<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234005-virtual-representations-of-nucleotide-sequences by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:01:08 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234005:VIRTUAL REPRESENTATIONS OF NUCLEOTIDE SEQUENCES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">VIRTUAL REPRESENTATIONS OF NUCLEOTIDE SEQUENCES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides oligonucleotide probes that can be used to hybridize to a representation of nucleic acid sequences. Compositions containing the probes such as microarrays are also provided. The invention also provides methods of using these probes and compositions in therapeutic, diagnostic, and research applications.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>VIRTUAL REPRESENTATIONS OF NUCLEOUDE SEQUENCES<br>
FIELD OF THE INVENTION<br>
[0001] This invention relates generally to molecular biology. More specifically,<br>
this invention relates to materials and methods for generating nucleotide sequences<br>
that are representative of a given source DNA (e.g., a genome).<br>
BACKGROUND OF THE INVFNTION<br>
[0002] Global methods for genomic analysis have provided useful insights into<br>
the pathophysiology of cancer and other diseases or conditions with a genetic<br>
component. Such methods include karyotyping, determination of ploidy,<br>
comparative genomic hybridizaton (CGH), representational difference analysis<br>
(RDA) (see, e.g., United States Patent 5,436,142), and analysis of genomic<br>
representations (WO 99/23256, published May 14,1999). Generally, these<br>
methods involve either using probes to interrogate the exprasstonof particular<br>
genes or examining changes in the genome itself.<br>
[0003] Using oligonucleotide arrays, these methods can be used to obtain a high<br>
resolution global image of genetic changes in cells. However, these methods<br>
require knowledge of the sequences of the particular probes. This is particularly<br>
limiting for cDNA arrays because such arrays only interrogate a limited set of<br>
genes. They also are limiting for genome-wide screening because many<br>
oligonucleotides designed for an array may be unrepresented in the interrogated<br>
population, resulting in inefficient or ineffective analysis.<br>
SUMMARY OF THE INVENTION<br>
[0004] This invention provides compositions and methods useful for<br>
interrogating populations of nucleic acid molecules. These compositions and<br>
methods can be used to analyze complex genomes (e.g., mammalian genomes),<br>
optionally in conjunction with the microarray technology. This invention features<br>
a plurality of at least 100 nucleic acid molecules (A) where (a) each of the nucleic<br>
acid molecules hybridizes specifically to a sequence in a genome of at least Z<br>
basepairs; and (b) at least P% of said plurality of nucleic acid molecules have-(i) a<br>
length of at least K nucleotides; (ii) hybridizes specifically to at least one nucleic<br>
acid molecule present in or predicted to be present in a representation derived from<br>
said genome, said representation having no more than R% of the complexity of<br>
said genome; and (iii) no more than X exact matches of L1 nucleotides to said<br>
genome (or said representation) and no fewer than Y exact matches of LI<br>
nucleotides to said genome (or said representation); and (B) where (a) Z =1 x 108;<br>
(b) 300 = K = 30; (c) 70 = R &gt; 0.001 ;(d)P = 90-R; (e) the integer closest to<br>
(log4{Z) + 2) = L1= the integer closest to log4(Z); (f) X is the integer closest to<br>
D1 x (K- L1+1); (g) Y is the integer closest to D2 x (K- L1+1); (h) 1.5 = D1= 1;<br>
and(i)1= D2 = 0.5.<br>
[0005] In some further embodiments, (1) the plurality of nucleic acid molecules<br>
comprises at least 500; 1,000; 2,500; 5,000; 10,000; 25,000; 50,000; 85,000;<br>
190,000; 350,000; or 550,000 nucleic acid molecules; (2) Z is at least 3 x 108,1 x<br>
109,1 x 1010 or 1 x 1011; {3) R is 0.001,1,2,4,10,15,20, 30,40,50 or 70;(4) P is<br>
independent of R and is at feast 70, 80,90,95,97 or 99; (5) D1 is 1 ;(6) L1 is 15,<br>
16,17,18,19,20,21,22,23 or 24;(7) P is 91,92,93,94,95,96,97,98,99 or<br>
100; and/or (8) K is 40,50, 60,70,80,90,100,110,120,140,160,180,200 or<br>
250. In some embodiments, a nucleic acid molecule that hybridizes specifically to<br>
another nucleic acid molecule has at least 90% sequence identity to a-sequence of<br>
the same length in the other nucleic acid molecule, In further embodiments, it has<br>
at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at<br>
least 97%, at least 98%, at least 99%, or 100% sequence identity.<br>
[0006] In some further embodiments, each of said P% of said plurality of nucleic<br>
acid molecules further have no more than A exact matches of L2 nucleotides to<br>
said genome and no fewer than B exact matches of L2 nucleotides to said genome,<br>
wherein (a) L1 = L2 = the integer closest to log4(Z) - 3, (b) A is the integer closest<br>
to D3 x ((K - L2+l) x (Z / 4L2)); (c) B is the integer closest to D4 x ((K - L2+1) x (Z<br>
/ 4L2)); (d) 4 = D3 = 1; and (e) 1 = D4 = 0.5.<br>
[0007] A representation of a DNA population can be produced by sequence-<br>
specific cleavage of said genome, e.g., accomplished with a restriction<br>
endonuclease. It can also be derived from another representation. That is, the<br>
resultant representation is a compound representation.<br>
[0008] The nucleic acid molecules of this invention can be identified by a<br>
method comprising: (a) cleaving said genome in silico with a restriction-enzyme to<br>
generate a plurality of predicted nucleic acid molecules; (b) generating a virtual<br>
representation of said genome by identifying predicted nucleic acid molecules -each<br>
having a length of 200-1,200 basepairs, inclusive, said virtual representation<br>
having a complexity of 0.001%-70%, inclusive, of said genome; (c) selecting an<br>
oligonucleotide having a length of 30-300 nucleotides, inclusive, and at least 90%<br>
sequence identity to a predicted nucleic acid molecule in"(b); (d) calculating the<br>
complexity of said virtual representation relative to said genome; (e) identifying all<br>
of the stretches of L1 nucleotides occurring in said oligonucleotide; and (f)<br>
confirming that the number of times each of said stretches occurs in said genome<br>
satisfies the various predetermined requirements.<br>
[0009] The nucleic acid molecules of this invention can be used as probes for<br>
analyzing sample DNA. These probes can be immobilized on the surface of a-solid<br>
phase, mcluding a semi-solid surface. Solid phases include, without limitation,<br>
nylon membranes, nitrocellulose membranes, glass slides, and microspheres (e.g.,<br>
paramagnetic microbeads). In some embodiments, the positions of the nucleic acid<br>
molecules on said solid phase are known, e.g., as used in a microarray format.<br>
The invention also features a method of analyzing a nucleic acid sample (e.g., a<br>
genomic representation), said method comprising (a) hybridizing the sample to the<br>
nucleic acid probes of this invention; and (b) determining to which of said plurality<br>
of nucleic acid molecules said sample hybridizes.<br>
[0010] This invention features also a method of analyzing copy number variation<br>
of a genomic sequence between two genomes, said method comprising: (a)<br>
providing two detectably labeled representations, each prepared from the<br>
respective genomes with at least one identical restriction enzyme; (b) contacting<br>
these two representations with the nucleic acid probes of this invention to allow<br>
hybridization between the representations and the probes; (c) analyzing the<br>
hybridization levels of the two representations to the probe set, wherein a<br>
difference in said levels to a member of the probe set indicates a copy number<br>
variation between the two genomes with regard to a genomic sequence targeted by<br>
said member. In some embodiments, the representations are distinguishably<br>
labeled; and/or the contacting of the two representations is simultaneous.<br>
[0011 ] This invention further features a method of comparing methylation status<br>
of a genomic sequence between two genomes, said method involves providing two<br>
detectably labeled representations from the respective genomes, each<br>
representation being prepared by a methylation sensitive method. For instance, a<br>
first representation of a first genome is prepared using a first restriction enzyme<br>
and a second representation of a second genome is prepared using a second<br>
restriction enzyme, wherein said first and second restriction enzymes recognize the<br>
same restriction site but one is methylation-sensitive and the other is not.<br>
Sequences with methyl-C can also be chemically cleaved after making a<br>
representation with a non-methylation sensitive restriction-enzyme, such that a<br>
representation derived from a methylated genome is distinguishable from a<br>
representation derived from a non-methylated genome. Then the two<br>
representations are contacted with the probes of this invention to allow<br>
hybridization between the representations and the probes. Hybridisation of the two<br>
representations to the probes is then analyzed, where a difference in hybridization<br>
levels between the representations with regard to a particular probe indicates a<br>
difference in methylation status between the two genomes with regard to a<br>
genomic sequence targeted by said probe.<br>
[0012] Similar methods can also be used to analyze polymorphism of a complex<br>
genome, as further illustrated below.<br>
[0013] In accordance with certain embodiments of the invention, an algorithm is<br>
provided for accurately and efficiently detecting and counting the number of times<br>
a word occurs in a genome. This algorithm, sometimes referred to herein as a<br>
search engine or a mer-engine, uses a transform of a genome (e.g., a Burrows-<br>
Wheeler Transform) and an auxiliary data structure to count the number of times a<br>
particular word occurs in the genome. A "word" refers to a nucleotide sequence of<br>
a defined length.<br>
[0014] In general, the engine searches for a particular word by first finding the<br>
last character of the word. Then it proceeds to look for the character immediately<br>
preceding the last character. If the first immediately preceding character is found,<br>
it then looks for the second immediately preceding character to the last character of<br>
the word, and so on until the word is found If further preceding characters are not<br>
found, it will be concluded that the word does not exist in the genome. If the first<br>
character of the word is found, then the number of times it occurs is the word count<br>
of that particular word.<br>
[0015] This particular algorithm is advantageous because it can be used to<br>
implement several practical applications involving genomic studies, as discussed<br>
below.<br>
[0016] Other features and advantages of the invention will be apparent from the<br>
following drawings, detailed description and claims.<br>
ACCOMPANYING<br>
BRIEF DESCRIPTION OF TEDDRAWINGS<br>
[0017] Figs. 1 A-1D demonstrate the predictability of informatics and accuracy of<br>
the array measurements using microarrays comprising 10,000 oligonucleotides.<br>
Fig. 1A shows the results where the samples hybridized are aBglII. representation<br>
and a RgIII representation depleted of fragments with a HindIII cleavage site. The<br>
Y-axis (Mean Ratio) is the mean measured ratio from two hybridizations of<br>
depleted representation to normal representation plotted in log scale. The X-axis<br>
(Index) is a false index constructed such that probes deriving from fragments<br>
defined as having an internal hindIII site are to the right side. Fig. 1B shows the<br>
reproducibility of the duplicate experiments used to generate the average ratio in<br>
Fig. 1 A. The Y-axis (Ratio Expl) is the measured ratio from experiment 1 and the<br>
X-axis (Ratio Exp2) is the measured ratio of experiment 2. Both axes are plotted<br>
in log scale. Fig. 1C graphs the normalized ratio on the Y-axis as a function of<br>
intensity of the sample that was not depleted on the X-axis. Both the ratio and<br>
intensity were plotted in log scale. Fig. 1Drepoesents datagenerated by<br>
simulation. The X-axis (Index) is a false index. Probes, in groups of 600, detect<br>
increasing copy number, from left to right. 600 flanking probes detect normal<br>
copy number. The Y-axis (Mean Ratio) is mean ratio plotted on a log scale.<br>
[0018] Figs. 2A1-2A3,2B1-2B3, and 2C1-2C3 show the genomic profiles for a<br>
primary breast cancer sample {CHTN159), with aneuploid nuclei compared to<br>
diploid nuclei from the same patient (Fig. 2A1-2A3), a breast cancer cell line<br>
compared to a normal male reference (Fig. 2B1-2B3), and a normal male to a<br>
normal male reference (Fig. 2C1-2C3), using the 10K printed array (Fig. 2A1, Fig.<br>
2B1, Fig. 2C1) and the 85K photoprint array (Fig. 2A2, Fig. 2B2, Fig. 2C2). In<br>
each case (Fig. 2A1, Fig. 2B1, Fig. 2C1 and Fig. 2A2, Fig. 2B2, Fig. 2C2) the Y-<br>
axis is the mean ratio, and the X-axis (Gen Index) is an index, which plots the<br>
probes in genomic order, concatenating the chromosomes, and allowing the<br>
visualization of the entire genome from chromosome 1 to Y. Fig. 2A3, Fig. 2B3,<br>
and Fig. 2C3 show the correspondence of the ratios measured from "brother"<br>
probes present in the 1 OK and the 85K microarrays. The Y-axis is the measured<br>
ratio from the 10K microarray and the X-axis is the measured ratio from the 85K<br>
microarray.<br>
(0019J Figs. 3A-3D show several chromosomes with varying copy number<br>
fluctuations from analysis of the tumor cell line SK-BR-3 as compared to the<br>
normal reference. The Y-axes (Mean Ratio) represents the mean ratio-of two<br>
hybridizations in log scale. The X-axes (Gen Index) is an index of the genomic<br>
coordinates. Fig. 3 A represents copy number fluctuations identified for<br>
chromosome 5, Fig. 3B for chromosome 8, fig. 3C for chromosome iy-andTig.<br>
3D for the X chromosome.<br>
[0020] Figs. 4A-4D show the mean segmentation 'calculated from the analysis of<br>
SK-BR-3 compared to the normal reference (Fig. 4A andT% 4B) and CHTN139<br>
(Fig. 4C and Fig. 4D). In Figs. 4A-4D, the Y-axis is the value of the mean<br>
segment for each probe in log scale. In Fig. 4A and Fig. AC, the X-axis (Mean<br>
Segment Index) is each listed in ascending value of their assigned mean'segment.<br>
In Fig. 4B and Fig. 4D the X-axis (Gen Index) is a genomic index, which, as<br>
described above, places the entire genome end to end. Plotted on top of the mean<br>
segment data is a copy number lattice extrapolated from the array data using<br>
formulas within the text (horizontal lines). Calculated copy number for each<br>
horizontal line is to the right of the lattice.<br>
[0021] Figs. 5 A-5D graph on the Y-axis (Mean Ratio SK-BR-3) the mean ratio<br>
of two hybridizations of SK-BR-3 compared to a normal reference in log scale.<br>
The X-axis (Gen Index) is a genomic index. Fig. 5A shows a region from X<br>
chromosome with a region of loss. Plotted over the measured array ratio is the<br>
calculated segmentation value. Fig. 5B shows a region of chromosome 8 (c-myc<br>
located to the right of the center of the graph) from results of SK-BR-3 in<br>
comparison to normal reference. Plotted on top of the data are the segmentation<br>
values for SK-BR-3 in comparison to normal reference in diagonal hatch and the<br>
segmentation values for the primary tumor CHTN159 in vertical hatch. Fig. 5C<br>
shows a lesion on chromosome 5 demonstrating the resolving power of the 85K as<br>
compared to the 10K array. Results are from SK-BR-3 compared to a normal<br>
reference. Open circles are from the 10K printed microarray and filled cilcles are<br>
from the 85K photoprint array. Horizontal lines are copy number estimates, based<br>
on modeling from mean segment values. Fig. 5D shows comparison of SK-BR-3<br>
to normal reference, displaying a region of homozygous deletion on chromosome<br>
19. The mean segment value is plotted as a white line, and the lattice are copy<br>
number estimates as described above.<br>
[0022] Figs. 6A-6D show the results of a normal compared to a normal, identical<br>
to that displayed in Fig. 2C2 with the exception that singlet probes have been<br>
filtered as described in the text. Fig. 6B illustrates the serial comparison of<br>
experimentsforasmall region from chromosome-4. The Y-axis is ihe mean ratio<br>
in log scale. The X-axis is a genomic index. The filled (85K) and open (10K)<br>
circles are from the comparison of SK-BR-3 to normal. The empty triangles are a<br>
comparison of a pygmy to the normal reference. Fig. 6C illustrates a lesion found<br>
in the normal population on chromosome 6. The filled circles are plotted by mean<br>
ratio for analysis of the pygmy to the normal reference. The vertical hatch line is<br>
the mean segment value for the pygmy to normal reference comparison. The<br>
diagonal hatch line is the mean segment value for the SK-3-BR-3 to normal<br>
reference comparison. The cross hatch line is the segment value from the primary<br>
tumor (CHTN159 aneuploid to diploid) comparison. Fig. 6D -shows a region of<br>
chromosome 2. The data shown in circles is from the comparison of SK-BR-3 to<br>
normal reference. The mean segment line for this comparison is shown in vertical<br>
hatch. The mean segment line for the comparison of a pygmy to the normal<br>
reference is shown in diagonal hatch and for the primary tumor CHTN159 in cross<br>
hatch. For Fig. 6C and Fig. 6D the calculated copy number for the horizontal lines<br>
is found to the right of the panel.<br>
[0023] Fig. 7 shows a block diagram of an illustrative system in accordance with<br>
certain embodiments of the invention.<br>
[0024] Fig. 8 shows a flow chart of an illustrative pre-processing step for<br>
performing exact word counts in accordance with certain embodiments of the<br>
invention.<br>
[0025] Figs. 9 A and 9B show a flow chart of an illustrative word counting<br>
algorithm in accordance with certain embodiments of the invention.<br>
[0026] Figs. 10A and 10B show an illustrative example of word counting<br>
algorithm of Figs. 9A and 9B in accordance with certain embodiments of the<br>
invention.<br>
[0027] Fig. 11 shows an illustrative suffix array having coordinate positions<br>
corresponding to the coordinates of the genome in accordance with certain<br>
embodiments of the invention.<br>
[0028] Fig. 12A shows a graphical representation of the variables and data<br>
structures used in connection with the algorithm according to certain embodiment<br>
of the invention.<br>
[0029] Fig. 12B shows a pseudo code representation of the algorithm aecording<br>
to certain embodiments of the invention.<br>
DETAILED DESCRIPTION Of THE INVENTION<br>
[0030] This invention features oligonucleotide probes /or analyzing<br>
representations of a DNA population (e.g., a genome, a chromosome or a mixture<br>
of DNAs). The oligonucleotide probes may be used in solution or they may be<br>
immobilized on a solid (including semi-solid) surface such as an array or a<br>
microbead (e.g., Lechner et al., Curr. Opin. Chem. Biol. 6:31-38 "(2001); Kwok,<br>
Annu. Rev. Genomics Human Genet 2:235-53 (2001); Aebersold et al., Nature<br>
422:198-207 (2003); and United States Patents 6,355,431 and 6,-429,027). A<br>
representation is a reproducible sampling of a DNA population in which the<br>
resulting DNA typically has a new format or reduced complexity or both (Lisitsyn<br>
et al., Science 258:946-51 (1993); Lucito et al., Proc. Natl. Acad. Sci. USA<br>
92:151-5(1998)). For example, a representation of a genome may consist of DNA<br>
sequences that are from only a small portion of the genome and are largely free of<br>
repetitive sequences. Analysis of genomic representations may reveal changes in a<br>
genome, including mutations such as deletions, amplifications, chromosomal<br>
rearrangements, and polymorphisms. When done in a clinical setting, the analysis<br>
can provide insight into the molecular basis of a disease as well as useful guides to<br>
its diagnosis and treatment.<br>
[0031] The oligonucleotide compositions of this invention can be used to<br>
hybridize to representations of a source DNA, where hybridization data are<br>
processed to provide generic profiles of the source DNA (e.g., disease-related<br>
genetic lesions and polymorphisms). It may be preferred that the representations<br>
(or "test representations" hereinafter) and at least a fraction of the oligonucleotide<br>
probes in the compositions are derived from the same species. DNA from any<br>
species may be utilized, including mammalian species (e.g., pig, mouse, rat,<br>
primate (e.g., human), dog, and cat), species of fish, species of reptiles, species of<br>
plants and species of microorganisms.<br>
I. OLIGONUCLEOTIDE PROBES<br>
[0032] The oligonucleotide probes of this invention are preferably designed by<br>
virtual representation of a source DNA, such as the genomic DNA of a reference<br>
individual Representation of the genome generally, but not invariably, results in a<br>
simplification of its complexity. The complexity of a reptesentation corresponds<br>
to the fraction of the genome that is represented therein. One way to calculate<br>
complexity is to divide the number of nucleotides in the representation by the<br>
number of nucleotides in the genome. The genomic complexity of a representation<br>
can range from below 1 % to as high as 95% of the total genome. Where DNA<br>
from an organism with a relatively simple genome is used, the representation may<br>
have a complexity of 100% of the total genome, e.g., the representation may be<br>
generated by restriction digest of total DNA without amplification.<br>
Representations associated with the invention typically have a complexity of<br>
between 0.001 % and 70%. Reduction of complexity allows for desirable<br>
hybridization kinetics.<br>
[0033J An "actual" representation of DNA involves laboratory procedures ("wet<br>
work") by which representational DNAs are selected. Virtual representations, on<br>
the other hand, take advantage of the fact that complete genomes, for example, the<br>
human genome, have been sequenced. Through computational analysis of the<br>
available genomic sequences, one can readily design a large number of<br>
oligonucleotide probes that hybridize to mapped regions of the genome and have a<br>
minimal degree of sequence overlap to the rest of the genome.<br>
[0034] By way of example, to design a set of oligonucleotide probes for human<br>
genetic analysis, one can perform an in silico (i.e., virtual) digestion of the human<br>
genome by locating all cleavage sites of a selected restriction endonuclease in the<br>
sequenced genome. One can then analyze the resulting fragments to identify those<br>
that are in a desired range (e.g., 200 - 1,200 bps, 100 - 400 bps and 400 - 600 bps)<br>
that can be amplified by, e.g., PCR. Such fragments are defined herein as<br>
"predicted to be present" in a representation. A restriction endonuclease may be<br>
selected based on the complexity of the representation desired. For example,<br>
restriction endonucleases that cut infrequently, such as those that recognize 6 bp or<br>
8 bp target sequences, will produce representations of lower complexity, whereas<br>
restriction endonucleases that cut frequently, such as those that recognize 4 bp<br>
target sequences, will produce representations of higher complexity. In addition,<br>
factors such as the G/C content of the genome analyzed will affect the frequency of<br>
cleavage of particular restriction endonucleases and consequently influence the<br>
selection of the restriction endonucleases. Generally, robust restriction<br>
endonucleases that do not exhibit star activity are used. Alternatively, cleavage<br>
based on methylation state of a target site may also be employed, e.g. through the<br>
use of a methylation-sensitive restriction enzyme or other enzyme such as McrBC,<br>
which recognizes methylated cytosines in DNA<br>
[0035] Sequences of all digested fragments of a desired range (e.g., 200 -1,200<br>
bps, 100 - 400 bps and 400 - 600 bps) are analyzed by computer, where regions of<br>
some of these fragments that are at least about 30 bps in length and have minimal<br>
homology to the rest of the genome can be selected as representational<br>
oligonucleotide probes for the human genome. Examples 1 and Section VI below<br>
further illustrate methods of identifying the oligonucleotides of this invention.<br>
[0036] Oligonucleotides of the invention may range in length from about 30<br>
nucleotides to about 1,200 nucleotides. The exact length of oligonucleotides<br>
chosen will depend on the intended use, e.g., the size of the source DNA from<br>
which the representation is prepared and whether they are used as components of<br>
an array. The oligonucleotides typically have a length of at least 35 nudeotides,<br>
e.g., at least 40,45,50,55,60,65,70,75, 80,85,90,95 or 100 nucleotides, but<br>
they may also be shorter having a length of, e.g., 20,21,22,23, 24,25,26,27,28,<br>
29 or 30 nucleotides. The oligonucleotides typically have a length of no more than<br>
600 nucleotides, e.g., no more than 550,500,450,400,350,300,250,200 or 150<br>
nucleotides. As would be recognized by one of skill in the art, the length of the<br>
oligonucleotides will depend on features of the genome analyzed, e.g., complexity<br>
and amount of repetitive sequences.<br>
II. OLIGONUCLEOTIDE ARRAYS<br>
[0037] The oligonucleotide probes of this invention can be used in an array<br>
format. An array comprises a solid support with nucleic acid probes attached<br>
thereto at defined coordinates, or addresses. Each address contains either many<br>
copies of a single DNA probe, or a mixture of different DNA probes. Nucleic acid<br>
arrays, also termed "microarrays" or "chips," have been generally described in the<br>
art. See, e.g., United States Patent 6,361,947 and references cited therein. We<br>
have termed genetic analysis using the new arrays "representational<br>
oligonucleotidemicroarray analysis" ("ROMA") or, where cleavage depends on<br>
methylation at the target site, "methyiation detection oligonucleotide microarray<br>
analysis" ("MOMA").<br>
[0038] To manufacture a microarray of this invention, pre-synthesized<br>
oligonucleotides are attached to a solid support, which may be made from glass,<br>
plastic (e.g., polypropylene or nylon), polyacrylamide, nitrocellulose, or other<br>
materials, and may be porous or nonporous. One method for attaching the nucleic<br>
acids to a surface is by printing on glass plates, as is described generally by Schena<br>
et al., Science 270:467-70 (1995); DeRisi et al., Nature Gen. 14:437450 (1996);<br>
Shalon et al., Genome Res. 6:639-45 (1996); and Schena et al., Proc. Natl. Acad.<br>
Sci. USA 93:10539-1286 (1995). For low density arrays, one can also use dot<br>
blots on a nylon hybridization membrane. See, e.g., Sambrook et al., Molecular<br>
Cloning - A Laboratory Manual (2nd Ed.), Vol. 1-3, Cold Spring Harbor<br>
Laboratory, Cold Spring Harbor, New York, 1989.<br>
[0039] Another method for making microarrays is by using photolithographic (or<br>
"photoprint") techniques to synthesize oligonucleotides directly on the array<br>
substrate, i.e., in situ. See, e.g., Fodor et al., Science 251:767-73 (1991); Pease-et<br>
al., Proc. Natl. Acad. Sci. USA 91:5022-6 (1994); Lipschutz et al., Nat Genet.<br>
21(1 Suppl):20-46 (1999); Nuwaysir et al., t3enome Res. 12(11): 1749-55 (2002);<br>
Albert et al., Nucl. Acids Res. 31(7):e35 (2003); and United States Patents<br>
5,578,832,5,556,752, and 5,510,270. Other methods for rapid synthesis and<br>
deposition of defined oligonucleotides can also be used. See, e.g., Blanchard et al.,<br>
Biosensors &amp; Bioelectronics 11:687-90 (1996); and Maskos and Southern, Nucl.<br>
Acids Res. 20:1679-1684 (1992).<br>
[0040] The arrays of the invention typically comprise at least 100 (e.g., at least<br>
500,1,000,5,000 or 10,000) oligonucleotide probes, and may comprise many more<br>
probes, for example, up to 25,000,50,000, 75,000,85,000, 100,000,200,000,<br>
250,000,500,000 or 700,000 probes. The arrays of the invention typically do not<br>
comprise more than 700,000 probes. However, they may comprise more, e.g., up<br>
to 800,000,900,000 or 1,000,000 probes. In some embodiments, the arrays are<br>
high density arrays with densities greater than about 60 different probes per 1 cm2.<br>
The oligonucleotides in the arrays may be single-stranded or double-stranded To<br>
facilitate manufacturing and use of the arrays, the oligonucleotide probes of this<br>
invention may be modified by, e.g., incorporating peptidyl structures and analog<br>
nucleotides, into the probes.<br>
HI. TEST REPRESENTATIONS<br>
[0041] The oligonucleotide arrays of this invention can be used to probe any<br>
nucleic acid sample of choice. For example, the sample may be a cDNA library, a<br>
genomic DNA library, or an RNA preparation. In other embodiments, the Arrays<br>
of this invention are used to probe DNA samples that are representations (or "test<br>
representations") of a complex DNA population, such as the genome of a higher<br>
organism.<br>
[0042] Representations and methods for their preparation are described in, e.g.,<br>
Lisitsyn et al., Proc. Natl. Acad. Sci. USA 92:151 (1995); Lucito et al., Proc. Natl.<br>
Acad, Sci. USA 95:4487-4492 (1998); and WO 99/23256. One approach for<br>
making a representation involves reproducibly cleaving a DNA population into<br>
fragments. Reproducible cleavage is generally accomplished by digesting with one<br>
or more restriction endonucleases (e.g. Dpnl or BgIII) or enzyme(s) that cleave at<br>
particular methylated sites (e.g. McrBC), but any method that reproducibly cleaves<br>
the DNA may be used. The resulting DNA fragments are linked to adaptor<br>
oligonucleotides. These fragments are then amplified by, e.g., polymerase chain<br>
reaction ("PCR") or ligase chain reaction, using primers complementary to the<br>
adaptors. The amplified fragments represent a subset of the starter DNA<br>
population. Due to the amplification step, representations can be made from very<br>
small amounts of starting material (e.g., from 5 ng of DNA). Representational<br>
difference analysis ("RDA") as described in Lisitsyn et al., Science 258:946-51<br>
(1993) and United States Patents 5,436,142 and 5,501,964 may be used to remove<br>
any known, unwanted sequences from the representation, including repetitive<br>
sequences.<br>
[0043] The starter DNA population can be large DNA molecules such as the<br>
genome of an organism, or a part thereof (e.g., a chromosome or region thereof).<br>
We refer to representations of such a DNA population as chromosomal or enbmic<br>
representations, respectively. The starter DNA populations can be obtained from,<br>
e.g., diseased tissue samples such as tumor biopsy samples, normal tissue samples,<br>
tumor cell lines, normal cell lines, cells stored as fixed specimens, autopsy<br>
samples, forensic samples, paleo DNA samples, microdissected tissue samples,<br>
isolated nuclei, isolated chromosomes or regions of chromosomes, and'fractionated<br>
cell or tissue samples. One can also make a representation of a representation (or a<br>
"compound representation"). Compound representations are useful for screening<br>
for polymorphisms. See, e.g., WO 99/23256.<br>
[0044] For comparative analysis of representations from two DNA sourees, such<br>
as comparing a genomic representation from a normal-cell with a genomic<br>
representation from a cancerous or otherwise diseased cell, it may be preferred to<br>
prepare the two representations in parallel, e.g., isolating the starter DNA from the<br>
two cells at the same time and in the same manner, preparing the representations<br>
from the same amount of starter DNA, and amplifying DNA fragments at the same<br>
time under the same conditions in the same thermal cycler. It may also be<br>
preferred that the normal cell and the diseased cell are taken from the same<br>
individual, although it is possible to obtain "normal" genomic DNA by combining,<br>
e.g., the DNA from both parents of the individual.<br>
[0045] The complexity of a representation is generally lower than that of the<br>
starter DNA population, because there are sequences present in the starter<br>
population that are not present in the representation. The complexity of a<br>
representation is related to the cutting frequency of the restriction endonuclease in<br>
a particular starter population. A more frequent cutter gives rise to a more<br>
complex representation. Because fragments between 200 —1,200 basepairs are<br>
preferentially amplified by PCR under typical conditions, one can obtain high<br>
complexity representations by cleaving the starter DNA such that a majority of the<br>
fragments are between 200 - 1,200 basepairs. Conversely, low complexity<br>
representations can be obtained by cleaving the DNA molecule such that fewer of<br>
the fragments are between 200 - 1,200 basepairs. For example, DpnII digestion of<br>
human genomic DNA may give rise to a representation having about 70% of the<br>
complexity of the entire human genome. Digestion by a less frequent cutter such<br>
as BamHI or BglII, on the other hand, may give rise to a representation having only<br>
about 2% of the complexity of the human genome. High complexity<br>
representations are useful for, e.g., determining gene copy number, deletion<br>
mapping, determining loss of heterozygosity, comparative genomic hybridization,<br>
and archiving of DNA. Generally, low complexity representations are useful for<br>
the same purposes, but give better hybridization kinetics than high complexity<br>
representations.<br>
[0046] The complexity of a representation may be further fine-tuned by using<br>
more than one restriction enzyme to generate fragments prior to ligation of the<br>
adaptors, and/or by using one or more additional restriction enzymes to cleave a<br>
subset of the fragments after ligation of the adaptors, thus depleting the resulting<br>
representation of those fragments. Any restriction enzyme, including methylation<br>
sensitive restriction enzymes, may be used to produce a representation for analysis<br>
as described herein.<br>
10047] The complexity of the representation also can be shaped by the choice of<br>
adaptors used for amplification. For example, which adaptors are used can<br>
influence the sizes of members of a representation. Where identical adaptors are<br>
ligated to both ends of the cleaved fragments, panhandle formation between<br>
adaptors within the single strands competes with primer annealing, thus inhibiting<br>
amplification by PCR. See Lukyanov et al., Anal. Biochem. 229:198-202 (1995).<br>
Amplification of shorter fragments is more likely to be inhibited because the<br>
adaptors are closer to each other in shorter fragments, resulting in a higher<br>
effective local concentration of the ligated adaptors and, thus, greater interaction.<br>
Adaptors that form panhandles of approximately 29 basepairs allow for<br>
amplification of fragments in the size range of 200-1,200 basepairs. Adaptors that<br>
form shorter panhandles, e.g., 24 basepairs, release some of the inhibition of the<br>
smaller fragments, resulting in the favoring of smaller PCR amplification products<br>
and therefore, a representation of altered complexity.<br>
IV. HYBRIDIZATION OF NUCLEIC ACID SAMPLES TO ARRAYS<br>
[0048] The mieroarrays of this invention are typically hybridized to samples of<br>
single-stranded nucleic acids in solution. Because the potential hybridization<br>
signal may vary from address to address in the hybridization chamber, the probe<br>
array may preferably be used as a comparator, measuring the ratio of hybridization<br>
between two differently labeled specimens (the sample) that are thoroughly mixed<br>
and therefore share the same hybridization conditions. Typically the two<br>
specimens will be from test (e.g., diseased) and control (e.g., disease-free) cells,<br>
respectively.<br>
[0049] Samples to be hybridized to the microarrays, e.g., the above-described<br>
test representations, can be detectably labeled by any means known to one of-skill<br>
in the art In some embodiments, the sample is labeled with a fluorescent moiety<br>
by, for example, random primer labeling or nick translation. When the sample is a<br>
representation, it may be labeled during the amplification step by inclusion of<br>
labeled nucleotides in the reaction. The fluorescent label may be, for example, a<br>
lissamine-conjugated nucleotide or a fluorescein-conjugated nuclotide analog. In<br>
some embodiments, two differentially labeled samples (e.g., one labeled with<br>
lissamine and the other with fluorescein) are used. In some embodiments, the<br>
samples are unlabeled.<br>
[0050] Hybridization and wash conditions are chosen such that the nucleic acid<br>
molecules in the sample specifically bind to complementary oligonucleotides on<br>
the array. Arrays containing double stranded oligonucleotides are generally<br>
subjected to denaturing conditions to Fender the oligonucleotides single-stranded<br>
prior to contacting with the sample. Optimal hybridization conditions will depend<br>
on the length and type (e.g., RNA orDNA) of the oligonucleotide probe and<br>
sample nucleic acids.<br>
[0051] Hybridization to an array of the invention may be detected by any method<br>
known to those of skill in the art. In some embodiments, the hybridization of<br>
flourescently labeled sample nucleotides is detected by laser scanner. In some<br>
embodiments, the hybridization of labeled or unlabeled sample nucleotides is<br>
detecting by measuring their mass. When two different fluofcesoent labels are used,<br>
the scanner may be one that is able to detect fluorescence of more than one<br>
wavelength, the wavelength corresponding to that of each fluorescent label,<br>
typically simultaneously or nearly simultaneously.<br>
V. EXEMPLARY USES FOR THE OLIGONUCLEOTIDE PROBES<br>
[0052] The oligonucleotide probes of the invention may be used to detect and<br>
quantify the changes in the copy number or methylation status of specific<br>
sequences in a genome. Where representations derived from a plurality oFDNA<br>
samples are hybridized to the same oligonucleotide probes, the relative intensity of<br>
hybridization between the two samples to a particular probe is mdiieative of the<br>
relative copy number or methylarion status of the sequence corresponding to that<br>
probe in the two samples. Genomes, for example, typically contain either extra<br>
copies of certain sequences due to amplification or fewer or no copies of eertain<br>
sequences due to deletion of specific regions. These methods can be used, for<br>
example, to analyze changes in the copy of number or methylation status of<br>
sequences between a reference sample and patient samples where amplification,<br>
deletion or methylation status of specific sequences is implicated in,for example,<br>
the predisposition, progression or staging of specific diseases including, for<br>
example, cancer, neurological diseases (e.g., autism), diabetes, cardiological<br>
diseases and inflammatory diseases (e.g., autoimmune diseases).<br>
[0053] In addition, positional information on the alteration of copy number or<br>
methylation status in a genome can be obtained because the sequences in the<br>
genome to which the oligonucleotide probes of the invention are complementary<br>
are known. Where the oligonucleotide probes are designed to hybridize frequently<br>
in the genome sequence and the sample is a high complexity representation, it is<br>
possible to map precisely regions of genome amplification, deletion or methylation<br>
status. Thus, the invention may be used to identify individual genes that may be<br>
involved in the predisposition, progression or staging of specific diseases. These<br>
genes may be oncogenes and tumor suppressor genes depending on whether the<br>
sequence is amplified, deleted or methylated/unmethylated in a cancer genome<br>
relative to a reference genome, respectively.<br>
[0054] The oligonucleotide probes of the invention also may be used to identify<br>
polymorphic sites, including single nucleotide polymorphisms (SNPs), both within<br>
an individual and between individuals. These polymorphisms are common and as<br>
many as 2-3% of oligonucleotide probes display polymorphic behavior even<br>
between "normal" individuals. Detectable polymorphisms may result from the loss<br>
or gain of restriction endonuclease fragments, e.g., due to point mutations,<br>
deletions, genomic rearrangements or gene conversions extending over<br>
heterozygous polymorphisms, where they are reflected in their presence or absence<br>
in a representation. For example, digestion of the a nuoleotide "sequence with a<br>
restriction enzyme may result in one large (i.e., uncleaved) or two small fragments<br>
depending on whether a restriction site is present. This polymorphic restriction site<br>
is known to exist in a test genome if the oligonucleotide probes detect one or both<br>
of the small fragments in the test representation.<br>
[0055] Similarly, genomic rearrangements, including translocations, insertions,<br>
inversions and deletions, may result in the creation of new resstriction endonuclease<br>
fragments spanning at least part of the rearrangement. Some of these new<br>
fragments may be ampliflable and, therefore, present in a representation of the<br>
rearranged genome but absent from a referenoe representation. Conversely,<br>
genomic rearrangements may result in the loss of a fragment from a repsesentation.<br>
In either case, a difference between the test and reference representations in<br>
hybridization to certain probes suggests that genomic rearrangements may have<br>
occurred in the test genome relative to the reference genome. By analyzing the<br>
sequences of these probes and the locations of these probes in the reference<br>
genome, one can obtain information on the genetic rearrangements, including the<br>
type of rearrangements and the junctions of the rearrangements.<br>
[0056] The ability to analyze copy number and other polymorphisms of specific<br>
sequences within and between individuals has many uses that will be apparent to<br>
one of skill in the art These may be, but are not limited to, identification of<br>
individuals, e.g., for forensic testing and paternity testing; breeding of plants or<br>
animals; discovery of polymorphisms that are genetically linked to an inherited<br>
trait, including the analysis of quantitative traits; determination of drug response in<br>
a patient, including predicting a beneficial or adverse response to a drug; diagnosis;<br>
and for patient identification and stratification in clinical trials.<br>
VI. AN EXEMPLARY SEARCH ENGINE<br>
[0057] The following describes an algorithm that can be used to obtain the<br>
above-mentioned oligonucleotide probes. It will be understood that the following<br>
description is not intended to show that this algorithm is the only means to obtain<br>
such probes. It will also be understood that this algorithm has applications other<br>
than generating the oligonucleotide probes of this invention. Some of the other<br>
applications are described herein.<br>
[0058] This algorithm, sometimes referred to herein as a search engine or a mer-<br>
engine, uses a transform of a genome (e.g., a Burrows-Wheeler- T'masfiorm) .and an<br>
auxiliary data structure to count the number of times a particular word occuts in<br>
the genome. A "word" refers to a nucleotide sequence of any length.<br>
[0059] In general, the engine searches for a particular word by first finding the<br>
last character of the word. Then it proceeds to look for the character immediately<br>
preceding the last character. If the first immediately preceding character is found,<br>
it then looks for the second immediately preceding character to the last-character of<br>
the word, and so on until the word is found. If further preceding characters are not<br>
found, it will be concluded that the word does not exist in the genome.<br>
[0060] This particular algorithm is advantageous because it can be used to<br>
implement several practical applications involving genomic studies, as discussed<br>
above. One application of the search engine is that it-can be used to annotate a<br>
nucleotide sequence such as a genome. Particularly, the genome can be annotated<br>
using substrings of a particular length that exist within the genome. The search<br>
engine can then count the number of times a particular length substring ogcurs in<br>
the genome. These counts provide an indication of the uniqueness of a particular<br>
substring, where lower counts represent a higher degree of uniqueness than higher<br>
counts.<br>
[0061] Probe design is another practical application that is advantageously<br>
enhanced by using the search engine. The engine's ability to rapidly count the<br>
number of times a particular word appears in a genome is particularly useful in<br>
designing probes that are unique and hybridize to a specific region of DNA, with<br>
minimal cross-hybridization. By using the search engine, potential cross-<br>
hybridizations can be minimized by requiring a probe to be comprised of<br>
constituent segments that are unique and meet certain stringency conditions "such as<br>
having low word counts or no word counts within the entire genome.<br>
[0052] Yet another application of the search-engine is to detect differences<br>
between two genomes. For example, as the human genome project progresses,<br>
new segments of the genome are mapped and released to the public. Using the<br>
search engine and probes that were designed on another version of the same<br>
genome, it can be determined how many of those probes can be applied to the new<br>
version of the genome.<br>
[0063] Yet still another application in which the search engine can be used is to<br>
verify whether a particular word exists in the enome. It may be desirable to find<br>
words that do not appear in the genome so that there is little chance that the word<br>
will hybridize to a section of the genome. These words may be generated<br>
randomly according to a predefined set of criteria. When a word is found, its<br>
complement is also submitted to the search engine to determine whether it appears<br>
in the genome. If both the word and its complement do not appear in the genome,<br>
it is known that both of these words will hybridize to each other and not to the<br>
genome.<br>
A. SYSTEM DESCRIPTION<br>
[0064] The search engine and applications thereof can be performed in<br>
accordance with the present invention using illustrative system 700 shown in<br>
Fig. 7. System 700 may include computer 710, user interface equipment 730,<br>
Internet 740, and optional laboratory equipment (not shown). System 700 may<br>
include multiple computers 710 and user interface equipment 730, but only one of<br>
each is illustrated in Fig. 7 to avoid complicating the drawing. Computer 710 is<br>
shown to be connected to user interface equipment 730, and Internet 740 via<br>
communication paths 790.<br>
[0065] Computer 710 may include circuitry such as a processor 712, database<br>
714 (e.g., a hard-drive), memory 716-(e.g., random-access-memory), and<br>
removable-media drive 718 (e.g., a floppy disk drive, a CD-ROM drive, or a DVD<br>
drive). This circuitry can be used to transmit data to, from, and/or between user<br>
interface equipment 730 and the Internet 740. Computer 710 may initiate<br>
techniques of the invention by responding to user input from user interface<br>
equipment 730. Computer 710 may also provide information to the user at user<br>
interface equipment 730 with respect to results obtained from operation of the<br>
search engine.<br>
[0066] Database 714 stores information that provides the search engine with<br>
data. More particularly, database 714 may include the"sequenee of a-genome or a<br>
particular portion of the genome. The invention may use the genome information<br>
stored on database 714 to construct a suffix array, which can also be stored on<br>
database 714. The suffix array is a data structure that is generated in preparation<br>
for constructing a transform of a genome or a portion thereof. Data representatiye<br>
of a .genome may be obtained, for example, from a readable medium(e.g., a floppy<br>
diskette, a CD-Rom, or a DVD) which can be accessed through removable-media<br>
drive 718. Alternatively, genome data may be obtained through Internet 740,<br>
where the data is transmitted from a server located, for example, at a research<br>
facility (e.g., the National Institutes of Health or a university). If desired, database<br>
714 may be updated with new genome data as it becomes available.<br>
[0067] Generally, the quantity of data representing thesuffix array is much larger<br>
than the quantity of data representing the genome. Therefore, database 714 may be<br>
more suitable for storing the suffix array than memory 712 because databases<br>
readily store more data than memory.<br>
[0068] User interface equipment 730 enables a user to input commands to<br>
computer 730 via input device 732. Input device 732 may be any suitable device<br>
such as a conventional keyboard, a wireless keyboard, a mouse, a touch pad, a<br>
trackball, a voice activated console, or any combination of such devices. Input<br>
device 732 may, for example, enable a user to enter commands to perform a word<br>
count of a particular word, or perform statistical analysis of potential probes. A<br>
user may monitor processes operating on system 700 on display device 734.<br>
Display device 734 may be a computer monitor, a television, a flat panel display, a<br>
liquid crystal display, a cathode-ray tube .(CRT), or any other suitable display<br>
device.<br>
[0069] Communication paths 790 may be any suitable -communications path such<br>
as cable link, a hard-wired link, a fiber-optic link, an infred link, a ribbon-wipe<br>
link, a blue-tooth link, an analog communications link, a digital communications<br>
link, or any combination of such links. Communications paths 790 are configured<br>
to enable data transfer between computer 710, user interface equipment 730, and<br>
Internet 740.<br>
[0070] Laboratory equipment may be provided in system 700 so that results<br>
obtained with the search engine can be directly applied to experiments, and vioe<br>
versa.<br>
[0071] An advantage of the search engine is that the techniquesibr counting<br>
exact word matches can take place entirely within the memory (eg., memory 716)<br>
of the computer. This provides for extremely fast and efficient querying of the<br>
genome for exact word matches. There is no need to aceess a database (e.g., a<br>
hard-drive). Such a need can substantially hamper search engine performance.<br>
The techniques used for counting exact word matches are 100% aecurate.<br>
B. SUFFIX ARRAY, BURROWS-WHEELER<br>
TRANSFORM AND ALPHABOUNDS<br>
[0072] Referring now to Fig. 8, illustrative flow chart 800 shows-steps in<br>
preparing a genome for use in the search-engine according to the principles of the<br>
present invention. Flow chart 800 uses techniques to build a suffix array data<br>
structure which provides the basis for-generating a transform of a particular<br>
genome. This transform provides a basis for the search engine of this invention,<br>
where the search engine can rapidly count the number of occurrences of a<br>
particular word (e.g., a word having a length of 15,21,70 or 80 characters). At<br>
step 810 a nucleotide sequence such as a genome or a portion of a genome is<br>
provided. The genome may be arranged as a string of characters having a length of<br>
N nucleotides, where N represents the total number of nucleotides in the string of<br>
characters representing the genome.<br>
[0073] The genome provided at step 810 can be derived from any organism or it<br>
can be randomly generated. For example, the entire known human genome may be<br>
provided or a portion of the human genome may be provided (e.g., a portion of the<br>
genome representing a chromosome or a region of a chromosome). If desired,<br>
non-human genome data may be provided such as the genomes of viruses, bacteria,<br>
single-celled and multi-celled organisms, including yeast, plant, and animals such<br>
as lizards, fish, and mammals (e.g., mice, rats, and nonhuman primates).<br>
[0074] At step 820, the genome is subjected to a transformation process which<br>
reorganizes the nucleotide arrangement of the genome according to a<br>
predetermined lexicographical order. The transform maintains the same<br>
constituent letters (e.g., As, Cs, Gs, and Ts) that appear in the genome, but these<br>
letters are arranged in a different order. In one embodiment of the invention, the<br>
genome is subjected to a known transform called the Burrows-Wheeler transform.<br>
The Burrows-Wheeler transform can be obtained from a suffix array. According to<br>
this invention, a suffix array is an N x N matrix representing all the cyclical<br>
permutations of the genome, where the permutations are arranged according to<br>
predetermined criteria (e.g., alphabetical, numerical, etc.). Advantageously, the<br>
Burrows-Wheeler transform represents the sorted N x N matrix of the cyclical<br>
permutations. Thus, when the search engine of the present invention searches<br>
through the Burrows-Wheeler transform, it, by extension, searches through the<br>
suffix array, which by further extension, searches through the original string<br>
representing the genome.<br>
[0075] Genome sequence assemblies can include an ambiguous character in<br>
addition to A, C, G, and T, thus extending the genome alphabet to five characters.<br>
This ambiguous character, commonly referred to as N, is typically used when the<br>
nucleotide at a particular position of a nucleic acid sequence is unknown.<br>
[0076] Because the Burrows-Wheeler transform represents a sorted suffix array,<br>
there is no need to access the suffix array when searching for a particular string of<br>
characters. Preferably, the transform is stored in memory, where searching<br>
functions can be executed much faster than when the transform is stored on a hard-<br>
disk. Moreover, because the quantity of data contained in a suffix array can be<br>
substantial, the suffix array may have to be stored in a hard-disk drive, as opposed<br>
to faster operating memory (e.g., random access memory of a computer). For<br>
example, the size of a suffix array for the human genome is on the order of twelve<br>
gigabytes. If such an array were to be stored in memory, the cost of a machine<br>
having twelve gigabytes of memory would be much more costly than a machine<br>
having, for example, three gigabytes of memory. Therefore, one advantage of the<br>
search engine is that it does not require expensive and memory-intensive machines<br>
because the transform represents a condensed version of the sorted suffix array.<br>
[0077] While the suffix array is not necessary for performing word-searches<br>
according this invention, it is useful to describe how such arrays are obtained in<br>
order to show the relationship between the transform and the array. The suffix<br>
array can be constructed by first obtaining the cyclical permutations of a nucleotide<br>
sequence. For example, Table 1 illustrates the cyclical permutations of the genome<br>
"AGACAGTCAT$," where "$" is provided to mark the end of the genome string.<br><br>
[0078] After the cyclic permutations are obtained, the rows atcsorted according<br>
to predetermined criteria to obtain a particular lexicographic order(e.g., an<br>
alphabetical lexicographic order). For example, Table 2 illustrates an alphabetical<br>
arrangement of permutations shown in Table 1, under the heading, Sorted Array.<br><br>
[0079J Once the cyclic permutations are sorted, the transform of the genome-can<br>
be obtained by taking the last letter of each row of the sorted array. These letters<br>
are reproduced under the column heading 'Transform," indicating that the<br>
transform of the genome "AGACAGTCATS" is "IGSCCATAAAG."<br>
(0080] In one embodiment, the suffix array of a genome such as the human<br>
genome can be built using a parallel radix sort using a 16-node cluster. Using this<br>
approach, the genome is divided into X number (e.g., 100) equal-isize substrings,<br>
each overlapping by seven nucleotides, with X being a predetermined number.<br>
The offsets into the genome (i.e., the "genome" coordinate) within each substring<br>
are assigned to one of 57 "prefix" bins according to the 7-mer (seven nucleotides)<br>
at each offset. The offsets within each prefix bin are sorted based on the sequence<br>
following the 7-mer piefix, thereby cteating the suffix array.<br>
[0081] At step 630, various statistics are computed to generate an auxiliary data<br>
structure, which can include an alphabounds data structure, a K-interval-data<br>
structure, and a dictionary counts data structure. The alphabounds indicate how<br>
many adenine, cytosine, guanine, and thymine nucleotides there are in the<br>
transform. For example, using the genome of Tabies 1 and 2, the alphabounds for<br>
A, C, G, and T are 4,2,2, and 2, respectively.<br>
[0082] The alphabounds can be used to delimit ranges in the transform that<br>
correspond to particular characters existing at the front of each row of the sorted<br>
suffix array. For example, a delimited range for the nucieotide A includes each<br>
row of the suffix array that starts with A. Referring to Table 2, it shows that rows<br>
1-4 of the sorted array start with A. Thus, the four rows correspond to the<br>
alphabounds computed for A. Table 2 shows that rows 5-6 begin with C, which<br>
correspond to the alphabounds computed for C Likewise, the G block<br>
corresponds to rows 7 and 8, and the T block to rows 9 and 10 of the transform.<br>
[0083] Step 830 may also generate K-intervals for-every K number of characters<br>
in the transform, where K is a predetermined number. K-intervals may be used to<br>
keep a running total of each nucleotide as they appear in the transform. These K-<br>
intervals may be used by the search engine of the ptesent invention to speed up the<br>
counting process, which is discussed below in connection with Figs. 3 and 4.<br>
Specifically, the use of the K-interval enables the search engine to outperform and<br>
use less space than conventional word counting techniques, especially when<br>
applied to nucleotide sequences greater than four million characters in length.<br>
[0084] The following example further explains how a transform is tabulated<br>
using K-intervals. Assume that the transform has the ten characters<br>
ACGTCAGTCA, and the K-intervals are stored every five characters. At the first<br>
interval, the K-interval includes one A, two Cs, one G, and one T. At the second<br>
interval {e.g., the tenth character) the K-interval includes a tabulation of all the<br>
nucleotides that have occurred in the transform thus Far. The second K-interval<br>
includes three As, three Cs, two Gs, and two Ts.<br>
[0085] At step 840, the Burrows-Wheeler string is compressed according to a<br>
predetermined compression ratio. Preferably the string is copressed using a 3-to-<br>
1 compression ratio. That is, for every three characters, the string is compressed to<br>
one character {e.g., 3000 characters is condensed down to 1000 characters). Those<br>
of skill in the art will appreciate that othercompcession ratios can be used. For<br>
example, a four-to-one or a five-to-one compression may be employed. The string<br>
can be compressed using a dictionary-based compression scheme, where one of<br>
125 distinct single byte codes represents one of each of the 53 possible three-tetter<br>
substrings (e.g., AAA, AAC,..., TTT). More specifically, the transform is divided<br>
up into three character substrings and each substring is compressed according the<br>
dictionary based compression scheme. For example, if a three character substring<br>
is AAA, it may be equivalent to byte 0 of the dictionary compression scheme.<br>
occurrences of the query pattern. The search region is defined using predefined<br>
criteria such as a particular character of the query pattern, alphabounds, and other<br>
data. A more detailed explanation of how the search region is defined is discussed<br>
in conjunction with the description accompanying Fig. 10.<br>
[0090] At step 920, the process determines how many times the next preceding<br>
character of the query pattern appears in the search region. At step 922, if the<br>
proceeding character count is zero, the query pattern does not exist and the process<br>
ends (step 924). If at least one character is found within the delimited range, the<br>
prooess proceeds to step 926. At step 926, it is determined whether the preceding<br>
character is the first character in the query pattern. If so, the process proceeds to<br>
step 928, where the count obtained in step 920 is outputted and the process ends.<br>
If the preceding character is not the first character of the query pattern, the prooess<br>
loops back to step 918 because it has not yet been determined if the query pattern<br>
does or does not exist in the genome. At step 918, the search region is redefined<br>
using predetermined criteria.<br>
[0091] More particularly, the search region is redefined using equations 1 and 2<br>
below:<br>
Start Position = A + Z (1)<br>
End Position = Start Position + M - 1 (2)<br>
where A is the start position of the preceding character according to the<br>
alphabounds, Z represents the number of times the preceding character occurs in<br>
the transform prior to the currently defined search region, and M represents tile<br>
number of times the preceding character occurs in the corrtently defined search<br>
region.<br>
[0092] The redefined search region also potentially contains all occurrenoes of<br>
the query pattern, but the newly defined search region further limits the character<br>
positions that need to be searched in step 920. After the new search region is<br>
defined, the process continues to step 920, where the next preceding character (i.e.,<br>
the character proceeding the last character used in the previous step 920) of the<br>
query pattern is counted within the newly defined search region. This loop may<br>
repeat as many times as necessary before finding the'first-character of the query<br>
pattern, and consequently the number of word-counts. If one of the pseeeding<br>
characters is not found in a search region, it will be concluded that no such pattern<br>
exists in the genome.<br>
[0093] Figs. 10A-B illustrate an example of the foregoing word counting<br>
algorithm. This example uses the illustrative genome (AGACAGTCATS), suffix<br>
array, Burrows-Wheeler transform (TGSCCATAAAG), and alphabounds<br>
described previously in connection with Tables 1 and 2. In this example, assume<br>
that a user wants to determine how many times the word "CAG" appears in the<br>
genome.<br>
[0094] In Fig. 10A, the process begins by delimiting the G block because G is<br>
the last letter in the word "CAG." As illustrated, the G block begins at position 7<br>
and ends at position 8 of the Burrow-Wheeler transform. These positions are<br>
obtained from the alphabounds. Once the G block is delimited, the engine searches<br>
for and counts the number of As, the next preceeding character of "CAG," existing<br>
within the G block. Fig. 10A-shows that two As appear in the G block, thus<br>
indicating that the genome contains two occurrences of "AG."<br>
[0095] If desired, K-intervals could be used to facilitate the step of counting the<br>
number of times a particular letter appears within a search region (e.g., counting<br>
the number of As in the G block) and can also be used to count the number of<br>
times a particular letter appears before a search region. To carry out such counting<br>
steps, the particular character is counted starring from a predetermined position<br>
(e.g., the start position) and progress to the nearest position that is a multiple of K.<br>
One advantage of using the K-intervals in connection with the search-engine is that<br>
the time it takes to determine haw many times a particular word appears in a<br>
genome is linear with respect to the K-intervals, the size of the word being<br>
searched, and the time required to access various memory addresses. Thus, the<br>
size of the genome is not a factor in determining the word count, unless the size of<br>
the compressed transform and the K-interval data structure are too lacge to fit in<br>
memory (e.g., random-access-memory). In one embodiment, the K-can be set to<br>
300 characters, or equivalently, 100 compressed bytes. With such an arrangement,<br>
the maximum number of counts that need to be performed does not exceed K/2.<br>
[0096] If desired, subintervals of size K^ within each K-interval may be used to<br>
maintain a running total of each character occurring within a particular K-interval.<br>
If the size of K is limited to be less than 28, for example, then the counts for each<br>
letter at every K-interval can be recorded using a single byte. This provides for<br>
increased density of the counting index by a factor of K/K^ while increasing the<br>
space requirements for the K-interval counts by a factor of only^ [[K/K ^)/4]. Such<br>
subintervals and size constraints have been-employed by the auxiliary data<br>
structure being used in connection with this algorithm. Depending on the choices<br>
of K and K^, a three to fivefold increase in query execution-speed has been<br>
achieved while maintaining a memory requirement of less than two gigabytes for<br>
the human genome.<br>
[0097] To further speed the counting process, the dictionary counts data structure<br>
can be used. Note that the compression scheme used is a 3:1-compression scheme,<br>
where bytes 0 through 124 decompress to "AAA" through "TTT" respectively.<br>
The dictionary-counts structure is a two-dimensional array that-can be thought of<br>
as a matrix with 125 rows with five columns. Each row corresponds to one of the<br>
compression dictionary entries, and each column corresponds to each letter of the<br>
genome alphabet, A through T. The following explains by way of example how<br>
the dictionary counts structure and K-intervals can be used to perform counting<br>
operations.<br>
[0098] Assume, for example, that the search engine is in the process of<br>
determining the number of As that occur before the search region. Using the K-<br>
interval counts structure described above, the engine can "jump" to within at least<br>
50 bytes of the current start position of the search region in a single look-up.<br>
Further assume that the start position is pointing to the third "T" in a compressed<br>
"ATT"(a byte) wich is the 49th byte of the intrval. For each of the 48<br>
preceeding bytes, the byte itself can be used as the row number in the dictionary<br>
counts data structure, and the tetter of interest, "A," represents the column number.<br>
Using this information as coordinates for accessing the dictionary counts array, the<br>
dictionary counts data structure provides the number times "A" appears in that<br>
compressed byte. Therefore, to determine how many As appear before the tart of<br>
the search region, the dictionary counts structure needs to be aecess48 times. In<br>
addition, the 49th byte may need to be decompressed in order to examine the first<br>
two letters "AT" of the "ATT" byte.<br>
[0099] Thus, when the dictionary counts data structure is combined with the K -<br>
intervals data structure, the step of counting any number characters requires only<br>
K/6+1 table look-ups, plus two character-comparisons in the worst case.<br>
[0100] Referring back to Fig. 10, the search engine then delimits the AG block<br>
within the transform so that it knows where to search for the next preceding<br>
character. The boundaries of the AG block are found by adding the number of<br>
times A precedes the G block in the transform to the first position that the A block<br>
begins in the transform. In this example, only one A occurs before the^G block.<br>
Therefore, using equation 1 above, where A is 1 and Z is 1, a start position of 2 for<br>
the AG block is obtained. The ending position of the AG is obtained using<br>
equation 2 above, where M is 2 number of As found in the G block). Equation 2<br>
yields an AG Block end position of 3, as shown in Fig. 10B.<br>
[0101 ] Once the AG block is found, the search engine counts the number of<br>
times C occurs therein. This count yields the number ofCAGs that oocur in the<br>
genome because C is the first character of the word "CAG." Thus, the seatch<br>
engine yields a word count of one.<br>
[0102] Fig. 11 shows an illustrative genome having cordinate positions and a<br>
sorted suffix array that has coordinate positions corresponding to the genome's<br>
coordinate positions. That is, the first character in each row of the sulfix array<br>
corresponds to one of the characters in the genome. For example, the second row<br>
of the array has a coordinate position of 2, which corresponds to position two of<br>
the genome. Thus the coordinate positions of the suffix anray correlate to the<br>
coord inate positions of the genome.<br>
[0103] If desired, the suffix array can be used to locate the coordinate position of<br>
a particular word. For example, if the coopdinate position of "CAG" is being<br>
sought, the suffix array of Fig. 11 can be accessed and it will indicate that CAG<br>
starts at position 3. However, as mentioned above, accessing the suffix array is a<br>
time consuming process because it requires hard-disk drive access. Therefore, it is<br>
desirable to obtain word coordinates by only accessing memory. This can be<br>
achieved by assigning preselected suffix array coordinates to the transform,<br>
thereby allowing a coordinate location algorithm to use the transform to locate the<br>
starting coordinate of a particular word.<br>
[0104] Such a coordinate location algorithm is explained by way of example.<br>
Assume that the circled portion of the suffix array is the transform of the genome<br>
and that only coordinates 3 and 7 have been carried over to the transform from the<br>
suffix array. Further assume that it is desired to find the coordinates of TC. (Note<br>
that if the transform had coordinates associated with the G that is affiliated with<br>
TC, the coordinates of TC would be known without having to resort to using the<br>
coordinate location algorithm.) It is known that the TC is associated with the last<br>
G in the transform. Starting with this G, the algorithm determines how many<br>
preceding G's there are. In this case, there is one preceding G.<br>
[0105] The alphabounds data structure and the number of preeceding G's are<br>
used to determine which letter precedes this particular G. Using the alphabounds,<br>
it is known that the G block starts at position 7. Since there is one pieeeding G, the<br>
algorithm adds this number to 7 to obtain 8. Thus, the A that corresponds to the<br>
suffix array string that starts with GT is the letter that precedes the afopementioned<br>
G. This completes one iteration of the coordinate location algorithm. Generally<br>
speaking, this iteration is repeated until a coordinate (e.g., 3 or 7) on the transform<br>
is reached. Once a coordinate is reached, the number of iterations is added to the<br>
coordinate and the resulting sum is the actual starting coordinate position of the<br>
desired word (e.g., TC).<br>
[0106] Continuing with the iterative process, it is known that two As precede the<br>
A associated with the suffix array string beginning with GT. Using the<br>
alphabounds and the number of preceding A's, the algorithm settles on the C<br>
associated with the suffix array begmning with AGT. Since there ate no Gs<br>
preceding this particular C, the algorithm settles on the A associated with the suffix<br>
array string starting with CAG. Because this A has a coordinate positrontion (e.g., 3),<br>
the actual position of the word TC can be determined by adding 3 the coordinate<br>
position of this A) to the number of iterations, which in this example is 3, resulting<br>
in a coordinate position of 6. Thus, TC begins at the coordinate position of 6 in the<br>
original genome.<br>
D. APPLICATIONS OF THE SEARCH ENGINE<br>
[0107] Now that the operative features of the search engine have been described,<br>
practical applications of the engine can be discussed. One application of the search<br>
engine is that it can be used to annotate a genome (or any other type of nucleotide<br>
sequence). Particularly, the genome can be annotated using substrings of a<br>
particular length that exist within the genome. The search engine can then count<br>
the number of times a particular length substring occurs in the genome. These<br>
counts provide an indication of the uniqueness of a particular substring, where<br>
lower counts represent a higher degree of uniqueness than higher counts.<br>
[0108] If desired, any region of the genome or the whole genome may be<br>
annotated based on its constituent "mer" frequencies. A "mer" is another term for<br>
a word or substring of a particular length. Thus when a genome or a portion<br>
thereof is being annotated, it is annotated based on mers of a particular length (e.g.,<br>
mer lengths of 15,18,21, and 24). Regardless of the mer length being annotated,<br>
every mer of that length that exists in the genome is counted. For example, if the<br>
mer length is 15, the search engine will determine the word-count for the first 15-<br>
mer and each 15-mer occurring thereafter. Each succeeding 15-mer overlaps the<br>
previous 15-mer word by one character. That is, characters 1 through 15 constitute<br>
a 15-mer, characters 2 through 16 constitute another 15-mer,-characters 3 through<br>
17 constitute yet another 15-mer, and so forth. This ensures that every annotated<br>
15-mer is assigned a word count, such that the word count represents the number<br>
of times that particular 15-mer occurs in the entire genome.<br>
[0109] Probe design is facilitated by using the search engine. The eoguae's<br>
ability to rapidly count the number of times a particular word appears in a-genome<br>
is useful in designing probes that are unique and hybridize to a specific region of<br>
DNA with minimal cross-hybridization. Byusin the search engine,potential<br>
cross-hybridizations can be minimized by selecting a candidate probe that is<br>
comprised of smaller mers that are unique and meet certain stringency conditions<br>
such as having low or no word counts within the entire genome. A unique word<br>
may be a particular string of nucleotides that have less than a predetermined<br>
number of word counts (e.g., less than 2,5,10,25,-50, or 100 word counts) or an<br>
absence of word counts (e.g., zero word counts) within a-genome of portion<br>
thereof.<br>
[0110] More particularly, candidate probes are obtained based on a set of<br>
predetermined criteria such as requiring the candidates to have a length, L1, and<br>
also requiring that the candidates have a predetermined word count (e.g., a<br>
candidate probe having a word count of one). In addition, the predetermined<br>
criteria can also require that the reverse complement of a candidate to have a<br>
predetermined word count (e.g., one). Once the candidates are obtained, they are<br>
subjected to additional predetermined criteria to determine which candidates are<br>
suitable for use as probes. These additional criteria are used to filter the candidates<br>
based on their constituent subregions (i.e., mers of a length contained within the<br>
candidate probe). For example, the filtering criteria may require a mer of length<br>
L2, where L2 is less than L1, to have word counts that are minimized relative to<br>
other probe candidates. Thus, a relationship exists among the criteria used in<br>
finding probes - a relationship between "hard" constraints (e.g., in which each<br>
candidate is unique with respect to the genome) and "soft" constraints (e.g., in<br>
which the constituent mer counts are minimized).<br>
[0111] One way to satisfy the "hard" constraints is to obtain candidates based on<br>
the results of a previously performed annotation. Using word count information,<br>
candidates can be selected from regions of the genome having low-concentrations<br>
of word counts (e.g., it is preferable to obtain candidates having a minimum mean<br>
value of word counts of a predetermined length, a geometric mean value of word<br>
counts of a predetermined length, a mode value of word counts of a predetermined<br>
length, a minimized maximum value of word counts of a predetermined length, a<br>
sum total value of word counts of a predetermined length, a product value of word<br>
counts of a predetermined length, a maximum length string of a particular<br>
nucleotide, or a combination thereof.<br>
[0112] To satisfy the "soft" constraint, the candidates can be annotated according<br>
to predefined criteria such as 15-mer counts, 17-mer counts, etc. The data obtained<br>
from the annotation is analyzed to determine whether a candidate is unique enough<br>
to be used as a probe. A candidate may be selected as a probe if, tor-example, it<br>
has the lowest sum of 15 mer counts of all the candidates. Other criteria such as<br>
minimal occurrences of composition bias (e.g., long strings of a particular<br>
nucleotide) can be applied to determine which probe is best. After the criteria are<br>
applied to each candidate, the one or more candidates are selected as suitable<br>
probes.<br>
[0113] Yet another application of the search engine is to detect changes from one<br>
genome to another. For example, as the human genome project progresses, new<br>
segments of the genome are mapped and released to the public. Using the Search<br>
engine and probes that were designed on another version of the same genome, it<br>
can be determined how many of those probes can be applied to the new version of<br>
the genome.<br>
[0114] Yet still another application in which the search engine can be used is to<br>
verify whether a particular word exists in the genome. It may be desirable to find<br>
words that do not appear in the genome so that there is little chanoe that the word<br>
will hybridize to a section of the genome. These words may be-generated<br>
randomly according to a predefined set of criteria. When a word is found, its<br>
complement is also submitted to the search-engine to determine whether it appears<br>
in the genome. If both the word and its complement do not appear in the genome,<br>
there is a minimal chance that this word and its complement will hybridise to fee<br>
genome. Such non-hybridizing probes can be used in hybridization as readable<br>
barcodes and in hybridization array controls, and can be added to nucleic acid<br>
probes for the purpose of enhancing hybridization signals through network<br>
formation.<br>
[0115] One way to minimize the chance of hybridization is to minimize the<br>
frequency of a particular word's constituent mers. That is, it is preferable to obtain<br>
probes that have as many constituent mer lengths that have word counts of zero.<br>
For example, assume that several 20-mer oligonucleotides are generated with the<br>
aim of not hybridizing to the human genome. Then, further assume that-each 20-<br>
rner is annotated for each of its constituent overlapping 49-mers, 18-mers, 17-mers,<br>
16-mers, down to, for example, 6-mers. Theoretically, the most desirable 20-mer<br>
would preferably have zero word counts for each length mer. In practise, a probe<br>
that has minimal chance for hybridization preferably has as many mer counts of<br>
zero as far down in the mer lengths as possible (e.g., a desirable probe may have<br>
zero word counts for mer lengths of 19,18,17,1-6,15,14, and 13). Thus, if one<br>
probe has zero counts of its constituent 15 and 14-mers, it is less likely to hybridize<br>
to the genome than a probe that has zero counts of its constituent 13-mers, but has<br>
one or more counts of its Constituent 14-mers. Thus, me former probe has less of<br>
an opportunity lo hybridize than the latter probe because it does not have any 14-<br>
mers that match with sections of the genome.<br>
[0116] Non-hybridizing oligonucleorides can be constructed using constituent<br>
raers of a particular mer that have a zero or low word-count. For example, if a<br>
particular 20-mer has a 13-mer that has a zero word count, this 13-mer can be used<br>
to build oligonucleotides that probably do not exist in the genome (e.g., two of<br>
these 13-mers can be attached to each other to create a unique 26-mer).<br>
[0117] In a laboratory setting, for-example, a .zero count word and its zero count<br>
complement (non-hybridizing oligonucleotides) can be attached to a (hybridizing)<br>
probe or target word. in an abstract sense, the words are the "arms" that are<br>
attached to the "body" (i.e., the probe). When hybridization commenees, the words<br>
("arms") only hybridize to each other, while the probe hybridizes to the genome.<br>
Because the words ("arms") typically carry a detectable material (e.g., a fluorescent<br>
label) the self hybridization helps a person distinguish the probe's location within<br>
the genome against background hybridization. Thus, the self hybridization of the<br>
arms serves to amplify visibility of the probe that is hybridized to the genome.<br>
[0118] The non-hybridizing oligonucleotides can also be used as tags to uniquely<br>
identify a particular sequence among a vast population of other sequences. The<br>
non-hybridizing oligonucleotides can be attached to the known sequence, thereby<br>
tagging or labeling a particular sequence.<br>
[0119] In yet another example, several different DNA sequences can be<br>
concatenated to form the single genome (e.g., provided for example, at step 810) of<br>
Fig. 8). Such a concatenated genome is useful, for example if it is desied to<br>
design a probe that detects the presence of a particular pathogen (e.g.,) virus)<br>
within a human blood sample. A concatenated genome is needed because the<br>
DNA extracted from human blood not only contams human DNA, but also DNA<br>
from other sources such as the pathogen. Therefore, in order for the probe to<br>
effectively detect the pathogen in human blood, it should not cross hybridize to the<br>
human genome.<br>
[0120] In the event that the pathogen probe is not completely unique with respect<br>
to the other genomes in a tissue sample (e,g., the patient's genome and genomes of<br>
other microorganisms found in the patient), it may be necessary to compare the<br>
word count for the probe in the pathogen genome with the word counts for the<br>
probe in the other genomes. This approach may require two search engines — one<br>
for the pathogen of interest, and the other for a combination of the other genomes.<br>
Note that in applying this dual search engine approach, it may be advantageous to<br>
design probes that have high mer counts within the pathogen-genome, as long as<br>
the probe counts in the other genomes in the tissue sample are disproportionately<br>
low.<br>
VII. EXAMPLES<br>
[0121] The Following examples are provided by way of illustration only. They<br>
are not intended to limit the scope of the invention disclosed herein.<br>
Example 1 - Selection of Oligonucleotides Complementary to a Representation<br>
[0122] This example demonstrates the identification of oligonucleotide probes<br>
that are complementary to a BglII-derived representation of a human genome.<br>
Similar approaches may be used to design oligonucleotides complementary to any<br>
population of nucleic acids whose sequences are known or predicted.<br>
Using the published draft assembly of the human genome sequence, we performed<br>
an in silico BglII digestion of the human genome by locating all BglII restriction<br>
sites within the draft assembly. We further selected all sequenees of BglII.<br>
fragments that were between 200 to 1,200 basepairs in length. We then analyzed<br>
the sequences of these fragments using an algorithm described herein. This<br>
algorithm (also called "a mer-engine")can be used to determine therapy number<br>
of any given oligonucleotide sequence in any sequenoed enome. Thiscopy<br>
number is also called the "word count" of the oligomucleolitle- squere in the<br>
genome.<br>
[0123] We annotated each BglII-digested 'fragment with the word counts of their<br>
constituent, overlapping 15- and 21-mers (i.e., oligonucleotides having 15 or21<br>
nucleotides) using the mer-engine constructed from the same draft assembly of the<br>
human genome. To do this, we generated in silico for each fragment every<br>
constituent, overlapping 70-mer oligonucleotide (e.g., a 100 basepair fragment<br>
would have 31 such 70-mers). The following attributes were determined for every<br>
such 70-mer of a fragment as described below: maximum 21-mer-count (or<br>
maximum 18-mer count), arithmetic mean of 15-mer counts, pepoent-G/C content,<br>
and quantity of each base, and the longest run of any single base.<br>
[0124] To determine the maximum 21 -mer count, we broke-each 70-mer into<br>
overlapping 21-mers and compared each of these 21-mers against all of the 21-mer<br>
sequences in the genome. We discarded all 70-mers whose maximum 21-mer<br>
count was greater than 1, i.e., those with a 21 -mer sequence that was 100%<br>
complementary to more than one 21-mer sequence in the genome. This was our<br>
initial set of 70-mer probes.<br>
[0125] We further optimized the 70-mer probe set by removing those with a GC<br>
content less than 30% or greater than 70%, a run of A/T greater than-6 bases, or a<br>
run of G/C greater than 4 bases. From the remaining 70-mers, we chose for each<br>
BglII fragment the one (or more) 70-mer which had a GC/AT proportionality<br>
closest to that of the genome as a whole. We further analyzed-each of the thus-<br>
chosen 70-mers by determining the genome word count for-each of the 70-mer 's<br>
constituent, overlapping 15-mers. We chose 70-mers that had the lowest mean 15-<br>
mer count.<br>
[0126] As a final check for overall uniqueness, the optimal 70-mer probes-for<br>
each BglII fragment were compared to the entire genome using the BLAST<br>
software program. Default parameters were used with the exception of filtration of<br>
low complexitysequence, which was not performed. Any 70-mer probe with any<br>
degree of homology along 50% or more of its length to any sequence other than<br>
itself was eliminated.<br>
[0127] The mer-engine algorithm affords rigor, flexibility and simplicity to the<br>
probe design process. The ability to rapidly determine word cunts for words of<br>
all sizes allows design criteria to be framed quantitatively in a way that is<br>
analogous to real hybridization events. The word counts can be considered as a<br>
quantitative measure of the degree to which sequences belong to two or mote sets<br>
of polynucleotides. For instance, the small probe "AGT'"can be thought of as a set<br>
containing six distinct words, namely "A", "G," "T," "AG," GT," and "AGT." If<br>
this probe were annotated with word counts for all words of all sizes, it would be<br>
discovered that the number of times each word appears in the first set, feat being<br>
the probe "AGT," would be vastly overshadowed by the number of times they<br>
appear in the second set, namely the three billion nucleotide genome.<br>
[0128] This relationship can be expressed as a ratio X/Y, where X is the sum of<br>
counts for all of a probe's constituent words relative to said probe and Y is the sum<br>
of counts for all of me same words within the genome. When-selecting a 70-mer<br>
probe that hybridizes to a target sequence with minimal cross-hybridization, one<br>
can maximize the ratio X/Y, where the maximum value of X/Y For probes derived<br>
from the genome sequence is 1. The technique of selecting only two word lengths<br>
with which to annotate is essentially one of many possible short cuts toward this<br>
goal.<br>
[0129] Id the event that no unique probes can be found within a genomic region<br>
of interest, it is possible to make use of non-unique probes to provide dear<br>
measurements of relative copy number differences or simply the quantity of<br>
matter. The problem then extends to a comparison between three sets of words:<br>
the probe, the encompassing region of interest, and the genome. Let Z represent<br>
the sum of all probe word counts relative to the encompassing region. Assume that<br>
X and Y still represent the sums of all probe word counts relative to the probe and<br>
the genome respectively. The goal is then to maximize the value of the expression,<br>
(X/Y)/(X/Z), or simply, Z/Y. In other words, one can find probes that are region-<br>
specific regardless of the overall copy number. This special case can be<br>
generalized to include any circumstance in which one is selecting probes to<br>
recognize one particular entity out of many through hybridization. A further<br>
example is me recognition of the DNA of one organism when-exposed to the DNA<br>
of many other organisms.<br>
[0130] Yet another application of this paradigm is mat of minimization of set<br>
membership. We have designed probes that acted as hybridization controls in<br>
microarray experiments. These probes were controls in the sense that they wer-e<br>
intended to hybridize to only those fragments of DNA that any other probe had an<br>
equal chance of recognizing. The goal in this case was simply to design a probe<br>
where Y was as close to zero as possible. Such a probe will also be useful, e.g., as<br>
hybridization-readable unique identifiers or as additions to other nucleic acid<br>
sequences for enhancing hybridization signal through networking.<br>
{0131) In addition to sums and arithmetic means of word counts, many other<br>
statistics can be used, including, for example, the variance of a probe's word<br>
counts for words of a particular size. This variance can act as a rapid prescreen ibr<br>
the selection of probes that must exist in a particular copy number. The maximum<br>
word count for a particular word size can be taken as an indication of the worst<br>
possible hybridization outcome for an otherwise unique probe. These quantitative<br>
measures are ideal for rapidly determining the fitness of a hybridization probe<br>
relative to other candidates. The mer engine algorthm in essence can reduee the<br>
process of probe selection to a single pass scan over the sequence of interest<br>
[0132] One of the probcsets that we designed consisted of83,000 70-mers,<br>
possessed a mean 18-mer count relative to the human genome of 1.2 with a<br>
standard deviation of 0/8. The mean was calculated over the set of all 18-mers of<br>
all probes combined. Compared with the prior art, in particular a published set of<br>
approximately 23,000 70-mer-expression array probes, the mean of the l$-mer<br>
counts for all probes combined was 1.9 with a standard deviation of 14.8.<br>
Therefore, this set of probes was the larger of the two by a factor of 4, and was<br>
more consistently unique by a factor of 18. The set of 85;0O0 probes in this<br>
example were selected by us based on the combination of a unique 21-mer<br>
constraint and a minimized aggFegate 15-mer count Constraint as previously<br>
described. The advantages included a large increase in confidence that probes that<br>
proved to perform well empirically were not simply hybridizing to a large<br>
heterogeneous population of DNA fragments and thereby increasing theirsignal.<br>
This further illustrates the precision with which probe sete canbe tlesignedto<br>
satisfy rigorously defined criteria, such as an extemly smallstandard deviation<br>
about a target mean word count.<br>
Example 2 - Preparation of Arrays<br>
[0133] We used two formats for constructing microarrays containing<br>
oligonucleotide probes designed according to Example 1. Id the first of these, the<br>
"print" format, we purchased about 10,000 oligonucleotides made with solid phase<br>
chemistry, and printed them with quills on a glass surface. Specifically, we used<br>
the Cartesian PixSys 5500 (Genetic Microsystems) to array our probccollection<br>
onto the slides using a 4 X 4 pin configuration. The dimensions oteach printed<br>
array was roughly 2 cm2. Our arrays were printed on commercially prepared<br>
silanated slides (Corning® ultraGAPS™ #40015). Pins used for the arrayer were<br>
from Majer Precision.<br>
[0134] In the second format, the "photoprint" format, oligonucleotides were<br>
TM<br>
synthesized by NimbleGen Systems, Inc. directly on a silica surface using laser-<br>
directed photochemistry. Approximately 700,000 unique 70-mer oligonucleotides<br>
were first screened for "performance" by arraying them onto-eight chips and<br>
hybridizing them with BglII and EcoRl -depleted BglII. representations of genomic<br>
DNA from a normal male J. Doe. We picked the 85,000 oligonucleotides that<br>
generated the strongest signal and arrayed them on a single chip.<br>
[0135] In both formats, we arrayed the oligonucleotides in a random order to<br>
ninimize the possibility that a geometric artifact during array hybridization would<br>
be incorrectly interpreted as a genomic lesion. In subsequent examples, we<br>
describe results with the 1 OK printed arrays and the 85K photoprint arrays.<br>
Example 3 - Preparation and Labeling of Test Representations<br>
[0136] For some experiments described herein, we uhose BglII to make the<br>
representations. BglII has characteristics useful for these particular experiments:<br>
It is a robust enzyme; its cleavage site is not affected by CpG methylation; it leaves<br>
a four base overhang; and its cleavage sites have a reasonably uniform distribution<br>
in the human genome. BgHlII representations are made up of short fragtnents,<br>
generally smaller than 1,200 bps. We estimated that there are about 200,000 of<br>
them, comprising about 2.5% of the human genome, with an everage spacing of 17<br>
kb.<br>
[0137] In all of the experiments described herein, we used comparative<br>
hybridization of representations prepared in parallel. The'DNA from two samples<br>
being compared were prepared at the same time, and reptsentations prepared from<br>
the same concentration of template, using the same protocols, reagents and thermal<br>
cycler. This would diminish the possible "noise" created by variable yield upon<br>
PCR amplification.<br>
[0138] We prepared BglII representations of buman genomic DNA as previously<br>
described by Lucito et al., 1998, supra. Briefly, we digested 3-10 ng of human<br>
genomic DNA with BglII underconditions suggested by the supplier. We purified<br>
the digest by phenol extraction and ethanol precipitation in the presence of 10 mg<br>
of tRNA. We resuspended the pellet in 30 ml of 1X T4 DNA ligase buffer with<br>
444 pmol of each adaptor (RBgl24 and RBlg 12; Lucito,R. and M. Wigter. 2003.<br>
"Preparation of Target DNA." In Microarray-based Representational Analysis of<br>
DNA Copy Number (eds. D. Bowteil &amp; J. Sambrook), pp. 386-393. Cold Spring<br>
Harbor Press, Cold Spring Harbor, NY). We placed the reaction mixture in a<br>
preheated 55 °C heat block and placed the heat block on ice for approximately 1<br>
hour until the temperature had dropped to 15°C. We then added 400 units of T4<br>
DNA lgase and incubated the reaction mixture at 15°C 12-18 hours.<br>
[0139] We added l/40th of the ligated material, 20 ml of 5X PCR buffer [335<br>
mM Tris-HCl, pH 8.8; 20 mM MgCl2; 80 mM (NH4)2 SO4; 50mM b.<br>
mercaptoethanol and 0.5 mg/ml BSA], 2'-dideoxynucleoside 5'-triphosphates to a<br>
final concentration of 0.32 mM, RBgl24 adaptor to a final-coneentration of 0:6 mM,<br>
1.25 U of Taq polymerase and water to a 250 ml tubes to bring the volume to 1-00<br>
ml. The tubes were placed into an MJ Research TETRAD thermocycler<br>
preheated at 72°C. We then conducted the amplification as follows: one cycle at<br>
72°C for 5 minutes, and then 20 cycles of 1 minute at 95°C and 3 minutes at 72°C<br>
followed by a 10 minute extension time at 72 °C. We cleaned the representations<br>
(i.e., the PCR products) by phenol:chloroform extraction and ethanol precipitation<br>
before resuspending in TE (pH 8) and determining the DNA congentration.<br>
[0140] For certain experiments, we prepared depleted representations by<br>
digestion with an additional restriction endonuclease to cleave those tragments that<br>
contained its restriction site. In these cascs, we digested the ligaticn mixtrue With<br>
the second restriction endonuclease just before the amplification step. In the<br>
experiments described below, the depleted BglII representation was ptoduoed<br>
using HindIII.<br>
[0141] We labeled fragments in the representations by placing the DNA in a 0.2<br>
ml PCR tube. We added 10 m1 of primers from the Amersham-Pharmacia<br>
Megaprime™ labeling kit and mixed them well with the DNA. We brought the<br>
volume up to 100 ml with water. We placed the tubes in an MJ Research<br>
TETRAD™ machine at 100°C for 5 minutes, placed on ice for 5 minutes and added<br>
20 ml of labeling buffer from the Amersham-Pharmacia Megaprime™ labeling kit,<br>
10 ml of label (either Cy3 -dCTP or Cy5 -dCTP) and 1 ml of New England<br>
BioLabs Klenow fragment. We incubated the tubes at 3 7°C for two hours,<br>
combined the labeled samples (Cy3 and CyS™) into one Eppendorf® tube, and<br>
then add 50 ml of 1 mg/ul Human Cot 1 DNA, 10 ml of 10 mg/ml stock yeast tRNA,<br>
and 80 ml of Low TE (3mM Tris pH 7.4, 02 mM EDTA). We loaded the sample<br>
on a Centricon Filter and centrifuged for 10 minutes at 12,500 rcf. We discarded<br>
the flow through and washed the filter with 450 ml of Low TE. We repeated the<br>
centrifugation and TE wash twice. We collected the labeled sample by inverting<br>
the Centricon column into a new tube and oentrifuging for 2 minutes at 12,600<br>
rcf. We transferred the labeled sample to a 200 mlPCR tube and adjusted the<br>
volume with Low TE to 10 ml.<br>
[0142] In addition, for some experiments, we digested DNA isolates from a<br>
primary ovarian cancer cell and from normal reference with McrBC and ligated<br>
linkers and amplified as described above.<br>
Example 4 - Hybridization of Test-Representations to Arrays<br>
[0143] We UV-crosslinked the oligonucleotide probes to the slide using a<br>
Stratagene® Stratalinker® set at 300 mJ, rotated the slide 180 degflses, keeping the<br>
slide in the same spot in the crosslinker, and repeated the treatment. We washed<br>
the slides for 2 minutes in 0.1% SDS, 2 minutes in Milli-Q® water, 5 minutes in<br>
boiled Milli-Q® water, and finally in ice cold 95% benzene- free ethanol. We dried<br>
the slides by placing them in a metal rack and spinning them for 5 minutes at 75<br>
rcf. We prehybridized the printed microarrays by placing them in a coplin jar or<br>
other slide processing chamber, adding prehybridization buffer (25 % deionized<br>
formamide, 5X SSC and 0.1% SDS) and preheating the chamber to 610C for two<br>
hours and then washed them in Milli-Q® water Tor 10'seconds. We again dried the<br>
slides by placing them in a metal slide rack and spinning Cor 5 minutes at 75 rcf.<br>
NimbleGenTM photoprinted arrays did not require UV-crossliking or<br>
prehybridization.<br>
[0144] We added 25 m1 of hybridization solution to 10 ml of labeled sample<br>
prepared as in Example 3 and mixed. For printed slides, the hybridization-solution<br>
was 25% formamide, 5X SSC, and 0.1% SDS. For NimbleGen™ photoprinted<br>
arrays, it was 50% formamide, 5X SSC, and 0.1% SDS. We denatured the samples<br>
in an MJ Research™ TETRAD™ at 95°C for 5 minutes and then incubated at 37°C<br>
for 30 minutes. We spun down the samples and pipetted them onto a slide<br>
prepared with lifter slip and incubated it in a hybridization oven (such as the<br>
Boekel InSlide Out oven) set at 58°C for printed arrays or 42°C for NimbloGen™<br>
photoprinted arrays for 14 to 16 hours.<br>
[0145] After hybridization, we washed the slides as follows: briefly in 0.2%<br>
SDS / 0.2X SSC to remove the coverslip; 1 minute in 0.2% SDS / 0.2X SSC, 3D<br>
seconds in 0.2X SSC; and 30 seconds in 0.05X SSC. We dried the slides as before<br>
by placing them in a rack and spinning them at 75 rcf for 5 minutes. We then<br>
scanned the slides immediately.<br>
[0146] We scanned the slides using an Axon GenePix® 4000B scanner set to a<br>
pixel size of 10 microns for printed arrays and 5 microns for photoprinted arrays.<br>
We quantified the intensity of the arrays using GenePixTM Pro 4.0 software and<br>
imported the data into S-PLUS® for further analysis. We calculated the ratios<br>
between two signals in an experiment using measured intensities without<br>
background subtraction. We normalized the data using an intensity based lowess<br>
curve fitting algorithm similar to that described in Yang et al., Nucl. Acids Res. 30:<br>
el 5-15 (2002). We averaged the data obtained from color peversal experittients<br>
and displayed it as presented in the figures.<br>
Example S - Performance and Validation of Arrays<br>
[0147] As discussed above in Example 1, we should be able to predict-based on<br>
the published human genome sequence which oligonucleotide probes«oould<br>
hybridize with which representations. To confirm this, we tested our 10K printed<br>
arrays by hybridizing them to BglII. representations of normal human genomic<br>
DNA labeled with one fluorescent dye and to HindIII-depleted BglII.<br>
representations of the same DNA labeled with another fluorescent dye.<br>
[0148] Fig. 1 illustrates results obtained with BglII. representations depleted by<br>
HindIII. In Fig. 1 A, we graph the ratios of hybridization intensity of each probe<br>
along the Y-axis. Each experiment was performed in color reversal, and the<br>
geometric mean of ratios from the separate experiments is plotted. Probes<br>
predicted to detect fragments in both the full and depleted tepresentations did<br>
hybridize to both (Fig. 1 A; left). There were about 8,000 of these probes. Probes<br>
predicted not to detect probes in the depleted leppesentation did not (Fig. 1 A;<br>
right). There were about l,800 of these probes. These results validate that: (l) the<br>
restriction profile of representational fragments was correctly predicted, (2) the<br>
oligonucleotides were correctly arrayed, and (3) the ohgonucleotides detected the<br>
predicted probes with acceptable signal intensity. In Fig. 1B, the agreement<br>
between the ratios of the color reversal experiments is graphed. These data<br>
confirm the reproducibility of our array.<br>
[0149] A very small number of oligonucleotide probes failed to hybridize to<br>
target fragments in the representations as predicted. For instance, of the 8000<br>
probes predicted to hybridize to fragments notxleaved by HindIII, about 16<br>
appeared to hybridize BglII fragments that were in fact cleaved. This could be due<br>
to a divergence between our sample and the published human sequence, which<br>
could result from polymorphism or sequencing errors. However, the data here<br>
show that the public human sequence is sufficiently reliable for the design of<br>
probes for representational oligonucleotide microarrays.<br>
Example 6 - Global Analysis of Tumor Genomes<br>
[0150] The oligoncleotide arrays of the invention readily detect large scale<br>
genomic lesions, whether they are deletions or amplifications, figs. 2A1-A3,2B1-<br>
B3, and 3C1-3C3 show the array hybridization data for three genomic<br>
comparisons: Figs. 2Al-A3 compare aneuploid-breast-cancer cells to normal,<br>
diploid cells from the same biopsy (CHTN159) (the two sample reprecntetions<br>
were prepared from about 100 ng of DNA,-each isolated from nuclei from aneuplid<br>
and diploid fractions separated by flow-cytometry); Figs. 2B1-B3 compare a breast<br>
cancer cell line (SK-BR.-3) derived from a patient of unknown ethnicity to an<br>
unrelated normal male J. Doe (of mixed European and African parentage; see<br>
Example 2); and Figs. 2C1-C3 compare cells from another normal male (African<br>
pygmy) vs. the same J. Doe. In each case, the samples were hybridized twice, with<br>
color reversal, and the geometric mean ratio (on a log scale) was plotted against the<br>
genome order of the oligonucleotide probes. Increased-copy number<br>
(amplification) is indicated by a ratio above 1, and dacreased copy number<br>
(deletion) by a ratio below 1. Data shown in Figs. 2A1,2B1 and 2C1 were<br>
obtained with the 10K print arrays. Data shown in Figs. 2A2,2B2 and 2C2 were<br>
obtained with the 85K photoprint arrays.<br>
[0151] There were clear profiles to the cancer genomes. The profiles of the two<br>
breast cancer lines were distinct, but each showed large regions of amplification<br>
and deletion in the genome (Figs. 2A1-A2 and 2B1-B2). In contrast, the profile-of<br>
the normal-normal was essentially flat, indicating no large scale amplifications or<br>
deletions between these genomes (Figs. 2C1-C2). These data-confirm that the<br>
oligonucleotide arrays of the invention can detect large scale genomic changes.<br>
[0152] The results also indicate that there were many oligonucleotide probes<br>
detecting minor losses and gains in all three genomes (the two cancer genomes and<br>
the African male's genome). These losses and gains are shown as stand-alone dots<br>
in Figs. 2A1-A2,2B1-B2 and 3C1-C2, and are manifest in Fig. 2C2 (the normal-<br>
normal comparison) as a "shell" or zone of probes that approach the ratios of 0.5<br>
and 2.0 throughout the genome. These losses and gains were likely the result of<br>
heterozygous BglII polymorphism between the individuals sampled.<br>
[0153] Further, comparison between the 10K print format and the 85K photoprint<br>
format clearly demonstrates that, although they had different resolutions, both<br>
captured a similar view of the lairge scale genomnrc features. We call Vpiobes<br>
"brothers" if they share-complementarity to the same BglII fragment. Brothers do<br>
not necessarily have overlapping sequences, although they may overlap by up to<br>
half of their length, or may be complementary across their entire length, in Figs.<br>
2A3,2B3 and 2C3, we plot the ratios of brother oligonucleotides from the 10K<br>
format (Y-axis) to ratios of their brother oligomrcleotitles from the 85K format (X-<br>
axis). There were in excess of 7,000 brother probes. There was rcmarkable<br>
agreement between the ratios of brother probes in the two formats Tor all three<br>
experiments, in spite of the facts that the probe sequences differed between<br>
formats, that their patterns of arraying were different, that the hybridization<br>
conditions differed, and the surfaces of the array were different. These data<br>
confirm the reproducibility of results obtained using arrays comprising<br>
oligonucleotides of the invention.<br>
[0154] In addition, analysis of MOMA representations produced by cleavage<br>
with McrBC showed regions of the genome with altered methylation status<br>
between the cancer cell and normal cell genomes. Normalization to copy number<br>
differences in these regions using the BglII representation-confirmed that the<br>
observed difference at many of these sites was due to a difference in methylation<br>
status and not in copy number.<br>
Example 7 - Automated Segmentation and Whole Genome Analysis<br>
[0155] We also analyzed the data from smaller regions of the genome to map the<br>
variation seen in Example 6. For example, we analyzed the data from one<br>
chromosome at a time using a statistical segmentation algorithm that parses the<br>
probe ratio data into segments of similar mean after taking variance into account<br>
(termed circular binary segmentation (CBS); see Olshen and Venkairaman,<br>
Change-Point Analysis of Array-Based Comparative Genomic Hybridization Data,<br>
Alexandria, VA, American Statistical Association, 2D02). The algorithm<br>
recursively identifies the best possible segmentation from each chromosome,<br>
rejecting or accepting-each proposed split based on the probability that the<br>
difference in mean could have arisen by chanee. This probability is determined by<br>
a randomization method Due to its nonparametric nature, the algorithm prevented<br>
us from identifying aberrations recognized by fewer than three probes.<br>
[0155] Figs. 3 A-D illustrate the output for this analysis on four chromosomes<br>
(chromosomes 5,8,17 and X in Figs. 3A-D, respectrvely) of the caheer cell line<br>
SK-BR-3 using the 85K array. We observed similar segmentation profiles and<br>
segment means when we used the data from the 10K array. Further lanalysis of the<br>
data enabled us to determine the ploidy level of the cells.<br>
[0157] Once segmented, we assigned to each obgonucleotide the mean ratio of<br>
the segment to which it belonged and plotted the mean-ratios in sorted order.<br>
These data are plotted for the cancer genomes of CHTN159 (Fig 4A) and SK-BR-<br>
3 (Fig. 4C). The figures demonstrate that the segment mean-ratios within-each<br>
genome were quantized, with major and minor plateaus of similar value. We<br>
deduced the copy number of these regions based on counting and the knowledge<br>
by flow analysis that CHTN159 was sub-triploid and SK-BR-3 was tetraploid. If<br>
each sample was roughly monoclonal, then the two major plateaus in-CHTN159<br>
would be two and three copies per cell, and the major plateaus SK-BR-3 would be<br>
three and four copies per cell.<br>
[0158] We used the copy number calculated for the major plateaus to solve the<br>
ploidy and Sn for each experiment. We used an equation:<br>
Rm = (RtxSn+1)/(Sn+1)<br>
where Rm was the mean measured ratio, Rt was the true ratio, and SN was an<br>
experimentally derived character measuring "specific to non-specific" noise. We<br>
selected Rm as the average of the probes of the segments in the plateau and set RT<br>
to Cn/P, where Cn was the true copy number known from the plateau, and P was<br>
the ploidy of the tumor genome. The combination provided two equations and two<br>
unknowns, P and Sn. For the CHTN159 experiment (Fig. 4A), we calculated the<br>
ploidy P to be 2.60, and SN to be 1.13. For the SK-BR-3 experiment (Fig. 4C), we<br>
calculated that P was 3.93, and SN was 1.21.<br>
[0159] We also used the equation to calculate what mean-ratios would be<br>
predicted for higher and lower copy numbers. We marked these predicted values<br>
on the respective graphs, from zero to a copy number of 12, with horizontal lines<br>
forming a "copy number lattice." The assigned mean-segment values'for probes<br>
are displayed in genome order, embedded with the expeeted copy number lattioe,<br>
in Figs. 4B and 4D. The copy number lattice fits remarkably well the minor<br>
plateaus of the data, especially for the higher copy numbers.<br>
ExampJe 8 - Analysis of Fine-Scale Gepomic Lesions<br>
[0160] We also analyzed the data to determine precise breakpoints in individual<br>
chromosomes that had amplifications or deletions. Our analysis demonstrated that<br>
the arrays of the invention can be used to identify genomics lesions at the<br>
resolution of individual genes. Accordingly, data obtained from the arrays may be<br>
used to predict the impact of aberrations in particular genes on conversion of a<br>
normal cell to a cancerous one.<br>
[0161] We first analyzed a region of a break in the X chromosome, seen in Fig.<br>
3D. SK-BR-3 cells, which were derived from a female, were compared to cells of<br>
an unrelated male. We expected that probes in chromosome X would have<br>
elevated ratios. This was the case through much of the long arm of chromosome<br>
X. But in the midst of Xq 13.3, there was a sharp break in copy number over a<br>
region spanning 27kb and ratios near one were observed for the remainder of the<br>
chromosome (Fig. 5 A). Thus, it was possible to draw boundaries of-genetic<br>
lesions from array data by segmentation. We have observed many other instances<br>
of sharp copy number transitions that must break genes.<br>
[0162] There were three to four narrow amplifications in the SK-BR-3 genome,<br>
each containing two or fewer genes, among which were transmembrane receptors.<br>
[0163] We then analyzed the data from chromosome 8 {Fig. 3B), which had an<br>
abundance of aberrations, including broad, distinct regions of amplification (Fig.<br>
5B). The rightmost peak was approximately a one megabase stretch, comprised of<br>
thirty-seven probes (probe coordinates 45099-45138, June genome-coordinates<br>
126815070-128207342). Yet it contained a single well characterized gene, c-myc.<br>
[0164] There was a second broad peak in SK-BR-3, ascending to the left of the c-<br>
myc peak, and off the graph (Fig. 5B). This broad peak had a broad shoulder on its<br>
right (probe coordinates 44994-45051, June genome coordinates 123975563-<br>
125564705), with a very narrow peak in its midst. We overlaid on this the<br>
segmentation data from the tumor genome, CHTN159, which had an even broader<br>
peak encompassing c-myc (probe coordinates 44996-45131, June genomic<br>
coordinates 1240735E5-127B28283). The peak in CHTN 159 also encompassed the<br>
shoulder of the second SK-BR-3 peak (Fig. 5B). Thus the shoulder may contain<br>
candidate oncogenes that merit attention. Within that region, at the narrow peak,<br>
we found TRC8, the target of a translocartion implicated in hereditary renal<br>
carcinoma (Gemmill et al. Proc. Natl. Acad. Sci. USA 95:9572-7 (l996)). these<br>
results illustrate the value of coordinating data from multiple genomes, and the<br>
need for automated methods for analyzing multiple data sets.<br>
[0165] We also analyzed a narrow deletion on chromosome5. Fig.5C shows the<br>
results of a combined 10K (emptyxircles) and 85K (filled circles) analysis overlaid<br>
on a copy number lattice. A deletion was evident at both the 10K and 85K<br>
resolutions (probe coordinates 29496-29540, June genomic coordinates 14231414-<br>
15591226), but the boundaries were much more clearly resolved at 85K. This<br>
region contained TRIO, a protein having a GEF domain, an SH3 domain, and a<br>
serine threonine kinase domain (Lin and Greenberg, Cell 101:230-42-(3000));<br>
ANKH, a transmembrane protein (Nurnberg et al., Nat.Genet 28:37-41 (2001));<br>
and F3XL, a component of the ubiquitin ligase mediated protein degradation<br>
pathway (IIyin et al., Genomics 67:40-47 (2000)).<br>
[0166] Finally, we analyzed a region of homozygous loss on chromosome 19<br>
affecting a cluster of zinc finger proteins (Fig. 5D; probe coordinates 77142-77198,<br>
June genomic coordinates 21893948-24955961). Some of these genes may encode<br>
transcription factors, whose deletion may havecole in tumorigenesrs. We observed<br>
an abundance of narrow hemizygous and homozygous lesions, some of which<br>
might be attributable to normal variation. See Example 9.<br>
Example 9 - Examining "Normal" Genomic Variation<br>
[0167] We also used the oligonucleotide arrays and methods of this invention to<br>
analyze copy number variation between two normal genomes and observed<br>
differences resulting from polymorphic variation. This analysis is important, e.g.,<br>
in situations where a tumor ONA sample cannot be matched against normal ONA<br>
and an unrelated normal DNA is used as a reference because differences observed<br>
may be the result of polymorphic variation. This variation can be of two sorts,<br>
point sequence variation of the sort that creates or destroys a BglII fragment, e.g.,<br>
 SNPs, or actual copy number fluctuation present in the human gene pool. The<br>
former has limited impost on analyses using the areays of the inyention as it will<br>
produce scattered "noise" that can lasgely be filteted by statistical means.<br>
[0158] in Fig. 6A (combined data from the 10K and 85K datasets), we<br>
demonstrate that a mild filtration algorithm (if a ratio was me most deviant of the<br>
surrounding four, we replaced it with the closer ratio of its two neighbors) can<br>
minimize the impact of point sequence variation and detect instances where there<br>
is actual copy number variation. The cloud of scattered polymorphisms present in<br>
an unfiltered sample (e.g., Fig. C2) is lifted in this presentation of the data<br>
revealing non-random clusters of deviant probe ratios, indicating large scale<br>
genomic differences between normal individuals.<br>
[0169] Polymorphic variation of the scattered variety can also be filtered by<br>
serial comparison of experiments. For example, Fig. 6B shows data from SK-BR-<br>
3 compared to normal donor, J. Doe, the 85K ratios displayed in filled circles, and<br>
the 1 OK in empty circles. On the same graph we display the ratios of J. Doe<br>
compared to another normal, DNA from an African pygmy, in green triangles. We<br>
see three probes of extreme ratio in the SK-BR-3-normal hybridization that can be<br>
identified as polymorphisms by comparison to hybridization between the two<br>
normal individuals. The simplest interpretation of these data is that J. Doe is +/+,<br>
pygmy +/- and SK-BR-3 -/- where + designates the presenge of a small BglII<br>
fragment (most likely a SNP at a BgIII site). In general, pairwise comparisons of<br>
three genomes allow interpretable calls of allele status. Thus, these kinds of data<br>
are especially useful when a malignant genome cannot be paired to a matched<br>
normal.<br>
[03 70] Polymorphism in copy number, however, presents a different "sort of<br>
problem. Fig. 6A demonstrates large regional differences in copy number in the<br>
normal-normal comparison. We applied "segmentation analysis to these data and<br>
identified multiple regions that display altered copy number between the two<br>
normal individuals. We observed about a dozen variant regions in any normal-<br>
normal comparison. They stretch from one hundred kilobases to greater than a<br>
megabase in length, can occur anywhere but are more frequently observed near<br>
telemeres and centromeres, and often encompass known genes.<br>
[0171] Close inspections of two such regions are displayed in Fig. 6C and Fig.<br>
6D, with ratios as connected circles and segmentation values as lattice. In Fig. 6C,<br>
the abnormal region is 135kb on chromosome 6p21 (probe coodinates 32518-<br>
32524, June genomic coordinates 35669083-35804705), and encompasses three<br>
known genes. In Fig. 6D, the region is a 620kb region from chromosome 3pl 1<br>
(probe coordinates 9927-9952, June genomic coordinates 887694-89385815)<br>
that contains a number of heavy chain variable regions.<br>
[0172] We analyzed the impact of the normal-normal variation on the<br>
interpretation of cancer-normal data, in Fig. 6C and Fig. 6D we have overlain the<br>
segmentation values from the analysis of SK-BR-3 in diagonal and vertical hatch,<br>
respectively. The copy number lattice for SK-BR-3 is plotted as lattige. Fig. 6C<br>
illustrates a region in SK-BR-3 that would be called a deletion in comparison to the<br>
normal. In SK-BR-3 the flanking region occurs at a copy number that we judge to<br>
be two copies per cell, and within that region, copy number becomes reduced to<br>
one. But the same region appears in the comparison of pygmy DNA to that<br>
normal. In Fig. 6D we observe an analogous condition on chromosome 2pl 1. In<br>
Fig. 6D we have also plotted segmentation data from the tumor. This region is<br>
evidently abnormal there as well.<br>
Example 10 - Annotation of a Genome of aJPortion Thereof<br>
[0173] The following examples are meant to illustrate uses the search engine.<br>
Suitable modifications and adaptations of the described conditions and parameters<br>
normally encountered in the art which are obvious to those skilled in the art are<br>
within the spirit and scope of the present invention.<br>
[0174] The search engine of the present invention can be used to perform<br>
computations on a genome or on subsets of a genome (e.g., a chromosome). In<br>
performing these computations, several regions having high word counts are found<br>
which are not detected by search tools such as Repeat Masker. It has been shown<br>
that the database of repeats used by Repeat Masker does not include region specific<br>
or chromosome specific repeats. Using the search engine described above in<br>
section VH, such repeats are easily found because exact mateh counting can form<br>
the basis For set algebra of the genome, in particular, a subset of the-genome can<br>
be made into transform strings, which ape examined to find chromosome specific<br>
repeats.<br>
[0175] A transform string from chromosome 1 was annotated with the word<br>
counts within itself and within the entire-genome. A search was performed to look<br>
for contiguous regions of chromosome 1, at least 100 bp in length, with .high 18-<br>
mer counts in which the exact matches were found to derive mainly ftom<br>
chromosome 1. Such regions were readily found, ranging in length from 100 bp to<br>
35 kb. Focusing on one such region, it was observed that its mer terrain was nearly<br>
a step function, composed of shorter sequences each with a signature modal<br>
frequency and length. The chromosome-specific legions containing one of these<br>
signature regions were collected and a family of chromosome 1 specific sequences<br>
was quickly identified. The chromosome 1 specific Tegion was selected by<br>
identifying 18 mers whose chromosome 1 counts exceeded 90% of their whole<br>
genome counts, these 18 mers were strung together to create the chromosome<br>
specific repeat. In addition, the space between the 18 mers that were strung was<br>
not allowed to exceed 100 base pairs. It was found at least once that this repeat has<br>
been annotated as overlapping a RefSeq-gene (acoession number NM_015383),<br>
with many exons that together encode a large predicted protein sequence having<br>
low homology to myosin.<br>
[0176] The same process by which chromosome specific repeats are identified<br>
can be applied to finding repetitive DNA throughout the genome, including those<br>
that are not recognized by Repeat Masker or other programs.<br>
Example 11- Probe Design Using the Mer Engine<br>
[0177] The above-mentioned search engine can be used in probe design. Probes<br>
are generally useful for their ability to hybridize specifically to complementary<br>
DNA, and therefore one of the primary objectives in probe design is to minimize<br>
cross hybridization. Previous probes designing applications have used-repeat<br>
masking to exclude repeat regions from consideration. This type of solution is<br>
problematic, in that it does not provide protection from regions that are repetitive<br>
such as chromosome specific repeats, and it excludes "repetitive" regions that are<br>
unique.<br>
[0178] While rules for hybridization between imperfectly matched-sequeriees are<br>
not well understood, it is known in the art that probes that have exact "small"<br>
matches to multiple regions of the genome should preferably be avoided. previous<br>
probe applications have chosen probes that minimise aggtegate exact 12 mer<br>
match counts, but for genomic probes, these methods are inadequate. Fast, it is<br>
unclear that exact matches of 12-mers have any effect on hybridization under<br>
normally stringent annealing conditions. Nor do 12 mer counts predict homology,<br>
let alone uniqueness in the genome. In fact, a comparison of 15-mer counts to the<br>
geometric mean of counts from their constituent 12-mers yielded a pooreonelation<br>
between two sequences that are essentially unique.<br>
[0179] A general protocol for probe design using the mer engine is described as<br>
follows. First, the genome is annotated according to a particular length mer such<br>
that sufficiently long stretches of uniqueness are found (i.e., candidate probes).<br>
Second, these candidate probes are annotated using at least one predefined length<br>
mer, preferably of a length shorter than the mer length used to find the candidate<br>
probes. One of the candidate probes is selected as the probe based on the<br>
minimum aggregate mer-counts of the predefined shorter lengths.<br>
[0180] Following the above mentioned protocol, 70-mer-candidate probes were<br>
selected from small BglII fragments, using uniqueness data obtained from 21-mer<br>
counts. Within these candidate probes, a 70-mer was selected with the lowest sum<br>
of 15-mer counts, with a cut off value of about 900. Additional'criteria that<br>
eliminated runs of single nucleotides and severe base composition bias were also<br>
applied to assist in determining which candidate probe to choose. Selected probes<br>
were synthesized and printed on glass to test their performance in micro-array<br>
hybridization conditions. It was found that substantially all the probes performed<br>
at or above specified performance-criteria. More particularly, a success rate of<br>
about 70% to about 98% was achieved with the probes designed using the above <br>
mentioned protocol, where success is defined as having a substantial (e.g., Jarge)<br>
signal/noise ratio.<br>
[0181] BLAST was used to test whether the selected probes were unique within a<br>
particular published genome sequence. 30,000 such probes were tested using the<br>
default parameters for MegaBLAST (filtration ofsimple sequence was turned off).<br>
It was found that greater than 99% of the selected probes were unique within the<br>
genome.<br>
Example 12 - Pseudo Code Representation of Algorithm<br>
[0182] To further illustrate how the algoriftm can be implernented to perform a<br>
word counting function, refer to FIGS. 12A scad 12B. FIG. 12A grapfteally<br>
defines variables and data structures used by the algorithm and FIG. 12B shows a<br>
pseudo code representation of the algorithm. As indicated above in section VII,<br>
the transform can be used as a navigational tool for a virtual Genome Dictionary<br>
or suffix array. In the simplest case, assume that it isdesited to determine whether<br>
a substring occurs in the genome, and if so, in how many copies. In this case,<br>
assume that the substring is the single character, "X." All occurrences of X can be<br>
viewed in the dictionary as a block (e.g., search region) where Fx and Lx are the<br>
indices of the first and last occurrence of X. Fx and Lx can be derived from the<br>
alphabounds data structure. The size of this block (e.g., searchr region) is kx = Lx-<br>
Fx +1, is also the number of occurrences of X. Note that this number can be<br>
determined by counting the number of occurrences of X in the transform.<br>
[0183] In a more difficult case, such as when two or more character words are to<br>
be counted, the Fx, Lx, and kx of each character X in the genome need to be<br>
determined. In other words, the Fx and Lx for each character X is stored in the<br>
data structure called alphabounds. Once the alphabounds data sfcucture is<br>
constructed, the algorithm can proceed to count the number of times a particular<br>
word, Z, occurs in the genome. Assume W is a suffix ofZ, W exists in the<br>
genome, and the alphabounds(e.g., Fw and Lwas show in FIG. 12A) of W are<br>
known. Next, a determination needs to be made as to whether XW exists as a<br>
substring, where X is the character preceding W in Z. In addition, the start and end<br>
indices (e.g., Fxw and Lxw) of the XW block need to determined.<br>
[0184] If and only if X occurs in the transform between Fw and Lw, then XW<br>
exists as a substring in the genome. Furthermore, the number of X's in the "W<br>
block" of the transform, indicated as kxw, is the word count of the substring XW in<br>
thegenome. The start and end indices of XW can be completed using: l)Fxw =<br>
Fx + bxw; and 2) Lxw—Fxw + kxw —1, where bxw is the number of words<br>
beginning with X in the Genome Dictionary that occur before XW. bxw can be<br>
determined by counting the number of Xs that occur before the W block of the<br>
transform.<br>
[0185] This procedure is reiterated, lengthening the suffx one charaeter at a<br>
time, stoppmg if the suffix does not exit in the Genome Diotionary. If the snffix<br>
W encompasses the entire word, Z, kw is the number of occmrenets of Z in the<br>
genomic string. An outline of this procedure is outline in pseudo code, as shown in<br>
FIG, 12B. With respect to FIG. 12B,Z is a string of leagth N,composed of<br>
characters from the genome alphabet, and the alphabounds data structure Contains<br>
the indices of the first and last occurrences in the genome dictionary for each<br>
character in the genome alphabet.<br>
[0186] Unless otherwise defined, all technical and scientfic ternts used hesein<br>
have the same meaning as commonly understood by one of ordinary skill in the art<br>
to which this invention belongs. All pubhcations and other references mentioned<br>
herein are incorporated by reference in their entirety. In case of conflict, the<br>
present specification, including definitions, will control. The materials, methods,<br>
and examples are illustrative only and not intended to be limiting. Throughout this<br>
specification, the word "comprise," or variations such as "comprises" or<br>
"comprising" will be understood to imply the inclusion of a stated integer or group<br>
of integers but not the exclusion of any other integer or group of integers.<br>
WE CLAIM:<br>
1. A plurality of nucleic acid molecules,<br>
wherein:<br>
(a) said plurality consists of N nucleic acid molecules;<br>
(b) each of said plurality of nucleic acid molecules has a nucleotide sequence that<br>
hybridizes specifically to a sequence in a genome of Z basepairs; and<br>
(c) at least P% of said plurality of nucleic acid molecules have<br>
(i) a length of K nucleotides;<br>
(ii) hybridizes specifically to at least one nucleic acid molecule present in or<br>
predicted to be present in a representation derived from said genome, said representation having<br>
no more than R% of the complexity of said genome; and<br>
(iii) between Y and X exact matches of L1 nucleotides to said genome; and<br>
wherein:<br>
(AN =500;<br>
(B)Z =lxlO8;<br>
(C) 300 = K &gt; 30;<br>
(D)70 = R = 0.001;<br>
(E) P = (N x R + (3 x sigma))/N;<br>
(F) sigma is the squareroot of (N x R% x (1-R%))<br>
(G) the integer closest to (log4(Z) + 2) = L1 = the integer closest to log4(Z);<br>
(H) X is the integer closest to D1 x (K - Li+1);<br>
(I) Y is the integer closest to D2 x (K - L1+1);<br>
(J) 1.5 =D1 =l;and<br>
(K)l =D2 =0.5.<br>
2. The plurality of nucleic acid molecules as claimed in claim 1, wherein N is selected<br>
from the group consisting of at least 500; at least 1,000; at least 2,500; at least 5,000; at least<br>
10,000; at least 25,000; at least 50,000; at least 85,000; at least 190,000; at least 350,000; and at<br>
least 550,000 nucleic acid molecules.<br>
3. The plurality of nucleic acid molecules as claimed in claim 1, wherein Z is selected<br>
from the group consisting of at least 3 x 108, at least 1 x 109, at least 1 x 1010 and at least 1 x<br>
10".<br>
4. The plurality of nucleic acid molecules as claimed in claim 1, wherein the genome<br>
is a mammalian genome.<br>
5. The plurality of nucleic acid molecules as claimed in claim 4, wherein the genome<br>
is a human genome.<br>
6. The plurality of nucleic acid molecules as claimed in claim 1, wherein R is selected<br>
from the group consisting of 0.001, 1, 2, 4, 10, 15, 20, 30, 40, 50 and 70.<br>
7. The plurality of nucleic acid molecules as claimed in claim 1, wherein P is selected<br>
from the group consisting of at least 70, at least 80, at least 90, at least 95, at least 97 and at least<br>
99.<br>
8. The plurality of nucleic acid molecules as claimed in claim 1, wherein D1 is 1.<br>
9. The plurality of nucleic acid molecules as claimed in claim 1, wherein D2 is 1.<br>
10. The plurality of nucleic acid molecules as claimed in claim 1, wherein L1 is<br>
selected from the group consisting of 15,16,17, 18,19, 20, 21, 22, 23 and 24.<br>
11. The plurality of nucleic acid molecules as claimed in claim 1, wherein<br>
each of said P% of said plurality of nucleic acid molecules further have no more than A<br>
exact matches of L2 nucleotides to said genome and no fewer than B exact matches of L2<br>
nucleotides to said genome; and wherein<br>
(a) L1 = L2 = the integer closest to log4(Z) - 3;<br>
(b) A is the integer closest to D3 x ((K - L2+l) x (Z / 4L2));<br>
(c) B is the integer closest to D4 x ((K - L2+l) x (Z / 4L2));<br>
(d)4 =D3 =l;and<br>
(e) 1 = D4 =0.5.<br>
12. The plurality of nucleic acid molecules as claimed in claim 11, wherein D3 = 3, 2,<br>
or 1.5.<br>
13. The plurality of nucleic acid molecules as claimed in claim 1, wherein said P% of<br>
said plurality of nucleic acid molecules have at least 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100%<br>
sequence identity to at least one nucleic acid molecule present or predicted to be present in said<br>
representation.<br>
14. The plurality of nucleic acid molecules as claimed in claim 1, wherein K is<br>
selected from the group consisting of 40, 50, 60, 70, 80, 90, 100, 110, 120, 140, 160, 180, 200<br>
and 250.<br>
15. A plurality of nucleic acid molecules, wherein:<br>
(a) said plurality consists of at least 100 nucleic acid molecules;<br>
(b) each of said plurality of nucleic acid molecules has a nucleotide sequence that is at<br>
least 90% identical to a sequence in a genome of at least Z basepairs; and<br>
(c) at least P% of said plurality of nucleic acid molecules have<br>
(i) a length of K nucleotides;<br>
(ii) at least 90% sequence identity to at least one nucleic acid molecule present in<br>
or predicted to be present in a representation derived from said genome, said representation<br>
having no more than R% of the complexity of said genome; and<br>
(iii) between Y and X exact matches of L1 nucleotides to said<br>
representation; and wherein:<br>
(A)Z=lx108;<br>
(B)300 =K =30;<br>
(C)70 =R= 0.001;<br>
(D) P = 90-R;<br>
(E) the integer closest to (log4((Z x R)/100) + 2) &gt; L1 &gt; the integer closest to log4((Z x<br>
R)/100);<br>
(F) X is the integer closest to D1 x (K - L1+1);<br>
(G) Y is the integer closest to D2 x (K - Lj+1);<br>
(H)1.5 =D1 =l;and<br>
(I)l= D2 =0.5.<br>
16. The plurality of nucleic acid molecules as claimed in claim 15, comprising at least<br>
500; at least. 1,000; at least 2,500; at least 5,000; at least 10,000; at least 25,000; at least 50,000;<br>
at least 85,000; at least 190,000; at least 350,000; or at least 550,000 nucleic acid molecules.<br>
17. The plurality of nucleic acid molecules as claimed in claim 15, wherein Z is<br>
selected from the group consisting of at least 3 x 108, at least 1 x 109, at least 1 x 1010 and at least<br>
1x1011.<br>
18. The plurality of nucleic acid molecules as claimed in claim 15, wherein the<br>
genome is a mammalian genome.<br>
19. The plurality of nucleic acid molecules as claimed in claim 18, wherein the<br>
genome is a human genome.<br>
20. The plurality as claimed in nucleic acid molecules of claim 15, wherein R is<br>
selected from the group consisting of 0.001, 1, 2, 4, 10, 15, 20, 30, 40, 50 and 70.<br>
21. The plurality of nucleic acid molecules as claimed in claim 15, wherein P is<br>
selected from the group consisting of at least 70, at least 80, at least 90, at least 95, at least 97<br>
and at least 99.<br>
22. The plurality of nucleic acid molecules as claimed in claim 15, wherein D1 is 1.<br>
23. The plurality of nucleic acid molecules as claimed in claim 15, wherein D2 is 1.<br>
24. The plurality of nucleic acid molecules as claimed in claim 15, wherein L1 is<br>
selected from the group consisting of 15, 16, 17, 18, 19, 20, 21, 22, 23 and 24.<br>
25. The plurality of nucleic acid molecules as claimed in claim 15, wherein<br>
each of said P% of said plurality of nucleic acid molecules further have<br>
between B and A exact matches of L2 nucleotides to said genome and wherein<br>
(a) L1 = L2 = the integer closest to log4(Z) - 3;<br>
(b) A is the integer closest to D3 x ((K - L2+l) x (Z / 4L2));<br>
(c) B is the integer closest to D4 x ((K - L2+l) x (Z / 4L2));<br>
(d) 4 = D3 = 1; and<br>
(e) 1 = D4 = 0.5.<br>
26. The plurality of nucleic acid molecules as claimed in claim 15, wherein said P%<br>
of said plurality of nucleic acid molecules have at least 91, 92, 93, 94, 95, 96, 97, 98, 99 or 100%<br>
sequence identity to at least one nucleic acid molecule present or predicted to be present in said<br>
representation.<br>
27. The plurality of nucleic acid molecules as claimed in claim 15, wherein K is<br>
selected from the group consisting of 40, 50, 60, 70, 80, 90, 100, 110, 120, 140, 160, 180, 200<br>
and 250.<br>
28. The plurality of nucleic acid molecules as claimed in claim 1, wherein said<br>
representation is produced by sequence-specific cleavage of said genome.<br>
29. The plurality of nucleic acid molecules as claimed in claim 28, wherein sequence-<br>
specific cleavage is accomplished with a restriction endonuclease.<br>
30. The plurality of nucleic acid molecules as claimed in claim 1, wherein said<br>
representation is a compound representation.<br>
31. The plurality of nucleic acid molecules as claimed in claim 1, wherein said<br>
plurality of nucleic acid molecules are immobilized on the surface of a solid phase.<br>
32. The plurality of nucleic acid molecules as claimed in claim 31, wherein said solid<br>
phase is selected from the group consisting of a nylon membrane, a nitrocellulose membrane, a<br>
glass slide, and a microsphere.<br>
33. The plurality of nucleic acid molecules as claimed in claim 31, wherein the<br>
positions of said plurality of nucleic acid molecules on said solid phase are known.<br>
34. The plurality of nucleic acid molecules as claimed in claim 33, wherein said<br>
plurality of nucleic acid molecules is on a microarray.<br>
35. The plurality of nucleic acid molecules as claimed in claim 33, wherein said<br>
plurality of nucleic acid molecules is immobilized on microspheres.<br>
36. A method of analyzing a nucleic acid sample, said method comprising:<br>
(a) hybridizing the sample to the plurality of nucleic acid molecules as claimed<br>
in claim 1; and<br>
(b) determining to which of said plurality of nucleic acid molecules said sample<br>
hybridizes.<br>
37. The method as claimed in claim 36, wherein said sample is a representation.<br>
38. The method as claimed in claim 36, wherein said plurality of nucleic acid<br>
molecules is immobilized on the surface of a solid phase.<br>
39. The method as claimed in claim 38, wherein said solid phase is selected from the<br>
group consisting of a nylon membrane, a nitrocellulose membrane, a glass slide, and a<br>
microsphere.<br>
40. The method as claimed in claim 38, wherein the positions of said plurality of<br>
nucleic acid molecules on said solid phase are known.<br>
41. The method as claimed in claim 40, wherein said plurality of nucleic acid<br>
molecules is on a microarray.<br>
42. The method as claimed in claim 38, wherein said plurality of nucleic acid<br>
molecules is immobilized on microspheres.<br>
43. A method of analyzing copy number variation of a genomic sequence between two<br>
genomes, said method comprising:<br>
(a) providing a first genome and a second genome;<br>
(b) preparing detectably labeled representations of each genome using at least one<br>
identical restriction enzyme;<br>
(c) contacting said representations with the plurality of nucleic acid molecules<br>
as claimed in claim 1 or 31 to allow hybridization between the representations and said plurality<br>
of nucleic acid molecules; and<br>
(d) comparing levels of the hybridization of said representations, wherein a<br>
difference in said levels indicates a copy number variation between the two genomes with regard<br>
to a genomic sequence targeted by said member.<br>
44. The method as claimed in claim 43, wherein the two representations are<br>
distinguishably labeled.<br>
45. The method as claimed in claim 44, wherein said representations are<br>
simultaneously contacted with said plurality of nucleic acid molecules.<br>
46. A method of comparing methylation status of a genomic sequence between two<br>
genomes, said method comprising:<br>
(a) providing a first genome and a second genome;<br>
(b) preparing detectably labeled representations of each genome using at least one<br>
identical enzyme, wherein said representations are prepared by a methylation-sensitive method;<br>
(c) contacting said representations with the plurality of nucleic acid molecules of<br>
claim 1 or 31 to allow hybridization between the representations and said plurality of nucleic<br>
acid molecules; and<br>
(d) comparing levels of the hybridization of said representations, wherein a<br>
difference in said levels indicates a difference in methylation status between the two genomes<br>
with regard to a genomic sequence targeted by said member.<br>
47. The method as claimed in claim 46, wherein said methylation sensitive method<br>
involves preparing a first representation using a first restriction enzyme and a second<br>
representation using a second restriction enzyme, wherein said first and second restriction<br>
enzymes recognize the same restriction site but one is methylation-sensitive and the other is not.<br>
48. The method as claimed in claim 46, wherein said methylation sensitive method<br>
involves chemical cleavage of methyl-C sequences after making a representation with a non-<br>
methylation sensitive restriction enzyme, such that a representation derived from a methylated<br>
genome is distinguishable from a representation derived from a non-methylated genome.<br>
49. A method of identifying an oligonucleotide that has:<br>
(a) a length of K nucleotides;<br>
(b) at least 90% sequence identity to at least one nucleic acid molecule present in or<br>
predicted to be present in a representation derived from a genome of at least Z basepairs, and<br>
(c) between Y and X exact matches of L, nucleotides to said genome; and<br>
wherein :<br>
(i)Z =1 x 108;<br>
(ii) 300 =K =30;<br>
(iii) the integer closest to (log4(Z) + 2) = L1 = the integer closest to log4(Z);<br>
(iv) X is the integer closest to D1 x (K - L1+1);<br>
(v) Y is the integer closest to D2 x (K - L1+1);<br>
(vi) 1.5 =D, =l;and<br>
(vii) 1 = D2 = 0.5;<br>
the method comprising:<br>
(A) cleaving said genome in silico with a restriction enzyme to generate a plurality of<br>
predicted nucleic acid molecules,<br>
(B) generating a virtual representation of said genome by identifying predicted nucleic<br>
acid molecules each having a length of 200-1,200 basepairs, inclusive;<br>
(C) selecting an oligonucleotide having a length of 30-300 nucleotides, inclusive, and at<br>
least 90% sequence identity to a predicted nucleic acid molecule in (B);<br>
(D) identifying all of the stretches of L1 nucleotides occurring in said oligonucleotide;<br>
and<br>
(E) confirming that the number of times each of said stretches occurs in said genome<br>
satisfies the requirements of (c).<br>
50. The method as claimed in claim 49, wherein step (E) comprises:<br>
providing a compressed transform of said genome;<br>
providing an auxiliary data structure that includes information related to said<br>
genome; and<br>
determining a word count for the L1 nucleotides using the compressed transform<br>
and the auxiliary data structure.<br>
51. The method as claimed in claim 49, wherein step (E) comprises:<br>
providing a compressed transform of said genome;<br>
iterating through each nucleotide of said stretch of L1 nucleotides, starting with<br>
the last nucleotide and advancing to the first nucleotide one character per iteration, wherein the<br>
nucleotide corresponding to a particular iteration is stored as-an index nucleotide, said iterating<br>
further comprising:<br>
defining a search region that delineates a contiguous range of nucleotides<br>
within said transform;<br>
counting the number of times the nucleotide preceding said index<br>
nucleotide occurs in said search range; and<br>
wherein said iterating ceases if no occurrences of the nucleotide preceding<br>
said index nucleotide occurs in said search range; and<br>
outputting the number of times the first nucleotide of said stretch of L1<br>
nucleotides is counted, this number being equivalent to the number times said stretch of L1<br>
nucleotides appears in said genome.<br>
52. The method as claimed in claim 51, further comprising:<br>
providing an auxiliary data structure, said auxiliary data structure comprising:<br>
a K-intervals data structure that maintains a running total of each<br>
nucleotide that has appeared in said transform up to and including a particular predetermined<br>
location in said compressed transform; and<br>
a dictionary-counts data structure that provides fast look-up access to the<br>
compressed transform; and<br>
wherein said counting and said defining are performed using said auxiliary data<br>
structure and said compressed transform.<br>
53. The method as claimed in claim 52, wherein said transform remains compressed<br>
while said counting is being performed.<br>
54. The method as claimed in claim 52, wherein said compressed transform is<br>
compressed such that every three characters in the uncompressed transform are compressed to<br>
form a byte, and wherein said counting uncompresses at most one such byte during one of said<br>
iterations.<br>
55. The method -------------- as claimed in claim 51, wherein said genome comprises at<br>
least three billion characters.<br>
56. The method-------------- as claimed in claim 51, wherein said compressed transform<br>
is a Burrows-Wheeler transform of the genome.<br>
57. The method ----------— as claimed in claim 51, further comprising providing data<br>
which is based on said transform, wherein said defining comprises using said data and said index<br>
nucleotide to define said search region.<br>
58. The method ------------- as claimed in claim 51, further comprising:<br>
providing data which is based on said transform; and<br>
determining a prior nucleotide count, said prior nucleotide count being the<br>
number of times the nucleotide preceding the index nucleotide occurs in said transform before<br>
the beginning of said search region;<br>
wherein said defining comprises using said data, said index nucleotide, and said<br>
prior nucleotide count to define said search region.<br>
59. The method ------------- as claimed in claim 58, wherein said prior nucleotide count<br>
is obtained using K-intervals, said K-intervals being stored at predetermined locations along said<br>
transform and maintain a running total of each nucleotide that has appeared in said transform up<br>
to and including a particular predetermined location.<br>
60. A plurality of oligonucleotides each of which produced by the method as claimed<br>
in claim 49, said plurality comprising at least 500 oligonucleotides.<br>
61. A plurality of oligonucleotides each of which produced by the method ' as claimed<br>
in claim 49, said plurality comprising at least 1,000; at least 2,500; at least 5,000; at least 10,000;<br>
at least 25,000; at least 50,000; at least 85,000; at least 190,000; at least 350,000; or at least<br>
550,000 oligonucleotides.<br>
The invention provides oligonucleotide probes that can be used to hybridize to a<br>
representation of nucleic acid sequences. Compositions containing the probes such as<br>
microarrays are also provided. The invention also provides methods of using these<br>
probes and compositions in therapeutic, diagnostic, and research applications.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1LT0xOUC0yMDA1LUNPUlJFU1BPTkRFTkNFLnBkZg==" target="_blank" style="word-wrap:break-word;">2358-KOLNP-2005-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1LT0xOUC0yMDA1LUZPUk0gMjcucGRm" target="_blank" style="word-wrap:break-word;">2358-KOLNP-2005-FORM 27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1LT0xOUC0yMDA1LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">2358-KOLNP-2005-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtcGEucGRm" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-pa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjM1OC1rb2xucC0yMDA1LWdyYW50ZWQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2358-kolnp-2005-granted-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234004-a-method-for-preparing-a-dermatological-compostion.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234006-agent-for-processing-synthetic-fibers-and-method-for-processing-sysnthetic-fibers.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234005</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2358/KOLNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Nov-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COLD SPRING HARBOR LABORATORY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1 BUNGTOWN ROAD, COLD SPRING HARBOR, NY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>WIGLER MICHAEL H</td>
											<td>52 SNAKE HILL ROAD, COLD SPRING HARBOR, NY 11724</td>
										</tr>
										<tr>
											<td>2</td>
											<td>HEALY JOHN</td>
											<td>32 DALY ROAD, EAST NORTHPORT, NY 11731</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LUCITO ROBERT</td>
											<td>1477 ERIC LANE, EAST MEADOW, NY 11554</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/016060</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/472,845</td>
									<td>2003-05-23</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/472,843</td>
									<td>2003-05-23</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234005-virtual-representations-of-nucleotide-sequences by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:01:09 GMT -->
</html>
