<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234034-a-novel-set-of-nucleotide-primers-for-phosphotidyl-inositol-in-bacillus-cereus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:02:06 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234034:A NOVEL SET OF NUCLEOTIDE PRIMERS FOR PHOSPHOTIDYL INOSITOL IN BACILLUS CEREUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A NOVEL SET OF NUCLEOTIDE PRIMERS FOR PHOSPHOTIDYL INOSITOL IN BACILLUS CEREUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a novel oligonucleotide primer for phosphotidyl inositol in Bacillus cereus. The present invention also relates to to a method for the detection of Bacillus cereus in food.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of invention<br>
A novel set of nucleotide primers for phosphotidyl inositol in bacillus cereus.   The present invention also relates to a method for the detection of Bacillus cereus in food. Background of the invention<br>
Among the predominantly occurring food borne pathogenic bacteria, Bacillus cereus, an opportunistic pathogen has been found to occur abundantly in Indian foods and also cause illnesses like diarrhea and/or emesis (Rakh et al. 1988). The illness has been attributed to the presence of enterotoxins and other toxins including haemolysins elaborated by strains of B. cereus. Conventionally, B. cereus is detected by its ability to grow on selective plating media containing egg yolk and inability to utilize mannitol. The isolates are further identified by morphological, cultural and biochemical characteristics. (Duguid, 1996).<br>
Advances made in detection methods have led to the use of polymerase chain reaction (PCR) for the specific detection of b. cereus. PCR protocols have been developed for the detection of B. cereus group of bacteria in pure culture systems and food samples using specific sets of primers.<br>
Reference is made to the work of Schrafts and Griffiths (1995), wherein primers for the cereolysin AB gene (M 24149) of B. cereus was designed. The detection limit for B. cereus by PCR in artificially contaminated milk samples was 103 CFU/ml without enrichment of the milk.<br>
Reference is made to the works of Agata et al. (1995) and Mantynen and Lindstrom (1998), wherein primers for the BceT gene was designed and used to study the distribution of the toxin gene in clinical and food isolates of B. cereus. Only qualitative observations were made on this work and no quantification has<br><br>
been reported. It was also postulated that the BceT gene could not be targeted to assess the enterotoxic potential of B. cereus strains.<br>
Reference is made to the work of Wang et al. (1997), wherein a universal protocol for PCR detection of a number of food borne pathogenic bacteria was devised using haemolysin as the target gene. Detection of toxin producing strains of B. cereus was accomplished using these primer, following overnight enrichment of various food samples in a laboratory growth medium. This work provides only qualitative information and quantification was not addressed.<br>
Reference is made to the work of Hsieh et al. (1999), wherein oligonucleotide primers were designed for sphingomyelinase gene and used for the PCR based detection of strains of B. cereus group in food samples. These primers can detect 100 cells/gram of the food sample only after an enrichment step for 8 hours indicating poor sensitivity.<br>
Reference is made to the work of Yamada et al (1999) disclosing spiked boiled rice sample with varying cell concentrations of B. cereus . The rice sample was enriched in nutrient broth for different time intervals. No amplification was observed with non-enriched food samples with a gyrase B specific primers, even when the initial cell number was 2.4 x 104 CPU of B. cereus per gram of boiled rice. Detection of low numbers of B. cereus by PCR was possible only after 15 hours enrichment in nutrient broth.<br>
Reference is made to the work of Tsen et al. (2000), wherein primers were designed for 16s ribosomal RNA (Ribo nucleic Acid) and used for PCR-based quantification of B. cereus spiked in food samples. Target<br><br>
Cells ranging from 1 to 9 CFU/g of food sample could be detected only after 8 hours enrichment in brain heart infusion broth supplemented with glucose.<br>
In German Patents DE 19915141 and DE 19915141 the sequences refer to 16s ribosomal RNA (Ribo Nucleic Acid) and gyrase B Specific primers used for the detection of Bacillus cereus.<br>
Reference is made to the work of Schrafts and Griffiths (1995), wherein a method for the isolation of target DNA from milk samples was devised. This method was elaborate comprising multitude of steps using combination of enzymes, detergents and column chromatography. The method also suffered from lack of sensitivity and could only detect 103 CFU/ml B. cereus by PCR using primers for cereolysin AB gene.<br>
Reference is made to the work of Yamada et al. (1999), wherein a protocol for the detection of B. cereus from boiled rice was described. The method included pre-enrichment step, two steps of filtration, followed by boiling of the samples prior to use in PCR. It was reported that zero hour, a moderately high count of 2.4 x 104 CFU of B. cereus per gram of boiled rice failed to yield any PCR amplified product. Low numbers of B. cereus could only be detected after 15 hours of enrichment.<br>
The drawback of all these methods have been non-specific detection of target organism i.e. B. cereus, lack of reproducibility, failure to detect all the isolates of B. cereus in a food system and lack of sensitiveness to detect low numbers of target organism. Besides, the methods are cumbersome and procedures are lengthy. The problem of formation of formation of spores by B. cereus group of organisms makes detection by PCR a difficult proposition, in most of the methods a step of enrichment in a suitable laboratory growth medium<br><br>
is included which may take 8 to 15 hours of incubation for building up of cell numbers which can result in target DNA for use in PCR detection.<br>
Object of the invention<br>
The main object of the present invention is to provide an iproved method for the detection of Bacillus cereus in foods which obviates the drawbacks detailed above.<br>
Another object of the present invention is to use a primer designed for a conserved region of a specific gene In the target organism.<br>
Still another object of the present invention is to use the designed primer in detecting isolates which belong to B. cereus group.<br>
Yet another object of the present invention is to detect B. cereus in food systems directly by PCR.<br>
Still another object of the present invention is to use a simple and effective method for the preparation of template DNA (Deoxyribo Nucleic Acid) of the organism directly from the foods.<br>
Yet another object of the present invention is to detect very low numbers of target organism in the food systems.<br>
Still another object of the present invention is to detect very low numbers of target organism in the food system.<br>
Summary of the invention<br>
Accordingly the present invention provides a novel set of nucleotide primers<br>
for phosphotidyl inositol in B.cereus said set of primers comprising<br>
PI - 1 (F) 5 ' AGTATGGGGAATGAG 3'        (Seq ID 10.1)<br>
PI - 2 (R )5' ACAATTTTCCCACGA 3'	(Seq ID NO. 2)<br><br>
The present invention also relates to method for the detection of B. cereus in foods said method<br>
comprising using primers specific for phosphotidyl inositol gene in B. cereus in a mixed microflora, said<br>
primers comprising<br>
PI - 1 (F) 5' AGTATGGGGAATGAG 3'<br>
PI - 2 (R) 5' ACAATTTTCCCACGA 3'.<br>
In one embodiment of the invention, the food matrices for detecting B. cereus in milk and cooked rice.<br>
In another embodiment of the invention, template DNA from B. cereus in cooked rice is extracted<br>
using Triton X-100, 0.5 - 2%, boiling at 96 - 100°C for 3 - 8 min and treatment with phenol : chloroform in the<br>
ratio of 22: 21 -28:27.<br>
In another embodiment of the invention, the template DNA from B. cereus in milk is extracted using<br>
diethyl ether: chloroform in the ratio of 1:1 - 1 : 3, urea 1.5 - 3.5 M and sodium dodecyl sulphate in a range of<br>
0.5 - 2%.<br>
In a funher embodiment of the invention, the PCR reaction mixture in a total volume of 25 uJ<br>
comprises of Tris HC1: 8 - 1 2 mM; KC1: 45 - 55 mM; MgCl2: 0.5 - 3.0 mM; gelatin: 0.005 - 0.02%; individual<br>
deoxynucleoside triphosphates: 150 - 300 ^M; each specific primer: 30 - 60 picomoles; Taq DNA polymerase:<br>
0.5 - 2.0 units and template DNA: 1 - 3 nl.<br>
In another embodiment of the invention, detection of B. cereus is effected by amplification profile of<br>
target gene from an initial denaturation at 90 - 98°C for 2 - 8 min, amplification cycles of 28 - 40, each cycle<br>
with a denaturation at 90 - 98°C for 40 - 70 seconds, annealing at 46 - 54°C for 40 - 80 seconds and an<br>
extension at 68 - 76°C for 45 - 75 seconds and final extension at 68 - 76°C for 4 - 12 min.<br>
In another embodiment of the invention, analysis of the PCR product is done in 1.2 - 1.8% agarose gel<br>
electrophoresis, visualization of the PCR product by staining with 0.5 (g/ml ethidium bromide and observation<br>
in a UV transilluminator.<br>
In yet another embodiment of the invention, detection of minimum number of cells of B. cereus is done<br>
in a food matrix by PCR.<br>
Detailed description of the invention<br>
The present invention relates to an improved PCR method for the detection of B. cereus in foods. The<br>
PCR method using the primers of the invention detects 1 to 106 cells of B. cereus directly in foods. Polymerase<br>
chain reaction method is used to selectively amplify phosphotidyl inositol gene in B. cereus. Milk and cooked<br>
rice samples were spiked with varying cell numbers of B. cereus ranging from 1 to 1,000,000. Protocols for<br>
extraction of template DNA from B. cereus present in food matrix were standardized using detergents and<br>
organic solvents. The PCR reaction mixture and amplification conditions were optimized for the specific<br>
amplification. Visualization of PCR products revealed that by the method followed, it is possible to detect cell<br>
numbers ranging from 1 to 1,000,000 in milk and cooked rice samples.<br>
The primers of the invention directly detect Bacillus cereus in food systems by PCR. This method can<br>
detect all the strains of B. cereus. The method is rapid and sensitive making it possible to detect even 1 cell in a<br>
food matrix overcoming any steps of enrichment.<br>
The following examples are given by way of illustrations of the present invention and therefore should<br>
not be construed to limit the scope if the present invention.<br>
EXAMPLE-1<br>
Oligonucleotide primers for phosphotidyl inositol gene of B. cereus were designed based on the gene<br>
sequence (M 30809) using the software programme Primer 3.0 This primer set amplifies a 342 base pair (bp)<br>
fragment of the gene, the sequence of which is given below. Sterilization of media and other solutions was<br>
achieved by autoclaving for 20 min at 121°C.<br>
PI - 1 (F) 5- AGTATGGGGAATGAG 3'<br>
PI - 2 (R) 5' ACAATTTTCCCACGA 3'<br>
Aliquots in 100 \\\ of a native food isolate of B. cereus was inoculated into sterile 10 ml brain heart<br>
infusion (BHI) broth and incubated for 18 h at 37°C in a shaker incubator with 140 rpm. Cells were harvested by<br>
centrifugation at 10,000 rpm for 10 min at 4°C. The cells were suspended in 10 ml sterile 0.85% saline to get a<br>
ceil concentration of 109 colony forming units per millilitre (CFU/ml). From this stock, serial dilutions in 9 ml<br>
sterile 0.85% saline were carried out to achieve cell concentrations ranging from 108 to 101 CFU/ml. The<br>
individual dilutions were used for spiking into milk samples.<br>
Twenty millilitres of pasteurized milk was taken in a sterile screw capped tube of 25 x 125 mm<br>
dimension, steamed for 30 min in a cooker without any pressure and cooled to 30°C. In individual 1.5 ml sterile<br>
microcentrifuge tube, 0.4 ml of the cooled milk sample was mixed with 0.4 ml of 0.85% saline suspension of A<br>
cereus to attain a final cell concentration ranging from of 106, 105, 104, 103, 102, 101 and 10° CFU/ml. To^ch<br>
tube was added 0.25 ml each of diethyl ether and chloroform were added to the samples and vortexed for 30<br>
seconds. The samples were centrifuged at 10,000 rpm for 15 min at 25°C. The aqueous phase was transferred to<br>
a fresh 1.5 ml sterile microcentrifuge tube and 0.5 ml of 6M urea and 0.1 ml of 10% sodium dodecyl sulphate<br>
were added. The samples were incubated at 37°C for 20 min and then centrifuged 10,000 rpm for 15 min at<br>
25°C. The supernatant was discarded and 0.1 ml of 0.2N NaOH was added to the samples and incubated at 37°C<br>
for 10 min. DNA was precipitated by adding 1.0 ml of chilled absolute ethanol and 0.1 ml of 3M sodium acetate<br>
(pH 4.8) and holding the samples at -20°C for 2 h. Samples were centrifuged at 10,000 rpm for 15 min at 4°C.<br>
The supernatant was discarded and excess salt in the DNA preparation was removed by adding 1.0 ml of chilled<br>
70% ethanol and centrifuging the samples at 10,000 rpm for 15 min at 4°C. The supernatant was discarded and<br>
the DNA pellet was air-dried and resuspended in 15 (1 of sterile ultrafiltered water.<br>
Amplification was performed in a total reaction volume of 25 \i\ which contained 2 ul of the DNA<br>
preparation from milk samples. The reaction mixture consisted of IX PCR buffer (lOmM Tris HCI, pH 9.0, 50<br>
mM KC1, 1.5 mM MgQ2, 0.01% gelatin), 200 uM of each deoxynucleoside triphosphate, 50 picomoles of each<br>
primer and 1.0 unit of Taq DNA polymerase. Template DNAs were initially denatured at 94°C for 5 min.<br>
Subsequently, a total of 35 amplification cycles were carried out in a programmable thermocycler. Each cycle<br>
consisted of denaturation for 1 min at 94°C, primer annealing for 1 min at 50°C and extension for 1 min at<br>
72°C. The last cycle was followed by a final extension at 72°C for 8 min.<br>
PCR products were analysed by agarose gel electrophoresis. Aliquots of 10 nl PCR products were<br>
mixed with 2.0 ul of loading dye and loaded onto 1.5% agarose gel and subjected to electrophoresis for 2 h at<br>
120 volts in IX TAE buffer. Gel was stained with ethidium bromide (0.5 ng/ml), de-stained with distilled water<br>
and examined on a UV transilluminator. A 100 bp ladder was used as molecular size marker. The amplification<br>
1<br>
profile in the gel was documented in a CCD-camera based Ciei Documentation System.<br>
The specific amplicons of 342 bp for phosphotidyl inositol were observed when PCR was performed<br>
with milk samples containing B. cereus cells ranging from 1 to 1,000,000.<br>
EXAMPLE - II<br>
Oligonucleotide primers for phosphotidyl inositol gene of B. cereus were designed based on the gene<br>
sequence (M 30809) using the software programme Primer 3.0 This primer set amplifies a 342 base pair (bp)<br>
fragment of the gene, the sequence of which is given below. Sterilization of media and other solutions was<br>
achieved by autoclaving for 20 min at 121°C.<br>
PI - 1 (F) 5' AGTATGGGGAATGAG 3'<br>
PI - 2 (R) 5' ACAATTTTCCCACGA 3'<br>
Aliquots in 100 ul of a native food isolate of B. cereus was inoculated into sterile 10 ml brain heart<br>
infusion (BHI) broth and incubated for 18 h at 37°C in a shaker incubator with 140 rpm. Cells were harvested by<br>
centrifugation at 10,000 rpm for 10 min at 4°C. The cells were suspended in 10 ml sterile 0.85% saline to get a<br>
cell concentration of 109 colony forming units per millilitre (CFU/ml). From this stock, serial dilutions in 9 ml<br>
sterile 0.85% saline were carried out to achieve cell concentrations ranging from 108 to 103 CFU/ml. The<br>
individual dilutions were used for spiking into cooked rice samples.<br>
Raw rice in 1000 g quantity was taken, cleaned and washed with running tap water. Cleaned rice was<br>
mixed with water in 1 : 2 proportion, taken in a stainless steel container and steam cooked in a pressure cooker<br>
for 20 min. Cooked rice in 100 g aliquots were taken in individual sterile 500 ml glass beakers and was spiked<br>
with 1.0 ml saline suspension of B. cereus to get a cell concentration of 107 CFU/g and mixed uniformly.<br>
Spiked cooked rice samples in 11 g aliquots was then added to 99 ml sterile 0.85% saline taken in a<br>
250 ml conical flask, mixed well and serial dilutions were prepared in sterile 0.85% saline to get individual cell<br>
concentrations of, 106, 10s, 104, 103, 102, 10' and 10° CFU/g. Aliquots of 1 ml of diluted samples were<br>
transferred to a 1.5 ml sterile microcentrifuge tubes. The samples were centrifuged at 10,000 rpm for 5 min at<br>
4°C. The pellet was washed thrice with 1.0 ml phosphate buffered saline of pH 7.4 and once with 1.0 ml sterile<br>
ultrafilter water by centrifugation at 10,000 rpm for 5 min at 4°C and discarding the washes. The pellet was<br>
resuspended in a mixture containing 50 (il sterile ultrafilter water and 450 |il sterile 1% Triton X-100. The<br>
samples were incubated in boiling water for 5 min. 0.5 ml phenol : chloroform (25 : 24) was added to the<br>
sample, vortexed briefly and centrifuged at 10,000 rpm for 15 min at 4°C. The aqueous phase was transferred to<br>
a fresh 1.5 ml sterile microcentrifuge tube and 0.5 ml chloroform was added to the sample. The samples were<br>
centrifuged at 10,000 rpm for 15 min at 4°C and the aqueous phase was transferred to a fresh 1.5 ml sterile<br>
microcentrifuge tube. DNA was precipitated by adding 1.0 ml chilled absolute ethanol and 0.1 ml of 3M sodium<br>
acetate (pH 4.8) and incubating the samples at -20°C for 2 h. The samples were centrifuged at 10,000 rpm for 15<br>
min at 4°C. Excess salt in the DNA pellet was removed by adding 1.0 ml chilled 70% ethanol and centrifuging<br>
the samples at 10,000 rpm for 15 min at 4°C. The supernatant was discarded. The DNA pellet was air dried and<br>
dissolved in 15 ul of sterile ultrafilter water.<br>
Amplification was performed in a total reaction volume of 25 |il containing 2 ul of the DNA<br>
preparation from milk samples. The reaction mixture consisted of IX PCR buffer (lOmM Tris HC1, pH 9.0, 50<br>
mM KC1, 1.5 mM MgCl2) 0.01% gelatin), 200 \iM of each deoxynucleoside triphosphate, 50 picomoles of each<br>
primer and 1.0 unit of Taq DNA polymerase. Template DNAs were initially denatured at 94°C for 5 min.<br>
Subsequently, a total of 35 amplification cycles were carried out in a programmable thermocycler. Each cycle<br>
consisted of denaturation for 1 min at 94°C, primer annealing for 1 min at 50°C and extension for 1 min at<br>
72°C. The last cycle was followed by a final extension at 72°C for 8 min.<br>
PCR products were analysed by agarose gel electrophoresis. Aliquots of 10 |al PCR products were<br>
mixed with 2.0 nl of loading dye and loaded onto 1.5% agarose gel and subjected to electrophoresis for 2 h at<br>
120 volts in IX TAB buffer. Gel was stained with ethidium bromide (0.5 (g/ml), destained with distilled water<br>
and examined on a UV transilluminator. A 100 bp ladder was used as molecular size marker. The amplification<br>
profile in the gel was documented in a CCD-camera based Gel Documentation System.<br>
The specific amplicons of 342 bp for phosphotidyl inositol were observed when PCR was performed<br>
with cooked rice samples containing B. cereus cells ranging from 1 to 1,000,000.<br>
The main advantages of the present invention are:<br>
1. The designed phosphotidyl inositol primers is specific for the detection of B. cereus.<br>
2. In a mixed microflora, the designed primer set specifically detects B. cereus with no cross reactivity.<br>
3. A simple and effective protocol for extraction of template DNA for B. cereus present in a varied food<br>
matrix.<br>
4. Standardized PCR conditions for the detection of 5. cereus present in milk and cooked rice.<br>
5. A rapid and sensitive PCR method which can detect even 1 cell of B. cereus in food system.<br><br><br><br><br>
We Claim:<br>
1.	A novel set of nucleotide primers for phosphotidyl inositol in B.cereus said set of<br>
primers comprising<br>
PI - 1 (F) 5 ' AGTATGGGGAATGAG 3'        (Seq ID No. 1)<br>
PI - 2 (R )5' ACAATTTTCCCACGA 3'	(Seq ID NO. 2)<br>
2.	A method for the detection of B. cereus in foods said method comprising using<br>
primers specific for phosphotidyl inositol gene in B. cereus in a mixed microflora,<br>
said primers comprising<br>
PI - 1 (F) 5' AGTATGGGGAATGAG 3' (Seq. ID No. 1) PI - 2 (R) 5' SVSSYYYY\AA/SVHS 3' (Seq. ID No. 2)<br>
3.	A method as claimed in claim 2 wherein the food matrices for detecting B. cereus are milk and cooked rice.<br>
4.	A novel set of nucleotide primer for phosphotidyl inositol in B. cereu substantially as herein described with reference to the examples.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA0NTYtZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00456-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDU2LURFTE5QLTIwMDMtQ2xhaW1zLSgyMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">456-DELNP-2003-Claims-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDU2LURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">456-DELNP-2003-Correspondence-Others-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDU2LURFTE5QLTIwMDMtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0oMjMtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">456-DELNP-2003-Description (Complete)-(23-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDU2LURFTE5QLTIwMDMtRm9ybS0yLSgyMy0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">456-DELNP-2003-Form-2-(23-10-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234033-a-process-for-vapor-phase-nitration-of-benzene-using-nitric-acid-over-molybdenum-silica-catalyst.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234035-an-improved-process-of-extracting-chili-capsicum-oleoresin.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234034</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00456/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>21/2005</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>22-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Apr-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG, NEW DELHI-110 001, INDIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BANADA PADMANABHA PADMAPRIYA</td>
											<td>DEPARTMENT OF FOOD MICROBIOLOGY, CENTRAL FOOD TECHNOLOGICAL RESEARCH INSTITUTE, MYSORE 570 013, INDIA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>AIYAGARI RAMESH</td>
											<td>DEPARTMENT OF FOOD MICROBIOLOGY, CENTRAL FOOD TECHNOLOGICAL RESEARCH INSTITUTE, MYSORE 570 013, INDIA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARUN CHANDRASHEKAR</td>
											<td>DEPARTMENT OF FOOD MICROBIOLOGY, CENTRAL FOOD TECHNOLOGICAL RESEARCH INSTITUTE, MYSORE 570 013, INDIA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KESHAVA NIRESHWALIA</td>
											<td>DEPARTMENT OF FOOD MICROBIOLOGY, CENTRAL FOOD TECHNOLOGICAL RESEARCH INSTITUTE, MYSORE 570 013, INDIA.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MANDYAM CHAKRAVARATHY VARADARAJ</td>
											<td>DEPARTMENT OF FOOD MICROBIOLOGY, CENTRAL FOOD TECHNOLOGICAL RESEARCH INSTITUTE, MYSORE 570 013, INDIA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IN01/0007</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-03-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234034-a-novel-set-of-nucleotide-primers-for-phosphotidyl-inositol-in-bacillus-cereus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:02:07 GMT -->
</html>
