<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234069-a-recombinant-nucleic-acid-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:03:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234069:A RECOMBINANT NUCLEIC ACID MOLECULE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A RECOMBINANT NUCLEIC ACID MOLECULE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention relates to a recombinant nucleic acid molecule, comprising a nucleic acid which codes for a polypeptide with chorismate mutase activity and derivatives thereof, whereby the derivatives have at least 10% of the chorismate mutase activity of the chorismate mutase, according to the identification number of the SEQ ID NO:2. The invention further relates to vectors containing nucleic acid molecules, to host cells comprising nucleic acid molecules and their use in methods for producing polypetides with chorismate mutase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention concerns a recombinant nucleic acid molecule, comprising a nucleic acid coding for a polypeptide with chorismate mutase activity and derivatives thereof, wherein the derivatives show at least 10% of the chorismate mutase activity of the chorismate mutase according to SEQ ID NO: 2. The invention further concerns vectors containing nucleic acid molecules, host cells containing nucleic acid molecules and processes for the production of polypeptides with chorismate mutase activity. The present invention further concerns the polypeptides with chorismate mutase activity and antibodies which specifically recognise these. In addition, the invention concerns processes for the production of auxotrophic yeast strains by means of the nucleic acid molecules and the use thereof in processes for the recombinant expression of heterologous genes.<br>
As single-cell, eukaryotic microorganisms, yeasts can be cultured without difficulty and have the great advantage of being readily genetically manipulable. Also, they can process and modify recombinant proteins in accordance with the patterns known from higher organisms. As far as is yet known, they contain no pathogenic substances and are therefore also suitable for the production of therapeutic proteins. Thus for example the first vaccine produced by heterologous gene expression, the hepatitis B vaccine, was heterologously expressed in the well-characterised baker's yeast Saccharomyces cerevisiae (Lepetic et al., 1996).<br>
While S. cerevisiae makes it possible to produce many different proteins (for a review, see Gellissen and Hollenberg, 1997), there are nonetheless also a few limiting properties. Thus for example the maximal content of heterologous protein is about 1 5% of the total protein content of the cell (Buckholz and Gleeson, 1991).<br>
Various other yeasts, such as for example Schizosaccharomyces pombe, KJuyveromyces lactis, Yarrowia lipolytica and also Hansenula polymorpha have been characterised and compared with S. cerevisiae in terms of their suitability as expression systems (Mullet et al., 1998). All these yeasts showed markedly stronger secretion of active protein than S. cerevisiae, the heterologous gene expression being dependent on the particular gene, but independent of the donor organism. Basically, methylotrophic yeasts are found to be attractive organisms for the production of recombinant proteins. The methylotrophic yeasts<br><br>
are subdivided into the four genera Hansenula, Pichia, Candida and Torulopsis, which all possess the ability to utifise methane, methylamine, formaldehyde or formate as carbon and energy source.<br>
Inter alia, the yeast Hansenula polymorpha from the Saccharomycetaceae family (Lodder, 1970) belongs to the relatively small group of the methylotrophic yeasts. Hansenula polymorpha does not ferment glucose under aerobic conditions, and is thus Crabtree-negative (Verduyn et al., 1992) and with a growth temperature optimum of37°C is classed among the thermotolerant yeasts; thus it is an exception among the methylotrophic genera. The natural habitat of the methylotrophic yeasts is locations rich in organic material.<br>
Hitherto, only a few genes of the yeast Hansenula polymorpha had been cloned and charact¬erised (Hansen and Hollenberg, 1996), and according to GenBank the number of cloned genes in Hansenula polymorpha amounts to &gt; 30. However, how many of these are thoroughly characterised or are suitable as marker genes for plasmid selection, is unknown to me. Hence, I would suggest that similarities of these genes with the homologous genes in Saccharomyces cerevisiae, insofar as these are present, are relatively limited (Dobson et al., 19S2). The key enzymes in methylotrophic metabolism are the enzymes methanol oxidase (MOX), dihydroxyacetone synthase (DAS) and formate dehydrogenase (FMD), whose controlled expression by very strongly regulated promoters opens up a variety of useful possibilities for heterologous gene expression. These facts make Hansenula polymorpha an industrially interesting organism (Gellissen et al., 1994).<br>
So far, only two auxotrophic strains of//, polymorpha are available, whose transformation can be selected by functional complementation of ura3- or leu2- deficiency respectively. The provision of further transformation systems, consisting of auxotrophic strains and of nucleic acids capable of complementing the auxotrophy hitherto failed because suitable genes capable of this complementation were not available. While a number of genes from the amino acid or nucleic acid biosynthesis pathway of the baker's yeast S. cerevisiae were known, it had nonetheless in the past been found that the differences between the genes of baker's yeast and the methylotrophic yeasts are sometimes considerable. Hence it cannot in general be expected that a S. cerevisiae gene will be suitable for complementation of an<br><br>
auxotrophy in a methylotrophic yeast, nor can it be expected that genes from a methylo-trophic yeast will be capable of complementing an auxotrophy in S. cerevisiae<br>
Hence a technical problem underlying the present invention is to provide a new gene from H. polymorpha, which can serve as a selectable marker for complementation in the transformation of suitable auxotrophic yeast strains. A further problem consists in the provision of vectors and host cells, which contain the gene. Further, it is to provide the polypeptide encoded by the gene and antibodies which specifically recognise the polypeptide. Finally, it is to provide suitable auxotrophic strains.<br>
This problem is solved according to the invention by a nucleic acid molecule comprising a nucleic acid coding for a polypeptide with chorismate mutase activity or the complementary strand thereof, wherein the nucleic acid is selected from<br>
(a)	a nucleic acid with the DNA sequence stated in SEQ ID NO: 1 or the RNA sequence corresponding thereto;<br>
(b)	a nucleic acid which hybridises with the complementary strand of a nucleic acid according to (a),<br>
(c)	a nucleic acid which on the basis ofthe genetic code is degenerate to the DNA sequences defined under (a) and (b),<br>
(d)	a nucleic acid which hybridises with one ofthe nucleic acids stated in (a) to (c) and the complementary strand thereof codes for a polypeptide with chorismate mutase activity,<br>
(e)	a nucleic acid which is at least 60% homologous to one ofthe nucleic acids stated in (a) to(d);<br>
(0   a variant ofthe nucleic acids stated in (a) to (e), wherein the variant had additions, deletions, insertions or inversions relative to the nucleic acids stated in (a) to (e);<br>
(g) a fragment of one ofthe nucleic acids stated in (a) to (f),<br><br>
(h) a combination of several of the nucleic acids stated in (a) to (g),<br>
wherein the polypeptide encoded by the nucleic acid or complementary strand thereof has at least 10% of the chorismate mutase activity of the chorismate mutase according to SEQ ID NO:2.<br>
In microorganisms and plants, the biosynthesis of aromatic amino acids proceeds firstly through 7 enzyme-catalysed reactions of the shikimate pathway from erythrose-4-phosphate and phosphoenol pyruvate to chorismate (figure I). Chorismate is the substrate of the first branching point. In the baker's yeast Saccharomyces cerevisiae, starting from this branching point, on the one hand anthranilate is formed via the enzyme anthranilate synthase (E.C, 4.1.3.27), and on the other prephenate via the enzyme chorismate mutase (E.C. 5.4.99.5). Finally, via further intermediate products, tyrosine and phenylalanine are formed from prephenate (Braus, 1991). In the baker's yeast Saccharomyces cerevisiae, the chorismate mutase is encoded bv the AR07 gene (Schmidheini et al, 1989), which is located on chromosome XVI. AR07 encodes a 0.95 kb mRNA and contains a 771 bp open reading frame, which codes for a protein consisting of 256 amino acids.<br>
Below, a number of terms are explained in more detail, in order to make clear how they should be understood in connection with the present application.<br>
The term "chorismate mutase", as used below in the description, includes complete chorismate mutase, chorismate mutase fragments, chorismate mutase mutants and fusion proteins thereof Polypeptides which have at least 10% of the chorismate mutase activity of the chorismate mutase according to SEQ ID NO:2 are regarded as being according to the invention.<br>
"Chorismate mutase activity" means the catalytic conversion of chorismate to prephenate as part of the phenylalanine and tyrosine biosynthesis which is catalysed by the enzyme chor¬ismate mutase [E.C. 5.4.99.5], The chorismate mutase activity of the polypeptide according to the invention can for example be measured spectrophotometrically through the acid-catalysed conversion of the product prephenate to phenylpyruvate, which absorbs at 320 nm (Schmidheini et al., 1989).<br><br>
For its growth or proliferation, a "prototrophic" microorganism needs only simple nutrients (carbon and nitrogen) and minerals, but can itself build up all needed amino acids. Hence it is capable of growing on '"minimal medium".<br>
In contrast to this, "auxotrophic" microorganisms need additional factors, for example amino acids, which they cannot themselves synthesise owing to a defect in the biosynthesis pathway for the factor concerned. In connection with the present invention, the auxotrophy is preferably a phenylalanine/tyrosine auxotrophy, which is caused through diminished or absent chorismate mutase activity, e.g. of an auxotrophic yeast strain prepared according to the invention.<br>
"Minimal medium" means a nutrient solution, which contains only the components which are necessary for the growth of a prototrophic microorganism. In connection with the present invention, the minimal medium is preferably a phenylalanine- and/or tyrosine-free medium, through which the selection of the phenylalanine/tyrosine auxotrophic form of the micro¬organism compared to the prototrophic form of the microorganism is made possible.<br>
'His" tag means a sequence of at least 6 histidine amino acids, which by appropriate cloning and fusion with an expressible sequence leads to a fusion protein with at least 6 His residues at the NH2 terminus, which can easily be purified by complexing with a Ni " column.<br>
A "heterologous gene" is understood to mean the coding region of a structural gene, which is either not expressed under the control of its own (homologous) promoter or not in the organism from which it is derived, or else is expressed neither under the control of its own promoter nor in the original organism.<br>
"Cloning" is intended to include all cloning methods known in the state of the technology, which could be used here, which are however not all described in detail, since they are among the obvious tools of the skilled person.<br>
"Recombinant expression in a suitable host cell" is to be understood to mean all expression methods known in the state of the technology in known expression systems, which could be used here, which are however not all described in detail, since they are among the obvious tools of the skilled person.<br><br>
By selection and sequencing of a genomic clone from Hansenula polymorpha which after transformation is capable of functionally complementing the phenylalanine/tyrosine auxo-trophy of the Saccharomyces cerevisicte aro7A-deletion strain, the inventors have for the first time succeeded in identifying a nucleic acid according to the invention, and producing auxotrophic mutants of methylotrophic yeasts with this nucleic acid. Thus a further urgently needed transformation system for methylotrophic yeasts is provided, which makes the targeted selection of transformed yeasts with simple media possible.<br>
The inventors have prepared a genomic bank from H. polymorpha by cloning of restriction fragments of chromosomal H. polymorpha DNA into a shuttle vector, the said shuttle vector having a high copy number in yeast and the Hansenula genes being expressed in S. cerevtsiae under the control of their endogenous promoter. This genomic bank was used for the complementation of auxotrophic Saccharomyces cerevtsiae strains, although it was not known each endogenous promoter of Hansenula polymorpha is active in S. cerevisiae.<br>
Surprisingly, it was now found that the chorismate mutase gene from Hamenula polymorpha is transcribed and translated in S. cerevisiae and is enzymatically active.<br>
For several reasons, it was not to be expected that the chorismate mutase gene in particular could be expressed in Saccharomyces cerevisiae.<br>
Thus it was not known whether the chorismate mutase gene from H. polymorpha contains introns and whether the 5. cerevisiae splicing apparatus is capable of the correct processing of the possible pre-RNA. From now on, it could be shown that the chorismate mutase gene from H. polymorpha has no introns.<br>
Further, it could not be foreseen that the translation product can be correctly folded and if necessary assembled in S. cerevisiae, in order to give a sufficiently active chorismate mutase which is capable of catalysing the reaction of chorismate to prephenate. The absence of predictability is supported by the fact that HLEU2, the LEU2 homologue from H. polymorpha, is not capable of complementing a corresponding mutation in S. cerevisiae (Agaphonov et a!., 1994).<br><br>
Further, it was not known that the biosynthesis of the aromatic amino acids tyrosine and phenylalanine in H. polymorpha involves the conversion of chorismate to prephenate with the aid of the enzyme chorismate mutase at all. In fact it is known that there are considerable differences in the biosynthesis of aromatic amino acids. Thus in some cyanobacteria prephenate is first converted to arogenate by transamination. This pathway is also used by plants (Jensen and Stenmark, 1975) and is more widespread in nature than the hydroxy-phenyl pyruvate/phenyl pyruvate pathway, which is used by Saccharomyces cerevisiae and E. coli.<br>
The nucleic acid contained in the nucleic acid molecule according to the invention can be genomic DNA, cDNA or synthetic DNA, where a synthetic DNA sequence is also understood to mean one such which contains modified internucleoside bonds. Furthermore the nucleic acid can be an RNA sequence, which e.g. can be necessary for expression by means of recombinant vector systems. The nucleic acid according to (b) is for example obtainable by use of a detectably labelled probe which corresponds to one of the sequences stated in (a) or a fragment or complementary strand thereof, for the screening of cDNA or genomic DNA libraries from microorganisms. Preferably the microorganism from which the bank is created belongs to a genus selected from: Pichia, Hansenula, Candida, Tomlopsis, Saccharomyces, Schizosaccharomyces, Khtyveromyces and Yarrowia. In particular, the microorganism belongs to a species selected from: Hansenula polymorpha, Saccharomyces cerevisiae, Schizosaccharomycespombe, Kluyveromyces lactis and Yarrowia lipolytica. In this, the identification of positive cDNA or genomic DNA clones is performed according to standard procedures. See Maniatis et al.. Molecular Cloning (1989), Cold Spring Harbor Laboratory Press.<br>
In a preferred embodiment, the hybridisation stated under (b) or (d) is performed under stringent conditions.  Stringent hybridisation conditions are e.g. incubation at 65°C overnight in 7% SDS, 1%BSA, ImMEDTA, 250 mM sodium phosphate buffer (pH 7.2) and subsequent washing at 65°C with 2 x SSC, 0.1% SDS.<br>
In a preferred embodiment, nucleic acids are provided which are at least 60% homologous to the nucleic acid sequence stated in (a). Preferred are nucleic acids at least 80% homologous<br><br>
/ to the nucleic acid sequence stated in (a). Especially preferred are nucleic acids at least 90% or 95% homologous to the nucleic acid sequence stated in (a).<br>
According to the invention, the expression "homology" means homology at the DNA level, which can be determined by known procedures, e.g. computer-aided sequence comparisons (Basic local alignment search tool, S. F. Altschul et al., J. Mol. Biol. 215 (1990), 403-410).<br>
The expression "homology", well-known to the skilled person, designates the degree of relatedness between two or more nucleic acid molecules, which is determined by the agree¬ment between the sequences. The "percentage homology" is obtained from the percentage of identical regions in two or more sequences taking account of gaps or other sequence features.<br>
The homology of mutually related nucleic acid molecules can be determined by means of known procedures. As a rule, special computer programs with the algorithms taking account of the special requirements are used.<br>
Preferred procedures for the determination of homology firstly generate the greatest agree¬ment between the sequences studied. Computer programs for determination of homology include, but are not limited to, the GCG program package, including GAP (Devereux, J., et al., Nucleic Acids Research 12 (12): 387 (1984); Genetics Computer Group University of Wisconsin, Madison, (WI)), and BLASTP, BLASTN and FASTA (Altschul S., et al., J. Mol. Biol., 215: 403-410 (1990)). The BLASTX program can be obtained from the National Centre for Biotechnology Information (NCBI) and from other sources (BLAST Handbook, Altschul S., et al., NCB NLM NTH Bethesda MD 20894, Altschul S., et al., J. Mol. Bio!., 215- 403-410 (1990)). The well-known Smith-Waterman algorithm can also be used for the determination of homologies.<br>
Preferred parameters for the nucleic acid sequence comparison include the following:<br>
Algorithm:	Needleman and Wunsch, J. Mol. Biol. 48: 443-453 (1970)<br>
Comparison matrix:    Matches	= + 10<br>
Mismatches    = 0<br>
Gap penalty	50<br>
Gap length penalty:    3<br><br>
The GAP program is also suitable for use with the above parameters. The above parameters are the default parameters for nucleic acid sequence comparisons.<br>
Further examples of algorithms, gap opening penalties, gap extension penalties and comparison matrices including those named in the program handbook, Wisconsin package, Version 9, September 1997, can be used. The choice will depend on the comparison to be performed and further on whether the comparison is performed between sequence pairs, when GAP or Best Fii are preferred, or between one sequence and a large sequence database, when FASTA or BLAST are preferred.<br>
An agreement of 60% determined with the aforesaid algorithm is described in the context of the present application as 60% homology. The same applies for higher degrees of homology.<br>
In a preferred embodiment, the nucleic acid according to the invention is a combination of several of the nucleic acids stated under (a) to (f), which can be obtained by the fusion and if necessary cloning known to the skilled person. These combinations can e.g. be of especial interest for the creation of immunogenic constructs.<br>
In a further preferred embodiment, the polypeptide encoded by the nucleic acid according to the invention has at least 50% of the chorismate mutase activity of the chorismate mutase SEQ ID NO:2. It is especially preferred that the polypeptide has at least 75% of the chorism¬ate mutase activity according to SEQ ID NO:2. Here, as already stated, the chorismate mutase activity is measured after Schmidheini et al., 1989.<br>
In a further embodiment, the nucleic acid molecule comprises a promoter suitable for its expression, the nucleic acid being under the control of the promoter. The choice of the promoter depends on the expression system used for expression, in particular on the choice of host organism. Preferred according to the invention are both constitutive promoters such as PGK- and GSPDH- promoters and also inducible promoters such as GAL4, ADH2 and the Cu-metallothionein promoter (survey in Recombinant Gene Expression Protocols in Methods in Molecular Biology Vol.63, Ed. R S Tuan, Chapter III; Schena, M. et ai. (1991) Vectors for constitutive and inducible expression in yeast, Methods in Enzymol. 194, 389-398). Further, the use of mammalian glucocorticoid response elements (GRE) for increasing transcription is also taken into account by the invention (Schena et al.. Science 241 (1988), 965-967).<br><br>
/<br>
Also preferred are the promoters of the methanol metabolism of methylotrophic yeasts, in particular Hansenutapolymorpha. The genes for the methanol metabolism enzymes in Nansetn/la polymorpha are among the most strongly expressed and regulated genes that have so far been described in yeasts (Van der Klei et al, 1991). The corresponding proteins can comprise up to 30% of the total protein content of the cell (Janowicz et al, 1985; Ledeboer et al, 1985). The expression of the methanol metabolism genes can on the one hand be induced by methanol, on the other hand however, the presence of glycerol also leads to derepression. Thus in this way the strong promoters of the methanol metabolism can also be used for heterologous gene expression under methanol-free culture conditions.<br>
Especially preferred promoters are the methanol oxidaseMTA'-promoter described in H. polymorpha, which at about 1.5 kb is unusually large and is among the strongest yeast promoters so far described. The presence of glucose leads to repression of the MOX-promoter, however the activity of this promoter can be increased more than 1000-fold by glycerol or methanol (Godecke et al, 1994). Further especially preferred is the dihydroxy-acetone synthase ZX46"-promoter (Ledeboer et al, 1985) and the Z-MD-promoter (European Patent 299108).<br>
In a farther preferred embodiment, the nucleic acid molecule also comprises a signal peptide-coding nucleic acid sequence, which ensures the export of the expressed protein, where the signal peptide-coding nucleic acid sequence is preferably directly 5' bound to the hetero¬logous gene to be expressed. For the secretion and modification of many eukaryotic proteins, it is necessary to fuse the protein sequence at the N-terminus with a signal sequence, in order to steer the polypeptides into the secretion apparatus. Possible for this are for example components from the S. occidenlalis gene GAM! (Dohmen et al, 1990) and from a hormone gene of the crab Carcimis maenas (Weydemann et al, 1989), which were successfully used for the secretion of hirudin (Weydemann et al, 1995).<br>
In a further preferred embodiment, the nucleic acid molecule also comprises at least a part of a vector, in particular control regions, wherein the vector can be selected from; bacteriophages such as X-derivatives, plasmids, adenoviruses, vaccinia viruses, baculoviruses, SV40 viruses and retroviruses, preferably MoMuLV (Moloney Murine Leukemia Virus).<br><br>
/<br>
Especially preferred are yeast transformation vectors, both integrative yeast plasmids (Yip) and also extrachromosomal plasmid vectors being possibilities. The extrachromosomal plasmid vectors subdivide into episomal yeast plasmids (YEp), replicative yeast plasmids (YRp) and yeast centromer plasmids (YCp) (see Singh, K.K. and Heinemann, J.A., Chapter 11 in Recombinant Gene Expression Protocols, v. supra). Further, artificial yeast chromo¬somes (YACs) are also possible as expression vectors according to the invention.<br>
Aiso especial ly preferred vectors are yeast replication plasmids, which contain a replication origin oh and an antibiotic resistance cassette, so that they can be propagated and selected in E. coli. Furthermore, they bear an ARS sequence for chromosomaJIy independent replication in the yeast cell, such as for example MARS I from H. polymorpha, and a metabolic yeast selection marker, such as for example URA3 or HLEU2 (Gellisen and Hollenberg, 1997).<br>
Many heterologous proteins have already been produced in H. polymorpha (Geliissen and Hollenberg, !997) and by the placement of more than one expression cassette on one trans¬formation vector, co-expression of different genes is also possible (Gilbert et al., 1994).<br>
Further, a nucleic acid molecule is preferred which in addition comprises a His-tag coding DNA sequence, which on expression of the construct leads to the formation of a fusion protein with a His-tag at the NH2 terminus, which facilitates the purification of the protein on a nickel column through chelate formation.<br>
According to the invention, host cells are provided, which contain the nucleic acid molecule and which are suitable for the expression of the heterologous gene. In the state of the technology, many prokaryotic and eukaryotic expression systems are known, where the host cells are for example selected from prokaryotic cells such as E. coli or B. siibtilis, from eukaryotic cells such as yeast cells, insect ceils and mammalian cells, e.g. CHO cells, COS cells or HeLa cells, and derivatives thereof. In the state of the technology for example, certain CHO production lines are known whose glycosylation patterns are altered compared to CHO cells.<br>
Preferably, the yeast belongs to a genus selected from: Pichia, Hametmla, Candida, Tomlopsis, Saccharomvces, Schizosaccharomyces, Kluyveromyces and Yarrowia. In particular, the microorganism belongs to a species selected from: Hansenulapolymorpha,<br><br>
Saccharomyces cerevisiae, Schizosaccharomyces pombe, Kluyveromyces lactis and Yarrowia lipolytica (see Recombinant Gene Expression Protocols in Methods in Molecular Biology, above).<br>
A further object of the present invention is a process for the production of a polypeptide with chorismate mutase activity. For this, the nucleic acid molecule according to the invention is expressed in a suitable host cell and the protein isolated from the host cell or the medium by standard procedures.<br>
Many procedures for the expression of DNA sequences are known to the skilled person, see Recombinant Gene Expression Protocols in Methods in Molecular Biology, above. The expression can be both constitutive and also inducible, inducers such as for example IPTG and Zn2"" being known to the skilled person. The polypeptide with chorismate mutase activity produced can, if a His-tag was fused to the NH2 terminus of the polypeptide, be purified by chelate formation on a nickel column.  Preferably the polypeptide with chorismate mutase activity is purified by ion exchange chromatography and/or gel filtration chromatography. The implementation of these measures is well-known to the skilled person.<br>
In a further preferred embodiment, the polypeptide with chorismate mutase activity produced according to the invention is modified. Here the modifications include the di-, oligo- and polymerisation of the monomeric starting product for example through crosslinking, eg with dicyciohexylcarbodiimide or pegylation or association (self-assembly). Other modifications include side-chain modifications, for example of e-amino-lysine residues of the polypeptide, or amino- or carboxy-terminal modifications. Other modifications include posttranslational events, e.g. the glycosylation or the partial or complete deglycosylation of the protein.<br>
In a preferred embodiment, the polypeptide obtained in recombinant expression in prokaryotes or glycosylation-deficient eukaryotes is non-glycosylated. Also taken into account according to the invention is a polypeptide with chorismate mutase activity which is glycosylated by recombinant expression in eukaryotes capable of glycosylation such as yeast cells, insect cells or mammalian ceils, such as CHO cells or HeLa cells.<br><br>
/ In a further embodiment, polypeptides with chorismate mutase activity are made available, which include an amino acid sequence, the amino acid sequence being encoded by one or several of the nucleic acid molecules according to the invention.<br>
Preferably, polypeptides with chorismate mutase activity are made available, which comprise the amino acid sequence in SEQ ID NO:2 or a fragment thereof, which has at least 10% of the chorismate mutase activity of the chorismate mutase according to SEQ ID NO:2, preferably more than 50% and especially preferably more than 75%,<br>
In a further embodiment, the invention provides polypeptides with chorismate mutase activity, obtainable by the recombinant production process or modifications thereof<br>
Further, non-glycosylated and glycosylated polypeptide with chorismate mutase activity, obtainable by expression in host cells capable or incapable of glycosylation, is provided. Depending on the intended use of the polypeptide, the glycosylation pattern of yeast, in particular methylotrophic yeast such as Hansenula pofymorpha, of COS or HeLa cells can be preferred.<br>
The invention further provides antibodies which specifically react with the polypeptide with chorismate mutase activity according to the invention and are obtainable by immunisation of an experimental animal with the polypeptide. Polyclonal antibodies can be obtained by immunisation of for example rabbits, mice or rats and subsequent extraction of antisera. Monoclonal antibodies can be obtained by standard procedures by immunisation of e.g. mice, extraction and immortalisation of the spleen cells and cloning of the hybridomas which produce antibodies specific for the polypeptide.<br>
In a further embodiment, the invention provides a process for the production of a phenyl¬alanine and tyrosine-auxotrophic yeast strain, comprising the destruction of the endogenous chorismate mutase gene of the corresponding yeast strain, wherein the mutant displays less than 10% of the chorismate mutase activity of the chorismate mutase according to SEQ ID NO:2.<br>
According to the invention, homologous recombination is possible as a procedure for the production of the mutant. The disruption of the chorismate mutase gene by homologous<br><br>
recombination is based on the replacement of the endogenous chromosomal copy of the gene by an inactivated copy   For the preparation of a disruption construct, the cloning of larger regions of the gene to be destroyed, is necessary for the efficient homologous recombination, preferably including the 5' and 3' regions. The cloned regions should preferably cover at least 2 kb.<br>
A Bglll digestion of genomic DNA from Hcmsetuila polymorpha and subsequent Southern Blot analysis with a chorismate mutase-specific probe such as e.g. SEQ ID NO:3 gave two bands of 3.2 kb and 3.0 kb respectively. Further investigation showed that the 3.2 kgBglll/ Bglll fragment includes 690 bp of SEQ ID NO.3 and contains the flanking 5' region. The 3.0 bp Bglll fragment includes 960 bp of SEQ H) NO:3 and contains the flanking 3' region. The isolation and directed fusion of the 3.2 kb fragment with the 3.0 kb fragment by means of standard procedures known to the skilled person leads to a 6.2 kb fragment, which includes the chorismate mutase gene and large 5' and 3' flanking regions.<br>
According to the invention, the nucleic acid to be destroyed, preferably a nucleic acid according to SEQ ID NO: 1 or a nucleic acid homologous thereto, especially preferably the nucleic acid according to SEQ ID NO:3 or a nucleic acid homologous thereto, in particular the nucleic acid including the 6.2 kb genomic DNA fragment, a nucleic acid homologous thereto or parts thereof, is cloned and mutated by oligonucleotide exchange, wherein the mutation can include additions, deletions, inversions or substitutions, which decrease the expression of the gene or lead to an inactive translation product. The mutation preferably takes place in at least one region or parts thereof, which are selected from the nucleic acid positions corresponding to the amino acid positions (SEQ ID NO:2) 10 to 20, 154 to 167, 192 to 196 and 240 to 247, these regions supposedly form the catalytic centre of the chorismate mutase. It is preferable to inactivate the nucleic acid coding for the chorismate mutase by cloning in a longer oligonucleotide.<br>
Preferably the endogenous chorismate mutase gene is present as a single copy gene.<br>
Preferably the oligonucleotide used for the oligonucleotide exchange comprises at least one selectable marker such as an antibiotic resistance or a metabolic marker. According to the invention, all selectable markers known in the state of the technology are included. Alternatively, the construct can also be prepared synthetically. For efficient homologous recombination, the construct should have a length of at least 2 kb.<br><br>
The cloned-in oligonucleotide is flanked both 5' and 3' by chorismate mutase-specific fragments, whose sequences correspond to the sequences stated in claim I (a) to 1 (h) or are complementary to these and can hybridise with the original chromosomal copy.<br>
The construct is first linearised for the production of a disruption mutant. Next, the proto¬trophic yeast strain intended for the production of the disruption mutant is transformed with the construct.  Since the construct comprises a selectable marker, the transformants are selected by appropriate selection pressure. The phenylalanine/tyrosine-auxotrophic transformants are identified by growth on pheny/alanine/tyrosine-free medium.<br>
In a preferred embodiment, the process comprises the steps:<br>
a)	Preparation of a construct comprising at least two fragments of a nucleic acid accordins to claim I (a) to 1 (h) suitable for homologous recombination, which flank a nucleic acid unsuitable for homologous recombination;<br>
b)	Transformation of cells of a yeast strain with an intact endogenous chorismate mutase gene with this construct<br>
and<br>
c)	Identification of the phenylalanine and tyrosine-auxotrophic transformants.<br>
Especially preferably, the construct also comprises a selection marker gene. Further the construct can comprise one or several recombination sites, which preferably 5' and 3' flank the selectable marker and make possible the excision of the selectable marker from the construct after disruption of the endogenous chorismate mutase gene has been effected.<br>
Preferably the recombination site is IoxP.  In an especially preferred embodiment, the process further includes in step b) that the cells of the yeast cells in step b) are brought into contact with nucleic acid suitable for the expression of Cre-recombinase, as a result of which the excision of the selectable marker by means of the loxP/Cre-recombinase system is made possible.<br><br>
In an especially preferred embodiment, the yeast strain is Hattsemdapolymorpha.<br>
In a further embodiment, the invention provides a phenylalanine and tyrosine-auxotrophic yeast strain, such as is obtainable e.g. by the process according to the invention.<br>
In a further embodiment, a process for the recombinant production of proteins is provided, which includes.<br>
a)	Transformation of an auxotrophic yeast strain according to the invention with a combination of a nucleic acid according to the invention suitable for expression and a heterologous gene suitable for expression under the control of a suitable promoter;<br>
b)	Culturing of the transformants under conditions suitable for the expression of the heterologous gene and of the nucleic acid molecule according to the invention and if necessary isolation of the protein which is encoded by the heterologous gene.<br>
In a preferred embodiment, the process further includes the step of the selection of the transformants on a phenylalanine and/or tyrosine deficient medium.<br>
The recombinant production process according to the invention can be performed with nucleic acid molecule according to the invention and heterologous gene spatially separate from one another, where the two can be provided in two vectors (binary vector system) and can be transformed independently of one another. Alternatively, the nucleic acid molecule according to the invention, which codes for a polypeptide with chorismate mutase activity, and the heterologous gene are present in one vector. These cointegrate vectors have the advantage that the selectable marker gene and the heterologous gene are covalently bound to one another, so that all prototrophic clones after transformation also contain the heterologous gene, as a result of which the effort of screening is reduced.<br>
Promoters and vectors preferred according to the invention for the expression of the heterologous gene correspond to the promoters and vectors stated above for the expression of the nucleic acid molecule according to the invention. The vector optionally also contains a signal peptide-coding nucleic acid sequence, which causes the secretion of the recombinant<br><br>
protein into the medium, and which is directly 5' bound to the heterologous gene. Also advantageous is the addition of a His-tag coding nucleic acid sequence to the 5' end of the heterologous gene, which after recombinant expression makes purification on a nickel chelate column possible.<br>
The invention further provides recombinant proteins which are obtainable by the process stated above.<br>
It is intended to illustrate the invention with the following figures and examples, but in no way to limit it. On the basis of the description and the examples, other embodiments are accessible to the skilled person, which are also included.<br>
Figure 1 shows the biosynthesis of the aromatic amino acids via the shikimate pathway.<br>
Figure 2 shows the ligation of the DNA fragments into the vector pRS426. The vector was linearised with the restriction enzyme BamHI and the genomic Sau3A DNA fragments from H. polymorpha were ligated into this cleavage site. This is possible, since the cleavage sites BamHI and Sau3A are compatible.<br>
Figure 3 shows the growth comparison of cells transformed with 2 complementing DNA fragments. To highlight the different growth rates, cell material of the cells complemented with a 5 kb (pME1524) or a 1.8 kb (pME1525 subclone of pME1524) DNA fragment was diluted in water and 20 JJ.1 of each dilution stage was applied onto minimal medium. The optical density of the different dilution stages is the same in each case, however since only the fundamentally different growth rates had to be illustrated, no accurate growth parameters were determined.<br>
Figure 4 shows the autoradiograph of a hybridisation of chromosomal DNA from different fungi with a 1.8 kb DNA fragment from H. pofymorpha, which is capable of complementing the phenylalanine/tyrosine auxotrophy of the S. cerevisiae aro7A selection strain. Chromo¬somal DNA from S. cerevisiae (\),A. nidtilans (2) and H. polymorpha (3) was cleaved with the restriction enzyme EcoRV and hybridised with a 32P radioactively labelled DNA probe, prepared using the genomic ].8 kb DNA fragment from H. polymorpha.<br><br>
Figure 5 shows the sequence of a genomic 1.8 kb fragment (SEQ ID NO:3) which is capable of complementing the phenylalanine/tyrosine auxotrophy of the S. cerevisiae aro7A selection strain. The 5' and 3' region are shown in small letters, the open reading frame (SEQ ID NO. 1) and the primary sequence (SEQ ID NO:2) of the derived gene product in large letters.<br>
Figure 6 shows the amplification of the flanking regions of the HAR07-ORF. The flanking regions of the HAR07 gene were amplified with the primers OLSK34/T7 and OLSK35/T3 respectively and then cleaved with Apal/Bglll and BgUI/Xbal respectively.<br>
This is possible, since the primers OLSK34 and OLSK35 contain Bglll recognition sequences.<br>
Figure 7 shows the production of various deletion constructs. The flanking regions of the HAR07-ORF were ligated into the Apal/Xbal cleaved vector pBluescriptKSII as Apal/Bglll and BgUI/Xbal fragments respectively. The vector pBluescript II KS is marketed by the firm Stratagene, USA. The vector was again opened with Bglll and the disruption cassettes cleaved with BamHI and Bglll respectively were ligated in.<br><br>
Material and Methods<br>
1. Material<br>
1.1	Chemicals<br>
Chemicals for the preparation of solutions, buffers and media were obtained from the firms<br>
Merck (Darmstadt, Germany), Boehringer Ingeiheim Bioproducts (Heidelberg, Germany),<br>
Carl Roth GmbH &amp; Co KG (Karlsruhe, Germany), Gibco BRL (Life Technologies GmbH,<br>
Karlsruhe, Germany), Fluka (Neu-Ulm, Germany) and Sigma-Aldrich Chemie GmbH<br>
(Steinheim, Germany).<br>
Restriction enzymes, DNA-modifying enzymes and polymerases were purchased from MBl<br>
Fermentas (Vilnius, Lithuania) and RNase A from Boehringer Mannheim GmbH (Mannheim,<br>
Germany). As the DNA size standard the "1 kb DNA-ladder" (Gene Ruler Plus) from MBl<br>
Fermentas was used. Agarose was obtained from Roth.<br>
For the preparation of plasmid DNA from Escherichia coli and for the extraction of DNA<br>
from agarose gels, Kits from the firm Qiagen (Hilden, Germany) were used.<br>
Synthetic oligonucleotides were purchased from Nucleic Acid Products Supply Gottingen<br>
GmbH (Gottingen, Germany) and Gibco BRL (Life Technologies GmbH, Karlsruhe,<br>
Germany).<br>
Contract sequencing was performed by the firm MWG-Biotech GmbH (Ebersberg,<br>
Germany).<br>
1.2	Strains, Plasmids and Oligonucleotides<br>
For cloning, the E. coli strain DH5a [F',cp80dIacZAM15,A(IacZY A-argF),U169,deoR,recAl, endAl,hsdR17,(rK-,mO,supE44,).-1thi-l,gyrA96,relAl] was used (Woodcock, 1989). For this work, the.//, polymorpha strain RBl l(ura3) was used (Weydemann et al., 1995).<br>
The S. cerevisiae strains used are set out in table 1.<br>
The plasmids and oligonucleotides needed for this work are listed in tables 2 and 3.<br><br><br><br>
2. Methods<br>
2.1	Culturingof the Microorganisms<br>
2.1.1	Escherichia coli<br>
The cells were cultured in Luria-Bertani medium (LB: 1% tryptone, 0.5% yeast extract, 1% NaCi) at 37°C. For strains with the ampicillin resistance marker, 50 mg/I ampicillin were added to the medium.<br>
2.1.2	Hansenula polymorpha<br>
The cells were cultured at 37°C either in "yeast extract peptone dextrose medium" (YEPD: 2% peptone, 1% yeast extract, 2% glucose) or in "yeast-nitrogen-base medium" (YNB: 0.15% yeast nitrogen base, 0.5% (NH^SO-t, 0.1% myo-inositol (200 mM), 5% glucose, supplemented with uracil for the present strain). All media were autoclaved before use and for solid media 2% agar was added.<br>
2.1.3	Saccharomyces cerevisiae<br>
The celts were cultured at 30°C either in "yeast extract peptone dextrose medium" (YEPD: 2% peptone, 1% yeast extract, 2% glucose) or in "minimal vitamins medium" (MV: 0.15% yeast nitrogen base, 0.52% ammonium sulphate, 2% glucose, 1% succinate, 0.3% KOH). Supplements such as L-amino acids or uracil were added after Sherman et al. (1986) and, like the antibiotics, sterile filtered or autoclaved and added to the sterile medium. For solid media, 2% agar was added. The growth of the yeast cells was followed by measurement of the optical density at 600 nm.<br>
2.2	Isolation of Nucleic Acids<br>
2.2.1  Qiagen Plasmid DNA Preparation from Escherichia coli<br>
Firstly the cultured bacteria were centrifuged off and the sediment formed was resuspended in 0.3 ml of buffer PI (buffer description as per manufacturer). Then 0.3 ml of buffer P2 were added and the mixture incubated for 5 mins at room temperature. In the next step,<br><br>
0.3 ml of the cooled buffer P3 were added and again incubated for 5 mins, this time on ice Next, the mixture was centrifuged for 10 mins (12,000 rpm) and the supernatant added to a Quiagen-tip 20 column previously equilibrated with buffer QBT. After the supernatant had passed through the column, this was washed four times with 1 ml portions of buffer QC. The bound DNA was eluted with 0.8 ml of buffer QF. The eluted DNA was precipitated with 0.7 volumes of isopropanol and centrifuged off (30 min, 10,000 rpm). Finally the DNA was washed with 1 ml of 70% ethanol, dried and taken up in H20.<br>
2.2.2	Isolation of Plasmid DNA from Escherichia coli (Birnboim and Doly, 1979)<br>
The E. coli cultures were cultured overnight in 5 ml of LB-Amp at 37°C on a rotary shaker. 1.5 ml of the culture were centrifuged off in an Eppendorf reaction vessel and the cells resuspended in 100 ul of solution 1 (50 mM glucose, 10 mM EDTA, 20 mM Tris-HCl, pH 8.4, 4 mg/ml lysozyme). After 7 mins incubation at RT, 200 u! of solution II (0.2 mM NaOH, 1% w/v SDS) were added, the mixture gently shaken and incubated for 5 mins on ice. 50 jil of solution III (3 MNa acetate, pH 4.8) were added, the mixture again gently shaken and incubated a further 7 mins on ice. The cell debris was centrifuged off and the super¬natant extracted with 450 ul of CH:Cl2-saturated phenol in TE buffer (10 mM Tris-HCl, 1 mM EDTA, pH 8.0) and then with 450 ul of CH2Cb. The plasmid was precipitated by addition of 1 ml of cold EtOH and incubation at -20°C. After 30 mins centrifugation at 4°C, the precipitate was washed with 200 ul of 70% EtOH, dried in vacuo and taken up in 50 ul of TE buffer (incl. 25 ug/mi RNase A). To dissolve the DNA this was heated for 5 mins at 65QC, and RNA degradation was effected using the heat-stable RNase A by incubation for 20 mins at 37°C. The plasmid DNA solution was stored at -20°C.<br>
2.2.3	Isolation of Chromosomal DNA from Hansenula polymorpha<br>
(Hofman and Winston, 1987)<br>
!0 m! of YEPD medium were inoculated and incubated for about 15 hrs at 37°C. The cells were centrifuged off, resuspended in 0,5 ml of distilled water and again centrifuged off. The supernatant was discarded and 0.2 ml oflysis buffer (2% Triton X-100, I % SDS, 100 mM NaCI, lOmMTris, pH 8.0, 1 mM EDTA), 0.2 m! of phenol/MeCI2/TE and 0.3 g of glass spheres were added   To disintegrate the cells this was shaken for 3-4 mins and then the cell<br><br>
/ debris was centrifiiged off (5 min, 13,000 rpm). The aqueous phase was placed in a new container and after addition of 1 ml of ethanol the DNA was precipitated and again centrifiiged off The sediment was resuspended in 0.4 m! of TE (10 mM Tris-HCl, 1 mM EDTA, pH 8.0) and 3 u.1 of RNase A (10 mg/ml) were added, whereupon the mixture was incubated for 5 mins at 37°C. Now 10 jil of 4M ammonium acetate and 1 ml of ethano! were added, the precipitated DNA was centrifiiged off, the supernatant discarded, dried and taken up in 50 ul TE [sic].<br>
2.3 Cloning Techniques<br>
2.3.1	Polymerase Chain Reaction ('PCR'; Saiki et al., 1985)<br>
Polymerase chain reactions were performed with the heat-stable enzyme Taq-polymerase (Fermentas) in GeneAmpTM reaction vessel (Eppendorf). Usually each time 5-50 nmol of primer oligonucleotide and 10-100 ng of DNA as matrix were used in 20-50 jil of reaction buffer in accordance with the manufacturer's instructions. As a rule, 30 cycles of the following temperature regime were performed: 30 sees denaturation at 94°C / 30 sees hybridisation at specific reaction temperature / 30 sees - 3 mins synthesis at 72°C for Taq-DNA polymerase. The PCR cycles were initiated by a 3 minute denaturation step and  , finished by a final synthesis step of 5 mins.<br>
2.3.2	DNA Restriction<br>
For analytical restriction reactions, ca. 0.5 pig of DNA were incubated with 1-2 units of restriction enzyme in a volume of 20 u.1 for 2-3 hrs at 37°C. For the restriction of preparative quantities of DNA, correspondingly larger volumes and quantities of enzyme were used. Reaction buffers were used in accordance with the manufacturer's instructions.<br>
2.3.3	Agarose Gel Electrophoresis<br>
0.1 vol. of DNA dye (25% w/v Ficoll 400, 0.25% w/v Bromphenol blue, 0.25% w/v xylenecyanol, 200 mM EDTA, pH 8.0) were added to the restriction mixture, and the DNA fragments were separated electrophoretically in a horizontal agarose gel in TAE buffer<br><br>
(40 mM Tris acetate, 20 mM NaOAc, 2 mM EDTA, pH 8.3) in the presence of 0.5 ug/m! ethidium bromide. DNA bands were detected with a UV Transilluminator (254 mm).<br>
2.3.4	Isolation of DNA Fragments from Agarose<br>
To isolate DNA fragments from the agarose gel, the columns and buffers of the QIAquick Gel Extraction Kit Protocol were used in accordance with the manufacturer's use directions. For this, the DNA fragments were firstly cut out from the gel and weighed. 3 volumes of the buffer QX1 were added, and incubated for 10 mins at 50°C. As soon as the gel fragment had completely dissolved, 1 volume of isopropanol was added and the mixture was placed on the QIAquick column and centrifuged off. To wash the DNA bound onto the column, 0.75 ml of PE buffer were added and again centrifuged. Finally, the DNA was eluted with 50 u,l of H2O.<br>
2.3.5	Ligation of DNA Fragments (Maniatis et al., 1989)<br>
Linear DNA fragments with sticky or smooth ends were ligated in a reaction mixture in a total volume of 20 jal with ligation buffer (20 mM Tris-HCl, 10 mM MgCl2, 10 mM DTT, 0.6 mM ATP, pH 7,6) and 5 units of T4-DNA ligase overnight at 15°C or for 5 hrs at room temperature. The concentration of DNA was between 1 and 10 u,g/ml, and the vector/insert DNA mole ratio between 1:5 and 1:10. Following the ligation reaction, the DNA was used for transformation without further purification.<br>
2.4 Transformation Methods<br>
2.4.1 Transformation of E. coli (Inoue et al., 1990)<br>
Cells of a 250 ml SOB culture (2% tryptone, 0.5% yeast extract, 10 mM NaCi, 2,5 mM KCI, 10 mM MgCi2, 10 mM MgSO.)) in the exponential phase were centrifuged off for 10 mins at 4°C (lOOOxg) and resuspended in 80 ml of TB buffer (10 mM PIPES, 15 mM CaCl2xH:0, 250 mM KCI, pH 6.7, 55 mM MnCI:xH20). After incubation (10 mins on ice), the ceils were again centrifuged off at 4°C and 2,500 rpm. Then the sediment was carefully resuspended in 20 ml of TB buffer and also carefully mixed thoroughly with 7% DMSO. After 10 min incubation on ice, the now competent cells were aliquotted, shock-frozen in liquid N2 and stored at -70°C. 200 nl of competent cells were added to 1-10 ^g of DNA and incubated on<br><br>
ice for 30 mins. After 30 sees heat shock at 42°C, the mixture was chilled on ice, treated with 800 ul of SOC medium (SOB, incl. 20 mM glucose) and shaken for 1 hr at 37°C. Various amounts (10-900 ul) of the cell culture were plated out on solid selective medium (agar plates with LB complete medium, supplemented with 50 ug/m! ampicillin) and finally the plates incubated overnight at 3 7°C to give transformed cells time for colony formation.<br>
2,4.2 S. cerevisiae Transformation (modified after Ito et a!., 19S3)<br>
0.5 ml of a fresh yeast culture, grown overnight, were used per transformation mix. The cells were centrifuged off at room temperature (5 mins, 3,500 rpm) and the supernatant was removed. Now the transforming DNA and 50 ug of carrier DNA were added and mixed. Finally, 0.5 ml of PEG (40% PEG3350, 0. IM LiOAc, 10 mM Tris pH 7.5, I mM EDTA, 0.1 M DTT) were added, and the transformation mixes were shaken and incubated for at least 12 hrs at room temperature. After a 20 min heat shock (42°C), the cells were carefully centrifuged, and incubated in 1 ml YEPD for 1 hr at 30°C. Finally the cells were centrifuged for 5 sees in the bench centrifuge, resuspended in the remaining liquid after discarding of the supernatant and plated out on selection media.<br>
2.5 Hybridisation Techniques<br>
2.5.1 Southern Hybridisation (Southern, 1975)<br>
For the Southern hybridisation, ca. 10 ug of chromosomal DNA were dissolved in TE buffer (10 mM Tris-HCl, 1 mM EDTA, pH 8.5) and subjected to restriction digestion for 12 hrs with a suitable restriction enzyme. The cleaved DNA was separated in a 1% agarose gel. The agarose gel was washed 3 times, for 20 mins each time, firstly in 0.25M HCl, then in 5M NaOH/l.OM NaCl and finally in IM NR,OAc. Now the separated DNA was transferred by 12-hour dry-blotting onto a nitrocellulose membrane. This was next washed for 2 mins in 2 x SSC (0. IM NaCl, 10 mM NaOAc, pH 7.0), dried in air, and finally "crosslinking" was performed for 5 mins under UV light (254 nm).<br>
The membrane was prehybridised for 30 mins at 65°C with 50 ml of Church buffer (7% SDS, 1 % BSA, I mM EDTA, 250 mM Na phosphate, pH 7.2), after which half of the hybridisation solution was discarded. Now the radioactively-labelled DNA probe was added and the hybridisation was performed overnight at 65°C. Following the hybridisation, the membrane<br><br>
was washed for 30 mins at 65°C with 2 x SSC/0.1% SDS and then exposed on an Xray film or on a phosphor imaging plate.<br>
2.5.2 Preparation of the Probe DNA<br>
For the preparation of the probe DNA, the Prime-It (Random Primer Labeling Kit) from the firm Stratagene was used in accordance with the manufacturer's instructions. The DNA required was previously amplified by PCR and then used for the probe preparation.<br>
2.6 Characterisation of the Chorismate Mutase from flansenufa polymorpka<br>
2.6.1	Crude Extract Preparation<br>
The yeast cells were cultured in YNB medium, individually supplemented for the particular yeast strains, and harvested at an OD546 of ca. 4. Crude extracts were prepared as described by Kradolfer et al. (1977), using an Amicon French Press.<br>
2.6.2	Determination of Protein Content<br>
The protein content of the protein solutions was determined by the method of Bradford (1976) withBSAas protein standard.<br>
2.6.3	Measurement of Enzyme Activity<br>
For the measurement of the chorismate mutase activity, stop tests, such as those developed by Schmidheini et al. (1989), with a few modifications were performed. All enzyme tests and extinction measurements were performed at 30°C. The 500 (jl reaction mixes contained 50 mM potassium phosphate buffer, pH 7.6, 2 mM EDTA, 20 mM DTT and 10 u.1 of the protein fraction concerned. The reaction, i.e. the conversion of chorismate to prephenate, was started by addition of chorismate at a final concentration of I mM and stopped after 10 mins with 500 p.1 of IM HCI, which also effects the conversion of prephenate into phenylpyruvate which is measurable at OD32o<br><br>
/ Neutralisation by addition of 4 ml of IM NaOH ended the reaction. The measurement of the<br>
OD3;o was performed against H2O, and similar solutions without enzyme were used as null<br>
values. The measurements were also performed in the presence of 5 uM L-tryptophan. The<br>
null values of the measurements without enzyme were subtracted from the absorption values.<br>
With the molecular extinction coefficient of phenylpyruvate at 30°C of 13095 u,l/(mol x cm),<br>
the conversion rate (umol product'(min x mg protein)) and hence the specific activity<br>
(conversion/mg protein) could be calculated.<br>
Results<br>
1.	Preparation of a Genomic Gene Bank from Hansenula polymorpha<br>
For the preparation of a genomic gene bank, chromosomal DN A from Hansenula poly¬morpha RB11 was isolated and this was subjected to partial restriction with the enzyme Sau3A. The recognition sequence of this enzyme is relatively frequently present in the genome, since it consists of only 4 bases. During the 5-hour reaction time, aliquots were taken at 15 min intervals, and the reaction in each stopped by addition of EDTA. In this way, it was ensured that the chromosomal DNA was cleaved into fragments of different size. The cleaved DNA was separated in an agarose gel, divided into different size fractions between 1 kb and 10 kb, and correspondingly extracted from the gel. The extracted fragment fractions were ligated into the vector pRS426 (Sikorski and Hieter, 1989), previously linearised with BamHl (figure 2).<br>
The plasmid library thus obtained was transformed into E. coli, during which the antibiotic resistance encoded by the bla-gene was selected for in the presence of ampicillin. Overall, about 150,000 transformants were obtained. These were washed from the plates with LB medium and stored at -20°C after addition of glycerine.<br>
2.	A Genomic 1.8 kb DNA Fragment from H. polymorpha Complements the aro7A-<br>
Phenotype in S. cerevisiae<br>
The plasmids of the genomic H. polymorpha bank were isolated from £ coli and transformed into the S. cerevisiae strain RH2185 (MATa(Aaro7.:LEU2 suc2-59 ura3-52 leu2-3 leu2-l 12 his4-519), in order to complement its Tyr/Phe auxotrophy, which is attributable to a deletion<br><br>
in the AR07 gene. The transformed yeast cells were firstly selected for uracil prototrophy and thus for the presence of the URA3 gene of the vector pRS426 (SC-Ura) and then transferred to minimal medium (YNB + Trp + His). After about 5 incubation stages at 30°C, 3 transformed yeast colonies were isolated which were capable of growing in the absence of uracil, tyrosine and phenylalanine   The plasmids were isolated and even after a retransform-ation were capable of complementing the AR07 deletion inRH2185. By restriction analysis, an approximately 5 kb additional DNA fragment was identified in the vector pRS426. For further localisation of the DNA fragment, a subcloning was performed wherein the fragments obtained were again ligated into the vector pRS426. These different plasmids were now, analogously to the plasmids of the genomic bank, examined for their ability to complement the Tyr/Phe auxotrophy in RH2185. An approximately 1.8 kb Apal/Xbal fragment was thus obtained, which was capable of doing this. This finding was confirmed by repeated ^trans¬formation. It was striking that the 1.8 kb H. polymorpha DNA fragment (in the plasmid pME! 525) in transformed yeast cells showed growth comparable with the wild type, while the 5 kb H. polymorpha fragment (in the plasmid pME1524) in transformed yeast cells resulted in markedly slower growth (figure 3).<br>
In order to confirm that the complementing 1.8 kb fragment originated from Hansenula polymorpha, this was used as the probe in a Southern hybridisation. The chromosomal DNA from H. polymorpha, A. nidulans and S. cerevisiae was cleaved with the restriction enzyme EcoRV and after hybridisation with the probe a clear signal could be seen with H. poly¬morpha, a weak signal with S. cerevisiae and no signal with A. nidulans. Furthermore, it could be shown by the Southern hybridisation that the gene occurs only once in the genome of//, polymorpha (figure 4).<br>
The 1.8 kb DNA fragment from H. polymorpha RBI I was sequenced completely and an open reading frame consisting of 843 bp could be identified (figure 5), which showed a 58% agreement with the AR07 gene from S. cerevisiae. It can thus be assumed that this is the gene of the yeast Hansenula polymorpha homologous to AR07, which is referred to below as HAR07.<br><br>
3. Preparation of Various Deletion Constructs<br>
On !he basis of the sequence of the 1.8 kb DNA fragment from H. polymorpha, the primers OLSK34 and OLSK35 were used for the preparation of various haro7A-deletion constructs. The various deletion constructs were transformed into H. polymorpha, in order to confirm the identity of the gene and to confirm that that no isoenzymes of chorismate mutase exist in H. polymorpha.<br>
By deletion of HAR07 in Hansenula polymorpha, the identity of this gene can be conclus¬ively proved and the existence of alternative genes checked. At the same time, the construct¬ion of such a deletion mutant opens up new selection possibilities. For the preparation of a haro7A deletion strain, three different disruption cassettes were constructed. For the necessary integration into the Hansenula polymorpha genome, the flanking regions of the HAR07 gene, which should effect the deletion of the HAR07 gene by homologous recombination, were used for all three constructs. The flanking region in the 3' direction was amplified using the primers OLSK34 and T7, and next cleaved with the restriction enzymes Apal and Bglll. The flanking region in the 5' direction was amplified using the primers OLSK35 and T3, and cleaved with the restriction enzymes Xbal and Bglll. This is possible as the primers OLSK34 and OLSK35 contain a Bglll cleavage site (figure 6).<br>
Both fragments were iigated into the Apal/Xbal cleaved vector pBluescript KSII and then the vector was again opened with Bglll, in order to introduce various disruption cassettes (figure 7).<br>
The hisG::URA3::hisG disruption cassette (Schneider et al., 1996) was cleaved with Bglll and Iigated into the vector described above. By means of the LTRA3 marker, integration of the cassette into the genome of//, polymorpha can be checked. Whether a homologous integration, i.e. a deletion of HAR07, has in fact taken place can be shown by selection for phenylalanine/tyrosine auxotrophy and by Southern hybridisation. The hisG components favour the conclusive excision of the URA3 marker, which can be checked by counter-selection with 5-FOA (5-fluoroorotic acid), since the URA3 gene product causes a toxic conversion of 5-FOA so that only cells without the URA3 marker are capable of surviving (Boekeetal., 1984).<br><br>
/<br>
The loxP;:kanMX::loxP disruption cassette (Guldener et al, 1996) was cieaved with BamHI and ligated into the vector described above. By means of the kanMX marker, integration of the cassette into the genome ofHanseimla polymorpha can be checked by selection for kanamycin resistance (G418). The karrr gene is flanked by the translation elongation factor (TEF) promoter and terminator (Steiner and Philippsen, 1994) of the filamentous fungus Ashbya gossypii. Whether a homologous integration has in fact taken place can be shown by selection for phenylalanine/tyrosine auxotrophy and by Southern hybridisation. In this system, the possibility exists of removing the marker again from the genome after the deletion has taken place. This occurs through the Cre-loxP recombination system of the bacteriophage PI (Guldener et al.. 1996). The plasmid pSH47 (Austin et al., 1981) contains the Cre recombinase gene with a GAL1 promoter connected upstream and in addition the components ARS, CEN" and a URA3 marker. Galactose induces the expression of the recombinase and this effects the removal of the marker from the genome by recombination of the flanking IoxP regions.<br>
The loxP::ODCl"loxP disruption cassette is analogous to the IoxP::kanMX::IoxP disruption cassette (Guldener et al., 1996), except that the kanMX gene has been replaced by the ODC1 gene (=L/RA3) from Hanseimla polymorpha (M Hiller, personal communication). In this case also, the possibility exists of removing the marker again from the genome after the deletion has taken place. The removal of the marker can in this case be checked by selection on 5-fiuoroorotic acid (5-FOA) medium, since the ODC! gene product causes a toxic conversion of 5-FOA so that only cells without the ODCl marker are capable of surviving (Boekeetal., 1984),<br>
The transformation of the hisG::URA3::hisG disruption cassette into H. polymorpha RBI 1 resulted in a mutant which had a tyrosine/phenylalanine auxotrophy and a uracil prototrophy, and thus possessed the phenotype to be expected.<br>
4. Example of Recombinant Expression of Proteins Using Auxotrophic Yeast Strains and Functional Complementation for Plasmid Selection:<br>
For the expression of a recombinant protein in H. polymorpha using a nucleic acid molecule according to the invention, suitable expression plasmids can be prepared. For example, the plasmid pFPMT121 from the firm Rhein Biotech GmbH serves as the basis for this. This<br><br>
bears the URA3 gene from S. cerevisiae for selection in H. polymorpha after transformation of uracil-auxotrophic strains such as e.g. RB11 (Weydemann et al., 1995) (odcl orura3). The FMD promoter (EP-B-0 299 108) from H. polymorpha, followed by the MOX terminator sequence (Godecke et al, 1994), serves as the expression module.<br>
After digestion with the restriction enzyme Ndel, a 5 kb DNA fragment can be isolated from pFPMTI2I. This corresponds to the plasmid without the URA3-coding sequence. After filling of the overhangs of this DNA fragment by Kienow treatment, this can be iigated with a \ .3 kb Eco72VSspl fragment from pSK69 (SEQ ID NO:3). The resulting plasmid now bears as a selectable marker gene the chommate mutase gene (HAR07) from H. polymorpha, which codes for the enzyme chorismate mutase. The selection is performed for prototrophy with respect to tyrosine and phenylalanine.<br>
Alternatively, the 1.3 kb Eco72VSspl fragment can be cloned into the plasmid pFPMT121 linearised with Smal. This procedure results in an expression plasmid which bears two marker genes, URA3 and HAR07.<br>
A recombinant DNA fragment for the expression of a heterologous protein can now be cloned into these plasmids into a suitable restriction cleavage site directly behind the FMD promoter region. Here, the coding DNA sequence is preferably fused with a DNA sequence which guarantees the secretion and processing of the expression product in H. polymorpha. As an example of such a control sequence, the MFai gene from S. cerevisiae (Arnold et al, 1998) may be mentioned.<br>
Transformed to the heterologous expression plasmids and positive transformants selected by growth on minima! medium.[sic] By alternate growth in minimal medium and complete medium, after many generations it is possible to identify from these ones in which the expression construct has been mitotically stably integrated into the genome. Then that strain in which the expression module is present in highest copy number within the host genome is preferably used as the expression strain. The expression of the heterologous protein preferably takes place after cuhuring of the strain in glucose-containing medium, by changing to medium with glycerine as the only C source. Under these conditions, there is derepression of the FMD promoter, linked with strong expression of the heterologous protein. Finally, this<br><br>
can be isolated in pure form from the purified culture supernatant by standard chromato¬graphic methods,<br>
5. The 1.8 kb Fragment from Hansenula polymorpha Codes for a Protein with Ch oris mate Mutase Activity<br>
To check whether the 1.8 kb fragment contains a gene which codes for a protein with chorismate mutase activity, ceil extracts of different plasmid-bearing yeast strains were prepared using a French Press and their chorismate mutase activities were measured. For the measurement of the specific enzyme activity, cell extracts from S. cerevisiae RH2185 (pRS426), H. polymorpha RB11, S. cerevisiae RH2185(pME 1524) and S. cerevisiae RH2185(pME1525) were prepared. It could be shown that both the 5 kb fragment (pME1524) and also the 1.8 kb fragment (pMEl 525) from H. polymorpha code for genes for a chorismate mutase activity (table 4).<br><br>
Tab.4: Specific chorismate mutase activities (U/mg protein) of various cell extracts.<br>
The table shows the mean value of 4 activity measurements in each case. The values for cell extracts from S. cerevisiae RH2185+pRS426, S. c.+pME1524, S. c + pME1525 and H. poly¬morpha RB 11 were measured. The measurements were in each case performed with and without 500 uM tryptophan.<br><br>
6. Identification of the HAR07 Gene of the Yeast Housenula polymorpha<br>
The complementing 1.8 kb DNA fragment was completely sequenced using a T3 and a T7 primer (figure 5).<br>
The enzyme chorismate mutase is encoded in the yeast Hanseimla polymorpha by a gene which consists of 843 bp and has been named HAR07. This gene was cloned by complementing the phenylalanine/tyrosine auxotrophy of a Saccharomyces cerevisiae aro7A deletion strain with a genomic gene bank from Hanseimla polymorpha. Hence the H. polymorpha promoter is functional in S. cerevisiae. No introns are present, and a 32 kDA protein consisting of 2S0 amino acids can be derived from the sequence.  The amino acid sequence of the chorismate mutase from Hanseimla polymorpha shows 58% identity to that from Saccharomyces cerevisiae and 44% to that from Aspergillus nidulans. A striking feature in the comparison of these amino acid sequences is an endpiece consisting of 23 amino acids in the sequence from H. polymorpha, which is present neither in S. cerevisiae nor in A. nidulans, nor in other eukaryotic chorismate mutase amino acid sequences.<br><br>
Literature<br>
Agaphonov, M. O , Poznyakovski, A. I., Bogdanova, A. I., and Ter-Avanesyan, M. D. (1994) Isolation and characterisation of the LEU2 gene of'Hansemtlapolymorpha. Yeast 10,509-513.<br>
Arnold, C. E., Pareish, R. N., Yang, W. and Wittrup, K. D. (1998) Leader peptide efficiency correlates with signal recognition partide dependence in Saccharomyces cerevisiae. Biotechnol. Bioeng. 59, 286-293.<br>
Austin, S., Ziese, M, and Sternberg, N. (1981) A novel role for site-specific recombination in maintenance of bacterial replicons. Cell 25, 729-736.<br>
Birnboim H. C. and Doly, J. (1979) A rapid alkaline extraction procedure for screening recombinant plasmid DNA. Nucl. Acids Res. 7, 15 i 3-1523.<br>
Boeke, J. D., Lacroute, F., and Fink, G. R. (1984) A positive selection of mutants lacking orotidine-5'-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197, 345-346.<br>
Bradford, M. M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72, 248-254.<br>
Braus, G. H. (1991) Aromatic amino acid biosynthesis in the yeast Saccharomyces cerevisiae: a model system for the regulation of an eukaryotic biosynthetic pathway. Microbiol. Rev. 55. 349-370.<br>
Buckholtz, R. G. and Gleeson, M. A. G, (1991) Yeast systems for the commercial production of heterologous proteins. Biotechnology 9, 1067-1072.<br>
Dobson, M. J., Tuite, M. F., Roberts, N. A., Kingsman, A. J., Kingsman, S. M, Perkins, R. E., Conroy, S. C, and Fothergill, L. A (1982) Conservation of high efficiency promoter sequences in Saccharomyces cerevisiae. Nucl. Acids Res. 10, 2625-2637.<br><br>
Dohmen, R. J., Strasser, A. W. M, Dahlems, U„ and Hollenberg, C. P. (1990) Cloning of the Schwanniomyces occidentals glucoamylase gene (GAMI) and its expression in Saccharo-myces cerevisiae. Gene 95, I! 1-121.<br>
Gellissen, G., Hollenberg, C P., and Janowicz, Z. A. (1994) Gene expression in methylo-trophic yeasts. In: Smith, A. (ed.) Gene expression in recombinant microorganisms. Marcel Dekker, New York, 195-239.<br>
Gellissen, G. and Hollenberg, C. P. (1997) Applications of yeasts in gene expression studies: a comparison of Saccharomyces cerevisiae, Hansenulapolymorpha and KJuyveromyces lactis - a review. Gene 190, 87-97<br>
Gilbert, S. C, Van Urk, H., Greenfield, D. M., McAvoy, M. J., Denton, K. A., Coghlan, D., Jones, G. D., and Mead, D. J. (1994) fncrease in copy number of an integrated vector during continuous culture of Hanssmtla polymorpha expressing functional human haemoglobin. Yeast 10, 1569-1580.<br>
Gbdecke, S., Eckart, M., Janowicz, Z. A., and Hollenberg, C. P. (1994) Identification of sequences responsible for transcriptional regulation of the strongly expressed methanol oxidase-encoding gene in Hansenulapolymorpha. Gene 139, 35-42.<br>
Guldener, U., Heck, S., Fiedler, T., Beinhauer, J., and Hegemann, J. H. (1996) A new efficient gene disruption cassette for repeated use in budding yeast. Nucl, Acids Res. 24, 2519-2524.<br>
Hansen, H. and Hollenberg, C. P. (1996) History of'Hansenulapolymorpha Research. In: Nonconventional Yeasts in Biotechnology. A Handbook. Klaus Wolf (ed). Springer-Verlag Berlin, Heidelberg, New York.<br>
Hoffman, C. S. and Winston, F. (1987) A ten-minute DNA preparation from yeast efficiently releases autonomous ptasmids for transformation of Escherichia coli. Gene 57, 267-272.<br>
Inoue, H., Nojima, H., and Okayama, H. (1990) High efficiency transformation of Escherichia coli with plasmids. Gene 96, 23-28.<br><br>
Ito, H., Jukuda, Y., Murata, K., and Kimura, A. (1983) Transformation of intact yeast cells treated with alkali cations. J. Bacterid. 155, 163-168.<br>
Tanowicz, Z. A., Eckart, M R,, Drewke, C, Roggenkamp, R. O., and Holienberg, C. P. (19S5) Cloning and characterisation of the DAS gene encoding the major methanol assimilatory enzyme from the methylotrophic yeast Hansemtla polymorpha. Nucl. Acids Res. 13, 3043-3062.<br>
Jensen, R. A. and Stenmark, S. L ((1975) The ancient origin of a second microbial pathway for L-tyrosine biosynthesis in pmkaryotes. J. Mol. Evol. 4, 249-259,<br>
Kradolfer, P., Zeyer, J., Miozzari, G., and Hiiiter, R. (1977) Dominant regulatory mutants in chorismate mutase of Saccharomyces cerevisiae. FEMS Microbiol. Lett. 2, 211-216.<br>
Ledeboer, A. M, Edens, L., Maat, J., Visser, C, Bos, J. W., Verrips, C, T., Janowicz, Z. A., Eckart, M., Roggenkamp, R., and Holienberg, C. P. (1985) Molecular cloning and characterisation of a gene coding for methanol oxidase in Hamemda polymorpha. Nucl. Acids Res.13, 3063-3082.<br>
Lepetic, A., Seigelchifer, M., Arduino, R C, Lazovski, J., Nacinivich, F., Sturba, E., and Stamboulian, D. (1996) Novel recombinant HB vaccine produced by a high-level expression Hansenula polymorpha yeast system. Clin. Infect. Dis. 23, 276.<br>
Lodder, J. (1970) The yeast, a taxonomic study. North- Holland Publishing, Amsterdam.<br>
Lorence, J. H. and Nester, E. W. (1967) Multiple molecular forms of chorismate mutase in Bacillus subtilis. Biochemistry 6, 1541-1552.<br>
Maniatis, T., Fritsch, E. F., and Sambrook, J. (1989) Molecular Cloning, a laboratory manual. Book 1. 2nd edn., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York.<br>
Miiller, S, Sandal, T., Kamp-Hansen, P., and Dalbge, H. (1998) Comparison of Expression Systems in the yeasts Saccharomyces cerevisiae, Hansenula polymorpha, Kluyveromyces<br><br>
/<br>
Iactis, Schizosaccharomyces pombe and Yairowia lipolytica. Cloning of two novel promoters from Yarrowia lipolytica. Yeast, 34, I267-J283.<br>
Rave, N., Crkvenjaker, R., and Bodtker, H. (1979) Identification of procollagen mRNAs transferred to diazobenzyloxymethyl paper from formaldehyde agarose gels. Nucl. Acids Res. 6, 3559-3567.<br>
Saiki, R. K., Schafr, S., Faloona, F„ Mullis, K. B„ Horn, G. T., Edich, H, E, and Arnheim, N. (1985) Enzymatic amplification of (-globin genomic structures and restriction site analysis for diagnosis of sickle cell anaemia. Science 230, 1350-1354.<br>
Schmidheini, T., Sperisen, P., Paravicini, G., Hotter, R., and Braus, G. H. (1989) A single point mutation results in a constitutively activated and feedback-resistant chorismate mutase of Saccharomyces cerevisiae. J. Bacteriol. 171, 1124-1153.<br>
Schneider, B. L., Steiner, B., Seufert, W., and Futcher, A. B. (1996) pMPY-ZAP; A reusable Polymerase Chain Reaction-directed gene disruption cassette for Saccharomyces cerevisiae. Yeast 12, 129-134.<br>
Sherman, F., Fink, G. R., and Hicks, J. B. (1986) Laboratory Course Manual for METHODS IN YEAST GENETICS. Cold Spring Harbor Laboratory, Cold Spring Harbor, New York.<br>
Sikorski, R. S. and Hieter, P. (1989) A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae. Genetics 122, 39-27.<br>
Southern, E. M. {J 975) Detection of specific sequences among DNA fragments separated by gel electrophoreses. J. Mol Biol 98, 503-517.<br>
Steiner, S. and Philippsen, P. (1994), Sequence and promoter analysis of the highly expressed TEFgene of the filamentous fungus Ashbyct gossypii. Mot. Gen. Genet. 242, 263-271.<br><br>
VanderKIei, I. J., Harder, W., and Veenhuis, M. (1991) Methanol metabolism in a peroxisome-deficient mutant of Hansenulapolymorpha: a physiological study. Arch. Microbiol. 156, 15-23<br>
Verduyn, C, Postman, E„ SchefFers, W. A., and Van Dijken, J. P. (1992) Effect of benzoic acid on metabolic fluxes in yeasts: a continuous-culture study on the regulation of respiration and alcoholic fermentation. Yeast 8, 501-517.<br>
Weidemann, W., Gromoll, J., and Keller, R. (1989) Cloning and sequence analysis of cDNA for precursor of a crustacean hyperglycaemic hormone. FEBS Lett. 257, 31-34.<br>
Weydemann, U., Keup, P., Piontek, M, Strasser, A. W. M., Schweden, J., Gellissen, G., and Janowicz, Z. A. (1995) High-level secretion of hirudin by Hansemrfa polymorpha - authetic processing of three different preprohiaidins. Appl. Microbiol. Biotechnol. 44, 377-385.<br>
Woodcock, D. M. (1989) Quantitative evaluation of Escherichia coli host strains for tolerance to cytosine methylation in plasmid and phage recombinants. Nucl. Acids Res. 17, 3469-3478.<br><br><br>
WE CLAIM:<br>
1.	A recombinant nucleic acid molecule, comprising a nucleic acid encoding a<br>
polypeptide with chorismate mutase activity or the complement thereof, wherein the<br>
nucleic acid is selected from (a) a nucleic acid comprising the DNA sequence set forth<br>
in SEQ ID NO: 1 or the RNA sequence corresponding thereto; and (b) a nucleic acid<br>
which on the basis of the genetic code is degenerate to the DNA sequences defined in<br>
(a); with the proviso that the nucleic acid molecule does not include the nucleic acid<br>
sequence of the AR07 gene from Saccharomyces cerevisiae.<br>
2.	The nucleic acid molecule according to claim 1, wherein said nucleic acid molecule is a desoxyribonucleic acid molecule.<br>
3.	The nucleic acid molecule according to claim 1, comprising a promoter suitable to control expression of the polypeptide encoded by said isolated nucleic acid, wherein the nucleic acid coding for a polypeptide with chorismate mutase activity is under the control of the promoter.<br>
4.	The nucleic acid molecule according to claim 3, wherein that the promoter is the MOX promoter or the FMD promoter from Hansenula polymorpha.<br>
5.	The nucleic acid molecule according to claim 3, comprising a heterologous nucleic acid sequence suitable to direct expression and optionally secretion of the polypeptide encoded by said isolated nucleic acid.<br>
6.	The nucleic acid molecule according to claim 3, wherein the nucleic acid molecule contains   at   least   a  part  of a  vector,     wherein  the  vector   is   selected   from:<br><br>
bacteriophages, plasmids, adenoviruses, vaccinia viruses, baculoviruses, SV40 virus<br>
and retroviruses.<br>
7.	The nucleic acid molecule according to claim 3, wherein the nucleic acid comprises<br>
a His-tag coding nucleic acid sequence and the expression of the nucleic acid<br>
molecule    leads    to    the    formation   of   a    fusion   protein   with    a   His-tag.<br>
8.	A process for the production of a polypeptide with chorismate mutase activity,<br>
wherein the nucleic acid molecule according to claim 1 is expressed in a microbial<br>
host cell suitable for the expression of a polypeptide encoded by said nucleic acid<br>
molecule and the protein is isolated if necessary.<br>
9.	A recombinant microbial host cell, comprising the nucleic acid molecule according<br>
to claim 3, wherein the host cell is a prokaryotic cell suitable for the expression of a<br>
polypeptide encoded by the nucleic acid molecule.<br>
10.	The microbial host cell according to claim 9 wherein the prokaryotic cell is<br>
selected from the group consisting of an E. coli cell and a Bacillus subtilis cell.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZm9ybS0xOS5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZm9ybS0yNi5wZGY=" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgcGN0IHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=aW4tcGN0LTIwMDEtMTQ3MC1jaGUgcGV0aXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">in-pct-2001-1470-che petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234068-gum-massager-tooth-brush.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234070-a-process-for-the-preparation-of-a-surface-nano-structured-and-or-nano-porous-coating.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234069</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>IN/PCT/2001/1470/CHE</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-May-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Oct-2001</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>RHEIN BIOTECH GESELLSCHAFT FUR NEUE BIOTECHNOLOGISCHE PROZESSE UND, PRODUKTE mbH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>Eichsfelder Strasse 11, 40595 Düsseldorf,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GELLISSEN, Gerd</td>
											<td>Ringstrasse 30, 42489 Wülfrath,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BRAUS, Gerhard</td>
											<td>Vor der Laakenbreite 13, 37075 Goettingen,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PRIES, Ralph</td>
											<td>Am Kreuze 46, 37075 Goettingen,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KRAPPMANN, Sven</td>
											<td>Ludwig-Prandtl-Strasse 52, 37077 Goettingen,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>STRASSER Alexander W</td>
											<td>Ringelsweide 16, 40225 Dusseldorf,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/90</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2000/003844</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-04-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>199 19 124.7</td>
									<td>1999-04-27</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234069-a-recombinant-nucleic-acid-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:03:17 GMT -->
</html>
