<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234114-a-method-for-detecting-the-invasion-and-or-metastatic-behaviour-of-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234114:A METHOD FOR DETECTING THE INVASION AND/OR METASTATIC BEHAVIOUR OF CELLS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR DETECTING THE INVASION AND/OR METASTATIC BEHAVIOUR OF CELLS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method for detecting the invasion and/or metastatic behaviour of cells ex vivo, comprising the steps of: a) contacting the cells with one or more human Hsp90 inhibitors; b) analyzing the migration of cells treated according to step a) by known technique; c) comparing the migration of the cells according to step a) with untreated cells by known technique; and d) optionally determining the percentage of migration in comparison with untreated cells by known technique.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Inhibitors of Extracellular Hsp90<br>
This invention was made with government support under CA81668 awarded by the National Institutes of Health. The government has certain rights in the invention.<br>
BACKGROUND OF THE INVENTION<br>
Malignant tumors shed cells, which migrate to new tissues and create secondary tumors. The process of generating secondary tumors is called metastasis and is a complex process in which tumor cells colonize sites distant from the primary tumor. Liotta ((1986) Cancer Res. 46, 1-7) has proposed a three-step hypothesis for the process of metastasis; The first step is tumor cell attachment via cell surface receptors. The anchored tumor cell next secretes hydrolytic enzymes or induces host cells to secrete enzymes, which can degrade the matrix locally. Matrix lysis most likely takes place in a highly localized region close to the tumor cell surface. The third step is tumor cell locomotion into the region of the matrix modified by proteolysis. Thus, invasion of the matrix is not merely due to passive growth pressure but requires active biochemical mechanisms. Degradation of the surrounding normal tissue is a central feature of invasiveness of malignant tumors. The process of metastasis formation depends on the invasiveness of tumor cells. It would, therefore, be useful to develop drugs, which inhibit invasiveness and therewith prevent metastasis of primary tumors.<br>
Recently, research has been focused on identifying specific proteins involved in metastasis, which can be used as a basis for better diagnostic or improved therapeutic strategies. A protein that has been identified as a molecular chaperone molecule and that is essential for the stability and function of several oncogenic proteins is heat shock protein 90 (Hsp90). This name is a generic term used to describe two isoforms termed Hsp90 a and {J. The structure and the function of the isoforms of Hsp90 is described in Csermely et al.: Pharmacol. Ther. Vol. 79, 1998, No. 2, The 90-kDa Molecular Chaperone Family: Structure, Function, and Clinical Applications. A Comprehensive Review, p. 131, p. 146. Hsp90 is one of the most abundant chaperones in the cytosol of eukaryotic cells and constitutes approximately 1-2 % of all proteins in the cell. The intracellular functions of Hsp90 include stabilization of proteins (steroid receptors) and maturation of proteins such as kinases and other signalling proteins. Hsp90 has been implicated, however, in a wide variety of functions including evolutionary stability of mutated proteins, cytoskeletal rearrangements, nuclear transport, cell proliferation and apoptosis, protein degradation, antigen presentation and lipopolysaccharide rec-<br><br>
ognition. Being very abundant in the cell Hsp90 has also been linked to many diseases, from cancer to autoimmune disease to cardiovascular disease. For example, a monoclonal antibody to the immunodominant LKVIRK epitope of Hsp90 showed therapeutic activity in a treatment against fungal infection and was used in a clinical trial b3' the firm Neutec under the trade name Mycogrip®.<br>
It has also been shown that Hsp90 is secreted from the cells in response to stress (Liao et al. (2000) J. Biol. Chem. 275, 189-96), but no known function has been associated with this secretion.<br>
While Hsp90 has well-established functions intracellular^, reports of extracellular occurrence and its function are scarce. Hsp90 has been found to be an effective antigenic peptide presenter to receptors on antigen presenting cells. It has also been found to be one of four proteins associated in lipid rafts on the extracellular surface of cells, which bind to lipopolysaccharide and initiate intracellular responses (Triantafilou et al. (2002) Trends in Immunology 23, 301-4). Hsp90 has also been found overexpressed on the surface of some tumor cells: microcitomas, melanoma, and hepatoma cell lines (Ferrarini et al. (1992) Int. J. Cancer 51, 613-19). It has been hypothesized that expression of Hsp90 on the surface of these cell lines is connected to antigen presentation, but clear evidence is not yet available.<br>
Hsp90 is also currently being assessed as an intracellular target in anti cancer drug development, due to its involvement in regulating several signalling pathways that are of importance in driving the phenotype of a tumor. Inhibition of Hsp90 function has been shown to cause selective degradation of signalling protein involved in cell proliferation, cell cycle regulation and apoptosis. Several known antibiotics (e.g. geldanamycin, radicicol, and coumermycin Al) have been shown recently to be inhibitors of Hsp90 and are described in WO 00/53169. In this document a method of inhibiting binding of a chaperone protein with its client is proposed, wherein the method proposes contacting a chaperone protein with coumarin or a coumarin derivative. However,, the teaching of WO 00/53169 is directed merely to the inhibition of intercellular Hsp90 protein.<br>
Inhibitors such as the geldanamycin analogue 17-AAG are already being tested in clinical trials but there are concerns about toxicity resulting from non-specific inhibition of the protein across all cellular compartments (Dunn (2002) J. Natl. Cancer Inst 94, 1194-5). Furthermore, the lack of understanding of the interaction of H-=vp90 imth client proteins in various cell signalling processes bears potential risks of inhibiting intracellular Hsp90 with toxic inhibitors.<br><br>
The determination of the physiological role of a protein is a prerequisite for deciding whether interference with this protein function might be a possible avenue for the treatment of disease or not. It must be kept in mind that in a physiological setting, i.e. in a naturally occurring tumor cell of a patient, Hsp90 acts together with other proteins, which can modulate and interfere with each other. It is the functional interplay between Hsp90 and interacting proteins that determines its physiological role.<br>
Hsp90 has also been reported to act as a molecular chaperone for transmembrane protein transport in the nucleus (Schlatter et al. (2002) Biochem. J. 362, 675-84), and has been implicated in drug efflux in leukemia, lung and ovarian carcinoma cells (Rappa et al (2002) Oncol. Res. 12, 113-9 and Rappa, et al (2000) Anticancer DrugDes 15, 127-34).<br>
The present invention demonstrates for the first time that the inhibition of extracellular Hsp90 leads to a reduction of the invasiveness of the tumor cells. The present invention shows a novel avenue to inhibit extracellular Hsp90 whereby side effects associated with attacking the intracellular Hsp90 can be prevented.<br>
The present invention also shows an interrelationship between Hsp90 inhibition and the secretion of matrix metalloproteases (MMPs). MMPs act in invasion by digesting surrounding extracellular matrix to allow cells to migrate through dense tissues. Our results showed that Hsp90 is critical for invasion of cancer cells by increasing secretion or activity of MMPs, when overexpressed in fibrosarcoma cells. We further showed that Hsp90-dependent invasion can be inhibited by using the molecules of the invention.<br>
The present invention relates to the use of molecules interfering with the function of extracellular Hsp90 on tumor cells for the treatment of specific cancers. Compounds, compositions and methods are provided that are useful for reducing or inhibiting the invasiveness and/or the metastatic potential of specific tumor cells. Furthermore, a method is provided that allows to determine whether a tumor cell depends on functional extracellular Hsp90 for its invasiveness and/or metastatic potential.<br><br>
SUMMARY OF THE INVENTION<br>
The present invention relates to molecules, which can specifically bind to the extracellular Hsp90. Furthermore, the molecules of the invention can be labeled with detectable groups, if desired, or can be part of a bioconjugate.<br>
The invention further relates to pharmaceutical compositions comprising an inhibitor of extracellular Hsp90.<br>
In a further embodiment the invention relates to nucleic acid molecules encoding inhibitors of extracellular Hsp90, if the inhibitors are selected from a group comprising polypeptides, antibodies or antibody fragments as well as to vectors comprising such a nucleic acid and to host cells comprising such a vector.<br>
In a further embodiment the invention relates to the use of molecules inhibiting extracellular Hsp90 function for the manufacture of a medicament for the treatment or prevention of invasion and/or metastatic potential of cancer cells.<br>
In a further embodiment the invention relates to a method of treating or preventing invasion and/or metastatic potential of cells in a patient, wherein the method comprises administering to a person in need thereof a therapeutically effective amount of the molecules, and/or the pharmaceutical composition according to the invention.<br>
In a further embodiment the invention relates to a method to determine the dependency of the invasiveness of cancer cell on the functionality of extracellular Hsp90.<br>
In a further embodiment the invention relates to a method for the identification of molecules useful for inhibiting the invasiveness of a cancer cell, particularly the identification of such ligands that bind to Hsp90.<br>
In a further embodiment the invention relates to the use of classes of molecules for reducing the activity of matrix metalloprotease (MMP).<br>
DEFINITIONS<br>
As used herein, the following definitions shall apply unless otherwise indicated.<br>
A "polypeptide" as used herein is a molecule comprising more than 10, preferably more than 20, most preferably more than 30, and less than 10000, more prefera-<br><br>
bly less than 2500, most preferably less than 1000 aminoacids. Also polypeptides containing modified or non-natural amino acids are encompassed.<br>
The terms "antibody" and "immunoglobulin", as used herein refer to any immunological binding agent, including polyclonal and monoclonal antibodies. Depending on the type of constant domain in the heavy chains, antibodies are assigned to one of five major classes: IgA, IgD, IgE, IgG, and IgM. Several of these are further divided into subclasses or isotypes, such as IgGl, IgG2, IgG3, IgG4, and the like.<br>
Antibodies may be also selected from modified immunoglobulins, for example chemically or recombinantly produced antibodies, CDR grafted antibodies or humanized antibodies, site directed mutagenized antibodies that exhibit substantial amino acid sequence identity in their CDR regions, particularly in their CDR3 region, to the corresponding antibody fragments of the invention and retain substantially the same affinity for Hsp90 binding as the corresponding antibody fragments.<br>
The CDRs (complementary determining region) of an antibody are the parts of these molecules that determine their specificity and make contact with specific ligands. The CDRs are the most variable parts of the molecule and contribute to the diversity of these molecules.<br>
Substantial amino acid sequence identity as used herein means that at least 70%, preferably at least 75%, 80%, 85%, 90%, more preferably all but 5, still more preferably all but 3 and even more preferably all but 1 of the amino acids of two aligned amino acid sequences, particularly of aligned CDRs, are identical.<br>
The term "antibody fragment" is used to refer to any fragment of an antibody-like molecule that has an antigen binding region, and this term includes antibody fragments such as scFv, dsFv, Fab', Fab, F(ab')2, Fv, single domain antibodies (DABs), diabodies, and the like. The techniques for preparing and using various antibody-based constructs and fragments are well known in the art (see Kabat et al. (1991) J. Immunol. 147, 1709-19), specifically incorporated herein by reference.<br>
"scFv" antibody fragments comprise the VH and VL domains of an antibody, wherein these domains are present in a single polypeptide chain. Generally, the scFv polypeptide further comprises a polypeptide linker between the VH and VL domains that enables the scFv to form the desired structure for antigen binding.<br>
A "Fv* fragment is the smallest antibody fragment that retains an intact antigen binding site.<br><br>
A BdsFv" is a disulfide stabilized Fv.<br>
A "Fab* fragment, is an antigen binding fragment, containing complete light chains paired with the VH and CHI domains of the heavy chain.<br>
A "Fab'*' fragment, is a reduced F(ab')2 fragment.<br>
A "F(ab')2w fragment, is a bivalent fragment comprising two Fab fragments linked by a disulfide bridge at the hinge region.<br>
A "single domain antibody (DAB)* is an antibody with only one (instead of two) protein chain derived from only one of the domains of the antibody structure. Dabs exploit the finding that, for some antibodies, half of the antibody molecule binds to its target antigen almost as well as the whole molecule (Davies et al.<br>
(1996) Protein Eng. 9: 531-537.<br>
"Diabodies* are bivalent or bispecific antibodies in which VH and VL domains are expressed on a single polypeptide chain, but using a linker that is too short to allow for pairing between the two domains on the same chain, thereby forcing the domains to pair with complementary domains of another chain and creating two antigen binding sites (Holliger et al. (1993) Proc. Natl. Acad. Sci. USA, 90, 6444-6448).<br>
The term "biogonjugate" as used herein means that an inhibitor of extracellular Hsp90 is covalently or non-covalently linked and/or coupled to or with, respectively, another protein, a solid matrix (e.g. like a bead), with itself to form mul-timers, a cytotoxic agent further enhancing the toxicity to targeted cells, a cytostatic agent, a prodrug, or an effector molecule, which is able to modify the cell expressing Hsp90 or to recruit immune cells.<br>
The terms "label" or "labeled" refers to a detectable marker or the incorporation of such, respectively, e.g., by incorporation of afluorophore-, chromophore- or radiolabeled amino acid or the attachment of a fluorophore-, chromophore- or radio-label to a ligand or the attachment of moieties that can be detected by a labeled second molecule containing a fluorescent marker or enzymatic activity that can be detected b)r an optical or a colorimetric method. An example for such a two-step detection system is the well known biotin-avidin system. Various methods of labeling polypeptides and glycoproteins are known in the art (for example, see Lobl et al. (1988) Anal. Biochem., 170, 502-511) and may be applied for different classes of molecules.<br><br>
An 'epitope" includes any protein determinant capable of specific binding to an immunoglobulin or an antibody fragment. Epitopic determinants usually consist of chemically active surface groupings of molecules such as exposed amino acids, aminosugars, or other carbohydrate side chains and usually have specific three-dimensional structural characteristics, as well as specific charge characteristics.<br>
"Treating metastatic tumors" or "treating micro-metastases", as used herein means that the metastasis of the tumor is stabilized, prevented, delayed, or inhibited by the molecule of the invention, either as a single medicament or in combination with other medicaments. Stable disease or "No Change" (NC) is a description for the course of the disease with either no change of the metastases or a reduction of less than 50% or an increase of less than 25 % over at least 4 weeks. Prevention can be, for example, that no new metastases are detected after the treatment is initiated. This can lead to a two- to three-fold median and/or a %-year survival rate of treated patients compared with untreated patients. A delay can signify a period of at least 8 weeks, 3 months, 6 months or even one year in which no new metastases are detected after the treatment is initiated. Inhibition can mean that the average size or the total number of new metastases is at least 30%, 40%, 50%, 60%, 70%, 80% or even 90% lower in a group treated with the molecule of the invention in comparison with an untreated group. Number, size and prevalence of metastases can be detected by a skilled practitioner in the field of oncology following generally accepted practice and diagnostic procedures for the detection of metastases, for example as outlined in Harrisons Principles of Internal Medicine 15th ed 2001 Mc Graw Hill.<br>
"Metastatic tumors" as used herein include both tumors at the primary site capable of metastasizing and metastasized tumors at a secondary site.<br>
"A micro-metastase* is an accumulation of tumor cells with a size smaller than 2 mm, which can usually only be detected by histological methods.<br>
"Invasiveness" as used herein is the ability of a cell to migrate through a layer of other cells or to migrate through the extracellular matrix. Invasiveness can be assessed by the Matrigel assay described in the Examples. Invasion is measured as the percentage of cells that reach the lower surface of the filter during a certain incubation period.<br>
"Metastatic potential"7 as used herein is the ability of a tumor cell to form a new tumor at a site distant from the primary tumor of which the tumor cell was derived (a metastase). "Therapeutically effective amounts* are amounts which eliminate or reduce the patient's tumor burden, or which prevent, delay or inhibit me-<br><br>
tastasis. The dosage will depend on many parameters, including the nature of the tumor, patient history, patient condition, the possible co-use of cytotoxic agents, and methods of administration. Methods of administration include injection (e.g., parenteral, subcutaneous, intravenous, intraperitoreal, etc), for which the molecule inhibiting extracellular Hsp90 function is provided in a nontoxic pharmaceu-tically acceptable carrier. In general, suitable carriers and diluents are selected so as not to significantly impair biological activity of the binding agent (e.g., binding specificity, affinity or stability), such as water, saline, Ringer's solution, dextrose solution, 5% human serum albumin, fixed oils, ethyloleate, or liposomes.). Acceptable carriers can include biocompatible, inert or bio-absorbable salts, buffering agents, oligo- or polysaccharides, polymers, viscoelastic compound such as hyaluronic acid, viscosity-improving agents, preservatives, and the like. In addition, the pharmaceutical composition or formulation may also include other carriers, adjuvants, or nontoxic, non-therapeutic, non-immunogenic stabilizers and the like. Typical dosages may range from about 0.01 to about 20 mg/kg, or more particularly from about 1 to about 10 mg/kg.<br>
Therapeutic methods employing molecules inhibiting Hsp90 function may be combined with chemotherapy, surgery, and radiation therapy, depending on type of the tumor, patient condition, other health issues, and a variety of factors. The molecules inhibiting Hsp90 function may also be used as the single effective medicament of a therapeutic composition.<br>
A "molecule inhibiting extracellular Hsp90 ", is a molecule resulting in inhibition of the biological activity of extracellular Hsp90, This inhibition of the biological activity of extracellular Hsp90 can be assessed by measuring one or more indicators of extracellular Hsp90's biological activity, such as Hsp90 dependent invasiveness. These indicators of Hsp90's biological activity can be assessed by one or more of several in vitro or in vivo assays (see Examples). Preferably, the ability of a molecule to inhibit Hsp90 activity is assessed by inhibition of Hsp90-induced invasiveness of invasive human cells, particularly the cells used in the Examples.<br>
A "molecule inhibiting extracellular Hsp90 function" of the invention is not a molecule which is a general inhibitor of protein function, like a protease, like a denaturing agent, e.g.urea or guanidinium hydrochloride, like heavy metal atoms or like small molecules (e.g. aldehydes or isocyanates) reacting covalently and non-specifically with biomolecules (lipids, proteins, sugars). Inhibition is understood as a decrease in function, when compared to a negative control with the same experimental conditions, but without the molecule of the invention.<br><br>
Additionally, in the case of a polypeptide of the invention, particularly an antibody or antibody fragment of the invention, the polypeptide of the invention is considered to inhibit the biological function of extracellular Hsp90 if it reduces the invasiveness of cancer cells in an experiment as described in the Examples by more than 30%, preferably more than 60%, when said antibody fragment is present at a concentration of InM to 50|iM, preferably around 20JIM.<br>
Additionally, with regard to the inhibitors of extracellular Hsp90, such inhibitor is encompasses by the present invention if it reduces the invasiveness of invasive cancer cells in an experiment as described in the Examples by more than 30%, preferably by more than 60%, when present at a concentration of lOnM to IOOJIM, preferably at around lfiM.<br>
"Extracellular Hsp90w as used herein is Hsp90 that is loosely associated with the cell membrane and outside of the cell and is not to be understood to encompass isolated Hsp90. This can include Hsp90 insertion in the membrane by post-translational modifications or physical integration so as to expose it to the extracellular face of the cell or secretion of Hsp90 to the extracellular space.<br>
The term "inhibitor" of extracellular Hsp90 is any entity, which binds extracellular Hsp90 and decreases its function, especially with regard to the cells properties such as invasiveness and/or metastatic potential. Such an entity can be an ionic or a non-ionic organic or non-organic small molecule (i.e. preferably having a molecular weigh of less than 1500Da), naturally and non-naturally occurring mac-romolecules, a polypeptide which can be modified, in particular with naturally and non-naturally occurring sugar moieties or otherwise chemically modified, an antibody, in particular a monoclonal antibody, antibody fragments etc.<br>
The term "a" is not to be construed to mean "one" and can also mean "one and/or more than one", if not otherwise stated.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
In order that the invention described herein may be more fully understood, the following detailed description is provided.<br>
The present invention relates to inhibitors, which can specifically bind extracellular Hsp90 and more particularly can inhibit Hsp90 function related to the invasiveness and/or metastatic potential of cancer cells. The inhibitor of extracellular<br><br>
Hsp90 is preferably either essentially unable to enter into the cell or is modified in such a manner that the modification essentially prohibits its cellular uptake. The modification of the inhibitors of the present invention can be achieved by conjugating the inhibitors to naturally occurring macromolecules such as polypeptides, sugars or to an artificially made, biologically compatible polymers.<br>
Preferably, the inhibitor of the present invention is a bioconjugate with, respectively, another protein, a solid matrix (e.g. like a bead), with itself to form mul-timers, a cj^totoxic agent further enhancing the toxicity to targeted cells, a cytostatic agent, a prodrug, or an effector molecule, which is able to modify the cell expressing Hsp90 or to recruit immune cells.<br>
The inhibitors of the present invention for inhibiting extracellular Hsp90 can be, thus, already known inhibitors of (intracellular Hsp90) such as geldanamycin and its analogues such as 17 AAG, radicicol, coumarin antibiotics such as novobiocin or purine based Hsp90 inhibitors such as PU3, when there are used to inhibit extracellular Hsp90. However, it is preferred that the known inhibitors are modified such that the cellular uptake is essentially prohibited so that any side effects associated with the possible toxicity of the inhibitors of (intracellular) Hsp90 are eliminated.<br>
A list of cytotoxic agents include, but is not limited to, daunorubicin, taxol, adri-amycin, methotrexate, 5 FU, vinblastin, actinomycin D, etoposide, cisplatin, doxorubicin, genistein, andribosome inhibitors (e.g., trichosantin), or various bac-teriaJtoxins (e.g., Pseudomonas exotoxin; Staphylococcus aureus protein A).<br>
The inhibitor of the present invention is preferably selected from a group comprising polypeptides, antibodies, antibody fragments, coumarin and/or purine based Hsp90 inhibitors and their analogs.<br>
Bioconjugates comprising the polypeptides of the invention, particularly the antibody fragment or antibody of the invention, together with a cytotoxic moieties are made using a variety of bifunctional protein coupling agents. Some examples of such reagents are N-succinimidyl 3-(2-p3Tidyldithio)-propionate (SPDP), bifunctional derivatives of imidoesters such a dimethyl adipimidate HC1, active esters such as disuccinimidyl suberate, aldehydes such as glutaraldehyde, bisazido compounds such as his (R-azidobenzoyl) hexanediamine, bisdiazonium derivatives such as bis-(R-diasoniumbensoyl)ethylenediamine, diisocyanates such as tolylene 2,6-diisocyanate, and bis-activated fluorine compounds such as l,5-difluoro-2,4-dinitrobenzene. Methods useful for the production of bioconjugates are described in detail in March's Advanced Organic Chemistry: Reactions, Mechanisms and<br><br>
Structure, 5th Edition, Wiley-Interscience; or Bioconjugate Techniques, Ed. Greg Hermanson, Academic Press.<br>
In another preferred embodiment the polypeptide of the invention is an antibody, in one preferred embodiment an antibody derived from a scFv antibody fragment, in another preferred embodiment a polyclonal or a monoclonal antibody, particularly a human monoclonal antibody.<br>
Anti-human Hsp90 antibodies binding to extracellular Hsp90 may be selected from modified immunoglobulins, for example chemically or recombinantly produced antibodies or humanized antibodies, site directed mutagenized antibodies, that exhibit substantial amino acid sequence identity in their CDR regions, particularly in their CDR3 region, to the corresponding antibody fragments of the invention and retain substantially the same affinity for Hsp90 binding as the corresponding antibody fragments.<br>
In another preferred embodiment the anti-human Hsp90 antibody is selected from the group consisting of IgA, IgD, IgE, IgG, and IgM, in particular IgG and IgM, more particularly IgGl, IgG2a, IgG2b, IgG3, IgG4.<br>
In another preferred embodiment of the invention the inhibitor of the invention is particularly an antibody fragment.<br>
Preferably the inhibitor of the invention is an antibody fragment, in particular a scFv, dsFv? Fab', Fab, F(ab')2, Fv, single domain antibody or diabody, more particularly a scFv, dsFv, Fv, single domain antibody or diabody, still more particularly a scFv, single domain antibody or diabody and even more preferably a scFv.<br>
In another embodiment, the antibody fragment of the invention specifically recognizes one or more epitopes of Hsp90, or epitopes of conserved variants of Hsp90, or peptide fragments of the Hsp90.<br>
As an example, one way to increase the biological activity of the molecules of the invention is to use the molecules (or ligands) in combination with CALI (Chromo-phore-assisted Laser/Light Inactivation).<br>
The principle of CALI (Chromophore-assisted Laser/Light Inactivation) is based on the local initiation of a photochemical reaction that leads to the generation of short-lived reactive species, which in turn selectively modify the target molecule and cause its functional inactivation. Highly specific but non-inhibitory ligands (e.g. antibodies, antibody fragments, small molecules) are labeled with a suitable<br><br>
fluorophore (e.g. fluorescein isothiocyante). After complex formation between the target molecule (e.g. proteins) and the ligand, the complex is irradiated with laser light or white light to excite the chromophore or fluorophore, respectively. The excitation triggers a photochemical reaction that initiates the generation of shortlived reactive species (e.g. hydroxyl radicals or highly reactive oxygen species). These reactive species modify the protein within a small radius around their site of generation. The distance that a reactive species can travel is very short due to its short lifetime. Therefore, the modifications of amino acid residues within the protein occur in close proximity to the binding site of the ligand. The damaging effect is restricted to a radius of 15-40A, which is well below the average distance of two proteins within a cell, which is at about 80A, (assuming an average cytoso-lie protein concentration of 300mg/ml and an average protein size of 50kDa) ensuring a high spatial resolution of the process. This principle is shown in Figure 12. In cases where the binding site of the ligand is close or within an important functional domain of the protein, these induced modifications lead to permanent inactivation of the protein. The functional inactivation of the protein is measured in an appropriate readout assay and evaluated in the context of disease relevant physiological functions like cell invasion, cell adhesion, cell signaling or apop-tosis.<br>
Inactivation of proteins with CALI was shown to be very specific to the respective protein. Linden et al showed that P-galactosidase could be efficiently inactivated with a malachite-green labeled anti-P-galactosidase antibody even in the presence of alkaline phosphatase in the same solution, p-galactosidase was inactivated by 95 % after 10 min of laser irradiation whereas alkaline was not effected at all (Linden et al. (1992) Biophys. J. 61, 956-962). Jay also demonstrated that a dye-labeled antibody bound to a single epitope of a protein was sufficient to inactivate acetylcholinesterase (Jay (1988) Proc. Natl. Acad. Sci. USA, 85, 5454-58).<br>
Henning et al. described that CALI was successfully used against a diverse array of proteins (Henning et al. Drug Discovery 62-71). These proteins include membrane proteins (eg. a-, p-, e-chains of T cell receptor, pi integrins, ephrin A5 or FAS receptor), signal transduction molecules (eg. Calicineurin, cyclophillin A or PKC), cytoskeletal proteins (eg. Actin, ezrin or kinesin) or transcription factors. Henning et al. further described that CALI can be used for identification of novel proteins as drug targets and at the same time the elucidation of their function in the biological context of interest (Henning et al. (2002) Current Drug Discovery May, 17-19).<br><br>
Several application examples of CALI show that this technique is able to convert specific but non-inhibitory ligands into blocking reagents. Therefore, these ligands can be used to modulate the action of inhibitory ligands, CALI can also be used to further enhance the inhibitory effect of a ligand that has already an inhibitory effect by itself. The experimental part shows Examples where CALI is integrated in an invasion or an adhesion assay.<br>
The chemical modification of the molecule can be the addition of chromophores or fluorophores. A chromophore is that part of a molecule that possesses high optical activity due to mobile electrons that interact with light. Chromophores are called fluorophores, if they absorb light in the visible range of the spectrum. Some examples of chromophores are, e.g., fluorescein derivatives, rhodamine derivatives, coumarin derivatives, porphyrin derivatives, phthalocyanine derivatives, naph-thalocyanines derivatives eosin derivatives, triphenylmethane derivatives or ac-ridine derivatives. A list of chromophores and useful derivatives for chemically modifying biomolecules is disclosed in The Sigma Aldrich Handbooks of Stains and Dyes, Ed., F.J. Green (1990) ISBN No. 0-941633-22-5.<br>
In another aspect of the present invention the inhibitor is labeled with a detectable label. Particularly, examples for detectable labels are radioisotopes, chromophores, fluorophores or enzymes.<br>
The expression of a cancer-associated Hsp90 antigen can be detected by using a bioconjugate or a polypeptide of the invention, particularly an antibody or an antibody fragment of the invention. A sample is taken from the subject, e.g., a biopsy specimen taken from tissue suspected of having a tumor. Generally, the sample is treated before an assay is performed. Assays, which can be employed include ELISA, RIA, EIA, Western Blot analysis, immunohistological staining and the like. Depending upon the assay used, the antigens or the antibodies can be labeled by an enzyme, a fluorophore or a radioisotope. (See, e.g., Coligan et al. (1994) Current Protocols in Immunology, John Wiley &amp; Sons Inc., New York, New York; and Frye et al. (1987) Oncogene 4, 1153-1157.)<br>
In another embodiment the modified polypeptide or the bioconjugate of the invention, binds to human, extracellular Hsp90 and reduces the invasiveness and/or metastatic potential of cancer cells. One method to measure a degree of the invasiveness of human cancer cells is given in the Examples.<br>
The present invention also relates to a pharmaceutical composition comprising effective amounts of an inhibitor of extracellular Hsp90 particularly for reducing<br><br>
the invasiveness and/or metastatic potential of cancer cells. Preferably, the inhibitor is either essentially unable to enter into a.cell or is modified in such away that the cellular uptake of the inhibitor is essentially prohibited. The pharmaceutical composition of the present invention can be used for the preparation of a medicament for the prevention and/or the treatment of the proliferative disorders such as cancer or metastasis.<br>
The present invention also relates to a pharmaceutical composition comprising effective amounts of at least one inhibitor, particularly one antibody or an antibody fragment and the use of the antibody or an antibody fragment for the preparation of a medicament for the prevention and/or treatment of proliferative disorders, cancer or metastasis.<br>
In another embodiment, the present invention encompasses a diagnostic kit. Such a kit comprises at least one bioconjugate and/or at least one inhibitor or a labeled version of these, and consists additionally of the reagents and materials necessary to carry out a standard competition or sandwich assay. Said diagnostic kit may be used for the determination of the invasive potential of biological samples, in particular of certain cancer cell types. A kit will further typically comprise a container.<br>
In another aspect, the present invention encompasses a method for screening or testing the invasion and/or the metastatic behavior of the cells ex vivo, comprising the steps of:<br>
a)	contacting the cell with one or more Hsp90 inhibitors under the conditions which essentially prohibit a cellular uptake of said Hsp90 inhibitor;<br>
b)	analyzing the migration of cells treated according to step a);<br>
c)	comparing the migration of the cells treated according to step a) with untreated the cells and optionally<br>
d)	determining the percentage of migration of the cells treated according to step a) with untreated cells.<br>
In a preferred embodiment of this aspect of the present invention the screening is performed such that the cells are contacted with a geHike matrix under the conditions suitable for the growth of the cells and step b) comprises, thus, analyzing<br>
the migration of the cells through the matrix.<br>
The term "gel-like matrix" as used herein is understood to be a semi-solid substance with a water content of at least 90%, which allows cultivation of cancer<br><br>
cells in contact with the matrix and allows migration of invasive cancer cells through a slab of said "gel-like matrix" of 0,1mm to 1mm, preferably 0,3mm thickness, but not migration of non-invasive cells. Examples for such a "gel-like matrix" are substances resembling the extracellular matrix in protein and carbohydrate composition, particularly the commercially available "Matrigel". Particularly the "gel-like matrix" comprises one of the proteins selected from the group consisting of the proteins collagen type IV, fibronectin and laminin. More particularly the gel-like matrix comprises the proteins collagen type IV, fibronectin and laminin. More preferable the gel-like matrix comprises the proteins collagen type IV, laminin. entactin, nidogen and heparan sulfate proteoglycans or collagen type IV, fibronectin, laminin, nidogen, entactin, and vitronectin.<br>
The present invention further provides a method of identifying an inhibitor binding specifically to Hsp90, by screening a library of ALDUs, wherein these inhibitors are capable of inhibiting Hsp90 related invasiveness, adhesiveness and/or metastatic potential of cancer cells. Said method comprises the steps of:<br>
a)	contacting a library of ampUfiable ligand-displaying units (ALDU) with a cancer<br>
cell, e.g. a sarcoma cell;<br>
b)	separating said cancer cell and the ALDUs bound thereto from ALDUs not<br>
bound to said cancer cell;<br>
c)	amplifying the ALDUs bound to said cancer cell;<br>
d)	identifying an ALDU or a ligand derived from an ALDU after step c), which affects said biological activity of Hsp90 by a functional screening assay;<br>
e)	identifying an ALDU or a ligand derived from an ALDU after step d) capable of binding to Hsp90<br>
f)	determining the chemical identity of the ligand.<br>
The preferred - but not the only possible - order of the steps is a, b, c, d, e and<br>
optionally f.<br>
As used herein "amplifiable ligand-displaying units" (ALDUs) are molecules or collections of molecules with dual function: they can bind to a target cell via a ligand; and they carry physically associated with the ligand an information about its identity, which allows individual units to be identified, and amplified. Examples of ALDUs are oligonucleotides, particularly RNAs, useful for the process of selection-amplification (SELEX), phages of phage displa libraries, viruses of viral display libraries, ribosomes of ribosome-display techniques, but also individualized beads carrying an identifiable molecule or ligand, particularly chemical entities, e.g. small molecules bound to beads, which are recognizable by inert chemi-<br><br>
cal tags. Preferred are such ALDUs, which comprise a nucleic acid as the identifiable component. Such ALDUs can be either amplified in vitro, e.g., by nucleic-acid amplification methods like RT-PCR, PCR, LCR and the like, or they can be amplified in vivo, for example an individual phage can infect a bacterium and yield, after several cycles of infection, millions of virtually identical progeny. A libraiy of ALDUs is a collection of similar, but in general non-identical ALDUs, for example phages of a phage library that display different scFvs in the context of an otherwise identical phage surface.<br>
A ligand "derived from an ALDUW as mentioned herein is a ligand, which had been displayed by such an ALDU, which had been selected by binding to the surface of a target cell. As an example, if the ALDU is a phage of a phage library displaying scFvs the "ligand derived from an ALDU" in this case is a polypeptide, here a scFv. As another example, in the case of ribosomal display the ligand "derived from an ALDU" is the polypeptide, which was presented by the complex comprising the ri-bosome, the RNA and the polypeptide.<br>
The method advantageously combines a screening step based on binding to the surface of a cancer cell with a screening step based on a functional assay. Therefore a library of ALDUs is brought in contact with a cancer cell in such a way that those ALDUs of the library with specificity for a surface antigen of the cancer cell can bind to structures associated with the surface of e.g. the cancer cell. For example phages of a phage library displaying antibodies or antibody fragments can be allowed to bind to cultured cancer cells or, as another example, RNA of an RNA-oligonucleotides libraiy can be allowed to bind to such cancer cells.<br>
The identity of ALDUs representing the antibody or the antibody fragment obtained with step d) can be determined by, e.g., sequencing the DNA encoding the antibody or the antibody fragment, or, in the case of a commercial libraiy with gridded or numbered phages, by determining the grid position or the number of the phage. The grid position or the number then can reveal the identity of the antibody or the antibody fragment represented by the ALDU.<br>
A "ligand" as used herein is a molecule displayable by an amplifiable ligand-displaying unit (ALDU). A ligand can be any entity that binds extracellular Hsp90 and inhibits its function. A ligand is that part of an ALDU through which the ALDU can bind to a target.<br>
A ligand "binding specifically to extracellular Hsp90" as mentioned herein can be a ligand, which binds to Hsp90 under the buffer conditions given in the Examples. The dissociation constant between the ligand and Hsp90 can be measured, e.g. by use of the so-called BIACORE System (see, for example, Fivash et al. Curr Opin<br><br>
Biotechnol. (1998) 9, 97-101) and "binding specifically" can then be understood to mean that the dissociation constant between the ligand and Hsp90 is lower than 10 jiM, preferably lower than 1 jiM, more preferably lower than 500, 400, 300, 200, 100, 50, 20 nM, most preferably from 0,1 nM to 20 nM if measured under standard conditions, for example at 20°C, ambient pressure and in a suitable buffer, e.g. 20 mM Tris, 100 mM NaCl,.0,l mM EDTA at an overall pH of 7.0.<br>
As used herein "contacting" of two components means allowing the two components to physically associate with one another and bind to one another.<br>
"Separating" as used herein means the physical separation of two or more components., for example a cell with an ALDU bound thereto can be separated from a free ALDU by centrifugation, wherein the cell with the ALDU bound thereto is pelleted and the free ALDU is still in the supernatant.<br>
"Amplifying" as used herein is any process that increases the number of ALDUs or ligands derived from ALDUs by at least a factor of two, preferably by a factor of 10, 100, 1.000, 10.000, 100.000, 1.000.000, 10.000.000 or even a billion. For example a single phage can be amplified by infecting a bacterium and the infected bacterium then produces several mostly identical copies of the infecting phage, or a DNA-molecule can be amplified by the process of PCR to yield 104 or more mostly identical daughter DNA-molecules.<br>
"Identifying" as used herein means locating or recognizing an individual component, for example an ALDU, with special properties and isolating said individual<br>
component.<br>
"Determining the chemical identity of a ligand" as used herein means determining the structural composition of a ligand. For example, if the ALDU library is a phage library displaying scFvs, then "determining the chemical identity of a ligand" would mean to determine the sequence of the scFv polypeptide, for example by sequencing the region encoding the scFv of the phage from which it was derived.<br>
"A screening assay" as used herein is aimed at detecting an individual, e.g. an ALDU, with special properties among other similar individuals with slightly different properties. In order to qualify for a screening assay at least 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50 or even 100 individuals must be tested in a certain functional assa3" to find an individual/individuals with the desired function detected among them.<br>
In the separating step (b.) according to the present invention, the cancer cell and the ALDUs bound thereto aire separated from unbound ALDUs, in order to select for those ALDUs with specificity for the cancer cell. The separation can be achieved, e.g., by removing the solution, in which the contacting step has been<br><br>
performed from cultured cancer cells grown adhesively in a culture flask and together with the solution the unbound ALDUs; or, as another example, cancer cells with RNAs of a SELEX library bound thereto can be pelleted by centrifugation, while the unbound RNAs remain in the supernatant as part of the solution, in which the contacting step had been performed. Alternatively, separation can be achieved by centrifugation, density centrifugation, filtration, FACS sorting, im-munoprecipitation or immobilization onto a solid support. For example an ALDU bound to a cancer cell can be separated from unbound ALDUs making use of the fact that the complex of an ALDU bound to a target has different biochemical properties compared to an unbound ALDU. Those changed biochemical properties can be size, specific binding, charge density, density or the like.<br>
Filtration can separate the bound ALDUs from the unbound ALDUs, e.g. if the size of the filter is such that ALDUs bound to the cancer cell are retained by the pores of the filter, while the unbound ALDUs can pass through the pores of the filter. Filtration by size is therefore useful if the target is large, like the cancer cell, and the ALDU is small, e.g. a phage, a virus, a nucleic acid or a ribosome displaying a ligand.<br>
Density centrifugation separates biomolecules according to their density. Density centrifugation can be applied if the density of an ALDU bound to a target is different from that of an unbound ALDU. For example, the density centrifugation can be applied if an ALDU is dense, e.g. a ribosome displaying a ligand, and the target is not as dense, like a sarcoma cell or vehicles derived from the membrane of a cancer cell.<br>
FACS can separate the complex of ALDU bound to a cancer cell from unbound ALDUs based on fluorescence of the target. FACS can be performed by labeling the cancer cell with a fluorescent dye and then passing the labeled cancer cell in suspending medium through a narrow dropping nozzle so that each cancer cell is in a small, separate droplet. A laser-based detector system is often used to excite the fluorescent dye and droplets with e.g. fluorescence-positive cancer cells are given an electric charge. Charged and uncharged droplets can then be separated as they pass charge plates and can be collected in two different tubes. In this way small droplets containing ALDUs that are bound to a cancer cell are separated from bulk solution containing unbound ALDUs.<br>
Immobilization of the cancer cell onto solid support can be performed in order to separate ALDUs bound to the immobilized cancer cell from unbound ALDUs. Immobilization of the cancer cell onto solid support is often performed by using a surface activated solid support or specific binding of the solid support surface with the cancer cell using e.g. biomolecules with specific binding properties to-<br><br>
wards the cancer cell. In order to separate ALDUs that are bound to the immobilized cancer cell from unbound ALDUs, the immobilized cancer cell is separated from the bulk solution containing unbound ALDUs.<br>
Separation of ALDUs bound to the immobilized cancer cells from the bulk solution containing unbound ALDUs can be performed by removing the bulk solution containing the unbound ALDUs from the immobilized cancer cells. For example, if the cancer cell is immobilized onto e.g. the well of a plastic micro titer plate via cova-lent interactions then the bulk solution containing the unbound ALDUs can be separated by removing the solution and washing the wells with washing solution.<br>
In the amplification step (c.) according to the method of the present invention the ALDUs bound to the target are amplified, in order to obtain high enough concentrations of the ALDUs so that they are useful for being applied in a functional screen. This amplifying step makes use of the ALDUs1 intrinsic property to be am-plifiable. For example phages displaying, e.g., an antibody or antibody fragment, which have been separated from the non-bound phages in step b), but are still bound to the cancer cell, can be amplified by first recovering the bound phages, for example by eluting the phages from the cancer cells and then using the recovered phage eluate to infect bacteria, e.g. Escherichia colt The amplification of the phages in Escherichia coli then leads to a significant increase in total phage number, but in such a way, that the concentration of those individual phages, which were capable of binding to the cancer cells, is dramatically increased. According to a preferred embodiment the step of amplification is realized by either eluting the ALDU bound to a cancer cell, or by lysing a caner cell under conditions where the ALDU remains amplifiable, and subsequent amplification. ALDUs bound to a cancer cells can be eluted from the cancer cells first, e.g. by use of solutions with low pH and/ or high salt, as demonstrated in the Examples, and subsequently amplified, e.g. such that the eluted phages are used to infect Escherichia coli in order to produce large numbers of progeny phages. However, the elution step from the target, while it may be convenient, is not necessary, as, e.g., in the case of phage libraries even the phages still bound to a cancer cell, are capable of infecting Escherichia coli and thus capable of producing large numbers of identical progeny, and thus are amplifiable even when still bound to the cancer cell. Accordingly, the phages need not necessarily be removed from the target cell, as a phage still bound to a target cell can infect added bacteria and thus be amplified.<br>
As another example, the mRNA of a ribosomal-display Ubrary, which is associated with the surface of a cancer cell by its attachment to a ribosome displaying the polypeptide corresponding to the mRNA can be directly used for RT-PCR, without the need of removing the intact ALDU from the target before amplifying the target.<br><br>
Thus, a step in which the ALDU bound to a target is recovered from the target before amplifying the ALDU is an optional step.<br>
Amplifying the ALDU can also consist in determining the chemical identity of the ligand of such ALDUs, which are bound to the cancer cell and then amplifying the ligands. That is to say, e.g., that in the case of a phage library, displaying an antibody or an antibody fragment, DNA encoding those ligands can be cloned from phages, which are bound to the cancer cell, e.g., by PCR, and the ligand, e.g., the antibody or antibody fragment, can then be produced recombinantly to yield sufficient amounts of the ligand to be useful for a functional screening assay. In such a way, not the ALDU itself is amplified, but the ligand displayed by the ALDU. This, however, is within the scope of the invention, as the aim of the amplification step is the generation of a ligand concentration high enough to be useful for a functional screening assay and this can be achieved by either amplifying the ligand or the ALDU displaying it.<br>
In the identifying step (d.) according to the method of the present invention, an ALDU or a ligand derived from an ALDU, which had bound to a cancer cell is identified based on its effects on a biological function of Hsp90 to be tested in a functional screening assay. For such a functional screening assay, the ALDUs or ligands derived from them are advantageously individualized, such that a signal or a pattern from the functional screening assay is relatable to an individual ALDU or an individual type of ligand. This can be done, e.g. by filling separate wells of a multi-well plate with individual ligands each and then performing an assay for a desired biological function of Hsp90 in those individual wells.<br>
In another example phages, which had bound to a cancer cell, and had been separated from unbound phages, can be individualized by infecting bacteria and plating the infected bacteria in, e.g. soft agar, so that individual plaques form, which represent clones of individual phages. Those individual plaques can then be tested for their effects in a biological screening assay for Hsp90 function. In such an assay, an individual plaque with an effect in the functional screening assay for Hsp90 function is identifiable. The ligand displayed by the phages of the plaque can then be identified, e.g. by sequencing the phage DNA of an isolated plaque, which represent a clone of a single phage, initially selected by its ability to bind to the cancer cell. Thus, in those cases where an ALDU itself is used in the functional screening assay, the identity of the ligand displayed by the ALDU can be determined, if desired.<br><br>
In a preferred embodiment the ligand is capable to inhibit a biological function of Hsp90. For example, the biological function of Hsp90 can be invasion, or adhesion.<br>
In another embodiment the ALDU or the ligand derived from an ALDU and used for the functional screening assay as in step d) of the method according the invention identifying an unmodified ligand according to the invention.<br>
In still another embodiment of the invention the ALDUs or the ligands derived from an ALDU which are used for the functional screening assay according to step d) of the method of identifying a ligand according to the invention are chemically modified to increase their biological activity or endow the ALDU or the ligand with biological activity. As an example, one way to increase the biological activity of an ALDU or a ligand or to endow the ALDU or the ligand with biological activity is to use the ALDU or the ligand in combination with Chromophore-Assisted Laser In-activation (CALI) as described earlier. CALI can also be used to further enhance the inhibitory effect of a ligand that has already a neutralizing and inhibitory effect by itself.<br>
This makes also clear that the nature of the ligand displayed on the ALDU is not critical to identify suitable molecules, it is more the ability of the ALDU to be identifiable and amplifLable, which has influence on a successful screening for Hsp90 inhibiting molecules according to the invention.<br>
However, as explained before, amplification of the ALDU can be achieved by increasing the number of an individual ALDU, but also by increasing the number of a ligand displayed by an individual ALDU. Since this can in principle also be achieved by an individual bead displaying a small chemical in combinatorial chemical libraries (see Ohlmeyer et al. (1993) Proc. Natl. Acad. Sci USA 90, 10922-26; Liu et al. J. Am. Chem. Soc. (2002) 124, 7678-80; Liang et al Science (1996) 274, 1520-2) such bead-format libraries are also libraries of ALDUs. Small molecules can be synthesized on microsphere beads, which can be indexed with inert chemical tags. During each step of the synthetic scheme a tag molecule that encodes the step number and the chemical reagent used in that step is attached to the bead. This array of tags can then be used as a binary code to record the reaction history of each bead. The identified compound can then easily be synthesized in a bigger scale.<br>
An ALDU used according to the invention is a biological ALDU whereby the information about its identity is stored as a nucleic acid, e.g. RNA or DNA. In a preferred embodiment the ALDU is selected from the group consisting of viruses useful for viral display, phages useful for phage display, bacteria useful for bacterial<br><br>
display, ribosomes useful for ribosome display, yeasts useful for yeast display and oligonucleotides useful for selection and amplification.<br>
Phages useful for phage display can be all phages that can be obtained from culture and that axe amenable to genetic engineering techniques and are capable of displaying a foreign ligand on their surface. Phage display has been disclosed in Smith et al. (1985) Science, 228, 1315-17, phage display of an antibody has been<br>
disclosed in WO 91/17271.<br>
Bacteria useful for bacterial display are bacteria, which can be kept in culture, which are amenable to genetic engineering techniques and which are able to display a Hgand on their surface. Examples for bacteria useful for bacterial display are disclosed in Dougherty et al., (1999) Protein Eng. 12,613-21; and Westerlund-Wickstrom B. (2000) Int. J. Meth. Microbiol. 290, 223-30.<br>
Ribosomes useful for ribosomal display have been disclosed in Shaffizel et al., (1999) J. Immunol. Methods 231, 119-35; and Willson et al., (2001)- Proc. Natl. Acad. Sci. USA 98, 3750-5.<br>
Oligonucleotides useful for selection/amplification (Selex) have been disclosed in Tuerk and Gold (1990) Science 249, 505-10, and Tuerk et al., (1992) Proc. Natl. Acad. Sci. USA 89, 6988-92.<br>
However, also lower eukaryotes which can be cultured and are amenable for genetic engineering techniques and, thus, able to display a foreign ligand on their surface are useful as ALDUs, e.g. genetically modified yeast, as disclosed in Boder and Wittrup (2000) Methods Enzymol. 328, 430-44.<br>
The method of the invention, the method of identifying a ligand of the invention comprises as the identification step d) a spatial separation of the different ALDUs of a pool of ALDUs and then the screening of the spatially separated ALDUs for their effect on a biological function of Hsp90 in such a way that the result obtained can be assigned to an individual ALDU. Spacial separation of the different ALDUs of a pool of ALDUs can be done, e.g., by filling separate wells of a multi-well plate with individual ligands each and then use those multi-well plates in a functional screening assay. If, e.g., in one well the biological function of Hsp90 is inhibited, then a Hgand inhibiting said biological function of Hsp90 is identified, as the experimentator knows the identity of each ligand he put into each individual well.<br>
In another preferred embodiment the method of identifying a ligand according to the invention can comprise the step of determining the identity of the ligand, wherein this determination step is achieved by sequencing the DNA of an identified ALDU, taking a PCR fingerprint of the nucleic acid of an identified ALDU or<br><br>
also, in the case of a ligand derived from an ALDU, by mass spectrometry of such a ligand. PCR fingerprinting of, e.g., phages is disclosed in Marks et al. (J. Mol. Biol. (1991) 222, 581-97). Mass spectrometrical analysis of a ligand, e.g. a polypeptide, is disclosed in Shevchenko et al. ((1996) Anal. Chem. 68, 850-58) or Spengler et al ((1992) Rapid. Commun. Mass. Spectrom. 6, 105-8.)<br>
The method of identifying a ligand according to the invention may further comprise at least one additional step of screening ALDUs, e.g. wherein the screening is based on biochemical properties of the ALDUs. Such an additional screening step may comprise a method selected from the group consisting of fLow cytometry, ELISA, immunoprecipitation, binding assays, immunohistochemical experiments, affinity studies, immunoblots and protein arrays. Biochemical properties can be determined by the size, the shape, the density, the charge density, the hydropho-bicity, or the binding specificity of an ALDU or the ligand derived from an ALDU. The biochemical properties of the ALDU or the ligand derived from an ALDU form the basis of the applicability of the above-mentioned methods for screening the ALDUs. Flow cytometry is usually performed by labeling cells with a fluorescent dye and then passing those labeled cells in suspending medium through a narrow dropping nozzle so that each cell is in a small, separate droplet. A laser-based detector system is often used to excite the fluorescent dye and droplets with e.g. fluorescence-positive cells are registered.<br>
In another preferred embodiment of the invention, the method of identifying a ligand of the invention can further comprise a substractive selection step. A sub-stractive selection step is a step, which removes ALDUs with an undesired property. For example a substractive selection step can be affected by removing the ALDUs capable of binding to a control cell, if the property of binding to a control cell is undesired. By way of example, if one wants to select for ALDUs specific for cancer cells one could first select those ALDUs which bind to cancer cells, elute the bound ALDUs, e.g. phages, and then remove those ALDUs, which are capable of binding to non-cancer-cells, by for example contacting the pool of eluted ALDUs with non-cancer-cells and removing those ALDUs bound to the non-cancer-cells. The ALDUs remaining in the supernatant are then ALDUs specific for cancer cells and can then be used in functional screening assays according to the method of identifying a ligand of the invention.<br>
In step e) the pool of ALDUs, e.g. phages, is enriched in ALDUs binding to Hsp90. Those ALDUs binding to Hsp90 can finally be identified in step f) by numerous methods known in the art.<br>
In a preferred embodiment of the invention, the above method comprises instead of steps e) and f) the farther steps of:<br><br>
e)	contacting ALDUs, e.g. isolated phages with recombinant Hsp90;<br>
f)	washing said Hsp90 with a buffered detergent and/or high salt solution; and<br>
g)	eluting ALDUs, e.g. phages bound to Hsp90; and<br>
h)  determining the identity of the ligand, e.g. an antibody or antibody fragment represented by said eluted ALDU, e.g. a phage.<br>
The "detergent" used can be a detergent solution, preferably buffered, and can be Tween in a concentration of 0.001- 0.5%, particularly 0.01-0.1 %. "High salt" as used herein means a high salt solution, preferably buffered, and has an ionic strength of 10 mM-lM, particularly 20-500 mM, more particularly 50-350 mM, even more preferably 80-250 mM. Typical useful anions are, for example, chloride, citrate, phosphate, hydrogen phosphate or borate. Typical useful cations are, for example, sodium, potassium, lithium, calcium or magnesium.<br>
buffered solution in the above paragraph typically has a pH of 7-8. For example, DMEM or PBS, particularly with 1-20%, more particularly 5-15%, even more preferably about 10% FCS, can be used as buffers.<br>
Isolation of cells with phages bound to them is effected by gentle centrifugation at g values from 200 to 300 for 3 to 20 minutes, particularly 5 to 10 minutes. Elu-tion of bound phages, both to cells and to immobilized Hsp90, is effected by a wash with 2-100 mM, particularly 4-50 mM, more particularly 5-20 mM, even more preferably around 10 mM Glycine at a pH of from 0 to 2.5, particularly from 1 to 2.5, more particularly from 1.5 to 2.5.<br>
Furthermore all the inhibitors of extracellular Hsp90 can be used according to the present invention to modulate and especially to reduce the activity and/or reduce the secretion of matrix metalloprotease (MMP).<br>
The following examples, including experiments conducted and results achieved, are provided for illustrative purposes only and are not to be construed as limiting upon the present invention.<br>
DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the invasion of stained HT-1080 cells through a Matrigel coated 8gm filter (left picture). The right picture shows invasion of stained HS-27 cells as a control. Fluorescence was quantified after six hours incubation at 37°C. Data presented are the mean of n ■ 3 wells +/- SD.<br>
Figure 2 shows the inhibitory effect of mAb 1.5.1 on the invasion of HT-1080 cells. Invasion was determined in a chemotaxis assay with a Matrigel<br><br>
coated cell migration chamber after light irradiation (with CALI). The invasion of HT-1080 cells in the absence of any inhibitory molecule was used as a control (left bar). mAb 1.5.1 inhibited the invasion of HT-1080 cells by about 35% (p-value 
Figure S shows the inhibitory effect of scFvl on the invasion of HT-1080 cells. Invasion was determined in a chemotaxis assay with a Matrigel coated cell migration chamber after light irradiation (with CALI). The invasion of HT-1080 cells in the absence of any inhibitory molecule was used as a control (left bars). scFvl inhibited the invasion of HT-1080 cells by about 46% (p-value 
Figure 4 shows a correlation of the inhibition of invasion with the antibody concentration. Antibodies specific for Hsp90, Hsp90a, Hsp90p, (all Stress-gen), alpha-actinin-4 (Martin R. Pollak, Children's Hospital, Boston, MA) and mAb 1.5.1 were FITC-labelled and tested in the invasion assay after light irradiation at 3 concentrations (lOjig/mL, 20ng/mL, and 40ng/mL). Positive control (51-integrin (Chemicon, 20fig/mL) and negative control (0, no antibody) are also shown. mAb 1.5.1, Hsp90, and Hsp90ct antibodies all demonstrated concentration-dependent inhibition of invasion, while Hsp90(5 and alpha-actinin-4 did not. Each bar is normalized to dark control and represents the mean +s.e.m of at least 2 experiments in triplicate.<br>
Figure 5 shows the result of immunoprecipitation experiments. mAb 1.5.1 or an anti-Hsp90 antibody (Stressgen #SPA-830) was incubated with lysates of HT-1080 cells. The immuno-complexes were separated by SDS-PAGE and immunoblotted with mAbl.5.1 or the anti-Hsp90 antibody. The same specific band is recognized by both antibodies, indicating that both antibodies bind the same protein.<br>
Figure 6 shows co-immunocytochemistry of Hsp90 and mAb 1,5.1. The first panel shows localization of Hsp9Oa/0 (Affinity Bioreagents #PA3012 8B PA3-013) near the leading edge of HT-1080 cells. The second panel shows the same localization near the leading edge using mAb 1.5.1. Complete protein co-localization can be seen when the images are overlapped (right picture). This demonstrates that mAb 1.5.1 and Hsp90 recognize the same protein. Images were collected (Zeiss Axiovert 10 using a Neofluar 40-x/0.75 numerical aperture lens) using excitation filter sets at 546 and 480 and emission filter sets at 590 and 535 for rho-damine (red, Hsp9Oa/0) and FITC (green, mAb 1.5.1). Scale bars in images represent lOjim.<br><br>
Figure 6.1 Surface immunocytochemistry on HT-1080 cells using antibodies against pl-integrin (c), and Hsp90a (e) show distinct surface staining. Negative controls [mouse secondary alone (a) and rabbit secondary alone (b)], a-actinin-4 (d), and Hsp90p (f) show no detectible surface staining. Images were visualised using an Olympus BH-3, and collected using a SPOT digital camera (Digital Instruments).<br>
Figure 6.2Surface immunocytochemistry on MD-MDA231 adenocarcinoma cells using antibodies against (31-integrin (c), and Hsp90a (e) show distinct surface staining. Negative controls [mouse secondary alone (a) and rabbit secondary alone (b)], a-actinin-4 (d), and Hsp90(J (f) show no detectible surface staining. Images were visualized using an Olympus BH-3, and collected using a SPOT digital camera (Digital Instruments).<br>
Figure 6.3Immunoblotting of HT-1080 conditioned media with Hsp90a and Hsp90p specific antibodies shows the extracellular location of Hsp90a, but not Hsp90p. Hsp90a is present in serum-free conditioned media from HT-1080 cells, but Hsp90(J is absent.<br>
Figure 7 shows a surface protein biotinylation experiment that was performed with live HT-1080 cells as previously described. Both, the surface bioti-nylated proteins and the unbiotinylated intracellular proteins were separated by SDS-PAGE. Hsp90 and alpha-actinin-4 antibodies were used for immunoblot analysis. Hsp90 shows both surface (S) and intracellular (IC) localization, but alpha-actinin is found only in the intracellular pool.<br>
Figure 8 shows a result of FACS analysis. The binding activity of scFvl to HT-1080 cells (bold line) and to HS-27 cells (thin line) as control is illustrated.<br>
Figure 9 shows peptide matches from mass spectrometry analysis. Trypsinized peptide fragments from the mAb 1.5.1 immunoprecipitation were run on a Survevor HPLC and LCO Deca Ion Trap mass spectrometer (Ther-moFinnigan) with a 75nM nanospray C18 column (New Objectives). The obtained PMF (peptide mass fragments) were used to search all entries for the species Homo sapiens in the NCBI and Swiss-Prot databases. The matched peptides cover 24% (172/732 residues) of Hsp90a and 33% (241/724 residues) of Hsp90p.<br>
Figure 10 shows the concentration-dependent effect of novobiocin on invasion of HT-1080 cells. HT-1080 cells were pre-treated for 1 hour with novobiocin or nalidixic acid and an invasion assay was performed. Novobiocin<br><br>
shows a dose-dependent inhibition of invasion (p  0.05mM by AN OVA). Nalidixic acid has no effect on invasion. Viability of cells in this assay was not affected (data not shown). Each data point is normalized to the no drug control and represents the mean +s.e.m. of 2 experiments in triplicate.<br>
Figure 11 shows that MMP secretion is decreased after addition of novobiocin. 40,000 HT-1080 cells were treated for 6 hours with novobiocin or nalidixic acid (as a control) at the given concentrations. MMP activity was decreased by &gt;75 % at the highest concentration of novobiocin tested and no reduction of enzyme activity was seen with nalidixic acid. Two bands (A and B) correspond to unidentified MMPs. (Image inverted for clarity.)<br>
Figure 12 shows the principle of Chromophore Assisted Laser/Light Inactivation (CALI)<br>
Figure 13 shows the vector map of the scFv display vector pXPIO and corresponding sequence<br>
Figure 14 shows the vector map of the scFv expression vector pXP14 and corresponding sequence<br>
Figure 15 shows the sequences for the construction primers for a mouse library.<br>
Figure 16 shows the amino acid sequences (SEQ ID NO.: 1) (CDR3 region is underlined) and nucleotide sequence (SEQ ID NO,: 2) of scFvl<br>
Figure 17 shows that an anti-Hsp90a antibody co-immunoprecipitates MMP2 from HT-1080 conditioned media. Immunoprecipitated proteins were im-munoblotted with anti-MMP2 or Hsp90a. Inhibition of Hsp90 inhibits MMP2 secretion.<br>
Figure 18 shows the MMP2 content in conditioned media after geldanamycin (GA) treatment. HT-1080 cells were treated with 20|iM geldanamycin (GA) in serum-free media and assayed by zymography. MMP2 (72kDa gelati-nase) activity was decreased by ~35% after GA treatment. Total protein content was visualized by silver staining to ensure equal loading.<br>
Figure 19 shows a correlation between the MMP2 content and the hsp90a content. HT-1080 cells were treated with geldanamycin-conjugated agarose beads (GA-beads) or control beads (beads) or left untreated (0). Conditioned media was immunoblotted for MMP2 or Hsp90a. Active MMP2 levels were decreased by 80%, and pro-MMP2 levels were decreased by 15% after GA-bead treatment.<br><br>
Figure 20 shows the decrease of invasiveness of tumor cells after treatment with different amounts of geldanamycin-conjugated agarose beads (GA-beads). HT-1080 cells were treated either with no beads, 5% or 10% (v/v) of GA-beads or control beads and assayed for invasiveness. GA-bead treated cells show a 45% reduction, whereas control bead treated cells only show a 15% reduction in invasiveness (p
EXAMPLES<br>
Example 1: Generation of monoclonal antibodies (e.g. mAbl.5.1) Mice were immunized with either fixed or lysed HT-1080 cells twice over three months. For fixation, confluent (lxl07/75cm2 flask) HT-1080 cells were washed once with PBS, then removed with PBS and scraping. Cells were resuspended by pipeting, centrifuged for 5 minutes at 220Xg, and fixed with lOmL of 2 % paraformaldehyde for 10 minutes at 4°C. Cells were washed with PBS, then resuspended in 1ML of PBS. Lysed cells were generated by trypsinizing confluent HT-1080 cells, washing once with PBS, then pelleting for 1 minute at 12000xg. Cells were then lysed in lysis buffer (0.67 Triton-X-100 in 0.33M Tris-HCI, pH 7.5) at room temperature for 5 minutes. Lysate was then centrifuged for 5 minutes at 12000xg. Triton-X-100 was removed by adding the lysate to PD-10 columns and eluting protein with 3.5mL PBS. Protein was quantitated using Micro-BCA kit (Pierce), and 150ng/mL/mouse was used for immunization.<br>
Generation of hybridomas was performed as described previously (Harlow and Lane, 1988).<br>
Hybridoma supernatants were first screened by fixing a confluent monolayer of HT-1080 cells (40,000 per well of a 96-well clear thin bottom plate) in 4% paraformaldehyde /PBS for 30 minutes at room temperature and then permeabilizing in 0.2% Triton-X-100 for 5 minutes. Following incubation with undiluted hybridoma supernatant, three 5 min 0.1% bovine serum albumin (BSA) in phosphate buffered saline (PBS) washes done. A FITC-labelled rabbit-anti-mouse secondary antibody was added for 1 hour at room temperature. The same 0.1%BSA/PBS washes were repeated and a tertiary FITC-labelled, goat-anti-rabbit antibody was added for 1 hour at room temperature. After a final round of washes, the cells were stored in PBS at 4°C until imaging. Images were collected (Zeiss Axiovert 10 using a Neofluar 40X/0.75 numerical aperture lens) using excitation filter set at 480 and emission filter set at 535 for FITC. Cells were scored for staining com-<br><br>
pared to positive controls (anti-fil integrin, Chemicon #MAB1963 and anti-gelsolin, Sigma #G4896) and negative controls (no primary antibody). The hybridomas that tested positive for HT-1080 binding were then tested for surface expression by surface immunocytochemistry. For surface protein immunocy-tochemistry, a confluent monolayer of HT-1080 cells (40,000 per well of a 96-well clear thin bottom plate) were incubated with hybridoma supernatant for 1 hour at 37°C/5%C02. After three 5 min 0.1% bovine serum albumin (BSA) in phosphate buffered saline (PBS) washes, the cells were fixed in 2% paraformaldehyde for 2 min. Another round of 0.1%BSA/PBS washes was performed, and a FITC-labelled rabbit-anti-mouse secondary antibody was added for 1 hour at room temperature. The same washes were repeated and a tertiary FITC-labelled go at-anti-rabbit antibody was added for 1 hour at room temperature. After a final round of washes, the cells were stored in PBS at 4°C until imaging. Surface staining was visualized by microscopy on a Zeiss Axiovert 10 using a Neofluar 40X/0.75 numerical aperture lens. Images were collected using excitation filter set at 480 and emission filter set at 535 for FITC. Cells were scored for surface staining compared to a positive control (anti-pi integrin, Chemicon #MAB1963 and anti-gelsolin, Sigma #G4896) and negative controls (no primary antibody).<br>
Example 2: Construction of an immune library<br>
Two BALB/c mice were each immunized intradermally with 2 x 107 paraformaldehyde fixed HT-1080 cells (human fibrosarcoma cell line; ATCC, CCL-121). Following the first immunization, the injections were repeated twice in a period of 39 days, the mice sacrificed and the spleens isolated and frozen in liquid nitrogen.<br>
Total RNA was isolated using the RNeasy Midi Kit (QIAGEN #75142) as described by the manufacturer using half of each spleen preparation. The RNA concentration and purity was determined by a denaturing formaldehyde gel and photometric<br>
measurement.<br>
cDNA was synthesized using 8.9 ng of freshly prepared RNA and 10 pmol of a<br>
primer mix IgGl-c, IgG2a-c, IgG2b-c, IgG3-c, VLL-c, VLK-c) using the Superscript™ II Kit (GibcoBRL Life Technologies #18064-014). These primers anneal to the RNA encoding the IgG heavy-chain (VH) genes and the light chain (VL) genes of the kappa and lambda family. VH genes were PCR amplified from 1 *il of cDNA using 36 individual combinations of 9 forward primers (M-VH1, M-VH2, M-VH3, M-VH4, M-VH5, M-VH6, M-VH7, M-VHS, M-VH9) and 4 backward primers (M-JH1, M-JH2, M-JH3, M-JH4) without restriction sites. VL genes were PCR amplified with one primer mix (M-VK1, M-VK2, M-VK3, M-VK4, M-VL1, M-JK1, M-JK2, M-JK3, M-JL1) without restriction sites. PCR products were gel-purified (QIAquick<br><br>
Gel Extraction Kit, #28706) and reamplified using individual combinations of 9 forward primers (MVH1 Sfil MVH2 Sfil MVH3 Sfil, MVH4 Sfil, MVH5 Sfil, MVH6 Sfil, MVH7 Sfil, MVH8 Sfil, MVH9 Sfil) and 4 backward primers (M-JH1 Sail, M-JH2 Sail, M-JH3 Sail, M-JH4 Sail) with restriction sites for VH and one primer mix (M-VK1 ApaLI, M-VK2 ApaLI, M-VK3 ApaLI, M-VK4 ApaLI, M-VL1 ApaLI, M-JK1 NotI, M-JK2 NotI, M-JK3 NotI, M-JL1 NotI) with restriction sites for VL. PCR products were gel-purified (QIAquick Gel Extraction Kit, #28706) and cloned into the phage display vector pXPIO using the restriction sites Sfil/ Sail for VH and ApaLI/NotI for VL. The ligation mix was transfected into E.coZx TG-1 by electropo-ration resulting in a library size of 107 independent clones (phages) expressing different single chain antibod}' fragments (scFv).<br>
Example 3: Selection and Screening of scFv (Selection on fixed cells)<br>
Single chain Fv were selected from a phage display library generated from mice immunized with fixed HT-1080 cells. The Hbrary was generated using the phage display vector pXPIO.<br>
Phages expressing scFv with high affinity to tumor cells were selected as follows: HT-1080 cells were harvested with 0.05% EDTA, fixed with paraformaldehyde, diluted to lxlO7 cells/ml in PBS and immobilized onto wells of a 96 well UV crosslink plate (Corning Costar). The wells of the UV cross-link plate were blocked with 5% Skim Milk Powder (#70166, Fluka) in PBS (MPBS). 10" cfu (colony forming units) of phage library/106 cells were pre-blocked for 1 hour at 25°C with MPBS and subsequently incubated for 1.5 hour at room temperature (RT) with the cells. The wells of the UV cross-link plate were washed six times with PBS + 0.05% Tween-20 followed by six washes with PBS. Bound phage were eluted by the addition of 10 mM Glycine pH 2.2, and neutralized with 1 M Tris/HCl pH 7.4. Typically, between 103 and 106 cfu were eluted in the 1st round of selection, thus the diversity of the enriched repertoire is decreased compared to the original repertoire. The eluate containing the enriched repertoire was amplified by infecting exponentially growing E. coli TGI. Phagemid containing 22. coli were selected and propagated by overnight (o/n) growth at 30°C on LB agar plates supplemented with 100^g/ml ampicillin and 1% glucose. Following this step, the enriched repertoire can either be amplified as a polyclonal pool and used for further rounds of selection in an iterative manner until convergence to desired properties is achieved or be spatialty separated and screened on a single clone level. Phage particles for the next round of selection were produced by super-infecting exponentially growing cultures of the previous round of selection with helper phage VCS-M13 (Stratagene, La Jolla, CA) and growing the cultures overnight at 20°C in 2xTY<br><br>
supplemented with lOOjxg/ml ampicillin (amp) and 50 fig/ml kaaamycin (kan). Selection ready phage were precipitated with 0.5 M NaCl/4% PEG-6000 from the cleared bacterial supernatant and re-suspended in PBS. One round of selection was performed, followed by screening on a single clone level as described in Example 4.<br>
Example 4: Selection and Screening of scFv (Screening on fixed cells)<br>
For screening, the genes encoding the selected scFv, contained in the phage display vector, were re-cloned to the expression vector pXP14. This vector directs the expression of scFv in fusion with a Strep-tag and E-tag and does not contain a filamentous phage gene-3. Expression vector containing B. coli TGI from single colonies were grown in individual wells of a micro titer plate so that each well contains only one scFv clone. The bacteria were grown at 30°C in 2xTY supplemented with lOO^ig/ml ampicillin and 0.1% glucose in 96-well micro titer plates (#9297, TPP) until an ODeoo of 0.7. Expression was induced with IPTG at a final concentration of 0.5 mM and continued at 25°C overnight. Single chain Fv containing cleared lysates were prepared by addition of hen-egg lysozyme (#L-6876, Sigma) to a final concentration of 50 fig/ml for 1 hour at 25°C and centrifugation for 15 minutes at 3000 x g. Prior to the screening ELISA, the cleared lysates were blocked by the addition of an equal volume of DMEM + 10% FCS for 1 hour. For the screening ELISA, HT-1080 cells were harvested with 0.05% EDTA, fixed with paraformaldehyde, diluted to lxl07cells/ml in PBS and immobilized onto wells of a 96 well UV cross-link plate (Corning Costar). The wells of the UV cross-link plate were blocked with MPBS and the scFv containing blocked cleared lysates added for 1.5 hours at 25°C. The plates were washed 2x with PBS + 0.1% Tween-20 and lx with PBS, incubated with HRP conjugated a-E-tag (#27-9413-01, Pharmacia Biotech; diluted 1:5000 in MPBS with 0.1 % Tween-20) for 1 hour, washed 3x with PBS + 0.1% Tween-20 and 3x with PBS, developed with POD (#1 484 281, Roche) and signals read at 370 nm.<br>
Positive clones were retested against HT-1080 cells and control human fibroblasts Hs-27 (ATCC CRL-1634) using the ELISA screening procedure described above and preserved in glycerol stocks. In a typical screen, 2760 (30x92) clones were screened for binding to HT-1080 cells with 5 % positives defined as clones giving a background subtracted signal &gt; 0.1. 155 positive clones were retested for specific binding to HT-1080 cells compared to the Hs-27 control cells with 28 % positives defined as clones giving a background subtracted signal on HT-1080 of at least twice the value of the signal on Hs-27 control cells.<br><br>
Example 5: Sequencing and large scale expression<br>
Sequencing of scFvl and its genes was performed by Sequiserve GmbH, Vaterstet-ten, Germany using the primer pXP2 Seq2 (5'-CCCCACGCGGTTCCAGC-3' and pXP2 Seql (5'TACCTATTGCCTACGGC-3'. The amino acid sequence and nucleotide sequence are shown in the Figures.<br>
Unique clones identified by sequencing were streaked out from glycerol stocks onto LB/Amp(100*ig/ml)/l% Glucose Agar plates and incubated o/n at 30DC. 10 ml LB/Amp/Glu (1%) media were inoculated with a single colony and grown o/n at 30°C and 200 rpm shaking. The next morning the overnight cultures were placed on ice until inoculation of 1L 2xTY media supplemented with 100)ig/ml Ampicillin and 0.1% Glucose in 2L Erlenmeyer-flasks. The cultures were grown at 25°C shaking until an ODeoo 0.5 - 0.6 was reached and then induced with IPTG 0.1 mM final concentration. Fresh ampicillin was added to 50 fig/ml and incubation was proceeded at 22°C o/n shaking. In the morning the cultures were centri-fuged at 5000 x g for 15 minutes at 4°C, supernatants discarded and the pellets re-suspended carefully on ice with a pipette in 10 ml pre-cooled PBS-0.5 M Na buffer containing protease inhibitors complete (#1697498, Roche). After re-suspension was completed, bacterial suspensions were transferred to 20 ml oak-ridge centrifuge tubes and hen-egg lysozyme (#L-6876, Sigma) added to a final concentration of 50 [ig/ml for 1 hour on ice. The lysed bacteria were centrifuged at 20000 x g for 15 minutes at 4°C and the supernatants (lysate) transferred to a 15 ml plastic tube. For affinity purification the lysates were loaded with 1 ml/min onto 1ml StrepTactin (# 2-1505-010, IBA) columns equilibrated with 10 column volumes (CV) PBS-0.5 M Na buffer via a parallel protein purification system. After a 10 CV wash with PBS the elution was done with 5 CV PBS/5mM Desthiobiotin (#D-1411, Sigma) and 1 ml fractions collected. The fractions were measured at UV280, protein containing fractions were pooled and concentrated with Amicon Ultra Centrifugal Filter Devices 10.000 MWCO (#UFC801024, Millipore) at 4700 x g. The concentrated scFv were checked on 12% Bis-Tris SDS-PAGE gels stained with Coomassie Blue for purity and frozen in aliquots with 20 % glycerol at -80°C.<br>
Example 6: FACS analysis for tumor cell specific binding<br>
To test the ability of purified anti HT-1080 scFv to bind specifically to target cells, we performed a fluorescence-activated cell sorter (FACS) analysis using HT-1080 cells (ATCC CCL-121) and Hs-27 cells (106 cells/ml) as control cell line (see Figures). Cells were incubated with 10 jig/ml of pure scFv in CellWash (BD (Becton, Dickinson and Company) #349524) for 20 min at 4DC, washed, and bound scFv's<br><br>
were detected with a secondary FITC labeled anti E-tag mab (Amersham #27-9412-01). Samples were washed and analyzed on a Becton Dickinson FACSscan. Figure 8 shows the log fluorescence intensity (FL1-H; x-axis) versus the relative cell numbers (counts; y-axis) for cells reacting with scFvl. The thin line represents the control cell line (HS-27) and the bold line the HT-1080 cells. scFvl specifically stains the tumor cell lines with up to 10 fold higher signal compared to the control cell line.<br>
Example 7: Labeling of scFv with FITC<br>
scFv were labeled with fluorescein isothiocyante (FITC) (Molecular Probes, Eugene, USA #F1906) by the following method: Aliquots of a lOmg/ml solution of FITC in dry dimethyl sulfoxide were added to lOOjig of scFvl dissolved in PBS/0.5M Na-HC03) pH 9.5 in a ratio of 30:1 (FITC:scFvl). The sample was incubated for two hours at room temperature with agitation, free FITC was separated using desalting columns (2 Micro Spin G-25, Pharmacia 27-5325-01). The ratio of labeling was determined via mass spectrometry and via UV/VIS spectroscopy, whereby the protein concentration was calculated at 280nm and the FITC concentration at 494nm.<br>
Example 7.1: Labeling of mAbl.5.1 with FITC<br>
A solution of freshly made lOmg/mL FITC (Molecular Probes, Eugene, USA #F1906) in dry dimethyl sulfoxide was added in a 1:5 ration with purified mAb 1.5.1 antibody (T-gel Absorbant, Pierce Biotechnology #20500). An equal volume of 0.5M NaHCC&gt;3, pH 9.5 was added to the reaction, and the reaction was incubated with rocking for two hours at room temperature. Free FITC was separated using a desalting column (PD-10, Amersham #17-0851-01). The ratio was determined by calculating the FITC concentration by VIS spectroscopy at 494nm and assuming complete protein recovery from the labeling.<br>
Example 8: Invasion assay for identification of inhibitory antibody fragments<br>
The ChemoTx®' system (Neuro Probe Inc.#106-8, Gaithersburg) is used as a disposable chemotaxis/cell migration chamber in a 96 well format with an 8nm filter Track etched Polycarbonate pore size, 5.7 mm diameter/site.<br>
13,3^1 of 0.3mg/ml Matrigel (Matrigel is a solubilised basement membrane preparation extracted from the Engelbreth-Holm-Swarm (EHS) mouse sarcoma, a tumor rich in extracellular matrix proteins. Its major component is laminin, followed by collagen IV, heparan sulfate proteoglycan, entactin and nidogen. It also contains<br><br>
TGF-p fibroblast growth factor, tissue plasminogen activator, and other growth factors which occur naturally in the EHS tumor) (Becton Dickenson, BD #356234) diluted in Dulbeccos PBS (Gibco #14040-091) is applied on the membrane filter of the 96-well plate on row B-H and on row A 1,2 ng/site of collagen S type I (Roche #10982929) is diluted in 0?05 M HC1 (Sigma #945-50) and is incubated over night at 20°C in a desiccator for gelation. HT-1080 cells are grown to 70-80% confluence in DMEM supplemented with GlutamaxI (862mg/l (Gibco #31966-021) with 10% FCS (Gibco #10270106). The cells are washed 2x with DM EM /GlutamaxI/ 0.1 % BSA (Sigma #A-7030) then labeled in situ with Bisbenzimide H 33342 (Sigma #B-2261) are diluted 1:100 in DMEM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% C02. Cells are washed 2x with DMEM/GlutamaxI/0.1 % BSA and are loaded with DMEM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% C02 for recovering. After washing 2x with PBS w/o Ca2+, Mg*+ (Gibco, 10010-015), the cells are detached with 0.5mM EDTA (Sigma #E8008), are collected with Dulbeccos PBS/0.1% BSA/lOmM Hepes (Gibco #15630-056), are washed 2x with Dulbeccos PBS/0.1% BSA/lOmM Hepes, are suspended in Dulbeccos PBS/0.1% BSA/lOmM Hepes and are diluted to 6,7 x 10* cells/ml with Dulbeccos PBS/0.1% BSA/lOmM Hepes. 6,7 x 106 cells/ml are incubated 1:1 with 40fig/ml of a control scFv as a negative control for inhibition of invasion and with HT-1080 specific scFv for lh on ice. After dilution to 6,7 x 105 cells/ml with DMEM/GlutamaxI/0.1 % BSA, HT-1080 ceUs and HT-1080 cell/scFv dilutions are pipetted in triplicate onto the chemotaxis chamber (row B-H) at a density of 3,4 x 104 cells/well and are incubated for 6 h at 37°C, 7,5% C02. DMEM/GlutamaxI with 5% FCS is used as a chemo attractant in the lower chamber. A standard curve from lxl04 to 4xl04 cells/site is performed on collagen S type I coated row A of the chemotaxis chamber. DMEM/GlutamaxI/0.1% BSA is used in the lower chamber (cells are not migrating). After scraping the non-migrating cells from the top of the membrane (except the Standard curve on row A) fluorescence of cells, which migrated through the membrane (not migrated in case of the Standard curve), is measured on the Flu-ostar Galaxy (bMG) microplate reader using excitation/emission wavelengths of 370/460nm.<br>
Example 9: Invasion assay for target identification with CALI<br>
This Example is in general identical to Example 8, except for the use of FITC-labeled scFv (see, Example 7 for labeling) and the integration of the CALI process within the invasion assay.<br><br>
The ChemoTx® system {Neuro Probe Inc.#106-8, Gaithersburg) was used as a disposable chemotaxis/cell migration chamber in a 96 well format with an 8fim filter Track etched Polycarbonate pore size, 5.7 mm diameter/site.<br>
13,3^1 of 0.3mg/ml Matrigel (see Example 8) diluted in Dulbeccos PBS (Gibco #14040-091) was applied on the membrane filter of the 96-well plate on row B-H and on row A 1,2 Mg/site of collagen S type I (Roche #10982929) was diluted in 0,05 M HC1 (Sigma #945-50) and was incubated over night at 20°C in a desiccator for gelation. HT-10S0 cells were grown to 70-80% confluence in DMEM supplemented with GlutamaxI (862mg/l (Gibco #31966-021) with 10% FCS (Gibco #10270106). The cells were washed 2x with DMEM/GlutamaxI/0.1 % BSA (Sigma #A-7030) then labeled in situ with Bisbenzimide H 33342 (Sigma #B-2261) and were diluted 1:100 in DMEM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% COs. Cells were washed 2x with DMEM/GlutamaxI/0.1 % BSA and loaded with DMEM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% C02 for recovering. After washing 2x with PBS w/o Ca2+, Mg2+ (Gibco, 10010-015), the cells were detached with 0.5mM EDTA (Sigma #E8008), collected with Dulbeccos PBS/0.1% BSA/lOmM Hepes (Gibco #15630-056), washed 2x with Dulbeccos PBS/0.1% BSA/lOmM Hepes, suspended in Dulbeccos PBS/0.1% BSA/lOmM Hepes and diluted to 6,7 x 106 cells/ml with Dulbeccos PBS/0.1% BSA/lOmM Hepes. 6,7 x 10* cells/ml were incubated 1:1 with 40jig/ml of FITC-labelled anti-betal integrin monoclonal antibody (JB1, Chemicon #MAB1963) as a control for inhibition of invasion after CALI and with HT-1080 specific FITC labelled scFv for lh on ice. 1,3x105 HT-1080 cells/well or HT-1080 cell/scFv or Ab dilution were pipetted in triplicate in two 96-well plate, black, ultra thin clear flat bottom special optics (Costar #3615). One plate was kept on ice in the dark while the other plate was irradiated on an ice block with continuous wave laser at 488nm (0.5W, 30 sec). After dilution to 6,7 x 10* cells/ml with DMEM/GlutamaxI/0.1 % BSA, HT-1080 cells and HT-1080 cell/scFv dilutions were pipetted in triplicate (non irradiated triplicate beside irradiated triplicate) onto the chemotaxis chamber (row B-H) at a density of 3,4 x 104 cells/well and incubated for 6 h at 37°C, 7,5% C02. DMEM/GlutamaxI with 5% FCS was used as a chemo attractant in the lower chamber. A standard curve from IxlO4 to 4xl04 cells/site was performed on collagen S t}'pe I coated row A of the chemotaxis chamber. DMEM/GlutamaxI/0.1% BSA was used in the lower chamber (cells were not migrating). After scraping the non-migrating cells from the top of the membrane (except the Standard curve on row A) fluorescence of cells, which had migrated through the membrane (not migrated in case of the Standard curve), was measured on the Fluostar Galaxy (bMG) microplate reader using excitation/emission wavelengths of 370/460nm. In a general experiment, a value of 45000 corresponded to 100% migrated cells.<br><br>
The invasion phenotype of HT-1080 cells was assessed by comparing their relative ability to invade tumor extracellular matrix (Matrigel) using the Transwell culture system described above. scFvl showed after CALI an inhibitory effect of 46% on the invasion of the HT-1080 cells. The result of the invasion assay with CALI is shown in Figure 3. Figure 3 shows that CALI converts scFvl in an inhibitory antibody fragment.<br>
Example 9.1: Invasion assay with antibodies (e.g. mAb 1.5.1) for target identification with CALI<br>
The ChemoTx® system (Neuro Probe Inc. #106-8, Gaithersburg) was used as a disposable chemotaxis/cell migration chamber in a 96 well format with an 8fim filter Track etched Polycarbonate pore size, 5.7 mm diameter/site.<br>
13.3|d of 0.3mg/ml Matrigel (see Example 8) diluted in cold Dulbeccos PBS (Gibco #14040-091) was applied on the membrane filter of the 96-well plate on row B-H and on row A0.3fig/site of mouse collagen, type IV (Becton-Dickenson #354233) was diluted in 0.05 M HCL (Sigma #945-50) and was incubated over night at 20°C in a desiccator for gelation, HT-1080 cells were grown to 70-80% confluence in DMEM supplemented with MEM non-essential amino acid? (IX, Gibco #11140050) and Penicillin/Streptomycin (IX, Gibco #15140122) with 10% FBS (Hyclone #SH30070.03). The cells were washed Ix with DMEM/MEM non-essential amino acids /0.1 % BSA/MEM amino acids /Penicillin-Streptomycin (BSA, Sigma #A-7030) then labeled in situ with 3jiM Cell Tracker Orange (Molecular Probes #C-2927) in DMEM/MEM non-essential amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin for 15 min at 37°C, 7.5% C02. Cells were washed lx with DMEM/MEM non-essential amino acids /0.1 % BSA/MEM amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin for 15 min at 37°C, 7.5% C02 for recovering. After washing lx with Hanks Bal-anced Salt Solution (HBSS) w/o Ca2+, Mg2+ (Gibco #14170112), the cells were detached with Versene (Gibco #15040066), collected with HBSS/MEM non-essential amino-acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin (HBSS, Gibco +14025092), washed 2x with HBSS/MEM non-essential amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin, suspended in HBSS/MEM non-essential amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin and diluted to 8 x 106 cells/ml with HBSS/MEM non-essential amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin. 8 x 105 cells/ml were incubated 1:1 with 40Mg/ml of FITC-labeled anti-betal integrin monoclonal antibody (JB1, Chemicon #MAB1963) as a control for inhibition of invasion after CALI and with mAbs for lh on ice, 3xl05 HT-1080 cells/well or HT-1080 cell/ mAbs were pipetted in triplicate in two<br><br>
clear 96-well plates (Costar #3370). One plate was kept on ice in the dark while the other plate was irradiated on ice block with 300W blue-filtered light (Roscolux, #69, Brilliant Blue) for one hour. After dilution to 8 x 10* cells/ml with DMEM/MEM non-essential amino acids /0.1 % BSA/MEM amino acids/Penicillin-Streptomycin, HT-1080 cells and HT-1080 cells/mAbs dilutions were pipetted in triplicate (non-irradiated triplicate be-side irradiated triplicate) onto the chemo-taxis chamber (row B-H) at a density of 4 x 104 cells/well and incubated for 6 h at 37°C, 7.5% C02, DMEM with 5% FCS was used as a chemoattractant in the lower chamber. A standard curve from lxlO4 to 4xI04 cells/site was performed on mouse collagen IV coated row A of the chemotaxis chamber. DMEM/MEM nonessential amino acids/0.1 % BSA/MEM amino acids/Penicillin-Streptomycin was used in the lower chamber (cells were not migrating). After scraping the non-migrating cells from the top of the membrane (except the Standard curve on row A) fluorescence of cells, which had migrated through the membrane (not migrated in case of the Standard curve), was measured on the Tecan Spectrafluor Plus fluorescence plate reader (excitation: 544, emission: 590nm). A positive control which recognizes pi integrin (data not shown) and mAb 1.5.1 both showed a significant (p 
Example 10: Cell-matrix adhesion assay<br>
96-well plates (TPP #9296) (cell culture treated) were coated in Row B-H with collagen S type I 1 ng/well (Roche #10982929) in Dulbeccos PBS (Gibco #14040-091) and in Row A well 10-12 were coated with 2% BSA (Sigma #A-7030)/Dulbeccos PBS at 4°C over night. The plate was washed with Dulbeccos PBS, blocked Row B-H and Row A well 10-12 with 2% BSA/Dulbeccos PBS for lh at 37CC and washed again with Dulbeccos PBS. HT-1080 cells were grown to 70-80% confluence in DMEM supplemented with GlutamaxI (862 mg/1 (Gibco #31966-021) with 10% FCS (Gibco #10270106). The cells were washed 2x with DM EM/GlutamaxI/0.1 % BSA (Sigma #A-7030) then labeled in situ with Bisbenzimide H 33342 (Sigma #B-2261) were diluted 1:100 in DMEM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% C02. Cells were washed 2x with DMEM/GlutamaxI/0.1 % BSA and loaded with DM EM/GlutamaxI/0.1 % BSA for 15 min at 37°C, 7,5% C02 for recovering. After washing 2x with PBS w/o Ca2+, Mg2+ (Gibco, 10010-015), the cells were detached with 0.5 mM EDTA (Sigma #E8008), collected with Dulbeccos PBS/0.1% BSA/lOmM Hepes (Gibco #15630-056), washed 2x with Dulbeccos PBS/0.1% BSA/lOmM Hepes, suspended in Dulbeccos PBS/0.1% BSA/lOmM Hepes and diluted to 6,7 x 106 cells/ml with Dulbeccos PBS/0.1% BSA/lOmM Hepes. 6,7 x 10^<br><br>
cells/ml were incubated 1:1 with 40 ng/ml of FITC-labeled anti-betal integrin monoclonal antibody (JB1, Chemicon #MAB1963) as a control for inhibition of adhesion after CALI and with HT-1080 specific FITC labeled scFv for lh on ice. I,3xl05 HT-1080 cells/well or HT-1080 cell/scFv or Ab dilution were pipetted in triplicate in two 96-well plate, black, ultra thin clear flat bottom special optics (Costar #3615). One plate was kept on ice in the dark while the other plate was irradiated on an ice block with continuous wave laser at 488 nm (0.5W, 30 sec). After dilution to 6,7 x 105 cells/ml with DMEM/GlutamaxI/0.1 % BSA, HT-1080 cells and HT-1080 cell/scFv dilutions were pipetted in triplicate (non irraditated triplicate beside irradiated triplicate) onto the coated and blocked plate. In Row A well 10-12 6,7 x 105 cells/ml with DMEM/GlutamaxI/0.1 % BSA were pipetted as a background control. Plate was incubated for lh at 37°C, 7,5% CO2 and washed 2x with Dulbeccos PBS, where non-adherent cells were washed away. In Row A well 1-9 a standard curve from lxlO4 to 4xl04 cells/well was performed, in all other wells 50 pi Dulbeccos PBS was pipetted. Fluorescence of cells, which had adhered to the Collagen S type 1 (not adhered in case of the Standard curve), was measured on the Fluostar Galaxy (bMG) microplate reader using excitation/emission wavelengths of 370/460 nm. scFvl inhibited the adhesion of HT-1080 cells to Collagen S type 1 by 50%. (Data not shown).<br>
Example 11: Immunoprecipitation<br>
For immunoprecipitation of mAbs, confluent monolayers of HT-1080 cells were lysed using imM Tris-Cl, pH 8.0 containing protease inhibitor cocktail (Bohrenger Mannheim) for 15 minutes at 4°C. Cells were further lysed using a dounce ho-mogenizer for 5 minutes. Lysates were pre-adsorbed for 1 hour at 4°C with anti-mouse IgG-agarose (Sigma) Ascities fluid from the target hybridoma clone (lO^L/l mg cell extract) was incubated overnight at 4°C. Antibody-protein complexes were isolated using anti-mouse IgG-agarose beads for 1 hour at 4°C. Immunoprecipi-tated proteins were analyzed by SDS-PAGE and Coomassie staining, and mAb im-munoprecipitates were further analyzed by immunoblots with the target ascities.<br>
For scFv immunoprecipitations, HT-1080 specific scFvs were coupled to StrepTac-tin Sepharose (50*ig/50[il resin) and the washed scFv-beads added to the cleared lysates (lmg total protein) for 2-3 h at 4°C. The scFv-target complexes were eluted with 50jil lOmM D-desthiobiotin in PBS 0.1% Tween 20. Immunoprecipitated proteins were analyzed by SDS-PAGE and silver staining.<br>
Example 12: Protein identification via mass spectroscopy<br><br>
For mAb target identification, coomassie-stained bands from the immunoprecipi-tates separated by SDS-PAGE were next subjected to in-gel digestion. The trypsinized peptide fragments were run on a Surveyor HPLC and LCQ Deca Ion<br>
Trap mass spectrometer (ThermoFinnigan) with a 75|iM nanospray C18 column (New Objectives), The obtained PMF (peptide mass fragments) were used to search all entries for the species Homo sapiens in the NCBI and SwissProt databases.<br>
For scFv target identification, stained bands were cut out and subjected to in-gel digestion with trypsin. The peptide fragments were extracted from the gel pieces, desalted on ZipTip JJLC18 and the eluted peptides spotted on a Teflon-coated MALDI target (Applied Biosystems). The samples were analyzed on a STR-DE Voyager MALDI mass spectrometer (Applied Biosystems) and the obtained peptide masses were used for protein identification via PMF (peptide mass fingerprint), searching all entries for the species Homo sapiens in the NCBI and Swiss-Prot databases.<br>
Example 13: Immunocytochemistry<br>
For surface protein immunocytochemistry, a confluent monolayer of HT-1080 cells (40,000 per well of a 96-well clear thin bottom plate) were incubated with hybri-doma supernatant for 1 hour at 37°C/7%C02. After three 5 minute 0.1% bovine serum albumin (BSA) in phosphate buffered saline (PBS) washes, the cells were fixed in 2% paraformaldehyde for 2 minutes. Another round of 0.1% BSA/PBS washes was performed and a FITC-rabbit-anti-mouse secondary antibody was added for 1 hour at room temperature. The same washes were repeated and a tertiary FITC-goat-anti-rabbit antibody was added for 1 hour at room temperature. After a final round of washes, the cells were stored in PBS at 4°C until imaging. Surface staining was visualized by microscopy on a Zeiss Axiovert 10 using a Neofluar 40X/0.75 numerical aperture lens. Images were collected using excitation filter set at 480 and emission filter set at 535 for FITC. Results are shown in Figures 6, 6.1 and 6.2.<br>
Example 14: Biotinylation of cell surfaces<br>
Biotinylation of cell surface proteins was performed as described in Hanwell at al (J Biol Chem 277:9772). The surface biotinylated pool and the unbiotinylated intracellular pool were separated on a SDS-PAGE gel, and transferred for im-munoblotting with anti-Hsp90 (Stressgen #SPA-830) or alpha-actinin-4 (Martin A. Pollak, Childrens Hospital, Boston, MA). Results are shown in Figure 7. Hsp90 shows both surface (S) and intracelluar (IC) localization, but alpha-actinin is<br><br>
found only in the intracellular pool. mAb 1.5.1 also shows surface and intracellular localization (data not shown).<br>
Example 15: Concentration dependency of Novobiocin or Nalidixic acid on the invasion of. HT-1080 cells<br>
8xl06 HT-1080cells/mL were pretreated for 1 hour with novobiocin or nalidixic acid at the concentrations shown and an invasion assay was performed as described in Example 9.1. Novobiocin shows a dose-dependent inhibition of invasion (p 0.5mM). Nalidixic acid has no effect on invasion. Viability of novobiocin-treated cells in this assay was not affected (data not shown). Each data point is normalized to the no drug control and represents the mean + s.e.m. of 2 experiments in triplicate. Results are shown in Figure 10.<br>
Example 16: Inhibition of MMP secretion<br>
40,000 HT-1080 cells were treated for 6 hours with novobiocin or nalidixic acid at the given concentrations. The conditioned media from treated cells was separated using a SDS-PAGE gel containing lOmg/mL gelantin. SDS was removed from the gel using 2.5% Triton-X-100 for 30 minutes at room temperature. Proteins were allowed to digest the gelatine in digestion buffer (0.1M Tris-Ck, pH 8.0, 5mM CaCl2, 0.04% NaN3) for 48 hours at 37°C. Gels were coomassie-stained and dried for densitometry. Parallel gels were silver stained to show equal loading of protein (data not shown). MMP activity was decreased by &gt;75% at the highest concentration of novobiocin tested and no reduction of enzyme activity was seen with nalidixic acid. Results are shown in Figure 11.<br>
Example 17: Covalent coupling of novobiocin to sepharose and inhibition of MMP<br>
activity<br>
This example demonstrates inhibition of Hsp90 function on the surface of HT-1080 cells. It has been shown that treatment of HT-1080 fibrosarcoma cells with novobiocin, a coumarin antibiotic, reduces the cell's ability to invade a basement membrane barrier and also significantly reduces matrix metalloprotease (MMP) activity/secretion. Since the experiment is performed on a short timescale (6 hours), significant depletion of intracellular Hsp90 targets can not take place.<br>
The covalent coupling of novobiocin to a macromolecular carrier like sepharose prohibits cellular uptake of novobiocin and consequently any intracellular inhibition of Hsp90. Marcu et al described a protocol for novobiocin conjugation to sepharose (Marcu et al. (2000) J Natl Cancer Inst 92, 242-8; Staudenbauer and Orr (1981)  Nucleic Acids Res.  9, 3589-603). In this procedure epoxy-activated<br><br>
Sepharose 6B (Sigma Chemical Co) is first washed and swollen with distilled water. The swollen beads are washed further with a coupling buffer (0.3M sodium carbonate, pH 9.5). Novobiocin is formally conjugated to the beads by incubation with the swollen beads in coupling buffer for 20 hours at 37°C. Unconjugated novobiocin is washed away using coupling buffer, and the remaining unbound ep-oxy-active groups are blocked with 1M ethanolamine for 12 hours at 30°C. Extensive washing of the beads using 0.5M NaCl in coupling buffer, distilled water, 0.5M NaCl in 0.1M sodium acetate (pH4.0) is done to remove any excess novobiocin and ethanolamine. The novobiocin-sepharose beads are then equilibrated in 25mM HEPES (pH 8.0) with ImM EDTA, 10% ethylene glycol, and 200mM KC1 at 4°C in the dark.<br>
Before proceeding to the gelatinase experiment, efficient binding of the novobiocin-sepharose conjugate to Hsp90 is demonstrated by an in vitro binding assay. HT-1080 cells are lysed in TNESV (1% Nonidet P-40, 2mM EDTA, and lOOmM NaCl, and ImM sodium orthovanadate) with protease inhibitors (tablet. Boe-hringer Mannheim). 300jig of this lysate are incubated with IOOJIL of the novobiocin-sepharose conjugate for 1 hour at 4°C. After extensive washing with TNESV, proteins bound to the beads are eluted with 2x SDS-PAGE loading buffer (125mM Tris-HCl pH6.8, 10% 2-mercaptoethanol, 10 SDS, 10% glycerol, and trace bromo-phenol blue) and boiling for 5 minutes. The proteins are run on 10% SDS-PAGE gels. These gels are either silver stained or transferred to nitrocellulose for im-munoblotting with an anti-Hsp90 antibody (AC88, Stressgen, Inc.)<br>
For the gelatinase assay, adherent HT-1080 cells are first gently removed from tissue culture dishes with Versene. 5xl04 cells in serum-free DMEM are combined with O.ljiL, lfiL or IOJUL of the novobiocin-sepharose conjugate and incubated at room temperature for one hour in siliconized eppendorf tubes with gentle rocking. Control samples containing sepharose beads not conjugated to novobiocin are done in parallel. After this incubation, the cells are transferred to a tissue culture treated 96-well plate and incubated further for 6 hours at 37°C/5% CO2. Conditioned media from these samples are then combined with 2x SDS-PAGE loading buffer (without 2-mercaptoethanol). These samples are run on a 10% SDS-PAGE gel containing lOmg/mL gelatine. SDS is then removed from the gel by incubating with 2.5% Triton-X-100 for 30 minutes at room temperature. Proteins are allowed to digest the gelatine in digestion buffer (0.1M Tris-HCl, pH 8.0, 5mM CaCl2, 0.04% NaNs) for 48 hours at 38°C. Gels will be coomassie-stained and dried for densitometry. Parallel gels are silver stained to show equal loading of protein. MMP activity decreases with increasing amounts of novobiocin-sepharose.<br><br>
Example   18:   Inhibition  of  MMP   activity  by  geldanamycin   and  immobilized<br>
geldanamycin (GA-beads)<br>
It could be shown that the matrix metalloproteinase MMP2 co-immunoprecipitated with Hsp90a from HT-1080 conditioned media (Figure 17). We then applied a known Hsp90 inhibitor, geldanamycin, to HT-1080 cells to test whether inhibition of Hsp90 reduced MMP2 activity. HT-1080 cells were treated with 20 M geldanamycin (GA) in serum-free media and secreted MMP2 was assayed by zymography. MMP2 (72 kDa gelatinase) activity was decreased by -35%; p
Geldanamycin is a well-known cell-permeant inhibitor of Hsp90. Treatment with 10 |iM geldanamycin (GA) for 1 hour caused a significant -35% decrease (p 
<br><br>
CLAIMS<br>
We claim:<br>
1.	An inhibitor of extracellular Hsp90, provided that the inhibitor is not a My-cograb® monoclonal antibody.<br>
2.	The inhibitor according to claim 1, wherein the inhibitor is modified such that the modification essentially prohibits its cellular uptake.<br>
3.	The inhibitor according to claim 1, wherein the modification of the inhibitor comprises conjugating the inhibitor to a carrier.<br>
4.	The inhibitor according to claim 1, wherein the inhibitor is selected from the group consisting of polypeptides, antibodies, antibody fragments, coumarin and purine based Hsp90 inhibitors and their analogs.<br>
5.	The inhibitor according to claim 4, wherein the Hsp90 inhibitor is one or more compounds selected from the group consisting of geldanamycin and its analogues such as 17 AAG, radicicol and coumarin antibiotics such as novobiocin and purine-based Hsp90 inhibitors such as PU3.<br>
6.	The inhibitor according to claim 1, wherein the inhibitor is an antibody, in particular a monoclonal antibody.<br>
7.	The inhibitor according to claim 1, wherein the inhibitor is an antibody fragment.<br>
8.	The inhibitor according to claim 1, wherein the inhibitor is labelled with a detectable label.<br>
9.	Pharmaceutical composition comprising an inhibitor according to claims 1-8, wherein the inhibitor is essentially unable to enter into a cell, provided that the inhibitor is not a Mycograb® monoclonal antibody.<br><br>
10.	Use of an inhibitor according to claims 1-8 for the preparation of a pharmaceutical composition for the prevention and/or treatment of proliferative disorders, cancer or metastasis.<br>
11.	Use of claim 10, wherein the inhibitor is selected from the group consisting of polypeptides, antibodies, antibody fragments, coumarine and purine based Hsp90 inhibitors.<br><br>
12.	Use according to claim 11, wherein the inhibitor is an antibody, in particular<br>
a monoclonal antibody.<br>
13.	Use according to claim 12. wherein the inhibitor is an antibody fragment.<br>
14.	A nucleic acid molecule encoding an inhibitor of extracellular Hsp90, wherein<br>
the inhibitor is an antibody or a fragment of an antibody.<br>
15.	A kit comprising an inhibitor according to any of the claims 1-S and suitable<br>
testing containers.<br>
16.	The kit according to claim 15. wherein the inhibitor is labelled.<br>
17.	A method for screening or testing the invasion and/or metastatic behavior of<br>
the cells ex vivo, comprising the steps of:<br>
a)	contacting the cell with one or more Hsp90 inhibitors under the conditions<br>
which essentially prohibit a cellular uptake of the said Hsp90 inhibitor;<br>
b)	analysing the migration of cells treated according to step a);<br>
c)	comparing the migration of the cells according to step a) with untreated cells<br>
and optionally<br>
d)	determining the percentage of migration in comparison with untreated cells.<br>
ps     .and  further comprising the stepsf. The method of claim 20, instead of step <br>
of:<br>
e)	contacting the ALDUs of step c) with immobilized Hsp?0;<br>
wiwn H-r.an and the ALDUs bound thereto from AL-<br>
f)	separating said immobilised H^p.U ana me<br>
DUs not bound to said immobilised Hsp90;<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1DSEVOUC0yMDA1IEFCU1RSQUNULnBkZg==" target="_blank" style="word-wrap:break-word;">2594-CHENP-2005 ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1DSEVOUC0yMDA1IENMQUlNUyBHUkFOVEVELnBkZg==" target="_blank" style="word-wrap:break-word;">2594-CHENP-2005 CLAIMS GRANTED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1DSEVOUC0yMDA1IENPUlJFU1BPTkRFTkNFIE9USEVSUy5wZGY=" target="_blank" style="word-wrap:break-word;">2594-CHENP-2005 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1DSEVOUC0yMDA1IENPUlJFU1BPTkRFTkNFIFBPLnBkZg==" target="_blank" style="word-wrap:break-word;">2594-CHENP-2005 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1DSEVOUC0yMDA1IEZPUk0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">2594-CHENP-2005 FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LWZvcm0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-form18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjU5NC1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2594-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234113-a-process-for-urea-production-as-well-as-to-a-plant-for-carrying-out-such-process.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234115-2-3-dioxo-1-2-3-4-tetrahydroquinoxalinyl-derivatives.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234114</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2594/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>29-May-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-May-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>TUFTS UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BALLOU HALL, MEDFORD, MA 02155,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JAY, DANIEL, G</td>
											<td>45 BIGELOW STREET, BRIGHTON, MA-02135,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>EUSTACE, BRENDA, K</td>
											<td>95 COOLIDGE STREET, #2, BROOKLINE, MA 02446,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SAKURAI, TAKASHI</td>
											<td>5-23-2 HIROO #505, SHIBUYA-KU, TOKYO 150-0012,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/47</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP04/02422</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-03-09</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/454,813</td>
									<td>2003-03-12</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>03015115.3</td>
									<td>2003-07-03</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234114-a-method-for-detecting-the-invasion-and-or-metastatic-behaviour-of-cells by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:45 GMT -->
</html>
