<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234275-a-method-for-producing-a-transgenic-bovine-cell-having-reduced-expression-of-functional-prion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234275:&quot;A METHOD FOR PRODUCING A TRANSGENIC BOVINE CELL HAVING REDUCED EXPRESSION OF FUNCTIONAL PRION PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR PRODUCING A TRANSGENIC BOVINE CELL HAVING REDUCED EXPRESSION OF FUNCTIONAL PRION PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for producing a transgenic bovine cell having reduced expression of functional prion protein, comprising introducing a first prion gene targeting vector into a bovine cell under conditions that allow homologous recombination between said first vector and a first allele of an endogenous prion nucleic acid in said cell, thereby introducing a hemizygous mutation in said cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Background of the Invention<br>
In general, the invention features cloned transgenic ungulates (e.g., bovines) in which prion prptein (PrP) activity is reduced by one or more genetically engineered mutations. Because such transgenic bovines with reduced prion protein activity should be resistant to prion-related diseases such as bovine spongiform encephalopy (BSE, also known as mad cow disease), they are a safer and preferred source for agricultural and pharmaceutical products, such as human therapeutic antibodies.<br>
Since the first case of BSE was discovered in the United Kingdom in 1986, this infectious disease has spread to other parts of the world, such as Japan. The threat of this disease has a great impact on the agricultural and pharmaceutical fields and has limited the use of cattle in these industries. Based on over a decade of research, prion protein has been identified as a substantial cause of this infectious disease. Animals having reduced prion protein activity are desirable-because of their expected resistance to prion protein-related infection.<br>
So far, transgenic mice having reduced prion protein activity have been created using conventional gene targeting strategies in homologous recombination-proficient murine embryonic stem (ES) cells. However, ES cells from ungulates such as cattle have been difficult to isolate, culture, and genetically modify.<br>
In the generation of prion knockout sheep, fetal fibroblasts were transfected with a promoterless knockout (KO) vector because the prion protein gene is very actively expressed in fetal fibroblasts (Denning et al., Nature Biotech., 19:559-562, 2001). By using this type of knockout vector, homologously targeted clones can be selected using an appropriate drug, such<br><br>
as G418 or puromycin, because the promoterless drug-resistant gene (e.g., a neomycin or puromycin-resistant gene) in the knockout vector can be expressed only when it is integrated into an actively expressed gene loci. However, only one live lamb was produced by nuclear transfer of these hemizygously targeted ovine flbroblasts, and then the lamb died about 12 days after birth. Unlike murine ES cells with an unlimited life-span, somatic flbroblasts have a limited life-span, which has made somatic gene targeting difficult because of the length of time required to culture the cells under stringent drug selection to select the desired knockout cells. Furthermore, in general, the frequency of homologous recombination in somatic flbroblasts is about 10-100 fold lower than in murine ES cells. These limitations pertaining to somatic flbroblasts in addition to the low success rate of traditional nuclear transfer methods have made it difficult to produce viable livestock with a desired site-specific mutation.<br>
Reduction of prion protein activity by gene targeting has been attempted in cattle. To our knowledge, successful generation of transgenic bovines with a mutation at the prion locus has not been reported, probably because of a lack of appropriate knockout vectors and/or nuclear transfer methods. Thus, improved knockout vectors are needed to mutate the prion locus in ungulate cells (e.g., bovine cells) with high efficiency. Additionally, improved methods for generating transgenic ungulates (e.g., bovines) from these genetically modified donor cells are desirable.<br>
Summary of the Invention<br>
The present invention features the design of knockout vectors with which very high frequency hemizygous and homozygous targeted integration (so-called homologous recombination) can be accomplished at the prion locus in ungulates (e.g., bovines) in donor cells such as fetal somatic flbroblasts. The invention also features methods for producing live calves having a hemizygous or homozygous mutation at the prion locus using genetically modified donor<br>
cells in any of the nuclear transfer methods described herein. These cattle are useful for producing pharmaceutical and agricultural products, such as therapeutic human antibodies for human use.<br>
The method of the present invention is achieved using several technologies, such as (i) prion gene knockout cells described herein, (ii) mammalian cloning methods such as nuclear transfer or chromatin transfer described in PCT Publication No. W002/051997, and (iii) introduction of a human artificial chromosome (HAC) such as delta HAC into an ungulate (PCT Publication No. W002/70648; Kuroiwa et al., Nature Biotechnol. 20:889-894, 2002). The prion knockout ungulate cell is used as source of donor genetic material in a mammalian cloning method, resulting in prion knockout (hemi or homo) offspring.<br>
Prion knockout ungulates may also have other useful features, such as production of human antibody. Such ungulates can be generated using a combination of the above technologies. For example, a prion knockout and human antibody-producing bovine can be generated by crossbreeding of a prion knockout ungulate and a human antibody-producing ungulate as described in PC Publication No. WO02/70648. Sequential manipulation of fetal fibroblast cells can also be used to generate such an ungulate, with or without breeding ungulates. Sequential manipulation of bovine fetal fibroblasts includes repeating the following steps: (i) genetic manipulation of an ungulate (e.g., a bovine) fibroblast cell, (ii) mammalian cloning using this cell, (iii) generation of a fetus, and (iv) isolation of a genetically-modified fetal fibroblast cell. For example, a prion knockout fibroblast cell can be sequentially manipulated to retain a HAC and to inactivate endogenous Ig genes.<br>
The resulting monoclonal or polyclonal xenogenous antibodies have a variety of uses; for example, they may be used as ingredients in prophylactic or therapeutic compositions for infection of pathogenic microorganisms such as bacteria or viruses.<br>
Transgenic ungulates and ungulate cells<br>
In one aspect, the invention provides an ungulate (e.g., bovine) or ungulate cell (e.g., bovine cell) having a non-naturally occurring mutation (e.g., a mutation after the initial ATC codon, such as a mutation that is within 10, 20, 50, or 100 nucleotides of this codon) in one or both alleles of an endogenous prion nucleic acid. Preferably, the mutation reduces or substantially eliminates the expression of functional prion protein. In preferred embodiments, expression of functional or total prion protein is decreased by at least 10, 20, 40, 60, 80, 90, 95, or 100%. The mutation may be hemizygous or homozygous. In some embodiments, the mutation includes an insertion of a positive selection marker (e.g., an antibiotic resistance gene) into the prion nucleic acid. Preferably, the positive selection marker is operably linked to a xenogenous promoter. For ungulates or ungulate cells with an antibiotic resistance gene inserted into both alleles of a prion nucleic acid, each allele may contain the same or a different antibiotic resistance gene. In a preferred embodiment, a negative selection marker (e.g., DT-A or Tk) is operably linked to a xenogenous promoter and is present in a vector used to mutate an endogenous prion allele. The mutation may or may not include the deletion of one or more nucleotides (e.g., contiguous nucleotides) in the prion nucleic acid.<br>
In preferred embodiments of the above aspect, the ungulate (e.g., bovine) or ungulate cell (e.g., bovine cell) has one or more transgenes and expresses an mRNA or protein (e.g., antibody) encoded by the transgene(s). Preferred ungulates contain naturally arranged segments of human chromosomes (e.g., human chromosomal fragments) or artificial chromosomes that comprise artificially engineered human chromosome fragments (i.e., the fragments may be rearranged relative to the human genome). In some embodiments, the xenogenous nucleic acid is contained within a chromosome fragment. The nucleic acid may be integrated into a chromosome of the ungulate or maintained in the ungulate cell independently from the host chromosome. In various embodiments, the nucleic acid is contained in a<br>
chromosome fragment, such as a AHAC or a AAHAC. In other embodiments, the xenogenous antibody is an antibody from another genus, such as a human antibody.<br>
Preferred ungulates and ungulate cells have one or more nucleic acids having a xenogenous antibody gene locus (e.g., a nucleic acid encoding all or part of a xenogenous immunoglobulin (Ig) gene that undergoes rearrangement and expresses at least one xenogenous Ig molecule) in one or more B-cells. Preferably, the nucleic acid has unrearranged antibody light chain nucleic acid segments in which all of the nucleic acid segments encoding a V gene segment are separated from all of the nucleic acid segments encoding a J gene segment by one or more nucleotides. Other preferred nucleic acids have unrearranged antibody heavy chain nucleic acid segments in which either (i) all of the nucleic acid segments encoding a V gene segment are separated from all of the nucleic acid segments encoding a D gene segment by one or more nucleotides and/or (ii) all of the nucleic acid segments encoding a D gene segment are separated from all of the nucleic acid segments encoding a J gene segment by one or more nucleotides. Other preferred ungulates have one or more nucleic acids encoding all or part of a rearranged xenogenous immunoglobulin (Ig) gene that expresses at least one xenogenous Ig molecule.<br>
In other preferred embodiments, the light chain and/or heavy chain of the xenogenous antibodies is encoded by a human nucleic acid. In preferred embodiments, the heavy chain is any class of heavy chain, such as \i, j, 6, e, or a, and the light chain is a lambda or kappa light chain. In other preferred embodiments, the nucleic acid encoding the xenogenous immunoglobulin chain or antibody is in its unrearranged form. In other preferred embodiments, more than one class of xenogenous antibody is produced by the ungulate. In various embodiments, more than one different xenogenous Ig or antibody is produced by the ungulate. The xenogenous antibody may be a polyclonal or monoclonal antibody.<br>
In various embodiments of the above aspect, the ungulate (e.g., bovine) or ungulate cell (e.g., bovine cell) has a mutation that reduces the expression of an endogenous antibody. Preferably, the mutation reduces the expression of functional IgM heavy chain or substantially eliminates the expression of functional IgM heavy chain. In other preferred embodiments, the mutation reduces the expression of functional Ig light chain or substantially eliminates the expression of functional Ig light chain. In yet other preferred embodiments, the mutation reduces the expression of functional IgM heavy chain and functional Ig light chain, or the mutation substantially eliminates the expression of functional IgM heavy chain and functional Ig light chain. Preferably, the ungulate also has a mutation in one or both alleles of an endogenous nucleic acid encoding alpha-(l,3)-galactosyltransferase and/or J chain. In other preferred embodiments, the ungulate has a nucleic acid encoding an exogenous J chain, such as a human J chain. Preferably, the mutation reduces or eliminates the expression of the endogenous alpha-(l,3)-galactosyltransferase enzyme, galactosyl(cd,3)galactose epitope, and/or J chain. Preferably, the ungulate produces human IgA or IgM molecules containing human J chain. Preferred ungulate cells (e.g., bovine cells) include, somatic cells, such as fetal fibroblasts or B-cells.<br>
The invention also features hybridomas that produce xenogenous (e.g., human) antibodies. In one such aspect, the invention provides a hybridoma formed from the fusion of a B-cell of the invention with a myeloma cell. Preferably, the antibody is reactive with an antigen of interest.<br>
Methods for producing transgenic ungulate cells<br>
The invention also features methods for generating ungulate cells (e.g., bovine cells) with a mutation in one or both alleles of a prion nucleic acid. These cells may be used as donor cells for the generation of transgenic ungulates (e.g., bovine prion knockout ungulates). Preferably, the mutation<br>
leads to a decrease in the amount of functional prion protein and/or a decreased incidence of prion related infections or diseases, such as BSE.<br>
Accordingly, in one such aspect the invention features a method for producing a transgenic ungulate (e.g., bovine). This method involves introducing a first prion gene-targeting vector into an ungulate cell under conditions that allow homologous recombination between the first vector and a first allele of an endogenous prion nucleic acid in the cell, thereby introducing a hemizygous mutation in the cell.   Preferably, the first vector includes a first region of homology having substantial sequence identity to a first region of an endogenous prion nucleic acid of the cell, a positive selection marker, and a second region of homology having substantial sequence identity to a second region of the prion nucleic acid. In a preferred embodiment, one region of homology is at least 1, 2, 3, 4, 5, 6, or 8 kilobases longer than the other region of homology. Preferably, the method also includes re-introducing the first vector into the cell under conditions that allow homologous recombination between the first vector and a second allele of an endogenous prion nucleic acid in the cell, thereby introducing a homozygous mutation in the cell. In other embodiments, the method also includes introducing a second prion gene targeting vector that has a different antibiotic resistance gene than the first vector into the cell under conditions that allow homologous recombination between the second vector and a second allele of an endogenous prion nucleic acid in the cell, thereby introducing a homozygous mutation in the cell. Preferably, the first and/or second vector is introduced into the cell in the presence of 1 mM of spermidine. Preferred cells include bovine fetal fibroblasts.<br>
In various embodiments of the invention, the nucleic acid used to mutate an endogenous prion nucleic acid (e.g., a knockout cassette which includes a promoter operably linked to a nucleic acid encoding a selectable marker and operably linked to a nucleic acid having substantial sequence identity to a prion nucleic acid) is not contained in a viral vector, such as an adenoviral vector or<br>
an adeno-associated viral vector. For example, the nucleic acid may be contained in a plasmid or artificial chromosome that is inserted into an ungulate cell, using a standard method such as transfection or lipofection that does not involve viral infection of the cell. In yet another embodiment, the nucleic acid used to mutate an endogenous prion nucleic acid (e.g., a knockout cassette which includes a promoter operably linked to a nucleic acid encoding a selectable marker and operably linked to a nucleic acid having substantial sequence identity to a prion nucleic acid) is contained in a viral vector, such as an adenoviral vector or an adeno-associated viral vector. According to this embodiment, a virus containing the viral vector is used to infect an ungulate cell, resulting in the insertion of a portion or the entire viral vector into the ungulate cell.<br>
Methods for producing transgenic ungulates<br>
The invention also provides methods for producing transgenic ungulates with one or more mutations in endogenous prion nucleic acids. One such method involves inserting a cell of any of the above aspects of the invention, a chromatin mass from the cell, or a nucleus from the cell into an oocyte. The cell has a first mutation in an endogenous prion nucleic acid. The oocyte or an embryo formed from the oocyte is transferred into the uterus of a host ungulate under conditions that allow the oocyte or the embryo to develop into a fetus. Preferably, the fetus develops into a viable offspring.<br>
In preferred embodiments, the first mutation is introduced into the cell by inserting a nucleic acid comprising a cassette that includes a promoter operably linked to a nucleic acid encoding a selectable marker and operably linked to one or more nucleic acids having substantial sequence identity to an endogenous prion nucleic acid, whereby the cassette is integrated into one endogenous allele of a prion nucleic acid. In other preferred embodiments, the mutation is introduced in the cell by inserting into the cell a nucleic acid comprising a first cassette that includes a first promoter operably linked to a nucleic acid encoding a first selectable marker and operably linked to a first<br>
nucleic acid having substantial sequence identity to a endogenous prion nucleic acid, whereby the first cassette is integrated into a first endogenous allele of a prion nucleic acid producing a first transgenic cell. Into the first transgenic cell is inserted a nucleic acid that includes a second cassette that includes a second promoter operably linked to a nucleic acid encoding a second selectable marker and operably linked to a second nucleic acid having substantial sequence identity to a prion nucleic acid. The second selectable marker differs from the first selectable marker, and the second cassette is integrated into a second endogenous allele of a prion nucleic acid producing a second transgenic cell.<br>
	In still other preferred embodiments, a cell is isolated from the embryo,<br>
the fetus, or an offspring produced from the fetus, and another mutation is introduced into a prion nucleic acid or another nucleic acid (e.g., an antibody heavy or light chain nucleic acid) of the cell. A second round of nuclear transfer is then performed using the resulting cell, a chromatin mass from the cell, or a nucleus from the cell to produce a transgenic ungulate with two or more mutations. The mutations are in the same or different alleles of a gene or are in different genes. The cell used in the first or optional second round of nuclear transfer encodes a xenogenous antibody. In particular embodiments, the cell includes one or more nucleic acids encoding all or part of a xenogenous Ig gene that is capable of undergoing rearrangement and expressing one or more xenogenous Ig molecules in B-cells. In preferred embodiments, the cell that is mutated is a fibroblast (e.g., a fetal fibroblast). Preferably, the endogenous gene that is mutated is operably linked to an endogenous promoter that is not active in a fibroblast. In other preferred embodiments, the endogenous promoter operably linked to the endogenous gene that is mutated is less than 80, 70, 60, 50, 40, 30, 20, 10% as active as an endogenous promoter operably linked to a endogenous housekeeping gene such as GAPDH. Promoter activity may be measured using any standard assay, such as assays that measure the level of mRNA or protein encoded by the gene (see, for example, Ausubel et al. Current Protocols in Molecular Biology, volume 2, p.<br>
11.13.1-11.13.3, John Wiley &amp; Sons, 1995).   This method for generating a transgenic ungulate has the advantage of allowing a gene that is not expressed in the donor cell (i.e., the cell that is the source of the genetic material used for nuclear transfer) to be mutated.<br>
Preferably, the cell used in the production of the transgenic ungulate has a mutation in one or both alleles of an endogenous nucleic acid encoding alpha-(l,3)-galactosyltransferase and/or J chain. In other preferred embodiments, the cell has a nucleic acid encoding an exogenous J chain, such as a human J chain, or a nucleic acid encoding a xenogenous antibody. Preferably, the xenogenbus nucleic acid encodes-all orpart of a xenogenous Ig gene, and the gene is capable of undergoing rearrangement and expressing more than one xenogenous Ig molecule in B-cells. In other preferred embodiments, the antibody is a polyclonal antibody. In yet other preferred embodiments, the immunogloblulin chain or antibody is expressed in serum and/or milk.<br>
We have previously disclosed a variety of improved methods for cloning mammals (e.g., ungulates such as bovines) that may be used to clone mammals with one or more mutations in a prion nucleic acid (see, e.g., U.S. Patent Publication No. 2002-0046722 Al and PCT Publication No. WO02/051997). In some of these methods, a permeabilized cell is incubated with a reprogramming media (e.g., a cell extract) to allow the addition or removal of factors from the cell, and then the plasma membrane of the permeabilized cell is resealed to enclose the desired factors and restore the membrane integrity of the cell. Some of these methods also involve the condensation of a donor nucleus (e.g., an isolated nucleus or a nucleus within a donor cell) into a chromatin mass to allow the release of nuclear components such as transcription factors that may promote the transcription of genes that are undesirable for the development of the nuclear transplant embryo into a viable offspring. If desired, the steps of any of these methods may be repeated one or more times or different reprogramming methods may be performed<br>
sequentially to increase the extent of reprogramming, resulting in greater viability of the cloned fetuses.<br>
One method for generating a transgenic ungulate (e.g., bovine) involves incubating a permeabilized cell of any the above aspects of the invention (e.g., a cell that has one or more mutations in an endogenous prion nucleic acid) with a reprogramming media (e.g., a cell extract) under conditions that allow the removal of a factor (e.g., a nuclear or cytoplasmic component such as a transcription factor) from a nucleus, chromatin mass, or chromosome of the permeabilized cell or the addition of a factor to the nucleus, chromatin mass, or chromosome, thereby forming a reprogrammed cell. The reprogrammed cell is inserted into an enucleated oocyte, and the resulting oocyte or an embryo formed from the oocyte is transferred into the uterus of a host ungulate under conditions that allow the oocyte or embryo to develop into a fetus. In preferred embodiments, the permeabilized cell is contacted with one or more of the following under conditions that allow formation of a chromatin mass: a mitotic extract in the presence or absence of an anti-NuMA antibody, a detergent and/or salt solution, or a protein kinase solution. In yet another preferred embodiment, the permeabilized cell is incubated with an interphase reprogramming media (e.g., an interphase cell extract). In still another preferred embodiment, the nucleus in the permeabilized cell remains membrane-bounded, and the chromosomes in the nucleus do not condense during incubation with this interphase reprogramming media. In certain embodiments, incubating the permeabilized cell in the reprogramming media does not cause DNA replication or only causes DNA replication in less than 50, 40, 30, 20, 10, or 5% of the cells. In other embodiments, incubating the permeabilized cell in the reprogramming media causes DNA replication in at least 60, 70, 80, 90, 95, or 100% of the cells. In various embodiments, the permeabilized cell is formed by incubating an intact cell with a protease such as trypsin, a detergent, such as digitonin, or a bacterial toxin, such as Streptolysin O. In a preferred embodiment, the reprogrammed cell is not<br>
incubated under conditions that allow the membrane of the reprogrammed cell to reseal prior to insertion into the oocyte. In yet another embodiment, the reprogrammed cell is incubated under conditions that allow the membrane of the reprogrammed cell to reseal prior to insertion into the oocyte. In other preferred embodiments, the reconstituted oocyte or the resulting embryo expresses lamin A, lamin C, or NuMA protein at a level that is less than 5-fold greater than the corresponding level expressed by a control oocyte or a control embryo with the same number of cells and from the same species.<br>
In another aspect, the invention provides another method for generating a transgenic ungulate (e.g., bovine). This method involves (a) incubating a donor nucleus from a cell of the invention (e.g., a nucleus that has one or more mutations in an endogenous prion nucleic acid) under conditions that allow formation of a chromatin mass without causing DNA replication, (b) inserting the chromatin mass into an enucleated oocyte, thereby forming a nuclear transfer oocyte and (c) transferring the nuclear transfer oocyte or an embryo formed from the nuclear transfer oocyte into the uterus of a host ungulate under conditions that allow the nuclear transfer oocyte or embryo to develop into a fetus. In a preferred embodiment, the donor nucleus is incubated with a reprogramming media (e.g., a cell extract) under conditions that allow nuclear or cytoplasmic components such as transcription factors, represser proteins, or chromatin remodeling proteins to be added to, or removed from, the nucleus or resulting chromatin mass. Preferably, the donor nucleus is contacted with one or more of the following under conditions that allow formation of a chromatin mass: a mitotic extract in the presence or absence of an anti-NuMA antibody, a detergent and/or salt solution, or a protein kinase solution. In other preferred embodiments, the reconstituted oocyte or the resulting embryo expresses lamin A, lamin C, or NuMA protein at a level that is less than 5-fold greater than the corresponding level expressed by a control oocyte or a control embryo with the same number of cells and from the same species. Preferably, the nucleus has fewer than four sets of homologous chromosomes (i.e., has fewer than two<br>
pairs of complete chromatids).<br>
Preferred methods for generating chimeric ungulates<br>
Other preferred ungulates are chimeric ungulates produced using cells from two or more embryos. For example, cells from a nuclear transfer embryo (e.g., an embryo formed by inserting a cell, nucleus, or chromatin mass into an enucleated oocyte) can be combined with cells from an in vitro fertilized, naturally-occurring, or parthenogenetically-activated embryo. Preferably, the majority of the cells and their progeny from the nuclear transfer embryo are incorporated into fetal tissue of the resulting chimeric embryo. At least some of the cells and their progeny from the second embryo are preferably incorporated into placental tissue and promote the viability of the resulting chimeric embryo. In preferred embodiments, the nuclear transfer embryo has a mutation in an endogenous prion nucleic acid and has a nucleic acid encoding a xenogenous antibody.<br>
Accordingly, in one such aspect, the invention provides a method of producing a transgenic ungulate by inserting a cell, nucleus, or chromatin mass of the invention (e.g., a cell, nucleus, or chromatin mass having a mutation in an endogenous prion nucleic acid and optionally having one or more nucleic acids encoding a xenogenous antibody) into an oocyte, thereby forming a first embryo. One or more cells from the first embryo are contacted with one or more cells from a second embryo, thereby forming a third embryo. The second embryo is an in vitro fertilized embryo, naturally-occurring embryo, or parthenogenetically-activated embryo. The third embryo is transferred into the uterus of a host ungulate under conditions that allow the third embryo to develop into a fetus.<br>
In another related aspect, the invention provides yet another method of generating a transgenic ungulate (e.g., a bovine). This method involves incubating a permeabilized cell of the invention (e.g., a cell having a mutation in an endogenous prion nucleic acid and optionally having one or more nucleic<br>
acids encoding a xenogenous antibody) in a reprogramming media (e.g., cell extract) under conditions that allow the removal of a factor from a nucleus, chromatin mass, or chromosome of the permeabilized cell or the addition of a factor from the reprogramming media to the nucleus, chromatin mass, or chromosome, thereby forming a reprogrammed cell. The reprogrammed cell is inserted into an enucleated oocyte, thereby forming a first embryo. One or more cells from the first embryo are contacted with one or more cells from an in vitro fertilized, naturally-occurring, or parthenogenetically-activated second embryo, forming a third embryo. The third embryo is transferred into the uterus of a host ungulate under conditions that allow the third embryo to develop into a fetus.<br>
In a preferred embodiment, the permeabilized cell is incubated with a reprogramming media (e.g., a cell extract) under conditions that allow nuclear or cytoplasmic components such as transcription factors to be added to, or removed from, the nucleus or resulting chromatin mass. In other preferred embodiments, the permeabilized cell is contacted with one or more of the following under conditions that allow formation of a chromatin mass: a mitotic extract in the presence or absence of an anti-NuMA antibody, a detergent and/or salt solution, or a protein kinase solution. In yet another preferred embodiment, the permeabilized cell is incubated with an interphase reprogramming media (e.g., an interphase cell extract). In still another preferred embodiment, the nucleus in the permeabilized cell remains membrane-bounded, and the chromosomes in the nucleus do not condense during incubation with this interphase reprogramming media. In some embodiments, incubating the permeabilized cell in the reprogramming media does not cause DNA replication or only causes DNA replication in less than 50, 40, 30, 20, 10, or 5% of the cells. In other embodiments, incubating the permeabilized cell in the reprogramming media causes DNA replication in at least 60, 70, 80, 90, 95, or 100% of the cells. In various embodiments, the permeabilized cell is formed by incubating an intact cell with a protease such<br>
as trypsin, a detergent, such as digitonin, or a bacterial toxin, such as Streptolysin 0. In a preferred embodiment, the reprogrammed cell is not incubated under conditions that allow the membrane of the reprogrammed cell to reseal prior to insertion into the oocyte. In yet another embodiment, the reprogrammed cell is incubated under conditions that allow the membrane of the reprogrammed cell to reseal prior to insertion into the oocyte.<br>
In still other embodiments, the ungulate is generated by contacting a donor nucleus from a cell of the invention (e.g., a nucleus having a mutation in an endogenous prion nucleic acid and optionally having one or more nucleic acids encoding a xenogenous antibody) with a reprogramming media (e.g., cell extract) under conditions that allow formation of a chromatin mass, and inserting the chromatin mass into an enucleated oocyte, thereby forming a first embryo. One or more cells from the first embryo are contacted with one or more cells from an in vitro fertilized, naturally-occurring, or parthenogenetically activated second embryo, forming a third embryo. The third embryo is transferred into the uterus of a host ungulate under conditions that allow the third embryo to develop into a fetus. In a preferred embodiment, the chromatin mass is formed by contacting a donor nucleus that has less than four sets of homologous chromosomes with a reprogramming media under conditions that allow formation of a chromatin mass without causing DNA replication. Preferably, the donor nucleus is contacted with one or more of the following under conditions that allow formation of a chromatin mass: a mitotic extract in the presence or absence of an anti-NuMA antibody, a detergent and/or salt solution, or a protein kinase solution.<br>
In preferred embodiments of any of the above aspects involving ungulates produced using cells from two embryos, at least one of the first embryo and the second embryo is a compaction embryo. In another embodiment, the first embryo and the second embryo are at different cell-stages. The first embryo and the donor cell used to produce the second embryo can be from the same species or from different genuses or species. Preferably,<br>
at least 10, 20, 30, 40, 50, 60, 70, 80, 90, 95, or 100% cells in the trophectoderm or placental tissue of the fetus are derived from the second embryo, or at least 30, 40, 50, 60, 70, 80, 90, 95, or 100% cells in the inner cell mass or fetal tissue of the fetus are derived from the first embryo. In other preferred embodiments, the first embryo or the third embryo expresses lamin A, lamin C, or NuMA protein at a level that is less than 5-fold greater than the corresponding level expressed by a control embryo with the same number of cells and from the same species.<br>
In other preferred embodiments of any of the above aspects involving ungulates produced using cells from two embryos, part or all of the zona pellucida of the first embryo or second embryo is removed before the cells from each embryo are contacted. In one embodiment, the cells from the first and second embryos are contacted by being placed adjacent to each other in solution or on a solid support. In another embodiment, standard techniques are used to inject cells from the first embryo into the second embryo. The cells can be injected into any region of the second embryo, such as the periphery of the embryo between the zona pellucida and the embryo itself. Naturally occurring embryos include embryos that are surgically or nonsurgically removed from a pregnant ungulate (e.g., a bovine) using standard methods. In vitro fertilized embryos include intra-cytoplasmic sperm injection embryos generated using standard methods. It is also contemplated that cells from more than two embryos (e.g., cells from three, four, five, six, or more embryos) can be combined to form a chimeric embryo for generation of a cloned ungulate.<br>
Preferred embodiments for generating ungulates<br>
In preferred embodiments of any of the above aspects, the reprogramming media (e.g., a cell extract) is modified by the enrichment or depletion of a factor, such as a DNA methyltransferase, histone deacetylase, histone, protamine, nuclear lamin, transcription factor, activator, or repressor. In other preferred embodiments, the level of expression of NuMA or AKAP95<br>
protein in the oocyte or chimeric embryo is at least 2-, 5-, 10-, or 20-fold greater in the nucleus than in the cytoplasm. In yet other embodiments, at least 30, 40, 50, 60, 70, 80, 90, or 100% of the AKAP95 protein in the oocyte or chimeric embryo is extracted with a solution of 0.1% Triton X-100, 1 mg/ml DNase I, and either 100 mM or 300 mM NaCl. Preferably, the chromatin mass is purified from the reprogramming media (e.g., extract) prior to insertion into the enucleated oocyte. In another preferred embodiment, inserting the chromatin mass into the enucleated oocyte involves contacting the chromatin mass and the oocyte with a fusigenic compound under conditions that allow the chromatin mass to enter the ooctye. In yet another preferred embodiment, the fetus develops into a viable offspring. Preferably, at least 1, 3, 5, 10, 20, 30, 40, 50, 60, 70, 80, or 90% of the nuclear transfer oocytes or embryos develop into viable offspring. In this method, the oocyte containing the chromatin mass or reprogrammed cell may be cultured under conditions that allow cell division and one of the resulting cells may be recloned one or more times. The donor nucleus, donor chromatin mass, or donor cell and the oocyte used in the method may be from the same species, or they may be from different species or genuses. Desirably, the donor cell or resulting transgenic ungulate expresses at least 10, 20, 40, 60, 80, 90, 95, or 100% less functional or total protein than a corresponding, naturally-occurring cell or ungulate. The oocyte may be fertilized or unfertilized. Preferably the donor nucleus, chromatin mass, or permeabilized cell is from a G] or GO phase cell. In addition, the genomic DNA of the cloned embryo, fetus, or ungulate is preferably substantially identical to that of the donor cell. It is also contemplated that the chromatin mass or reprogrammed cell may be inserted into an embryo for the production of a chimeric embryo, fetus, or ungulate containing a mixture of cells with DNA substantially identical to that of the chromatin mass or reprogrammed cell and cells with DNA substantially identical to that of the naturally-occurring cells in the embryo. It is also contemplated that a nucleated oocyte may be used in the methods of the invention.<br>
The reprogramming media used in any of the aspects of the invention may or may not contain exogenous nucleotides. In other preferred embodiments, a chromatin mass in a reprogramming media or formed in a permeabilized cell is contacted with a vector having a nucleic acid encoding a gene of interest under conditions that allow random integration or homologous recombination between the nucleic acid in the vector and the corresponding nucleic acid in the genome of the chromatin mass, resulting in the alteration of the genome of the chromatin mass. Due to the lack of an intact plasma membrane and the lack of a nuclear membrane, a chromatin mass in a permeabilized cell or in solution may be easier to genetically modify than a naturally-occurring cell. Examples of cells that may be used to generate reprogramming extracts include embryonic stem cells and adult stem cells from brain, blood, bone marrow, pancreas, liver, skin, or any other organ or tissue. Other exemplary reprogramming cell extracts include oocyte extracts (e.g., bovine or sea urchin oocyte extracts) and male germ cell extracts (e.g., spermatogonia, spermatocyte, spermatid, or sperm extracts from vertebrates, invertebrates, or mammals such as bovine). The donor or permeabilized cell can be non-immortalized or naturally, spontaneously, or genetically immortalized. The donor cell, permeabilized cell, recipient cell, or cytoplast can be from a source of any age, such as an embryo, fetus, youth, or adult mammal (e.g., ungulate). Cells from younger sources may have acquired fewer spontaneous mutations and may have a longer lifespan after insertion into an oocyte.<br>
Methods for breeding ungulates<br>
In preferred embodiments of any of the above methods for generating ungulates or ungulate cells, an ungulate of the invention is mated with another ungulate (e.g., an ungulate with a mutation in an antibody heavy or light chain nucleic acid or an ungulate with a nucleic acid encoding a xenogenous antibody) to produce a fetus or live offspring with two or more genetic<br>
modifications. Preferably, one or more cells are isolated from the fetus or offspring, and one or more additional genetic modifications are introduced into the isolated cell(s).<br>
Methods for producing antibodies<br>
The invention also provides method for producing antibodies using an ungulate of the invention that expresses xenogenous antibodies (e.g., human antibodies). One such method involves administering one or more antigens of interest to an ungulate of the invention having nucleic acid encoding a xenogenous antibody gene locus. The nucleic acid segments in the gene locus undergo rearrangement resulting in the production of antibodies specific for the antigen. Antibodies are recovered from the ungulate. The antibodies may be monoclonal or polyclonal and are preferably reactive with an antigen of interest. Preferably, the antibodies are recovered from the serum or milk of the ungulate.<br>
In a related aspect, the invention provides another method for producing antibodies that involves recovering xenogenous antibodies from an ungulate of the invention having nucleic acid encoding a xenogenous antibody gene locus. The nucleic acid segments in the gene locus undergo rearrangement resulting in the production of xenogenous antibodies. The antibodies may be monoclonal or polyclonal and are preferably reactive with an antigen of interest. Preferably, the antibodies are recovered from the serum or milk of the ungulate.<br>
Nucleic acids<br>
The invention also provides nucleic acids (e.g., vectors) that are useful for mutating a prion gene in ungulates (e.g., bovines). One such nucleic acid has a cassette which includes, in 5'to 3'order, a first region of homology having substantial sequence identity to a first region of an endogenous prion nucleic acid of an ungulate cell, a positive selection marker, and a second region of homology having substantial sequence identity to a second region of<br>
the priori nucleic acid. In one embodiment, the first region of homology (e.g., a region of less than 3, 2, 1.5, or 1 kilobases) is shorter than the second region of homology (e.g., a region of at least 7, 8, 9, or 10 kilobases), and the cassette is capable of integrating into an endogenous prion nucleic acid of the cell. In another embodiment, the first region of homology is longer than the second region of homology. Preferably, the first and/or second region of homology is isogenic to the corresponding region of an endogenous prion nuleic acid in a cell to be mutated.<br>
In a related aspect, the invention provides a nucleic acid that has a cassette which includes a first region of homology having substantial sequence identity to a first region of an endogenous prion nucleic acid of an ungulate cell, a positive selection marker, a second region of homology having substantial sequence identity to a second region of the prion nucleic acid, and a negative selection marker (e.g., DT-A or Tk) in the opposite orientation as the positive selection marker (e.g., a negative selection marker for which the promoter is linked to the end of the shorter region of homology). The cassette is capable of integrating into an endogenous prion nucleic acid of the cell. Desirably, one region of homology is at least 7, 8, 9, or 10 kilobases in length, and the other region of homology is less than 3, 2, 1.5, or 1 kilobases in length. In some embodiments, part or all of the negative selection marker is within the shorter region of homology. Preferably, the first and/or second region of homology is isogenic to the corresponding region of an endogenous prion nuleic acid in a cell to be mutated.<br>
In another aspect, the invention provides a composition that includes a nucleic acid of the invention and spermidine (e.g., 0.1 to 10 mM spermidine).<br>
Preferred embodiments of above aspects<br>
Preferably, the ungulate antiserum or milk has polyclonal human immunoglobulins. Preferably, the antiserum or milk is from a bovine, ovine, porcine, or caprine. In another preferred embodiment, the Igs are directed against a desired antigen. In preferred embodiments, the antiserum is used as intravenous immunoglobulin (IVIG) for the treatment or prevention of disease in humans. In another preferred embodiment, an antigen of interest is administered to the ungulate, and Igs directed against the antigen are produced by the ungulate. Preferably, the nucleic acid segments in the xenogenous immunoglobulin gene locus rearrange, and xenogenous antibodies reactive with the antigen of interest are produced. Preferably, the antiserum and/or milk contains at least 2, 5, 10, 20, or 50 fold more xenogenous antibody than endogenous antibody, or contains no endogenous antibody. If desired, hybridomas and monoclonal antibodies can be produced using xenogenous B-cells derived from the above-described transgenic ungulates (for example, transgenic bovines). It is also contemplated that xenogenous antibodies (e.g., human antibodies) isolated from ungulates may be subsequently chemically modified so that they are covalently linked to a toxin, therapeutically active compound, enzyme, cytokine, radiolabel, fluorescent label, or affinity tag.   If desired, the fluorescent or radiolabel may be used for imaging of the antibody in vitro or in vivo.<br>
Preferably, the mutation in a prion nucleic acid decreases the generation of the infectious form of prion protein. In preferred embodiments, the amount of the infectious form of prion protein produced by the ungulate is less than 80, 70, 60, 50, 40, 30, 20, 10, or 5% of that produced by a control ungulate without the mutation. Preferably, no infections prion protein is produced.<br>
In preferred embodiments, the mutation is the insertion of part or all of a prion nucleic acid (e.g., the coding region) from another animal (e.g., a different ungulate species or breed). Preferably, this xenogenous prion nucleic acid replaces part or all of an endogenous prion nucleic acid. In preferred<br>
embodiments, the xenogenous prion nucleic acid is from a species or breed with increased resistance to a prion infection, such as sheep that are resistant to scrapie. Preferably, the percentage of ungulates with the mutation that have a prion infection is less than 80, 70, 60, 50, 40, 30, 20, 10, or 5% of the percentage of control ungulates without the mutation that have a prion infection.<br>
Preferred ungulates and donor cells<br>
Ungulates include members of the orders Perissodactyla and Artiodaetyla, such as any member of the genus Bos. Other preferred ungulates include sheep, big-horn sheep, goats, buffalos, antelopes, oxen, horses, donkeys, mule, deer, elk, caribou, water buffalo, camels, llama, alpaca, pigs, and elephants.<br>
Preferred donor cells include differentiated cells such as epithelial cells, neural cells, epidermal cells, keratinocytes, hematopoietic cells, melanocytes, chondrocytes, B-lymphocytes, T-lymphocytes, erythrocytes, macrophages, monocytes, fibroblasts, and muscle cells; and undifferentiated cells such as embryonic cells (e.g., stem cells and embryonic germ cells). In another preferred embodiment, the cell is from the female reproductive system, such as a mammary gland, ovarian cumulus, granulosa, or oviductal cell. Other preferred cells include fetal cells and placental cells. Preferred cells also include those from any organ, such as the bladder, brain, esophagus, fallopian tube, heart, intestines, gallbladder, kidney, liver, lung, ovaries, pancreas, prostate, spinal cord, spleen, stomach, testes, thymus, thyroid, trachea, ureter, urethra, and uterus. Preferably, the donor cell, donor nucleus, donor chromatine mass, or reconstituted oocyte is not tetraploid.<br>
In yet another preferred embodiment, the nucleus, permeabilized cell, or chromosomes are from a transgenic cell or ungulate or contain a mutation not found in the donor cell or not found in a naturally-occurring cell. Preferred transgenic donor nuclei and donor cells encode proteins that confer improved<br>
resistance to disease or parasites in the cloned ungulate. Alternatively, the donor nuclei or donor cells may be engineered so that the cloned ungulate produces a recombinant product, such as the production of a human protein in the urine, blood, or milk of a bovine. For example, proteins may be expressed in the urine of cattle by inserting a polynucleotide sequence encoding a human protein under the control of an uroplakin promoter. Therapeutic proteins that may be produced in the milk of cloned bovines include human clotting factors such as any of factors I to XIII (Voet and Voet, Biochemistry, John Wiley &amp; Sons, New York, 1990). These heterologous proteins may be expressed under the control of a prolactin promoter or any other promoter suitable for expression in the milk of a bovine. Recombinant proteins from these or other tissues or fluids may be purified using standard purification methods (see, for example, Ausubel et al, supra).<br>
Definitions<br>
As used herein, by "artificial chromosome" is meant a mammalian chromosome or fragment thereof which has an artificial modification such as the addition of a selectable marker, the addition of a cloning site, the deletion of one or more nucleotides, the substitution of one or more nucleotides, and the like. By "human artificial chromosome" or "HAC" is meant an artificial chromosome generated from one or more human chromosome(s). An artificial chromosome can be maintained in the host cell independently from the endogenous chromosomes of the host cell. In this case, the HAC can stably replicate and segregate along side endogenous chromosomes. Alternatively, it may be translocated to, or inserted into, an endogenous chromosome of the host cell. Two or more artificial chromosomes can be introduced to the host cell simultaneously or sequentially. For example, artificial chromosomes derived from human chromosome #14 (comprising the Ig heavy chain gene), human chromosome #2 (comprising the Ig kappa chain gene), and human chromosome #22 (comprising the Ig lambda chain gene) can be introduced. Alternatively,<br>
an artificial chromosome(s) comprising both a xenogenous Ig heavy chain gene and Ig light chain gene, such as AHAC or AAHAC, may be introduced. Preferably, the heavy chain loci and the light chain loci are on different chromosome arms (i.e., on different side of the centromere). In still other preferred embodiments, the total size of the HAC is less than or equal to approximately 10, 9, 8, or 7 megabases.<br>
By "a nucleic acid in its pre-arranged or unrearranged form" is meant a nucleic acid that has not undergone V(D)J recombination. In preferred embodiments, all of the nucleic acid segments encoding a V gene segment of an antibody light chain are separated from all of the nucleic acid segments encoding a J gene segment by one or more nucleotides. Preferably, all of the nucleic acid segments encoding a V gene segment of an antibody heavy chain are separated from all of the nucleic acid segments encoding a D gene segment by one or more nucleotides, and/or all of the nucleic acid segments encoding a D gene segment of an antibody heavy chain are separated from all of the nucleic acid segments encoding a J gene segment by one or more nucleotides. Preferably, a nucleic acid in its unrearranged form is substantially human. In other preferred embodiments, the nucleic acid is at least 70, 80, 90, 95, or 99% identical to the corresponding region of a naturally-occurring nucleic acid from a human.<br>
By "reducing the quantity and/or activity of endogenous antibody" is meant reducing the amount of endogenous, functional antibodies produced by a B-cell or a population of B-cells, as described, for example, in U.S.S.N. 10/441,503, filed May 19, 2003. This reduction in the amount of endogenous antibodies may be due to a decrease in the amount of endogenous antibodies produced per B-cell, a decrease in the number of functional endogenous B-cells, or a combination thereof. Preferably, the amount of an endogenous antibody secreted by a B-cell or expressed on the surface of a B-cell expressing or secreting endogenous antibody is reduced by at least 25, 50, 75, 90, or 95%. In another preferred embodiment, the number of endogenous B-cells in a<br>
sample from the recipient mammal, such as a blood sample, is reduced by at least 25, 50, 75, 90, or 95%.<br>
By "bifunctional antibody" is meant an antibody that includes an antibody or a fragment of an antibody covalently linked to a different antibody or a different fragment of an antibody. In one preferred embodiment, both antibodies or fragments bind to different epitopes expressed on the same antigen. Other preferred bifunctional antibodies bind to two different antigens, such as to both an antibody light chain and an antibody heavy chain. Standard molecular biology techniques such as those described herein may be used to operably link two nucleic acids so that the fusion nucleic acid encodes a bifunctional antibody.<br>
By "fragment" is meant a polypeptide having a region of consecutive amino acids that is identical to the corresponding region of an antibody of the invention but is less than the full-length sequence. The fragment has the ability to bind the same antigen as the corresponding antibody based on standard assays, such as those described herein. Preferably, the binding of the fragment to the antigen is at least 20, 40, 60, 80, or 90% of that of the corresponding antibody.<br>
By "purified" is meant separated from other components that naturally accompany it. Typically, a factor is substantially pure when it is at least 50%, by weight, free from proteins, antibodies, and naturally-occurring organic molecules with which it is naturally associated. Preferably, the factor is at least 75%, more preferably, at least 90%, and most preferably, at least 99%, by weight, pure. A substantially pure factor may be obtained by chemical synthesis, separation of the factor from natural sources, or production of the factor in a recombinant host cell that does not naturally produce the factor. Proteins, vesicles, and organelles may be purified by one skilled in the art using standard techniques such as those described by Ausubel et al. (supra). The factor is preferably at least 2, 5, or 10 times as pure as the starting material, as measured using polyacrylamide gel electrophoresis, column chromatography,<br>
optical density, HPLC analysis, or western blot analysis (Ausubel et at, supra). Preferred methods of purification include immunoprecipitation, column chromatography such as immunoaffmity chromatography, magnetic bead immunoaffmity purification, and panning with a plate-bound antibody.<br>
By "chromatin mass" is meant more than one chromosome not enclosed by a membrane. Preferably, the chromatin mass contains all of the chromosomes of a cell. An artificially induced chromatin mass containing condensed chromosomes may be formed by exposure of a nucleus to a mitotic reprogramming media (e.g., a mitotic extract) as described herein. Alternatively, an artificially induced chromatin mass containing decondensed or partially condensed chromosomes may be generated by exposure of a nucleus to one of the following, as described herein: a mitotic extract containing an anti-NuMA antibody, a detergent and/or salt solution, or a protein kinase solution. A chromatin mass may contain discrete chromosomes that are not physically touching each other or may contain two or more chromosomes that are in physical contact.<br>
If desired, the level of chromosome condensation may be determined using standard methods by measuring the intensity of staining with the DNA stain, DAPI. As chromosomes condense, this staining intensity increases. Thus, the staining intensity of the chromosomes may be compared to the staining intensity for decondensed chromosomes in interphase (designated 0% condensed) and maximally condensed chromosomes in mitosis (designated 100% condensed). Based on this comparison, the percent of maximal condensation may be determined. Preferred condensed chromatin masses are at least 50, 60, 70, 80, 90, or 100% condensed. Preferred decondensed or partially condensed chromatin masses are less than 50, 40, 30, 20, or 10% condensed.<br>
By "nucleus" is meant a membrane-bounded organelle containing most or all of the DNA of a cell. The DNA is packaged into chromosomes in a<br>
decondensed form. Preferably, the membrane-encapsulating the DNA includes one or two lipid bilayers or has nucleoporins.<br>
By "nucleus that has less than four sets of homologous chromosomes" is meant a nucleus that has a DNA content of less than 4n, where "n" is the number of chromosomes found in the normal haploid chromosome set of a mammal of a particular genus or species. Such a nucleus does not have four copies of each gene or genetic locus. Preferably, the nucleus is diploid and thus has two sets of homologous chromosomes but has less than two complete pairs of chromatids.<br>
     By "pronucleus" is meant a haploid nucleus resulting from meiosis or a nuclear transfer pronucleus. The female pronucleus is the nucleus of the oocyte or ovum before fusion with the male pronucleus. The male pronucleus is the sperm nucleus after it has entered the oocyte or ovum at fertilization but before fusion with the female pronucleus. A nuclear transfer pronucleus is a pronucleus (e.g., a diploid pronucleus) that forms after introduction of a donor cell, nucleus, or chromatin mass into an oocyte. The nuclear transfer pronucleus has less than four sets of homologous chromosomes.<br>
By "donor cell" is meant a cell from which a nucleus or chromatin mass is derived, or a permeabilized cell.<br>
By "permeabilization" is meant the formation of pores in the plasma membrane or the partial or complete removal of the plasma membrane.<br>
By "reprogramming media" is meant a solution that allows the removal of a factor from a cell, nucleus, chromatin mass, or chromosome or the addition of a factor from the solution to the cell, nucleus, chromatin mass, or chromosome. Preferably, the addition or removal of a factor increases or decreases the level of expression of an mRNA or protein in the donor cell, chromatin mass, or nucleus or in a cell containing the reprogrammed chromatin mass or nucleus. In another embodiment, incubating a permeabilized cell, chromatin mass, or nucleus in the reprogramming media alters a phenotype of the permeabilized cell or a cell containing the reprogrammed chromatin mass<br>
or nucleus relative to the phenotype of the donor cell. In yet another embodiment, incubating a permeabilized cell, chromatin mass, or nucleus in the reprogramming media causes the permeabilized cell or a cell containing the reprogrammed chromatin mass or nucleus to gain or lose an activity relative to the donor cell.<br>
Exemplary reprogramming media include solutions, such as buffers, that do not contain biological molecules such as proteins or nucleic acids. Such solutions are useful for the removal of one or more factors from a nucleus, chromatin mass, or chromosome. Other preferred reprogramming medias are extracts, such as cellular extracts from cell nuclei, cell cytoplasm, or a combination thereof. Exemplary cell extracts include extracts from oocytes (e.g., mammalian, vertebrate, or invertebrate oocytes), male germ cells (mammalian, vertebrate, or invertebrate germ cells such as spermatogonia, spermatocyte, spermatid, or sperm), and stem cells (e.g., adult or embryonic stem cells). Yet other reprogramming media are solutions or extracts to which one or more naturally-occurring or recombinant factors (e.g., nucleic acids or proteins such as DNA methyltransferases, histone deacetylases, histones, protamines, nuclear lamins, transcription factors, activators, repressors, chromatin remodeling proteins, growth factors, interleukins, cytokines, or other hormones) have been added, or extracts from which one or more factors have been removed. Still other reprogramming media include solutions of detergent (e.g., 0.01% to 0.1%, 0.1% to 0.5%, or 0.5% to 2% ionic or non-ionic detergent such as one or more of the following detergents: SDS, Triton X-100, Triton X-114, CHAPS, Na-deoxycholate, n-octyl glucoside, Nonidet P40, IGEPAL, Tween 20, Tween 40, or Tween 80), salt (e.g., -0.1, 0.15, 0.25, 0.5, 0.75, 1, 1.5, or 2 M NaCl or KC1), polyamine (e.g., ~1 uM, 10 pM, 100 pM, 1 mM or 10 mM spermine, spermidine, protamine, or poly-L-lysine), a protein kinase (e.g., cyclin-dependent kinase 1, protein kinase C, protein kinase A, MAP kinase, calcium/calmodulin-dependent kinase, CK1 casein kinase, or CK2 casein kinase), and/or a phosphatase inhibitor (e.g., ~10 pM, 100 fiM, 1 mM,<br>
10 mM, 50 mM, 100 mM of one or more of the following inhibitors: Na-orthovanadate, Na-pyrophosphate, Na-fluoride, NIPP1, inhibitor 2, PNUTS, SDS22, AKAP149, or ocadaic acid). In some embodiments, the reprogramming medium contains an anti-NuMA antibody. If desired, multiple reprogramming media may be used simultaneously or sequentially to reprogram a donor cell, nucleus, or chromatin mass.<br>
By "interphase reprogramming media" is meant a media (e.g., an interphase cell extract) that induces chromatin decondensation and nuclear envelope formation.<br>
By "mitotic reprogramming media" is meant a media (e.g., a mitotic cell extract) that induces chromatin condensation and nuclear envelope breakdown.<br>
By "reprogrammed cell" is meant a cell that has been exposed to a reprogramming media. Preferably, at least 1, 5, 10, 15, 20, 25, 50, 75, 100, 150, 200, 300, or more mRNA or protein molecules are expressed in the reprogrammed cell that are not expressed in the donor or permeabilized cell. In another preferred embodiment, the number of mRNA or protein molecules that are expressed in the reprogrammed cell, but not expressed in the donor or permeabilized cell, is between 1 and 5, 5 and 10, 10 and 25, 25 and 50, 50 and 75, 75 and 100, 100 and 150, 150 and 200, or 200 and 300, inclusive. Preferably, at least 1, 5, 10, 15, 20, 25, 50, 75, 100, 150, 200, 300, or more mRNA or protein molecules are expressed in the donor or permeabilized cell that are not expressed in the reprogrammed cell. In yet another preferred embodiment, the number of mRNA or protein molecules that are expressed in the donor or permeabilized cell, but not expressed in the reprogrammed cell, is between 1 and 5, 5 and 10, 10 and 25, 25 and 50, 50 and 75, 75 and 100, 100 and 150, 150 and 200, or 200 and 300, inclusive. In still another preferred embodiment, these mRNA or protein molecules are expressed in both the donor cell (i.e., the donor or permeabilized starting cell) and the reprogrammed cell, but the expression levels in these cells differ by at least 2, 5, 10, or 20-fold, as measured using standard assays (see, for example, Ausubel et al., supra).<br>
By "addition of a factor" is meant the binding of a factor to chromatin, a chromosome, or a component of the nuclear envelope, such as the nuclear membrane or nuclear matrix. Alternatively, the factor is imported into the nucleus so that it is bounded or encapsulated by the nuclear envelope. Preferably, the amount of factor that is bound to a chromosome or located in the nucleus increases by at least 25, 50, 75, 100, 200, or 500%.<br>
By "removal of a factor" is meant the dissociation of a factor from chromatin, a chromosome, or a component of the nuclear envelope, such as the nuclear membrane or nuclear matrix. Alternatively, the factor is exported out of the nucleus so that it is no longer bounded or encapsulated by the nuclear envelope. Preferably, the amount of factor that is bound to a chromosome or located in the nucleus decreases by at least 25, 50, 75, 100, 200, or 500%.<br>
By "enrichment or depletion of a factor" is meant the addition or removal of a naturally-occurring or recombinant factor by at least 20, 40, 60, 80, or 100% of the amount of the factor originally present in an reprogramming media (e.g., a cell extract). Alternatively, a naturally-occurring or recombinant factor that is not naturally present in the reprogramming media may be added. Preferred factors include proteins such as DNA methyltransferases, histone deacetylases, histones, protamines, nuclear lamins, transcription factors, activators, and repressers; membrane vesicles, and organelles. In one preferred embodiment, the factor is purified prior to being added to the reprogramming media, as described below. Alternatively, one of the purification methods described below may be used to remove an undesired factor from the reprogramming media.<br>
By "recloned" is meant used in a second round of cloning. In particular, a cell from an embryo, fetus, or adult generated from the methods of the invention may be incubated in a mitotic reprogramming media (e.g., a mitotic cell extract) to form a chromatin mass for insertion into an enucleated oocyte, as described above. Alternatively, the cell may be permeabilized, incubated in a reprogramming media, and inserted into an enucleated oocyte, as described<br>
above. Performing two or more rounds of cloning may result in additional reprogramming of the donor chromatin mass or donor cell, thereby increasing the chance of generating a viable offspring after the last round of cloning.<br>
By "viable offspring" is meant a mammal that survives ex utero. Preferably, the mammal is alive for at least one second, one minute, one hour, one day, one week, one month, six months, or one year from the time it exits the maternal host. The mammal does not require the circulatory system of an in utero environment for survival.<br>
By "nuclear transfer oocyte" or "nuclear transplant oocyte" is meant an oocyte in which a donor cell, nucleus, or chromatin mass is inserted or fused. An embryo formed from the oocyte is referred to as a "nuclear transfer" or "nuclear transplant" embryo.<br>
By "embryo" or "embryonic" is meant a developing cell mass that has not implanted into the uterine membrane of a maternal host. Hence, the term "embryo" may refer to a fertilized oocyte; an oocyte containing a donor chromatin mass, nucleus, or reprogrammed cell; a pre-blastocyst stage developing cell mass; or any other developing cell mass that is at a stage of development prior to implantation into the uterine membrane of a maternal host and prior to formation of a genital ridge. An embryo may represent multiple stages of cell development. For example, a one cell embryo can be referred to as a zygote; a solid spherical mass of cells resulting from a cleaved embryo can be referred to as a morula, and an embryo having a blastocoel can be referred to as a blastocyst. An "embryonic cell" is a cell isolated from or contained in an embryo.<br>
By "cells derived from an embryo" is meant cells that result from the cell division of cells in the embryo.<br>
By "chimeric embryo" is meant an embryo formed from cells from two or more embryos. The resulting fetus or offspring can have cells that are derived from only one of the initial embryos or cells derived from more than one of the initial embryos. If desired, the percentage of cells from each embryo<br>
that are incorporated into the placenta! tissue and into the fetal tissue can be determined using standard FISH analysis or analysis of a membrane dye added to one embryo.<br>
By "chimeric ungulate" is meant an ungulate formed from cells from two or more embryos. The ungulate can have cells that are derived from only one of the initial embryos or cells derived from more than one of the initial embryos. If desired, the percentage of cells from each embryo that are incorporated into the placental tissue and into the fetal tissue can be determined using standard FISH analysis or analysis of a membrane dye added to one embryo.<br>
By "precompaction embryo" is meant an embryo prior to compaction. A precompaction embryo expresses essentially no E-cadherin on the surface of its blastomereres. Preferred precompaction embryos express at least 3, 5, 10, 20, 30, or 40-fold less E-cadherin than a fully compacted embryo of the same species, or express no E-adherin.<br>
By "compaction embryo" is meant an embryo undergoing compaction or following compaction. The blastomeres of a compaction embryo express E-cadherin on their surface. This E-cadherin expression can be measuring using standard methods with an anti- E-cadherin antibody. E-cadherin increases the adherence between blastomeres. Preferred compaction embryos include embryos in which the compaction process is completed. Other preferred compaction embryos express at least 3, 5, 10, 20, 30, or 40-fold more E-cadherin than a precompaction embryo of the same species.<br>
By "fetus" is meant a developing cell mass that has implanted into the uterine membrane of a maternal host. A fetus may have defining features such as a genital ridge that is easily identified by a person of ordinary skill in the art. A "fetal cell" is any cell isolated from or contained in a fetus.<br>
By "parthenogenesis" or "parthenogenetic activation" is meant development of an oocyte or ovum without fusion of its nucleus with a male<br>
pronucleus to form a zygote. For example, an oocyte can be induced to divide without fertilization.<br>
By "zona pellucida" is meant a translucent, elastic, noncellular layer surrounding the oocyte or ovum of many mammals.<br>
By "trophectoderm" is meant the outermost layer of cells surrounding the blastocoel during the blastocyst stage of mammalian embryonic development. Trophectoderm gives rise to most or all of the placental tissue upon further development.<br>
By "inner cell mass" is meant the cells surrounded by the trophectoderm. The inner cell mass cells give rise to most of the fetal tissues upon further development.<br>
By "mRNA or protein specific for one cell type" is meant an mRNA or protein that is expressed in one cell type at a level that is at least 10, 20, 50, 75, or 100 fold greater than the expression level in all other cell types. Preferably, the mRNA or protein is only expressed in one cell type.<br>
By "mutation" is meant an alteration in a naturally-occurring or reference nucleic acid sequence, such as an insertion, deletion, frameshift mutation, silent mutation, nonsense mutation, or missense mutation. Preferably, the amino acid sequence encoded by the nucleic acid sequence has at least one amino acid alteration from a naturally-occurring sequence. Examples of recombinant DNA techniques for altering the genomic sequence of a cell, embryo, fetus, or mammal include inserting a DNA sequence from another organism (e.g., a human) into the genome, deleting one or more DNA sequences, and introducing one or more base mutations (e.g., site-directed or random mutations) into a target DNA sequence. Examples of methods for producing these modifications include retroviral insertion, artificial chromosome techniques, gene insertion, random insertion with tissue specific promoters, homologous recombination, gene targeting, transposable elements, and any other method for introducing foreign DNA. All of these techniques are well known to those skilled in the art of molecular biology (see, for example,<br>
Ausubel et al, supra). Chromatin masses, chromosomes, and nuclei from transgenic cells containing modified DNA or donor transgenic cells may be used in the methods of the invention.<br>
By "immortalized" is meant capable of undergoing at least 25, 50, 75, 90, or 95% more cell divisions than a naturally-occurring control cell of the same cell type, genus, and species as the immortalized cell or than the donor cell from which the immortalized cell was derived. Preferably, an immortalized cell is capable of undergoing at least 2-, 5-, 10-, or 20-fold more cell divisions than the control cell. More preferably, the immortalized cell is capable of undergoing an unlimited number of cell divisions. Immortalized cells include cells that naturally acquire a mutation in vivo or in vitro that alters their normal growth-regulating process. Still other preferred immortalized cells include cells that have been genetically modified to express an oncogene, such as ras, myc, abl, bc!2, or neu, or that have been infected with a transforming DNA or RNA virus, such as Epstein Barr virus or SV40 virus (Kumar et al., Immunol. Lett. 65:153-159, 1999; Knight et al, Proc. Nat. Acad. Sci. USA 85:3130-3134, 1988; Shammah et al, J. Immunol. Methods 160-19-25, 1993; Gustafsson and Hinkula, Hum. Antibodies Hybridomas 5:98-104, 1994; Kataoka et al, Differentiation 62:201-211, 1997; Chatelut et al, Scand. J. Immunol. 48:659-666, 1998). Cells can also be genetically modified to express the telomerase gene (Roques et al, Cancer Res. 61:8405-8507, 2001).<br>
By "non-immortalized" is meant not immortalized as described above.<br>
By "fusigenic compound" is meant a compound that increases the probability that a chromatin mass or nucleus is inserted into a recipient cell when located adjacent to the cell. For example, the fusigenic compound may increase the affinity of a chromatin mass or a nucleus for the plasma membrane of a cell. The fusigenic compound may also promote the joining of the nuclear membrane of a nucleus with the plasma membrane of a cell.<br>
By "substantially identical" is meant having a sequence that is at least 60, 70, 80, 90, or 100% identical to that of another sequence. Sequence<br>
identity is typically measured using sequence analysis software with the default parameters specified therein (e.g., Sequence Analysis Software Package of the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710 University Avenue, Madison, WI 53705). This software program matches similar sequences by assigning degrees of homology to various substitutions, deletions, and other modifications.<br>
Advantages<br>
The present invention provides a number of advantages related to the production of transgenic ungulates (e.g., bovines) with a mutation in one or both alleles of the prion gene. For example, the prion knockout vector, cell culture method, and transfection method described herein inactivated the prion gene in bovine cells at a surprisingly high frequency. The large number of correctly targeted donor cells in combination with the improved nuclear transfer methods of the invention (e.g., chromatin transfer and SLOT) greatly facilitate the generation of cloned transgenic ungulates (e.g., bovines) with reduced or no expression of prion protein. These resulting transgenic ungulates are desirable sources of agricultural products (e.g., meat) and pharmaceutical products (e.g., human therapeutic antibodies encoded by one or more xenogenous nucleic acids in the transgenic ungulates).<br>
Other features and advantages of the invention will be apparent from the following detailed description and from the claims.<br>
Brief Description of the Drawings<br>
The application file contains drawings executed in color (Figs. 31, 34A-34C, 36A, 36B, 42A-42D, 45A, 46D, 47A, 47C, 48,49B, 50A-50D, and 51C-5 IE). Copies of this patent or patent application with color drawings will be provided by the Office upon request and payment of the necessary fee.<br>
Fig. 1A depicts an overview of the procedures used to produce a cow that contains an Ig knockout and human artificial chromosome. The time line in Fig. 1A is based on an estimated 18 months to prepare the Ig knockout vector and generate knockout cells, 2 months to generate fetuses from the knockout cells, 9 months to perform subsequent knockouts, 9 months of gestation for calves to be born, 12 months before embryos can be produced from calves, and 6 months to perform the HAC transfers.<br>
Fig. IB depicts an overview of the methods used to produce a cow that contains a mutation in an endogenous Ig gene and contains AHAC or AAHAC. For the time line in Fig. IB, it is estimated that 250 colonies are screened per week for a total of 3,000 colonies in 3 months to isolate male and female knockout cells. It is assumed that one or more knockout colonies are produced per 1,500 colonies. Homozygous knockout ungulates may be produced by (1) introducing a second Ig mutation in an isolated knockout cell before nuclear transfer, (2) introducing a second Ig mutation in a cell obtained from a embryo, fetus (e.g., fetus at ~60 gestation days), or offspring produced from a first round of nuclear transfer and using the resulting homozygous cell as the donor cell in a second round of nuclear transfer, or (3) mating hemizygous ungulates. In Figs. 1A and IB, "Homo" denotes homozygous; "Hemi" denotes hemizygous; "H" denotes heavy chain; "L" denotes light chain; "HAC" denotes human artificial chromosome; "HAC 1" denotes either HAC; and "HAC2" denotes a second HAC.<br>
Fig. 2A contains a mu (IgM heavy chain) knockout construct according to the invention. Figure 2B is a restriction map of immunoglobulin loci from a Holstein cattle.<br>
Figs. 3A-3B contain schematic illustrations of constructs pSTneoB and pLoxP-StneoB. Respectively, that were used to produce the mu knockout DNA construct.<br>
Fig. 3C depicts mu knockout DNA construct.<br>
Fig. 3D depicts the polynucleotide sequence of the 1.5 kb region of the genomic bovine mu heavy chain locus that was used as the first region of homology in the mu knockout construct (SEQ ID NO: 47).<br>
Fig. 3E depicts the polynucleotide sequence of the 3.1 kb region of the genomic bovine mu heavy chain locus that was used as the second region of homology in the mu knockout construct (SEQ ID NO: 48). In this sequence, each "n" represents any nucleotide or no nucleotide. The region of consecutive "n" nucleotides represents an approximately 0.9 to 1.0 kb region for which the polynucloetide sequence has not been determined.<br>
Fig. 3F depicts a puromycin resistant, bovine mu heavy chain knockout construct.<br>
Fig. 3G depicts the polynucleotide sequence of a bovine kappa light chain cDNA (SEQ ID NO: 60). All or part of this sequence may be used in a kappa light chain knockout construct. Additionally, this kappa light chain may be used to isolate a genomic kappa light chain sequence for use in a kappa light chain knockout construct.<br>
Fig. 4 is a schematic illustration of the construction of AHAC and AAHAC.<br>
Fig. 5 is a photograph of an agarose gel showing the presence of genomic DNA encoding human heavy and light chains in AHAC fetuses.<br>
Fig. 6 is a photograph of an agarose gel showing the expression of human Cmu exons 3 and 4 in a AHAC fetus at 77 gestational days (fetus #5996).<br>
Fig. 7 is a photograph of an agarose gel showing the rearrangement of endogenous bovine heavy chain in AHAC fetus #5996<br>
Fig. 8 is a photograph of an agarose gel showing the expression of rearranged human heavy chain in AHAC fetus #5996.<br>
Fig. 9 is a photograph of an agarose gel showing the expression of the spliced constant region from the human heavy chain locus in AHAC fetus #5996<br>
Fig. 10 is a photograph of an agarose gel showing the expression of rearranged human heavy chain in AHAC fetus #5996.<br>
Fig. 11A is the polynucleotide sequence of a rearranged human heavy chain transcript from AHAC fetus #5996 (SEQ ID NO: 49).<br>
Fig. 11B is a sequence alignment of a region of this sequence ("Query") with a human anti-pneumococcal antibody ("Sbjct") (SEQ ID NOs: 50 and 51, respectively). For the query sequence from AHAC fetus #5996, only those nucleotides that differ from the corresponding nucleotides of the human anti-pneumococcal antibody sequence are shown.<br>
Figs. 12A-12B depict two additional polynucleotide sequences (SEQ ID NOs: 52 and 54) and their deduced amino acid sequences (SEQ ID NOs: 53 and 55, respectively) of rearranged human heavy chain transcripts from AHAC fetus #5996.<br>
Fig. 13 is a photograph of an agarose gel demonstrating that AAHAC fetus #5580 contains both human heavy and light chain immunoglobulin loci.<br>
Fig. 14 is a photograph of an agarose gel demonstrating that AAHAC fetuses #5442A, and 5442B contain both human heavy and light chain loci.<br>
Fig. 15 is a photograph of an agarose gel showing the expression of the spliced mu constant region from the human heavy chain locus in AAHAC fetus #5542A.<br>
Fig. 16 is a photograph of an agarose gel showing the rearrangement and expression of the human heavy chain locus in AAHAC fetus #5868A.<br>
Fig. 17 is a photograph of an agarose gel showing rearrangement and expression of the human Ig lambda locus in AAHAC fetuses #5442A and 5442B.<br>
Fig. 18 is a photograph of an agarose gel showing rearrangement and expression of the human Ig lambda locus in AAHAC fetus #5442A.<br>
Fig. 19 is a photograph of an agarose gel showing rearrangement and expression of the human Ig lambda locus in AAHAC fetus #5868A.<br>
Fig. 20 depicts the polynucleotide sequence and the corresponded deduced amino acid sequence of a rearranged human light chain transcript from AAHAC fetus #5442A (SEQ ID NOs: 56 and 57, respectively).<br>
Fig. 21 depicts another polynucleotide sequence and the corresponding deduced amino acid sequence of a rearranged human light chain transcript from AAHAC fetus #5442A (SEQ ID NOs: 58 and 59, respectively).<br>
Figs. 22A-22H are graphs of a FACS analysis of expression of human lambda light chain and bovine heavy chain proteins by AAHAC fetuses #5442A (Figs. 22A-22D) and 5442B (Figs. 22E-22H). Lymphocytes from the spleens of these fetuses were reacted with a phycoerytherin labeled anti-human lambda antibody (Figs. 22C and 22D), a FITC labeled anti-bovine IgM antibody (Figs. 22D and 22H), or no antibody (Figs. 22A, 22B, (22E, and 22F) and then analyzed on a FASCalibur cell sorter. The percent of cells that were labeled with one of the antibodies is displayed beneath each histogram.<br>
Fig. 23 is a schematic illustration of the a-(l,3)-galactosyltransferase knockout vector used to insert a puromycin resistance gene and a transcription termination sequence into the endogenous a-(l,3)-galactosyltransferase gene in bovine cells.<br>
Fig. 24 is a schematic illustration of a BamHl-Xhol fragment containing exons 2, 3, and 4 that was used as a backbone for the AAV targeting vector. A neomycin resistance marker was used for insertional mutagenesis of the locus by insertion into exon 4. The location of the annealing sites for the PCR<br><br>
primers that were used for subsequent confirmation of appropriate targeting is indicated.<br>
Fig. 25 is a schematic illustration of the construction of an adeno-associated viral construct designed to remove endogenous bovine IgH sequence.<br>
Fig. 26 is a photograph of an agarose gel showing the PCR analysis of individually transduced clones for appropriate targeting events. The vector used in this experiment is shown in Fig. 25. PCR products indicative of appropriate targeting are marked with asterisks.<br>
Fig. 27 is a table listing pregnancy rates for HAC carrying embryos.<br>
Figs. 28A-28J are photographs of FACS analysis of peripheral blood lymphocytes from either experimental fetuses injected with an anti-bovine IgM antibody (das6) to inhibit B-cell development or control fetuses. Figures 28A-28E are photograph of the FACS analysis performed using an anti-bovine IgM antibody to detect IgM molecules expressed on the surface of B-cells.   As illustrated in Fig. 28 A and Fig. 28B, approximately 19.82 to 26.61% of the peripheral blood lymphocytes from the control fetuses expressed IgM. In contrast, 7.78, 11.80, or 3.95% of the peripheral blood lymphocytes from the three fetuses injected with the anti-bovine IgM antibody expressed IgM (Figs. 28C-28E, respectively). Figures 28F-28J are photographs of the FACS analysis performed using an anti-bovine light chain antibody (das 10) to detect antibody light chain molecules expressed on the surface of B-cells. As illustrated in Fig. 28F and Fig. 28G, approximately 12.43 to 29.47% of the peripheral blood lymphocytes from the control fetuses expressed antibody light chain molecules. In contrast, 2.54, 13.77, or 3.99% of the peripheral blood lymphocytes from the three fetuses injected with the anti-bovine IgM antibody expressed antibody light chain molecules (Figs. 28H-28J, respectively).<br>
Figs. 29A-29B illustrate the immunodetection of nuclear envelope and nuclear matrix proteins in bovine preimplantation embryos. Fig. 29A is a picture of in v//ro-fertilized bovine embryos at the pronuclear and 8-cell stage examined using<br>
the same antibodies. Arrows in Fig. 29A to anti-NuMA and anti-AKAP95 labeling in the female pronucleus of pronuclear stage embryos. Insets in Fig. 29A are pictures of DNA labeled with 0.1 ug/ml Hoechst 33342 (bars, 20 urn). Fig. 29B is the immunoblotting analysis of bovine fibroblasts (upper rows) and pronuclear stage embryos (lower rows). Molecular weight markers are shown in kDa on the right of Fig. 29B,<br>
Fig. 30 is a photograph of bovine donor fibroblasts (Donor cell), nuclear transplant embryos at the premature chromatin condensation stage (three hours post-fusion), nuclear transplant embryos at the pronuclear stage (19 hours post-fusion), and parthenogenetic pronuclear stage embryos activated as described herein. Disassembly of the donor nucleus and assembly of the new pronuclei were monitored at the premature chromatin condensation stage three hours post injection "hpi" ("PCC") and seven hours post injection ("NT PN"), using anti-lamin B, lamins A/C, NuMA, and AKAP95 antibodies. Female pronuclei formed after parthenogenetic activation of Mil oocytes with 10 mM SrCl2 were also analyzed five hours after start of activation treatment ("Parth. PN"). Lamins A/C were assembled in pronuclei of bovine pronuclear stage nuclear transplant embryos. DNA was counterstained with 0.1 fig/ml Hoechst 33342. TRITC refers to labeling with TRITC-conjugated secondary antibodies (bars, 20 urn).<br>
Fig. 31 is a graph demonstrating that AKAP95 is more strongly anchored in pronuclei of nuclear transplant embryos compared to parthenogenetic embryos. This graph shows the relative percent of unextracted lamin B, AKAP95, and DNA labeling in pronuclei of parthenotes, nuclear transplant embryos, and somatic donor nuclei after in situ extraction with 0.1% Triton X-100 and 1 mg/ml DNAse I together with 100 or 300 mM NaCl for 30 minutes at room temperature prior to fixation with 3% paraformaldehyde. Localization of B-type lamins (red) and AKAP95 (green) was examined by double immunofluorescence. Fluorescence labeling intensity in each channel -red, (lamin B), blue (DNA), and green (AKAP95) - was quantified. The<br>
eference value (100% unextracted) represents relative amounts of B-type lamins, DNA, and AKAP95 staining in embryos or cells permeabilized with 0.1% Triton X-100 only prior to fixation. Approximately 30 embryos were examined in each group.<br>
Fig. 32 is a photograph of bovine pronuclear nuclear transplant embryos produced by fibroblast fusion and oocyte activation with either 5 uM ionomycin for four minutes followed by 10 }ig/ml cycloheximide/2.5 fig/ml cytochalasin D for four hours (b', -), ionomycin/cycloheximide/cytochalasin D as in (b') followed by an additional nine hours of culture with 10 ug/ml cycloheximide (b", CHX) or incubation as in (b') together with 1 |ig/ml actinomycin D during the entire activation treatment (b'"). Anti-lamin B (rabbit polyclonal) and anti-lamins A/C (mAb) antibodies were used on the same preparations. Insets are pictures of DNA labeling with 0.1 ug/ml Hoechst 33342 (bars, 20 urn).<br>
Fig. 33 is a graph of chromosome condensation and nuclear envelope breakdown in mitotic cytoplasmic extract (M-S15), mitotic cytosolic extract (M-S200), and oocyte extract (MII-S15) (n=300-400 nuclei examined in 3-5 replicates).<br>
Figs. 34A-34C are sets of pictures of immunofluorescence analysis of purified input bovine fibroblast nuclei (Fig. 34A) and condensed chromatin produced in mitotic cytosolic extract (Fig. 34B) and oocyte extract (Fig. 34C). The indicated nuclear markers were examined. DNA was counterstained with propidium iodide (red) (bars, 10 um).<br>
Fig. 35 is a series of photographs of immunofluorescence analysis of condensed chromatin obtained in oocytes following conventional nuclear transplant (NT) or nuclear injection (NI) methods and following injection of chromatin masses into oocytes (CT) using the methods of the present invention. Both detectable lamins B and A/C appear to be solubilized (bar, 10 u,m).<br>
Figs. 36A-36B are a series of photographs of immunofluorescence analysis of pronuclei resulting from chromatin transfer, nuclear transplant, or<br>
nuclear injection. Embryos were fixed at 19 hours post nuclear transplant, nuclear injection, or chromatin transfer and labeled. Control parthenogenetic pronuclei (Part.) were also examined. Fig. 36A shows the analysis of lamins A/CandB. Fig. 36B shows the analysis of AKAP95 and NuMA. Lamins A/C (green label) only appear in nuclear transplant and nuclear injection pronuclei (bars, 30 fim).<br>
Fig. 37 is a schematic illustration of a procedure for production of cloned Tc calves. Construction of HAC is shown with hChr22 and hChr!4 regions containing Ig^. and IgH genes and the position of the neo selection marker. From a CHO clone, the HAC was transferred into fetal bovine fibroblasts by means of a MMCT technique. Tc fibroblasts were fused with an enucleated oocyte for nuclear transfer. The reconstituted Tc embryos were cultured in vitro to the blastocyst stage then implanted into recipient cows. Around 60 days of gestation Tc fetuses were recovered; fibroblast cell lines were reestablished, evaluated and used for further nuclear transfer. Recloned Tc embryos were transferred to recipients to produce Tc calves.<br>
Figs. 38A-38D illustrate the analysis of Tc fetuses. G418 selection of regenerated Tc fibroblast line and control non-transgenic fibroblasts (Fig. 38A). Genomic PCR of IgH and Igl loci in Tc fetuses and controls. The three fetuses, #5968 (lane 1), #6032 (lane 2) and #6045 (lane 3) were derived from AHAC fibroblasts, fetus #5580 (lane 4) was from AAHAC fibroblasts. As a control, a non-transgenic fetus (lane N) was recovered and evaluated (Fig. 38B). In all Tc fetuses and a positive control human liver DNA sample (lane P), both human IgH and Igl loci were detected by PCR, but not in the negative control (lane N). Rearranged and expressed human Ig/j. and 7g/l transcripts amplified by RT-PCR from negative control non-transgenic bovine spleen (lane N), from brain (lane 1), liver (lane 2) and spleen (lane 3) of cloned Tc fetus and positive control human spleen (lane P) (Fig. 38C). A representative nucleotide and deduced amino acid sequence of human Ig/j. and IgA transcripts amplified by RT-PCR from a cloned Tc fetus recovered at 91 days (Fig. 38D).<br><br>
RT-PCR was carried out as described (Kuroiwa et al, Nature Biotech 18:1086-1090, 2000). For human Igu transcripts, VH1/5BACK, VH3BACK, and VH4BACK were used as a 5' primers and Cu-2 was used as a 3' primer. For human IgX transcripts, VX1LEA1, VA2MIX and VA3MIX were used as 5' primers, and C1MIX was used as a 3' primer. The amplified cDNAs were subcloned using a TA cloning kit (Invitrogen) and sequenced using a DNA autosequencer (ABI3700 system).<br>
Figs. 39A-39C are photographs illustrating the analysis of cloned Tc calves.   Four cloned Tc calves; male calf (# 50) from cell line #6045 and female calves (#1064, #1065, #1066) from cell line #5968 (Fig. 39A). Genomic PCR of IgHand IgA loci from PBLs from cloned Tc calves and controls; calf #1064 (lane 1), #1065 (lane 2), #1066 (lane 3), #50 (lane 4), #1067 (lane 5) and #1068 (lane 6) (Fig. 39B). In all the Tc calves and positive control human liver DNA (lane P) both human IgH and IgA loci were detected by genomic PCR, but not in a negative control, non-transgenic calf (lane N). FISH analysis in metaphase chromosome spreads in a cell showing a single signal and a cell showing a double signal (Fig. 39C). Arrows indicate location of HACs amongst surrounding bovine chromosomes. HAC painting was done using digoxigenin labeled human COT-1 DNA as a probe and detected with an anti-digoxigenin-rhodamine.<br>
Figs. 40A-40B illustrate the immunofluorescence distribution of lamin B, lamin A/C, NuMA, and AKAP95 in bovine fetal fibroblasts (Fig. 40A) and pronuclear and 8-16-cell stage bovine embryos cultured in vitro (Fig. 40B). Arrows point to NuMA and AKAP95 labeling in the female pronucleus (the smallest of both pronuclei). Insets, DNA labeled with 0.1 ug/ml Hoechst 33342.<br>
Fig. 40C illustrates the immunoblotting analysis of lamin B, lamin A/C, NuMA, and AKAP95 in bovine fibroblasts (Fib) and pronuclear embryos (PN). Bars, 10 urn (Fig. 40A), 50 urn (Fig. 40B).<br>
Figs. 41A-41D illustrate the dynamics of the nuclear envelope, NuMA, and AKAP95 in bovine NT embryos. The disassembly of the donor fibroblast nucleus and assembly of the new pronucleus were immunofluorescently monitored at the PCC and PN stage using indicated antibodies (TRITC) (Fig. 41 A). Parthenogenetic pronuclei following a similar activation procedure were also examined (Part. PN). Percent (SD) embryos labeled with each antibody is shown (n~20 embryos/group in three replicates). Recipient oocytes were activated with either 5 uM ionomycin for four minutes followed by 10 fig/ml CHX/2.5 fig/ml cytochalasin D for four hours (b),<br>
ionomycin/CHX/cytochalasin D as in (b) followed by nine hours with 10 ug/ml CHX (b'), or as in (b) together with 5 ug/ml ActD during the activation treatment (b") (Fig. 4IB). DNA (insets in B) was labeled with Hoechst 33342. Bars, 10 jim. Figure 41C is a bar graph of the meanSD immunofluorescence intensity of lamins A/C and B and NuMA in CHX- and ActD-treated NT embryos relative to untreated (-) NT embryos (100% fluorescence; n=30 embryos/group in three replicates). Figure 41D is a bar graph of the intranuclear association of AKAP95 in pronuclei of NT embryos. PN stage parthenotes (Part.) and NT embryos (NT), and donor flbroblasts (Fib) were extracted with 0.1% Triton X-100/1 mg/ml DNAse 1/300 mM NaCl prior to fixation. Lamin B and AKAP95 were examined by immunofluorescence, and DNA was stained with Hoechst 33342. Data are shown as mean +SD fluorescence labeling intensity of lamin B, DNA, and AKAP95 relative to fluorescence in non-extracted parthenogenetic or NT embryos, or in somatic cells (100% fluorescence; n~30 embryos/group in three replicates). Bars, 10<br>
um.<br>
Figs. 42 A-42D demonstrate that CT produces embryos devoid of lamin A/C and NuMA. Redistribution of lamins A/C and B, NuMA and AKAP95 was monitored by immunofluorescence in bovine fibroblast nuclei incubated in extract from mitotic bovine flbroblasts (Chaudhary et al., J. Cell Biol. 122:295-306, 1993) (Fig. 42A). DNA was labeled with 0.1 ug/ml propidium iodide.<br>
Percentages of nuclei displaying disassembly (disappearance of labeling) of indicated markers are shown (n&gt;200 nuclei/marker in three replicates). PN embryos were produced by CT, NT, NI, or parthenogenesis, and distribution of lamins A/C and B, NuMA and AKAP95 was examined by immunofluorescence 14 hours after start of activation (Poccia et at, Trends Biochem. Sci. 17:223-227, 1992 and Chaudhary et al., J. Cell Biol. 122:295-306, 1993) (Fig. 42B). DNA was labeled with Hoechst 33342. Mean SD immunofluorescence intensity of indicated markers in CT, NI, or parthenogenetic embryos relative to fluorescence intensity in PN-stage NT embryos (NT, 100% fluorescence; n= 15-20 embryos/marker in three replicates) (Fig. 42C). Immunolabeling intensity of lamin B, DNA, and AKAP95 in PN-stage NT and CT embryos after extraction with Triton X-100/DNAse I/NaCl (Fig. 42D). Values are expressed as meanSD fluorescence intensity relative to non-extracted pronuclear NT and CT embryos, respectively (n~15 embryos/marker/treatment in three replicates). Bars 10 jim (Fig. 42A), 20 |im (Fig. 42B).<br>
Figs. 43A-43B illustrate the in vitro donor nucleus disassembly and alteration of TBP localization in reconstituted pronuclei by CT. Dynamics of TBP during NT and CT was followed by immunofluorescence analysis of donor fibroblasts (Fib), condensed chromatin at the PCC stage (NT PCC), donor chromosomes condensed in mitotic extract (MS 15 CC), NT pronuclei (NT PN), and CT pronuclei (CT PN) are compared in Fig. 43B. DNA was labeled with Hoechst 33342. Bars, 20 \an. TBP/AKAP95 and TBP/DNA fluorescence intensity ratios at stages shown in Fig. 43A and in parthenogenetic pronuclei (Part. PN) (Fig. 43B). MeanSD fluorescence intensity in 10-13 embryos per group is shown.<br>
Fig. 44A is a schematic illustration of a prion protein knockout vector of the invention containing a neomycin-resistant gene (pBPrP(H)KOneo vector). This scheme illustrates how to identify the correctly targeted clones.<br><br>
Fifr44B is a schematic illustration of a prion protein knockout vector of the invention containing a puromycin-resistant gene (pBPrP(H)KOpuro vector) and a method for identifying the correctly targeted clones.<br>
Fig. 44C is a table listing the frequency of homologous recombination at the prion protein locus by using the prion protein knockout vector of the present invention in different bovine fibroblast cell lines.<br>
Fig. 45A is a photograph illustrating the level of lamin B, lamin A/C, NuMA, AKAP95, and DNA in pronuclei from NT and SLOT embryos.<br>
Fig. 45B is a graph of the fluorescence intensity from Fig. 45 A.<br>
Figs. 46A-46D depict the dynamics of fibroblast nuclei after NT. Fig. 46A shows the immunofluorescence distribution of lamin B, lamin A/C, and TBP in bovine fibroblasts and IVP pronuclear (PN) and 8-16-cell stage bovine embryos. Insets, DNA labeled with Hoechst 33342. Bars, (left) 10 urn, (right) 50 ^m. Fig. 46B is a picture of an immunoblot of lamin B, lamin A/C, and TBP in fetal fibroblasts and pronuclear embryos. Fig. 46C is a picture of immunofluorescence analysis of lamin B, lamin A/C, and TBP at the PCC and pronuclear (NT-PN) stages in NT embryos. Bars, 10 jam. Fig. 46D is a bar graph of the immunolabeling intensity of indicated proteins in male (MPN) and female (FPN) pronuclei of IVP embryos and in pronuclei of NT embryos (NT-PN). Data are expressed as meanSD ratio of antibody fluorescence over Hoechst 33342 (DNA) fluorescence. Over 20 embryos per marker were analyzed in 3-5 replicates in (A, C, D).<br>
Figs. 47A-47C illustrate the characterization of pronuclei in NT embryos. Fig. 47 A is a bar graph of the immunolabeling intensity (meanSD) of TBP and DNA (Hoechst 33342) in pronuclear IVP and NT embryos and in fibroblasts after in situ extraction with 1% Triton X-100/1 mg/ml DNAse 1/300 mM NaCl. Labeling intensity is expressed relative to that of non-extracted embryos (n=30 embryos or cells per group). Fig. 47B is a photograph illustrating expression of markers in NT embryos activated as (b) described herein, or (b') in the presence of 10 ng/ml CHX or (b") 5 (ig/ml ActD (n=30<br>
embryos/group in 2-3 replicates). Insets, DNA". Bar, 10 jam. Fig~ 47C is a bar graph of the ratio (meanSD) of immunolabeling intensity of indicated proteins over Hoechst 33342 (DNA) fluorescence in embryos activated as in Fig. 47B (n=l 5-20 embryos/treatment/marker).<br>
Fig. 48 illustrates the SLOT procedure. In step one, donor fibroblasts are reversibly permeabilized for 30 minutes with 500 ng/ml SLO. In step two, permeabilized cells are washed and incubated in a mitotic extract containing an ATP-regenerating system to elicit chromosome condensation and promote removal of nuclear components (arrows). In step three, the extract is removed, and the cells are optionally resealed in culture with 2 mM CaCl2 for two hours. In step four, cells are fused to enucleated recipient oocytes, and in step five, oocytes are activated as for NT to elicit pronuclear formation and development.<br>
Figs. 49A-49C show the induction of somatic nuclear breakdown in mitotic extract. Fig. 49A demonstrates that the mitotic extract does not elicit apoptosis. Intact bovine fibroblasts (lane 1), fibroblasts induced into apoptosis with 5 ug/ml nocodazole for 20 hours (lane 2), or SLO-permeabilized fibroblasts exposed to mitotic (lane 3) or interphase (lane 4) extract for one hour were immunoblotted using anti-PARP antibodies (upper panel). DNA degradation was evaluated by gel electrophoresis in 0.8% agarose and staining with ethidium bromide (lower panel). Fig. 49B is a picture of the immunofluorescence analysis of lamin B, lamins A/C, TBP, and AKAP95 after a 45-minute incubation of permeabilized fibroblasts in mitotic extract with (+ATP) or without (-ATP) an ATP-regenerating system. Bars, 10 (im. Over 1,000 cells analyzed in 4 replicates displayed consistent labeling as shown. Fig. 49C is a picture of the immunoblotting analysis interphase fibroblasts (lane 1), fibroblasts synchronized at mitosis with 1 ng/ml nocodazole (lane 2), chromatin isolated from permeabilized fibroblasts exposed to mitotic extract with or without the ATP-regenerating system (lanes 3, 4) or to interphase extract (lane 5), and whole permeabilized fibroblasts exposed to mitotic extract<br>
(lane 6); Blots were probed using antibodies to indicated proteins. Histone H4 was used as a protein loading control in the gel.<br>
Figs. 50A-50D demonstrate the characterization of nuclei of SLOT embryos. Fig. 50A illustrates chromatin morphology (arrow) within 30 minutes of introduction into an oocyte by SLOT or NT as indicated. Dotted line delineates the oocyte cytoplast. Panels on the right show enlargements of donor chromatin in a SLOT and NT embryo. Bar, 20 um. Fig. 50B illustrates the distribution of lamin B, lamin A/C, and TBP in pronuclear (upper two panels) and 8-16-cell stage (lower two panels) embryos produced by NT or SLOT. Bars, 20 urn (n=20-30 embryos/ marker in 3 replicates). Fig. 50C is a bar graph of the ratio (meanSD) of immunolabeling intensity over Hoechst 33342 (DNA) fluorescence intensity for indicated protein in pronuclei of NT or SLOT embryos (n~20 embryos/marker in 3 replicates). Fig. SOD is a bar graph of the immunolabeling intensity (meanSD) of TBP and DNA (Hoechst 33342) in pronuclear NT and SLOT embryos after extraction with 0.1% Triton X-100/1 mg/ml DNAse 1/300 mM NaCl, relative to that in non-extracted embryos (n=15 embryos/group in 2 replicates).<br>
Figs. 51A-51E illustrate the development and morphological evaluation of SLOT and NT clones. Fig. 51A is a bar graph of the rates of in vivo development of NT and SLOT clones. Pregnancy rates were assessed by ultrasonography at indicated days of gestation. Percentages of calves born and of live calves at 24 hour post-partum are also shown. Twelve SLOT and 23 NT calves were born from 59 and 211 embryos transfers, respectively, over eight weeks. P values were produced by Fisher's Exact Tests. Fig. 5 IB is a picture of SLOT clones (Simintal x Angus cross identical females) at 5-6 weeks of age. Fig. 51C is a graph of the scores of individual SLOT and NT calves and placentas within 24 hours of birth. Fig. 51D is a graph of the birth weights of individual SLOT and NT clones. In Figs. 51C and 5ID within SLOT (dots) or NT (squares) clones, one color represents calf score, placenta score, and birth<br>
weight for the same individual. Fig. 5IE is the box plot analysis of calf scores, placenta scores, and birth weights of SLOT and NT clones.<br>
Detailed Description<br>
The present invention is based in part on the surprisingly high frequency of inactivation of the bovine prion locus using the methods described herein. The large number of correctly targeted prion knockout cells greatly facilitates the generation of prion knockout ungulates (e.g., bovines). Additionally, the invention provides improved methods for increasing the efficiency of nuclear transfer to further simplify the production of prion knockout ungulates. These resulting transgenic ungulates are desirable sources of agricultural products (e.g., meat) and pharmaceutical products (e.g., human therapeutic antibodies encoded by one or more xenogenous nucleic acids in the transgenic ungulates).<br>
The knockout vector used to inactivate the prion locus in donor bovine cells was based in part on the sequence of the bovine prion protein gene locus deposited in Genbank (AJ298878). The source of the genomic sequence of the prion protein gene used for construction of the knockout vector matches that of strain of fibroblasts that were genetically modified. In particular, the Holstein genomic sequence of prion protein gene was used for construction of the knockout vector for efficient targeting in Holstein-derived fibroblasts. The knockout vector contained two regions of homology to the prion locus that are contiguous in the prion locus and was designed to insert a transcript!onal termination (STOP) sequence together with a drug-resistant gene flanked by two loxP sites into the prion locus. An alternative strategy involves using two noncontiguous regions of homology in the knockout vector to delete the region of the endogenous prion protein gene between the two regions of homology. To reduce the activity of prion protein, the STOP sequence was inserted just behind its initial ATG codon in exon 3. For efficient homologous recombination at the locus, one region of homology was longer than the other region of homology. In particular, the length of the 3' region was 8.3 kb, which is much longer than that of the 1.1 kb 5' region of homology. To<br>
concentrate homologous recombinants, a negative selection marker, such as DT-A gene, was attached to the short 5' region of homology in an opposite direction relative to the prion protein gene. The knockout vector was linearlized before tranfection by an appropriate restriction enzyme so that the basal plasmid vector, such as pUC of pBluescript, could be attached to the end of the negative selection marker. Furthermore, to enhance homologous recombination, the linearlized knockout vector was treated with HBS (Hepes buffered saline) containing 1 mM spermidine before addition to fibroblasts.<br>
Polyamine and any other compound can be added to the targeting vector to facilitate the efficiency of homologous recombination. A preferable concentration of polyamine is 0.1 to 10 mM; most preferably, the concentration is 1 mM. Polyamine may be selected from, e.g., spermidine, spermine, cadaverine, and putrescine. A preferred polyamine is spermidine. The other compound may be selected from compounds that can and form an ionic bond to the phosphate residue of DNA, such as poly-L-lysine or poly-L-arginine.<br>
Further homologous recombination efficiency can be facilitated by placing the negative selection marker so that it is not exposed on the terminus of the linearized targeting vector. Preferably, the 5' and 3' termini of the functional unit of the negative selection marker are located at least 1Kb, preferably at least 2Kb, away from the 5' and 3' termini of the linearized targeting vector, respectively. Generally, homologous genomic sequence is located on the 5' and 3' side of the negative selection marker, resulting in a sufficient distance of the negative selection marker from the termini of the linearized targeting vector. In the linearized targeting vector, the other side of the negative selection markers can be designed to retain at least 1Kb, preferably at least 2Kb extra sequence. For example, the pUC vector sequence in the targeting vector can be used as this extra sequence.<br>
In one particular method, a Holstein-derived knockout vector was electroporated into Holstein fetal fibroblasts, and hemizygously targeted clones were obtained at a surprisingly high frequency of more than 50%. The large<br>
number of correctly targeted clones should be sufficient to efficiently produce healthy fetuses and calves. To produce homozygously targeted fetuses and calves, hemizygous knockout fibroblasts isolated from hemizygous targeted fetuses can be electroporated with the same knockout vector or a knockout vector with a different antibiotic resistance gene to mutate the remaining allele of the prion locus. Alternatively, homozygous knockout calves can be produced by mating male and female hemizygous knockout cattle.<br>
Methods for producing ungulates that express xenogenous antibodies<br>
Homozygous prion protein knockout bovines producing human antibodies (prion protein KO/hAb bovine) can be generated by introduction of human immunoglobulin loci into bovine fibroblasts with a homozygous prion protein knockout genotype, e.g., as described in WO02/70648 and Kuroiwa et al. Nature Biotechnol. 20; 889-894, 2002. Suitable approaches include, for example, the insertion of a human artificial chromosome (HAC) containing both heavy and light chain immunoglobulin genes into an ungulate or the insertion of human B-cells or B-cell precursors into an ungulate during its fetal stage or after it is born (e.g., an immune deficient or immune suppressed ungulate) (see, for example, PCT Publication Nos. WO01/35735 and WO02/074938).<br>
Standard methods can be used to introduce a desired nucleic acid that contains genes (preferably, entire gene loci) for producing antibodies of a particular species (e.g., a human) into a donor cell for use in generating a transgenic ungulate that expresses xenogenous antibodies. Preferably, human artificial chromosomes are used, such as those disclosed in PCT Publication Nos. WO97/07671 and-WOOO/10383. These human artificial chromosomes also are described in issued Japanese patent JP 30300092. Each of these publications is incorporated by reference in their entirety herein. Also, the construction of artificial human chromosomes or transgenes that contain and<br>
express human immunoglobulin genes is disclosed in Shen et al., Hum. Mol. Genet. 6:1375-1382 (1997); Kuroiwae/a/., Nature Biotechnol. 18:1086-1090 (2000); Loupert et al, Chromosome 107:255-259 (1998); PCT Publication Nos. WOOO/10383, W098/24893, WO96/33735, W097/13852, W098/24884, and WO97/07671; and U.S. Patent Nos. 5,877,397, 5,874,299, 5,814,318, 5,789,650, 5,770,429, 5,661,016, 5,633,425, 5,625,126, 5,569,825, and 5,545,806, each of which is incorporated by reference in its entirety herein. Human artificial chromosomes may also be utilized to introduce xenogenous antibody genes into wild-type animal cells; this is accomplished using the methods described above. Introduction of artificial chromosome into animal cells, especially fetal fibroblast cells can be performed by microcell fusion as described herein.<br>
In an alternative to the use of human artificial chromosome, nucleic acid encoding immunoglobulin genes may be integrated into the chromosome using a YAC vector, BAG vector, or cosmid vector. Vectors comprising xenogenous Ig genes (described, e.g., in PCT Publication Nos. WO98/24893, WO96/33735, W097/13852, and WO98/24884, and U.S. Patent Nos. 5,849,992 and 5,827,690) can be introduced to fetal fibroblasts cells using known methods, such as electroporation, lipofection, fusion with a yeast spheroplast comprising a YAC vector, and the like. Further, vectors comprising xenogenous Ig genes can be targeted to the endogenous Ig gene loci of the fetal fibroblast cells, resulting in the simultaneous introduction of the xenogenous Ig gene and the disruption of the endogenous Ig gene.<br>
Integration of a nucleic acid encoding a xenogenous immunoglobulin gene may also be carried out as described in the patents by Lonberg et al. (supra}. In the "knockin" construct used for the insertion of xenogenous immunoglobulin genes into a chromosome of a host ungulate, one or more immunoglobulin genes and an antibiotic resistance gene may be operably-linked to a promoter which is active in the cell type transfected with the construct. For example, a constitutively active, inducible, or tissue specific<br>
promoter may be used to activate transcription of the integrated antibiotic resistance gene, allowing transfected cells to be selected based on their resulting antibiotic resistance. Alternatively, a knockin construct in which the knockin cassette containing the Ig gene(s) and the antibiotic resistance gene is not operably-linked to a promoter may be used. In this case, cells in which the knockin cassette integrates downstream of an endogenous promoter may be selected based on the resulting expression of the antibiotic resistance marker under the control of the endogenous promoter. These selected cells may be used in the nuclear transfer procedures described herein to generate a transgenic ungulate containing a xenogenous immunoglobulin gene integrated into a host chromosome.<br>
Using similar methodologies, it is possible to produce and insert artificial chromosomes containing genes for expression of Ig of different species such as dog, cat, other ungulates, non-human primates among other species. As discussed above, and as known in the art, immunoglobulin genes of different species are well known to exhibit substantial sequence homology across different species.<br>
Once it has been determined that the inserted artificial chromosome, for example, a human artificial chromosome, has been stably introduced into a cell line, e.g., a bovine fetal fibroblast, it is utilized as a donor for nuclear transfer. This may be determined by PCR methods. The resulting transgenic calves comprise a stably introduced nucleic acid, such as a human artificial chromosome. After calves have been obtained which include the stably incorporated nucleic acid (for example, human artificial chromosome), the animals are tested to determine whether they express human Ig genes in response to immunization and affinity maturation.<br>
Methods for reducing production of endogenous antibodies in ungulates that also express xenogenous antibodies<br>
Modifications of the overall procedure described above may also be performed. For example, to reduce the amount of endogenous antibody expressed by the resulting transgenic ungulate, one or more endogenous Ig genes may be inactivated in the donor cell before or after insertion of the xenogenous nucleic aicd. Further, an animal retaining xenogenous Ig genes may be mated with an animal in which an endogenous Ig gene is inactivated. An exemplary scheme for the sequential manipulation of donor cells (e.g., bovine fibroblasts) is described below.<br>
First, generation of homozygous prion protein knockout fetal fibroblasts are generated. Then, hemizygous bovine immunoglobulin heavy chain (BIgH) knockout fibroblasts are generated by gene targeting, and fetuses are producing using nuclear transfer (e.g., SLOT or chromatin transfer). Homozygous BIgH knockout fibroblasts are produced by second gene targeting, and then fetuses are produced using nuclear transfer. Hemizygous bovine immunoglobulin light chain (BIgL) knockout fibroblasts are produced by gene targeting, followed by production of fetuses by nuclear transfer. Homozygous BIgL knockout fibroblasts are generated through second gene targeting, and fetuses are produced by nuclear transfer. Then, human immunoglobulin gene loci are introduced into the above sequential knockout fibroblasts having a homozygous prion protein/BIgH/BIgL knockout genotype, and fetuses and calves are produced by means of nuclear transfer.<br>
Exemplary methods for generating ungulates of the invention<br>
Preferred methods for generating an ungulate having a mutation in an endogenous prion nucleic acid and optionally having one or more nucleic acids encoding a xenogenous antibody involve remodeling the donor genetic material before it is inserted into a recipient oocyte to form the transgenic ungulate. Remodeling refers to any morphological change that improves development of the resulting nuclear transplant oocyte over that derived from either transferring whole cells or intact nuclei into a recipient oocyte. Reprogramming is<br>
achieved by incubating a donor nucleus that has a mutation in an endogenous prion nucleic acid and optionally has one or more nucleic acids encoding a xenogenous antibody in a reprogramming media (e.g., a mitotic extract, detergent and/or salt solution, or protein kinase solution) resulting in nuclear envelope dissolution and possibly chromatin condensation. This nuclear envelope breakdown and chromatin condensation allows the release of transcription regulatory proteins that were attached to the chromosomes and that would otherwise promote the transcription of genes undesirable for oocyte, embryo, or fetus development. Additional regulatory proteins may be removed by purifying the chromatin mass prior to transferring it into a recipient oocyte. Alternatively, specific regulatory proteins that are released from the chromosomes may be immunodepleted or otherwise removed from the reprogrammed media (e.g., a cell extract) to prevent them from re-binding the chromosomes. After nuclear transfer, new proteins from the oocyte cytoplasm may be bound to the chromosomes during decondensation of the chromatin and nuclear envelope formation in the oocyte. These proteins promote the transcription of genes that allow the oocyte to develop into a viable offspring.<br>
Another cloning method that can be used to produce the transgenic ungulate involves reprogramming a permeabilized cell (i.e., a cell having a mutation in an endogenous prion nucleic acid and optionally having one or more nucleic acids encoding a xenogenous antibody) by incubating it in a reprogramming media (e.g., a cell extract) to allow the addition or removal of factors from the cell. The plasma membrane of the permeabilized cell is preferably resealed to enclose the desired factors and restore the membrane integrity of the cell. The reprogrammed cell is then transferred into a recipient ooctye for the production of a cloned mammal. This cloning method has been used to produce fetuses without a xenogenous immunoglobulin nucleic acid that have survived past day 60. Preliminary results indicate that fetal survival between day 40 and day 60 is higher for fetuses formed using this method (7/10; 70%) than for conventional nuclear transfer fetuses (8/16; 50%), Similar<br>
results are expected for fetuses having a mutation in an endogenous prion nucleic acid and/or having one or more nucleic acids encoding a xenogenous antibody<br>
In alternative methods, chimeric embryos in which the majority of the placental tissue is from one genetic source and the majority of the fetal tissue is from another genetic source are generated. These chimeric embryos may have fewer placental abnormalities and thus may have an increased survival rate. In one such method, cells from an in vitro fertilized or naturally-occurring embryo are contacted with cells from an embryo that has a mutation in an endogenous prion nucleic acid and optionally has one or more nucleic acids encoding a xenogenous antibody and that is produced using traditional nuclear transfer methods or any of the other cloning methods described herein. For example, cells from an in vitro fertilized embryo can be injected into the periphery of a nuclear transfer embryo (e.g., between the zona pellucida and the embryo itself). This method was used to produce chimeric embryos without xenogenous antibody genes that had a 67% survival rate at day 40 compared to a 25% survival rate for control nuclear transfer embryos. Similar results are expected using cells from nuclear transfer embryos having a mutation in an endogenous prion nucleic acid and/or having one or more nucleic acids encoding a xenogenous antibody. In an alternative method, cells from a precompaction, in vitro fertilized or naturally-occurring embryo are incubated with cells from a precompaction nuclear transfer embryo under conditions that allow cells from each embryo to reorganize to produce a single chimeric embryo (Wells and Powell, Cloning 2:9-22, 2000). In both methods, the cells from the in vitro fertilized or naturally-occurring embryo are preferentially incorporated into the placenta, and the cells from the nuclear transfer method are preferentially incorporated into the fetal tissue.<br>
State of the art prior to present invention<br>
While human Ig has been expressed in mice, it was unpredictable whether human Ig will be fractionally rearranged and expressed in bovines, or other ungulates, because of differences in antibody gene structure, antibody production mechanism, and B-cell function. In particular, unlike mice, cattle and sheep differ from humans in their immunophysiology (Lucier et al, J. Immunol. 161:5438, 1998; Parng et al, J. Immunol. 157:5478, 1996; and Butler, Rev. Sci. Tech. 17:43, 2000). For example, antibody gene diversification in bovines and ovines relies much more on gene conversion than gene rearrangement as in humans and mice. Also, the primary location of B-cells in humans and mice is in the bone marrow, whereas in bovines and ovines B-cells are located in the illeal Peyer's patch. Consequently, it would have been difficult, if not impossible, prior to the present invention, to predict whether immunoglobulin rearrangement and diversification of .a human immunoglobulin loci would take place within the bovine (or other ungulate) B-cell lineage. In addition, it would also have been unpredictable whether a bovine would be able to survive, i.e., elicit its normal immune functions, in the absence of its endogenous Ig or with interference from human antibodies. For example, it was not certain if bovine B-cells expressing human Ig would correctly migrate to the illeal Peyer's Patch in bovines because this does not happen in humans. Also, it is not clear if human Fc receptor function, which mediates complement activation, induction of cytokine release, and antigen removal, would be normal in a bovine system. It was unpredictable whether such ungulates would survive because it is uncertain whether human Igs will be functionally expressed, or expressed in sufficient amounts to provide for adequate immune responses. Also, it was uncertain whether human chromosomes wouldl be stably maintained in transgenic ungulates.<br>
While the approaches to be utilized in the invention have been described above, the techniques that are utilized are described in greater detail below. These examples are provided to illustrate the invention, and should not be<br>
construed as limiting. In particular, while these examples focus on transgenic bovines, the methods described may be used to produce and test any transgenic ungulate.<br>
EXAMPLE 1: Transgenic ungulates having reduced prion protein activity<br>
Bovine fibroblast cell clones in which both alleles of the PrP locus are mutated were generated by sequential homologous recombination. The DNA construct for generating the PrP homozygous knockout cells was used to prevent transcription of functional, full-length PrP mRNA by inserting both a neomycin or puromycin-resistance gene (neo orpuro, described in Example 3) and a transcription termination cassette (STOP) in exon 3 just behind its initial ATG codon. Thus, the resulting immature PrP transcripts lack the functional coding domain. The DNA construct containing neo gene (i.e., the PrPKOneo vector) was firstly electroporated into bovine fibroblast cell lines, and then neomycin-resistant colonies were isolated. Based on PCR analysis, homologous recombination in the exon 3 occurred in some colonies. Thus, bovine fibroblast cell lines in which one allele of PrP locus is mutated were generated. From the hemizygously mutated (hemi-PrPKO) fibroblasts, three fetuses in which one allele of PrP locus is mutated were generated and the hemi-PrPKO fibroblast cell lines were reestablished. Then another DNA construct containingpuro gene (i.e., the PrPKOpuro vector) was secondly electroporated into the hemi-PrPKO fibroblasts, and then puromycin-resistant colonies were isolated. Based on PCR analysis, homologous recombination in the exon 3 of the remaining allele occurred in some clones at a frequency of ~5 %. In this way, bovine fibroblast clones in which both alleles of PrP locus are mutated (homo-PrPKO) were generated.<br>
From the homozygously mutated (homo-PrPKO) fibroblasts, ten fetuses in which both alleles of PrP locus are mutated were generated and the homo-PrPKO fibroblast cell lines were reestablished. In these homo-PrP KO fibroblasts, the complete absence of PrP expression was confirmed by RT-PCR<br>
analysis. The homo-PrPKO fibroblasts can be also generated by a combination of the same one knockout vector and a higher concentration of antibiotic to select for homozygous knockout cells. By using nuclear transfer, homo-PrPKO calves will be generated from the homo-PrPKO fibroblasts. These methods are described further below.<br>
Construction of a prion protein knockout vector<br>
The prion protein knockout vector was generated as follows (Fig. 44A). To isolate genomic DNA around exon 3 of the prion protein gene, a DNA probe was PCR amplified using the following primer pair 5'-CCA CAT AGG CAG TTG GAT CC-3' (SEQ ID NO: 69) and 5'-ATA AGA GGC CTG CTC ATG GC-3' (SEQ ID NO: 70). Using this probe, a bovine (Holstein) genomic A, phage library was screened, and four positive A, phage clones were identified. One clone out of the four clones was analyzed further by restriction mapping. The 8.3 kb BamHl genomic fragment (3' homologous arm) containing exon 3 and the 1.2 kb BamHl-BgRl fragment were subcloned into pBluescript II SK(-), and then neo and STOP cassettes were inserted at the Bam HI site, which is just behind its initial ATG codon. The Bglll site was converted to blunt ends, and a M?/I-linker was inserted at this site. A diphtheria toxin gene (DT-A, Gibco) was then added to the newly created Notl site. DT-A was inserted in the reverse orientation relative to the neomycin-resistance gene in the targeting cassette to kill cells in which the targeting cassette was randomly integrated in the genome (pBPrP(H)KOneo vector). A similar knockout vector containing a puro gene was also constructed (pBPrP(H)KOpuro vector; Fig. 44B). These prion protein knockout vectors can be linearlized using the Srf I restriction enzyme (TOYOBO) so that the pBluescript II SK(-) can be attached to the DT-A gene.<br>
Transfection/knockout procedures<br>
Transfection of fetal male Holstein fibroblasts cell lines was performed using a standard electroporation protocol as follows. The medium used to culture the bovine fetal fibroblasts contained 500 ml Alpha MEM (Gibco, 12561-049), 50 ml fetal calf serum (Hy-Clone #ABL13080), 5 ml penicillin-streptomycin (Sigma), and 1 ml 2-mercaptoethanol (Gibco/BRL #21985-023). On the day prior to transfection, cells were seeded on a T175 tissue culture flask with a confluency of 80-100%, as determined by microscopic examination. On the day of transfection, about 107 bovine fibroblasts cells were trypsinized and washed once with alpha-MEM medium. After resuspension of the cells in 800 ul of alpha-MEM, 30 |ig of the pBprion protein(H)KOneo vector which was already dissolved in HBS (Hepes buffer saline) containing 1 mM spermidine, was added to the cell suspension and mixed well by pipetting. The cell-DNA suspension was transferred into an electroporation cuvette and electroporated at 550 V and 50 u,F. After that, the electroporated cells were plated onto sixty 24-well plates with the alpha-MEM medium supplemented with the serum. After a 48 hour-culture, the medium was replaced with medium containing 500 ug/ml of G418, and the cells were cultured for 2-3 weeks to select neomycin resistant cells. After selection, all colonies that reached close to 100% confluency were picked and divided in replica plates (24-well plates). Cells in one plate were used for genomic DNA extraction, and cells in the other plate were used for nuclear transfer. Genomic DNA was extracted from the colonies to screen for the desired homologous recombination events by PCR. Typically, about 100 G418-colonies were analyzed.<br>
Screening for targeted integrations<br>
As described above, the genomic DNA was extracted from each 24-well independently using the PUREGENE DNA isolation Kit (Gentra Systems) according to the manufacturer's protocol. Each genomic DNA sample was<br>
fesuspended in 20 ul of 10 mM Tris-Cl (pH 8.0) and 1 mM EDTA (EDTA). Screening by PCR was performed using the following primer pair F7 (5'-TGT CAA AGA GAC ACT CCT CAT TTG TCT TCC-3', SEQ ID NO: 71) and R7 (5'-TCA TAG CCG AAT AGC CTC TCC ACC C-3', SEQ ID NO: 72). The sequence of one primer is located in the knockout vector, and the sequence of the other primer is located just outside of the integrated vector in the targeted endogenous locus (Fig. 44A). Therefore, the expected PCR product should be detected only when the knockout vector is integrated into the targeted locus by homologous recombination. The PCR reaction mixtures contained 17.9 ul water, 3 ul of 1 OX LA PCR buffer II (Mg2+ plus), 4.8 ul of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 2 ul of genomic DN A, and 0.3 ul of LA Taq. Thirty cycles of PCR were performed by incubating the reaction mixtures at the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, and 68C for 5 minutes. After PCR, the reaction mixtures were analyzed by electrophoresis. PCR products of the expected size were generated from approximately 50% of 94 screened neomycin-resistant clones (Fig. 44B). These positive clones were also confirmed by PCR-amplification another primer pair F10 using (5'-TGA AGA TGT CCC AGT GCT GGG GAT-3', SEQ ID NO: 73) and RIO (5'-GAG TTA GGG GCG GGA CTA TGG TTG C-3', SEQ ID NO: 74), giving PCR products of the expected size of 2.7 kb. The high frequency of homologous recombination of more than 50% was reproduced in another fibroblast cell line, female Fl (Holstein x Jersey) cell line (Fig. 44C). Thus, bovine fibroblast cell lines in which one allele of the prion locus is mutated by the knockout vector were successfully generated, irrespective of the initial cell line used.<br>
Chromatin transfer<br>
The hemizygous prion knockout cells or the nuclei from these cells can be used in any of the nuclear transfer methods described herein to generate a hemizygous prion knockout ungulate (e.g., a knockout bovine calf). If desired,<br>
cells from a knockout fetus or live ungulate can be used as described below to generate homozygous prion knockout cells. In one particular method, fibroblasts (e.g., bovine primary fetal fibroblasts) were synchronized in mitosis with 1 fig/ml nocodazole for 18 hours, harvested by mitotic shake-off, and washed twice in phosphate buffered saline and once in cell lysis buffer (20 mM Hepes, pH 8.2, 5 mM MgCl2, 10 mM EDTA, 1 mM DTT and protease inhibitors). Sedimented cells were resuspended in one volume of ice-cold cell lysis buffer, allowed to swell on ice for one hour, and Dounce-homogenized using a tight-fitting glass pestle. The lysate was centrifuged at 15,000 x g for 15 minutes at 4C, and the supernatant (mitotic extract) was aliquoted, frozen in liquid nitrogen, and stored at -80C. Fresh or frozen extracts were used. In vzYro-matured oocytes were enucleated at 20 hpm. Transfected bovine fetal fibroblasts from selected colonies were washed in Ca2+/Mg2+-free Hank's Balanced Salt Solution (HBSS) and permeabilized by incubation of cells in suspension with 31.2 U Streptolysin O (SLO; Sigma) in 100 (il HBSS for 30 minutes in an approximately 37C water bath. Permeabilization was assessed by uptake of the membrane impermeant DNA stain, propidium iodide (0.1 u/ml). Permeabilized fibroblasts were sedimented, washed, and incubated in 40 ul mitotic extract containing an ATP-generating system (1 mM ATP, 10 mM creatine phosphate, and 25 ng/ml creatine kinase) for 30-45 minutes at approximately 37C. Aliquots were labeled with 0.1 ug/ml Hoechst 33342 to monitor chromatin condensation. At the end of incubation, the reaction mixture was diluted with 500 u.1 Alpha MEM/10% fetal bovine serum (Hyclone). Cells were fused to enucleated oocytes; oocytes were activated at 28 hpm, and embryos cultured to the blastocyst stage in vitro. Two embryos were transferred per recipient female. Pregnancies were monitored by ultrasonography, and C-sections were performed on recipients to recover fetuses for cell line production.<br>
Second transfection/knockout procedures<br>
Transfection of the hemi-PrPKO fetal fibroblasts cell lines where the first KO vector is already integrated into "allele A" was performed as follows. The medium used to culture the bovine fetal fibroblasts contained 500 ml Alpha MEM (Gibco, 12561-049), 50 ml fetal calf serum (Hy-Clone #ABL 13080), 5 ml penicillin-streptomycin (SIGMA), and 1 ml 2-mercaptoethanol (Gibco/BRL #21985-023). On the day prior to transfection, cells were seeded on a T175 tissue culture flask with a confluency of 80-100 %, as determined by microscopic examination. On the day of transfection, about 107 bovine fibroblasts cells were trypsinized and washed once with alpha-MEM medium. After resuspension of the cells in 800 |il of alpha-MEM, 30 |j.g of the KO vector {pBPrP(H)KOpuro vector} which was already dissolved in HBS (Hepes buffer saline) containing 1 mM spermidine was added to the cell suspension and mixed well by pipetting. The cell-DNA suspension was transferred into an electroporation cuvette and electroporated at 550 V and 50 uJ7. After that, the electroporated cells were plated onto thirty 48-well plates with the alpha-MEM medium supplemented with the serum. After a 48 hour-culture, the medium was replaced with medium containing 1 jag/ml of puromycin, and the cells were cultured for 2-3 weeks to select puromycin resistant cells. After selection, all colonies which reached close to 100 % confluency were picked up and divided to replica plates (24-well and 48-well plates); one was for genomic DNA extraction, the other for nuclear transfer. Genomic DNA was extracted from the colonies to screen for the desired homologous recombination events by PCR.<br>
Screening for homozygously targeted integrations<br>
As described above, the genomic DNA was extracted from each 24-well independently using the PUREGENE DNA isolation Kit (Centra Systems) according to the manufacturer's protocol. Each genomic DNA sample was resuspended in 20 ul of 10 mM Tris-Cl (pHS.O) and 1 mM EDTA (EOTA).<br><br>
Screening by PCR wase performed using the following primer pair F14; 5'-TTG CTC CAC CTT CCG TCT TCT GTC-3' (SEQ ID NO: 3) and R14; 5'-GTT GGC GCC TAG CGG TGG ATG TG-3' (SEQ ID NO: 4). The sequence of one primer is located in the KO vector, and the sequence of the other primer is located just outside of the integrated vector in the targeted endogenous locus (Fig. 1). Therefore, the expected PCR product should be detected only when the KO vector is integrated into the targeted locus by homologous recombination. The PCR reaction mixtures contained 17.9 (il water, 3 |il of 10X LA PCR buffer II (Mg2+ plus), 4.8 ul of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 2 |al of genomic DNA, and 0.3 ul of LA Taq. Thirty cycles of PCR were performed by incubating the reaction mixtures at the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, and 68C for 5 minutes. After PCR, the reaction mixtures were analyzed by electrophoresis. As a result of screening of 332 clones, 15 clones (#26, 86, 113, 116, 118, 173, 177, 213, 235, 250, 251, 266, 285, 318, 330) showed the expected PCR products, confirmed by sequencing the PCR product. Based on the polymorphic sequence, it turned out that the KO vector was integrated into "allele B" of the ex on 3 of PrP locus in all the clones. The eight clones were subjected to nuclear transfer to generate fetuses and three fetuses (#1395, 1468, 1476) from #285 and seven fetuses (#1439-1, 1439-2, 1487, 3583, 3632, 3682-1, 3682-2, 4961) from #113 proved to be homozygous PrP KO where the puroKO vector and the neoKO vector are integrated into "allele B" and "allele A," respectively. Thus, ten bovine fibroblast cell lines in which both alleles of the PrP locus are mutated by the KO vectors were successfully generated.<br>
able 1 summarizes the data pertaining to pregnancies following chromatin transfer using homozygous PrP KO clones derived from hemizygous PrP KO cell lines.<br><br>
Verification of absence of PrP expression in the homo-PrP fetuses<br>
From some fetuses, fibroblasts were established and total RNA was<br>
extracted by using RNeasy Mini kit (QIAGEN). One microliter of total RNA was subjected to first-strand cDNA synthesis (Superscript First-Strand Synthesis System for RT-PCR; Invitrogen), followed by RT-PCR. The RT-PCR was carried out as below. The primers used were 5'-AAG AAG CGA CCA AAA CCT GG-3' and 5'-GTA ACG GTG CAT GTT TTC ACG-3'. The PCR reaction mixtures contained 32.5 fil water, 5 fil of 10X Ex Taq buffer (TAKARA), 8 ul of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 2 [il of the first-strand cDNA, and 0.5 |il of Ex Taq (TAKARA). Thirty five cycles of PCR were performed by incubating the reaction mixtures at the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 62C for 30 seconds and 72C for 1 minute. After PCR, the reaction mixtures were analyzed by electrophoresis. Whereas the clear PCR products were detected in both wild-type and hemi-PrP KO cells, there were no positive PCR products in any of the ten homo-PrP KO cell lines. This result verifies the complete absence of PrP expression in the homo-PrP KO cell lines.<br>
EXAMPLE 2: Introduction and rearrangement of HAC Summary of procedures for insertion of HACs<br>
For the generation of ungulates (e.g., bovines) that have a mutation in one or both alleles of the prion gene and that express xenogenous antibody, standard methods may be used to insert a nucleic acid encoding a xenogenous<br>
antibody (e.g., a HAC) into a cell. For example, a HAC can be inserted before or after one or both alleles of the prion gene are inactivated. If desired, one or more endogenous antibody genes may also be mutated in the cell. The cell is then used in standard nuclear transfer procedures to produce the desired transgenic ungulate, as described in more detail below.<br>
Essentially, male and female bovine fetal fibroblast cell lines containing human artificial chromosome sequences (e.g., #14fg., #2fg., and #22fg.) are obtained and selected and used to produce cloned calves from these lines.<br>
For example, HACs derived from human chromosome #14 ("#14fg," comprising the Ig heavy chain gene), human chromosome #2 ("#2fg," comprising the Ig kappa chain gene) and human chromosome #22 ("#22fg," comprising the Ig lambda chain gene) can be introduced simultaneously or successively.<br>
The transmission of these chromosome fragments is tested by mating a male #14fg. animal to female #2fg. and #22fg. animals and evaluating offspring. If transmission is successful then the two lines are mated to produce a line containing all three chromosome fragments.<br>
Also, #14fg., #2fg., and #22fg. chromosome fragments may be inserted into fetal cells (e.g., transgenic Homo H/L fetal cells) and used to generate cloned calves or cross transgenic HAC calves with other transgenic calves (e.g., Homo H/L calves). Alternatively, other HACs, such as AHAC or AAHAC, may be introduced as described below or introduced using any other chromosome transfer method.<br>
Rationale<br>
Germline transmission of HACs should be useful for introducing the HACs into the animals (e.g., Ig knockout animals) and in propagating animals in production herds. The concern in propagation of HACs through the germline is incomplete pairing of chromosomal material during meiosis. However,<br>
germline transmission has been successful in mice as shown by Tomizuka et al. (Proc. Natl. Acad. Sci. USA, 97:722, 2000).<br>
The strategy outlined in Fig. 1A consists of inserting #14fg. into a male line of cells and #2fg. and #22fg. each into female cell lines. Calves retaining a HAC are produced and germline transmission can be tested both through females and males. Part of the resulting offspring (-25%) should contain both heavy and light chain HACs. Further crossing should result in a line of calves containing all three chromosomal fragments. These animals are optionally used for crossing with transgenic animals (e.g., Homo H/L animals), produced from fetal cells as previously described.<br>
Experimental design<br>
Cells are obtained from the original screening of cell lines. These may be Holstein or different lines than those used above. This allows crossing while maintaining as much genetic variation in the herd as possible. Introduction of HACs into cell lines and selection of positive cell lines is then effected. Selected cell lines are used for nuclear transfer and calves are produced. Starting at 12 months of age semen and eggs are collected, fertilized, and transferred into recipient animals. Cell samples are taken for DNA marker analysis and karyotyping. Beginning at birth, blood samples are taken and analyzed for the presence of human Ig proteins.<br>
As indicated above, HACs are also optionally transferred into Homo H/L cell lines using the procedures developed in the above experiments.<br>
The following examples are additionally provided as further exemplification of the invention.<br>
Exemplary procedures for insertion of HACs<br>
Additional experiments were carried out to demonstrate that xenogenous (e.g., human) immunoglobulin heavy chain (mu) and lambda light chain may be produced by a bovine host, either alone or in combination. In addition,<br>
these experiments demonstrated that the human immunoglobulin chains were rearranged and that polyclonal sera was obtained. In these procedures, immunoglobulin-expressing genes were introduced into bovine fibroblasts using human artificial chromosomes. The fibroblasts were then utilized for nuclear transfer, and fetuses were obtained and analyzed for antibody production. These procedures and results are described in more detail below. Similar results are expected from fibroblasts that are isolated from a fetus with a mutation in one or both alleles of the prion gene and that are used in the methods described below for the introduction of a HAC. Alternatively, a HAC can be introduced into a cell, and then a mutation can be introduced (i) into the prion gene of the cell with the HAC or (ii) into the prion gene of a cell from a cloned fetus generated using the cell with the HAC.<br>
HAC constructs<br>
The human artificial chromosomes (HACs) were constructed using a previously described chromosome-cloning system (Kuroiwa et al., Nature Biotech. 18: 1086-1090,2000). Briefly, for the construction of AHAC, the previously reported human chromosome 22 fragment (hChr22) containing a loxP sequence integrated at the HCF2 locus was truncated at the AP000344 locus by telomere-directed chromosomal truncation (Kuroiwa et a/., Nucleic Acid Res., 26: 3447-3448, 1998). Next, cell hybrids were formed by fusing the DT40 cell clone containing the above hChr22 fragment (hCF22) truncated at the AP000344 locus with a DT40 cell clone (denoted "R clone") containing the stable and germline-transmittable human minichromosome SC20 vector. The SC20 vector was generated by inserting a loxP sequence at the RNR2 locus of the S20 fragment. The SC20 fragment is a naturally-occurring fragment derived from human chromosome 14 that includes the entire region of the human Ig heavy chain gene (Tomizuka et al., Proc. Natl. Acad. Sci. USA 97:722, 2000). The resulting DT40 cell hybrids contained both hChr fragments. The DT40 hybrids were transfected with a Cre recombinase-<br>
expression vector to induce Cre/loxP-mediated chromosomal translocation between HCF22 and the SC20 vector. The stable transfectants were analyzed using nested PCR to confirm the cloning of the 2.5 megabase hChr22 region, defined by the HCF2 and AP000344 loci, into the loxP-cloning site in the SC20 vector. The PCR-positive cells that were expected to contain AHAC were then isolated by FACS sorting based on the fluorescence of the encoded green fluorescent protein. Fluorescent in situ hybridization (FISH) analysis of the sorted cells was also used to confirm the presence of AHAC, which contains the 2.5 megabase hChr22 insert.<br>
Similarly, AAHAC was also constructed using this chromosome-cloning system. The hChr22 fragment was truncated at the AP000344 locus, and then the loxP sequence was integrated into the AP000553 locus by homologous recombination in DT40 cells. The resulting cells were then fused with the R clone containing the SC20 minichromosome vector. The cell hybrids were transfection with a Cre-expression vector to allow Cre/loxP-mediated chromosomal translocation. The generation of AAHAC, which contains the 1.5 megabase hChr22 insert, defined by the AP000553 and AP000344 loci, was confirmed by PCR and FISH analyses.<br>
The functionality of AHAC and AAHAC in vivo was assessed by the generation of chimeric mice containing these HACs. These HACs were individually introduced into mouse embryonic stem (ES) cells, which were then used for the generation of chimeric mice using standard procedures (Japanese patent number 2001-142371; filed May 11, 2000). The resulting mice had a high degree of chimerism (85-100% of coat color), demonstrating a high level of pluripotency of the ES cells containing these HACs and the mitotic stability of these HACs in vivo. Furthermore, AHAC was transmitted through the germline of the AHAC chimeric mouse to the next offspring, demonstrating the meiotic stability of this HAC.<br>
Chicken DT40 cells retaining these HACs have been deposited under the Budapest treaty on May 9, 2001 in the International Patent Organism<br>
Depository, National Institute of Advanced Industrial Science and Technology, Tsukuba Central 6, 1-1, Higashi 1-Chome Tsukuba-shi, Ibaraki-ken, 305-8566 Japan. The depository numbers are as follows: AHAC (PERM BP-7582), AAHAC (PERM BP-7581), and SC20 fragment (PERM BP-7583). Chicken DT40 cells retaining these HACs have also been deposited in the Food Industry Research and Development Institute (FIRDI) in Taiwan. The depository numbers and dates are as follows: AHAC (CCRC 960144; November 9, 2001), AAHAC (CCRC 960145; November 9, 2001), and SC20 fragment (the cell line was deposited under the name SC20 (D); CCRC 960099; August 18, 1999).<br>
The 2.5 megabase (Mb) hChr22 insert in AHAC is composed of the following BAG contigs, which are listed by their GenBank accession number: AC002470, AC002472, AP000550, AP000551, AP000552, AP000556, AP000557, AP000558, AP000553, AP000554, AP000555, D86995, D87019, D87012, D88268, D86993, D87004, D87022, D88271, D88269, D87000, D86996, D86989, D88270, D87003, D87018, D87016, D86999, D87010, D87009, D87011, D87013, D87014, D86991, D87002, D87006, D86994, D87007, D87015, D86998, D87021, D87024, D87020, D87023, D87017, AP000360, AP00361, AP000362, AC000029, AC000102, U07000, AP000343, and AP000344. The 1.5 Mb hChr22 insert in AAHAC is composed of the following BAG contigs: AP000553, AP000554, AP000555, D86995, D87019, D87012, D88268, D86993, D87004, D87022, D88271, D88269, D87000, D86996, D86989, D88270, D87003, D87018, D87016, D86999, D87010, D87009, D87011, D87013, D87014, D86991, D87002, D87006, D86994, D87007, D87015, D86998, D87021, D87024, D87020, D87023, D87017, AP000360, AP00361, AP000362, AC000029, AC000102, U07000, AP000343, and AP000344 (Dunham et al, Nature 402:489-499, 1999).<br>
Generation of bovine fetal fibroblasts<br>
To generate bovine fetal fibroblasts, day 45 to 60 fetuses were collected from disease-tested Holstein or Jersey cows housed at Trans Ova (Iowa), in which the pedigree of the male and female parents were documented for three consecutive generations. The collected fetuses were shipped on wet ice to Hematech's Worcester Molecular Biology Division for the generation of primary fetal fibroblasts. Following arrival, the fetuses were transferred to a non-tissue culture grade, 100 mm plastic petri dish in a tissue culture hood. Using sterile forceps and scissors, the extraembryonic membrane and umbilical cord were removed from the fetus. After transferring the fetus to a new plastic petri dish, the head, limbs and internal organs were removed. The eviscerated fetus was transferred to a third petri dish containing approximately 10 ml of fetus rinse solution composed of: 125 ml IX Dulbecco's-PBS (D-PBS) with Ca2+ and Mg 2+ (Gibco-BRL, cat#. 14040); 0.5 ml Tylosine Tartrate (8 mg/ml, Sigma, cat#. T-3397); 2 ml Penicillin-Streptomycin (Sigma, cat#. P-3539); and 1 ml Fungizone (Gibco-BRL, cat#. 15295-017) (mixed and filtered through a 0.2 um nylon filter unit [Nalgene, cat#. 150-0020).<br>
Each fetus was washed an additional three times with the fetus rinse solution to remove traces of blood, transferred to a 50 ml conical tissue culture tube, and finely minced into small pieces with a sterile scalpel. The tissue pieces were washed once with IX D-PBS without Ca2+ and Mg 2+ (Gibco-BRL, cat#. 14190). After the tissue pieces settled to the bottom of the tube, the supernatant was removed and replaced with approximately 30 ml of cell dissociation buffer (Gibco-BRL, cat#. 13151-014). The tube was inverted several times to allow mixing and incubated at 38.5C/5% CO2 for 20 minutes in a tissue culture incubator. Following settling of the tissue to the bottom of the tube, the supernatant is removed and replaced with an equivalent volume of fresh cell dissociation buffer. The tissue and cell dissociation buffer mixture was transferred to a sterile, 75 ml glass trypsinizing flask (Wheaton Science Products, cat#. 355393) containing a 24 mm, round-ended, spin bar. The flask<br>
was transferred to a 38.5C/5% C02 tissue culture incubator, positioned on a magnetic stir plate, and stirred at a sufficient speed to allow efficient mixing for approximately 20 minutes. The flask was transferred to a tissue culture hood; the tissue pieces allowed to settle, followed by removal of the supernatant and harvesting of the dissociated cells by centrifugation at 1,200 rpm for five minutes. The cell pellet was re-suspended in a small volume of complete fibroblast culture media composed of: 440 ml alpha MEM (BioWhittaker, cat# 12-169F); 50 ml irradiated fetal bovine serum; 5 ml GLUTAMAX-I supplement (Gibco-BRL, cat# 25050-061); 5 ml Penicillin-Streptomycin (Sigma, cat# P-3539); 1.4 ml 2-mercaptoethanol (Gibco-BRL, cat# 21985-023) (all components except the fetal bovine serum were mixed were filtered through 0.2 urn nylon filter unit [Nalgene, cat# 151-4020]), and stored on ice. The dissociation process was repeated three additional times with an additional 30 ml of cell dissociation solution during each step. Cells were pooled; washed in complete fibroblast media; passed sequentially through 23 and 26 gauge needles, and finally through a 70 urn cell strainer (B-D Falcon, cat# 352350) to generate a single cell suspension. Cell density and viability were determined by counting in a hemacytometer in the presence of trypan blue (0.4% solution, Sigma, cat#. T-8154).<br>
Primary fibroblasts were expanded at 38.5C/5% CO2 in complete fibroblast media at a cell density of 1 x 106 viable cells per T75 cm2 tissue culture flask. After 3 days of culture or before the cells reached confluency, the fibroblasts were harvested by rinsing the flask once with IX D-PBS (without Ca2+ and Mg2+) and incubating with 10 ml of cell dissociation buffer for 5 to 10 minutes at room temperature. Detachment of cells was visually monitored using an inverted microscope. At this step, care was taken to ensure that cell clumps were disaggregated by trituration. After washing and quantitation, the dissociated fibroblasts were ready for use in gene targeting experiments.   These cells could also be cryopreserved for long-term storage.<br>
Introduction of HAGs into bovine fetal fibroblasts<br>
AHAC and AAHAC were transferred from the DT40 cell hybrids to Chinese hamster ovary (CHO) cells using microcell-mediated chromosome transfer (MMCT) (Kuroiwa et al. Nature Biotech. 18: 1086-1090, 2000).   The CHO clone containing AHAC ("D15 clone") was cultured in F12 (Gibco) medium supplemented with 10% FBS (Gibco), 1 mg/ml G418, and 0.2 mg/ml hygromycin B at 37C and 5% C02. The D15 clone was expanded into twelve T25 flasks. When the confluency reached 80-90%, colcemid (Sigma) was added to the medium at a final concentration of 0.1 ng/ml. After three days, the medium was exchanged with DMEM (Gibco) supplemented with 10 (ig/ml of cytochalacin B (Sigma). The flasks were centrifuged for 60 minutes at 8,000 rpm to collect microcells. The microcells were purified through 8-, 5-, and 3-um filters (Costar) and then resuspended in DMEM medium. The microcells were used for fusion with bovine fibroblasts as described below.<br>
Bovine fetal fibroblasts were cultured in a-MEM (Gibco) medium supplemented with 10% FBS (Gibco) at 37C and 5% CO2. The fibroblasts were expanded in a T175 flask. When the confluency reached 70-80%, the cells were detached from the flask with 0.05% trypsin. The fibroblast cells were washed twice with DMEM medium and then overlayed on the microcell suspension. After the microcell-fibroblast suspension was centrifuged for five minutes at 1,500 rpm, PEG 1500 (Roche) was added to the pellet according to the manufacturer's protocol to enable fusion of the microcells with the bovine fibroblasts. After fusion, the fused cells were plated into six 24-well plates and cultured in a-MEM medium supplemented with 10% FBS for 24 hours. The medium was then exchanged with medium containing 0.7 mg/ml of G418. After growth in the presence of the G418 antibiotic for about two weeks, the G418 resistant, fused cells were selected. These G418-resistant clones were used for nuclear transfer, as described below.<br>
Similarly, AAHAC from the CHO clone C13 was transferred into bovine fetal fibroblasts by means of MMCT. The selected G418-resistant clones were used for nuclear transfer.<br>
Nuclear transfer, activation, and embryo culture<br>
The nuclear transfer procedure was carried out essentially as described earlier (Cibelli et al., Science 1998: 280:1256-1258). In vitro matured oocytes were enucleated about 18-20 hours post maturation (hpm) and chromosome removal was confirmed by bisBenzimide (Hoechst 33342, Sigma) labeling under UV light. These cytoplast-donor cell couplets were fused, by using single electrical pulse of 2.4 kV/cm for 20 |^sec (Electrocell manipulator 200, Genetronics, San Diego, CA). After 3-4 hrs, a random sub-set of 25% of the total transferred couplets was removed, and the fusion was confirmed by bisBenzimide labeling of the transferred nucleus. At 30 hpm reconstructed oocytes and controls were activated with calcium ionophore (5 |J.M) for 4 minutes (Cal Biochem, San Diego, CA) and 10 (ig Cycloheximide and 2.5 fig Cytochalasin D (Sigma) in ACM culture medium for 6 hours as described earlier (Lin et al., Mol. Reprod. Dev. 1998: 49:298-307; Presicce et al., Mol. Reprod. Dev. 1994:38:380-385). After activation eggs were washed in HEPES buffered hamster embryo culture medium (HECM-Hepes) five times and placed in culture in 4-well tissue culture plates containing irradiated mouse fetal fibroblasts and 0.5 ml of embryo culture medium covered with 0.2 ml of embryo tested mineral oil (Sigma). Twenty five to 50 embryos were placed in each well and incubated at 38.5C in a 5% CO2 in air atmosphere. On day four 10% PCS was added to the culture medium.<br>
Embryo transfer<br>
Day 7 and 8 nuclear transfer blastocysts were transferred into day 6 and 7 synchronized recipient heifers, respectively. Recipient animals were synchronized using a single injection of Lutalyse (Pharmacia &amp; Upjohn, Kalamazoo, MI) followed by estrus detection. The recipients were examined<br>
on day 30 and day 60 after embryo transfer by ultrasonography for the presence of conceptus and thereafter every 30 days by rectal palpation until 270 days. The retention of a HAC in these bovine fetuses is summarized in Table 2 and is described in greater detail in the sections below.<br>
Table 2: Summary of HAC retention in bovine fetuses<br><br>
Introduction of a HAC containing a fragment of chromosome #14<br>
The SC20 fragment, a human chromosome #14 fragment ("hchr.Hfg," containing the Ig heavy chain gene), was introduced into fetal fibroblast cells in substantially the same manner as described above. Any other standard chromosome transfer method may also be used to insert this HAC or another HAC containing a human Ig gene into donor cells. The resulting donor cells may be used in standard nuclear transfer techniques, such as those described above, to generate transgenic ungulates with the HAC.<br>
The pregnancy status of the 28 recipients to whom cloned embryos were transferred from cells containing the hchr.!4fg was checked by ultrasonography. The results are summarized in Table 3.<br>
If desired, a cell from the resulting HAC fetus or HAC offspring can be used in a second round of nuclear transfer to generate additional cloned offspring. Cells from the initial HAC fetus or HAC offspring may also be frozen to form a cell line to be used as a source of donor cells for the generation of additional HAC ungulates.<br>
Table 3: Pregnancy at 40 days using donor cells containing hchr.!4fg<br><br>
The pregnancy rates were lower than anticipated. This is believed to be attributable to extremely abnormally hot weather during embryo transfer.<br>
As illustrated in Fig. 27, pregnancy rates for HAC carrying embryos appear to be equivalent to non-transgenic cloned pregnancies. One recipient carrying a AAHAC calf gave birth recently to a live healthy calf. Others will be born over the next several months.<br>
Demonstration of rearrangement and expression of human heavy chain locus in a AHAC bovine fetus<br>
Cloned AHAC-transgenic bovine fetuses were removed at various gestational days and analyzed for the presence, rearrangement, and expression of the human immunoglobulin loci. Analysis of genomic DNA and cDNA obtained by RT-PCR from spleen and nonlymphoid tissues (liver and brain) of<br>
one of these fetuses indicated the presence, rearrangement, and expression of the AH AC.<br>
Presence of human heavy and/or light chain in AHAC fetuses<br>
To determine whether the human heavy and light chains were retained in AHAC fetuses, liver DNA was isolated from AHAC fetuses and analyzed by PCR for the presence of genomic DNA encoding human heavy and light chains.<br>
For the detection of genomic heavy chain DNA, the following primers were used: VH3-F 5'-AGT GAG ATA AGC AGT GGA TG-3' (SEQ ID NO: 1) and VH3-R 5'-CTT GTG CTA CTC CCA TCA CT-3' (SEQ ID NO: 2). The primers used for detection of lambda light chain DNA were IgL-F 5-GGA GAC CAC CAA ACC CTC CAA A-3' (SEQ ID NO: 3) and IgL-R 5'-GAG AGT TGC AGA AGG GGT YGA CT-3' (SEQ ID NO: 4). The PCR reaction mixtures contained 18.9 ul water, 3 ul of 10X Ex Taq buffer, 4.8 jal of dNTP mixture, 10 pmol forward primer and 10 pmol of reverse primer, 1 ul of genomic DNA, and 0.3 fil of Ex Taq. Thirty-eight cycles of PCR were performed by incubating the reaction mixtures at the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 56C for 30 seconds, and 72C for 30 seconds.<br>
As shown in Fig. 5, fetuses #5968, 6032 and 6045 each contained both human heavy chain (u) and light chain (A.) loci. Fetus #5996 contained only the human heavy chain locus. Fetus #5983 did not contain the human heavy chain and may not have contained the human light chain. Fetus #5846 did not contain either human sequence. Thus, fetuses #5983 and 5846 may not have retained the HAC. These results suggested that AHAC can be stably retained up to gestational day 58 in bovines.<br>
Presence of human Cmu exons in AHAC fetus #5996<br>
Primers specific for a mRNA transcript including portions of Cmu 3 and Cmu 4 were used to determine whether AHAC was present and expressing transcripts encoding the constant region of the human mu locus of fetus #5996.<br>
For this RT-PCR analysis of the genomic constant region of the human mu heavy chain, primers "CH3-F1" (5'-ace ace tat gac age gtg ac-31, SEQ ID NO: 5) and "CH4-R2" (5'-gtg gca gca agt aga cat cg-3', SEQ ID NO: 6) were used to generate a RT-PCR product of 350 base pairs. This PCR amplification was performed by an initial denaturing incubation at 95C for five minutes. Then, 35 cycles of denaturation, annealing, and amplification were performed by incubation at 95C for one minute, 59C for one minute, and 72C for two minutes. Then, the reaction mixtures were incubated at 72C for 10 minutes. Rearranged bovine heavy chain was detected using primers I7L and P9, as described below (Fig. 7). As an internal control, levels of GAPDH RNA was detected using primers "GAPDH forward" (5'-gtc ate ate tct gcc cct tct g-31, SEQ ID NO: 7) and "GAPDH reverse" (5'-aac aac ttc ttg atg tea tea t-3', SEQ ID NO: 8). For this amplification of GAPDH RNA, samples were incubated at 95C for five minutes, followed by 35 cycles of incubation at 95C for one minute, 55C for one minute, and 72C for two minutes. Then, the mixtures were incubated at 72C for seven minutes.<br>
This analysis showed that RT-PCR analysis of the spleen of fetus #5996 produced a band (lane 3) matching the amplification products generated using control human spleen cDNA (lane 4) and cDNA obtained from a AHAC chimeric mouse (lane 5) (Fig. 6). No such band was detected in nonlymphoid tissues: bovine liver (lane 1) or bovine brain (lane 2). The capacity of these tissues to support RT-PCR was shown by the successful amplification of the housekeeping gene, GADPH, in both liver (lane 10 of Fig. 6) and brain (lane 6 of Fig. 7).<br>
Rearrangement of bovine heavy chain locus by 77 gestational days<br>
The AHAC fetus #5996 was tested to determine whether it had undergone the developmental processes necessary for the expression and activation of the recombination system required for immunoglobulin heavy chain locus rearrangement. For this analysis, standard RT-PCR analysis was performed to detect the presence of mRNA transcripts encoding mu-VH rearrangements. RNA isolated from the spleen, liver, and brain of fetus #5996 was analyzed by RT-PCR using primers "17L" (5'-ccc tec tct ttg tgc tgt ca-31, SEQ ID NO: 9) and "P9" (5'-cac cgt get etc ate gga tg-3', SEQ ID NO: 10). The PCR reaction mixtures were incubated at 95C for 3 minutes, and then 35 cycles of denaturation, annealing, and amplification were performed using the following conditions: 95C for one minute, 58C for one minute, and 72C for two minutes. The reaction mixture was then incubated at 72C for 10 minutes.<br>
Lane 5 of Fig. 7 shows that a product of the size expected for amplification of a rearranged bovine heavy chain (450 base pairs) was obtained. This product migrated to a position equivalent to that of a control bovine Cmu heavy chain cDNA known to contain sequences corresponding to rearranged bovine heavy chain transcripts (lane 7). As expected, the rearranged heavy chain was expressed in the spleen (lane 5), but absent from the brain (lane 2) and liver (lane 3) at this point in development.<br>
Rearrangement and expression of the human heavy chain locus in the AHAC fetus #5996<br>
The rearrangement and expression of the human heavy chain locus was demonstrated by the amplification of a segment of DNA including portions of Cmu and VH regions. Primers specific for RNA transcripts including portions of Cmu (Cmul) and VH (VH3-30) were used to determine if RNA transcripts containing rearranged human Cmu-VDJ sequences were present (Fig. 8).<br>
For this RT-PCR analysis, primers "Cmul" (5'-cag gtg cag ctg gtg gag tct gg-3', SEQ ID NO: 11) and "VH3-30" (5'-cag gag aaa gtg atg gag tc-31, SEQ<br>
ID NO: 12) were used to produce a RT-PCR product of 450 base pairs. This RT-PCR was performed by incubating reaction mixtures at 95 for 3 minutes, followed by 40 cycles of incubation at 95 for 30 minutes, 69 for 30 minutes, and 72 for 45 minutes, and one cycle of incubation at 72 for 10 minutes. This RT-PCR product was then reamplified with the same primers by one cycle of incubation at 95C for three minute, 40 cycles of incubation at 95C for one minute, 59C for one minute, 72C for one minute, and one cycle of incubation at 72C for 10 minutes. As an internal control, RT-PCR amplification of GAPDH was performed as described above.<br>
The gel in Fig. 8 shows that RT-PCR analysis of the spleen from fetus #5996 produced a band (lane 5) matching the amplification products generated using human spleen cDNA (lane 4) or AHAC chimeric mouse spleen cDNA (lane 1). No such band was detected in bovine liver (lane 2) or bovine brain (lane 3). As a positive control, amplification of GADPH RNA (lanes 8 and 9) showed the capacity of these tissues to support RT-PCR.<br>
Rearrangement and expression of the human heavy chain region in fetus #5996 was also demonstrated by RT-PCR analysis using primers CH3-F3 (5-GGA GAC CAC CAA ACC CTC CAA A-3'; SEQ ID NO: 13) and CH4-R2<br>
(5-GTG GCA GCA AGT AGA CAT CG-3'; SEQ ID NO: 14). These PCR reaction mixtures contained 18.9 |al water, 3 ul of 10X Ex Taq buffer, 4.8 fil of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 1 fil of cDNA, and 0.3 fil of Ex Taq. Forty PCR cycles were performed by incubating the reaction mixtures under the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 60C for 30 seconds, and 72C for 30 seconds.<br>
As shown in lanes 6 and 7 of Fig. 9, an amplified sequence from the spleen of fetus #5996 was the same size as the spliced constant region fragments from the two positive controls: a sample from a human spleen (lane<br>
8) and a AHAC chimeric mouse spleen (lane 9). As expected, the negative controls from a normal mouse spleen and a bovine spleen did not contain an amplified sequence (lanes 1 and 2). Samples from the liver and brain of fetus #5996 did not contain an amplified spliced sequence of the same size as the spliced human mu heavy chain constant region fragments but did contain a amplified sequence of an unspliced genomic fragment derived from genomic DNA contaminating the RNA sample (lanes 3, 4, and 5).<br>
VDJ rearrangement of the human heavy chain locus in a AHAC<br>
fetus<br>
RT-PCR analysis was also performed to further demonstrate VDJ rearrangement in the heavy chain locus in AHAC fetus #5996. Nested RT-PCR was performed using primer Cmu-1 (5-CAGGAGAAAGTGATGGAGTC-3'; SEQ ID NO: 15) for the first reaction, primer Cmu-2 (5'-AGG CAG CCA ACG GCC ACG CT-3'; SEQ ID NO: 16) for the second reaction, and primer VH3-30.3 (5'-CAG GTG CAG CTG GTG GAG TCT GG-3'; SEQ ID NO: 17) for both reactions. The RT-PCR reaction mixtures contained 18.9 jil water, 3 ul of 10X Ex Taq buffer, 4.8 (il of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 1 ul of cDNA, and 0.3 ul of Ex Taq. The RT-PCR was performed using 38 cycles under the following conditions for the first reaction: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 65C for 30 seconds, and 72C for 30 seconds. For the second reaction, 38 cycles were performed under the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 65C for 30 seconds, and 72C for 30 seconds using primers VH3-30.3 and Cmu-2 (5'-AGG CAG CCA ACG GCC ACG CT-3'; SEQ ID NO: 16).<br>
As shown in lanes 6 and 7 of Fig. 10, RT-PCR analysis of the spleen of fetus #5996 produced a heavy chain band of the same size as the positive controls in lanes 8 and 9. Samples from the liver and brain of fetus #5996<br>
contained some contaminating rearranged DNA (lanes 3 and 5). The negative controls in lanes 1 and 2 produced bands of the incorrect size.<br>
Verification Of AHAC rearrangement By sequencing<br>
The cDNA obtained by reverse transcription of RNA from the spleen of the AHAC fetus #5996 was amplified with primers specific for rearranged human mu and run on an agarose gel. The band produced by amplification with the Cmul-VH3-30 primer pair was excised from the gel. The amplified cDNA was recovered from the band and cloned. DNA from a resulting clone that was PCR-positive for rearranged human mu was purified and sequenced (Fig. 11 A).<br>
The sequence from this AHAC fetus is greater than 95% homologous to over 20 known human heavy chain sequences. For example, the mu chain of a human anti-pneumococcal antibody is 97% homologous to a region of this sequence (Fig. 11B).<br>
Additional sequences from rearranged human heavy chains were also obtained by RT-PCR analysis of the spleen of fetus #5996 using primers Cmu-1 and VH3-30.3, followed by reamplification using primers Cmu-2 and VH3-30.3. The RT-PCR products were purified using CHROMA SPIN column (CLONETECH) and cloned into the pCR2.1 TA-cloning vector (Invitrogen) according to manufacturer's protocol. The Dye Terminator sequence reaction (ABI Applied System) was performed in a 10 (il volume reaction mixture composed of BigDye Terminator reaction mixture (3 pi), template plasmid (200 ng), and the Cmu-2 primer (1.6 pmol). The sequencing reaction was performed using a ABI 3700 sequencer. For this analysis, twenty-five cycles were conducted under the following conditions: 96 C for one minute, 96C forlO seconds, 55C for five seconds, and 60C for four minutes.<br>
At least two rearranged human heavy chain transcripts were identified, which were VH3-1 l/D7-27/JH3/C^ and VH3-33/D6-19/JH2/C|i (Figs. 12A and 12B). These results demonstrate that VDJ rearrangement of the human mu<br><br>
heavy chain locus occurs in the AHAC in the spleen of fetus #5996. The identification of more than one rearranged heavy chain sequence from the same fetus also demonstrates the ability of AHAC fetuses to generate diverse human immunoglobulin sequences.<br>
Rearrangement and expression of human heavy and light chain loci in AAHAC fetus<br>
Cloned fetuses derived from bovine fetal fibroblasts transchromosomal for the AAHAC were removed from recipient cows at various gestational days. The fetuses were analyzed for the presence and rearrangement of the HAC-borne human immunoglobulin heavy and lambda light chain loci. Studies of genomic DNA from these tissues indicated the presence of the human immunoglobulin heavy and light chains in some of the fetuses. Examination of cDNA derived from the spleens of these fetuses indicated rearrangement and expression of the immunoglobulin heavy and light chain loci in some of these fetuses. FACS analysis also demonstrated the expression of human lambda light chain protein on the surface of splenic lymphocytes in two of the fetuses.<br>
Presence of human heavy and light chain loci in AAHAC fetuses<br>
To determine whether AAHAC fetuses retained the human heavy and light chain loci, PCR analysis was performed on genomic DNA from the liver of 58 day fetus #5580, 57 day fetus # 5848, and 91 day fetuses #5442A and 5442B.   The PCR primers used for detection of the heavy chain loci were VH3-F<br>
(5-ACT GAG ATA AGC AGT GGA TG-3'; SEQ ID NO: 18) and VH3-R (5'-CTT GTG CTA CTC CCA TCA CT-3'; SEQ ID NO: 19), and the primers used for the detection of the light chain were IgL-F (5'-GGA GAC CAC CAA ACC CTC CAA A-3'; SEQ ID NO: 20) and IgL-R (5'-GAG AGT TGC AGA AGG GGT YGA CT-3'; SEQ ID NO: 21). The PCR reaction mixtures contained 18.9 ul water, 3ul of 10X Ex Taq buffer, 4.8 ul of dNTP mixture, 10<br>
pmol forward primer, 10 pmol of reverse primer, 1 ul of genomic DNA, and 0.3 ul of Ex Taq. Thirty-eight PCR cycles were performed as follows: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 56C for 30 seconds, and 72C for 30 seconds (Figs. 13 and 14).<br>
As illustrated in Figs. 13 and 14, positive control 58 day fetus #5580 contained both human heavy and light chain immunoglobulin loci. Additionally, the 91 day fetuses #5442A and 5442B also contained both heavy and light chain loci (Fig. 14). In contrast, fetus #5848 did not contain either human loci and may not have contained AAHAC. These results suggested that AAHAC can be stably retained up to gestational day 91 in bovine.<br>
Rearrangement and expression of human heavy chain locus in AAHAC fetus #5442A<br>
RT-PCR was used to detect expression of rearranged human heavy chain RNA transcripts in AAHAC fetus #5542A. The RT-PCR primers used were CH3-F3 (5'-GGA GAC CAC CAA ACC CTC CAA A-31; SEQ ID NO: 22) and CH4-R2 (5'-GAG AGT TGC AGA AGG GGT GAC T-3'; SEQ ID NO: 23). The RT-PCR reaction mixtures contained 18.9 [il water, 3 ul of 10X Ex Taq buffer, 4.8 ul of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 1 ul of cDNA, and 0.3 ul of Ex Taq. Forty cycles of RT-PCR cycles were performed as follows: 85C for three minutes, 94C for one minute, 98C for 10 seconds, 60C for 30 seconds, and 72C for 30 seconds.<br>
Lanes 4 and 5 of Fig. 15 contained amplified spliced mu heavy chain constant region sequences from the spleen of fetus #5442A that are similar in size to that of the positive control samples. These results indicate that fetus #5442A expressed a rearranged mu heavy chain transcript in its spleen. Faint bands were also seen in the region of the unspliced genomic sequence, which are amplified from genomic DNA contaminated in the RNA sample. Control<br>
samples from the liver and brain of fetus #5442 A did not produce a band of the size expected for an amplified rearranged heavy chain sequence.<br>
Rearrangement and expression of human heavy chain locus in<br>
AAHAC fetus #5868A<br>
RT-PCR was used to detect expression of rearranged human heavy chain RNA transcripts in the spleen of a AAHAC fetus at 119 gestational days (fetus #5868A). The primers used for this analysis were VH30-3 (5-cag gtg cag ctg gtg gag tct gg-3'; SEQ ID NO: 24) and CM-1 (5-cag gag aaa gtg atg gag tc-31; SEQ ID NO: 25). Additionally, primers "GAPDH up" (5'-gtc ate ate tct gcc cct tct g-3'; SEQ ID NO: 26) and "GAPDH down" (5'-aac aac ttc ttg atg tea tea t-3'; SEQ ID NO: 27) were used to amplify GAPDH control transcripts. For this PCR analysis, the reaction mixture was incubated at 95C for five minutes and then multiple cycles of denaturation, annealing, and amplification were performed by incubation at 95C for one minute, 58C for one minute, and 72C for two minutes. Then, the mixture was incubated at 72C for 10 minutes.<br>
Lane 3 of Fig. 16 contains the RT-PCR product produced from this analysis of AAHAC fetus# 5868 A. This RT-PCR product was the size expected for the amplification of a rearranged human heavy chain (470 base pairs) and migrated to the same position in the gel as the control cDNA known to contain sequences corresponding to rearranged human heavy chain transcripts. As controls, both AAHAC fetus# 5868A fetal spleen cDNA and normal bovine cDNA samples generated a product when amplified with GAPDH primers, demonstrating the capacity of the cDNA to support amplification (lanes 7 and 8).<br>
Rearrangement and expression of human lambda locus in AAHAC fetuses #5442A and 5442B<br>
Primers specific for amplification of a transcript including portions of human lambda were used to detect RNA transcripts from a rearranged human lambda light chain locus.<br>
For the RT-PCR analysis shown in Fig. 17, an equimolar mixture of primers CU (5'-GGG AAT TCG GGT AGA AGT TCA CTG ATC AG -3'; SEQ ID NO: 28), CX2-3 (5'-GGG AAT TCG GGT AGA AGT CAC TTA TGA G -3'; SEQ ID NO: 29), and CA.7 (5'-GGG AAT TCG GGT AGA AGT CAC TTA CGA G -3'; SEQ ID NO: 30) was used with primer VX1 LEA1 (5'-CCC AAG CTT RCC KGS TYY CCT CTC CTC -3'; SEQ ID NO: 31). The RT-PCR reaction mixtures contained 18.9 ul water, 3 |il of 10X Ex Taq buffer, 4.8 jal of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 1 jj.1 of cDNA and 0.3 [al of Ex Taq. The RT-PCR conditions were as follows: 40 cycles of 85C for three minutes, 94C for one minute, 98C for 10 seconds, 60C for 30 seconds, and 72C for one minute.<br>
As shown in Fig. 18, this RT-PCR analysis was also performed using an equimolar mixture of primers VX3LEA1 (5-CCC CCA AGC TTG CCT GGA CCC CTC TCT GG-3'; SEQ ID NO: 32), V?i3JLEAD (5'- ATC GGC AAA GCT TGG ACC CCT CTC TGG CTC AC-3'; SEQ ID NO: 33), VXBACK4 (5'-CCC CCA AGC TTC TCG GCG TCC TTG CTT AC-3'; SEQ ID NO: 34) and an equimolar mixture of primers C^l (5'-GGG AAT TCG GGT AGA AGT TCA CTG ATC AG-3'; SEQ ID NO: 35) CX2-3 (5'-GGG AAT TCG GGT AGA AGT CAC TTA TGA G-3'; SEQ ID NO: 36) and CV7 (5'-GGG AAT TCG GGT AGA AGT CAC TTA CGA G-3'; SEQ ID NO: 37). The RT-PCR reaction conditions were the same as those described above for Fig. 7.<br>
Lanes 6 and 7 of Fig. 17 and lanes 4 and 5 of Fig. 18 contained RT-PCR products from the spleen of fetus #5442A that are similar in size to the positive control bands, indicating the presence of rearranged light chain RNA<br>
transcripts in this fetus. The spleen sample from fetus #5442B produced very weak bands (that are not visible in the picture) of the appropriate size. This RT-PCR product indicates that fetus #5442B also expressed a rearranged light chain immunoglobulin transcript in its spleen. As expected, samples from the brain of fetuses #5442A and 5442B did not express human rearranged lambda light chain transcripts.<br>
Rearrangement and expression of human lambda locus in AAHAC<br>
Fetus #5868A<br>
RNA transcripts from a rearranged human lambda light chain locus were also detected in AAHAC fetus# 5868A. For this analysis, primers specific for amplification of a transcript including portions of human lambda were used to detect AAHAC-encoded expression of transcripts encoding portions of a rearranged human lambda locus. Primer VL1 LEAI (5'-ccc cca age ttR ccK gSt Yyc etc tec tc-3'; SEQ ID NO: 38 ) and an equimolar mixture of primers CL1 (5'-ggg aat teg ggt aga agt cac tga tea g-3'; SEQ ID NO: 39), CL2-3 (5-ggg aat teg ggt aga agt cac tta tga g-3'; SEQ ID NO: 40), and CL7 (5-ggg aat teg ggt aga agt cac tta cga g-3'; SEQ ID NO: 41) were used for this analysis. For this RT-PCR reaction, the reaction mixtures were incubated at 95C for 5 minutes and then multiple cycles of denaturation, annealing, and amplification were performed by incubation at 95C for one minute, 60C for one minute, and 72C for two minutes. Then, the mixtures were incubated at 72C for 10 minutes.<br>
This analysis demonstrated that spleen cDNA from AAHAC #5868A (lane 4 of Fig. 19) produced a RT-PCR product of the same size as the TC mouse spleen cDNA (lane 6) positive control. No such RT-PCR product was detected using either brain or liver cDNA from AAHAC #5868A (lanes 2 and 3, respectively). The capacity of each of these tissues to support RT-PCR was shown by successful amplification of the housekeeping gene, GAPDH using primers "GAPDH up" and "GAPDH down" (lanes 8 and 10).<br>
Verification of AAHAC rearrangement by sequencing<br>
RT-PCR analysis was performed on a spleen sample from fetus #5442A using an equimolar mixture of primers CA.1, CX2-3, and CX7 with primer VA,1LEA1, or an equimolar mixture of primersVXSLEAl, VX.3JLEAD, and V?iBACK4 and an equimolar mixture of primers CX,1, CA2-3, and C'kl. The PCR products were purified using a CHROMA SPIN column (Clontech) and cloned into the pCR2.1 TA-cloning vector (Invitrogen), according to manufacturer's protocol. The Dye Terminator sequence reaction (ABI Applied System) was carried out using the CK\, CX2-3, and CA/7 primers in an equimolar mixture. Twenty-five cycles were performed at 96C for one minute, 96C for 10 seconds, 55C for five seconds, and 60C for four minutes. The 10 ul reaction mixture contained BigDye Terminator reaction mixture (3 ul), template plasmid (200 ng), and the C^l, CA.2-3, and Ckl primers (1.6 pmol). The reaction mixture was analyzed using a ABI 3700 sequencer.<br>
At least two rearranged human lambda light chain transcripts were identified (Vl-17/JL3/C?i and V2-13/JL2/CX). These results demonstrate that VJ rearrangement of human lambda light chain genes occurs in the AAHAC in the spleen of fetus #5442A (Figs. 20 and 21).<br>
FACS analysis of expression of human lambda light chain and bovine heavy chain in AAHAC Fetus #5442A and 5442B Splenic lymphocytes from AAHAC Fetus #5442A and 5442B were analyzed for the expression of human lambda light chain and bovine heavy chain proteins. These cells were reacted with a phycoerytherin labeled anti-human lambda antibody (Figs. 22C and 22D), a FITC labeled anti-bovine IgM antibody (Figs. 22D and 22H), or no antibody (Figs. 22A, 22B, 22E, and 22F) for 20 minutes at 4C. Cells were then washed twice with PBS plus 2% PCS and analyzed on a FASCalibur cell sorter. The percent of cells reacting with the antibody was calculated using the non antibody controls to electronically<br>
set the gates. These percentages are displayed beneath each histogram. Fetus # 5442A (Figs. 22A-22D) and fetus #5442B (Figs. 22E-22H) expressed both human lambda light chain protein and bovine heavy chain protein.<br>
Expression of human antibody protein in HAC calves<br>
As described above, a novel procedure was developed for producing transchromosomic (Tc) calves (Fig. 37). This method overcame the limitations due to the limited life span of only about 35 population doublings for primary bove fibroblasts and the requirement for a large DNA insert to be introduced and maintained in the donor cells. In particular, a human artificial chromosome (HAC) vector was used to introduce both the entire unrearranged human Ig heavy (IgH) and lambda light chain (/gA,) loci into bovine primary fibroblasts. Selected fibroblast clones were rejuvenated and expanded by producing cloned fetuses. Cloned fetal cells were selected and recloned to produce four healthy, Tc calves that functionally rearranged both heavy and light chain human Ig loci and produced human polyclonal antibodies. These results demonstrate the feasibility of using HAC vectors for production of transgenic livestock. More importantly, Tc cattle containing human Ig genes may be used to produce novel human polyclonal therapeutics with applications ranging from the prevention of antibiotic resistant infections to combating bioterrorism. This method is described in more detail below.<br>
HACs were introduced into bovine fetal fibroblasts from CHO clones using the MMCT technique described herein. The fibroblasts were placed under selection for the neo gene marker on the HAC vector with G418 (700 ug/ml) until colonies began to appear. Complete antibiotic selection and DNA-based screening was avoided at this step to minimize cell divisions prior to nuclear transfer. Colonies were picked on the basis of growth and morphology and nuclear transfer was carried out as previously described (Lucier et al, J. Immunol. 161:5438-5444, 1998). Because the cells were only useful for<br>
nuclear transfer for a few days, final selection was done after rejuvenation and expansion of cells by production of cloned fetuses.<br>
Development to the blastocyst stage ranged from 17 to 21% and pregnancy at 40 days ranged from 22 to 50% with no differences among cell lines. At 56 to 58-days, four AHAC and two AAHAC fetuses were recovered and fibroblast cell lines were regenerated and cryopreserved for further analysis and nuclear transfer. Retention of the HACs in the fibroblast lines derived from the fetuses collected at 56 to 5 8-day s and seven additional fetuses collected between 77 and 119-day s was evaluated by G418-resistance (Fig 38A) and genomic PCR of IgH and 7g/l loci (Fig 38B). Nine of 13 fetuses were resistant to G418 and eight of them showed the presence of both human IgH and IgA loci. Three AHAC (#5968, #6032 and #6045) and one AAHAC (#5580) positive 56 to 58-day fetuses were used for recloning and production of offspring. Five positive 77 to 119-day fetuses were evaluated for expression and rearrangement of the human Ig loci by RT-PCR analysis, followed by sequencing of the amplified products. Human IgH and 7g/l genes were expressed (Fig. 38C) in all fetuses and showed evidence of proper V(D)J recombination (Fig. 38D).<br>
Recloned AHAC cell lines produced one male (from cell line #6045) and 5 female (from cell lines #5968 and #6032) calves from 37 recipients (16%, Fig. 39A). The two calves derived from cell line #6032 died within 48 hours after birth. One calf was born from non-regenerated AAHAC cells. All five live calves were healthy and phenotypically normal. Retention of the HAC was confirmed in all the calves by G418 selection, genomic PCR and fluorescent in situ hybridization (FISH) analyses (Figs. 39B and 39C). Results of FISH analysis indicated that the HAC was retained as an independent chromosome and the proportion of cells retaining the HAC was 78 to 100%. No obvious differences were observed in retention rates between peripheral blood lymphocytes (PBLs, 91%) and fibroblasts (87%). Donor cell line #6045 may have had a higher retention rate than donor cell line #5968 (97% and 86%,<br>
respectively). Interestingly, retention rate in the Tc bovine may be higher than we previously observed in mouse. To determine whether the human Ig loci were rearranged and expressed in calves as well as in fetuses, we performed RT-PCR analysis on PBLs. We observed the expression of both human IgH and 7gA genes in the PBLs and the diversity of the human IgH and 7g/l repertoire was determined by sequence analysis (Table 4). A representative set of the sequences showed a wide utilization of Vn/VA, DH and JH/J/l segments distributed over the loci. In the 7g/l transcripts, the frequent utilization of V segments from VH! and V}}3 was observed, which is similar to the usage of VH segments in human. Addition of non-germline nucleotides (N-addition), as well as nucleotide deletion, was also observed in both IgH and /g/3, transcripts. This produced a high degree of diversification in the third complementarity determining regions (CDRSs) of both heavy and light chains. Furthermore, human Ig protein was detected at levels ranging from 13 to 258 ng/ml (Ig expression is typically very low to undetectable in newborn calves) in blood samples collected prior to colostrum feeding in five of the seven calves as determined by solid phase ELISA. These data indicate that the HAC transfer can be accomplished efficiently in primary cells using a recloning strategy and that human Ig genes carried by the HAC can be properly processed and expressed with a high degree of diversity in Tc calves.<br>
The results of this study demonstrate that a combination of chromosome-cloning, chromosome transfer and somatic cell recloning technologies can be used to produce healthy calves retaining a HAC vector carrying Mb-sized genomic transgenes. Furthermore, these technologies were used to demonstrate the transfer and retention of the entire loci for both the human IgH and IgA genes in cattle. Interestingly, both loci were demonstrated to have undergone proper processing and express functionally rearranged human immunoglobulin genes in a species with substantially different immunophysiology than either the human or mouse. This HAC system may be useful for the expression of a variety of complex human proteins (e.g.,<br>
hemoglobulin) or large collections of proteins for pharmaceutical applications. For example, the Tc calves produced in this study, which retain both the human IgH and 7g/l loci, are useful for production of human polyclonal antibodies. Human polyclonal antibodies are currently only available from human blood or plasma donors. Consequently, there are limitations in supply and application of human polyclonal products. Tc cows may be hyperimmunized to produce large quantities of novel polyclonal therapeutics for treatment of a wide variety of human diseases.<br>
Table 4. Repertoire analysis of human immunoglobulin heavy and lambda chain transcripts in cloned Tc calves. Human fi and A,-specific mRNAs were amplified by RT-PCR, cloned and sequenced. Nucleotide sequences of V(D)J junctions of each of 10 independent |i and X clones are shown, divided into VH/VA,, DH, JH/JA and N segments, as identified by homology to published germline sequences (Ig-BLAST).<br>
EXAMPLE 3: Transgenic ungulates producing xenogenous antibodies that have a mutation in one or more endogenous antibodies<br>
In an ungulate that expressed xenogenous antibodies and that has a mutation in a prion gene as described in Example 2, the expression of endogenous antibodies may be optionally reduced by mutating one or more endogenous antibody genes. By increasing the number of functional xenogenous immunoglobulin heavy or light chain genes relative to the number of functional endogenous heavy or light chain genes, the percentage of B-cells expressing the desired xenogenous antibodies (e.g., human therapeutic antibodies) should increase.<br>
To generate these transgenic ungulates, AHAC or AAHAC transgenic ungulates may be mated with transgenic ungulates containing a mutation in one or both alleles of an endogenous immunoglobulin chain (e.g., a mu heavy chain or a lambda or kappa light chain). If desired, the resulting transgenic ungulates may be mated with (i) transgenic ungulates containing a mutation in one or both alleles of an endogenous alpha-(l,3)-galactosyltransferase, prion, and/or J chain nucleic acid or (ii) transgenic ungulates containing an exogenous J chain nucleic acid (e.g., human J chain). Alternatively, a cell (e.g., a fetal fibroblast) from a AHAC or AAHAC transgenic fetus may be genetically modified by the mutation of one or more endogenous immunoglobulin genes. In another possible method, AHAC or AAHAC is introduced into a cell (e.g., a fetal fibroblast) in which endogenous immunoglobulins (mu heavy and/or lambda light chains) are hemizgously or homozygously inactivated. In any of the above methods, the cells may also be genetically modified by (i) the introduction of a mutation, preferably a knockout mutation, into one or both alleles of an endogenous alpha-(l,3)-galactosyltransferase, prion, and/or J chain nucleic acid or (ii) the introduction of an exogenous J chain nucleic acid. The resulting transgenic cell may then be used in nuclear transfer procedures to generate the desired transgenic ungulates. Exemplary methods are described below.<br>
DNA constructs<br>
The mu heavy chain (Fig. 2A), lambda light chain, kappa light chain, alpha-(l,3)-galactosyltransferase, prion, and/or J chain knockout constructs described above may be used. Alternatively, the puromycin resistant mu heavy chain construct described below may be used (Fig. 3F). This knockout construct was designed to remove the 4 main coding exons of the bovine mu heavy chain locus but leave the transmembrane domain intact, resulting in the inactivation of the mu heavy chain locus.<br>
The puromycin resistant construct was assembled as follows. A 4.4 kilobase Xhol fragment containing the region immediately proximal to coding exon 1 was inserted into the^TzoI site of pBluescript II SK+. Plasmid pPGKPuro, which contains a puromycin resistant gene, was obtained from Dr. Peter W. Laird, Whitehead Institute, USA. A 1.7 Kb Xhol fragment containing a puromycin resistance gene was subcloned adjacent to, and downstream of, the 4.4 Kb fragment into the Sail site present in the polylinker region. This 1.7 Kb puromycin marker replaces the coding exons CHI, CH2, CHS and CH4 of the bovine immunoglobulin heavy chain locus. AnXbal fragment containing a 4.6 Kb region of the mu locus that is downstream of these four exons in the wild-type genomic sequence was added to this construct for use as the second region of homology.<br>
To generate the final targeting construct, a subclone of this construct was generated by cutting the three assembled fragments with Notl and MM. The MM restriction digestion truncates the 4.6 Kb fragment down to 1.4 Kb. The Notl site lies in the polylinker and does not cut into the subcloned DNA itself. The MM site was filled in with a Klenow fragment to generate a blunt end, and the TVofl/filled in Mlul fragment was subcloned into a fresh pBluescript II SK+ vector using the Notl and Smal sites present in the pBluescript vector. For gene targeting, the final vector is linearized with Notl.<br>
Gene targeting by electroporation and drug selection of transfected fibroblasts<br>
For electroporation, a single cell suspension of 1 x 10 bovine fetal fibroblasts (e.g, fibroblasts obtained as described in Example 2 from a AHAC or AAHAC transgenic fetus) that had undergone a limited number of population doublings is centrifuged at 1200 rpm for five minutes and re-suspended in 0.8 ml of serum-free Alpha-MEM medium. The re-suspended cells are transferred to a 0.4 cm electroporation cuvette (Invitrogen, cat#. P460-50). Next, 30 ug of a restriction enzyme-linearized, gene targeting vector DNA is added, and the<br>
contents of the cuvette are mixed using a 1 ml pipette, followed by a two minute incubation step at room temperature. The cuvette is inserted into the shocking chamber of a Gene Pulser II electroporation system (Biorad) and then electroporated at 1000 volts and 50 uF. The cuvette is quickly transferred to a tissue culture hood and the electroporated cells are pipetted into approximately 30 ml of complete fibroblast medium. The cells are equally distributed into thirty 100 mm tissue culture dishes (Corning, cat# 431079), gently swirled to evenly distribute the cells, and incubated at 38.5C/5% CO2 for 16 to 24 hours. The media is removed by aspiration and replaced with complete fibroblast medium containing the selection drug of choice. The media is changed every two days and continued for a total time period of 7 to 14 days. During the drug selection process, representative plates are visually monitored to check for cell death and colony formation. Negative control plates are set up that contained fibroblasts that are electroporated in the absence of the gene targeting vector and should yield no colonies during the drug selection process.<br>
Picking of drug-resistant fibroblast colonies and expansion of cells<br>
Following completion of the drug selection step (usually 7 to 14 days), the drug resistant colonies are macroscopically visible and ready for transfer to 48 well tissue culture plates for expansion. To assist in the transferring process, individual colonies are circled on the bottom of the tissue culture plate using a colored marker (Sharpie). Tissue culture plates containing colonies are washed 2X with IX D-PBS (without Ca2+ and Mg2+) and then 5 ml of a 1:5 dilution of the cell dissociation buffer is added per plates. Following a three to five minute room temperature incubation step, individual colonies start to detach from the bottom of the tissue culture dish. Before the colonies detached, they are individually transferred to a single well of a 48 well tissue culture plate using a P200 pipetmen and an aerosol barrier pipette tip (200 or 250 ul). Following transfer, the colony is completely dissociated by pipeting up-and-down and 1 ml of complete fibroblast medium is added. To ensure that<br>
the cells are drug resistant, drug selection is continued throughout the 48 well stage. The transferred colonies are cultured at 38.5C/5% CO2 and visually monitored using an inverted microscope. Two to seven days later, wells that are approaching confluency are washed two times with IX D-PBS (without Ca2+ and Mg2+) and detached from the bottom of the well by the addition of 0.2 ml of cell dissociation buffer, followed by a five minutes room temperature incubation step. Following detachment, the cells are further dissociated by pipeting up-and-down using a PI000 pipetmen and an aerosol pipette tip (1000 ul). Approximately 75% of the dissociated fibroblasts are transferred to an individual well of a 24 well tissue culture plate to expand further for subsequent PCR analysis and the remaining 25% is transferred to a single well of a second 24 well plate for expansion and eventually used for somatic cell nuclear transfer experiments. When cells in the plate containing 75% of the original cells expanded to near confluency, DNA is isolated from that clone for genetic analysis.<br>
DNA preparation<br>
The procedure used to isolate DNA for genetic analyses is adapted from Laird et al, Nucleic Acids Research, 1991, Volume 19, No. 15. In particular, once a particular clone has attained near-confluency in one well of a 24 well plate, culture medium is aspirated from that well and the adherent cells are washed twice with PBS. The PBS is aspirated off and replaced with 0.2 ml buffer to lyse the cells and digest excess protein from the DNA to be isolated. This buffer is composed of 100 mM Tris-HCl (pH 8.5), 5 mM EDTA, 0.2% SDS, 200 mM NaCl and 100 jag/ml proteinase K. The 24 well plate is returned to the tissue culture incubator for a minimum of three hours to allow the release of the DNA and digestion of protein. The viscous product of this procedure is transferred to a 1.5 ml microcentrifuge tube and 0.2 ml of isopropanol added to precipitate the DNA. The precipitate is recovered by centrifugation, the DNA pellet is rinsed with 70% ethanol, and after air-drying the pellet is resuspended<br>
in 25-50 ul of buffer containing 10 mM Tris, pH 8, and 1 mM EDTA. This DNA is used for PCR analyses of clones.<br>
Screening of clones<br>
Two different approaches are used to screen clones, both employing the polymerase chain reaction (PCR). All approaches described in this section are adaptable to the targeting of any other gene, the only difference being the sequences of the primers used for genetic analysis.<br>
According to the first approach, two separate pairs of primers are used to independently amplify products of stable transfection. One pair of primers is used to detect the presence of the targeting vector in the genome of a clone, regardless of the site of integration. The primers are designed to anneal to DNA sequences both present in the targeting vector. The intensity of the PCR product from this PCR reaction may be correlated with the number of copies of the targeting vector that have integrated into the genome. Thus, cells containing only one copy of the targeting vector tend to result in less intense bands from the PCR reaction. The other pair of primers is designed to detect only those copies of the vector that integrated at the desired locus. In this case, one primer is designed to anneal within the targeting vector and the other is designed to anneal to sequences specific to the locus being targeted, which are not present in the targeting vector. In this case, a PCR product is only detected if the targeting vector has integrated directly next to the site not present in the targeting vector, indicating a desired targeting event. If product is detected, the clone is used for nuclear transfer.<br>
For the neomycin resistant heavy chain knockout construct, primers Neol (5 -CTT GAA GAC GAA AGO GCC TCG TGA TAG GCC-3'; SEQ ID NO: 42) and IN2521 (5'-CTG AGA CTT CCT TTC ACC CTC CAG GCA CCG-3'; SEQ ID NO: 43) are used to detect the presence of the targeting vector in cells, regardless of the location of integration. Primers Neol and OUT3570 (5'-CGA TGA ATG CCC CAT TTC ACC CAA GTC TGT C-3'; SEQ ID NO:<br>
44) are used to specifically amplify only those copies of the targeting construct that integrated into the mu heavy chain locus.<br>
For these PCR reactions to analyze the integration of the neomycin resistant heavy chain knockout construct, a Qiagen PCR kit is used. The PCR reaction mixture contains 1 pmole of each primer, 5 ul of 10X reaction buffer, 10 ul of Q solution, 5 ul of DNA, and 1 ul of dNTP solution. The reaction mixture is brought to a total volume of 50 ul with I^O. This PCR amplification is performed using an initial denaturing incubation at 94C for two minutes. Then, 30 cycles of denaturation, annealing, and amplification are performed by incubation at 94C for 45 seconds, 60C for 45 seconds, and 72 C for two minutes. Then, the reaction mixture is incubated at 72C for five minutes and at 4C until the mixture is removed from the PCR machine.<br>
In the alternative approach, a single primer set is used to amplify the targeted locus and the size of the PCR products is diagnostic for correct targeting. One primer is designed to anneal to a region of the locus not present in the targeting vector and the other primer is designed to anneal to a site present in the targeting vector but also present in the wild type locus. In this case, there is no detection of targeting vector that had integrated at undesirable sites in the genome. Because the region deleted by the targeting vector is different in size from the drug selection marker inserted in its place, the size of the product depended on whether the locus amplified is of wild-type genotype or of targeted genotype. Amplification of DNA from clones containing incorrect insertions or no insertions at all of the targeting vector results in a single PCR product of expected size for the wild type locus. Amplification of DNA from clones containing a correctly targeted ("knocked out") allele results in two PCR products, one representing amplification of the wild type allele and one of altered, predictable size due to the replacement of some sequence in the wild-type allele with the drug resistance marker, which is of different length from the sequence it replaced.<br>
For the puromycin resistant heavy chain knockout construct, primers Shortend (5'-CTG AGC CAA GCA GTG GCC CCG AG-3'; SEQ ID NO: 45) and Longend (5-GGG CTG AGA CTG GGT GAA CAG AAG GG-3'; SEQ ID NO: 46) are used. This pair of primers amplifies both the wild-type heavy chain locus and loci that have been appropriately targeted by the puromycin construct. The size difference between the two bands is approximately 0.7 Kb. The presence of the shorter band is indicative of appropriate targeting.<br>
For this PCR reaction to analyze the integration of the puromcying resistant heavy chain knockout construct, a Promega Master Mix kit is used. The PCR reaction mixture contains 1 pmole of each primer, 2.5 ul of DNA, and 25 ul of 2X Promega Master Mix. The reaction mixture is brought to a total volume of 50 ul with H2O. This PCR amplification is performed using an initial denaturing incubation at 94C for two minutes. Then, 30 cycles of denaturation, annealing, and amplification are performed by incubation at 94C for 45 seconds, 60C for 45 seconds, and 72 C for two minutes. Then, the reaction mixture is incubated at 72C for five minutes and at 4C until the mixture is removed from the PCR machine.<br>
First round of nuclear transfer<br>
Selected fibroblast cells in which an immunoglobulin gene has been inactivated may be used for nuclear transfer as described in Example 2 to generate a transgenic ungulate containing a mutation in an endogenous immunoglobulin gene and containing a HAC encoding a xenogenous immunoglobulin gene. Alternatively, nuclear transfer may be performed using standard methods to insert a nucleus or chromatin mass (i.e., one or more chromosomes not enclosed by a membrane) from a selected transgenic fibroblast into an enucleated oocyte (described, for example, in U.S.S.N. 60/258,151; filed December 22, 2000). These methods may also be used for cells in which an endogenous alpha-(l,3)-galactosyltransferase, prion, and/or J chain nucleic acid has been mutated.<br>
Second Round of Mutagenesis and Nuclear Transfer<br>
If desired, a cell (e.g., a fetal fibroblast) may be obtained from a transgenic ungulate generated from the first round of nuclear transfer. Another round of gene targeting may be performed as described above to inactivate the second allele of the gene inactivated in the first round of targeting. Alternatively, another immunoglobulin (e.g., mu heavy chain, lambda light chain, kappa light chain, or J chain), alpha-(l,3)-galactosyltransferase, or prion gene may be inactivated in this round of targeting. For this second round of targeting, either a higher concentration of antibiotic may be used or a knockout construct with a different antibiotic resistance marker may be used. Antibiotic resistance cells may be selected as described above. The selected cells may be used in a second round of nuclear transfer as described above to generate, for example, a transgenic ungulate containing two mutations in endogenous immunoglobulin genes and containing a HAC encoding a xenogenous immunoglobulin gene. Alternatively, the selected antibiotic resistant cells may first be treated to isolate Gl phase cells as described below, which are used for the second round of nuclear transfer.<br>
To isolation of Gl cells for nuclear transfer, 5.0 x 105 cells are plated onto 100 mm tissue culture plates containing 10 ml of a-MEM + PCS, twenty four hours prior to isolation. The following day, plates are washed with PBS and the culture medium is replaced for 1-2 hours before isolation. The plates are then shaken for 30-60 seconds on a Vortex-Genie 2 (Fisher Scientific, Houston, TX, medium speed), the medium is removed, spun at 1000 G for five minutes and the pellet is re-suspended in 250 ul of MEM +FCS. Newly divided cell doublets attached by a cytoplasmic bridge, are then selected, as these cells are in early Gl. This isolation procedure is referred to as the "shake-off method.<br>
EXAMPLE 4: Additional methods to mutate endogenous immunoglobulin genes<br>
In some embodiments of the present approach, xenogenous immunoglobulin production in an ungulate with one or more mutations in a prion gene is accomplished essentially by the combined use of homologous recombination techniques, introduction of artificial chromosomes carrying entire xenogenous Ig loci, nuclear transfer, and administration of an antibody to eliminate endogenous antibody. More specifically, the process preferably involves the targeted disruption of one or both alleles of the IgM heavy chain gene, and optionally one or both alleles of the Ig light chain gene, although xenogenous antibody production can also be accomplished in animals without mutations in Ig nucleic acids. Gene knockouts may be effected by sequential homologous recombination, then another mating procedure. In a preferred embodiment, this is effected by initially effecting targeted disruption of one allele of the IgM heavy chain gene of a male or female ungulate (for example, bovine) fetal fibroblast in tissue culture using a suitable homologous recombination vector. The use of fetal fibroblasts is preferred over some other somatic cells as these cells are readily propagated and genetically manipulated in tissue culture. However, the use of fetal fibroblasts is not essential to the invention, and indeed other cell lines may be substituted therefor with equivalent results.<br>
This process, of course, entails constructing a DNA construct having regions of homology to the targeted IgM heavy chain allele such that the construct upon integration into an IgM heavy chain allele in the ungulate genome disrupts the expression thereof. An exemplary vector for carrying out such targeted disruption of an IgM allele is described in the example that follows. In this regard, methods for constructing vectors that provide for homologous recombination at a targeted site are well known to those skilled in the art. Moreover, in the present instance, the construction of a suitable vector is within the level of skill in the art, given especially that the sequence of the<br>
bovine IgM heavy chain and Ig lambda light chain genes are known, as are the sequences of immunoglobulon genes from other ungulates (see below). In order to facilitate homologous recombination, the vectors used to effect homologous recombination and inactivation of the IgM gene, respectively, comprise portions of DNA that exhibit substantial sequence identity to the ungulate IgM heavy and Ig light chain genes. Preferably, these sequences possessing at least 98% sequence identity, more preferably, at least 99% sequence identity, and still more preferably will be isogenic with the targeted gene loci to facilitate homologous recombination and targeted deletion or inactivation.<br>
Typically, and preferably the construct will comprise a marker gene that provides for selection of desired homologous recombinants, for example, fibroblast cells, wherein the IgM heavy chain gene and/or Ig light chain gene has been effectively disrupted. Exemplary marker genes include antibiotic resistance markers, drug resistance markers, and green fluorescent protein, among others. A preferred construct is shown in Fig. 2A and starting materials used to make this construct in Figs. 3A and 3B. Other constructs containing two regions of homology to an endogenous immunoglobulin gene, which flank a positive selection marker (e.g., an antibiotic resistance gene) that is operably linked to a promoter, may be generated using standard molecular biology techniques and used in the methods of the present invention.<br>
The mu knockout construct shown in Figs. 2A and 3C was designed to remove the exons encoding the bovine immunoglobulin heavy chain constant region, designated as "C-mu exons 1 -4" and the two exons encoding the transmembrane domain, designated "TM exons".<br>
To construct this vector, the region designated as "1", anXbal-Xhol fragment from the genomic mu heavy chain bovine sequence, was subcloned into the commercial DNA vector, pBluescript (Stratagene, LaJolla, California), previously cut with the enzymes Xbal and Xhol. Once this fragment was cloned, there was a Notl restriction enzyme recognition sequence adjacent to<br>
iheXbal site, used to insert a Notl fragment of approximately 3.5 Kb. This fragment contains a neomycin resistance marker, described further below. If desired, other mu knockout constructs may be constructed using the genomic mu heavy chain sequence from another ungulate breed, species, or genus (e.g., the mu heavy chain sequence deposited as Genbank accession number U63637 from a Swiss Bull/Holstein cross).<br>
Once fragment "1" and the neomycin resistance marker were joined together into pBluescript, there remained a Sad site adjacent to the neomycin resistance marker. The new construct was linearized with Sad and converted to a blunt end by filling in the sticky ends left from the Sad digest, using DNA polymerase.<br>
The fragment designated "2" was isolated as an Xhol-Bstl\01\ fragment and converted to a blunt-ended fragment by filling in the sticky ends left from the Xhol and 55/11071 enzymes, using DNA polymerase.<br>
Once finished, the final construct contained region 2, the neomycin resistance marker and region 1, respectively.<br>
For transfection of bovine fibroblasts, the construct was digested with the restriction enzyme, Kpnl (two Kpn\ sites are shown in the diagram) and the DNA fragment was used for homologous recombination.<br>
The neomycin resistance construct was assembled as follows. A construct designated "pSTneoB" (Katoh et al., Cell Struct. Funct. 12:575, 1987; Japanese Collection of Research Biologicals (JCRB) deposit number VE039) was designed to contain a neomycin resistance gene under the control of an SV40 promoter and TK enhancer upstream of the coding region. Downstream of the coding region is an SV40 terminator sequence. The neo cassette was excised from "pSTneoB" as a Xhol fragment. After the ends of the fragment were converted to blunt ends using standard molecular biology techniques, the blunt ended fragment was cloned into the EcoRV site in the vector, pBS246 (Gibco/Life Technologies). This site is flanked by loxP sites. The new construct, designated "pLoxP-STNeoR", was used to generate the mu<br>
knockout DNA construct. The desired fragment of this construct is flanked by loxP sites and Notl sites, which were originally present in the pBS246 cloning vector.   The desired Notl fragment, which contains loxP-neo-loxP, was used for replacement of the immunoglobulin mu constant region exons. The SV40 promoter operably linked to the neomycin resistance gene activates the transcription of the neomycin resistance gene, allowing cells in which the desired Notl fragment has replaced the mu constant region exons to be selected based on their resulting antibiotic resistance.<br>
After a cell line is obtained in which the IgM heavy chain allele has been effectively disrupted, it is used as a nuclear transfer donor to produce a cloned ungulate fetus (for example, a cloned bovine fetus) and eventually a fetus or animal wherein one of the IgM heavy alleles is disrupted. Thereafter, a second round of gene targeted disruption can be effected using somatic cells derived therefrom, e.g., fibroblasts, in order to produce cells in which the second IgM heavy chain allele is inactivated, using a similar vector, but containing a different selectable marker.<br>
Preferably, concurrent to the first targeted gene disruption, a second ungulate (for example, bovine) somatic cell line is also genetically modified, which similarly may be of male or female origin. If the first cell line manipulated is male, it is preferable to modify a female cell line; vice versa if the first cell line manipulated is female, it is preferable to select a male cell line. Again, preferably, the manipulated cells comprise ungulate (for example, bovine) fetal fibroblasts.<br>
In a preferred embodiment, the female fetal fibroblast is genetically modified so as to introduce a targeted disruption of one allele of the Ig lambda light chain gene. This method similarly is carried out using a vector having regions of homology to the ungulate (for example, bovine) Ig lambda light chain, and a selectable marker, which DNA construct is designed such that upon integration and homologous recombination with the endogenous Ig light chain results in disruption (inactivation) of the targeted Ig lambda light gene.<br>
Once a female fibroblast cell line is selected having the desired targeted disruption, it similarly is utilized as a donor cell for nuclear transfer or the DNA from such cell line is used as a donor for nuclear transfer.<br>
Alternatively, this cell may be subjected to a second round of homologous recombination to inactivate the second Ig lambda light chain using a similar DNA construct to that used to disrupt the first allele, but containing a different selectable marker.<br>
Methods for effecting nuclear transfer, and particularly for the production of cloned bovines and cloned transgenic bovines have been reported and are described in U.S. Patent No. 5,945,577. Alternatively, the nuclear transfer techniques disclosed in any one of PCT Publication Nos. WO95/16670, WO96/07732, W097/07669, or WO97/07668 (collectively, Roslin Methods) may be used. The Roslin methods differ from the University of Massachusetts techniques in that they use quiescent rather than proliferating donor cells. All of these patents are incorporated by reference herein in their entirety. These nuclear transfer procedures will produce a transgenic cloned fetus which can be used to produce a cloned transgenic bovine offspring, for example, an offspring which comprises a targeted disruption of at least one allele of the Ig light chain gene and/or IgM gene. After such cell lines have been created, they can be utilized to produce a male and female heavy and light chain hemizygous knockout (M and F Hemi H/L) fetus and offspring. Moreover, these techniques are not limited to use for the production of transgenic bovines; the above techniques may be used for nuclear transfer of other ungulates as well.<br>
Following nuclear transfer, production of desired animals may be affected either by mating the ungulates or by secondary gene targeting using the homologous targeting vector previously described.<br>
As noted previously, a further object of the invention involves creating male and female heavy and light chain hemizygous knockouts wherein such hemizygous knockouts are produced using the cell lines already described.<br>
This may be affected either by mating of the offspring produced according to the above described methods, wherein an offspring which comprises a disrupted allele of the IgM heavy chain gene is mated with another offspring which comprises a disrupted allele of the Ig light chain. Alternatively, this may be affected by secondary gene targeting by manipulating a cell that is obtained from an offspring produced according to the above-described procedures. This will comprise effecting by homologous recombination targeted disruption of an allele of the IgM heavy chain gene or allele of the Ig light chain. After a cell line is produced which comprises a male and female heavy and light chain hemizygous knockout (M and F Hemi H/L) it will be used to produce a fetus or calf that comprises such a knockout. As noted, this is effected either by mating or secondary gene targeting.<br>
Once the male and female heavy and light chain hemizygous knockouts are obtained, cells from these animals may be utilized to create homozygous knockout (Homo H/L) fetuses. Again, this is affected either by sequential gene targeting or mating. Essentially, if affected by mating, this will involve mating the male heavy and light chain hemizygous knockout with a female heavy and light chain hemizygous knockout and selection of an offspring that comprises a homozygous knockout. Alternatively, the cells from the hemizygous knockout described above may be manipulated in tissue culture, so as to knock out the other allele of the IgM or Ig light chain (lambda) gene. Secondary gene targeting may be preferred to mating as this may provide for more rapid results, especially given that the gestation period of ungulates, such as bovines, is relatively long.<br>
During this sequential knockout and/or breeding strategy, one or both allele of the prion gene can be mutated in any donor cell produced during this process (e.g., a donor cell with or without a mutation in an endogenous Ig nucleic acid and with or without a xenogenous Ig nucleic acid). The ungulates generated using the methods described herein can thus have any number of desired mutations in a prion gene, Ig heavy chain gene, Ig light chain gene,<br>
alpha-(l,3)-galactosyltransferase gene, and/or J chain gene and can have one or more xenogenous Ig and/or J chain genes.<br>
Knockout procedures to produce transgenic ungulates that express human Igs<br>
Approaches for the production of Homo H/L fetuses or calves are summarized in Fig. 1. There are three schemes outlined therein. The first relies on successive knockouts in regenerated fetal cell lines. This approach is the technically most difficult and has the highest level of risk but as noted above potentially yields faster results than breeding approaches. The other two schemes rely on breeding animals. In the second scheme, only single knockouts of heavy and light chain genes are required in male and female cell lines, respectively. This scheme does not rely on regeneration of cell lines and is technically the simplest approach but takes the longest for completion. Scheme 3 is an intermediate between schemes 1 and 2. In all schemes only Homo H/L fetuses are generated because of potential difficulties in survival and maintenance of Homo H/L knockout calves. If necessary, passive immunotherapy can be used to increase the survival of Homo H/L knockout calves.<br>
Experimental design<br>
The present invention preferably involves the production of a hemizygous male heavy chain knockout (M Hemi H) and a hemizygous female light chain knockout (F Hemi L) and the production of 40 day fetuses from these targeted deletions. The cells from the embryos are harvested, and one allele of the light locus is targeted in the M Hemi H cells and one allele of the heavy chain locus is targeted in the F Hemi L cells resulting in cells with hemizygous deletions of both the H and L loci (Hemi H/L). These cells are used to derive 40 day fetuses from which fibroblasts are isolated.<br>
The M Hemi H/L fibroblasts are targeted with the other H chain allele to create M Homo H/Hemi L, and the F Hemi H/L are targeted with the other L chain allele to create F Homo L/Hemi H. In order to create homozygous deletions, higher drug concentrations are used to drive homozygous targeting. However, it is possible that this approach may not be successful and that breeding may be necessary. An exemplary strategy that relies on cre/lox targeting of the selection cassette allows the same selective systems to be used for more than one targeted deletion. These fibroblasts are cloned and 40 day fetuses harvested and fibroblast cells isolated. The fetal cells from this cloning are targeted to produce homozygous deletions of either the H or L loci resulting in M Homo H/L and F Homo H/L fetal fibroblasts. These fibroblasts are cloned and 40 day fetuses derived and fibroblasts isolated. The Homo H/L fetal fibroblasts are then used for incorporation of the HAC optionally by the use of breeding procedures.<br>
Library construction<br>
Fetal fibroblast cells are used to construct a genomic library. Although it is reported to be significant that the targeting construct be isogenic with the cells used for cloning, it is not essential to the invention. For example, isogenic, substantially isogenic, or nonisogenic constructs may be used to produce a mutation in an endogenous immunoglobulin gene. In one possible method, Holstein cattle, which genetically contain a high level of inbreeding compared to other cattle breeds, are used. We have not detected any polymorphisms in immunoglobulin genes among different animals. This suggests that sequence homology should be high and that targeting with nonisogenic constructs should be successful.<br>
A library is constructed from one male cell line and one female cell line at the same time that the "clonability" testing is being conducted. It is envisioned that at the end of the process, a library will be produced and a<br><br>
number of different fetal cell lines will be tested and one cell line chosen as the best for cloning purposes.<br>
Genomic libraries are constructed using high molecular weight DNA isolated from the fetal fibroblast cells. DNA is size fractionated and high molecular weight DNA between 20-23 Kb is inserted into the lambda phage vector LambdaZap or LambdaFix. The inventors have had excellent success with Stratagene prepared libraries. Therefore, DNA is isolated and the size selected DNA is sent to Stratagene for library preparation. To isolate clones containing bovine heavy and light chains, radiolabeled IgM cDNA and radiolabeled light chain cDNA is used. Additionally, light chain genomic clones are isolated in case it is necessary to delete the locus. Each fetal cell library is screened for bovine heavy and light chain containing clones. It is anticipated that screening approximately 105 - 106 plaques should lead to the isolation of clones containing either the heavy chain or light chain locus. Once isolated, both loci are subcloned into pBluescript and restriction mapped. A restriction map of these loci in Holsteins is provided in Fig. 2B (Knight et al. J Immunol 140: 3654-3659, 1988). Additionally, a map from the clones obtained is made and used to assemble the targeting construct.<br>
Production of targeting constructs<br>
Once the heavy and light chain genes are isolated, constructs are made. The IgM construct is made by deleting the IgM constant region membrane domain. As shown by Rajewsky and colleagues in mice, deletion of the membrane domain of IgM results in a block in B-cell development since surface IgM is a required signal for continued B-cell development (Kitamura et al, Nature 350:423-426). Thus homozygous IgM cattle lack B-cells. This should not pose a problem since in the present strategy no live births of animals lacking functional Ig are necessary. However, if necessary, passive immunotherapy may be used to improve the survival of the animals until the last step when the human Ig loci are introduced.<br>
An exemplary targeting construct used to effect knockout of the IgM heavy chain allele is shown in Fig. 2A. For the heavy chain, the membrane IgM domain is replaced with a neomycin cassette flanked by lox P sites. The attached membrane domain is spliced together with the neo cassette such that the membrane domain has a TAG stop codon inserted immediately 5' to the lox P site ensuring that the membrane domain is inactivated. This is placed at the 5' end of the targeting construct with approximately 5-6 kilobases of 3' chromosomal DNA.<br>
If increasing drug concentrations does not allow deletion of the second allele of either IgM heavy or light chains, the cre/lox system (reviewed in Sauer, 1998, Methods 14:381-392) is used to delete the selectable marker. As described below, the cre/lox system allows the targeted deletion of the selectable marker. All selectable markers are flanked with loxP sequences to facilitate deletion of these markers if this should be necessary.<br>
The light chain construct contains the bovine lambda chain constant region (e.g., the lambda light chain constant region found in GenBank accession number AF396698 or any other ungulate lambda light chain constant region) and a puromycin resistance gene cassette flanked by lox P sites and will replace the bovine gene with a puromycin cassette flanked by lox P sites. Approximately 5-6 kilobases of DNA 3' to the lambda constant region gene will be replaced 3' to the puromycin resistance gene. The puromycin resistance gene will carry lox P sites at both 5' and 3' ends to allow for deletion if necessary. Due to the high degree of homology between ungulate antibody genes, the bovine lambda light chain sequence in GenBank accession number AF396698 is expected to hybridize to the genomic lambda light chain sequence from a variety of ungulates and thus may be used in standard methods to isolate various ungulate lambda light chain genomic sequences. These genomic sequences may be used in standard methods, such as those described herein, to generate knockout constructs to inactivate endogenous lambda light chains in any ungulate.<br>
A kappa light chain knockout construct may be constructed similarly using the bovine kappa light chain sequence in Fig. 3G or any other ungulate kappa light chain sequence. This bovine kappa light chain may be used as a hybridization probe to isolate genomic kappa light chain sequences from a variety of ungulates. These genomic sequences may be used in standard methods, such as those described herein, to generate knockout constructs to inactivate endogenous kappa light chains in any ungulate.<br>
Additional ungulate genes may be optionally mutated or inactivated. For example, the endogenous ungulate Ig J chain gene may be knocked out to prevent the potential antigenicity of the ungulate Ig J chain in the antibodies of the invention that are administered to humans. For the construction of the targeting vector, the cDNA sequence of the bovine Ig J chain region found in Genbank accession number U02301 may be used. This cDNA sequence may be used as a probe to isolate the genomic sequence of bovine Ig J chain from a BAG library such as RFC 1-42 (BACPAC in Oakland, CA) or to isolate the genomic sequence of the J chain from any other ungulate. Additionally, the human J chain coding sequence may be introduced into the ungulates of present invention for the functional expression of human IgA and IgM molecules. The cDNA sequence of human J chain is available from GenBank accession numbers AH002836, M12759, and M12378. This sequence may be inserted into an ungulate fetal fibroblast using standard methods, such as those described herein. For example, the human J chain nucleic acid in a HAG, YAC vector, BAG vector, cosmid vector, or knockin construct may be integrated into an endogenous ungulate chromosome or maintained independently of endogenous ungulate chromosomes. The resulting transgenic ungulate cells may be used in the nuclear transfer methods described herein to generate the desired ungulates that have a mutation that reduces or eliminates the expression of functional ungulate J chain and that contain a xenogenous nucleic acid that expresses human J chain.<br>
Additionally, the ungulate a-(l,3)-galactosyltransferase gene may be mutated to reduce or eliminate expression of the galactosyl(ccl,3)galactose epitope that is produced by the a-(l,3)-galactosyltransferase enzyme. If human antibodies produced by the ungulates of the present invention are modified by this carbohydrate epitope, these glycosylated antibodies may be inactivated or eliminated, when administered as therapeutics to humans, by antibodies in the recipients that are reactive with the carbohydrate epitope. To eliminate this possible immune response to the carbohydrate epitope, the sequence of bovine alpha-(l,3)-galactosyltransferase gene may be used to design a knockout construct to inactive this gene in ungulates (Genbank accession number J04989; Joziasse et al, J. Biol. Chem. 264:14290-14297, 1989). This bovine sequence or the procine alpha-(l,3)-galactosyltransferase sequence disclosed in U.S. Patent Nos. 6,153,428 and 5,821,117 may be used to obtain the genomic alpha-(l,3)-galactosyltransferase sequence from a variety of ungulates to generate other ungulates with reduced or eliminated expression of the galactosyl(al,3)galactose epitope.<br>
If desired, the ungulate prion gene may be mutated or inactivated to reduce the potential risk of an infection such as bovine spongiform encephalopathy (BSE). For the construction of the targeting vector, the genomic DNA sequence of the bovine prion gene may be used (GenBank accession number AJ298878), as described in Example 1. Alternatively, this genomic prion sequence may be used to isolate the genomic prion sequence from other ungulates. The prion gene may be inactivated using the methods described herein. Alternatively, the previously described methods for mutating the alpha-(l,3)-galactosyltransferase gene or prion gene in sheep (Denning et al., Nature Biotech., 19: 559-562, 2001) can be adapted based on the teachings described herein. For example, the improved methods that are described herein for culturing donor cells (e.g., fetal fibroblasts), generating knckout vectors, transfecting the donor cells with the knockout vectors (e.g., transfection in the presence of spermidine), and/or transfering the genetic material from the<br>
knockout cells into an oocyte for generation of a transgenic ungulate can be used.<br>
For targeting the second allele of each locus, it may be necessary to assemble a new targeting construct containing a different selectable marker, if the first selectable marker remains in the cell. As described in Table 5, a variety of selection strategies are available and may be compared and the appropriate selection system chosen. Initially, the second allele is targeted by raising the drug concentration (for example, by doubling the drug concentration). If that is not successful, a new targeting construct may be employed.<br>
The additional mutations or the gene inactivation mentioned above may be incorporated into the ungulates of the present invention using various methodologies. Once a transgenic ungulate cell line is generated for each desired mutation, crossbreeding may be used to incorporate these additional mutations into the ungulates of the present invention. Alternatively, fetal flbroblast cells that have these additional mutations can be used as the starting material for the knockout of endogenous Ig genes and/or the introduction of xenogenous Ig genes. Also, fetal flbroblast cells having a knockout mutation in endogenous Ig genes and/or containing xenogenous Ig genes can be uses as a starting material for these additional mutations or inactivations.<br>
Targeting constructs are introduced into embryonic fibroblasts, e.g., by electroporation. The cells that incorporate the targeting vector are selected by the use of the appropriate antibiotic. Clones that are resistant to the drug of choice will be selected for growth. These clones are then subjected to negative selection with gancyclovir, which will select those clones that have integrated appropriately. Alternatively, clones that survive the drug selection are selected by PCR. It is estimated that it will be necessary to screen at least 500-1000 clones to find an appropriately targeted clone. The inventors' estimation is based on Kitamura (Kitamura et al., Nature 350:423-426, 1991), who found that when targeting the membrane domain of IgM heavy chain constant region approximately 1 in 300 neo resistant clones were properly targeted. Thus, it is proposed to pool clones into groups of 10 clones in a 96 well plate and screen pools of 10 clones for the targeted clones of choice. Once a positive is identified, single clones isolated from the pooled clone will be screened. This strategy should enable identification of the targeted clone.<br>
Because fibroblasts move in culture it is difficult to distinguish individual clones when more than approximately ten clones are produced per dish. Further, strategies may be developed for clonal propagation with high efficiency transfection. Several reasonable strategies, such as dilution cloning, may be used.<br>
Cre/Lox excision of the drug resistance marker<br>
As shown above, exemplary targeting constructs contain selectable markers flanked by loxP sites to facilitate the efficient deletion of the marker using the cre/lox system. Fetal fibroblasts carrying the targeting vector are transfected via electroporation with a Cre containing plasmid. A recently described Cre plasmid that contains a GFPcre fusion gene (Gagneten et al, Nucleic Acids Res. 25:3326-31, 1997) may be used. This allows the rapid selection of all clones that contain Cre protein. These cells are selected either by FACS sorting or by manual harvesting of green fluorescing cells via<br>
micromanipulation. Cells that are green are expected to carry actively transcribed Cre recombinase and hence delete the drug resistance marker. Cells selected for Cre expression are cloned and clones analyzed for the deletion of the drug resistance marker via PCR analysis. Those cells that are determined to have undergone excision are grown to small clones, split and one aliquot is tested in selective medium to ascertain with certainty that the drug resistance gene has been deleted. The other aliquot is used for the next round of targeted deletion.<br>
Application of targeting strategies to altering immunoglobulin genes of other ungulates<br>
To alter immunoglobulin genes of other ungulates, targeting vectors are designed to contain three main regions. The first region is homologous to the locus to be targeted. The second region is a drug selection marker that specifically replaces a portion of the targeted locus. The third region, like the first region, is homologous to the targeted locus but is not contiguous with the first region in the wild type genome. Homologous recombination between the targeting vector and the desired wild type locus results in deletion of locus sequences between the two regions of homology represented in the targeting vector and replacement of that sequence with a drug resistance marker. In preferred embodiments, the total size of the two regions of homology is approximately 6 kilobases, and the size of the second region that replaces a portion of the targeted locus is approximately 2 kilobases. This targeting strategy is broadly useful for a wide range of species from prokaryotic cells to human cells. The uniqueness of each vector used is in the locus chosen for gene targeting procedures and the sequences employed in that strategy. This approach may be used in all ungulates, including, without limitation, goats (Capra hircus), sheep (Ovis aries), and the pig (Sus scrufa), as well as cattle (Bos taurus).<br>
The use of electroporation for targeting specific genes in the cells of ungulates may also be broadly used in ungulates. The general procedure described herein is adaptable to the introduction of targeted mutations into the genomes of other ungulates. Modification of electroporation conditions (voltage and capacitance) may be employed to optimize the number of transfectants obtained from other ungulates.<br>
In addition, the strategy used herein to target the heavy chain locus in cattle (i.e., removal of all coding exons and intervening sequences using a vector containing regions homologous to the regions immediately flanking the removed exons) may also be used equally well in other ungulates. For example, extensive sequence analysis has been performed on the immunoglobulin heavy chain locus of sheep' (Ovis aries), and the sheep locus is highly similar to the bovine locus in both structure and sequence (GenBank accession numbers Z71572, Z49180 through Z49188, M60441, M60440, API72659 through API72703). In addition to the large number of cDNA sequences reported for rearranged Ovis aries immunoglobulin chains, genomic sequence information has been reported for the heavy chain locus, including the heavy chain 5'enhancer (GenBank accession number Z98207), the 3'mu switch region (Z98680) and the 5'mu switch region (Z98681). The complete mRNA sequence for the sheep secreted form of the heavy chain has been deposited as accession number X59994. This deposit contains the entire sequence of four coding exons, which are very homologous to the corresponding bovine sequence.<br>
Information on the sheep locus was obtained from GenBank and used to determine areas of high homology with bovine sequence for the design of primers used for PCR analysis. Because non-isogenic DNA was used to target bovine cells, finding areas of high homology with sheep sequence was used as an indicator that similar conservation of sequences between breeds of cow was likely. Given the similarity between the sequences and structures of the bovine and ovine immunoglobulin loci, it would be expected that the targeting<br>
strategies used to remove bovine immunoglobulin loci could be successfully applied to the ovine system. In addition, existing information on the pig (Sus scrofa, accession number S42881) and goat (Capra hircus, accession number AF140603) indicates that the immunoglobulin loci of both of these species are also sufficiently similar to the bovine loci to utilize the present targeting strategies.<br>
EXAMPLE 5: Bovine IgM knockout<br>
The following procedures were used to generate bovine fibroblast cell lines in which one allele of the immunoglobulin heavy chain (mu) locus is disrupted by homologous recombination. A DNA construct for effecting an IgM knockout was generated by the removal of exons 1-4 of the Mu locus (corresponds to IgM heavy chain gene), which were replaced with a copy of a neomycin resistance gene. Using this construct, neomycin resistant cell lines have been obtained which were successfully used in nuclear transfer procedures, and blastocysts from these cell lines have been implanted into recipient cows. Additionally, some of these blastocysts were tested to confirm that targeted insertion occurred appropriately in the mu locus using PCR procedures. Blastocysts resulting from nuclear transfer procedures from several of the cell lines obtained indicated that heterozygous IgM-knockout fetuses were in gestation. Additionally, both male and female cell lines that comprise a single IgM heavy chain (mu) knockout have been produced. It is anticipated that mating of animals cloned from these cell lines will give rise to progeny wherein both copies of mu are inactivated. As described herein, the progeny can be mated with prion knockout ungulates to produce offspring with mutations in the IgM and prion genes. Alternatively, a cell from the progeny can be genetically modified to inactivate one or both alleles of the prion gene as described in Example 1. These procedures are discussed in greater detail below.<br>
DNA construct<br>
The DNA used in all transfections described in this document was generated as follows. The four main exons (excluding the transmembrane domain exons), CHI-4, are flanked by anXhoI restriction site at the downstream (CH4) end and anXbal site at the upstream (CHI) end. The construct used for the transfection procedure consisted of 1.5 kb of genomic sequence downstream of theXhol site and 3.1 Kb of genomic sequence upstream of the Xbal site (Figs. 3D and 3E). These sequences were isolated as described herein from a Holstein cow from a dairy herd in Massachusetts.   A neomycin resistance marker was inserted between these two fragments on a 3.5 Kb fragment, replacing 2.4 Kb of DNA, originally containing CH1-4, from the originating genomic sequence. The backbone of the vector was pBluescriptll SK+ (Stratagene) and the insert of 8.1 Kb was purified and used for transfection of bovine fetal fibroblasts. This construct is shown in Figs. 3A-3C.   Other mu knockout constructs containing other homologous regions and/or containing another antibiotic resistance gene may also be constructed using standard methods and used to mutate an endogenous mu heavy chain gene.<br>
Transfection/knockout procedures<br>
Transfection of fetal bovine was performed using a commercial reagent, Superfect Transfection Reagent (Qiagen, Valencia, CA, USA).<br>
Bovine fibroblasts were generated from disease-tested male Charlais cattle at Hematech's Kansas facility and sent to Hematech's Worcester Molecular Biology Labs for use in all experiments described. Any other ungulate breed, genus, or species may be used as the source of donor cells (e.g., somatic cells such as fetal fibroblasts). The donor cells are genetically modified to contain a mutation that reduces or eliminates the expression of functional, endogenous Ig.<br>
The medium used for culture of bovine fetal fibroblasts consisted of the following components: 500 ml Alpha MEM (Bio-Whittaker #12-169F); 50 ml fetal calf serum (Hy-Clone #A-1111-D); 2 ml antibiotic/antimyotic (Gibco/BRL #15245-012); 1.4 ml 2-mercaptoethanol (Gibco/BRL #21985-023); 5.0 ml L-Glutamine (Sigma Chemical #G-3126); and 0.5 ml tyrosine tartrate (Sigma Chemical #T-6134)<br>
On the day prior to transfection procedures, cells were seeded in 60 mm tissue culture dishes with a targeted confluency of 40-80% as determined by microscopic examination.<br>
On the day of transfection, 5 jig of DNA, brought to a total volume of 150 jil in serum-free, antibiotic-free medium, was mixed with 20 ul of Superfect transfection reagent and allowed to sit at room temperature for 5-10 minutes for DNA-Superfect complex formation. While the complex formation was taking place, medium was removed from the 60 mm tissue culture dish containing bovine fibroblasts to be transfected, and cells were rinsed once with 4 ml of phosphate-buffered saline. One milliliter of growth medium was added to the 170 ul DNA/Superfect mixture and immediately transferred to the cells in the 60 mm dish. Cells were incubated at 38.5C, 50% carbon dioxide for 2.5 hours. After incubation of cells with the DNA/Superfect complexes, medium was aspirated off and cells were washed four times with 4 ml PBS. Five ml of complete medium were added and cultures were incubated overnight at 38.5C, 5% C02- Cells were then washed once with PBS and incubated with one ml of 0.3% trypsin in PBS at 37C until cells were detached from the plate, as determined by microscopic observation. Cells from each 60 mm dish were split into 24 wells of a 24 well tissue culture plate (41.7 ul/well). One milliliter of tissue culture medium was added to each well and plates were allowed to incubate for 24 hours at 38.5C and 5% CO2 for 24 hours.<br>
During all transfection procedures, sham transfections were performed using a Superfect/PBS mixture containing no DNA, as none of those cells would be expected to contain the neomycin resistance gene and all cells would<br>
be expected to die after addition of G418 to the tissue culture medium. This served as a negative control for positive selection of cells that received DNA.<br>
After the 24 hour incubation, one more milliliter of tissue culture medium containing 400 jig G418 was added to each well, bringing the final G418 concentration to 200 ug/ml. Cells were placed back into the incubator for 7 days of G418 selection. During that period, both transfected and sham transfection plates were monitored for cell death and over 7 days, the vast majority of wells from the sham transfections contained few to no live cells while plates containing cells that received the DNA showed excellent cell growth.<br>
After the 7 day selection period, the cells from wells at 90-100% confluency were detached using 0.2 ml 0.3% trypsin in PBS and were transferred to 35 mm tissue culture plates for expansion and incubated until they became at least 50% confluent, at which point, cells were trypsinized with 0.6 ml 0.3% trypsin in PBS. From each 35 mm tissue culture plate, 0.3 ml of<br>
A<br>
the 0.6 ml cell suspension was transferred to a 12.5 cm tissue culture flask for further expansion. The remaining 0.3 ml was reseeded in 35 mm dishes and incubated until they attained a minimal confluency of approximately 50%, at which point cells from those plates were processed for extraction of DNA for PCR analysis. Flasks from each line were retained in the incubator until they had undergone these analyses and were either terminated if they did not contain the desired DNA integration or kept for future nuclear transfer and cryopreservation.<br>
Screening for targeted integrations<br>
As described above the DNA source for screening of transfectants containing the DNA construct was a 35 mm tissue culture dish containing a passage of cells to be analyzed. DNA was prepared as follows and is adapted from a procedure published by Laird el al, Nucleic Acids Res., 19:4293). Briefly, DNA was prepared as follows. A cell lysis buffer was prepared with<br>
the following components: 100 mM Tris-HCl buffer, pH 8.5; 5 mM EDTA, pH 8.0; 0.2% sodium dodecyl sulfate; 200 mM NaCl; and 100 ug/ml Proteinase K.<br>
Medium was aspirated from each 35 mm tissue culture dish and replaced with 0.6 ml of the above buffer. Dishes were placed back into the incubator for three hours, during which time cell lysis and protein digestion were allowed to occur. Following this incubation, the lysate was transferred to a 1.5 ml microfuge tube and 0.6 ml of isopropanol was added to precipitate the DNA. Tubes were shaken thoroughly by inversion and allowed to sit at room temperature for 3 hours, after which the DNA precipitates were spun down in a microcentrifuge at 13,000 rpm for ten minutes. The supernatant from each tube was discarded and the pellets were rinsed with 70% ethanol once. The 70% ethanol was aspirated off and the DNA pellets were allowed to air-dry. Once dry, each pellet was resuspended in 30-50 ul of Tris (10 mM)-EDTA (1 mM) buffer, pH 7.4 and allowed to hydrate and solubilize overnight.   5-7 microliters of each DNA solution was used for each polymerase chain reaction (PCR) procedure.<br>
Two separate PCR procedures were used to analyze transfectants. The first procedure used two primers that were expected to anneal to sites that are both located within the DNA used for transfection. The first primer sequence is homologous to the neomycin resistance cassette of the DNA construct and the second is located approximately 0.5 Kb away, resulting in a short PCR product of 0.5 Kb. In particular, primers Neol (5 '-CTT GAA GAC GAA AGO GCC TCG TGA TAG GCC-3'; SEQ ID NO: 42) and IN2521 (5'-CTG AGA CTT CCT TTC ACC CTC CAG GCA CCG-3'; SEQ ID NO: 43) were used. A Qiagen PCR kit was used for this PCR reaction. The PCR reaction mixture contained 1 pmole of each primer, 5 ul of 10X reaction buffer, 10 ^1 of Q solution, 5 jal of DNA, and 1 jal of dNTP solution. The reaction mixture was brought to a total volume of 50 |il with H2O. This PCR amplification was performed using an initial denaturing incubation at 94C for two minutes. Then, 30 cycles of denaturation, annealing, and amplification were performed<br>
by incubation at 94C for 45 seconds, 60C for 45 seconds, and 72 C for two minutes. Then, the reaction mixture was incubated at 72C for five minutes and at 4C until the mixture was removed from the PCR machine. Alternatively, any other primers that are homologous to the region of the knockout construct that integrates into the genome of the cells may be used in a standard PCR reaction under appropriate reaction conditions to verify that cells surviving G418 selection were resistant as a result of integration of the DNA construct.<br>
Because only a small percentage of transfectants would be expected to contain a DNA integration in the desired location (the Mu locus), another pair of primers was used to determine not only that the DNA introduced was present in the genome of the transfectants but also that it was integrated in the desired location. The PCR procedure used to detect appropriate integration was performed using one primer located within the neomycin resistance cassette of the DNA construct and one primer that would be expected to anneal over 1.8 Kb away, but only if the DNA had integrated at the appropriate site of the IgM locus (since the homologous region was outside the region included in the DNA construct used for transfection). The primer was designed to anneal to the DNA sequence immediately adjacent to those sequences represented in the DNA construct if it were to integrate in the desired location (DNA sequence of the locus, both within the region present in the DNA construct and adjacent to them in the genome was previously determined). In particular, primers Neol and OUT3570 (5'-CGA TGA ATG CCC CAT TTC ACC CAA GTC TGT C-3;, SEQ ID NO: 44) were used for this analysis. This PCR reaction was performed using a Qiagen PCR kit as described above for the first PCR reaction to confirm the integration of the targeting construct into the cells. Alternatively, this PCR analysis may be performed using any appropriate reaction conditions with any other primer that is homologous to a region of the knockout construct that integrates into the genome of the cells and any other<br>
primer that is homologous to a region in the genome of the cells that is upstream or downstream of the site of integration.<br>
Using these methods, 135 independent 35 mm plates were screened for targeted integration of the DNA construct into the appropriate locus. Of those, DNA from eight plates was determined to contain an appropriately targeted DNA construct and of those, three were selected for use in nuclear transfer procedures. Those cells lines were designated as "8-1C", "5-3C" and "10-1C." Leftover blastocysts not used for transfer into recipient cows were used to extract DNA which was subjected to additional PCR analysis. This analysis was effective using a nested PCR procedure using primers that were also used for initial screening of transfected lines.<br>
As noted above, three cell lines were generated using the gene targeting<br>
construct designed to remove exons 1-4 of the mu locus. These lines all tested<br>
positive for targeted insertions using a PCR based test and were used for<br>
nuclear transfers. Leftover blastocysts resulting from those nuclear transfers<br>
were screened by PCR testing the appropriately targeted construct. The<br>
following frequencies of positive blastocysts were obtained:<br>
Cell Line 8-1C:	6/8<br>
Cell Line 10-1C:	2/16<br>
Cell Line 5-3C:	0/16<br>
Although at forty days of gestation, 11 total pregnancies were detected by ultrasound, by day 60, seven fetuses had died. The remaining four fetuses were processed to regenerate new fetal fibroblasts and remaining organs were used to produce small tissue samples for PCR analysis. The results of the analyses are below:<br>
Line 8-1C:    two fetuses, one fetus positive for targeted insertion by PCR Line 10-1C:   one fetus, positive for targeted insertion by PCR Line 5-3C:     one fetus, negative for targeted insertion by PCR<br>
Surprisingly, although the frequency of 10-1C blastocysts testing positive for targeted insertion was only 2/16, the one viable 60-day fetus<br>
obtained from that cell line was positive as determined by PCR. A positive fetus from 8-1C was also obtained. Southern blot analysis of DNA of all tissue samples is being effected to verify that the construct not only targeted correctly at one end (which is determined by PCR of the shorter region of homology present in the original construct) but also at the other end. Based on results to date, it is believed that two heavy chain knockout fetuses from two independent integration events have been produced. Also, since these fetuses were derived from two different lines, at least one is likely to have integrated construct correctly at both ends. Once the Southern blot analyses have confirmed appropriate targeting of both ends of targeting construct, further nuclear transfers is performed to generate additional fetuses that are carried to term.<br>
Nuclear transfer and embryo transfer<br>
Nuclear transfers were performed with the K/O cell line (8-1-C (18)) and eight embryos were produced. A total of six embryos from this batch were transferred to three disease free recipients at Trans Ova Genetics ("TOG"; Iowa).<br>
Frozen embryos have been transferred to ten disease free recipients to obtain disease free female fibroblast cell lines. Fetal recoveries are scheduled after confirming the pregnancies at 35-40 days.<br>
Pregnancy diagnosis and fetal recovery<br>
Pregnancy status of the eighteen recipients transferred with cloned embryos from knockout fetal cells was checked by ultrasonography. The results are summarized in Table 6.<br>
Pregnancy diagnosis<br>
Pregnancy status of the three recipients to whom cloned embryos were transferred from knockout cells (8-1C) was checked; one was open and the other two required reconfirmation after one month.<br>
Fetal recoveries and establishment of cell lines<br>
Eleven pregnancies with the K/O embryos at 40 days were obtained. Four live fetuses were removed out of these at 60 days. Cell lines were established from all four and cryopreserved for future use. Also we collected and snap froze tissue samples from the fetuses and sent them to Hematech molecular biology laboratory for PCR/Southern blot analysis.<br>
All four of the cell lines were male. In order to secure a female cell line, cell lines were established and cryopreserved for future establishment of K/O cells from the fetuses (six) collected at 55 days of gestation from the pregnancies established at Trans Ova Genetics with disease free recipients. Recently, the existence of a female cell line containing a mu knockout was confirmed. This female cell line may be used to produce cloned animals which may be mated with animals generated from the male cell lines, and progeny screened for those that contain the double mu knockout.<br>
If desired, a cell from the resulting knockout fetus or knockout offspring can be used in a second round of nulear transfer to generate additional cloned offspring. Cells from the initial knockout fetus or knockout offspring may also be frozen to form a cell line to be used as a source of donor cells for the generation of additional knockout ungulates.<br>
EXAMPLE 6: Transgenic ungulates having eeduced a-1,3-galactosyltransferase activity<br>
If desired, transgenic ungulates in which a-1, 3-galactosyltransferase is mutated can be generated to prevent undesired glycosylation of xenogenous antibodies with a galactose a(l,3)-galactose epitope. Bovine fibroblast cell lines in which one allele of the a-l,3-galactosyltransferase locus is mutated were generated by homologous recombination. The DNA construct for generating the a-galactosyltransferase knockout cells was used to prevent transcription of functional, full-length a-galactosyltransferase mRNA by inserting both a puromycin-resistance gene (puro, described herein) and a transcription termination cassette (STOP) in exon 9, which contains the catalytic domain. Thus, the resulting immature a-galactosyltransferase transcripts lack the catalytic domain. The DNA construct (i.e., the a-galactosyltransferase knockout vector) was electroporated into three independent bovine fibroblast cell lines, and then puromycin-resistant colonies were isolated. Based on PCR analysis, homologous recombination in the exon 9 region occurred in some colonies. Thus, bovine fibroblast cell lines in which one allele of al,3-galactosyltransferase locus is mutated were generated. If desired, the second allele can be mutated by using the same knockout vector and a higher concentration of antibiotic to select for homozygous knockout cells or using another knockout vector with a different antibiotic resistance gene. This method may also be applied to cells from other ungulates to generate transgenic cells for use in the nuclear transfer methods described herein to produce transgenic ungulates of the present invention.<br>
These methods are described further below.<br>
Construction of an a-l,3-galactosyltransferase knockout vector<br>
The a-l,3-galactosyltransferase knockout vector was generated as follows (Fig. 23). To isolate genomic DNA around exon 9 of the a-1,3-<br>
galactosyltransferase gene, a DNA probe was amplified by PCR using the following primer pair: 5'-gat gat gtc tec agg atg cc-3' (SEQ ID NO: 61) and 5'-gac aag ctt aat ate cgc agg-3' (SEQ ID NO: 62). Using this probe, a bovine genomic A. phage library was screened, and 7 positive A, phage clones were identified. One clone, which contained DNA from a male Charolais bovine fibrolast cell, was analyzed further by restriction mapping. The Notl-Xhol genomic fragment containing exon 9 was subcloned into pBluescript II SK(-), and then both puro and STOP cassettes were inserted at the Avil site in the Notl-Xhol genomic fragment that is 5' to the catalytic domain. A diphtheria toxin gene (DT-A, Gibco) was also added to the vector construct to kill cells in which the targeting cassette was integrated nonhomologously.<br>
Transfection/knockout procedures<br>
Transfection of three fetal flbroblasts cell lines (two from a male Jersey bovine and one from a female Jersey bovine) was performed using a standard electroporation protocol as follows. The medium used to culture the bovine fetal flbroblasts contained 500 ml Alpha MEM (Gibco, 12561-049), 50 ml fetal calf serum (Hy-Clone #ABL13080), 5 ml penicillin-streptomycin (SIGMA), and 1 ml 2-mercaptoethanol (Gibco/BRL #21985-023). On the day prior to transfection, cells were seeded on a T175 tissue culture flask with a targeted confluency of 80-100 %, as determined by microscopic examination. On the day of transfection, about 107 bovine flbroblasts cells were trypsinized and washed once with alpha-MEM medium. After resuspension of the cells in 800 ul of alpha-MEM, 30 ug of DNA was added to the cell suspension and mixed well by pipetting. The cell-DNA suspension was transferred into an electroporation cuvette and electroporated at 1,000 V and 50 fiF. After that, the electroporated cells were plated onto twenty 24-well plates with the alpha-MEM medium supplemented with the serum. After a 48 hour-culture, the medium was replaced with medium containing 1 ng/ml of puromycin, and the cells were cultured for 2-3 weeks to select puromycin resistant cells. After<br>
selection, all colonies that reached close to 100 % confluency were picked, and genomic DNA was extracted from the colonies to screen for the desired homologous recombination events by PCR.<br>
Screening for targeted integrations<br>
As described above, the genomic DNA was extracted from each 24-well independently using the PUREGENE DNA isolation Kit (Centra Systems) according to the manufacturer's protocol. Each genomic DNA sample was resuspended in 20 ul of 10 mM Tris-Cl (p H8.0) and 1 mM EDTA (EOTA). Screening by PCR was performed using the following primer pair 5'-aag aag aga aag gta gaa gac ccc aag gac-3' (SEQ ID NO: 63) and 5'-cct ggg tat aga cag gtg ggt att gtg c-3' (SEQ ID NO: 64). The sequence of one primer is located in the alpha-l,3-galactosyltransferase knockout vector, and the sequence of the other primer is located just outside of the integrated vector in the targeted endogenous locus (Fig. 23). Therefore, the expected PCR product should be detected only when the knockout vector is integrated into the targeted locus by homologous recombination.<br>
The PCR reaction mixtures contained 18.9 ul water, 3 ul of 10X LA PCR buffer II (Mg2+ plus), 4.8 ul of dNTP mixture, 10 pmol forward primer, 10 pmol of reverse primer, 1 ul of genomic DNA, and 0.3 ul of LA Taq. Forty cycles of PCR were performed by incubating the reaction mixtures at the following conditions: 85C for three minutes, 94C for one minute, 98C for 10 seconds, and 68C for 15 minutes. After PCR, the reaction mixtures were analyzed by electrophoresis. Puromycin-resistant clones that generated PCR products of the expected size were selected (Fig. 23). Thus, bovine fibroblast cell lines in which one allele of the a-l,3-galactosyltransferase locus is mutated by the knockout vector were successfully generated.<br>
EXAMPLE 7: Alternative method for producing transgenic ungulates using adeno-associated viruses to mutate an endogenous gene<br>
Adeno-associated virus (AAV) can be used for specific replacement of targeted sequences present in the genome of cells (Inoue et al, Mol. Ther. 3: 526-530, 2001); Hirata et al., J. Virol. 74: 16536-16542, 2000); Inoue et al., J. Virol. 73: 7376-7380, 1999); and Russell et al., Nat. Genet. 18: 325-330, 1998)). The gene targeting rate is highly efficient in comparison to more conventional gene targeting approaches. AAV has a broad range of host and tissue specificities, including specificity for both bovine and human skin fibroblasts. Thus, AAV can be used to produce transgenic ungulate cells containing one or more mutations in an endogenous immunoglobulin (e.g., mu heavy chain, lambda light chain, kappa light chain, or J chain), alpha-(l,3)-galactosyltransferase, or prion gene. These transgenic cells can then be used in the nuclear transfer methods described herein to produce transgenic ungulates of the present invention.<br>
Using AAV resulted in homologous recombination of the bovine immunoglobulin heavy chain locus at higher frequencies than previously obtained using traditional gene targeting strategies (i.e., electroporation and lipofection procedures). In the first round of gene targeting experiments, five appropriately targeted fibroblast clones were obtained out of 73 stable transductants containing the DNA introduced through an AAV vector.<br>
These experiments were carried out as follows.<br>
AAV knockout vectors<br>
AAV constructs can disrupt a gene either by simple insertion of foreign sequences or replacement of endogenous sequences with new sequence present in the AAV vector. Figure 24 shows an AAV construct in which all four coding exons of the bovine immunoglobulin heavy chain mu constant region are present on a 2822 base pair BamHl-XhoI fragment. A 1.16 Kb fragment containing a neomycin resistance marker present in the commercially available<br>
vector, pMClNeo, was inserted into a Sacll site present in exon 4 of the mu heavy chain locus from a Holstein bovine. This locus is the one contained in the phage clone isolated to generate the knockout vector described in Example 3. To generate the AW vector, the Sacll site in the mu heavy chain locus was filled in to create blunt ends, which were then ligated to blunt Sail linkers (New England Biolabs). Then, the Xhol fragment of pMClNeo, which contains the neomycin resistance gene, was ligated to the Sail site added to the locus through the Sail linker. This ligation can be performed because the Xhol and Sail restriction sites have compatible ends. This knockout vector causes a disruptional insertion of the neomycin resistance gene into the endogenous mu heavy chain gene, thereby inactivating the mu heavy chain gene. This gene inactivation occurs without deleting regions of the endogenous mu locus.<br>
An alternative vector was designed to remove exons 3 and 4 from the endogenous locus during targeting, resulting in the replacement of these two exons with a functional copy of the neomycin resistance gene (Fig. 25). This construct was generated using PCR amplification of genomic DNA from a female Jersey bovine. In particular, the 3'region of homology was amplified using the following primers: 5'-GGG GTC TAG Age aga cac tac act gat ggg ccc ttg gtc c-3'(SEQ ID NO: 65), which adds anXbal restriction site, and 5'-GGG GAA GCT Teg tgt ccc tgg tec tgt ctg aca cag-3' (SEQ ID NO: 66), which adds a Hindlll restriction site. The 5'region of homology was amplified with primers 5'-GGG GCT CGA Ggt egg cga agg atg ggg gga ggt g-3' (SEQ ID NO: 67), which adds a Xhol restriction site, and 5'-GGG GGG TAC Cgc tgg get gag ctg ggc aga gtg gg-3'(SEQ ID NO: 68), which adds a Kpnl restriction site. The capitalized nucleotides in these primer sequences are nucleotides that do not anneal to the mu heavy chain locus but are included in the primers to add restriction sites to facilitate later subcloning steps. The first four guanines are added to separate the restriction sites from the very end of the primers because restriction enzymes do not cleave sites that are at the very end of primers as well as internal sites. The 5'region of homology is 1.5 Kb long and contains<br>
exons 1 and 2. The 5'region of homology also contains the first 25 nucleotides of exon 3 to maintain the splice acceptor site of exon 3. The splice acceptor site allows exon 3 to be used for splicing and thus prevents the possible splicing of exons 1 and 2 to the downstream transmembrane domain to form an aberrant membrane-bound product. The 3'region of homology is 1.24 Kb long and contains the region immediately downstream of exon 4.<br>
For the construct shown in Fig. 24, the targeting cassette was inserted into the AAV vector reported by Ryan et al. (J. Virol. 70:1542-1553, 1996), which contains viral long terminal repeat (LTR) sequences, using standard methods. The AAV vector was packaged into eapsids using the TtetA2 packaging cell line as previously described (Inoue and Russell, J. Virol. 72: 7024-7031, 1998) and purified as previously described (Zolorukhin et al, Gene Therapy, 6: 973-985, 1999). For the construct shown in Fig. 25, the above method or any other standard method can be used to insert the targeting cassette into the AAV vector described by Ryan et al. or any other AAV vector (such as a commercially available vector from Stratagene) and generate viruses containing the vector.<br>
Transduction procedures<br>
Fibroblasts from a female Jersey bovine were seeded into one well of a 48-well tissue culture plate at 40,000 cells per well and cultured in complete medium at 38.5C and 5% CO2 until cells attached to the bottom surface of the well. Once cells adhered, the medium was removed and replaced with 0.2 ml of fresh medium containing AAV particles with the vector shown in Fig. 24 at a multiplicity of infection (MOI) of 500-20,000 particles/cell. The MOI was chosen based on pilot experiments that determined the resulting numbers of colonies and the spacing of the colonies during the drug selection phase. Plates were incubated overnight. After this incubation, the transduced wells were rinsed with calcium and magnesium-free PBS and detached from the wells using either trypsin or the cell dissociation buffer described above. A uniform<br>
cell suspension was obtained by gentle pipetting of the detached cells, and the cells from the well were redistributed among ten 100 mm tissue culture dishes. Dishes were incubated with complete medium overnight.<br>
Following this incubation of the 100 mm dishes, the medium was replaced with selective medium containing G418 at a concentration of 350 micrograms/ml. Selective medium was changed every 2-3 days until colonies were macroscopically visible on the surface of the dish. At that point, individual colonies were picked and transferred into their own vessels.<br>
Regions containing colonies were marked on the outer surface of the tissue culture dish. Once all colonies were circled, medium was aspirated off the plates, and the plates were washed three times with calcium and magnesium-free PBS. After washing, the plates were flooded with a 1:25 dilution of IX trypsin and allowed to sit at room temperature until the colonies had visibly begun to detach from the surface of the plate. Plates were kept stationary to prevent detached colonies from floating to another location of the plate. A pipet tip was used to pick up cell clumps in a volume of 50 microliters, and the contents of the pipet tip were transferred into one well of a 24 well tissue culture plate. Once all colonies were transferred, complete medium containing G418 was added, and the isolated clones were allowed to proliferate to near confluency.<br>
When an individual well was close to confluency, it was washed twice with calcium and magnesium free PBS. Cells were detached using 0.2 ml of cell dissociation buffer. Of this cell suspension, 20 \JL\ was transferred to a new 24 well plate, and the remaining cells were allowed to reattach to the surface of the original 24 well plate following the addition of 2.0 ml of complete medium. The original plate was incubated to 100% confluency. The new plate serves as a source of appropriately targeted cells for future bovine cloning procedures.<br>
When a well from the original 24 well plate became 100% confluent, the medium was removed, and the cells were washed once with PBS. PBS was removed and replaced with a cell lysis buffer adopted from Laird et al.<br>
(Nucleic Acids Res. 19:4293, 1991). Briefly, 0.2 ml of lysis buffer containing 200 mM NaCl, lOOmM Tris-HCl pH 8.5, 5 mM EDTA, 0.2% SDS, and 100 ug/ml proteinase K was added to the well. The plate was returned to the incubator for between three hours and overnight. The viscous cell lysate was then transferred to a microfuge tube. An equal volume of isopropanol was added to precipitate DNA. Following a 10 minute spin in a microfuge, the supernatant was discarded, and the pellet was washed once with 0.5 ml of 70% ethanol. After removal of the ethanol, the DNA pellet was air-dried and resuspended in 35 microliters of TE buffer (10 mM Tris pH 8 and 1 mM EDTA). Aliquots of 3 ul were used for PCR analysis.<br>
PCR analysis<br>
DNA samples from drug resistant clones transduced with AAV particles were screened for appropriate targeting of the vector using PCR analysis. This screening strategy used one primer that anneals within the DNA encoding the drug selection marker and another primer that anneals within the targeted locus, but outside the sequence present in the AAV targeting particles. PCR products are only detected if the AAV targeting DNA has integrated into the desired location of the endogenous genome.<br>
Results from a single targeting experiment using these AAV particles are shown in Fig. 26. Based on this analysis, five out of 73 independent clones contained the appropriate targeted vector DNA.<br>
This method may also be used with the AAV vector shown in Fig. 25 or with any other appropriate adenovirus or adeno-associated viral vector. If desired, the second mu heavy chain allele can be mutated in the isolated colonies by transducing them with an AAV vector with a different antibiotic resistance gene (i.e., a gene other than a neomycin resistance gene). To select the resulting homozygous knockout cells, the infected cells are cultured in the presence of the corresponding antibiotic. Alternatively, the isolated colonies can be transduced with an AAV vector containing a neomycin resistance gene<br>
and cultured in the presence of a high concentration of antibiotic (i.e., a concentration of antibiotic that kills heterozygous knockout cells but not homozygous knockout cells).<br>
EXAMPLE 8: Evidence for nuclear reprogramming deficiencies in traditional bovine nuclear transplant embryos<br>
Traditional nuclear transplant techniques generally produce a low percentage of live births. As described below, this in efficiency may be due to, at least in part, the inability of the reconstituted oocyte to reprogram the donor cell or donor nucleus to promote the transcription of genes desirable for development of the oocyte and to inhibit the transcription of genes undesirable for development. Examples 11 and 12 describe improved cloning methods that can be used to produce transgenic ungulates with a mutation in a prion gene and optionally with a xenogenous Ig gene in the methods of the present invention.<br>
Distribution of nuclear envelope, nuclear matrix, and chromatin-matrix interface components during bovine preimplantation development<br>
To determine the distribution of nuclear envelope (B-type and A/C-type lamins), nuclear matrix (NuMA), and chromatin-matrix interface (AKAP95) components in preimplantation embryos, bovine embryos were produced by in vitro fertilization (IVF) and examined by immunofluorescence analysis. Bovine in vitro fertilization was performed as described previously (Collas et al, Mol. Reprod. Devel. 34: 212-223, 1993). Briefly, frozen-thawed bovine sperm from a single bull was layered on top of a 45-90% Percoll gradient and centrifuged for 30 minutes at 700 x g. The concentration of sperm in the pellet was determined, and the sperm was diluted such that the final concentration at fertilization was 106 sperm/ml. At 22 hours post maturation, oocytes were washed three times in TL HEPES and placed in 480 ^il fertilization medium.<br>
Twenty microliters sperm suspension was added at 106 sperm/ml for 50 oocytes. Embryos were placed in culture in four-well tissue culture plates containing a monolayer of mouse fetal fibroblasts in 0.5 ml of embryo culture medium covered with 0.3 ml of embryo tested mineral oil (Sigma). Between 25 and 50 embryos were placed in each well and incubated at 38.5C in a 5% CC"2 air atmosphere. Fertilization rates were over 90% as determined by pronuclear development.<br>
For the immunofluorescence analysis of these in vitro fertilized bovine embryos, anti-human lamin B antibodies were obtained from Dr. Jean-Claude Courvalin, CNRS, Paris, France. Anti-lamins A/C monoclonal antibodies were purchased from Santa-Cruz Biotechnology, and anti-NuMA antibodies were obtained from Transduction Laboratories. Anti-rat AKAP95 affinity-purified rabbit polyclonal antibodies were obtained from Upstate Biotechnologies. The in vitro fertilized bovine embryos were settled onto poly-L-lysine-coated glass coverslips, fixed with 3% paraformaldehyde for 15 minutes, and permeabilized with 0.1% Triton X-100 for 15 minutes (Collas et al, J. Cell Biol. 135:1715-1725, 1996). The proteins were blocked with 2% BSA in PBS/0.01% Tween 20 (PBST) for 15 minutes. Primary antibodies (anti-AKAP95, anti-lamin B, anti-LBR, anti-NuMA, and anti-lamins A/C) and secondary antibodies were incubated each for 30 minutes and used at a 1:100 dilution in PBST-BSA. DNA was counterstained with 0.1 ug/ml Hoechst 33342 incorporated in the antifade mounting medium. Samples were mounted onto slides and coverslips sealed with nail polish. Immunofluorescence observations were made on an Olympus BX60 epifluorescence microscope and photographs were taken with a JVC CCD camera and AnalySIS software. Images were processed using the Aldus Photostyler software. Relative quantification of fluorescence signals was performed using the AnalySIS quantification program. Data were expressed as mean relative fluorescence intensities.<br>
Immunofluorescence analysis of bovine embryos showed that B-type lamins were detected at the nuclear periphery (Fig. 29A). Lamins A/C,<br>
however, were not detected at the pronuclear or 8-cell stage. This failure to detect lamins A/C at these early cell stages is expected for a marker of differentiated cells (Guilli et al, EMBO J. 6:3795-3799, 1987). The nuclear matrix structural protein, NuMA, was detected in all the stages that were examined (Fig. 29A). However, in bovine pronuclear stage embryos, NuMA labeling was restricted to the female pronucleus (FPN), the smallest of both pronuclei (Fig. 29A arrows). AKAP95, which was recently characterized in early mouse embryos (Bomar et al, 2002 manuscript submitted) and detected using affinity-purified anti-rat AKAP95 antibodies, was also restricted to the female pronucleus (Fig. 29A). Nevertheless, intranuclear distribution of AKAP95 was observed in nuclei of all blastomeres in subsequent developmental stages (Fig. 29A).<br>
Specificity of immunofluorescence labeling was verified by Western blot analysis of bovine primary fetal fibroblasts and pronuclear stage in vitro fertilized embryos (Fig. 29B). For this analysis, proteins were resolved by 10% SDS-PAGE at 40 mA per gel. Proteins were electrophoretically transferred onto a nitrocellulose membrane in transfer buffer (25 mM TrisHCl, pH 8.3, 192 mM glycine, 20% methanol, and 0.1% SDS) at 100 V for one hour. Membranes were washed for 10 minutes with Tris-buffered saline (TBS; i.e., 140 mM NaCl, 2.7 mM KC1, and 25 mM Tris-HCl at pH 8.0), blocked for one hour with TEST (TBS with 0.05% Tween-20) containing 5% milk, and incubated for 1.5 hours with the following primary antibodies: anti-AKAP95 (1:250 dilution), anti-lamin B (1:1000), anti-LBR (1:500), anti-NuMA (1:500), and anti-lamins A/C (1:500). Blots were washed twice for 10 minutes in TEST and incubated for one hour with horse radish peroxidase (HRP)-conjugated secondary antibodies. Blots were washed twice for 10 minutes in TBS and developed using enhanced chemiluminescence (ECL, Amersham).<br>
All proteins were detected at their expected apparent Mr: 68 kDa (B-type lamins), 70 and 60 kDa (lamins A and C, respectively), -180 kDa (NuMA), and 95 kDa (AKAP95). Altogether, these results indicate that preimplantation<br>
bovine embryos express nuclear structural proteins that can be detected with cross-reacting antibodies. Notably, lamins A/C are not immunologically detected in bovine preimplantation embryos. Because lamins A/C are expressed in somatic cells (Fig. 29B), they potentially constitute molecular markers for nuclear reprogramming in nuclear transplant embryos.<br>
Dynamics of nuclear envelope, NuMa, and AKAP95 in nuclear transplant bovine embryos<br>
The dynamics of nuclear envelope and nuclear matrix structures was examined during traditional nuclear transplantation procedure in bovine. These structures were investigated using antibodies to lamins A/C and B, NuMA, and AKAP95, respectively. To determine the dynamics of these markers during nuclear remodeling, bovine nuclear transplant embryos were produced using primary fetal fibroblasts, which were isolated as described previously, as the donor cells (Kasinathan et al, Biol. Reprod. 64: 1487-1493, 2001). Briefly, cells were harvested from bovine fetuses by trypsinization using 0.08% trypsin and 0.02% EDTA in PBS (trypsin-EDTA). Cells were seeded in a T75 culture flask (Corning) in a-MEM (Gibco) supplemented with 10% fetal bovine serum (FBS; Hyclone), 0.15 g/ml glutamine (Sigma), 0.003% (3-mercaptoethanol (Gibco), and an antibiotic-antimycotic (Gibco). On day three after seeding, cells were harvested with trypsin-EDTA and frozen in a-MEM/DMSO. Gl cells were isolated as described previously (Kasinathan et al, Biol. Reprod. 64: 1487-1493, 2001). Briefly, 24 hours before isolation, 5.0 x 105 cells were plated in a T75 flask containing 10 ml of MEM/FBS. The following day, the plates were washed with PBS, the culture medium was replaced for 1-2 hours, and the plates were shaken for 30-60 seconds on a Vortex at medium speed. The medium was removed, centrifuged at 500 x g for five minutes, and the pellet was resuspended in 250 )il of MEM/FBS. Cell doublets attached by a cytoplasmic bridge were selected using a micropipette and used for nuclear transfer.<br>
Bovine nuclear transfer was carried out as described earlier (Kasinathan et al, Biol. Reprod. 64: 1487-1493, 2001). In vz/ro-matured oocytes were enucleated 18-20 hours post-maturation. After transferring Gl donor cells into the perivitelline space, they were fused using a single electrical pulse of 2.4 kV/cm for 20 microseconds (Electrocell Manipulator 200, Genetronics). At 28-30 hours post maturation (i.e., 28-30 hours after oocytes were placed in maturation medium after collection from ovaries and at least two hours after fusion with donor cells) reconstructed oocytes and parthenogenetic controls were activated with calcium ionophore (5 uM) for four minutes (Cal Biochem) followed by 10 ug cycloheximide and 2.5 ug cytochalasin D (Sigma) in ACM medium (100 mM NaCl, 3 mM KC1, 0.27 mM CaCl2, 25 mM NaHC03, 1 mM sodium lactate, 0.4 mM pyruvate, 1 mM L-glutamine, 3 mg/ml BSA, 1% BME amino acids, and 1% MEM nonessential amino acids, for five hours (Liu et al., Mol. Reprod. Dev. 49: 298-307, 1998). After activation, nuclear transplant embryos or oocytes eggs were washed five times and co-cultured with mouse fetal fibroblasts at 38.5C in a 5% CO2 atmosphere.<br>
Reconstituted embryos were activated using standard methods, and three hours post-fusion, embryos at the premature chromatin condensation (PCC) stage were fixed with paraformaldehyde and analyzed by immunofluorescence using antibodies to lamins A/C, lamin B, NuMA, and AKAP95 (Fig. 30, PCC). Furthermore, groups of nuclear transplant embryos that were allowed to progress to the pronuclear (PN) stage (i.e., 15 hour post-fusion bovine embryos) were analyzed similarly (Fig. 30, nuclear transplant-PN). As controls, parthenogenetic oocytes activated as described herein were also examined at the pronuclear stage (Fig. 30, Parth. PN).<br>
As expected, somatic donor cells (bovine fetal fibroblasts, Fig. 30) expressed all markers with a distribution anticipated from the literature. At the premature chromatin condensation stage, distinct condensed chromosome masses were evidenced by DNA staining with Hoechst 33342. Lamins A/C and B were not detected on or near the condensed chromosomes (Fig. 30,<br><br>
PCC), presumably as a result of their dispersal in the egg cytoplasm. Some labeled NuMA was detected; this NuMA was presumably associated with the spindle poles maintaining the condensed chromosomes. AKAP95, in contrast, was associated with the condensed (PCC) chromosomes. This result is reminiscent of AKAP95 labeling in mitotic human cells (Collas et al, J. Cell Biol. 147:1167-1180, 1999; Steen et al, J. Cell Biol. 150:1251-1262,2000). At the pronuclear stage, all markers were detected. Lamins A/C were present at the pronuclear envelope (Fig. 2, nuclear transplant-PN). This contrasted with their absence from the envelope of control parthenote pronuclei (Fig. 30) and from the envelope of fertilized pronuclei (Fig. 29A). Lamin B was detected in nuclear transplant pronuclei, as in control pronuclei. Likewise, NuMA and AKAP95 decorated the nuclear interior except for the nucleoli. NuMA labeling was consistently brighter in nuclear transplant pronuclei than in control parthenogenetic pronuclei (compare nuclear transplant PN and Parth. PN, Fig. 30). Collectively, these observations indicate that pronuclei of nuclear transplant embryos reassemble the somatic nuclear markers lamins A and C and display strong NuMA staining.<br>
Differential anchoring of AKAP95 in pronuclei of parthenogenetic embryos and nuclear transplant embryos<br>
The A-kinase anchoring protein AKAP95 is a nuclear protein implicated in mitotic chromosome condensation. For use as another molecular marker affecting reprogramming of somatic nuclei after nuclear transplant, the intranuclear anchoring properties of AKAP95 were characterized in bovine nuclear transplant pronuclear stage embryos formed from fetal fibroblasts. Anchoring of AKAP95 in pronuclei from parthenogenetic embryos and nuclei of somatic donor cells was also examined.<br>
Intranuclear anchoring of AKAP95 in pronuclear embryos was examined in situ by extraction of embryos with 0.1% Triton X-100, 1 mg/ml DNAse I, and either 100 or 300 mM NaCl for 30 minutes at room temperature.<br>
As noted above, male pronuclei did not harbor any AKAP95. In contrast, a significant amount of AKAP95 and DNA was resistant to DNAse I and 300 mM NaCl in pronuclei of nuclear transplant embryos, and in donor nuclei in bovine (Fig. 31). B-rype lamins were not extracted by DNAse I and 300 mM NaCl in parthenote or nuclear transplant pronuclei (Fig. 31), suggesting that alterations in AKAP95 and DNA distributions did not result from gross changes in nuclear architecture. These data indicate that, as in somatic nuclei, AKAP95 is more tightly anchored to intranuclear structures in nuclear transplant pronuclei than in parthenogenetic pronuclei in the bovine. Whether this association imposes constraints on DNA organization or results from altered genome organization in nuclear transplant embryos remains to be determined. As DNAse I-resistant DNA is transcriptionally silent, incomplete remodeling of AKAP95 anchoring after nuclear transplantation likely impairs expression of developmentally important genes.<br>
Transcriptional misregulation of lamins A/C in nuclear transplant bovine embryos<br>
A striking observation was that lamins A/C reassemble at the periphery of pronuclei in bovine nuclear transplant embryos, whereas this somatic-specific marker is absent from in vitro fertilized, and parthenogenetic pronuclei. Thus, we investigated whether reassembly of lamins A/C resulted from (i) re-targeting of somatic lamins disassembled at the premature chromatin condensation stage (Fig. 30), (ii) translation and assembly of lamins from a pool of maternal lamin A/C mRNA, or (iii) de novo transcription of the somatic lamin A (LMNA) gene in nuclear transplant pronuclei.<br>
To distinguish between these possibilities, bovine nuclear transplant embryos were produced by either the "traditional" nuclear transplant procedure as described herein, nuclear transplant followed by activation of reconstituted embryos with the protein synthesis inhibitor cycloheximide (CHX), or by nuclear transplant followed by activation in the presence of the RNA<br>
polymerase II (PolII) inhibitor actinomycin D (ActD) to inhibit de novo transcription. For culruring bovine nuclear transplant embryos in cycloheximide, oocytes were activated after nuclear transfer as described above except that oocytes were incubated for 14 hours in cycloheximide (CHX). At 14 hours after activation, oocytes were washed five times and placed in ACM culture medium containing 15 |ig/ml Hoechst 33342 (Sigma) for one hour. After incubation, pronuclear development was observed by epifluorescence microscopy. Pronuclear embryos were then fixed in 3% paraformaldehyde in PBS, washed, and mounted on slides. For culturing bovine nuclear transplant oocytes in actinomycin D, oocytes were activated after nuclear transfer as described above except 5 ug/ml actinomycin D (ActD) was added to the cycloheximide incubation step. After five hours, eggs were washed five times and placed in ACM culture medium containing 5 ng/ml actinomycin D. At 14 hours after activation, eggs were washed five times and placed in ACM culture medium containing 15 |ig/ml Hoechst 33342 (Sigma) for one hour. After incubation, pronuclear development was observed by epifluorescence microscopy. Pronuclear stage embryos were fixed in 3% paraformaldehyde in PBS, washed, and mounted on slides.<br>
Lamin B assembly around nuclear transplant pronuclei was not affected by either protein or RNA synthesis inhibition. This result indicates that lamin B was reassembled from either a previously disassembled somatic pool and/or from a large pool of lamin B in the oocyte cytoplasm. Lamins A/C, which<br>
were detected in nuclear transplant pronuclei (Fig. 30), were absent from nuclei<br>
j reformed after activation with cycloheximide. This result indicates that lamins<br>
A/C assembly requires de novo protein synthesis and that these lamins are not re-targeted from a disassembled somatic pool brought into the oocyte by donor nucleus injection or cell fusion. Furthermore, lamins A/C are not reassembled when embryos are activated in the presence of actinomycin D. This result indicates that lamins A/C reassembly in nuclear transplant pronuclei results from de novo transcription of the LMNA gene in the reconstituted pronucleus.<br>
NuMA, which was detected in nuclear transplant pronuclei, is not reassembled in pronuclei of nuclear transplant embryos activated with cycloheximide, but is faintly detected in pronuclei of actinomycin D-treated nuclear transplant embryos. This finding strongly suggests that NuMA reassembly in nuclear transplant pronuclei requires de novo translation that occurs, at least in part, from a pool of maternal NuMA mRNA. The consistent observation that anti-NuMA labeling is weaker in pronuclei of actinomycin D-treated nuclear transplant embryos compared to control untreated nuclear transplant embryos (compare b' and b'" in Fig. 32) suggests that part of NuMA assembly in nuclear transplant pronuclei results from de novo transcription of the NuMA gene at the pronuclear stage.<br>
Collectively, these results indicate that the LMNA gene is not turned off upon nuclear remodeling after nuclear transplantation. Similarly, the NuMA gene apparently remains active in pronuclear nuclear transplant embryos. It is likely that transient inactivation of these genes takes place during premature chromatin condensation, as anticipated from the highly condensed nature of the chromatin (Fig. 30). These results clearly illustrate incomplete nuclear reprogramming in nuclear transplant embryos produced under the conditions described herein. As discussed earlier for AKAP95, we propose that the persistence of lamins A/C in nuclear transplant pronuclei affects gene expression, such as expression of developmentally important genes. The previously reported interactions of lamins A and C with chromatin proteins and DNA, and the association of these lamins with transcription factors also support this hypothesis.<br>
EXAMPLE 9: Exemplary nuclear reprogramming deficiencies in traditional bovine nuclear transplant embryos<br>
Exemplary differences between naturally-occurring embryos and traditional nuclear transfer embryos are also described below. These differences include differences in pronuclear assembly of differentiated cell-<br>
specific A-type nuclear lamins, enhanced pronuclear NuMA and TATA binding protein concentrations, and increased sensitivity of nuclear matrix -chromatin interface component AKAP95 and DNA to extraction with detergent, DNAse and salt.<br>
For these studies, bovine fetal fibroblasts cell lines were established as described previously (Kasinathan et al., Nat. Biotechnol. 19:1176-1178, 2001 and Kasinathan et al., Biol. Reprod. 64:1487-1493, 2001). Gl-phase fibroblast doublets were isolated from cultures using a previously described shake-off method (Kasinathan et al., Nat. Biotechnol. 19:1176-1178, 2001). In vitro fertilization with in vz'/ro-matured oocytes was carried out as described previously (Collas et al, Mol. Reprod. Dev. 34:224-231, 1993). For nuclear transplantation (NT) and oocyte activation, nuclear transplantation using Gl-phase donor cells was performed at ~20 hours post-maturation (hpm) as reported previously (Kasinathan et al, Nat..Biotechnol. 19:1176-1178, 2001 and Kasinathan et al, Biol. Reprod. 64:1487-1493, 2001). Reconstituted embryos were activated at 28-30 hpm (T=0) with 5 uM calcium ionophore for four minutes followed by 10 u,g/ml CHX and 2.5 ug/ml cytochalasin D for five hours. Embryos were washed and co-cultured with mouse fetal fibroblasts (Kasinathan et al, Biol. Reprod. 64:1487-1493, 2001). When reconstituted embryos were cultured in CHX, oocytes were activated as above and cultured with 2.5 u,g/ml CHX for another nine hours (total, 14 hours in CHX). Embryos were washed thoroughly and cultured as described (Kasinathan et al, Biol. Reprod. 64:1487-1493, 2001). When reconstituted embryos were exposed to ActD, oocytes were activated as above except that 5 (ig/ml ActD was added to the five hour CHX incubation step.<br>
For immunological analysis, cells, oocytes, embryos, nuclei (Example 14), and chromatin masses (Example 14), and were settled onto poly-L-lysine-coated coverslips, fixed with 3% paraformaldehyde for 15 minutes, and permeabilized with 0.1% Triton X-100 for 15 minutes. Proteins were blocked using PBS/2% BSA/0.01% Tween 20. Primary and secondary antibodies<br>
(1:100 dilution) were incubated each for 30 minutes. In particular, rabbit polyclonal antibodies against a peptide of human lamin B were used (Chaudhary etal, J. Cell Biol. 122:295-306, 1993). Goat anti-lamin B polyclonal antibodies, anti-lamin A/C monoclonal antibodies, and anti-TBP antibodies from Santa-Cruz Biotechnology were also used. Anti-NuMA monoclonal antibodies were from Transduction Laboratories, and anti-rat AKAP95 affinity-purified polyclonal antibodies were from Upstate Biotechnologies. DNA was counterstained with 0.1 ug/ml Hoechst 33342. Photographs were taken with a JVC CCD camera, and quantification of immunofluorescence intensity was performed using the AnalySIS software. Data were expressed as a meanSD fluorescence intensity relative to a control in at least three replicates. For in situ extractions, embryos and cells settled on coverlips were incubated for 15 minutes with 0.1% Triton X-100, 1 mg/ml DNAse I, and 300 mM NaCl in Tris-HCl (pH 7.2) prior to immunofluorescence analysis. For immunoblotting, 100 embryos were dissolved in 20 jal SDS sample buffer, proteins resolved by 10% SDS-PAGE, and analyzed with the following antibodies: anti-lamin B, 1:1,000; anti-lamin A/C, 1:250; anti-NuMA, 1:500; anti-AKAP95, 1:250.<br>
Based on the above immunological analysis, the distribution of A/C-and B-type lamins, NuMA and AKAP95 was characterized in bovine fetal fibroblasts commonly used for NT (Fig. 40). In in v/fro-produced bovine preimplantation embryos, lamin B was detected at the nuclear periphery as early as the pronuclear (PN) stage (Fig. 40). Lamin A/C was absent, as expected from a marker of differentiated cells. NuMA and AKAP95 were restricted to the female pronucleus at the pronuclear stage but decorated all nuclei in subsequent stages. Specificity of immunofluorescence data was verified on immnunoblots (Fig. 40).<br>
The dynamics of lamins A/C and B, NuMA, and AKAP95 was examined during morphological nuclear remodeling associated with transplantation of bovine fibroblasts into enucleated oocytes by electrofusion<br>
(Kasinathan et al. Biol. Reprod. 64:1487-1493, 2001). Donor nuclei underwent premature chromatin condensation (PCC) within three hours of fusion (Fig. 41 A). Nuclear lamins and NuMA were redistributed in the oocyte cytoplasm and were absent from PCC chromosomes (Fig. 41 A). AKAP95 was associated with PCC chromosomes, a property reminiscent of mitotic cells. Fourteen hours after start of activation treatment of recipient oocytes, NT embryos displayed fully developed pronuclei (Fig. 41A, NT PN). However, in contrast to pronuclei of parthenogenetic or fertilized embryos, essentially all NT pronuclei expressed strong lamin A/C and NuMA immunoreactiviry, two characteristics of the somatic donor cells (Fig. 41A).<br>
Recipient oocyte activation in the presence of 10 ug/ml of the protein synthesis inhibitor cycloheximide (CHX) or 5 fig/ml of the RNA polymerase (Pol) II inhibitor actinomycin D (ActD), both compatible with pronuclear formation, inhibited pronuclear lamin A/C assembly (Figs. 4IB and 41C). This result indicates that assembly of these somatic lamins results from transcription of the somatic lamin A (LMNA) gene. Lamin B assembly was not perturbed by CHX or ActD, indicating that it was re-targeted from a disassembled somatic pool and/or from a maternal pool of B-type lamins (Figs. 41B and 41C). Essentially no NuMA was detected after CHX exposure; however, 40% of NuMA immunoreactivity in NT pronuclei was detected after ActD treatment (Figs. 41B and 41C). This result suggests that NuMA assembles as a result of translation from maternal mRNA and of de novo transcription. As lamin A/C and NuMA are abnormally transcribed in NT pronuclei, these proteins can be used as markers to determine the ability of nuclear transfer methods to reprogram the donor genetic material.<br>
As discussed in Example 8, the intranuclear anchoring properties of AKAP95, a structural multivalent protein of the nuclear matrix-chromatin interface (Collas et al., J. Cell Biol. 147:1167-1179, 1999) and enriched in hypoacetylated chromatin, can also be used as a marker for reprogramming of donor genetic material. AKAP95 was the only marker investigated that was<br>
detected in somatic donor nuclei on PCC chromosomes and in NT pronuclei with a labeling intensity similar to that of parthenotes (Fig. 41 A) and PN embryos (Fig. 40B). AKAP95 association with NT pronuclei was maintained by inhibition of protein or RNA synthesis. Thus, a major fraction of PN AKAP95 in NT embryos is of somatic origin. AKAP95 anchoring was examined by extraction of NT embryos, parthenotes, and donor flbroblasts with 0.1% Triton X-100,1 mg/ml DNAse I and 300 mM NaCl. In parthenotes, -90% of AKAP95 and DNA were extracted; however, 35% of AKAP95 in NT pronuclei resisted extraction (Fig. 4ID). Sensitivity of AKAP95 and DNA to DNAse I and Nad resembled that of fibroblast nuclei (Fig. 4ID). Lamin B was not extracted under these conditions, indicating that differences in AKAP95 and DNA extractability did not result from gross alterations in nuclear architecture. These results imply that NT pronuclei are characterized by tight anchoring of AKAP95 and restricted DNA accessibility to DNAse I. Thus, pronuclei produced by somatic NT appear to display structural abnormalities as a result of incomplete morphological remodeling of donor nuclei and/or transcriptional misregulation of somatic genes.<br>
EXAMPLE 10: Exemplary nuclear reprogramming deficiencies in traditional bovine nuclear transplant embryos<br>
As described in Example 9, expression patterns in Nuclear Transplant (NT) embryos were compared to those in in v/fro-produced (IVP) preimplantation embryos. For this comparison, in vitro fertilization was performed, and embryos were cultured as previously described (Collas et al., Mol. Reprod. Dev. 34:224-231, 1993 and Kasinathan et al., Biol. Reprod. 64:1487-1493, 2001). NT was carried out by fusing donor bovine fetal flbroblasts to enucleated oocytes (Kasinathan et ai, Nat. Biotechnol. 19:1176-1178, 2001 and Kasinathan et al., Biol. Reprod. 64:1487-1493, 2001). Recipient oocytes were activated at 28-30 hours post-maturation (hpm) with 5 uM calcium ionophore for 4 minutes followed by 10 ng/ml CHX and 2.5 p.g/ml<br>
eytochalasin D for 5 hours and washed. Embryos were co-cultured with mouse fetal fibroblasts (Kasinathan et al., Biol. Reprod. 64:1487-1493, 2001). For CHX treatment, oocytes were activated as above and embryos were cultured with 2.5 (_ig/ml CHX for another nine hours before culture. For ActD treatment, oocytes were activated as above except that 5 ng/ml ActD was added to the five-hour CHX incubation step and embryos were maintained in 5 ug/ml ActD for another nine hours prior to culture. NT embryos were cultured to the blastocyst stage in vitro, and two embryos were transferred per recipient female. Pregnancies were monitored by ultrasonography, and C-sections were performed. Calves were scored by veterinarians within 24 hours of birth. For analysis of protein levels in NT and IVP embyros, anti-lamin B polyclonal antibodies, anti-lamin A/C monoclonal antibodies, and anti-TBP polyclonal or monoclonal antibodies from Santa-Cruz Biotechnology were used. Anti-AKAP95 antibodies were from Upstate Biotechnologies. Immunofluorescence analysis wasxperformed as described (Bomar et al., J. Cell Sci. 115:2931-2940, 2002). Briefly, cells and embryos were settled onto poly-L-lysine-coated coverslips, fixed with 3% paraformaldehyde for 15 minutes, permeabilized with 0.1% Triton X-100 for 15 minutes, and proteins were blocked in PBS/2% BSA/0.01% Tween 20. Samples were incubated with primary and secondary antibodies (1:100 dilutions) each for 30 minutes. DNA was counterstained with 0.1 |ig/ml Hoechst 33342. Photographs were taken with a JVC CCD camera, and quantification of immunofluorescence intensity performed using the AnalySIS software. When indicated, samples were extracted on coverslips with a cocktail of 1% Triton X-100, 1 mg/ml DNAse I, and 300 mM NaCl in Tris-HCl (pH 7.2) for 15 minutes prior to immunofluorescence analysis. For immunoblotting, protein samples (30 |ig) were resolved by 10% SDS-PAGE, blotted onto nitrocellulose, and probed with indicated antibodies.<br>
Dynamics of the donor nucleus in nuclear transplant embryos<br>
To investigate the dynamics of somatic nuclei during bovine nuclear transplantation (NT), the distribution of two structural components of the nuclear envelope, the ubiquitously expressed B-type lamins (referred to as lamin B) and the differentiated cell-specific A-type lamins (lamin A/C) was examined (Gruenbaum et at., J. Struct. Biol. 129:313-323, 2000). Nuclear lamins anchor nuclear membranes to chromatin and have been suggested to promote nuclear expansion after (pro)nuclear reconstitution in vitro. Lamins have also been shown to be essential for cell survival, as failure to assemble B-type lamins leads to cell death. As a marker of the transcription machinery, the dynamics of the TATA-binding protein, TBP, a transcription factor for virtually all genes, was analyzed. Perinuclear distribution of lamin B and colocalization of TBP with DNA in bovine fetal fibroblasts and in in vitro-produced (IVP) preimplantation embryos were consistent with observations in other species (Holy et al., Dev. Biol. 168:464-478, 1995; Houliston et al, Development 102:271-278, 1988; and Worrad et al, Development 120:2347-2357, 1994). (Fig. 46A). Lamin A/C was not detected during preimplantation development (Fig. 46A) as expected from a marker of differentiated cells. Specificity of immunofluorescence labeling was verified on immunoblots (Fig. 46B).<br>
Following transplantation of fibroblast nuclei into enucleated oocytes, both lamins A/C and B were disassembled from the prematurely condensed chromosomes (PCC; three hours post-fusion), while TBP remained associated with the chromosomes (Fig. 46C, PCC). Fourteen hours after initiation of activation of the recipient oocytes, all NT embryos contained fully developed pronuclei with perinuclear lamin B labeling and TBP co-localized with DNA (Fig. 1C, NT-PN). However, in contrast to IVP embryos, 95-99% of NT embryos displayed lamin A/C expression as early as the pronuclear stage (Fig. 46C), and expression persisted during early development (see below). Relative amounts of immunolabeled lamin B, lamin A/C, and TBP in pronuclei of NT<br>
and IVP embryos were quantified by measuring the ratio of secondary antibody fluorescence intensity to that of DNA (Hoechst 33342) to account for DNA content (haploid vs diploid) in the nuclei examined. Whereas relative amounts of lamin B were similar in pronuclei of NT and IVP embryos, relative mounts of lamin A/C and TBP were higher in NT pronuclei than in male (MPN) or female (FPN) pronuclei in IVP embryos (Fig. 46D).<br>
TBP, DNA, and A-type lamins in pronuclei of NT embryos<br>
Higher amounts of TBP in NT pronuclei was associated with a greater resistance to in situ extraction with a combination of detergent (1% Triton X-100), nuclease (1 mg/ml DNAse I), and salt (0.3 M NaCl). Quantification of immunofluorescence labeling intensity in extracted embryos relative to that of non-extracted controls shows that ~35% of TBP remained unextracted in male (MPN) or female (FPN) pronuclei of IVP embryos (Fig. 47 A). However, TBP of NT pronuclei displayed strong resistance to extraction as in fibroblast nuclei (Fig. 47 A). Similarly, DNA of NT pronuclei displayed a 4.5-fold increase in resistance to extraction under these conditions compared to pronuclei of IVP embryos (Figs. 47 A, DNA), suggestive of a more compact chromatin organization.<br>
To determine the origin of lamin B, lamin A/C, and TBP in pronuclei of NT embryos, recipient oocytes were activated in the presence of the RNA polymerase (Pol) II inhibitor actinomycin D (ActD; 5 jig/ml) or with the protein synthesis inhibitor cycloheximide (CHX; 10 ug/ml) (Knott et al., Biol. Reprod. 66:1095-1103, 2002) as described herein. Assembly of lamin A/C, lamin B, and TBP was examined by densitometric analysis of immunofluorescently labeled pronuclear embryos. Both inhibitors prevented pronuclear lamin A/C assembly (Figs. 47B and 48C), suggesting that assembly of these somatic lamins in NT embryos resulted from transcription of the lamin A gene at the pronuclear stage. Lamin B assembly was not perturbed by CHX or ActD treatment (Figs. 47B and 47C), suggesting that lamin B was assembled<br>
from somatic lamins solubilized in the oocyte cytoplasm after NT and/or from a maternal pool of lamins. Similar amounts of TBP were detected in untreated embryos or after inhibition of RNA or protein synthesis (Figs. 47B and 47C). As TBP associates with condensed chromosomes during PCC, and since the metaphase II oocyte cytoplasm is devoid of detectable TBP, TBP of somatic origin probably remains associated with the donor genome during NT. Thus, in addition to expressing A-type lamins, pronuclei of NT embryos display higher amounts and enhanced intranuclear anchoring of TBP.<br>
EXAMPLE 11: Use of reprogrammed donor chromatin masses to clone mammals<br>
To overcome the problem of incomplete reprogramming in traditional nuclear transfer embryos that was demonstrated in Examples 8-10, new methods were developed to more efficiently reprogram donor chromatin prior to nuclear transfer (PCT/USO1/50406, filed December 21, 2001). These methods involve incubating a nucleus (e.g., a nucleus that encodes a xenogenous antibody) from a donor cell in a reprogramming media (e.g., a cell extract) that results in nuclear envelope dissolution and possibly chromatin condensation. This nuclear envelope breakdown and chromatin condensation allows the release of transcription regulatory proteins that were attached to the chromosomes and that would otherwise promote the transcription of genes undesirable for oocyte, embryo, or fetus development. Additionally, regulatory proteins from the reprogramming media may bind the chromatin mass and promote the transcription of genes desirable for development.<br>
To generate an ungulate with a mutation in a prion gene and optionally with a xenogenous Ig gene, the donor nucleus or chromatin mass can be modified before, during, or after reprogramming by mutation of one or more alleles of the prion gene and optionally by insertion of one or more nucleic acids encoding a xenogenous antibody. If desired, a cell from the cloned fetus or the cloned offspring can be used in a second round of nulear transfer to<br>
generate additional cloned offspring. Cells from the initial cloned fetus or cloned offspring may also be frozen to form a cell line to be used as a source of donor cells for the generation of additional cloned ungulates.<br>
Bulk preparation of donor nuclei for use in cloning<br>
As many as several million nuclei may be isolated from synchronized or unsynchronized cell populations in culture. The cell populations may be synchronized naturally or chemically. Preferably, at least 40, 60, 80, 90, or 100% of the cells in a population are arrested in G0 or G\ phase. To accomplish this, cells may be incubated, for example, in low serum, such as 5%, 2%, or 0% serum, for 1, 2, 3, or more days to increase the percentage of cells in G0 phase. To synchronize cells in G], the cells may be grown to confluence as attached cells and then incubated in 0.5-1 ng/ml nocodazole (Sigma Chemicals, St. Louis, MO) for 17-20 hours, as described previously (see, for example, Collasefa/., J. Cell Biol. 147:1167-1180, 1999 and references therein). The flasks containing the attached cells are shaken vigorously by repeatedly tapping the flasks with one hand, resulting in the detachment of mitotic cells and G] phase doublets. The GI phase doublets are pairs of elongated cells at the end of the division process that are still connected by a thin bridge. Detached GI phase doublets may be isolated from the media based on this characteristic doublet structure. The GI phase doublets may remain attached or may divide into two separate cells after isolation.<br>
The synchronized or unsynchronized cells are harvested in phosphate buffered saline (PBS) using standard procedures, and several washing steps are performed to transfer the cells from their original media into a hypotonic buffer (10 mM HEPES, pH 7.5, 2 mM MgCl2, 25 mM KC1, 1 mM DTT, 10 |iM aprotinin, 10 fiM leupeptin, 10 uM pepstatin A, 10 (iM soybean trypsin inhibitor, and 100 uM PMSF). For example, the cells may be washed with 50 ml of PBS and pelleted by centrifugation at 500 x g for 10 minutes at 4C. The PBS supernatant is decanted, and the pelleted cells are resuspended in 50 ml of<br>
PBS and centrifuged, as described above. After this centrifugation, the pelleted cells are resuspended in 20-50 volumes of ice-cold hypotonic buffer and centrifuged at 500 x g for 10 min at 4C. The supernatant is again discarded and approximately 20 volumes of hypotonic buffer are added to the cell pellet. The cells are carefully resuspended in this buffer and incubated on ice for at least one hour, resulting in the gradual swelling of the cells.<br>
To allow isolation of the nuclei from the cells, the cells are lysed using standard procedures. For example, 2-5 ml of the cell suspension may be transferred to a glass homogenizer and Bounce homogenized using an initial 10-20 strokes of a tight-fitting pestle. Alternatively, the cell suspension is homogenized using a motorized mixer (e.g., Ultraturrax). If desired, cell lysis may be monitored using phase contrast microscopy at 40-fold magnification. During this homogenization, the nuclei should remain intact and most or preferably all of the originally attached cytoplasmic components such as vesicles, organelles, and proteins should be released from the nuclei. If necessary, 1-20 fig/ml of the cytoskeletal inhibitors, cytochalasin B or cytochalasin D, may be added to the aforementioned hypotonic buffer to facilitate this process. Homogenization is continued as long as necessary to lyse the cells and release cytoplasmic components from the nuclei. For some cell types, as many as 100, 150, or more strokes may be required. The lysate is then transferred into a 15 ml conical tube on ice, and the cell lysis procedure is repeated with the remainder of the suspension of swollen cells. Sucrose from a 2 M stock solution made in hypotonic buffer is added to the cell lysate (e.g., 1/8 volume of 2 M stock solution is added to the lysate), resulting in a final concentration of 250 mM sucrose. This solution is mixed by inversion, and the nuclei are pelleted by centrifugation at 400 x g in a swing out rotor for 10 to 40 minutes at 4C. The supernatant is then discarded, and the pelleted nuclei are resuspended in 10-20 volumes of nuclear buffer (10 mM HEPES, pH 7.5, 2 mM MgCl2, 250 mM sucrose, 25 mM KC1, 1 mM DTT, 10 ^M aprotinin, 10 leupeptin, 10 jiM pepstatin A, 10 ^M soybean trypsin inhibitor, and 100<br>
uM PMSF). The nuclei are sedimented and resuspended in 1-2 volumes of nuclear buffer, as described above. The freshly isolated nuclei may either be used immediately for in vitro reprogramming and nuclear transfer as described below or stored for later use. For storage, the nuclei are diluted in nuclear buffer to a concentration of approximately 106/ml. Glycerol (2.4 volumes of 100% glycerol) is added and mixed well by gentle pipetting. The suspension is aliquoted into 100-500 ul volumes in 1.5-ml tubes on ice, immediately frozen in a methanol-dry ice bath, and stored at -80C. Prior to use, aliquots of the nuclei are thawed on ice or at room temperature. One volume of ice cold nuclear buffer is added, and the solution is centrifuged at 1,000 x g for 15 minutes in a swing out rotor. The pelleted nuclei are resuspended in 100-500 (il nuclear buffer and centrifuged as described above. The pelleted nuclei are then resuspended in a minimal volume of nuclear buffer and stored on ice until use.<br>
Preparation of mitotic extract or media for use in reprogramming donor genetic material<br>
For the preparation of a mitotic extract, a somatic cell line (e.g., fibroblasts) is synchronized in mitosis by incubation in 0.5-1 fig/ml nocodazole for 17-20 hours (see, e.g., Collas et al., J. Cell Biol. 147:1167-1180, 1999 and references therein) and the mitotic cells are detached by vigorous shaking, as described above. The detached G] phase doublets may be discarded, or they may be allowed to remain with the mitotic cells that constitute the majority off the detached cells (typically at least 80%). The harvested detached cells are centrifuged at 500 x g for 10 minutes in a 10 ml conical tube at 4C. Several cell pellets are pooled, resuspended in a total volume of 50 ml of cold PBS, and centrifuged at 500 x g for 10 minutes at 4C. This PBS washing step is repeated. The cell pellet is resuspended in approximately 20 volumes of ice-cold cell lysis buffer (20 mM HEPES, pH 8.2, 5 mM MgCl2, 10 mM EDTA, 1 mM DTT, 10 uM aprotinin, 10 ^M leupeptin, 10 uM pepstatin A, 10 uM<br>
soybean trypsin inhibitor, 100 |iM PMSF, and optionally 20 ng/ml cytochalasin B), and the cells are sedimented by centrifugation at 800 x g for 10 minutes at 4C. The supernatant is discarded, and the cell pellet is carefully resuspended in no more than one volume of cell lysis buffer. The cells are incubated on ice for one hour to allow swelling of the cells. The cells are lysed by either sonication using a tip sonicator or Bounce homogenization using a glass mortar and pestle. Cell lysis is performed until at least 90% of the cells and nuclei are lysed, which may be assessed using phase contrast microscopy. The sonication time required to lyse at least 90% of the cells and nuclei may vary depending on the type of cell used to prepare the extract.<br>
The cell lysate is placed in a 1.5-ml centrifuge tube and centrifuged at 10,000 to 15,000 x g for 15 minutes at 4C using a table top centrifuge. The tubes are removed from the centrifuge and immediately placed on ice. The supernatant is carefully collected using a 200 ul pipette tip, and the supernatant from several tubes is pooled and placed on ice. This supernatant is the "mitotic cytoplasmic" or "MS 15" extract. This cell extract may be aliquoted into 50 ^,1 or 10 ul volumes of extract per tube on ice, depending on whether the regular or micromethod for generation of chromatin masses will be used. The extracts are immediately flash-frozen on liquid nitrogen and stored at -80C until use. Alternatively, the cell extract is placed in an ultracentrifuge tube on ice (e.g., fitted for an SW55 Ti rotor; Beckman). If necessary, the tube is overlayed with mineral oil to the top. The extract is centrifuged at 200,000 x g for three hours at 4C to sediment membrane vesicles contained in the MS 15 extract. At the end of centrifugation, the oil is discarded. The supernatant is carefully collected, pooled if necessary, and placed in a cold 1.5 ml tube on ice. This supernatant is referred to as "MS200" or "mitotic cytosolic" extract. The extract is aliquoted and frozen as described for the MS 15 extract.<br>
If desired, the extract can be enriched with additional nuclear factors. For example, nuclei can be purified from cells of the cell type from which the reprogramming extract is derived or from cells of any other cell type and lysed<br>
by sonication as described above. The nuclear factors are extracted by a 10-60 minute incubation in nuclear buffer containing NaCl or KC1 at a concentration of 0.15-800 mM under agitation. The lysate is centrifuged to sediment unextractable components. The supernatant containing the extracted factors of interest is dialyzed to eliminate the NaCl or KC1. The dialyzed nuclear extract is aliquoted and stored frozen. This nuclear extract is added at various concentrations to the whole cell extract described above prior to adding the nuclei for reprogramming.<br>
Mitotic extracts can also be prepared from germ cells, such as oocytes or male germ cells. For example, metaphase II oocytes that are naturally arrested at this stage can be harvested, washed, and lysed as described above for the generation of an oocyte extract. To prepare a male germ cell extract, germ cells are isolated from testes obtained from the abattoir by mincing the organ and by differential centrifugation of the harvested cells on a sucrose or percoll gradient. Germ cells are separated from somatic (Leydig and Sertoli) cells, washed by suspension, and sedimentation in PBS. The cells are then washed once in ice-sold cell lysis buffer as described above and lysed by sonication. The lysate is cleared by centrifugation at 15,000 x g for 15 minutes at 4C, and the supernatant (i.e., the germ cell extract) is aliquoted and snap-frozen in liquid nitrogen.<br>
As an alternative to a cell extract, a reprogramming media can also be formed by adding one or more naturally-occurring or recombinant factors (e.g., nucleic acids or proteins such as DNA methyltransferases, histone deacetylases, histones, protamines, nuclear lamins, transcription factors, activators, repressers, chromatin remodeling proteins, growth factors, interleukins, cytokines, or other hormones) to a solution, such as a buffer. Preferably, one or more of the factors are specific for oocytes or stem cells.<br>
Formation of condensed chromatin masses<br>
An aliquot of MS 15 or MS200 extract or the mitotic media is thawed on ice. An ATP-generating system (0.6 fil) is added to 20 u,l of extract or media and mixed by vortexing. For the preparation of the ATP-generating system, equal proportions of 100 mM ATP stock, 1 M creatine phosphate, and 2.5 mg/ml creatine kinase stock solutions (lOOx) made in F^O are mixed and stored on ice until use. After addition of the ATP generating system to the extract, the final concentrations are 1 mM ATP, 10 mM creatine phosphate, and 25 jig/ml creatine kinase.<br>
The nuclei suspension is added to the extract or media at a concentration of 1 u.1 nuclei per 10 uJ of extract or media, mixed well by pipetting, and incubated in a 30, 33, 35, 37, or 39C water bath. The tube containing the mixture is tapped gently at regular intervals to prevent chromosomes from clumping at the bottom of the tube. Nuclear envelope breakdown and chromosome condensation is monitored at regular intervals, such as every 15 minutes, under a microscope. When the nuclear envelope has broken down and chromosomes have started to condense, the procedure for recovery of chromatin masses from the extract or media is started.<br>
Alternatively, chromatin masses that are not condensed or only partially condensed may be formed by performing the above procedure after pre-loading the isolated nuclei with an antibody to the nuclear matrix protein NuMA (Steen el al, J. Cell Biol. 149, 531-536, 2000). This procedure allows the removal of nuclear components from chromatin by the dissolution of the nuclear membrane surrounding the donor nuclei; however, the condensation step is inhibited by addition of the anti-NuMA antibody. Preventing chromosome condensation may reduce the risk of chromosome breakage or loss while the chromosomes are incubated in the mitotic extract.<br>
For this procedure, purified cell nuclei (2,000 nuclei/fil) are permeabilized in 500 \\\ nuclear buffer containing 0.75 p.g/ml lysolecithin for 15 minutes at room temperature. Excess lysolecithin is quenched by adding 1<br>
ml of 3% BSA made in nuclear buffer and incubating for 5 minutes on ice. The nuclei are then sedimented and washed once in nuclear buffer. The nuclei are resuspended at 2,000 nuclei/fil in 100 u.1 nuclear buffer containing an anti-NuMA antibody (1:40 dilution; Transduction Laboratories). After a one hour incubation on ice with gentle agitation, the nuclei are sedimented at 500 x g through 1 M sucrose for 20 minutes. The nuclei are then resuspended in nuclear buffer and added to a mitotic extract or media containing an ATP regenerating system, as described in the previous section. Optionally, the anti-NuMA antibody may be added to the extract or media to further prevent chromosome condensation.<br>
Chromatin masses that are not condensed or only partially condensed may also be formed by exposure to a detergent or protein kinase. Detergent may be used to solubilize nuclear components that are either unbound or loosely bound to the chromosomes in the nucleus, resulting in the removal of the nuclear envelope. For this procedure, purified cell nuclei (2,000-10,000 nuclei/fil) are incubated in nuclear buffer supplemented with a detergent, such as 0.1% to 0.5% Triton X-100 or NP-40. To facilitate removal of the nuclear envelope, additional salt, such as NaCl, may be added to the buffer at a concentration of approximately 0.1, 0.15, 0.25, 0.5, 0.75, or 1 M. After a 30-60 minute incubation on ice with gentle shaking, the nuclei are sedimented by centrifugation at 1,000 x g in a swing-out rotor for 10-30 minutes, depending on the total volume. The pelleted nuclei are resuspended in 0.5 to 1 ml nuclear buffer and sedimented as described above. This washing procedure is repeated twice to ensure complete removal of the detergent and extra salt.<br>
Alternatively, the nuclear envelope may be removed using recombinant or naturally-occurring protein kinases, alone or in combination. Preferably, the protein kinases are purified using standard procedures or obtained in purified form from commercial sources. These kinases may phosphorylate components of the nuclear membrane, nuclear matrix, or chromatin, resulting in removal of the nuclear envelope (see, for example, Collas and Courvalin, Trends Cell Biol.<br>
10: 5-8, 2000). Preferred kinases include cyclin-dependent kinase 1 (CDK1), protein kinase C (PKC), protein kinase A (PKA), MAP kinase, calcium/calmodulin-dependent kinase (CamKII), and CK1 casein kinase, or CK2 casein kinase. For this method, approximately 20,000 purified nuclei are incubated in 20 fil of phosphorylation buffer at room temperature in a 1.5 ml centrifuge tube. A preferred phosphorylation buffer for CDK1 (Upstate Biotechnology) contains 200 mM NaCl, 50 mM Tris-HCl (pH 7.2-7.6), 10 mM MgSO4, 80 mM p-glycerophosphate, 5 mM EGTA, 100 uM ATP, and 1 mM DTT. For PKC, a preferred buffer contains 200 mM NaCl, 50 mM Tris-HCl (pH 7.2-7.6), 10 mM MgS04, 100 uM CaCl2, 40 ug/ml phosphatidylserine, 20 uM diacylglycerol, 100 uM ATP, and 1 mM DTT. If both PKC and CDK1 are used simultaneously, the CDK1 phosphorylation buffer supplemented with 40 jig/ml phosphatidylserine and 20 uM diacylglycerol is used. A preferred phosphorylation buffer for PKA includes 200 mM NaCl, 10 mM MgSO4, 10 mM Tris, pH 7.0, ImM EDTA, and 100 uM ATP. For MAP kinase, the PKA phosphorylation buffer supplemented with 10 mM CaCl2, and 1 mM DTT may be used. For CamKII, either PKA buffer supplemented with 1 mM DTT or a Cam Kinase assay kit from Upstate Biotechnology (Venema et al.L Biol. Chem 272:28187-90, 1997) is used.<br>
The phosphorylation reaction is initiated by adding a protein kinase to a final amount of 25-100 ng. The reaction is incubated at room temperature for up to one hour. Nuclear envelope breakdown may be monitored by microscopy during this incubation, such as at 15 minute intervals. After nuclear envelope breakdown, nuclei are washed three times, as described above for the removal of the detergent solution.<br>
Recovery of chromatin masses<br>
The extract or solution containing the condensed, partially condensed, or not condensed chromatin masses is placed under an equal volume of 1 M sucrose solution made in nuclear buffer. The chromatin masses are sedimented<br>
by centrifugation at 1,000 x g for 10-30 minutes depending on the sample volume in a swing out rotor at 4C. The supernatant is discarded, and the pelleted chromatin masses are carefully resuspended by pipetting in 0.1-1.0 ml nuclear buffer or lipofusion buffer (150 mM Nad, 10 uM aprotinin, 10 u.M leupeptin, 10 uM pepstatin A, 10 jiM soybean trypsin inhibitor, and 100 uM PMSF in either 20 mM HEPES around pH 7.0 or pH 7.5 or 20 mM MES around pH 6.2) and centrifuged at 1,000 x g for 10-30 minutes. The supernatant is discarded, and the pelleted chromatin masses are resuspended in nuclear buffer or lipofusion buffer and stored on ice until use. Each chromatin mass is transferred to a 20 |il drop of HEPES-buffered medium under oil in a micromanipulation dish. One chromatin mass is inserted into each enucleated oocyte, as described below.<br>
Micromethod for preparation of chromatin masses<br>
A 10-20 j-il drop of MS 15 or MS200 extract or mitotic media containing an ATP generating system, a detergent and/or salt solution, or a protein kinase solution as described above is placed in a petri dish. A 50-|il drop of isolated GI phase cell doublets or G0 phase cells in culture medium, a separate 50 ul "lysis" drop of HEPES- or bicarbonate-buffered medium containing 0.1% Triton X-100 or NP-40 for use in facilitating cell lysis, and a 50-ul drop of oocyte injection medium is then added. Each of these drops is covered with CC&gt;2 equilibrated mineral oil. A 50 jil "wash drop" of culture medium is also added to the petri dish for use in washing the lysed cells or nuclei.<br>
Cells are transferred to the lysis drop using a micropipette. The cell membranes are lysed in the pipette by gentle repeated aspirations. When the cell is lysed, the lysate is gently expelled into the wash drop, and the nucleus is immediately reaspirated to remove detergent. Optionally, the nuclei may be permeabilized and incubated with anti-NuMA antibodies prior to being added to the mitotic extract or media. The nucleus is then expelled into the drop of MS 15, MS200, or media, detergent and/or salt solution, or protein kinase<br>
solution. Nuclear breakdown and chromosome condensation is monitored as described above. Once the nuclear envelope has broken down and, if a mitotic extract without anti-NuMA antibodies was used, the chromosomes have started to condense, a single intact chromatin mass is isolated with a micropipette and transferred to an enucleated recipient oocyte, as described below.<br>
Enucleation of oocytes<br>
Preferably, the recipient oocyte is a metaphase II stage oocyte. At this stage, the oocyte may be activated or is already sufficiently activated to treat the introduced chromatin mass as it does a fertilizing sperm. For enucleatation of the oocyte, part or preferably all of the DNA in the oocyte is removed or inactivated. This destruction or removal of the DNA in the recipient oocyte prevents the genetic material of the oocyte from contributing to the growth and development of the cloned mammal. One method for destroying the pronucleus of the oocyte is exposure to ultraviolet light (Gurdon, in Methods in Cell Biology, Xenopus Laevis: Practical Uses in cell and Molecular Biology, Kay and Peng, eds., Academic Press, California, volume 36:pages 299-309, 1991). Alternatively, the oocyte pronucleus may be surgically removed by any standard technique (see, for example, McGrath and Solter, Science 220:1300-1319, 1983). In one possible method, a needle is placed into the oocyte, and the nucleus is aspirated into the inner space of the needle. The needle may then be removed from the oocyte without rupturing the plasma membrane (U.S. Patent Numbers 4,994,384 and 5,057,420).<br>
Lipofusion for insertion of chromatin masses into oocytes<br>
Chromatin may be introduced into recipient oocytes by lipofusion as described below or by standard microinjection or electrofusion techniques (see, for example, U.S. Patent Nos. 4,994,384 and 5,945,577). The following lipofusion method may also be used in other applications to insert chromosomes into other recipient cells.<br>
Chromatin masses are isolated from the mitotic extract, detergent and/or salt solution, or protein kinase solution by centrifugation, and then washed with lipofusion buffer, as described above. The chromatin masses may be in stored in ice-cold lipofusion buffer until use. Alternatively, the chromatin masses are aliquoted, frozen in liquid nitrogen or in a methanol-dry ice bath, and stored frozen at -80C. The lipofusion solution is prepared by mixing one or more fusigenic reagents with the lipofusion buffer in respective proportions ranging from 5:1 to 1:10 approximately. The fusigenic reagents consist of, but are not limited to, polyethylene glycol (PEG) and lipophilic compounds such as Lipofectin, Lipofectamin, DOTAP {N-[l-(2,3-Dioleoyloxy)propyl]-N,N,N-trimethylamonium methylsulfate; C43H83NO8S}, DOSPA {2,3-dioleyloxy-N-[2(sperminecarboxamido)ethyl]-N,N-dimethyl-l-propanaminium trifuoroacetate}, and DOPE (dioleoyl phosphatidylethanolamine).<br>
Other preferred lipids include neutral and monovalent or multivalent cationic lipids, such as those containing quaternary ammonium groups. Additional preferred lipids have a cholesterol moiety such as that formed from the reaction of the hydroxyl group in cholesterol with a group in the lipid. Still other preferred lipids have a saturated or unsaturated fatty acid that preferably contains between 5 and 10, 10 and 15,15 and 20, or 20 and 30 carbon atoms, inclusive. These lipids may be synthesized using standard chemical synthesis techniques, obtained from naturally-occurring sources, or purchased from commercially available source (Summers et al, Biophys J. 71: 3199-3206, 1996; Nabekura et al, Pharm Res. 13: 1069-1072, 1996; Walter et al, Biophys J. 66: 366-376, 1994; Yang et al, Biosci Rep.13: 143-157, 1993; Walter and Siegel, Biochemistry. 6:32: 3271-3281, 1993). Other preferred fusigenic compounds are phospholipids such as membrane vesicle fractions from sea urchin eggs or any other source (Collas and Poccia, J. Cell Science 109: 1275-1283, 1996). Preferably, contacting chromosomes with the membrane vesicle fraction does not result in the chromosomes being encapsulated by an intact membrane.<br>
For example, a cationic lipid, such as DOTAP, may be used at a concentration of approximately 0.1 to 30 |ig/ml in lipofusion buffer. Alternatively, a liposome formulation consisting of a mixture of a cationic lipid and a neutral lipid, such as DOPE, may be used.<br>
The chromatin masses, either freshly prepared or frozen and thawed, are mixed with the lipofusion solution to allow coating of the chromatin masses with the compound. Incubation takes place at a temperature of 20-30C for a period of approximately 10-30 minutes. Microdrops containing the chromatin masses in the lipofusion solution are placed under CO2 equilibrated mineral oil. A drop containing the enucleated recipient oocytes is also prepared. The chromatin masses coated with the lipofusion reagent are picked up in a micropipette and inserted in the perivitellin space, between the oocyte cytoplasm and the zona pellucida. The chromatin mass is placed next to the oocyte membrane to ensure contact with the oocyte. The chromatin mass-oocyte complexes are maintained at a temperature of 20-30C, and fusion is monitored under the microscope. Once fusion has occurred, reconstituted oocytes are activated as described below.<br>
Activation, culturing, and transplantation of reconstituted oocytes<br>
To prevent polar body extrusion and chromosome loss, the oocyte may be activated in the presence of cytochalasin B, or cytochalasin B may be added immediately after activation (Wakayama et al, PNAS 96: 14984-14989, 1999; Wakayama et al, Nature Genetics 24: 108-109, 2000). Either electrical or nonelectrical means may be used for activating reconstituted oocytes. Electrical techniques for activating cells are well known in the art (see, for example, U.S. Patent Nos. 4,994,384 and 5,057,420). Non-electrical means for activating cells may include any method known in the art that increases the probability of cell division. Examples of non-electrical means for activating an oocyte include incubating the oocyte in the presence of ethanol; inositol trisphosphate; Ca++ ionophore and a protein kinase inhibitors; a protein synthesis inhibitor;<br><br>
phorbol esters; thapsigargin, or any component of sperm. Other non-electrical methods for activation include subjecting the oocyte to cold shock or mechanical stress. Alternatively, one to three hours after nuclear transfer, oocytes may be incubated for approximately six hours in medium containing Sr2+ to activate them and cytochalasin B to prevent cytokinesis and polar body extrusion (Wakayama et al., PNAS 96: 14984-14989, 1999; Wakayama et al, Nature Genetics 24: 108-109, 2000). Depending on the type of mammal cloned, the preferred length of activation may vary. For example, in domestic animals such as cattle, the oocyte activation period generally ranges from about 16-52 hours or preferably about 28-42 hours.<br>
After activation, the oocyte is placed in culture medium for an appropriate amount of time to allow development of the resulting embryo. At the two-cell stage or a later stage, the embryo is transferred into a foster recipient female for development to term. For bovine species, the embryos are typically cultured to the blastocyst stage (e.g., for approximately 6-8 days) before being transferred to maternal hosts. For other cloned animals, an appropriate length for in vitro culturing is known by one skilled in the art or may be determined by routine experimentation.<br>
Methods for implanting embryos into the uterus of a mammal are also well known in the art. Preferably, the developmental stage of the embryo is correlated with the estrus cycle of the host mammal. Once the embryo is placed in the uterus of the mammal, the embryo may develop to term. Alternatively, the embryo is allowed to develop in the uterus until a chosen time, and then the embryo (or fetus) is removed using standard surgical methods to determine its health and viability. Embryos from one species may be placed into the uterine environment of an animal from another species. For example, bovine embryos can develop in the oviducts of sheep (Stice and Keefer, Biology of Reproduction 48: 715-719, 1993). Any cross-species relationship between embryo and uterus may be used in the methods of the invention.<br>
Lipofusion of nuclei with oocytes or other recipient cells<br>
The lipofusion solution is prepared by mixing one or more fusigenic reagents with lipofusion buffer in respective proportions ranging from approximately 5:1 to 1:10, as described above. Nuclei, either freshly prepared or frozen and thawed as described above, are mixed with the lipofusion solution to allow coating of the nuclei with the compound. Incubation takes place at a temperature of 20-30C for a period of approximately 10-30 minutes. Microdrops containing nuclei in the lipofusion solution are placed under CC&gt;2 equilibrated mineral oil. A drop containing the recipient cell, preferably an enucleated cell, is also prepared. Enucleated recipient cells are prepared by physically removing the chromosomes or the nucleus by micromanipulation or by damaging the genetic material by exposure to UV light, as described above. For insertion into oocytes, the nuclei coated with the lipofusion reagent are picked up in a micropipette and inserted in the perivitellin space, between the oocyte cytoplasm and the zona pellucida. For insertion into other recipient cells, the coated nuclei are preferably placed next to the cell membrane to ensure contact with the cell. The nucleus-cell complexes are maintained at a temperature of 20-30C, and fusion is monitored using a microscope. Once fusion has occurred, reconstituted oocytes are activated as described above.<br>
EXAMPLE 12: Use of reprogrammed permeabilized cells to clone mammals<br>
Cells may also be reprogrammed without requiring the isolation of nuclei or chromatin masses from the cells. In this method, cells are permeabilized and then incubated in an interphase or mitotic reprogramming media under conditions that allow the exchange of factors between the media (e.g., a cell extract) and the cells. If an interphase media is used, the nuclei in the cells remain membrane-bounded; if a mitotic media is used, nuclear envelope breakdown and chromatin condensation may occur. After the nuclei<br>
are reprogrammed by incubation in this media, the plasma membrane is preferably resealed, forming an intact reprogrammed cell that contains desired factors from the media. If desired, the media can be enriched with additional nuclear factors as described in Example 11. The reprogrammed cells are then fused with recipient oocytes, and embryos formed from the reconstituted oocytes are inserted into maternal recipient mammals for the generation of cloned mammals.<br>
For the production of an ungulate with a mutation in a prion gene and optionally with a xenogenous Ig gene, the donor cells are modified before, during, or after reprogramming by mutation of one or more alleles of the prion gene and optionally by insertion of one or more nucleic acids encoding a xenogenous antibody. If desired, a cell from the cloned fetus or the cloned offspring can be used in a second round of nulear transfer to generate additional cloned offspring. Cells from the initial cloned fetus or cloned offspring may also be frozen to form a cell line to be used as a source of donor cells for the generation of additional cloned ungulates.<br>
Permeabilization of cells<br>
Cells that may be reprogrammed using this procedure include unsynchronized cells and cells synchronized in G0, G^ S, 62, or M phase or a combination of these phases. The cells are permeabilized using any standard procedure, such as permeabilization with digitonin or Streptolysin O. Briefly, cells are harvested using standard procedures and washed with PBS. For digitonin permeabilization, cells are resuspended in culture medium containing digitonin at a concentration of approximately 0.001 - 0.1% and incubated on ice for 10 minutes. For permeabilization with Streptolysin 0, cells are incubated in Streptolysin O solution (see, for example, Maghazachi et al., FASEB J. 11: 765-774, 1997, and references therein;) for -15, 30, or 60 minutes at room temperature. After either incubation, the cells are washed by centrifugation at 400 x g for 10 minutes. This washing step is repeated twice<br>
by resuspension and sedimentation in PBS. Cells are kept in PBS at room temperature until use. Preferably, the permeabilized cells are immediately added to the interphase or mitotic media for reprogramming, as described below.<br>
Preparation of the reprogramming media<br>
To prepare an interphase reprogramming extract, interphase cultured cells are harvested using standard methods and washed by centrifugation at 500 x g for 10 minutes in a 10 ml conical tube at 4C. The supernatant is discarded, and the cell pellet is resuspended in a total volume of 50 ml of cold PBS. The cells are centrifuged at 500 x g for 10 minutes at 4C. This washing step is repeated, and the cell pellet is resuspended in approximately 20 volumes of ice-cold interphase cell lysis buffer (20 mM HEPES, pH 8.2, 5 mM MgCl2, 1 mM DTT, 10 fiM aprotinin, 10 ^M leupeptin, 10 uM pepstatin A, 10 jiM soybean trypsin inhibitor, 100 [iM PMSF, and optionally 20 p.g/ml cytochalasin B). The cells are sedimented by centrifugation at 800 x g for 10 minutes at 4C. The supernatant is discarded, and the cell pellet is carefully resuspended in no more than one volume of interphase cell lysis buffer. The cells are incubated on ice for one hour to allow swelling of the cells. The cells are lysed by either sonication using a tip sonicator or Bounce homogenization using a glass mortar and pestle. Cell lysis is performed until at least 90% of the cells and nuclei are lysed, which may be assessed using phase contrast microscopy. The sonication time required to lyse at least 90% of the cells and nuclei may vary depending on the type of cell used to prepare the extract.<br>
The cell lysate is placed in a 1.5-ml centrifuge tube and centrifuged at 10,000 to 15,000 x g for 15 minutes at 4C using a table top centrifuge. The tubes are removed from the centrifuge and immediately placed on ice. The supernatant is carefully collected using a 200 fil pipette tip, and the supernatant from several tubes is pooled and placed on ice. This supernatant is the "interphase cytoplasmic" or "IS 15" extract. This cell extract may be aliquoted<br>
into 20 |il volumes of extract per tube on ice and immediately flash-frozen on liquid nitrogen and stored at -80C until use. Alternatively, the cell extract is placed in an ultracentrifuge tube on ice (e.g., fitted for an SW55 Ti rotor; Beckman). If necessary, the tube is overlayed with mineral oil to the top. The extract is centrifuged at 200,000 x g for three hours at 4C to sediment membrane vesicles contained in the IS 15 extract. At the end of centrifugation, the oil is discarded. The supernatant is carefully collected, pooled if necessary, and placed in a cold 1.5 ml tube on ice. This supernatant is referred to as "IS200" or "interphase cytosolic" extract. The extract is aliquoted and frozen as described for the IS 15 extract.<br>
If desired, the extract can be enriched with additional nuclear factors. For example, nuclei can be purified from cells of the cell type from which the reprogramming extract is derived or from cells of any other cell type and lysed by sonication as described above. The nuclear factors are extracted by a 10-60 minute incubation in nuclear buffer containing NaCl or KC1 at a concentration of 0.15-800 mM under agitation. The lysate is centrifuged to sediment unextractable components. The supernatant containing the extracted factors of interest is dialyzed to eliminate the NaCl or KC1. The dialyzed nuclear extract is aliquoted and stored frozen. This nuclear extract is added at various concentrations to the whole cell extract described above prior to adding the cells for reprogramming.<br>
Interphase extracts can also be prepared from germ cells, such as oocytes or male germ cells. For example, oocytes are activated as described above and cultured for five hours to allow entry into interphase. Oocytes are then treated as described in Example 11 for metaphase II oocyte extracts except that EDTA is omitted from the lysis buffer. Male germ cell extracts can be prepared as described in Example 11.<br>
As an alternative to a cell extract, a reprogramming media can also be formed by adding one or more naturally-occurring or recombinant factors (e.g., nucleic acids or proteins such as DNA methyltransferases, histone<br>
deaeetylases, histones, protamines, nuclear lamins, transcription factors, activators, repressers, chromatin remodeling proteins, growth factors, interleukins, cytokines, or other hormones) to a solution, such as a buffer. Preferably, one or more of the factors are specific for oocytes or stem cells.<br>
Reprogramming of cells in a media<br>
The permeabilized cells are suspended in an interphase reprogramming media described above or one of the mitotic reprogramming medias described in Example 11 at a concentration of approximately 100-1,000 cells/ul. The ATP generating system and GTP are added to the extract as described above, and the reaction is incubated at 30-37C for up to two hours to promote translocation of factors from the extract into the cell and active nuclear uptake or chromosome-binding of factors. The reprogrammed cells are centrifuged at 800 x g, washed by resuspension, and centrifuged at 400 x g in PBS. If desired, the cells are resuspended in culture medium containing 20-30% fetal calf serum (PCS), RPMI1640 containing 2 mM CaCl2 (added from a 1 M stock in H2O), or in a-MEM medium containing 2 mM CaC^and incubated for 1-3 hours at 37C in a regular cell culture incubator to allow resealing of the cell membrane. The cells are then washed in regular warm culture medium (10% PCS) and cultured further using standard culturing conditions. As described in Example 17, the reprogrammed permeablized cells may also be used for genetic transfer to oocytes without resealing the cell membrane.<br>
Alternative method of reprogramming permeabilized cells<br>
Alternatively, the cells can be permeabilized while placed on coverslips to minimize the handling of the cells and to eliminate the centrifugation of the cells, thereby maximizing the viability of the cells. Cells (e.g., fibroblasts) are grown on 16-mm poly-L-lysine-coated coverslips in RPMI1640 to 50,000-100,000 cells/coverslip in 12-well plates. Cells are permeabilized in 200 ng/ml Streptolysin O in Ca2+-free Hanks Balanced Salt Solution (Gibco-BRL) for 50<br>
minutes at 37C in regular atmosphere. If desired, the percent of cells that are permeabilized under these conditions can be measured based on propidium iodide uptake. Streptolysin O is aspirated; coverslips are overlaid with 80-100 ul of reprogramming media; and the cells are incubated for thirty minutes to one hour at 37 C in C02 atmosphere. The reprogramming media preferably contains the ATP generating system and 1 mM each of ATP, CTP, GTP and UTP. To optionally reseal plasma membranes, a-MEM medium containing 2 mM CaCl2, medium containing 20-30% fetal calf serum, or RPMI1640 containing 2 mM CaCl2 is added to the wells, and the cells are incubated for two hours at 37 C. Alternatively, the plasma membrane is not resealed.<br>
Effect of Streptolysin O treatment on the percentage of permeabilized and resealed cells<br>
To assess the percent of permeabilized and resealed cells, dose and time titrations of Streptolysin 0 incubation were performed (Table 7). Permeabilization of cells was assessed by uptake of 0.1 |ig/ml of the DNA stain propidium iodide at the end of Streptolysin O treatment. Resealing was assessed similarly at the end of the resealing treatment in a separate group of cells.<br><br>
Assessment of viability of bovine fibroblasts permeabilized with Streptolysin O treatment and exposed to mitotic extract<br>
TUNEL analysis was performed to evaluate apoptosis in cells permeabilized with 0 or 500 ng/ml Streptolysin O and resealed, or in cells permeabilized with Streptolysin 0, exposed to mitotic extract for 30 or 60 minutes, and resealed. TUNEL-positive cells are cells undergoing apoptosis (i.e., cell death). The data show that Streptolysin O itself does not induce apoptosis (Table 8). Exposure of Streptolysin O-treated cells to the mitotic extract for 60 minutes, but not 30 minutes, induces a 10% increase in apoptotic rate, based on TUNEL analysis (Table 8). Based on these data, a 30-minute incubation of donor cells in the extract is more preferable than a 60-minute incubation. Thirty-minute incubations were shown by immunofluorescence analysis of cells to induce nuclear envelope breakdown in the majority of nuclei examined (-90%, n&gt;100).<br>
Additionally, purified nuclei incubated in extract and washed in either buffer N or TL-HEPES and sucrose as described in Example 11 for the<br><br>
chromatin transfer method do not undergo apoptosis (2/34 and 3/47 TUNEL positive, respectively).<br>
Table 8. TUNEL analysis of Streptolysin O and Streptolysin O plus extract-treated bovine fibroblasts<br><br>
The permeabilization method chosen for these cloning methods was 500 ng/ml SLO for 30 minutes at 38C. The resealing method chosen for forming an intact membrane surrounding the reprogrammed cells was a two hour incubation in a-MEM medium containing 2 mM CaCl2.<br>
Alternatively cell permeabilized method using a protease<br>
In an exemplary cell permeabilization method using a protease, cells (e.g., fibroblasts) are grown to confluency in a 35 mm plate overnight. The fibroblasts are removed from the plate using the normal trypsin-EDTA (0.5%-5.3 mM) procedure for five minutes. The fibroblasts are washed in PBS and then resuspended in 1 ml TL Hepes. One ml of 3 mg/ml protease in TL Hepes is added to 1 ml of cell suspension (final protease concentration is 1.5 mg/ml) and incubated at room temperature for one minute. The cells are washed in 10 ml HBSS and resuspended in 1 ml HBSS and counted. Approximately 105 cells in minimal volume are used for mitotic extract incubation. Forty ul of MS-15 mitotic extract is placed on the cells for 30 minutes at 37C. Cells are<br>
then washed in TL Hepes and used in one of the nuclear transfer procedures described herein.<br>
These cells show signs of permeabilization because after incubation in the mitotic extract, 70-80% of the cells have undergone nuclear envelope breakdown and premature chromosome condensation, indicating that MPF (maturation promoting factor) from the extract or some component of MPF has traversed the membrane. This cell permeabilization method can be used with a variety of proteases such as trypsin and with a variety of germ, somatic, embryonic, fetal, adult, differentiated, and undifferentiated cells.<br>
Formation, activation, culturing, and transplantation of reconstituted oocytes<br>
The reprogrammed cells are inserted into, or fused with, recipient oocytes using standard microinjection or electrofusion techniques (see, for example, U.S. Patent Nos. 4,994,384 and 5,945,577). For example, the cells can be placed next to the oocytes in standard cell medium in the presence or absence of sucrose (e.g., 2.5 % sucrose), and the cells can be drawn into an injection pippette. The pipette is then aspirated a few times to lyse the cells and remove cytoplasmic components from the nucleus which is then injected into the oocyte. The reconstituted oocytes are then activated, cultured, and transplanted into maternal recipient mammals using standard methods such as those described in Example 11 to produce cloned mammals. EXAMPLE 13: Evidence for more complete nuclear reprogramming using chromatin transfer and Streptolysin O-transfer<br>
As illustrated in Examples 8-10, incomplete nuclear remodeling and reprogramming occurs in traditional nuclear transplant pronuclear stage embryos. This finding was demonstrated by the assembly of lamins A/C in the nuclear envelope of pronuclear nuclear transplant embryos and excess NuMA immunofluorescence labeling. More complete nuclear reprogramming was achieved using the chromatin mass transfer method described in Example 11<br>
and the cell permeabilization and reprogramming method (also referred to as SLOT) described in Example 12.<br>
In particular, the cloning methods of the present invention produced embryos with protein expression patterns that more closely resembled in vitro fertilized embryos than cloned embryos produced using traditional cloning methods. As illustrated in Examples 8-10, chromatin transfer embryos expressed much less lamin A/C protein than traditional nuclear transfer embryos. Lamins A/C are somatic-specific components of the nuclear lamina that are naturally expressed in differentiated cells, but not expressed in embryos. Because of the reported interaction of lamins with transcription factors, chromatin proteins, and DNA, it is likely that the expression of lamins A/C in traditional nuclear transfer embryos promotes the expression of proteins specific for somatic cells that are undesirable for embryo development. Thus, the chromatin transfer embryos of the present invention may express fewer undesirable somatic-specific proteins than traditional nuclear transfer embryos. Additionally, the chromatin transfer embryos had expression patterns for NuMA, a main component of the nuclear matrix that is implicated in transcriptional regulation, that more closely resembled in vitro fertilized embryos than traditional nuclear transplant embryos. This result also indicates that chromatin transfer embryos are more efficiently reprogrammed than traditional nuclear transplant embryos.<br>
Assessment of in vitro nuclear breakdown of bovine fibroblast nuclei<br>
Extracts prepared from mitotic bovine fibroblasts consistently supported breakdown of-80% of input purified fibroblast nuclei (Fig. 33). An extract from metaphase II oocytes (i.e., an extract from oocytes naturally arrested in metaphase II prior to fertilization) also successfully supported nuclear breakdown (75% of nuclei within 30 minutes).<br>
Input interphase nuclei (Fig. 34A), chromatin masses obtained from nuclei incubated in a MS 15 mitotic extract (Fig. 34B), and chromatin masses<br>
obtained from nuclei incubated in an oocyte extract (Fig. 34C) were examined for the expression of the following markers: lamin B receptor (LBR), an integral protein of the inner nuclear membrane (membrane marker); lamin B, a ubiquitous component of the nuclear lamina; lamins A/C, a somatic-specific component of the nuclear lamina present only in differentiated cells and absent in embryos; NuMA, a main component of the nuclear matrix; AKAP95, a PKA-anchoring protein of the nucleus; and DNA. Both somatic cytosolic MS 15 and oocyte MS 15 extracts induced solubilization of lamin B, lamins A/C, LBR, and NuMA in ~100% of chromatin units examined (Figs. 34B and 34C). As expected, AKAP95 remained associated with chromosomes, as observed previously in mitotic human cells (Collasef al., J. Cell Biol. 147:1167-1180, 1999). This result was also described in Example 8 for bovine nuclear transplant embryos at the premature chromatin condensation stage. Both the mitotic extract and the oocyte extract appeared to be as efficient as intact oocytes in promoting nuclear envelope solubilization, regardless of the method used, i.e., traditional nuclear transplant, nuclear injection (NI), or chromatin transfer (Fig. 35).<br>
Comparison of pronuclear embryos produced by chromatin transfer and pronuclei from nuclear transplant and nuclear injection embryos<br>
To generate chromatin transfer embryos, in v^ro-matured oocytes were enucleated about 18-20 hours post maturation. Nuclei from interphase bovine fetal fibroblasts were incubated in a MS 15 mitotic extract that was prepared from bovine fetal cells as described herein. Chromatin masses were isolated from the extract when after nuclear envelope breakdown had occurred and before chromatin condensation was completed. In particular, the chromatin masses were isolated when the chromatin was approximately 50-60% condensed, compared to the level of condensation of chromosomes in interphase (designated 0% condensed) and the maximum level of condensation<br>
of chromosomes in mitotsis (designated 100% condensed.) At this stage, individual chromosomes in the chromatin mass could not be distinguished and the edges of the chromatin mass had an irregular shape. Chromatin masses that had been isolated from the mitotic extract were placed in a microdrop of TL HEPES with 2.5% sucrose along with enucleated oocytes. The sucrose was added to the buffer to minimize damage to the ooctyes from the subsequent injection procedure. Chromatin masses were injected into the oocytes using a beveled microinjection pipette using a Burleigh Piezo Drill (Fishers, NY) (frequency 2 Hz for 75 microseconds at an amplitude of 70 V). Typically multiple pulses, such as 2, 3, 4, or 5 pulses, were performed so that the needle sufficiently penetrated the oocyte for injection. After injection, oocytes were washed in serial dilutions of TL HEPES in sucrose to minimize osmotic shock. At 28-30 hours post maturation (i.e., 28-30 hours after oocytes were placed in maturation medium after collection from ovaries, which is also at least two hours after injection of chromatin masses), reconstructed oocytes and controls for parthenogenetic development were activated with calcium ionophore (5 |iM) for four minutes (Cal Biochem, San Diego, CA) and 10 |_ig/ml cycloheximide and 2.5 ug/ml cytochalasin D (Sigma) in ACM culture medium (100 mM NaCl, 3 mM KC1, 0.27 mM CaCl2, 25 mM NaHCO3, 1 mM sodium lactate, 0.4 mM pyruvate, 1 mM L-glutamine, 3 mg/ml BSA (fatty acid free), 1% BME amino acids, and 1% MEM nonessential amino acids (Sigma)), for five hours as described earlier (Liu et al., Mol. Reprod. Dev. 49: 298-307, 1998). After activation, eggs were washed five times and placed in culture in four-well tissue culture plates containing mouse fetal fibroblasts and 0.5 ml of embryo culture medium covered with 0.3 ml of embryo tested mineral oil (Sigma). Between 25 and 50 embryos were placed in each well and incubated at 38.5C in a 5% C02 air atmosphere. If desired, calcium (e.g., -.0.5, 1.0, 1.5 , 2.0, 2.5, 3, 3.5, 5 mM, or more CaC^) can be added to the culture medium for ~0.5, 1.0, 1.5, 2.0, 2.5, 3.0, or more hours to promote resealing of the oocyte after injection. The resealed oocytes are likely to have increased survival rates<br>
due to the intact layer surrounding the oocytes when they are implanted into the recipient mammal using the standard methods described herein.<br>
Nuclear injection embryos were formed as described above for chromatin transfer embryos, except that interphase bovine fetal fibroblasts nuclei that had not been incubated in an extract were injected into the ooctyes instead of chromatin masses. Nuclear transplant embryos were generated using the conventional methods described in Example 8.<br>
Nuclear transplant, nuclear injection, and chromatin transfer pronuclei reassemble lamin B (Fig. 36A, red label) and AKAP95 (Fig. 36B, red label) as anticipated. Nuclear transplant and nuclear injection pronuclei also reassemble lamins A/C, a somatic-specific component (Fig. 36A, green label), consistent with the results reported above for nuclear transplant embryos. However, chromatin transfer pronuclei and control parthenote pronuclei do not reassemble lamins A/C (Fig. 36A). Nuclear transplant pronuclei also contain NuMA (green label), unlike most chromatin transfer or parthenote pronuclei (Fig. 36B, green label). A proportion of parthenote nuclei and chromatin transfer nuclei assemble a low level of NuMA, as reported above.<br>
In vitro disassembly of nuclei followed by chromatin transfer results in pronuclei that are morphologically similar to control parthenote pronuclei. In contrast, nuclear transplant and nuclear injection pronuclei harbor somatic-specific components (lamins A/C and extensive NuMA labeling). This result is indicative of incomplete nuclear remodeling after traditional nuclear transplant or nuclear injection procedures. As described above, lamins A/C detected in nuclear transplant and nuclear injection pronuclei originate from lamins transcribed de novo at the pronuclear stage. Because nuclear lamins and possibly NuMA are implicated in transcription regulation and disease in humans, persistence of lamins A/C in conventional nuclear transplant pronuclei might be indicative of improper functional reprogramming. We conclude that in vitro nuclear disassembly and chromatin transfer produces more normal pronuclei than traditional nuclear transplant or nuclear injection.<br>
Cloning Efficiency using Reprogrammed Chromatin Masses or Permeabilized Cells as Donor Source<br>
As described in Example 12, a cloning procedure denoted "SLOT" was developed that involves Streptolysin 0 (SLO)-induced permeabilization of primary fetal bovine fibroblasts, exposure of permeabilized cells to a reprogramming media (e.g., a mitotic extract) for 30 minutes, optional resealing of the fibroblasts with 2 mM calcium in culture, and transfer of the chromatin into oocytes using standard cell fusion methods.<br>
For this cloning method, a vial of Streptolysin 0 (Sigma S-5265; 25,000 units stored in store powder form at 4C) was dissolved in 400 (al H2O and mixed well. All contents were transferred to a 15-ml conical tube, and then 3.6 ml H20 was added and mixed by vortexing. Aliquots of 10 ul were frozen at -20C at a stock concentration of 0.062 U/ul. Cells (-100,000) were suspended in 100 ul HBSS (Gibco BRL, cat. No. 14170-120) at room temperature. These cells were confluent, and thus ~80-85% of the cells were in Gl phase, and the majority of the other cells were in S phase. Streptolysin O stock solution (5 ul) (i.e., 500 ng/ml or 0.3 U/ul final concentration) was added, and the mixture was incubated at 38C for 25 minutes in a water bath. The tube was gently tapped 2-3 times during incubation to ensure that the cells remained in suspension. Room temperature PBS (200 |al) was added and mixed well by gentle pipetting. The cells were centrifuged cells at 5,000 rpm for five minutes at room temperature in a tabletop centrifuge. All the supernatant was discarded. At this stage, the pellet is small and may not be clearly visible. Mitotic extract containing the ATP-generating system (40 p.1, "MS 15") was added and mixed well. The extract was prepared during the centrifugation of the cells by thawing one vial of 40 ul extract and adding 1.2 ul of ATP-generating system, mixing well, and incubating at room temperature. This mitotic extract was the same extract used for the generation of chromatin masses in the section above. The mixture was incubated at 38C in water bath for 30 minutes, and the tube<br>
was occasionally gently tapped. Room temperature resealing medium (RM, 500 uL) (complete a-MEM [Bio-Whittaker] medium supplemented with CaCl2 to 2 mM from a 1 M stock) was added. The tube was left open and incubated in a CC&gt;2 incubator for two hours with occasional tapping of the tube to ensure that the cells remained in suspension. The cells were centrifuged at 5,000 rpm for five minutes at room temperature in a tablettop centrifuge. The cell pellet was resuspended in 100 (il of room temperature TL HEPES (Bio-Whittaker, cat. No. 04-616F), and another 900 ul TL HEPES was added. The nuclear transfer was performed using standard procedures. Oocytes were activated and transferred to recipient mammals as described in the previous section for chromatin transfer.<br>
The development of embryos formed using this SLOT method and the chromatin transfer method of the present invention is summarized in Table 7. Development to the blastocyst stage was slightly lower for SLOT embryos compared to conventional nuclear transfer embryos. The differences between SLOT and nuclear transfer development at the blastocyst stage could be due to the effect of using a greater precentage of cells in the Gl phase of the cell cycle for nuclear transfer than for SLOT. The survival rate was lower for chromatin transfer embryos, which is expected for an invasive procedure.<br>
Pregnancy rates were comparable for nuclear transfer and SLOT embryos at 40 days of gestation (Table 9). Survival from 40 days of pregnancy to 60 days tended to be higher for SLOT embryos than for nuclear transfer embryos produced using conventional methods.<br><br>
As noted above, the survival rate for chromatin transfer embryos may be increased by incubating the reconstituted oocytes in calcium for a few hours to allow the oocytes to reseal prior to be inserted into recipient mammals. Survival rates for SLOT embryos may also be increased by reducing the amount of time between when the cells are taken out of culture and when they are fused with oocytes. For example, the length of time for the incubation in Streptolysin O, the incubation in the reprogramming medium, and/or the incubation in the resealing medium may be decreased. In particular, the incubation in the resealing medium may be decreased to approximately one hour or less. This shortened resealing treatment may be performed in the presence of 2 mM calcium as described above or in the presence of a higher concentration of calcium (e.g., -2.5, 3.0, 3.5, 4.0, 4.5, 5.0, or 6.0 mM calcium) to increase the rate of resealing. By reducing the amount of time the cells are treated prior to being fused with oocytes, the cells are less likely to enter S phase and begin DNA replication which reduces the survival rate of the reconstituted oocyte.<br>
EXAMPLE 14: Evidence for more complete nuclear reprogramming using chromatin transfer<br>
As discussed in Example 11, a strategy was developed to enhance remodeling of donor nuclei, promote repression of somatic genes, and produce embryos with a pronuclear architecture similar to that of fertilized zygotes. In a particular example of this general chromatin transfer method, isolated intact bovine fibroblast nuclei were incubated in a cytoplasmic extract of mitotic bovine fibroblasts in the presence of an ATP-generating system, which was required to drive nuclear disassembly.<br>
To generate a mitotic reprogramming extract for conversion of donor nuclei into chromatin masses, fibroblasts were synchronized in mitosis with 0.5 (ig/ml nocodazole for ~18 hours, harvested by mitotic shake-off, and washed twice in ice-cold PBS and once in ice-cold cell lysis buffer (20 mM Hepes, pH 8.2, 5 mM MgCl2, 10 mM EDTA, 1 mM DTT, and protease inhibitors) (Collas et al., J. Cell Biol. 147:1167-1179, 1999). Packed cells were resuspended one volume of cell lysis buffer, allowed to swell on ice for one hour and Dounce-homogenized on ice using a tight fitting pestle until all cells were lysed. The lysate was centrifuged at 15,000 x g for 15 minutes at 4C, and the supernatant (mitotic extract) was collected, aliquoted, and snap-frozen in liquid nitrogen and stored at -80C.<br>
To generate donor chromatin, unsynchronized confluent fibroblasts were harvested, washed, and resuspended in -20 volumes of ice-cold hypotonic nuclear isolation buffer (10 mM Hepes, pH 7.5, 2 mm MgCl2, 25 mM KC1, 1 mm DTT, and a cocktail of protease inhibitors) (Collas et al, J. Cell Biol. 147:1167-1179, 1999). After one hour on ice, cells were Dounce-homogenized with a tight-fitting pestle. Sucrose was added from a 2 M stock to a concentration of 250 mM and nuclei sedimented at 400 x g for 10 minutes at 4C. Nuclei were washed in nuclear isolation buffer (same buffer as above but with 250 mM sucrose) and were either used fresh or frozen in nuclear isolation buffer/70% glycerol (Collas *t al. J. Cell Biol. 147:1167-1179. 1999X<br>
For reprogramming, isolated fibroblast nuclei were incubated in 40 (0.1 of mitotic extract containing an ATP-generating system (1 mM ATP, 10 mM creatine phosphate, and 25 |ig/ml creatine kinase) at 4,000 nuclei/ul for 30 minutes in a 38C H20 bath. Nuclear envelope breakdown and chromatin condensation were monitored by phase contrast microscopy. At the end of incubation, the reaction mix was diluted with 500 ul TL Hepes containing 2.5% sucrose, and chromatin was recovered by sedimentation at 2,000 x g for five minutes. Chromatin masses were resuspended in TL Hepes/sucrose and transferred to TL Hepes/sucrose under mineral oil together with enucleated oocytes. Individual chromatin masses were injected into in vitro-matured oocytes enucleated at 20 hpm with a beveled microinjection pipette using a Burleigh Piezo Drill (Fishers, NY) (2 Hz, 2 u,s, 70 V). After injection, oocytes were washed in serial dilutions of sucrose in TL Hepes to minimize osmotic shock and cultured. At 28 hpm, reconstructed oocytes and parthenogenetic controls were activated as described for NT and cultured ((Kasinathan et a/., Nat. Biotechnol. 19:1176-1178, 2001 and Kasinathan et al, Biol. Reprod. 64:1487-1493, 2001). For nuclear injections (NI), purified fibroblast nuclei were exposed to cell lysis buffer instead of mitotic extract and injected into enucleated oocytes as for CT.<br>
As a result of reprogramming, lamin A/C, lamin B, and NuMA were readily disassembled, while AKAP95 remained associated with condensed chromosomes (Fig. 42A). TUNEL analysis showed that no apoptosis occurred in the extract. Similar nuclear breakdown studies with human HeLa nuclei and mitotic extracts indicated that condensed chromatin masses were capable of supporting nuclear reconstitution (Steen et al, J. Cell Biol. 150:1251-1562, 2000) and transcription in interphase cytoplasm. Thus, in vitro nuclear disassembly produces condensed chromatin capable of reforming functional nuclei. Condensed fibroblast chromatin was recovered by sedimentation, and individual chromatin masses were injected into enucleated recipient oocytes. After recovery in culture, oocvtes were activated as described in Example 9 for<br>
NT oocytes. To control for artifacts generated by handling of nuclei, intact nuclei exposed to extract buffer alone were also injected.<br>
CT and control nuclear injection (NI) results are shown in Figs. 42A-42D. NI produced pronuclei which, like NT pronuclei, contained lamins A/C and B, NuMA, and AKAP95 (Figs. 42B and 42C). CT resulted in PN formation in over 80% of embryos that survived injection and activation (~50% of oocytes injected, n&gt;2,000). However, CT pronuclei displayed no detectable lamin A/C and a 3-fold reduction of anti-NuMA immunolabeling compared to NT and NI pronuclei (Figs. 42B and 42C). This pattern was similar to that of parthenogenetic pronuclei. Extractability of AKAP95 and DNA with Triton X-100/DNAse I/NaCl as described above was enhanced 8-fold in CT pronuclei compared to NT pronuclei, reflecting a beneficial effect of CT on pronuclear AKAP95 anchoring and DNAse I accessibility (Fig. 42D). As chromatin-bound AKAP95 co-fractionates with primarily transcriptionally repressed (hypoacetylated) chromatin, this result suggests that CT enhances formation of euchromatin upon PN assembly.<br>
The dynamics of the TATA binding protein, TBP, was examined during fibroblast donor nucleus remodeling by NT and CT procedures carried out in parallel. TBP facilitates assembly of the general transcription machinery for virtually all genes (Sharp et al, Cell 68:819-821, 1992). TBP co-localized with AKAP95 and DNA in donor fibroblast nuclei (Fig. 43A). PCC chromosomes obtained after NT contained 5-fold more TBP than chromosomes condensed in mitotic extract, as shown by TBP/AKAP95 and TBP/DNA fluorescence intensity ratios (Fig. 43B). TPB labeling intensity of chromosomes condensed in vitro resembled that of mitotic fibroblasts. At the PN stage, TBP was barely detectable in CT embryos but displayed strong immunoreactivity in NT and NI embryos (Fig. 43 A). Pronuclear TBP in NT embryos was of somatic origin because inhibition of transcription or translation maintained strong TBP labeling. Pronuclear TBP concentration in the mouse has been shown to increase during progression through interphase (Worrad et al., Development<br>
120:2347-2357, 1994). However, the similarity in kinetics of PN formation from PCC- or in v/fro-condensed chromatin between reconstructed NT and CT embryos indicated that the enhanced TBP concentration in NT pronuclei was not due to more a advanced cell cycle stage. Thus, in vitro disassembly of donor nuclei promotes dissociation of TBP from the chromatin, such that resulting CT pronuclei contain -10-fold less TBP than NT pronuclei at the same stage of reconstitution.<br>
Thus, five structural and functional markers of incomplete reprogramming by NT include pronuclear assembly of lamins A/C, enhanced pronuclear NuMA and TBP concentrations, and increased sensitivity of AKAP95 and DNA to extraction with detergent, DNAse, and salt. In contrast, B-type lamins, essential for proper nuclear reformation and cell survival (Steen et al., J. Cell Biol. 153:621-626, 2001) appear to assemble normally in NT pronuclei, although we were not able to determine whether the pool of assembled B-type lamins was of somatic (and therefore re-targeted) or of maternal origin. The LMNA gene remains active in NT pronuclei, resulting in the assembly of differentiated cell-specific A-type lamins. Through interactions with chromatin and the transcription machinery, nuclear lamins have been suggested to participate in transcription regulation (Cohen et al., Trends Biochem. Sci. 26:41-47, 2001); thus, assembly of the correct set(s) of nuclear lamins is most likely critical for proper pronuclear function in NT embryos.<br>
In the present methods, somatic nuclear components are dispersed in the extract and typically do not come in contact with the oocyte cytoplasm. This prohibits re-targeting of somatic-specific molecules to the developing nucleus, such as B-type lamins whose composition may differ from that of maternal lamins (Cohen et al., Trends Biochem. Sci. 26:41-47, 2001). In vitro chromatin condensation may also promote release of DNA-bound factors such as chromatin remodeling enzymes (Sif et al., Genes Devel. 12:2842-2851, 1998) and transcription factors, thereby "stripping" the donor genome of<br>
potentially inhibitory somatic components. In particular, TBP removal may result in inactivation of somatic-specific genes in reconstituted CT pronuclei and duplicate the low transcriptional activity of the male pronucleus after fertilization (Poccia et al., Trends Biochem. Sci. 17:223-227, 1992).<br>
An implication of removing factors from the donor nucleus is that loading of maternal components onto chromatin and remodeling into a physiological pronucleus may be facilitated. CT increases the sensitivity of AKAP95 and DNA to nucleases and salt. DNAse I-resistant DNA is mostly transcriptionally silent; thus, incomplete remodeling of AKAP95 anchoring by NT may impair expression of developmentally important genes, such as genes involved in placental development, maintenance of late pregnancy and postnatal survival of cloned animals.<br>
The following characteristics of CT oocytes indicate that nuclear transfer of a chromatin mass incubated in a mitotic extract significantly improves the functional characteristics of the resulting reconstituted oocyte. A nuclear matrix protein that is expressed at high levels by somatic donor cells, NuMA, was expressed 3-fold less in CT pronuclei (i.e., pronuclei formed after introduction of a chromatin mass into an oocyte) than in NT pronuclei. This result indicates that the level of NuMA in CT oocytes is more similar to the level of NuMA in naturally-occurring oocytes than in NT oocytes. CT pronuclei also expressed 10-fold less of the general transcription factor TBP than NT pronuclei. This removal of TBP from the donor chromatin mass by incubation in the mitotic extract may result in inactivation of undesired, somatic-specific genes in the resulting CT oocyte. Lamin A/C, a nuclear envelope protein that is specific for differentiated cells and is not detected in in vitro fertilized or parthenogenically activated oocytes, was also not detected in CT pronuclei but was detected in NT pronuclei. Because nuclear lamins may regulate transcription, the lack of detectable lamin A/C in CT pronuclei may result in more appropriate regulation of transcription in CT oocytes than in NT<br>
oocytes. The sensitivity of AKAP95, a structural protein of the nuclear matrix-chromatin interface which is enriched in transcriptionally repressed (hypoacetylated) chromatin, and DNA to extraction with detergent, DNase, and salt was measured in the pronuclei to characterize the anchoring of AKAP95 to DNA and to characterize DNA accessibility in the pronuclei. Extractability was enhanced 8-fold in CT pronuclei compared to NT pronuclei, reflecting a beneficial effect of incubation of a donor chromatin mass in a mitotic extract on AKAP95 anchoring and morphological remodeling of the donor DNA. Because AKAP95 and DNase I-resistant DNA is associated with transcriptionally repressed or silent DNA, the increased sensitivity of AKAP95 and DNA to nucleases, salt, and detergent suggests that CT oocytes may have increased expression of developmentally important genes.<br>
Similar results are expected with donor nuclei from cells with one or more mutations in a prion gene.<br>
EXAMPLE 15: Evidence for more complete nuclear reprogramming using Streptolysin O-transfer<br>
The reprogramming of a chromatin mass during incubation of a permeabilized cell in a mitotic extract using SLOT is also demonstrated in Figs. 45A and 45B. In this study, the expression of a nuclear matrix protein that is expressed at high levels by somatic donor cells, NuMA, and the expression of a nuclear envelope protein that is specific for differentiated cells, lamin A/C, was compared for oocytes produced using SLOT and oocytes produced using traditional nuclear transfer of a cell containing a nucleus that has not been incubated in an extract ("NT"). Fourteen hours after activation, the SLOT pronuclei (i.e., pronuclei formed after introduction of a donor cell containing a chromatin mass into an oocyte) expressed significantly less NuMA and lamin A/C than NT pronuclei (n=15-20 embryos/marker in three replicates). These results demonstrate that reconstituted SLOT oocytes are more efficiently reprogrammed than NT oocytes and more closely resemble<br>
naturally-occurring, fertilized oocytes. Because nuclear lamins may regulate transcription, the significantly lower level of lamin A/C in SLOT pronuclei may result in more appropriate regulation of transcription in SLOT oocytes than in NT oocytes.<br>
In addition to reducing the expression of undesired factors in the resulting oocyte, SLOT increases the viability of the resulting fetuses compared to traditional nuclear transfer methods (Table 10). Thus, the many structural and functional differences of SLOT donor cells substantially improve the ability of the reconstituted oocytes to form non-human embryos and non-human mammals.<br>
Table 10. Development of bovine embryos produced using permeabilized donor cells with a chromatin mass ("SLOT") compared to bovine embryos produced using donor cells with a nucleus ("NT")<br><br>
EXAMPLE 16: Exemplary Evidence for More Complete Nuclear Reprogramming Using Streptolysin O-transfer (SLOT)<br>
As described in Examples 12 and 15, a novel in vitro nuclear remodeling system has been developed. The system involves permeabilization of the donor cell, induction of chromatin condensation in a mitotic cell extract, and washing of the permeabilized cell to remove nuclear factors solubilized during condensation of the chromatin. Pronuclei of bovine chromatin transplant embryos exhibit an expression pattern of several markers that closely resembles that of normal embryos as opposed to nuclear transplant embryos, which resemble somatic cells. Eight healthy calves were produced using this chromatin transfer system. Chromatin transfer shows trends of increased survival to term, lower incidence of large calves, and significantly enhanced survival after birth. The results demonstrate the successful manipulation of a<br>
somatic donor nucleus prior to transplantation. As described further below, the disassembly of a somatic nucleus in a mitotic extract followed by transfer of the condensed chromatin into an oocyte enhances nuclear remodeling and shows evidence of improved development and viability of clones. This procedure can be used to further characterize the mechanism of nuclear reprogramming, if desired.<br>
A chromatin transfer strategy<br>
To alleviate defects identified in pronuclear NT embryos, fibroblast nuclei were manipulated in vitro prior to transfer into recipient oocytes. This SLOT system is outlined in Fig. 48. For generation of a mitotic extract, fibroblasts were synchronized in mitosis with 1 |ag/ml nocodazole for 18 hours, harvested by mitotic shake-off, and washed twice in phosphate buffered saline and once in cell lysis buffer (20 mM Hepes, pH 8.2, 5 mM MgCl2, 10 mM EDTA, 1 mM DTT and protease inhibitors). Sedimented cells were resuspended in one volume of ice-cold cell lysis buffer, allowed to swell on ice for one hour, and Bounce-homogenized using a tight-fitting glass pestle. The lysate was centrifuged at 15,000 x g for 15 minutes at 4C and the supernatant (mitotic extract) was aliquoted, frozen in liquid nitrogen, and stored at -80C. Fresh or frozen extracts were used without noticeable differences on efficiency of nuclear breakdown.<br>
Bovine fetal fibroblasts from confluent cultures were washed in Ca2+/Mg2+-free Hank's Balanced Salt Solution (HBSS) and permeabilized by incubation of 100,000 cells in suspension with 31.2 U Streptolysin O (SLO; Sigma) in 100 (il HBSS for 30 minutes in an approximately 38.5C H20 bath. Permeabilization was assessed by uptake of the membrane impermeant DNA stain, propidium iodide (0.1 ug/ml). Permeabilized fibroblasts were sedimented, washed, and incubated in 40 fil mitotic extract containing an ATP-generating system (1 mM ATP, 10 mM creatine phosphate, and 25 |ag/ml creatine kinase) for 30-45 minutes at approximately 38.5C to promote nuclear<br>
disassembly and removal of nuclear components.   Aliquots were labeled with 0.1 ug/ml Hoechst 33342 to monitor chromatin condensation. After the incubation, fibroblasts were recovered from the extract by sedimentation and washed. The reaction mixture was diluted with 500 ul Alpha MEM/10% fetal bovine serum (Gibco-BRL) containing 2 mM CaCl2 for membrane resealing, and the cells were cultured for two hours at 38.5C (Hakelien et al, Nat. Biotechnol. 20:460-466, 2002). Resealing was monitored by propidium iodide uptake. Resealed cells were fused with in v^ro-matured oocytes that had been enucleated at 20 hpm; oocytes were activated at 28 hpm, and embryos cultured as described for NT in Example 10.<br>
SLOT embryos were cultured to the blastocyst stage in vitro, and two embryos were transferred per recipient female. Pregnancies were monitored by ultrasonography, and C-sections were performed. Calves were scored by veterinarians within 24 hours of birth.<br>
Breakdown of fibroblast nuclei in mitotic extract<br>
The mitotic extract consisted of a 15,000 x g supernatant from a lysate of mitotic bovine fibroblasts and contained an ATP-regenerating system. The extract did not induce apoptosis, as judged by the absence of proteolysis of poly(ADP)ribosyl polymerase (PARP) and DNA fragmentation characteristic of apoptotic fibroblasts (Fig. 49A). Thus, the extract was suitable to promote remodeling of somatic nuclei.<br>
The extract elicited ATP-dependent condensation of chromosomes, disassembly of A/C and B-type lamins from chromatin as judged by cytoplasmic labeling of these lamins, and removal of TBP from chromatin (Fig. 49B). These events were confirmed by immunoblotting analysis of condensed chromatin purified from the fibroblasts after recovery from the mitotic extract (Fig. 49C; compare lanes 1 and 3). In this experiment, the A-kinase anchoring protein AKAP95 was used as a marker of a nuclear component that remains associated with the condensed chromosomes, as normally occurs at mitosis (Collas etal, I Cell Biol. 147:1167-1180, 1999). Histone H4 was used as a<br>
protein loading control in the gel (Fig. 49C). Disassembly of nuclear lamins and TBP from chromatin in mitotic extract was dependent on an ATP-regenerating system (Figs. 49B and 49C, lanes 3 and 4) and was reminiscent of that occurring in mitotic cells (Fig. 49C, lane 2). Furthermore, immunoblotting analysis of whole permeabilized fibroblasts (as apposed to isolated chromatin) after exposure to mitotic extract showed that a fraction of solubilized lamin A/C and all detectable TBP were eliminated from the cells and/or proteolysed (Fig. 49C, lane 6). Lastly, a control extract from interphase fibroblasts (Fig. 49C, lane 5) or cell lysis buffer alone both containing an ATP-regenerating system failed to promote nuclear disassembly, indicating that nuclear breakdown was ATP-dependent and specific for the mitotic extract. Permeabilized fibroblasts exposed to mitotic extract and resealed with CaCl2 could be cultured over several passages, indicating that membrane permeabilization, incubation of the permeabilized cells in the extract, and membrane resealing were viable procedures.<br>
Characterization of nuclei in embryos produced by chromatin transfer<br>
Fusion of resealed fibroblasts to recipient oocytes occurred as efficiently (over 70%) as with non-permeabilized cells. The donor chromatin was in a condensed form at the time of introduction into oocytes (Fig. 50A, SLOT). In contrast, chromatin of fibroblasts used for NT was still decondensed within 30 minutes effusion (Fig. 50A, NT). Thus, resealing of mitotic extract-treated fibroblasts with CaCl2 prior to transfer into oocytes did not promote nuclear 'reformation in the donor cells. This observation was supported by the absence of a nuclear envelope around the condensed chromatin in SLOT embryos immediately after fusion, as judged by immunofluorescence analysis of several lamina and inner nuclear membrane proteins.<br>
Immunolabeling of nuclear lamins and TBP in nuclei of SLOT and NT embryos, and immunolabeling intensity of these markers relative to DNA<br>
fluorescence intensity, are shown in Figs. SOB and 50C. Perinuclear lamin B labeling intensity was similar in SLOT and NT pronuclei. Remarkably however, in contrast to NT pronuclei, lamin A/C was undetected in pronuclei and up to at least the 8-16-cell stage in SLOT embryos (Fig. SOB). SLOT pronuclei also displayed a 4-fold reduction in TBP labeling compared to NT pronuclei (Fig. 50C). Pronuclear TBP concentration in the mouse has been shown to increase during progression through interphase (Worrad et al, Development 120:2347-2357, 1994). However, as kinetics of pronuclear formation from PCC- or in vz/ro-condensed chromatin were similar in NT and SLOT embryos, it is unlikely, albeit not formally excluded, that enhanced TBP concentration in NT pronuclei was due to a more advanced cell,cycle stage. Resistance of TBP to extraction with 1% Triton X-100, 1 mg/ml DNAse I, and 0.3 M NaCl was decreased by over 2-fold (Fig. SOD), indicating a weaker association of TBP with intranuclear ligands. Likewise, resistance of DNA to DNAse I was reduced nearly 4-fold in SLOT pronuclei, suggesting that SLOT favors the establishment of a looser chromatin configuration in pronuclei (Fig. SOD). Thus, disassembly of fibroblast nuclei in mitotic extract followed by transfer of the condensed chromatin into oocytes enhanced morphological remodeling of the donor nuclei and alleviated defects detected in pronuclei of NT embryos.<br>
Chromatin transfer produces healthy clones<br>
SLOT resulted in development to term of cloned embryos (Figs. 51A and SIB). Pregnancy rates following transfer of blastocysts into recipient females were significantly higher at 40 days for SLOT embryos than for NT embryos (P^O.02; Fisher's Test), and the trend was maintained up to development to term (12/59 and 23/211 calves born, respectively; P=0.05) (Fig. 51 A). Additionally, SLOT enhanced survival rate of clones beyond 24 hours post-partum (10/59 and 17/211, respectively; P=0.04) (Figs. 51A and 51B).<br>
Health of NT (Example 10) and SLOT clones born was evaluated by scoring animals and placentas on a scale of 1 (normal) to 5 (grossly abnormal). Placental scores included parameters such as placental edema, cotyledon number, size and morphology, color of amniotic fluids, morphology of uterus, and umbilicus. Animal scores included functional evaluation and general appearance of respiratory, cardiovascular, digestive, urinary, muscular, skeletal and nervous systems. Data for individual calves are shown in Fig. 51C. Box plot analyses show that scores of animals and placentas derived from NT were more dispersed that those produced by SLOT (Fig. 5IE). Mean birth weight of SLOT and NT clones was not significantly different; however, the proportion of SLOT calves over 45 kg at birth was lower (P=0.02; Fisher) than that of NT animals (Figs. 5ID and 5IE). Altogether, these results indicate that chromatin transfer produces live offspring and shows evidence of improved development and viability.<br>
In comparison with NT (Example 10) and despite the limited number of SLOT offspring produced (n=8), SLOT significantly enhances pregnancy rates at 40 days (P=0.02) and survival of calves born beyond 24 hours (P=0.04). A trend towards an improvement in the health of animals produced by SLOT is also reflected in box plot analyses. Furthermore, the lower incidence of large (over 45 kg) calves produced by SLOT has practical and economical implications on animal management.<br>
Several nuclear defects have been identified in NT embryos, including assembly of lamin A/C, enhanced pronuclear TBP content and increased resistance of DNA to DNAse I. These defects may result from incomplete remodeling of the fibroblast nuclei and/or from misregulation of expression of differentiated cell-specific (e.g., lamin A) genes. Remodeling of nuclei in vitro and transplantation of condensed chromatin into oocytes alleviates these defects.<br>
Remodeling nuclei through SLOT increases DNA sensitivity to DNAse I and may promote the formation of transcriptionally active (or potentially<br>
active) chromatin. This effect may in turn facilitate expression of developmentally important genes, such as genes involved in placental development, maintenance of late pregnancy, and post-natal survival. SLOT also induces repression of lamin A gene expression in cloned embryos. In vitro and in vivo manipulations of nuclear lamina composition have shown that failure to assemble a correct set of lamins invariably leads to apoptosis (Steen et al., J. Cell Biol. 153:621-626, 2001). Moreover, as lamins interact with DNA, chromatin and the transcription machinery, proper lamina reconstitution is likely to be essential for normal nuclear function (Gruenbaum et al., J. Struct. Biol. 129:313-323, 2000 and Cohen et al, Trends Biochem. Sci. 26:41-47, 2001) in cloned embryos.<br>
Chromatin condensation at mitosis or in vitro is associated with the release of DNA-bound components such as chromatin remodeling enzymes, transcription factors (e.g., TBP), or other potentially inhibitory somatic components. Removal of TBP from donor somatic chromatin may facilitate repression or down-regulation of somatic-specific genes in SLOT embryos, which may impair development. An implication of removing factors from the donor nucleus is that loading of maternal components onto chromatin and subsequent remodeling into a physiological pronucleus may be facilitated.<br>
In conclusion, it is possible to directly remodel a somatic nucleus in a cell extract and produce live offspring. In vitro manipulation of nuclei for cloning or transdifferentiation purposes (Landsverk et al., EMBO Rep. 3:384-389, 2002 and Hakelien et al., Nat. Biotechnol. 20:460-466, 2002) is a useful tool for optional further investigation of the mechanisms of nuclear reprogramming. Chromatin transfer shows evidence of improved development to term and viability of clones. Additional manipulation of the system might lead to further improvements in the efficiency of mammalian cloning.<br>
EXAMPLE 17: Additional Evidence for More Complete Nuclear Reprogramming Using Streptolysin O-transfer (SLOT)<br>
The SLOT method described in Example 12 produced improved results when the donor permeabilized cells were are not resealed prior to genetic transfer. Additionally, use of donor cells in GI phase instead of confluent cells may result in increased viability of the reconstituted oocyte and resulting embryo. These experiments are described further below.<br>
Preparation of buffers<br>
A 1 molar DTT solution was prepared by placing 1 ml H2O in an eppendorf tube, adding 154 mg refrigerated DTT, and mixing well by vortexing. The solution was aliquoted into 10 u.1 volumes and frozen at -20C. Aliquots can be thawed and re-frozen several times. A 100 ml volume of cell lysis buffer for preparation of cell extracts for nuclear assembly/disassembly assays contained NaCl (50 mM, 1 ml of 5 M stock), MgCl^ (5 mM, 0.5 ml of 1 M stock), Hepes, pH 8.2 (20 mM, 2 ml of 1 M stock, pH 8.2), and H2O (96.5 ml). The buffer was aliquoted and stored at 4C. There was a drop of ~1 pH unit upon lysate preparation. For preparation of mitotic extracts, 10 mM EGTA was added (1 ml of a 1 M stock) and 95.5 ml H2O was added instead of 96.5 ml. Prior to use, the following were added: DTT (1 |il/ml solution, 1 M stock at -20C, 1 mM final concentration), PMSF (10 pi/ml solution 100 mM stock, 1 mM final concentration), CAL mix (10 \i\ CAL cocktail at -20C per ml solution , i.e., a final concentration of 10 fig/ml each of chymostatin, aprotinin, and leupeptin), Pepstatin A (10 (il stock at -20C per ml solution, 10 Hg/ml final concentration), and Cytochalasin D (1 [il/ml from 1 mg/ml stock at -20C, 1 fig/ml final concentration). For the nocodozole 1000X stock solution, 1 mg/ml of nocodozole in DMSO was prepared and stored in 160 |il aliquots.<br>
To prepare Streptolysin O stock (SLO), a vial of SLO (Sigma S-5265; 25,000 units stored as a powder at 4C) was dissolved in 400 u.1 H20 and mixed well. The entire content was transferred to a 1.5-ml conical tube, divided into 10 u.1 aliquots, and frozen at -20C. The stock concentration was "10X." To prepare the protease solution, 3 ml TL Hepes and 9 mg Protease (Sigma P-8811) were added to a 15 ml conical tube and mixed by vortexing. The solution was filtered through a 0.22 um syringe filter directly into TL Hepes. Because the cells doubled the volume, the final concentration was 1.5 mg/ml. For HECM Hepes, NaCl (114 mM, 6.662 g), KC1 (3.2 mM, 0.239 g), CaCl2 2H2O (2.0 mM, 0.294 g), MgCl2 6H2O (0.5 mM, 0.102 g), Pen/Strep (10 ml, Sigma P3539 Pen/Strep, 100 U/ml and 100 ug/ml final concentration), Phenol Red (5 ug/ml, 1 ml), and H2O (in a sufficient amount to increase the total volume to 990 ml) were combined. Then, 100 X A.A. (10 ml), Na lactate (10 mM, 1.44 ml), Na pyruvate (0.1 mM, 0.011 g), NaHCO3 (2 mM, 0.168 g), and HEPES (10 mM, 2.38 g) were added. The final solution had an osmolarity of 260-270 mOsM. Then, 3 g bovine serum albumin (Fraction V) was added, and the pH was adjusted to 7.4. The solution was filtered through a 0.22 uM filter and stored at 4C.<br>
For preparation of the ATP stock solution (Sigma A3377: 100 mM Stock, 100 x), H20 (1 ml) and ATP (0.055 g) were combined, and 10 u.1 aliquots were frozen at -20C. For preparation of creatine phosphate (Sigma P7936: 1 M stock, lOOx), H2O (1 ml) and creatine phosphate (0.255 g) were combined, and 10 ul aliquots were frozen at -20C. To prepare creatine kinase (Sigma C3755: 2.5 mg/ml stock, lOOx), H2O (1 ml) and creatine kinase (0.0025 g) were combined, and 10 u,l aliquots were frozen at -20C. For preparation of the ATP-generating system, equal proportions of 100 mM ATP stock, 1 M creatine phosphate, and 2.5 mg/ml creatine kinase stock solutions (lOOx) were made using H2O, mixed, and stored as a frozen solution. The ATP generating system was kept on ice until use. The ATP generating system (1.2 p.1) was added to the extract (40 u.1), and the solution was mixed by vortexing. The<br>
final concentration of the ATP generating system in the extract was 1 mM ATP, 10 mM creatine phosphate, and 25 u,g/ml creatine kinase.<br>
Preparation of mitotic extract<br>
One vial of 1.5 to 2 million cells was thawed. Cells were split into two T75 flasks and grown for two days or until they were confluent. The cells were passaged in 6-8 T75 flasks and grown to confluency. The cells were trypsinized and counted using a hemocytometer, and 3 million cells were added to each of as many T150 flasks as could be used with the number of cells available. The cell line (e.g., fibroblasts such as primary fibroblasts, epithelial cells, or immortalized and disease-free cells such as MDBK cells) was synchronized at 70-80% confluency in mitosis with 0.5-1 (ig/ml nocodazole for 17-20 hours using standard procedures (e.g., Collas et al,, J. Cell Biol. 147:1167-1180, 1999 and references therein). The synchronized cells were harvested by mitotic shake-off. Each flask containing cells was shaken vigorously by repeatedly tapping it with one hand. The mitotic cells detached and floated in the culture medium. The harvested cells were centrifuged at 500 x g for 10 minutes in a 50 ml conical tube at 4C. The supernatant was discarded, and the cell pellets were resuspended in a total of 50 ml of cold phosphate buffered saline/Ca/Mg Free (PBS). If desired, several cell pellets can be pooled into a single 50 ml tube. The cells were centrifuged at 500 x g for 10 minutes at 4C, and the above washing step was repeated. The volume of the cell pellet was determined, and the cell pellet was resuspended in approximately 20 volumes of ice-cold cell lysis buffer containing protease inhibitors (i.e., DTT and PMSF).<br>
Then, the cells were sedimented at 500 x g for five minutes at 4C. The supernatant was discarded, and the cell pellet volume was determined. The cell pellet was resuspended in no more than one volume of cell lysis buffer containing all of the protease inhibitors. The cells were incubated on ice for one hour to allow swelling of the cells. Using a tip sonicator, the cell<br>
suspension was sonicated until all cells were broken open. Cell lysis was monitored under a phase contrast microscope. Desirably, 90% of the cells were lysed before proceeding to the next step. Sonication can be prolonged as long as necessary; the sonication time varies with the cell type used to prepare the extract. Alternatively, cells are lysed by Bounce homogenization using a glass mortar and pestle (homogenizer), desirably until at least 90% of the cells are lysed. The cell lysate was placed in a 1.5-ml centrifuge tube and centrifuged at ~15,000 x g for 15 minutes at 4C using a table top refrigerated centrifuge. The centrifuge was placed in a cold room or refrigerator and allowed to equilibrate. The tubes were removed from the centrifuge and immediately placed on ice. The supernatant was carefully collected using a 200 ul pipette tip. This supernatant is the mitotic cytoplasmic extract. The extract was placed in another tube on ice. Extracts collected from several tubes were pooled.<br>
Two alternatives exist at this stage. The cell extract was aliquoted into tubes on ice, 41 fil of extract per tube. The extract was immediately snap-frozen in liquid nitrogen and stored in a -80C freezer until use. Such extracts prepared from a 15,000 x g centrifugation are called MS 15 (or mitotic cytoplasmic extract). Alternatively, the MS 15 extract is placed in an ultracentrifuge tube on ice (e.g., fitted for an SW55 Ti rotor; Beckman). The tube is overlayed with mineral oil to the top if necessary to prevent collapsing of the tube upon ultracentrifugation. The extract was centrifuged at 200,000 x g for three hours at 4C to sediment membrane vesicles contained in the MS 15. At the end of centrifugation, the oil was discarded. The supernatant was carefully collected and placed in a cold 1.5-ml tube on ice. Several supernatants were used if necessary. This supernatant is referred to as MS200 (or mitotic cytosolic extract). The MS200 extract was aliquoted and frozen as described for the M515 extract.<br>
Chromatin transfer<br>
On the day before the cloning procedure, one 35 mm Nunc was prepared with 1 million fibroblast cells, or six T75 flasks containing 500,000 cells were prepared for shake-off. On the morning of the cloning procedure, the Alpha MEM with 15% FBS irradiated, complete media in all flasks was changed to remove debris and dead cells. Prior to performing the cell permeabilization and mitotic extract reaction, the lowest concentration of SLO necessary to permeabilize 80-90% of the cells was determined by serial dilution of the SLO stock solution (e.g., dilutions of IX, 0.5X, 0.3X, and 0.1X), incubating for 30 minutes at approximtely 38.5 C in a water bath, and then staining with propidium iodide.<br>
The cells were dissociated from confluent culture or newly transfected cell culture using trypsin or cell dissociation buffer, placed in 15 ml conical tube, and washed once by centrifugation using Hank's Balanced Salt Solution, Ca/Mg free (HBSS). The cell pellet was resuspended in 1 ml HBSS, and the cells were counted to determine the concentration. Approximately 50,000-100,000 cells were suspended in 100 ul HBSS (Gibco BRL, cat. No. 14170-120) at room temperature. The 5 u,l SLO stock solution at the previously determined concentration was added. The mixture was incubated at approximately 38.5 C for 30 minutes in a water bath. The tube was gently tapped 2-3 times during incubation to ensure that the cells remained in suspension. A 200 u,l volume of room temperature PBS (Ca/Mg free) was added and mixed well by gentle pipetting. The cells were centrifuged at 500 x g for five minutes at room temperature in table top centrifuge. All of the supernatant was discarded. The pellet was small and may not be clearly visible.<br>
A 40 u.1 volume of mitotic extract containing the ATP-generating system was added, and the solution was mixed well. The extract was prepared during the 30 minute incubation above. One vial of 40 ul extract was thawed and added to 1.2 ul of ATP-generating system. The solution was mixed well and<br>
kept at room temperature. The mixture was incubated at 38.5 C in a water bath for 30 minutes, and the tube was occasionally gently tapped. A 500 |il volume of room temperature complete media (Alpha MEM + 15% Fetal calf serum) was added and stored in an approximately 38.5C incubator with the lid open until use. The cells were centrifuged at 500 x g for five minutes at room temperature in a table top centrifuge. The cell pellet was resuspended in 1 ml room temperature TL Hepes and transferred to a 15 ml conical tube. A 1 ml volume of 3 mg/ml Protease in TL Hepes (filtered) was added for a final protease concentration on the cells of 1.5 mg/ml, and the mixture was incubated for 30 seconds. TL Hepes was added to fill the 15 ml conical tube, and the tube was capped and centrifuged at 2300 rpm for five minutes. TL Hepes was removed from cells, and 150 ul of TL Hepes was added to cells. The tube was labeled for the appropriate cell line and placed on a warming stage.<br>
During the time that the cells were being prepared, the manipulation station was prepared for cell transfer using an appropriately sized cell transfer pipette. Approximately 50 enucleated oocytes were placed into a 50 ul drop of TL Hepes under heavy mineral oil in a 100 mm dish. The washed cells were placed in a drop with enucleated oocytes. Enough cells were used for genetic transfer to all oocytes. Care was taken to avoid using so many cells that they sterically blocked the manipulation tools. One enucleated oocyte was placed on the holder, and one cell was placed in the perivitellin space under the zona, such that the cell was touching the plasma membrane of the oocyte. To facilitate fusion, the cell was placed adjacent to, or opposite to the polar body. After all enucleated oocytes had cells transferred to them, oocytes were removed from the microdrop and placed in a pre-warmed 35 mm dish of HECM Hepes for fusion.<br>
Results<br>
Comparing the results in Table 12 below for donor cells in which the membrane was not resealed prior to genetic transfer with the results in Table 11 for donor cells in which the membrane was resealed prior to genetic transfer indicates that increased cloning efficiency may be obtained by not resealing the permeablized cell membrane. Additionally, Table 12 demonstrates that the use of donor cells in G] phase instead of confluent cells may result in increased viability of the reconstituted oocyte and resulting embryo. The data in Table 11 was obtained using an extract from bovine primary fibroblasts and donor bovine fetal fibroblasts.   MDBK cells have also been successfully used to generate reprogramming extracts.<br><br>
EXAMPLE 18: Methods for the generation of chimeric mammals<br>
Many spontaneous abortions that occur using traditional methods to clone mammals are thought to result from placental abnormalities rather than from problems with the fetus. Thus, methods have been developed to produce chimeric embryos with placental tissue primarily from one origin (e.g., an in vitro fertilized, naturally-occurring, or parthenogenetically activated embryo) and fetal tissue primarily from another origin (e.g., a nuclear transfer embryo encoding a xenogenous antibody). Chimeric embryos with placental tissue derived primarily from cells from in vitro fertilized, naturally-occurring, or<br>
parthenogenetically activated embryos may better resemble naturally-occurring placental tissue and result in increased production of viable offspring. Preferably, the majority of the cells of the offspring are derived from cells from the nuclear transfer embryo and thus have a genome that is substantially identical to that of the donor cell used to generate the nuclear transfer embryo. In one such method, cells from an in vitro fertilized embryo are injected into the periphery of a compaction embryo encoding a xenogenous antibody (e.g., between the zona pellucida and the embryo itself) that was produced using traditional nuclear transfer methods or any of the other cloning methods described herein. In an alternative method, cells from a precompaction, in vitro fertilized embryo are incubated with cells from a precompaction embryo encoding a xenogenous antibody produced using one of the cloning methods of the present invention (e.g., using a reprogrammed chromatin mass or a permeabilized cell as the donor source) under conditions that allow cells from each embryo to reorganize to produce a single chimeric embryo (Wells and Powell, Cloning 2:9-22, 2000). In both methods, the cells from the in vitro fertilized embryo are preferentially incorporated into the placenta, and the cells from the nuclear transfer method are preferentially incorporated into the fetal tissue. These methods are described further below. These results were generated using nuclear transfer embryos that do not contain a xenogenous antibody gene; however, similar results are expected for nuclear transfer embryos containing a xenogenous antibody gene and/or containing a mutation in a prion gene.<br>
Isolation of Gl fibroblasts<br>
For the isolation of Gl fibroblasts as donor cells to produce nuclear transfer embryos, the previously described "shake-off method was used (Kasinathan et al., Nature biotech. 19:1176-1178, 2001). Briefly, 24 hours prior to isolation, 5.0 x 105 cells were plated onto 100 mm tissue culture plates containing 10 ml of a-MEM plus PCS. The following day, plates were washed<br>
with PBS, and the culture medium was replaced for one to two hours before isolation. The plates were then shaken for 30-60 seconds on a Vortex-Genie 2 (Fisher Scientific, Houston, TX, medium speed). The medium was removed, spun at 500 x g for five minutes, and the pellet was re-suspended in 250 ul of MEM plus PCS. This cell suspension consisted of newly divided cell doublets attached by a cytoplasmic bridge, some single cells, and metaphase or anaphase cells. The cell doublets attached by a cytoplasmic bridge were used as donor cells for nuclear transfer.<br>
Nuclear transplantation, activation, and embryo culture<br>
The nuclear transfer procedure using the isolated Gl fibroblasts was performed essentially as previously described (Cibelli et al., Nature Biotech. 16: 642-646, 1998; Kasinathan et al, Biol. Reprod. 64: 1487-1493, 2000). In vitro matured oocytes were enucleated about 18-20 hours post maturation, and chromosome removal was confirmed by bisBenzimide (Hoechst 33342, Sigma) labeling under UV light. These cytoplast-donor cell couplets were fused using a single electrical pulse of 2.4 kV/cm for 20 mircoseconds (Electrocell manipulator 200, Genetronics, San Diego, CA). At 30 hours past maturation, reconstructed oocytes and controls were activated with calcium ionophore (5 uM) for four minutes (Cal Biochem, San Diego, CA) and 10 ug cycloheximide and 2.5 ug cytochalasin D (Sigma) in ACM culture medium (100 mM NaCl, 3 mM KC1, 0.27 Mm CaCl2, 25 mM NaHCO3, 1 mM sodium lactate, 0.4 mM Pyruvate, 1 mM L-glutamine, 3 mg/ml BSA (fatty acid free), 1% BME amino acids, and 1% MEM nonessential amino acids; all from Sigma) for six hours as described previously (Liu et al, Mol. Reprod. Dev. 49:298-307, 1998; Presicce et al, Mol. Reprod. Dev. 38:380-385, 1994). After activation, eggs were washed in HEPES buffered hamster embryo culture medium (HECM-HEPES, 114 mM NaCl, 3.2 mM KC1, 2 mM CaCl2, 10 mM Sodium Lactate, 0.1 mM sodium pyruvate, 2 mM NaHCO3, 10 mM HEPES, and 1% BME amino acids; Sigma) five times and placed in culture in 4-well tissue culture plates<br>
containing mouse fetal fibroblasts and 0.5 ml of embryo culture medium covered with 0.2 ml of embryo tested mineral oil (Sigma). Twenty five to 50 embryos were placed in each well and incubated at 38.5C in a 5 % C02 in air atmosphere. On day four, 10% PCS was added to the culture medium. On days seven and eight, development to the blastocyst stage was recorded.<br>
Bovine in vitro fertilization<br>
In vitro fertilization was performed as described earlier to produce bovine in vitro fertilized embryos (Collas et al., Mol. Reprod. Dev. 34:224-231, 1993). A 45% and 90% isotonic Percoll gradient was prepared with sperm TL stock (Parrish et al., Theriogenology 24:537-549, 1985). Frozen-thawed bovine sperm from a single bull was layered on top of the gradient and centrifuged for 30 minutes at 700 x g (2000 rpm using a 6.37 inch tip radius). The concentration of sperm in the pellet was determined, and the sperm was diluted in sperm TL (sperm TL stock, 1 mM pyruvate, 6 mg/ml BSA, and 1% PS) such that the final concentration at fertilization was 106 sperm/ml. At 22 hours post maturation, oocytes were wash three times in TL HEPES and placed in 480 [il of fertilization TL (Bavister et al., Biol. Reprod. 28:235-247, 1983) in Nunc wells containing 6 mg/ml BSA, 0.2 mM pyruvate, 20 uM penicillamine, 10 uM hypotaurine, 1 mM epinepherine (Leibfried et al., J. Reprod. Fertil. 66:87-93, 1982), and 0.004 fig/ml heparin. Twenty microliters of sperm were added to generate a final concentration of 106 sperm/ml to 50 oocytes. Culture conditions were the same as those described above for nuclear transfer. Fertilization rates were over 90% based on pronuclear development.<br>
Chimeric nuclear transfer embryos<br>
In vitro fertilized embryos at 8-cell stage (6-12 blastomeres) were harvested at approximately 96 hours post fertilization, prior to compaction. The zona pellucida was removed with protease (3 mg/ml in TL-HEPES). The zona dissolution was carefully monitored using a dissecting microscope. When<br>
the zona first appeared to dissolve (~ two minutes), the embryos were removed and washed in TL-HEPES and transferred to 30 mm petri dishes containing Hank's balanced salt solution and incubated at 37.5C for 30 minutes. The blastomeres from these precompaction embryos were transferred into microdrops (50 jil) of TL-HEPES under mineral oil in 100 mm petridish. Nuclear transfer embryos on day four at the 8-16 cell stage were selected and transferred into the same microdrops containing the blastomeres. These nuclear transfer embryos included both precompaction embryos (e.g., 8 cell stage embryos) and compaction embryos (e.g., 16 stage embryos). Then 4-6 blatomeres were transferred into the nuclear transfer embryos with the beveled micropipette (35 urn diameter) using standard micromanipulation techniques. After transferring the blastomeres, the embryos were cultured as described for nuclear transfer embryos.<br>
On days seven and eight, the development to blastocyst of the chimeric embryos was evaluated. The blastocysts were also analyzed for the presence of the membrane dye Dil that was added to the cells from the in vitro fertilized embryo before they were injected into the nuclear transfer embryo. The cells were labeled on day four and observed on day seven. This dye is maintained for a few cell divisions in the progeny of the originally dyed cells, allowing the chimeric embryo to be analyzed after a few cell divisions. Based on this analysis, cells from the in vitro fertilized embryo were incorporated into the chimeric embryo. If desired, fluorescence in situ hybridization (FISH) with a probe specific for a nucleic acid in either the in vitro fertilized embryo or the nuclear transfer embryo can be performed using standard methods (see, for example, Ausubel et al, Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York, pp. 14.7.1-14.7.12, 1995). This FISH analysis can be used to determine the distribution of cells derived from each embryo in the chimeric embryo (e.g., to determine what percent of the cells are incorporated into the inner cell mass and what percent are incorporated into the trophectoderm) while it is cultured in vitro and in the fetus or the offspring generated from the<br>
embryo. Alternatively, a reporter gene such as green fluorescent protein can be added to cells from one of the embryos and used to monitor the incorporation of the cells into the placenta and various fetal tissues of the chimeric embryo.<br>
Embryo transfer<br>
Days seven and eight, nuclear transfer blastocysts of grade 1 and 2, derived from nuclear transfer embryos and chimeric nuclear transfer embryos were transferred into day six and seven synchronized recipient heifers. Recipients were synchronized using a single injection of Lutalyse (Parmacia &amp; Upjohn, Kalamazoo, MI) followed by estrus detection. The recipients were examined on days 30 and 60 after embryo transfer by ultrasonography for the presence of conceptus and thereafter every 30 days by rectal palpation until 240 days. The pregnancy results at day 40 for the chimeric embryos and for control embryos produced by fusing a transgenic bovine fibroblast with an oocyte are compared in Table 13. These results indicate that a greater number of chimeric embryos survived until day 40.<br>
(Table Remove)  Alternative methods for production of chimeric embryos<br>
Standard methods can be used to modify the above method for producing chimeric embryos. For example, a naturally-occurring embryo can be surgically isolated from a mammal (e.g., a bovine) or an oocyte can be parthenogenetically activated using standard techniques and used instead of the in vitro fertilized embryo. If desired, fewer cells from the in vitro fertilized, naturally-occurring, or parthenogenetically activated embryos (e.g., 1, 2, 3, 4, or 5 cells) can be injected into the nuclear transfer embryo to reduce the percent<br><br>
of the injected cells and their progeny that become incorporated into fetal tissue. Alternatively, more cells (e.g., 6, 7, 8, 9, 10, 11 or more cells) can be injected to increase the percent of the injected cells and their progeny that are incorporated into placental tissue. Moreover, cells from embryos in other cell stages can be used. For example, in vitro fertilized, naturally-occurring, or parthenogenetically activated embryos at the 4, 8, 16, 32, 64, 128, 256, 512, or later cell stage can be injected into nuclear transfer embryos at the 4, 8, 16, 32, 64, 128, 256, 512, or later cell stage. The injected cells and the nuclear transfer embryo can be at the same cell stage or at different cell stages. In one embodiment, the in vitro fertilized, naturally-occurring, or parthenogenetically activated embryo has increased ploidy (e.g., a DNA content of 4n) relative to the nuclear transfer embryo, which further biases the injected cells to the trophectoderm (i.e., the outermost layer of cells of the embryo that primarily forms the placental tissue). If desired, all or part of the zona pellucida can be kept surrounding the injected cells, rather than removed prior to injection.<br>
In other alternative methods, cells from a precompaction or compaction in vitro fertilized, naturally-occurring, or parthenote embryo are incubated with cells from a precompaction nuclear transfer embryo under conditions that allow cells from each embryo to reorganize to produce a single chimeric embryo (Wells and Powell, Cloning 2:9-22, 2000). Cells from in vitro fertilized, naturally-occurring, or parthenote embryo are expected to contribute primarily to the trophectoderm and eventually to the placental tissue, and cells from the nuclear transfer embryo are expected to contribute primarily to the inner cell mass and eventually to the fetal tissue. Cells from both embryos can be at the same cell stage or at different cell stages, and the same or different numbers of cells from each embryo can be combined to form the aggregation embryo.<br>
If desired, a cell from the resulting cloned fetus or the cloned offspring can be used in a second round of nulear transfer to generate additional cloned offspring. Cells from the initial cloned fetus or cloned offspring may also be<br>
frozen to form a cell line to be used as a source of donor cells for the generation of additional cloned ungulates.<br>
EXAMPLE 19: Testing for Human Ig Expression<br>
Testing calves retaining a HAC or other nucleic acid encoding a xenogenous antibody may begin shortly after birth and includes evaluation for (1) xenogenous Ig expression, (2) response to immunization, (3) affinity maturation, and (4) transmission of the HACs to offspring.<br>
Human Ig expression may be monitored by bleeding the animals and assaying for the presence of human heavy and light chain expression by ELISA, RT-PCR, or FACS analysis (see, for example, PCT Publication No. WO01/35735). Once it has been determined that the animals produce human Ig, animals are immunized with tetanus toxoid in adjuvant. Animals are bled once a week following immunization and responses to antigen determined via ELISA or FACS and compared to pre-bleeds collected before immunization. One month after the initial immunization, animals are boosted with an aqueous form of the antigen. One week following the boost, the animals are bled and response to antigen is measured via ELISA or FACS and compared to the prebleed. The ELISA or FACS assay permits measurement of most of the titer of the response as well as the heavy chain isotypes produced. This data allows a determination of an increase in antibody titer as well as the occurrence of class switching. Estimates of average affinity are also measured to determine if affinity maturation occurs during the response to antigen.<br>
After the transgenic bovines have been obtained as described above, they are utilized to produce transgenic Igs, preferably human, but potentially that of other species, e.g. dog, cat, non-human primate, other ungulates such as sheep, pig, goat, murines such as mouse, rat, guinea pig, rabbit, etc. As noted, Ig genes are known to be conserved across species.<br>
Transgenic antisera and milk containing xenogenous antibodies<br>
The bovine (or other ungulate) yields transgenic antisera directed to whatever antigen(s) it is endogenously exposed, or to exogenously administered antigen(s). For example, antigens may be administered to the ungulate to produce desired antibodies reactive with the antigens, including antigens such as pathogens (for example, bacteria, viruses, protozoans, yeast, or fungi), tumor antigens, receptors, enzymes, cytokines, etc. Exemplary pathogens for antibody production include, without limitation, hepatitis virus (for example, hepatitis C), immunodeficiency virus (for example, HIV), herpes virus, parvovirus, enterovirus, ebola virus, rabies virus, measles virus, vaccinia virus, Streptococcus (for example, Streptococcus pneumoniae), Haemophilus (for example, Haemophilus influenza), Neisseria (for example, Neisseria meningitis), Coryunebacterium diptheriae, Haemophilus (for example, Haemophilus pertussis), Clostridium (for example, Clostridium botulinium), Staphlococcus, Pseudomonas (for example, Pseudomonas aeruginosa), and respiratory syncytial virus (RSV).<br>
One or more pathogens may be administered to a transgenic ungulate to generate hyperimmune serum useful for the prevention, stabilization, or treatment of a specific disease. For example, pathogens associated with respiratory infection in children may be administered to a transgenic ungulate to generate antiserum reactive with these pathogens (e.g., Streptococcus pneumoniae, Haemophilus influenza, and/or Neissaria meningitis). These pathogens may optionally be treated to reduce their toxicity (e.g., by exposure to heat or chemicals such as formaldehyde) prior to administration to the ungulate.<br>
For the generation of broad spectrum Ig, a variety of pathogens (e.g., multiple bacteria] and/or viral pathogens) may be administered to a transgenic ungulate. This hyperimmune serum may be used to prevent, stabilize, or treat infection in mammals (e.g., humans) and is particularly useful for treating mammals with genetic or acquired immunodeficiencies.<br>
In addition, antibodies produced by the methods of the invention may be used to suppress the immune system, for example, to treat neuropathies, as well as to eliminate particular human cells and modulate specific molecules. For example, anti-idiotypic antibodies (i.e., antibodies which inhibit other antibodies) and antibodies reactive with T-cells, B-cells, or cytokines may be useful for the treatment of autoimmune disease or neuropathy (e.g., neuropathy due to inflammation). These antibodies may be obtained from transgenic ungulates that have not been administered an antigen, or they may be obtained from transgenic ungulates that have been administered an antigen such as a B-cell, T-cell, or cytokine (e.g., TNFa).<br>
Transgenic antisera generated from transgenic ungulates that have not been administered an antigen may be used to manufacture Pharmaceuticals comprising human polyclonal antibodies, preferably human IgG molecules. These human antibodies may be used in place of antibodies isolated from humans as Intraveneous Immunoglobulin (IVIG) therapeutics.<br>
Transgenic antiserum may optionally be enriched for antibodies reactive against one or more antigens of interest. For example, the antiserum may be purified using standard techniques such as those described by Ausubel et al. (Current Protocols in Molecular Biology, volume 2, p. 11.13.1-11.13.3, John Wiley &amp; Sons, 1995). Preferred methods of purification include precipitation using antigen or antibody coated beads, column chromatography such as affinity chromatography, magnetic bead affinity purification, and panning with a plate-bound antigen. Additionally, the transgenic antiserum may be contacted with one or more antigens of interest, and the antibodies that bind an antigen may be separated from unbound antibodies based on the increased size of the antibody/antigen complex. Protein A and/or protein G may also be used to purify IgG molecules. If the expression of endogenous antibodies is not eliminated, protein A and/or an antibody against human Ig light chain lambda (Pharmingen) may be used to separate desired human antibodies from endogenous ungulate antibodies or ungulate/human chimeric antibodies.<br>
Protein A has higher affinity for human Ig heavy chain than for bovine Ig heavy chain and may be used to separate desired Ig molecules containing two human heavy chains from other antibodies containing one or two ungulate heavy chains. An antibody against human Ig light chain lambda may be used to separate desired Ig molecules having two human Ig lambda chains from those having one or two ungulate Ig light chains. Additionally or alternatively, one or more antibodies that are specific for ungulate Ig heavy or light chains may be used in a negative selection step to remove Ig molecules containing one or two ungulate heavy and/or light chains.<br>
The resultant antisera may itself be used for passive immunization against an antigen. Alternatively, the antisera has diagnostic, prophylactic, or purification use, e.g. for attaining purification of antigens.<br>
Alternatively, after antisera administration, B-cells may be isolated from the transgenic bovine and used for hybridoma preparation. For example, standard techniques may be used to fuse a B-cell from a transgenic ungulate with a myeloma to produce a hybridoma secreting a monoclonal antibody of interest (Mocikat, J. Immunol. Methods 225:185-189, 1999; Jonak et al, Hum. Antibodies Hybridomas 3:177-185, 1992; Srikumaran et al., Science 220:522, 1983). Preferred hybridomas include those generated from the fusion of a B-cell with a myeloma from a mammal of the same genus or species as the transgenic ungulate. Other preferred myelomas are from a Balb/C mouse or a human. In this instance, hybridomas are provided that make xenogenous monoclonal antibodies against a particular antigen. For example, this technology may be used to produce human, cat, dog, etc. (dependent upon the specific artificial chromosome) monoclonal antibodies that are specific to pathogens. Methods for selecting hybridomas that produce antibodies having desirable properties, i.e., enhanced binding affinity, avidity, are well known.<br>
Alternatively, a B-cell from a transgenic ungulate may be genetically modified to express an oncogene, such as ras, myc, abl, bc!2, or neu, or infected with a transforming DNA or RNA virus, such as Epstein Barr virus or<br>
SV40 virus (Kumar et al., Immunol. Lett. 65:153-159, 1999; Knight et al., Proc. Nat. Acad. Sci. USA 85:3130-3134, 1988; Shammah et al, J. Immunol. Methods 160-19-25, 1993; Gustafsson and Hinkula, Hum. Antibodies Hybridomas 5:98-104, 1994; Kataoka et al, Differentiation 62:201-211, 1997; Chatelut et al, Scand. J. Immunol. 48:659-666, 1998). The resulting immortalized B-cells may also be used to produce a theoretically unlimited amount of antibody. Because Ig is also secreted into the milk of ungulates, ungulate milk may also be used as a source of xenogenous antibodies.<br>
Other embodiments<br>
From the foregoing description, it will be apparent that variations and modifications may be made to the invention described herein to adopt it to various usages and conditions. For example, other incubation times or temperatures (such as 25, 30, 35, 37, 38.5, or 40C or a higher or lower temperature) can be used for permeablizing membranes, reprogramming donor genetic material, resealing membranes, and/or tranfer of genetic material into oocytes. Such embodiments are also within the scope of the following claims.<br>
All publications mentioned in this specification are herein incorporated by reference to the same extent as if each independent publication or patent application was specifically and individually indicated to be incorporated by reference.<br><br><br><br><br>
WE CLAIM:<br>
1.	An in-vitro method for producing a transgenic bovine cell having reduced<br>
expression of functional prion protein, comprising introducing a first prion<br>
gene targeting vector into a bovine cell under conditions that allow homologous<br>
recombination between said first vector and a first allele of an endogenous<br>
prion nucleic acid in said cell, thereby introducing a non-naturally occurring<br>
hemizygous mutation in said cell, wherein the expression of functional prion<br>
protein in said cell is substantially eliminated and a nucleic acid for reducing<br>
the expression prion protein comprises a cassette which included in 5' to 3'<br>
order a first region of homology having at least 60% sequence identity to a first<br>
region of an endogenous prion nucleic acid of a bovine cell, a positive selection<br>
marker, and a second region of homology having at least 60% sequence identity<br>
to a second region of said prion nucleic acid, and wherein said cassette is<br>
capable of integrating into an endogenous prion nucleic acid of said cell.<br>
2.	An in-vitro method as claimed in claim 1, optionally comprising introducing said first vector into said cell under conditions that allow homologous recombination between said first vector and a second allele of an endogenous prion nucleic acid in said cell, thereby introducing homozygous mutation in said cell.<br>
3.	An in-vitro method as claimed in claim 1, wherein said mutation comprises an insertion of a positive selection marker into a prion nucleic acid.<br>
4.	An in-vitro method as claimed in claim 3, wherein said positive selection marker is operably linked to a xenogenous promoter.<br>
5.	An in-vitro method as claimed in claim 3, wherein said positive selection marker is an antibiotic resistance gene.<br><br>
6.	An in-vitro method as claimed in claim 5, optionally comprising<br>
introducing a second prion gene targeting vector that has a different antibiotic<br>
resistance gene than said first vector into said cell under conditions that allow<br>
homologous recombination between said second vector and a second allele of<br>
an endogenous prion nucleic acid in said cell, thereby introducing a<br>
homozygous mutation in said cell.<br>
7.	An in-vitro method as claimed in claim 5, wherein each allele comprises the same antibiotic resistance gene.<br>
8.	An in-vitro method as claimed in claim 5, wherein each allele comprises a different antibiotic resistance gene.<br>
9.	An in-vitro method as claimed in claim 2, wherein said mutation comprises an insertion of a transcription termination sequence into a prion nucleic acid.<br>
10.	An in-vitro method as claimed in claim 9, wherein said transcription<br>
termination sequence is inserted downstream of the initial ATG codon in exon<br>
3 of said prion nucleic acid.<br>
11.	An in-vitro method as claimed in claim 10, wherein said transcription termination sequence is inserted within 10 nucleotides of said ATG codon.<br>
12.	An in-vitro method as claimed in claim 9, wherein said transcription termination sequence is a STOP cassette.<br>
13.	An in-vitro method as claimed in claim 1, wherein said mutation comprises<br>
a deletion of one or more nucleotides in a prion nucleic acid.<br><br>
14.	An in-vitro method as claimed in any one of claims 1 to 13, wherein said cell comprises one or more nucleic acids comprising one or more transgenes and expressing an mRNA or protein expressed by said transgene(s).<br>
15.	An in-vitro method as claimed in any one of claims 1 to 13, wherein said cell comprises one or more nucleic acids comprising all or part of a xenogenous immunoglobulin (Ig) gene, wherein said gene is capable of undergoing rearrangement and expressing one or more xenogenous Ig molecules in B-cells.<br><br>
16.	An in-vitro method as claimed in claim 14 or 15, wherein said cell comprises one or more nucleic acids encoding a xenogenous antibody.<br>
17.	An in-vitro method as claimed in claim 16, wherein said xenogenous antibody is a human antibody.<br><br>
18.	An in-vitro method as claimed in any one of claims 1 to 17, wherein said cell comprises a mutation that reduces the expression of an endogenous antibody.<br>
19.	An in-vitro method as claimed in claim 18, wherein said mutation reduces the expression of functional IgM heavy chain.<br>
20.	An in-vitro method as claimed in any one of claims 1 to 19, wherein said cell comprises a mutation in one or both alleles of an endogenous nucleic acid encoding alpha-( 1,3)-galactosyltransferase.<br>
21.	An in-vitro method as claimed in any one of the claims 1 to 20, wherein said<br>
cell is a bovine fibroblast.<br><br>
22.	An in-vitro method as claimed in claim 21, wherein said cell is a bovine fetal fibroblast.<br>
23.	An in-vitro method as claimed in any one of claims 1 to 20, wherein said cell is a B-cell.<br>
24.	An in-vitro method as claimed in claim 1, wherein a nucleic acid for<br>
reducing the expression of functional prion protein, comprises a cassette which<br>
includes in 5' to 3' order a first region of homology having at least 60%<br>
sequence identity to a first region of an endogenous prion nucleic acid of a<br>
bovine cell, a positive selection marker, and a second region of homology<br>
having at least 60% sequence identity to a second region of said prion nucleic<br>
acid, wherein said first region of homology is longer than said second region of<br>
homology, and wherein said cassette is capable of integrating into an<br>
endogenous prion nucleic acid of said cell.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUFic3RyYWN0LSgwMi0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Abstract-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUNsYWltcy0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Claims-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUNsYWltcy0oMTQtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Claims-(14-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Correspondence-Others-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Correspondence-Others-(14-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMDItMDctMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-description (complete)-02-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMTQtMDgtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-description (complete)-14-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUZvcm0tMS0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Form-1-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMoMTAtMDctMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13(10-07-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMoMTEtMDUtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13(11-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMoMTItMTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13(12-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMoMTQtMDUtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13(14-05-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMtKDAzLTA5LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13-(03-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUZvcm0tMi0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Form-2-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUZvcm0tMy0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Form-3-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tNigwMy0wOS0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-6(03-09-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWZvcm0tNi0xMi0xMi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-form-6-12-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LUdQQS0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-GPA-(02-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC1ub3RpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-notification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1kZWxucC0yMDA1LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1614-delnp-2005-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYxNC1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMDItMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1614-DELNP-2005-Petition-137-(02-07-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234274-an-anti-inflammatory-antiarthritic-and-a-vasculodilator-herbal-oil-from-momordica-charantia-l-bitter-gourd.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234276-a-gasket-assembly.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234275</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1614/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>15-May-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Apr-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>KIRIN PAHRMA KABUSHIKI KAISHA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>26-1 JINGUMAE 6-CHOME,SHIBUYA-KU,TOKYO,JAPAN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>POOTHAPPILLAI KASINATHAN</td>
											<td>1901 S. DOROTHY AVENUE, SIOUXFALLS, SD 57106, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JAMES ROBL</td>
											<td>26601 TUCKER DRIVE, BRANDON, SD 57005, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KAZUMA TOMIZUKA</td>
											<td>401, RM2 TAKASAKI, 110-1, YASHIMA-CHO, TAKASAKI-SHI, GUNMA 370-0849, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ISAO ISHIDA</td>
											<td>352-6 KUSHIHASHI, ISEHARA-SHI, KANAGAWA-KEN 259-1136, JAPAN.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>YOSHIMI KUROIWA</td>
											<td>5015 S. GLENVIEW, SIOUX FALLS, SD 57108, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>PHILIPPE COLLAS</td>
											<td>NYGAARDSALLE 4A, N-0871 OSLO, NORWAY.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>EDDIE SULLIVAN</td>
											<td>410 EVERETT STREET, TEA, SD 57064, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/035720</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-11-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/425,056</td>
									<td>2002-11-08</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/506,901</td>
									<td>2003-09-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234275-a-method-for-producing-a-transgenic-bovine-cell-having-reduced-expression-of-functional-prion-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:01 GMT -->
</html>
