<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234296-a-truncated-enzymatically-active-thermolabile-antarctic-phosphatase-tap by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234296:A TRUNCATED ENZYMATICALLY ACTIVE THERMOLABILE ANTARCTIC PHOSPHATASE (TAP)</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A TRUNCATED ENZYMATICALLY ACTIVE THERMOLABILE ANTARCTIC PHOSPHATASE (TAP)</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A truncated enzymatically active thermolabile Antarctic phosphatase (TAP), wherein the I truncation corresponds to a deletion of a signal sequence, the phosphatase having a C- terminal and N-terminal end, wherein the N-terminal end is covalently linked to a hydrophi)ic positively charged oligopeptide leader sequence and wherein the TAP is substantially inactivated at about 65°C within 30 minutes and is active in a Bis- Tris propane buffer at a pH between 5.5 and 6.0.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
OVEREXPRESSION, PURIFICATION AND CHARACTERIZATION OF A<br>
THERMOLABILE PHOSPHATASE<br>
BACKGROUND OF THE INVENTION<br>
Purified phosphatases are used by molecular biologists to remove phosphate groups from linearized DNA, RNA, nucleotides and proteins in vitro. Removal of both 5'-terminal phosphate residues from a linearized DNA vector prevents recircularization of the vector DNA in a ligation reaction because DNA ligase catalyzes the formation of a phosphodiester bond between adjacent nucleotides only if one nucleotide contains a 5'-phosphate and the other a 3'-hydroxyl group. However, foreign DNA segments with 5'-terminal phosphates can be ligated efficiently into the dephosphorylated vector to give an open circular molecule containing two nicks that can easily be transformed into competent cells.<br>
Several different phosphatases have been developed for removal of phosphate groups from biological molecules, and each has its own advantages and disadvantages. The first phosphatase to be used for this purpose was Bacterial Alkaline Phosphatase (BAP) purified from Escherichia coli (E. coli). BAP has the advantage of having good activity against all types of DNA ends. However, it is difficult to remove from the reaction as it is very resistant to heat and detergents (Sambrook, et al., Molecular Cloning, A Laboratory Manual Sections 1.53-1.72 (1989)). A mutant BAP has been prepared (U.S. Patent No. 5,891,699, EP Patent No. 0441252) which reportedly had increased thermostability. Calf Intestinal Alkaline Phosphatase (CIP or CIAP) is another phosphatase that has been extensively used in molecular biology techniques (U.S. Patent Nos. 5,773,226 and 5,707,853). CIAP is not as active on DNA as BAP, but it is slightly easier to remove from a reaction requiring the use of either Proteinase K treatment followed by phenol .'chloroform extractions or a heat step in the presence of EDTA followed by a phenohchloroform extraction (Sambrook, et al., supra (1989)). More recently a phosphatase isolated from Arctic shrimp Pandalus borealis (SAP) (U.S. Patent Nos. 6,387,634 and 6,379,940) has proved easier to use.   It has good activity against all types of DNA<br><br>
ends like BAP, but it is reported to have the advantage that it is easily removed from the reaction by heat inactivation at 65°C for 15 minutes (Amersham Bioscience, Piscataway, NJ).<br>
Other reports have occurred in the literature for other thermolabile phosphatases including one purified from a psychrophilic strain TAB5 isolated from Antarctica referred to as Thermolabile Antarctic Phosphatase (TAP) (Rina, et ah, Eur. J. Biochem. 267:1230-1238 (2000)). Advantages of TAP over other phosphatases include heat lability and high specific activity. Rina, et al. reported that this phosphatase had a specific activity of 1650 units/mg of protein for p-nitrophenyl phosphate (pNPP) substrate which was significantly higher than the activity of any other known phosphatase. However, the protein produced by the clone reported by Rina, et al., {supra (2000)) had a number of problems associated with overexpression and purification. For example, the purification protocol described by Rina, et al., {supra (2000)) requires multiple ultracentrifiigation steps to extract the TAP from cell membranes with which it was apparently associated. This protocol is not suited for large scale manufacture (for example, manufacturing protocols involving production of 300 g or more of cell paste). In addition, overexpressing the TAP gene using the protocol described in Rina, et al., resulted in yields of the enzyme that were very low and consequently not cost effective for large scale manufacture.<br>
SUMMARY OF THE INVENTION<br>
Present embodiments of the invention provide an alkaline phosphatase with increased heat lability at neutral pH and enhanced activity and a hydrophilic leader sequence at the N-terminal end of the protein. The hydrophilic leader sequence may be an oligopeptide, for example, an oligopeptide with a net positive charge such as a His tag. Advantages of this phosphatase include the ability to eliminate its activity by raising the temperature to a level at which other reactants and products remain unaffected. An example of a phosphatase having these properties is TAP which is described in detail in the examples. Although the method described herein relies on the presence of a hydrophilic leader sequence, such as His tag, fused to the N-terminal<br><br>
end of the protein, the protein is preferably not purified using a nickel column because the nickel column buffer had an adverse effect on activity. Nonetheless, the presence of the hydrophilic leader sequence surprisingly enhanced yields during column purification using other column compositions.<br>
In an embodiment of the invention, a truncated enzymatically active TAP is provided in which the truncation corresponds to a deletion of a signal sequence, the phosphatase having a C-terminal and N-terminal end, wherein the N-terminal end is covalently linked to a hydrophilic leader sequence, such as a positively charge oligopeptide, for example, a His tag. The truncated TAP can be substantially inactivated at about 65°C and in less than 15 minutes. In particular, the activity of the truncated TAP is substantially stable in Tris-HCL at a pH greater than pH 6 more particularly at about pH 7.4.<br>
In addition, a DNA is described that encodes TAP (also listed in GenBank Y18016) but which lacks some or all of the signal sequence associated with the naturally occurring TAP gene. The DNA may further include sequences that encode a plurality of His amino acids preferably at the N-terminal of the expressed TAP gene. This DNA may be expressed in a vector under control of a strong promoter, for example, the T7 promoter.<br>
In a preferred embodiment, vectors are provided that contain the DNA described above. An example of these vectors is pEGTAP7.4.L In additional embodiments of the invention, host cells are provided which have been transformed with at least one vector. In the examples, the host cell is E. coli although the use of other host cell types is not excluded.<br>
In additional embodiments of the invention, a formulation of TAP is provided which is substantially stable at 37°C in a buffer which includes Tris-HCL, MgCk&gt; DTT and glycerol and more particularly where the pH is about 7.4.<br><br>
In an embodiment of the invention, a method is provided for overexpressing TAP, that includes (a) operably linking to a T7 promoter, a truncated TAP gene fused to a sequence for expressing a hydrophilic oligopeptide, such as a His tag; (b) transforming a host cell; and (c) overexpressing TAP. In further embodiments, the TAP gene encodes a TAP protein having an N-terminal and a C-terminal end, wherein the TAP protein is truncated at the N-terminal end, the truncation corresponding to a signal sequence, the N-terminal end having the hydrophilic oligopeptide attached hereto.<br>
In an additional embodiment of the invention, a method is provided for obtaining a purified TAP, that includes (a) obtaining a transformed host cell as described above; (b) disrupting the host cells to yield a soluble fraction; and (c) purifying TAP from the soluble fraction. Moreover, the method may additionally include a column separation wherein the TAP is eluted from the column. In this context, purification refers to an increase in the percentage of TAP protein relative to total protein in the disrupted cell mixture. For example, purification may be achieved when the percentage of TAP protein is more than 5% of the total protein in the mixture. Preferably, the percentage of TAP protein may be more than 50%, 80% or 90% of the total protein. In a particular embodiment, the percentage of TAP protein is purified to about 95% of the total protein.<br>
In an additional embodiment of the invention, a method is provided for assaying for TAP activity, that includes: (a) adding TAP to a substrate in a buffer at about pH 5.5-7.0; and (b) determining the amount of phosphate removed from the substrate in a defined time and at a defined temperature to determine TAP activity. In a preferred embodiment, the buffer includes ZnCb and MgCh salts. More particularly the buffer is Bis-Tris Propane.<br>
In an additional embodiment, a method of dephosphorylating a phosphorylated substrate is provided that includes adding TAP to the substrate in a buffer at pH 5.5-7.0 and causing an amount of phosphate to be removed from the substrate in a defined time and at a defined temperature.<br><br>
The substrate may be a nucleic acid or a terminal nucleotide on a nucleic acid, a sugar phosphate, or a phosphorylated peptide or protein.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the result of purification of TAP from an E. coli strain containing pNl. Lanes 1 through 6 are sequential steps in a purification. Lane 1 is total crude extract after induction. Lane 2 is the solublized membrane fraction. Lane 3 is flow through from a DEAE column. Lane 4 is pooled fractions from a Q-sepharose column. Lane 5 is flow through from a HiTrap S column. Lane 6 is flow through from a Hydroxyapatite column. Lane 7 is purified TAP obtained from the pNI clone. Molecular weight standards (S) are denoted in kD. The band corresponding to 47.5 kD in lane 7 represents substantially purified TAP.<br>
Figure 2 shows a schematic diagram of the steps used to construct vectors for over-expressing the TAP gene. The rectangle at the top of the figure is a schematic drawing of the gene for TAP including an N-terminal and C-terminal portion of the corresponding expression product of the gene. At the N-terminal end, the location of the methionine start codon and the region around the processing site encoding the hypothesized signal sequence is shown. At the C-terminal end, amino acids of the gene product are shown with the stop codon marked by an asterisk. Primers used in PCR amplification of the TAP gene are drawn above (not to scale) (for the forward primers A, B and C) and below (for the reverse primers D and E) the rectangle at the site at which each primer hybridizes to the TAP gene. The N and X above the primers show engineered cleavage sites for the restriction endonucleases Ndel or Xhol . Primer C shows 6 Hfs following the Ndel site. These correspond to 6 codons encoding histidine inserted downstream of and adjacent to the methionine start codon in the primer to create an N-terminal His-tag fusion with TAP. The next set of lines following "PCR" indicate the resulting products produced by using the pairs of primers indicated. As above, the asterisk indicates a stop codon and the 6 H's refer to 6 histidine codons. 5 different plasmid are shown which were created by inserting<br><br>
PCR products into pET21a. pEGTAP 1.1.1 encodes for intact TAP with no His-tag. pEGTAPl.2.1 encodes for truncated TAP with no His-tag. pEGTAP 1.3.1 encodes for intact TAP with a C-terminal His-tag. pEGTAPl.4.1 encodes for truncated TAP with a C-terminal His-tag. In a later experiment pEGTAP7.4.1 was constructed to encode truncated TAP with an N-terminal His-tag.<br>
Figure 3 shows an SDS-PAGE analysis of TAP produced from different clones in which the TAP gene is either truncated or not truncated in the presence or absence of a His-tag. "Test" purification refers to a sample size of less than 1 ml. Lane 1 is the soluble crude extract from clone pEGTAP 1.1.1 with the intact TAP gene and with no His-tag. Lane 2 is the soluble crude extract from clone pEGTAPl.2.1 with the truncated TAP gene with no His-tag. Lanes 3 and 4 are soluble crude extracts from two different isolates of the clone pEGTAPl.3.1 with the intact TAP gene fiised to a His-tag. Lanes 5 and 6 are soluble crude extracts from two different isolates of the clone pEGTAPL4.1 with the truncated TAP gene fused to a His-tag. Molecular weight standards (S) are denoted in kD. Those clones having a truncated TAP gene (lanes 2, 5 and 6) provided the strongest bands corresponding to TAP.<br>
Figure 4 shows SDS-PAGE analysis of soluble TAP purified under non-denaturing conditions from an E. coli (ER2566) clone containing the truncated gene with a C-terminal His-tag (pEGTAPl.4.1). 10 |al samples of the total crude extract (T), the clarified crude extract (load) (L), the flow-through (FT) from the Ni-NTA spin column (Qiagen, Studio City, CA), the wash from the Ni-NTA spin column (W) and the first and second samples eluted from the Ni-NTA spin column with the elution buffer (El and E2) were run on a 10-20% Tris tricine PAG (Invitrogen Corp., Carlsbad, CA) and stained with Coomassie Brilliant Blue. Molecular weight standards (S) denoted in kD.<br>
Figure 5 shows the result of a purification protocol similar to that used in Figure 4 except that the clone analyzed was ER2566 (pEGTAP7.4.1). Also, instead of running the total cell extract on the gel, the pellet resulting from spinning the crude extract was resuspended and run as insoluble pellet (P). 10 ^1 of pellet (P), soluble load (L),<br><br>
flow through from Ni-NTA spin column (FT), wash from Ni-NTA spin column (W), elutions from Ni-NTA spin column with imidazole (E) and molecular weight standard (S) denoted in kD were run on a 10-20% Tris tricine PAG and stained with Coomassie Brilliant Blue.<br>
Figure 6 shows a comparison of TAP activity at different pH and in different buffers. pH optimum was determined using the phosphate release assay. 50 mM of each buffer was incubated in a reaction mixture containing the P labeled duplex DNA, 1 mM MgCl2 and 0.1 mM ZnCb. The reaction was incubated at 37°C for 5 minutes and the amount of trichloroacetic acid (TCA) soluble counts was measured.<br>
Figure 7 shows the effect of different salts on the activity of TAP using the phosphate release assay. 100 mM of each salt, KC1, NaCl, potassium acetate or sodium acetate were tested.<br>
Figure 8 shows a comparison of the thermolability of CIAPS SAP and TAP. Each enzyme was incubated in the presence of DNA at 65°C for up to 30 minutes. Samples were taken at different time points during the incubation and were assayed for activity with pNPP.<br>
Figure 9 shows the stability of TAP at 37°C and 25°C by measuring the release of phosphate from pNPP by TAP over time. Sample tubes containing 1 unit of TAP and unreacted pNPP were incubated at either 25°C or 37°C. Reactions were stopped at 5-minute intervals for 0-30 minutes by adding NaOH to a final concentration of 5 N. Release of phosphate (accumulation of pNP) was measured by spectrophotometer at 405nm (yellow) and graphed vs reaction time.<br>
Figure 10 shows a comparison of the efficiency of TAP to SAP in the removal of phosphate groups from different types of DNA termini: 55 overhang, blunt ends and 3' overhang were tested. These ends were produced by digestion of the vector with Hindlll, EcoRV and PstI respectively.<br><br>
Figure 11 shows the reaction time course of deoxynucleotidase activity for TAP and SAP. The composition of the dATPase reaction mixture was measured by Cap-HPLC at 5-minute intervals. The results are expressed for each component as a percentage of the total nucleotide present.<br>
Figure 12 shows the release of inorganic phosphate from pyrophosphate by TAP. Increasing amounts of TAP were added to each reaction mix containing 0.32 mM sodium pyrophosphate. Inorganic phosphate was measured essentially by the method of Heinonen and Lahti (Heinonen, J.K. and Lahti, R.J. "A new and convenient colorimetric determination of inorganic orthophosphate and its application to the assay of inorganic pyrophosphatase."Anal. Biochem. 113(2): 313-7, 1981) by comparison to known phosphate standards. Thermostable inorganic pyrophosphate (TIPP) was used in a parallel reaction to demonstrate equivalence. The results shown are for a 10 minute assay at 37°C for TAP and at 75°C for TIPP.<br>
Figure 13 provides a table showing the specific activity of TAP as compared to CIAP on phosphorylated myelin basic protein.<br>
Figure 14 shows the activity of TAP on three phospho-peptides:phospho-serine (graph A), phospho-threonine (graph B) and phospho-tyrosine (graph C) as measured by Cap-HPLC. The area of the absorbance peaks for the phospho-peptide and the peptide are measured and plotted on the graph as a percent of total area versus the number of units of TAP.<br>
Figure 15 shows TLC analysis of a titer of TAP on three different sugar-phosphates, glucose-1-phosphate, glucose-6-phosphate and mannose-6-phosphate. 3^g pf Glucose (G) and mannose (M) were spotted as positive controls. Negative controls (-) contain just the reaction mix containing buffer and the sugar phosphate substrate incubated without enzyme. Lanes marked 1 through 6 are the serial dilution (1:1) of the enzyme in the reaction mix starting with reaction 1 containing 12.5 units of TAP down to reaction 6 containing 0.4units of TAP. Arrows indicate the distances that the sugars and the sugar-phosphates migrate on the TLC.<br><br>
Figure 16 shows the sequence of the primers (SEQ ID NOS:6 and 7) used to PCR a 1.1 kb fragment from the pNl plasmid. The expected N-terminal amino acid sequence of the protein (SEQ ID NO:8) resulting from expression from the cloned PCR fragment is indicated below the sequence of the forward primer. The Ndel and Xhol cloning sites contained within the primers are also indicated.<br>
Figure 17 shows the DNA (SEQ ID NO:9) and amino acid sequence (SEQ ID NO:10) of the intact TAP gene. Indicated in the box is the start of the protein as determined by N-terminal sequence analysis (Rina, et al., supra). Arrows indicate the location at which the PCR primers used to clone the truncated gene would anneal.<br>
DETAILED DESCRIPTION OF THE EMBODIMENTS<br>
TAP gene has been isolated and cloned (pNl) by Rina, et al., and the DNA sequence has been published. However, when attempts were made to overexpress and purify TAP from this clone using the published protocol, a number of problems were encountered including obtaining insufficient amounts of purified enzyme for reagent purposes and the need for multiple ultracentrifugation steps to separate the enzyme from membrane fragments with which it was associated. A protocol is provided herein in which the TAP gene is reliably overexpressed to provide yields of enzyme that are cost effective for large scale manufacture. Furthermore, a purification protocol has been developed that avoids the need for multiple ultracentrifugation steps and is suited for large scale manufacture. Uses for TAP have been established for removing phosphates from nucleic acids, peptides, and sugars. Other substrates which may act as a substrate for TAP are those described by Reid and Wilson in The Enzymes" vol IV (1971) ch 17, p 373.<br>
In a preferred embodiment, "sugars" here refer to monosaccharides, oligosaccharides or polysaccharides. Sugars include any pentose, hexose or heptulose compound. A phosphate may be cleaved using TAP from any carbon in the sugar wherever the phosphate or phosphates are positioned irrespective of whether the<br><br>
phosphate is an alpha or beta isomer. Examples of some sugar phosphates are provided at the web site http://www.arabidopsis.org. Examples of sugar phosphates include: phosphates of mannose, glucose, fucose, galactose, N-acetylgalactosamine, N-acetylgluosamine, xylose, and rhamnose.<br>
Purification of TAP from ER2575 (ER2566 pLysS) containing the pNl plasmid was attempted using modifications of the published protocol to improve yield and ease of purification. The transformed E.coli strain was grown and induced and the phosphatase was purified as described in Rina, et al. (supra), with the following exceptions. After the soluble membrane fraction was isolated by ultracentrifugation, the proteins were dialyzed in buffer A (20 mM Tris pH 7.6, 10 mM MgCfe, 50 mM NaCl, 0.2% Triton X-100) then loaded onto a DEAE column which had been equilibrated with buffer A. The flow-through which had the phosphatase activity as determined with a pNPP assay was loaded on to a Q-sepharose column as described in Rina, et al. The pooled fractions from the Q-sepharose column were dialyzed in buffer B (Potassium phosphate buffer pH 6.6, 25 mM NaCl, 10 mM MgCl2) and loaded on a Hi-trap SP column which had been equilibrated in buffer B, The flow-through which had the phosphatase activity was dialyzed in buffer C (Potassium phosphate buffer pH 7.2, 50 mM NaCl, 10 mM MgCfe) and loaded onto a hydroxyapatite column which had been equilibrated with buffer C. The phosphatase activity flowed through. Samples from each step were run on a 10-20% Tris tricine PAG (Invitrogen Corp., Carlsbad, CA) and stained with Coomassie Brilliant Blue. As can be seen in Figure 1, the yield of phosphatase in lanes 1-6 was disappointing.<br>
With the disappointing results obtained using the above protocols, alternative approaches were investigated that involved cloning the TAP gene as described in Figure 2.<br>
Specifically, the TAP gene was cloned directly behind a strong promoter (T7 promoter) and attached a His-tag (affinity tag) to facilitate purification. The His-tag sequence shown in Figures 13 and 14 has six histidines fused to the N-terminal end of the protein (51 end of the gene). However, this number of histidines is not critical. The<br><br>
number of histidines in the tag may be as few as three or as many as desired in excess of six to achieve the purpose described herein. Several different constructs were made. In two constructs, the TAP gene was truncated by removing a putative signal sequence consisting of 22 amino acids at the N-terminal end of the protein adjacent to the methionine start codon. A C-terminal His-tag was added to form a fusion protein with the truncated phosphatase for one clone but not for another. For example, pEGTAPl .4.1 had a His-tag while pEGTAP 1.2.1 did not. In two other constructs, the gene retained the putative signal sequence and either additionally contained a C-terminal His-tag (pEGTAPl.3.1) or not (pEGTAPl.1.1) (Figure 2).<br>
Five different plasmids are shown in Figure 2 which were created by inserting PCR products into pET21a (Example I, Figures 2, 11 and 12). pEGTAP 1.1.1 encodes for intact TAP with no His-tag. pEGTAP 1.2.1 encodes for truncated TAP with no His-tag. pEGTAP 1.3.1 encodes for intact TAP with a C-terminal His-tag. pEGTAPl.4.1 encodes for truncated TAP with a C-terminal His-tag. pEGTAP7.4.1 encodes truncated TAP with an N-terminal His-tag.<br>
Plasmids containing the intact TAP gene with the putative signal sequence, with or without the His-tag at the C-terminal end, failed to produce any obvious soluble phosphatase on SDS-PAGE after induction (Figure 3, lanes 1, 3 and 4). However, those plasmids constructed without the putative signal sequence either with or without the His-tag produced a very strong band of the correct size on SDS-PAGE (Figure 3, lanes 2, 5 and 6). However, problems occurred during purification of the protein. The transformed host cell ER2566 containing pEGTAPl.4.1 (truncated TAP and C-terminal His) produced predominantly insoluble TAP in non-denaturing conditions (Figure 4). Purification was attempted under denaturing conditions followed by refolding of the denatured phosphatase. However, this approach yielded relatively small amounts of active enzyme. E. coli ER2566 containing pEGTAP 1.2.1 (truncated TAP with no His-tag) produced soluble TAP, however, the protein failed to bind to most columns.<br><br>
To overcome the above problems, a fifth clone was constructed in which a sequence encoding His-tag was placed at the N-terminal end instead of the C-terminal end of the TAP gene (pEGTAP7.4.1) (Figure 2). This construct produced good yields of TAP in the soluble fraction (Figure 5) which could be readily purified without multiple ultracentrifugation steps (Example II). Specifically, the presence of His-tag at the N-terminal end of the phosphatase enabled purification to near homogeneity on only 2 or 3 columns not including a nickel column. While not wishing to be limited by theory, it appears that the primary use of the His-tag was to facilitate production of soluble phosphatase rather than as an affinity tag.<br>
The activity of the TAP phosphatase prepared by the above method (Example I and II) was compared with SAP and CIAP phosphatase activity by means of assays employing DNA and protein substrates. (Example VII) and TAP was found to have enhanced activity. For example, linear DNA dephosphorylation activity of TAP and SAP was determined using a phosphate release assay described in Example VII, for 5' overhangs', 3' overhangs and blunt ends of DNA. When the dephosphorylation of these different structures were compared using identical pNPP unit amounts, TAP proved to be consistently more efficient than SAP (Figure 10).<br>
TAP is also able to remove phosphate groups from serine/threonine and tyrosine residues on phosphorylated proteins as efficiently as CIAP (Example VII, Figure 13).<br>
In addition to increased activity of TAP compared with other phosphatases, TAP was found to have enhanced thermolability (Example IV, Figure 8). When the heat lability of CIAP, SAP and TAP was compared, it was found that TAP had lost 98% of its activity after 5 minutes at 65° C compared with SAP which had lost 70% activity and CIAP which had lost only 10% of its activity at this temperature.<br>
In other embodiments of the invention, ZnCfe and MgCfe were found to enhance the activity of the phosphatase (Example III). The activity of TAP was optimized at a pH of about 5.5-7.0, for example, using Bis-Tris Propane buffer (Example III).<br><br>
The present invention is further illustrated by the following Examples. These Examples are provided to aid in the understanding of the invention and are not construed as a limitation thereof.<br>
The references cited above and below are hereby incorporated by reference herein.<br>
EXAMPLES<br>
Example I: Construction and expression of a TAP His-tag fusion gene<br>
The truncated phosphatase gene derived from an unclassified Antarctic strain TAB5 was cloned with an N-terminal 6-histidine residue tag. Briefly, a 1.1 kb fragment was amplified by PCR from the recombinant plasmid pNl (Rina, et al., Eur. J. Biochem. 267:1230-1238 (2000)). The forward primer (Figure 2 primer C), in addition to containing the sequence homologous to the 5' end of the gene without the putative signal sequence (63 bp) (Figures 16 and 17), contained the Ndel restriction site as part of the ATG start and 6 codons encoding for 6 histidine residues placed between the ATG start and the first codon of the truncated phosphatase gene (Figures 16 and 17). The reverse primer (Figure 2 primer D), in addition to containing the sequence reverse complementary to the 3' end of the gene, contained an Xhol restriction site immediately downstream of the TAA stop codon (Figures 16 and 17). Once amplified, the 1.1 kb fragment was digested with Ndel and Xhol and ligated into the T7 expression vector pET21a (Novagen, Madison, WI) similarly digested with Ndel and Xhol (Sambrook, et al., Molecular Cloning, A Laboratory Manual, Sections 1.53-1.72 (1989)). The resulting ligation was transformed into the E. coli strain ER2688 (New England Biolabs, Beverly, MA). Plasmids from the transformation were isolated and characterized by restriction mapping and DNA sequencing and when characterized proved to contain an insert of the correct size and sequence. The plasmid constructs were transformed into E. coli strain ER2566 (New England Biolabs, Beverly, MA) for expression of the phosphatase gene from the T7 promoter. pEGTAP7.4.1 was found to provide the best yield and purification profile (Figures 3-5).<br><br>
Example II: Production and purification of the TAP His-tag fusion protein<br>
(a)	Small scale purification of TAP<br>
ER2566 (pEGTAP7.4.1) was grown overnight at 30°C from an isolated colony in 5 ml Rich Broth ((10 g Tryptone (Difco Laboratories, Livonia, MI), 5 g Yeast Extract (Difco Laboratories, Livonia, MI), 5 g NaCl ph to 7.2 with NaOH per liter)) with 100 mg/ml carbenicillin. 3 ml of the overnight culture was used to inoculate 300 ml of Rich Broth with 100 mg/ml carbenicillin. The culture was grown at 30°C to mid log (Klett 70) and chilled on ice, induced with 0.4 mM isopropyl-Q-D-thiogalactoside (IPTG) and grown at 15°C for 20 hours. 35 ml of cells were harvested by centrifligation at 8000 rpm for 10 minutes at 4°C. The supernatant was removed, the cell pellet was weighed and placed at -20°C for 2 hours. The pellet was thawed on ice and the cells were resuspended in 1 ml lysis buffer (50 mM NaH2P04, 300 mM NaCl, 10 mM imidazole pH 8.0). Lysozyme was added to 1 mg/ml and the sample was incubated on ice for 30 minutes. Following incubation the cells were broken by sonication on ice for 6 times at 10 seconds each time with a 1 minute rest halfway through. The crude extract was spun at 14,000 rpm for 20 minutes at 4° C to remove any unbroken cells and any other insoluble material to generate a clarified crude extract. 0.6 ml of the clarified crude extract was loaded on to a Ni-NTA spin column which had been equilibrated with 0.6 ml lysis buffer. The sample was centrifuged at 700 x g for 2 min. The flow-through was removed and the column was washed with 2 x 0.6 ml wash buffer (50 mM NaH2P04, 300 mM NaCl, 20 mM imidazole pH 8.0). The protein was eluted with 2x 0.2 ml elution buffer (50 mM NaH2P04, 30 mM NaCl, 250 mM imidazole pH 8.0). 10 ^1 of each step were analyzed on SDS-PAGE. The results for pEGTAP7.4.1 are shown in Figure 5.<br>
(b)	Large scale purification of TAP from pEGTAP7.4.1 in ER2566<br>
E. coli strain ER2566 harboring the pEGTAP7.4.1 plasmid was grown at 25°C in 2000 ml Fermentation Rich Broth (500 g Amberex yeast extract (Sensient Technologies, Milwaukee, WT), 1 kg CE90MS Tryptone (Marcor Development Corp., Carlstadt, NJ), 27 g NaOH, 10 g ampicillin, 5.5 g polypropylene glycol antifoam agent (Arco Chemical Co., South Charleston, WV) per 100 liters) at 25°C. This<br><br>
culture was used to inoculate 100 liters of Fermentation Rich Broth. The cells were grown aerobically at 25°C for 5 hours until they reached a Klett of 70. The fermentor was then cooled to 15°C and IPTG was added to a final concentration of 0.3 mM when the Klett reached 90. The cells were then grown aerobically at 15°C for a further 17 hours, reaching stationary phase with a final Klett of 305. One 100 liter fermentation was required to harvest 321 grams of wet cell pellet. The 321 gram cell pellet was suspended in 963 ml buffer A (20 mM Potassium phosphate buffer (pH 7.4), 50 mM NaCl, 5% Gycerol) and passed through a Gaulin homogenizer at -12,000 psig. The lysate was centrifuged at ~13,000xG for 40 minutes and the supernatant collected (1150 ml).<br>
The supernatant solution was applied to a 500 ml Heparin Hyper-D column (Ciphergen Biosystems, Inc., Fremont, CA) equilibrated in buffer A. A 1.0 L wash of buffer A was applied, then a 2 L gradient of NaCl from 0.05 M to 1 M in buffer A was applied and fractions of 50 ml were collected. Fractions were assayed for phosphatase activity by incubating samples with 0.1 M pNPP in 1M diethanolamine/HCL buffer (pH 8.5) containing 10 mM MgCfe. Reactions were carried out at 37°C for 1-5 minutes and activity was measured as generation of yellow color spectrophotometrically at 405 nm. Phosphatase activity eluted from 0.05 - 0.35 M NaCl.<br>
The Heparin Hyper-D column fractions containing the phosphatase activity were pooled, then dialyzed against buffer A overnight. 100 ml of this 800 ml pool was applied to a 105 ml Source Q column (Pfizer, Inc., NY, NY). A 210 ml wash with buffer A was applied followed by a 1.0 L gradient from 0.05 M to 1.0 M NaCl in buffer A and fractions of 15 ml were collected. Fractions were assayed using the pNPP assay described above and the phosphatase activity eluted from 0.15 - 0.18 M NaCl. The remaining 700 ml of the Heparin Hyper-D pool was similarly applied and eluted; then the gradient fractions containing phosphatase activity were pooled.<br>
The combined Source Q pool was dialyzed against buffer A and supplemented with 50% glycerol. Forty ml of this 120 ml pool was diluted to 500 ml with buffer A,<br><br>
then applied to a 400 ml PEI column (Whatman, Kent, U.K.) which had been pre-equilibrated with buffer A. A wash of 400 ml buffer A was applied followed by a linear gradient from 0.05 M to 1.0 M NaCl in buffer A and fractions of 15 ml were collected. Fractions were assayed using the pNPP assay described above. Phosphatase activity was eluted between 0.14 M and 0.2 M NaCl. The remaining 80 ml of the Source Q pool was similarly applied and eluted; then the gradient fractions containing phosphatase activity were pooled. This pool was dialyzed into storage buffer containing 10 mM Tris (pH 7.4), 1 mM MgCl2, ImM DTT, 50% glycerol.<br>
Example III: Determination of Optimal pH and Salt Conditions for Removal of a Phosphate group by TAP<br>
Using TAP purified as described in Example II, the optimum conditions for removal of a phosphate group from DNA was determined. Dephosphorylation was measured by means of a phosphate release assay as follows: In a 50 ^il reaction, two complementary 40mer oligonucleotides<br>
5'-ACGTATGTTAGGTTAGGTTAGGTTAGGTTAGGTTAGGCTC-3^ (SEQ ID NO:l)<br>
3,-TGCATACAATCCAATCCAATCCAATCCAATCCAATCCGAG-5f (SEQ ID NO:2)<br>
were annealed and end-labeled as recommended by the manufacturer using T4 polynucleotide kinase (New England Biolabs, Inc., Beverly, MA) and gamma 32P-ATP. This radioactive dimer was used to 'spike' 1 mg of a mixture of lambda Hindlll fragments which served as a phosphatase substrate (20 ^g/ml final). TAP activity was determined by release of radioactivity following incubation of the phosphatase with the substrate mix (0.01 pNPP units per reaction in a buffer containing 1 mM MgCfe and 0.1 mM ZnCfe) at 37°C for 5 minutes. Release was measured by precipitation of the DNA substrate with TCA and scintillation counting the radioactivity which was not precipitable. TCA precipitation consisted of adding 100 ^il of Herring Sperm DNA (2 mg/ml) to each reaction as carrier. 150 \il of 20% cold TCA was then added. The samples were vortexed and chilled on ice for 5 minutes. Each reaction was centrifuged in a microflige for 5 minutes at 14,000 X g. 150 \i\ of each supernatant (50%) was added to 2 ml of scintillant and counted for 0.5 minutes.<br><br>
TAP activity was found to be linear over the range of 0.001 to 0.01 units measured by the pNPP colorimetric assay.<br>
A reaction buffer with a pH range of 5.5 to 9.5 was tested using three different buffers: Tris-HCL pH range 6 to 9.5, Bis-Tris Propane buffer pH range 6 to 7 and Bis-Tris Propane buffer pH range 5.5 to 7. The pH range which gave optimum activity on a DNA substrate in Bis-Tris Propane buffer proved to be between 5.5 and 6.0 (Figure 6).<br>
The cation requirement was also tested. It was determined that contrary to Rina, et al., (supra (2000)) which reported that Zn2+ ions were inhibitory in assays on pNPP, the assay described above revealed that the presence of ZnCfe gave a ten-fold increase phosphate release. 1 mM MgCfe, 0.1 mM ZnCfe was subsequently included in TAP activity assays. Addition of other salts including KC1, NaCl, potassium acetate and sodium acetate at concentrations of 100 mM were observed to have no effect on TAP activity (Figure 7).<br>
Example IV: Determination of Thermolability of TAP<br>
A unit of pNPP activity corresponds to the amount of enzyme required to hydrolyze 1 Hmol of pNPP to p-nitrophenol in a reaction volume of 1 ml in 1 minute at room temperature. Ten pNPP units each of TAP, CIAP and SAP phosphatase were mixed with 1 \ig lambda Hindlll fragments in the recommended reaction buffer for each enzyme. The mixture was incubated at 37°C for 10 minutes and then placed on ice. The reactions were then placed in a 65°C water bath with samples removed and placed on ice after 5, 10, 20 and 30 minutes. Following heat treatment the samples were assayed for pNPP activity. Activity remaining was calculated as a percentage of the 0 time point. After 5 minutes the TAP had lost greater than 98% of its activity. In that same time period SAP still had 30% activity remaining and CIAP had almost 90% activity remaining. After 30 minutes SAP still had 20% activity remaining and CIP had 40% remaining showing that TAP is much more heat labile than either SAP or CIAP (Figure 8).<br><br>
Example V: Stability of TAP at 37°C and 25°C<br>
The stability of TAP activity at 37°C and 25°C was measured by release of free phosphate from pNPP in a colorimetric assay. Activity in this assay was determined by the generation of color that occurs when pNPP is converted to p-nitrophenol (pNP). A phosphatase reaction mix was prepared containing 1 M diethanolamine, (pH 8.5), 10 mM MgCk, 10 mM pNPP and 10 units TAP per ml. This mix was aliquoted to 3 ml reaction tubes, 0.1 ml per tube , and the reactions were initiated by placing the tubes at 25°C or 37°C. Reactions were terminated at 5-minute intervals from 0-30 minutes by adding 0.1 ml of 10 N NaOH. Each reaction was diluted with 1 ml of reaction mix (without pNPP or TAP) to bring it to a suitable volume for spectrophotometry at 405 nm. Released phosphate was determined by measuring accumulated pNP and comparison to a known pNP standard. The results were expressed as nmoles phosphate released at either 25°C or 37°C over a 30 minute time course (Figure 9). These results demonstrated that TAP activity is nearly two-fold higher at 37°C than at 25°C and that TAP is stable for at least 30 minutes at either temperature.<br>
Example VI: Optimal conditions for storage of TAP<br>
TAP was incubated at 75°C for 5 minutes to determine optimum conditions for storage. Three different buffers were tested, Bis-Tris Propane buffer pH 6.0, Phosphate buffer pH 7.0 and Tris-HCL buffer pH 7.4, the latter proved the best at stabilizing the TAP. Different concentrations of NaCl were tested with no NaCl proving better than 50, 100 or 200 mM at maintaining enzyme activity. Where EDTA was inhibitory to TAP enzyme activity, both 1 mM DTT and 200 |ag/ml BSA stabilized TAP activity. In a buffer containing Tris-HCL (pH 7.4), 1 mM MgCb, 1 mM DTT, 200 ng/ml BSA and 50% glycerol TAP was found to be stable for over 12 months with no decline in activity.<br>
Example VII: Phosphatase activity<br>
(a) Comparison of the enzyme activity of TAP to SAP in a DNA ligation assay normalized for pNPP activity: Dephosphorylation of 5', 3* and blunt ends.<br><br>
Efficiency is defined as the amount of DNA which can be dephosphorylated by 1 pNPP unit of enzyme based on the dilution which can dephosphorylate 1 ng of DNA.<br>
SAP and TAP were both adjusted to 1 unit per ^il using the pNPP assay. Litmus 28 DNA (New England Biolabs, Inc., Beverly, MA) was digested with either Hindlll (5' overhang), EcoRV (blunt) or PstI (3' overhang). Aliquots of each cut vector DNA (1 mg/50 nl) were treated with several dilutions of each phosphatase for 30 minutes at 37°C in their recommended buffers. TAP was heat killed for 5 minutes at 65°C. SAP was heat killed for 15 minutes at 65°C. Cut and dephosphorylated DNAs were then recircularized using the Quick Ligase Kit (New England Biolabs, Inc., Beverly, MA) according to the instructions. Ligated vectors were then transformed into E. coli and plated on ampicillin plates overnight. Phosphatase activity was considered complete if the number of colonies was less than 5% of the control (vector cut but not dephosphorylated). Whereas 1 pNPP unit of SAP could dephosphorylate 5 pg of 5' overhang DNA, the same number of pNPP units of TAP could dephosphorylate 50 ^g of the same DNA (Figure 10). TAP was therefore shown to be 10 times more efficient at removing phosphate groups from 5' overhangs on DNA. On blunt ends TAP was 50 times more efficient than SAP and on 3' overhangs TAP was 8 times more efficient.<br>
(b) TAP can remove phosphate groups from deoxynucleotides.<br>
The activity of TAP as a deoxynucleotidase was measured by release of free phosphate in a colorimetric assay essentially as described by Heinonen and Lahti {supra). Activity in this assay was determined relative to known phosphate standards. 0.1 ml of four different deoxynucleotide reaction mixes containing 50 mM Bis Tris-Propane (pH 6.0), 1 mM MgCl2, 0.1 mM ZnCl2 and 1 mM Sodium dATP, dCTP, dGTP or TTP were placed into four separate 1.5 ml tubes. 2.5 units of TAP were added to each tube. The reactions were initiated by placing the tubes at 37°C and terminated after 15 minutes by placing the tubes at 65°C for 5 minutes. The entire<br><br>
contents of each reaction mix tube (0.1 ml) was added to a glass tube containing 2.4 ml of measurement solution containing 1.25 N H2SO4, 4 mM Ammonium Molybdate and 50% acetone. The tube was vortexed briefly and incubated at room temperature for 15 minutes. The measurement reactions were stopped by adding citric acid to a final concentration of 0.33 M and the OD390 of each tube was recorded. A standard curve was generated by adding 10 \il each of 0-30 mM phosphate standards to mock reaction tubes which were treated and measured identically to TAP reaction tubes. It was determined that dATP, dCTP, dGTP and TTP were nearly equivalent substrates for TAP; generating 1-2 nmoles of phosphate per minute per unit under these conditions.<br>
The deoxynucleotidase activities of SAP and TAP were compared. dATP was used as the test substrate, but it was concluded that similar results would be obtained using dCTP, dGTP or TTP. A deoxynucleotidase reaction mix was prepared containing 50 mM Bis Tris-Propane (pH 6.0), 1 mM MgCl2, 0.1 mM ZnCl2 and 0.8 mM dATP. 0.1 ml of this mix was aliquoted into seven 1.5 ml reaction tubes. 5 units of either TAP or SAP were added to each tube. The reaction was initiated by placing the tubes at 37°C and terminated at 5-minute intervals for 0-30 minutes by placing the tubes at 65°C for 5 minutes. Reaction products were subjected to capillary high-performance liquid chromatograph (Cap-HPLC) for a comparison to a profile containing standards for dATP, dADP, dAMP and deoxyadenosine. An Agilent 1100 Series Cap-HPLC equipped with an automated sample injector, column heater and a diode array detector was used to perform all analysis. Standards of dATP, dADP, dAMP and dA were purchased from Sigma Chemicals. A 3 urn, 150 x 1 mm C18 reverse-phase Cap-HPLC Develosil® column (Nomura Chemical Co. Ltd., Aichi, Japan) was used to separate the four species. An isocratic separation using 95% 0.1M K2HPO4, pH 6.0 with KOH and 5% acetonitrile at a flow rate of 20 ^l/min at 30°C using a 100 nl flow sensor was found to produce a good resolution of all four species. Typical retention times for the dATP, dADP, dAMP and dA were 4.7, 5.2, 7.0 and 21.6 minutes, respectively. TAP- and SAP-treated samples were diluted to 0.2 ^il in the above buffer and 4 |al were injected per run. Data was collected using Agilent ChemStation Software (Agilent Technologies, Palo Alto, CA) for the 254 nm and 280<br><br>
nm absorbance and the peak sizes were quantitated by the software. The results were expressed for each reaction component as a percentage of the total nucleotide present (Figure 11). These results demonstrated that TAP was capable of functioning as a deoxynucleotidase, removing phosphate groups from dNTP, dNDP or dNMP (where N represents adenosine, cytosine, guanosine or thymidine) and that TAP had a higher specific activity as a deoxynucleotidase than SAP (Heinonen and Lahti (supra)).<br>
(c) TAP can release inorganic phosphate from pyrophosphate.<br>
The activity of TAP as a pyrophosphatase was measured by release of free phosphate in a colorimetric assay essentially as described by Heinonen and Lahti (supra). Activity in this assay was determined relative to known phosphate standards using a known pyrophosphatase as a control. A pyrophosphatase reaction mix was prepared containing 50 mM Bis Tris-Propane (pH 6.0), 1 mM MgCl2&gt; 0.1 mM ZnCl2 and 0.32 mM Sodium pyrophosphate. This mix was aliquoted to 1.5 ml reaction tubes, 0.5 ml per tube and serial 2-fold TAP dilution was performed, starting with 10 units in the first tube. The reaction was intitiated by placing the tubes at 37°C and terminated after 10 minutes by placing the tubes at 65°C for 5 minutes. The phosphate measurement solution was prepared containing 1.25 N H2SO4, 4 mM Ammonium Molybdate and 50% acetone. The entire contents of each reaction mix tube (0.5 ml) was added to a glass tube containing 2 ml of measurement solution, vortexed briefly and incubated at room temperature for 15 minutes. The measurement reactions were stopped by adding citric acid to a final concentration of 0.33 M and the OD390 of each tube was recorded. A standard curve was generated by adding 10 ^1 each of 0-30 mM phosphate standards to mock reaction tubes which were treated and measured identically to TAP reaction tubes. A reaction containing TIPP was run in parallel. The TIPP reaction mix contained 2 units of TIPP, 50 mM Tricine (pH 8.0), 1 mM MgCl2 and 0.32 mM Sodium pyrophosphate; and was incubated at 75°C. This reaction was stopped after 10 minutes by placing the tube at room temperature where TIPP is inactive. Measurement of released phosphate was performed as with TAP, This experiment showed that 10 units of TAP release approximately the same amount of inorganic phosphate as 2 units of TIPP (Figure 12).   These results demonstrate that TAP is<br><br>
capable of functioning as a pyrophosphatase, cleaving pyrophosphate (PPi) into inorganic phosphate (Pi).<br>
(d) TAP removes phosphate groups from phosphorylated peptides.<br>
Preparation of33 P labeled Myelin Basic Protein substrate (MyBP).<br>
Serine/threonine (Ser/Thr) phosphorylated MyBP substrate was prepared using the Protein Serine/Threonine phosphatase (PSP) Assay System kit from New England Biolabs, Inc., Beverly, MA (catalog number #P0780S). Ser/Thr labeled MyBP substrate was diluted to a concentration of 50 ^iM (5X concentrated) with respect to the incorporated 33P.<br>
Tyrosine (Tyr) phosphorylated MyBP substrate was prepared using the Protein Tyrosine phosphatase (PTP) Assay System kit from New England Biolabs, Inc., Beverly, MA (catalog number #P0785S). Tyr labeled MyBP substrate was diluted to a concentration of 25 jaM (5X concentrated) with respect to the incorporated   P.<br>
10 ^1 of either Ser/Thr (50 \xM) or Tyr labeled MyBP protein substrate (25 ^iM) was incubated with serial dilutions of the TAP phosphatase, 10 pNPP units of CIAP phosphatase or no enzyme in the recommended buffer for 1 hour at 37° C. The reactions were terminated by adding 200 \il of cold 20% TCA and incubating on ice for 5-10 minutes. The samples were then centrifuged at 12,000 x g for 5 minutes. 150 |al of the supernatant is carefully removed and added to 2 mis of scintillation fluid and counted in a scintillation counter. The results are shown in Figure 13.<br>
The specific activity for TAP against phosphorylated Ser/Thr residues is 1910 nmol/min/mg which is slightly higher than the specific activity for CIAP against the same substrate. The specific activity for TAP against phosphorylated Tyr residues is also greater than the specific activity for CIAP (Figure 13).<br>
The results of the above experiment the activity of TAP were confirmed using three peptide substrates that each contained a single phospho-amino acid residue. These<br><br>
peptides  were  prepared  using  standard  FMOC-chemistry.   Peptide   1   contained phosphoserine residue, Peptide 2 contained a phosphothreonine residue and Peptide contained a phosphotyrosine residue indicated by pSer, pThr and pTyr, respective!} Listed below are the sequences for each peptide.<br>
1.	H-Val-Pro-Ile-Pro-Gly-Arg-Phe-Asp-Arg-Arg-Val-pSer-Val-Ala-Ala-Glu-NH2   (SE(<br>
ID NO:3)<br>
2.	H-Thr-Ala-Asp-Ser-Gln-His-Ser-pThr-Pro-Pro-Lys-Lys-Lys-Arg-Lys-Val-Glu-OI<br>
(SEQ ID NO:4)<br>
3.	H-Glu-Trp-Met-Arg-Glu-Asn-Ala-Glu-pTyr-Leu-Arg-Val-AIa-OH (SEQ ID NO:5)<br>
Control peptides were also synthesized which had identical sequences but lacked th phosphate in each peptide.<br>
A 50 ul reaction containing 5 ug of each phospho-peptide in IX TAP buffer (se Example VIII) was incubated at 37°C for 10 min with a serial dilution of TAP. Afte incubation Cap-HPLC was used to analyzed the samples. All three peptide de phosphorylation reactions were analyzed using a Vydac 5 urn, CI8 1 x 250 mm column z a flow rate of 25 ^il/min. However an optimized gradient was required to separate th phosphorylated and dephosphorylated peptides for each peptide set. The Cap-HPL( gradient program timetables using an increasing percentage of acetonitrile (%B) relativ to 0.1% TFA in water (v/v) were as follows:<br>
Table 1<br><br><br>
Peptide 2<br><br>
A ratio of 80%A and 20%B , 91.5%A and 8.5%B and 95%A and 5%B was used to equilibrate the column for peptides 1, 2 and 3 respectively. Data was collected using Agilent ChemStation Software (Agilent Technologies, Palo Alto, CA) for the 214 nm, 254 nm and 280 nm absorbances and the peak sizes were calculated by the software. To verify the identity of the species present in the reactions the reactions were also examined on Matrix-assisted laser desorption/ionization time-of-flight mass spectrometer (MALDI-TOF MS). An ABI Voyager DE Mass Spectrometer was used with a-cyano-4-hydroxycinnamic acid as the matrix, an accelerating voltage of 20 kV and a delay time of approximately 150 Dsec in positive ion mode. Figure 14 consists of three graphs of the compiled data showing the decreased amount of the phospho-peptide and increasing amount of dephosphorylated peptide with increasing amounts of TAP. This data shows that TAP is most active on the phospho-tyrosine peptide with only 0.02 units of enzyme being required to remove all the phosphates from 5^g of the peptide in 10 mins. 2.5 units and 10 units of TAP were required to completely remove the phosphates from the phospho-serine and phospho-threonine peptides respectively. This data shows that TAP can be used to remove phosphate groups from peptides.<br><br>
Example VIII: TAP can dephosphorylate sugar phosphates regardless of the position of the phosphate on the sugar.<br>
To test if TAP could cleave the phosphate residue from a sugar phosphate irrespective of position, its activity for three sugar phosphates glucose-1-phosphate, glucose-6-phosphate and mannose-6-phosphate was determined. A dilution series of the reaction mixture was prepared for each sugar phosphate where each reaction mixture contained 100 pi of the 5 mg/ml solution of one of the sugar phosphates, 30 pi of the 10X TAP Buffer (500 mM Bis Tris-Propane, 10 mM MgCl2, 1 mM ZnCl2, pH 6.0)and 210 nl H20. 50 pi of the reaction mix was added to the first tube and 25 pi of the reaction mix to five subsequent tubes. This was repeated for each of the three sugar phosphates to be tested. These tubes were placed on ice and allowed to chill. Once cold 5 pi of the TAP (5 U/pI) was added to the first tube containing 50 pi reaction mix. After mixing 25 |il was removed from this tube and placed in the next tube which contained 25 pi of the reaction mix. After mixing 25 pi was removed from this second tube and placed in the next tube. This 1:1 dilution series was continued for a total of 6 tubes. After a dilution series of the TAP had been prepared for each sugar phosphate, the tubes were removed from the ice and placed at 37°C for 15 minutes. After incubation 3 pi of each sample was spotted in a tight band on silica gel glass backed thin layer chromatography (TLC) plate. Negative controls containing just the mix incubated with no enzyme and positive controls containing 1 mg/ml glucose or mannose were also spotted on the TLC. The spots were completely dried with a hot air gun (temperature not exceeding 70°C). The plate was developed until the solvent front moved 9 cm in isopropanol, ethanol and water (2.5:1:0.5 v:v:v). Following chromatography, the plate was dried and sprayed with 10% perchloric acid. The sugars were visualized by charring the plate with a heat gun. Increased sensitivity was achieved by visualizing the products under UV at 366 nm. Complete digestion of 42 pg of glucose-1-phosphate, glucose-6-phosphate and mannose-6-phosphate was achieved in 15 minutes at 37°C using 6 U of TAP (Figure 15).<br>
The linkage of the phosphate on the sugar (i.e. 1 or 6) had no apparent effect on the activity of the enzyme. In addition the activity of the TAP on mannose sugar phosphates was the same as on glucose sugar phosphates.<br><br><br><br>
WE CLAIM:<br>
1.	A method for overexpressing a thermolabile Antarctic phosphatase (TAP),<br>
comprising<br>
(a)	operably linking to a T7 promoter, a truncated TAP gene fused to a sequence for expressing a hydrophilic oligopeptide;<br>
(b)	transforming a host cell; and<br>
(c)	overexpressing TAP.<br><br>
2.	A method according to claim 1, wherein the TAP gene encodes a TAP protein having an N-terminal and a C-terminal end, wherein the TAP protein is truncated at the N-terminal end, the truncation corresponding to a signal sequence, the N-terminal end having the hydrophilic oligopeptide attached thereto.<br>
3.	A method according to claim 2, wherein the hydrophilic oligopeptide is a peptide with a positive charge.<br>
4.	A method according to claim 3, wherein the peptide with a positive charge is a histidine tag.<br>
5.	A method for obtaining a purified thermolabile Antarctic phosphatase (TAP), comprising:<br><br>
(a)	obtaining a transformed host cell according to claim 1,<br>
(b)	disrupting the host cells to yield a soluble fraction; and<br>
(c)	purifying TAP from the soluble fraction.<br><br>
6.	A method according to claim 5, wherein the step of purifying TAP further comprises a column separation wherein the TAP binds to and is eluted from the column, effecting fractionation.<br>
7.	A method of assaying for thermolabile Antarctic phosphatase (TAP) activity, comprising:<br><br>
(a)	adding TAP to a substrate in a buffer at about pH 5.5-7.0; and<br>
(b)	determining the amount of phosphate removed from the substrate in a defined time and at a defined temperature to determine TAP activity.<br><br>
8.	A method according to claim 7, wherein the buffer includes ZnCfe and MgCb salts.<br>
9.	A method according to claim 7, wherein the buffer is Bis-Tris Propane.<br><br>
10.	A method according to claim 7, wherein the substrate is a nucleic acid.<br>
11.	A method according to claim 10, wherein the substrate is a terminal nucleotide on the nucleic acid,<br>
12.	A method according to claim 7, wherein the substrate is a sugar phosphate.<br>
13.	A method according to claim 7, wherein the substrate is a phosphorylated peptide or protein.<br>
14.	A truncated enzymatically active thermolabile Antarctic phosphatase (TAP),<br>
wherein the truncation corresponds to a deletion of a signal sequence, the phosphatase<br>
having a C-terminal and N-terminal end, wherein the N-terminal end is covalently<br>
linked to a hydrophilic leader sequence.<br>
15.	A phosphatase according to claim 14, wherein the hydrophilic leader sequence is an oligopeptide.<br>
16.	A phosphatase according to claim 15, wherein the oligopeptide is a positively charged oligopeptide.<br>
17.	A phosphatase according to claim 16, wherein the positively charged oligopeptide is a histidine tag.<br>
18.	A phosphatase according to claim 14, wherein the TAP is substantially inactivated at about 65°C and in less than 30 mins.<br>
19.	A phosphatase according to claim 14, wherein the TAP activity is substantially stable in TRIS-HCL at a pH greater than pH 6.<br><br>
20.	A phosphatase according to claim 15, wherein the TAP activity is substantially<br>
stable in TRIS-HCL buffer at about pH 7.4.<br>
21.	A DNA encoding the phosphatase of claim 15, wherein a sequence encoding a<br>
Histidine tag is fused to the 5' end of the TAP DNA sequence.<br>
22.	A vector comprising the DNA of claim 21.<br>
23.	A host cell transformed with the vector of claim 22.<br>
24.	A substantially stable preparation of TAP having a formulation comprising: TRIS-<br>
HCL buffer, MgC12, DTT and glycerol.<br>
25.	A substantially stable preparation of TAP according to claim 19, wherein the pH<br>
is about 7.4.<br>
26.	A method of dephosphorylating a phosphorylated substrate, comprising:<br>
(a)	adding thermolabile Antarctic phosphatase (TAP) to the substrate in a buffer at pH<br>
5.5-7-0; and<br>
(b)	causing an amount of phosphate to be removed from the substrate in a defined<br>
time and at a defined temperature.<br>
27.	A method for overexpressing a thermolabile Antarctic phosphatase (TAP), a<br>
method for obtaining a purified thermolabile Antarctic phosphatase (TAP), a method<br>
of assaying for thermolabile Antarctic phosphatase (TAP) activity, a truncated<br>
enzymatically active thermolabile Antarctic phosphatase (TAP), a DNA encoding, a<br>
vector, a host cell and a method of dephosphorylating a phosphorylated substrate,<br>
substantially as hereindescribed with reference to the given examples and drawings.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk1My1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1953-chenp-2005-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234295-a-method-for-producing-an-aromatic-diamine-derivative.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234297-wet-flue-gas-desulfurization-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234296</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1953/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>24/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>12-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-May-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>18-Aug-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NEW ENGLAND BIOLABS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>240 COUTY ROAD, IPSWICH, MA 01938,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GUTHRIE, ELLEN</td>
											<td>416 LOWELL STREET, ANDOVER, MASSACHUSETTS 01810,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DAVIS, THEODORE</td>
											<td>139 DEPOT ROAD, BOXFORD, MASSACHUSETTS 01921,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BENNER II, JACK</td>
											<td>101 SAGAMORE ROAD, SOUTH HAMILTON, MASSACHUSETTS 1982,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 21/07</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/004996</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/449,711</td>
									<td>2003-02-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234296-a-truncated-enzymatically-active-thermolabile-antarctic-phosphatase-tap by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:43 GMT -->
</html>
