<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234498-method-for-modifying-yield-and-or-height-of-a-plant-comprising-modifying-expression-of-a-nucleic-acid-encoding-map-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234498:&quot;METHOD FOR MODIFYING YIELD AND/OR HEIGHT OF A PLANT COMPRISING MODIFYING EXPRESSION OF A NUCLEIC ACID ENCODING MAP PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR MODIFYING YIELD AND/OR HEIGHT OF A PLANT COMPRISING MODIFYING EXPRESSION OF A NUCLEIC ACID ENCODING MAP PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method for modifying yield, seed yield or plant height relative to corresponding wild type plants, comprising modifying expression in a plant of a nucleic acid encoding a methionine aminopeptidase (MAP protein) represented by SEQ ID NO: 2 or a homologue thereof having at least 35% sequence identity to SEQ ID NO: 2.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD FOR MODIFYING PLANT GROWTH CHARACTERISTICS<br>
The present invention concerns a method for modifying plant growth characteristics. More specifically, the present invention concerns a method for modifying plant growth characteristics by modifying expression of a nucleic acid encoding a methionine aminopeptidase (MAP protein) and/or by modifying level and/or activity of a MAP protein in a plant. The present invention also concerns plants having modified expression of a nucleic acid encoding a MAP protein and/or modified level and/or activity of a MAP protein, which plants have modified growth characteristics relative to corresponding wild type plants.<br>
Given the ever-increasing world population, it remains a major goal of agricultural research to improve the efficiency of agriculture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogeneous genetic components that may not always result in the desirable trait being passed on from parent plants. In contra st, advances in molecular biology have allowed mankind to more precisely manipulate the germplasm of plants. Genetic engineering of plants entails trie isolation and manipulation of genetic material {typically in the form of DNA or RNA) and the subsequent Introduction of that genetic material Into a plant. Such technology has led to the development of plants having various improved economic, agronomic or horticultural traits, for example improved yield.<br>
The ability to influence one or more of the plant growth characteristics, would have many applications in areas such as crop enhancement, plant breeding, production of ornamental plants, arboriculture, horticulture, forestry, production of algae or plants (for use as bioreactors for example, for the production of Pharmaceuticals such as antibodies or vaccines, or for the byconversion of organic waste, or for use as fuel in the case of high-yielding algae and plants).<br>
It has now been found that modifying expression in a plant of a nucleic acid encoding a MAP protein, and/or modifying in a plant level and/or activity of a MAP protein, gives plants having modified growth characteristics.<br>
Accordingly, the present invention provides a method for modifying plant growth characteristics relative to corresponding wild type plants, comprising modifying expression in a plant of a nucleic acid encoding a methionine aminopeptidase (MAP) protein and/or modifying level and/or activity in a plant of a MAP protein.<br>
The term "modifying" as used herein is taken to mean increasing, decreasing and/or changing in place and/or time. Modifying expression of a nucleic acid encoding a MAP protein or modifying level and/or the activity of a MAP protein encompasses altered expression of a gene and/or altered level and/or activity of a gene product, namely a polypeptide, in specific cells or tissues, when compared to expression, level and/or activity of a MAP gene or protein in corresponding wild-type plants. The modified gene expression may result from modified expression of an endogenous MAP gene and/or may result from modified expression of a MAP gene previously introduced into a plant Similarly, modified levels and/or activity of a MAP protein may be due to modified expression of an endogenous MAP nucleic acid or protein and/or due to modified expression of a MAP nucleic acid or protein previously introduced into a plant. Modified expression of a nucleic acid/gene and/or modified level and/or activity of a gene product/protein may be effected, for example, by chemical means and/or recombinant means.<br>
According to a preferred embodiment of the present invention, modifying expression of a nucleic acid encoding a MAP protein and/or modifying level and/or activity of a MAP protein may be effected by recombinant means. Such recombinant means may comprise a direct and/or indirect approach.<br>
For example, an indirect recombinant approach may comprise introducing, into a plant, a nucleic acid capable of modifying expression of a MAP gene and or capable of modifying level and/or activity of a MAP protein. Examples of such nucleic acids to be introduced into a plant include nucleic acids encoding transcription factors or activators or inhibitors that bind to the promoter of a MAP gene or that interact with a MAP protein. Methods to test these types of interactions and methods for isolating nucleic acids encoding such interactors include yeast one-hybrid or yeast two-hybrid screens in which the MAP gene/protein is used as bait. Therefore, modifying expression of a nucleic acid encoding a MAP protein and/or modifying level and/or activity of the MAP protein may be effected by decreased or increased levels of factors that control expression of the MAP gene or that directly or indirectly (in)activate the MAP protein. Further, modifying level and/or activity of a MAP protein may be effected by modifying levels of a factor capable of interacting with a MAP protein. Such factors may include a ligand or a natural target/substrate of a MAP protein. An example of such a target includes the eukaryotic initiation factor elF2, which is part of the protein translation initiation complex. Also encompassed by an indirect recombinant approach for modifying expression of a MAP gene and/or modifying level and/or activity of a MAP protein is the provision of, or the inhibition or stimulation of regulatory elements that drive expression of MAP gene, for example the endogenous MAP gene. For example, such regulatory elements to be introduced Into a plant may be a promoter capable of driving expression of an endogenous MAP gene.<br>
A preferred recombinant approach for modifying expression of a nucleic acid encoding a MAP protein and/or modification of level and/or activity of a MAP protein comprises introducing into a plant a nucleic acid capable of modifying expression of a nucleic acid encoding a MAP protein and/or capable of modifying level and/or activity of a MAP protein.<br>
Accordingly, the present invention provides a method for modifying plant growth characteristics as described above, wherein the modifying expression, level and/or activity is effected by introducing into a plant a nucleic acid capable of modifying expression of a nucleic acid encoding a MAP protein and/or capable of modifying level and/or activity of a MAP protein. According to a more direct and further preferred embodiment of such a method, that nucleic acid is a nucleic acid encoding a MAP protein or a variant thereof as described herein below, or is a variant of a nucleic acid encoding a MAP protein. This nucleic acid encoding a MAP protein may be wild type, i.e. the native or endogenous. Alternatively, the nucleic add may be heterologous, i.e. derived from the same or another species, which nucleic acid is introduced as a transgene. This transgene may ba substantially modified from its native form in composition and/or genomic environment through deliberate human manipulation.<br>
Additionally or alternatively, modifying expression of the nudeic acid encoding a MAP protein and/or modifying level and/or activity of the MAP protein itself may be effected by a chemical approach. Such a chemical approach may involve exogenous application of one or more compounds or elements capable of modifying expression of a MAP nucleic acid (endogenous gene or introduced into the plant) and/or capable of modifying level and/or activity of a MAP protein (endogenous or introduced into the plant). The term "exogenous application" as defined herein is taken to mean the contacting or administering of a suitable compound or element to a plant, plant cell, tissue or organ. The compound or element may be exogenously applied to a plant in a form suitable for plant uptake (such as through application to the soil for uptake via the roots, or in the case of some plants, by applying directly to the leaves, for example by spraying). The exogenous application may take place on wild-type plants or on transgenic plants that have previously been transformed with a MAP nuclei c acid/gene or other transgene. Suitable compounds or elements for exogenous application include MAP proteins or MAP nucleic acids. Alternatively, suitable compounds or elements include those capable of directly or indirectly binding or (in)activating a MAP protein. Suitable compounds also include antibodies that can recognise or mimic the function of a MAP protein. Such antibodies may comprise "plantibodies", single chain antibodies, IgG antibodies and heavy chain camel antibodies, as well as fragments thereof. Other suitable compounds or elements for chemical modification of expression, activity and/or level of a MAP gene or protein, include mutagenic substances, such as N-nitroso-N-ethylurea, ethylene imine, ethyl methanesulphonate and diethyl sulphate. Mutagenesis may also be achieved by exposure to ionising radiation, such as X-rays or gamma-rays or ultraviolet light. Methods for introducing mutations, and for testing the effect of mutations (such as by monitoring gene expression and/or protein activity), are well known in the art.<br>
Therefore, according to one aspect of the present invention, there is provided a method for modifying plant growth characteristics, comprising exogenous application of one or more compounds or elements capable of modifying expression of a MAP gene and/or capable of modifying level and/or activity of a MAP protein.<br>
Methbnine aminopeptidases (MetAP or MAP) are described in literature to be responsible for removal of (the initiator methionine residue of a peptide during protein synthesis (Bradshaw and Yi, 2002, Essays Biochem 38:65-78). Methionine aminopeptidases are specific and ubiquitous enzymes that belong to the family of metalloenzymes. It has been found that Eukaryotes have two classes of methionine aminopeptidase (MAPI and MAP2), while prokaryotes only have one. MAP2 is also known as the eukaryotic initiation factor 2 alpha (elF2alpha) associated protein p67. It has been demonstrated that rat p67 in addition to its peptidase function, also plays an important role in translational regulation by preventing the phosphorylation of the alpha subunit of initiation factor-2. Accordingly, MAP2 proteins have, In addition to their peptidase activity, a non-proteolysis function to protect elF2alpha against phosporylation (POEP), which elF2alpha is inactive in the phosphorylated state (Data R et al. Bonhomie. 2001 83:919-31). MAP proteins from various organisms have been studied with respect to their function as well as their structure. Based on sequence analysis, MAP proteins have been found to have a MAP signature and a peptidase domain. For example, typical structural features of a MAPI protein are a MAPI signature (PROSITE PS00680 =[MFY]-x-G-H-G-[LIVMCHGSH]-x{3)-H-x(4HLiVM]-x-[HN]-fYWVH]) and a pFAM PEPTIOASE_M24 domain. Typical structural features of a MAP2 protein include a MAP2 signature (PROSITE PS01202 =[DA]-[LIVMY]-x-K-[LIVM]-D-x-G-x-IHQ]-[LIVM]-pNS]-G-x(3)-[DN]) and a pFAM PEPTIDASE_M24 domain. MAP2 proteins, such as plant MAP2 proteins, additionally comprise<br><br>
at least one lysine-rich domain at the N-terminus. MAP proteins isolated from yeast have been described in Li and Chang, (1995) Proc Natl Acad Sci USA. 92(26): 12357-61, Human MAP proteins have been described in Li and Chang (1998) Biochem Biophys Res Commun 227:152-159 and six MAP cDNAs were cloned from Arabidopsis thaliana and corresponding proteins were characterized in vivo and in vitro (Giglione et al. EMBO J. 2000 19:5916-29). One example of an Arabidopsis thaliana MAP protein is herein represented by SEQ ID NO 2, and its encoding sequence by SEQ ID NO 1. Another example of an Arabidopsis thaliana MAP protein is herein represented by SEQ ID NO 4, and its encoding sequence by SEQ ID NO 3.<br>
The term "MAP protein" as used herein encompasses a methionine aminopeptidases (MAP), for example a MAP of SEQ ID NO 2 or 4, as well as a variant thereof (or a proteins essentially similar thereto). The terms "MAP gene" or "MAP nucleic acid" or "nucleic acid encoding a MAP protein" are used herein interchangeably and also encompass variant MAP nucleic acids, for example variants of SEQ ID NO 1 or 3. The terms "a variant of and "essentially similar to" are used interchangeably herein. Variant MAP proteins or variant nucleic acids encoding a MAP protein include:<br>
(i)       Functional portions of a MAP nucleic acid, for example a MAP nucleic acid of SEQ<br>
IDNO1 or 3; (ii)       Sequences capable of hybridising with a MAP nucleic acid, for example with a MAP<br>
nucleic acid of SEQ ID N01 or 3; (iii)      Alternative splice variants of a MAP nucleic acid, for example a MAP nucleic acid of<br>
SEQ ID N01 or 3; (iv)      Allelic variants of a MAP nucleic acid, for example a MAP nucleic acid of SEQ ID<br>
NO 1 or 3; and<br>
(v)       Homologues, derivatives and active fragments of a MAP protein , for example a MAP protein of SEQ ID NO 2 or 4.<br>
Advantageously, the methods according to the invention may be practised using variant MAP proteins and variant MAP nucleic acids. Suitable variants include variants of SEQ ID NO 2 or 4 and/or variants of SEQ ID NO 1 or 3. However, it should be clear that the applicability of the invention is not limited to the use of the nucleic acid represented by SEQ ID NO 1 or 3, nor to the nucleic acid encoding the amino acid sequence represented by SEQ ID NO 2 or 4, but that other nucleic acids encoding variants of SEQ ID NO 2 may be useful in the methods of the present invention.<br>
For use in the methods according to the present invention, the MAP protein preferably comprises any one or both of the following domains: a MAP signature<br>
a)	a peptidase_M24 domain<br>
A preferred MAP protein comprises both of these domains in the above rank order.<br>
The term "variant" also includes variants in the form of a complement, DNA, RNA, cDNA or genomic DNA. The variant nucleic acid may be synthesized in whole or in part, it may be a double-stranded nucleic acid or a single-stranded nucleic acid. Also, the term Variant" encompasses a variant due to the degeneracy of the genetic code; a family member of the gene or protein; and variants that are interrupted by one or more intervening sequences such as introns or transposons.<br>
One variant nucleic acid encoding a MAP protein is a functional portion of a nucleic acid encoding a MAP protein. Advantageously, the method of the present invention may also be practised using portions of a nucleic acid encoding a MAP protein. A functional portion refers to a piece of DNA derived from or prepared from an original (larger) DNA molecule, which portion, retains at least part of the functionality of the original, DNA and which, when expressed in a plant, gives plants having modified growth characteristics. The portion may comprise many genes, with or without additional control elements or may contain spacer sequences. The portion may be made by making one or more deletions and/or truncations to the nucleic acid. Techniques for introducing truncations and deletions into a nucleic acid are well known in the art. Portions suitable for use in the methods according to the invention may readily be determined by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the portion to be tested for functionality.<br>
Another variant of a nucleic acid encoding a MAP protein is a nucleic acid capable of hybridising with a nucleic acid encoding a MAP protein, for example to any of the nucleic acids encoding a protein represented by SEQ ID NO 2 or 4. Hybridising sequences suitable for use in the methods according to the invention may readily be determined, for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the hybridising sequence.<br>
The term "hybridising" as used herein means annealing to a substantially homologous complementary nucleotide sequences in a hybridization process. The hybridisation process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, nuclease S1 mapping, primer extension, reverse transcription, cDNA synthesis, differential display of RNAs, and DNA sequence determination. The hybridisation process can also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridisation process can furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-cellulose or nylon membrane or immobilised by e.g. photolithography to e.g. a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process include RNA and DNA gel blot analysis, colony hybridisation, plaque hybridisation, in situ hybridisation and microarray hybridisation. In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, sodium/salt concentration and hybridisation buffer composition. High stringency conditions for hybridisation include high temperature and/or low salt concentration (salts include NaCI and Na3-citrale) and/or the inclusion of formamide in the hybridisation buffer and/or lowering the concentration of compounds such as SOS (sodium dodecyl sulphate detergent) in the hybridisation buffer and/or exclusion of compounds such as dextran sulphate or polyethylene glycol (promoting molecular crowding) from the hybridisation buffer. Conventional hybridisation conditions are described in, for example, Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York, but the skilled craftsman will appreciate that numerous different hybridisation conditions can be designed in function of the known or the expected homology and/or length of the nucleic acid. Sufficiently low stringency hybridisation conditions are particularly preferred (at least in the first instance) to isolate nucleic acids heterologous to the DNA sequences of the invention defined supra. An example of low stringency conditions is 4-6x SSC / 0.1-0.5% w/v SDS at 37-45"C for 2-3 hours. Depending on the source and concentration of the nucleic acid involved in the hybridisation, alternative conditions of stringency may be employed, such as medium stringency conditions. Examples of medium stringency conditions include 1-4x SSC / 0.25% w/v SDS at fe 45°C for 2-3 hours. With specifically hybridising is meant hybridising under stringent conditions. An example of high stringency conditions includes 0.1-2XSSC, 0.1XSDS, and 1X SSC, 0.1X SDS at 60°C for 2-3 hours.<br>
The methods according to the present invention may also be practised using an alternative splice variant of a nucleic acid encoding a MAP protein, for example an alternative splice variant of SEQ ID NO 1 or 3. The term "alternative splice variant" as used herein encompasses<br>
variants of a nucleic acid in which selected introns and/or exons have been excised, replaced or added. Such splice variants may be found in nature or may be manmade. Methods for making such splice variants are well known in the art. Splice variants suitable for use in the methods according to the invention may readily be determined, for example, by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the splice variant.<br>
Another variant MAP nucleic acid useful in practising the method for modifying plant growth characteristics, is an allelic variant of a MAP gene, for example an allelic variant of SEQ ID NO 1 or 3. Allelic variants exist in nature and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants also encompass Single Nucleotide Polymorphisms (SNPs) as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms. Allelic variants suitable for use in the methods according to the invention may readily be determined for example by following the methods described in the Examples section by simply substituting the sequence used in the actual Example with the allelic variant.<br>
The present invention provides a mathod for modifying plant growth characteristics, comprising modifying expression in a plant of an alternative splice variant or of an allelic variant of a nucleic acid encoding a MAP protein and/or by modifying level and/or activity in a plant of a MAP protein encoded by an alternative splice variant or allelic variant.<br>
One example of a variant MAP protein useful in practising the methods of the present invention is a homologue of a MAP protein. "Homologues" of a particular MAP protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having an amino acid substitution, deletion and/or insertion relative to that particular MAP protein and having similar biological and functional activity as a MAP protein. Homologues of a MAP proteins may be manmade via the techniques of genetic engineering and /or protein engineering. To produce such homologues, amino acids of the protein may be replaced by other amino acids having similar properties (such as similar hydrophobicity, hydrophilicity, antigenicity, propensity to form or break ex-helical structures or p-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company).<br>
Additionally and/or alternatively, homologues of a particular MAP protein exist in nature and may be found in the same or different species or organism from which the particular MAP<br>
protein is derived. Two special forms of homologues, orthologues and paralogues, are evolutionary concepts used to describe ancestral relationships of genes. The term "orthologues" relates to homologous genes in different organisms due to ancestral relationship. The term "paralogues" relates to gene-duplications within the genome of a species leading to 5 paralogous genes. The term "homologues" of a MAP protein as used herein therefore encompasses paralogues and orthologues of the MAP protein, which are also useful for practising the methods of the present invention.<br>
The homologues useful in the method according to the invention have, in increasing order of<br>
    preference, at least 20%. 21 %, 22%, 23%, 24%, 25%, 26%, 27%, 28%, 29%, 30%, 31 %, 32%,<br>
33%, 34%, 35%, 36%, 37%, 38%, 39%, 40%, 41%, 42%, 43%, 44%, 45%, 46%, 47%, 48%,<br>
49%, 50%, 51%, 52%, 53%, 54%, 55%, 56%, 57%, 58%, 59%, 60%, 61%, 62%, 63%, 64%,<br>
65%, 66%, 67%, 68%, 69%, 70%, 71%, 72%, 73%, 74%, 75%, 76%, 77%, 78%, 79%, 80%,<br>
81%, 82%, 83%, 84%. 85%, 86%, 87%, 88%, 89%, 90%, 91%, 92%. 93%, 94%, 95%, 96%,<br>
    97%, 98% or 99% sequence identity with the sequence of SEQ ID NO 2 or with the sequence<br>
of SEQ ID NO 4, preferably over the full length of SEQ ID NO 2 or 4..<br>
The percentage of sequence identity is calculated using a pairwise global alignment program implementing the algorithm of Needleman-Wunsch (J. Mol. Biol. 48: 443-453, 1970) which 20 maximizes the number of matches and minimizes the number of gaps. For calculation of the above-mentioned percentages, the program Align X (as part of the Vector NTI suite 5.5) is used with the standard parameters and the variable parameters'gap opening penalty 10 and gap extension penalty 0.1.<br><br><br>
Multiple alignment of MAP proteins from various sources is generated using the program Clustal X with the standard fixed parameters and the variable parameters Gap opening penalty 10 and Gap extension penally 0.2. Percentage of identity between the proteins of this multiple alignment was calculated using the BoxShade software (available at http://www.isrec.isb-sib.ch/ftp-sen/er/boxshadeA. Table I gives an overview of these percentages.<br><br>
(Table Remove)<br><br>
The homologues useful in the methods according to the invention may be derived (either directly or indirectly, i.e. if subsequently modified) from any source provided that the sequence, when expressed in a plant, leads to modified growth characteristics. The nucleic acid may be isolated from a microbia) source, such as bacteria, yeast or fungi, or from a plant, algae, insect or animal (including human) source. .<br>
The nucleic acid encoding a MAP homologue is preferably isolated from a plant. Further preferably, the nucleic acid is isolated from a dicotyledoneous plant, preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana. MAP proteins of Arabidopsis have been subdivided into different classes on the basis of their level of sequence homology (Giglione et el. (2000, EMBO J. 19: p 5916-5929). Class MAPI (with A, B, C and D isoforms) and class MAP2 (with A and B isoforms) were identified in Arabidopsis. These classes and isoforms are also encompassed by the term "homologue" as used herein. Advantageously, these different classes or isoforms of MAP proteins, or their encoding nucleic acids, may be used in the methods of the present inventbn. Accordingly, the present invention provides a method as described hereinabove, wherein the MAP nucleic acid or MAP protein is obtained from a plant, preferably from a dicotyledoneous plant, further preferably from the family Brassicaceae, more preferably from Arabidopsis thaliana. According to a further embodiment, a MAP nucleic acid or protein is a MAP2 nucleic acid or protein or a MAPI nucleic acid or protein. According to a further embodiment of the invention, the MAP nucleic acid encodes or the MAP protein is, an isoform of the MAP protein represented by SEQ ID NO 2 or 4. According to a further embodiment of the invention, the MAP nucleic acid encodes or the MAP protein is the Arabidopsis thaliana MAP2B protein represented by SEQ ID NO 2.<br>
Other preferred MAP homologues and their encoding sequences may be found in (public) sequence databases. Methods for the search and identification of MAP homologues in sequence databases would be well within the realm of persons skilled in the art. Such methods, involve screening sequence databases with the sequences provided by the present invention, for example SEQ ID NO 2 or 4 (or SEQ ID NO 1 or 3), preferably in a computer readable form. Useful sequence databases include, but are not limited, to Genbank (http://www.ncbi.nlm.nih.gov/web/Genbank), the European Molecular Biology Laboratory Nucleic acid Database (EMBL) (http:/w.ebi.ac.uk/ebi-docs/embl-db.html) or versions thereof or the MIPS database (http://mips.gsf.de/). Different search algorithms and software for the alignment and comparison of sequences are well known in the art. Such software includes for example GAP, BESTFIT, BLAST, FASTA and TFASTA. Preferably the BLAST software is used, which calculates percent sequence identity and performs a statistical analysis of the<br>
similarity between the sequences. The suite of programs referred to as BLAST programs has 5 different implementations: three designed for nucleotide sequence queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology: 76-80,1994; Birren et al., GenomeAnalysis, 1:543,1997). The software for performing BLAST analysis is publicly available through the National Centre for Biotechnology Information.<br>
Orthologues of a MAP protein in other plant species may easily be found by performing a reciprocal Blast search. This method encompasses searching one or more sequence databases with a query gene or protein of interest (for example SEQ ID NO 1,2, 3 or 4), using for example the BLAST program. The highest-ranking subject genes that result from the search are then used as a query sequence in a similar BLAST search. Only those genes that have as a highest match again the original query sequence (SEQ ID NO 1, 2, 3 or 4) are considered to be orthologous genes. For example, to find a rice orthologue of an Arabidopsis thaliana gene, one may perform a BLASTN or TBLASTX analysis on a rice database such as (but not limited to) the Oryza saliva Nipponbare database available at the NCBI website (http://www.ncbi.nlm.nih.gov) or the genomic sequences of rice (cuftivars /no7ca orjaponica). In a next step, the obtained rice sequences are used in a reverse BLAST analysis using an Arabidopsis database. The method can be used to identify orthologues from many different species, for example from com.<br>
Paralogues of a MAP protein in the same species may easily be found by performing a Blast search on sequences of the same species from which the MAP protein is derived. From the sequences that are selected by the Blast search, the true paralogues may be identified by looking for identity between the sequences or for conservation of typical MAP domains. A search for paralogues of the Arabidopsis thaliana MAP protein has been carried out by Giglione etal. (2000, EMBO J. 19: p 5916-5929).<br>
A BLAST using default parameters and using SEQ ID NO 2 or SEQ ID NO 1 as search sequence was performed and resulted in the identification of the following MAP nucleic acids and proteins. Sequences substantially identical to SEQ ID NO 1 (AtMAP2B) have been published under Genbank accession numbers NM-115862 (genomic DNA), BT000063 (At3g59990 mRNA), AY084710, AY065161 and AF300880. One isoform of SEQ ID NO 1 and 2, named herein AIMAP2A, was found in the database under Genbank accession number AF250964. Isoforms of a different class were found in the database under Genbank accession numbers AF250960 ((A4MAP1A and represented herein by SEQ ID NO 3 and SEQ ID NO 4), AF250961 (A1MAP1B), AF250962 (AtMAPIC) and AF250963 (AtMAPID). Also MAP homologues from different plant species have been identified, which homologues are useful in the methods of the present invention. There is a high degree of conservation among the MAP proteins of plants (see Table I). Such homologues include for example, the MAP protein from Qryza sativa as published in Genbank database under accession number BAD03108, the O/yza sativa protein under accession number AK122063, the Oryza sativa protein under Genbank accession number AK107616 and the Zea Mays protein under Genbank accession number AY105027. The genes encoding MAP homologues of crop plants may be especially useful in practising the methods of the invention in crop plants. In another embodiment of the invention, the genes encoding MAP homologues of a dicot plant may be used to practise the methods of the present invention in a monocot plant, or vice versa.<br>
The genome sequences of Arabidopsis thaliana and Oryza sativa are now available in public databases such as Genbank and other genomes are currently being sequenced. Therefore, it is expected that further homologues will readily be identifiable by sequence alignment with SEQ ID NO 1 or 3 or with SEQ ID NO 2 or 4 using the programs Blast X or BlastP or other programs. *<br>
A phylogenetic tree may be constructed with all the homologues, paralogues and ortho!ogues as defined hereinabove. Multiple alignments are made using the program ClustalX as described hereinabove. The phylogenetic tree is made by the Phylip software package available at http://evolution.qenetics.washington.edu/phvlip.html. Sequences clustering around one or more of the 6 MAP proteins of Arabidopsis thaliana identify proteins, and their corresponding genes, suitable for use in the methods of the present invention.<br>
The above-mentioned software analyses for comparing sequences, for the calculation of sequence identity, for the search for homologues, orthologues or paralogues or for the making of a phylogenetic tree, is preferentially done with full-length sequences. Alternatively, these software analyses may be carried out on with a conserved region of the MAP protein or DMA sequence. Accordingly, these analyses may be based on the comparison and calculation of sequence identity between conserved regions, functional domains, motifs or boxes.<br>
The identification of such domains or motifs, for example, the domains represented by SEQ ID NO 5 (MAPI signature), SEQ ID NO 6 (MAP2 signature), SEQ ID NO 7 (peptidase_M24 domain of AIMAP1A), SEQ ID NO 8 (peptidase_M24 domain of AtMAP2B), SEQ ID NO 9 (lysine-rich domain of AIMAP2B), would also be well within the realm of a person skilled in the art and involves screening a computer readable format of MAP proteins for the presence of conserved protfcin domains, motifs and boxes. Protein domain information is available in the PRODOM (http://www.biochem.ucl.ac.uk/bsm/dbbrowser/jj/prodomsrchjj.htrnl), PlR (http://pir.geofqeiown.edu/). PROSITE (http://au.expasy.org/PROSITE/) or pFAM (http://pFAM .wustl.edu/) database. Software programs designed for such domain searching include, but are not limited to, MotifScan, MEME, SIGNALSCAN, and GENESCAN. MotifScan is a preferred software program and is available at (http://hits.isb-sib.ch/cgi-bin/PFSGAN, which program uses the protein domain information of PROSITE and pFAM . A MEME algorithm (Version 3.0) may be found in the GCG package; or at http://www.sdsc.edu/MEME/meme. SIGNALSCAN version 4.0 information is available at http://biosci.cbs.umn.edu/software/sigscan.html. GENESCAN may be found at http://gnomic.stanford.edu/GENESCANW.html.<br>
The term "MAP signature" means a MAP-specific domain. Examples of such MAP signatures are MAPI or MAP2 signatures. The MAPI signature is described in the PROSITE database under accession number PS00680 and is herein represented by the consensus sequence of SEQ ID NO 5: [MFY]-x-G-H-G-tLIVMC]-tGSH]-x(3)-H-x(4HLIVM]-x-[HN]-[YWVH], In MAPI proteins, the MAPI signature may be located within the peptidase domain. The MAP2 signature is further described in the PROSITE database under accession number PS01202 and is herein represented by the consensus sequences of SE Q ID NO 6: [DA]-[LIVMY]-x-K-[LIVM]-D-x-G-x-[HQ]-[LIVMHDNS]-G-x(3HDN]. In case of MAP2 proteins, the MAP2 signature is preferably located upstream of the peptidase domain. A person skilled in the art will recognize that a MAP signature may deviate, with for example 1 or 2 mismatches, from the above-mentioned consensus MAP signatures, without loosing its functionality. One example is found in the Drosophila MAP2 protein (see Figure 3), having a K at the twelfth position of its MAP2 signature, instead of D, N, or S<br>
The term "peptidase_M24 domain", as used herein, refers to a peptidase domain, which occurs in MAP proteins. The peptidase_M24 domain is described in the pFAM database under accession number PF00557. A consensus sequence for this domain is not given in the pFAM database, but nearly 500 proteins were categori/ed as having a PEPTIDASE_M24 domain. This domain may be identified by its folding and tertiary structure, rather than by its primary structure, which may be variable. Different MAP proteins therefore also exhibit substantial variation in the primary structure (amino acid sequence) of this peptidase_M24 domain (see Figure 3). A person skilled in the art would readily know how to determine the presence of a peptidase_M24 domain in a protein sequence. One example is to submit the protein sequence to a software program capable of determining conserved domains, for example the program MotifScan as described hereinbefore. One example of a peptidasejv124 domain is given in SEQ ID NO 7, which is the peptidase_M24 domain of AtMAPIA. Another example is given in SEQ ID NO 8, which is the peptidase_M24 domain of AtMAP2B. Preferably, the MAP protein used in the methods of the present invention has a peptidase_M24 domain, which is at least 70% identical to SEQ ID NO 7 or to SEQ ID NO 8.<br>
Optionally, for example the case of MAP2 proteins, the MAP protein useful in the methods of the present invention has at least one lysine-rich domain. Preferably such lysine-rich domain is located between the N-terminus and the MAP signature. The term "lysine-rich region" means an amino acid region, which is enriched with lysine amino acids. Typically a lysine-rich domain is an amino acid sequence of which more than 50% of the amino acids are lysine (K). For example, in AIMAP2B, 12 out of 14 continuous residues of the lysine-rich domain are lysine, which corresponds to 85% of lysine residues. Optionally a stretch of continuous lysines may be present, for example a stretch of at least 3 lysine residues, for example of 4, 5, 6 or more lysine residues. Accordingly, substantial variation between the lysine-rich domains of different MAP proteins exists, as is illustrated in Figure 3. The lysine-rich domain of the Arabidopsis thaliana MAP2B protein is represented by SEQ ID NO 9.<br>
Based on the presence and conservation of the above-mentioned structural domains, persons skilled in the art have been able to readily recognize MAP proteins of different organism, for example from plants (see Giglione et al., (2000), EMBO J. 19: p 5916-5929).<br>
Some of the variants as mentioned hereinabove may occur in nature. Once the sequence of a variant is known, and its corresponding coding sequence, the person skilled in the art will be able to isolate the corresponding MAP gene or variant from biological material, for example by the technique of PCR. One example of such an experiment is outlined in Example 1.<br>
Alternatively and/or additionally, the variants as mentioned above may be manmade via techniques involving, for example, mutation (substitution, insertion or deletion) or derivation. These variants are herein referred to as "derivatives", which derivatives are also useful in the methods of the present invention. Derivatives of a protein may readily be made using peptide synthesis techniques well known in the art, such as solid phase peptide synthesis and the like, or by recombinant DNA manipulations. The manipulation of DNA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA are well known to those skilled in the art and Include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols. One example of a derivative is a substitutional variant. The term "substitutional variants" of a protein refers to those variants in which at (oast one residue in an amino acid sequence has been removed and a different amino acid inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions usually are of the order of about 1-10 amino acids, and deletions can range from about 1-20 amino acids. Preferably, amino acid substitutions comprise conservative amino acid substitutions.<br>
Other derivatives are "insertional variants" in which one or more amino acids are introduced into a predetermined site in the MAP protein. Insertions may comprise amino-terminal and/or carboxy-terminal fusion as well as intra-sequence insertion of single or multiple amino acids. Generally, insertions within the amino acid sequence will are of the order of about 1 to 10 amino acids. Examples of amino- or carboxy-terminal fusions include fusion of the binding domain or activation domain of a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins, (histidine)6-tag, glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofoiate reductase, Tag»100 epitope, c-myc epitope, FLAGs-epitope, lacZ, CMP (calmodulin-binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
Other derivatives of a MAP protein are "deletion variants", characterised by the removal of one or more amino acids from the protein.<br>
Another "derivative" of a MAP protein is characterised by substitutions, and/or deletions and/or additions of naturally and non-naturally occurring amino acids compared to the amino acids of a naturally-occurring MAP protein. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived. Such non-amino acid substituents include for example non-naturally occurring amino acids, a reporter molecule or other ligand, covalently or non-covalently bound to the amino acid sequence. Such as reporter molecule may be bound to facilitate the detection of the MAP protein.<br>
Another variant of a MAP protein useful in the methods of the present invention is an active fragment of a MAP protein. "Active fragments" of a MAP protein encompass at least five contiguous amino acid residues of a MAP protein, which residues retain similar biological and/or functional activity to a naturally occurring protein or a part thereof. Suitable fragments include fragments of a MAP protein starting at the second or third or further internal methionine residues. These fragments originate from protein translation, starting at internal ATG codons. Functional fragments of a MAP protein useful in practising the methods of the present invention may have one or more of the conserved domains of a MAP proteins, whilst retaining its functionality in the methods of the present invention.<br>
According to a preferred embodiment of the present invention, a method for modifying plant growth characteristics, comprises enhanced or increased expression of a nucleic acid encoding a MAP protein. Methods for obtaining enhanced or increased expression of genes or gene products are well documented in the art and include, for example, overexpression driven by a (strong) promoter, the use of transcription enhancers or translation enhancers. The term overexpression as used herein means any form of expression that is additional to the original wild-type expression level. Preferably the nucleic acid to be introduced into the plant and/or the nucleic acid that is to be overexpressed in the plant is in the sense direction with respect to the promoter to which it is operably linked. Preferably, in the methods of the present invention a nucleic acid encoding a MAP protein is overexpressed in a plant, such as a MAP nucleic acid of SEQ ID NO 1 or a variant thereof, such as a portion of SEQ ID NO 1 or a sequence capable of hybridising therewith. However, it should be clear that the applicability of the inventbn is not limited to use of the nucleic acid represented by SEQ ID NO 1 nor to the nucleic acid encoding the amino acid sequence of SEQ ID NO 2, but that other nucleic acids encoding homologues, derivatives or active fragments of SED ID NO 2 may be useful in the methods of the present invention.<br>
Alternatively and/or additionally, increased expression of a MAP gene or increased level, and/or activity of a MAP protein in a plant cell, is achieved by mutagenesis. For example these mutations may be responsible for altered control of the MAP gene, resulting in more expressbn of the gene, relative to the wild-type gene. Mutations can also cause conformational changes in a protein, resulting in higher levels and/or more activity of the MAP protein. Such mutations or such mutant genes may be selected, or isolated and/or introduced into the same or different plant species in order to obtain plants having modified growth characteristics. Examples of such mutants include dominant positive mutants of a MAP gene.<br>
Modifying gene expression (whether by a direct or indirect approach) encompasses altered transcript levels of a gene. Altered transcript levels can be sufficient to induce certain phenotypic effects, for example via the mechanism of co-suppression. Here the overall effect of overexpression of a transgene is that there is less activity in the cell of the protein encoded by a native gene having homology to the introduced transgene. Therefore, according to another embodiment of the present invention, there is provided a method for modifying plant growth characteristics, comprising inhibiting or decreasing expression of a gene encoding a MAP protein or decreasing level and/or activity of a MAP protein. Examples of decreasing expression, level and/or activity of a protein are also well documented in the art and include, for example, downregulation of expression by anti-sense techniques, RNAi techniques, small interference RNAs (SIRNAs) and microRNA (miRNA).<br>
Another method for downregulation of gene expression or gene silencing comprises use of ribozymes, for example as described in Atkins et a/. 1994 (WO 94/00012), Lenee era/. 1995 (WO 95/03404), Lutziger atal. 2000 (WO 00/00619), Prinsen era/. 1997 (WO 97/3865) and Scott ef a/. 1997 (WO 97/38116).<br>
Gene silencing may also be achieved by insertion mutagenesis (for example, T-DNA insertion or transposon insertion) or by gene silencing strategies as described by, among others, Angell and Baulcombe 1998 (WO 98/36083), Lowe et al. 1989 (WO 98/53083), Lederer et al. 1999 (WO 99/15682) or Wang ef al. 1999 (WO 99/53050).<br>
Expression of an endogenous MAP gene may also be reduced by mutation. Such a mutation or such a mutant gene may be isolated and introduced into the same or different plant species in order to obtain plants having modified growth characteristics. Examples of such mutants include dominant negative mutants of a MAP gene.<br>
Genetic constructs aimed at silencing gene expression may comprise the nucleotide sequence encoding a MAP protein, for example a nucleic acid represented by SEQ ID NO 1 (or a variant thereof), in a sense and/or anti-sense orientation relative to the promoter sequence. The sense and/or anti-sense copies of at least part of the endogenous gene in the form of direct or inverted repeats or in the form of a hairpin may be utilised in the methods according to the invention. The growth of plants may also be modified by introducing into a plant at least part of an anti-sense version of a nucleotide sequence encoding a MAP protein.<br>
According to a further aspect of the present invention, there is provided genetic constructs and vectors to facilitate introduction into a plant cell and/or facilitate expression and/or facilitate maintenance of the nucleotide sequence capable of modifying expression of a nucleic acid encoding a MAP protein and/or capable of modifying level and/or activity of a MAP protein. Therefore, according to a further aspect of the present invention, there is provided a genetic construct comprising:<br>
(a)	A nucleic acid encoding a plant MAP protein or a variant thereof or a variant of a<br>
nucleic acid encoding a plant MAP protein;<br>
(b)	One or more control sequences capable of driving expression of the nucleic acid of in a<br>
plant (a); and optionally<br>
(c)	 A transcription termination sequence.<br>
Constructs useful in the methods according to the present invention may be constructed using recombinant DMA technology well known to persons skilled in the art. The genetic constructs may be inserted into vectors, which may be commercially available, suitable for transformation into plants and suitable for maintenance and expression of a MAP gene in the transformed cells. Preferably the genetic construct is a plant expression vector.<br>
The nucleic acid according to (a) is advantageously any of the nucleic acids described hereinbefore. A preferred nucleic acid is a nucleic acid represented by SEQ ID NO 1 or 3, or a variant thereof as hereinbefore defined, or is a nucleic acid encoding a protein represented by SEQ ID NO 2 or 4, or a variant thereof as hereinbefore defined.<br>
The terms "regulatory element? and "control sequence", are used herein interchangeably and are taken in a broad context refer to regulatory nucleic acid sequences capable of driving and/or regulating expression of the sequences to which they are ligated and/or operably linked. Preferably, the control sequence of (b) is operable in a plant, most preferably the control sequence is a derived from a plant sequence. Encompassed by the terms "control sequence" are promoters. A "promoter" encompasses transcriptional regulatory sequences derived from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) arid additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers), which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included within the term is a transcriptional regulatory sequence of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. The term "regulatory element" also encompasses a synthetic fusion molecule or derivative, which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ. The term "operably linked" as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiate transcription of the gene of interest. Preferably, the promoter is operably linked to the gene of interest in a sense orientation.<br>
Advantageously, any type of promoter may be used in the methods of the present invention. For example, a meristem-specific promoter, such as the RNR (rlbonucleotide reductase), cdc2a promoter and the cyc07 promoter; or a seed-specific promoter, such as 2S2 albumin, prolamin, or oleosin promoter. A promoter may be selected in order to increase expression of a MAP protein at the moment of germination. Alternatively, a promoter expressed only in one or more of the seed tissues, such as the aleurone, embryo, scutellum or endosperm may be used. A flower-specific promoter, such as the leafy promoter, may be used if the desired outcome would be to modify expression of a MAP gene in flower organs. An anther-specific promoter may be used to modify MAP expression in male reproductive organs. Further, a root-specific promoter may be used, particularly in crops of which the roots are to be harvested; such crops include sugar beet, turnip, carrot, and potato, A vascular-specific promoter may be used or a nodule-specific promoter or a stress-inducible promoter. A cell wall-specific promoter may be used or promoters expressed preferably in one or more of the above-ground tissues of the plant, such as green tissues, shoot, stem, leaves, fruits, and young expanding tissues.<br>
According to a preferred embodiment of the invention, the MAP nucleic acid in the genetic construct as described above, is operably linked to a constitutive promoter. The term "constitutive" as defined herein refers to a promoter that is expressed substantially continuously in and substantial! y in all tissues of a plant. Examples of useful constitutive promoters are selected from the rice GOS2 promoter, maize GOS2 promoter, CaMV35S promoter, ubiquitin promoter, enolase promoter, actin-2 promoter and L-41 promoter or other promoters with similar expression patterns. Promoters with similar expression patterns may be found by coupling them to a reporter gene and checking the function of the reporter gene in different tissues of a plant. One suitable reporter gene is beta-glucuronidase and the colorimetric GUS staining to visualize the beta-glucuronidase activity in a plant tissue is well known to a person skilled in the art.<br>
Optionally, one or more transcription termination sequences may also be incorporated in the genetic construct. The term "transcription termination sequence" encompasses a control sequence at the end of a transcriptional unit, which signals 3' processing and poly-adenylation of a primary transcript and termination of transcription. Additional regulatory elements, such as transcriptional or translational enhancers, may be incorporated in the genetic construct. Those skilled in the art will be aware of terminator and enhancer sequences, which may be suitable for use in the invention. Such sequences would be known or may readily be obtained by a person skilled in the art.<br>
The genetic constructs of the invention may further include an origin of replication which is required for maintenance and/or replication in a specific cell type. One example is when a genetic construct is required to be maintained in a bacterial cell as an episomal genetic element (e.g. plasmid or cosmid molecule) in a cell. Preferred origins of replication include, but are not limited to, fl-ori and colE1 ori. The genetic construct may optionally comprise a selectable marker gene. As used herein, the term "selectable marker gene" includes any gene, which confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells, which are transfeded or transformed, with a genetic construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance. Cells containing the recombinant DMA will thus be able to survive in the presence of antibiotic or herbicide concentrations that kill untransformed cells. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as nptll encoding neomycin phosphotransferase capable of phosphorylating neomycin and kanamycin, or hpt encoding hygromycin phosphotransferase capable of phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example beta-glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof). Further examples of suitable selectable marker genes include the ampicillin resistance gene (Ampr), tetracycline resistance gene (Tcr), bacterial kanamycin resistance gene (Kanr), phosphinothricin resistance gene, and the chloramphenicol acetyltransferase (CAT) gene, amongst others<br>
According to another aspect of the present invention, there is provided a method for the production of plants, having modified growth characteristics relative to corresponding wild type plants, comprising modifying expression of a nucleic acid encoding a MAP protein and/or modifying level and/or activity of a MAP protein in the plant. According to one embodiment of the invention, such a method comprises introducing into a plant cell a nucleic acid capable of modifying expression of a nucleic acid encoding a MAP protein and/or capable of modifying level and/or activity of a MAP protein.<br>
According to a further embodiment of the invention, there is provided a method for the production of plants having modified growth characteristics, which method comprises:<br>
(a)	Introducing into a plant cell a nucleic acid encoding a MAP protein or variant thereof, or<br>
introducing a variant of a nucleic acid encoding a MAP protein; and<br>
(b)	Cultivating said plant cell under conditions promoting plant growth.<br>
According to a further preferred embodiment, the nucleic acid of (a) is as represented by SEQ ID NO 1 or 2, or a variant thereof, or the nucleic acid of (a) encodes a MAP protein as represented by SEQ ID NO 2 or 4, or a variant thereof. Cultivating the plant cell under conditions promoting plant growth, may or may not include regeneration of the plant cell into a plant. Cultivating the plant cell under conditions promoting plant growth may or may not include growth to reach maturity, including for example fruit production, seed formation, seed ripening and seed setting.<br>
Methods for modifying expression of a MAP nucleic acid and/or for modifying level and/or activity of a MAP protein in a plant cell, include the introduction of the protein directly into said cell, for example by microinjection or ballistic means. Alternatively, these methods include the introduction of a nucleic acid encoding a MAP protein into a plant cell transient.<br>
The MAP nucleic acid is preferably introduced into a plant by transformation. The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer.<br>
Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention. The choice of tissue depends on the particular plant species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem).The nucleic acid may be transiently or stably introduced into a plant cell and may be maintained non-iitegrated, for example, as a plasmid. Preferably, the nucleic acid encoding a MAP protein is stably introduced into the genome of the transformed plant cell. Stable introduction into the genome of a plant cell may be achieved, for example, by using a plant transformation vector or a plant expression vector having T-DNA borders which flank the nucleic acid to be introduced into the genome.<br>
Transformation of a plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce a MAP nucleic acid into a plant cell. Transformation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microprojection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens, F.A. et a!., 1882, Nature 296, 72-74; Negrutiu I. et a/., June 1987, Plant Mol. Biol. 8, 363-373); electroporation of protoplasts (Shillito R.D. et a/., 1985 Bio/Technol 3, 1099-1102); microinjection into plant material (Crossway A. et a/., 1986, Mol. Gen Genet 202, 179-185); DNA or RNA-coated particle bombardment (Klein T.M. et a/., 1987, Nature 327, 70) infection with (non-integrative) viruses and the like, A preferred transformation method for the production of iransgenic plants according to the present invention, is an Xlgro/bacterium-mediated transformation method.<br>
Transgenic rice plants are preferably produced via ^groDacterav/n-mediated transformation using any of the well-known methods for rice transformation, such as the ones described in any of the following: published European patent application EP 1198985 A1, Aldemita and Hodges (Planta, 199, 612-617, 1996); Chan et al. (Plant Mol. Biol. 22 (3) 491-506, 1993); Hiei et al. (Plant J. 6 (2) 271-282, 1994); which disclosures are incorporated by reference herein as if fully set forth. In the case of corn transformation, the preferred method is as described in either Ishida et al. (Nat. Biotechnol. 1996 Jun; 14(6): 745-50) or Frame et al. (Plant Physiol. 2002 May; 129(1): 13-22), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more of the above-mentioned selectable marker genes, which are co-transformed with the MAP<br>
The resulting transformed plant cells, cell groupings, or plant tissue may than be used to regenerate a whole plant via techniques well known to persons skilled in the art.<br>
Subsequently, putatively transformed plant cells or plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organisation. Alternatively or additionally, expression levels of the introduced nucleic acid may be undertaken using northern and/or Western analysis, both techniques being well known to persons skilled in the art.<br>
The regenerated transformed plants may be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation transformed plant (T1 plant) may be selfed to give homozygous second generation transformed plants (T2 plants), which T2 plants may be further propagated through classical breeding techniques.<br>
The generated transformed plants may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transformants (e.g., all cells transformed to contain the genetic construct according to the invention); grafts of transformed and untransformed tissues (e.g., a transformed rootstock grafted to an untransformed scion). The present invention clearly extends to plants obtainable by any of the methods according to the present invention, which plants have modified growth characteristics. The present invention clearly extends to any plant part and propagules of such plants. The present invention extends further to encompass the progeny of a primary transformed cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the only requirement being that progeny exhibit the same genotypic and/or phenotypic characteristics) as the parent produced by the methods according to the invention. The invention also includes host cells having modified expression of a MAP gene and/or modified level and/or activity of a MAP protein. Particularly, the invention includes host cells comprising an isolated nucleic acid encoding a MAP protein. Such host cells preferably comprise a genetic construct as mentioned hereinabove. Preferred host cells according to the invention may be selected from bacteria, algae, fung i, yeast, insect plant or animal host cells. The present invention extends to a transgenic plant cell or plant having modified growth characteristics, which plant has modified expression of a MAP gene and/or modified level and/or activity of a MAP protein. Preferably said transgenic plant cell or plant comprising an isolated nucleic acid encoding a MAP protein or a variant thereof, more preferably comprises a genetic construct as mentioned hereinabove. The invention also extends to any part of the plants according to the invention, preferably a harvestable part, such as, but not limited to, a seed, leaf, fruit, flower, stem culture, stem, rhizome, root, tuber, bulb and cotton fiber.<br>
The term "plant" or "plants" as used herein encompasses whole plants, ancestors and progeny of plants and plant parts, including seeds, shoots, stems, roots (including tubers), and plant cells, tissues and organs. The term "plant" also therefore encompasses suspension cultures, embryos, meristematic regions, callus tissue, gametophytes, sporophytes, pollen, and microspores. Plants that are particularly useful in the methods of the invention include all plants which belong to the superfamily Viridiplantae, in particular monocotyledonous and dicotyledonous plants including a fodder or forage legume, ornamental plant, food crop, tree, or shrub selected from the list comprising Acacia spp., Acer spp., Actinidia spp.,Aesculus spp., Agathis australis, Albizia amara, Alsophila tricolor, Andropogon spp., Arachis spp, Areca catechu, Astelia fragrans, Astragalus deer, Baikiaea plurijuga, Betula spp., Brassica spp., Bruguiera gymnorrhiza, Burkea africana, Butea frondosa, Cadaba farinosa, Calliandra spp, Camellia sinensis, Canna indica, Capsicum spp., Cassia spp., Centroema pubescens, Chaenomeles spp.,Cinnamomum cassia, Coffea arabica, Colophospermum mopane, Coronillia varia, Cotoneaster serotlna, Crataegus spp., Cucumis spp., Cupressus spp., Cyathea d&amp;albata, Cydonia oblonga, Cryptomeria japonica, Cymbopogon spp., Cynthea dealbata, Cydonia oblonga, Dalbergia monetaria, Davallia divaricata, Desmodium spp., Dicksonia squarosa, Diheteropogon amplectens, Dioclea spp, Dolichos spp., Dorycnium  rectum, Echinochloa pyramidalis, Ehrartia spp., Eleusine coracana, Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia villosa, Fagopyrum spp., Feijoa sellowiana, Fragaria spp., Flemingia spp, Freycinetia banksii, Geranium thunbergii, Ginkgo biloba, Glycine javanica,   Gliricidia  spp,   Gossypium  hirsutum,   Grevillea  spp.,   Guibourtia  coleosperma, Hedysarum spp., Hemarthia altissima, Heteropogon contortus, Hordeum vulgare, Hyparrhenia rufa, Hypericum enectum, Hyperthelia dissoluta, Indigo incamata, Iris spp., Leptarrhena pyrolifolia, Lespediza spp., Letiuca spp., Leucaena leucocephala, Loudetia simplex, Lotonus bainesii, Lotus spp., Macrotyloma axillare, Malus spp., Manihot esculenta, Medicago sativa, Metasequoia   glyptostroboides,   Musa   sapientum,   Nicotianum   spp.,   Onobrychis   spp., Omithopus spp., Oryza spp., Peltophorum africanum, Pennisetum spp., Persea gratissima, Petunia spp., Phaseolus spp.. Phoenix canariensis, Phormium cookianum, Photinia spp., Picea  glauca,   Pinus  spp.,   Pisum   sativum,   Podocarpus   totara,   Pogonarthria   fleckii, Pogonarthria   squarrosa,   Populus   spp.,   Prosopis   cineraria,   Pseudotsuga   menziesii, Pterolobiutn   stellatum,    Pyrus   communis,    Quercus   spp.,    Rhaphiolepsis   umbellata, Rhopalostylis sapida, Rhus natalensis, Ribes grossularia, Ribes spp., Robinia pseudoacacia, Rosa spp., Rubus spp.,  Salix spp., Schyzachyrium sanguineum, Sciadopitys verticillata, Sequoia  sempervirens,   Sequoiadendron  giganteum,   Sorghum   bicolor,   Spinacia   spp., Sporobolus fimbriatus, Stiburus alopecuroides, Stylosanthos humilis, Tadehagi spp, Taxodium distichum, Themeda triandra, Trifolium spp., Triticum spp., Tsuga heterophylla, Vaccinium spp.,  We/a spp.Vitis vinifera, Watsonia pyramidata, Zantedeschia aethiopica, Zea mays, amaranth, artichoke, asparagus, broccoli, brussel sprout, cabbage, canola, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, straw, sugarbeet, sugar cane, sunflower, tomato, squash, and tea, trees and algae amongst others. According to a preferred embodiment of the present invention, the plant is a crop plant such as soybean, sunflower, canola, alfalfa, rapeseed, cotton, tomato,  potato, tobacco, squash, papaya, poplar, leguminosa, flax, lupinus and sorghum. According to another preferred embodiment of the present invention, the plant is a monocotyledonous plant, such as sugarcane, further preferably a cereal such as rice, maize (corn), wheat, barley, millet, rye and oats.<br>
Accordingly, the present invention provides any of the methods as described hereinabove, or a transgenic plant as described hereinabove, wherein the plant is a monocot, preferably a cereal, more preferably wherein the plant is rice or wherein the plant is corn.<br>
Advantageously, performance of the methods according to the present invention leads to plants having various modified growth characteristics. The term "modified" as used herein means increased or improved, decreased or changed over time and/or place. Preferably, with the methods of the present invention tne plant growth characteristics are improved. The term "growth characteristic" as used herein, preferably refers to, but is not limited to yield/biomass and plant height or any one or more of the growth characteristics as described hereinbelow.<br>
The term "yield" refers to the amount of harvestable material and is normally defined as the measurable produce of economical value of a crop. For crop plants, "yield" also means the amount of harvested material per acre or unit of production. Yield may be defined in terms of quantity or quality. The harvested material may vary from crop to crop, for example, it may be seeds (e.g. for rice, sorghum or corn when grown for seed); above-ground biomass (e.g. for corn, when used as silage), roots (e.g. for sugar beet, turnip, potato), fruits (e.g. for tomato, papaya), cotton fibers, or any other part of the plant which is of economic value. "Yield" also encompasses yield stability of the plants. High yield stability means that year after year similar yield from the progeny of the plants is obtained. "Yield" also encompasses yield potential, which is the maximum obtainable yield.<br>
Yield may be dependent on a number of yield components, which may be monitored by certain parameters. These parameters are well known to persons skilled in the art and vary from crop to crap. For example, breeders are well aware of the specific yield components and the corresponding parameters for the crap they are aiming to improve. For example key yield parameters for corn include number of plants per hectare or acre, number of ears per plant, number of rows (of seeds) per ear, number of kernels per row, and thousand kernel weight. For siiage corn, typical parameters are the above-ground biomass and energy content. Key yield parameters for rice include number of plants per hectare or acre, number of panicles per plant, number of flowers (spikelets) per panicle, seed filling rate (number of filled seeds per spiketet) and thousand kernel weight. Preferentially methods for increasing yield of rice encompass increasing number of filled seeds.<br>
According to one particular embodiment, the term "yield" encompasses "seed yield". The plants of the present invention are characterized by increased harvested seed yield. The plants are characterized by an increased number of filled seeds and increased total weight of harvested seeds. Plants according to the present invention may also have increase d total number of seeds per plant. The methods of the present invention are particularly favorable in cereals, such as rice and corn. Accordingly, a particular embodiment of the present invention relates to a method to increase yield of corn, for example seed yield, comprising modifying expression of a nucleic acid encoding a MAP protein.<br>
The term "yield" also encompasses harvest index, which is the ratio between the harvested biomass over the total amount of biomass. The plants of the present invention are characterized by increased harvest index. Advantageously, the methods of the present invention may be used to increase harvest index of cereals and in particular the harvest index of com or rice.<br>
The term "yield" may also encompass Thousand Kernel Weight (TKW), which is a parameter for the biomass of a single seed. Plants of the present invention may also exhibit increased TKW. Increased TKW is also an indication of increased seed volume and/or increased seed density. Advantageously, the methods of the present invention may be used to increase the thousand Kernel weight of cereals and in particular the thousand kernel weight of com or rice may.<br>
The term "yield" also encompasses typical biomass components, such as above-ground biomass. General biomass parameters include above-ground area and /or above-ground dry weight. The plants of the present invention are characterized by increased above-ground area. Therefore, the methods of the present invention are particularly favorable for crops grown for their green tissue and/or grown for their above-ground biomass. The methods of the present invention are particularly useful for grasses, forage crops (such as forage corn, clover and medicago), trees and sugar cane.<br>
The yield increase as obtained by the methods of the invention, may result from increase or improvement of one or more of the above mentioned yield components and/or parameters.<br>
The term "growth characteristic" as used herein also encompasses plant height. The plants according to the present invention may also exhibit increased plant height.<br>
The invention further relates to the use of an isolated nucleic acid encoding a MAP protein or use of an isolated MAP protein to modify plant growth characteristics. Furthermore, the invention embodies the use of a MAP gene or a MAP protein as a growth regulator, such as a herbicide or a growth stimulator. Also the present invention embodies a composition comprising a MAP nucleic acid or a MAP protein or a genetic construct as herein described above, for use as a regulator of plant growth characteristics <e.g. growth regulator such as a stimulator further this composition comprises suitable carrier diluent or excipient.></e.g.>
Alternatively, MAP genes and MAP proteins may be considered as interesting targets for agrochemical compounds, such as a growth regulator or growth stimulator. Accordingly, the invention embodies the use of a MAP gene or a MAP protein as a target of an agrochemical compound, such as or a growth regulator. Since the plants of the present invention have excellent growth characteristics and have high yield, they are suitable for the production of enzymes, Pharmaceuticals oragrochemicals. Also, they are suitable for the production of food or feed products. The invention clearly extends to enzymes, Pharmaceuticals or agrochemicals as well as food or feed products isolated or produced from these plants.<br>
Furthermore, a nucleic acid encoding a MAP protein, a MAP protein and/or the constructs of the present invention may be used in breeding programs aiming at the development of plants with increased yield. Furthermore, allelic variants as defined above may also be used in particular conventional breeding programs, such as in marker-assisted breeding. Such breeding programs sometimes n eed the introduction of allelic variation in the plants by mutagenic treatment of a plant. One suitable mutagenic method is EMS mutagenesis. Identification of allelic variants then takes place by, for example, PCR. This is followed by a selection step for selection of superior allelic variants of the MAP sequence, which give rise to altered growth characteristics in a plant. Selection is typicafly carried out by monitoring growth performance of plants containing different allelic variants of the MAP sequence, for example different allelic variants of the MAP sequence of SEQ ID NO 1. Monitoring growth performance can be done in a greenhouse and/or in the field. Subsequently, plants having modified growth characteristics are selected. Such growth characteristics may be any one or more of the growth characteristics as described hereinabove. Further optional steps include crossing the selected plants in which the superior allelic variant was identified with another plant, for example a plant with an economically valuable genotype. These crossing methods may be used, for example, to make a combination of interesting phenotypic features or traits.<br>
According to another type of breeding program, a DNA marker is identified which may be genetically linked to the gene capable of modifying expression of a MAP gene and/or modifying level and/or activity of a MAP protein in a plant (which gene may be the gene encoding the MAP protein itself or another gene capable of modifying expression of a MAP gene or capable of modiiying levef and/or activity of the MAP protein). This DNA marker is then used in breeding programs to select plants having modified growth characteristics. Such growth characteristics may be any one or more of the growth characteristics as described hereinabove.<br>
The methods according to the present invention may also be practised by introducing into a plant at least a part of a (natural or artificial) chromosome (such as a Bacterial Artificial Chromosome (BAC)), which chromosome contains at least a gene encoding a MAP protein, optionaly together with one or more related gene family members. Therefore, according to a further aspect of the present invention, there is provided a method for modifying plant growth characteristics comprising introducing into a plant at least a part of a chromosome comprising at least a gene encoding a MAP protein.<br>
The present invention will now be described with reference to the following figures in which:<br>
Fig. 1 is a map of the plant expression vector comprising an expression cassette for a MAP gene under control of a constitutive promoter. CDS0430 is the internal code for the Arabidopsis lhaliana cDNA encoding methionine aminopeptidase MAP2B. PRO0129 is the internal code for the rice GOS2 promoter. The MAP expression cassette also comprises the double transcription termination sequence T-zein and T-rbcS-deltaGA. This expression cassette is located within the left border (LB Ti C58) and the right border (RB Ti C58) of the nopaline Ti plasmid. Cloned within these T-borders are also a screenable marker and a selectable marker both under control of a constitutive promoter and followed by a NOS transcription termination sequence . Furthermore, this vector also contains an origin of replication (pBR322 (ori + bom) for bacterial replication and a bacterial selectable marker (Sm/SpR) for bacterial selection.<br>
Fig. 2 lists all the sequences used in the description of the present invention<br>
Fig. 3 shows a multiple alignment of animal and plant MAP proteins, with the annotation of the different domains. From the N-terminus to the C-terminus: lysine-rich domain, MAP2 signature or MAPI signature and Peptidase_M24 domain. The peptidase domain is annotated corresponding to the peptidase domain of MAP2 proteins (see for example SEQ ID NO 8). The peptidase domain of MAPI proteins extends further than this annotation (see for example SEQ ID NO 7).<br>
Examples<br>
The present invention will now be described with reference to the following examples, which are by way of illustration alone.<br>
DNA Manipulation<br>
Unless otherwise stated, recombinant DNA techniques are performed according to standard protocols described in Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York or in Volumes 1 and 2 of Ausubel et a/. (1988), Current Protocols in Molecular Biology, Current Protocols. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfase (1993) by R.D.D. Cray, published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications (UK).<br>
Example 1.   Cloning of CDS0430 encoding Arabidopsis thaliana MAP2B<br>
A gene encoding a methionine aminopeptidase MAP2B was amplified by PCR from an Arabidopsis thaliana seedling cDNA library (Invitrogen, Paisley, UK). After reverse transcription of RNA extracted from seedlings, the cDNAs were cloned into pGMV Sport 6.0. Average insert size of the bank was about 1.5 kb, and original number of clones was about 1.59x107cfu. Original titre was determined to be 9.6x105 cfu/ml. after first amplification of 6x1011 cfu/ml. After plasmid extraction, 200 ng of template was used in a 50pl PCR mix. The primers used for PCR amplification, prm01642 with the sequence 5' ACAAGTTTGTACAAAAAAGCAGGCTTCA CAATGG.CGAGCGAAAGTCC 3' as represented by SEQ ID NO 10 and prm01643 with the sequence 5' ACCCAGCTTTCTTGTACAAAGTGGTA GGATCTGAATCAGTAGTCGTCTC 3' as represented by SEQ ID NO 11, include an attB site for Gateway recombination (italics). PCR was performed using Hifi Taq DNA polymerase in standard conditions. A PCR fragment of 1381 bp was amplified and purified using standard methods. The first step of the Gateway procedure, the BP reaction, was then performed, during which the PCR fragment recombines in vivo with the pDONR201 plasmid to produce the entry done p1753. Plasmid pDONR201 was purchased from Invitrogen, as part of the Gateway® technology.<br>
Example 2: Construction of expression cassette CD2231 (pGOS2::AtMAP2B)<br>
The entry clone p1753 was subsequently used in an LR Gateway recombination reaction with p0640, a Gateway destination vector suitable for rice transformation. Vector p0640 contains as functional elements within the T-DNA borders, a plant selectable marker and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the entry clone. Upstream of this Gateway cassette lies the rice GOS2 promoter (PRO0129) for constitutive expression of the gene of interest (De Pater et a/., Plant J. 2(6) 837-844,1992). After the recombination step, the resulting expression vector with the expression cassette CD2231 (Figure 1) can be transformed into Agrobacterium strain LBA4404 and subsequently into Oryza sativa var. Nipponbare plants. Transformed rice plants were allowed to grown and were examined for various growth characteristics as described in Example 3.<br>
Example 3: Evaluation of TO, T1 and T2 rice plants transformed with pGOS2::AtMAP2B<br>
Approximately 15 to 20 independent TO transformants were generated. The primary transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. Six events of which the T1 progeny segregated 3/1 for presence/absence of the transgene were retained. "Null plants" or "Null segregants" or "Nullizygotes" are the plants treated in the same way as a transgenic plant, but from which the transgene has segregated. Null plants can also be described as the homozygous negative transformants. For each of these events, approximately 10 T1 seedlings containing the transgene (hetero-and homo-zygotes), and approximately 10 T1 seedlings lacking the transgene (nullizygotes), were selected by PCR.<br>
Based on the results of the T1 evaluation, three events, which showed improved growth characteristics at the T1 level, were chosen for further characterisation in the T2 and further generations. To this extent, seed batches from the positive T1 plants (both hetero-and homozygotes), were screened by monitoring marker expression. For each chosen event, the heterozygote seed batches were then selected for T2 evaluation. An equal number of positive and negative within each seed batch were transplanted for evaluation in the greenhouse (i.e., for each event 40 plants, of which 20 positives for the transgene and 20 negative for the transgene, were grown). For the three events therefore, a total amount of 120 plants was evaluated in the T2 generation.<br>
T1 and T2 plants were transferred to a greenhouse and were evaluated for vegetative growth parameters and seed parameters, as described hereunder.<br>
(I) Statistical analysis of numeric data<br>
A two factor ANOVA (analyses of variance) corrected for the unbalanced design was used as statistical evaluation model for the numeric values of the observed plant phenotypic characteristics. The numerical values are submitted to a t-test and an F test. The p-value is obtained by comparing the t value to the t distribution or alternatively, by comparing the F value to the F distribution. The p-value stands the probability that the null hypothesis (null hypothesis being "there is no effect of the transgene") is correct.<br>
A t-test was performed on all the values of all plants of one event. Such a t-test was repeated for each event and for each growth characteristic. The t-test was carried out to check for an effect of the gene within one transformation event, also named herein a "line-specific effect". In the t-test, the threshold for a significant line-specific effect is set at 10% probability level. Therefore, data with a p-value of the t test under 10% indicate a "line-specific" effect, meaning that the phenotype observed in the transgenic plants of that line is caused by the presence of the gene. Within one population of transformation events, some events may be under or below this threshold. This difference may be due to the difference in position of the transgene in the genome. It is not uncommon that a gene might only have an effect in certain positions of the genome. Therefore, the above-mentioned "line-specific effect" is also referred to ad "position-dependent effect".<br>
An F-test was carried out on all the values measured for all plants of all events. An F-test was repeated for each growth characteristic. The F-test was earned out to check for an effect of the gene over all the transformation events and to verify an overall effect of the gene, also named herein "gene effect". In the F-test, the threshold for a significant global gene effect is set at 5% probability level. Therefore, data with a p-value of the F test under 5% indicate a "gene effect", meaning that the phenotype observed is caused by more than just the presence of the gene and or the position of the transgene in the genome. A "gene effect" is an indication for the wide applicability of the gene in transgenic plants.<br>
(Ill Vegetative growth measurements<br>
The selected plants were grown in a greenhouse. Each plant received a unique barcode label to link unambiguously the phenotyping data to the corresponding plant. The selected plants were grown on soil in 10 cm diameter pots under the following environmental settings: photoperiod= 11.5 h, daylight intensity= 30,000 lux or more, daytime temperature= 28°C or higher, night time temperature= 22°C, relative humidity= 60-70%. Transgenic plants and the corresponding nullizygotes were grown side-by-side at random positions. From the stage of sowing until the stage of maturity (which is the stage were there is no more increase in biomass) the plants were passed weekly through a digital imaging cabinet. At each time point digital images (2048x1536 pixels, 16 million colours) were taken of each plant from at least 6 different angles. The parameters described below were derived in an automated way from the digital images using image analysis software.<br>
(a) Aboveground area<br>
Plant above-ground area was determined by counting the total number of pixels from above-ground plant parts discriminated from the background. This value was averaged for the pictures taken on the same time point from the different angles and was converted to a physical surface value expressed in square mm by calibration. Experiments show that the above-ground plant area, which corresponds to the total maximum area, measured this way correlates with the biomass of plant parts above-ground.<br>
These results of the maximum above-ground area are summarized in Table 1. On average, transgenic plants show an increase in above-ground area of 10%. In one particular line the increase in above-ground area was as high as 26%. These results indicate that the MAP gene has an effect on the above-ground biomass of MAP transgenic plants. Table 1: Maximum above-ground area of MAP transgenic T2 plants. Each row corresponds to one event, for which the average maximum above-ground area (expressed in mm2) was determined for the transgenics (TR) and the null plants (null). The difference between the transgenic plants and the null plants of each event is presented in absolute values (dif.), as well as in percentage of difference (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test.<br>
(Table Remove) <br>
(HI) Measurement of seed-related parameters<br>
The mature primary panicles were harvested, bagged, barcode-labelled and then dried for three days in the oven at 37°C. The panicles were then threshed and all the seeds collected. The filled husks were separated from the empty ones using an air-blowing device. After separation, both seed lots were then counted using a commercially available counting machine. The empty husks were discarded. The filled husks were weighed on an analytical balance. This procedure resulted in the set of seed-related parameters described below.<br>
(a) Total number of filled seeds per plant<br>
The number of filled seeds was determined by counting the number of filled husks that remained after the separation step. These numbers are summarized in Table 2. On average, transgenic plants show an increase in number of filled seeds of 52%. In one particular line the increase in number of filled seeds was as high as 76%. These results indicate that the MAP gene has an effect on the number of filled seeds of MAP transgenic plants.<br>
Table 2: Number of filled seeds of MAP transgenic T2 plants. Each row corresponds to one event, for which number of filled seeds was determined for the transgenics (TR) and the null plants (null). The difference between the transgenic plants and the null plants of each event is presented in absolute values (dif.), as well as in percentage of difference (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test. <br><br><br>
(Table Remove) <br>
 (b) Total seed yield per plant<br>
The total seed yield was measured as total seed weight, by weighing all filled husks harvested from a plant The values of total seed weight are summarized in Table 3. On average, transgenic plants show an increase in total seed weight of 55%. In one particular line the increase in total seed weight and thus in seed yield was as high as 79%. These results indicate that the MAP gene has an effect on the total seed weight and seed yield of MAP transgenic plants.<br>
Table 3: Total seed weight per plant of MAP transgenic T2 plants. Each row corresponds to one event, for which the average total seed weight (in grams) was determined for the transgenics (TR) and the null plants (null). The difference between the transgenic plants and the null plants of each event is presented in absolute values (drf.), as well as in percentage of difference (% dif). P stands for the probability produced by the t-test for each event The last row presents the average numbers calculated for all the events. Here the p-value is produced by the F-test.<br>
(Table Remove) <br>
 (c) Harvest index<br>
The harvest index in the present invention is defined as the ratio between the total seed yield and the above-ground area (mm2), multiplied by a factor 106. The values for harvest index are summarized in Table 4. On average, transgenic plants show an increase in harvest index of 36%. In one particular line the increase in harvest index and thus the increase in yield was as high as 50%. These results indicate that the MAP gene has an effect on the harvest index and on yield of the MAP transgenic plants. Table 4: Harvest index of MAP transgenic T2 plants. Each row corresponds to one event, for which the average harvest index was determined for the transgenics fJR) and the null plants (null). The difference between the transgenic plants and the null plants of each event is presented in absolute values (dif.), as well as in percentage of difference (% dif). P stands for the probability produced by the t-test for each event. The last row presents the average numbers calculated from all the events. Here the p-value is produced by the F-test<br>
(Table Remove) <br>
 (d) Thousand kernel weight<br>
TKW is extrapolated from the number of filled seeds counted and their total weight Some MAP transgenic plants showed an increase in Thousand kernel weight, which is an indicator of increased seed density and/or increased seed size.<br>
(IV) Measurement of plant Height<br>
Plant height was determined by the distance between the horizontal lines going through the upper pot edge and the uppermost pixel corresponding to a plant part above ground. This value was averaged for the pictures taken on the same time point from the different angles (taken as described above) and was converted, by calibration, to a physical distance expressed in mm. Experiments showed that plant height measured this way correlate with plant height measured manually with a ruler. Some MAP transgenic plants showed an increase of plant height.<br>
Example 4. Evaluation of transgenic corn plants transformed with MAP<br>
The methods of the invention described herein are also used in corn (Zea mays). To this aim, a MAP encoding gene, for example a corn MAP ortholog ue, is cloned under control of a promoter operable in corn, in a plant transformation vector suitable for Agrobacterium-mediated com transformation. The promoter operable in corn may for example be a constitutive promoter, selected from GOS2 promoters, ubiquitine promoters, L-41 promoters, actine-2 promoters and enolase promoters. Methods to use for com transformation have been described in literature (Ishida et al., Nat Biotechnol. 1996 Jun;14(6):745-50; Frame et al., Plant Physiol. 2002 May;129(1):13-22). Transgenic (inbred) lines made by these methods may be crossed with another non-transgenic or transgenic (inbred) line or be self/sib-pollinated. Importantly, transgenic (inbred) lines may be used as a female or male parent. Inheritability and copy number of the transgene are checked by quantitative real-time PCR and Southern blot analysis and expression levels of the transgene are determined by reverse PCR and Northern analysis. Transgenic events with single copy insertions of the transgene and with varying levels of transgene expression are selected for further evaluations in subsequent generations.<br>
Progeny seeds obtained as described hereinabove are germinated and grown in the greenhouse in conditions well adapted for corn (16:8 photoperiod, 26-28°C daytime temperature and 20-24°C night time temperature) as well under water-deficient, nitrogen-deficient, and excess NaCI conditions. Null segregants from the same parental line (inbred line or hybrids), as well as wild type plants of the same inbred line or hybrids are used as controls. The progeny plants are evaluated on different biomass and developmental parameters, including but not limited to plant height, stalk width, nodes below ear, nodes above ear, brace roots, number of leaves, leaf greenness, leaf angle, total above-ground area time to tassel, time to silk, time to maturity, ear height, ear number, ear length, ear weight, row number, kernel number, grain moisture. Kernel traits include but are not limited to kernel size, kernel weight, starch content, protein content, and oil content are also monitored. Corn yield is calculated according to well-known methods. Corn plants transformed with a MAP protein show improved growth characteristics. More particularly they show an improvement in any one or more of the abovementioned biomass and developmental parameters.<br>
Transgenic events that are most significantly improved compared to corresponding control lines are selected for further field-testing and marker-assisted breeding, with the objective of transferring the field-validated transgenic traits into another germplasm. The phenotyping of maize for growth and yield-related parameters in the field is conducted using well-established protocols. The com plants are particularly evaluated on yield components at different plant densities and under different environmental conditions. Subsequent improvements for 30 introgressing specific loci (such as transgene containing loci) from one germplasm into another is also conducted using well-established protocols including but not limited to MAS.<br>
Example  5.  Evaluation of transgenic plants transformed with Arabidopsis thaliana MAPI A<br>
35 The methods described In Examples 1, 2, 3 and 4 are also repeated with Arabidopsis thaliana MAP1A. To this aim, the AtMAPIA encoding gene, represented by SEQ ID NO 3, is cloned under control of a promoter operable in rice or maize, in a plant transformation vector suitable for Agnobacterium-medlaied transformation of rice or corn. One suitable promoter is a constitutive promoter, such as a GOS2 promoter.<br>
In case of rice plants, the protocol for isolation of cDNA and for vector construction is followed as described in Example 1 and 2, except that the primers are specific for SEQ ID NO 3. The protocols for plant transformation, plant growth and plant evaluation are followed as described in Example 3. Rice plants transformed with AtMAPIA have modified growth characteristics and show any one or more of the modified growth characteristics as described in Example 3.<br>
In case of corn plants, the protocols for plant transformation, plant growth, plant propagation, plant selection, and plant evaluation are followed as described in.Example 4. Corn plants transformed with AtMAPIA have modified growth characteristics and show any one or more of the improved growth characteristics as described in Example 4.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	Method for modifying yield, seed yield or plant height relative to corresponding wild type plants, comprising modifying expression in a plant of a nucleic acid encoding a methionine aminopeptidase (MAP protein) represented by SEQ ID NO: 2 or a homologue thereof having at least 35% sequence identity to SEQ ID NO: 2.<br>
2.	Method as claimed in claim 1, wherein said MAP protein is a plant MAP2 protein comprising a MAP2 signature sequence.<br>
3.	Method as claimed in claims 1 and 2, wherein said modifying expression<br>
comprises increased expression.<br>
4.	Genetic construct comprising:<br>
a)	a nucleic acid encoding a methionine aminopeptidase (MAP protein) represented by SEQ ID NO: 2 or a homologue thereof having at least 35% sequence identity to SEQ ID NO: 2 as claimed in claim 1;<br>
b)	one or more control sequences capable of driving expression of the nucleic acid of (a) in a plant; and optionally<br>
c)	a transcription termination sequence.<br><br>
5.	Genetic construct as claimed in claim 4, wherein the nucleic acid of a) is represented by SEQ ID NO. 1 or a variant thereof encoding a MAP protein represented by SEQ ID NO: 2 or a homologue thereof.<br>
6.	Genetic construct as claimed in claims 4 and 5, wherein said control sequence of (b), is a constitutive promoter, such as a GOS2 promoter.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUFic3RyYWN0LSgyMC0wOC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Abstract-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUNsYWltcy0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Claims-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-Correspondence Others-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Correspondence-Others-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Correspondence-Others-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMjAtMDgtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-description (complete)-20-08-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUZvcm0tMS0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Form-1-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUZvcm0tMi0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Form-2-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LUdQQS0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-GPA-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LUdQQS0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-GPA-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bWVudC0oMjAtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Others-Document-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bWVudC0oMjktMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-Others-Document-(29-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LVBDVC0yMTAtKDIwLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-PCT-210-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LVBDVC0yMjAtKDIwLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-PCT-220-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LVBDVC0yMzctKDIwLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-PCT-237-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LVBDVC0zMDQtKDIwLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-PCT-304-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxOUC0yMDA1LVBDVC0zMDgtKDIwLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELNP-2005-PCT-308-(20-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1kZWxucC0yMDA1LVBldGl0aW9uIE90aGVycy0oMjYtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3347-delnp-2005-Petition Others-(26-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtQWJzdHJhY3QtKDE5LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Abstract-(19-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtQ2xhaW1zLSgxOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Claims-(19-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgxOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Correspondence-Others-(19-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtRm9ybS0xLSgxOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Form-1-(19-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtRm9ybS0yLSgxOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Form-2-(19-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ny1ERUxQLTIwMDUtT3RoZXJzLURvY3VtZW50LSgxOS0xMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">3347-DELP-2005-Others-Document-(19-12-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234497-bioerodible-sustained-release-drug-delivery-systems.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234499-a-process-for-the-preparation-of-yeast-strain-characterized-in-protease-enzyme.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234498</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3347/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Jul-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3,B-9052 ZWIJNAARDE,BELGIUM,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VALDIMIR MIRONOV</td>
											<td>ZEBRASTRAAT 11,B-9000 GENT,BELGIUM,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2004/050092</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-02-06</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03075363.6</td>
									<td>2003-02-06</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234498-method-for-modifying-yield-and-or-height-of-a-plant-comprising-modifying-expression-of-a-nucleic-acid-encoding-map-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:13 GMT -->
</html>
