<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234499-a-process-for-the-preparation-of-yeast-strain-characterized-in-protease-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234499:A PROCESS FOR THE PREPARATION OF YEAST STRAIN CHARACTERIZED IN PROTEASE ENZYME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PROCESS FOR THE PREPARATION OF YEAST STRAIN CHARACTERIZED IN PROTEASE ENZYME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a process for the preparation of yeast strain characterized in deficient in protease enzyme. The novelty of the process is inherent in the unique approach followed for constructing the strains. The invention relates more particularly to the fission yeast Schizosaccharomyces pombe (S. pombe).The yeast strains so prepared can be used for multiple purposes that include their use as hosts for the expression of commercially important proteins. The said strains are useful for the expression of any mammalian, viral, plant, yeast, fungal or bacterial proteins. The said strains allow use of the commonly used vectors based on the S. pombe ura4+ gene or the S. cerevisiae URA3 gene. It also allows use of the commonly used vectors based on the LHU2 gene of S. cerevisiae or the leul +gene of S. pombe.The strains are also useful because the protease deficiencies can be followed by simple plate assays.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a process for the preparation of yeast strains that are deficient in proteases. The novelty of the process is inherent in the unique approach followed for constructing the strains. The invention relates more particularly to the fission yeast Schizosaccharomyces pombe (S.pombe). The yeast strains so prepared can be used for multiple purposes that include their use as hosts for the expression of commercially important proteins. The said strains are useful for the expression of any mammalian, viral, plant, yeast, fungal or bacterial proteins. The said strains allow use of the commonly used vectors based on the the S.pombe ura4+ gene or the S.cerevisiae URA3 gene. It also allows use of the commonly used vectors based on the LEU2 gene of S.cerevisiae or the Ieu1+ gene of S.pombe. The strains are also useful because the protease deficiencies can be followed by simple plate assays.<br>
Yeasts are excellent model systems for investigating many aspects of basic cell biology. This is because they have all the basic features of plant and animal cells, and are easy and inexpensive to grow. In addition, there is a large number of molecular and genetic techniques available for introducing DNA into these cells and for analyzing and recovering proteins from yeasts thus making them accessible to genetic as well as biochemical investigations. These are also the reasons why yeasts are being used as hosts for the expression of a variety of proteins including those from mammals, plants and viruses. Most laboratories have been using the yeast Saccharomyces cerevisiae as well as the methanolotrophic yeasts Hansenula polymorpha and Pichia pastoris for protein expression studies.<br>
The yeast Schizosaccharomyces pombe has some special advantages over the other yeasts that are currently being exploited for overexpression studies. One of the major reasons is that these yeasts resemble the higher eukaryotic systems more closely in many processes. Thus there is an increasing interest in wanting to express many proteins of mammalian or plant origin in this yeast so that they are folded and processed more accurately and in a fashion that resembles the orignal environment most authentically (Giga-Hama and Kumagai Ed. Foreign gene expression in fission yeast Schizosaccharomyces pombe, Springer-Verlag, 1997). Several proteins have thus been expressed in S.pombe. These include the human lipocortin I protein ( Giga-Hama et.al, Biotechnology, 12, 400, 1994), human gastric lipase (Smerden et.al, Appl. Microb.Biotech. 49, 45, 1998) the mouse alpha amylase (Tokunaga et.al., Yeast, 9, 379, 1993) and the fungal cellobiohydrolase II (Okada et.al, Appl. Microb. Biotech.49, 301, 1998).<br>
A major drawback in the present S.pombe host strains that are available is that owing to the presence of inherent proteases in the cell, many native and recombinant proteins that are sensitive to proteases have inefficient recoveries and yields from these cells (Suarez-Rendueles et.al, FEMS Lett., 81, 215, 1991; Tokunaga et.al., Yeast, 9, 379-383, 1993). As a result numerous difficulties are encountered during the purification process and precautions need to be taken. For example, sufficient levels of protease inhibitors need to be added which are both expensive arid toxic. Protease inhibitors are chemical compounds that inactivate proteases. An alternate approach, that has been successfully employed in Saccharomyces cerevisiae is through the<br><br>
construction of strains that are deficient in several of the proteases that are responsible for the proteolytic degradation. These strains have been found to be very beneficial for all biochemical and purification studies.<br>
In the case of Schizosaccharomyces pombe, however, although several different proteases and proteolytic activities have been identified in S.pombe, a process for the construction of such strains that are deficient in several proteases has not been described so far. Some of the proteases that have been identified include the carboxypeptidase Y, (cpylp) (Simeon et.al., Yeast<br>
11,	271-282, 1993), the carboxypeptidase, sxa2p (Imai et.al., Mol.Cell.Biol.,<br>
12,	1827-1834, 1994), , the aspartyl protease, sxal (Imai et.al., Mol.Cell.Biol.,<br>
12, 1827, 1994), the serine protease, isp6p (Sato, S. et.al., Curr.Genet., 26,<br>
31, 1994), the Kex2-like protease Krp (Davey, J. et.al., EMBO J., 13, 5910-<br>
5921, 1994) and the aminopeptidase, ysplp (Arbescu et.al., Yeast, 9, 637,<br>
1993). The genes for many of these proteases have been isolated and<br>
characterized. In some cases mutants that are defective in these proteases<br>
have been isolated. In other cases,   disruptions of these genes have   been<br>
carried out by a techniques known as gene replacement through homologous<br>
recombination (Rothstein, Methods in Enzymology, 194, 281-301,1991). By<br>
disruption of a gene it is meant the deletion of the gene encoding for a<br>
particular protein which results in null activity of the protein. So far, however, in<br>
S.pombe only those strains where a single protease gene has been disrupted<br>
are available. One reason for this has been the relative lack of sufficient<br>
auxotrophic markers that have been employed in the majority of the laboratory<br>
strains of S.pombe. Auxotrophy refers to the ability of a strain of microrganisms<br><br>
to proliferate or grow in a medium only when supplemented with a particular nutrient. An auxotrophic marker refers to a strain of microorganisms that can be identified by it's auxotrophic requirement for a particular nutrient. Thus for example, a strain that has a mutation or deletion in the ura4 gene in S.pombe would be recognized by it's ability to grow only in minimal medium supplemented with uracil. Similarly a Ieu1 mutation in S.pombe would be recognized by the ability to grow only when supplemented by leucine in the medium and an ade6 mutation would be recognized by it's ability to grow only in medium supplemented with adenine.<br>
Thus there is a need to have S.pombe strains that do not suffer from the above drawbacks and would be deficient in the important proteoiytic activities of the cell, as well as would have the ura 4 marker and other auxotrophic markers such as ade6 and Ieu1 free for other genetic and molecular manipulations.Thus in the present invention the drawbacks of the yeast strains have been overcome and a novel strain of S.pombe has been constructed which is protease deficient, hence has wide range of use chiefly as expression host for proteins. The strain has been deposited in the Microbial Type Culture Collection and Gene Bank, a National facility located in the 'Institute of Microbial Technology1, and has been assigned an Accession no. MTCC Y0050, where Y stands for yeast. The characteristics of this strain are as follows: The cells are globose and cylindroidal and reproduce by fission. The sugars glucose, sucrose and maltose are fermented; nitrate is not assimilated; the strain shows a positive starch test in Wickerham's liquid medium; urea is hydrolyzed and Diazonium Blue B is negative.<br>
In the organism of the present invention the protease genes are disrupted to make this strain protease-deficient and useful for the expression of any mammalian, viral, plant, yeast, fungal or bacterial proteins.<br>
The main objective of the present invention is, therefore, to provide yeast strains that are deficient in the major proetolytic activities of the cell.<br>
Another objective of the present invention is to construct and provide protease deficient strains where the individual protease deficiencies can be detected either by easily assayed markers or by enzymatic plate assays.<br>
Yet another objective of the present invention is to construct protease deficient strains, that, in addition to being protease deficient, have other auxotrophic mutations such as the ade6, leul and ura4 auxotrophic markers available for subsequent manipulations of this yeast.<br>
Accordingly, the present invention provides an process for the preparation of yeast strain characterized in deficient in protease enzyme and useful for expression of commercially important proteins which comprises isolating gene encoding protease enzyme from the said yeast strain (Schizosaccharamyces pombe) such as herein described, disrupting the said protease encoding genes using selection markers, mutagens such as herein described, reconstructing the yeast strain with resultant disrupted protease encoding gene such as herein described to get the desired strain of yeast.<br>
In an embodiment of the present invention, protease genes cpyl+, sxal+, sxa2+, isp6+, dpal+, yspl+, particularly sxal+ and isp6+ may be isolated by standard Polymerase Chain reaction (PCR) amplification and cloning methods. The markers used for the disruption of the protease genes<br><br><br>
may be such as the auxotrophic and anitbiotic resistant markers, for example, arg3+, his3+, his7+, Iys1+, hisS*, Kanr particularly his3+ and arg3+ The reconstruction is effected by making the S.pombe cells competent to take up the disrupted protease gene by standard chemical or electroporation methods followed by selecting and then confirming cells for the disrupted protease activity.<br>
The mutations may be effected by using mutagens such as ethyl<br>
methane     sulphonate,     ultraviolet     irradiation,	N-methyl-N'-nitro-N-<br>
nitrosoguanidine, in particular ethyl methane sulphonate to mutate protease genes such as cpy1+, sxa1+, sxa2+, isp6+, dpa1+, ysp1+ in particular cpy1+. The protease deficient S.pombe strains thus obtained can be followed by convenient markers and plate assays. Furthermore the strains retain several auxotrophic markers such as the ura4, Ieu1 and ade6 available for further genetic and molecular manipulations.<br>
The process of the present patent thus involves the following steps:<br>
1)	Isolation and cloning of the yeast protease genes sxa1+ and isp6+ from<br>
S.pombe, using a known method<br>
2)	Disruption of the protease genes on the plasmids with the S.pombe genes<br>
his3+ and arg3+ as selection markers.<br>
3)	Construction of  S.pombe strains that have multiple auxotrophic markers<br>
and are also mutated in the carboxypeptidase Y gene.<br>
4)	Construction of S.pombe strains that  have  multiple  markers but are<br>
disrupted in either the aspartyl protease (sxatp) or the serine protease<br>
(isp6p) or both.<br>
5)	Preparation of strains that  are deficient in sxalp, isp6p and cpylp, while<br>
having the commonly used auxotrophic markers also available for genetic<br>
and molecular manipulations.<br>
6)	Use of strains prepared as in steps 3 to 6 as hosts for the production and<br>
purification of proteins<br>
The details of the present invention are further provided below:<br>
The S.pombe strain ABP20 that was used in this invention was obtained from the Fission Yeast Course, Cold Spring Harbor Laboratories, NY. Strain ABP590 which was also used in this invention was a strain previously constructed in this lab by random spore analysis of ABP20 with strains obtained from Kathy Gould, USA and from Scott-Wadell, U.K.<br>
The sxa1+ and isp6+ genes of S.pombe were amplified by using the standard Polymerase Chain Reaction (PCR) method using Tag polymerase enzyme, specific oligonucieotide primers, and using S.pombe genomic DMA as a template DNA for the PCR reaction. The S.pombe genomic DMA was isolated from S.pombe by a modification of the glass bead lysis method used for S.cerevisiae ( Kaiser et.al., Methods in Yeast Genetics: A laboratory manual, Cold Spring Harbor Press 1997).The oligonucieotide primers used for the PCRamplification of sxa1+ were sxalfor and sxalrev. The sequences of these oligonucieotide primers are: sxalfor, 5'-acgcatacaccgtctattag-3';<br>
sxa1rev,5'-agccggcagtgtaacgaaaga-3'. The oligonucleotide primers used for the PCR amplification of isp6+ were ispSf and ispSr.The oligonucleotide sequence of the primers are: isp6f, 5'-gcctcgtcgactcagcatcaacatgag-3' and isp6r, 5'-tcaggatcagacttattgaaaacatacgaatg-3'. Here, the letters a,c,g, and t are abbreviations for the four bases adenine, cytosine, guanine and thymidine that are found in DNA. The amplified sxa1+ and /sp6+ genes were purified from agarose gels and after digestion with restriction enzymes were ligated into appropriate restriction sites in cloning vectors pSP2 and pSP1 respectively to create vectors pSXA1 H and pLSPT respectively. Agarose gel electrophorses, DNA purification, DNA isolations in particular plasmid DNA, Restriction enzyme digestions, ligations and all other recombinant DNA methods were carried out using standard protocols (Sambrook et.al., Molecular Cloning : A laboratory manual. Cold Spring Harbor Press Ltd., 1989). For disruption of the sxa1+ protease gene, an S.pombe his3+ gene was excised from the vector pKLG497 (Burke and Gould, Mol. Gen. Genet., 242, 169, 1994) by restriction digestion with Bgl II enzyme and the purified 2.2.kb Bgl II his3+ fragment was inserted into the Bgl II site of the sxa) clone to create a sxalv his3+ disrupted plasmid. Similarly an S.pombe arg3+ gene was excised from the plasmid paR3 (Wadell and Jenkins, Nucleic Acids Research, 23, 1836, 1995) and inserted into the Sea I sites of the /sp6+ gene in pLSP7 to create plasmid plSP6-R4 that contained isp6D::arg3+. These plasmids were constructed to create disruptions in the sxa1+ and the /sp6+ genes in an S.pombe strain that was deficient in carboxypeptidase Y activity. The strategy to create disruptions of these known<br>
proteases by this combination of his3+ and arg3+ markers without using the ura4, Ieu1 and ade6 markers was a novel approach taken during this study<br>
To construct strains deficient in carboxypeptidase Y that also contained additional auxotrophic markers, an S.pombe strain ABP20, h-ade6-210 leu1-32 ura4-D18 was mutagenized by 2% ethyl methyl sulphonate using known conditions (Alfa et.al., Experiments with fission yeast. A laboratory manual. Cold Spring Harbor Laboratory Press Ltd., 1992) and cells after mutagenization were diluted and screened by a simple colorimetric plate assay (Jones, Methods in Enzymology, 194, 428-452,1991) for the carboxypeptidase Y enzyme activity. Mutants deficient in carboxypeptidase Y were isolated. These were classified and characterized by conventional genetic techniques such as random spore analysis and complementation analysis (Alfa et.al., Experiments with fission yeast. A laboratory manual. Cold Spring Harbor Laboratory Press Ltd., 1992). One of these mutants cpy1-1, (strain ABP 600 : h- ade6-216 Ieu1-32 ura4-D18 cpy1-1) which was carboxypeptidase Y deficient was mated with a wild type S.pombe ABP 590 strain (of opposite mating type) h+ ade6-M216 leu1-32 ura4-D18 his3-D1 arg3-D4 and contained the additional markers, his3-D1 and arg3-D4. The mating reaction was carried out on Malt Extract Agar medium using known procedures (Moreno et.al., Methods in Enzymology, 194, 795, 1991), and after 3 days at 25C the mating mix was incubated overnight with glusulase (0.5%) to digest the ascus cell wall as well as the parent haploids. The glusulase was washed of , d aliquots were diluted and plated, onto YES plates. Only the spores survive and the spores that grew were derived from the above cross . The spores were analyzed for the different<br>
phenotypes and the spores with the desired phenotypes that had a deficiency in cpyl as well as had all the other markers in both mating types were preserved. This entire procedure is referred to as random spore analysis ( Moreno et.al., Methods in Enzymology, 194, 795, 1991). The strain ABP 600 whose genotype is : h-ade6-216 ura4-D18 leu1-32 arg3-D4 his3-D1 cpy1-1 was prepared in this fashion.<br>
The sxa1::his3+ fragment was digested with Sma I and Sal I and the 3.9Kb fragment containing the sxa1::his3+ region was used to transform the S.pombe strain ABP 600 (that was constructed as described above) by a modified lithium acetate method (Chaudhuri et.al, Genetics,145, 75,1997). By transformation it is meant here the process by which the host organism is made to accept and receive the DNA fragment. The S.pombe strain that was used for transformation was ABP 600 : h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4 cpy1-1. The transformants were selected on Edinburgh's minimal medium (Moreno et.al., Methods in Enzymology, 194,795, 1991) that was supplemented with different amino acids and other supplements but which lacked histidine. Colonies that were able to grow these histidine lacking plates were picked up, purified by single colony streaking on YES medium (0.5% Yeast extract, 3% glucose, 50mg/litre each of uracil, leuine, adenine, arginine, histidine) and checked for disruption at the sxal locus using the Polymerase Chain Reaction (PCR) at standard PCR conditions. The disruption at the sxal locus occurs by a process known as gene replacement by homologous recombination . In this way strains were constructed that were deficient in both cpy1+ activity and sxa1+ activity and the sxal deficiency was marked by the<br>
his 3 marker.The strain obtained was ABP 631 whose genotype was h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4 cpy1-1 sxa1::his3+.<br>
To construct strains that were also deficient in isp6p, a fragment containing the isp6D::arg3+ region was amplified using PCR from the plasmid plSP6-R4 constructed as described above. This PCR amplified fragment was purified from the gel and was used to transform S.pombe ABP 631 : h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4 cpy1-1 sxa1::his3+. Transformants were selected for growth on arginine-lacking medium. For other selection markers appropriate selection medium has to be used to select and confirm the transformatnts. The transformants were picked up, colony purified and confirmed to a carry a disruption in the isp6 gene by PCR. The isp6p deficiency was marked by an arg3+ marker. In this way strains ABP 687 and ABP688 were prepared that were deficient in isp6p in addition to being deficient in cpyl p and sxal p and had the auxotrophic markers ade6, ura4 and Ieu1 available for subsequent manipulations.<br>
Example 1<br>
This example describes the construction of strains that are deficicient in sxalp activity and this deficiency is marked by the S.pombe his3+ marker.<br>
The S.pombe strain ABP20, was obtained from the Fission Yeast Course, Cold Spring Harbor Laboratory; strain ABP 590 was a strain previously constructed in our lab and was derived from ABP20 and S.pombe strains obtained from Dr. Kathy Gould, USA , as well from strains obtained from Dr. Scott-Wadell, U.K..<br>
The sxa1+ gene of S.pombe was amplified by the Polymerase Chain Reaction (PCR) at an annealing temperature of 55 C using Taq polymerase enzyme and using S.pombe genomic DMA as a template. The S.pombe genomic DMA was isolated from S.pombe by a modification of the glass bead lysis method used for S.cerevisiae ( Kaiser et.al., Methods in Yeast Genetics: A laboratory manual, Cold Spring Harbor Press 1997).The amplified sxa1+ gene fragment was purified from agarose gels and digested with the restriction enzyme Hind III. The 1.6kb Hind III fragment was then cloned into the Hind III site of the cloning vector pSP2 (Cottarell etal., Curr. Genet., 23, 547-548, 1993). All recombinant DMA methods were carried out using standard protocols (Sambrook et.al., Molecular Cloning: A laboratory manual. Cold Spring Harbor Press, Ltd. 1989). An S.pombe his3+ containing selection marker was excised from the vector pKLG497 (Burke and Gould, Mol.Gen.Genet., 242,179,1994) by restriction digestion with Bgl II enzyme and the purified 2.2.kb Bgl II his3+ fragment was inserted into the Bgl II site of the sxal clone to create a sxa1::his3+ disrupted plasmid. The sxa1::his3+ fragment was digested with Sma I and Sal I and the 3.9Kb fragment containing the sxa1::his3+ region was used to transform an S.pombe strain by a modified lithium acetate method (Chaudhuri et.al, Genetics, 145, 75, 1997). By transformation it is meant here the process by which the host organism is made to accept and receive the DNA fragment. Transformation has alwaysbeen carried out with a known procedure. The S.pombe strain that was used for transformation was ABP 511 : h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4. The transformants were selected on Edinburgh's minimal medium<br>
(Moreno et.al., Methods in Enzymology, 194, 795, 1991) that was supplemented with different amino acids and other supplements but which lacked histidine. Colonies that were able to grow these histidine lacking plates were picked up, purified by single colony streaking on YES medium (0.5% Yeast extract, 3% glucose , 50mg/litre each of uracil, leuine, adenine, arginine, histidine) and checked for disruption at the sxa1+ locus using the PCR using gene specific primers and at standard PCR condiitons. The disruption at the sxal locus occurs by a process known as gene replacement by homologous recombination . In this way strains were constructed that were deficient in sxal p activity and were marked by the his 3 marker.The strain obtained was ABP 632 whose genotype was h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4 sxa1::his3+.<br>
Example 2<br>
This example describes the construction of strains deficient in isp6p acticity, and this deficiency is marked by the arg3+ marker.<br>
The isp6+ gene was amplified by PCR at an annealing tempereature of 60 Â°C. The PCR product was purified from agarose gels after electrophoresis and the PCR product was digested with the restriction enzymes Sac I and Nsi I and cloned into the Sac I and Pst I sites of plasmid pSP1 (Cottarell et.al., Curr. Genet, 23, 547-548, 1993). The resulting plasmid, pLSP7, containing a fragment of the /sp6+ gene was digested with the enzyme Sea I, and purified after gel electrophoresis. An arg3+ ,.:sne was excised from plasmid paR3 (Wadell and Jenkins, Nucleic Acids Research, 23, 1836, 1995) by digestion with the enzymes Sma I and Pst I and the 1.1 Kb fragment was blunted with<br>
Klenow enzyme and cloned into the Sea I site of the pLSP7 DNA that had been purified after Sea I digestion. The disruption plasmid isp6D::arg3+ was thus generated (plSP6-R4). A fragment containing the isp6D::arg3+ region was amplified using PCR from the plasmid plSP6-R4 and this amplified fragment was purified from the gel and was used to transform an S.pombe strain as described in Example 1. The S.pombe strain that was used for transformation was ABP 511 : h- ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4. Transformants were selected for growth on arginine-lacking medium. The transformants were picked up, colony purified and confirmed to a carry a disruption in the isp6+ gene by PCR. In this way strains were prepared that were deficient in isp6p and this deficiency was marked by an arg3+ marker.<br>
Example 3<br>
This example describes the construction of strains deficient in carboxypeptidase Y which can be followed by,a smple plate test, and the strains also contain additional auxotrophic markers.<br>
An S.pombe strain ABP20, h-ade6-210 leu1-32 ura4-D18 was mutagenized by 2% ethyl methane sulphonate using known conditions (Alfa et.al., Experiments with fission yeast. A Laboratory Manual. Cold Spring Harbor Laboratory Press Ltd., 1992) and cells after mutagenization were diluted and screened by a simple colorimetric plate assay (Jones, Methods in Enzymology, 494,428-451, 1991) for the carboxypeptidase Y enzyme activity.<br>
Mutants deficient in carboxypeptidase Y were isolated. They were classified and characterized by conventional genetic techniques such as random spore<br>
analysis and complementation analysis. One of these mutants cpy1-1 , (strain ABP 600 : h- ade6-216 leu1-32 ura4-D18 cpy1-1) which was carboxypeptidase Y deficient was mated with a wild type S.pombe ABP 590 strain (of opposite mating type) h+ ade6-M216 leu1-32 ura4-D18 his3-D1 arg3-D4 and contained the additional markers, his3-D1 and arg3-D4. The mating reaction was carried out on Malt Extract Agar medium using known procedures (Moreno et.al., Methods in Enzymology, 194, 795, 1991), and after 3 days at 25C the mating mix was incubated overnight with glusulase (0.5%) to digest the ascus cell wall as well as the parent haploids. The glusulase was washed off and aliquots were diluted and plated onto YES plates. Only the spores survive and the spores that grew were derived from the above cross . The spores were analyzed for the different phenotypes and the spores with the desired phenotypes that had a deficiency in cpylp as well as had all the other markers in both mating types were preserved. This entire procedure is referred to as random spore analysis ( Moreno et.al., Methods in Enzymology, 194, 795, 1991). The strain ABP 600 whose genotype is : h-ade6-216 ura4-D18 leu1-32 arg3-D4 his3-D1 cpy1-1 was prepared in this fashion.<br>
Example 4<br>
This example describes the construction of an S.pombe strain that is deficient in cpy activity as well as sxal activity.<br>
S.pombe ABP 600 h- ade6-216 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy1-1 was transformed with the sxa1::his3+ fragment as described in example 1. Transformants that were his"1" were picked up and those having an sxal<br>
disruption in the chromosome were confirmed by PCR. S.pombe strain ABP 631 whose genotype is :/?- ade6-216 leu1-32 ura4-D18 arg3-D4 his3-D1cpy1-1 sxa1::his3+ was prepared in this manner.<br>
Example 5<br>
This describes a process where strains deficient in both cpyl and isp6 were constructed.<br>
The isp6D::arg3+ plasmid (plSP6-R4) that was constructed as described in example 2 was used to amplify the isp6D::arg3+ region and the isp6D::arg3* fragment was purified and used to transform S.pombe ABP 600 rr ade6-210 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy 1-3. Transformants that were arg+ were picked up and those having an isp6 disruption in the chromosome were confirmed by PCR. These strains were deficient in isp6p and sxalp. Strain ABP 713 was prepared in this manner.<br>
Example 6<br>
A strain that was triply deficient in cpyl p, sxal p and isp6p proteases was constructed using the procedure as below..<br>
An S.pombe strain ABP 631, that was deficient in both cpylp and sxalp was transformed with a an isp6D::arg3+ fragment and transformants were selected that were arg+. These transformants were confirmed for disruption at the /sp6+ gene locus by Polymerase Chain Reaction. S.pombe strain ABP 687 whose genotype is h- ade6-216 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy1-1 sxa1::his3+ isp6D::arg3+ was deficent in all three proteases.<br>
Example 7<br>
For the construction of protease deficient strains in different mating types, strain ABP 687 whose genotype was h- ade6-M216 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy1-3 sxa1::his3+ isp6D::arg3+ was crossed with strain ABP 509 h+ade6-216 leu1-32 ura4-D18 his3-D1 arg3-D4. Random spore analysis for strain construction was carried out as described in Example 3. Strains deficient in different proteases and in different mating types h- and h+ were constructed and analyzed by the known plate assays and auxotrophies. The strains that were prepared by this method were ABP 711: h+ade6-216 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy1-1 sxa1::his3+ isp6D::arg3+ and ABP 712 : h+ ade6-216 ura4-D18 leu1-32 his3-D1 arg3-D4 cpy1-1 sxa1::fiis3+ isp6D::arg3+<br>
Advantages of the present invention:<br>
1)	Strains  defective  in  single  or  multiple  proteases were  constructed.<br>
Furthermore any combination of markers and protease deficiencies can be<br>
obtained by using the different molecular and genetic strategies of strain<br>
constructions<br>
2)	The strains have been disrupted for proteases without making use of the<br>
marker ade6 which is routinely used in all genetic experiments in S.pombe<br>
3)	The strains have been disrupted for proteases without making use of the<br>
ura4 and Ieu1 markers that are otherwise commonly used in S.pombe. As a<br>
result it allows the ability to use vectors  based on the S.pombe ura4 gene<br>
or the S.cerevisiae URA3 gene. It allows use of the commonly used vectors based on the Ieu1 gene of S.pombe and the LEU2 gene of S.cerevisiae.<br>
4) The process for the construction of the strains deficient in the different proteases has been carried out such that each of the protease deficiencies can be followed by simple plate assays. Carboxypeptidase Y deficiency can be detected by the simple plate assay, the APE test. Disruption in isp6+ can be identified by prototrophy on minimal medium plates lacking arginine. Disruption in sxa1+ can be identified by prototrophy in minimal medium plates lacking histidine.<br><br><br><br><br><br>
claim<br>
1.	A process for the preparation of yeast strain characterized in deficient in protease enzyme and useful for expression of commercially important proteins which comprises isolating gene encoding protease enzyme from the said yeast strain (Schizosaccharamyces pombe) such as herein decribed, disrupting the said protease encoding genes using selection markers, mutagens such as herein described, reconstructing the yeast strain with resultant disrupted protease encoding gene such as herein described to get the desired strain of yeast.<br>
2.	A process as claimed in claim 1 wherein protease genes yeast. cpyl+, sxal+ , sxa2+, isp6+, dpal+, ysplf, in particular sxal+ and isp6* arc isolated by standard Polymerase Chain reaction (PCR) amplification and cloning methods the markers used for the disruption of the protease genes may be such as the auxotrophic and anitbiotic resistant markers, for example, arg3+. his3+, his7+, lysl+, his5+, Kanr particularly his3+ and arg3+.<br>
3.	A process as claimed in claims 1 and 2 wherein the reconstruction is effected by making the S.pombe cells competent to take up the disrupted protease gene by standard chemical or electroporation methods followed by selecting and then confirming cells for the disrupted protease activity.<br>
4.	A process as claimed in claims 1 to 3 wherein the mutations are effected by using mutagens such as ethyl methane sulphonate, ultraviolet irradiation, N-methyl-N'-nitro-N-nitrosoguanidine, in particular ethyl methane sulphonate to mutate protease genes such as cpyl + sxal\ sxa2+, isp6+ dpa+ yspl+ in particular cpy1+<br><br><br>
5. A process for the preparation of yeast strain characterized in deficient in protease enzyme substantially as herein described with reference to examples accompanying this specification.<br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLUFic3RyYWN0LSgxOS0wNS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Abstract-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">469-del-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLUNsYWltcy0oMTktMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Claims-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">469-del-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Correspondence-Others-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">469-del-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">469-del-2003-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE5LTA1LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Description (Complete)-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">469-del-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">469-del-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">469-del-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLUZvcm0tMi0oMTktMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Form-2-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">469-del-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLUZvcm0tMy0oMTktMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Form-3-(19-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LWRlbC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">469-del-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDY5LURFTC0yMDAzLVBldGl0aW9uLTEzNy0oMTktMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">469-DEL-2003-Petition-137-(19-05-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234498-method-for-modifying-yield-and-or-height-of-a-plant-comprising-modifying-expression-of-a-nucleic-acid-encoding-map-protein.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234500-a-method-of-controlling-a-communication-device.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234499</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>469/DEL/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>02-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Mar-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RAFI MARG, NEW DELHI-110001, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PARUL AGGARWAL</td>
											<td>MICROBIAL TECHNOLOGY,CHANDIGARH-160036.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ANAND KUMAR BACHHAWAT</td>
											<td>MICROBIAL TECHNOLOGY, CHANDIGARH-160036</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 001/16</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234499-a-process-for-the-preparation-of-yeast-strain-characterized-in-protease-enzyme by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:15 GMT -->
</html>
