<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234408-cephalosporin-acylase-mutant-and-method-for-preparing-7-aca-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:14:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234408:CEPHALOSPORIN ACYLASE MUTANT AND METHOD FOR PREPARING 7-ACA USING SAME</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CEPHALOSPORIN ACYLASE MUTANT AND METHOD FOR PREPARING 7-ACA USING SAME</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to a mutant of a cephalosporin C (CPC) acylase composed of an a-subunit according to SEQ ID NO: 4 and a P-subunit according to SEQ ID NO: 5, or a functionally equivalent CPC acylase derivative thereof, characterized in that at least one amino acid selected from the group consisting of Vail2la, Glyl39a and Phel69a of CPC acylase a-subunit of SEQ ID NO: 4 and PheSSp, His70p, Ile75p, Ilel76p and Ser471p of CPC acylase p-subunit of SEQ ID NO: 5 is replaced by another amini.- acid, wherein the CPC acylase mutant or the functionally equivalent CPC acylase derivative thereof catalyzes the conversion of a compound of formula II from a compound of formula I: wherein, R is acetoxy (-OCOCH3), hydroxy (-0H), hydrogen (-H) Â« a salt thereof.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
CEPHALOSPORIN C ACYLASE MUTANT AND METHOD FOR PREPARING 7-ACA USING SAME<br>
Field of the Invention<br>
The present invention relates to a cephalosporin C (CPC) acylase mutant polypeptide derived from encoding said polypeptide; an expression vector containing said gene; a microorganism transformed with said expression vector; a method for preparing said CPC acylase mutant; and a method for preparing 7-ACA using said CPC acylase mutant.<br>
Baclcground of the Invention<br>
Cephalosporin C (hereinafter, referred to as "CPC") is one of P-lactam family antibiotics produced by filamentous fungi such as Acremonium chrysogenum. CPC shows antibiotic activity against Gram-negative bacteria by hindering cell wall synthesis, but it is not active enough against the growth of Gram-negative bacteria. Accordingly, CPC has been mainly used for preparing intermediates for the production of semi-synthetic cephalosporin antibiotics. In particular, 7-aminoacephalosporanic acid (hereinafter, referred to as "7-ACA") prepared by removing the D-a-aminoadipoyl side chain from CPC has been used for the production of most semi-synthetic cephalosporin antibiotics that account over 40% share of the world antibiotics market.<br>
There are two known methods for preparing 7-ACA from CPC, chemical and enzymatic methods. The chemical method of synthesizing 7-ACA from CPC uses silyl protecting groups for both amine and carboxyl groups and gives a yield of over 90%. However, this method is complicated, uneconomical and environmentally unfavorable. Therefore, the chemical method has been replaced by an enzymatic method for preparing 7-ACA which is regarded as an environmentally acceptable method.<br>
A two-step enzymatic method widely used commercially at present comprises the two steps of converting CPC into glutaryl-7-aminocephalosporanic acid (hereinafter, referred to as "GL-7-ACA") by D-amino acid oxidase (hereinafter, referred to as "DAG") (the first step) and GL-7-ACA into 7-ACA by GL-7-ACA acylase (the second step) (see Fig. 1). However, this method gives a lower yield of 7-ACA than the chemical method.<br>
^<br><br>
due to a large quantity of by-products generated by the. reaction of hydrogen peroxide produced in the first step with the DAO substrate or the reaction product. Therefore, there has been a need to develop an efficient one-step enzymatic method for directly converting CPC into 7-ACA using a modified CPC acylase which is capable of breaking an amide linkage at the 7'*' position of CPC and removing the aminoadipoyl side chain.<br>
CPC acylase (also called together with CPC amidase or CPC amidohydrolase) active toward CPC has been found in several microorganisms, such as Pseudomonas sp., Bacillus megaterium, Aeromonas sp., Arthrobacter viscous etc., and some CPC acylase genes have been cloned and sequenced: Pseudomonas sp. SE83 derived acyll gene (Matsuda et al., J. Bacteriol. 169: 5821-5829, 1987); Pseudomonas sp. N176 derived CPC acylase gene (USP 5,192,678); Pseudomonas sp. V22 derived CPC acylase gene (Aramori et al., J. Ferment. Bioeng, 72: 232-243, 1991); Pseudomonas vesicularis B965 derived CPC amidohydrolase gene (USP 6,297,032); Bacillus megaterium derived CPC amidase gene (USP 5,229,274); and Pseudomonas sp. 130 derived CPC acylase gene (Li et al., Eur. J. Biochem. 262: 713-719, 1999). However, these CPC acylases are not hydrolytically active enough to cleave the amide linkage at the 7* position of CPC, and thus, it is not suitable for a one-step en2ymatic process for preparing 7-ACA from CPC.<br>
Several genetic engineering studies have been attempted to increase the enzyme activity of CPC acylase toward CPC. For example, Fujisawa Pharmaceutical Co. (Japan) developed a CPC acylase mutant derived from Pseudomonas sp. N176 which shows about 2.5-fold higher specific activity toward CPC than that of a wild-type (USP 5,804,429; USP 5,336,613, Japanese Patent Publication No. 1995-222587; Japanese Patent Publication No. 1996-098686; and Japanese Patent Publication No. 1996-205864). However, such a mutant still has insufficient specific activity toward CPC to directly produce 7-ACA from CPC and exhibits end-product inhibition by 7-ACA. Therefore, it can't be used practically in the direct conversion of CPC to 7-ACA.<br>
For the purpose of developing a CPC acylase mutant applicable to a one-step enzymatic method for preparing 7-ACA, the tertiary structure of GL-7-ACA acylase has been investigated by the present inventors, who have identified for the first time the tertiary structures of apoenzyme (Kim, et al.. Structure 8: 1059-1068, 2000) and CAD-GL-7-ACA complex (Kim, et al., Chem. Biol. 8: 1253-1264, 2001; Kim, et al., J. Biol. Chem. 276: 48376-48381, 2001) associated with GL-7-ACA acylase derived from Pseudomonas sp. KAC-<br>
J&gt;<br><br>
1 (Kim, et al., Biotech. Lett. 23: 1067-1071, 2001; hereinafter, referred to as "CAD") (see Fig. 2). The structures of the two CAD binary complexes suggest that the most extensive interactions between the GL-7-ACA acylase and a substrate took place in the glutaryl moiety of GL-7-ACA. Therefore, it is suggested that the hydrophilic and hydrophobic interactions between the side-chain of substrate and its binding pocket are the dominating factors in recognizing the substrate in GL-7-ACA acylase. When the chemical structures of CPC having extremely lower substrate-binding affinity and GL-7-ACA are compared, their P-lactam structures are the same, but there are some differences in the side chain (see Fig. 3). Namely, unlike the side chain of GL-7-ACA which is composed of glutaric acid, that of CPC is a D-a-aminoadipic acid. Therefore, the modeling of the structure of CAD-CPC complex suggests that the glutaric acid side chain of GL-7-ACA and the binding-related key residues of CAD are sterically crowded with respect to the carboxyl and D-form amino groups at the terminal end of D-a-aminoadipic acid side chain (see Fig. 4). Thus, if an enough space for accommodating the CPC side chain (that contains a carbon backbone greater than that of the GL-7-ACA side chain and the D-amino group) can be secured in the substrate binding site of CAD, it is considered that the specific activity of GL-7-ACA for CPC can be increased. Accordingly, the structural analysis about the "enzyme-substrate" complex of CAD may provide important information for developing a CPC acylase mutant having an increased specific activity to CPC by introducing the mutations at the active site of GL-7-ACA acylase derived from Pseudomonas sp. that show relatively low substrate-binding affinity.<br>
The present inventors have therefore endeavored to develop a CPC acylase mutant having irnproyedjreactivity to CPC which can be used in a one-<br>
^ep enzymatic method for preparing 7-ACA from CPC. Summary of the Invention<br>
Accordingly, it is an object of the present invention to provide a CPC acylase mutant having improved reactivity to CPC which can be advantageously used in converting CPC into 7-ACA and a functionally equivalent derivative thereof<br>
Another object of the present invention is to provide a nucleotide sequence encoding said CPC acylase mutant and a functionally equivalent derivative thereof<br>
^<br><br>
A further object of the present invention is to provide an expression vector comprising said nucleotide sequence and a microorganism transformed with said nucleotide sequence or said expression vector.<br>
A further object of the present invention is to provide a method for preparing a CPC acylase mutant using said transformed microorganism.<br>
A further object of the present invention is to provide a method for preparing 7-ACA or a salt thereof from CPC using said CPC acylase mutant, a composition containing said CPC acylase mutant or the processed CPC acylase mutant.<br>
In accordance with one aspect of the present invention, there is provided a CPC acylase mutant or a functionally equivalent derivative thereof, characterized in that at least one of the amino acids selected from the group consisting of Vall2ia, Glyl39a and Phel69a of CPC acylase a-subunit of SEQ ID NO: 4 and Met31p, Phe58p, HisTOp, Ile75j3, Ilel76P and Ser471p of CPC acylase P-subunit of SEQ ID NO: 5 is replaced by another amino acid.<br>
Among the CPC acylase mutants of the present invention, preferred are those, wherein:<br>
Vall21a is replaced by alanine;<br>
Glyl39a is replaced by serine;<br>
Phel69a is replaced by tyrosine;<br>
Met3ip is replaced by leucine;<br>
Phe58p is replaced by alanine, methionine, leucine, valine, cysteine or asparagine;<br>
His70P is replaced by serine or leucine;<br>
Ile75p is replaced by threonine;<br>
Ilel76P is replaced by valine; or<br>
Ser47ip is replaced by cysteine.<br>
Brief Description of the Drawings<br>
The above and other objects and features of the present invention will become apparent from the following description of the invention, when taken in conjunction with the accompanying drawings which respectively show;<br>
s<br><br>
Fig. 1: schematic procedures of one-step and two-step enzymatic methods for preparing 7-ACA from CPC;<br>
Fig. 2: the tertiary structure of Pseudomonas sp. KAC-1 derived CAD-GL-7-ACA complex (A) and a general scheme for the binding residues adjacent to its active site (B);<br>
Fig. 3: the binding pattern of Pseudomonas sp. KAC-1 derived CAD with GL-7-ACA;<br>
Fig. 4: modeling of Pseudomonas sp. KAC-1 derived CAD-CPC complex;<br>
Fig. 5: a schematic representation of the procedure for preparing the inventive CPC acylase mutants;<br>
Fig. 6: comparison of the end-product inhibition by 7-ACA observed for<br>
the wild-type CPC acylase (-â¢-, Sem), F169aY/M3ipL/F58pM/I176pV<br>
fourfold	CPC	acylase	mutant	(H70)	and<br>
F169aY/M3ipL/F58pM/H70pS/I176pV fivefold CPC acylase mutant (-0-, ml 176);<br>
Fig. 7: comparison of the reactivities toward CPC of F169aY/M3ipL/F58pM/I176pV fourfold CPC acylase mutant (-0-. H70), V121aA/F169aY/M3ipL/F58pM/I176pV fivefold CPC acylase mutant (-0-, #213), G139aS/F169aY/M3ipL/F58pM/I176pV fivefold CPC acylase mutant (-<br>
â 	-, #120) and V121aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold CPC<br>
acylase mutant (-â¢-, TnS5);<br>
Fig. 8: comparison of the reactivities toward CPC of<br>
V121aA/G139aS/F169aY/F58pN/I176pV fivefold CPC acylase mutant (-V-,<br>
mF58), V121aA/G139aS/F169aY/F58pC/I176pV fivefold CPC acylase mutant<br>
(-â -), V121aA/G139aS/F169aY/M3ipL/F58pN/I176pV sixfold CPC acylase<br>
mutant (-0-, TnS5ap), V121aA/G139aS/F169aY/M3ipL/F58pC/I176pV<br>
sixfold	CPC	acylase	mutant	(-â¼-)	and<br>
V121aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold CPC acylase mutant (-â¢-,TnS5);<br>
Fig. 9: comparison of the reactivities toward CPC of V121aA/G139aS/F58pN/I176pV fourfold CPC acylase mutant (-T-, F169), V121(xA/G139aS/F58pN/I75pT/I176pV fivefold CPC acylase mutant (-V-, #59), V121aA/G139aS/F58pN/I176pV/S471pC fivefold CPC acylase mutant (-<br>
â 	-, #76), V121aA/Gl39aS/F169aY/F58pN/I176pV fivefold CPC acylase<br>
mutant (-0-, mF58) and V121aA/G139aS/F169aY/M31pL/F58pN/I176pV<br>
sixfold CPC acylase mutant (-â¢-, TnS5aP);<br>
t<br><br>
Fig. 10a and 10b: comparison of the reactivities to CPC (A) and the end-<br>
product	inhibition	by	7-ACA	(B)	observed	for<br>
V121aA/G139aS/F58pN/I75pT/I176pV/S47ipC sixfold CPC acylase mutant (-<br>
â¼ -, SI 2) with those observed for<br>
V121aA/G139aS/F169aY/M3ipL/F58pN/I176pV sixfold CPC acylase mutant<br>
(-â¢-, TnS5aP) and V121aA/G139aS/F58pN/I176pV/S471pC sixfold CPC<br>
acylase mutant {-0-, #76);<br>
Figs. I la to lie: the molecular weight of Pseudomonas sp. SE83 derived wild-type CPC acylase;<br>
11a: SDS-PAGE (M: size marker; 1: cell-free extract; 2: the purified enzyme)<br>
lib: non-denaturing PAGE (M: size marker; 1: the purified enzyme)<br>
lie: MALDI-TOF mass spectrophotometry<br>
Fig. 12: the results of gel electrophoresis and Comassie blue staining of a crude enzyme isolated from E. coli BL21(DE3) culture solution comprising pET29-TnS5 plasmid;<br>
M: standard size marker,	1: 0% lactose, 48 hr cultivation<br>
2: 0.02% lactose, 48 hr cultivation,     3: 0.2% lactose, 48 hr cultivation<br>
4: 2% lactose, 48 hr cultivation,	5: 2% lactose, 72 hr cultivation<br>
Fig. 13: comparison of the conversion rates of CPC into 7-ACA observed for the wild-type CPC acylase, TnS5 and S12 CPC acylase mutants;<br>
Fig. 14: the HPLC analysis result for determining 7-ACA produced from CPC by the action of the inventive S12 CPC acylase mutant.<br>
Detailed Description of the Invention<br>
As used therein, the term "increase in the reactivity toward CPC" means increased specific activity toward CPC and/or decreased end-product inhibition by 7-ACA.<br>
The term "functionally equivalent derivative" as used herein means a CPC acylase derivative retaining the same fiinctional property as the inventive CPC acylase mutant. Namely, the functionally equivalent derivative includes all possible variants such as a native, synthetic or recombinant polypeptide which may be modified, i.e., by sequence mutation, deletion, insertion, substitution, inversion of single or several nucleotides and a combination thereof, capable of flinctioning as a CPC acylase mutant.<br>
The term "processed CPC acylase mutant" as used herein means a CPC<br>
;<br><br>
acylase mutant in an immobilized state not but in a free state. A CPC acylase mutant may be immobilized to a conventional carrier generally employed in the art by a conventional method such as a covalent bond, ionic bond, hydrophilic bond, physical bond, microencapsulation and so on. Further, a processed CPC acylase mutant can be prepared by a whole cell immobilization method that immobilizes a microorganism producing the CPC acylase mutant as is without further purification.<br>
The term "GL-7-ACA acylase" as used herein generally means an enzyme which is capable of converting GL-7-ACA into 7-ACA. The term "CPC acylase" as used herein means an enzyme having specific activity toward CPC among GL-7-ACA acylases, which directly produces 7-ACA from CPC by cleaving the amide linkage at the 7* position of CPC to remove the aminoadipoyl side chain. The term "CAD" as used herein means Pseudomonas sp. KAC-1 derived GL-7-ACA acylase.<br>
The present invention provides an amino acid sequence of a CPC acylase mutant having an improved reactivity to CPC which is derived from Pseudomonas sp. SE83 and a fiinctionally equivalent derivative thereof; a nucleotide sequence encoding said CPC acylase mutant and fiinctionally equivalent derivative thereof<br>
CPC acylase gene ("acyll gene") derived from Pseudomonas sp. SE83 has been used as a starting gene for developing a CPC acylase mutant in the present invention, wherein the acyll gene is newly synthesized using a DNA synthesizer based on the nucleotide sequence of CPC acylase indentified by Matsuda et al. {J. Bacterial. 169: 5821-5826, 1987; GenBank Ml8278). To increase the efficiency of protein synthesis using E. coli, the previously known amino acid sequence of Acyll is modified such that it is compatible with the codons preferred used in the E. co/z-mediated protein synthesis. For example, in the previously known amino acid sequence encoded by the acyll gene, the only codon for phenylalanine is TTC and a codon for arginine is predominantly CGC or CGG, and therefore, the amino acid sequence encoded by the acyll gene synthesized in the present invention is carried out using TTT and. CGT codons for phenylalanine and arginine, respectively. The acyll gene synthesized in the present invention consists of 2,325 base pairs having the nucleotide sequence of SEQ ID NO: 1. In SEQ ID NO: 1, the open reading frame correspondmg to base numbers 1 to 2,322 (2,323-2,325: termination codon) is a fiiU-length protein encoding region and the predicted amino acid<br><br>
sequence derived therefrom is shown in SEQ ID NO: 2 which consists of 774 amino acids. The amino acid sequence of SEQ ID NO: 2 encoded by the acyll gene synthesized in the present invention is identical to that of the previously well-known CPC acylase, but 18% of the base pairs in the nucleotide sequence of SEQ ID NO: I is different from those of the existing CPC acylase gene.<br>
The amino acid sequences deduced from most GL-7-ACA acylase (or CPC acylase) genes are composed of a-subunit, a spacer peptide, and p-subunit, in that order. The wild-type CPC acylase derived from Pseudomonas sp. SE83 is generated in the form of an inactive single chain polypeptide having about 84 kDa in size after undergoing transcription and translation in a host cell. After then, twice self-digestions occur between the 230* and the 231*' amino acids, and the 239* and the 240* amino acids in the amino acid sequence of SEQ ID NO: 2, which results in removing the spacer peptide consisting of 9 amino acids, and separating into a 25 kDa a-subunit and an about 58 kDa p-subunit. One of the a-subunit separated above is coupled to one of the p-subunit through hydrophobic interactions, to form an about 83 kDa dimer having acylase activity. As is generally known, the first amino acid, methionine codon needed for translation initiation during a protein synthesis in a prokaryote, is removed after the translation. Thus, an active form of the wild-type CPC acylase used in the present invention is composed of a a-subunit consisting of 229 amino acids (SEQ ID NO: 4) and a p-subunit consisting of 535 amino acids (SEQ ID NO: 5).<br>
If a proper self-digestion does not occur in a host cell after transcription and translation of a corresponding gene, GL-7-ACA acylase (or CPC acylase) is generated in the form of an inactive polypeptide. Therefore, the self-digestion efficiency in a host cell plays an important role in obtaining an active protein. It has also been reported that the self-digestion efficiency may be lowered by several causes such as over-production of acylase in a host cell or a mutation of a specific amino acid at a spacer peptide or a mature protein, resulting in the production of an inactive protein (Li, et al, Eur. J. Biochem. 262: 713-719, 1999; Kim, et al., J. Biol Chem, 276: 48376-48381, 2001). Therefore, when the improvement of the reaction property of CPC acylase is intended, the question of whether a mutation to be introduced at a specific amino acid to increase the enzyme reactivity may affect to the self-digestion efficiency must be properly addressed. Namely, since the change of self-digestion efficiency of an inactive polypeptide generated in a host cell after transcription/translation of a CPC acylase gene is directly related to the productivity of an active CPC acylase, it is difficult to select a CPC acylase mutant having an improved<br><br>
reactivity toward CPC without knowing the reactivity characteristics of the purified active CPC acylase mutant.   To solve such problems, the present inventors have contrived a CPC acylase gene having the nucleotide sequence of SEQ ID NO: 3 based on the nucleotide sequence of SEQ ID NO: 1, which is capable of generating an active CPC acylase without the self-digestion step by individually synthesizing a- and P-subunits during the translation.   The amino acid sequence encoded by the CPC acylase gene of SEQ ID NO: 3 is identical to that of SEQ ID NO: 2 encoded by CPC aclyase gene acyll, except that the nucleotide sequence encoding the spacer peptide corresponding to the 231' to the 239"" region of SEQ ID NO: 2 is modified.   Namely, a translation stop codon is inserted behind the glycine codon encoding the last amino acid of the a-subunit; and a nucleotide sequence comprising a random nucleotide sequence including a restriction enzjmie site, a nucleotide sequence encoding a ribosome binding site represented as -AGGA-, another random nucleotide sequence, and a translation initiation codon (methionine), arranged in a consecutively fashion, is inserted in front of the serine codon encoding the first amino acid of P-subunit. The present inventors have contrived to form the CPC acylase gene in one operon structure by making each of the nucleotide sequences encoding the a-subunit and p-subunit to individually maintain its own structural gene.   The CPC acylase gene having the nucleotide sequence of SEQ ID NO: 3 and individually expressing a-subunit and P-subunit is designated sem.   Therefore, it can be expected that if the inventive CPC acylase gene sem is inserted into an appropriate expression vector and transformed to a host cell, the a-subunit and p-subunit would be individually synthesized and coupled to each other within a host cell, to obtain the dimeric form of active CPC acylase.   Comparison of the characteristic features of CPC acylase purified from the E. coli transformant containing the sem gene of SEQ ID NO: 3 with that purified from the E. coli transformant containing the acyll gene of SEQ ID NO: 1 actually shows that the characteristic features of sem derived CPC acylase, e.g., molecular weight, specific activity, substrate specificity, optimal reaction temperature and pH, are identical to those of acyll derived CPC acylase.    Therefore, the separate expression of the a-subunit and P-subunit may remove the cause responsible for the self-digestion efiiciency of inactive polypeptide, making it easy to detect the change in the characteristic features induced by a mutation introduced at a specific amino acid residue.   Thus, in the present invention the acyll gene of SEQ ID NO: 3 is employed as a starting DNA for developing a CPC acylase mutant.<br>
/O<br><br>
The literatural nomenclature of the amino acid sequence of Pseudomonas sp. SE83 derived CPC acylase (Acyll, SEQ ID NO: 2) employed in the present invention has adopted a consecutively numbering system starting from the methinine codon at the N-terminal end of the a-subunit as the first, to the alanine codon at the C-terminal end of the p-subunit as the last. However, since the first amino acid, methinine, and the spacer peptide are removed in a host cell after transcription and translation of a CPC aclyase gene, they do not exist in the amino acid sequence of a mature protein. Namely, the amino acid sequence of an active protein consists of the amino acid sequence of the a-subunit and the amino acid sequence of the P-subunit. Accordingly, the numbering system employed in the present invention is to consecutively number each of the a- and p-subunits existing in the active form of mature CPC acylase. Naming of a sp'ecific residue of the amino acid sequence of CPC acylase is carried out by following the conventional nomenclature as follows: the threonine, the 1** residue of a-subunit in the amino acid sequence of SEQ ID NO: 4, is designated Thrla; and the serine, the 1^ residue of P-subunit in the amino acid sequence of SEQ ID NO: 5, Serip. Accordingly, Thrla and Serip mean the 2"** residue thereonine and the 240* residue serine in the amino acid sequence of SEQ ID NO: 2, respectively.<br>
The naming of a specific residue introduced with a mutation among the amino acid sequence of CPC acylase is also carried out following the conventional nomenclature as follows: the mutated residue replacing the 169* residue phenylalanine of a-subunit by a tyrosine is represented by F169aY and the mutated residue replacing the 176* residue isoleucine of P-subunit by a valine, 1176pV.<br>
The CPC acylase mutant of the present invention has been prepared using the CPC acylase gene sent of SEQ ID NO: 3 as a starting DNA using conventional point mutation methods and genetic engineering techniques as follows (see Fig. 5).<br>
First, in order to increase the specific activity of CPC acylase mutant to CPC, the amino acid residue to be mutated has been selected from the amino acid sequence of Acyll, based on the result of virtual mutagenesis in the tertiary structural modeling of CAD-CPC complex using a computer program. An artificial oligonucleotide including a nucleotide sequence corresponding to the mutated amino acid sequence selected above has been synthesized and subjected to PCR for inducing a site-directed mutagenesis at the CPC acylase gene.   The<br>
II<br><br>
mutated gene has been transformed into a microorganism and the transformant was cultivated under a suitable condition to produce the CPC acylase. Then, the enzyme activities of CPC acylases thus produced have been measured to select a CPC acylase mutant having an improved reactivity to CPC. The nucleotide sequence of gene encoding the selected CPC acylase mutant has been analyzed. As a result, it has been found that the mutation introduced at one or more of amino acid residues selected from the group consisting of Phel69a, Met3ip, Phe58p, His70p and Ilel76p in the Acyll amino acid sequence of SEQ ID NOs: 4 and 5 enhances the CPC acylase reactivity to CPC.<br>
To further increase the reactivity of CPC acylase mutant to CPC, the selected CPC acylase mutant gene has been subjected to error-prone PCR to induce a point mutation at a random site. The mutated gene has been transformed to a host cell to construct a mutant library and the host cell containing the CPC acylase mutant having increased reactivity to CPC has been screened from the mutant library. The enzyme activities of CPC acylase mutants thus produced have been measured to select a CPC acylase mutant having an improved CPC acylase reactivity and the nucleotide sequence of the gene encoding the CPC acylase mutant selected above has been analyzed. As a result, it has been found that the mutation introduced at one or more of amino acid residues selected from the group consisting of Vall21a, Glyl39a, IleTSP and Ser47ip in the Acyll amino acid sequence of SEQ ID NOs: 4 and 5 increases the efficiency of CPC acylase convertmg CPC into 7-ACA.<br>
Therefore, the CPC acylase mutant of the present invention preferably has an amino acid sequence which is characterized in that at least one amino acid selected from the group consisting of Vall21a, Glyl39a and Phel69a of CPC acylase a-subunit of SEQ ID NO: 4 and Met31p, PheSSp, His70p, Ile75p, Ilel76p and Ser47ip of CPC acylase p-subunit of SEQ ID NO: 5 is replaced by another amino acid.<br>
In the above amino acid substitution for increasing the CPC acylase reactivity to CPC, it is preferable to replace the specified amino acid residue with another amino acid selected from the group consisting of glycine, alanine, valine, leucine, serine, threonine, cysteine, isoleucine, methionine, proline, phenylalanine, tyrosine, tryptophan, aspartic acid, glutamic acid, asparagine, glutamine, histidine, lysine and arginine. More preferably, VaI121a is replaced by alanine; Glyl39a, by serine; Phel69a, by tyrosine; Met3ip, by leucine; PheSSp, by asparagine, methionine, alanine, leucine, valine or cysteine; His70p, by serine or leucine; Ile75P, by thereonine; Ilel76P, by valnine; and Ser471p, by<br>
/^<br><br>
cysteine.<br>
In a preferred embodiment of the present invention, a CPC acylase mutant gene designated S12 having Vall21a replaced by alanine; Glyl39a, by serine; PheSSP, by asparagine; IleTSp, by thereonine; Ilel76p, by vahiine; and Ser471p, by cysteine has been prepared. The S12 mutant gene has an increased specific activity to CPC but shows decreased end-product inhibition by 7-ACA. The S12 mutant gene has the nucleotide sequence of SEQ ID NO: 6 and the active CPC acylase mutant encoded by the S12 mutant gene consists of the a-subunit of SEQ ID NO: 7 and the p-subunit of SEQ ID NO: 8.<br>
The present invention also includes a functionally equivalent derivative of the CPC acylase mutant. A functionally equivalent derivative in the present invention means a CPC acylase derivative retaining the general feature of the inventive CPC acylase mutant. Namely, the functionally equivalent derivative includes all possible variants such as native, synthetic or recombinant polypeptides modified by sequence mutation, deletion, insertion, substitution, inversion of single or several nucleotides and a combination thereof, which are capable of fimctioning as a CPC acylase mutant.<br>
Further, the present invention includes the polynucleotide encoding said CPC acylase mutant having an improved reactivity to CPC or a functionally equivalent derivative thereof Preferably, the polynucleotide has the nucleotide sequence of SEQ ID NO: 6. The functionally equivalent derivative in the present invention means a polynucleotide and its derivative retaining the crucial functional property of the polynucleotide(s) encoding the a-subunit and/or P-subunit of the CPC acylase mutant. Therefore, the present invention includes within its scope not only the polynucleotide encoding the CPC acylase mutant which comprises the a-subunit linked to the P-subunit with a specific spacer peptide but also a polynucleotide encoding both the a- and p-subunits of the CPC acylase mutant without a spacer peptide. Furthermore, a polynucleotide encoding only the a-subunit of CPC acylase mutant and a polynucleotide encoding only the P-subunit of CPC acylase also fall within the scope of the present invention.<br>
The present invention also includes, in its scope, a polynucleotide comprising a nucleotide sequence of the inventive CPC acylase mutant gene or a mucleotide sequence deduced fi"om the CPC acylase mutant amino acid sequence as well as a nucleotide sequence generated through the codon degeneracy of genetic code in the nucleotide sequence of the inventive CPC acylase mutant gene.<br>
/3<br><br>
Further, the present invention includes a recombinant expression vector comprising the inventive CPC acylase mutant encoding gene. Said recombinant expression vector may be prepared by inserting a DNA fragment containing the CPC acylase mutant gene and a suitable transcription/translation regulatory sequence into an appropriate expression vector according to a conventional method (Sambrook, et al., Molecular Cloning, Cold Spring Harbor Laboratory, 1989). Any expression vector capable of expressing a foreign gene in a host cell can be employed in the present invention. For example, preferable expression vectors include, but are not to limited to, a plasmid, a phage vector and an integration vector.<br>
The recombinant expression vector prepared above may be introduced into a suitable host cell according to a conventional transformation method (Sambrook, et al., the supra). Bacteria, actinomyces, yeast, fungi, an animal cell, an insect cell or a plant cell can be employed as a host cell suitable for the expression of a recombinant DNA, Among these host cells, preferred are bacteria such as E. coli or Bacillus sp.; actinomyces such as Streptomyces sp.; yeast such as Scaaharomyces sp., Humicola sp. or Pichia sp.; fungi such as Aspergillus sp. or Trichoderma sp.; and a CPC producing microorganism such as Cephalosporium sp. or Acremonium sp.<br>
In a preferred embodiment of the present invention, E. coli BL21(DE3) is transformed with the recombinant expression vector comprising the inventive CPC acylase mutant gene S12 (SEQ ID NO: 6) to obtain an E. coli transformant designated E. coli BL21(DE3)/pET-S12 which was deposited on July 30, 2003 with the Korean Collection for Type Cultures (KCTC) (Address: Korea Research Institute of Bioscience and Biotechnology (KRIBB), #52, Oun-dong, Yusong-ku, Taejon, 305-333, Republic of Korea) under the accession number KCTC 10503BP, in accordance with the terms of Budapest Treaty on the Intemational Recognition of the Deposit of Microorganism for the Purpose of Patent Procedure.<br>
The CPC acylase mutant may be prepared by culturing said transformed microorganism introduced with the expression vector comprising the CPC acylase mutant encoding gene or functionally equivalent derivative thereof in a suitable medium under a proper condition.<br>
Further, the CPC acylase mutant may be also prepared by culturing a microorganism transformed with the expression vector comprising only an a-subunit encoding gene of CPC acylase mutant or a functionally equivalent derivative thereof and a microorganism transformed with the expression vector<br>
I'l<br><br>
comprising only a p-subunit encoding gene of CPC acylase mutant or a functionally equivalent derivative thereof in a suitable medium under a proper condition, separately producing the a-subunit protein and p-subunit protein in respective transformed microorganisms and mixing the two subunit proteins purified therefrom in vitro.<br>
It is possible to use the CPC acylase mutant thus prepared as is or in a purified form to produce the desired product, 7-ACA, in a one-step enzymatic method. The CPC acylase mutant may be purified by a conventional protein purification method using various column chromatographic techniques based on the characteristics of the CPC acylase, with or without minor modifications in accordance with specific purposes. Further, it is also possible to purify the CPC acylase mutant by affinity chromatography, taking advantage of the binding affinity such as the binding affinity of a histidine peptide to a nickel column or the binding affinity of a cellulose-binding domain (CBD) with a cellulose.<br>
The inventive CPC acylase mutant may be also used in an immobilized form. The immobilization of CPC acylase mutant may be conducted by a conventional method using a carrier, e.g., a natural polymer such as cellulose, starch, dextran and agarose; a synthetic polymer such as polyaciylamide, polyacrylate, polymetacrylate and Eupergit C; or a mineral such as silica, bentonite and metal. The CPC acylase mutant may be coupled to said carrier by a conventional immobilization method such as a covalent bond, ionic bond, hydrophilic bond, physical absorption or microencapsulation. In addition, it is also capable to immobilize the CPC acylase mutant by forming a covalent bond between the carrier and the enzyme via the action of glutaraldehyde or cyanogen bromide. Preferably, the microorganism cell producing the CPC acylase mutant may be immobilized as is by a whole cell immobilization method without purifying the CPC acylase mutant. In order to increase the reactivity of CPC acylase mutant produced by a microorganism, it is also possible to make a hole at the cell wall or apply a cell surface expression technique thereto.<br>
Further, the present invention provides a method for preparing 7-ACA of formula II or a salt thereof from CPC of formula I using said CPC acylase mutant.<br>
In the following formula, R is acetoxy (-OCOCH3), hydroxy (-0H), hydrogen (-H) or a salt thereof Preferably, R is acetoxy (-OCHCH3) and a salt is an alkali metal salt such as a sodium, potassium or lithium salt.<br>
ir<br><br><formula><br><br><br>
HOOCâCHâ(CH2)3-<br>
NH2<br>
(I)<br><br><formula><br>
H2N-<br><br>
-N\^<br><br>
CH2-R<br><br>
(n)<br><br>
COOH<br>
7-ACA (II) may be produced by contacting CPC (I) with the inventive CPC acylase mutant, wherein the CPC acylase may be used in the form of a culture solution of the CPC acylase mutant producing strain, or in the form of an composition comprising the CPC acylase mutant, the purified free enzyme itself or an immobilized form of the enzyme. Preferably, the contact reaction of the CPC acylase mutant with CPC (I) may be carried out in an aqueous solution. The preferable concentration of CPC (I) ranges from 1 to 500 mM; the amount of added CPC acylase mutant, from 0.1 to 100 U/ra^; pH of the reaction mixture, from 7 to 10; the reaction time, from 0.1 to 24 hr; and the reaction temperature, 4 to 40*0. 7-ACA (II) prepared by above enzyme reaction can be isolated and purified from the reaction mixture by conventional methods.<br>
Further, it is possible to produce 7-ACA (II) by contacting the inventive CPC acylase mutant with CPC (I) in vivo. In particular, 7-ACA (II) may be produced fay the steps of introducing the CPC acylase mutant encoding gene or a fiinctionally equivalent derivative thereof into a microorganism having a biosynthetic activity of CPC such as Acremonium crysozenum; culturing the transfromant in a suitable medium under a proper condition; and spontaneously contacting the CPC acylase mutant with the CPC (I) biosynthesized in said transformant.<br>
The present invention is further defined in the following Examples. It should be understood that these Examples, while indicating preferred embodiments of the invention, are given by way of illustration only.   From the<br>
/6<br><br>
above discussion and these Examples, one skilled in the art can ascertain the essential characteristics of this invention, and without departing from the spirit and scope thereof, can make various changes and modifications of the invention to adapt it to various usage and conditions.<br>
Example 1; Preparation of CPC acylase mutant having an improved<br>
reactivity to CPC<br> Preparation  of CPC acylase mutant based on  the structural<br>
information<br> Selection of Acyll amino acid residue to be mutated based on the<br>
structural information<br>
The tertiary structure of CAD-GL-7-ACA complex has been determined lately (Kim, et al., Chem. Biol. 8:1253-1264, 2001; Kim, et al., /. Biol. Chem. 276: 48376-48381, 2001). As illustrated in Fig. 3, Arg57p, Tyr33p, Tyrl49a and Argl55a existing in the substrate-binding site of CAD directly form hydrogen bonds with GL-7-ACA, and Phel77p, Leu24p and Val70p bind to GL-7-ACA via hydrophobic interactions. In addition, although GlnSOp makes no direct interaction with GL-7-ACA, it forms hydrogen bonds with Arg57P and Tyrl49a to arrange these residues at suitable positions for a catalytic reaction. Meanwhile, in the coupling of CAD and GL-7-ACA, a beta-lactam core of GL-7-ACA comprising an acetoxy group, a six-membered ring and a four-membered beta-lactam ring does not significantly affect on the binding with active site residues. If anything, a glutaric acid region as GL-7-ACA side chain has the largest effect on a substrate binding by precisely coupling with Arg57p, Tyr33p, Tyrl49a and Phel77p (Figs. 2 and 3). In addition, it has been presumed that Leu24p and Val70P play a role in stimulating a catalytic reaction of Serlp by making a substrate occupied to a suitable position.<br>
Meanwhile, as a result of overlapping the tertiary structure of CAD-GL-7-ACA complex with that of CAD-CPC complex, key residues of CAD such as Arg57P, Tyr33p, Phel77p and Tyrl49a involving in the coupling with a glutaric acid side chain of GL-7-ACA collide with the carboxyl group and the D-form amino group at a terminal region of D-a-aminoadipic acid side chain of CPC (Fig. 4). From these modeling results, if a space enough for accommodating the CPC side chain (namely, said a carbon backbone and a D-amino group additionally existed at the CPC side chain as compared with the GL-7-ACA side chain) within a substrate-binding site of CAD can be secured, it might be<br><br>
expected to increase the specific activity of GL-7-ACA acylase for CPC.<br>
Meanwhile, it has been presumed from the result of comparing structures of CAD, penicillin G acylase and Pseudomonas sp. derived GL-7-ACA acylase that Pseudomonas sp. derived GL-7-ACA acylase shows similar substrate-binding and reaction patterns to the rest (Kim, et al., Chem. Biol. 8: 1253-1264, 2001; Fritz-Wolf, et al.. Protein Sci. 11: 92-103, 2002). Further, it has been reported that while CAD has no activity to CPC, Pseudomonas sp. derived GL-7-ACA acylase (Acyll) shows about 5% level of acylase activity to CPC as compared with GL-7-ACA (Matsuda, et al., J. Bacterial. 169: 5815-5820, 1987). Thus, there has an advantage in developing a CPC acylase mutant based on Acyll having a constant level of enzyme activity to CPC rather than based on CAD having no enzyme activity to CPC. Accordingly, the present invention has employed Pseudomonas sp. SE83 derived CPC acylase gene {acyll) as a fundamental gene for developing a CPC acylase mutant.<br>
The present invention intended to prepare a CPC acylase mutant having a better specific activity for CPC than the wild-type Acyll by selecting an active site of Acyll based on the tertiary structure of CAD-CPC complex and a virtual mutagenesis information, and then, performing a site-directed mutagenesis to Acyll residues be involved in interfering with a CPC binding among the selected active site. Table 1 shows Acyll residues corresponding to active site resides of CAD subjected to site-directed mutagenesis.<br><table>
<br>
CAD residue	Acyll residue<br>
Tyr33p	Met31p<br>
Phe58p	Phe58p<br>
Val70P	His70p<br>
Phel77P	Ilel76p<br>
Tyrl49a	Phel69a<br>
Based on the results of the tertiary structural modeling of CAD-CPC complex, site-directed mutagenesis was conducted to Phe58p, Met3ip, Ilel76p and Phel69a residues of Acyll corresponding to Phe58p, Tyr33p, Phel77p and Tyrl49a residues of CAD that are presumed to seriously inhibit the binding of CPC side chain, and His70p residue of Acyll corresponding to Val70p residue of CAD which stimulates to catalytic reaction of Serlp active site (Fig. 5).<br>
/8<br><br><l-l-2> Preparation of pBCPC and pBSEM plasmids<br>
pBCPC plasmid was constructed by inserting the acyll structural gene DNA fragment of SEQ ID NO: 1 into Xhol/Xbal sites of pBC KS(+) vector (Stratagene, USA) as follows. First, the acyll structural gene was synthesized by using a DNA synthesizer based on the DNA nucleotide sequence of Pseudomonas sp. SE83 derived CPC acylase gene {acyll) and the amino acid sequence of protein Acyll encoded thereby (GenBank Accession No. Ml8278). At this time, the amino acid sequence encoded by the acyll gene was identical to that published in the literatures, but its nucleotide sequence has been synthesized to be made some modifications that the acyll gene may includes one or more preferred codons in E. coli.<br>
PCR was performed to introduce a recognition site of restriction enzyme and a ribosome-binding site into the acyll structural gene. PCR reaction solution (100 id) contained 10 ng of the synthesized acy/Zgene DNA, 50 pmol each of CPC-F primer (SEQ ID NO: 13) and CPC-R primer (SEQ ID NO: 14), 0.2 mM dNTP mixture, Taq buffer solution (5 mM KCl, 5 mM Tris-HCl, (pH 8.3), 1.5 mM MgCh), and 2.5 unit of ExTaq polymerase (Takara, Japan). Reactions were initiated by pre-denaturation for 5 min at 95*0 in a programmable thermal cycler (Peltier Thermal Cycler PTC-200; MJ Research, USA). PCR conditions consisted of 25 cycles of 1 min at 95 "C (denaturation), 30 sec at 58 Â°C (annealing), and 1 min 30 sec at 72Â°C (polymerization), with a final elongation of 10 min at llX!, (post-polymerization). After the PCR amplification, about 2.5 kb of PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pBC KS(+) vector DNA was digested with Xbal/Xhol and subjected to dephosphorylation with CIP, to obtain a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at \6X&gt; for 12 to 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 25 /ig/niC of chloramphenicol and cultured at 30 "C incubator for overnight to select a transformant. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed. Out of this, pBCPC plasmid containing the acyll structural gene of SEQ ID NO: 1 was prepared and the E. coli transformant containmg pBCPC plasmid was designated E. coli MC1061(pBCPC).<br>
n<br><br>
pBSEM plasmid was constructed by inserting the sem structural gene DNA fragment of SEQ ID NO: 3 into Xhol/Xbal sites of pBC KS(+) vector (Stratagene, USA). A series of PCR amplifications were performed as follows: pBCPC plasmid as a template and M13-R primer (SEQ ID NO: 11) and aORF-R primer (SEQ ID NO: 15) to obtain 0.8 kb of PCR product; pBCPC plasmid as a template and pORF-F primer (SEQ ID NO: 16) and Hindlll-R primer (SEQ ID NO: 18) to obtain about 0.96 kb of PCR product; and pBCPC plasmid as a template and Hindlll-F primer (SEQ ID NO: 17) and M13-F primer (SEQ ID NO: 12) to obtain 0.75 kb of PCR product. 0.8 kb, 0.96 kb and 0.75 kb of PCR products obtained above were mixed and subjected to PCR under the same PCR condition described above except that primers did not added, to obtain about 2.5 kb of PCR product which is ligated three PCR products in one. After then, about 2.5 kb of PCR product was subjected to PCR using T3 primer (SEQ ID NO: 9) and T7 primer (SEQ ID NO: 10) to amplify the sem gene DNA fragment of about 2.5 kb in size. After the PCR amplification, about 2.5 kb of PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pBC KS(+) vector DNA was digested with Xbal/Xhol and subjected to dephosphorylation with CIP to obtain a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at 16*0 for 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli stram was spread onto a LB agar plate containing 25 iiglmi of chloramphenicol and cultured at 30"C incubator for overnight to select a transformant. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed. As a result, pBSEM plasmid containing the sem structural gene of SEQ ID NO: 3 was prepared and the E. coli transformant containing pBSEM plasmid was designated E. coli MC1061(pBSEM).<br>
To compare the productivity of CPC acylase by pBCPC plasmid derived E. coli transformant with that by pBSEM plasmid derived E. coli transformant, a CPC acylase crude enzyme solution obtained from each E. coli transformant was prepared as follows. Each E. coli transformant was inoculated into 3 mC of a LB medium (1% Bacto-Tryptone, 0.5% Yeast Extract, 0.5% NaCl) containing 25 //g/m^ of chloramphenicol and cultured at 30*0, 200 rpm for 16 hr with vigorous shaking. 50 id oftheculturesolution was transformed to 50 mC of a new LB medium containing 25 //g/mC of chloramphenicol and fiirther cultured at 25*0, 200 rpm for 48 hr with vigorous shaking.   The culture<br>
JlÂ©<br><br>
solution was subjected to centrifugation at 4t;, 8,000 rpm for 10 min to separate a precipitate and the precipitate was washed twice with 0.1 M Tris-HCl buffer solution (pH 8.0). This precipitate was suspended in 5 niÂ£ of the same buffer solution and subjected to ultra-sonication at 4*0 for 10 min. And then, the suspension was subjected to centriftigation at 4"C, 15,000 rpm for 20 min, to separate a supematant which can be used as a CPC acylase crude enzyme solution.<br>
The enzyme activity of CPC acylase mutant to CPC was measured according to the method described by Park et al. with minor modifications as follows (Park, et al., Kor. J. Appl. Microbiol. Biotechnol 23: 559-564, 1995). A substrate solution was prepared by dissolving CPC (purity 14.2%; CJ Corp., Korea) in 0.1 M Tris-HCl buffer solution (pH 8.0) at a concentration of 20 mg/rofi and adjusting pH to 8 with 1 N NaOH. 20 fd of the CPC solution was mixed with 20 fd of the crude enzyme solution prepared above and the reaction mixture was incubated at 3lV for 5 min. After the reaction, the mixture (200 (d) of 50 mM NaOH and 20% glacial acetic acid (1:2) was added thereto to stop the reaction. 200 fd of the supematant recovered from the reaction mixture by centriftigation was mixed with 40 fd of 0.5% (w/v) PDAB (p-dimethylaminobenzaldehyde; Sigma, USA) dissolved in a methanol and the reaction mixture was incubated at 37*0 for 10 min. After the reaction, the absorbance of reaction mixture was measured at 415 nm and quantified by comparing with a calibration curve of standard material. At this time, 1 unit has been defined as the amount of enzyme capable to produce 1 jmiole of 7-ACA from CPC per minute. Meanwhile, the specific activity of CPC acylase mutant for CPC was determined by measuring the amount of protein remaining in the enzyme solution according to the method described by Bradford (Bradford, M., Anal. Biochem. 72: 248-254, 1976) and representing it as an active unit corresponding to 1 mg of protein. The reactivity of CPC acylase mutant to CPC was determined by the following steps of adding the same amount of protein to a reaction mixture, performing an enzyme reaction for a fixed time, measuring the amount of 7-ACA produced in the reaction mixture, and representing it as a relative value. The end-product inhibition by 7-ACA was determined by adding a protein corresponding to the same active unit to a reaction mixture, performing an enzyme reaction for a fixed time; measuring the amount of 7-ACA produced in the reaction mixture, and representing it as a relative value.<br>
As a result of measuring the enzyme activity to CPC, while the<br>
M<br><br>
productivity of CPC acylase produced by E. coli transformant MC1061(pBCPC) was about 97 unit/Â« , that by E. coli transformant MC1061(pBSEM) was only about 11 xxmXii .<br><l-l-3> Preparation of Met3ip/Phe58p double mutant<br>
To develop a CPC acylase mutant having an improved reactivity to CPC, Met3ip/Phe58p double mutant was prepared as follows. As shown in the tertiary structural modeling of CAD-CPC complex, since the CPC side chain additionally contains a carbon backbone and a D-amino group as compared with the GL-7-ACA side chain, CAD needs a large space for binding with CPC than GL-7-ACA. To secure the space enough for binding with CPC, two residues within the substrate-binding site of Acyll was subjected to a simultaneous mutation. With fixing His57p of Acyll corresponding to Arg57P of CAD as the most important residue for the binding of CPC side chain, Met3ip of Acyll being regarded to collide with a carboxyl group and a D-amino group at the terminal end of CPC side chain was replaced by leucine. At the same time, to further secure the space for the binding of CPC side chain by inducing a change of torsion rotation at His57P side chain of Acyll, PheSSp of Acyll was replaced by another amino acid residue having a relatively small side chain such as alanine, valine, leucine, methionine, cysteine or asparagine. As a result, M31PL/F58PM, M3ipL/F58pC, M31pL/F58pL, M31pL/F58pA, M31pL/F58pV and M31pL/F58pN double mutants were prepared by overlapping PCR (Ho, et al., Gene 15: 51-59, 1989). The procedure for preparing these double mutants was set forth in detail as follows.<br>
PCR reaction solution (100 /d) for a site-directed mutagenesis contained 10 ng of template DNA, 50 pmol each of forward and reverse primers, 0.2 mM dNTP mixture, Taq buffer solution (5 mM KCl, 5 mM Tris-HCl, (pH 8.3), 1.5 mM MgCla), and 2.5 unit of ExTaq polymerase (Takara, Japan). Reactions were initiated by denaturation for 5 min at 95 "C in a programmable thermal cycler (Peltier Thermal Cycler PTC-200; MJ Research, USA). PCR conditions consisted of 25 cycles of 1 min at 95t;, 30 sec at 58*0, and 60 to 90 sec at 72 "C, with a final elongation of 10 min at 72 "C.<br>
In particular, a series of PCR amplifications for preparing M3ipL/F58pM double mutant were performed using the following template and primers: pBSEM plasmid as a template and M13-R primer (SEQ ID NO: 11) and M31pL-R primer (SEQ ID NO: 19) to obtain 1.0 kb of PCR product; and<br>
d^<br><br>
pBSEM plasmid as template and F58pM-F primer (SEQ ID NO: 20) and MI3-F primer (SEQ ID NO: 12) to obtain 1.6 kb of PCR product.<br>
1.0 kb and 1.6 kb of PCR products obtained above were mixed and subjected to PCR under the same condition except that primers did not added, to obtain about 2.5 kb of PCR product which is formed in one by connecting two PCR products with each other. After then, about 2.5 kb of PCR product was subjected to PCR using T3 primer (SEQ ID NO: 9) and T7 primer (SEQ ID NO: 10) to amplify the double mutant DNA fragment of about 2.5 kb in size. After the PCR amplification, about 2.5 kb of PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pBC KS(+) vector DNA was digested with Xbal/Xhol and subjected to dephosphorylation with CIP, to obtain a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at 16Â°C for 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 25 iigM of chloramphenicol and cultured at 30*0 incubator for overnight to select a transformant containing the double mutant gene. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed to confirm the mutated residue.<br>
M31pL/F58pC, M31pL/F58pL, M31pL/F58pA, M31pL/F58pV and M3ipL/F58pN double mutants were prepared according to the same method described above. At this time, for M31pL/F58pC, M13-R primer (SEQ ID NO: 11) and M31pL-R primer (SEQ ID NO: 19), and F58PC-F primer (SEQ ID NO: 21) and M13-F primer (SEQ ID NO: 12) were employed for the overlapping PCR amplification; for M3ipL/F58pL, M13-R primer (SEQ ID NO: 11) and M3 IpL-R primer (SEQ ID NO: 19), and F58PL-F primer (SEQ ID NO: 24) and M13-F primer (SEQ ID NO: 12); for M3ipL/F58PA, M13-R primer (SEQ ID NO: 11) and M3ipL-R primer (SEQ ID NO: 19), and F58PA-F primer (SEQ ID NO: 22) and M13-F primer (SEQ ID NO: 12); for M31pL/F58pV, M13-R primer (SEQ ID NO: 11) and M3ipL-R primer (SEQ ID NO: 19), and F58pV-F primer (SEQ ID NO: 23) and M13-F primer (SEQ ID NO: 12); and for M3ipL/F58pN, M13-R primer (SEQ ID NO: 11) and M3ipL-R primer (SEQ ID NO: 19), and F58PN-F primer (SEQ ID NO: 25) and M13-F primer (SEQ ID NO: 12).<br>
The specific activity of double mutants obtained above for CPC was measured by using the crude enzyme solution of each CPC acylase mutant<br>
^<br><br>
according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the specific activity of M3ipL/F58pM, M31pL/F58pC, M31pL/F58pN, M31pL/F58pL, M3ipL/F58pA and M31pL/F58pV double mutants show 2.4-, 2.3-, 2.0-, 1.8-, 1.6- and 1.6-fold higher than that of the wild-type Acyll, respectively.<br><l-l-4> Preparation of Met3ip/Phe58p/His70p triple mutant<br>
To further increase the specific activity of M3ipL/F58pM showing the highest increased specific activity among 6 double mutants, His70p of Acyll corresponding to ValTOP of CAD which stimulates a catalytic reaction of active site Sip was replaced by serine or leucine to prepare a triple mutant.<br>
M3ipL/F58pM/H70pS triple mutant was prepared by using a QuickChange^'^ site-directed mutagenesis kit (Stratagene, USA) according to the manufacturer's instruction. PCR reaction solution (50 fd) contained 40 ng of M31PL/F58PM double mutant DNA, 100 pmol each of H70pS-F primer (SEQ ID NO: 26) and H70pS-R primer (SEQ ID NO: 27), 1 fd of dNTP mixture, 5 fd of buffer solution (100 mM KCl, 100 mM (NH4)2S04, 200 mM Tris-HCl, (pH 8.8), 20 mM MgS04, 1% Triton X-100 and 1 mg/mC of BSA), and 2.5 unit of pfiiTurboâ¢ DNA polymerase (Strategene, USA). Reactions were initiated by denaturation for 30 sec at 95*0 in a programmable thermal cycler (Peltier Thermal Cycler PTC-200; MJ Research, USA). PCR conditions consisted of 16 cycles of 30 sec at 95Â°C, 1 min at 55Â°C, and 12 min at 68Â°C. After the PCR amplification, 10 unit oiDpnl restriction enzyme was added to the PCR reaction solution and incubated at 37*0 for 1 hr to remove the template DNA wherein any mutation does not occur. The mutant DNA was purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany) and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 25 uglmi of chloramphenicol and cultured at 301) incubator for overnight to select a transformant. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed to confirm the mutated residue.<br>
M31pL/F58PM/H70pL triple mutant was prepared by using a QuickChange^"^ site-directed mutagenesis kit (Stratagene, USA) according to the same method as described above except that H70PL-F primer (SEQ ID NO: 28) and H70pL-R primer (SEQ ID NO: 29) were employed.<br>
The specific activity of triple mutant for CPC was measured by using<br>
y<br><br>
the crude enzyme solution of each CPC acylase mutant according to the same method as described in Example <l-l-2>. As a result, the specific activity of M3ipL/F58PM/H70pS and M3ipL/F58pM/H70pL triple mutants showed 3.2-and 2.4-fold higher than that of M3ipL/F58pM double mutant, respectively.<br><l-i-5> Preparation of Phel69a/Met3ip/Phe58p/His70p fourfold mutant<br>
To further increase the specific activity of M3ipL/F58pM/H70pS showing the highest increased specific activity between two triple mutants, Phel69a of Acyll corresponding to Tyrl49a of CAD which helps CPC to be located at a suitable position for an efficient catalytic reaction of active site Sip was replaced by tyrosine to prepare a fourfold mutant, F169aY/M3 lpL/F58pM/H70pS. Since a tyrosine side chain additionally has a hydroxyl group (-0H) as compared with a phenylalanine side chain, it can stimulate a catalytic reaction of Sip by forming a hydrogen bond with a CPC side chain and/or adjacent residues thereof.<br>
F169aY/M3ipL/F58pM/H70pS fourfold mutant was prepared by using a QuickChange^^ site-directed mutagenesis kit (Stratagene, USA) according to the same method as described in Example <l-l-4> except that M3ipL/F58pM/H70pS triple mutant as a template and F169aY-F primer (SEQ ID NO: 30) and F169aY-R primer (SEQ ID NO: 31) were employed.<br>
The specific activity of F169aY/M3ipL/F58pM/H70pS fourfold mutant for CPC was measured by using the crude enzyme solution of fourfold mutant according to the same method as described in Example <l-l-2>. As a result, the specific activity of F169aY/M3ipL/F58pM/H70pS fourfold mutant increased 2.1-fold higher than that of M3ipL/F58pM/H70pS triple mutant.<br><l-l-6> Preparation of Phel69o/Met3ip/Phe58p/His70p/Ilel76p fivefold mutant<br>
To fiirther increase the specific activity of F169aY/M31pL/F58pM/H70pS fourfold mutant to CPC, Ilel76p of Acyll corresponding to Phel77P of CAD which may be involved in interfering with the binding of a CPC side chain was replaced by valine to prepare a fivefold mutant, F169aY/M3ipL/F58pM/H70pS/Ilel76pV. In Example , Phe58P of Acyll was replaced by methionine. Meanwhile, Ilel76p of Acyll is very closely located to Phe58p of Acyll as well as supposed to interfere with the<br><br>
CPC binding. Thus, for the more efficient binding of CPC substrate side chain, Ilel76P was replaced by valine which has a side chain having a similar structure but smaller size than isoleucine.<br>
F169aY/M3ipL/F58pM/H70pS/I176pV fivefold mutant was prepared by using a QuickChange'^^ site-directed mutagenesis kit (Stratagene, USA) according to the same method as described in Example <l-l-4> except that F169aY/M31pL/F58pM/H70pS fourfold mutant as a template and Ilel76pV-F primer (SEQ ID NO: 32) and Ilel76pV-R primer (SEQ ID NO: 33) were employed.<br>
The specific activity of F169aY/M3ipL/F58pM/H70pS/I176pV fivefold mutant for CPC was measured by using the crude enzyme solution of fivefold mutant according to the same method as described in Example <l-l-2>. As a result, the specific activity of F169aY/M3ipL/F58pM/H70pS/I176pV fivefold mutant increased 2.4-fold higher than F169aY/M31pL/F58pM/H70pS fourfold mutant.<br><l-2> Preparation of CPC acylase mutant having an improved reactivity to<br>
CPC<br><l-2-l> Preparation of Phel69a/Met3ip/Phe58p/Ilel76p fourfold mutant<br>
The reason why it is difficult to apply an existing CPC acylase to a one-<br>
step enzymatic method for producing 7-ACA on a large scale is that the<br>
conversion efficiency of CPC into 7-ACA is very low due to the end-product<br>
inhibition by 7-ACA as well as the CPC acylase showing a low specific activity<br>
for CPC. Meanwhile, it can be known from a series of site-directed<br>
mutagenesis for increasing the specific activity of CPC acylase mutant in<br>
Example  that the mutation of His70p significantly increases the specific<br>
activity of CPC acylase mutant for CPC but severely increases the level of end-<br>
product inhibition by 7-ACA. Thus, F169aY/M31pL/F58pM/H70pS/I176pV<br>
fivefold mutant of Example <l-l-6> was subjected to a reverse mutagenesis of<br>
H70PS into a wild-type residue, histidine, to prepare<br>
FI69aY/M3ipL/F58pM/H76pV        fourfold        mutant        (Fig.	5).<br>
F169aY/M3ipL/F58pM/Il76pv fourfold mutant was prepared according to the same method as described in Example <l-l-4> except that F169aY/M3ipL/F58pM/H70pS/I176pV fivefold mutant as a template and H70P-F primer (SEQ ID NO: 34) and H70P-R primer (SEQ ID NO: 35) were employed.<br>
cM.<br><br>
The end-product inhibition was measured by using an enzyme solution corresponding to the same active unit after the preparation of each crude enzyme solution of wild-type, F169aY/M31pL/F58pM/H70pS/I176pV fivefold mutant and F169aY/M31pL/F58PM/I176pV fourfold mutant by the same method as described in Example <l-i-2>. As a result, it has been confirmed that the reverse mutation of H70pS in F169aY/M3ipL/F58pM/H70pS/I176pV fivefold mutant into a wild-type residue, histidine decreases the end-product inhibition by 7-ACA to the similar level of wild-type enzyme (Fig. 6).<br><l-2-2>   Preparation   of  GIyl39a/PheI69o/Met31p/Phe58p/nel76p   and Vall21o/Phel69o/Met31p/Phe58p/Ilel76P fivefold mutants<br>
To further increase the reactivity of F169aY/M31pL/F58pM/Il76pV fourfold mutant to CPC, the fourfold mutant DNA was subjected to error-prone PCR to construct a mutant library having a point mutation at a random site. At this time, an error rate in said error-prone PCR was adjusted to be occtured a substitution at one amino acid residue for one fourfold mutant gene on the average. The particular procedure of constructing a mutant library was as follows.<br>
PCR reaction solution (100 id) for error-prone PCR contained 5 ng of F169aY/M3ipL/F58pM/I176pV fourfold mutant DNA, 50 pmol each of T3 primer (SEQ ID NO: 9) and T7 primer (SEQ ID NO: 10), 0.2 mM each of dATP and dGTP, 1.0 mM each of dCTP and dTTP, 5 mM KCl, 5 mM Tris-HCl, (pH 8.3), 3.5 mM MgCh, 0.025 mM MnCla and 5 unit of rTaq DNA polymerase (Takara, Japan). Reactions were initiated by denaturation for 3 min at 95 "C in a programmable thermal cycler (Peltier Thermal Cycler PTC-200; MJ Research, USA). PCR conditions consisted of 20 cycles of 1 min at 95*0, 30 sec at 58"C, and 90 sec at 72'C, with a final elongation of 10 min at 72*0. After the error-prone PCR amplification, about 2.5 kb of PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pBSEM plasmid DNA was digested with Xbal/Xhol to obtain 3.4 kb of DNA fragment used as a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at 16*0 for 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 25 /zg/m^ of chloramphenicol and cultured at 30 "C incubator for overnight to construct a mutant library.<br>
'?<br><br>
The CPC acylase mutant having an improved reactivity to CPC was screened from the mutant library obtained above as follows.<br>
E. coli MCI061 transformant containing the CPC acylase mutant gene introduced a point mutation at a random site by error-prone PCR was inoculated in a 96-well plate filling 200 id of LB medium containing 25 {iglid of chloramphenicol and cultivated at 30t), 180 rpm for 60 to 70 hr with vigorous shaking. 100 JJZ of each culture solution taken from the well plate was transferred to a new 96-well plate. 100 /iC of a cell lysis solution (0.1 M Tris-Cl buffer solution (pH 8.0) containing 2 mg/M lysozyme, 4 mM EDTA, 0.4% Triton X-100) was added thereto and leaved at 301) for 2 hr. After then, 50 id of the mixture of 2.5% (w/v) CPC solution dissolved in 0.1 M Tris-Cl buffer solution (pH 8.0) and 5 mM 7-ACA was added to each well and the well plate was kept at 28*0 for 14 to 16 hr to induce a hydrolysis reaction of CPC. At this time, the reason why 7-ACA was added to the CPC solution is for facilitating the screening of CPC acylase mutant showing an improved specific activity for CPC and/or the decreased end-product inhibition by 7-ACA. After the hydrolysis reaction of CPC, the reaction mixture was subjected to centrifiigation at 4,200 rpm for 20 min to separate a supernatant and 50 [d of the supernatant was transferred to a new 96-well plate. After 160 id of a stop solution (acetic acid : 250 mM NaOH, 2:1) was added to each well to stop the enzyme reaction, 40 /^ of a developing agent (0.5% (w/v) PDAB solution dissolved in a methanol) was added thereto and the well plate leaved at room temperature for 10 min. The well plate was then loaded on a microplate reader to measure an absorbance at 415 m and the CPC acylase mutant having an improved specific activity for CPC was selected by comparing the measured absorbance value.<br>
As a result of randomly screening about 25,000 colonies from said random mutant library, 2 mutants (#120 and #213) showing the higher absorbance value than that of F169aY/M31pL/F58pM/I176pV fourfold mutant used as a template for a random mutagenesis were selected. It has been confirmed by a sequence analysis that #120 mutant has the substitution of Glyl39a by serine (G139aS/F169aY/M31pL/F58pM/I176pV fivefold mutant) and #213 mutant, the substitution of Vall21a by alanine (V121aA/F169aY/M31pL/F58pM/I176pV fivefold mutant) (Fig. 5).<br>
Further, the reactivity of #120 and #213 mutants to CPC were measured by using the crude en2yme solution of each mutant according to the same method as described in Example <l-l-2>.   As a result, it has been confirmed<br>
n<br><br>
that #120 and #213 mutants show the further increased reactivity to CPC than F169aY/M3 lpL/F58pM/I176pV fourfold mutant of Example <l-2-l> (Fig. 7).<br><l-2-3> Preparation of Vall21a/GIyl39a/Phel69a/Met3ip/Phe58p/Ilel76p sixfold mutant<br>
It has been confirmed from the results of Example <l-2-2> that the<br>
incorporation of V121aA or G139aS mutation into<br>
F169aY/M31pL/F58pM/I176pV fourfold mutant increases the reactivity of<br>
CPC acylase mutant to CPC. Thus, in order to further increase the reactivity to<br>
CPC, Vl21aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant was<br>
prepared by incorporating G139aS mutation of #120 mutant into #213 mutant<br>
(Fig. 5). The inventive sixfold mutant was prepared according to the same<br>
method as described in Example <l-l-4> except that #213 mutant DNA as a<br>
template and G139aS-F primer (SEQ ID NO: 36) and G139aS-R primer (SEQ<br>
ID NO; 37) were employed for PCR. The present invention has designated the<br>
CPC	acylase	mutant	gene	encoding<br>
V12laA/G139aS/F169aY/M31PL/F58PM/I176pv sixfold mutant as TnS5.<br>
The reactivity of V121aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant to CPC was measured by using the crude enzyme solution of sixfold mutant according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the reactivity of V121aA/G139aS/F169aY/M3ipL/F58pM/I176pV sixfold mutant (TnS5) to CPC increases rather than those of #213 and #120 mutants (Fig. 7).<br><l-3> Preparation of pBC-TnS5ap plasmid<br>
The	TnS5	gene	encoding<br>
V121aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant is the CPC acylase mutant gene which is designed to express a-subunit and P-subunit separately in a host cell and generate the active form of CPC acylase via a spontaneous contact of these two subunits such as the sem gene.<br>
To induce the formation of active CPC acylase mutant fi:om the TnS5 gene which consists of each one of a- and P-subunits obtained through a self-digestion process after the transcription and translation of CPC acylase mutant gene in a host cell such as the wild-type acylase gene {acyll), the present invention prepared a TnS5ap gene encoding the sixfold CPC acylase mutant<br>
^<br><br>
wherein the spacer peptide of wild-type CPC acylase (Acyll) was inserted into<br>
between	a-subunit	and	P-subunit	of<br>
V121aA/G139aS/F169aY/M31pL/F58pM/I176pV   sixfold   mutant   (Fig.   5). The method for preparing the TnSSaP gene was in detail as follows.<br>
The TnS5 gene was inserted into Xhol/Xbal sites of pBC-KS(+) vector to prepare pBC-TnS5 plasmid. Twice PCR amplifications were performed by using pBC-TnS5 plasmid as a template and M13-R primer (SEQ ID NO: 11) and aCPC-R primer (SEQ ID NO: 38) to obtain 0.8 kb of PCR product and pBC-TnS5 plasmid as a template and pCPC-F primer (SEQ ID NO: 39) and M13-F primer (SEQ ID NO: 12) to obtain 1.7 kb of PCR product. 0.8 kb and 1.7 kb of PCR products obtained above were mixed and subjected to PCR under the same condition except that primers did not added, to obtain about 2.5 kb of PCR product which is formed in one by connecting two PCR products with each other. After then, 2.5 kb of PCR product was subjected to PCR using T3 primer (SEQ ID NO: 9) and T7 primer (SEQ ID NO: 10) to amplify the DNA fragment containing the TnS5ap gene. After the PCR amplification, about 2.5 kb of PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pBC KS(+) vector DNA was digested with Xbal/Xhol and subjected to dephosphorylation with CIP, to obtain a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at 16*0 for 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 25 iigM of chloramphenicol and cultured at 30Â°C incubator for overnight to select a transformant. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed. Out of this, pBC-TnS5aP plasmid containing the TnS5aP gene was prepared.<br>
In order to examine the productivity of CPC acylase mutant by the TnS5aP gene, each of TnS5 and TnS5aP genes encoding V121aAyG139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant was inserted at pBC-KS(+) vector to prepare pBC-TnS5 and pBC-TnS5aP recombinant plasmids, respectively, and each recombinant plasmid was transformed to E. coli MC1061 to obtain E. coli transformant MC1061(pBC-TnS5) and MC1061(pBC-TnS5ap). Each Â£. co//transformant was cultivated in 50 m^ of a LB broth containing 25 ixglvA of chloramphenicol at 25 "C, 200 rpm for 48 hr with vigorous shaking and the enzyme activity to CPC was measured by using the crude enzyme solution prepared from the transformant culture solution<br>
jo<br><br>
according to the same method as described in Example <l-l-2>. As a result, the productivity of CPC acylase produced by each E. coli transformant MC1061(pBC.TnS5) and MCI061(pBC-TnS5ap) was about 78 unHli and 162 unit/^ , respectively. From these results, it has been confirmed that the sixfold CPC acylase mutant formed by a self-digestion process after the transcription and translation of CPC acylase mutant gene in a host cell such as TnSSaP shows about 2-fold higher productivity than the sixfold CPC acylase mutant formed by a spontaneous contact of a-subunit and P-subunit expressed separately in a host cell such as TnS5.<br>
Further, the crude enzyme solution prepared from the culture solution of E. coli transformant MCl061(pBC-TnS5ap) was subjected to a denaturing polyacrylamide gel electrophoresis (12% SDS-PAGE) to examine the expression pattern of protein. As a result, there was no inactive precursor band of about 83 kDa in size, which means that most of the inactive precursor form convert into the active form of CPC acylase mutant by an efficient self-digestion after the transcription and translation of TnSSaP gene under the culture condition of the present invention (Fig. 7). Therefore, the TnSSaP gene has been employed as a template DNA for developing a CPC acylase mutant in the following.<br><l-4> Preparation of CPC acylase mutant having a additionally increased<br>
reactivity to CPC<br><l-4-l> Preparation of Vall21a/G!yl39a/Phel69a/Phe58p/Ilel76p fivefold<br>
mutant<br>
The present invention prepared the CPC acylase mutant genes TnS5 and TnS5aP encoding V121aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant having an improved reactivity to CPC by a series of site-directed mytagenesis and a random mutagenesis in Examples  to <l-3>. In these mutagenesis procedures for preparing the TnSSaP sixfold mutant, Met3 ip being supposed to collide with a carboxyl group and a D-amino group at the terminal end of CPC side chain was replaced by leucine and Phe58p was replaced by methionine, cysteine or asparagine at the same time to secure the space enough for the efficient binding of CPC side chain by changing a torsion rotation at His57p side chain. Thus, the present invention optimized Met3ip and Phe58p residues in V121aA/G139aS/F169aY/M3ipL/F58pM/I176pV sixfold mutant to further increase the reactivity to CPC as follows.<br>
3(<br><br>
The present invention performed a reverse mutagenesis of M31pL into a<br>
wild-type residue, methionine and a substitution of F58pM by asparagine or<br>
cysteine using the TnS5aP sixfold mutant DNA as a template to prepare 4<br>
mutants	(V121 oA/G 139aS/F 169aY/F58pN/1176pV,<br>
V121aA/G139aS/F169aY/F58pC/I176pV, V12laA/G139aS/F169aY/M3ipL/F58pN/I176pV,<br>
V121aA/G139aS/F169aY/M3ipL/F58pC/I176pV). These mutants were<br>
prepared according to the same method for preparing M3ipL/F58pM double<br>
mutant as described in Example <l-l-3>. At this time, the overlapping PCR<br>
was performed using the following primers: for<br>
V121aA/G139aS/F169aY/M31pL/F58pC/I176pV, M13-R primer (SEQ ID NO:<br>
11) and M3 IpL-R primer (SEQ ID NO: 19), and F58pC-F primer (SEQ ID NO:<br>
21) and M13-F primer (SEQ ID NO: 12); for<br>
V121aA/G139aS/F169aY/M3ipL/F58pN/I176pV, M13-R primer (SEQ ID NO:<br>
11) and M3ipL-R primer (SEQ ID NO: 19), F58PN-F primer (SEQ ID NO: 25)<br>
and        M13-F        primer        (SEQ        ID        NO:	12);	for<br>
V121aA/G139aS/F169aY/F58pC/I176pV, M13-R primer (SEQ ID NO: 11) and M31P-R primer (SEQ ID NO: 42). and F58pC-F primer (SEQ ID NO: 21) and M13-F primer (SEQ ID NO: 12); for V121aA/G139aS/F169aY/F58pN/I176pV, M13-R primer (SEQ ID NO: 11) and M31p-R primer (SEQ ID NO: 42), and F58PN-F primer (SEQ ID NO: 25) and M13-F primer (SEQ ID NO: 12).<br>
The enzyme reactivity to CPC was measured by using the crude enzyme solution of each mutant according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the mutant having a substitution of F58pM by asparagine in TnS5aP sixfold mutant shows the considerably increased reactivity to CPC and the mutant having a reverse mutation of M3ipL into methionine in TnSSaP sixfold mutant more efficiently produces 7-ACA from CPC although its specific activity decreases somewhat (Fig. 8). From these results, the present invention has selected V121aA/G139aS/F169aY/F58pN/I176pV fivefold mutant (mF58) having a further increased reactivity to CPC than TnS5aP sixfold mutant (Fig. 5).<br><l-4-2> Preparation of VaI121o/GIyl39a/ Phe58p/IIel76p fourfold mutant<br>
The present invention has replaced Phel69a by tyrosine in M3ipL/F58pM/H70pS triple mutant to further increase the specific reactivity of CPC acylase mutant for CPC in Example <l-l-5>.   Meanwhile, H70pS residue<br>
32^<br><br>
in F169aY/M31pL/F58pM/H70pS/I176pV fivefold mutant was subjected to a reverse mutagenesis into histidine in Example <l-2-l> to reduce the end-product inhibition by 7-ACA, and M3ipL residue in VI21aA/G139aS/F169aY/M31pL/F58pM/I176pV sixfold mutant was also subjected to a reverse mutagenesis into a wild-type residue, methionine in Example <l-4-l>. Further, F58pM mutated residue and I176p residue in V121aA/G139aS/F169aY/F58pM/I176pV fivefold mutant were replaced by asparagine and valine, respectively. Thus, V121aA/G139aS/F58pN/I176pV fourfold mutant (F169) was prepared by a reverse mutagenesis of F169aY mutated residue into a wild-type residue, phenylalanine (Fig. 5).<br>
V12laAyG139aS/F58pN/I176pV fourfold mutant was prepared by the<br>
same method for preparing M3ipL/F58pM/H70pS triple mutant DNA as<br>
described	in	Example	<l-l-4>	except	that<br>
V121aA/G139aS/F169aY/F58pN/I176pV fivefold mutant DNA as a template and F169a-F primer (SEQ ID NO: 45) and F169a-R primer (SEQ ID NO: 46) were employed for PCR.<br>
The reactivity of V121aA/G139aS/F58pN/I176pV fourfold mutant to CPC was measured by using the crude enzyme solution of mutant enzyme according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the reverse mutation of F169aY into a wild-type residue, phenylalanine in V121aA/G139aS/FI69aY/F58pN/I176pV fivefold mutant further increases the reactivity of CPC acylase mutant to CPC (Fig. 9).<br><l-4-3> Preparation of Vall21o/Glyl39o/Phe58p/Ile75p/Ilel76p and Vall21a/Glyl39a/ Phe58p/Ilel76p/Ser471p fivefold mutants<br>
To further increase the reactivity of V121aA/G139aS/F58pN/I176pV fourfold mutant to CPC, V121aA/G139aS/F58pN/I176pV fourfold mutant was subjected to error-prone PCR to construct a random mutant library. A mutant showing the fiirther increased reactivity to CPC than V121aA/G139aS/F58pN/I176pV fourfold mutant was screened from the random mutant library constructed above. At this time, the reaction condition of error-prone PCR, the methods for preparing said mutant library and screening the mutant therefrom were the same as described in Example <l-2-2> except that when the mutant showing the further increased reactivity to CPC was screened from the mutant library, 5 mM 7-ACA (final concentration) was added to the reaction mixture.<br>
33<br><br>
As a result of screening about 15,000 colonies from said random mutant library, 2 mutants (#59 and #76) showing the higher absorbance value than V121aA/G139aS/F58pN/I176pV fourfold mutant used as a template for a random mutagenesis were selected. It has been confirmed from the results of sequence analysis that #59 mutant has the substitution of Ile75p by threonine (V121aA/G139aS/F58pN/I75pT/I176pV fivefold mutant) and #76 mutant, the substitution of Ser47ip by cysteine (V121aA/G139aS/F58pN/I176pV/S47ipC fivefold mutant) (Fig. 5),<br>
Further, the reactivity of #59 and #76 mutants to CPC were measured by using the crude enzyme solution of each mutant according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the reactivity of #59 and #76 mutants to CPC are higher than that of V121aA/G139aS/F58pN/I176pV fourfold mutant (F169) (Fig. 9).<br><m> Preparation  of VaI121a/Glyl39a/Phe58p/Ile75p/Ilel76p/Ser47ip sixfold mutant<br>
It has been confirmed above that the reactivity to CPC is increased by introducing Ile75pT or S471pC mutation into V121aA/G139aS/F58pN/I176pV fourfold mutant. Therefore, to additionally increase the reactivity to CPC, the present invention prepared V121aA/G139aS/F58pN/I75pT/I176pV/S47ipC sixfold mutant by introducing Ile75PT mutation of #59 mutant into #76 mutant (Fig. 5). The sixfold mutant of the present invention was prepared by the same method as described in Example <l-l-4> except that #76 mutant DNA as a template and I75pT-F primer (SEQ ID NO: 43) and I75pT-R primer (SEQ ID NO: 44) were employed for the PCR amplification. The present invention has designated the CPC acylase mutant gene encoding V121aA/G139aS/F58pN/I75pT/I176pV/S471pC sixfold mutant as S12.<br>
The reactivity of V121aA/G139aS/F58pN/I75PT/I176pV/S47ipC sixfold mutant to CPC was measured by using the crude enzyme solution of sixfold mutant according to the same method as described in Example <l-l-2>. As a result, it has been confirmed that the reactivity of V12IaA/G139aS/F58pN/I75PT/I176pV/S471pC sixfold mutant (S12) to CPC is higher than that of #76 mutant (V121aA/G139aS/F58pN/I176pV/SpC fivefold mutant) (Fig. 10).<br>
Example 2: Purification of CPC acylase mutant and analysis of its reaction<br><br>
property<br> Purification of CFC acylase mutant having an increased specific<br>
activity to CFC<br>
E. coli MCI061 transformant containing the recombinant plasmid introduced with the wild-type or the inventive CPC acylase mutant gene was cultivated in 1 Â« of a terrific broth (1.2% Bacto-Trypton, 2.4% Yeast Extract, 0.4% glycerol, 0.17 M KH2PO4, 0.72 M K2HPO4) containing 25 iiglmH of chloramphenicol at 30*0 for 16 hr to obtain a pre-culture solution. 10 nvC of the pre-culture solution was inoculated into the same medium, and then, cultured at 25*0, 200 rpm for 48 hr with vigorous shaking. The culture solution was subjected to centrifugation at 4*0, 8,000 rpm for 10 min to separate a precipitate, and the precipitate was washed twice with 20 mM Tris-HCl buffer solution (pH 8.0). The precipitate was suspended in 100 m^ of the same buffer solution. This suspension was subjected to ultra sonication at 4*0 for 20 min to destruct a cell wall and centrifugation at 410, 15,000 rpm for 20 min to remove insoluble materials and cell debris, which results in obtaining a supernatant used as a crude enzyme solution in the following.<br>
Ammonium sulfate was added to the crude enzyme solution to be adjusted its final saturation rate to 30% and the reaction mixture was stirred at 4IC for 1 hr to saturate. After then, the reaction mixture was subjected to centrifugation at 4*0, 12,000 rpm for 15 min to remove a precipitate and ammonium sulfate was added to the supematant to be adjusted a final saturation rate to 60%. The reaction mixture was stirred at 4*0 for 1 hr to saturate and subjected to centrifugation at 4^, 12,000 rpm for 15 min to recover a precipitate. The precipitate was suspended in 20 mM Tris-HCl buffer solution (pH 8.0) and subjected to dialysis with 20 mM Tris-HCl buffer solution (pH 8.0) containing 50 mM NaCl for ovemight.<br>
The protein solution obtained by dialysis was subjected to DEAE-sepharose anion exchange column chromatography (5x 18.5 cm) equilibrated with 20 mM Tris-HCl buffer solution (pH 8.0) containing 50 mM NaCl and the column was washed with triple volume of the same buffer solution to remove protein molecules that do not absorb thereto. The absorbed protein was eluted by gradually increasing the concentration of NaCl ranging from 50 to 200 mM. The fraction showing a CPC acylase activity was collected and concentrated with a polyethylene glycol (M.W, 20,000) in a dialysis bag. The concentrated protein was subjected to dialysis with 20 mM Tris-HCl buffer solution (pH 8.0)<br>
%&gt;^<br><br>
containing 25% ammonium sulfate.<br>
The difiusate obtained above was subjected to Phenyl-Toyoperl column chromatography (2.5x 6 cm) equilibrated with 20 mM Tris-HCl buffer solution (pH 8.0) containing 25% ammonium sulfate to absorb the protein to the column, and the absorbed protein was eluted by gradually decreasing the concentration of ammonium sulfate ranging from 25 to 0%. After confirming the CPC acylase activity of each fraction, the fraction having the enzyme activity was recovered, concentrated with a polyethylene glycol and subjected to dialysis with 20 mM Tris-HCl buffer solution (pH 8.0), to purify the wild-type and the inventive CPC acylase mutant.<br>
The enzyme activity was measured by the same method as described in Example <l-l-2>.<br> Analysis of physical property and reaction feature of wild-type CPC acylase<br>
The wild-type CPC acylase was purified from E. coli transformant containing pBCPC or pBSEM plasmid prepared in Example <l-l-2> according to the same method as described in Example . As a result of analyzing the physical property and reaction feature of pBCPC plasmid derived wild-type CPC acylase (Acyll) and pBSEM plasmid derived wild-type CPC acylase (Sem), respectively, they showed the same physical property (e.g., molecular weight) and reaction feature (e.g., specific activity, optimal temperature, optimal pH).<br>
Further, the active fraction of wild-type CPC acylase obtained in Example  was subjected to non-denaturing PAGE and Coomasie blue staining. As a result, a single band was detected at a position corresponding to the molecular weight of about 83 kDa, which means that the wild-type CPC acylase was purely purified (Figs. 11 a to 11 c). Together with the result of non-denaturing PAGE, MALDI-TOF mass spectrophotometry analysis confirmed that the molecular weight of purified enzyme (that is, the active CPC acylase) is about 83 kDa. Furthermore, about 25 kDa of band corresponding to a-subunit and about 58 kDa of band corresponding to P-subunit were detected by denaturing SDS-PAGE, From these results, it has been confirmed that the inventive CPC acylase mutant is a dimer consisting of about 25 kDa of a-subunit and about 58 kDa of P-subunit one by one.<br>
a- and P-subunits were purified from the denaturing SDS-PAGE gel,<br>
34<br><br>
respectively, and the molecular weight of each subunit fragment was measured by MALDI-TOF mass spectrophotometry analysis. As a result, it has been supposed that the wild-type CPC acylase (Acyll) derived from pBCPC plasmid is separated into about 25 kDa of a-subunit and about 58 kDa of P-subunit by removing a spacer peptide consisting of 9 amino acids via a self-digestion occurred at two positions between the 230* and the 231^' amino acids, and the 239* and the 240* amino acids in the amino acid sequence of SEQ ID NO: 2, respectively.<br>
As a result of examining the specific activity of wild-type CPC acylase, the wild-type CPC acylase showed about 23.1 unit/mg protein of the specific activity to GL-7-ACA and about 0.33 unit/mg protein of the specific activity to CPC. Therefore, it has been known that the Pseudomonas sp. SE83 derived Acyll shows the specific activity to CPC corresponding to only about 1.4% of thattoGL-7-ACA.<br>
After performing the en2yme reaction according to the same method as described in Example <l-l-2>, a kinetic parameter of the purified enzyme was determined by Lineweaver-Burk plot method. As a result, Km, Kcat and a catalytic efficiency (that is, Kcat/Rm) of the wild-type CPC acylase Acyll were 50 mM, 0.9/sec and 0.02/sec/mM, respectively.<br>
Further, as a result of examining the optimal reaction temperature and pH of the wild-type CPC acylase Acyll, the optimal reaction temperature and pH to CPC were 40 Â°C and 9.0, respectively.<br> Analysis of reaction feature of CPC acylase mutant having an increased specific activity<br>
Each CPC acylase mutant was purified from the recombinant E. coli<br>
transformed with the plasmid containing each CPC acylase mutant gene<br>
encoding M31pL/F58pM double mutant, M31pL/F58pM/H70pS triple mutant,<br>
Fl69aY/M31pL/F58pM/H70pS	fourfold	mutant	and<br>
F169aY/M31pL/F58pM/H70pS/I176pV fivefold mutant, respectively, according to the same method as described in Example .<br>
As a result of analyzing the reaction feature of each purified mutant enzyme, their reaction feature (e.g., optimal temperature and pH) and physical property (e.g., molecular weight) were the same as the wild-type CPC acylase. However, as a result of examining the specific activity of each mutant enzyme, the specific activity of each mutant to CPC gradually increased as a mutagenesis<br>
7<br><br>
makes steady progress (Table 2). From these successive mutagenesis, F169aY/M3ipL/F58pM/H70pS/I176pV fivefold CPC acylase mutant showing the increased specific activity of about 11.2-fold higher than the wild-type CPC acylase to CPC was obtained.<br><table>
<br>
Mutant en2yme	Relative activity to CPC (fold)<br>
Wild-type	1.0<br>
M31PL/F58PM	1.2<br>
M3ipL/F58pM/H70pS	5.2<br>
F169aY/M3 lpL/F58pM/H70pS	7.3<br>
F169aY/M31 pL/F58PM/H70pS/1176pV	11.2<br>
After performing the enzyme reaction according to the same method as described in Example <l-l-2>, a kinetic parameter of F169aY/M31pL/F58pM/H70pS/I176pV fivefold mutant enzyme was determined by Lineweaver-Burk plot method. As a result, Km, Kcat and Kcat/Km were 8 mM, 2.4/sec and 0.30/sec/niM, respectively. F169aY/M31pL/F58pM/H70pS/I176pV fivefold mutant showed about 6.3-fold loweF&amp;m^ value and-.about-15-fold higher reaction, efficiency to CPC as ^mpared with the wild-type enzyme. From these results, it has been ~corifiraied that the inventive CPC acylase mutant obtained by a series of site-directed mutagenesis based on the tertiary structural information of CAD has an increased binding affinity and a specific activity to CPC.<br> Purification of CPC acylase mutant having an increased reactivity to CPC and analysis of reaction feature thereof<br>
Each sixfold CPC acylase mutant was purified from the recombinant E. coli transformed with the plasmid containing the CPC acylase mutant gene (TnS5aP) encoding V121aA/G139aS/F169aY/M3ipL/F58pM/Il76pV sixfold mutant of Example <l-3> or the CPC acylase mutant gene (SI2) encoding V121aA/G139aS/F58pN/I75pT/I176pV/S471pC sixfold mutant of Example  according to the same method as described in Example .<br>
As a result of analyzing the reaction feature of each purified mutant enzyme TnS5aP and SI2, their reaction feature (e.g., optimal temperature and<br>
3?<br><br>
pH) and physical property (e.g., molecular weight) were the same as the wild-type CPC acylase.<br>
To examine the specific activity to CPC and the end-product inhibition by 7-ACA of each CPC acylase mutant, the enzyme reaction was performed according to the same method as described in Example <l-l-2> except that pH of the reaction mixture was adjusted to 8.5. As a result, the specific activity of TnSSap and S12 mutant enzymes to CPC were 1.5 unit/mg protein and 5.8 unit/mg protein, respectively, which corresponds to 2.3- and 8.5-fold higher than that of the wild-type CPC acylase. Further, as a result measuring the inhibition constant (Ki) by 7-ACA of S12 mutant enzyme, while Ki value of the wild-type enzyme was 0.4 mM, Ki value of S12 mutant enzyme, 1.9 mM, which means that the end-product inhibition by 7-ACA of S12 mutant enzyme was significandy decreased as compared with that of the wild-type enzyme. From these results, it has been confirmed that the inventive S12 mutant enzyme (V121aA/G139aS/F58pN/I75pT/I176pV/S47ipC sixfold mutant) shows the increased specific activity to CPC but the decreased end-product inhibition by 7-ACA.<br>
Example 3: Production of CPC acylase mutant in E, coli<br> Production of CPC acylase mutant using pBC KS(+) vector<br>
V121aA/G139aS/F169aY/M3ipL/F58pM/I176pV sixfold CPC acylase<br>
mutant	encoding	genes	TnS	and	TnSSap,	and<br>
V121aAyG139aS/F58pN/I75pT/I176pV/S471pC sixfold CPC acylase mutant encoding gene S12 were inserted into pBC KS(+) vector to obtain pBC-TnS5, pBC-TnS5aP and pBC-S12 recombinant plasmids, respectively, and E. coli MCI061 was transformed with each resulting recombinant plasmid.<br>
Each E. coli transformant was cultivated in 50 mC of a LB broth containing 25 iiglmi of chloramphenicol at 25*0, 200 rpm for 48 hr with vigorous shaking and the productivity of CPC acylase was measured by using the crude enzyme solution prepared from the culture solution according to the same method as described in Example <l-l-2>. At this time, the recombinant plasmids pBSEM and pBCPC containing the wild-type acylase encoding gene acyll and sem, respectively, were used as a control. As a result, E. coli trasnformant containing pBC-S12 plasmid showed the maximum productivity of CPC acylase at a level of 712 unil/Â£ .<br>
;'<br><br><table>
<br>
The formation pattern of CPC acylase	Plasmid (CPC acylase)	The productivity of CPC acylase (unit/^ )<br>
Respective expression of each subunit	pBSEM (wild-type)	11<br><br>
	pBC-TnS5 (TnS5 sixfold mutant)	78<br>
Self-digestion	pBCPC (wild-type)	97<br><br>
	pBC-TnS5ap (TnSSap sixfold mutant)	162<br><br>
	pBC-S12(S12 sixfold mutant)	712<br> Production of CPC acylase mutant using pET29-a(+) vector  Preparation of pET29-TnS5ap and pET29-S12 plasmids<br>
The recombinant plasmids pBC-TnS5aP and pBC-S12 were subjected to the PCR amplification using pET29-F primer (SEQ ID NO: 40) and pET29-R primer (SEQ ID NO: 41), respectively, to obtain 2.5 kb of PCR product each. After the PCR amplification, the PCR product was digested with Xbal/Xhol and purified with a purification kit (QIAEX II Gel Extraction Kit; Qiagen, Germany), to obtain an insert DNA. Further, pET29-a(+) vector DNA (Novagen, USA) was digested with Xbal/Xhol and subjected to dephosphorylation with CIP, to obtain a vector DNA. The insert DNA and vector DNA were subjected to ligation using T4 DNA ligase (Roche, Germany) at 1610 for 16 hr and transformed to E. coli MCI061 strain by electrophoration. The E. coli strain was spread onto a LB agar plate containing 20 jt/g/m^ of kanamycin and cultured at 30 "C incubator for overnight to select a transformant containing the mutant gene. The plasmid was purified from the selected transformant and the nucleotide sequence of insert DNA was analyzed, to obtain pET29-TnS5ap and pET29-S12 plasmids.<br> Production of CPC acylase mutant by E, coli transformant containing pET29-TnS5ap or pET29-S12 plasmid<br>
Each of pET29-TnS5ap and pET29-S12 plasmids was transformed into E. coli BL21(DE3) by electrophoration to prepare the recombinant E. coli<br>
(fG<br><br>
containing pET29-TnS5ap and pET29-S12 plasmid, respectively, and the productivity of sixfold CPC acylase mutant in each recombinant E. coli was measured The TnSSaP and S12 CPC acylase mutant gene in these recombinant plasmids was transcribed by the action of T7 promoter which is under the control of Lad operator existed on pET29-a(+) vector.<br>
The recombinant E. coli containing pET-TnS5aP plasmid was inoculated in 3 m^ of a LB broth containing 20 fig/id of kanamycin and cultivated at BOX), 200 rpm for 16 hr with vigorous shaking. 50 fd of the culture solution was transferred to 50 mC of a new LB broth containing 20 /ig/mi of kanamycin and 0, 0.02, 0.2 and 2% lactose each and cultured at 25"C, 200 rpm for 80 hr with vigorous shaking. At this time, to measure the productivity of CPC acylase mutant according to the time course of cultivation, 5 fd of the culture broth was taken from the culture flask at 24, 36, 48, 72 and 80 hr during the cultivation, respectively. Each culture solution was subjected to centrifugation at 4*0, 8,000 rpm for 10 min to separate a precipitate and the precipitate was washed twice with 0.1 M Tris-HCl buffer solution (pH 8.0). After the precipitate was suspended in 500 }d of the same buffer solution, it was subjected to ultra sonication at 4*0 for 1 min and centrifugation at 4*0, 15,000 rpm for 20 min to separate a supernatant, which can be used as a crude enzyme solution of the sixfold CPC acylase mutant. The activity of sixfold CPC acylase mutant to CPC was measured using said crude enzyme solution according to the same method as described in Example <l-l-2>.<br>
Since IPTG generally used as an inducer for a high expression of foreign gene in an expression system of pET29 vector and E. coli BL21(DE3) is too expensive, it is difficult to apply IPTG to the industrial scale of cultivation. Thus, the present invention has employed a low-priced inducer, lactose (DDFCO, USA) instead of IPTG for the mass-production of CPC acylase mutant. Further, it has been generally known in the art that the expression system of pET29 vector and E. coli BL21(DE3) highly expresses the foreign gene within a short period by culturing a transformant for a fixed time without an inducer at the early stage of cultivation to propagate it and further culturing with the addition of inducer at a fixed point. However, since it was confirmed that the inducible expression method for producing the CPC acylase described above generated a large quantity of inactive polypeptide as a by-product, the present invention cultured the E. coli transformant with the addition of lactose at the early stage of cultivation to induce a constitutive expression. As a result, in case of culturing with 2% lactose, the productivity of CPC acylase was<br>
^i<br><br>
increased by the action of lactose as an inducer according to the time course of cultivation (Table 4).<br>
In addition, each crude enzyme solution was prepared from three culture broth containing a different concentration of lactose (0.02, 0.2 and 2%) taken at 48 hr after the cultivation and subjected to denaturing SDS-PAGE (12% SDS-polyacrylamide gel) to examine the expression pattern of a protein. As a result, the CPC acylase mutant was produced in a large quantity in case of culturing with 2% lactose (Fig. 12). Further, there was no inactive precursor band corresponding to the molecular weight of about 83 kDa, which confirms that most of the mactive precursor form convert into the active form of CPC acylase by an efficient self-digestion.<br><table>
<br>
Concentration of lactose	Period of cultivation (hr)	Cell growth (ODfioo)	Productivity of CPC acylase mutant (unit/Â£ )<br>
0%	24	6.8	5<br><br>
	36	6.9	6<br><br>
	48	5.8	3<br>
0.02%	24	4.7	35<br><br>
	36	4.2	36<br><br>
	48	3.5	.32<br>
0.2%	24	5.8	71<br><br>
	36	5.2	72<br><br>
	48	4.4	64<br>
2%	24	5.7	118<br><br>
	36	7.8	132<br><br>
	48	9.5	168<br><br>
	60	11.2	304<br><br>
	72	11.5	334<br><br>
	80	12.2	312<br>
Further, the productivity of CPC acylase by S12 mutant enzyme in the E. coli BL2I(DE3) trasnfromant containing pET29-SI2 recombinant plasmid was measured according to the same method as described above except that 50 ml of a LB broth containing 20 jt/g/mC of kanamycin and 2% lactose was employed for a main cultivation and the E. coli trasnfromant was cultured at 251), 200<br>
k2^<br><br>
rpm for 72 hr with vigorous shaking. As a result, the productivity of V121aA/G139aS/F58pN/I75pT/I176pV/S471pC sixfold CPC acylase mutant (S12) was 1.207 unit/^ .<br>
E. coli BL21(DE3) transformed with pET29-S12 plasmid containing the inventive V121aA/G139aS/F58pN/I75pT/I176pV/S471pC sixfold CPC acylase mutant gene S12 has been designated E. coli BL21(DE3)/pET-S12 which was deposited on July 30, 2003 with the Korean Collection for Type Cultures (KCTC) (Address: Korea Research Institute of Bioscience and Biotechnology (KRIBB), #52, Oun-dong, Yusong-ku, Taejon, 305-333, Republic of Korea) under the accession number KCTC 10503BP, in accordance with the terms of Budapest Treaty on the International Recognition of the Deposit of Microorganism for the Purpose of Patent Procedure.<br>
Example 4: One-step conversion of CPC into 7-ACA using CPC acylase<br>
mutant	^â<br>
E. coli BL2I(DE3) trasnfromant containing each one of pET29-TnS5aP and pET29-S12 plasmids of Example  was cultivated in 5 ^ of a LB broth containing 20 //g/rrtC of kanamycin and 2% lactose at 25 Â°C, 200 rpm for 72 hr with vigorous shaking. Further, E. coli MC1061(pBCPC) containing the wild-type CPC acylase gene was also cultivated in 10 ^ of a LB broth containing 25 iiglvd of chloramphenicol at 25 Â°C, 200 rpm for 48 hr with vigorous shaking. After then, the wild-type CPC acylase, TnS5aP and S12 CPC acylase mutants were purified from each culture broth according to the same method as described in Example , respectively. The purified protein obtained above was subjected to dialysis with 50 mM phosphate buffer solution (pH 8.5), and the diffusate obtained by dialysis was used for a CPC conversion reaction.<br>
After CPC was dissolved in 50 mM phosphate buffer solution (pH 8.5) at a final concentration of 50 mM, the purified enzyme solution dissolved in the same buffer solution was added to the CPC solution obtained above at a final concentration of 5 unit/mC to prepare 50 mC of the reaction mixture. The reaction mixture was stirred at 25*0 for 1 hr to induce a CPC conversion reaction. At this time, in order to prevent pH of the reaction mixture from decreasing during the conversion reaction, when pH of the reaction mixture was reached to 8.4, 0.2 N NaOH solution was automatically injected to the reaction mixture by a pH regulator to constantly maintain pH of the reaction mixture at<br>
^3<br><br>
8.5. 100 id of the reaction mixture was taken from at a fixed time during the conversion reaction and immediately subjected to a HPLC analysis to quantify the amount of CPC and 7-ACA in the reaction mixture. For the HPLC analysis, Symmetry C18 column (Waters, USA) and the mixture (90:10) of 20 mM ammonium acetate buffer solution (pH 5.0) and acetonitrile as a mobile phase were employed. In addition, the flow rate of mobile phase was 0.6 td/min, the sample (20 (d) to be injected was prepared by appropriately diluting the reaction mixture with 50 mM phosphate buffer solution (pH 8.5), and its absorbance was detected at UV 250 ran.<br>
As a result, while the wild-type CPC acylase showed 60% level of CPC conversion rate under the reaction condition described above, TnS5ap and S12 CPC acylase mutants of the present invention, 86% and 98% level of CPC conversion rate, respectively (Fig. 13). In particular, the inventive S12 CPC acylase mutant efficiently converted CPC into 7-ACA at a high level of 7-ACA yield (90% or more) (Figs. 13 and 14).<br>
While the embodiments of the subject invention have been described and illustrated, it is obvious that various changes and modifications can be made therein without departing from the spirit of the present invention which should be limited only by the scope of the appended claims.<br><br><br>
1. A mutant of a cephalosporin C (CPC) acylase composed of an a-subunit according to SEQ ID NO: 4 and a p-subunit according to SEQ ID NO: 5, or a functionally equivalent CPC acylase derivative thereof, characterized in that at least one amino acid selected from the group consisting of Vall21a, Glyl39a and Phel69a of CPC acylase a-subunit of SEQ ID NO: 4 and Phe58p, His70p, Ile75p, Ilel76p and Ser471p of CPC acylase P-subunit of SEQ ID NO: 5 is replaced by another amino acid, wherein the CPC acylase mutant or the functionally equivalent CPC acylase derivative thereof catalyzes the conversion of a compound of formula II from a compound of formula I:<br><br>
wherein, R is acetoxy (-OCOCH3), hydroxy (-0H), hydrogen (-H) w a salt thereof.<br>
2.       The CPC acylase mutant as claimed in claim 1, wherein: Vall21a is replaced by alanine; Glyl39a is replaced by serine; Phel69a is replaced by tyrosine;<br>
Phe58p is replaced by alanine, methionine, leucine, valine, cysteine or<br>
asparagine;<br>
His70p is replaced by serine or leucine;<br>
Ile75p is replaced by threonine;<br>
Ilel76p is replaced by valine; or<br>
Ser47ip is replaced by cysteine.<br><br>
3.	The CPC acylase mutant as claimed in claim 2, wherein PheSSp is replaced by alanine, methionine, leucine, valine, cysteine or aspai-agine, and further MetSip is replaced by leucine.<br>
4.	The CPC acylase mutant as claimed in claim 3, wherein His70p is further replaced by serine or leucine.<br>
5.	The CPC acylase mutant as claimed in claim 4, wherein Phel69a is further replaced by tyrosine.<br>
6.	The CPC acylase mutant as claimed in claim 5, wherein Ilel76p is further replaced by valine.<br>
7.	The CPC acylase mutant as claimed in claim 3, wherein Phel69a is further replaced by tyrosine and Ilel76P is further replaced by valine.<br>
8.	The CPC acylase mutant as claimed in claim 7, wherein Vall21a is further replaced by alanine.<br>
9.	The CPC acylase mutant as claimed in claim 7, v/herein Glyl39a is further replaced by serine.<br>
10.	The CPC acylase mutant as claimed in claim 7, v/herein Vall21a is further replaced by alanine and Glyl39a is further replaced by serine.<br>
11.	The CPC acylase mutant as claimed in claim 2, wherein Vall21a is replaced by alanine; Glyl39a is replaced by serine; PheSSp is replaced by alanine, methionine, leucine, valine, cysteine or asparagine; and Ilel76p is replaced by valine.<br>
12.	The CPC acylase mutant as claimed in claim 11, wherein Phel69a is further replaced by tyrosine.<br>
13.	The CPC acylase mutant as claimed in claim 11, wherein Ile75p is further replaced by threonine.<br><br>
14.	The CPC acylase mutant as claimed in claim 11, wherein Ser47ip is further replaced by cysteine.<br>
15.	The CPC acylase mutant as claimed in claim 11, wherein Ile75p is further replaced by threonine and Ser47ip is further replaced by cysteine.<br>
16.	The CPC acylase mutant as claimed in claim 15, which has an amino acid sequence comprising CPC acylase a-subunit of SEQ ID NO: 7 and CPC acylase p-subunit of SEQ ID NO: 8.<br>
17.	A gene encoding the CPC acylase mutant or a fimctionally equivalent CPC acylase derivative thereof, wherein the CPC acylase mutant or the functionally equivalent derivative thereof is as claimed in claim 1.<br>
18.	The gene as claimed in claim 17, which has the nucleotide sequence described by SEQ ID NO: 6.<br>
19.	A recombinant expression vector containing the gene as claimed in claim 17.<br>
20.	The recombinant expression vector as claimed in claim 19, which is pET29-S12.<br>
21.	A microorganism transformed with the recombinant expression vector as claimed in claim 19.<br>
22.	The microorganism as claimed in claim 21, which is E. coli BL21(DE3) (pET29-S12) (Accession No: KCTC 10503BP).<br>
23.	A method for preparing the CPC acylase mutant as claimed in claim 1, which comprises the steps of cultivating the microorganism as claimed in claim 21 under a suitable condition and recovering the CPC acylase mutant from the culture broth.<br>
24.	The method as claimed in claim 23, which further comprises the step of adding lactose to the culture broth as an inducer at the early stage of cultivation.<br><br>
25.	A composition comprising the CPC acylase mutant as claimed in claim 1 for<br>
preparing 7-aminocephalosporanic acid (7-ACA) by a one-step enzymatic<br>
method.<br>
26.	A method for preparing the compound of Formula II, which comprises the step<br>
of contacting a compound of Formula I with the CPC acylase mutant as<br>
claimed in claim 1:<br><br>
wherein, R is acetoxy (-OCOCH3), hydroxy (-0H), hydrogen (-H) or a salt thereof<br>
27.	The method as claimed in claim 26, wherein the CPC acylase mutant is used in the form of a culture broth of microorganism of claim 21, the composition of claim 25, a free form of the CPC acylase mutant purified from the culture solution, or an immobilized form of the CPC acylase mutant.<br>
28.	The method as claimed in claim 26, wherein the contact reaction of the compound of Formula I with the CPC acylase mutant is carried out in an aqueous solution.<br>
29.	The method as claimed in claim 26, the concentration of compound of Formula<br>
I ranges from 1 to 500 mM and the amount of the added CPC acylase mutant<br>
ranges from 0.1 to 100 U/ml.<br><br>
30. The method as claimed in claim 26, wherein the contact reaction of the compound of Formula I with the CPC acylase mutant is carried out at pH of 7 to 10, 4 to 40'C for 0.1 to 24 hr.<br><br>
</table>
<br><span id="LblCompatibility" style="color:White;font-family:Arial;font-size:Small;">Best View in Resolution of 1024x768 or later. Enable Javascript for Better Performance.</span>
        
    

<script type="text/javascript">
//<![CDATA[
(function() {var fn = function() {$get("ToolkitScriptManager_HiddenField").value = '';Sys.Application.remove_init(fn);};Sys.Application.add_init(fn);})();//]]>
</script></l-l-2>
</table></l-l-2></l-l-2></l-3></l-l-2>
</table></l-l-2></l-l-2></l-l-2></l-l-2></l-l-4></m></l-l-2></l-2-2></l-4-3></l-l-2></l-l-4></l-4-l></l-2-l></l-l-5></l-4-2></l-l-2></l-l-3></l-3></l-4-l></l-4></l-l-2></l-3></l-l-2></l-l-4></l-2-2></l-2-3></l-2-l></l-l-2></l-2-2></l-i-2></l-l-4></l-l-6></l-2-l></l-2></l-l-2></l-l-4></l-l-6></l-l-2></l-l-4></l-i-5></l-l-2></l-l-4></l-l-2></l-l-3></l-l-2>
</table></formula></formula></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYgLSBhYnN0cmFjdC5qcGc=" target="_blank" style="word-wrap:break-word;">515-chenp-2006 - abstract.jpg</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgQUJTVFJBQ1QucGRm" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgQ0xBSU1TIEdSQU5URUQucGRm" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 CLAIMS GRANTED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYgY29tcGxldGUgc3BlY2lmaWNhdGlvbiBhcyBncmFudGVkLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006 complete specification as granted.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgQ09SUkVTUE9OREVOQ0UgUE8ucGRm" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYgZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">515-chenp-2006 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgRk9STSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LUNIRU5QLTIwMDYgUEVUSVRJT05TLnBkZg==" target="_blank" style="word-wrap:break-word;">515-CHENP-2006 PETITIONS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">515-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">515-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZm9ybSAyNi5wZGY=" target="_blank" style="word-wrap:break-word;">515-chenp-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE1LWNoZW5wLTIwMDYtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">515-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234407-an-environmentally-friendly-lubricant-and-a-method-of-making-the-same.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234409-a-process-for-the-manufacture-of-tetrazole-derivative.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234408</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>515/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-May-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Feb-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANDOZ AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>LICHTSTRASSE 35, CH-4056 BASEL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JUNG, KYUNG, HWA,</td>
											<td>SINANJUGONG 1-CHA Apt. 111-802, SINAN-DONG, JINJU-SI, YUNGSANGNAM-DO 660-100, KOREA;</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KIM , YOUNGSOO,</td>
											<td>EUNMA APT. 10-203, #316 , DAECHI-DONG, KANGNAM-KU, SEOUL 135-280 KOREA;</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SHIN, YONG, CHUL,</td>
											<td>HYUNDAI Apt. 118-302, #360, JUYAK-DONG, JINJU-SI, KYUNGSANGNAM-DO 660-290</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JEON, JOHN, YJ,</td>
											<td>HANYANG Apt. 1-907, #54, MYUNGIL 2-DONG, KANGDONG-KU, SOUL 134-072 KOREA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>PARK, MI, RAN,</td>
											<td>WORLDVILLE 103, #559-49, SANSU 1-DONG, DONG-GU, GWANGJU 501-091, KOREA;</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/55, 9/14</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/KRO4/02005</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-10</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10-2003-0055259</td>
									<td>2003-08-11</td>
								    <td>Republic of Korea</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234408-cephalosporin-acylase-mutant-and-method-for-preparing-7-aca-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:14:15 GMT -->
</html>
