<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234508-a-sprayable-composition-for-application-to-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:30 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234508:&quot;A SPRAYABLE COMPOSITION FOR APPLICATION TO PLANT&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A SPRAYABLE COMPOSITION FOR APPLICATION TO PLANT&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The subject invention relates to the surprising discovery that toxin complex (TC) proteins, obtainable from Xenorhabdus, Photorhabdus, and Paenibucillus, can be used interchangeably with each other. In particularly preferred embodiments of the subject invention, the toxicity of a &quot;stand-alone&quot; TC protein (from Photorhabdus, Xenorhabdus, and Paenibucillus, for example) is enhanced by one or more TC protein &quot;potentiators&quot; derived from a source organism of a different genus from which the toxin was derived. As one skilled in the art will recognize with the benefit of this disclosure, this has broad implications and expands the range of utility that individual types of TC proteins will new be recognized to have. Among the most important advantages is that one skilled in the art will now be able to use a single set of potentiators to enhance the activity of a stand-alone Xenorhabdus protein toxin as well as a stand alone Photorlzabdus protein toxin. (As one skilled in the art knows, Xenorhabdus toxin proteins tend to be more desirable for controlling lepidopterans while Photorhabdus toxin proteins tend to be more desirable for controlling coleopterans). This reduces the number of genes, and transformation events, needed to be expressed by a transgenic plant to achieve effective control of a wider spectrum of target pests. Certain preferred combinations of heterolugous TC proteins are also disclosed herein. Other objects, advantages, and features of the subject invention will be apparent to cue skilled in the art having the benefit of the subject disclosure.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DESCRIPTION<br>
MIXING AND MATCHING TC PROTEINS FOR PEST CONTROL<br>
Background of the Invention<br>
Insects and other pests cost farmers billions of dollars annually in crop losses and in the expense of keeping these pests under control. The losses caused by insect pests in agricultural production environments include decreases in crop yield, reduced crop quality, and increased harvesting costs. Insect pests are also a burden to vegetable and fruit growers, to producers of ornamental flowers, and to home gardeners and homeowners.<br>
Cultivation methods, such as crop rotation and the application of high levels of nitrogen<br>
fertilizers, have partially addressed problems caused by agricultural pests. However, various<br>
demands on the utilization of farmland restrict the use of crop rotation.    In addition,<br>
overwintering traits of some insects are disrupting crop rotations in some areas.<br>
[0003]	Thus, synthetic chemical insecticides are relied upon most heavily to achieve a sufficient<br>
level of control. However, the use of synthetic chemical insecticides has several drawbacks. For example, the use of these chemicals can adversely affect many beneficial insects. Target insects have also developed resistance to some chemical pesticides. Furthermore, rain and improper calibration of insecticide application equipment can result in poor control. The use of insecticides often raises environmental concerns such as contamination of soil and water supplies when not used properly, and residues can also remain on treated fruits and vegetables. Working with some insecticides can also pose hazards to the persons applying them. Stringent new restrictions on the use of pesticides and the elimination of some effective pesticides could limit effective options for controlling damaging and costly pests.<br>
[0004]	The replacement of synthetic chemical pesticides, or combination of these agents with<br>
biological pesticides, could reduce the levels of toxic chemicals in the environment. Some biological pesticidal agents that are now being used with some success are derived from the soil microbe Bacillus thuringiensis (B.t.). While most B.t. strains do not exhibit pesticidal activity, some B.t. strains produce proteins that are highly toxic to pests, such as insects, and are specific<br>
in their toxic activity. Genes that encode 5-endotoxin proteins have been isolated. Other species of Bacillus also produce pesticidal proteins.<br>
[ooo5]	Recombinant DNA-based B.t. products have been produced and approved for use. In<br>
addition, with the use of genetic engineering techniques, various approaches for delivering these toxins to agricultural environments are being perfected. These include the use of plants genetically engineered with toxin genes for insect resistance and the use of stabilized intact microbial cells as toxin delivery vehicles. Thus, isolated Bacillus toxin genes are becoming commercially valuable.<br>
[0006]	B.t. protein toxins were initially formulated as sprayable insect control agents.   A<br>
relatively more recent application of B.t. technology has been to isolate and transform plants with genes that encode these toxins. Transgenic plants subsequently produce the toxins, thereby providing insect control. See U.S. Patent Nos. 5,380,831; 5,567,600; and 5,567,862 to Mycogen Corporation. Transgenic B.t. plants are quite efficacious, and usage is predicted to be high in some crops and areas.<br>
[0007]	There are some obstacles to the successful agricultural use of Bacillus (and other<br>
biological) pesticidal proteins. Certain insects can be refractory to the effects of Bacillus toxins. Insects such as boll weevils, black cutworm, and Helicoverpa zea, as well as adult insects of most species, heretofore have demonstrated no significant sensitivity to many B.t. 6-endotoxins.<br>
[0008]	Another potential obstacle is the development of resistance to B. t. toxins by insects. The<br>
potential for wide-spread use of B.t. plants has caused some concern that resistance management issues may arise more quickly than with traditional sprayable applications. While a number of insects have been selected for resistance to B.t. toxins in the laboratory, only the diamondback moth (Plutella xylostella) has demonstrated resistance in a field setting (Ferre, J. and Van Rie, J., Anriu. Rev. Entomol. 47:501-533, 2002).<br>
[0009]	Resistance management strategies inB.t. transgene plant technology have become of great<br>
interest. Several strategies have been suggested for preserving the ability to effectively use B. thwingiensis toxins. These strategies include high dose with refuge, and alternation with, or co-deployment of, different toxins (McGaughey et al. (1998), "B.t. Resistance Management," Nature Biotechnol 16:144-146), as in a natural bacterium, for example.<br>
[0010]	Thus, there remains a great need for developing additional genes that can be expressed in<br>
plants in order to effectively control various insects. In addition to continually trying to discover<br>
new B.t. toxins (which is becoming increasingly difficult due to the numerous B.t. toxins that have already been discovered), it would be quite desirable to discover other bacterial sources (distinct from B.t.) that produce toxins that could be used in transgenic plant strategies.<br>
The' relatively more recent efforts to clone insecticidal toxin genes  from the<br>
Photorhabdus/Xenorhabdus group of bacteria present potential alternatives to toxins derived<br>
from B. thuringiensis. The genus Xenorhabdus is taxonomically defined as a member of the<br>
Family Enterobacteriaceae, although it has certain traits atypical of this family. For example,<br>
strains of this genus are typically nitrate reduction negative and catalase negative. Xenorhabdus<br>
has only recently been subdivided to create a second genus, Photorhabdus, which is comprised of<br>
three species, Photorhabdus asymbiotica, Photorhabdus temperata, and P. luminescens.    P.<br>
luminescens has three recognized subspecies, Photorhabdus luminescens subsp. akhurstii,<br>
Photorhabdus luminescens subsp. lawnondii, and Photorhabdus luminescens subsp. luminescent<br>
(Type species). (Fischer-Le Saux, M., Viallard, V., Brunei, B., Normand, P., Boemare, N. E.<br>
Title Polyphasic classification of the genus Photorhabdus and proposal of new taxa: P.<br>
luminescens subsp. luminescens subsp. nov., P. luminescens subsp. akhurstii subsp. nov., P.<br>
luminescens subsp. lawnondii subsp. nov., P. temperata sp. nov., P. temperata subsp. temperata<br>
subsp. nov. and P. asymbiotica sp. nov. Int. J. Syst. Bacteriol. 49; 1645-1656, (1999)). This<br>
differentiation is based on several distinguishing characteristics easily identifiable by the skilled<br>
artisan. These differences include the following: DNA-DNA characterization studies; phenotypic<br>
presence (Photorhabdus) or absence {Xenorhabdus) of catalase activity; presence (Photorhabdus)<br>
or absence (Xenorhabdus) of bioluminescence; the Family of the nematode host in that<br>
Xenorhabdus is found mSteinernematidae and Photorhabdus is found mffeterorhabditidae); as<br>
well as comparative, cellular fatty-acid analyses (Janse et al. 1990, Lett. Appl. Microbiol. 10,<br>
131-135; Suzuki etal 1990, J. Gen. Appl Microbiol, 36,393-401). In addition, recent molecular<br>
studies focused on sequence (Rainey et al. 1995, Int. J. Syst. Bacterial., 45, 379-381) and<br>
restriction analysis (Brunei et al., 1997, App. Environ. Micro., 63,574-580) of 16S rRNA genes<br>
also support the separation of these two genera.<br>
The expected traits for Xenorhabdus are the following: Gram stain negative rods, white to yellow/brown colony pigmentation, presence of inclusion bodies, absence of catalase, inability to reduce nitrate, absence of bioluminescence, ability to uptake dye from medium, positive gelatin<br>
hydrolysis, growth on Enterobacteriaceae selective media, growth temperature below 37° C, survival under anaerobic conditions, and motility.<br>
[0013]	Currently, the bacterial genus Xenorhabdus is comprised of four recognized species,<br>
Xenorhabdus nematophihis, Xenorhabdus poinarii, Xenorhabdus bovienii and Xenorhabdus beddingii (Brunei et al., 1997, App. Environ. Micro., 63, 574-580). A variety of related strains have been described in the literature (e.g., Akhurst and Boemare 1988 J. Gen. Microbiol., 134, 1835-1845; Boemare etal 1993 Int. J. Syst. Bacterial. 43, pp. 249-255; Putz et al. 1990, Appl. Environ. Microbiol, 56,181-186, Brunei etal., 1997, App. Environ. Micro., 63,574-580,Rainey et al. 1995, Int. J. Syst. Bacterial, 45, 379-381).<br>
[0014]	Photorhabdus    and    Xenorhabdus    spp.     are     Gram-negative    bacteria    that<br>
entomopathogenically and symbiotically associate with soil nematodes. These bacteria are found in the gut of entomopathogenic nematodes that invade and kill insects. When the nematode invades an insect host, the bacteria are released into the insect haemocoel (the open circulatory system), and both the bacteria and the nematode undergo multiple rounds of replication; the insect host typically dies. These bacteria can be cultured away from their nematode hosts. For a more detailed discussion of these bacteria, see Forst and Nealson, 60 Microbiol Rev. 1 (1996), pp. 21-43. Unfortunately, as reported in a number of articles, the bacteria only had pesticidal activity when injected into insect larvae and did not exhibit biological activity when delivered orally.<br>
[0015]	Xenorhabdus and Photorhabus bacteria secrete a wide variety of substances into the<br>
culture medium. See R.H. ffrench-Constant et al. 66 AEMNo. 8, pp. 3310-3329 (Aug. 2000), for a review of various factors involved in Photorhabdus virulence of insects.<br>
[0016]	It has been difficult to effectively exploit the insecticidal properties of the nematode or its<br>
bacterial symbiont. Thus, proteinaceous agents from Photorhabdus/Xenorhabdus bacteria that have oral activity are desirable so that the products produced therefrom could be formulated as a sprayable insecticide, or the genes encoding said proteinaceous agents could be isolated and used in the production of transgenic plants.<br>
[0017]	There has been substantial progress in the cloning of genes encoding insecticidal toxins<br>
from both Photorhabdus luminescens sndXenorhabdus nematophilus. Toxin-complex encoding genes from P. luminescens were examined first. See WO 98/08932. Parallel genes were more<br>
recently cloned from X. nematophilus. See, e.g., Morgan et al, Applied and Environmental Microbiology' 2001,67:2062-69. The degree of "parallelism" is discussed in more detail below.<br>
[0018]	WO 95/00647 relates to the use of Xenorhabdus protein toxin to control insects, but it<br>
does not recognize orally active toxins. WO 98/08388 relates to orally administered pesticidal agents fromXenorhabdus. U.S. Patent No. 6,048,838 relates to protein toxins/toxin complexes, having oral activity, obtainable from Xenorhabdus species and strains.<br>
[0019]	Four different toxin complexes (TCs)—Tea, Tcb, Tec and Ted—have been identified in<br>
Photorhabdus spp. Each of these toxin complexes resolves as either a single or dimeric species on a native agarose gel but resolution on a denaturing gel reveals that each complex consists of a range of species between 25-280 kDa. The ORFs that encode the typical TCs from Photorhabdus, together with protease cleavage sites (vertical arrows), are illustrated in Figure 7. See also R.H. ffrench-Constant and Bowen, 57 Cell. Mol. Life Sci. 828-833 (2000).<br>
[0020]	Genomic libraries of P. luminescens were screened with DNA probes and with<br>
monoclonal and/or polyclonal antibodies raised against the toxins. Four tc loci were cloned: tea, tcb, tec and ted. The tea locus is a putative operon of three open reading frames (ORFs), tcaA, tcaB, and tcaC, transcribed from the same DNA strand, with a smaller terminal ORF (tcaZ) transcribed in the opposite direction. The tec locus also is comprised of three ORFs putatively transcribed in the same direction (tccA, tccB, and tccC). The tcb locus is a single large ORF (tcbA), and the ted locus is composed of two ORFs {tcdA and tcdB}\ tcbA and tcdA, each about 7.5 kb, encode large insect toxins. It was determined that many of these gene products were cleaved by proteases. For example, both TcbA and TcdA are cleaved into three fragments termed i, ii and iii (e.g. TcbAi, TcbAii and TcbAiii). Products of the tea and tec ORFs are also cleaved. See Figure 7. See also R.H. ffrench-Constant and DJ. Bowen, Current Opinions in Microbiology', 1999, 12:284-288.<br>
??/;	As reported in WO 98/08932, protein toxins from the genus Photorhabdus have been<br>
shown to have oral toxicity against insects. The toxin complex produced by Photorhabdus luminescens (W-14), for example, has been shown to contain ten to fourteen proteins, and it is known that these are produced by expression of genes from four distinct genomic regions: tea, tcb, tec, and ted. WO 98/08932 discloses nucleotide sequences for many of the native toxin genes.<br>
[oo22]	Bioassays of the Tea toxin complexes revealed them to be highly toxic to first instar<br>
tomato hornworms (Manduca sexto) when given orally (LD50 of 875 ng per square centimeter of artificial diet). R.H. ffrench-Constant and Bowen 1999. Feeding was inhibited at Tea doses as low as 40 ng/cm~. Given the high predicted molecular weight of Tea, on a molar basis, P. luminescens toxins are highly active and relatively few molecules appear to be necessary to exert a toxic effect. R.H. ffrench-Constant and Bowen, Current Opinions in Micriobiology, 1999, 12:284-288.<br>
[0023]	None of the four loci showed overall similarity to any sequences of known function in<br>
GenBank. Regions of sequence similarity raised some suggestion that these proteins (TcaC and TccA) may overcome insect immunity by attacking insect hemocytes. R.H. ffrench-Constant and Bowen, Current Opinions in Microbiology, 1999, 12:284-288.<br>
[0024]	TcaB, TcbA, and TcdA all show amino acid conservation (-50% identity), compared with<br>
each other, immediately around their predicted protease cleavage sites. This conservation between three different Tc proteins suggests that they may all be processed by the same or similar proteases. TcbA and TcdA also share -50% identity overall, as well as a similar predicted pattern of both carboxy- and amino-terminal cleavage. It was postulated that these proteins might thus be homologs (to some degree) of one another. Furthermore, the similar, large size of TcbA and TcdA, and also the fact that both toxins appear to act on the gut of the insect, may suggest similar modes of action. R.H. ffrench-Constant and Bowen, Current Opinions in Microbiology, 1999, 12:284-288.<br>
[0025]	Deletion/knock-out studies suggest that products of the tea and ted loci account for the<br>
majority of oral toxicity to lepidopterans. Deletion of either of the tea or ted genes greatly reduced oral activity against Manduca sexta. That is, products of the tea and ted loci are oral lepidopteran toxins on their own; their combined effect contributed most of the secreted oral activity. R.H. ffrench-Constant and DJ. Bowen, 57 Cell Mol. Life. Sci. 831 (2000). Interestingly, deletion of either of the tcb or tec loci alone also reduces mortality, suggesting that there maybe complex interactions among the different gene products. Thus, products of the tea locus may enhance the toxicity of ted products. Alternatively, ted products may modulate the toxicity of tea products and possibly other complexes. Noting that the above relates to oral activity against a single insect species, tcb or tec loci may produce toxins that are more active against other groups of insects (or active via injection directly into the insect haemocoel—the<br>
normal route of delivery when secreted by the bacteria in vivo). R.H. ffrench-Constant and Bowen, Current Opinions in Microbiology, 1999,12:284-288.<br>
[0026]	The insect midgut epithelium contains both columnar (structural) and goblet (secretory)<br>
cells. Ingestion of tea products by M. sexta leads to apical swelling and blebbing of large cytoplasmic vesicles by the columnar cells, leading to the eventual extrusion of cell nuclei in vesicles into the gut lumen. Goblet cells are also apparently affected in the same fashion. Products of tea act on the insect midgut following either oral delivery or injection. R.H. ffrench-Constant and D.J. Bowen, Current Opinions in Microbiology, 1999, 12:284-288. Purified tea products have shown oral toxicity against Manduca sexta (LDso of 875 ng/cm2). R.H. ffrench-Constant and D.J. Bowen, 57 Cell Mol. Life Sci. 828-833 (2000).<br>
[0027]	WO 99/42589 and U.S. Patent No. 6,281,413 disclose TC-like ORFs from Photorhabdus<br>
luminescens. WO 00/30453 and WO 00/42855 disclose TC-like proteins from Xenorhabdus. WO 99/03328 and WO 99/54472 (and U.S. Patent Nos. 6,174,860 and 6,277,823) relate to other toxins from Xenorhabdus and Photorhabdus.<br>
[0028]	WO 01/11029 and U.S. Patent No. 6,590,142 Bl disclose nucleotide sequences that<br>
encode TcdA and TcbA and have base compositions that have been altered from that of the native genes to make them more similar to plant genes. Also disclosed are transgenic plants that express Toxin A and Toxin B. These references also disclose Photorhabdus luminescens strain W-14 (ATCC 55397; deposited March 5, 1993) and many other strains.<br>
[0029]	Of the separate toxins isolated from Photorhabdus luminescens (W-14), those designated<br>
Toxin A and Toxin B have been the subject of focused investigation for their activity against target insect species of interest (e.g., corn rootworm). Toxin A is comprised of two different subunits. The native gene tcdA encodes protoxin TcdA. As determined by mass spectrometry, TcdA is processed by one or more proteases to provide Toxin A. More specifically, TcdA is an approximately 282.9 kDa protein (2516 aa) that is processed to provide TcdAi (the first 88 amino acids), TcdAii (the next 1849 aa; an approximately 208.2 kDa protein encoded by nucleotides 265-5811 of tcdA), and TcdAiii, an approximately 63.5 kDa (579 aa) protein (encoded by nucleotides 5812-7551 of tcdA). TcdAii and TcdAiii appear to assemble into a dimer (perhaps aided by TcdAi), and the dimers assemble into a tetramer of four dimers. Toxin B is similarly derived from TcbA.<br><br>
While the exact molecular interactions of the TC proteins with each other, and their mechanism(s) of action, are not currently understood, it is known, for example, that the Tea toxin complex of Photorhabdus is toxic to Manduca sexta. In addition, some TC proteins are known to have "stand alone" insecticidal activity, while other TC proteins are known to potentiate or enhance the activity of the stand-alone toxins. It is known that the TcdA protein is active, alone, against Manduca sexta, but that TcdB and TccC, together, can be used to enhance the activity of TcdA. Waterfield, N. et a!., Appl. Environ. Microbiol. 2001, 67:5017-5024. TcbA (there is only one Tcb protein) is another stand-alone toxin from Photorhabdus. The activity of this toxin (TcbA) can also be enhanced by TcdB together with TccC-like proteins.<br>
[0031]	U.S. Patent Application 20020078478 provides nucleotide sequences for two potentiator<br>
genes, tcdB2 and tccC2, from the ted genomic region of Photorhabdus luminescens W-14. It is shown therein that coexpression of tcdB and tccCl with tcdA in heterologous hosts results in enhanced levels of oral insect toxicity compared to that obtained when tcdA is expressed alone in such heterologous hosts. Coexpression of tcdB and tccCl with tcdA or tcbA provide enhanced oral insect activity.<br>
As indicated in the chart below, Tec A has some level of homology wi th the N terminus of TcdA, and TccB has some level of homology with the C terminus of TcdA. TccA and TccB are much less active on certain test insects than is TcdA. TccA and TccB from Photorhabdus strain W-14 are called "Toxin D." "Toxin A" (TcdA), "Toxin B" (TcbA), and "Toxin C" (TcaA and TcaB) are also indicated below.<br>
Furthermore, TcaA has some level of homology with TccA and likewise with the N terminus of TcdA. Still further, TcaB has some level of homology with TccB and likewise with the N terminus of TcdA.<br>
TccA and TcaA are of a similar size, as are TccB and TcaB. TcdB has a significant level of similarity (both in sequence and size) to TcaC.<br>
(Table Removed)<br>
[0035]	Relatively more recent cloning efforts in Xenorhabdus nematophilus also appear to have<br>
identified novel insecticidal toxin genes with homology to the P. luminescens tc loci. See, e.g., WO 98/08388 and Morgan et al, Applied and Environmental Microbiology 2001, 67:2062-69. InR.H. ffrench-Constant and DJ. Bowen, Current Opinions in Microbiology', 1999,12:284-288, cosmid clones were screened directly for oral toxicity to another lepidopteran, Pieris brassicae. One orally toxic cosmid clone was sequenced. Analysis of the sequence in that cosmid suggested that there are five different ORFs with similarity to Photorhabdus tc genes; orf2 and orf5 both have some level of sequence relatedness to both icbA and tcdA, whereas orfl is similar to tccB, or/3 is similar to tccC and orf4 is similar to tcaC. A number of these predicted ORFs also share the putative cleavage site documented in P. luminescens, suggesting that active toxins might also be proteolytically processed.<br>
The finding of somewhat similar, toxin-encoding loci in these two different bacteria is interesting in terms of the possible origins of these virulence genes. The J£ nematophilus cosmid also appears to contain transposase-like sequences, the presence of which might suggest the potential that these loci can be transferred horizontally between different strains or species of bacteria. A range of such transfer events may also explain the apparently different genomic organization of the tc operons in the two different bacteria. Further, only a subset of X. nematophilus and P. luminescens strains appear to be toxic to M. sexta, suggesting either that different strains lack the tc genes or that they carry a different tc gene compliment. Detailed analysis of the phytogeny of strains and toxins within., and between, these bacterial species should help clarify the likely origin of the toxin genes and how they are maintained in different bacterial populations. R.H. ffrench-Constant and Bowen, Current Opinions in Microbiology, 1999, 12:284-288.<br><br>
 [0037]	There are five typical Xenorhabdus TC proteins: XptAl, XptA2, XptBl, XptCl, and<br>
XptDl. XptAl is a "stand-alone" toxin. XptA2 is another TC protein from Xenorhabdus that has stand-alone toxin activity. XptBl and XptCl are potentiators that can enhance the activity of either (or both) of the XptA toxins. XptDl has some level of homology with TccB. XptCl has some level of similarity to Photorhabdus TcaC. The XptA2 protein of Xenorhabdus has some degree of similarity to the Photorhabdus TcdA protein. XptBl has some level of similarity to Photorhabdus TccC.<br>
[0038]	TC proteins and genes have more recently been described from other insect-associated<br>
bacteria such as Serratia entomophila, an insect pathogen. Pseudomonas species were found to have potentiators. Waterfield et al, TRENDS in Microbiology, Vol. 9, No. 4, April 2001.<br>
[0039]	Bacteria of the genus Paenibacillus are distinguishable from other bacteria by distinctive<br>
rRNA and phenotypic characteristics (C. Ash et al. (1993), "Molecular identification of rRNA group 3 bacilli (Ash, Farrow, Wallbanks and Collins) using a PCR probe test: Proposal for the creation of a new genus Paenibacillus," Antonie Van Leeiiwenhoek 64:253-260). Some species in this genus are known to be pathogenic to honeybees (Paenibacillus larvae) and to scarab beetle grubs (P. popilliae and P. lentimorbus). P. larvae, P. popilliae, and P. lentimorbus are considered obligate insect pathogens involved with milky disease of scarab beetles (D.P. Stahly et al. (1992), "The genus Bacillus: insect pathogens, "pp. 1697-1745, In A. Balows et al, ed., The Procaryotes, 2nd Ed., Vol. 2, Springer-Verlag, New York, NY).<br>
[0040]	A crystal protein, Cry 18, has been identified in strains of Paenibacillus popilliae and<br>
Paenibacillus lentimorbus. Cry IS has scarab and grub toxicity, and has about 40% identity to Cry2 proteins (Zhang et al, 1997; Harrison et al, 2000). TC proteins and lepidopteran-toxic Ciy proteins have very recently been discovered in Paenibacillus. See U.S. Serial No. 60/392,633 (Bintrim et al), filed June 28, 2002. Six TC protein ORFs were found in that strain of Paenibacillus. ORF3 and ORF1 are shown there to each have some level of homology with TcaA. ORF4 and ORF2 are shown there to have some level of homology with TcaB. ORFS appears to be a TcaC-like potentiator, and ORF6 has homology with the TccC potentiator.<br>
[0041]	Although some Xenorhabdus TC proteins were found to "correspond" (have a similar<br>
function and some level of sequence homology) to some of the Photorhabdus TC proteins, a given Photorhabdus protein shares only about 40% sequence identity with the "corresponding" Xenorhabdus protein. This is illustrated below for four "stand-alone" toxins:<br><br>
(Table Removed) <br>
(For a more complete review, see, e.g., Morgan et al, "Sequence Analysis of Insecticidal Genes from Xenorhabdus nematophiles PMFI296," Vol. 67, Applied and Environmental Microbiology, May 2001, pp. 2062-2069.) This approximate degree of sequence relatedness is also observed<br>
1<br>
when comparing the more recently discovered TC proteins from Paenibacillus (those proteins and that discovery are the subject of co-pending U.S. Serial No. 60/392,633) to their Xenorhabdus and Photorhabdus "counterparts."<br>
[0042]	While Photorhabdus toxins have been used successfully, and Xenorhabdus toxins have<br>
been used successfully (apart from Photorhabdus toxins), enhancing the activity of a TC protein toxin from one of these source organisms (such as ^Photorhabdus) with one or more TC protein potentiators from the other (aXenorhabdus, for example) has not heretofore been proposed or demonstrated.<br>
Brief Summary of the Invention<br>
(0043]	The subject invention relates to the surprising discovery that toxin complex (TC) proteins,<br>
obtainable from organisms such as Xenorhabdus, Photorhabdus, and Paenibacillus, can be used interchangeably with each other. As one skilled in the art will recognize with the benefit of this disclosure, this has broad implications and expands the range of utility that individual types of TC proteins will now be recognized to have. This was not previously contemplated, and it would not have been thought possible, especially given the high level of divergence at the sequence level of the TC proteins from Photorhabdus compared to "corresponding" TC proteins of Xenorhabdus and Paenibacillus, for example.<br>
[0044]	In particularly preferred embodiments of the subject invention, the toxicity of a "stand-<br>
alone" TC protein (from Photorhabdus, Xenorhabdus, or Paenibacillus, for example) is enhanced by one or more TC protein "potentiators" derived from a different source organism. The subject invention provides one skilled in the art with many surprising advantages. One of the most important advantages is that one skilled in the art will now be able to use a single pair of potentiators to enhance the activity of a stand-alone Xenorhabdus protein toxin and a stand-alone<br>
Photorhabdus protein toxin. (As one skilled in the art knows, Xenorhabclus toxin proteins tend to be more desirable for controlling lepidopterans while Photorhabdus toxin proteins tend to be more desirable for controlling coleopterans.) This reduces the number of genes (and transformation events) needed to be co-expressed by individual transgenic plants and/or plant cells to achieve effective control of a wider spectrum of target pests.<br>
[0045]	Stated another way, the subject invention relates to the discovery that Xenorhabclus TC<br>
proteins could be used to enhance the activity of Photorhabdus TC proteins and vice versa. Similarly, and also surprisingly, it was discovered that TC proteins Hom'Paenibacilltis could be used in place of Xenorhabdus/Photorhabdus TC proteins, and vice versa. Again, there was no expectation that proteins from these divergent organisms would be compatible with each other; this was not previously proposed or demonstrated. The subject invention was surprising especially in light of the notable differences between Xenorhabclus, Photorhabdus, and Paenibacillus TC proteins (as well as those from other genera) notwithstanding some characteristics they have in common.<br>
[oo46]	Certain preferred combinations of heterologous TC proteins are also disclosed herein.<br>
[0047]	Many objects, advantages, and features of the subject invention will be apparent to one<br>
skilled in the art having the benefit of the subject disclosure.<br>
Brief Description of the Figures<br>
[0048]	Figure 1 shows the orientation of ORFs identified in pDAB2097.<br>
[0049]	Figure 2 shows expression vector plasmid pBT-TcdA.<br>
[0050]	Figure 3 shows expression vector plasmid pET280 vector.<br>
[0051]           '    Figure 4 shows expression vector plasmid pCot-3.<br>
[0052]	Figure 5 is a schematic diagram of pBT constructions.<br>
[0053]	Figure 6 is a schematic diagram of pET/pCot constructions.<br>
[0055]	Figure 7 shows the TC operons from Photorhabdus .<br>
Brief Description of the Sequences<br>
[0055[	SEQ ID NO :1 is the N-terminus of ToxinxwiA 220 IcDa protein (XptA2Wi).<br>
[0056]	SEQ ID NO:2 is an internal peptide of ToxinXwiviA purified toxin (XptA2wi).<br>
 [0057]	SEQ ID NO:3 is an internal peptide of ToxinxwiA purified toxin (XptA2wi).<br>
[oo88]	SEQ ID NO:4 is an internal peptide of ToxinxwA purified toxin (XptA2W;).<br>
[0059]	SEQ ID NO:5 is an internal peptide of ToxinXwiA purified toxin (XpfA2Wi).<br>
[0060]	SEQ ID NO:6 is the pDAB2097 cosmid insert: 39,005 bp.       .	'.<br>
t<br>
[0061]	SEQ ID NO:7 is the pDAB2097 cosmid ORF1: nucleotides 1-1,533 of SEQ DD NO:6.<br>
[0062]	SEQ ID NO:8 is the pDAB2097 cosmid ORF1 deduced protein: 511 aa.'<br>
[oo63]	SEQ ID NO:9 is the pDAB2097 cosmid ORP2 (xptD1m): nucleotides 1,543-5,715 of<br>
SEQ ID N0:6.<br>
[0064]	SEQ ID NO:10 is the pDAB2097 cosmid ORF2 deduced protein:  1,391 aa.<br>
[0065]	SEQ ID NO:11 is the pDAB2097 cosmid ORF3: nucleotides 5,764-7,707 of SEQ ID<br>
NO: 6.<br>
SEQ ED NO :12 is the pDAB2097 cosmid ORF3 deduced protein: 648 aa.<br>
SEQ ID NO:13 is thepDAB2097 cosmid ORF4 (xptAlm)'- nucleotides 10,709-18,277 of<br>
SEQ ID N0:.6.<br>
[0066]	SEQ ID NO:14 is the pDAB2097 cosmid ORF4 deduced protein: 2,523 aa.<br>
[0067]	SEQ ID NO:15 is the pDAB2097 cosmid ORF5 (xptBlm): nucleotides 18,383-21,430<br>
(C)ofSEQIDNO:6.<br>
[0070]	SEQ ID NO:16 is the pDAB2097 cosmid ORF5 deduced protein: 1,016 aa.<br>
SEQ ID NO:17 is the pDAB2097 cosmid OKF6 (xptClwi): nucleotides 21,487-25,965<br>
(C)ofSEQEDNO:6.<br>
SEQ ID NO:18 is the pDAB2097 cosmid ORF6 deduced protein:  1,493 aa.<br>
[0073]	SEQ ID NO:19 is the pDAB2097 cosmid ORF7 (yptA2in): nucleotides 26,021-33,634<br>
(C)ofSEQIDNO:6.<br>
[0074]	SEQ ID NO:20 is the pDAB2097 cosmid ORF7 deduced protein: 2,538 aa.<br>
[0075]	SEQ ID NO:21 is the TcdA gene and protein sequence from GENB ANK Accession No.<br>
API 88483.<br>
[0076]	SEQ ID NO:22 istheTcdBl gene and protein sequence from GENB ANK Accession No.<br>
AF346500.<br>
[0077]	SEQ ID NO:23 is the forward primer used to amplify the TcdB 1 sequence from plasmid<br>
pBC-AS4.<br>
SEQ ID NO:24 is the reverse primer used to amplify the TcdB 1 sequence from plasmid<br>
pBC-AS4.<br>
[0079]	SEQ ID NO:25 is the gene and protein sequence for TccC 1 from GENBAMC Accession<br>
No.AAC38630.1.<br>
[0080]	SEQ ID NO:26 is the forward primer used to amplify TccCl from the pBC KS+ vector.<br>
[0081]	SEQ ID NO:27 is the reverse primer used to amplify TccCl from the pBC KS+ vector.<br>
[0082]	SEQ ID NO:28 is the forward primer used to amplify xptA2wi.<br>
[0083]	SEQ ID NO:29 is the reverse primer used to amplify xptA2fp,.'<br>
[oos4]	SEQ ID NO:30 is the forward primer used to amplify xptClwi,-.<br>
[oo85]	SEQ ID NO:31 is the reverse primer used to amplify xptCl rwi.<br>
[oos6]	SEQ ID NO:32 is the forward primer used to amplify xptBl wi-<br>
[0087]	SEQ ID NO:33 is the reverse primer used to amplify xptBl w,.<br>
[oo88]	SEQ ID NO:34 is the amino acid sequence of the XptA2w protein from Xenorhubmis<br>
nematophilus Xwi.<br>
[0089]	SEQ ID NO:35 is the nucleic acid sequence of ORP3, of Paenibacillus strain DAS 1529,<br>
which encodes a tcaA-like protein.<br>
[0090]	SEQ ID NO:36 is the amino acid sequence encoded by Paenibacillus ORF3.<br>
[0091]	SEQ ID NO:37 is the nucleic acid sequence of ORF4, of Paenibacillus strain DAS 1529,<br>
which encodes a teaB-like protein.<br>
[0092]	SEQ ID NO:38 is the amino acid sequence encoded by Paenibacillus ORF4.<br>
[0093]	SEQ ID NO:39 is the nucleic acid sequence of ORF5, of Paenibacillus strain DAS 1529,<br>
which encodes a tcaC-like protein (pptBl 1529).<br>
[00941	SEQ ID NO:40 is the amino acid seqvience encoded by Paenibacillus ORP5 (PptBl 1529).<br>
[0095]	SEQ ID NO:41 is the nucleic acid sequence of ORF6 (short), of Paenibacillus strain<br>
DAS 1529, which encodes a tccC-like protein (pptCl 15795)-<br>
[0096]	SEQ  ID  NO:42 is a protein sequence encoded by Paenibacillus (short) ORF6<br>
(PptCl 1529S).<br>
[0097]	SEQ ID NO:43 is an alternate (long) protein sequence    (PptCl 15291) encoded by<br>
Paenibacillus ORF6 (long) of SEQ ID NO.55.<br>
[0098]	SEQ ID NO:44 is the nucleotide sequence for TcdB2.<br>
SEQ ID NO:45 is the ainino acid sequence of the TcdB2 protein.<br>
 (00100]	SEQIDNO:46 is the nucleotide sequence of TccC3.<br>
(00101]	SEQ ID NO:47 is the amino acid sequence of the TccC3 protein.<br>
(00102]        •    SEQ ID NO:4S is the native xptB.!xb coding region (4521 bases).<br>
[00103]	SEQ ID NO:49 is the native XptBIxb protein encoded by SEQ ID N0:48 (1506 amino<br>
acids).<br>
[00 104]	SEQ ID NO:50 is the native xptClxb coding region (2889 bases).<br>
(00105]	SEQ ID NO:51 is the native XptClxb protein encoded by SEQ ID NO:50 (962 amino<br>
acids).<br>
[001o6]	SEQ ID NO:52 is the Xba I to Xho I fragment of expression plasraid pDAB6031<br>
comprising the vativexptBlxb, coding region, where bases 40 to 4557 encode the protein of SEQ<br>
ID NO:49 (4595 bases).<br>
(00107]	SEQ ID NO:53 is the Xba I to Xho I fragment of expression plasmid pDAB6032<br>
comprising the nativexp(Clxb coding region, where bases 40 to 2925 encode the protein of SEQ<br>
ID NO:51 (2947 bases).<br>
(00108]	SEQ ID NO:54 is the Xba I to Xho I fragment of expression plaswid pDAB6033<br>
comprising the nativexptBlxb and nativexptClxvCodmg regions, where bases 40 to 4557 encode<br>
the protein of SEQ ED NO:49, and bases 4601 to 7486 encode the protein of SEQ ID NO:51<br>
(7508 bases).<br>
['00109]	SE Q ID NO: 55 is the nucleic acid sequence of ORF6 (long; pptCl 15291), ofPaen ibaciihis<br>
strain DAS 1529, which encodes a tccC-like protein (PptCl 15291.) disclosed in SEQ ID NO:43.<br>
(00110]	SEQ ID NO:56 is the gene and protein sequence for TcaC from GENBANK Accession<br>
No. AF346497.1.<br>
(00111]	SEQ ID NO:57 is the gene and protein sequence for TccC5 from GENBANK Accession<br>
No. AF346500.2.<br>
SEQ ID NO:58 is the protein sequence for TccC2 from GENBANK Accession No.<br>
AAL18492.<br>
SEQ ID NO:59 shows the amino acid sequence for the TcbAw-14 protein.<br>
(00112]	SEQ ID NO:60 shows the amino acid sequence for the SepB protein.<br>
(00115]	SEQ ID-NO: 61 shows the amino acid sequence for the SepC protein.<br>
(00116]       •     SEQ ID NO:62 shows the amino acid sequence for the TcdA2w-14 protein.<br>
(00117]	SEQ ID NO:63 shows the amino acid sequence for the TcdA4w-14 protein.<br>
SEQ ID NO: 64 shows the amino acid sequence for the TccC4w-i4 protein.<br>
Detailed Description of the Invention<br>
[00! 19]	The subject invention relates to the novel use of toxin complex (TC) proteins, obtainable<br>
from organisms such asXenorhabdus, Photorhabdus, and Paenibacillus. As discussed below, one or more TC potentiators were used to enhance the activity of a TC toxin protein that is different from the TC toxin which one or both of the potentiators enhance in nature. As one skilled in the art will recognize with the benefit of this disclosure, this has broad implications and expands the range of utility that individual types of TC proteins will now be recognized to have.<br>
(00120]	It was known that some TC proteins have "stand alone" insecticidal activity, and other TC<br>
proteins were Icnown to enhance the activity of the stand-alone toxins produced by the same given organism. In particularly preferred embodiments of the subject invention, the toxicity of a "stand-alone" TC protein (from Photorhabdus. Xenorhabdus, or Paenibacillus, for example) is enhanced by one or more TC protein "potentiators" derived from a source organism of a different genus.<br>
(00121]	There are three main types of TC proteins. As referred to herein, Class A proteins<br>
("Protein A") are stand alone toxins. Native Class A proteins are approximately 280 kDa.<br>
[00122]	Class B proteins ("Protein B") and Class C proteins ("Protein C") enhance the toxicity of<br>
Class A proteins. As used referred to herein, native Class B proteins are approximately 1 70 kDa, and native Class C proteins are approximately 112 kDa.<br>
(00123]	Examples of Class A proteins are TcbA, TcdA, XptAl, and XptA2. Examples of Class B<br>
proteins are TcaC, TcdB, XptBlxb, and XptClwi- Examples of Class C proteins are TccC, XptClxb, andXptBlwi.<br>
[00124]	The exact mechanism of action for the toxicity and enhancement activities are not<br>
currently Icnown, but the exact mechanism of action is not important. What is important is that the target insect eats or otherwise ingests the A, B, and C proteins.<br>
[00125]	It was known that the TcdA protein is active, alone, against Manduca sexta. It was also<br>
known that TcdBl and TccC, together, can be used to enhance the activity of TcdA. TcbA is another stand-alone Photorhabdus toxin. One combination of TC proteins currently 'contemplated in the art is Tca"C (or TcdB) and TccC (as potentiators) together with TcdA or<br>
TcbA. Similarly mXenorhabdus, it was known that XptB 1 and XptC 1 enhanced the activity of XptAl or XptA2, the. latter of which are each "stand alone" toxins.<br>
[00126]	Although the complex of (TcbA or TcdA) + (TcaC + TccC) might appear to'be a similar<br>
arrangement as the complex of (XptAl or XptA2) + (XptC2 + XptBl), each Photorhabdus component shares only about 40% (approximately) sequence identity with the "corresponding" Xenorhabdus component. The unique TC proteins from Paenibacillus also share only about 40% sequence identity with "corresponding" Photorhabdus and Xenorhabdus TC proteins (those proteins and that discovery are the subject of co-pending U.S. application serial no. 60/392,633, Bintrim et al, filed June 28, 2002).<br>
[ooi27]	It is in this context that it was discovered, as described herein, that Xenorhabdus TC<br>
proteins could be used to enhance the activity of Photorhabdus TC proteins and vice versa. Paenibacillus TC proteins are also surprisingly demonstrated herein to potentiate the activity of Xenorhabdus (and Photorhabdus) TC toxins. This was not previously proposed or demonstrated, and was very surprising especially in light of the notable differences between Xenorhabdus, Photorhabdus, and Paenibacillus TC proteins. There was certainly no expectation that divergent proteins from these divergent organisms would be compatible with each other.<br>
[oo128]	The subject invention can be performed in many different ways.   A plant can be<br>
engineered to produce two types of Class A proteins and a single pair of potentiators (B and C proteins). Every cell of the plant, or every cell in a given type of tissue (such as roots or leaves) can have genes to encode the two A proteins and the B and C pair.<br>
[oo129]	Alternatively, different cells of the plant can produce only one (or more) of each of these<br>
proteins, hi this situation, when an insect bites and eats tissues of the plant, it could eat a cell that produces the first Protein A, another cell that produces the second Protein A, another cell that produces the B protein, and yet another cell that produces the C protein. Thus, what would be important is that the plant (not necessarily each plant cell) produces two A proteins, the B protein, and the C protein of the subject invention so that insect pests eat all four of these proteins when they eat tissue of the plant.<br>
[00130]	Aside from transgenic plants, there are many other ways of administering the proteins, in<br>
a combination of the subject invention, to the target pest. Spray-on applications are known in the<br>
'art. Some or all of the A, B, and C proteins can be sprayed (the plant could produce one or more<br>
of the proteins and the others could be sprayed).   Various types of bait granules for soil<br>
applications, for example, are also known in the art and can be used according to the subject invention.<br>
Many combinations of various TC proteins are shown herein to function in surprising, new ways. One example set forth herein shows the use of TcdBl and TccCl to enhance the activity of XptA2 against corn earworm, for example. Another example set forth herein is the use of XptBl together with TcdBl to enhance the activity of TcdA against corn rootworm, for example. Similarly, and also surprisingly, it was further discovered that TC proteins from Paenibacillus could be used to enhance the activity of TcdA-like and "XptA2xwi-like proteins. Some of the examples included herein are as follows:<br><br>
(Table Removed)<br>
The use of these and other combinations will now be apparent to those skilled in the art having the benefit of the subject disclosure.<br>
[oo132]	Stand-alone toxins such as TcbA, TcdA, XptAl, and XptA2 are each in the approximate<br>
size range of 280 kDa. TcaC, TcdBl, TcdB2, and XptCl are each approximately 170 kDa. TccCl, TccC3, and XptBl are each approximately 112 kDa. Thus, preferred embodiments of the subject invention include the use of a 280-kDa type TC protein toxin (as described herein) with a 170-kDa class TC protein (as described herein) together with a 112-kDa class TC protein (as described herein), wherein at least one of said three proteins is derived from a source organism (such as Photorhabdus, Xenorhabdus, or Paenibacillus) that is of a different genus than the source organism from which one or more of the other TC proteins is/are derived.<br>
[00133]	The subject invention provides one skilled in the art with many surprising advantages.<br>
Among the most important advantages is that one skilled in the art will now be able to use a single pair of potentiators to enhance the activity of a stand-alone Xenorhabdus protein toxin, for example, as well as a stand-alone Photorhabdus protein toxin, for example. (As one skilled in the art knows, Xenorhabdus toxin proteins tend to be more desirable for controlling lepidopterans while Photorhabdus toxin proteins tend to be more desirable for controlling coleopterans.) This reduces the number of genes (and transformation events) needed to be expressed by a transgenic<br>
plant to achieve effective control of a wider spectrum of target pests. That is, rather than having to express six genes—two toxins and two pairs of potentiators—the subject invention allows for the expression of only four genes—two toxins and one pair of potentiator proteins.<br>
[00134]	Thus, the subject invention includes a transgenic plant and/or a transgenic plant cell that<br>
co-expresses a polynucleotide or polynucleotides encoding two (or more) different stand-alone TC protein toxins, and a polynucleotide or polynucleotides encoding a single pair of TC protein potentiators—-a Class B protein and a Class C protein—wherein one or both of said potentiators is/are derived from a bacterium of a genus that is different from the genus from which one of the stand-alone TC protein toxins is derived. Accordingly, one can now obtain a cell having two (or more) TC protein toxins (Class A proteins) that are enhanced by a single pair of protein potentiators (a Class B and a Class C protein). There was no previous suggestion to produce such cells, and certainly no expectation that both (or all) such toxins produced by said cell would be active to adequate levels (due to the surprising enhancement as reported herein). TC proteins, as the term is used herein, are known in the art. Such proteins include stand-alone toxins and potentiators. Bacteria known to produce TC proteins include those of the following genera: Photorhabdus, Xenorhabdus, Paenibacillus, Serraiia, and Pseudomonas, See, e.g., Pseudomonas syringae pv. syringae B728a (GenBank Accession Numbers gi:23470933 and gi:23472543). Any of such TC proteins can be used according to the subject invention.<br>
[oo135]	Examples of stand-alone (Class A) toxins, as the term is used herein, include TcbA and<br>
TcdA from Photorhabus, and XptAl and XptA2 from Xenorhabdus. Toxins in this class are about 280 kDa. Further examples of stand-alone toxins include SepA from Serratia entomophila (GenBank Accession No. AAG09642.1). Class A proteins can be -230 kDa (especially if truncated), -250-290 kDa, -260-285 kDa, and -270 kDa, for example.<br>
[0136]	•   There are two main types or classes of potentiators, as the tennis used herein. 'Examples<br>
of the "Class B" of potentiators (sometimes referred to herein as Potentiator 1) include TcaC, TcdBl, and TcdB2 from Photorhabus, XptCl from Xenorhabdus, and the protein product of ORF5 otPaenibacillus strain DAS 1529. Potentiators in this class are typically in the size range of about 170 kDa. Further examples of-170 kDa class potentiators are SepB from Serratia entomophila (GenBank Accession No. AAG09643.1; reproduced here as SEQ ID NO:60), TcaC homologs from Pseudomonas syringae pv. syringae B728a (GenBank Accession Numbers gi23472544 and gi23059431), and X! nematophilus PO ORF268 (encoded by bases 258-1991 of<br><br>
Figure 2 of WO 20/004855). A preferred -170 kDa potentiator is TcdB2 (SEQ ED NOs:44-45). Class B proteins can be -130-180 kDa, -140-170 kDa, -150-165 kDa, and -155 kDa, for example.<br>
[00137]	Examples of the "Class C" potentiators (sometimes referred to herein as Potentiator 2)<br>
include TccCl andTccC3 fromPhotorhabus, XptBl fromXenorhabdus, and the protein product of ORF6 of Paenibacillus strain DAS 1529. Potentiators in this class are typically in the size range of about 112 kDa. Further examples of-112 kDa class potentiators are SepC from Serratia entomophila (GenBank Accession No. AAG09644.1; reproduced here as SEQ ID N0:61), and TccC homologs from Psendomonas syringae pv. syringae B728a (GenBanlc Accession Numbers gi:23470227, gr.23472546, gi:23472540, 01:23472541, gi:23468542, gi:23472545, gi:2305S175, gi:23058176, gi:23059433, gi:23059435, and gi:23059432). A preferred-112 kDa potentiator is TccC3 (SEQ ID NOs:46-47). Class C proteins can be-90-120 kDa, -95-115 kDa, -100-110 kDa, and -105-107 kDa, for example.<br>
]00118]	WO 02/94867, U.S. Patent Application 20020078478, and Waterfield atal. (TRENDSin<br>
Microbiology Vol. 10, No. 12, Dec. 2002, pp. 541-545) disclose TC proteins that can be used according to the subject invention. For example, Waterfield et al. disclose tcdB2, tccC3, tccC5, tcdA2, tcdA3, and tcdA4 genes and proteins. Any of the relevant TC proteins disclosed by relevant references discussed above in the Background section (and any other references relating to TC proteins) can also be used according to the subject invention.<br>
[00139]	Thus, one embodiment of the subject invention includes a transgenic plant or plant cell<br>
that produces one, two, or more types of stand-alone TC protein toxins, and a single pair of potentiators: Potentiator 1 and Potentiator 2 (examples of each of these three components are given above and elsewhere herein) wherein at least one of said TC proteins is derived from an organism of a genus that is different from the genes from which one or more of the other TC proteins is derived.<br>
[00140]	It should be clear that examples of the subject invention include a transgenic plant or<br>
plant cell that produces/co-expresses one type of a Photorhabdus toxin (e.g., TcbA or TcdA). one type of a Xenorhabdus toxin (e.g., XptAl or XptA2), and a single (one and only one) pair of potentiator proteins (e.g., TcaC and TccC, without XptCl or XptBl; or XptCl and XptBl, without TcaC or TccC; or TcaC and Paenibacillus ORF6 without any other potentiators; or TcdB and XptB 1 without any other potentiators; these combinations are only exemplary; many other<br><br>
combinations would be clear to one skilled in the art having the benefit of the subject disclosure). Additional potentiators could be used according to the subject invention to enhance heterologous toxins, but multiple types of potentiator pairs are not essential. This is one very surprising aspect of the subject invention.<br>
It should also be clear that the subject invention can be defined in many ways—other than in terms of what is co-expressed by a transgenic plant or plant cell. For example, the subject invention includes methods of potentiating the activity of one or more stand-alone TC protein toxin(s) by coexpressing/coproducing it (or them) with a single pair of potentiators, wherein one or both of the potentiators is/are derived from an organism of a genus that is different from the genus of the organism from which the TC protein toxin is derived. The subject invention also includes methods of controlling insect (and like) pests by feeding them one or more types of TC protein toxins together with one or more pairs of potentiators (e.g., TcbA and XptAl and XptCl and XptBl, possibly without TcaC and TccC), including cells that produced this combination of proteins, wherein one or both of the potentiators is/are derived from an organism of a genus that differs from one or both of the stand-alone toxins.<br>
'00142}	Such arrangements were not heretofore contemplated or expected to have activity. One<br>
way of understanding why the subject results were surprising is to consider the sequence relatedness of some of the protein components exemplified herein. For example, XptA2, a standalone toxin from Xenorhabdus, has about 43% sequence identity with TcdA and about 41% identity with TcbA. TcdA and TcbA are each stand-alone toxins from Photorhabdus. XptAl (another stand-alone toxin from Xenorhabdus) has about 45% identity with TcdA and TcbA.<br>
[0143]	TcaC (a Photorhabdus ~170 IcDa potentiator) has about 49% sequence identity with<br>
XptCl (a~170 WL)Z. Xenorhabdus potentiator). TccC (a~l 12 KDz. Photorhabdus potentiator) has about 48% sequence identity with XptBl-'(a ~112 IcDa Xenorhabdus potentiator). 'Heretofore, TcaC+TccC, for example, would not have been expected to enhance the activity of a protein (XptAl or XptA2) that has only 40-45% sequence identity with the native "target" of the TcaC-KTccC association. (The scores reported above were obtained by using the program FASTA 6.0 and are from Morgan et al, "Sequence Analysis of Insecticidal Genes from Xenorhabdus nematophiles PMFI296," Vol. 67, Applied and Environmental Microbiology, May 2001, pp. 2062-2069):<br>
Some examples of components for use according to the subject invention, and their relatedness to each other, include:<br>
Class A Proteins<br><br><br><br><br>
(Table Removed)<br>
Class B Proteins<br>
(Table Removed)<br>
Class C Proteins<br><br>
(Table Removed) <br>
[00145]	Thus, referring to the genus of a bacterium from which a TC protein was derived is not<br>
simply a matter of arbitrary nomenclature. As illustrated above, doing so helps define a class of TC proteins that are relatively conserved amongst themselves (such as a given type of TC protein<br><br>
produced by Photorhabdus species and strains) but which are relatively quite divergent from other "corresponding" TC proteins derived from a different microbial genus (sueh as those produced by various Xenorhabdus species and strains).<br>
[00146]	Another way to define each TC protein component of the subject invention is by a given<br>
protein's degree of sequence identity to a given toxin or potentiator. Means for calculating identity scores are provided herein. Thus, one specific embodiment of the subject invention includes a transgenic plant or plant cell co-producing a toxin having at least 75% sequence identity with XptA2, a toxin having at least 75% identity with TcdA'or TcbA, a potentiator having at least 75% sequence identity with TcdBl or TcdB2, and a potentiator having at least 75% sequence identity with TccCl or TccCS. Other TC proteins can be substituted into the above formula, in accordance with the teachings of the subject invention. Other, more specific ranges of identity scores are provided elsewhere herein.<br>
[00147)	Yet another way of defining a given type of TC protein component of the subject<br>
invention is by the hybridization characteristics of the polynucleotide that encodes it. Much more detailed infomiation regarding such "tests" and hybridization (and wash) conditions is provided throughout the subject specification. Thus, TC proteins for use according to the subject invention can be defined by the ability of a polynucleotide that encodes the TC protein to hybridize with a given "tc" gene.<br>
[0014&amp;]	Applying that guidance to a particular example, an XptA2-type toxin of the subject<br>
invention could be defined as being encoded by a polynucleotide, wherein a nucleic acid sequence that codes for said XptA2-type toxin hybridizes with the xptA2 gene of SEQ ID NO: 19, wherein hybridization is maintained after hybridization and wash under any such conditions described or suggested herein (such as the examples of low, moderate, and high stringency hybridization/wash conditions mentioned herein). Any of the other exemplified or suggested TC proteins (including potentiators or other toxins) could be substituted for XptA2 in this definition, such as TcdB2, TccCS, TcdA, and TcbA.<br>
[00149]	Thus, the subject invention includes a transgenic plant, a transgenic plant cell, or a<br>
bacterial cell that co-expresses certain combinations of polynucleotides that encode TC proteins of the subject invention. It should be clear that the subject invention includes a transgenic plant or plant cell that co-expresses two toxin genes and on1y one pair of potentiators. Thus, the subject invention includes a transgenic plant or plant cell comprising one or more<br><br>
polynucleotides encoding a toxin in a class of a toxin indicated below as Toxin Pair 1, 2, 3, or 4 as follows, and wherein said plant or cell consists of DNA encoding one pair of potentiators selected from the group consisting of proteins in the class of potentiators shown in Potentiator Pair 1, 2, 3, 4, 5, or 6, as indicated below. Stated another way, said plant or cell consists of a polynncleotide segment encoding one potentiator of Potentiator Pair 1, 2, 3, 4, 5, or 6, and said plant or cell consists of another polynucleotide segment encoding the other potentiator of the selected Potentiator Pair.<br><br>
(Table Removed) <br>
[00150]	The plant or cell can comprise genes encoding additional TC protein toxins (e.g., so that<br>
the cell produces TcbA as well as TcdA, and/or XptAl and XptA2), but only one pair of potentiators is used according to preferred embodiments of the subject invention. (Of course, the cell or plant will produce multiple copies of the potentiators; the key is that additional transformation events can be avoided.)<br>
[00151]	Further embodiments of the subject invention include a transgenic cell or plant that co-<br>
expresses a stand-alone protein toxin and a single (no more than one) potentiator pair comprising at least one "heterologous" (derived from a bacterium of a genus that is other than the genus of the organism from which the toxin is derived) TC protein. The subject invention also includes potentiating the insecticidal activity of a TC protein toxin with a pair of TC proteins that are potentiators, wherein at least one (one or both) of said TC protein potentiators is a heterologous TC protein, with respect to the TC protein toxin it helps to potentiate. Sets of toxins and the potentiators used to enhance the toxin include the following combinations:<br>
(Table Removed)<br>
It should be clear that the above matrices are intended to include, for example, TcdB2 + TccC3 (a preferred pair of potentiators) with any of the toxins such as XptAl and/or XplA2 (together with TcbA and/or TcdA).<br>
Other embodiments and combinations will be apparent to one skilled in the art having the benefit of this disclosure.<br>
[00151]	The subject invention also provides "mixed pairs"-of potentiators such as Potentiator<br>
-Pairs 3. 4, and 5 as illustrated above. Such combinations were not heretofore expected (or suggested) to be active as TC protein toxin enhancers. Thus, such "heterologous" combinarions of potentiators can now be selected to maximize their ability to enhance two (for example) insecticidal toxins. That is, one might now find that, for a. given use, TcdB 1 and XptB 1 is a more desirable pair of potentiators than is XptCl and XptBl, for example. Again, this is surprising given the relative degree of sequence divergence between a given Photorhabdus potentiator and a Xenorhabdus potentiator for which it is substituted, as well as the degree of difference between the natural "target" toxins which the potentiators would naturally enhance. Therefore, it should be clear that the subject invention also provides heterologous potentiator pairs (i.e., where the Class B (-170 kDa) potentiator is derived from a bacterial genus that is different from the bacterial genus from which the Class C (~112 kDa) potentiator is derived).<br>
The subject invention is not limited to 280 kDa TC protein toxins and a heterologous 112 kDa and/or 170 kDa TC protein potentiator. As this is the first observation of the ability to "mix and match" Xenorhabdus and Photorhabdus, for example, TC proteins, the subject invention includes any substitution of a Xenorhabdus TC protein with a "corresponding" Photorhabdus TC protein, and vice versa. For example, one skilled in the art will also now seek to use various heterlogous combinations involving "Toxin C" components (as discussed above in the Background section) and "Toxin D" components (e.g., TccA + XptDl).<br>
[00156]	The subject invention also includes the use of a transgenic plant-producing a subject TC<br>
protein combination together with one or more Bacillus thuringiensis Cry proteins, for example.<br>
[00157]	Proteins and toxins.   The present invention provides easily administered, functional<br>
proteins. The present invention also provides a method for delivering insecticidal toxins that are functionally active and effective against many orders of insects, preferably lepidopteran insects. By "functional activity" (or "active against") it is meant herein that the protein toxins function as orally active insect control agents (alone or in combination with other proteins), that the proteins have a toxic effect (alone or in combination with other proteins), or are able to disrupt or deter insect growth and/or feeding which may or may not cause death of the insect. When an insect comes into contact with an effective amount of a "toxin" of the subject invention delivered via transgenic plant expression, formulated protein composition(s), sprayable protein composition(s), a bait matrix or other delivery system, the results are typically death of the insect, inhibition of the growth and/or proliferation of the insect, and/or prevention of the insects from feeding upon the source (preferably a transgenic plant) that makes the toxins available to the insects. Functional proteins of the subject invention can also work together or alone to enhance or improve the activity of one or more other toxin proteins. The terms "toxic," "toxicity," or "toxin" as used herein are'meant to convey that the subject "toxins'" have "functional activity" as ' defined herein.<br>
[00158]	Complete lethality to feeding insects is preferred, but is not required to achieve functional<br>
activity. If an insect avoids the toxin or ceases feeding, that avoidance will be useful in some applications, even if the effects are sublethal or lethality is delayed or indirect. For example, if insect resistant transgenic plants are desired, the reluctance of insects to feed on the plants is as useful as lethal toxicity to the insect's because the ultimate objective is avoiding insect-induced plant damage.<br>
There are many other ways in which toxins can be incorporated into an insect's diet. For example, it is possible to adulterate the larval food source with the toxic protein by spraying the food -with a protein solution, as disclosed herein. Alternatively, the purified protein could be genetically engineered into an otherwise harmless bacterium, which could then be grown in culture, and either applied to the food source or allowed to reside in the soil in an area in which insect eradication was desirable. Also, the protein could be genetically engineered directly into an insect food source. For instance, the major food source for many insect larvae is plant material. Therefore the genes encoding toxins can be transferred to plant material so that said plant material expresses the toxin of interest.<br>
[oo160]	Transfer of the functional activity to plant or bacterial systems typically requires nucleic<br>
acid sequences, encoding the amino acid sequences for the toxins, integrated into a protein expression vector appropriate to the host in which the vector will reside. One way to obtain a nucleic acid sequence encoding a protein with functional activity is to isolate the native genetic material from the bacterial species which produce the toxins, using information deduced from the toxin's amino acid sequence, as disclosed herein. The native sequences can be optimized for expression in plants, for example, as discussed in more detail below. Optimized polynucleotide can also be designed based on the protein sequence.<br>
(00161]	The subject invention provides classes of TC proteins having toxin activities. One way to<br>
characterize these classes of toxins and the polynucleotides that encode them is by defining a polynucleotide by its ability to hybridize, under a range of specified conditions, with an exemplified nucleotide sequence (the complement thereof and/or a probe or probes derived from either strand) and/or by their ability to be amplified by PCR using primers derived from the exemplified sequences.<br>
[00162] ' There are a number of methods for obtaining the 'pesticidal toxins for use according to the subject invention. For example, antibodies to the pesticidal toxins disclosed herein can be used to identify and isolate other toxins from a mixture of proteins. Specifically, antibodies may be raised to the portions of the toxins which are most constant and most distinct from other toxins. These antibodies can then be used to specifically identify equivalent toxins with the characteristic activity by immunoprecipitation, enzyme linked immunosorbent assay (ELISA), or immuno-blotting. Antibodies to the toxins disclosed herein, or to equivalent toxins, or to fragments of these toxins, can be readily prepared using standard procedures. Such antibodies are an aspect of<br>
the subject invention.   Toxins of the subject invention can be obtained from a variety of sources/source microorganisms.<br>
[00163]	One skilled in the art would readily recognize that toxins (and genes) of the subject<br>
invention can be obtained from a variety of sources. A toxin "from" or "obtainable from" any of the subject isolates referred to or suggested herein means that the toxin (or a similar toxin) can be obtained from the isolate or some other source, such as another bacterial strain or a plant. "Derived from" also has this connotation, and includes proteins obtainable from a given type of bacterium that are modified for expression in a plant, for example. Ohe skilled in the art will readily recognize that, given the disclosure of a bacterial gene and toxin, a plant can be engineered to produce the toxin. Antibody preparations, nucleic acid probes (DNA and RNA), and the like may be prepared using the polynucleotide and/or amino acid sequences disclosed herein and used to screen and recover other toxin genes from other (natural) sources.<br>
[0o164]	Polvnucleotides and probes. The subject invention further provides nucleotide sequences<br>
that encode the TC proteins for use according to the subject invention. The subject invention further provides methods of identifying and characterizing genes that encode proteins having toxin activity. In one embodiment, the subject invention provides unique nucleotide sequences that are useful as hybridization probes and/or primers for PCR techniques. The primers produce characteristic gene fragments that can be used in the identification, characterization, and/or isolation of specific toxin genes. The nucleotide sequences of the subject invention encode toxins that are distinct from previously described toxins.<br>
[0015]	The polynucleotides of the subject invention can be used to form complete "genes" to<br>
encode proteins or peptides in a desired host cell. For example, as the skilled artisan would readily recognize, the subject polynucleotides can be appropriately placed under the control of a promoter in a host of interest, as is readily known in the art.<br>
[oo 166]	As the skilled artisan knows, DNA typically exists in a double-stranded form. In this<br>
arrangement, one strand is complementary to the other strand and vice versa. As DNA is replicated in a plant (for example), additional complementary strands of DNA are produced. The "coding strand" is often used in the art to refer to the strand that binds with the anti-sense strand. The mRNA is transcribed from the "anti-sense" strand of DNA. The "sense" or "coding" strand has a series of codons (a codon is three nucleotides that can be read as a three-residue unit to specify a particular amino acid) that can be read as an open reading frame (ORF) to form a<br>
protein or peptide of interest. In order to produce a protein in vivo, a strand of DNA is typically transcribed into a complementary strand of mRNA which is used as the template for the protein. Thus, the subject invention includes the use of the exemplified polyiiucleotides shown in the attached sequence listing and/or equivalents including the complementary strands. RNA and PNA (peptide nucleic acids) that are functionally equivalent to the exemplified DNA are included in the subject invention.<br>
(00167]	In one embodiment of the subject invention, bacterial isolates can be cultivated under<br>
conditions resulting in high multiplication of the microbe. After treatirig the microbe to provide single-stranded genomic nucleic acid, the DNA can be contacted with the primers of the invention and subjected to PCR amplification. Characteristic fragments of toxin-encoding genes will be amplified by the procedure, thus identifying the presence of the toxin-encoding gene(s).<br>
[00168]	Further aspects of the subject invention include genes and isolates identified using the<br>
methods and nucleotide sequences disclosed herein. The genes thus identified encode toxins active against pests.<br>
[oo169]	Proteins and genes for use according to the subject invention can be identified and<br>
obtained by using oligonucleotide probes, for example. These probes are detectable nucleotide sequences which may be detectable by virtue of an appropriate label or may be made inherently fluorescent as described in International Application No. WO 93/16094. The probes (and the polynucleotides of the subject invention) may be DNA, RNA, or PNA. hi addition to adenine (A), cytosine (C), guanine (G), thymine (T), and uracil (U; for RNA molecules), synthetic probes (and polynucleotides) of the subject invention can also have inosine (a neutral base capable of pairing with all four bases; sometimes used in place of a mixture of all four bases in synthetic probes). Thus, where a synthetic, degenerate oligonucleotide is referred to herein, and "N"' or "n" is used generically, "N" or "n" can be G, A, T, C, or inosine. Ambiguity codes as used herein are in accordance with standard IUPAC naming conventions as of the filing of the subject application (for example, R means A or G, Y means C or T, etc.).<br>
[00170]	As is well known in the art, if a probe molecule hybridizes with a nucleic acid sample, it<br>
can be reasonably assumed that the probe and sample have substantial homo logy/similarity/identity. Preferably, hybridization of the polynucleotide is first conducted followed, by washes'under conditions of low, moderate, or high stringency by techniques well-known in the art, as described in, for example, Keller, G.H., M.M. Manak (1987) DNA Probes,<br>
Stockton Press, New York, NY, pp. 169-170. For example, as stated therein, low stringency conditions can be achieved by first washing with 2x SSC (Standard Saline Citrate)/0.1 % SDS (Sodium Dodecyl Sulfate) for 15 minutes at room temperature. Two washes are typically performed. Higher stringency can then be achieved by lowering the salt concentration and/or by raising the temperature. For example, the wash described above can be followed by two washings with O.lx SSC/0.1% SDS for 15 minutes each at room temperature followed by subsequent washes with O.lx SSC/0.1% SDS for 30 minutes each at 55° C. These temperatures can be used with other hybridization and wash protocols set forth herein and as would be known to one skilled in the art (SSPE can be used as the salt instead of SSC, for example). The 2x SSC/0.1% SDS can be prepared by adding 50 ml of 20x SSC and 5 ml of 10% SDS to 445 ml of water. 20x SSC can be prepared by combining NaCl (175.3 g/0.150 M), sodium citrate (88.2 g/0.015 M), and water, adjusting pH to 7.0 with 10 N NaOH, then adjusting the volume to 1 literlO% SDS can be prepared by dissolving 10 g of SDS in 50 ml of autoclaved water, then diluting to 100ml.<br>
[00171]	Detection of the probe provides a means for determining in a known manner whether<br>
hybridization has been maintained. Such a probe analysis provides a rapid method for identifying toxin-encoding genes of the subject invention. The nucleotide .segments which are used as probes according to the invention can be synthesized using a DNA synthesizer and standard procedures. These nucleotide sequences can also be used as PCR primers to amplify genes of the subject invention.<br>
[00172]	Hybridization characteristics of a molecule can be used to define polynucleotides of the<br>
subject invention. Thus the subject invention includes polynucleotides (and/or their complements, preferably their full complements) that hybridize with a polynucleotide exemplified herein. That is, one way to define a tcdA-like gene (and the protein it encodes), for example, is by its ability to hybridize (under any of the conditions specifically disclosed herein) with a previously known, including a specifically exemplified, tcdA gene. The same is true for xptA2-, tcaC-, tcaA-, tcaB-, tcdB-, tccC-, and xptBl-like genes and related proteins, for example. This also includes the tcdB2 and tccC3 genes.<br>
(00173]	As used herein, "stringent" conditions for hybridization refers to conditions which<br>
achieve the same, or about the same, degree of specificity of hybridization as the conditions employed by the current applicants. Specifically, hybridization of immobilized DNA on<br>
Southern blots with 32P-labeled gene-specific probes was performed by standard methods (see, e.g., Maniatis, T., E.F. Fritsch, J. Sambrook [1982] Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY). In general, hybridization and subsequent washes were carried out under conditions that allowed for detection of target sequences. For double-stranded DNA gene probes, hybridization was carried out overnight at 20-25° C below the melting temperature (Tm) of the DNA hybrid in 6x SSPE, 5x Denhardt's solution, 0.1% SDS, 0.1 mg/ml denatured DNA. The melting temperature is described by the following formula (Beltz, G.A., K.A. Jacobs, T.H. Eickbush, P.T. Cherbas, and F.C. Kafatos [1983] Methods of Enzymology, R. Wu, L. Grossman and K. Moldave [eds.] Academic Press, New York 100:266-285):<br>
Tm - 81.5° C + 16.6 Log[Na+] + 0.41 (%G+C) - 0.61(%formamide) - 600/length of<br>
duplex in base pairs.<br>
[10174]	Washes are typically carried out as follows:<br>
(1)	Twice at room temperature for 15 minutes in Ix SSPE, 0.1 % SDS (low stringency<br>
wash).<br>
(2)	Once at Tm-20°C for 15 minutes in 0.2x SSPE, 0.1 % SDS (moderate stringency<br>
wash).<br>
[00 75]	For oligonucleotide probes, hybridization was earned out overnight at 10-20 ° C below the<br>
melting temperature (Tm) of the hybrid in 6x SSPE, 5x Denhardt's solution, 0.1% SDS, 0.1 mg/ml denatured DNA. Tm for oligonucleotide probes was determined by the following formula:<br>
Tm (c C) = 2(number T/A base pairs) + 4(number G/C base pairs)<br>
(Suggs, S.V., T. Miyake, E.H. Kawashime, M.J. Johnson, K. Itakura, and R.B. Wallace [1981]<br>
ICN-UCLA Symp. Dev. Biol. Using Purified Genes, D.D. Brown [ed.], Academic Press, New<br>
York, 23:683-693).<br>
[00176]	Washes were typically carried out as follows:<br>
())       Twice at room temperature for 15 minutes 1 x SSPE, 0.1 % SDS (low stringency wash).<br>
(2)       Once at the hybridization temperature for 15 minutes in Ix SSPE, 0.1% SDS (moderate stringency wash).<br>
 (00177]	m general, salt and/or temperature can be altered to change stringency. With a labeled<br>
DNA fragment &gt;70 or so bases in length, the following conditions can be used:<br>
Low:	1 or 2x SSPE, room temperature<br>
Low:	lor2xSSPE,42° C<br>
Moderate:       0.2x or Ix SSPE, 65° C<br>
High:	O.lxSSPE, 65° C.<br>
(00178]	Duplex formation and stability depend on substantial complementarity between the two<br>
strands of a hybrid, and, as noted above, a certain degree of mismatch can be tolerated.<br>
Therefore, the probe sequences of the subject invention include mutations (both single and<br>
multiple), deletions, insertions of the described sequences, and combinations thereof, wherein<br>
said mutations, insertions and deletions permit formation, of stable hybrids with the target<br>
polynucleotide of interest. Mutations, insertions, and deletions can be produced in a given<br>
polynucleotide sequence in many ways, and these methods are known to an ordinarily skilled<br>
artisan. Other methods may become known in the future.	' .<br>
[00179]	PCR technology. Polymerase Chain Reaction (PCR) is a repetitive, enzymatic, primed<br>
synthesis of a nucleic acid sequence. This procedure is well known and commonly used by those skilled in this art (see Mullis, U.S. Patent Nos. 4,683,195, 4,683,202, and 4,800.159; Saiki, Randall K., Stephen Scharf, Fred Faloona, Kary B. Mullis, Glenn T. Horn, Henry A. Erlich, Norman Arnheim [1985] 'Enzymatic Amplification of ß-Globin Genomic Sequences and Restriction Site Analysis for Diagnosis of Sickle Cell Anemia," Science 230:1350-1354). PCR is based on the enzymatic amplification of a DNA fragment of interest that is flanked by two oligonucleotide primers that hybridize to opposite strands of the target sequence. The primers are oriented with the 3' ends pointing towards each other. Repeated cycles of heat denaturation of the template, annealing of the primers 'to their complementary sequences, and extension of the annealed primers with a DNA polymerase result in the amplification of the segment defined by the 5' ends of the PCR primers. The extension product of each primer can serve as a template for the other primer, so each cycle essentially doubles the amount of DNA fragment produced in the previous cycle. This results in the exponential accumulation of the specific target fragment, up to . several million-fold in a few hours. By using a thermostable DNA polymerase such as Tag polymerase, isolated from the thermophilic bacterium Thermits aquaticus, the amplification<br><br>
process can be completely automated.  Other enzymes which can be used are known to those skilled in the art.<br>
The DNA sequences of the subject invention can be used as primers for PCR amplification. In performing PCR amplification, a certain degree of mismatch can be tolerated between primer and template. Therefore, mutations, deletions, and insertions (especially additions of nucleotides to the 5' end) of the exemplified primers fall within the scope of the subject invention. Mutations, insertions, and deletions can be produced in a given primer by methods known to an ordinarily skilled artisan.<br>
(00181]	Modification of genes and toxins. The genes and toxins useful according to the subject<br>
invention include not only the specifically exemplified full-length sequences, but also portions, segments and/or fragments (including internal and/or terminal deletions compared to the full-length molecules) of these sequences, variants, mutants, chimerics, and fusions thereof. Proteins of the subject invention can have substituted amino acids so long as they retain the characteristic pesticidal/ functional activity of the proteins specifically exemplified herein. "Variant" genes have nucleotide sequences that encode the same toxins or equivalent toxins having pesticidal activity equivalent to an exemplified protein. The terms "variant proteins" and "equivalent toxins" refer to toxins having the same or essentially the same biological/functional activity against the target pests and equivalent sequences as the exemplified toxins. As used herein, reference to an "equivalent" sequence refers to sequences having amino acid substitutions, deletions, additions, or insertions which improve or do not adversely affect pesticidal activity. Fragments retaining pesticidal activity are also included in this definition. Fragments and other equivalents that retain the same or similar function, or "toxin activity," as a corresponding fragment of an exemplified toxin are within the scope of the subject invention. Changes, such as amino acid substitutions or additions, can be made for a variety of purposes, such as increasing (or decreasing) protease stability of the protein (without materially/substantially decreasing the functional activity of the toxin).<br>
(00182]	Equivalent toxins and/or genes encoding these equivalent toxins can be obtained/derived<br>
from wild-type or recombinant bacteria and/or from other wild-type or recombinant organisms using the teachings provided herein. Other Bacillus, Serratia, Paenibacillus, Photorhabdus, and Xenorhabdus species, for example, can be used as source isolates.<br><br>
(00183]	Variations of genes may be readily constructed using standard techniques for making<br>
point mutations, for example. In addition, U.S. Patent No. 5,605,793, for example, describes methods for generating additional molecular diversity by using DNA reassembly after random fragmentation. Variant genes can be used to produce variant proteins; recombinant hosts can be used to produce the variant proteins. Using these "gene shuffling" techniques, equivalent genes and proteins can be constructed that comprise any 5,10, or 20 contiguous residues (amino acid or nucleotide) of any sequence exemplified herein. As one skilled in the art knows, the gene shuffling techniques can be adjusted to obtain equivalents having, for example, 3,4,5, 6, 7, 8,9, 10, 11,12,13,14, 15, 16, 17, 18,19, 20, 21, 22, 23, 24, 25,26, 27, 28, 29; 30, 31, 32, 33, 34, 35, 36, 37, 38, 39,40, 41,42, 43,44,45, 46,47,48,49, 50, 51, 52, 53, 54, 55; 56, 57,58, 59, 60,61, 62,63,64, 65,66, 67, 68, 69, 70, 71, 72, 73,74, 75, 76, 77,78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104,105, 106,107, 108, 109, 110, 111, 112,113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125,126, 127, 128, 129, 130.. 131,1.32, 133, 134, 135, 136, 137, 138, 139, 140, 141,142, 143, 144, 145, 146, 147, , 148, 149, 150, 151, 152, 153, 154, 155,156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, . 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178,179, 180, 181, 182, 183, 184, 185, , 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199,200,201,202,203,204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 291, 292, 293, 294, 295, 296, 297, 298, 299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310, 311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 324,325, 326, 327, 328, 329,330-, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400, 401, 402, 403, 404, 405, 406, 407, 408, 409, 410, 411, 412, 413, 414, 415, 416, 417, 418, 419, 420, 421, 422, 423, 424, 425, 426, 427, 428, 429, 430, 431, 432, -433, 434, 435, 436, 437, 438, 439, 440, 441. 442, 443, 444, 445, 446, 447, 448, 449, 450, 451, 452, 453, 454, 455, 456, 457, 458, 459, 460, 461, 462, 463, 464, 465, 466, 467, 468, 469, 470,<br><br>
471, 472, 473, 474, 475, 476, 477, 478, 479,480, 481, 482, 483, 484, 485, 486, 487,4S8, 489, 490, 491, 492, 493, 494, 495, 496, 497, 498, 499, or_500 contiguous residues (ammo acid or nucleotide), corresponding to a segment (of the same size) in any of the exemplified or suggested sequences (or the complements (full complements) thereof). Similarly sized segments, especially those for conserved regions, can also be used as probes and/or primers.<br>
(00184]	Fragments of full-length genes can be made using commercially available exonucleases or<br>
endonucleases according to standard procedures. For example, enzymes such as BciB 1 or site-directed mutagenesis can be used to systematically cut off nucleotides irom the ends of these genes. Also, genes which encode active fragments may be obtained using a variety of restriction enzymes. Proteases may be used to directly obtain active fragments of these toxins.<br>
It is within the scope of the invention as disclosed herein that toxins (and TC proteins) may be truncated and still retain functional activity. By "truncated toxin" is meant that a portion of a toxin protein may be cleaved and yet still exhibit activity after cleavage. Cleavage can be achieved by proteases inside or outside of the insect gut. Furthermore, effectively cleaved proteins can be produced using molecular biology techniques wherein the DNA bases encoding said toxin are removed either through digestion with restriction endonucleases or other techniques available to the skilled artisan. Alter truncation, said proteins can be expressed in heteroiogous systems such as E. coli, baculoviruses, plant-based viral systems, yeast and the like and then placed in insect assays as disclosed herein to determine activity. It is well-known in the art that truncated toxins can be successfully produced so that they retain functional activity while having less than the entire, full-length sequence. It is well known in the art that B.t. toxins can be used in a truncated (core toxin) form. See, e.g., Adang et al., Gene 36:289-300 (1985), "Characterized full-length and truncated plasmid clones of the crystal protein of Bacillus thuringiensis subsp 'kurstaki HD-73 and their toxicity to Manduca'sexta:" There are other examples of truncated proteins that retain insecticidal activity, including the insect juvenile hormone esterase (U.S. Pat. No. 5,674,485 to the Regents of the University of California). As used herein, the term "toxin" is also meant to include functionally active truncations.<br>
In some cases, especially for expression in plants, it can be advantageous to use truncated genes that express truncated proteins. Hofte et al. 1989, for example, discussed in the Background Section above, discussed protoxin and core toxin segments of B.t. toxins. Preferred truncated genes will typically encode 40,41,42,43,44, 45,46,47,48,49, 50,51,52,53, 54,55,<br><br>
56, 57, 58,59, 60, 61, 62, 63, 64,65, 66, 67,68,69,70, 71, 72, 73, 74, 75, 76, 77, 78,79, 80, 81, 82, 83, 84, 85, 86,87, 88, 89,90,91, 92, 93,94,95,96, 97, 98, or 99% of the full-length protein. The Background section also discusses protease processing and reassembly of the segments of TcdA and TcbA, for example.<br>
(00187]	Certain toxins/TC proteins of the subject invention have been specifically exemplified<br>
herein. As- these toxins/TC proteins are merely exemplary of the proteins of the subject<br>
invention, it should be readily apparent that the subject invention comprises variant or equivalent<br>
proteins (and nucleotide sequences coding for equivalents thereof) having the same or similar<br>
toxin activity of the exemplified proteins. Equivalent proteins will have amino acid similarity<br>
(and/or homology) with an exemplified toxin/TC protein. The amino acid identity will typically<br>
be greater than 60%, preferably greater than 75%, more preferably greater than 80%, even more<br>
preferably greater than 90%, and can be greater than 95%. Preferred polynucleotides and proteins<br>
of the subject invention can also be defined in terms of more particular identity and/or similarity<br>
ranges. For example, the identity and/or similarity can be 49, 50. 51. 52, 53, 54, 55, 56, 57, 58,<br>
59, 60, 61, 62, 63,64, 65,66,67,68, 69,70, 71, 72: 73,74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84,<br>
85, 86, 87, 88, 89.. 90, 91, 92, 93, 94, 95, 96, 97, 98. or 99% as compared to a sequence<br>
exemplified or suggested herein. Any number listed above can be used to define the upper and<br>
lower limits. For example, a Class A protein can be defined as having 50-90% identity with a<br>
given TcdA protein. Thus, a TcdA-like protein (and/or a tcdA-like gene) can be defined by any<br>
numerical identity score provided or suggested herein, as compared to any previously known<br>
TcdA protein, including any TcdA protein (and likewise with XptA2 proteins) specifically<br>
exemplified herein. The same is true for any other protein or gene, to be used according to the<br>
subject invention, such as TcaC-, TcaA-, TcaB-, TcdB-, TccC-, and XptB2-like proteins and<br>
genes. Thus, this applies to potentiators (such as TcdB2 and TccC3) and stand-alone toxins.<br>
(00188]	Unless otherwise specified, as used herein, percent sequence identity and/or similarity of<br>
two nucleic acids is determined using the algorithm of Karlin and Altschul (1990), Proc. Natl. Acad. Sci. USA 87:2264-2268, modified as in Karlin and Altschul (1993), Proc. Natl. Acad. Sci. USA 90:5873-5877. Such an algorithm is incorporated into the NBLAST and XBLAST programs of Altschul et al. (1990), J. Mol. Biol. 215:402-410. BLAST nucleotide searches are performed with the NBLAST program, score = 100, wordlength = 12. Gapped BLAST can be used as described in Altschul et al. (1997), Nucl. Acids Res. 25:3389-3402. When utilizing<br><br>
BLAST and Gapped BLAST programs, the default parameters of the respective programs (NBLAST and XBLAST) are used. See NCBI/NIH website. The scores can also be calculated using the methods and algorithms of Crickmore et al. as described in the Background section, above. To obtain gapped alignments for comparison purposes, the AlignX function of Vector NTI Suite 8 (InforMax, Inc., North Bethesda, MD, U.S.A.), was used employing the default parameters. These were: a Gap opening penalty of 15, a Gap extension penalty of 6.66, and a Gap separation penalty range of 8.<br>
[oo189]	The amino acid homology/similariry/identity will be highest in critical regions of the<br>
protein that account for its toxin activity or that are involved in the determination of three-dimensional configurations that are ultimately responsible for the toxin activity. In this regard, certain amino acid substitutions are acceptable and can be expected to be tolerated. For example, these substitutions can be in regions of the protein that are not critical to activity. Analyzing the crystal structure of a protein, and software-based protein structure modeling, can be used to identify regions of a protein that can be modified (using site-directed mutagenesis, shuffling, etc.) to actually change the properties and/or increase the functionality of the protein.<br>
. Various properties and three-dimensional features of the protein can also be changed without adversely affecting the toxin activity/functionality of the protein. Conservative amino acid substitutions can be expected to be tolerated/to not adversely affect the three-dimensional configuration of the molecule. Amino acids can be placed in the following classes: non-polar^ uncharged polar, basic, and acidic. Conservative substitutions whereby an amino acid of one class is replaced with another amino acid of the same type fall within the scope of the subject invention so long as the substitution is not adverse to the biological activity of the compound. Table 1 provides a listing of examples of amino acids belonging to each class.<br>
Table 1.<br>
(Table Removed)<br>
Iii some instances, non-conservative substitutions can also be made. The critical factor is that these substitutions must not significantly detract from the functional/biological/toxin activity of the protein.<br>
[00192]	As used herein, reference to "isolated" polynucleotides and/or "purified" toxins refers to<br>
these molecules when they are not associated with the other molecules with which they would be found in nature. Thus, reference to "isolated" and/or "purified" signifies the involvement of the "hand of man" as described herein. For example, a bacterial toxin "gene" of the subject invention put into a plant for expression is an "isolated polynucteotide." Likewise, a Paenibacillus protein, exemplified herein, produced by a plant is an "isolated protein."<br>
[oo193]	Because of the degeneracy/redundancy of the genetic code, a variety of different DNA<br>
sequences can encode the amino acid sequences disclosed herein. It is well within the skill of a person trained in the art to create alternative DNA sequences that encode the same, or essentially the same, toxins. These variant DNA sequences are within the scope of the subject invention.<br>
(00194]	Optimization of sequence for expression in plants.   To obtain high expression of<br>
•heterologous genes in plants it may be preferred to reengineer said genes so that they are more efficiently expressed in (the cytoplasm of) plant cells. Maize is one such plant where it may be preferred to re-design the heterologous gene(s) prior to transformation to increase the expression level thereof in said plant. Therefore, an additional step in the design of genes encoding a bacterial toxin is reengineering of a heterologous gene for optimal expression.<br>
[ool95]	One reason for the reengineering of a bacterial toxin for expression in maize is due to the<br>
non-optimal G+C content of the native gene. For example, the very low G+C content of many native bacterial gene(s) (and consequent skewing towards high A+T content) results in the generation of sequences mimicking or duplicating plant gene control sequences that are known to • be highly A+T rich. The presence of some A+T-rich sequences within the DNA of gene(s) introduced into plants (e.g., TATA box regions normally found in gene promoters) may result in aberrant transcription of the gene(s). On the other hand, the presence of other regulatory sequences residing in the transcribed mRNA (e.g., polyadenylation signal sequences (AAUAAA), or sequences complementary to small nuclear RNAs involved in pre-mRNA splicing) may lead to RNA instability. Therefore, one goal in the design of genes encoding a bacteria] toxin for maize expression, more preferably referred to as plant optimized gene(s), is to generate a DNA sequence having a higher G+C content, and preferably one close to that of maize<br><br>
genes coding for metabolic enzymes. Another goal in the design of the plant optimized gene(s) encoding a bacterial toxin is to generate a DNA sequence in which the sequence modifications do not hinder translation.<br>
(00196)	The table below (Table 2) illustrates how high the G+C content is in maize. For the data<br>
in Table 2, coding regions of the genes were extracted from GenBank (Release 71) entries, and base compositions were calculated using the Mac Vector™ program (Accelerys, San Diego, California). Intron sequences were ignored in the calculations.<br>
(00196]	Due to the plasticity afforded by the redundancy/degeneracy of the genetic code (i.e.,<br>
some amino acids are specified by more than one codon), evolution of the genomes in different organisms or classes of organisms has resulted in differential usage of redundant codons. This "codon bias" is reflected in the mean base composition of protein coding regions. For example, organisms with relatively low G+C contents utilize codons having A or T in the third position of redundant codons, whereas those having higher G+C contents utilize codons having G or C in the third position. It is thought that the presence of "minor" codons within an mKNA may reduce the absolute translation rate of that mRNA, especially when the relative abundance of the charged tRNA corresponding to the minor codon is low. An extension of this is that the diminution of translation rate by individual minor codons would be at least additive for multiple minor codons. Therefore, mRNAs having high relative contents of minor codons would have correspondingly low translation rates. This rate would be reflected by subsequent low levels of the encoded protein.<br>
[001981	In engineering genes encoding a bacterial toxin for maize (or other plant, such as cotton •<br>
or soybean) expression, the codon bias of the plant has been determined. The codon bias for maize is die statistical codon distribution that the plant uses for coding its proteins and the • preferred codon usage is shown in Table 3. After determining the bias, the percent frequency of the codons in the gene(s) of interest is determined. The primary codons preferred by the plant should be determined, as well as the second, third, and fourth choices of preferred codons when multiple choices exist. A new DNA sequence can then be designed which encodes the amino sequence of the bacterial toxin, but the new DNA sequence differs from the native bacterial DNA sequence (encoding the toxin) by the substitution of the plant (first preferred, second preferred, third preferred, or fourth preferred) codons to specify the amino acid at each position within the toxin amino acid sequence. The new sequence is then analyzed for restriction enzyme sites that<br><br>
might have been created by the modification. The identified sites are further modified by replacing the codons with first, second, third, or fourth choice preferred codons. Other sites in the sequence which could affect transcription or translation of the gene of interest are the exon:intron junctions (5' or 3'), poly A addition signals, or RNA polymerase termination signals. The sequence is further analyzed and modified to reduce the frequency of TA or GC doublets. In addition to the doublets, G or C sequence blocks that have more than about four residues that are the same can affect transcription of the sequence. Therefore, these blocks are also modified by replacing the codons of first or second choice, etc. with the next preferred codon of choice.<br>
(Table Removed)<br>
1Q199J	It is preferred that the plant optimized gene(s) encoding a bacterial toxin contain about<br>
63% of first choice codons, between about 22% to about 37% second choice codons. and between about 15% to about 0% third or fourth choice codons, wherein the total percentage is 100%. Most preferred the plant optimized gene(s) contains about 63% of first choice codons, at least about 22% second choice codons, about 7.5% third choice codons, and about 7.5% fourth choice codons, wherein the total percentage is 100%. The preferred codon usage for engineering genes for maize expression are shown in Table 3. The method described above enables one skilled in the art to modify gene(s) that are foreign to a particular plant so that the genes are optimally expressed in plants. The method is further illustrated in PCT application WO 97/13402.<br>
'00200]	In order to design plant optimized genes encoding a bacterial toxin, a DNA sequence is<br>
designed to encode the amino acid sequence of said protein toxin utilizing a redundant genetic code established from a codon bias table compiled for the gene sequences for the particular plant, as shown in Table 2. The resulting DNA sequence has a higher degree of codon diversity, a.<br><br>
desirable base composition, can contain strategically placed restriction enzyme recognition sites, and lacks sequences that might interfere with transcription of the gene, or translation of the product mRNA.<br>
(Table Removed)<br>
*The first and second preferred codons for maize.<br>
'00201]	Thus, synthetic .-genes that are functionally equivalent to .the toxins/genes of the subject<br>
invention can be used to transform hosts, including plants. Additional guidance regarding the production of synthetic genes can be found in, for example, U.S. Patent No. 5,380,831.<br>
'00202]	Transgenic hosts. The toxin-encoding genes of the subject invention can be introduced<br>
into a wide variety of microbial or plant hosts. In preferred embodiments, transgenic plant cells and plants are used. Preferred plants (and plant cells) are corn, .maize, and cotton.<br>
[00203] • In preferred embodiments, expression of the toxin gene results, directly or indirectly, in the intracellular production (and maintenance) of the pesticide proteins. Plants can be rendered insect-resistant in this manner. When transsemc/recombinant/transformed/transfected host cells<br><br>
(or contents thereof) are ingested by the pests, the pests will ingest the toxin. This is the preferred manner in which to cause contact of the pest with the toxin. The result is control (killing or making sick) of .the pest. Sucking pests can also he controlled in a similar manner. Alternatively, suitable microbial hosts, e.g.., Psendomonas such as P.fluorescens, can be applied<br>
where target pests are present; the microbes can proliferate there, and are ingested by the target<br>
 pests.  The microbe hosting the toxin gene can be treated under conditions that prolong the<br>
activity of the toxin and stabilize the cell. The treated cell, which retains the toxic activity, can then be applied to the environment of the target pest.<br>
[00204]	Where the toxin gene is introduced via a suitable vector into a microbial host, and said<br>
host is applied to the environment in a living state, certain host microbes should be used. Microorganism hosts are selected which are known to occupy the "phytosphere" (phylloplane, phyllosphere, rhizosphere, and/or rhizoplane) of one or more crops of interest. These microorganisms are selected so as to be capable of successfully competing in the particular environment (crop and other insect habitats) with the wild-type microorganisms, provide for stable maintenance and expression of the gene expressing the polypeptide pesticide, and, desirably, provide for improved protection of the pesticide from environmental degradation and inactivation.<br>
00205]	A large number of microorganisms are known to inhabit the phylloplane (the surface of<br>
the plant leaves) and/or the rhizosphere (the soil surrounding plant roots) of a wide variety of important crops. These microorganisms include bacteria, algae, and fungi. Of particular interest are microorganisms, such as bacteria, e.g., genetaPseudomonas, Envinia, Serratia, Klebsiella, Xanthomonas, Strsptomvces, Rhizobiwn, Rhodopseudomonas, Methylophilius, Agrobacteriwn, Acetobacter, Lactobacillus, Arthrobacter, Azotobacter, Leitconostoc, and Alcaligenes\ fungi, particularly yeast, e.g., genera Saccharomyces, Gyptococcus, Kluy\&gt;eromyces', Sporobolomyces, Rhodotontla, a&amp;dAureobasidiwn. Of particular interest are such phytosphere bacterial species as Pseudomouas syringae, Pseudomonasfliiorescens, Serratia marcescens, Acetobacter xylinum, Agrobacterium tumefaciens, Rhodopseudomonas spheroides, Xanthomonas campestris, Rhizobiwn melioti, Alcaligenes entrophus, and Azotobacter vinlandii', and phytosphere yeast species such as Rhodotorula mbra, R. glutinis, R. marina, R. aurantiaca, Cryptococcus albidus, C. diffluens, C. laurentii, Saccharomyces rosei, S. pretoriensis, S. cerevisiae, Sporobolomyces<br><br>
roseus, S. odorus, Khtyveromyces veronae, and Aureobasidium pollulans. Also of interest are pigmented microorganisms.<br>
[00206]	Insertion of genes to form transgenic hosts.  One aspect of the subject invention is the<br>
transfonnation/transfection of plants, plant cells, and other host cells with polynucleotides of the subject invention that express proteins of the subject invention. Plants transformed in this manner can be rendered resistant to attack by the target pest(s).<br>
['00207]	A wide variety of methods are available for introducing a gene encoding a pesticidaJ<br>
protein into the target host under conditions that allow for stable maintenance and expression of the gene. These methods are well known to those skilled in the art and are described, for example, in United States Patent No. 5,135,867.<br>
(00208]	For example, a large number of cloning vectors comprising a replication system in E. coli<br>
. and a marker that permits selection of the transformed cells are available for preparation for the insertion of foreign genes into higher plants. The vectors comprise, for example, pBR322, pUC series, M13mp series, pACYC184, etc. Accordingly, the sequence encoding the toxin can be inserted into the vector at a suitable restriction site. The resulting plasmid is used for transfonnation into E. coli. The E. coli cells are cultivated in a suitable nutrient medium, then harvested and lysed. The plasmid is recovered. Sequence analysis, restriction analysis, electrophoresis, and other biochemical-molecular biological methods are generally carried out as methods of analysis. After each manipulation, the DNA sequence used can be cleaved and joined to the next DNA sequence. Each plasmid sequence can be cloned in the same or other plasmids. Depending on the method of inserting desired genes into the plant, other DNA sequences maybe necessary. If, for example, the Ti or Ri plasmid is used for the transformation of the plant cell, then at least the right border, but often the right and the left border of the Ti or Ri plasmid T-DNA, has to be joined as the flanking region of the genes' to be'inserted. The use of T-DNA'for the transformation of plant cells has been intensively researched and described in EP 120 516; Hoekema (1985) In: The Binaiy Plant Vector System, Offset-durkkerij Kanters B.V., Alblasserdam, Chapter.5; Fraley et al, Crit. Rev. Plant Sci. 4:1 -46; and An et al. (1985) EMBO J. 4:277-287.<br>
(00209]	A large number of techniques are available for inserting DNA into a plant host cell.<br>
Those techniques include transformation with T-DNA using Agrobacierium tumefaciens or Agrobacteriitm rhizogenes as transformation agent, .fusion, injection, biolistics (microparticle<br><br>
bombardmen t), or electroporation as well as other possible methods. If Agrobacteria are used for the transformation, the DNA to be inserted has to be cloned into special plasmids, namely either into an intermediate vector or into a binary vector. The intermediate vectors can be integrated into the Ti or Ri plasmid by homologous recombination owing to sequences that are homologous to sequences in the T-DNA. The Ti or Ri plasmirl also comprises the vir region necessary for the transfer of the T-DNA. Intermediate vectors cannot replicate themselves in Agrobacteria. The intermediate vector can be transferred into Agrobacterium tumefaciens by means of a helper plasmid (conjugation). Binary vectors can replicate themselves both in E. coli and in Agrobacteria. They comprise a selection marker gene and a linker or polylinker which are framed by the right and left T-DNA border regions. They can be transformed directly into Agrobacteria (Holstersetal. [1978]Mo/. Gen. Genet. 163:181-187). TheAgrobacteriwnuscdus host cell is to comprise a plasmid carrying a vir region. The vir region is necessary for the transfer of the T-DNA into the plant cell. Additional T-DNA may be contained. The bacterium so transformed is used for the transformation of plant cells. Plant explants can advantageously be cultivated with Agrobacterium tumefaciens orAgrobaclerimn rhiiogenes for the transfer of the DNA into the plam cell. Whole plants can then be regenerated from the infected plant material (for example, pieces of leaf, segments of stalk, roots, but also protoplasts or suspension-cultivated cells) in a suitable medium, which may contain antibiotics or biocid.es for selection. The plants so obtained can then be tested for the presence of the inserted DNA. No special demands are made of the plasmids in the case of injection and electroporation. It is possible to use ordinary plasmids, such as, for example, pUC derivatives.<br>
The transformed cells grow inside the plants in the usual manner. They can form germ cells and transmit the transformed trait(s) to progeny plants. Such plants can be grown in the nonnal manner and crossed with plants that have the same transformed hereditary factors or other hereditary factors. The resulting hybrid individuals have the corresponding phenotypic properties.<br>
(00211]	In some preferred embodiments of the invention, genes encoding the bacterial toxin are<br>
expressed from transcriptional units inserted into the plant genome. Preferably, said transcriptional units are recombinant vectors capable of stable integration into the plant genome and enable selection of transformed plant lines expressing niRNA encoding the proteins.<br>
[00212]	' Once the inserted DNA has been integrated in the genome, it is relatively stable there (and<br><br>
does not come out again). It normally contains a selection marker that confers on the transformed plant cells resistance to a biocide or an antibiotic, such as kanamycin, G418, bleomycin, hygromycin, or chloramphenicol, inter alia. The individually employed marker should accordingly permit the selection of transformed cells rather than cells that do not contain the inserted DNA. The gene(s) of interest are preferably expressed either by constitutive or inducible promoters in the plant cell. Once expressed, the mRNA is translated into proteins, thereby incorporating amino acids of interest into protein. The genes encoding a toxin expressed in the plant cells can be under the control of a constitutive promoter, a tissuejspecific promoter, or an inducible promoter.<br>
[00213]	Several techniques exist for introducing foreign recombinant vectors into plant cells, and<br>
for obtaining plants that stably maintain and express the introduced gene. Such techniques include the introduction of genetic material coated qnto microparticles directly into cells (U.S. Pat. Nos. 4,945,050 to Cornell and 5,141,131 to DowElanco, now Dow AgroSciences, LLC). In addition, plants may be transformed using Agrobacterium technology, see U.S. Pat. No. 5,177,010 to University of Toledo; 5,104,310 to Texas A&amp;M; European Patent Application 0131624B1; European Patent Applications 120516, 159418B1 and 176,112 to Schilperoot; U.S. Pat. Nos. 5,149,645,5,469,976,5,464,763 and 4,940,838 and 4,693,976 to Sclulperoot; European Patent Applications 116718,290799, 320500- all to Max Planck; European Patent Applications 604662 and 627752, and U.S. Pat. No. 5,591,616, to Japan Tobacco; European Patent Applications 0267159 and 0292435, and U.S. Pat. No. 5,231,019, all to Ciba Geigy, now Novartis; U.S. Pat. Nos. 5,463,174 and 4,762,785, both to.Calgene; and U.S. Pat. Nos. 5:004,863 and 5,159,135, both to Agracetus. Other, transformation technology includes whiskers technology. See U.S. Pat. Nos. 5,302,523 and 5,464,765, both to Zeneca. Electroporation technology has also been used to transform plants. See WO 87/06614 to Boyce Thompson Institute; U.S. Pat. Nos. 5,472,869 and 5,384,253, both to Dekalb; and WO 92/09696 and WO 93/21335, both to Plant Genetic Systems. Furthermore, viral vectors can also be used to produce transgenic plants expressing the protein of interest. For example, monocotyledonous plant can be transformed with a viral vector using the methods described in U.S. Pat. Nos. 5,569,597 to Mycogen Plant Science and Ciba-Giegy, now Novartis, as well as U.S. Pat. Nos. 5,589,367 and 5,316,931, both to Biosource.<br><br>
(00214]	As mentioned previously, the manner in which the DNA construct is introduced into the<br>
plant host is not critical to this invention. Any method which provides for efficient trans formation may be employed. For example, various methods for plant cell transformation are described herein and include the use of Ti or Ri-plasmids and the like to perform Agrobacterium mediated transformation. In many instances, it will be desirable to have the construct used for transformation bordered on one or both sides by T-DNA borders, more specifically the right border. This is particularly useful when the construct uses Agrobacterium twnefaciens or Agrobacteriwn rhizogenes as a mode for transformation, although T-DNA borders may find use with other modes of transformation. Where Agi-obacterium is used for plant cell transformation, a vector may be used which may be introduced into the host for homologous recombination with T-DNA or the Ti or Ri plasmid present in the host. Introduction of the vector may be performed via electroporation, tri-parental mating and other techniques for transforming gram-negative bacteria which are known to those skilled in the art. The manner of vector transformation into the Agrobacterium host is not critical to this invention. The Ti or Ri plasmid containing the T-DNA for recombination may be capable or incapable of causing gall formation, and is not critical to said invention so long as the vir genes are present in said host.<br>
(00215]	In some cases where Agrobacterium is used for transformation, the expression construct<br>
being within the T-DNA borders will be inserted into a broad spectrum vector such as pRK2 or<br>
derivatives thereof as described in Ditta etal, (PNAS USA (19SO) 77:7347-7351 and EPO 0 120<br>
515, which are incorporated herein by reference. Included within the expression construct and the<br>
T-DNA will be one or more markers as described herein which allow for selection of transformed<br>
Agrobacterium and transformed plant cells. The particular marker employed is not essential to<br>
this invention, with the preferred marker depending on the host and construction used.<br>
(00216]	For transformation of plant cells using Agrobacterium, explants ma}' be combined and<br>
incubated with the transformed Agrobacterium for sufficient time to allow transformation thereof. After transformation, the Agrobacteria are killed by selection with the appropriate antibiotic and plant cells are cultured with the appropriate selective medium. Once calli are formed, shoot formation can be encouraged by employing the appropriate plant hormones according to methods well known in the art of plant tissue culturing and plant regeneration. However, a callus intermediate stage is not always necessary. After shoot formation, said plant<br>
«<br>
cells can be transferred to medium which encourages root formation thereby completing plant<br><br>
regeneration. The plants may then be grown to seed and said seed can be used to establish future generations. Regardless of transformation technique, the gene encoding a bacterial toxin is preferably incorporated into a gene transfer vector adapted to express said gene in a plant cell by including in the vector a plant promoter regulatory element, as well as 3' non-translated transcriptional termination regions such as Nos and the like.<br>
[0217]	In addition to numerous technologies for transforming plants, the type of tissue which is<br>
contacted with the foreign genes may vary as well. Such tissue would include but would not be limited to embryogenic tissue, callus tissue types I, II, and HI, hypocotyl,'meristem, root tissue, tissues for expression in phloem, and the like. Almost all plant tissues maybe transformed during dedifferentiation using appropriate techniques described herein.<br>
[0218]	As mentioned above, a variety of selectable markers can be used, if desired. Preference<br>
for a particular marker is at the discretion of the artisan, but any of the following selectable markers may be used along with any other gene not listed herein which could function as a selectable marker. Such selectable markers include but are not limited to aminoglycoside phosphotransferase gene of transposon Tn5 (Aph II) which encodes resistance to the antibiotics kanamycin, neomycin and G418, as well as those genes which encode for resistance or tolerance to glyphosate; hygromycin; methotrexaie; phosphinothricin (bialaphos); imidazolinones, sulfonyiureas and triazolopyrimidine herbicides, such as chlorsulfuron; bromoxynil, dalapon and the like.<br>
[00219]	In addition to a selectable marker, it may be desirous to use a reporter gene. In some<br>
instances a reporter gene may be used with or without a selectable marker. Reporter genes are genes which are typically not present in the recipient organism or tissue and typically encode for proteins resulting in some phenotypic change or enzymatic property. Examples of such genes are provided in K. Wising et al. Ann. Rev. Genetics, 22, 421 (1988). Preferred reporter genes include the beta-glucuronidase (GUS) of the uidA locus of E. coli, the chloramphenicol acetyl transferase gene from Tn9 of E. coli, the green fluorescent protein from the bioluminescent jellyfish Aequorea victoria, and the luciferase genes from firefly Photinuspyralis. An assay for detecting reporter gene expression may then be performed at a suitable time after said gene has been introduced into recipient cells. A preferred such-assay entails the use of the gene encoding beta-glucuronidase (GUS) of the uidA locus of E. coli as described by Jefferson et al., (1987 Biochem. Soc. Trans. 15, 17-19) to identify transformed cells.<br><br>
[00220]	In addition to plant promoter regulatory elements, promoter regulatory elements from a<br>
variety of sources can be used efficiently in plant cells to express foreign genes. For example, promoter regulatory elements of bacterial origin, such as the octopine synthase promoter, the nopaline synthase promoter, the mannopine synthase promoter; promoters of viral origin, such as the cauliflower mosaic virus (35S and 19S), 35T (which is a re-engineered 35S promoter, see U.S. Pat. No. 6,166,302, especially Example 7E) and the like may be used. Plant promoter regulatory elements include but are not limited to ribulose-1,6-bisphosphate (RUBP) carboxylase small subunit (ssu), beta-conglycinin promoter, beta-phaseolin promoter, ADH promoter, heat-shock promoters, and tissue specific promoters. Other elements such as matrix attachment regions, scaffold attachment regions, introns, enhancers, polyadenylation sequences and the like may be present and thus may improve the transcription efficiency or DNA integration. Such elements may or may not be necessary for DNA function, although they can provide better expression or functioning of the DNA-by affecting transcription, mRNA stability, and the like. Such elements may be included in the DMA as desired to obtain optimal performance of the transformed DNA in the plant. Typical elements include but are not limited to Adh-intron 1, Adh-intron 6, the alfalfa mosaic virus coat protein leader sequence, the maize streak virus coat protein leader sequence, as well as others available to a skilled artisan. Constitutive promoter regulatory elements may also be used thereby directing continuous gene expression in all cells types and at all times (e.g., actin, ubiquitin, CaMV 35S, and the like). Tissue specific promoter regulatory elements are responsible for gene expression in specific cell or tissue types, such as the leaves or seeds (e.g., zein, oleosin, napin, ACP, globulin and the like) and these may also be used.<br>
[0022[	Promoter regulatory elements may also be active during a certain stage of the plant's<br>
development as well as active in plant tissues and organs. Examples of such include but are not limited to pollen-specific, embryo-specific, com-silk-specific, cotton-fiber-specific, root-specific, seed-endosperm-specific promoter regulatory elements and the like. Under certain circumstances it may be desirable to use an inducible promoter regulatory element, which is responsible for expression of genes in response to a specific signal, such as: physical stimulus (heat shock genes), light (RUBP carboxylase), hormone (Em), metabolites, chemical, and stress. Other desirable transcription and translation elements that function in plants may be used. Numerous plant-specific gene transfer vectors are known in the art.<br><br>
[0222]	Standard molecular biology techniques may be used to clone and sequence the toxins<br>
described herein. Additional information may be found in Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989), Molecular Cloning, A Laboratory Manual, Cold Spring Harbor Press, which is incorporated herein by reference.<br>
[0223]	Resistance Management. With increasing commercial use of insecticidal proteins in<br>
transgenic plants, one consideration is resistance management. That is, there are numerous companies using Bacillus thuringiensis toxins in their products, and there is concern about insects developing resistance to B.t. toxins. One strategy for insect resistance management would be to combine the TC toxins produced byXenorhabdus, Photorhabdus, and the like with toxins such as B.t. crystal toxins, soluble insecticidal proteins from Bacillus stains (see, e.g., WO 98/18932 and WO 99/57282), or other insect toxins. The combinations could be formulated for a sprayable application or could be molecular combinations. Plants could be transformed with bacterial genes that produce two or more different insect toxins (see, e.g., Gould, 38 Bioscience 26-33 (1988) and U.S. Patent No. 5,500,365; likewise, European Patent Application 0 400 246 Al and U.S. Patents 5,866,784; 5,908,970; and 6,172,281 also describe transformation of a plant with two B. t. crystal toxins). Another method of producing a transgenic plant that contains more than one insect resistant gene would be to first produce two plants, with each plant containing an insect resistance gene. These plants could then be crossed using traditional plant breeding techniques to produce a plant containing more than one insect resistance gene. Thus, it should be apparent that the phrase "comprising a polynucleotide" as used herein means at least one polynucleotide '(and possibly more, contiguous or not) unless specifically indicated otherwise.<br>
[oo224]	Formulations and Other Delivery Systems. Formulated bait granules containing cells<br>
and/or proteins of the subject invention (including recombinant microbes comprising the genes described herein) can be applied to the soil. Formulated product can also be applied as a seed-coating or root treatment or total plant treatment at later stages of the crop cycle. Plant and soil treatments of cells may be employed as wettable powders, granules or dusts, by mixing with various inert materials, such as inorganic minerals (phyllosilicates, carbonates, sulfates, phosphates, and the like) or botanical materials (powdered corncobs, rice hulls, walnut shells, and the like). The formulations may include spreader-sticker adjuvants, stabilizing agents, other pesticidal additives, or surfactants. Liquid formulations maybe aqueous-based or non-aqueous<br>
and employed as foams, gels, suspensions, emulsifiable concentrates, or the like. The ingredients may include rheological agents, surfactants, emulsifiers, dispersants, or polymers.<br>
[00225}	As would be appreciated by a person skilled in the art, the pesticidal concentration will<br>
vary widely depending upon the nature of the particular formulation, particularly whether it is a concentrate or to be used directly. The pesticide will be present in at least 1 % by weight and may be 100% by weight. The dry formulations will have from about 1 -95% by weight of the pesticide while the liquid formulations will generally be from about 1-60% by weight of the solids in the liquid phase. The formulations will generally have from about 102 to ab'out 104 cells/mg. These formulations will be administered at about 50 mg (liquid or dry) to 1 kg or more per hectare.<br>
[00226!	The formulations can be applied to the environment of the pest, e.g., soil and foliage, by<br>
spraying, dusting, sprinkling, or the like.<br>
(00227]	Another delivery scheme is the incorporation of the genetic material of toxins into a<br>
baculovims vector. Baculoviruses infect particular insect hosts, including those desirably targeted with the toxins. Infectious baculovirus harboring an expression construct for the toxins could be introduced into areas of insect infestation to thereby intoxicate or poison infected insects.<br>
'[00228]	Insect viruses, or baculoviruses, are known to infect and adversely affect certain insects.<br>
The effect of the viruses on insects is slow, and viruses do not immediately stop the feeding of insects. Thus, viruses are not viewed as being optimal as insect pest control agents. However, combining the toxin genes into a baculovirus vector could provide an efficient way of transmitting the toxins, hi addition, since different baculoviruses are specific to different insects, it may be possible to use a particular toxin to selectively target particularly damaging insect pests. A particularly useful vector for the toxins genes is the nuclear polyhedrosis virus. Transfer vectors using this virus have been described and are now the vectors of choice for transferring foreign genes into insects. The virus-toxin gene recombinant may be constructed in an orally transmissible form. Baculoviruses normally infect insect victims through the mid-gut intestinal mucosa. The toxin gene inserted behind a strong viral coat protein promoter would be expressed and should rapidly kill the infected insect.<br>
[0229]	In addition to an insect virus or baculovirus or transgenic plant delivery system for the<br>
protein toxins of the present invention, the proteins may be encapsulated using Bacillus tlniringiensis encapsulation technology such as but not limited to U.S. Pat. Nos. 4,695,455; 4,695,462; 4,861,595 which are all incorporated herein by reference. Another delivery system for<br><br>
the protein toxins of the present invention is formulation of the protein into a bait matrix, which could then be used in above and below ground insect bait stations. Examples of such technology include but are not limited to PCT Patent Application WO 93/23998, which is incorporated herein by reference.<br>
[00230]	Plant RNA viral based systems can also be used to express bacterial toxin. In so doing,<br>
the gene encoding a toxin can be inserted into the coat promoter region of a suitable plant virus which will infect the host plant of interest. The toxin can then be expressed thus providing protection of the plant from insect damage. Plant RNA viral based systems are described in U.S. Pat. Nos. 5,500,360 to Mycogen Plant Sciences, Inc. and US. Pat. Nos. 5,316,931 and 5,589,367 to Biosource Genetics Corp.<br>
[00231]	In addition to producing a transformed plant, there are other delivery systems where it<br>
may be desirable to engineer the bacterial gene(s). For example, a protein toxin can be constructed by fusing together a molecule attractive to insects as a food source with a toxin. After purification in the laboratory such a toxic agent with "built-in" bait could be packaged inside standard insect trap housings.<br>
[00232]	Mutants. Mutants of bacterial isolates can be made by procedures that are well known in<br>
the art. For example, asporogenous mutants can be obtained through ethylmethane sulfonate (EMS) rnutagenesis of an isolate. The mutants can be made using ultraviolet light and nitrosoguanidine by procedures well known in the art.<br>
[o233j	All patents, patent applications, provisional applications, and publications referred to or<br>
cited herein are incorporated by reference in their entirety to the extent they are not inconsistent with the explicit teachings of this specification.<br>
[oo234]	Following are examples that illustrate procedures for practicing the invention. These<br>
examples should not be construed as limiting. All percentages are by weight and all solvent mixture proportions are by volume unless otherwise noted.<br>
Example 1 - TC Proteins and Genes Obtainable from Xenorhabdus strain Xwi<br>
[00235]	It was shown previously (U.S. Patent No. 6,048,838) thai Xenorhabdus nematophilus<br>
strain Xwi (NRRL B-21733, deposited on April 29, 1997) produced extracellular proteins with<br><br>
oral insecticidal activity against members of the insect orders Coleoptera, Lepidoptera, Diptera,<br>
and Acarina. Full-length gene and TC protein sequence from strain Xwi are disclosed below.<br>
The methods used to obtain them are more fully discussed in concurrently filed U.S. provisional<br>
application by Bintrim et al. (Serial No. 60/ 441,717), entitled "Xenorhabdus TC Proteins and<br>
Genes for Pest Control." These sequences, including N-tenninal and internal peptide sequences<br>
(SEQ ED NOs:l-5) are also summarized above in the Brief Description of the Sequences section.<br>
00236J	In summary, a 39,005 bp fragment of genomic DNA was obtained from strain Xwi and<br>
was cloned as cosmid pDAB2097. The sequence of the cosmid insert (SEQ ID NO. 6) was analyzed using the Vector NTI™ Suite (Informax, hie. North Bethesda, MD, USA) to identify encoded ORFs (Open Reading Frames). Six full length ORFs and one partial ORF were identified (Figure 1 and Table 4).<br>
(Table Removed)<br>
* (C) designates complementary.strand of SEQ ID NO: 6<br>
[0237]	The nucleotide sequences of the identified ORFs and the deduced amino acid sequences<br>
encoded by these ORFs were used to search the databases at the National Center for Biotechnology Information by using BLASTn, BLASTp, and BLASTx, via the government (".gov") website of ncbi/nih for BLAST. These analyses showed that the ORFs identified in the pDAB2097 insert had significant amino acid sequence identity to genes previously identified in Photorhabdus luminescens and Xenorhabdus nematophilus (Table 5). It is noteworthy that the xpt gene sequences presented in GenBank accession number AJ308438 were obtained from a recombinant cosmid that expressed oral insecticidal activity.<br><br>
Table 5. Similarity of Deduced Proteins encoded by pDAB2Q97 ORPs to Known Genes<br>
(Table Removed)<br>
*Deduced Amino Acid Positions with Identity to Database Sequence<br>
W3SJ	Since ORF2, ORF4, ORF5, ORF6, and ORF7 were shown to have at least 95% amino<br>
acid sequence identity to previously identified genes, the same gene nomenclature was adopted for further studies on the ORFs identified in the pDAB2097 insert sequence (Table 6).<br>
W39J	As used throughout this application, XptA2, for example, signifies a protein and xptA2,<br>
for example, signifies a gene. Furthermore, the source isolate for the gene and protein is indicated with subscript. An illustration of this appears in Table 6.<br>
Table 6. Nomenclature of ORPs identified in pDAB2097 insert sequence<br>
(Table Removed)<br>
Example 2 - Heterologous Expression of Toxin Complex Genes from Photorhabdus and Xenorhabdus<br>
'00240]	A series of experiments was done in which Photorhabdus and Xenorhabdus genes were<br>
expressed in E. coli. It is shown that co-expression of either the tcdA orxptA2 genes with specific combinations of the tcdBl, tccCl, xptBl and xptCl genes, results in significant activity in bioassay against sensitive insects. It is also demonstrated here that expression of the Photorhabdus genes tcdA and tcdB with the Xenorhabdus gene xptBl results in significant activity against Southern corn rootworm (Diabrotica undecimpunctata howardii). Likewise,<br><br>
expression of XenorhabdnsxptA2 with Photorhabdus tcdBl and tccCl produces activity against corn earworm (Helicoverpa zed).<br>
['00241]	Two E.   coli expression systems were employed  for testing Photorhabdus and<br>
Xenorhabdus genes. The first relied on an E. coli promoter present in the expression vector pBT-TcdA (Figure 2). Several plasmids were constructed in which polycistronic arrangements of up to three genes were constructed. Each gene contained a separate ribosome binding site and start codon, a coding sequence and a stop codon. The second system was mediated by the strong T7 phage promoter and T7 RNA polymerase (Figure 3, pET; Figure 4, pCot). Similarly, in some constructions polycistronic arrangements of coding sequences were used. In other experiments, compatible plasmids were used for co-expression. Schematic diagrams describing all of the constructions used in the experiments are shown in Figures 5 and 6.<br>
['00242}	Construction of pBT-TcdA.  The expression plasmid pBT-TcdA is composed of the<br>
replication and antibiotic selection components of plasmid pBC KS+ (Stratagene) and the expression components (i.e. a strong E. coli promoter, lac operon represser and operator, upstream of a multiple cloning site) from plasmid pTrc99a (Amersham Biosciences Corp., Piscataway, N.J.). AnNco I site was removed from the chloramphenicol resistance gene of pBC KS+ using in vitro mutagenesis. The modification did not change the amino acid sequence of the chloramphenicol acetyl transferase protein. As previously described (Example 27 of WO 98/08932, Insecticidal Protein Toxins from Photorhabdus), the TcdA coding sequence (GenBank Accession No. AF188483; reproduced here as SEQ ID NO:21) was modified using PCR to engineer both the 5' and 3' ends. This modified coding sequence was subsequently cloned into pTrc99a. Plasmid pBT-TcdA was made by joining the blunted Sph I/Pvu I fragment of pTrc-TcdA with the blunted Asn I/Pvu I fragment of the pBC KS+. The result is plasmid pBT-TcdA (Figures 2 and 5).<br>
[00243}	Construction of pBT-TcdA-TcdB. The TcdBl coding sequence (GenBank Accession No.<br>
AF346500; reproduced here as SEQ ID N0:22) was amplified from plasmid pBC-AS4 (R. ffrench-Constant University of Wisconsin) using the forward primer:<br>
5' ATATAGTCGACGAATTTTAATCTACTAGTAAAAAGGAGATAACCATGCAGAATTC ACAAACATTCAGTGTTACC3'. (SEQIDNO:23)<br>
[00244)	This primer does not change the protein coding sequence and adds Sal I and Spe I sites in<br>
the 5' non coding region. The reverse primer used was:<br><br>
5'ATAATACGATCGTTTCTCGAGTCATTACACCAGCGCATCAGCGGCCGTATCATTCT C3'. (SEQIDNO:24)<br>
[00245]	Again, no changes were made to the protein coding sequence but an Xho I site was added<br>
to the 3' non coding region. The amplified product was cloned into pCR2.1 (Invitrogen) and the DNA sequence was determined. Two changes from the predicted sequence were noted, a single A deletion in the Spe I site of the forward primer (eliminating the site) and an A-to-T substitution at corresponding ainino acid position 1041 that resulted in the conservative substitution of Asp-to-Glu. Neither change was corrected. Plasmid pBT-TcdA was digested with XIio I and Pvu I (cutting at the 3' end of the TcdA coding sequence). Plasmid pCR2.1-TcdBl was cut with Sal I and Pvu I. The fragments were ligated and pBT-TcdA-TcdB 1 recombinants (Figure 5) were isolated. The Xho I and Sal I ends are compatible, but both sites are eliminated upon ligation. The plasmid encodes a polycistronic TcdA-TcdBl RNA. Each coding region carries separate stop and start codons, and each is preceded by separate ribosome binding sites.<br>
roo246]	Construction of pDAB3Q59.   The coding sequence for the TccCl protein (GenBank<br>
Accession No. AAC38630.1; reproduced here as SEQ ID NO:25) was amplified from a pBC KS+ vector (pTccC chl; from R. ffrench-Constant, the University of Wisconsin) containing the three-gene Tec operon. The forward primer was:<br>
5' GTCGACGCACTACTAGTAAAAAGGAGATAACCCCATGAGCCCGTCTGAGACTACT    '. CTTTATACTCAAACCCCAACAG 3' (SEQ ID NO:26)<br>
roo247]	This primer did not change the coding sequence of the tccCl gene, but provided 5' non<br>
coding Sal I and Spe I sites as well as a ribosome binding site and ATG initiation codon. The reverse primer was: 5'<br>
CGGCCGCAGTCCTCGAGTCAGATTAATTACAAAGAAAAAACTCGTCGTGCGGCTCCC' 3'(SEQIDNO:27)<br>
roo248]	This primer also did not alter the tccCl coding sequence, but provided 3' Not I and XIio I<br>
cloning sites. Following amplification with components of an EPICENTRE FailSafe PCR kit (EPICENTRE; Madison, WI) the engineered TccCl coding sequence was cloned into pCR2.1-TOPO (Invitrogen). The coding sequence was cut from pCR2.1 and transferred to a modified pET vector (Novagen; Madison WI) via the 5' Sal I and 3' Not I sites. The pET vector contains a gene conferring resistance to spectinomycin/streptomycin, and has a modified multiple cloning<br>
site. A PCR-induced mutation found via DNA sequencing was corrected using the pTccC chl plasmid DNA as template, and the plasmid containing the corrected coding region was named pDAB3 059. Double-stranded DNA sequencing confirmed that the mutation had been corrected.<br>
(00249]	Construction of pBT-TcdA-TccCl. Plasmid pBT-TcdA DNA was cut with Xlio I, and<br>
ligated to pDAB3059 DNA cut with Sal I and Xlw I. The tccCl gene was subsequently ligated downstream of the tcdA gene to create pBT-TcdA-TccCl (Figure 5).<br>
(00250]	Construction of pBT-TcdA-TcdB 1 -TccC 1. Plasmid pBT-TcdA-TcdBl DNA was cut<br>
with Klio I and ligated to pDAB3059 DNA cut with Sal I and Xlio I. Recombinants were screened for insertion of the tccCl gene behind the tcdB gene to create plasmid pBT-TcdA-TcdBL-TccCl (Figure 5).<br>
'[00251)	Construction of pBT-TcdA-TcdB 1 -XptB 1. Plasmid pBT-Tcd A-TcdB 1 DNA was cut<br>
with Xlio I and shotgun ligated with pET280-XptBl DNA which was cut with Sal I and Xlio I. Recombinants representing insertion of the xptBl coding region into the Xlio I site of pBT-TcdA-TcdBl where identified to create plasmid pBT-TcdA-TcdBl-XptBl (Figure 5).<br>
[00252]	Construction of pET28-TcdA. The description of this plasmid can be found elsewhere as<br>
Example 27 of WO 98/08932, Insecticidal Protein Toxins from Photorhabdus.<br>
[00253]	Construction of pCot-TcdB 1. Plasmid pCR2. i-TcdB 1 was cut with Xlw I and Sal I and<br>
ligated into the Sal I site of the T7 expression plasmid pCot-3 (Figure 4). Plasmid pCot-3 has a pACYC origin of replication, making it compatible with plasmids bearing a ColEl origin (pBR322 derivatives). In addition, it carries a chloramphenicol antibiotic resistance marker gene and a T7 RNA polymerase-specific promoter for expression of coding regions inserted into the multiple cloning site.<br>
[00254]	Construction of pCot-TccCl -TcdB.l. The TccC 1 coding region was cut from pDAB3059<br>
DNA with Spe I arid Not I and ligated into the multiple cloning site of plasmid pET280-K (a modified pET28 which has had the multiple cloning site replaced). This resulted in acquisition of a Swa I site upstream of the Spe I site and an Xho I site downstream of the Not I site. DNA of plasmid pET280-K-TccC 1 was cut with Swa I and Xho I to release the TccC 1 coding sequence, wliich was then ligated into the Swa ! and Sal I sites of plasmid pCot-3-TcdB 1 to create plasmid pCot-3-TccCl-TcdB (Figure 6).<br>
Construction of pET2SO-XptA2. pET280-XptCL and pET280-Xptfil.   The coding sequences for the XptA2, XptCl, and XptBl proteins were each PCR amplified from<br><br>
pDAB2097, a recombinant cosmid containing the three genes that encode these proteins. The PCR primer sets used to amplify these coding sequences are listed in Table 7. hi all of these primer sets, the forward primer did not change the coding sequence of the gene but provided 5' non coding Sal I and Xba I sites as well as a ribosome binding site. The reverse primers also did not alter the corresponding coding sequences, but provided a 3' Xlio I cloning site. Following amplification with components of the EPICENTRE Fail Safe PCR kit, the engineered XptA2, XptCl, and XptBl coding sequences were each cloned into pCR2.1. The cloned amplified products were sequence confirmed to ensure that PCR-induced mutations did not alter the coding sequences. Recombinant plasmids that contained unaltered coding sequences for XptA2, XptC 1, and XptBl were identified and designated as pDAB3056, pDAB3064, and pDAB3055, respectively. The coding sequences were each cut from the pCRZ.l derivatives and transferred to a modified pET vector (pET2SO-SS is a pET2S derivative which has had the multiple cloning site replaced, and a streptomycin/spectinomycin resistance gene inserted into the backbone to provide a selectable marker [Figure 3]), via the 5' Xba 1 and 3' XJio 1 sites to create plasmids pET2SO-XptA2, pET280-XptCl, and pET280-XptBl.<br><br>
(Table Removed)<br>
[0056]	Construction of pET280-XptA2-XptCl. Plasmid pET280-XptA2 DNA was cut with Xho<br>
I and ligated into the unique Sal I site in pDAB3064. The resulting ligated product contained both pCR2.1 and pET280-SS vector backbones and could be recovered by antibiotic selection using a combination of streptomycin (25 µg/mL), spectinomycin (25 ug/mL), and ampicillin (100 µg/mL). DNA of the recovered plasmids was digested withXho I to check fragment orientation. A plasmid with the XptCl coding region immediately downstream of the XptA2 coding region was obtained and the DNA was digested with A7zo 1 to remove the pCR2.1 vector backbone. The resulting construct, which contains the pET280-SS vector backbone and the coding sequences for XptA2 and XptCl, was self-ligaterl to produce pET280-XptA2-XptCl.<br>
[00257]Construction of pET280-XptCl-XptBl. Plasmid pET280-XptCl DNA was cut with Xh7io<br>
I and ligated into the unique Sal I site in pDAB305 5. The resulting ligated product contained both pCR2.1 and pET280-SS vector backbones and could be recovered by antibiotic selection using a combination of streptomycin (25 ug/mL), spectinomycin (25 ug/mL), and ampicillin (100 ug/mL). DNA of the recovered plasmids was digested with Xlio I to check fragment orientation. A plasmid with the XptBl coding region immediately downstream of the XptCl coding region was obtained and the DNA was digested withXlw I to remove the pCR2.1 vector backbone. The resulting construct, which contains thepET2SO-SS vector backbone and the coding sequences for XptCl and XptBl, was self-ligated to produce pET280-XptCl-XptBl.<br>
W25SJ	Construction of pET280-XptA2-XptBl. Plasmid pET280-XptA2 DNA was cut withXho<br>
I and ligated into the unique Sal I site in pDAB305 5. The resulting ligated product contained both pCR2.1 and pET280-SS vector backbones and could be recovered by antibiotic selection using a combination of streptomycin (25 µg/mL), spectinomycin (25 ug/mL), and ampicillin (100 µg/mL). DNA of the recovered plasmids was digested wiih Xho I to check fragment orientation. A plasmid with the XptBl coding region immediately downstream of the XptA2 coding region was obtained and the DNA was digested with Xlw I to remove the pCR2.1 vector backbone. The resulting construct, which contains thepET2SO-SS vector backbone and the coding sequences for XptA2 and XptBl, was self-ligated to produce pET280-XptA2-XptBl.<br>
F00259]	Construction of pET280-XptA2-XptC 1 -XptB 1. Plasmid pET280-XptA2-XptCl DNA<br>
was cut with Xlio I and ligated into the unique Sal I site in pDAB3Q55. The resulting ligated product contained both pCR2.1 and pET280-SS vector backbones and could be recovered by antibiotic selection using a combination of streptomycin (25 µg/mL), spectinomycin (25 µg/mL),<br><br>
and ampicillin (100 µg/mL). The recovered plasmids were digested withXho I to check fragment orientation. A plasmid with the XptBl coding region immediately downstream of the XptCl coding region was obtained and the DNA was digested withXho I to remove the pCR2.1 vector backbone. The resulting construct, which contains the pET280-SS vector backbone and the XptA2,XptCl, and XptBl coding sequences, was self-ligated to producepET280-XptA2-XptCl-XptBl.<br>
'[00260]	Expression of pBT-based constructions. The pBT expression plasmids were transformed<br>
into E. coli strain BL21 cells and plated on LB agar containing 50 jag/mL chloramphenicol and 50 mM glucose, and transforrnants were grown at 37°C overnight. Approximately 10-100 well isolated' colonies were used to inoculate 200 mL of sterile LB containing 50 ug/mL chloramphenicol plus 75 ^M isopropyl-p-D-thiogalatopyranoside (IPTG ) in 500 mL baffled flasks. The cultures were shaken at 200 rpm at 28°C for 24 hours. Cells were collected by centrifugation (approximately 3000 x g) and resuspended in phosphate buffer (30 mM, pH 7.4; NutraMax; Gloucester. MA) to a cell density of 30-120 ODgoo units/mL. Diluted cells were then used for insect bioassay.<br>
[0061]	Alternatively, the cells were chilled on ice after growth for 24 hours and adjusted to 20-30<br>
OD6oo units/ml with phosphate buffer. The cells were lysed with a probe sonicator (Soniprep 150, MSE), using 2 x 45 second bursts at 20 microns amplitude with 1/3 volume 0.1 mm glass beads (Biospec; Bartistsville, OK). The lysates were cleared in an Eppendorf microfuge at 14,000 rpm for 10 minutes. Cleared lysates were concentrated in UltraFree 100 kDa units (Millipore; Bedford, MA), collected, adjusted to 10 mg/mL in phosphate buffer, and submitted for insect bioassay.<br>
10262]	Expression of T7 Based Constructions. The T7 based expression plasmids were handled<br>
the same as the pBT expression plasmids described above, with the exception that they were transformed into the T7 expression strain BL21(DE3) (Novageri, Madison, WT), and a combination of streptomycin (25 ug/mL) and spectinomycin (25 ng/mL) was used for the antibiotic selection.<br>
Example 3 - Insect bioassay results of heterologously expressed toxin complex genes<br>
[0263]	A series of expression experiments was performed using the pBT expression system as<br>
described above. E. coli cells were transformed, induced and grown overnight at 28°C. The cells<br>
were collected, washed, normalized to equal concentrations and applied to Southern com rootworm diet and bioassayed. As-shown in Table 8, only when all three Photorhabdus genes, tcdA, tcdBl and tccCl were expressed in the same cell was significant mortality observed. Other combinations of genes did not result in significant mortality. For example, the specific combination of tcdBl and tccCl genes, which showed no insect killing activity, is shown in Table 8. Mortality was observed routinely when the genes on plasmid pBT-TcdA-TcdB 1 -TccC 1 were expressed, and storing the cells at 4°C for 24 hours before application to insect diet did not decrease or increase mortality significantly in these experiments (Table 8). Southern com rootworm mortality was also observed if the cells were lysed following co-expression of the genes on plasmid pBT-TcdA-TcdBl-TccC 1. Activity did not appear to decrease if the lysate was stored frozen at -70°C for one week (Table 9).<br>
(Table Removed)<br>
Whole E. coli cells were washed with phosphate buffer, concentrated, adjusted to equal cell concentration, and applied to insect diet. Day 1 samples were assayed immediately. Day 2 samples were the same preparations of cells, but had been stored overnight at 4°C before application to insect diet.<br>
Grading Scale represents % mortality of Southern com rootworm (0 = 0-10%; + = 11-20%; ++ = 21-40%; +++ = 41-60%; ++++ = 61-80%; +++-H- = 81-100%).<br><br>
Table 9. Bioassay of pBT-E\pressed Photorhabdus Toxin Complex Genes on<br>
(Table Removed)<br>
Whole E. coli cells were washed with phosphate buffer, concentrated, adjusted to equal cell concentrations, and applied to insect diet preparations. Alternatively, lysates were prepared by sonication and applied to diet either fresh or after being frozen at -70°C for 7 days. Grading Scale represents % mortality of Southern com rootworm (0 = 0-10%; + = 11 -20%; ++ = 21-40%; +++ = 41-60% = ++++, 61-80%; +++++ = 81-100%).<br>
[00264]	In another series of experiments, the Xenorhabdus xptBl gene was substituted for the<br>
Photorhabdus tccCl gene and expressed as part of the polycistronic operon of plasmid pBT-TcdA-TcdB 1-XptB 1. These experiments demonstrated that the Xenorhabdus xptBl gene was able to substitute for the Photorhabdus tccCl gene, resulting in mortality of Southern corn root worm in bioassay of whole E. coli cells (Table 10).<br>
Table 10. Bioassay of pBT Expressed Photorhabdus and Xenorhabdus Toxin<br><br>
(Table Removed)<br>
Whole E.  coli cells were washed with phosphate buffer, concentrated, adjusted to equal cell concentrations, and applied to insect diet preparations.<br>
Grading Scale represents % mortality of Southern corn rootworm (0 = 0-10%; + = 11-20%; ++ = 21-40%; +++ = 41-60% = +++-K 61-80%; +++++ = 81-100%).<br>
[0065]	Expression of the various Photorhabdus genes from separate plasmids also resulted in<br>
Southern corn root worm mortality. When tcdA was present on the pET expression plasmid, and the tccCl and tcdBl genes were on the compatible expression vector pCot-3, significant activity was observed as compared to control combinations of these plasmids (Table 11). As noted above, the presence of the tcdBl and tccCl genes alone did not result in significant activity (Table 11).<br><br>
Table 11. Bioassay of pCoT/pET (T7 promoter) Expressed Photorhabdus Toxin Complex Genes on Southern Com Rootworm<br>
(Table Removed)<br>
Whole E. coli cells were washed with phosphate buffer, concentrated, adjusted to equal cell concentrations, and applied to insect diet preparations.<br>
Grading Scale represents % mortality of Southern com rootworm (0 = 0-10%; -i- = 11-20%; ++= 21-40%; -H-+ = 41-60% = ++++, 61-80%; •+++++ = 81-100%).<br>
0206}	Bioassay Results of Heterologously Expressed Xenorhabdus Toxin Complex_Genes. A<br>
series of expression experiments was performed using the pET expression system as described above. E. coli cells were transformed, induced and grown overnight at 28°C. The cells were collected, washed, normalized to equal concentrations, and tested for insecticidal activity against Osfrinia mibilalis European corn borer (ECB), com earworm (CEW), and tobacco budworm (TBW). As shown in Table 12, the highest levels of insecticidal activity were observed when xptA2,xptCJ, and xptBl were present in the same construct.<br><br>
(Table Removed)<br>
*Whole E. coli cells were washed with phosphate buffer, concentrated, adjusted to equal cell concentrations, and applied to insect diet preparations.<br>
Grading Scale represents % growth inhibition relative to controls (0 = 0-10%; + = 11-20%; ++ = 21-40%; -H-+ = 41-60% =++++, 61-80%; -H-+-H- = 81-100%).<br>
Bioassav results pfhgterologously expressed xptA2. tcdBL and tccCl, E. coli cells were co-transformed with the pET280 and pCoT constructs listed in Table 13. Transformants were induced, processed and bioassayed as described above. In these assays, co-transformants that<br><br>
contained either PCOT/pET280-XptA2-XptCl-XptB 1 or pCoT-TcdBl-TccCl/pET280-XptA2 plasmid combinations exhibited the highest levels of insecticidal activity. These experiments show that the Photorhabdus tcdBl and tccCl genes, even in trans relative to xptA2, were able to substitute for the Xenorhabdus xptCl and xptBl genes, resulting in qualitatively similar levels of enhanced insecticidal activity.<br>
Table 13. Bioassay of Heterologously ExpressedxptA2, tcdBl,<br>
(Table Removed)<br>
* Whole E. coli cells were washed with phosphate buffer, concentrated, adjsuted to equal cell concentrations, and applied to insect diet preparations. Grading Scale represents % growth inhibition relative to controls (0 = 0-10%; + = 11 -20%; ++ = 21 -40%; +++ = 41 -60% = ++++, 61-80%; +++++ = 81-100%}.<br>
Example 4 - Complementation of Xenorhabdus XptA2 Toxin with Paenibacillus Strain DAS 1529 TC Proteins<br>
[0026S]	This example provides additional data relating to co-pending U.S. provisional application<br>
serial no. 60/392,633, which is discussed in the Background section above. This data is relevant<br>
to the present application because it provides experimental evidence of the ability of<br>
Paenibacillus strain DAS 1529 TC proteins (expressed here as a single operon) to complement,<br>
for example, the XptA2 toxin from Xenorhabdus nematophilus Xwi (see SEQ ID NO:34). Two<br>
independent experiments were carried out to express the DAS 1529 TC operon and XptA2<br>
independently, or to co-express the XptA2 gene and the TC operon in the same E. co/rcells.<br>
Whole cells expressing different toxins/toxin combinations were tested for activity against two<br>
lepidopteran insects: corn earworm (Heliothis zea; CEW) and tobacco budworm (Heliothis<br>
virescens; TBW). The data from both experiments indicate that DAS 1529 TC proteins are able<br>
to enhance Xenorhabdus XptA2 activity against both insect species tested.<br>
[00269]	A. Co-expression of DAS 1529 TC Proteins and Xenorhabdus TC XptA2 Toxin<br>
[Q0270]	Expression of the DAS 1529 TC operon was regulated by T7 promoter/toe operator in the<br>
pETlOl .D expression vector that carries a ColEl replication origin and an ampicillin resistance selection marker (Invitrogen). A comprehensive description of cloning and expression of the<br><br>
 Paenibacillus TC operon can be found in Example 8 of U.S. Serial No. 60/392,633. The XptA2 gene was cloned in the pCo.t-3 expression vector, which carries a chloramphenicol resistance selection marker and a replication origin compatible with the ColEl. The pCot-3 vector expression system is also regulated by the T7 promoter//ac operator. Hence, compatible replication origins and different selection markers form the basis for co-expression of the TC operon and XptA2 in the same E. coli cells. Plasmid DNAs carrying the TC operon and XptA2 were transformed into E. coli, BL21 Star™ (DE3) either independently or in combination. Transformants were selected onLB agarplates containing 50 fig/ml carbenicillin forpETl0l.D-TC operon, 50µg/ml chloramphenicol forpCot-3-XptA2, and both antibiotics forpETl0l.D-TC operon/pCot-3-XptA2. To suppress basal toxin expression, glucose at a final concentration of 50 mM was included in both agar and liquid LB medium.<br>
[0071]	For toxin production, 5 mL and 50 mL of LB medium containing antibiotics and 50 mM<br>
glucose were inoculated with overnight cultures growing on the LB agar plates. Cultures were grown at 30°C on a shaker at 300 rpm. Once the culture density reached an O.D. of ~ 0.4 at 600 nm, IPTG at a final concentration of 75 µM was added to the culture medium to induce gene expression. After 24 hours, E. coli cells were harvested for protein gel analysis by the NuP AGE system (Invitrogen). Cell pellets from 0.5 mL IX culture broth was resuspended in 100 µL of IX NuP AGE LDS sample buffer. Following brief soaication and boiling for 5 min, 5 ^L of the sample was loaded onto 4 to 12% NuP AGE bis-tris gradient gel for total protein profile analysis. XptA2 expressed to detectable levels when expressed independently or in the presence of the TC operon. Based on gel scan analysis by a Personal Densitometer SI (Molecular Dynamics), XptA2 expressed nearly SX as high by itself as when co-expressed with the TC operon. For the 5 mL induction experiment, there is a nearly equal expression of XptA2.<br>
[00272]	B. Bioassay for Insecticidal Activity<br>
[00273]	As described in Example 8 of U.S. Serial No. 60/392,633: DAS1529 TC OPxFs, when<br>
expressed independently or as an operon, did not appear to be active against TBW and CEW. The following bioassay experiments focused on determining whether Paenibacillus (DAS 1529) TC proteins (of OPvFs 3-6; TcaA-, TcaB-, TcaC-, and TccC-like proteins; see SEQ ED NOs:35-43) could potentiate Xenorhabdus TC protein (XptA2 is exemplified) activity. Bioassay samples were prepared as whole E. coli cells in 4 X cell concentrate for the 5 mL induction experiment, both the XptA2 and XptA2/TC operon cells contained very low but nearly equal amount of<br><br>
XptA2. Data in Table 14 showed that at the 4X cell concentration, the combination of the Paenibacillus TC proteins ("TCs" in Table 14) +XptA2 was active against CEW. This demonstrates a complementation effect of Paenibacillus DAS1529 TCs onXenorhabdus XptA2.<br>
Table 14. Bioassay of DAS1529 TC potentiation<br><br>
(Table Removed)<br>
* -, ++, +++ = no, moderate and high activity, respectively<br>
W274]	For the second bioassay experiment, the amount of XptA2 protein in the XptAZ cells and<br>
the X.ptA2 + TC operon cells was normalized based on densitometer gel scan analysis. As shown in Table 15, XptA2 per se had moderate activity at 40X on TBW (H. virescens), but the activity dropped to a level undetectable at and below 20X. However, when co-expressed with the Paenibacillus TC proteins, high levels of activity were very apparent in the presence of 1 OX and 5X XptA2, and low activity was still noticeable at 1.25X XptA2. These observations indicate there is a significant potentiation effect of these DAS 1529 TC proteins on XptA2 against H. virescens. At the highest doses tested, neither the negative control nor the TC operonperse had any activity against this pest.<br>
(Table Removed)<br>
* n.d. - not determined; -, +, ++, +++ = no, low, moderate, and high activity, respectively<br>
Example 5 -Xenorhabdus bovienii B and C protein mixed complementation<br>
Example 5A. Overview<br>
00275]	The identification and isolation of genes encoding factors that potentiate or synergize the<br>
activity of the insect active proteins Photorhabdus TcdA and Xenorhabdus XptA2Wi were accomplished using a cosmid complementation screen. Individual Escherichia coli clones from a<br><br>
cosmid genomic library ofXenorhabdus bovienii (strain ILM104) were used to create crude cell extracts which were mixed with purified toxins and bioassayed. Lysates were assayed with purified Photorhabdus toxin TcdA against southern corn rootworm larvae (Diabrotica undecimpunctata howardi). Likewise, lysates were also mixed with purified Xenorhabdus XptA2Wi protein and assayed against tobacco budworm (Heliothis virescens) or corn earworm (Helicoverpa zed) larvae. Cosmid lysates were scored as positive if the combination of lysate plus purified toxin had activity greater than either component alone.<br>
[00276]	The primary screen samples (in 96-well format) were tested in duplicate and scored<br>
compared to controls for insecticidal activity. Positive samples were re-grown and tested in the secondary screen. Cosmids identified as positive through primary and secondary screens were screened a third time. Larger culture volumes were utilized for tertiary screens (see below), tested for biological activity in a 128-well format bioassay.<br>
[00277]	DNA from one of the cosmids identified as having potentiating activity in this screen was<br>
subcloned. The DNA sequence of a single subclone which retained activity was determined and shown to contain two open reading frames, designated xptB1Xb and xptdxb. These coding regions were subcloned into pET plasmids and expressed in E. coli. A dramatic increase in insect activity was seen when either TcdA or XptA2wi protein was mixed with lysates co-expressing both XptBlXb and XptClxb- Lysates containing only XptBlXb or only XptClxb had minimal affects when mixed with purified TcdA or XptA2Wi. Example 5B. Insect Bioassay Methodology<br>
[00278]	Insect bioassays were conducted using artificial diets in either 96-well microtiter plates<br>
(Becton Dickinson and Company, Franklin Lakes, NJ) or 128-well trays specifically designed for insect bioassays (C-D International, Pitman, NJ). Eggs from 2 lepidopteran species were used for bioassays conducted in 96-well microtiter plates: the com earworm (Helicoverpa zea (Boddie)) and the tobacco budworm (Heliothis virescens (F.)). Neonate larvae were used for bioassays conducted in 128-well trays. The lepidopteran species tested in this format included the com earworm, the tobacco budworm, and the beet armyworm (Spodoptera exigua (Hiibner)). A single coleopteran species, the southern com rootworm (Diabrotica undecimpunctata howardii (Barber)) was also tested in this bioassay format.<br><br>
The data recorded in these bioassays included the total number of insects in the treatment,<br>
number of dead insects, the number of insects whose growth was stunted, and the weight of<br>
surviving insects. In cases where growth inhibition is reported, this was calculated as follows:<br>
[0080]	% Growth Inhibition = [ 1 - (Average Weight of Insects in Treatment/ Average Weight of<br>
Insects in the Vector-Only Control)]* 100<br>
Example 5C. Other Experimental Protocol<br>
[0081]	This is described in more detail in concurrently filed application by Apel-Birkhoid et al,<br>
entitled "Toxin Complex Proteins and Genes fromXenorhabdus bovienli" under attorney docket<br>
no. DAS-114P (Serial No.	).<br>
Example 5D.<br><br>
[0082]	Plasmid pDAB6026 was shown to encode activities which synergized the insect toxic<br>
activities of TcdA and XptA2wi. E. coli cells containing plasmid pDAB6026 or the pBCKS+ vector control were inoculated into 200 mL of LB containing chloramphenicol (50 µg/mL) and 75 µM 1PTG (isopropyl-ß-D-thiogalactopvranoside) and grown for two days at 28°C with shaking at 180 rpm. The cells were then centrimged for 10 min at 3500 x g. The pellets were resuspended in 5 mL of Butterfield's phosphate solution (Fisher Scientific) and transferred to 50 mL conical tubes containing 1.5 mL of 0.1 mm diameter glass beads (Biospec, Bartlesville, OK, catalog number 1107901). The cell-glass bead mixture was chilled on ice, then the cells were lysed by sonication with two 45 second bursts using a 2 mm probe with a Branson Sonifier 250 (Danbury CT) at an output of ~20, chilling completely between bursts. The supernatant was transferred to 2 mL microcentrifuge tubes and centrifuged for 5 min at 16,000 x g. The supernatants were then transferred to 15 mL tubes, and the protein concentration was measured. Bio-Rad Protein Dye Assay Reagent was diluted 1:5 witht2O and 1 mL was added to 10µLof a 1:10 dilution of each sample and to bovine serum albumin (BSA) at concentrations of 5, 10,15, 20 and 25 µg/mL. The samples were then read on a spectrophotometer measuring the optical density at the wavelength of 595 in the Shimadzu UV160U spectrophotometer (Kyoto, JP). The amount of protein contained in each sample was then calculated against the BSA standard curve and adjusted to between 3-6 mg/mL with phosphate buffer. Six hundred nanograms of XptA2w, toxin protein were added to 500 pL of the E. coli lysate prior to testing in insect feeding bioassays. The combination of pDAB6026 and XptA2 was shown to have potent activity (Table 16).<br><br>
Table 16. Response of 2 lepidopteran species to pDAB6026 lysates alone and with purified XptA2v j)rotem.<br>
(Table Removed)<br>
Example 5E. Discovery, Engineering and Tcsting of xptBlxb, and xptCIxb Genes<br>
[0083]	DNA of plasmid pDAB6026 was sent to vSeqWright DNA Sequencing (Houston, TX) for<br>
DNA sequence determination. Two complete open reading frames (ORFs) of substantial size were discovered. The first (disclosed as SEQ ID NO:48) has significant similarity to known toxin complex genes belonging to the "B" class. This ORF was therefore called xptBlXb and encodes the protein disclosed as SEQ ID NO:49. The second ORF (SEQ ID NO:50) encodes a protein (SEQ ID N0:51) with homology to toxin complex "C" proteins and therefore was named<br>
xptClSb-<br>
The xptBlxb and xptClxb genes were engineered (using the polymerase chain reaction; PCR) for high level recombinant expression by addition of restriction sites 5' and 3' to the coding regions, as well as provision of ribosome binding sequences and optimal translational stop signals. In addition, silent mutations (no change in amino acid sequence) were introduced into the 5' end of the coding regions to reduce potential secondary structure of the niRNA and hence increase translation. The strategy was to amplify/engineer segments at the 5' and 3' ends of the genes, join the distal fragments using 'Splice Overlap Extensions' reactions, then add the non-amplified center portion of the open reading frames via restriction sites. This approach minimized the potential of PCR-induced changes in the DNA sequence. The engineered coding regions were cloned into pET expression plasmids (Novagen, Madison, Wl) as either separate coding regions (SEQ ID NO:52 and SEQ ID NO:53) or a dicistronic operon (SEQ ID NO:54). The names of the expression plasmids are shown in Table 17.<br>
Table 17. Expression plasmids containing various coding regions<br>
cloned into the pET vector.	<br>
(Table Removed)<br>
 [0085]	Competent cells of the E. coli T7 expression strain BL21 Star    (DE3) (Stratagene, La<br>
Jolla, CA) were freshly transformed with DNA of either the pET (control) vector or plasmids pDAB6031, pDAB6032 or pDAB6033, and inoculated into 250 mL ofLB containing 50 µg/mL chloramphenicol arid 75 µM. BPTG. After growth for 24 hrs at 2S°C with shaking at 180 rpm, the cells were centrifuged for 10 min at-5500-xg. The pellets were resuspended in 5 mL of phosphate solution and transferred to 50 mL conical tubes containing 1.5 mL of 0.1 mm diameter glass beads, then were sonicated for two 45 sec bursts at "constant" and a setting of 30 as described above. The samples were centrifuged at 3000 x g.for 15 min, the supernatant was transferred to 2 mL microcentrifuge tubes, centrifuged for 5 min at 14,000 rpm, and the supernatants were then transferred to 15 mL tubes. The protein concentrations were measured as described above and the lysates were adjusted to 5 mg/mL with phosphate buffer. A set of three samples per lysate was submitted for insect bioassay. To the first sample, phosphate buffer was added in place of purified toxin; to the second sample, sufficient TcdA protein was added to provide a dose of 50 ng/crrf in the insect bioassay well,- and to the third sample, sufficient<br>
»j<br>
XptA2wj protein was added to provide a dose of 250 ng/cm in the insect bioassay well.<br>
wise]	The results of the bioassay are shown in Table 18. Control samples, which were not<br>
supplemented with low levels of added TcdA or XptA2wi protein, (e.g. samples from vector, . pDAB6031, pDAB6032 and pDAB6033), had little impact on the insects. Likewise, samples which contained low levels of TcdA or XptA2Wj, with either pDAB6031 or pDAB6032 lysates, had minimal effects. In contrast, significant activity was observed in the samples which included low levels of TcdA or XptA2Wj with pDAB6033 lysates.<br>
(Table Removed)<br>
Example 5F. Identification, Purification, and Characterization of XptB 1^ and XptClxh proteins of Xenorhabdus bovienii strain ILM104<br>
[0087]	Bioassay driven fractionation of a pDAB6033-containing E. coli lysate resulted in the<br>
identification by MALDI-TOF of two co-purifying proteins; XptBlxb and XptClxb- Peaks containing these 2 proteins effectively potentiated the activity of TcdA and XptA2wi.<br>
[0088]	Active fractions were identified based en their ability to synergize or potentiate the<br>
activity of TcdA against southern corn rootworm or XptA2wj against com earworm. All bioassays were conducted in the 128-well format described above in Example 5A.<br>
[0089]	Two peaks of activity were detected from protein fractions eluting between 22-24 mS/cm<br>
conductance (Peak 1 and Peak 2). An example of the potentiating activity of Peaks 1 and 2 is shown in Table 19. Subsequent purification and analysis were performed on both Peak 1 and Peak 2<br>
[00290]	Gels from both Peak 1 and Peak 2 contained two predominant bands, one migrating, at<br>
-170 kDa and the other migrating at -80 kDa. The gel from Peak 1 contained three additional proteins that migrated at approximately 18,33 and 50 kDa. Retrospective analysis revealed that<br><br>
the -170 kDa and -80 kDa bands were abundant at the initial stages of purification and became progressively enriched at each step<br>
[oo29]	Extracted peptides were analyzed using MALDI-TOF mass spectrometry to produce<br>
peptide mass fingerprints (PMF) on a Voyager DE-STR MALDI-TOF mass spectrometer (PerSeptive Biosystems, Framingham, MA). Analysis of the samples extracted from the -170 kDa band confirmed the identity as XptBlxb- Analysis of the samples extracted from the ~80 kDa band confirmed the identity as XptClxb- Although the predicted molecular weight of the XptClxb protein as calculated from the gene sequence (SEQ ID NO:50) is 108 kDa, the extracted protein ran significantly faster than expected in the SDS/PAGE. The presence of peptide fragments representing the entire peptide sequence indicated that the protein as extracted is full length.<br>
Table 19. Biological activity of purified Peak 1 and Peak 2 from pDAJB6033.<br>
(Table Removed)<br>
walues in column labeled Sample represent the concentration of Peak 1 or Peak 2 XptB 1 xb/XptC 1 Xb proteins appl ied to the diet (in ng/cnr). For bioassays against com earworm, 250 ng/cm2 of XptA2xi was included in the bioassay. For bioassays against southern com rootworm, 100 ng/cm" of TcdA was included in the bioassay. A total of eight larvae were used per sample.'<br>
roo292]	Example 6 - Additional Mix and Match Example<br>
roo293]	In this example, it is demonstrated that potent insect suppression is obtained with a<br>
combination of three toxin complex (TC) proteins. Compelling insect activity is observed when a Class A protein is mixed with a Class B and Class C protein. The present invention is surprising in that many combinations of a Class A, Class B and Class C protein result in powerful insect repression. The Toxin Complex, proteins may be from widely divergent sources and may only share a limited amount of amino acid identity with other functional members of its class.<br><br>
Example 6A. Introduction<br>
[00294]	The insecticidal and growth inhibition activities encoded by fifteen different toxin<br>
complex genes were tested separately and in combination with one another. Several examples from each of the described classes, A, B or C, were tested. The genes were derived from three genera (Photorhabdus, Xenorhabdus and Paenibacillus; both gram negative and gram positive bacteria) and four different species. The results within this example are consistent with the observation that Toxin Complex Class A proteins (e.g. TcdA and XptA2Wj) have significant activity alone. This was recently shown in transgenic plants by Liu et al. (Liu, D., Burton, S., Glancy, T., Li, Z-S., Hampton, R., Meade, T. and Merlo, D.J. "Insect resistance conferred by 283-kDa Photorhabdus luminescens protein TcdA in Arabidopsis tlialiana" Nature Biotechnology October 2003. Volume 21, number 10 pages 1222-1228). The results also agree with the observation that co-expression of three toxin complex genes (Class A, Class B and Class C) from within the same operon, strain, or genus result in greater insect activity than the Class A gene alone, or any single or double combination of the three classes (Hurst, Ad., Glare, T.. Jackson, T. and Ronson, C. "Plasmid-Located Pathogenicity Determinants of Serratia entomophila, the Causal Agent of Amber Disease of Grass Grub, Show Similarity to the Insecticidal Toxins ofPhotorhabdus luminescens". Journal of Bacteriology, Sept 2000, Volume 182, Number 18, pages 5127-5138; Morgan, J.A., Sergeant, M., Ellis, D., Ousley, M. and Jarrett, P. "Sequence Analysis of Insecticidal Genes from Xenorhabdus nematophilus PMFI296". Applied and Environmental Microbiology, May 2001, p. 2062-2069, Vol. 67, No. 5; Waterfield, N., Dowling, A., Sharma, S., Daborn, P., Potter, U. and Ffrench-Constant., R. "Oral Toxicity of Photorhabdus luminescens W-14 Toxin Complexes in Escherichia coli,'" Applied and Environmental Microbiology, Nov.2001, Volume 67, Number 11, pages 50] 7-5024).<br>
W295J	Surprisingly, the data below document the discovery that toxin complex" Class A proteins<br>
maybe mixed and matched with, for example, lysates prepared from E. coli cells programmed to express Class B and Class C genes from widely divergent sources to produce stunning insecticidal and insect growth inhibition activity. For example, a Class A protein from Xenorhabdus may be mixed with a lysate programmed to express a Class B gene from - Photorhabdus and a Class C gene from Paenibacillus to provide an insect active combination.. Likewise, a Class A protein from Photorhabdus may be mixed with a lysate programmed to express a Class B and Class C gene from Xenorhabdus, and vice versa. Many combinations are<br><br>
possible; many are shown below to result in potent insect activity. It was an unexpected revelation that toxin complex Class A, B, and C components from strains noted for either coleopteran (Photorhabdus luminescens strain W-14) or lepidopteran activity (Xenorhabdus nematophilus strain Xwi) may be functionally mixed and matched. Additionally surprising was the discovery of the degree of divergence possible for individual A, B or C proteins. For example, individual Class A's (e.g. TcdA and XptA2Wi) which function with a Class B/Class C combination may only share 41 % ammo acid identity with each other. Likewise any individual Class B may only share 41% identity with another functional Class B protein. Similarly, any given Class C may share only 35% identity with another Class C protein.<br>
Example 6B. Protein Sources and Constructions<br>
o296]	The Class A proteins TcdA and XptA2wi were utilized in a purified form prepared from<br>
cultures ofPseudonionasfluorescens heterologously expressing the proteins. Preparations of the TcdA and XptA2Wi from other heterologous sources (plant; bacterial) were functionally equivalent in the assays. The Class B and Class C proteins were tested as components of E. coii lysates. The use of lysates was validated by comparison to purified preparations of several Class B and Class C combinations. Reading frames encoding Class B and Class C proteins were engineered for expression in E. coli by cloning into pET plasmids (Novagen, Madison WI). Each coding region contained an appropriately spaced ribosome binding site (relative to the start codon) and termination signal. The DNA sequences at the 5' end of some of the genes were modified to reduce predicted secondary structure of the RNA and hence increase translation. These base changes were silent and did not result in amino acid changes in the protein. In cases where a Class B gene was tested with a Class C gene, an operon was constructed in the pET expression plasmid with the Class B coding sequence being transcribed first, followed by the Class C coding sequence. The two coding regions were separated by a linker sequence which contained a ribosome binding site appropriately spaced relative to the start codon of the Class C protein coding region. The DNA sequence between the, coding regions in the dicistronic constructions is shown in the 5' to 3'orientation. Tables 20-27 contain lists of the proteins encoded by the various expression plasmids, the source of the coding regions and the plasmid reference number. Tables 22B, 23B, and 28-31 show linker sequences used in expression plasmids.<br><br><br><br><br>
(Table Removed)<br>
Example 6C. Expression Conditions and Lysate Preparations<br>
[00297]	The pET expression plasmids listed in Tables 20-27 were transformed into the E. coli T7<br>
expression strains BL21(DE3) (Novagen, Madison WI) or BL21 Star™ (DE3) (Stratagene, La Jolla, CA) using standard methods. Expression cultures were initiated with 10-200 freshly transformed colonies into 250 mL LB 50 µg/ml antibiotic and 75 uM IPTG. The cultures were grown at 28°C for 24 hours at 180-200 rpm. The cells were collected by centrifugation in 250 ml Nalgene bottles at 3,400 x g for 10 minutes at 4°C. The pellets were suspended in 4-4.5 mL Butterfield's Phosphate solution (Hardy Diagnostics, Santa Maria, CA; 0.3 mM .potassium phosphate pH 7.2). The suspended cells were transferred to 50 mL polypropylene screw cap • centrifuge tubes with 1 mL of 0.1 mm diameter glass beads (Biospec, BartlesviJle, OK, catalog number 1107901). The cell-glass bead mixture was chilled on ice, then the cells were lysed by sonication with two 45 second bursts using a 2 mm probe with a Branson Sonifier 250 (Danbury CT) at an output of-20, chilling completely between bursts. The lysates were transferred to 2 mL Eppendorf tubes and centrifuged 5 minutes at 16,000 x g. The supematants were collected and the protein concentration measured. Bio-Rad Protein Dye Assay Reagent was diluted 1:5 with H2O and 1 mL was added to 10 µL of a 1:10 dilution of each sample and to bovine serum albumin (BSA) at concentrations of 5,10,15,20 and 25 µg/mL. The samples were then read on a spectrpphotometer measuring the optical density at the wavelength of 595 nm in the Shimadzu UV160U spectrophotometer (Kyoto, JP). The amount of protein contained in each sample was then calculated against the BSA standard curve and adjusted to between 3-6 mg/mL with phosphate buffer. The lysates were typically assayed fresh, however no loss in activity was observed when stored at -70°C.<br>
Example 6D. Bioassav Conditions<br>
[00298]	Insect bioassays were conducted with neonate larvae on artificial diets in 128-well trays<br>
specifically designed for insect bioassays (C-D International, Pitman, NJ). The species assayed were the southern com rootworm, Diabrotica undecimpunctata howardii (Barber), the com<br><br>
earworm, Helicoverpa zea (Boddie), the tobacco budworm, Heliothis virescens (F.), and the beet armywomi, Spodoptera exigua (Hubner).<br>
00299]	Bioassays were incubated under controlled environmental conditions (28°C, -40% r.h.,<br>
16:8 [L:D]) for 5 days at which point the total number of insects in the treatment, the number of dead insects, and the weight of surviving insects were recorded. Percent mortality and percent growth inhibition were calculated for each treatment. Growth inhibition was calculated as follows:<br>
% Growth Inhibition = [1 - (Average Weight of Insects in Treatment/ Average Weight of Insects in the Vector-Only Control)]* 100<br>
00300]	In cases where the average weight of insects in treatment was greater that of insects in the<br>
vector only control, growth inhibition was scored as 0%.<br>
'oo3oi]	The biological activity of the crude lysates alone or with added TcdA or XptA2Wi toxin<br>
proteins was assayed as follows. Crude E. coli lysates (40 µL) of either control cultures or those expressing potentiator proteins were applied to the surface of artificial diet in 8 wells of a bioassay tray. The average surface area of treated diet in each well was —1.5 cm2. The lysates were adjusted to between 2-5 mg/'mL total protein and were applied with and without TcdA or XptA2,w1. The TcdA or XptA2W1 added were highly purified fractions from bacterial cultures heterologously expressing the proteins. The final concentrations of XptA2wi and TcdA on the diet were 250 ng/cm2 and 50 ng/cm2, respectively.<br>
'00302]	The results of bioassays are summarized in Tables 32-39.  Little to no effect on the<br>
survival or growth of the insect species tested was observed when larvae were fed lysates from E. coli clones, engineered to express a Class B or Class C protein alone (Tables 32 and 33). Similarly, little to no effect was observed when larvae were fed combinations of Class B and Class C proteins in the absence of a purified toxin (Tables 34-39, "none" column). Significant effects on survival and/or growth were commonly observed when larvae were fed lysates from E. coli clones engineered to express a combination of a Class B and Class C protein with a purified toxin (Tables 2C-2H, "TcdA" and "XptA2wi" columns). Class B and Class C combinations tested with purified TcdA most typically exerted an effect on southern com rootworm while combinations tested with purified XptA2Wi typically exerted an effect on one of the 3 lepidopteran species with corn earworm being the most consistently sensitive species. It is notable that many Class B and Class C combinations produced an observable effect of XptA2wl- on southern corn<br><br>
rootworm. The converse, that these combinations would produce an observable effect of TcdA on lepidopteran species, did not hold true.<br>
[0303J	Tables 32-39 show biological activity of E. coli lysates fed to insect larvae alone and<br>
combined with Pholorhabdus or Xenorhabdus toxin proteins. The gene contained in each E. coli clone corresponds to those contained in Tables 20-27. Biological activity is classified using the following scale: 0 = average mortality  50% of the average weight of empty vector/no toxin treatment, + = average weight _50% OR average' weight <_20 of the average weight empty vector toxin treatment and mortality> 95% OR average weight 
<br><br>
(Table Removed)<br>
Example 7 -Additional Mixing and Matching of TC Proteins<br>
00304]	To demonstrate the presently discovered versatility of TC proteins, additional E. coli<br>
expression experiments were done employing double plasmid expression systems. A T7 promoter based system utilized a pACYC derivative (called pCot-3 or 4, chloramphenicol resistant) to express either the TcdA or XptA2 proteins while a compatible T7 promoterpET280 plasmid (kanamycin resistant) expressed various combinations of the TcdBl (SEQ ED NO:22), TcdB2 (SEQ ID NO:45), XptCl (SEQ ID NO: 18), TccCl (SEQ ID NQ:25), TccC3 (SEQ ID N0:47) and XptBl (SEQ ID NO: 16) proteins, all within the same cell. Likewise, in another series of experiments, an E. coli promoter system was used that utilized a different pACYC derivative (called pCTS, spectino'mycin/ streptomycin resistant) to express either TcdA (SEQ ID N0:21) or XptA2 (SEQ ID NO:34) proteins while a compatible pBT280 plasmid (chloramphenicol resistant) expressed various combinations of TcdBl, TcdB2, XptCl, TccCl, TccC3 and XptBl. Both systems produced proteins of similar activities when bioassayed.<br>
'00305]	The T7 promoter based experiments were done by first preparing stocks of competent<br>
BL21(DE3) cells containing either pCot-3, pCot-TcdA or pCot-XptA2. These cells were then transformed with either control pET280 plasmid or any of the combinations of TC genes noted above in the pET280 vector. Cells containing both plasmids were selected on media containing chloramphenicol and kanamycin. Similarly, for the E. coli promoted system, competent BL21 cells containing either pCTS, pCTS-TcdA or pCTS-XptA2 were prepared. The competent cells were then transformed with either pBT280 control plasmid or any of TC combinations noted above in the pBT280 vector. When more than one TC gene was present on a particular plasmid, they were arranged as a two gene operon with a single promoter at the 5' end. The first coding ' region was followed by translational termination signals; a separate ribosome binding site (Shine-Dalgamo sequence) and translational start signal were used to initiation translation of the second coding region. The methods described in Examples 2 and 3 were used to grow expression cultures, prepare lysates and assess insect activity. Some experiments utilized a modified assay method where enriched preparations of proteins TcdA and XptA2 were added to lysates containing either singly or in combination TcdBl, TcdB2, XptCl, TccCl, TccC3 and XptBl (Tables 40 and 41).<br><br>
(Table Removed)<br>
Whole E. coli cells were lysed and the soluble protein generally normalized within an experiment to between 5-10 mg/ml. The lysates were bioassayed as described by top loading onto insect diets. Grading Scale represents % growth inhibition relative to controls (0 = 0-25%; + = 26-50%; ++ = 51-65%; +++ = 66-80% = ++++, 81-95%; +++++ = 96-100%).<br><br><br>
(Table Removed)<br>
Whole E. coH cells were lysed and the soluble protein was adjusted to equal sample concentrations of between 8-15 mg/ml. * TcdA protein was added to the samples for a final concentration of 50ng/cnr when applied on top of the insect diet preparations.<br>
Grading Scale represents % growth inhibition of surviving insects fed the treatment plus TcdA Toxin Protein relative to surviving insects fed the treatment in the absence of TcdA Toxin Protein (0 = 0-10%; + = 11-20%; ++ = 21 -40%; ++=21-40%+++ = 41 -60% = +++, 61 -80%; +++++ &gt;80%).<br><br>
(Table Removed)<br>
Whole E. coli cells were lysed and the soluble protein was adjusted to equal sample concentrations of between 8-15 mg/ml. * XptA2 protein was added to the samples for a final concentration of 250ng/'cnr when applied OR top of the insect diet preparations.<br>
Grading Scale represents % growth inhibition of surviving insects fed the treatment plus XptA2 Toxin Proiein .relative to surviving insects fed the treatment in the absence of XptA2 Toxin Protein (0 = 0-10%; -r = 11-20%; ++ - 21-40%; -H-+' = 41-60% = ++++, 61-80%: Mill &gt;80%).<br>
Example 8 - Summary of Mix &amp; Match Assays and Sequence Relatedness<br>
•00306}	The following Tables summarize and compare proteins used in the assays described<br>
above. Tables 43-45 compare A, B, and C class proteins. Tables 46-48 compare A, B, and C class genes (bacterial). Any of the numbers in these tables can be used as upper and/or lower limits for defining proteins and polynucleotides for use according to the subject invention. Table 49 compares the sizes of various TC proteins. Again, any of the numbers in this table ca.n be used to define the upper and/or lower size limits of proteins (and polynucleotides) for use according to the subject invention.<br><br>
o307]	These tables help to show that even highly divergent proteins (in the -40-75% identity<br>
range) can surprisingly be used and substituted for each other according to the subject invention. TcdA2W-i4 is reproduced here as SEQ ID NO:62, TcdA4W-i4 as SEQ ID NO:63, and TccCw-u as SEQ ID NO:64.<br><br><br>
(Table Removed)<br><br>
NOTE: tcdA3 is a pseudo gene (does not encode a full-length protein) so is left out of this analysis<br><br>
(Table 44 Removed)<br>
(Table 45 Removed)<br>
(Table 46 Removed)<br>
(Table 47 Removed)<br>
(Table 48 Removed)<br>
(Table 49 Removed)<br><br><br><br><br><br>
The Applicant hereto submits that the corresponding U.S. case having Application Number 10/754.115 is allowed, and a patent will issue on February 17 as U.S. Patent No. 7,491,698.<br>
The claims are amended as follows.<br>
21.	(currently amended) A method of controlling or inhibiting an insect wherein said<br>
method comprises contacting said insect with effective amounts of a Protein A, a<br>
Protein B, and a Protein C, wherein said Protein A is an approximately 230-290 kDa<br>
complex-forming protein having at least 99% sequence identity with SEQ ID NO:34<br>
(XptA2xwi);<br>
said Protein B is an approximately 130-180 kDa complex-forming protein comprising an amino acid sequence selected from the group consisting of SEQ ID NO:22 (TcdB1), SEQ ID NO:56 (TcaC), and an amino acid sequence having at least 99% sequence identity with SEQ ID NO:45 (TcdB2);<br>
said Protein C is an approximately 90-120 kDa complex-forming protein comprising an amino acid sequence selected from the group consisting of SEQ ID NO:25 (TccC1), SEQ ID NO:57 (TccC5): and an amino acid sequence having at least 99% sequence identity with SEQ ID NO:47 (TccC3);<br>
said Protein A has activity against an insect and said activity is potentiated by said Protein B and said Protein C; and said Protein B and said Protein C potentiate the activity of said Protein A.<br>
22.	(currently amended) The method of claim 21 wherein said Protein C comprises the polypeptide of SEQ ID NO:47 (TccC3).<br>
23.	(currently amended) The method of claim 21 wherein said Protein B comprises the polypeptide of SEQ ID NO:45 (TcdB2).<br><br>
24.	(currently amended) The method of claim 21 wherein said Protein C comprises the polypeptide of SEQ ID NO:57 (TccC5).<br>
25.	(currently amended) The method of claim 21 wherein said Protein B comprises the polypeptide of SEQ ID NO:47 (TcdB2) and said Protein C comprises the polypeptide of SEQ ID NO:47 (TccC3).<br>
34.	(currently amended) A method of inhibiting an insect wherein said method<br>
comprises contacting said insect with an A component and a B component,<br>
wherein said components form an insecticidal toxin complex, wherein said A<br>
component is a 230-290 kDa complex-forming protein having at least 99%<br>
sequence identity with SEQ ID NO:34 (XptA2xwi);<br>
said B component is a 130-180 kDa complex-forming protein comprising a polypeptide [[an acid sequence]] selected from the group consisting of SEQ ID NO:22 (TcdB1), SEQ ID NO:56 (TcaC), and an amino acid sequence having at least 99% sequence identity with SEQ ID NO:45 (TcdB2);<br>
wherein said A component has activity against an insect, and wherein said B component is a potentiator of said A component.<br>
35.	(currently amended) The method of claim 34 wherein said A component is the polypeptide of SEQ ID NO:34 (XptA2xwi).<br>
36.	(currently amended) A method of inhibiting an insect wherein said method comprises contacting said insect with an A component and a C component, wherein said components form an insecticidal toxin complex,<br>
wherein said A component is a 230-290 kDa complex-forming protein having at least 99% sequence identity with SEQ ID NO:34 (XptA2xwi);<br><br>
said C component is a 90-120 kDa complex-forming protein comprising an amino acid sequence selected from the group consisting of SEQ ID NO:25 (TccC1), SEQ ID NO:57 (TccC5), and an amino acid sequence having at least 99% sequence identity with SEQ ID NO:47 (TccC3);<br>
wherein said A component has activity against an insect, and wherein said C component is a potentiator of said A component.<br>
37. (currently amended) The method of claim 36 wherein said C component comprises the polypeptide of SEQ ID NO:47 (TccC3).<br>
40.	(currently amended) The method of claim 35 wherein said B component is the polypeptide of SEQ ID NO:45 (TcdB2).<br>
41.	(currently amended) The method of claim 36 wherein said A component is the polypeptide of SEQ ID NO:34 (XptA2xwj).<br><br>
43.	(currently amended) The method of claim 21 wherein said Protein A comprises the polypeptide of SEQ ID NO:34 (XptA2xwi).<br>
44.	(currently amended) The method of claim 23 wherein said Protein A comprises the polypeptide of SEQ ID NO:34 (XptA2xwi); and said Protein C comprises the polypeptide of SEQ ID NO:47(TccC3).<br>
45.	(currently amended) The method of claim 25 wherein said Protein A comprises the polypeptide of SEQ ID NO:34 (XptA2xwj).<br>
46.	(currently amended) The method of claim 34 wherein said B component is the polypeptide of SEQ ID NO:45 (TcdB2).<br>
47.	(currently amended) The method of claim 36 wherein said C component is the polypeptide of SEQ ID NO:57 (TccC5).<br><br>
48.	 (currently amended) The method of claim 41 wherein said C component is the polypeptide of SEQ ID NO:47 (TccC3).<br>
49.	(currently amended) The method of claim 41 wherein said C component is the polypeptide of SEQ ID NO:57 (TccC5).<br>
51. (currently amended) A method of inhibiting an insect wherein said method comprises contacting said insect with an A component, a B component, and a C component, wherein said components form an insecticidal toxin complex,<br>
wherein said A component is a 230-290 kDa complex-forming protein comprising the polypeptide of SEQ ID NO:34 (XptA2xwi);<br>
said B component is a 130-180 kDa complex-forming protein comprising the polypeptide of SEQ ID NO:45(TcdB2);<br>
said C component is an approximately 90-120 kDa complex-forming protein comprising the polypeptide of SEQ ID NO:47 (TccC3); wherein said A component has activity against an insect, and wherein said B component and said C component are potentiators of said A component.<br></_20></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1ERUxOUC0yMDA1LUNsYWltcy0oMTItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-DELNP-2005-Claims-(12-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy0oMDItMDMtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-correspondence-others-(02-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-DELNP-2005-Correspondence-Others-(12-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjctMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-DELNP-2005-Correspondence-Others-(27-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWRyYXdpbmdzLSgwMi0wMy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-drawings-(02-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMTMtKDAyLTAzLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-13-(02-03-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1ERUxOUC0yMDA1LUdQQS0oMTItMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-DELNP-2005-GPA-(12-02-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LXBjdC0zMjYucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1kZWxucC0yMDA1LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">3089-delnp-2005-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA4OS1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMjctMDItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3089-DELNP-2005-Petition-137-(27-02-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234507-semiconductor-module-and-method-of-producing-a-semiconductor-module.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234509-a-linkage-device-especially-for-adjustment-of-rear-view-mirrors-for-motor-vehicles.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234508</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3089/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>03-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Jul-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW AGROSCIENCES LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>9330 ZIONSVILLE ROAD, INDIANAPOLIS, IN 46268-1054, U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BINTRIM, SCOTT, B.</td>
											<td>14189 AUTUMN WOODS DRIVE, WESTFIELD, IN 46074, U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NI, WEITING</td>
											<td>13848 RIVERWOOD WAY, CARMEL, IN 46032, U.S.A.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MITCHELL, JON, C.</td>
											<td>1320 PALMER DRIVE #311, WEST LAFAYETTE, IN 47906, U.S.A.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LI,ZE,SHENG</td>
											<td>1437 ESPRIT DRIVE, WESTFIELD, IN 46074, U.S.A.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ZHU, BAOLONG</td>
											<td>7244 SHORELINE DRIVE, APARTMENT 139, SAN DEIGO, CA 92122, U.S.A.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MERLO, DONALD, J.</td>
											<td>11845 DURBIN DRIVE, CARMEL, IN 46032, U.S.A.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>APEL-BIRKHOLD, PATRICIA, C.</td>
											<td>1371 SAYLOR COURT, ZIONSVILLE, IN 46077, U.S.A.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>HEY, TIMOTHY, D.</td>
											<td>1653 CATALINA WAY, ZIONSVILLE, IN 46077, U.S.A.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>SCHLEPER, AMANDA, D.</td>
											<td>15753 WILDRYE DRIVE, WESTFIELD, IN 46074, U.S.A.</td>
										</tr>
										<tr>
											<td>10</td>
											<td>BEVAN, SCOTT, A.</td>
											<td>1109 WESTFIELD COURT WEST, APARTMENT 6, INDIANAPOLIS, IN 46220, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/000394</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-01-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/441, 723</td>
									<td>2003-01-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234508-a-sprayable-composition-for-application-to-plant by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:17:31 GMT -->
</html>
