<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234535-a-method-for-generating-or-increasing-the-resistance-of-a-plant-to-a-plant-pathogen-of-the-phylum-oomyceta by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:18:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234535:A METHOD FOR GENERATING OR INCREASING THE RESISTANCE OF A PLANT TO A PLANT PATHOGEN OF THE PHYLUM OOMYCETA</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR GENERATING OR INCREASING THE RESISTANCE OF A PLANT TO A PLANT PATHOGEN OF THE PHYLUM OOMYCETA</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a novel method for increasing the resistance, of a plant, in particular of a Solanaceae, preferably of potato and tomato, to plant pathogens of the phylum Oomyceta comprising increasing the activity of the polypeptid of the present invention. The invention further relates to polynucleotides and vectors comprising these polynucleotides. The invention furthermore relates to corresponding vectors, cells, transgenic plants and transgenic propagation material derived from them, methods to produce them and to their use for the production of foodstuffs, feeding stuffs, seed, pharmaceuticals or fine chemicals.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FUNGUS   RESISTANT   PLANTS  AND  THEIR  USES<br>
The present invention relates to a novel method for increasing the resistance of a plant, in particular of a Solanaceae, preferably of potato and tomato, to plant pathogens of the phylum Oomycetes comprising increasing the activity of the polypeptide of the present invention. The invention further relates to polynucleotides and vectors comprising these polynucleotides. The invention furthermore relates to corresponding vectors, cells, transgenic plants and transgenic propagation material derived from them, 'methods to produce them and to their use for the production of foodstuffs, feeding stuffs, seed, Pharmaceuticals or fine chemicals.<br>
The aim of plant biotechnology work is the generation of plants with advantageous novel properties, for example for increasing agricultural productivity, increasing the quality in the case of foodstuffs, or for producing specific chemicals or Pharmaceuticals (Dunwell JM (2000) J Exp Bot 51 Spec No:487-96). The plant1 s natural defence mechanisms against pathogens are frequently insufficient Fungal diseases alone result in annual yield losses of many billions of US$. The introduction of foreign genes from plants, animals or microbial sources can increase the defences. Examples are the protection of tobacco against feeding damage by insects by expressing Bacillus thur-ingiensis endotoxins under the control of the 35S CaMV promoter (Vaeck et al. (1987) Nature 328:33-37) or the protection of tobacco against fungal infection by expressing a bean chitinase under the control of the CaMV promoter (Broglie et al. (1991) Science 254:1194-1197). However, most of the approaches described only offer resistance to a single pathogen or a narrow spectrum of pathogens.<br>
Despite the notorious Irish potato famine of the mid-19th century, late blight still continues to be one of the most devastating of all diseases in crop plants. Late blight is caused by the oomycete fungus Phytophthora infestans, a specialised pathogen, primarily causing disease on the foliage and fruits of a range of Solanaceae species, especially potato and tomato. The fungus was first observed in Mexico and for several reasons Mexico is believed to be the centre of origin of the fungus. Both of the mating types A1 and A2 are permanently present in for example the Toluca area. Also, P. infestans is reported on native Solanum species in remote areas of Mexico. Furthermore, many species of tuber bearing Solanum with a high level of resistance to late blight are found in Mexico. Prevailing measures to prevent crop failures or reduced yields imply the application of fungicides that prevent or cure an infection by P. infestans. Instead of the massive use of chemical pesticides an alternative approach for controlling late blight could be advantageous: the use of cultivars, which harbour partial or complete resistance tp late blight. To obtain late blight resistance, breeders have in the past focussed on the introgression of dominant R genes from Solanum de-missum, a wild potato species indigenous to Mexico. Eleven such R genes have been<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
identified, several of which have been mapped to specific loci on the genetic map of potato (reviewed in Gebhardt and Valkonen, 2001) and recently the R1 gene has been cloned. R1 and R2 are located on chromosomes 5 and 4, respectively. R3, R6 and R7 are located on chromosome 11. Unknown R genes conferring race specific resistance to late blight have also been described in S. tuberosum ssp. andigena and S. berthaultii and S. pinnatisectum. The resistance induced by these R-genes was (nearly) complete but appeared not to be durable in any case. Because of the high level of resistance and ease of transfer, many cultivars contain S. demissum derived resistance. Unfortunately, S. demissum derived race specific resistance, although nearly complete, is not durable. Once newly bred potato cultivars were grown on larger scale in commercial fields, new virulences emerged in P. infestans, which rendered the pathogen able to overcome the introgressed resistance. More durable field resistance to late blight, often quantitative in nature and presumed to be race non-specific, can be found in several Mexican and Central and South American Solanum species. However this type of resistance is difficult to transfer into potato cultivars through crossing and phenotypic selection.<br>
Diploid S. bulbocastanum from Mexico and Guatemala is one of the tuber bearing species that is long known for its high levels of resistance to late blight. Unfortunately, classic transfer of resistance from wild Solanum species to cultivated potato is frequently prevented due to differences in ploidy and Endosperm Balance Number (EBN). Despite these problems, introgression of the S. bulbocastanum resistance trait has been successful. Recently, somatic hybrids of S. bulbocastanum and S. tuberosum and backcrossed germplasm were found to be highly resistant to late blight, even under extreme disease pressure (Helgeson et al., 1998). Despite reports of suppression of recombination, resistance in the backcrossed material appeared to be on chromosome 8 within an approximately 6 cM interval between the RFLP markers CP53 and CT64. A CAPS marker derived from the tomato RFLP probe CT88 cosegregated with resistance.<br>
Accordingly, in the recent years the development of plants resistant to pathogens of the phylum Oomyceta forged ahead. However, 40 years of intense and continuous research and breeding efforts with available germplasm has still not resulted in market introduction of resistant cultivars. The prevailing number of genes identified in the recent years confers merely race specific resistance. Further, the achieved resistance was not durable. In addition, the application of crop protectants is widely considered to be a burden for the environment. Thus, in several Western countries, legislation becomes more restrictive and partly prohibitive to the application of specific fungicides, making chemical control of the disease more difficult. Further, chemical control is expensive. Finally, another restriction is the development of resistance by the fungus to specific fungicides such as metalaxyl, which has been reported from many countries in the world.<br><br>
WO 2005/014631	PC17EP2004/008683<br>
Accordingly, the problem underlying the present invention is to provide novel means and methods for an efficient protection of plants against late blight and related diseases.<br>
The solution of the technical problem is achieved by providing the embodiments characterized in the claims.<br>
Accordingly, the present invention relates to a method for generating or increasing the resistance of a plant to plant pathogen of the phylum Oomycetes comprising increasing the activity of Rpi-blb2 protein in the plant or a tissue, organ or cell of the plant or a part thereof.<br>
Rpi-blb2 is a LZ-NBS-LRR type of R gene and shows sequence homology to the tomato gene Mi-1, that confers resistance to three species of root knot nematodes (Meloidogyne spp.) as well as to the potato aphid Macrosiphum euphorbiae (Vos et al., 1998; Rossi et al., 1998; Milligan et al., 1998) and to both B- and Q-biotypes of whitefly Bemisia tabaci (Nombela et al., 2003). As was found for Rpi-blb, Rpi-blb2 also confers full resistance to a range of P. infestans isolates carrying multiple virulence factors and race-specificity has not yet been demonstrated.<br>
The term *Rpi-blb2" refers to a polynucleotide encoding a polypeptide having the herein mentioned Rpi-blb2 protein activity or a polypeptide having said Rpi-blb2 protein activity. Whether in the following the term "Rpi-blb2" relates to a polypeptide or a polynucleotide is clear from the context of its usage.<br>
By the term "generating" or "increasing" or "stimulating" "the resistance of a plant" is meant that the resistance of a plant or a part thereof is increased or generated or stimulated in comparison to a reference.<br>
"Conferring", "existing", "generating", "stimulating" or "increasing" a pathogen resistance means that the defence mechanisms of a specific plant species or variety is increasingly resistant to one or more pathogens due to the use of the method according to the invention in comparison with the wild type of the plant, to which the method according to the invention has not been applied, under otherwise identical conditions (such as, for example, climatic conditions, growing conditions, pathogen species and the like). The increased resistance manifests itself preferably in a reduced manifestation of the disease symptoms, disease symptoms comprising - in addition to the abovementioned adverse effects - for example also the penetration efficiency of a pathogen into the plant or plant cells or the proliferation efficiency in or on the same. In this context, the disease symptoms are preferably reduced by at least 10% or at least<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
20%, especially preferably by at least 40% or 60%, very especially preferably by at least 70% or 80% and most preferably by at least 90% or 95%.<br>
By the term "increased" it is hereby meant that an activity of a gene product is higher than in a reference. Thus, the term "increased" includes that an activity, e.g. the activity of Rpi-blb2 gene product or of an other gene product, is generated de novo, if that activity, e.g the herein described Rpi-blb2 activity, was not found in the reference. Th
Thus, in the following, the term "increasing" also comprises the terms "stimulating" and "generating".<br>
"Pathogen resistance" denotes the reduction or weakening of disease symptoms of a plant following infection by a pathogen. The symptoms can be manifold, but preferably encompass those which directly or indirectly have an adverse effect on the quality of the plant, the quantity of the yield, the suitability for use as feeding stuff or foodstuff, or else which make sowing, planting, harvesting or processing of the crop difficult.<br>
"Pathogen" within the scope of the invention means by way of example but not by limitation viruses or viroids, bacteria, fungi, animal pests such as, for example, insects or nematodes.<br>
The term uRpi-blb2 protein" relates to a protein or polypeptide which expression in a plant or a "part confers resistance of the plant or a part of the plant to one of the pathogens described herein in comparison to a non-resistant strain.<br>
The plant or a tissue, organ or cell of the plant or a part thereof comprising increased activity of Rpi-blb2 protein is less susceptible to an infection by a pathogen, in particular to pathogen of the phylum Oomycetes, preferably to P. infestans, than a plant or a part thereof which has the identical genetic background but not the genetic elements necessary to allow an expression of Rpi-blb2 (herein named as "wild type" or "reference"). Assays for the testing of the resistance of a plant or a part thereof are well known to a person skilled in the art. The resistance to P. infestans can be defined as sporulation index according to Flier, 2001. Flier describes the sporulation index as a level of sporulation per 1 cm2. Thust a reduction of sporulation per 1 cm2 of 20% com-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
pared to a wild type is herein defined as resistance. In the examples illustrating the present invention, the speculation index was defined as level of sporulation per lesion. Thus, by the term "resistance" can be alternatively meant a reduction of sporulation per lesion of 20% compared to a wild type. The later definition is preferred.<br>
in preferred embodiments the sporulation in an assay is reduced by 30%, more preferred is a reduction of 50%, even more preferred are 70%, even more preferred are more than 80%, more preferred are 85% and 90%. Most preferred is a reduction of 95% or more.<br>
Accordingly, in the present invention by "activity" of a Rpi-blb2 protein is meant, that the protein expression confers said reduction in the sporulation index. Further, it was observed, that a typical response for plants containing Rpi-blb2 to a P. infestans infection is the presence of small lesions, without any clear sporulation, at the end of the growing season. Thus, in one embodiment, the activity of Rp'hblb2 is defined as the presence of small lesions without any clear sporulation in experiments as described. Rpi-blb2 resistance shows necrotic regions that contain a low level of sporulation. An experiment performed with detached leaves exemplifies the activity of Rpiblb2. The experiment is described in example 17 and figure 18The difference between Rpi-blb2 and other P. infestans resistance genes is that Rpi-blb2 allows a low level of sporulation (Figure 18). A detached leaf assay in which the lesions present on Rpi-blb2 genotype (ARD 92-1197-16) shows a low level of sporangia in relation to complete absence of sporangia on a genotype containing the S. demissum gene R2. The sporulation index is only 1.1% of a susceptible phenotype (cv. Bintje) (Table 7 and Figure 18).<br>
Field experiments have also shown that Rpi-blb2 allows a low level of infection. Late blight symptoms developed at a low level during the growing season (Figure 3, ARF87-801) or at the end of the growing season (Figure 2, ARF87-601; Figure 3, ARF87-507 andARF87-601).<br>
Thus, in one embodiment, the activity of Rpi-blb2 is further defined as resulting after expression in a plant in necrotic regions that contain a low level of sporulation in experiments as described.<br>
Thus, in one embodiment, the method of the present invention produces plants showing necrotic regions that contain a low level of sporuiation or less.<br>
The term "reference" relates to an organism or a part thereof, e.g. a cell, which is essentially as identical as possible in genome, proteome, and/or metabolome to the rele-<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
vant organism or part thereof, e.g. a cell, for example to the plant of the present invention.<br>
Thus, the term "reference" relates for example to an organism or a part thereof, e.g. a cell, which is essentially genetically, proteomically, and/or metabolically identical to the organism of the present invention or a part thereof but an activity of a specific gene product, e.g. Rpi-blb2, cannot be observed as there is a relevant difference in the reference's genome, proteome or metabolome. Thus, the reference can be a plant or a part thereof which does not express or expresses too little of a relevant active gene product, e.g. it does not encode a Rpi-blb2 or does not transcribe a Rpi-blb2 encoding gene or does not translate an active Rpi-blb2 mRNA. Thus, the reference does not provide the modification creating an active gene product in a sufficient quantity to result in an phenotype as described.. Whether two plants are essentially genetically identical can be tested with assays known to a person skilled in the art, e.g. via fingerprint analysis, e.g. as described in Roldan-Ruiz, Theor. Appl. Genet., 2001,1138-1150. The expression pattern of proteins can be tested as described in the art e.g. via gel elec-trophoresis (1D, 2D, 3D), mass spectrometric analysis and other methods as described for example in www.waters.com. www.proteine.org.au, www.proteomesci.com. www.sdu.dk/Nat/CPA. The metabolome can be analysed by the skilled as described in the art, e.g. via HPLC, GC, OPLC, LC-MS, GC-MS, LC-MS-MS, and other methods as described e.g. in www.metabolic-explorer.com. www.ki.se/icsb2002/pdf/ICSBJ209.pdf, www.qenomics.ruq.nl/technologies.htm. Fiehn et al., Nature Biotech, 18 (2000), 1157, Raamsdonk et al., Nature Biotech, 19 (2991), 45-50, Buchholz, Anal. Biochem, 295 (2001) 129-137, Soga et al., Anal Chem. 74 (2002) 2233-2239.<br>
In order to increase the resistance to a pathogen the reference organism or the part<br>
thereof is susceptible to the infection with the pathogen, e.g. a plant pathogen, e.g. P.<br>
infestans.	.  .<br>
Preferably, the reference is a clone of that organism in which for example a relevant polynucleotide, e.g. the polynucleotideof the invention, or an activator, e.g. an activator of a relevant gene product mediating the activity, e.g. an activator increasing the expression of a relevant polynucleotide or a derivate of said polynucleotide, or an activator of a relevant polypeptide, e.g. of the polypeptide of the present invention, and/or a corresponding the relevant gene product encoding vector has been introduced. For example, a preferred reference in the method of the present invention is an organism or a part thereof, which is a clone of the organism or part thereof, e.g. a cell which has been transfected or transformed with the polynucleotide or vector of the invention.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
If the clone as described can not be identified it is state of the art to cleave out, to knock out or to switch off those elements which essentially mediate the relevant activity, e.g. mediating an increased Rpi-blb2 activity, e.g. mediating an increased expression, in the organism, e.g. in the plant. It is well known to skilled person, how to reduce or inhibit the activity of a relevant gene product, e.g. by reducing or inhibiting the expression of e.g. Rpi-blb2 . Such a clone can than be compared with an organism produced according to the method of the present invention, e.g. a P. infestans resistant, Rpi-Wb2 expressing genotype.<br>
The term "plant" as used herein refers to all genera and species of higher and lower plants of the Plant Kingdom. The term includes the mature plants, seed, shoots and seedlings and their derived parts, propagation material, plant organs, tissue, protoplasts, callus and other cultures, for example cell cultures, and any other type of plant cell grouping to give functional or structural units. "Mature plant" refers to a plant at any desired developmental stage beyond that of the seedling. Seedling refers to a young immature plant at an early developmental stage. "Plant" encompasses all annual and perennial monocotyledonous and dicotyledonous plants. Preferred within the scope of the invention are those plants which are employed as foodstuffs or feeding stuffs, for example monocotyledonous or dicotyledonous genera, in particular species, like the above-described ones, e.g. cereal species or members of the Solanaceae family, respectively, most preferably potato and tomato.<br>
As known to a person skilled in the art, the method of the present invention comprises further selecting those plants in which, as opposed or as compared to the reference piani, ihe resistance to at least ens said pathogen exists cr is increased.<br>
"Selection* with regard to plants in which - as opposed or as compared to the reference plant - resistance to at least one pathogen exists or is increased means all those methods which are suitable for recognizing an existing or increased resistance to pathogens. These may be symptoms of pathogen infection but may also comprise the herein described symptoms which relate to the quality of the plant, the quantity of the yield, the suitability for use as feeding stuff or foodstuff and the like.<br>
Accordingly, in one embodiment of the method of present invention the Rpi-blb2 protein is encoded by a polynucleotide comprising a nucleic acid molecule selected from the group consisting of:<br>
a) nucleic acid molecules encoding at least the mature form of the polypeptide depicted in SEQ ID NO: 2 or 4;<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
b)	nucleic acid molecules comprising the coding sequence as depicted in SEQ ID<br>
NO: 1 or 3, or 5 or 6 encoding at least the mature form of the polypeptide;<br>
c)	nucleic acid molecules the nucleotide sequence of which is degenerate as a result<br>
of the genetic code to a nucleotide sequence of (a) or (b);<br>
d)	nucleic acid molecules encoding a polypeptide derived from the polypeptide en<br>
coded by a polynucleotide of (a) to (c) by way of substitution, deletion and/or addi<br>
tion of one or several amino acids of the amino acid sequence of the polypeptide<br>
encoded by a polynucleotide of (a) to (c);<br>
e)	nucleic acid molecules encoding a polypeptide the sequence of which has an iden<br>
tity of 70% or more to the amino acid sequence of the polypeptide encoded by a<br>
nucleic acid molecule of (a) or (b);<br>
f)	nucleic acid molecules comprising a fragment or a epitope-bearing portion of a<br>
polypeptide encoded by a nucleic acid molecule of any one of (a) to (e);<br>
g)	nucleic acid molecules comprising a polynucleotide having a sequence of a nucleic<br>
acid molecule amplified from a nucleic acid library using the primers as listed in<br>
Tab 3b, in particular ARF1F and ARF1R;<br>
h)  nucleic acid molecules encoding a fragment beginning with amino acid: 1, 30, 50, 100, 200, 300, 500, or 1000 and stopping with amino acid 1276,1000, 500, 300, 200, 50, 30, or 1 of a polypeptide encoded by any one of (a) to (g);<br>
i)    nucleic acid molecules comprising at least 20 nucleotides of a polynucleotide of any one of (a) or (d);<br>
j)    nucleic acid molecules encoding a polypeptide being recognized by a monoclonal antibody that have been raised against a polypeptide encoded by a nucleic acid molecule of any one of (a) to (h);<br>
k)   nucleic acid molecules obtainable by screening an appropriate library under stringent conditions with a probe having the sequence of the nucleic acid molecule of any one of (a) to (j) or of a fragment thereof of at least 15, preferable 30, 60, 100 or more nucleotides; and<br>
I)   nucleic acid molecules the complementary strand of which hybridises under stringent conditions with a nucleic acid molecule of any one of (a) or (k); ■ or the complementary strand of any one of (a) to (I);<br>
or expressing a polypeptide encoded by a segment or linkage group 6 of Solanum bul-bocastanum which co-segregates with a marker selected from table 3A and which mediates resistance to pathogens, in particular to pathogens selected from the group consisting of phylum Oomycetes;<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
In one embodiment, the polynucleotide of the method of the invention does not consist of the sequence depicted in Seq. ID NO.: 7 and/or 9 and/or does not consist of the sequence of a nucleic acid molecule encoding a protein depicted in Seq. ID NO.: 8 and/or 10.<br>
In one embodiment, the polynucleotide of the method of the invention does not consist of the sequence of a nucleic acid molecule of Mil .1 or Mil .2 and/or of a nucleic acid molecule encoding a Mi1.1 or Mil.2 protein.<br>
Thus, in one embodiment, the polynucleotide of method of the present invention may not consist of the sequences shown in Rossi et al. 1998, PNAS USA 95:9750-9754, Milligan et al.f 1998. Plant Cell 10:1307-1319; and/or WO 9806750. A comparison of the sequences of Rpi-blb2, Mi1.1 and Mil .2 is shown in Figures 15 to 17.<br>
The term "linkage group" as used herein relates to two or more traits and/or loci and/or genes and/or markers that tend to be inherited together as a consequence of an association between said traits and/or loci and/or genes and/or markers. The closer together the traits and/or loci, and/or genes and/or markers are, the lower the probability that they will be separated during DNA repair or replication processes such as mitosis or meiosis in eukaryotes, and hence the greater the probability that they will be inherited together. There are as many linkage groups as there are homologous pairs of chromosomes.<br>
The tsrri "Snksgs group 6" relates to 2 linkage group of potato or tomato which is affiliated to chromosome 6, such affiliation established by identifying markers of known chromosomal position based on work published by Bernatzky and Tanksley (1986) and Tanksley.ef al. (1992). Linkage groups bear the same numbers as their respective chromosomes. In tomato, the chromosomes are numbered according to their length measured in pachytene. Such numbers have been applied by Barton (1950); chromosome 1 is the longest, chromosome 12, the shortest. In addition to length, such features as positions of centromere and amount and distribution of heterochromatin serve to identify each chromosome. Short arms are symbolized by HS", long ones by "L"; thus "1S" designates the short arm of chromosome 1; as e.g. in Barton, D.W. (1950) American Journal of Botany. 37,639-643, Bernatzky, R,and Tanksley, S.D. (1986) Genetics 112, 887-898, Tanksley, S.D., et al., (1992) Genetics 132, 1141-1160.<br>
The term "co-segregation" as used herein relates to the tendency for two or more closely linked traits and/or loci and/or genes and/or markers to be inherited together.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
For example, the more concrete region of chromosome 6 that co-segregates with Rpi-b!b2 is the short arm that, in tomato, bears the morphological marker Ml.<br>
Accordingly, in one embodiment the present invention relates to the method of the present invention, wherein the Rpi-blb2 protein is encoded by the polynucleotide of the present invention, e.g. encoded by a polynucleotide shown in Seq. ID. 1 or 3 or 5 or 6 or a fragment thereof.<br>
On basis of a BLASTX search the genes with the highest homology identified to the identified Rpi-blb2 sequences were the MH.1- and Mil .2-genes and proteins; see figures 15 to 17. Both genes have a high identity to the sequence depicted in Seq. ID NO.: 1 or 3 or 5 or 6 but do not confer resistance to the plant pathogen of the phylum Oomycetes. Therefore the activity of MH.1 and Mil.2 is an other activity as the activity of the polypeptid of the present invention. The sequence of MI1.1 and Mil .2 ORF and encoded proteins is herein shown in Seq. ID NO.: 7 to 10, Further, the application EP 401764.4 relates to the Mi-genes. The sequence of prior art Mi1.1- and Mil .2-genes is excluded from the polynucleotide of the present invention, in particular Seq. ID NO.: 7 and 9 are excluded; Also included may be polynucleotide sequences encoding the polypeptide of Seq. ID NO.: 8 or 10, Thus, in an embodiment also sequences encoding the Mill and Mit.2 protein are excluded. Proteins with a lower homology to the polypeptide encoded by the polynucleotide of the present invention are Hero Resistance proteins 1 and 2 (Genbank AccNo.: gi26190252 and gi'26190254), Tospovirus resistance proteins Af B, C, D and E [Genbank AccNos.:gi15418709, gi15418710, gi15418712, gi15418713, gi15418714]; R1 [Genbank AccNo.: gi17432423] and Prf [Genbank AccNo.: gi8547237] which sequences or encoded sequences are as well excluded from the sequences of the present invention.<br>
The terms "gene(s)", "polynucleotide", "nucleic acid sequence", "hucleotide sequence", or "nucleic acid molecule(s)H as used herein refer to a polymeric form of nucleotides of any length, either ribonucleotides or deoxyribonucleotides. This term refers only to the primary structure of the molecule.<br>
Thus, this term includes double- and single-stranded DNA, and RNA. It also includes known types of modifications, for example, methylation, "caps" substitution of one or more of the naturally occurring nucleotides with an analogue. Preferably, the DNA sequence of the invention comprises a coding sequence encoding the herein defined polypeptide.<br>
A "coding sequence" is a nucleotide sequence which is transcribed into mRNA and/or translated into a polypeptide when placed under the control of appropriate regulatory<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
sequences. The boundaries of the coding sequence are determined by a translation start codon at the 5'-terminus and a translation stop codon at the 3'-terminus. A coding sequence can include, but is not limited to mRNA, cDNA, recombinant nucleotide sequences or genomic DNA, while introns may be present as well under certain circumstances.<br>
By "hybridising" it is meant that such nucleic acid molecules hybridise under conventional hybridisation conditions, preferably under stringent conditions such as described by, e.g., Sambrook (Molecular Cloning; A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989)). An example of one such stringent hybridisation condition is hybridisation at 4XSSC at 65°C, followed by a washing in 0.1XSSC at 65°C for one hour. Alternatively, an exemplary stringent hybridisation condition is in 50 % formamide, 4XSSC at 42°C. Further, the conditions during the wash step can be selected from the range of conditions delimited by low-stringency conditions (approximately 2X SSC at 50°C) and high-stringency conditions (approximately 0.2X SSC at 50°C, preferably at 65°C) (20X SSC: 0.3M sodium citrate, 3M NaCI, pH 7.0). In addition, the temperature during the wash step can be raised from low-stringency conditions at room temperature, approximately 22°C, to higher-stringency conditions at approximately 65°C. Both of the parameters salt concentration and temperature can be varied simultaneously, or else one of the two parameters can be kept constant while only the other is varied. Denaturants, for example formamide or SDS, may also be employed during the hybridisation. In the presence of 50% formamide, hybridisation is preferably effected at 42°C. Some further examples of conditions for hybridisation and wash step are shown herein below:<br>
(1)     Hybridisation conditions can be selected, for example, from the following conditions:<br>
a)	4X SSC at 65°C,<br>
b)	6X SSC at 45°C,<br>
c)	6X SSC, 100 mg/ml denatured fragmented fish sperm DNA at 68°C,<br>
d)	6X SSC, 0.5% SDS, 100 mg/ml denatured salmon sperm DNA at 68°C,<br>
e)	6X SSC, 0.5% SDS, 100 mg/ml denatured fragmented salmon sperm DNA, 50%<br>
formamide at 42°C,<br>
f)	50% formamide, 4X SSC at 42°C,<br>
g)	50% (vol/vol) formamide, 0.1% bovine serum albumin, 0.1% Ficoll, 0.1% polyvi-<br>
nylpyrrolidone, 50 mM sodium phosphate buffer pH 6.5, 750 mM NaCI, 75 mM<br>
sodium citrate at 42°C,<br>
h)      2X or 4X SSC at 50°C (low-stringency condition), or<br>
i)       30 to 40% formamide, 2X or 4X SSC at 42°C (low-stringency condition).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
(2)    Wash steps can be selected, for example, from the following conditions:<br>
a)	0.015 M NaCl/0.0015 M sodium citrate/0.1% SDS at 50°C.<br>
b)	0.1XSSCat65°C.<br>
c)	0.1X SSC, 0.5 % SDS at 68°C.<br>
d)	0.1X SSC, 0.5% SDS, 50% formamide at 42°C.<br>
e)	0.2X SSC, 0.1% SDS at42°C.<br>
f)	2X SSC at 65°C (low-stringency condition).<br>
In one embodiment of the present invention, the polynucleotide of the invention comprises a polynucleotide which hybridises to a nucleic acid molecule comprising or consisting of a nucleic acid molecule having the sequence shown in Seq ID No. 1 or 3 or 5 or 6 or a fragment thereof. The fragment comprises or consists preferably of 15, 20, 30, 40, 70, 100, 300, 500, 700, 1000 or more residues of Seq ID No. 1 or 3 or 5 or 6.<br>
In a preferred embodiment, the poiynucleotide of the invention comprises a polynucleotide which hybridises under "stringent" hybridisation conditions with a nucleic acid molecule comprising or consisting of a nucleic acid molecule having the sequence shown in Seq ID No. 1 or 3 or 5 or 6 or a fragment thereof.<br>
The term "under stringent hybridisation conditions" as used herein refers to any of the herein mentioned stringent hybridisation conditions. In a further embodiment, the term "under stringent hybridisation conditions" refers to the hybridisation conditions mentioned in the examples or used in Sambrook (Molecular Cloning; A Laboratory Manual, 2nd Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989).<br>
In one preferred embodiment, the term "under stringent hybridisation conditions" as used herein refers to all of the herein mentioned stringent hybridisation conditions, meaning that a polynucleotide hybridises under all mentioned stringent conditions.<br>
Rpi-blb2 derived from other organisms, may be encoded by other DNA sequences which hybridise to the sequences shown in Seq ID No. 1 or 3 or 5 or 6 under relaxed hybridisation conditions and which code on expression for peptides having the activity of Rpi-blb2. Further, some applications have to be performed at low stringency hybridisation conditions, without any consequences for the specificity of the hybridisation. For example, a Southern blot analysis of total DNA could be probed with a polynucleotide of the present invention and washed at low stringency (55°C in 2xSSPE, 0,1% SDS). The hybridisation analysis could reveal a simple pattern of only genes encoding Rpi-blb2. A further example of such low-stringent hybridisation conditions are 4XSSC at<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
immunogens (i.e., substances capable of eliciting an immune response) are antigens; however, some antigen, such as haptens, are not immunogens but may be made im-munogenic by coupling to a carrier molecule. The term "antigen" includes references to a substance to which an antibody can be generated and/or to which the antibody is specifically immunoreactive. In one embodiment the present invention relates to a epi-tope of Rpi-blb2.<br>
The term 'one or several amino acids" relates to at least one amino acid but not more than that number of amino acids which would result in a homology of below 70% identity. Preferably, the identity is more than 75% or 80%, more preferred are 85%, 90% or 95%, even more preferred are 96%, 97%, 98%, or 99% identity.<br>
The terms "polynucleotide" and "nucleic acid molecule* also relate to "isolated" polynu-cleotides or nucleic acids molecules. An "isolated" nucleic acid molecule is one which is separated from other nucleic acid molecules which are present in the natural source of the nucleic acid. Preferably, an "isolated" nucleic acid is free of sequences which naturally flank the nucleic acid (i.e., sequences located at the 5' and 3' ends of the nucleic acid) in the genomic DNA of the organism from which the nucleic acid is derived. For example, in various embodiments, the polynucleotide of the present invention can contain less than about 5 kb, 4kb, 3kb, 2kb, 1 kb, 0.5 kb or 0.1 kb or less of nucleotide sequences which naturally flank the nucleic acid molecule in genomic DNA of the cell from which the nucleic acid is derived. Moreover, the polynucleotides of the present invention, in particular an "isolated" nucleic acid molecule, such as a cDNA molecule, can be substantially free of other cellular material, or culture medium when produced by recombinant techniques, or chemical precursors or other chemicals when chemically synthesized.<br>
Further, the polynucleotide of the invention comprises a nucleic acid molecule which is a complement of one of the nucleotide sequences of above mentioned polynucleotides or a portion thereof. A nucleic acid molecule which is complementary to one of the nucleotide sequences shown in SEQ ID No;1 or 3 or 5 or 6 is one which is sufficiently complementary to one of the nucleotide sequences shown in SEQ ID No: 1 or 3 or 5 or 6 such that it can hybridise to one of the nucleotide sequences shown in SEQ ID No; 1 or 3 or 5 or 6, thereby forming a stable duplex.<br>
The polynucleotide of the invention comprises a nucleotide sequence which is at least about 70%, preferably at least about 75%, more preferably at least about 80%, 90%, or 95%, and even more preferably at least about 96%, 97%, 98%, 99% or more homologous to a nucleotide sequence shown in SEQ ID No: 1 or 3 or 5 or 6, or a portion thereof. The polynucleotide of the invention comprises a nucleotide sequence which<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
hybridises, preferably hybridises under stringent conditions as defined herein, to one of the nucleotide sequences shown in SEQ ID No: 1 or 3 or 5 or 6, or a portion thereof.<br>
Moreover, the polynucleotide of the invention can comprise only a portion of the coding region of one of the sequences in SEQ ID No: 1 or 3 or 5 or 6, for example a fragment which can be used as a probe or primer or a fragment encoding a biologically active portion of the Rpi-blb2 protein coding gene. The nucleotide sequences determined from the cloning of the present Rpi-blb2 protein encoding gene allows for the genera-tion of probes and primers designed for use in identifying and/or cloning its homo-bgues in other cell types and organisms. The probe/primer typically comprises sub-stantially purified oligonucleotides. The oligonucleotide typically comprises a region of nucleotide sequence that hybridises under stringent conditions to at least about 12, 15 preferably about 20 or 25, more preferably about 40, 50 or 75 consecutive nucleotides of a sense strand of one of the sequences set forth, e.g., in SEQ ID No. No: 1 or 3 or 5 or 6, an anti-sense sequence of one of the sequences, e.g., set forth in SEQ ID No.: 1 or 3 or 5 or 6, or naturally occurring mutants thereof. Primers based on a nucleotide of invention can be used in PCR reactions to clone Rpi-blb2 homologues, e.g. as the primers described in the examples of the present invention, e.g. as shown in tab 3a or 3b, preferably the primers ARF1F and ARF1R are used. A PCR with the primers univ24R and univ14L will result in a fragment of Rpi-blb2 which can be used as described herein. Said primer sets are interchangeable. The person skilled in the art knows to combine said primers to result in the desired product, e.g. in a full length clone or a partial sequence. Probes based on the Rpi-blb2 nucleotide sequences can be used to detect transcripts or genomic sequences encoding the same or homologous proteins. The probs can further comprise a label group attached thereto, e.g. the label group can be a radioisotope, a fluorescent compound, an enzyme, or an enzyme co-factor. Such probes can be used as a part of a genomic marker test kit for identifying cells which express a Rpi-blb2, such as by measuring a level of a Rpi-blb2-encoding nucleic acid molecule in a sample of cells, e.g., detecting Rpi-blb2 mRNA levels or determining whether a genomic Rpi-blb2 gene has been mutated or deleted.<br>
The polynucleotide of the invention encodes a polypeptide or portion thereof which includes an amino acid sequence which is sufficiently homologous to the amino acid sequence of SEQ ID No: 2 or 4 such that the protein or portion thereof maintains the ability to participate in resistance to pathogens, in particular a Rpi-blb2 protein activity as described in the examples in plants. As used herein, the language "sufficiently homologous" refers to proteins or portions thereof which have amino acid sequences which include a minimum number of identical or equivalent (e.g., an amino acid residue which has a similar side chain as an amino acid residue in one of the sequences of the polypeptide of the present invention), amino acid residues to an amino acid sequence<br><br>
WO 2005/014631	PCTVEP2004/008683<br>
of Seq. ID No.: 2 or 4 such that the protein or portion thereof is able to participate in the resistance of plants to said pathogens. Examples of a Rpi-blb2 protein activity are described herein. Thus, the function of a Rpi-blb2 protein contributes either directly or indirectly to the resistance to plant pathogens, preferably to the pathogens mentioned herein, more preferred to P. infestans.<br>
The protein is at least about 70%, preferably at least about 75%, and more preferably at least about 80%, 90%, 95%, and most preferably at least about 96%, 97%, 98%, 99% or more homologous to an entire amino acid sequence of SEQ ID No; 2 or 4.<br>
Portions of proteins encoded by the polynucieotide of the invention are preferably biologically active.<br>
As mentioned hereinr the term "biologically active portion" is intended to include a portion, e.g., a domain/motif, that confers resistance to an oomycete plant pathogen and/or Bemisia tabaci and/or aphids or has an immunological activity such that it binds to an antibody binding specifically to Rpi-blb2 protein or it has an activity as set forth in the Examples or as described herein.<br>
Additional nucleic acid fragments encoding biologically active portions of the polypep-tide of the present invention can be prepared by isolating a portion of one of the sequences in SEQ ID No: 1 or 3 or 5 or 6, expressing the encoded portion of the Rpi-blb2 protein or peptide (e.g., by recombinant expression in vitro) and assessing the activity of the encoded portion of the protein.<br>
The invention further encompasses polynucleotides that differ from one of the nucleo-tide sequences shown in SEQ ID No: 1 or 3 or 5 or 6 (and portions thereof) due to degeneracy of the genetic code and thus encode a Rpi-blb2 polypeptide as that encoded by the sequences shown in SEQ ID No: 2 or 4. Further the polynucieotide of the invention has a nucleotide sequence encoding a protein having an amino acid sequence shown in SEQ ID No: 2 or 4. In a still further embodiment, the polynucieotide of the invention encodes a full length protein which is substantially homologous to an amino acid sequence of SEQ ID No: 2 or 4.<br>
In addition, it will be appreciated by those skilled in the art that DNA sequence polymorphisms that lead to changes in the amino acid sequences may exist within a population (e.g., the S. bulbocastanum population). Such genetic polymorphism in the Rpi-blb2 gene may exist among individuals within a population due to natural variation.<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
As used herein, the terms "gene" and "recombinant gene" refer to nucleic acid molecules comprising an open reading frame encoding a Rpi-blb2, preferably a S. bulbocastanum Rpi-blb2. Such natural variations can typically result in 1-5% variance in the nucleotide sequence of the Rpi-blb2 gene. Any and all such nucleotide variations and resulting amino acid polymorphisms in Rpi-blb2 that are the result of natural variation and that do not alter the functional activity of Rpi-blb2 are intended to be within the scope of the invention.<br>
Polynucleotkjes corresponding to natural variants and non-S. bulbocastanum homo-logues of the Rpi-blb2 cDNA of the invention can be isolated based on their homology to S. bulbocastanum Rpi-blb2 polynucleotides disclosed herein using the polynucleo-tide of the invention, or a portion thereof, as a hybridisation probe* according to standard hybridisation techniques under stringent hybridisation conditions. Accordingly, in another embodiment, a polynucleotide of the invention is at least 20 nucleotides in length. Preferably it hybridises under stringent conditions to the nucleic acid molecule comprising a nucleotide sequence of the polynucleotide of the present invention, e.g. SEQ ID No: 1 or 3 or 5 or 6. In other embodiments, the nucleic acid is at least 20, 30, 50,100, 250 or more nucleotides in length. The term "hybridises under stringent conditions" is defined above and is intended to describe conditions for hybridisation and washing under which nucleotide sequences at least 65% identical to each other typically remain hybridised to each other. Preferably, the conditions are such that sequences at least about 70%, more preferably at least about 75% or 80%, and even more preferably at least about 85%, 90% or 95% or more identical to each other typically remain hybridised to each other. Preferably, polynucleotide of the invention that hybridises urtder stringent conditions to a sequence of SEQ ID NO: 1 or 3 or 5 or 6 corresponds to a naturally-occurring nucleic acid molecule.<br>
As used herein, a "naturally-occurring" nucleic acid molecule refers to an RNA or DNA molecule having a nucleotide sequence that occurs in nature (e.g., encodes a natural protein). Preferably, the polynucleotide encodes a natural S. bulbocastanum Rpi-blb2.<br>
In addition to naturally-occurring variants of the Rpi-blb2 sequence that may exist in the population, the skilled artisan will further appreciate that changes can be introduced by mutation into a nucleotide sequence of the polynucleotide encoding Rpi-blb2, thereby leading to changes in the amino acid sequence of the encoded Rpi-blb2, without altering the functional ability of the Rpi-blb2. For example, nucleotide substitutions leading to amino acid substitutions at "non-essential" amino acid residues can be made in a sequence of the polynucleotide encoding Rpi-blb2, e.g. SEQ ID No: 1 or 3 or 5 or 6. A "non-essential" amino acid residue is a residue that can be altered from the wild-type sequence of the Rpi-blb2 protein without altering the activity of said Rpi-blb2 protein,<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
whereas an "essential" amino acid residue is required for Rpi-blb2 protein activity. Other amino acid residues, however, (e.g., those that are not conserved or only semi-conserved in the domain having Rpi-blb2 activity) may not be essential for activity and thus are likely to be amenable to alteration without altering Rpi-blb2 activity.<br>
Accordingly, a person skilled in the art knows that the codon usage between organisms can differ. Therefore he will adapt the codon usage in the polynucleotide of the present invention to the usage of the organism in which the polynucleotide or polypeptide is expressed.<br>
Accordingly, the invention relates to polynucleotides encoding Rpi-blb2 that contain changes in amino acid residues that are not essential for Rpnblb2 activity. Such Rpi-blb2s differ in amino acid sequence from a sequence contained in SEQ ID No: 2 or 4 yet retain the Rpi-blb2 activity described herein. The polynucleotide can comprise a nucleotide sequence encoding a polypeptide, wherein the polypeptide comprises an amino acid sequence at least about 70% identical to an amino acid sequence of SEQ ID No: 2 or 4 and is capable of participation in the resistance to a plant pathogen. Preferably, the protein encoded by the nucleic acid molecule is at least about 70% identical to the sequence in SEQ ID No: 2 or 4t more preferably at least about 75% identical to one of the sequences in SEQ ID No: 2 or 4, even more preferably at least about 80%, 90%, 95% homologous to the sequence in SEQ ID No: 2 or 4, and most preferably at least about 96%, 97%, 98%, or 99% identical to the sequence in SEQ ID No: 2 or 4.<br>
To determine the percent homology of two amino acid sequences (e.g., one of the sequences of Seq. ID No.: 2 or 4 and a mutant form thereof) or of two nucleic acids, the sequences are aligned for optimal comparison purposes (e.g., gaps can be introduced in the sequence of one protein or nucleic acid for optimal alignment with the other protein or nucleic acid). The amino acid residues or nucleotides at corresponding amino acid positions or nucleotide positions are then compared. When a position in one sequence (e.g., one of the sequences of SEQ ID No: 2 or 4) is occupied by the same amino acid residue or nucleotide as the corresponding position in the other sequence (e.g., a mutant form of the sequence selected), then the molecules are homologous at that position (i.e., as used herein amino acid or nucleic acid "homology" is equivalent to amino acid or nucleic acid "identity"). The percent homology between the two sequences is a function of the number of identical positions shared by the sequences (i.e., % homology = numbers of identical positions/total numbers of positions x 100).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
preferably at least 80%, by preference at least 90%f especially preferably at least 95%, very especially preferably at least 98%, homology with one of the polypeptides as shown in SEQ ID NO: 2 or 4 according to the invention and are distinguished by essentially the same properties as the polypeptide as shown in SEQ ID NO: 2 or 4.<br>
Functional equivalents derived from the nucleic acid sequence as shown in SEQ ID NO: 1 or 3 or 5 or 6 according to the invention by substitution, insertion or deletion have at least 70%, preferably at least 80%, by preference at least 90%, especially preferably at least 95%, very especially preferably at least 98%, homology with one of the polypeptides as shown in SEQ ID NO: 2 or 4 according to the invention and encode polypeptides having essentially the same properties as the polypeptide as shown in SEQ ID NO: 2 or 4.<br>
"Essentially the same properties" of a functional equivalent is above all understood as meaning conferring a pathogen-resistant phenotype or conferring or increasing the resistance to at least one pathogen while increasing the amount of protein, activity or function of said functional Rpi-blb2 equivalent in a plant or in a tissue, part or cells of the same. The sporulation and lesion phenotype after infection in combination with said increase of the amount of protein, activity or function of the functional equivalent is furthermore understood as an essential property.<br>
A nucleic acid molecule encoding a Rpi-blb2 homologous to a protein sequence of SEQ ID No: 2 or 4 can be created by introducing one or more nucleotide substitutions, additions or deletions into a nucleotide sequence of the polynucleotide of the present invention, in particular of SEQ ID No: 1 or 3 or 5 or 6 such that one or more amino acid substitutions, additions or deletions are introduced into the encoded protein. Mutations can be introduced into the sequences of, e.g., SEQ ID No: 1 or 3 or 5 or 6 by standard techniques, such as site-directed mutagenesis and PCR-mediated mutagenesis. Preferably, conservative amino acid substitutions are made at one or more predicted non-essential amino acid residues. A "conservative amino acid substitution" is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), non-polar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, me-thionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan, histidine). Thus, a predicted nonessential amino acid residue in a Rpi-blb2 is preferably replaced with another amino acid residue from the same family. Alternatively, in another embodi-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
merit, mutations can be introduced randomly along all or part of a Rpf-blb2 coding sequence, such as by saturation mutagenesis, and the resultant mutants can be screened for a Rpi-blb2 activity described herein to identify mutants that retain Rpi»blb2 activity. Following mutagenesis of one of the sequences of SEQ ID No: 1 or 3 or 5 or 6, the encoded protein can be expressed recombinantly and the activity of the protein can be determined using, for example, assays described herein (see Examples).<br>
In one embodiment in the method of present invention the activity of Rpi-blb2 protein and of a further resistance protein is increased.<br>
It is expected, that under field conditions the presence of more than one resistance gene is beneficial, in particular genes conferring resistance to the same pathogen. In case a pathogen isolate, e.g. a P. infestans race, is present that is able to overcome resistance of one of the R-genes, the other one or more R-gene(s) is/are still functional making ft impossible to infect the plant. The present of two undefeated R-genes strongly reduces the chance that a pathogen, in particular a P. infestans race, is able to mutate into a race that can overcome two or more R-genes.<br>
In the following "resistance polypeptide' or "resistance protein" relates to a polypeptide which (increased) activity will confer resistance to a susceptible genotype ("wild type* or "reference'). Accordingly, Rpi-blb2 is a resistance protein as well as e.g. Rpi-blb (or RB or Sbu1). A "further resistance protein" relates to an other resistance protein than the protein of the present invention, whereas the term "resistance protein" comprises the poJypeptiu of the present invention as we!! as one or mere further resistance pro-tein(s). It is further understood, that the term "and a further resistance protein" relates to one or more further resistance proteins. Thus, the activity of one or more resistance proteins can be increased. Further resistance proteins are described below. However, generally any other known resistance protein can be co-expressed with the polypeptid of the present invention or its activity can be increased by any of the methods described herein for Rpi-b(b2.<br>
In a preferred embodiment, the further resistance protein comprises a LRR domain and a P-loop.<br>
The cloning and molecular characterisation of over 30 plant disease resistance (R) genes conferring resistance to bacteria, fungi, oomycetes, viruses, nematodes, or insects has allowed their classification in structural classes regardless of pathogen specificity (reviewed in Dangl and Jones, 2001). The most abundant class of characterised R genes, comprising about 0.5 percent of the genes predicted in the Arabidopsts<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
nome, is predicted to encode intracellular proteins that carry leucine-rich repeat (LRR) and nucleotide-binding site (NBS) domains, motifs also found in other receptor and signal transduction proteins. NBS-LRR R proteins differ primarily at the N-terminus that either exhibits sequence similarity to the Drosophila Toll protein and the mammalian interleukin-1 receptor domain (TIR-NBS-LRR), or code for a coiled-coils structure (CC-NBS-LRR), sometimes in the form of a leucine zipper (LZ-NBS-LRR). Although maybe membrane associated, NBS-LRR proteins are predicted to be cytoplasmic. In contrast, two other classes of R proteins that carry LRRs are predicted to span the cell membrane, with an extracellular LRR domain: the LRR-transmembrane (LRR-TM) Cf proteins and the LRR-TM-kinase Xa21 protein. Characterised R proteins that lack LRRs are the Pto gene from tomato, the Hs1pro-1 gene from beet, the mlo gene from barley, the Rpw8 genes from Arabidopsis and the Rpg1 gene from barley.<br>
According to the gene-for-gene hypothesis, disease resistance follows perception by plant R proteins of pathogen effector molecules with avirulence (Avr) function, thereby initiating through some kind of elicitor recognition complex, signal transduction pathways leading to a hypersensitive response (HR). In common with other receptors it is generally considered that NBS-LRR R proteins have a modular structure with separate recognition and signalling domains, whereby the LRR is the candidate recognition domain and the IM-terminal region including the NBS, the major signalling domain. Functional analysis of recombinant R proteins indicates that recognition specificity indeed resides in the LRR. Moreover, the LRR is the most variable region in closely related NBS-LRR proteins and is under selection to diverge. However, evidence is accumulating that LRRs also contribute to signalling through negative regulation involving putative intramolecular interactions. Currently, five R genes have been cloned from potato, including two R genes conferring resistance to late blight, and all belong to the CC/LZ-NBS-LRR class of plant R genes. While the S. demissum derived R1 gene confers race specific.resistance to late blight, the recently cloned S. bulbocastanum derived gene Rpi-blb (or RB or Sbu1) confers full resistance to a range of P. infestans isolates carrying multiple virulence factors and race-specificity has not yet been demonstrated. Furthermore, as described before, progeny plants of somatic hybrids containing Rpi-blb were unaffected by late blight on field experiments in Mexico, where nearly every race of the fungus is found. Through complementation of the susceptible phenotype in cultivated potato and tomato the potential of interspecific transfer of broad-spectrum late blight resistance to cultivated Solanaceae from sexually incompatible host species by transformation with single cloned R genes was demonstrated. US 6,127,607 describes resistance proteins with LRR domains and P-loops. The content of US 6,127,607 is herewith incorporated by reference. In particular columns 6 to 8 and col. 11 describe LRR domains and P-loops. Furthermore Song, 2003, PNAS 100 (16), 9128-9133 shows a comparison of Rpi-blb LRR motifs in Fig. 4 and gives on pages 9132 an over-<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
view about LRR domains. The domains of the polypeptid of the present invention are shown in Rg. 14 as well as in Fig, 15.<br>
Preferably the activity of one or more resistance protein(s) selected from the group consisting of Rpnblb (synonym RB or Sbu1), Rpi-ABPT1, Rpi-blb3, Rpi-mcd, R1, R-ber (synonym R12), Rpli, R2, R3a, R3b, R4, R5, R6, R7, R8, R9, R10, R11( Ph-1, Ptv2 and Ph-3 is increased. Preferred is that in addition to Rpi-blb2 at least also the Rpwblb activity is increased.<br>
In one embodiment of the present invention, the expression of an, e.g. transgenic, Rpi-blb2 protein is increased and further a transgenic resistance gene's expression is increased. The resistance protein coexpressed with the Rpi-blb2 (or RB or Sbu1) is preferably one of the resistance proteins mentioned herein, in particular Rpi-blb, Rpi-ABPT1, Rpf-blb3, R1, RpI1, R-ber, Rpi-mcd, R2, R3a, R3b, R6, R7, Ph-1, Ph-2 or Ph-3 but can also be one of the others resistance to plant pathogens conferring proteins known to a person skilled in the art.<br>
As mentioned, the term "increased expression" according to this invention also includes a de novo-Expression of a polynucleotide or polypeptide.<br>
Most preferred is an increase of resistance via coexpression of the polypeptid of the present invention together with Rpi-blb. Rpi-blb and Rpi-blb2 provide both full resistance in detached leaf assays to P. infestans isolates as described in the examples, and in Song 2003t PNAS 100 (16), 9128.<br>
Said resistance conferring genes are for example described in<br>
RB or Sbu1 (synonym of Rpi-blb): AY336128 [gi: 32693280], (Song et al.j 2003). BAC clones 177 013 and CB3A14 comprising the Rpi-blb gene have been deposited in GenBank with accession nos AY303171 and AY303170.<br>
R1: AF447489 [gi; 9117432422], (Ballvora et al., 2002)<br>
Rpi1: Kuhl, J.C., Hanneman, R.E., and Havey, MJ., (2201) Characterization and mapping of Rpi1, a late blight resistance locus from diploid (1EBN) Mexican Solanum pin-natisectum. Molecular genet. Genomics 265: 977-985.<br>
Rrber: Ewing, E.E., Simko, K, Smart, CD., Bonierbale, M.W., Mizubuti, E.S.G., May, G.D., and Fry, W.E., (2000) Genetic mapping from field tests of qualitative and quanti-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
tative resistance to Phytophthora infestans in a population derived from Solanum tube-rosum and Solanum berthaultii. Molecular breeding 6:25-36.<br>
R2: Li, X., vanEck, H.J., vanderVoort, J.NAM., Huigen, D.J., Stam, P., and Jacobsen, E. (1998) Autotetraploids and genetic mapping using common AFLP markers: the R2 allele conferring resistance to Phytophthora infestans mapped on potato chromosome 4. Theoretical and Applied Genetics 96 (8): 1121-112.<br>
R3f R6, R7: Elkharbotly, A., Palominosanchez, C, Salamini, F., Jacobsen, E., and Gebhardt, C. (1996) R6 and R7 alleles of potato conferring race-specific resistance to Phytophthora infestans (Mont) de Bary identified genetic loci clustering with the R3 locus on chromosome XI. Theoretical and Applied.Genetics 92 (7): 880-884.<br>
Ph-1: Bonde and Murphy (1952) Main Agric. Exp. Stn. Bull. No 497<br>
Ph-2: Moreau, P., Thoquet, P., Olivier, J.( Laterrot, HM and Grimsley, N.H. (1998) Genetic mapping of Ph-2, a single locus controlling partial resistance to Phytophthora infestans in tomato. Molecular Plant Microbe Interactions 11 (4): 259-269.<br>
Ph-3: Chunwongse, J., Chunwongse, C, Black, L, and Hanson, P. (2002) Molecular mapping of the Ph-3 gene for late blight resistance in tomato. Journal of Horticultural Science &amp; Biotechnology 77 (3): 281-286.<br>
Rpi-blb3, Rpi-ABPT1 and Rpkmcd: Park, T.H., Van der Vossen, E., Vleeshouwers, V.GAA., Tan, A., Visser, R.G.F. and Van Eck, H.J. 2004. Major resistance genes for tuber and leaf resistance to Phytophthora infestans in potato: An outline of a PhD project. Crop Functional Genomics 2004, July 2004, Jeju, Korea, page 93.<br>
R3a and R3b: Huang, S., Vleeshouwers, V.GAA., Werij, J.S., Hutten, R.C.B., Van Eckt H.J.f Visser, R.G.F, and Jacobsen, E. (2004). The R3 resistance to Phytophthora infestans in potato is conferred by two closely linked R genes with distinct specificities. MPMI 17(4), 428-435.<br>
In one embodiment, the activity of the Rpi-blb2 is increased according to the present invention, e.g. the polynucleotide of the invention's expression is increased and the expression of at least one nucleic acid molecule is increased encoding Rpi-blb, Rpi-ABPT1, Rpi-blb3, Rpi-mcd R1, R-ber, Rpi1, R2, R3a, R3b, R6, R7, Ph-1, Ph-2 and/or Ph-3 whereby the nucleic acid molecule is selected from the group consisting of:<br>
a)  nucleic acid molecule encoding at least a mature form of at least<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
a Rpnblb (or RB- or Sbu1-) polypeptide, preferably as encoded by the sequence shown in GenBank Accession no.: AY336128 [gi: 32693280];<br>
a R1 polypeptide, preferably as encoded by the sequence shown in GenBank Accession no.: AF447489 [gi 9117432422];<br>
a Rp"hblb3, Rpi-ABPT1 and/or Rpi-mcd polypeptide, preferably encoded by the sequence shown in or derivable by the information given in Park, T.H., Van der Vossen, E, VJeeshouwers, V.GAA., Tan, A., Visser, R.G.F. and Van Eck, HJ. 2004. Major resistance genes for tuber and leaf resistance to Phytophthora infestans in potato: An outline of a PhD project Crop Functional Genomics 2004, July 2004, Jeju, Korea, page 93;<br>
a R3a and/or R3b polypeptide, preferably encoded by the sequence shown in or derivable by the information given in Huang, S., Vleeshouwers, V.GAA., Werij, J.S., Hutten, R.C.B., Van Eck, H J., Visser, R.G.F, and Jacobsen, E. (2004). The R3 resistance to Phytophthora infestans in potato is conferred by two closely linked R genes with distinct specificities. MPMI 17 (4), 428-435 and/or<br>
a pathogen, preferably P. infestans, resistance conferring protein mapped and characterized as described, e.g. as for<br>
for Rpi1 in Kuhl, J.C., Hanneman, R.E., and Havey, MJ., (2001) Characterization and mapping of Rpi1, a late blight resistance locus from diploid (1EBN) Mexican Solarium pinnatisectUTTi. Molecular genet. Gencrnics 265: 977-985;<br>
for R-berin Ewing, E.E., Simko, I., Smart, CD., Bonierbale, M.W., Mizubuti, E.S.G., May, G.D., and Fry, W.E., (2000) Genetic mapping from field tests of qualitative and quantitative resistance to Phytophthora infestans in a population derived from Solanum tuberosum and Solanum berthaultii. Molecular breeding 6:25-36;<br>
for R2 in Li, X., vanEck, H.J., vanderVoort, J.NAM., Huigen, D.J., Start, P., and Jacobsen, E. (1998) Autotetraploids and genetic mapping using common AFLP markers: the R2 allele conferring resistance to Phytophthora infestans mapped on potato chromosome 4. Theoretical and Applied Genetics 96 (8): 1121-1128;<br>
for R3, R6, R7 in Elkharbotly, A., Palominosanchez, C, Salamini, F., Jacobsen, EM and Gebhardt, C. (1996) R6 and R7 alleles of potato conferring race-specific resistance to Phytophthora infestans (Mont) de Bary identified genetic loci clustering<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
with the R3 locus on chromosome XI Theoretical and Applied.Genetics 92 (7): 880-884;<br>
for Ph-1 in Bonde and Murphy (1952) Main Agric. Exp. Stn. Bull. No 497; or<br>
for Ph-2 in Moreau, P.f Thoquet, P., Olivier, J., Laterrot, HM and Grimsley, N.H. (1998) Genetic mapping of Ph-2, a single locus controlling partial resistance to Phytophthora infestans in tomato. Molecular Plant Microbe Interactions 11 (4); 259-269; and/or<br>
for Ph-3 in Chunwongse, J., Chunwongse, C, Black, L, and Hanson, P. (2002) Molecular mapping of the Ph-3 gene for late blight resistance in tomato. Journal of Horticultural Science &amp; Biotechnology 77 (3): 281-286;<br>
or a pathogen resistance conferring polypeptide, preferably P. infestans resistance conferring polypeptide derivable from said publications;<br>
&gt;)  nucleic acid molecule the nucleotide sequence of which is degenerate as a result of the genetic code to a nucleotide sequence of (a);<br>
i)  nucleic acid molecule encoding a polypeptide derived from the polypeptide encoded by a polynucleotide of (a) or (b) by way of substitution, deletion and/or addition of one or several amino acids of the amino acid sequence of the polypeptide encoded by a polynucleotide of (a) or (b);<br>
1)  nucleic acid molecule encoding a polypeptide the sequence of which has an identity of 70% or more to the amino acid sequence of the polypeptide encoded by a nucleic acid molecule of (a);<br>
s)  nucleic acid molecules comprising a fragment or a epitope-bearing portion of a polypeptide encoded by a nucleic acid molecule of any one of (a) to (d);<br>
)   nucleic acid molecule encoding a fragment beginning with amino acid: 1, 30, 50, 100, 200, 500 or 1000, and stopping with amino acid 1267, 1000, 500, 300, 200, 50, 30, or 1 of a polypeptide encoded by any one of (a) to (e) and with one of said activities;<br>
l)  nucleic acid molecule comprising at least 20 nudeotides of a polynucleotide of any one of (a) or (b);<br>
i)  nucleic acid molecule encoding a polypeptide being recognized by a monoclonal antibody that have been raised against a polypeptide encoded by a nucleic acid molecule of any one of (a) to (f);<br>
i    nucleic acid molecule obtainable by screening an appropriate library under stringent conditions with a probe having the sequence of the nucleic acid molecule of<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
any one of (a) to (h) or of a fragment tnereoT OT at least 20, preferable 30 or more nucleotides; and<br>
j)   nucleic acid molecule the complementary strand of which hybridises under stringent conditions with a nucleic acid molecule of any one of (a) or (0;<br>
or the complementary strand of any one of (a) to (I)<br>
Accordingly, the method of present invention confers resistance of one of said plants, plant tissue or plant cell of the present invention to a plant pathogen of a phylum Oo mycetes, preferably to a pathogen of the order Pythiales or Peronosperales, more preferred to the family Pythiaceae or Peronosporaceae, more preferred of the genus Phy-tophthora or Bremia or Peronospera or Plasmopara, most preferred wherein the pathogen is of the species Phytophthora parasitica var. nicotianae (causing, amongst others, black shank in tobacco), Phytophthora sojae (causing Phytophthora root rot in soybean), Phytophthora capsici (causing rots in pepper and cucurbits and tomato), Phytophthora erythroseptica (causing Pink rot in potato), Plasmopara viticola (causing grapevine downy mildew), Bremia lactuca (causing downy mildew in lettuce) or Pero-nospora tabaci (causing blue mould in tobacco).<br>
The activity of Rpi-blb2 in a plant, a plant cell, a plant tissue, a plant organ or part thereof according to the present invention can be increased, generated or stimulated via methods which are well known to a person skilled in the art and e.g. are described in Sambrook et al., Cold Spring Harbor Laboratory Press, NY, 1989.<br>
Thus, in a preferred embodiment, the present invention relates to the method of the invention, wherein the expression is a de novo expression.<br>
The term "de novo-Expression" in a cell, a tissue or in an organism or in a part thereof as understood herein relates to the expression of a gene product after a previous non-detectability of said gene product or an activity of said gene product, in particular of a corresponding polypeptide or polynucleotide in a cell, a tissue or in an organism or in a part thereof. Preferred is that the gene encoding a polypeptide or a polynucleotide in a cell, a tissue or in an organism or in parts thereof and which should be de novo-expressed is not present in the genome in a cell, a tissue or in an organism or in parts thereof. If the expression of a gene product can not be detected in a cell, a tissue or in an organism or in parts thereof, it is generally assumed that no expression occurs in a cell, a tissue or in an organism or in parts thereof. Accordingly, if the activity can not be detected, it is generally assumed that no corresponding activity exists. A person skilled in the art, however, knows that the detection methods and means develop to higher sensitivity. Thus, in a preferred embodiment, the term "de novo-Expression" relates to a novel or additional expression in systems, where the level of activity, e.g. due to a low<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
expression level or the expression of an (nearly) inactive gene product is too low to confer any resistance to a plant pathogen, in particular to P. infestans. A comparison of a knock out strain and a low and/or high-expression strain-phenotype can show, whether any difference in resistance to any of the herein mentioned pathogens is observable.<br>
Accordingly, in another embodiment of the present invention, the endogenous activity of a Rpi-blb2 and/or a further resistance protein is increased.<br>
The level of expression in a cell can be increased by methods known to a person skilled in the art Several techniques are described herein, e.g. the transgenic expression of the polynucleotide or polypeptide of the present invention. The poly-nucleotide or polypeptide can be of foreign origin. Preferred is that a polynucleotide of the same genetic origin as the host cell, plant cell, plant tissue, or plant is introduced.<br>
The activity, in particular an endogenous activity but also the activity of a transgenic expressed Rpi-blb2 can be increased by several methods. Accordingly, in a preferred embodiment, the activity of the resistance proteins described herein is increased by one or more of the following steps<br>
a)	stabilizing the resistance protein;<br>
b)	stabilizing the resistance protein encoding mRNA;-<br>
c)	increasing the specific activity of the resistance protein;<br>
d)	expressing or increasing the expression of a homologous or artificial transcription<br>
factor for resistance expression;<br>
e)	stimulate resistance protein activity through exogenous inducing factors;<br>
f)	expressing a transgenic resistance gene; and/or<br>
g)	increasing the copy number of the resistance-encoding gene.<br>
In general an activity in an organism, in particular in a plant cell, a plant, or a plant tissue can be increased by increasing the amount of the specific protein, i.e. of the resistance protein, in said organism. "Amount of protein" is understood as meaning the amount of a polypeptide, preferably Rpi-blb2, in an organism, a tissue, a cell or a cell compartment. "Increase" of the amount of protein means the quantitative increase of the amount of a protein in an organism, a tissue, a cell or a cell compartment - for example by one of the methods described herein below - in comparison with the wild type of the same genus and species, to which this method had not been applied, under otherwise identical conditions (such as, for example, culture conditions, plant age and the like). The increase amounts to at least 10%, preferably at least 20% or at least 50%,<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
especially preferably at least 70% or 90%, very especially preferably at least 100%, most preferably at least 200% or more.<br>
"Increase" of the activity is understood as meaning the increase of the total activity of a protein in an organism, a tissue, a cell or a cell compartment in comparison with the wild type of the same genus and species, to which this method had not been applied, under otherwise identical conditions (such as, for example, culture conditions, plant age and the like). The increase amounts to at least 10%, preferably at least 20% or at least 50%, especially preferably at least 70% or 90%, very especially preferably at leas 100%, most preferably at least 200% or more.<br>
In this context, the efficacy of the pathogen resistance can deviate both downward or upward in comparison with a value obtained when increasing one of the Rpnblb2 proteins as shown in SEQ ID NO: 2 or 4. Preferred functional equivalents are those in which the efficacy of the pathogen resistance - measured, for example, by the penetration efficacy of a pathogen or as described herein - differs by not more than 50%, preferably 25%, especially preferably 10% from a comparative value obtained by reducing a Rpi-blb2 protein as shown in SEQ ID NO: 2 or 4. Especially preferred are those sequences where the increase increases the efficacy of pathogen resistance quantitatively by more than 50%, preferably 100%, especially preferably 500%, very especially preferably 1000% based on a comparative value obtained by reducing one of the Rpi-blb2 proteins as shown in SEQ ID NO: 2 or 4.<br>
Any comparison is preferably carried out under analogous conditions. "Analogous conditions5 means thai aii conditions such as, for exampie, culture or growing conditions, assay conditions (such as buffer, temperature, substrates, pathogen concentration and the like) are kept identical between the experiments to be compared and that the set-ups differ only by the sequence of the Rpi-blb2 polypeptides to be compared, their organism of origin and, if appropriate, the pathogen. When choosing the pathogen, each comparison requires that the pathogen be chosen which is most similar to the other equivalent, taking into consideration the species specificity.<br>
Due to the increased Rpi-blb2 activity, the resistance of a plant or a part thereof is increased. In a preferred embodiment, the method of the present invention results in reduction in the sporulation index of at least 30% after infection with P. infestans compared to a wild type, more preferred is a reduction of 50%, even more preferred are 70%, even more preferred are more than 80%, more preferred are 85% and 90%. Most preferred is 95% or more.<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Accordingly, the present invention also relates to said polynudeotide of the invention, as defined above encoding a Rpi-blb2 protein comprising a nucleic acid molecule selected from the group consisting of;<br>
a)	nucleic acid molecules encoding at least the mature form of the polypeptide de<br>
picted in SEQ ID NO: 2 or 4;<br>
b)	nucleic acid molecules comprising the coding sequence as depicted in SEQ ID<br>
NO: 1 or 3 or 5 or 6 or encoding at least the mature form of the pofypeptide;<br>
c)	nucleic acid molecules the nucleotide sequence of which is degenerate as a result<br>
of the genetic code to a nucleotide sequence of (a) or (b);<br>
d)	nucleic acid molecules encoding a polypeptide derived from the polypeptide en<br>
coded by a potynucleotide of (a) to (c) by way of substitution, deletion and/or addi<br>
tion of one or several amino acids of the amino acid sequence of the polypeptide<br>
encoded by a polynudeotide of (a) to (c);<br>
e)	nucleic acid molecules encoding a polypeptide the sequence of which has an iden<br>
tity of 70% or more to the amino acid sequence of the polypeptide encoded by a<br>
nucleic acid molecule of (a) or (b);<br>
f)	nucleic acid molecules comprising a fragment or a epitope-bearing portion of a<br>
polypeptide encoded by a nucleic acid molecule of any one of (a) to (e);<br>
g)	nucleic acid molecules comprising a polynudeotide having a sequence of a nucleic<br>
acid molecule amplified from a nucleic acid library using the primers as listed in<br>
Tab. 3b, preferably ARF1F or ARF1R;<br>
h)  nucleic acid molecules encoding polypeptide fragment beginning with amino acid: 1, 30, 50, 100, 200, 300, 500( or 1000 and stopping with, amino acid 1267, 1000, 500, 300, 200t 50, 30, or 1 of a polypeptide encoded by any one of (a) to (g);<br>
i)   nucleic acid molecules comprising at least 20 nucleotides of a polynudeotide of any one of (a) or (d);<br>
j) nucleic acid molecules encoding a polypeptide being recognized by a monoclonal antibodies that have been raised against a polypeptide encoded by a nucleic acid molecule of any one of (a) to (h);<br>
k)  nucleic acid molecules obtainable by screening an appropriate library under stringent conditions with a probe having the sequence of the nucleic acid molecule of any one of (a) to (j) or of a fragment thereof of at least 15, preferable 30, 60, 90 or more nucleotides; and<br>
I) nucleic acid molecules the complementary strand of which hybridises under stringent conditions with a nucleic acid molecule of any one of (a) or (k);<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
or the complementary strand of any one of (a) to (I);<br>
or encoding a polypeptide encoded by a segment of chromosome 6 or of linkage group 6 of Solanum bulbocastanum which co-segregates with a marker selected from table 3a or 3b and which mediates resistance to plant pathogens, preferably of the phylum Oomycetes;<br>
In one embodiment the polynucleotide of the invention does not consist of the sequence depicted in Seq. ID NO.: 7 and/or 9 and/or does not consist of the sequence of a nucleic acid molecule encoding a protein depicted in Seq. ID NO.: 8 and/or 10.<br>
In one embodiment, the polynucleotide of the present invention does not consist of the sequence of a nucleic acid molecule of Mi1.1 or MM .2 and/or of a nucleic acid molecule encoding a Mi 1.1 or Mil.2 protein.<br>
Thus, in one embodiment, the polynucleotide of the present invention may not consist of the sequences shown in Rossi et al. 1998, PNAS USA 95:9750-9754, Milligan et al., 1998. Plant Cell 10:1307-1319; and/or WO 9806750..<br>
In an further embodiment, the polynucleotide of the present invention is derived or isolated from the genome of a organism selected from the group consisting of Menyan-thaceae, Solanaceae, Sclerophytacaceae, Duckeodendraceae, Goetzeaceae, Convol-vulaceae, Cuscutaceae, Polemoniaceae, and Hydrophyllaceae according to the Sys-tema Naturae 2000, Brands, SJ.r Amsterdam or has its origin thereof, more preferably it is selected from ine group consisting of Atropa, Srowaliia, Brunfslsis, Capsicum, Oestrum, Cyphomandra, Datura, Fabiana, Frariciscea, Hyoscyamus, Lycium, Mandragora, Nicandra, Nicotiana, Petunia, Physalis, Schizanthus and Solanum according to the Systema Naturae 2000, Brands, S J.f Amsterdam or has its origin thereof, even more preferred i§ a selection out of the group consisting of Solanaceae family, preferably S. bulbocastanum, potato (S. tuberosum), tomato (S. lycopersicum), petunia, tree.to-mato (S. betaceum), pear melon (S. muricatum) and eggplant (S. melongena). Even more preferred are tomato or potato or S. bulbocastanum as source for the polynucleotide of the present invention. Most preferred is S. bulbocastanum as source.<br>
Rpi-blb2 has been isolated from S. tuberosum material derived form ABPT. Thus, from taxonomic perspective the Rpi-blb2 described is also S. tuberosum-derived. However, the gene was present on an introgression fragment presumably derived from S. bulbocastanum. A lot of S. tuberosum varieties contain introgression fragments of related Solanum species, but still are S. tuberosum. Therefore, S. tuberosum can according to the taxonomical system also be a source for the polynucleotide of the pre-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
sent invention, in particular ABPT-derived S. tuberosum, as well as other varieties of other Solanum species varieties derived in a similar way.<br>
Accordingly, in another embodiment the polynucleotide of the present invention is derived from S. tuberosum.<br>
A polynucleotide of the present invention, e.g., a nucleic acid molecule having a nucleotide sequence of Seq ID NO: 1 or 3 or 5 or 6, or a portion thereof, can be isolated using standard molecular biology techniques and the sequence information provided herein. For example, Rpi-blb2 cDNA can be isolated from a library using all or portion of one of the sequences.of the polynucleotide of the present invention as a hybridisation probe and standard hybridisation techniques (e.g., as described in Sambrook et al.. Molecular Cloning: A Laboratory Manual. 2nd, ed., Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989). Moreover, a polynucleotide encompassing all or a portion of one of the sequences of the polynucleotide of the present invention can be isolated by the polymerase chain reaction using oligonucleotide primers designed based upon this sequence (e.g., a nucleic acid molecule encompassing all or a portion of one of the sequences of polynucleotide of the present invention). For example, mRNA can be isolated from cells, e.g. S. bulbocastanum or another plant (e.g., by the guanidinium-thiocyanate extraction procedure of Chirgwin et al. (1979) Biochemistry 18: 5294-5299) and cDNA can be prepared using reverse transcriptase (e.g., Moloney MLV reverse transcriptase, available from Gibco/BRL, Bethesda, MD; or AMV reverse transcriptase, available from Seikagaku America, Inc., St. Petersburg, FL). Synthetic oiigonucleotide primers for polymerase chain reaction amplification can be designed based upon one of the nucleotide sequences shown in SEQ ID No: 1 or 3 or 5 or 6. A polynucleotide of the invention can be amplified using cDNA or, alternatively, genomic DNA, as a template and appropriate oiigonucleotide primers according to standard PCR amplification techniques. The polynucleotide so amplified can be cloned into an appropriate vector and characterized by DNA sequence analysis. Furthermore, oligonucleotides corresponding to a Rpi-blb2 nucleotide sequence can be prepared by standard synthetic techniques, e.g., using an automated DNA synthesizer.<br>
In an embodiment of the present invention the Rpi-blb 2 protein is encoded by a segment of chromosome 6 or linkage group 6 of Solanum bulbocastanum or S. tuberosum. -<br>
Further the present invention comprises a segment of chromosome 6 or linkage group 6 of S. bulbocastanum or S. tuberosum. In one preferred embodiment in the method of the present invention the Rpi-blb2 protein expressed is encoded<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
by a polynucleotide comprising a segment of chromosome 6 or linkage group 6 of S. bulbocastanum. Preferably said segment a group comprises further cis acting element, e.g. promoters, enhancers, binding sites etc. or trans acting elements, like cofactors, activators or other resistance proteins, which confer an increased resistance. Genomic fragments comprising the Rpi-blb2 gene and further regulatory elements are depicted in Seq. ID NO.: 5 and 6.<br>
A person skilled in the art knows how to obtain a chromosome segment, e.g. by cloning chromosome fragments into BACs, as for example Song, 2003t PNAS 100 (16), 9128 or as described herein and in the references cited herein.<br>
Accordingly, in a further embodiment, the polynucleotide of the present invention or a polynucleotide encoding the Rpi-blb2 protein co-segregates with a marker selected from table 3a or comprises a replication site or hybridisation site for said marker. As described in detail in the examples, the resistance to P. infestans could be mapped with the markers depicted in table 3a or 3b. As closer a marker is localized to a gene,  . as higher is the percentage of lines, i.e. offspring clones, in which the gene co-segregates with said marker. Therefore in a preferred embodiment, Jthe polynucleotide of the present invention co-segregates with the Marker E40M58, CT119 and/or CT216.<br>
In a further embodiment, the present invention relates to a method for making a recombinant vector comprising inserting the polynucleotide.of the present invention into a vector or inserting said polynucleotide and a further resistance protein into a vector.<br>
Accordingly, in one further embodiment, the present inveniion relates io a vector containing the polynucleotide of the present invention or said polynucleotide and a further resistance gene produced by the method of the present invention.<br>
As used herein, the term 'vector" refers to a nucleic acid molecule capable of transporting a polynucleotide to which it has been linked. One type of vector is a "plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is a viral vector, wherein additional DNA or RNA segments can be ligated into the viral genome. Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication and episomal mammalian vectors). Other vectors (e.g., non-episomal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. Moreover, certain vectors are capable of directing the expression of genes to which they are operatively linked. Such vectors are referred to herein as "expression vectors". In general, expression vectors of utility in recombinant DNA tech-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
niques are often in the form of plasmids. In the present specification, "plasmid" and "vector* can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is intended to include such other forms of expression vectors, such as viral vectors (e.g.t replication defective retroviruses, adenoviruses and adeno-associated viruses), which serve equivalent functions.<br>
The present invention also relates to cosmids, viruses, bacteriophages and other vectors used conventionally in genetic engineering that contain a nucleic acid molecule according to the invention. Methods which are well known to those skilled in the art can be used to construct various plasmids and vectors; see, for example, the techniques described in Sambrook, Molecular Cloning A Laboratory Manual, Cold Spring Harbor Laboratory (1989) N.Y. and Ausubel, Current Protocols in Molecular Biology, Green Publishing Associates and Wiley Interscience, N.Y. (1989). Alternatively, the nucleic acid molecules and vectors of the invention can be reconstituted into liposomes for delivery to target cells.<br>
In another embodiment, the vector of the present invention or the method of the present invention is characterized therein, that the polynucleotide, encoding Rpi-blb2 protein or a further resistance protein is operatively linked to expression control sequences and/or a linked to a nucleic acid sequence encoding a transgenic expression regulating signal allowing expression in prokaryotic or eukaryotic host cells.<br>
In a preferred embodiment, the present invention relates to a vector of the present invention or the method of the present invention in which the polynucleotide encoding Rpi-blb2 protein and/or the further resistance protein is operatively linked to expression control sequences of the same species origin as the polynucleotide encoding Rpi-blb2 protein and/or the further resistance protein.<br>
In the case that a nucleic acid molecule according to the invention is expressed in a cell it is in principle possible to modify the coding sequence in such a way that the protein is located in any desired compartment of the plant cell. These include the nucleus, endoplasmatic reticulum, the vacuole, the mitochondria, the plastids like amyloplasts, chloroplasts, chromoplasts, the apoplast, the cytoplasm, extracellular space, oil bodies, peroxisomes and other compartments of plant cells (for review see Kermode, Crit. Rev. Plant Sci. 15, 4 (1996), 285-423 and references cited therein). The polynucleotide can then operatively be fused to an appropriate polynucleotide, e.g., a vector, encoding a signal for the transport into the desirable compartment.<br>
In another preferred embodiment of the present invention relates to a vector in which the polynucleotide of the present invention is operativety linked to expression control<br><br>
WO 2005/014631	PCTVEP2004/008683<br>
sequences allowing expression in proKaryouc or eukaryotic host cells. The nature of such control sequences differs depending upon the host organism. In prokaryotes, control sequences generally include promoter, ribosomal binding site, and terminators. In eukaryotes, generally control sequences include promoters, terminators and, in some instances, enhancers, transactivators; or transcription factors.<br>
The term "control sequence" is intended to include, at a minimum, components the presence of which are necessary for expression, and may also include additional advantageous components.<br>
The term " operatively finked" refers to a juxtaposition wherein the components so described are in a relationship permitting them to function in their intended manner. A control sequence 'operatively linked" to a coding sequence is ligated in such a way that expression of the coding sequence is achieved under conditions compatible with the control sequences. In case the control sequence is a prompter, it is obvious for a skilled person that double-stranded nucleic acid is used.<br>
Operable linkage is to be understood as meaning, for example, the sequential arrangement of a promoter with the nucleic acid sequence to be expressed and, if appropriate, further regulatory elements such as, for example, a terminator in such a way that each of the regulatory elements can fulfil its function when the nucleic acid sequence is expressed recornbinantly, depending on the arrangement of the nucleic acid sequences in relation to sense or antisense RNA. To this end, direct linkage in the chemical sense is not necessarily required. Genetic control sequences such as, for example, enhancer sequences, csn slso exsrt their function OP. the targe! sequence from positions which are further away, or indeed from other DNA molecules. Preferred arrangements are those in which the nucleic acid sequence to be expressed recombi-nantly is positioned behind the sequence acting as promoter, so that the two sequences are linked covalently to each other The distance between the promoter sequence and the nucleic acid sequence to be expressed recombinantly is preferably less than 200 base pairs, especially preferably less than 100 base pairs, very especially preferably less than 50 base pairs.<br>
Operable linkage, and an expression cassette, can be generated by means of customary recombination and cloning techniques as are described, for example, in Maniatis T, Fritsch EF and Sambrook J (1989) Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor (NY), in Silhavy TJ, Berman ML and Enquist LW (1984) Experiments with Gene Fusions, Cold Spring Harbor Laboratory, Cold Spring Harbor (NY), in Ausubel FM et al. (1987) Current Protocols in Molecular Biology, Greene Publishing Assoc. and Wiley Interscience and in Gelvin et al. (1990)<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
In: Plant Molecular Biology Manual. However, further sequences which, for example, act as a linker with specific cleavage sites for restriction enzymes, or as a signal pep-tide, may also be positioned between the two sequences. The insertion of sequences may also lead to the expression of fusion proteins. Preferably, the expression cassette, consisting of a linkage of promoter and nucleic acid sequence to be expressed, can exist in a vector-integrated form and be inserted into a plant genome, for example by transformation.<br>
Such regulatory sequences are described, for example, in Goeddel; Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990) or see: Gruber and Crosby, in: Methods in Plant Molecular Biology and Biotechnology, CRC Press, Boca Raton, Florida, eds.:Glick and Thompson, Chapter 7, 89-108 including the references therein. Regulatory sequences include those which direct constitutive expression of a nucleotide sequence in many types of host cell and those which direct expression of the nucleotide sequence only in certain host cells or under certain conditions. It will be appreciated by those skilled in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein desired, etc. The expression vectors of the invention can be introduced into host cells to thereby produce proteins or peptides, including fusion proteins or peptides, encoded by polynucleotides as described herein.<br>
The recombinant expression vectors of the invention can be designed for expression of said resistance proteins, preferably Rpi-blb2, in prokaryotic or eukaryotic cells. For example, genes encoding the polynucleotide of the invention can be expressed in bacterial cells such as E. coli, C. glutamicum, Agrobacterium tumefaciens, insect cells (using baculovirus expression vectors), yeast and other fungal cells (see Romanos, (1992), Yeast 8: 423-488; van den Hondel, (1991) J.W. Bennet &amp; LL Lasure, eds., p. 396-428: Academic Press: San Diego; and van den Hondel, (1991) in: Applied Molecular Genetics of Fungi, Peberdy, eds., p. 1-28, Cambridge University Press: Cambridge), algae (Falciatore et al., 1999, Marine Biotechnology. 1, 3:239-251), and multicellular plant cells (see Schmidt, R. (1988), Plant Cell Rep.: 583-586); Plant Molecular Biology and Biotechnology, C Press, Boca Raton, Florida, chapter 6/7, S.71-119 (1993); F.F. White, B. Jenes et al., Techniques for Gene Transfer, in: Transgenic Plants, Vol. 1, Engineering and Utilization, eds.:Kung und R. Wu, Academic Press (1993), 128-43; Potrykus, Annu. Rev. Plant Physiol. Plant Molec. Biol. 42 (1991), 205-225 (and references cited therein) or mammalian cells. Suitable host cells are discussed further in Goeddel, Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, CA (1990). Alternatively, the recombinant expression vector can be transcribed and translated in vitro, for example using T7 promoter regulatory sequences and T7 polymerase.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
Expression of proteins in prokaryotes is most often carried out with vectors containing constitutive or inducible promoters directing the expression of either fusion or non-fusion proteins. Fusion vectors add a number of amino acids to a protein encoded therein, usually to the amino terminus of the recombinant protein but also to the C-terminus or fused within suitable regions in the proteins. Such fusion vectors typically serve three purposes: 1) to increase expression of recombinant protein; 2) to increase the solubility of the recombinant protein; and 3) to aid in the purification of the recombinant protein by acting as a ligand in affinity purification. Further, the fusion vector can also encode for additional proteins, which expression supports an increase of the activity of Rpi-blb2 or of the resistance of a plant against plant pathogens, e.g. other resistance proteins. Often, in fusion expression vectors, a proteolytic cleavage site is introduced at the junction of the fusion moiety and the recombinant protein to enable separation of the recombinant protein from the fusion moiety subsequent to purification of the fusion protein. Such enzymes, and their cognate recognition sequences, include Factor Xaf thrombin, and enterokinase.<br>
Typical fusion expression vectors include pGEX (Pharmacia Biotech Inc; Smith, D.B. and Johnson, K.S. (1988) Gene 67:31-40), pMAL (New England Biolabs, Beverly, MA) and pRIT5 (Pharmacia, Piscataway, NJ).<br>
Examples of suitable inducible non-fusion E. coli expression vectors include pTrc (Amann (1988) Gene 69:301-315) and pET 11d (Studier et al., Gene Expression Technology: Methods ih Enzymology 185, Academic Press, San Diego, California (1990) 50-83.<br>
One strategy to maximize recombinant protein expression is to express the protein in a host bacterium with an impaired capacity to proteolytically cleave the recombinant protein (Gottesman, S., Gene Expression Technology: Methods in Enzymology 185, Academic Press, San Diego, California (1990) 119-128). Another strategy is to alter the nucleic acid sequence of the nucleic acid to be inserted into an expression vector so that the individual codons for each amino acid are those preferentially utilized in the bacterium chosen for expression, such as E. coli or C. glutamicum (Wada et al. (1992) Nucleic Acids Res. 20:2111-2118). Such alteration of nucleic acid sequences of the invention can be carried out by standard DNA synthesis techniques.<br>
Further, the vector can be a yeast expression vector. Examples of vectors for expression in yeast S. cerivisae include pYepSed (Baldari, et al., (1987) Embo J. 6:229-234), pMFa (Kurjan and Herskowitz, (1982) Cell 30:933-943), pJRY88 (Schultz et al., (1987) Gene 54:113-123), and pYES2 (Invitrogen Corporation, San Diego, CA).<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Preferably, the polynucleotide of the present invention or described herein is opera-tively linked to a plant expression control sequence, e.g. an expression cassettes. A plant expression cassette preferably contains regulatory sequences capable to drive gene expression in plants cells and which are operatively linked so that each sequence can fulfil its function such as termination of transcription such as polyadenylation signals. Preferred polyadenylation signals are those originating from Agrobacterium tume-faciens t-DNA such as the gene 3 known as octopine synthase of the Ti-plasmid pTiACH5 (Gielen et a)., EMBO J. 3 (1984), 835 ff) or functional equivalents thereof but also all other terminators functionally active in plants are suitable.<br>
As plant gene expression is very often not limited on transcriptional levels as plant expression cassette preferably contains other operatively linked sequences like trans-lational enhancers such as the overdrive-sequence containing the 5'-untranslated leader sequence from tobacco mosaic virus enhancing the protein per RNA ratio (Gal-lie et al 1987, NucL Acids Research 15:8693-8711).<br>
Accordingly, the polynucleotide described herein can be operatively linked to an appropriate promoter conferring gene expression in a timely, cell or tissue specific manner. Preferred are promoters driving constitutitive expression (Benfey et al.a EMBO J. 8 (1989) 2195-2202) like those derived from plant viruses like the 35S CAMV (Franck et al., Cell 21(1980) 285-294), the 19S CaMV (see also US5352605 and WO8402913) or plant promoters like those from Rubisco small subunit described in US 4962028.<br>
The term plant-specific promoters is understood as meaning, in principle, any promoter which is capable of governing the expression of genes, in particular foreign genes, in plants or plant parts, plant cells, plant tissues or plant cultures. In this context, expression can be, for example, constitutive, inducible, or development-dependent.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
The following are preferred:<br>
a)	Constitutive promoters<br>
Preferred vectors are those which make possible constitutive expression in plants (Benfey et al.(1989) EMBO J 8:2195*2202). "Constitutive" promoter is understood as meaning those promoters which ensure expression in a large number of, preferably all, tissues over a substantial period of plant development, preferably at all stages of plant development In particular a plant promoter or a promoter derived from a plant virus are preferably used. Particularly preferred is the promoter of the CaMV cauliflower mosaic virus 35S transcript (Franck et al. (1980) Cell 21:285-294; Odell et al. (1985) Nature 313:810-812; Shewmaker et al. (1985) Virology 140:281-288; Gardner et al. (1986) Plant Mol Bid 6:221- 228) or the 19S CaMV promoter (US 5,352,605; WO 84/02913; Benfey et al. (1989) EMBO J 8:2195-2202). Another suitable constitutive promoter is the "Rubisco small subunit (SSU)" promoter (US 4,962,028), the leguminB promoter (GenBank Ace. No. X03677), the Agrobacterium nopaline synthase promoter, the TR dual promoter, the Agrobacterium OCS (octopine synthase) promoter, the ubiquitin promoter (Holtorf S et al. (1995) Plant Mol Biol 29:637-649), the ubiquitin 1 promoter (Christensen et al. (1992) Plant Mol Biol 18:675-689; Bruce et al. (1989) Proc Natl Acad Sci USA 86:9692-9696), the Smas promoter, the cinnamyl alcohol dehydro-genase promoter (US 5,683,439), the promoters of the vacuolar ATPase subunits or the promoter of a proline-rich protein from wheat (WO 91/13991), and further promoters of genes whose constitutive expression in plants is known to the skilled worker.<br>
b)	Tissue-specific promoiers<br>
Preferred are furthermore promoters with specificity for the anthers, ovaries, flowers, leaves, stems, roots, and seeds.   -<br>
Seed-specific promoters<br>
such as, for example, the phaseolin promoter (US 5,504,200; Bustos MM et al. (1989)<br>
Plant Cell 1(9):839-53), the 2S albumin gene promoter (Joseffson LG et al. (1987) J<br>
Biol Chem 262:12196-12201), the legumin promoter (Shirsat A et al. (1989) Mol Gen<br>
Genet 215(2): 326-331), the USP (unknown seed protein) promoter (BSumiein H et al.<br>
(1991) Mol Gen Genet 225(3):459-67), the napin gene promoter (US 5,608,152;<br>
Stalberg K et al. (1996) L Planta 199:515-519), the sucrose binding protein promoter<br>
(WO 00/26388) or the legumin B4 promoter (LeB4; BSumiein H et al. (1991) Mol Gen<br>
Genet 225:121-128; Baeumlein et al. (1992) Plant Journal 2(2):233-9; Fiedler U et al.<br>
(1995) Biotechnology (NY) 13(10):1090f), the Arabidopsis oleosin promoter<br>
(WO 98/45461), the Brassica Bce4 promoter (WO 91/13980). Further suitable seed-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
specific promoters are those of the genes encoding the high-molecular-weight glutenin (HMWG)( gliadin, branching enzyme. ADP glucose pyrophosphatase (AGPase) or starch synthase. Furthermore preferred are promoters which permit seed-specific expression in monocots such as maize, barley, wheat, rye. rice and the like. The following can be employed advantageously; the promoter of the Ipt2 or Ipt1 gene (WO 95/15389, WO 95/23230) or the promoters described in WO 99/16890 (promoters of the hordein gene, the glutelin gene, the oryzin gene, the prolamin gene, the gliadin gene, the glute-I'm gene, the zein gene, the kasirin gene or the secalin gene).<br>
Tuber-, storage-root-, or root-specific promoters such as, for example, the patatin promoter class I (B33), the potato cathepsin D inhibitor promoter.<br>
Leaf-specific promoters<br>
such as the potato cytosolic FBPase promoter (WO 97/05900), the Rubisco (ribulose-1,5-bisphosphate carboxylase) SSU (small subunit) promoter or the ST-LSI promoter from potato (Stockhaus et al. (1989) EMBO J 8:2445-2451). Very especially preferred are epidermis-specific promoters such as, for example, the OXLP gene (oxalate-oxidase-like protein) promoter (Wei et al. (1998) Plant Mol. Biol. 36:101-112).<br>
Flower-specific promoters<br>
such as, for example, the phytoene synthase promoter (WO 92/16635) or the promoter<br>
of the P-rr gene (WO 98/22593),<br>
Anther-specific promoters<br>
such as the 5126 promoter (US 5.689,049, US 5,689.051), the glob-l promoter and the<br>
y-zein promoter.<br>
c)      Chemically inducible promoters<br>
The expression cassettes can also comprise a chemically inducible promoter (review article: Gatz et al. (1997) Annu Rev Plant Physiol Plant Mol Biol 48:89-108), by which the expression of the exogenous gene in the plant at a particular point in time can be controlled. Such promoters such as, for example, the PRP1 promoter (Ward et al. (1993) Plant Mol Biol 22:361-366), a salicylic-acid-inducible promoter (WO 95/19443), a benzenesulfonamide-inducible promoter (EP 0 388 186), a tetracycline-inducible promoter (Gatz et al. (1992) Plant J 2:397-404), an abscisic-acid-inducible promoter (EP 0 335 528) or an ethanol- or cyclohexanone-inducible promoter (WO 93/21334) can likewise be used.<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
gen-inducible and epidermis-specific promoters being most preferred. Also preferred is the natural promoter, which is e.g. comprised in the genomic fragment depicted in Seq. ID NO.: 5 and 6.<br>
Furthermore, further promoters may be linked operatively to the nucleic acid sequence to be expressed, which promoters make possible the expression in further plant tissues or in other organisms, such as, for example, E. coli bacteria. Suitable plant promoters are, in principle, all of the above-described promoters.<br>
The term "genetic control sequences* is to be understood in the broad sense and refers to also all those sequences which have an effect on the materialization or the function of the expression cassette according to the invention. For example, genetic control sequences modify the transcription and translation in prokaryotic or eukaryotic organisms. Preferably, the expression cassettes according to the invention encompass the promoter with specificity for the embryonic epidermis and/or the flower 5'-upstream of the nucleic acid sequence in question to be expressed recombinantly, and 3'-downstream a terminator sequence as additional genetic control sequence and, if appropriate, further customary regulatory elements, in each case linked operatively to the nucleic acid sequence to be expressed recombinantly.<br>
Genetic control sequences also encompass further promoters, promoter elements, or minimal promoters, all of which can modify the expression-governing properties. Thus, for example, the tissue-specific expression may additionally depend on certain stress-ors, owing to genetic control sequences. Such elements have been described, for example, for water stress, absclsic acid (Lam E and Chua NH, J Biol Chem 1991; 266(26): 17131 -17135) and heat stress (Schoffl F et al., Molecular &amp; General Genetics 217(2-3):246-53, 1989).<br>
Further advantageous control sequences are, for example, the Gram-positive promoters amy and SPO2, and the yeast or fungal promoters ADC1, MFa t AC, P-60, CYC1, GAPDH, TEF, rp28, ADH.<br>
In principle, all natural promoters with their regulatory sequences like those mentioned above may be used for the method according to the invention. In addition, synthetic promoters may also be used advantageously.<br>
Genetic control sequences furthermore also encompass the 5'-untranslated regions, introns or noncoding 3'-region of genes, such as, for example, the actin-1 intron, or the Adh1-S introns 1, 2, and 6 (general reference: The Maize Handbook, Chapter 116, Freeling and Walbot, Eds., Springer, New York (1994)). It has been demonstrated that<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
 V<br>
they may play a significant role in the regulation of gene expression. Thus, it has been demonstrated that 5'-untranslated sequences can enhance the transient expression of heterologous genes. Examples of translation enhancers which may be mentioned are the tobacco mosaic virus 5' leader sequence (Gallie et al. (1987) Nucl Acids Res 15:8693-8711) and the like. Furthermore, they may promote tissue specificity (Rouster J et al. (1998) Plant J 15:435-440).<br>
The expression cassette may advantageously comprise one or more of what are known as enhancer sequences, linked operatively to the promoter, which make possible an increased recombinant expression of the nucleic acid sequence. Additional advantageous sequences, such as further regulatory elements or terminators, may also be inserted at the 3' end of the nucleic acid sequences to be expressed recombinantly. One or more copies of the nucleic acid sequences to be expressed recombinantly may be present in the gene construct.<br>
In one embodiment the natural terminator sequence comprised in the genomic fragment depicted in Seq ID No.: 5 and/or 6 is used.<br>
Polyadervylation signals which are suitable as control sequences are plant polyadenyla-tion signals, preferably those which essentially correspond to T-DNA polyadenylation signals from Agrobacterium tumefaciens, in particular gene 3' of the T-DNA (octopin synthase) of the Ti plasmid pTiACHS (Gielen et al. (1984) EMBO J 3:835 et seq.) or functional equivalents thereof. Examples of terminator sequences which are especially suitable are the OCS (octopin synthase) terminator and the NOS (nopalin synthase)<br>
iS      iOr<br>
Control sequences are furthermore to be understood as those which make possible homologous recombination or insertion into the genome of a host organism or which permit removal from the genome. In the case of homologous recombination, for example the natural promoter of a particular gene may be exchanged for a promoter with specificity for the embryonic epidermis and/or the flower. Methods such as the cre/lox technology permit a tissue-specific, if appropriate inducible, removal of the expression cassette from the genome of the host organism (Sauer B (1998) Methods. 14(4):381-92). In this method, specific flanking sequences (lox sequences), which later allow removal by means of ere recombinase, are attached to the target gene.<br>
An expression cassette and the vectors derived from it may comprise further functional elements. The term functional element is to be understood in the broad sense and refers to all those elements which have an effect on the generation, amplification, or func-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
tion of the expression cassettes, vectors, ortransgenic organisms according to the invention. The following may be mentioned by way of example, but not by limitation:<br>
a)	Selection markers which confer a resistance to a metabolism inhibitor such as<br>
2~deoxyglucose-6-phosphate (WO 98/45456), antibiotics or biocides, preferably<br>
herbicides, such as, for example, kananriycin, G 418, bleomycin or hygromycin,<br>
or else phosphinothricin and the like. Especially preferred selection markers are<br>
those which confer resistance to herbicides. Examples which may be mentioned<br>
are: DNA sequences which encode phosphinothricin acetyl transferases (PAT)<br>
and which inactivate glutamine synthase inhibitors (bar and pat genes), 5-enol-<br>
pyruvylshikimate-3-phosphate synthase genes (EPSP synthase genes), which<br>
confer resistance to Glyphosater (N-(phosphonomethyl)glycine), the gox gerve,<br>
which encodes Glyphosater-degrading enzymes (Glyphosate oxidoreductase),<br>
the deh gene (encoding a dehalogenase which inactivates dalapon), sulfonyl-<br>
urea- and imidazolinone-inactivating acetolactate synthases, and bxn genes,<br>
which encode bromoxynil-degrading nitrilase enzymes, the aasa gene, which<br>
confers resistance to the antibiotic apectinomycin, the streptomycin phospho-<br>
transferase (SPT) gene, which allows resistance to streptomycin, the neomycin<br>
phosphotransferase (NPTII) gene, which confers resistance to kanamycin or<br>
geneticidin, the hygromycin phosphotransferase (HPT) gene, which mediates<br>
resistance to hygromycin, the acetolactate synthase gene (ALS), which confers<br>
resistance to sutfonylurea herbicides (for example mutated ALS variants with, for<br>
example, the S4 and/or Hra mutation).<br>
b)	Reporter genes which encode readily quantifiable proteins and, via their color or<br>
enzyme activity, make possible an assessment of the transformation efficacy, the<br>
site of expression or the time of expression. Very especially preferred in this con<br>
text are genes encoding reporter proteins (Schenborn E, Groskreutz D. Mol Bio-<br>
technol. 1999; 13(1):29-44) such as the green fluorescent protein (GFP) (Sheen<br>
et al.(1995) Plant Journal 8(5):777-784; Haseloff et al.(1997) Proc Natl Acad Sci<br>
USA 94(6):2122-2127; Reichel et al.(1996) Proc Natl Acad Sci USA 93(12):5888-<br>
5893; Tian et al. (1997) Plant Cell Rep 16:267-271; WO 97/41228; Chui WL et al.<br>
(1996) Curr Biol 6:325-330; Leffel SM et al. (1997) Biotechniques. 23(5):912-8),<br>
chloramphenicol transferase, a luciferase (Ow et aL (1986) Science 234:856-<br>
859; Millar et al. (1992) Plant Mol Biol Rep 10:324-414), the aequorin gene<br>
(Prasher et al. (1985) Biochem Biophys Res Commun 126(3):1259-1268), p-<br>
galactosidase, R locus gene (encoding a protein which regulates the production<br>
of anthocyanin pigments (red coloring) in plant tissue and thus makes possible<br>
the direct analysis of the promoter activity without addition of further auxiliary<br>
substances or chromogenic substrates; Dellaporta et al., In: Chromosome Struc-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
ture and Function: Impact of New Concepts, 18th Stadler Genetics Symposium, 11:263-282, 1988), with (Hjlucuronidase being very especially preferred (Jefferson et aL, EMBO J. 1987, 6, 3901-3907).<br>
c)	Origins of replication, which ensure amplification of the expression cassettes or<br>
vectors according to the invention in, for example, E. colL Examples which may<br>
be mentioned are ORI (origin of DNA replication), the pBR322 ori or the P15A ori<br>
(Sambrook et al.; Molecular Cloning. A Laboratory Manual, 2nded. Cold Spring<br>
Harbor Laboratory Press, Cold Spring Harbor, NY, 1989).<br>
d)	Elements which are necessary for Agrobacterium-mediated plant transformation,<br>
such as, for example, the right or left border of the T-DNA or the vir region.<br>
To select cells which have successfully undergone homologous recombination, or else to select transformed cells, it is, as a rule, necessary additionally to introduce a selectable marker, which confers resistance to a biocide (for example herbicide), a metabolism inhibitor such as 2-deoxyglucose-6-phosphate (WO 98/45456) or an antibiotic to the cells which have successfully undergone recombination. The selection marker permits the selection of the transformed cells from untransformed ones (McCormick et al. (1986) Plant Cell Reports 5:81-84).<br>
The introduction of an expression cassette according to the invention into an organism or cells, tissues, organs, parts or seeds thereof (preferably into plants or plant cells, tissue, organs, parts or seeds) can be effected advantageously using vectors which<br>
comprise the expression cassettes. The expression cassette Can be introduced into the<br>
vector (for example a plasmid) via a suitable restriction cleavage site. The plasmid formed is first introduced into E. coli. Correctly transformed E. coli are selected, grown, and the recombinant plasmid is obtained by the methods familiar to the skilled worker. Restriction analysis and sequencing may serve to verify the cloning step.<br>
Further promoters for expression in specific plant parts are e.g. the napin-gene promoter from rapeseed (US5608152), the USP-promoter from Vicia faba (Baeumlein et al., Mol Gen Genet, 1991, 225 (3):459-67), the oleosin-promoter from Arabidopsis (WO9845461), the phaseolin-promoter from Phaseolus vulgaris (US5504200), the Bce4-promoter from Brassica (WO9113980) or the legumin B4 promoter (LeB4; Baeumlein et al., 1992, Plant Journal, 2 (2):233-9) as well as promoters conferring seed specific expression in monocot plants like maize, barley, wheat, rye, rice etc. Suitable promoters to note are the Ipt2 or Ipt1-gene promoter from barley (WO9515389 and WO9523230) or those described in WO9916890 (promoters from the barley hordein-gene, the rice glutelin gene, the rice oryzin gene, the rice prolamin gene, the wheat<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
gliadin gene, wheat glutelin gene, the maize zein gene, the oat glutelin gene, the Sorghum kasirin-gene, the rye secalin gene).<br>
. Further, the polynucleotide of the invention can be cloned into the expression vector in an antisense orientation. That is, the DNA molecule is operativeiy linked to a regulatory sequence in a manner which allows for expression (by transcription of the DNA molecule) of an RNA molecule which is antisense to the mRNA encoded by the polynucleotide of the present invention. Regulatory sequences operativeiy.linked to a nucleic acid cloned in the antisense orientation can be chosen which direct the continuous expression of the antisense RNA molecule in a variety of cell types, for instance viral promoters and/or enhancers, or regulatory sequences can be chosen which direct constitutive, tissue specific or cell type specific expression of antisense RNA. The antisense expression vector can be in the form of a recombinant plasmid, phagemid or attenuated virus in which antisense nucleic acid molecules are produced under the control of a high efficiency regulatory region, the activity of which can be determined by the cell    . type into which the vector is introduced. For a discussion of the regulation of gene expression using antisense genes see Weintraub, H. et al., Antisense RNA as a molecular tool for genetic analysis, Reviews - Trends in Genetics, Vol. 1(1) 1986 and Mol et al.,.1990, FEBS Letters 268:427-430.<br>
In one embodiment the present invention relates to a method of making a recombinant host cell comprising introducing the vector or the polynucleotide of the present invention or said vector or said polynucleotide and a vector for expressing a further resistance protein into a host cell.<br>
Vector DNA can be introduced into prokaryotic or eukaryotic cells via conventional transformation or transfection techniques. As used herein, the terms "transformation" and "transfection", conjugation and transduction are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., DNA) into a host cell, including calcium phosphate or calcium chloride co-precipitation, DEAE-dextran-mediated transfection, lipofection, natural competence, chemical-mediated transfer, or electroporation. Suitable methods for transforming or transfecting host cells including ■ plant cells can be found in Sambrook, et al. (Molecular Cloning: A Laboratory Manual. 2nd, ed.t Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989) and other laboratory manuals such as Methods in Molecular Biology, 1995, Vol. 44, Agrobacterium protocols, ed: Gartland and Davey, Humana Press, Totowa, New Jersey.<br>
For stable transfection of eukaryotic cells, it is known that, depending upon the expression vector and transfection technique used, only a small fraction of cells may integrate<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
the foreign DNA into their genome. In order to identify and select these integrants, a gene that encodes a selectable marker (e.g., resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Preferred selectable markers include those which confer resistance to drugs, such as G418, hygromycin, and methotrexate or in plants that confer resistance towards a herbicide such as glyphosatc or glufosinate. Nucleic acid encoding a selectable marker can be introduced into a host cell on the same vector as that encoding the polypeptide of the present invention or can be introduced on a separate vector. Cells stably transfected with the introduced nucleic add can be identified by, for example, drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die).<br>
Further host cells can be produced which contain selection systems which allow for regulated expression of the introduced gene. For example, inclusion of the polynucleo tide of the invention on a vector placing it under control of the lac operon permits expression of the polynudeotide only in the presence of IPTG. Such regulatory systems are well known in the art<br>
Preferably, the.introduced nucleic acid molecule is foreign to the host cell<br>
By "foreign" it is meant that the nucleic acid molecule is either heterologous with, respect to the host cell, this means derived from a cell or organism with a different ge-nomic background, or is homologous with respect to the host cell but located in a different genomic environment than the naturally occurring counterpart of said nucleic acid molecule. This means that, if the nucleic acid molecule is homologous with respect to the host ceii, it is not located in its natural location in the genome of said host cell, in particular it is surrounded by different genes. In this case the nucleic add molecule may be either under the control of its own promoter or under the control of a heterologous promoter. The vector or nucleic acid molecule according to the invention which is present in the host cell may either be integrated into the genome of the host cell or it may be maintained in some form extrachromosomally. In this respect, it is also to be understood that the nucleic acid molecule of the invention can be used to restore or create a mutant gene via homologous recombination (Paszkowski (ed.), Homologous Recombination and Gene Silencing in Plants. Kluwer Academic Publishers (1994)).<br>
Accordingly, in another embodiment the present invention relates to a host cell genetically engineered with the polynucleotide of the invention or the vector of the invention, or said vector or said polynucleotide and a vector or a polynucleotide for expressing a further resistance protein.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
The terms "host cell" arid "recombinant host cell" are used interchangeably herein. It is understood that such terms refer not only to the particular subject cell but also to the progeny or potential progeny of such a cell. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent cell, but are still included within the scope of the term as used herein.<br>
For example, a polynucleotide of the present invention can be introduced in bacterial cells, insect cells, fungal cells or mammalian cells (such as Chinese hamster ovary cells (CHO) or COS cells), algae, ciliates, plant cells or fungi. Suitable host cells are known to those skilled in the art Preferred are E. coli, baculovirus, Agrobacterium, or plant cells.<br>
Further, the host cell can also be transformed such that further enzymes and proteins are (over)expressed which expression supports an increase of resistance of a plant to pathogens. Preferably, a further resistance gene is also expressed, preferably one or more resistance genes, preferably the genes as mentioned herein, is/are also expressed. Most preferred is a coexpression of Rpi-blb2 and Rpi-blb.<br>
Further preferred are cells of one of herein mentioned plants, in particular, of one of the above-mentioned Solanaceae, most preferred are potato, tomato, petunia, tree tomato, pear melon, or eggplant.<br>
In another embodiment, the present invention relates to a process for the production of the polypeptide of the present invention, in particular of a protein having Rpi-blb2 activity comprising culturing the host cell of the invention and recovering the polypeptide encoded by said polynucleotide and expressed by the host cell from the culture or the cells.<br>
The term "expression" means the production of a protein or nudeotide sequence in the cell. However, said term also includes expression of the protein in a cell-free system. It includes transcription into an RNA product, post-transcriptional modification and/or translation to a protein product or polypeptide from an DNA encoding that product, as well as possible posMranslational modifications.<br>
Depending on the specific constructs and conditions used, the protein may be recovered from the cells, from the culture medium or from both. For the person skilled in the art it is well known that it is not only possible to express a native protein but also to express the protein as fusion polypeptides or to add signal sequences directing the protein to specific compartments of the host cell, e.g., ensuring secretion of the protein<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
into the culture medium, etc. Furthermore, such a protein and fragments thereof can be chemically synthesized and/or modified according to standard methods described, for example herein below.<br>
A host cell of the invention, such as a prokaryotic or eukaryotic host cell in culture, can be used to produce (I.e., express) the polypeptide encoded by the polynucleotide of the invention, preferably a pofypeptid having Rpi-blb2 activity. An alternate method can be applied in addition in plants by the direct transfer of DNA into developing flowers via electroporation or Agrobacterium mediated gene transfer. Accordingly, the invention further provides methods for producing Rpi-blb2 using the host cells of the invention. In one embodiment, the method comprises culturing the host cell of invention in a suitable medium such that the potypeptid of the present invention is produced. Further, the method comprises isolating and/or recovering said polypeptid from the medium or the host cell.<br>
The polypeptide of the present invention is preferably produced by recombinant DNA techniques. For example, a nucleic acid molecule encoding the protein is cloned into an expression vector (as described above), the expression vector is introduced into a host cell (as described above) and said polypeptide is expressed in the host cell Said polypeptide can then be isolated from the cells by an appropriate purification scheme using standard protein purification techniques. Alternative to recombinant expression, the polypeptide or peptide of the present invention can be synthesized chemically using standard peptide synthesis techniques. Moreover, native Rpi-blb2 can be isolated from cells (e.g., endothelial cells), for example using the antibody of the present invention as described below, in particular, an anti-Rpi-blb2 antibody, which can be produced by standard techniques utilizing the polypeptid of the present invention or fragment thereof, i.e., the polypeptide of this invention.<br>
In one embodiment, the present invention relates to a Rpi-blb2 protein or a protein having Rpi-blb2 activity.<br>
In one embodiment, the present invention relates to a polypeptide having the amino acid sequence encoded by a polynucleotide of the invention or obtainable by a process of the invention.<br>
In one embodiment the polypeptide of the does not consist of the sequence depicted in Seq. ID NO.: 8 and/or 10 and/or does not consist of the sequence encoded by a nucleic acid molecule depicted in Seq. ID NO.: 7 and/or 9.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
In one embodiment, the polypeptiae of the present invention does not consist of the sequence of Mi1.1 or MI1.2 protein and/or of a protein encoded by a nucleic acid molecule encoding a Mil.l or MI1.2 protein.<br>
Thus, in one embodiment, the polypeptide of the present invention may not consist of the sequences shown in Rossi et al. 1998, PNAS USA 95:9750-9754, Milligan et at., 1998. Plant Cell 10:1307-1319; and/or WO 9806750.<br>
The terms "protein" and "polypeptide1' used in this application are interchangeable. "Polypeptide" refers to a polymer of amino acids (amino acid sequence) and does not refer to a specific length of the molecule. Thus peptides and oligopeptides are included within the definition of polypeptide. This term does also refer to or include post-transiationa) modifications of the polypeptide, for example, glycosylations, acetylations, phosphorylations and the like. Included within the definition are, for example, poly-peptides containing one or more analogs of an amino acid (including, for example, unnatural amino acids, etc.), polypeptides with substituted linkages, as well as other modifications known in the art, both naturally occurring and non-naturally occurring.<br>
Preferably, the polypeptide is isolated. An "isolated" or "purified" protein or biologically active portion thereof is substantially free of cellular material when produced by re-combinant DNA techniques, or chemical precursors or other chemicals when chemically synthesized.<br>
The language "substantially free of cellular material" includes preparations of the polypeptide of the invention in which the protein is separated from cellular components of the cells in which it is naturally or recombinantly produced. In one embodiment, the language "substantially free of cellular material" includes preparations having less than about 30% (by dry weight) of "contaminating protein", more preferably less than about 20% of "contaminating protein", still more preferably less than about 10% of "contaminating protein", and most preferably less than about 5% "contaminating protein". The term "Contaminating protein" relates to polypeptides which are not polypeptides of the present invention. When the polypeptide of the present invention or biologically active portion thereof is recombinantly produced, it is also preferably substantially free of culture medium, i.e., culture medium represents less than about 20%, more preferably less than about 10%, and most preferably less than about 5% of the volume of the protein preparation. The language "substantially free of chemical precursors or other chemicals" includes preparations in which subject of the present invention, e.g. the polypeptide of the present invention, is separated from chemical precursors or other chemicals which are involved in the synthesis of the protein. The language "substantially free of chemical precursors or other chemicals" includes preparations having less<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
length Rpi-blb2 protein or the full length protein which is homologous to a Rpi-blb2 protein depicted herein, and exhibit at least one activity of Rpi-blb2 protein. Typically, biologically (or immunological) active portions i.e. peptides, e.g., peptides which are, for example, 5,10,15, 20, 30f 35, 36, 37, 38, 39, 40, 50, 100 or more amino acids in length comprise a domain or motif with at least one activity or epitope of a Rpi-blb2 protein. Moreover, other biologically active portions, in which other regions of the poly-peptide are deleted, can be prepared by recombinant techniques and evaluated for one or more of the activities described herein.<br>
Manipulation of the Rpi-blb2 polynucleotide of the invention may result in the production of Rpi-blb2 having functional differences from the wild-type Rpi-blb2 protein. These proteins may be improved in efficiency or activity, may be present in greater numbers in the cell than is usual, or may be decreased in efficiency or activity.<br>
Any mutagenesis strategies for Rpi-blb2 to result in increased said resistance or a resistance to another plant pathogen species or an other strain of a plant pathogen species aforementioned, of said compound are not meant to be limiting; variations on these strategies will be readily apparent to one skilled in the art Using such strategies, and incorporating the mechanisms disclosed herein, the polynucleotide and polypep-tide of the invention may be utilized to generate plants or parts thereof," expressing wild type Rpi-blb2 or mutated Rpi-blb2 polynucleotide and protein molecules such that the yield, production, and/or efficiency of production of a desired compound is improved. This desired compound may be any natural product of plants, which includes the final products of biosynthesis pathways and intermediates of naturally-occurring metabolic pathways, as well as molecules which do not naturally occur in the metabolism of said cells, but which are produced by a said cells of the invention.<br>
The invention also provides chimeric or fusion proteins.<br>
As used herein, a "chimeric protein" or "fusion protein" comprises an polypeptide operatively linked to a non- Rpi-blb2 polypeptide.<br>
An "Rpi-blb2 polypeptide" refers to a polypeptide having an amino acid sequence corresponding to polypeptide having a Rpi-blb2 activity, whereas a "non-Rpi-blb2 polypeptide* refers to a polypeptide having an amino acid sequence corresponding to a protein which is not substantially homologous to the Rpi-blb2, e.g., a protein which does not confer the resistance described herein, in particular does not confer resistance to P. infestans and which is derived from the same or a different organism. Within the fusion protein, the term "operatively linked" is intended to indicate that the Rpi-blb2 polypeptide and the non-Rpi-blb2 polypeptide are fused to each other so that<br><br>
WO 2005/014631	PCT/F.P2004/0086R3<br>
both sequences fulfil the proposed function addicted to the sequence used. The non-Rpi-blb2 polypeptide can be fused to the N-terminus or C-terminus of the Rpi-blb2 polypeptide. For example, in one embodiment the fusion protein is a GST-LMRP fusion protein in which the RpM&gt;Ib2 sequences are fused to the C-terminus of the GST sequences. Such fusion proteins can facilitate the purification of recombinant Rpi-blb2. In another embodiment, the fusion protein is a Rpi-blb2 containing a heterologous signal sequence at its N-terminus. In certain host cells (e.g., mammalian host cells), expression and/or secretion of a Rpi-blb2 can be increased through use of a heterologous signal sequence.<br>
Preferably, a Rpi-blb2 chimeric or fusion protein of the invention is produced by starv dard recombinant DNA techniques. For example, DNA fragments coding for the different polypeptide sequences are ligated together in-frame in accordance with conventional techniques, for example by employing blunt-ended or stagger-ended termini for ligatlon, restriction enzyme digestion to provide for appropriate termini, filling-in of cohesive ends as appropriate, alkaline phosphatase treatment to avoid undesirable joining, and enzymatic ligation. The fusion gene can be synthesized by conventional techniques including automated DNA synthesizers. Alternatively, PCR amplification of gene fragments can be carried out using anchor primers which give rise to complementary overhangs between two consecutive gene fragments which can subsequently be annealed and reamplified to generate a chimeric gene sequence (see, for example, Current Protocols in Molecular Biology, eds. Ausubel et al. John Wiley &amp; Sons: 1992). Moreover, many expression vectors are commercially available that already encode a fusion moiety (e.g., a GST potypeptide). The polynucleotide of the invention can be cloned into such an expression vector such that the fusion moiety is linked in-frame to the encoded protein.<br>
Furthermore, folding simulations and computer redesign of structural motifs of the protein of the invention can be performed using appropriate computer programs (Ol-szewski, Proteins 25 (1996), 286-299; Hoffman, Comput. Appl. Biosci. 11 (1995), 675-679). Computer modelling of protein folding can be used for the conformational and energetic analysis of detailed peptide and protein models (Monge, J. Mol. Biol, 247 (1995), 995-1012; Renouf, Adv. Exp. Med. Biol. 376 (1995), 37-45). In particular, the appropriate programs can be used for the identification of interactive sites of mitogenic cyplin and its receptor, its ligand or other interacting proteins by computer assistant searches for complementary peptide sequences (Fassina, Immunomethods (1994), 114-120. Further appropriate computer systems for the design of protein and peptides are described in the prior art, for example in Berry, Biochem. Soc. Trans. 22 (1994), 1033-1036; Wodak, Ann. N. Y. Acad. Sci. 501 (1987), 1-13; Pabo, Biochemistry 25 (1986), 5987-5991. The results obtained from the above-described computer analysis<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
can be used for, e.g., the preparation of peptidomimetics of the protein of the invention or fragments thereof. Such pseudopeptide analogues of the, natural amino acid sequence of the protein may very efficiently mimic the parent protein (Benkirane, J. Biol. Chem. 271 (1996), 33218-33224). For example, incorporation of easily available achiral Q-amino acid residues into a protein of the invention or a fragment thereof results in the substitution of amide bonds by polymethylene units of an aliphatic chain, thereby providing a convenient strategy for constructing a peptidomimetic (Banerjee, Biopolymers 39 (1996), 769-777).<br>
Superactive peptidomimetic analogues of small peptide hormones in other systems are described in the prior art (Zhang, Biochem. Biophys. Res. Commun. 224 (1996), 327-331). Appropriate peptidomimetics of the protein of the present invention can also be identified by the synthesis of peptidomimetic combinatorial libraries through successive amide alkylation and testing the resulting compounds, e.g., for their binding and immu-nological properties. Methods for the generation and use of peptidomimetic combinatorial libraries are described in the prior art, for. example in Ostresh, Methods in Enzymol-ogy 267 (1996), 220-234 and Dorner, Bioorg. Med. Chem. 4 (1996), 709-715.<br>
Furthermore, a three-dimensional and/or crystallographic structure of the protein of the invention can be used for the design of peptidomimetic inhibitors of the biological activity of the protein of the invention (Rose, Biochemistry 35 (1996), 12933-12944; Rutenber, Bioorg. Med. Chem. 4 (1996)t1545-1558).<br>
In a further embodiment, the present invention relates to an antibody that binds specifically to the polypeptide of the present invention or parts, i.e. specific fragments or epi-topes of such a protein.<br>
The antibodies of the invention can be used to identify and isolate Rpi-blb2 and genes in any organism, preferably plants, prepared in plants described herein. These antibodies can be monoclonal antibodies, polyclonal antibodies or synthetic antibodies as well as fragments of antibodies, such as Fab, Fv "or scFv fragments etc. Monoclonal antibodies can be prepared, for example, by the techniques as originally described in Kdhler and Miistein, Nature 256 (1975), 495, and Galfr6. Meth. Enzymol. 73 (1981), 3, which comprise the fusion of mouse myeloma cells to spleen cells derived from immunized mammals.<br>
Furthermore, antibodies or fragments thereof to the aforementioned peptides can be obtained by using methods which are described, e.g., in Harlow and Lane "Antibodies, A Laboratory Manual", CSH Press, Cold Spring Harbor, 1988. These antibodies can be used, for example, for the immunoprecipitation and immunolocalization of proteins ac-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
cording to the invention as well as for the monitoring of the synthesis of such proteins, for example, in recombinant organisms, and for the identification of compounds interacting with the protein according to the invention. For example, surface plasmon resonance as employed in the BlAcore system can be used to increase the efficiency of phage antibodies selections, yielding a high increment of affinity from a single library of phage antibodies which bind to an epitope of the protein of the invention (Schier, Human Antibodies Hybridomas 7 (199(5), 97-105; Malmborg, J. Immunol. Methods 183 (1995), 7-13). In many cases, the binding phenomena of antibodies to antigens is equivalent to other ligand/anti-Iiganc binding.<br>
In one embodiment the present invention relates to an antisense nucleic acid molecule comprising the complementary sequence of the polypeptide of the present invention.<br>
Methods to modify the expression levels and/or the activity are known to persons skilled in the art and include for instance overexpression, co-suppression, the use of ribozymes, sense and anti-sense strategies, gene silencing approaches. "Sense strand" refers to the strand of a double-stranded DNA molecule that is homologous to a mRNA transcript thereof. The "anti-sense strand" contains an inverted sequence which is complementary to that of the "sense strand".<br>
An "antisense" nucleic acid molecule comprises a nucleotide sequence which is complementary to a "sense" nucleic acid molecule encoding a protein, e.g., complementary to the coding strand of a double-stranded cDNA molecule or complementary to an mRNA sequence. Accordingly, an antisense nucleic acid molecule can hydrogen bond to a sense nucleic acid mOieCuie. The sntiSens© nucleic acid molacule can be comple-mentary to an entire Rpi-blb2 coding strand, or to only a portion thereof. Accordingly, an antisense nucleic acid molecule can be antisense to a "coding region" of the coding strand of a nucleotide sequence of a polynucleotide of the present invention. The term "coding region" refers to the region of the nucleotide sequence comprising codons which are translated into amino acid residues. Further, the antisense nucleic acid molecule is antisense to a "noncoding region" of the coding strand of a nucleotide sequence encoding Rpi-blb2. The term "noncoding region" refers to 5' and 31 sequences which flank the coding region that are not translated into a polypeptide, i.e., also referred to as 51 and 31 untranslated regions (5'-UTR or 3'-UTR).<br>
Given the coding strand sequences encoding Rpi-blb2 disclosed herein, antisense nucleic acid molecules of the invention can be designed according to the rules of Watson and Crick base pairing. The antisens® nucleic acid molecule can be complementary to the entire coding region of Rpi-blb2 mRNA, but can also be an oligonucleotide which is antisense to only a portion of the coding or noncoding region of Rpi-blb2 mRNA. For<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
example, the antisense oligonucleotide can be complementary to the region surrounding the translation start site of Rpi-blb2 mRNA. An antisense oligonucleotide can be, for example, about 5f 10,15, 20, 25, 30, 35, 40, 45, or 50 nucleotides in length. An antisense nucleic acid molecule of the invention can be constructed using chemical synthesis and enzymatic ligation reactions using procedures known in the art For example, an antisense nucleic acid molecule (e.g., an antisense oligonucleotide) can be chemically synthesized using naturally occurring nucleotides or variously modified nucleotides designed to increase the biological stability of the molecules or to increase the physical stability of the duplex formed between the antisense and sense nucleic acids, e.g., phosphorothbate derivatives and acridine substituted nucleotides can be used. Examples of modified nucleotides which can be used to generate the antisense nucleic acid include 5-fluorouracil, 5-bromouracil, 5-chlorouracil, 5-iodouracil, hypoxan-thine, xanthine, 4-acetylcytosine, 5-(carboxyhydroxylmethyl) uracil, 5-carboxymethyl-aminomethyl-2-thiouridine, 5-carboxymethyiaminomethyluracil, dihydrouracil, beta-D-galactosylqueosine, inosine, NS-isopentenyladenine, 1-methylguanine, 1-methyl-inosine, 2,2-dimethyIguanine, 2-methyladenine, 2-methylguanine, 3-methylcytosine, 5-methylcytosine, N6-adenine, 7-methylguanine, 5-methylaminomethyluracil, 5-meth-oxyaminomethyl-2-thiouraciI, beta-D-mannosylqueosine, 5'-methoxycarboxymethyl-uracil, 5-methoxyuracil, 2-methylthio-N6-isopentenyladenine, uradl-5-oxyacetic acid (v), wybutoxosine, pseudouracil, queosine, 2-thiocytosine, 5-methyl-2-thiouracil, 2-thio-uracil, 4-thiouraciI, 5-methyluracil, uracil-5- oxyacetic acid methylester, uracil-5-oxy-acetic acid (v), 5-methyl-2-thiouracil, 3-(3-amino-3-N-2-carboxypropyl) uracil, (acp3)w, and 2,6-diaminopurine. Alternatively, the antisense nucleic acid can be produced biologically using an expression vector into which a polynucleotide has been subcloned in an antisense orientation (i.e., RNA transcribed from the inserted polynucleotide will be of an antisense orientation to a target polynucleotide of interest, described further in the following subsection).<br>
The antisense nucleic acid molecules of the invention are typically administered to a cell or generated in situ such that they hybridise with or bind to cellular mRNA and/or genomic DNA encoding a Rpi-blb2 to thereby inhibit expression of the protein, e.g., by inhibiting transcription and/or translation. The hybridisation can be by conventional nucleotide complementarity to form a stable duplex, or, for example, in the case of an antisense nucleic acid molecule which binds to DNA duplexes, through specific interactions in the major groove of the double helix. The antisense molecule can be modified such that it specifically binds to a receptor or an antigen expressed on a selected cell surface, e.g., by linking the antisense nucleic acid molecule to a peptide or an antibody which binds to a cell surface receptor or antigen. The antisense nucleic acid molecule can also be delivered to cells using the vectors described herein. To achieve sufficient intracellular concentrations of the antisense molecules, vector constructs in<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
which the antisense nucleic acid molecule is placed under the control of a strong pro karyotic, viral, or eukaryotic including plant promoters are preferred. .<br>
In a further embodiment, the antisense nucleic acid molecule of the invention can be an anomeric nucleic acid molecule. An a-anomeric nucleic acid molecule forms specific double-stranded hybrids with complementary RNA in which, contrary to the usual units, the strands run parallel to each other (Gaultier et al. (1987) Nucleic Acids. Res. 15:6625-6641). The antisense nucleic acid molecule can also comprise a 2(-o-methyl-ribonucleotlde (Inoue et al. (1987) Nucleic Acids Res. 15:6131-6148) or a chimeric RNA-DNA analogue (Inoue et al. (1987) FEBS Lett. 215:327-330).<br>
Further the antisense nucleic add molecule of the invention can be a ribozyme. Ribo- . zymes are catalytic RNA molecules with ribonuclease activity which are capable of cleaving a single-stranded nucleic acid, such as ah mRNA, to which they have a complementary region. Thus, ribozymes (e.g., hammerhead ribozymes (described in Haselhoff and Geriach (1988) Nature 334:585-591)) can be used to catalytically cleave Rpi-blb2 mRNA transcripts to thereby inhibit translation of mRNA. A ribozyme having specificity for a Rpi-blb2 -encoding nucleic acid molecule can be designed based upon the nucleotide sequence of a Rpi-blb2 cDNA disclosed herein or on the basis of a heterologous sequence to be isolated according to methods taught in this invention. For example, a derivative of a Tetrahymena L-19 IVS RNA can be constructed in which the nucleotide sequence of the active site is complementary to the nucleotide sequence to be cleaved in an encoding mRNA. See, e.g., Cech et al. U.S. Patent No. 4,987,071 and Cech et al. U.S. Patent No. 5,116,742. Alternatively, Rpi-blb2 mRNA can be used iu select a catalytic RNA having 3 specific ribonuclcase activity<br>
from a pool of RNA molecules. See, e.g., Bartel, D. and Szostak, J.W. (1993) Science 261:1411-1418.<br>
The antisense molecule of the present invention comprises also a polynucleotide comprising a nucleotide sequences complementary to the regulatory region of a Rpi-blb2 nucleotide sequence, e.g., Its promoter and/or enhancers, e.g. to form triple helical structures that prevent transcription of the gene in target cells. See generally, Helene, C. (1991) Anticancer Drug Des. 6(6):569-84; Helene, C. et al. (1992) Ann. N.Y. Acad. Sci. 660:27-36; and Maher, L.J. (1992) Bioassays 14(12):807-15.<br>
In addition, in one embodiment, the present invention relates to a method for the production of transgenic plants, plant cells or plant tissue comprising the introduction of the polynucleotide or the vector of the present invention into the genome of said plant, plant tissue or plant cell. In a preferred embodiment, said vector or said polynucleotide and a vector or a polynucleotide for the expression of a further resistance gene, in par-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
ticular for Rpi-blb, is also introduced into the genome of said plant, plant tissue or plant cell, before, after or together.<br>
For the expression of the nucleic acid molecules according to the invention in sense or antisense orientation in plant cells, the molecules are placed under the control of regulatory elements which ensure the expression in plant cells. These regulatory elements may be heterologous or homologous with respect to the nucleic acid molecule to be expressed as well with respect to the plant species to be transformed and are described above in detail.<br>
In general, such regulatory elements comprise a promoter active in plant cells. To obtain expression in all tissues of a transgenic plant, e.g. constitutive promoters are used, such as the 35 S promoter of CaMV (Odell, Nature 313 (1985), 810-812) or promoters of the polyubiquitin genes of maize (Christensen, Plant Mol. Biol. 18 (1982), 675-689). In order to achieve expression in specific tissues of a transgenic plant it is possible to use tissue specific promoters (see, e.g., Stockhaus, EMBO J. 8 (1989), 2245-2251). Known are also promoters which are specifically active in tubers of potatoes or in seeds of different plants species, such as maize, Vicia, wheat, barley etc. Inducible promoters may be used in order to be able to exactly control expression. Inducible promoters comprise also promoters, which are induced by infections of plants. Further embodiments are described above.<br>
In one embodiment, the present invention relates to a method for producing a plant or a part thereof resistant to a pathogen of the phylum Oomycetes comprising the steps: expressing in the plant or a part thereof the polypeptide of the present invention and a further resistance protein.<br>
Accordingly in one further embodiment, the present invention relates to transgenic plant or plant tissue of the invention or produced according to the method of the invention, which upon the presence of the polynudeotide or the vector is resistant to said pathogens.<br>
The generation of a transformed organism (or of a transformed cell or tissue) requires introducing the DNA, RNA, or protein in question into the relevant host cell. A multiplicity of methods are available for this procedure, which is termed transformation (or transduction or transfection) (Keown et al. (1990) Methods in Enzymology 185:527-537). For example, the DNA or RNA can be introduced directly by microinjec-tion or by bombardment with DNA-coated microparticles. Also, the cell can be perme-abilized chemically, for example using polyethylene glycbl, so that DNA can enter the cell by diffusion. The DNA can also be introduced by protoplast fusion with other DNA-<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
(EP 120 516; Hoekema. In: The Binary Plant Vector System, Offsetdrukkerij Kanters B.V., Alblasserdam, Chapter V; An et al. (1985) EMBO J 4:277-287). Various binary vectors are known, some of which are commercially available such as, for example, PBI101.2 or pBIN19 (Clontech Laboratories, Inc. USA).<br>
Further promoters which are suitable for expression in plants have been described (Rogers et al. (1987) Meth in Enzymol 153:253-277; Schardl et al. (1987) Gene 61:1-11; Berger et al. (1989) Proc Natl Acad Sci USA 86:8402-8406).<br>
Direct transformation techniques are suitable for any organism and cell type.<br>
The plasmid used need not meet any particular requirements in the case of the injection or electroporation of DNA or RNA into plant cells. Simple plasmids such as those of the pUC series can be used. If complete plants are to be regenerated from the transformed cells, it is necessary for an additional selectable marker gene to be located on the plasmid.<br>
Stably transformed cells, i.e. those which contain the introduced DNA integrated into the DNA of the host cell, can be selected from untransformed cells when a selectable marker is part of the DNA introduced. Examples of genes which can act as markers are all those which are capable of conferring resistance to antibiotics or herbicides (such as kanamycin, G 418, bleomycin, hygromycin or phosphinothricin) (see above). Transformed cells which express such marker genes are capable of surviving in the presence of concentrations of a corresponding antibiotic or herbicide which kill an untransformed wild type. Examples are mentioned above and preferably comprise the bar gene, which confers resistance to the herbicide phosphinothricin (Rathore KS et al. (1993) Plant Mol Biol 21(5):871-884), the nptll gene, which confers resistance to kanamycin, the hpt gene, which confers resistance to hygromycin, or the EPSP gene, which confers resistance to the herbicide Glyphosate. The selection marker permits the selection of transformed cells from untransformed cells (McCormick et al. (1986) Plant Cell Reports 5:81-84). The resulting plants can be bred and hybridised in the customary fashion. Two or more generations should be grown in order to ensure that the ge-nomic integration is stable and hereditary.<br>
The abovementioned methods are described, for example, in Jenes B et al.(1993) Techniques for Gene Transfer, in: Transgenic Plants, Vol. 1, Engineering and Utilization, edited by SD Kung and R Wu, Academic Press, pp. 128-143 and in Potrykus (1991) Annu Rev Plant Physiol Plant Moiec Biol 42:205-225). The construct to be expressed is preferably cloned into a vector which is suitable for the transformation of<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
formation, electroporation of partially permeabilized cells, introduction of DNA using glass fibers, etc.<br>
The term "transformation1' as used herein, refers to the transfer of an exogenous polynudeotide into a host cell, irrespective of the method used for the transfer. The polynudeotide may be transiently or stably introduced into the host cell and may be maintained non-integrated, for example, as a plasmid or as chimeric links, or alternatively, may be integrated into the host genome. The resulting transformed plant cell can then be used to regenerate a transformed plant in a manner known by a skilled person.<br>
Accordingly, in one embodiment, the present invention relates to a plant cell comprising the polynudeotide the vector of the present invention or obtainable by the method of the present invention. Preferably, the cell comprises a further resistance conferring polynudeotide or vector, more preferred is a Rpi-blb encoding vector or polynucleotide.<br>
Thus, the present invention relates also to transgenic plant cells which contain (preferably stably integrated into the genome) a polynucleotide according to the invention linked to regulatory elements which allow expression of the polynucleotide in plant cells and wherein the polynucleotide is foreign to the transgenic plant cell. For the meaning of foreign; see supra.<br>
Thus, the present invention also relates to transgenic plants and plant tissue comprising transgenic plant cells according to the invention. Due to the (over)expression of a polypeptide of the invention, said plant or plant tissues are resistance to plant pathogens, in particular to Oomycetes, Preferably the plants are also resistance to other pathogen, e.g. to sucking plant pathogens. Further pathogens are described herein. Preferred is that said plants or plant tissue is resistant to Phytophthora species, most preferred to P. infestans.<br>
For example, to obtain transgenic plants expressing the Rpi-blb2 gene, its coding region can be cloned, e.g., into the pBinAR vector (H6fgen und Willmitzer, Plant-Science, 66, 1990, 221-230). For example, following a polymerase chain reaction (PCR) technology the coding region of Rpi-blb2 can be amplified using Primers as shown in the examples and figures, e.g., in Table 3b in particular ARF1F and ARF1R. The obtained PCR fragment can be purified and subsequently the fragment can be cloned into a vector, the resulted vector can be transferred into Agrobacterium tumefaciens. This strain can be used to transform and transgenic plants can then be selected. In another embodiment, the present invention relates to a transgenic plant or plant tissue comprising the plant cell of the present invention.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
"Transgenic", for example regarding a nucleic acid sequence, an expression cassette or a vector comprising said nucleic add sequence or an organism transformed with said nucleic acid sequence, expression cassette or vector, refers to all those constructs originating by recombinant methods in which either<br>
a)	the Rpi-blb2 nucleic acid sequence, or<br>
b)	a genetic control sequence Finked operably to the Rpi-blb2 nucleic acid se<br>
quence, for example a promoter, or<br>
c)	(a) and (b)<br>
are not located in their natural genetic environment or have been modified by recombinant methods, an example of a modification being a substitution, addition, deletion, inversion or insertion of one or more nucleotide residues. Natural genetic environment refers to the natural chromosomal locus in the organism of origin, or to the presence in a genomic library. In the case of a genomic library, the natural genetic environment of the nucleic add sequence is preferably retained, at least in part. The environment flanks the nucleic acid sequence at least at one side and has a sequence of at least 50 bp, preferably at least 500 bp, especially preferably at least 1000 bp, very especially preferably at least 5000 bp, in length. A naturally occurring expression cassette - for example the naturally occurring combination of the Rpi-blb2 promoter with the corresponding Rpi-blb2 gene - becomes a transgenic expression cassette when it is modi-fied by non-natural, synthetic 'artificial' methods such as, for example, mutagenization. Such methods have been describee! (US 5,565,350, WO 00/15315; also 566 above).<br>
Further, the plant cell, plant tissue or plant can also be transformed such that further enzymes and proteins are (over)expressed which expression supports an increase of the plant's or the plant tissue's resistance, for example Rpi-bib (synonyms Rpi-blbi, RB or Sbu1), R1, Rpi-mcd. R-ber (synonym R12), Rpi1, Rpi-blb3, Rpi-ABPT1, R2, R3a or R3b, R4, R5, R6, R7, R8, R9, R10, R11, Ph-1, Ph-2 and/or Ph-3-proteins. Preferred is the coexpression of Rpi-bib and Rpi-blb2.<br>
The present invention also relates to cultured plant tissues comprising transgenic plant cells as described above which shew expression of a protein according to the invention.<br>
Host or starting organisms which are preferred as transgenic organisms are mainly plants in accordance with the above definition. Included within the scope of the invention are all genera and species of higher and lower plants of the Plant Kingdom. Furthermore included are the mature plants, seed, shoots and seedlings, and parts,<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
propagation material and cultures derived there from, for example cell cultures which have an increased Rpi-blb2 activity. Mature plants refers to plants at any developmental stage beyond that of the seedling. The term seedling refers to a young immature plant in an early developmental stage.<br>
Any transformed plant obtained according to the invention can be used in a conventional breeding scheme or in in vitro plant propagation to produce more transformed plants with the same characteristics and/or can be used to introduce the same characteristic in other varieties of the same or related species. Such plants are also part of the invention. Seeds obtained from the transformed plants genetically also contain the same characteristic and are part of the invention. As mentioned before, the present invention is in principle applicable to any plant and crop that can be transformed with any of the transformation method known to those skilled in the art<br>
In general, the plants which can be modified according to the invention and which either show overexpression of a protein according to the invention or a reduction of the synthesis of such a protein can be derived from any desired plant species. They can be monocotyledonous plants or dicotyledonous plants, preferably they belong to plant species of interest in agriculture, wood culture or horticulture interest, such as crop plants (e.g. maize, rice, barley, wheat, rye, oats etc.), potatoes, oil producing plants (e.g. oilseed rape, sunflower, pea nut, soy bean, etc.), cotton, sugar beet, sugar cane, leguminous plants (e.g. beans, peas etc.), wood producing plants, preferably trees, etc. However, plants which can be infected by Phytophthora species are preferred.<br>
Accordingly, in one embodiment the plant, plant cell or plant tissue of the invention or produced according to the method of the invention is selected from the group consisting of Menyanthaceae, Solanaceae, Sclerophylacaceae, Duckeodendraceae, Goetze-aceae, Convolvulaceae, Cuscutaceae, Polemoniaceae, and Hydrophyllaceae according to the Systema Naturae 2000, Brands, S.J., Amsterdam or has its origin thereof. Preferably said plant, plant cell or plant tissue of the invention or produced according to the method of the invention is  a Solanaceae, preferably selected from the group of Atropa, Browallia, Brunfelsia, Capsicum, Cestrum, Cyphomandra, Datura, Fabiana, Franciscea, Hyoscyamus, Lycium, Mandragora, Nicandra, Nicotiana, Petunia, Physalis, Schizanthus and Solanum according to the Systema Naturae 2000, Brands, S.J., Amsterdam or has its origin thereof.<br>
More preferred, the plant, plant cell or plant tissue of the invention or produced according to the method of the present invention is a S. bulbocastanum, S. tuberosum (potato), S. lycopersicum (tomato), petunia, S, betaceum (tree tomato), S. muricatum (pear melon) or S.melongena (eggplant). Even more preferred, the plant, plant tissue<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
or plant cell is a S. tuberosum or S. lycopersicum. Most preferred is S. tuberosurn. In other systems, the classification will be similar. The person skilled in the art knows the differences, e.g. more common, tomato is named systematically Lycopersicon lycopersicum (L) Karsten ex Fan/veil.<br>
In yet another aspect, the inventiori also relates to harvestable parts and to propagation material of the transgenic plants according to the invention which either contain transgenic plant cells expressing a nucleic acid molecule and/or the polypeptide according to the invention or which contains cells which show an increased level of the polypeptide of the invention.<br>
Harvestable parts can be in principle any useful parts of a plant, for example, flowers, pollen, seedlings, tubers, leaves, stems, fruit, seeds, roots etc. Propagation material includes, for example, seeds, fruits, cuttings, seedlings, tubers, rootstocks etc. Preferred are potatoes, tomatoes, eggfruits or pear melons as harvestable or propagation material. In case, the plant of the invention is petunia, the present invention relates in one embodiment to the flowers of petunia as harvestable part.<br>
The invention furthermore relates to the use of the transgenic organisms according to the invention and of the cells, cell cultures, parts - such as, for example, roots, leaves and the like in the case of transgenic plant organisms - derived from them, and to transgenic propagation material such as seeds or fruits, for the production of foodstuffs or feeding stuffs, Pharmaceuticals or fine chemicals. In particular, potatoes can serve for the production of fine chemicals.<br>
Accordingly in another embodiment, the present invention relates to the use of the polynucleotide, the plant, plant cell or plant tissue, the vector, or the polypeptide of the present invention for making fatty acids, carotenoids, isoprenoids, vitamins, lipids, wax esters, (poly)saccharides and/or polyhydroxyalkanoates, and/or its metabolism products, in particular, steroid hormones, cholesterol, prostaglandin, triacylgiycerols, bile acids and/or ketone bodies producing cells, tissues and/or plants. There are a number of mechanisms by which the yield, production, and/or efficiency of production of fatty acids, carotenoids, isoprenoids, vitamins, wax esters, lipids, (poly)saccharides and/or polyhydroxyalkanoates, and/or its metabolism products, in particular, steroid hormones, cholesterol, triacylglycerols, prostaglandin, bile acids and/or ketone bodies or further of above defined fine chemicals incorporating such an altered protein can be affected. In the case of plants, by e.g. increasing the expression of acetyl-CoA which is the basis for many products, e.g., fatty acids, carotenoids, isoprenoids, vitamins, lipids, (poly)saccharides, wax esters, and/or polyhydroxyalkanoates, and/or its metabolism products, in particular, prostaglandin, steroid hormones, cholesterol, triacylglycerols,<br><br>
VO 2005/014631	PCT/EP2004/008683<br>
bile acids and/or ketone bodies in a cell, it may be possible to increase the amount of the produced said, compounds thus permitting greater ease of harvesting and purification or in case of plants more efficient partitioning. Further, one or more of said metabolism products, increased amounts of the cofactors, precursor molecules, and intermediate compounds for the appropriate biosynthetic pathways maybe required. Therefore, by increasing the number and/or activity of transporter proteins involved in the import of nutrients, such as carbon sources (i.e., sugars), nitrogen sources (i.e., amino acids, ammonium salts), phosphate, and sulphur, it may be possible to improve the production of acetyl CoA and its metabolism products as mentioned above, due to the removal of any nutrient supply limitations on the biosynthetic process. In particular, it may be possible to increase the yield, production, and/or efficiency of production of said compounds, e.g. fatty adds, carotenoids, isoprenoids, vitamins, was esters, lipids, (poiy)saccharides, and/or pofyhydroxyalkanoates, and/or its metabolism products, in particular, steroid hormones, cholesterol, prostaglandin, triacylgfycerols, bile acids and/or ketone bodies molecules etc. in plants.<br>
Furthermore preferred is a method for the recombinant production of Pharmaceuticals or fine chemicals in host organisms, wherein a host organism is transformed with one of the above-described expression cassettes and this expression cassette comprises one or more structural genes which encode the desired fine chemical or catalyse the biosynthesis of the desired fine chemical, the transformed host organism is cultured, and the desired fine chemical is isolated from the culture medium. This method can.be applied widely to fine chemicals such as enzymes, vitamins, amino acids, sugars, fatty acids, and natural and synthetic flavorings, aroma substances and colorants. Especially preferred is the production of tocopherols and tocotrienols and carotenoids. The transformed host organisms are cultured and the products are isolated from the host organisms or the culture medium by methods known to the skilled worker. The production of Pharmaceuticals such as, for example, antibodies or vaccines, is described by Hood EE, Jilka JM. Curr Opin Biotechnol. 1999 Aug; 10(4):382-6; Ma JK, Vine ND. Curr Top Microbiol Immunol. 1999; 236:275-92.<br>
In one embodiment, the present invention also relates to the use of the polyhucieotide, the vector, or the polypeptide of the present invention for producing a plant or a plant tissue, plant organ, or a plant cell or a part thereof resistant to said.<br>
Furthermore, in one embodiment, the present invention relates to a method for the identification of a compound stimulating resistance to a said plant pathogen comprising:<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
a)	contacting cells which express the polypeptide of the present invention or its<br>
mRNA with a candidate compound under cell cultivation conditions;<br>
b)	assaying an increase in expression of said polypeptide or said mRNA;<br>
c)	comparing the expression level to a standard response made in the absence of<br>
said candidate compound; whereby, an increased expression over the standard<br>
indicates that the compound is stimulating resistance.<br>
Said compound may be chemically synthesized or microbiologically produced and/or comprised in, for example, samples, e.g., cell extracts from, e.g., plants, animals or microorganisms, e.g. pathogens. Furthermore, said compound(s) may be known in the art but hitherto not known to be capable of suppressing or activating Rpi-blb2. The reaction mixture may be a cell free extract or may comprise a cell or tissue culture. Suitable set ups for the method of the invention are known to the person skilled in the art and are, for example, generally described in Alberts et a!., Molecular Biology of the Cell, third edition (1994), in particular Chapter 17. The compounds may be, e.g., added to the reaction mixture, culture medium, injected into the cell or sprayed onto the plant.<br>
If a sample containing a compound is identified in the method of the invention, then it is either possible to isolate the compound from the original sample identified as containing the compound capable of activating or increasing resistance to said pathogens, or one can further subdivide the original sample, for example, if it consists of a plurality of different compounds, so as to reduce the number of different substances per sample and repeat the method with the subdivisions of the original sample. Depending on the complexity of the samples, the steps described above can be performed several times, preferably until the sample identified according to the method of the invention oniy comprises a limited number of or only one substance(s). Preferably said sample comprises substances of similar chemical and/or physical properties, and most preferably said substances are identical. Preferably, the compound identified according to the above-described method or its derivative is further formulated in a form suitable for the application in plant breeding or plant cell and tissue culture.<br>
The compounds which can be tested and identified according to a method of the invention may be expression libraries, e.g., cDNA expression libraries, peptides, proteins, nucleic acids, antibodies, small organic compounds, hormones, peptidomimetics, PNAs or the like (Milner, Nature Medicine 1 (1995), 879-880; Hupp, Cell 83 (1995), 237-245; Gibbs, Cell 79 (1994), 193-198 and references cited supra). Said compounds can also be functional derivatives or analogues of known inhibitors or activators. Methods for the preparation of chemical derivatives and analogues are well known to those skilled in the art and are described in, for example, Beilstein, Handbook of Organic Chemistry, Springer edition New York Inc., 175 Fifth Avenue, New York, N.Y. 10010 U.SA and<br><br>
WO 2005/014631	PCT7EP2004/00**«<br>
Organic Synthesis, Wiley, New York, USA. Furthermore, said derivatives and analogues can be tested for their effects according to methods known in the art. Furthermore, peptidomimetics and/or computer aided design of appropriate derivatives and analogues can be used, for example, according to the methods described above. The cell or tissue that may be employed in the method of the invention preferably is a host cell, plant cell or plant tissue of the invention described in the embodiments hereinbefore.<br>
Determining whether a compound is capable of suppressing or activating said resistance can be done, as described in the examples, in particular via sporulation index determination. The activator identified by the above-described method may prove useful as a fungicide or crop protectant Thus, in a further embodiment the invention relates to a compound obtained or identified according to the method of the invention said compound being an agonist of Rpi-blbZ<br>
Accordingly, in one embodiment, the present invention further relates to a compound identified by the method of the present invention.<br>
Said compound is, for example, a homologue of Rpi-blb2. Homologues of the polypep-tid of the present invention can be generated by mutagenesis, e.g., discrete point mutation or truncation of Rpi-blb2. As used herein, the term "homologue" refers to a variant form of the protein which acts as an agonist of the activity of the Rpi-blb2. An agonist of said protein can retain substantially the same, or a subset, of the biological activities of Rpi-blb2.<br>
In one embodiment, the invention relates to an antibody specifically recognizing the compound of the present invention.<br>
The invention also relates to a diagnostic composition comprising at least one of the aforementioned polynucleotides, nucleic acid molecules, vectors, proteins, antibodies or compounds of the invention and optionally suitable means for detection.<br>
The diagnostic composition of the present invention is suitable for the isolation of mRNA from a cell and contacting the mRNA so obtained with a probe comprising a nucleic acid probe as described above under hybridising conditions, detecting the presence of mRNA hybridised to the probe, and thereby detecting the expression of the protein in the cell. Further methods of detecting the presence of a protein according to the present invention comprises immunotechniques well known in the art, for example enzyme linked immunosorbent assay. Furthermore, it is possible to use the nucleic<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
acid molecules according to the invention in particular the markers described in the examples, e.g. in table 3a or 3b as molecular markers or primer in plant breeding.<br>
Suitable means for detection are well known to a person skilled in the art, e.g. buffers and solutions for hybridisation assays, e.g. the aforementioned solutions and buffers, further and means for Southern-, Western-, Northern- etc. -blots, as e.g; described in Sambrook et al. are known.<br>
In another.embodiment, the present invention relates to a kit comprising the poiynucieotide, the vector, the host cell, the polypeptide, the antisense nucleic acid, the antibody, plant pell, the plant or plant tissue, the harvestable part, the propagation material or the compound of the invention.<br>
The compounds of the kit of the present invention may be packaged in containers such as vials, optionally with/in buffers and/or solution. If appropriate, one or more of said components may be packaged in one and the same container. Additionally or alternatively, one or more of said components may be adsorbed to a solid support as, e.g. a nitrocellulose filter, a glass plate, a chip, or a nylon membrane or to the well of a mi-crotiterplate. The kit can be used for any of the herein described methods and embodiments, e.g. for the production of the host cells, transgenic plants, pharmaceutical compositions, detection of homologous sequences, identification of antagonists or agonists, etc.<br>
Further, the kit can comprise instructions for the use of the kit for any of said embodiments, in particular for its use for increasing the resistance to one or more of said pathogens of a plant cell, plant tissue or plant.<br>
In a preferred embodiment said kit comprises further a poiynucieotide encoding one or more of the aforementioned resistance protein(s), preferably Rpi-blb, and/or an antibody, a vector, a host cell, an antisense nucleic acid, a plant cell or plant tissue and/or a plant related to said resistance protein(s), preferably to Rpi-blb.<br>
In a further embodiment, the present invention relates a method for the production of a crop protectant providing the poiynucieotide, the vector or the polypeptide of the invention or comprising the steps of the method of the invention; and formulating the poiynucieotide, the vector or the polypeptide of the invention or the compound identified in step (c) of said method in a form applicable as plant agricultural composition.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
In another embodiment, the present invention relates to a method for the production of a crop protectant composition comprising the steps of the method of the present invention; and<br>
(a)     formulating the compound identified in step (c) in a form acceptable as agricultural composition.<br>
Under "acceptable as agricultural composition" is understood, that such a composition is in agreement with the laws regulating the content of fungicides, plant nutrients, herbicides, etc. Preferably such a composition is without any harm for the protected plants and the animals (humans included) fed therewith.<br>
The present invention also pertains to several embodiments relating to further uses and methods. The polynucleotide, pofypeptide, protein homologues, fusion proteins, primers, vectors, host cells, described herein can be used in one or more of the following methods: identification of plants resistant to plant pathogens as mentioned and related organisms; mapping of genomes; identification and localization of sequences of interest; evolutionary studies; determination of regions required for function; modulation of an activity.<br>
Accordingly, the polynucleotides of the present invention have a variety of uses. First, they may be used to identify an organism as. being S. bulbocastanum or a close relative thereof. Also, they may be used to identify the presence of S. bulbocastanum or a relative thereof in a mixed population of plants. By probing the extracted genomic DNA of a culture of a unique or mixed population of plants under stringent conditions with a probe spanning a region of the gene of the present invention which is unique to this S. bulbocastanum, one can ascertain whether the present invention has been used or whether S. bulbocastanum or a relative, e.g. a close relative, is. present.<br>
Further, the polynucleotide of the invention may be sufficiently homologous to the sequences of related species such that these nucleic acid molecules may serve as markers for the construction of a genomic map in related organism.<br>
The polynucleotides of the invention are also useful for evolutionary and protein structural studies. By comparing the sequences of the Rpi-blb2 of the present invention to those encoding similar enzymes from other organisms, the evolutionary relatedness of the organisms can be assessed. Similarly, such a comparison permits an assessment of which regions of the sequence are conserved and which are not, which may aid in determining those regions of the protein which are essential for the functioning of the enzyme. This type of determination is of value for protein engineering studies and may<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
give an indication of what the protein can tolerate in terms of mutagenesis without losing function.<br>
These and other embodiments are disclosed and encompassed by the description and examples of the present invention. Further literature concerning any one of the methods, uses and compounds to be employed in accordance with the present invention may be retrieved from public libraries, using for example electronic devices. For example the public database "Medline" may be utilized which is available on the Internet, for example under hftp7/www.ncbi.nlm.nih.gov/PubMed/medline.html. Further databases and addresses, such as hftp"7/www.ncbi.nlm.nih.gov/, hftp://www.infobiogen. fr/, hftp://viww.fmi.ch/biology/research"tools.html, hftp://www.tigr.org/I are known to the person skilled in the art and can also be obtained using, e.g., hftp://www.lycos.com. An overview of patent information in biotechnology and a survey of relevant sources of patent information useful for retrospective searching and for current awareness is given in Berks, TIBTECH 12 (1994), 352-364.<br>
Tables:<br>
Table 1: Sequences:<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
The figures show:<br>
Figure 1. Schematic representation of the development of the complex interspecific hybrid clones designated as 'ABPT (1a) and the S. tuberosum mapping populations that were derived from two of these clones: ABPT clone 55 and ABPT clone 60 (1b to d). A; Solanum acaule, B; S. bulbocastanum, P; S. pureja, T; S. tubersosum, 2x ; diploid (2n=2x=24), 3x; triploid, 4x; tetraploid, 6x; hexaploid, cv; cultivar. Codes in italics indicate mapping populations.<br>
Figure 2. Disease progress curves for clone ARF 87-601 and susceptible control culti-vars (cv) Bildtstar, Eersteling and the partial resistant control cultivar Pimpernel in a field test for foliar resistance to late blight in Toluca Valley, Mexico in 1991. At eight time points after planting, the percentage-blighted foliage due to a natural late blight infection was scored on the 1 to 9 CIP scale (Estrada-Ramos, 1983).<br>
Figure 3. Disease progress curves for clone ARF 87-507, ARF 87-601, ARF 87-801, the susceptible control cultivar (cv) Granola and the partial resistant breeding clone AR 85-96-13 in a field test for foliar resistance to late blight in Benguet Province, Philippines in 1992. At six time points between August 25th to November 24th, the percentage-blighted foliage due to a natural late blight infection was scored on the 1 to 9 CIP scale (Estrada-Ramos, 1983).<br>
Figure 4. Typical phenotypes in tetraploid resistant and susceptible parental clones and progeny clones segregating for Rpi-blb2 mediated resistance to late blight in the annual field trial at Marknesse, The Netherlands, approximately 6 weeks after inoculation with isolate IPO82001 of P. infestans. Six plant plots with a clone showing the resistant phenotype (within black solid line) that shows no or hardly any sporulating lesions and with a clone showing the susceptible phenotype (within white dotted line) that shows completely blighted foliage.<br>
Figure 5. Genetic map based on 109 progeny clones of S. tuberosum mapping population ARG 95-15 showing 7 AFLP markers that were found to cosegregate with the Rpi-blb2 locus. Numbers left to the vertical line indicate the genetic distance between flanking markers or the Rpi-blb2 locus in centimorgan (cM).<br>
Figure 6. Genetic map based on 137 progeny clones of S. tuberosum mapping population ARG 95-3 showing 15 AFLP markers and RGA marker S1E00 that were found to cosegregate with the Rpi-blb2 locus. Phenotypes of the progeny clones were obtained with detached leaf assays. Numbers left to the vertical line indicate the genetic distance between flanking markers or the Rpi-blb2 locus in centimorgan (cM).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
Figure 7. Genetic map based on 178 progeny clones of S, tuberosum mapping population ARG 95-3 showing 5 markers that were found to cosegregate with the Rpi-blb2 locus on linkage group 6 of S. tuberosum. Phenotypes of the progeny clones were determined in the field trial at Marknesse, the Netherlands in 1998. Markers E40M58 and E46M52 were scored either as AFLP, CAPS, SCAR or extended (suffix: e) marker (table 3A). Partly, marker CT119 was scored as marker CT119N (table 3a). Marker CT216 was scored as SCAR marker. The number left to the vertical line indicates the genetic distance between flanking markers or the Rpi-blb2 locus in centimorgan (cM). For each marker, the number of recombinants between marker and phenotype and the total number of progeny clones scored is given in parenthesis.<br>
Figure 8. Genetic maps based on 886 progeny clones of S. tuberosum mapping population ARG 95-3 and on 170 progeny clones of S. tuberosum mapping population ARP 96-11, showing markers that were found to cosegregate with the Rpi-blb2 locus on linkage group 6 of S. tuberosum. Phenotypes of the progeny clones were determined in the field trial at Marknesse, the Netherlands in 2000. The number left to the vertical line indicates the genetic distance between flanking markers in centimorgan (cM). The marker interval which delimitates the position of the Rpi-blb2 gene, based on detected recombination events in progeny clones, is indicated by double arrow headed lines.<br>
Figure 9. Physical map of the genomic region containing Rpi-blb2 in S. tuberosum (upper horizontal line) and S. bulbocastanum (lower horizontal line). Vertical lines indicate the relative position of markers linked to resistance. Numbers above the horizontal lines are the number of recombinants identified between the flanking markers in 1056 and 1899 progeny plants of S. tuberosum, derived from complex species hybrids "ABPT" (Figure 1), and S. bulbocastanum progeny plants respectively. ABPT-derived progeny comprises clones from both the mapping populations ARG 95-3 and ARP 96-11. Rectangles represent bacterial artificial chromosome (BAC) clones from the ARD 1197-16 BAC library except for BAC clones with prefix "Bib" which were from the S. bulbocastanum Bib 2002 BAC library. The marker interval which delimitates the position of the Rpi-blb2 gene, based on detected recombination events in progeny clones, is indicated by double arrow headed lines. Small arrows indicate positions of Resistance Gene Candidates (RGC's).<br>
Figure 10. Schematic representation of the development of the diploid, intraspecifc mapping population B6 of S. bulbocastanum. Codes in italics indicate mapping populations.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
Figure 11. Genetic map based on 1899 progeny clones of S. bulbocastanum mapping population B6, showing markers that were found to cosegregate with the Rpi-blb2 locus on chromosome 6 of S. bulbocastanum. Phenotypes of the progeny clones were determined by detached leaf assays. The number left to the vertical line indicates the genetic distance between flanking markers in centimorgan (cM). The marker interval which delimitates the position of the Rpi-blb2 gene, based on detected recombination events in progeny clones, is indicated by a double arrow headed line.<br>
Figure 12. Genetic complementation for late blight susceptibility. Typical disease phenotypes of potato (S. tuberosum) leaves, 6 days after inoculation with a sporangiospore suspensions of P. infestans isolate 655-2A. Leaf derived from kanamycin resistant cv Kondor plants transformed with pBINPLUS (control; A)? leaves derived from cv Kondor plants harbouring BAG SPB39 derived (B) or BAC 211 derived RGC5 (C), leaf derived from kanamycin resistant cv Impala plants transformed with pBINPLUS (control; D), leaves derived from cv Impala plants harbouring BAC SPB39 derived (E) or BAC 211 derived RGC5 (F). Panels A and D depict typical susceptible responses with extensive sporulating lesions of P. infestans. Panels B, C, E and F depict typical resistance reactions observed at the sites of inoculation on transgenic potato plants harbouring Rpi-blb2.<br>
Figure 13, Nucleic acid sequences coding for the Rpi-blb2 gene. A. Coding nucleic acid sequence of the Rpi-blb2 gene. B. Coding nucleic acid sequence of the Rpi-blb2 gene including the intron sequence (position 43-128). C. Sequence of the 7967 bp Sau3AI genomic DNA fragment of ARD 1197-16 BAC 211 present in p211F-C12, one of the two the genetic constructs used for genetic complementation for late blight resistance. The genomic fragment harbours the Rpi-blb2 gene including natural regulatory elements necessary for correct expression of the gene. The initiation codon (ATG position 1546-1548) and the termination codon (TAG position 5433-5435) are underlined. D. Sequence of the 9949 bp Sau3AI genomic DNA fragment of S. bulbocastanum 2002 BAC BlbSP39 present in pSP39-20, one of the two the genetic constructs used for genetic complementation for late blight resistance. The genomic fragment harbours the Rpi-blb2 gene including natural regulatory elements necessary for correct expression of the gene. The initiation codon (ATG position 1413-1415) and the termination codon (TAG position 5300-5303) are underlined.<br>
Figure 14. Putative Rpi-blb2 gene structure and deduced Rpi-blb2 protein sequence. A. Schematic representation of the Rpi-blb2 gene structure. Horizontal lines indicate exons. Open boxes represent coding sequence. Lines angled downwards indicate the positions of intron sequences. B. Deduced Rpi-blb2 protein sequence. The amino acid sequence deduced from the DNA sequence of Rpi-blb2 is divided into three domains<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
(Figure 1a) with late blight (LB) susceptible S. tuberosum cultivar Oberambacher Friihe as first parent and S. tuberosum cultivars Arkula (Figure 1b) and Blanka (Figure 1c) respectively as second parents. Similarly, BC2-clone ARF 87-601 was obtained by successive crossing on ABPT-clone 60 with LB susceptible S. tuberosum cultivars Alcmaria and Blanka (Figure 1d).<br>
Clone ARF 87-601 was tested as part of a field test for screening of LB-resistance in the Toluca area in Mexico in 1991. A plot of clone ARF 87-601 with seven plants was evaluated in comparison to plots with nine plants each of the control cultivars Bildtstar, Eersteling and Pimpernel. According to the ratings for resistance to late blight in the Dutch National list of recommended potato cultivars of 1988, these control cultivars scored 3, 3 and 8 respectively on a scale from 3 to 8 of increasing resistance. Cultivar Pimpernel is considered as a source of partial resistance (Colon et al., 1985). About forty days after planting, natural infection by P. infestans established. The development of LB in the foliage then was monitored eight times during the period from July 16th to September 2nd (Figure 2). There was a clear difference between the disease progress curves for ARF 87-601 in comparison to the control cultivars. At 74 days after planting, foliage of the control cultivars was completely or nearly completely blighted whereas clone ARF 87-601 showed no visible symptoms (Figure 2). Clones ARF 87-507, ARF 87-801 and again clone ARF 87-601 showed comparable results in a field test for screening of LB-resistance in the Benguet Province of the Philippines in 1992 (Figure 3). Ten plants each of the three BC2 clones, control cultivar Granola and the moderately LB resistant breeding clone AR 85-96-13, which was used as female parent to obtain AR 92-1197 (Figure 1d), were planted on August 25th. The percentage of blighted foliage was scored six times after occurrence of natural infection by P. infestans. Disease progress curves of ABPT derived BC2-clones were markedly different when compared to cultivar Granola and clone AR 85-96-13 (Figure 3). BC2-clones showed no or little LB symptoms and no clear disease progress during the scoring period whereas cultivar Granola had almost completely blighted foliage at the third scoring date.<br>
Clones ARF 87-601, ARF 87-507 and ARF 87-801 were used for further backcrossing with LB susceptible cultivars and breeding clones of S. tuberosum (Figure 1b to 1d). This breeding work resulted in four different mapping populations, tetraploid BC3-population ARG 95-15, tetraploid BC4-populations ARG 95-3 and ARP 96-11 and dip-loid BC4-population DP1. During the successive steps of this breeding work resistant clones ARF 87-507, ARF 87-601, ARF 87-801, AR 91-1263, AR 91-1292 and AR 92-1197 were selected on the basis of agronomic performance in common practice breeding evaluations as well as by screening their parents and relevant progenies in a field trial at Marknesse, the Netherlands, that was inoculated with the complex isolate<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
IPO82001 of P. infestans. The diploid (2n=2x=24) clone ARD 1197-16 was selected ampng the progeny of cross AR 92-1197 x Phu 81-101 (Figure 1d), the latter parental clone being known for its capacity to induce parthenogenic seed set in the female parent (Hermsen and Verdenius, 1973). Initially, resistance to LB in ARD 1197-16 was found after repeated detached leaf assays using P. infestans isolates IPO82001, IPO655-2A and IPO428-2 and verified in a field trial in 1998 at Marknesse. The diploid status of clone ARD 1197-16 was confirmed by flow cytometry (Plant cytometry services, Schijndel, the Netherlands).<br>
Clear segregation for the LB-resistance trait in ABPT-derived progeny and mapping populations was observed during successive years of field testing at the trial site of Marknesse, approximately 6 weeks after inoculation with isolate IPO82001 of P. infestans. Typically, resistant clones showed no or hardly any sporulating lesions whereas susceptible clones showed completely blighted foliage (Figure 4) In 2000t a total of 2851 clones from the mapping populations ARG 95-3 and ARP 96-11 were screened as single plant plots. On average, 24 percent of the clones showed pheno-types that could not unambiguously be classified as resistant or susceptible. Clones that could be classified as such showed segregation ratio's of resistant to susceptible phenotypes of 1 to 1 and 1 to 1.5 for populations ARG 95-3 and ARP 96-11, respec-tively (Table 2).<br>
Detached leaf assays with ABPT-derived progeny and mapping populations where found to be less accurate for phenotyping than screening under field conditions. Never-theless, results of detached leaf assays were considered suitable for the initial determi-nation of the phenotype of individual clones and thus, for construction of mapping populations.<br>
Example 2:    Genetic mapping of the Rp'hblb2 resistance locus in ABPT derived back cross populations.<br>
In all four mapping populations (Figure 1), resistance segregated as expected for a monogenic trait, suggesting the presence of a dominant resistance allele at a single locus (Table 2). This locus was designated the Rpi-blb2 locus.<br>
In order to identify markers linked to Rpi-blb2, an initial AFLP analysis with 14 primer combinations (pc) was carried out on DNA of 10 resistant and 10 susceptible ARG 95-15 progeny plants, based on detached leaf assay, including the parental clones. The testing of 21 potentially linked markers on an additional 89 plants identified several markers linked to resistance (Figure 5). Subsequent bulked segregant analysis (BSA) with 160 pc's on 2 resistant and 2 susceptible DNA pools, each containing genomic<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
DNA of 8 resistant or susceptible ARG 95-15 progeny plants, respectively, identified a total of 58 AFLP markers potentially linked to resistance (Figure 5). When a number of these markers were tested on 137 progeny plants of ARG 95-3, they were also linked to resistance in this population, suggesting that the resistance in the two populations was determined by the same locus (Figure 6). These cosegregating markers mapped 3 to 28 centimorgan (cM) and 1 to 7.2 cM to one side of the locus in ARG 95-15 and ARG 95-3 respectively, suggesting that Rpi-blb2 could be situated at a distal position on a chromosome.<br>
To determine the position of the Rpi-blb2 on the genetic map of potato, the two cosegregating AFLP markers E40M58 and E46M52 (Figure 6) were cloned into the pGEM-T vector (Promega, the Netherlands) and sequenced. Primers designed on the ends of the sequences of the cloned AFLP fragments (Table 3) were used to develop cleaved amplified polymorphic sequence (CAPS) marker E40M58 that was found to be cosegregating with the resistance trait in 25 resistant and 25 susceptible clones of ARG 95-3. CAPS marker E40M58 was subsequently tested on 46 progeny plants of the CxE mapping population (van Eck et al., 1995). These data were added to the existing marker scores of the CxE population. Joinmap (Stam, 1993) linkage analyses mapped E40M58 8 cM distal to GP79 (Gebhardt et al., 1991), positioning Rpi-blb2 on the short arm of chromosome 6. In 178 progeny plants of population ARG 95-3 no recombination between Rpi-blb2 and AFLP markers E40M58, E40M60 and CAPS marker CT119 was observed. AFLP marker E46M52 and sequence characterised amplified region (SCAR) marker CT216 mapped 2.2 cM proximal to the gene (Figure 7).<br>
Example 3:     Identification of a RGA marker linked to Rpi-blb2<br>
In an attempt to identify functionally relevant markers linked to resistance, primers designed on the conserved motifs of the NBS domain of plant R genes (Leister et al., 1996), were used in an adapted AFLP protocol (RGA-AFLP) to identify resistance gene analogue (RGA) specific markers.<br>
Using the P-loop based primer S1 from Leister et al. (1996) in combination with the EcoOO AFLP primer, an RGA specific marker, S1E00 was developed which cosegre-gated with resistance and markers E40M58 and CT119 in the ARG 95-3 mapping population (Figure 6 and 7).<br><br><br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Example 5:    Mapping of Rpi-blb2 in a diploid mapping population derived from ABPT material<br>
A total of 149 progeny clones of diploid mapping population DP1 were screened with markers E40M58e and E46M52e. No recombination was found between these markers suggesting suppressed recombination in the genomic region studied when compared to the tetraploid mapping population ARG 95-3 (Figure 7). A subset of 112 clones was screened for resistance to P. infestans isolates IPO82001, IPO655-2A and IPO428-2 in a partially repeated detached leaf assay. Eleven of the clones (11%) showed intermediate reactions and were classified as having unknown phenotypes. Another 51 and 50 clones were classified as resistant and susceptible respectively. Three progeny clones DP1-28, DP1-79 and DP1-81 were identified that were putatively recombined between the Rpi-blb2 locus and the markers E40M58e and E46M52e. In 2000, a subset of 50 out of the 112 phenotyped clones was tested for resistance to LB in the field at the trial site of Marknesse. Conclusive results on the phenotype for LB resistance were obtained for 33 out of the 50 clones. The phenotype of clones 28 and 81 as determined with the detached leaf assay appeared to be erroneous. Thus, it was concluded that these clones did not represent recombination events between Rpi-blb2 and the markers used. The phenotype of clone DP1-79 could not be verified conclusively under field conditions and this clone may represent the only recombination event between the Rpi-blb2 locus and the markers E40M58e and E46M52e in 101 progeny clones of DP1 (1 cM). Since it was shown that two markers, linked to the resistance trait in ARG 95-15, ARG 95-3 and ARP 98-11 f cosegregated with the same locus for LB-resistance in DP1, it was concluded that the DP1 parental clone ARD 1197-16 was suitable as a source for Rpi-blb2 gene isolation in a map based cloning approach.<br>
Example 6:     Physical mapping of the ABPT derived Rpi-blb2 locus<br>
The resistant clone ARD 1197-16, heterozygous for the Rpi-blb2 locus, was used as source DNA for the construction of a BAC library (hereafter referred to as the ARD 1197-16 BAG library). High molecular weight DNA preparation and BAC library construction were carried out as described in Rouppe van der Voort et al. (1999). Initially, a total of 67968 clones with an average insert size of 100 kb, which corresponds to approximately 7 genome equivalents, were individually stored in 177 384-well micro titer plates at -80°C. Marker screening of the ARD 1197-16 BAC library was carried ou as described in Rouppe van der Voort et al. (1999). Essentially, DNA pools generated for each 384-well plate were screened by PCR with SCAR or CAPS markers linked to the Rpi-blb2 locus in order to build a BAC contig across the Rpi-blb2 locus.<br><br>
WO 2005/014631	PCT7EP20O4/008683<br>
Screening of the ARD 1197-16 BAC library with markers E40M58e, S1E00 and CT119 identified several positive BAC clones, which served as seed BACs from which a chromosome walk across the Rpi-blb2 locus was initiated. Marker E40M58e was used to isolate the BAC clones 69 and 141 whereas BAC clones 14, 24,123 and 133 were positive for marker S1E00. Marker CT119 was used to isolate BAC 67. After sequencing the left (L) and right (R) borders of these BAC clones, a new set of markers was developed; 14L, 24L, 24R, 69L, 69R, 141R, 123L, 123R, 133R and 67L Screening of the isolated BAC clones with these markers showed that the following pairs of BAC clones shared overlap: the right side of 123 with the left side of 133,14 completely with 24, and the left side of 69 with the right side of 141. BAC 67 did not share overlap with the other BAC clones. The finding that the S1E00 positive BAC clones 14, 24,123, and 133 did not form a single contig indicated that S1E00 was a repetitive sequence. This, together with the finding that the right BAC-end sequences of BAC clones 24 and 123 showed high homology to different regions of the Mil resistance gene from tomato (Mtl-ligan et al., 1998, Simons et al., 1998), suggested that the Rpi-blb2 locus harboured more than one RGA. Screening of the initial ARD 1197-16 BAC library with markers 141 Rt 24L, 24R and 123L did not lead to contig extension. However, screening of the library with markers 123R and 133R resulted in the isolation of BAC clones 99 and 113, thereby extending the BAC 123/133 contig in one direction. BAC-end sequencing of these two BAC clones lead to the development of two new markers, 99L and 113R. Screening of the ARD 1197-16 BAC library with 69R lead to the extension of the 141/69 contig. Consecutive screening of the BAC library with markers derived from BAC clones that further extended this contig lead to the isolation of BAC clones 36, 41 and 112, and the development of markers 36L, 41L and 112L<br>
In an attempt to complete the BAC contig across the Rpi-blb2 locus, the ARD 1197-16 BAC library was enlarged with an additional 38864 BAC clones of -100kb (384-well plate numbers 178-273). this second library was screened with markers 24Lf 24R, 123L, and 141R, leading to the identification of BAC clones positive for both 24R and 123L (e.g. 191) and BAC clones positive for 24L (211, 242). In this way, the gap between BAC 24 and 123 was closed and the 24/14 contig was extended towards BAC clone 141. There were no new clones in the extended ARD 1197-16 library that were positive for marker 141R.<br>
Example 7:     Construction of additional markers in BAC 123/133 region.<br>
In an attempt to develop additional polymorphic markers from BAC 123 and 133, a 10 kb sub-clone library was constructed of both BAC 123 and 133. BAC DNA was partially cleaved with Sau3AI and fragments of approximately 10 kbp were cloned in the BarnHI site of vector pBINPLUS. In order to select clones containing the original BAC-end se-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
quence, 288 subclones of BAC 123 and 192 of BAC 133 were screened with the BAC-end markers 123L or 133R. In total 14 subclones were positive for marker 123L and 11 for marker 133R. Subsequently, the orientation of the BAC-end positive clones was determined by several PCRs using either the forward or reverse primer of the relevant BAC-end marker in combination with primers M13F or M13R (Table 3). For marker 123L three sub-clones and two sub-clones for marker 133R were selected and the ends not containing the 123L or 133R marker were sequenced (approximately 500 bp). Based on the new sequence two new primers were designed for subclone 123 resulting in marker 123L2 and two new primers were designed for subclone 133 resulting in marker 123R2. SCAR marker 123L2, which was located 10 kbp proximal to marker 123L. appeared to be polymorphic in mapping populations ARG 95-3, ARP 96-11 and as CAPS in B6; SCAR marker 133R2, which was located 10 kbp distal to marker 133R, was only polymorphic in mapping populations ARG 95-3 and ARP 95-11.<br>
Example 8:    Fine mapping of the Rpi-blb2 locus in ABPT derived mapping populations..<br>
In order to fine map the Rpi-blb2 locus in ABPT derived mapping populations a total of 2283 new progeny clones of mapping population ARG 95-3 and 598 clones of mapping population ARP 96-11 were tested for resistance to LB in the field at the trial site of Marknesse in 2000 (Table 2). In population ARG 95-3 846 clones (37%) were scored susceptible and 886 clones resistant (39%). The phenotypes of the remaining 551 clones were unclear. In population ARP 96-11 256 clones (45%) were scored susceptible and 170 clones (30%).resistant. The phenotypes of the remaining 142 (25%) were unclear (Table 2). The 846 and 170 resistant clones from mapping populations ARG 95-3 and ARP 96-11, were selected for recombinant screening with SCAR marker CT216 and CAPS marker 41L or 36L, respectively. In total 85 (9.6 cM) and 22 (12.9 cM) recombinants were obtained in mapping populations ARG 95-3 and ARP 96-11 respectively, that were subsequently screened with CAPS marker 67L, reducing the number of recombinants to 5 (0.56 cM) in the marker interval 67L - 36L in case of mapping population ARG 95-3 and to 4 recombinants (2.35 cM) in the marker interval 67L - 41L in case of the mapping population ARP 96-11 (Figure 8). These remaining 9 recombinants were further analysed with SCAR and CAPS markers 113R, 99L, 133R, 133R2, 123R, 123L, 24R, 14Lt 24L, 141R, 69L, E40M58e and 69R. The latter two markers were scored only in mapping population ARG 95-3.<br>
In population ARG 95-3 two clones showed recombination between markers E40M58e and 69L, positioning the Rpi-blb2 gene 0.23 cM proximal to marker E40M58e. Two other clones were recombined between markers 113R and 67L and one was recom-<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
bined between markers 133R2 and 133R, positioning the Rpi-b!b2 gene 0.11 cM distal to marker 133R.<br>
In population ARP 96-11r no recombination was detected between markers 41L and 69L, positioning the Rpi-blb2 gene 0.58 cM proximal to marker 36L Two progeny clones were recombined between markers 113R and 67L, and one clone was recom-bined between markers 99L and 133R, positioning the Rpi-blb2 gene 0.58 cM distal to marker 99L (Figure 8; Figure 9),<br>
Example 9:    Evaluation and genetic mapping of late blight resistance in a S. bulbocastanum intrasperific mapping population.<br>
In order to develop an intraspecific mapping population of S. bulbocastanum, a resistant clone Bib 2002 was obtained from an inter accession cross (Rgure 10); This clone was reciprocally crossed with a susceptible done Bib 48-5 that was selected also in progeny from an inter accession cross (Figure 10). The resulting population was designated B6 with synonyms B6a, Bib 99-229, Bib 00-7 and Bib 00-8.<br>
Initially a small group of 47 progeny plants of the B6 population was screened for resistance to P. infestans in a partially repeated detached leaf assay using a sporangio-spore solution of isolate IPO655-2A of P. infestans as inoculum. Plants with leaves that clearly showed sporulating lesions 6 to 9 days after inoculation were considered to have a susceptible phenotype whereas plants with leaves showing no visible symptoms or necrosis at the side of inoculation in the absence of clear sporulation were considered to be resistant. Of the 47 seedlings, 23 scored resistant and 24 susceptible. These data indicated that the progeny of mapping population B6 gave clear segregation of the resistance trait in the detached leaf assay and that resistance could be due to a single dominant gene or a tightly linked gene cluster, in order to determine the chromosome position of this locus, 46 seedlings were analysed with markers 112L and E46M52e. Marker 112L was found to be linked in repulsion with the resistant phenotype, as only two recombinants were obtained between this marker and the phenotype of the 46 seedlings (4 cM). Also, marker E46M52e was found to be linked in repulsion with the resistant phenotype. Here, five recombinants were obtained between marker E46M52e and the phenotype (11 cM). Furthermore, markers 69R, 69L and 141R were used for analysis of the seven recombinants between markers 112L and E40M58e with an additional group of 6,15 and 14 non recombined seedlings respectively, and found to be completely linked in either coupling (marker 69R) or repulsion phase (markers 69L and 141R) to resistance, indicating that the resistance gene was located at the same locus, i.e. Rpi-blb2, as in the ABPT-derived mapping populations.<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
In order to determine the position of Rpi-blb2 more precisely relative to the available markers, another 849 seedlings of the B6 mapping population and 1054 seedlings from the reciprocal cross (Figure 10) were grown and analysed for recombination between the markers E46M52e and 112L Thus, in addition to the initial 47 seedlings, a total of 1903 individual offspring clones of the B5 population were screened. Recombination between markers E46M52e and 112L was defected in a total of 138 of these seedlings (7,25 cM). Fine mapping of the Rpi-blb2 locus was carried out in two steps. Firstly, the group of 138 recombinants was reduced to 19 by additional screening with markers 14Lb, 113R, 123L2, 24L, 141R and 69L (Table 3), derived from left (L) and right (R) border sequences of BAC clones isolated from the ARD 1197-16 BAC library and subsequent selection of all the seedlings that were recombined between markers 113R and 69L Possibly due to double recombination, 4 recombinants gave patterns for the markers scored that deviated from scores expected in the case of single recombination events in the genetic interval studied and when assuming co-linearity of markers. These were withdrawn from further analyses. Secondly, the remaining 15 recombinants were analysed with markers from border sequences of BAC clones isolated from the Bib 2002 library, SPB39L and SPB30L, or with MiGA markers 24L9spec, 24Lspec and 14L24L (Table 3). Results of marker analyses of these remaining 15 recombinants, which gave clearly interpretable marker scores and phenotypes, positioned the Rpi-blb2 locus between markers 69L and 24L, on a 0.11 cM (n=1899) genetic interval (Figure 11).<br>
Example 10:   MiGA markers<br>
Southern analysis of BAC clones 14, 24, 123 and 133 using markers 123R, 14L( or 24L as probes showed that these BAC clones contained several resistance gene analogs (RGAs). In view of the homology between the sequences of markers 14L, 24L and 123R with the Mil gene from tomato, RGAs within the Rpi-blb2 region are hereafter referred to as Mi gene analogs (MiGAs). In an attempt to develop additional polymorphic markers within the Rpi-blb2 interval, PCR fragments generated from BAC clones 24 and 123 with the primer combination 14LR and 24LF were cloned into the pGEM-T vector (Promega, the Netherlands) and partially sequenced. Based on the alignment of these partial sequences, a set of universal primers were designed, univ14L and univ24L (Table 3), with the aim to amplify the corresponding region of as many as possible MiGAs within the Rpi-blb2 interval. This universal primer set was subsequently used to develop MiGA specific SCAR/CAPS markers linked to Rpi-blb2 (e.g. markers 14L24L, 24Lspec, 24L9spec; Figure 9).<br><br><br><br>
WO 2005/014631	PCT/EP2004/008683<br>
quences using the primers univ24L and univ14L (Table 3). Positive clones were selected for further characterisation. Based on the restriction pattern of the 14L24L fragments digested with the enzymes Rsal, Taql. Alul, DpnII or Msel, the different groups of MiGAs were identified. The MiGA harbouring the marker 24L, which was completely present on BAC clones BlbSP39, 211 and 242 was not detected with the universal primers univ14L and univ24L<br>
The relative position of the MiGA sequences in the 10kbp subclones was determined by PCR using internal primers 123Mi and 14L2 for the 51 end and univ14L and 24L2 for the 3' end in combination with primers derived from pBlNPLUS vector sequences (ARO 295 and 296; Table 3). Two subclones per RGA of each BAC-library were selected for transformation.<br>
For complementation analysis, the selected subclones were transferred to the susceptible potato cultivars Impala and Kondor through Agrobacterium mediated transformation using isolate UIA143 (Farrand et al., 1989) or AGLO (Lazo et ah, 1991). Primary transformants harbouring the transgenes of interest were tested for resistance to P. infestans in detached leaf assays using isolate IPO655-2A and IPO82001 (Table 4). Only the genetic constructs harbouring RGC5, both derived from S. tuberosum and S. bulbocastanum, were able to complement the susceptible phenotype both in cultivar Impala and in Kondor; in total 18 out of 19 RGC5 containing primary transformants were resistant (Table 4, Figure 12) whereas all RGC1. RGC2, RGC3, RGC4, RGC6 RGC7 or RGC8 genes containing primary transformants were susceptible to P. infestans. As the RGC5 transformants showed similar resistance phenotypes as the resistant S. bulbocastanum parent of mapping population B6, RGC5 was designated the Rpi-blb2gene. The homologue RGC24L can also be transferred to the described susceptible potato cultivars and tested for resistance to P. infestans in a detached leaf assay.<br>
A selection of primary transformants containing RGC5 was analysed for copy number by Southern analysis. EcoRI digested genomic DNA was hybridised with a nptll probe (Table 3). Based op the presence of the number of nptll hybridising fragments, the primary transformants contained at least 1 to 11 transgene inserts. In total, 4 single copy integrations in cultivar Impala and 6 in cultivar Kondor were observed of which one cultivar Kondor transformant appeared to have a P. infestans susceptible phenotype.<br>
To investigate whether Rpi-blb2 can also complement the susceptible phenotype in tomato, primary transformants of cultivar Moneymaker harbouring the Rpi-blb2 gene construct were produced and tested with the potato derived isolates IPO82001 and<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
IPO655-2A. The disease resistance assay revealed that RGC5 is also able to complement a susceptible tomato phenotype (Table 6)..<br>
Example 13:   Rpi-blb2 gene structure and putative amino acid sequence<br>
The inserts of the RGC5 containing binary subclones 211F/C12 and SP39-20 were sequenced by a primer walk strategy whereby consecutive rounds of sequencing were carried out using a set of nested primers which were designed as the contiguous sequence was extended. The first set of sequences was generated using the M13F and M13R primers. The complete sequences of the inserts of clones 211F/C12 and SP39-20 consisted of 7967 and 9949 nudeotides (nt), respectively (Figure 13). The sequence of clone 211F/C12 wad identical to the corresponding sequence within clone SP39-20. The position and putative structure of Rpi-blb2 was predicted using GEN-SCAN (Burge and Karlin, 1997), GeneMark (Lukashin and Borodovsky 1998) and through alignment to the gene sequences of Mi1.1 and Mil.2.<br>
The exact length and structure of the coding sequence was determined through 5' and 3' rapid amplification of cDNA ends (RACE) using the GeneRacer™ kit (lnvitrogen™f Groningen, the Netherlands). RACE identified 5' and 3' Rpi-blb2 specific cDNA fragments comprising 5' and 3' untranslated regions (UTRs) of 767.and 201 nudeotides (nt), respectively. The Rpi-blb2 gene contains two introns. Intron 1 is 626 nt long and positioned within the 51 UTR ending 32 nudeotides upstream of the ATG start codon. Intron 2 is 86 nt long starting 43 nudeotides downstream of the ATG start codon of the gene. The coding sequence of the Rpi-blb2 transcript is 3304 nudeotides.<br>
The deduced open reading frame of the Rpi-blb2 gene encodes a predicted polypep-tide of 1267 amino acids with an estimated molecular weight of 146 kD (Figure 14). Several functional motifs present in R genes of the NBS-LRR class of plant R genes are apparent in the encoded protein. As illustrated in Figure 14, the Rpi-blb2 protein belongs to the leucine zipper (LZ) subset of NBS-LRR resistance proteins. The N-terminal half of the Rpi-bib2 protein contains a potential LZ region between amino acids 413 and 434 and six conserved motifs indicative of a nucleotide-bindtng site (van der Biezen and Jones, 1998). The C-terminal half of Rpi-blb comprises a series of 15 irregular LRRs that can be aligned according to the consensus sequence hxxhxxLxxLxLxxC/N/Sx(x)LxxLPxx observed in other cytoplasmic R proteins, whereby h can be L, lt M, V or F, and x any amino acid residue (Jones and Jones, 1997).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
Example 14:   Homology to known state of the art R gene sequences<br>
To identify in silico homologues of the Rpi-blb2 genef BLAST searches (Altschul et at., 1990) were carried out with the coding sequence of the Rpi-blb2 gene. BLA.STN searches identified a number of sequences with significant homology to the Rpi-blb2 gene. Using the alignment programme ClustalW (standard settings) in the DIMAStar software package, we determined that the Rpi-b!b2 coding sequence shares the highest homology to MI-1.1 (89.8%) and MM .2 (89.7%) (Genbank accession numbers AF039681 and AF039682, respectively). The latter sequence corresponds to the Mi gene from tomato that confers resistance to three of the most damaging species of the root knot nematodes (Meloidogyne spp.) (Mii)igan et a!., 1998). In addition nucleotides 2410-3461 of the Rpi-bib2 coding sequence share 87.8% sequence homoiogy to a partial NBS-LRR sequence from Solanum nigrum (Genbank accession number AY055116.1). At the amino add level the putative Rpi-blb2 protein sequence shares the highest homology to Mi-1.1 (82% identity) and Mi-1.2 (81% identity) (Genbank accession numbers AF039681 and AF039682).<br>
Through ClustalW alignment of the deduced amino acid sequences of Rpi-blb2, Mi-1.1 and Mi-1.2 we have identified 200 amino acid (aa) residues which are unique to Rpi-blb2 (Figure 15). Of these, 31 are found at hypervariable positions, i.e. the residue at this position is different in all three sequences and 11 are encoded by small insertions (one 3 aa residue insertion and one 8 aa residue insertion). The rest are Rpi-blb2 specific in that the aa residues encountered at corresponding positions in Mi-1.1 and Mi-1.2 are different from the Rpi-blb2 residue but conserved in the two Mi protein sequences (Figure 15). Interestingly, the VLDL motif that is conserved in the third LRR of many NBS-LRR proteins including Mi (Axtell et ah, 2001; Banerjee et al.» 2001), is not conserved in Rpi-blb2 (Figure 15).<br>
Example 15:   Rpi-blb2 allele mining in wild Solanum species<br>
Using primers ARF1F and ARF1R (Table 3B), designed around the start and stop codon of the Rpi-blb2 gene, it is possible to amplify by PCR, alleles of Rpi-blb2 from any Solanum species. The amplification products can be cloned between transcrip-tional regulatory sequences in a binary plasmid and transferred to S. tuberosum through Agrobacterium mediated transformation or any method known to those skilled in the art. The resulting primary transformants can subsequently be analysed for resistance to P. infestans or to any pathogen for which potato is a host plant.<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Example 16:   Material and methods<br>
Plant material and development of mapping populations in (1) Solanum tuberosum. Complex interspecifc hybrid clones, designated ABPT, were made by Hermsen and co-workers (Hermsen, 1966; Hermsen and Ramanna, 1969; Ramanna and Hermsen, 1971; Hermsen and Ramanna, 1973; Hermseh, 1983; Hermsen, 1994) (Figure 1a). The chromosome doubling step with colchicines was described by Hermsen (1966) and Hermsen and De Boer (1971). The resistance in some of the ABPT clones to P. infestans is believed to be derived from either one or both of the accessions from S. bulbocastanum BGRC 8007 (CGN 21306; Pi 275196) and BGRC 8008 (CGN 17693; Pi 275198) that were used in the initial cross to produce hybrids between S. acaule and S. bulbocastanum, since all other parents that were used in the breeding scheme for ABPT-clones were susceptible or only partially resistant to P. infestans in detached leaf assays (Hermsen and Ramanna, 1973). Tubers from 19 clones of population [(ABPT clone number 55 x cultivar (cv) Oberambacher FrOhe) x cv Arkula], from 7 clones of population [(ABPT clone number 55 x cv Oberambacher FrOhe) x cv Blanka] and from 5 clones of population {(ABPT clone number 60 x cv Alcmaria) x cv Blanka] were received in 1988 from the former Department of Plant Breeding of the Wageningen Agricultural University (Wageningen, the Netherlands). Clones ARF 87-507, ARF 87-801 and ARF 87-601 were selected from these populations respectively. They represented offspring from a second backcross (BC2) with the complex interspecific ABPT-clones and were used for further back crosses that resulted in one tetraplold BC3 population, two tetraploid BC4 populations and one diploid BC4 population that were used for genetic mapping of the Rpi-blb2 gene (Figure 1). The tetraploid<br><br>
P. infestans resistant clone ARF 87-507 with the susceptible cultivar Alkon. Tetraploid population ARG 95-3 was produced by crossing P. infestans resistant clone AR 91-1263 with the susceptible cultivar Cosmos. Tetraploid population ARP 96-11 was produced by crossing resistant clone AR 92-1292 with the susceptible cultivar Celeste. The diploid population DP1 was obtained by crossing the resistant clone ARD 1197-16 with the susceptible clone ARD 93-2090 (Figure 1).<br>
Plant material and development of mapping populations in (2) Solanum bulbocastanum.<br>
The diploid S. bulbocastanum mapping population, designated B6 (synonym B6a, Bib 99-229, Bib 00-7 and Bib 00-8), was developed by crossing a P. infestans resistant clone Bib 2002 (synonym M94-81-C) with a susceptible clone Bib 48-5. Results from reciprocal crosses of population B6 were combined. The resistant parental clone of population B6 was obtained from a cross between S. bulbocastanum clone Bib 93-D26-3 (accession BGRC 8002; CGN 17690; Pi 275187) as female parent and<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
S. bulbocastanum clone Bib 93-60-10 (accession BGRC 8006; Pi 275194) as male parent. The susceptible parental clone of population B6 was obtained from a cross between S. bulbocastanum clones from accessions BGRC 8005 (CGN 17692, PI 275193) and BGRC 8006 (Figure 2).<br>
Disease assays;(1) Phytophthora infestans isolates<br>
Three different P. infestans isolates were obtained from Plant Research International B.V. (Wageningen, the Netherlands). Isolates had different race structures and mating types as follows: IPO82001: race structure 1.2.3.4.5.6.7.10.11, mating type A2; IPO655-2A: race structure 1.2.3.4.5.6.7.8.9.10.11, mating type A1; IPO428-2: race structure 1.2.3.4.5.6.7.8.9.10.11, mating type A2 (Flier et al., 2003).<br>
Disease assays; (2) field trials<br>
Glasshouse grown seedling tubers or field grown seed potatoes were planted at trial sites in Marknesse, the Netherlands from 1985 tot 2002, in the Toiuca area of Mexico in 1991 or at a site in the Benguet Province in the Philippines in 1992. For individual clones, plots were planted consisting of 1 to 10 tubers. Approximately 8 weeks after planting, the field at Marknesse was inoculated with a sporangiospore solution of P. infestans isolate IPO82001 and disease scores were collected 3 to 6 weeks after inoculation. Clones that were free or nearly free from late blight were classified as having a resistant phenotype whereas clones with a complete of nearly complete blighted foliage were classified as susceptible. Clones with intermediate reactions to late blight were classified as having an unknown phenotype. At the field trials in Mexico and the Philippines, natural infection had to occur. Once this natural infection by P. infestans established, the percentage of blighted foliage of plants on each plot was scored on 8 and 6 days respectively on a 1-9 scale were estimated percentages of blighted foliage from 1 tot 9 were: 0, 3, 10, 25, 50, 75, 90, 97 and 100 (Estrada-Ramos et al., 1983).<br>
Disease assays; (3) detached leaves<br>
For the detached leaf assay, leaves from plants grown for 6 to 12 weeks in the greenhouse were placed in pieces of water-saturated florists foam, approximately 35x4x4 cm, and put in a tray (40 cm width, 60 cm length and 6 cm height) with a perforated bottom. Each leaf was inoculated with two droplets (25 pi each) of sporangiospore solution on the abaxial side. Subsequently, the tray was placed in a plastic bag on top of a tray, in which a water-saturated filter paper was placed, and incubated in a climate room at 17°C and a 16h/8h day/night photoperiod with fluorescent light (Philips TLD50W/84HF and OSRAM L58W/21-840). After 6 to 9days, the leaves were evaluated for the development of P. infestans disease symptoms.<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Evaluation:<br>
Plants with leaves that clearly showed sporulating lesions 6 to 9 days after inoculation were considered to have a susceptible phenotype, whereas plants with leaves showing no visible symptoms or necrosis at the side of inoculation in the absence of clear sporulation were considered to be resistant<br>
Plant DNA marker screening<br>
Genomic DNA was extracted from young leaves according to Bendahmane et al. (1997). For PCR analysist 15 pi reaction mixtures were prepared containing 0.5 *jg DNA, 15 ng of each primer, 0.2 mM of each dNTP, 0.6 units Taq-polymerase (15 VI\A, SphaeroQ, Leiden, the Netherlands), 10 mM Tris-HCI pH 9,1.5 mM MgCI2, 50 mM KCI, 0.1% Triton X-100 and 0.01% (w/v) gelatine. The PCRs were performed using the following cycle profile: 25 seconds DNA denaturation at 94°C, 30 seconds annealing and 40 seconds elongation at 72°C. As a first step in PCR-amplification DNA was denatured for 5 min at 94°C and finalised by an extra 5 min elongatiprt step at 72°C. The amplification reactions were performed in a Biometra* T-Gradient or Biometra* Uno-ll thermocyder (Westburg, Leusden, the Netherlands). Depending on the marker, the PCR product was digested with an appropriate restriction enzyme. An overview of the markers including primer sequences, annealing temperature and restriction enzymes if appropriate, is given in Table 3. Subsequently, the (cleaved) PCR products were analysed by electrophoresis in agarose or acrylamide gels. For acrylamide gel analysis, the CleanGel DNA Analysis Kit and DNA Silver Staining Kit (Amersham Pharmacia Biotech Benelux, Roosendaal, the Netherlands) were used.<br>
Elongation ofAFLP fragments by Thermal asymmetric interlaced (TAIL)-PCR Elongation of the sequence of an AFLP fragment was performed by TAIL-PCR according to Liu and Whittier (1995). Shortly, elongation ofAFLP fragments was performed using 2 or 3 nested specific primers (sp) in combination with an arbitrary degenerate (AD) primer. The first PCR was performed with primers sp1 and AD, the second with sp2 and AD and the third with sp3 and AD according to the scheme described in Table 5. The PCR was performed in 25 pi reactions containing the standard PCR mix as described before, except that 30 ng of primer AD was used. The elongated fragments were cloned in pGEM-T (Promega, the Netherlands) and sequenced.<br>
BAC library construction and screening<br>
The resistant clone ARD 1197-16, heterozygous for the Rpi-blb2 locus, was used as source DNA for the construction of the S tuberosum BAG library. The resistant clone Bib 2002 heterozygous for the Rpi-blb2 locus, was used as source DNA for the construction of the S. bulbocastanum BAC library. High molecular weight DNA preparation and BAC library construction were carried out as described in Rouppe van der Voort et<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
al. (1999). For the S. tuberosum BAC library, approximately 120.000 clones with an average insert size of 100 kb, which corresponds to 8 to 10 genome equivalents were finally obtained. A total of approximately 70.000 clones were individually stored in 177 384-well microtiter plates at -80°C. Another 50.000 clones were stored as 14 bacterial pools containing approximately 4000 white colonies. These were generated by scraping the colonies from the agar plates into Luria Broth medium containing 18% glycerol and 12.5 pg/ml chloramphenicol using a sterile glass spreader. These so-called super pools were also stored at -80°C. Finally, another 37.000 clones were added to the S. tuberosum BAC library. The S. bulbocastanum BAC library consisted of 48 super pools of approximately 2.000 colonies.<br>
Marker screening of the BAC library harbouring the individually stored BAC clones was carried out as described in Rouppe van der Voort et al. (1999). For the screening of the BAC library stored as super pools, plasmid DNA was isolated from each pool of clones using the standard alkaline lysis protocol and PCR was carried out to identify positive pools. Bacteria corresponding to positive pools were diluted and plated on Luria Broth agar plates containing chloramphenicol (12.5 pg/ml) Individual white colonies were subsequently picked into 384-well microtiter plates and single positive BAC clones subsequently identified as described above. Names of BAC clones isolated from the super pools carry the prefix SP (e.g. SP39).<br>
Subcloning of candidate genes<br>
Candidate RGAs were subcioned from BAC clone 24, 211, 242, BLBSP39 and BLBSP30 as follows. Aliquots of approximately 1 pg BAC DNA were digested with 1U, 0.1U or 0.01 U of Sau3AI restriction enzyme for 30 min. The partially cleaved BAC DNA was subjected to CHEF electrophoresis at 4°C in 0.5 X TBE using a linear increasing pulse time of 1-10 sec and a field strength of 6 V/cm for 16 hr. After eiectrophoresis, the agarose gel was stained with ethidium bromide to locate the region of the gel containing DNA fragments of approximately 10 kbp in size. This region was excised from the gel and treated with GELASE (Epicentre Technologies, USA) according to the manufacturer. The size selected DNA was ligated to the BamHI-cleaved and dephos-phorylated binary vector pBINPLUS (van Engelen et al., 1995) followed by transformation to ElectroMAX E. coli DH10B competent cells (Life Technologies, UK). A total of 192 clones were PCR screened for the presence of RGC sequences using the primers of marker 24L14L (Table 3). Positive clones were selected for further characterisation. Identification of clones harbouring RGC1, RGC2, RGC3, RG4, RGC5, RGC6, RGC7, RGC8 and RGC24L was carried out by sequencing 14L24L PCR fragments derived from positive clones. The relative position of the RGAs within a subclone was determined by PCR analysis using internal primers (24L2, 123Mi) in combination with pBIN-PLUS specific primers (Table 3).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
Agrobacterium tumefaciens mediated transformation of potato Binary plasmids harbouring the candidate genes were transformed to A. tumefaciens strains AGLO (Lazo et aL, 1991) or U1A143 (Farrand et al.91989), the latter containing the helper plasmid pCH32 (Hamilton et aL, 1996). Overnight cultures of the transformed A. tumefaciens strains were used to transform potato tuber discs (cvs Impala and Kondor) according to standard protocols (Hoekema et al., 1989; Rllati et al., 1987). Shortly, certified seed potatoes of cultivars Impala and Kondor were peeled and surface sterilised for 30 min in a 1% sodium hypochlorate solution containing 0.1% Tween-20. Tubers were then washed thoroughly in large volumes of sterile distilled water (4 times, 10 min). Discs of approximately 2 mm thickness and 7 mm in diameter were sliced from cylinders of tuber tissue prepared with a corkbore. The tuber discs were transferred into liquid MS30 medium containing A. tumefaciens and incubated for 15 min. After removing the A. tumefaciens solution, the tuber discs were transferred to regeneration medium containing MS30, 0.9 mg/l IAA, 3.6 mg/l zeatine riboside and 8 g/l agar (Hoekema et al., 1989). The plates were incubated at 24°C, 16 hour day-length (Philips TLD50W/B4HF). After 48 hours of co-cultivation, the tuber discs were rinsed for 5 min in liquid MS medium including antibiotics, 200 mg/l vancomycin, 250 mg/l cefotaxim and 75 mg/l kanamycin. and transferred to regeneration medium supplemented with the same antibiotics. The plates were incubated at 24°C, 16 hour day-length (Philips TLD50W/84HF). Every three weeks, the tuber discs were transferred to fresh medium. Regenerating shoots were transferred to MS30 medium containing 75 mg/l kanamycin. Rooting shoots were propagated in vitro and tested for absence of A. tumefaciens cells by incubating a piece of stem in 3 ml Luria Broth medium (3 weeks, 37DC, 400 rpm). One plant of each transformed regenerani was transferred to the greenhouse.<br>
Agrobacterium tumefaciens mediated transformation of tomato Seeds of the susceptible tomato line Moneymaker were rinsed in 70% ethanol to dissolve the seed coat and washed with sterile water. Subsequently, the seeds were surface-sterilised in 1.5% sodium hypochlorite for 15 minutes, rinsed three times in sterile water and placed in containers containing 140 ml MS medium pH 6.0 (Murashige and Skoog, 1962) supplemented with 10 g/l sucrose (MS 10) and 160 ml vermiculite. The seeds were left to germinate for 8 days at 25°C and 0.5 W/M2 light.<br>
Eight day old cotyledon explants were pre-cultured for 24 hours in Petri dishes containing a two week old feeder layer of tobacco suspension cells plated on co-cultivation medium (MS30 pH 5.8 supplemented with Nitsch vitamines (Duchefa Biochemie BV, Haarlem, the Netherlands), 0.5 g/l MES buffer and 8 g/l Daichin agar).<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
Overnight cultures of A tumefaciens were centrifuged and the pellet was resuspended in cell suspension medium (MS30 pH 5.8 supplemented with Nitsch vitamines, 0.5 g/l MES buffer, pH 5.8) containing 200 pM acetosyringone to a final O.D.600 of 0.25. The explants were then infected with the diluted overnight culture of A tumefaciens UIA143 containing pBINRGC5 for 25 minutes, blotted dry on sterile filter paper and co-cultured for 48 hours on the original feeder layer plates. Culture conditions were as described above.<br>
Following the co-cultivation, the cotyledons explants were transferred to Petri dishes with selective shoot inducing medium (MS pH 5.8 supplemented with 10 g/l glucose, including Nitsch vitamines, 0.5 g/l MES buffer, 5 g/l agargel, 2 mg/l zeatineriboside. 400 mg/l carbenicilline, 100 mg/l kanamicine, 0.1 mg/l IAA) and cultured at 25°C with 3-5 W/m2 light The explants were sub-cultured every 3 weeks onto fresh medium. Emerging shoots were dissected from the underlying callus and transferred to contain-ers with selective root inducing medium (MS 10 pH 5.8 supplemented with Nitsch vitamines, 0.5 g/l MES buffer, 5 g/I agargel, 0.25 mg/l IBA, 200 mg/l carbenicillin and 100 mg/l kanamycine).<br>
RNA extraction<br>
Total RNA was isolated using Trizol® according to the protocol supplied by the manufacturer (Invitrogen™, Groningen, the Netherlands) with minor modifications. Briefly, 0.5 g of young leaf tissue was ground in liquid nitrogen and the powder suspended in 5 ml Trizol®. After a 5 min incubation at room temperature (RT), 0.5 ml chloroform was added, the suspension was vortexed and incubated for 2 min. After centrifugation (15 min, 11404 x g, 4°C) the supernatant was transferred to a new tube and 2.5 ml isopro-panol was added. After 10 min at RT, nucleic acids were precipitated (10 min, 11404 x g, 4°C). The pellet was washed with 5 ml 70% ethanol (5 min, RT) and after centrifugation (5 min, 6415 x g, 4°C), the pellet was dried and resuspended in 100 y\ sterile distilled water. PoIyA+ RNA was extracted from total RNA using the Oligotex™ rnRNA midi kit (Qiagen, GmbH, Germany).<br>
Rapid amplification of cDNA ends.<br>
The 5' and 3' ends of the Rpi-blb2 cDNA and confirmation of putative intron positions<br>
was determined by rapid amplification of cDNA ends (RACE) using the GeneRacer™ kit (Invitrogen™, Groningen, the Netherlands). 5' RACE was carried out on cDNA syn-<br><br>
WO 2005/014631	PCTYEP2004/008683<br>
thesised with primer GSP4 (ARO 772). Subsequently, primer GSP6 (ARO 774) was used in combination with the GeneRacer™ 5' primer and the final amplification was carried out with GSP6 in combination with the GeneRacer™ 5' nested primer. 3' RACE was carried out with the nested primers GSP1 (ARO 769) and GSP2 (ARO 770) in combination with the GeneRacer 3' primer. The final amplification was carried out with GSP3 (ARO 771) in combination with GeneRacer nested 3' primer.<br>
Both 3' and 5' RACE amplification steps were canied out using Accuprime (Invitro-gen™, Groningen, the Netherlands) instead of the Taq polymerase supplied by the GeneRacer™ kft<br>
AFLP fingerprinting and cloning ofAFLP fragments<br>
Template preparation and AFLP fingerprinting were essentially performed as described in Vos et al. (1995). In order to clone specific fragments 33P-labelled AFLP fragments were excised out of the acrylamide gel by overlaying the polyacrylamide gels, dried on Whatmann 3MM paper, with autoradiogram images. The pieces of gel/paper underneath the band of interest were cut out and transferred to 200 pi of TE and incubated for 1 h at room temperature. Five microlitres of supernatant was used to re-amplify the fragment, using a PCR in which the EcoRI+0 in combination with Msel+0 were used as primers. The re-amplified AFLP fragment was subsequently cloned into the pGEM-T cloning vector (Promega, the Netherlands) and the inserts of several clones se-quenced..<br>
The DNA sequence of the excised AFLP band was used to design locus-specific primers. The amplification product obtained with such primers was screened for internal polymorphisms with restriction enzymes. After restriction, the fragments were separated on a 2-3% agarose gel including ethidiumbromide.<br>
RGA-AFLP analysis<br>
Template preparation was essentially performed as described in Vos et al. (1995). However, the second amplification step was canied out with the P-loop based primer S1 from Leister et al. (1996) in combination with the EcoRI+0 AFLP primer. A 10 pl reaction mixture [0.5 pi 33P-labelled S1 primer (10 ng/l); 0.5 pi EcoR1+0 primer (10 ng/pl); 0.8 pi dNTPs (5mM); 2 pi 10xGoIdstar™ PCR buffer (Eurogenetc. Belgium); 1.2 pi MgCI2 (25.mM); 0.06 pi Goldstar™ DNA polymerase (5U/pl) (Eurogentec, Bel-gium); 14.94 pi MQ water] was added to 10 pi diluted template (20x diluted in MQ water) and a PCR reaction performed using the following cycle profile: 45 seconds DNA denaturation at 94°C, 45 seconds primer annealing at 49°C and 2 min elongation step at 72°C (35 cycles). Prior to the cycling the template DNA was denatured for 2 min<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
at 94°C and the PCR was finalised by a applying an extra 5 min elongation step at 72°C. The amplification reactions were performed in a Perkin Elmer 9600 thermocycler. The labelled PCR products fragments were separated on a 6% polyacrylamide gel and the individual bands visualized by autoradiography according to standard procedures.<br>
Example 17: Phenotype of Rpi-blb2 expression<br>
Material &amp; Methods<br>
Four lesions (6 days after inoculation at standard conditions) of infected leaflets (IPO82001) were rinsed in 3 ml H2O. The concentration was determined using a haemocytometer Fuchs-Rosenthal (W. Schreck Hofheim/Ts.)<br>
Definition:<br>
Sporulation index is the amount of sporangia per ml detected on lesions of infected leaflets.<br><br>
The difference between Rpi-blb2 and other P. infestans resistance genes is that Rpi-blb2 allows a low level of sporulation (Figure 18). This is demonstrated by a detached leaf assay in which the lesions present on Rpi-blb2 genotype (ARD 92-1197-16) show a low level of sporangia in relation to complete absence of sporangia on a genotype containing the S. demissum gene R2. However, the sporulation index is only 1.1% of a susceptible phenotype (cv. Bintje) (Table 7 and Figure 18.<br>
Field experiments have also shown that Rpi-blb2 allows a low level of infection. Late blight symptoms developed at a low level during the growing season (Figure 3, ARF87-801) or at the end of the growing season (Figure 2, ARF87-601; Figure 3t ARF87-507 andARF87-601).<br><br>
WO 2005/014631	PCT/EP2004/008683<br>
References<br>
Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, DJ. (1990) Basic local alignment search tool. Journal pf Molecular Biology 215, 403-410. Axtell, M.J., McNellis, T.W., Mudget M.B., Hsu, C.S., and Staskawicz, B J. (2001) Mutational analysis of the Arabidopsis RPS2 disease resistance gene and the corresponding Pseudomonas syringae avrRpt2 avirulence gene.. Molecular Plant-Microbe interactions 14,181-188.<br>
Ballvora, A., Ercolano, M.R., Weis, J., Meksem, K., Bormann, CA, Oberhagen, P., Salamini, F., and Gebhardt, C. (2002) The R1 gene for potato resistance to late blight (Phytophthora infestans) belongs to the leucine zipper/NBS/LRR class of plant resistance genes. Plant Journal 30, 361-371.<br>
Banerjee, D., Zhang, X., and Bent, A.F. (2001) The teudne-rich repeat domain can determine effective interaction between RPS2 and other host factors in Arabidopsis RPS2-mediated disease resistance. Genetics 158, 439-450. Bendahmane, A., Kanyuka, K., and Baulcombe, D.C. (1997) The High-resolution genetic and physical mapping of the Rx gene for extreme resistance to potato virus X in tetraploid potato. Theoretically Applied Genetics 95,153-162. Burge, C.B. and Karlin, S. (1997) Prediction of complete gene structures in human genomic DNA. Journal of Molecular Biology 268, 78-94. Colon, LT., Turkensteen, L.J., Prummel, W., Budding, DJ. and Hoogendoom, J. (1985) Durable resistance to late blight (Phytophthora infestans) in old potato cultivars. European Journal of Plant Pathology 101 f 387-397.<br>
Dangl, J.L and Jones, J.D.G. (2001) Plant pathogens and integrated defence responses to infection. Nature 411, 826-633.<br>
Estrada-Ramos, N.,P6rez-Alvarez, O., Henfling, J. and Malamud, O. (1983) In: WJ. Hooker (ed), Research for the potato in the year 2000. International Potato Center, Lima, Peru, pp 78-79.<br>
Farrand, S.K., O'Morchoe, S.P., and McCutchan, J. (1989) Construction of an Agro-bacterium tumefaciens C58 recA mutant Journal of Bacteriology 171, 5314-5321. Fillati, J.J., Kiser, J., Rose, R., and Comai, L (1987) Efficient transfer of a glyphos-phate tolerance gene into tomato using a binary Agrobacterium tumefaciens vector. Bio Technology 5, 726-730.<br>
Flier, W.G. (2001) Variation in Phytophthora infestans, sources and implications. PhD thesis Wageningen University, Wageningen, the Netherlands, p. 93. Flier, W.G., vandenBosch, G.B.M., and Turkensteen, LJ. (2003) Stability and partial resistance in potato cultivars exposed to aggressive strains of Phytophthora infestans. Plant Pathology 52 (3), 326-337.<br>
Gebhardt, C, Ritter, E., Barone, A., Debener, T., Walkemeier, B., Schachtschabel, U., Kaufmann. H., Thompson, R.D, Bonierbale, M.W., Ganal, M.W., Tanksley, S.D., and<br><br>
WO 2005/014631	PCT7EP2004/008683<br>
Salamini, F. (1991) RFLP maps of potato and their alignment with the homologous tomato genome. Theoretically Applied Genetics 83,49-57.<br>
Hamilton, CM., Frary, A., Lewis, C.f and Tanksley, S.D. (1996) Stable transfer of intact high molecular weight DNA into plant chromosomes. Proceedings of National Academy of Science USA 93, 9975*9979.<br>
Hemnsen, J.G.Th. (1966) Crossabiiity, fertility and cytogenetic studies in Solanum acaule x Solanum bulbocastanum. Euphytica 15,149-155.   . Hermsen, J.G.Th. and Ramanna, M.S. (1969) Meiosis in different Ft-hybrids of Solanum acaule Bitt. x S. bulbocastanum Dun. and if s bearing on genome relationship, fertility and breeding behaviour. Euphytica 18, 27-35.<br>
Hermsen, J.G.Th., and De Boer, A.J.E. (1971) The effect of colchicines treatments on Solanum acaule and S. bulbocastanum; a complete analysis of ploidy chimeras in S. bulbocastanum. Euphytica 20,171-180<br>
Hermsen, J.G.Th. and Ramanna, M.S. (1973) Double-bridge hybrids of Solanum bulbocastanum and cultivars of Solanum tuberosum. Euphytica 22,457-466 Hermsen, J.G.Th. and Verdenius, J. (1973) Selection from Solanum tuberosum group Phureja of genotypes combining high-frequency haploid induction with homozygosity for embryo-spot lEuphytica 22,244-259.<br>
Hermsen, J.G.Th. (1983) Utilization of wide crosses in potato breeding. In: Report of a planning conference on present and future strategies for potato breeding and improvement. International Potato Center, Lima, Peru, pp 115-132. Hermsen, J.G.Th. (1994) Introgression of genes from wild species, including molecular and cellular approaches. In: J.E. Bradshaw and G.R. Mackay (eds), Potato Genetics, CAB International, Wallingford, UK. pp 515-538.<br>
Hijmans, R.J., Forbes, G.A., and Walker, T.S. (2000) Estimating the global severity of potato late blight with GIS-linked disease forecast models. Plant Pathology 49, 697-705.<br>
Hoekema, A., Hirsch, P.R., Hooykaas, PJJ., and Schilperoort, RA (1983) A binary plant vector strategy based on separation of vir- and T-region of the Agrobacterium tumefaciens Ti-plasmid. Nature 303,179-180.<br>
Hoekema, A., Huisman, M.J., Molendijk, L., van den Elzen, P.J.M., and Cornelissen, BJ.C. (1989) The genetic engineering of two commercial potato cultivars for resistance to potato virus X. Bio/Technology 7,273-278.<br>
Jones, D.A. and Jones, J.D.G. (1997) The role of leucine-rich repeat proteins in plant defences. Adv. Bot Res. 24, 89-167.<br>
Lazo, G.R., Stein, PA and Ludwig, RA (1991) A DNA transformation-competent Ara-bidopsis genomic library in Agrobacterium. Bio/Technology 9, 963-967. Leach, J.E., Vera Cruz, CM., Bai, J., and Leung, H. (2001) Pathogen fitness penalty as a predictor of durability of disease resistance genes. Annual Review Phytopathology 39, 187-224.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgQUJTVFJBQ1QucGRm" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgREVTQ1JJUFRJT04gKENPTVBMRVRFKS5wZGY=" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgRk9STSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgUEVUSVRJT05TLnBkZg==" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 PETITIONS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LUNIRU5QLTIwMDYgUE9XRVIgT0YgQVRUT1JORVkucGRm" target="_blank" style="word-wrap:break-word;">507-CHENP-2006 POWER OF ATTORNEY.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">507-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtY29ycmVzcG9uZG5lY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">507-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTA3LWNoZW5wLTIwMDYtcGN0LnBkZg==" target="_blank" style="word-wrap:break-word;">507-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234534-triazolopyrimidine-compound-as-glycogen-synthase-kinase-3-inhibitors.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234536-benzoxazinone-derived-sulfonamide-compounds-and-process-for-preparation-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234535</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>507/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Feb-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>KWEEK-EN RESEARCHBEDRIJF AGRICO B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NL-8300, AA EMMELOORD,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAN DER VOSSEN, EDWIN, ANDRIES, GERARD</td>
											<td>OORTLAAN 20, NL-3572 ZM UTRECHT,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALLEFS, JOSEPHUS, JACOBUS, HENDRICUS, MARIA</td>
											<td>NAARDENSTRAAT 8, NL-8304 EJ EMMELOORD,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MUSKENS, MARINA, WOUTERA, MARIA</td>
											<td>MIDDELPLAAT 8, NL-8303 MK EMMELOORD,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 5/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP04/08683</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>03018266.1</td>
									<td>2003-08-11</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234535-a-method-for-generating-or-increasing-the-resistance-of-a-plant-to-a-plant-pathogen-of-the-phylum-oomyceta by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:18:21 GMT -->
</html>
