<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234680-a-method-for-modifying-the-sterol-content-of-host-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:22:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234680:A METHOD FOR MODIFYING THE STEROL CONTENT OF HOST CELL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD FOR MODIFYING THE STEROL CONTENT OF HOST CELL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;A method for modifying the sterol content of a host cell&quot; This invention relates to a metiiod for modifying the sterol content of a host cell, comprising transforming a host cell with a recombinant construct containing a regulatory sequence which is heterologous and functional in plants, operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide selected from the group consisting of SEQ ID NOs.2, 4, 6, 8, 10, 11, 73 and 75 or an acyl CoA: cholesterol acyltransferase-like polypeptide selected from SEQ ID NO.33 or 42 and culturing said host cell, in a known manner, under conditions wherein said host cell expresses a lecithin: cholesterol acyltransferase-like polypeptide or an acyl CoA: cholesterol acyltransferase-like polypeptide such that said host cell has a modified sterol composition as compared to host cells without the recombinant construct. This invention also relates to a method for altering oil production by a host cell.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention is directed to plant acyltransferase-like nucleic acid and amino acid sequences and constructs, and methods related to their use in altering sterol composition and/or content, and oil composition and/or content in host cells and plants.<br>
Related Art<br>
Through the development of plant genetic engineering techniques, it is now possible to produce transgenic varieties of plant species to provide plants, which have novel and desirable characteristics. For example, it is now possible to genetically engineer plants for tolerance to environmental stresses, such as resistance to pathogens and tolerance to herbicides. It is also possible to improve the nutritional characteristics of the plant, for example to provide improved fatty acid, carotenoid, sterol and tocopherol compositions.<br>
However, the number of useful nucleotide sequences for the engineering of such characteristics is thus far limited.<br>
There is a need for improved means to obtain or manipulate compositions of sterols from biosynthetic or natural plant sources. The ability to increase sterol production or alter the sterol compositions in plants may provide for novel sources of sterols for use in human and animal nutrition.<br>
Sterol biosynthesis branches from the farnesyJ diphosphate intermediate in the isoprenoid pathway. Sterol biosynthesis occurs via a mevalonate dependent pathway in mammals and higher plants (Goodwin, (1981) Biosynthesis of Isoprenoid Compounds, vol 1 (Porter, J. W. &amp; Spurgeon, S. L., eds) pp. 443-480, John Wiley and Sons, New York), while in green algae sterol biosynthesis is thought to occur via a mevalonate independent pathway (Schwender, et al. (1997) Physiology, Biochemistry, and Molecular Biology of Plant Lipids, (Williams, J. P., Khan, M U, and Lem, N. W., eds) pp. 180-182,Kluwer Academic Publishers, Norwell, MA).<br>
The solubility characteristics of sterol esters suggests that this is the storage form of sterols (Chang, etal., (1997) Annu. Rev. Biochem., 66: 613-638). Sterol O-acyltransferase enzymes such as acyl CoA: cholesterol acyltransferase (ACAT) and lecithin : cholesterol acyltransferase (LCAT) catalyze the formation of cholesterol esters, and thus are key to controlling the intracellular cholesterol storage. In yeast, it has been reported that over expression ofLAOl, a homolog of human LCAT, and phospholipid: diacylglycerol acyltransferase increased lipid synthesis (Oeikers et al., (2000) J, Biol. Chem., 26: 1560915612; Dahlqvist et al., (2000) Proc. Natl.Acad. Sci. USA, 97: 6487-6492).<br>
The characterization of various acyltransferase proteins is useful for the further study of plant sterol synthesis systems and for the development of novel and/or alternative sterol sources. Studies of plant mechanisms may provide means to further enhance, control, modify, or otherwise alter the sterol composition of plant cells. Furthermore, such alterations in sterol content and/or composition may provide a means for obtaining tolerance to stress and insect damage. Of particular interest are the nucleic acid sequences of genes encoding proteins which may be useful for applications in genetic engineering.<br>
SUMMARY OF THE INVENTION<br>
The present invention is directed to lecithin: cholesterol acyltransferase-like polypeptides (also referred to herein as LCAT) and acyl CoA: cholesterol acyltransferase-like polypeptides<br>
2<br><br>
(also referred to herein as ACAT). In particular the invention is related to polynucleotides encoding such sterol: acyltransferases, polypeptides encoded by such polynucleotides, and the use of such polynucleotides to after sterol composition and oil production. The polynucleotides of the present invention include those derived from plant sources.<br>
One aspect of the invention, therefore, is an isolated nucleic acid sequence encoding a plant lecithin: cholesterol acyltransferase-like polypeptide, a fragment of a plant lecithin: cholesterol acyltransferase-like polypeptide, a plant acyl CoA: cholesterol acyltransferase-like polypeptide or a fragment of a plant acyl CoA: cholesterol acyltransferase-like<br>
polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of SEQ ID NO: or 75. Also provided is an isolated nucleic acid sequence consisting of SEQ ID NO: 2,4,6,8,10-29,43-51,73 or75.<br>
Still another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 3 or SEQ ID NO: 3 with at least one conservative amino acid substitution; SEQ ID NO: 2; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 2; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 2; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 2 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Still another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula5'X- (R1)n-(R2)n-(R3)n-Y 3'where X is a hydrogen, Y is a hydrogen or a metal, Ri and R2 are any nucleic acid, n is an integer between 0-3000, andR.2 is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 3 or SEQ ID NO: 3 with at least one conservative amino acid substitution; SEQ ID NO: 2; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 2; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 2; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 2 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucieotide encoding a polypeptide of SEQ IDNO:5 or SEQ ID NO: 5 with at least one conservative amino acid substitution; SEQ ID NO: 4; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 4; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 4; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 4 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formu!a5'X- (R1)n-(R2)n-(R=3)n-Y 3'where X is a hydrogen, Y is a hydrogen or a metal, R1 and R2 are any nucleic acid, n is an integer between 0-3000, andR2 is<br>
3<br><br>
selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO; 5 or SEQ ID NO:5 with at least one conservative amino acid substitution; SEQ ID NO: 4; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 4; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 4; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 4 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ IDNO: 7 or SEQ ID NO: 7 with at least one conservative amino acid substitution; SEQ ID NO: 6; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 6; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 6; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 6 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula5'X- (Ri)n-(R2)n-(R3)n-Y 3'where X is a hydrogen, Y is a hydrogen or a metal,R1 and R2 are any nucleic acid, n is an integer between 0-3000, andR; is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 7 or SEQ ID NO: 7 with at least one conservative amino acid substitution; SEQ ID NO: 6; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ IDNO: 6; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 6; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 6 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ IDNO: 9 or SEQ ID NO: 9 with at least one conservative amino acid substitution; SEQ ID NO: 8; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 8; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 8; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 8 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formula5'X- (R1) n-(R2)n-(R3)n-Y3'where X is a hydrogen, Y is a hydrogen or a metal, R, and R2 are any nucleic acid, n is an integer between 0-3000, andR2 is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 9 or SEQ ID NO: 9 with at least one conservative amino acid substitution; SEQ ID NO: 8; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ IDNO: 8; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 8; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under<br>
4<br><br>
stringent conditions to SEQ ID NO: S and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 74 or SEQ ID NO: 74 with at least one conservative amino acid substitution; SEQ ID NO: 73; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 73; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 73; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 73 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucieotide of the formula5'X- (R1)n-(R2)n-(R3)n-Y3 'where X is a hydrogen, Y is a hydrogen or a metal, Ri and R2 are any nucleic acid, n is an integer between 0-3000, andR2 is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 74 or SEQ ID NO: 74 with at least one conservative amino acid substitution; SEQ ID NO: 73; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 73; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 73; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucieotide that hybridizes under stringent conditions to SEQ ID NO: 73 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 76 or SEQ ID NO: 76 with at least one conservative amino acid substitution; SEQ ID NO: 75; an isolated polynucieotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 75; an isolated polynucieotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 75; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 75 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucieotide of the formula5'X- (R1)B (R2)n (R3)n Y 3'where X is a hydrogen, Y is a hydrogen or a metal, R] and R2 are any nucleic acid, n is an integer between 0-3000, andR2 is selected from the group consisting of an isolated polynucleotide encoding a polypeptide of SEQ ID NO: 76 or SEQ ID NO: 76 with at least one conservative amino acid substitution; SEQ ID NO: 75; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 75; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 75; an isolated polynucieotide complementary to any of the foregoing; and an isolated polynucieotide that hybridizes under stringent conditions to SEQ ID NO: 75 and encodes a plant lecithin: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence comprising a polynucleotide selected from the group consisting of SEQ ID NO: 42 or a degenerate variant thereof; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ<br>
5<br><br>
ID NO: 42; an isolated polynucleotide of at least 10 amino acids thai hybridizes under stringent conditions to SEQ ID NO: 42; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 42 and encodes an acyl CoA: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides an isolated nucleic acid sequence consisting essentially of a polynucleotide of the formulaS'X- (Ri)n-(R2V(R3)n-Y3'where X is a hydrogen, Y is a hydrogen or a metal, R, and R2 are any nucleic acid, n is an integer between 0-3000, andR2 is selected from the group consisting of SEQ ID NO: 42 or a degenerate variant thereof; an isolated polynucleotide that has at least 70%, 80%, 90%, or 95% sequence identity with SEQ ID NO: 42; an isolated polynucleotide of at least 10 amino acids that hybridizes under stringent conditions to SEQ ID NO: 42; an isolated polynucleotide complementary to any of the foregoing; and an isolated polynucleotide that hybridizes under stringent conditions to SEQ ID NO: 42 and encodes a acyl CoA: cholesterol acyltransferase-like. polypeptide.<br>
Also provided is a recombinant nucleic acid construct comprising a regulatory sequence operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide and/or an acyl CoA: cholesterol acyltransferase-like polypeptide. In one embodiment, the sterol acyl transferases are plant sterol acyl transferases. In another embodiment, the recombinant nucleic acid constructs further comprises a termination sequence. The regulatory sequence can be a constitutive promoter, an inducible promoter, a developmentally regulated promoter, a tissue specific promoter, an organelle specific promoter, a seed specific promoter or a combination of any of the foregoing. Also provided is a plant containing this recombinant nucleic acid construct and the seed and progeny from such a plant. This recombinant nucleic acid construct can also be introduced into a suitable host cell to provide yet another aspect of the invention. If the host cell is a plant host cell, the eel! can be used to generate a plant to provide another aspect of the invention. Further aspects include seed and progeny from such a plant.<br>
Yet another aspect is a purified polypeptide comprising, SEQ ID NO: 3, SEQ ID NO: 5, SEQ ID NO: 7, SEQ ID NO: 9, SEQ ID NO: 74, SEQ ID NO: 76, or any of the preceding sequences with at least one conservative amino acid substitution.<br>
Still another aspect provides a purified immunogenic polypeptide comprising at least 10 consecutive amino acids from an amino acid sequence selected from the group consisting of SEQ ID NO: 3,5,7,9,74,76 and any of the preceding sequences containing at least one conservative amino acid substitution. Also provided are antibodies, either polyclonal or monoclonal, that specifically bind the preceding immunogenic polypeptides.<br>
One aspect provides a method for producing a lecithin: cholesterol acyltransferase like polypeptide or an acyl CoA: cholesterol acyltransferase-like polypeptide comprising culturing a host cell containing any recombinant nucleic acid construct of the present invention under condition permitting expression of said lecithin: cholesterol acyltransferase-like polypeptide or acyl CoA: cholesterol acyltransferase-like polypeptide.<br>
Another aspect provides a method for modifying the sterol content of a host cell, comprising transforming a host eel) with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide and culturing said host cell under conditions wherein said host cell expresses a<br>
6<br><br>
lecithin: cholesterol acyltrantisferase-like polypeptide such that said host cell has a modified sterol composition as compared to host cells without the recombinant construct.<br>
An additional aspect is a method for modifying the sterol content of a host cell comprising transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoA: cholesterol acyltransferase-like polypeptide and culturing said host cell under conditions wherein said host cell expresses an acy] CoA: cholesterol acyltransferase-like polypeptide such that said host cell has a modified sterol composition as compared to host cells without the recombinant construct.<br>
A further aspect is a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in modified sterol composition of said plant as compared to the same plant without said recombinant construct.<br>
Another aspect provides a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoA: cholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in modified sterol composition of said plant as compared to the same plant without said recombinant construct.<br>
In a further aspect is provided oil obtained from any of the plants or host cells of the present invention.<br>
In still another aspect is provided a method for producing an oil with a modified sterol composition comprising providing any of the plants or host cells of the present invention and extracting oil from the plant by any known method. Also provided is an oil produced by the preceding method.<br>
Still another aspect provides a method for altering oil production by a host cell comprising, transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide and culturing the host cell under conditions wherein the host cell expresses a lecithin: cholesterol acyltransferase-like polypeptide such that the host cell has an altered oil production as compared to host cells without the recombinant construct.<br>
Another aspect provides a method for altering oil production by a host cell comprising, transforming a host cell with a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl CoA: cholesterol acyltransferase-like polypeptide and culturing the host cell under conditions wherein the host cell expresses an acyl CoA: cholesterol acyltransferase-like polypeptide such that the host cell has an altered oil production as compared to host cells without the recombinant construct<br>
Also provided is a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding a lecithin: cholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in an altered production of oil by said plant as compared to the same plant without said recombinant construct.<br>
In a further aspect is provided a plant comprising a recombinant construct containing a regulatory sequence operably linked to a polynucleotide encoding an acyl<br>
7<br><br>
CoA: cholesterol acyltransferase-like polypeptide wherein expression of said recombinant construct results in an altered production of oil by said plant as compared to the same plant without said recombinant construct.<br>
Additional aspects provide a food, food ingredient or food product comprising any oil, plant or host cell of the present invention; a nutritional or dietary supplement comprising any oil, plant or host cell of the present invention; and a pharmaceutical composition comprising any oil, plant or host cell of the present invention along with a suitable diluent, carrier or<br>
excipient.<br>
Additional aspects will be apparent from the descriptions and examples that follow.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
These and other features, aspects, and advantages of the present invention will become better understood with regard to the following description, appended claims and accompanying figures where:<br>
Figure 1 shows an alignment of yeast, human and rat lecithin: cholesterol acyltransferase<br>
protein sequences with Arabidopsis LCAT1, LCAT2, LCAT3, and LCAT4 deduced amino acid sequences.<br>
Figure 2 shows the results of NMR sterol ester analysis on T2 seed from plant expressing LCAT4 under the control of the napin promoter (pCGN9998).<br>
Figure 3 shows the results of HPLC/MS sterol analysis on oit extracted from T2 seed from control lines (pCGN8640) and lines expressing LCAT3 (pCGN9968) under the control of the napin promoter.<br>
Figure 4 shows the results of HPLC/MS sterol analysis on oil extracted from T2 seed from control lines (pCGN8640), and plant line expressing LCAT1 (pCGN9962), LCAT2 (pCGN9983), LCAT3 (pCGN9968), and LCAT4 (pCGN9998) under the control of the napin promoter. Additionally, data from 3 lines expressing LCAT4 under the control of the 35S promoter (pCGN9996) are shown.<br>
Figure 5 shows the results of Nir analysis of the oil content of T2 seed from control lines (pCGN8640), and plant lines expressing LCAT1 (pCGN9%2), LCA.T2 (pCGN99&amp;3), and LCAT3 (pCGN9968) under the control of the napin promoter.<br>
Additionally, data from 16 lines expressing LCAT2 under the control of the 35S promoter (pCGN9981) are shown.<br>
DETAILED DESCRIPTION<br>
The following detailed description is provided to aid those skilled in the art in practicing the present invention. Even so, this detailed description should not be construed to unduly limit the present invention as modifications and variations in the embodiments discussed herein can be made by those of ordinary skill in the art without departing from the spirit or scope of the present inventive discovery.<br>
All publications, patents, patent applications and other references cited in this application are herein incorporated by reference in their entirety as if each individual publication, patent, patent application or other reference were specifically and individually indicated to be<br>
8<br><br>
incorporated by reference.<br>
The present invention relates to lecithin: cholesterol acyltransferase, particularly the isolated nucleic acid sequences encoding lecithin: cholesterol-like polypeptides (LCAT) from plant sources and acyl CoA: cholesterol: acyltransferase, particularly the isolated nucleic acid sequences encoding acyl CoA: cholesterol acyltransferase-Uke polypeptides (ACAT) from plant sources. Lecithin: cholesterol acyltransferase-Like as used herein includes any nucleic acid sequence encoding an amino acid sequence from a plant source, such as a protein, polypeptide or peptide, obtainable from a cell source, which demonstrates the ability to utilize lecithin (phosphatidyl choline) as an acyl donor for acylation of sterols or glycerides to form esters under enzyme reactive conditions along with such proteins polypeptides and peptides. Acyl CoA: cholesterol acyltransferase-like as used herein includes any nucleic acid sequence encoding an amino acid sequence from a plant source, such as a protein, polypeptide or peptide, obtainable from a cell source, which demonstrates the ability to utilize acyl CoA as an acyl donor for acylation of sterols or glycerides to form esters under enzyme reactive conditions along with such proteins polypeptides and peptides. By” enzyme reactive conditions” is meant that any necessary conditions are available in an environment (i. e., such factors as temperature, pH, lack of inhibiting substances), which will permit the enzyme to function.<br>
The term “sterol” as applied to plants refers to any chwal tetracyclic isopentenoid which may be formed by cyclization of squalene oxide through the transition state possessing stereochemistry similar to the trans-syn-trans-anti-trans-anti configuration, for example, protosteroid cation, and which retains a polar group at C-3 (hydroxyl or keto), an all-trans-anti stereochemistry in the ring system, and a side-chain 2QR-cotvfiguration (Parker, et al.(1992) In Nes, et al., Eds., Regulation of Isopentenoid Metabolism, ACS Symposium Series No. 497, p. 110; American Chemical Society, Washington, D. C).<br>
Sterols may or may not contain a C-5-C-6 double bond, as this is a feature introduced late in the biosynthetic pathway. Sterols contain aC8-C, 0 side chain at the C-17 position.<br>
The term “phytosterol” which applies to sterols found uniquely in plants, refers to a sterol containing a C-5, and in some cases a C-22, double bond. Phytosterols are further characterized by alkylation of the C-l 7 side-chain with a methyl or ethyl substituent at the C-24 position. Major phytosterols include, but are not limited to, sitosterol, stigmasterol, campesterol, brassicasterol, etc. Cholesterol, which lacks a C-24 methyl or ethyl side chain, is found in plants, but is not unique thereto, and is not a “phytosterol.”<br>
“Phytostanols” are saturated forms of phytosterols wherein the C-5 and, when present, C-22 double bond (s) is (are) reduced, and include, but are not limited to, sitostanol, campestanol, and 22-dihydrobrassicastanol.<br>
“Sterol esters” are further characterized by the presence of a fatty acid or phenolic acid moiety rather than a hydroxyl group at the C-3 position.<br>
The term “sterol “includes sterols, phytosterols, phytosterol esters, phytostanols, and phytostanol esters.<br>
The term “sterol compounds” includes sterols, phyotsterols, phytosterol esters, phytostanols, and phytostanol esters.<br>
9<br><br>
The term “phytosterol compound” refers to at least one phytosterol, at least one phytostero! ester, or a mixture thereof.<br>
The term “phytostanol compound” refers to at least one phytostanoi, at least one phytostanol ester, or a mixture thereof.<br>
The term “glyceride” refers to a fatty acid ester of glycerol and includes mono-, di-, and tri-acylglycerols.<br>
As used herein, “recombinant construct” is defined either by its method of production or its structure. In reference to its method of production, e. g., a product made by a process, the process is use of recombinant nucleic acid techniques, e. g., involving human intervention in the nucleotide sequence, typically selection or production.<br>
Alternatively, in terms of structure, it can be a sequence comprising fusion of two or more nucleic acid sequences which are not naturally contiguous or operatively linked to each other As used herein, “regulatory sequence” means a sequence of DNA concerned with controlling expression of a gene; e. g. promoters, operators and attenuators. A “heterologous regulatory sequence” is one which differs from the regulatory sequence naturally associated with a gene.<br>
As used herein, “polynucleotide” and “oligonucleotide” are used interchangeably and mean a polymer of at least two nucleotides joined together by a phosphodiester bond and may consist of either ribonucleotides or deoxynucleotides.<br>
As used herein, “sequence” means the linear order in which monomers appear in a polymer, for example, the order of amino acids in a polypeptide or the order of nucleotides in a<br>
polynucleotide.<br>
As used herein, “polypeptide”, “peptide”, and “protein” are used interchangeably and mean a compound that consist of two or more amino acids that are linked by means of peptide bonds.<br>
As used herein, the terms “complementary” or “complementarity” refer to the pairing of bases, purines and pyrimidines, that associate through hydrogen bonding in double stranded nucleic acids. For example, the following base pairs are complementary: guanine and cytosine; adenine and thymine; and adenine and uracil. The terms, as used herein, include complete and partial complementarity.<br>
Isolated proteins, Polypeptides and Polynucleotides<br>
A first aspect of the present invention relates to isolated LCAT polynucleotides.<br>
The polynucleotide sequences of the present invention include isolated polynucleotides that encode the polypeptides of the invention having a deduced amino acid sequence selected from the group of sequences set forth in the Sequence Listing and to other polynucleotide sequences closely related to such sequences and variants thereof.<br>
The invention provides a polynucleotide sequence identical over its entire length to each coding sequence as set forth in the Sequence Listing. The invention also provides the coding sequence for the mature polypeptide or a fragment thereof, as well as the coding sequence for the mature polypeptide or a fragment thereof in a reading frame with other coding sequences, such as those encoding a leader or secretory sequence, a pre-, pro-, or prepro-protein<br>
10<br><br>
sequence. The polynucleotide can also include non-coding sequences, including for example, but not limited to, non-coding 5'and 3'sequences, such as the transcribed, untranslated sequences, termination signals, ribosome binding siies, sequences that stabilizem RNA, introns, polyadenylation signals, and additional coding sequence that encodes additional amino acids. For example, a marker sequence can be included to facilitate the purification of the fused polypeptide. Polynucleotides of the present invention also include polynucleotides comprising a structural gene and the naturally associated sequences that control gene expression.<br>
The invention also includes polynucleotides of the formu!a:X- (R1)-- (R2)- (R3)--y wherein, at the 5'end, X is hydrogen, and at the 3'end, Y is hydrogen or a metal, R, andR3 are any nucleic acid residue, n is an integer between 0 and 3000, preferably between 1 and 1000 andR2 is a nucleic acid sequence of the invention, particularly a nucleic acid sequence selected from the group set forth in the Sequence Listing and preferably SEQ ID<br>
NOs: 2,4,6,8,10-29,33,42-51,73 and 75. In the formula,R: is oriented so that its 5' end residue is at the left, bound to R1, and its 3'end residue is at the right, bound to R3.<br>
Any stretch of nucleic acid residues denoted by either R group, where R is greater than 1. may be either a heteropolymer or a homopolymer, preferably a heteropolymer.<br>
The invention also relates to variants of the polynucleotides described herein that encode for variants of the polypeptides of the invention. Variants that are fragments of the polynucleotides of the invention can be used to synthesize full-length polynucleotides of the invention. Preferred embodiments are polynucleotides encoding polypeptide variants wherein 5 to 10,1 to 5,1 to 3,2,1 or no amino acid residues of a polypeptide sequence of the invention are substituted, added or deleted, in any combination. Particularly preferred are substitutions, additions, and deletions that are silent such that they do not alter the properties or activities of the polynucleotide or polypeptide.<br>
Further preferred embodiments of the invention that are at least 50%, 60%, or 70% identical over their entire length to a polynucleotide encoding a polypeptide of the invention, and polynucleotides that are complementary to such polynucleotides. More preferable are polynucleotides that comprise a region that is at least 80% identical over its entire length to a polynucleotide encoding a polypeptide of the invention and polynucleotides that are complementary thereto. In this regard, polynucleotides at least 90% identical over their entire length are particularly preferred, those at least 95% identical are especially preferred. Further, those with at least 97% identity are highly preferred and those with at least 98% and 99% identity are particularly highly preferred, with those at least 99% being the most highly preferred.<br>
Preferred embodiments are polynucleotides that encode polypeptides that retain substantially the same biological function or activity as determined by the methods described herein as the mature polypeptides encoded by the polynucleotides set forth in the Sequence Listing.<br>
The invention further relates to polynucleotides that hybridize to the above described sequences. In particular, the invention relates to polynucleotides that hybridize under stringent conditions to the above-described polynucleotides. An example of stringent hybridization conditions is overnight incubation at42 C in a solution comprising50% formamide, 5x SSC (150 mMNaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH<br>
11<br><br>
7.6), 5x Denhardt’s solution, 10% dextran sulfate, and 20 micrograms/mi Hi liter denatured, sheared salmon sperm DNA, followed by washing the hybridization support in0. I x SSC at approximately65 C. Also included are polynucleotides that hybridize under a wash stringency of O. IX SSC or <rti sspe c. other hybridization and wash conditions are well known exemplified in></rti>
Sambrook, etal., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY (1989), particularly Chapterl 1.<br>
The invention also provides a polynucleotide consisting essentially of a polynucleotide sequence obtainable by screening an appropriate library containing the complete gene for a polynucleotide sequence set for in the Sequence Listing under stringent hybridization conditions with a probe having the sequence of said polynucleotide sequence or a fragment thereof; and isolating said polynucleotide sequence. Methods for screening libraries are well known in the art and can be found for example in Sambrook, etal., Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor, NY (1989), particularly Chapter 8 and Ausubel et al., Short Protocols in Molecular Biology,3rd ed, Wiley and Sons,1995, chapter 6. Nucleic acid sequences useful for obtaining such a polynucleotide include, for example, probes and primers as described herein and in particular SEQ ID NO:2,4,6,8,10-29,33,42-51,73 and 75. These sequences are particularly useful in screening libraries obtained from Arabidopsis, soybean and corn for sequences encoding lecithin: cholesterol acyltransferase and lecithin: cholesterol acyltransferase-like polypeptides and for screening libraries for sequences encoding acyl<br>
CoA: cholesterol acyl transferase and acyl CoA: cholesterol acyl transferase-like polypeptides.<br>
As discussed herein regarding polynucleotide assays of the invention, for example, polynucleotides of the invention can be used as a hybridization probe for RNA, cDNA , or genomic DNA to isolate full length cDNAs or genomic clones encoding a polypeptide and to isolatec DNA or genomic clones of other genes that have a high sequence similarity to a polynucleotide set forth in the Sequence Listing and in particular SEQ ID NO:2,4,6,8, 10-29,33,42-51,73 and 75. Such probes will generally comprise at least 15 bases.<br>
Preferably such probes will have at least 30 bases and can have at least 50 bases. Particularly preferred probes will have between 30 bases and 50 bases, inclusive.<br>
The coding region of each gene that comprises or is comprised by a polynucleotide sequence set forth in the Sequence Listing may be isolated by screening using a DNA sequence provided in the Sequence Listing to synthesize an oligonucleotide probe. A labeled oligonucleotide having a sequence complementary to that of a gene of the invention is then used to screen a library of cDNA, genomic DNA form RNA to identify members of the library which hybridize to the probe. For example, synthetic oligonucleotides are prepared which correspond to the LCAT EST sequences. The oligonucleotides are used as primers in polymerase chain reaction (PCR) techniques to obtain 5'and 3'terminal sequence of LCAT genes. Alternatively, where oligonucleotides of low degeneracy can be prepared from particular LCAT peptides, such probes may be used directly to screen gene libraries for LCAT gene sequences. In particular, screening of cDNA libraries in phage vectors is useful in such methods due to lower levels of background hybridization.<br>
Typically, a LCAT sequence obtainable from the use of nucleic acid probes will show 60-<br>
12<br><br>
70% sequence identity between the target LCAT sequence and the encoding sequence used as a probe. However, lengthy sequences with as little as 50-60% sequence identity may also be obtained. The nucleic acid probes may be a lengthy fragment of the nucleic acid sequence, or may also be a shorter, oligonucleotide probe. When longer nucleic acid fragments are employed as probes (greater than about 100 bp), one may screen at lower stringencies in order to obtain sequences from the target sample, which have 20-50% deviation (i. e-, 50-80% sequence homology) from the sequences used as probe. Oligonucleotide probes can be considerably shorter than the entire nucleic acid sequence encoding an LCAT enzyme, but should be at least about 10, preferably at least aboutl5, and more preferably at least about 20 nucleotides. A higher degree of sequence identity is desired when shorter regions are used as opposed to longer regions. It may thus be desirable to identify regions of highly conserved amino acid sequence to design oligonucleotide probes for detecting and recovering other related LCAT genes. Shorter probes are often particularly useful for polymerase chain reactions (PCR), especially when highly conserved sequences can be identified. (See, Gould, etal., PNAS USA (1989) 86: 1934-1938.).<br>
Another aspect of the present invention relates to LCAT polypeptides. Such polypeptides include isolated polypeptides set forth in the Sequence Listing, as well as polypeptides and fragments thereof, particularly those polypeptides which exhibit LCAT activity and also those polypeptides which have at least 50%, 60% or 70% identity,' preferably at least 80% identity, more preferably at least 90% identity, and most preferably at least 95% identity to a polypeptide sequence selected from the group of sequences set forth in the Sequence Listing, and also include portions of such polypeptides, wherein such portion of the polypeptide preferably includes at least 30 amino acids and more preferably includes at least 50 amino acids.<br>
‘Identity"” as is well understood in the art, is a relationship between two or more polypeptide sequences or two or more polynucleotide sequences, as determined by comparing the sequences. In the art, “identity” also means the degree of sequence relatedness between polypeptide or polynucleotide sequences, as determined by the match between strings of such sequences. “Identity” can be readily calculated by known methods including, but not limited to, those described in Computational Molecular Biology, Lesk,<br>
A. M., ed., Oxford University Press, New York (1988); Biocomputing : Informatics and Genome Projects, Smith, D. W.,ed., Academic Press, New York, 1993; Computer Analysis of Sequence Data, Part I, Griffin, A. M. and Griffin, H. G., eds., Humana Press, New Jersey (1994); Sequence Analysis in Molecular Biology, vonHeinje, G., Academic Press (1987); Sequence Analysis Primer, Gribskov, M. and Devereux, J., eds., Stockton Press, New York (1991); and Carillo, H., and Lipman, D.,SIAM J Applied Math, 48: 1073 (1988).<br>
Methods to determine identity are designed to give the largest match between the sequences tested. Moreover, methods to determine identity are codified in publicly available programs-Computer programs which can be used to determine identity between two sequences include, but are not limited to, GCG (Devereux, J., et al., Nucleic Acids<br>
Research 12(1): 387(1984); suite offive BLAST programs, three designed for nucleotide sequences queries (BLASTN, BLASTX, and TBLASTX) and two designed for protein sequence queries (BLASTP and TBLASTN) (Coulson, Trends in Biotechnology, 12 : 7680 (1994); Birren, etal., Genome Analysis, 1 : 543-559 (1997)). The BLAST X program is publicly available from NCBI and other sources (BLAST Manual, Altschul, S., etal., NCBI NLMNIH, Bethesda, MD 20894; Altschul, S., etal., J Mol. Bio!., 215: 403-410 (1990)). The well known Smith Waterman algorithm can also be used to determine identity.<br>
13<br><br>
Parameters for polypeptide sequence comparison typically include the following: Algorithm: Needleman and Wunsch, J. Mol. Biol. 48: 443-453 (1970) Comparison matrix: BLOSSUM62 from Hentik off and Hentik off, Proc.Natl-<br>
Acad. Scf USA 89: 10915-10919 (1992)<br>
Gap Penalty: 12<br>
Gap length Penalty: 4<br>
A program which can be used with these parameters is publicly available as the “gap”<br>
program from Genetics Computer Group, Madison Wisconsin. The above parameters along with no penalty for end gap are the default parameters for peptide comparisons.<br>
Parameters for polynucleotide sequence comparison include the following:<br>
Algorithm: Needleman and Wunsch, J. Mol. Biol. 48: 443-53 (1970)<br>
Comparison matrix: matches = +10; mismatches = 0<br>
Gap Penalty: 50<br>
Gap Length Penalty: 3<br>
A program which can be used with these parameters is publicly available as the “gap”<br>
program from Genetics Computer Group, Madison Wisconsin. The above parameters are the<br>
default parameters for nucleic acid comparisons.<br>
The invention also includes polypeptides of the formula: XOROnORz) OR3) n I' wherein, at the amino terminus, X is hydrogen, and at the carboxyl terminus, Y is hydrogen or a metal, R, and R3 are any amino acid residue, n is an integer between 0 and 1000, andR2 is an amino acid sequence of the invention, particularly an amino acid sequence selected from the group set forth in the Sequence Listing and preferably SEQ ID<br>
NOs: 3,5,7,9,74 and 76. In the formula, R2 is oriented so that its amino terminal residue is at the left, bound to R,, and its carboxy terminal residue is at the right, bound to R3.<br>
Any stretch of amino acid residues denoted by either R group, where R is greater than I, may be either s heteropolymer or a homopolymer, preferably a heteropolymer.<br>
Polypeptides of the present invention include isolated polypeptides encoded by a polynucleotide comprising a sequence selected from the group of a sequence contained in SEQ ID NOs: 2,4,6,8,73 and 75.<br>
The polypeptides of the present invention can be mature protein or can be part of a fusion protein.<br>
Fragments and variants of the polypeptides are also considered to be a part of the invention. A fragment is a variant polypeptide which has an amino acid sequence that is entirely the same as part but not all of the amino acid ssquence of the previously described polypeptides. The fragments can be”free-standing”or comprised within a larger polypeptide of which the fragment forms a part or a region, most preferably as a single continuous region. Preferred fragments are biologically active fragments which are those fragments that mediate activities of the polypeptides of the invention, including those with similar activity or improved activity or with a decreased activity. Also included are those polypeptides and polypeptide fragments that are antigenic or immunogenic in an animal, particularly a human and antibodies, either polyclonal or monoclonal that specifically bind the antigenic fragments. In one preferred embodiment, such antigenic or immuniogenic fragments comprise at least 10 consecutive<br>
14<br><br>
amino acids from the amino acid sequences disclosed herein or such sequences with at least one conservative amino acid substitution.<br>
In additional embodiments, such antigenic or immunogenic fragments comprise at least 15, at least 25, at least 50 or at least 100 consecutive amino acids from the amino acid sequences disclosed herein or such sequences with at least one conservative amino acid substitution. Methods for the production of antibodies from polypeptides and polypeptides conjugated to carrier molecules are well known in the art and can be found for example in Ausubel et al., ShortProtocols in Molecular Biology, 3rd ed., Wiley &amp; Sons, 1995, particularly chapter 11.<br>
Variants of the polypeptide also include polypeptides that vary from the sequences set forth in the Sequence Listing by conservative amino acid substitutions, substitution of a residue by another with like characteristics. Those of ordinary skill in the art are aware that modifications in the amino acid sequence of a peptide, polypeptide, or protein can result in equivalent, or possibly improved, second generation peptides, etc., that display equivalent or superior functional characteristics when compared to the original amino acid sequence. The present invention accordingly encompasses such modified amino acid sequences. Alterations can include amino acid insertions, deletions, substitutions, truncations, fusions, shuffling of subunit sequences, and the like, provided that the peptide sequences produced by such modifications have substantially the same functional properties as the naturally occurring counterpart sequences disclosed herein.<br>
One factor that can be considered in making such changes is the hydropathic index of amino acids. The importance of the hydropathic amino acid index inconferring interactive biological function on a protein has been discussed by Kyte and Doolittle(J.<br>
Mol.BioI., 157: 105-132,1982). It is accepted that the relative hydropathic character of amino acids contributes to the secondary structure of the resultant protein. This, in turn, affects the interaction of the protein with molecules such as enzymes, substrates, receptors, DNA, antibodies, antigens, etc.<br>
Based on its hydrophobicity and charge characteristics, each amino acid has been assigned a hydropathic index as follows: isoleucine (+4.5); valine (+4.2); leucine (+3.8); phenylalanine (+2.8); cysteine/cystine (+2.5); methionine (+1.9); alanine (+1.8); glycine (- 0.4); threonine (-0.7); serine (-0.8); tryptophan (-0.9); tyrosine (-1.3); proline (-1.6); histidine (-3.2); glutamate/glutamine/aspartate/asparagine (-3.5); lysine (-3.9); and arginine (-4.5).<br>
As is known in the art, certain amino acids in a peptide or protein can be substituted for other amino acids having a similar hydropathic index or score and produce a resultant peptide or protein having similar biological activity, i. e., which still retains biological functionality. In making such changes, it is preferable that amino acids having hydropathic indices within+2 are substituted for one another. More preferred substitutions are those wherein the amino acids have hydropathic indices within 1 Most preferred substitutions are those wherein the amino acids have hydropathic indices within0.5.<br>
Like amino acids can also be substituted on the basis of hydrophilicity. U. S. Patent No. 4,554,101 discloses that the greatest local average hydrophilicity of a protein, as governed by the hydrophilicity of its adjacent amino acids, correlates with a biological<br>
property of the protein. The following hydrophilicity values have been assigned to amino<br>
15<br><br>
acids: arginine/lysine (+3.0); aspartate/glutamate (+3.0+1); serine (+0.3);asparagine/glutamine (+0.2); glycine (0); threonine (-0.4); proline (-0.51); alanine/histidine (-0.5); cysteine (-1.0); methionine (-1.3); valine (-1.5); leucine/isoleucine (-1.8); tyrosine (-2.3); phenylalanine (-2.5); and tryptophan (-3.4). Thus, one amino acid in a peptide, polypeptide, or protein can be substituted by another amino acid having a similar hydrophilicitj' score and still produce a resultant protein having similar biological activity, i. e., still retaining correct biological function. In making such changes, amino acids having hydropathic indices within2 are preferably substituted for one another, those within+1 are more preferred, and those within0.5 are most preferred.<br>
As outlined above, amino acid substitutions in the peptides of the present invention can be based on the relative similarity of the amino acid side-chain substituents, for example, their hydrophobicity, hydrophilicity, charge, size, etc. Exemplary substitutions that take various of the foregoing characteristics into consideration in order to produce conservative amino acid changes resulting in silent changes within the present peptides, etc., can be selected from other members of the class to which the naturally occurring amino acid belongs. Amino acids can be divided into the following four groups: (I) acidic amino acids; (2) basic amino acids; (3) neutral polar amino acids; and (4) neutral nonpolar amino acids. Representative amino acids within these various groups include, but are not limited to: (1) acidic (negatively charged) amino acids such as aspartic acid and glutamic acid; (2) basic (positively charged) amino acids such as arginine, histidine, and lysine; (3) neutral polar amino acids such as glycine, serine, threonine, cysteine, cystine, tyrosine, asparagine, and glutamine; and (4) neutral non-polar amino acids such as alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan, and methionine. It should be noted that changes which are not expected to be advantageous can also be useful if these result in the production of functional sequences.<br>
Variants that are fragments of the polypeptides of the invention can be used to produce the corresponding full length polypeptide by peptide synthesis. Therefore, these variants can be used as intermediates for producing the full-length polypeptides of the invention.<br>
The polynucleotides and polypeptides of the invention can be used, for example, in the transformation of host cells, such as plant cells, animal cells, yeast cells, bacteria, bacteriophage, and viruses, as further discussed herein.<br>
The invention also provides polynucleotides that encode a polypeptide that is a mature protein plus additional amino or carboxy1-terminal amino acids, or amino acids within the mature polypeptide (for example, when the mature form of the protein has more than one polypeptide chain). Such sequences can, for example, play a role in the processing of a protein from a precursor to a mature form, allow protein transport, shorten or lengthen proteinhalf-life, or facilitate manipulation of the protein in assays or production. It is contemplated that cellular enzymes can be used to remove any additional amino acids from the mature protein.<br>
A precursor protein, having the mature form of the polypeptide fused to one or more prosequences may be an inactive form of the polypeptide. The inactive precursors generally are activated when the prosequences are removed. Some or all of the prosequences may be removed prior to activation. Such precursor protein are generally called proproteins.<br>
Preparation of Expression Constructs and Methods of Use<br>
16<br><br>
Of interest is the use of the nucleotide sequences in recombinant DNA constructs to direct the transcription or transcription and translation (expression) of the acyltransferase sequences of the present invention in a host cell. Of particular interest is the use of the polynucleotide sequences of the present invention in recombinant DNA constructs to direct the transcription or transcription and translation (expression) of the acyltransferase sequences of the present invention in a host plant cell.<br>
The expression constructs generally comprise a regulatory sequence functional in a host cell operably linked to a nucleic acid sequence encoding a lecithin: cholesterol acyltransferase-like polypeptide or acyl CoA: cholesterol acyltransferase-like polypeptide of the present invention and a transcriptional termination region functional in a host plant cell. Of particular interest is the use of promoters (also referred to as transcriptional initiation regions) functional in plant host cells.<br>
Those skilled in the art will recognize that there are a number of promoters which are functional in plant cells, and have been described in the literature including constitutive, inducible, tissue specific, organelle specific, developmentally regulated and environmentally regulated promoters. Chloroplast and plastid specific promoters, chloroplast or plastid functional promoters, and chloroplast or plastid operable promoters are also envisioned.<br>
One set of promoters are constitutive promoters such as the CaMV35S orFMV35S promoters that yield high levels of expression in most plant organs. Enhanced or duplicated versions of the CaMV35S and FMV35S promoters are useful in the practice of this invention (Odell, et al. (1985) Nature 313: 810-812; Rogers, U. S. Patent Number 5,378,619). Other useful constitutive promoters include, but are not limited to, the mannopine synthase (mas) promoter, the nopaline synthase (nos) promoter, and the octopine synthase(ocs) promoter.<br>
Useful inducible promoters include heat-shock promoters (Ou-Lee et al. (1986)<br>
Proc. Natl. Acad. Sci. USA 83: 6815; Ainley et al. (1990)Plant Mol. Biol. 14: 949), anitrate- inducible promoter derived from the spinach nitrite reductase gene (Back et al.<br>
(1991) Plant Mol. Biol. 17: 9), hormone-inducible promoters(Yamaguchi-Shinozaki et al.<br>
(1990) Plant Mol. Biol. 15: 905; Kares et al. (1990) Plant Mol. Biol. 15: 905), and light-inducible promoters associated with the small subunitof RuBP carboxylase and LHCP gene families(Kuhlemeier et al. (1989) PlantCell 1: 471; Feinbaum et al.(1991) Mol. Gen. Genet. 226: 449; Weisshaar et<br>
Lasquerella hydroxylase, and barley aldose reductase promoters (Bartels (1995) Plant!. 7: 809-822), the EA9 promoter(U. S. Patent 5,420,034), and theBce4 promoter (U. S. Patent 5,530,194).Useful embryo-specific promoters include the corn globulin 1 and oleosin promoters. Useful endosperm-specific promoters include the riceglutelin-1 promoter, the promoters for thelow-pl ss amylase gene (Amy32b) (Rogers et al. (1984)J. Biol. Chem.<br>
259: 12234), the high-piss amylase gene(Amy 64) (Khurseed et al. (1988)J. Biol. Chem. <br>
263: 18953), and the promoter for a barley thiol protease gene (“Aleurain”) (Whittier et al. (1987) <br>
Nucleic Acids Res. 15:2515).<br>
Of particular interest is the expression of the nucleic acid sequences of the present invention<br>
17<br><br>
from transcription initiation regions which are preferentially expressed in a plant seed tissue. Examples of such seed preferential transcription initiation sequences include those sequences derived from sequences encoding plant storage protein genes or from genes involved in fatty acid biosynthesis in oilseeds. Examples of such promoters include the 5'regulatory regions from such genes as napin (Kridl etal., Seed ScLRes. 1: 209 : 219 (1991)), phaseolin, zein, soybean trypsin inhibitor, ACP, stearoyl-ACP desaturase, soybean a'subunitof ss-conglycinin (soy 7s, (Chen etal., Proc. Natl.Acad. Sci., 83: 85608564 (1986))) and oleosin. Seed-specific gene regulation is discussed in EP 0 255 378 Bl and U. S. Patents 5,420,034 and 5,608,152. Promoter hybrids can also be constructed to enhance transcriptional activity(Hoffman, U. S. Patent No. 5,106,739), or to combine desired transcriptional activity and tissue specificity.<br>
It may be advantageous to direct the localization of proteinsconferring LCAT to a particular subcellular compartment, for example, to the mitochondrion, endoplasmic reticufum, vacuoles, chloroplast or other plastidic compartment. For example, where the genes of interest of the present invention will be targeted to plastids, such as chloroplasts, for expression, the constructs will also employ the use of sequences to direct the gene to the plastid. Such sequences are referred to herein as chloroplast transit peptides (CTP) or plastid transit peptides (PTP). In this manner, where the gene of interest is not directly inserted into the plastid, the expression construct will additionally contain a gene encoding a transit peptide to direct the gene of interest to the plastid. The chloroplast transit peptides may be derived from the gene of interest, or may be derived from a heterologous sequence having a CTP. Such transit peptides are known in the art. See, for example, VonHeijne et al. (1991)PlantMol. Biol. Rep. 9: 104-126;Clark etal. (]989)L Biol.Chem.<br>
264: 17544-17550; della-Cioppa et al. (1987)Plant Physiol. 84: 965-968; Romer et al.<br>
(1993) Biochem. Biophys. Res Commun. 196: 1414-1421; and,Shah et al. (1986) Science 233:478-481.<br>
Depending upon the intended use, the constructs may contain the nucleic acid sequence which encodes the entire LCAT protein, a portion of the LCAT protein, the entire AC AT protein, or a portion of the ACAT protein. For example, where antisense inhibition of a given LCAT or ACAT protein is desired, the entire sequence is not required. Furthermore, where LCAT or ACAT sequences used in constructs are intended for use as probes, it may be advantageous to prepare constructs containing only a particular portion of a LCAT or ACAT encoding sequence, for example a sequence which is discovered to encode a highly conserved region.<br>
The skilled artisan will recognize that there are various methods for the inhibition of expression of endogenous sequences in a host cell. Such methods include, but are not limited to antisense suppression (Smith, etal. (1988) Nature 334: 724-726), cosuppression (Napoli, et al. (1989) PfantCell 2 : 279-289), ribozymes (PCT Publication WO 97/10328), and combinations of sense and antisense Waterhouse, et al. (1998) Proc.<br>
NatLAcad. Sci, USA 95: 13959-13964. Methods for the suppression of endogenous sequences in a host cell typically employ the transcription or transcription and translation of at least a portion of the sequence to be suppressed. Such sequences may be homologous to coding as well as non-coding regions of the endogenous sequence.<br>
Regulatory transcript termination regions may be provided in plant expression constructs of<br>
18<br><br>
this invention as well. Transcript termination regions may be provided by the DNA sequence encoding the diacylglycerol acyltransferase or a convenient transcription termination region derived from a different gene source, for example, the transcript termination region which is naturally associated with the transcript initiation region. The skilled artisan will recognize that any convenient transcript termination region which is capable of terminating transcription in a plant cell may be employed in the constructs of the present invention.<br>
Alternatively, constructs may be prepared to direct the expression of the LCAT or ACAT sequences directly from the host plant cell plastid. Such constructs and methods are known in the art and are generally described, for example, in Svab, et al. (1990) Proc.<br>
Natl.Acad. Sci. USA 87: 8526-8530 and Svab and Maliga (1993) Proc.Natl. <br>
Acad. Sci.<br><br>
USA 90: 913-917 and in U. S. Patent Number 5,693,507.<br>
A plant cell, tissue, organ, or plant into which the recombinant DNA constructs containing the expression constructs have been introduced is considered transformed, transfected, or transgenic. A transgenic or transformed cell or plant also includes progeny of the cell or plant and progeny produced from a breeding program employing such a transgenic plant as a parent in a cross and exhibiting an altered phenotype resulting from the presence of a LCAT nucleic acid sequence.<br>
Plant expression or transcription constructs having a plant LCAT as the DNA sequence of interest for increased or decreased expression thereof may be employed with a wide variety of plant life, particularly, plant life involved in the production of vegetable oils for edible and industrial uses. Most especially preferred are temperate oilseed crops.<br>
Plants of interest include, but are not limited to, rapeseed (Canola and High Erucic Acid varieties), sunflower, safflower, cotton, soybean, peanut, coconut and oil palms, and com.<br>
Depending on the method for introducing the recombinant constructs into the host cell, other DNA sequences may be required. Importantly, this invention is applicable to dicotyledyons and monocotyledons species alike and will be readily applicable to new and/or improved transformation and regulation techniques.<br>
Of particular interest, is the use of plant LCAT and ACAT constructs in plants to produce plants or plant parts, including, but not limited to leaves, stems, roots, reproductive, and seed, with a modified content of lipid and/or sterol esters and to alter the oil production by such plants.<br>
Of particular interest in the present invention, is the use of ACAT genes in conjunction with the LCAT sequences to increase the sterol content of seeds. Thus, overexpression of a nucleic acid sequence encoding an ACAT and LCAT in an oilseed crop may find use in the present invention to increase sterol levels in plant tissues and/or increase oil production.<br>
It is contemplated that the gene sequences may be synthesized, either completely or in part, especially where it is desirable to provide plant-preferred sequences. Thus, all or a portion of the desired structural gene (that portion of the gene which encodes the LCAT or ACAT protein) may be synthesized using codons preferred by a selected host. Hostpreferred codons may be determined, for example, from the codons used most frequently in the<br>
19<br><br>
proteins expressed in a desired host species.<br>
One skilled in the art will readily recognize that antibody preparations, nucleic acid probes (DNA and RNA) and the like may be prepared and used to screen and recover “homologous”or”related”sequences from a variety of plant sources. Homologous sequences are found when there is an identity of sequence, which may be determined upon comparison of sequence information, nucleic acid or amino acid, or through hybridization reactions between a known LCAT and a candidate source. Conservative changes, such as GIu/Asp,Val/Ile, Ser/Thr, Arg/Lys andGln/Asn may also be considered in determining sequence homology. Amino acid sequences are considered homologous by as little as 25% sequence identity between the two complete mature proteins. (See generally, Doolittle, R. F., OF URFS and ORFS (University Science Books, CA, 1986.) Thus, other LCATs may be obtained from the specific sequences provided herein.<br>
Furthermore, it will be apparent that one can obtain natural and synthetic sequences, including modified amino acid sequences and starting materials for synthetic-protein modeling from the exemplified LCAT and ACAT sequences and from sequences which are obtained through the use of such exemplified sequences. Modified amino acid sequences include sequences which have been mutated, truncated, increased and the like, whether such sequences were partially or wholly synthesized. Sequences which are actually purified from plant preparations or are identical or encode identical proteins thereto, regardless of the method used to obtain the protein or sequence, are equally considered naturally derived.<br>
For immunological screening, antibodies to the protein can be prepared by injecting rabbits or mice with the purified protein or portion thereof, such methods of preparing antibodies being well known to those in the art. Either monoclonal or polyclonal antibodies can be produced, although typically polyclonal antibodies are more useful for gene isolation. Western analysis may be conducted to determine that a related protein is present in a crude extract of the desired plant species, as determined by crossreaction with the antibodies to the encoded proteins. When cross-reactivity is observed, genes encoding the related proteins are isolated by screening expression libraries representing the desired plant species. Expression libraries can be constructed in a variety of commercially available vectors, including lambda gtll, as described in Sambrook, et al.<br>
(MolecularCloning : A Laboratory Manual, Second Edition (1989) Cold Spring Harbor Laboratory, Cold Spring Harbor, New York).<br>
To confirm the activity and specificity of the proteins encoded by the identified nucleic acid sequences as acyltransferase enzymes, in vitro assays are performed in insect cell cultures using baculovirus expression systems. Such baeulovirus expression systems are known in the art and are described by Lee, etal. U. S. Patent Number 5,348,886, the entirety of which is herein incorporated by reference.<br>
In addition, other expression constructs may be prepared to assay for protein activity utilizing different expression systems. Such expression constructs are transformed into yeast or prokaryotic host and assayed for acyltransferase activity. Such expression systems are known in the art and are readily available through commercial sources.<br>
The method of transformation in obtaining such transgenic plants is not critical to the instant invention, and various methods of plant transformation are currently available.<br>
20<br><br>
Furthermore, as newer methods become available to transform crops, they may also be directly applied hereunder. For example, many plant species naturally susceptible to Agrobacterium infection may be successfully transformed via tripartite or binary vector methods of Agrobacterium mediated transformation. In many instances, it will be desirable to have the construct bordered on one or both sides by T-DNA, particularly having the left and right borders, more particularly the right border. This is particularly useful when the construct uses A. tumefaciens or A. rhizogenes as a mode for transformation, although the T-DNA borders may find use with other modes of transformation. In addition, techniques of microinjection, DNA particle bombardment, and electroporation have been developed which allow for the transformation of various monocot and dicot plant species.<br>
Normally, included with the DNA construct will be a structural gene having the necessary regulatory regions for expression in a host and providing for selection of transformant cells. The gene may provide for resistance to a cytotoxic agent, e. g. antibiotic, heavy metal, toxin, etc., complementation providing prototrophy to an auxotrophic host, viral immunity or the like. Depending upon the number of different host species the expression construct or components thereof are introduced, one or more markers may be employed, where different conditions for selection are used for the different hosts.<br>
Non-limiting examples of suitable selection markers include genes that confer resistance to bleomycin, gentamycin, glyphosate, hygromycin, kanamycin, methotrexate, phleomycin, phosphinotricin, spectinomycin, streptomycin, sulfonamide and sulfonylureas. Maliga et al., Methods in Plant Molecular Biology, Cold Spring Harbor<br>
Laboratory Press, 1995, p. 39. Examples of markers include, but are not limited to, alkaline phosphatase (AP), myc,hemagglutinin (HA), glucuronidase (GUS), luciferase, and green fluorescent protein (GFP).<br>
Where Agrobacterium is used for plant cell transformation, a vector may be used which may be introduced into the Agrobacterium host for homologous recombination with T-DNA or the Ti-or Ri-plasmid present in the Agrobacterium host. The Ti-or Ri-plasmid containing the T-DNA for recombination may be armed (capable of causing gall formation) or disarmed (incapable of causing gall formation), the latter being permissible, so long as the vir genes are present in the transformed Agrobacterium host. The armed plasmid can give a mixture of normal plant cells and gall.<br>
In some instances where Agrobacterium is used as the vehicle for transforming host plant cells, the expression or transcription construct bordered by the T-DNA border region (s) will be inserted into a broad host range vector capable of replication in E. coli and Agrobacterium, there being broad host range vectors described in the literature.<br>
Commonly used is pRK2 or derivatives thereof. See, for example, Ditta, etal., (Proc. Nat.<br>
Acad. Sci., U. S. A. (1980) 77:7347-7351) and EPA 0 120 515, which are incorporated herein by reference. Alternatively, one may insert the sequences to be expressed in plant cells into a vector containing separate replication sequences, one of which stabilizes the vector in E. coli, and the other in Agrobacterium. See, for example, McBride and Summerfelt (PlantMol. Biol. (1990)14: 269-276), wherein the pRiHRI (Jouanin, etal., Mol. Gen. Genet. (1985) 201: 370-374) origin of replication is utilized and provides for added stability of the plant expression vectors in host Agrobacterium cells.<br>
21<br><br>
Included with the expression construct and the T-DNA can be one or more markers, which allow for selection of transformed Agrobacterium and transformed plant cells. A number of markers have been developed for use with plant cells, such as resistance tochloramphenicol, kanamycin, the aminoglycoside G418, hygromycin, or the like. The particular marker employed is not essential to this invention, one or another marker being preferred depending on the particular host and the manner of construction.<br>
For transformation of plant cells using Agrobacterium, explants may be combined and incubated with the transformed Agrobacterium for sufficient time for transformation, the bacteria killed, and the plant cells cultured in an appropriate selective medium. Once callus forms, shoot formation can be encouraged by employing the appropriate plant hormones in accordance with known methods and the shoots transferred to rooting medium for regeneration of plants. The plants may then be grown to seed and the seed used to establish repetitive generations and for isolation of vegetable oils.<br>
Thus, in another aspect of the present invention, methods for modifying the steroland/or stanol composition of a host cell. Of particular interest are methods for modifying the sterol and/or stanol composition of a host plant cell. In general the methods involve either increasing the levels of sterol ester compounds as a proportion of the total sterol compounds. The method generally comprises the use of expression constructs to direct the expression of the polynucleotides of the present invention in a host cell.<br>
Also provided are methods for reducing the proportion of sterol ester compounds as a percentage of total sterol compounds in a host plant cell. The method generally comprises the use of expression constructs to direct the suppression of endogenous acyltransferase proteins in a host cell.<br>
Of particular interest is the use of expression constructs to modify the levels of sterol compounds in a host plant cell. Most particular, the methods find use in modifying the levels of sterol compounds in seed oils obtained from plant seeds.<br>
Also of interest is the use of expression constructs of the present invention to alter oil production in a host cell and in particular to increase oil production. Of particular interest is the use of expression constructs containing nucleic acid sequences encoding LCAT and/or ACAT polypeptides to transform host plant cells and to use these host cells to regenerate whole plants having increase oil production as compared to the same plant not containing the expression construct.<br>
The oils obtained from transgenic plants having modified sterol compound content find use in a wide variety of applications. Of particular interest in the present invention is the use of the oils containing modified levels of sterol compounds in applications involved in improving human nutrition and cardiovascular health. For example, phytostanols are beneficial for lowering serum cholesterol (Ling, et al. (1995) Life Sciences 57: 195-206).<br>
Cholesterol-lowering compositions comprise the oils and sterol ester compound compositions obtained using the methods of the present invention. Such cholesterol lowering compositions include, but are not limited to foods, food products, processed foods, food ingredients, food additive compositions, or dietary/nutritional supplements that contain oils and/or fats. Non-limiting examples include margarines; butters; shortenings; cooking oils; frying oils;<br>
22<br><br>
dressings, such as salad dressings; spreads; mayonnaises; and vitamin/mineral supplements. Patent documents relating to such compositions include, U. S. Patents 4,588,717 and 5,244,887, and PCT International Publication Nos. WO 96/38047, WO 97/42830, WO 98/06405, and WO 98/06714.<br>
Additional non-limiting examples include toppings; dairy products such as cheese and processed cheese; processed meat; pastas; sauces; cereals; desserts, including frozen andshelf-stable desserts; dips; chips; baked goods; pastries; cookies; snack bars; confections; chocolates; beverages; unextracted seed; and unextracted seed that has been ground, cracked, milled, rolled, extruded, pelleted, defatted, dehydrated, or otherwise processed, but which still contains the oils, etc., disclosed herein.<br>
The cholesterol-lowering compositions can also take the form of pharmaceutical compositions comprising a cholesterol-lowering effective amount of the oils or sterol compound compositions obtained using the methods of the present invention, along with a pharmaceutically acceptable carrier, excipient, or diluent. These pharmaceutical compositions can be in the form of a liquid or a solid. Liquids can be solutions or suspensions; solids can be in the form of a powder, a granule, a pill, a tablet, a gel, or an extrudate. U. S. Patent 5,270,041 relates to sterol-containing pharmaceutical compositions.<br>
Thus, by expression of the nucleic acid sequences encoding acyltransferase-like sequences of the present invention in a host cell, it is possible to modify the lipid content and/or composition as well as the sterol content and/or composition of the host cell.<br>
The invention now being generally described, it will be more readily understood by reference to the following examples which are included for purposes of illustration only and are not intended to limit the present invention.<br>
EXAMPLES<br>
Example I: RNA Isolations<br>
Total RNA from the inflorescence and developing seeds of Arabidopsis thaliana was isolated for use in constru tion of complementary(cDNA) libraries. The procedure was an adaptation of the DNA isolation protocol of Webb and Knapp (D. M. Webb and S. J.<br>
Knapp, (1990) Plant Molec. Reporter, 8,180-185). The following description assumes the use of 1 g fresh weight of tissue. Frozen seed tissue was powdered by grinding under liquid nitrogen. The powder was added to1Oml REC buffer (50mMTris-HCl, pH 9,0.8MNaCl, 10mM EDTA, 0.5% w/v CTAB (cetyltrimethyl-ammonium bromide)) along with 0.2g insoluble polyvinylpolypyrrolidone, and ground at room temperature. The homogenate was centrifuged for 5 minutes at 12,000xg to pellet insoluble material. The resulting supernatant fraction was extracted with chloroform, and the top phase was recovered.<br>
The RNA was then precipitated by addition of 1 volume RecP (50mM Tris-HCL pH9,IOmM EDTA and 0.5% (w/v) CTAB) and collected by brief centrifugation as before.<br>
The RNA pellet was redissolved in 0.4 ml oflM NaCI. The RNA pellet was redissolved in water and extracted withphenol/chloroform. Sufficient 3M potassium acetate(pH 5) ws added to make the mixture 0.3M in acetate, followed by addition of two volumes of ethanol to precipitate the RNA. After washing with ethanol, this final RNA precipitate was dissolved in water and stored frozen.<br>
23<br><br>
Alternatively, total RNA may be obtained using TKLzo) reagent (BRL Lifetechnologies, Gaithersburg, MD) following the manufacturer's protocol. The RNA precipitate was dissolved in water and stored frozen.<br>
Example 2: Identification of LCAT Sequences<br>
Searches were performed on a Silicon Graphics Unix computer using additional Bioaccellerator hardware and GenWeb software supplied by Compugen Ltd. This software and hardware enabled the use of the Smith-Waterman algorithm in searching DNA and protein databases using profiles as queries. The program used to query protein databases was profilesearch. This is a search where the query is not a single sequence but a profile based on a multiple alignment of amino acid or nucleic acid sequences. The profile was used to query a sequence data set, i. e., a sequence database. The profile contained all the pertinent information for scoring each position in a sequence, in effect replacing the”scoring matrix”sused for the standard query searches. The program used to query nucleotide databases with a protein profile was tprofilesearch. Tprofilesearch searches nucleic acid databases using an amino acid profile query. As the search is running, sequences in the database are translated to amino acid sequences in six reading frames. The output file for tprofilesearch is identical to the output file for profiiesearch except for an additional column that indicates the frame in which the best alignment occurred.<br>
The Smith-Waterman algorithm, (Smith and Waterman(1981) J. Molec. Biol.<br>
147: 195-197), was used to search for similarities between one sequence from the query and a group of sequences contained in the database.<br>
A protein sequence of Lecithin: cholesterol acyltransferase from human (McLean J, et al. (1986) NucleicAcids Res. 14 (23): 9397-406 SEQ IDNO: 1)) was used to search the NCBI non-redundant protein database using BLAST. Three sequences were identified from Arabidopsis, GenBank accessions AC004557 (referred to herein as LCAT1, SEQ IDNO: 2), AC003027 (referred to herein as LCAT2, SEQ IDNO: 4), and AL024486 (referred to herein<br>
as LCAT3, SEQ IDNO: 6). The deduced amino acid sequences are provided in<br>
SEQ ID NOs: 3,5, and 7, respectively.<br>
The profile generated from the queries using PSI-BLAST was excised from the hyper text markup language (html) file. The worldwide web (www)/html interface to psiblast at ncbi stores the current generated profile matrix in a hidden field in the htm! file that is returned after each iteration of psiblast. However, this matrix has been encoded into string62 (s62) format for ease of transport through html. String62 format is a simple conversion of the values of (he matrix into html legal ascii characters.<br>
The encoded matrix width (x axis) is 26 characters, and comprise the consensus characters, the probabilities of each amino acid in the order A, B, C, D, E, F, G, H, I, K, L, M, N, P, Q, R, S, T, V, W, X, Y, Z (where B represents D and N, and Z represents Q and E, and X represents any amino acid), gap creation value, and gap extension value.<br>
The length (y axis) of the matrix corresponds to the length of the sequences identified by PSI-BLAST. The order of the amino acids corresponds to the conserved amino acid sequence of the sequences identified using PSI-BLAST, with the N-terminal end at the top of the matrix. The probabilities of other amino acids at that position are represented for each amino acid<br>
24<br><br>
along the x axis, below the respective single letter amino acid abbreviation.<br>
Thus, each row of the profile consists of the highest scoring (consensus) amino acid, followed by the scores for each possible amino acid at that position in sequence matrix, the score for opening a gap that that position, and the score forcontinuing a gap at that position.<br>
The string62 file is converted back into a profile for use in subsequent searches.<br>
The gap open field is set to 11 and the gap extension field is set to 1 along the x axis. The gap creation and gap extension values are known, based on the settings given to the PSI BLAST algorithm. The matrix is exported to the standard GCG profile form. This format can be read by GenWeb.<br>
The algorithm used to convert the string62 formatted file to the matrix is outlined in Table 1.<br>
Table 1 1. if encoded character z then the value is blast score min 2. if encoded character Z then the value is blast score max 3. else if the encoded character is uppercase then its value is(64- (ascii # of char)) 4. else if the encoded character is a digit the value is( (ascii # of char)-48) 5. else if the encoded character is not uppercase then the value is( (ascii # of char)-87) 6. ALL B positions are set to minof D and N amino acids at that row in sequence matrix 7. ALL Z positions are set to minof Q amd E amino acids at that row in sequence matrix 8. ALL X positions are set to min of all amino acids at that row in sequence matrix 9.kBLAST~SCORE~MAX=999; 10. kBLASTSCOREMIN=-999; 11. all gap opens are set to 11 12. all gap lens are set to provided in SEQ ID NO: 75 and the protein sequence is provided in SEQ ID NO: 76.<br>
Seven EST sequences were identified from soybean libraries as being LCAT sequences.<br>
Two sequences from soy (SEQ ID NOs: 12 and 13) are most closely related to the Arabidopsis LCAT1 sequence, a single sequence was identified as being most closely related to LCAT2 (SEQ ID NO: 14), three were closely related to LCAT3 (SEQ ID NOs: 15-17), and an additional single sequence was identified (SEQ IDNO: 18). A total of 11 com EST sequences were identified as being related to the Arabidopsis LCAT sequences.<br>
Two com EST sequences (SEQ ID NOs: 19 and 20) were most closely related to LCAT1, two sequences were identified as closely related to LCAT2 (SEQ ID NOs: 21 and 22), four corn EST sequences were identified as closely related to LCAT3 (SEQ ID NOs: 23-26), and an additional three com EST sequences were also identified (SEQ ID NOs: 27-29).<br>
Example 3; Identification of ACAT Sequences<br>
Since plant ACATs are unknown in the art, searches were performed to identify known and related ACAT sequences from mammalian sources from public databases.<br>
These sequences were then used to search public and proprietary EST databases to identify plant ACAT-like sequences.<br>
A public database containing mouse Expressed Sequence Tag (EST) sequences (dBEST) was searched for ACAT-like sequences. The search identified two sequences (SEQ ID 30 and31)<br>
which were related (approximately 20% identical), but divergent, to known ACAT sequences.<br>
25<br><br><br><br>
The entire coding region of the Arabidopsis ACAT sequence (SEQ ID NO: 42) was amplified from the EST clone LIB25-088-C7 using oligonucleotide primers 5'-TCGACCTGCAGGAAGCTTAGAAATGGCGATTTTGGATTC-3' (SEQ ID NO: 60) and 5'-GGATCCGCGGCCGCTCATGACATCGATCCTTTTCGG-3' (SEQ ID NO:61) in a polymerase chain reaction (PCR).<br>
Each resulting PCR product wassubcloned into pCR2.1 Topo (Invitrogen) and labeled pCGN9964(LCATl), pCGN9985 (LCAT2), pCGN9965 (LCAT3), pCGN9995 (LCAT4), pCGN]0964(LCAT5), pCGN10963(LR01), and pCGN8626 (ACAT).<br>
Double stranded DNA sequence was obtained to verify that no errors were introduced by the PCR amplification.<br>
4A. Baculovirus Expression Constructs<br>
Constructs are prepared to direct the expression of the Arabidopsis LCAT and yeast LCAT sequences in cultured insect cells. The entire coding region of the LCAT proteins was removed from the respective constructs by digestion with NotI and Sse8387I, followed by gel electrophoresis and gel purification. The fragments containing the LCAT coding sequences were cloned intoNotl and PstI digested baculovirus expression vectorpFastBacI (Gibco-BRL, Gaithersburg, MD). The resulting baculovirus expression constructs were referred to as PCGN9992(LCAT1), pCGN9993 (LCAT2), pCGN9994 (LCAT3), pCGN10900 (LCAT4), pCGN10967(LCAT5), and pCGN10962 (LROI).<br>
4B. Plant Expression Construct Preparation<br>
A plasmid containing the napin cassette derived from pCGN3223 (described in<br>
U. S. Patent No. 5,639,790, the entirety of which is incorporated herein by reference) was modified to make it more useful for cloning large DNA fragments containing multiple restriction sites, and to allow the cloning of multiple napin fusion genes into plant binary transformation vectors. An adapter comprised of the self annealed oiigonucleotide of sequence 5'<br>
CGCGATTTAAATGGCGCGCCCTGCAGGCGGCCGCCTGCAGGGCGCGCCATTTA<br>
AAT-3' (SEQ IDNO: 62) was ligated into the cloning vector pBC SK+(Stratagene) after digestion with the restriction endonuclease BssHII to construct vector pCGN7765.<br>
Plamids pCGN3223 and pCGN7765 were digested with NotI and ligated together. The resultant vector, pCGN7770, contained the pCGN7765 backbone with the napin seed specific expression cassette from pCGN3223.<br>
The cloning cassette, pCGN7787, contained essentially the same regulatory elements as pCGN7770, with the exception of the napin regulatory regions of pCGN7770 have been replaced with the double CAMV 35S promoter and the tml polyadenylation and transcriptional termination region.<br>
A binary vector for plant transformation, pCGN5139, was constructed from pCGN1558 (McBride and Summerfelt, (1990) Plant Molecular Biology, 14: 269-276). InpCGN5139, the polylinker of pCGN1558 was replaced as aHindIII/Asp718 fragment with a polylinker containing unique restriction endonuclease sites, AscI, PacI,XbaI, SwaI,BamHI, and NotI. TheAsp718 and HindIII restriction endonuclease sites are retained in pCGN5139.<br>
28<br><br>
A series of turbo binary vectors was constructed to allow for the rapid cloning of DNA sequences into binary vectors containing transcriptional initiation regions (promoters) and transcriptional termination regions.<br>
The plasmid pCGN8618 was constructed by ligating oligonucleotides 5'-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3'(SEQ ID NO; 63) and 5'-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCC-3' (SEQ ID NO: 64) intoSaII/XhoI-digested pCGN7770. A fragment containing the napin promoter, polylinker and napin 3'region was excised from pCGN8618 by digestion withAsp7181; the fragment was blunt-ended by filling in the 5'overhangs with Klenow fragment then Iigated into pCGN5139 that had been digested withAsp7181 and HindIII and blunt-ended by filling in the 5'overhangs with Klenow fragment. A plasmid containing the insert oriented so that the napin promoter was closest to the bluntedAsp718I site of pCGN5139 and the napin 3' was closest to the blunted HindIII site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8622.<br>
The plasmid pCGN8619 was constructed by ligating oligonucleotides 5'-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCC-31 (SEQ ID NO: 65) and 5'-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGG-3' (SEQ ID NO: 66) intoSafI/XhoI-digested pCGN7770. A fragment containing the napin promoter, polylinker and napin 3'region was removed from pCGN86f9 by digestion withAsp7(8I; the fragment was blunt-ended by filling in the 5'overhangs with Klenow fragment then ligated into pCGN5139 that had been digested withAsp7t8I and HindIII and blunt-ended by filling in the 5'overhangs with Klenow fragment. A plasmid containing the insert oriented so that the napin promoter was closest to the bluntedAsp7I8I site of pCGN5139 and the napin 3' was closest to the blunted HindIII site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8623.<br>
The plasmid pCGN8620 was constructed by ligating oligonucleotides5'-TCGAGGATCCGCGGCCGCAAGCTTCCTGCAGGAGCT-31 (SEQ ID NO: 67) and 5'-CCTGCAGGAAGCTTGCGGCCGCGGATCC-3' (SEQ IDNO: 68) intoSaII/Sad- digested pCGN7787. A fragment containing the d35S promoter, polylinker and tml 3' region was removed from pCGN8620 by complete digestion withAsp718I and partial digestion with NotI. The fragment was blunt-ended by filling in the 5'overhangs with Klenow fragment then ligated intopCGN5139 that had been digested withAsp7181 and HindIII and blunt-ended by filling in the 5'overhangs with Klenow fragment. A plasmid containing the insert oriented so that the d35S promoter was closest to the bluntedAsp7181 site of pCGN5139 and the tml 3'was closest to the blunted HindIII site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8624.<br>
The plasmid pCGN862I was constructed by ligating oligonucleotides 5'-TCGACCTGCAGGAAGCTTGCGGCCGCGGATCCAGCT-3' (SEQ ID NO: 69) and 5'-GGATCCGCGGCCGCAAGCTTCCTGCAGG-3' (SEQ IDNO: 70) intoSaII/Sad- digestedpCGN7787. A fragment containing the d35S promoter, polylinker and tml 3' region was removed from pCGN862 J by complete digestion withAsp7181 and partial digestion with NotI. The fragment was blunt-ended by filling in the 5'overhangs with Klenow fragment then ligated iniopCGN5139 that had been digested withAsp718I and HindIII and blunt-ended by filling in the 5'overhangs with Klenow fragment. A plasmid containing the insert oriented so that the d35S promoter was closest to the bluntedAsp7181<br>
29<br><br>
site ofpCGN5139 and the tml 3'was closest to the blunted HindIII site was subjected to sequence analysis to confirm both the insert orientation and the integrity of cloning junctions. The resulting plasmid was designated pCGN8625.<br>
The plasmid construct pCGN8640 is a modification of pCGN8624 described above. A 938bp PstI fragment isolated from transposon Tn7 which encodes bacterial spectinomycin and streptomycin resistance (Fling et al. (1985), Nucleic Acids Research 13 (19): 7095-7106), a determinant for E. coli and Agrobacterium selection, was blunt ended with Pfu polymerase. The blunt ended fragment was ligated into pCGN8624 that had been digested with Spel and blunt ended with Pfu polymerase. The region containing the PstI fragment wassequenced to confirm both the insert orientation and the integrity of cloning junctions.<br>
The spectinomycin resistance marker was introduced into pCGN8622 and pCGN8623 as follows. A 7.7 Kbp AvrII-SnaBI fragment from pCGN8640 was ligated to a 10.9 KbpAvrfl-SnaBI fragment from pCGN8623 or pCGN8622, described above. The resulting plasmids were pCGN8641 and pCGN8643, respectively.<br>
The plasmid pCGN8644 was constructed by ligating oligonucleotides 5'-GATCACCTGCAGGAAGCTTGCGGCCGCGGATCCAATGCA-3' (SEQ ID NO: 71) and S'-TTGGATCCGCGGCCGCAAGCTTCCTGCAGGT-S1 (SEQ ID NO: 72) into BamHI-PstI digested pCGN8640.<br>
4C. Plant LCAT Expression Construct Preparation<br>
The coding sequence of LCAT 1 was cloned from pCGN9964 as aNotII Sse8387I fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9960, pCGN996l, pCGN9962, and pCGN9963, respectively.<br>
The construct pCGN9960 was designed to express the LCA1 coding sequence in the sense orientation from the constitutive promoter CaMV35S. The construct pCGN9961 was designed to express the LCAT1 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9962 was designed to express the LCAT1 coding sequence in the sense orientation from the napin promoter. The construct pCGN9963 was designed to express the LCAT1 coding sequence in the antisense orientation from the constitutive promoter CaMV 35S.<br>
The coding sequence of LCAT2 was cloned from pCGN9985 as a NotlI Sse8387I fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9981, pCGN9982, pCGN9983, and pCGN9984, respectively.<br>
The construct pCGN9981 was designed to express the LCAT2 coding sequence in the sense orientation from the constitutive promoter CaMV 35S. The construct pCGN9982 was designed to express the LCAT2 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9983 was designed to express the LCAT2 coding sequence in the sense orientation from the napin promoter. The construct pCGN9984 was designed to express the LCAT2 coding sequence in the antisense orientation from the constitutive promoter CaMV35S.<br>
The coding sequence of LCAT3 was cloned from pCGN9965 as aNotII Sse8387I fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9966, pCGN9967, pCGN9968, and pCGN9969, respectively.<br>
30<br><br>
The construct pCGN9966 was designed to express the LCAT3 coding sequence in the sense orientation from the constitutive promoterCaMV 35S. The construct pCGN9967 was designed to express the LCAT3 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9968 was designed to express the LCAT3 coding sequence in the sense orientation from the napin promoter. The construct pCGN9969 was designed lo express the LCAT3 coding sequence in the antisense orientation from the constitutive promoter CaMV35S.<br>
The coding sequence of LCAT4 was cloned from pCGN9995 as aNotll Sse83871 fragment into pCGN8640, pCGN8641, pCGN8643, and pCGN8644 to create the expression constructs pCGN9996, pCGN9997, pCGN9998, and pCGN9999, respectively.<br>
The construct pCGN9996 was designed to express the LCAT4 coding sequence in the sense orientation from the constitutive promoter CaMV35S. The construct pCGN9997 was designed to express the LCAT4 coding sequence in the antisense orientation from the napin promoter. The construct pCGN9998 was designed to express the LCAT4 coding sequence in the sense orientation from the napin promoter. The construct pCGN9999 was designed to express the LCAT4 coding sequence in the antisense orientation from the constitutive promoterCaMV 35S.<br>
The coding sequenceof LCAT5 was cloned from pCGN10964 asaNotI/Sse8387I fragment into pCGN9977 and pCGN9979, to create the expression constructs pCGN10965, and pCGN10966, respectively. The construct pCGN10965 was designed to express the LCAT5 coding sequence in the sense orientation from the constitutive promoter CaMV35S. The construct pCGN 10966 was designed to express the LCAT5 coding sequence in the sense orientation from the napin promoter.<br>
The coding sequence of LRO1 was cloned from pCGN10963 as aNotll Sse8387I fragment into pCGN9977 and pCGN9979, to create the expression constructs pCGN10960, and pCGN 10961, respectively. The construct pCGN10960 was designed to express theLoi coding sequence in the sense orientation from the constitutive promoter CaMV35S. The construct pCGN10961 was designed to express theLRO1 coding sequence in the sense orientation from the napin promoter.<br>
4D. Plant ACAT Expression Construct Preparation<br>
A fragment containing the Arabidopsis ACAT-like coding region was removed from pCGN8626 by digestion withSse8387l and Notl. The fragment containing the ACAT-like sequence was ligated into Pstl-Not I digested pCGN8622. The resulting plasmid was designated pCGN8627. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis ACAT-like coding region (SEQ ID NO: 42) was removed from pCGN8626 by digestion withSse83871 and NotI. The fragment was ligated into Pstl-Not I digested pCGN8623. The resulting plasmid was designated pCGN8628. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis ACAT-like coding region was removed from pCGN8626 by digestion with Sse8387 and Not I. The fragment was ligated into Pstl-Not I digested pCGN8624. The resulting plasmid was designated pCGN8629. DNA<br>
31<br><br>
sequence analysis confirmed the integrity of the cloning junctions.<br>
A fragment containing the Arabidopsis ACAT-like coding region was removed from pCGN8626 by digestion with Sse8387 and NotI. The fragment was ligated into PstI-Not I digested pCGN8625. The resulting plasmid was designated pCGN8630. DNA sequence analysis confirmed the integrity of the cloning junctions.<br>
An additional expression construct for the suppression of endogenous ACAT-like activity was also prepared. The construct pCGN8660 was constructed by cloning approximately 1 Kb of the Arabidopsis ACAT-like coding region from pCGN8626 in the sense orientation, and the full-length Arabidopsis ACAT-like coding region in the antisense orientation under the regulatory control of the napin transcription initiation sequence.<br>
For expression of the rat ACAT-like sequence in plants, the NotI-Sse8387I fragment of pCGN8592 was cloned into NotI-PstI digested binary vectors pCGN8621, pCGN8622, and PCGN8624 to yield plasmids, pCGN 9700, pCGN9701, and pCGN9702, respectively. Plasmid pCGN9700 expresses a sense transcript of the rat ACAT-likecDNA under control of a napin promoter, plasmid pCGN970l expresses an antisense transcript of the rat ACAT-like cDNA under control of a napin promoter, and plasmid pCGN9702 expresses a sense transcript of the rat ACAT-like cDNA under control of a double 35S promoter. Plasmids pCGN 9700, pCGN9701, and pCGN9702 were introduced in Agrobacterium tumefaciens EHA101.<br>
Constructs were prepared to direct the expression of the rat ACAT-like sequence in the seed embryo of soybean and theendosperm of corn. For expression of the rat ACATlike DNA sequence in soybean, a 1.5 kbNotHSse8387I fragment from pCGN8592 containing the coding sequence of the rat ACAT-like sequence was blunt ended using Mung bean nuclease, and ligated into the Smal site of the turbo 7S binary/cloning vector pCGN8809 to create the vector pCGN8817 for transformation into soybean by particle bombardment. The vector pCGN8817 contained the operably linked components of the promoter region of the soybean a'subunitof-conglycinin (7S promoter, (Chen etal., (1986), Proc. Natl.Acad. Sci., 83: 8560-8564), the DNA sequence coding for the entire rat ACAT-like protein, and the transcriptional termination region of pea RuBisCo small subunit, referred to as E9 31 (Coruzzi, et al.(1984) EMBO J. 3: 1671-1679 and Morelli, et al. (1985) Nature 315: 200-204). This construct further contained sequences for the selection of positive transformed plants by screening for resistance to glyphosate using the CP4 EPSPS (U. S. Patent 5,633,435) expressed under the control of the figwort mosaic virus (FMV) promoter (U. S. Patent Number 5,378,619) and the transcriptional termination regionof E9.<br>
For expression of the rat ACAT-like sequence in the cornendosperm, a 1.5 kb NotIISse8387I fragment from pCGN8592 containing the coding sequence of the rat ACAT-like sequence was blunt ended using Mung bean nuclease, and ligated into theBamHl site of the rice pGtl expression cassette pCGN8592 for expression from the pGtl promoter (Leisy, D. J. et al., Plant Mol. Biol. 14 (1989) 41-50) and the HSP70 intron sequence (U. S. Patent Number 5,593,874). This cassette also included the transcriptional termination region downstream of the cloning site of nopaline synthase, nos 3' (Depicker et al., J.Molec. Appl. Genet. (1982) 1: 562-573). A 7.5 kb fragment containing the pGtl promoter, the DNA sequence encoding the rat ACAT-like protein, and the nos transcriptional termination sequence was cloned into the binary vector pCGN8816 to create the vector pCGN8818 for<br>
32<br><br>
transformation into corn. This construct also contained sequences for the selection of positive transformants with kanamycin using the kanamycin resistance gene from Tn5 bacteria under the control of the CAMV 35S promoter and tml transcriptional termination regions.<br>
Example 5: Expression in Insect Cell Culture<br>
A baculovirus expression system was used to express the LCAT cDNAs in cultured insect cells.<br>
The baculovirus expression constructs pCGN9992, pCGN9993, pCGN9994, pCGN 10900, pCGN10962, and pCGN10967 were transformed and expressed using the BAC-to-BAC Baculovirus Expression System (Gibco-BRL, Gaithersburg, MD) according to the manufacturer's directions.<br>
The transformed insect cells were used to assay for acyltransferase activities using methods known in the art (see Example 8).<br>
Example 6: Plant Transformation<br>
A variety of methods have been developed to insert a DNA sequence of interest into the genome of a plant host to obtain the transcription or transcription and translation of the sequence to effect phenotypic changes. Transgenic plants were obtained by<br>
Agrobacteriummediated transformation as described by Radke et aI.(Theor. Appl. Genet.<br>
(1988)75:685-694;<br>
Plant Cell Reports (1992) 11: 499-505). Alternatively,microprojectile bombardment methods, such as described by Klein et al.(BiolTechnology 10: 286-291) may also be used to obtain nuclear transformed plants. Other plant species may be similarly transformed using related techniques.<br>
The plant binary constructs described above were used in plant transformation to direct the expression of the sterol acyltransferases in plant tissues. Binary vector constructs were transforme<br>
High-resolution spectra were measured at 11.7T(1H=500MHz, I3C=125mHz) using Varian NMR Instruments (Palo Alto, CA)Inova NMR spectrometers equipped with carbon-observe MASNanoprobesTM. The 13C spectra were acquired without a fieldfrequency lock at ambient temperature (approx.21-22 C) for 14 hours using the following conditions: spectral width = 29.996 kHz, acquisition time = 2.185 seconds, p/2 pulse (3.8 ms) with no relaxation delay, 1H g B2 = 2.5 kHz with Waltz decoupling. Data processing conditions were typically: digital resolution = 0.11 Hz, 0.3 to 1.5 Hz line broadening and time-reversed linear prediction of the first three data points. Chemical shifts were referenced by adding neat tetram ethyl si lane (TMS) to Arabidopsis seeds and using the resulting referencing parameters for subsequent spectra. The 13C resolution was 2-3 Hz for the most narrow seed resonances. Spectral resolution was independent of MAS spinning speeds (1.5-3.5 kHz) and data were typically obtained with 1.5 kHz spinning speeds. Spinning sidebands were approx. 1% of the main resonance. Phytosterol 13C assignments were based on model samples composedof triolein, ss-sitosterol and cholesterol oleate. Triacylglycerol 13C assignments were made from comparison with literature assignments or with shifts computed from aI3C NMR database (Advanced<br>
Chemical Development, Inc., version 3.50, Toronto Canada).<br>
The results of these analyses are displayed in Figure 2 and show that there was a trend of an<br>
33<br><br>
approximately 2 fold increase of phytosterols in the seeds derived from plant line 5 expressing the LCAT 4 gene (pCGN9998) under the control of the napin promoter.<br>
During the course of this analysis jl was also noted that die average oil content of seed from plants expressing the LCAT2 construct (pCGN9983) under the control of the napin promoter was higher than that of controls. This is the first in planta evidence supporting the concept that overexpression of a nucleotide sequence encoding a lecithin: cholesterol acyltransferase-like polypeptide can increase oil content.<br>
7B: HPLC/MS of T2 seed<br>
Seed oil from T2 plants expressing LCAT1 through 4 under the control of the napin promoter was extracted using an accelerated solvent extractor (ASE) method. Seed samples were ground, using a mortar and pestle, to achieve a fine homogeneous meal. Oil was obtained using a Dionex Accelerated Solvent Extractor (ASE). Clean ground seed was added to an equal amount of diatom aceous earth. The ground seed sample and the diatomaceous earth were thoroughly mixed until a homogeneous texture was achieved.<br>
The sample was then loaded into the instrument and oil extraction was achieved using hexane under validated laboratory protocols.<br>
Oil from these seed samples was then analyzed for sterol ester analysis using<br>
HPLC/MS for freecampesterol, stigmasterol, and sitosterol and their fatty acid esters. To the autosampler vial containing approximately 0.) grams oil was added 0.3 mLs CDCI3.<br>
One-hundredmicroSiters of this solution was added to 900 microliters CHC13. FivemicroJiters of this diluted sample was subsequently injected into an HPLC/MS with positive ion atmospheric pressure ionization. The individual components in the oils weie separated using two 4.6 x 50 mmC8 Zorbax columns in series and a gradient using acetonitrile and acetonitrile with 40%CHCI3. The sterol concentrations were calculated assuming each sterol and its fatty acids have the same molar responses. This was observed to be the case with cholesterol and its esters and was assumed to be the case forcampesterol, stigmasterol, and sitosterol. In the present study, the sterol identified as stigmasterol was actually an isomer of this compound.<br>
The results of these analyses are displayed in Figures 3 and 4 and show that there were sterol ester enhancements on the order of 50%. in the seeds derived from six out of seven T2 plant lines expressing LCAT3 (pCGN9968) under the control of the napin promoter.<br>
Example 8: Baculovirus Insect Cell Culture for Sterol Esterification Activity Baculovirus expression construct pCGN9992, pCGN9993, pCGN9994 and pCGNJ0900 (see Example 4) were transformed and expressed using the BAC-TOBAC Baculoviras Expression System (Gibco-BRL, Gaithersburg, MD) according to the manufacturer's instructions except harvesting of recombmant viruses was done 5 days post-transfection. The supernatant from the transfection mixture was used for generating virus stock which in turn was used for infecting Sf9 cells used in the assay.<br>
The transformed cells were assayed for lecithin: sterol acyltransferase activities using the method described herein. Insect cells were centrifuged and the resulting cell pellet was either used immediately or stored at-70 C for later analysis. Cells were resuspended in Medium A (100 mM Tricine/NaOH, pH 7.8,10% (w/v) glycerol, 280 mMNaCl with: O.lpM Aprottnin,<br>
34<br><br>
1M Leupeptin, and 100I1M Pefabloc (all from<br>
Boehringer Mannheim, Germany) and lysed by sonication (2 x 10 sec). Cell walls and other debris were pelleted by centrifugation (14,000 x g, 10 min,4 C). The supernatant was transferred to a new vial and membranes pelleted by centrifugation (100,000 x g, Ti 70.1 rotor, 46,000 rpm for 1 hour at4 C). Total membranes were resuspended in Medium A. Lecithin: sterol acyltransferase activity was assayed in a 0.1 ml reaction mixture containing 100 mMTris/HCI, pH 7,28 mMNaCl, 0.03% Triton X-100,0.1 mM sitosterol, 20pM 1, 2- ['4C]-palmitoyl-phosphatidy! choline (246420dpm/nmole), and 0.05-20 mg of membrane protein. After 15 minutes at 30 C, the reaction was terminated by addition of a 0.5 ml solution of methylene chloride: methanol (4: I, v/v) containing lOOig cholesterol and cholesterol ester as cold carriers. A portion (0.1 ml) of the bottom organic layer was removed and evaporated under nitrogen gas. The concentrated extract was resuspended in 30pl of hexane and spotted onto a silica gel-G thin layer chromatographic plate. The plate was migrated in hexane: diethyl ether: acetic acid (80:20: 1) to the top, then air dried. <br>
Radioactivity was determined by exposure to a Low Energy Phosphor-imaging <br><br>
Screen. Following exposure, the screen was read on a phosphorimager.<br>
The LCAT 4 protein from pCGN10900 in baculovirus membranes showed a radioactive spot in the region of the TLC plate where cholesterol ester migrates indicating that LCAT 4 has the ability to catalyze the transfer of an acyl group from lecithin (PC) to sitosterol to make a sitosterol ester.<br>
Example 9: Plant Assay for Modified Lipid Content<br>
Nir (near infrared spectroscopy spectral scanning) can be used to determine the total oil content of Arabidopsis seeds in a non-destructive way provided that a spectral calibration curve has been developed and validated for seed oil content. A seed oil spectral calibartion curve was developed using seed samples from 85 Arabidopsis plants.<br>
Seed was cleaned and scanned using a FossNIR model 6500 (Foss-Nirs Systems, Inc.).<br>
Approximately 50 to 100 milligrams of whole seeds, per sample, were packed in a mini sample ring cup with quartz lens[IH-0307] consisting a mini-insert [IH-0337] and scanned in reflectance mode to obtain the spectral data. The seed samples were then ground, using a mortar and pestle, to achieve a fine homogeneous meal. The ground samples were measured for oil using an accelerated solvent extractor (ASE).<br>
Measurement for the total oil content was performed on the Dionex Accelerated Solvent Extractor (ASE). Approximately 500 mg of clean ground seed was weighed to the nearest 0.1 mg onto a 9 x 9 cm weigh boat. An equal amount of diatomaceous earth was added using a top-loading balance accurate to the nearest 0.01 g. The ground seed sample and the diatomaceous earth were thoroughly mixed until a homogeneous texture was achieved. The sample was loaded on to the instrument and oil extraction was achieved using hexane under validated laboratory protocols. Standard Rapeseed samples were obtained from the Community Bureau of Reference (BCR). The ASE extraction method was validated using the BCR reference standards. A total percent oil recovery of 99% to 100% was achieved. “As-is”oil content was calculated to the nearest 0.01 mass percentage using the formula: Oil Content^ 100% x (vial plus extracted oil wt-initial vial wt)/ (sample wt) The analytical data generated by ASE were used to perform spectral calibrations.<br>
35<br><br>
Nir calibration equations were generated using the built-in statistical package within the NirSytems winisi softu-are. The spectral calibration portion of the software is capable of calibration and seJf-validation. From a total of 85 samples, 57 samples were used to generate the total percent oil calibration. The remaining samples were used to validate the oil calibrations. Optimized smoothing, derivative size, and mathematical treatment (modified partial least square) was utilized to generate the calibration. The samples that were not used in building respective calibrations were used as a validation set. Statistical tools such as correlation coefficient (R), coefficient of determination2), standard error of prediction (SEP), and the standard error of prediction corrected for bias (SEPC) were used to evaluate the calibration equations.<br>
T2 seeds from plants that had been transformed with the LCAT genes were cleaned and scanned using a Foss NIR model 6500 (Foss-Nirs Systems, Inc.). Approximately 50 to 100 milligrams of whole seeds, per sample, were packed in a mini sample ring cup with quartz lens[IH-0307] consisting a mini-insert [IH-0337] and scanned in reflectance mode to obtain the spectral data. Oil percentage in each seed sample was determined using the seed oil spectral calibration curve detailed above.<br>
The results of these analyses are found in Figure 5 and Table 2 and show that there was a significant increase in the oil level in seed from T2 plants expressing the LCAT2 gene. This increase in oil was seen in plants when LCAT2 was driven by either the35S constitutive promoter or the seed-specific napin promoter. These results show that over expression of a nucleic acid sequence encoding a lecithin: cholesterol acyltransferase like polypeptide can increase seed oil production in plants.<br><br><br>
In light of the detailed description of the invention and the examples presented above, it can be appreciated that the several aspects of the invention are achieved.<br>
It is to be understood that the present invention has been described in detail by way of illustration and example in order to acquaint others skilled in the art with the invention, its principles, and its practical application. Particular formulations and processes of the present invention are not limited to the descriptions of the specific embodiments presented, but rather the descriptions and examples should be viewed in terms of the claims that follow and their equivalents. While some of the examples and descriptions above include some conclusions about the way the invention may function, the inventors do not intend to be bound by those conclusions and functions, but put them forth only as possible explanations.<br>
It is to be further understood that the specific embodiments of the present invention as set forth are not intended as being exhaustive or limiting of the invention, and that many alternatives, modifications, and variations will be apparent to those of ordinary skill in the art in light of the foregoing examples and detailed description. Accordingly, this invention is intended to embrace all such alternatives, modifications, and variations that fall within the spirit and scope of the following claims,</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">351-che-2006 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLUNIRS0yMDA2IENMQUlNUyBHUkFOVEVELnBkZg==" target="_blank" style="word-wrap:break-word;">351-CHE-2006 CLAIMS GRANTED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">351-che-2006-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">351-che-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">351-che-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxLWNoZS0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">351-che-2006-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234679-an-optimized-interaction-method-of-user-terminal-selecting-access-mobile-network-in-wireless-local-area-network.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234681-electric-power-steering-apparatus.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234680</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>351/CHE/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>11-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Feb-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERGH BOULEVARD, SAINT LOUIS, MISSOURI 63167</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VAN EENENNAAM ALISON</td>
											<td>856 BURR STREET, DAVIS, CA 95616</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LASSNER, MICHAEL</td>
											<td>515 GALVESTON DRIVE, REDWOOD CITY, CA 94063</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US00/23863</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2000-08-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/152,493</td>
									<td>1999-08-30</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234680-a-method-for-modifying-the-sterol-content-of-host-cell by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:22:58 GMT -->
</html>
