<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234835-a-pharmaceutical-composition-comprising-a-recombinant-mva-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234835:&quot;A PHARMACEUTICAL COMPOSITION COMPRISING A RECOMBINANT MVA VIRUS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A PHARMACEUTICAL COMPOSITION COMPRISING A RECOMBINANT MVA VIRUS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The invention provides modified virus Ankara (MVA), a replication-deficient strain of vaccinia virus, expressing human immunodeficiency virus (HIV) env, gag, and pol genes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Brief Description of the Drawings<br>
Figure I. Phylogenetic relationships of HIV-1 and HIV-2 based on identity of pol gene sequences. SIVcpz and SrVsmm are subhuman primate lentiviruses recovered from a chimpanzee and sooty mangabey monkey, respectively.<br>
Figure II. Phylogenetic relationships of HIV-1 groups M, N and O with four different SIVcpz  isolates based on Mi-length pol gene sequences. The bar indicates a genetic distance of 0.1 (10% nucleotide divergence) and the asterisk positions group N HIV-1 isolates based on env sequences.<br>
Figure III. Tropic and biologic properties of HTV-1 isolates.<br>
Figure IV. HIV-encoded proteins. The location of the HIV genes, the sizes of primary translation products (in some cases polyproteins), and the processed mature viral proteins are indicated.<br>
Figure V. Schematic representation of a mature HTV-1 virion.<br>
Figure VI. Linear representation of the HIV-1 Env glycoprotein. 'The arrow indicates the site of gp!60 cleavage to gp!20 and gp41. In gp!20, cross-hatched areas represent variable domains (V1 to V5) and open boxes depict conserved sequences (C1 to C5). In the gp41 ectodomain, several domains are indicated: the N-terminal fusion peptide, and the two ectodomain helices (N- and C-helix). The membrane-spanning domain is represented by a black box. In the gp41 cytoplasmic domain, the Tyr-X-X-Leu (YXXL) endocytosis motif (SEQ ID NO: 9) and two predicted helical domains (helix-1 and -2) are shown. Ainino acid numbers are indicated.<br>
Figure 1. Temporal frequencies of Gag-specific T cells. (A) Gag-specific CD8 T cell responses raised by DNA priming and rMVA booster immunizations. The schematic presents mean Gag-CM9-tetramer data generated in the high-dose i.d. DNA-immumzed animals. (B) Gag-specific IFN-γ ELISPOTs in A*01 (open bars) and non-A*01 (filled bars) macaques at various times before challenge and at two weeks after challenge. Three pools of 10 to 13 Gag peptides (22-mers overlapping by 12) were used for the analyses. The numbers above data bars represent the arithmetic mean ± SD for the ELISPOTs within each group. The numbers at the top of the graphs designate individual animals. *, data not available; #, 
 <br>
Figure 2. Temporal viral loads, CD4 counts, and survival after challenge of vaccinated and control animals. (A) Geometric mean viral loads and (B) geometric mean CD4 counts. (C) Survival curve for vaccinated and control animals. The dotted line represents all 24 vaccinated animals. (D) Viral loads and (E) CD4 counts for individual animals in the vaccine and control groups. The key to animal numbers is presented in (E). Assays for the first 12 weeks after challenge had a detection level of 1000 copies of RNA per niilliliter of plasma. Animals with loads below 1000 were scored with a load of 500. For weeks 16 and 20, the detection level was 300 copies of RNA per milliliter. Animals with levels of virus below 300 were scored at 300.<br>
Figure 3. Postchallenge T cell responses in vaccine and control groups. (A) Temporal tetramer+ cells (dashed line) and viral loads (solid line). (B) Intracellular cytokine assays for IFN-γ production in response to stimulation with the Gag-CM9 peptide at two weeks after challenge. This ex vivo assay allows evaluation of the functional status of the peak postchallenge tetramer+ cells displayed in Figure 1 A. (C) Proliferation assay at 12 weeks after challenge. Gag-Pol-Env (open bars) and Gag-Pol (hatched bars) produced by transient transfections were used for stimulation. Supernatants from mock-transfected cultures served as control antigen. Stimulation indices are the growth of cultures in the presence of viral antigens divided by the growth of cultures in the presence of mock antigen.<br>
Figure 4. Lymph node histomorphology at 12 weeks after challenge. (A) Typical lymph node from a vaccinated macaque showing evidence of follicular hyperplasia characterized by the presence of numerous secondary follicles with expanded germinal centers and discrete dark and light zones. (B) Typical lymph node from an infected control animal showing follicular depletion and paracortical lymphocellular atrophy. (C) A representative lymph node from an age-matched, uninfected macaque displaying nonreactive germinal centers. (D) The percentage of the total lymph node area occupied by germinal centers was measured to give a non-specific indicator of follicular hyperplasia. Data for uninfected controls are for four age-matched rhesus macaques.<br>
Figure 5. Temporal antibody responses. Micrograms of total Gag (A) or Env (B) antibody were determined with ELISAs. The titers of neutralizing antibody for SHIV-89.6 (C) and SHIV-89.6P (D) were determined with MT-2 cell killing and neutral red staining (D.C. Montefiori e.t al. 1988 J Clin Microbiol 26:231). Titers are the reciprocal of the<br><br>
serum dilution giving 50% neutralization of the indicated viruses grown in human PBMC. Symbols for animals are the same as in Figure 2.<br>
Figure 6. Gag-CM9-Mamu-A*01 tetramer-specific T cells in Mamu-A*01 vaccinated and control macaques at various times before challenge and at two weeks after challenge. The number at the upper right comer of each plot represents the frequency of tetramer-specific CD8 T cells as a % of total CD8 T cells. The numbers above each column of FACS data designate individual animals,<br>
Figure A. Map and sequence of plasmid transfer vector pLW-48.<br>
Figure B. Sequences of plasmid transfer vector pLW-48, Psy II promoter (which controls ADA envelope expression), ADA envelope truncated, PmH5 promoter (which controls HXB2 gag pol expression), and HXB2 gag pol (with safety mutations, A integrase).<br>
Figure C. Plasmid transfer vector pLW-48 and making MVA recombinant virus MVA/HIV 48.<br>
Figure D. A clade B gag pol.<br>
Figure E. Sequence of new Psyn II promoter.<br>
Detailed Description of the Preferred Embodiment Recombinant MVA Virus<br>
Vaccinia virus, a member of the genus Orthopoxvirus in the family of Poxviridae, was used as live vaccine to immunize against the human smallpox disease. Successful worldwide vaccination with vaccinia virus culminated in the eradication of variola virus, the causative agent of the smallpox (The global eradication of smallpox. Final report of the global commission for the certification of smallpox eradication. History of Public Health, No. 4, Geneva: World Health Organization, 1980). Since that WHO declaration, vaccination has been universally discontinued except for people at high risk of poxvirus infections (e.g. laboratory workers).<br>
More recently, vaccinia viruses have also been used to engineer viral vectors for recombinant gene expression and for the potential use as recombinant live vaccines (Mackett, M. et al. 1982 PNAS USA 79:7415-7419; Smith, G.L. et al. 1984 Biotech Genet Engin Rev 2:383-407). This entails DNA sequences (genes) which code for foreign antigens being introduced, with the aid of DNA recombination techniques, into the genome of the vaccinia viruses. If the gene is integrated at a site in the viral DNA which is non-<br><br>
essential for the life cycle of the virus, it is possible for the newly produced recombinant vaccinia virus to be infectious, that is to say able to infect foreign cells and thus to express the integrated DNA sequence (EP Patent Applications No. 83,286 and No. 110,385). The recombinant vaccinia viruses prepared in this way can be used, on the one hand, as live vaccines for the prophylaxis of infectious diseases, on the other hand, for the preparation of heterologous proteins in eukaryotic cells.<br>
For vector applications health risks would be lessened by the use of a highly attenuated vaccinia virus strain. Several such strains of vaccinia virus were especially developed to avoid undesired side effects of smallpox vaccination. Thus, the modified vaccinia Ankara (MVA) has been generated by long-term serial passages of the Ankara strain of vaccinia virus (CVA) on chicken embryo fibroblasts (for review see Mayr, A. et al. 1975 Infection 3:6-14; Swiss Patent No. 568,392). The MVA virus is publicly available from American Type Culture Collection as ATCC No. VR-1508. MVA is distinguished by its great attenuation, that is to say by diminished virulence and ability to replicate in primate cells while maintaining good immunogenicity. The MVA virus has been analyzed to determine alterations in the genome relative to the parental CVA strain. Six major deletions of genomic DNA (deletion I, II, III IV, V, and VI) totaling 31,000 base pairs have been identified (Meyer, H. et al. 1991 J Gen Virol 72:1031-1038). The resulting MVA virus became severely host cell restricted to avion cells.<br>
Furthermore, MVA is characterized by its extreme attenuation. When tested in a variety of animal models, MVA was proven to be avirulent even in immunosuppressed animals. More importantly, the excellent properties of the MVA strain have been demonstrated in extensive clinical trials (Mayr A. et al. 1978 Zentralbl Bakteriol [B] 167:375-390; Stickl et al. 1974 Dtsch Med Wschr 99:2386-2392). During these studies in over 120,000 humans, including high-risk patients, no side effects were associated with the use of MVA vaccine.<br>
MVA replication in human cells was found to be blocked late in infection preventing the assembly to mature infectious virions. Nevertheless, MVA was able to express vira! and recombinant genes at high levels even in non-permissive cells and was proposed to serve as an efficient and exceptionally safe gene expression vector (Sutter, G. and Moss, B. 1992 PNAS USA 89:10847-10851). Additionally, novel vaccinia vector vaccines were established on the basis of MVA having foreign DNA sequences inserted at<br><br>
the site of deletion III within the MVA genome (Sutter, G. et al. 1994 Vaccine 12:1032-1040).<br>
The recombinant MVA vaccinia viruses can be prepared as set out hereinafter. A DNA-construct which contains a DNA-sequence which codes for a foreign polypeptide flanked by MVA DNA sequences adjacent to a naturally occurring deletion, e.g. deletion III, or other non-essential sites, within the MVA genome, is introduced into cells infected with MVA, to allow homologous recombination. Once the DNA-construct has been introduced into the eukaryotic cell and the foreign DNA has recombined with the viral DNA, it is possible to isolate the desired recombinant vaccinia virus in a manner known per se, preferably with the aid of a marker. The DNA-construct to be inserted can be linear or circular. A plasmid or polymerase chain reaction product is preferred. The DNA-construct contains sequences flanking the left and the right side of a naturally occurring deletion, e.g. deletion III, within the MVA genome. The foreign DNA sequence is inserted between the sequences flanking the naturally occurring deletion. For the expression of a DNA sequence or gene, it is necessary for regulatory sequences, which are required for the transcription of the gene, to be present on the DNA. Such regulatory sequences (called promoters) are known to those skilled in the art, and include for example those of the vaccinia 11 IcDa gene as are described in EP-A-198,328, and those of the 7.5 kDa gene (EP-A-110,385). The DNA-construct can be introduced into the MVA infected cells by transfection, for example by means of calcium phosphate precipitation (Graham et al. 1973 Viral 52:456-467; Wigler et al. 1979 Cell 16:777-785), by means of electroporation (Neumann et al. 1982 EMBO J 1:841-845), by microinjection (Graessmann et al. 1983 Meth Enzymol 101:482-492), by means of liposomes (Straubinger et al. 1983 Meth Enzymol 101:512-527), by means of spheroplasts (Schaffner 1980 PNAS USA 77:2163-2167) or by other methods known to those skilled in the art. HIVs and Their Replication<br>
The etiological agent of acquired immune deficiency syndrome (AIDS) is recognized to be a retrovirus exhibiting characteristics typical of the lentivirus genus, referred to as human immunodeficiency virus (HIV). The phylogenetic relationships of the human lentiviruses are shown in Figure 1 HIV-2 is more closely related to SlVsmm, a virus isolated from sooty mangabey monkeys in the wild, than to HIV-1. It is currently believed<br><br>
that HIV-2 represents a zoonotic transmission of SIVsmm to man. A series of lentiviral isolates from captive chimpanzees, designated SlVcpz, are close genetic relatives of HIV-1.<br>
The earliest phylogenetic analyses of HIV-1 isolates focused on samples from Europe/North America and Africa; discrete clusters of viruses were identified from these two areas of the world. Distinct genetic subtypes or clades of HIV-1 were subsequently defined and classified into three groups: M (major); O (outlier); and N (non-M or O) (Fig. II). The M group of HIV-1, which includes over 95% of the global virus isolates, consists of at least eight discrete clades (A, B, C, D, F, G, H, and J), based on the sequence of complete viral genomes. Members of HIV-1 group O have been recovered from individuals living in Cameroon, Gabon, and Equatorial Guinea; their genomes share less than 50% identity in nucleotide sequence with group M viruses. The more recently discovered group N HIV-I strains have been identified in infected Cameroonians, fail to react serologically in standard whole-virus enzyme-linked immunosorbent assay (ELISA), yet are readily detectable by conventional Western blot analysis.<br>
Most current knowledge about HIV-1 genetic variation comes from studies of group M viruses of diverse geographic origin. Data collected during the past decade indicate that the HIV-1 population present within an infected individual can vary from 6% to 10% in nucleotide sequence. HIV-1 isolates within a clade may exhibit nucleotide distances of 15% in gag and up to 30% in gp120 coding sequences. Interclade genetic variation may range between 30% and 40% depending on the gene analyzed.<br>
All of the HIV-1 group M subtypes can be found in Africa. Clade A viruses are genetically the most divergent and were the most common HIV-1 subtype in Africa early in the epidemic. With the rapid spread of HIV-1 to southern Africa during the mid to late 1990s, clade C viruses have become the dominant subtype and now account for 48% of HIV-1 infections worldwide. Clade B viruses, the most intensively studied HIV-1 subtype, remain the most prevalent isolates in Europe and North America.<br>
High rates of genetic recombination are a hallmark of retroviruses. It was initially believed that simultaneous infections by genetically diverse virus strains were not likely to be established in individuals at risk for HIV-1. By 1995, however, it became apparent that a significant fraction of the HIV-1 group M global diversity included interclade viral recombinants. It is now appreciated that HIV-1 recombinants will be found in geographic areas such as Africa, South America, and Southeast Asia, where multiple HIV-1 subtypes<br><br>
coexist and may account for more than 10% of circulating HIV-1 strains. Molecularly, the genomes of these recombinant viruses resemble patchwork mosaics, with juxtaposed diverse HIV-1 subtype segments, reflecting the multiple crossover events contributing to their generation. Most HIV-1 recombinants have arisen in Africa and a majority contain segments originally derived from clade A viruses. In Thailand, for example, the composition of the predominant circulating strain consists of a clade A gag plus pol gene segment and a clade E env gene. Because the clade E env gene in Thai HIV-1 strains is closely related to the clade E env present in virus isolates from the Central African Republic, it is believed that the original recombination event occurred in Africa, with the subsequent introduction of a descendent virus into Thailand. Interestingly, no full-length HIV-1 subtype E isolate (i.e., with subtype E gag, pol, and env genes) has been reported to date.<br>
The discovery that α and ß chemokine receptors function as coreceptors for virus fusion and entry into susceptible CD4+ cells has led to a revised classification scheme for HIV-1 (Fig. III). Isolates can now be grouped on the basis of chemokine receptor utilization in fusion assays in which HIV-1 gp120 and CD4+ coreceptor proteins are expressed in separate cells. As indicated in Figure III, HIV-1 isolates using the CXCR4 receptor (now designated X4 viruses) are usually T cell line (TCL)-tropic syncytium inducing (SI) strains, whereas those exclusively utilizing the CCR5 receptor (R5 viruses) are predominantly macrophage (M)-tropic and non-syncytium inducing (NSI). The dual-tropic R5/X4 strains, which may comprise the majority of patient isolates and exhibit a continuum of tropic phenotypes, are frequently SI.<br>
As is the case for all replication-competent retroviruses, the three primary HIV-1 translation products, all encoding structural proteins, are initially synthesized as polyprotein precursors, which are subsequently processed by viral or cellular proteases into mature particle-associated proteins (Fig. IV). The 55-kd Gag precursor Pr55Gag is cleaved into the matrix (MA), capsid (CA), nucleocapsid (NC), and p6 proteins. Autocatalysis of the 160-kd Gag-Pol polyprotein, Prl60Gag-po1, gives rise to the protease (PR), the heterodimeric reverse transcriptase (RT), and the integrase (IN) proteins, whereas proteolytic digestion by a cellular enzyme(s) converts the glycosylated 160-kd Env precursor gp160 to the gp120 surface (SU) and gp41 transmembrane (TM) cleavage products. The remaining six HIV-1-<br><br>
encoded proteins (Vif, Vpr, Tat, Rev, Vpu, and Nef) are the primary translation products of<br>
spliced mRNAs.<br>
Gag<br>
The Gag proteins of HIV, like those of other retroviruses, are necessary and sufficient for the formation of noninfectious, virus-like particles. Retroviral Gag proteins are generally synthesized as polyprotein precursors; the HIV-1 Gag precursor has been named, based on its apparent molecular mass, Pr55Gag. As noted previously, the mRNA for Pr55Gag is the unspliced 9.2-kb transcript (Fig. IV) that requires Rev for its expression in the cytoplasm. When the pol ORF is present, the viral protease (PR) cleaves Pr55Gag during or shortly after budding from the cell to generate the mature Gag proteins pi? (MA), p24 (CA), p7 (NC), and p6 (see Fig. IV). In the virion, MA is localized immediately inside the lipid bilayer of the viral envelope, CA forms the outer portion of the cone-shaped core structure in the center of the particle, and NC is present in the core in a ribonucleoprotein complex with the viral RNA genome (Fig. V).<br>
The HIV Pr55Gag precursor oligomerizes following its translation and is targeted to the plasma membrane, where particles of sufficient size and density to be visible by EM are assembled. Formation of virus-like particles by Pr55Qag is a self-assembly process, with critical Gag-Gag interactions taking place between multiple domains along the Gag precursor. The assembly of virus-like particles does not require the participation of genomic RNA (although the presence of nucleic acid appears to be essential), pol-encoded enzymes, or Env glycoproteins, but the production of infectious virions requires the encapsidation of the viral RNA genome and the incorporation of the Env glycoproteins and the Gag-Pol polyprotein precursor Prl6QGag-Po1. Pol<br>
Downstream of gag lies the most highly conserved region of the HIV genome, the pol gene, which encodes three enzymes: PR, RT, and IN (see Fig. IV). RT and IN are required, respectively, for reverse transcription of the viral RNA genome to a double-stranded DNA copy, and for the integration of the viral DNA into the host cell chromosome. PR plays a critical role late in the life cycle by mediating the production of mature, infectious virions. The pol gene products are derived by enzymatic cleavage of a 160-kd Gag-Pol fusion protein, referred to as Prl60aaE-p°l. This fusion protein is produced by ribosomal frameshifting during translation of Pr55Gag (see Fig. IV). The frame-shifting<br><br>
mechanism for Gag-Pol expression, also utilized by many other retroviruses, ensures that the pol-derived proteins are expressed at a low level, approximately 5% to 10% that of Gag. Like Pr55Gag, the N-terminus of Prl60Gag-po1 is myrisrylated and targeted to the plasma membrane. Protease<br>
Early pulse-chase studies performed with avian retroviruses clearly indicated that retroviral Gag proteins are initially synthesized as polyprotein precursors that are cleaved to generate smaller products. Subsequent studies demonstrated that the processing function is provided by a viral rather than a cellular enzyme, and that proteolytic digestion of the Gag and Gag-Pol precursors is essential for virus infectivity. Sequence analysis of retroviral PRs indicated that they are related to cellular "aspartic" proteases such as pepsin and rerun. Like these cellular enzymes, retroviral PRs use two apposed Asp residues at the active site to coordinate a water molecule that catalyzes the hydrolysis of a peptide bond in the target protein. Unlike the cellular aspartic proteases, which function as pseudodimers (using two folds within the same molecule to generate the active site), retroviral PRs function as true dimers. X-ray crystallographic data from HIV-1 PR indicate that the two monomers are held together in part by a four-stranded antiparallel ß-sheet derived from both N- and C-terminal ends of each monomer. The substrate-binding site is located within a cleft formed between the two monomers. Like their cellular homologs, the HIV PR dimer contains flexible "flaps" that overhang the binding site and may stabilize the substrate within the cleft; the active-site Asp residues lie in the center of the dimer. Interestingly, although some limited ammo acid homology is observed surrounding active-site residues, the primary sequences of retroviral PRs are highly divergent, yet their structures are remarkably similar. Reverse Transcriptasg<br>
By definition, retroviruses possess the ability to convert their single-stranded RNA genomes into double-stranded DNA during the early stages of the infection process. The enzyme that catalyzes this reaction is RT, in conjunction with its associated RNaseH activity. Retroviral RTs have three enzymatic activities: (a) RNA-directed DNA polymerization (for minus-strand DNA synthesis), (b) RNaseH activity (for the degradation of the tRNA primer and genomic RNA present in DNA-RNA hybrid intermediates), and (c) DNA-directed DNA polymerization (for second- or plus-strand DNA synthesis).<br>
The mature HIV-1 RT holoenzyme is a heterodimer of 66 and 51 kd subunits. The 51-kd subunit (p51) is derived from the 66-kd (p66) subunit by proteolytic removal of the C-terminal 15-kd RNaseH domain of p66 by PR (see Fig. IV). The crystal structure of HIV-1 RT reveals a highly asymmetric folding in which the orientations of the p66 and p51 subunits differ substantially. The p66 subunit can be visualized as a right hand, with the poiymerase active site within the palm, and a deep template-binding cleft formed by the palm, fingers, and thumb subdomains. The poiymerase domain is linked to RNaseH by the connection subdomain. The active site, located in the palm, contains three critical Asp residues (110, 185, and 186) in close proximity, and two coordinated Mg2+ ions. Mutation of these Asp residues abolishes RT polymerizing activity. The orientation of the three active-site Asp residues is similar to that observed in other DNA polymerases (e.g., the Klenow fragment of E. coli DNA poll). The p51 subunit appears to be rigid and does not form a polymerizing cleft; Asp 110, 185, and 186 of this subunit are buried within the molecule. Approximately 18 base pairs of the primer-template duplex lie in the nucleic acid binding cleft, stretching from the poiymerase active site to the RNaseH domain.<br>
In the RT-primer-template-dNTP structure, the presence of a dideoxynucleotide at the 3 end of the primer allows visualization of the catalytic complex trapped just prior to attack on the incoming dNTP. Comparison with previously obtained structures suggests a model whereby the fingers close in to trap the template and dNTP prior to nucleophilic attack of the 3-OH of the primer on the incoming dNTP. After the addition of the incoming dNTP to the growing chain, it has been proposed that the fingers adopt a more open configuration, thereby releasing the pyrophosphate and enabling RT to bind the next dNTP. The structure of the HIV-1 RNaseH has also been determined by x-ray crystallography; this domain displays a global folding similar to that of E. coli RNaseH. Integrase<br>
A distinguishing feature of retrovirus replication is the insertion of a DNA copy of the viral genome into the host cell chromosome following reverse transcription. The integrated viral DNA (the provirus) serves as the template for the synthesis of viral RNAs and is maintained as part of the host cell genome for the lifetime of the infected cell. Retroviral mutants deficient in the ability to integrate generally fail to establish a productive infection.<br>
The integration of viral DNA is catalyzed by integrase, a 32-kd protein generated by PR-mediated cleavage of the C-terminal portion of the HIV-1 Gag-Pol polyprotein (see Fig. IV).<br>
Retroviral IN proteins are composed of three structurally and functionally distinct domains: an N-terminal, zinc-finger-containing domain, a core domain, and a relatively nonconserved C-terminal domain. Because of its low solubility, it has not yet been possible to crystallize the entire 288-amino-acid HIV-1 IN protein. However, the structure of all three domains has been solved independently by x-ray crystallography or NMR methods. The crystal structure of the core domain of the avian sarcoma virus IN has also been determined. The N-terminal domain (residues 1 to 55), whose structure was solved by NMR spectroscopy, is composed of four helices with a zinc coordinated by amino acids His-12, His-16, Cys-40, and Cys-43. The structure of the N-terminal domain is reminiscent of helical DNA binding proteins that contain a so-called helix-tum-helix motif; however, in the HIV-1 structure this motif contributes to dimer formation. Initially, poor solubility hampered efforts to solve the structure of the core domain. However, attempts at crystallography were successful when it was observed that a Phe-to-Lys change at IN residue 185 greatly increased solubility without disrupting in vitro catalytic activity. Each monomer of the HIV-1 IN core domain (IN residues 50 to 212) is composed of a five-stranded p-sheet flanked by helices; this structure bears striking resemblance to other polynucleotidyl transferases including RNaseH and the bacteriophage MuA transposase. Three highly conserved residues are found in analogous positions in other polynucleotidyl transferases; in HIV-1 IN these are Asp-64, Asp-116 and Glu-152, the so-called D,D-35-E motif. Mutations at these positions block HIV IN function both in vivo and in vitro. The close proximity of these three amino acids in the crystal structure of both avian sarcoma virus and HIV-1 core domains supports the hypothesis that these residues play a central role in catalysis of the polynucleotidyl transfer reaction that is at the heart of the integration process. The C-terminal domain, whose structure has been solved by NMR methods, adopts a five-stranded ß-barrel folding topology reminiscent of a Src homology 3 (SH3) domain. Recently, the x-ray structures of SIV and Rous sarcoma virus IN protein fragments encompassing both the core and C-terminal domains have been solved.<br>
Env<br>
The HIV Env glycoproteins play a major role in the virus life cycle. They contain the determinants that interact with the CD4 receptor and coreceptor, and they catalyze the fosion reaction between the lipid bilayer of the viral envelope and the host cell plasma membrane, hi addition, the HIV Env glycoproteins contain epitopes that elicit immune responses that are important from both diagnostic and vaccine development perspectives.<br>
The HIV Env glycoprotein is synthesized from the singly spliced 4.3-kb Vpu/Env bicistronic mRNA (see Fig. IV); translation occurs on ribosomes associated with the rough endoplasmic reticulum (ER). The 160-kd polyprotein precursor (gp!60) is an integral membrane protein that is anchored to cell membranes by a hydrophobic stop-transfer signal in the domain destined to be the mature TM Env glycoprotein, gp41 (Fig. VI). The gp160 is cotranslationally glycosylated, forms disulfide bonds, and undergoes oligomerization in the ER, The predominant oligomeric form appears to be a trimer, although dimers and tetramers are also observed. The gp!60 is transported to the Golgi, where, like other retroviral envelope precursor proteins, it is proteolytically cleaved by cellular enzymes to the mature SU glycoprotein gp120 and TM glycoprotein gp41 (see Fig. VI). The cellular enzyme responsible for cleavage of retroviral Env precursors following a highly conserved Lys/Arg-X-Lys/Arg-Arg motif is furin or a form-like protease, although other enzymes may also catalyze gp!60 processing. Cleavage of gpl60 is required for Env-induced fusion activity and virus infectivity. Subsequent to gp!60 cleavage, gp!20 and gp41 form a noncovalent association that is critical for transport of the Env complex from the Golgi to the cell surface. The gpl20-gp41 interaction is fairly weak, and a substantial amount of gp120 is shed from the surface of Env-expressing cells.<br>
The HIV Env glycoprotein complex, in particular the SU (gp120) domain, is very heavily glycosylated; approximately half the molecular mass of gp160 is composed of oligosaccharide side chains. During transport of Env from its site of synthesis in the ER to the plasma membrane, many of the side chains are modified by the addition of complex sugars. The numerous oligosaccharide side chains form what could be imagined as a sugar cloud obscuring much of gp120 from host immune recognition. As shown in Figure VI, gp120 contains interspersed conserved (C1 to C5) and variable (V1 to V5) domains. The Cys residues present in the gp120s of different isolates are highly conserved and form disulfide bonds that link the first four variable regions in large loops.<br>
A primary function of viral Env glycoproteins is to promote a membrane fusion reaction between the lipid bilayers of the viral envelope and host cell membranes. This membrane fusion event enables the viral core to gain entry into the host cell cytoplasm. A number of regions in both gp120 and gp41 have been implicated, directly or indirectly, in Env-mediated membrane fusion. Studies of the HA2 hemagglutinin protein of the orthomyxoviruses and the F protein of the paramyxoviruses indicated that a highly hydrophobic domain at the N-terminus of these proteins, referred to as the fusion peptide, plays a critical role in membrane fusion. Mutational analyses demonstrated that an analogous domain was located at the N-terminus of the HIV-1, HIV-2, and SIV TM glycoproteins (see Fig. VI), Nonhydrophobic substitutions within this region of gp41 greatly reduced or blocked syncytium formation and resulted in the production of noninfectious progeny virions.<br>
C-terminal to the gp41 fusion peptide are two amphipathic helical domains (see Fig. VI) which play a central role in membrane fusion. Mutations in the N-terminal helix (referred to as the N-helix), which contains a Leu zipper-like heptad repeat motif, impair infectivity and membrane fusion activity, and peptides derived from these sequences exhibit potent antiviral activity in culture. The structure of the ectodomain of HIV-1 and SIV gp41, the two helical motifs in particular, has been the focus of structural analyses in recent years. Structures were determined by x-ray crystallography or NMR spectroscopy either for fusion proteins containing the helical domains, a mixture of peptides derived from the N- and C-helices, or in the case of the SIV structure, the intact gp41 ectodomain sequence from residue 27 to 149. These studies obtained fundamentally similar trimeric structures, in which the two helical domains pack in an antiparallel fashion to generate a six-helix bundle. The N-helices form a coiled-coii in the center of the bundle, with the C-helices packing into hydrophobic grooves on the outside.<br>
In the steps leading to membrane fusion CD4 binding induces conformation changes in Env that facilitate coreceptor binding. Following the formation of a ternary gp120/CD4/coreceptor complex, gp41 adopts a hypothetical conformation that allows the fusion peptide to insert into the target lipid bilayer. The formation of the gp41 six-helix bundle (which involves antiparallel interactions between the gp41 N- and C-helices) brings the viral and cellular membranes together and membrane fusion takes place.<br>
Use of Recombinant MVA Virus To Boost CD+8 Cell Immune Response<br>
The present invention relates to generation of a CD8+ T cell immune response against an antigen and also eliciting an antibody response. More particularly, the present invention relates to "prime and boost" immunization regimes in which the immune response induced by administration of a priming composition is boosted by administration of a boosting composition. The present invention is based on inventors1 experimental •demonstration that effective boosting can be achieved using modified vaccinia Ankara (MVA) vectors, following priming with any of a variety of different types of priming compositions including recombinant MVA itself.<br>
A major protective component of the immune response against a number of pathogens is mediated by T lymphocytes of the CD8+ type, also known as cytotoxic T lymphocytes (CTL). An important function of CD8+ cells is secretion of gamma interferon (EFNy), and this provides a measure of CD8+ T cell immune response. A second component of the immune response is antibody directed to the proteins of the pathogen.<br>
The present invention employs MVA which, as the experiments described below show, has been found to be an effective means for providing a boost to a CD8+ T cell immune response primed to antigen using any of a variety of different priming compositions and also eliciting an antibody response.<br>
Remarkably, the experimental work described below demonstrates that use of embodiments of the present invention allows for recombinant MVA virus expressing an HIV antigen to boost a CD8+ T cell immune response primed by a DNA vaccine and also eliciting an antibody response. The MVA was found to induce a CD8+ T cell response after intradermal, intramuscular or mucosal immunization. Recombinant MVA has also been shown to prime an immune response that is boosted by one or more inoculations of recombinant MVA.<br>
Non-human primates immunized with plasmid DNA and boosted with the MVA were effectively protected against intrainucosal challenge with live virus. Advantageously, the inventors found that a vaccination regime used intradermal, intramuscular or mucosal immunization for both prime and boost can be employed, constituting a general immunization regime suitable for inducing CD8+ T cells and also eliciting an antibody response, e.g. in humans.<br>
The present invention in various aspects and embodiments employs an MVA vector encoding an HIV antigen for boosting a CD8+ T cell immune response to the antigen primed by previous administration of nucleic acid encoding the antigen and also eliciting an antibody response.<br>
A general aspect of the present invention provides for the use of an MVA vector for boosting a CD8+T cell immune response to an HIV antigen and also eliciting an antibody response.<br>
One aspect of the present invention provides a method of boosting a CD8+ T cell immune response to an HIV antigen in an individual, and also eliciting an antibody response, the method including provision in the individual of an MVA vector including nucleic acid encoding the antigen operably linked to regulatory sequences for production of antigen in the individual by expression from the nucleic acid, whereby a CD8+ T cell immune response to the antigen previously primed in the individual is boosted.<br>
An immune response to an HIV antigen may be primed by immunization with plasmid DNA or by infection with an infectious agent.<br>
A further aspect of the invention provides a method of inducing a CD8+ T cell immune response to an HIV antigen in an individual, and also eliciting an antibody response, the method comprising administering to the individual a priming composition comprising nucleic acid encoding the antigen and then administering a boosting composition which comprises an MVA vector including nucleic acid encoding the antigen operably linked to regulatory sequences for production of antigen in the individual by expression from the nucleic acid.<br>
A further aspect provides for use of an MVA vector, as disclosed, in the manufacture of a medicament for administration to a mammal to boost a CD8+ T cell immune response to an HIV antigen, and also eliciting an antibody response. Such a medicament is generally for administration following prior administration of a priming composition comprising nucleic acid encoding the antigen.<br>
The priming composition may comprise any viral vector, such as a vaccinia virus vector such as a replication-deficient strain such as modified vaccinia Ankara (MVA) or NYVAC (Tartaglia et al. 1992 Virology 118:217-232), an avipox vector such as fowlpox or canarypox, e.g. the strain known as ALVAC (Paoletti et al. 1994 Dev Biol Stand 82:65-69), or an adenovirus vector or a vesicular stomatitis virus vector or an alphavirus vector.<br>
The priming composition may comprise DNA encoding the antigen, such DNA preferably being in the form of a circular plasmid that is not capable of replicating in mammalian cells. Any selectable marker should not be resistance to an antibiotic used clinically, so for example Kanamycin resistance is preferred to Ampicillin resistance. Antigen expression should be driven by a promoter which is active in mammalian cells, for instance the cytomegalovirus immediate early (CMV IE) promoter.<br>
In particular embodiments of the various aspects of the present invention, administration of a priming composition is followed by boosting with a boosting composition, or first'and second boosting compositions, the first and second boosting compositions being the same or different from one another. Still further boosting compositions may be employed without departing from the present invention. In one embodiment, a triple immunization regime employs DNA, then adenovirus as a first boosting composition, then MVA as a second boosting composition, optionally followed by a further (third) boosting composition or subsequent boosting administration of one or other or both of the same or different vectors. Another option is DNA then MVA then adenovirus., optionally followed by subsequent boosting administration of one or other or both of the same or different vectors.<br>
The antigen to be encoded in respective priming and boosting compositions (however many boosting compositions are employed) need not be identical, but should share at least one CD8+ T cell epitope. The antigen may correspond to a complete antigen, or a fragment thereof. Peptide epitopes or artificial strings of epitopes may be employed, more efficiently cutting out unnecessary protein sequence in the antigen and encoding sequence in the vector or vectors. One or more additional epitopes may be included, for instance epitopes which are recognized by T helper cells, especially epitopes recognized in individuals of different HLA types.<br>
An HIV antigen of the invention to be encoded by a recombinant MVA virus includes polypeptides having immunogenic activity elicited by an amino acid sequence of an HIV Env, Gag, Pol, Vif, Vpr, Tat, Rev, Vpu, or Nef amino acid sequence as at least one CD8+ T cell epitope. This amino acid sequence substantially corresponds to at least one 10-900 amino acid fragment and/or consensus sequence of a known HIV Env or Pol; or at least one 10-450 amino acid fragment and/or consensus sequence of a known HIV Gag; or at<br>
least one 10-100 amino acid fragment and/or consensus sequence of a known HIV Vif, Vpr, Tat,Rev,Vpu,or Nef.<br>
Although a full length Env precursor sequence is presented for use in the present invention, Env is optionally deleted of subsequences. For example, regions of the gp120 surface and gp41 transmembrane cleavage products can be deleted.<br>
Although a full length Gag precursor sequence is presented for use in the present invention, Gag is optionally deleted of subsequences. For example, regions of the matrix protein (p17), regions of the capsid protein (p24), regions of the nucleocapsid protein (p7), and regions of p6 (the C-terminal peptide of the Gag polyprotein) can be deleted.<br>
Although a full length Pol precursor sequence is presented for use in the present invention, Pol is optionally deleted of subsequences. For example, regions of the protease protein (p10), regions of the reverse transcriptase protein (p66/p51), and regions of the integrase protein (p32) can be deleted.<br>
Such an HIV Env, Gag, or Pol can have overall identity of at least 50% to a known Env, Gag, or Pol protein amino acid sequence, such as 50-99% identity, or any range or value therein, while eliciting an immunogenic response against at least one strain of an HIV.<br>
Percent identify can be determined, for example, by comparing sequence information using the GAP computer program, version 6.0, available from the University of Wisconsin Genetics Computer Group (UWGCG). The GAP program utilizes the alignment method of Needleman and Wunsch (J Mol Biol 1970 48:443), as revised by Smith and Waterman (Adv Appl Math 1981 2:482). Briefly, the GAP program defines identity as the number of aligned symbols (i.e., nucleotides or amino acids) which are identical, divided by the total number of symbols in the shorter of the two sequences. The preferred default parameters for the GAP program include: (1) a unitary comparison matrix (containing a value of 1 for identities and 0 for non-identities) and the weighted comparison matrix of Gribskov and Burgess (Nuct Acids Res 1986 14:6745), as described by Schwartz and Dayhoff (eds., Atlas of Protein Sequence and Structure, National Biomedical Research Foundation, Washington, D.C. 1979, pp. 353-358); (2) a penalty of 3.0 for each gap and an additional 0.10 penalty for each symbol in each gap; and (3) no penalty for end gaps.<br>
In a preferred embodiment, an Env of the present invention is a variant form of at least one HIV envelope protein. Preferably, the Env is composed of gp120 and the<br>
membrane-spanning and ectodomain of gp41 but lacks part or all of the cytoplasmic domain of gp41.<br>
Known HIV sequences are readily available from commercial and institutional HIV sequence databases, such as GENBANK, or as published compilations, such as Myers et al. eds., Human Retroviruses and AIDS, A Compilation and Analysis of Nucleic Acid and Amino Acid Sequences, Vol. I and H, Theoretical Biology and Biophysics, Los Alamos, N. Mex. (1993), or http://hiv-web.lanl.gov/.<br>
Substitutions or insertions of an HIV Env, Gag, or Pol to obtain an additional HIV Env, Gag, or Pol, encoded by a nucleic acid for use in a recombinant MVA virus of the present invention, can include substitutions or insertions of at least one amino acid residue (e.g., 1-25 amino acids). Alternatively, at least one amino acid (e.g., 1-25 amino acids) can be deleted from an HIV Env, Gag, or Pol sequence. Preferably, such substitutions, insertions or deletions are identified based on safety features, expression levels, immunogenicity and compatibility with high replication rates of MVA.<br>
Amino acid sequence variations in an HIV Env, Gag, or Pol of the present invention can be prepared e.g., by mutations in the DNA. Such HIV Env, Gag, or Pol include, for example, deletions, insertions or substitutions of nucleotides coding for different amino acid residues within the amino acid sequence. Obviously, mutations that will be made in nucleic acid encoding an HIV Env, Gag, or Pol must not place the sequence out of reading frame and preferably will not create complementary domains that could produce secondary mRNA structures.<br>
HIV Env, Gag, or Pol-encoding nucleic acid of the present invention can also be prepared by amplification or site-directed mutagenesis of nucleotides in DNA or RNA encoding an HIV Env, Gag, or Pol and thereafter synthesizing or reverse transcribing the encoding DNA to produce DNA or RNA encoding an HIV Env, Gag, or Pol, based on the teaching and guidance presented herein.<br>
Recombinant MVA viruses expressing HIV Env, Gag, or Pol of the present invention, include a finite set of HIV Env, Gag, or Pol-encoding sequences as substitution nucleotides that can be routinely obtained by one of ordinary skill in the art, without undue experimentation, based on the teachings and guidance presented herein. For a detailed description of protein chemistry and structure, see Schulz, G.E. et al., 1978 Principles of Protein Structure, Springer-Verlag, New York, N.Y., and Creighton, T.E., 1983 Proteins:<br>
Structure and Molecular Properties, W. H. Freeman &amp; Co., San Francisco, CA. For a presentation of nucleotide sequence substitutions, such as codon preferences, see Ausubel et al. eds. Current Protocols in Molecular Biology, Greene Publishing Assoc., New York, N.Y. 1994 at §§ A.U-A.1.24, and Sambrook, J. et al. 1989 Molecular Cloning: A Laboratory Manual, Second Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. at Appendices C and D.<br>
Thus, one of ordinary skill in the art, given the teachings and guidance presented herein, will know how to substitute other amino acid residues in other positions of an HIV env, gag, or pol DNA or RNA to obtain alternative HIV Env, Gag, or Pol, including substitutional, deletional or insertional variants.<br>
Within the MVA vector, regulatory sequences for expression of the encoded antigen will include a natural, modified or synthetic poxvirus promoter. By "promoter" is meant a sequence of nucleotides from which transcription may be initiated of DNA operably linked downstream (i.e. in the 3' direction on the sense strand of double-stranded DNA). "Operably linked" means joined as part of the same nucleic acid molecule, suitably positioned and oriented for transcription to be initiated from the promoter. DNA operably linked to a promoter is "under transcriptional initiation regulation" of the promoter. Other regulatory sequences including terminator fragments, polyadenylation sequences, marker genes and other sequences may be included as appropriate, in accordance with the knowledge and practice of the ordinary person skilled in the art: see, for example, Moss, B. (2001). Poxviridae: the viruses and their replication. In Fields Virology, D.M. Knipe, and P.M. Howley, eds. (Philadelphia, Lippincott Williams &amp; Wilkins), pp. 2849-2883. Many known techniques and protocols for manipulation of nucleic acid, for example in preparation of nucleic acid constructs, mutagenesis, sequencing, introduction of DNA into cells and gene expression, and analysis of proteins, are described in detail in Current Protocols in Molecular Biology, 1998 Ausubel et al. eds., John Wiley &amp; Sons.<br>
Promoters for use in aspects and embodiments of the present invention must be compatible with poxvirus expression systems and include natural, modified and synthetic sequences.<br>
Either or both of the priming and boosting compositions may include an adjuvant, such as granulocyte macrophage-colony stimulating factor (GM-CSF) or encoding nucleic acid therefor.<br>
Administration of the boosting composition is generally about 1 to 6 months after administration of the priming composition, preferably about 1 to 3 months.<br>
Preferably, administration of priming composition, boosting composition, or both priming and boosting compositions, is intradermal, intramuscular or mucosal immunization.<br>
Administration of MVA vaccines may be achieved by using a needle to inject a suspension of the virus. An alternative is the use of a needleless injection device to administer a vims suspension (using, e.g., Biojector™ needleless injector) or a resuspended freeze-dried powder containing the vaccine, providing for manufacturing individually prepared doses that do not need cold storage. This would be a great advantage for a vaccine that is needed in rural areas of Africa.<br>
MVA is a virus with an excellent safety record in human immunizations. The generation of recombinant viruses can be accomplished simply, and they can be manufactured reproducibly in large quantities. Intradermal, intramuscular or mucosal administration of recombinant MVA virus is therefore highly suitable for prophylactic or therapeutic vaccination of humans against AIDS which can be controlled by a CD8+ T cell response.<br>
The individual may have AIDS such that delivery of the antigen and generation of a CD8+ T cell immune response to the antigen is of benefit or has a therapeutically beneficial effect.<br>
Most likely, administration will have prophylactic aim to generate an immune response against HIV or AIDS before infection or development of symptoms.<br>
Components to be administered in accordance with the present invention may be formulated in pharmaceutical compositions. These compositions may comprise a pharmaceutically acceptable excipient, carrier, buffer, stabilizer or other materials well known to those skilled in the art. Such materials should be non-toxic and should not interfere with the efficacy of the active ingredient. The precise nature of the carrier or other material may depend on the route of administration, e.g. intravenous, cutaneous or subcutaneous, nasal, intramuscular, intrapentoneal routes.<br>
As noted, administration is preferably intradermal, intramuscular or mucosal.<br>
Physiological saline solution, dextrose or other saccharide solution or glycols such as ethylene glycol, propylene glycol or polyethylene glycol may be included.<br>
For intravenous, cutaneous, subcutaneous, intramuscular or mucosal injection, injection at the site of affliction, the active ingredient will be in the form of a parenterally acceptable aqueous solution which is pyrogen-free and has suitable pH, isotonicity and stability. Those of relevant skill in the art are well able to prepare suitable solutions using, for example, isotonic vehicles such as Sodium Chloride Injection, Ringer's Injection, Lactated Ringer's Injection. Preservatives, stabilizers, buffers, antioxidants and/or other additives may be included as required.<br>
A slow-release formulation may be employed.<br>
Following production of MVA particles and optional formulation of such particles into compositions, the particles may be administered to an individual, particularly human or other primate. Administration may be to another mammal, e.g. rodent such as mouse, rat or hamster, guinea pig, rabbit, sheep, goat, pig, horse, cow, donkey, dog or cat.<br>
Administration is preferably in a "prophylactically effective amount" or a "therapeutically effective amount" (as the case may be, although prophylaxis may be considered therapy), this being sufficient to show benefit to the individual. The actual amount administered, and rate and time-course of administration, will depend on the nature and severity of what is being treated. Prescription of treatment, e.g. decisions on dosage etc, is within the responsibility of general practitioners and other medical doctors, or in a veterinary context a veterinarian, and typically takes account of the disorder to be treated, the condition of the individual patient, the site of delivery, the method of administration and other factors known to practitioners. Examples of the techniques and protocols mentioned above can be found in Remington's Pharmaceutical Sciences, 16th edition, 1980, Osol, A. (ed.).<br>
In one preferred regimen, DNA is administered at a dose of 250 µg to 2.5 mg/injection, followed by MVA at a dose of 106 to 109 infectious virus particles/injection.<br>
A composition may be administered alone or in combination with other treatments, either simultaneously or sequentially dependent upon the condition to be treated.<br>
Delivery to a non-human mammal need not be for a therapeutic purpose, but may be for use in an experimental context, for instance in investigation of mechanisms of immune responses to an antigen of interest, e.g. protection against HIV or AIDS.<br>
Further aspects and embodiments of the present invention will be apparent to those of ordinary skill in the art, in view of the above disclosure and following experimental<br>
exemplification, included by way of illustration and not limitation, and with reference to the attached figures.<br>
EXAMPLE 1<br>
Control of a Mucosal Challenge and Prevention of AIDS by a Multiprotein DNA/MVA<br>
Vaccine<br>
Here we tested DNA priming and poxvirus boosting for the ability to protect against a highly pathogenic mucosal challenge. The 89.6 chimera of simian and human immunodeficiency viruses (SHIV-89,6) was used for the construction of immunogens and its highly pathogenic derivative, SHIV-89.6P, for challenge (G.B. Karlsson et al. 1997 J Virol 71:4218). SHIV-89.6 and SHIV-89.6P do not generate cross-neutralizing antibody (D.C... Montefiori et al. 1998 J Virol 72:3427) and allowed us to address the ability of vaccine-raised T cells and non-neutralizing antibodies to control an immunodeficiency virus challenge. Modified vaccinia Ankara (MVA) was used for the construction of the recombinant poxvirus. MVA has been highly effective at boosting .DNA-primed CD8 T cells and enjoys the safety feature of not replicating efficiently in human or monkey cells (H.L. Robinson et al. 2000 AIDS Reviews 2:105).<br>
To ensure a broad immune response both the DNA and recombinant MVA (rMVA) components of the vaccine expressed multiple immunodeficiency virus proteins. The DNA prime (DNA/89,6) expressed simian immunodeficiency virus (SIV) Gag, Pol, Vif, Vpx, and Vpr and human immunodeficiency virus-1 (HIV-1) Env, Tat, and Rev from a single transcript (R.J. Gorelick et al. 1999 Virology 253:259; M.M. Sauter et al 1996 / Cell Biol 132:795).<br>
Molecularly cloned SHIV-89.6 sequences were cloned into the vector pGA2 using Clal and Rsrll sites. This cloning deleted both long terminal repeats (LTRs) and nef. The SHIV-89.6 sequences also were internally mutated for a 12-base pan: region encoding the first four ammo acids of the second zinc finger in nucleocapsid. This mutation renders SHIV viruses noninfectious (R.J. Gorelick et al 1999 Virology 253:259). A mutation in gp41 converted the tyrosine at position 710 to cysteine to achieve better expression of Env on the plasma membrane of DNA-expressing cells (M.M. Sauter et al. 1996 J Cell Biol 132:795). pGA2 uses the CMV immediate early promoter without intron A and the bovine growth hormone polyadenylation sequence to express vaccine inserts. Vaccine DNA was<br>
produced by Althea (San Diego, CA). In transient transfections of 293T cells, DNA/89.6 produced about 300 ng of Gag and 85 ng of Env per IxlO6 cells.<br>
The rMVA booster (MVA/89.6) expressed SIV Gag, Pol, and HIV-1 Env under the control of vaccinia virus early/late promoters.<br>
The MVA double recombinant virus expressed both the HIV 89.6 Env and the SIV 239 Gag-Pol, which were inserted into deletion n and deletion III of MVA, respectively. The 89.6 Env protein was truncated for the COOH-terminal 115 amino acids of gp41. The modified H5 promoter controlled the expression of both foreign genes.<br>
Vaccination was accomplished by priming with DNA at 0 and 8 weeks and boosting with rMVA at 24 weeks (Fig. 1 A).<br>
Id. and i.m. DNA immunizations were delivered in phosphate-buffered saline (PBS) with a needleless jet injector (Bioject, Portland, OR) to deliver five i.d. 100-ul injections to each outer thigh for the 2.5-mg dose of DNA or one i.d. 100-ul injection to the right outer thigh for the 250-µg dose of plasmid. I.m. deliveries of DNA were done with one 0.5-ml injection of DNA in PBS to each outer thigh for the 2.5-mg dose and one 100-µl injection to the right outer thigh for the 250-µg dose. 1x108 pfu of MVA/89.6 was administered both i.d. and i.m. with a needle. One 100-µl dose was delivered to each outer thigh for the i.d. dose and one 500-µl dose to each outer thigh for the i.m dose. Control animals received 2.5 nig of the pGA2 vector without vaccine insert with the Bioject device to deliver five 100-µl doses i.d. to each outer thigh. The control MVA booster immunization consisted of 2x108 pfu of MVA without an insert delivered i.d. and i.m. as described for MVA/89.6.<br>
Four groups of six rhesus macaques each were primed with either 2.5 mg (high-dose) or 250 µg (low-dose) of DNA by intradermal (i.d.) or intramuscular (i.m.) routes using a needleless jet injection device (Bioject, Portland, OR) (T.M. Allen et al. 2000 J Immunol 164:4968).<br>
Young adult rhesus macaques from the Yerkes breeding colony were cared for under guidelines established by the Animal Welfare Act and the NTH "Guide for the Care and Use of Laboratory Animals" with protocols approved by the Emory University Institutional Animal Care and Use Committee. Macaques were typed for the Mamu-A*01 allele with polymerase chain reaction (PCR) analyses (M.A. Egan et al. 2000 J Virol 74:7485; 1. Ourmanov et al 2000 ./ Virol 74:2740). Two or more animals containing at<br>
least one Mamu-A*01 allele were assigned to each group. Animal numbers are as follows: 1, RBr-5*; 2, RIm-5*; 3, RQf-5*; 4, RZe-5; 5, ROm-5; 6, RDm-5; 7, RAj-5*; 8, RJi-5*; 9, RA1-5*; 10, RDe-5*; 11, RAi-5; 12, RPr-5; 13, RKw-4*; 14, RWz-5*; 15, RGo-5; 16, RLp-4; 17, RWd-6; 18, RAt-5; 19, RPb-5*; 20, RIi-5*; 21, RIq-5; 22, RSp-4; 23, RSn-5; 24, RGd-6; 25, RMb-5*; 26, RGy-5*; 27, RUs-4; and 28, RPm-5. Animals with the A*01 allele are indicated with asterisks.<br>
Gene gun deliveries of DNA were not used because these had primed non-protective immune responses in a 1996 - 98 trial (H.L. Robinson et al 1999 Nat Med 5:526). The MVA/89.6 booster immunization (2x108 plaque-forming units, pfu) was injected with a needle both i.d. and i.m. A control group included two mock immunized animals and two naive animals. The challenge was given at 7 months after the rMVA booster to test for the generation of long-term immunity. Because most HIV-1 infections are transmitted across mucosal surfaces, an intrarectal challenge was administered.<br>
DNA priming followed by rMVA boosting generated high frequencies of virus-specific T cells that peaked at one week following the rMVA booster (Fig. 1). The frequencies of T cells recognizing the Gag-CM9 epitope were assessed by means of Mamu-A*01 tetramers, and the frequencies of T cells recognizing epitopes throughout Gag were assessed with pools of overlapping peptides and an enzyme-linked immunospot (ELISPOT) assay (C.A. Power et al. 1999 J Immunol Methods 227:99).<br>
For tetramer analyses, about 1xlO6 peripheral blood mononuclear cells (PBMC) were surface-stained with antibodies to CD8 conjugated to fluorescein isothiocyanate (F1TC) (FN-18; Biosource International, Camarillo, CA), CD8 conjugated to peridinin chlorophyl protein (PerCP) (SKI; Becton Dickinson, San Jose, CA), and Gag-CM9 (CTPYDINQM)-Mamu-A *01 tetramer (SEQ ID NO: 6) conjugated to allophycocyanin (APC), in a volume of 100µl at 8° to 10°C for 30 min. Cells were washed twice with cold PBS containing 2% fetal bovine serum (FBS), fixed with 1% paraformaldehyde in PBS, and analyzed within 24 hrs on a FACScaliber (Becton Dickinson, San Jose, CA). Cells were initially gated on lymphocyte populations with forward scatter and side scatter and then on CD3 cells. The CD3 cells were then analyzed for CD8 and tetramer-binding cells. About 150,000 lymphocytes were acquired for each sample. Data were analyzed using FloJo software (Tree Star, San Carlos, CA).<br>
For interferon-γ (IFN-γ) ELISPOTs, MULTISCREEN 96 well filtration plates (Millipore Inc. Bedford, MA) were coated overnight with antibody to human IFN-y (Clone B27, Pharmingen, San Diego, CA) at a concentration of 2 µg/ml in sodium bicarbonate buffer (pH 9.6) at 8° to 10°C. Plates were washed two times with RPMI medium and then blocked for 1 hour with complete medium (RPMI containing 10% FBS) at 37°C. Plates were washed five more times with plain RPMI medium, and cells were seeded in duplicate in 100µl complete medium at numbers ranging from 2x104  to 5xl05 cells per well. Peptide pools were added to each well to a final concentration of 2 µg/ml of each peptide in a volume of 100 µl in complete medium. Cells were cultured at 37°C for about 36 hrs under 5% CO2. Plates were washed six times with wash buffer (PBS with 0.05% Tween-20) and then incubated with 1 µg of biotinylated antibody to human IFN-y per milliliter (clone 7-86-1; Diapharma Group, West Chester, OH) diluted in wash buffer containing 2% FBS. Plates were incubated for 2 hrs at 37°C and washed six times with wash buffer. Avidin-horseradish peroxidase (Vector Laboratories, Burlingame, CA) was added to each well and incubated for 30 to 60 min at 37°C. Plates were washed six times with wash buffer and spots were developed using stable DAB as substrate (Research Genetics, Huntsville, AL). Spots were counted with a stereo dissecting microscope. An ovalbumin peptide (SHNFEKL) (SEQ ID NO: 7) was included as a control in each analysis. Background spots for the ovalbumin peptide were generally 20) were considered significant. The frequencies of ELISPOTs are approximate because different dilutions of cells have different efficiencies of spot formation in the absence of feeder cells (C.A. Power et al. 1999 J Immunol Methods 227: 99). The same dilution of cells was used for all animals at a given time point, but different dilutions were used to detect memory and acute responses.<br>
Gag-CM9 tetramer analyses were restricted to macaques that expressed the Mamu-A*OJ histocompatibility type, whereas ELISPOT responses did not depend on a specific histocompatibility type. As expected, the DNA immunizations raised low levels of memory cells that expanded to high frequencies within 1 week of the rMVA booster (Fig. 1 and 6). In  Mamu~A *01 macaques, CD8 cells specific to the Gag-CM9 epitope expanded to frequencies as high as 19% of total CD8 T cells (Fig. 6). This peak of specific cells underwent a 10- to 100-fold contraction into the DNA/MVA memory pool (Fig. 1A and 6).<br>
ELISPOTs for three pools of Gag peptides also underwent a major expansion (frequencies up to 4000 spots for 1xlO6 PBMC) before contracting from 5- to 20-fold into the DNA/MVA memory response (Fig.1B), The frequencies of ELISPOTs were the same in macaques with and without the A*01 histocompatibility type (P&gt;0.2).<br>
Simple linear regression was used to estimate correlations between postbooster and postchallenge ELISPOT responses, between memory and postchallenge ELISPOT responses, and between logarithmically transformed viral loads and ELISPOT frequencies. Comparisons between vaccine and control groups and A*01 and nonA*01 macaques were performed by means of two-sample t tests with logarithmically transformed viral load and ELISPOT responses. Two-way analyses of variance were used to examine the effects of dose and route of administration on peak DNA/MVA ELISPOTs, on memory DNA/MVA ELISPOTs, and on logarithmically transformed Gag antibody data.<br>
At both peak and memory phases of the vaccine response, the rank order for the height of the ELISPOTs in the vaccine groups was 2.5 mg i.d. &gt; 2.5 mg i.m. &gt; 250 ug id. &gt; 250 ug i.m. (Fig. 1B). The IFN-γ ELISPOTs included both CD4 and CD8 cells. Gag-CM9-specific CD8 cells had good lytic activity after restimulation with peptide.<br>
The highly pathogenic SHIV-89.6P challenge was administered intrarectally at 7 months after the rMVA booster, when vaccine-raised T cells were in memory (Fig. 1).<br>
The challenge stock (5.7 x 109 copies of viral RNA per milliliter) was produced by one intravenous followed by one intrarectal passage in rhesus macaques of the original SHIV-89.6P stock (G.B. Karlsson et al. 1997 J Virol 71:4218). Lymphoid cells were harvested from the intrarectally infected animal at peak viremia, CD8-depleted, and mitogen-stimulated for stock production. Before intrarectal challenge, fasted animals were anesthetized (ketamine, 10 mg/kg) and placed on their stomach with the pelvic region slightly elevated. A feeding tube (8Fr (2.7 mm) x 16 inches (41 cm); Sherwood Medical, St. Louis, MO) was inserted into the rectum for a distance of 15 to 20 cm. Following insertion of the feeding tube, a syringe containing 20 intrarectal infectious doses in 2 ml of RPMI-1640 plus 10% FBS was attached to the tube and the inoculum was slowly injected into the rectum. After delivery of the inoculum, the feeding tube was flushed with 3.0 ml of RPMI without FBS and then slowly withdrawn. Animals were left in place, with pelvic regions slightly elevated, for a period of ten minutes after the challenge.<br>
The challenge infected all of the vaccinated and control animals (Fig. 2). However, by 2 weeks after challenge, titers of plasma viral RNA were at least 10-fold lower in the vaccine groups (geometric means of IxlO7 to 5xl07) than in the control animals (geometric mean of 4xl08) (Fig. 2A) (S. Staprans et al in: Viral Genome Methods K. Adolph, ed. CRC Press, Boca Raton, FL, 1996 pp. 167-184; R. Hofinann-Lehmann et al. 2000 AIDS Res Hum Retroviruses 16:1247).<br>
For the determination of SHIV copy number, viral RNA from 150 µl of ACD anticoagulated plasma was directly extracted with the QIAamp Viral RNA kit (Qiagen), eluted in 60 µl of AVE buffer, and frozen at -80°C until SHIV RNA quantisation was performed. Five microliters of purified plasma RNA was reverse-transcribed in a final 20-µl volume containing 50 mM KC1, 10 mM Tris-HCl (pH 8.3), 4 mM MgCl2, 1 mM each deoxynucleotide triphosphate (dNTP), 2.5 uM random hexamers, 20 units MultiScribe RT, and 8 units ribonuclease inhibitor. Reactions were incubated at 25°C for 10 min, followed by incubation at 42°C for 20 min, and inactivation of reverse transcriptase at 99°C for 5 min. The reaction mix was adjusted to a final volume of 50 µl containing 50 mM KC1, 10 mM Tris-HCl (pH 8.3), 4 mM MgCl2,0.4 mM each dNTP, 0.2 uM forward primer, 0.2 uM reverse primer, 0.1 µM probe, and 5 units AmpliTaq Gold DNA pblymerase (all reagents from PerkmElmer Applied Biosystems, Foster City, CA). The primer sequences within a conserved portion of the SIV gag gene are the same as those described previously (S. Staprans et al in: Viral Genome Methods K. Adolph, ed. CRC Press, Boca Raton, FL, 1996 pp. 167-184). A PerkinElmer Applied Biosystems 7700 Sequence Detection System was used with the PCR profile: 95°C for 10 min, followed by 40 cycles at 93°C for 30 s, and 59.5°C for 1 min, PCR product accumulation was monitored with the 7700 sequence detector and a probe to an internal conserved gag gene sequence: 6FAM-CTGTCTGCGTCATTTGGTGC-Tamra (SEQ ID NO: 8), where FAM and Tamra denote the reporter and quencher dyes. SHTV RNA copy number was determined by comparison with an external standard curve consisting of virion-derived SIVmac239 RNA quantified by the SIV bDNA method (Bayer Diagnostics, Emeryville, CA). All specimens were extracted and amplified in duplicate, with the mean result reported. With a 0.15-ml plasma input, the assay has a sensitivity of 103 RNA copies per milliliter of plasma and a linear dynamic range of 103 to 108 RNA copies (R2 = 0.995). The intraassay coefficient of variation was 104 SHIV RNA copies per milliliter, and<br>
By 8 weeks after challenge, both high-dose DNA-primed groups and the low-dose i.d. DNA-primed group had reduced their geometric mean loads to about 1000 copies of viral RNA per milliliter. At this time, the low-dose i.m. DNA-primed group had a geometric mean of 6x103 copies of viral RNA and the nonvaccinated controls had a geometric mean of 2 x 106. By 20 weeks after challenge, even the low-dose i.m. group had reduced its geometric mean copies of viral RNA to 1000. Among the 24 vaccinated animals, only one animal, animal number 22 in the low-dose i.m. group, had intermittent viral loads above 1xlO4 copies per milliliter (Fig 2D).<br>
By 5 weeks after challenge, all of the nonvaccinated controls had undergone a profound depletion of CD4 cells (Fig 2B). All of the vaccinated animals maintained their CD4 cells, with the exception of animal 22 in the low dose i.m. group (see above), which underwent a slow CD4 decline (Fig. 2E). By 23 weeks after challenge, three of the four control animals had succumbed to AIDS (Fig. 2C). These animals had variable degrees of enterocolitis with diarrhea, cryptosporidiosis, colicystitis, enteric canipylobacter infection, splenomegaly, lymphadenopathy, and SIV-associated giant cell pneumonia. In contrast, all 24 vaccinated animals maintained their health.<br>
Containment of the viral challenge was associated with a burst of antiviral T cells (Fig. 1 and 3A). At one week after challenge, the frequency of tetramer+ cells in the peripheral blood had decreased, potentially reflecting the recruitment of specific T cells to the site of infection (Fig. 3A). However, by two weeks after challenge, tetramer+ cells in the peripheral blood had expanded to frequencies as high as, or higher than, after the rMVA booster (Fig- 1 and 3A). The majority of the tetramer+ cells produced IFN-γ in response to a 6-hour peplide stimulation (Fig. 3B) (S.L. Waldrop ct al 1997 J Clin Invest 99:1739) and<br><br>
did not have the "stunned" IFN-y negative phenotype sometimes observed in viral infections (F. Lechner et al 2000 JExp Med 191:1499).<br>
For intracellular cytokine assays, about 1x106 PBMC were stimulated for 1 hour at 37°C in 5 ml polypropylene tubes with 100 µg of Gag-CM9 peptide (CTPYDINQM) (SEQ ID NO: 6) per milliliter in a volume of 100 µl RPMI containing 0.1% bovine serum albumin (BSA) and 1 ng of antibody to human CD28 and 1 µg of antibody to human CD49d (Pharmingen, San Diego, CA) per milliliter. Then, 900 µl of RPMI containing 10% FBS and monensin (10 µg/ml) was added, and the cells were cultured for an additional 5 hrs at 37°C at an angle of 5° under 5% CO2. Cells were surface stained with antibodies to CD8 conjugated to PerCP (clone SKI, Becton Dickinson) at 8° to 10°C for 30 min, washed twice with cold PBS containing 2% FBS, and fixed and permeabilized with Cytofix/Cytoperm solution (Pharmingen). Cells were then incubated with antibodies to human CD3 (clone FN-18; Biosource International, Camarillo, CA) and IFN-y (Clone B27; Pharmingen) conjugated to FITC and phycoerythrin, respectively, in Perm wash solution (Pharmingen) for 30 min at 4°C. Cells were washed twice with Perm wash, once with plain PBS, and resuspended in 1% paraformaldehyde in PBS. About 150,000 lymphocytes were acquired on the FACScaliber and analyzed with FloJo software.<br>
The postchallenge burst of T cells contracted concomitant with the decline of the viral load. By 12 weeks after challenge, virus-specific T cells were present at about one-tenth of their peak height (Figs. 1A and 3A). In contrast to the vigorous secondary response in the vaccinated animals, the naive animals mounted a modest primary response (Fig. IB and 3 A). Tetramer+ cells peaked at less than 1% of total CD8 cells (Fig. 3 A), and IFN-γ-producing ELlSPOTs were present at a mean frequency of about 300 as opposed to the much higher frequencies of 1000 to 6000 in the vaccine groups (Fig. IB) (PO.05).<br>
The tetramer+ cells in the control group, like those in the vaccine group, produced IFN-y after peptide stimulation (Fig. 3B). By 12 weeks after challenge, three of the four controls had undetectable levels of IFN-γ-producing ELlSPOTs. This rapid loss of antiviral T cells in the presence of high viral loads may reflect the lack of CD4 help.<br>
T cell proliferative responses demonstrated that virus-specific CD4 cells had survived the challenge and were available to support the antiviral immune response (Fig. 3C).<br>
About 0.2 million PBMC were stimulated in triplicate for 5 days with the indicated antigen in 200 µl of RPMI at 37°C under 5% CO2. Supernatants from 293T cells transfected with DNA expressing either SHIV-89.6 Gag and Pol or SHIV-89.6 Gag, Pol and Env were used directly as antigens (final concentration of -0.5 µg of p27 Gag per milliliter). Supernatants from mock DNA (vector alone)-trausfected cells served as negative controls. On day six, cells were pulsed with 1µCi of tritiated thymidine per well for 16 to 20 hours. Cells were harvested with an automated cell harvester (TOMTEC, Harvester 96, Model 1010, Hamden, CT) and counted with a Wallac 1450 MICROBETA Scintillation counter (Gaithersburg, MD). Stimulation indices are the counts of tritiated thymidine incorporated in PBMC stimulated with 89.6 antigens divided by the counts of tritiated thymidine incorporated by the same PBMC stimulated with mock antigen.<br>
At 12 weeks after challenge, mean stimulation indices for Gag-Pol-Env or Gag-Pol proteins ranged from 35 to 14 in the vaccine groups but were undetectable in the control group, Consistent with the proliferation assays, intracellular cytokine assays demonstrated the presence of virus-specific CD4 cells in vaccinated but not control animals. The overall rank order of the vaccine groups for the magnitude of the proliferative response was 2.5 mg i.d. &gt; 2.5 mg i.m. &gt; 250 µg i.d. &gt; 250 µg i.m.<br>
At 12 weeks after challenge, lymph nodes from the vaccinated animals were morphologically intact and responding to the infection, whereas those from the infected controls had been functionally destroyed (Fig. 4). Nodes from vaccinated animals contained large numbers of reactive secondary follicles with expanded germinal centers and discrete dark and light zones (Fig. 4A). By contrast, lymph nodes from the non-vaccinated control animals showed follicular and paracortical depletion (Fig. 4B), while those from unvaccinated and unchallenged animals displayed normal numbers of minimally reactive germinal centers (Fig. 4C). Germinal centers occupied 
 <br>
undergone a profound depletion in CD4 T cells, the cytomorphology of infected lymph node cells was consistent with a macrophage phenotype.<br>
The prime/boost strategy raised low levels of antibody to Gag and undetectable levels of antibody to Env (Fig. 5). Postchallenge, antibodies to both Env and Gag underwent anamnestic responses with total Gag antibody reaching heights approaching 1 rag/ml and total Env antibody reaching heights of up to 100µg/ml.<br>
Enzyme-linked immunosorbent assays (ELISAs) for total antibody to Gag used bacterially produced SIV gag p27 to coat wells (2µg per milliliter in bicarbonate buffer). ELISAs for antibody to Env antibody used 89.6 Env produced in transiently transfected 293T cells and captured with sheep antibody against Env (catalog number 6205; International Enzymes, Fairbrook CA). Standard curves for Gag and Env ELISAs were produced with serum from a SHIV-89.6-infected macaque with known amounts of immunoglobulin G (IgG) specific for Gag or Env. Bound antibody was detected with peroxidase-conjugated goat antibody to macaque IgG (catalog # YNGMOIGGFCP; Accurate Chemical, Westbury, NY) and TMB substrate (Catalog # T3405; Sigma, St. Louis, MO). Sera were assayed at threefold dilutions in duplicate wells. Dilutions of test sera were performed in whey buffer (4% whey and 0.1% tween 20 in IX PBS). Blocking buffer consisted of whey buffer plus 0.5% nonfat dry milk. Reactions were stopped with 2M H2SO4 and the optical density read at 450 nm. Standard curves were fitted and sample concentrations were interpolated as ug of antibody per ml of serum using SOFTmax 2.3 software (Molecular Devices, Sunnyvale, CA).<br>
By 2 weeks after challenge, neutralizing antibodies for the 89.6 immunogen, but not the SHIV-89.6P challenge, were present in the high-dose DNA-primed groups (geometric mean liters of 352 in the i.d. and 303 in the i.m. groups) (Fig. 5C) (D.C. Montefiori et al 1988 J Clin Microbiol 26:231). By 5 weeks after challenge, neutralizing antibody to 89.6P had been generated (geometric mean titers of 200 in the high-dose i.d. and 126 in the high-dose i.m. group) (Fig. 5D) and neutralizing antibody to 89.6 had started to decline. By 16 to 20 weeks after challenge, antibodies to Gag and Env had fallen in most animals.<br>
Our results demonstrate that a multiprotein DNA/MVA vaccine can raise a memory immune response capable of controlling a highly virulent mucosal immunodeficiency virus challenge. Our levels of viral control were more favorable than have been achieved using only DNA (M.A. Egan et al. 2000 ./ Virol 74:7485) or rMVA vaccines (I. Ourmanov et al<br>
2000 J Virol 74:2740) and were comparable to those obtained for DNA immunizations adjuvanted with interleukin-2 (D.H. Barouch et al 2000 Science 290:486). All of these previous studies have used more than three vaccine inoculations, none have used mucosal challenges, and most have challenged at peak effector responses and not allowed a prolonged post vaccination period to test for "long term" efficacy.<br>
The dose of DNA had statistically significant effects on both cellular and humoral responses (P
The DNA/MVA vaccine controlled the infection, rapidly reducing viral loads to near or below 1000 copies of viral RNA per milliliter of blood. Containment, rather than prevention of infection, affords the opportunity to establish a chronic infection (H.L. Robinson et al. 1999 Nat Med 5:526). By rapidly reducing viral loads, a multiprotein DNA/MVA vaccine will extend the prospect for long-term non-progression and limit HIV transmission. (J.W. Meilors et al. 1996 Science 272:1167; T.C. Quinn et al 2000 N EnglJ Med 342:921).<br>
EXAMPLE 2 MVA Expressing Modified HIV Env, Gag, and Pol Genes<br>
This disclosure describes the construction of a modified vaccinia Ankara (MVA) recombinant virus, MVA/HIV clade B recombinant virus expressing the HIV strain ADA env and the HXB2 gag pol (MVA/HIV ADA env + HXB2 gag pol). For amplification, the lab name of MVA/HIV 48 will be used, which denotes the plasmid from which the construct comes.<br>
The HIV gag-pol genes were derived from the Clade B infectious HXB2 virus. The gag-pol gene was truncated so that most of the integrase coding sequences were removed and amino acids 185, 266, and 478 were mutated to inactivate reverse transcriptase, inhibit strand transfer activity, and inhibit the RNaseH activity, respectively. The Clade B CCR5 tropic envelope gene was derived from the primary ADA isolate; TTTTTNT sequences were mutated without changing coding capacity to prevent premature transcription<br>
termination and the cytoplasmic tail was truncated in order to improve surface expression, immunogenicity, and stability of the MVA vector. The HIV genes were inserted into a plasmid transfer vector so that gag-pol gene was regulated by the modified H5 early/late vaccinia virus promoter and the env gene was regulated by the newly designed early/late Psyn n promoter to provide similar high levels of expression. A self-deleting GUS reporter gene was included to allow detection and isolation of the recombinant virus. The HIV genes were flanked by MVA sequences to allow homologous recombination into the deletion 3 site so that the recombinant MVA would remain TK positive for stability and high expression in resting cells. The recombinant MVA was isolated and shown to express abundant amounts of gag-pol-env and to process gag. Production of HIV-like particles was demonstrated by centrifugation and by electron microscopy. The presence of env in the HTV-like particles was demonstrated by immunoelectron microscopy.<br>
Table of Sequences<br><br>
(Table Removed)<br>
Plasmid Transfer Vector<br>
The plasmid transfer vector used to make the MVA recombinant virus, pLW-48, (Figure C) by homologous recombination was constructed as follows:<br>
1. From the commercially obtained plasmid, pGem-4Z (Promega), flanking areas on either side of deletion III, designated flank 1 and flank 2, containing 926 and 520 base pairs respectively, were amplified by PCR from the MVA stains of vaccinia virus. Within these flanks, a promoter, the mH5, which had been modified from the originally published sequence by changing two bases that had been shown by previously published work to increase the expression of the cloned gene, was added.<br>
2.	A clade B gag pol (Figure D) was truncated so that the integrase was removed<br>
and was cloned into the plasmid so that it was controlled by the mH5 promoter. This gene<br>
contained the complete HXB2 sequence of the gag. The pol gene has reverse transcriptase<br>
safety mutations in amino acid 185 within the active site of RT, in amino acid 266 which<br>
inhibits strand transfer activity, and at amino acid 478 which inhibits the RNaseH activity.<br>
In addition, the integrase gene was deleted past EcoRI site.<br>
3.	A direct repeat of 280 basepairs, corresponding to the last 280 base pairs of MVA<br>
flank 1, was added after flank 1.<br>
4.	The pll promoter and GUS reporter gene were added between the two direct<br>
repeats of flank 1 so that this screening marker could initially be used for obtaining the<br>
recombinant virus, yet deleted out in the final recombinant virus (Scheiflinger, F. et al.<br>
1998 Arch Virol 143:467-474; Carroll, M.W. and B. Moss 1995 BioTechniques 19:352-<br>
355).<br>
5.	A new promoter, Psyn II, was designed to allow for increased expression of the<br>
ADA env. The sequence of this new early/late promoter is given in Figure E.<br>
6.	A truncated version of the ADA envelope with a silent 5TNT mutation was<br>
obtained by PCR and inserted in the plasmid under the control of the Psyn n promoter.<br>
The envelope was truncated in the cytoplasmic tail of the gp41 gene, deleting 115 amino<br>
acids of the cytoplasmic tail.    This truncation was shown to increase the amount of<br>
envelope protein on the surface of infected cells and enhance immunogenicity of the<br>
envelope protein in mice, and stability of the recombinant virus in tissue culture.<br>
Recombinant MVA Construction<br><br>
1.	MVA virus, which may be obtained from ATCC Number VR-1508, was plaque<br>
purified three times by terminal dilutions in chicken embryo fibroblasts (CEF),which were<br>
made from 9 day old SPF Premium SPAFAS fertile chicken eggs, distributed by B and E<br>
Eggs, Stevens, PA.<br>
2.	Secondary CEF cells were infected at an MOI of 0.05 of MVA and transfected<br>
with 2µg of pLW-48, the plasmid described above.   Following a two day incubation at<br>
37°C, the virus was harvested, frozen and thawed 3x, and plated out on CEF plates.<br>
3.	At 4 days, those foci of infection that stained blue after addition of X-gluc<br>
substrate, indicating that recombination had occurred between the plasmid and the infecting<br>
virus, were picked and inoculated on CEF plates. Again, those foci that stained blue were picked.<br>
4.	These GUS containing foci were plated out in triplicate and analyzed for GUS<br>
staining (which we wanted to now delete) and ADA envelope expression. Individual foci<br>
were picked from the 3rd replicate plates of those samples that had about equal numbers of<br>
mixed populations of GUS staining and nonstaining foci as well as mostly envelope<br>
staining foci.<br>
5.	These foci were again plated out in triplicate, and analyzed the same way. After<br>
5 passages, a virus was derived which expressed the envelope protein but which had deleted<br>
the GUS gene because of the double repeat. By immunostaining, this virus also expressed<br>
the gag pol protein.<br>
Characterization of MVA Recombinant Virus. MVA/HIV 48<br>
1.	Aliquots   of   MVA/HIV   48   infected   cell   lysates   were   analyzed   by<br>
radioimmunoprecipitation and immunostaining with monoclonal antibodies for expression<br>
of both the envelope and gag pol protein. In both of these tests, each of these proteins was<br>
detected.<br>
2.	The recombinant virus was shown to produce gag particles in the supernatant of<br>
infected cells by pelleting the 35S-labeled particles on a 20% sucrose cushion.<br>
3.	Gag particles were also visualized both outside and budding from cells as well as<br>
within vacuoles of cells in the electron microscope in thin sections. These gag particles had<br>
envelope protein on their surface.<br>
Unless otherwise indicated, all nucleotide sequences determined by sequencing a DNA molecule herein were determined using an automated DNA sequencer, and all amino acid sequences of polypeptides encoded by DNA molecules determined herein were predicted by translation of a DNA sequence determined as above. Therefore, as is known in the art for any DNA sequence determined by this automated approach, any nucleotide sequence determined herein may contain some errors. Nucleotide sequences determined by automation are typically at least about 90% identical, more typically at least about 95% to at least about 99.9% identical to the actual nucleotide sequence of the sequenced DNA molecule. The actual sequence can be more precisely determined by other approaches including manual DNA sequencing methods well known in the art. As is also known in the art, a single insertion or deletion in a determined nucleotide sequence compared to the actual sequence will cause a frame shift in translation of the nucleotide sequence such that the predicted amino acid sequence encoded by a determined nucleotide sequence will be completely different from the amino acid sequence actually encoded by the sequenced DNA molecule, beginning at the point of such an insertion or deletion. Summary<br>
In summary, we have made a recombinant MVA virus, MVA/HIV 48, which has high expression of the ADA truncated envelope aud the HXB2 gag pol. The MVA recombinant virus is made using a transiently expressed GUS marker that is deleted in the final virus. High expression of the ADA envelope is possible because of a new hybrid early/late promoter, Psyn II. In addition, the envelope has been truncated because we have shown truncation of the envelope enhances the amount of protein on the surface of the infected cells, and hence enhances immunogenicity; stability of the recombinant is also enhanced. The MVA recombinant makes gag particles which has been shown by pelleting the particles through sucrose and analyzing by PAGE. Gag particles with envelope protein on the surface have also been visualized in the electron microscope.<br>
EXAMPLE 3<br>
Additional Modified or Synthetic Promoters Designed for Gene Expression in MVA Or<br>
Other Poxviruses<br>
Additional modified or synthetic promoters were designed for gene expression in MVA or other poxviruses. Promoters were modified to allow expression at early and late times after infection and to reduce possibility of homologous recombination between identical sequences when multiple promoters are used in same MVA vector. Promoters are placed upstream of protein coding sequence.<br>
m7.5 promoter (SEQ ID NO: 10): CGCTTTTTATAGTAAGTTTTTCACCCATAAATAATAAATACAATAATTAATTTCT<br>
CGTAAAAATTGAAAAACTATTCTAATTTATTGCACGGT<br>
Psyn II promoter (SEQ ID NO:2): TAAAAAATGAAAAAATATTCTAATTTATAGGACGGTTTTGATTTTCTTTTTTTCT<br>
ATGCTATAAATAATAAATA<br>
Psyn III promoter (SEQ ID NO:11):<br>
TAAAAATTGAAAAAATATTCTAATTTATAGGACGGTTTGATTTCTTTTTTTCT ATACTATAAATAATAAATA<br>
Psyn IV promoter (SEQ ID NO: 12):<br>
TAAAAATTGAAAAACTATTCTAATTTATAGGACGGTTTTGATTTTCTTTTTTTCT ATACTATAAATAATAAATA<br>
PsynV promoter (SEQ ID NO: 13):<br>
AAAAAATGATAAAGTAGGTTCAGTTTTATTGCTGGTTTAAAATCACGCTTTCGA GTAAAAACTACGAATATAAAT<br>
EXAMPLE 4 Tables A-F<br>
TableA: MVA/48 immunization — guinea pigs.<br>
Groups of guinea pigs were immunized at days 0 and 30 with 1 xlO8 infectious units of MVA/48 by either the intramuscular (IM) or intradermal (ID) route. As a control another group was immunized IM with the same dose of non-recombinant MVA. Sera taken before as well as after each immunization was analyzed for neutralizing activity against HIV-l-MN. Titers are the reciprocal serum dilution at which 50% of MT-2 cells were protected from virus-induced killing. Significant neutralizing activity was observed in all animals after the second immunization with MVA/48 (day 49).<br>
Table B: Frequencies of HIV-1 gag-specific T cells following immunization of mice with MVA/48.<br>
Groups of BalbC mice were immunized at days 0 and 21 with 1 xlO7 infectious units of MVA/48 by one of three routes: intraperitoneal (IP), intradermal (ID), or intramuscular (IM). A control group was immunized with non-recombinant MVA. At 5 weeks after the last immunization, splenocytes were prepared and stimulated in vitro with an immunodominant peptide from HIV-1 p24 for 7 days. The cells were then mixed either with peptide-pulsed P815 cells or with soluble peptide. Gamma interferon-producing cells were enumerated in an ELISPOT assay. A value of &gt;500 was assigned to wells containing too many spots to count. Strong T cell responses have been reported in mice immunized IP<br>
with other viruses. In this experiment, IP immunization of mice with MVA/48 elicited very strong HIV-1 gag-specific T cell responses.<br>
Table C: DNA prime and MVA/48 boost - total ELISPOTS per animal.<br>
Ten rhesus macaques were primed (weeks 0 and 8) with a DNA vaccine expressing HIV-1 antigens including Ada envelope and HXB2 gagpol. At week 24 the animals were boosted intramuscularly with 1 xlO8 infectious units of MVA/48. Fresh peripheral blood mononuclear cells (PBMC) were analyzed for production of gamma interferon in an ELISPOT assay as follows: PBMC were incubated for 30-36 hours in the presence of pools of overlapping peptides corresponding to the individual HIV-1 antigens in the vaccines. The total number of gamma interferon-producing cells from each animal is shown in the table. T cell responses to DNA vaccination were limited (weeks 2-20). However, boosting with MVA/48 resulted in very strong HW-1-specific T cell responses in all animals (week 25).<br>
Table D: Antibody response following immunization of macaques with MVA/SHIV KB9.<br>
Groups of rhesus macaques were immunized with 2 xlO8 infectious units of MVA/SHIV-KB9 at weeks 0 and 4 by one of several routes: Tonsilar, intradermal (ID), or intramuscular (EM). Another group was immunized with non-recombinant MVA using the same routes. Serum samples from 2 weeks after the second immunization were analyzed for binding to KB9 envelope protein by ELISA and for neutralization of SHIV-89.6P and SHIV-89.6. hi the ELISA assay, soluble KB9 envelope protein was captured in 96 well plates using an antibody to the C-terminus of gp120. Serial dilutions of sera were analyzed and used to determine the endpoint titers. Neutralization of SHIV-89.6P and SHIV-89.6 was determined in an MT-2 cell assay. Titers are the reciprocal serum dilution at which 50% of the cells were protected from virus-induced killing. In in vitro neutralization assays, SHIV-89.6P and SHIV-89.6 are heterologous, i.e. sera from animals infected with one of the viruses does not neutralize the other virus. Thus, two immunizations with MVA/SHIV-KB9 elicited good ELISA binding antibodies in all animals and neutralizing antibodies to the homologous virus (SHIV-89.6P) in some animals. In addition, heterologous neutralizing antibodies were observed in a subset of animals.<br>
Table E: Frequencies of gag CM-9-specific CD3/CD8 T cells following immunization of macaques with MVA/SHIV-KB9.<br>
Groups of MamuA*01 positive rhesus macaques were immunized with 2 xlO8 infectious units of MVA/SHTV-KB9 at weeks 0 and 4 by one of several routes: tonsilar, intradermal (ID), or intramuscular IM). Another group was immunized with non-recombinant MVA. The frequencies of CD3+/CD8+ T cells that bound tetrameric complex containing the SIV gag-specific peptide CM9 were determined by flow cytometry at various times after each immunization. Time intervals were as follows: la, Ib, and Id were one, two, and four weeks after the first immunization, respectively; 2a, 2b, 2c, and 2d were one, two, three, and twelve weeks after the second immunization, respectively. Values above background are shown in bold face. Strong SIV gag-specific responses were observed after a single immunization with MVA/SHIV-KB9 in all immunized animals. Boosting was observed in most animals following the second immunization. In addition, measurable tetramer binding was still found twelve weeks after the second immunization.<br>
Table F: Frequencies of specific T cells following immunization of macaques with MVA/SHIV KB9.<br>
Groups of macaques were immunized with MVA/SHIV-KB9 as described above. MVA/SHIV-KB9 expresses 5 genes from the chimeric virus, SHIV-89.6P: envelope, gag, polymerase, tat, and nef. Thus, the frequencies of T cells specific for each of the 5 antigens was analyzed using pools of peptides corresponding to each individual protein. Fresh PBMC were stimulated with pools of peptides for 30-36 hours in vitro. Gamma interferon-producing cells were enumerated in an ELISPOT assay. The total number of cells specific for each antigen is given as "total # spots". In addition, the number of responding animals and average # of spots per group is shown. PBMC were analyzed at one week after the first immunization (la) and one week after the second immunization (2a). Another group of 7 animals was immunized with non-recombinant MVA, In these animals, no spots above background levels were detected. Thus, a single immunization with MVA/SHIV-KB9 elicited strong SHIV-specific T cell responses in all animals. Gag and envelope responses were the strongest; most animals had responses to gag, all animals had responses to envelope. The Elispot responses were also observed after the second immunization with MVA/SHIV-KB9, albeit at lower levels. At both times, the rank order of responses was: tonsilar &gt; ID &gt; IM. We show good immune response to nef and some immune response to tat.<br>
TABLE A<br>
MVA/48 immunization - guinea pigs HIV-MN neutralizing antibody - reciprocal titer<br>
(Table Removed)<br>
TABLE B<br>
Frequencies of HTV-gag-specific T cells following immunization of mice with MVA/48<br>
(Table Removed)	<br><br>
TABLE C<br>
DNA prime and MVA/48 boost Total ELISPOTS per Animal<br>
(Table Removed)<br>
DNA primes were at 0 and 8 weeks and MVA/48 boost was at 24 weeksBackground (2x the number of ELISPOTs in the unstimulated control +10} stimulatory antibodies were added to the ELISPOT incubations * Animals from this bleed date exhibited higher than usual ELISPOTs,<br>
TABLE D<br>
Antibody response following immunization of macaques with MVA/SHIV KB9<br>
(Table Removed)<br><br>
TABLE E<br>
Frequencies of gag CM9-speciflc CD3/CD8 T cells following immunization of macaques witb<br><br>
MVA/SHIVKB9<br>
(Table Removed)<br><br><br>
TABLE F<br>
Frequencies of specific T cells folloning immunization of macaques with MVA/SHTV KB9<br>
(Table Removed)<br>
While the present invention has been described in some detail for purposes of clarity and understanding, one skilled in the art will appreciate that various changes in form and detail can be made without departing from the true scope of the invention. All patents, patent applications and publications referred to above are hereby incorporated by reference.<br><br>
We Claim:<br>
1.	A pharmaceutical composition comprising:<br>
(a)	a recombinant MVA virus encoding   HIV Gag,    and a gp120<br>
protein and the membrane-spanning domain and ectodomain of gp41 protein of<br>
HIV Env, but lacking all or part of the cytoplasmic domain of gp41 protein, such<br>
as herein described, wherein HIV Pol lacks the integrase domain; and<br>
(b)	a pharmaceutically acceptable carrier such as herein described.<br><br>
2.	The composition of claim  1, wherein the HIV Pol encoded by the<br>
recombinant MVA virus has an amino acid change that inhibits reverse transcriptase<br>
activity.<br>
3.	The composition of claim  1, wherein the HIV Pol encoded by the<br>
recombinant MVA virus has an amino acid change that inhibits strand transfer activity.<br>
4.	The composition of claim  1, wherein the HIV Pol encoded by the<br>
recombinant MVA virus has an amino acid change that inhibits RNaseH activity.<br>
5.	The composition of claim 1, wherein recombinant MVA virus encodes<br>
gp41 protein lacking at least 115 amino acids of the cytoplasmic domain of gp41.<br>
6.	The pharmaceutical composition of claim 1, wherein the HIV pol is from<br>
clade A, clade B, Clade C, clade D, clade E, clade F, clade G, clade H or clade J.<br>
7.	The   pharmaceutical   composition   of  claim   1,   wherein   the   MVA<br>
additionally encodes at least one HIV protein selected from the group consisting of: vif,<br>
vpr, tat, rev, vpu and nef.<br>
8. The pharmaceutical composition of claim 1, wherein the MVA encodes the HXB2 Gag/Pol of MVA 48 (SEQ ID NO:5) and the truncated ADA envelope of MVA 48 (SEQ ID N0:3).<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUNsYWltcy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Claims-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Correspondence-Others-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDgtMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Correspondence-Others-(08-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjgtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Correspondence-Others-(28-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDA4LTA0LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Description (Complete)-(08-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUZvcm0tMS0oMDgtMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Form-1-(08-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUZvcm0tMi0oMDgtMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Form-2-(08-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLUZvcm0tMjctKDI4LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Form-27-(28-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLU90aGVycy1Eb2N1bWVudC0oMDgtMDQtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-Others-Document-(08-04-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1kZWxucC0yMDAzLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1509-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTUwOS1ERUxOUC0yMDAzLVBDVC00MDktKDA4LTA0LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1509-DELNP-2003-PCT-409-(08-04-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="234834-a-process-for-the-preparation-of-a-natural-antioxidant-fraction-from-fruits-of-the-plant-cinnamomum-zeylanicum.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234836-an-aqueous-polyurethane-dispersion.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234835</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1509/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Sep-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY, DEPARTMENT OF HEALTH AND HUMAN SERVICES</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NATIONAL INSTITUTE OF HEALTH, OFFICE OF TECHNOLOGY TRANSFER, 6011 EXECUTIVE BOLUEVARD,SUITE 325, ROCKVILLE, MD 20852-3804 (US)</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MOSS, BERNARD</td>
											<td>10301 DICKENS AVENUE, BETHESDA, MD 20814, (US)</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WYATT, LINDA</td>
											<td>11201 HUNTOVER DRIVE, ROCKVILLE, MD 20852 (US)</td>
										</tr>
										<tr>
											<td>3</td>
											<td>EARL, PATRICIA</td>
											<td>3520 WOODBINE STREET, CHEVY CHASE, MD 20815 (US)</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ROBINSON HARRIET L.</td>
											<td>1051 MASON WOODS DRIVE, ATLANTA, GA 30329, U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/06713</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-03-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/274,434</td>
									<td>2001-03-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234835-a-pharmaceutical-composition-comprising-a-recombinant-mva-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:03 GMT -->
</html>
