<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/234902-oligonucleotide-primers-for-use-in-detecting-the-presence-of-genetic-markers-in-a-bovine-subject by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:30:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 234902:&quot;OLIGONUCLEOTIDE PRIMERS FOR USE IN DETECTING THE PRESENCE OF GENETIC MARKERS IN A BOVINE SUBJECT&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;OLIGONUCLEOTIDE PRIMERS FOR USE IN DETECTING THE PRESENCE OF GENETIC MARKERS IN A BOVINE SUBJECT&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to oligonucleotide phmers for use in detecting the presence in a bovine subject of at least one genetic marker associated with bovine complex vertebral malformation (CVM). Genetic markets for identifying bovine carriers of complex vertebral malformation (CVM) disease gene are described. The genetic -markers, including the microsatellite markers BM4129, INRAA003, BMS2790, ILSTS029, INRA123, BM220, HUJ246, BMS862, BMS937, BL1048, BMS2095 and BMS1266 and the bovine SLC35A3 gene, are. located on bovine chromosome BTA3. The G/T polymorphism at position 559 of the bovine SLC35A3 gene is identified asbeing causative and diagnostic tor CVM in cattle.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The present invention relates to oligonucleotide primers for use in detecting the presence in a bovine subject of at least one genetic marker associated with bovine complex vertebral malformation. This invention relates generally to a genetic disease observed in bovines termed Complex Vertebral Malformation {CVM). More particularly, the invention relates to molecular markers for identirying potential bovine carriers of CVM and for identifying the CVM pene locus and mutations thereof responsible for complex vertebral malformation in Dovines,<br>
Background of the inventiori<br>
Complex: Vertebral Malformation (CVM) is 6 congenital vertebral disorder detected in Holstein-Frlesian (HF) black ano white dalP;' cattle. The disease has recently been described (Agerholm et a!., 20DO). In Denmarl:, all cases diaonosed until today (Octooer 17, 2000) have been genetically related to the former elite US Holstein bull Carlin-M vanhoe Bell According to the present date, CVM appears tc be inherited as ar. autosomBl recessive disease.<br>
The disease is ciiaracterised by e congentta^l Pilatersl symmetric arthrogryposis of the distal joints and malformations of the columnB, mainly at the cervico-thoracic junction comPined with, reduced body weight (Agerholm et al., 1994),<br>
£&gt;:temally, there are the following major flndlnns: In many cases the cervical and/or the thoracic part of the coiumna seems to be short, Moderate bilateral symmetric contraction of the carpal ioints and severe contrsction and supination of the phatangD-metacarpa! joint (fetlocic) are constant findings. Contractiori and pronation of the phalBngD-metatarsal joint and slight extension of the tarsus are also common findings, In most cases an Irregular course of the columns around the cervlco-thDrBCic junction is observed, Scoliosis may be observed, and lesions ma&gt;' be present in other regions of the columna. The irreputar course is often recognised by inspection and palpation of the ventral aspect of the co-inmna. However, lesions may be minima! and restricted to two or tew vertebrae. In such cases the columns may be of almost normal length. Therefore, radiological examination of the columna is recommended to exclude vertebral matformations in suspected cases. Tne spinal cord is of normal size lying v\rith the vertebral canal without obvious compressions. Using radiology, complex vertebral malformations consisting of hemivertebrae, fused and matshaped vertebrae, scoliosis, and anchylosis are found at varying degrees. This is best<br><br>
demonstrated following removal of the arcus vertebrae. In some cases malformations of<br>
the heart are present, mostly as a high interventricular septal defect and eccentric hypertrophy<br>
of the right ventricle. Malformations of the large vessels may occur. In the lungs<br>
fetal atelectasis is present. Serohemorrhagic fluids are most present in the thoracic cavity.<br>
A variety of other malformations have been observed, but these are not constant or common<br>
findings. Lesions due to dystocia are often found.<br>
Malformations have been observed both in aborted fetuses, prematurely born calves and in<br>
stillborn calves born at term. Cases among older calves have not yet been observed. In<br>
general the body weight is reduced, and the body weight is lower In premature born calves<br>
than in calves born at term.<br>
Additionally, there seems to be an increased frequency of abortions in cows inseminated<br>
with semen of carrier bulls. At present the cause of this is unknown.<br><br>
Presently, the only tool available for CVM diagnosis is patho-anatomical diagnosis based on<br>
the above described presence of bilateral symmetric arthrogryposis of the distal joints and<br>
malformations of the columna, mainly at the cervico-thoracic junction combined with<br>
reduced body weight. However, symmetric contractions of the limbs are common and<br>
general findings in vertebral malformations in calves. Therefore, differential diagnostic<br>
problems do exist as it is often difficult to differentiate between CVM and other malformations.<br>
The fact that the genetic defect appears to be spread by the bull Carlin-M Ivanhoe Bell<br>
which has been used intensively all over the world makes It of significant economic importance<br>
to be able to test whether current and potential breeding bulls are carriers of the<br>
defect.<br>
In order to obtain an estimate of the frequency of potential CVM carrier animals within the<br>
Danish cattle population, the present inventors have extracted pedigree information from<br>
the Danish national cattle database. At the time of the extraction (October 2000) there<br>
were registered 919,916 pure-bred cows and heifers, and 169,821 pure-bred bulls and<br>
male calves. Bell wris found 707,915 times in the pedigrees of the cows and heifers and<br>
1«1,043 times in the male pedigrees. In Tables 1 and 2 below, the number of occurrences<br>
of Bell in each generation of the pedigrees is shown.<br>
(TABLE Removed) <br>
Although these numbers also include some double and triple occurrences of Bell in the<br>
pedigrees, the data clearly show that a majority of the Danish Holstein cattle are potential<br>
carriers of CVM. Clearly, the problem is immense on a global scale.<br>
Thus, there is great demand In the cattle Industry for a genetic test that permits the identification<br>
of cattle In various breeds that are potential carriers of CVM (e.g. before detectable<br>
onset of clinical symptoms).<br>
Prior to the present invention, mlcrosatelllte mapping has not been applied to the gene<br>
causing the above complex vertebral malformations which has not been Isolated or<br>
characterised. Thus, to the inventors' best knowledge, the diagnostic method according to<br>
the invention described in further detail In the following has not previously been suggested<br>
OK disclosed.<br>
Accordingly, the present invention, which comprises mapping of the disease locus for CVM,<br>
has provided a DNA test based on microsatellite markers located on bovine chromosome<br>
BTA3. The ability of the test to define the carrier status of animals descending from Bell<br>
has been confirmed which appears from the examples below.<br>
Summary of the invention<br>
In its broadest aspect, the present invention provides a method for detecting the presence<br>
in a bovine subject of a genetic marker associated with bovine complex vertebral malformation<br>
(CVM), comprising the steps of provic ing a bovine genetic material, and detecting<br>
in the genetic material the presence or absence of at least one genetic marker that is<br>
linked to a bovine complex vertebral malformation disease trait or a specific nucleotide<br>
polymorphism which causes the complex vertebral malformation disease trait.<br>
In a further aspect, the invention pertains to a diagnostic kit for use in detecting the presence<br>
in a bovine subject of at least one genetic marker associated with bovine complex<br>
vertebral malformation (CVM), comprising at least one oligonucleotide sequence selected<br>
from the group consisting of SEQ ID NO:1, SEQ ID NO:2, SEQ ID NO:3, SEQ ID NO:4, SEQ<br>
ID N0:5, SEQ ID NO:6, SEQ ID NO:7, SEQ ID NO:8, SEQ ID NO:9, SEQ ID NO:10, SEQ ID<br>
NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO: 15 , SEQ ID NO:16,<br>
SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37 and SEQ<br>
ID NO: 38, and combinations thereof. Furthermore, the invention relates to a diagnostic<br>
method including a diagnostic kit for the detection of a G/T polymorphism in the bovine<br>
SLC35A3 gene causative and diagnostic for CVM in cattle.<br>
Detailed disclosure of the invention<br>
One primary objective of the present invention is to enable the identification of cattle carrying<br>
bovine complex vertebral malformation (CVM). This is achieved by a method which<br>
detects the presence of a .genetic marker associated with bovine CMV in a bovine subject.<br>
More specifically, the genetic marker may be the bovine SLC35A3 gene or even more<br>
specifically specific polymorphisms in the bovine SLC35A3 gene.<br>
As used herein, the term a "bovine subject" refers to cattle of any breed. Thus, any of the<br>
various cow or ox species, whether male or female, are Included In the term, and both<br>
adult and new-born animals are Intended to be covered. The term does not denote a particular<br>
age. One example of a bovine subject Is a member of the Holstein-Friesian cattle<br>
population.<br>
The term "genetic marker" refers to a variable nucleotide sequence (polymorphic) that is<br>
present in bovine genomic DNA on a chromosome and which Is identifiable with specific<br>
oligonucleotides. Such a variable nucleotide sequence Is e.g. distinguishable by nucleic acid<br>
amplification and observation of a difference in size or sequence of nucleotides due to the<br>
polymorphism. In useful embodiments, such genetic markers may be identified by several<br>
techniques known to those skilled in the art, and include typing of microsatellites or short<br>
tandem repeats (STR), restriction fragment length polymorphisms (RFLP), detection of<br>
deletion or insertion sites, and random amplified polymorphic DNA (RAPD) as well as the<br>
typing of single nucleotide polymorphism (SNP) by methods including restriction-fragmentlength<br>
polymerase chain reaction, aliele-specific oligomer hybridisation, oligomer-specific<br>
ligation assays, mini-sequencing, direct sequencing, fluorescence-detected 5'-exonuclease<br>
assays, and hybridisation with PNA and LNA probes and others. However, it will be<br>
appreciated that other genetic markers and techniques may be applied in accordance with<br>
the invention.<br>
As described above, "bovine complex vertebral malformation" (CVM) is a congenital vertebral<br>
disorder. Presently, the disease has only been detected in Holstein-Friesian (HF)<br>
black and white dairy cattle; however, it is also contemplated that other bovine races may<br>
be affected. The disease has recently been described by Agerholm et al., 2000. Accordingly,<br>
in the present context CVM and bovine complex vertebral malformation disease<br>
trait are to be understood as a disease resulting in the clinical symptoms previously<br>
described herein, and as reported and defined by Agerholm et al., 2000.<br>
The method according to the invention includes the provision of a bovine genetic material.<br>
Such material include bovine DNA material which may be provided by any conventional<br>
method or means. The bovine DNA material may e.g. be extracted, isolated and purified<br>
from blood (e.g., fresh or frozen), tissue samples (e.g., spleen, buccal smears), hair samples<br>
containing follicular cells and semen.<br>
As previously described, the method of the present invention further comprises a step of<br>
detecting in the genetic material the presence or absence of a genetic marker that is linked<br>
to a bovine complex vertebral malformation disease trait.<br>
In order to detect if the genetic marker is present in the genetic material, standard<br>
methods well known to persons skilled in the art may be applied, e.g. by the use of nucleic<br>
acid amplification. In order to determine if the genetic marker is genetically linked to the<br>
complex vertebral malformation disease trait, a lod score can be applied. A lod score,<br>
which is also sometimes referred to as Zmu, Indicates the probability (the logarithm of the<br>
ratio of the likelihood) that a genetic marker locus and a specific gene locus are linked at a<br>
fiarticular distance. Lod scores may e.g. be calculated by applying a computer programme<br>
such as the MLINK programme of the LINKAGE package (Lathrop et al., 1985). A lod score<br>
of greater than 3.0 is considered to be significant evidence for linkage between the genetic<br>
marker and the complex vertebral malformation disease trait or gene locus.<br>
In one embodiment of the invention, the genetic marker is located on bovine chromosome<br>
BTA3. The region of bovine chromosome BTA3 comprising the genetic markers that are<br>
useful in the method of the present invention is indicated in Figure 2.<br>
Accordingly, genetic markers located on bovine chromosome BTA3 in the region flanked by<br>
and including the polymorphic microsatellite markers BM4129 and BMS1266, may be<br>
useful according to the present invention. In one specific embodiment, the at least one genetic<br>
marker is located in the region from about 59.5 cM to about 67.9 cM on bovine<br>
chromosome BTA3.<br>
In a further useful embodiment, the at least one genetic marker is located on the bovine<br>
chromosome BTA3 in the region flanked by and Including the polymorphic microsatellite<br>
markers INRM003 and BMS937.<br>
In a further aspect, the at least one genetic marker is located on the jbovine chromosome<br>
BTA3 in the region flanked by and including the polymorphic microsatellite markers<br>
INRAA003 and ILSTS029.<br>
In another advantageous embodiment, the at least one genetic marker Is selected from<br>
the group consisting of microsatellite markers BM4129, INRAA003, BMS2790, ILSTS029,<br>
INRA123, BM220, HUJ246, BMS862, BMS937, BL1048, BMS2095 and BMS1266.<br>
As described in the examples, the at least one genetic marker may be linked to a gene<br>
causing the bovine complex vertebral malformation disease. Thus, in one embodiment, the<br>
at least one genetic marker is located on bovine chromosome BTA3 in the region flanked<br>
by and including the polymorphic mtcrosatellite markers BM4129 and BMS1266 and<br>
genetically linked to the CVM disease trait or the CVM gene locus at a lod score of at least<br>
3.0, such as at least 4.0, including at least 5.0, such as at least 6.0, including at least 7.0<br>
such as at least 8.0, including at least 9.0 such as at least 10.0, including at least 11.0,<br>
such as at least 12.0.<br>
The specific definition and locus of the above polymorphic microsatellite markers can be<br>
found in the USDA genetic map (Kappes et al., 1997).<br>
«.<br>
It will be appreciated that in order to detect the presence or absence in a bovine subject of<br>
a genetic marker associated with CVM, more than one genetic marker may be applied in<br>
accordance with the invention. Thus, the at least one marker can be a combination of two<br>
or more genetic markers which are shown to be informative whereby the accuracy of the<br>
test can be increased.<br>
Accordingly, as further exemplified below, in one useful embodiment, two or more of the<br>
microsatellite markers 1NRAA003, BMS2790, ILSTS029, INRA 123, BM220, HUJ246,<br>
BMS862, BMS937 can be used in combination.<br>
In accordance with the invention, the nucleotide sequences of the primer pairs for amplifying<br>
the above microsatellite markers are described in Table 4 below.<br>
The comparative maps (Solinas-Toldo et al., 1995) show that most of bovine chromosome<br>
BTA3 corresponds to a part of human chromosome 1 (HSA1). The genetic mapping of the<br>
CVM locus presented herein makes it possible to use the information available about human<br>
genes and to concentrate the search for the candidate gene to genes present on<br>
human chromosome 1. This will greatly limit the number of candidate genes and facilitate<br>
the search for the CVM causative gene.<br>
Genetic markers of the present invention can be made using different methodologies<br>
known to those skilled in the art. Thus, it will be understood that with the knowledge<br>
presented herein, the nucleotide sequences of the above described polymorphic<br>
microsatellite markers of bovine chromosome BTA3 have been identified as being<br>
genetically linked to the CVM gene locus, and additional markers may be generated from<br>
the known sequences or the indicated location op bovine chromosome BTA3 for use in the<br>
method of the present invention.<br>
For example, using the map illustrated in Figure 2, the CVM region of bovine chromosome<br>
BTA3 may be micro-dissected, and fragments cloned into vectors to isolate DNA segments<br>
which can be tested for linkage with the CVM gene locus. Alternatively, with the nucleotide<br>
sequences provided in Table 4, isolated DNA segments can be obtained from the CVAf<br>
region by nucleic add amplification (e.g., polymerase chain reaction) or by nudeotide<br>
sequencing of the relevant region of bovine chromosome BTA3 ("chromosome walking").<br>
Additionally, the above described homology between bovine chromosome BTA3 and human<br>
chromosome 1 (HSA1) Indicates that any gene or expressed sequence tag that is mapped<br>
to this analogous region in human may also map to the CVM region of bovine chromosome<br>
gTA3. Thus, genes or conserved sequences that map on human chromosome HSA1 may<br>
be analysed for linkage to the CVM gene locus using routine methods.<br>
Genotyping is based on the analysis of genomic DNA which can be provided by using<br>
standard DNA extraction methods as described herein. When the genomic DNA is isolated<br>
and purified, nucleic acid amplification (e.g. polymerase chain reaction) can be used to<br>
8<br>
amplify the region of the DMA corresponding to each genetic marker to be used in the<br>
analysis for detecting the presence in a bovine subject of a genetic marker associated with<br>
CVM. Accordingly, a diagnostic kit for use in such an embodiment comprises, in a separate<br>
packing, at least one oligonucleotide sequence selected from the group consisting of SEQ<br>
ID N0:l, SEQ ID NO:2, SEQ ID NO:3, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:6, SEQ ID<br>
N0:7, SEQ ID NO:8, SEQ ID NO:9, SEQ ID NO:1D, SEQ ID NO:11, SEQ ID NO:12, SEQ ID<br>
N0:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO: 33, SEQ ID NO: 34,<br>
SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37 and SEQ ID NO: 38, and combinations<br>
thereof.<br>
Identification of a mutation In the bovine SLC35A3 gene causative and diagnostic for CVM<br>
in cattle<br>
Having established the genomic localisation of the CVM gene delimited by polymorphic<br>
microsateilite markers, a search for the identification of the structural gene and the causative<br>
mutation herein was performed which can be used as an ultimate genetic marker for<br>
CMV.<br>
The human genome sharing sequence homology to the CVM region, defined as the region<br>
between the markers INRA003 and ILSTS029, was identified. The marker BMS2790 is<br>
located in this interval (Figure 1). Initially, 5 different clones from a bovine BAG library<br>
(RPCI-42, constructed and made available by P. de Jonge and co-workers) were identified,<br>
each harbouring one of the markers INRA003, ISLTS029, or BMS2790. Using sequence<br>
information obtained from these BAG clones, a single region on human chromosome 1<br>
which contained a regionJof homology to the marker-containing BACs was Identified using<br>
the BLASTN programme on public sequence databases. This region spans about 6 million<br>
base pairs and is located In position approx. 107.4-113.5 (Rgure 3) (ENSEMBL viewer, The<br>
Sanger Centre).<br>
Isolation and sequencing of the SLC35A3 cDNA<br>
Based on a homology alignment of the SLC35A3 gene between homo sapiens and canis<br>
familiaris, 2 oligosJSLIF and SL8R) were designed for amplification of almost the entire<br>
cDNA for bovine SLC35A3, Including the start codon. PCR was performed on cDNA Isolated<br>
from heart tissue samples collected from a wlldtype animal, a CVM carrier, and an affected<br>
animal, respectively. To obtain the sequence of the 3'-end of the gene, the resulting PCR<br>
fragment was sequenced and a new oligo designed (SL5F). To amplify the 3' end of<br>
SLC35A3, SL5F was used in combination with an ollgo (bSLCBVIR), designed using the<br>
published sequence of a partial bovine EST (genbank, dbEST). The cDNA sequence (SEQ<br>
ID NO: 18) and the translated peptide sequence (SEQ ID NO: 17) of bovine SLC35A3 is<br>
shown in Figure 4. The protein encoded by bovine SLC35A3 contains 326 amino acids and<br>
shares homology to a family of previously known proteins involved in the transport of<br>
nucleotide sugars from the cytosol into the Golgi lumen. The alignment depicted in Figure<br>
5 shows the homology to SLC35A3 proteins previously described in human (Ishida et al.<br>
1999) and in dog (Guillen et al. 1998).<br>
Detection of a polymorphism In the 5LC35A3 oer)e<br>
To detect potential polymorphisms in the bovine SLC35A3 gene, PCR amplification of the<br>
gene was performed using cDNA isolated from heart tissue samples collected from a CVM<br>
carrier and an affected animal, respectively. Sequencing of the fragment Isolated from the<br>
affected animal revealed a sequence identical to the wildtype, except for the affected<br>
animal being homozygous for the nucleotide T in nucleotide position 559, compared to the<br>
wildtype animal being homozygous for G in the corresponding position (see Figure 4).<br>
Sequencing of the cDNA from an animal being carrier of the CVM-defect showed this<br>
animal to be heterozygous having both T and G in position 559.<br>
The exchange of G to T in position 559 affects the sequence of the resulting peptide in<br>
changing a valine in position 180 to a phenylalanine (see Figure 4).<br>
Typing the SLC35A3 polymorphism by a DNA sequencing based assay<br>
Figure 6 shows the results obtained from sequencing a PCR fragment amplified from genomic<br>
DNA and containing the region (from 544 to 572 of the SLC35A3 cDNA, for numbering<br>
see Figure 4) containing the G/T mutation. The left and right panels show forward<br>
and reverse sequencing, respectively. The upper row (marked by -/-) shows the wildtype<br>
result, showing a G in the polymorphic position using forward sequencing and a C in the<br>
similar position on the other strand using reverse sequencing (marked by asterisks). The<br>
lower row +/+ shows the results from an affected calf, showing a T in the polymorphic<br>
position using forward sequencing and an A in the similar position on the other strand<br>
using reverse sequencing (marked by asterisks). The heterozygote (+/-) Is shown in the<br>
middle panel and expectedly displays as a mixture of the wildtype and affected signal and<br>
thus has both a T and a G signal using forward sequencing and an A and a C signal on the<br>
other strand.<br>
All, .calves affected.Js^CVM are homozygous for the T-allele<br>
Genotyping of 39 calves affected by CVM was performed by sequencing of a PCR product<br>
amplified from genomic DNA (see Figure 6 and example 6). All of these animals were<br>
homozygous for the T-allele, confirming the initial results.<br>
The T-allele is not found in animals unrelated to Bell<br>
Since calves affected by CVM have been reported only in pedigrees containing the widely<br>
used bull BELL, it was investigated whether the T-allele was present in animals unrelated<br>
to BELL. Taking advantage of the Danish Cattle Database, 496 animals of the Holstein<br>
breed without BELL in their pedigree were identified and sampled. Genotyping of these<br>
animals was performed by sequencing of a PCR product amplified from genomic DNA (see<br>
Figure 6 and example 6). None of these animals contained the T-allele, suggesting that<br>
this allele is found exclusively In the line of animals closely related to BELL.<br>
By sequencing, more than 326 unrelated (at least for the last three generations) animals<br>
of 12 different breed were also genotyped. All of these animals were homozygous for the<br>
wildtype allele (G-allele) again demonstrating the lack of the CVW-related allele (the T-allele)<br>
In the general cattle population.<br>
Typing the SLC35A3 polymorphism bv an allele-soedfic PCR assay fAS-PCR)<br>
In order to type the G/T polymorphism efficiently, an allele-specific PCR assay using<br>
BIOLASE Diamond DNA Polymerase from Bioline was developed. This polymerase requires<br>
a perfect match of the 3' end of the primer to the template, and a mismatch at this<br>
position will result in no (or very weak) amplification. In this way, it is possible to distinguish<br>
between wildtype, carrier or sick animals by identifying the presence or absence of<br>
allele-specific PCR products (Figure 7). The left part of Figure 7 shows the Allele-Spedfic<br>
PCR products of the coding strand. As expected, wild type animals show amplification with<br>
the G-speclfic primer but not with the T-specific primer. The carriers show amplification of<br>
both the G- and T-spedfic primers, while sick animals only show amplification of the Tspeclflc<br>
primer. The right part shows the Allele-Spedfic PCR products of the non-coding<br>
strand, and as expected, the patterns are the same as the coding strand. Wild type anlntels<br>
are homozygotic C, carriers are heterozygotic C/A, and sick animals are homozygotic<br>
A.<br>
From the above described results of using a positional candidate gene approach, a bovine<br>
gene was identified which is homologous to the human gene SLC35A3 encoding a UDP-Nacetylglucosamine<br>
transporter. Within this gene a G/T polymorphism was identified which<br>
alters the amino acid sequence of the protein. All affected calves analysed (39) are<br>
homozygous for the T-allele (T/T) and known carriers (108) are all heterozygous for the<br>
polymorphism (G/T). Analysis of more than 500 animals of the Holstein breed, chosen as<br>
being unrelated to Bell, failed to identify any animals carrying the T-allele. More than 1500<br>
animals were analysed having Bell In the pedigree without finding any unaffected animal<br>
being homorygous for the T-allele. Furthermore, more than 300 cattle selected from 12<br>
different breeds were analysed without detecting the T-allele in any of these animals.<br>
Taken together, the findings described in the present application demonstrate that the Tallele<br>
is present in a single copy in animals which are carriers of CVM and in two copies in<br>
animals affected by CVM. Detection of the T-allele In position 559 (numbering from Rgure<br>
4) is therefore diagnostic for CVM and ideal for detection of carriers of the CVM defect.<br>
As the G/T polymorphism has been identified as being causative for CMV, any genetic<br>
markers closely coupled to this polymorphism may be diagnostic for CMV in a bovine<br>
population. Accordingly, the present invention describes a method to identify bovine<br>
subjects either affected by CMV or carriers of CMV by determining the presence of the G/T<br>
polymorphism at position 559 of the bovine SLC35A3 gene, either Indirectly by analysing<br>
any genetic markers, such as microsatellites described herein, coupled to the bovine<br>
SLC35A3 gene or directly by analysing the sequence of the bovine SLC35A3 gene, e.g. as<br>
described above and in further details in the examples.<br>
Within the scope of the present invention is therefore a method for detecting and/or quantifying<br>
the presence of a genetic marker associated with bovine CVM In a bovine subject in<br>
order to be able to identify the CMV affected bovine subjects or carriers of CMV. The steps<br>
of the method comprises:<br>
a) providing a bovine genetic material, and<br>
b) detecting, in said genetic material, the presence or absence of at least one genetic<br>
marker that is linked to a bovine complex vertebral malformation disease trait.<br>
The at least one genetic marker is linked to a gene causing bovine CMV disease, said gene<br>
being identified herein to be the bovine SLC35A3 gene which encodes the bovine SLC35A3<br>
protein comprisingan amino acid sequence as shown in SEQ ID NO: 17.<br>
More specifically, the present invention relates to a method for detecting bovine CMV,<br>
.wherein the genetic marker is a single nucleotide polymorphism at a position equivalent to<br>
nucleotlde 559 of SEQ ID NO: 18, said single nucleotide polymorphism being a G/T polymorphism.<br>
The present application furthermore describes an efficient assay for the genotyping of the<br>
present polymorphism using allele-specific PCR. This is but one of a battery of methods for<br>
the typing of SNPs, other methods which could be employed include, but are not limited to,<br>
mini-sequencing, primer-extension, pyro-sequencing, PCR-RFLP, allele-specific rolling<br>
circle-amplification, primer-extension followed by MALDI-TOF mass-spectrometry as well<br>
as a range of other enzymatic and hybridisation-based methods.<br>
A phenotype resembling CVM has been demonstrated to exist in mice mutated in the gene<br>
lunatic fringe (Evrad et a)., 1998). Similar to calves affected by CVM, mice homozygous for<br>
a null mutation in lunatic fringe exhibit an altered somite segmentation and patterning,<br>
having a shortened body axis, vertebral- and rib-fusions and incompletely formed<br>
vertebrae (Evrad et al., 1998). Fringe seems to participate in the definition of boundary<br>
formation and somite patterning by modulating the activity of notch receptors (Klein and<br>
Arias, 1998; Moloney et al., 2000, Bruckner et al., 2000). The Notch-modulating activity<br>
seems to be mediated by an N-acetylglucosaminyl-transferase activity of Fringe, which In<br>
Golgi initiates the elongation of O-linked fucose residues attached to EGF-like sequence<br>
repeats of Notch (Moloney et al., 2000; Bruckner et al., 2000).<br>
Furthermore, as the bovine SLC35A3 gene is homologous to the human SLC35A3 gene, it<br>
is, with the information given herein, obvious to analyse the coding sequence of the<br>
human SLC35A3 gene for causative and diagnostic mutations when studying human developmental<br>
defects, especially Involving somite-segmentation and patterning.<br>
The effect of a mutation in the Golgi-located N-acetylglucosaminyl transporter (SLC35A3)<br>
affecting transport of N-acetylglucosamlne from the cytosol Into the lumen of Golgi would<br>
be expected to deprive the Fringe family of proteins for their substrate. This would affect<br>
the ability of Fringe to modulate Notch activity and thereby cause a segmentation defect<br>
like CVM. It therefore seems very plausible that the mutation in SLC35A3, apart from<br>
being diagnostic for CVM, is also the mutation causing this widespread genetic defect.<br>
The invention is described In further details in the following examples:<br>
Examples<br>
Examjjje J.<br>
Geneticjrtappjnq of Complex Vertebral Malformation (CVM)<br>
This example illustrates the localisation of the CVM gene locus to bovine chromosome<br>
BTA3. Additionally, this embodiment describes the identification of markers linked to the<br>
CVM gene locus, and thus the characterisation of the CVM region of bovine chromosome<br>
BTA3.<br>
In order to map the locus responsible for CVM, samples were obtained from animals participating<br>
In a breeding study. Briefly, approx. 300 cows and heifers descending from the<br>
bull T Burma and inseminated with semen from the bull KOL Nixon were selected for the<br>
breeding study. Thirteen affected calves were selected on the basis of the post mortem<br>
examination, as described in Agerholm et al., 2000. These 13 calves as well as their<br>
parents, in total 28 animals, were used in the initial genome scan. The calves were separated<br>
by 4 generations to their common ancestor, the purebred bull Carlin-M Ivanhoe Bell.<br>
The genome scan was conducted, covering all 29 autosomes, using a battery of microsatellite<br>
markers picked from the USDA genome map (Kappes et al., 1997). Markers were<br>
selected with pair-wise distances between 10 antl 20 cM. In areas of doubt due to low<br>
marker informativity, new markers were included and typed. A total of 194 markers were<br>
used. PCR reactions were performed in duplexes in a volume of 5 nl In an ABI 877 PCR<br>
robot (Applied Biosystems), containing 12 ng of genomic DMA, Ix PCR buffer, 0.4 U<br>
AmpliTaq Gold (Applied Biosystems), 20 pmol of each primer and 2.0 rnM MgCI2. All<br>
markers were run at the same touchdown PCR conditions: incubation at 94°C for 12<br>
minutes to activate the enzyme, 35 cycles at 94°C, 30 sec; Ta, 45 sec; 72°C, 20 sec,<br>
ending with a final extension at 72°C for 10 min. The first ten cycles Ta decreased from<br>
67°C to 58°C, one degree for each cycle, and the remaining 25 cycles Ta were fixed at<br>
58°C. PCR products were pooled and 5 to 9 different markers were run in each lane on an<br>
ABI 377 (Applied Biosystems), and gels were analysed with the accompanying software.<br>
Alleles were assigned with the Genotyper programme (Version 2.1, Applied Biosystems).<br>
Eor three markers, two-point lod scores were calculated using the MLINK programme of<br>
the LINKAGE package (Lathrop et al., 1985). Due to the pedigree structure (Figure 1) with<br>
multiple inbreeding loops, the pedigree was divided into thirteen small families, one for<br>
each affected calf including the Sire (KOL Nixon), the dam and the maternal grandsire (T<br>
Burma). The disease was assumed to be recessively Inherited with a complete penetrance<br>
of the genotype.<br>
Significant linkage was found for all three markers. The highest lod score (Z) was observed<br>
with BMS2790 and 1LSTS029 with Z=10.35 at 9=0. Furthermore, the nearby marker<br>
INRA003 was also significantly linked to the CVM locus (Table 3).<br>
(TABLE Removed) <br>
The above results locate the CVM locus to BTA3 (Figure 2) according to the USDA genetic<br>
map (Kappes et al., 1997).<br>
• Eleven calves were homozygous for the interval defined by INRA003, BMS2790,<br>
ILSTS029, BMS862 and HU3246, while BMS2790 and 1LSTS029 alone were homozygous<br>
in all thirteen calves as depicted In Figure 1. It was possible to construct<br>
haplotypes of these markers, allowing us to deduce the most likely CVM haplotype<br>
(Figure 1). The haplotypes are defined by the size of the marker alleles which are<br>
numbered from 1 to N where 1 defines the shortest allele of the amplified marker and<br>
N defines the longest allele. The actual length of the alleJes associated with CVM in Bell<br>
is as follows;<br>
INRA003 (allele no. 3): 176 base pairs<br>
BMS2790 (allele no. 3): 118 base pairs<br>
ILSTS029 (allele no. 2): 164 base pairs<br>
BMS862 (allele no. 1): 130 base pairs<br>
HUJ246 (allele no. 3): 262 base pairs<br>
The actual length 6f the alleles will depend upon the primers used to amplify the marker,<br>
and the fragment lengths shown above is based on using the primers described in Table 4.<br>
Furthermore, seventeen additional affected calves sampled as part of the Danish surveillance<br>
programme for genetic defects were included in the study. All affected animals had<br>
the pure-bred bull Bell as a common ancestor. DNA was extracted from blood samples or<br>
semen using standard procedures.<br>
The 17 calves and their mothers were genotyped with the 8 markers INRA003, BMS2790,<br>
1LSTS029, BM220, INRA123, BMS862, BMS937, and HUJ246 spanning the region on BTA3<br>
from approximately 59.5 cM to 67.9 cM, and since the CVM gene in the additional 17<br>
calves, like in the initial breeding study, was assumed to originate from the common<br>
ancestor bull Bell, a similar region of identity by descent (IBD) was expected to exist in all<br>
affected calves. This also turned out to be the case: in 17 out of 17 calves the chromosome<br>
segment defined by 1NRA003 and BMS2790 was homozygous, sharing the same<br>
alleles as the animals from the breeding study. In two of the nineteen animals, heteroTygosity<br>
was observed in the ILSTS029 and BMS862 locus explained by single recombination<br>
events between BMS2790 and ILSTS029. Thus, based on the combined genotyping results,<br>
we have found that the CVM genetic defect is most likely located In an interval of less than<br>
6 cM, flanked by the markers INRA003 and ILSTS029 as illustrated in Figure 2 and<br>
denoted "CVM region".<br>
The sequences of the primers for the applied 8 markers INRA003, BMS2790, ILSTS029,<br>
BM220, INRA123, BMS862, BMS937, and HUJ246, are depicted in Table 4 below.<br>
TABLE Removed<br><br>
Example 2<br>
Diagnostic test to identify CVM carriers:<br>
A diagnostic test to determine bovine carriers of CVM was established by determining<br>
whether descendants from Bell were carriers of the disease-associated haplotype.<br>
The test was based upon the 8 microsatellite markers INRA003, BMS2790, ILSTS029,<br>
BM220, INRA123, BMS862, BMS937 and HUJ246, and relied upon the recognition of the<br>
disease specific alleles or haplotype (see Figure 1) in animals descending from Bell.<br>
Animals were thus determined to be carriers if they had inherited the disease-associated<br>
alleles in the region defined by the markers INRA003, BMS2790, ILSTS029, and BM220<br>
from Bell or from animals descending from Bell. If the animals had not inherited the disease-<br>
associated haplotype from Bell or from animals descending from Bell, they were<br>
determined to be non-carriers. In cases where the Bell haplotype had been split by recombination,<br>
the animals were designated as indeterminable. The four additional markers<br>
were only used when the information content in the test markers was decreased due to<br>
inability to distinguish between maternal and paternal inheritance.<br>
Like all diagnostic genetic tests based upon linked DNA markers, the CVM test suffers from<br>
the drawback that a double recombination event (one event at each side of the causative<br>
gene, between the gene and the flanking markers) cannot be detected. In the present<br>
case, this event will be extremely rare due to the tight linkage between the markers and<br>
the CVM gene, and the reliability of the test is estimated to be higher than 99%.<br>
Example 3<br>
Tissue Dissection and RNA isolation and cDNA synthesis:<br>
Roughly 5 grams of heart tissue were dissected from dead-bom calves within 3 hours of<br>
delivery, immediately frozen in liquid nitrogen and stored at -BO°C. 250 mg of tissue was<br>
used for RNA isolation. RNA was isolated using the RNA Isolation Kit from Stratagene (cat.<br>
200345).<br>
cDNA was synthesised by mixing 2.5 ng of total RNA with 1 pi of oligo (dT)i2-i8 (500<br>
ng/ml), 1 nl of 10 mM dNTP mix and H2O to give a final volume of 12 pi. The resulting<br>
mixture was heated at 65°C for 5 min, chilled on Ice and spun briefly. Following the<br>
addition of 4 \i\ of 5 x first-strand buffer, 2 pi of 0.1 M DTT and 1 pi of H2O, the contents<br>
were mixed and incubated at 42°C for 2 min, after which 1 pi (200 U) of Superscript II<br>
(GibcoBRL® Lifetechnologies) was added and the incubation allowed to continue at 42°C<br>
for 1.5 hours. The reaction was inactivated at 70°C for 15 min. To remove RNA, the cDNA<br>
was incubated at 37°C for 20 min with 1 U RNase H (Roche Molecular Biochemicals).<br>
Example 4<br>
Sequencing of SLC35A3:<br>
Based on a homology alignment of the SLC35A3 gene between homo sapiens and canis<br>
familiaris, 2 oligos (SLIP and SL8R) were designed for amplification of almost the entire<br>
cDNA for bovine SLC35A3, including the start codon. To obtain the 3' end of the gene, the<br>
resulting PCR fragment was sequenced and a new ollgo designed (SL5F). To amplify the 3'<br>
end of SLC35A3, SL5F was used in combination with an oligo (bSLCBVIR), based on a<br>
published sequence of a bovine EST.<br>
The same PCR conditions were applied for both primer sets.<br>
The PCR reactions were performed In a GeneAmp® PCR System 9700 (Applied Biosystems)<br>
in a final volume of 10 pi consisting of 1 µl of lOxNH, reaction buffer, 0.5 µl of 50<br>
mM MgCI2, 0.8 pi of dNTPs (2.5 rnM of each), 5.65 µl of H2O, 1 pi of forward and reverse<br>
primer (5 pmol of each) and 0.05 µl of 5 U/ul BIOTAQ DNA polymerase (Bioline).<br>
The touchdown PCR reaction consisted of an initial heat activation step at 95°C for 2 min<br>
followed by 10 cydes of denaturation for 30 sec at 95°C, annealing at 62°C for 30 sec<br>
(0.5°C decrements), and elongation for 20 sec at 72°C, plus an additional 30 cycles with a<br>
denaturation step for 30 sec at 95"C, an annealing temperature of 57°C for 30 sec and an<br>
elongation step at 72°C for 30 sec.<br>
The following primers were used to amplify the complete SLC35A3 cDNA:<br>
SLIP: 5'-GGA GGC AAA TGA AGA TAA AAC-3' (SEQ ID NO: 19)<br>
SL8R: 5'-CTA TGC TTT AGT GGG ATT3-' (SEQ ID NO: 20)<br>
SL5F: 5'-GAG TTG CTT TTG TAC AGT GG-3' (SEQ ID NO: 21)<br>
bgLCBVIR: 5'-ACT GGC TAC TAT CTA GCA CAG GA-3' (SEQ ID NO: 22)<br>
The complete cDNA sequence was obtained by applying these primers in four separate<br>
cycle sequencing reactions using purified PCR products as the template. The PCR products<br>
were purified using SPIN-X* (Corning Incorporated) from a 0.8% Seakem agarose gel.<br>
Cycle sequencing reactions were carried out in a GeneAmp® PCR System 9700 (Applied<br>
Biosystems) and included an initial step at 96°C for 2 min followed by 99 cycles of 96°C for<br>
10 sec, 55°C for 5 sec and 60°C for 4 min. Sequencing products were precipitated with two<br>
volumes of ethanol and 1/10 volume of 3 M NaAc (pH 5.5), washed with 70% ethanol,<br>
resuspended in 5 pi of loading buffer and run on 4% acrylamide sequencing gels using an<br>
AB1377 automatic sequencer.<br>
The cDNA sequence of the bovine SLC35A3 is shown in Figure 4 (SEQ ID NO: 18).<br>
Example 5<br>
Identification and isolation of BACs containing microsatelllte markers<br>
Filter hybridisation:<br>
The filters were pre-hybridised in hybridisation solution (SxSSC (52.6 g of NaCI, 26.46 g<br>
of sodium citrate per litre), 5x Denhardt (2 g of ficoll (type 400, Pharmacia), 5 g of<br>
polyvinylpyrrolidone, 5 g of bovine serum albumin (Fraction V, Sigma), 0.5% SDS and 50<br>
ug/ml SS-DNA) at 65°C for 3 hours with rotation. 100 pi of 5'-end labelled oligonucleotide<br>
was then incubated with the filters for 16 hours at 65°C. For end labelling, 5 pmol of<br>
oligonucleotide was combined with 5 pi (50 pCI) of gamma-^P-ATP (specific activity &gt;<br>
5000 Ci/mrnole), 2 ul of lOx kinase buffer, 11 pi of H2O and 1 pi (10 U) of T4<br>
polynucleotide kinase (New England Biolabs Inc.), and the mix was incubated at 37°C for<br>
1.5 hours followed by 5 min of boiling to heat inactivate the enzyme. The labelled probe<br>
was NaAc/ethanol precipitated using standard procedures, and after a wash in 70%<br>
ethanol, the probe was redfssolved in 100 pi of H2O. Following hybridisation the filters were<br>
washed once with wash solution I (2xSSC, 0.2% SDS) and twice at 65°C with wash<br>
solution II (O.lxSSC, 0.5% SDS), and exposed to Kodak BIOMAX™ MS film for 2 days at -<br>
80°C.<br>
The following 5'-end labelled oligonucleotides were used to identify ILSTS029 and INRA003<br>
positive BAG clones:<br><br>
ltsTS029 oligo: 5'-CAC ACC GCT GTA CAG GAA AAA GTG TGC CAA CCC TGG TCT AAA<br>
TCC AAA ATC CAT TAT CTT CCA ACT ACA T-3' (SEQ ID NO: 23)<br>
INRA003 oligo: 5'-CGT CCC CTA TGC GCT TAC TAC ATA CAC TCA AAT GGA AAT GGG<br>
AAA ACT GGA GGT GTG TGA GCC CCA TIT A-3' (SEQ ID NO: 24)<br>
PCR screening of the bovine BAG library:<br>
BAG pools were prepared from the BAG library and screened by PCR. The PCR reactions<br>
were performed in a GeneAmp® PCR System 9700 (Applied Biosystems) in a final volume<br>
of 10 ul consisting of 1 pi of 10xNH
dNTPs (2.5 mM of each), 5.65 pi of H2O, 1 ul of forward and reverse primer (5 pmol of<br>
each) and 0.05 pi of 5 U/pl BIOTAQ DNA polymerase (Bioline).<br>
The following primers were used to identify BMS2790 containing BAG clones by PCR:<br>
BMS2790F: 5'-AAG ACA AGG ACT TTC AGC CC-3' (SEQ ID NO: 25)<br>
BMS2790R: 5'-AAA GAG TCG GAC ATT ACT GAG C-3' (SEQ ID NO: 26)<br>
The touchdown PCR reaction consisted of an Initial heat activation step at 95°C for 2 min<br>
followed by 10 cycles of denaturation for 30 sec at 95°C, annealing at 70°C for 30 sec<br>
(0.5"C decrements), and elongation for 20 sec at 72°C, plus an additional 30 cycles with a<br>
denaturation step at 95°C for 30 sec, annealing at 65°C for 30 sec and elongation at 72°C<br>
for 20 sec.<br>
BAG DNA isolation and sequencing;<br>
BAG DNA was prepared according to Qiagens Large Construct Kit, and approximately 1 pg<br>
of BAG DNA was used as the template for cyde sequendng performed with the BigDye™<br>
Terminator Cycle Sequencing Kit (PE Applied Biosystems). The cyde sequencing reactions<br>
were performed in a final volume of 6 pi containing 1 ul of Big Dye™ Terminator mix, 1 pi<br>
of primer (5 pmo)), 1 ul of reaction buffer and 2 pi of H2O. Cycle sequendng reactions<br>
were carried out in a GeneAmp® PCR System 9700 (Applied Biosystems) and induded an<br>
initial step at 96°C for 2 min followed by 130 cydes of 96°C for 10 sec, 55°C for 5 sec and<br>
60°C for 4 min. Sequencing products were precipitated with two volumes of ethanol and<br>
1/10 volume of 3 M NaAc (pH 5.5), washed with 70% ethanol, resuspended in 2 ul of<br>
loading buffer and run on 4% acrylamide sequencing gels using an A6I377 automatic<br>
sequencer.<br>
Sequencing primers:<br>
T7: ^-TTA TAG GAC TCA CTA TAG GG-3' (SEQ ID NO: 27)<br>
$P6: 5'-ATT TAG GTG ACA CTA TAG-3' (SEQ ID NO: 28)<br>
INRA003F: 5'-CTG GAG GTG TGT GAG CCC CAT TTA-3' (SEQ ID NO: 29)<br>
INRA003R: 5 ' -CTA AGA GTC GAA GGT GTG ACT AGG-3' (SEQ ID NO: 30)<br><br>
Example 6<br>
Determination of Q/M status by sequencing:<br>
PCR reactions (2 |jl of purified template/sample genomic DNA, 2 ul of lOxPCR buffer, 2 ul<br>
of 25 mM MgCI2, 3.3 M! of 0.2 mM of each dNTP (Ultrapure dNTP, 27-2033-01; Amersham<br>
Pharmacia Biotech), 6 pmol of primer (forward: CBFEX1, 5 '-GGC CCT CAG ATT CTC-3'<br>
(SEQ ID NO: 31); reverse: CBTEXR, 5' -GTT GAA TGT TTC TTA-3') (SEQ ID NO: 32),<br>
0.165 U Taq polymerase (Biotaq, M95801B; Bioline), dH2O ad total volume) were<br>
performed oil-free in 96-well plates using a Primus HT (MWG Biotech AG). Cycling<br>
conditions: 95°C 120 sec, 35x [95"C 60 sec, 60°C 30 sec, 72°C 110 sec]. Post-reaction<br>
clean-up was done by gel filtration (Milllpore Filtration System) with Sephadex G-50<br>
Superfine (17-0041-01, Amersham Pharmacia Biotech) carried out according to the<br>
manufacturer's recommendation, using 50 ul of dH2O for final sample elution. Forward and<br>
reverse sequencing reactions were performed with the same primers as used for the<br>
generation of the PCR product (2 ul of PCR product, 8 pi of Sequencing Mix, 0.6 ul of 6<br>
pmol Primer (see above), dH2O ad 20 ul; DYEnamic ET Dye Terminator Cycle Sequencing<br>
Kit (US81095). After thermocycling (30x [95°C 20 sec, 55°C 15 sec, 60°C 70 sec]) samples<br>
were cleaned by gel filtration essentially as described above and analysed on a<br>
MegaBACElOOO (Amersham Pharmacia Biotech) using LPA long-read matrix and the<br>
following sequencing conditions: 90 sec injection at 3 kV and 35 min run time at 9 kV.<br>
Example 7<br>
Ajleie-speclfic PCR assay:<br>
Primers were designed from the cDNA sequence and the four allele-specific primers were<br>
designed to have the 3' base at the position of the mutation.<br>
Primer sequences:<br>
T_fwd: 5'-CAG TGG CCC TCA GAT TCT CAA GAG CTT AATTCT AAG GAA CTTTCA<br>
GCT GGC TCA CAA TTT GTA GGT CTC ATG GCA T-3' (SEQ ID NO: 33)<br>
G_fwd: 5'-CAC AAT TTG TAG GTC TCA TGG CAG-3' (SEQ ID NO: 34)<br><br>
A_rev: 5'-GCC ACT GGA AAA ACA TGC TGT GAG AAA-3' (SEQ ID NO: 35)<br>
C_revjink*: 5'-aat get act act att agt aga att gat gcc ace ttt tea get cgc gee cca aat<br>
gaa aat ata get aaa cag gtt att gac cat ttg cga aat gta tct aat ggt caa act<br>
ttt ttC TGG AAA AAC ATG CTG TGA GAA C -3' (SEQ ID NO: 36)<br>
Fwd: 5'-GGC CCT CAG ATT CTC AAG AGC-3' (SEQ ID NO: 37)<br>
Rev: S'-CGA TGA AAA AGG AAC CAA AAG GG-3' (SEQ ID NO: 38)<br>
The C_rev._!ink primer contains a linker sequence from the M13 phage (shown in lower<br>
case letters). This linker was added to obtain a longer PCR product In order to be able to<br>
multiplex the C- and A-primers in one PCR reaction. The 3' base at the position of the<br>
mutation is shown in bold. The C- and G-primers are specific for the wildtype allele, while<br>
the T- and A-speclfic primers are specific for the mutation.<br>
Primer pairs:<br>
C- and A-specific multiplex: Fwd + A_rev + C_rev_link (lower strand)<br>
T-specifk reaction: T_fwd + Rev (upper strand)<br>
G-speclfic reaction: G_fwd + Rev (upper strand)<br>
AS-PCR conditions:<br>
Each PCR reaction was carried out in a 10 &gt;J volume containing 20-100 ng of genomic<br>
DNA, 0.025 units/ul of BIOLASE Diamond DNA polymerase, 0.75 mM dNTPs, 3 mM MgCI2,<br>
0.25 pmol/fil primer (0.125 pmol/nl of the two reverse primers in the multiplex) in 1 x NH4<br>
buffer (Bioline). PCR was carried out in a GeneAmp* PCR System 9700 (PE Applied<br>
Biosystems) under the following conditions: 95°C for 4 min, 35 cydes of 94°C for 30 s,<br>
62°C (56°C for the T- and G-reaction) at ramp 80% for 30 s, and 72°C for 30 s followed by<br>
a final extension at 72°C for 7 min and storage at 4°C. PCR was followed by electrophoresis<br>
in a 2% agarose gel at 200 V for 30 min.<br>
The results of the allele-spedfic PCR analysis of two wildtype, two carriers, and two sick<br>
animals are shown in Figure 7.<br>
Figure legends<br>
Figure l shows the pedigree used to locate the bovine complex vertebral malformation<br>
(CVM) locus and haplotypes of five microsatellite markers on bovine chromosome 3. The<br>
most likely CVM haplotypes are in bold. Filled black squares represent affected calves.<br>
Double lines between the sire and the dams indicate inbreeding loop. N refers to the<br>
number of animals. Genotypes of the thirteen different dams are for simplicity reasons not<br>
shown.<br>
Figure 2 shows the genetic map of bovine chromosome 3. Numbers on the sides refer to<br>
the genetic distances given In centlMorgan (cM) along the chromosome. The most likely<br>
location of the bovine complex vertebral malformation (CVM) locus is Indicated.<br>
F]gure_3 shows the relative distance In cM between the 3 microsatellite markers ILSTS029,<br>
BMS2790 and INRA003 (shown on the line denoted Contig markers on bovine Chr. 3) on<br>
the bovine chromosome 3 as depicted by the U.S. Meat Animal Research Center (Kappes<br>
et al. 1997). BACs containing these 3 markers were isolated either by hybridisation to high<br>
density replica filters (ILSTS029 and INRA003), or by PCR screening of the RPCI-42 bovine<br>
BAG library (BMS2790). The identified BACs are shown In black bars and annotated by<br>
plate number/well number. These BACs were subjected to end-sequencing using SP6 and<br>
T7 primers or to sequencing using primers extending from the microsatellite. The resulting<br>
sequences were blasted against the human chromosome 1 using the Ensemble Server at<br>
the Sanger Centre. The accession numbers from the blast search are shown as numbers<br>
under the human chromosome 1 and the relative distance between the hits is given in MB.<br>
Selected genes in the region are shown in boxes.<br>
Figure 4 shows the cDNA sequence and translation of the SLC35A3 gene (SEQ ID NO: 18)<br>
and the encoded amlno acid sequence (SEQ ID NO: 17). The polymorphic nucleotide In<br>
position 559 and the affected valine-180 is Indicated In bold.<br>
Figure 5 shows a comparison of the deduced amino add sequence of cow SLC35A3 with<br>
human (AB021981) (Ishida et al. 1999) and dog (AF057365) (Guillen et al. 1998)<br>
sequences. Dots Indicate residues that match the Bos Taurus sequence. Dashes indicate<br>
gaps that have been introduced to optimise the alignment.<br>
Figure 6 shows the results obtained from sequencing the region (from nucleotide 544 to<br>
572 of SLC35A3, see Figure 4) showing the G/T polymorphism in position 559 In determination<br>
of CVM status by sequencing. The left and right panels show forward and reverse<br>
sequencing, respectively. The upper row (-/-) shows the sequencing of a wildtype animal,<br>
the middle row shows the sequencing of a carrier (heterozygote), and the lower row shows<br>
the sequencing of an affected animal.<br><br>
Figure 7 is a picture showing the Allele-Specific PCR products from two wildtype, two<br>
carriers, and two sick animals. Annotations: WT: wildtype, C: carrier, S: sick, neg:<br>
negative control, M: marker (size ladder). Arrows show the allele-specific PCR products: C:<br>
220 bp, A: 98 bp, T: 340 bp, and G: 288 bp.<br>
References<br>
1. Agerholm JS, Bendixen, C., Andersen O., Arnbjerg, J. (2000) LK meddelelser October<br>
2000.<br>
2. Barendse W, Vaiman D, Kemp SJ, Sugimoto Y, Armitage SM, Williams JL, Sun HS,<br>
Eggen A, Agaba M, Aleyasin SA, Band M, Bishop MD, Buitkamp J, Byrne K, Collins F,<br>
Cooper L, Coppettiers W, Denys B, Drinkwater RD, Easterday K, Elduque C, Ennis S,<br>
Erhardt G, Li L, Ul L (1997) A medium-density genetic linkage map of the bovine genome.<br>
Mamm Genome 8, 21-28.<br>
3. Bruckner, K., Perez, L., Clausen, H., &amp; Cohen, S., (2000) Glycosyltransferase activity of<br>
Fringe modulates Notch-Delta interactions Nature 406, pp. 411-415.<br>
4. Evrad, Y. A., Lun, Y., Aulehla, A,. Gan, L, &amp; 3ohnson, R. L. (1998) Lunatic fringe is an<br>
essential mediator of somite segmentation and patterning. Nature 394, pp. 377-381.<br>
5. Guillen, E., Abeijon, C., &amp; Hirschberg, C. B. (1998) Mammalian Golgi apparatus UDP-Nacetylglucosamine<br>
transporter: Molecular cloning by phenotypic correction of a yeast<br>
mutant. Proc. Wat/. Acad. Sci.USA. 95, pp. 7888-7892.<br>
6. Ishida, N., Yoshioka, S., Chlba, Y., Takeuchi, M., &amp; Kawakita, M. (1999) Molecular<br>
cloning and functional expression of the human golgi UDP-N-acetylglucosamine<br>
transporter. 3. Biochem. 126, pp. 68-77.<br>
7. Kappes SM, Keele JW, Stone RT, McGraw RA, Sonstegard TS, Smith TP, Lopez-Corrales<br>
NL, Beattie CW (1997) A second-generation linkage map of the bovine genome.<br>
Genome Res 7, 235-249.<br>
8. Klein, T., &amp; Arias, M. (1998) Interactions among Delta, Serrate and Fringe modulate<br>
Notch activity during drosophila wing development. Development 125, pp. 2951-2962.<br>
9. Lathrop GM, Lalouel jfo, Julier C, Ott J (1985) Multilocus linkage analysis in humans:<br>
detection of linkage and estimation of recombination. Am J Hum Genet 37, 482-498<br>
10. Moloney, DJ., Panin, V. M., Johnston, S. H., Chen, 3., Shao, L, Wilson, Y., Stanley, P.,<br>
Irvine, K. D., Haltiwanger, R. S., &amp; Vogt, T. F. (2000) Fringe Is a glycosyltransferase<br>
that modifies Notch. Nature 4O6, pp 369-375.<br>
11. Solinas-Toldo S, Lengauer C, Fries R (1995) Comparative genome map of human and<br>
cattle Genomlcs 27, 489-496.<br><br>
We claim:<br>
1. An oligonucleotide primer for use in detecting the presence in a bovine subject of at least one genetic marker associated with bovine complex vertebral malformation (CVM), wherein said oligonucleotide primer is selected from SEQ ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 36, SEQ ID NO: 37 or SEQ ID NO: 38, and combinations thereof.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktREVMTlAtMjAwMy1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">00909-DELNP-2003-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLTA5LTA5LTIwMDgucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-description (complete)-09-09-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktREVMTlAtMjAwMy1Gb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">00909-DELNP-2003-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1mb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1wY3QtMjEwLnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1wY3QtMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1wY3QtNDA5LnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA5MDktZGVsbnAtMjAwMy1wY3QtNDE2LnBkZg==" target="_blank" style="word-wrap:break-word;">00909-delnp-2003-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQWJzdHJhY3QtKDA5LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Abstract-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQWJzdHJhY3QtKDIwLTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Abstract-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQ2xhaW1zLSgwOS0wOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Claims-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQ2xhaW1zLSgyMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Claims-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgwOS0wOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Correspondence-Others-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Correspondence-Others-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRGVzY3JpcHRpb24gKENvbXBsZXRlKS0oMjAtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Description (Complete)-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRHJhd2luZ3MtKDA5LTA5LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Drawings-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRm9ybS0xLSgyMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Form-1-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRm9ybS0yLSgwOS0wOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Form-2-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRm9ybS0yLSgyMC0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Form-2-(20-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTA5LURFTE5QLTIwMDMtRm9ybS0zLSgwOS0wOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">909-DELNP-2003-Form-3-(09-09-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="234901-a-composition-for-whitening-teeth.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="234903-a-skin-lightening-composition.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>234902</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>00909/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>19-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Jun-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AARHUS UNIVERSITET</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NORDRE RINGGADE 1, DK-8000 ARHUS C, DENMARK.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PANITZ, FRANK</td>
											<td>ØSTER BORDINGVæNGET 19B, DK-8600 SIKEBORG, DENMARK.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JONKER, MARC</td>
											<td>ENGBOULEVARDEN 37, ST. TH., DK-8900 RANDERS, DENMARK.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BENDIXEN, CHRISTIAN</td>
											<td>AMSTRUPVEJ 7, DK-8860 ULSTRUP, DENMARK.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>SVENDSEN, SOREN</td>
											<td>RINDSVEJ 13, ST. IV.,DK-8900 RANDERS, DENMARK.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>JENSEN, HELLE</td>
											<td>SCL. JØRGENSVEJ 24, LEJL. 11, DK-8800 VIBORG, DENMARK.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>AASBERG, ANDERS</td>
											<td>SKEJBYTOFTEN 122, DK-8200 ÅRHUS N, DENMARK.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>HOLM, LARS-ERIK</td>
											<td>SOLSORTEVEJ 1, DK-8370 HADSTEN, DENMARK.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>HORN, PER</td>
											<td>NØRRESØKOLLEGIET 13, DK-8800 VIBORG, DENMARK.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>HØJ, ANETTE</td>
											<td>VIBORGVEJ 1, HAMMERSHØJ, DK-8830 TJELE, DENMARK.</td>
										</tr>
										<tr>
											<td>10</td>
											<td>THOMSEN, BO</td>
											<td>RYHAVEN 49, DK-8210 ÅRHUS V, DENMARK.</td>
										</tr>
										<tr>
											<td>11</td>
											<td>JEPPESEN, METTE</td>
											<td>RØNNEVEJ 36, DK-8800 VIBORG, DENMARK.</td>
										</tr>
										<tr>
											<td>12</td>
											<td>NIELSEN, VIVI HUNNICKE</td>
											<td>VELDSPARKEN 72, ØRUM, DK-8830 TJELE, DENMARK.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/DK01/00756</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-11-15</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2000 01717</td>
									<td>2000-11-16</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2001 00765</td>
									<td>2001-05-15</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/234902-oligonucleotide-primers-for-use-in-detecting-the-presence-of-genetic-markers-in-a-bovine-subject by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:30:15 GMT -->
</html>
