<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/235016-a-fungal-consensus-phytase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:33:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 235016:A FUNGAL CONSENSUS PHYTASE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A FUNGAL CONSENSUS PHYTASE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a fungal consensus phytase which has the amino acid sequence shown in Figure 2 or any variants thereof wherein the variants basic&#x27; properties such as enzymatic activity (type of and specific activity), i, thermo stability, activity in a certain ph-range (PH-stability) have not significantly been changed. A food, feed or pharmaceutical composition comprising the consensus protein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Phytases (myo-inositol hexakisphosphate phosphohydrolases; EC 3.1.3.8) are enzymes that hydrolyze phytate (myo-inositol hexakisphosphate) to myoinositol and inorganic phosphate and are known to be valuable feed additives.<br>
A phytase was first described in rice bran in 1907 [Suzuki et ah, Bull. Coll. Agr. Tokio Imp. Univ. 7, 495 (1907)] and phytases from Aspergillus species in 1911 [Dox and Golden, J. Biol. Chem. 10, 183-186 (1911)].  Phytases have also been found in wheat bran, plant seeds, animal intestines and in microorganisms [Howsen and Davis, Enzyme Microb. Technol. |3, 377-382 (1983), Lambrechts et al., Biotech. Lett. 14, 61-66 (1992), Shieh and Ware, Appl. Microbiol. 16, 1348-1351 (1968)].<br>
The cloning and expression of the phytase from Aspergillus niger (ficuum) has been described by Van Hartingsveldt et al., in Gene, 127, 87-94 (1993) and in European Patent Application, Publication No. (EP) 420 358 and from Aspergillus niger var. awamori by Piddington et al., in Gene 133, 55-62 (1993).<br>
Cloning, expression and purification of phytases with improved properties have been disclosed in EP 684 313. However, since there is a still ongoing need for further improved phytases, especially with respect to their thermostability, it is an object of the present invention to provide the following process which is, however, not only applicable to phytases.<br>
A process for the preparation of a consensus protein, whereby such process is characterized by the following steps:<br><br>
a)	at least three preferably four amino acid sequences of a defined protein family are aligned by any standard alignment program known in the art;<br>
b)	amino acids at the same position according to such alignment are compared regarding their evolutionary similarity by any standard program known in the art, whereas the degree of similarity provided by such a program which defines the least similarity of the amino acids that is used for the determination of an amino acid of corresponding positions is set to a less stringent number and the parameters are set in such a way that it is possible for the program to determine from only 2 identical amino acids at a corresponding position an amino acid for the consensus protein; however, if among the compared amino acid sequences are sequences that show a much higher degree of similarity to each other than to the residual sequences, these sequences are represented by their consensus sequence determined as defined in the same way as in the present process for the consensus sequence of the consensus protein or a vote weight of 1 divided by the number of such sequences is assigned to every of those sequences.<br>
c)	in case no common amino acid at a defined position can be identified by the program, any of the amino acids of all sequences used for the comparison, preferably the most frequent amino acid of all such sequences is selected or an amino acid is selected on the basis of the consideration given in Example 2.<br>
d)	once the consensus sequence has been defined, such sequence is back-translated into a DNA sequence, preferably using a codon frequency table of the organism in which expression should take place;<br>
e)	the DNA sequence is synthesized by methods known in the art and used either integrated into a suitable expression vector or by itself to transform an appropriate host cell;<br>
f)	the transformed host cell is grown under suitable culture conditions and the consensus protein is isolated from the host cell or its culture medium by methods known in the art.<br><br>
In a preferred embodiment of this process step b) can also be defined as follows:<br>
b) amino acids at the same position according to such an alignment are compared regarding their evolutionary similarity by any standard program known inthe art, whereas the degree of similarity provided by such program is set at the lowest possible value and the amino acid which is the most similar for at least half of the sequences used for the comparison is selected for the corresponding position in the amino acid sequence of the consensus protein.<br>
A preferred embodiment of this whole process can be seen in a process in which a sequence is choosen from a number of highly homologous sequences and only those amino acid residues are replaced which clearly differ from a consensus sequence of this protein family calculated under moderately stringent conditions, while at all positions of the alignment where the method is not able to determine an amino acid under moderately stringent conditions the amino acids of the preferred sequence are taken.<br>
It is furthermore an object of the present invention to provide such a process, wherein the program used for the comparison of amino acids at a defined position regarding their evolutionary similarity is the program "PRETTY". It is more specifically an object of the present invention to provide such a process, wherein the defined protein family is the family of phytases, especially wherein the phytases are of fungal origin.<br>
It is furthermore an object of the present invention to provide such processes, wherein the host cell is of eukaryotic, especially fungal, preferably Aspergillus or yeast, preferably Saccharomyces or Hansenula origin.<br>
It is also an object of the present invention to provide a consensus protein obtainable preferably obtained, by such processes and specifically the consensus protein, which has the amino acid sequence shown in Figure 2 or a variant thereof. A "variant" refers in the context of the present invention to a consensus protein with amino acid sequence shown in Figure 2 wherin at one or more positions amino acids have been deleted, added or replaced by one or more other amino acids with the provisor that the resulting sequence provides for a protein whose basic properties like enzymatic activity (type of and specific activity), thermostability, activity in a certain pH-range (pH-stability) have<br><br>
not significantly been changed. "Signaficantly means in this context that a man skilled in the art would say that the properties of the variant may still be different but would not be unobvious over the ones of consensus protein with the amino acid sequence of Figure 2 itself.<br>
A mutein refers in the context of the present invention to replacements of the amino acid in the amino acid sequences of the consensus proteins shown in Figure 2 which lead to consensus proteins with further improved properties e. g. activity. Such muteins can be defined and prepared on the basis of the teachings given in European Patent Application number 97810175.6, e. g. Q50L, Q50T, Q50G, Q50L-Y51N, or Q50T-Y51N. "Q50L" means in this context that at position 50 of the amino acid sequence the amino acid Q has been replaced by amino acid L.<br>
In addition, a food, feed or pharmaceutical composition comprising a consensus protein as defined above is also an object of the present invention.<br>
In this context "at least three preferably three amino acid sequences of such defined protein family" means that three, four, five, six to 12, 20, 50 or even more sequences can be used for the alignment and the comparison to create the amino acid sequence of the consensus protein. "Sequences of a defined protein family" means that such sequences fold into a three dimensional structure, wherein the a-helixes, the (3-sheets and-turns are at the<br>
same position so that such structures are, as called by the man skilled in the art, superimposable. Furthermore these sequences characterize proteins which show the same type of biological activity, e.g. a defined enzyme class, e.g. the phytases. As known in the art, the three dimensional structure of one of such sequences is sufficient to allow the modelling of the structure of the other sequences of such a family. An example, how this can be effected, is given in the Reference Example of the present case. "Evolutionary similarity" in the context of the present invention refers to a schema which classifies amino acids regarding their structural similarity which allows that one amino acid can be replaced by another amino acid with a minimal influence on the overall structure, as this is done e.g. by programs, like "PRETTY", known in the art. The phrase "the degree of similarity provided by such a program...is set to less stringent number" means in the context of the present invention that values for the parameters which determine the degree of similarity in the prgram used in the practice of the present invention are chosen in a way to allow the program to define a common amino acid for a maximum of positions of the whole amino<br><br>
acid sequence, e. g. in case of the program PRETTY a value of 2 or 3 for the THRESHOLD and a value of 2 for the PLURALITY can be choosen. Furthermore, "a vote weight of one devided by the number of such sequences" means in the context of the present invention that the sequences which define a group of sequences with a higher degree of similarity as the other sequences used for the determination of the consensus sequence only contribute to such determination with a factor which is equal to one devided by a number of all sequences of this group.<br>
As mentioned before should the program not allow to select the most similar amino acid, the most frequent amino acid is selected, should the latter be impossible the man skilled in the art will select an amino acid from all the sequences used for the comparison which is known in the art for its property to improve the thermostability in proteins as discussed e.g. by<br>
Janecek, S. (1993), Process Biochem. 28, 435-445 or<br>
Fersht, A. R. &amp; Serrano, L. (1993), Curr. Opin. Struct Biol 3, 75-83.<br>
Alber, T. (1989), Annu. Rev. Biochem. 58, 765-798 or<br>
Matthews, B. W. (1987), Biochemistry 26, 6885-6888.<br>
Matthews, B. W. (1991), Curr. Opin. Struct Biol. 1, 17-21.<br>
The stability of an enzyme is a critical factor for many industrial applications. Therefore, a lot of attempts, more or less successful, have been made to improve the stability, preferably the thermostability of enzymes by rational (van den Burg et a/., 1998) or irrational approaches (Akanuma et at, 1998). The forces influencing the thermostability of a protein are the same as those that are responsible for the proper folding of a peptide strand (hydrophobic interactions, van der Waals interactions, H-bonds, salt bridges, conformational strain (Matthews, 1993). Furthermore, as shown by Matthews et al. (1987), the free energy of the unfolded state has also an influence on the stability of a protein. Enhancing of protein stability means to increase the number and strength of favorable interactions and to decrease the number and strength of unfavorable interactions. It has been possible to introduce disulfide linkages ( Sauer et al.t 1986) to replace glycine with alanine residues or to increase the proline content in order to reduce the free energy of the unfolded state (Margarit et at, 1992; Matthews, 1987a). Other groups concentrated on the importance of additional H-bonds or salt bridges for the stability of a protein (Blaber et at, 1993) or tried to fill cavities in the protein interior to increase the buried hydrophobic surface area and the van der Waals<br><br>
interactions   (Karpusas   et   al,   1989).   Furthermore,   the   stabilization   of secondary structure elements, especially a-helices, for example, by improved<br>
helix capping, was also investigated (Munoz &amp; Serrano, 1995).<br>
However, there is no fast and promising strategy to identify amino acid replacements which will increase the stability, preferably the thermal stability of a protein. Commonly, the 3D structure of a protein is required to find locations in the molecule where an amino acid replacement possibly will stabilize the protein's folded state. Alternative ways to circumvent this problem are either to search for a homologous protein in a thermo- or hyperthermophile organism or to detect stability-increasing amino acid replacements by a random mutagenesis approach. This latter possibility succeeds in only 103 to 104 mutations and is restricted to enzymes for which a fast screening procedure is available (Arase et al., 1993; Risse et al., 1992). For all these approaches, success was variable and unpredictable and, if successful, the thermostability enhancements nearly always were rather small.<br>
Here we present an alternative way to improve the thermostability of a protein. Imanaka et al. (1986) were among the first to use the comparisons of homologous proteins to enhance the stability of a protein. They used a comparison of proteases from thermophilic with homologous ones of mesophilic organisms to enhance the stability of a mesophilic protease. Serrano et al. (1993) used the comparison of the amino acid sequences of two homologous mesophilic RNases to construct a more thermostable Rnase. They mutated individually all of the residues that differ between the two and combined the mutations that increase the stability in a multiple mutant. Pantoliano et al. (1989) and, in particular, Steipe et al. (1994) suggested that the most frequent amino acid at every position of an alignment of homologous proteins contribute to the largest amount to the stability of a protein. Steipe et al. (1994) proved this for a variable domain of an immunoglobulin, whereas Pantoliano et al. (1989) looked for positions in the primary sequence of subtilisin in which the sequence of the enzyme chosen to be improved for higher stability was singularly divergent. Their approach resulted in the replacement M50F which increased the Tm of subtilisin by 1.8 °C.<br>
Steipe et al. (1994) proved on a variable domain of immunoglobulin that it is possible to predict a stabilizing mutation with better than 60% success<br><br>
rate just by using a statistical method which determines the most frequent amino acid residue at a certain position of this domain. It was also suggested that this method would provide useful results not only for stabilization of variable domains of antibodies but also for domains of other proteins. However, it was never mentioned that this method could be extended to the entire protein. Furthermore, nothing is said about the program which was used to calculate the frequency of amino acid residues at a distinct position or whether scoring matrices were used as in the present case.<br>
It is therefore an object of the present invention to provide a process for the preparation of a consensus protein comprising a process to calculate an amino acid residue for nearly all positions of a so-called consensus protein and to synthesize a complete gene from this sequence that could be expressed in a pro- or eukaryotic expression system.<br>
DNA sequences of the present invention can be constructed starting from genomic or cDNA sequences coding for proteins, e.g. phytases known in the state of the art [for sequence information see references mentioned above, e.g. EP 684 313 or sequence data bases, for example like Genbank (Intelligenetics, California, USA), European Bioinformatics Institute (Hinston Hall, Cambridge, GB), NBRF (Georgetown University, Medical Centre, Washington DC, USA) and Vecbase (University of Wisconsin, Biotechnology Centre, Madison, Wisconsin, USA) or disclosed in the figures by methods of in vitro mutagenesis [see e.g. Sambrook et al., Molecular Cloning, Cold Spring Harbor Laboratory Press, New York]. A widely used strategy for such "site directed mutagenesis", as originally outlined by Hurchinson and Edgell [J. Virol. 8, 181 (1971)], involves the annealing of a synthetic oligonucleotide carrying the desired nucleotide substitution to a target region of a single-stranded DNA sequence wherein the mutation should be introduced [for review see Smith, Annu. Rev. Genet. _19, 423 (1985) and for improved methods see references 2-6 in Stanssen et al., Nucl. Acid Res., J/7, 4441-4454 (1989)]. Another possibility of mutating a given DNA sequence which is also preferred for the practice of the present invention is the mutagenesis by using the polymerase chain reaction (PCR). DNA as starting material can be isolated by methods known in the art and described e.g. in Sambrook et al. (Molecular Cloning) from the respective strains. For strain information see, e.g. EP 684 313 or any depository authority indicated below. Aspergillus niger [ATCC 9142],<br><br>
Myceliophthora thermophila [ATCC 48102], Talaromyces thermophilus [ATCC 20186] and Aspergillus fumigatus [ATCC 34625] have been redeposited according to the conditions of the Budapest Treaty at the American Type Culture Cell Collection under the following accession numbers: ATCC 74337, ATCC 74340, ATCC 74338 and ATCC 74339, respectively. It is however, understood that DNA encoding a consensus protein in accordance with the present invention can also be prepared in a synthetic manner as described, e.g. in EP 747 483 or the examples by methods known in the art.<br>
Once complete DNA sequences of the present invention have been obtained they can be integrated into vectors by methods known in the art and described e.g. in Sambrook et al. (s.a.) to overexpress the encoded polypeptide in appropriate host systems. However, a man skilled in the art knows that also the DNA sequences themselves can be used to transform the suitable host systems of the invention to get overexpression of the encoded polypeptide. Appropriate host systems are for example fungi, like Aspergilli, e.g. Aspergillus niger [ATCC 9142] or Aspergillus ficuum [NRRL 3135] or like Trichoderma, e.g. Trichoderma reesei or yeasts, like Saccharomyces, e.g. Saccharomyces cerevisiae or Pichia, like Pichia pastoris, or Hansenula polymorpha, e.g. H. polymorpha (DSM5215) or plants, as described, e.g. by Pen et al., Bio/Technology 11, 811-814 (1994). A man skilled in the art knows that such microorganisms are available from depository authorities, e.g. the American Type Culture Collection (ATCC), the Centraalbureau voor Schimmelcultures (CBS) or the Deutsche Sammlung fiir Mikroorganismen und Zellkulturen GmbH (DSM) or any other depository authority as listed in the Journal "Industrial Property" [(1991) I, pages 29-40], Bacteria which can be used are e.g. E. coli, Bacilli as, e.g. Bacillus subtilis or Streptomyces, e.g. Streptomyces lividans (see e.g. Anne and Mallaert in FEMS Microbiol. Letters 114, 121 (1993). E. coli, which could be used are E. coli K12 strains e.g. M15 [described as DZ 291 by Villarejo et al. in J. Bacteriol. 120, 466-474 (1974)], HB 101 [ATCC No. 33694] or E. coli SG13009 [Gottesman et al., J. Bacteriol. 148, 265-273 (1981)].<br>
Vectors which can be used for expression in fungi are known in the art and described e.g. in EP 420 358, or by Cullen et al. [Bio/Technology 5, 369-376 (1987)] or Ward in Molecular Industrial Mycology, Systems and Applications for Filamentous Fungi, Marcel Dekker, New York (1991), Upshall et al. [Bio/Technology 5, 1301-1304(1987)] Gwynne et al. [Bio/Technology 5, 71-79<br><br>
(1987)], Punt et al. [J. Biotechnol. 17, 19-34 (1991)] and for yeast by Sreekrishna et al. [J. Basic Microbiol. 28, 265-278 (1988), Biochemistry 28, 4117-4125 (1989)], Hitzemann et al. [Nature 293, 717-722 (1981)] or in EP 183 070, EP 183 071, EP 248 227, EP 263 311. Suitable vectors which can be used for expression in E. coli are mentioned, e.g. by Sambrook et al. [s.a.] or by Fiers et al. in Procd. 8th Int. Biotechnology Symposium" [Soc. Franc, de Microbiol., Paris (Durand et al., eds.), pp. 680-697 (1988)] or by Bujard et al. in Methods in Enzymology, eds. Wu and Grossmann, Academic Press, Inc. Vol. 155, 416-433 (1987) and Stiiber et al. in Immunological Methods, eds. Lefkovits and Pernis, Academic Press, Inc., Vol. IV, 121-152 (1990). Vectors which could be used for expression in Bacilli are known in the art and described, e.g. in EP 405 370, Procd. Natl. Acad. Sci. USA 81, 439 (1984) by Yansura and Henner, Meth. Enzymol. !L85, 199-228 (1990) or EP 207 459. Vectors which can be used for the expression in H. Polymorpha are known in the art and described, e.g. in Gellissen et al., Biotechnology 9, 291-295 (1991).<br>
Either such vectors already carry regulatory elements, e.g. promotors, or the DNA sequences of the present invention can be engineered to contain such elements. Suitable promotor elements which can be used are known in the art and are, e.g. for Trichoderma reesei the cbhl- [Haarki et al., Biotechnology 1_, 596-600 (1989)] or the pkil-promotor [Schindler et al., Gene 130, 271-275 (1993)], for Aspergillus oryzae the amy-promotor [Christensen et al., Abstr. 19th Lunteren Lectures on Molecular Genetics F23 (1987), Christensen et al., Biotechnology 6, 1419-1422 (1988), Tada et al, Mol. Gen. Genet. 229, 301 (1991)], for Aspergillus niger the glaA- [Cullen et al, Bio/Technology 5, 369-376 (1987), Gwynne et al., Bio/Technology 5, 713-719 (1987), Ward in Molecular Industrial Mycology, Systems and Applications for Filamentous Fungi, Marcel Dekker, New York, 83-106 (1991)], alcA- [Gwynne et al, Bio/Technology 5, 718-719 (1987)], sucl- [Boddy et al, Curr. Genet. 24, 60-66 (1993)], aphA- [MacRae et al., Gene 71. 339-348 (1988), MacRae et al., Gene 132, 193-198 (1993)], tpiA-[McKnight et al, Cell 46, 143-147 (1986), Upshall et al., Bio/Technology 5, 1301-1304 (1987)], gpdA- [Punt et al., Gene 69, 49-57 (1988), Punt et al., J. Biotechnol. 1/7, 19-37 (1991)] and the pkiA-promotor [de Graaff et al., Curr. Genet. 22, 21-27 (1992)]. Suitable promotor elements which could be used for expression in yeast are known in the art and are, e.g. the pho5-promotor [Vogel et al., Mol. Cell. Biol., 2050-2057 (1989); Rudolf and Hinnen, Proc. Natl. Acad. Sci. 84, 1340-1344 (1987)] or the gap-promotor for expression in Saccharomyces<br><br>
cerevisiae and for Pichia pastoris, e.g. the aoxl-promotor [Koutz et al., Yeast J&gt;, 167-177 (1989); Sreekrishna et al., J. Basic Microbiol. 28, 265-278 (1988)], or the FMD promoter [Hollenberg et al, EPA No. 0299108] or MOX-promotor [Ledeboer et al., Nucleic Acids Res. JL3, 3063-3082 (1985)] for H. polymorpha.<br>
Accordingly vectors comprising DNA sequences of the present invention, preferably for the expression of said DNA sequences in bacteria or a fungal or a yeast host and such transformed bacteria or fungal or yeast hosts are also an object of the present invention.<br>
It is also an object of the present invention to provide a system which allows for high expression of proteins, preferably phytases like the consensus phytase of the present invention in Hansenula characterized therein that the codons of the encoding DNA sequence of such a protein have been selected on the basis of a codon frequency table of the organism used for expression, e.g. yeast as in the present case (see e.g. in Example 3) and optionally the codons for the signal sequence have been selected in a manner as described for the specific case in Example 3. That means that a codon frequency table is prepared on the basis of the codons used in the DNA sequences which encode the amino acid sequences of the defined protein family. Then the codons for the design of the DNA sequence of the signal sequence are selected from a codon frequency table of the host cell used for expression whereby always codons of comparable frequency in both tables are used.<br>
Once such DNA sequences have been expressed in an appropriate host cell in a suitable medium the encoded protein can be isolated either from the medium in the case the protein is secreted into the medium or from the host organism in case such protein is present intracellularly by methods known in the art of protein purification or described in case of a phytase, e.g. in EP 420 358. Accordingly a process for the preparation of a polypeptide of the present invention characterized in that transformed bacteria or a host cell as described above is cultured under suitable culture conditions and the polypeptide is recovered therefrom and a polypeptide when produced by such a process or a polypeptide encoded by a DNA sequence of the present invention are also an object of the present invention.<br>
Once obtained the polypeptides of the present invention can be characterized regarding their properties which make them useful in agriculture<br><br>
any assay known in the art and described e.g. by Simons et al. [Br. J. Nutr. 64, 525-540 (1990)], Schoner et al. [J. Anim. Physiol, a. Anim. Nutr. 66, 248-255 (1991)], Vogt [Arch. Geflugelk. 56, 93-98 (1992)], Jongbloed et al. [J. Anim. Sci., 70, 1159-1168 (1992)], Perney et al. [Poultry Sci. 72, 2106-2114 (1993)], Farrell et al., [J. Anim. Physiol, a. Anim. Nutr. 69, 278-283 (1993), Broz et al., [Br. Poultry Sci. 35, 273-280 (1994)] and Diingelhoef et al. [Animal Feed Sci. Technol. 49, 1-10 (1994)] can be used.<br>
In general the polypeptides of the present invention can be used without being limited to a specific field of application, e.g. in case of phytases for the conversion of inositol polyphosphates, like phytate to inositol and inorganic phosphate.<br>
Furthermore the polypeptides of the present invention can be used in a process for the preparation of a pharmaceutical composition or compound food or feeds wherein the components of such a composition are mixed with one or more polypeptides of the present invention. Accordingly compound food or feeds or pharmaceutical compositions comprising one or more polypeptides of the present invention are also an object of the present invention. A man skilled in the art is familiar with their process of preparation. Such pharmaceutical compositions or compound foods or feeds can further comprise additives or components generally used for such purpose and known in the state of the art.<br>
It is furthermore an object of the present invention to provide a process for the reduction of levels of phytate in animal manure characterized in that an animal is fed such a feed composition in an amount effective in converting phytate contained in the feedstuff to inositol and inorganic phosphate.<br>
Before describing the present invention in more detail a short explanation of the Tables and enclosed Figures is given below.<br>
Table 1: Vote weights of the amino acid sequences of the fungal phytases used. The table shows the vote weights used to calculate the consensus sequence of the fungal phytases.<br>
Table 2: Homology of the fungal phytases. The amino acid sequences of the phytases used in the alignment were compared by the program GAP (GCG program package, 9; Devereux et al., 1984) using the standard parameters. The comparison was restricted to the part of the sequence that was also used for<br><br>
the alignment (see legend to Figure 1) lacking the signal peptide which was ratherdivergent. The numbers above and beneath the diagonal represent the amino acid identities and similarities, respectively.<br>
Table 3: Homology of the amino acid sequence of fungal consensus phytase to the phytases used for its calculation. The amino acid sequences of all phytases were compared with the fungal consensus phytase sequence using the program GAP (GCG program package, 9.0). Again, the comparison was restricted to that part of the sequence that was used in the alignment.<br>
Table 4: Primers used for the introduction of single mutations into fungal consensus phytase. For the introduction of each mutation, two primers containing the desired mutation were required (see Example 8). The changed triplets are highlighted in bold letters.<br>
Table 5: Temperature optimum and Tm-value of fungal consensus phytase and of the phytases from A fumigatus, A niger, A nidulans, andM thermophila. The temperature optima were taken from Figure 3. aThe Tm-values were determined by differential scanning calorimetry as described in Example 10 and shown in Figure 7.<br>
Figure 1: Calculation of the consensus phytase sequence from the alignment of nearly all known fungal phytase amino acid sequences. The letters represent the amino acid residues in the one-letter code. The following sequences were used for the alignment: phyA from Aspergillus terreus 9A-1 (Mitchell et al., 1997; from amino acid (aa) 27), phyA from Aspergillus terreus cbsll6.46 (van Loon et al., 1997; from aa 27), phyA from Aspergillus niger var. awamori (Piddington et al., 1993; from aa 27), phyA from Aspergillus niger T213; from aa 27), phyA from Aspergillus niger strain NRRL3135 (van Hartingsveldt et al., 1993; from aa 27), phyA from Aspergillus fumigatus ATCC 13073 (Pasamontes et al., 1997b; from aa 25), phyA from Aspergillus fumigatus ATCC 32722 (van Loon et al., 1997; from aa 27), phyA from Aspergillus fumigatus ATCC 58128 (van Loon et al, 1997; from aa 27), phyA from Aspergillus fumigatus ATCC 26906 (van Loon et al., 1997; from aa 27), phyA from Aspergillus fumigatus ATCC 32239 (van Loon et al, 1997; from aa 30), phyA from Aspergillus nidulans (Pasamontes et al., 1997a; from aa 25), phyA from Talaromyces thermophilus (Pasamontes et al., 1997a; from aa 24), and phyA from<br><br>
Myceliophthora thermophila (Mitchell et aL, 1997; from aa 19). The alignment was calculated using the program PILEUP. The location of the gaps was refined by hand. Capitalized amino acid residues in the alignment at a given position belong to the amino acid coalition that establish the consensus residue. In bold, beneath the calculated consensus sequence, the amino acid sequence of the finally constructed fungal consensus phytase (Fcp) is shown. The gaps in the calculated consensus sequence were filled by hand according to principals stated in Example 2.<br>
Figure 2: DNA sequence of the fungal consensus phytase gene (fcp) and of the primers synthesized for gene construction. The calculated amino acid sequence (Figure 1) was converted into a DNA sequence using the program BACKTRANSLATE (Devereux et aL, 1984) and the codon frequency table of highly expressed yeast genes (GCG program package, 9.0). The signal peptide of the phytase from A. terreus cbs was fused to the Af-terminus. The bold bases represent the sequences of the oligonucleotides used to generate the gene. The names of the respective oligonucleotides are noted above or below the sequence. The underlined bases represent the start and stop codon of the gene. The bases written in italics show the two introduced Eco RI sites.<br>
Figure 3: Temperature optimum of fungal consensus phytase and other phytases used to calculate the consensus sequence. For the determination of the temperature optimum, the phytase standard assay was performed at a series of temperatures between 37 and 85 °C. The phytases used were purified according to Example 5. V, fungal consensus phytase; T, A. fumigatus 13073 phytase; □, A niger NRRL3135 phytase; O, A. nidulans phytase; ■, A. terreus 9A-1 phytase; •, A terreus cbs phytase.<br>
Figure 4: The pH-dependent activity profile of fungal consensus phytase and of the mutant Q50L, Q50T, and Q50G. The phytase activity was determined using the standard assay in appropriate buffers (see Example 9) at different pH-values. Plot a) shows a comparison of fungal consensus phytase (•) to the mutants Q50L (V), Q50T (▼), and Q50G (O) in percent activity. Plot b) shows a comparison of fungal consensus phytase (O) to mutant Q50L (•) and Q50T (V) using the specific activity of the purified enzymes expressed in H. polymorpha.<br><br>
figure 5: The pH-dependent activity profile of the mutants Q50L, Y51N and ^50TfY51N in comparison to the mutants Q50T and Q50L of fungal consensus )hytase. The phytase activity was determined using the standard assay in appropriate buffers (see Example 9) at different pH-values. Graph a) shows the .nfluence of the mutation Y51N (•) on mutant Q50L (O). Graph b) shows the .nfluence of the same mutation (•) on mutant Q50T (O).<br>
Figure 6: Substrate specificity of fungal consensus phytase and its mutants Q50L, Q50T, and Q50G. The bars represent the relative activity in comparison to the activity with phytic acid (100%) with a variety of known natural and synthetic phosphorylated compounds.<br>
Figure 7: Differential scanning calorimetry (DSC) of fungal consensus phytase and its mutant Q50T. The protein samples were concentrated to ca. 50-60 mg/ml and extensively dialyzed against 10 mM sodium acetate, pH 5.0. A constant heating rate of 10 °C/min was applied up to 90 °C. DSC of consensus phytase Q50T (upper graph) yielded in a melting temperature of 78.9 °C, which is nearly identical to the melting point of fungal consensus phytase (78.1 °C, lower graph).<br>
Examples<br>
Reference Example<br>
Homology Modeling of A. fumigatus and A terreus cbs!16.46 phytase<br>
The amino acid sequences of A. fumigatus and A. terreus cbsll6.46 phytase were compared with the sequence of A. niger NRRL 3135 phytase (see Figure 1) for which the three-dimensional structure had been determined by X-ray crystallography.<br>
A multiple amino acid sequence alignment of A. niger NRRL 3135 phytase, A. fumigatus phytase and A. terreus cbsll6.46 phytase was calculated with the program "PILEUP" (Prog. Menu for the Wisconsin Package, version 8, September 1994, Genetics Computer Group, 575 Science Drive, Madison Wisconcin, USA 53711). The three-dimensional models of A. fumigatus phytase and A. terreus cbsll6.46 phytase were built by using the structure of A. niger NRRL 3135 phytase as template and exchanging the amino acids of A. niger NRRL 3135 phytase according to the sequence alignment to amino acids of A.<br><br>
fumigatus and A. terreus cbsll6.46 phytases, respectively. Model construction and energy optimization were performed by using the program Moloc (Gerber and Muller, 1995). C-alpha positions were kept fixed except for new insertions/deletions and in loop regions distant from the active site.<br>
Only small differences of the modelled structures to the original crystal structure could be observed in external loops. Furthermore the different substrate molecules that mainly occur on the degradation pathway of phytic acid (/nyo-inositol-hexakisphosphate) by Pseudomonas sp. bacterium phytase and, as far as determined, by A. niger NRRL 3135 phytase (Cosgrove, 1980) were constructed and forged into the active site cavity of each phytase structure. Each of these substrates was oriented in a hypothetical binding mode proposed for histidine acid phosphatases (Van Etten, 1982). The scissile phosphate group was oriented towards the catalytically essential His 59 to form the covalent phosphoenzyme intermediate. The oxygen of the substrate phosphoester bond which will be protonated by Asp 339 after cleavage was orientated towards the proton donor. Conformational relaxation of the remaining structural part of the substrates as well as the surrounding active site residues was performed by energy optimization with the program Moloc.<br>
Based on the structure models the residues pointing into the active site cavity were identified. More than half (60%) of these positions were identical between these three phytases, whereas only few positions were not conserved (see Figure 1). This observation could be extended to four additional phytase sequences (A. nidulans, A. terreus 9A1, Talaromyces thermophilus, Myceliophthora thermophila).<br>
Example 1<br>
Alignment of the amino acid sequence of the fungal phytases<br>
The alignment was calculated using the program PILEUP from the Sequence Analysis Package Release 9.0 (Devereux et aL, 1984) with the standard parameter (gap creation penalty 12, gap extension penalty 4). The location of the gaps was refined using a text editor. The following sequences (see Figure 1) without the signal sequence were used for the performance of the alignment starting with the amino acid (aa) mentioned below:<br><br>
phyA gene from Aspergillus terreus 9A-1, aa 27 (Mitchell et al., 1997)<br>
phyAgene from Aspergillus terreus cbsll6.46, aa 27 (van Loon et aL, 1997)<br>
phyA gene from Aspergillus niger var. awamori, aa 27 (Piddington e£ aL, 1993)<br>
p/i^A gene from Aspergillus niger T213, aa 27<br>
phyA gene from Aspergillus niger strain NRRL3135, aa 27 (van Hartingsveldt<br>
etaL, 1993)<br>
p%A gene from Aspergillus fumigatus ATCC 13073, aa 26 (Pasamontes et al.,<br>
1997)<br>
phyA gene from Aspergillus fumigatus ATCC 32722, aa 26 (van Loon et aL,<br>
1997)<br>
phyA gene from Aspergillus fumigatus ATCC 58128, aa 26 (van Loon et aL,<br>
1997)<br>
phyA gene from Aspergillus fumigatus ATCC 26906, aa 26 (van Loon et aL,<br>
1997)<br>
phyA gene from Aspergillus fumigatus ATCC 32239, aa 30 (van Loon et aL,<br>
1997)<br>
phyA gene from Aspergillus nidulans , aa 25 (Roche Nr. R1288, Pasamontes et<br>
aL, 1997a)<br>
phyA gene from Talaromyces thermophilus ATCC 20186, aa 24 (Pasamontes et<br>
aL, 1997a)<br>
phyA gene from Myceliophthora thermophila, aa 19 (Mitchell et aL, 1997)<br>
Table 2 shows the homology of the phytase sequences mentioned above.<br>
Example 2<br>
Calculation of the amino acid sequence of fungal consensus phytases<br>
Using the refined alignment of Example 1 as input, the consensus sequence was calculated by the program PRETTY from the Sequence Analysis Package Release 9.0 (Devereux et aL, 1984). PRETTY prints sequences with their columns aligned and can display a consensus sequence for the alignment. A vote weight that pays regard to the similarity between the amino acid sequences of the phytases aligned were assigned to all sequences. The vote weight was set such as the combined impact of all phytases from one sequence subgroup (same species of origin but different strains), e. g. the amino acid sequences of all phytases from A. fumigatus, on the election was set one, that<br><br>
□leans that each sequence contributes with a value of 1 divided by the number i&gt;f strain sequences (see Table 1). By this means, it was possible to prevent that very similar amino acid sequences, e. g. of the phytases from different A fumigatus strains, dominate the calculated consensus sequence.<br>
The program PRETTY was started with the following parameters: The plurality defining the number of votes below which there is no consensus was set on 2.0. The threshold, which determines the scoring matrix value below which an amino acid residue may not vote for a coalition of residues, was set on 2. PRETTY used the PrettyPep.Cmp consensus scoring matrix for peptides.<br>
Ten positions of the alignment (position 46, 66, 82, 138, 162, 236, 276, 279, 280, 308; Figure 1), for which the program was not able to determine a consensus residue, were filled by hand according to the following rules: if a most frequent residue existed, this residue was chosen (138, 236, 280); if a prevalent group of chemically similar or equivalent residues occurred, the most frequent or, if not available, one residues of this group was selected (46, 66, 82, 162, 276, 308). If there was either a prevalent residue nor a prevalent group, one of the occurring residues was chosen according to common assumption on their influence on the protein stability (279). Eight other positions (132, 170, 204, 211, 275, 317, 384, 447; Figure 1) were not filled with the amino acid residue selected by the program but normally with amino acids that occur with the same frequency as the residues that were chosen by the program. In most cases, the slight underrating of the three A niger sequences (sum of the vote weights: 0.99) was eliminated by this corrections.<br>
Table 3 shows the homology of the calculated fungal consensus phytase amino acid sequence to the phytase sequences used for the calculation.<br>
Example 3<br>
Conversion of the fungal consensus phytase amino acid sequence to a DNA sequence<br>
The first 26 amino acid residues of A. terreus cbsll6.46 phytase were used as signal peptide and, therefore, fused to the N-terminus of all consensus phytases. For this stretch, we used a special method to calculate the corresponding   DNA   sequence.   Purvis   et   al.   (1987)    proposed   that   the<br><br>
incorporation of rare codons in a gene has an influence on the folding efficiency of theprotein. Therefore, at least the distribution of rare codons in the signal sequence of A terreus cbsll6.46, which was used for the fungal consensus phytase and which is very important for secretion of the protein, but converted into the S. cerevisiae codon usage, was transferred into the new signal sequence generated for expression in S. cerevisiae. For the remaining parts of the protein, we used the codon frequency table of highly expressed S. cerevisiae genes, obtained from the GCG program package, to translate the calculated amino acid sequence into a DNA sequence.<br>
The resulting sequence of the fcp gene are shown in Figure 2.<br>
Example 4<br>
Construction and cloning of the fungal consensus phytase genes<br>
The calculated DNA sequence of fungal consensus phytase was divided into oligonucleotides of 85 bp, alternately using the sequence of the sense and the anti-sense strand. Every oligonucleotide overlaps 20 bp with its previous and its following oligonucleotide of the opposite strand. The location of all primers, purchased by Microsynth, Balgach (Switzerland) and obtained in a PAGE-purified form, is indicated in Figure 2.<br>
In three PCR reactions, the synthesized oligonucleotides were composed to the entire gene. For the PCR, the High Fidelity Kit from Boehringer Mannheim (Boehringer Mannheim, Mannheim, Germany) and the thermo cycler The Protokol™ from AMS Biotechnology (Europe) Ltd. (Lugano, Switzerland) were used.<br>
Oligonucleotide CP-1 to CP-10 (Mix 1, Figure 2) were mixed to a concentration of 0.2 pMol/|il per each oligonucleotide. A second oligonucleotide mixture (Mix 2) was prepared with CP-9 to CP-22 (0.2 pMol/(il per each oligonucleotide). Additionally, four short primers were used in the PCR reactions:<br>
CP-a:	Eco RI<br>
5'-TAT ATG AAT TCA TGG GCG TGT TCG TC-3'<br><br><br><br><br>
The resulting PCR product (1.4 kb) was purified as mentioned above, digested with Eco RI, and ligated in an Eco Rl-digested and dephosphorylated pBsk(-)-vector (Stratagene, La Jolla, CA, USA). 1 pi of the ligation mixture was used to transform E. coli XL-1 competent cells (Stratagene, La Jolla, CA, USA). All standard procedures were carried out as described by Sambrook et al. (1987). The constructed fungal consensus phytase gene (fcp) was verified by sequencing (plasmid pBsk"-fcp).<br>
Example 5<br>
Expression of the fungal consensus phytase gene fcp and its variants in Saccharomyces cerevisiae and their purification from culture supernatant<br>
A fungal consensus phytase gene was isolated from the plasmid pBsk"/ep ligated into the Eco RI sites of the expression cassette of the Saccharomyces cerevisiae expression vector pYES2 (Invitrogen, San Diego, CA, USA) or subcloned between the shortened GAPFL (glyceraldhyde-3-phosphate dehydrogenase) promoter and the pho5 terminator as described by Janes et al. (1990). The correct orientation of the gene was checked by PCR. Transformation of S. cerevisiae strains, e. g. INVScl (Invitrogen, San Diego, CA, USA) was done according to Hinnen et al. (1978). Single colonies harboring the phytase gene under the control of the GAPFL promoter were picked and cultivated in 5 ml selection medium (SD-uracil, Sherman et al., 1986) at 30°C<br><br>
under vigorous shaking (250 rpm) for one day. The preculture was then added to 500 ml YPD medium (Sherman et al, 1986) and grown under the same conditions. Induction of the gall promoter was done according to manufacturer's instruction. After four days of incubation cell broth was centrifuged (7000 rpm, GS3 rotor, 15 min, 5°C) to remove the cells and the<br>
supernatant was concentrated by way of ultrafiltration in Amicon 8400 cells (PM30 membranes) and ultrafree-15 centrifugal filter devices (Biomax-30K, Millipore, Bedford, MA, USA). The concentrate (10 ml) was desalted on a 40 ml Sephadex G25 Superfine column (Pharmacia Biotech, Freiburg, Germany), with 10 mM sodium acetate, pH 5.0, serving as elution buffer. The desalted sample was brought to 2 M (NH4)2S04 and directly loaded onto a 1 ml Butyl Sepharose 4 Fast Flow hydrophobic interaction chromatography column (Pharmacia Biotech, Feiburg, Germany) which was eluted with a linear gradient from 2 M to 0 M (NH4)2S04 in 10 mM sodium acetate, pH 5.0.<br>
Phytase was eluted in the break-through, concentrated and loaded on a 120 ml Sephacryl S-300 gel permeation chromatography column (Pharmacia Biotech, Freiburg, Germany). Fungal consensus phytase and fungal consensus phytase 7 eluted as a homogeneous symmetrical peak and was shown by SDS-PAGE to be approx. 95% pure.<br>
Example 6<br>
Expression of the fungal consensus phytase genes fcp and its variants in Hansenula polymorpha<br>
The phytase expression vectors, used to transform H. polymorpha, was constructed by inserting the Eco RI fragment of pBsk"/cp encoding the consensus phytase or a variant into the multiple cloning site of the H. polymorpha expression vector pFPMT121, which is based on an ura3 selection marker and the FMD promoter. The 5' end of the fcp gene is fused to the FMD promoter, the 3' end to the MOX terminator (Gellissen et al, 1996; EP 0299 108 B). The resulting expression vector are designated pFPMT/ep and pBsk" fcp7.<br>
The constructed plasmids were propagated in E. coli. Plasmid DNA was purified using standard state of the art procedures. The expression plasmids were transformed into the H. polymorpha strain RP11 deficient in orotidine-5'-phosphate decarboxylase (ura3) using the procedure for preparation of<br><br>
competent cells and for transformation of yeast as described in Gelissen et ah (1996). Each transformation mixture was plated on YNB (0.14% w/v Difco YNB and 0.5% ammonium sulfate) containing 2% glucose and 1.8% agar and incubated at 37 °C. After 4 to 5 days individual transformant colonies were picked and grown in the liquid medium described above for 2 days at 37 °C. Subsequently, an aliquot of this culture was used to inoculate fresh vials with YNB-medium containing 2% glucose. After seven further passages in selective medium, the expression vector integrates into the yeast genome in multimeric form. Subsequently, mitotically stable transformants were obtained by two additional cultivation steps in 3 ml non-selective liquid medium (YPD, 2% glucose, 10 g yeast extract, and 20 g peptone). In order to obtain genetically homogeneous recombinant strains an aliquot from the last stabilization culture was plated on a selective plate. Single colonies were isolated for analysis of phytase expression in YNB containing 2% glycerol instead of glucose to derepress the fmd promoter. Purification of the fungal consensus phytases was done as described in Example 5.<br>
Example 7<br>
Expression of the fungal consensus genes fcp and its variants in Aspergillus niger<br>
Plasmid pBsk"/ep or the corresponding plasmid of a variant of the fcp gene were used as template for the introduction of a Bsp Hi-site upstream of the start codon of the genes and an Eco RV-site downstream of the stop codon. The Expand™ High Fidelity PCR Kit (Boehringer Mannheim, Mannheim, Germany) was used with the following primers:<br>
Primer Asp-1:<br>
Bsp EI 5'-TAT ATCATGAGC GTG TTC GTC GTG CTA CTG TTC-3'<br>
Primer Asp-2 for cloning of fcp and fcpl:<br>
3'-ACC CGA CTT ACA AAG CGA ATT CTA TAG ATA TAT-5*<br>
EcoW<br><br>
The reaction was performed as described by the supplier. The PCR-amplified fcp gene had a new Bsp HI site at the start codon, introduced by primer Asp-1, which resulted in a replacement of the second amino acid residue glycine by serine. Subsequently, the DNA-fragment was digested with Bsp HI and Eco RV and ligated into the Nco I site downstream of the glucoamylase promoter of Aspergillus niger (glaA) and the Eco RV site upstream of the Aspergillus nidulans tryptophan C terminator (trpC) (Mullaney et aL, 1985). After this cloning step, the genes were sequenced to detect possible failures introduced by PCR. The resulting expression plasmids which basically corresponds to the pGLAC vector as described in Example 9 of EP 684 313, contained the orotidine-5'-phosphate decarboxylase gene (pyr4) of Neurospora crassa as a selection marker.Transformation of Aspergillus niger and expression of the consensus phytase genes was done as described in EP 684 313. The fungal consensus phytases were purified as described in Example 5.<br>
Example 8<br>
Construction of muteins of fungal consensus phytase<br>
To construct muteins for expression in A. niger, S. cerevisiae, or H. polymorpha, the corresponding expression plasmid containing the fungal consensus phytase gene was used as template for site-directed mutagenesis. Mutations were introduced using the "quick exchange™ site-directed mutagenesis kit" from Stratagene ( La Jolla, CA, USA) following the manufacturer's protocol and using the corresponding primers. All mutations made and the corresponding primers are summarized in Table 4. Clones harboring the desired mutation were identified by DNA sequence analysis as known in the art. The mutated phytase were verified by sequencing of the complete gene.<br>
Example 9<br>
Determination of the phytase activity and of the temperature optimum of the consensus phytase and its variants<br>
Phytase activity was determined basically as described by Mitchell et aL (1997). The activity was measured in a assay mixture containing 0.5% phytic<br><br>
acid (*5 mM), 200 mM sodium acetate, pH 5.0. After 15 min incubation at 37 °Cfthe reaction was stopped by addition of an equal volume of 15% trichloroacetic acid. The liberated phosphate was quantified by mixing 100 |il of the assay mixture with 900 jil H20 and 1 ml Of 0.6 M H2S04, 2% ascorbic acid and 0.5% ammonium molybdate. Standard solutions of potassium phosphate were used as reference. One unit of enzyme activity was defined as the amount of enzyme that releases 1 jimol phosphate per minute at 37 °C. The protein concentration was determined using the enzyme extinction coefficient at 280 nm calculated according to Pace et ah (1995): fungal consensus phytase, 1.101; fungal consensus phytase 7, 1.068.<br>
In case of pH-optimum curves, purified enzymes were diluted in 10 mM sodium acetate, pH 5.0. Incubations were started by mixing aliquots of the diluted protein with an equal volume of 1% phytic acid (=10 mM) in a series of different buffers: 0.4 M glycine/HCl, pH 2.5; 0.4 M acetate/NaOH, pH 3.0, 3.5, 4.0, 4.5, 5.0, 5.5; 0.4 M imidazole/HCl, pH 6.0, 6.5; 0.4 M Tris/HCl pH 7.0, 7.5, 8.0, 8.5, 9.0. Control experiments showed that pH was only slightly affected by the mixing step. Incubations were performed for 15 min at 37 °C as described<br>
above.<br>
For determination of the substrate specificities of the phytases, phytic acid in the assay mixture was replaced by 5 mM concentrations of the respective phosphate compounds. The activity tests were performed as described above.<br>
For determination of the temperature optimum, enzyme (100 \il) and substrate solution (100 \xl) were pre-incubated for 5 min at the given temperature. The reaction was started by addition of the substrate solution to the enzyme. After 15 min incubation, the reaction was stopped with trichloroacetic acid and the amount of phosphate released was determined.<br>
The pH-optimum of the original fungal consensus phytase was around pH 6.0-6.5 (70 U/mg). By introduction of the Q50T mutation, the pH-optimum shifted to pH 6.0 (130 U/mg), while the replacement by a leucine at the same position resulted in a maximum activity around pH 5.5 (212 U/mg). The exchange Q50G resulted in a pH-optimum of the activity above pH 6.0 (see Figure 4). The exchange of tyrosine at position 51 with asparagine resulted in a relative increase of the activity below pH 5.0 (see Figure 5). Especially by the<br><br>
Q50L mutation, the specificity for phytate of fungal consensus phytase was drastically increased (see Figure 6).<br>
The temperature optimum of fungal consensus phytase (70 °C) was 15-25 °C higher than the temperature optimum of the wild-type phytases (45-55 °C) which were used to calculate the consensus sequence (see Table 5 and Figure 3).<br>
Example 10<br>
Determination of the melting point by differential scanning calorimetry (DSC)<br>
In order to determine the unfolding temperature of the fungal consensus phytases, differential scanning calorimetry was applied as previously published by Brugger et al. (1997). Solutions of 50-60 mg/ml homogeneous phytase were used for the tests. A constant heating rate of 10 °C/min was applied up to 90 °C.<br>
The determined melting points clearly show the strongly improved thermostability of the fungal consensus phytase in comparison to the wild-type phytases (see Table 5 and Figure 7). Figure 7 shows the melting profile of fungal consensus phytase and its mutant Q50T. Its common melting point was determined between 78 to 79 °C.<br>
References:<br>
van den Burg, B., Vriend, G., Veltman, O. R., Venema &amp; G., Eijsink, V. G. H. (1998). Engineering an enzyme to resist boiling. Proc. Natl Acad. Sci. (USA) 95, 2056-2060.<br>
Akanuma, S., Yamagishi, A., Tanaka, N. &amp; Oshima, T. (1998). Serial increase in the thermal stability of 3-isopropylmalate dehydrogenase from Bacillus subtilis by experimental evolution. Prot. Sci, 7, 698-705.<br>
Matthews, B. W. (1993). Structural and genetic analysis of protein stability. Annu. Rev. Biochem. 62, 139-160.<br><br>
Serrano, L., Day, A. G. &amp; Fersht, A. R. (1993). Step-wise mutation of barnase to binase. A procedure for engineering increased stability of proteins and an experimental analysis of the evolution of protein stability. J. Mol Biol 233, 305-312.<br>
Matthews, B. W. (1987a). Genetic and structural analysis of the protein stability problem. Biochemistry 26, 6885-6888.<br>
Sauer, R., Hehir, K., Stearman, R., Weiss, M., Jeitler-Nilsson, A., Suchanek, E. &amp; Pabo, C. (1986). An engineered intersubunit disulfide enhances the stability and DNA binding of the N-terminal domain of X-repressor. Biochemistry 25, 5992-5999.<br>
Margarit, I., Campagnoli, S., Frigerio, F., Grandi, G., Fillipis, V. D. &amp; Fontana, A. (1992). Cumulative stabilizing effects of glycine to alanine substitutions in Bacillus subtilis neutral protease. ProL Eng. 5, 543-550.<br>
Matthews, B. W., Nicholson, H. &amp; Becktel, W. (1987). Enhanced protein thermostability from site-directed mutations that decrease the entropy of unfolding. Proc. Natl Acad. Sci. (USA) 84, 6663-6667.<br>
Blaber, M., Lindstrom, J. D., Gassner, N., Xu, J., Heinz, D. W. &amp; Matthews, B. W. (1993). Energetic cost and structural consequences of burying a hydroxyl group within the core of a protein determined from Ala-*Ser and Val-^Thr substitutions in T4 lysozyme. Biochemistry 32, 11363-11373.<br>
Karpusas, M., Baase, W. A., Matsumura, M. &amp; Matthews, B. W. (1989). Hydrophobic packing in T4 lysozyme probed by cavity-filling mutants. Proc. Natl Acad. Scl(USA) 86, 8237-8241.<br>
Munoz, V. &amp; Serrano, L. (1995). Helix design, prediction and stability. Curr. Opin. Biotechnol 6, 382-386.<br>
Arase, A., Yomo, T., Urabe, I., Hata, Y., Katsube, Y. &amp; Okada, H. (1993). Stabilization of xylanase by random mutagenesis. FEBS Lett. 316, 123-127.<br>
Risse, B., Stempfer, G., Rudolph, R., Schumacher, G. &amp; Jaenicke, R. (1992). Characterization of the stability effect of point mutations of pyruvate oxidase from Lactobacillus plantarum: protection of the native state by modulating coenzyme binding and subunit interaction. Prot. Scl 1, 1710-1718.<br><br>
Imanaka, T., Shibazaki, M. &amp; Takagi, M. (1986). A new way of enhancing the thermostability of proteases. Nature 324, 695-697.<br>
Pantoliano, M. W., Landner, R. C, Brian, P. N., Rollence, M. L., Wood, J. F. &amp; Poulos, T. L. (1987). Protein engineering of subtilisin BPN': enhanced stabilization through the introduction of two cysteines to form a disulfide bond. Biochemistry 26, 2077-2082.<br>
Steipe, B., Schiller, B., Plueckthun, A. &amp; Steinbach, S. (1994). Sequence statistics reliably predict stabilizing mutations in a protein domain. J. Mol. Biol 240, 188-192.<br>
Mitchell, D. B., Vogel, K., Weimann, B. JM Pasamontes, L. &amp; van Loon, A. P. G. M. (1997) The phytase subfamily of histidine acid phosphatases: isolation of genes for two novel phytases from the fungi Aspergillus terreus and Myceliophthora thermophila, Microbiology 143, 245-252.<br>
van Loon, A. P. G. M., Simoes-Nunes, C., Wyss, M., Tomschy, A., Hug, D., Vogel, K. &amp; Pasamontes, L. (1997). A heat resistant phytase of Aspergillus fumigatus with superior performance in animal experiments. Phytase optimization and natural variability. In Proceedings book of the workshop on plant phytate and phytases. Kluwer Academic Press.<br>
Pasamontes, L., Haiker, M., Wyss, M., Tessier, M. &amp; van Loon, A. P. G. M. (1997) Cloning, purification and characterization of a heat stable phytase from the fungus Aspergillus fumigatus, AppL Environ. Microbiol. 63, 1696-1700.<br>
Pasamontes, L., Haiker, M., Henriquez-Huecas, M., Mitchell, D. B. &amp; van Loon, A. P. G. M. (1997a). Cloning of the phytases from Emericella nidulans and the thermophilic fungus Talaromyces thermophilus. Biochim. Biophys. Acta 1353, 217-223.<br>
Piddington, C. S., Houston, C. S., Paloheimo, M., Cantrell, M., Miettinen-Oinonen, A. Nevalainen, H., &amp; Rambosek, J. (1993) The cloning and sequencing of the genes encoding phytase (phy) and pH 2.5-optimum acid phosphatase (aph) from Aspergillus nigervsx. awamori. Gene 133, 55-62.<br><br>
van Hartingsveldt, W., van Zeijl, C. M. F., Harteveld, G. M., Gouka, R. J., Suykerbuyk, M. E. G„ Luiten, R. G. M., van Paridon, P. A., Selten, G. C. M., Veenstra, A. E., van Gorcom, R. F. M., &amp; van den Hondel, C. A. M. J. J. (1993) Cloning, characterization and overexpression of the phytase-encoding gene (phyA) of Aspergillus niger. Gene 127, 87-94.<br>
Gerber, P. and Miiller, K (1995) Moloc molecular modeling software. J. Comput. Aided Mol Des. 9, 251-268<br>
Van Etten, R.L. (1982) Human prostatic acid phosphatase: a histidine phosphatase. Ann. NY Acad. Sci. 390,27-50<br>
Cosgrove, D.J. (1980) Inositol phosphates - their chemistry, biochemistry and physiology: studies in organic chemistry, chapter 4. Elsevier Scientific Publishing Company, Amsterdam, Oxford, New York.<br>
Devereux, J., Haeberli, P.&amp; Smithies, O. (1984) A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12, 387-395.<br>
Purvis, I. J., Bettany, A. J. E., Santiago, T. C, Coggins, J. R., Duncan, K., Eason, R. &amp; Brown, A. J. P. (1987). The efficiency of folding of some proteins is increased by controlled rates of translation in vivo. J. Mol. Biol. 193, 413-417.<br>
Sambrook, J., Fritsch, E. F. &amp; Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.<br>
Janes, M., Meyhack, B., Zimmermann, W. &amp; Hinnen, A. (1990) The influence of GAP promoter variants on hirudine production, average plasmid copy number and cell growth in Saccharomyces cerevisiae. Curr. Genet. 18, 97-103.<br>
Hinnen, A., Hicks, J. B. &amp; Fink, G, R. (1978) Transformation of yeast. Proc. Natl. Acad. Sci. USA 75, 1929-1933.<br>
Sheman, J. P., Finck, G. R. &amp; Hicks, J. B. (1986) Laboratory course manual for methods in yeast genetics. Cold Spring Harbor University.<br>
Gellissen, G., Piontek, M., Dahlems, U., Jenzelewski, V., Gavagan, J. E., DiCosimo, R., Anton, D. I. &amp; Janowicz, Z. A. (1996) Recombinant Hansenula polymorpha as a biocatalyst: coexpression of the spinach glycolate oxidase (GO) and the S. cerevisiae catalase T (CTT1) gene. Appl. Microbiol. BiotechnoL 46, 46-54.<br><br>
Mullaney, E. J., Hamer, J. E., Roberti, K. A., Yelton, M. M. &amp; Timberlake, W. E. (1985) Primary structure of the trpC gene from Aspergillus nidulans. Mol. Gen. Genet. 199, 37-46.<br>
Pace, N. C., Vajdos, F., Fee, L., Grimsley, G. &amp; Gray, T. (1995). How to measure and predict the molar absorption coefficient of a protein. ProL Sci. 4, 2411-2423.<br>
Brugger, R., Mascarello, F., Augem, S., van Loon, A. P. G. M. &amp; Wyss, M. (1997). Thermal denaturation of fungal phytases and pH 2.5 acid phosphatase studied by differential scanning calorimetry. In Proceedings book on the workshop on plant phytate and phytase. Kluwer Academic Press.<br><br>
Table 1<br>
Aspergillus terreus 9A-1 phytase:	0.50<br>
Aspergillus terreus cbsll6.46 phytase:	0.50<br>
Aspergillus niger var. awamori phytase:	0.3333<br>
Aspergillus niger T213 phytase:	0.3333<br>
Aspergillus niger NRRL3135 phytase:	0.3333<br>
Aspergillus fumigatus ATCC 13073 phytase:	0.20<br>
Aspergillus fumigatus ATCC 32722 phytase:	0.20<br>
Aspergillus fumigatus ATCC 58128 phytase:	0.20<br>
Aspergillus fumigatus ATCC 26906 phytase:	0.20<br>
Aspergillus fumigatus ATCC 32239 phytase:	0.20<br>
Aspergillus nidulans phytase:	1.00<br>
Talaromyces thermophilus ATCC 20186 phytase:	1.00<br>
Myceliophthora thermophila phytase:	1.00<br><br><br><br><br><br><br><br>
Claims<br>
1. A process for the preparation of a consensus protein, whereby such process is characterized by the following steps:<br>
a)	at least three, preferably four amino acid sequences are aligned by any<br>
standard alignment program known in the art;<br>
b)	amino acids at the same position according to such alignment are<br>
compared regarding their evolutionary similarity by any standard<br>
program known in the art, whereas the degree of similarity provided by<br>
such a program which defines the least similarity of the amino acids that<br>
is used for the determination of an amino acid of corresponding positions<br>
is set to a less stringent number and the parameters are set in such a way<br>
that it is possible for the program to determine from only 2 identical<br>
amino acids at a corresponding position an amino acid for the consensus<br>
protein; however, if among the compared amino acid sequences are<br>
sequences that show a much higher degree of similarity to each other than<br>
to the residual sequences, these sequences are represented by their<br>
consensus sequence determined as defined in the same way as in the<br>
present process for the consensus sequence of the consensus protein or a<br>
vote weight of 1 divided by the number of such sequences is assigned to<br>
every of those sequences.<br>
c)	in case no common amino acid at a defined position is identified by the program, any of the amino acids, preferably the most frequent amino acid of all such sequences is selected;<br>
d)	once the consensus sequence has been defined, such sequence is back-translated into a DNA sequence, preferably by using a codon frequency table of the organism in which expression should take place;<br>
e)	the DNA sequence is synthesized by methods known in the art and used either integrated into a suitable expression vector or by itself to transform an appropriate host cell;<br><br>
f)   the transformed host cell is grown under suitable culture conditions and the consensus protein is isolated from the host cell or its culture medium by methods known in the art.<br>
2.	A process as claimed in claim 1 wherein the program used for the<br>
comparison of amino acids at a defined position regarding their evolutionary<br>
similarity is the program "PRETTY".<br>
3.	A process as claimed in claim 1 or 2, wherein the defined protein family is<br>
the family of phytases.<br>
4.	A process as claimed in claim 3, wherein the phytases are of fungal origin.<br>
5.	A process as claimed in any one of claims 1 to 4, wherein the host cell is of<br>
eukaryotic origin.<br>
6.	A process as claimed in claim 5, wherein eukaryotic means fungal,<br>
preferably Aspergillus or yeast, preferably Saccharomyces or Hansenula.<br>
7.	A consensus protein obtainable, preferably obtained by a process as claimed<br>
in any one of claims 1 to 6.<br>
8.	A consensus protein which has the amino acid sequence shown in Figure 2 or<br>
any variants or muteins thereof.<br>
9.	A mutein of the consensus protein of claim 8 characterized therein that in<br>
the amino acid sequence of Figure 2 the following replacements have been effected<br>
Q50L, Q50T, Q50G, Q50T-Y51N or Q50L-Q51N.<br>
10.	A food, feed or pharmaceutical composition comprising a consensus protein<br>
as claimed in any of the claims 7 to 9.<br>
11.	A process    for the prepar-ation of  a consensus protein<br>
substantially    a s  herein descri-bed with reference  to  the<br>
accompanying drawings.<br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1597-mas-1998-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1597-mas-1998-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1jb3JyZXNwb25kZW5jZSBwby5wZGY=" target="_blank" style="word-wrap:break-word;">1597-mas-1998-correspondence po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">1597-mas-1998-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">1597-mas-1998-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">1597-mas-1998-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">1597-mas-1998-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTU5Ny1tYXMtMTk5OC1mb3JtIDQucGRm" target="_blank" style="word-wrap:break-word;">1597-mas-1998-form 4.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="235015-payment-transaction-system-and-method.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="235017-a-method-of-configuring-a-lens-surface-and-a-lens-cone-gured-and-produced-by-the-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>235016</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1597/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>17-Jul-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DSM IP ASSETS B V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>HET OVERLOON 1, 6411 TE HEERLEN,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>MARTIN LEHMANN</td>
											<td>101A RIEHENSTRASSE, D-79594 INZLINGEN,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/16</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>97112688.3</td>
									<td>1997-07-24</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/235016-a-fungal-consensus-phytase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:33:53 GMT -->
</html>
