<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/235081-a-composition-comprising-an-isolated-m-tuberculosis-peptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:36:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 235081:A COMPOSITION COMPRISING AN ISOLATED M TUBERCULOSIS PEPTIDE .</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A COMPOSITION COMPRISING AN ISOLATED M TUBERCULOSIS PEPTIDE .</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method is provided for identifying mycobacterial genes the expression of which is down-regulated during a stationary phase culture of mycobacteria under nutrient-starving conditions when compared with an exponential phase culture of mycobacteria under culture conditions that are not nutrient-starving and that support exponential growth of said mycobacteria. The described method optionally provides for identifying mycobacterial genes that are simultaneously down-regulated under low DOT conditions. The down-regulated genes of the present invention form the basis of nucleic acid vaccines, or provide targets to allow preparation of attenuated mycobacteria for vaccines against mycobacterial infections. Similarly, peptides encoded by said down-regulated genes are employed in vaccines. In a further embodiment, the identified genes/peptides provide the means for identifying the presence of a mycobacterial infection in a clinical sample by nucleic acid probe or antibody detection.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>AN ISOLATED M. TUBERCULOSIS PEPTIDE<br>
The present invention relates to a method of identilying a gene in mycobacteria the<br>
expression of which is down-regulated during mycobacterial latency, to the isolated<br>
peptide products, variants, derivatives or fragments thereof and to inhibitors thereof,<br>
to antibodies that bind to said peptides, variants, derivatives or fragments, to DNA<br>
and RNA vectors that express said polypeptide, variants, derivatives or fragments,<br>
to attenuated mycobacteria in which the activity of at least one of said genes has<br>
been modified, to vaccines against mycobacterial infections, and to methods of<br>
detecting the presence of a mycobacterial infection.<br>
Many microorganisms are capable of forming intracellular infections. These<br>
include: infections caused by species of Salmonella, Yersinia, Shigella,<br>
Campylobacter, Chlamydia and Mycobacteria. Some of these infections are<br>
exclusively intracellular, others contain both intracellular and extracellular<br>
components. However, it is the intracellular survival cycle of bacterial infection<br>
which is suspected as a main supportive factor for disease progression.<br>
Generally, these microorganisms do not circulate freely in the body, for example,<br>
in the bloodstream, and are often not amenable to drug treatment regimes. Where<br>
drugs are available, this problem has been exacerbated by the development of<br>
multiple drug resistant microorganisms.<br>
A number of factors have contributed to the problem of microbial resistance. One<br>
is the accumulation of mutations over time and the subsequent horizontal and<br>
vertical transfer of the mutated genes to other organisms. Thus, for a given<br>
pathogen, entire classes of antibiotics have been rendered inactive. A further factor<br>
has been the absence of a new class of antibiotics in recent years. The emergence<br>
of multiple drug-resistant pathogenic bacteria represents a serious threat to public<br>
health and new forms of therapy are urgently required.<br>
For similar reasons, vaccine therapies have not proved effective against such<br>
intracellular microorganisms. Also, increased systemic concentration of antibiotics<br>
to improve bioavailability within cells may result in severe side effects.<br>
Mycobacterium tuberculosis (TB) and closely related species make up a small<br>
group of mycobacteria known as the Mycobacterium tuberculosis complex (MTC).<br>
This group comprises five species M. tuberculosis, M. microti, M. bovis, M. caneti,<br>
and M. africanum which are the causative agent in the majority of tuberculosis (TB)<br>
cases throughout the world.<br>
M. tuberculosis is responsible for more than three million deaths a year world-wide.<br>
Other mycobacteria are also pathogenic in man and animals, for example M. avium<br>
subsp. paratuberculosis which causes Johne's disease in ruminants, M. bovis which<br>
causes tuberculosis in cattle, M. avium and M. intracellulars which cause<br>
tuberculosis in immunocompromised patients (eg. AIDS patients, and bone marrow<br>
transplant patients) and M. leprae which causes leprosy in humans. Another<br>
important mycobacterial species is M. vaccae.<br>
M. tuberculosis infects macrophage cells within the body. Soon after macrophage<br>
infection, most M. tuberculosis bacteria enter and replicate within cellular<br>
phagosome vesicles, where the bacteria are sequestered from host defences and<br>
extracellular factors.<br>
It is the intracellular survival and multiplication or replication of bacterial infection<br>
which is suspected as a main supportive factor for mycobacterial disease<br>
progression.<br>
A number of drug therapy regimens have been proposed for combatting M.<br>
tuberculosis infections, and currently combination therapy including the drug<br>
isoniazid has proved most effective. However, one problem with such treatment<br>
regimes is that they are long-term, and failure to complete such treatment can<br>
promote the development of multiple drug resistant microorganisms.<br>
A further problem is that of providing an adequate bioavailability of the drug within<br>
the cells to be treated. Whilst it is possible to increase the systemic concentration<br>
of a drug (eg. by administering a higher dosage) this may result in severe side<br>
effects caused by the increased drug concentration.<br>
The effectiveness of vaccine prevention against M. tuberculosis has varied widely.<br>
The current M. tuberculosis vaccine, BCG, is an attenuated strain of M. bovis. It is<br>
effective against severe complications of TB in children, but it varies greatly in its<br>
effectiveness in adults particularly across ethnic groups. BCG vaccination has been<br>
used to prevent tuberculous meningitis and helps prevent the spread of M.<br>
tuberculosis to extra-pulmonary sites, but does not prevent infection.<br>
The limited efficacy of BCG and the global prevalence of TB has led to an<br>
international effort to generate new, more effective vaccines. The current paradigm<br>
is that protection will be mediated by the stimulation of a Th1 immune response.<br>
BCG vaccination in man was given orally when originally introduced, but that route<br>
was discontinued because of loss of viable BCG during gastric passage and of<br>
frequent cervical adenopathy. In experimental animal species, aerosol or intra-<br>
tracheal delivery of BCG has been achieved without adverse effects, but has varied<br>
in efficacy from superior protection than parenteral inoculation in primates, mice and<br>
guinea pigs to no apparent advantage over the subcutaneous route in other studies.<br>
Conventional mycobacterial vaccines, including BCG, protect against disease and<br>
not against infection. Ideally a new mycobacterial vaccine will impart sterile<br>
immunity post-exposure vaccine capable of boosting the immune system to<br>
kill latent mycobacteria or prevent reactivation to active disease-causing<br>
microorganisms would also be valuable against latent infection.<br>
There is therefore a need for an improved and/or alternative vaccine or therapeutic<br>
agent for combatting mycobacterial infections.<br>
An additional major problem associated with the control of mycobacterial infections,<br>
especially M. tuberculosis infections, is the presence of a large reservoir of<br>
asymptomatic individuals infected with mycobacteria. Dormant mycobacteria are<br>
, even more resistant to front-line drugs.<br>
Infection with mycobacteria (eg. M. tuberculosis) rarely leads to active disease, and<br>
most individuals develop a latent infection which may persist for many years before<br>
reactivating to cause disease (Wayne, 1994). The current strategy for controlling<br>
such infection is early detection and treatment of patients with active disease.<br>
Whilst this is essential to avoid deaths and control transmission, it has no effect on<br>
eliminating the existing reservoir of infection or on preventing new cases of disease<br>
through reactivation.<br>
Furthermore, conventional methods for the detection of a latent mycobacterial<br>
infection by skin testing may be compromised. For example, current TB detection<br>
methods based on tuberculin skin testing are compromised by BCG vaccination and<br>
by exposure to environmental mycobacteria.<br>
Thus, new strategies are required for more effective diagnosis, treatment and<br>
prevention of mycobacterial latent infection.<br>
To develop specific strategies for addressing latent mycobacterial infection it is<br>
necessary to elucidate the physiological, biochemical and molecular properties of<br>
these microorganisms.<br>
However, at present, there is no suitable in vivo model for studying mycobacterial<br>
latent infection and such a model is unlikely to provide sufficient microbial material<br>
to enable detailed analysis of the physiological and molecular changes that occur.<br>
In this respect, conventional mycobacterial culture systems for analysing gene and<br>
protein expression profiles have been based on simple, crude batch-type systems,<br>
such as those described in Sherman, D. R et al (2001) PNAS Vol. 98, No. 13, pp.<br>
7534-7539; Imboden, P. (1998) Gene 213, pp. 107-117; Boon, C. et al (2001) J.<br>
Bacteriol. Vol. 183, No. 8, pp. 2672-2676; Cunningham, A. F et al (1998), J.<br>
Bacteriol. Vol. 180, No. 4, pp. 801-808; and Murugasu-Oei, B. et al (1999), Mol.<br>
Gen. Genet., Vol. 262, pp. 677-682. In these crude batch systems, mycobacterial<br>
growth follows a typical sigmoid growth curve involving an exponential growth<br>
phase and a stationary phase. The transition from exponential phase to stationary<br>
phase involves rapid and transient switches in terms of gene and protein<br>
expression, which switches are initiated by a complex set of undefined or poorly<br>
defined interactive stimuli as the mycobacteria become starved of essential<br>
nutrients.<br>
In summary, studies to date have used either static cultures that allow tubercle<br>
bacilli to generate oxygen-depletion-gradients and enter a non-replicating persistent<br>
state in the sediment layer, or agitated sealed liquid cultures (Wayne and Lin, 1982;<br>
Cunningham and Spreadbury, 1998; Wayne and Hayes, 1996). Transition to a<br>
non-replicating persistent state in these models coincides with a shift-down to<br>
glyoxylate metabolism, resistance to isoniazid and rifampicin and susceptibility to<br>
the anaerobic bactericidal action of metronidazole (Wayne and Hayes, 1996).<br>
Such studies are poorly defined and controlled, and experiments relying on<br>
self-generated oxygen-depletion gradients have yielded inconsistent results.<br>
Furthermore, these studies do not consider other physicochemical stimuli that may<br>
be important in vivo, and have been conducted over a short duration, in many cases<br>
2-3 weeks.<br>
In view of the above, there is a need for a defined and controlled model for studying<br>
mycobacterial (eg. TB) persistence, which simulates key features of the in vivo<br>
environment.<br>
According to a first aspect of the invention there is provided an isolated<br>
mycobacterial peptide, or a fragment or derivative or variant thereof, wherein the<br>
peptide is encoded by a mycobacterial gene the expression of which gene is down-<br>
reguTated during a stationary phase culture of mycobacteria under nutrient-starving<br>
culture conditions when compared with an exponential phase culture of<br>
mycobacteria under culture conditions that are not nutrient-starving and which<br>
support exponential growth of said mycobacteria.<br>
The following various embodiments (eg. preferred culture conditions, and sampling<br>
times) described for the first aspect of the present invention apply equally to the<br>
second and subsequent aspects of the present invention.<br>
During infection, mycobacteria (eg. M. tuberculosis) encounter a dynamic host<br>
environment and modulate the expression of genes required for infection.<br>
The selection conditions of the present invention permit identification of<br>
mycobacterial genes that are not required for maintenance of mycobacterial<br>
. latency. Such selection conditions therefore permit the identification of genes that<br>
may be expressed in vivo during active mycobacterial infection, or during/following<br>
re-activation of a mycobacterium from a latent state. Active infection refers to an<br>
infectious state in which the mycobacteria demonstrate high metabolic activity and<br>
undergo cell division. Latency is discussed in more detail below.<br>
The term latency is synonymous with persistence, and describes a reversible state<br>
of low metabolic activity in which mycobacterial cells can survive for extended<br>
periods without cell division, During latency (ie. latent infection), the clinical<br>
symptoms associated with a mycobacterial infection do not become manifest, and<br>
the host suffers no significant ill.<br>
However, re-activation of latent mycobacteria may be induced by environmental<br>
stimuli, resulting in the development of an active mycobacterial infection with the<br>
associated clinical symptoms. The present inventors believe that stimuli such as an<br>
increase in nutrient availability, optionally together with an increase in the local<br>
dissolved oxygen concentration may induce re-activation.<br>
For example, re-activation of latent mycobacteria may occur when previously<br>
limiting nutrients, and optionally oxygen, become available. This may happen, for<br>
example, following breakdown of a mycobacterial infection granuloma structure.<br>
Studies of TB pathogenesis have established that tubercle bacilli reside in a number<br>
of different locations in the lung, for example lining the walls of cavitary lesions<br>
where the close proximity to bronchioles provides a source of air; within<br>
macrophages where oxygen availability may be more limited or within granulomas.<br>
Studies of granuloma structure, where the bacilli are encased by layers of<br>
macrophages and lymphocytes and may persist for prolonged periods, have led<br>
scientists to the conclusion that the environment within a granuloma is most likely<br>
to be anoxic. Therefore TB may encounter a range of atmospheric conditions from<br>
the point of transmission and inhalation through to long-term persistence in a<br>
granuloma.<br>
The present inventors believe that mycobacteria such as M. tuberculosis must be<br>
able to adapt and respond to changes in nutrient, and optionally oxygen, availability<br>
in order to cause infection. The initial stages of infection, which include<br>
dissemination and phagocytosis, occur under aerobic conditions, and tubercle bacilli<br>
replicate rapidly in the aerobic environments of the cavitary lesion as well as within<br>
macrophages.<br>
In vitro studies have demonstrated that mycobacteria such as M. tuberculosis can<br>
adapt and survive under nutrient- and oxygen-depleted conditions, and can grow<br>
over a range of oxygen tensions. Adaptation to carbon starvation, and optionally to<br>
a low dissolved oxygen tension, triggers transition to a non-replicating persistent<br>
state in vitro that may be analogous to latency in vivo.<br>
Antigens repressed during latency and that have been identified according to the<br>
present invention may play an early and important role in the development of an<br>
effective immune response against replicating bacilli during the active stages of<br>
disease, and consequently represent good vaccine candidates, in addition, genes<br>
repressed under latency conditions are important therapeutic targets for preventing<br>
the establishment, spread and reactivation of disease. The identified genes are<br>
likely to be required for re-activation of latent mycobacteria, and replication of said<br>
re-activated bacilli. They are therefore key targets for the development of<br>
therapeutic agents and post-exposure vaccines to prevent re-activation of latent<br>
infection. Alternatively, said genes may be exploited in a treatment regime designed<br>
to trigger re-activation of latent bacilli. When re-activated, tubercle bacilli are more<br>
susceptible to treatment regimes. Thus, the antigens and genes of the present<br>
invention may form the basis for vaccines against pre- and post-mfections by<br>
mycobacteria.<br>
The term "nutrient-starving" in the context of the present invention means that the<br>
concentration of the primary carbon, and preferably the primary energy source, is<br>
insufficient to support optimal "exponential growth" of the mycobacteria. "Nutrient-<br>
starving" is a term that is generally associated with the stationary phase of a batch<br>
culture growth curve, or with the onset of late exponential/early stationary phase<br>
when the concentration of carbon is approaching depletion and starts to restrict the<br>
growth rate. Essential nutrients other than carbon (eg. nitrogen, iron, and oxygen)<br>
may also limit growth. Under such conditions the mycobacteria become<br>
metabolically stressed, rather than simply reduced in growth rate. By comparison,<br>
"nutrient limiting" is a term associated with continuous culture where growth is<br>
controlled/limited by the rate of addition of an essential nutrient to the culture<br>
system. .<br>
., In more detail, exponential growth is that period of growth that is associated with a<br>
logarithmic increase in mycobacterial cell mass (also known as the "log" phase)<br>
when the bacteria are multiplying at their maximum specific growth rate for the<br>
prevailing culture conditions. During this period of growth the concentrations of<br>
essential nutrients diminish and those of end-products increase. However, once the<br>
primary carbon and/or primary energy source falls to below a critical level, it is no<br>
longer possible for all of the mycobacterial cells within the culture to obtain sufficient<br>
carbon and/or energy needed to support optimal cellular function and cell division.<br>
Once this occurs, exponential growth stops and the mycobacteria enter stationary<br>
phase. Growth may also cease due to the accumulation of inhibitory secondary<br>
metabolites, or as a result of pH changes.<br>
Carbon starvation normally refers to a supply of exogenous carbon that is<br>
insufficient to enable the bacteria to grow/replicate. However, there may be other<br>
energy sources (eg. endogenous reserves, secondary metabolites) that are<br>
available to maintain essential cellular functions and viability without supporting<br>
growth. Thus, carbon starvation is associated with a stationary phase condition in<br>
which the carbon source has become depleted and bacterial growth has<br>
substantially ceased.<br>
During onset of stationary phase, DNA synthesis may continue for some time after<br>
net increase in cell mass has ceased, and the mycobacteria may divide to produce<br>
cells small in size and low in RNA content. Many proteins, including extracellular<br>
enzymes, may be synthesized during the stationary phase.<br>
The onset of stationary phase vis-a-vis addition of a mycobacterial inoculum to the<br>
culture vessel will depend on a number of factors such as the particular<br>
mycobacterial species/strain, the composition of the culture media (eg. the<br>
particular primary carbon and energy source), and the physical culture parameters<br>
employed.<br>
However, as a guide, the end of exponential phase and the onset of stationary<br>
phase generally corresponds to that point in the growth phase associated with the<br>
maximum number of viable counts of mycobacteria.<br>
In use of the present invention, the exponential phase mycobacterial cells are<br>
harvested from the culture vessel at a point in the growth phase before the<br>
maximum number of total viable counts has been achieved. This point in the growth<br>
phase may be mimicked under continuous culture conditions employing a steady<br>
state growth rate approximating µmax and providing a generation time of<br>
approximately 18-24 hours. In a preferred embodiment, the exponential phase<br>
mycobacterial cells are harvested when a value of between 2 and 0.5 (more<br>
preferably between 1 and 0.5) log units of viable counts per ml of culture medium<br>
less than the maximum number of viable counts per ml of culture medium has been<br>
achieved. Thus, the "exponential" phase cells are generally harvested during mid-<br>
log phase.<br>
For example, if the maximum viable count value is 1x1010 per ml, then the<br>
"exponential" phase cells would be preferably harvested once a value of between<br>
1x108 and 1x109.5 (more preferably between 1x109 and 1x109.5) viable counts per ml<br>
has been achieved. In the case of M. tuberculosis, this would be approximately 3-<br>
10, preferably 4-7 days post-inoculation.<br>
Similarly, in use of the present invention, the stationary phase mycobacterial cells<br>
are harvested from the culture vessel at a point in the growth phase after the<br>
maximum number of total viable counts has been achieved. This point in the growth<br>
phase may be mimicked under continuous culture conditions supporting a<br>
generation time of at least 3 days. In a preferred embodiment, the stationary phase<br>
mycobacterial cells are harvested when the viable counts per ml of culture medium<br>
has fallen to between 0.5 and 3 (more preferably between 1.5 and 2.5) log units<br>
less than the maximum number of viable counts per ml of culture medium. Thus,<br>
the "stationary" phase cells are generally harvested during mid- to late-stationary<br>
phase.<br>
For example, if the maximum viable count value is 1x1010 per ml, then the stationary<br>
phase cells would be preferably harvested once the viable count number had fallen<br>
to a value of between 1x109.5 and 1x107 (more preferably between 1x108.5 and<br>
1*107,5) viable counts per ml. In the case of M. tuberculosis, this would be<br>
approximately day 25-60, preferably day 30-50, more preferably day 40-50, post-<br>
inoculation. For mycobacteria generally, the stationary phase cells are preferably<br>
harvested at least 25, preferably at least 30, more preferably at least 40 days post-<br>
inoculation. Longer post-inoculation harvesting times of at least 100 days, even at<br>
least 150 days may be employed. The above harvesting times apply to all aspects<br>
of the present invention.<br>
The preferred culture method employed by the present invention is that of batch<br>
fermenter culture. In contrast to the crude prior art Wayne-type, simple batch<br>
culture systems, the batch fermenter system of the present invention permits careful<br>
monitoring of growth culture parameters such as pH, temperature, available<br>
nutrients, and dissolved oxygen tension (DOT). In particular, temperature and DOT<br>
may be strictly controlled.<br>
Thus, in use of the present method it is possible to ensure that the principal latency<br>
induction parameter employed is starvation of the primary carbon, and preferably<br>
the starvation of the primary carbon and energy source. In contrast, the crude prior<br>
art, simple batch systems simultaneously expose the cultured mycobacteria to a<br>
complex range of interactive environmental stimuli, which stimuli may obscure or<br>
modify the cellular effects associated with a single, principal stimulus in isolation.<br>
Thus, the present invention substantially avoids the accidental induction or up-<br>
regulation of genes which are solely responsive to other environmental switches<br>
may be substantially prevented. Accordingly, false-positive identification of genes<br>
that are down-regulated under conditions unrelated to carbon starvation and/or<br>
energy limitation may be substantially avoided.<br>
Suitable media for culturing mycobacteria are described in Wayne, L.G. (1994) [in<br>
Tuberculosis: Pathogenesis, Protection, and Control published by the American<br>
Society for Microbiology, pp. 73-83]. These include Middlebrook 7H9 Medium [see<br>
Barker, LP., et al. (1998) Molec. Microbiol., vol. 29(5), pp. 1167-1177; and<br>
WO00/52139 in the name of the present Applicant].<br>
In use of the method, the starting concentration of the primary carbon source (and<br>
preferably the primary energy source) is at least 0.5, preferably at least 1 gl-1 of<br>
culture medium. Such concentrations are generally considered to be non nutrient-<br>
starving. Similarly, the onset of the stationary phase is generally associated with a<br>
primary carbon and energy source concentration of less than 0.5, preferably less<br>
than 0.2, and more preferably less than 0.1 gl-1 of culture medium<br>
In a preferred embodiment, the primary carbon and energy source is glycerol. The<br>
starting concentration of this component is at least 1, preferably 1-3, more<br>
preferably approximately 2 gl-1 of culture medium. The onset of stationary phase is<br>
associated with a concentration of less than 0.2, preferably less than 0.1 gl-1 of<br>
culture medium.<br>
Other primary carbon and energy sources may be employed such as glucose,<br>
pyruvate, and fatty acids (eg. palmitate, and butyrate). These sources may be<br>
employed at substantially the same concentrations as for glycerol.<br>
The pH of the culture medium is preferably maintained between pH 6 and 8, more<br>
preferably between pH 6.5 and 7.5, most preferably at about pH 6.9.<br>
In one embodiment, the dissolved oxygen tension (DOT) is maintained throughout<br>
the culture process at at least 40 % air saturation when measured at 37°C, more<br>
preferably between 50 and 70 % air saturation when measured at 37°C, most<br>
preferably at 50% air saturation when measured at 37°C.<br>
The dissolved oxygen tension parameter is calculated by means of an oxygen<br>
electrode and conventional laboratory techniques at 37°C. Thus, 100 % air<br>
saturation corresponds to a solution which is saturated with air, whereas 0%<br>
corresponds to a solution which has been thoroughly purged with an inert gas such<br>
as nitrogen. Calibration is performed under standard atmospheric pressure<br>
conditions and 37 ºC, and with conventional air comprising approximately 21 %<br>
oxygen. Thus, the DOT values quoted in the present application concern DOT when<br>
measured at 37 °C and standard atmospheric pressure. A reference temperature<br>
and pressure is indicated, as DOT may vary with temperature and pressure.<br>
In another embodiment of the present invention, latency may be induced by a<br>
combination of carbon and/or energy source starvation, and a low DOT.<br>
In a preferred embodiment, the DOT is maintained at at least 40 % air saturation,<br>
more preferably between 50 and 70 % air saturation, until the mycobacterial culture<br>
has entered early-mid log phase. The DOT may be then lowered so as to become<br>
limiting, for example in increments over a 5 or 6 day period, and the culture<br>
maintained at a DOT of 0-10, preferably at a DOT of approximately 5 % or less until<br>
the stationary phase cells are harvested.<br>
The present inventors believe that the carbon and energy starvation, and optional<br>
low oxygen tension, induction conditions of the present invention are culture<br>
conditions that are conducive for a mycobacterium to express at least one gene that<br>
would be normally expressed in vivo during infection.<br>
In use, it is preferred that those genes (ie. as represented by cDNAs in the<br>
detection assay) that are down-regulated by at least 1.5-fold under stationary<br>
phase, nutrient-starving conditions vis-a-vis exponential phase, non nutrient-<br>
starving conditions are selected. In more preferred embodiments, the corresponding<br>
down-regulation selection criterion is at least 2-fold, more preferably 3-fold, most<br>
preferably 4-fold. Jn further embodiments down-regulation levels of at least 10-fold,<br>
preferably 50-fold may be employed. The above down-regulation criteria apply to<br>
all aspects of the present invention.<br>
The mycobacterium is selected from the species M. phlei, M. smegmatis, M.<br>
africanum, M. caneti, M. fortuitum, M. marinum, M. ulcerans, M. tuberculosis, M.<br>
bovis, M. microti, M. avium, M. paratuberculosis, M. leprae, M. lepraemurium, M.<br>
intracellulars, M. scrofulaceum, M. xenopi, M. genavense, M. kansasii, M. simiae,<br>
M. szulgai, M. haemophilum, M. asiaticum, M. malmoense, M. vaccae, M. caneti,<br>
and M. shimoidei. Of particular interest are members of the MTC, preferably M.<br>
tuberculosis.<br>
The term peptide throughout this specification is synonymous with protein.<br>
Use of mycobacterial peptide compositions according to the present invention<br>
provide excellent vaccine candidates for targeting mycobacteria in patients infected<br>
with mycobacteria.<br>
The terms "isolated," "substantially pure," and "substantially homogenous" are used<br>
interchangeably to describe a peptide which has been separated from components<br>
which naturally accompany it. A peptide is substantially pure when at least about<br>
60 to 75% of a sample exhibits a single peptide sequence. A substantially pure<br>
peptide will typically comprise about 60 to 90% w/w of a protein sample, more<br>
usually about 95%, and preferably will be over about 99% pure. Peptide purity or<br>
homogeneity may be indicated by, for example, polyacrylamide gel electrophoresis<br>
of a protein sample, followed by visualizing a single polypeptide band upon staining<br>
the gel. Alternatively, higher resolution may be provided by using, for example,<br>
HPLC.<br>
A peptide is considered to be isolated when it is separated from the contaminants<br>
which accompany it in its natural state. Thus, a peptide which is chemically<br>
synthesized or synthesized in a cellular system different from the cell from which<br>
it naturally originates will be substantially free from its naturally associated<br>
components.<br>
The present invention provides peptides which may be purified from mycobacteria'<br>
as well as from other types of cells transformed with recombinant nucleic acids<br>
encoding these peptides.<br>
If desirable, the amino acid sequence of the proteins of the present invention may<br>
be determined by protein sequencing methods.<br>
The terms "peptide", "oligopeptide", "polypeptide", and "protein" are used<br>
interchangeably and do not refer to a specific length of the product. These terms<br>
embrace post-translational modifications such as glycosylation, acetylation, and<br>
phosphorylation.<br>
The term "fragment" means a peptide having at least five, preferably at least ten,<br>
more preferably at least twenty, and most preferably at least thirty-five amino acid<br>
residues of the peptide which is the gene product of the down-regulated gene in<br>
question. The fragment preferably includes at least one epitope of the gene product<br>
in question.<br>
The term "variant" means a peptide or peptide fragment having at least seventy,<br>
preferably at least eighty, more preferably at least ninety percent amino acid<br>
sequence homology with the peptide that is the gene product of the down-reguiated<br>
gene in question. An example of a "variant" is a peptide or peptide fragment of a<br>
down-regulated gene which contains one or more analogs of an amino acid (eg. an<br>
unnatural amino acid), or a substituted linkage. The terms "homology" and "identity"<br>
are considered synonymous in this specification. In a further embodiment, a<br>
"variant" may be a mimic of the peptide or peptide fragment, which mimic<br>
reproduces at least one epitope of the peptide or peptide fragment. The mimic may<br>
be, for example, a nucleic acid mimic, preferably a DNA mimic.<br>
For sequence comparison, typically one sequence acts as a reference sequence,<br>
to which test sequences may be compared. When using a sequence comparison<br>
algorithm, test and reference sequences are input into a computer, subsequent<br>
coordinates are designated, if necessary, and sequence algorithm program<br>
parameters are designated. The sequence comparison algorithm then calculates<br>
the percentage sequence identity for the test sequence(s) relative to the reference<br>
sequence, based on the designated program parameters.<br>
Optimal alignment of sequences for comparison may be conducted, for example,<br>
by the local homology alignment algorithm of Smith and Waterman [Adv. Appl.<br>
Math. 2: 484 (1981)], by the algorithm of Needleman &amp; Wunsch [J. Mol. Biol. 48:<br>
443 (1970)] by the search for similarity method of Pearson &amp; Lipman [Proc. Nat'l.<br>
Acad. Sci. USA 85:2444 (1988)], by computer implementations of these algorithms<br>
(GAP, BESTFIT, FASTA, and TFASTA - Sequence Analysis Software Package of<br>
the Genetics Computer Group, University of Wisconsin Biotechnology Center, 1710<br>
University Avenue, Madison, Wis. 53705), or by visual inspection [see Current<br>
Protocols in Molecular Biology, F.M. Ausbel et al, eds, Current Protocols, a joint<br>
venture between Greene Publishing Associates, In. And John Wiley &amp; Sons, Inc.<br>
(1995 Supplement) Ausbubel].<br>
Examples of algorithms suitable for determining percent sequence similarity are the<br>
BLAST and BLAST 2.0 algorithms [see Altschul (1990) J. Mol. Biol. 215: pp. 403-<br>
410; and "http://www.ncbi.nlm.nih.govr of the National Center for Biotechnology<br>
Information].<br>
In a preferred homology comparison, the identity exists over a region of the<br>
sequences that is at least 10 amino acid, preferably at least 20 amino acid, more<br>
preferably at least 30 amino acid residues in length.<br>
The term "derivative" means a protein comprising the peptide (or fragment, or<br>
variant thereof) which peptide is the gene product of the down-regulated gene in<br>
question. Thus, a derivative may include the peptide in question, and a further<br>
peptide sequence which may introduce one or more additional epitopes. The further<br>
peptide sequence should preferably not interfere with the basic folding and thus<br>
conformational structure of the peptide in question. Examples of a "derivative" are<br>
afusion protein, a conjugate, and a graft. Thus, two or more peptides (orfragments,<br>
or variants) may be joined together to form a derivative. Alternatively, a peptide (or<br>
fragment, or variant) may be joined to an unrelated molecule (eg. a second,<br>
unrelated peptide). Derivatives may be chemically synthesized, but will be typically<br>
prepared by recombinant nucleic acid methods. Additional components such as<br>
lipid, and/or polysaccharide, and/or polyketide components may be included.<br>
All of the molecules "fragment", "variant" and "derivative" have a common antigenic<br>
cross-reactivity and/or substantially the same in vivo biological activity as the" gene<br>
product of the down-regulated gene in question from which they are derived. For<br>
example, an antibody capable of binding to a fragment, variant or derivative would<br>
be also capable of binding to the gene product of the down-regulated gene in<br>
question. It is a preferred feature that the fragment, variant and derivative each<br>
possess the active site of the peptide which is the down-reguiafed peptide in<br>
question. Alternatively, all of the above embodiments of a peptide of the present<br>
invention share a common ability to induce a "recall response" of a T-lymphocyte<br>
which has been previously exposed to an antigenic component of a mycobacterial<br>
infection.<br>
In a preferred embodiment, the peptide is selected from the group consisting of<br>
SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39,<br>
41, 43, and 45.<br>
According to a second aspect of the present invention there is provided a method<br>
of identifying a mycobacterial gene the expression of which is down-regulated<br>
during mycobacterial latency, said method comprising:-<br>
. culturing a first mycobacterium under culture conditions that are nutrient-<br>
starving and which do not support exponential growth of the first mycobacterium;<br>
culturing a second mycobacterium under culture conditions that are not<br>
nutrient-starving and which support exponential growth of the second<br>
mycobacterium;<br>
obtaining first and second mRNA populations from said first and second<br>
mycobacteria respectively, wherein said first mRNA population is obtained from the<br>
first mycobacterium which has been harvested during stationary phase and<br>
wherein said second mRNA is obtained from the second mycobacterium which has<br>
been harvested during exponential phase growth;<br>
preparing first and second cDNA populations from said first and second<br>
mRNA populations respectively, during which cDNA preparation a detectable label<br>
is introduced into the cDNA molecules of the first and second cDNA populations;<br>
isolating corresponding first and second cDNA molecules from the first and<br>
second cDNA populations, respectively;<br>
comparing relative amounts of label or corresponding signal emitted from the<br>
label present in the isolated first and second cDNA molecules;<br>
identifying a greater amount of label or signal provided by the isolated<br>
second cDNA molecule than that provided by the isolated first cDNA molecule; and<br>
identifying the first cDNA and the corresponding mycobacterial gene which<br>
is down-regulated during mycobacterial latency.<br>
Reference to "down-regulated" embraces switched-off. A gene is switched-off when<br>
there is substantially no transcription of said gene.<br>
Reference to gene throughout this specification embraces open reading frames<br>
(ORFs).<br>
The term "corresponding first and second cDNA molecules from the first and<br>
second cDNA populations" refers to cDNAs having substantially the same<br>
nucleotide sequence. Thus, by isolating the cDNA copies relating to a given gene<br>
under each culture condition (ie. exponential phase, and stationary phase), it is<br>
possible to quantify the relative copy number of cDNA for that gene for each culture<br>
condition. Since each cDNA copy has been produced from an mRNA molecule, the<br>
cDNA copy number reflects the corresponding mRNA copy number for each culture<br>
condition, and thus it is possible to identify down-regulated genes.<br>
In one embodiment, the first and second cDNA molecules are isolated from the<br>
corresponding first and second cDNA populations by hybridisation to an array<br>
containing immobilised DNA sequences which are representative of each known<br>
gene (or ORF) of a particular mycobacterial species' genome. Thus, a first cDNA<br>
may be considered "corresponding" to a second cDNA if both cDNAs hybridise to<br>
the same immobilised DNA sequence. Alternatively, representative DNA sequences<br>
from a particular mycobacterial strain, or from a number of different species and/or<br>
strains may be employed in the array.<br>
In another embodiment, the first and second cDNAs are prepared by incorporation<br>
of a fluorescent label. The first and second cDNAs may incorporate labels which<br>
fluoresce at different wavelengths, thereby permitting dual fluorescence and<br>
simultaneous detection of two cDNA samples.<br>
The type of label employed naturally determines how the output of the detection<br>
method is read. When using fluorescent labels, a confocal laser scanner is<br>
preferably employed.<br>
According to one embodiment, fluorescently labelled cDNA sequences from<br>
stationary and exponential phase cultured systems were allowed to hybridise with<br>
a whole mycobacterial genome array. The first cDNA population was labelled with<br>
fluorescent label A, and the second cDNA population was labelled with fluorescent<br>
label B. The array was scanned at two different wavelengths corresponding to the<br>
excitable maxima of each dye and the intensity of the emitted light was recorded.<br>
Multiple arrays were then preferably prepared for each cDNA and a mean intensity<br>
value was calculated across the two cDNA populations for each spot with each<br>
dye, against which relative induction or up-regulation was quantified.<br>
In addition to the above mRNA isolation and cDNA preparation and labelling,<br>
genomic DNA may be isolated from the first and second mycobacteria. Thus, in a<br>
preferred embodiment, labelled DNA is also prepared from the Isolated DNA. The<br>
labelled DNA may be then included on each array as a control.<br>
According to a third aspect of the present invention, there is provided an inhibitor<br>
of a mycobacterial peptide, wherein the peptide is encoded by a gene the<br>
expression of which is down-regulated during a stationary phase culture of<br>
mycobacteria under nutrient-starving culture conditions when compared with an<br>
exponential phase culture of mycobacteria under culture conditions that are not<br>
nutrient-starving and that support exponential growth of said mycobacteria, and<br>
wherein the inhibitor is capable of preventing or inhibiting the mycobacterial peptide<br>
from exerting its native biological effect.<br>
Such inhibitors may be employed to prevent the onset of, or to cause a break in the<br>
period of mycobacterial latency (ie. induce re-activation). In this respect,<br>
mycobacteria are more susceptible to treatment regimens when in a non-latent<br>
state, and the combined use of drugs to kill latent mycobacteria (eg. TB) would<br>
significantly reduce the incidence of mycobacteria by targeting the reservoir for new<br>
disease and would thereby help reduce the problem of emerging drug-resistant<br>
strains.<br>
The inhibitor may be a peptide, carbohydrate, synthetic molecule, or an analogue<br>
thereof. Inhibition of the mycobacterial peptide may be effected at the nucleic acid<br>
level (ie. DNA, or RNA), or at the peptide level. Thus, the inhibitor may act directly<br>
on the peptide. Alternatively, the inhibitor may act indirectly on the peptide by, for<br>
example, causing inactivation of the down-regulated mycobacterial gene.<br>
In preferred embodiments, the inhibitor is capable of inhibiting one or more of the<br>
following:- endoglucanase, endo-1,4-beta-glucanase, carboxymethyl cellulase,<br>
inorganic phosphate transporter protein, transcriptional regulatory protein, 50S<br>
ribosomal protein L3, ribosomal protein S1, 30S ribosomal protein S4, uroporphyrin<br>
III C-methyltransferase, uroporphyrinogen III methylase, urogen 111 methylase,<br>
crystathionine gamma synthase, 0-succinylhomoserine[thiol]-lyase, and zinc<br>
metalloprotease.<br>
In a further embodiment, the inhibitor may be an antibiotic capable of targeting the<br>
down-regulated mycobacterial gene identifiable by the present invention, or the<br>
product.<br>
In a further embodiment, the inhibitor may act on a gene or gene product the latter<br>
of which interacts with the down-regulated gene. Alternatively, the inhibitor may act<br>
on a gene or gene product thereof upon which the gene product of the down-<br>
regulated gene acts.<br>
Inhibitors of the present invention may be prepared utilizing the sequence<br>
information provided herein. For example, this may be performed by<br>
overexpressing the peptide, purifying the peptide, and then performing X-ray<br>
crystallography on the purified peptide to obtain its molecular structure. Next,<br>
compounds are created which have similar molecular structures to all or portions<br>
of the peptide or its substrate. The compounds may be then combined with the<br>
peptide and attached thereto so as to block one or more of its biological activities.<br>
Also included within the invention are Isolated or recombinant polynucleotides that<br>
bind to the regions of the mycobacterial chromosome containing sequences that<br>
are associated with down-regulation under carbon starvation and optionally low<br>
oxygen tension (ie. virulence), including antisense and triplex-forming<br>
polynucleotides. As used herein, the term "binding" refers to an interaction or<br>
complexation between an oligonucleotide and a target nucleotide sequence,<br>
. mediated through hydrogen bonding or other molecular forces. The term "binding"<br>
more specifically refers to two types of Internucleotide binding mediated through<br>
base-base hydrogen bonding. The first type of. binding is "Watson-Crick-type"<br>
binding interactions in which adenine-thymine (or adenine-uracil) and<br>
guanine-cytosine base-pairs are formed through hydrogen bonding between the<br>
bases. An example of this type of binding is the binding traditionally associated with<br>
the DNA double helix and in RNA-DNA hybrids; this type of binding is normally<br>
detected by hybridization procedures.<br>
The second type of binding is "triplex binding". In general, triplex binding refers to<br>
any type of base-base hydrogen""bonding of a third polynucleotide, strand with a<br>
duplex DNA (or DNA-RNA hybrid) that is already paired in a Watson-Crick manner.<br>
In a preferred embodiment, the inhibitor may be an antisense nucleic acid<br>
sequence which is complementary to at least part of the down regulated<br>
gene.<br>
The inhibitor, when in the form of a nucleic acid sequence, in use, comprises at<br>
least 15 nucleotides, preferably at least 20 nucleotides, more preferably at least 30<br>
nucleotides, and most preferably at least 50 nucleotides.<br>
According to a fourth aspect of the invention; there is provided an antibody that<br>
binds to a peptide encoded by a gene, or to a fragment or variant or derivative of<br>
said peptide, the expression of which gene is down-regulated during a stationary<br>
phase culture of mycobacteria under nutrient-starving culture conditions when S<br>
compared with an exponential phase culture of mycobacteria under culture<br>
conditions that are not nutrient-starving.<br>
The antibody preferably has specificity for the peptide in question, and following<br>
binding thereto may initiate coating of a mycobacterium expressing said peptide.<br>
Coating of the bacterium preferably leads to opsonization thereof. This, in turn,<br>
leads to the bacterium being destroyed. It is preferred that the antibody is specific<br>
for the mycobacterium (eg. species and/or strain) which is to be targeted.<br>
In use, the antibody is preferably embodied in an isolated form.<br>
Opsonization by antibodies may influence cellular entry and spread of mycobacteria<br>
in phagocytic and non-phagocytic cells by preventing or modulating receptor-<br>
mediated entry and replication in macrophages.<br>
The peptides, fragments, variants or derivatives of the present invention may be<br>
used to produce antibodies, including polyclonal and monoclonal. If polyclonal<br>
antibodies are desired, a selected mammal (eg. mouse, rabbit, goat, horse, etc.)<br>
is immunized with an immunogenic polypeptide. Serum from the immunized animal<br>
is collected and treated according to known procedures. If serum containing<br>
polyclonal antibodies to a desired mycobacterial epitope contains antibodies to<br>
other antigens, the polyclonal antibodies may be purified by immunoaffinity chromatography.<br>
Alternatively, general methodology for making monoclonal antibodies by<br>
hybridomas involving, for example, preparation of immortal antibody-producing cell<br>
lines by cell fusion, or other techniques such as direct transformation of B<br>
lymphocytes with oncogenic DNA, or transfection with Epstein-Barr virus may be<br>
employed.<br>
The antibody employed in this aspect of the invention may belong to any antibody<br>
isotype family, or may be a derivative or mimic thereof. Reference to antibody<br>
throughout this specification embraces recombinantly produced antibody, and any<br>
part of an antibody which is capable of binding to a mycobacterial antigen.<br>
In one embodiment the antibody belongs to the IgG, IgM or IgA isotype families.<br>
In a preferred embodiment, the antibody belongs to the IgA isotype family.<br>
Reference to the IgA isotype throughout this specification includes the secretory<br>
form of this antibody (ie. slgA). The secretory component (SC) of slgA may be<br>
added in vitro or in vivo. In the latter case, the use of a patient's natural SC labelling<br>
machinery may be employed.<br>
In one embodiment, the antibody may be raised against a peptide from a member<br>
of the MTC, preferably against M. tuberculosis.<br>
In a preferred embodiment, the antibody is capable of binding to a peptide selected<br>
from the group consisting of SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25,<br>
27, 29, 31, 33, 35, 37, 39, 41, 43, 45, (or a fragment, variant, or derivative thereof).<br>
In a further embodiment, the antigen is an exposed component of a mycobacterial<br>
bacillus. In another embodiment, the antigen is a cell surface component of a<br>
mycobacterial bacillus.<br>
The antibody of the present invention may be polyclonal, but is preferably<br>
monoclonal.<br>
Without being bound by any theory, it is possible that following mycobacterial<br>
infection of a macrophage, the macrophage is killed and the bacilli are released. It<br>
is at this stage that the mycobacteria are considered to be most vulnerable to<br>
antibody attack. Thus, it is possible that the antibodies of the present invention act<br>
on released bacilli following macrophage death, and thereby exert a post-infection<br>
effect.<br>
It is possible that the passive protection aspect (ie. delivery of antibodies) of the<br>
present invention is facilitated by enhanced accessibility of the antibodies of the<br>
present invention to antigens on mycobacterial bacilli. It is also possible that<br>
antibody binding may block macrophage infection by steric hindrance or disruption<br>
of its oligomeric structure. Thus, antibodies acting on mycobacterial bacilli released<br>
from killed, infected macrophages may interfere with the spread of re-infection to<br>
fresh macrophages. This hypothesis involves a synergistic action between<br>
antibodies and cytotoxic T cells, acting early after infection, eg. ?d and NKT cells,<br>
but could later involve also CD8 and CD4 cytotoxic T cells.<br>
According to a fifth aspect of the invention, there is provided an attenuated<br>
mycobacterium in which a gene has been modified thereby rendering the<br>
mycobacterium substantially reduced in ability to enter a latent state, wherein said<br>
gene is a gene the expression of which is down-regulated during a stationary phase<br>
culture of mycobacteria under nutrient-starving culture conditions when compared<br>
with an exponential phase culture of mycobacteria under culture conditions that are<br>
not nutrient-starving. The modification preferably inactivates the gene in question,<br>
and preferably renders the mycobacterium substantially non-pathogenic.<br>
The term "modified" refers to any genetic manipulation such as a nucleic acid or<br>
nucleic acid sequence replacement, a deletion, or an insertion which renders the<br>
mycobacterium substantially reduced in ability to enter a latent state. In one<br>
embodiment the entire down-regulatable gene may be deleted.<br>
In a preferred embodiment, gene to be modified has a wild-type coding sequence<br>
selected from the group consisting of SEQ ID NO: 2, 4, 6, 8, 10, 12, 14, 16, 18, 20,<br>
22, 24, 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46.<br>
It will be appreciated that the above wild-type sequences may include minor<br>
variations depending on the Database employed. The term "wild-type" indicates that<br>
the sequence in question exists as a coding sequence in nature.<br>
According to a sixth aspect of the invention, there is provided an attenuated<br>
microbial carrier, comprising a peptide encoded by a gene, or a fragment or variant<br>
or derivative of said peptide, the expression of which gene is down-regulated during<br>
a stationary phase culture of mycobacteria under nutrient-starving culture<br>
conditions when compared with an exponential phase culture of mycobacteria<br>
under culture conditions that are not nutrient-starving.<br>
In use, the peptide (or fragment, variant or derivative) is either at least partially<br>
exposed at the surface of the carrier, or the carrier becomes degraded in vivo so<br>
that at least part of the peptide (or fragment, variant or derivative) is otherwise<br>
. exposed to a host's immune system.<br>
In a preferred embodiment, the attenuated microbial carrier is attenuated<br>
salmonella, attenuated vaccinia virus, attenuated fowlpox virus, or attenuated M.<br>
bows (eg. BCG strain).<br>
In a preferred embodiment, the peptide is selected from the group consisting of<br>
SEQ ID NO: 1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39,<br>
41,43,45.<br>
According to a seventh aspect of the invention, there is provided a DNA plasmid<br>
comprising a promoter, a polyadenylation signal, and a DNA sequence that<br>
encodes a gene or a fragment or variant of derivative of said gene, the expression<br>
of which gene is down-regulated during a stationary phase culture of mycobacteria<br>
under nutrient-starving culture conditions when compared with an exponential<br>
phase culture of mycobacteria under culture conditions that are not nutrient-<br>
starving, wherein the promoter and polyadenylation signal are operably linked to the<br>
DNA sequence. Reference to gene preferably means the peptide-coding sequence<br>
of the down-regulated gene.<br>
The term DNA "fragment" used in this invention will usually comprise at least about<br>
5 codons (15 nucleotides), more usually at least about 7 to 15 codons, and most<br>
preferably atleast about 35 codons. This number of nucleotides is usually about the<br>
minimal length required for a successful probe that would hybridize specifically with<br>
such a sequence.<br>
In preferred embodiments, the DNA "fragment" has a nucleotide length which Is at<br>
least 50%, preferably at least 70%, and more preferably at least 80% that of the<br>
coding sequence of the corresponding down-regulated gene.<br>
The term DNA "variant" means a DNA sequence which has substantial homology<br>
or substantial similarity to the coding sequence (or a fragment thereof) of a down-<br>
regulated gene. A nucleic acid or fragment thereof is "substantially homologous" (or<br>
"substantially similar") to another if, when optimally aligned (with appropriate<br>
nucleotide insertions or deletions) with the other nucleic acid (or its complementary<br>
strand), there is nucleotide sequence identity in at least about 60% of the nucleotide<br>
bases, usually at least about 70%, more usually at least about 80%, preferably at<br>
least about 90%, and more preferably at least about 95 to 98% of the nucleotide<br>
bases. Homology determination is performed as described supra for peptides.<br>
Alternatively, a DNA "variant" is substantially homologous (or substantially similar)<br>
with the coding sequence (or a fragment thereof) of a down-regulated gene when<br>
they are capable of hybridizing under selective hybridization conditions. Selectivity<br>
of hybridization exists when hybridization occurs which is substantially more<br>
selective than total lack of specificity. Typically, selective hybridization will occur<br>
when there is at least about 65% homology over a stretch of at least about 14<br>
nucleotides, preferably at least about 70%, more preferably at least about 75%, and<br>
most preferably at least about 90%. See, Kanehisa (1984) Nuc. Acids Res. 12:203-<br>
213. The length of homology comparison, as described, may be over longer<br>
stretches, and in certain embodiments will often be over a stretch of at least about<br>
17 nucleotides, usually at least about 20 nucleotides, more usually at least about<br>
24 nucleotides, typically at least about 28 nucleotides, more typically at least about<br>
32 nucleotides, and preferably at least about 36 or more nucleotides.<br>
Nucleic acid hybridization will be affected by such conditions as salt concentration<br>
(eg. NaCi), temperature, or organic solvents, in addition to the base composition,<br>
length of the complementary strands, and the number of nucleotide base<br>
mismatches between the hybridizing nucleic acids, as will be readily appreciated<br>
. by those skilled in the art. Stringent temperature conditions are preferably<br>
employed, and generally include temperatures in excess of 30°C, typically in<br>
excess of 37°C and preferably in excess of 45°C. Stringent salt conditions will<br>
ordinarily be less than 1000 mM, typically less than 500 mM, and preferably less<br>
than 200 mM. The pH is typically between 7.0 and 8.3. However, the combination<br>
of parameters is much more important than the measure of any single parameter.<br>
See, for example, Wetmur and Davidson (1968) J. Mol. Biol. 31:349-370.<br>
The term DNA "derivative" means a DNA polynucleotide which comprises a DNA<br>
sequence (or a fragment, or variant thereof) corresponding to the coding sequence<br>
of the down-regulated gene and an additional DNA sequence which is not naturally<br>
associated with the DNA sequence corresponding to the coding sequence. The<br>
comments on peptide derivative supra also apply to DNA "derivative". A<br>
"derivative" may, for example, include two or more coding sequences of a<br>
mycobacterial operon that is induced during nutrient starvation. Thus, depending<br>
on the presence of absence of a non-coding region between the coding sequences,<br>
the expression product/s of such as "derivative" may be a fusion protein, or<br>
separate peptide products encoded by the individual coding regions.<br>
The above terms DNA "fragment", "variant", and "derivative" have in common with<br>
each other that the resulting peptide products have cross-reactive antigenic<br>
properties which are substantially the same as those of the corresponding wild-type<br>
peptide. Preferably all of the peptide products of the above DNA molecule<br>
embodiments of the present invention bind to an antibody which also binds to the<br>
wild-type peptide. Alternatively, all of the above peptide products are capable of<br>
inducing a "recall response" of a T lymphocyte which has been previously exposed<br>
to an antigenic component of a mycobacterial infection.<br>
The promoter and polyadenylation signal are preferably selected so as to ensure<br>
that the gene is expressed in a eukaryotic cell. Strong promoters and<br>
polyadenylation signals are preferred.<br>
In a related aspect, the present invention provides an isolated RNA molecule that<br>
is encoded by a DNA sequence of the present invention, or a fragment or variant<br>
or derivative of said DNA sequence.<br>
An "isolated" RNA is an RNA which is substantially separated from other<br>
mycobacterial components that naturally accompany the sequences of interest, eg.,<br>
ribosomes, polymerases, and other mycobacterial polynucleotides such as DNA<br>
and other chromosomal sequences.<br>
The above RNA molecule may be introduced directly into a host cell as, for<br>
example, a component of a vaccine.<br>
Alternatively the RNA molecule may be incorporated into an RNA vector prior to<br>
administration.<br>
The polynucleotide sequences (DNA and RNA) of the present invention include a<br>
nucleic acid sequence that has been removed from its naturally occurring<br>
environment, and recombinant or cloned DNA isolates and chemically synthesized<br>
analogues or analogues biologically synthesized by heterologous systems.<br>
The term "recombinant" as used herein intends a polynucleotide of genomic, cDNA,<br>
semisynthetic, or. synthetic origin which, by virtue of its origin or manipulation: (1)<br>
is not associated with all or a portion of a polynucleotide with which it is associated<br>
in nature; or (2) is linked to a polynucleotide other than that to which it is linked in<br>
nature; and (3) does not occur in nature. This artificial combination is often<br>
accomplished by either chemical synthesis means, or by the artificial manipulation<br>
of isolated segments of nucleic acids, eg., by genetic engineering techniques. Such<br>
is usually done to replace a codon with a redundant codon encoding the same or<br>
a conservative amino acid, while typically introducing or removing a sequence<br>
recognition site. Alternatively, it is performed to join together nucleic acid segments<br>
of desired functions to generate a desired combination of functions.<br>
In embodiments of the invention the polynucleotides may encode a peptide (or<br>
fragment, variant, or derivative) that is down-regulated under nutrient-starving<br>
conditions. A nucleic acid is said to "encode" a peptide if, in its native state or when<br>
manipulated, it can be transcribed and/or translated to produce the peptide (or<br>
fragment, variant or derivative thereof). The anti-sense strand of such a nucleic acid<br>
is also said to encode the peptide (or fragment, variant, or derivative).<br>
Also contemplated within the invention are expression vectors comprising the<br>
polynucleotide of interest. Expression vectors generally are repiicable<br>
polynucleotide constructs that encode a peptide operably linked to suitable<br>
transcriptional and translational regulatory elements. Examples of regulatory<br>
elements usually included in expression vectors are promoters, enhancers,<br>
ribosomal binding sites, and transcription and translation initiation and termination<br>
sequences. These regulatory elements are operably linked to the sequence to be<br>
translated. A nucleic acid sequence is operably linked when it is placed into a<br>
functional relationship with another nucleic acid sequence. For instance, a promoter<br>
is operably linked to a coding sequence if the promoter affects its transcription or<br>
expression. Generally, operably linked means that the DNA sequences being linked<br>
are contiguous and, where necessary to joint two protein coding regions, contiguous<br>
and in reading frame. The regulatory elements employed in the expression vectors<br>
containing a polynucleotide encoding a virulence factor are functional in the host<br>
cell used for expression.<br>
The polynucleotides of the present invention may be prepared by any means known<br>
in the art. For example, large amounts of the polynucleotides may be produced by<br>
replication in a suitable host cell. The natural or synthetic DNA fragments coding<br>
for a desired fragment will be incorporated into recombinant nucleic acid constructs,<br>
typically DNA constructs, capable of introduction into and replication in a<br>
prokaryotic or eukaryotic cell. Usually the DNA constructs will be suitable for<br>
autonomous replication in a unicellular host, such as yeast or bacteria, but may also<br>
be intended for introduction to and integration within the genome of a cultured<br>
insect, mammalian, plant or other eukaryotic cell lines.<br>
The polynucleotides of the present invention may also be produced by chemical<br>
synthesis, eg. by the phosphoramidite method or the triester method, and may be<br>
performed on commercial automated oligonucleotide synthesizers. A<br>
double-stranded fragment may be obtained from the single stranded product of<br>
chemical synthesis either by synthesizing the complementary strand and annealing<br>
the strand together under appropriate conditions or by adding the complementary<br>
strand using DNA polymerase with an appropriate primer sequence.<br>
DNA constructs prepared for introduction into a prokaryotic or eukaryotic host will<br>
typically comprise a replication system recognized by the host, including the<br>
intended DNA fragment encoding the desired peptide, and will preferably also<br>
include transcription and translational initiation regulatory sequences operably<br>
linked to the polypeptide encoding segment. Expression vectors may include, for<br>
example, an origin of replication or autonomously replicating sequence (ARS) and<br>
expression control sequences, a promoter, an enhancer and necessary processing<br>
information sites, such as ribosome-binding sites, RNA splice sites, polyadenylation<br>
sites, transcriptional terminator sequences, and mRNA stabilizing sequences.<br>
Secretion signals from polypeptides secreted from the host cell of choice may also<br>
be included where appropriate, thus allowing the protein to cross and/or lodge in<br>
cell membranes, and thus attain its functional topology or be secreted from the cell.<br>
Appropriate promoter and other necessary vector sequences are selected so as to<br>
be functional in the host, and may, when appropriate, include those naturally<br>
associated with mycobacterial genes. Promoters such as the trp, lac and phage<br>
promoters, tRNA promoters and glycolytic enzyme promoters may be used in<br>
prokaryotic hosts. Useful yeast promoters include the promoter regions for<br>
metallothionein, 3-phosphoglycerate kinase or other glycolytic enzymes such as<br>
enolase or glyceraldehyde-3-phosphate dehydrogenase, enzymes responsible for<br>
maltose and galactose utilization, and others.<br>
Appropriate non-native mammalian promoters may include the early and late<br>
promoters from SV40 or promoters derived from murine moloney leukemia virus,<br>
mouse mammary tumour virus, avian sarcoma viruses, adenovirus II, bovine<br>
papilloma virus or polyoma. In addition, the construct may be joined to an<br>
amplifiable gene (eg. DHFR) so that multiple copies of the gene may be made.<br>
While such expression vectors may replicate autonomously, they may less<br>
preferably replicate by being inserted into the genome of the host cell.<br>
Expression and cloning vectors may contain a selectable marker, a gene encoding<br>
a protein necessary for the survival or growth of a host cell transformed with the<br>
vector. The presence of this gene ensures the growth of only those host cells which<br>
express the inserts. Typical selection genes encode proteins that (a) confer<br>
resistance to antibiotics or other toxic substances, eg. ampicillin, neomycin,<br>
methotrexate, etc.; (b) complement auxotrophic deficiencies; or (c) supply critical<br>
nutrients not available from complex media, e.g. the gene encoding D-alanine<br>
racemase for Bacilli. The choice of appropriate selectable marker will depend on<br>
the host cell.<br>
The vectors containing the nucleic acids of interest can be transcribed in vitro and<br>
the resulting RNA introduced into the host cell (eg. by injection), or the vectors can<br>
be introduced directly into host cells by methods which vary depending on the type<br>
of cellular host, including electroporation; transfection employing calcium chloride,<br>
, rubidium chloride, calcium phosphate, DEAE-dextran, or other substances;<br>
microprojectile bombardment; lipofection; infection (where the vector is an<br>
infectious agent, such as a retroviral genome). The cells into which have been<br>
introduced nucleic acids described above are meant to also include the progeny of<br>
such cells.<br>
Large quantities of the nucleic acids and peptides of the present invention may be<br>
prepared by expressing the nucleic acids or portions thereof in vectors or other<br>
expression vehicles in compatible prokaryotic or eukaryotic host cells. The most<br>
commonly used prokaryotic hosts are strains of Escherichia coli, although other<br>
prokaryotes, such as Bacillus subtilis or Pseudomonas may also be used.<br>
Mammalian or other eukaryotic host cells, such as those of yeast, filamentous fungi,<br>
plant, insect, amphibian or avian species, may also be useful for production of the<br>
proteins of the present invention. Propagation of mammalian cells in culture is per<br>
se well known. Examples of commonly used mammalian host cell lines are VERO<br>
and HeLa cells, Chinese hamster ovary (CHO) cells, and W138, BHK, and COS cell<br>
lines, although other cell lines may be appropriate, e.g., to provide higher<br>
expression, desirable glycosylation patterns.<br>
Clones are selected by using markers depending on the mode of the vectdr<br>
construction. The marker may be on the same or a different DNA molecule,<br>
preferably the same DNA molecule. The transformant may be screened or,<br>
preferably, selected by any of the means well known in the art, e.g., by resistance<br>
to such antibiotics as ampicillin, tetracycline.<br>
The polynucleotides of the invention may be inserted into the host cell by any<br>
means known in the art, including for example, transformation, transduction, and<br>
electroporation. As used herein, "recombinant host cells", "host cells", "cells", "cell<br>
lines", "cell cultures", and other such terms denoting microorganisms or higher<br>
eukaryotic cell lines cultured as unicellular entities refer to cells which can be, or<br>
have been, used as recipients for recombinant vector or other transfer DNA, and<br>
include the progeny of the original cell which has been transformed. It is understood<br>
that the progeny of a single parental cell may not necessarily be completely<br>
identical in morphology or in genomic or total DNA complement as the original<br>
parent, due to natural, accidental, or deliberate mutation. "Transformation", as used<br>
herein, refers to the insertion of an exogenous polynucleotide into a host cell,<br>
irrespective of the method used for the insertion, for example, direct uptake,<br>
transduction, f-mating or electroporation. The exogenous polynucleotide may be<br>
maintained as a non-integrated vector, for example, a plasmid, or alternatively, may<br>
be integrated into the host cell genome.<br>
In one embodiment, a DNA plasmid or RN A vector may encode a component of the<br>
immune system that is specific to an immune response following challenge with a<br>
peptide, wherein said peptide is encoded by a mycobacterial gene that is down-<br>
regulated under nutrient-starving culture conditions.<br>
An example of such a component is an antibody to the peptide product of the down-<br>
regulated gene. Thus, in one embodiment, the nucleic acid sequence (eg. DNA<br>
plasmid or RNA vector) encodes the antibody in question.<br>
In the context of the present invention, the term plasmid is equivalent to vector.<br>
Thus, a plasmid may be either linear or circularised nucleic acid.<br>
An eighth aspect provides use of said aforementioned peptide or fragment or<br>
variant or derivative thereof, inhibitor, antibody, attenuated mycobacterium,<br>
attenuated microbial carrier, DNA sequence corresponding to the coding sequence<br>
of a mycobacterial gene that is down-regulated under nutrient-starving conditions<br>
or a fragment or variant or derivative of said DNA sequence, DNA plasmid<br>
comprising said DNA sequence or said fragment or variant or derivative, RNA<br>
sequence encoded by said DNA sequence or said fragment or variant or derivative,<br>
and/or RNA vector comprising said RNA sequence, in the manufacture of a<br>
medicament for treating or preventing a mycobacterial infection.<br>
The term "preventing" includes reducing the severity/intensity of, or initiation of, a<br>
mycobacterial infection.<br>
The term "treating" includes post-infection therapy and amelioration of a<br>
mycobacterial infection.<br>
In a related aspect, there is provided a method of treating or preventing a<br>
mycobacterial infection, comprising administration to a subject of a medicament<br>
selected from the group consisting of said aforementioned peptide or fragment or<br>
variant or derivative thereof, inhibitor, antibody, attenuated mycobacterium,<br>
, attenuated microbial carrier, DNA sequence corresponding to the coding sequence<br>
of a mycobacterial gene that is down-regulated under nutrient-starving conditions<br>
or a fragment or variant or derivative of said DNA sequence, DNA plasmid<br>
comprising said DNA sequence or said fragment or variant or derivative, RNA<br>
sequence encoded by said DNA sequence or said fragment or variant or derivative,<br>
and/or RNA vector comprising said RNA sequence.<br>
The immunogenicity of the epitopes of the peptides of the invention may be<br>
enhanced by preparing them in mammalian or yeast systems fused with or<br>
assembled with particle-forming proteins such as, for example, that associated with<br>
hepatitis B surface antigen. Vaccines may be prepared from one or more<br>
immunogenic peptides of the present invention.<br>
Typically, such vaccines are prepared as injectables, either as liquid solutions or<br>
suspensions; solid forms suitable for solution in, or suspension in, liquid prior to<br>
injection may also be prepared. The preparation may also be emulsified, or the<br>
peptide encapsulated in liposomes. The active immunogenic ingredients are often<br>
mixed with excipients which are pharmaceutically acceptable and compatible with<br>
the active ingredient. Suitable excipients are, for example, water, saline, dextrose,<br>
glycerol, ethanol, or the like and combinations thereof. In addition, if desired, the<br>
vaccine may contain minor amounts of auxiliary substances such as wetting or<br>
emulsifying agents, pH buffering agents, and/or adjuvants which enhance the<br>
effectiveness of the vaccine. Examples of adjuvants which may be effective include<br>
but are not limited to: aluminum hydroxide,<br>
N-acetyl-muramyl-L-threonyl-D-isoglutamine (thr-MDP),<br>
N-acetyl-nor-muramyl-L-alanyl-D-isoglutamine (CGP 11637, referred to as<br>
nor-MDP), N-acetylmuramyl-L-alanyl-D-isoglutaminyl-L-alanine-2-(1'-2'-dipalmito<br>
yl-sn -glycero-3-hydroxyphosphoryloxy)-ethylam ine (CGP 19835A, referred to as<br>
MTP-PE), and RIBI, which contains three components extracted from bacteria,<br>
monophosphoryl lipid A, trehalose dimycolate and cell wall skeleton<br>
(MPL+TDM+CWS) in a 2% squalene/Tween 80 emulsion.<br>
The vaccines are conventionally administered parenterally, by injection, for<br>
example, either subcutaneously or intramuscularly. Additional formulations which<br>
are suitable for other modes of administration include suppositories and, in some<br>
cases, oral formulations or formulations suitable for distribution as aerosols. For<br>
suppositories, traditional binders and carriers may include, for example,<br>
polyalkylene glycols or triglycerides; such suppositories may be formed from<br>
mixtures containing the active ingredient in the range of 0.5% to 10%, preferably<br>
1%-2%. Oral formulations include such normally employed excipients as, for<br>
example, pharmaceutical grades of mannitol, lactose, starch, magnesium stearate,<br>
sodium saccharine, cellulose, magnesium carbonate, and the like. These<br>
compositions take the form of solutions, suspensions, tablets, pills, capsules,<br>
sustained release formulations or powders and contain 10%-95% of active<br>
ingredient, preferably 25%-70%.<br>
The peptides may be formulated into the vaccine as neutral or salt forms.<br>
Pharmaceutically acceptable salts include the acid addition salts (formed with free<br>
amino groups of the peptide) and which are formed with inorganic acids such as,<br>
for example, hydrochloric or phosphoric acids, or with organic acids such as acetic,<br>
oxalic, tartaric, maleic, and the like. Salts formed with the free carboxyl groups may<br>
also be derived from inorganic bases such as, for example, sodium, potassium,<br>
ammonium, calcium, or ferric hydroxides, and such organic bases as<br>
isopropylamine, trimethylamine, 2-ethylamino ethanol, histidine, procaine, and the<br>
like.<br>
The vaccines are administered in a manner compatible with the dosage formulation,<br>
and in such amount as will be prophylactically and/or therapeutically effective. The<br>
quantity to be administered, which is generally in the range of 5 micrograms to 250<br>
micrograms of antigen per dose, depends on the subject to be treated, capacity of<br>
the subject's immune system to synthesize antibodies, and the degree of protection<br>
desired. Precise amounts of active ingredient required to be administered may<br>
depend on the judgment of the practitioner and may be peculiar to each subject.<br>
The vaccine may be given in a single dose schedule, or preferably in a multiple<br>
dose schedule. A multiple dose schedule is one in which a primary course of<br>
vaccination may be with 1-10 separate doses, followed by other doses given at<br>
subsequent time intervals required to maintain and or re-enforce the immune<br>
response, for example, at 1-4 months for a second dose, and if needed, a<br>
subsequent dose(s) after several months. The dosage regimen will also, at least in<br>
part, be determined by the need of the individual and be dependent upon the<br>
judgment of the practitioner.<br>
In addition, the vaccine containing the immunogenic mycobacterial antigen(s) may<br>
. be administered in conjunction with other immunoregulatory agents, for example,<br>
immunoglobulins, as well as antibiotics.<br>
The medicament may be administered by conventional routes, eg. intravenous,<br>
intraperitoneal, intranasal routes.<br>
The outcome of administering antibody-containing compositions may depend on the<br>
efficiency of transmission of antibodies to the site of infection. In th.e case of a<br>
mycobacterial respiratory infection (eg. a M. tuberculosis infection), this may be<br>
facilitated by efficient transmission of antibodies to the lungs.<br>
In one embodiment the medicament may be administered intranasally (i.n.). This<br>
mode of delivery corresponds to the route of delivery of a M. tuberculosis infection<br>
and, in the case of antibody delivery, ensures that antibodies are present at the site<br>
of infection to combat the bacterium before it becomes intracellular and also during<br>
the period when it spreads between cells.<br>
An intranasal composition may be administered in droplet form having approximate<br>
diameters in the range of 100-5000 µm, preferably 500-4000 µm, more preferably<br>
1000-3000 µm. Alternatively, in terms of volume, the droplets would be In the<br>
approximate range of 0.001-100 µl, preferably 0.1-50 µl, more preferably 1.0-25 µl.<br>
Intranasal administration may be achieved by way of applying nasal droplets or via<br>
a nasal spray.<br>
In the case of nasal droplets, the droplets may typically have a diameter of<br>
approximately 1000-3000 µm and/or a volume of 1-25 µl.<br>
. In the case of a nasal spray, the droplets may typically have a diameter of<br>
approximately 100-1000 µm and/or a volume of 0.001-1 µl.<br>
It is possible that, following i.n; delivery of antibodies, their passage to the lungs is<br>
facilitated by a reverse flow of mucosal secretions, although mucociliary action in<br>
the respiratory tract is thought to take particles within the mucus out of the lungs.<br>
The relatively long persistence in the lungs' lavage, fast clearance from the bile and<br>
lack of transport to the saliva of some antibodies suggest the role of mucosal site<br>
specific mechanisms.<br>
In a different embodiment, the medicament may be delivered in an aerosol<br>
formulation. The aerosol formulation may take the form of a powder, suspension or<br>
solution.<br>
The size of aerosol particles is one factor relevant to the delivery capability of an<br>
aerosol. Thus, smaller particles may travel further down the respiratory airway<br>
towards the alveoli than would larger particles. In one embodiment, the aerosol<br>
particles have a diameter distribution to facilitate delivery along the entire length of<br>
the bronchi, bronchioles, and alveoli. Alternatively, the particle size distribution may<br>
be selected to target a particular section of the respiratory airway, for example the<br>
alveoli.<br>
In the case of aerosol delivery of the medicament, the particles may have diameters<br>
in the approximate range of 0.1-50 urn, preferably 1-25 urn, more preferably 1-5<br>
urn.<br>
The aerosol formulation of the medicament of the present invention may optionally<br>
contain a propellant and/or surfactant.<br>
By controlling the size of the droplets which are to be administered to a patient to<br>
within the defined range of the present invention, it is possible to avoid/minimise<br>
inadvertent antigen delivery to the alveoli and thus avoid alveoli-associated<br>
pathological problems such as inflammation and fibrotic scarring of the lungs.<br>
I.n. vaccination engages both T and B cell mediated effector mechanisms in nasal<br>
and bronchus associated mucosal tissues, which differ from other mucosae-<br>
associated lymphoid tissues.<br>
The protective mechanisms invoked by the intranasal route of administration may<br>
include: the activation of T lymphocytes with preferential lung homing; upregulation<br>
of co-stimulatory molecules, eg. B7.2; and/or activation of macrophages or<br>
secretory IgA antibodies.<br>
Intranasal delivery of antigens may facilitate a mucosal antibody response is<br>
invoked which is favoured by a shift in the T cell response toward the Th2<br>
phenotype which helps antibody production. A mucosal response is characterised<br>
by enhanced IgA production, and a Th2 response is characterised by enhanced IL-<br>
4 production.<br>
Intranasal delivery of mycobacterial antigens allows targeting of the antigens to<br>
submucosal B cells of the respiratory system. These B cells are the major local IgA-<br>
producing cells in mammals and intranasal delivery facilitates a rapid increase in<br>
IgA production by these cells against the mycobacterial antigens.<br>
In one embodiment administration of the medicament comprising a mycobacterial<br>
antigen stimulates IgA antibody production, and the IgA antibody binds to the<br>
mycobacterial antigen. In another embodiment, a mucosal and/or Th2 immune<br>
response is stimulated.<br>
In another embodiment monoclonal antibodies, in particular, may be used to raise<br>
anti-idiotype antibodies. Anti-idiotype antibodies are immunoglobulins which carry<br>
an "internal image" of the antigen of the infectious agent against which protection<br>
is desired. These anti-idiotype antibodies may also be useful for treatment,<br>
vaccination and/or diagnosis of mycobacterial infections.<br>
According to a ninety aspect of the present invention, the peptides (including<br>
fragments, variants, and derivatives thereof) of the present invention and antibodies<br>
that bind thereto are useful in immunoassays to detect the presence of antibodies<br>
to mycobacteria, or the presence of the virulence associated antigens in biological<br>
samples. Design of the immunoassays maybe subject to a great deal of variation,<br>
and many formats are known in the art. The immunoassay may utilize at least one<br>
epitope derived from a peptide of the present invention. In one embodiment, the<br>
immunoassay uses a combination of such epitopes. These epitopes may be<br>
derived from the same or from different bacterial peptides, and may be in separate<br>
recombinant or natural peptide or together in the same recombinant peptide.<br>
An immunoassay may use, for example, a monoclonal antibody directed towards<br>
a virulence associated peptide epitope(s), a combination of monoclonal antibodies<br>
directed towards epitopes of one mycobacterial antigen, monoclonal antibodies<br>
directed towards epitopes of different mycobacterial antigens, polyclonal antibodies<br>
directed towards the same antigen, or polyclonal antibodies directed towards<br>
different antigens. Protocols may be based, for example, upon competition, or direct<br>
reaction, or sandwich type assays. Protocols may also, for example, use solid<br>
supports, or may be by immunoprecipitation. Most assays involve the use of<br>
labelled antibody or polypeptide; the labels may be, for example, enzymatic,<br>
fluorescent, chemiluminescent, radioactive, or dye molecules. Assays which amplify<br>
the signals from the probe are also known; examples of which are assays which<br>
utilize biotin and avidin, and enzyme-labelled and mediated immunoassays, such<br>
as ELISA assays.<br>
Typically, an immunoassay for an antibody(s) to a peptide, will involve selecting and<br>
preparing the test sample suspected of containing the antibodies, such as a<br>
biological sample, then incubating it with an antigenic (ie. epitope-containing)<br>
peptide(s) under conditions that allow antigen-antibody complexes to form, and<br>
then detecting the formation of such complexes. The immunoassay may be of a<br>
standard or competitive type.<br>
The peptide is typically bound to a solid support to facilitate separation of the<br>
sample from the peptide after incubation. Examples of solid supports that can be<br>
used are nitrocellulose (eg. in membrane or microtiter well form), polyvinyl chloride<br>
(eg. in sheets or microtiter wells), polystyrene latex (eg. in beads or microtiter<br>
plates, polyvinylidine fluoride (known as Immulon), diazotized paper, nylon<br>
membranes, activated beads, and Protein A beads. For example, Dynatech<br>
Immulon microtiter plates or 60 mm diameter polystyrene beads (Precision Plastic<br>
Ball) may be used. The solid support containing the antigenic peptide is typically<br>
washed after separating it from the test sample, and priorto detection of bound antibodies.<br>
Complexes formed comprising antibody (or, in the case of competitive assays, the<br>
amount of competing antibody) are detected by any of a number of known<br>
techniques, depending on the format. For example, unlabelled antibodies tn the<br>
complex may be detected using a conjugate of antixenogeneic Ig complexed with<br>
a label (eg. an enzyme label).<br>
In immunoassays where the peptides are the analyte, the test sample, typically a<br>
biological sample, is incubated with antibodies directed against the peptide under<br>
conditions that allow the formation of antigen-antibody complexes. It may be<br>
desirable to treat the biological sample to release putative bacterial components<br>
prior to testing. Various formats can be employed. For example, a "sandwich assay"<br>
may be employed, where antibody bound to a solid support is incubated with the<br>
. test sample; washed; incubated with a second, labelled antibody to the analyte, and<br>
the support is washed again. Analyte is detected by determining if the second<br>
antibody is bound to the support. In a competitive format, a test sample is usually<br>
incubated with antibody and a labelled, competing antigen is also incubated, either<br>
sequentially or simultaneously.<br>
Also included as an embodiment of the invention is an immunoassay kit comprised<br>
of one or more peptides of the invention, or one or more antibodies to said<br>
peptides, and a buffer, packaged in suitable containers.<br>
As used herein, a "biological sample" refers to a sample of tissue or fluid isolated<br>
from an individual, including but not limited to, for example, plasma, serum, spinal<br>
fluid, lymph fluid, the external sections of the skin, respiratory, intestinal, and<br>
genitourinary tracts, tears, saliva, milk, blood cells, tumours, organs, and also<br>
samples of in vitro ceil culture constituents (including but not limited to conditioned<br>
medium resulting from the growth of cells in cell culture medium, putatively virally<br>
infected cells, recombinant cells, and cell components).<br>
In a related diagnostic assay, the present invention provides nucleic acid probes for<br>
detecting a mycobacterial infection.<br>
Using the polynucleotides of the present invention as a basis, oligomers of<br>
approximately 8 nucleotides or more can be prepared, either by excision from<br>
recombinant polynucleotides or synthetically, which hybridize with the<br>
mycobacterial sequences, and are useful in identification of mycobacteria. The<br>
probes are a length which allows the detection of the down-regulated sequences<br>
by hybridization. While 6-8 nucleotides may be a workable length, sequences of<br>
10-12 nucleotides are preferred, and at least about 20 nucleotides appears optimal.<br>
These probes can be prepared using routine methods, including automated<br>
oligonucleotide synthetic methods. For use as probes, complete complementarity<br>
is desirable, though it may be unnecessary as the length of the fragment is<br>
increased.<br>
For use of such probes as diagnostics, the biological sample to be analyzed, such<br>
as blood or serum, may be treated, if desired, to extract the nucleic acids contained<br>
therein. The resulting nucleic acid from the sample may be subjected to gel<br>
electrophoresis or other size separation techniques; alternatively, the nucleic acid<br>
sample may be dot blotted without size separation. The probes are usually labeled.<br>
Suitable labels, and methods for labeling probes are known in the art, and include,<br>
for example, radioactive labels incorporated by nick translation or kinasing, biotin,<br>
fluorescent probes, and chemiluminescent probes. The nucleic acids extracted from<br>
the sample are then treated with the labeled probe under hybridization conditions<br>
of suitable stringencies.<br>
The probes may be made completely complementary to the virulence encoding<br>
polynucleotide. Therefore, usually high stringency conditions are desirable in order<br>
to prevent false positives. The stringency of hybridization is determined by a<br>
number of factors during hybridization and during the washing procedure, including<br>
temperature, ionic strength, length of time, and concentration of formamide.<br>
It may be desirable to use amplification techniques in hybridization assays. Such<br>
techniques are known in the art and include, for example, the polymerase chain<br>
reaction (PCR) technique.<br>
The probes may be packaged into diagnostic kits. Diagnostic kits include the probe<br>
DNA, which may be labeled; alternatively, the probe DNA may be unlabeled and<br>
the ingredients for labeling may be included in the kit in separate containers. The<br>
kit may also contain other suitably packaged reagents and materials needed for the<br>
particular hybridization protocol, for example, standards, as well as instructions for<br>
conducting the test.<br>
In a preferred embodiment, a peptide (or fragment or variant or derivative) of the<br>
present invention is used in a diagnostic assay to detect the presence of a T-<br>
lymphocyte, which T-lymphocyte has been previously exposed to an antigenic<br>
component of a mycobacterial infection in a patient.<br>
In more detail, a T-lymphocyte which has been previously exposed to a particular<br>
antigen will be activated on subsequent challenge by the same antigen. This<br>
activation provides a means for identifying a positive diagnosis of mycobacterial<br>
infection. In contrast, the same activation is not achieved by a T-lymphocyte which<br>
has not been previously exposed to the particular antigen.<br>
The above "activation" of a T-lymphocyte is sometimes referred to as a "recall<br>
response" and may be measured, for example, by determining the release of<br>
interferon (eg. IFN-Y) from the activated T-lymphocyte. Thus, the presence of a<br>
mycobacterial infection in a patient may be determined by the release of a minimum<br>
concentration of interferon from a T-lymphocyte after a defined time period following<br>
in vitro challenge of the T-lymphocyte with a peptide (or fragment or variant or<br>
derivative) of the present invention.<br>
In use, a biological sample containing T-lymphocytes is taken from a patient, and<br>
then Challenged with a peptide (or fragment, variant, or derivative thereof) of the<br>
present invention.<br>
The above T-lymphocyte diagnostic assay may include an antigen presenting cell<br>
(APC) expressing at least one major histocompatibility complex (MHC) class II<br>
molecule expressed by the patient in question. The APC may be inherently<br>
provided in the biological sample, or may be added exogenously. In one<br>
embodiment, the T-lymphocyte is a CD4 T-lymphocyte;<br>
BRIEF DESCRIPTION OF THE ACCOMPANYING DRAWINGS<br>
Brief mention is now made to the Figures of the present application, in which:-<br>
Fig. 1 illustrates the viable counts for M. tuberculosis during culture under batch<br>
fermentation conditions at a DOT of 50 % air saturation (37 °C);<br>
Fig. 2 illustrates the concentration of glycerol (as the primary carbon and energy<br>
source) during culture of M. tuberculosis under batch fermentation conditions at the<br>
separate DOT concentrations of 0.5% and Aerobic culture;<br>
Fig. 3 illustrates the DOT within the medium of the mycobacterial culture described<br>
in Example 18; and<br>
Fig. 4 illustrates the viable counts for M. tuberculosis during the batch fermentation<br>
conditions of Example 18 (ie. carbon-starvation, and oxygen limiting conditions).<br>
Two alternative mycobacterial culture methods have been employed to study-genes<br>
which are down-regulated during mycobacterial latency. The first method is<br>
described in Examples 1-8, whereas the second method is described in Example<br>
18.<br>
Example 1 - in vitro model of mycobacterial persistence under aerobic,<br>
nutrient-starved conditions<br>
Materials and Methods<br>
Strain<br>
Studies were performed with M. tuberculosis strain H37Rv (NCTC cat. no. 7416) -<br>
a representative strain of M.tuberculosis. Stock cultures were grown on<br>
Middlebrook 7H10 + OADC for 3 weeks at 37 ± 2°C.<br>
Culture Medium —<br>
Persistence cultures were established in Middlebrook 7H9 medium supplemented<br>
with Middlebrook ADC enrichment, 0.2% Tween 80 and 0.2% glycerol (Table 1).<br>
The medium was prepared with high quality water from a Millipore water purification<br>
system and filter sterilised by passage through a 0.1 µm pore size cellulose acetate<br>
membrane filter capsule (Sartorius Ltd). The pH was adjusted to 6.6 with<br>
concentrated hydrochloric acid.<br>
Middlebrook 7H10 + OADC agar was used to prepare inoculum cultures,<br>
enumerate the number of culturable bacteria in samples, and to assess culture<br>
purity.<br>
Culture system<br>
We previously developed a process for the culture of mycobacteria under controlled<br>
and defined conditions- Patent Application No. PCT/GB00/00760 (WO00/52139).<br>
We used this culture system operated as a batch fermenter for the following studies<br>
of mycobacterial persistence.<br>
Culture experiments were performed in a one litre glass vessel operated at a<br>
working volume of 750 ml. The culture was agitated by a magnetic bar placed in the<br>
culture vessel coupled to a magnetic stirrer positioned beneath the vessel. Culture<br>
conditions were continuously monitored by an Anglicon Microlab Fermentation<br>
System (Brighton Systems, Newhaven), linked to sensor probes inserted into the<br>
culture through sealed ports in the top plate. The oxygen concentration was<br>
monitored with a galvanic oxygen electrode (Uniprobe, Cardiff) and was controlled<br>
through sparging the culture with a mixture of air and .oxygen free-nitrogen.<br>
Temperature was monitored by an Anglicon temperature probe, and maintained by<br>
a heating pad positioned beneath the culture vessel. Culture pH was measured<br>
using an Ingold pH electrode (Mettler-Toledo, Leicester).<br>
Inoculation and culture<br>
The vessel was filled with 750 ml of sterile culture medium and parameters were<br>
allowed to stabilise at 37 °C = 2 °C, pH 6.9 = 0.3 and a dissolved oxygen tension<br>
of approximately 70 % air saturation. A dense inoculum suspension was prepared<br>
by resuspending Middlebrook agar cultures, grown at 37 °C = 2 °C for 3 weeks, in<br>
sterile deionised water. The inoculum was aseptically transferred to the culture<br>
vessel, to provide an initial culture turbidity of approximately 0.25 at 540 nm,.<br>
The culture were maintained at 37 °C with an agitation rate of 500 to 750 rpm. The<br>
dissolved oxygen tension was maintained between 50 - 70% air saturation with the<br>
aid of culture sparging. The initial culture pH was set at approximately 6.7 and was<br>
and gene expression.<br>
Growth and survival<br>
Bacterial growth and survival was assessed by determining the number of viable<br>
cells in the culture system at specific time points. This was achieved by preparing<br>
a decimal dilution series of the sample in sterile water and plating 100 µl aliquots<br>
onto Middlebrook 7H10 + OADC plates. The plates were incubated at 37 °C for up<br>
to 4 weeks before enumerating the number of colonies formed.<br>
Nutrient utilisation<br>
Glycerol is the primary carbon and energy source present in Middlebrook 7H9<br>
medium with ADC, 0.2 % Tween and 0.2 % Glycerol. The rate at which glycerol<br>
was utilised was determined using the Glycerol Determination Kit Cat. No. 148 270<br>
Boehringer Mannheim.<br>
Microarray experiments<br>
RNA was extracted from culture samples collected at different time points during<br>
the experiment. A fluorescently-labelled cDNA was then transcribed from each<br>
sample of RNA. The cDNA was labelled by the incorporation of either Cy3 or Cy5<br>
labelled dCTP (Dyes are supplied by Amersham Pharmacia Biotech).<br>
Whole M. tuberculosis genome arrays were prepared from M. tuberculosis genomic<br>
DNA using ORF-specific primers. PCR products corresponding to each ORF were<br>
spotted in a grid onto a standard glass microscope slide using a BioRobotics<br>
microgrid robot (MWG Biotech) at a resolution of &gt;4000 spots/cm2.<br>
In each microarray experiment a whole genome array was hybridised with labelled<br>
cDNA from one culture sample (Test sample). Each array was also hybridised with<br>
control DNA incorporating a different Cy dye and prepared from DNA extracted<br>
from M. tuberculosis strain H37Rv (control sample).<br>
Each array was scanned at two different wavelengths corresponding to the<br>
excitation maxima of each dye and the intensity of the emitted light was recorded.<br>
The ration of the intensity values for the test and control samples was determined<br>
for each array. The slides were scanned using an Affymetrix 428 scanner. The raw<br>
expression of each gene.<br>
Results<br>
After inoculation the culture entered exponential growth and continued to grow<br>
exponentially until 10 days after inoculation (see Fig. 1). Cessation of exponential<br>
growth coincided with depletion of the primary carbon and energy source - glycerol<br>
(see Fig. 2). As the culture entered stationary phase, viability started to decline and<br>
continued to decline steadily over the duration of the study. After 40 days in<br>
stationary phase, approximately 1% of the culture was still culturable on<br>
Middlebrook agar.<br>
The gene expression profiles for samples collect at day 5 and day 50 were<br>
compared. Three arrays were prepared for each sample and the test data were<br>
normalised against the control data on each chip. The normalised data for each set<br>
of arrays were then averaged and the two sets of data were compared. Those<br>
genes that were expressed at least 1.5-fold less at day 50 relative to day 5 were<br>
selected. SEQ IDs 1 -20 identified by this method are listed in Table 2, together with<br>
assigned biological functions for each of the encoded peptide.<br>
The coding sequences (nucleic acid sequences are given from the transcription .<br>
start site to the stop codon) for these genes and their corresponding amino acid<br>
sequences are listed in the accompanying sequence listing, in which the first SEQ<br>
ID NO is the amino acid sequence and the second SEQ ID NO is the corresponding<br>
nucleic acid coding sequence.<br>
Example 2 - RNA extraction from M. tuberculosis for microarray analysis<br>
Materials and Methods<br>
Trizol (Life Technologies)- formulation of phenol and guanidine thiocyanate.<br>
GTC lysis solution containing: 5 M guanidine thiocyanate, 0.5 % N-lauryl sarcosine,<br>
25 mM tri-sodium citrate, 0.1 M 2-mercaptoethanol, and 0.5 % Tween 80.<br>
Chloroform<br>
Isopropanol<br>
3M sodium acetate<br>
70 % Ethanol<br>
microfuge<br>
ribolyser<br>
Sterile plasticware-Falcon tubes, screw capped eppendorfs, gilson tips -all RNase<br>
free<br>
Glassware - baked at 160 °C for at least 16 hours<br>
Method<br>
Steps performed at Containment level 3; within a Class III microbiological safety<br>
cabinet.<br>
Remove 10 or 20 ml of culture (109/ml) and immediately add this to 4 volumes of<br>
GTC lysis buffer in a plastic specimen pot. Seal the pot tightly.<br>
Incubate the cells in GTC lysis buffer for 1 hour at room temperature. Surface<br>
decontaminate the plastic pot with 5 % Hycolin for 5 minutes. Transfer the sample to<br>
the pass box and place it into a plastic carry tin with a sealable lid. Close the container<br>
securely and transport it to a non-toxic cabinet CL3 cabinet.<br>
Equally distribute the lysis mixture between Falcon tubes. Place these tubes into<br>
centrifuge buckets and seal the buckets tightly. Surface-decontaminate the buckets<br>
for 5 minutes with 5 % Hycolin. Then transfer them to the centrifuge (Baird and Tatlock<br>
Mark IV refrigerated bench-top centrifuge). Spin the tubes at 3,000 rpm for 30 minutes.<br>
Return the unopened buckets to the cabinet. Remove the centrifuge tubes and pour<br>
the supernatant into a waste bottle for GTC lysis buffer.<br>
Resuspend each pellet in 1 ml of Trizol (formulation of phenol and GTC cat no. 15596-<br>
026). The manufacturers guidelines recommend lysing cells by repetitive pipetting.<br>
Although this action alone will not lyse M. tuberculosis, it is important to completely<br>
resuspend the pellet in Trizol.<br>
Transfer 1 ml of cells into a FastRNA tube and ribolyse it at power setting 6.5 for 45<br>
seconds.<br>
Leave the tube to incubate at room temperature for 5 minutes.<br>
Remove the aqueous layer from the tube and add this to 200 pi of chloroform in a<br>
screw-capped eppendorf tube. Shake each tube vigorously for about 15 seconds.<br>
Incubate for 2-3 minutes at room temperature.<br>
Spin the tube at 13,000 rpm for 15 minutes. Following centrifugation, the liquid<br>
separates into red phenol/chloroform phase, an interface, and a clear aqueous phase.<br>
Carefully remove the aqueous phase and transfer it to a fresh eppendorf tube<br>
containing 500 pi of chloroform/isoamyl alcohol (24:1). Spin the tubes at 13,000 rpm<br>
for 15 minutes.<br>
Transfer the aqueous phase to an eppendorf tube containing 50 µl of sodium acetate<br>
and 500 µl of isopropanol.<br>
Surface decontaminate the eppendorf tube with 5% Hycolin for 5 minutes. Remove<br>
the tube from the CL3 laboratory and continue with the procedure in laboratory 157.<br>
Steps performed at Containment level 2:<br>
Precipitate the RNA at -70 °C for at least 30 minutes-can do this step overnight.<br>
Spin the precipitated RNA down at 13,000 rpm for 10 minutes. Remove the<br>
supernatant and wash the pellet in 70 % ethanol. Repeat centrifugation.<br>
Remove the 70 % ethanol and air-dry the pellet. Dissolve the pellet in RNAse free<br>
water.<br>
Freeze the RNA at -70 °C to store it<br>
Example 3 - isolation of genomic DNA from Mycobacterium tuberculosis<br>
grown<br>
in chemostat culture. DNA then used to generate Cy3 or Cy5 labelled DNA for<br>
use as a control in microarray experiments.<br>
Materials and Methods<br>
Beads 0.5 mm in diameter<br>
Bead beater<br>
Bench top centrifuge<br>
Platform rocker<br>
Heat block<br>
Falcon 50 ml centrifuge tubes<br>
Sorvall RC-5C centrifuge<br>
250 ml polypropylene centrifuge pots.<br>
Screw capped eppendorf tubes<br>
Pipettes 1 ml, 200 pi, 10 ml, 5 ml<br>
Breaking buffer<br>
50 mM Tris HCI pH 8.0<br>
10 mM EDTA<br>
100 mM NaCI<br>
Procedure<br>
Mechanical disruption of M. tuberculosis cells<br>
* 150 ml of chemostat cells (O.D of 2.5 at 540 nm) are spun down at 15,000 rpm for<br>
15 minutes in 250 ml polypropylene pots using centrifuge Sorvall RC-5C.<br>
* The supernatant is discarded.<br>
* Cells are re-suspended in 5 ml of breaking buffer in a 50 ml Falcon tube and<br>
centrifuged at 15,000 rpm for a further 15 minutes.<br>
* The supernatant is removed and additional breaking buffer is added at a volume of<br>
5 ml. Beads are used to disrupt the cells. These are used at a quantity of 1ml of beads<br>
for 1 ml of cells. Place the sample into the appropriate sized chamber. Place in the<br>
bead beater and secure the outer unit (containing ice) and process at the desired<br>
speed for 30 seconds.<br>
* Allow the beads to settle for 10 minutes and transfer cell lysate to a 50 ml Falcon<br>
centrifuge tube<br>
* Wash beads with 2-5 ml of breaking buffer by pipetting washing buffer up and down<br>
over the beads.<br>
* Add this washing solution to the lysate in the falcon tube<br>
Removal of proteins and cellular components<br>
* Add 0.1 volumes of 10% SDS and 0.01 volumes proteinase K.<br>
* Mix by inversion and heat at 55 °C in a heat block for 2-3 hours<br>
* The resulting mix should be homogenous and viscous. Additional SDS may be<br>
added to assist here to bring the concentration up to 0.2 %<br>
* Add an equal volume of phenol/chloroform/lsoamyl alcohol in the ratio: 25/24/1.<br>
* Gently mix on a platform rocker until homogenous<br>
* Spin down at 3,000 rpm for 20 minutes<br>
* Remove the aqueous phase and place in a fresh tube<br>
* Extract the aqueous phase with an equal volume of chloroform to remove traces of<br>
cell debris and phenol. Chloroform extractions may need to be repeated to remove all<br>
the debris.<br>
* Precipitate the DNA with 0.3 M sodium acetate and an equal volume of isopropanol.<br>
* Spool as much DNA as you can with a glass rod<br>
* Wash the spooled DNA in 70 % ethanol followed by 100 % ethanol<br>
* Leave to air dry<br>
* Dissolve the DNA in sterile deionised water (500 pi)<br>
* Allow DNA to dissolve at 4 °C for approximately 16 hours.<br>
* Add RNase 1 (500U) to the dissolved DNA<br>
* Incubate for 1 hour at 37 °C.<br>
* Re-extract with an equal volume of phenol/chloroform followed by a chloroform<br>
extraction and precipitate as before<br>
* Spin down the DNA at 13,000 rpm<br>
* Remove the supernatant and wash the pellet in 70% ethanol<br>
* Air dry<br>
* Dissolve in 200-500 µl of sterile water.<br>
Example 4 - preparation of Cy3 or Cy5 labelled DNA from DNA<br>
Incubate the microarray slide in the pre-heated prehybridisation solution at 65 °C for<br>
20 min. Rinse slide thoroughly in 400 ml H20 for 1 min followed by rinse in 400 ml<br>
propan-2-ol for 1 min and centrifuge slide in 50 ml centrifuge tube at 1,500 rpm for 5<br>
min to dry. Store slide in dark, dust-free box until hybridisation (
c\ Purify Cv3/Cv5 labelled DNA - Qiaaen MinElute Purification<br>
* Combine Cy3 and Cy5 labelled DNA samples in single tube and add 500 ul Buffer<br>
PB. ......._<br>
* Apply to MinElute column in collection tube and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 500 pi Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 250 ul Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Centrifuge at 13,000 rpm for an additional 1 min to remove residual ethanol.<br>
* Place the MinElute column into a fresh 1.5 ml tube.<br>
* Add 10.5 µl H2O to the centre of the membrane and allow to stand for 1 min.<br>
* Centrifuge at 13,000 rpm for 1 min.<br>
Example 5 - preparation of Cy3 or Cy5 label cDNA from RNA<br>
Incubate the microarray slide in the pre-heated prehybridisation solution at 65 °C for<br>
20 min. Rinse slide thoroughly in 400 ml H2Ofor 1 min followed by rinse in 400 ml<br>
propan-2-ol for 1min and centrifuge slide in 50 ml centrifuge tube at 1500 rpm for 5<br>
min to dry. Store slide in dark, dust-free box until hybridisation (
c) Purify Cy3/Cy5 labelled cDNA - Qiagen MinElute Purification<br>
* Combine Cy3 and Cy5 labelled DNA samples in single tube and add 250 µl Buffer<br>
PB.<br>
* Apply to MinElute column in collection tube and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 500 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 250 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Centrifuge at 13,000 rpm for an additional 1 min to remove residual ethanol.<br>
* Place the MinElute column into a fresh 1.5 ml tube.<br>
* Add 10.5 µl H2O to the centre of the membrane and allow to stand for 1 min.<br>
* Centrifuge at 13,000 rpm for 1 min.<br>
Example 6 - hybridise slide with Cy3/Cy5 labelled cDNA/DNA<br>
Place the prehybridise microarray slide in the hybridisation cassette and add two 15<br>
ml aliquots of H2O to the wells in the cassette. Mix resuspended Cy3/Cy5 labelled<br>
cDNA/DNA sample with hybridisation solution.<br>
Heat hybridisation solution at 95 °C for 2 min. Do not snap cool on ice but allow to cool<br>
slightly and briefly centrifuge. Pipette the hybridisation solution onto the slide at the<br>
edge of the arrayed area avoiding bubble formation. Using forceps carefully drag the<br>
edge of a cover slip along the surface of the slide towards the arrayed area and into<br>
the hybridisation solution at the edge of the array. Carefully lower the cover slip down<br>
over the array avoiding any additional movement once in place. Seal the hybridisation<br>
cassette and submerge in a water bath at 60 °C for 16-20 h.<br>
Wash slide.<br>
Remove microarray slide from hybridisation cassette and initially wash slide carefully<br>
in staining trough of Wash A to remove cover slip. Once cover slip is displaced place<br>
slide(s) in slide rack and continue agitating in Wash A for a further 2 min.<br>
Place slide into a 50 ml centrifuge tube and centrifuge at 1500 rpm for 5 mins to dry<br>
slide, and then scan fluorescence using a ScanArray 3000 dual-laser confocal<br>
scanner and analyse data.<br>
Example 7 - preparation of the arrays<br>
PCR-amplified products are generated from M. tuberculosis genomic DNA using ORF-<br>
specific primers. Each gene of the genome is represented. These are spotted in a grid<br>
onto a standard glass microscope slide using a BioRobotics microgrid robot (MWG<br>
Biotech) at a resolution of &gt;4000 spots/cm2.<br>
Example 8 - scanning and analysis of data<br>
The slides were scanned using an Affymetrix 428 scanner.<br>
Dual fluorescence is used, allowing simultaneous detection of two cDNA samples. The<br>
output of the arrays is read using a confocal laser scanner (Affymetrix 428 scanner<br>
from MWG Biotech). More detailed information can be found web site<br>
www.sghms.ac.uk/depts/medmicro/bugs; Mujumdar, R.B. (1993) Bioconjugate<br>
Chemistry, 4(2). pp.105-111; Yu, H. (1994) Nucl. Acids Res. 22, pp.3226-3232; and<br>
Zhu, Z. (1994) Nucl. Acids Res. 22, pp. 3418-3422.<br>
The raw data were initially analysed in software known as ImaGene, which was<br>
supplied with the scanner. The scanned images were then transferred to another<br>
software package known as GeneSpring. This is a very powerful tool, which draws<br>
information from many databases allowing the complete analysis of the expression of<br>
each gene.<br>
Example 9 - delete one or more of the genes from M. tuberculosis in order to<br>
attenuate its virulence while retaining immunogenicity<br>
One or more genes that are identified may be disrupted using allelic exchange. In<br>
brief, the gene of interest is cloned with 1-2 kb of flanking DNA either side and is<br>
inactivated by deletion of part of the coding region and insertion of an antibiotic<br>
resistance marker, such as hygromycin.<br>
The manipulated fragment is then transferred to a suitable suicide vector e.g. pPR23<br>
and is transformed into the wild-type parent strain of M. tuberculosis. Mutants are<br>
recovered by selecting for antibiotic resistant strains. Genotypic analysis (Southern<br>
Blotting with a fragment specific to the gene of interest) is performed on the selected<br>
strains to confirm that the gene has been disrupted.<br>
The mutant strain is then studied to determine the effect of the gene disruption on the<br>
phenotype. In order to use it as a vaccine candidate it would be necessary to<br>
demonstrated attenuated virulence. This can be done using either a guinea pig or<br>
mouse model of infection. Animals are infected with the mutant strain and the<br>
progression of disease is monitored by determining the bacterial load in different<br>
organs, in particular the lung and spleen, at specific time points post infection, typically<br>
up to 16 weeks.<br>
Comparison is made to animals infected with the wild-type strain which should have<br>
a significantly higher bacterial load in the different organs. Long-term survival studies<br>
and histopathology can also be used to assess virulence and pathogenicity.<br>
Once attenuated virulence has been established, protection and immunogenicity<br>
studies can be performed to assess the potential of the strain as a vaccine. Suitable<br>
references for allelic exchange and preparation of TB mutants are McKinney et al.,<br>
2000 and Pelicic et al., 1997, [1,2].<br>
Example 10 - select one or more of the genes identifiable by the present<br>
invention, which encode proteins that are immunogenic, and put them into BCG<br>
or an attenuated strain of M. tuberculosis to enhance its overall immunogenicity<br>
The gene of interest is amplified from the M. tuberculosis genome by PCR. The<br>
amplified product is purified and cloned into a plasmid (pMV306) that integrates site<br>
specifically Into the mycobacterial genome at the attachment site (attB) for<br>
mycobacteriophage L5 [3].<br>
BCG is transformed with the plasmid by electroporation, which involves damaging the<br>
cell envelope with high voltage electrical pulses, resulting in uptake of the DNA. The<br>
plasmid integrates into the BCG chromosome at the attB site generating stable<br>
recombinants. Recombinants are selected and are checked by PCR or Southern<br>
blotting to ensure that the gene has been integrated. The recombinant strain is then<br>
used for protection studies.<br>
Example 11- Use of recombinant carriers such as attenuated salmonella and<br>
the<br>
Vaccinia virus to express and present TB genes.<br>
One of the best examples of this type of approach is the use of Modified Vaccinia virus<br>
Ankara (MVA) [4]. The gene of interest is cloned into a vaccinia virus shuttle vector,<br>
e.g. pSC11. Baby Hamster Kidney (BHK) cells are then infected with wild-type MVA<br>
and are transfected with the recombinant shuttle vector. Recombinant virus is then<br>
selected using a suitable selection marker and viral plaques, selected and purified.<br>
Recombinant virus is normally delivered as part of a prime-boost regime where<br>
animals are vaccinated initially with a DNA vaccine encoding the TB genes of interest<br>
under the control of a constitutive promoter. The immune response is boosted by<br>
administering recombinant MVA carrying the genes of interest to the animals at least<br>
2 weeks later.<br>
Example 12 - Sub-unit vaccines containing a single peptide/protein or a<br>
combination of proteins<br>
To prepare sub-unit vaccines with one or more peptides or proteins it is first of all<br>
necessary to obtain a supply of protein or peptide to prepare the vaccine. Up to now,<br>
this has mainly been achieved in mycobacterial studies by purifying proteins of interest<br>
from TB culture. However, it is becoming more common to clone the gene of interest<br>
and produce a recombinant protein.<br>
The coding sequence for the gene of interest is amplified by PCR with restriction sites<br>
inserted at the N terminus and C terminus to permit cloning in-frame into a protein<br>
expression vector such as pET-15b. The gene is inserted behind an inducible<br>
promoter such as lacZ. The vector is then transformed into E. coli which is grown in<br>
culture. The recombinant protein is over-expressed and is purified.<br>
One of the common purification methods is to produce a recombinant protein with an<br>
N-terminal His-tag. The protein can then be purified on the basis of the affinity of the<br>
His-tag for metal ions on a Ni-NTA column after which the His-tag is cleaved. The<br>
purified protein is then administered to animals in a suitable adjuvant [5].<br>
Example 13 - Plasmid DNA vaccines carrying one or more of the identified<br>
genes<br>
DNA encoding a specific gene is amplified by PCR, purified and inserted into<br>
specialised vectors developed for vaccine development, such as pVAX1. These<br>
vectors contain promoter sequences, which direct strong expression of the introduced<br>
DNA (encoding candidate antigens) in eukaryotic cells (eg. CMV or SV40 promoters),<br>
and polyadenlyation signals (eg. SV40 or bovine growth hormone) to stabilise the<br>
mRNA transcript.<br>
The vector is transformed into E. coli and transformants are selected using a marker,<br>
such as kanamycin resistance, encoded by the plasmid. The plasmid is then<br>
recovered from transformed colonies and is sequenced to check that the gene of<br>
interest is present and encoded properly without PCR generated mutations.<br>
Large quantities of the plasmid is then produced in E. co// and the plasmid is<br>
recovered and purified using commercially available kits (e.g. Qiagen<br>
Endofree-plasmid preparation). The vaccine is then administered to animals for<br>
example by intramuscular injection in the presence or absence of an adjuvant.<br>
Example 14 - Preparation of DNA expression vectors<br>
DNA vaccines consist of a nucleic acid sequence of the present invention cloned into<br>
a bacterial plasmid. The plasmid vector pVAX1 is commonly used in the preparation<br>
of DNA vaccines. The vector is designed to facilitate high copy number replication in<br>
E. coli and high level transient expression of the peptide of interest in most mammalian<br>
cells (for details see manufacturers protocol for pVAX1 (catalog No. V260-20<br>
www.invitrogen.com).<br>
The vector contains the following elements:-<br>
* Human cytomegalovirus Immediate-early (CMV) promoterfor high-level expression<br>
in a variety of mammalian cells<br>
* T7 promoter/priming site to allow in vitro transcription in the sense orientation and<br>
sequencing through the insert<br>
* Bovine growth hormone (BGH) polyadenylation signal for efficient transcription<br>
termination and polyadenylation of mRNA<br>
* Kanamycin resistance gene for selection in E. coli<br>
* A multiple cloning site<br>
* pUC origin for high-copy number replication and growth in E. coli<br>
* BGH reverse priming site to permit sequencing through the insert<br>
Vectors may be prepared by means of standard recombinant techniques which are<br>
known in the art, for example Sambrook et al. (1989). Key stages in preparing the<br>
vaccine are as follows:<br>
* The gene of interest is ligated into pVAX1 via one of the multiple cloning sites<br>
* The ligation mixture is then transformed into a competent E. coli strain (e.g. TOP 10)<br>
and LB plates containing 50µg/ml kanamycin are used to select transformants.<br>
* Clones are selected and may be sequenced to confirm the presence and orientation<br>
of the gene of interest.<br>
* Once the presence of the gene has been verified, the vector can be used to transfect<br>
a mammalian cell line to check for protein expression. Methods for transfection are<br>
known in the art and include, for example, electroporation, calcium phosphate, and<br>
lipofection.<br>
* Once peptide expression has been confirmed, large quantities of the vector can be<br>
produced and purified from the appropriate cell host, e.g E. coli.<br>
pVAX1 does not integrate into the host chromosome. All non-essential sequences<br>
have been removed to minimise the possibility of integration. When constructing a<br>
specific vector, a leader sequence may be included to direct secretion of the encoded<br>
protein when expressed inside the eukaryotic cell.<br>
Other examples of vectors that have been used are V1 Jns.tPA and pCMV4 (Lefevre<br>
et al., 2000 and Vordermeier et al., 2000).<br>
Expression vectors may be used that integrate into the genome of the host, however,<br>
it is more common and more preferable to use a vector that does not integrate. The<br>
example provided, pVAX1, does not integrate. Integration would lead to the generation<br>
of a genetically modified host which raises other issues.<br>
Example 15 - RNA vaccine<br>
As discussed on page 15 of US5,783,386, one approach is to introduce RNA directly<br>
into the host<br>
Thus, the vector construct (Example 14) may be used to generate RNA in vitro and<br>
the purified RNA then injected into the host. The RNA would then serve as a template<br>
for translation in the host cell. In this embodiment, integration would not normally<br>
occur.<br>
Another option is to use an infectious agent such as the retroviral genome carrying<br>
RNA corresponding to the gene of interest. In this embodiment, integration into the<br>
host genome will occur.<br>
Another option is the use of RNA replicon vaccines which can be derived from virus<br>
vectors such as Sindbis virus or Semliki Forest virus. These vaccines are<br>
self-replicating and self-limiting and may be administered as either RNA or DNA which<br>
is then transcribed into RNA replicons in vivo. The vector eventually causes lysis of<br>
the transfected cells thereby reducing concerns about integration into the host<br>
genome. Protocols for RNA vaccine construction are detailed in Cheng et al., (2001).<br>
Example 16 - Diagnostic assays based on assessing T cell responses<br>
For a diagnostic assay based on assessing T cell responses it would be sufficient to<br>
obtain a sample of blood from the patient. Mononuclear cells (monocytes, T and B<br>
lymphocytes) can be separated from the blood using density gradients such as Ficoll<br>
gradients.<br>
Both monocytes and B-lymphocytes are both able to present antigen, although less<br>
efficiently than professional antigen presenting cells (APCs) such as dendritic cells.<br>
The latter are more localised in lymphoid tissue.<br>
The simplest approach would be to add antigen to the separated mononuclear cells<br>
and incubate for a week and then assess the amount of proliferation. If the individual<br>
had been exposed to the antigen previously through infection, then T-cell closes<br>
specific to the antigen should be more prevalent in the sample and should respond.<br>
It is also possible to separate the different cellular populations should it be desired to<br>
control the ratio of T cells to APC's.<br>
Another variation of this type of assay is to measure cytokine production by the<br>
responding lymphocytes as a measure of response. The ELISPOT assay described<br>
below in Example 17 is a suitable example of this variation.<br>
Example 17 - detection of latent mycobacteria<br>
A major problem for the control of tuberculosis is the presence of a large reservoir of<br>
asymptomatic individuals infected with, tubercle bacilli. Dormant bacilli are more<br>
resistant to front-line drugs.<br>
The presence of latent mycobacteria-associated antigen may be detected indirectly<br>
either by detecting antigen specific antibody or T-cells in blood samples.<br>
The following method is based on the method described in Lalvani et al. (2001) in<br>
which a secreted antigen, ESAT-6, was identified as being expressed by members of<br>
the M. tuberculosis complex but is absent from M. Bovis BCG vaccine strains and<br>
most environmental mycobacteria. 60 - 80% of patients also have a strong cellular<br>
immune response to ESAT-6. An ex-vivo ELISPOT assay was used to detect ESAT-6<br>
specific T cells.<br>
As applied to the present invention:<br>
A 96 well plate is coated with cytokine (e.g. interferon-?, IL-2) -specific antibody.<br>
Peripheral blood monocytes are then isolated from patient whole blood and are<br>
applied to the wells.<br>
Antigen (ie. one of the peptides, fragments, derivatives or variants of the present<br>
invention) is added to stimulate specific T cells that may be present and the plates are<br>
incubated for 24h. The antigen stimulates cytokine production which then binds to the<br>
specific antibody.<br>
The plates are washed leaving a footprint where antigen-specific T cells were present.<br>
A second antibody coupled with a suitable detection system, e.g. enzyme, is then<br>
added and the number of spots are enumerated after the appropriate substrate has<br>
been added.<br>
The number of spots, each corresponding to a single antigen-specific T cell, is related<br>
to the total number of cells originally added.<br>
The above Example also describes use of an antigen that may be used to distinguish<br>
TB infected individuals from BCG vaccinated individuals. This could be used in a more<br>
discriminative diagnostic assay.<br>
Example 18 - In vitro model for mycobacterial persistence under the joint<br>
conditions of carbon-starvation and oxygen-limitation (a variation on Examples<br>
1-8)<br>
Materials and Methods<br>
Studies were performed with M. tuberculosis strain H37Rv (NCTC cat No. 7416) - a<br>
representative strain of M. tuberculosis. Stock cultures were grown on Middlebrook<br>
7H10 + OADC for 3 weeks at 37±2 °C.<br>
Culture Medium<br>
. Persistence cultures were established in Middlebrook 7H9 medium supplemented with<br>
Middlebrook ADC enrichment, 0.2 % Tween 80 and 0.2 % glycerol (see below). The<br>
medium was prepared with high quality water from a Millipore water purification<br>
system and filter sterilised by passage through a 0.1 urn pore size cellulose acetate<br>
membrane filter capsule (Sartorius Ltd). The pH was adjusted to 6.6 with concentrated<br>
hydrochloric acid.<br>
Middlebrook 7H10 + OADC agar was used to prepare inoculum cultures, enumerate<br>
the number of culturable bacteria in samples, and to assess culture purity..<br>
Culture system<br>
We used the culture system described in WO00/52139, operated as a patch<br>
fermenter, for this Example.<br>
Culture experiments were performed in a one litre glass vessel operated at a working<br>
volume of 750 ml. The culture was agitated by a magnetic bar placed in the culture<br>
vessel coupled to a magnetic stirrer positioned beneath the vessel. Culture conditions<br>
were continuously monitored by an Anglicon Microlab Fermentation System (Brighton<br>
Systems, Newhaven), linked to sensor probes inserted into the culture through sealed<br>
ports in the top plate. The oxygen concentration was monitored with a galvanic oxygen<br>
electrode (Uniprobe, Cardiff) and was controlled through sparging the culture with a<br>
mixture of air and oxygen free-nitrogen. Temperature was monitored by an Anglicon<br>
temperature probe, and maintained by a heating pad positioned beneath the culture<br>
vessel. Culture pH was measured using an Ingold pH electrode (Mettler-Toledo,<br>
Leicester).<br>
Inoculation and culture<br>
The vessel was filled with 750 ml of sterile culture medium and parameters were<br>
allowed to stabilise at 37 °C ± 2 °C, pH 6.9 ± 0.3 and a dissolved oxygen tension of<br>
approximately 70 % air saturation. A dense inoculum suspension was prepared by<br>
resuspending Middlebrook agar cultures, grown at 37±2 °C for 3 week, in sterile<br>
deionised water. The inoculum was aseptically transferred to the culture vessel, to<br>
provide an initial culture turbidity of approximately 0.25 at 540 nm. The culture was<br>
maintained at 37 °C with an agitation rate of 500 to 750 rpm.<br>
After inoculation, the dissolved oxygen tension (DOT) of the culture was maintained<br>
at approximately 40% air saturation at 37 °C until the culture had entered early<br>
exponential growth. The DOT was then lowered in increments down to 1 % air<br>
saturation over a six day period (Fig. 3). The culture was then maintained at a DOT<br>
of 0 - 5 % until 50 days after inoculation and samples were removed regularly to<br>
monitor growth and survival, nutrient utilisation and gene expression.<br>
Growth and survival<br>
Bacterial growth and survival was assessed by determining the number of viable cells<br>
in the culture system at specific time points. This was achieved by preparing a decimal<br>
dilution series of the sample in sterile water and plating 100(1 aliquots onto<br>
Middlebrook 7H10 + OADC plates. The plates were incubated at 37 °C for up to 4<br>
weeks before enumerating the number of colonies formed.<br>
Nutrient utilisation<br>
Glycerol is the primary carbon and energy source present in Middlebrook 7H9 medium<br>
with ADC, 0.2 % Tween and Glycerol. The rate at which glycerol was utilised was<br>
determined using the Glycerol Determination Kit Cat. No. 148 270 Boehringer<br>
Mannheim.<br>
Microarrav experiments<br>
RNA was extracted from culture samples collected at different time points during the<br>
experiment A fluorescently-labelled cDNA was then transcribed from each sample of<br>
RNA. The cDNA was labelled by the incorporation of either Cy3 or Cy5 labelled dCTP<br>
(Dyes are supplied by Amersham Pharmacia Biotech).<br>
Whole M. tuberculosis genome arrays were prepared from M. tuberculosis genomic<br>
DNA using ORF-specific primers. PCR products corresponding to each ORF were<br>
spotted in a grid onto a standard glass microscope slide using a BioRobotics microgrid<br>
robot (MWG Biotech) at a resolution of &gt;4000 spots/cm2. Arrays were supplied by Dr<br>
P Butcher, St George's Hospital Medical School London.<br>
In each microarray experiment a whole genome array was hybridised with labelled<br>
cDNA from one culture sample (Test sample). Each array was also hybridised with<br>
control DNA incorporating a different Cy dye and prepared from DNA extracted from<br>
M. tuberculosis strain H37Rv (control sample). Each array was scanned, using an<br>
Affymetrix 428 scanner, at two different wavelengths corresponding to the excitation<br>
maxima of each dye and the intensity of the emitted light was recorded. The raw data<br>
was processed by ImaGene software before performing comparative analysis using<br>
. GeneSpring.<br>
Analysis of viable count data indicated that the culture grew exponentially until 10 to<br>
12 days post infection (Fig. 4). As the culture entered stationary phase, viability started<br>
to decline and continued to decline steadily over the duration of the study. After 40<br>
days in stationary phase, approximately 0.1 % of the culture was still cultivable on<br>
Middlebrook agar. The rate of glycerol utilisation was slower than observed in the<br>
culture established under aerobic conditions, indicating that the metabolic activity of<br>
the low-oxygen culture was restricted by limited oxygen availability. Nevertheless, the<br>
principal carbon and energy source was depleted within 15 days after inoculation (Fig.<br>
2).<br>
Samples were collected for microarray analysis as outlined. The gene expression<br>
profiles for samples collected at day 5 and 50 were compared. Three arrays were<br>
prepared for each sample and the test data was normalised against the control data<br>
on each chip. The normalised data for each set of arrays was then averaged and the<br>
two data sets were compared. Those genes that were expressed 5-fold higher at day<br>
5 relative to day 50 were selected. The gene list was compared with the list generated<br>
from the carbon starvation model (ie. Table 2), and a sub-list of genes which were only<br>
down-regulated under the combined conditions of carbon starvation and<br>
oxygen-limitation was generated (see Table 3).<br>
Liquid medium formulation for persistence cultures - Middlebrook 7H9 medium<br>
supplemented with ADC, 0.2 % Tween 80 and 0.2 % Glycerol<br>
1. RNA extraction from M. tuberculosis for microarray analysis<br>
Materials and Methods<br>
* Trizol (Life Technologies) - formulation of phenol and guanidine thiocyanate.<br>
* GTC lysis solution containing: 5 M guanidine thiocyanate, 0.5 % N-lauryl sarcosine,<br>
25 mM tri-sodium citrate, 0.1 M 2-mercaptoethanol, and 0.5 % Tween 80.<br>
* Chloroform<br>
* Isopropanol<br>
* 3 M sodium acetate<br>
*70%Ethanol<br>
* microfuge<br>
* ribolyser<br>
* Sterile plasticware-Falcon tubes, screw capped eppendorfs, gilson tips -all RNase<br>
free<br>
* Glassware - baked at 160 °C for at least 16 hours<br>
Method<br>
* Steps performed at Containment level 3; within a Class III microbiological safety<br>
cabinet.<br>
* Remove 10 or 20 ml of culture (109/ml) and immediately add this to 4 volumes of<br>
GTC lysis buffer in a plastic specimen pot. Seal the pot tightly.<br>
* Incubate the cells in GTC lysis buffer for 1 hour at room temperature. Surface<br>
decontaminate the plastic pot with 5 % Hycolin for 5 minutes. Transfer the sample to<br>
. the pass box and place it into a plastic carry tin with a sealable lid. Close the container<br>
securely and transport it to a non-toxic cabinet CL3 cabinet.<br>
* Equally distribute the lysis mixture between Falcon tubes. Place these tubes into<br>
centrifuge buckets and seal the buckets tightly. Surface-decontaminate the buckets<br>
for 5 minutes with 5 % Hycolin. Then transfer them to the centrifuge (Baird and Tatlock<br>
Mark IV refrigerated bench-top centrifuge). Spin the tubes at 3,000 rpmfor30 minutes.<br>
* Return the unopened buckets to the cabinet. Remove the centrifuge tubes and pour<br>
-the supernatant into a waste bottle for GTC lysis buffer.<br>
* Resuspend each pellet in 5 ml of Trizol (formulation of phenol and GTC cat No.<br>
15596-026). The manufacturers guidelines recommend lysing cells by repetitive<br>
pipetting. Although this action atone will not lyse M. tuberculosis, it is important to<br>
completely resuspend the pellet in Trizol.<br>
* Transfer 1 ml of cells into each FastRNA tube and ribolyse them at power setting 6.5<br>
for 45 seconds.<br>
* Leave the tubes to incubate at room temperature for 5 minutes.<br>
* Remove the aqueous layer from each tube and add this to 200 ul of chloroform in<br>
a screw-capped eppendorf tube. Shake each tube vigorously for about 15 seconds.<br>
Incubate for 2-3 minutes at room temperature.<br>
* Spin the tubes at 13,000 rpm for 15 minutes. Following centrifugation, the liquid<br>
separates into red phenol/chloroform phase, an interface, and a clear aqueous phase.<br>
* Carefully remove the aqueous phase and transfer it to fresh eppendorf tubes<br>
containing 500 µl of chloroform/isoamyl alcohol (24:1). Spin the tubes at 13,000 rpm<br>
for 15 minutes.<br>
* Transfer the aqueous phase to eppendorf tubes containing 50 µl of sodium acetate<br>
and 500 ul of isopropanol.<br>
* Surface decontaminate the eppendorf tubes with 5% Hycolin for 5 minutes. Remove<br>
the tubes from the CL3 laboratory and continue with the procedure in laboratory 157.<br>
* Steps performed at Containment level 2:<br>
* Precipitate the RNA at -70 °C for at least 30 minutes (optionally overnight).<br>
* Spin the precipitated RNA down at 13,000 rpm for 10 minutes. Remove the<br>
supernatant and wash the pellet in 70 % ethanol. Repeat centrifugation.<br>
* Remove the 70 % ethanol and air-dry the pellet. Dissolve the pellet in RNAse free<br>
water.<br>
* Freeze the RNA at -70 °C to store it<br>
2. Isolation of genomic DNA from Mycobacterium tuberculosis grown in chemostat<br>
culture. DNA then used to generate Cy3 or Cy5 labelled DNA for use as a control in<br>
microarray experiments<br>
Procedure<br>
Mechanical disruption of Mtb cells<br>
* 150 ml of chemostat cells (O.D of 2.5 at 540 nm) are spun down at 15,000 rpm for<br>
15 minutes in 250 ml polypropylene pots using centrifuge Sorvall RC-5C.<br>
* The supernatant is discarded.<br>
* Cells are re-suspended in 5 ml of breaking buffer in a 50 ml Falcon tube and<br>
centrifuged at 15,000 rpm for a further 15 minutes.<br>
* The supernatant is removed and additional breaking buffer is added at a volume of<br>
5 ml. Beads are used to disrupt the cells. These are used at a quantity of 1 ml of beads<br>
for 1 ml of cells. Place the sample into the appropriate sized chamber. Place in the<br>
bead beater and secure the outer unit (containing ice) and process at the desired<br>
speed for 30 seconds.<br>
* Allow the beads to settle for 10 minutes and transfer cell lysate to a 50 ml Falcon<br>
centrifuge tube<br>
* Wash beads with 2-5 ml of breaking buffer by pipetting washing buffer up and down<br>
over the beads.<br>
* Add this washing solution to the lysate in the falcon tube<br>
Removal of proteins and cellular components.<br>
* Add 0.1 volumes of 10 % SDS and 0.01 volumes proteinase K.<br>
* Mix by inversion and heat at 55 °C in a heat block for 2-3 hours<br>
*The resulting mix should be homogenous and viscous. If it isn't then add more SDS<br>
to bring the concentration up to 0.2 %<br>
* Add an equal volume of phenol/chloroform/lsoamyl alcohol in the ratio: 25/24/1.<br>
* Gently mix on a platform rocker until homogenous<br>
* Spin down at 3,000 rpm for 20 minutes<br>
* Remove the aqueous phase and place in a fresh tube<br>
* Extract the aqueous phase with an equal volume of chloroform to remove traces of<br>
cell debris and phenol. Chloroform extractions may need to be repeated to remove all<br>
the debris.<br>
* Precipitate the DNA with 0.3 M sodium acetate and an equal volume of isopropanol.<br>
* Spool as much DNA as you can with a glass rod<br>
* Wash the spooled DNA in 70 % ethanol followed by 100 % ethanol<br>
* Leave to air dry<br>
* Dissolve the DNA in sterile deionised water (500 µl)<br>
* Allow DNA to dissolve at 4 °C for approximately 16 hours.<br>
* Add RNase 1 (500U) to the dissolved DNA<br>
* Incubate for 1 hour at 37 °C.<br>
* Re-extract with an equal volume of phenol/chloroform followed by a chloroform<br>
extraction and precipitate as before<br>
* Spin down the DNA at 13,000 rpm<br>
* Remove the supernatant and wash the pellet in 70 % ethanol<br>
* Air dry<br>
* Dissolve in 200-500 µl of sterile water.<br>
Incubate the microarray slide in the pre-heated prehybridisation solution at 65 °C for<br>
20 min. Rinse slide thoroughly in 400 ml H2O for 1 min followed by rinse in 400 ml<br>
propan-2-ol for 1 min and centrifuge slide in 50 ml centrifuge tube at 1,500 rpm for 5<br>
min to dry. Store slide in dark, dust-free box until hybridisation (
c) Purify Cy3/Cy5 labelled DNA - Qiagen MinElute Purification<br>
* Combine Cy3 and Cy5 labelled DNA samples in single tube and add 500 µl Buffer<br>
PB.<br>
* Apply to MinElute column in collection tube and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 500 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 250 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Centrifuge at 13,000 rpm for an additional 1 min to remove residual ethanol.<br>
* Place the MinElute column into a fresh 1.5 ml tube.<br>
* Add 10.5 µl H2O to the centre of the membrane and allow to stand for 1 min.<br>
* Centrifuge at 13,000 rpm for 1 min.<br>
Incubate the microarray slide in the pre-heated prehybridisation solution at 65 °C for<br>
20 min. Rinse slide thoroughly in 400 ml H2O for 1 min followed by rinse in 400 ml<br>
propan-2-ol for 1 min and centrifuge slide in 50 ml centrifuge tube at 1500 rpm for 5<br>
min to dry. Store slide in dark, dust-free box until hybridisation (
c) Purify Cy3/Cy5 labelled cDNA - Qiagen MinElute Purification<br>
* Combine Cy3 and Cy5 labelled DNA samples in single tube and add 250 µl Buffer<br>
PB.<br>
* Apply to MinElute column in collection tube and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube..<br>
* Add 500 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1 min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Add 250 µl Buffer PE to MinElute column and centrifuge at 13,000 rpm for 1min.<br>
* Discard flow-through and place MinElute column back into same collection tube.<br>
* Centrifuge at 13,000 rpm for an additional 1 min to remove residual ethanol.<br>
* Place the MinElute column into a fresh 1.5 ml tube.<br>
* Add 10.5 µl H2O to the centre of the membrane and allow to stand for 1 min.<br>
* Centrifuge at 13,000 rpm for 1 min.<br>
5. Hybridise slide with Cy3/Cy5 labelled cDNA/DNA<br>
Place the prehybridise microarray slide in the hybridisation cassette and add two 15<br>
µl aliquots of H2O to the wells in the cassette. Mix resuspended Cy3/Cy5 labelled<br>
cDNA sample with hybridisation solution.<br>
Heat hybridisation solution at 95 °C for 2 min. Do NOT snap cool on ice but allow to<br>
cool slightly and briefly centrifuge. Pipette the hybridisation solution onto the slide at<br>
the edge of the arrayed area avoiding bubble formation. Using forceps carefully drag<br>
the edge of a cover slip along the surface of the slide towards the arrayed area and<br>
into the hybridisation solution at the edge of the array. Carefully lower the cover slip<br>
down over the array avoiding any additional movement once in place. Seal the<br>
hybridisation cassette and submerge in a water bath at 65 °C for 16-20 hours.<br>
Wash slide<br>
Remove microarray slide from hybridisation cassette and initially wash slide carefully<br>
in staining trough of Wash A preheated to 65 °C to remove cover slip. Once cover slip<br>
is displaced place slide(s) in slide rack and continue agitating in Wash A for a further<br>
2 min.<br>
Transfer slide(s) to a clean slide rack and agitate in first trough of Wash B for 2 min.<br>
Wash in second trough of Wash B with agitation for 2 min.<br>
Place slide into a 50 ml centrifuge tube and centrifuge at 1500 rpm for 5 mins to dry<br>
slide and then scan fluorescence.<br>
The genes identified in accordance with Example 18 are listed In Table 3. Those<br>
genes overlapping with Table 2 have been omitted from Table 3. The coding<br>
sequences for these genes (nucleic acid sequences are given from the transcription<br>
start site to the stop codon) and corresponding amino acid sequences are listed in the<br>
accompanying Sequence Listing starting from SEQ ID NO. 21 et seq (amino acid<br>
sequences are followed immediately by corresponding coding sequences for each<br>
gene).<br>
References:<br>
1. McKinney, J.D., et a!., Persistence of Mycobacterium tuberculosis in<br>
macrophages and mice requires the glyoxylate shunt enzyme isocitrate lyase [see<br>
comments]. Nature, 2000.406(6797): p. 735-8.<br>
2. Pelicic, V., et al., Efficient allelic exchange and transposon mutagenesis in<br>
Mycobacterium tuberculosis. Proc Natl Acad Sci USA, 1997. 94(20): p. 10955-60.<br>
3. Lee, M.H., et al., Site-specific integration of mycobacteriophage L5: integration-<br>
proficient vectors for Mycobacterium smegmatis, Mycobacterium tuberculosis, and<br>
bacille Calmette-Guerin. Proc Natl Acad Sci U S A, 1991.88(8): p. 3111-5.<br>
4. McShane, H., et al., Enhanced immunogenicity of CD4(+) t-cell responses and<br>
protective efficacy of a DNA-modified vaccinia virus Ankara prime-boost vaccination<br>
regimen for murine tuberculosis. Infect Immun, 2001. 69(2): p. 681-6.<br>
5. Movahedzadeh, F., M.J. Colston, and E.O. Davis, Characterization of<br>
Mycobacterium tuberculosis LexA: recognition of a Cheo (Bacillus-type SOS) box.<br>
Microbiology, 1997.143(Pt 3): p. 929-36.<br>
Additional References:<br>
Cunningham, A. F. and C. L. Spreadbury. 1998. Mycobacterial stationary phase<br>
induced by low oxygen tension: cell wall thickening and localization of the 16-kilodalton<br>
alpha-crystallin homolog. J. Bacterid. 180:801-808.<br>
Lalvani, A. et al., 2001. Enhanced contact tracing and spatial tracking of<br>
Mycobacterium tuberculosis infection by enumeration of antigen-specific T cells. The<br>
Lancet 357:2017-2021.<br>
Rook, G. A. W. and B. R. Bloom. 1994. Mechanisms of pathogenesis in tuberculosis,<br>
pp 460-485. In B. R. Bloom (ed), Tuberculosis- pathogenesis, protection and control.<br>
ASM Press, Washington DC.<br>
Wayne, L. G. 1994. Dormancy of Mycobacterium tuberculosis and latency of disease.<br>
Eur. J. Clin. Microbiol. Infect. Dis. 13:908-914.<br>
Wayne, L. G. and L G. Hayes. 1996. An in vitro model for sequential study of<br>
shift-down of Mycobacterium tuberculosis through two stages of non-replicating<br>
persistence. Infect. Immun. 64:2062-2069.<br>
Wayne, L. G. and K. Lin. 1982. Glyoxylate metabolism and adaptation of<br>
Mycobacterium tuberculosis to survival under anaerobic conditions. Infect. Immun.<br>
37:1042-1049.<br>
Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory<br>
manual, 2nd ed. Cold Spring Harbour Laboratory Press, Cold Spring Harbour, N. Y.<br>
Lefever, P., O. Denis, L. De Wit, A. Tanghe, P. Vandenbussche, J. Content, and K.<br>
Huygen. 2000. Cloning of the gene encoding a 22-kilodalton cell surface antigen of<br>
Mycobacterium bovis BCG and analysis of its potential for DNA vaccination against<br>
tuberculosis. Infection and Immunity. 68:1040-1047.<br>
Vordermeire, H. M., P. J. Cockle, A. O. Whelan, S. Rhodes, M. A. Chambers, D.<br>
Clifford, K. Huygen, R. Tascon, D. Lowrie, M. J. Colston, and R. G. Hewinson. 2000.<br>
Effective DNA vaccination of cattle with the mycobacterial antigens MPB83 and MPB70<br>
does not compromise the specificity of the comparative intradermal tuberculin skin test.<br>
Vaccine. 19:1246-1255.<br>
Cheng, W., C. Hung, C. Chai, K. Hsu, L. He, C. Rice, M. Ling, and T. Wu. 2001.<br>
Enhancement of Sindbis virus self-replicating RNA vaccine potency by linkage of<br>
Mycobacterium tuberculosis heat shock protein 70 gene to an antigen. J. Immunol.<br>
166:6218-6226.<br>
WE CLAIM:<br>
1. An isolated M. tuberculosis peptide having an amino acid sequence selected<br>
from the group consisting of:<br>
(i) SEQ ID NOs: 1, 5, 7, 11, 15, 17, 21, 25, 27, 29, 31, 33, 35, 37, 41, 43, 45,<br>
or a variant thereof having at least 70% amino acid homology therewith, or a<br>
derivative thereof, or a fragment thereof having at least: 35 amino acid residues,<br>
wherein said variant, derivative or fragment has a common antigenic cross-reactivity<br>
to said isolated peptide; or<br>
(ii) SEQ ID NO: 3, or a variant thereof having at least 80% amino acid<br>
homology therewith, or a derivative thereof, or a fragment thereof having at least 35<br>
amino acid residues, wherein said variant, derivative or fragment has a common<br>
antigenic cross-reactivity to said isolated peptide; or<br>
(iii) SEQ ID NOs: 23 or 39, or a variant thereof having at least 90% amino acid<br>
homology therewith, or a derivative thereof, or a fragment thereof having at least 35<br>
amino acid residues, wherein said variant, derivative or fragment has a common<br>
antigenic cross-reactivity to said isolated peptide; or<br>
(iv) SEQ ID NO: 19, or a derivative thereof, wherein said derivative has a<br>
common antigenic cross-reactivity to said isolated peptide.<br>
wherein the peptide is encoded by an M. tuberculosis gene the expression of which<br>
is down-regulated during a stationary phase culture of M. tuberculosis under nutrient-<br>
starving culture conditions when compared with an exponential phase culture of M.<br>
tuberculosis under culture conditions that are not nutrient-starving and that support<br>
exponential growth of said M. tuberculosis.<br>
2. An isolated M. tuberculosis peptide as claimed in Claim 1, wherein said M.<br>
tuberculosis gene is down-regulated during a stationary phase culture of M.<br>
tuberculosis under nutrient-starving culture conditions and a dissolved oxygen<br>
tension of less than 10% when measured at 37°C.<br>
3. An isolated M. tuberculosis peptide as claimed in Claim 1 or Claim 2, wherein<br>
said gene is down-regulated at a time point at least 25, preferably at least 30, more<br>
preferably at least 40 days post-inoculation.<br>
4. An isolated M. tuberculosis peptide as claimed in any of Claims 1-3, in the<br>
manufacture of a medicament for treating or preventing a mycobacterial infection.<br>
5. An isolated M. tuberculosis peptide as claimed in any of Claims 1-3, in the<br>
manufacture of a diagnostic reagent for identifying an M. tuberculosis infection.<br><br>
A method is provided for identifying mycobacterial genes the expression of which is down-regulated during a stationary<br>
phase culture of mycobacteria under nutrient-starving conditions when compared with an exponential phase culture of mycobacteria<br>
under culture conditions that are not nutrient-starving and that support exponential growth of said mycobacteria. The described<br>
method optionally provides for identifying mycobacterial genes that are simultaneously down-regulated under low DOT conditions.<br>
The down-regulated genes of the present invention form the basis of nucleic acid vaccines, or provide targets to allow preparation of<br>
attenuated mycobacteria for vaccines against mycobacterial infections. Similarly, peptides encoded by said down-regulated genes<br>
are employed in vaccines. In a further embodiment, the identified genes/peptides provide the means for identifying the presence of<br>
a mycobacterial infection in a clinical sample by nucleic acid probe or antibody detection.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LUtPTE5QLTIwMDQtKDAzLTAxLTIwMTIpLUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">518-KOLNP-2004-(03-01-2012)-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZm9ybSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1jb3JyZXNwb25kZW5jZS5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1leGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-form 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1ncGEucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1yZXBseSB0byBleGFtaW5hdGlvbiByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtZ3JhbnRlZC1zcGVjaWZpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-granted-specification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtcmVwbHkgdG8gZXhhbWluYXRpb24gcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-reply to examination report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTE4LWtvbG5wLTIwMDQtc3BlY2lmaWNhdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">518-kolnp-2004-specification.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="235080-mixer-and-method-of-mixing-gypsum-slurry-therewith.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="235082-a-pharmaceutical-composition-of-2-5-mg-of-the-pentasaccharide-methyl-o-2-3-a-tri-o-methyl-6-o-sulfo-d-glucopyranosyl-1-4-o-2-3-di-o-methyl-d-glucopyranosyl-uronic-acid-1-4-o-2-3-6-tri-o-sulfo-d-gl">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>235081</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>518/KOLNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>26/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>26-Jun-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Apr-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>HEALTH PROTECTION AGENCY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>PORTON DOWN, SALISBURY, WILTSHIRE SP4 0JG</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JAMES BRIAN WILLIAM</td>
											<td>HEALTH PROTECTION AGENCY, PORTON DOWN, SALISBURY SP4 0JG</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MARSH PHILIP</td>
											<td>HEALTH PROTECTION AGENCY, PORTON DOWN, SALISBURY SP4 0JG</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HAMPSHIRE TOBIAS</td>
											<td>HEALTH PROTECTION AGENCY, PORTON DOWN, SALISBURY SP4 0JG</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/435</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2002/04718</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-10-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0125535.5</td>
									<td>2001-10-24</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/235081-a-composition-comprising-an-isolated-m-tuberculosis-peptide by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:36:00 GMT -->
</html>
