<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/235121-an-isolated-polypeptide-of-chlamydia-and-its-gene-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:37:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 235121:AN ISOLATED POLYPEPTIDE OF CHLAMYDIA AND ITS GENE SEQUENCE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED POLYPEPTIDE OF CHLAMYDIA AND ITS GENE SEQUENCE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>2199/MAS/98 &quot;AN ISOLATED POLYPEPTIDE OF CHLAMYDIA AND ITS GENE SEQUENCE&quot; The present invention relates to an isolated polypeptide comprising an amino acid sequence at least 95% identical to amino acids 29-1012 of SEQ ID NO: 2, wherein said polypeptide is recognizable by an antibody preparation that specifically binds to a polypeptide consisting of the amino acid sequence of SEQ ID NO: 2.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
The present invention generally relates to a high molecular weight ("HMW") protein of Chlamydia, the amino acid sequence thereof, and antibodies, including cytotoxic antibodies, that specifically bind the HMW protein. The invention further encompasses prophylactic and therapeutic compositions comprising the HMW protein, a fragment thereof, or an antibody that specifically binds the HMW protein or a portion thereof or the nucleotide sequence encoding the HMW protein or a fragment thereof, including vaccines. The invention additionally provides methods of preventing, treating or ameliorating disorders in mammals and birds related to Chlamydia infections and for inducing immune responses to Chlamydia. The invention further provides isolated nucleotide sequences and degenerate sequences encoding the HMW protein, vectors having said sequences, and host cells containing said vectors. Diagnostic methods and kits are also included.<br>
2.        BACKGROUND OF THE INVENTION<br>
Chlamydia are prevalent human pathogens causing disorders such as sexually transmitted diseases, respiratory diseases including pneumonia, neonatal conjunctivitis, and blindness. Chlamydia are obligate intercellular bacteria that infect the epithelial lining of the lung, conjunctivae or genital tract. The most common species of Chlamydia include Chlamydia trachomatis. Chlamydia psittaci, Chlamydia pecorum and Chlamydia pneumoniae. Recently, the newly designated species of Chlamydia, C. pneumoniae (formerly C. trachomatis TWAR), has been implicated as a major cause of epidemic human pneumonitis and perhaps may play a role in atherosclerosis.<br>
There are currently 18 recognized C trachomatis serovars, causing trachoma and a broad spectrum of sexually transmitted diseases: with the A, B and C serovars being most frequently associated with trachoma, while the D-K serovars are the most common cause of genital infections.<br>
C trachomatis is the major cause of sexually transmitted disease in many industrialized countries, including the United States. While the exact incidence of C. trachomatis infection in the U.S. is not known, current epidemiological studies indicate that more than 4 million chlamydial infections occur each year, compared to an estimated 2 million gonococcal infections. While all racial, ethnic and socioeconomic groups are affected, the greatest prevalence of<br><br>
chlamydial infections occur among young, 12 to 20 year-old, sexually active individuals. Most<br>
genitourinary chlamydial infections are clinically asymptomatic. Prolonged carriage in both men<br>
and women is common. As many as 25% of men and 75% of women diagnosed as having<br>
chlamydial infections have no overt signs of infection. As a consequence, these asymptomatic<br>
individuals constitute a large reservoir that can sustain transmission of the agent within the<br>
community.<br>
Far from being benign, serious disease can develop from these infections including: urethritis,<br>
lymphogranuloma venereum (LGV), cervicitis, and epididymitis in males. Ascending infections<br>
from the endocervix commonly gives rise to endometritis, pelvic inflammatory disease (PID) and<br>
salpingitis which can cause tubal occlusion and lead ultimately to infertility.<br>
C. trachomatis infection of neonates results from perinatal exposure to the mother's infected<br>
cervix. Nearly 70% of neonates bom vaginally to mothers with chlamydial cervicitis become<br>
infected during delivery. The mucus membranes of the eye, oropharynx, urogenital tract and<br>
rectum are the primary sites of infection. Chlamydia conjunctivitis has become the most common<br>
form of ophthalmia neonatorum. Approximately 20-30% of exposed infants develop inclusion<br>
conjunctivitis within 14 days of delivery even after receiving prophylaxis with either silver<br>
nitrate or antibiotic ointment. C. trachomatis is also the leading cause of infant pneumonia in the<br>
United States. Nearly 10-20% of neonates delivered through an infected cervix will develop<br>
chlamydial pneumonia and require some type of medical intervention.<br>
In developing countries, ocular infections of C. trachomatis cause trachoma, a chronic follicular<br>
conjunctivitis where repeated scar formation leads to distortion of the eyelids and eventual loss of<br>
sight. Trachoma is the world's leading cause of preventable blindness. The World Health<br>
Organization estimates that over 500 million people worldwide, including about 150 million<br>
children, currently suffer from active trachoma and over 6 million people have been blinded by<br>
this disease.<br>
In industrialized countries, the costs associated with treating chlamydial infections are enormous.<br>
In the U.S., the annual cost of treating these diseases was estimated at $2.5-3 billion in 1992 and<br>
has been projected to exceed $8 billion by the year 2000.<br>
One potential solution to this health crisis would be an effective chlamydial vaccine. Several<br>
lines of evidence suggest that developing an effective vaccine is feasible.<br>
Studies in both humans and primates have shown that short-term protective immunity to C.<br>
trachomatis can be produced by vaccinating with whole Chlamydia. However, protection was<br>
characterized as short lived, serovar specific, and due to mucosal antibody. Additionally, in some<br><br>
vaccinees disease was exacerbated when these individuals became naturally infected with a serovar different from that used for immunization. This adverse reaction was ultimately demonstrated to be due to a delayed-type hypersensitivity response. Thus, the need exists to develop a subunit-based chlamydial vaccine capable of producing an efficacious but nonsensitizing immune response. Such a subunit vaccine may need to elicit both mucosal neutralizing secretory IgA antibody and/or cellular immune response to be efficacious. Subunit vaccine development efforts to date have focused almost exclusively on the major outer membrane protein (MOMP). MOMP is an integral membrane protein of approximately 40 kDa in size and comprises up to about 60% of the infectious elementary body (EB) membrane protein (Caldwell, H. D., J. Kromhout, and L. Schachter. 1981. Infect. Immun.. 31:1161-1176). MOMP imparts structural integrity to the extracellular EB and is thought to function as a porin-like molecule when the organism is growing intracellularly and is metabolically active. With the exception of four surface exposed variable domains (VDI-VDIV), MOMP is highly conserved among all 18 serovars. MOMP is highly immunogenic and can elicit a local neutralizing anti-Chlamydia antibody. However, problems exists with this approach.<br>
To date, most MOMP-specific neutralizing epitopes that have been mapped are located within the VD regions and thus give rise only to serovar-specific antibody. Attempts to combine serovar-specific epitopes in various vaccine vectors (e.g. poliovirus) to generate broadly cross-reactive neutralizing antibodies have been only marginally successful (Murdin, A. D., H. Su, D. S. Manning, M. H. Klein, M. J. Pamell, and H. D. Caldwell. 1993. Infect. Immun.. 61:4406-4414; Murdin, A. D., H. Su, M. H. Klein, and H. D. Caldwell. 1995. Infect. Immun.. 63:1116-1121).<br>
Two other major outer membrane proteins in C. trachomatis, the 60 kDa and 12 kDa cysteine-rich proteins, as well as the surface-exposed lipopolysaccharide, are highly immunogenic but, unlike MOMP, have not been shown to induce a neutralizing antibody (Cerrone et al., 1991, Infect. Immun.. 59:79-90). Therefore, there remains a need for a novel subunit-based chlamydial vaccine.<br>
3.        BRIEF SUMMARY OF THE INVENTION<br>
An object of the present invention is to provide an isolated and substantially purified high molecular weight protein of a Chlamydia sp. ("HMW protein"), wherein the HMW protein has an apparent molecular weight of about 105-115 kDa, as determined by SDS-PAGE, or a fragment or analogue thereof Preferably the HMW protein has substantially the amino acid sequence of any<br><br>
of SEQ ID Nos.: 2, 15 and 16. Preferred fragments of the HMW protein include SEQ ID Nos: 3, 17, and 25-37. As used herein, "substantially the sequence" is intended to mean that the sequence is at least 80%, more preferably at least 90% and most preferably at least 95% identical to the referenced sequence. Preferably, the HMW protein is an outer membrane protein. More preferably, the outer membrane HMW protein is surface localized. Preferably, the HMW protein has a heparin binding domain. Preferably, the HMW Protein has a porin-like domain. It is intended that all species of Chlamydia are included in this invention, however preferred species include Chlamydia trachomatis. Chlamydia psittaci, Chlamydia percorum and Chlamydia pneumoniae. The substantially purified HMW protein is at least 70 wt % pure, preferably at least about 90 wt % pure, and may be in the form of an aqueous solution thereof Also included in this invention are recombinant forms of the HMW protein, wherein in transformed E. coli cells, the expressed recombinant form of the HMW protein has an apparent molecular weight of about 105-115 kDa, as determined by SDS-PAGE, or a fragment or analogue thereof. The term HMW-derived polypeptide is intended to include fragments of the HMW protein; variants of wild-type HMW protein or fragment thereof, containing one or more amino acid deletions, insertions or substitutions; and chimeric proteins comprising a heterologous polypeptide fiised to the C-terminal or N-terminal or internal segment of a whole or a portion of the HMW protein.<br>
As used herein and in the claims, the term "HMW protein" refers to a native purified or recombinant purified high molecular weight protein of a species of Chlamydia wherein the apparent molecular weight (as determined by SDS-PAGE) is about 105-115 kDa. As used herein and in the claims, the term "rHMW protein" refers to recombinant HMW protein. Another object of the present invention is to provide an isolated substantially pure nucleic acid molecule encoding a HMW protein or a fragment or an analogue thereof Preferred is the nucleic acid sequence wherein the encoded HMW protein comprises the amino acid sequence of any of SEQ ID Nos.: 2, 15 and 16, or a fragment thereof, particularly SEQ ID Nos.: 3, 17, 25-37. Also included is an isolated nucleic acid molecule comprising a DNA sequence of any of SEQ ID Nos.: 1, 23-24 or a complementary sequence thereof; a fragment of the HMW DNA sequence having the nucleic acid sequence of any of SEQ ID Nos.: 4-14, 18-22 or the complimentary sequence thereto; and a nucleic acid sequence which hybridizes under stringent conditions to any one of the sequences described above. The nucleic acid that hybridizes under stringent condition preferably has a sequence identity of about 70% with any of the sequences identified above, more preferably about 90%.<br><br>
The production and use of derivatives and analogues of the HMW protein are within the scope of the present invention. In a specific embodiment, the derivative or analogue is functionally active, i.e., capable of exhibiting one or more functional activities associated with a full-length, wild-type HMW protein. As one example, such derivatives or analogues which have the desired immunogenicity or antigenicity can be used, for example, in immunoassays, for immunization, etc. A specific embodiment relates to a HMW fragment that can be bound by an anti-HMW antibody. Derivatives or analogues of HMW can be tested for the desired activity by procedures known in the art.<br>
hi particular, HMW derivatives can be made by altering HMW sequences by substitutions, additions or deletions that provide for functionally equivalent molecules. Due to the degeneracy of nucleotide coding sequences, other DNA sequences which encode substantially the same amino acid sequence as a HMW gene may be used in the practice of the present invention. These include but are not limited to nucleotide sequences comprising all or portions of genes which are altered by the substitution of different codons that encode a functionally equivalent amino acid residue within the sequence, thus producing a silent change. Likewise, the HMW derivatives of the invention include, but are not limited to, those containing, as a primary amino acid sequence, all or part of the amino acid sequence of a HMW protein including altered sequences in which functionally equivalent amino acid residues are substituted for residues within the sequence resuhing in a silent change. For example, one or more amino acid residues within the sequence can be substituted by another amino acid of a similar polarity which acts as a functional equivalent, resulting in a silent alteration. Substitutes for an amino acid within the sequence may be selected from other members of the class to which the amino acid belongs. For example, the nonpolar (hydrophobic) amino acids include alanine, leucine, isoleucine, valine, proline, phenylalanine, tryptophan and methionine. The polar neutral amino acids include glycine, serine, threonine, cysteine, tyrosine, asparagine, and glutamine. The positively charged (basic) amino acids include arginine, lysine and histidine. The negatively charged (acidic) amino acids include aspartic acid and glutamic acid.<br>
hi a specific embodiment of the invention, proteins consisting of or comprising a fragment of a HMW protein consisting of at least 6 (continuous) amino acids of the HMW protein is provided, hi other embodiments, the fragment consists of at least 7 to 50 amino acids of the HMW protein, hi specific embodiments, such fragments are not larger than 35, 100 or 200 amino acids. Derivatives or analogues of HMW include but are not limited to those molecules comprising regions that are substantially homologous to HMW or fi-agments thereof (e.g., in various<br><br>
embodiments, at least 60% or 70% or 80% or 90% or 95% identity over an amino acid sequence of identical size or when compared to an aligned sequence in which the alignment is done by a computer homology program known in the art) or whose encoding nucleic acid is capable of hybridizing to a coding HMW sequence, unde stringent, moderately stringent, or nonstringent conditions.<br>
By way of example and not limitation, useful computer homology programs include the following: Basic Local Alignment Search Tool (BLAST) (www.ncbi.nlm.nih.gov) (Altschul et al, 1990, J. of Molec. Biol., 215:403-410, "The BLAST Algorithm; Altschul et al, 1997, Nuc. Acids Res. 25:3389-3402) a heuristic search algorithm tailored to searching for sequence similarity which ascribes significance using the statistical methods of Karlin and Altschul 1990, Proc. Nafl Acad. Sci. USA, 87:2264-68; 1993 Proc. Nafl Acad. Sci. USA 90:5873-77. Five specific BLAST programs perform the following tasks:<br>
(1)	The BLASTP program compares an amino acid query sequence against a protein sequence database.<br>
(2)	The BLASTN program compares a nucleotide query sequence against a nucleotide sequence database.<br>
(3)	The BLASTX program compares the six-frame conceptual translation products of a nucleotide query sequence (both strands) against a protein sequence database.<br>
(4)	The TBLASTN program compares a protein query sequence against a nucleotide sequence database translated in all six reading frames (both strands).<br>
(5)	The TBLASTX program compares the six-frame translations of a nucleotide query sequence against the six-frame translations of a nucleotide sequence database.<br>
Smith-Waterman (database: European Bioinformatics Institute wwwz.ebi.ac.uk^ic_sw/) (Smith-Waterman, 1981, J. of Molec. Biol., 147:195-197) is a mathematically rigorous algorithm for sequence alignments.<br>
FASTA (see Pearson et al, 1988, Proc. Nat'l Acad. Sci. USA, 85:2444-2448) is a heuristic approximation to the Smith-Waterman algorithm. For a general discussion of the procedure and benefits of the BLAST, Smith-Waterman and FASTA algorithms see Nicholas et al, 1998, "A Tutorial on Searching Sequence Databases and Sequence Scoring Methods" (www.psc.edu) and references cited therein.<br>
The HMW derivatives and analogues of the invention can be produced by various methods known in the art. The manipulations which result in their production can occur at the gene or protein level. For example, the cloned HMW gene sequence can be modified by any of numerous<br><br>
strategies known in the art (Sambrook et al, 1989, Molecular Cloning, A Laboratory Manual, 2d Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York). The sequence can be cleaved at appropriate sites with restriction endonuclease(s), followed by further enzymatic modification if desired, isolated, and ligated in vitro. In the production of the gene encoding a derivative or analogue of HMW, care should be taken to ensure that the modified gene remains within the same translational reading frame as HMW, uninterrupted by translational stop signals, in the gene region where the desired HMW activity is encoded.<br>
Additionally, the HMW-encoding nucleic acid sequence can be mutated in vitro or in vivo, to create and/or destroy translation, initiation, and/or termination sequences, or to create variations in coding regions and/or form new restriction endonuclease sites or destroy preexisting ones, to facilitate further in vitro modification. Any technique for mutagenesis known in the art can be used, including but not limited to, chemical mutagenesis, in vitro site-directed mutagenesis (Hutchinson, C, et al, 1978, J. Biol. Chem 253:6551), use of TAB® linkers (Pharmacia), etc. Manipulations of the HMW sequence may also be made at the protein level. Included within the scope of the invention are HMW protein fragments or other derivatives or analogues which are differentially modified during or after translation, e.g., by glycosylation, lipidation, acetylation, phosphorylation, amidation, derivatization by known protecting^locking groups, proteolytic cleavage, linkage to an antibody molecule or other cellular ligand, etc. Any of numerous chemical modifications may be carried out by known techniques, including but not limited to specific chemical cleavage by cyanogen bromide, trypsin, chymotrypsin, papain, V8 protease, NaBH24 ; acetylation, formylation, oxidation, reduction; metabolic synthesis in the presence of tunicamycin; etc.<br>
In addition, analogues and derivatives of HMW can be chemically synthesized. For example, a peptide corresponding to a portion of a HMW protein which comprises the desired domain, or which mediates the desired activity in vitro, can be synthesized by use of a peptide synthesizer. Furthermore, if desired, nonclassical amino acids or chemical amino acid analogues can be introduced as a substitution or addition into the HMW sequence. Non-classical amino acids include but are not limited to the D-isomers of the common amino acids, a-amino isobutyric acid, 4-aminobutyric acid, Abu, 2-amino butyric acid, y-Abu, e-Ahx, 6-amino hexanoic acid, Aib, 2-amino isobutyric acid, 3-amino propionic acid, ornithine, norleucine, norvaline, hydroxyproline, sarcosine, citrulline, cysteic acid, t-butylglycine, t-butylalanine, phenylglycine, cyclohexylalanine, p-alanine, fluoro-amino acids, designer amino acids such as P-methyl amino<br><br>
acids, Ca-methyl amino acids, Na-methyl amino acids, and amino acid analogues in general.<br>
Furthermore, the amino acid can be D (dextrorotary) or L (levorotary).<br>
Another object of the invention is to provide a recombinant expression vector adapted for<br>
transformation of a host or for delivery of a HMW protein to a host comprising the nucleic acid<br>
molecule of SEQ ID No.: 1, 23 or 24 or any fragment thereof Preferably, the recombinant<br>
expression vector is adapted for transformation of a host and comprises an expression means<br>
operatively coupled to the nucleic acid molecule for expression by the host of said HMW protein<br>
or the fragment or analogue thereof More preferred is the expression vector wherein the<br>
expression means includes a nucleic acid portion encoding a leader sequence for secretion from<br>
the host or an affinity domain coupled to either the N- or C-terminus of the protein or the<br>
fragment or analogue thereof<br>
A further aspect of the invention includes a transformed host cell containing an expression vector<br>
described above and the recombinant HMW protein or fragment or analogue thereof producible<br>
by the transformed host cell.<br>
Still a fiirther aspect of the invention is directed to a HMW protein recognizable by an antibody<br>
preparation that specifically binds to a peptide having the amino acid sequence of SEQ ID No. 2,<br>
15-16 or a fragment or conservatively substituted analogue thereof<br>
Antigenic and/or immunogenic compositions are another aspect of the invention wherein the<br>
compositions comprise at least one component selected from the following group:<br>
(a)	a HMW protein, wherein the molecular weight is about 105-115 kDa, as determined by SDS-PAGE, or a fragment or analogue thereof;<br>
(b)	an isolated nucleic acid molecule encoding a HMW protein, or a fragment or analogue thereof;<br>
(c)	an isolated nucleic acid molecule having the sequence of SEQ ID Nos. 1,22, 23 or 24, the complimentary sequence thereto or a nucleic acid sequence which hybridizes under stringent conditions thereto or fragment thereof;<br>
(d)	an isolated recombinant HMW protein, or fragment or analogue thereof, producible in a transformed host comprising an expression vector comprising a nucleic acid molecule as defined in (b) or (c) and expression means operatively coupled to the nucleic acid molecule for expression by the host of said HMW protein or the fragment or analogue thereof;<br>
(e)	a recombinant vector comprising a nucleic acid encoding a HMW protein or fragment or analogue thereof;<br><br>
(f) a transformed cell comprising the vector of (e) and optionally an adjuvant, and a<br>
pharmaceutically acceptable carrier or diluent therefor, said composition producing an<br>
immune response when administered to a host.<br>
Preferred adjuvants include cholera holotoxin or subunits, E. coli heat labile holotoxin, subunits<br>
and mutant forms thereof, alum, QS21, and MPL. Particularly, preferred are alum, LTR192G,<br>
mLTandQS21.<br>
Also included are methods for producing an immune response in a mammal or a bird comprising administering to said mammal, an effective amount of the antigenic or the immunogenic composition described above.<br>
Another aspect of the invention is directed to antisera raised against the antigenic or immunogenic composition of the invention, and antibodies present in the antisera that specifically bind a HMW protein or a fragment or analogue thereof Preferably the antibodies bind a HMW protein having the amino acid sequence of SEQ ID Nos.: 2, 15-16 or fragment or a conservatively substituted analogue thereof Also included are monoclonal antibodies that specifically bind a HMW protein or a fragment or analogue thereof<br>
A further aspect of the invention includes pharmaceutical and vaccine compositions comprising an effective amount of at least one component selected fi*om the follov^'ing group:<br>
(a)	a HMW protein, wherein the isolated protein molecular weight is about 105-115 kDa, as determined by SDS-PAGE, or a fi-agment or analogue thereof;<br>
(b)	an isolated nucleic acid molecule encoding a HMW protein, or a fi"agment or analogue thereof;<br>
(c)	an isolated nucleic acid molecule having the sequence of SEQ ID Nos.: 1, 22,23 or 24 the complimentary sequence thereto or a nucleic acid sequence which hybridizes under stringent conditions thereto or a fragment thereof;<br>
(d)	an isolated recombinant HMW protein, or fragment or analogue thereof producible in a transformed host comprising an expression vector comprising a nucleic acid molecule as defined in (b) or (c) and expression means operatively coupled to the nucleic acid molecule for expression by the host of said HMW protein of a Chlamydia species or the fragment or analogue thereof;<br>
(e)	a recombinant vector, comprising a nucleic acid encoding a HMW protein or fragment or analogue thereof;<br><br>
(f) a transformed cell comprising the vector of (e), (g) antibodies that specifically bind the component of (a), (b), (c), (d) or (e), and a pharmaceutically acceptable carrier or diluent therefor. Preferred are vaccine compositions which are effective at the mucosal level. The invention also includes a diagnostic reagent which may include any one or more of the above mentioned aspects, such as the native HMW protein, the recombinant HMW protein, the nucleic acid molecule, the immunogenic composition, the antigenic composition, the antisera, the antibodies, the vector comprising the nucleic acid, and the transformed cell comprising the vector.<br>
Methods and diagnostic kits for detecting Chlamydia or anti-Chlamydia antibodies in a test sample are also included, wherein the methods comprise the steps of: (a) contacting said sample with an antigenic composition comprising Chlamydia HMW protein or a fragment or analogue thereof or immunogenic composition or antibodies thereto to form Chlamydia antigen: anti-Chlamydia antibody immunocomplexes, and further,<br>
(b) detecting the presence of or measuring the amount of said immunocomplexes formed during step (a) as an indication of the presence of said Chlamydia or anti-Chlamydia antibodies in the test sample. The diagnostic kits for detecting Chlamydia or antibodies thereto comprise antibodies, or an antigenic or immunogenic composition comprising Chlamydia HMW protein or a fragment or analogue thereof, a container means for contacting said antibodies or composition with a test sample suspected of having anti-Chlamydia antibodies or Chlamydia and reagent means for detecting or measuring Chlamydia antigen: anti-Chlamydia antibody immunocomplexes formed between said antigenic or immunogenic composition or said antibodies and said test sample. A further aspect of the present invention provides methods for determining the presence of nucleic acids encoding a HMW protein or a fragment or analogue thereof in a test sample, comprising the steps of:<br>
(a)	contacting the test sample with the nucleic acid molecule provided herein to produce duplexes comprising the nucleic acid molecule and any said nucleic acid molecule encoding the HMW protein in the test sample and specifically hybridizable therewith; and<br>
(b)	determining the production of duplexes.<br>
The present invention also provides a diagnostic kit and reagents therefor, for determining the presence of nucleic acid encoding a HMW protein or fragment or analogue thereof in a sample, comprising:<br>
(a) the nucleic acid molecule as provided herein;<br><br>
(b) means for contacting the nucleic acid with the test sample to produce duplexes<br>
comprising the nucleic acid molecule and any said nucleic acid molecule encoding the<br>
HMW protein in the test sample and specifically hybridizable therewith; and c) means for<br>
determining the production of duplexes.<br>
Also included in this invention are methods of preventing, treating or ameliorating disorders<br>
related to Chlamydia in an animal including mammals and birds in need of such treatment<br>
comprising administering and effective amount of the pharmaceutical or vaccine composition of<br>
the   invention.   Preferred   disorders   include   a   Chlamydia   bacterial   infection,   trachoma,<br>
conjunctivitis,  urethritis,   lymphogranuloma  venereum  (LGV),  cervicitis,  epididymitis,  or<br>
endometritis, pelvic inflammatory disease (PID), salpingitis, tubal occlusion, infertility, cervical<br>
cancer, and artherosclerosis. Preferred vaccine or pharmaceutical compositions include those<br>
formulated for in vivo administration to a host to confer protection against disease or treatment<br>
therefor caused by a species of Chlamydia. Also preferred are compositions formulated as a<br>
microparticle, capsule, liposome preparation or emulsion.<br>
4.        ABBREVIATIONS<br>
anti-HMW	=	HMW polypeptide antibody or antiserum<br>
ATCC	=	American Type Culture Collection<br>
immuno-reactive	~	capable of provoking a cellular or humoral immune response<br>
kDa	=	kilodaltons<br>
OG	—	n-octyl P-D-glucopyranoside or octyl glucoside<br>
OMP	=	outer membrane protein<br>
OMPs	=	outer membrane proteins<br>
PBS	=	phosphate buffered saline<br>
PAGE	=	polyacrylamide gel electrophoresis<br>
polypeptide		a peptide of any length, preferably one having ten or more amino acid residues<br>
SDS	=	sodium dodecylsulfate<br>
SDS-PAGE		sodium dodecylsulfate polyacrylamide gel electrophoresis<br>
Nucleotide or nucleic acid sequence	defined herein are represented by one-letter symbols for the<br>
bases as follows:		<br>
A (adenine)		<br>
C (cytosine)		<br>
G (guanine)		<br>
T (thymine)		<br><br>
U (uracil) M (A or C) R(AorG) W (A or T/U) S (C or G)<br>
Y	(C or T/U) K (G or T/U)<br>
Y	(A or C or G; not T/U) H (A or C or T/U; not G) D (A or G or T/U; not C) B (C or G or T/U; not A)<br>
N (A or C or G or T/U) or (unknown) Peptide and polypeptide sequences defined herein are represented by one-letter symbols for amino acid residues as follows:<br>
A (alanine)<br>
R (arginine)<br>
N (asparagine)<br>
D (aspartic acid)<br>
C (cysteine)<br>
Q (glutamine)<br>
E (glutamic acid)<br>
G (glycine)<br>
H (histidine)<br>
I (isoleucine)<br>
L (leucine)<br>
K (lysine)<br>
M (methionine)<br>
F (phenylalanine)<br>
P (proline)<br>
S (serine)<br>
T (threonine)<br>
W (tryptophan)<br>
Y	(tyrosine)<br><br>
V (valine)<br>
X (unknown) The present invention may be more fully understood by reference to the following detailed description of the invention, non-limiting examples of specific embodiments of the invention and the appended figures.<br>
5.        BRIEF DESCRIPTION OF THE DRAWINGS/FIGURES<br>
Figure 1: Western blot analysis of C. trachomatis L2 elementary bodies (EBs). Gradient purified EBs were solubilized in standard Laemmli SDS-PAGE sample buffer containing 2-mercaptoethanol, boiled for ~3 minutes and loaded onto a 4-12% Tris-glycine gradient gel containing SDS and electrophoresed at lOOV. Immediately following electrophoresis, proteins were electroblotted onto PVDF membranes at 4°C. for -2.5 hours at ~50V. The blocked membrane was probed using a 1/5,000 dilution of anti-rHMWP' antibody (K196) for 1.5 hours at room temperature. Following washing, the membrane was treated with a 1/5,000 dilution of a goat anti-rabbit IgG antibody conjugated to HRP for 1 hour at room temperature. The blot was developed using a standard TMB substrate system. Three immunoreactive bands detected in EBs and RBs. Dot indicates HMW Protein of about 105-115 kDa.<br>
Figure 2. Consensus Nucleic Acid Sequence encoding the open reading frame of the HMW protein from C. trachomatis LGV L2.<br>
Figure 3. Deduced Amino Acid Sequence of the HMW protein from the PCR open reading frame from C. trachomatis LGV L2.<br>
Figure 4. SDS-PAGE of partially purified recombinant HMW protein from C. trachomatis LGVL2 expressed in E. coli. Counterstained and prestained SDS-PAGE standards were used as molecular weight markers. The positions of the molecular weight markers in the gel are noted on the left and right side of the figure by lines to the molecular weights (kDa) of some of the markers. See Text Example 10 for details. Lane A: Mark 12 Wide Range Molecular Weight Markers (Novex); myosin, 200 Kdal; B-galactosidase, 116.3 Kdal; phosphorylase B, 97.4 Kdal; bovine serum albumin, 66.3 Kdal. Lane B: C. trachomatis L2 recombinant HMWP. Lane C: SeeBlue Prestained Molecular Weight markers (Novex); myosin, 250 Kdal; bovine serum albumin, 98 Kdal; glutamic dehydrogenase, 64 Kdal.<br>
Figure 5. Map of plasmids pAH306, pAH310, pAH312, pAH316 and the PCR open reading frame.<br><br>
Figure 6. Predicted amino acid sequences, of HMW protein for C trachomatis L2, B, and F. The C. trachomatis L2 sequence is given in the top line and begins with the first residue of the mature protein, E. Potential eucaryotic N-glycosylation sequences are underlined. A hydrophobic helical region flanked by prolinerich segments and of suitable length to span the lipid bilayer is underlined and enclosed in brackets. Amino acid differences identified in the B and F serovars are designated below the L2 HMWP protein sequence.<br>
Figure 7. Indirect florescence antibody staining of C. trachomatis Nil (serovar F) inclusion bodies using anti-rHMWP' antibody. Panel A: Post-immunization sera from rabbit K196. Chlamydia inclusion bodies are stained yellow. Panel B: Pre-immunization sera fi-om rabbit K196.<br>
6.        DETAILED DESCRIPTION OF THE INVENTION<br>
The term "antigens" and its related term "antigenic" as herein and in the claims refers to a<br>
substance that binds specifically to an antibody or T-cell receptor. Preferably said antigens are<br>
immunogenic.<br>
The term "immunogenic" as used herein and in the claims refers to the ability to induce an<br>
immune response, e.g., an antibody and/or a cellular immune response in a an animal, preferably<br>
a mammal or a bird.<br>
The term "host" as used herein and in the claims refers to either in vivo in an animal or in vitro in<br>
mammalian cell cultures.<br>
An effective amount of the antigenic, immunogenic, pharmaceutical, including, but not limited to<br>
vaccine, composition of the invention should be administered, in which "effective amount" is<br>
defined as an amount that is sufficient to produce a desired prophylactic, therapeutic or<br>
ameliorative response in a subject, including but not limited to an immune response. The amount<br>
needed will vary depending upon the immunogenicity of the HMW protein, fragment, nucleic<br>
acid or derivative used, and the species and weight of the subject to be administered, but may be<br>
ascertained using standard techniques. The composition elicits an immune response in a subject<br>
which produces antibodies, including anti-HMW protein antibodies and antibodies that are<br>
opsonizing or bactericidal. In preferred, non-limiting embodiments of the invention, an effective<br>
amount of a composition of the invention produces an elevation of antibody titer to at least three<br>
times the antibody titer prior to administration. In a preferred, specific, non-limiting embodiment<br><br>
of the invention, approximately 0.01 to 2000 \ig and preferably 0.1 to 500 \ig are administered to a host. Preferred are compositions additionally comprising an adjuvant.<br>
Immunogenic, antigenic, pharmaceutical and vaccine compositions may be prepared as injectables, as liquid solutions or emulsions. The HMW protein may be mixed with one or more pharmaceutically acceptable excipient which is compatible with the HMW protein. Such excipients may include, water, saline, dextrose, glycerol, ethanol, and combinations thereof Immunogenic, antigenic, pharmaceutical and vaccine compositions may further contain one or more auxiliary substance, such as wetting or emulsifying agents, pH buffering agents, or adjuvants to enhance the effectiveness thereof. Immunogenic, antigenic, pharmaceutical and vaccine compositions may be administered parenterally, by injection, subcutaneously or intramuscularly.<br>
Alternatively, the immunogenic, antigenic, pharmaceutical and vaccine compositions formed according to the present invention, may be formulated and delivered in a manner to evoke an immune response at mucosal surfaces. Thus, the immunogenic, antigenic, pharmaceutical and vaccine compositions may be administered to mucosal surfaces by, for example, the nasal, oral (intragastric), ocular, branchiolar, intravaginal or intrarectal routes. Alternatively, other modes of administration including suppositories and oral formulations may be desirable. For suppositories, binders and carriers may include, for example, polyalkalene glycols or triglycerides. Oral formulations may include normally employed incipients such as, for example, pharmaceutical grades of saccharine, cellulose and magnesium carbonate. These compositions can take the form of solutions, suspensions, tablets, pills, capsules, sustained release formulations or powders and contain about 0.001 to 95% of the HMW protein. The immunogenic, antigenic, pharmaceutical and vaccine compositions are administered in a manner compatible with the dosage formulation, and in such amount as will be therapeutically effective, protective or immunogenic. Further, the immunogenic, antigenic, pharmaceutical and vaccine compositions may be used in combination with or conjugated to one or more targeting molecules for delivery to specific cells of the immune system, such as the mucosal surface. Some examples include but are not limited to vitamin B12, bacterial toxins or fragments thereof, monoclonal antibodies and other specific targeting lipids, proteins, nucleic acids or carbohydrates.<br>
The quantity to be administered depends on the subject to be treated, including, for example, the capacity of the individual's immune system to synthesize antibodies, and if needed, to produce a cell-mediated immune response. Precise amounts of active ingredient required to be administered depend on the judgment of the practitioner. However, suitable dosage ranges are readily<br><br>
determinable by one skilled in the art and may be of the order of 0.1 to 1000 micrograms of the HMW protein, fragment or analogue thereof. Suitable regimes for initial administration and booster doses are also variable, but may include an initial administration followed by subsequent administrations. The dose may also depend on the route(s) of administration and will vary according to the size of the host.<br>
The concentration of the HMW protein in an antigenic, immunogenic or pharmaceutical composition according to the invention is in general about 0.001 to 95%. A vaccine which contains antigenic material of only one pathogen is a monovalent vaccine. Vaccines which contain antigenic material of several pathogens are combined vaccines and also belong to the present invention. Such combined vaccines contain, for example, material for various pathogens or from various sfrains of the same pathogen, or from combinations of various pathogens. The antigenic, immunogenic or pharmaceutical preparations, including vaccines, may comprise as the immunostimulating material a nucleotide vector comprising at least a portion of the gene encoding the HMW protein, or the at least a portion of the gene may be used directly for immunization.<br>
To efficiently induce humoral immune responses (HIR) and cell-mediated immunity (CMI), immunogens are typically emulsified in adjuvants. Immunogenicity can be significantly improved if the immunogen is co-administered with an adjuvant. Adjuvants may act by retaining the immunogen locally near the site of administration to produce a depot effect facilitating a slow, sustained release of antigen to cells of the immune system. Adjuvants can also attract cells of the immune system to an immunogen depot and stimulate such cells to elicit immune responses.<br>
Many adjuvants are toxic, inducing granulomas, acute and chronic inflammations (Freund's complete adjuvant, FCA), cytolysis (saponins and Pluronic polymers) and pyrogenicity, arthritis and anterior uveitis (LPS and MDP). Although FCA is an excellent adjuvant and widely used in research, it is not licensed for use in human or veterinary vaccines because of its toxicity. Desirable characteristics of ideal adjuvants include:<br>
(1)	lack of toxicity;<br>
(2)	ability to stimulate a long-lasting immune response;<br>
(3)	simplicity of manufacture and stability in long-term storage;<br>
(4)	ability to elicit either CMI or HIR or both to antigens administered by various routes, if required;<br>
(5)	synergy with other adjuvants;<br><br>
(6)	capability of selectively interacting with populations of antigen presenting cells (APC);<br>
(7)	ability to specifically elicit appropriate THI or TH2 cell-specific immune responses; and<br>
(8)	ability to selectively increase appropriate antibody isotype levels (for example, IgA) against antigens.<br>
Immunostimulatory agents or adjuvants have been used for many years to improve the host immune responses to, for example, vaccines. Intrinsic adjuvants, such as lipopolysaccharides, normally are the components of the killed or attenuated bacteria used as vaccines. Extrinsic adjuvants are immunomodulators which are typically non-covalently linked to antigens and are formulated to enhance the host immune responses. Thus, adjuvants have been identified that enhance the immune response to antigens delivered parenterally. Aluminum hydroxide and aluminum phosphate (collectively commonly referred to as alum) are routinely used as adjuvants in human and veterinary vaccines. The efficacy of alum in increasing antibody responses to diphtheria and tetanus toxoids is well established and a HBsAg vaccine has been adjuvanted with alum.<br>
Other extrinsic adjuvants may include saponins complexed to membrane protein antigens (immune stimulating complexes), pluronic polymers with mineral oil, killed mycobacteria in mineral oil, Freund's complete adjuvant, bacterial products, such as muramyl dipeptide (MDP) and lipopolysaccharide (LPS), as well as lipid A, and liposomes.<br>
International Patent Application, PCT/US95/09005 incorporated herein by reference describes mutated forms of heat labile toxin of enterotoxigenic E. coli ("mLT"). U.S. Patent 5,057,540, incorporated herein by reference, describes the adjuvant, Qs21, an HPLC purified non-toxic fraction of a saponin from the bark of the South American tree Quiliaja saponaria molina 3D-MPL is described in great Britain Patent 2,220,211, and is incorporated herein by reference. U.S. Patent 4,855,283 granted to Lockhoff er al on August 8, 1989 which is incorporated herein by reference, teaches glycolipid analogues including N-glycosylamides, N-glycosylureas and N-glycosylcarbamates, each of which is substituted in the sugar residue by an amino acid, as immuno-modulators or adjuvants. Lockhoff reported that N-glycosphospholipids and glycoglycerolipids, are capable of eliciting strong immune responses in both herpes simplex virus vaccine and pseudorabies virus vaccine. Some glycolipids have been synthesized from long chain-alkylamines and fatty acids that are linked directly with the sugars through the anomeric carbon atom, to mimic the functions of the naturally occurring lipid residues.<br><br>
U.S. Patent 4,258,029 granted to Moloney, incorporated herein by reference thereto, teaches that octadecyl tyrosine hydrochloride (OTH) functioned as an adjuvant when complexed with tetanus toxoid and formalin inactivated type I, II and III poliomyelitis virus vaccine. Lipidation of synthetic peptides has also been used to increase their immunogenicity.<br>
Therefore, according to the invention, the immunogenic, antigenic, pharmaceutical, including vaccine, compositions comprising a HMW protein, or a fragment or derivative thereof or a HMW encoding nucleic acid or fragment thereof or vector expressing the same, may further comprise an adjuvant, such as, but not limited to alum, mLT, QS21 and all those listed above. Preferably, the adjuvant is selected from alum, LT, 3D-mPL, or Bacille Calmette-Guerine (BCG) and mutated or modified forms of the above, particularly mLT and LTR192G. The compositions of the present invention may also further comprise a suitable pharmaceutical carrier, including but not limited to saline, bicarbonate, dextrose or other aqueous solution. Other suitable pharmaceutical carriers are described in Remington's Pharmaceutical Sciences. Mack Publishing Company, a standard reference text in this field, which is incorporated herein by reference in its entirety.<br>
Immunogenic, antigenic and pharmaceutical, including vaccine, compositions may be administered in a suitable, nontoxic pharmaceutical carrier, may be comprised in microcapsules, and/or may be comprised in a sustained release implant.<br>
Immunogenic, antigenic and pharmaceutical, including vaccine, compositions may desirably be administered at several intervals in order to sustain antibody levels, and/or may be used in conjunction with other bacteriocidal or bacteriostatic methods.<br>
As used herein and in the claims, "antibodies" of the invention may be obtained by any conventional methods known to those skilled in the art, such as but not limited to the methods described in Antibodies A Laboratorv Manual (E. Harlow, D. Lane, Cold Spring Harbor Laboratory Press, 1989) which is incorporated herein by reference in its entirety. The term "antibodies" is intended to include all forms, such as but not limited to polyclonal, monoclonal, purified IgG, IgM, IgA and fragments thereof, including but not limited to fi-agments such as Fv, single chain Fv (scFv), F(ab')2, Fab fragments (Harlow and Leon, 1988, Antibody, Cold Spring Harbor); single chain antibodies (U.S. Patent 4,946,778) chimeric or humanized antibodies (Morrison et ai, 1984, Proc. Nat'l Acad. Sci. USA 81:6851); Neuberger et al., 1984, Nature 81:6851) and complementary determining regions (CDR), (see Verhoeyen and Windust, in Molecular Immunology 2ed., by B. D. Hames and D. M. Glover, IRL Press, Oxoford University Press, 1996, at pp. 283-325), etc.<br><br>
In general, an animal (a wide range of vertebrate species can be used, the most common being mice, rats, guinea pig, bovine, pig, hamsters, sheep, birds and rabbits) is immunized with the HMW protein or nucleic acid sequence or immunogenic fragment or derivative thereof of the present invention in the absence or presence of an adjuvant or any agent that enhances the immunogen's effectiveness and boosted at regular intervals. The animal serum is assayed for the presence of desired antibody by any convenient method. The serum or blood of said animal can be used as the source of polyclonal antibodies.<br>
For monoclonal antibodies, animals are treated as described above. When an acceptable antibody titre is detected, the animal is euthanized and the spleen is aseptically removed for fusion. The spleen cells are mixed with a specifically selected immortal myeloma cell line, and the mixture is then exposed to an agent, typically polyethylene glycol or the like, which promotes the fusion of cells. Under these circumstances fiision takes place in a random selection and a fused cell mixture together with unfused cells of each type is the resulting product. The myeloma cell lines that are used for fiision are specifically chosen such that, by the use of selection media, such as HAT: hypoxanthine, aminopterin, and thymidine, the only cells to persist in culture from the fiision mixture are those that are hybrids between cells derived from the immunized donor and the myeloma cells. After fusion, the cells are diluted and cultured in the selective media. The culture media is screened for the presence of antibody having desired specificity towards the chosen antigen. Those cultures containing the antibody of choice are cloned by limiting dilution until it can be adduced that the cell culture is single cell in origin.<br>
Antigens. Immunogens and Immunoassays<br>
The HMW protein or nucleic acid encoding same, and fragments thereof are useful as an antigen or immunogen for the generation of anti-HMW protein antibodies or as an antigen in immunoassays including enzyme-linked immunosorbent assays (ELISA), radioimmunoassays (RIA) and other non-enzyme linked antibody binding assays or procedures known in the art for the detection of anti-bacterial, anti-Chlamydia, and anti-HMW protein antibodies. In ELISA assays, the HMW protein is immobilized onto a selected surface, for example, a surface capable of binding proteins such as the wells of a polystyrene microtiter plate. After washing to remove incompletely absorbed HMW protein, a nonspecific protein solution that is known to be antigenically neutral with regard to the test sample may be bound to the selected surface. This allows for blocking of nonspecific absorption sites on the immobilizing surface and thus reduces the background caused by nonspecific bindings of antisera onto the surface.<br><br>
The immobilizing surface is then contacted with a sample, such as clinical or biological materials, to be tested in a manner conducive to immune complex (antigen/antibody) formation. This may include diluting the sample with diluents, such as solutions of bovine gamma globulin (BGG) and/or phosphate buffered saline (PBS)/Tween. The sample is then allowed to incubate for from 2 to 4 hours, at temperatures such as of the order of about 20*^0 37°C. Following incubation, the sample-contacted surface is washed to remove non-immunocomplexed material. The washing procedure may include washing with a solution, such as PBS/Tween or a borate buffer. Following formation of specific immunocomplexes between the test sample and the bound HMW protein, and subsequent washing, the occurrence, and even amount, of immunocomplex formation may be determined by subjecting the immunocomplex to a second antibody having specificity for the first antibody. If the test sample is of human origin, the second antibody is an antibody having specificity for human immunoglobulins and in general IgG. To provide detecting means, the second antibody may have an associated activity such as an enzymatic activity that will generate, for example, a color development upon incubating with an appropriate chromogenic substrate. Detection may then be achieved by detecting color generation. Quantification may then be achieved by measuring the degree of color generation using, for example, a visible spectrophotometer and comparing to an appropriate standard. Any other detecting means known to those skilled in the art are included.<br>
Another embodiment includes diagnostic kits comprising all of the essential reagents required to perform a desired immunoassay according to the present invention. The diagnostic kit may be presented in a commercially packaged form as a combination of one or more containers holding the necessary reagents. Such a kit may comprise HMW protein or nucleic acid encoding same or fragment thereof, a monoclonal or polyclonal antibody of the present invention in combination with several conventional kit components. Conventional kit components will be readily apparent to those skilled in the art and are disclosed in numerous publications, including Antibodies A Laboratory Manual (E. Harlow, D. Lane, Cold Spring Harbor Laboratory Press, 1989) which is incorporated herein by reference in its entirety. Conventional kit components may include such items as, for example, microtitre plates, buffers to maintain the pH of the assay mixture (such as, but not limited to Tris, HEPES, etc.), conjugated second antibodies, such as peroxidase conjugated anti-mouse IgG (or any anti-lgG to the animal from which the first antibody was derived) and the like, and other standard reagents.<br><br>
Nucleic Acids and Uses Thereof<br>
The nucleotide sequences of the present invention, including DNA and RNA and comprising a sequence encoding the HMW protein or a fragment or analogue thereof, may be synthesized using methods known in the art, such as using conventional chemical approaches or polymerase chain reaction (PRC) amplification using convenient pairs of oligonucleotide primers and ligase chain reaction using a battery of contiguous oligonucleotides. The sequences also allow for the identification and cloning of the HMW protein gene from any species of Chlamydia, for instance for screening chlamydial genomic libraries or expression libraries.<br>
The nucleotide sequences encoding the HMW protein of the present invention are useful for their ability to selectively form duplex molecules with complementary stretches of other protein genes. Depending on the application, a variety of hybridization conditions may be employed to achieve varying sequence identities. In specific aspects, nucleic acids are provided which comprise a sequence complementary to at least 10, 15, 25, 50, 100, 200 or 250 nucleotides of the HMW protein gene (Figure 2). In specific embodiments, nucleic acids which hybridize to an HMW protein nucleic acid (e.g. having sequence SEQ ID NO: 1, 23 or 24) under annealing conditions of low, moderate or high stringency conditions.<br>
For a high degree of selectivity, relatively stringent conditions are used to form the duplexes, such as, by way of example and not limitation, low salt and/or high temperature conditions, such as provided by 0.02 M to 0.15 M NaCl at temperatures of between about 50°C to 70°C. For some applications, less stringent hybridization conditions are required, by way of example and not limitation such a 0.15 M to 0.9 M salt, at temperatures ranging from between about 20°C to 55°C. Hybridization conditions can also be rendered more stringent by the addition of increasing amounts of formamide, to destabilize the hybrid duplex. Thus, particular hybridization conditions can be readily manipulated, and will generally be a method of choice depending on the desired results. By way of example and not limitation, in general, convenient hybridization temperatures in the presence of 50% formamide are: 42°C for a probe which is 95 to 100% homologous to the target fragment, S^C for 90 to 95% homology and 32°C for 70 to 90% homology. Low, moderate and high stringency conditions are well known to those of skill in the art, and will vary predictably depending on the base composition and length of the particular nucleic acid sequence and on the specific organism from which the nucleic acid sequence is derived. For guidance regarding such conditions see, for example, Sambrook et al., 1989, Molecular Cloning, A Laboratory Manual, Second Edition, Cold Spring Harbor Press, N.Y., pp. 9.47-9.57; and<br><br>
Ausubel et al., 1989, Current Protocols in Molecular Biology, Green Publishing Associates and Wiley Interscience, N.Y. which is incorporate herein, by reference.<br>
In the preparation of genomic libraries, DNA fragments are generated, some of which will encode parts or the whole of Chlamydia HMW protein. The DNA may be cleaved at specific sites using various restriction enzymes. Alternatively, one may use DNase in the presence of manganese to fragment the DNA, or the DNA can be physically sheared, as for example, by sonication. The DNA fragments can then be separated according to size by standard techniques, including but not limited to, agarose and polyacrylamide gel electrophoresis, column chromatography and sucrose gradient centrifiigation. The DNA fragments can then be inserted into suitable vectors, including but not limited to plasmids, cosmids, bacteriophages lambda or T4, bacmids and yeast artificial chromosome (YAC). (See, for example, Sambrook et al., 1989, Molecular Cloning, A Laboratory Manual, 2d Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York; Glover, D. M. (ed.), 1985, DNA Cloning: A Practical Approach, MRL Press, Ltd., Oxford, U.K. Vol. I, II.) The genomic library may be screened by nucleic acid hybridization to labeled probe (Benton and Davis, 1977, Science 196:180; Grunstein and Hogness, 1975, Proc. Natl. Acad. Sci. U.S.A. 72:3961).<br>
The genomic libraries may be screened with labeled degenerate oligonucleotide probes corresponding to the amino acid sequence of any peptide of HMW protein using optimal approaches well known in the art. In particular embodiments, the screening probe is a degenerate oligonucleotide that corresponds to the sequence of SEQ ID NO: 4. In another embodiment, the screening probe may be a degenerate oligonucleotide that corresponds to the sequence of SEQ ID N0:5. In an additional embodiment, any one of the oligonucleotides of SEQ ID NOs: 6-9, 12-14 and 18-21 are used as the probe. In fiirther embodiments, any one of the sequences of SEQ ID NOs: 1, 10-11, 22-24 or any fragments thereof, or any complement of the sequence or fragments may be used as the probe. Any probe used preferably is 15 nucleotides or longer. Clones in libraries with insert DNA encoding the HMW protein or fragments thereof will hybridize to one or more of the degenerate oligonucleotide probes. Hybridization of such oligonucleotide probes to genomic libraries are carried out using methods known in the art. For example, hybridization with the two above-mentioned oligonucleotide probes may be carried out in 2X SSC, 1.0% SDS at 50°C and washed using the same conditions.<br>
In yet another aspect, clones of nucleotide sequences encoding a part or the entire HMW protein or HMW-derived polypeptides may also be obtained by screening Chlamydia expression libraries. For example. Chlamydia DNA or Chlamydia cDNA generated from RNA is isolated<br><br>
and random fragments are prepared and ligated into an expression vector (e.g., a bacteriophage, plasmid, phagemid or cosmid) such that the inserted sequence in the vector is capable of being expressed by the host cell into which the vector is then introduced. Various screening assays can then be used to select for the expressed HMW protein or HMW-derived polypeptides. In one embodiment, the various anti-HMW antibodies of the invention can be used to identify the desired clones using methods known in the art. See, for example, Harlow and Lane, 1988, Antibodies: A Laboratory Manual Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, Appendix IV. Clones or plaques from the library are brought into contact with the antibodies to identify those clones that bind.<br>
In an embodiment, colonies or plaques containing DNA that encodes HMW protein or HMW-derived polypeptide could be detected using DYNA Beads according to Olsvick et al, 29th ICAAC, Houston, Tex. 1989, incorporated herein by reference. Anti-HMW antibodies are crosslinked to tosylated DYNA Beads M280, and these antibody-containing beads would then be used to adsorb to colonies or plaques expressing HMW protein or HMW-derived polypeptide. Colonies or plaques expressing HMW protein or HMW-derived polypeptide is identified as any of those that bind the beads.<br>
Alternatively, the anti-HMW antibodies can be nonspecifically immobilized to a suitable support, such as silica or Celite™' resin. This material would then be used to adsorb to bacterial colonies expressing HMW protein or HMW-derived polypeptide as described in the preceding paragraph. In another aspect, PCR amplification may be used to produce substantially pure DNA encoding a part of or the whole of HMW protein from Chlamydia genomic DNA. Oligonucleotide primers, degenerate or otherwise, corresponding to known HMW protein sequences can be used as primers. In particular embodiments, an oligonucleotide, degenerate or otherwise, encoding the peptide having an amino acid sequence of SEQ ID NO: 2, 3 or 15-17 or any portion thereof may be used as the 5' primer. For fragment examples, a 5' primer may be made from any one of the nucleotide sequences of SEQ ID NO: 4-7, 10, 12, 22-24 or any portion thereof. Nucleotide sequences, degenerate or otherwise, that are reverse complements of SEQ ID NO: 11, 13 or 14 may be used as the 3' primer.<br>
PCR can be carried out, e.g., by use of a Perkin-Elmer Cetus thermal cycler and Taq polymerase (Gene Amp^'^). One can choose to synthesize several different degenerate primers, for use in the PCR reactions. It is also possible to vary the stringency of hybridization conditions used in priming the PCR reactions, to allow for greater or lesser degrees of nucleotide sequence similarity between the degenerate primers and the corresponding sequences in Chlamydia DNA.<br><br>
After successfiil amplification of a segment of the sequence encoding HMW protein, that segment may be molecularly cloned and sequenced, and utilized as a probe to isolate a complete genomic clone. This, in turn, will permit the determination of the gene's complete nucleotide sequence, the analysis of its expression, and the production of its protein product for functional analysis, as described infra.<br>
In a clinical diagnostic embodiment, the nucleic acid sequences of the HMW protein genes of the present invention may be used in combination with an appropriate indicator means, such as a label, for determining hybridization. A wide variety of appropriate indicator means are known in the art, including radioactive, enzymatic or other ligands, such as avidin/biotin and digoxigenin-labelling, which are capable of providing a detectable signal. In some diagnostic embodiments, an enzyme tag such as urease, alkaline phosphatase or peroxidase, instead of a radioactive tag may be used. In the case of enzyme tags, colorimetric indicator substrates are known which can be employed to provide a means visible to the human eye or spectrophotometrically, to identify specific hybridization with samples containing HMW protein gene sequences. The nucleic acid sequences of the HMW protein genes of the present invention are usefiil as hybridization probes in solution hybridizations and in embodiments employing solid-phase procedures. In embodiments involving solid-phase procedures, the test DNA (or RNA) from samples, such as clinical samples, including exudates, body fluids (e.g., serum, amniotic fluid, middle ear effusion, sputum, semen, urine, tears, mucus, bronchoalveolar lavage fluid) or even tissues, is absorbed or otherwise affixed to a selected matrix or surface. The fixed, single-stranded nucleic acid is then subjected to specific hybridization with selected probes comprising the nucleic acid sequences of the HMW protein encoding genes or fragments or analogues thereof of the present invention under desired conditions. The selected conditions will depend on the particular circumstances based on the particular criteria required depending on, for example, the G+C contents, type of target nucleic acid, source of nucleic acid, size of hybridization probe etc. Following washing of the hybridization surface so as to remove non-specifically bound probe molecules, specific hybridization is detected, or even quantified, by means of the label. It is preferred to select nucleic acid sequence portions which are conserved among species of Chlamydia. The selected probe may be at least 15 bp and may be in the range of about 30 to 90 bp.<br><br>
Expression of the HMW protein Gene<br>
Plasmid vectors containing replicon and control sequences which are derived from species compatible with the host cell may be used for the expression of the genes encoding the HMW protein or fragments thereof in expression systems. Expression vectors contain all the necessary elements for the transcription and translation of the inserted protein coding sequence. The vector ordinarily carries a replication site, as well as marking sequences which are capable of providing phenotype selection in transformed cells. For example, E. coli may be transformed using pBR322 which contains genes for ampicillin and tetracycline resistance cells. Other commercially available vectors are useful, including but not limited to pZERO, pTrc99A, pUC19, pUC18, pKK223-3, pEXl, pCAL, pET, pSPUTK, pTrxFus, pFastBac, pThioHis, pTrcHis, pTrcHis2, and pLEx. The plasmids or phage, must also contain, or be modified to contain, promoters which can be used by the host cell for expression of its own proteins.<br>
In addition, phage vectors containing replicon and control sequences that are compatible with the host can be used as a transforming vector in connection with these hosts. For example, the phage in lambda GEM^*^-! 1 may be utilized in making recombinant phage vectors which can be used to transform host cells, such as E. coli LE392.<br>
Promoters commonly used in recombinant DNA construction include the P-lactamase (penicillinase) and lactose promoter systems and other microbial promoters, such as the T7 promoter system as described in U.S. Patent 4,952,496. Details concerning the nucleotide sequences of promoters are known, enabling a skilled worker to ligate them functionally with genes. The particular promoter used will generally be matter of choice depending upon the desired results.<br>
In accordance with this invention, it is preferred to make the HMW protein by recombinant methods, particularly when the naturally occurring HMW protein as isolated from a culture of a species of Chlamydia may include trace amounts of toxic materials or other contaminants. This problem can be avoided by using recombinantly produced HMW protein in heterologous systems which can be isolated from the host in a manner to minimize contaminants in the isolated material. Particularly desirable hosts for expression in this regard include Gram positive bacteria which do not have LPS and are, therefore endotoxin free. Such hosts include species of Bacillus and may be particularly useful for the production of non-pyrogenic rHMW protein, fragments or analogues thereof.<br>
A variety of host-vector systems may be utilized to express the protein-coding sequence. These include but are not limited to mammalian cell systems infected with virus {e.g., vaccinia virus,<br><br>
adenovirus, etc.); insect cell systems infected with virus {e.g., baculovirus); microorganisms such as yeast containing yeast vectors, or bacteria transformed with bacteriophage DNA, plasmid DNA, or cosmid DNA. Hosts that are appropriate for expression of the HMW protein genes, fragments, analogues or variants thereof, may include E. coli. Bacillus species, Haemophilus, fungi, yeast, such as Saccharomyces pichia, Bordetella, or the baculovirus expression system may be used. Preferably, the host cell is a bacterium, and most preferably the bacterium is E. coli, B. subtilis or Salmonella.<br>
The expression elements of vectors vary in their strengths and specificities. Depending on the host-vector system utilized, any one of a number of suitable transcription and translation elements may be used. In a preferred embodiment, a chimeric protein comprising HMW protein or HMW-derived polypeptide sequence and a pre and/or pro sequence of the host cell is expressed. In other preferred embodiments, a chimeric protein comprising HMW protein or HMW-derived polypeptide sequence fused with, for example, an affinity purification peptide, is expressed. In further preferred embodiments, a chimeric protein comprising HMW protein or HMW-derived polypeptide sequence and a useful immunogenic peptide or protein is expressed. In preferred embodiments, HMW-derived protein expressed contains a sequence forming either an outer-surface epitope or the receptor-binding domain of native HMW protein. Any method known in the art for inserting DNA fragments into a vector may be used to construct expression vectors containing a chimeric gene consisting of appropriate transcriptional/translation control signals and the protein coding sequences. These methods may include in vitro recombinant DNA and synthetic techniques and in vivo recombinants (genetic recombination). Expression of a nucleic acid sequence encoding HMW protein or HMW-derived polypeptide may be regulated by a second nucleic acid sequence so that the inserted sequence is expressed in a host transformed with the recombinant DNA molecule. For example, expression of the inserted sequence may be controlled by any promoter/enhancer element known in the art. Promoters which may be used to control expression of inserted sequences include, but are not limited to the SV40 early promoter region (Bemoist and Chambon, 1981, Nature 290:304-310), the promoter contained in the 3' long terminal repeat of Rous sarcoma virus (Yamamoto et al., 1980, Cell 22:787-797), the herpes thymidine kinase promoter (Wagner et al., 1981, Proc. Natl. Acad. Sci. U.S.A. 78:1441-1445), the regulatory sequences of the metallothionein gene (Brinster et al., 1982, Nature 296:39-42) for expression in animal cells; the promoters of P-lactamase (Villa-Kamaroff e/ al., 1978, Proc. Natl. Acad. Sci. U.S.A. 75:3727-3731), tac (DeBoer et al., 1983, Proc. Natl. Acad. Sci. U.S.A. 80:21-25), PL, or trc for expression in bacterial cells (see also<br><br>
"Useftil proteins from recombinant bacteria" in Scientific American, 1980, 242:74-94); the nopaline synthetase promoter region or the cauliflower mosaic virus 35S RNA promoter (Gardner et al., 1981, Nucl. Acids Res. 9:2871), and the promoter of the photosynthetic enzyme ribulose biphosphate carboxylase (Herrera-Estrella et al., 1984, Nature 310:115-120) for expression implant cells; promoter elements from yeast or other fungi such as the Gal4 promoter, the ADC (alcohol dehydrogenase) promoter, PGK (phosphoglycerol kinase) promoter, alkaline phosphatase promoter.<br>
Expression vectors containing HMW protein or HMW-derived polypeptide coding sequences can be identified by three general approaches: (a) nucleic acid hybridization, (b) presence or absence of "marker" gene functions, and (c) expression of inserted sequences such as reactivity with anti-HMW antibody. In the first approach, the presence of a foreign gene inserted in an expression vector can be detected by nucleic acid hybridization using probes comprising sequences that are homologous to the inserted HMW protein or HMW-derived polypeptide coding sequence. In the second approach, the recombinant vector/host system can be identified and selected based upon the presence or absence of certain "marker" gene functions (e.g., thymidine kinase activity, resistance to antibiotics, transformation phenotype, occlusion body formation in baculovirus, etc.) caused by the insertion of foreign genes in the vector. For example, if the HMW protein or HMW-derived polypeptide coding sequence is inserted within the marker gene sequence of the vector, recombinants containing the insert can be identified by the absence of the marker gene function. In the third approach, recombinant expression vectors can be identified by assaying the foreign gene product expressed by the recombinant. Such assays can be based, for example, on the physical or functional properties of HMW protein or HMW-derived polypeptide in in vitro assay systems, e.g., binding to an HMW ligand or receptor, or binding with anti-HMW antibodies of the invention, or the ability of the host cell to hemagglutinate or the ability of the cell extract to interfere with hemagglutination by Chlamydia.<br>
Once a particular recombinant DNA molecule is identified and isolated, several methods known in the art may be used to propagate it. Once a suitable host system and growth conditions are established, recombinant expression vectors can be propagated and prepared in quantity. As explained above, the expression vectors which can be used include, but are not limited to, the following vectors or their derivatives: human or animal viruses such as vaccinia virus or adenovirus; insect viruses such as baculovirus; yeast vectors; bacteriophage vectors (e.g., lambda), and plasmid and cosmid DNA vectors, to name but a few.<br><br>
In addition, a host cell strain may be chosen which modulates the expression of the inserted sequences, or modifies and processes the gene product in the specific fashion desired. Expression from certain promoters can be elevated in the presence of certain induces; thus, expression of the genetically engineered HMW protein or HMW-derived HMW may be controlled. Furthermore, different host cells have characteristic and specific mechanisms for the translational and post-translational processing and modification of proteins. Appropriate cell lines or host systems can be chosen to ensure the desired modification and processing of the foreign protein expressed. The proteins, polypeptides, peptides, antibodies and nucleic acids of the invention are useful as reagents for clinical or medical diagnosis of Chlamydia infections and for scientific research on the properties of pathogenicity, virulence, and infectivity of Chlamydia, as well as host defense mechanisms. For example, DNA and RNA of the invention can be used as probes to identify the presence of Chlamydia in biological specimens by hybridization or PCR amplification. The DNA and RNA can also be used to identify other bacteria that might encode a polypeptide related to the Chlamydia HMW protein. The proteins of the invention may be used to prepare polyclonal and monoclonal antibodies that can be used to further purify compositions containing the proteins of the invention by affinity chromatography. The proteins can also be used in standard immunoassays to screen for the presence of antibodies to Chlamydia in a sample.<br>
7.        BIOLOGICAL DEPOSITS<br>
Certain plasmids that contain portions of the gene having the open reading frame of the gene encoding the HMW protein of Chlamydia that are described and referred to herein have been deposited with the America Type Culture Collection (ATCC) located at 10801 University Boulevard, Manassas, Virginia 20110-2209, U.S.A., pursuant to the Budapest Treaty and pursuant to 37 CFR 1.808 and prior to the filing of this application. The identifications of the respective portions of the genes present in these plasmids are shown below. Samples of the deposited materials will become available to the public upon grant of a patent based upon this United States patent application. The invention described and claimed herein is not to be limited by the scope of the plasmids deposited, since the deposited embodiment is intended only as an illustration of the invention. Any equivalent or similar plasmids that encode similar or equivalent proteins or fragments or analogues thereof as described in this application are within the scope of the invention.<br><br>
Microorganism	ATCC Accession No.        Date Deposited<br>
pAH342	ATCC 985538	September 8, 1997<br>
8.        EXAMPLES<br>
The above disclosure generally describes the present invention. A more specific description is provided below in the following examples. The examples are described solely for the purpose of illustration and are not intended to limited the scope of the invention. Changes in form and substitution of equivalents are contemplated as circumstances suggest or render expedient. Although specific terms have been employed herein, such terms are intended in a descriptive sense and not for purposes of limitation.<br>
Methods of molecular genetics, protein biochemistry and immunology used but not explicitly described in the disclosure and examples are amply reported in the scientific literature and are well within the ability of those skilled in the art.<br>
8.1.      EXAMPLE     1;     ISOLATION    AND    PURIFICATION    OF    MATURE CHLAMYDIA PROTEINS<br>
McCoy cells were cultured either in standard 225 cm^ tissue culture flasks or in Bellco spinner flasks (Cytodex microcarrier, Pharmacia) at 37°C in 5% CO2 using DMEM media supplemented with 10% Chlamydia-mtihody free fetal bovine serum, glucose and nonessential amino acids. C. trachomatis L2 elementary bodies (ATCC VR-902B) were prepared fi-om lysates of infected McCoy cells. Basically, McCoy cells infected with C. trachomatis L2 (LGV) were sonicated and cellular debris was removed by centrifiigation. The supernatant containing chlamydial elementary bodies (EBs) was then centrifuged and the pellet containing EBs was resuspended in Hanks' balanced salts solution (HBSS). RNase/DNase solution was added and incubated at 37°C for 1 hour with occasional mixing. The EB containing solution was layered onto a discontinuous density gradient (40%, 44% and 54%) of Angiovist 370 (mixture of diatrizoate melgumine and diatrizoate sodium, Berlex Laboratories, Wayne, NJ) and ultracentrifiiged for separation of the EBs on the gradient. After centrifiigation the EBs were harvested fi-om the gradient between the interface of the 44% and 54% Angiovist 370 layers. The EBs were washed in phosphate buffered saline and resuspended in HBSS.<br>
Purified EBs were sequentially extracted with 0.1% OGP [high ionic strength] in HBSS to remove peripheral surface proteins and held on ice. The same EB preparation was then extracted with 1.0% OGP, 10 mM DTT, 1 mM PMSF, 10 mM EDTA, in a 50 mM Tris pH 7.4 buffer. Extracts were dialyzed (3500 MWCO) to remove detergent and other reagents and concentrated<br><br>
by lyophilization. Protein containing extracts were diluted in HBSS and passed over commercially available heparin-sepharose columns (HiTrap Col., Pharmacia). After samples were applied to the heparin column nonadhered proteins were removed by washing with excess HBSS. Bound proteins were batch eluted with PBS containing 2 M sodium chloride. Eluents were dialyzed extensively to remove salt and then lyophilized. The heparin-binding proteins were size fractionated by SDS-PAGE and visualized by silver staining or analyzed by Western blotting. Protein(s) of about 105-115 KDa present in moderate amounts were detected as shown in Figure 1. The isoelectric point of the native HMW protein was determined to be about 5.95. To obtain one N-terminal amino acid sequence, sufficient quantities of the HMW protein (&gt;5 ^lg) were electroblotted onto a PVDF membrane (Applied Biosystems), and stained with Coomassie blue. Immobilized HMW protein was released from the membrane and treated in situ with low levels of endopeptidase Lys-C, endopeptidase Arg-C and/or endopeptidase Glu-C to fragment the native protein. The resulting peptide fragments were purified by HPLC and their N-terminal amino acid sequences determined using an ABI 430 Protein Sequenator and standard protein sequencing methodologies. The N-terminal amino acid sequence is:<br>
E-I-M-V-P-Q-G-I-Y-D-G-E-T-L-T-V-S-F-X-Y<br>
and is denoted SEQ ID No.: 3. When a composite PDB+SwissProt+PIR+GenPept database (&gt;145 K unique sequences) was searched with the HMW protein N-terminal sequence (20 residues) using rigorous match parameters, no precise homologies were found. Thus the HMW protein is a novel chlamydial protein. Since this protein was isolated under conditions that should release only peripheral membrane proteins (e.g. Omp2), these data indicate that the HMW protein is a surface-associated protein.<br>
8.2.      EXAMPLE 2: PREPARATION OF ANTIBODIES TO WHOLE CHLAMYDIA EBs<br>
To aid in the characterization of the HMW protein, hyperimmune rabbit antisera was raised against whole EBs from C. trachomatis L2. Each animal was given a total of three immunizations of about 250 ^g of Chlamydia EBs per injection (beginning with complete Freund's adjuvant and followed with incomplete Freund's adjuvant) at approximately 21 day intervals. At each immunization, approximately half of the material was administered intramuscularly (i.m.) and half was injected intranodally. Fourteen days after the third vaccination a fourth booster of about<br><br>
100 ng of EBs was given i.m. and the animals exsanguinated 7-10 days later. A titre of 1:100,000 was obtained as determined by ELISA.<br>
8.3.     EXAMPLE      3:      DETERMNIATION      OF      POST-TRANSLATIONAL MODIFICATIONS<br>
Recently, several C. trachomatis membrane-associated proteins have been shown to be post-translationally modified. The 18 kDa and 32 kDa cysteine-rich EB proteins, which are lectin-binding proteins, have been shown to carry specific carbohydrate moieties (Swanson, A. F. and C. C. Kuo. 1990. Infect. Immun. 58:502-507). Incorporation of radiolabelled palmitic acid has been used to demonstrate that the about 27 kDa C. trachomatis Mip-like protein is lipidated (Lundemose, A. G., D. A. Rouch, C. W. Penn, and J. H. Pearce. 1993. J. Bacteriol. 175:3669-3671). Swanson et al. have discovered that the MOMP from the L2 serovar contains N-acetylglucosamine and/or N-acetylgalactosamine and these carbohydrate moieties mediate binding of MOMP to Hela cell membranes.<br>
To ascertain whether the HMW protein is glycosylated, EBs are grown on McCoy cells in the presence of tritiated galactose or glucosamine, subjected to heparin affinity chromatography and the heparin binding proteins analyzed by SDS-PAGE and autoradiography. Briefly, McCoy cells are grown in T225 flasks under standard conditions (DMEM+10% FCS, 35 ml per flask, 10% CO2) to about 90% confluency and inoculated with sufficient EBs to achieve 90%-100% infectivity. Following a 3 hour infection period at 37°C cycloheximide is added (1 |J.g/ml) to inhibit host cell protein synthesis and the cultures reincubated for an additional 4-6 hours. Approximately 0.5 mCi of tritiated galactose (D-[4,5-^H(N)]galactose, NEN) or glucosamine (D-[1,6-^ H(N)glucosamine, NEN) is then be added to each flask and the cultures allowed to incubate for an additional 30-40 hours. Cells are harvested by scraping and EBs purified by gradient centrifugation. HMW protein is isolated from 1.0% OGP surface extracts by affinity chromatography, eluted with NaCl and analyzed by SDS-PAGE using '" C-labelled molecular weight markers (BRL) then subjected to autoradiography. Dried gels are exposed for 1-4 weeks to Kodak X-AR film at -70°C.<br>
To determine post synthesis lipid modification, C. trachomatis serovar L2 is cultivated on monolayers of McCoy cells according to standard procedures. Approximately 24 hours postinfection, conventional culture media (DMEM+10% FCS) is removed and replaced with a serum-free medium containing cycloheximide (1 ^ig/ml) and [U-'"* CJpalmitic acid (0.5 mCi/T225 flask, NEN) and incubated for a further 16-24 hours to allow protein lipidation to<br><br>
occur. Surface EB extracts are prepared, heparin-binding proteins are isolated and analyzed by autoradiography as described above.<br>
The functionality of glycosylated or lipidated moieties is assessed by treating whole EBs or OGP surface extracts with appropriate glycosidases. Following carbohydrate removal, extracts are subjected to affinity chromatography and SDS-PAGE to determine whether the HMW protein retains the ability to bind to heparin sulfate.<br>
8.4.      EXAMPLE 4; CLONING OF THE N-TERMINAL SEGMENT OF THE HMW PROTEIN GENE<br>
Degenerate oligonucleotides were designed based on the N-terminal amino acid sequence of the HMW protein and were synthesized. These oligonucleotides were then used to generate gene-specific PCR products that were employed as hybridization probes to screen a C. trachomatis L2 5iZAPII DNA library to isolate the gene for the HMW protein.<br>
Briefly, appropriate low degeneracy peptide segments were identified fi-om the N-terminal and<br>
internal amino acid sequence data by computer analysis (MacVector, IBI) and used to guide the<br>
design of low degeneracy sequence-specific oligonucleotide PCR primer sets.<br>
Using the N-terminal primary sequence as a guide, four degenerate oligonucleotide probes<br>
complementary to the first six residues of the HMW peptide E-I-M-V-P-Q (residues 1-6 of SEQ<br>
ID NO.: 3), and comprising all possible nucleotide combinafions (total degeneracy=192<br>
individual sequences), have been designed and employed as forward amplification primers.<br>
SEQ ID N0.4	5'-GAA-ATH-ATG-GTN-CCN-CAA-3'.<br>
SEQ ID N0.5	5'-GAA-ATH-ATG-GTN-CCN-CAG-3'<br>
SEQ ID N0.6	5'-GAG-ATH-ATG-GTN-CCN-CAA-3'<br>
SEQ ID N0.7	5'-GAG-ATH-ATG-GTN-CCN-CAG-3'<br>
Two additional oligonucleotide probes representing the reverse complement DNA sequence of the internal five residue peptide Y-D-G-E-T (residues 9-13 of SEQ ID No.: 3), and comprising all possible nucleotide combinations (total degeneracy=128 individual sequences), have been designed and employed as reverse amplification primers.<br>
SEQ ID N0.8	5'-NGT-YTC-NCC-RTC-ATA-3'<br>
SEQ ID N0.9	5'-NGT-YTC-NCC-RTC-GTA-3'<br>
Oligonucleotides were synthesized on an ABI Model 380B DNA synthesizer using a 0.2 |j.mol scale   column   (tritylon,   auto-cleavage)   and   standard   phosphoramidite   chemistry.   Crude<br><br>
oligonucleotides were manually purified over C-18 syringe columns (OP Columns, ABI). Purity and yield were ascertained spectrophotometrically (230/260/280 ratios).<br>
Standard PCR amplification reactions (2 mM Mg^"", 200 umol dNTPs, 0.75 units AmpliTaq, 50 H\ final volume) were programmed using about 0.2 jig C. trachomatis La DNA (about 3x10^ copies of the HMW protein gene if single copy) and about 100 pmol of each forward (N-terminal oligo) and reverse (internal oligo) primer. Higher than normal concentrations of primers (-20 pmol/50 ^1) were used for amplification, at least initially, in order to compensate for primer degeneracy. Amplification of target sequences was achieved using a standard 30-cycle, three-step thermal profile, i.e. 95°C, 30 sec; 60°C, 45 sec, 72°C, 1 min. Amplification was carried out in sealed 50 \i\ glass capillary tubes using a Idaho Technologies thermal cycler. To verify that the PCR products generated during these amplification reactions were specific for the HMW protein coding sequence and were not "primer-dimer" or other DNA amplification artifacts, amplimers were purified using silica-gel spin columns (QIAGEN), cloned into the PCR cloning vector pZERO (StrataGene), and subjected to direct DNA sequence analysis.<br>
The DNA sequence for the cloned PCR products were determined using conventional dideoxy-terminator sequencing chemistry and a modified T7 DNA polymerase (Sequenase, USB). Briefly, each double stranded plasmid template was denatured by a brief treatment with NaOH. Following neutralization, each denatured template was used to program 4 separate sequencing reactions. Each reaction contained the Ml 3 universal forward sequencing primer (21-mer) but a different ddNTP/dNTP termination mix (i.e. A,G,C, or T). Termination products were labelled by including [a-^^S]dATP in the reaction (-50 uCi/reaction, &gt;3000 Ci/mmol, Amersham). Individual extension products were denatured (formamide, ~95°C) and subjected to high resolution denaturing polyacrylamide gel electrophoresis (6% acrylamide, 8M urea, TAE buffer, -500 V, -90 min). Sequencing gels were then transferred to filter paper (Whatmann 3MM), dried under vacuum, and then autoradiographed at -70°C for 24-72 hours. Base ladders were read manually fi-om each gel and a consensus sequence determined.<br>
HMW protein-specific amplimers suitable for library screening and/or Southern blotting were produced by PCR and uniformly radiolabelled during the amplification process by adding [a-^^ PJdNTPs (about 50 p.Ci each dNTP, Amersham, &gt;5000 Ci/mmol) to the reaction mixture. Labelling reactions were performed as above except reactions were performed in 0.5 ml microcentrifuge tubes using a Bellco Thermal Cycler. Unincorporated label and amplification primers were removed from the reaction mixture using centrifijgal size-exclusion chromatography columns (BioSpin 6 columns, BioRad).<br><br>
A highly redundant C. trachomatis serovar L2 DNA library (&gt;50,000 primary clones) has been constructed by cloning size^iractionated fragments &gt;10 Kbp produced from a partial EcoRI digest of genomic DNA into the lambda cloning vector &gt;ZAPII (Stratagene). Radiolabelled HMW protein-specific PCR products were used to screen this library for recombinant clones that carry all or part of the HMW protein coding sequence. Standard recombinant DNA procedures and methodologies were employed for these experiments. All phage that hybridized with these probes were purified to homogeneity by sequential rounds of plating and hybridization screening. Once reactive phage were purified, insert-containing phagmids (pBluescript SK-derivatives) were excision-rescued from the parental phage by coinfecting host cells with an appropriate helper phage, e.g. R408 or VCSM13 (Stratagene). Individual phagmids were further purified by streak-plating on LB agar containing ampicillin (100 ^g/ml) and selecting for individual colonies. To confirm purified phagemid derivatives carried the HMW protein sequences, plasmid DNA was prepared and used to program amplification reactions containing the HMW protein-specific PCR primer sets. The presence of HMW protein-specific inserts was confirmed by the production of the appropriate sized PCR product.<br>
Plasmid pAH306 is one HMW protein-containing derivative that was isolated by these methodologies.<br>
Phvsical Mapping of pAH306<br>
The inserts from pAH306 were physically mapped and the location of HMW protein gene determined using appropriate six-base restriction endonucleases (e.g. EcoRI, Hindlll, BamHI, PstI, Smal, Kpnl, etc.) and HMW protein coding sequences localized by Southern hybridization employing radiolabelled N-terminal-specific PCR products as probes. The orientation and extent of HMW protein-specific sequences were determined by PCR analysis using primer sets consisting of HMW protein-specific forward primers and reverse primers complementary to either the T3 or T7 promoter sequences located in the cloning vector.<br>
Plasmid pAH306 was determined to contain a single ~6.6 Kbp EcoRI fragment of chlamydial origin. Directional PCR analysis of pAH306 demonstrated this derivative encodes roughly 1.5 Kbp of the N-terminal region of the HMW protein gene.<br>
The DNA sequence for the HMW protein gene encoded on pAH306 was obtained for both strands via conventional "sequence-walking" coupled with asymmetric PCR cycle sequencing methodologies (ABI Prism Dye-Terminator Cycle Sequencing, Perkin-Elmer). Sequencing<br><br>
reactions were programmed with undigested plasmid DNA (~0.5 ng/rxn) as a template and appropriate HMW protein-specific sequencing primers (~3.5 pmol/rxn).<br>
hi addition to the template and sequencing primer, each sequencing reaction (~20 JAI) contained the four different dNTPs (i.e. A,G,C, and T) and the four corresponding ddNTPs (i.e. ddA, ddG, ddC, and ddT) terminator nucleotides; with each terminator being conjugated to one of four different fluorescent dyes. Single strand sequencing elongation products were terminated at random positions along the template by the incorporation of the dye-labelled ddNTP terminators. Fluorescent dye-labelled termination products were purified using microcentrifuge size-exclusion chromatography columns (Princeton Genetics), dried under vacuum, suspended in a Template Resuspension Buffer (Perkin-Elmer), denatured at 95°C for ~5 min, and resolved by high resolution capillary electrophoresis on an ABI310 Automated DNA Sequenator (Perkin-Elmer). DNA sequence data produced from individual reactions were collected and the relative fluorescent peak intensities analyzed automatically on a PowerMAC computer using ABI Sequence Analysis Software (Perkin-Elmer). Individually autoanalyzed DNA sequences were edited manually for accuracy before being merged into a consensus sequence "string" using AutoAssembler software (Perkin-Elmer). Both strands of the HMW protein gene segment encoded by pAH306 were sequenced and these data compiled to create a composite sequence for the HMW protein gene segment. The sequence encoding the segment of HMW protein is listed as SEQ ID NO.: 10 and is represented by nucleotides 382 to 1979 in Figure 2. A map of pAH306 is shown in Figure 5.<br>
Database analysis (e.g. primary amino acid homologies, hydropathy profiles, N-0-glycosylation sites, functional/conformational domain analyses) of the DNA and predicted amino acid sequences for the HMW protein was performed using GeneRunner and Intelligentics software, indicating the HMW protein is novel.<br>
8.5.     EXAMPLE 5; CLONING OF THE C-TERMINAL SEGMENT OF THE HMW PROTEIN GENE<br>
Chromosome walking was employed to isolate the C-terminal portion of the HMW protein gene. A -0.6 Kbp BamHI-EcoRI fragment distal to the N-terminal sequence of the mature HMW protein and proximal to the T3 promoter sequence of the vector was chosen as the probe for the initial chromosome walk. Briefly, pAH306 was digested to completion with BamHI and EcoRI and the digestion products size fractionated by agarose gel electrophoresis (0.8% agarose in TAE buffer). The desired ~0.6 Kbp BamHI/EcoRI (B/E) band was excised from the gel and purified<br><br>
using commercially available silica gel microcentrifuge chromatography columns and reagents (QIAGEN).<br>
The purified 0.6 Kbp B/E fragment was radiolabelled with [a-dATP] (&gt;3000 Ci/mmol, Amersham) via random-priming labelling methodologies employing commercially available reagents (Boehringer Mannheim) and used to probe Southern blots of C. trachomatis L2 genomic DNA that had been digested to completion with Hindlll.<br>
The 0.6 Kbp B/E probe from pAH306 hybridized to a -1.4 Kbp Hindlll genomic fragment. Based on the experimentally derived restriction map of the HMW protein gene segment encoded on pAH306, this fragment encodes -0.2 Kbp of the C-terminal HMW protein sequence. The radiolabelled 0.6 Kbp B/E fragment was used subsequently to probe a moderately redundant (-5,000 primary clones) C. trachomatis L2 library to identify clones that contain the -1.4 Kbp Hindlll fragment. Briefly, C trachomatis L2 genomic DNA was digested to completion using a -10-fold excess of the restriction endonuclease Hindlll (-10 units per 1 |j,g of genomic DNA, 37°C, 18-24 hours). Digestion products were size fractionated by agarose gel electrophoresis (0.8% agarose, TAE) and DNA fragments ranging in size from -1.0 Kbp to 2.0 Kbp were excised from the gel. Excised agarose strips contain the desired DNA fragment sizes were dissolved in a soIubilization^inding solution (QXl,QIAGEN) and purified using commercially available silica-gel spin columns (QIAGEN). Purified 1.0-2.0 Kbp genomic Hindlll fragments were then cloned into the pBlueScript SK- plasmid which had been previously digested to completion with Hindlll and treated with calf intestinal phosphatase to prevent vector religatlon. Vector/insert ligations were performed in a -50 |4,1 final reaction volume (50 mM Tris-HCl, pH 7.00; 10 mM NaCl; 1 mM ATP; 0.5 mM DTT) at 25'C for -16-24 hours using T4 DNA Ugase (-10 units/reaction) and a vector:insert molar ratio of approximately 1:10. Following ligation, aliquots (-50 ng ligated DNA) were used to electroporate a competent E. coli host, e.g. E. coli TOPIO. Electroporated cells were then plated onto LB agar containing -100 ng/ml ampicillin to select for plasmid-harboring clones. Approximately 1,000 plasmid-harboring Ap"^ transformants were transferred directly from LB Ap'°° agar plates onto nylon membranes (HyBond N+, Amersham) by capillary action.<br>
Following transfer, plates were re-incubated at 37'C to regenerate viable colonies for flirther manipulation. Colonies transferred to membranes were lysed and DNA liberated by treating the colony blots with a denaturing SDS/NaOH solution. A Tris buffered NaCI solution was used to neutralize and stabilize lysis material. Released DNA was immobilized onto the membranes by UV irradiation. Standard recombinant DNA procedures and methodologies were employed to<br><br>
probe the colony blots with the radiolabeled 0.6 Kbp B/E fragment and identify recombinant derivatives which carry the desired -1.4 Kbp Hindlll fragment.<br>
Plasmid pAH310 was one derivative isolated by these procedures and the coding segment of the HMW protein is represented by nucleotides 994-2401 in FIG. 2.<br>
Restriction analysis using Hindlll and EcoRI, individually and in combination, together with DNA sequence analysis of purified plasmid DNA confirmed pAH310 encodes the expected -1.4 Kbp Hindlll fragment. These analyses also demonstrated that the -1.4 Kbp insert consists of the same -1.2 Kbp Hindlll-EcoRI fragment that is present in pAH306 and a unique -0.2 Kbp EcoRI-Hindlll fragment that encodes C-terminal HMW protein-specific DNA.<br>
The -0.2 Kbp EcoRI-Hindlll (E/H) fragment was chosen as the probe for the second chromosome walk. Briefly, pAHSlO was digested to completion with EcoRI and Hindlll and the digestion products size fractionated by agarose gel electrophoresis (0.8% agarose in TAE buffer). The desired -0.2 Kbp (E/H) band was excised from the gel, purified, radiolabelled with [a-P^^ ]dATP, and used as a probe to identify clones in the original C. trachomatis L2 XLA?\l genomic library that encode the C-terminal segment of the HMW protein gene.<br>
Plasmid pAH316 is one derivative isolated by these procedures. Restriction analysis of pAH316 demonstrated that this derivative contains a C. trachomatis L2 insert of-4.5 Kbp which consists of two EcoRI fragments of-2.5 Kbp and -2.0 Kbp in size. Southern hybridization analysis using the -0.2 Kbp E/H fragment as a probe localized this sequence to the -2.5 Kbp EcoRI fragment of pAH316. Directional PCR analyses employing purified pAH316 plasmid DNA as a template and amplification primer sets specific for -0.2 Kbp E/H fragment and T3 and T7 vector sequences demonsfrated pAH316 encodes the C-terminal segment of the HMW protein gene. The coding segment of the HMW protein is represented by nucleotides 1974 to 3420 in FIG. 2, and is listed asSEQIDNO.:ll.<br>
8.6.     EXAMPLE  6:  PRODUCTION OF TRUNCATED HMW RECOMBINANT PROTEIN<br>
The N-terminal half of the HMW protein was PCR cloned as a -1.5 Kbp fragment into the commercially available E. coli expression plasmid pTrcHisB (Invitrogen). The forward primer used in these reactions was designated 140FXHO (57-mer), listed as SEQ ID No. 18, and contains sequences complementary to the first 10 N-terminal residues of the mature HMW protein. In addition to the HMW proteincoding sequences, this forward primer also carries a unique Xhol restriction site located optimally located upstream of the first residue of the mature<br><br>
HMW protein (Glu/E) for proper fusion to the (His)6 affinity purification domain encoded on the<br>
vector plasmid, and 5' terminal 6 base G/C clamp for effective amplification and a 12 base<br>
internal spacer for effective endonuclease recognition and digestion.<br>
SEP ID No. 18	5'-AAG-GGC-CCA-ATT-ACG-CAG-AGC-TCG-<br>
AGA-GAA-ATT-ATG-GTT-CCT-CAA-GGA-ATT-TAC -GAT-3'<br>
SEP ID No. 19	5'-CGC-TCT-AGA-ACT-AGT-GGA-TC-3'<br>
The commercially available reverse sequencing primer SK (20mer, StrataGene), SEP ID No. 19. which is complementary to phagemid sequences downstream of the EcoRI site in pAH306, was used as the reverse amplification primer in these reactions. To obtain acceptable yields of the HMW protein PRE product (-1.5 Kbp), PCR amplification was performed using a mixture of thermostable DNA polymerases consisting of T. thermophilus DNA polymerase (Advantage Polymerase), as the primary amplification polymerase and a minor amount of a second high fidelity thermostable DNA polymerase to provide additional 5'-3' proofreading activity (CloneTech). An anti-Tth DNA polymerase antibody was added to the reaction mixture to provide automatic "hot-start" conditions which foster the production of large &gt;2 Kbp amplimers. pAH306 plasmid DNA purified using a commercially available alkaline/SDS system (QIAGEN) and silica gel spin columns (QIAGEN) was used to program these amplification reactions (-0.2 ng/reaction).<br>
The -1.5 Kbp amplimer was purified from unincorporated primers using silica gel spin columns and digested to completion using an excess of Xhol and EcoRI (-10 units per 1 ^g DNA). The purified and digested N-terminal truncated HMW protein PRE was then be cloned into the commercially available expression plasmid pTrcHisB that had been previously digested with both Xhol and EcoRI (5:1, insert.-vector ratio). Aliquots from the ligation reaction were then be used to electrotransform a suitable E. coli host {e.g. TPPIO).<br>
Mini-prep DNA fi"om ampicillin-resistant transformants picked at random were prepared, digested to completion with Xhol, EcoRI, or a combination of both and examined for the presence and orientation of the -1.5 Kbp truncated HMW protein PRE insert by agarose gel electrophoresis. Mini-prep DNA from clones determined to carry the -1.5 Kbp XhoI/EcoRI insert was prepared and used to program asymmetric PCR DNA sequencing reactions to confirm the fidelity of the junction formed between the HMW protein fi-agment and the (His)6 affinity purification domain of the expression vector. Plasmid pJJ36-J was one recombinant derivative isolated by these procedures and is represented by nucleotides 466 to 1977 in FIG. 2. The<br><br>
deduced amino acid sequence of the truncated fragment of HMW protein is represented by amino acids 29 to 532 in FIG. 3 and is listed as SEQ ID NO.: 17.<br>
8.7.     EXAMPLE 7; DETERMINATION OF PRESENCE IN OTHER SPECIES<br>
Polymerase chain reaction analyses were undertaken to establish the presence of the HMW gene in several clinically recognized C. trachomatis strains and as well as other chlamydial species, e.g., C. pneumoniae. Chlamydia trachomatis strains as frozen stocks from the ATCC (Rockville, Md.) were used to infect subconfluent monolayers (about 80%) of McCoy cells according to standard procedures. Infected monolayers were either centrifuged in a Sorvall RT6000B centrifuge (~l,300 rpm, 25°C, 30 min) and/or treated with dextran sulfate (-50 ^g/ml) at the time of infection to enhance initial attachment of the low infectivity biovars (non-LGV) to host cells and thus increase the final EB yield. Roughly 48 hours later, infected monolayers were collected by scraping and host cells disrupted by sonication to release elementary bodies (EBs). Total DNA was extracted from purified EBs (-10^ -10*) of each strain using the proteinase K/Nonidet P40 method described by Denamur, et al, J. Gen. Microbiol. 137:2525-2530 (1991), incorporated herein by reference, and further purified by phenol/chloroform extraction and salt precipitation. Purified Chlamydia pneumoniae (AR-139) genomic DNA was purchased from Advanced Biotechnologies Inc.<br>
To determine the presence of the HMW protein gene in these strains, amplification reactions were programmed using total Chlamydia DNA as template and the HMW protein segment-specific oligonucleotide primer (21mers) sets listed below.<br>
SEOIDNo.20	5'-ATG-GTT-CCT-CAA-GGA-ATT-TAC-G-3'<br>
SEOIDNo.21	5'-GGT-CCC-CCA-TCA-GCG-GGA-G-3'<br>
Briefly, standard PCR amplification reactions (2 mM Mg^"^, 100 ixmol dNTPs, 0.75 units AmpliTaq polymerase, 50 \i\ final volume) were programmed using approximately 15 p,l of the crude C. trachomatis DNA extracts (-10 |il of the commercially available C. pneumoniae DNA) and -20 pmol of each forward and reverse HMW protein-specific amplification primers of SEQ ID No. 20 and 21. Amplification of small target sequences (
 <br>
matched the size of the desired PCR product plus 2 min {i.e. desired PCR product=6 Kbp, extension time=8 min).<br>
Both conventional and long-distance PCRs were carried out using 0.2 ml thin-walled polypropylene microcentrifuge tubes in an ABI 2400 Thermal Cycler (Perkin-Elmer). Following thermal cycling, aliquots (-20 |j.l) of the reactions were analyzed and PCR products identified by standard agarose gel electrophoresis (0.8% agarose in TAE buffer) and ethidium bromide staining. The resuhs showed that the HMW protein is highly conserved in clinically relevant serovars; the HMW gene was present in all Chlamydia samples strains tested, including serovars B, Ba, D, E, F, G, H, I, J, K, Li, L2 and MoPn and in C. pneumoniae.<br>
8.8.      EXAMPLE 8; DETERMINATION OF SEQUENCE VARIATION<br>
To establish the degree of DNA and amino acid sequence variation among different Chlamydia<br>
strains, the gene for the HMW protein was PCR-cloned from both a C. trachomatis B serovar<br>
(representing the trachoma group of organisms)  and  from  a C.   trachomatis F serovar<br>
(representing the low infectivity STD biovars) and compared to the HMW protein consensus C.<br>
trachomatis L2 sequence.<br>
Briefly, LD-PCR was used to generate -6 Kbp HMW protein-specific DNA fragments from C.<br>
trachomatis B and F genomic DNA that contain the complete coding sequence for the mature<br>
HMW protein. Amplification conditions for these LD-PCR exercises were as described in<br>
Example 6. The reverse amplification primer employed in these reactions (p316Kpn-RC, 56mer),<br>
listed as SEQ ID No. 13, is complementary to a sequence located ~3 Kbp downstream of the<br>
predicted HMW protein termination codon. As an aide to cloning the desired ~6 Kbp amplimer, a<br>
single Kpnl restriction endonuclease site 5' to the chlamydial sequence was engineered into the<br>
p316Kpn-RC primer. The forward amplification primer used for these reacfions (p306Kpn-F,<br>
56mer), listed as SEQ ID No. 12, contains the sequence complementary to the first 10 amino acid<br>
residues (30 nucleotides) specifying the mature HMW protein as well as a 5' sequence specifying<br>
a Kpnl site. p306Kpn-F was designed such that the sequence encoding the N-terminus of the<br>
mature HMW protein could be linked in-frame to a hexa-His affinity domain encoded<br>
downstream of the highly efficient trc promoter on the E. coli expression vector pTrcHisB<br>
(ClonTech) when the -6 Kbp amplimer was inserted into the Kpnl site of this vector.<br>
SEOIDN0.I2	5'-AAG-GGC-CCA-ATT-ACG-CAG-AGG-GTA-<br>
CCG-AAA-TTA-TGG-TTC-CTC-AAG-GAA-TTT-ACG -AT-3'<br><br>
SEP ID No. 13	5'-AAG-GGC-CCA-ATT-ACG-CAG-AGG-GTA-<br>
CCC-TAA-GAA-GAA-GGC-ATG-CCG-TGC-TAG-CGG -AG-3'<br>
The ~6 Kbp HMW protein products were purified using silica-gel spin columns (QIAGEN) and the fragments subjected to two 8-10 hour cycles of Kpnl digestion using a 10-fold excess of Kpnl (-10 units per 1 ^g of purified fragment, 37°C). Following the second digestion, residual restriction enzyme activity was removed using QIAGEN spin columns and the ~6 Kbp Kpnl HMW protein fragments cloned into the pTrcHisB plasmid which had been previously digested to completion with Kpnl and treated with calf intestinal phosphatase to prevent vector religation. Vector/insert ligations were performed in a -50 ^^1 final reaction volume (50 mM Tris-HCl, pH 7.00; 10 mM NaCl; 1 mM ATP; 0.5 mM DTT) at 25°C for -2 hours using T4 DNA ligase (-10 units/reaction) and a vector:insert molar ratio of approximately 1:5. Following ligation, aliquots (-50 ng ligated DNA) was used to electroporate a competent E. coli host, e.g. E. coli TOP 10. Plasmid-harboring transformants were selected by plating elecfrotransformed cells onto LB agar containing 100 (^g/ml ampicillin. Ampicillin-resistant (Ap*^) transformants appearing after a -18-24 hour incubation period at 37°C were picked at random and restreaked onto the same selective media for purification.<br>
A single, purified Ap*^ colony from each initial transformant was used to inoculate -5 ml of LB broth and grown overnight at 37°C in a incubator shaker with mild aeration (-200 rpm). Cells from broth cultures were harvested by centriftigation and used to prepare small quantities of plasmid DNA. Commercially available reagents (QIAGEN Plasmid Mini Kits) were employed for these plasmid extractions. Plasmid derivatives carrying inserts were presumptively identified by electrophoresing the non-digested plasmid DNA in agarose gels (0.8% agarose in TAE buffer) and identifying derivatives greater in size than vector plasmid. Insert-containing derivatives were confirmed and the orientation of the HMW protein inserts relative to vector sequences were determined using appropriate restriction endonucleases (Kpnl, EcoRI, Hindlll, BamHI, etc.), either separately or together in various combinations.<br>
The DNA sequence of the C. trachomatis B and F HMW protein genes were obtained for both strands using "sequence walking" the asymmetric dye-terminator PCR cycle sequencing methodology (ABI Prism Dye-Terminator Cycle Sequencing, Perkin-Elmer) described in Example 4. Reactions were programmed with plasmid mini-prep DNA and individual HMW protein sequence-specific primers that were employed in the sequencing of the HMW protein gene from the L2 type strain.<br><br>
DNA sequence data were collected using the ABI 310 Sequenator and analyzed automatically on a PowerMAC computer and appropriate computer software as described in Example 4. Individually autoanalyzed DNA sequences were edited manually for accuracy before being merged into a consensus sequence "string" using AutoAssembler software (Perkin-Elmer). Both strands of the HMW protein gene from the C. trachomatis B and F seovars were sequenced and these data compiled to create composite consensus sequences for both the C. trachomatis B and F HMW protein genes. These sequences are listed as SEQ ID NOs.: 14 and 15. Sequence comparisons of the L2, F and B strains are presented in Figure 6.<br>
8.9.      EXAMPLE 9; PRODUCTION OF RECOMBINANT PROTEIN<br>
To produce sufficient quantities of recombinant HMW protein for both immunogenicity and animal protection studies, the HMW gene has been PCR cloned into suitable E. coli and baculovirus expression systems. Large quantities of rHMW protein are produced in an E. coli-based system as a chimeric fusion protein containing an N-terminal (Hls)6 affinity purification domain. The complete HMW protein open reading frame (ORF) was PCR-cloned from the C. trachomatis L2 genome as a single Kpnl fragment and fused in the proper orientation and in the correct reading frame to the (His)6 affinity purification domain encoded on the high expression plasmid vector pTrcHisB (CloneTech) as described in Example 5.<br>
The (His)6 affinity purification domain is part of a high expression locus consisting of the highly efficient tac promoter (IPTG-inducible) and consensus Shine and Delgamo ribosome binding site (RBS) located immediately upstream of the (His)6 affinity purification domain. The HMW protein genes from C trachomatis LGV L2, C. trachomatis B, and C. trachomatis F were PCR cloned as ~3.0 Kbp fi-agments. The forward primer (56-mer) used in these reactions was designated p306Kpn-F and contains sequences complementary to the first 10 N-terminal amino acid residues of the mature HMW protein, listed as SEQ ID No 12. In addition to the HMW protein coding sequences, this forward primer also carries a unique Kpnl restriction site located optimally located upstream of the first residue of the mature HMW protein (Glu) for proper fusion to the (His)6 affinity purification domain encoded on the vector plasmid, and 5' terminal 6 base G/C clamp for effective amplification and a 12 base internal spacer for effective endonuclease recognition/digestion. The reverse PCR primer, designated p316Kpn-3RC, contains a reverse complement sequence to a C. trachomatis sequence located -0.2 Kbp downstream of the HMW protein termination codon, listed as SEQ ID No. 14. As with p306Kpn-<br><br>
F, the reverse primer also contains a Kpnl restriction site 5' to the C. trachomatis sequences, a 6 base G/C clamp, and a 12 base internal spacer.<br>
To obtain acceptacle yields of the HMW protein ORF product (about 3,500 bp), PCR amplification was performed using a mixture of thermostable DNA polymerases consisting of T. thermophilus DNA polymerase as the primary amplification polymerase and a minor amount of a second high fidelity thermostable DNA polymerase to provide additional 5'-3' proofreading activity (Advantage Polymerase, CloneTech). An anti-Tth DNA polymerase antibody was added to the reaction mixture to provide automatic 'hot-start' conditions which foster the production of large (&gt;2 Kbp) amplimers.<br>
Genomic DNA from the various C. trachomatis strains was isolated from EBs as described in the example above and used to program these reactions. Following amplification, the desired reaction products were purified from excess primers using commercially available silica-gel spin columns and reagents (QIAGEN) and digested to completion with an excess of Kpnl (-10 units per 1 ng DNA). The purified and digested Kpnl HMW protein ORP was then be cloned into the Kpnl predigested pTrcHisB expression plasmid (5:1, insert:vector ratio). Aliquots from the ligation reaction were then used to electrotransform a suitable E. coli host (e.g. TOP 10). Mini-prep DNA from ampicillin-resistant transformants picked at random were prepared, digested to completion with KnI, Hindlll, or a combination of both and examined for the presence and orientation of the -3.2 Kbp HMW protein ORF insert by agarose gel electrophoresis and ethidium bromide staining. Mini-prep DNA was used to program asymmetric PCR DNA sequencing reactions as described in example(s) above to confirm the fidelity of the junction formed between the HMW protein fragment and the (His)6 affinity purification domain of the vector. Plasmid pAH342 was one derivative isolated by these procedures, which contains the HMW protein gene ORF from C. trachomatis La and is represented by nucleotides 446 to 3421 in Figure 2.<br>
Recombinants were grown in 2X-YT broth containing 100 |ig/ml Ap to mid-log phase (-0.5 O.D.600) and induced with IPTG (1 mM final) for an additional 4-5 hours to activate transcription from the vectors trc promoter. Cells were harvested by centrifugation and crude cell lysates prepared by lysis using a French pressure cell.<br>
Alternatively, expression of rHMW protein may be obtained by using a baculovirus expression system. Here, the HMW protein ORF from C. trachomatis L2 and C. trachomatis F were PCR-cloned as -3 Kbp PCR products into a baculovirus transfer vector {e.g. pFastBacHTb) that had been previously digested to completion with Kpnl and treated with CIP to minimize vector<br><br>
religation in essentially the same manner as described for pTrcHisB. The HMW protein expression cartridge generated in this cloning exercise {i.e. the baculovirus polyhedron promoter, N-terminal (His)6 affinity purification domain, HMW protein gene ORF) was then transferred to the baculovirus genome by site-specific transposition using a commercially available bacmid system (Bac-to-Bac, Gibco)<br>
Briefly, the HMW protein baculovirus expression plasmid was used to transform competent E. coli DHlObac (Gibco) cells containing a bacmid (a hybrid baculovirus-plasmid replicon) to gentamicin resistance using standard transformation and selection methodologies. Transformants where the HMW protein expression cartridge had successfully transposed from the expression plasmid to the appropriate receptor site within the lacZ gene located on the bacmid replicon were identified using a standard IPTG/X-gal blue-white selection.<br>
White, Gm"^ transformants were picked at random and restreaked for purification. Bacmid DNA was prepared from broth cultures by the method of Ish-Horowitz, N. A. R. 9:2989-2993 (1981) incorporated herein by reference, and is used to transfect Spodoptera frugiperda 9 cells. Following plaque purification, recombinant HMW protein baculovirus is used to infect large scale Spodoptera suspension cultures. A yeast expression system is used to generate a glycosylated form of HMW protein.<br>
8.10.    EXAMPLE 10; PURIFICATION OF RECOMBINANT PROTEIN<br>
Recombinant HMW protein was purified to homogeneity using standard preparative immobilized metal affinity chromatography (IMAC) procedures. Briefly, an E. coli strain harboring an expression plasmid containing HMW protein gene was grown in Luria broth in a 51 fermenter (New Brunswick) at 37°C with moderate aeration until mid-log phase (-0.5 O.D.600) and induced with IPTG (1 mM final) for 4-5 hours. Cell paste was collected, washed in PBS and stored at -20°C. Aliquots of frozen cell paste (-9-10 grams wet weight) were suspended in -120 ml of D-PBS by mechanical agitation and lysed by passage through a French pressure cell (2X, 14,000 psi, 4°C). Soluble protein was then removed from rHMW protein inclusion bodies by high speed centrifugation (-10,000Xg, 4°C, 30 min).<br>
The insoluble pellet containing rHMW protein was suspended in -20 ml of ice cold D-PBS by homogenization and centriftjged as above. Washed rHMW protein inclusion bodies were then denatured by suspension in a sodium phosphate buffer (0.1 M, pH 7.0) containing 6M guanidine hydrochloride and loaded onto a Ni^"^ -affinity column (1.5 cm X 25 cm, bed volume -15 ml) prepared from Fast-Flow Chelating Sepharose (Pharmacia) and charged with Ni^^ or Zn^"^ ions by<br><br>
standard procedures. Unbound material was removed by washing the column with -5-10 column volumes of a sodium phosphate buffer (0.1 M, pH 7.0) containing 8M urea. Recombinant HMW protein bound to the affinity resin by virtue of the N-terminal (His)6 affinity purification domain was eluted using a pH 4.0 sodium phosphate/8M urea buffer (-20 ml). Eluted material was neutralized by the addition of 1.0 M Tris-HCl (-2.5 ml, pH 7.5) and dialyzed against TE buffer containing SDS (0.5%) to remove the urea. Dialyzed material was concentrated using a Centricon-30 centrifugal concentrator (Amicon, 30,000 MWCO) and mixed with a 1/5 volume of 5X SDS gel sample buffer containing 1 mM 2-mercaptoethanol (Lammeli) and boiled at 100°C for 5 minutes.<br>
Samples were loaded onto Tris/glycine preparative acrylamide gels (4% stacking gel, 12% resolving gel, 30:0.8 acrylamiderbis solution, 3 mm thickness). A prestained molecular weight standard (SeeBlue, Novex) was run in parallel with the rHMW protein samples to identify size fractions on the gel. The area of the gel containing proteins having molecular masses of-50-70 Kdal was excised and the proteins electroeluted using an Elu-Trap device and membranes (S&amp;S) as specified by the manufacturer. Electroeluted protein was dialyized to remove SDS. The protein concentration of the sample was determined using a Micro-BCA system (Pierce) and BSA as a concentration standard. The purity of rHMW protein was determined using conventional SDS-PAGE and commercially available silver staining reagents (Silver Stain Plus, Novex) as shown in Figure 4.<br>
The apparent molecular weight of the isolated mature rHMW is about 105-115 kDa as determined by SDS-PAGE.<br>
8.11.    EXAMPLE 11: PREPARATION OF ANTIBODIES TO HMW PROTEIN<br>
Polyvalent antisera directed against the HMW protein were generated by vaccinating rabbits with the purified HMW protein or fragments thereof Each animal was given a total of three immunizations of about 250 ^g HMW protein or fragment thereof per injection (beginning with complete Freund's adjuvant and followed with incomplete Freund's adjuvant) at approximately 21 day intervals. At each immunization, approximately half of the material was administered intramuscullarly (i.m.) and half was injected intranodally. Fourteen days after the third vaccination a fourth booster of about 100 ^g HMW protein was given i.m. and the animals exsanguinated 7-10 days later. Anti-HMW protein titers were measured by ELISA using purified HMW protein (1.0 ^g/well) or C. trachomatis Lz EBs (whole and crude protein extracts) as<br><br>
capture ligands. Immunogen specific IgG ELISA litres of 1/320,000 were observed using purified rHMW truncated protein and 1/2500 using either EBs or RBs.<br>
Serial dilutions of antisera were made in PBS and tested by ELISA in duplicate. Goat HRP-conjugated anti-rabbit antibody diluted 1/1000 was used as the second reporter antibody in these assays. Titers are expressed as the greatest dilution showing a positive ELISA reaction, i.e. an O.D.450 value &gt;2SD above the mean negative control value (prebleed rabbit sera). Hyperimmune antisera was then used to probe Western blots of crude EB or RB extracts as well as 1.0% OOP EB extract preparations to determine whether other C. trachomatis serovars and Chlamydia species express the HMW protein. C. trachomatis serovars B, Ba, D, F, G, I, J, K, Li, L2, L3, MoPn and Chlamydia pneumoniae were tested and found to have a protein of an apparent molecular weight of 105-115 KDa reactive with antisera generated against HMW protein.<br>
8.12.   EXAMPLE 12; SURFACE LOCALIZATION<br>
Surface localization of the HMW protein on different Chlamydia strains and derivatives were examined by indirect fluorescence antibody (IFA). IFA was performed using the procedures generally known in the art using hyperimmune anti-HMW protein as the primary antibody. Hak cells infected with whole EBs from one of C. trachomatis serovars L2, B, and F, C. pneumoniae or C psittaci are achieved by the following method.<br>
McCoy or Hak cells were grown to confluence in D-MEM media on 12 mm plain coverslips inside 24 well tissue culture plates then centrifugally inoculated with ~5X10' inclusion forming units (IFU) of the C. trachomatis strain Nl 1 (serover F). After -24 hours incubation, the culture media was removed and infected cells fixed inmethanol for 10 min. The fixed monoloayer was then washed with PBS (IX) to remove fixative and overlayer with &gt;300 ^1 of anti-60Kdal truncated HMWP rabbit antibody that had been diluted -1/100 in PBS. After 1 hour incubation with the primary antibody, the cells were washed gently with PBS then incubated for -30 min. with a 1/200 dilution of a mouse anti-rabbit IgG antibody conjugated with FITC. The second antibody was diluted using a PBS solution containing 0.0091% Evans Blue as a counter stain to visualize the monolayer. Cells were washed 2X in PBS to remove the secondary antibody, the coverslips removed from the culture plates, and mounted onto microscope slides using a fluorescent mounting medium.<br>
Identical cell samples stained with prebleed rabbit antibody or FITC-conjugated second antibody alone were processed in parallel and served as antibody specificity (negative) controls. Counterstained samples were examined at a 1000-X magnification with a Zeiss Axioskop<br><br>
photomicroscope equipped with plan-neoflur objectives. Results using C. trachomatis Nil (F serovar) are shown in FIG. 7. The results show that enhanced fluorescence of samples stained with HMW protein antibody compared to the controls confirmed the surface location of the HMW protein. Furthermore, fluorescence of samples stained with HMW protein antibodies show binding to surface localized HMW protein from L2, B and MoPn serovars and C. pneuomoniae.<br>
9.        EXAMPLE; IN VITRO NEUTRALIZATION MODEL<br>
The in vitro neutralization model has been used to show that protective antiserum inhibited chlamydial infection (neutralization) of various tissue culture cell lines. Animal models are also essential for testing vaccine efficacy with both small animal (non-primate) and primate models necessary for preclinical evaluation. The guinea-pig has been used for studying experimental ocular and genital infection by the Guinea-pig inclusion conjunctivitis agent (GPIC), C. psittaci. The mouse offers a consistent and reproducible model of genital tract infection using human genital tract isolates. This mouse model is a generally accepted pre-clinical assay, and was used to evaluate MOMP as a subunit vaccine. Another model is known as the primate model of trachoma infection wherein the induction of secretory IgA was shown to be a prime component of protection. Vaccinogenic ability of new subunit antigen candidates is determined using the above-mentioned generally accepted in vitro neutralization and animal model systems. As a preliminary exercise to the animal protection studies, hyperimmune anti-HMW antibody was evaluated for its ability to block the infectivity of various C. trachomatis serovars (e.g. L2, B, F) in vitro. Although McCoy cells were used to propagate Chlamydia, these cells also allow antibody-mediated uptake via Fc receptors. Therefore, to evaluate anti-HMW antibody inhibition of infectivity, Hak cells, which do not display Fc receptors, were used in these analyses. Cells were grown on coverslips in 24-well plates to a subconfluent monolayer (about 90% confluency=lX10^ cells/ml) at 37°C in 5% CO2. Anti-HMW-antibody was diluted to about 100 ^g/ml (total protein) in sucrose-phosphate-glutamate (SPG) buffer and then serially diluted in SPG buffer. Frozen aliquots of pretitered Chlamydia was diluted in SPG buffer to about 2X10'* IFU/ml. EBs were premixed with the diluted anti-HMW-antibody to about 10-20 IFU/^l and incubated 30 minutes at 37°C on a rocking platform.<br>
Prepared Hak cells were washed in HBSS and then incubated with the anti-HMW-antibody/Chlamydia EB mixture in triplicate for each antibody using 500 IFU/ml. Plates were incubated for 2 hours at 37'C, then the inoculum removed and plates washed 3 times with HBSS. Tissue culture media containing 1 |xg/ml of cyclohexamide was added and plates incubated for<br><br>
about 24-36 hours at 37°C in 5% CO2 to allow inclusion bodies to develop. After incubation, the media was removed and cell monolayers washed 3X in PBS. Plates were fixed in methanol for 20 minutes and re-washed in PBS.<br>
Cells were stained to visualize inclusions by incubating with mti-Chlamydia LPS antibody (diluted about 1:500, ViroStat), cells washed 3 times in PBS, followed by incubation with FITC-conjugated goat secondary antibody for 30 minutes at 37°C. Coverslips were washed, air dried, and mounted in glycerol on glass coverslips. Inclusions were counted in five fields through the midline of the coverslip on a Zeiss fluorescence photomicroscope. Results are reported as the percent reduction of inclusion-containing cells with respect to a heterogenous antibody control (rabbit prebleed sera).<br>
10.      EXAMPLE:  VACCINE  EFFICACY  (MOUSE MODEL  OF  SALPINGITIS AND FERTILITY)<br>
10.1.    METHODS<br>
10.1.1 IMMUNIZATION AND CHALLENGE<br>
The Tuffrey murine infertility model was employed to evaluate the efficacy of rHMWP to protect animals against Chlamydia trachomatis-'mduced salpingitis and infertility. Three groups of 17 female C3H HeOuJ mice (~6 weeks of age, Jackson Labs) were employed for this evaluation. The test group of 17 animals was immunized at weeks 0, 2, and 3 by intranasal administration of -20 \i\ of a vaccine formulation containing approximately 10-12 ^g of gel-purified rHMWP and ~5 \ig mLT (SmithKline Beecham) as adjuvant. Prior to immunization mice were sedated using an anesthesia cocktail consisting of 16% Ketaject and 16% Xylaject in 68% pyrogen-free PBS (100 ^l i.p./animal). Sedated animals were placed on their backs and using a standard laboratory pipette administered the vaccine formulation; -10 ^1 of the vaccine solution per nostril with a 5-10 minute wait period between applications.<br>
Two groups of 17 female mice (per test group) were immunized similarly but with a preparation containing only 5 ^g mLT (i.e. adjuvant only, no antigen). One of these groups was subsequently challenged with C. trachomatis (shamimmunized, infected) and served as the negative fertility control while the other group was not challenged (sham immunized, sham Infected) and served as the positive fertility control.<br>
At week 4, all animals were administered a single i.p. dose of progesterone (2.5 mg in pyrogen-free PBS, Depo-Provera, Upjohn) to stabilize the uterine epithelium. At week 5, animals immunized with rHMWP and animals in the negative control eroun were, infected by bilateral<br><br>
intrauterine inoculation with -5X10^ inclusion forming units (IFU) of C. trachomatis Nil (serovar F) in 100 jtl of sucrose phosphate glutamate buffer (SPG). To mimic the manipulations to the reproductive tract experienced by the other two groups, animals in the positive control were bilaterally inoculated with 100 ^,1 of a McCoy cell extract that contained no C. trachomatis. At week 7, 5-7 animals from each group were sacrificed by CO2 asphyxiation and the complete genital tract (both upper and lower reproductive tracts) removed for histopathological analysis. At week 9, the remaining females from each group were caged with 8-10 week old male C3H mice for a 2 month breeding period to assess fertility (1 male for every 2 females per cage with weekly rotation of the males within each group, animals from different experimental groups were not mixed). Palpation and periodic weighing were used to determine when animals in each pair became pregnant. The parameters used to estimate group fertility were: F, the number of mice which littered at least once during the mating period divided by the total number of mice in that study group; M, the number of newborn mice (bom dead or alive) divided by the number of litters produced in that group during the mating period; and N, the number of newborn mice (bom dead or alive) divided by the total number of mice in that group.<br>
10.1.2.	HISTOPATHOLOGY<br>
Genital tracts were treated for &gt;24 hrs in Bouin's fixative, progressively dehydrated in 50%, 70%, and 100%) methanol, soaked in toluol, and either paraffin embedded or directly embedded in OCT compound (Tissue-TEK, Miles) and subsequently snap frozen in liquid nitrogen. Tissue sections (~6 pm) were stained with hematoxylin and eosin (after deparaffinization of the Bouin fixed samples). Inflammatory changes in the oviducts and ovaries were graded as follows: 0, no apparent inflammatory reaction; 1, a few mononuclear cells infiltrating the periovarial space or the submucosa of the oviduct; 2, same as 1 but to a greater extent; 3, same as 2 but with a thickened oviductal submucosa and the presence of inflammatory cells in the oviductal lumen; 4, same as 3 but to a greater extent. Inflammation in the cervix/vagina was scored based on the level of the intraepithelial infiltrate observed.<br>
10.1.3.	DETERMINATION      OF      rHMWP-SPECIFIC      HUMORAL<br>
RESPONSES<br>
Blood samples were collected periodically during the immunization and challenge phases by retroorbital bleeding and serum prepared by centrifugation. Vaginal secretions were collected by repeated injection of 50 ^l of sterile PBS into the vagina with a standard laboratory pipettor and<br><br>
immediately withdrawing the solution. Two-to-three injection/withdrawal cycles were performed.<br>
Quantitation of antibody (Ab) responses by ELISA were performed as described in Section 8.11. Microwell ELISA plates (Maxisorb, NUNC) for determining Ab levels were coated overnight at 4°C with -0.5-1.0 ng of gel-purified rHMWP per well in 10 mM carbonate/bicarbonate buffer (pH 9.6), washed with PBS containing 0.1% Tween-20 (washing buffer) and blocked for ~1 hr at 37°C with a PBS solution containing 3% BSA. For the determination of antigen-specific serum IgG levels, test sera were serially diluted in washing buffer containing 0.5% BSA and aliquots (100 |i,l) incubated in the antigen-coated wells for ~2 hr at 37°C. The plates were then washed and incubated for -1 hr at 37'C with a horseradish peroxidase (HRP)-conjugated goat anti-mouse IgG second antibody (Sigma). A HRP-conjugated goat anti-mouse IgA secondary antibody was used to detect the presence of rHMWP-specific IgA in vaginal secretions. After incubation with the appropriate secondary Ab, the plates were washed and incubated for ~20-30 minutes at room temperature with TMB substrate (Sigma). Reactions were stopped by the addition of 2M H2 SO4 and the absorbance determined at 450 nm on a Molecular Devices SpectroMax microplate reader. Titers were determined as the reciprocal of the sample dilution corresponding to an optical density of 1.0 at 450 nm.<br>
10.1.4.	DETERMINATION OF rHMWP-SPECIFIC CELLULAR<br>
RESPONSES<br>
Groups of 6 female C3H HeOuJ mice (Jackson Labs) were sedated and immunized at weeks 0, 2 and 3 by intranasal administration with the rHMWP+mLT vaccine as described in Section 10.1.3. At weeks 4 and 5 immediately prior to progesterone treatment and intrauterine challenge, respectively, 3 animals from each group were sacrificed by CO2 asphyxiation and spleens aseptically removed and single cell suspensions prepared using conventional methodologies. Spleen cells from immunized animals were analyzed separately. For both the positive control group (sham immunized and sham infected) and the negative control group (sham immunized, infected) spleen cells were pooled and tested for restimulation.<br>
For the measurement of spleen cell proliferation, spleens were ground (5 to 10 rounds) in 5 ml of RPMI 1640 Glutamax I supplemented with 10% fetal calf serum, 25 mM HEPES, 50 U/ml penicillin, 50 ^ig/ml streptomycin, 1 mm sodium pyruvate, nonessential amino acids, and 50 pM 2-mercaptoethanol (Gibco-BRL). Live cells were counted by Trypan Blue staining and diluted in the same media to reach a density of 1.0-2.0X10^ cells/ml (Falcon 2063 polypropylene tubes).<br><br>
Triplicate cultures were set-up in round bottom 96-well culture plates (Nunclon, Nunc) using -5X10^ responder cells per well in 200 ^l of media Cells were stimulated with either 1.0 ^g/ml of rHMWP (antigen-specific proliferation) or with 4 |xg/ml concanavalin A (Boerhinger Mannheim) as a positive stimulation control; unrestimulated cell cultures were used as a negative control of cellular activation. After 72-96 hours of incubation at 37°C in 5% CO2 cells were pulsed labelled for -18 hrs with 1.0 fiCi ^H-thymidine (Amersham) per well. Pulsed cells were harvested onto glass-fiber sheets using a Tomtec Cell Harvester (Mk III) and counted for beta-emission in a 3-channel Wallac 1450 Trilux Liquid Scintillation Counter. The stimulation index (SI) for a sample (individual or pooled) was defined as the mean of the antigen or ConA-stimulated T-cell uptake of ^H-thymidine for triplicate wells divided by the mean of the unstimulated uptake for triplicate wells. Sis for both antigen-specific (rHMWP-specific) and ConA-specific proliferation were determined.<br>
10.2.   Results<br>
10.2.1	EFFECT ON MOUSE FERTILITY AFTER A HETEROTYPIC<br>
CHALLENGE<br>
Evidence that mucosal immunization with rHMWP combined with mLT can afford protection against infertility caused by a human clinical isolate of C. trachomatis (strain Nil, serovar F) is shown in Table 1. Animals immunized with the rHMWP displayed a significantly higher fertility rate (70%, i.e. number of fertile females in group/total number of animals in the group) than animals in the negative control group (30%, sham immunized and infected). Similarly, the rHMWP immunized group produced more offspring and exhibited a group fecundity greater than those observed in the negative control group (51 ys 24 offspring and 5.1±4.7 ys 2.4±4.6 fecundity scores, respectively). As a group, animals immunized with the rHMWP vaccine displayed a comparable fertility rate, total number of offspring, and a fecundity score to those observed in the sham infected positive control group (80% fertility rate, 56 total offspring, 4.9±2.7 fecundity). The protection against C. trachomatis-induced infertility obtained in this experiment also demonstrates the utility of the rHMWP to afford cross-biovar and cross-serovar protection against C. trachomatis disease. The recombinant HMWP antigen employed in this experiment was cloned from a strain belonging to the C. trachomatis lymphogranuloma venereum (LGV) group (strain L2) which causes systemic as well as more localized mucosal infections of the eye and genital tract. The C. trachomatis challenge organism used in these experiments, strain Nil is<br><br><br>
10.2.2.	EFFECT ON CELLULAR IMMUNE RESPONSE<br>
The rHMWP-specific activation of the cellular immune system was demonstrated using a conventional spleen cell proliferation assay. When spleen cells were tested during week 4 (immediately prior to progesterone treatment) (Table 2) and week 5 (~7 days after hormone treatment but before intrauterine challenge) (Table 3), all samples collected from rHMWP-immunized animals developed a strong antigen-specific proliferative immune response. The antigen-specific Stimulation Indexes (Sis) obtained prior to progesterone treatment from rHMWP-immunized animals were equal to or greater than the Sis obtained via mitogenic stimulation with ConA (mean values for antigen and ConA stimulation obtained fi-om 3 rHMWP-immunized animals: 26.2 vs 18.4, respectively). Spleen cells obtained from either sham immunized animals or naive animals (i.e. animals that were not exposed to either the rHMWP antigen or mLT) did not respond to in vitro restimulation with the rHMWP material, thus establishing the specificity of the proliferative response observed in the immunized animals. Progesterone treatment did not affect the antigen-specific proliferative response observed in rHMWP immunized animals. Antigen-specific Sis obtained with spleen cells obtained after hormone treatment were greater than obtained via mitogenic stimulation (mean values for antigen<br><br>
and ConA stimulation obtained from 3 rHMWP-immunized animals: 92.4 vs 37.8, respectively). Again samples collected from sham immunized or naive animals failed to demonstrate any antigen-specific proliferative response.<br><br>
10.2.3.	EFFECT ON HUMORAL IMMUNE RESPONSE<br>
To demonstrate that immunization with the full length rHMWP produces a humoral immune response, IgG titers were measured by ELISA on sera collected at week 5 immediately prior to challenge (i.e. approximately 2 weeks after the third immunization). As shown in Table 4,<br><br>
immunization of C3H mice with three doses of-10-12 p.g rHMWP produced detectable levels of anti-rHMWP IgG in all animals. Vaginal secretions were also collected from these animals at the same time and tested for the presence of anti-rHMWP mucosal IgA. Antigen-specific vaginal IgA was detected in three animals (Table 4).<br><br>
11.       EXAMPLE; CONSTRUCTION OF PJJ701<br>
A plasmid containing the entire C. trachomatis L2 HMWP gene was constructed by selectively removing the EcoRI site upstream to the HMWP N-terminus in pAH306 (Described in Section 8.4). This was accomplished by digesting pAH306 to completion with Xhol and then religating the plasmid to create pAH306-XhoA-l. The -2.5 Kbp EcoRI fragment from pAH316 (described in Section 8.5) which contains the remaining HMWP C-terminus was isolated from preparative agarose gels that had been loaded with a complete EcoRI digestion of pAH316. An agarose gel slice containing the appropriate -2.5 Kbp fragment was then excised, dissolved with Nal buffer and purified from residual agarose by hydroxyapatite spin column chromatography (QiaGen).<br><br>
The purified C-terminal HMWP EcoRI fragment was then ligated into the single EcoRI site of pAH306-XhoA-l, located at the 3'-end of the HMWP N-terminal coding sequence, using T4 DNA ligase and standard molecular biology protocols. E. coli ToplO cells were transformed with an aliquot of the pAH306-XhoA-l (EcoRI digested and phosphatase treated vector) and the 2.5 Kbp EcoRI C-terminal fragment ligation reaction and recombinants selected on 2X-YT agar containing 100 ^g/ml ampicillin. Ampicillin-resistant transformants were picked at random, Plasmid DNA was isolated from individual Ap'* transformants using a QiaGen Mini-Prep Plasmid DNA Isolation System and screened for the presence of plasmids greater in size than pAH306-XhoA-l by conventional agarose gel electrophoresis and ethidium bromide staining. Derivatives of pAH306-XhoA-l carrying the 2.5 Kbp HMWP fragment in the proper orientation that would allow expression of the full length HMWP were identified by restriction analyses using EcoRI and/or Xhol. Plasmid pAH374 was one derivative isolated from this experiment. A PCR-based site-directed mutagenesis procedure (Quik-Change Site-Directed Mutagenesis System, Stratagene) was employed to effect a desired DNA change, namely to remove the Ndel site within the HMWP coding sequence of pAH374. Mutagenic PCR primers, 41 bases in length and complementary to the sequencing containing the Ndel site and designated 140-Nde-FX and 140-NdeRCX, were designed so as to eliminate the Ndel recognition site but not change the corresponding protein coding sequence. The sequences of the two PCR mutagenic primers employed to remove the Ndel site in pAH374 are given below.<br>
140-Nde-FX(SEQ ID NO: 38)<br>
5'-GGG TTT GGG AAT CAG CAC ATG AAA ACC TCA TAT ACA TTT GC-3'<br>
140-Nde-RCX(SEQ ID NO:39)<br>
5'-GCA AAT GTA TAT GAG GTT TTC ATG TGC TGA TTC CCA AAC CC-3'<br>
Following Pfti DNA polymerase (Stratagene) mutagenesis and Dpnl digestion, to cleave any unaltered pAH374 parental plasmid, mutated plasmid DNA was then transformed into E. coli XL 1-Blue. Plasmid harboring transformants were selected on 2X-YT agar containing 100 fig/ml ampicillin. Antibiotic resistant transformants were picked at random and screened for plasmids of the size as pAH374. The identity of plasmids isolated from transformants was determined by restriction enzyme digestion using EcoRI. The absence of the Ndel site in these plasmids was determined by digestion using Ndel. To verily the loss of the HMWP Ndel site and to ensure no<br><br>
unwanted DNA sequence changes had occurred in this region during the mutagenesis procedure, mutagenized plasmids were further subjected to DNA sequence analysis using a sequence-specific sequencing primer located upstream of the Ndel site. Plasmid pAH374-NdeA-l was one plasmid isolated fi"om this experiment.<br>
A DNA fragment encoding the C. trachomatis L2 HMWP without the internal Ndel site, plasmid pAH374-NdeA-l, was PCR amplified irom reactions programmed with plasmid pAH374-NdeA-1 (-50 ng) and primers 306-Nde-Metl and 312H6Xbal. Primer 306NdeMetl was designed to contain a central Ndel site for directed cloning onto pMG81. The Ndel site in 306NdeMetl overlapped the ATG start codon for HMWP signal sequence and was flanked by a 20 base G/C clamp on the 5' side and sequences complementary to the first 15 residues of the HMWP signal sequence on the 3' side. Primer 312H6Xbal was designed to contain sequences complementary to the C-terminus of the HMWP followed by a (CAT)6 motif specifying a hexa-histidine affinity purification domain. This primer also contained two UAA termination codons, an Xbal recognition sequence, and a 20 base G/C clamp at the 3' end of the primer. The sequences of the 306NdeMetl and 312H6Xbal PCR primers are given below.<br>
306NdeMetl (SEQ ID NO: 40)<br>
5'-AAG GGC CCA ATT ACG CAG ACA TAT GGA AAC GTC TTT CCA TAA GTT<br>
CTTTCTTTCA-3'<br>
312H6Xbal(SEQIDNO:41)<br>
5'-AAG GGC CCA ATT ACG CAG AGT CTA GAT TAT TAA TGA TGA TGA TGA<br>
TGA TGG AAC CGG ACT CTA CTT CCT GCA CTC AAA CC-3'<br>
PCR amplification conditions described in Section 8.4 were used to generate the Ndel-Xbal HMWP gene cassette. Following amplification, the PCR product was purified using hydroxyapatite spin columns (QiaGen) and digested overnight at 2&gt;TC with a ~10-fold excess of Ndel and Xbal to generate the required 'overhangs' at the ends of the fragment. The digested fi-agment was again purified using spin columns and -250 ng ligated to -50 ng pMGSl plasmid DNA that had been previously digested to completion with Ndel and Xbal and subsequently treated with CIP to prevent vector religation. An aliquot of the ligation reaction was used to transform E. coli strain AR58 which had been made competent by the method of Lederberg and Cohen. Transformants were selected on 2X-YT agar containing 40 ng/ml kanamycin sulfate. Due<br><br>
to the temperature inducible promoter on pMG81, the transformed cells were grown at 30°C. Kanamycin-resistant transformants were picked at random and screened for the presence of plasmids -3.0 Kbp larger in size than pMG81. Insert containing derivatives of pMG81 were confirmed by restriction enzyme analysis using Ndel, Xbal, EcoRI and Ncol. Plasmid pJJ701 was one plasmid isolated from this exercise.<br>
12.       EXAMPLE 16; PRODUCTION OF FULL LENGTH rHMWP FROM AR58 fPJJ70n<br>
One milliliter of a frozen stock of E. coli strain AR58 containing plasmid pJJ701 was used to inoculate -100 ml of 2X-YT broth containing 40 ^.g/ml kanamycin and grown overnight at 30°C to prepare a fermentor seed culture. Approximately 20 ml of the overnight seed culture was then used to inoculate a New Brunswick Bioflow 3000 fermentor loaded with -2.01 of 2X-YT broth containing 40 \ig/m\ kanamycin. The AR58 (pJJVOl) culture was grown at 30°C with vigorous aeration until an O.D.625 value of 0.5-0.6 was attained. Expression of rHMWP was induced by increasing the temperature of the fermentor culture to ~39°C to 42°C. Incubation at the elevated temperature was continued for approximately 4-5 hours.<br>
At the end of the induction period, the E. coli culture, with some cells displaying classic recombinant protein inclusion bodies, was harvested by continuous flow centrifugation using an Heraeus Contifiige 28RS centrifuge. Following centrifugation, cell mass was scraped from the centrifuge bowl and stored at -70°C until processed.<br>
Approximately 15 gm of the AR58 (pJJ701) frozen cell paste was resuspended by vortexing and trituration in -40 ml of ice cold 10 mM sodium phosphate buffer, pH7.3. Once suspended, lysozyme (Chicken egg white, Sigma) and DNase I (Bovine pancreas, Sigma) were added to final concentrations of 1.0 mg/ml and 0.01 mg/ml, respectively, and the mixture incubated on ice for 30-45 minutes. Cells were disrupted by 2 sequential passes through a pre-cooled (-4°C) SLM Aminco French Pressure Cell (-14 Kpsi, 1" diameter bore). The cell lysate was then centrifuged for 5 min at ~500xg (4°C) in a Sorvall SS34 rotor to remove unbroken cells. Insoluble material containing the rHMWP was isolated (pelleted) by centrifugation for 45 min at ~20,000xg (4°C) in a Sorvall SS34 rotor. The supernatant from this centrifugation was discarded and the insoluble fraction stored at -20°C in pellet form.<br>
To selectively extract contaminating proteins and remove endotoxin, the rHMWP-containing insoluble pellet was thawed on ice and washed twice with 10 ml of PBS buffer containing 2.0% Triton X-100. Washing was performed at room temperature and suspension of the gelatinous<br><br>
rHMWP-containing pellet was accomplished by vortexing and homogenization in a conventional glass tissue grinder. Insoluble material containing the rHMWP was recovered after washing by centrifiigation at ~10,000Xg for 20 minutes (room temperature) in a Sorvall SS34 rotor. Insoluble material was then washed (again by vortexing and homogenization) 2-times with 10 ml of a 4.0 M urea solution containing 2.0 M NaCl. Washed rHMWP material was recovered by centrifiigation as above. The insoluble rHMWP fraction was fiarther washed 2-times with 10 ml of a PBS solution containing 1.0% Zwittergent 3-14 (Sigma).<br>
The rHMWP pellet recovered after centrifiigation of the final wash solution was then solubilized for 2 hours at room temperature in standard Laemelli SDS-PAGE sample buffer containing 4 M urea. Solubilized rHMWP was size fractionated into a single protein band of ~110 Kdal by electrophoresis through a standard ~14 cmX~20 cmX~3 mm 10% polyacrylamide (36:1, acrylamide:bis-acrylamide) Tris/glycine/SDS preparative gel. A 4% polyacrylamide stacking gel formed using a 5-well, ~500 |il/well preparative comb was polymerized on top of the resolving gel. Electrophoresis was carried out on a BioRad Protean unit for -12 hours at -22°C (-80-85 volts, constant voltage) using a conventional Tris/glycine/SDS running buffer (BioRad). Prestained molecular weight standards (SeeBlue, Novex) were loaded into a parallel lane and were used to gauge the degree and efficiency of separation of the protein species. Following electrophoresis, the gel sandwich was disassembled and a vertical slice was removed from the rHMWP sample lane adjacent to the molecular weight markers and stained with coomassie blue R250 to visualize the rHMWP band. The stained section was then repositioned onto the remaining unstained preparative gel and the strip of acrylamide containing the rHMWP identified and excised.<br>
rHMWP was eluted from the gel slice using a Schleicher and Schuell EluTrap electroelution device. Electroelution was carried out according to the manufacturers recommendations except 1/4-strength SDS running buffer (Novex) was used as the elution buffer. Elution was carried out at -40 mA for -12-14 hours, at room temperature. At the end of the elution period the polarity of the cell was reversed for -2-3 minutes to remove any protein absorbed to the BTl membrane. The rHMWP-containing solution was removed from the collection chamber and stored in a polypropylene conical tube at 4°C.<br>
Excess SDS detergent was removed using an SDS precipitation system (SDS-OUT Precipitation kit, Pierce Chemical). Removal of excess detergent from the gel-eluted protein solution was accomplished following the manufacturer's protocol. Detergent extracted rHMWP was diluted approximately 15 fold with sterile, endotoxin-free 10 m Molar sodium phosphate buffer (pH 7.4)<br><br>
and  concentrated  to  approximately   1.0  mg/ml  by  ultrafiltration  in  an  Amicon  stirred<br>
concentration cell using a YM30 ultrafiltration membrane.<br>
Residual endotoxin was removed from the concentrated rHMWP solution by polymyxin B AfFi-<br>
Prep Polymyxin Matrix (BioRad) treatment. Affi-Prep treatment was performed overnight at 4°C<br>
in a batch mode according to the manufacturers recommendations.<br>
The protein concentration of the concentrated, polymyxin B-treated rHMWP was determined<br>
using the Micro BCA method (Pierce Chem.) and BSA as a standard.<br>
Purified rHMWP (~0.9-1.2 mg/ml protein concentration) was evaluated for purity, identity, and<br>
residual endotoxin burden by SDS-PAGE, Western blot, and a colorimetric endotoxin assay<br>
(BioWhittaker), respectively. The gel-purified rHMWP material displayed a purity of &gt;95% as a<br>
single band of the expected molecular size (~110 Kdal) by gel analysis and reacted vigorously<br>
with rHMWP-specific K196 antibody in Western blots. Residual endotoxin was calculated to be<br><br><br>
WE CLAIM:<br>
1.	An isolated polypeptide comprising an amino acid sequence at least 95% identical to amino acids 29-1012 of SEQ ID NO: 2, wherein said polypeptide is recognizable by an antibody preparation that specifically binds to a polypeptide consisting of the amino acid sequence of SEQ ID NO: 2.<br>
2.	The polypeptide as claimed in claim 1, which comprises amino acids 29-1013 of SEQ ID NO: 15.<br>
3.	The polypeptide as claimed in claim 2, wherein said polypeptide is encoded by nucleotides 82-3036 of SEQ ID NO: 23.<br>
4.	The polypeptide as claimed in claim 2, which comprises SEQ ID NO: 15.<br>
5.	The polypeptide as claimed in claim 4, wherein said polypeptide is encoded by SEQ ID NO: 23.<br>
6.	The polypeptide as claimed in claim 1, which comprises amino acids 29-1013 of SEQ ID NO: 16.<br>
7.	The polypeptide as claimed in claim 6, wherein said polypeptide is encoded by nucleotides 85-3039 of SEQ ID NO: 24.<br>
8.	The polypeptide as claimed in claim 6, which comprises SEQ ID NO: 16.<br>
9.	The polypeptide as claimed in claim 8, wherein said polypeptide is encoded by SEQ ID NO: 24.<br><br>
10.	The polypeptide as claimed in claim 1, which comprises amino acids 29-1012 of SEQ ID NO: 2.<br>
11.	The polypeptide as claimed in claim 10, as expressed by the insert in plasmid pAH342from£. coli BL21.<br>
12.	The polypeptide as claimed in claim 10, wherein said polypeptide is encoded by nucleotides 466-3417 of SEQ ID NO: 1.<br>
13.	The polypeptide as claimed in claim 1, which comprises SEQID NO: 2.<br>
14.	The polypeptide as claimed in claim 13, wherein said polypeptide is encoded by SEQ ID NO: 1.<br>
15.	The isolated polypeptide as claimed in claim 1, which comprises an amino acid sequence at least 95% identical to amino acids 29-533 of SEQ ID NO: 2.<br>
16.	The polypeptide as claimed in claim 15, which comprises amino acids 29-533 of SEQ ID NO: 2.<br>
17.	The polypeptide as claimed in claim 16, wherein said polypeptide is encoded by nucleotides 466 to 1980 of SEQ ID NO: 1.<br>
18.	The polypeptide as claimed in claim 16, as expressed by the insert in plasmid pJJ36-J from E. coli TOP 10.<br><br>
19.	The polypeptide as claimed in claim 15, which comprises amino acids 29-534 of SEQIDNO: 15.<br>
20.	The polypeptide as claimed in claim 19, wherein said polypeptide is encoded by nucleotides 82-1599 of SEQ ID NO: 23.<br>
21.	The polypeptide as claimed in claim 15, which comprises amino acids 29-534 of SEQIDNO: 16.<br>
22.	The polypeptide as claimed in claim 21, wherein said polypeptide is encoded by nucleotides 85-1602 of SEQ ID NO: 24.<br>
23.	The polypeptide as claimed in any one of claims 1-22, wherein said polypeptide is produced by Chlamydia trachomatis. Chlamydia pecorum, or Chlamydia pneumoniae.<br>
24.	The polypeptide as claimed in any one of claims 1-23, comprising a heterologous polypeptide.<br>
25.	The polypeptide as claimed in claim 24, wherein the heterologous polypeptide is a pre or pro sequence, an affinity purification peptide, a heterologous immunogenic peptide, or a combination of two or more of said heterologous polypeptides.<br>
26.	A composition comprising: the polypeptide of any one as claimed in claims 1-25; and a carrier.<br>
27.	The composition as claimed in claim 26, comprising an adjuvant.<br><br>
28.	The composition as claimed in claim 27, wherein said adjuvant is alum, mLT, QS21, MPS, Freund's complete adjuvant, or a combination of two or more of said adjuvants.<br>
29.	The composition as claimed in any one of claims 26-28, comprising a targeting molecule combined with or conjugated with said polypeptide.<br>
30.	The composition as claimed in claim 29, wherein the targeting molecule is vitamin B12, a bacterial toxin or fragment thereof, a monoclonal antibody, a protein, a nucleic acid, a carbohydrate, or a combination of two or more of said targeting molecules.<br>
31.	The composition as claimed in any one of claims 26-30, which is formulated as a microparticle, a capsule, a liposome preparation, or an emulsion.<br>
32.	The composition as claimed in any one of claims 26-31, which induces a humoral immune response (HIR) or cell mediated immune response (CMI) against said polypeptide when administered to a mammal or a bird.<br>
33.	A diagnostic reagent comprising: the polypeptide as claimed in any one of claims 1-25.<br>
34.	A method for detecting anti-Chlamydia antibodies in a test sample comprising the steps of:<br><br>
(a)	contacting said sample with the polypeptide of any one as claimed in claims 1-25 to form Chlamydia antigen: anti-Chlamydia antibody immunocomplexes; and<br>
(b)	detecting the presence of or measuring the amount of said immunocomplexes formed during step (a).<br><br>
35. A diagnostic kit for detecting an antibody or antigen-binding fragment thereof to Chlamydia, wherein said kit comprises the polypeptide as claimed in claims 1-25, a container means for contacting said polypeptide or composition with a test sample, and a reagent means for detecting or measuring Chlamydia antigen: anti-Chlamydia antibody immunocomplexes formed between said polypeptide and said antibody or antigen-binding fragment thereof.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2199-mas-1998 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2199-mas-1998 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2199-mas-1998 correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBjb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">2199-mas-1998 correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBkZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2199-mas-1998 description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">2199-mas-1998 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTE5LnBkZg==" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTQucGRm" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-4.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBmb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">2199-mas-1998 form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE5OS1tYXMtMTk5OCBwZXRpdGlvbi5wZGY=" target="_blank" style="word-wrap:break-word;">2199-mas-1998 petition.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="235120-a-single-stage-process-for-the-depyrogenation-of-an-injectable-solution-of-contrast-agent.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="235122-a-fibre-reactive-organic-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>235121</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2199/MAS/1998</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>29/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>01-Oct-1998</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>EMERGENT PRODUCT DEVELOPMENT GAITHERSBURG INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>300 PROFESSIONAL DRIVE, GAITHERSBURG MD 20879</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PACE, JOHN, L.,</td>
											<td>20408 AFTERNOON LANE, GERMANTOWN, MD 20874</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JACKSON, JAMES</td>
											<td>1687 ARMISTICE WAY, MARRIOTTSVILLE, MD 21104</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07H21/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>08/942,596</td>
									<td>1997-10-02</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/235121-an-isolated-polypeptide-of-chlamydia-and-its-gene-sequence by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:37:21 GMT -->
</html>
