<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/235110-a-triplex-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:36:58 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 235110:A TRIPLEX COMPLEX</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A TRIPLEX COMPLEX</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Triplex complexes contain a single-stranded probe bound to a double-stranded nucleic acid target, in which the probe includes a heteropolymeric nucleic acid or a heteropolymeric nucleic acid analog. All base triplets of the complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C. A cation-facilitated assay includes detecting the presence of such triplex complexes to determine the degree of complementarity between the probe and target sequence. The assay preferably detects a change in fluorescent intensity of a label as a function of binding affinity between the probe and target. The label can be covalently tethered to the probe or to the target, or can be an intercalating fluorophore in the reaction medium.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>CATION MEDIATED TRIPLEX HYBRIDIZATION ASSAY<br>
SPECIFICATION<br>
BACKGROUND OF THE INVENTION<br>
1.	Field of Invention<br>
The invention relates to nucleic acid triplexes, and more particularly to methods of accurately assaying triplex nucleic acid complexes employing fluorescent intensity measurements.<br>
2.	Description of Related Art<br>
Fluorescent dyes have been used to detect and quantitate nucleic acids for decades. In their most basic form, fluorescent intensity-based assays have typically comprised contacting a target with a f luorophore-containing probe, removing any unbound probe from bound probe, and detecting fluorescence in the washed sample. Homogeneous assays improve upon such basic assays, in that the former do not require a washing step or the provision of a non-liquid phase support.<br>
For example, U.S. Patents Nos. 5,538,848 to Livak et al. and 4,220,450 to Maggio disclose homogeneous fluorescence-based assays of nucleotide sequences using oligonucleotide probes in solution. However, these patents require the use of a quenching agent in combination with a i-eporting agent, so as to distinguish between the signals generated by hybridized probes and unhybridized probes. Livak et al. also requires the use of enzymes in its disclosed method. Quenching agents and enzymes add complexity and expense to the methods.<br>
U.S. Patent No. 5,332,659 to Kidwell discloses a method for detecting nucleotide sequences in solution using probes comprising at least two fluorophore moieties. The fluorophores must be selected to electronically interact with each other when close enough to vary the wavelength<br><br>
dependence of their spectra. Unhybridized probes are much more flexible than probes hybridized to the target sequence, and consequently the two fluorophore moieties on each probe are more likely to be close to each other when the probe is unhybridized than when the probe is hybridized. Thus, a change in emission wavelength correlated with free probe can be monitored as an indication of the amount of free probe in the sample.<br>
U.S. Patent No. 5,846,729 to Wu et al. also discloses homogeneous fluorescence-based assays for detecting nucleic acid.<br>
In addition to the aforementioned developments which detect fluorescent intensity, some have touted the advantages of fluorescent polarization assays. However, there are significant drawbacks to polarization-based assays. The degree of change in polarization as a function of binding can be unpredictable, and interpretation of data to conform inconsistent data to theoretical expectations can require more effort than is desirable in an analytical method, particularly when the method is to be automated. There are as well constraints arising from the molecular weight of the molecules whose motion is being evaluated in a fluorescent polarization assay.<br>
Conventidnal assays for nucleic acids have generally been based on a duplex hybridization model, wherein a single-stranded probe specifically binds to a complementary single-stranded target sequence. Triplex hybridization of nucleic acids has been previously identified in the art; however, hybridization among three strands was largely believed to be confined to very limited species of nucleic acids (e.g., polypurine or polypyrimidine sequences). See, e.g., Floris et al. , "Effect of cations on purine-purine-pyrimidine triple helix formation in mixed-valence salt solutions," 260 Eur. J. Biochem. 801-809 (1999). Moreover, such triplex formation or hybridization<br><br>
"Was based on Hoogsteen binding between limited varieties of adjacent nucleobases, rather than Watson-Crick base pairing. See, e.g., Floris et al. and U.S. Patent No. 5,874,555 to Dervan et al.<br>
Despite the foregoing developments, a need has continued to exist in the art for additional simple, highly sensitive, effective and rapid methods for analyzing interaction between nucleic acids and/or nucleic acid analogs.<br>
All references cited herein are incorporated herein by reference in their entireties.<br>
SUMMARY OF THE INVENTION<br>
The invention provides triplex complexes comprising a single-stranded probe bound to a double-stranded nucleic acid target, wherein the probe comprises a heteropolymeric nucleic acid or a heteropolymeric nucleic acid analog, and all base triplets of the complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C.<br>
Also provided is a method for assaying binding, said method comprising:<br>
providing a double-stranded nucleic acid comprising a<br>
target sequence, wherein said target sequence<br>
coritains at least one purine base and at least<br>
one pyrimidine base;<br>
providing a probe comprising a nucleic acid sequence<br>
or a nucleic acid analog sequence; providing a cation;<br>
adding said probe, said target sequence and said cation to a medium to provide a test sample containing a triplex complex comprising said probe bound to said target sequence, wherein all base triplets of said complex are members selected from the group consisting of A-T-A,<br><br>
T-A-T, U-A-T,  T-A-U, A-U-A,  U-A-U,  G-C-G and C-G-C;<br>
irradiating said test sample with exciting radiation to cause test sample to emit fluorescent radiation;<br>
detecting an intensity of said fluorescent radiation, wherein said intensity is correlated with a binding affinity between said probe and said target sequence,- and<br>
determining from said intensity an extent of matching between said probe and said target sequence.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention will be described in conjunction with the following drawings in which like reference numerals designate like elements and wherein:<br>
Figs. 1A, IB, 2A, 2B, 2C, 3A, 3B, 3C, 4A, 4B, 4C, 5A, 5B, 5C, 5D and 5E are composite graphs of fluorescent intensity plotted as a function of wavelength for each sample analyzed.<br>
DETAILED DESCRIPTION OF PREFERRED EMBODIMENTS The invention provides triplex complexes comprising a single-stranded probe bound to a double-stranded nucleic acid target, wherein the probe comprises a heteropolymeric nucleic acid'or a heteropolymeric nucleic acid analog, and all base triplets of the complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C.<br>
Unlike certain Hoogsteen triplexes disclosed by the prior art, the triplexes of the invention are stable at pH values greater than 7.6. Moreover, the inventive triplexes do not require the presence of homopyrimidine sequences or homopurine sequences, as in certain prior art triplexes. For example, the target sequence can contain 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order.<br><br>
Preferably the single-stranded nucleic acid or nucleic acid analog of the triplex is 5 to 30 bases long and the double-stranded nucleic acid target is 8 to 3.3 X 109 base pairs long.<br>
Triplex formation according to the invention is suitable for a variety of uses. For example, probes covalently bound to a double-stranded nucleic acid cleaving agent can be used to specifically cleave target sequences of double-stranded nucleic acids. Probes covalently bound to a chemotherapeutic agent can be used to specifically treat target sequences of double-stranded nucleic acids.<br>
In preferred embodiments, the invention provides a rapid, sensitive, environmentally friendly, and safe method for assaying binding between a double-stranded target and a single-stranded probe, wherein the target comprises a nucleic acid sequence or a nucleic acid analog sequence and the probe comprises a nucleic acid sequence or a nucleic acid analog sequence.<br>
Unlike certain prior art assays, the invention not only detects the presence of specific probe-target binding, but also provides qualitative and quantitative information regarding the nature of interaction between a probe and target. Thus, the invention enables the practitioner to distinguish "among a perfect match, a one base pair mismatch, a two base pair mismatch, a three base pair mismatch, a one base pair deletion, a two base pair deletion and a three base pair deletion arising between a base sequence in the probe and in a strand of the double-stranded target.<br>
Embodiments of the invention comprise calibrating the measured signal (e.g., fluorescent intensity) for a first probe-target mixture against the same type of signal exhibited by other probes combined with the same target, wherein each of the other probes differs from the first probe by at least one base.<br><br>
A calibration curve can be generated, wherein the magnitude of the measured signal (e.g., fluorescent intensity) is a function of the binding affinity between the target and probe. As the binding affinity between the target and a plurality of different probes varies with the number of mismatched bases, the nature of the mismatch(es) (A-G vs. A-C vs. T-G vs. T-C, etc.), the location of the mismatch(es) within the triplex, etc., the assay of the invention can be used to sequence the target.<br>
In embodiments, the signal measured can be the fluorescent intensity of a fluorophore included in the test sample. In such embodiments, the binding affinity between the probe and target can be directly or inversely correlated with the intensity, depending on whether the fluorophore signals hybridization through signal quenching or signal amplification. Under selected conditions, the fluorescent intensity generated by intercalating agents can be directly correlated with probe-target binding affinity, whereas the intensity of preferred embodiments employing a non-intercalating fluorophore covalently bound to the probe can be inversely correlated with probe-target binding affinity. The fluorescent intensity decreases for non-intercalating fluorophores as the extent of matching between the pfrobe and target increases, preferably over a range inclusive of 0-2 mismatches and/or deletions, more preferably over a range inclusive of 0-3 mismatches and/or deletions.<br>
The invention enables quantifying the binding affinity between probe and target. Such information can be valuable for a variety of uses, including designing antisense drugs with optimized binding characteristics.<br>
Unlike prior art methods, the assay of the invention is preferably homogeneous. The assay can be conducted without separating the probe-target complex from the free probe and target prior to detecting the magnitude of the<br><br>
measured signal. The assay does not require a gel separation step, thereby allowing a great increase in testing throughput. Quantitative analyses are simple and accurate. Consequently the binding assay saves a lot of time and expense, and can be easily automated. Furthermore, it enables binding variables such as buffer, pH, ionic concentration, temperature, incubation time, relative concentrations of probe and target sequences, intercalator concentration, length of target sequences, length of probe sequences, and possible cofactor requirements to be rapidly determined.<br>
The assay can be conducted in, e.g. , a solution within a well, on an impermeable surface or on a biochip.<br>
Moreover, the inventive assay is preferably conducted without providing a signal quenching agent on the target or on the probe.<br>
Although the inventors have previously disclosed the advantages of fluorescent intensity assays for hybridization (see, e.g., U.S. Patent Application No. 09/224,505, filed December 31, 1998), assays according to the present invention specifically detect triplexes of the probe and the double-stranded target, thus obviating the need to denature the target. While nucleic acid (and nucleic acid* analog) probes have been known to form triplexes with certain limited classes of targets (see, e.g., Floris et al. , supra, Dervan et al., supra, Egholm et al., 365 Nature 566 (1993), and Tomac et al., 118 J.Am.Chem.Soc. 5544 (1996)), it is surprising that the inventors have been able to specifically assay triplexes formed between single-stranded nucleic acid (e.g., ssDNA and RNA) probes and double-stranded nucleic acid (e.g., dsDNA) targets, wherein the interaction between the probes and targets is based on Watson-Crick base pairing (at least in the sense that A binds to T (or U, in the case of RNA) and G binds to C), rather than the very limited Hoogsteen<br><br>
model of triplex hybridization of, e.g., Dervan et al. The term "Watson-Crick triplex," which is employed herein, is intended to crystallize these differences by limiting the nature of base pairing between the single-stranded probe and the double-stranded target to A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and/or C-G-C (including C+-G-C, and/or any other ionized species of base). These three-member groups are hereinafter denoted Watson-Crick base triplets and the resulting structures denoted Watson-Crick triplexes.<br>
Suitable probes for use in the inventive assay include, e.g., ssDNA, RNA, PNA and other nucleic acid analogs having uncharged or partially-charged backbones. Probe sequences having any length from 8 to 20 bases are preferred since this is the range within which the smallest unique DNA sequences of prokaryotes and eukaryotes are found. Probes of 12 to 18 bases are particularly preferred since this is the length of the smallest unique sequences in the human genome. In embodiments, probes of 5 to 30 bases are most preferred. However, a plurality of shorter probes can be used to detect a nucleotide sequence having a plurality of non-unique target sequences therein, which combine to uniquely identify the nucleotide sequence. The length of th£ probe can be selected to match the length of the target.<br>
In parent U.S. Application Serial No. 09/468,679, the inventors disclosed the surprising development that they were able to specifically assay a wide-variety of triplexes formed in a Watson-Crick base-pair dependent manner between single-stranded nucleic acid (e.g., ssDNA, RNA, ssPNA and other analogs of DNA or RNA) probes and double-stranded nucleic acid (e.g., dsDNA) targets. The inventors disclosed that triplex formation and/or stabilization is enhanced by the presence of an intercalating agent in the sample being tested.<br><br>
The instant disclosure expands upon the earlier one by disclosing that Watson-Crick triplex formation and/or stabilization is enhanced by the presence of cations in the sample being tested. Suitable cations include, e.g., monovalent cations, such as Na* (preferably at a concentration of 50mM to 125mM) , K+, and other alkali metal ions; divalent cations, such as alkaline earth.metal ions (e.g., Mg+2 and Ca+2) and divalent transition metal ions (e.g., Mn*2, Ni+2, Cd+2, Co*2 and Zn+2) ; and cations having a positive charge of at least three, such as Co(NH3)6+3, trivalent spermidine and tetravalent spermine. Mn+2 is preferably provided at a concentration of lOmM to 30mM. Mg*2 is preferably provided at a concentration of 15niM to 20mM. Ni*2 is preferably provided at a concentration of about 20mM. In embodiments, Mg+a and Mn+2 are provided in combination at a concentration of lOmM each, 15mM each or 20mM each (i.e., 10-20 mM each).<br>
The amount of cation added to the medium in which the triplex forms depends on a number of factors, including the nature of the cation, the concentration of probe, the concentration of target, the presence of additional cations and the base content of the probe and target. The preferred cation concentrations and mixtures can routinely be discovered' experimentally.<br>
The instant invention does not require the use of radioactive probes, which are hazardous, tedious and time-consuming to use, and need to be constantly regenerated. Probes of the invention are preferably safe to use and stable for years. Accordingly, probes can be made or ordered in large quantities and stored.<br>
In embodiments, the probe is labeled with a multi-molecule signaling complex or a redox pair, or with a label that elicits chemiluminescent or electrochemiluminescent properties.<br><br>
It is preferred that the probe or target (preferably the probe) have a fluorescent label covalently bound thereto. The label is preferably a non-intercalating fluorophore. In such embodiments, the fluorophore is preferably bound to the probe at either end. Preferred fluorescent markers include biotin, rhodamine and fluorescein, and other markers that fluoresce when irradiated with exciting energy.<br>
The excitation wavelength is selected (by routine experimentation and/or conventional knowledge) to correspond to this excitation maximum for the fluorophore being used, and is preferably 200 to 1000 nm. Fluorcphores are preferably selected to have an emission wavelength of 200 to 1000 nm. In preferred embodiments, an argon ion laser is used to irradiate the fluorophore with light having a wavelength in a range of 400 to 540 nm, and fluorescent emission is detected in a range of 500 to 750 nm.<br>
The assay of the invention can be performed over a wide variety of temperatures, such as, e.g., from 5 to 85°C. Certain prior art assays require elevated temperatures, adding cost and delay to the assay. On the other hand, the invention can be conducted at room temperature cfr below (e.g., at a temperature below 25°C) .<br>
The reliability of the invention is independent of guanine arid cytosine content in said target. Since G-C base pairs form three hydrogen bonds, while A-T base pairs form only two hydrogen bonds, target and probe sequences with a higher G or C content are more stable, possessing higher melting temperatures. Consequently, base pair mismatches that increase the GC content of the hybridized probe and target region above that present in perfectly matched hybrids may offset the binding weakness associated with a mismatched probe. Triplexes containing every possible base pair mismatch between the probe and the<br><br>
target proved to be more unstable than perfectly matched tTriplexes, always resulting in lower fluorescent intensities than did perfectly complementary hybrids, when an intercalating fluorophore was used.<br>
The inventive assay is extremely sensitive, thereby obviating the need to conduct PCR amplification of the target. For example, it is possible to assay a test sample having a volume of about 20 microliters, which contains about 10 femtomoles of target and about 10 femtomoles of probe. Embodiments of the invention are sensitive enough to assay targets at a concentration of 5 X 10~9 M, preferably at a concentration of not more than 5 x 10"10 M. Embodiments of the invention are sensitive enough to employ probes at a concentration of 5 X 10"9 M, preferably at a concentration of not more than 5 x 10"10 M. It should go without saying that the foregoing values are not intended to suggest that the method cannot detect higher concentrations.<br>
The medium in which triplexes form can be any conventional medium known to be suitable for preserving nucleotides. See, e.g., Sambrook et al., "Molecular Cloning: A Lab Manual," Vol. 2 (1989). For example, the liquid medium can comprise nucleotides, water, buffers and standard salt&gt; concentrations. When divalent cations are used exclusively to promote triplex formation, chelators such as EDTA or EGTA should not be included in the reaction mixtures.<br>
Specific binding between complementary bases occurs under a wide variety of conditions having variations in temperature, salt concentration, electrostatic strength, and buffer composition.  Examples of these conditions and methods for applying them are known in the art.<br>
Unlike many Hoogsteen-type triplexes, which are unstable or non-existent at pH levels above about 7.6, the Watson-Crick triplexes of the invention are stable over a<br><br>
wide range of pH levels, preferably from about pH 5 to 9bout pH 9.<br>
It is preferred that triplexes be formed at a temperature of about S°C to about 25°C for about one hour or less. Longer reaction times are not required, but incubation for up to 24 hours in most cases did not adversely affect the triplexes. The fast binding times of Watson-Crick triplexes of the invention contrast with the much longer binding times for Hoogsteen triplex-based assays.<br>
Although not required, it is possible to facilitate triplex formation in solution by using certain reagents in addition to cations. Preferred examples of these reagents include single stranded binding proteins such as Rec A protein, T4 gene 32 protein, E. coli single stranded binding protein, major or minor nucleic acid groove binding proteins, viologen and intercalating substances such as ethidium bromide, actinomycin D, psoralen, and angelicin. Such facilitating reagents may prove useful in extreme operating conditions, for example, under abnormal pH levels or extremely high temperatures.<br>
The inventive assay can be used to, e.g., identify accessible regions in folded nucleotide sequences, to determine the number of mismatched base pairs in a hybridization complex, and to map genomes.<br>
The inventors may sometimes herein suggest that Watson-Crick triplexes result from hybridization of the probe to duplex target. While fluorophores tethered to the probe produced quenched fluorescent emissions upon being exposed to duplex targets containing a strand of Watson-Crick complementary bases, which indicates the occurrence of some kind of binding event, the inventors are not sure that what occurs in the Watson-Crick triplex is best described as hybridization in the sense traditionally associated with Watson-Crick duplex formation. While the<br><br>
formation of a Watson-Crick triplex may sometimes be referred to as a hybridization event herein, that is merely for convenience and is not intended to limit the scope of the invention with respect to how the formation of a Watson-Crick triplex can be best characterized.<br>
The invention will be illustrated in more detail with reference to the following Examples, but it should be understood that the present invention is not deemed to be limited thereto.<br>
EXAMPLES Example 1<br>
Sense and antisense 50-mer ssDNA target sequences, derived from exon 10 of the human cystic fibrosis gene (Nature 380,  207  (1996)}  were synthesized on a DNA synthesizer (Expedite 8909, PerSeptive Biosystems)  and purified by HPLC.   Equimolar amounts of complementary oligonucleotides were denatured at 95°C for 10 min and allowed to anneal gradually as the temperature cooled to 21°C over  1.5  hours.   Double  stranded DNA  (dsDNA) oligonucleotides were dissolved in ddH2O at a "concentration of 1 pmole//ul.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO:1): 5'-TGG CAC CAT TAA AGA AAA TAT CAT CTT TGG TGT TTC CTA TGA TGA ATA TA-31.<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ IDNOtl): 5'-TAT ATT CAT CAT AGG AAA CAC CAA AGA TGA TAT TTT CTT TAA TGG TGC CA- 3 ' .<br>
SEQ ID NO: 2 was a 50-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:1) except for a one base pair mutation (underlined) at amino acid position 507 at which the wild-type sequence CAT was changed to CGT.<br>
Sequence^ for the sense strand of SEQ ID NO:2: 5'-TGG CAC CAT TAA AGA AAA TAT CGT CTT TGG TGT TTC CTA TGA TGA ATA TA-31.<br>
Sequence for the antisense strand of SEQ ID NO.-2: 5 ' -TAT ATT CAT CAT AGG AAA CAC CAA AGA CGA TAT TTT CTT TAA TGG TGC CA.-3'.<br>
SEQ ID NO: 3 was a 47-mer mutant dsDNA target sequence identical to wild-type target DNA (SEQ ID NO:1) except for a consecutive three base pair deletion (indicated by an ellipsis) at amino acid positions 507 and 508 at which the wild-type sequence CTT is deleted.<br>
Sequence for the sense strand of SEQ ID N0:3: 5'-TGG<br>
CAT TAA AGA AAA TAT CAT . . . TGG TGT TTC CTA TGA TGA ATA TA-3 ' .<br>
Sequence for the antisense strand of SEQ ID NO:3: 5 ' -TAT ATT CAT CAT AGG AAA CAC CA . . . A TGA TAT TTT CTT TAA TGG TGC CA-31.<br>
Probe No. 1 (SEQ ID NO:4) , a 15-mer ssDNA probe with an attached fluorescein moiety at the 5' position, was designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DMA (SEQ ID NO:1), overlapping amino acid positions 505 to 510 (Nature 380, 207 (1996)). Probe No. 1 was synthesized on a DNA synthesizer, purified by HPLC, and dissolved in ddH2O at a concentration of 1 pmole//il.<br>
Sequence for SEQ ID N0:4: 5'-Flu-CAC CAA AGA TGA TAT-3'.<br>
The hybridization reaction mixture (40//1) contained the following: 0.4 pinoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA Probe No. 1, 10 mM Tris-HCl, pH 7.5 and 0, 10, 25, 50, 75, 100, 125 or 150 mM NaCl. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The maximum fluorescent intensities occurred at a wavelength of 525 nm, the emission wavelength for fluorescein. The intensity of fluorescence was plotted as a function of wavelength for each sample analyzed.<br>
In the absence of NaCl or presence of 10 mM or 25 mM NaCl, no hybridization between the dsDNA targets and the ssDNA-F probe was detected, resulting in similar fluorescent intensities observed when wild-type target SEQ ID NO:1 or mutant target SEQ ID NO: 2 were mixed with Probe No. 1 (SEQ ID NO:4) or when Probe No. 1 was present alone (data not shown).<br>
After a one-hour incubation at 21°C in the presence of 50mM NaCl, dsDNA:ssDNA-F triplexes consisting of perfectly complementary sequences (SEQ ID NO:1 + Probe No. 1) formed readily, resulting in a 49% decrease in fluorescent intensity compared to that emitted by Probe No. 1 alone (labeled ssDNA-F) (Fig. 1A). In contrast, incompletely complementary dsDNA:ssDNA-F triplexes containing a 1 bp G-T mismatch (SEQ ID NO:2 + Probe No. 1} were less stable in these reaction conditions, yielding only an 11% decrease in fluorescent intensity compared to that exhibited by Probe No. 1 alone.<br>
Incubation for one hoxir in the presence of 75 mM NaCl was slightly less conducive to triplex formation, resulting in a 30% decrease in fluorescent intensity for the perfectly matched dsDNA:ssDNA-F triplex (Fig. IB) . Minimal formation of the 1 bp G-T mismatched dsDNA:ssDNA-F triplex was observed, resulting in only a 0.4% decrease in fluorescence.<br>
The presence of 100 mM and 125 mM NaCl also facilitated maximum triplex DNA formation between the perfectly matched SEQ ID NO:1 target and Probe No. 1, and less stable triplex DNA formation between the 1 bp G-T mismatched SEQ ID NO:2 and Probe No. 1 hybrid (data not shown) . At '150 mM NaCl, no triplex DNA formation was evident.<br>
Therefore, the inclusion of monovalent cations such as Na* and K4 at specific concentrations, was sufficient to allow detection of triplex formation between dsDNA targets and ssDNA probes labeled with f luorescein in the absence of prior denaturation. Moreover, the reaction occurred at room temperature within just one hour of incubation at a ratio of probe to target of 10 to 1, using natural dsDNA. The dsDNA targets and ssDNA probe used in this example contained a 33% GC content, and did not contain homopurine or homopyrimidine stretches of DNA. Despite the presence<br>
of 6 pyrimidine bases interspersed within the 15 nucleotide ssDNA probe, DNA triplexes formed easily. Significantly, the hybridization assay of the invention was able to discriminate between perfectly complementary DNA sequences and those containing a single 1 bp mismatch using natural DNA. Example 2<br>
To ensure that the hybridization assay, which used 5'-fluorescein labeled ssDNA probes and dsDNA targets in the absence of prior denaturation, would apply to probe and target DNAs possessing dramatically different percent GC contents (and potentially different annealing preferences) , new 15-mer ssDNA-F probes and 50-mer dsDNA target sequences were synthesized, purified and annealed as above. Both ssDNA-F probes and dsDNA targets were dissolved in ddH2O at a concentration of 1 pmole//j.l.<br>
SEQ ID NO: 5 was a 50-mer dsDNA target sequence modified from SEQ ID N0:l, wherein the percent GC content was changed from 30% to 52%.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO: 5): 5'-GAG CAC CAT GAC AGA CAC TGT CAT CTC TGG TGT GTC CTA CGA TGA CTC TG-3 ' .<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO: 5): 5'-GAG ACT CAT CGT AGG ACA CAC CAG AGA TGA CAG TGT CTG TCA TGG TGC TC-3 ' .<br>
SEQ ID NO: 6 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:5, except for a one base pair mutation (underlined), at which the sequence CTC was changed to CTT.<br>
Sequence for the sense strand of mutant SEQ ID NO: 6: 5 ' -GAG CAC CAT GAC AGA CAC TGT CAT CTT TGG TGT GTC CTA CGA TGA CTC TG-31.<br>
Sequence for the antisense strand of mutant SEQ ID NO: 6: 5 '-CAG ACT CAT CGT AGG ACA CAC CAA AGA TGA CAG TGT CTG TCA TGG TGC TC-3'.<br>
SEQ ID NO: 7 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:5, except for a one base pair mutation (underlined), at which the sequence CAT was changed to CGT.<br>
Sequence for the sense strand of mutant SEQ ID NO:7: 5 ' -GAG CAC CAT GAC AGA CAC TGT CGT CTC TGG TGT GTC CTA CGA TGA CTC TG-31.<br>
Sequence for the antisense strand of mutant SEQ ID NO: 7: 5 ' -CAG ACT CAT CGT AGG ACA CAC CAG AGA CGA CAG TGT CTG TCA TGG TGC TC-31.<br>
SEQ ID NO:8 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:5, except for a one base pair mutation (underlined), at which the sequence CAT was changed to CTT.<br>
Sequence for the sense strand of mutant SEQ ID NO:8: 5' -GAG CAC CAT GAC AGA CAC TGT CTT CTC TGG , TGT GTC CTA CGA TGA CTC TG-3 ' .<br>
Sequence for the antisense strand of mutant SEQ ID NO: 8: 5'-CAG ACT CAT CGT AGG ACA CAC CAG AGA AGA CAG TGT CTG TCA TGG TGC TC-31.<br>
SEQ ID NO:9 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:5, except for a one base pair mutation (underlined), at which the sequence CTC was changed to CCC.<br>
Sequence for the sense strand of mutant SEQ ID NO:9: 5'-GAG CAC CAT GAC AGA CAC TGT CAT CCC TGG TGT GTC CTA CGA TGA CTC TG-3'.<br>
Sequence for the antisense strand of mutant SEQ ID NO: 9: 5'-CAG ACT CAT CGT AGG ACA CAC CAG GGA TGA CAG TGT CTG TCA TGG TGC TC-3'.<br>
SEQ ID NO:10 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO: 5, except for a one base pair mutation (underlined), at which the sequence CTC was changed to CGC.<br>
Sequence for the sense strand of mutant SEQ ID NO:10: 5'-GAG CAC CAT GAG AGA CAC TGT CAT CGC TGG TGT GTC CTA CGA TGA CTC TG-3 ' .<br>
Sequence for the antisense strand of mutant SEQ ID NO: 10: 5 '-CAG AGT CAT CGT AGG ACA CAC CAG CGA TGA CAG TGT CTG TCA TGG TGC TC-3 ' .<br>
SEQ ID NO: 11 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO:5, except for a consecutive two base pair mutation (underlined), at which the sequence CAT was changed to ACT.<br>
Sequence for the sense strand of mutant SEQ ID NO: 11: 5' -GAG CAC CAT GAG AGA CAC TGT ACT CTC TGG TGT GTC CTA CGA TGA CTC TG-3 ' .<br>
Sequence for the antisense strand of mutant SEQ ID NO: 11: 5'-CAG AGT CAT CGT AGG ACA CAC CAG AGA GTA CAG - TGT CTG TCA TGG TGC TC-3 ' .<br>
SEQ ID NO:12 was a 50-mer dsDNA target sequence modified from SEQ ID NO:1, wherein the percent GC content was changed from 30% to 72%.<br>
Sequence for the sense strand of the wild-type target DNA (SEQ ID NO-.12): 5'-GAG CAC CCT CCC AGG CAC GGT CGT CCC TGG TGC GAG CTC CGA CGA GCG TG-31.<br>
Sequence for the antisense strand of the wild-type target DNA (SEQ ID NO: 12): 5 '-CAC GCT CGT CGG AGG TCG CAC CAG GGA CGA CCG TGC CTG GGA GGG TGC TC-3 ' .<br>
SEQ ID NO:13 was a 50-mer mutant dsDNA target sequence identical to SEQ ID NO: 12, except for a one base pair mutation (underlined) , at which the sequence CGT was changed to CAT.<br>
Sequence for the sense strand of mutant SEQ ID NO: 13: 5 ' -GAG CAC CCT CCC AGG CAC GGT CAT CCC TGG TGC GAC CTC CGA CGA GCG TG-3'.<br>
Sequence for the antisense strand of mutant SEQ ID NO: 13: 5'-CAC GCT CGT CGG AGG TCG CAC CAG GGA TGA CCG TGC CTG GGA GGG TGC TC-3'.<br><br>
Probe No. 2 (SEQ ID NO:14), a 15-mer ssDNA probe with an attached fluorescein moiety at the 5' position, was designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DNA (SEQ ID NO:5).<br>
Sequence for SEQ ID NO: 14: 5 ' -Flu-CAC CAG AGA TGA GAG-3'.<br>
Probe No. 3 (SEQ ID NO:15) was a 15-mer 5'-fluorescein labeled ssDNA probe designed to be completely complementary to a 15 nucleotide segment of the sense strand of the 50-mer wild-type target DNA (SEQ ID NO:12) .<br>
Sequence for SEQ ID NO: 15: 5'-Flu-CAC CAG GGA CGA CCG-3' .<br>
The triplex DNA hybridization assays performed in Example 1 were facilitated by the addition of monovalent cations in the reaction mixtures. The specificity of the hybridization assay was further examined utilizing divalent cations (instead of monovalent cations) to promote triplex DNA formation with dsDNA targets and ssDNA-F probes possessing various percent GC contents.<br>
The hybridization reaction mixture (40//1) contained the following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5 and 5 mM to 3O mM MnCl2 or 5 mM to 30 mM MgCl2 or 5 mM to 30 mM NiCl2. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The samples were saved and allowed to incubate at room temperature overnight for a total of 22 hours, at which time a second fluorescent intensity measurement was taken following irradiation with the argon ion laser beam. The intensity of fluorescence was plotted as a function of wavelength for each sample analyzed.<br><br>
When the ssDNA-F Probe No. 2 (with a 53% GC content)<br>
was incubated with the 50-mer wild-type dsDNA target (SEQ<br>
ID NO: 5) and mutant dsDNA targets (SEQ ID NO: 6 to SEQ ID<br>
NO:11)  in the presence of 10 mM MnCl2,  dsDNA:ssDNA-F<br>
triplexes  were  formed  at  room  temperature  under<br>
non-denaturing conditions.  While perfectly matched DNA<br>
triplexes yielded the maximum decrease in fluorescent<br>
intensity (a 43% decrease after a one-hour incubation), the<br>
less stable dsDNA:ssDNA-F triplexes containing a 1 bp T-G<br>
mismatch (SEQ ID NO:6 + Probe No. 2) produced a fluorescent<br>
intensity that was 20% lower than that observed with Probe<br>
No.  2 alone after a one-hour incubation  (Fig.  2A) .<br>
dsDNA-.ssDNA-F triplexes that resulted in a 1 bp G-T<br>
mismatch (SEQ ID N0:7 + Probe No. 2), a 1 bp T-T mismatch<br>
(SEQ ID NO:8 + Probe No. 2), a 1 bp C-A mismatch (SEQ ID<br>
NO: 9 + Probe No. 2) and a consecutive 2 bp A-G and C-T<br>
mismatch (SEQ ID NO: 11 + Probe No. 2) were all less stable<br>
than the perfectly matched DNA triplex (SEQ ID NO:5 + Probe<br>
No. 2) yielding fluorescent intensities in between that<br>
observed for Probe No. 2 alone and that observed for the<br>
perfectly matched DNA triplex (data not shown) . Except for<br>
the 1 bp T-T mismatched DNA triplex, which was the least<br>
stable (resulting in only a 5% decrease in fluorescent<br>
intensity after 1 hour), all of the other mismatched DNA<br>
triplexes generated very similar fluorescent intensities.<br>
Only the dsDNA:ssDNA-F triplex that contained a 1 bp G-A<br>
mismatch (SEQ ID NO: 10 + Probe No. 2) yielded a fluorescent<br>
intensity lower than that produced by the perfectly matched<br>
DNA triplex (data not shown).<br>
DNA triplex formation was more efficient after a 22-hour incubation in the presence of 10 mM MnCl2. Nevertheless, a more prominent discrimination between DNA triplexes containing perfectly matched sequences and DNA triplexes containing base pair mismatched sequences was observed. As illustrated in Fig. 2B, the dsDNA:ssDNA-F<br><br>
The inclusion of 20 mM MgCl2 or 20 mM MnCl2 or 20 mM NiCl2 also facilitated dsDNA:ssDNA triplex formation when the ssDNA-F Probe No. 3 (possessing a 73% GC content) was reacted with the corresponding 50-mer wild-type dsDNA target (SEQ ID NO: 12) and mutant dsDNA target (SEQ ID NO: 13) for one hour (data not shown). As expected, the perfectly matched DNA triplexes generated the maximum decreases in fluorescent intensity, while the less stable 1 bp A-C mismatched DNA triplexes (SEQ ID NO: 13 + Probe No. 3) produced intermediate levels of fluorescence (data not shown). The perfectly matched DNA triplexes formed very efficiently in the presence of 10 mM MnCl2 after a 22 hour incubation, yielding an 89% decrease in fluorescent intensity. -The 1 bp A-C mismatched DNA triplexes were formed with equal efficiency in these reaction conditions, generating a 90% decrease in fluorescence compared to that observed with Probe No. 3 alone (data not shown). Therefore, better discrimination was achieved between the perfectly matched and 1 bp mismatched 73% GC DNA triplexes following short incubation times of 1 hour in the presence of 20 mM divalent cations.<br>
Perfectly matched dsDNA:ssDNA-F triplexes (possessing a 33% GC content) (SEQ ID NO:1 + Probe No. 1) formed readily within 1 hour in the presence of 10 mM MnCl2, resulting in a 57% decrease in fluorescent intensity<br><br>
compared to that emitted by Probe No. 1 alone (data not "shown). These reaction conditions were highly unfavorable for DNA triplexes that contained a 1 bp G-T mismatch (SEQ ID NO-.2 + Probe No. 1), resulting in an increased fluorescence compared to that observed by Probe No. 1 alone (data not shown). Similar results were obtained following a 22 hour incubation in the presence of 15 mM MgCl2.<br>
Regardless of the percent GC content of the dsDNA targets and ssDNA probes, the addition of divalent cations such as Mn+2, Mg+2 or Ni+2 promoted DNA triplex formation under non-denaturing conditions, to allow accurate discrimination between perfectly complementary sequences and those containing 1 bp mutations. Example 3<br>
The triplex DNA hybridization assays in Examples 1 and 2 were performed in the presence of one type of raonovalent or divalent cation. The next examples will demonstrate the reliability of the assay of the invention to differentiate between perfect matches and 1 bp mismatches in triplex DNA when combinations of divalent cations are present in the reaction mixtures.<br>
The hybridization reaction mixture (40fJ.~L) contained the following: 0.4 pmoles of target dsDNA, 4 pmoles of 5'-fluorescein labeled ssDNA probe, 10 mM Tris-HCl, pH 7.5 and 5 mM MgCl2 and 5 mM MnCl2, or 10 mM MgCl2 and 10 mM MnCl2, or 15 mM MgCl2 and 15 mM MnCl2, or 20 mM MgCl2 and 20 mM MnCl2. The reaction mixtures were incubated at room temperature (21°C) for 1 hour, without prior denaturation. Samples were placed into a quartz cuvette, irradiated with an argon ion laser beam having a wavelength of 488 nm and monitored for fluorescent emission. The samples were saved and allowed to incubate at room temperature overnight for a total of 22 hours, at which time a second fluorescent intensity measurement was taken following irradiation with the argon ion laser beam. The intensity of fluorescence<br><br>
was plotted as a function of wavelength for each sample<br>
In all mixtures of dsDNA target and ssDNA-F probe, the addition of 5 mM MgCl2 and 5 mM MnCl2 was insufficient to allow detection of triplex DNA formation (data not shown) . When the ssDNA-F Probe No. 3 (with a. 73% GC content) was incubated for one hour with the 50-mer wild- type dsDNA target (SEQ ID NO: 12) in the presence of 10 mM MgCl2 and 10 mM MnCl2, or 15 mM MgCl2 and 15 mM MnCl2/ perfectly complementary dsDNA:ssDNA-F triplexes were formed with equal efficiency, generating a 29% decrease in fluorescence compared to that emitted by Probe No. 3 alone. Both reaction conditions were highly unfavorable for DNA triplexes that contained a 1 bp A-C mismatch (SEQ ID NO: 13 -i- Probe No. 3) , resulting in a 14% increase in fluorescence compared to that observed with Probe No . 3 alone . The fluorescent spectra obtained after a one hour incubation in the presence of 15 mM MgCl2 and 15 mM MnCl2 are shown in Fig. 3A.<br>
Incubation for 22 hours yielded more DNA triplex formation.    The  dsDNA: ssDNA-F  triplexes  containing perfectly matched sequences (SEQ ID NO: 12 + Probe No. 3) or a 1 bp A-C mismatch (SEQ ID NO: 13 + Probe No. 3) produced fluorescent intensities that were 62% and 21% lower, respectively, than that achieved by Probe No. 3 alone, following a 22 hour incubation in the presence of 10 mM MgCl2 and 10 mM MnCl2 (Fig. 3B) . Very similar results were obtained with the samples containing 15 mM MgCl2 and 15 mM MnCl2 after 22 hours (data not shown) .<br>
Treatment with 20 mM MgCl2 and 20 mM MnCl2 for just one hour, resulted in a 46% and a 3% reduction in fluorescence for perfectly matched DNA triplexes and 1 bp A-C mismatched DNA triplexes, respectively (Fig. 3C) . In this case, no benefit was achieved by further incubating the samples for 22 hours (data not shown) .<br><br>
When dsDNA targets containing a 73% GC content are Rested in the hybridization assay of the invention, a one-hour treatment with 20 mM MgCl2 and 20 mM MnCl2 provides the maximum difference in stability and fluorescence between perfectly complementary DNA triplexes and DNA triplexes containing a 1 bp mismatch. Example 4<br>
When the ssDNA-F Probe No. 1 (with a 33% GC content) was incubated with the wild-type dsDNA target (SEQ ID N0:l) or mutant dsDNA targets (SEQ ID NO: 2 and SEQ ID NO: 3), in the presence of 10 mM MgCl2 and 10 mM MnCl2, minimal DNA triplex formation was observed (data not shown) . However, incubation in the presence of 15 mM MgCl2 and 15 mM MnCl2 for one hour facilitated perfectly matched DNA triplex formation, as evidenced by the 49% decrease in fluorescent intensity observed, compared to that obtained by Probe No. I (Fig. 4A) . dsDNA:ssDNA-F triplexes that resulted in a 1 bp G-T mismatch (SEQ ID N0:2 + Probe No. 1) or a 3 bp deletion (SEQ ID NO:3 + Probe No. 1) were very unstable in the presence of 15 mM MgCl2 and 15 mM MnCl2/ yielding a 2% decrease in fluorescence and a 5% increase in fluorescence, respectively, compared to that emitted by Probe No. 1 alone (Fig. 4A) .<br>
Treatment with 20 mM MgCl2 and 20 mM MnCl2 for 1 hour, resulted in a 68%, a 48% and a 6% reduction in fluorescence for perfectly matched DNA triplexes, and for dsDNA:ssDNA-F triplexes containing a 1 bp G-T mismatch or a 3 bp deletion, respectively, compared to that observed with Probe No. 1 alone (Fig. 4B) . Optimum discrimination between the 33% GC DNA triplexes containing wild-type sequences or base pair mismatches was achieved when these same samples were incubated for 22 hours. The perfectly complementary DNA triplexes (SEQ ID NO.-l- + Probe No. 1) remained stable over the 22 hours, producing a 62% decrease in fluorescent intensity, compared to that achieved by<br><br>
No- ! alone (Fig. 4C) . By contrast, the DNA:ssDNA-F triplexes containing a 1 bp G-T mismatch (SEQ ID NO:2 + Probe No. 1) or a 3 bp deletion (SEQ ID NO:3 + Probe No. 1) proved to be very unstable during the 22 hour incubation, generating a 1% and a 13% increase in fluorescence, respectively, compared to that emitted by Probe No. 1 alone (Fig. 4C). Example 5<br>
Perfectly matched dsDNA:ssDNA-F triplexes (possessing a 53% GC content) (SEQ ID NO:5 + Probe No. 2) formed readily within one hour in the presence of 10 mM MgCl2 and 10 mM MnCl2, resulting in a 68% decrease in fluorescence compared to that observed by Probe No. 2 alone (Fig. 5A) . The DNA triplexes that contained a 1 bp T-G mismatch (SEQ ID NO:6 + Probe No. 2) were less stable, generating a 20% decrease in fluorescent intensity compared to that achieved by Probe No. 2 alone (Fig. 5A).<br>
Incubation of the same samples for 22 hours produced an even more dramatic difference in fluorescence achieved by the perfectly matched or mismatched DNA triplexes. As illustrated in Fig. 5B, the dsDNA:ssDNA-F triplexes containing perfectly complementary sequences (SEQ ID NO:5 + Probe No. 2) or a 1 bp T-G mismatch (SEQ ID NO:6 + Probe No. 2) generated fluorescent intensities that were 92% and 33% lower, respectively, than that emitted by Probe No. 2 alone, in the presence of 10 mM MgCl2 and 10 mM MnCl2.<br>
In a similar experiment, while the perfectly matched DNA triplex (SEQ ID NO: 5 + Probe No. 2) yielded a 85% decrease in fluorescence compared to that observed with Probe No. 2 alone following a 22 hour incubation in the presence of 10 mM MgCl2 and 10 mM MnCl2, the dsDNA:ssDNA-F triplexes that resulted in a 1 bp G-T mismatch (SEQ ID NO: 7 + Probe No. 2), a 1 bp C-A mismatch (SEQ ID NO:9 + Probe No. 2) and a consecutive 2 bp A-G and C-T mismatch (SEQ ID NO: 11 + Probe No. 2) produced a 43%, a 69% and a 32%<br><br>
.^duction in fluorescence (Fig. 5C) . Only the dsDNA: ssDNA-F triplex that contained a 1 bp G-A mismatch (SEQ ID NO: 10 + Probe No. 2) yielded a fluorescent intensity slightly lower than that produced by the perfectly matched DNA triplex {data not shown).<br>
Optimum discrimination between the 53% GC DNA triplexes containing perfectly complementary sequences or base pair mismatches was achieved following a one-hour incubation in the presence of 15 mM MgCl2 and 15 mM MnCl2. These reaction conditions greatly facilitated DNA triplex formation between the perfectly matched DNA sequences (SEQ ID NO:5 + Probe No. 2) , resulting in a 74% reduction in fluorescence compared to that achieved by Probe No. 2 alone (Fig. 5D) . By contrast, dsDNA:ssDNA-F triplexes that contained a 1 bp T-G mismatch {SEQ ID NO:6 + Probe No. 2} were much less stable in the presence of 15 mM MgCl2 and 15 mM MnCl2, yielding a 15% decrease in fluorescence compared to that emitted by Probe No. 2 alone after a one-hour incubation {Fig. 5D) .<br>
Similarly, DNA triplexes that resulted in a 1 bp G-T mismatch {SEQ ID NO:7 + Probe No. 2), a 1 bp C-A mismatch (SEQ ID NO: 9 + Probe No. 2) , a 1 bp G-A mismatch (SEQ ID NO: 10 + Probe No. 2) and a consecutive 2 bp A-G and C-T mismatch (SE@ ID NO: 11 + Probe No. 2) were all much less stable than the perfectly matched DNA triplex (data not shown) . The 1 bp G-A mismatched DNA triplex that formed relatively easily in the presence of 10 mM MnCl2 or 10 mM MgCl? and 10 mM MriCl2, was now disrupted in the presence of 15 mM MgCl2 and 15 mM MnCl2_ producing only a 7% reduction in fluorescence compared to that observed with Probe No. 2 alone (data not shown) . When Probe No. 2 (containing a 53% GC content) was reacted with the dsDNA target SEQ ID NO: 3 (containing a 33% GC content) , a 3% increase in fluorescence was observed compared to that obtained by Probe No. 2 alone (Fig. 5D) , indicative of no DNA triplex<br><br>
formation. This result was expected considering this probe and target combination would result in a 5 bp mismatch.<br>
Treatment with 15 mM MgCl2 and 15 mM MnCl2 for 22 hours, generated a 76% and a 44% decrease in fluorescent intensity for dsDNA-.ssDNA-F triplexes containing perfectly complementary sequences (SEQ ID NO: 5 4 Probe No. 2) and a 1 bp T-G mismatch (SEQ ID NO:6 + Probe No. 2), respectively, compared to that obtained with Probe No. 2 alone (Fig. 5E).<br>
Collectively, the above Examples demonstrated that the addition of at least one type of cation to a hybridization medium promotes DNA triplex formation between dsDNA targets and fluorescent ly- labeled ssDNA probes, possessing dramatically different percent GC contents, to allow accurate and reliable discrimination between perfectly complementary sequences and those containing various mutations.<br>
While the invention has been described in detail and with reference to specific examples thereof, it will be apparent to one skilled in the art that various changes and modifications can be made therein without departing from the spirit and scope thereof.<br><br><br><br>
We claim:<br>
1.	A triplex complex comprising a single-stranded probe bound to a double-stranded nucleic<br>
acid target, wherein: (a) said probe comprises a heteropolymeric nucleic acid or a single stranded heteropolymeric nucleic acid analog, (b) said triplex complex comprises base triplets, (c) all said base triplets of said complex are members selected from the group consisting of A-T-A, T-A-T, U-A-T, T-A-U, A-U-A, U-A-U, G-C-G and C-G-C, (d) and wherein at least one of said base triplets is A-T-A, A-U-A or G-C-G, and (e) at least one other of said base triplets is T-A-T, U-A-T, T-A-U, U-A-U or C-G-C.<br>
2.	The complex as claimed in claim 1, wherein a pH of a medium in which said complex is present is greater than 7.6.<br>
3.	The complex as claimed in claim 1, wherein said single-stranded nucleic acid or nucleic acid analog is 5 to 30 bases long and said double-stranded nucleic acid target is 8 to 3.3 X 109 base pairs long.<br>
4.	The complex as claimed in claim 1, wherein said target sequence contains 25% to 75% purine bases and 75% to 25% pyrimidine bases in any order.<br>
5.	The complex as claimed in claim 1, wherein said probe is covalently bound to a double-stranded nucleic acid cleaving agent.<br>
6.	The complex as claimed in claim 1, wherein said probe is covalently bound to a chemotherapeutic agent.<br>
7.	The complex as claimed in claim 1, wherein said probe is covalently bound to a label.<br>
8.	The complex as claimed in claim 7, wherein said label is a multi-molecule signaling complex, a redox pair, a chemiluminescent agent or an electrochemiluminescent agent.<br>
9.	The complex as claimed in claim 7, wherein said label is a fluorophore.<br>
10.	The complex as claimed in claim 9, wherein a fluorescent intensity of said complex is directly correlated with a binding affinity between said probe and said target sequence.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUFic3RyYWN0LSgyNC0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Abstract-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUFzc2lnbm1lbnQtKDE2LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Assignment-(16-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUNsYWltcy0oMTYtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Claims-(16-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUNsYWltcy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Claims-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWNvbXBsZXRlIHNwZWNpZmljYXRpb24gKGdyYW50ZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjEtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Correspondence Others-(21-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Correspondence-Others-(16-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Correspondence-Others-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy0oMjgtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-correspondence-others-(28-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMjQtMDctMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-description (complete)-24-07-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LURyYXdpbmdzLSgyNC0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Drawings-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUZvcm0tMi0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Form-2-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUZvcm0tMy0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-Form-3-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUdQQS0oMTYtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-GPA-(16-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUdQQS0oMjEtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-GPA-(21-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1ERUxOUC0yMDA0LUdQQS0oMjQtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-DELNP-2004-GPA-(24-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBjdC1yZXF1ZXN0IGZvcm0ucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-pct-request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY1Ni1kZWxucC0yMDA0LXBldGl0aW9uLTEzNy0oMjgtMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1656-delnp-2004-petition-137-(28-07-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="235109-a-veterinary-formulation-for-treatment-and-prophylaxis-of-diseases-and-infections-of-pigs-and-poultry.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="235111-a-pharmaceutical-composition-for-treating-systemic-lupus-erythematosus-sle.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>235110</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1656/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Jun-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INGENEUS CORPORATION,</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHANCERY HOUSE, HIGH STREET, BRIDGETOWN,, BARBADOS</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ERIKSON GLEN HENRYK</td>
											<td>P.O. BOX 799, PROVIDENCIALES, TURKS &amp; CAICOS ISLANDS, BRITISH WEST INDIES</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DAKSIS JASMINE</td>
											<td>36 CARTIER CRESCENT, RICHMOND HILL, ONTARIO L4C 2NC, CANADA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PICARD PIERRE</td>
											<td>18 PIXLEY PRIVATE, OTTAWA, ONTARIO, KIG 4CG, CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB01/01538</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-07-09</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/613,263</td>
									<td>2000-07-10</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/235110-a-triplex-complex by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:36:59 GMT -->
</html>
