<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/235334-antibodies-to-vla-i by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:44:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 235334:&quot;ANTIBODIES TO VLA-I&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ANTIBODIES TO VLA-I&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A recombinant anti-VLA-1 antibody or antigen-binding fragment thereof whose light chain complementarity determining regions are defined by amino acid residues 24 to 33, 49 to 55, and 88 to 96 of SEQ ID NO: 1, and whose heavy chain complementarity determining regions are defined by amino acid residues 31 to 35, 50 to 65, and 98 to 107 of SEQ ID NO: 2.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ANTIBODIES TO VLA-1<br>
FIELD OF THE INVENTION<br>
This invention relates to antibodies to VLA-1 integrin and the use of these antibodies in treating inflammatory diseases and other immunological disorders.<br>
This invention also relates to the crystal structure of the complex between one such antibody and the α1-I domain of VLA-1, and to the use of this structural information for computational drug design.<br>
BACKGROUND OF THE INVENTION<br>
Integrins are a superfamily of cell surface receptors that mediate cell-cell and cell-matrix adhesion. These proteins are known to provide anchorage as well as signals for cellular growth, migration and differentiation during development and tissue repair. They have been implicated in immune and inflammatory processes.<br>
Integrins are heterodimeric proteins composed of two noncovalently linked polypeptide chains, α and β. The amino terminus of each chain forms a globular head that contributes to interchain linking and to ligand binding. The globular heads are connected to the transmembrane segments by stalks. The cytoplasmic tails are usually less than 50 amino acid residues long. Integrin subfamilies were originally defined on the basis of which P subunit was used to form the heterodimers. The βI-containing integrins are also called VLA molecules, referring to "very late activation" antigens. VLA-1 to VLA-6 refer to βI subfamily members containing αl to α6 (i.e., CD49a to CD49f), respectively. For general<br>
review, see Cellular and Molecular Immunology, eds. Abul K. Abbas et al., W.B.<br>
Saunders Company, Philadelphia, PA, 2000.<br>
Collagen (both types I and IV) and laminin are known ligands of al β1 integrin (i.e., VLA-1). VLA-1 has been implicated in cell adhesion and migration on collagen (Keely et al., 1995, J. Cell Sci. 108:595-607; and Gotwals et al., 1996, J. Gin. Invest. 97:2469-2477); in promoting contraction and reorganization of collagen matrices, a critical component of wound healing (Gotwals et al., supra; and Chiro, 1991, Cell 67:403-410); and in regulating the expression of genes involved in extracellular matrix remodeling (Riikonen et al., 1995, .7. Biol. Chem. 270:1-5; and Langholzet al., 1995, J. Cell Biol. 131:1903-1915). Thus, improper regulation of VLA-1 may result in certain pathological conditions such as fibrosis.<br>
Moreover, it has been suggested that VLA-1 may play a role in T cell /monocyte-driven diseases. Anti-VLA-1 antibodies block T-cell dependent cytokine expression (Miyake et al., l993,J.Exp.Med. 177:863-868). Expression of VLA-1 is increased in persistently activated, 2 to 4 week old cultured T cells (Hemler et al.,<br>
1985,	Eur. J. Immunol 15:502-508). VLA-1 is also expressed on a high percentage of<br>
T cells isolated from the synovium of patients with rheumatoid arthritis (Hemler et al.,<br>
1986,	J. Clin. Invest. 78:692-702).<br>
Several crystal structures of integrin a sub units have been determined, including the structures of the α2-I domain of α2βl (PDB accession code laox; Emsley et al., 1997, J. Biol. Chem. 272:28512-28517); the al-I domain of rat α1βl (PDB accession number Ick4; NoJte et al., 1999, FEES Lett. 452:379-385; WO 00/20459); the al subunit of human dpi (PDB accession code Iqc5; Rich et al., 1999,/. Biol. Chem. 274:24906-24913); the aL-I and aM-I domains; and vWF-A3 (Lee et al., 1995, Cell 80:631-635; Lee et ai, 1995, Structure 3:1333-1340; Qu et al., 1995, Proc. Natl. Acad. Sci. USA 92:10277-10281; Qu et al., 1996, Structure 4:931-942). The α2βl structure revealed a helix (i.e., the C-helix) that created a trench or groove on one face of the protein at the metal-ion binding site (Emsley et al., supra). The crystal structure of the α 2-I domain complexed to a short collagen-based triple helical peptide revealed that the collagen-based peptide was bound to that trench, where the α2-I amino acids that made intermolecular or metal contacts included<br>
Asp 151, Asnl54, Tyrl57, Gln215, Asp219, Leu220, Thr221, Asp254, Glu256, His25R, Tyr2R5, Tgii.286, Ash289,- Leu291, Asn295, and T.ys298 (PDB acceasssion code Idzi; Emsley et al., 2000, Cell 101:47-56; WO 01/73444). The amino acid numbering immediately above is based on PDB accession code Idzi and herein referred to as "crystal numbering." The crystal structures of the rat and human αI-I domains revealed a similar trench.<br>
SUMMARY OF THE INVENTION<br>
The present invention provides anti-VLA-1 antibodies and methods of using these antibodies to treat a variety of inflammatory and immunological disorders.<br>
Specifically, the invention embraces an antibody that specifically binds to VLA-1 (e.g., human VLA-1). This antibody contains light chain complementarity determining regions ("CDR"s) defined by amino acid residues 24 to 33, 49 to 55, and 88 to 96 of SEQ ID NO:1, and/or heavy chain complementarity determining regions defined by amino acid residues 31 to 35, 50 to 65, and 98 to 107 of SEQ ED N0:2. These CDRs may contain mutations (e.g., deletions, insertions and/or substitutions) in the non-antigen-contacting portions, as determined from the crystal structure disclosed herein, without affecting the VLA-1-binding activity of the antibody. Exemplary mutations are S24N, G92S and D101A in the light chain CDRs, and G55S in the heavy chain CDR2. In one embodiment, the antibody of this invention contains a light chain variable domain sequence of SEQ ID NO:1 and/or a heavy chain variable domain sequence of SEQ ID N0:2.<br>
In a related embodiment, the antibody of this invention contains the same heavy and light chain polypeptide sequences as an antibody produced by hybridoma mAQC2, deposited on April 18, 2001 at the American Type Culture Collection ("ATCC"), 10801 University Boulevard, Manassas, VA 20110-2209 and having ATCC accession number PTA3273. (All ATCC deposits recited herein were made under the Budapest Treaty). This antibody can be produced by, for example, hybridoma mAQC2, or cells containing nucleic acid sequences isolated from that hybridorna that encode the heavy and light chains of the mAQC2 monoclonal antibody.<br>
In another embodiment, the antibody is a humanized antibody comprising at least one (e.g., 2, 3.4, or 5)<br>
Ql, L4, P46, W47 and Y71; or at least one (e.g., 2, 3,4, 5, 6 or 7) of the following residues in its heavy chain: Dl, V12, S28, F29, A49, T93, R94 (Kabat numbering convention). For instance, the antibody comprises Ql, L4 and Y71 in the light chain; and/or (i) F29, A49, T93 and R94, or (ii) A49'and T93, in the heavy chain.<br>
The humanized antibody of this invention may contain a light chain variable domain sequence defined by amino acid residues 1 to 106 of SEQ ID N0:3, and/or a heavy chain variable domain sequence defined by amino acid residues 1 to 118 of SEQ ID NO:4. The humanized antibody may comprise the same heavy and/or light chain polypeptide sequences as an antibody produced by cell line hAQC2 (ATCC accession number PTA3275; deposited on April 18, 2001).<br>
In another embodiment, the humanized antibody of this invention may contain a mutation (e.g., deletion, substitution or addition) at one or more (e.g., 2, 3, 4, 5, 6, 7 or 8) of certain positions in the heavy chain such that an effector function of the antibody (e.g., the ability of the antibody to bind to a Fc receptor or a complement factor) is altered without affecting the antibody's ability to bind to VLA-1 (U.S. Patent 5,648,260). These heavy chain positions include, without limitation, residues 234, 235, 236, 237, 297, 3 1 8, 320 and 322 (EU numbering system). The humanized antibody can, for instance, contain the mutations L234A (i.e., replacing leucine at position 234 of an unmodified antibody with alanine) and L235A (EU numbering system) in its heavy chain. In one related embodiment, the antibody comprises the same heavy chain polypeptide sequence as an antibody produced by cell line hsAQC2 (ATCC accession number PTA3356; deposited on May 4, 2001).<br>
In yet another embodiment, the humanized antibody of this invention may contain a mutation (e.g., deletion or substitution) at an amino acid residue that is a site for glycosylation, such that the glycosylation site is eliminated. Such an antibody may be clinically beneficial for having reduced effector functions or other undesired functions while retaining its VLA-1 binding affinity. Mutations of glycosylation sites can also be beneficial for process development (e.g., protein expression and purification). For instance, the heavy chain of the antibody may<br>
contain the mutation N297Q (EU numbering system) such that the heavy chain can no longer be glycosylated at this site. In one related embodiment, the humanized antibody may comprise the same heavy chain polypeptide sequence as an antibody, produced by cell line haAQC2 (ATCC accession number PTA3274; deposited on April 18,2001).<br>
In still other embodiments, the heavy and/or light chains of the antibody of this invention contain mutations that increase affinity for binding to VLA-1 and thereby increase potency for treating VLA-1 -mediated disorders.<br>
Embraced in this invention are also a composition containing an antibody of the invention and a pharmaceutically acceptable carrier; an isolated nucleic acid containing a coding sequence for SEQ ID N0:l; an isolated nucleic acid containing a coding sequence for SEQ ID N0:2; an isolated nucleic acid containing a coding sequence for the light chain of an antibody produced by hybridoma mAQC2; an isolated nucleic acid containing a coding sequence for the heavy chain of an antibody produced by hybridoma mAQC2; an isolated nucleic acid containing a coding sequence for the light chain of an antibody produced by cell line hAQC2; an isolated nucleic acid containing a coding sequence for the heavy chain of an antibody produced by cell line hAQC2; an isolated nucleic acid containing a coding sequence for the heavy chain of an antibody produced by cell line haAQC2; an isolated nucleic acid containing a coding sequence for the heavy chain of an antibody produced by cell line hsAQC2; an isolated nucleic acid containing a coding sequence for residues 1 to 106 of SEQ ID NO:3; an isolated nucleic acid containing a coding sequence for residues 1 to 118 of SEQ ID NO:4; cells of hybridoma mAQC2; cells from cell line hAQC2; cells from cell line haAQC2; and cells from cell line hsAQC2.<br>
The present invention also provides a method of treating a subject with an immunological disorder mediated by VLA-1, including administering to the subject an effective amount of an antibody of this invention. For instance, this method is used to treat a human subject to palliate, ameliorate, stabilize, reverse, prevent, slow or delay progression of the disorder. Alternatively, this method is used prophylactically to treat a human subject at risk for developing this immunological disorder so as to<br>
prevent or delay the onset of the disorder. An "effective amount" of the composition<br>
can be administered in one or more dosages.<br>
VLA-1 mediated immunological disorders include, but are not limited to, disorders in which the VLA-1 activity level is elevated in one or more tissues as compared to a normal subject. Examples of such disorders are skin related conditions (e.g., psoriasis, eczema, burns, dermatitis, and abnormal proliferation of hair follicle cells), fibrosis (e.g., kidney or lung fibrosis), allergic rhinitis, respiratory distress syndrome, asthma, bronchitis, tendinitis, bursitis, fever, migraine headaches, gastrointestinal conditions (e.g., inflammatory bowel disease, Crohn's disease, gastritis, irritable bowel syndrome, colitis and colorectal cancer), vascular diseases (e.g., atherosclerosis), periarteritis nodosa, thyroiditis, aplastic anemia, Hodgkin's Disease, rheumatic fever, osteoarthritis, autoimmune diseases (e.g., type I diabetes, myasthenia gravis, rheumatoid arthritis, systemic lupus erythematosus, and multiple sclerosis), sarcoidosis, nephrotic syndrome, renal failure, Bechet's Syndrome, polymyositis, gingivitis, hypersensitiviry (e.g., delayed type hypersentivity or immediate hypersensitiviry), graft and transplant rejections, graft versus host disease (GVHD), conjunctivitis, swelling occurring after injury, myocardial ischemia, and endotoxin shock syndrome.<br>
The present invention also provides a method of determining the level of VLA-1 in a tissue (e.g., tissue specimen and body fluid) comprising contacting the tissue (e.g., in vivo-or in vitro) with the antibody of the invention, and then detecting the binding of the antibody to the tissue, thereby determining the level of VLA-l in the tissue.<br>
As used herein, the antibody of this invention can be, for instance, a murine antibody, a humanized antibody, or a chimeric antibody. It can be a whole antibody (i.e., with two full length light chains and two full length heavy chains) of any isotype and subtypes (e.g., IgM, IgD, IgG1, IgG2, IgG3, IgG4, IgE, IgA1, and IgA2; with either kappa or lambda light chain). Alternatively, the antibody of this invention refers to an antigen-binding fragment (e.g., Fab, F(ab')2, and single chain Fv) of a whole antibody.<br>
The present invention further provides crystallizable compositions and crystals of complexes formed by a rat-human chimeric αl-I domain (mutant RAH) and a hAQC2 Fab fragment, and methods for using such compositions and crystals. This invention also provides the structure coordinates and binding sites of the chimeric domain and the hAQC2 Fab fragment. The atomic coordinates derived from the crystal structure described herein provide a structural basis for the biological activities of hAQC2 as well as a basis for rational design of VLA-1 agonists or antagonists with predicted biological activities (e.g., small molecule compounds or antibodies such as hAQC2 variants).<br>
The crystal structure disclosed herein is the first crystal structure of an α1-I domain of an α1βl  integrin/ Fab complex. This structure shows the residues critical for Fab binding by al-I domain. In addition, the Fab binds in the putative' collagen-binding site and inhibits collagen binding. Amino acid residues found in the binding site on the al-I domain include Aspl54, Serl56, Asnl57, Serl58, Tyrl60, Glul92, Glu218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering). Residues on the Fab fragment found to bind to the al-I domain include light chain residues Asn30, Tyr48, Trp90, Ser91, Asn93 and Trp95, and heavy chain residues Ser30, Arg31, Trp47, Ser52, Gly53, His56, Tyr58, Phe99, GlylOO and AsplOl (crystal numbering).<br>
This invention also provides a computer for producing a three-dimensional representation of a molecular complex, where the molecular complex is defined by the set of structure coordinates of a complex of a chimeric I domain of an al pi integrin RAH and humanized antibody hAQC2, according to Fig. 19; or a homologue of the molecular complex, the homologue having a root mean square deviation from the backbone atoms of the ammo acids of not more than 0.65 A. The computer includes a machine-readable data storage medium including a data storage material encoded with machine-readable data, where the data contains at least a portion of the structure coordinates of the complex according to Fig. 19; a working memory for storing instructions for processing the machine-readable data; a central-processing unit coupled to the working memory and to the machine-readable data storage medium for processing the machine readable data into the three-dimensional<br>
representations; and a display coupled to the central-processing unit for displaying the<br>
This invention further provides a computer for producing a three-dimensional representation of a molecule or molecular complex including a binding site defined by structure coordinates of hAQC2 amino acids including at least seven (e.g., 7, 8, 9, 10, 11, 12, 13, 14, 15, or 16) of light chain residues Asn30, Tyr48, Trp90, Ser913 Asn93 and Trp95, and heavy chain residues Ser30, Arg31, Trp47, Ser52, Gly53, His56, Tyr58, Phe99, Gly100and Asp 101 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of the hAQC2 amino acids of not more than 1.10 A. This invention also provides a computer for producing a three-dimensional representation of: a binding site defined by structure coordinates of hAQC2 amino acids including at least seven (e.g., 7, 8, 9, 10, 1 1, 12, 13, 14, 15, or 16) of light chain residues Asn30, Tyr48, Trp90, Ser91, Asn93 and Trp95, and heavy chain residues Ser30, Arg31, Trp47, Ser52, Gly53, His56, Tyr58, Phe99, GlylOO and AsplOl (crystal numbering), according to Fig. 19; a binding site of a homologue that has a root mean square deviation from the backbone atoms of the hAQC2 amino acids of not more than 1.10 A.<br>
This invention also provides a method for identifying an inhibitor of an I domain of an integrin including the steps of using structure coordinates of hAQC2 amino acids including at least seven (e.g., 7, 8, 9, 10, 11, 12, 13, 14, 15, or 16) of light chain residues Asn30, Tyr48, Trp90, Ser91 , Asn93 and Trp95, and heavy chain residues Ser30, Arg31, Trp47, Ser52, Gly53, His56, Tyr58, Phe99, GlylOO and Aspl0l (crystal numbering), according to Fig. 19 or ± a root mean square deviation from the backbone atoms of the hAQC2 amino acids not more than 1.10 A, to generate a three-dimensional structure of a binding site; employing the three-dimensional structure to design or select a potential antagonist; synthesizing the potential antagonist; and contacting the potential antagonist with hAQC2 to determine the ability of the potential antagonist to interact with hAQC2, where the ability of the potential antagonist to interact with hAQC2 indicates that the potential antagonist is<br>
an inhibitor of the I domain. This invention further provides an inhibitor of I domain of integrin identified by this method.<br>
This invention also provides a computer for producing a<br>
three-dimensional representation of a molecule or molecular complex including: a binding site defined by structure coordinates of I domain amino acid residues Asp 154, Serl56, Asnl57, Serl58, Tyrl60, Glul925 Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a second binding site that has a root mean square deviation from the backbone atoms of the I domain amino acids not more than 0.65 A. This invention also provides a computer for producing a three-dimensional representation of: a first binding site defined by structure coordinates of I domain amino acids residues Aspl54, Serl56, Asnl57, Serl58, Tyrl60, Glul92, Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19; or a binding site of a homologue that has a root mean square deviation from the backbone atoms of the I domain amino acids not more than 0.65 A.<br>
This invention also provides a computer for producing a<br>
three-dimensional representation of a molecule or molecular complex including: a binding site defined by structure coordinates of I domain amino acids including at least three of residues Glul92, Gln218, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a second binding site that has a root mean square deviation from the backbone atoms of the I domain amino acids not more than 1.0 A. The invention further provides a computer for producing a three-dimensional representation of a binding site defined by structure coordinates of I domain ammo acids including at least three of residues Glul92, Gln218, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19; or a binding site of a homologue that has a root mean square deviation from the backbone atoms of the I domain amino acids not more than 1.0 A-<br>
This invention further provides methods for using these three-dimensional representations to design chemical entities that associate with the chimeric domain or the hAQC2 Fab fragment, or portions thereof; and act as potential inhibitors of the chimeric domain or the hAQC2 Fab fragment, or portions thereof. This invention also relates to compositions including chemical entities, such as inhibitors and variants of the chimeric domain or variants of the hAQC2 Fab fragment, where such chemical entities and variants are rationally designed by means of the structure coordinates of the chimeric domain or the hAQC2 Fab fragment, or binding sites. The invention further relates to use of the above-identified chemical .entities to treat or prevent conditions associated with inappropriate or abnormal α1β1 activity in a subject.<br>
This invention further provides a method for identifying an inhibitor of an I domain of an integrin including the steps of using the structure coordinates of I domain amino acids residues Aspl54, Serl56, Asnl57, Serl58, Tyrl60, Glul92, Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19, to generate a three-dimensional structure of a binding site; employing the three-dimensional structure to design or select a potential antagonist; synthesizing the potential antagonist; and contacting the potential antagonist with I domain to determine the ability of the potential antagonist to interact with I domain, where the ability of the potential antagonist to interact with the I domain indicates that the potential antagonist is an inhibitor of the I domain.<br>
This invention also provides a method for identifying an inhibitor of an I domain of an integrin including the steps of using the structure coordinates of at least three of,I domain amino acids including residues Glu 192, Gln218, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19, or ± a root mean square deviation from the backbone atoms of the I domain amino acids not more than 0.65 A, to generate a three-dimensional structure of a binding site; employing the three-dimensional structure to design or select a potential antagonist; synthesizing the potential antagonist; and contacting the potential antagonist with I domain to determine the ability of the potential antagonist to interact with I domain of integrin,<br>
where the ability of the potential antagonist to interact with the I domain indicates that the potential antagonist is an inhibitor of the I domain. This invention also provides an inhibitor of I domain of integrin identified by this method.<br>
Other features and advantages of the invention will be apparent from the following detailed description, and from the claims.<br>
BRIEF DESCRIPTION OF THE DRAWINGS Figure 1. Collagen-binding integrins α1βl  and α2βl  on activated leukocytes. (A). Flow cytometric analysis of al and a2pl integrin expression on lL-2-activated splenocytes (d 11). Cells were labeled with either anti-al mAb, anti-a2 mAb, or non-binding control mAb (grey lines), and followed by FITC-anti-hamster irnmunoglobulin. (B) Effect of anti-al and anti-a2 mAbs on leukocyte adhesion to collagen. 105 IL-2 activated splenocytes were treated with indicated mAbs for 15 min before plating onto either type IV or type I collagen-coated wells for 1 h at 37°C. Adhesion was calculated as illustrated in Example 1, and expressed as % adhesion relative to control mAb-treated cells. Adhesion assays were done in triplicate, and at least three independent experiments were performed. One representative experiment is shown.<br>
Figure 2. Effect of anti-integrin mAbs on the effector phase of delayed-type hypersensitivity. SRBC-sensitized mice were injected i.p. with the indicated mAbs 1 h prior to SRBC challenge. Footpad thickness was measured 20 h after antigen challenge, and results shown as % increase in footpad thickness ± SEM as illustrated in Example 2. These data represent a summary of eight experiments with n = 79 (PBS), 68 (control hamster Ig), 68 (anti-al), 29 (anti-a2), 18 (anti-al + anti-a2), 45 (anti-a4), 18 (anti-a5), 20 (anti-a6), and 10 (anti-pi). The mAbs used were: Ha4/8 (control hamster Ig group 2), Ha31/8 (anti-al), Hal/29 (anti-o2), PS/2 (anti-a4), 5H10-27 (anti-a5), GoH3 (anti-a6), and HMβl-1 (anti-pi).<br>
Figure 3.  Effect of anti-integrin mAbs on the effector phase of contact hypersensitivity. FITC-sensitized mice were injected i.p. with the indicated mAbs 4 h prior to FTTC challenge. Ear thickness was measured at baseline and 24 h later, and results shown as % increase in ear thickness ± SEM as illustrated in Example 3.<br>
These data represent a summary of nine experiments with n = 74 (PBS), 60 (control<br>
-hamster Ig), 26 (anti-ICAM-1), 44 (anti-all 44 (anti-α 2), 38 (anti-αl + anti-α2). 36<br>
(anti-α4), 16 (anti-α5), 26 (anti-α4 + anti-α5), 24 (anti-α6), and 22 (anti-β1). The<br>
hamster mAbs used were: Ha4/8 (control hamster Ig group 2), Ha31/8 (anti-al),<br>
Hal/29 (anti-α2), HMp 1-1 (anti-β1), 3E2 (anti-ICAM-1); the rat mAbs used were:<br>
R35-95 and R35-38 (control rat IgG2a and rat IgG2b, respectively), PS/2 (anti-α4),<br>
5H10-27 (anti-α5), GoH3 (anti-α6).	<br>
Figure 4. Contact hypersensitivity responses in α1-deficient mice compared to wild-type mice. FITC-sensitized mice were injected i.p. with indicated mAbs 4 h prior to FITC challenge. Ear thickness was measured at baseline and 24 h later, and results shown as % increase in ear thickness ± SEM as illustrated in Example 4. Groups of four to five mice per condition were used, and all experiments were performed a minimum of three times. One representative experiment is shown.<br>
Figure 5. Effect of anti-α1 and anti-α2 mAbs on croton oil-induced non-specific inflammation. Mice were injected i.p. with indicated mAbs 4 h prior to ear painting with croton oil. Ear thickness was measured at baseline and 24 h later, and results shown as % increase in ear thickness ± SEM as illustrated in Example 5. Groups of four to five mice per condition were used, and all experiments were performed a minimum of three times. One representative experiment is shown.<br>
Figure 6. Effect of anti-αl and α2 mAbs in collagen mAb-induced arthritis. Mice were injected i.p. with anti-collagen mAbs at d 0, followed by LPS on day 3. Mice were injected i.p. with indicated mAbs every 3M day starting on d 0. Clinical arthritis was apparent 2-3 d following LPS injection and continued for several weeks. Each limb was evaluated on a 0 to 4 scale every 3rd day as illustrated in Example 6 and results are expressed as the mean arthritic score between d 9 and d 15 (± SEM) of all four limbs. These data represent a summary of four experiments with each experiment consisting of groups of three to four mice per condition.<br>
.   Figure 7. Effect of anti-al and a2 mAbs in collagen mAb-induced arthritis. A. Preventative treatment of mice with either anti-αl or anti-α2 mAb decreases arthritic score. Mice were treated with anti-collagen mAbs at d 0, followed by LPS on d 3. Arthritis was apparent by d 6 and continued for several weeks. Mice<br>
were treated with the indicated mAbs every 3rd day starting on d 0. Each limb was evaluated and scored on a 0 to 4 scale every 3rd day. Results are xpressed as the mean arthritic score between d 9 and d 15 (± SEM) of all four limbs (maximum score of 16). Groups of 4 mice per condition were used; the average of 12 experiments is shown. B. αl-deficient mice have a reduced arthritic score comparable to anti-al mAb-treated wild-type mice. Experimental details and scoring are as outlined above. Groups of 4 mice per condition were used; the average of 2 experiments is shown.<br>
Figure 8. Development of arthritis is delayed in the absence of lymphocytes and inhibition of arthritis by anti-αl mAb occurs in the absence of lymphocytes. Wild-type B6,129 or RAG-1-deficient B6,129 mice were treated with anti-collagen mAbs at day 0, followed by LPS on day 3. Arthritis was apparent by day 6 and continued for several weeks. Mice were treated with the indicated mAbs every 3rd day starting on day 0. Each limb was evaluated and scored on a 0 to 4 scale every 3rd day. Results are expressed as the mean arthritic score per limb (maximum score of 4). Groups of 4 mice per condition were used.<br>
Figure 9. Dose response of anti-al mAb inhibition of arthritis. Wild-type Balb/c mice were treated with anti-collagen mAbs at day 0, followed by LPS on day 3. Arthritis was apparent by day 6 and continued for several weeks. Mice were treated i.p. with the indicated dose of either Ha4/8 (isotype control) or Ha31/8 (anti-al) mAbs every 3rd day starting on day 0. Each limb was evaluated and scored on a 0 to 4 scale every 3rd day. Results are expressed as the mean arthritic score per limb (maximum score of 4). Groups of 4 mice per condition were used.<br>
Figure 10. Therapeutic treatment with anti-al mAb can decrease arthritic score. Wild-type Balb/c mice were treated with anti-collagen mAbs at day 0,-followed by LPS on day 3. Arthritis was apparent by day 6 and continued for several weeks. Mice were treated i.p. with mAbs (250 µg) or Ig fusion protein (200µg) every 3rd day starting on day 4. Mice received either mAb (Ha4/8 isotype control or Ha31/8 anti-αl), Ig fusion protein (Isotype control Ig or TNF-R55-Ig) or a combination of both (250 ug Ha31/8 and 200 ug TNF-R55-Ig). Each limb was evaluated and scored on a 0 to 4 scale every 3rd day. Results are expressed as the mean arthritic score per limb (maximum score of 4). Groups of 4 mice per condition were used.<br>
-14-<br>
Figure 11. Location of the Epitope for the anti-αI I domain Blocking mAbs. A. Amino acid sequence of the rat (top; SEQ ID NO:63) and human (below; residues 91-97 of SEQ ID NO:64) al-I domain. The residues that comprise the MIDAS (metal ion dependent adhesion site) motif are shown in bold. The human amino acids that replaced the corresponding rat residues (RAH) are shown below the rat sequence in the boxed region. For clarity, residue numbering in the text refers to this figure,, unless otherwise designated, e.g., as crystal numbering. B. Increasing concentrations of mAb AJH10 (ATCC No. PTA-3580; deposited under the Budapest Treaty with the American Type Culture Collection, Manassas, VA, USA on August 2, 2001) were bound to plates coated with 30 µg/ml human (circles), rat (triangles) or RAH (squares) al-I domain. Data shown is representative of three experiments.<br>
Figure 12. Amino acid sequence of the human αl-I domain (SEQ ID NO:64).<br>
Figure 13. Identification of a blocking mAb to the al-I domain. A. Increasing concentration of mAbs AEF3 (triangles) or AJH10 (circles) were bound to plates coated with 30 fig/ml αl-I domain. B. The al-I domain was treated with increasing concentrations of mAb AJH10 (diamonds) or mAb BGC5 (squares) and bound collagen IV (2 ug/ml) coated plates. C. K562-αl cell were treated with increasing concentration of mAbs AEF3 (triangles) or AJH10 (circles) and bound to collagen IV (5 µg/ml) coated plates. 45-50% of cells added to each well adhered to collagen IV. Data shown is representative of three independent experiments.<br>
Figure 14. Species Cross-reactivity of the blocking mAbs analyzed by fluorescence activated cell sorter (FACS). Rabbit vascular smooth muscle cells were incubated with either mAb AJH10 (bottom) or murine IgG control (top) and analyzed by fluorescence activated cell sorter (FACS).<br>
Figure 15. The αl-1domain binds collagen. A. Increasing<br>
concentrations of the human al-I domain were bound to plates previously coated with 1µg/ml collagen I (squares) or collagen IV (circles). Values shown have been corrected for background binding to BSA. B. 2 µg/ml human al-I domain was mixed with increasing concentration of an anti- human αl integrin antibody 5E8D9 (squares) or an anti- human a2- integrin antibody A2IIE10 (circles), and then bound<br>
to plates previously coated with 1 fag/ml collagen IV. C. Plates were coated with 1 µg/ml collagen IV or 3% BSA. αl-I domain (2 µg/ml) was subsequenctly bound to coated plates plates in the presence of 1 mM Mn2+, 1 mM Mg2+, or 5 mM EDTA. Data shown is representative of three independent experiments.<br>
Figure 16. Characterization of Humanized AQC2 Forms. mAQC2 (triangles), chAQC2 (circles), hAQC2 (inverted triangles) and hAQC2' (squares) were evaluated.<br>
A.	Inhibition of VLA-1 binding to type IV collagen.<br>
B.	Inhibition of al-I domain binding to type IV collagen.<br>
C.	Binding to immobilized αl-I domain.<br>
D.	Competition with biotinylated mAQC2 for binding to immobilized<br>
αl-I domain.<br>
Figure 17. Characterization of Humanized AQC2 Forms by FACS. Figure 18. Characterization of Humanized AQC2 Forms by FACS. Figure 19. Atomic structure coordinates for the al-I domain/Fab complex, as derived by X-ray crystallography from crystals of that complex in Protein Data Bank (PDB) format. The coordinates of the two complexes in the asymmetric unit are listed as follows.<br>
Complex 1:     molecule A = I domain of integrin<br>
molecule H = heavy chain of hAQC2 Fab molecule L = light chain of hAQC2 Fab molecule M = Mn+2 Complex 2:     molecule B = I domain of integrin<br>
molecule X = heavy chain of hAQC2 Fab molecule Y = light chain of hAQC2 Fab molecule M = Mn+2<br>
Figure 20. I domain-Fab complex. A. Ribbon diagram of the I domain-Fab complex. The I domain and the antibody heavy and light chain are labeled. The Mn+2 ion is shown as a sphere. B. Close-up of the MIDAS (Metal-Ion-Dependent-Adhesion-Site) site showing the coordination of the metal ion (sphere) by Asp 101 (crystal numbering). The<br>
protein backbones are shown as ribbons and the side chains in the ball-and-stick representation. The cylinders represent interactions between the metal ion and protein atoms. The thin lines represent H-bonds. Fig. 20 was made with the software program RIBBONS (Carson, 1991, J. Appl. Cryst. 24:958-961).<br>
Figure 21. A diagram of a system used to carry out the instructions encoded by the storage medium of Figs. 22 and 23.<br>
Figure 22. A cross section of a magnetic storage medium.<br>
Figure 23. A cross-section of an optically-readable data storage medium.<br>
DETAILED DESCRIPTION OF THE INVENTION It is a discovery of the present invention that an antibody to an integrin (e.g., VLA-1) and fragment thereof, particularly, an αl-integrin subunit, can block the interaction of pro-inflammatory leukocytes with components of the extracellular matrix including, but not limited to collagens, laminin and fibronectin. This discovery illustrates the importance of adhesion molecules of the integrin family, particularly αlβ1, in the peripheral tissue environment during conditions related to inflammation. It also extends the role of integrins family and fragments thereof in inflammation beyond leukocyte attachment and extravasation at the endothelial interface by highlighting the importance of the matrix-rich peripheral tissue environment to immune responses and it reveals peripheral tissues as a new point of intervention for adhesion based therapies. I. Anti-Integrin Antibodies<br>
The methods of the present invention contemplate the use of antibodies to integrins where the integrins contemplated include molecules which comprise a P chain, including but not limited to β1, β2, β3, β4, β5, β6, β7, β8, non-covalently bound to an a chain, including but not limited to αl, α2, α3, α4, α5, α6, α7, α8, α9, α10, αV, αL, αM, αX, αD, αE, αIIb. Examples of the various integrins contemplated for use in the invention include, but are not limited to:<br>
αlβl, α2βl, α3βl, α4βl, α5βl, α6βl, α7βl, α8βl, α9βl, αl0βl αVβl, αLβl, αMβl, αXβl,αDβl,αIIbβl,αEβl<br>
αlβ2, α2β2, α3β2, α4β2, α5β2, α6β2, α7β2, α8β2, α9β2, αl0β2, αVβ2, αLβ2, αMβ2, αXβ2, αDβ2, αIIbβ2,αEβ2;<br>
αlβ3, α2β3,α3β3,α4β3,α5β3,α6β3, α7β3,α8β3,α9αβ3,α10β3, αVβ3, αLβ3, αMβ3, αXβ3, αDβ3, αIIbβ3, αEβ3;<br>
alp4, a2p4, a3p4, a4p4, cc5p4, a6p4, a?p4, a8p4, a9p4, a!0p4, aVf}4, aLp4, aMp4, aXp4 aDp4; allbp4, aEp4;<br>
αlβ5, α2β5, α3β5, α4β5, α5β5, α6β5, α7β5, α8β5, α9β5, α10β5, αVβ5, αLβ5, αMβ5, αXβ5, αDβ5, αIIbβ5, αEβ5;<br>
αlβ6, α2β6, α3β6, α4β6, α5β6, α6β6, α7β6, α8β6, α9β6, α10β6, αVβ6, αLβ6,αMβ6,αXβ6, αDβ6, αllbβ6, αEβ6;<br>
αlβ7, α2β7, α3β7, α4β7, α5β7, α6β7, α7β7, α8β7, α9β7, α10β7, αVβ'7, αLβ7, αMβ7, αXβ7, αDβ7, αIIbβ7, αEβ7;<br>
αlβ8, α2β8, α3β8, α4β8, α5β8, α6β8, α7β8, α8β8, α9β8, α10β8, αVβ8, αLβ8, αMβ8, αXβ8, αDβ8, αIIbβ8, αEβ8;<br>
The methods of the present invention also contemplate the use of antibodies to integrin fragments including for example antibodies to a P chain alone, including but not limited to β1, β2, β3, β4, β5, β6, β7, β8, as well as an a chain 20    alone, including but not limited to αl, α2, α3, α4, α5, α6, α7, α8, α9, α10, αV, αL, αM, αX, αD, αE, αIIb. In addition, the methods of the present invention further contemplate the use of antibodies to integrin fragments including for example<br>
antibodies to the I domain of the a chain, including but not limited to the I domain from alpl (Briesewitz et al., 1993, J. Biol. Chem. 268:2989); α2βl ( Takada and Hemler, 1989, J Cell Biol 109:397), αLβ2 (Larson et al., 1989, J Cell Biol 108:703), aMp2 (Corbi et al., 1988, J Biol Chem 263:12403),αXβ2 (Corbi et al., 1987, EMBO J6:4023), αDβ2 (Grayson et al., 1988, J Exp Med 188:2187), αEβ7 (Shaw et al., 1994, J Biol Chem 269:6016). In one embodiment, the αl-I domain antigenic determinant includes an amino acid sequence of at least 6 contiguous amino acids, wherein the contiguous sequence is found within the sequence of Fig. 12. In a related embodiment, the contiguous sequence is Val-Gln-Arg-Gly-Gly-Arg (residues 91-97 of SEQ ID NO:64).<br>
Methods for producing integrins for use in the present invention are known to those of skill in the art (see, e.g., Springer et al., 1990, Nature 346:425-434).<br>
Embodiments of the present invention further include anti-integrin polyclonal and monoclonal antibodies. Embodiments of the present invention include a monoclonal antibody such an anti-αl monoclonal antibody. Antibodies for treatment, in particular for human treatment, include human antibodies, humanized antibodies, chimeric antibodies, and antigen-binding fragments of whole antibodies such as Fab, Fab', F(ab')2 and F(v) antibody fragments. Some antibodies of this invention may also include proteins containing one or more immunoglobulin light chains and/or heavy chains, such as monomers and homo-or hetero-multimers (e.g., dimers or trimers) of these chains, where these chains are optionally disulfide-bonded or otherwise cross-linked. These antibodies may be capable of binding to one or more antigens (e.g., al, a2, a6 or alpha-I domain containing integrin subunits).<br>
An α1βl function blocking antibody as used herein refers to an antibody that binds to the αl-I domain, for example, residues 92-97 of Fig. 12, and blocks α1β1 function as tested, for example, by their ability to inhibit K562-al dependent adhesion to Collagen IV (see Example 15).<br>
The following describes the various methods of making the antibodies of this invention. Methods that are known in the art but not specifically described herein are also within the scope of this invention. For instance, antibodies of this<br>
invention can also be identified using phage-displayed antibody libraries, such as those described in Smith, 1985, Science 228:1315-7; U.S. Patents 5,565,332,<br>
5,733,743, 6,291,650, and 6,303,313. Additional antibodies of this invention can be made by coupling the heavy chains identified herein with a noncognate light chain, e.g., a light chain identified by phage display technology. II. Non-Human Hybridoma Antibodies<br>
The monoclonal antibodies of this invention can be generated by well known hybridoma technology. For instance, β1 -/- animals (e.g., mice, rats or rabbits) can be immunized wife purified or crude α1β1 preparations, cells transfected with cDNA constructs encoding α1,β1 or both antigens, cells that constitutively express α1,β1 and the like. The antigen can be delivered as purified protein, protein expressed on cells, protein fragment or peptide thereof, or as naked DNA or viral vectors encoding the protein, protein fragment, or peptide. Sera of the immunized animals are then tested for the presence of anti-α1β1antibodies. B cells are isolated from animals that test positive, and hybridomas are made with these B cells.<br>
Antibodies secreted by the hybridomas are screened for their ability to bind specifically to VLA-1 (e.g., binding to α1transfected cells and not to untransfected parent cells) and for any other desired features, e.g., having the desired CDR consensus sequences, inhibiting (or not inhibiting in the case of nonblockers) the binding between collagen and VLA-1.<br>
Hybridoma cells that test positive in the screening assays are cultured in a nutrient medium under conditions that allow the cells to secrete the monoclonal antibodies into the culture medium. The conditioned hybridoma culture supernatant is then collected and antibodies contained in the supernatant are purified. Alternatively, the desired antibody may be produced by injecting the hybridoma cells into the peritoneal cavity of an unimmunized animal (e.g., a mouse). The hybridoma cells proliferate in the peritoneal cavity, secreting the antibody which accumulates as ascites fluid. The antibody may then be harvested by withdrawing the ascites fluid from the peritoneal cavity with a syringe.<br>
The monoclonal antibodies can also be generated by isolating the antibody-coding cDNAs from the desired hybridomas, transfecting mammalian host cells (e.g., CHO or NSO cells) with the cDNAs, culturing the transfected host cells, and recovering the antibody from the culture medium.<br>
III.	Chimeric Antibodies<br>
The monoclonal antibodies of this invention can also be generated by engineering a cognate hybridoma (e.g., murine, rat or rabbit) antibody. For instance, a cognate antibody can be altered by recombinant DNA technology such that part or all of the hinge and/or constant regions of the heavy and/or light chains are replaced with the corresponding components of an antibody from another species (e.g., human). Generally, the variable domains of the engineered antibody remain identical or substantially so to the variable domains of the cognate antibody. Such an engineered antibody is called a chimeric antibody and is less antigenic than the cognate antibody when administered to an individual of the species from which the hinge and/or constant region is derived (e.g., a human). Methods of making chimeric antibodies are well known in the art. Human constant regions include those derived from IgGl and IgG4.<br>
IV.	Fully Human Antibodies<br>
The monoclonal antibodies of this invention also include fully human antibodies. They may be prepared using in vitro-primed human splenocytes, as described by Boemeret al., 1991, J. Immunol. 147:86-95, or using phage-displayed antibody libraries, as described in, e.g., U.S. Patent 6,300,064.<br>
Alternatively, fully human antibodies may be prepared by repertoire cloning as described by Persson et al., 1991 , Proc. Nat. Acad. Sci. USA 88: 2432-2436; and Huang and Stollar, 1991, J. Immunol. Methods 141: 227-236. In addition, U.S. Patent 5,798,230 (Aug. 25,1998) describes preparation of human monoclonal antibodies from human B cells, wherein human antibody-producing B cells are immortalized by infection with an Epstein-Barr virus, or a derivative thereof, that expresses Epstein-Barr virus nuclear antigen 2 (EBNA2), a protein required for immortalization. The EBNA2 function is subsequently shut off, resulting in an increase in antibody production.<br>
Some other methods for producing fully human antibodies involve the use of non-human animals that have inactivated endogenous Ig loci and are transgenic for un-rearranged human antibody heavy chain and light chain genes. Such transgenic animals can be immunized with α1β1, and hybridomas are then made from B cells<br>
derived therefrom. These methods are described in, e.g., the various Gen Pharm/Medarex (Palo Alto, CA) publications/patents concerning transgenic mice containing human Ig miniloci (e.g., Lonberg U.S. Patent 5,789;650); the various  Abgenix (Fremont, CA) publications/patents with respect to XENOMIGE (e.g., Kucherlapati U.S. Patents 6,075,181, 6,150,584 and 6,162,963f Green et al., 1994, Nature Genetics 7:13-21; and Mendez et al., 1997, Nature Genetics 15(2): 146-56); and the various Kirin (Japan) publications/patents concerning "transomic" mice (e.g., EP 843 961, and Tomizuka et al., 1997, Nature Genetics 16:133-1443). V. Humanized Antibodies<br>
The monoclonal antibodies of this invention also include humanized versions of cognate anti-α1β1 antibodies derived from other species. A humanized antibody is an antibody produced by recombinant DNA technology, in which some or all of the amino acids of a human immunoglobulin light or heavy chain that are not required for antigen binding (e.g., the constant regions and the framework regions of the variable domains) are used to substitute for the corresponding amino acids from the light or heavy chain of the cognate, nonhuman antibody. By way of example, a humanized version of a murine antibody to a given antigen has on both of its heavy and light chains (1) constant regions of a human antibody, (2) framework regions from the variable domains of a human antibody; and (3) CDRs from the murine antibody. When necessary, one or more residues in the human framework regions can be changed to residues at the corresponding positions in the murine antibody so as to preserve the binding affinity of the humanized antibody to the antigen. This change is sometimes called "back mutation." Humanized antibodies generally are less likely to elicit an immune-response in humans as compared to chimeric human antibodies because the former contain considerably fewer non-human components.<br>
The methods for making humanized antibodies are described in, e.g., Winter EP 239 400; Jones et al., 1986, Nature 321:522-525; Riechmann et al., 1988, Nature 332:323-327 (1988); Verhoeyen et al., 1988, Science 239:1534-1536; Queen et ai., 1989, Proc. Nat, Acad. Sci. USA 86:10029; U.S. Patent 6,180,370; and Orlandi et al., 1989, Proc. Natl Acad. Sci. USA 86:3833. Generally, the transplantation of murine (or other non-human) CDRs onto a human antibody is achieved as follows.<br>
The cDNAs encoding heavy and light chain variable domains are isolated from <br>
a. The DNA sequences of the variable domains, including the CDRs, are<br>
determined by sequencing. The DNAs encoding the CDRs are transferred to the corresponding regions of a human antibody heavy or light chain variable domain coding sequence by site directed mutagenesis. Then human constant region gene segments of a desired iso.type (e.g, γl for CH and k for CL) are added. The humanized heavy and light chain genes are co-expressed in mammalian host ceils (e.g., CHO or NSO cells) to produce soluble humanized antibody. To facilitate large scale production of antibodies, it is often desirable to produce such humanized antibodies in bioreactors containing the antibody-expressing cells, or to produce transgenic mammals (e.g., goats, cows, or sheep) that express the antibody in milk (see, e.g., U.S. Patent 5,827,690).<br>
At times, direct transfer of CDRs to a human framework leads to a loss of antigen-binding affinity of the resultant antibody. This is because in some cognate antibodies, certain amino acids within the framework regions interact with the CDRs and thus influence the overall antigen binding affinity of the antibody, hi such cases, it would be critical to introduce "back mutations" (supra) in the framework regions of the acceptor antibody in order to retain the antigen-b hiding activity of the cognate antibody.<br>
The general approach of making back mutations is known in the art. For instance, Queen et al. (supra), Co et al., 1991, Proc.Nat. Acad. Sci, USA 88:2869-2873, and WO 90/07861 (Protein Design Labs .Inc.) describe an approach that involves two key steps. First, the human V framework regions are chosen by computer analysis for optimal protein sequence homology to the V region framework of the cognate murine antibody. Then, the tertiary structure of the murine V region is modeled by computer in order to visualize framework amino acid residues that are likely to interact with the murine CDRs, and these murine ammo acid residues are then superimposed on the homologous human framework.<br>
Under this two-step approach, there are several criteria for designing humanized antibodies. The first criterion is to use as the human acceptor the framework from a particular human immunoglobulin that is usually homologous to<br>
the non-human donor irnmunoglobulin, or to use a consensus framework from many human antibodies. The second criterion is to use the donor amino acid rather than the acceptor if the human acceptor residue is unusual and the donor residue is typical for human sequences at a specific residue of the framework. The third criterion is to use the donor framework amino acid residue rather than the acceptor at positions immediately adjacent to the CDRs.<br>
One may also use a different approach as described in, e.g., Tempest, 1991, Biotechnology 9: 266-271. Under this approach, the V region frameworks derived from NEWM and REI heavy and light chains, respectively, are used for CDR-grafting without radical introduction of mouse residues. An advantage of using this approach is that the three-dimensional structures of NEWM and REI variable regions are known from X-ray crystallography and thus specific interactions between CDRs and V region framework residues can be readily modeled.<br>
VI.	Other Moieties<br>
The monoclonal antibodies of this invention may further include other moieties to effect the desired functions. For instance, the antibodies may include a toxin moiety (e.g., tetanus toxoid or ricin) or a radionuclide (e.g., '"In or 90Y) for killing of cells targeted by the antibodies (see, e.g., U.S. Patent 6,307,026). The antibodies may include a moiety (e.g., biotin, fluorescent moieties, radioactive moieties, histidine tag or other peptide tags) for easy isolation or detection. The antibodies may also include a moiety that can prolong their serum half life, for example, a polyethylene glycol (PEG) moiety, and a member of the immunoglobulin super family or fragment thereof (e.g., a portion of human IgGl heavy chain constant region such as the hinge, CH2 and CH3 regions).<br>
VII.	Crystallizable Compositions and Crystals<br>
This invention also provides a crystallizable composition containing a complex of: (1) a rat-human chimeric αl-I domain (e.g., mutant RAH), or a portion thereof (e.g., a polypeptide including 135 to 336 amino acids of the rat-human chimeric αl-I domain); and (2) a Fab fragment of hAQC2, or a portion thereof (e.g., a polypeptide including 3 to 213 amino acids of the light chain and/or a polypeptide including 3 to 219 amino .acids of the heavy chain). An exemplary complex is shown<br>
in Fig. 20. The RAH al-I domain can include, e.g., amino acid residues 145 to 336<br>
(crystal numbering) (SEQ ID N0:59, infra) of the rat αl sabunit. The hAQC2 Fab<br>
fragments may include light chain amino acid residues 1 to 106 (e.g., 1-213) of SEQ ID NO:3 and heavy chain amino acid residues 1 to 118 (e.g., 1-219) of SEQ ID NO:4. The hAQG2 Fab fragments may be obtained by papain digestion of the whole antibody or made by recombinant methods. The Fab fragments include at least an antigen-binding portion of the variable domains of the light chain and/or the heavy chains of hAQC2.<br>
145       TQLDIV<br>
151	IVLDGSNSIY PWESVIAFLN DLLKRMDIGP KQTQVGIVQY<br>
191	GENVTHEFNL NKYSSTEEVL VAANKIVQRG GRQTMTALGI<br>
231	DTARKEAFTE ARGARRGVKK VMVIVTDGES HDNYRLKQVI<br>
271	QDCEDENIQR FSIAILGHYN RGNLSTEKFV EEIKSIASEP<br>
311.     TEKHFFNVSD ELALVTIVKA LGERIF<br>
(SEQ ID NO:59)<br>
Some crystallizable compositions and crystals of this invention may contain a molecule or molecular complex that is homologous to the αl-I domain and/or the hAQC2 Fab fragment by amino acid sequence or by three-dimensional structure. Examples of homologues include, but are not limited to: the al-I domain and/or the hAQC2 Fab fragment with mutations, such as conservative substitutions, additions, deletions or a combination thereof. "Conservative substitutions" refer to replacement residues that are physically similar in size, shape, hydrophobicity, charge, and/or chemical properties to the corresponding reference residues. Methods for identifying a "corresponding" amino acid are known in the art and are based upon sequence, structural alignment, its functional position or a combination thereof as compared to the crystal structure solved in the present invention. For example, corresponding amino acids may be identified by superimposing the backbone atoms of the amino acids in the αl-I domain/hAQC2 complex and a al-I domain and/or hAQC2 homologue using well known software applications, such as QUANTA (Molecular Simulations, Inc., San Diego, CA ©1998,2000). The corresponding<br>
amino acids may also be identified using sequence alignment programs such as the "bestfit" program available from the Genetics Computer Group, which uses the local homology algorithm described by Smith and Waterman in Adv. Appl. Math. 2:482 (1981).<br>
Crystallizable compositions of this invention may further include one or more components feat promote crystallization and/or is compatible with crystallization conditions. Such components may include, but are not limited to, buffer, salts, precipitating agents and other reagents. One component can be 30% weight/volume Polyethylene Glycol 1500 (PEG1500).<br>
The instant invention also provides methods of making crystals from crystallizable compositions including a.complex of al-I domain and an antigen-binding portion of hAQC2 (e.g., Fab, Fab' or other fragments, supra). Various techniques of crystallization can be used in the claimed invention, including, but not limited to, vapor-diffusion, dialysis, microbatch, batch, and liquid-liquid diffusion. Vapor diffusion methods include, but are not limited too, sitting-drop, hanging-drop and sandwich-drop techniques. Vapor-diffusion methods can use techniques to control the rate of crystallization, such as the addition of oils on the drops or reservoir solution. Crystallization methods can include mixing a reservoir solution containing precipitating agent with an aqueous solution of a complex of al-I domain and an antigen-binding portion of hAQC2 to produce a crystallizable composition. The mixture or crystallizable composition may then be crystallized using the various above-listed techniques. The crystallizable composition of this invention may be an aqueous solution of a complex of al-I domain and an antigen-binding portion of hAQC2 containing the complex at a concentration of about 1 to 50 mg per mL, such as a concentration of about 5 to 15 mg per mL (e.g., 11 mg per mL). VIII. Crystal Structures and Structure Coordinates<br>
This invention further provides the three-dimensional structure of a crystal including a complex of mutant RAH, and a hAQC2 Fab fragment at 2.8 A resolution (Example 24, infra). The three-dimensional structures of other related crystals may also be determined using techniques described herein and those known in the art The three-dimensional structure of this complex is defined by a set of<br>
stracture coordinates set forth in Fig, 19. These structure coordinates are Cartesian atomic coordinates derived from mathematical equations related to the patterns<br>
obtained from diffraction of a monochromatic beam of X-rays by the atoms or scattering centers of the crystalline complex of the αl-I domain and the hAQC2 Fab fragment. Diffraction data are first used to calculate an electron density map of the repeating unit of the crystal. The electron density map is then used to establish the positions of individual atoms of the complex.<br>
This invention provides a molecule or a molecular complex defined by all or part of the structure coordinates of all amino acids set forth in Fig. 19, as well as a homologue of the molecule or molecular complex, where the homologue has a root mean square deviation from the backbone atoms of these amino acids between 0.00 A and 0.65 A, such as between 0.00 A and 0.60 A (e.g., between 0.00 A and 0.50 A). The term "root mean square deviation" or "r.m.s. deviation" means the square root of the arithmetic mean of the squares of the deviations from the mean. It is a way to express the deviation or variation from a trend or object. For purposes of this invention, the "root mean square deviation" or "r.m.s. positional deviation" defines the variation in the backbone of a protein from the relevant portion of the backbone of the polypeptide as defined by the structure coordinates described herein.<br>
A molecule or a molecular complex of this invention may also include a binding site defined by structure coordinates of at least seven amino acids of the hAQC2 Fab fragment selected from the group including of light chain residues AsnSO,,  Tyr48, Trp90, Ser9l, Asn93 and Trp95, and heavy chain residues Ser30, ArgSl, Trp47, Ser52, Gly33,.His56, Tyr58, Phe99, GlylOO and Asp 101 (crystal numbering) according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of one or more of these amino acids between 0.00 A and 1.10 A, such as between 0.00 A and 1.00 A (e.g., between 0.00 A and 0.50. A). The term "binding site" as used herein, refers to a region of a molecule or molecular complex that, as a result of its shape and charge, favorably associates with another chemical entity. The term "site" includes, but is not limited to, trench, cleft, channel or pocket For instance, binding sites on the α 1 -I domain may include a collagen-binding site<br>
 (Emsley et al., 1997, supra), an antibody-binding site, and an allosteric (or IDAS) binding site (Huth et al., 2000, Proc. Natl. Acad. Sci. U.S.A. 97:5231-5236). The term "chemical entity" includes, but is not limited to, any molecule, molecular complex, compound or fragment thereof. The term "associate with" refers to an association or binding in a condition of proximity between a chemical entity, or portions thereof, and a binding pocket or binding site on a protein. The association may be non-covalent — where the juxtaposition is energetically favored by hydrogen bonding or van der Waals or electrostatic interactions— or it may be covalent.<br>
A molecule or molecular complex of this invention can include a binding site defined by structure coordinates of αl-I domain amino acids selected from the group consisting of residues Asp 154, Serl56, Asnl57, Serl58, Tyrl60, Glu.192, Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19, or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of the αl-I domain amino acids between 0.00 A and 0.92 A.<br>
A molecule or molecular complex of this invention also may include a binding site defined by structure coordinates of α1-I domain amino acids selected from the group consisting of residues Glul92, Gln218, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of the αl-I domain amino acids between 0.00 A and 0.30 A.<br>
Those of skill in the art will understand that a set of structure coordinates for a polypeptide is a relative set of points that define a shape in three dimensions. Thus, it is possible that an entirely different set of coordinates that define a similar or identical shape could be generated using mathematical manipulations of the structure coordinates in Fig. 19. For example, the structure coordinates could be manipulated by-crystallographic permutations of the structure coordinates, fractionalization of the structure coordinates, integer additions or subtractions to sets of the structure coordinates, inversion of the structure coordinates, or any combination<br>
thereof. Moreover, slight variations in the individual coordinates will have little effect on overall shape.<br>
Alternatively, modification in the crystal structure due to mutations, such as additions, substitutions, and/or deletions of amino acids, or other changes in any of the polypeptide components (e;g., a hAQC2 Fab fragment or a αl-I domain) that make up the crystal can also account for variations in structure coordinates. If such variations are within an acceptable standard error as compared to the original coordinates, the resulting three-dimensional shape is considered to be the same as that of the unmodified crystal.<br>
It is therefore necessary to determine whether an entity is sufficiently similar to all or parts of the structure described herein as to be considered the same. Such analyses may be carried out using current software applications, such as QUANTA (Accelrys, Inc. and Molecular Simulations, Inc., San Diego, CA ©1998,2000) and 0 (Jones et al., 1991, Acta Cryst. A47:110-119), and accompanying User Guides. The Molecular Similarity application of QUANTA and the LSQ application of O permit comparisons between different structures, different conformations of the same structure, and different parts of the same structure. The general procedure used in both applications is to input the structures to be compared, define the equivalent atomic positions in these structures, perform a fitting operation, and analyze the results.<br>
When each structure is input into the application, it is given a name, and identified as the fixed structure or a moving structures. Atom equivalency is usually defined by equivalent atoms such as protein backbone atoms (N, Co, C and 0) for all conserved residues between the two structures being compared. The moving structure is translated and rotated to obtain an optimum or least-squares fit with the fixed structure. The root mean square difference of the fit over the specified pairs of equivalent atom is reported by both programs in angstroms.<br>
For the purpose of this invention, any molecular complex that has a root mean square deviation of conserved residue backbone atoms (N, Cα, C, 0) between 0.00 A and 1.50 A, such as between 0.00 A and 1.00 A (e.g., between 0.00 A and 0.50 A), when superimposed on the relevant backbone atoms described by<br>
structure coordinates listed in Fig. 19 are considered identical. IX. Determining Other Crystal Structures<br>
The structure coordinates set forth in Fig. 19 can-also be used to aid in obtaining structural information about another crystallized molecular entity, such as another hAQC2 containing amino acid substitutions in one of it's CDRs. This may be achieved by any well-known techniques, including molecular replacement, an especially useful method for determining the structures of mutants and homologues of α 1-I domain/Fab.<br>
The structure coordinates set forth in Fig. 19 can also be used for determining at least, a portion of the three-dimensional structure of molecular entities that contain at least some structural features similar to at least a portion of the αl-I domain or the hAQC2 Fab. Therefore, another embodiment of this invention provides a method of utilizing molecular replacement to obtain structural information about a crystallized molecule or molecular complex with unknown structure including the steps of: (a) generating an X-ray diffraction pattern from the crystallized molecule or molecular complex; and (b) applying at least a portion of the structure coordinates set forth in Fig. 19 to the X-ray diffraction pattern to generate a three-dimensional electron density map of the molecule or molecular complex with unknown structure.<br>
By using molecular replacement, all or part of the structure coordinates set forth in.Fig. 19 can be used to determine the unknown structure of a crystallized molecular entity more rapidly and efficiently than attempting to determine such information ab initio. Molecular replacement provides an accurate estimation of the phases for an unknown structure. Phases are a factor in equations used to solve crystal structures that cannot be determined directly. Obtaining accurate values for the phases, by methods other than molecular replacement, can often be a time-consuming process that involves iterative cycles of approximations and refinements and greatly hinders the solution of crystal structures. However, when the crystal structure of a protein containing at least a homologous portion has been solved, the phases from the known structure can often provide a satisfactory estimate of the phases for the unknown structure.<br>
Thus, molecular replacement involves generating a preliminary model<br>
of a molecule or molecular complex whose structure coordinates are unknown, by orienting and positioning the relevant portion of the complex according to Fig. 19 within the unit cell of the crystal of the unknown molecule or molecular complex, so as best to account for the observed X-ray diffraction pattern of the crystal of the molecule or molecular complex whose structure is unknown: Phases can then be calculated from this model and combined with the observed X-ray diffraction pattern amplitudes, to generate an electron density map of the structure whose coordinates are unknown. This, in turn, can be subjected to any well-known model building and structure refinement techniques to provide a final, accurate structure of the unknown crystallized molecule or molecular complex (Lattman, 1985, Meth. Enzymol, 115:55-77; Rossmann, ed., "The Molecular Replacement Method", Int. Sci. Rev. Sen, No. 13, Gordon &amp; Breach, New York, 1972). The structure ofany portion of any crystallized molecule or molecular complex that is sufficiently homologous to any portion of the αl-I domain and/or the hAQG2 Fab fragment (according to Fig. 19) can be solved by this method. X. Computer and Storage Medium<br>
To use the structure coordinates of this invention, e.g., those set forth in Fig. 19, it is usually necessary to convert the coordinates into a three-dimensional representation or shape. Commercially available graphical software programs including, but not limited to, O (Jones et al., 1991,Acto Cryst. A47:110-119) and INSIGHTII (© Accelrys, Inc. and Molecular Simulations, Inc., San Diego, CA) are capable of generating three-dimensional representations of molecules or molecular complexes, or portions thereof, from a set of structure coordinates.<br>
In accordance with the present invention, the structure coordinates of the molecular entities of this invention are stored in a storage medium readable by machine (e.g., a computer). Using a computer and appropriate software, such data may be used for a variety of purposes, such as drug discovery and X-ray crystallographic analysis of other protein crystals.<br>
Accordingly, a machine-readable data storage medium may include a data storage material encoded with machine-readable data including at least a portion of the structure coordinates set forth in Fig. 19. The computer may further include<br>
instructions to produce three-dimensional representations of the molecular complexes of αl-I domain and the hAQC2 Fah fragment by processing-the machine-readable, data-of this invention. The computer of this invention may also include a display, a graphical interface for displaying, or an input device for moving and manipulating the three-dimensional graphical representation of the structure coordinates.<br>
This invention also provides a computer for determining at least a portion of the structure coordinates corresponding to X-ray diffraction data obtained from a molecular complex of a 101 .integrin and the Fab fragment of hAQG2 antibody, where the computer includes a machine-readable data storage medium including a data storage material encoded with machine-readable data, where the data includes at least a portion of the structure coordinates of the molecular complex of al-I domain and the hAQG2 Fab fragment according to Fig. 19, or X-ray diffraction data obtained from the crystalline molecular complex. The computer further includes instructions for performing a Fourier transform of the machine readable coordinate data, and instructions for processing this machine readable diffraction data into structure coordinates. This computer may further include: a working memory for storing instructions for processing the machine-readable data; a central-processing unit coupled to the working memory and to the machine-readable data; and optionally a graphical interface or display coupled to the central-processing unit for displaying the three-dimensional graphical representation of the structure coordinates of the molecule or molecular complex.<br>
This invention further provides a computer for producing a three-dimensional representation of: a molecule or a molecular complex defined by at least a portion or all of the structure coordinates of all the α1 -I domain and the hAQC2 Fab fragment amino acids  set forth in Fig. 19, or a homologue of the molecule or molecular complex, where the homologue has a root mean square deviation from the backbone atoms of the amino acids of between 0.00 A than 1.50 A, such as between 0.00 A and 1.00 A, (e.g., between 0.00 A and 0.50 A). Further in this invention the computer includes: a machine-readable data storage medium including a data storage material encoded with machine-readable data, where the data includes at least a portion or all of the structure coordinates of all of the al-I domain and the Fab<br>
hAQC2 fragment amino acids set forth in Fig. 19.<br>
A computer of this invention may also produce a three-dimensional representation pf a molecule or molecular complex including a binding site. The binding site may be defined by structure coordinates of at least seven amino acids of: the hAQC2 Fab fragment-selected from the group including light chain residues . Asn30, Tyr48, Trp90, ,Ser91, Asn93 and Trp95, and heavy chain residues Ser30, Arg31,Trp47, Ser52, Gly53, His56, Tyr58, Phe99, GlylOO and Aspl0l (crystal numbering) according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of the at least one amino acid of the hAQC2 Fab fragment of.between 0.00 A and 1.10 A, such as between 0.00 A and 1.00 A, (e.g., between 0.00 A and 0.50 A). Further, the computer of this invention includes: a machine-readable data storage medium including a data storage material encoded with machine-readable data, where the data includes the structure coordinates of at least seven amino acids of the hAQC2 Fab fragment selected from the group consisting of light chain residues Asn30, Tyr48, Trp90, Ser91, Asn93 and Trp95, and heavy chain residues Ser30, Arg31, Trp47, Ser52, Gly53, His56, Tyr58, Phe99, GlylOO and Aspl0l (crystal numbering) according to Fig. 19.<br>
This invention also provides a computer for producing a<br>
three-dimensional representation of: a molecule or molecular complex including a binding site defined by structure coordinates I domain amino acids selected from the group consisting of residues Aspl54, Serl56, Asnl57, Serl58, Tyrl60, Glul92, Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of the I domain amino acids between 0.00 A and 0.92 A. Further in this invention, the computer includes: a machine-readable data storage medium including a data storage material encdded with machine-readable data, where the data includes the structure coordinates of I domain amino acids selected from the group consisting of residues Aspl54, Serl56, Asnl57, Serl58, Tyrl60, Glul92, Gln218, Arg219, Gly220, Gly221,<br>
Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering), according to Fig. 19.<br>
This invention also provides a computer for producing a<br>
three-dimensional representation of a molecule or molecular complex including a binding site defined by structure coordinates of I domain amino acids selected from the group consisting of residues Glul92, Gln218, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19; or a homologue of the molecule or molecular complex, where the homologue includes a binding site that has a root mean square deviation from the backbone atoms of I domain amino acids between 0.00 A and 0.30 A. Further in this invention the computer includes: a machine-readable data storage medium including a data storage material encoded with machine-readable data, where the data includes the structure coordinates I domain amino acids selected from the group consisting of residues Glul92, Gln2l8, Arg219, Gly220, and Gly221 (crystal numbering), according to Fig. 19.<br>
Fig. 21 demonstrates one such embodiment. System 10 includes a computer 11 including a central-processing unit ("CPU") 20, a working memory 22 which may be, e.g., RAM (random-access memory) or "core" memory, mass storage memory 24 (such as one or more disk or tape drives or CD-ROM or DVD-ROM drives), one or more cathode-ray tube ("CRT") display terminals 26, one or more keyboards 28, one or more input lines 30, and one or more output lines 40, all of which are interconnected by a conventional bidirectional system bus 50.<br>
Input hardware 36, coupled to computer 11 by input lines 30, may be implemented in a variety of ways. Machine-readable data of this invention may be inputted via the use of a modem or modems 32 connected by a telephone line or dedicated data line 34. Alternatively or additionally, the input hardware 36 may include CD-ROM or DVD-ROM drives or tape or disk drives 24. In conjunction with display terminal 26, keyboard 28 may also be used as an input device.<br>
Output hardware 46, coupled to computer 11 by output lines 40, may similarly be implemented by conventional devices. By way of example, output hardware 46 may include CRT display terminal 26 for displaying a graphical representation of a binding site of this invention using a program such as QUANTA as<br>
described herein. Output hardware might also include a printer 42, so that hard copy output may be produced, or a disk drive 24, to store system output for later use.<br>
In operation, CPU 20 coordinates the use of the various input and output devices 36, .46, coordinates data accesses from mass storage 24 and accesses to and from working memory 22, and determines the sequence of data processing steps. A number of programs may.be used to process the machine-readable data of this invention. .Such programs are discussed in reference to the computational methods of drug discovery as described herein. Specific references to components of the hardware system 10 are included as appropriate throughout the following description of the data storage medium.<br>
Fig. 22 shows a cross-section of a magnetic data storage medium 100 which can be encoded with machine-readable data that can be carried out by a system such as system 10 of Fig. 21. Medium 100 can be a conventional floppy diskette or hard disk, having a suitable substrate 101, which may be conventional, and a suitable coating 102, which may be conventional, on one or both sides, containing magnetic domains (not visible) whose polarity or orientation can be altered magnetically. Medium 100 may also have an opening (not shown) for receiving the spindle of a disk drive or other data storage device 24.<br>
The magnetic domains of coating 102 of medium 100 are polarized or oriented so as to encode in manner which may be conventional, machine readable data such as that described herein, for execution by a system such as system 10 of Fig. 21.<br>
Fig. .23 shows a cross-section of an optically-readable data storage medium 110 which also can be encoded with such machine-readable data, or a set of instructions, which can be carried out by a system such as system 10 of Fig. 21. Medium .110 can be a conventional compact disk or DVD disk read only memory (CD-ROM or DVD-ROM) or a rewritable medium, such as a magneto-optical disk which is optically readable and magneto-optically writable. Medium 100 has a suitable substrate 111, which may be conventional, and a suitable coating 112, which may be conventional, usually of one side of substrate 111.<br>
In the case of CD-ROM, as is well known, coating 112 is reflective and is impressed with a plurality of pits 113 to encode the machine-readable data. The<br>
arrangement of pits is read by reflecting laser light off the surface of coating 112. A protective coating 114, which is substantially transparent, is provided on top of coating 112.  .<br>
In the case of a magneto-optical disk, as is well known, coating 112 has no pits 113, but has a plurality of magnetic domains whose polarity or orientation can be changed magnetically when heated above a certain temperature, as by a laser (not shown). The orientation of the domains can be read by measuring the polarization of laser light reflected from coating 112. The arrangement of the domains encodes the data as described above. XI. Rational Drug Design<br>
The present invention permits the use of structure-based and rational drug design techniques to design, select, and synthesize or isolate chemical entities, such as inhibitors of the α1 -I domain and to improve known inhibitors of this domain. These inhibitors may be capable of blocking the collagen-binding site of VLA-1. This invention also permits the use of structure-based and rational drug design techniques to design variants that may act as inhibitors of collagen binding.<br>
The three-dimensional representation of this invention can be used experimentally or computationally to design potential inhibitors, other chemical entities, variants of the Fab fragment or combinations of chemical entities that may bind to and effect the biological functions of the hAQC2 Fab fragment or the chimeric α1 -I domain of the current invention.<br>
One skilled in the art can use one of several methods to screen chemical entities for their ability to associate with the complex of the hAQC2 Fab fragment or the chimeric αl-I domain of the current invention and more particularly with a binding site of either the I domain or the Fab fragment. This process may begin by visual inspection of, for example, the binding site for either the I domain or the Fab fragment on the computer screen, based on the coordinates of the complex in Fig. 19. Selected chemical entities may then be positioned in a variety of orientations, or docked, within an individual binding site of either the I domain or the Fab fragment Docking may be accomplished using software such as QUANTA, followed by energy minimization and molecular dynamics with standard molecular mechanics forcefields,<br>
such as CHARMM (Molecular Simulations, Inc., Burlington, MA ©1994) and<br>
AMBER(P.A.Kollman), University of California	at San Francisco,©1994)<br>
Specialized computer programs may also assist in the process of selecting chemical entities. These include, inter alia:<br>
1 .        GRID (Goodford, P.J., 1985, J. Med. Chem. 28:849-857). GRID is available from Oxford University, Oxford, UK.<br>
2.	MCSS (Miranker, A. and M. Karplus, 1991, Proteins: Structure, Function<br>
and Genetics 1 1 :29-34). MCSS is available from Molecular Simulations,<br>
Burlington, MA.<br>
3.	AUTODOCK (Goodsell, D.S. and A.J. Olsen, 1990, Proteins: Structure,<br>
Function, and Genetics 8:195-202). AUTODOCK is available from Scripps<br>
Research Institute, La Jolla, CA.<br>
4.	DOCK (Kuntz, ID. et al., 1982, J. Mol. Biol. 161:269-288). DOCK is<br>
available from University of California, San Francisco, CA.<br>
Once suitable chemical entities have been selected, they can be assembled into a single compound. Assembly may proceed by visual inspection of the relationship of the entities to each other on the three-dimensional image displayed on a computer screen in relation to the structure coordinates of the complex of hAQC2 Fab fragment and the chimeric α1-I domain. This is followed by manual model building using software such as Quanta or Sybyl.<br>
The above-described evaluation process for chemical entities may be performed in a similar fashion for compounds or for variants that may bind the cd-I domain.<br>
Useful programs to aid one of skill in the art in connecting the individual chemical entities include:<br>
1 .        CAVEAT (Bartlett, P.A. et al, "CAVEAT:" A Program to Facilitate the<br>
Structure-Derived Design of Biologically Active Molecules". In "Molecular Recognition in Chemical and Biological Problems", Special Pub., 1989, Royal Chem. Soc., 78:182-196). CAVEAT is available from the University of California, Berkeley, C A.<br>
2.        3D Database systems such as MACCS-3D (MDL Information Systems, San Leandro, CA). This area is reviewed in Martin, Y.C., 1992,J. Med. Chem. 35:2145-2154.<br>
3.        HOOK (available from Molecular Simulations, Burlington, MA).<br>
Instead of proceeding to build an inhibitor or  binding compound in a step-wise fashion one chemical entity at a time, as described above, binding compounds may be designed as a whole or "de novo" using either an empty binding site (such as a binding site of the al-I domain or the hAQC2 Fab fragment) or optionally including some portion(s) of a known al-I domain or the hAQC2 Fab fragment binding compound. These methods include:<br>
1.	LUDI (Bohm, H.-J., 1992, J. Comp. Aid. Molec. Design 6:61-78). LUDI is<br>
available from Biosym Technologies, San Diego, CA.<br>
2.	LEGEND (Nishibata, Y. and A. Itai, 1991, Tetrahedron 47:8985). LEGEND<br>
is available from Molecular Simulations, Burlington, MA.<br>
3.	LeapFrog (available from Tripos Associates, St. Louis, MO).<br>
Other molecular modeling techniques may also be employed in accordance with this invention. See, e.g., Cohen, N.C. et al., 1990, J. Med. Chem. 33:883-894. See also Navia, M.A.-and M.A. Murcko, 1992, Curr. Opin. Struct. Biol 2:202-210.<br>
Once an entity has been designed or selected by the above methods, the efficiency with which that entity may bind to the al-I domain or the hAQC2 Fab fragment can be tested and optimized by computational evaluation. For example, a compound that has been designed or selected to function as a αl-I domain binding compound can traverse a volume not overlapping that occupied by the binding site when it is bound to the chimeric αl-I domain. An effective al-I domain binding compound can demonstrate a relatively small difference in energy between its bound and free states (i.e., a small deformation energy of binding). Thus, the most efficient αl-I domain binding compound should be designed with a deformation energy of binding of not greater than about 10 kcaymole, e.g., not greater than 7 kcal/mole. al-I domain binding compounds may interact with the α 1-I domain in more than one conformation that is similar in overall binding energy. In those cases, the deformation energy of binding is taken to be the difference between the energy of the free compound and the average energy of the conformations observed when the compound binds to the protein.<br>
A compound designed or selected as binding to-α1-I domain may be further computationally optimized so that in its bound state it would lack repulsive electrostatic interaction with the target protein. Such non-complementary (e.g., electrostatic) interactions include repulsive charge-charge, dipole-dipole and charge-dipole interactions. Specifically, the sum of all electrostatic interactions between the compound and the protein when the compound is bound to αl-I domain, should make a neutral or favorable contribution to the enthalpy of binding.<br>
Specific computer software is available in the art to evaluate compound deformation energy and electrostatic interaction. Examples of programs designed for such uses include: Gaussian 92, revision C (MJ. Frisch, Gaussian, Inc., Pittsburgh, PA ©1992); AMBER, version 4.0 (P.A. Kollman, University of California at San Francisco, ©1994); QUANTA/CHARMM (Molecular Simulations, Inc., Burlington, MA ©1994); and Insight IE/Discover (Biosysm Technologies Inc., San Diego, CA ©1994). These programs may be implemented, for instance, using a Silicon Graphics workstation. Other hardware systems and software packages will be known to those skilled in the art.<br>
One other useful drug design technique enabled by this invention is iterative drug design. Iterative drug design is a method for optimizing associations between a protein and a compound (that compound includes an antibody) by determining and evaluating the three-dimensional structures of successive sets of protein/compound complexes. In iterative drug design, a series of crystals of a protein complexed with entities that bind the protein are obtained and then the three-dimensional structure of each molecular complex is solved. Such an approach provides insight into the associations between the proteins and other entities of each complex. This is accomplished by selecting chemical entities with inhibitory activity, obtaining crystals of these new complexes, solving the three-dimensional structure of the complexes, and comparing the associations between the new complexes and the previously solved complex. Associations within a complex can be optimized by observing how changes in the components of the complex affect associations.<br>
In some cases, iterative drug design is carried out by forming successive complexes and then crystallizing each new complex. Alternatively, a<br>
pre-formed protein crystal is soaked in the presence of another chemical entity,<br>
thereby forming a complex and obviating the need to crystallize each individual<br>
complex.<br>
XII. Pharmaceutical Compositions<br>
The pharmaceutical compositions of this invention contains one or more VLA-1 antagonists of the present invention (e.g., anti-VLA-1 antibodies and the small molecular VLA-1 antagonists identified by the above-described rational drug design methods), or pharmaceutically acceptable derivatives thereof. The compositions may further contain a pharmaceutically acceptable carrier, such as an -adjuvant, a vehicle, a buffer, and a stabilizer.<br>
The pharmaceutical compositions of this invention may be given orally, topically, intravenously, subcutaneously, intraperitoneally, intramuscularly, intramedullarily, intraarterially, intra-articularly, intra-synovially, intrastemally, intrathecally, intrahepatically, intraspinally, intracranially as desired, or just locally at sites of inflammation or tumor growth. The pharmaceutical compositions of this invention may also be administered by inhalation through the use of, e.g., a nebulizer, a dry powder inhaler or a metered dose inhaler, or by implantation of an infusion pump or a biocompatible sustained release implant into the subject.<br>
The pharmaceutical compositions may be in the form of a sterile injectable preparation, for example a sterile injectable aqueous or oleaginous suspension. This suspension may be formulated according to techniques known in the art using suitable dispersing, wetting, and suspending agents. If given orally, the pharmaceutical compositions can be administered in form of capsules, tablets, aqueous suspensions or solutions. For topical applications, the pharmaceutical compositions may be formulated in a suitable ointment.<br>
The dosage and dose rate of the VLA-1 antagonists of this invention effective to produce the desired effects will depend on a variety of factors, such as the nature of the disease to be treated, the size of the subject, the goal of the treatment, the specific pharmaceutical composition used, and the judgment of the treating physician. Dosage levels of between about 0.001 and about 100 mg/kg body weight per day, for example between about 0.1 and about 50 mg/kg body weight per day, of the active<br>
ingredient compound are useful. For instance, an antibody of the invention will be administered at a dose ranging between about 0.01 mg/kg body weight/day and about 20 mg/kg body weight/day, e.g., ranging between about 0.1 mg/kg body weight/day and about 10 mg/kg body weight/day, and at intervals of every one to fourteen days. In another embodiment, the antibody is administered at a dose of about 0.3 to 1 mg/kg body weight when administered intraperitoneally. In yet another embodiment, the antibody is administered at a dose of about 5 to 12.5 mg/kg body weight when administered intravenously. In one embodiment, an antibody composition is administered in an amount effective to provide a plasma level of antibody of at least 1 mg/ml. XIII. Diseased Conditions And Animal Models<br>
The VLA-1 antagonists of the invention are useful hi the treatment, including prevention, of α1,β1-mediated diseases such as those enumerated above. The treatments of this invention are effective on both human and animal subjects afflicted with these conditions. Animal subjects to which the invention is applicable extend to both domestic animals and livestock, raised either as pets or for commercial purposes. Examples are dogs, cats, cattle, horses, sheep, hogs and goats.<br>
The efficacy of the VLA-1 antagonists of the invention can be tested in various animal models. For instance, useful psoriasis and arthritis models include those described in WO 00/72881. Kidney fibrosis models include those described hi WO 99/61040, the Alport's syndrome kidney model described in Cosgrove et al., 2000, Am. J. Path. 157:1649-1659, and the SNF1 mouse model of lupus nephritis described in Kalled et al., 2001, Lupus 10:9-22. Vascular fibrosis models for restenosis include a rat carotid balloon injury model described in Smith et al., 1999, Circ. Res. 84:1212-1222. Lung fibrosis models for idiopathic pulmonary fibrosis and scleroderma-associated pulmonary fibrosis include a bleomycin-induced pulmonary fibrosis model described hi Wang et al., 1999, TJwrax 54:805-812. Liver cirrhosis models for hepatitis C- or alcohol-induced cirrhosis include the bile duct iigation model described in George et al., 1999, Proc, Nad. Acad. Sci. USA 96:12719-12724 and the CCL4-induced liver fibrosis model described in Shi et al, 1997, Proc. Natl. Acad. Sci. USA 94:10663-10668.<br>
The efficacy of the treatments of this invention may be measured by a number of available diagnostic tools, including physical examination, blood tests, proteinuria measurements, creatinine levels and creatinine clearance, pulmonary function tests, chest X-rays, bronchoscopy, brorichoalveolar lavage, lung biopsy, plasma blood urea nitrogen (BUN) levels, observation and scoring of scarring or fibrotic lesions, deposition of extracellular matrix such as collagen, smooth muscle actin and fibronectin, kidney function tests, ultrasound, magnetic resonance imaging (MRI), and CT scan. XIV. Diagnostic Methods<br>
The antibodies of this invention can be used to diagnose diseased conditions associated with altered α1,β1 expression levels. A tissue sample from a subject, such as a tissue biopsy, body fluid sample or lavage (e.g., alveolar lavage), can be tested in an antigen capture assay, ELIS A, immunohistochemistry assay, and the like using the antibodies. A tissue sample from a normal individual is used as control.<br>
Practice of the present invention will employ, unless indicated otherwise, conventional techniques of cell biology, cell culture, molecular biology, microbiology, recombinant DNA, protein chemistry, and immunology, which are within the skill of the art. Such techniques are described in the literature. See, for example, Molecular Cloning: A Laboratory Manual, 2nd edition (Sambrook et al., Eds.), l989;Oligonucleotide Synthesis, (M.J. Gait, Ed.), 1984; U.S. Patent 4,683,195 to Mullis et al.; Nucleic Acid Hybridization, (B.D. Hames and SJ. Higgins), 1984; Transcription and Translation, (B.D. Hames and SJ. Higgins), 1984; Culture of Animal Cells (R.I. Ereshney, Ed.), 1987; Immobilized Cells and Enzymes, IRL Press, 1986; A Practical Guide to Molecular Cloning (B. Perbal), 1984; Methods in Enzymology, Volumes 154 and 155 (Wu et al., Eds.), Academic Press, New York; Gene Transfer Vectors for Mammalian Cells (J.H. Miller and M.P. Calos, Eds.), 1987; Immunochemical Methods in Cell and Molecular Biology (Mayer and Walker, Eds.), 1987; Handbook of Experiment Immunology, Volumes I-IV (D.M. Weir and C.C. Blackwell, Eds.), 1986; Manipulating the Mouse Embryo, 1986.<br>
Unless otherwise defined, all technical and scientific terms used herein<br>
have the same meaning as commonly understood by one of ordinary skill in the art to which this invention belongs. Exemplary methods and materials are described below, although methods and materials similar or equivalent to those described herein can also be used in the practice or testing of the present invention. All publications and other references .mentioned herein are incorporated by reference in their entirety. In ease of conflict, the present specification, including definitions, will control. The materials, methods, and examples are illustrative only and not intended to be limiting. Throughout this specification and claims, the word "comprise," or variations such as "comprises" or "comprising" will be understood to imply the inclusion of a stated integer or group of integers but not the exclusion of any other integer or group of integers.<br>
The following Examples are provided to illustrate the present invention, and should not be construed as limiting thereof.<br>
EXAMPLES Chemical reagents<br>
Fluorescein isothiocyanate (FITC) was purchased from Sigma Chemical Co. (St. Louis, MO). Croton oil was purchased from ICN Biochemicals (Aurora, OH). Whole sheep blood in Alsevers solution was obtained from East Acres Biologicals (Southbridge, MA). Type I rat tail collagen and type IV mouse collagen were purchased from Collaborative Research Inc. (Bedford, MA) and Gibco (Gaithersburg, MD), respectively.<br>
Balb/c female mice of 6-8 weeks of age were purchased from Taconic (Germantown, NY) and the al pi integral-deficient mice on a Balb/c background were as previously described (3). Example j[<br>
Monoclonal Antibodies. Function-blocking mAbs to murine antigens were prepared in an azide-free and low endotoxin format: Ha31/8 (hamster anti-CD49a; integrin αl) (Mendrick et al. 1995. Lab. Invest. 72:367-375), Hal/29 (hamster anti-CD49b; integrin α2)( β1) (Mendrick et al. 1995. Lab. Invest. 72:367-375; Mendrick, D.L. and D.M. Kelly 1993 Lab. Invest. 69:690-702), hamster group II<br>
control mAb Ha4/8 (hamster anti-KLH)(Mendrick, D.L. and D.M. Kelly 1993 Lab. Invest. 69:690-702), and PS/2 (rat,anti-CD49d; integrin α4β1 chain) (Miyake etal 1991 J. Exp. Med. 173:599-607), In addition, the following function-blocking mAbs to murine antigens were purchased as no-azide/low endotoxin preparations from      Pharmingen (San Diego, CA): HMβl-1 (hamster anti-CD29; integrm β1 chain) (Noto etal. 1995 Int. Immunol. 7:835-842), Ha2/5 (hamster anti-CD29; integrin pichain)( Mendrick, D.L. and D.M. Kelly 1993 Lab. Invest. 69:690-702), 3E2 (hamster anti-CD54, ICAM-1X Scheynius et,al.l993 J. Immunol, 150:655-663), 5H10-27 (rat anti-CD49e; integrin α5)( Kinashi, T., and T.A. Springer.  1994. Blood Cells. 20:25-44), GoH3 (rat anti-CD49f; integrin a6)( Sonnenberg et al. 1987 J. Biol. Chem. 262:10376-10383), and the rat isotype control mAbs R35-95 (rat IgG2a) and R35-38 (rat IgG2b).<br>
Adhesion Assay. Splenocytes from Balb/c mice were cultured with 20 ng/ml IL-2 for 7-12 d. Adhesion of cells to type land type IV collagen was as previously described (Gotwals et al. 1996 J. din. Invest. 97:2469-2477). Briefly, 96-well Maxisorp plates (Nunc, Napierville, EL) were coated with either 10 µg/ml type IV or 5 µg/ml type I collagen and non-specific sites blocked with 1% BSA. IL-2 activated splenocytes were labeled with 2 µM BCECF [2',7'-bis(carboxyethyl)-5(6) carboxyl fluorescein penta acetoxymethylester] (Molecular Probes, Eugene, OR) and incubated with 10 µg/ml of indicated mAbs for 15 min. 10s cells in 0.25% BSA in RPMI were then added to coated wells and incubated for 60 min at 37°C. Unbound cells were removed by washing three times with 0.25% BSA in RPMI. Adhesion was quantified using a CytoFluor 2350 fluorescent plate reader (Millipore, Bedford,-MA). The ratio of bound cells to input cells was measured and percent adhesion relative to control mAb-treated cells (normalized to 100%) calculated. Background values due to cell adhesion on wells coated with BSA alone were subtracted.<br>
Expression and functional blockade of αlβ1 and α2β1 on activated leukocytes. Given the key role leukocytes play in inflammation, we decided to test whether anti-αl and anti-α2 mAbs were capable of blocking leukocyte adhesion to collagens. In order to obtain leukocytes expressing high levels of both al and α2, murine T cells were stimulated in vitro with EL-2 for 7-12 d. These cells expressed<br>
high levels of both αl and α2 (Fig. 1A), and bound well to both collagen type IV and .type I-coated surfaces (Fig. 1B) Adhesion to type IV collagen was partially inhibited by anti-α1 mAb alone and was not inhibited by anti-α2 mAb alone. In contrast, adhesion to type I collagen was completely inhibited by anti-α2 mAb and anti-αl mAb alone showed only partial inhibition. Both anti-β1 mAb and the combination of anti-al and anti-α2 mAbs completely inhibited adhesion to types I and IV collagen. Having demonstrated that the α1βl and α2βl integrins are expressed on activated T cells and that anti-al and a2 mAbs are able to functionally block leukocyte adhesion to collagens, we used these mAbs to investigate the in vivo role of these integrins in animal models of inflammatory disorders. Example2<br>
Inhibition of DTH responses by anti-integrin mAbs.   SRBC-induced delayed type hypersensitivity (DTH) responses were adapted from a previously published protocol (Hurtrel et al., 1992, Cell. Immunol 142:252-263). Briefly, mice were immunized s.c. in the back with 2 x 107 SRBC in 100 ul PBS on d 0. The mice were challenged on d 5 by injecting 1 x 108 SRBC in 25 ul PBS s.c into the right hind footpad. Footpad thickness was measured with an engineer's caliper (Mitutoyo/MTI, Paramus, NT) 20 h after antigen challenge, and the degree of footpad swelling calculated. Results are reported as the mean percent increase footpad thickness ± SEM and calculated as % increase = [1- (Right footpad thickness 20 h after antigen challenge/Uninjected left footpad thickness 20 h after antigen challenge)] x 100. To block the effector phase of the SRBC-induced DTH response, therapeutic or control mAb (10.0 ug), which were prepared according to the methods described in Example l, was given i.p. 1 h prior to antigen challenge.on d 5.<br>
SRBC-induced DTH is a well characterized in vivo model of inflammation, and in particular psoriasis, that has been used to demonstrate the importance of a variety of cytokines and adhesion molecules in inflammation (Tedder et al?, 1995, J.. Exp. Med. 181:2259-2264, Terashita et al.,'1996, J. Immunol. 156:4638-4643). SRBC-sensitized mice received anti-integrin mAbs 1 h prior to footpad antigen challenge and inflammation was assessed 20 h later as measured by increased footpad thickness. PBS and control hamster Ig-treated mice showed a 60-<br>
70% increase in footpad thickness 20 h after antigen challenge (Fig. 2). Compared to control hamster Ig treatment anti-α1 or anti-α2 mAbs resulted in a 68% and 60% inhibition in footpad thickness, respectively. The combination of anti-αl and α2 mAbs resulted in 71% inhibition, demonstrating little additive effect over anti-al or anti-α2 mAbs alone. Treatment with other anti-integrin mAbs was also effective at inhibiting DTH effector response. The degree of inhibition seen with the various mAb treatments was 49% (anti-cc4), 23% (anti-a5)3 and 57% (anti-oc6). Lastly, mAb blockade of the common β1 integrin subunit (mAb HMBI-1) inhibited the effector DTH response by 67%. Example 3<br>
Inhibition of CHS effector responses by anti-integrin mAbs. Contact hypersensitivity (CHS) to FITC was assayed as previously described (Gaspari et al., 1991, In Current Protocols in Immunology. J.E. Coligan, A.M. Kruisbeek, D.H. Margulies, E.M. Shevach, and W. Strober, editors. John Wiley &amp; Sons, New York. Section 4.2:1). Briefly, mice were sensitized by painting 100 ul 0.5% FITC in 1:1 acetone/dibutylphthalate onto the shaved back on d 0. 10 d later, animals were challenged by applying 5 ul 0.5% FITC onto both sides of each ear. Ear swelling response was determined by ear thickness measured with an engineer's caliper (Mirutoyo/MTI, Paramus, NJ) at the time of antigen challenge (d 10) and 24 h later, and the results reported as mean percent increase in baseline ear thickness ± SEM. Increase in ear thickness was calculated as % increase = [1- (Ear thickness 24 h after antigen challenge/Ear thickness at the time of antigen challenge)] x 100. To block the effector phase of the CHS response, therapeutic or control mAb (250 ug) was given i.p. 4 h prior to antigen challenge on d 10. Mice that were antigen-sensitized and ear challenged with vehicle only .(vehicle control) or mice that were ear challenged without prior sensitization (irritant control) served as negative controls (never exceeded 2% increase in ear thickness).<br>
Given that CHS is mechanistically distinct from DTH and involves different effector cells, we investigated what effect anti-integrin mAbs had on the effector phase of the CHS response. Mice were hapten-sensitized using FITC applied to their shaved backs, followed 10 d later with FITC challenge to the ear resulting in<br>
an inflammatory response the next day. FITC-sensitized mice demonstrated a 60-70% increase in thickness 24 h.after antigen challenge (Fig. 31 Consistent with published results (Scheynius et.aL, J. Immunol. 150:655-663), anti-ICAM-1 mAb treatment resulted in 51% inhibition of ear swelling. Compared to control hamster mAb, treatment of mice with anti-al or anti-α2 mAb 4 h prior to antigen challenge resulted in 37% and 57% inhibition in ear swelling, respectively (Fig. 3). The combination of anti-al and anti-α2 mAbs resulted in slightly greater inhibition of ear swelling (65%). Treatment with other mAbs to β1 integrins revealed that while anti-α4 and anti-α5 mAbs resulted in no inhibition of FITC-induced CHS effector response when compared to control rat mAb, treatment with anti-α6 mAb resulted in an 86% inhibition of effector responses. Lastly, mAb blockade of the common β1 integrin subunit inhibited CHS effector responses by 74%. Similar CHS results were obtained using different strains of mice (C57/BL6, 129/Sv) and a different sensitizing agent (oxazolone) (data not shown). Similar to the results seen in the SRBC-induced DTH model, histologic analysis of inflammed ears revealed that both edema formation and leukocytic infiltration were inhibited by anti-αl and anti-α2 mAb treatment.<br>
Consistent with the finding that αlβl and α2βl can be expressed on IL-2-activated splenocytes, analysis of lymph nodes from antigen-sensitized mice (FITC or oxazolone) revealed αlβl and α2βl to be expressed exclusively on CD44hl LFA-11"' activated CD4+ and CD8+ T cells (data not shown). Treatment of mice with anti-al and anti-α2 mAbs did not result in deletion of these cells, as the numbers of activated T cells in both spleen and lymph nodes seen in response to antigen sensitization in the CHS model was unaffected. In addition, effector cells were not functionally deleted as prolonged treatment of antigen-sensitized mice with anti-al and anti-α2 mAbs (d 10-16) did not affect the inflammatory response of mice challenged with antigen at d 20 (data not shown).<br>
Example 4<br>
CHS effector responses are decreased in α1β1-deficient mice. To<br>
exclude the possibility that the inhibitory role of αl β1 in the effector response of FlTC-mediated CHS was mAb-mediated, experiments were carried out in wild-type and αlβl-integrin deficient mice (Fig. 4). MAb inhibition of the effector phase in<br>
wild-type mice was consistent with previous results, with 56% inhibition in ear thickness seen with anti-al, 56% with anti-α2, and 62% with a combination of anti-α1 and anti-α2. The effector phase of CHS was significantly reduced in untreated αlβ1-deficient mice as compared to untreated wild-type mice (30% vs 71% increase in ear thickness, respectively). As expected, the level of ear swelling in untreated αlβ1-deficient mice was equivalent to the level of ear swelling seen in anti-αl mAb-treated wild-type mice. Lastly, mAb blockade of α2βl in the αlβl-deficient mice resulted in only slightly increased inhibition of ear swelling, consistent with the results seen in wild-type mice treated with a combination of anti-al and anti-α2 mAbs. Example 5<br>
To further exclude the possibility that the inhibitory effect of the anti-integrin mAbs seen in both the DTH and CHS models of inflammation is caused by a general anti-inflammatory effect mediated by the anti-al and anti-α2 mAbs, the effect of these mAbs on irritant dermatitis was studied.<br>
To assess irritant dermatitis, mice were painted with 5 ul of 0.8% croton oil in acetone on both sides of each ear. Therapeutic or control antibodies were given 4 h prior to the application of the irritant. Ear swelling was measured 24 h later as described above and compared to ear thickness prior to croton oil application. Results are reported as mean percent increase in baseline ear thickness ± SEM as described above. Mice painted with acetone only (vehicle control) served as a negative control.<br>
24 h later, ears of mice treated with croton oil showed a significant increase in ear thickness (48%), when compared to mice receiving vehicle only (acetone). Toxic ear swelling caused by croton oil was not significantly affected in mice pretreated with anti-al or anti-α2 mAbs when compared to either PBS or control mAb-treated animals (Fig. 5). Histologic examination of the croton oil-treated ears revealed no differences in numbers or types of infiltrating cells or edema formation in mice treated with anti-αl or anti-α2 mAbs, as compared to control mAb-treated mice or PBS-treated mice (data not shown).<br>
Example 6<br>
Inhibition of arthritis by α1β1 and α2β1.As α1β1 is well expressed<br>
on infiltrating cells in the synovium of arthritis patients, we decided to examine whether anti-al or anti-α2 mAbs would be inhibitory in an accelerated model of arthritis previously described (Terato -et al., 1992, J. Immunol. 148:2103-2108; Terato etal., 1995, Autoimmunity 22:137-147).<br>
Arthrogen-CIA Antibody kits were purchased from Stratagene  (La Jolla, CA) and arthritis induced using a well established protocol (Terato et al., 1992, J.Immunol, 148:2103-2108; Terato etal, 1995, Autoimmunity 22: 137-147). Briefly, arthritis was induced through i.p. injection of a cocktail of 4 anti-collagen type n mAbs ( 1 mg each) on d 0, followed by i.p. injection of 50 ug LPS on d 3.   Over the course of the next 3-4 d, the mice developed swollen wrists, ankles and digits. Therapeutic or control mAb (250 ug) was administered i.p. 4 h prior to injection of the anti-collagen mAbs on d 0, and again 4 h prior to LPS administration on d 3, and then continuing every 3rd day for the length of the experiment. Beginning on d 3, mice were evaluated for the development of arthritis. Severity of arthritis in each limb was scored using a four point system. 0=normal; l=mild redness, slight swelling of ankle or wrist; 2=moderate swelling of ankle or wrist; 3=severe swelling including some digits, ankle, and foot; 4=maximally inflamed.<br>
Severe arthritis in Balb/c mice developed within 72 h after LPS injection and persisted for more than 3 weeks. Neither injection of anti -collagen mAbs alone nor LPS alone induced arthritis. Mice receiving control mAb treatment displayed equally severe arthritis as than seen in PBS-treated mice (Fig. 6). In contrast, treatment with anti-al mAb alone resulted in a marked reduction (78%) in arthritis, lasting the duration of the experiment. Treatment with anti-α2 mAb alone also had abeneficial effect, resulting in a 32% decrease in the arthritic score as compared to control mAb-treated mice. The combination of anti-αl and anti-α2 mAbs resulted in a similar degree of inhibition as seen with anti-αl mAb alone.<br>
Example 7<br>
Histological analysis of effect of anti-α1 and anti-α2 mAb treatment<br>
on the inflammatory cellular infiltrate. Further histological analysis of the SRBC-induced DTH response confirmed the ability of anti-eel and anti-α2 mAb treatment to modulate the elicited inflammatory response. An unchallenged  footpad from an SRBC-sensitized mouse showed virtually no inflammatory cellular infiltrate when compared to an SRBC-chailenged footpad from the same mouse. Treatment of SRBC-sensitized mice with anti-αl and anti-α2 mAbs either alone or combined greatly reduced the number of these infiltrating cells found in SRBC-challenged footpads when compared to control mAb-treated mice. Closer examination of the infiltrating cells revealed most cells to be composed of neutrophils, with some monocytes and lymphocytes present, and confirmed that anti-al and anti-α2 mAb treatment greatly decreased the numbers of these cells. Example 8<br>
Immunohistochemical demonstration of αl-expressing cells in the inflammatory cellular infiltrate. Immunohistochemistry was performed to more precisely determine the nature of the infiltrating cells and whether they express collagen-binding integrins. Infiltrating cells from an inflamed footpad of an untreated mouse were examined for expression of αl β1 integrin and cell lineage markers, αlβl integrin was found to be expressed on many infiltrating leukocytes. Dual immunohistochemistry was-utilized to identify the nature of the infiltrating cells and the distribution of αl β1 expression. Using cell lineage markers  the infiltrate was found to be composed largely of granulocyte/monocytes (Mac-l+), with many of these cells being neutrophils (Grl+), along with a smaller number of T- lymphocytes (CD3+). Expression of αlβ1 integrin was found among all three subsets of cells, with al expressed on a subset of Mac-l+ granulocyte/monocytes, a subset of Grl+ neutrophils, and on the majority of infiltrating CD3+ T lymphocytes. Detailed immunohistochemical analysis revealed that although anti-αl and anti-α2 mAb treatment reduced the numbers of infiltrating cells, no change in the cellular composition of the infiltrate was seen (data not shown). Immunohistochemistry<br>
staining with a FITC anti-hamster mAb confirmed the ability of the anti-al and anti-α2 mAb to localize to the inflamed footpad (data not shown). Example 9<br>
Inhibition of arthritis by mAbs to αlβ1 and α2βl and in αl-deficient mice. As αl β1 is well expressed on infiltrating cells in the synovium of arthritis patients, we decided to examine whether anti-αl or anti-α2 mAbs would be inhibitory in an accelerated model of arthritis previously described (Terato et al., 1992, J. Immunol 148:2103-2108; Terato et al., 1995, Autoimmunity 22:137-147). This model involves injection of a cocktail of anti-collagen type n mAbs into mice, followed later by LPS administration, resulting in the development of arthritis over the next 3-7 d. Mice were given mAb every 3rd day starting at d 0, and scored for the development of arthritis every 3rd day. Severe arthritis developed in all mice within 72 h after LPS injection and persisted for more than 3 weeks. Neither injection of anti-collagen mAbs alone nor LPS alone induced arthritis. Mice receiving control mAb treatment displayed equally severe arthritis as than seen in PBS-treated mice (Fig. 7). In contrast, treatment with anti-αl mAb alone resulted in a marked reduction (79% and higher) in arthritis, lasting the duration of the experiment. Treatment with anti-α2 mAb alone also had a beneficial effect, resulting in a 37% decrease in the arthritic score as compared to control mAb-treated mice. The combination of anti-al and anti-α2 mAbs resulted in a similar degree of inhibition as seen with anti-al mAb alone. Reduction of arthritic score with anti-αl mAb treatment was seen in all mice and compares favorably with several other mAb-based treatments for arthritis such as soluble TNF receptor Ig fusion protein (Mori et al., 1996,J. Immunol. 157:3178-3182), anti-Mac-1 (Taylor et al., 1996, Immunology. 88:315-321), anti-α4 (Seiffge, 1996, J.Rheumatol 23:2086-2091), and anti-ICAM-1 (Kakimoto et al., 1992, Cell Immunol. 142:326-337). In agreement with mAb-based data showing an important role for αlβl in arthritis, untreated αl-deficient mice showed significant reduction in arthritic score when compared to wild-type mice.<br>
Example 10<br>
Effect of anti-α1 mAb treatment on the immunopathology of arthritic joints. Joints from wild-type arthritic mice (day 8) receiving either control mAb or anti-αl mAb treatment were compared visually and histologically to joints from a normal untreated mouse. Visually, joints from control mAb-treated mice demonstrated redness and swelling of the entire foot including digits, while anti-αl mAb-treated mice showed little if any signs of inflammation in either joints or digits. Histologic examination showed severe changes in control mAb-treated arthritic joints, with extensive infiltration of the subsynovial tissue with inflammatory cells, adherence of cells to the joint surface, and marked cartilage destruction as evidenced by proteoglycan loss. Consistent with previous reports (Terato et al, 1992, J. Immunol 148:2103-2108; Terato et al., 1995, Autoimmunity 22:137-147), the majority of the infiltrating cells in this model are neutrophils. Anti-αl mAb treatment of mice dramatically reduced the amount of inflammatory infiltrate and the degree of cartilage destruction. Example 11<br>
Development of arthritis is delayed in the absence of lymphocytes and inhibition of arthritis by anti-αl mAb occurs in the absence of lymphocytes. To determine what cell types might be important in the collagen mAb-induced arthritis model we compared the ability of wild-type B6-129 mice and RAG-1-deficient B6-129 mice to develop arthritis (Fig. 8). Genetic deletion of the RAG-1 (recombination activating gene-1) gene results in a complete loss of mature T and B lymphocytes (Mombaerts et al., 1992, Cell 68:869-877). Both the wild-type and RAG-1 -deficient mice developed arthritis, though the kinetics of induction in the RAG-1-deficient mice is significantly slower (Fig. 8). These results suggest that while lymphocytes are involved in this model of arthritis, they are not required for the development and progression of the disease. Published reports examining the effect of the RAG-1-deficient mice in other models of arthritis also found that loss of T and B lymphocytes delayed the onset of arthritis (Plows et al., 1999, J. Immunol. 162:1018-1023). Treatment of either wild-type or RAG-1-deficient mice with anti-al mAb completely inhibited arthritis (Fig. 8). These results demonstrate that the effectiveness of anti-αl<br>
mAb in this model is not dependent on the presence of lymphocytes and that as suggested by previous experiments (Fig. 7), the efficacy of anti-αl mAb in preventing disease may be through its action on other αl-expressing cells, such as macrophages and neutrophils. Example 12<br>
Dose response of  anti-αl mAb inhibition of'arthritis. Given the striking effects of anti-al mAb treatment on preventing arthritis, we extended these studies to include a dose response analysis (Fig. 9). Different doses of mAb were admininstered i.p. every 3rd day starting at day 0. In agreement with earlier data, a 250 ug dose of anti-al mAb resulted in near complete prevention of arthritis. A lower dose of 100 ug of anti-al mAb was partially effective at preventing arthritis in this model, while lower doses did not have any discemable effect on arthritic score (Fig. 9). Example 13<br>
Therapeutic treatment with anti-al mAb can decrease arthritic score. Given the effectiveness of anti-αl mAb in preventing arthritis, we attempted to treat mice that are on their way to develop disease. Arthritis was induced in mice by injection of a cocktail of anti-collagen type II mAbs on day 0, followed by LPS administration on day 3, Mice were then treated with either anti-al mAb or a soluble TNF receptor Ig fusion protein starting on day 4. Progression of arthritis was completely blocked in mice receiving anti-al mAb starting at day 4, when compared to mice receiving control hamster mAb starting at day 4 (Fig. 10). The degree of inhibition seen with therapeutic administration of anti-al mAb was complete and was equal to that seen with preventative treatment of anti-al mAb (started at day 0) (Fig. 10). In comparison, treatment with TNF receptor Ig fusion protein from day 4 onwards resulted in only a 60-70% inhibition in arthritic score when compared to control Ig fusion protein (Fig. 10). Combined treatment of anti-al mAb and TNF receptor Ig fusion together was effective at completely inhibiting arthritic score, which is not surprising given the complete effectiveness of anti-αl mAb treatment alone in suppressing arthritis. In summary, these results indicate that therapeutic treatment<br>
with anti-αl mAb is effective at inhibiting arthritic score, and compares favorabl; therapeutic treatment with a TNF antagonist Example 14<br>
Cloning and mutagenesis of the al-I domain. Human and rat αl β1 integrin I domain sequences were amplified from full length cDNAs (Kern, et al., 1994,J.Biol Chem. 269, 22811-22816; Ignatius etal., 1990, J. CellBiol. 111, 709-720) by the polymerase chain reaction (PCR) (PCR CORE Kit; Boehringer Mannheim, GmbH Germany), using either human specific primers, 5'-CAGGATCCGTCAGCCCCACATTTCAA-3' [forward] (SEQ ID NO:7), and 5'-TCCTCGAGGGCTTGCAGGGCAAATAT-3' [reverse] (SEQ ID NO:8), or rat specific primers,<br>
5'-CAGGATCCGTCAGTCCTACATTTCAA-3' [forward] (SEQ ID NO:9), and 5'-TCCTCGAGCGCTTCCAAAGCGAATAT-3! [reverse] (SEQ ID NO: 10).<br>
The resulting PCR amplified products were purified, ligated into pGEX4t-i (Pharmacia), and transformed into competent DH5α cells (Life Technologies). Ampicillin resistant colonies were screened for the expression of the —45 kDa glutathione S-transferase-I domain fusion protein. The sequences from inserts of plasmid DNA of clones that were selected fpr further characterization were confirmed by DNA sequencing.<br>
A rat/human chimeric αl-I domain (RAH) was generated (MORPH    . Mutagenesis kit; 5 prime - 3 prime), exchanging the rat residues G92, R93, Q94, and L97 (Fig. 11) for the corresponding human residues, V, Q, R, and R, respectively. Clones harboring the RAH I domain were identified by the loss of a diagnostic Stu 1 restriction enzyme site, and the inserts confirmed by DNA sequencing. The amino acid sequence of the human al-I domain is shown in Fig. 12. Example 15<br>
Generation of mAbs specific to the αl-I domain. Monoclonal antibodies have proved to be very useful probes in studying the relationship between structure and function of integrin subunits. For example, mAbs were used extensively to study regions of the β1 subunit associated with an activated conformation (Qu, A., and Leahy, D. J. (1996) Structure 4, 931-942). Thus, to identify potential probes for<br><br>
conformational changes of the αl-I domain, we generated a panel of mAbs to the human αl-I domain.<br>
Generation of anti-αl I domain Monoclonal Antibodies. Female Robertsonian mice (Jackson Labs) were immunized intraperitoneally (i.p.) with 25 µg of purified human αlβ1 (Edwards et al, 1995; .7. Biol. Chem. 270,12635-12640; Gotwals et al., 1999, Biochemistry 38:8280-8) emulsified with complete Fruend's adjuvant (LifeTechnologies). They were boosted three times i.p. with 25 µg of αlβ1 emulsified with incomplete Freunds's adjuvant (LifeTechnologies). The mouse with the highest anti-αl-I domain titer was boosted i.p. with 100 µg of alpl three days prior to fusion, and intravenously with 50 µg of αlβ1 one day prior to fusion. Spleen cells were fused with FL653 myeloma cells at a 1:6 ratio and were plated at 100,000 and 33,000 per well into 96 well tissue culture plates.<br>
Supematants were assessed for binding to the αlβl integrin by single color FACS. Prior to FACS analysis, supematants were incubated with untransfected K562 cells to eliminate IgG that bound solely to the β subunit. Subsequently, 3-5 X 104 K562 cells transfected with the al integrin subunit (K562-αl) suspended in FACS buffer (1% fetal calf serum (FCS) in PBS containing 0.5% NaN3) were incubated with supernatant for 45 minutes at 4° C, washed and incubated with anti-mouse IgG conjugated to phycoerythrin. After washing twice with FACS buffer, cells were analyzed in a Becton Dickinson Flow Cytometer.<br>
Supemantants from the resulting hybridomas were screened for binding to the al-I domain. Briefly, 50 µl of 30 µg/ml human αl-I domain-GST fusion in PBS was coated onto wells of a 96 -well plate (Nunc) overnight at 4° C. The plates were washed with PBS, blocked with 1% BSA in PBS and the hybridoma supernatant was incubated with the I domain at room temperature for 1 hour. After extensive washing with PBS containing 0.03% Tween 20, alkaline phosphatase linked anti-mouse IgG (Jackson ImmunoResearch) was added for an additional hour. After a final wash, 1 mg/ml p-nitrophenylphosphate (pNPP) in 0.1 M glycine, 1 mM ZnCI2, and 1 mM MgCl2 was added for 30 minutes at room temperature, and the plates were read at O.D. 405.<br>
Selected supernatants were tested for their ability to inhibit K562-al dependent adhesion to Collagen IV. K562-αl cells were labeled with 2 mM 2,7(bis-2-carboxyethyl-5 and 6) carboxyfiuorescein penta acetoxymethylester (BCECF; Molecular Probes) in DMEM containing 0.25% BSA at 37° C for 30 minutes. Labeled cells were washed with binding buffer (10 mM Hepes, pH 7.4; 0.9% NaCl; and 2% glucose) and resuspended in binding buffer plus 5 mM MgCl2 at a final concentration of 1 X 106 cells/ml. 50 µl of supernatant was incubated with an equal-volume of 2 X 105 K562-αl cells in wells of a 96 well plate. The plate was then centrifuged and the supernatants removed. Cells were resuspended in binding buffer and transferred to wells of a collagen-coated plate and incubated for 1 hour at 37° C. Following incubation, the non-adherent cells were removed by washing three times with binding buffer. Attached cells were analyzed on a Cytofluor (Millipore).<br>
We initially identified 19 hybridomas, the supernatants of which bound to human leukemia K562 cells expressing the α1β1 integrin (K562-αl) and to the αl-I domain. The irnmunoglobulins were purified from each of these hybridomas and tested for the ability to block either K562-αl or αl-1 domain binding to collagen IV. The mAbs fall into two classes: those that block and those that do not block αlβl function. For example, while the mAbs produced by clones AEF3, BGC5, AQC2 and AJH10 bind the ctl-I domain (Fig. 13A, data not shown for BGC5), only mAbs AJH10 and AQC2 inhibit αl-I domain-dependent (Fig. 13B; Fig. 16B) or K562-αl (Fig. 13C; Fig. 16C) adhesion to collagen IV.<br>
Sequencing of the Complementarity Determining Regions. To establish the clonal origin of this panel of mAbs, we amplified by PCR and sequenced the CDRs from 12 of the 19 antibodies (data not shown).<br>
2 µg of mRNA, isolated from 107 hybridomas (FastTrack mRNA isolation kit, Invitrogen), was reverse transcribed (Ready-To-Go You Prime First Strand Kit, Pharmacia Biotech) using 25 pM each of the following primers: heavy chain VH1FOR-2 (Michishita et al., 1993, Cell 72:857-867); light chain, VK4FOR, which defines four separate oligos (Kern et al., 1994, J. Biol. Chem. 269:22811-22816). For each hybridoma, heavy and light chains were amplified in four separate PCR reactions .using various combination of the following oligos: 1) Heavy chain:<br>
VH1FR1K (Kamata et al., 1995, J. of Biol. Chem. 270:12531-12535), VH1BACK, VH1BACK (Baldwin et al.(1998)-Structure 6, 923-935), VHfrla, VHfrlb, VHfrle, VHfrlf, VHfrlg (Ignatius et al. (1990) J. CellBiol. 111, 709-720), or VH1FOR-2 .(Michishita, M., Videm, V., and Amaout, M. A. (1993) 'Cell 72, .857-867); 2) Light chain: VK1BACK (Baldwin et al. (1998) Structure 6,923-935), VK4FOR, VK2BACK oligos (Kem et al. (1994) J. Biol Chem. 269,22811-22816), or VKfrla, VHfrlc, VHfrle, VHfrlf (Ignatius et al. (1990) J. Cell Biol. 111, 709-720). Products were amplified (5 min at 95° C, 50 cycles of 1 min at 94° C, 2 min at 55° C, 2 min at 72° C, and a final cycle of 10 min at 72° C), gel purified (QIAquick, Qiagen), and sequenced directly using various of the listed oligos on an ABI 377 Sequencer.<br>
Sequences from clones producing function-blocking mAbs were nearly identical across all the complementarity-determining regions (CDRs) and the intervening framework regions suggesting that these hybridomas are clonally related. Example 16<br>
Immunoblotting and FACS Analysis. Sequences of the variable regions of the non-blocking antibodies were markedly different from the clonally related family of sequences found for the blocking antibodies. As the blocking antibodies appear to originate from a single clone, we chose two (AJH10 and AQC2) to characterize further.<br>
Immunoblotting The smooth muscle cell layer dissected from sheep aorta, and K562-αl cells were extracted with 1% Triton X-l 00 in 50 mM Hepes, pH 7.5,150 mM NaCl, 10 mM phenylmethylsulfonyl flouride (PMSF), 20 µg/ml aprotinin, 10 µg/ml leupeptin, 1.0 mM ethylenediaminetetraacetic acid (EDTA). Samples were subjected to 4-20% gradient SDS-PAGE, and electroblotted onto nitrocellulose membranes. The blots were blocked with 5% dry milk in TBS; washed in TBS containing 0.03% Tween-20, and incubated with antibodies in blocking buffer containing 0.05% NaN3 for 2 hours. Blots were then washed as before, incubated with horseradish peroxidase conjugated anti-mouse IgG for one hour, washed again and then treated with ECL reagent (Amersham). Blots were then exposed to film (Kodak) for 30 to 60 seconds, and developed.<br>
Immunoblotting and .FACS analysis (Fig. 14) demonstrate that AJH10 reacts with human, rabbit, and sheep, but not rat αlβl integrin suggesting that the blocking mAbs bind to an evolutionarily conserved, linear epitope. The non-blocking mAbs were neither efficient at immunoblotting nor did they react with species other than human. Example 17<br>
Binding of the αl-1 domain to collagen is divalent cation-dependent<br>
A.	Purification of the αl-I domains.<br>
The al-I domains were expressed in E. coli as GST (glutathione-S-transferase) fusion proteins containing a thrombin cleavage site at the junction of the sequences. The clarified supernatant from cells lysed in PBS was loaded onto a glutathione Sepharose 4B column (Pharmacia) which was washed extensively with PBS. The al-I domain-GST fusion protein was eluted with 50 mM Tris-HCl, pH 8.0, 5 mM glutathione (reduced). For denaturation studies, the I domain was cleaved with thrombin in 50 mM Tris, pH 7.5, and purified from the GST fusion partner. DTT was added to 2 mM and the sample was loaded on a glutathione Sepharose 4B column. The flow-through and wash fractions were pooled and loaded onto a Q Sepharose FF column (Pharmacia). The αl-I domain was eluted with 50 mM Tris HC1, pH 7.5, 10 mM 2-mercaptoethanol, 75 mM NaCl. The purified I domain displayed its predicted mass (Lee et al. (1995) Structure 3,1333-1340, 871 Da) by electrospray ionization-mass spectrometry (ESI-MS), migrated as a single band by SDS-PAGE, and the protein eluted as a single peak of appropriate size by size exclusion chromotography on a Superose 6 FPLC column (Pharmacia).<br>
B.	Functional Analysis<br>
96 well plates were coated overnight at 4° G with 1 µg/ml collagen IV (Sigma) or collagen Type I (Collaborative Biomedical), washed with Triton buffer (0.1% TritonX-100; 1 mM MnCl2; 25 mM Tris-HCl; 150 mM NaCl), and blocked with 3% bovine serum albumin (BSA) in 25 mM Tris-HCl; 150 mM NaCl (TBS). Serial dilutions of the αl-1 domain-GST fusion protein in TBS containing I mM MnCl2 and 3% BSA were incubated on the coated plates at room temperature for 1 hour, and washed in Triton buffer. Bound αl-1 domain was detected with serial<br>
additions of 10 µg/ml biotinylated anti-GST polyclonal antibody (Pharmacia); ExtrAvidin-horseradishperoxidase (Sigma) diluted 1:3000 in TBS containing 1 mM MnCl2 and 3% BSA, and 1-Step ABTS (2,2'-Azine-di[3-ethylbenzthiazoline sulfonate]; Pierce). Plates were read at O.D. 405 on a microplate reader (Molecular Devices).<br>
Results.<br>
The human and rat (95% identity to human)αl-I domains were expressed in E. coli as GST-fusion-proteins and purified over glutathione sepharose. Both proteins were examined for binding to collagen I and IV using a variation of an ELISA-based assay previously described (Qu, A., and Leahy, D. J. (1995) Proc. Natl. Acad. Sci. USA 92, 10277-10281). The human al-I domain binds collagen IV with better efficiency than collagen I (Fig. 15A). An antibody specific to the αl-I domain, but not an antibody specific to the a2-I domain (Fig. 15B) abrogated binding to both ligands (data for collagen I is not shown). Both Mn2+ and Mg2+ stimulated binding, and EDTA reduced binding to background levels (Fig. 15C). No measurable differences in ligand binding were detected between the human and rat αl-I domains suggesting that the sequence differences between species are not functionally relevant (data not shown). Thus, the αl-I domain, specifically, require cation for efficient ligand binding. Example 18<br>
A Cation-Dependent Epitope Resides near the MIDAS motif. We exploited the observation that AJH10 recognizes the human, but not the rat al-I domain sequences to map the epitope for the αlβ1 function-blocking mAbs. The human and rat sequences differ by only 12 amino acids, 4 of which lie in a stretch of 6 amino acids (aa 92-97, Fig. 11 A) adjacent to the critical threonine (Fig. 11 A, aa 98) within the MIDAS motif. To test the hypothesis that the 6 amino acid residues, Val-Gln-Arg-Gly-Gly-Arg (residues 91-97 of SEQ ID NO:64), comprise the epitope for the blocking mAbs, we constructed a chimeric I domain (R∆H), exchanging the rat residues G92, R93, Q94, and L97 for the corresponding human residues, V, Q, R, and R, respectively. AJH10, along with all the function-blocking mAbs, recognizes the chimeric I domain (RAH; Fig. 11B).<br><br>
To orient these residues with respect to the MIDAS domain in the<br>
tertiary structure of the αl-I domain, we modeled the al-I domain using the coordinates of the crystal structure of the α2 I domain<br>
A homology model of the human αl I-domain was built using the X-ray crystal structure of the human α2 I-domain (Ward et al. (1989) Nature 341, 544-546). The model was built using the homology modeling module of Insight n (version 2.3.5; Biosym Technologies). The program CHARMM (Clackson et al. (1991) Nature 352, 624-628) was used with the all-hydrogen parameter set 22 with a distant dependent dielectric constant of two times the atom separation distance. We first did 1000 steps of steepest descent minimization with mass-weighted harmonic positional constraints of lkcal/(mol A2) on all atoms of the al-I domain. This minimization was followed by another 1000 steps of steepest descent and 5000 steps of Adopted-Basis Newton Raphson with constraints of 0.1 kcal/( mol A2) on the C-a atoms of the al-I domain to avoid significant deviations from the a2-I domain X-ray crystal structure.<br>
The αlβl and α2βl integrin sequences exhibit 51% identity with no insertions or deletions, suggesting that the overall structure of the two I domains will be similar. The metal coordination site is predicted to be the same in the al-I domain as in the α2-I domain, and the residues that comprise the epitope for the blocking mAbs lie on a loop between helix a3 and helix a4 which contains the threonine within the MIDAS motif critical for cation binding. The αl-I domain model predicts that the amide nitrogen of Q92 (Fig. 11A) hydrogen bonds with the carbonyl group of 133, the residue adjacent to S32. Thus, the loop that contains the epitope may play a functional role in stabilizing the MIDAS region. Example 19<br>
Monoclonal antibody AQC2 (i.e., mAQC2; "m" for murine) (Example 15, supra) is an IgG,, kappa antibody. To identify the nucleotide sequences encoding the heavy and light chains of this antibody, total cellular RNA from AQC2 murine hybridoma cells was obtained by using a QIAGEN RNEASY midi kit in accordance with the manufacturer's instructions. Then cDNAs encoding the variable regions of the heavy and light chains were cloned by RT-PCR from total cellular RNA using a GBCO BRL SUPERSCRIPT Preamplification System for First Strand cDNA<br>
Synthesis following the manufacturer's recommended protocol. Random hexamers<br>
were used for priming.	<br>
The heavy chain  variable domain of mAQC2 was amplified by  PCR from the first strand cDNA with the primers: 5' TGA GGA GAC GGT GAC CGT GGC CCT TGG CCC C 3' (SEQ ID NO: 11) and 5! AGG TSM ARC TGC AGS ACT CWG G 3' (S=C/G, M=A/C, R=A/G, and W=A/T) (SEQ ID NO:12). The PCR was subjected to 30 cycles using Clontech's Advantage Taq polymerase: denature 30 sec at 94°C, anneal 1 min at 50°C, and elongate 1.5 min at 68°C. The mAQC2 light chain with its signal sequence was amplified by PCR using the primers: 5' ACT AGT CGA CAT GGA TTT WCA GGT GCA GAT TWT CAG CTT C 3' (W=A/T) (SEQ ID NO: 13) and 5' ACT GGA TGG TGG GAA GAT GGA 3' (SEQ ID NO: 14). The PCR was subjected to 30 cycles using Stratagene's cloned Pfu polymerase: denature 1 min at 94°C, anneal 1 min at 50°C, and elongate 2 min at 72°C. The PCR products for the heavy and light chains were gel-purified using a QIAGEN QIAQUICK gel extraction kit following the manufacturer's recommended protocol.<br>
Purified heavy chain product was subcloned into Invitrogen's pCR2.1-TOPO TA vector using its TOPO TA cloning kit. Purified light chain was subcloned into Invitrogen's pCRblunt IITOPO vector using its Zero blunt TOPO cloning kit following the manufacturer's recommended protocol. Inserts from multiple independent subclones were sequenced. With the exception of degenerate positions within the PCR primers, the insert sequences of the independent subclones were identical.<br>
The polypeptide sequences of mAQC2 were deduced from their coding sequences. The N-terminal amino acid sequence for the mature light chain predicted by the cDNA sequence from the PCR product amplified with a signal sequence exactly matched the N-terminal sequence of purified mAQC2 light chain derived from Edman degradation (DVKVVESGG; SEQ ID N0:15). BLAST analyses of the variable domain sequences confirmed their immunoglobulin identity.<br>
The polypeptide sequence of the light chain variable domain of mAQC2 is shown below:<br>
.1    QIVLTQFPAL	MSASPGEKVT	MTCSASSSVN HMFWYQQKPK<br>
41   SSPKPWIYLT	SNLASGVPAR	FSGSGSGTSY  SLTISSMEAE<br>
81   DAATYYCQQW	SGNPWTFGGG	TKLEIK   106<br>
(SEQIDNO:1)<br>
The CDRs are shown in boldface. The CDRs are defined according to Kabat et al., Sequences of Proteins of Immunological Interest, 5th Edition, The United States Department of Health and Human Services, The United States Government Printing Office, 1991. Using the Kabat numbering system, SEQ ID NO:1 is represented as. follows, where a dash denotes the absence of an ammo acid:<br>
1 QIVLTQFPAL MSASPGEKVT MTCSASS-SV NHMFWYQQKP 41 KSSPKPWIYL TSNI&amp;SGVPA RFSGSGSGTS YSLTISSMEA 81 EDAATYYCQQ WSGNPWTFGG GTKLEIK 107<br>
The polypeptide sequence of the heavy chain variable domain of mAQC2is:<br>
1	DVKWESGGG	LVKPGGSLKL	ACAASGFSFS	RYTMSWVRQI<br>
41        PEKRLEWVAT	ISGGGHTYYL	DSVKGRFTIS	RDNAKNTLYL<br>
81        QMSSLRSEDT	AMYYCTRGFG	DGGYFDVWGQ	GTTVTVSS<br>
(SEQIDNO:2)<br>
The CDRs are shown in boldface. Usmg the Kabat numbering system, SEQ ID N0:2 is represented as follows, where positions numbers are consecutive numerals unless otherwise indicated:<br>
1	DVKVVESGGG     LVKPGGSLKL     ACAASGFSFS     RYTMSWVRQI<br>
41	PEKRLEWVAT     ISGGGHTYYL     DSVKGRFTIS     RDNAKNTLYL<br>
81	QM<br>
82a-c	SSL<br>
83	RSEDTAMY	YCTRGFGDGG<br>
lOOa-b   YF<br>
101       DVWGQGTTVT  VSS     113<br>
As used herein, residue position numbers of variable domains are designated in accordance with the Kabat numbering system unless otherwise indicated. Example 20<br>
This example describes the generation of a murine-human chimeric antibody, chAQC2.<br>
The cDNAs encoding the variable regions of the mAQC2 heavy and light chains were used to construct chAQC2 expression vectors, in which the mAQC2 variable regions were linked to human IgG1 and kappa constant regions.<br>
The heavy chain chimera was constructed as follows. A 0.33 kb Pstl-BstEII fragment from the mAQC2 heavy chain plasmid pANDO83 was subcloned into the phosphatased 2.82 kb Pstl-BstEII vector fragment from the 5a8 heavy chain plasmid pLCB7, so as to add a murine heavy chain signal-encoding sequence and a murine splice donor site to the cDNA of the mAQC2 heavy chain variable region. 5a8 is a molecularly cloned CD4-specific mAb (see, e.g., Boon et al, 2002, Toxicology 172:191-203). In the mature heavy chain encoded by the resultant plasmid (pAND092), the N-terminus differed by five residues from the N-terminus (DVKVVE; SEQ ID NO: 16) of the cognate mAQC2 heavy chain.<br>
To correct the heavy chain N-terminus, pAND092 was subjected to unique site elimination (USE) mutagenesis using an USE mutagenesis kit (Amersham Pharmacia Biotech) following the manufacturer's recommended protocol. The Q1D, Q3K, L4V, Q5V, Q6E "substitutions were encoded by the mutagenic primer 5' GCA CCA GGT GCC CAC TCC GAC GTC AAG GTG GTG GAG TCA GGG GGA GGC TTA GTG 3' (SEQ ID NO: 17). Mutated plasmid clones were identified by their new Aatn and Hinfl sites and eliminated PstI site. The heavy chain coding sequence was then confirmed by DNA sequencing. The correctly mutated plasmid was called pAND094. The 0.43 kb Notl-HindIIIfragment from pAND094 and the 1.21 kb HindII-Notl fragment from the plasmid pEAG964 (containing a coding sequence for<br>
a human IgG1 constant region) were subcloned into the NotI site of pCH269, a plasmid derived from the pCEP4 EBV expression vector (Invitrogen). The resultant plasmid was named pAND099.<br>
The light chain chimera was generated as follows. A 0.46 kb EcoRI fragment from the mAQC2 light chain variable domain plasmid pANDO8l was subcloned into the phosphatased 2.7 kb vector fragment of the pUC-derived pNN09 cloning vector, to add a 5' NotI site. The resulting plasmid, pAND091, was subjected to mutagenesis using the Amersham USE kit (supra) to introduce a BgIII site at the 3' end of the coding sequence. The mutagenic primer had the sequence 5' GGA GGC AGC AAG CTG GAG ATG TAA CGG GCT GAT GCT GC 3' (SEQ ID NO: 18). The correctly mutated plasmid was identified by its BgIII and BstYI site changes. The light chain coding sequence in the resultant plasmid pAND093 was confirmed by DNA sequencing. Then the 0.44 kb Notl-BgIII light chain variable domain fragment from pAND093 and the 0.68 kb BcII-NotI fragment from the plasmid pEAG963 (containing a coding sequence for a human kappa light chain constant domain) were subcloned into the NotI site of pCH269 (supra), producing plasmid pAND102. To create an unblocked kappa light chain (Q1E), pAND093 was subjected to USE mutagenesis with the mutagenic primer 5' CAT AAT GTC CAG GGG AGA AAT TGT TCT CAC CCA G 3' (SEQ ID NO: 19), to introduce an XmnI site. The mutated plasmid was identified by screening for an XmnI site change. The light chain sequence in the resultant plasmid pAND097 was confirmed by DNA sequencing. The 0.44 kb NotI-BgIII light chain variable domain fragment from pAND097 and the 0.68 kb BclI-NotI fragment from the plasmid pEAG963 (containing a human kappa light chain constant domain) were subcloned into the NotI site of pCH269, producing plasmid pAND098.<br>
To generate chAQC2 antibodies, expression vectors (chAQC2 heavy chain vector pAND099 + chAQC2 light chain vector pAND 102, and chAQC2 heavy chain vector p AND 099 + chAQC2 unblocked light chain vector pAND098) were co-transfected into 293-EBNA cells. The transfectants were tested for antibody secretion and specificity. The controls were cells transfected with the corresponding vectors without an insert or with DNA constructs encoding ch5c8 (a molecularly<br>
cloned CD154-specific mAb described in, e.g., Elster et al., 2001, Transplantation 72:1473-1478') or chCBE11 (a molecularly cloned LTβR-specific mAb described in, e.g., Browning et al., 1996, J. Biol Chem. 271:24934-24938).<br>
Then transfectants with the desired antibody secretion were lysed, and protein A immunoprecipitation was performed on the lysates and conditioned medium. Western blot analysis of the precipitates performed with anti-human heavy and light chain antibodies indicated that chAQC2-transfected cells synthesized and efficiently secreted heavy and light chains at levels similar to ch5c8-transfected and chCBEll-transfected cells. Further, huVLA-1-expressing K562α cells were stained with the conditioned medium from the transfected cells, and FACS analysis was performed on the stained cells. The results indicated that the chAQC2 antibody produced staining patterns similar to those of mAQC2, while conditioned media from mock-transfected and ch5c8-transfected cells failed to stain K562αl cells. Chimeric AQC2 produced from scaled-up transient transfection was purified and shown to bind to VLA-1 by FACS titration. Chimeric AQC2 with either a wildtype or a genetically unblocked light chain bound to VLA-1. See also Figs. 16A-D (discussed below). Example 21<br>
This example describes a method of humanizing the mAQC2 monoclonal antibody.<br>
Analysis of the mAQC2 variable domains. The variable domains in me light and heavy chains of mAQC2 were compared with the consensus sequences for mouse and human subgroups (Kabat et al, supra) using the software program FAST A. The light chain variable domain was found to be a member of mouse subgroup VI with 89% identity in a 109 ammo acid overlap. This domain also corresponded to human subgroup I with 72% identity in a 113 amino acid overlap. The heavy chain variable domain was found to be a member of mouse subgroup IIId with 86% identity in a 129 amino acid overlap. This heavy chain variable domain also corresponded to human subgroup in with 79% identity in a 130 amino acid overlap.<br>
The CDRs were categorized into canonical classes according to Chothia et al., Nature 342, pp. 877-883 (1989). The key residues defining each<br>
canonical class determine to a large extent the structural conformation of the CDR loop, and thus should be retained in the reshaped antibody The LI loop of mAOC2 fell into canonical class 1 (10 residue loop), L2 into class 1 (7 residue loop) and L3 into class 1 (9 residue loop). The H1 loop fell into class 1 (5 residue loop) and the H2 loop into class 1 (16 residue loop) residues. The H3 loop did not seem to belong to-any canonical class. The canonical residues important for these classes were all included in the humanized antibodies.<br>
Unusual framework residues in mAQC2 were determined by analyzing all mouse and human variable chain sequences in the September 1999 version of the Kabat database. It was believed that mAQC2-specific differences might indicate somatic mutations that enhance binding affinity if these differences were close to the binding site. Unusual mAQC2 residues further away from the binding site and unusual human framework residues were removed in case they would create immunogenic epitopes in the humanized antibody. Unusual framework residues<br>
found in mAQC2 were 7(F), 10(L), and 41 (K) in the light chain; and 4(V), 21 (A), and 40(1) in the heavy chain. None of these unusual mouse framework residues were retained the humanized antibodies.<br>
Modeling the structure of the variable regions. The light and heavy chains of mAQC2 were aligned against a nonredundant database to determine which structural frames to use to construct three-dimensional models of the mAQC2 light and heavy chains. Using FASTA, the light chain was found to have 82% sequence identity to monoclonal murine antibody ab57 (1CLOL), whereas the heavy chain was found to have 76% sequence identity to murine 6d9 Fab fragment (1HYY). Using the molecular modeling software package SYBYL (Tripos Inc.), the approximate three- -dimensional structures of the mAQC2 light and heavy chains were built using the light chain of ab57 and the heavy chain of 6d9, respectively. The structural integrity of the models was assessed at the console and was found to be reasonable,<br>
Design of the reshaped variable regions. Two approaches were used to choose human acceptor frameworks to "accept" mAQC2's CDRs. The first approach was by homology matching and the other by using consensus human Ig sequences. Under the homology approach, the Kabat database, the nonredundant<br>
database from NCBI, ENTREZ (The National Institutes of Health), and the Incyte database were searched using the software programs FASTA and BLAST. The choice of human acceptor frameworks was made based on sequence identity between mAQC2 frameworks and human frameworks (excluding frameworks from previously humanized antibodies) and the source of the antibody.<br>
The frameworks from an immunoglobulin variable region gene having a GENBANK. accession number of gi:587330 (human kappa subgroup I Vκ-lcl47) were eventually chosen for the light chain of the humanized antibody (Welschof et al., J. Immunol. Meth. 179:203-14 (1995)). The frameworks from Amulc11 (Kabat ED 044469; human subgroup III) were chosen for the heavy chain of the humanized antibody (Huang et al., J. Immunol 151:5290-300 (1993)).<br>
Back mutations of the human frameworks. Strategies for determining which back mutations to make are available on the H unionization bY Design web sites under mirrored urls: http://mathbio.nimr.mrc.ac.uk/isaldan and http://www.crvst.bbk.ac.uk/~ubcg07s. Previous experiments have shown that it is important to retain canonical residues, interface packing residues and unusual murine residues that are close to the binding site. In addition, residues in the "Vernier Zone," which forms a platform on which the CDRs rest (Foote et al., J. Mol Biol. 224, p. 487 (1992)) and those close to CDR H3 should be considered.<br>
Four reshaped versions were designed for each of the variable light and heavy chains, as shown in Table 1. Two of the four versions for each chain were designed by homology matching (designated huAQC2-hl and -h2) and the other two versions by consensus matching (huAQC2-cl and -c2). It should be noted that the sequences for huAQC-hl heavy chain and huAQC-cl heavy chain are identical. Table   1,   Sequences of mAQC2,   huAQC2,   and human frameworks<br>
(Table Removed)<br>
*Dashes indicate identity with the iLAQC2 amino acid sequence. **Part of SEQ ID NO:1; ***Part of SEQ ID NO:2. SEQ ID NOS: 67, 44, 42, 2, 42 and 68, respectively, in order of appearance.<br>
in all versions. (2) heavy chain:<br>
1 E-&gt;D     This back mutation was made in h1 (i.e., cl) only. 121-&gt;V    The residue I is unusual in human and was retained in the h2 only.<br>
28	T-&gt;S    This is a vernier residue and was retained in h1 only.<br>
29	V-&gt;F   This is a canonical residue and was retained in all versions.<br>
49 S-&gt;A   This is a vernier residue and was retained in all versions.<br>
93	A-&gt;T   This is a vernier residue and interfacial and was retained in all<br>
versions.<br>
94	S-&gt;R    This is a canonical residue and was retained in both versions.<br>
The huAQC2 variable regions were made by USE mutagenesis as described above, using the chAQC2 variable domain plasmids as starting templates. The human acceptor framework ("FR") cDNA sequences were Kabat #Z37334 for the light chain and Kabat #U00490 for the heavy chain. To facilitate identification of mutated plasmids, silent mutations were introduced to change restriction sites. Mutated plasmids were identified by the restriction site changes. The variable region cDNA sequences in the resultant plasmids were confirmed by DNA sequencing.<br>
The h1 and cl versions of heavy chain (which were identical) were made by using plasmid pAND094 as template. The mutagenic primers were: FR1 primer 5'GGT GCC CAC TCC GAC GTC CAG CTG GTC GAG TCA GGG GGA GGC TTA GTC CAG CCT GGA GGG TCC CTG AGA CTC TCC TGT GCA GCC TCT GGA TTC 3' (SEQ ID N0:20), which introduced TaqI and PvuII" sites, and eliminated a Ddel site; FR2 primer 5' ATG TCT TGG GTT CGC CAG GCT CCG GGG AAG GGG CTG GAG TGG GTC GCA ACC 31 (SEQ ID NO:21), which introduced a NciI site, and eliminated BspEI and Earl sites; FR3 primer 5' TTC ACC ATC TCC AGA GAC AAT TCC AAG AAC ACC CTG TAG CTG CAG ATG AAC AGT CTG AGG GCC GAG GAC ACA GCC GTG TAT TAG TGT ACA AGA 3' (SEQ ED N0:22), which introduced PstI and Ddel sites; and FR4 primer 5' TGG GGC CAA GGT ACC CTG GTC ACC GTC TCC TCA GGT GAG 3' (SEQ ID N0:23),<br>
which introduced KpnI and Eco0109I sites. The resultant hi (i.e., cl) heavy chain plasmid was designated pAND l04.<br>
The: c2 version of heavy chain were made by using p AND 104 as template with the following mutagenic primers: FR1 primer 5' TCC TGT GCA GCC TCT GGA TTC ACC TTC AGT AGG TAT ACT ATG TCT TGG GTT 3' (SEQ ID NO:24), which introduced an AccI site; and FR1 primer 5' GCA CCA GGT GCG CAC TCC GAG GTC GAG CTG GTC GAG TCA 3' (SEQ ID NO:25), which introduced an FspI site and eliminated an AatII site. The resultant c2 heavy chain plasmid was designated pAND l15.<br>
The h2 version of heavy chain were made by using p AND 115 as template with the following primer: FR1 primer 5' GAG TCA GGG GGA GGC TTA ATC CAG CCT GGA GGG TCC CTG 3' (SEQ ID NO:26), which eliminated a Ddel site. The resultant h2 heavy chain plasmid was designated pANDl 13.<br>
To generate expression vectors for the huAQC2 heavy chains, the 0.43 kb No tI-Hindin heavy chain variable domain fragment from p AND 104, pANDl 15, or pANDl 13, and the 1.21 kb HindIII-Notr fragment from pEAG964 (supra) were subcloned into the NotI site of pCH269 (supra). The resultant heavy chain expression plasmids were designated p AND 114 (h1), pAND121 (c2), and pAND124 (h2), respectively.<br>
The h1 version of light chain were made by using plasmid pAND093 as template. The mutagenic primers were: FR1 primer 5' CAA ATT GTT CTC ACC CAG TCT CGA TCC TCC CTG TCT GCG TCT GTA GGG GAC AGA GTC ACC ATC ACA TGC AGT GCC AGC TCA 3' (SEQ ID N0:27), which removed BstEII and PstI sites; FR2 primer 5' TTC TGG TAT CAG CAG AAG CCC GGG AAA GCC CCC AAA CCC TGG ATT 3' (SEQ ID NO:28), which introduced an Neil site; FR3 primer 51 GCT TCT GGA GTC CCT TCA CGC TTC AGT GGC AGT GGG TCT GGG ACA GAT TAC ACT CTC ACA ATC AGC AGC CTG CAA CCT GAA GAT TTT GCC ACT TAT TAG TGC CAG 3' (SEQ ID NO:29), which introduced a Ddel site and eliminated Eco0109I and Avail sites; and FR4 primer 5' GGT GGA GGC ACT AAG GTG GAG ATC TAA CGG GCT 3' (SEQ ID NO:30), which introduced Ddel and Styl sites. The resultant h1 light chain plasmid was designated pANDl 03.<br>
The h2 version of light chain were made by using pAND 103 as template with the followinng primer: FR2 primer 5'CCC GGG AAA GCG CCC AAA GTC CTG ATT TAT CTC ACA TCC 3' (SEQ ID NO:31), which introduced HhaI and Haell sites. The resultant h2 light chain plasmid was designated pANDl 16.<br>
The cl version of light chain used plasmid pAMD103 template with the following primers: FRl primer 5' GCC TCA GTC ATA ATG TCC CGG GGA CAA ATT CAG CTC ACC CAG TCT CCA TCC 3' (SEQ ID NO:32), which introduced Smal, NeiI, and HpaII sites; FR4 primer 51 GGT AAC CCG TGG ACG TTC GGT CAG GGC ACT AAG GTG GAG ATC TAA CGG GCT 3' (SEQ ID NO:33), which introduced a Bspl286I site. The resultant cl light chain plasmid was designated pAND118.<br>
The c2 version of light chain were made by using plasmid p AND 116 template with the following primers: FRl primer 51 GCC TCA GTC ATA ATG TCC CGG GGA CAA ATT CAG CTC ACC CAG TCT CCA TCC 3' (SEQ ID N0:34), which introduced Smal, NciI, and HpaII sites; FR4 primer 5' GGT AAC CCG TGG ACG TTC GGT CAG GGC ACT AAG GTG GAG ATC TAA CGG GCT 3' (SEQ ID NO:35), which introduced a Bspl286I site. The resultant c2 light chain plasmid was designated p AND 119.<br>
To generate expression vectors for the huAQC2 light chains, the 0.44 kb Notl-Bgin light chain variable domain fragment from pAND103, pANDl 16, pANDl 18, or pANDl 19, and the 0.68 kb BclI-NotI fragment from pEAG963 (supra} were subcloned into the NotI site of pCH269 (supra). The resultant light chain expression vectors were designated pANDl 17 (h1), pAND120 (h2), pAND122 (cl), and pAND123(c2), respectively.<br>
The expression vectors were co-transfected into 293-EBNA cells, and transfected cells were tested for antibody secretion and specificity. Cells transfected with an empty vector served as negative control. The whole cell lysates and the conditioned medium were immuno-precipitated with protein A Western blot analysis of the precipitates (developed with anti-human heavy and light chain antibodies) indicated that huAQC2-transfected cells synthesized and efficiently secreted heavy and light chains at levels similar to chAQC2-transfected cells.<br>
FACS analysis of VLA-1 -expressing K562αl cells stained with conditioned medium from the transfected cells was then performed. To do so, the K562αl cells were incubated with the conditioned medium on ice for 120 min. The cells were then washed three times with a FACS buffer (PBS with .5% FBS and 0.05% sodium azide). The washed cells were resuspended in the buffer and incubated with PE-conjugated anti-human IgG (H + L) (Jackson ImmunoResearch Laboratories, Inc.) on ice for 30 min on ice. After the incubation, the cells were washed three times with the FACS buffer, and resuspended in the FACS buffer for analysis. The data are shown in Table 2, in which HuAQC2-hl refers to an mAb consisting of the h1 version of the huAQC2 heavy chain (HC) and the h1 version of the huAQC2 light chain (LC) (see Table 1). Likewise, huAQC-h2 is an mAb consisting of the h2 versions of the heavy and light chains, huAQC2-cl the cl versions, and huAQC2-c2 the c2 versions. In the table, relative MFI refers to mean MFI normalized to that observed for chAQC2 blocked. Data shown represents the average from two independent transfections. These data indicated that the huAQC2-h2 and -c2 mAbs bound less well than huAQC2-hl and -cl relative to chAQC2. Table 2. FACS staining of K562α1 cells by chAQC2 and huAQC2<br><br>
(Table Removed)<br>
Co-transfections of 293-EBNA cells with chAQC2 and huAQC2-hl, -h2, -cl and -c2 were scaled up. Antibodies in the conditioned media were purified with Protein A-Sepharose. Purified mAbs were assayed by FACS for activity. The protocol as follows.<br><br>
1.	Count cells from flask that was split 1:4 on the day prior to the assay.<br>
2.	Pellet cells and resuspend at 2.5e5 cells/ml in FACS bufferY5% FBS in PBS- with<br>
.0.02%NaAzide).<br>
3.	Pipette 100 µl of cells into the wells of a 96 well V bottom plate.<br>
4.	Prepare 1:3 seriald ilutidns of AQC2 starting at 3 µg/ml in FACS buffer.<br>
5.	Pellet the cells for 5 minutes at 800 X g and flick plate to remove buffer.<br>
6.	Resuspend the cells in 100 µl of the diluted antibody series.<br>
7. Incubate for 2 hours on ice.<br><br>
8.	Wash plate. Pellet the cells for 3 minutes at 800 X g and flick plate to remove<br>
buffer.<br>
9.	Resuspend the cells in 100 µl of secondary antibody (diluted 1:100 in FACS<br>
buffer).<br><br>
10.	Incubate for 30 minutes on ice.<br>
11.	Wash plate (see above).<br>
12.	Resuspend cells in 25 µl of FACS buffer.<br>
13.	Centrifuge the FACS tubes briefly to ensure that the 50 µl is in the bottom of the<br>
tubes.<br>
14.	Vortex each tube vigorously and collect 5000 events.<br>
The data are shown in Fig. 17. These data confirmed that huAQC2-h2 and -c2 bound less well than huAQC2-hl and -cl relative to chAQC2.<br>
The consensus versions of huAQC2 were studied further because they would be less immunogenic when used to treat patients With chronic indications. Mix-and-match cotransfectidns were perfonned to identify whether a single chain was responsible for the apparent decrease in binding seen with huAQC2-c2. The co-transfections suggested that the.reduction could be attributed to the c2 light chain (encoded by pAND123), which differed from the cl light chain (encoded by pAND122) at only two residues in the FR2 region: P46L and W47L.<br>
To examine the individual contributions of each of these two changes, new c2 light chain expression vectors were constructed. Plasmid pAND125, the L47W variant of the c2 light chain was made using pANDl 19 as a template with the<br>
following mutagenic primer: FR2 primer 5' GGG AAA GCA CCC AAA CTC TGG ATC TAT CTC ACA TCC AAC 3' (SEQ ID NO:36), which introduced HhaI and Haell sites. Plasmid pAND126, the L46P variant of the c2 light chain, was made by using pAND 119 as a template with the following mutagenic primer: FR2 primer 5' AAG CCC GGG AAG GGG CCC AAA CCC CTG ATT TAT CTC. ACA TCC AAC y (SEQ ID NO:37), which introduced BsaHI, BanI, and NarI sites. Expression vectors for these new huAQC2 light chains were made by subcloning-the 0.44 kb Notl-Bglll light chain variable domain fragment from pAND125 or pAND126, and the 0.68 kb BcII-NotI fragment from pEAG963 (supra) into the NotI site of pCH269 (supra). The resultant plasmids were designated pAND132 (c2 with L47W), and p AND 13 3 (c2 with L46P), respectively.<br>
Co-transfections of the new light chain plasmids with each of the huAQC2 heavy chain plasmids were performed. VLA-l binding was examined by FACS. The data demonstrate that the L47W back mutation failed to improve binding. The L46P mutation improved the peak of the binding curve, but the EC50 was still right-shifted relative to the behavior of huAQC2 version 1 (Table 2, supra). These results suggested that both back mutations were needed for full binding activity.<br>
A genetically unblocked cl light chain was also made, since the Q1D variant would be one residue more "humanized." The QID mutant, designated p AND 148, was made with the template pANDl 18 with the following mutagenic primer: FR1 primer 5' GTC ATA ATG TCC CGG GGA GAT ATC CAG CTC ACC CAG TCT 3' (SEQ ID NO.38), which introduced a new EcoRI site and removed an Apol site. An expression vector for this last variant of the huAQC2 light chain was made by subcloning the 0.44 kb Notl-BgIII light chain variable domain fragment from pAND148 and the 0.68 kb BcII-NotI fragment from pEAG963 into the NotI site of pCH269, producing the light chain expression vector pANDl5O (cl with unblocked N-terminus Q1D). Co-expression of the genetically unblocked light chain with the c2 heavy chain (i.e., "huAQC2 DC cl unblocked/HC c2"; designated huAQC2-c4) was equivalent to that of "huAQC2 LC cl/HC c2" (designated as huAQC2-c3). VLA-l binding was confirmed by FACS on VLA1 -expressing TC562ccl cells (Table 2).<br>
Co-transfections of 293-EBNA cells with chAQC2 and huAQC2-hl, -h2, -cl, -c2, -c3, and -c4. Antibodies in the conditioned media were purified on Protein A-Sepharose. The purified mAbs were assayed for activity (Figs. 17 and 18). HuAQC2-c3 was chosen as the drug candidate, since its properties were more similar to chAQC2. Vectors were then designed for stable expression of huAQC2-c3 in CHO cells. The vectors contained a cDNA for the huAQC2 cl LC or c2 HC, with the 5' and 31 UTRs eliminated and the heavy chain C-terminal lysine genetically deleted to. ensure product homogeneity. The final vectors were pAND 162 (light chain), pAND 160 (heavy chain). As used herein, huAQC2-c3 is also called hAQC2.<br>
The full polypeptide sequences of hAQC2 are as follows.<br>
Light Chain (Plasmid: pAND 162)<br>
1	QIQLTQSPSS  LSASVGDRVT  ITCSASSSVN HMFWYQQKPG<br>
KAPKPWIYLT<br>
51        SNLASGVPSR  FSGSGSGTDY  TLTISSLQPE  DFATYYCQQW<br>
SGNPWTFGQG<br>
101     TKVEIKRTVA APSVFIFPPS   DEQLKSGTAS  WCLLNNFYP<br>
REAKVQWKVD<br>
151     NALQSGNSQE  SVTEQDSKDS   TYSLSSTLTL  SKADYEKHKV<br>
YACEVTHQGL<br>
201     SSPVTKSFNR GEC<br>
(SEQIDNO:3)<br>
Heavy Chain (Plasmid: pAND160)<br>
1	EVQLVESGGG  LVQPGGSLRL  SCAASGFTFS  RYTMSWVRQA<br>
PGKGLEWVAT<br>
:51        ISGGGHTYYL DSVKGRFTIS. RDNSKNTLYL QMNSLRAEDT AVYYCTRGFG<br>
101     DGGYFDVWGQ  GTLVTVSSAS   TKGPSVFPLA  PSSKSTSGGT AALGCLVKDY<br>
151     FPEPVTVSWN SGALTSGVHT  FPAVLQSSGL  YSLSSVVTVP SSSLGTQTYI<br>
201     CNVKHKPSNT  KVDKKVEPKS   CDKTHTCPPC  PAPELLGGPS<br>
VFLFPPKPKD<br>
251     TLMISRTPEV. TCWVDVSHE  DPEVKFNWYV. DGVEVHNAKT<br>
KPREEQYNST<br>
301     YRWSVLTVL  HQDWLNGKEY  KCKVSNKALP  APIEKTISKA<br>
KGQPREPQVY<br>
351     TLPPSRDELT  KNQVSLTCLV  KGFYPSDIAV  EWESNGQPEN<br>
NYKTTPPVLD<br>
401     -SDGSFFLYSK LTVDKSRWQQ  GNVFSCSVMH  EALHNHYTQK SLSLSPG<br>
(SEQE)NO:4)<br>
Other heavy and light chain polypeptide and nucleotide sequences are shown below.<br>
A. chAQC2 heavy chain (Pand099) (SEQ ID N0s:39 and 40. The former No refers to the nucleotide sequence and the latter to the polypeptide sequence.  The same order is used in the following numbering.)<br>
1<br>
GACGTCAAGGTGGTGGAGTCAGGGGGAGGCTTAGTGAAGCCTGGAGGGTCCCTGAAA CTC<br>
DVKVVESGGGLVKPGGSL K  L<br><br>
GCCTGTGCAGCCTCTGGATTCAGTTTCAGTAGATATACTATGTCTTGGGTTCGCCAG ATT   .<br>
A     C     A     A     S     G     F     S     F     S     R     Y     T     M     S     W     V     R Q     I<br>
121<br>
CCGGAGAAGAGGCTGGAGTGGGrCGCAACCATTAGTGGTGGTGGTCACACCTACTAT<br>
CTA<br>
PEKRLEWVATI      SGGGHTY Y   ,L<br>
181<br>
GACAGTGTGAAGGGCCGATTCACCATCTCCAGAGACAATGCCAAGAACACCCTGTAC.<br>
CTG<br>
DSVKGRFTISRDNAKNTL Y     L<br>
241<br>
CAAATGAGCAGTCTGAGGTCTGAGGACACAGCCATGTATTACTGTACAAGAGGTTTT<br>
GGA<br>
QMSSLRSEDTAMYYCTRG F     G<br>
301 GACGGGGGGTACTTCGATGTCTGGGGCCAAGGGACCACGGTCACCGTCTCCTCA<br>
DGGYFDVWGQGTTVTVSS<br>
B.   hAQC2  HC  h1   and  c1    (pAND114)    (SEQ  ID NOs:41  and  42)<br>
1<br>
GACGTCCAGCTGGTCGAGTCAGGGGGAGGCTTAGTCGAGCCTGGAGGGTGCCTGAGA CTC<br>
DVQ'LVESGGGL.VQPGGSL R   L<br>
61<br>
TCCTGTGCAGCCTCTGGATTCAGTTTCAGTAGATATACTATGTCTTGGGTTCGCCAG GCT<br>
SCAASGFSFSRYTMSWVR Q A<br>
121<br>
CCGGGGAAGGGGCTGGAGTGGGTCGCAACCATTAGTGGTGGTGGTGACACCTACTAT<br>
CTA<br>
PGKGLEWVATISGGGHTY<br>
Y     L      . ••    .<br>
181<br>
GACAGTGTGAAGGGCCGATTCACCATCTCCAGAGACAATTCCAAGAACACCCTGTAC<br>
CTG<br>
DSVKGRFTISRDNSKNTL Y     L<br>
241<br>
CAGATGAACAGTCTGAGGGCCGAGGACACAGCCGTGTATTACTGTACAAGAGGTTTT<br>
GGA<br>
QMNSLRAEDTAVYYCTRG .F     G<br>
301 GACGGGGGGTACTTCGATGTCTGGGGCCAAGGTACCCTGGTCACCGTCTCCTCA<br>
DGGYFDVWGQGTLVTVSS<br>
C.   hAQC2 h2 heavy chain   (pAND124)    (SEQ ID NOs:43  and 44)<br>
1<br>
GAGGTCCAGCTGGTCGAGTCAGGGGGAGGCTTAATCCAGCCTGGAGGGTCCCTGAGA CTC<br>
EVQLVESGGGLIQPGGSL R  L<br>
€1<br>
TCCTGTGCAGCCTCTGGATTCACCTTCAGTAGGTATACTATGTCTTGGGTTCGCCAG GCT<br>
SCAASGFTFSRYTMSWVR<br>
Q     A<br>
121<br>
CCGGGGAAGGGGCTGGAGTGGGTCGCAACCATTAGTGGTGGTGGTCACACCTACTAT<br>
CTA<br>
PGKGLEWVATISGGGHTY Y     L<br>
181<br>
GACAGTGTGAAGGGCCGATTCACCATCTCCAGAGACAATTCCAAGAACACGCTGTAC<br>
CTG<br>
DSVKGRFTISRDNSKNTL<br>
Y     L<br>
241<br>
CAGATGAACAGTCTGAGGGCCGAGGACACAGCCGTGTATTACTGTACAAGAGGTTTT<br>
GGA<br>
QMNSLRAEDTAVYYCTRG F     G<br>
301<br>
GACGGGGGGTACTTCGATGTCTGGGGCCAAGGTACCCTGGTCACCGTCTCCTCAGG DGGYFDVWGQGTLVTVSS<br>
D.   hAQC2   c2   heavy  chain   (pAND121)    (SEQ   ID  NOs:45   AND   69)<br>
1<br>
GAGGTCCAGCTGGTCGAGTCAGGGGGAGGCTTAGTCCAGCCTGGAGGGTCCCTGAGA CTC<br>
EVQLVESGGGLVQPGGSL R  L<br>
61<br>
TCCTGTGCAGCCTCTGGATTCACCTTCAGTAGGTATACTATGTCTTGGGTTCGCCAG GCT<br>
SCAAS     GFTFSRYTMSWVR Q    A<br>
121<br>
CCGGGGAAGGGGCTGGAGTGGGTCGCAACCATTAGTGGTGGTGGTCACACCTACTAT<br>
CTA<br>
PGKGLEWVATI      SGGGHTY Y     L<br>
181<br>
GACAGTGTGAAGGGCCGATTCACCATCTCCAGAGACAATTCCAAGAACACCCTGTAC<br>
CTG<br>
DSVKGRFTISRDNSKNTL Y     L<br>
241<br>
CAGATGAACAGTCTGAGGGCCGAGGACACAGCCGTGTATTACTGTACAAGAGGTTTT<br>
GGA<br>
QMNSLRAEDTAVYYCTRG<br>
301<br>
GACGGGGGGTACTTCGATGTCTGGGGCCAAGGTACCCTGGTCACCGTCTCCTCAGG DGGYFDVWGQGTLVTVSS<br>
E.   chAQC2  blocked light chain   (Pandl02)    (SEQ  ID N0s:46 and 47)<br>
1 CAAATTGTTCTCACCCAGTTTCCAGCACTCATGTCTGCGTCTCGAGGGGAGAAGGTCACC<br>
QIVLTQFPALMSASPGEK V  T<br>
61<br>
ATGACCTGCAGTGCCAGCTGAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCA AAA<br>
M  T  C  S  A  S  S  S  V N  H  M  F  W  Y  Q  Q  K P  K<br>
121<br>
TCCTCCCCCAAACCGTGGATTTATGTCACATCCAACCTGGCTTCTGGAGTCCCTGCT<br>
CGC<br>
SSPKPWIYLTSNLASGVP A  R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACCTCTTACTCTCTCACAATCAGCAGCATGGAGGCT<br>
GAA<br>
FSGSGSGTSYSLTI  SSME A  E<br>
241<br>
GATGCTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTGGA<br>
GGC<br>
DAATYYCQQWSGNPWTFG-G     G 301   ACCAAGCTGGAGATCAAA<br>
T     K     L     E     I     K<br>
F. hAQC2 h1 light chain (pAND117) (SEQ ID N0s:48 and 49)<br>
1  CAAATTGTTCTCACCCAGTCTCCATCCTCCCTGTCTGCGTCTGTAGGGGACAGAGTCACC<br>
Q     T     V     LL    T     Q    S    P    S    S   L.    S     A    S.  V    G     D     R V     T<br>
61<br>
' ATCACATGCAGTGCGAGCTCAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCC GGG<br>
ITC     S     A     S     SV     N     H     M     F     W     Y     Q     K<br>
•P     G	.<br>
121<br>
AAAGCCCCCAAACCCTGGATTTATCTCACATCCAACCTGGGTTCTGGAGTGCCTTCA CGC<br>
KAPKPWIYLTSNLASGVP S  R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACAGATTACACTCTCACAATCAGCAGCCTGCAACCT<br>
GAA<br>
FSGSGSGTDYTLTI      SSLQ P     E<br>
241<br>
GATTTTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTGGA<br>
GGC<br>
DFATYYCQQWSGNPWTFG' G     G<br>
301  ACTAAGGTGGAGATCAAA T     K     V     E      I      K<br>
G.   hAQC2  h2   light   chain   (pAND120)    (SEQ  ID NOs:50   and 51)<br>
1<br>
CAAATTGTTCTCACCCAGTCTCCATCCTCCCTGTCTGCGTCTGTAGGGGACAGAGTC ACC<br>
QIVLTQSPSSLSASVGDR V  T<br>
61 ATCACATGCAGTGCCAGCTCAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCC<br>
GGG		_.<br>
ITCSASSSVNHMFWYQQK P     G<br>
121<br>
AAAGCGCCCAAACTCCTGATTTATCTCACATCCAACCTGGCTTCTGGAGTCCCTTCA<br>
CGC<br>
KAPKLLIYLTSNLASGVP S     R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACAGATTACACTCTCACAATCAGCAGCCTGCAACCT<br>
GAA<br>
FSGSGSGTDYTLTISSLQ P     E<br>
241<br>
GATTTTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTGGA<br>
GGC<br>
DFATYYCQQWSGNPWTFG G     G<br>
301   ACTAAGGTGGAGATCAAA T     K     V     E     I     K<br>
H.   hAQC2   cl   light   chain   (pAND122)    (SEQ   ID  NOs:52   and  70)<br>
1<br>
CAAATTCAGCTCACCCAGTCTCCATCCTCCCTGTCTGCGTCTGTAGGGGACAGAGTC ACC<br>
Q     I     QLTQS     PS      S     L     S     A     S     VGD     R V     T<br>
61<br>
ATCACATGCAGTGCCAGC.TCAAGTGTAAATCACATGTTCTGGTATCAGGAGAAGCCC GGG<br>
ITCSASSSVNHMFWYQQK P  G<br>
121<br>
AAAGCCCCCAAACCCTGGATTTATCTCACATCCAACCTGGCTTCTGGAGTCCCTTCA<br>
CGC<br>
KAPKPWIYLTSNLASGVP S     R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACAGATTACACTCTCACAATCAGCAGCCTGCAACCT<br>
GAA<br>
FSG.SGSGTDYTLTI      SSLQ P     E<br>
241<br>
GATTTTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTCAG<br>
GGC<br>
DFATYYCQQWSGNPWTF'G Q    G<br>
301  ACTAAGGTGGAGATCAAA T     K     V     E     I     K<br>
I.  hAQC2  c2  light chain   (pAND123)    (SEQ ID N0s:53  and 54}<br>
1 CAAATTCAGCTCACCGAGTCTCCATCCTCCCTGTCTGCGTCTGTAGGGGACAGAGTC<br>
QIQLTQSPSSLSASVGDR V  T<br>
61<br>
ATCACATGCAGTGCCAGCTCAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCC GGG<br>
ITCSASSSVNHMFWYQQK P  G<br>
121<br>
AAAGCGCCCAAACTCCTGATTTATCTCACATCCAACCTGGCTTCTGGAGTCCCTTCA<br>
CGC<br>
KAPKLLIYLTSNLASGVP S  R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACAGATTACACTCTCACAATCAGGAGCCTGCAACCT<br>
GAA<br>
FSGSGSGTDYTLTISSLQ P  E<br>
241<br>
GATTTTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTCAG<br>
GGC<br>
DFATYYCQQWSGNPWTFG<br>
Q  G<br>
301 ACTAAGGTGGAGATCAAA T  K  V  E  I  K<br>
J. chAQC2 unblocked light chain (pAND098) (SEQ ID NOs:55 and 56)<br>
1<br>
GAAATTGTTCTCACGCAGTTTCCAGCACTCATGTCTGGGTCTCCAGGGGAGAAGGTC ACC<br>
EIVLTQFPALMSASPGEK V  T<br>
61<br>
ATGACCTGCAGTGCCAGCTCAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCA AAA<br>
MTCSASSSVNHMFWYQQK P  K<br>
121<br>
TCCTCCCCCAAACCCTGGATTTATCTCACATCCAACCTGGCTTCTGGAGTCCCTGCT<br>
CGC<br>
SSPKPWIY     TSNLASGVP A  R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACCTCTTACTCTCTCACAATCAGCAGCATGGAGGCT<br>
GAA<br>
FSGSGSGTSYSLTISSME A  E<br>
241<br>
GATGCTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTGGA<br>
GGC<br>
DAATYYCQQWSGNPWTFG G     G<br>
301 ACCAAGCTGGAGATCAAA T     K   L    E     I   K<br>
K. huAQC2 unblocked cl light chain (pANDl5O) (SEQ ID N0s:57 and 58)<br>
1<br>
GATATCCAGCTCACCCAGTCTCCATCCT.CCCTGTCTGCGTCTGTAGGGGACAGAGTC ACC<br>
D     I     QLTQS     P .    S     S     L     SASVGDR V     T<br>
61<br>
ATGACATGCAGTGCCAGCTCAAGTGTAAATCACATGTTCTGGTATCAGCAGAAGCCC GGG<br>
ITCSASSSVNHMFWYQQK P  G<br>
121<br>
AAAGCCCCCAAACCCTGGATTTATCTCACATCCAACCTGGCTTCTGGAGTCCCTTCA<br>
GGC<br>
KAPKPWIYLTSNLASGVP S     R<br>
181<br>
TTCAGTGGCAGTGGGTCTGGGACAGATTACACTCTCAGAATCAGCAGCCTGCAACCT GAA   .    .<br>
F     S     G     S     G     S     G     T     D     Y     T     L     T     I     S     S     L     Q P     E<br>
241<br>
GATTTTGCCACTTATTACTGCCAGCAGTGGAGTGGTAACCCGTGGACGTTCGGTCAG<br>
GGC<br>
DFATYYCQQWSGNPWTFG Q     G<br>
301 ACTAAGGTGGAGATCAAA<br>
T     K    V     E     I     K<br>
Example '22<br>
This example describes the characterization of various AQC2 antibodies of the invention.<br>
Solid-phase assay for αl I domain binding. Fifty µl of 10mg/ml αl I domain-GST fusion protein was added to a CORNING COSTAR EASY WASH polystyrene 96-well plate (Gotwals et al., Biochemistry, 38, 8280-8 (1999)). Following incubation at 4°C for 16 hrs, the plate was washed four times with 350 µl of 0.1 % Tween-20 in PBS in a plate washer. The plate was blocked by addition of 180 µl of 3% BSA in TBS at 25°C for 60 min, and then washed as above. Dilutions of antibodies (50 ul/well) in TBS containing 1 mg/ml BSA (assay buffer) were prepared in a 96-well roundbottom plate, transferred to the all domain-coated plate, and incubated for 60 min at 25°C. Following a final wash, 100 µI/well of TMB reagent (Pierce) was added. After 10 min, lOO.ul of 1 M sulfuric acid was added, and the absorbance at 450 run was read on a UV-Vis 96-well spectrophotometer.<br>
Electrochemiluminescence assays for binding of αl β1 integrin or al I domain to collagen. Tosyl-activated DYNABEADS M-280 (Dynal, Inc.) were coated with 100 fig/ml type IV collagen (Sigma) according to the manufacturer's instructions. Cell lysates from αl-transfected K562 cells were prepared as follows. Cells were collected by centrifugation, resuspended at 10s cells/ml in a lysis buffer containing 25 mM Tris, pH 7.4,1% NP-40,1 mM CaCl2,1 mM MnCl2,1 mM MgCl2,2% BSA, and 1 mM PMSF, and incubated at 4°C for 60 min. Cell debris was removed by centrifugation at 12,000 rpm for 30 min and the resulting supernatant was used hi subsequent experiments. Anti-pi activating antibody TS2/16 and polyclonal anti-GST antibody (Pharmacia) were labeled with TAG-NHS ester (IGEN International, Inc., Gaithersburg, MD) according to the manufacturer's instructions. Labeled antibodies were purified by gel filtration chromatography on SEPHADEX G25M (Pharmacia).<br>
To cany out the binding assay, collagen-coated beads (1 mg/ml) were blocked for 5 mm with 8% Lewis rat plasma in an assay buffer containing 50 mM HEPES, pH 7.5,150 mM NaCl, and 0.1% Triton X-100. For the αlβl binding assay,<br>
serial dilutions of antibodies were incubated with 10 µg, of beads, cell lysate prepared from-10s αl-traflsfected K562 cells (supra), and 0.1 ng/ml of TAG-TS2/16 in an assay buffer containing 1 mM MnCl2. For the al I domain binding assay, the antibodies were incubated with 10 ug of beads, 0.1 µg/ml αl I domain GST fusion protein, and 1 µg/ml of TAG-anti-GST in an assay buffer containing 1 mM MnCl2. After one to two hours of agitation at room temperature, 200 ul of the assay buffer was added and the samples were read on an ORIGEN 1.5 electrochemiluminescence detector (IGEN). Plots are presented with arbitrary electrochemiluminescence units (ECL) on the ordinate axis.<br>
Biotinylated mAQC2 competition assay. A 96-well plate was coated with 50 µl of 5 µg/ml all domain GST fusion protein and blocked with 3% BSA in TBS as described above. Dilutions of antibodies (60 µl/well) in the assay buffer were prepared in a 96-well roundbottom plate, and 60 µl of 0.1 µg/ml biotinylated murine AQC2 in the assay buffer was added. Fifty microliters from each well was transferred to the coated plate and incubated for 3 hrs at 25°C. The plate was then washed as above, 50 ul of 1 µg/ml peroxidase-conjugated EXTRAVIDIN (Sigma) was added, and the plate was incubated another 2 hrs at 25°C. After a final wash, 100 µl/well of TMB reagent (Pierce) was added. After 10 min, 100 µl of 1 M sulfuric acid was added, and the absorbance at 450 nm was read on a UV-Vis 96-well spectrophotom eter.<br>
Experimental results. The experimental results are shown in Figs. 16A-D and Table 3. The ability of mAQC2, chAQC2, hAQC2, and hAQC2' (i.e., huAQC2-c4; differing from hAQC2 only in that residue 1 of the hAQC2' light chain was D instead of Q) to (1) bind to human αl- transfected K562 cells (by FACS); (2) bind to immobilized al-I domain (by ELIS A); (3) compete with mAQC2 for binding to αl-I domain (ELISA); (4) block αl-I domain binding to collagen (Electrochemiluminescence assay); or (5) block αlβl integrin binding to collagen (Electrochemiluminescence assay) was determined. The results are shown in Figs. 16A-D, and calculated IC50 (for inhibition) or EC50 (for binding) values are given in Table 3. In each assay, each of the humanized AQC2 forms showed a similar ability to either bind VLA1 (or the al domain) or block binding to collagen (Note that in<br><br>
panel C, the observed difference in intensity between mAQC2 and the humanized forms derives from  the use of an anti-murine IgG secondary antibody, instead of an anti-human-IgG).<br>
Table 3. Summary of assay results (all values in nM)<br><br>
(Table Removed 3)<br>
We next tested whether changes at certain conservative residues in the CDRs could preserve the VLA-1 binding activity of hAQC2. DNA constructs encoding variants of hAQC2 with the following mutations were made by site-directed mutagenesis: (1) G55S in the heavy chain CDR2; (2) S24N in the light chain CDR1 (introducing an occupied N-linked glycosylation site); (3) G92S in the light chain CDR3; (4) a combination of (1) and (2); and (5) a combination of (1) and (3). The DNA constructs encoding both the heavy and light chains were then co-transfected into 293-EBNA cells, and the conditioned medium of the transfectants was assayed for antibody expression by Western blot and ELISA, The results indicated that the hAQC2 variants were expressed as efficiently as cognate hAQC2. FACS analysis using VLA-1-expressing K562 cells further showed that the VLA-1 -binding activities<br>
of these variants were similar to hAQC2 itself. In sum, the amino acid substitutions did not alter the VLA-1 binding activity of hAQC2. Indeed, X-ray crystal structure of the  RAH/hAQC2 Fab complex (infra) shows that S24 and G92 of the light chain and G55 of the heavy chain are not in the binding pocket that is in contact with the αl-I domain. Example 23<br>
The effector functions of an immunoglobulin couple the immunoglobulin's antigen-binding activity to the inflammatory, cytotoxic and stimulatory arms of the immune system. Effector functions may impair the safety and efficacy of an immunoglobulin therapeutic product. To reduce the potential effector functions of hAQC2, mutations of L234A and L235A were made to its heavy chain to generate hsAQC2. For the same reason, a single mutation of N297Q was made in the heavy chain of hAQC2 to generate an aglycosylated form of hAQC2, named haAQC2. Studies can be done to compare their efficacy, residual effector function, stability and immunogenicity to cognate hAQC2. Unless otherwise indicated, residue position numbers in constant regions as used herein are designated in accordance with the EU numbering convention.<br>
The heavy chain polypeptide sequence of haAQC2 is as follows (Plasmid: pAND161):<br>
1	EVQLVESGGG LVQPGGSLRL  SCAASGFTFS   RYTMSWVRQA<br>
PGKGLEWVAT<br>
51        ISGGGHTYYL DSVKGRFTIS   RDNSKNTLYL  QMNSLRAEDT<br>
AVYYCTRGFG<br>
101     DGGYFDVWGQ  GTLVTVSSAS   TKGPSVFPLA   PSSKSTSGGT<br>
ZVALGCLVKDY<br>
151     FPEPVTVSWN  SGALTSGVHT  FPAVLQSSGL  YSLSSWTVP<br>
SSSLGTQTYI<br>
201     CNVNHKPSNT  KVDKKVEPKS   CDKTHTCPPC   PAPELLGGPS<br>
VFLFPPKPKD<br>
251     TLMISRTPEV TCVWDVSHE  DPEVKFNWYV  DGVEVHNAKT<br>
KPREEQYQST<br>
.301     YRWSVLTVL HQDWLNGKEY  KCKVSNKALP  APIEKTISKA KGQPREPQVY<br>
351      TLPPSRDELT  KNQVSLTCLV  KGFYPSDIAV  EWESNGQPEN<br>
NYKTTPPVLD<br>
401      SDGSFFLYSK LTVDKSRWQQ  GNVFSCSVMH  EALHNHYTQK  SLSLSPG<br>
(SEQ ID N0:5)<br>
The heavy chain polypeptide sequence of hsAQC2 is as follows (Plasmid: pAND171):<br>
1	EVQLVESGGG LVQPGGSLRL- SCAASGFTFS  RYTMSWVRQA<br>
PGKGLEWVAT<br>
' 51        ISGGGHTYYL DSVKGRFTIS   RDNSKNTLYL  QMNSLRAEDT AVYYGTRGFG<br>
101     DGGYFDVWGQ  GTLVTVSSAS   TKGPSVFPLA  PSSKSTSGGT AALGCLVKDY<br>
151      FPEPVTVSWN  SGALTSGVHT  FPAVLQSSGL  YSLSSWTVP SSSLGTQTYI<br>
201      CNVNHKPSNT  KVDKKVEPKS   CDKTHTCPPC  PAPEAAGGPS VFLFPPKPKD<br>
251      TLMISRTPEV TCVWDVSHE  DPEVKFNWYV DGVEVHNAKT KPREEQYNST<br>
301     YRWSVLTVL HQDWLNGKEY  KCKVSNKALP APIEKTISKA KGQPREPQVY<br>
351     TLPPSRDELT  KNQVSLTCLV  KGFYPSDIAV EWESNGQPEN NYKTTPPVLD<br>
401      SDGSFFLYSK LTVDKSRWQQ  GNVFSCSVMH EALHNHYTQK  SLSLSPG-(SEQ ID NO:6) Example 24<br>
This example describes a method for determining the crystal structure of the complex of a rat/human chimeric αl-I domain of the αlβl integrin and the hAQC2 Fab fragment. Preparation of the protein complex<br>
The hAQC2 Fab fragment was prepared from hAQC2 antibody using .a variation of the procedure of the IMMUNOPURE® Fab preparation kit (Cat#44885,<br>
Pierce, Rockford, EL). The intact hAQC2 antibody was concentrated to 12 mg/ml in a buffer containing 20 mM phosphate, 10 mM EDTA and 25 mM cysteine (pH 7.0). Immobilized papain was added at an enzyme to substrate ratio of 1:50, and digestion was allowed to occur overnight at 37° C. The immobilized papain was removed and the crude digest was dialyzed against 20 mM sodium acetate buffer (pH 4.5). The Fab fragment was separated from residual intact antibody, dimeric Fab fragment, and Fc fragment by cation exchange chromatography using a S-column (Poros HS/M , PERSEPTIVE Biosytems #PQ42M26) with a shallow salt gradient. The Fab fragment was then exchanged into 0.1 M Hepes buffer (pH 8.0).<br>
The chimeric αl-I domain used in the present invention is a rat/human chimeric J domain construct (mutant RAH) containing residues Thrl45-Phe336 of the rat αl integrin chain, where residues Gly217, Arg218, Gln219 and Leu222 (crystal numbering) have been substituted with equivalent human residues Val, Gin, Arg and Arg, respectively, in order to restore antibody binding. The amino acid sequences of chimeric RAH, rat, and human αl-I domains are given below in SEQ ID NOs:59, 60 and 61, respectively. Recombinant al-I domain was expressed in E. coli as a GST-fusion protein. The RAH al-I domain was cleaved with thrombin and purified from a Pichia pastor is clone as described previously (Gotwals et al., 1999, Biochemist™ 3 8 :8280-8288).<br>
145	TQLDIV<br>
151	IVLDGSNSIY PWESVIAFLN DLLKRMDIGP KQTQVGTVQY<br>
191	GENVTHEFNL NKYSSTEEVL VAANKIVQRG GRQTMTALGI<br>
231	DTARKEAFTE ARGARRGVKK VMVI VTDGES HDNYRLKQVI<br>
271	QDCEDENIQR FSIAILGHYN RGNLSTEKFV EEIKSIASEP<br>
311	TEKHFFNVSD ELALVTIVKA LGERIF<br>
;SEQ ID N0:59)<br>
L45  TQLDIV<br>
L51  IVLDGSNSIY PWESVIAFLN DLLKRMDIGP KQTQVGIVQY<br>
191     GENVTHEFNL NKYSSTEEVL VAANKIGRQG  GLQTMTALGI 231     DTARKEAFTE ARGARRGVKK VMVIVTDGES  HDNYRLKQVI<br>
271     QDCEDENIQR  FSIAILGHYN RGNLSTEKFV EEIKSIASEP<br>
311     TEKHFFNVSD  ELALVTIVKA LGERIF<br>
(SEQIDNO:60)<br>
145	TQLDIV<br>
151	IVLDGSNSIY'PWDSVTAFLN DLLKRMDIGP  KQTQVGIVQY<br>
191	GENVTHEFNL  NKYSSTEEVL VAAKKIVQRG  GRQTMTALGI<br>
231	DTARKEAFTE ARGARRGVKK VMVIVTDGES   HDNHRLKKVI<br>
271	QDCEDENIQR  FSIAILGSYN RGNLSTEKFV  EEIKSIASEP<br>
311	TEKHFFNVSD  EIALVTIVKT LGERIF<br>
(SEQIDNO:61)<br>
The hAQC2 Fab fragment was mixed with excess chimeric αl-I domain and incubated at 37' C for 15.minutes. The saturated αl/Fab complexes were separated from uncomplexed αl-I domain by size exclusion chromatography using a S200 Sephacryl column (Pharmacia, Gibco). The complex was further concentrated to 11 mg/ml in a 20 mM Tris (pH 7:4), 150 mM NaCl, 1 mM MnCl2, 5 mM β-mercaptoethanol. Preparation of crystals<br>
Crystallization conditions were found using the CRYSTAL SCREEN™ KITs from Hampton Research (Laguna Niguel, CA). Crystals of the complex described above were grown at 20 C by vapor diffusion using an equal amount of protein complex solution and a 20-30% PEG 1500 reservoir solution. Typically, 2 µL of protein complex was added to 2 µL of well solution to yield drops of 4 µL. Crystals grew in two to seven days as hexagonal rods with dimensions 0.8 x 0.05 x 0.05 mm3. The presence of the al-I domain and hAQC2 Fab fragment was confirmed by SDS-PAGE analysis of dissolved crystals. In order to reduce the inherent radiation damage during data collection, X-ray diffraction data was collected at approximately 100 K. To prepare the crystals for data collection at this low<br>
temperature, crystals were gradually equilibrated into a cryoprotectant solution -containing 25% PEG 400 and 30% PEG 1500, and flash cooled in liquid nitrogen. Structure determination<br>
Native X-ray diffraction data to 2.8 A resolution were collected from a single crystal at about 100 K using an ADSC Quantum 4 charged-coupled device detector at beamline X4A of the Brookhaven National Laboratory (BNL) National Synchrotron Light Source (NSLS). Data was processed using the software programs DENZO and SCALEPACK (Otwinowski &amp; Minor, 1997, Methods in Enzymol 276:307-326). Crystals belonged to the space group P61or its enantiomorph P65, with unit cell dimensions a = b = 255.09 A, c=38.64 A. The data set was 96,6% complete and had an R-merge of 8.3%. The Matthews coefficient (Matthews, 1968, J. Mol. Biol. 33:491-497) was 2.59 A3 Da'1 with a solvent content of 52.1 %, which indicated that there were two complexes in the asymmetric unit. The two complexes in the asymmetric unit were related by non-crystallographic 2-fold symmetry, Data statistics are shown in Table 4.<br>
Molecular replacement searches were done with the program AMoRe (Navaza, 1994, Acta Cryst. A50:157-163) from the CCP4 program package (Collaborative Computational Project No.4. The CCP4 Suite: programs for protein crystallography. 1994, Acta Gyst. D50:760-763), and molecular graphics manipulations were done with the program QUANTA. A single αl-I domain from the structure of the rat αl-I domain of αlβ1 integrin (Protein Data Bank (PDB) accession number Ick4; Nolte et al., 1999,FEBS Lett, 452:379-385) was used as amodel or probe for rotation and translation searches. The translation function search indicated that the 1s1 and 9th highest peaks of the rotation function corresponded to the correct solutions for the two al-I domains in the asymmetric unit (correlation coefficient (cc) = 21.1%, R=53.1 %) and that the space group was P65. Subsequently, searches for the hAQC2 Fab fragments were done, keeping the I domain solutions fixed and using a model of the Fv domain of the hAQC2 Fab as a search probe. A clear solution was found for one of the two Fv domains (cc=22.1%, R=52.6 %), but the second Fv could not be located. The position of the second Fv was derived using the non-crystallographic 2-fold symmetry. Rigid body refinement of the two I domains and<br>
two Fv domains reduced the R-factor to 43.6% (R-free = 42.7 %). An 2Fo-Fc electron density map showed clear electron density for the constant domain (Fconst) of the first Fab fragment, but no density for the Fconst domain of the second Fab fragment. A model of the Fconst domain of the first Fab was manually fit in the observed electron density. Subsequent rigid body refinement with the software program CNX (Accelrys Inc., San Diego, CA ©2000; Brunger,1998, Acta Cryst. D54:905-921), using data in the 500-2.8 A resolution range, optimized the position of all domains, reducing the R-factor to 39.7 % (R-free = 38.9%).<br>
All subsequent refinement steps were carried out with the CNX program. To reduce model bias, partial models were used for 2Fo-Fc map calculation and model refinement. The initial partial model, was subjected to simulated annealing and grouped B-factor refinement with non-crystallographic symmetry restraints. The R-working and R-free factors dropped to 28.3% and 32.9%, respectively. Several cycles consisting of iterative model building, maximum likelihood positional refinement and B-factor refinement followed. Only model adjustments that resulted in a drop in the R-free factor were accepted. A bulk-solvent correction was employed after the complete model was built. The R-working and R-free factors of the final model are 21.3 % and 27.2 %, respectively for the data (F &gt; 2σ) in the 500-2.8 A resolution range.<br>
The final 2Fo-Fc electron density map is of good quality for most of the complex with the exception of amino acid residues 288-295 of one I domain fragment (molecule A in Fig. 19) that are associated with weak electron density and have not been included in the model. In addition, the entire constant domain of one Fab fragment has no visible electron density, which indicates that it is disordered. This appears to be consequence of the absence of crystal contacts for the constant domain of the Fab fragment due to its position within a large solvent channel. This domain was also not included in the final model that consists of 1030 amino acid residues, constituting 6 polypeptide chains, and 2 manganese ions. The r.m.s. positional deviation between equivalent residues from the two complexes in the asymmetric unit is small (0.37 A for 1660 equivalent main chain atoms). Stereochemistry statistics were calculated with the software programs PROCHECK<br>
 (Laskowski et al, 1993, J. Appl. Cryst. 26:283-291; Morris et al., 1992, Proteins 12:345-364) and CNX. Hydrogen bonds (
 <br>
Table 4: Summary of Data Statistics and Crystallographic Analysis<br>
(Table Removed 4)<br>
Example 25<br>
This example describes the crystal structure of the complex of a rat/human<br>
chimeric αl-I domain of the αlβl integrin and the hAQC2 Fab fragment. Architecture of Crystal Structure<br>
The crystal structure of the complex of the rat/human chimeric αl-I domain of the alpl integrin and the hAQC2 Fab fragment has an elongated shape (Fig. 20). The dimensions of the complex are 100 A x 50 A x 35 A.<br>
The Fab fragment exhibits the typical immunoglobulin fold. The light chain and heavy chains of the Fab fragment each form two broad sheets of anti-parallel P-strands which pack tightly together to form a scaffold for the complementarity determining region (CDR) loops which extend from the packed sheets. Both the light chain and the heavy chain contain three CDR loops. The light chain loops are called L1, L2 and L3, while the heavy chain loops are referred to as H1, H2 and H3.  The complementarity determining region (CDR) loops correspond to canonical structure 1 for light chain LI, L2 and L3 loops and for heavy chain III and H2 loops (Chothia et al., 1989, Nature 342:877-883). The heavy chain H3 loop has a tight β-hairpin-like conformation that is stabilized by internal hydrogen bonds as well as two aromatic residues (Tyrl04 and Phel05) that are packed against the light chain. Residue Thr50 of L2 adopts mainchain dihedral angles that fall in the disallowed regions of the Ramachandran diagram. The same observation for the corresponding residue has been made for other antibodies (Muller et al., 1998, Structure 6, pp. 1153-11567) which indicates that this is a natural characteristic of L2 loops.<br>
The αl-I domain in the present invention has a structure very similar to the uncomplexed al-I domain (PDB accession number Ick4; Nolte et al., 1999, FEBS Lett. 452:379-385; PDB accession code Iqc5; Rich et al.,1999, J. Biol.Chem. 274:24906-24913). The I domain structure exhibits a "dinucleotide-binding" or "Rossman" fold (Rao &amp; Rossman, 1973, J. Mol. Biol. 76:241-256) in which a central sheet of five parallel β-strands and one small antiparallel-strand is surrounded on both sides by a total of seven α-helices. The six P-strands of the structure in this invention will be referred to as βA, βB, βC, βD, βE, and βF and the seven a-helices are called αl, α2, α3, α4, α5, α6 and a7.<br>
Three characteristic structural features exist for I domains. The first characteristic feature is the presence of an inserted small helix in the βE-α6 loop, termed as the C helix. Most of the C helix loop of molecule A (Fig. 19) in the present invention is associated with weak electron density, which suggests disorder. This appears to be a consequence of absence of crystal contacts or contacts with the Fab that would have stabilized the loop. However, the same loop in molecule B (Fig. 19) in the present invention has well-defined electron density and has been included in the model. The second characteristic feature of α 1-1 domains is the MIDAS or Metal-Ion-Dependent-Adhesion-Site where metal ions and ligands are implicated to bind to the I domain. Five key residues which form part of the MIDAS are referred to as the "DxSxS-T-D" motif. These residues, which are completely conserved among I domains, coordinate the metal ion (Gotwals et al., 1999, Biochemistry 38:8280-8288). The crystals in the present invention were grown in the presence of manganese and the MIDAS site of the I domain in this structure is observed to contain a Mn+2 metal ion. The ion is directly coordinated by the side chains of residues Serl56, Serl58 and Thr224. The 2Fo-Fc electron density map shows no evidence that MIDAS residues Aspl54 and Asp257 make water-mediated indirect coordination of the metal ion (Fig. 20). However, the apparent absence of water molecules could be a consequence of the limited resolution (2.8 A) of the electron density map. The third feature of I domains is that all determined structures of I domains belong to one of two conformations called "open" and "closed". The differences between the open and closed conformation include a different mode of metal ion coordination and a significant (about 10 A) positional shift of the C-terminal helix of the I domain. The I.. domain in the complex in the present invention is in the closed conformation.<br>
In the structure of the complex in the present invention, the Fab fragment binds to its epitope on the front upper surface of the I domain with a footprint 35 A by 30 A. The total buried surface area in the antibody-antigen interface is 1534 A2 which is typical of other antibody-antigen complexes (Davies et al., 1996, Proc. Natl. Acad. Sci. USA 93:7-12; Jones &amp; Thornton, 1996, Proc. Nad. Acad. Sci. USA 93:13-20). The surface is 25% hydrophobic and 75% hydrophilic in character. The heavy chain contributes 65% of the buried surface area for the complex, while the<br><br>
remaining 35% is contributed by the light chain. The antibody epitope consists of residues located in four loops of the I domain (Emsley et al., 2000, Cell 101:47-56).<br>
Three of the loops form the MIDAS site: loop 1 (PA- al) which contains the conserved DXSXS sequence, loop 2 (α3-α4) which contains the MIDAS Thr224 and loop 3 (βD-α5) that contains MIDAS residue Asp257. The fourth loop is the C-helix loop and is involved in only in minor contacts.<br>
The central feature of the antigen-antibody interaction is the coordination of the MIDAS site metal ion by AsplOl from the CDR H3 of the antibody (Fig. 20). The distance between the ion and O61 of AsplOl is 2.4 A. In addition, the O62 atom of Aspl0l is interacting with His261 of the I domain. Interestingly, the CDR H3 contains several glycine residues adjacent to Aspl0l (sequence GFGDGGY)(SEQ ID N0:62), presumably to allow enough flexibility to the CDR loop to permit proper coordination of the metal ion. The CDR H3 sequence is essentially invariant in monoclonal antibodies that were raised against the same antigen and found to belong in the same class. Most of the antibody residues that are involved in antibody-antigen contacts are located in L3, HI, H2 and H3 CDR loops. A few residues from the LI (Asn30) and L2 (Tyr48) loops appear to form minor Van Der Waals contacts. L3 primarily contributes to contacts through two large hydrophobic residues, Trp90 and Trp95. In addition, Asn93 from L3 forms hydrogen bonds with Gln223 of the I domain. The side chains of His56 and Tyr58 from the H2 loop form hydrogen bonds with main chain atoms of loop 2 of the I domain. Arg31 of HI is in contact with Arg291 of loop 4 of the I domain. Arg222 from loop 2 of the I<br>
domain is sandwiched between several antibody residues including Tyr58, Trp95 and<br><br>
Asn93. This is the only residue out of the four mutated in the RAH I domain, that is involved in contacts with the Fab. It is therefore likely to be the only residue responsible for restoring the binding of the antibody after the mutagenesis. Comparison of the crystal structure of the complex of a rat/human chimeric αl-I domain and the hAQC2 Fab fragment with other I domain structures<br>
The chimeric RAH al-I domain has four sequence differences with the rat al-I domain (rat residues: 217G, 218R, 219Q and 222L), eight sequence    . differences with the human al-I domain (human residues: 163D, 166T, 214K, 264H,<br><br>
268K, 288S, 3221 and 38OT), and ten sequence differences with the clone used in the crystal structure studies of human αl-I domain (clone residues: 163Dr 166T, 174E,<br>
214K, 2301,264H,268K, 2888,3221 and 380T). In the unliganded rat αlβl αl-I domain crystal structure (PDB accession code Ick4; Nolte et al., 1999, FEES Lett. 452:379-385), the ccl-I domain contains no bound metal ions and adopts the "closed" conformation. In the unliganded human αl-I domain crystal structure (accession code 1Iqc5; .Rich etal.,. 1999, J. Biol. Chem. 274:24906-24913), the ccl-I domain contains bound Mg+2 and similarly adopts the closed conformation. Superimposition of these two structures with the complexed chimeric αl-I domain indicates that there are only minor conformational changes upon hAQC2 antibody binding. The r.m.s. positional deviation between the rat and chimeric αl-I domain is 1.04 A for all 768 main chain atoms. The r.m.s. positional deviation between the human and chimeric αl-I domain is 0.69 A for all 764 main chain atoms. The biggest differences (human and chimeric al-I domain pair) are observed in loop 1 (r.m.s. deviations 1.24 A for main chain atoms of residues 154-161) and the loop 4 (C helix loop) of the al-I domain (r.m.s. deviations 1.55 A for main chain atoms of residues 288-296). However, these differences can be more accurately described as shifts of the whole secondary structure elements rather than complex conformational changes. These are likely to be within the normal range of conformational flexibility of proteins. The r.m.s. positional deviation between the human and chimeric al-I domain for backbone atoms of amino acid residues GIul92, Gln218, Arg219, Gly220, and Gly221 (crystal numbering) is 0.33 A. The r.m.s. positional deviation between the rat and chimeric al-I domain for backbone atoms of amino acid residues Aspl54, Serl56, Asnl57, SerlSS, Tyrl60, Glul92, Gln218, Arg219, Gly220, Gly221, Arg222, Gln223, Thr224, Asp257, His261, Asn263, Arg291, and Leu294 (crystal numbering) is 0.97 A.<br>
The I domain maintains the "closed" I domain conformation that has been observed only for unliganded I domains crystallized in the absence of ligands or pseudo-ligands bound to the MIDAS site. The r.m.s. positional deviation of the C-terminal helices of the human and chimeric I domains (calculated for the main chain atoms of residues 321-335) is 0.64 A. A simulated annealing omit map calculated for the final refined model unambiguously confirms that the position of the<br>
C-terminal helix and adjacent.structural elements are consistent with the closed conformation.<br>
In order to investigate the effects of ligand binding to the modes of ,. metal ion coordination, the structure of the present invention was superimposed with the structures of the unliganded α2-I domain (PDB accession code laox; Emsley et al., 1997,J. Biol. Chem. 272:28512-28517) and the a2-I domain complexed with a collagen peptide (PDB accession code Idzi; Emsley et al., 2000, Cell 101:47-56). The coordination of the metal ion by Aspl0l from the antibody is remarkably similar to the coordination of the metal ion of the α2-I domain by a glutamic acid from the; collagen peptide. Another feature that is conserved is the simultaneous interaction of the acidic group with His261 (His258 in the α2-I domain). All MIDAS residues of the I domain-Fab complex except Serl56 and Serl58 adopt conformations very similar to those observed in the unliganded I domain. In contrast, the side chains of Serl56 and Serl58, as well as the metal, adopt conformations similar with those of the liganded I domain. It is clear that the coordination of the metal ion by AsplOl does not allow the ion to maintain the position and coordination distances that are observed in the unliganded state. Thus, the metal ion is not directly coordinated by Asp257, a fact that permits the ion to maintain high electrophilicity. Biological Implications<br>
In the present invention, there is no direct coordination of the metal by Asp257, which may permit high affinity binding by lowering the energy barrier between a closed (no ligand bound) and open (ligand bound) conformation. However, the coordination of the metal by an aspartic acid from the-antibody is not sufficient to induce the open conformation to the I domain in the present invention. The I domain - Fab complex structure indicates that it is possible to have strong binding to the I domain that adopts the closed conformation and that coordination of the metal ion by an acidic residue from the ligand may be necessary but not sufficient to induce a conformational change to the open state. Binding of the antibody is expected to stabilize the low affinity state of the integrin and prevent the outside-in signaling that would have accompanied integrin binding to collagen.<br>
Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be apparent to those skilled in the art that certain changes  and modifications will be practiced. Therefore, the description and examples should not be construed as limiting the scope of the invention.<br><br><br><br>
WE CLAIM:<br>
1.	A recombinant anti-VLA-1 antibody or antigen-binding fragment thereof<br>
whose light chain complementarity determining regions are defined by amino<br>
acid residues 24 to 33, 49 to 55, and 88 to 96 of SEQ ID NO: 1, and whose<br>
heavy chain complementarity determining regions are defined by amino acid<br>
residues 31 to 35, 50 to 65, and 98 to 107 of SEQ ID NO: 2.<br>
2.	The antibody as claimed in claim 1, wherein the antibody comprises at least one of the following residues in its heavy chain: V12, F29, A49, T93 and R94.<br>
3.	The antibody as claimed in claim 1, wherein the antibody comprises at least one of the following residues in its light chain: Ql, L4 and Y71.<br>
4.	The antibody or antigen-binding fragment thereof as claimed in claim 1, wherein the antibody comprises:<br><br>
(a)	a light chain variable domain sequence of SEQ ID NO: 1 and a heavy chain variable domain sequence of SEQ ID NO:2; or<br>
(b)	the same heavy and light chain polypeptide sequences as an antibody produced by hybridoma mAQC2 (ATCC accession number PTA3273).<br>
5.	The antibody or antigen-binding fragment thereof as claimed in claim 1,<br>
wherein the antibody or fragment is a humanized antibody.<br><br>
6.	The antibody or antigen-binding fragment thereof as claimed in claim 5,<br>
wherein the antibody or fragment comprises:<br>
(a)	at least one of the following residues in its light chain: Ql, L4, P46,<br>
W47 and Y71; or at least one of the following residues in its heavy chain: Dl,<br>
V12, S28, F29, A49, T93, R94 (Kabat numbering convention); or<br>
(b)	a light chain variable domain sequence defined by amino acid<br>
residues 1 to 106 of SEQ ID NO:3, and a heavy chain variable domain<br>
sequence defined by amino acid residues 1 to 118 of SEQ ID NO:4; or<br>
(c)	the same heavy and light chain polypeptide sequences as an antibody<br>
produced by cell line hAQC2 (ATCC accession number PTA3275); or<br>
(d)	the same heavy and light polypeptide sequences as an antibody<br>
produced by cell line hsAQC2 (ATCG accession number PTA3356); or<br>
(e)	the same heavy and light chain polypeptide sequences as an antibody<br>
produced by cell line haAQC2 (ATCC accession number PTA3274).<br>
7.	The antibody or antigen-binding fragment thereof as claimed claim 5, wherein the heavy chain is mutated at one or more of amino acid residues selected from the group consisting of residues 234, 235, 236, 237, 297, 318, 320 and 322 (EU numbering system), thereby causing an alteration in an effector function while retaining binding to VLA-1 as compared with an unmodified antibody.<br>
8.	The antibody or antigen-binding fragment thereof as claimed in claim 7, wherein the antibody or fragment comprises the mutations L234A and L235A (EU numbering system) in its heavy chain as compared with an unmodified antibody.<br><br>
9.	The antibody or antigen-binding fragment thereof as claimed in claim 5,<br>
wherein the antibody or fragment is mutated at an amino acid residue that is a<br>
glycosylation site, thereby eliminating the glycosylation site.<br>
10.	The antibody or antigen-binding fragment thereof as claimed in claim 9, wherein the antibody or fragment comprises the mutation N297Q in its heavy chain (EU numbering system).<br>
11.	A composition comprising an antibody or antigen-binding fragment thereof as claimed in any one of claims 1 to 10, and a pharmaceutically acceptable carrier.<br>
12.	An isolated nucleic acid sequence comprising DNA which is selected from<br>
the group consisting of:<br>
(a)	a coding sequence for SEQ ID NO: 1 and a coding sequence for SEQ ID NO:2;<br>
(b)	a coding sequence for the light chain of an antibody produced by hybridoma mAQC2 (ATCC accession number PTA3273) and a coding sequence for the heavy chain of an antibody produced by hybridoma mAQC2 (ATCC accession number PTA3273);<br>
(c)	a coding sequence for the light chain of an antibody produced by cell line hAQC2 (ATCC accession number PTA3275) and a coding sequence for the heavy chain of an antibody produced by cell line hAQC2 (ATCC accession number PTA3275);<br>
(d)	a coding sequence for the heavy chain of an antibody produced by cell line haAQC2 (ATCC accession number PTA3274);<br><br>
(e)	a coding sequence for the heavy chain of an antibody produced by cell line hsAQC2 (ATCC accession number PTA3356);<br>
(f)	a coding sequence for residues 1 to 106 of SEQ ID NO:3 and a coding sequence for residues 1 to 118 of SEQ ID NO:4; and<br>
(g)	coding sequences for an anti-VLA-1 antibody, or antigen-binding<br>
fragment thereof, whose light chain complementarity determining regions are<br>
defined by amino acid residues 24 to 33, 49 to 55, and 88 to 96 of SEQ ID<br>
NO:l, and whose heavy chain complementarity determining regions are defined<br>
by amino acid residues 31 to 35, 50 to 65, and 98 to 107 of SEQ ID NO: 2.<br>
13.	An in vitro method of determining the level of VLA-1 in a tissue, comprising<br>
contacting the tissue with the antibody as claimed in claim 1, and detecting the<br>
binding of the antibody to the tissue, thereby determining the level of VLA-1 in<br>
the tissue.<br>
14.	Use of the composition as claimed in claim 11 for the preparation of a<br>
composition for the determination of the level of VLA-1 in a tissue in vitro for<br>
use in the diagnosis of an immunological disorder.<br>
15.	An in vitro method for identifying an inhibitor of an I domain of an integrin<br>
comprising the steps of:<br>
(a) using structure coordinates of hAQC2 amino acids comprising light chain residues Asn30, Tyr48 , Trp90, Asn93 and Trp95, and heavy chain residues Arg31, His56, Tyr58, GlylOO and AsplOl, according to Fig. 19 or ± a root mean square deviation from the backbone atoms of said hAQC2 amino<br><br>
acids not more than 1.10 A, to generate a three-dimensional structure of a binding site;<br>
(b)	employing said three-dimensional structure to design or select a<br>
potential antagonist;<br>
(c)	synthesizing said potential antagonist; and<br>
(d)	contacting said potential antagonist with hAQC2 in vitro to determine the ability of said potential antagonist to interact with hAQC2, wherein the ability of said potential antagonist to interact with hAQC2 indicates that the potential antagonist is an inhibitor of the I domain.<br>
16.	The use as claimed in claim 14, wherein the immunological disorder is<br>
selected from the group consisting of a gastrointestinal disorder, an<br>
autoimmune disease, a fibrosis, a skin disorder, a vascular disease, allergic<br>
rhinitis, respiratory distress syndrome, asthma, bronchitis, tendonitis, bursitis,<br>
a migraine headache, periareritis nodosa, thyroiditis, aplastic anemia,<br>
Hodgkin's Disease, rheumatic fever, osteoarthritis, sarcoidosis, nephrotic<br>
syndrome, renal failure, Bechet's syndrome, polymyositis, gingivitis,<br>
hypersensitivity, graft or transplant rejection, graft versus host disease,<br>
conjunctivitis, swelling after injury, myocardial ischemia, and endotoxin shock<br>
syndrome.<br>
17.	The antibody, or antigen-binding fragment thereof, as claimed in any one<br>
of claims 1 to 10, wherein the antigen-binding fragment is selected from the<br>
group consisting of a Fab, a F(ab')2 and a single chain Fv.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLUFic3RyYWN0LSgwNC0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Abstract-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWFzc2lnbm1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-assignments.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLUNsYWltcy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Claims-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLUNsYWltcy0oMTgtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-Claims-(18-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Correspondence-Others-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLURyYXdpbmdzLSgwNC0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Drawings-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMTkucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-19.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLUZvcm0tMy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Form-3-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLUdQQS0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-GPA-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLU90aGVycy1Eb2N1bWVudHMtKDA0LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Others-Documents-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0xMDUucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-105.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLVBDVC0yMTAtKDA0LTA2LTIwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-PCT-210-(04-06-209).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0zMDEucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-301.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC00MDEucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1kZWxucC0yMDAzLXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">1798-delnp-2003-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLVBldGl0aW9uLTEzNy0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Petition-137-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc5OC1ERUxOUC0yMDAzLVBldGl0aW9uLTEzOC0oMDQtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1798-DELNP-2003-Petition-138-(04-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="235333-antibiotic-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="235335-a-process-for-the-preparation-of-the-polycyclitols.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>235334</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>01798/DELNP/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2009</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>31-Jul-2009</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Jun-2009</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>31-Oct-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOGEN, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>14 CAMBRIDGE CENTER, CAMBRIDGE, MASSACHUSETTS 02142,USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PAUL D. LYNE</td>
											<td>72 ALLSTON STREET #3, CAMBRIDGE, MA 02139,USA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ELLEN A. GARBER</td>
											<td>14 DONNELL STREET, CAMBRIDGE, MA 02138, USA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JODE W. SALDANHA</td>
											<td>21 FILLERBROOK AVENUE, ENFIELD, MIDDLESEX EN1 3BD, ENGLAND.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MICHAEL KARPUSAS</td>
											<td>PLATONOS 7, AGIOS BASILIOS, 26500 PATRA, GREECE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US02/11521</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2002-04-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/303,689</td>
									<td>2001-07-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/283,794</td>
									<td>2001-04-13</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/235334-antibodies-to-vla-i by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:44:25 GMT -->
</html>
