<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250151-isolated-nucleotide-sequence-useful-for-the-prevention-treatment-of-hiv-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:35:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250151:ISOLATED NUCLEOTIDE SEQUENCE USEFUL FOR THE PREVENTION/TREATMENT OF HIV INFECTION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ISOLATED NUCLEOTIDE SEQUENCE USEFUL FOR THE PREVENTION/TREATMENT OF HIV INFECTION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to isolated nucleotide sequences which have targets in HIV genome and a method of identification thereof. Using multiple software, targets to six human microRNAs [miRNAs] were discovered in the nef, vpr, env, and I vif genes. The miRNAs were identified as hsa-miR-29a, hsa-mi&#x27;R-29b, hsa-mi&#x27;R-29c, hsa-mir-149, hsa-mir-324-5p, hsa-mir-378. These miRNAs or its homologues can be used as therapeutics against HIV infection. The invention further relates to a novel strategy to target genes of HIV-1 by human microRNAs, or its homologues, to inactivate or block HIV activity. The computational approach towards identification of human miRNA targets in the HIV genome and the variation in the microRNA levels was further validated experimentally.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of Invention<br>
The present invention relates to human microRNA targets in HIV genome and a method of identification thereof. It specifically relates to a novel strategy to target genes of HIV-1 by human microRNAs, or its homologues, to inactivate or block HIV activity.<br>
Background of the Invention<br>
Human immunodeficiency virus - 1 (HIV - 1) has been identified as the etiological agent responsible for acquired immune deficiency syndrome (AIDS), a fatal condition that arises by the invasion of the virus on various cells of the human immune system, mainly the T-cells. HIV is a member of the lentivirus, a subfamily of retrovirus, which have a single stranded ribonucleic acid (RNA) as the genetic material. The virus has the ability to integrate its genetic information into the genome of the host cell thereby blocking normal genetic flow of information in the cell. The virus then takes over the cellular machinery and replicates itself by synthesizing its own proteins and packaging the genetic material. Another feature of HIV infection is that the virus does not keep replicating itself all the time. There are "window periods" after infection, during which replication is not observed. This is regarded as a silent phase where the effect of the infection during which immune deficiency is not detected. A critical aspect of HIV infection is that, it infects and invades cells of the immune system, resulting in breakdown of the body's immune system and rendering the patient susceptible to opportunistic infections and conditions like neoplasms which arise due to immune system inactivation. HIV-1 is cytopathic for T4 lymphocytes, cells of the immune system that express the cell surface differentiation antigen CD4. In addition to CD4+ T cells, the host range of HIV includes cells of the mononuclear phagocytic lineage, including blood monocytes, tissue macrophages, Langerhans cells of the skin, and dendritic reticulum cells within lymph nodes.<br>
Worldwide, researchers have been engaged in studies to develop effective therapeutic antiviral agents and vaccines against this deadly virus. Currently accepted strategies are mainly based on non-nucleotide analog inhibitors of reverse transcriptase, such as Nevirapine  (BI-RG-587),   TIBO  (R82913),   pyrinodes   (such  as  R-697,661   and   L-<br><br>
696,227), bis(heteroary) piperazines (BHAPs, such as U-87201E and U-90,152), atevirdine mesylate (ATV) and R-89431; HIV protease inhibitors, include substrate analogs and non-analogs, such as Ro 31-8959, A-77003 and A-80987; HIV Tat protein inhibitors, such as Ro 5-3335 and Ro 27-7429; blockers of viral entry into cells, such as soluble CD4 protein (sCD4), and chimeric sCD4 derivatives, such as CD4-lgG and CD4-PE40; blockers of HIV RNaseH activity, such as the AZT derivative azidothymidine monophosphate; drugs that alter the intracellular milieu to create conditions less favorable for viral replication, such as the free-radical scavengers and glutathione-level restoring drugs (N-acetylcysteine and similar drugs), and thalidomine (which seems to lower blood TNF-.alpha. levels). Efforts have also focused on manipulation of the immune system and viral replication with naturally-occurring cytokines and lymphokines, or other agonists or antagonists of these systems. One of the drugs most frequently used is azidothimidine (AZT). Antiviral therapies with pharmaceuticals that inhibit the replication of HIV have been found to decrease plasma virus to undetectable levels. Combination antiviral HIV therapy with Protease Inhibitors (PI) and Reverse Transcriptase (RT) inhibitors has provided Highly Active Antiviral Therapy (HAART). HAART results in the rapid cessation of viral replication and the decline of plasma virus to undetectable levels within 4-8 weeks. More recently, short interfering RNA (siRNA) has been tried against HIV infection. The property of siRNA that is made use here is that the binding of these small RNAs to mRNA would result in the cleavage of the mRNA which consequently inhibits protein synthesis.<br>
Even before siRNA-based therapy has reached the clinical trial stage for use as an anti-HIV mechanism, probability of failure exist owing to the fact that exact complementarity is a necessity for siRNA activity. HIV is observed to constantly mutate its genome and this would result in rendering the siRNA-mediated inhibition a failure. However, miRNAs may offer a more effective alternative since they require incomplete complementarity.<br>
Recently it was shown that human miRNAs can downregulate Hepatitis C Virus (HCV) and Primate Foamy Virus-1 (PFV-1) mRNA . The inventors have observed that human microRNAs Msa-mir-29a, hsa-mir-29b and hsa-mir-29c, which were predicted by all the<br><br>
programs, exclusively target the nef gene of HIV-1. Viral genes like tat and the gene coding for reverse transcriptase, as well as genes of host origin required for viral transcription have previously been targeted using siRNAs. In the present study, the inventors have shown that human microRNAs may target HIV-1 genes. These miRNAs have highly conserved targets in HIV-1 and related clade sequences. They are expressed in T-cells, the natural site of infection by HIV-1 infection and their expression level may vary between individuals. In summary, our study implies that human miRNAs have the potential to affect expression of HIV-1 genes and may in future be used to develop therapeutic approaches to inhibit HIV-1.<br>
Drawbacks of existing therapies<br>
Although AZT has proved effective in many cases, lowering mortality rates, the virus develops resistance to AZT, and the drug has significant and adverse side effects.<br>
In most instances, HAART alone does not lead to complete immune recovery. Another drawback of HAART is that HIV develop resistance to it. Moreover, it now appears that many individuals may not be able to take HAART indefinitely, due to serious long-term side effects. As many as 40%-60% of patients who have received HAART for greater than one year have developed symptoms of Cushing's Syndrome, with hyperglycemia, hyperlipidemia, centipetal fat distribution, and peripheral muscle wasting.<br>
Even before siRNA-based therapy has reached the clinical trial stage for use as an anti-HIV mechanism, probability of failure exist owing to the fact that exact complementarity is a necessity for siRNA activity. HIV is observed to constantly mutate its genome and this would result in rendering the siRNA-mediated protein inhibition a failure.<br>
Accordingly, there is a need for new safe and efficient therapeutic and preventive methods for HIV-infections. The invention addresses this need in the field.<br><br>
Objects of the invention<br>
The main object of the invention is thus to provide a novel strategy to target genes of HIV-1 by human microRNAs, or its homologues, to inactivate or block HIV activity and obviate the drawbacks mentioned above.<br>
Yet another objective is to use synthetic miRNA, based on modified nucleosides, as therapeutic to prevent or inhibit the progression of disease.<br>
Still another object is to provide miRNA expression variation as a genetic basis for long term progression of HIV infection.<br>
Another object is to provide miRNA mediated inhibition of protein synthesis in HIV-1.<br>
Still another object of the invention is to use variation of expression of human miRNAs as a prognostic genetic bio-marker in HIV-1 infection.<br>
Novelty of invention<br>
•	a novel strategy to target HIV genes by human microRNA<br>
•	use of microRNA to inactivate or block HIV activity<br>
•	provides a method to repress synthesis of proteins of HIV-1<br>
•	use of human microRNA expression variation as a prognostic marker in HIV-1 infection<br>
•	method to predict targets for HIV-1 genes with microRNAs<br>
•	use and synthesis of miRNA, based on modified nucleosides, to block HIV-1 proteins<br>
Description of the figures<br>
Figure 1 shows positions of the five microRNA targets on the HIV-1 genome.<br>
Figure 2: Scheme 1 : Protocol for Luciferase assay<br>
Figure 3: Effect of hsa-miR-29a on nefgene using Anti-miR 29a Figure 4: Effect of hsa-miR-29b on nef gene using Anti-miR 29b Figure 5: Effect of hsa-miR-29c on nef gene using Anti-miR 29c Figure 6: Expression of miR 29a,b, c<br>
Summary of the invention<br>
The invention relates to the identification of targets to human encoded microRNA on the HIV genome and a method of targeting HIV genes with human microRNAs. The invention provides drug targets in HIV genome based on microRNAs for HIV activity inhibition.<br>
MicroRNAs are short RNA molecules which have the ability to repress protein synthesis by binding to messenger RNAs. In the present invention the applicants have screened HIV-1 reference genome computationally using human microRNAs for identifying targets for HIV activity inhibition. The invention also provides microRNA expression variation as a genetic basis for long term progression of HIV-1 infection.<br>
Accordingly, the present invention relates to the targets for human microRNAs in HIV genome and a method of identification thereof.<br>
The invention provides targets of six human microRNAs in HIV-1 reference genome comprising nucleotide stretches of SEQ ID Nos. 1 to 6.e invention specifically provides microRNA targets for the HIV genome wherein hsa-miR-29a, hsa-miR-29b and hsa-miR-29c target the nucleotide stretches of SEQ ID NOS 1,2 and 3 in the nef gene.<br>
The invention further provides microRNA targets for HIV genome wherein hsa-mir-149, hsa-mir-324-5p and hsa-mir-378 target the nucleotide stretches of SEQ ID NOS 4, 5 and 6 in the vpr, vif and env genes respectively.<br>
The invention also discloses a method of targeting HIV genes with human miRNAs which comprises:<br>
i.     Downloading and shuffling computationally the whole genome HIV sequence from publicly available database at URL:<br>
http://www.ncbi.nlm.nih.qov/entrez/query.fcgi?db=Genome&amp;cmd=search&amp;term-NC 001802.1<br>
ii.     Predicting computationally,  the targets for microRNAs  in the shuffled  HIV genome sequences and computing the cut-off.<br>
iii.     Deriving consensus predictions for microRNA-target pairs (which have scores &gt; the cutoff of step 3)  for six human microRNA.<br>
iv.     Mapping computationally the human microRNA targets in HIV-1 genome.<br>
Another aspect of the invention is to provide targets for miRNA sequences (SEQ ID NOS: 1 to 6) in 12 other related HIV-1 clades which represent the various subtypes of HIV-1.<br>
Still another aspect of the invention is to provide use of miRNA expression variation as a genetic basis for long term progression of HIV infection.<br>
In an embodiment to the invention, the software programs used for computational predictions consist of software available in public domain, viz., RNAhybrid, miRanda, DIANA-micro-T and microlnspector.<br>
In another embodiment to the invention, publically available software- ShuffleSeq- was used for shuffling downloaded HIV genome to minimize error due to sequence compositional bias.<br>
The parameters used for prediction of targets comprised of sequence complimentarity, minimum free energy of the duplex and continuous seed complementarity towards the 5' end of the microRNA.<br>
Still another aspect of the invention is the use of miRNA or its homologues as therapeutics to prevent or inhibit the progression of disease by microRNA mediated inhibition of protein synthesis in HIV-1.<br>
Yet another aspect of the invention is the use of synthetic oligomers that can bind as microRNA which can inhibit or repress protein synthesis in HIV-1. The ribonucleosides of the oligomer can be modified using strategies like Locked Nucleic Acid (LNA) or 2'-O-methyl RNA (OMe) resulting in better stability and binding to the target mRNA strand, thus enabling the repression of the HIV-1 proteins.<br>
Still another aspect of the invention is the association of miRNA expression levels with disease progression and use of variation of expression of human miRNAs as a prognostic genetic bio-marker in HIV-1 infection.<br>
Detailed description of the present invention<br>
Data Retrieval<br>
The human microRNA mature sequences were downloaded from The miRNA Registry. For querying probable targets in the HIV - 1 genome, the RefSeq validated HIV-1 reference sequence, obtained from http://ncbi.nlm.nih.gov was used along with 12 representative related strain sequences, as identified by the International Committee on Taxonomy of Viruses (ICTV). The clade sequences were obtained from ICTV database (http://www.ncbi.nlm.nih.gov/ICTVdb/lctv/fs_retro.htm ).<br>
Prediction of miRNA targets in HIV-1<br>
The applicants used four well-established microRNA target prediction softwares -miRanda , RNAhybrid, Micro Inspector and DIANA-MicroT to predict targets for the 211 human miRNAs obtained from The microRNA Registry, in the HIV-1 reference sequence and the 12 representative clade sequences. HIV-1 targets to human microRNAs were initially predicted using miRanda with default parameters (Gap Open Penalty: -8.0; Gap Extend: -2.0;  Score Threshold:  50.0;  Energy Threshold: -20.0<br>
kcal/mol; Scaling Parameter: 2.0). In order to increase the stringency, a cut-off score was derived above which the miRNA-target pairs were selected. A cut-off score of 120 was derived by running the same program on a shuffled sequence of HIV-1 reference strain with the same set of miRNAs. HIV-1 genome sequence was shuffled using the EMBOSS2 program shuffleseq. This enabled filtering of probable false positive hits and selection of the most probable and high-scoring values. These short-listed HIV-1 targets to human microRNAs were also found to be highly probable targets on the other prediction software. Prior to running the RNAhybrid program, the RNAcalibrate module was used to derive the xi and theta values for calculation of Extreme Value Distribution. The xi - theta values thus obtained were included as one of the parameters while using RNAhybrid for target prediction. This minimizes the base composition bias. Also, the helix parameters were set to include maximum continuous complementarity towards the 5' end of the miRNA. It was observed that out of the several probable targets predicted by RNAhybrid, the six filtered pairs from miRanda had the lowest minimum free energy. Similar observations were made when the other two software were employed with default parameters, viz., minimum free energy of -20. Similarly the targets were predicted on the representative clade sequences also using miRanda. The target regions were mapped to the genomes of the 12 HIV clades. The target sites for the respective miRNAs were aligned using TCoffee server.<br>
Validation of miRNA targets in HIV genome<br>
The target of miRNA 29a, 29b and 29c found in the nef gene of HIV genome was further validated by experimental means. The validation was carried out in a cell culture model employing HeLa cells. Primer extension based methods described below were used to ensure that HeLa cells express the miRNAs being tested. A vector with the firefly luciferase gene under the control of a constitutive promoter was used to monitor the activity of the miRNA. Cultured HeLa cells were transfected with various constructs bearing reporter gene which carried testable target regions in their 3'Untranslated regions. Subsequently, the reporter gene activity was monitored using enzymatic assays. The expression level of the reporter would be expected to get downregulated if<br>
the cellular miRNA binds to the 3'UTR and results in translational block of the target HIV-1 gene (scheme 1). Furthermore, specificity of the interaction was validated by introducing an anti-miRNA, complimentary in sequence to the miRNA being tested. The DNA oligonucleotide which acts as an anti-miRNA molecule interferes in the action of the miRNA and restores the reporter gene activity.<br>
Targets for 29a, 29b and 29c within the nef gene showed dependence on the miRNA in the Hela cell since expression levels from the clone carrying the target region were downregulated compared to the expression from the vector without the target regions. Anti-miRNA against 29a and 29b could partially restore the reporter activity (Figure 2 and Figure 3). However an unrelated anti-miRNA i.e. anti-mir 149 had no such effect proving that this is not a non-specific effect of DNA oligos.<br>
Expression profile analysis of miRNA<br>
Microarray based expression data was retrieved from ArrayExpress database. The raw intensity data for each experiment was log transformed and then used for the calculation of Z scores. Z scores were calculated by subtracting the overall average gene intensity (within a single experiment) from the raw intensity data for each gene, and dividing the result by the standard deviation of all of the measured intensities, according to the formula:<br>
Z score=(lntensity G-mean intensity G1...Gn)/SDG1...Gn.<br>
We adopted a consensus target prediction' approach using multiple miRNA target prediction softwares, vis, miRanda, RNAhybrid, DIANA-MicroT and Microlnspector. Firstly, ther inventors identified highly probable targets to six human miRNAs in the HIV-1 genome using the miRNA target prediction software miRanda which takes into account continuous base complementarity at the 5' end of the microRNA and minimum free energy of the bound complex. These were also subsequently predicted by the other three softwares - RNAhybrid, Microlnspector and DIANAMicro-T.<br>
miRNA expression levels were monitored using a primer extension based protocol (see below) standardized for the purpose. As shown in Figure 4, the miRNA expression levels could be detected by monitoring a radioactively labeled anti-miRNA probe in a sequence dependent manner (Figure 4). Expression levels of miRNAs 29a and 29b which showed anti-nef activity were expressed at different levels in different control individuals.<br>
Detection of miRNA using Primer extension<br>
The mixture of total RNA and double autoclaved water in the ratio 1:10 is heated in a boiling water bath for 5-10 minutes followed by chilling in ice for the same duration. Subsequently, it is kept at room temperature for 10-15 minutes followed by addition of diluted dATP, dGTP and dTTP and 10X RT buffer. 1µl of α-P-32-dCTP was added after which RT enzyme is added. The reaction mix was then incubated at 37°C for 30min. The reaction was stopped by adding 2µl 1N NaOH and 0.5µl 0.5M EDTA and the sample was incubated at 65°C for 30min. After 30 min 7µl 1M Tris-HCI (pH 7.5) was added to the mixture. The samples were prepared as explained below and run on 18% polyacrylamide gel containing urea (8M). Sample Preparation: 16 M Urea was added to the samples to make the final cone, of urea to 8M. The samples were then heated at 65°C for 5-10min, mixed with loading dye and loaded in 18% urea-containing PAGE. After running, the gel was kept in fixing solution (10% Methanol, 10% Glacial Acetic Acid) for 1hr on a rocker. After fixing, the gel was washed with water twice, wrapped in Saran Wrap and was put for exposure.The image was scanned after overnight exposure.<br>
Luciferase assay: Preparation of Lysate:<br>
Lysate is prepared by suspending HeLa cells in 5X lysis buffer (CCLR, RLB or PLB) after removal of the growth medium by rinsing with PBS buffer followed by freeze thaw. The suspension is centrifuged at 12,000Xg for 15 seconds at room temperature followed by centrifugation at 4 degree centigrade for 2 minutes. The supernatant (cell lysate) is stored at minus 70 degree centigrade.<br>
Luciferase assay using luminometer:<br>
Dispense 100µl of the Luciferase Assay Reagent into luminometer tubes, one tube per sample. Program the luminometer to perform a 2-second measurement delay followed by a 10-second measurement read for luciferase activity. The read time may be shortened if sufficient light is produced. Add 20ul of cell lysate to a luminometer tube containing the Luciferase Assay Reagent. Mix by pipetting 2.3 times or vortex briefly. Place the tube in the luminometer and initiate reading.<br>
Mapping of miRNA targets:<br>
The miRNAs hsa-mir-29a, hsa-mir-29b and hsa-mir-29c target the nef gene whereas hsa-mir-149, hsa-mir-324-5p and hsa-mir-378 target vpr, vif and env respectively, nef, vpr and vif are accessory genes of HIV-1 which are required for HIV-1 infection. The Nef protein is involved in both replication and pathogenesis of HIV-1. It has been established that Nef enhances Tat mediated gene expression from the LTR by activating signalling pathways as a consequence of which there is an increase in the activation of general transcription factors as well as the translocation of repression factors from the nucleus. The Vpr protein is involved in a range of functions which include induction of cell cycle arrest in the G(2) phase, transactivation of the viral promoter, nuclear import of preintegration complexes, and induction of apoptosis in the infected cell. The Vif protein is understood to play an important role during viral assembly in producer cells to ensure infectivity in infected cells. The env gene product is Envelope surface glycoprotein which is important in entry of the HIV to the cell. Entry of HIV into target cells requires sequential interactions of the viral envelope protein (Env) with CD4 and appropriate chemokine receptor on the target cell.<br>
Comparison of target sequences in related HIV-1 strains:<br>
The variability of viral genomes can pose a problem in using RNA interference. Therefore we compared the sequence conservation at the target site amongst different HIV -1 clades. It was observed that the target regions were significantly conserved. The genes to which the target regions belong varied among some of the strains, mostly in<br>
the Clade O. However, the clade O is known to be a highly polymorphic strain when compared to the other sub-groups M and N.<br>
Expression variation studies:<br>
Recently microarray experiments have been used to generate data on tissue and cell type specific expression profiles of microRNAs. MicroRNAs can be effective anti-viral regulatory strategies of host cells only if the microRNA is expressed at the site of infection by the virus. Probes for hsa-mir-29c, hsa-mir-378 and hsa-mir-324-5p were not available on the arrays. Expression data reported from previous studies show that the miRNAs hsa-mir-29a, hsa-mir-29b and hsa-mir-149 are expressed in T cells.<br>
There have been reports of defective accessory genes being a probable cause of delayed response to HIV-1 infection, nef gene function loss has been, arguably, reported to play an important role in disease progression. Individuals who do not develop Acquired Immuno Deficiency Syndrome (AIDS) even several years after infection have been shown to harbour defective or non-functional genes in the viral genome. Polymorphisms in miRNA sequences that target the HIV-1 or variation in the expression levels of these miRNAs between individuals, hence, may also contribute to disease progression. However, neither published literature nor database entries report polymorphisms in the mature miRNAs that target HIV-1 as well as in their respective pre-miRNA sequences. The inventors used recently reported microarray based miRNA expression profiles to identify the expression pattern of the five miRNAs which target the HIV-1 genes. The raw expression values were retrieved and re-normalised. The data from all experiments was scaled to enable comparison.<br>
The scaled and normalised expression levels of the microRNAs in 73 samples were calculated. In addition hsa-mir-29a and hsa-mir-29b miRNAs expressed in T-cells can repress nef function and thus influence disease progression, nef is also known to be regulated by a HIV-1 miRNA. HIV-1 strains from long-term non-progressors have been shown to harbor defective nef genes. The limited amount of expression data currently available implies that these miRNAs show variable levels of expression in different<br>
samples. Study of the expression levels of these miRNAs, in normal individuals and infected individuals who do not develop disease after prolonged periods of infection can, in future, reveal the role of human miRNA expression in accounting for differences in disease progression.<br>
Another aspect of the invention is associating variations in expression levels of miRNA with progression of disease. This is carried out by microarray experiments and other high throughput primer extension based methods wherein miRNA expression levels are detected in patients and control individuals. The expression levels of miRNAs may provide prognostic information about HIV progression.<br>
Use of chemically modified miRNAs to target HIV:<br>
Another aspect of the invention is targeting HIV genes using chemically modified synthetic oligomers that act as miRNAs.<br>
The following examples are given by way of illustration of the present invention and therefore should not be construed to limit the scope of the present invention<br>
Example 1: Data Retrieval:<br>
The human microRNA mature sequences were downloaded from the database of<br>
miRNA maintained by the Sanger Center named - The miRNA Registry<br>
(microrna.sanger.ac.uk/sequences/). The entire list of miRNA that were downloaded<br>
are given as Table 1. For querying probable targets in the HIV - 1 genome, the<br>
inventors used the RefSeq validated HIV-1 reference sequence, obtained from<br>
http://ncbi.nlm.nih.gov and 12 representative related strain sequences, as identified by<br>
the International Committee on Taxonomy of Viruses (ICTV). The clade sequences<br>
were	obtained	from	ICTV	database<br>
(http://www.ncbi.nlm.nih.gov/ICTVdb/lctv/fs_retro.htm). The accession numbers of the clades are given in Table 2.<br>
Example 2: Prediction of miRNA targets in HIV-1:<br>
Four well-established microRNA target prediction softwares - miRanda, RNAhybrid, Microlnspector and DIANA-MicroT were used to predict targets for the 211 human miRNAs in the HIV-1 reference sequence and the 12 representative clade sequences. HIV-1 targets to human microRNAs were initially predicted using miRanda with default parameters (Gap Open Penalty: -8.0; Gap Extend: -2.0; Score Threshold: 50.0; Energy Threshold: -20.0 kcal/mol; Scaling Parameter: 2.0). In order to increase the stringency, a cut-off score was derived above which the miRNA-target pairs were selected. The cutoff score of 120 was derived by running the same program on a shuffled sequence of HIV-1 reference strain with the same set of miRNAs. HIV-1 genome sequence was shuffled using the EMBOSS2 program shuffleseq. This enabled filtering of probable false positive hits and selection of the most probable and high-scoring values. These short-listed HIV-1 targets to human microRNAs were also found to be highly probable targets on the other prediction software. The top scoring miRNA-target pairs are tabulated in Table 3. Prior to running the RNAhybrid program, the RNAcalibrate module was used to derive the xi and theta values for calculation of Extreme Value Distribution. The xi - theta values thus obtained were included as one of the parameters while using RNAhybrid for target prediction. This minimizes the base composition bias. Also, the helix parameters were set to include maximum continuous complementarity towards the 5' end of the miRNA. It was observed that out of the several probable targets predicted by RNAhybrid, the six filtered pairs from miRanda had the lowest minimum free energy. Similar observations were made when the other two software were employed with default parameters, viz., minimum free energy of -20. Similarly the targets were predicted on the representative clade sequences also using miRanda. The target regions were mapped to the genomes of the 12 HIV clades. The target sites for the respective miRNAs were aligned using TCoffee server.<br>
Validation of miRNA targets in HIV genome:<br>
The target of miRNA 29a, 29b and 29c found in the nef gene of HIV genome was further validated by experimental means. The validation was carried out in a cell culture model employing HeLa cells. Primer extension based methods described below were used to ensure that HeLa cells express the miRNAs being tested. A vector with the firefly luciferase gene under the control of a constitutive promoter was used to monitor the activity of the miRNA. Cultured HeLa cells were transfected with various constructs bearing reporter gene which carried testable target regions in their 3'Untranslated regions. Subsequently, the reporter gene activity was monitored using enzymatic assays. The expression level of the reporter would be expected to get downregulated if the cellular miRNA binds to the 3'UTR and results in translational block of the target HIV-1 gene (scheme 1). Furthermore, specificity of the interaction was validated by introducing an anti-miRNA, complimentary in sequence to the miRNA being tested. The DNA oligonucleotide which acts as an anti-miRNA molecule interferes in the action of the miRNA and restores the reporter gene activity.<br>
Targets for 29a, and 29b within the nef gene showed dependence on the miRNA in the Hela cell since expression levels from the clone carrying the target region were downregulated compared to the expression from the vector without the target regions. Anti-miRNA against 29a and 29b could partially restore the reporter activity (Figure 2 and Figure 3). However an unrelated anti-miRNA i.e. anti-mir 149 had no such effect proving that this is not a non-specific effect of DNA oligos.<br>
Example 3: Mapping of miRNA targets:<br>
The miRNAs hsa-mir-29a, hsa-mir-29b and hsa-mir-29c target the nef gene whereas hsa-mir-149, hsa-mir-324-5p and hsa-mir-378 target vpr, vif and env respectively, nef, vpr and vif are accessory genes of HIV-1. The Nef protein is involved in both replication and pathogenesis of HIV-1. It has been established that Nef enhances Tat mediated gene expression from the LTR by activating signalling pathways as a consequence of which there is an increase in the activation of general transcription factors as well as the translocation of repression factors from the nucleus. The Vpr protein is involved in a<br>
range of functions which include induction of cell cycle arrest in the G(2) phase, transactivation of the viral promoter, nuclear import of preintegration complexes, and induction of apoptosis in the infected cell. The Vif protein is understood to play an important role during viral assembly in producer cells to ensure infectivity in infected cells. The env gene product is Envelope surface glycoprotein which is important in entry of the HIV to the cell.<br>
Example 4: Comparison of target sequences in related HIV-1 strains:<br>
The variability of viral genomes can pose a problem in using RNA interference. Therefore we compared the sequence conservation at the target site amongst different HIV -1 clades. It was observed that the target regions were significantly conserved. The genes to which the target regions belong varied among some of the strains, mostly in the Clade 0. However, the clade O is known to be a highly polymorphic strain when compared to the other sub-groups M and N. The genes which encompass the target regions are tabulated in Table 4.<br>
Example 5: Expression profile analysis of miRNA:<br>
Microarray based expression data was retrieved from ArrayExpress database. The raw intensity data for each experiment was log transformed and then used for the calculation of Z scores. Z scores were calculated by subtracting the overall average gene intensity (within a single experiment) from the raw intensity data for each gene, and dividing the result by the standard deviation of all of the measured intensities, according to the formula:<br>
Z score=(lntensity G-mean intensity G1 ...Gn)/SDG1 ...Gn.<br>
The graph representing the varying levels of miRNA expression is shown in Figure 2.<br>
In addition hsa-mir-29a and hsa-mir-29b miRNAs expressed in T-cells can repress nef function and thus influence disease progression, nef is also known to be regulated by a HIV-1  miRNA. HIV-1 strains from long-term non-progressors have been shown to<br>
harbor defective nef genes. The limited amount of expression data currently available implies that these miRNAs show variable levels of expression in different samples. Study of.the expression levels of these miRNAs, in normal individuals and infected individuals who do not develop disease after prolonged periods of infection can, in future, reveal the role of human miRNA expression in accounting for differences in disease progression.<br>
Another aspect of the invention is associating variations in expression levels of miRNA with progression of disease. This is carried out by microarray experiments wherein miRNA expression levels are detected in patients and control individuals. Detection of miRNA using Primer extension protocol<br>
1.	In an Eppendarf tube, 1µg of total RNA and 1 µl of primer (10pmole/ul) were taken<br>
and the final volume was made to 10ul using double autoclaved water.<br>
Note:   If the RNA is at a high conc, and had been stored at -20°C for long, then warm the RNA before use.<br>
2.	The mixture was heated in a boiling water bath for 5-10min and then chilled in ice for 5-10min. The tube was then kept at room temp, for 10-15min.<br>
3.	dATP, dGTP and dTTP were diluted 5 times from their stock of 2mM each. 2µl of each of them was then added to the reaction mixture. 10X RT Buffer was also added. Then 1µl of α-P-32-dCTP was added to the reaction mixture and finally RT enzyme was added. The reaction mix was then incubated at 37°C for 30min.<br>
4.	The reaction was stopped by adding 2µl 1N NaOH and 0.5µl 0.5M EDTA and the sample was incubated at 65°C for 30min. After 30 min 7µl 1M Tris-HCI (pH 7.5) was added to the mixture.<br>
5.	The samples were prepared as explained below and run on 18% polyacrylamide gel containing urea (8M).<br>
6.	Sample Preparation: 16 M Urea was added to the samples to make the final cone, of urea to 8M. The samples were then heated at 65°C for 5-10min, mixed with loading dye and loaded in 18% urea-containing PAGE.<br>
Note: Wash the wells properly before loading the samples.<br>
7.	After running, the gel was kept in fixing solution(10% Methanol, 10% Glacial Acetic Acid) for 1 hr on a rocker.<br>
8.	After fixing, the gel was washed with water twice, wrapped in Saran Wrap and was put for exposure.<br>
9.	The image was scanned after overnight exposure.<br>
Luciferase assay protocol:<br>
Preparation of Lysate:<br>
1.	Add 4 volumes of water to 1 volume of 5X lysis buffer. Equilibrate 1X lysis buffer to room temperature before use.<br>
2.	Carefully remove the growth medium from cells to be assayed. Rinse cells with PBS, being careful to not dislodge attached cells. Remove as much of the PBS rinse as possible.<br>
3.	Add enough 1X lysis buffer (CCLR, RLB or PLB) to cover the cells (e.g., 400µl/60mm culture dish, 900µl/100mm culture dish or 20µl per well of a 96-well plate). If using RLB, perform a single freeze-thaw to ensure complete lysis.<br>
4.	Rock culture dishes several times to ensure complete coverage of the cells with lysis buffer. Scrape attached cells from the dish. Transfer cells and all liquid to a microcentrifuge tube. Place the tube on ice.<br>
5.	Vortex the microcentrifuge tube 10.15 seconds, then centrifuge at 12,000 x g for 15 seconds (at room temperature) or up to 2 minutes (at 4°C). Transfer the supernatant to a new tube.<br>
6.	Store the supernatant/cell lysate at 70°C<br>
Luciferase assay using luminometer:<br>
1.	Dispense 100ul of the Luciferase Assay Reagent into luminometer tubes, one tube per sample.<br>
2.	Program the luminometer to perform a 2-second measurement delay followed by a 10-second measurement read for luciferase activity. The read time may be shortened if sufficient light is produced.<br>
Note: When using shorter assay times, validate the luminometer over that time period<br>
to ensure that readings are taken at a flat portion of the signal curve.<br>
.3. Add 20ul of cell lysate to a luminometer tube containing the Luciferase Assay<br>
Reagent. Mix by pipetting 2.3 times or vortex briefly.<br>
4. Place the tube in the luminometer and initiate reading.<br>
Example 6: Use of chemically modified miRNAs to target HIV:<br>
Another aspect of the invention is targeting HIV genes using chemically modified synthetic oligomers that act as miRNAs. The nucleosides of the oligomer can be modified using strategies like Locked Nucleic Acid (LNA) or 2'-0-methyl RNA (OMe) resulting in better stability and binding to the target mRNA strand, thus enabling the repression of the HIV-1 proteins.<br>
Table 1: The 211 human microRNAs used in the analyses. Nomenclature as per the microRNA Registry<br>
Table 2: Details of the H1V-1 clades.<br>
Table 3: Top scoring miRNA-target pairs.<br>
Table 4: Genes encompassing target regions in the HIV-1 clades:<br>
(Table Removed)<br>
Sequence Listing<br>
(Sequance Removed)<br>
Advantages:<br>
1.	The main advantage of the invention is to provide a novel strategy to target genes of HIV-1 by human microRNAs, or its homologues, to inactivate or block HIV activity and obviate the drawbacks mentioned above.<br>
2.	Another advantage is to use synthetic miRNA, based on modified nucleosides, as therapeutic to prevent or inhibit the progression of disease.<br>
3.	Still another advantage is to provide miRNA expression variation as a genetic basis for long term progression of HIV infection.<br>
4.	Another advantage is to provide miRNA mediated inhibition of protein synthesis in HIV-1.<br>
5.	Yet another advantage of the invention is to use variation of expression of human miRNAs as a prognostic genetic bio-marker in HIV-1 infection.<br><br><br><br><br><br><br><br>
We Claim:<br>
1.	An isolated nucleotide represented by SEQ ID No. 7 useful for the prevention<br>
and/or treatment of HIV infection, wherein the said nucleotide having the<br>
sequence of:<br>
(Sequance Removed)<br>
2.	A nucleotide as claimed in claim 1, wherein it targets the nucleotide sequence of SEQ ID No. 3 in the nef gene of HIV genome.<br>
3.	A nucleotide as claimed in claim 1, wherein the variation in expression thereof is useful as a prognostic biomarker for indicating HIV infection.<br>
4.	An isolated nucleotide useful for the prevention and/or treatment of HIV infection substantially as herein described with reference to the foregoing examples.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1BYnN0cmFjdC0oMDktMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Abstract-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2729-del-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1DbGFpbXMtKDA5LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Claims-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1DbGFpbXMtKDIyLTAyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Claims-(22-02-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDA5LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Correspondence-Others-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">2729-del-2005-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-del-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1kZXNjcmlwdGlvbiAocHJvdmlzaW9uYWwpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-del-2005-description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1Gb3JtLTMtKDA5LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Form-3-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1kZWwtMjAwNS1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">2729-del-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjcyOS1ERUwtMjAwNS1QZXRpdGlvbiAxMzctKDA5LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2729-DEL-2005-Petition 137-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1BYnN0cmFjdC0oMTItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Abstract-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1DbGFpbXMtKDEyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Claims-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDEyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Correspondence Others-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLSgxMi0wOS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Description (Complete)-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1EcmF3aW5ncy0oMTItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Drawings-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1Gb3JtLTItKDEyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Form-2-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1Gb3JtLTMtKDEyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Form-3-(12-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjkyOS1ERUwtMjAwNS1QZXRpdGlvbi0xMzctKDEyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2929-DEL-2005-Petition-137-(12-09-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250150-security-document-with-upconverting-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250152-coloring-composition-and-coloring-method.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250151</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2729/DEL/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>50/2011</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>16-Dec-2011</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Oct-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC &amp; INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ANUSANDHAN BHAWAN, RAFI MARG, NEW DELHI-110 001, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SAMIR KUMAR BRAHMACHARI</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007, INDIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MANOJ HARIHARAN</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007, INDIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>VINOD SCARIA</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007, INDIA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>BEENA PILLAI</td>
											<td>INSTITUTE OF GENOMICS AND INTEGRATIVE BIOLOGY, MALL ROAD, DELHI-110007, INDIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>H12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250151-isolated-nucleotide-sequence-useful-for-the-prevention-treatment-of-hiv-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:35:46 GMT -->
</html>
