<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250295-an-isolated-human-monoclonal-antibody-directed-against-egfrviii by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:31:03 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250295:AN ISOLATED HUMAN MONOCLONAL ANTIBODY DIRECTED AGAINST EGFRvIII</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED HUMAN MONOCLONAL ANTIBODY DIRECTED AGAINST EGFRvIII</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to novel antibodies, particularly antibodies directed against deletion mutants of epidermal growth factor receptor and particularly to the type III deletion mutant, EGFRvIII. The invention also relates to human monoclonal antibodies directed against deletion mutants of epidermal growth factor receptor and particularly to EGFRvIII. Diagnostic and therapeutic formulations of such antibodies, and immunoconjugates thereof, are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FIELD OF THE INVENTION<br>
[0002] The present embodiments relate to novel antibodies, particularly antibodies directed against deletion mutants of epidermal growth factor receptor and particularly to the type III deletion mutant, EGFRvIIl. The embodiments also relate to human monoclonal antibodies directed against deletion mutants of epidermal growth factor receptor and particularly to EGFRvIIl. The embodiments also relate to variants of such antibodies. Diagnostic and therapeutic formulations of such antibodies, and immunoconjugates thereof, are also provided.<br>
BACKGROUND OF THE INVENTION<br>
[0003| Tumor specific molecules to aid in better diagnosis and treatment of human and animal cancer have been sought since llic last ccntiuy. Hard evidence of tumor-specific substances, based on molecular structural data, has been difficult to provide in most types of human cancer except those based on virally-induccd cancer and involving molecular structures specified by the virus genome. There have been extremely few examples of tumor-specific molecules based on novel molecular structures. In the case of malignant human gliomas and other tumors potentially associated with amplification or changes in the epidermal growth factor receptor molecule, such as carcinoma of the breast and other human carcinomas, there have been no unequivocal demonstrations of structurally altered molecules with unique sequences.<br>
[0004] The epidermal growth factor receptor (EGFR) is the 170 kilodalton membrane glycoprotein product of the proto-oncogene c-erb B. The sequence of the EGFR gene is known (Ullrich et al. (1984). Human Epidermal Growth Factor Receptor cDNA Sequence and Aberrant Expression of the Amplified Gene in A431 Epidermoid Carcinoma Cells. Nature 309:418-425). The EGFR gene is the cellular homolog of the erb B oncogene originally identified in avian erythroblastosis viruses (Downward et al. (1984). Close Similarity of Epidermal Growth Factor Receptor and v-erb B Oncogene Protein Sequence. Nature 307:521-527, Ullrich, et al. (1984)). Activation of this oncogene by gene amplification has been observed in a variety of human tumors (Haley et al. (1987A). The Epidermal Growth Factor Receptor Gene in: Oncogenes, Genes, and Growth Factors Edited by: Guroff, G. 12th Edition. Chapter 2. pp. 40-76. Wiley, N.Y.), and in particular, those of glial origin (Libermann et al. (1985). Amplification, Enhanced Expression and Possible Rearrangement of EGF Receptor Gene in Primary Human Brain Tumours of Glial Origin.<br><br>
Nature 313:144-147; Wong et al. (1987). Increased Expression of the Epidermal Growth Factor Receptor Gene in Malignant Gliomas is Invariably Associated with Gene Amplification, Proc. Natl. Acad. Sci. USA 84:6899-6903; Yamazaki et al. (1988). Amplification of the Structurally and Functionally Altered Epidermal Growth Factor Receptor Gene (c-erbB) in Human Brain Tumors. Molecular and Cellular Biology 8:1816-1820; Maiden et al., (1988). Selective Amplification of the Cytoplasmic Domain of the Epidermal Growth Factor Receptor Gene in Glioblastoma Multiforme. Cancer Research 4:2711-2714).<br>
[0005] EGF-r has been demonstrated to be overexpressed on many types of human solid tumors. Mendelsohn Cancer Cells 7:359 (1989), Mendelsohn Cancer Biology 1:339-344 (1990), Modjtahedi and Dean lnt'1 J. Oncology 4:277-296 (1994). For example, EGFR overexpression has been observed in certain lung, breast, colon, gastric, brain, bladder, head and neck, ovarian, kidney and prostate carcinomas. Modjtahedi and Dean Int'l J. Oncology 4:277-296 (1994). Both epidermal growth factor (EOF) and transforming growth factor-alpha (TGF-alpha.) have been demonstrated to bind to EGF-r and to lead to cellular proliferation and tumor growth.<br>
|0006] One major difference between v-erb B oncogenes and the normal EGFR gene is that the viral oncogenes are amino-truncated versions of the normal receptor; they lack most of the extracytoplasmic domain but retain the transmembrane and tyrosine kinase domains (Fung et al., (1984) Activation of the Cellular Oncogene c-erb B by LTR Insertion: Molecular Basis for Induction of Erythroblastosis by Avian Leukosis Virus. Cell 33:357-368; Yamamoto et al., (1983). A New Avain Erythroblastosis Virus, AEV-H Carries erbB Gene Responsible for the Induction of Both Erythroblastosis and Sarcoma. Cell 34:225-232, Nilsen et al., (1985). c-erbB Activation in ALV-Induced Erythroblastosis: Novel RNA Processing and Promoter Insertion Results in Expression of an Amino-Truncated EOF Receptor. Cell 41:719-72'), Gammett et al., (1986). Differences in Sequences Encoding the Carboxy-Terminal Domain of the Epidermal Growth Factor Receptor Correlate with Differences in the Disease Potential of Viral erbB Genes. Proc. Natl. Acad. Sci. USA 83:6053-6057). This results in a protein that is unable to bind epidermal growth factor (EGF) but can still phosphorylate other substrates (Gilmore et al., (1985). Protein Phosphorlytion at Tyrosine is Induced by the v-erb B Gene Product in Vivo and In Vitro. Cell 40:609-618; Kris et al., (1985). Antibodies Against a Synthetic Peptide as a Probe for the Kinase Activity of the Avian EGF Receptor and v-erB Protein. Cell 40:619-625), and has led to speculation that the v-erb B proteins are oncogenic because the kinase domain is unregulated and constitutively active (Downward et al., 1984).<br>
[0007] A variety of genetic alterations can occur in viral erb B oncogenes, e.g. amino acid substitutions and deletions in the carboxy terminus of the gene. Available evidence, however, argues lliiit the umino truncation is critical to carcinogcncsis. Amino truncations arc a feature of all v-erb B oncogenes, including those that arise by promoter insertion or retroviral transduction (Nilsen et al., (1985). c-erbB Activation in ALV-Induced Erythroblastosis: Novel RNA Processing and Promoter Insertion Results in Expression of an Amino-Truncated EGF Receptor. Cell 41:719-726;<br><br>
Gammett et al., (1986). Differences in Sequences Encoding the Carboxy-Terminal Domain of the Epidermal Growth Factor Receptor Correlate with Differences in the Disease Potential of Viral erbB Genes. Proc. Natl. Acad. Sci. USA 83:6053-6057).<br>
(0008) In contrast, carboxy-terminal deletions appear to be associated only with tumors that arise through retroviral transduction and seem to determine host range and tumor type specificity (Gammett et al., 1986; Raines et al., (1985). c-erbB Activation in Avian Leukosis Virus-Induced Erythroblastosis: Clustered Integration Sites and the Arrangement of Provirus in the c-erbB Alleles. Proc. Natl. Acad. Sci. USA 82:2287-2291). Transfection experiments with amino-truncated avian c-erb B genes or chimeric viral oncogenc-human EGF receptors demonstrates that this deletion is sufficient alone to create a transforming protein (Pelley et al., (1988). Proviral-Activated c-erbB is Leukemogenic but not Sarcomagenic: Characterization of a Replication—Competent Retrovirus Containing the Activated c-erbB. Journal of Virology 62: 1840-1844; Wells et al., (1988). Genetic Determinants of Neoplastic Transformation by the Retroviral Oncogene v-erbB. Proc. Natl. Acad. Sci. USA 85:7597-7601).<br>
[0009] Amplification of the EGFR gene occurs in 40% of malignant human gliomas (Libermann et al., (1985) Amplification, Enhanced Expression and Possible Rearrangement of EGF Receptor Gene in Primary Human Brain Tumours of Glial Origin. Nature 313:144-147; Wong et al., (1987). Increased Expression of the Epidermal Growth Factor Receptor Gene in Malignant Gliomas is Invariably Associated with Gene Amplification. Proc. Natl. Acad. Sci. USA 84:6899-6903), Rearrangement of the receptor gene is evident in many of the tumors with gene amplification. The structural alterations seem to preferentially affect the amino terminal half of the gene (Yamazaki et al., (1985). Amplification, Enhanced Expression and Possible Rearrangement of EGF Receptor Gene in Primary Human Brain Tumours of Glial Origin. Nature 313:144-147; Maiden et al., (1988). Selective Amplification of the Cytoplasmic Domain of the Epidermal Growth Factor Receptor Gene in Glioblastoma Multiforme. Cancer Research 4:2711-2714), but the nature of the rearrangements had not at that time been precisely characterized in any tumor.<br>
|0010] Size variant EGFR genes and amplification have been reported in several human cancers. (Humphrey et al., (1988). Amplification and Expression of the Epidermal Growth Factor Receptor Gene in Human Glioma Xenografts. Cancer Research 48:2231-2238; Bigner et al., (1988) J. Neuropathol. Exp. Neurol., 47:191-205; Wong et al., (1987). Increased Expression of the Epidermal Growth Factor Receptor Gene in Malignant Gliomas is Invariably Associated with Gene Amplification. Proc. Natl. Acad. Sci. USA 84:6899-6903; and Humphrey et al. Amplification and expression of the epidermal growth factor receptor gene in human glioma xenografts. Cancer Res. 48(8):2231-8 (1988)) There had been no determination, however, of the molecular basis for the altered EGFR molecules in cells.<br>
[0011) In 1989, work of Drs. Bigner and Vogelstein elucidated the sequence of a EGF receptor mutant that has become known as the type III mutant (also referred to as delta-EGFr or<br><br>
EGFrvIII). This work is described in U.S. Patent Nos. 6,455,498, 6,127,126, 5,981,725, 5,814,317, 5,710,010, 5,401,828, and 5,212,290.<br>
[0012]        EGFR variants are caused by gene rearrangement accompanied by EGFR gene<br>
amplification. There are eight major variants of EGFr that are known: (i) EGFRvI lacks a majority<br>
of the extracellular domain of EGFR, (ii) EGFRvIl consists of an 83 aa in-frame deletion in the<br>
extracellular domain of EGFR, (iii) EGFRvID consists of a 267 aa in-frame deletion in the<br>
extracellular domain of EGFR, (iv) EGFRvIV contains deletions in the cytoplasmic domain of<br>
EGFR, (v) EGFRvV contains deletions in cytoplasmic domain of EGFR, (vi) EGFR.TDM/2-7<br>
contains a duplication of exons 2-7 in the extracellular domain of EGFR, (vii) EGFR.TDM/18-25<br>
contains a duplication of exons 18-26 in the tyrosine kinase domain of EGFR, and    (viii)<br>
EGFR.TDM/18-26 contains a duplication of exons 18-26 in the tyrosine kinase domain of EGFR<br>
(Kuan et al. EGF mutant receptor vIII as a molecular target in cancer therapy. Endocr Relat Cancer,<br>
8(2}:83-96 (2001)). In addition, there is a second, more rare, EGFRvIII mutant (EGFRvIIIM12-13)<br>
that possesses a second deletion that introduces a novel histidine residue at the junction of exons 11<br>
and 14 (Kuan et al. EGF mutant receptor vID as a molecular target in cancer therapy. Endocr Relat<br>
Cancer. 8(2):83-96 (2001)).<br>
|0013| EGFRvIII is the most commonly occurring variant of the epidermal growth factor (EGF) receptor in human cancers (Kuan et al. EGF mutant receptor vIII as a molecular target in cancer therapy. Endocr Relal Cancer. 8(2):83-96 (2001)). During the process of gene amplification, a 267 amino acid deletion occurs in the extracellular domain creating a novel junction lo which tumor specific monoclonal antibodies can be directed. This variant of the EGF receptor contributes to tumor progression through consli.utive signaling in a ligand independent manner, EGFrVIIJ is not know to be expressed on any normal tissues (Wikstrand, CJ. et al. Monoclonal antibodies against EGFRVI11 are tumor specific and react with breast and lung carcinomas malignant gliomas. Cancer Research 55(14): 3140-3148 (1995); Olapade-Olaopa, EO. et al. Evidence for the differential expression of a variant EGF receptor protein in human prostate cancer. Br J Cancer. 82(l):186-94 (2000)). Yet, EGFRvffl shows significant expression in tumor cells, e.g., 27-76% brcasl cancer biopsies express EGFRvIII (Wikstrand, CJ. et al. Monoclonal antibodies against EGFRvIII are tumor specific and react with breast and lung carcinomas malignant gliomas. Cancer Research 55(14): 3140-3148 (1995); Ge H. et al. Evidence of high incidence of EGFRvIII expression and coexpression with EGFR in human invasive breast cancer by laser capture microdissection and immunohistochemical analysis. Int J Cancer. 98(3):357-61 (2002)), 50-70% gliomas express EGFRvIII (Wikstrand, CJ. et al. Monoclonal antibodies against EGFRVIII are tumor specific and react with breast and lung carcinomas malignant gliomas. Cancer Research 55(14): 3140-3148 (1995); Moscatello, G. et al. Frequent expression of a mutant epidermal growth factor receptor in multiple human tumors. Cancer Res. 55(23).5536-9 (1995)), 16% NSCL cancers express EGFRvID<br><br>
(Garcia de Palazzo, IE. et al. Expression of mutated epidermal growth factor receptor by non-small cell lung carcinomas. Cancer Res. 53(14):3217-20 (1993)), 75% ovarian cancers express EGFRvUI (Moscatello, G. et al. Frequent expression of a mutant epidermal growth factor receptor in multiple human tumors. Cancer Res. 55(23):5536-9 (1995)), and 68% prostate cancers express EGFRvIH (Olapade-Olaopa, EO. et al. Evidence for the differential expression of a variant EGF receptor protein in human prostate cancer. Br J Cancer. 82(1): 186-94 (2000)).<br>
[0014] The deletion of 267 amino acids with a Glycine substitution creates a unique junction that may be capable of antibody targeting. Further, in view of EGFRvIIFs expression in certain tumors and its lack of expression in normal tissues, EGFRvIIl may be an ideal target for drug targeting in tumor therapy. In particular, EGFRvIIl would appear to be an ideal candidate for immunoconjugate therapy of tumors (e.g., an antibody conjugated to an antineoplastic agent or toxin). Another method of treatment of cancers which over-express EGFRvIU involved the use of a tumor-specific ribozyme targeted specifically to the variant receptor which did not cleave normal EGFR. The ribozyme was found to significantly inhibit breast cancer growth in athymic nude mice (Luo et al. Int. J. Cancer. 104(6);716-21 (2003)).<br>
[0015] General antibodies for the entire EGFRvIIl protein have been described. See International Patent Application No. WO 01/62931 and Kuan et al. EGF mutant receptor vIII as a molecular target in cancer therapy. Endocr Relat Cancer. 8(2):83-96 (2001), Kuan et al. EGFRvin as a promising target for antibody-based brain tumor therapy. Brain Tumor Pathol. 17(2):71-78 (2000), Kuan et al. Increased binding affinity enhances targeting of glioma xenografts by EGFRvIH-specific scFv. International Journal of Cancer. 88(6):962-969 (2000), Landry et al. Antibody recognition of a conformational epitope in a peptide antigen: Fv-peptide complex of an antibody fragment specific for the mutant EGF receptor, EGFRvUI. Journal of Molecular Biology. 308(5):883-893 (2001), Reist et al. Astatine-211 labeling of internalizing anti-EGFRvIII monoclonal antibody using N-succinimidyl 5-[21 lAt]astato-3-pyridinecarboxylate. Nuclear Medicine and Biology. 26(4):405-411 (1999), Reist et al. In vitro and in vivo behavior of radiolabeled chimcric anti-EGFRvIll monoclonal antibody: comparison with its murine parent. Nuclear Medicine and Biology. 24(7):639-647 (1997), Wikstrand et al. Generation of anti-idiotypic reagents in the EGFRvIIl tumor-associated antigen system. Cancer Immunology, Immunotherapy. 50(12):639-652 (2002), Wikstrand et al. Monoclonal antibodies against EGFRvIIl are tumor specific and react with breast and lung carcinomas malignant gliomas. Cancer Research. 55(14):3140-3148 (1995), Wikstrand et al. The class III variant of the epidermal growth factor receptor (EGFRVI1I): characterization and utilization as an immunothcrapeutic target. J.Neurovirol. 4(2):148-158 (1998), Wikstrand et al. The class III variant of the epidermal growth factor receptor (EGFRvUI): characterization and utilization as an immunotherapeutic target. J.Neurovirol. 4(2):148-158 (1998), Jungbluth et al. A monoclonal antibody recognizing human cancers with amplification/ovcrexpression of the human epidermal growth factor receptor. Proc Natl Acad Sci USA.<br><br>
100(2):639-44 (2003), Mamot et al. Epidermal Growth Factor Receptor (EGFR)-targeted Immunoliposomes Mediate Specific and Efficient Drug Delivery to EGFR- and EGFRvITJ-overexpressing Tumor Cells. Cancer Research 63:3154-3161 (2003)). Each of these above-mentioned antibodies, however, possess or contain murine sequences in either the variable and/or constant regions. The presence of such murine derived proteins can lead to the rapid clearance of the antibodies or can lead to the generation of an immune response against the antibody in a patient. In addition, such antibodies are relatively low affinity, on the order of 2.2 x 10-8through 1.5 x 10"", even after affinity maturation. (Kuan et al. EGF mutant receptor vIII as a molecular target in cancer therapy. Endocr Rclat Cancer. 8(2):83-96 (2001)).<br>
[0016] In order to avoid the utilization of murine or rat derived antibodies, researchers have introduced human antibody function into rodents so that the rodents can produce fully human antibodies. See e.g., Mendez et al. Functional transplant of megabase human immunoglobulin loci recapitulates human antibody response in mice. Nat Genet.l5(2): 146-56 (1997). This approach has been used in connection with the generation of successful antibodies directed against wild type EGFR. See e.g., Yang X et al. Development of ABX-EGF, a fully human anti-EGF receptor monoclonal antibody, for cancer therapy. Crit Rev Oncoi Hemato 38(l):17-23 (2001); Yang X-D et al. Eradication of Established Tumors by a Fully Human Monoclonal Antibody to the Epidermal Growth Factor Receptor without Concomitant Chemotherapy. Cancer Research 59(6): 1236-1243 (1999); and U.S. Patent No. 6,235,883.<br>
SUMMARY OF THE INVENTION<br>
[0017) In one embodiment, the invention comprises an isolated human monoclonal antibody that specifically binds to EGFRvIII and a peptide that comprises the sequence L E E K K G N Y V V T D H C (SEQ ID NO: 56). In one embodiment, a therapeutic agent may be conjugated to the antibody. In one embodiment, a toxin is used. In another embodiment, the invention comprises an isolated human monoclonal antibody that specifically binds to an epitope contained within a sequence comprising LEEKKGNYVVTDHC (SEQ ID NO: 56), wherein the residues required for binding, as determined by Alanine scanning in a SPOTs array, are selected from the group consisting of EEK, KK.NYV, LEK, EKNY and EEKGN.<br>
|0018] Further embodiments include an isolated human monoclonal antibody that comprises a heavy chain variable region amino sequence that is encoded by a VH3-33 gene. The heavy chain variable region amino sequence can include an amino acid sequence that is encoded by a JH4b gene, or an ammo acid sequence that is encoded by a D gene that is selected from the group consisting of D6-13 and D3-9.<br>
[0019] Other embodiments include an isolated human monoclonal antibody that comprises a light chain variable region amino sequence that is encoded by a A23(VK2) gene. The light chain variable region amino sequence can include an amino acid sequence that is encoded by a JK1 gene.<br><br>
[0020] Other embodiments include an isolated antibody, or fragment thereof, that binds to EGFRvIU and that comprises a heavy chain amino acid sequence selected from the group consisting of the heavy chain amino acid sequence of antibody 13.1.2, 131, 170, 150, 095, 250, 139, 211, 124, 318, 342 and 333 as identified in (SEQ ID NO: 138, 2, 4, 5, 1, 9, 10, 12, 13, 15, 16, and 17). The antibody can be a monoclonal antibody, a chimeric antibody, a humanized antibody or a human antibody. The antibody or fragment can be associated with a pharmaceutically acceptable carrier or diluent, and can be conjugated to a therapeutic agent. The therapeutic agent can be a toxin. The therapeutic agent can be a toxin such as DM-1, AEFP, AURISTATIN E, or ZAP. The agent can be associated with the antibody via a linker. The toxin can be associated with the antibody via a secondary antibody. Further embodiments include a hybridoma cell line producing the antibody, and a transformed cell comprising a gene encoding the antibody. The cell can be, for example, a Chinese hamster ovary cell.<br>
[0021] Further embodiments include a method of inhibiting cell proliferation associated with the expression of EGFRvIU, comprising treating cells expressing EGFRvIU with an effective amount of the antibody or fragment. In one emobidment, the antibody comprises a heavy chain amino acid sequence selected from the group consisting of the heavy chain amino acid sequence of antibody 13.1.2 (SEQ ID NO: 138), 131 (SEQ ID NO: 2), 170 (SEQ ED NO: 4), 150 (SEQ ID NO: 5), 095 (SEQ ID NO: 7), 250 (SEQ ED NO: 9), 139 (SEQ ID NO: 10), 211 (SEQ ID NO: 12), 124 (SEQ ID NO: 13), 318 (SEQ ID NO: 15), 342 (SEQ ID NO: 16), and 333 (SEQ ID NO: 17). The method can be performed in vivo, and performed on a mammal, such as a human, who suffers from a cancer involving epithelial cell proliferation, such as a lung, colon, gastric, renal, prostate, breast, glioblastoma or ovarian carcinoma.<br>
(0022) Further embodiments include a method of killing a targeted cell. This is achievd by contacting the targeted cell with an antibody associated with a toxin. The antibody binds to a peptide LEEKKGNY (SEQ ID NO: 133). In one embodiment, the antibody has a binding affinity greater than 1.3*10"9M to the peptide. In one embodiment the toxin is selected from AEFP, MMAE, DM-1, and ZAP. In one embodiment, the antibody toxin compound is 10 fold more toxic to targeted cells than to cells without the peptide. In one embodiment, the antibody comprises a heavy chain amino acid sequence selected from the group consisting of the heavy chain amino acid sequence of antibody 13.1.2 (SEQ ID NO: 138), 131 (SEQ ED NO: 2), 170 (SEQ DO NO: 4), 150 (SEQ ED NO: 5), 095 (SEQ ID NO: 7), 250 (SEQ BO NO: 9), 139 (SEQ ID NO: 10), 211 (SEQ ID NO: 12), 124 (SEQ ID NO: 13), 318 (SEQ DO NO: 15), 342 (SEQ ID NO: 16), and 333 (SEQ DO NO: 17). In another embodiment, the antibody is associated with a toxin via a peptide linker or a second antibody.<br>
[0023] Further embodiments of the invention include an isolated antibody that binds to EGFRvIU and that comprises a heavy chain amino acid sequence comprising the following complementarity determining regions (CDRs):<br><br>
(a)	CDR1 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for the CDR1 region of antibodies 13.1.2, 131, 170, 150,095,250, 139,211, 124,<br>
318, 342 and 333 as identified in SEQ ID NO: 138,2,4,5,7,9, 10, 12, 13, 15, 16, and 17;<br>
(b)	CDR2 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for the CDR2 region of antibodies 13.1.2, 131, 170, 150, 095, 250, 139, 211, 124,<br>
318, 342 and 333 as identified in SEQ ID NO: 138, 2, 4, 5, 7, 9, 10, 12, 13, 15, 16, and 17;<br>
and<br>
(c)	CDR3 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for  the ('I)IU region of  antibodies 13 1.2.13l. 170, 150, 095, 250. \V), 21 I, 17.1,<br>
118,  192; AND  n I „., hi,,Mi I MM in M'l.)SEQ  NO., 9,10,12,13,15,16,and 17 <br>
In one embodiment, the antibody is a monoclonal antibody, a chimcric antibody, human, or a<br>
humanized antibody. In one embodiment, the antibody is associated with a pharmaceutically<br>
acceptable carrier, diluent, and/or therapeutic agent.   In one embodiment, the therapeuctic<br>
agent is a toxin. In one embodiment, the the toxin is DM-1 or Auristatin E.<br><br>
(24)	Also included is an isolated antibody, or fragment thereof, that binds to<br>
EGFRvIlI and that comprises a light chain amino acid sequence selected from the group consisting of<br>
the light chain amino acid sequence of antibody 13.1.2, 131, 170, 150, 123,095, 139,250,211, 318,<br>
342, and 333 as identified in SEQ ED NO: 140, 19, 20, 21, 29, 23, 25, 26, 28, 33, 31 and 32.    The<br>
antibody can be a monoclonal antibody, a chimeric antibody, a humanized antibody, or a human<br>
antibody. It can be associated with a pharmaceutically acceptable carrier or diluent, or conjugated to<br>
a therapeutic agent, such as a toxin, for example DM1 or AURISTATIN E.   In one embodiment a<br>
hybridoma cell line or a transformed cell producing an antibody comprising a light chain amino acid<br>
sequence selected from the group consisting of the light chain amino acid sequence of antibody<br>
13.1.2, 131, 170, 150, 123,095, 139,250,211,318, 342, and 333 as identified in SEQ ID NO: 140,<br>
19, 20, 21, 29, 23, 25, 26, 28, 33, 31 and 32 is contemplated.<br>
(25)	Further embodiments include a hybridoma cell line producing such an antibody,<br>
and a transformed cell, such as a Chinese hamster ovary cell, comprising a gene encoding the<br>
antibody.<br>
[0026] Yet another embodiment includes a method of inhibiting cell proliferation associated with the expression of EGFRvIII, comprising treating cells expressing EGFRvIlI with an effective amount of the antibodies or fragments described above. The method can be performed in vivo and on a mammal, such as a human, who suffers from a cancer involving epithelial cell proliferation such as lung, colon, gastric, renal, prostate, breast, glioblastoma or ovarian carcinoma.<br>
(0027) Yet another embodiment includes an isolated antibody that binds to EGFRvIlI and that comprises a light chain amino acid sequence comprising the following complementarity determining regions (CDRs):<br><br>
(a)	CDR1 consisting of a sequence selected from the group consisting of the amino acid<br>
scquenccsforthe.CDR1 region of antibodies 13.1.2, 131, 170, 150, 123,095, 139,250,211,<br>
318, 342, and 333 as identified in SEQ ID NO: 140, 19, 20, 21, 29, 23, 25, 26, 28, 33, 31 and<br>
32;<br>
(b)	CDR2 consisting of a sequence selected from the group consisting of amino acid<br>
sequences for the CDR1 region of antibodies 13.1.2, 131, 170, 150, 123, 095, 139, 250, 211,<br>
318, 342, and 333 as identified in SEQ ID NO: 140, 19, 20, 21,29, 23,25, 26, 28, 33, 31 and<br>
32; and<br>
(c)	CDR3 consisting of a sequence selected from the group consisting of amino acid<br>
sequences for the CDR1 region of antibodies 13.1.2, 131, 170,150, 123,095, 139,250,211,<br>
318, 342, and 333 as identified in SEQ ID NO: 140,19, 20, 21, 29, 23, 25, 26, 28, 33, 31 and<br>
32.<br>
[0028]        The antibody described in the paragraph above can also include a heavy chain amino acid sequence comprising the following complementarity determining regions (CDRs):<br>
(a)	CDR1 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for the CDR1 region of antibodies 13.1.2, 131, 170, 150,095,250,139,211, 124,<br>
318, 342 and 333 as identified in SEQ ID NO: 138,2,4,5,7,9, 10, 12, 13,15, 16, and 17;<br>
(b)	CDR2 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for the CDR2 region of antibodies 13.1.2, 131, 170, 150,095,250,139,211, 124,<br>
318, 342 and 333 as identified in SEQ ID NO: 138, 2, 4, 5, 7, 9, 10, 12, 13, 15, 16, and 17;<br>
and<br>
(c)	CDR3 consisting of a sequence selected from the group consisting of the amino acid<br>
sequences for the CDR3 region of antibodies 13.1.2, 131, 170, 150,095,250, 139,211, 124,<br>
318, 342 and 333 as identified in SEQ ID NO: 138, 2,4, 5,7, 9, 10, 12, 13, 15, 16, and 17.<br>
(0029|         Further embodiments include a method of inhibiting cell proliferation associated<br>
with the expression of EGFRvlll, comprising treating cells expressing EGFRvIII with an effective amount of the antibody or fragment described above. The method can be performed in vivo, on a mammal, such as a human, suffering from a cancer involving epithelial cell proliferation, such as lung carcinoma, breast carcinoma, head &amp; neck cancer, prostate carcinoma or glioblastoma.<br>
[0030) Further embodiments include an isolated polynucleotide molecule comprising a nucleotide sequence encoding a heavy chain amino acid sequence, or a fragment thereof, selected from the group consisting of the heavy chain amino acid sequence of antibodies 13.1.2, 131, 170, 150, 095, 250, 139, 211, 124, 318, 342, and 333 as identified in SEQ ID NO: 138, 2,4, 5,7, 9, 10, 12, 13, 15, 16, and 17, or an isolated polynucleotide molecule comprising a nucleotide sequence encoding a light chain amino acid sequence, or a fragment thereof, selected from the group consisting of the light chain amino acid sequence of antibodies 13.1.2, 131, 170, 150, 123, 095, 139, 250, 211, 318, 342, and 333, as identified in SEQ ID NO: 140, 19, 20, 21, 29, 23, 25, 26, 28, 33, 31 and 32.<br><br>
[0031] Further embodiments include an article of manufacture comprising a container, a composition contained therein, and a package insert or label indicating that the composition can be used to treat cancer characterized by the expression of EGFRvIlI, wherein the composition comprises an antibody as described above. Such cancers include a lung carcinoma, breast carcinoma, head &amp; neck cancer, prostate carcinoma or glioblastoma. Also included is an assay kit for the detection of EGFRvIlI in mammalian tissues or cells in order to screen for lung, colon, gastric, renal, prostate or ovarian carcinomas, the EGFRvlII being an antigen expressed by epithelial cancers, the kit comprising an antibody that binds the antigen protein and means for indicating the reaction of the antibody with the antigen, if present. The antibody can be a labeled monoclonal antibody, or the antibody can be an unlabeled first antibody and the means for indicating the reaction comprises a labeled second antibody that is anti-immunoglobulin. The antibody that binds the antigen can be labeled with a marker selected from the grcip consisting of a fluorochrome, an enzyme, a Radionuclide and a radiopaque material. The antibody that binds the antigen can also bind to over-expressed wtEGFR. The kit can be used clinically for patient selection.<br>
(0032] A further embodiment includes an antibody which specifically recognizes the epitope of EGFRvlII containing the novel Gly residue.<br>
|0033] A further embodiment includes a protein variant of EGFRvlII. The variant may have a pFLAG insert, may consist of the amino acids in SEQ ID NO: 56, and can exist in silico.<br>
J0034) Another embodiment includes an antibody, or variant thereof, which binds to the recognition sequence EEKKGNYVVT (SEQ ID NO: 57).<br>
|0035) Another embodiment includes an antibody variant that specifically binds to EGFRvlII. The antibody variant can further bind to a peptide that comprises SEQ ID NO: 57. The antibody variant can have residues that interact with residues EK.NY or EEKGN in the peptide. In one embodiment, the antibody variant binds to the peptide sequence ten fold more tightly than it does to a wild-type EGFR protein. In one embodiment, the antibody binds specifically binds to EGFRvlII and the peptide of SEQ ID NO: 56. In one embodiment, the isolated antibody or variant has a complementarity determining region comprising a deep cavity, wherein the cavity is created by CDR2 and CDR3 of the heavy chain, CDR3 of the light chain, and a small portion from CDR1 of the , light chain. In one embodiment, the isolated antibody or variant has residues 31, 37, 95-101, 143-147, 159, 162-166, 169-171, 211-219, 221, and 223 within 5 angstroms of a binding cavity. In one embodiment, the isolated antibody or variant ha:; a complementarity determining region comprising a narrow groove, wherein the groove is created by heavy chain CDR2 and CDR3, and light chain CDR1, CDR2, and CDR3. In one embodiment, the isolated antibody or variant has residues 31, 33, 35-39, 51, 54-56, 58-61, 94-101, 144-148, 160, 163-166, 172, and 211-221 within 5 angstroms of a binding groove. In one embodiment, the isolated antibody or variant has residues 31- 33, 35, 37, 55, 96-101, 148, 163, 165, 170, 172, 178, 217, and 218 within 5 angstroms of a binding groove. In one embodiment, the isolated antibody or variant has a paratope configured so that when the epitope of<br><br>
peptide EEKKGN (SEQ ID NO 127) binds to the paratope of the antibody, at least one bond is formed between two residues selected from the group consisting, of E2 and Y172, K3 and H31, K.4 and H31, N6 and D33, N6 and Y37, and N6 and K55. In one embodiment, the isolated antibody or variant has a paratope configured so that when the epitope of peptide EEKKGNY (SEQ ED 131) binds to the paratope of the antibody, at least one bond is formed between two residues selected from the group consisting of K4 and Q95, K4 and Q95, N6 and Q98, G5 and H31, Y7 and H31, Y7 and W165. In one embodiment, the antibody has a structure or interaction with a structure that is determined in silica.<br>
(0036) Another embodiment provides a method for selecting variants that bind to EGFRvIII with particular binding characteristics, the method comprising the use of a molecular structure to create a paratope, the use of a molecular structure to create an epitope, calculating the interaction energy between the two and comparing that energy level to the energy level of the epitope and a second paratope of a mAb variant, and selecting a variant based on the differences in the energy levels. The method can further include using an interaction energy between a second variant of the paratope and the epitope to determine a third interaction energy and comparing the third interaction energy and the second interaction energy lo determine which variant to select. In one embodiment, the variant is created and tested for binding.<br>
|0037| Another embodiment provides a method for selecting variants that bind to EGFRvIII with particular binding characteristic;,, the method comprising examining residues of an epitope which interact with a paratope, selecting important residues lo create a recognition sequence, using this sequence to create a EGFRvIII variant, and using the EGFRvIII variant to select the mAb variant.<br>
|0038) Another embodiment provides a method for making antibody variants to EGFRvIII, said method comprising analyzing the residues of an epitope which interact with a paratope, selecting the more important residues of an epitope to create a recognition sequence, using the recognition sequence to create an EGFRvIII variant, and using the EGFRvIII variant to select antibody variants. In one embodiment, the selection of the antibodies is achieved in silica. In one embodiment, the selection of the antibodies through the use of the EGFRvIII variant is achieved by raising antibodies against EGFRvIII variant.<br>
|0039] In the embodiment where the isolated antibody variant binds to EGFRvIII and the peptide of SEQ ID NO: 57, the antibody can further comprise a point mutation of the following: Tyrl72Arg, Leu99Glu, ArglOlGlu, Leu217Glu, Leu99Asn, Leu99His, L99T, ArglOlAsp, or some combination thereof In one embodiment, the antibody is a monoclonal antibody, a chimeric antibody, a humanized antibody, or a human antibody.<br>
[0040] In one embodiment, the antibody or variant thereof binds to the sequence EEKKGNYWT (SEQ ID NO: 57), and the antibody or variant has subnanomolar binding ability.<br><br><br>
[0041] In a further embodiment, the antibody binds to EGFRvIII and the antibody has a paratope that binds to an epitope, and the epitope has 2 set of residues that interact with the paratope that include E, K, N, and Y. In one embodiment, the antibody is antibody 131.<br>
[0042]        In a further embodiment, the antibody binds to EGFRvIII and the antibody has a paratope that binds to an epitope that has a set of residues that interact with the paratope comprising: ' E, E, K, G, and N. In one embodiment, the primary structure of the epitope is EEKKGNY (SEQ ED NO: 131). In one embodiment, the antibody is 13.1.2.<br>
[0043] In a further embodiment, th: antibody that binds to EGFRvIII and has a KD of less than 1.3* 109 M, less than 1.0*10-9 M, or less than 500 pM. In one embodiment, the antibody is specific for SEQ ED NO: 56 compared to a wild type EGFR peptide. In one embodiment, the nonspecific binding of the antibody to the wild type EGFR peptide (SEQ ED NO: 134) is less than 10% of that of the specific binding of the antibody to EGFRVIII (SEQ ED NO: 135). In one embodiment, the antibody is selected from the group consisting of 131, 139, and 13.1.2. In one embodiment, the antibody is internalized. In one embodiment, the intemalization occurs for at least about 70% or at least about 80% of the antibody.<br>
[0044] In one embodiment, the variant human monoclonal antibody preferentially binds to an epitope that is substantially unique to an EGFRvIII protein compared to a wild-type EGFR protein or variant thereof (SEQ ED NO: 134). In one embodiment, the variant comprises a heavy chain complementarity determining region (CDR1) corresponding to canonical class 1. In one embodiment, the variant comprises a heavy chain complementarity determining region (CDR2) corresponding to canonical class 3. In one embodiment, the variant comprises a light chain complementarity determining region (CDR1) corresponding to canonical class 4. In one embodiment, the variant comprises a light chain complementarity determining region (CDR2) corresponding to canonical class 1. In one embodiment, the variant comprises a light chain complementarity determining region (CDR3) corresponding to canonical class 1. In one embodiment, the variant comprises a first heavy chain complementarity determining region (CDR1) corresponding to canonical class 1, a second neavy chain complementarity determining region (CDR2) corresponding to canonical class 3, a first light chain complementarity determining region (CDR1) corresponding to canonical class 4, a second light chain complementarity determining region (CDR2) corresponding to canonical class 1; and a third light chain complementarity determining region (CDR3) corresponding to canonical class 1, wherein the complementary determining regions are configured to allow the variant to bind to an epitope that is substantially unique to an EGFRvIII protein as compared to a EGFR protein.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[0045] FIG. 1 is an alignment between wild type EGFR and EGFRvIII showing the 267 amino acid deletion and G substition.<br><br>
[0046] FIG. 2 is a diagram of the design of the EGFRvHI PEP3 14-mer peptide. In FIG. 2A, the N-terminal sequence of EGFRvIII with amino acids LEEKK (SEQ ID NO: 58) (1-5) that are identical to the N-terminal seqence of EGFR, followed by the unique Glysine residue, followed by amino acids that are identical to residues 273 through 280 in EGFR. FIG. 2B represents the amino acids of EGFR that are deleted in EGFRvlIl (6-272).<br>
(0047] FIGs. 3A-L provide sequences of antibodies of the invention. For each antibody provided, a nucleotide and amino acid sequence is provided for both a heavy chain and a light chain variable region. Accordingly, four sequences are provided for every antibody listed.<br>
|0048] FIG. 4 is a table comparing the 13.1.2 antibody heavy chain regions to a particular germ line heavy chain region, "-"s indicate that the amino acid residue of the hybridoma heavy chain region is the same as the germ line for that particular position. Deviation from the germline is indicated by the appropriate amino acid residue.<br>
|0049] FIG. 5 is a table comparing the 13.1.2 antibody light chain regions to a particular germ line light chain region, "-"s indicate that the amino acid residue of the hybridoma light chain region is the same as the germ line for that particular position. Deviation from the germline is indicated by the appropriate amino acid residue.<br>
[0050] FIG. 6 is a table comparing various hybridoma derived antibody heavy chain regions to a particular germ line heavy chain region, "-"s indicate that the amino acid residue of the hybridoma heavy chain region is the same as the germ line for that particular position. Deviation from the germline is indicated by the appropriate amino acid residue.<br>
(0051) FIG. 7 is a table comparing various hybridoma derived antibody light chain regions to a particular germ line light chain region, "-"s indicate that the amino acid residue of the hybridoma light chain region is the same as the germ line for that particular position. Deviation from the germline is indicated by the appropriate amino acid residue.<br>
|0052] FIG. 8 is a representative figure showing binding of recombinant EGFRvIII mAbs to cells expressing EGFRvIII (NR6 cells). Diamonds represent 95, triangles represent 133, squares represent 139, "x" represent 150, asterixes represent 170, circles represent 221, lines 230, and rectangles represent 250.<br>
|0053] FIG. 9A shows FACS staining analysis for a human anti-EGFR antibody (ABX-EGF) to H80.<br>
|0054|         FIG. 9B shows FACS staining analysis for antibody 131 to H80.<br>
|0055]         FIG. 9C shows FACS staining analysis for antibody 139 to H80.<br>
[0056]         FIG. 9D shows FACS staining analysis for antibody 13.1.2 to H80.<br>
|0057]         FIG. 9E shows FACS staining analysis for ABX-EGF to H1477.<br>
[0058]         FIG. 9F shows FACS staining analysis for antibody 131 to H1477.<br>
[0059]         FIG. 9G shows FACS staining analysis for antibody 139 to H1477.<br>
|0060|         FIG. 9H shows FACS staining analysis for antibody 13.1.2 to II1477.<br><br>
[0061]        FIG. 91 shows FACS staining analysis foi ABX-EGF to A549.<br>
[0062]        FIG. 9J shows FACS staining analysis for antibody 131 to A549.<br>
|0063)        FIG. 9K shows FACS staining analysis for antibody 139 to A549.<br>
|0064)        FIG. 9L shows FACS staining analysis for antibody 13.1.2 to A549.<br>
[0065) FIG. 9M is a graph displaying binding of EGFRvIII mAbs to glioblastoma cells. Filled triangles represent antibody 131 binding to H1477. Filled squares represent antibody 13.1.2 binding to HI477. Empty triangles represent antibody 131 binding to H80. Empty squares represent antibody 13.1.2 binding to H80.<br>
[0066] FIG. 9N is a graph displaying the binding of EGFRvIII mAbs to human epidermoid carcinoma cell line A431. The filled squares represent antibody 13.1.2. The filled tnangles represent antibody 131.<br>
[0067] FIG. 9O is a graph displaying the binding of antibody 13.1.2 to NR6 murine fibroblast cell lines. The squares represent NR6. The triangles represent NR6 with with wild type EDFR. The circles represent NR6 with EGFRvIII.<br>
[0068] FIG. 9P is a graph displaying the binding of antibody 131 to murine fibroblast cell lines. The squares represent NR6. The triangles represent NR6 with wild type EGFR. The circles represent NR6 with EGFRvIII.<br>
(0069)	FIG.   10A shows FACS staining analysis for a human anti-EGFR antibody<br>
(ABX-EGF) binding to cells expressing EGFR (A431).<br>
[0070)	FIG. 10B shows FACS staining analysis for antibody 131 to cells expressing<br>
EGFR(A431).<br>
(0071)	FIG. IOC shows FACS staining analysis for antibody 139 to cells expressing<br>
EGFR(A431).<br>
[0072)	FIG. 10D shows FACS staining analysis for antibody 13.1.2 to cells expressing<br>
EGFR(A431).<br>
[0073| FIG. 11 shows the molecular surface of antibody 131 structure model. The six CDRs are shaded different shades to mark their boundries. The binding cavity is located close to the center.<br>
(0074)	FIG. 12 shows a structural model of the molecular surface of antibody 13.1.2.<br>
The six  CDR regions are shaded and  identified by number.     The  long groove  is located<br>
approximately along the vertical centerline.<br>
[0075)	FIG.  13A is a possible docking model of the 13.1.2 antibody and peptide<br>
EEKJCGN (SEQ ID NO: 127) complex. The CDR regions are shaded to denote boundries.<br>
[0076] FIG. ) 3B shows the hydrogen bonds-in the docking model of the 13.1.2 antibody and peptide EEKKGN (SEQ ID NO: 127) complex. Shading of the CDR loops and residues is the same as in FIG. 12. The peptide residue is numbered from the N-terminus, at the top of the figure, to the C-terminus as 1 through 6. Six hydrogen bonds are indicated by dashed lines. The six pairs of<br><br>
amino acids forming hydrogen bonds are: E2...Y172, K3...H31, K4...H31, N6...D33, N6...Y37, andN6...K55.<br>
|0077]        FIG. 14 is a graph demonstrating a correlation between the epitope-antibody binding energy and the logrithm of Kd for one of the docking models selected.<br>
|0078)        FIG. 15 is a depiction of a refined docking model for the peptide-13.1.2 antibody complex. The peptide is rendered in a space-filling manner..<br>
[0079] FIG. 16 is a depiction representing the hydrogen bonds in the refined docking model.<br>
|0080| FIG. 17 is a graph that depicts the linear fitting of antibody-antigen binding energy versus the logrithm of relative affinities.<br>
DETAILED DESCRIPTION OF THE PREFERRED EMBODIMENTS<br>
[0081] As discussed above, EGFRvIII is a deletion mutant of EGFR in which 267 amino acids in the extracellular domain of EGFr are deleted with a single amino acid substitution of Glycine at the junction. These features are showr in a sequence alignment between wild type EGFR and EGFRvIII in FIG. 1. In view of the amino acid substitution of Glycine at the junction of the deletion, it becomes theoretically possible to generate antibodies to the novel epitope present in EGFRvIII that is not present in wild type EGFR. Thus, a peptide for immunization and screening was designed, termed PEP3, as shown in FIG. 2 (Kuan et al. EOF mutant receptor vlll as a molecular target in cancer therapy. Endocr Relat Cancer. 8(2):83-96 (2001)). Such 14-mer peptide possesses the 5 n-terminal amino acids common to EGFRvIII and wild type EGFR, the unique Glycine junction site, and 8 amino acid residues contained in the conserved sequences between wild type EGFR (corresponding to residues 273-280) and EGFRvIII (corresponding to residues 7-14). In addition, glioblasloma cell and cells (B300.19 cells) transfected with the gene encoding EGFRvIII were also utilized for immunization and screening (sometimes referred to herein as B300.19/EGFRvIlI transfectants).<br>
|0082| In order to generate human antibodies against EGFRvIII, transgenic XenoMouse® mice were immunized with combinations of glioblastoma cells/EGFRvlII, B300.19/EGFRvlII cells, and peptides (PEP3) directed to the junction region in the novel extracellular domain represented in EGFRvIII as compared to wild type EGFR. B cells from immunized mice were isolated and cither used to produce hybridomas followed by screening for binding to liGFRvIIl or used directly in screening for binding to EGFRvIII using XenoMax™/SLAM™ technologies (Babcook ct al. A novel strategy for generating monoclonal antibodies from single, isolated lymphocytes producing antibodies of defined specificities. Proc Natl Acad Sci U S A.93(15):7843-8 (1996), and U.S. Patent No. 5,627,052). Antibodies identified that bound to EGFRvIII were screened in a series of assays to ascertain specific recognition of EGFRvIII. Through this process, panels of human monoclonal antibodies that bound to and were specific for EGFRvIII were generated, isolated, and characterized. Subsequent epitope mapping demonstrated<br><br>
unique but overlapping specificities. All antibodies were further evaluated in vitro for their ability to be internalized by cells for the purpose of delivering cytotoxic drugs to cells. Antibodies demonstrating efficient drug delivery were directly conjugated with a cytotoxic drug and examined for their ability to kill tumor cells expressing EGFRvIII in vitro and in vivo. These studies provide the basis for the next generation of antibody drug conjugates for treating cancer in patients whose tumor harbor specific genetic lesions.<br>
[0083| Through the processes described above, panels of fully human anti-EGFRvIIl antibodies were generated. Using the hybridoma approach, several antibodies, including antibody 13.1, 13.2, 13.3, and 13.4 that were positive on SLISA for binding with the PEP3, were generated with limited cross-reactivity with wild type EGFR. Out of these, antibody 13.1 (and, particularly, its subclone 13.1.2) was selected for further research and development. Using the XenoMax approach a panel of antibodies, including antibody 131, 139, 250, and 095, were generated that were highly specific for binding with the pep3 oligonucleotide and had limited cross-reactivity with wild type EGFR. Of these, the 131 antibody has very interesting properties. The sequences for each of the antibodies are displayed in FIGs. 4-7 (SEQ ID NO: 1-33 and 141-144). A comparison of the sequences and binding abilities of the various antibodies was made and the results are displayed in FIGs. 4-10. As can be seen in FIGs. 9A-9L, and FIGs. 10A-10D antibodies 131, 139, and 13.1.2 all demonstrated superior selectivity for EGFRvIII expressing cells (HI477) as compared to ABX-EGF. Some of the results are shown in graph form in FIGs. 9M-9P, which demonstrates that at least two of the antibodies, 13.1.2 and 131 demonstrated superior specificity for EGFRvin expressing cells compared to simply EGFRvIII cells. Finally, based on predicted structural models, variants of the antibodies were made in order to obtain antibodies with altered binding characteristics.<br>
|0084] Further, antibodies of the invention are highly useful for the screening of other antibodies that bind to the same or similar cpitopes. Antibodies of the invention can be utilized in cross competition studies for the elucidation of other antibodies that are expected to have the same or improved effects with respect to characteristics of the antigen-antibody complex that is formed.<br>
[0085) Each of the 131 antibody and the 13.1.2 possessed very high affinities for EGFRvIII, were internalized well by cells, ard appeared highly effective in cell killing when conjugated to toxins. Intriguingly, both of the antibodies, despite having been generated in different immunizations of XenoMouse mice, and utilizing different technologies, were derived from very similar germline genes. Based upon epitope mapping work, however, each of the antibodies appears to bind to slightly different epitopes on the EGFRvIII molecule and have slightly different residues on EGFRvIII that are essential for binding. These results indicate that the germline gene utilization is of importance to the generation of antibody therapeutics targeting EGFRvIII and that small changes can modify the binding and effects of the antibody in ways that allow for the further design of antibodies and other therapeutics based upon these structural findings.<br><br>
(0086)        Antibodies that bind to the same epitope as, or compete for binding with, the 13.1.2 and 131 antibodies are highly desirable. As discussed in more detail below, through Alanine scanning on SPOTs arrays important residues for binding of certain antibodies have been elucidated. Accordingly, antibdodies that share critical binding residues are also highly desirable. Definitions<br>
|0087] Unless otherwise defined, scientific and technical terms used herein shall have the meanings that are commonly understood by those of ordinary skill in the art. Further, unless otherwise required by context, singular terms shall include pluralities and plural terms shall include the singular. Generally, nomenclatures utilized in connection with, and techniques of, cell and tissue culture, molecular biology, and protein and oligo- or polynucleotide chemistry and hybridization described herein are those well known and commonly used in the art. Standard techniques are used for recombinant DNA, oligonucleotide synthesis, and tissue culture and transformation (e.g., electroporation, lipofection). Enzymatic reactions and purification techniques are performed according to manufacturer's specifications or as commonly accomplished in the art or as described herein. The foregoing techniques and procedures are generally performed according to conventional methods well known in the art and as described in vanous general and more specific references that are cited and discussed throughout the present specification. See e.g., Sambrook et al. Molecular Cloning: A Laboratory Manual (2d ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989). The nomenclatures utilix.cd in connection with, and the laboratory procedures and techniques of, analytical chemistry, synthetic organic chemistry, and medicinal and pharmaceutical chemistry described herein are those well known and commonly used in the art. Standard techniques are used for chemical syntheses, chemical analyses, pharmaceutical preparation, formulation, and delivery, and treatment of patients.<br>
|0088] The term "isolated polynucleotide" as used herein shall mean a polynucleotide of genomic, cDNA, or synthetic origin or some combination thereof, which by virtue of its origin the "isolated polynucleotide" (1) is not associated with all or a portion of a polynucleotide in which the "isolated polynucleotide" is found in nature, (2) is operably linked to a polynucleotide which it is not linked to in nature, or (3) does not occur in nature as part of a larger sequence.<br>
[0089] The term "isolated protein" referred to herein means a protein of cDNA, recombinant RNA, or synthetic origin or some combination thereof, which by virtue of its ongin, or source of derivation, the "isolated protein" (1) is not associated with proteins found in nature, (2) is free of other proteins from the same source, e.g. free of marine proteins, (3) is expressed by a cell from a different species, or (4) does not occur in nature.<br>
(0090) The term "polypeptide" is used herein as a generic term to refer to native protein, fragments, or analogs of a polypeptide sequence. Hence, native protein, fragments, and analogs are species of the polypeptide genus. Preferred polypeptides in accordance with the invention comprise the human heavy chain immunoglobulin molecules and the human kappa light chain immunoglobulin<br><br>
molecules, as well as antibody molecules formed by combinations comprising the heavy chain immunoglobulin molecules with light chain immunoglobulin molecules, such as the kappa light chain immunoglobulin molecules or lambda light chain immunoglobulin molecules, and vice versa, as well as fragments and analogs thereof.<br>
[0091] The term "naturally-occurring" as used herein as applied to an object refers to the fact that an object can be found in nature. For example, a polypeptide or polynucleotide sequence that is present in an organism (including viruses) that can be isolated from a source in nature and which has not been intentionally modified by man in the laboratory or otherwise is naturally-occurring.<br>
|0092| The term "operably linked" as used herein refers to positions of components so described are in a relationship permitting them to function in their intended manner. A control sequence "operably linked" to a coding sequence is ligated in such a way that expression of the coding sequence is achieved under conditions compatible with the control sequences.<br>
[0093] The term "control sequence" as used herein refers to polynucleotide sequences which are necessary to effect the expression and processing of coding sequences to which they are ligated. The nature of such control sequences differs depending upon the host organism; in prokaryotes, such control sequences generally include promoter, ribosomal binding site, and transcription termination sequence; in eukaryotes, generally, such control sequences include promoters and transcription termination sequence. The term "control sequences" is intended to include, at a minimum, all components whose presence is essential for expression and processing, and can also include additional components whose presence is advantageous, for example, leader sequences and fusion partner sequences.<br>
(0094) The term "polynucleotide" is referred to herein means a polymeric form of nucleotides of at least 10 bases in length, either nbonucleotides or deoxynucleotides or a modified form of either type of nucleotide. The term includes single and double stranded forms of DNA.<br>
[0095] The term "oligonucleotide" referred to herein includes naturally occurring, and modified nucleotides linked together by naturally occurring, and non-naturally occurring oligonucleotide linkages. Oligonucleotides are a polynucleotide subset generally comprising a length of 200 bases or fewer. Preferably Oligonucleotides are 10 to 60 bases in length and most preferably 12, 13, 14, 15, 16, 17, 18, 19, or 20 to 40 bases in length. Oligonucleotides are usually single stranded, e.g. for probes; although Oligonucleotides may be double stranded, e.g. for use in the construction of a gene mutant. Oligonucleotides of the invention can be either sense or antisense Oligonucleotides.<br>
[0096] The term "naturally occurring nucleotides" referred to herein includes deoxyribonucleotides and ribonucleotides. The term "modified nucleotides" referred to herein includes nucleotides with modified or substituted sugar groups and the like. The term "oligonucleotide linkages" referred to herein includes Oligonucleotides linkages such as<br><br>
phosphorothioate,	phosphorodithioate,	phosphoroselenoate,	phosphorodiselenoate,<br>
phosphoroanilothioate, phosphoraniladate, phosphoroamidate, and the like. See e.g., LaPlanche et al. Nucl. Acids Res. 14:9081 (1986); Stec et al. J. Am. Chem. Soc. 106:6077 (1984); Stein et al. Nucl. Acids Res. 16:3209 (1988); Zon et al. Anti-Cancer Drug Design 6:539 (1991); Zon et al. Oligonucleotides and Analogues: A Practical Approach, pp. 87-108 (F. Eckstein, Ed., Oxford University Press, Oxford England (1991)); Stec et al. U.S. Patent No. 5,151,510; Uhlmann and Peyman Chemical Reviews 90:543 (1990). An oligonucleotide can include a label for detection, if desired.<br>
[0097] The term "variant" as used herein, is a polypeptide, polynucleotide, or molecule that differs from the recited polypeptide or polynucleotide, but only such that the activity of the protein is not detrimentally altered. There may be variants of epitopes. There may be variants of antibodies. In a preferred embodiment, the ability of a protein variant to bind to the epitope is not detrimentally altered. In one embodiment, the protein variant can bind with 10-500% of the ability of the wild type mAb. For example, the protein variant can bind with 10%, 50%, 110%, 500%, or greater than 500% of the ability of the wild type mAb. In one embodiment, the range of binding abilities between 10-500% is inlcuded. Binding ability may be reflected in many ways, including, but not limited to the ka, kj, or KD of the variart to an epitope. In one preferred embodiment, the epitope is one described in the present specification.<br>
(0098) In one embodiment, variant antibodies can differ from the wild-type sequence by substitution, deletion or addition of five amino acids or fewer. Such variants may generally be identified by modifying one of the disclosed polypeptide sequences, and evaluating the binding properties of the modified polypeptide using, for example, the representative procedures described herein. In another embodiment, polypeptide variants preferably exhibit at least about 70%, more preferably at least about 90% and most preferably at least about 95% identity to the identified polypeptides. Preferrably, the variant differs only in conservative substitutions and/or modifications. Variant proteins include those that are structurally similar and those that are functionally equivalent to the protein structures described in the present specification. In another embodiment, the protein is a variant if it is functionally equivalent to the proteins described in this specification, so long as the paratope of variant is similar to the paratopes described in the specification. In one embodiment, any substance with a shape that is similar to the paratope described in FIG. 11 is a variant. In one embodiment, any substance with a shape that is similar to the paratope described in FIG. 12 is a variant. In one embodiment, any substance that has a shape that is similar to the interaction surface described in FIG. 13 A and 13B is a variant.<br>
[0099] In one embodiment, the antibody is a variant if the nucleic acid sequence can selectively hybridize to wild-type sequence under stringent conditions. In one embodiment, suitable moderately stringent conditions include prewas.hing in a solution of SxSSC; 0.5% SDS, 1.0 mM EDTA (pH 8:0); hybridizing at 50°C-65°C, SxSSC, overnight or, in the event of cross-species<br><br>
homology, at 45°C with O.SxSSC; followed by washing twice at 65°C for 20 minutes with each of 2x, 0.5x and 0.2xSSC containing 0.1% SDS. Such hybridizing DNA sequences are also within the scope of this invention, as are nucleotide sequences that, due to code degeneracy, encode an antibody polypeptide that is encoded by a hybridizing DNA sequence.The term "selectively hybridize" referred to herein means to detectably and specifically bind. Polynucleotides, oligonucleotides and fragments thereof in accordance with the invention selectively hybridize to nucleic acid strands under hybridization and wash conditions that minimize appreciable amounts of detectable binding to nonspecific nucleic acids. High stringency conditions can be used to achieve selective hybridization conditions as known in the art and discussed herein. Generally, the nucleic acid sequence homology between the polynucleotides, oligonucleotides, and fragments of the invention and a nucleic acid sequence of interest will be at least 80%, and more typically with preferably increasing homologies of at least 85%, 90%, 95%, 99%, and 100%. Two amino acid sequences are homologous if there is a partial or complete identity between their sequences. For example, 85% homology means that 85% of the amino acids are identical when the two sequences are aligned for maximum matching. Gaps (in either of the two sequences being matched) are allowed in maximizing matching; gap lengths of 5 or less are preferred with 2 or less being more preferred. Alternatively and preferably, two protein sequences (or polypeptide sequences derived from them of at least 30 amino acids in length) are homologous, as this term is used herein, if they have an alignment score of at more than 5 (in standard deviation units) using the program ALIGN with the mutation data matrix and a gap penalty of 6 or greater. See Dayhoff, M.O., in Atlas of Protein Sequence and Structure, pp. 101-110 (Volume 5, National Biomedical Research Foundation (1972)) and Supplement 2 to this volume, pp. 1-10. The two sequences or parts thereof are more preferably homologous if their amino acids are greater than or equal to 50% identical when optimally aligned using the ALIGN program. The term "corresponds to" is used herein to mean that a polynucleotide sequence is homologous (i.e., is identical, not strictly evolutionarily related) to all or a portion of a reference polynucleotide sequence, or that a polypeptide sequence is identical to a reference polypeptide sequence. In contradistinction,<br>
llif Iciili "(. oiii|&gt;li UK nt.ii y U&gt;" r. ir.nl lii'irm In nir.ni Ihnl Ilir i i&gt;iii|ilcinciilni y '.c|i&gt;|jc&gt;ii'.<br>
to all or a portion of a reference polynucleotide sequence. For illustration, the nucleotide sequence "TATAC" corresponds to a reference sequent e "TATAC" and is complementary to a reference sequence "GTATA".<br>
[0100] The following terms arc used to describe the sequence relationships between two or more polynucleotide or amino acid sequences: "reference sequence", "comparison window", "sequence identity", "percentage of sequence identity", and "substantial identity". A "reference sequence" is a defined sequence used as a basis for a sequence comparison; a reference sequence may be a subset of a larger sequence, for example, as a segment of a full-length cDNA or gene sequence given in a sequence listing or may comprise a complete cDNA or gene sequence. Generally, a reference sequence is at least 18 nucleotides or 6 amino acids in length, frequently at<br><br>
least 24 nucleotides or 8 amino acids in length, and often at least 48 nucleotides or 16 amino acids in length. Since two polynucleotides or amino acid sequences may each (1) comprise a sequence (i.e., a portion of the complete polynucleotide or amino acid sequence) that is similar between the two molecules,   and  (2)  may   further  comprise   a  sequence   that   is  divergent between  the  two polynucleotides or amino acid sequences, sequence comparisons between two (or more) molecules are typically performed by comparing sequences of the two molecules over a "comparison window" to identify and compare local regions of sequence similarity.   A "comparison window", as used herein, refers to a conceptual segment of at least 18 contiguous nucleotide positions or 6 amino acids wherein a polynucleotide sequence or amino acid sequence may be compared to a reference sequence of at least 18 contiguous nucleotides or 6 amino acid sequences and wherein the portion of the polynucleotide sequence in the comparison window may comprise additions, deletions, substitutions, and the like (i.e., gaps) of 20 percent or less as compared to the reference sequence (which does not comprise additions or deletions) for optimal alignment of the two sequences. Optimal alignment of sequences for aligning a comparison window may be conducted by the local homology algorithm of Smith and Waterman Adv. Appl. Math. 2:482 (1981), by the homology alignment algorithm of Needleman and Wunsch J. Moi Biol. 48:443 (1970), by the search for similarity method of Pearson and Lipman Proc. Nail. Acad. Sci. (U.S.A.) 85:2444 (1988), by computerized implementations of these algorithms (GAP, BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package Release 7.0, (Genetics Computer Group, 575 Science Dr., Madison, Wis.), Geneworks, or MacVector software packages), or by inspection, and the best alignment (i.e., resulting in the highest percentage of homology over the comparison window) generated by the various methods is selected. <br>
[0101]         The term "sequence identity" means that two polynucleotide or amino acid sequences are identical (i.e., on a nuc)eotide-by-nucleotide or residue-by-residue basis) over the comparison window.   The term "percentage of sequence identity" is calculated by comparing two optimally aligned sequences over the window of comparison, determining the number of positions at which the identical nucleic acid base (e.g., A, T, C, G, U, or I) or residue occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number ' of positions in the comparison window (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity. The terms "substantial identity" as used herein denotes a characteristic of a polynucleotide or ammo acid sequence, wherein the polynucleotide or amino acid comprises a sequence that has at least 85 percent sequence identity, preferably at least 90 to 95 percent sequence identity, more usually at least 99 percent sequence identity as compared to a reference sequence over a comparison window of at least 18 nucleotide (6 amino acid) positions, frequently over a window of at least 24-48 nucleotide (8-16 amino acid) positions, wherein the percentage of sequence identity is calculated by comparing the reference sequence to the sequence which may include deletions or additions which total 20 percent or less of the reference sequence over the comparison window. The reference sequence may be a subset of a larger sequence. Amino<br><br>
acids or nucleic acids with substantial identity to the wild-type protein or nucleic acid are examples of variants of the wild-type protein or nucleic acid.<br>
[0102] As used herein, the twenty conventional amino acids and their abbreviations follow conventional usage. See Immunology - A Synthesis (2nt Edition, E.S. Golub and D.R. Gren, Eds., Sinaucr Associates, Sunderland, Mass. (1991)). Stereoisomers (e.g., D-amino acids) of the twenty conventional amino acids, unnatural amino acids such as a-, a-disubstituted amino acids, N-alkyl amino acids, lactic acid, and other unconventional amino acids may also be suitable components for polypcptides of the present invention. Examples of unconventional amino acids include: 4-hydroxyproline, y-carboxyglutamate, E-N,N,N-trimethyllysine, e-N-acetyllysine, O-phosphoserine, N-acetylserine, N-formylmethionine, 3-methylhistidine, 5-hydroxylysine, a-N-methylarginine, and other similar amino acids and imino acids (e.g., 4-hydroxyproline). In the polypeptide notation used herein, the left-hand direction is the amino terminal direction and the right-hand direction is the carboxy-terminal direction, in accordance with standard usage and convention.<br>
[0103] Similarly, unless specified otherwise, the left-hand end of single-stranded polynucleptide sequences is the 5' end; the left-hand direction of double-stranded polynucleotide sequences is referred to as the 5' direction. The direction of 5' to 3' addition of nascent RNA transcripts is referred to as the transcription direction; sequence regions on the DNA strand having the same sequence as the RNA and which are 5' to the 5' end of the RNA transcript are referred to as "upstream sequences"; sequence regions on the DNA strand having the same sequence as the RNA and which are 3' to the 3' end of the RNA transcript are referred to as "downstream sequences".<br>
|0104) As applied to polypeptides, the term "substantial identity" means that two pcptide sequences, when optimally aligned, such as by the programs GAP or BESTFIT using default gap weights, share at least 80 percent sequence identity, preferably at least 90 percent sequence identity, more preferably at least 95 percent sequence identity, and most preferably at least 99 percent sequence identity. Preferably, residue positions which are not identical differ by conservative amino acid substitutions. Conservative amino acid substitutions refer to the interchangeability of residues having similar side chains. For example, a group of amino acids having aliphatic side chains is glycine, alanine, valine, leucine, and isoleucine; a group of amino acids having aliphatic-hydroxyl side chains is serine and threonine; a group of amino acids having amide-containing side chains is asparagine and glutaminc; a group of amino acids having aromatic side chains is phenylalanine, tyrosine, and tryptophan; a group of amino acids having basic side chains is lysine, arginine, and histidine; and a group of amino acids having sulfur-containing side chains is cysteine and methionine. Preferred conservative amino acids substitution groups are: valine-leucine-isoleucine, phenylalanine-tyrosine, lysine-arginine, alanine-valine, glutamic-aspartic, and asparagine-glutamine. Polypeptides with substantial identity can be variants.<br><br>
[0105] Variant proteins also include proteins with minor variations. As discussed herein, minor variations in the amino acid sequences of antibodies or immunoglobulin molecules are contemplated as being encompassed by the present invention, providing that the variations in the amino acid sequence maintain at least 75%, more preferably at least 80%, 90%, 95%, and most preferably 99%. In particular, conservative amino acid replacements are contemplated.<br>
|0106] Conservative replacements are those that take place within a family of amino acids that are related in their side chains. Genetically encoded amino acids are generally divided into families: (1) acidic=aspartate, glutamate; (2) basic=lysine, arginine, histidine; (3) non-polar=alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan; and (4) uncharged polar=glycine, asparagine, glutamme, cysteine, serine, threonine, tyrosine. More preferred families are: serine and threonine are aliphatic-hydroxy family; asparagine and glutamine are an amide-containing family; alanine, valine, leucine and isoleucine are an aliphatic family; and phenylalanine, tryptophan, and tyrosine are an aromatic family. For example, it is reasonable to expect that an isolated replacement of a leucine with an isoleucine or valine, an aspartate with a glutamate, a threonine with a serine, or a similar replacement of an amino acid with a structurally related amino acid will not have a major effect on the binding or properties of the resulting molecule, especially if the replacement does not involve an amino acid within a framework site. Whether an amino acid change results in a functional peptidc can readily be determined by assaying the specific activity of the polypeptide derivative, Assays are described in detail herein. Fragments or analogs of antibodies or immunoglobulin molecules can be readily prepared by those of ordinary skill in the art. Preferred amino- and carboxy-lermini of fragments or analogs occur near boundaries of functional domains. Structural and functional domains can be identified by comparison of the nucleotide and/or amino acid sequence data to public or proprietary sequence databases. Preferably, computerized comparison methods are used to identify sequence motifs or predicted protein conformation domains that occur in other proteins of known structure and/or function. Methods to identify protein sequences that fold into a known three -dimensional structure are known. Bowie et al. Science 253:164 (1991). Thus, the foregoing examples demonstrate that those of skill in the art can recognize sequence motifs and structural conformations that may be used to define structural and functional domains in accordance with the antibodies described herein.<br>
[0107] Preferred ammo acid substitutions are those which: (1) reduce susceptibility to proteolysis, (2) reduce susceptibility to oxidation, (3) alter binding affinity for forming protein complexes, (4) alter binding affinities, and (4) confer or modify other physicochemical or functional properties of such analogs. Analogs can include various muteins of a sequence other than the naturally-occurring peptidc sequence. For example, single or multiple amino acid substitutions (preferably conservative amino acid substitutions) may be made in the naturally-occurring sequence (preferably in the portion of the polypeptide outside the domain(s) forming intermolecular contacts. A conservative amino acid substitution should not substantially change the structural characteristics<br><br>
of the parent sequence (e.g., a replacement amino acid should not tend to break a helix that occurs in the parent sequence, or disrupt other types of secondary structure that characterizes the parent sequence). Examples of art-recognized polypeptide secondary and tertiary structures are described in Proteins, Structures and Molecular Principles (Creighton, Ed., W. H. Freeman and Company, New York (1984)); Introduction to Protein Structure (C. Branden and J. Tooze, eds., Garland Publishing, New York, N.Y. (1991)); and Thornton et at. Nature 354:105 (1991).<br>
(0108) The term "polypeptide fragment" as used herein refers to a polypeptide that has an ammo-terminal and/or carboxy-terminal deletion, but where the remaining amino acid sequence is identical to the corresponding positions in the naturally-occurring sequence deduced, for example, from a full-length cDNA sequence. Fragments typically are at least 5, 6, 8 or 10 amino acids long, preferably at least 14 amino acids long, more preferably at least 20 amino acids long, usually at least 50 amino acids long, and even more preferably at least 70 urnino acids long. The term "analog" as used herein refers to polypeptides which are comprised of a segment of at least 25 amino acids that has substantial identity to a portion of a deduced amino acid sequence. Analogs typically are at least 20 amino acids long, preferably at least 50 amino acids long or longer, and can often be as long as a full-length naturally-occurring polypeptide. Both fragments and analogs are forms of variants<br>
[0109] Peptide analogs are commonly used in the pharmaceutical industry as non-peptide drugs with properties analogous to those of the template peptide. These types of non-peptide compound are termed "peptide mimetics" or "peptidomimetics". Fauchere, J. Adv. Drug Res. 15:29 (1986); Veber and Freidingcr TINS p.392 (1985); and Evans et al. J. Med. Chem. 30:1229 (1987). Such compounds are often developed with the aid of computerized molecular modeling. Peptide mimetics that are structurally similar to therapeutically useful peptides may be used to produce an equivalent therapeutic or prophylactic effect. Generally, peptidomimetics are structurally similar to a paradigm polypeptide (i.e., a polypeptide that has a biochemical property or pharmacological activity), such as human antibody, but have one or more peptide linkages optionally replaced by a linkage selected from the group consisting of: -CHjNH--, --CH2S--, --CH2-CH2--, -CH=CH--(cis and trans), ~COCH2~, --CH(OH)CH2--, and -CH2SO--, by methods well known in the art. Systematic substitution of one or more amino acids of a consensus sequence with a D-amino acid of the same type (e.g., D-lysine in place of L-lysine) may be used to generate more stable peptides. In addition, constrained peptides comprising a consensus sequence or a substantially identical consensus sequence variation may be generated by methods known in the art (Rizo and Gierasch Ann. Rev. Biochem. 61:387 (1992)); for example, by adding internal cysteine residues capable of forming intramolecular disulfide bridges which cyclize the peptide. Peptide mimetics and peptidomimetics ' are both forms of variants.<br>
|0110] "Antibody" or "antibody peptide(s)" refer to an intact antibody, or a binding fragment thereof that competes with the intact antibody for specific binding. Binding fragments are produced by recombinant DNA techniques, or by enzymatic or chemical cleavage of intact<br><br>
antibodies. Binding fragments include Fab, Fab', F(ab')2, Fv, and single-chain antibodies. An antibody other than a "bispecific" or "bifunctional" antibody is understood to have each-of its binding sites identical. An antibody substantially inhibits adhesion of a receptor to a counterreceptor when an excess of antibody reduces the quantity of receptor bound to counterreceptor by at least about 20%, 40%, 60% or 80%, and more usually greater than about 85% (as measured in an in vitro competitive binding assay).<br>
[0111)        The term "epitope" includes any protein determinant capable of specific binding<br>
to an immunoglobulin or T-cell receptor or otherwise interacting with a molecule.    Epitopic<br>
determinants generally consist of chemically active surface groupings of molecules such as amino<br>
acids or carbohydrate or sugar side chains and generally have specific three-dimensional structural<br>
characteristics,  as  well  as specific charge characteristics.     An epitope may be "linear" or<br>
"conformational."  In a linear epitope, all of the points of interaction between the protein and the<br>
interacting molecule (such as an antibody) occur linearally along the primary amino acid sequence of<br>
the protein.  In a conformational epitope, the points of interaction occur across amino acid residues<br>
on the protein that are separated from one another.   AJI antibody is said to specifically bind an<br>
antigen when the dissociation constant is £1 |oM, preferably 
and even more preferably 
to generate antibodies to that epitope, e.g., using the techniques described in the present invention.<br>
Alternatively, during the discovery process, the generation and characterization of antibodies may<br>
elucidate information about desirable epitopes.    From this information, it is then possible to<br>
competitively screen antibodies for binding to the same epitope. An approach to achieve this is to<br>
conduct cross-competition studies to find antibodies that competively bind with one another, e.g., the<br>
antibodies compete for binding to the antigen.  A high throughput process for "binning" antibodies<br>
based upon their cross-competition is described in International Patent Application No. WO<br>
03/48731. As will be appreciated by one of skill in the art, practically anything to which an antibody<br>
can spccificially bind could be an epitope.   An epitope can comprises those residues to which the<br>
antibody binds.   In one embodiment, the epitope is the EGFRvIlI epitope.   In a more preferred<br>
embodiment, the epitope is that described in Example 4 of this specification. In one embodiment, the<br>
epitope is the epitope described in Example 14.   In one embodiment, the epitope comprises the<br>
sequence LEEKKGNYVVTD (SEQ ED NO: 59).   In one embodiment, the epitope comprises the<br>
sequence EEKKGNYVVT (SEQ ED NO: 57).    In one embodiment, the epitope comprises the<br>
sequence EKNY (SEQ ID NO: 60).    In on.; embodiment, the epitope comprises the sequence<br>
EEKGN (SEQ ED NO: 61).   One of skill in the art will appreciate that these need not be actually<br>
assembled in this order on a single peptide, rather, these are the residues that form the eptiope which<br>
interacts with the paratope.   As will be appreciated by one of skill in the art, the space that is<br>
occupied by a residue or side chain that creates the shape of a molecule helps to determine what an<br><br>
epitope is. Likewise, any functional groups associated with the epitopc, van der Waals interactions, degree of mobility of side chains, etc. can all determine what an epitope actually is. Thus an epitope may also include energetic interactions.<br>
[0112] The term "paratope" is meant to describe the general structure of a binding region that determines binding to an epitope. This structure influences whether or not and in what manner the binding region might bind to an epitope. Paratope can refer to an antigenic site of an antibody that is responsible for an antibody or fragment thereof, to bind to an antigenic determinant. Paratope also refers to the idiotope of the antibody, and the complementary determining region (CDR) region that binds to the epitope. In one embodiment, the paratope is the region of the antibody that is LI 10, L2 30, L3 50, HI 20, H2 40, and H3 60 in FIG. 11. In one embodiment, the paratope is the region of the antibody that comprises the CDR sequences in Example 16 for LI, L2, L3, HI, H2, and H3. In one embodiment, the paratope is the region of the antibody that is LI 110, L2 130, L3 150, HI 120, H2 140, and H3 160 in FIG. 12. In one embodiment, the paratope is the region of the antibody that comprises the CDR sequences in Example 18 for LI, L2, L3, HI, H2, and H3. In one embodiment, the paratope comprises the sequences listed in Example 18. In one embodiment, the paratope comprises the residues; that interact with the epitope, as shown in FIG. 13A and FIG. 13B. The solid black structure is the peptide structure. In one embodiment, the paratope comprises residue Tyrl72Arg of the 13.1.2 mAb. In one embodiment, the paratope of the 13.1.2 mAb comprises at least one residue selected from the group consisting of: Tyr 172Arg, ArglOlGlu, Leu99Asn, Leu99His, ArglOlAsp, Leu217Gln, Leu99Thr, Leu217Asn, ArglOlGln, and Asn35Gly. As will be appreciated by one of skill in the art, the paratope of any antibody, or variant thereof, can be determined in the manner set forth by the present application. Residues are considered "important" if they are predicted to contribute 10% of the binding energy. In one embodiment, residues are considered "important" if they are predicted to contribute 2% of the binding energy. In one embodiment, residues are considered "important" if they are predicted to contribute 50% of the binding energy. In one embodiment, residues are considered "important" if they are predicted to interact with the surface of the epitope, or the surface of the paratope. In one embodiment, residues are considered "important" if changing the residue results in a loss in binding.<br>
(0113) The terms "specifically" or "preferentially" binds to, or similar phrases are not meant to denote that the antibody exclusively binds to that epitope. Rather, what is meant is that the antibody, or variant thereof, can bind to that epitope, to a higher degree than the antibody binds to at least one other substance to which the antibody is exposed to. In one embodiment, the specifically binding antibody will bind to the EGFRvllI protein with an affinity greater than (more tightly, or lower KD) it will to the EGFR protein. For example, the specifically binding antibody will bind more tightly by at least a minimal increase to 1, 1-2, 2-5, 5-10, 10-20, 20-30, 30-50, 50-70, 70-90, 90-120, 120-150, 150-200, 200-300, 300-500, 500-1000 percent or more.<br><br>
[0114] The shorthand of amino acid, number, amino acid, e.g., Leu217Gln, denotes a mutation at the numbered amino acid, from the first amino acid, to the second amino acid. Thus, Tyrl72Arg would mean that while the wild type protein has a tyrosine at position 172, the mutant has an argimnc at position 172.<br>
[0115] The term "agent" is used herein to denote a chemical compound, a mixture of chemical compounds, a biological macromolecule, or an extract made from biological materials.<br>
|0116] "Mammal" when used herein refers to any animal that is considered a mammal. Preferably, the mammal is human.<br>
|0117] Digestion of antibodies with the enzyme, papain, results in two identical antigen-binding fragments, known also as "Fab" fragments, and a "Fc" fragment, having no antigen-binding activity but having the ability to crystallize. Digestion of antibodies with the enzyme, pepsin, results in the a F(ab')2 fragment in which the two arms of the antibody molecule remain linked and comprise two-antigen binding sites. The F(ab')2 fragment has the ability to crosslink antigen.<br>
|0118] "Fv" when used herein refers to the minimum fragment of an antibody that retains both antigen-recognition and antigen-binding sites, These fragments can also be considered variants of the antibody.<br>
[0119] "Fab" when used herein refers to a fragment of an antibody which comprises the constant domain of the light chain and the CHI domain of the heavy chain.<br>
[0120)        The term "mAb" refers to monoclonal antibody.<br>
[0121) The description of XenoMax method generated antibody sequences is coded as follows: "AB"-referring to antibody, "EGFRvU.l"-referring to antibody's binding specificity, "X" referring to XenoMouse mouse derived, "Gl"-referring to IgGl isotype or "G2" referring to lgG2 isotype, the last three digits refer to the single cell number from which the antibody was derived, for example: AB- EGFRvIH -XG1-095 would be an antibody with binding specificity to EGFRvHI from XenoMouse mouse of a IgGl isotype and cell number 95.<br>
[0122] The term "SC" refers to single cell and a particular XenoMax method derived antibody may be referred to as SC followed by three digits, or just three digits, referring to the single cell number from which the antibody was derived herein.<br>
[0123] The description of hybridoma derived antibody sequences is coded as follows: "AB"-referring to antibody, "EGFRvIIT'-refers to the antibody's binding specificity, "X" refers to XenoMouse mouse derived, "Gl"-refers to IgGl isotype or "G2" refers to lgG2 isotype, "K" refers to kappa, "L* refers to lambda. The last three digits referring to the clone from which the antibody was derived, for example: AB-EGFRvIII-XGlK-13.1.2<br>
[0124] "Label" or "labeled" as used herein refers to the addition of a detectable moiety to a polypeptide, for example, a radiolabel, fluorescent label, enzymatic label chemiluminescent labeled or a biotinyl group. Radioisotopes or radionuclides may include 3H, I4C, 1SN, 3SS, 90Y, wTc, 1MIn, I2S1, ml, fluorescent labels may include rhodamine, lanthanide phosphors or F1TC and<br><br>
enzymatic   labels   may   include  horseradish   peroxidase,   (i-galactosidase,   luciferase,   alkaline phosphatase.<br>
[0125] The term "pharmaceutical agent or drug" as used herein refers to a chemical compound or composition capable of inducing a desired therapeutic effect when properly administered to a patient. Other chemistry terms herein are used according to conventional usage in the art, as exemplified by The McGraw-Hill Dictionary of Chemical Terms (Parker, S., Ed., McGraw-Hill, San Francisco (1985)).<br>
[0126] As used herein, "substantially pure" means an object species is the predominant species present (i.e., on a molar basis it is more abundant than any other individual species in the composition), and preferably a substantially purified fraction is a composition wherein the object species comprises at least about 50 percent (on a molar basis) of all macromolecular species present. Generally, a substantially pure composition will comprise more than about 80 percent of all macromolecular species present in the composition, more preferably more than about 85%, 90%, 95%, 99%, and 99.9%, Most preferably, the object species is purified to essential homogeneity (contaminant species cannot be detected in the composition by conventional detection methods) wherein the composition consists essentially of a single macromolecular species.<br>
[0127)        The term "patient" includes human and veterinary subjects.<br><br>
[0128|        The term "SLAM* Technology" refers to the "Selected Lymphocyte Antibody<br>
Method" (Babcook et al., Proc. Nail. Acad. Sci. USA, i93:7843-7848 (1996), and Schrader, US Patent No. 5,627,052.<br>
[0129]         The   term   "XenoMax™"   refers   to   the   use   of SLAM  Technology   with XenoMouse* mice (as described below). Antibody Structure<br>
[0130] The basic antibody structural unit is known to comprise a tetramer. Each tetramer is composed of two identical pairs of polypeptide chains, each pair having one "light" (about 25 kDa) and one "heavy" chain (about 50-70 kDa). The amino-terminal portion of each chain includes a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. The carboxy-terminal portion of each chain defines a constant region primarily responsible for effector function. Human light chains are classified as kappa and lambda light chains. Heavy chains are classified as mu, delta, gamma, alpha, or epsilon, and define the antibody's isotype as IgM, IgD, IgG, IgA, and IgE, respectively. Within light and heavy chains, the variable and constant regions are joined by a "J" region of about 12 or more amino acids, with the heavy chain also including a "D" region of about 10 more amino acids. See generally, Fundamental Immunology Ch. 7 (Paul, W., ed., 2nd ed. Raven Press, N.Y. (1989))). The variable regions of each light/heavy chain pair form the antibody binding site.<br>
[0131] Thus, an intact antibody has two binding sites. Except in bifunctional or bispecific antibodies, the two binding sites are the same.<br><br>
[0132]<br>
 The chains all exhibit the same general structure of relatively conserved framework regions (FR) joined by three hyper variable regions, also called complementarity determining regions or CDRs. The CDRs from the two chains of each pair are aligned by the framework regions, enabling binding to a specific epitope. From N-terminal to C-terrninal, both light and heavy chains comprise the domains FR1, CDR1, FR2, CDR2, FR3, CDR3 and FR4. The assignment of amino acids to each domain is in accordance with the definitions of Kabat Sequences of Proteins of Immunological Interest (National Institutes of Health, Bethesda, Md. (1987 and 1991)), or Chothia &amp; Lesk J. Mol. Biol. 196:901-917 (1987); Chothia et al. Nature 342:878-883 (1989).<br>
[0133] A bispecific or bifunctional antibody is an artificial hybrid antibody having two different heavy/light chain pairs and two different binding sites. Bispecific antibodies can be produced by a variety of methods including fusion of hybridomas or linking of Fab' fragments. See, e.g., Songsivilai &amp; Lachmann Clin. Exp. Immunol. 79: 315-321 (1990), Kostelny et al. /. Immunol. 148:1547-1553 (1992). Production of bispecific antibodies can be a relatively labor intensive process compared with production of conventional antibodies and yields and degree of purity are generally lower for bispecific antibodies. Bispecific antibodies do not exist in the form of fragments having a single binding site (e.g., Fab, Fab', and Fv).<br>
[0134] In addition to the general structural aspects of antibodies, the more specific interaction between the paratope and the epitope may be examined through structural approaches. In one embodiment, the structure of the CDRs form a paratope, through which an antibody is able to bind to an epitope. The structure of such a paratope may be determined in a number of ways. Traditional structural examination approaches may be used, such as NMR or x-ray crystalography. These approaches may examine the structure of the paratope alone, or while it is bound to the epitope. Alternatively, molecular models may be generated in silica. A structure can be generated through homology modeling, aided with a commercial package, such as Insightn modeling package from Accelrys (San Diego, CA). Briefly, one can use the sequence of the antibody to be examined to search against a database of proteins of known structures, such as the Protein Data Bank. After one identifies homologous proteins with known structures, these homologous proteins are used as modeling templates. Each of the possible templates can be aligned, thus producing structure based sequence alignments amoung the templates. The sequence of the antibody with the unknown structure can then be aligned with these templates to generate a molecular model for the antibody with the unknown structure. As will be appreciated by one of skill in the art, there are many alternative methods for generating such structures in silica, any of which may be used. For instance, a process similar to the one described in Hardman et al., issued U.S. Pat. No. 5,958,708 employing QUANTA (Polygen Corp., Waltham, Mass.) and CHARM (Brooks, B. R., Bruccoleri, R. E., Olafson, B. D., States, D. J., Swaminathan, S. and Karplus, M., 1983, J. Comp. Chem,. 4:187) may be used.<br><br>
[0135] Not only is the shape of the paralope important in determining whether and how well a possible paratope will bind to an epitope, but the interaction itself, between the epitope and the paratope is a source of great information in the design of variant antibodies. As appreciated by one of skill in the art, there are a variety of ways in which this interaction can be studied. One way is to use the structural model generated, perhaps as described above, and then to use a program such as InsightU (Accelrys, San Diego, CA), which has a docking module, which, among other things, is capable of performing a Monte Carlo search on the conformational and orientational spaces between the paratope and its epitope. The result is that one is able to estimate where and how the epitope interacts with the paratope. In one embodiment, only a fragment, or variant, of the epitope is used to assist in determining the relevant interactions. In one embodiment, the entire epitope is used in the modeling of the interaction between the paratope and the epitope. As will be appreciated by one of skill in the art, these two different approaches have different advantages and disadvantages. For instance, using only a fragment of the epitope allows for a more detailed examination of the possible variations of each side chain, without taking huge amounts of time. On the other hand, by using only a fragment of the epitope, or simply the epitope instead of the entire protein, it is possible that the characteristics of the epitope fragment may not be the same as the characteristics for the whole epitope, thus possibly increasing the risk of being mislead during the computational modeling. In one embodiment, both approaches are used to a limited extent, in order to cross check the results. In a preferred embodiment, if a variant of an epitope is used, it will be optimized so that the variant of the epitope comprises the most important residues of the epitope. The identity of the most important residues can be determined in any number of ways, for instance as described in Examples 4 and 14 of the present specification.<br>
[0136] Through the use of these generated structures, one is able to determine which residues are the most important in the interaction between the epitope and the paratope. Thus, in one embodiment, one is able to read)ly select which residues to change in order to alter the binding characteristics of the antibody. For instance, it may be apparent from the docking models that the side chains of certain residues in the paratope may sterically hinder the binding of the epitope, thus altering these residues to residues with smaller side chains may be beneficial. One can determine this in many ways. For example, one may simply look at the two models and esitmate interactions based on functional groups and proximety. Alternatively, one may perform repeated pairings of epitope and paratope, as described above, in order to obtain more favorable energy interactions. One can also determine these interactions for a variety of variants of the antibody to determine alternative ways in which the antibody may bind to the epitope. One can also combine the various models to determine how one should alter the structure of the antibodies in order to obtain an antibody with the particular characteristics that are desired.<br>
[0137]         The models determined above can be tested through various techniques.   For example, the interaction energy can determined with the programs discussed above in order to<br><br>
determine which of the variants to further examine. Also, coulumbic and van der Waals interactions are used to determine the interaction energies of the epitope and the variant paratopes. Also site directed mutagenesis is used to see if predicted changes in antibody structure actually result in the desired changes in binding characteristics. Alternatively, changes may be made to the epitope to verify that the models are correct or to determine general binding themes that may be occurring between the paratope and the epitope.<br>
[0138) The above methods for modeling structures can be used to determine what changes in protein structure will result in particular desired characteristics of an antibody. These methods can be used to determine what changes in protein structure will not result in the desired characteristics.<br>
[0139] As will be appreciated by one of skill in the art, while these models will provide the guidance necessary to make the antibodies and variants thereof of the present embodiments, it may still be desirable to perform routine testing of the in silico models, perhaps through in vitro studies. In addition, as will be apparent to one of skill in the art, any modification may also have additional side effects on the activity of the antibody. For instance, while any alteration predicted to result in greater binding, may induce greater binding, it may also cause other structural changes which might reduce or alter the activity of the antibody. The determination of whether or not this is the case is routine in the art and can be achieved in many ways. For Example, the activity can be tested through an EL1SA test, as in Example 21. Alternatively, the samples can be tested through the use of a surface plasmon resonance device. Antibodies Binding, and Variant Antibodies for Superior Binding<br>
[0140] In one embodiment, the models described above are used to increase the binding ability of the antibody to its epitope. The antibody can bind to the epitope more readily, and thus have a higher association constant (ka). Alternatively, the antibody may dissociate from the epitope slower, and thus have a lower dissociation constant (k,j), or the KD of the epitope-paratope interaction can be smaller in value, thus making the extent of the binding between the epitope and paratope higher.<br>
|0141] In some embodiments, the variant antibodies are designed to bind with the opposite characteristics. That is, the antibodies do not bind as tightly or perhaps as quickly.<br>
[0142] In other embodiments, the variant antibodies are not different in their KD from the wild-type antibodies, but the variant antibodies are more specific for a particular epitope. This may mean that the paratopes of the designed antibodies have a lower risk of binding to other epitopes. The antibodies can have other characteristics that have been altered. For example, a variant may be more immune to nonspecific antibody binding or may stay solvated in solution even when the antibody is present in high concentrations. Such a variant may be present in the discussed antibodies. For instance, while the higher concentrations of some variant antibodies examined below resulted in slower binding components in Bia^ore experiments, for instance 13.1.2 mAb, certain<br><br>
variants did not exhibit this slower component, even at the same concentrations, L217N-2.1, for example.<br>
|0143] The variants predicted by the models determined above can be created and then tested to determine if they actually bind with the desired characteristics. Mutants with a greater total interaction energy with the epitope can be selected for further testing. The interaction energy can be determined in a number of ways, one of which is described above.<br>
[0144] These variants can be tested in a number of ways. Exemplary options include and are not limited to KinExA (e.g., Lackie, Issued Pat. No. 5,372,783, Dec 13, 1994) (Sapidyne Instruments Inc., ID, Boise), surface plasmon resonance (SPR)(e.g., BIACORE™ Biacore, Inc., Pistcataway, N.J.), stopped-flow fluorescence, resonant mirror, and fluorescence polarization. Many of these options are able to not only record the data, but also provide ready means for fitting the data to various theoretical curves and thus determine the k,, kd, and KD, as well as other properties. It is important to note that the fitting of these curves to the resulting data is not without the possibility for some variation. Because of this, the relevant association, dissociation, and equilibrium constants can be looked at, not only through these curve fitting mechanisms, but also in direct comparison with each other, and in light of the knowledge of one of skill in the art. Human Antibodies and Humanization of Antibodies<br>
[0145] Human antibodies avoid some of the problems associated with antibodies that possess murine or rat variable and/or constant regions. The presence of such murine or rat derived proteins can lead to the rapid clearance of the antibodies or can lead to the generation of an immune response against the antibody by a patient. In order to avoid the utilization of murine or rat derived antibodies, fully human antibodies can be generated through the introduction of human antibody function into a rodent so that the rodent produces fully human antibodies.<br>
|0146| The ability to clone and reconstruct megabase-sized human loci in YACs and to introduce them into the mouse germline provides a powerful approach to elucidating the functional components of very large or crudely mapped loci as well as generating useful models of human disease. Furthermore, the utilization of such technology for substitution of mouse loci with their human equivalents could provide unique insights into the expression and regulation of human gene products during development, their communication with other systems, and their involvement in disease induction and progression.<br>
|0147] An important practical application of such a strategy is the "humanization" of the mouse humoral immune system. Introduction of human immunoglobulin (Ig) loci into mice in which the endogenous Ig genes have been inactivated offers the opportunity to study the mechanisms underlying programmed expression and assembly of antibodies as well as their role in B-cell development. Furthermore, such a strategy could provide an ideal source for production of fully human monoclonal antibodies (mAbs)-an important milestone towards fulfilling the promise of antibody therapy in human disease. Fully human antibodies are expected to minimize the<br><br>
immunogenic and allergic responses intrinsic to mouse or mouse-derivatized mAbs and thus to increase the efficacy and safety of the administered antibodies. The use of fully human antibodies can be expected to provide a substantial advantage in the treatment of chronic and recurring human diseases, such as inflammation, autoimmunity, and cancer, which require repeated antibody administrations.<br>
[0148] One approach towards this goal was to engineer mouse strains deficient in mouse antibody production with large fragments of the human Ig loci in anticipation that such mice would produce a large repertoire of human antibodies in the absence of mouse antibodies. Large human Ig fragments would preserve the large variable gene diversity as well as the proper regulation of antibody production and expression. By exploiting the mouse machinery for antibody diversification and selection and the lack of immunological tolerance to human proteins, the reproduced human antibody repertoire in these mouse strains should yield high affinity antibodies against any antigen of interest, including human antigens. Using the hybridoma technology, antigen-specific human mAbs with the desired specificity could be readily produced and selected. This general strategy was demonstrated in connection with our generation of the first XenoMouse mouse strains, as published in 1994. (See Green et al. Nature Genetics 7:13-21 (1994)) The XenoMouse strains were engineered with yeast artificial chromosomes (YACs) containing 245 kb and 190 kb-sized germline configuration fragments of the human heavy chain locus and kappa light chain locus, respectively, which contained core variable and constant region sequences. Id. The human Ig containing YACs proved to be compatible with the mouse system for both rearrangement and expression of antibodies and were capable of substituting for the inactivated mouse Ig genes. This was demonstrated by their ability to induce B-cell development, to produce an adult-like human repertoire of fully human antibodies, and to generate antigen-specific human mAbs. These results also suggested that introduction of larger portions of the human Ig loci containing greater numbers of V genes, additional regulatory elements, and human Ig constant regions might recapitulate substantially the full repertoire that is characteristic of the human humoral response to infection and immunization. The work of Green et al. was recently extended to the introduction of greater than approximately 80% of the human antibody repertoire through introduction of megabase sized, germline configuration YAC fragments of the human heavy chain loci and kappa light chain loci, respectively. See Mendez et al. Nature Genetics 15:146-156 (1997) and U.S. patent application Ser. No. 08/759,620, filed Dec. 3, 1996.<br>
[0149] The production of the XenoMouse mice is further discussed and delineated in U.S. Patent Application Serial Nos. 07/466,003, filed January 12, 1990, 07/610,515, filed November 8, 1990, 07/919,297, filed July 24, 1992, 07/922,649, filed July 30, 1992, filed 08/031,801, filed March 15,1993, 08/112,848, filed August 27, 1993, 08/234,145, filed April 28, 1994, 08/376,279, filed January 20, 1995, 08/430, 938, April 27, 1995, 08/464,584, filed June 5, 1995, 08/464,582, filed June 5, 1995, 08/463,191, filed June 5, 1995, 08/462,837, filed June 5, 1995, 08/486,853, filed June<br><br>
5, 1995, 08/486,857, filed June 5, 1995, 08/486,859, filed June 5, 1995, 08/462,513, filed June 5, 1995, 08/724,752, filed October 2, 1996, and 08/759,620, filed December 3, 1996 and U.S. Patent Nos. 6,162,963, 6,150,584, 6,114,598, 6,075,181, and 5,939,598 and Japanese Patent Nos. 3 068 180 B2, 3 068 506 B2, and 3 068 507 B2. See also Mendez et al. Nature Genetics 15:146-156 (1997) and Green and Jakobovits J. Exp. Med. 188:483-495 (1998). See also European Patent No., EP 0 463 151 Bl, grant published June 12, 1996, International Patent Application No., WO 94/02602, published February 3, 1994, International Patent Application No., WO 96/34096, published October 31, 1996, WO 98/24893, published June 11, 1998, WO 00/76310, published December 21, 2000, WO 03/47336.<br>
(0150) In an alternative approach, others, including GenPharm International, Inc., have utilized a "minilocus" approach. In the minilocus approach, an exogenous Ig locus is mimicked through the inclusion of pieces (individual genes) from the Ig locus. Thus, one or more VH genes, one or more DH genes, one or more JH genes, a mu constant region, and a second constant region (preferably a gamma constant region) are formed into a construct for insertion into an animal. This approach is described in U.S. Patent No. 5,545,807 to Surani et al. and U.S. Patent Nos. 5,545,806, 5,625,825, 5,625,126, 5,633,425, 5,661,016, 5,770,429, 5,789,650, 5,814,318, 5,877,397, 5,874,299, and 6,255,458 each to Lonberg and Kay, U.S. Patent No. 5,591,669 and 6,023.010 to Krimpenfort and Berns, U.S. Patent Nos. 5,612,205, 5,721,367, and 5,789,215 to Bems et al., and U.S. Patent No. 5,643,763 to Choi and Dunn, and GenPharm International U.S. Patent Application Serial Nos. 07/574,748, filed August 29, 1990, 07/575,962, filed August 31, 1990, 07/810,279, filed December 17, 1991, 07/853,408, filed March 18, 1992, 07/904,068, filed June 23, 1992, 07/990,860, filed December 16, 1992, 08/053,131, filed April 26, 1993, 08/096,762, filed July 22, 1993, 08/155,301, filed November 18, 1993, 08/161,739, filed December 3, 1993, 08/165,699, filed December 10, 1993, 08/209,741, filed March 9, 1994. See also European Patent No. 0 546 073 Bl, International Patent Application Nos. WO 92/03918, WO 92/22645, WO 92/22647, WO 92/22670, WO 93/12227, WO 94/00569, WO 94/25585, WO 96/14436, WO 97/13852, and WO 98/24884 and U.S. Patent No. 5,981,175. See further Taylor et al., 1992, Chen et al., 1993, Tuaillon et al., 1993, Choi et al., 1993, Lonberg et al., (1994), Taylor et al., (1994), and Tuaillon et al., (1995), Fishwild et al., (1996).<br>
[0151] Kirin has also demonstrated the generation of human antibodies from mice in which, through microccll fusion, large pieces of chromosomes, or entire chromosomes, have been introduced. See European Patent Application Nos. 773 288 and 843 961. Xenerex Biosciences is developing a technology for the potential generation of human antibodies. In this technology, SCTD mice are reconstituted with human lymphatic cells, e.g., B and/or T cells. Mice are then immunized with ah antigen and can generate an immune esponse against the antigen. See U.S. Patent Nos. 5,476,996, 5,698,767, and 5,958,765.<br>
|0152] Human anti-mouse antibody (KAMA) responses have led the industry to prepare chimcnc or otherwise humanized antibodies. While chimeric antibodies have a human constant<br><br>
region and a marine variable region, it is expected that certain human anti-chimeric antibody (HACA) responses will be observed, particularly in chronic or multi-dose utilizations of .the antibody.  Thus, it would be desirable to provide fully human antibodies against EGFRvDI in order to vitiate concerns and/or effects of HAMA or HACA response. Antibody Therapeutics<br>
(0153) As discussed herein, the function of the EGFRvIH antibody appears important to at least a portion of its mode of operation. By function, it is meant, by way of example, the activity of the EGFRvIH antibody in operation with EGFRvUJ. Accordingly, in certain respects, it may be desirable in connection with the generation of antibodies as therapeutic candidates against EGFRvIII that the antibodies be capable of fixing complement and recruiting cytotoxic lymphocytes thus participating in CDC and ADCC. There are a number of isotypes of antibodies that are capable of the same, including, without limitation, the following: murine IgM, murine IgG2a, murine IgG2b, murine IgG3, human IgM, human IgGl, human IgG3, and human IgA. Also, it may be desirable in connection with the generation of antibodies as therapeutic candidates against EGFRvin that the antibodies be capable of activating antibody-dependent celluclar cytotoxicity (ADCC), through engagement of Fc receptors on effectors cells such as natural killer (NK) cells. There are a number of isotypes of antibodies that are capable of ADCC, including, without limitation, the following: murine IgG2a, murine IgG2b, murine IgG3, human IgGl, and human IgG3. It will be appreciated that antibodies that are generated need not initially possess such an isotype but, rather, the antibody as generated can possess any isotype and the antibody can be isotype switched thereafter using conventional techniques that are well known in the art. Such techniques include the use of direct rccombinant techniques (see e.g., U.S. Patent No. 4,816,397) and cell-cell fusion techniques (see e.g., U.S. Patent Nos. 5,916,771 and 6,207,418), among others.<br>
|0154] In the cell-cell fusion technique, a myeloma or other cell line is prepared that possesses a heavy chain with any desired isotype and another myeloma or other cell line is prepared that possesses the light chain. Such cells can, thereafter, be fused and a cell line expressing an intact antibody can be isolated.<br>
[0155] By way of example, certain anti-EGFRvIII antibodies discussed herein are human anti-EGFRvIll IgGl antibodies. If such antibody possessed desired binding to the EGFRvIII molecule, it could be readily isotype switched to generate a human IgM, human IgG3, or human IgGA while still possessing the same variable region (which defines the antibody's specificity and some of its affinity). Such molecules, including IgGl, would then be capable of fixing complement and participating in CDC, and, if comprising and IgGl or IgG3 constant region, such molecules would also be capable of participating in antibody-dependent cellular cytotoxicity (ADCC) through recruiting cytotoxic lymphocytes.<br><br>
[0156)        Accordingly, as antibody candidates are generated that meet desired "structural" attributes as discussed above, they can generally be provided with at least certain of the desired "functional" attributes through isotype switching. Design and Generation of Other Therapeutics<br>
[0157]	Based on the activity of the antibodies that are produced and characterized<br>
herein with respect to EGFRvIII, the design of other therapeutic modalities beyond antibody moieties<br>
is facilitated.  Such modalities include, without luritation, advanced antibody therapeutics, such as<br>
bispecific   antibodies,   immunotoxins,   and   radiolabeled   therapeutics,   generation   of  peptide<br>
therapeutics, gene therapies, particularly intrabodies, antisense therapeutics, and small molecules.<br>
[0158)	In connection with the generation of advanced antibody therapeutics, where<br>
complement fixation and recruitment of cytoxic lymphocytes is a desirable attribute, it is possible to<br>
enhance cell killing through the use of bispecifics, immunotoxins, or radiolabels, for example.<br>
[0159) For example, in connection with bispecific antibodies, bispecific antibodies can be generated that comprise (i) two antibodies one with a specificity to EGFRvIII and another to a second molecule that are conjugated together, (ii) a single antibody that has one chain specific to HGFRvni and a second chain specific to a second molecule, or (iii) a single chain antibody that has specificity to EGFRvIII and the other molecule. Such bispecific antibodies can be generated using techniques that are well known for example, in connection with (i) and (ii) see e.g., Fanger et al. Immunol Methods 4:72-81 (1994) and Wright and Harris, supra, and in connection with (iii) see e.g., Traunecker et al. Int. J. Cancer (Suppl.) 7:51-52 (1992). In each case, the second specificity can be made to the Fc chain activation receptors, including, without limitation, CD 16 or CD64 (see e.g., Deo et al. 18:127 (1997)) CD3 (Micromet's BiTE technology) or CD89 (see e.g., Valerius et al. Blood 90:4485-4492 (1997)). Bispecific antibodies prepared in accordance with the foregoing would be likely to kill cells expressing EGFRvIII, and particularly those cells in which the EGFRvIII antibodies of the invention are effective.<br>
[0160] In connection with immunotoxins, antibodies can be modified to act as immunotoxins utilizing techniques that are well known in the art. See e.g., Viterta Immunol Today 14:252 (1993). See also U.S. Patent No. 5,194,594. In connection with the preparation of radiolabeled antibodies, such modified antibodies can also be readily prepared utilizing techniques that arc well known in the art. See e.g., Junghans et al. in Cancer Chemotherapy and Biotherapy 655-686 (2d edition, Chafner and Longo, eds., Lippincott Raven (1996)). See also U.S. Patent Nos. 4,681,581, 4,735,210, 5,101,827, 5,102,990 (RE 35,500), 5,648,471, and 5,697,902. Each of immunotoxins and radiolabeled molecules would be likely to kill cells expressing EGFRvIII, and particularly those cells in which the antibodies described herein are effective.<br>
[0161) The antibodies can be designed to bind more quickly, or to dissociate more slowly from the epitopc, and thus the antibodies themselves can be designed therapeutics. The<br><br>
altered characterisitics of the antibodies can be used, for example, in the administration of a therapeutic to a patient. Therapeutic Immunoconiueates<br>
[0162)	As will be appreciated,  antibodies conjugated to drugs, toxins, or other<br>
molecules (immunoconjugates or immunotoxins) are highly useful in the targeted killing of cells that<br>
express a molecule that can be specifically bound by a specific binding molecule, such as an<br>
antibody.   As discussed above, EGFRvIII is not known to be expressed on any normal tissues.<br>
Further,  EGFRvIII  shows  significant expression in numerous human tumors.     Accordingly,<br>
EGFRvlll is a highly attractive molecule for targeting with an immunotoxin.<br>
(0163)	Many reports have appeared on the attempted specific targeting of tumor cells<br>
with monoclonal antibody-drug conjugates (Sela et al. in Immunoconjugates 189-216 (C. Vogel, ed.<br>
1987); Ghose et al, in Targeted Drugs 1-22 (£. Goldberg, ed. 1983); Diener et al, in Antibody<br>
Mediated Delivery Systems 1-23 (J. Rodwell, ed. 1988); Pietersz et al, in Antibody Mediated<br>
Delivery Systems 25-53 (J. Rodwell, ed. 1988); Bumol et al, in Antibody Mediated Delivery Systems<br>
55-79 (J. Rodwell, ed. 1988). Cytotoxic drugs such as methotrexate, daunorubicin, doxorubicin,<br>
vincristine, vinblastine, melphalan, mitomycin C, and chlorambucil have been conjugated to a variety<br>
of murine monoclonal antibodies. In some cases, the drug molecules were linked to the antibody<br>
molecules through an intermediary earner molecule such as serum albumin (Garnett et al. Cancer<br>
Res. 46:2407-2412 (1986); Ohkawa et al. Cancer Immumol. Immunother. 23:81-86 (1986); Endo et<br>
al. Cancer Res. 47:1076-1080 (1980)), dextran (Hurwitz et al. Appl. Uiochem. 2:25-35 (1980);<br>
Manabi et al. Biochem. Pharmacol. 34:289-291 (1985); Dillman et al. Cancer Res. 46:4886-4891<br>
(1986); Shoval et al. Proc. Natl. Acad. Sci. 85: 8276-8280 (1988)), or polyglutamic acid (Tsukada et<br>
al. J. Natl. Cane. List. 73:721-729 (I9H4); Kato et al. J. Mcd. Chem. 27:1602-1607 (1984); Tsukada<br>
et al. Br. J. Cancer 52:111-116 (1985)).<br>
(0164] A wide array of linker technologies has been employed for the preparation of such immunoconjugates and both cleavable and non-cleavable linkers have been investigated. In most cases, the full cytotoxic potential of the drugs could only be observed, however, if the drug molecules could be released from the conjugates in unmodified form at the target site.<br>
[0165] One of the cleavable linkers that has been employed for the preparation of antibody-drug conjugates is an acid-labile linker based on cis-aconitic acid that takes advantage of the acidic environment of different intracellular compartments such as the endosomes encountered during receptor mediated endocytosis and the lysosomes. Shen and Ryser introduced this method for the preparation of conjugates of daunorubicin with macromolecular carriers (Biochem. Biophys. Res. Commun. 102:1048-1054 (1981)). Yang and Reisfeld used the same technique to conjugate daunorubicin to an anti-melanoma antibody (J. Natl. Cane. Inst. 80:1154-1159 (1988)). Recently, Dillman et al. also used an acid-labile linker in a similar fashion to prepare conjugates of daunorubicin with an anti-T cell antibody (Cancer Res. 48:6097-6102 (1988)).<br><br>
[0166] An alternative approach, explored by Trouet et al. involved linking daunorubicin to an antibody via a peptide spacer arm (Proc. Mail. Acad. Sci. 79:626-629 (1982)). This was done under the premise that free drug could be released from such a conjugate by the action of lysosomal peptidases.<br>
|0167| In vitro cytotoxicity tests, however, have revealed that antibody-drug conjugates rarely achieved the same cytotoxic potency as the free unconjugated drugs. This suggested that mechanisms by which drug molecules are released from the antibodies are very inefficient. In the area of immunotoxins, conjugates formed via disulfide bridges between monoclonal antibodies and catalytically active protein toxins were shown to be more cytotoxic than conjugates containing other linkers. See, Lambert et al, J. Biol. Chem. 260:12035-12041 (1985); Lambert et al. in Immunotoxins 175-209 (A. Frankel, ed. 1988); Ghetie et al. Cancer Res. 48:2610-2617 (1988). This was attributed to the high intracellular concentration of glutathione contributing to the efficient cleavage of the disulfide bond between an antibody molecule and a toxin. Despite this, there are only a few reported examples of the use of disulfide bridges for the preparation of conjugates between drugs and macromolecules. Shen et al. described the conversion of methotrexate into a mercaptoethylamide derivative followed by conjugation with poly-D-lysinc via a disulfide bond (J. Biol. Chem. 260:10905-10908 (1985)). In addition, a report described the preparation of a conjugate of the trisulfide-containing toxic drug calicheamycin with an antibody (Menendez et al. Fourth International Conference on Monoclonal Antibody Immunoconjugates for Cancer, San Diego, Abstract 81 (1989)). Another report described the preparation of i conjugate of the trisulfidc-containing toxic drug calicheamycin with an antibody (Hinman et al, 53 Cancer Res. 3336-3342 (1993)).<br>
[0168] One reason for the lack of disulfide linked antibody-drug conjugates is the unavailability of cytotoxic drugs that bear a sulfur atom containing moiety that can be readily used to link the drug to an antibody via a disulfide bridge. Furthermore, chemical modification of existing drugs is difficult without diminishing their cytotoxic potential.<br>
|0169| Another major drawback with existing antibody-drug conjugates is their inability to deliver a sufficient concentration of drug to the target site because of the limited number of targeted antigens and the relatively moderate cytotoxicity of cancerostatic drugs like methotrexate, daunorubicin and vincristine. In order to achieve significant cytotoxicity, linkage of a large number of drug molecules either directly to the antibody or through a polymeric carrier molecule becomes necessary. However such heavily modified antibodies often display impaired binding to the target antigen and fast in vivo clearance from the blood stream.<br>
(0170] Maytansmoids are highly cytotoxic drugs. Maytansine was first isolated by Kupchan et al. from the east African shrub Maytenus serrata and shown to be 100 to 1000 fold more cytotoxic than conventional cancer chemotherapeutic agents like methotrexate, daunorubicin, and vincristine (U.S. Pat. No. 3,896,111). Subsequently, it was discovered that some microbes also produce maytansinoids, such as maytansinol and C-3 esters of maytansinol (U.S. Pat. No. 4,151,042).<br><br>
Synthetic C-3 esters of maytansinol and analogues of maytansinol have also been reported (Kupchan et al. J. Med. Chem. 21:31-37 (1978); Higashide et al. Nature 270:721-722 (1977); Kawai et al. Chem. Pharm. Bull. 32:3441-3451 (1984)). Examples of analogues of maytansinol from which C-3 esters have been prepared include maytansinol with modifications on the aromatic ring (e.g. dechloro) or at the C-9, C-14 (e.g. hydroxylated methyl group), C-15, C-18, C-20 and C-4,5.<br>
|0171] The naturally occurring and synthetic C-3 esters can be classified into two groups:<br>
(a)	C-3 esters with simple carboxylic acids (U.S. Pat. Nos. 4,248,870; 4,265,814;<br>
4,308,268; 4,308,269; 4,309,428; 4,317,821; 4,322,348; and 4,331,598), and<br>
(b)	C-3 esters with derivatives of N-methyl-L-alanine (U.S. Pat. Nos. 4,137,230;<br>
4,260,608; 5,208,020; and Chem. Pharm. Bull. 12:3441 (1984)).<br>
10172) Esters of group (b) were found to be much more cytotoxic than esters of group (a).<br>
[0173] Maytansine is a mitotic inhibitor. Treatment of L1210 cells in vivo with maytansine has been reported to result in 67% of the cells accumulating in mitosis. Untreated control cells were reported to demonstrate a mitotic index ranging from between 3.2 to 5.8% (Sieber et al. 43 Comparative Leukemia Research 1975, Bibl. Haemal. 495-500 (1976)). Experiments with sea urchin eggs and clam eggs have suggested that maytansine inhibits mitosis by interfering with the formation of microtubules through the inhibition of the polymerization of the microtubule protein, tubulin (Remillard et al. Science 189:1002-1005 (1975)).<br>
[0174) In vitro, P388, L1210, and LY5178 murine leukemic cell suspensions have been found to be inhibited by maytansine at doses of 10-3 to 10-1 .mu.g/.mu.l with the P388 line being the most sensitive. Maytansine has also been shown to be an active inhibitor of in vitro growth of human nasopharyngeal carcinoma cells, and the human acute -lymphoblastic leukemia line CEM was reported inhibited by concentrations as low as 10'7 mg/ml (Wolpert-DeFillippes et al. Biochem. Pharmacol. 24:1735-1738 (1975)).<br>
[0175] In vivo, maytansine has also been shown to be active. Tumor growth in the P388 lymphocytic leukemia system was shown to be inhibited over a 50- to 100-fold dosage range which suggested a high therapeutic index; also significant inhibitory activity could be demonstrated with the LI210 mouse leukemia system, the human Lewis lung carcinoma system and the human B-16 melanocarcinoma system (Kupchan, Ped. Proc. 33:2288-2295 (1974)).<br>
[0176] Current methods of conjugation of maytansinoids with cell binding agents (such as antibodies) involve two reaction steps. A cell binding agent, for example an antibody, is first modified with a cross-linking reagent such as N-succinimidyl pyridyldithiopropionate (SPDP) to introduce dithiopyridyl groups into the antibody (Carlsson et al. Biochem. J. 173:723-737 (1978); U.S. Pat. No. 5,208,020). In a second step, a reactive ma'ytansinoid having a thiol group, such as DM1 (formally termed N2 -deacetyl-N2 -(3-mcrcapto-l-oxopropyl)-maytansine, as the starting<br><br>
reagent., is added to the modified antibody, resulting in the displacement of the thiopyridyl groups in the modified antibodies, and the production of disulfide-linked cytotoxic maytansinoid/antibody conjugates (U.S. Pat. No. 5,208,020). A one-step process for conjugation of maytansinoids is described in U.S. Patent No. 6,441,163. Maytansinoid-based immunotoxin technology is available from Immunogen Corporation (Cambridge, MA).<br>
[0177] Another important toxin technology is based upon auristatin toxins. Auristatins are derived from Dolastatin 10 that was obtained from the Indian Ocean sea hare Dolabella, as a potent cell growth inhibitory substance. See U.S. Patent Nos. 4,816,444 and 4,978,744. With respect to other Dolastatins, see also U.S. Patent Nos. 4,414,205 (Dolastatin-1, 2, and 3), 5,076,973 (Dolastaun-3), 4,486,414 (Dolastatin-A and B),; 4,986,988 (DolasVatin-13), 5,138,036 (Dolastatin-14), and 4,879,278 (dolastatin-15). Isolated and synthesized by Dr. Pettit and colleagues at the University of Arizona, a variety of auristatine derivatives have been tested and shown to be highly toxic to cells. See Pettit et al. Antineoplastic agents 337. Synthesis of dolastatin 10 structural modifications. Anticancer Drug Des. 10(7):529-44 (1995), Woyke et al. In vitro activities and postantifungal effects of the potent dolastatin 10 structural modification auristatin PHE. Antimicrobial Agents and Chemotherapy. 45:3580-3584 (2001), Pettit et al. Specific activities of dolastatin 10 and peptide derivatives against Cryptococcus neoformans. Antimicrobial Agents and Chemotherapy. 42:2961-2965 (1998), WoykeThree-dimensional visualization of microtubules during the Cryptococcus neoformans cell cycle and the effects of auristatin PHE on microtubule integrity and nuclear localization. Submitted, Antimicrobial Agents and Chemotherapy.<br>
[0178] Recently, additional auristatin derivatives have been developed that appear quite effective when delivered as payloads on antibodies. For example monomethyl auristatin E (MMAE) has been shown as a potent toxin for tumor cells when conjugated to tumor specific antibodies. Doronina et al. Development of potent monoclonal antibody auristatin conjugates for cancer therapy. Nature Biotechnology. (2003) (available online), Francisco et al. cAClO-vcMMAE, an anti-CD30-monomethyl auristatin E conjugate with potent and selective antitumor activity. Blood. (2003) May 8 [Epub ahead of print], Epub 2003 Apr 24 (available online). In addition to the toxicity of the auristatin molecule, research has shown that peptide-linked conjugates are more stable , and, thus, more specific and less toxic to normal tissues than other linker technologies in buffers and plasma. Doronina et al. Development of potent monoclonal antibody auristatin conjugates for cancer therapy. Nature Biotechnology. (2003) (available online), Francisco et al. cAClO-vcMMAE, an anti-CD30-monomethyl auristatin E conjugate with potent and selective antitumor activity. Blood. (2003) May 8 [Epub ahead of print]. Epub 2003 Apr 24 (available online). Such linkers are based on a branched peptide design and include, for example, mAb-valine-citrulline-MMAE and mAb-phenylalanine-lysine-MMAE conjugates. Doronina et al. Development of potent monoclonal antibody auristatin conjugates for cancer therapy. Nature Biotechnology. (2003) (available online), Francisco et al. cAClO-vcMMAE, an anti-CD30-monomethyl auristatin E conjugate with potent and selective<br><br>
antitumor activity. Blood. (2003) May 8 [Epub ahead of print]. Epub 2003 Apr 24 (available online). Such designs and conjugation techniques are described, for example, by King et al. Monoclonal antibody conjugates of doxorubicin prepared with branched peptide linkers: inhibition of aggregation by methoxytriethyleneglycol chains. J Med Chem. 45(19):4336-43 (2002) and Dubowchik et al. Cathepsin B-sensitive dipeptide prodrugs. 2. Models of anticancer drugs paclitaxel (Taxol), mitomycin C and doxorubicin. Bioorg Med Chem Lett. 8(23):3347-52 (1998). Auristatin E-based immunotoxin technology based upon the foregoing is available from Seattle Genetics Corporation (Seattle, WA).<br>
|0179] There are a large number of novel microtubule effecting compounds obtained from natural sources-extracts, and semisynthetic and synthetic analogs that appear to possess potential as toxins for the generation of immunoconjugates. (see the website at newmedinc "dot" com). Such molecules and examples of drug products utilizing them, include the following: Colchicine-site Binders (Curacin), Combretastatins (AVE806, Combretastatin A-4 prodrug (CA4P), Oxi-4503), Cryptophycins (LY355703), Discodermolide, Dolastatin and Analogs (Auristatin PHE, Dolastatin 10, 1LX-651, Symplostatin 1, TZT-1027), Epolhilones (BMS-247550, BMS-310705, EP0906, KOS-862, ZK-EPO), Hleutherobin, FR182877, Halichondrin B (E7389), Halimide (NPI-2352 and NPI-2358), Hemiasterlins (HTI-286), Laulimalide, Maytansinoids ("DMl")(Bivatuzumab mertansine, Cantuzumab mertansine, huN901-DM 1/BB-10901 TAP, MLN591DM1, My9-6-DMl, Trastuzumab-DMl), PC-SPES, Peloruside A, Resveratiol, S-allylmercaptocysteine (SAMC), Spongistatins, Vitilevuamide, Molecular Motor-Kinesins (SB-715992), Designed Colchicine-Site Binders (A-289099, A-293620/A-318315, ABT-751/E7010, D-24851/D-64131, ZD6126), Other Novel Spindle Poisons (2-Methoxyestradiol (2-ME2), Bezimidazole Carbarnates (ANG 600 series, Mebendazolc), CP248/CP461, HMN-214, R440, SDX-103, T67/T607). Further, additional marine derived toxins arc reviewed in Mayer, A.M.S. Marine Pharmacology in 1998: Antitumor and Cytotoxic Compounds. The Pharmacologist. 41(4):159-164 (1999). Therapeutic Administration and Formulations<br>
[0180]A prolonged duration of action will allow for less frequent and more convenient dosing schedules by alternate parenteral routes such as intravenous, subcutaneous or intramuscular injection.<br>
[0181] When used for in vivo administration, antibody formulations described herein should be sterile. This is readily accomplished, for example, by filtration through sterile filtration membranes, prior to or following lyophilization and reconstitution. Antibodies ordinarily will be stored in lyophilized form or in solution. Therapeutic antibody compositions generally are placed into a container having a sterile access port, for example, an intravenous solution bag or vial having an adapter that allows retrieval of the formulation, such as a stopper pierceable by a hypodermic injection needle.<br><br>
(0182) The route of antibody administration is in accord with known methods, e.g., injection or infusion by intravenous, intraperitoneal, intracerebral, intramuscular, intraocular, intraarterial, intrathecal, inhalation or intralesional routes, or by sustained release systems as noted below. Antibodies are preferably administered continuously by infusion or by bolus injection.<br>
[0183] An effective amount of antibody to be employed therapeutically will depend, for example, upon the therapeutic objectives, the route of administration, and the condition of the patient. Accordingly, it is preferred for the therapist to liter the dosage and modify the route of administration as required to obtain the optimal therapeutic effect. Typically, the clinician will administer antibody until a dosage is reached that achieves the desired effect. The progress of this therapy is easily monitored by conventional assays or by the assays described herein.<br>
[0184] Antibodies as described herein can be prepared in a mixture with a pharmaceutically acceptable carrier. Therapeutic compositions can be administered intravenously or through the nose or lung, preferably as a liquid or powder aerosol (lyophilized). Composition can also be administered parenterally or subcutaneously as desired. When administered systemically, therapeutic compositions should be sterile, pyrogen-free and in a parenterally acceptable solution having due regard for pH, isotonicity, and stability. These conditions are known to those skilled in the art. Briefly, dosage formulations of the compounds are prepared for storage or administration by mixing the compound having the desired degree of purity with physiologically acceptable carriers, excipients, or stabilizers. Such materials are non-toxic to the recipients at the dosages and concentrations employed, and include buffers such as THIS HCI, phosphate, citrate, acetate and other organic acid salts; antioxidants such as ascorbic acid; low molecular weight (less than about ten residues) peptides such as polyarginine, proteins, such as serum albumin, gelatin, or immunoglobulins; hydrophilic polymers such as polyvinylpyrrolidinone; amino acids such as glycine, glutamic acid, aspartic acid, or arginine; monosaccharides, disaccharides, and other carbohydrates including cellulose or its derivatives, glucose, mannose, or dextrins; chelating agents such as EDTA; sugar alcohols such as mannitol or sorbitol; counterions such as sodium and/or nonionic surfactants such as TWEEN, PLURON1CS orpolyethyleneglycol.<br>
[0185] Sterile compositions for injection can be formulated according to conventional pharmaceutical practice as described in Remington's Pharmaceutical Sciences (18* ed, Mack Publishing Company, Easton, PA, 1990). For example, dissolution or suspension of the active compound in a vehicle such as water or naturally occurring vegetable oil like sesame, peanut, or cottonseed oil or a synthetic fatty vehicle like ethyl oleate or the like may be desired. Buffers, preservatives, antioxidants and the like can be incorporated according to accepted pharmaceutical practice.<br>
[0186] Suitable examples of sustained-release preparations include semipermeable matrices of solid hydrophobic polymers containing the polypeptide, which matrices are in the form of shaped articles, films or microcapsules. Examples of sustained-release matrices include polyesters,<br><br>
hydrogels (e.g., poly(2-hydroxyethyl-methacrylate) as described by Langer et al., J. Biomed Mater. Res., (1981) 15:167-277 and Langer, Chem. Tech., (1982) 12:98-105, or poly(vinylalcohol)), polylactides (U.S. Pat. No. 3,773,919, EP 58,481), copolymers of L-glutamic acid and gamma ethyl-L-glutamate (Sidman et al., Biopolymers, (1983) 22:547-556), non-degradable ethylene-vinyl acetate (Langer et al., supra), degradable lactic acid-glycolic acid copolymers such as the LUPRON Depot™ (injectable microspheres composed of lactic acid-glycolic acid copolymer and leuprolidc acetate), and poly-D-(-)-3-hydroxybutyric acid (EP 133,988).<br>
[0187) While polymers such as ethylene-vinyl acetate and lactic acid-glycolic acid enable release of molecules for over 100 days, certain hydrogels release proteins for shorter time periods. When encapsulated proteins remain in the body for a long time, they may denature or aggregate as a result of exposure to moisture at 37°C, resulting in a loss of biological activity and possible changes in immunogenicity. Rational strategies can be devised for protein stabilization depending on the mechanism involved. For example, if the aggregation mechanism is discovered to be intermolecular S-S bond formation through disulfide interchange, stabilization may be achieved by mollifying sulfliydryl residues, Ivophili/.inK from acidic solutions, controlling moisture content, using appropriate additives, and developing specific polymer matrix compositions.<br>
[0188] Sustained-released compositions also include preparations of crystals of the antibody suspended in suitable formulations capable of maintaining crystals in suspension. These preparations when injected subcutaneously or intraperitoneally can produce a sustain release effect. Other compositions also include liposomally entrapped antibodies. Liposomes containing such antibodies are prepared by methods known per se: U.S. Pat. No. DE 3,218,121; Epstein et al., Proc. Natl. Acad. Sci. USA, (1985) 82:3688-3692; Hwang et al., Proc. Natl. Acad. Sci. USA, (1980) 77:4030-4034; EP 52,322; EP 36,676; EP 88,046; EP 143,949; 142,641; Japanese patent application 83-118008; U.S. Pat. Nos. 4,485,045 and 4,544,545; and EP 102,324.<br>
[0189J The dosage of the antibody formulation for a given patient will be determined by the attending physician taking into consideration various factors known to modify the action of drugs including severity and type of disease, body weight, sex, diet, time and route of administration, other medications and other relevant clinical factors. Therapeutically effective dosages may be determined by either in vitro or in vivo methods.<br>
[0190] An effective amount of the antibody to be employed therapeutically will depend, for example, upon the therapeutic objectives, the route of administration, and the condition of the patient. Accordingly, it is preferred that the therapist liter the dosage and modify the route of administration as required to obtain the optimal therapeutic effect. A typical daily dosage might range from about 0.001 mg/kg to up to 100 mg/kg or more, depending on the factors mentioned above. Typically, the clinician will administer 'he therapeutic antibody until a dosage is reached that<br><br>
achieves the desired effect. The progress of this therapy is easily monitored by conventional assays<br>
or as described herein.	<br>
[0191] It will be appreciated that administration of therapeutic entities in accordance with the compositions and methods herein will be administered with suitable carriers, excipients, and other agents that are incorporated into formulations to provide improved transfer, delivery, tolerance, and the like. A multitude of appropriate formulations can be found in the formulary known to all pharmaceutical chemists: Remington's Pharmaceutical Sciences (18th ed, Mack Publishing Company, Easton, PA (1990)), particularly Chapter 87 ty Block, Lawrence, therein. These formulations include, for example, powders, pastes, ointments, jellies, waxes, oils, lipids, lipid (cationic or anionic) containing vesicles (such as Lipofectin™), DNA conjugates, anhydrous absorption pastes, oil-in-water and water-in-oil emulsions, emulsions carbowax (polyethylene glycols of various iiiolccuhit weights), semi :&gt;hi! Ki'ls, ;&gt;nil .'.run-solid mixUuc.s i iitiUiiunt-', c;iil&gt;ow;ix. Any l the foregoing mixtures may be appropriate in treatments and therapies in accordance with the present invention, provided that the active ingredient in the formulation is not inactivated by the formulation and the formulation is physiologically compatible and tolerable with the route of administration. See also Baldrick P. "Pharmaceutical excipient development: the need for preclinical guidance." Regul. Toxicol. Pharmacol. 32(2):210-8 (2000), Wang W. "Lyophilization and development of solid protein Pharmaceuticals." Int. J. Pharm. 203(1-2): 1-60 (2000), Charman WN "Lipids, lipophilic drugs, and oral drug delivery-some emerging concepts." J Pharm Sci ,89(8):967-78 (2000), Powell et al. "Compendium of excipients for parenteral formulations" PDA J Pharm Sci Technol. 52:238-311 (1998) and the citations therein for additional information related to formulations, excipients and carriers well known to pharmaceutical chemists. Preparation of Antibodies<br>
(0192] Antibodies, as described herein, were prepared through the utilization of the XenoMouse® technology, as described below. Such mice, then, are capable of producing human immunoglobulin molecules and antibodies and are deficient in the production of murine immunoglobulin molecules and antibodies. "';chnologies utilized for achieving the same are disclosed in the patents, applications, and references disclosed herein. In particular, however, a one embodiment of transgenic production of mice and antibodies therefrom is disclosed in U.S. Patent Application Serial No. 08/759,620, filed December 3, 1996 and International Patent Application Nos. WO 98/24893, published June 11, 1998 and WO 00/76310, published December 21, 2000, the disclosures of which are hereby incorporated by reference. See also Mendez et al, Nature Genetics 15:146-156(1997).<br>
(0193) Through use of such technology, fully human monoclonal antibodies to a variety of antigens can be produced. In one cmbodiement, XenoMouse® lines of mice are immunized with an antigen of interest (e.g. EGFRvIIl), lymphatic cells are recovered (such as B-cells) from the mice<br><br>
that expressed antibodies, and such cells are fused with a myeloid-type cell line to prepare immortal hybridoma cell lines, and such hybridoma cell lines are screened and selected to identify hybridoma cell lines that produce antibodies specific to the antigen of interest. Provided herein are methods for the production of multiple hybridoma cell lines that produce antibodies specific to EGFRvIII. Further, provided herein are characterization of the antibodies produced by such cell lines, including nucleoride and amino acid sequences of the heavy and light chains of such antibodies.<br>
|0194] Alternatively, instead of being fused to myeloma cells to generate hybridomas, the antibody produced by recovered cells, isolated from immunized XenoMouse® lines of mice, are screened further for reactivity against the initial antigen, preferably EGFRvlll protein. Such screening includes ELISA with EGFRvIII protein, in vitro binding to NR6 M cells stably expressing full length EGFRvIII and intemalization of EGFRvIII receptor by the antibodies in NR6 M cells. Single B cells secreting antibodies of interest are then isolated using a EGFRvlII-specific hemolytic plaque assay (Babcook et al., Proc. Nail. Acad. Sci. USA, i93:7843-7848 (1996)). Cells targeted for lysis are preferably sheep red blood cells (SRBCs) coated with the EGFRvIII antigen. In the presence of a B cell culture secreting the immunoglobulin of interest and complement, the formation of a plaque indicates specific EGFRvIII-mediated lysis of the target cells. The single antigen-specific plasma cell in the center of the plaque can be isolated and the genetic information that encodes the specificity of the antibody is isolated from the single plasma cell. Using reverse-transcriptase PCR, the DNA encoding the variable region of the antibody secreted can be cloned. Such cloned DNA can then be further inserted into a suitable expression vector, preferably a vector cassette such as a pcDNA, more preferably such a pcDNA vector containing the constant domains of immunglobulin heavy and light chain. The generated vector can then be transfected into host cells, preferably CHO cells, and cultured in conventional nutrient media modified as appropriate for inducing promoters, selecting transformants, or amplifying the genes encoding the desired sequences. Herein, we describe the isolation of multiple single plasma cells that produce antibodies specific to EGFRvIII. Further, the genetic material that encodes the specificity of the anti- EGFRvIII antibody is isolated, introduced into a suitable expression vector that is then transfected into host cells.<br>
(01951 B ce^s fr°m XenoMouse mice may be also be used as a source of genetic material from which antibody display libraries may be generated. Such libraries may be made in bactenophage, yeast or in vitro via ribosome display using ordinary skills in the art. Hyperimmunized XenoMouse mice may be a rich source from which high-affinity, antigen-reactive antibodies may be isolated. Accordingly, XenoMouse mice hyperimmunized against EGFRvIII may be used to generate antibody display libraries from which high-affinity antibodies against EGFRvIII may be isolated. Such libraries could be screened against the pep3 oligopeptide and the resultingly derived antibodies screening against cells expressing EGFRvIII to confirm specificity for the natively display antigen. Full IgG antibody may then be expressed using recombinant DNA technology. See e.g., WO 99/53049.<br><br>
[0196]        In general, antibodies produced by the above-mentioned cell lines possessed<br>
fully human IgGl or IgG2 heavy chains with human kappa light chains.   In one embodiment, the<br>
9 antibodies possessed high affinities, typically possessing Kd's of from about 10"   through about 10"<br>
M, when measured by either solid phase and solution phase. In other embodiments the antibodies<br>
possessed lower affinities, from about 10-6 through about 10-8 M.<br>
|0197]        As appreciated by one of skill in the art, antibodies in accordance with the<br>
present embodiments can be expressed in cell lines other than hybridoma cell lines.   Sequences<br>
encoding particular antibodies can be used for transformation of a suitable mammalian host cell.<br>
Transformation can be by any known method for introducing polynucleotides into a host cell, including, for example packaging the polynucleotide in a virus (or into a viral vector) and transducing a host cell with the virus (or vector) or by transfection procedures known in the art, as exemplified by U.S. Patent Nos. 4,399,216, 4,912,040, 4,740,461, and 4,959,455). The transformation procedure used depends upon the host to be transformed. Methods for introduction of heterologous polynucleotides into mammalian cells are well known in the art and include dextran-mediated transfection, calcium phosphate precipitation, polybrene mediated transfection, protoplast fusion, electroporation, encapsulation of the polynucleotide(s) in liposomes, and direct microinjection of the DNA into nuclei.<br>
[0198] Mammalian cell lines available as hosts for expression are well known in the art and include many immortalized cell lines available from the American Type Culture Collection (ATCC), including but not limited to Chinese hamster ovary (CHO) cells, HeLa cells, baby hamster kidney (BHK) cells, monkey kidney cells (COS), human hepatocellular carcinoma cells (e.g., Hep G2), and a number of other cell lines. Cell lines of particular preference are selected through determining which cell lines have high expression levels and produce antibodies with constitutive EGFRvFII binding properties.<br>
EXAMPLES<br>
[0199) The following examples, including the experiments conducted and the results achieved are provided for illustrative purposes only and are not to be construed as limiting upon the present invention.<br>
[0200] The strategy for generating EGFRvIII-specific antibodies initially involved immunization of XenoMouse mice with combinations of antigens (peptide, various soluble proteins, antigen-expressing cells) followed by isolation of antibody producing cells, either as through fusions to produce hybridomas or isolation of B cell cells through the XenoMax™/SLAM™ technology. Antibody producing cells were subjected to a primary screen for specificity by ELISA and a secondary screen for cell surface binding by FMAT and/or FACS. Internalization assays were then conducted to identify antibodies that would be useful for drug delivery. Affinities of the antibodies were measured. Certain antibodies were selected for epitope mapping. In addition, certain<br><br>
antibodies were selected for in vitro and in vivo tests to analyze the efficacy of such antibodies for treatment of cancers<br>
EXAMPLE 1<br>
ANTIGEN PREPARATION A.        EGFRvIII PEP3-KLH Antigen Preparation<br>
|0201] In connection with Example 2, the 14-mer human EGFRvIII PEP3 (L E E K K G NYVVTDHC (SEQ ID NO: 56)) peptide was custom synthesized by R&amp;D Systems. The PEP3 peptide was then conjugated to keyhole limpet hemocyanin (KLH), as follows: EGFRvIII PEP3 (200 meg) (R&amp;D) was mixed with 50 meg of keyhole limpet hemocyanin (KLH; Pierce, Rockford, 1L) to a final volume of 165 mcl using distilled water. 250 mcl of conjugation buffer (0.1M MES, 0.9M NaCl, pH 4.7) was added and EGFRvIII PHP3 and KLH were crosslinked by the addition of 25 mcl of 10 mg/ml stock solution of l-ethyl-3-[3-dimethylaminopropyl]carbodiimide hydrochloride (EDC, Pierce, Rockford, IL). Conjugate was incubated for 2 hours at room temperature and the unreacted EDC was removed by centnfugation tarough a 1 kDa filter (Centrifugal filter; Millipore, Bedford, MA) using PBS pH 7.4.<br>
[0202) In connection with Example 3, the 14-mer human EGFRvIII PEP3 (L E E K K G NYVVTDHC (SEQ ID NO: 56)) peptide was custom synthesized. The PEP3 peptide was then conjugated to KLH, as follows: human EGFRvIII PEP3 (200 meg) was mixed with 50 meg of keyhole limpet hemocyanin (KLH; Pierce, Rockford, IL) to a final volume of 165 mcl using distilled water. 250 mcl of conjugation buffer (0.1M MES, 0.9M NaCl, pH 4.7) was added and EGFRvIII PEP3 and KLH were crosslinked by the addition of 25 mcl of 10 mg/ml stock solution of l-ethyl-3-[3-dimethylaminopropyl]c:arbodiimide hydrochloride (EDC, Pierce, Rockford, IL). Conjugate was incubated for 2 hours at room temperature and the unreacted EDC was removed by centnfugation through a 1 kDa filter (Centrifugal filter; Millipore, Bedford, MA) using PBS pH 7.4. B. B3Q0.19/EGFRvIII Transfectants<br>
|0203] In order to prepare the B300.19/EGFRvffl transfectants, wild type EGFR was initially cloned from A431 cells and EGFR gene was modified to code for EGFRvIII to delete the codons encoding residues 6-273, with a codon encoding a Glycine residue created at the junction of the deletion. The deletion occurs within the codons surrounding the deletion GTT (Valine) and CGT (Arginine), such that the resulting codon after the deletion is GOT (Glycine). (Wikstrand et al. J Neurovirol.4(2):148-58(1998))<br>
1.         Cloning of wild type EGFR Construct!<br>
|0204] PolyA+mRNA was extracted from A431 (ATCC) cells usingMicro-fast RNA kit (Invirrogen, Burlington, ON). Total cDNA was synthesized from polyA+ mRNA with random pdN6 primers and M-MuLV reverse transcnptase (NEB, New England Biolabs, Beverly, Mass.). A 2,3kb PCR product was amplified from A431 cDNA with the following primers:<br><br>
sense S'-GGATCTCGAGCCAGACCGGAACGACAGGCCACCTC-3'; (SEQ ID NO: 62)<br>
anti-sense 5'-CGGAlCTCGAGCCGGA(JCCCAGCACTTrGATCTT-3' (SEQ ID NO: 63)<br>
using Pfu DNA polymerase.<br>
|0205] The PCR product was digested with Xhol, gel purified and ligated into plasmid pWBFNP (see International Patent Application No. WO 99/45031) linearized with Xhol to yield plasmid Wt-EGFR/pWBFNP.<br>
2.         Generation of EGFRvUI Construct:<br>
t [0206]        PCR products amplified from plasmid Wt-EGFR/pWBFNP template with primer<br>
pairs C13659/C29538 and C29539/C14288 (BioSource International), in which the C29538 and C29539 were phosphorylated with T4 Polynucleotide kinase (NEB, New England Biolabs, Beverly, Mass.):<br>
C13659: 5'-CGGATGAATTCCCAGACCGGACGACAGGCCACCTC-3'(Sense) (SEQ ID NO: 64);<br>
C29538; 5'-ClTIXrnTTCCTCCAGAGCC-3'(Anti-Sense)(SEQ ID NO: 65);<br>
C29539: 5'-GTAATTATGTGGTGACAGATC-3'(Sense)(SEQ ID NO: 66);<br>
C14288:5'-CGGATCTCGAGCTCAAGAGAGCTTGGTTGGGAGCT-3 (Anti-Sense) (SEQ ID NO: 67).<br>
were ligated to introduce a deletion in the secuence encoding amino acids 6 through 273 of the EGFR extracellular domain and subcloned into expression vector pWBDHFR2 (see International Patent Application No. WO 99/45031).<br>
|0207) A 232 bp fragment representing the 5'end of the deletion was generated with primer pair C13659/C29538 from Wt-EGFR/pWBFNP template amplified with Pfu polymerase (NED, New England Biolabs, Beverly, Mass.). The PCR product was digested with EcoRl (NEB, New England Biolabs, Beverly, Mass.) and gel purified. A 1273 bp fragment representing the 3'end of the deletion was generated with primer pair C29539/C14288 from Wt-EGFR/pWBFNP and the template amplified with Pfia polymerase. The PCR product was digested with Xhol (NEB, New England Biolabs, Beverly, Mass.) and gel purified. Fragments were ligated into EcoRl/Xhol digested pWBDHFR2 with T4 DNA ligase (NEB, New England Biolabs, Beverly, Mass.) to yield construct EGFRvIIl/pWBDHFR<br>
[0208] The inrracellular domain of EGFR was introduced into the resulting construct as follows: A 1566bp Dralll/XhoI fragment was isolated from plasmid Wt-EGFR/pWBFNP and ligated into DraJll/XhoI digested EGFRvIIl/pWBDHFR to yield EGFRvIII-FL/pWBDHFR.<br><br>
3.         Transfection of B300.19 cells with EGFRvIII-FL/pWBDHFR:<br>
[0209] B300.19 cells (8xl06) were used per transfection in 700 µl DMEM/H1 medium. 20 µg EGFRvllI-FL/pWBDHFR and 2 µg CMV-Puro plasmid DNA were added. Cells were elecrroporated at 300 volts/960uF with Bio-Rad Gene Pulser. Following electroporation, cells were cooled on ice for 10 minutes and, thereafter, 10 ml non-selection medium (DMEM/HI Glucose, 10% FBS, 50 µM BME, 2mM L-Glutamine, 100 units Penicillin-G/ml, 100 units MCG Streptomycin/ml) was added. Cells were incubated for 48hrs at 37°C 7.5% CO2.<br>
[0210] Following incubation, cells were split into selection medium (DMEM/HI Glucose, 10% FBS, 2 mM L-Glutamine, 50 µM BME, 100 units Penicillin-G/ml, 100 units MCG Streptomycin/ml, 2µg/ml puromycin) at 2xl04, 0.4x10"' and 0.08xl04 cells/ well in 96 well plate and were selected in selection medium for 14 days to generate stable clones. Puro resistant clones were stained with E752 mAb (an anti-EGFR antibody, described in Yang et al., Crit Rev Oncol Hematol., 38(1): 17-23 (2001)) and goat anti-human IgG PE then analyzed on FACS Vantage (Becton Dickinson). C. Construction of EGFRvlll-RbFc Expres_sion Constructs.<br>
|0211] In order to generate the EGFRvlll rabbit Fc fusion, protein, we first constructed a vector containing DNA encoding rabbit Fc. This was ligated with DNA encoding EGFRvlll. This approach is described in more detail below:<br>
1.	Construction of RbFc/peDNA3,l Hvtzro:<br>
|0212| Primers 1322/867 (below) were used to amplify a 721bp fragment encoding the llinge-CH2-CH3 domain of rabbit IgG.<br>
#1322 (sense): S'-GGTGGCGGTACCTGGACAAGACCG'ITGCG-S' (SEQ ID NO: 68)<br>
#867   (antisense):   5'-ATAAGAATGCGGCCGCTCATTTACCCGGAGAGCGGGA-3'   (SEQ  ID<br>
NO: 69)<br>
|0213] The resulting PCR product was digested with Kpnl and NotI, gel purified and ligated into Kpnl/Notl digested pcDNA3.1(+)/Hygro (Invivrogen, Burlington, ON) to yield plasmid RbFc/pcDNA3.1 Hygro.<br>
2.	Construction of EGFRvm-RbFc/pCEP4:<br>
[0214] Primers 1290/1293 (below) were used to amplify an 1165bp product from EGFRvllI-FL/pWBDHFR plasmid template with Pfu polymerase<br>
#1290 (sense): 5'-CTACTAGCTAGCCACCATGCGACCCTCCGGGA-3' (SEQ ID NO: 70)<br><br>
#1293 (anti-sense): 5'-CGGGGTACCCGGCGATGGACGGGATC-3' (SEQ ID NO: 71)<br>
|0215] The PCR product was digested with Nhel and Kpnl, gel purified and ligated into Nhel/Kpnl digested RbFc/pcDNA3.1 Hygro to yield plasmid EGFRvIII-RbFc/pcDN A3.1 Hygro.<br>
|0216] A 2170 bp SnaBI/XhoI fragment was isolated from EGFRvIII-RbFc/pcDNA3.1 Hygro and subcloned into SnaBI/XhoI digested pCEP4 (Invitrogen, Burlington, ON) to yield plasmid EGFRvIII-RbFc/pCEP4.<br>
3.         Generation of 293F EGFRvIU-RbFc stable Cell Lines:<br>
[0217] Plasmid EGFRvlII-RbFc/pCEP4 was introduced into 293F cells (Gibco, Grand Island, NY) by Calcium Phosphate transfection, as follows: one day prior to transfection, lx!06293F cells were plated on a gelatin coated 100mm tissue culture petridish and incubated at 5% CO2, 37°C, Cells were fed with 10ml of fresh non-selective media (DMEM/F12, 10% FBS, 2mM L-Glutaminc, lOOU/ml Penicillin G, lOOU/ml MCG Streptomycin) 2-3 hours before transfection. Transfection reagents were prepared in a microfuge tube, as follow: lOug of DNA (EGFRvlIl-RbFc/pCEP4) was mixed with 62ul of 2M Calcium Phosphate and deionized water to make the final volume SOOul. In another tube pipette 500|al of 2XHBS is drawn aid used to transfer the transfection reagents.<br>
[0218] The solution in the tube pipette was added to the cells drop by drop, while maintaining proper pH by leaving cells in a 5% CO2 incubator until transfection was performed. 15-20 hours after transfection, cells were washed with PBS and feed with 10ml of fresh 293F non-selective media. Expressing cells were harvested with trypsin 48-72 post-transfection and cells were plated at O.OSxlO4 cells/well in a 96 well plate in 293F selective media (DMEM/F12, 10% FBS, 2mM L-Glutamme, lOOU/ml Penicillin G, lOOU/ml MCG Streptomycin, 250ug/m1 Hygromycin) for 14 days.<br>
[0219] Hygromycin resistent clones were screened by EL1SA using anti-EGFR antibody E763 (US Patent No. 6,235,883) as the capture antibody at lug/ml and detecting with a goat anti-rabbit IgG HRPO (C'alTag) at 1:100 dilution.<br>
EXAMPLE 2 PRODUCTION OF ANTl-EgfrViii ANTIBODIES THROUGH<br>
HYBR1DOMA GENERATION<br>
[0220] Eight XenoMouse mice that produce antibodies with a gamma-1 constant region (XenoMouse Gl mice) were immunized on day 0 and boosted on days 11, 21, 32, 44 and 54 for this protocol and fusions were performed on day 58. All immunizations were conducted via subcutaneous administration at the base of tail plus intrapcritoncal administartion for all injections. The day 0 immunization was done with 1.5 x 107 B300.19/EGFRvLll transfccted cells (Example 1A) suspended in pyrogen free DPBS admixed 1:1 v/v with complete Freunds adjuvant (CFA) (Sigma, St. Louis, MO) Boosts on days 11, 7.1, and 32 were done with 1.5 x 107 B300.19/EGFRvIIl<br><br>
transfected cells in DHBS admixed 1:1 v/v with incomplete Freunds adjuvant (IFA) (Sigma, St. Louis, MO). The boosts on day 44 was done with 5 µg of the PEP3 (EGFRvlII peptide) - KLH conjugate (Example 1) in DPBS admixed 1:1 v/v with EFA and final boost, on day 54, was done with 5ug PEP3 (EGFRvlII peptide) - KLH conjugate in DPBS without adjuvant.<br>
(02211 0" day 58, mice were euthanized, and then inguinal and Lumbar lymph nodes were recovered. Lymphocytes were released by mechanical disruption of the lymph nodes using a tissue grinder then depleted of T cells by CD90 negative selection. The fusion was performed by mixing washed enriched B cells and non-secretory myeloma P3X63Ag8.653 cells purchased from ATCC, cat. # CRL 1580 (Kearney et al, J. Immunol. 123:1548-1550 (1979)) at a ratio of 1:1. The cell mixture was gently pelleted by centrifugahon at 800 g. After complete removal of the supernatant, the cells were treated with 2-4 mL of Pronase solution (CalBiochem, cat. # 53702; 0.5 mg/ml in PBS) for no more than 2 minutes. Then, 3-5 ml of FBS was added to stop the enzyme activity and the suspension was adjusted to 40 rnl total volume using electro cell fusion solution, ECFS (0.3M Sucrose, Sigma, Car# S7903, O.lmM Magnesium Acetate, Sigma, Cat# M2545, 0.1 mM Calcium Acetate, Sigma, Cat# C4705 (St. Louis, MO)).<br>
|0222] The supernatant was removed after centrifugation and the cells washed by resuspension in 40 ml ECFS. This wash step was repeated and the cells again were resuspended in ECFS to a concentration of 2xl06 cells/ml. Electro-cell fusion was performed using a fusion generator, model ECM2001, Genetronic, Inc., San Diego, CA. The fusion chamber size used was 2.0 ml, and using the following instrument settings: Alignment condition: voltage: 50 v, time; 50 s, Membrane breaking at: voltage: 3000 v, time: 30 (is, Post-fusion holding time: 3 s. After fusion, the cells were re-suspended in DMEM (JRH Biosciences),15% PCS (Hyclone), containing HAT, and supplemented with L-glutamine, pen/strep, OP1 (oxaloacetate, pyruvate, bovine insulin) (all from Sigma, St. Louis, MO) and 1L-6 (boehringer Mannheim) for culture at 37 °C and 10% CO2 in air.<br>
[0223] Cells were plated in flat bottomed 96-well tissue culture plates at 4xl04 cells per well. Cultures were maintained in HAT (hypoxanthine, aminopterin and thymidine) supplemented media for 2 weeks before transfer to HT (hypoxanthine and thymidine) supplemented media. Hybridomas were selected for by survival in HAT medium and supematants were screened for antigen reactivity by EL1SA. The ELISA format entailed incubating supernatants on antigen coated plates (EGFRvlII peptide-OVA coated plates and wild type EGFr peptide-OVA coated plates as a counter screen) and detecting EGFRvIlI-specific binding using horseradish peroxidase (HRP) labeled mouse anti-human IgG (see Table 2.1).<br><br>
TABLE 2.1<br><br>
(Table Removed) <br><br>
[0224] As will be observed, at least four antigen specific hybridomas were detected: 13.1, 13.2, 13.3, and 13.4. These hybridomas that were positive in the ELISA assay EGFRvIII specificity were confirmed by FACS on stably transfected 300.19 cells expressing EGFRvIII versus 300.19 untransfected parental cells.<br>
(0225) Cloning was performed on selected antigen-positive wells using limited dilution plating. Plates were visually inspected for the presence of single colony growth and supematants from single colony wells then screened by antigen-specific ELISAs and FACS confirmation as described above. Highly reactive clones were assayed to verify purity of human gamma and kappa chain by multiplex ELISA using a Luininex instrument. Based on EGFRvIII specificity in the ELISA and FACS assay, Clone 13.1.2 was selected as the most promising candidate for further screening and analysis. The nucleotide and amino acid sequences of the heavy and light chains of 13.1.2 antibody are shown in FIG. 3L and SEQ ID NO: 137 and 139 for heavy and light chain nucleic acids and 138 and 140 for heavy and light chain amino acid sequences.. In addition, a comparison of the 13.1.2 heavy chain and light chain sequences with the germline sequence from which they were derived us shown in FlGs 4 and 5.<br>
EXAMPLE 3<br>
ANTIBODY GENERATION THROUGH USE OF XENOMAX TECHNOLOGY Immunization of XenoMouse animals<br>
|0226] Human monoclonal antibodies against human EGFRvIII were developed by sequentially immunizing XenoMouse mice that produce antibodies with a gamma-1 constant region (XenoMouse Gl mice), XenoMouse mice that produce antibodies with gamma-2 constant regions (XenoMouse XMG2 mice), and XenoMouse mice that produce antibodies with a gamma-4 constant region (XenoMouse G4 mice).<br><br>
10227] To generate mAbs by through XenoMax technology, cohorts of XenoMouse Gl and XMG2 mice were immunized with EGFRvlII PEP3 (Example 1A) and EGFRvIII-expressing 300.19 cells (Example IB) or with bacterially expressed extracellular domain of EGFRvlII protein (EGFRvIlI-ECD) (Dr. Signer, Duke University) and EGFRvIII-expressing 300.19 cells or with EGFRvIIl-Rabbit Fc fusion protein (EGFRvIII-RbFc) (Example 1C) and EGFRvIII-expressing 300.19 cells or with EGFRvIIl-RbFc only via foot pad (FP), or via base of the tail by subcutaneous injection and intraperitoneum (BIP).<br>
10228) For footpad immunizations, the initial immunization was with or without 10 X 106 EGFRvIII-expressing 300.19 cells and with or without 10 u.g of EGFRvlII PEP3 or EGFRvffl-ECD or EGFRvIIl-RbFc mixed 1:1 v/v with Titermax gold (Sigma, Oakville, ON) per mouse. The subsequent boosts were performed with half of the amount of immunogen used in the initial immunization. The first four boosts were done by taking the immunogen mixed with alum (Sigma, Oalcville, ON), adsorbed overnight, per mouse as shown in the Table 3.1 below. This was followed by one injection with the respective immunogen in Titermax gold, one injection with alum and then a final boost of the immunogen in PBS as shown in Table 3.1. In particular, animals were immunized on days 0, 3, 7, 10, 14, 17, 21 and 24. The animals were bled on day 19 to obtain sera and determine the liter for harvest selection. The animals were harvested on Day 28.<br>
TABLE 11 Footpad immunization schedule<br>
(Table Removed)<br>
[0229] The initial BIP immunization with the respective immunogen, as described for the footpad immunization, was mixed 1:1 v/v with Complete Freund's Adjuvant (CFA, Sigma, Oakville, ON) per mouse. Subsequent boosts were made first with the immunogen respectively, mixed 1:1 v/v with Incomplete Freund's Adjuvant (1FA, Sigma, Oakville, ON) per mouse, followed by a final boost in PBS per mouse. The animals were immunized on days 0, 14, 28, 42, 56, and day 75 (final boost) as shown in Table 3.2 below. The animals were bled on day 63 to obtain sera and determine the liter for harvest selection. The animals were harvested on Day 78.<br>
TABLE 3.2 Bip Immunization schedule<br>
(Table Removed)<br><br>
Selection of animals for harvest By liter determination<br>
[0230] Anti-hEGFRvIII antibody tilers were determined by ELISA. EGFRvUI-RbFc (2.5 ug/ml) or a control RbFc (2 µg/ml) or EGFRvIIIpeptide-OVA (2 µg/ml) (Example 1) or control OVA (4 µg/ml) were coated onto Costar Labcoat Universal Binding Polystyrene 96-well plates (Corning, Acton, MA) overnight at four degrees. The solution containing unbound antigen was removed and the plates were treated with UV light (365nm) for 4 minutes (4000 microjoules). The plates were washed five times with dH2O. Sera from the EGFRvIIl immunized XenoMouse® animals, or na'i've XenoMouse® animals, were titrated in 2% milk/PBS at 1:2 dilutions in duplicate from a 1:100 initial dilution. The last well was left blank. The plates were washed five times with dHjO. A goat anti-human IgG Fc-specific horseradish peroxidase (HRP, Pierce, Rockford, EL) conjugated antibody was added at a final concentration of 1 µg/mL for 1 hour at room temperature.<br>
The plates were washed five times with dH20. The plates were developed with the addition of TMB chromogcnic substrate (Gaithersburg, MD) for 30 minutes and the ELISA was stopped by the addition of 1 M phosphoric acid. The specific liter of individual XenoMouse® animals was determined from the optical density at 450 nm and is shown in Tables 3.3 and 3.4. The liter represents the reciprocal dilution of the serum and therefore the higher the number the greater the humoral immune response to hEGFRvIII.<br>
[0231] For the mice immunized via base of the tail by subcutaneous injection and intraperitoneum, the litre was determined exactly as above except the plates were coated with EGFRvIII-RbFc (2.0 ng/ml) or a control RbFc (2.5 µg/ml).<br>
TABLE 3.3<br>
(Table Removed)<br><br>
 [0232]        All the XenoMouse animals from group 5 and XenoMouse animals 0743-5 from group 6 from Table 3.3 were selected for XenoMax harvests based on the serology.<br>
Table 3.4<br>
(Table Removed)<br><br>
 [0233]        XenoMouse animals (0695-1, 0695-3 and 0695-4) were selected for harvests based on the serology data in Table 3.4. Selection of B cells.<br>
|0234| B-cells from the above-discussed animals were harvested and cultured. Those secreting EGFRvIII-peptide specific antibodies were isolated as described in Babcook et al., Proc. Natl. Acad. Sci. USA, 93:7843-7848 (1996). ELISA was used to identify primary EGFRvIII-peptide-OVA -specific wells. About 5 million B-cells were cultured from XenoMouse animals in 245 96 well plates at 500 or 150 or 50 cells/well, and were screened on EGFRvIII-peptide-OVA to identify the antigen-specific wells. About 515 wells showed ODs significantly over background, a representative sample of which are shown in Table 3.5.<br>
TABLE 3.5<br>
(Table Removed) <br><br><br>
(0235) 244 of EGFRvIII-pcptide-OVA-Elisa positive wells of OD &gt; 0.5 were screened again on EGFRvIIl-peptide-OVA and on OVA to confirm that they were EGFRvIII-peptide specific. A representative example of these results is shown in Table 3.6.<br>
TABLE 3.6<br>
(Table Removed)<br><br>
Limited antigen assay and analysis<br>
|0236| The limited antigen analysis is a method that affinity-ranks the antigen-specific antibodies present in B-ccll culture supematants relative to all other antigen-specific antibodies. In the presence of a very low coating of antigen, only the highest affinity antibodies should be able to bind to any detectable level at equilibrium. (See, e.g., International Patent Application No. WO 03/48730)<br>
[0237) EGFRvIll peptide-OVA was coated to plates at three concentrations; 7.5 ng/ml, 1.5 ng/ml and 0.03 ng/ml for overnight at 4° C on 96-well Elisa plates. Each plate was washed 5 limes with dH3O, before 50ul of 1% milk in PBS with 0.05% sodium azide were added to the plate, followed by 4 µl of B cell supernatant added to each well. After 18 hours at room temperature on a shaker, the plates were again washed 5 times with dH2O. To each well was added 50ul of Gt anti-Human (Fc)-HRP at 1 ug/ml. After 1 hour at room temperature, the plates were again washed 5 times with dHjO and 50 u.1 of TMB substrate were added to each well. The reaction was stopped by<br>
the addition of 50uL of 1M phosphoric acid to each well and the plates were read at wavelength 450nm and the results shown in Table 3.7.<br>
Table 3.7<br>
(Table Removed)<br><br>
[0238] The results generated from limited antigen analysis were compared to the total OD obtained in high antigen assay. A relative ranking of affinity was done by taking the ratio of the OD obtained in limited antigen assay Vs that obtained in high antigen assay. Antibodies with higher ratio will have the highest affinity. Table 3.7 shows the sample of B-cell culture supernatants that were ranked based on limited antigen assay OD (for the lowest antigen plating concentration of 0.03 ng/ml) Vs the high antigen assay OD. Native cell binding assay by FMAT<br>
[0239] EGFRvIII peptide-OVA-Elisa positive well supernatants were analyzed for their ability to bind to the native form of EGFRvIII stably expressed on NR6 cells (NR6 M cells) (See, Barra et al. Epidermal growth factor ligand-independent, unregulated, cell-transforming potential of a naturally occurring human mutant EGFRvIII gene. Cell Growth Differ. 6(10): 1251-9 (1995)). NR 6 M cells were seeded at 8000 cells per well and incubated over night in 96 well FMAT plates. Media was then removed leaving 15 µl in the well. 15 µl B-ccll culture supernatants were added and 15 ul anti-human IgU Fc Cy5 al I µg/ml final concentration added to wells. It is then lell incubated at 4"(J for 2 hours. The cells were washed with 150 ul PBS, and fixed before reading on FMAT. The results were expressed as total fluorescent intensity (Table 3.8). Human anti-EGFRvIII mAb 13.1.2 was used as a positive control starting at 1 µg/ml final concentration and negative control was PK 16.3.1 at the same conecnlnilion. 134 of the 244 samples tested hound to NR6M cells of which 62 had a total fluorescence of greater than 8000. 6 of these 134 binders were false positives.<br>
[0240] The same type of native binding assay was done on NR6 Wt cells (NR6 cells expressing EOF receptor) (See Batra ct al. Epidermal growth factor ligand-independent, unregulated, cell-transforming potential of a naturally occurring human mutant EGFRvIII gene. Cell Growth<br>
Differ. 6(10): 1251-9 (1995)) to eliminate the binding is due to binding to Wt receptor (Table 3.8). ABX-EGF was used as a positive control and PK 16.3.1 at the same concentration was used as a negative control antibody. 3 out the 134 NR6 M binders were binding strongly to NR6 Wt cells. 190 of the 244 wells bound EGFRvFII peptide in Elisa were also bound to the native form on cells. Examples are given in Table 3.8.<br>
TABLE 3.8<br>
(Table Removed)<br><br>
In Table 3.8, supernatant from well 187A4 is identified as a Wt binder and 141C6 was a false positive for NR6 M cells binding.   Wells 129H6 and 183E11 are strong peptide binders with no native binding. Inlemalization assay<br>
|0241| The top 60 native binding B cell culture supematants were further assayed for their ability to internalize the receptor. NR6 M cells were seeded at 8000 cells/well into 96 well FMAT plates and incubated overnight. Media was removed and 10-15 ul B-Cell culture supernatant in a total volume of 30 µl media, in duplicate was added. Next, IS ul of secondary antibody (SS Alexa 647 anti-human IgG Fab at 1.5 u.g/ml final concentration) was added and the mixture was incubated on ice for 1 hr. An irrelevant B-Cell Culture supernatant was used to see the effect of the culture media. Human anti-EGFRvIII mAb 13.2.1 was used as a positive control starting at 1 ug/ml (final concentration) and negative control was PK 16.3.1 (human anti-KLH IgG2 antibody) at the same concentration. After incubation, the cells were washed with cold PBS, 50 µ1 media was added to all of the wells, one of the duplicates were incubated at 37 °C for 30 mins while the other duplicate remained on ice. After the incubations media was removed, l00ul of cold 50 mM glutathione was added to the set incubated at 37 °C and 100 ul of cold media added to the other set, both sets-were then left on ice for 1 hr. The cells were then washed with 100 µl cold PBS and then fixed with 1% paraformaldehyde and read in FMAT. The results were expressed as % internalized, calculated as<br><br>
total fluorescence in the presence of glutathione/ total fluorescence in the absence of glutathione X 100. Representative information is given in Table 3.9.<br>
TABLE 3.9<br>
(Table Removed)<br><br>
EGFRvIII-specific Hemolytic Plaque Assay<br>
(0242]         A number of specialized reagents were needed to conduct this assay.   These reagents were prepared as follows.<br>
1.	liioiinylation of Sheep red blood cells (SRBC). SRBCs were stored in RJPM1<br>
media as a 25% stock.  A 250 µl SRBC packed-cell pellet was obtained by aliquoting 1.0 ml<br>
of SRBC to a fresh eppendorf tube. The SRBC were pelleted with a pulse spin at 8000 rpm<br>
(6800 rcf) in microfuge, the supernatant drawn off, the pellet re-suspended in 1.0 ml PBS at<br>
pH 8.6, and the centrifugation repeated.   The wash cycle was repeated 2 times, then the<br>
SRBC pellet was transferred to a 15-ml falcon tube and made to 5 ml with PBS pH 8.6. In a<br>
separate 50 ml falcon tube, 2.5 mg of Sulfo-NHS biotin was added to 45 ml of PBS pH 8.6.<br>
Once the biotin had completely dissolved, the 5 ml of SRBCs were added and the tube<br>
rotated at RT for 1 hour.   The SRBCs were centrifuged at 3000rpm for 5 min and the<br>
supernatant drawn off.  The biotinylated SRBCs were transferred to an eppendorf tube and<br>
washed 3 times as above but with PBS pH 7.4 and then made up to 5 ml with immune cell<br>
media (RPMI 1640) in a 15 ml falcon tube (5% B-SRBC stock). Stock was stored at 4° C<br>
until needed.<br>
2.	Streptavidin (SA) coating of B-SRBC.   1 ml of the 5% B-SRBC stock was<br>
transferred into a fresh eppendorf tube.  The B-SRBC cells were washed 3 times as above<br>
and resuspended in 1.0 ml of PBS at pH 7.4 to give a final concentration of 5% (v/v). 10 ul of a 10 mg/ml streptavidin (CalBiochem, San Diego, CA) stock solution was added and the tube mixed and rotated at RT for 20 min. The washing steps were repeated and the SA-SRBC were re-suspended in 1ml PBS pH 7.4 (5% (v/v)).<br>
3.	EGFRvIII   coating   of SA-SRBC.   The   SA-SRBCs   were   coated   with<br>
biotmylated-EGFRvIllpetide-OVA at 10 ug/ml, mixed and rotated at RT for 20 min.  The<br>
SRBC were washed twice with 1.0 ml of PBS at pH 7.4 as above.  The EGFRvIII-coated<br>
SRBC were re-suspended in RPMI (+10%FCS) to a final concentration of 5% (v/v).<br>
4.	netirrminntion      of     the.      quality      of     EGFRvIllpeptide-SRBC      by<br>
immnmofluorescence IF).   10 µl of V'/o SA-SRUC anil 10 µl b% LvCJl'RvUl peplide-coatcd<br>
SRBC were each added to a separate fresh 1.5 ml eppendorf tube containing 40ul of PBS. A<br>
control human anti-EGFRvIII antibody was added to each sample of SRBCs at 45 ng/ml.<br>
The tubes were rotated at RT for 25 min, and the cells were then washed three times with<br>
100 µl of PBS. The cells svere re-suspended in 50 ul of PBS and incubated with 40 mcg/mL<br>
Gt-anti Human IgG Fc antibody conjugated to Alexa488 (Molecular Probes, Eugene, OR).<br>
The tubes were rotated at RT for 25 min, and then washed with 100 u,l PBS and the cells re-<br>
sti.spcmlcd  in   10  |il   I'US.     10  (il  of llic  stained  cells  wcic  spoiled  onlo a  clean  glass<br>
microscope slide, covered with a glass coverslip, observed under fluorescent light, and<br>
scored on an arbitrary scale of 0-4.<br>
5.	Preparation of plasma cells.   The contents of a single microculture well<br>
previously   identified   by   various  assays   as  containing  a   B   cell   clone   secreting  the<br>
immunoglobulin of interest were harvested.  Using a 100-1000 ul pipetman, the contents of<br>
the well were recovered by adding 37C RPMI (10% PCS).  The cells were re-suspended by<br>
pipetting and then transferred to a fresh 1.5 ml eppendorf tube (final vol. approx 500-700 ul).<br>
The cells were centrifuged in a microfuge at 2500 rpm (660 rcf) for 1 minute at room<br>
temperature, and then the tube was rotated 180 degrees and spun again for 1 minute at 2500<br>
rpm.   The freeze media was drawn off and the immune cells resuspended in 100 nl RPMI<br>
(10% PCS), then centrifuged.   This washing with RPMI (10% FCS) was repeated and the<br>
cells re-suspended in 60 ul RPMI (10% FCS) and stored on ice until ready to use.<br>
6.	Micromanipulation of plasma cells.  Glass slides (2x3 inch) were prepared<br>
in advance with silicone edges and allowed to cure overnight at RT.  Before use, the slides<br>
were treated with approx. 5ul of SigmaCoat (Sigma, Oakville, ON) wiped evenly over glass<br>
surface, allowed to dry and then wiped vigorously. To a 60 u,l sample of cells was added 60<br>
Hi each of EGFRvIIlpeptide-coated SRBC (5% v/v stock), 4x guinea pig complement<br>
(Sigma, Oakville, ON) slock prepared in RPMI (10% FCS), and 4x enhancing sera stock<br>
(1:150 in RPMI with 10% FCS).   The mixture was spotted (10-15 ul) onto the prepared<br>
slides and the spots covered with undi'uted paraffin oil.  The slides were incubated at 37° C<br>
for a minimum of 45 minutes.   The EGFRvIII-specific plasma cells were identified from plaques and rescued by micromanipulation (see Table 3.10).<br>
TABLE 3.10<br>
(Table Removed)<br><br>
Single cell I'CR, Cloning, Expression, Purification and Characterization of Recombinant anti-EGFRvIlI Antibodies.<br>
[0249) The genes encoding the variable regions were rescued by RT-PCR on the single micromanipulalcd plasma cells. inRNA was extracted and reverse transcriptasc PCR was conducted to generate cDNA. The cDNA encoding the variable heavy and light chains was specifically amplified using polymerase chain reaction. The human variable heavy chain region was cloned into an IgGl expression vector. This vector was generated by cloning the constant domain of human IgGl into the multiple cloning site of pcDNA3.1+/Hygro (Invitrogen, Burlington, ON). The human variable light chain region was cloned into an IgK expression vector. These vectors were generated by cloning the constant domain of human IgK into the multiple cloning site of pcDNA3.1+/Neo<br><br>
(Invitiogen, Burlington, ON). The heavy chain and the light chain expression vectors were then co-lipofected into a 60 mm dish of 70% confluent human embryonal kidney 293 cells and the transfected cells were allowed to secrete a recombinant antibody with the identical specificity as the original plasma cell for 24-72 hours. The supernatant (3 mL) was harvested from the HEK 293 cells and the secretion of an intact antibody was demonstrated with a sandwich ELISA to specifically detect human IgG (Table 3.11). Specificity was assessed through binding of the recombinant antibody to EGFRvIII using ELISA (Table 3.11).<br>
TABLE 3.11<br>
(Table Removed)<br><br>
[0250] The secretion ELISA tests were performed as follows. For Ab secretion, 2 HB/mL of Goat anti-human IgG II+L and for antigen binding, 1.5 µg/ml of EGFRvIlI-Rab Ig Fc fusion protein was coated onto Costar Labcoat Universal Binding Polystyrene 96 well plates and held overnight at four degrees. The plates were washed five times with dH2O. Recombinant antibodies were titrated 1:2 for 7 wells from the undiluted minilipofection supernatant. The plates were washed five times with dH2O. A goat anti-human IgG Fc-specific HRP-conjugated antibody was added at a final concentration of 1µg/mL for ' hour at RT for the secretion plates and binding plates detected with 1 |ag/ml Rb anti Hu Fc for 1 hour at room temperature. The plates were washed five times with dH20. The plates were developed with the addition of TMB for 30 minutes and the ELISA was stopped by the addition of 1 M phosphoric acid. Each ELISA plate was analyzed to determine the optical density of each well at 450 nm. Sequencing and sequence analysis<br>
[0251]         The cloned heavy and light chain cDNAs were sequenced in both directions and analyzed to determine the germlme sequence derivation of the antibodies and identify changes from<br><br>
germline sequence. Such sequences are provided in FIGs. 3A-3K and (SEQ ID NO: 34-55). A comparison of each of the heavy and light chain sequences and the germline sequences from which they are derived is provided in FIGs 4-7. In addition, the sequence of the hybridoma derived 13.1.2 antibody is compared to its germline sequence in FIGs. 4 and 5.<br>
[0252) As will be appreciated from the discussion herein, each of the 131 antibody and the 13.1.2 antibody possess very high affinities for EGFRvlII, are internalized well by cells, and appear highly effective in cell killing when conjugated to toxins. Intriguingly, each of the antibodies, despite having been generated in different immunizations of XenoMouse mice, and utilizing different technologies, each are derived from very similar germline genes. Based upon epitope mapping work (described herein), each of the antibodies, however, appear to bind to slightly different epitopes on the EGFRvlII molecule and have slightly different residues on EGFRvlII that are essential for binding. These results indicate that the germline gene utilization is of importance to generation of antibody therapeutics targeting EGFRvlII and that small changes can modify the binding and effects of the antibody in ways that allow further design of antibody and other therapeutics based upon these structural findings. Binding of Anli-EGFRvIII mAbs to native EGFRvlII expressed on cells<br>
[0253] In this example, binding of anti-EGFRvIU antibodies to NR6 M cells was measured. Specifically, unquantitated supernatants of XenoMax derived IgGl recombinant antibodies were assayed for their ability to bind to NR6 M and NR6 WT cells. Cells were seeded at 10000 / well and incubated overnight at 37 C it FMAT 96 well plates. Media was removed and 40 ul mini lipo supernatant (titrated down) was added, the cells were incubated on ice for 1 hr. The human 13.1.2 EGFRvlII antibodies and ABX EGF (E7.6.3, U.S. Patent No. 6,235,883) antibodies were added as positive controls. The PK 16.3.1 antibody was used as a negative control. The cells were washed with Cold PBS, secondary antibody was added (SS Alexa antihuman IgG Fc) at 1 ug/ml, 40 µl/well and incubated on ice for 1 hr. The cells were then washed with Cold PBS and fixed and read by FMAT. All antibodies were tested for specificity for binding by counter screening against NR6 WT cells. Purification of Recombinant Anti-EGFRvlll Antibodies<br>
[0254] For larger scale production, heavy and light chain expression vectors (2.5 ug of each chain/dish) were lipofected into ten 100 mm dishes that were 70% confluent with HEK 293 cells. The tiunsfcclcd cells were incubated at 37°C for 4 clays, the supernatant (6 mL) was harvested and replaced with 6 mL of fresh media. At day 7, the supernatant was removed and pooled with the initial harvest (120 itiL total from 10 plates), liueh untibody was purified from the supernatant using a Protcin-A Sepharose (Amersham Biosciences, Piscataway, NJ) affinity chromatography (1 mL). The antibody was eluted from the Protem-A column with 500 mcL of 0.1 M Glycine pH 2.5. The eluate was dialyzed in PBS, pH 7.4 and filter-sterilized. The antibody was analyzed by non-reducing SDS-PAGE to assess purity and yield. Concentration was also measured by LTV analysis at OD 250.<br><br>
Intemalization of EGFRvIII receptor by recombinant anti-EGFRvIII mAbs<br>
[0255) XcnoMax derived IgGl recombinant antibodies were expressed, purified and quantitated as described previously. Antibodies were further assayed for their ability to internalize the EGFRvIII receptor in NRG M cells. 250,000 NR6 M cells were incubated with primary antibody (SC95, SC131, SC133, SC139, SC150, SC170, SC211, SC230, SC250 and human 13.1.2 as a control) at 0.25 ug/ml, 7 mins on ice in 96 well v-bottomed plate in triplicate. The cells were washed with cold 10% PCS in PBS and secondary antibody (SS Alexa antihuman IgG Fab) at 3 ug/ml Fab was added and incubated for 7 mins on ice. The cells were washed with cold 10% PCS in PBS once and then resuspended in cold media. Next, two j*ts of the triplicate were incubated at 37 °C and the remaining set was incubated at 4 °C for 1 hr. After that the cells incubated at 4 °C and one set of the cells incubated at 37 °C were treated with glutathione (as previously mentioned) for 1 hr on ice. Then the cells were washed and resuspended in 100 ul of cold 1% PCS in PBS and analyzed by FACS. The % internalization was calculated from the geometric mean obtained from the FACS analysis [(mean at 37 °C with glutathione - mean at 4 °C with glutathione) / (mean at 37 °C without glutathione - mean at 4 C with glutathione)]. NA means that a FACS analysis was performed but the data was not provided in Table 3.12.<br>
TABLE 3.12<br>
(Table Removed)<br><br>
(0257) 13.1.2 is an antibody that was generated through hybridoma generation (Example 2) that was directed against the EGl-'RvIII epitope previously and was used as a positive control in this experiment. These results in Table 3.12 demonstrate the presence of two subsets of antibodies, those that are efficiently internalized (70-80%) and those that are not (22% or less).<br>
EXAMPLE4 EPITOPE MAPPING OF HUMAN ANTI EefrViii ANTIBODIES<br>
[0258] In order to determine the e-otopes to which certain of the antibodies of the present invention bound, the epitopes of 6 human and 3 murine monoclonal antibodies (mabs) against EGFRvIII were mapped using synthetic peptides derived from the specific EGFRvIJI peptide sequence. The antibodies mapped were the human hybridoma derived anti-EGFRvIII 13.1.2 antibody, the human XenoMax derived anti-EGFRvIII 131, 139, 250, 095, and 211 antibodies and the murine anti-EGFRvIII H10, Y10, and B9 antibodies (from Dr. D. Bigner, Duke University).<br>
|0259| The approach that was used was a custom SPOTs peptide array (Sigma Genosys) to study the molecular interaction of the human anti-EGFrVHI antibodies with their peptide epitope. SPOTs technology is based on the solid-phase synthesis of peptides in a format suitable for the systematic analysis of antibody epitopes. Synthesis of custom arrayed oligopeptides is commerically available from Sigma-Genosys. A peptide array of overlapping oligopeptides derived from the amino-acid sequence of the EGFr VIII variant was ordered from Sigma-Genosys.<br>
(0260] A series of nine 12-mer peptides were synthesized as spots on polypropylene membrane sheets. The peptide array spanned residues 1 - 20 of the EGFrVIII sequence, representing the deletion of amino acids 6-273 in the extracellular domain of wtEGFr, and the generation of a glycine (G) residue at the junction point. Each consecutive peptide was offset by 1 residue from the previous one, yielding a nested, overlapping library of arrayed oligopeptides. The membrane carrying the 9 peptides was reacted with 9 different anti EGFrVIII antibodies (1 ug/ml). The binding of the mAbs to the membrane-bound peptides was assessed by an enzyme-linked immunosorbent assay using HRP-conjugated secondary antibody followed by enhanced chemiluminescence (ECL). The array utilized is shown in Table 4.1.<br>
TABLE 4.1 Spot Array Sequence:<br>
(Table Removed)<br>
 [0261] In addition, functional epitopes were mapped by combinatorial Alanine scanning. In this process, a combinatorial Alanine-scanning strategy was used to identify amino acids in the EGFvVlll peptide necessary for interaction with anti-EGFRvIII mAbs. To accomplish<br>
this, a second set of SPOTs arrays was ordered for Alanine scanning. A panel of variants peptides with alanine substitutions in each of the 12 residues was scanned as above. Spot #1, the unmutated sequence, is a positive control for antibody binding. The array utilized is shown in Table 4.2.<br>
Table. 4.2 Alanine Scanning Array:<br>
(Table Removed)<br>
 (0262] Epitopes of all 9 mAbs to the human EGFrVin were mapped and identified by SPOTs procedure. All 9 antibodies were reactive with the peptides. The results obtained with 3 murine antibodies and 6 XenoMouse mouse derived human antibodies are presented in Table 4.3. Highlighted residues are those which we mutated to alanine and abrogated binding by the test antibody. These are therefore relevant residues for binding to the antibody.<br>
TABLE 4.3<br>
(Table Removed)<br><br>
[0264] The shaded amino acids shown in Table 4.3 are the most relevant residues in the eiptope for antibody recognition. The minimal lengths of epitopes of all ten of the mAbs were precisely mapped using peptides of overlapping sequences, and the tolerance for mAb binding to<br>
mutated epitopes was determined by systematically replacing each residue in the epitope with Alanine.<br>
[0265) In Table 4.4, additional characteristics of the antibodies are summarized. Specifcially, a subset of the antibodies were tested for their binding of to lysates of tumor cell lines in Western plates of polyacrylamide gel electrophoresis under either non-reducing or reducing conditions. Purifed recombinant protein is also included. Antibodies binding in both reducing and non-reducing conditions suggest that the epitope is linear. Sample identifications:<br>
|0266]        EGFRvIII - the rabbit Fc fusion protein<br>
[0267)	HI477 -   H80 human tumor cell line transfected with EGFRvIII expression<br>
contruct. These cells express both EGFR and EGFRvIII.<br>
(0268)	EGFR - purified wild-type EGFR protein<br>
[0269]        A431 - human tumor cell line expressing only wild-type EGFR [0270J        A549 - human tumor cell line expressing only wild-type EGFR <br>
[0271]        H80 - human tumor cell line expressing only wild-type EGFR [0272]        EGFR Biacore - mAbs were tested in Biacore for binding to purifed EGFR as a highly sensitive test for specificity<br>
TABLE 4.4<br>
(Table Removed)<br><br><br>
[0273] The results showed that most of these mAbs have essentially the same binding specificity, seven of the mAbs were shownω bind specifically to the EGFrVIII variant, while 2 mAbs cross reacted with wildtype EGFr (murine H10 and human 211) in Western blots of purified protein and in lysate of A431 cells. Note, however, that while antibody 211 binds to both native and<br>
reduced purified EGFRvIlI in Western blots, it binds slightly more strongly to the non-reduced protein. In tests against a lysate of A431 cells, antibody 211 binds strongly to a band of the size of wild-type EGFR in the non-reduced sample but there is no signal in the reduced sample. This suggests that the binding of antibody 211 is due to a conformational epitope present in wild-type EGFR and represented differently in the EGFRvIlI variant. The epitopes of 5 of the mAbs are within residues 2-12 spanning the EGFRvIIl variant specific Glycine residue, whereas the epitope of 4 of the mAbs (including H10 and 211) spans residues 7-16 which are common to the EGFRvin and wildtype EGFr. Antibody 131 binds to A431 and A549 cells in FACS. These cells are apparently negative for expression of EGFRvIlI while positive for EGFR expression. Antibody 131 does not bind to non-reduced or non-reduced purified EGFR or to reduced or non-reduced lysates of A43 and A549 cells in Westerns suggesting that antibody 131 may be binding to a variant of EGFR expressed on the cell surface of some human tumor cell lines. This variant would be sensitive to denaturation.<br>
EXAMPLE 5 CHARACTERIZATION OF SPECIFICITY OF ANTI-EgfrViii<br>
ANTIBODIES IN VITRO<br>
[0274] The specificity of the purified antibodies was ascertained by performing FACS analysis on NR6 cells that were transfected with either wild type or mutant EGFR. Cells were incubated on ice with 5 ωg/ml of the respective antibody for 1 hr, washed in FACs buffer and subsequently incubated with PE-conjugated goat anti-human IgG.<br>
EXAMPLE 6<br>
Cross-Reactivity With Amplified EGFR<br>
[0275] Antibodies directed to variant EGF receptors have been shown to cross-react with subsets of wild type EGF receptors on cells in which gene amplification has occurred (Johns et al., Int. J. Cancer. 98: 398, 2002). To determine whether the human EGFRvIlI antibodies identified had similar properties, they were tested for their ability to recognize wild type EGF receptors on a variety of cells in culture. Antibodies were incubated with the indicated cell lines at 4°C. After washing in FACS buffer, a secondary antibody conjugated with phyoerythrin was added and the incubation was continued. All cell lines analyzed expressed vuld type EGFR. A subset of wild type EGFRs was recognized by the antibody XG1-131 on both A431 and SF-539 cells but not on A498 or SKRC-52 cells. Another antibody to EGFRvIlI, 13.1.2, did not recognize this subset of wild type EGFRs. When considered together these data indicate that only a subset of antibodies directed to the mutant EGFRvIlI are able to recognize wild type EGFRs on the surface of cells. The ability of certain antibodies directed to mutant EGFRvIlI to recognize a subpopulation of wild type EGF receptors is not dependent on total EGFR density but likely represents a novel conforrriational epitope that is unique to tumor cells. The ability of antibodies directed to EGFRvIlI to cross-react with subpopulations of wild type receptors may be determined by both the specific epitope within the<br>
junction of the mutant receptor and the affinity of the antibody for this unique epitope (See the results of the epitope mapping and affinity determination section herein).<br>
EXAMPLE 7 CHARACTERIZATION OF SPECIFICITY OF ANTI-EefrViii ANTIBODIES IN VITRO:<br>
BINDING<br>
OF THE ANTIBODIES TO CELL LINES<br>
[0276] The specificity of the purified antibodies was ascertained by performing FACS analysis on a panel of cell lines. H80, a human glioblastoma line, and H1477 (HSO-EGFRvETI) that expresses high levels of EGFRvIll, A431, a human epidermoid carcinoma line, and A549, a human lung carcinoma cell line were used as the cell lines. All cell lines were from Dr. Bigner except A431 and A549, which were from ATCC (Rockville, MD, U.S.A.). Cells were incubated on ice with 10 ^.g/ml of the respective antibody for 30 min., washed in FACS buffer and subsequently incubated with PE-conjugated goat anti-human IgG from Jackson ImmunoResearch (West Grove, PA, U.S.A.). In FIGs. 9A-9L and 10A-10D, the darkened histogram indicates cells stained with an irrelevant IgG, the outlined, or white histogram, represents the staining of the relevant antibodies. The anti-EGFRvIII antibodies 13.1.2, 131 and 139 bind to the EGFRvlII protein on the transfected cell lines. A graph summarizing some of the results is displayed in FIGs. 9M-9P.<br>
[0277] Antibodies directed to variant EGF receptors have been shown to cross-react with subsets of wild type EGF receptors on cells in which gene amplification has occurred (Johns et al., Int. J. Cancer. 98: 398, 2002). In this example, A431 and A549 stained by XG1-131 and XG1-139. FIG. 10B and FIG. 10 C show that 131 and 139 have certain cross reactivity with the wild type EGFR instead of just recognizing a subset of population in H80, A431 and A549 line. However, this cross reactivity is only at 10% of the level of ABX-EGF (E7.6.3) staining on these cell lines. The results are provided in FIGs 9A-9P and 10A-10D.<br>
[0278] Antibodies directed to cell surface antigens can be used as delivery vehicles that specifically transport drugs or toxins into cells. If the antibody stimulates intemalization of antigen, the drug or toxin can result in death of the cell, perhaps after the drug or toxin is cleaved from the antibody. Such a mechanism can be utilized to specifically kill tumor cells in animals and in patients. One way to select antibodies that can deliver drugs to cells is through secondary cytotoxicity assays. In these assays the primary antibody binds to the cell surface and a secondary antibody that is conjugated with a drug or toxin is added. If the primary antibody stimulates antigen intemalization, the secondary antibody will be co-internalized and upon cleavage of the drug or toxin result in cell killing.<br>
EX^vlPLES<br>
SECONDARY CYTOTOXICITY ASSAYS<br>
[0279] This example demonstrates how the antibodies could be used to direct toxins conjugated via secondary antibodies to cells expressing the target epitope (target cell). Antibodies<br>
conjugated to a toxin are administered to cells expressing the target peptide. The death of those cells indicates the effectiveness of the antibody toxin combination.<br>
[0280) The amount of specific killing required can vary depending upon the particular use. In one embodiment, any reduction in possibly cancerous cells is sufficient. For example, a reduction of 0-1, 1-5, 5-10, 10-20, 20-30, 30-40, 40-50, 50-60, 60-70, 70-80, 80-90, 90-95, 95-99, or 100% of the target cells will be sufficient. In another embodiment, the desired reduction in target cell number is also a function of the nonspecific lethality of the antibody combination. For example, antibody/toxin combinations that only have a 10 % decrease in target cell number may be sufficient, if there is very little nonspecific targeting and lethality by the antibody. For example, the antibody toxin combination kills less than 10% of a non-target population. Again, the particular amount will depend on the particular need and situation. Particularly useful are antibodies that are highly selective for a target cell and bind well to the target cell, or proteins associated with the cell. In one embodiment, the target is the EGFRvIH protein, or a fragment thereof. In one embodiment, antibodies that are human, or humanized, efficient at being internalized, specific to the EGFRvIII protein or fragments thereof, associate tightly with the EGFRvIII protein or fragment, and are associated with an effective toxin, are taught from these examples.<br>
HXAMPLI- 9<br>
SECONDARY CYTOTOX1CITY CLONOGEN1C ASSAYS<br>
[0281] In addition to the secondary Cytotoxicity assays, EGFRvIIl-specific antibodies can also be used for clonogenic assays. As above, antibodies associated with toxin are administered to cells. The ability of the cells to proliferate is monitored. A decrease in proliferation indicates that the antibody toxin combination is effective.<br>
EXAMPLE- 10 ANTI-EefrViii ANTIBODY (13.1.2) DIRECT CONJUGATES IN THE CYTQTOXICITY<br>
ASSAYS<br>
(0282) In addition to the indirectly conjugated examples above, these tests can also be performed with antibodies that are directly conjugated to the toxins. An antibody that is directly conjugated to a toxin is used in the same manner as described for the antibody toxin conjugate in Example 8.<br>
EXAMPLE 11<br>
IN VIVO ANTl-EgfrViii ANTIBODIES CHARACTERIZATION.<br>
[0283] The antibody toxin conjugates can also be tested in vivo. An antibody toxin conjugate is administered to a test organism that has target cells that are expressing the target peptide. A decrease in the number of target cells in the test organism indicates the ability of the antibody conjugated toxin to work in an in vivo setting.<br>
EXAMPLE 12<br>
EXPRESSION OF EgfrViii IN CANCER PATIENTS/HUMAN TUMORS <br>
[0284] The expression of EGFRvIII on human tumors was determined by staining frozen tissue sections from a variety of cancer patients with a combination of 2 murine monoclonal antibodies (B9, IgGl and Y10, IgG2 (Dr. Bigner, Duke University)) known to bind specifically to EGFRvllI. The same sections were stained with isotype matched control antibodies. A summary of the staining results obtained from all patient samples is presented in Table 12.1.<br>
TABLE 12.1 SUMMARY OF STAINING RESULTS FROM PATIENT SAMPLES<br>
(Table Removed)<br><br>
EGFRvIII&gt;+: include all tumors that express EGFRvIIT<br>
EGFRvllI&gt;++: include only those tumors that express at least 10% or more EGFRvllI<br>
[0286] The expression was found primarily on the cell membrane and/or cytoplasm. Significant numbers of breast (31%), NSCL (47%), and head &amp; neck (42%) cancer specimens stained positively for EGFRvllI. In certain instances, in order to obtain high quality IHC staining, the use of two antibodies can be better than the use of one antibody. Frozen tissue specimens were superior over fixed tissues.<br>
[0287] As appreciated by one of skill in the art, it may be advantageous to test patients before using therapeutic antibodies to ensure that the tumor which is being treated expresses EGFRvllI.<br>
EXAMPLE 13 JN VIVO ANTl-F.gfrViii ANTIBODIES CHARACTERIZATION.<br>
[0288] The method of Example 11 will be used to treat lung cancer and gliomas. This will be broadly examined by producing animal models. Animal models for glioblastoma and lung cancer are developed as follows: lung cancer cells that express wt-EGFR are transfected with EGFRvllI. The cells are injected into the lungs of nu/nu mice and tumors allowed to progress to a comparable stage to that above. Anti-EGFRvIII conjugates will then be injected intravenously as above every I lo 10 days as needed. I lie si/c iind prevention or suppression of continued growth of these cancer cells will then be monitored, to determine the effectiveness of these Anti-EGFRvIII<br>
antibodies and antibody-toxins combinations. As appreciated by one of skill in the art, this can be done for any of the antibodies disclosed herein.<br>
EXAMPLE 14 FUNCTIONAL CHARACTERIZATION OF EPITOPES BY SUBSTITIQNAL ANALYSES<br>
[0289] In order to further resolve the identity of those amino acid residues that are indispensable for binding within the EGFRvIEl epitope, substitutional analyses of the amino acids in the epitope peptides were performed. The starting point was the sequence that was derived from Example 4, LEEKKGNYWTD (SEQ ID NO 59). In this example each amino acid of the mapped epitope was substituted one-at-a-time by all 20 L-amino acids, thus, all possible single site substitution analogs were synthesized and screened to provide detailed information on the mode of peptide binding. Discrete substitution patterns were identified for mAbs 131 and 13.1.2. The results from the substitutions arc summarized in Table 14.1.<br>
TABLE 14.1<br>
(Table Removed)<br><br>
[0290] It appears that for mAb 13.1.2, 5 residues are important for binding (bold), while only 4 residues are essential for the binding of mAb 131. The rest of the residues were replaced by various amino acids without significant loss of binding. Although the 131 and 13.1.2 epitopes are identical by sequence and length, the binding pattern for each appears different. Binding of mAb 131 is strongly dependent on the residues EKNY (SEQ ID NO: 60). On the other hand, the data revealed that residues FFKGN (SF.Q ID NO: 61) are involcd in binding of mAb 13.1.2.<br>
Mxumplc 15 MAbs CHALN SHUFFLING<br>
[0291] Heavy and light chains of mAbs 131 and 13.1.2 were shuffled and transiently transfected into 293T cells. Seventy-two hours later supematants were collected and assayed for secretion and binding to EGFrVlll antigen by ELISA.<br>
[0292] The results demonstrated that antibodies derived from expression of 131 heavy chain with 13.1.2 kappa chain, and vice versa were expressed well but binding activity was reduced by 75% probably due to the different binding pattern of these two mAbs to EGFrVDI antigen, (data not shown). This demonstrates the difference between the two paratopes of the 131 and 13.1.2 mAbs, again suggesting that the structural characteristics of the epitope selected for between the two mAbs are different.<br>
EXAMPLE 16<br>
MOLECULAR MODELING OF 131 AND ITS PARATOPE<br>
[0293] This example demonstrates how three-dimensional structures can be generated for the proteins of the embodiments. The three-dimensional structure model of the variable region of antibody 131 was generated through a homology modeling approach using the Insightll modeling package from Accelrys (San Diego, CA). The model was built from the variable region sequences described below, Table 16.1. The residue numbering starts with the light chain amino acids, and continues to heavy chain amino acids.<br>
TABLE 16.1<br>
(Table Removed)<br>
[0294] Antibody 131 sequences were used to search against the Protein Data Bank to identify homologous antibodies and their structures. Based on the homologous antibodies' sequence similarity to the 131 antibody, several structures were selected. The structures selected for modeling samples from the Protein Data Bank had the Protein Data Bank identifications of 1HEZ, 2H1P, 1AQK, 1DQL, 1MF2 and 1FLR. These template structures were then aligned by superposition and used to generate structure-based sequence alignments among the templates. The sequences of antibody 131's variable region were then aligned to the template sequences. The structure and sequence alignments were used to generate the molecular model for the variable region of the 131 antibody. The sequence for CDR1, light chain was: RSSQSLVHSDGNTYLS (SEQ DD NO 103). The sequence for CDR2, light chain was: RJSRRFS (SEQ ID NO 103). The sequence for CDR3, light chain was: MQSTHVPWT (SEQ ID NO 105). The sequence for CDR1, heavy chain was: NYGMH (SEQ ID NO 108). The sequence for CDR2, heavy chain was: VrWYDGSDKYYADSVRG (SEQ ID NO 110). The sequence for CDR3, heavy chain was: DGYDILTGNPRDFDY (SEQ ID NO 112).<br><br>
[0295] The interaction surface for antibody 131 was calculated from the structure model and shown in FIG. 11. The various CDRs are identified as follows: LI (light CDR1) 10, HI (heavy CDR1) 20, L2 30, H2 40, L3 50 and H3 60. A prominent feature on the predicted antibody 131 interaction surface is a deep cavity,. The cavity is mainly surrounded by heavy chain CDR2, CDR3 and light chain CDR3, with a small portion contributed by light chain CDR1. The cavity is probably the binding pocket. Within 5 Angstroms of the binding cavity are residues 31, 37, 95-101, 143-147, 159, 162-166, 169-171, 211-219, 221 and 223. These residues are likely to comprise the paratope and make key contacts in the binding of EGFRvIIl epitope. It is also likely that the residues provide important structural features to the binding site in general.<br>
EXAMPLE 17 SITE-DIRECTED MUTAGENESIS CONFIRMING THE MODEL FOR ANTIBODY 131<br>
[0296) This example demonstrates one method l&gt;y which models that suggest residues that arc important in binding may be tested. The Example also results in several antibody variants. Antibody variants of the 131 clone were generated by single residue mutations introduced to the heavy and the light chain of mAb 131. These variants were then analyzed to determine how the altered side chains from the point mutation contributed to antigen binding.<br>
[0297] Changes were made in the heavy and light chains of mAb 131. On the heavy chain 1.216 was changd by site directed mutagenesis to R On the light chain, V99 w;is changed to F. Both mutations affected the expression and secretion of the variant antibodies compared to the wildtype sequence. Both mutations resulted in n loss of binding of the mAb variant to the EGFRvIlII antigen. This demonstrates L216 and V99 probably have significant contacts with the EGFRvIIl antigen since substitutions of these residues to R and F respectively resulted in reduced activity. Of course, it is always an option that these substitutions are disruptive to the antibody's general structure.<br>
EXAMPLE 18 MOLECULAR MODELING OF 13.1.2 AND ITS PARATOPE<br>
[0298] The three-dimensional structure model of the variable region of the 13.1.2 antibody was generated through homology modeling approach with the Insightll modeling package from Accelrys (San Diego, CA). The model was built from the variable region sequences, shown below in Table 18.1, using the published x-ray crystal structures as templates.<br>
TABLE 18.1<br>
(Table Removed)<br>
 [0299] The sequence for CDR1, light chain was: RSSQSLVHSDGNTYLS (SEQ ID NO: 101). The sequence for CDR2, light chain was: KISNRFS (SEQ ID NO: 116). The sequence for CDR3, light chain was: MQATQLPRT (SEQ ID NO: 118). The sequence for CDR1, heavy chain was: SYGMH (SEQ ED NO: 121). The sequence for CDR2, heavy chain was: VIWYDGSNKYYVDSVKG (SEQ ID NO: 123). The sequence for CDR3, heavy chain was: DGWQQLAPFDY (SEQ ID NO: 125).<br>
[0300] Antibody 13.1.2 sequences were used to search the Protein Data Bank to identify homologous antibodies. The structures with the Protein Data Bank identifications of 1HEZ, 2H1P, 8FAB and 1AQK were selected as modeling templates, based on their sequence similarity to antibody 13.1.2. The template structures were aligned by superposition and used to generate structure-based sequence alignments among the templates. The sequences of the variable regions of the 13.1.2 antibody were then aligned to the template sequences. The structure and sequence alignments were used to generate the molecular model for antibody 13.1.2 variable region.<br>
[0301] The interaction surface was calculated for the model and is shown in FIG. 12. A major feature of the 13.1.2 model is a long and narrow groove on the surface of the CDR region. The groove is outlined by heavy chain CDR2 140, CDR3 160, and light chain CDR1 110, CDR2 130 and CDR3 150. One end of the groove touches the rest oflight chain CDR3 150, and the other end opens to the wider area of heavy chain CDR3 160 near the heavy chain-light chain interface. The groove is probably the binding pocket for the antigen. Within 5 Angstroms of the binding groove are residues 31, 33, 35-39, 51, 54-56, 58-61, 94-101, 144-148, 160, 163-166, 172, and 211-221. These residues are likely to comprise the paratope for the binding of EGFRvIII epitope. It is also likely that the residues provide important structural features to the binding site in general.<br>
EXAMPLE 19 DOCKING MODELS OF A PEPTIDE TO AN ANTIBODY<br>
(0302)	1 he epitope mapping studies in Example 14 revealed lliat   the relevant ammo<br>
acids required for binding of the epitope to the paratope of 13.2.1 mAb reside in the six-residue peptide EEKKGN (SEQ ID NO: 127). Therefore, docking models of this six-residue peptide complexed to the CDR region of 13.1.2 structure model were generated. First, a model of the peptide EEKXGN (SEQ ID NO: 127) was produced. This was done, similarly as described before, except this time using the x-ray crystal structure of 1181, as identified in the Protein Data Bank, as the<br>
template. Next, this peptide structure was manually placed into the long groove to form an initial assembly complex. A Monte Carlo search in the conformational and orientational spaces were then automatically performed with the Docking module in InsightII The peptide conformation was allowed to be flexible by giving each of the Phi, Psi and Chi angles full rotational freedom. During the docking process, the residues within 5 Angstroms of the binding groove were allowed to move while the other residues of the antibody were fixed. The plausible configurations found by Monte Carlo search were subjected to simulated annealing and energy minimization to reach the final complex structure models. For each docking model obtained, the interaction energy between the antibody and the peptide was calculated with the Discover_3 module of Insightn package. The interaction energies for all docking models were assessed and the model with the strongest overall antibody-peptide interaction was examined and is shown in FIG. 13A and 13B.<br>
(0303) In this docking model, there are six hydrogen bonds between peptide EEKKGN (SEQ ID NO: 127) and antibody 13.1.2, as shown in FIG. 13B. The peptide residue number is labeled from N-termmus to the C-terminus as 1 through 6. Six hydrogen bonds are indicated by green dashed lines. The six pairs of amino acids forming hydrogen bonds are: E2...Y172, K3...H31, K4...H31, N6...D33, N6...Y37, and N6...K55. In this docking model, the peptide is bound to the groove in an extended p-strand conformation. Residues in the peptide alternately face the solvent and the antibody surface. The residues facing the binding groove with the most significant contacts to the antibody are E2, K4 and N6. This indicates that these three residues may be important to peptide binding, consistent with the epitope mapping results. The interaction energies for each of the six peptide residues with the 13.1.2 paratope was calculated with the Discover_3 module and the results are shown in Table 19.1. Table 19.1 shows the interaction energies for each of the six peptide residues with the 13.1.2 paratope. All energies are in the unit of kcal/mol.<br>
[0304] The residues with the strongest interaction energies are in the order of N6, K.4 and E2, confirming that these residues are key contributors on the antigen side in the antibody-antigen interaction, again consistent with experimental data. These data provided strong evidence to support the docking model. In this embodiment, the paratope is defined as the residues within 5 Angstroms of the docked peptide. The 20 residues comprising the paratope as so defined are residues 31-33, 35, 37, 55, 96-101, 148, 163, 165, 170, 172, 178 and 217-218. To evaluate, on an individual residue basis, the contribution of each of these residues of the antibody in the antibody-antigen interaction, the interaction energy between the paratope residues and the peptide EEKKGN (SEQ ED NO: 127) was calculated for each of the above 20 residues. The results are listed in Table 19.2. Table 19.2 shows the interaction energies for each of the 20 paratope residues with the peptide EEKKGN (SEQ ID NO: 127). All energies are in the unit of kcaVmol. The residues with the strongest interaction energies with the peptide are Lys55 and His31, followed by Tyrl72, Ala96, Asp33, Tyr37, Leu99, Thr97, Gln98, Lysl78 and Asnl70.<br>
TABLE 19.1<br>
(Table Removed)<br><br>
TABLE 19.2<br>
(Table Removed)<br><br>
EXAMPLE 20<br>
RATIONAL DESIGN FOR AFFINITY-IMPROVED ANTIBODIES [0305) This Example demonstrates how the docking model can be used as the basis of rational design for affinity-improved antibodies by site-directed mutagenesis. Each of the 13.1.2 paratope residues was mutated to all 19 other amino acids in silica, resulting in a total of 19x20 or 380 virtual mutants. The mutation was done by residue replacement followed by 50 steps of energy minimization to account for any local conforrnational changes that could be induced by the side chain change. The interaction energy between the whole peptide and the whole paratope was calculated for each mutant. Mutants with a total interaction energy stronger than the wild type 13.1.2 could potentially have a higher affinity for the peptide EEKKGN (SEQ ID NO: 127), and perhaps even the whole EGFRvIlI protein. These mutants most y have stronger coulumbic interactions than the wild type 13.1.2, but some of them have weaker van der Waals (VdW) interactions than the wild type antibody. Considering that in the wild type 13.1.2 antibody, the VdW interacting energy is -9.689 kcal/mol, mutants with VdW interaction energy weaker than -8.5 kcal/mol were filtered out. The<br><br>
rest of the mutants that have stronger, total interaction energy, than the wild type 13.1.2 are listed in Table 20.1. The wild type data are listed at the bottom for comparison. All energies are in the units of kcal/mol.<br>
TABLE 20.1<br>
(Table Removed)<br><br>
 (0306] The mutants listed in Table 20.1 could be candidates for engineering of affinity-improved antibodies. For the top 14 candidates in the list, per residue contributions on the antigen side and on the antibody side were further analyzed to examine the impact of the proposed modifications. The 10 mutants selected for in vitro site-directed mutagenesis were Tyrl72Arg, ArglOlGlu, Leu99Asn, Leu99His, ArglOlAsp, Leu217Gln, Leu99Thr, Leu217Asn, ArglOlGln and Asn35Gly. The results can be seen in Example 21.<br>
EXAMPLE 21 SITE-DIRECTED MUTAGENESIS CONFIRMING THE MODEL FOR 13.1.2<br>
[0307] This example demonstrates one method by which the above models, which suggest residues that are important in binding, can be tested. The Example also results in several antibody variants. Antibody variants of 13.1.2 were generated by single residue mutations introduced into the heavy and the light chains of the 13.1.2 mAb. The variants were analyzed to determine the contribution that the various side chains had in antigen binding. A list of the mutations introduced by site directed mutagenesis are summarized in Table 21.1.<br>
TABLE 21.1<br>
(Table Removed)<br><br>
[0308] Each of the 10 mutations in Table 21.1 was introduced into the heavy or light chain of the 13.1.2 mAb. Each mutated chain was then transfected with the complementary wild-type chain in 293 cells. Supernatants were then tested for expression and secretion of human IgG<br>
antibodies, and for binding to EGFrVlll antigen.   The results, as determined by an ELISA, are summarized in Table 21.2.<br>
TABLE 21.2<br>
(Table Removed) <br><br><br>
EXAMPLE 22 PREPARATION OF EGFRVIII/PFLAG VARIANT CONSTRUCT<br>
[0310| This example demonstrates how a variant to EGFRvIII can be made. A 1092bp fragment encoding the extracellular domain of EGFRvIII was generated with primer pairs 9712 and 9713 (Qiagen, Valencia, CA):<br>
Primer # 9712: 5'-ataaaagcttctggaggaaaagaaaggtaatta-3' (sense) (SEQ ID NO 128)<br>
Primer # 9713: 5'-TTA'ITGGTACCTCAGGCGATGGACGGGATCTTA- 3' (antisense) (SEQ ID NO 129)<br>
from plasmid template EGFRvIII-rbIgG/pCEP4 (as described above) amplified using Pfu DNA polymerasc enzyme (Strat:igenc, l.u Jollc, CA). Primer #9712 introduced a Hindlll site and primer # 9713 introduced a Kpnl site. 1 he 1'C'R pioduct was column purified (Qiagen column purification kit, Valencia, CA) digested with Hindlll and Kpnl (NEB, New England Biolabs, Beverly, Mass.) and gel purified (Qiagen gel purification kit, Valencia, CA). Fragments were ligated with T4 DNA Ligase (NEB, New England Biolabs, Beverly, Mass.) into pFLAG-CMV-1 (Sigma, St. Louis, MO) linearized with Hindlfl and Kpnl (NEB, New England Biolabs, Beverly, Mass.). The resulting vector was designated EGFRvIiypFLAG-CMV-1 # 1.<br>
EXAMPLE 23 PREPARATION OF EgfrViii/PFLAG RECOMBINANT PROTEIN<br>
[0311) This example demonstrates how a variant EGFRvIII protein can be made. First, 500 ug of EGFRvIII/pFLAG-CMV-l# 1 plasmid DNA was resuspended in 25 ml of Opti-MEMI (Invitrogen, Burlington, ON) and combined with 500 ul of 293fectin (Invitrogen, Burlington, ON) resuspended in 25 ml of Opti-MEMI. The mixture was incubated for 20 min at room temperature then mixed with 293T cells (IxlO9) prepared n 1L 293 FreeStyle media (Invitrogen, Burlington, ON), supplemented with 2% FBS and 50 µg/ml 0418 (Invitrogen, Burlington, ON). Cells are grown for 7 days at 37°C in 8% CO, with shaking at 125 rpm.<br>
[0312] EGFRvIlI-FLAG fusion protein purification was carried out with Anti-FLAG M2 Affinity Chromatography kit (Sigma, St. Louis, MO) according to the manufacture's protocol.<br>
(0313) Monomeric fusion protein was produced as follows. First, purified protein (1508 ug), was reduced with DTT in a final concentration of 10 mM for 30 minutes at 55 °C. Then IAA (iodoacetic acid) (Sigma, St. Louis, MO) was added to 22 mM and incubated 15 minutes at room temperature in the dark then dialyzed against PBS at 4 °C in 7k MWCO dialysis cassettes (Pierce, Rockford, 111.).<br>
EXAMPLES 24-30 BINDING STUPES OF ANTIBODY VARIANTS<br>
[0314)        The   following   examples   involve   Biacore   experiments   (surface   plasmon resonance) and KinExA experiments.   These examples demonstrate how one can test the various antibodies and variants thereof produced by the above examples to determine if they have the desired binding characteristics. All of the variants examined were variants in the 13.1.2 background. Instrumentation<br>
[0315]        All surface plasmon resonance experiments were performed using Biacore 2000 optical biosensors (Biacore, Inc., Piscataway, NJ).   All Kinetic Exclusion Assays were performed using a KinExA 3000 instrument (Sapidyne Instruments, Inc., Boise, ID). Reagents<br>
10316)Pep-3, NH2-LEEKKGNYVVTNYVVDUG-OH (MW - 1590 Da) (Sl-Q 11) NO. 130), was custom synthesized and purchased from Anatcch, Inc. (San Jose, CA). All mAbs were prepared in-house. The antigen EGFRvlIlpflag (iodoacetic acid reacted in order to block aggregation through free sulfhdryl groups), MW 39,907, was prepared in-house. Bovine serum albumin (BSA) fraction V (#BP1605-100) was purchased from Fisher Scientific (Pittsburgh, PA). All other general reagents were purchased from Sigma-Aldrich, Inc (St. Louis, MO).<br>
[0317] All antigen and mAb samples for Biacore and KinExA analysis were prepared in vacuum-degassed HBS-P buffer (0.01 M HEPES, 0.15 M NaCI, 0.005% surfactant P-20, Biacore Inc., Uppsala, Sweden) containing 100 µg/mL BSA. Biacore amine-coupling reagents, l-ethyl-3-(3-dimethylaminopropyl) carbodiimide (EDC), N-hydroxysuccmimide (NHS), and ethanolamine were purchased from Biacore, Inc. Biacore surface regeneration was with a 12 second pulse of 26 mM NaOH for the pep-3/mAb 131 experiment. All other mAbs dissociated to baseline within 20 minutes. Research grade CMS biosensor chips were purchased from Biacore, Inc.<br>
[0318) The KinExA detection antibody was Cy5-labeled goat anti-human IgG, Fey specific (Jackson ImmunoResearch Laboratories, Inc., West Grove, PA, #109-175-008) and was diluted 1000-fold in HEPES buffer (0.01 M HEPES, 0.15 M NaCI, pH 7.2) from a 0.5 mg/mL stock (1 X PBS, pH 7.4). The solid phase particles used for the KinExA experiments were NHS-activated Sepharose 4 Fast Flow beads (Pharmacia Biotech AB, Uppsala, Sweden, #17-0906-01). Prior toreacting the sepharose beads with antigen, a bead stock aliquot of 1.5 mL in a microcentrifuge tube was spun down and washed at least six times with cold deionized H20. After rinsing the beads once with sodium carbonate buffer (0.05 M, pH 9.3), antigen (-40 µg) in sodium carbonate buffer was added to the sepharose beads. The sepharose/antigen tube was rocked overnight at 4°C. After rocking, the sepharose was spun and rinsed twice with 1 M Tris buffer, pH 8.3. The antigen-coated beads were then rocked for 1 hour at room temperature in 1 M Tris buffer with 2% BSA. Biacore Measurements<br>
[0319]	Standard   EDC/NHS   and  carbohydrate  coupling  was  used  to  covalently<br>
immobilize mAbs to a CMS sensor chip.   To minimize mass transport and crowding mAbs were<br>
immobilized at levels that gave a maximum antigen binding response (Rmax) of no more than 50 -<br>
100 RU. A reference flow cell on each chip was activated and blocked with no mAb immobilization<br>
to serve as a control.<br>
(0320)	All Biacore kinetic experiments were conducted at 23°C. For each experiment, a<br>
series of six to eight antigen concentrations (starting with 1.01 u.M pep-3) was prepared using 2-fold<br>
dilutions.   Antigen samples were randomly injected over the biosensor surface in triplicate at 100<br>
µL/min.   Several buffer blanks were injected intermittently over the course of an experiment for<br>
double referencing. Each pep-3 concentration and blank were injected for 90 seconds. Dissociation<br>
was followed for 13 to 180 minutes. Dissociation data for pep-3 binding to mAb 131 were acquired<br>
by alternating three additional injections of 251 nM pep-3 with three additional blank injections and<br>
following the dissociation phase for 3-4 hours.<br>
[0321] All Biacore sensorgrams were processed using Scrubber software (Version l.lf, BioLogic Software, Australia). Sensorgrams were first zeroed on the y-axis and then x-aligned at the beginning of the injection. Bulk refractive index changes were .removed by subtracting the reference flow cell responses. The average response of all blank injections was subtracted from all analyte and blank sensorgrams to remove systematic artifacts between the experimental and reference flow cells. CLAMP biosensor data analysis software (Version 3.40, BioLogic Software, Australia) was used to determine k, and k,) from the processed data sets. Data from all flow cells were globally fit to a 1:1 bimolecular binding model that included a mass transport tenn. For several of the mAbs the injections corresponding to the first or second concentration of the pep-3 series were excluded in the nonlinear kinetic fit where it was obvious that the sensorgrams were not described well by a 1:1 interaction model. The KD was calculated from the quotient kd/ka. KinExA Equilibrium Measurements<br>
[0322] All KinExA experiments were conducted at room temperature (~23°C). For all equilibrium experiments, antigen was serially ciluted into solutions having a constant mAb binding site concentration. For the first 10 titration points the dilutions were 2-fold and the IIth and 12"1 serial dilutions were 10-fold. The sample flow rate for all experiments was 0.25 mL/min and the<br>
labeling antibody flow rate was 0.5 mL/min. Antigen/antibody samples were then allowed to reach equilibrium, which took ~48-72 hr to reach. For the pep-3/mAb 131 KinExA experiment the starting concentration of pep-3 in the Ko-controlled titration was 352 nM and the constant [mAb binding site] = 219 pM; for the mAb-controlled titration the starting [pep-3] = 251 nM and the [mAb binding site] = 11 nM. During the KD-controlled experiment with pep-3/mAb 131, 1.25 mL of each sample was drawn through the flow cell. A sample volume of 250 uL was analyzed for the antibody-controlled experiment. Two or three replicates of each sample were measured for all equilibrium experiments. The equilibrium titration data were fit in a dual curve analysis to a 1:1 binding model using KinExA software (Version 2.4, Sapidyne Instruments).<br>
[0323] The EGFRvlllpflag/mAb 131 complex was studied with KinExA under KD-controlled conditions only. The starting [EGFRvIIIpflag] was 198 nM and the [mAb binding site] was 150 pM. A sample volume of 1 mL was drawn through the flow cell. Duplicate measurements were collected for all samples. The equilibrium titration data were fit in a dual curve analysis to a 1:1 binding model using KinExA software (Version 2.4, Sapidyne Instruments). See Example 28 below for results and predicted equilibrium constant.<br>
[0324] For the KinExA titrations of the EGFRvIIIpflag/mAb 13.1.2 complex the starting concentration of EGFRvDI was 5.26 u,M (mAb-controlled), 230.1 nM (KD-controlled) and [mAb binding site] = 9.59 nM (mAb-controlled), 498 pM (KD-controlled). During the KD-controlled experiment, 1.30 mL of each sample was drawn through the flow cell. A sample volume of 250 uL was analyzed for the antibody-controlled experiment. Two or three replicates of each sample were measured for all equilibrium experiments. The equilibrium titration data were fit in a dual curve analysis to a 1:1 binding model using KinExA software (Version 2.4, Sapidyne Instruments).<br>
EXAMPLE 24 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR ANTIBODIES<br>
[0325] The binding kinetics of the wild type mAb 131 was observed by using a Surface Plasmon Resonance (SPR) instrument from Biacore. The KD was very low, 380 pM, owing to the very slow kj and a rapid ka. Estimates for the other kinetic parameters, derived from curve fitting, were ^=2.246*106 and kd=8.502*10'4.<br>
[0326] In one embodiment, improved or variant antibodies with improved kinetics are taught. By improved kinetics, it is meant that one of the kinetic elements of antibody binding to an epitope is superior to the same element in previously known antibodies for the same epitope. For example, an antibody that binds to pep-3 with a KD of greater (in binding ability) than 1.3*10"' M would be an improved antibody. As such, antibodies with a KD of less than 500 nM, 500-300 nM, 300-lOOnM, 100-lnM, 1.3 nM, 1.3 nM to 1000 pM, 1000 pM to 900 pM, 900-500pM, 500-400 pM, 400-300 pM, 300-100 pM, 100-50 pM, 50-1 pM, or smaller KD are contemplated.<br>
EXAMPLE 25<br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR ANTIBODIES <br>
[0327]         Similar to Example 24, the binding kinetics of mAb!3.1.2 to Pep-3 (EGFRvIII epitope) were examined.   The estimated KD was 67nM, but varied slightly between experiments. Estimates, for the other kinetic parameters, derived from curve fitting, were k,=2.835*105 and k,d=0.01922.<br>
EXAMPLE 26<br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR ANTIBODIES <br>
(0328)        Similar to Example 24, the binding kinetics of mAb 095 to Pep-3 (EGFRvIII epitope) were examined.   The estimated KD was 66nM.    Estimates, for the kinetic parameters, derived from curve fitting, were k,=1.491»105 and k<i></i>
EXAMPLE 27<br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR ANTIBODIES <br>
[0329]        Similar to Example 24, the binding kinetics of mAb 139 to Pep-3 (EGFRvIII epitope) were examined.   The estimated KD was 290nM.   Estimates, for the kinetic parameters, derived from curve fitting, were ka=10328 and kd==2.981*10"3.<br>
EXAMPLE 28<br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR ANTIBODIES <br>
[0330]        In order to more fully analyze the binding characteristics of the antibodies, KinExA experiments were performed to determine the binding characteristics of the mAb 131. The KD determined from a dual curve analysis was 1.74*10-10.   In a KinExA experiment, the KD for EGFRvlIIpflag to mAb 131 was 6.266* 10'".<br>
EX AMPLE 29 JN VITRO DETERMINATION OF BINDING CONSTANTS FOR VARIANT ANTIBODIES<br>
[0331) In order to more fully analyze the binding characteristics of the 13.1.2 antibodies, a KinExA experiment was performed to deteremine the binding characteristics of the mAb 13.1.2. The KD determined from a dual curve analysis was 7.538*10"'°. Additionally, the antigen in this example was the LGI'Kvlllpllag variant and was reacted with iodoacetic acid (IAA).<br>
EXAMPLE 30<br>
COMPARISON OF B1ACORE RESULTS AND KINEXA RESULTS (0332)        The results of the previous Examples and the KinExA tests are presented in Table 30.1 below.   Numbers in parentheses in Table 30.1 are 95% confidence intervals.   "ND," means not determined and "*" denotes binding to EGFRvlIIpflag (iodoacetic acid reacted), instead of pep-3.<br>
[0333] As is evidenced by the ra.c. constants, mAb 131 appears to have the greatest association constant and the lowest dissociation constant, thus giving mAb 131 the lowest KD-<br>
TABLE30.1<br>
(Table Removed)<br>
EXAMPLE 31<br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR L991-S.3 VARIANT<br>
ANTIBODIES<br>
[0334] The binding kinetics of mAb L99T-5.3 to Pep-3 (EGFRvIII epitope) were examined. 1 he lirsl slcj) was ID iiiiiiioliilr/c 5,(&gt;()() resonance units (Mi).to H.OOO RU ufinAb L99T-5.3 to two flow cells (Fc) of a CMS sensor chip and 5,600 resonance units (RU) to 8,000 RU of mAb 13.1.2 to one Fc using standard EDC/NHS coupling chemistry. This surface density yielded a binding signal with pep-3 of less than 100 RU, Two CMS sensor chips were used in total to immobilize both mAbs. With the previously collected data, this produced a total of 5 independent experiments for both antibodies that allows the 95% confidence intervals to be calculated. Biacore 2000 optical biosensors were used for all studies.<br>
[0335] Next, pep-3 was flowed across the mAb immobilized biosensor surfaces. The starting concentration of pep-3 was 1.25 µM, which was followed with eight two-fold serial dilutions in randomized triplicate injections. Blank injections were run every sixth sample throughout the injection series for double referencing purposes.<br>
[0336) Finally, the biosensor data was processed with Scrubber and the data was fit to curves utilizing Clamp with a 1:1 interaction model with a term included for mass transport. The high concentration injections, 1.25 µM, were excluded from the kinetic fits because it was apparent that the data was not consistent with a 1:1 interaction model. Most likely, this deviation is caused by non-specific interactions occurring at high concentrations of pep-3. All the kinetic data fit a 1:1 interaction model satisfactorily.<br>
[0337]        The estimated KD varied from 54-70 nM.    Estimates, for the other kinetic parameters, which also varied slightly between runs, were ka=2.238*105 and kd=0.01217. Examples 32-38<br>
[0338] Examples 32-38 further examined the binding kinetics of the variant mAbs through the use of a Biacore device. The first step in these examples involved the immobilization of 5,600 resonance units (RU) to 8,000 RU of each mAb tested to one flow cell (Fc) of a CMS sensor chip using standard EDC/NHS coupling chemistry. This surface density yielded a binding signal with pep-3 of less than 100 RU. Three CM5 sensor chips were used in total to immobilize all mutant<br><br>
mAbs with a unique mAb immobilized to each flow cell. MAb 13.1.2 was included on one flow cell for two out of the three CMS sensor chips. Biacore 2000 optical biosensors were used for all studies. [0339] Next, pep-3 was run across the mAb immobilized biosensor surfaces. The starting concentration of pep-3 was 4.98 µM, followed by eight to eleven two-fold serial dilutions in randomized duplicate or triplicate injections. Blank injections were run every sixth sample throughout the injection series for double referencing purposes.<br>
[0340] Finally, the biosensor data was processed with Scrubber and fitted utilizing Clamp with a 1:1 interaction model wi.th a term included for mass transport. Some high concentration injections (4.98 - 1.25 uM), depending upon the mAb and its affinity, were excluded from the kinetic fits when it was apparent that the data was not consistent with a 1:1 interaction model. Most likely, this deviation is caused by non-specific interactions occurring at high concentrations of pep-3. All the kinetic data fit a 1:1 interaction model.<br>
EXAMPLE 32 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR L217q-10.1 VARIANT<br>
ANTIBODIES<br>
[0341] The binding kinetics of mAb L217Q-10.1 to Pep-3 (EGFRvIIl epitope) were examined. The estimated KD was 92 nM Estimates, for the other kinetic parameters, derived from curve fitting, were ka=2.04*105 and kd-0.01885.<br>
EXAMPLE 33 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR L217n-2.1 VARIANT<br>
ANTIBODIES<br>
|0342] Similar to Example 32, the binding kinetics of mAb L217N-2.1 to Pep-3 (EGFRvllI epitope) were examined. The estimated KD was 185 nM. Estimates, for the other kinetic parameters, derived from curve fitting, were k,=2.198*105 and kd=0.04069.<br>
EXAMPLE 34 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR N35g-3.1 VARIANT<br>
ANTIBODIES<br>
J03431 Similar to Example 32, the binding kinetics of mAb N35G-3.1 to Pep-3 (EGFRvllI epitope) were examined. The estimated KD was 204 nM. Estimates, for the other kinetic parameters, derived from curve fitting, were k,=1.497*105 and kd=0.03057.<br>
EXAMPLE 35 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR VARIANT ANTIBODIES<br>
[0344] Similar to Example 32, the binding kinetics of mAb L99H-9.2 to Pep-3 (EGFRvIIl spitope) were examined. The estimated KD was 395 nM. Estimates, for the other kinetic parameters, derived from curve fitting, were k,-83390 and kd=0.03293.<br>
EXAMPLE 36<br><br>
IN VITRO DETERMINATION OF BINDING CONSTANTS FOR VARIANT ANTIBODIES<br>
[0345] Similar to Example 32, the binding kinetics of mAb Y172R-1.2 to Pep-3 (EGFRvIlI epitope) were examined. The estimated KD was 927 nM. Estimates, for the other kinetic parameters, derived from curve fitting, were ka=82237 and kd=0.07622,<br>
EXAMPLE 37 IN VITRO DETERMINATION OF BINDING CONSTANTS FOR VARIANT ANTIBODIES<br>
(03461 Similar to Example 32, the binding kinetics of mAb L99N-4.1 to Pep-3 (EGFRvIlI epitope) were examined. The estimated KD was 1.4 µM. MAb L99N-4.1 was fit using a steady-state (equilibrium) binding model in order to determine the KD because the kinetics were too fast to be fitted.<br>
EXAMPLE 38<br>
COMPARISON OF 13.1.2 WITH DESIGNED VARIANTS<br>
(0347] As can be seen in Table 38.1 a mAb with improved binding characteristics was developed. The 95% confidence intervals are shown in parentheses. L99T-5.3 exhibited an enhanced k,, a decreased kd and thus a slower KD overall. While statistically there appears to be little if any significant difference in the equilibrium dissociation constants and kinetic rate constants of Pep-3 binding to mAbs 13.1.2 and L99T-5.3 (at the 95% confidence interval), there still seems to be an intuitive bias for a marginal increase in affinity for Pep-3 binding to L99T-5.3. Moreover, when the same biosensor chip was used, L99T-5.3 seemed to always have a higher affinity than 13.1.2.<br>
TABLE 38.<br>
(Table Removed)<br><br>
Additional Docking Models and Methods of Selecting Models and Predicting Binding Affinity<br>
[0348] In other embodiments, the examples described above can be performed with various length peptides rather than just peptides that are 6 amino acids in length, as long as the key binding residues are included in the peptide. For example, instead of the six amino acid peptide,<br>
EEKKGN (SEQ ID NO: 127), a seven amino acid peptide, EEKKGNY (SEQ ID NO: 131) can be used. Any size peptide for the epitopc can be used. In other embodiments, the peptide is selected from the following peptides: LEEKKGNYWTDHC (SEQ ID NO: 56), LEEKKGNYVVTD (SEQ ID NO: 59), LEEKKGNYWT (SEQ ID N0:132), and EEKKGNYVVT (SEQ ID NO:57). Any sized peptide between the short fragments disclosed herein, to the full length peptide, or variants thereof, can be used.<br>
|0349] As appreciated by one of skill in the art, the presence of additional amino acids can alter the manner in which the peptide binds to the antibody. Not only does the presence of the additional amino acid allow for alternative and additional bonds to be formed between the peptide and the antibody, but the additional amino acid can change the structure of the peptide and the structure of the antibody upon binding of the peptide with the antibody. Thus, in one embodiment, various lengths of the epitope peptide, e.g. EEKKGN (SEQ ID NO: 127) and EEKK.GNY (SEQ ID NO: 131), can be examined for binding properties and binding optimization. Not only will the longer fragments of the peptide provide an accurate depiction of the peptide-antibody interaction for longer segments of the peptide, but an examination of the changes in binding strength and the residues involved in binding will allow additional information concerning longer peptides to be extrapolated from the data.<br>
[0350] In addition, and perhaps complementary to the testing of longer peptide fragments, additional filtering steps can be performed on the various docking models in order to select a refined docking model. An additional filtering step can allow one to filter through numerous docking models to find those that are consistent with available experimental data.<br>
[0351) In one embodiment, the filter is based on fine-resolution epitope mapping data, e.g., the experimentally characterized individual residue binding profile, which could be correlated with the computed binding energy profile for each amino acid in the peptide. The binding energy profile of a seven amino acid peptide, for example, can be used to select docking models that contain similar binding energy profiles. A binding energy profile is an assignment of the binding energy of each ammo acid in a peptide to the particular ammo acid to create a profile of the peptide in terms of each amino acid's binding energy in that model. For example, in one docking model, given a peptide comprising amino acids A and B, where A has a binding energy of-5 and B has a binding energy of-20, one would have a profile of Al (at -5) and B2 (at -20). This profile could be used as a filter to select other docking models. For example, the use of this binding energy profile as a filter or "template" would result in other docking models being selected if the peptide in the candidate model liiid a relatively low value attributed to positioi A, and a relatively high (larger negative, higher absolute value) value attributed to position B. In an alternative embodiment, the template requires additional limitations; for example, that the value at position B is four fold higher than the value at position A.<br>
[0352] One can compare the binding energy profile template with the profiles of the peptide in the other docking models in a variety of ways. If the binding energy profile template is similar to the desired binding energy profile, then the filter can be used to pick out favorable docking models for further examination. If the binding energy profile template is dissimilar to the desired binding energy profile, then the filter can be used to eliminate unfavorable docking models. In one embodiment, the filtering process includes a template with both favorable and unfavorable binding energies and the filter is used to both select and exclude docking models. As appreciated by one of skill in the art, there are many possible different binding energy profiles, and thus many different binding energy profile templates that can be used depending upon the situation.<br>
[0353] In one embodiment, one can define a binding energy profile template as a template that has a series of relatively high binding energies at particular positions in the peptide. In a preferred embodiment, the binding energy profile template, and the binding energy profile selected by the template, will have relatively high binding energy at position 2, 4, or 6 of the peptide, EEKKGNY (SEQ ID NO: 131). In another embodiment, the binding energy profile template will have a relatively high binding energy at positions 2, 4, and 6 of the peptide EEKKGNY (SEQ ID NO: 131). In another embodiment, the binding energy profile template will have a relatively low binding energy attributed to position 3 of the peptide EEKKGNY (SEQ ID NO: 131). In the above discussion, the positions are assigned as follows: El, E2, K3, K4 G5, N6, Y7.<br>
(0354] In one embodiment, the filtering process first involves a comparison of the binding energies at K3 and K4. Docking models that result in a relatively higher binding energy for K4 compared to K3 are selected, while docking models that result in a lower binding energy for K4 compared lo K3 arc filtered out. Thus, by "relatively high," it is mount that the binding energy for K4 is greater (more negative value, larger absolute value) than K3. Next, the docking models are again filtered through the binding energy profile template, this time, those binding models with relatively higher energies at positions 2, 4, and 6 are selected for, while the other models can be removed. Thus, by "relatively high," it is meant that the binding energy at positions 2, 4, and 6 are higher (more negative value, larger absolute value) than the lowest binding energy in the peptide. Thus, in this embodiment, the binding energy profile template could be summarized as follows: El can be any value, E2 should be greater than the lowest value, K3 should be less than K4, K4 should be greater than the lowest value, G5 can be any value, N6 should be greater than the lowest value, Y7 can be any value. Thus, El, G5, and Y7 could be any value, as long as at least one (or K3) is lower than at least one of E2, K4, and N6. 'In another embodiment, "relatively high" can be set to a standard value as determined through modeling or experimentation. In one embodiment, that the docking models pass .the first filter is more important than the docking model pass the second filtering step. As appreciated by one of skill in the art, one need not perform these two steps sequentially and they can be performed simultaneously.<br>
[0355] Additionally, these profile templates for filtering through results will vary depending upon the pcptide, the antibody, and the binding conditions. One of skill in the art, given the present disclosure, especially with reference to Example 14, could determine the appropriate binding energy profile template. For example, as shown in Table 14.1, there are several possible important residues for peptide binding both in the 131 and in the 13.1.2 antibody. In the 131 mAb, positions E2, K4, N6, and Y7 are important for the particular peptide tested. In the 13.1.2 mAb, positions El, E2, K4, G5, and N6 are important for the particular peptide tested. Those residues that are important can be residues involved in the creation of a binding energy profile template. As clear from the discussion below, the binding energy profile template in Example 39 appears to be different from that suggested by an analysis of Example 14. Example 39 is a less stringent version of the template that allows more models to pass through the screening step. If one wanted to reduce the number of models that made it through the screening step, one could further add requirements concerning El and G5.<br>
[0356) The following example demonstrates both the use of a longer peptide, how it can alter the results demonstrated above, what such changes can mean, as well as demonstrating the use of one of the above filters for selecting particular docking models.<br>
EXAMPLE 39 EPITOPE-ANTIBODY DOCKING MODEL FOR A SEVEN AMFNO ACID PEPTIDE<br>
(0357] This example demonstrates the generation of a set of docking models for a seven-residue peptide complcxed to the CDR region of 13.1.2 structure model. Additionally, this example demonstrates methods for selecting one docking model over another docking model.<br>
[0358] First, a structural model for the seven-residue peptide EEKKGNY (SEQ ID NO: 131) was built in an extended conformation and energy minimized with Discover 3 module in Insightll modeling package. Next, the peptide structure was manually placed into the combining site to form an initial assembly. A Monte Carlo search in the translational and rotational spaces was then automatically performed with relaxed energy constraints in the Docking module in Insightll. During the docking process, the residues within 5 Angstroms of the binding groove were allowed to move while other antibody residues were fixed. The peptide was constrained to within 5 Angstroms of the starting position. The plausible configurations found by the Monte Carlo search were followed by simulated annealing and energy minimization to reach the final complex structure models. A total of 63 docking models were obtained.<br>
[0359) For each docking model, the interaction energy between the antibody and the individual residue in the peptide was calculated with Discover_3. The profile of individual residue contribution in epitope-antibody binding was inspected to select the docking models that are consistent with the fine-resolution epitope mapping data, i.e., "the binding energy profile template." 19 out of 63 docking models passed this check. A typical individual residue binding energy profile is shown in Table 39.1. Consistent with the epitope mapping data, in Example 14, the binding energy<br>
for K4 is prominent, and those for N6 and E2 are large. This binding energy profile template placed particular emphasis on the fact that K4 is greater than K3. This binding energy profile also placed an emphasis on the requirement that E2, K4, and N6 are relatively large. In other words, the binding energies of E2, K.4, and N6 were not the lowest (least negative or smallest absolute value) binding energies in the peptide.<br>
TABLE 39.1.<br>
'BINDING ENERGY PROFILE FOR INDIVIDUAL RESIDUE IN THE SEVEN-RESIDUE PEPTTDE TO ANTIBODY 13.1.2 IS CONSISTENT WITH EPITQPE MAPPING DATA IN<br>
(Table Removed)<br>
[0360] For the 19 models that passed the filter based on the binding energy profile, epitope-antibody binding energetics simulations were performed on each of the seven mutants with affinity data (Tyrl72Arg, Example 36; Leu217Asn, Example 33; Leu217Gln, Example 32; Asn35Gly, Example 34; Leu99Asn, Example 37; Leu99His, Example 35; and Leu99Thr, Example 31). Since the extent of electrostatic interaction in this complex had to be approximated, a number of different dielectric constants were used in a series of calculations. The mutation was done with residue replacement followed by 30-100 steps of energy minimization to account for any local conformational changes that are induced by the side chain change. For each docking model, the interaction energy between the seven-residue peptide and the whole antibody was calculated for each mutant for the selected parameters. For each set of 8 binding energies (7 mutants plus the wild type), a linear fitting procedure was done on each set of the binding data, in comparison with the logarithm of Kd. A correlation coefficient was calculated for each linear fitting. The best correlation was obtained for one model, the model with the data described in Table 39.1, with a dielectric constant 1 *r and 50 steps of energy minimization. The epitope-antibody binding energies for this model are shown in Table 39.2. The correlation coefficient was 0.80 with all the data. As the KD for Leu99Asn was not measured with high degree of accuracy, see Example 37 above, a separate linear fitting was performed excluding the data for Leu99Asn. An excellent correlation coefficient of 0.91 was obtained, as shown in FIG. 14. The refined docking model is thus well represented by the above selected model. The model with space-filling peptide is shown in FIG. 15, and the hydrogen bonds are shown in FIG. 16. L3 150 is the lower section and H3 160 is the upper section on FIG. 16. H2 140 is to the right of the peptide binding area. The peptide itself is placed into the binding site with El being positioned at the top of the page in a light shade, down through K3, K4, G5, N6, and Y7, progressively getting darker. The antibody residues involved in hydrogen bonding are shown in FIG.<br><br>
16.   The model produced from this example demonstrates that there are seven hydrogen bonds: K4...Q95, K4...Q95,N6...Q98,G5...H3l,Y7...H31,andY7...W165.<br>
TABLE 39.2. SIMULATION OF EPITOPE-ANTIBODY BINDING ENERGETICS. IN COMPARISON<br>
(Table Removed)<br>
(0361] As can be seen from the model selected in Example 39, which is represented in FIG. 15, the docking model revealed some unexpected results. One interesting result is that while residues E2, K4, and N6 are important residues in the binding of the peptide as a whole, not all of these amino acids are modeled as involved in forming H-bonds with the antibody. It appears that K4 is involved in the formation of two H-bonds, both with Q95, which is consistent with K4's importance in the binding energy profile and profile template. It also appears that N6 is modeled to bond to Q98; however, in this particular model, E2 does not appear to form H-bonds in the model. One interesting trend that is consistent is that each of the key residues from the binding energy profile template (e.g., E2, K4, and N6) are mostly buried and thus in close contact with the antibody binding groove. Thus, this docking model selection can account for the fact that these key residues are important because of their close interaction with the antibody. Additionally, it is possible that El is involved in a hydrogen bond with YV214.<br>
[0362] Example 39 also demonstrates that the above described method results in a strong correlation between binding energy and KD, suggesting that models created by this method will also allow optimization or at least a prediction of the KD of the antibody-peptide complex.<br>
[0363) As can be seen from a comparison of Example 39 and Example 19, there are some residues that are important between the two models, some residues that appear only in the seven amino acid docking model, as well as some residues that do not appear to be as important in the seven ammo acid docking model. For example, the seven peptide epitope appears to create H bonds between K4...Q95, K4...Q95, N6...Q98, G5...H31, Y7...H31, and Y7...W165. On the other hand, the six peptide epitope appears to create H bonds between E2...Y172, K3...H31, K4...H31, N6...D33, N6...Y37, and N6...KS5. As can be seen from the above data, both the six and the seven amino acid peptide models emphasize the importance of H 31, as both models involve H31 forming rwo hydrogen bonds with the peptide. While there are other possible trends between the two data sets, it also appears that many of the binding interactions have changed from the six amino acid<br>
model to the seven amino acid model. However, these examples demonstrate that variations due to epitope size can be detected with these models and thus the scaling up from shorter to longer epitope peptides should not be problematic in light of the present disclosure. The presence of amino acids that consistently demonstrate their importance in various binding models allows one to bias the importance of the various interactions accordingly so that shorter peptide models can be more representative of longer peptide interactions.<br>
[0364] As appreciated by one of skill in the art, any of the above discussion or examples concerning the six ammo acid peptide, EEKKGN (SEQ ID NO: 127), can also be applied towards the seven amino acid peptide, EEKKGNY (SEQ ID NO: 131), or any longer peptide. For instance, Example 20 can be repeated with the information from Example 39 for rational design for affinity-improved antibodies by site-directed mutagenesis. Furthermore, Example 21 can be repeated, using the results of Example 39, following an attempt of rational design for affinity-improved antibodies by site-directed mutagenesis to test any new antibodies derived from Example 20.<br>
[0365] In one embodiment, the results from Example 39 are used to redefine the interaction area between the antibody and the peptide. For example, the paratope, for EEKKGNY (SEQ ID NO: 131), can be defined as including the other residues on the antibody that are predicted to interact with the peptide, for example, residue 95. Alternatively, as in Example 19, the paratope can be defined as all residues within 5 Angstroms of the docked peptide. In Silico Affinity Maturation In Different Proteins<br>
[0366] Antibody affinity maturation has been successfully done in vitro in a number of different studies. Typically, randomized mutant libraries need to be constructed by molecular biology methods and selection/screening assays need to be developed to enrich the clones with good binding capability. Selected variants then need to be purified to determine affinities. This process requires a scries of lengthy and laborious experiments. The following example demonstrates that it is possible to accurately predict affinity maturation through in silica selection utilizing an antibody-antigen complex structure alone.<br>
EXAMPLE 40 IN S/I/CQjS.EFJNrTY MATURATION THROUGH ANTIBODY-ANTIGEN BINDING<br>
1'Nl-RGin'lCS SIMUI.ATIONS<br>
[0367] This Example demonstrates that in silica antibody-antigen binding energetics simulations can be used for affinity maturation. In particular, this example demonstrates that the binding kinetics of a Fab-12 (IgG form known as rhuMAb VEGF)-to VEGF (vascular endothclial growth factor) can be predicted through the above described in silica process.<br>
[0368| The crystal structure of the VEGF-Fab complex used was located in the PDB database with the accession number 1BJ1, at a resolution of 2.4 angstroms. Published experimental affinity data for a series of mutants of an anti-VEGF Fab were used to test the concept. The 3-D coordinates of the VEGF-Fab structure were used for carrying out in silica mutation for the following<br>
mutants: H97Y, SlOOaT, T28D, 28D31H, 28D31H97Y100aT, N31H, Y53W, 71I73K, 71V73V. The affinity data were obtained from the paper by Chen, Y ct al., (J Mol Biol., 293(4):865-81 (1999)). The energetics simulations were carried out between the various VEGF-Fab mutants, as described in Example 39. The results are listed in Table 40.1. The results from this example demonstrate that a significant correlation between the binding energy and affinity ranking was obtained through this process. The linear fitting of the binding energy versus logarithm of the relative affinity is shown in FIG. 17. The correlation coefficient of-0.91 indicates that the in silico simulation accurately captures the detailed interaction at the atomic level,<br>
TABLE 40.1. ANTIBODY-ANTIGEN BINDING ENERGY SIMULATION COMPARED WITH AFFINITY<br>
(Table Removed) <br><br>
 [0369]	As is clear from the Examples above, the simulation can be extrapolated to<br>
identify higher affinity mutants without the use of in vitro experimentation.  Additionally, it is clear<br>
that this approach is useful for different antibodies and for different peptides. This methodology can<br>
be generally applied to perform affinity maturation in silico, using only a high-resolution antibody-<br>
antigen complex structure.   In one embodiment, this use of in silico affinity maturation will save<br>
tremendous amounts of time and resource.<br>
EXAMPLE 41 DETERMINATION OF CANONICAL CLASSES OF ANTIBODIES<br>
(0370)	Cliotliiii, el ;il liuve described antibody structure in terms of "canonical classes"<br>
for the hypervanable regions of each immunoglobulin chain (J. Mol. Biol. 1987 Aug 20;196(4):901-<br>
17). The atomic structures of the Fab and VL fragments of a variety of immunoglobulins were<br>
analyzed to determine the relationship between their amino acid sequences and the three-dimensional<br>
structures of their antigen binding sites. Chothia, et al. found that there were relatively few residues<br>
that, through their packing, hydrogen bonding or the ability to assume unusual phi, psi or omega<br>
conformations, were primarily responsible for the main-chain conformations of the hypervariable<br>
regions. These residues were found to occur at sites within the hypervariable regions and in the<br>
conserved beta-sheet framework   By examining sequences of immunoglobulins having unknown<br>
structure, Chothia, et al show that many immunoglobuins have hypervariable regions that are similar<br>
in size to one of the known structures and additionally contained identical residues at the sites responsible for the observed conformation.<br>
(0371)	Their discovery implied that these hypervariable regions have conformations<br>
close to those in the known structures. For five of the hypervariable regions, the repertoire of<br>
conformations appeared to be limited to a relatively small number of discrete structural classes.<br>
These commonly occurring main-chain conformations of the hypervariable regions were termed<br>
"canonical structures". Further work by Chothia, et al. (Nature. 1989 Dec 21-28;342(6252):877-83)<br>
and others (Martin, et al. J Mol Biol. 1996 Nov 15;263(5):800-15) confirmed that that there is a small<br>
repertoire of main-chain conformations for at least five of the six hypervariable regions of antibodies.<br>
[0372]	Some of the antibodies described above were analyzed to determine the<br>
canonical class for each of the antibody's complementarity determining regions (CDRs).   As is<br>
known, canonical classes have only been assigned for CDR1 and CDR2 of the antibody heavy chain,<br>
along with CDR1, CDR2 and CDR3 of the antibody light chain. The table below (41.1) summarizes<br>
the results of the analysis.  The Canonical Class data is in the form of *HCDR1-HCDR2-LCDR1-<br>
LCDR2-LCDR3, wherein "HCDR" refers to the heavy chain CDR and "LCDR" refers to the light<br>
chain CDR.    Thus, for example, a canonical class of 1-3-2-1-5 refers to an antibody that has a<br>
HCDR1 that falls into canonical class 1, a HCDR2 that falls into canonical class 3, a LCDR1 that<br>
falls into canonical class 2, a LCDR2 that falls into canonical class 1, and a LCDR3 that falls into<br>
canonical class 5.<br>
TABLE 41.1<br>
(Table Removed) <br><br>
[0374] Each CDR (except for H3) was assigned to a canonical structure if it satisfies the length requirement and matches the key residues defined in the canonical class. The amino acids defined for each antibody can be found, for example, in the articles by Chothia, et al. referred to above.<br>
[0375] The foregoing description and Examples detail certain preferred embodiments of the invention and describes the best mode contemplated by the inventors. It will be appreciated, however, that no matter how detailed the foregoing may appear in text, the invention may be practiced in many ways and the invention should be construed in accordance with the appended claims and any equivalents thereof.<br><br><br><br><br>
We Claim:<br>
1.	An isolated human monoclonal antibody that comprises a heavy chain<br>
polypeptide and a light chain polypeptide, wherein the antibody comprises<br>
a)	i) the heavy chain polypeptide comprising an amino acid sequence that is<br>
at least 90% identical to the amino acid sequence selected from the group consisting of:<br>
SEQ ID NO: 2 and SEQ ID NO: 142, and<br>
ii) the light chain polypeptide comprising an amino acid sequence that is at least 90% identical to the amino acid sequence selected from the group consisting of: SEQ ID NO: 19 and SEQ ID NO: 144;<br>
b)	the heavy chain polypeptide comprising the following complementarity determining regions (CDRs): a heavy chain CDR1 that is a CDR1 in SEQ ID NO: 142; a heavy chain CDR2 that is a CDR2 in SEQ ID NO: 142; a heavy chain CDR3 that is a CDR3 in SEQ ID NO: 142; and the light chain polypeptide comprising the following CDRs: a light chain CDR1 that is a CDR1 in SEQ ID NO: 144; a light chain CDR2 that a CDR2 in SEQ ID NO: 144; and a light chain CDR3 that is a CDR3 in SEQ ID NO: 144;<br>
c)	the heavy chain polypeptide comprising the following complementarity determining regions (CDRs): a heavy chain CDR1 that is a CDR1 in SEQ ID NO: 2; a heavy chain CDR2 that is a CDR2 in SEQ ID NO: 2; a heavy chain CDR3 that is a CDR3 in SEQ ID NO: 2; and the light chain polypeptide comprising the following CDRs: a light chain CDR1 that is a CDR1 in SEQ ID NO: 19; a light chain CDR2 that is a CDR2 in SEQ ID NO: 19; and a light chain CDR3 that is a CDR3 in SEQ ID NO: 19;<br>
d)	or any combination of a), b) or c).<br>
2.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises the amino acid sequence of SEQ ID NO: 2.<br>
3.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the light chain polypeptide comprises the amino acid sequence of SEQ ID NO: 19.<br>
4.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the antibody binds to a peptide that consists of the sequence LEEKKGNYVVTDHC (SEQ ID NO: 56).<br>
5.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises an amino acid sequence that is at least about 95% identical to the amino acid sequence of SEQ ID NO: 2 and the light chain polypeptide comprises an amino acid sequence that is at least about 95% identical to the amino acid sequence of SEQ ID NO: 19.<br>
6.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises at least one of the following amino acid sequences: SEQ ID NO: 108, SEQ ID NO: 110, and SEQ ID NO: 112.<br>
7.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 108, SEQ ID NO: 110, and SEQ ID NO: 112.<br>
8.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the light chain polypeptide comprises at least one of the following amino acid sequences: SEQ ID NO: 101, SEQ ID NO: 103, and SEQ ID NO: 105.<br>
9.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the light chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 101, SEQ ID NO: 103, and SEQ ID NO: 105.<br>
10.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises an amino acid sequence that is at least about 95% identical to the amino acid sequence of SEQ ID NO: 142 and the light chain polypeptide comprises an amino acid sequence that is at least about 95% identical to the amino acid sequence of SEQ ID NO: 144.<br>
11.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises at least one of the following amino acid sequences: SEQ ID NO: 121, SEQ ID NO: 123, and SEQ ID NO: 125.<br>
12.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the heavy chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 121, SEQ ID NO: 123, and SEQ ID NO: 125.<br>
13.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the light chain polypeptide comprises at least one of the following amino acid sequences: SEQ ID NO: 101, SEQ ID NO:  116, and SEQ ID NO: 118.<br>
14.	The isolated human monoclonal antibody as claimed in Claim 1, wherein the light chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 101, SEQ ID NO: 116, and SEQ ID NO: 118.<br>
15.	The isolated human monoclonal antibody as claimed in Claim 1 comprising:<br>
the heavy chain polypeptide comprising the following complementarity determining regions (CDRs): a heavy chain CDR1 that is a CDR1 in SEQ ID NO: 142; a heavy chain CDR2 that is a CDR2 in SEQ ID NO: 142; a heavy chain CDR3 that is a CDR3 in SEQ ID NO: 142; and<br>
the light chain polypeptide comprising the following CDRs: a light chain CDR1 that is a CDR1 in SEQ ID NO: 144; a light chain CDR2 that a CDR2 in SEQ ID NO: 144; and a light chain CDR3 that is a CDR3 in SEQ ID NO: 144.<br>
16.	The isolated human monoclonal antibody as claimed in Claim 15, wherein the heavy chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 121, SEQ ID NO: 123, and SEQ ID NO: 125.<br>
17.	The isolated human monoclonal antibody as claimed in Claim 15, wherein the light chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 101, SEQ ID NO: 116, and SEQ ID NO: 118.<br>
18.	The isolated human monoclonal antibody as claimed in Claim 1 comprising:<br>
the heavy chain polypeptide comprising the following complementarity determining regions (CDRs): a heavy chain CDR1 that is a CDR1 in SEQ ID NO: 2; a heavy chain CDR2 that is a CDR2 in SEQ ID NO: 2; a heavy chain CDR3 that is a CDR3 in SEQ ID NO: 2; and<br>
the light chain polypeptide comprising the following CDRs: a light chain CDR1 that is a CDR1 in SEQ ID NO: 19; a light chain CDR2 that is a CDR2 in SEQ ID NO: 19; and a light chain CDR3 that is a CDR3 in SEQ ID NO: 19.<br>
19.	The isolated human monoclonal antibody as claimed in Claim 18, wherein the heavy chain polypeptide comprises all three of the following amino acid sequences: SEQ ID NO: 108, SEQ ID NO: 110, and SEQ ID NO: 112.<br>
20.	The isolated human monoclonal antibody as claimed in Claim 18, wherein the light chain polypeptide comprises all three of the following sequences: SEQ ID NO: 101, SEQ ID NO:  103, and SEQ ID NO: 105.<br>
21.	An isolated human monoclonal antibody that binds to an epitope on an EGFRvIII<br>
protein, wherein said antibody binds to the amino acid sequence LEEKKGNYVVTDHC (SEQ<br>
ID NO: 56), wherein the second lysine in SEQ ID NO: 56 is part of said epitope.<br>
22.	An isolated human monoclonal antibody that binds to an EGFRvIII protein,<br>
wherein said antibody competes for binding to EGFRvIII with the antibody as claimed in Claim<br>
1.<br>
23.	A kit for the detection of EGFRvIII in mammalian tissues or cells comprising at<br>
least one antibody from Claim 1 to Claim 22 and Claim 28 to Claim 30 and means for indicating<br>
the binding of the antibody with EGFRvIII, if present.<br>
24.	The kit as claimed in Claim 23, wherein the antibody is a monoclonal antibody.<br>
25.	The kit as claimed in Claim 24, wherein the antibody is labeled.<br>
26.	The kit as claimed in Claim 23, wherein the antibody is an unlabeled first<br>
antibody and the means for indicating the reaction comprises a labeled second antibody that is an<br>
anti-immunoglobulin.<br>
27.	The kit as claimed in Claim 23, wherein the antibody is labeled with a marker<br>
selected from the group consisting of a fluorochrome, an enzyme, a Radionuclide and a<br>
radiopaque material.<br>
28.	The isolated human monoclonal antibody as claimed in Claim 1, wherein amino acids in the CDRs are defined by the method of Chothia.<br>
29.	The isolated human monoclonal antibody as claimed in Claim 1, wherein amino acids in the CDRs are defined by the method of Kabat.<br>
30.	The isolated human monoclonal antibody as claimed in Claim 29, wherein the amino acids in the CDRs include those defined by the method of Chothia and the method of Kabat.<br>
31.	A composition comprising:<br>
an antibody of at least one of the antibodies as claimed in Claim 1 to Claim 22 and Claim 28 to Claim 30; and<br>
a pharmaceutically acceptable carrier or diluent.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUFic3RyYWN0LSgwNy0xMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Abstract-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUFic3RyYWN0LSgyNC0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Abstract-(24-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNsYWltcy0oMDctMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Claims-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNsYWltcy0oMTAtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Claims-(10-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNsYWltcy0oMjUtMDMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Claims-(25-03-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNsYWltcy0oMzAtMDYtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Claims-(30-06-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDYtMjAxMSkuLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence Others-(10-06-2011)..pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence Others-(10-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence Others-(24-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(16-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(16-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(25-03-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMzAtMDYtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Correspondence-Others-(30-06-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LURyYXdpbmdzKDE2LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Drawings(16-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LURyYXdpbmdzLSgwNy0xMi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Drawings-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMS0oMDctMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-1-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMS0oMTctMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-1-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMS0oMjQtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-1-(24-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMTMtKDE2LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-13-(16-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMi0oMDctMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-2-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMi0oMTctMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-2-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMi0oMjQtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-2-(24-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMjYtKDE3LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-26-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMy0oMTAtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-3-(10-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMy0oMTYtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-3-(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tMy0oMjYtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-3-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUZvcm0tNS0oMDctMTItMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Form-5-(07-12-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LWZvcm0tNi0oMTctMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-form-6-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LUdQQS0oMTAtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-GPA-(10-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bWVudHMtKDE3LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Others-Documents-(17-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bWVudHMtKDI2LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Others-Documents-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LU90aGVycy1Eb2N1bW5lbnQoMTYtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Others-Documnent(16-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LVBDVC0yMTAtKDI2LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-PCT-210-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LVBDVC0yMjAtKDI2LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-PCT-220-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LVBDVC0yMzctKDI2LTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-PCT-237-(26-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1kZWxucC0yMDA1LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">6073-delnp-2005-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjA3My1ERUxOUC0yMDA1LVBldGl0aW9uLTEzNy0oMTAtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">6073-DELNP-2005-Petition-137-(10-06-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250294-method-for-the-management-and-reduction-of-the-incidence-and-intensity-of-microbial-resistance.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250296-a-process-for-optimizing-the-production-performance-of-a-spinning-machine.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250295</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6073/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>51/2011</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>23-Dec-2011</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>21-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AMGEN FREMONT INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>6701 KAISER DRIVE,FREMONT,CALIFORNIA 94555 USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RATHANASWAMI, PALANISWAMI</td>
											<td>7878 FRENCH STREET, VANCOUVER, BRITISH COLUMBIA V6P 4V7 (CA)</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAYA, ROBERT</td>
											<td>3367 GREENWOOD DRIVE, FREMONT, CALIFORNIA 94536 (US)</td>
										</tr>
										<tr>
											<td>3</td>
											<td>YANG, XIAO, DONG</td>
											<td>2833 BRYANT STREET, PALO ALTO, CALIFORNIA 94306 (US)</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CORVALAN, JOSE</td>
											<td>125 WILLIAMS STREET, FOSTER CITY CALIFORNIA 94404 (US)</td>
										</tr>
										<tr>
											<td>5</td>
											<td>FOLTZ, IAN</td>
											<td>2108 KNIGHTSWOOD PLACE, BURNABY, BRITISH COLUMBIA V5A 4B9 (CA)</td>
										</tr>
										<tr>
											<td>6</td>
											<td>JIA, XIAO-CHI</td>
											<td>22 MURRAY COURT, SAN MATEO, CALIFORNIA 94403 (US)</td>
										</tr>
										<tr>
											<td>7</td>
											<td>KANG, JASPAL</td>
											<td>16727 109 AVENUE SURREY, BRITISH COLUMBIA V4N 5H6 (CA)</td>
										</tr>
										<tr>
											<td>8</td>
											<td>KING, CHADWICK, T.</td>
											<td>1325 MOODY AVE, NORTH VANCOUVER BC, CANADA V7L 3T5</td>
										</tr>
										<tr>
											<td>9</td>
											<td>KLAKAMP, SCOTT, L.</td>
											<td>1160 CHOPIN TERRACE #301, FREMONT, CALIFORNIA 94538 (US)</td>
										</tr>
										<tr>
											<td>10</td>
											<td>SU, QIAOJUAN, JANE</td>
											<td>4073 FILAN WAY, SAN JOSE, CALIFORNIA 95135 (US)</td>
										</tr>
										<tr>
											<td>11</td>
											<td>WEBER, RICHARD</td>
											<td>2537 PACHECO STREET, SAN FRANCISCO, CALIFORNIA 94116 (US)</td>
										</tr>
										<tr>
											<td>12</td>
											<td>FENG, XIAO</td>
											<td>135 RED CEDAR LANE, UNION CITY, CALIFORNIA 94587 (US)</td>
										</tr>
										<tr>
											<td>13</td>
											<td>FOORD, ORIT</td>
											<td>712 CAYMAN LANE, FOSTER CITY, CALIFORNIA 94404 (US)</td>
										</tr>
										<tr>
											<td>14</td>
											<td>GREEN, LARRY</td>
											<td>464 HILL STREET, SAN FRANCISCO, CALIFORNIA 94114 (US)</td>
										</tr>
										<tr>
											<td>15</td>
											<td>GUDAS, JEAN</td>
											<td>36178 CRYSTAL SPRINGS DRIVE, NEWARK, CALIFORNIA 94560 (US)</td>
										</tr>
										<tr>
											<td>16</td>
											<td>KEYT, BRUCE</td>
											<td>1180 LAKEVIEW DRIVE, HILLSBOROUGH CALIFORNIA 94010 (US)</td>
										</tr>
										<tr>
											<td>17</td>
											<td>LIU, YING</td>
											<td>143 MONROE DR. PALO ALTO, CA 94306 (US)</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/020564</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-06-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/562,453</td>
									<td>2004-04-15</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/483,145</td>
									<td>2003-06-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/525/570</td>
									<td>2003-11-26</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250295-an-isolated-human-monoclonal-antibody-directed-against-egfrviii by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:31:04 GMT -->
</html>
