<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250332-an-in-vitro-method-for-enhancing-the-level-and-or-quality-of-protein-production by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:29:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250332:AN IN VITRO METHOD FOR ENHANCING THE LEVEL AND/OR QUALITY OF PROTEIN PRODUCTION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN VITRO METHOD FOR ENHANCING THE LEVEL AND/OR QUALITY OF PROTEIN PRODUCTION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed herein are methods and compositions for enhanced protein production and over expression from an exogenous sequence using engineered zinc finger proteins to inactivate first and second genes in a cell including the exogenous sequence.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> AN IN VITRO METHOD FOR ENHANCING THE LEVEL AND/OR QUALITY OF PROTEIN PRODUCTION<br><br>
WO 2006/033859 PCT/US2005/032157<br>
COMPOSITIONS AND METHODS FOR PROTEIN PRODUCTION<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
This application claims the benefit of United States provisional patent<br>
applications No. 60/610,853 (filed September 16, 2004) and 60/661,841 (filed March<br>
15,2005) the disclosures of which are incorporated by reference in their entireties for<br>
all purposes.<br>
STATEMENT OF RIGHTS TO INVENTIONS<br>
MADE UNDER FEDERALLY SPONSORED RESEARCH<br>
Not applicable.<br>
TECHNICAL FIELD<br>
The present disclosure is in the field of transcriptional regulation, particularly<br>
protein expression and overproduction, for example, large-scale production of<br>
therapeutic proteins.<br>
BACKGROUND<br>
Controlled regulation of transcription is useful in the fields of research,<br>
diagnostics and therapeutics. For example, regulation of transcription of a nucleotide<br>
sequence encoding a protein can facilitate production of large amounts of<br>
recombinant protein for use as a therapeutic, for screening, for lead optimization and<br>
for target validation.<br>
Typical expression systems are characterized by a host cell line comprising an<br>
expression vector containing a heterologous promoter operatively linked to a cDNA<br>
encoding a gene product of interest. One such system for protein expression (and<br>
overexpression) uses the SRa promoter, which is composed of a fusion between the<br>
SV40 early promoter and the R segment and part of the U5 sequence form the long<br>
terminal repeat of human T-cell leukemia virus type 1. Takebe et al. (1988) Mol.<br>
Cell. Biol 8:466-472. Another overexpression system utilizes a human<br>
cytomegalovirus (CMV) immediate early promoter. U.S. Patents 5,168,062 and<br>
5,385,839. Both of these promoters are regulated by endogenous, naturally-occurring<br>
1<br><br>
WO 2006/033859 PCT/US2005/032157<br>
transcription factors, whose availability or activity may limit the amount of<br>
transcription, and hence the amount of protein produced, in these systems. Moreover,<br>
overexpression of the naturally-occurring transcription factors that regulate these<br>
promoters could lead to aberrant expression of genes normally regulated by these<br>
factors, with potential detrimental effects on expression of the desired gene product.<br>
Additional methods for protein production involve integration of promoters or<br>
other regulatory sequences into a chromosome adjacent to a gene whose expression is<br>
to be regulated. See, e.g., U.S. Patents 5,272,071; 5,641,670; 5,733,761; 5,968,502<br>
and 6,361,972. These, too, depend of the action of endogenous transcription factors<br>
and thus are subject to the potential limitations discussed above for the SRa and CMV<br>
systems. Moreover, they also suffer from difficulties in achieving precisely targeted<br>
chromosomal integration of exogenous polynucleotides.<br>
Expression systems which yield levels of protein that are higher than those<br>
obtained by using the SRa and CMV promoters, and which do not depend on random<br>
integration of exogenous polynucleotide sequences, would be desirable. Moreover,<br>
expression systems utilizing exogenous transcription factors allow the design of<br>
customized transcription factors, provide greater flexibility and provide the potential<br>
to obtain higher levels of expression of gene products of interest.<br>
SUMMARY<br>
In one aspect, a method for regulating the transcription of a nucleotide<br>
sequence in a cell is provided, the method comprising expressing, in the cell, a protein<br>
that binds to a target site comprising SEQ ID NO:1, wherein SEQ ID NO:1 is<br>
operatively linked to the nucleotide sequence. In certain embodiments, a plurality of<br>
target sites are operatively linked to the nucleotide sequence.<br>
In another aspect, described herein is a method for regulating transcription of<br>
first and second nucleotide sequences in a cell, the method comprising expressing, in<br>
the cell, a protein that binds to a target site comprising SEQ ID NO:1 and wherein<br>
SEQ ID NO:1 is operatively linked to the first and second nucleotide sequences. A<br>
plurality of target sites may be operatively linked the first nucleotide sequence, to the<br>
second nucleotide sequence or to both the first and second nucleotide sequences.<br>
In any of them methods described herein, the protein may comprise, for<br>
example, SEQ ID NO:25 or equivalents thereof (e.g., SEQ ID NO:2, SEQ ID NO:3,<br>
SEQ ID NO:7, SEQ ID NO:8). Thus, described herein are methods for regulating<br>
2<br><br>
WO 2006/033859 PCT/US2005/032157<br>
transcription of a first (and/or second) nucleotide sequence(s) in a cell by expressing<br>
SEQ ID NO:25 or an equivalent in the cell. In certain embodiments, the protein<br>
further comprises a transcriptional activation domain (e.g., VP16).<br>
In any of the methods described herein, the nucleotide sequence can encode a<br>
mRNA that is translated to yield one or more protein(s). Furthermore, in any of the<br>
methods described herein, the nucleotide sequence(s) may comprises a cDNA<br>
sequence, for example one or more cDNA sequence encoding antibody polypeptides<br>
(e.g., antibody heavy chain or light chain). In additional embodiments, the nucleotide<br>
sequence can encode a RNA molecule that is not translated into protein such as, for<br>
example, siRNA, micro RNA, rRNA, tRNA, snRNA or scRNA.<br>
In another aspect, described herein are methods for regulating the transcription<br>
of a nucleotide sequence in a cell, the method comprising expressing, in the cell, a<br>
protein comprising SEQ ID NO:25, SEQ ID NO:2, SEQ ID NO:3, SEQ ID NO:7,<br>
SEQ ID NO:8 or equivalents, wherein the protein binds to a target site; wherein the<br>
target site is operatively linked to the nucleotide sequence. In certain embodiments,<br>
the target site comprises SEQ ID NO: 1. One or more target sites may be operatively<br>
linked to the nucleotide sequence. In certain embodiments, the nucleotide sequence<br>
comprises a cDNA sequence. The protein may further comprise a transcriptional<br>
activation domain, for example a VP16 domain.<br>
In yet another aspect, described herein is a method for regulating the<br>
transcription of first and second nucleotide sequences in a cell, the method<br>
comprising: expressing, in the cell, a protein comprising SEQ ID NO:25, SEQ ID<br>
NO:2, SEQ ID NO:3, SEQ ID NO:7, SEQ ID NO:8 or equivalents, wherein the<br>
protein binds to a target site; wherein the target site is operatively linked to both of the<br>
first and second nucleotide sequences. In certain embodiments, the target site<br>
comprises SEQ ID NO: 1. One or more target sites may be operatively linked to the<br>
first and/or second nucleotide sequences. In certain embodiments, the first and/or<br>
nucleotide sequence comprises a cDNA sequence, for example a cDNA sequence<br>
encoding antibody polypeptides (e.g., antibody light chains and/or antibody heavy<br>
chains). The protein may further comprise a transcriptional activation domain, for<br>
example a VP16 domain.<br>
In yet another aspect, polypeptides comprising SEQ ID NO:25 are provided.<br>
In certain embodiments, the polypeptides comprise SEQ ID NO:2 or SEQ ID NO:7 or<br>
equivalents thereof.<br>
3<br><br>
WO 2006/033859 PCT/US2005/032157<br>
In a still further aspect, polynucleotides encoding any of the polypeptides<br>
disclosed herein are provided.<br>
In yet another aspect, cells comprising any of the polypeptides and/or<br>
polynucleotides disclosed herein are provided.<br>
In another aspect, provided herein are vectors comprising SEQ ID NO:1; cells<br>
comprising any of these vectors as well as a cell comprising one or more copies of<br>
SEQ ID NO:1 integrated into its genome.<br>
These and other embodiments will be readily apparent to one of skill in the art<br>
upon reading the present disclosure.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the nucleotide sequence (SEQ ID NO:9) of a hybrid SV40-R-<br>
U5 promoter (SRα promoter), synthesized as described in Example 1. The target site<br>
for the various ZFPs described in Example 2 is underlined.<br>
Figure 2 shows the amino acid sequence of the 2392/00 protein (SEQ ID<br>
NO:10). The domains in the protein are as follows. Amino acids 3-9; nuclear<br>
localization sequence; amino acids 15-109: zinc finger domain; amino acids 119-185:<br>
VP16 transcriptional activation domain; amino acids 196-203: FLAG epitope tag.<br>
Figure 3 shows the sequence of a polynucleotide (SEQ ID NO: 11) encoding<br>
the 2392/00 protein. Portions of the sequence encoding the various domains in the<br>
protein are as follows. Nucleotides 7-27: nuclear localization sequence; nucleotides<br>
43-327: zinc finger domain; nucleotides 355-555: VP16 transcriptional activation<br>
domain; nucleotides 586-609: FLAG epitope tag.<br>
Figure 4 shows the amino acid sequence of the 2392/10 protein (SEQ ID<br>
NO: 12). The domains in the protein are as follows. Amino acids 3-9: nuclear<br>
localization sequence; amino acids 15-109: zinc finger domain; amino acids 119-185:<br>
VP16 transcriptional activation domain; amino acids 196-203: FLAG epitope tag.<br>
Figure 5 shows the sequence of a polynucleotide (SEQ ID NO: 13) encoding<br>
the 2392/10 protein. Portions of the sequence encoding the various domains in the<br>
protein are as follows. Nucleotides 7-27: nuclear localization sequence; nucleotides<br>
43-327: zinc finger domain; nucleotides 355-555: VP16 transcriptional activation<br>
domain; nucleotides 586-609: FLAG epitope tag.<br>
4<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Figure 6 shows the amino acid sequences of a number of three-finger zinc<br>
finger domains that were obtained from a two-hybrid selection system to recognize<br>
the target sequence GCTGTGGAA (SEQ ID NO:1). See Example 2 for details.<br>
Figure 7 shows levels of immunoglobulin kappa chain mRNA in cells<br>
containing an integrated transcription unit comprising a SRα promoter and kappa<br>
chain cDNA. Cells were transfected with plasmids encoding a VP16 activation<br>
domain (NVF) or the 2393/00 ZFP fused to the VP16 activation domain (2392-<br>
VP16). Numbers on the abscissa refer to nanograms of DNA transfected. See<br>
Example 3 for details.<br>
Figure 8 shows levels of secreted antibody in two cell lines (A and B)<br>
transfected with a plasmid encoding the 2392/00 ZFP-VP16 fusion protein (2392),<br>
compared to cells transfected with a plasmid encoding the VP16 activation domain<br>
(NVF). GFP: cells transfected with a plasmid encoding a green-fluorescent protein;<br>
Mock: mock-transfected cells; ntf: non-transfected cells. See Example 3 for details.<br>
Figure 9 shows levels of immunoglobulin gamma heavy chain and<br>
immunoglobulin kappa light chain mRNA in cells containing amplified gamma and<br>
kappa chain cDNAs, both under the transcriptional control of the SRα promoter<br>
("High Producer Line" in Figure). Results from cells transfected with a plasmid<br>
encoding the 2392/00-VP16 fusion protein are indicated by "ZFP" along the abscissa;<br>
non-transfected cells are indicated by "NT." mRNA levels were normalized to those<br>
of GAPDH. See Example 3 for details.<br>
Figure 10 shows levels of immunoglobulin G secreted from cells stably<br>
transfected with sequences encoding the 2392/00-VP16 fusion protein (labeled "ZFP"<br>
in the Figure). Secreted IgG levels from untransfected cells (labeled "Control") are<br>
also shown.<br>
Figure 11 shows relative IgG levels in cells transfected with two different<br>
plasmids containing both a heavy chain-encoding transcription unit and a light chain-<br>
encoding transcription unit. In one of the plasmids, each transcription unit is under<br>
the transcriptional control of a SRα promoter (denoted SRa in the figure). In the other<br>
plasmid, each transcription unit is under the transcriptional control of a SRα promoter<br>
to which 8 additional copies of SEQ ID NO:1 have been appended (denoted SRa in<br>
the figure). 2392/10-7 refers to a clonal isolate of CHO cells stably transfected with a<br>
nucleic acid encoding the 2392/10-VP16 fusion protein. DG44 refers to the parental,<br>
untransfected CHO cell line.<br>
5<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Figure 12 shows levels of green fluorescent protein mRNA (normalized to<br>
GAPDH mRNA) in 2392/10-7 cells, which contain sequences encoding a ZFP-VP16<br>
fusion protein. Cells were transfected with plasmids containing various modified<br>
CMV promoters operatively linked to sequences encoding green fluorescent protein<br>
(GFP). The number of copies of target site SEQ ID NO:1 inserted adjacent to the<br>
CMV promoter in each construct is shown below the graph, as is the orientation of the<br>
inserted target sites. The rightmost bar in each pair shows GFP mRNA levels in the<br>
ZFP-containing cell line; the leftmost bar shows GFP mRNA levels in a parental cell<br>
line that does not express the ZFP.<br>
Figure 13 shows levels of erythropoietin (Epo) secreted from cells transfected<br>
with different Epo expression constructs. SRαZ6 refers to a construct in which Epo<br>
expression is controlled by a SRα promoter containing seven copies of SEQ ID NO:1.<br>
CMV refers to a construct in which Epo expression is controlled by a CMV promoter.<br>
CMVz10 refers to a construct in which Epo expression is controlled by a CMV<br>
promoter containing ten copies of SEQ 3D NO: 1. The rightmost bar in each pair<br>
shows levels of Epo secreted from the ZFP-containing cell line 2392/10-7; the<br>
leftmost bar show levels of Epo secreted from a parental cell line that does not<br>
express the ZFP. See Example 9 for details.<br>
DETAILED DESCRIPTION<br>
General<br>
Practice of the methods, as well as preparation and use of the compositions<br>
disclosed herein employ, unless otherwise indicated, conventional techniques in<br>
molecular biology, biochemistry, chromatin structure and analysis, computational<br>
chemistry, cell culture, recombinant DNA and related fields as are within the skill of<br>
the art. These techniques are fully explained in the literature. See, for example,<br>
Sambrook et al. MOLECULAR CLONING: A LABORATORY MANUAL, Second edition,<br>
Cold Spring Harbor Laboratory Press, 1989 and Third edition, 2001; Ausubel et al.,<br>
CURRENT PROTOCOLS IN MOLECULAR BIOLOGY, John Wiley &amp; Sons, New York,<br>
1987 and periodic updates; the series METHODS IN ENZYMOLOGY, Academic Press,<br>
San Diego; Wolffe, CHROMATIN STRUCTURE AND FUNCTION, Third edition,<br>
Academic Press, San Diego, 1998; METHODS INENZYMOLOGY, Vol. 304,<br>
"Chromatin" (P.M. Wassarman and A. P. Wolffe, eds.), Academic Press, San Diego,<br>
6<br><br>
WO 2006/033859 PCT/US2005/032157<br>
1999; and METHODS IN MOLECULAR BIOLOGY, Vol. 119, "Chromatin Protocols"<br>
(P.B. Becker, ed.) Humana Press, Totowa, 1999.<br>
Definitions<br>
The terms "nucleic acid," "polynucleotide," and "oligonucleotide" are used<br>
interchangeably and refer to a deoxyribonucleotide or ribonucleotide polymer, in linear or<br>
circular conformation, and in either single- or double-stranded form. For the purposes of<br>
the present disclosure, these terms are not to be construed as limiting with respect to the<br>
length of a polymer. The terms can encompass known analogues of natural nucleotides, as<br>
well as nucleotides that are modified in the base, sugar and/or phosphate moieties (e.g.,<br>
phosphorothioate backbones). In general, an analogue of a particular nucleotide has the<br>
same base-pairing specificity; i.e., an analogue of A will base-pair with T.<br>
The terms "polypeptide," "peptide" and "protein" are used interchangeably to refer<br>
to a polymer of amino acid residues. The term also applies to amino acid polymers in<br>
which one or more amino acids are chemical analogues or modified derivatives of a<br>
corresponding naturally-occurring amino acids.<br>
Techniques for determining nucleic acid and amino acid sequence identity are<br>
known in the art. Typically, such techniques include determining the nucleotide<br>
sequence of the mRNA for a gene and/or determining the amino acid sequence of a<br>
protein encoded by a gene or mRNA, and comparing these sequences to a second<br>
nucleotide or amino acid sequence. Genomic sequences can also be determined and<br>
compared in this fashion. In general, identity refers to an exact nucleotide-to-<br>
nucleotide or amino acid-to-amino acid correspondence of two polynucleotides or<br>
polypeptide sequences, respectively. Two or more sequences (polynucleotide or<br>
amino acid) can be compared by determining their percent identity. The percent<br>
identity of two sequences, whether nucleic acid or amino acid sequences, is the<br>
number of exact matches between two aligned sequences divided by the length of the<br>
shorter sequences and multiplied by 100. An approximate alignment for nucleic acid<br>
sequences is provided by the local homology algorithm of Smith and Waterman,<br>
Advances in Applied Mathematics 2:482-489 (1981). This algorithm can be applied<br>
to amino acid sequences by using the scoring matrix developed by Dayhoff Atlas of<br>
Protein Sequences and Structure, M.O. Dayhoff ed., 5 suppl. 3:353-358, National<br>
Biomedical Research Foundation, Washington, D.C., USA, and normalized by<br>
Gribskov, Nucl. Acids Res. 14(6);6745-6763 (1986). An exemplary implementation<br>
7<br><br>
WO 2006/033859 PCT/US2005/032157<br>
of this algorithm to determine percent identity of a sequence is provided by the<br>
Genetics Computer Group (Madison, WI) in the "BestFit" utility application. The<br>
default parameters for this method are described in the Wisconsin Sequence Analysis<br>
Package Program Manual, Version 8 (1995) (available from Genetics Computer<br>
Group, Madison, WI). A preferred method of establishing percent identity in the<br>
context of the present disclosure is to use the MPSRCH package of programs<br>
copyrighted by the University of Edinburgh, developed by John F. Collins and Shane<br>
S. Sturrok, and distributed by IntelliGenetics, Inc. (Mountain View, CA). From this<br>
suite of packages the Smith-Waterman algorithm can be employed where default<br>
parameters are used for the scoring table (for example, gap open penalty of 12, gap<br>
extension penalty of one, and a gap of six). From the data generated the "Match"<br>
value reflects sequence identity. Other suitable programs for calculating the percent<br>
identity or similarity between sequences are generally known in the art, for example,<br>
another alignment program is BLAST, used with default parameters. For example,<br>
BLASTN and BLASTP can be used using the following default parameters: genetic<br>
code = standard; filter = none; strand = both; cutoff = 60; expect =10; Matrix =<br>
BLOSUM62; Descriptions = 50 sequences; sort by = HIGH SCORE; Databases =<br>
non-redundant, GenBank + EMBL + DDBJ + PDB + GenBank CDS translations +<br>
Swiss protein + Spupdate + PIR. Details of these programs can be found at the<br>
following internet address: http://www.ncbi.nlm.gov/cgi-bin/BLAST. With respect to<br>
sequences described herein, the range of desired degrees of sequence identity is<br>
approximately 20% to 100% and any integer value therebetween. Typically the<br>
percent identities between sequences are at least 70-75%, preferably 80-82%, more<br>
preferably 85-90%, even more preferably 92%, still more preferably 95%, and most<br>
preferably 98% sequence identity.<br>
Alternatively, the degree of sequence similarity between polynucleotides can<br>
be determined by hybridization of polynucleotides under conditions that allow<br>
formation of stable duplexes between homologous regions, followed by digestion<br>
with single-stranded-specific nuclease(s), and size determination of the digested<br>
fragments. Two nucleic acid, or two polypeptide sequences are substantially<br>
homologous to each other when the sequences exhibit at least about 70%-75%,<br>
preferably 80%-82%, more preferably 85%-90%, even more preferably 92%, still<br>
more preferably 95%, and most preferably 98% sequence identity over a defined<br>
length of the molecules, as determined using the methods above. As used herein,<br>
8<br><br>
WO 2006/033859 PCT/US2005/032157<br>
substantially homologous also refers to sequences showing complete identity to a<br>
specified DNA or polypeptide sequence. DNA sequences that are substantially<br>
homologous can be identified in a Southern hybridization experiment under, for<br>
example, stringent conditions, as defined for that particular system. Defining<br>
appropriate hybridization conditions is within the skill of the art. See, e.g., Sambrook<br>
et al., supra; Nucleic Acid Hybridization: A Practical Approach, editors B.D. Hames<br>
and S.J. Higgins, (1985) Oxford; Washington, DC; IRL Press).<br>
Selective hybridization of two nucleic acid fragments can be determined as<br>
follows. The degree of sequence identity between two nucleic acid molecules affects<br>
the efficiency and strength of hybridization events between such molecules. A<br>
partially identical nucleic acid sequence will at least partially inhibit the hybridization<br>
of a completely identical sequence to a target molecule. Inhibition of hybridization of<br>
the completely identical sequence can be assessed using hybridization assays that are<br>
well known in the art (e.g., Southern (DNA) blot, Northern (RNA) blot, solution<br>
hybridization, or the like, see Sambrook, et al., Molecular Cloning: A Laboratory<br>
Manual, Second Edition, (1989) Cold Spring Harbor, N.Y.). Such assays can be<br>
conducted using varying degrees of selectivity, for example, using conditions varying<br>
from low to high stringency. If conditions of low stringency are employed, the<br>
absence of non-specific binding can be assessed using a secondary probe that lacks<br>
even a partial degree of sequence identity (for example, a probe having less than<br>
about 30% sequence identity with the target molecule), such that, in the absence of<br>
non-specific binding events, the secondary probe will not hybridize to the target.<br>
When utilizing a hybridization-based detection system, a nucleic acid probe is<br>
chosen that is complementary to a reference nucleic acid sequence, and then by<br>
selection of appropriate conditions the probe and the reference sequence selectively<br>
hybridize, or bind, to each other to form a duplex molecule. A nucleic acid molecule<br>
that is capable of hybridizing selectively to a reference sequence under moderately<br>
stringent hybridization conditions typically hybridizes under conditions that allow<br>
detection of a target nucleic acid sequence of at least about 10-14 nucleotides in<br>
length having at least approximately 70% sequence identity with the sequence of the<br>
selected nucleic acid probe. Stringent hybridization conditions typically allow<br>
detection of target nucleic acid sequences of at least about 10-14 nucleotides in length<br>
having a sequence identity of greater than about 90-95% with the sequence of the<br>
selected nucleic acid probe. Hybridization conditions useful for probe/reference<br>
9<br><br>
WO 2006/033859 PCT/US2005/032157<br>
sequence hybridization, where the probe and reference sequence have a specific<br>
degree of sequence identity, can be determined as is known in the art (see, for<br>
example, Nucleic Acid Hybridization: A Practical Approach, editors B.D. Hames and<br>
S.J. Higgins, (1985) Oxford; Washington, DC; IRL Press).<br>
Conditions for hybridization are well-known to those of skill in the art.<br>
Hybridization stringency refers to the degree to which hybridization conditions<br>
disfavor the formation of hybrids containing mismatched nucleotides, with higher<br>
stringency correlated with a lower tolerance for mismatched hybrids. Factors that<br>
affect the stringency of hybridization are well-known to those of skill in the art and<br>
include, but are not limited to, temperature, pH, ionic strength, duration of the<br>
hybridization reaction and concentration of organic solvents such as, for example,<br>
formamide and dimethylsulfoxide. As is known to those of skill in the art,<br>
hybridization stringency is increased by higher temperatures, lower ionic strength and<br>
lower solvent concentrations.<br>
With respect to stringency conditions for hybridization, it is well known in the art<br>
that numerous equivalent conditions can be employed to establish a particular stringency<br>
by varying, for example, the following factors: the length and nature of the sequences,<br>
base composition of the various sequences, concentrations of salts and other hybridization<br>
solution components, the presence or absence of blocking agents in the hybridization<br>
solutions (e.g., dextran sulfate, and polyethylene glycol), hybridization reaction<br>
temperature and time parameters, as well as, varying wash conditions. The selection of a<br>
particular set of hybridization conditions is selected following standard methods in the art<br>
(see, for example, Sambrook, et al., Molecular Cloning: A Laboratory Manual Second<br>
Edition, (1989) Cold Spring Harbor, N.Y.).<br>
"Binding" refers to a sequence-specific, non-covalent interaction between<br>
macromolecules (e.g., between a protein and a nucleic acid). Not all components of a<br>
binding interaction need be sequence-specific (e.g., contacts with phosphate residues<br>
in a DNA backbone), as long as the interaction as a whole is sequence-specific. Such<br>
interactions are generally characterized by a dissociation constant (Kd) of 10-6 M-1 or<br>
lower. "Affinity" refers to the strength of binding: increased binding affinity being<br>
correlated with a lower Kd.<br>
A "binding protein" is a protein that is able to bind non-covalently to another<br>
molecule. A binding protein can bind to, for example, a DNA molecule (a DNA-binding<br>
protein), an RNA molecule (an RNA-binding protein) and/or a protein molecule (a<br>
10<br><br>
WO 2006/033859 PCT/US2005/032157<br>
protein-binding protein). In the case of a protein-binding protein, it can bind to itself (to<br>
form homodimers, homotrimers, etc.) and/or it can bind to one or more molecules of a<br>
different protein or proteins. A binding protein can have more than one type of binding<br>
activity. For example, zinc finger proteins have DNA-binding, RNA-binding and protein-<br>
binding activity.<br>
A "zinc finger DNA binding protein" (or zinc finger binding domain) is a protein,<br>
or a domain within a larger protein, that binds DNA in a sequence-specific manner through<br>
one or more zinc fingers, which are regions of amino acid sequence within the binding<br>
domain whose structure is stabilized through coordination of a zinc ion. The term zinc<br>
finger DNA binding protein is often abbreviated as zinc finger protein or ZFP.<br>
Zinc finger binding domains can be "engineered" to bind to a predetermined<br>
nucleotide sequence. Non-limiting examples of methods for engineering zinc finger<br>
proteins are design and selection.<br>
A designed zinc finger protein is a protein not occurring in nature whose<br>
design/composition results principally from rational criteria. Rational criteria for<br>
design include application of substitution rules and computerized algorithms for<br>
processing information in a database storing information of existing ZFP designs and<br>
binding data. See, for example, US Patents 6,140,081; 6,453,242; and 6,534,261;<br>
see also WO 98/53058; WO 98/53059; WO 98/53060; WO 02/016536 and<br>
WO 03/016496.<br>
A "selected" zinc finger protein is a protein not found in nature whose<br>
production results primarily from an empirical process such as phage display,<br>
interaction trap or hybrid selection. See e.g., US 5,789,538; US 5,925,523;<br>
US 6,007,988; US 6,013,453; US 6,200,759; WO 95/19431; WO 96/06166;<br>
WO 98/53057; WO 98/54311; WO 00/27878; WO 01/60970 WO 01/88197 and<br>
WO 02/099084.<br>
Thus, an "engineered zinc finger protein" refers to a protein that contains one<br>
or more zinc fingers, that has been constructed to bind in a sequence-specific fashion<br>
to a predetermined nucleotide sequence. Generally, engineered zinc finger proteins<br>
are non-naturally-occurring proteins and/or contain naturally-occurring zinc fingers in<br>
non-naturally-occurring arrangements and/or combinations. Methods for engineering<br>
the binding specificity of zinc fingers and constructing engineered zinc finger proteins<br>
include, but are not limited to, rational design, randomization/selection techniques,<br>
polysome selection, cis-display, one- and two-hybrid systems, and selection from<br>
11<br><br>
WO 2006/033859 PCT/US2005/032157<br>
randomized libraries of both engineered and naturally-occurring zinc fingers. See, for<br>
example, U.S. Patents 5,789,538; 6,007,988; 6,013,453; 6,140,466; 6,242,568;<br>
6,410,248; 6,453,242; 6,479,626; 6,503,717; 6,534,261; 6,706,470; 6,733,970;<br>
and 6,746,838; U.S. Patent Application publications 2003/0044957; 2003/0068675;<br>
2003/0104526; 2003/0108880; 2003/0166141 and 2004/0091991; and PCT<br>
Publications WO 98/53057; WO 98/53058; WO 98/53059; WO 98/53060; WO<br>
01/53480; WO 01/88197; and WO 02/77227, the disclosures of which are<br>
incorporated herein by reference in their entireties for all purposes. The term<br>
"engineered zinc finger protein" does not refer to a cloned, naturally-occurring zinc<br>
finger protein.<br>
"Chromatin" is the nucleoprotein structure comprising the cellular genome.<br>
Cellular chromatin comprises nucleic acid, primarily DNA, and protein, including<br>
histones and non-histone chromosomal proteins. The majority of eukaryotic cellular<br>
chromatin exists in the form of nucleosomes, wherein a nucleosome core comprises<br>
approximately 150 base pairs of DNA associated with an octamer comprising two<br>
each of histones H2A, H2B, H3 and H4; and linker DNA (of variable length<br>
depending on the organism) extends between nucleosome cores. A molecule of<br>
histone H1 is generally associated with the linker DNA. For the purposes of the<br>
present disclosure, the term "chromatin" is meant to encompass all types of cellular<br>
nucleoprotein, both prokaryotic and eukaryotic. Cellular chromatin includes both<br>
chromosomal and episomal chromatin.<br>
A "chromosome," is a chromatin complex comprising all or a portion of the<br>
genome of a cell. The genome of a cell is often characterized by its karyotype, which<br>
is the collection of all the chromosomes that comprise the genome of the cell. The<br>
genome of a cell can comprise one or more chromosomes.<br>
An "episome" is a replicating nucleic acid, nucleoprotein complex or other<br>
structure comprising a nucleic acid that is not part of the chromosomal karyotype of a<br>
cell. Examples of episomes include plasmids and certain viral genomes.<br>
A "target site" or "target sequence" is a nucleic acid sequence that defines a<br>
portion of a nucleic acid to which a binding molecule (e.g., a binding protein) will<br>
bind, provided sufficient conditions for binding exist. For example, the sequence<br>
5'-GAATTC-3' is a target site for the Eco RI restriction endonuclease.<br>
An "accessible region" is a site in cellular chromatin in which a target site<br>
present in the nucleic acid can be bound by an exogenous molecule which recognizes<br>
12<br><br>
WO 2006/033859 PCT/US2005/032157<br>
the target site. Without wishing to be bound by any particular theory, it is believed<br>
that an accessible region is one that is not packaged into a nucleosomal structure. The<br>
distinct structure of an accessible region can often be detected by its sensitivity to<br>
chemical and enzymatic probes, for example, nucleases.<br>
An "exogenous" molecule is a molecule that is not normally present in a cell,<br>
but can be introduced into a cell by one or more genetic, biochemical or other<br>
methods. "Normal presence in the cell" is determined with respect to the particular<br>
developmental stage and environmental conditions of the cell. Thus, for example, a<br>
molecule that is present only during embryonic development of muscle is an<br>
exogenous molecule with respect to an adult muscle cell. Similarly, a molecule<br>
induced by heat shock is an exogenous molecule with respect to a non-heat-shocked<br>
cell. An exogenous molecule can comprise, for example, a functioning version of a<br>
malfunctioning endogenous molecule or a malfunctioning version of a normally-<br>
functioning endogenous molecule.<br>
An exogenous molecule can be, among other things, a small molecule, such as<br>
is generated by a combinatorial chemistry process, or a macromolecule such as a<br>
protein, nucleic acid, carbohydrate, lipid, glycoprotein, lipoprotein, polysaccharide,<br>
any modified derivative of the above molecules, or any complex comprising one or<br>
more of the above molecules. Nucleic acids include DNA and RNA, can be single- or<br>
double-stranded; can be linear, branched or circular; and can be of any length.<br>
Nucleic acids include those capable of forming duplexes, as well as triplex-forming<br>
nucleic acids. See, for example, U.S. Patent Nos. 5,176,996 and 5,422,251. Proteins<br>
include, but are not limited to, DNA-binding proteins, transcription factors, chromatin<br>
remodeling factors, methylated DNA binding proteins, polymerases, methylases,<br>
demethylases, acetylases, deacetylases, kinases, phosphatases, integrases,<br>
recombinases, ligases, topoisomerases, gyrases and helicases.<br>
An exogenous molecule can be the same type of molecule as an endogenous<br>
molecule, e.g., an exogenous protein or nucleic acid. For example, an exogenous<br>
nucleic acid can comprise an infecting viral genome, a plasmid or episome introduced<br>
into a cell, or a chromosome that is not normally present in the cell. Methods for the<br>
introduction of exogenous molecules into cells are known to those of skill in the art<br>
and include, but are not limited to, lipid-mediated transfer (i.e., liposom.es, including<br>
neutral and cationic lipids), electroporation, direct injection, cell fusion, particle<br>
13<br><br>
WO 2006/033859 PCT/US2005/032157<br>
bombardment, calcium phosphate co-precipitation, DEAE-dextran-mediated transfer<br>
and viral vector-mediated transfer.<br>
By contrast, an "endogenous" molecule is one that is normally present in a<br>
particular cell at a particular developmental stage under particular environmental<br>
conditions. For example, an endogenous nucleic acid can comprise a chromosome,<br>
the genome of a mitochondrion, chloroplast or other organelle, or a naturally-<br>
occurring episomal nucleic acid. Additional endogenous molecules can include<br>
proteins, for example, transcription factors and enzymes.<br>
A "fusion" molecule is a molecule in which two or more subunit molecules are<br>
linked, preferably covalently. The subunit molecules can be the same chemical type<br>
of molecule, or can be different chemical types of molecules. Examples of the first<br>
type of fusion molecule include, but are not limited to, fusion proteins (for example, a<br>
fusion between a ZFP DNA-binding domain and a transcriptional regulatory domain)<br>
and fusion nucleic acids (for example, a nucleic acid encoding the fusion protein<br>
described supra). Examples of the second type of fusion molecule include, but are<br>
not limited to, a fusion between a triplex-forming nucleic acid and a polypeptide, and<br>
a fusion between a minor groove binder and a nucleic acid.<br>
Expression of a protein (e.g., a fusion protein) in a cell can result from<br>
delivery of the protein to the cell or by delivery of a polynucleotide encoding the<br>
protein to a cell, wherein the polynucleotide is transcribed, and the transcript is<br>
translated, to generate the protein. Delivery to the cell of a RNA molecule, which is<br>
subsequently translated in the cell, can also be used to express a protein in a cell.<br>
Trans-splicing, polypeptide cleavage and polypeptide ligation can also be involved in<br>
expression of a protein in a cell. Methods for polynucleotide and polypeptide delivery<br>
to cells are known in the art and exemplary methods are presented elsewhere in this<br>
disclosure.<br>
A "gene," for the purposes of the present disclosure, includes a DNA region<br>
encoding a gene product (see infra), as well as all DNA regions which regulate the<br>
production of the gene product, whether or not such regulatory sequences are adjacent<br>
to coding and/or transcribed sequences. Accordingly, a gene includes, but is not<br>
necessarily limited to, promoter sequences, terminators, translational regulatory<br>
sequences such as ribosome binding sites and internal ribosome entry sites, enhancers,<br>
silencers, insulators, boundary elements, replication origins, matrix attachment sites<br>
and locus control regions.<br>
14<br><br>
WO 2006/033859 PCT/US2005/032157<br>
"Gene expression" refers to the conversion of the information, contained in a<br>
gene, into a gene product. A gene product can be the direct transcriptional product of<br>
a gene (e.g., rnRNA, tRNA, rRNA, antisense RNA, dsRNA, ribozyme, structural<br>
RNA or any other type of RNA), a processed transcript such as, for example siRNA,<br>
or a protein produced by translation of a mRNA. Gene products also include RNAs<br>
which are modified, by processes such as capping, polyadenylation, methylation, and<br>
editing, and proteins modified by, for example, methylation, acetylation,<br>
phosphorylation, ubiquitination, ADP-ribosylation, myristilation, and glycosylation.<br>
"Modulation" of gene expression refers to a change in the activity of a gene.<br>
Modulation of expression can include, but is not limited to, gene activation and gene<br>
repression.<br>
"Eucaryotic" cells include, but are not limited to, fungal cells (such as yeast),<br>
plant cells, animal cells, mammalian cells and human cells.<br>
A "region of interest" is any region of cellular chromatin, such as, for<br>
example, a gene or a non-coding sequence within or adjacent to a gene, in which it is<br>
desirable to bind an exogenous molecule. Binding can be for the purposes of, e.g.,<br>
transcriptional regulation. A region of interest can be present in a chromosome, an<br>
episome, an organellar genome (e.g., mitochondrial, chloroplast), or an infecting viral<br>
genome, for example. A region of interest can be within the coding region of a gene,<br>
within transcribed non-coding regions such as, for example, leader sequences, trailer<br>
sequences or introns, or within non-transcribed regions, either upstream or<br>
downstream of the coding region. A region of interest can be as small as a single<br>
nucleotide pair or up to 2,000 nucleotide pairs in length, or any integral value of<br>
nucleotide pairs.<br>
The terms "operative linkage" and "operatively linked" (or "operably linked")<br>
are used interchangeably with reference to a juxtaposition of two or more components<br>
(such as sequence elements), in which the components are arranged such that the<br>
components function normally and allow the possibility that at least one of the<br>
components can mediate a function that is exerted upon at least one of the other<br>
components. By way of illustration, a transcriptional regulatory sequence, such as a<br>
promoter, is operatively linked to a coding sequence if the transcriptional regulatory<br>
sequence controls the level of transcription of the coding sequence in response to the<br>
presence or absence of one or more transcriptional regulatory factors. A<br>
transcriptional regulatory sequence is generally operatively linked in cis with a coding<br>
15<br><br>
WO 2006/033859 PCT/US2005/032157<br>
sequence, but need not be directly adjacent to it. For example, an enhancer is a<br>
transcriptional regulatory sequence that is operatively linked to a coding sequence,<br>
even though they are not contiguous.<br>
With respect to fusion polypeptides, the term "operatively linked" can refer to<br>
the fact that each of the components performs the same function in linkage to the<br>
other component as it would if it were not so linked. For example, with respect to a<br>
fusion polypeptide in which a ZFP DNA-binding domain is fused to a transcriptional<br>
regulatory domain, the ZFP DNA-binding domain and the regulatory domain are in<br>
operative linkage if, in the fusion polypeptide, the ZFP DNA-binding domain portion<br>
is able to bind its target site and/or its binding site, while the regulatory domain is able<br>
to modulate (e.g., activate or repress) transcription.<br>
A "functional equivalent" or "functional fragment" of a protein, polypeptide or<br>
nucleic acid is a protein, polypeptide or nucleic acid whose sequence is not identical<br>
to the full-length protein, polypeptide or nucleic acid, yet retains the same function as<br>
the full-length protein, polypeptide or nucleic acid. A functional fragment can<br>
possess more, fewer, or the same number of residues as the corresponding native<br>
molecule, and/or can contain one ore more amino acid or nucleotide substitutions.<br>
Methods for determining the function of a nucleic acid (e.g., coding function, ability<br>
to hybridize to another nucleic acid) are well-known in the art. Similarly, methods for<br>
determining protein function are well-known. For example, the DNA-binding<br>
function of a polypeptide can be determined, for example, by filter-binding,<br>
electrophoretic mobility-shift, or immunoprecipitation assays. DNA cleavage can be<br>
assayed by gel electrophoresis. See Ausubel et at, supra. The ability of a protein to<br>
interact with another protein can be determined, for example, by co-<br>
immunoprecipitation, two-hybrid assays or complementation, both genetic and<br>
biochemical. See, for example, Fields et al. (1989) Nature 340:245-246; U.S. Patent<br>
No. 5,585,245 and PCT WO 98/44350.<br>
Engineered Zinc Finger Proteins and Target Sequences<br>
Disclosed herein are compositions and methods for regulation of transcription,<br>
which are useful, for example, for enhanced production of RNA and/or protein.<br>
These include fusion proteins comprising an engineered zinc finger protein and a<br>
functional domain such as, for example, a transcriptional activation domain. Suitable<br>
functional domains are known in the art and include, without limitation,<br>
16<br><br>
WO 2006/033859 PCT/US2005/032157<br>
transcriptional activation domains such as, for example, VP16, VP64 and p65.<br>
Moreover, one or more of the same or different functional domains (e.g.,<br>
transcriptional activation domains) can be present in a given fusion protein. See co-<br>
owned U.S. Patent Application Publication No. 2002/0160940, incorporated by<br>
reference, for disclosure of exemplary functional domains.<br>
In certain embodiments, a zinc finger protein is engineered to bind to a<br>
sequence comprising the target sequence GCTGTGGAA (SEQ ID NO: 1). This<br>
sequence is present in the SRα promoter (Takebe et al, supra), a promoter commonly<br>
used for protein production, but one or more copies SEQ ID NO:1 can be inserted in<br>
or adjacent to any promoter known in the art (e.g., a CMV promoter).<br>
An exemplary three-finger zinc finger protein, SBS2392/00, that has been<br>
engineered to bind to SEQ ID NO:1 has the amino acid sequence:<br>
KKKQHICHIQGCGKVYGORSNLVRHLRWHTGERPFMCTWSYCGKRFTRSDA<br>
LSRHKRTHTGEKKFACPECPKRFMOSSDLRRHIKTHQNK (SEQ ID NO:2).<br>
The underlined amino acid residues in SEQ ID NO:2 correspond to residues -<br>
1 through +6 with respect to the start of the alpha-helical portion of a zinc finger and<br>
are denoted the "recognition regions" because one or more of these residues<br>
participate in sequence specificity of nucleic acid binding. Accordingly, proteins<br>
comprising the same three recognition regions in a different polypeptide backbone<br>
sequence are considered equivalents to the protein identified as SEQ ED NO:2, since<br>
they will have the same DNA-binding specificity.<br>
Thus, in certain embodiments, the three recognition regions (underlined in<br>
SEQ ID NO:2 above) can be placed in any zinc finger backbone (see, e.g., U.S.<br>
Patents 6,453,242 and 6,534,261) and the resulting protein can be used to regulate<br>
transcription, e.g., to enhance protein production. Accordingly, engineered zinc<br>
finger proteins having the following sequence can be used in the disclosed methods:<br>
C-X2-4-C-X5-QRSNLVR-H-X3-5-H-X7-C-X2-4-C-X5-RSDALSR-H-X3-5-H-X7-C-X2-4-<br>
C-X5-QSSDLRR-H-X3-5-H (SEQ ID NO:3).<br>
Within the recognition region, residues -1, +3 and +6 are primarily<br>
responsible for protein-nucleotide contacts. Accordingly, non-limiting examples of<br>
additional equivalents include proteins comprising three zinc fingers wherein the first<br>
finger contains a Q residue at -1, a N residue at +3 and a R residue at +6<br>
(QXXNXXR, SEQ ID NO:4); the second finger contains a R residue at -1, an A<br>
residue at +3 and a R residue at +6 (RXXAXXR, SEQ ID NO:5); and the third finger<br>
17<br><br>
WO 2006/033859 PCT/US2005/032157<br>
contains a Q residue at -1, a D residue at +3 and a R residue at +6 (QXXDXXR, SEQ<br>
ID NO:6). Additional equivalents comprise any ZFP that binds to a sequence<br>
comprising the target sequence GCTGTGGAA (SEQ ID NO:1).<br>
An additional exemplary three-finger zinc finger protein engineered to bind<br>
the target sequence GCTGTGGAA (SEQ ID NO:1), SBS2392/10, has the following<br>
amino acid sequence:<br>
KKKOHICHIOGCGKVYGOSSNLARHLRWHTGERPFMCTWSYCGKRF<br>
TRSDALTRHKRTHTGEKKFACPECPKRFMOSCDLTRHIKTHQNK (SEQ ID<br>
NO:7)The underlined amino acid residues in SEQ ID NO:7 correspond to residues -1<br>
through +6 with respect to the start of the alpha-helical portion of a zinc finger and are<br>
denoted the "recognition regions" because one or more of these residues participate in<br>
sequence specificity of nucleic acid binding. Accordingly, proteins comprising the<br>
same three recognition regions in a different polypeptide backbone sequence are<br>
considered equivalents to SEQ ID NO:7, since they will have the same DNA-binding<br>
specificity.<br>
Thus, in certain embodiments, the three recognition regions (underlined in<br>
SEQ ID NO:7 above) can be placed in any zinc finger backbone (see, e.g., U.S.<br>
Patents 6,453,242 and 6,534,261) and the resulting protein can be used to regulate<br>
transcription, e.g., to enhance protein production. Accordingly, engineered zinc<br>
finger proteins having the following sequence can be used in the disclosed methods;<br>
C-X2-4-C-X5-QSSNLAR-H-X3-5-H-X7-C-X2-4-C-X5-RSDALTR-H-X3-5-H-X7-C-X2-4-<br>
C-X5-QSCDLTR-H-X3-5-H (SEQ ID NO:8).<br>
Within the recognition region, residues -1, +3 and +6 are primarily<br>
responsible for protein-nucleotide contacts. Accordingly, non-limiting examples of<br>
additional equivalents include proteins comprising three zinc fingers wherein the first<br>
finger contains a Q residue at -1, a N residue at +3 and a R residue at +6<br>
(QXXNXXR, SEQ ID NO:4); the second finger contains a R residue at -1, an A<br>
residue at +3 and a R residue at +6 (RXXAXXR, SEQ ID NO:5); and the third finger<br>
contains a Q residue at -1, a D residue at +3 and a R residue at +6 (QXXDXXR, SEQ<br>
ID NO:6). Thus, for example, proteins comprising SEQ ID NO:25 are considered<br>
equivalents for use in the disclosed methods.<br>
C-X2-4-C-X5-QXXNXXR-H-X3-5-H-X7-C-X2-4-C-X5-RXXAXXR-H-X3-5-H-<br>
X7-C-X2-4-C-X5-QXXDXXR-H-X3-5-H (SEQ ID NO:25)<br>
18<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Additional equivalents comprise any ZFP that binds to a sequence comprising<br>
the target sequence GCTGTGGAA (SEQ ID NO:1).<br>
Correspondences between amino acids at the -1, +3 and +6 contact residues of<br>
the recognition region of a zinc finger, and micleotides in a target site, have been<br>
described. See, for example, U.S. Patent Nos. 6,007,988; 6,013,453 and 6,746,838;<br>
as well as PCT Publications WO 96/06166; WO 98/53058; WO 98/53059 and WO<br>
98/53060. Accordingly, also to be considered equivalents are three-finger zinc finger<br>
proteins in which the first finger contains Q at -1; N at +3 and R, K, S or T at +6; the<br>
second finger contains R at -1; A, S or V at +3 and R, K, S or T at +6; and the third<br>
finger contains N, Q, H or T at -1; S, D, E, L, T, or V at +3 and R, K, S or T at +6.<br>
Fusion proteins and encoding polynucleotides<br>
As previously stated, the engineered zinc finger DNA-binding domains<br>
disclosed herein can comprise a portion of a fusion protein, wherein the fusion protein<br>
also contains one or more functional domains (e.g., transcriptional regulatory<br>
domains), nuclear localization sequences, epitope tags, etc. For example, the amino<br>
acid sequence of a fusion protein comprising a nuclear localization sequence (NLS), a<br>
zinc finger binding domain (ZFP), the VP16 transcriptional activation domain (VP16)<br>
and a FLAG epitope tag (FLAG), denoted 2392/00, is shown in Figure 2 (SEQ ID<br>
NO: 10). The nucleotide sequence encoding this protein is shown in Figure 3 (SEQ ID<br>
NO:11).<br>
An additional exemplary protein (2392/10) has the amino acid sequence (SEQ<br>
ID NO:12) shown in Figure 4. The sequence of a polynucleotide (SEQ ID NO:13)<br>
encoding the 2392/10 protein is shown in Figure 5.<br>
Functional Domains<br>
Any DNA-binding domain can optionally be associated with one or more<br>
"functional domains" or "regulatory domains" which facilitate, e.g., DNA processing<br>
(e.g., DNA cleavage) or modulation of gene expression. The binding domain can be<br>
covalently or non-covalently associated with one or more regulatory domains,<br>
alternatively two or more regulatory domains, with the two or more domains being<br>
two copies of the same domain, or two different domains. The regulatory domains<br>
can be covalently linked to the binding domain, e.g., via an amino acid linker, as part<br>
of a fusion protein. A DNA-binding domain can also be associated with a regulatory<br>
19<br><br>
WO 2006/033859 PCT/US2005/032157<br>
domain via a non-covalent dimerization domain, e.g., a leucine zipper, a STAT<br>
protein N terminal domain, or an FK506 binding protein (see, e.g., O'Shea, Science<br>
254: 539 (1991), Barahmand-Pour etal, Curr. Top. Microbiol Immunol. 211:121-<br>
128 (1996); Klemm et al, Annu. Rev. Immunol. 16:569-592 (1998); Klemm et al.,<br>
Annu. Rev. Immunol. 16:569-592 (1998); Ho et al., Nature 382:822-826 (1996); and<br>
Pomeranz et al, Biochem. 37:965 (1998)). The regulatory domain can be associated<br>
with the binding domain at any suitable position, including the C- or N-terminus of<br>
the binding domain.<br>
Common regulatory domains include, e.g., effector domains from transcription<br>
factors (activators, repressors, co-activators, co-repressors), silencers, nuclear<br>
hormone receptors, oncogene transcription factors (e.g., myc, jun, fos, myb, max,<br>
mad, rel, ets, bcl, myb, mos family members etc.); DNA repair enzymes and their<br>
associated factors and modifiers; DNA rearrangement enzymes and their associated<br>
factors and modifiers; chromatin associated proteins and their modifiers (e.g., kinases,<br>
acetyiases and deacetylases); and DNA modifying enzymes (e.g., methyltransferases,<br>
topoisomerases, helicases, ligases, kinases, phosphatases, polymerases,<br>
endonucleases, integrases) and their associated factors and modifiers.<br>
Transcription factor polypeptides from which one can obtain a regulatory<br>
domain include those that are involved in regulated and basal transcription. Such<br>
polypeptides include transcription factors, their effector domains, coactivators.,<br>
silencers, nuclear hormone receptors (see, e.g., Goodrich et al., Cell 84:825-30 (1996)<br>
for a review of proteins and nucleic acid elements involved in transcription;<br>
transcription factors in general are reviewed in Barnes &amp; Adcock, Clin. Exp. Allergy<br>
25 Suppl. 2:46-9 (1995) and Roeder, Methods Enzymol. 273:165-71 (1996)).<br>
Databases dedicated to transcription factors are known (see, e.g., Science 269:630<br>
(1995) and TRANSFAC). Nuclear hormone receptor transcription factors are<br>
described in, for example, Rosen et al, J. Med. Chem. 38:4855-74 (1995). The<br>
C/EBP family of transcription factors are reviewed in Wedel et al., Immunobiology<br>
193:171-85 (1995). Coactivators and co-repressors that mediate transcription<br>
regulation by nuclear hormone receptors are reviewed in, for example, Meier, Eur. J.<br>
Endocrinol. 134(2):158-9 (1996); Kaiser et al., Trends Biochem. Sci. 21:342-5<br>
(1996); and Utley et al., Nature 394:498-502 (1998)). GATA transcription factors,<br>
which are involved in regulation of hematopoiesis, are described in, for example,<br>
Simon,Nat. Genet. 11:9-11 (1995); Weiss et al., Exp. Hematol 23:99-107. TATA<br>
20<br><br>
WO 2006/033859 PCT/US2005/032157<br>
box binding protein (TBP) and its associated TAF polypeptides (which include<br>
TAF30, TAF55, TAF80, TAF110, TAF150, and TAF250) are described in Goodrich<br>
&amp;Tjian, Curr. Opin. Cell Biol. 6:403-9 (1994) and Hurley, Curr. Opin. Struct. Biol.<br>
6:69-75 (1996). The STAT family of transcription factors are reviewed in, for<br>
example, Barahmand-Pour et al., Curr. Top. Microbiol. Immunol. 211:121-8 (1996).<br>
Transcription factors involved in disease are reviewed in Aso et al., J. Clin. Invest.<br>
97:1561-9 (1996).<br>
In one embodiment, the KOX repression domain and/or the KRAB repression<br>
domain from the human KOX-1 protein is used as a transcriptional repressor. Thiesen<br>
et al., New Biologist 2:363-374 (1990); Margolin et al., PNAS 91:4509-4513 (1994);<br>
Pengue et al., Nucl. Acids Res. 22:2908-2914 (1994); Witzgall et al, PNAS 91:4514-<br>
4518 (1994). In another embodiment, KAP-1, a KRAB co-repressor, is used with<br>
KRAB or KOX. Friedman et al., Genes Dev. 10:2067-2078 (1996). Alternatively,<br>
KAP-1 can be used alone as a functional domain. Other preferred transcription<br>
factors and transcription factor domains that act as transcriptional repressors include<br>
MAD (see, e.g., Sommer et al., J. Biol. Chem. 273:6632-6642 (1998); Gupta et al.,<br>
Oncogene 16:1149-1159 (1998); Queva et al., Oncogene 16:967-977 (1998); Larsson<br>
et al., Oncogene 15:737-748 (1997); Laherty et al., Cell 89:349-356 (1997); and<br>
Cultraro et al., Mol Cell. Biol. 17:2353-2359 (19977)); FKHR (forkhead in<br>
rhapdosarcoma gene; Ginsberg et al., Cancer Res. 15:3542-3546 (1998); Epstein et<br>
al, Mol. Cell Biol. 18:4118-4130 (1998)); EGR-1 (early growth response gene<br>
product-1: Yan et al., PNAS 95:8298-8303 (1998); and Liu et al., Cancer Gene Ther.<br>
5:3-28 (1998)); the ets2 repressor factor repressor domain (ERD; Sgouras et al.,<br>
EMBOJ. 14:4781-4793 ((19095)); and the MAD smSIN3 interaction domain (SID;<br>
Ayer et al., Mol. Cell. Biol. 16:5772-5781 (1996)).<br>
In one embodiment, the HSV VP16 activation domain is used as a<br>
transcriptional activator (see, e.g., Hagmann et al., J. Virol. 71:5952-5962 (1997)).<br>
Other transcription factors from which activation domains can be obtained include<br>
nuclear hormone receptors (see, e.g., Torchia et al., Curr. Opin. Cell. Biol. 10:373-<br>
383 (1998)); the p65 subunit of nuclear factor kappa B (Bitko &amp; Barik, J. Virol.<br>
72:5610-5618 (1998) and Doyle &amp; Hunt, Neuroreport 8:2937-2942 (1997)); and<br>
EGR-1 (early growth response gene product-1; Yan et al., PNAS 95:8298-8303<br>
(1998); and Liu et al., Cancer Gene Ther. 5:3-28 (1998)). An additional synthetic<br>
21<br><br>
WO 2006/033859 PCT/US2005/032157<br>
activation domain is the VP64 activation domain (Seipel et al., EMBO J. 11:4961-<br>
4968 (1996)).<br>
Kinases, phosphatases, methylases, demethylases, acetylases, deacetylases,<br>
and other proteins that modify polypeptides involved in gene regulation are also<br>
useful as regulatory domains. Such modifiers are often involved in switching on or<br>
off transcription mediated by, for example, hormones. Kinases involved in<br>
transcription regulation are reviewed in Davis, Mol. Reprod. Dev. 42:459-67 (1995),<br>
Jackson et al., Adv. Second Messenger Phosphoprotein Res. 28:279-86 (1993), and<br>
Boulikas, Crit. Rev. Eukaryot. Gene Expr. 5:1-77 (1995), while phosphatases are<br>
reviewed in, for example, Schonthal &amp; Sernin, Cancer Biol. 6:239-48 (1995).<br>
Nuclear tyrosine kinases are described in Wang, Trends Biochem. Sci. 19:373-6<br>
(1994).<br>
As described, useful domains can also be obtained from fixe gene products of<br>
oncogenes (e.g., myc, jun, fos, myb, max, mad, rel, ets, bcl, myb, mos family<br>
members) and their associated factors and modifiers. Oncogenes are described in, for<br>
example, Cooper, Oncogenes, 2nd ed., The Jones and Bartlett Series in Biology,<br>
Boston, MA, Jones and Bartlett Publishers, 1995. The ets transcription factors are<br>
reviewed in Waslylke? et al., Eur. J. Biochem. 211:7-18 (1993) and Crepieux et al.,<br>
Crit. Rev. Oncog. 5:615-38 (1994). Myc oncogenes are reviewed in, for example,<br>
Ryan et al., Biochem. J. 314:713-21 (1996). The jun and fos transcription factors are<br>
described in, for example, The Fos and Jun Families of Transcription Factors, Angel<br>
&amp; Herrlich, eds. (1994). The max oncogene is reviewed in Hurlin et al., Cold Spring<br>
Harb. Symp. Quant. Biol. 59:109-16. The myb gene family is reviewed in Kanei-Ishii<br>
et al., Curr. Top. Microbiol Immunol. 211:89-98 (1996). The mos family is reviewed<br>
in Yew et al., Curr. Opin. Genet. Dev. 3:19-25 (1993).<br>
Regulatory domains can also be obtained from DNA repair enzymes and their<br>
associated factors and modifiers. These include, for example, nucleases (exo- and<br>
endo-), recombinases, helicases, integrases, polymerases and single-stranded DNA-<br>
binding proteins (SSBs). DNA repair systems are reviewed in, for example, Vos,<br>
Curr. Opin. Cell Biol. 4:385-95 (1992); Sancar, Ann. Rev. Genet. 29:69-105 (1995);<br>
Lehmann, Genet. Eng. 17:1-19 (1995); and Wood, Ann. Rev. Biochem. 65:135-67<br>
(1996). DNA rearrangement enzymes and their associated factors and modifiers can<br>
also be used as regulatory domains (see, e.g., Gangloff et al., Experientia 50:261-9<br>
(1994); Sadowski, FASEB J. 7:760-7 (1993)).<br>
22<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Similarly, regulatory domains can be derived from DNA modifying enzymes<br>
(e.g., DNA methyltransferases, topoisomerases, helicases, ligases, kinases,<br>
phosphatases, polymerases) and their associated factors and modifiers. Helicases are<br>
reviewed in Matson et al., Bioessays, 16:13-22 (1994), and methyltransferases are<br>
described in Cheng, Curr. Opin. Struct. Biol. 5:4-10 (1995). Chromatin associated<br>
proteins and their modifiers (e.g., kinases, acetylases and deacetylases), such as<br>
histone deacetylase (Wolffe, Science 272:371-2 (1996)) are also useful functional<br>
domains, In one embodiment, the regulatory domain is a DNA methyl transferase<br>
that acts as a transcriptional repressor (see, e.g., Van den Wyngaert et al., FEBS Lett.<br>
426:283-289 (1998); Flynn et al., J. Mol Biol. 279:101-116 (1998); Okano et al.,<br>
Nucleic Acids Res. 26:2536-2540 (1998); and Zardo &amp; Caiafa, J. Biol. Chem.<br>
273:16517-16520 (1998)). In another embodiment, endonucleases such as Fokl<br>
provide functional domains to catalyze targeted DNA cleavage, which facilitates<br>
processes such as transcriptional repression and homologous recombination. See,<br>
e.g., U. S. Patents 5,436,150; 5,792,640 and 6,265,196; U.S. Patent Application<br>
Publication No. 2003/0232410 and WO 03/87341.<br>
Factors that control chromatin and DNA structure, movement and localization<br>
and their associated factors and modifiers; factors derived from microbes (e.g.,<br>
prokaryotes, eukaryotes and virus) and factors that associate with or modify them can<br>
also be used to obtain functional domains for the construction of chimeric proteins or<br>
fusion molecules. In one embodiment, recombinases and integrases are used as<br>
regulatory domains. In another embodiment, histone acetyltransferase is used as a<br>
transcriptional activation domain (see, e.g., Jin &amp; Scotto, Mol. Cell. Biol. 18:4377-<br>
4384 (1998); Wolffe, Science 272:371-372 (1996); Taunton et al., Science 272:408-<br>
411 (1996); and Hassig et al., PNAS 95:3519-3524 (1998)). In another embodiment,<br>
histone deacetylase is used as a transcriptional repression domain (see, e.g., Jin &amp;<br>
Scotto, Mol Cell. Biol. 18:4377-4384 (1998); Syntichaki &amp; Thireos, J. Biol Chem.<br>
273:24414-24419 (1998); Sakaguchi et al., Genes Dev. 12:2831-2841 (1998); and<br>
Martinez et al., J. Biol. Chem. 273:23781-23785 (1998)).<br>
Another suitable repression domain is methyl binding domain protein 2B<br>
(MBD-2B) (see, also Hendrich et al. (1999) Mamm Genome 10:906-912 for<br>
description of MBD proteins). Another useful repression domain is that associated<br>
with the v-ErbA protein. See, for example, Damm, et al. (1989) Nature 339:593-597;<br>
Evans (1989) Int. J. Cancer Suppl. 4:26-28; Pain et al. (1990) New Biol. 2:284-294;<br>
23<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Sap et al. (1989) Nature 340:242-244; Zenke et al. (1988) Cell 52:107-119; and<br>
Zenke et al. (1990) Cell 61:1035-1049. Additional exemplary repression domains<br>
include, but are not limited to, thyroid hormone receptor (TR, see infra), SID, MBD1,<br>
MBD2, MBD3, MBD4, MBD-like proteins, members of the DNMT family (e.g.,<br>
DNMT1, DNMT3A, DNMT3B), Rb, MeCP1 and MeCP2. See, for example, Bird et<br>
al. (1999) Cell 99:451-454; Tyler et al. (1999) Cell 99:443-446; Knoepfler et al.<br>
(1999) Cell 99:447-450; and Robertson et al. (2000) Nature Genet. 25:338-342.<br>
Additional exemplary repression domains include, but are not limited to, ROM2 and<br>
AtHD2A. See, for example, Chern et al. (1996) Plant Cell 8:305-321; and Wu et al.<br>
(2000) Plant J. 22:19-27.<br>
Certain members of the nuclear hormone receptor (NHR) superfamily,<br>
including, for example, thyroid hormone receptors (TRs) and retinoic acid receptors<br>
(RARs) are among the most potent transcriptional regulators currently known. Zhang<br>
et al., Annu. Rev. Physiol 62:439-466 (2000) and Sucov et al., Mol Neurobiol 10(2-<br>
3):169-184 (1995). In the absence of their cognate ligand, these proteins bind with<br>
high specificity and affinity to short stretches of DNA (e.g., 12-17 base pairs) within<br>
regulatory loci (e.g., enhancers and promoters) and effect robust transcriptional<br>
repression of adjacent genes. The potency of their regulatory action stems from the<br>
concurrent use of two distinct functional pathways to drive gene silencing: (i) the<br>
creation of a localized domain of repressive chromatin via the targeting of a complex<br>
between the corepressor N-CoR and a histone deacetylase, HDAC3 (Guenther et al.,<br>
Genes Dev 14:1048-1057 (2000); Urnov et al, EMBO J 19:4074-4090 (2000); Li et<br>
al., EMBO J 19, 4342-4350 (2000) and Underhill et al., J. Biol. Chem. 275:40463-<br>
40470 (2000)) and (ii) a chromatin-independent pathway (Urnov et al., supra) that<br>
may involve direct interference with the function of the basal transcription machinery<br>
(Fondell et al., Genes Dev 7(7B):1400-1410 (1993) and Fondell et al.,Mol Cell Biol<br>
16:281-287 (1996).<br>
In the presence of very low (e.g., nanomolar) concentrations of their ligand,<br>
these receptors undergo a conformational change which leads to the release of<br>
corepressors, recruitment of a different class of auxiliary molecules (e.g.,<br>
coactivators) and potent transcriptional activation. Collingwood et al., J. Mol.<br>
Endocrinol. 23(3):255-275 (1999).<br>
The portion of the receptor protein responsible for transcriptional control (e.g.,<br>
repression and activation) can be physically separated from the portion responsible for<br>
24<br><br>
WO 2006/033859 PCT/US2005/032157<br>
DNA binding, and retains full functionality when tethered to other polyp eptides, for<br>
example, other DNA-binding domains. Accordingly, a nuclear hormone receptor<br>
transcription control domain can be fused to a DNA-binding domain (e.g., a zinc<br>
finger protein) such that the transcriptional regulatory activity of the receptor can be<br>
targeted to a chromosomal region of interest (e.g., a gene) by virtue of the DNA-<br>
binding domain.<br>
Moreover, the structure of TR and other nuclear hormone receptors can be<br>
altered, either naturally or through recombinant techniques, such that it loses all<br>
capacity to respond to hormone (thus losing its ability to drive transcriptional<br>
activation), but retains the ability to effect transcriptional repression. This approach is<br>
exemplified by the transcriptional regulatory properties of the oncoprotein v-ErbA.<br>
The v-ErbA protein is one of the two proteins required for leukemic transformation of<br>
immature red blood cell precursors in young chicks by the avian erythroblastosis<br>
virus. TR is a major regulator of erythropoiesis (Beug et al., Biochim Biophys Acta<br>
1288(3):M35-47 (1996); in particular, in its unliganded state, it represses genes<br>
required for cell cycle arrest and the differentiated state. Thus, the administration of<br>
thyroid hormone to immature erythroblasts leads to their rapid differentiation. The v-<br>
ErbA oncoprotein is an extensively mutated version of TR; these mutations include:<br>
(i) deletion of 12 amino-terminal amino acids; (ii) fusion to the gag oncoprotein; (iii)<br>
several point mutations in the DNA binding domain that alter the DNA binding<br>
specificity of the protein relative to its parent, TR, and impair its ability to<br>
heterodimerize with the retinoid X receptor; (iv) multiple point mutations in the<br>
ligand-binding domain of the protein that effectively eliminate the capacity to bind<br>
thyroid hormone; and (v) a deletion of a carboxy-terminal stretch of amino acids that<br>
is essential for transcriptional activation. Stunnenberg et al., Biochim Biophys Acta<br>
1423(1):F15-33 (1999). As a consequence of these mutations, v-ErbA retains the<br>
capacity to bind to naturally occurring TR target genes and is an effective<br>
transcriptional repressor when bound (Urnov et al., supra; Sap et al., Nature<br>
340:242-244(1989); and Ciana et al., EMBO J. 17(24):7382-7394 (1999). In<br>
contrast to TR, however, v-ErbA is completely insensitive to thyroid hormone, and<br>
thus maintains transcriptional repression in the presence of thyroid hormones or<br>
retinoids.<br>
Accordingly, in one aspect, v-ErbA or its functional fragments are used as a<br>
repression domain. In additional embodiments, TR or its functional domains are used<br>
25<br><br>
WO 2006/033859 PCT/US2005/032157<br>
as a repression domain in the absence of ligand and/or as an activation domain in the<br>
presence of ligand (e.g., 3,5,3'-triiodo-L-thyronine or T3). Thus, TR can be used as a<br>
switchable functional domain (i.e., a bifunctional domain); its activity (activation or<br>
repression) being dependent upon the presence or absence (respectively) of ligand.<br>
Additional exemplary repression domains are obtained from the DAX protein<br>
and its functional fragments. Zazopoulos et al., Nature 390:311-315 (1997). In<br>
particular, the C-terminal portion of DAX-1, including amino acids 245-470, has been<br>
shown to possess repression activity. Altincicek et al., J. Biol. Chem. 275:7662-7667<br>
(2000). A further exemplary repression domain is the RBP1 protein and its functional<br>
fragments. Lai et al., Oncogene 18:2091-2100 (1999); Lai et al., Mol. Cell. Biol.<br>
19:6632-6641 (1999); Lai et al., Mol. Cell. Biol. 21:2918-2932 (2001) and WO<br>
01/04296. The full-length RBP1 polypeptide contains 1257 amino acids. Exemplary<br>
functional fragments of RBP1 are a polypeptide comprising amino acids 1114-1257,<br>
and a polypeptide comprising amino acids 243-452.<br>
Members of the TIBG family of transcription factors contain three repression<br>
domains known as Rl, R2 and R3. Repression by TIEG family proteins is achieved<br>
at least in part through recruitment of mSIN3A histone deacetylases complexes.<br>
Cook et al. (1999) J. Biol. Chem. 274:29,500-29,504; Zhang et al. (2001) Mol. Cell.<br>
Biol. 21:5041-5049. Any or all of these repression domains (or their functional<br>
fragments) can be fused alone, or in combination with additional repression domains<br>
(or their functional fragments), to a DNA-binding domain to generate a targeted<br>
exogenous repressor molecule.<br>
Furthermore, the product of the human cytomegalovirus (HCMV) UL34 open<br>
reading frame acts as a transcriptional repressor of certain HCMV genes, for example,<br>
the US3 gene. LaPierre et al. (2001) J. Virol. 75:6062-6069. Accordingly, the UL34<br>
gene product, or functional fragments thereof, can be used as a component of a fusion<br>
polypeptide also comprising a zinc finger binding domain. Nucleic acids encoding<br>
such fusions are also useful in the methods and compositions disclosed herein.<br>
Yet another exemplary repression domain is the CDF-1 transcription factor<br>
and/or its functional fragments. See, for example, WO 99/27092.<br>
The Ikaros family of proteins are involved in the regulation of lymphocyte<br>
development, at least in part by transcriptional repression. Accordingly, an Ikaros<br>
family member (e.g., Ikaros, Aiolos) or a functional fragment thereof, can be used as<br>
26<br><br>
WO 2006/033859 PCT/US2005/032157<br>
a repression domain. See, for example, Sabbattini et al. (2001) EMBO J. 20:2812-<br>
2822.<br>
The yeast Ashlp protein comprises a transcriptional repression domain.<br>
Maxon et al. (2001) Proc. Natl. Acad. Sci. USA 98:1495-1500. Accordingly, the<br>
Ashlp protein, its functional fragments, and homologues of Ashlp, such as those<br>
found, for example, in, vertebrate, mammalian, and plant cells, can serve as a<br>
repression domain for use in the methods and compositions disclosed herein.<br>
Additional exemplary repression domains include those derived from histone<br>
deacetylases (HDACs, e.g., Class I HDACs, Class II HDACs, SIR-2 homologues),<br>
HDAC-interacting proteins (e.g., SIN3, SAP30, SAP15, NCoR, SMRT, RB, p107,<br>
p130, RBAP46/48, MTA, Mi-2, Brgl, Brm), DNA-cytosine methyltransferases (e.g.,<br>
Dnmt1, Dnmt3a, Dnmt3b), proteins that bind methylated DNA (e.g., MBD1, MBD2,<br>
MBD3, MBD4, MeCP2, DMAP1), protein methyltransferases (e.g., lysine and<br>
arginine methylases, SuVar homologues such as Suv39H1), polycomb-type repressors<br>
(e.g., Bmi-1, eed1, RING1, RYBP, E2F6, Mel18, YY1 and CtBP), viral repressors<br>
(e.g., adenovirus Elb 55K protein, cytomegalovirus UL34 protein, viral oncogenes<br>
such as v-erbA), hormone receptors (e.g., Dax-1, estrogen receptor, thyroid hormone<br>
receptor), and repression domains associated with naturally-occurring zinc finger<br>
proteins (e.g., WT1, KAP1). Further exemplary repression domains include members<br>
of the polycomb complex and their homologues, HPH1, HPH2, HPC2, NC2, groucho,<br>
Eve, tramtrak, mHP1, SIP1, ZEB1, ZEB2, and Enxl/Ezh2. In all of these cases,<br>
either the full-length protein or a functional fragment can be used as a repression<br>
domain for fusion to a zinc finger binding domain. Furthermore, any homologues of<br>
the aforementioned proteins can also be used as repression domains, as can proteins<br>
(or their functional fragments) that interact with any of the aforementioned proteins.<br>
Additional repression domains, and exemplary functional fragments, are as<br>
follows. Hesl is a human homologue of the Drosophila hairy gene product and<br>
comprises a functional fragment encompassing ammo acids 910-1014. In particular, a<br>
WRPW (trp-arg-pro-trp) motif can act as a repression domain. Fisher et al. (1996)<br>
Mol. Cell. Biol. 16:2670-2677.<br>
The TLE1, TLE2 and TLE3 proteins are human homologues of the<br>
Drosophila groucho gene product. Functional fragments of these proteins possessing<br>
repression activity reside between amino acids 1-400. Fisher et al., supra.<br>
27<br><br>
WO 2006/033859 PCT/US2005/032157<br>
The Tbx3 protein possesses a functional repression domain between amino<br>
acids 524-721. He et al. (1999) Proc. Natl. Acad. Sci. USA 96:10,212-10,217. The<br>
Tbx2 gene product is involved in repression of the p14/p16 genes and contains a<br>
region between amino acids 504-702 that is homologous to the repression domain of<br>
Tbx3; accordingly Tbx2 and/or this functional fragment can be used as a repression<br>
domain. Carreira et al. (1998) Mol. Cell. Biol. 18:5,099-5,108.<br>
The human Ezh2 protein is a homologue of Drosophila enhancer of zeste and<br>
recruits the eed1 polycomb-type repressor. A region of the Ezh2 protein comprising<br>
amino acids 1-193 can interact with eed1 and repress transcription; accordingly Ezh2<br>
and/or this functional fragment can be used as a repression domain. Denisenko et al.<br>
(1998) Mol. Cell. Biol. 18:5634-5642.<br>
The RYBP protein is a corepressor that interacts with polycomb complex<br>
members and with the YY1 transcription factor. A region of RYBP comprising<br>
amino acids 42-208 has been identified as functional repression domain. Garcia et al.<br>
(1999)EMBO J. 18:3404-3418.<br>
The RING finger protein RING1A is a member of two different vertebrate<br>
polycomb-type complexes, contains multiple binding sites for various components of<br>
the polycomb complex, and possesses transcriptional repression activity.<br>
Accordingly, RTNG1A or its functional fragments can serve as a repression domain.<br>
Satjin et al. (1997) Mol. Cell. Biol. 17:4105-4113.<br>
The Bmi-1 protein is a member of a vertebrate polycomb complex and is<br>
involved in transcriptional silencing. It contains multiple binding sites for various<br>
polycomb complex components. Accordingly, Bmi-1 and its functional fragments are<br>
useful as repression domains. Gunster et al. (1997) Mol. Cell., Biol. 17:2326-2335;<br>
Hemenway et al. (1998) Oncogene 16:2541-2547.<br>
The E2F6 protein is a member of the mammalian Bmi-1 -containing polycomb<br>
complex and is a transcriptional repressor that is capable or recruiting RYBP, Bmi-1<br>
and RING1A. A functional fragment of E2F6 comprising amino acids 129-281 acts<br>
as a transcriptional repression domain. Accordingly, E2F6 and its functional<br>
fragments can be used as repression domains. Trimarchi et al. (2001) Proc. Natl.<br>
Acad. Sci. USA 98:1519-1524.<br>
The eed1 protein represses transcription at least in part through recruitment of<br>
histone deacetylases (e.g., HDAC2). Repression activity resides in both the N- and C-<br>
28<br><br>
WO 2006/033859 PCT/US2005/032157<br>
terminal regions of the protein. Accordingly, eed1 and its functional fragments can be<br>
used as repression domains, van der Vlag et al. (1999) Nature Genet. 23:474-478.<br>
The CTBP2 protein represses transcription at least in part through recruitment<br>
of an HPC2-polycomb complex. Accordingly, CTBP2 and its functional fragments<br>
are useful as repression domains. Richard et al. (1999) Mol. Cell. Biol. 19:777-787.<br>
Neuron-restrictive silencer factors are proteins that repress expression of<br>
neuron-specific genes. Accordingly, a NRSF or functional fragment thereof can serve<br>
as a repression domain. See, for example, US Patent No. 6,270,990.<br>
Additional repression domains include PLZF, BCL-6, BAZF, ZNF274, PRH,<br>
TEL, TGIF, and G9A.<br>
It will be clear to those of skill in the art that, in the formation of a fusion<br>
protein (or a nucleic acid encoding same) between a DNA-binding domain and a<br>
functional domain, either a repressor or a molecule that interacts with a repressor is<br>
suitable as a functional domain. Essentially any molecule capable of recruiting a<br>
repressive complex and/or repressive activity (such as, for example, histone<br>
deacetylation) to the target gene is useful as a repression domain of a fusion protein.<br>
Additional exemplary activation domains include, but are not limited to, p300,<br>
CBP, PCAF, SRC1 PvALF, AtHD2A and ERF-2. See, for example, Robyr et al.<br>
(2000) Mol. Endocrinol. 14:329-347; Collingwood et al. (1999) J. Mol. Endocrinol.<br>
23:255-275; Leo.et al. (2000) Gene 245:1-11; Manteuffel-Cymborowska (1999)<br>
Acta Biochim. Pol. 46:77-89; McKenna et al. (1999) J. Steroid Biochem. Mol. Biol.<br>
69:3-12; Malik et al. (2000) Trends Biochem. Sci. 25:277-283; and Lemon et al.<br>
(1999) Curr. Opin. Genet. Dev. 9:499-504. Additional exemplary activation domains<br>
include, but are not limited to, OsGAI, HALF-1, C1, AP1, ARF-5, -6, -7, and -8,<br>
CPRF1, CPRF4, MYC-RP/GP, and TRAB1. See, for example, Ogawa et al. (2000)<br>
Gene 245:21-29; Okanami et al. (1996) Genes Cells 1:87-99; Goff et al. (1991)<br>
Genes Dev. 5:298-309; Cho et al. (1999) Plant Mol. Biol. 40:419-429; Ulmason et<br>
al. (1999) Proc. Natl. Acad. Sci. USA 96:5844-5849; Sprenger-Haussels et al. (2000)<br>
Plant J. 22:1-8; Gong et al. (1999) Plant Mol. Biol. 41:33-44; and Hobo et al. (1999)<br>
Proc. Natl. Acad. Sci. USA 96:15,348-15,353.<br>
Additional transcriptional activation domains can be obtained from the<br>
following proteins: ATF2, myc, GATA-1, GATA-3, NF-E2, Octl, CTF1, Spl, GR-<br>
AF1, a zeste deletion, HSF-1, p53, myoD, and CARβ.<br>
29<br><br>
WO 2006/033859 PCT/US2005/032157<br>
It will be clear to those of skill in the art that, in the formation of a fusion<br>
protein (or a nucleic acid encoding same) between a DNA-binding domain and a<br>
functional domain, either an activation domain or a molecule that interacts with an<br>
activation domain is suitable as a functional domain. Essentially any molecule<br>
capable of recruiting an activating complex and/or activating activity (such as, for<br>
example, histone acetylation) to the target gene is useful as an activating domain of a<br>
fusion protein.<br>
Insulator domains, localization domains, and chromatin remodeling proteins<br>
such as ISWI-containing domains and/or methyl binding domain proteins suitable for<br>
use as functional domains in fusion molecules are described, for example, in co-<br>
owned U.S. Patent Applications 2002/0115215 and 2003/0082552 and in co-owned<br>
WO 02/44376.<br>
In a further embodiment, a DNA-binding domain (e.g., a zinc finger domain)<br>
is fused to a bifunctional domain (BFD). A bifunctional domain is a transcriptional<br>
regulatory domain whose activity depends upon interaction of the BFD with a second<br>
molecule. The second molecule can be any type of molecule capable of influencing<br>
the functional properties of the BFD including, but not limited to, a compound, a<br>
small molecule, a peptide, a protein, a polysaccharide or a nucleic acid. An<br>
exemplary BFD is the ligand binding domain of the estrogen receptor (ER). In the<br>
presence of estradiol, the ER ligand binding domain acts as a transcriptional activator;<br>
while, in the absence of estradiol and the presence of tamoxifen or 4-hydroxy-<br>
tamoxifen, it acts as a transcriptional repressor. Another example of a BFD is the<br>
thyroid hormone receptor (TR) ligand binding domain which, in the absence of<br>
ligand, acts as a transcriptionai repressor and in the presence of thyroid hormone (T3),<br>
acts as a transcriptional activator. An additional BFD is the glucocorticoid receptor<br>
(GR) ligand binding domain. In the presence of dexamethasone, this domain acts as a<br>
transcriptional activator; while, in the presence of RU486, it acts as a transcriptional<br>
repressor. An additional exemplary BFD is the ligand binding domain of the retinoic<br>
acid receptor. In the presence of its ligand all-trans-retinoic acid, the retinoic acid<br>
receptor recruits a number of co-activator complexes and activates transcription. In<br>
the absence of ligand, the retinoic acid receptor is not capable of recruiting<br>
transcriptional co-activators. Additional BFDs are known to those of skill in the art.<br>
See, for example, US Patent Nos. 5,834,266 and 5,994,313 and PCT WO 99/10508.<br>
30<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Another class of functional domain derived from nuclear receptors are those<br>
whose functional activity is regulated by a non-natural ligand. These are often<br>
mutants or modified versions of naturally-occurring receptors and are sometimes<br>
referred to as "switchable" domains. For example, certain mutants of the<br>
progesterone receptor (PR) are unable to interact with their natural ligand, and are<br>
therefore incapable of being transcriptionally activated by progesterone. Certain of<br>
these mutants, however, can be activated by binding small molecules other than<br>
progesterone (one example of which is the antiprogestin mifepristone). Such non-<br>
natural but functionally competent ligands have been denoted anti-hormones. See,<br>
e.g., U.S. Patents 5,364,791; 5,874,534; 5,935,934; Wang et al., (1994) Proc. Natl.<br>
Acad. Sci. USA 91:8180-8184; Wang et al. (1997) Gene Ther. 4:432-441.<br>
Accordingly, a fusion comprising a targeted DNA-binding domain (e.g., ZFP),<br>
a functional domain, and a mutant PR ligand binding domain of this type can be used<br>
for mifepristone-dependent activation or repression of an endogenous gene of choice,<br>
by designing or selecting the DNA-binding domain such that it binds in or near the<br>
gene of choice. Such fusions can further comprise a degron domain. If the fusion<br>
contains an activation domain, mifepristone-dependent activation of gene expression<br>
is obtained; if the fusion contains a repression domain, mifepristone-dependent<br>
repression of gene expression is obtained. Additionally, polynucleotides encoding<br>
such fusion proteins are provided, as are vectors comprising such polynucleotides and<br>
cells comprising such polynucleotides and vectors. It will be clear to those of skill in<br>
the art that modified or mutant versions of receptors other than PR can also be used as<br>
switchable domains. See, for example, Tora et al. (1989) EMBO J. 8:1981-1986.<br>
Host cells, vectors and promoters<br>
In the practice of the disclosed methods, an engineered zinc finger protein, as<br>
described above, is expressed in a cell and binds to a target site to regulate<br>
transcription. A protein can be expressed in a cell, e.g., by delivering the protein to<br>
the cell or by delivering a polynucleotide encoding the protein to a cell. If a DNA<br>
molecule encoding the protein is delivered to a cell, it is transcribed into a mRNA<br>
molecule which can be translated to produce the protein. Alternatively, a RNA<br>
molecule can be delivered to a cell and is translated to generate the protein. Methods<br>
for polynucleotide and polypeptide delivery to cells are known in the art and<br>
exemplary methods are presented elsewhere in this disclosure. Expression of a<br>
31<br><br>
WO 2006/033859 PCT/US2005/032157<br>
protein in a cell can also be accomplished by transfecting a cell with a nucleic acid<br>
encoding the protein, and selecting a cell line in which the nucleic acid has stably<br>
integrated into a chromosome or is otherwise stably and heritably maintained in the<br>
cell. In these cases, expression of the protein encoded by the stably-maintained<br>
sequences can be inducible (e.g., tetracycline- and doxycycline-regulated systems) or<br>
constitutive.<br>
The disclosed compositions and methods can be used for the transcriptional<br>
regulation of any nucleotide sequence in any cell, including cultured cells, primary<br>
cells, cells in an organism and cells removed from an organism which are then<br>
returned to the organism after delivery of a protein or a polynucleotide encoding the<br>
protein to a cell. In this regard, transcription of both endogenous and exogenous<br>
sequences can be regulated. To regulate transcription of endogenous sequences, one<br>
or more copies of SEQ ID NO: 1 can be inserted into or in the vicinity of the<br>
endogenous sequence to be regulated. Exemplary methods for targeted insertion of<br>
exogenous sequences into cellular genomes are disclosed in U.S. Patent Application<br>
Publication No. 2003/0232410 (Dec. 18,2003), PCT Publication WO 03/87341 (Oct.<br>
23, 2003) and co-owned U.S. Patent Application Serial No. 10/912,932, filed on<br>
August 6, 2004, the disclosures of which are hereby incorporated by reference in their<br>
entireties for all purposes.<br>
As noted, the disclosed methods and compositions can also be used to regulate<br>
transcription of exogenous sequences (e.g., a vector comprising a cDNA sequence<br>
which is introduced into a cell). For example, a cDNA can be cloned into a vector<br>
containing a promoter, e.g., a SRα promoter, such that transcription of the cDNA<br>
sequences is controlled by the promoter. In these cases, the cDNA-containing vector<br>
is introduced into a cell, and an engineered ZFP as disclosed herein (or a<br>
polynucleotide encoding an engineered ZFP as disclosed herein) is introduced into the<br>
same cell. Alternatively, the cDNA is introduced into a stable cell line containing an<br>
integrated (or otherwise stably maintained) copy of a polynucleotide encoding an<br>
engineered ZFP as disclosed herein. In another alternative, an engineered ZFP or a<br>
polynucleotide encoding an engineered ZFP is introduced into a cell containing an<br>
integrated (or otherwise stably maintained) copy of a polynucleotide sequence<br>
comprising a cDNA.<br>
In cells containing stably integrated sequences encoding an engineered ZFP, a<br>
number of exogenous transcription units, each containing one or more target sites for<br>
32<br><br>
WO 2006/033859 PCT/US2005/032157<br>
the ZFP, can be introduced (e.g., by transfection) to obtain coordinate regulation of<br>
the exogenous transcription units. Also in such cells, one or more target sites can be<br>
introduced upstream, within or adjacent to any number of endogenous transcription<br>
units, to obtain coordinate regulation of multiple genes.<br>
Suitable vectors for the propagation of nucleotide sequences that are<br>
exogenous to a cell, and promoters for regulation of the transcription of such<br>
exogenous nucleotide sequences, are known in the art. Exemplary promoters include<br>
the SRα and CMV promoters. Regulation of transcription of sequences operatively<br>
linked to a SRα promoter can be accomplished directly using the disclosed proteins,<br>
since a copy of the target site SEQ ID NO:1 is present in the SRα promoter. Higher<br>
levels of transcription (e.g., for overexpression of proteins) are achieved by placing<br>
additional copies of SEQ ID NO:1 into the SRα promoter. Furthermore, modification<br>
of a promoter other than SRα (e.g., the CMV promoter), by insertion of one or more<br>
copies of SEQ ID NO:1, leads to expression levels that are higher than those obtained<br>
with the unmodified promoter.<br>
The disclosed methods and compositions can be used in any type of cell<br>
including, but not limited to, prokaryotic cells, fungal cells, Archaeal cells, plant cells,<br>
insect cells, animal cells, vertebrate cells, mammalian cells and human cells. Suitable<br>
cell lines for protein expression are known to those of skill in the art and include, but<br>
are not limited to COS, CHO (e.g., CHO-S, CHO-K1, CHO-DG44, CHO-DUXB11),<br>
VERO, MDCK, WI38, V79, B14AF28-G3, BHK, HaK, NSO, SP2/0-Ag14, HeLa,<br>
HEK293 (e.g., HEK293-F, HEK293-H, HEK293-T), perC6, insect cells such as<br>
Spodoptera fugiperda (Sf), and fungal cells such as Saccharomyces, Pischia and<br>
Schizosaccharomyces. Progeny, variants and derivatives of these cell lines can also<br>
be used.<br>
Exemplary promoters include SRa, CMV, phosphoglycerate kinase (PGK),<br>
human ubiquitin C (UBC), elongation factor1α (EF-1α), herpes thymidine kinase<br>
(TK), SV40 early and late promoters, human keratin-14 (K14) and Rous Sarcoma<br>
virus (RSV).<br>
In certain embodiments, two or more promoters can be regulated with the<br>
same engineered ZFP, provided that one or more target sites for the ZFP is present in<br>
or near each promoter. This is particularly useful for recombinant production of<br>
antibodies, as sequences encoding a heavy chain can be placed under the<br>
transcriptional control of a first promoter and sequences encoding a light chain can be<br>
33<br><br>
WO 2006/033859 PCT/US2005/032157<br>
placed under the transcriptional control of a second promoter. The two promoters can<br>
be the same, or they can be different promoters. In additional embodiments, two<br>
copies of the same promoter, each containing a different number of ZFP target sites,<br>
can be used to obtain differential expression of sequences operatively linked to each<br>
promoter. Two different promoters, each containing a different number of target<br>
sites, can also be used.<br>
It will be clear to those of skill in the art that it is possible to design a ZFP to<br>
bind to any predetermined target sequence; and therefore is possible to use one or<br>
more copies of such a sequence, along with a ZFP the binds to the sequence, to<br>
regulate transcription for e.g. overexpression.<br>
Fusion nucleic acids and expression vectors<br>
In certain embodiments, a fusion polypeptide is encoded by a fusion nucleic<br>
acid. In such cases, the nucleic acid can be cloned into intermediate vectors for<br>
transformation into prokaryotic or eukaryotic cells for replication and/or expression.<br>
Intermediate vectors for storage or manipulation of the fusion nucleic acid or<br>
production of fusion protein can be prokaryotic vectors, (e.g., plasmids), shuttle<br>
vectors, insect vectors, or viral vectors for example. A fusion nucleic acid can also<br>
cloned into an expression vector, for administration to a bacterial cell, fungal cell,<br>
protozoal cell, plant cell, or animal cell, preferably a mammalian cell, more preferably<br>
a human cell.<br>
A nucleic acid encoding a fusion protein can be cloned into a vector for<br>
transformation into prokaryotic or eukaryotic cells for replication and/or expression.<br>
Vectors can be prokaryotic vectors, e.g., plasmids, or shuttle vectors, insect vectors,<br>
or eukaryotic vectors. A nucleic acid encoding a ZFP can also be cloned into an<br>
expression vector, for administration to a plant cell, animal cell, preferably a<br>
mammalian cell or a human cell, fungal cell, bacterial cell, or protozoal cell.<br>
To obtain expression of a cloned gene or nucleic acid, sequences encoding a<br>
fusion protein are typically subcloned into an expression vector that contains a<br>
promoter to direct transcription. Suitable bacterial and eukaryotic promoters are well<br>
known in the art and described, e.g., in Sambrook et al., Molecular Cloning, A<br>
Laboratory Manual (2nd ed. 1989; 3rd ed., 2001); Kriegler, Gene Transfer and<br>
Expression: A Laboratory Manual (1990); and Current Protocols in Molecular<br>
Biology (Ausubel et al., supra. Bacterial expression systems for expressing the ZFP<br>
34<br><br>
WO 2006/033859 PCT/US2005/032157<br>
are available in, e.g., E. coli, Bacillus sp., and Salmonella (Palva et al., Gene 22:229-<br>
235 (1983)). Kits for such expression systems are commercially available.<br>
Eukaryotic expression systems for mammalian cells, yeast, and insect cells are well<br>
known by those of skill in the art and are also commercially available.<br>
The promoter used to direct expression of a protein-encoding nucleic acid<br>
depends on the particular application. For example, a strong constitutive promoter is<br>
typically used for expression and purification of protein. In contrast, when a protein<br>
is administered in vivo for gene regulation, either a constitutive or an inducible<br>
promoter is used, depending on the particular use of the protein. In addition, a<br>
preferred promoter for administration of a protein can be a weak promoter, such as<br>
HSV TK or a promoter having similar activity. The promoter typically can also<br>
include elements that are responsive to transactivation, e.g., hypoxia response<br>
elements, Gal4 response elements, lac repressor response element, and small molecule<br>
control systems such as tet-regulated systems and the RU-486 system (see, e.g.,<br>
Gossen &amp; Bujard, PNAS 89:5547 (1992); Oligino et al., Gene Ther. 5:491-496<br>
(1998); Wang et al., Gene Ther. 4:432-441 (1997); Neering et al., Blood 88:1147-<br>
1155 (1996); and Rendahl et al., Nat. Biotechnol. 16:757-761 (1998)).<br>
In addition to the promoter, the expression vector typically contains a<br>
transcription unit or expression cassette that contains all the additional elements<br>
required for the expression of the nucleic acid in host cells, either prokaryotic or<br>
eukaryotic. A typical expression cassette thus contains a promoter operably linked,<br>
e.g., to a nucleic acid sequence encoding the ZFP, and signals required, e.g., for<br>
efficient polyadenylation of the transcript, transcriptional termination, ribosome<br>
binding sites, or translation termination. Additional elements of the cassette may<br>
include, e.g., enhancers, and heterologous splicing signals.<br>
The particular expression vector used to transport the genetic information into<br>
the cell is selected with regard to the intended use of the protein, e.g., expression in<br>
plants, animals, bacteria, fungus, protozoa, etc. (see expression vectors described<br>
below). Standard bacterial expression vectors include plasmids such as pBR322-<br>
based plasmids, pSKF, pET23D, and commercially available fusion expression<br>
systems such as GST and LacZ. An exemplary fusion protein is the maltose binding<br>
protein, "MBP." Such fusion proteins facilitate purification of the protein. Epitope<br>
tags, e.g., c-myc, hemagglutinin (HA) or FLAG, can also be added to recombinant<br>
35<br><br>
WO 2006/033859 PCT/US2005/032157<br>
proteins to provide convenient methods of isolation, for monitoring expression, and<br>
for monitoring cellular and subcellular localization.<br>
Expression vectors containing regulatory elements from eukaryotic viruses are<br>
often used in eukaryotic expression vectors, e.g., SV40 vectors, papilloma virus<br>
vectors, and vectors derived from Epstein-Barr virus. Other exemplary eukaryotic<br>
vectors include pMSG, pAV009/A+, pMTO10/A+, pMAMneo-5, baculovirus<br>
pDSVE, and any other vector allowing expression of proteins under the direction of<br>
the SV40 early promoter, SV40 late promoter, metallothionein promoter, murine<br>
mammary tumor virus promoter, Rous sarcoma virus promoter, polyhedrin promoter,<br>
or other promoters shown effective for expression in eukaryotic cells.<br>
Some expression systems have markers for selection of stably transfected cell<br>
lines such as thymidine kinase, hygromycin B phosphotransferase, and dihydrofolate<br>
reductase. High yield expression systems are also suitable, such as using a<br>
baculovirus vector in insect cells, with a protein coding sequence under the direction<br>
of the polyhedrin promoter or other strong baculovirus promoters.<br>
The elements that are typically included in expression vectors also include a<br>
replicon that functions in E. coli or other prokaryotic bacteria, a gene encoding<br>
antibiotic resistance to permit selection of bacteria that harbor recombinant plasmids,<br>
and unique restriction sites in nonessential regions of the plasmid to allow insertion of<br>
recombinant sequences.<br>
Standard transfection methods are used to produce bacterial, mammalian,<br>
yeast or insect cell lines that express large quantities of protein, which are then<br>
purified using standard techniques (see, e.g., Colley et al., J. Biol. Chem. 264:17619-<br>
17622 (1989); Guide to Protein Purification, in Methods in Enzymology, vol. 182<br>
(Deutscher, ed., 1990)). Transformation of eukaryotic and prokaryotic cells are<br>
performed according to standard techniques (see, e.g., Morrison, J. Bad. 132:349-351<br>
(1977); Clark-Curtiss &amp; Curtiss, Methods in Enzymology 101:347-362 (Wu et al., eds,<br>
1983).<br>
Any of the well known procedures for introducing foreign nucleotide<br>
sequences into host cells may be used. These include the use of calcium phosphate<br>
transfection, polybrene, protoplast fusion, electroporation, liposomes, microinjection,<br>
naked DNA, plasmid vectors, viral vectors, both episomal and integrative, and any of<br>
the other well known methods for introducing cloned genomic DNA, cDNA,<br>
synthetic DNA or other foreign genetic material into a host cell (see, e.g., Sambrook<br>
36<br><br>
WO 2006/033859 PCT/US2005/032157<br>
et al., supra). It is only necessary that the particular genetic engineering procedure<br>
used be capable of successfully introducing at least one gene into the host cell capable<br>
of expressing the protein of choice.<br>
Nucleic acids encoding fusion proteins and delivery to cells<br>
Conventional viral and non-viral based gene transfer methods can be used to<br>
introduce nucleic acids encoding fusion proteins and/or engineered ZFPs in cells (e.g.,<br>
mammalian cells) and target tissues. Such methods can also be used to administer<br>
nucleic acids encoding fusion proteins to cells in vitro. In certain embodiments,<br>
nucleic acids encoding fusion protiens are administered for in vivo or ex vivo gene<br>
therapy uses. Non-viral vector delivery systems include DNA plasmids, naked<br>
nucleic acid, and nucleic acid complexed with a delivery vehicle such as a liposome<br>
or poloxamer. Viral vector delivery systems include DNA and RNA viruses, which<br>
have either episomal or integrated genomes after delivery to the cell. For a review of<br>
gene therapy procedures, see Anderson, Science 256:808-813 (1992); Nabel &amp;<br>
Feigner, TIBTECH 11:211-217 (1993); Mitani &amp; Caskey, TIBTECH 11:162-166<br>
(1993); Dillon, TIBTECH 11:167-175 (1993); Miller, Nature 357:455-460 (1992);<br>
Van Brunt, Biotechnology 6(10):1149-1154 (1988); Vigne, Restorative Neurology<br>
and Neuroscience 8:35-36 (1995); Kremer &amp; Perricaudet, British Medical Bulletin<br>
51(1):31-44 (1995); Haddada et al., in Current Topics in Microbiology and<br>
Immunology Doerfler and Böhm (eds) (1995); and Yu et al., Gene Therapy 1:13-26<br>
(1994).<br>
Methods of non-viral delivery of nucleic acids encoding engineered ZFPs<br>
include electroporation, lipofection, microinjection, biolistics, virosomes, liposomes,<br>
immunoliposomes, polycation or lipid:nucleic acid conjugates, naked DNA, artificial<br>
virions, and agent-enhanced uptake of DNA. Lipofection is described in e.g., US<br>
5,049,386, US 4,946,787; and US 4,897,355) and lipofection reagents are sold<br>
commercially (e.g., Transfectam™, Lipofectin™, Lipofectamine®). Cationic and<br>
neutral lipids that are suitable for efficient receptor-recognition lipofection of<br>
polynucleotides include those of Feigner, WO 91/17424, WO 91/16024. Delivery can<br>
be to cells (ex vivo administration) or target tissues (in vivo administration).<br>
The preparation of lipid:nucleic acid complexes, including targeted liposomes<br>
such as immunolipid complexes, is well known to one of skill in the art (see, e.g.,<br>
37<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Crystal, Science 270:404-410 (1995); Blaese et al., Cancer Gene Ther. 2:291-297<br>
(1995); Behr et al., Bioconjugate Chem. 5:382-389 (1994); Remy et al., Bioconjugate<br>
Chem. 5:647-654 (1994); Gao et al., Gene Therapy 2:710-722 (1995); Ahmad et al.,<br>
Cancer Res. 52:4817-4820 (1992); U.S. Pat Nos. 4,186,183,4,217,344, 4,235,871,<br>
4,261,975, 4,485,054,4,501,728,4,774,085,4,837,028, and 4,946,787).<br>
The use of RNA or DNA viral based systems for the delivery of nucleic acids<br>
encoding fusion proteins and/or engineered ZFPs take advantage of highly evolved<br>
processes for targeting a virus to specific cells in the body and trafficking the viral<br>
payload to the nucleus. Viral vectors can be administered directly to patients (in vivo)<br>
or they can be used to treat cells in vitro and the modified cells are administered to<br>
patients (ex vivo). Conventional viral based systems for the delivery of ZFPs include,<br>
but are not, limited to, retroviral, lentivirus, adenoviral, adeno-associated, vaccinia and<br>
herpes simplex virus vectors for gene transfer. Integration in the host genome is<br>
possible with the retrovirus, lentivirus, and adeno-associated virus gene transfer<br>
methods, often resulting in long term expression of the inserted transgene.<br>
Additionally, high transduction efficiencies have been observed in many different cell<br>
types and target tissues.<br>
The tropism of a retrovirus can be altered by incorporating foreign envelope<br>
proteins, expanding the potential target population of target cells. Lentiviral vectors<br>
are retroviral vectors that are able to transduce or infect non-dividing cells and<br>
typically produce high viral titers. Selection of a retroviral gene transfer system<br>
depends on the target tissue. Retroviral vectors are comprised of as-acting long<br>
terminal repeats with packaging capacity for up to 6-10 kb of foreign sequence. The<br>
minimum cis-acting LTRs are sufficient for replication and packaging of the vectors,<br>
which are then used to integrate the therapeutic gene into the target cell to provide<br>
permanent transgene expression. Widely used retroviral vectors include those based<br>
upon murine leukemia virus (MuLV), gibbon ape leukemia virus (GaLV), Simian<br>
Immunodeficiency virus (SIV), human immunodeficiency virus (HTV), and<br>
combinations thereof (see, e.g., Buchscher et al., J. Virol. 66:2731-2739 (1992);<br>
Johann et., J. Virol. 66:1635-1640 (1992); Sommerfelt et al., Virol. 176:58-59<br>
(1990); Wilson et al., J. Virol. 63:2374-2378 (1989); Miller et al., J. Virol. 65:2220-<br>
2224 (1991); PCT/US94/05700).<br>
In applications in which transient expression of a fusion protein is preferred,<br>
adenoviral based systems can be used. Adenoviral based vectors are capable of very<br>
38<br><br>
WO 2006/033859 PCT/US2005/032157<br>
high transduction efficiency in many cell types and do not require cell division. With<br>
such vectors, high titer and high levels of expression have been obtained. This vector<br>
can be produced in large quantities in a relatively simple system. Adeno-associated<br>
virus ("AAV") vectors are also used to transduce cells with target nucleic acids, e.g.,<br>
in the in vitro production of nucleic acids and peptides, and for in vivo and ex vivo<br>
gene therapy procedures (see, e.g., West et al., Virology 160:38-47 (1987); U.S.<br>
Patent No. 4,797,368; WO 93/24641; Kotin, Human Gene Therapy 5:793-801 (1994);<br>
Muzyczka, J. Clin. Invest. 94:1351 (1994). Construction of recombinant AAV<br>
vectors are described in a number of publications, including U.S. Pat. No. 5,173,414;<br>
Tratschin et al., Mol. Cell. Biol. 5:3251-3260 (1985); Tratschin, et al., Mol. Cell. Biol.<br>
4:2072-2081 (1984); Hermonat &amp; Muzyczka, PNAS 81:6466-6470 (1984); and<br>
Samulski et al., J. Virol. 63:03822-3828 (1989).<br>
At least six viral vector approaches are currently available for gene transfer in<br>
clinical trials, which utilize approaches that involve complementation of defective<br>
vectors by genes inserted into helper ceil lines to generate the transducing agent.<br>
pLASN and MFG-S are examples of retroviral vectors that have been used in<br>
clinical trials (Dunbar et al., Blood 85:3048-305 (1995); Kohn et al., Nat. Med.<br>
1:1017-102 (1995); Malech et al., PNAS 94:22 12133-12138 (1997)). PA317/pLASN<br>
was the first therapeutic vector used in a gene therapy trial. (Blaese et al., Science<br>
270:475-480 (1995)). Transduction efficiencies of 50% or greater have been<br>
observed for MFG-S packaged vectors. (Ellem et al., Immunol Immunother. 44(1):10-<br>
20 (1997); Dranoff et al, Hum. Gene Ther. 1:111-2 (1997).<br>
Recombinant adeno-associated virus vectors (rAAV) are a promising<br>
alternative gene delivery systems based on the defective and nonpathogenic<br>
parvovirus adeno-associated type 2 virus. All vectors are derived from a plasmid that<br>
retains only the AAV 145 bp inverted terminal repeats flanking the transgene<br>
expression cassette. Efficient gene transfer and stable transgene delivery due to<br>
integration into the genomes of the transduced cell are key features for this vector<br>
system. (Wagner et al., Lancet 351:9117 1702-3 (1998), Kearns et al, Gene Ther.<br>
9:748-55 (1996)).<br>
Replication-deficient recombinant adenoviral vectors (Ad) can be produced at<br>
high titer and readily infect a number of different cell types. Most adenovirus vectors<br>
are engineered such that a transgene replaces the Ad El a, Elb, and/or E3 genes;<br>
subsequently the replication defective vector is propagated in human 293 cells that<br>
39<br><br>
WO 2006/033859 PCT/US2005/032157<br>
supply deleted gene function in trans. Ad vectors can transduce multiple types of<br>
tissues in vivo, including nondividing, differentiated cells such as those found in liver,<br>
kidney and muscle. Conventional Ad vectors have a large carrying capacity. An<br>
example of the use of an Ad vector in a clinical trial involved polynucleotide therapy<br>
for antitumor immunization with intramuscular injection (Sterman et al., Hum. Gene<br>
Ther. 7:1083-9 (1998)). Additional examples of the use of adenovirus vectors for<br>
gene transfer in clinical trials include Rosenecker et al., Infection 24:1 5-10 (1996);<br>
Sterman et al., Hum. Gene Ther. 9:7 1083-1089 (1998); Welsh et al., Hum. Gene<br>
Ther. 2:205-18 (1995); Alvarez etal, Hum. Gene Ther. 5:597-613 (1997); Topf etal,<br>
Gene Ther. 5:507-513 (1998); Sterman et al., Hum. Gene Ther. 7:1083-1089 (1998).<br>
Packaging cells are used to form virus particles that are capable of infecting a<br>
host cell. Such cells include 293 cells, which package adenovirus, and \r2 cells or<br>
PA317 cells, which package retrovirus. Viral vectors used in gene therapy are usually<br>
generated by a producer cell line that packages a nucleic acid vector into a viral<br>
particle. The vectors typically contain the minimal virai sequences required for<br>
packaging and subsequent integration into a host (if applicable), other viral sequences<br>
being replaced by an expression cassette encoding the protein to be expressed. The<br>
missing viral functions are supplied in trans by the packaging cell line. For example,<br>
AAV vectors used in gene therapy typically only possess inverted terminal repeat<br>
(ITR) sequences from the AAV genome which are required for packaging and<br>
integration into the host genome. Viral DNA is packaged in a cell line, which<br>
contains a helper plasmid encoding the other AAV genes, namely rep and cap, but<br>
lacking ITR sequences. The cell line is also infected with adenovirus as a helper. The<br>
helper virus promotes replication of the AAV vector and expression of AAV genes<br>
from the helper plasmid. The helper plasmid is not packaged in significant amounts<br>
due to a lack of ITR sequences. Contamination with adenovirus can be reduced by,<br>
e.g., heat treatment to which adenovirus is more sensitive than AAV. Alternatively,<br>
adenovirus helper functions can be provided on a plasmid.<br>
In many gene therapy applications, it is desirable that the gene therapy vector<br>
be delivered with a high degree of specificity to a particular tissue type. Accordingly,<br>
a viral vector can be modified to have specificity for a given cell type by expressing a<br>
ligand as a fusion protein with a viral coat protein on the outer surface of the virus.<br>
The ligand is chosen to have affinity for a receptor known to be present on the cell<br>
type of interest. For example, Han et al., Proc. Natl Acad. Sci USA 92:9747-9751<br>
40<br><br>
WO 2006/033859 PCT/US2005/032157<br>
(1995), reported that Moloney murine leukemia virus can be modified to express<br>
human heregulin fused to gp70, and the recombinant virus infects certain human<br>
breast cancer cells expressing human epidermal growth factor receptor. This principle<br>
can be extended to other virus-target cell pairs, in which the target cell expresses a<br>
receptor and the virus expresses a fusion protein comprising a ligand for the cell-<br>
surface receptor. For example, filamentous phage can be engineered to display<br>
antibody fragments (e.g., Fab or Fv) having specific binding affinity for virtually any<br>
chosen cellular receptor. Although the above description applies primarily to viral<br>
vectors, the same principles can be applied to nonviral vectors. Such vectors can be<br>
engineered to contain specific uptake sequences which favor uptake by specific target<br>
cells.<br>
Gene therapy vectors can be delivered in vivo by administration to an<br>
individual patient, typically by systemic administration (e.g., intravenous,<br>
intraperitoneal, intramuscular, subdermal, or intracranial infusion) or topical<br>
application, as described below. Alternatively, vectors can be delivered to ceils ex<br>
vivo, such as cells explanted from an individual patient (e.g., lymphocytes, bone<br>
marrow aspirates, tissue biopsy) or universal donor hematopoietic stem cells,<br>
followed by reimplantation of the cells into a patient, usually after selection for cells<br>
which have incorporated the vector.<br>
Ex vivo cell transfection for diagnostics, research, or for gene therapy (e.g., via<br>
re-infusion of the transfected cells into the host organism) is well known to those of<br>
skill in the art. In certain embodiments, cells are isolated from the subject organism,<br>
transfected with a nucleic acid (gene or cDNA), and re-infused back into the subject<br>
organism (e.g., patient). Various cell types suitable for ex vivo transfection are well<br>
known to those of skill in the art (see, e.g., Freshney et al., Culture of Animal Cells, A<br>
Manual of Basic Technique (3rd ed. 1994)) and the references cited therein for a<br>
discussion of how to isolate and culture cells from patients).<br>
In one embodiment, stem cells are used in ex vivo procedures for cell<br>
transfection and gene therapy. The advantage to using stem cells is that they can be<br>
differentiated into other cell types in vitro, or can be introduced into a mammal (such<br>
as the donor of the cells) where they will engraft in the bone marrow. Methods for<br>
differentiating CD34+ cells in vitro into clinically important immune cell types using<br>
cytokines such a GM-CSF, IFN-y and TNF-a are known (see Inaba et al., J. Exp.<br>
Med. 176:1693-1702(1992)).<br>
41<br><br>
WO 2006/033859 PCTAJS2005/032157<br>
Stem cells are isolated for transduction and differentiation using known<br>
methods. For example, stem cells are isolated from bone marrow cells by panning the<br>
bone marrow cells with antibodies which bind unwanted cells, such as CD4+ and<br>
CD8+ (T cells), CD45+ (panB cells), GR-1 (granulocytes), and lad (differentiated<br>
antigen presenting cells) {see Inaba et al., J. Exp. Med. 176:1693-1702 (1992)).<br>
Vectors (e.g., retro viruses, adenoviruses, liposomes, etc.) containing<br>
therapeutic nucleic acids can also be administered directly to an organism for<br>
transduction of cells in vivo. Alternatively, naked DNA can be administered.<br>
Administration is by any of the routes normally used for introducing a molecule into<br>
ultimate contact with blood or tissue cells including, but not limited to, injection,<br>
infusion, topical application and electroporation. Suitable methods of administering<br>
such nucleic acids are available and well known to those of skill in the art, and,<br>
although more than one route can be used to administer a particular composition, a<br>
particular route can often provide a more immediate and more effective reaction than<br>
another route.<br>
Pharmaceutically acceptable carriers are determined in part by the particular<br>
composition being administered, as well as by the particular method used to<br>
administer the composition. Accordingly, there is a wide variety of suitable<br>
formulations of pharmaceutical compositions available, as described below {see, e.g.,<br>
Remington's Pharmaceutical Sciences, 17th ed., 1989).<br>
DNA constructs may be introduced into the genome of a desired plant host by<br>
a variety of conventional techniques. For reviews of such techniques see, for<br>
example, Weissbach &amp; Weissbach Methods for Plant Molecular Biology (1988,<br>
Academic Press, N.Y.) Section VIII, pp. 421-463; and Grierson &amp; Corey, Plant<br>
Molecular Biology (1988, 2d Ed.), Blackie, London, Ch. 7-9. For example, the DNA<br>
construct may be introduced directly into the genomic DNA of the plant cell using<br>
techniques such as electroporation and microinjection of plant cell protoplasts, or the<br>
DNA constructs can be introduced directly to plant tissue using biolistic methods,<br>
such as DNA particle bombardment (see, e.g., Klein et al (1987) Nature 327:70-73).<br>
Alternatively, the DNA constructs may be combined with suitable T-DNA flanking<br>
regions and introduced into a conventional Agrobacterium tumefaciens host vector.<br>
Agrobacterium tumefaciens-mediated transformation techniques, including disarming<br>
and use of binary vectors, are well described in the scientific literature. See, for<br>
example Horsch et al (1984) Science 233:496-498, and Fraley et al (1983) Proc. Nat'l.<br>
42<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Acad. Sci. USA 80:4803. The virulence functions of the Agrobacterium tumefaciens<br>
host will direct the insertion of the construct and adjacent marker into the plant cell<br>
DNA when the cell is infected by the bacteria using binary T DNA vector (Bevan<br>
(1984) Nuc. Acid Res. 12:8711-8721) or the co-cultivation procedure (Horsch et al<br>
(1985) Science 227:1229-1231). Generally, the Agrobacterium transformation system<br>
is used to engineer dicotyledonous plants (Bevan et al (1982) Ann. Rev. Genet<br>
16:357-384; Rogers et al (1986) Methods Enzymol. 118:627-641). The<br>
Agrobacterium transformation system may also be used to transform, as well as<br>
transfer, DNA to monocotyledonous plants and plant cells. See Hernalsteen et al<br>
(1984) EMBO J3:3039-3041; Hooykass-Van Slogteren et al (1984) Nature<br>
311:763-764; Grimsley et al (1987) Nature 325:1677-179; Boulton et al (1989) Plant<br>
Mol. Biol. 12:31-40.; and Gould et al (1991) Plant Physiol. 95:426-434.<br>
Alternative gene transfer and transformation methods include, but are not<br>
limited to, protoplast transformation through calcium-, polyethylene glycol (PEG)- or<br>
electroporation-mediated uptake of naked DNA (see Paszkowski et al. (1984) EMBO<br>
J3:2717-2722, Potrykus et al. (1985) Molec. Gen. Genet 199:169-177; Fromm et al.<br>
(1985) Proc. Nat. Acad. Sci. USA 82:5824-5828; and Shimamoto (1989) Nature<br>
338:274-276) and electroporation of plant tissues (D'Halluin et al. (1992) Plant Cell<br>
4:1495-1505). Additional methods for plant cell transformation include<br>
microinjection, silicon carbide mediated DNA uptake (Kaeppler et al. (1990) Plant<br>
Cell Reporter 9:415-418), and microprojectile bombardment (see Klein et al. (1988)<br>
Proc. Nat. Acad. Sci. USA 85:4305-4309; and Gordon-Kamm et al. (1990) Plant Cell<br>
2:603-618).<br>
Transformed plant cells which are produced by any of the above<br>
transformation techniques can be cultured to regenerate a whole plant which<br>
possesses the transformed genotype and thus the desired phenotype. Such<br>
regeneration techniques rely on manipulation of certain phytohormones in a tissue<br>
culture growth medium, typically relying on a biocide and/or herbicide marker which<br>
has been introduced together with the desired nucleotide sequences. Plant<br>
regeneration from cultured protoplasts is described in Evans, et al., "Protoplasts<br>
Isolation and Culture" in Handbook of Plant Cell Culture, pp. 124-176, Macmillian<br>
Publishing Company, New York, 1983; and Binding, Regeneration of Plants, Plant<br>
Protoplasts, pp. 21-73, CRC Press, Boca Raton, 1985. Regeneration can also be<br>
obtained from plant callus, explants, organs, pollens, embryos or parts thereof. Such<br>
43<br><br>
WO 2006/033859 PCT/US2005/032157<br>
regeneration techniques are described generally in Klee et al (1987) Ann. Rev. of<br>
Plant Phys. 38:467-486.<br>
Nucleic acids introduced into a plant cell can be used to confer desired traits<br>
on essentially any plant. A wide variety of plants and plant cell systems may be<br>
engineered for the desired physiological and agronomic characteristics described<br>
herein using the nucleic acid constructs of the present disclosure and the various<br>
transformation methods mentioned above. In certain embodiments, target plants and<br>
plant cells for engineering include, but are not limited to, those monocotyledonous<br>
and dicotyledonous plants, such as crops including grain crops (e.g., wheat, maize,<br>
rice, millet, barley), fruit crops (e.g., tomato, apple, pear, strawberry, orange), forage<br>
crops (e.g., alfalfa), root vegetable crops (e.g., carrot, potato, sugar beets, yam), leafy<br>
vegetable crops (e.g., lettuce, spinach); flowering plants (e.g., petunia, rose,<br>
chrysanthemum), conifers and pine trees (e.g., pine fir, spruce); plants used in<br>
phytoremediation (e.g., heavy metal accumulating plants); oil crops (e.g., sunflower,<br>
rape seed) and plants used for experimental purposes (e.g., Arabidopsis). Thus, the<br>
disclosed methods and compositions have use over a broad range of plants, including,<br>
but not limited to, species from the genera Asparagus, Avena, Brassica, Citrus,<br>
Citrullus, Capsicum, Cucurbita, Daucus, Glycine, Hordeum, Lactuca, Lycopersicon,<br>
Malus, Manihot, Nicotiana, Oryza, Persea, Pisum, Pyrus, Prunus, Raphanus, Secale,<br>
Solanum, Sorghum, Triticum, Vitis, Vigna, and Zea.<br>
One of skill in the art will recognize that after the expression cassette is stably<br>
incorporated in transgenic plants and confirmed to be operable, it can be introduced<br>
into other plants by sexual crossing. Any of a number of standard breeding<br>
techniques can be used, depending upon the species to be crossed.<br>
A transformed plant cell, callus, tissue or plant may be identified and isolated<br>
by selecting or screening the engineered plant material for traits encoded by the<br>
marker genes present on the transforming DNA. For instance, selection may be<br>
performed by growing the engineered plant material on media containing an<br>
inhibitory amount of the antibiotic or herbicide to which the transforming gene<br>
construct confers resistance. Further, transformed plants and plant cells may also be<br>
identified by screening for the activities of any visible marker genes (e.g., the<br>
P-glucuronidase, luciferase, B or C1 genes) that may be present on the recombinant<br>
nucleic acid constructs. Such selection and screening methodologies are well known<br>
to those skilled in the art.<br>
44<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Physical and biochemical methods also may be used to identify plant or plant<br>
cell transformants containing inserted gene constructs. These methods include but are<br>
not limited to: 1) Southern analysis or PCR amplification for detecting and<br>
determining the structure of the recombinant DNA insert; 2) Northern blot, S1 RNase<br>
protection, primer-extension or reverse transcriptase-PCR amplification for detecting<br>
and examining RNA transcripts of the gene constructs; 3) enzymatic assays for<br>
detecting enzyme or ribozyme activity, where such gene products are encoded by the<br>
gene construct; 4) protein gel electrophoresis, Western blot techniques,<br>
immunoprecipitation, or enzyme-linked immunoassays, where the gene construct<br>
products are proteins. Additional techniques, such as in situ hybridization, enzyme<br>
staining, and irnmunostaining, also may be used to detect the presence or expression<br>
of the recombinant construct in specific plant organs and tissues. The methods for<br>
doing all these assays are well known to those skilled in the art.<br>
Effects of gene manipulation using the methods disclosed herein can be<br>
observed by, for example, northern blots of the RNA (e.g., mRNA) isolated from the<br>
tissues of interest. Typically, if the amount of mRNA has increased, it can be<br>
assumed that the corresponding endogenous gene is being expressed at a greater rate<br>
than before. Other methods of measuring gene activity can be used. Different types<br>
of enzymatic assays can be used, depending on the substrate used and the method of<br>
detecting the increase or decrease of a reaction product or by-product. In addition, the<br>
levels of protein expressed can be measured immunochemically, i.e., ELISA, RIA,<br>
EIA and other antibody based assays well known to those of skill in the art, such as by<br>
electrophoretic detection assays (either with staining or western blotting). The<br>
transgene may be selectively expressed in some tissues of the plant or at some<br>
developmental stages, or the transgene may be expressed in substantially all plant<br>
tissues, substantially along its entire life cycle. However, any combinatorial<br>
expression mode is also applicable.<br>
The present disclosure also encompasses seeds of the transgenic plants<br>
described above wherein the seed has the transgene or gene construct. The present<br>
disclosure further encompasses the progeny, clones, cell lines or cells of the<br>
transgenic plants described above wherein said progeny, clone, cell line or cell has the<br>
transgene or gene construct.<br>
45<br><br>
WO 2006/033859 PCTAJS2005/032157<br>
Delivery vehicles<br>
An important factor in the administration of polypeptide compounds, such as<br>
fusion proteins, is ensuring that the polypeptide has the ability to traverse the plasma<br>
membrane of a cell, or the membrane of an intra-cellular compartment such as the<br>
nucleus. Cellular membranes are composed of lipid-protein bilayers that are freely<br>
permeable to small, nonionic lipophilic compounds and are inherently impermeable to<br>
polar compounds, macromolecules, and therapeutic or diagnostic agents. However,<br>
proteins and other compounds such as liposomes have been described, which have the<br>
ability to translocate polypeptides across a cell membrane.<br>
For example, "membrane translocation polypeptides" have amphiphilic or<br>
hydrophobic amino acid subsequences that have the ability to act as membrane-<br>
translocating carriers. In one embodiment, homeodomain proteins have the ability to<br>
translocate across cell membranes. The shortest internalizable peptide of a<br>
homeodomain protein, Antennapedia, was found to be the third helix of the protein,<br>
from amino acid position 43 to 58 (see, e.g., Prochiantz, Current Opinion in<br>
Neurobiology 6:629-634 (1996)). Another subsequence, the h (hydrophobic) domain<br>
of signal peptides, was found to have similar cell membrane translocation<br>
characteristics (see, e.g., Lin et al., J. Biol Chem. 270:1 4255-14258 (1995)).<br>
Examples of peptide sequences which can be linked to a protein, for<br>
facilitating uptake of the protein into cells, include, but are not limited to: an 11 amino<br>
acid peptide of the tat protein of HTV; a 20 residue peptide sequence which<br>
corresponds to amino acids 84-103 of the pl6 protein (see Fahraeus et al., Current<br>
Biology 6:84 (1996)); the third helix of the 60-amino acid long homeodomain of<br>
Antennapedia (Derossi et al., J. Biol. Chem. 269:10444 (1994)); the h region of a<br>
signal peptide such as the Kaposi fibroblast growth factor (K-FGF) h region (Lin et}<br>
al., supra); or the VP22 translocation domain from HSV (Elliot &amp; O'Hare, Cell<br>
88:223-233 (1997)). Other suitable chemical moieties that provide enhanced cellular<br>
uptake may also be chemically linked to ZFPs. Membrane translocation domains<br>
(i.e., internalization domains) can also be selected from libraries of randomized<br>
peptide sequences. See, for example, Yeh et al. (2003) Molecular Therapy 7(5):S461,<br>
Abstract #1191.<br>
Toxin molecules also have the ability to transport polypeptides across cell<br>
membranes. Often, such molecules (called "binary toxins") are composed of at least<br>
two parts: a translocation/binding domain or polypeptide and a separate toxin domain<br>
46<br><br>
WO 2006/033859 PCT/US2005/032157<br>
or polypeptide. Typically, the translocation domain or polypeptide binds to a cellular<br>
receptor, and then the toxin is transported into the cell. Several bacterial toxins,<br>
including Clostridium perfringens iota toxin, diphtheria toxin (DT), Pseudomonas<br>
exotoxin A (PE), pertussis toxin (PT), Bacillus anihracis toxin, and pertussis<br>
adenylate cyclase (CYA), have been used to deliver peptides to the cell cytosol as<br>
internal or amino-terminal fusions (Arora et al., J. Biol Chem., 268:3334-3341<br>
(1993); Perelle et al., Infect. Immun., 61:5147-5156 (1993); Stenmark et al., J. Cell<br>
Biol. 113:1025-1032 (1991); Donnelly et al., PNAS90:3530-3534 (1993); Carbonetti<br>
et al., Abstr. Annu. Meet. Am. Soc. Microbiol. 95:295 (1995); Sebo et al., Infect.<br>
Immun. 63:3851-3857 (1995); Klimpel et al., PNAS U.S.A. 89:10277-10281 (1992);<br>
and Novak et al, J. Biol Chem. 267:17186-17193 1992)).<br>
Such peptide sequences can be used to translocate ZFPs and other types of<br>
fusion proteins across a cell membrane. Polypeptide sequences can be conveniently<br>
fused to or derivatized with such translocation sequences. Typically, the translocation<br>
sequence is provided as part of a fusion protein. Optionally, a linker can be used to<br>
link the translocation sequence to the remainder of the fusion protein. Any suitable<br>
linker can be used, e.g., a peptide linker. See supra.<br>
A fusion protein can also be introduced into an animal cell, preferably a<br>
mammalian cell, via a liposomes and liposome derivatives such as immunoliposomes.<br>
The term "liposome" refers to vesicles comprised of one or more concentrically<br>
ordered lipid bilayers, which encapsulate an aqueous phase. The aqueous phase<br>
typically contains the compound to be delivered to the cell, e.g. a fusion protein<br>
comprising a degron domain and a ZFP DNA-binding domain.<br>
The liposome fuses with the plasma membrane, thereby releasing the protein<br>
into the cytosol. Alternatively, the liposome is phagocytosed or taken up by the cell<br>
in a transport vesicle. Once in the endosome or phagosome, the liposome either<br>
degrades or fuses with the membrane of the transport vesicle and releases its contents.<br>
In current methods of drug delivery via liposomes, the liposome ultimately<br>
becomes permeable and releases the encapsulated compound (in this case, a fusion<br>
protein) at the target tissue or cell. For systemic or tissue specific delivery, this can be<br>
accomplished, for example, in a passive manner wherein the liposome bilayer<br>
degrades over time through the action of various agents in the body. Alternatively,<br>
active drug release involves using an agent to induce a permeability change in the<br>
liposome vesicle. Liposome membranes can be constructed so that they become<br>
47<br><br>
WO 2006/033859 PCT/US2005/032157<br>
destabilized when the environment becomes acidic near the liposome membrane (see,<br>
e.g., PNAS 84:7851 (1987); Biochemistry 28:908 (1989)). When liposomes are<br>
endocytosed by a target cell, for example, they become destabilized and release their<br>
contents. This destabilization is termed fusogenesis.<br>
Dioleoylphosphatidylethanolamine (DOPE) is the basis of many "fusogenic" systems.<br>
Such liposomes typically comprise a protein and a lipid component, e.g., a<br>
neutral and/or cationic lipid, optionally including a receptor-recognition molecule<br>
such as an antibody that binds to a predetermined cell surface receptor or ligand (e.g.,<br>
an antigen). A variety of methods are available for preparing liposomes as described<br>
in, e.g., Szoka et al., Ann. Rev. Biophys. Bioeng. 9:467 (1980), U.S. Pat. Nos.<br>
4,186,183, 4,217,344,4,235,871,4,261,975,4,485,054,4,501,728,4,774,085,<br>
4,837,028, 4,235,871, 4,261,975, 4,485,054,4,501,728, 4,774,085,4,837,028,<br>
4,946,787, PCT Publication No. WO 91\17424, Deamer &amp; Bangham, Biochim.<br>
Biophys. Acta 443:629-634 (1976); Fraley, et al., PNAS 76:3348-3352 (1979); Hope<br>
et al., Biochim. Biophys. Acta 812:55-65 (1985); Mayer et al., Biochim. Biophys. Acta<br>
858:161-168 (1986); Williams et al., PNAS 85:242-246 (1988); Liposomes (Ostro<br>
(ed.), 1983, Chapter 1); Hope et al., Chem. Phys. Lip. 40:89 (1986); Gregoriadis,<br>
Liposome Technology (1984) and Lasic, Liposomes: from Physics to Applications<br>
(1993)). Suitable methods include, for example, sonication, extrusion, high<br>
pressure/homogenization, microfluidization, detergent dialysis, calcium-induced<br>
fusion of small liposome vesicles and ether-fusion methods, all of which are known to<br>
those of skill in the art.<br>
In certain, embodiments, it is desirable to target liposomes using targeting<br>
moieties that are specific to a particular cell type, tissue, and the like. Targeting of<br>
liposomes using a variety of targeting moieties (e.g., ligands, receptors, and<br>
monoclonal antibodies) has been described. See, e.g., U.S. Patent Nos. 4,957,773 and<br>
4,603,044.<br>
Examples of targeting moieties include monoclonal antibodies specific to<br>
antigens associated with neoplasms, such as prostate cancer specific antigen and<br>
MAGE. Tumors can also be targeted by detecting gene products resulting from the<br>
activation or over-expression of oncogenes, such as ras or c-erbB2. In addition, many<br>
tumors express antigens normally expressed by fetal tissue, such as the<br>
alphafetoprotein (AFP) and carcinoembryonic antigen (CEA). Sites of viral infection<br>
can be targeted using various viral antigens such as hepatitis B core and surface<br>
48<br><br>
WO 2006/033859 PCT/US2005/032157<br>
antigens (HBVc, HBVs) hepatitis C antigens, Epstein-Barr virus antigens, human<br>
immunodeficiency type-1 virus (HIV1) and papilloma virus antigens. Inflammation<br>
can be detected using molecules specifically recognized by surface molecules which<br>
are expressed at sites of inflammation such as integrins (e.g., VCAM-1), selectin<br>
receptors (e.g., ELAM-1) and the like.<br>
Standard methods for coupling targeting agents to liposomes can be used.<br>
These methods generally involve incorporation into liposomes of lipid components,<br>
e.g., phosphatidylethanolamine, which can be activated for attachment of targeting<br>
agents, or derivatized lipophilic compounds, such as lipid derivatized bleomycin.<br>
Antibody targeted liposomes can be constructed using, for instance, liposomes which<br>
incorporate protein A (see Renneisen et al., J. Biol. Chem., 265:16337-16342 (1990)<br>
and Leonetti et al., PNAS 87:2448-2451 (1990).<br>
Dosages<br>
For therapeutic applications, the dose administered to a patient, in the context<br>
of the present disclosure, should be sufficient to effect a beneficial therapeutic<br>
response in the patient over time. In addition, particular dosage regimens can be<br>
useful for determining phenotypic changes in an experimental setting, e.g., in<br>
functional genomics studies, and in cell or animal models. The dose will be<br>
determined by the efficacy and Kd of the particular ZFP employed, the nuclear<br>
volume of the target cell, and the condition of the patient, as well as the body weight<br>
or surface area of the patient to be treated. The size of the dose also will be<br>
determined by the existence, nature, and extent of any adverse side-effects that<br>
accompany the administration of a particular compound or vector in a particular<br>
patient.<br>
As an example, the maximum therapeutically effective dosage of ZFP for<br>
approximately 99% binding to target sites is calculated to be in the range of less than<br>
about 1.5x105 to 1.5x106 copies of the specific ZFP molecule per cell. The number of<br>
ZFPs per cell for this level of binding is calculated as follows, using the volume of a<br>
HeLa cell nucleus (approximately 1000 um3 or 10-12 L; Cell Biology, (Altaian &amp;<br>
Katz, eds. (1976)). As the HeLa nucleus is relatively large, this dosage number is<br>
recalculated as needed using the volume of the target cell nucleus. This calculation<br>
also does not take into account competition for ZFP binding by other sites. This<br>
49<br><br>
WO 2006/033859 PCT/US2005/032157<br>
calculation also assumes that essentially all of the ZFP is localized to the nucleus. A<br>
value of l00x Kd is used to calculate approximately 99% binding of to the target site,<br>
and a value of 10x Kd is used to calculate approximately 90% binding of to the target<br>
site. For this example, Kd = 25 nM<br>
ZFP + target site  complex<br>
i.e., DNA + protein  DNA:protein complex<br>
Kd= [DNA] [protein]<br>
[DNA:protein complex]<br>
When 50% of ZFP is bound, Kd = [protein]<br>
So when [protein] = 25 nM and the nucleus volume is 10-12 L<br>
[protein] = (25x10-9 moles/L) (10-12 L/nucleus) (6x1023<br>
molecules/mole)<br>
= 15,000 molecules/nucleus for 50% binding<br>
When 99% target is bound; 100x Kd = [protein]<br>
100x Kd = [protein] = 2.5 uM<br>
(2.5x10-6moles/L) (10-12L/nucleus) (6x1023 molecules/mole)<br>
= about 1,500,000 molecules per nucleus for 99% binding of<br>
target site.<br>
The appropriate dose of an expression vector encoding a ZFP fusion protein<br>
can also be calculated by taking into account the average rate of ZFP expression from<br>
the promoter and the average rate of ZFP degradation in the cell. In certain<br>
embodiments, a weak promoter such as a wild-type or mutant HSV TK promoter is<br>
used, as described above. The dose of fusion protein in micrograms is calculated by<br>
taking into account the molecular weight of the particular protein being employed.<br>
In determining the effective amount of the protein to be administered in the<br>
treatment or prophylaxis of disease, the physician evaluates circulating plasma levels<br>
of the protein or nucleic acid encoding the protein, potential toxicities due to the<br>
protein, progression of the disease, and the production of antibodies to the protein.<br>
Administration can be accomplished via single or divided doses.<br>
Pharmaceutical compositions and administration<br>
Fusion proteins and expression vectors encoding such proteins can be<br>
administered directly to the patient for targeted gene regulation, targeted DNA<br>
50<br><br>
WO 2006/033859 PCT/US2005/032157<br>
cleavage and/or recombination, and for therapeutic or prophylactic applications, for<br>
example, cancer, ischemia, diabetic retinopathy, macular degeneration, rheumatoid<br>
arthritis, psoriasis, HIV infection, sickle cell anemia, Alzheimer's disease, muscular<br>
dystrophy, neurodegenerative diseases, vascular disease, cystic fibrosis, stroke, and<br>
the like. Examples of microorganisms that can be inhibited by ZFP gene therapy<br>
include pathogenic bacteria, e.g., chlamydia, rickettsial bacteria, mycobacteria,<br>
staphylococci, streptococci, pneumococci, meningococci and conococci, klebsiella,<br>
proteus, serratia, pseudomonas, legionella, diphtheria, salmonella, bacilli, cholera,<br>
tetanus, botulism, anthrax, plague, leptospirosis, and Lyme disease bacteria; infectious<br>
fungus, e.g., Aspergillus, Candida species; protozoa such as sporozoa (e.g.,<br>
Plasmodia), rhizopods (e.g., Entamoeba) and flagellates (Trypanosoma, Leishmania,<br>
Trichomonas, Giardia, etc.);viral diseases, e.g., hepatitis (A, B, or C), herpes virus<br>
(e.g., VZV, HSV-1, HSV-6, HSV-H, CMV, and EBV), HIV, Ebola, adenovirus,<br>
influenza virus, flaviviruses, echovirus, rhinovirus, coxsackie viras, coronavirus,<br>
respiratory syncytial virus, mumps virus, rotavirus, measles virus, rubella virus,<br>
parvovirus, vaccinia virus, HTLV virus, dengue virus, papillomavirus, poliovirus,<br>
rabies virus, and arboviral encephalitis viras, etc.<br>
Administration of therapeutically effective amounts is by any of the routes<br>
normally used for introducing a protein or nucleic acid into ultimate contact with the<br>
tissue to be treated. The fusion proteins or encoding nucleic acids are administered in<br>
any suitable manner, preferably with pharmaceutically acceptable carriers. Suitable<br>
methods of administering such modulators are available and well known to those of<br>
skill in the art, and, although more than one route can be used to administer a<br>
particular composition, a particular route can often provide a more immediate and<br>
more effective reaction than another route.<br>
Pharmaceutically acceptable carriers are determined in part by the particular<br>
composition being administered, as well as by the particular method used to<br>
administer the composition. Accordingly, there is a wide variety of suitable<br>
formulations of pharmaceutical compositions that are available (see, e.g., Remington's<br>
Pharmaceutical Sciences, 17th ed. 1985)).<br>
Fusion proteins or their encoding nucleic acids, alone or in combination with<br>
other suitable components, can be made into aerosol formulations (i.e., they can be<br>
"nebulized") to be administered via inhalation. Aerosol formulations can be placed<br>
51<br><br>
WO 2006/033859 PCT/US2005/032157<br>
into pressurized acceptable propellants, such as dichlorodifluoromethane, propane,<br>
nitrogen, and the like.<br>
Formulations suitable for parenteral administration, such as, for example, by<br>
intravenous, intramuscular, intradermal, and subcutaneous routes, include aqueous<br>
and non-aqueous, isotonic sterile injection solutions, which can contain antioxidants,<br>
buffers, bacteriostats, and solutes that render the formulation isotonic with the blood<br>
of the intended recipient, and aqueous and non-aqueous sterile suspensions that can<br>
include suspending agents, solubilizers, thickening agents, stabilizers, and<br>
preservatives. The disclosed compositions can be administered, for example, by<br>
intravenous infusion, orally, topically, intraperitoneally, intravesically or intrathecally.<br>
The formulations of compounds can be presented in unit-dose or multi-dose sealed<br>
containers, such as ampules and vials. Injection solutions and suspensions can be<br>
prepared from sterile powders, granules, and tablets of the kind previously described.<br>
Applications<br>
The disclosed proteins can also be used in vitro. For example, the engineered<br>
ZFP (or a transcription/translation lysate) can be incubated with a polynucleotide<br>
(e.g., plasmid) e.g., in a coupled transcription/translation system for protein<br>
production in vitro.<br>
In additional embodiments, the target site for an engineered ZFP can be placed<br>
on either side of a coding sequence, either endogenous or exogenous. In this<br>
situation, binding of an engineered ZFP (optionally fused to, e.g., an insulator<br>
domain; see e.g., WO 01/02553; WO 02/44376) to the target sites can protect the<br>
coding sequence against the influence of e.g., position effects and<br>
heterochromatinization.<br>
The disclosed methods can be used in concert with existing methods (e.g.,<br>
methotrexate-based selection for amplification of sequences adjacent to a<br>
dihydrofolate reductase gene) to obtain increased levels of protein expression based<br>
on both increased template number and higher levels of transcription.<br>
In additional embodiments, a target sequence for an engineered ZFP can be<br>
introduced adjacent to or overlapping a binding site for another transcriptional<br>
regulatory molecule such that regulation can be further modulated by the engineered<br>
ZFP. Engineered ZFPS fused to either transcriptional activation domains or<br>
52<br><br>
WO 2006/033859 PCT/US2005/032157<br>
transcriptional repression domain can be used, as can ZFPs not fused to a functional<br>
domain, which can sterically modulate binding of other regulatory molecules.<br>
Inactivation of endogenous genes to facilitate protein production<br>
In certain embodiments, the function of one or more cellular gene products is<br>
inactivated to enhance the level and/or quality of proteins produced using the methods<br>
and compositions disclosed herein. Gene function can be inactivated by, e.g.,<br>
disruption of one or more alleles of the endogenous cellular gene that encodes the<br>
gene product (e.g., by gene "knock-out") and/or by repressing transcription of an<br>
endogenous cellular gene as described, for example, in co-owned U.S. Patent No.<br>
6,534,261.<br>
Disruption of an endogenous cellular gene can be accomplished by<br>
mutagenesis (e.g., chemical or radiation-induced) or by insertion of exogenous DNA<br>
sequences into the cellular genome, optionally followed by selection for the desired<br>
mutant cells. Insertion of exogenous sequences can be random (e.g., following<br>
retroviral infection or contact of a cell with a DNA molecule) or targeted (e.g., U.S.<br>
Patent No. 5,614,396). Targeted insertion of exogenous sequences can also be<br>
accomplished by targeted cleavage of genomic DNA in combination with introduction<br>
of exogenous DNA. Co-owned PCT WO 2005/014791 and United States Provisional<br>
Patent Application 60/702,394 (filed July 26,2005), the disclosures of which are<br>
incorporated by reference, provide methods and compositions for homology-<br>
dependent and homology independent methods, respectively, for targeted insertion of<br>
exogenous sequences. In addition, PCT WO 2005/014791 also discloses methods for<br>
targeted mutagenesis by targeted DNA cleavage followed by non-homologous end-<br>
joining.<br>
By way of example, one or more genes involved in the process of apoptosis<br>
(or the function thereof) can be inactivated. Exemplary genes involved in apoptosis<br>
are shown in Table 1.<br>
53<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Table 1<br>
caspase1<br>
caspase2<br>
caspase3<br>
caspase4<br>
caspase5<br>
caspase6<br>
caspase7<br>
caspase8<br>
caspase9<br>
caspase10<br>
caspase11<br>
caspase12<br>
caspase13<br>
caspase14<br>
bax<br>
bak<br>
bik<br>
APAF-1 (apoptosome associated factor 1)<br>
c-jun<br>
lactate dehydrogenase<br>
cytochrome c<br>
Genes whose products negatively regulate cell cycle progression can be<br>
inactivated. Exemplary genes involved in regulation of the cell cycle are shown in<br>
Table 2.<br>
Table 2<br>
p16<br>
p19<br>
Rb (retinoblastoma protein)<br>
p53<br>
p73<br>
Telomerase<br>
Because the bioactivity of certain proteins can be affected by their<br>
glycosylation pattern, it may be desirable, in certain circumstances, to inactivate<br>
certain genes whose products are involved in protein glycosylation. Exemplary genes<br>
involved in glycosylation are shown in Table 3.<br>
Table 3<br>
a-1,6-fucosyltransferase<br>
CMP-N-acetylneuraminic acid hydroxylase<br>
CMP-sialic acid hydroxylase<br>
Glucosamine-6-phosphate isomerase<br>
54<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Genes encoding receptors for infection (e.g., viral or bacterial receptors) can<br>
be inactivated to enhance protein production. For example, the gene encoding the<br>
minute virus of mice (MVM) receptor can be inactivated.<br>
In certain cases, it may be desirable to integrate an exogenous DNA molecule<br>
encoding a transgene and a selection marker (e.g., to select cells containing the<br>
integrated transgene). If the selection marker is encoded by the genome of the cell<br>
into which it is desired to integrate the exogenous DNA molecule, it can be useful to<br>
inactivate the endogenous gene encoding that selection marker. Inactivation can be<br>
partial (such that reduced levels of the marker are produced) or complete (such that no<br>
marker is produced). Exemplary marker genes that can be so inactivated are shown in<br>
Table 4.<br>
Table 4<br>
dihydrofolate reducatase<br>
Glutamine synthetase<br>
hypoxanthine phosphoribosyl transferase<br>
Genes encoding proteases can be inactivated to enhance protein production.<br>
Exemplary proteases are shown in Table 5.<br>
Table 5<br>
serine proteases<br>
elastase<br>
collagenase<br>
plasminogen activator<br>
For safety and compliance with regulatory requirements, genes encoding<br>
prions (PRPs)and/or pseudo-prions can be inactivated. Pseudo-prions include, for<br>
example, PRNP/PRP, Prion-like protein Doppel (PRND) and Shadow of Prion<br>
(SPRN) See, for example, Liao et al. (1986) Science 233:364-367 (PRNP); Lu et al.<br>
(2000) Biochemistry 39:13575-13583 (PRND) and Premzl et al. (2003) Gene 314:89-<br>
102 (SPRN).<br>
In cells which are being used for production of antibody molecules, it can be<br>
useful to inactivate genes encoding proteins which are bound by the antibody or<br>
antibodies being produced. For example, genes encoding the antigen(s) recognized<br>
by the antibody or antibodies can be inactivated by any of the methods disclosed<br>
herein.<br>
55<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Additional cellular processes which can affect protein yield and quality<br>
include, for example, replication, transcription, RNA processing, translation, amino<br>
acid biosynthesis, cellular metabolism, protein folding, protein degradation, protein<br>
transport, stress responses and plasmid copy number control. It is to be understood<br>
that either positive or negative regulation of a gene (including inactivation or gene<br>
"knock-out") involved in any of the aforementioned processes can be used to enhance<br>
the yield and/or quality of a protein expressed using the methods and compositions<br>
disclosed herein.<br>
EXAMPLES<br>
The following examples are presented as illustrative of, but not limiting, the<br>
claimed subject matter.<br>
Example 1: Construction of a SRα promoter<br>
The S V40 early promoter was generated by PCR from the plasmid pRL-SV40<br>
(Promega) and included base pairs 80-449 from the pRL-SV40 sequence and<br>
incorporated the HindIII site of the promoter at the 3'end. Bases 80-88 comprise the<br>
9bp target site for the 2392/00 and 2932/10 ZFPs. A Bpul0I restriction site was<br>
added to the PCR primer immediately upstream of the ZFP binding site to facilitate<br>
subsequent cloning into the backbone vector, pcDNA3.1/zeo, from which the original<br>
CMV promoter had been removed.<br>
The R-U5 composite sequence was generated by PCR from the plasmid<br>
pDrive01-NSE(r)-RU5 v02 (Invivogen) and included base pairs 1774-2066 from the<br>
plasmid sequence, which encompass the HindIII restriction site within the 5' primer<br>
binding site. An EcoRI restriction site was added to the 3' PCR primer to facilitate<br>
cloning into the backbone vector.<br>
The SV40 and RU5 sequences were linked at their respective HindHI sites to<br>
form the SV40-RU5 fusion. The sequence of this hybrid promoter (SEQ ID NO:9) is<br>
shown in Figure 1, with the ZFP target site underlined.<br>
Example 2: Engineering zinc finger proteins to bind to a target site in the SRα promoter<br>
To increase expression of proteins whose coding regions are under the control<br>
of the SRα promoter, a three-finger zinc finger protein, SBS2392/00, was synthesized<br>
56<br><br>
WO 2006/033859 PCT/US2005/032157<br>
(according to methods disclosed in co-owned U.S. Patents 6,453,242 and 6,534,261)<br>
to bind a nine-nucleotide target sequence in the SRα promoter. The target site was the<br>
sequence GCTGTGGAA (SEQ ID NO:1), located as shown in Figure 1. The<br>
sequence of the protein is as follows, with the recognition regions of the zinc fingers<br>
underlined:<br>
KKKQHICHIQGCGKVYGORSNLVRHLRWHTGERPFMCTWSYCGKRFTRSDA<br>
LSRHKRTHTGEKKFACPECPKRPMQSSDLRHIKTHQNK (SEQ ID NO:2).<br>
Additional zinc finger domains capable of binding to SEQ ID NO.1 were<br>
obtained using a two-hybrid selection system. See, e.g., U.S. Patent Application<br>
Publication No. 2003/0044787 (Mar. 6,2003) and Joung et al. (2000) Proc. Nail.<br>
Acad. Sci. USA 97:7382-7287. Their amino acid sequences are shown in Figure 6.<br>
Nucleotide sequences encoding these zinc finger proteins were fused to<br>
nucleotide sequences encoding a VP16 activation domain, a nuclear localization<br>
signal (NLS) and a FLAG epitope tag to generate engineered transcription factors.<br>
Exemplary poiynucleotide sequences are shown in Figures 3 and 5, with then-<br>
encoded amino acid sequences shown in Figures 2 and 4, respectively.<br>
Example 3: Enhancement of SRα promoter-driven expression of<br>
immunoglobulin genes by an engineered zinc finger protein<br>
A DG44 CHO cell line containing an integrated antibody expression construct<br>
driven by the SRα promoter was transiently transfected with 20-250 ng of plasmid<br>
encoding the VP16 activation domain (NVF), or ZFP 2392/00 (SEQ ID NO:2)linked<br>
to VP16 (2392-VP16). Immunoglobulin kappa chain mRNA expression was<br>
measured by real-time PCR (Taqman®). The results, shown in Figure 7, indicate that<br>
SRα-driven transcription of kappa chain mRNA is increased 4-5-fold by the 2392/00<br>
transcription factor.<br>
In an experiment to measure the effect of the 2392/00-VP16 fusion on protein<br>
levels, two different DG44 CHO cell-derived cell lines (Line A and Line B), that<br>
stably express gamma heavy chains and kappa light chains, the expression of both of<br>
which is driven by the SRα promoter, were transiently transfected with plasmid<br>
encoding the VP16 activation domain (NVF) or ZFP 2392/00 linked to the VP16<br>
activation domain (2392). Expression of secreted immunoglobulin G was measured<br>
57<br><br>
WO 2006/033859 PCT/US2005/032157<br>
by ELISA. The results, shown in Figure 8, indicate that, in both cell lines, expression<br>
of IgG was increased by the 2392/00 ZFP-VP16 fusion.<br>
To determine whether the 2392/00-VP16 fusion protein can increase<br>
expression in an optimized expression system, a cell line containing amplified gamma<br>
heavy chain and kappa light chain cDNAs, obtained by methotrexate selection, was<br>
used. Sequences encoding the 2392/00-VP16 fusion ZFP were introduced into cells<br>
of this "high producer" line (ZFP) and the level of mRNA expression was compared<br>
with the same cells in the absence of ZFP (NT). Figure 9 shows that optimized levels<br>
of both gamma chain and kappa chain mRNAs can be increased a further three-fold<br>
when the 2392/00-VP16 protein is introduced into these cells.<br>
A cell line containing integrated cDNAs encoding the gamma heavy chain and<br>
kappa light chain components of immunoglobulin G, both under the transcriptional<br>
control of the SRα promoter, was transfected with a plasmid encoding the 2392/00-<br>
VP16 fusion protein. Transfected cells were selected for stable ZFP expression.<br>
Secreted IgG was measured by ELISA from a clonal iine stably expressing the<br>
2392/00-VP16 fusion protein. The results, shown in Figure 10, reveal an<br>
approximately 4-fold increase in secreted IgG in cells stably transfected with the<br>
2392/00-VP16 protein, compared to control untransfected cells.<br>
Example 4: Selection and properties of a stable cell line containing<br>
integrated sequences encoding a 2392/10-VP16 fusion protein<br>
Chinese hamster ovary (CHO) DG44 cells growing adherent on 6 well plates<br>
were transfected with a plasmid encoding a 2392/10-VP16 fusion protein, as follows.<br>
Cells were incubated with 1 ug DNA and 4ul Lipofectamine 2000 in lml serum-free<br>
growth medium for 4hrs, then medium was aspirated and 2ml of regular growth<br>
medium added. After 3 days, cells were split in various dilutions into 25 cm dishes<br>
and subjected to Zeocin selection. When selection was complete, individual colonies<br>
were harvested from the appropriate dilution plate, transferred into 24 well dishes and<br>
grown up. Criteria for completion of selection were that individual cell colonies<br>
appear and no new cell death occurs, while complete cell death is observed in non-<br>
transfected DG44 cells that had been placed under selection at the same time. 22<br>
clonal lines were analyzed for ZFP expression, cell growth properties and activation<br>
of reporter constructs containing mulitmerized ZFP binding sites.<br>
58<br><br>
WO 2006/033859 PCT/US2005/032157<br>
One of these lines was subcloned and genomic DNA was extracted from two<br>
of the subclones. The DNA was digested completely with BamHI restriction<br>
endonuclease, resolved on a 0.9% TAE agarose gel, transferred to Nytran+ membrane<br>
and the membrane was probed with a 290bp radiolabelled DNA fragment encoding<br>
the VP16 activation domain. A single band of approximately 12 kbp was detected in<br>
the two ZFP-containing cell lines, suggesting a single copy of the DNA encoding the<br>
fusion proteins is present. This band was not detected in DNA from wild type DG44<br>
CHO cells.<br>
Example 5: Construction of a SRα promoter containing multiple copies<br>
of a target site for a ZFP-VP16 fusion protein<br>
A promoter denoted "SRαZ6" was constructed by joining a DNA fragment<br>
containing six copies of the target site for the 2392/00 and 2392/10 ZFPs (SEQ ID<br>
NO:1) to an SRα promoter-containing DNA fragment. The Z6 promoter thus contains<br>
7 copies of SEQ ID NO:1. These additional target sites confer a much greater level of<br>
ZFP-mediated activation of this promoter. See infra.<br>
The insert containing the additional six binding sites was inserted as a<br>
BamHI/NdeI fragment after first inserting those restriction sites (shown in bold text<br>
below) upstream of the SV40 portion of the SRα promoter. The seven iterations of<br>
the target site are underlined. The target site downstream of the Ndel site is the site<br>
originally present in the SV40 portion of the SRα promoter.<br>
The sequence of that portion of the SRaZ6 promoter containing the ZFP target<br>
sites is as follows:<br>
BamHI<br>
ggatccgagcbgbggaabgagagcbgbggaabgagagcbgbggaabgagagc<br>
bgtggaatgagagctgtggaatgagagctgtggaatgacatatggcbgtggaatgtg<br>
bgbcagbba NdeI<br>
(SEQ ID NO:23)<br>
Example 6: Construction of a CMV promoter containing multiple copies<br>
of a target site for a ZFP-VP16 fusion protein<br>
A promoter denoted "CMVz10" was constructed by inserting a DNA<br>
fragment, comprising multiple copies of a target site (SEQ ID NO:1) for the 2392/00<br>
59<br><br>
WO 2006/033859 PCT/US2005/032157<br>
and 2392/10 engineered ZFPs, into a MM restriction site immediately upstream of<br>
the CMV promoter. The CMVz10 promoter contains 9 perfect iterations of the target<br>
site plus a single 8/9 match in the tenth iteration (a C to T substitution). The sequence<br>
of the CMVz10 promoter is shown below, with the MluI restriction sites shown in<br>
bold and the target sites underlined. Note that, since this sequence was inserted in the<br>
reverse orientation, the sequence shown reflects the complement of each binding site.<br>
MluI<br>
acgcgttcattccacagctctcattccacagctctcattccacagctctcat<br>
tccacagctctcattccacagctctcattccacagctctcattccacagctctcatt<br>
ccacagctctcattccacagctctcattccacagtcacgcgt<br>
MluI<br>
(SEQ ID NO:24)<br>
Additional promoter constructs, which include different numbers of binding<br>
sites and different orientations of the binding sites with respect to the promoter, have<br>
also been constructed and tested.<br>
Example 7: Enhanced protein production from a SRα promoter<br>
containing multiple copies of a target site for a ZFP-VP16 fusion protein<br>
A clonal isolate (2392/10-7) of the DG44 CHO cell line containing the stably<br>
integrated 2392/10 ZFP (Example 4), and parent DG44 cells, were transfected with<br>
two different antibody-expressing constructs. In one, heavy chain and the light chain<br>
transcription units were each under the control of an SRα promoter (SRa); in the<br>
other, both the heavy chain and light chain transcription units were under the control<br>
of an SRα promoter containing an additional 8 target sites for the ZFP immediately<br>
upstream of the promoter (SRa 2393BS). Three days after transfection, IgG secretion<br>
was measured by ELISA.<br>
The results are shown in Figure 11, and indicate that IgG expression level is<br>
dependent upon the presence of the 2392/10 ZFP and that more copies of the 2392/10<br>
target site result in higher IgG levels.<br>
60<br><br>
WO 2006/033859 PCT/US2005/032157<br>
Example 8: Enhanced transcription from a CMV promoter containing<br>
multiple copies of a target site for a ZFP-VP16 fusion protein<br>
To test the effect of the 2392/10-VP16 fusion protein on the transcriptional<br>
activity of the CMV promoter, a reporter construct, in which a CMV promoter was<br>
operatively linked to sequences encoding green fluorescent protein (GFP) was<br>
constructed. Variants of this reporter construct were then constructed, which<br>
contained multiple copies of the target site for the 2392/10 ZFP upstream of the core<br>
CMV promoter sequences. Promoters containing different numbers of target sites in<br>
different orientations were tested by transfecting them into the 2392/10-7 cell line<br>
(Example 4), followed by analysis of GFP mRNA levels by real-time PCR<br>
(Taqman®). Representative results are shown in Figure 12. The results indicate that<br>
more copies of the target site lead to higher steady-state mRNA levels, and that the<br>
effect is independent of the orientation of the target sites with respect to the remainder<br>
of the CMV promoter sequences. No effect of either number or orientation of target<br>
sites is observed in a parental cell line that does not express the 2392/10 ZFP.<br>
Example 9: Enhanced expression of erythropoietin, mediated by the<br>
2392/10-VP16 fusion protein, from SRα and CMV promoters containing multiple<br>
copies of the 2392/10 target site<br>
Three different erythropoietin (Epo)-expressmg constructs were constructed.<br>
In the first, Epo expression was controlled by a CMV promoter fused to a P-globin<br>
intron (CMV). In the second, Epo expression was controlled by a CMV promoter<br>
containing ten upstream copies of the 2392/10 binding site, fused to a P-globin intron<br>
(CMVzlO, Example 6). In the third, Epo expression was controlled by a SRα<br>
promoter with six additional copies of the 2392/10 binding site immediately upstream<br>
of the promoter (SRαZ6, Example 5). The three constructs were transiently<br>
transfected into either parental DG44 CHO cells or the ZFP-expressing 2392/10-7 cell<br>
line. After 24 hours, Epo secretion was measured by ELISA.<br>
The results, shown in Figure 13, confirm previous observations that, in the<br>
absence of the 2392/10-VP16 ZFP, the CMV promoter is stronger than the SRα<br>
promoter. However, activation of the SRαZ6 promoter by the 2393/10-VP16 protein<br>
increases its activity such that it is comparable to that of the CMV promoter.<br>
Moreover, insertion of 2392/10 target sites adjacent to the CMV promoter increases<br>
61<br><br>
WO 2006/033859 PCT/US2005/032157<br>
its already strong activity an additional three-fold in 2392/10-VP16-expressing cells.<br>
Thus, the activity of the strongest known naturally-occurring promoter has been<br>
improved by the use of an engineered ZFP transcriptional activator.<br>
All patents, patent applications and publications mentioned herein are hereby<br>
incorporated by reference in their entirety.<br>
Although disclosure has been provided in some detail by way of illustration<br>
and example for the purposes of clarity of understanding, it will be apparent to those<br>
skilled in the art that various changes and modifications can be practiced without<br>
departing from the spirit or scope of the disclosure. Accordingly, the foregoing<br>
descriptions and examples should not be construed as limiting.<br>
62<br><br>
We Claim :<br>
1.	An in vitro method for enhancing the level and/or quality of protein production from an<br>
exogenous sequence in a cell the method comprising:<br>
(a)	providing a cell in which a first gene is inactivated; and<br>
(b)	expressing first and second fusion proteins in the cell, each of the first and second<br>
fusion proteins comprising a zinc finger DNA-binding domain and a nuclease, wherein<br>
the first and second fusion proteins bind to and cleave at a first site in a second gene,<br>
wherein the first and second genes are inactivated;<br>
(c)	expressing an exogenous sequence encoding the protein in the cell in which<br>
the first and second genes are inactivated;<br>
thereby enhancing- the level and/or quality of protein production from the<br>
exogenous sequence expressed in the cell.<br>
2.	The method as claimed in claim 1, wherein cleavage in the second gene is followed<br>
by non-homologous end joining.<br>
3.	The method as claimed in claim 1. comprising expressing third and fourth<br>
fusion proteins in the cell, each of the third and fourth fusion proteins comprising a zinc<br>
finger protein and a nuclease, wherein the third and fourth fusion proteins bind to and<br>
cleave at a second site in the second gene.<br>
4.	The method as claimed in claim 1. comprising contacting the cell with an<br>
exogenous nucleic acid sequence;<br>
thereby resulting in integration of the exogenous nucleic acid sequence into the<br>
second gene.<br>
5.	The method as claimed in any of claims 1 to 4, wherein the first gene is box.<br><br>
6.	The method as claimed in any of claims 1 to 4, wherein the first gene is bac.<br>
7.	The method as claimed in claim 5, wherein the first gene is bax and the second gene is<br>
bac.<br>
8.	The method as claimed in claim 6, wherein the first gene is bac and the second gene is<br>
box.<br>
9.	The method as claimed in any of claims 1 to 4, wherein the first gene is glutamine<br>
synthetase.<br>
10.	The method as claimed in any of claims 1 to 4, wherein the first gene is<br>
dihydrofolate reductase.<br>
11.	The method as claimed in claim 9, wherein the first gene is giutamine synthetase<br>
and the second gene is dihydrofolate reductase.<br>
12.	The method as claimed in claim 10, wherein the first gene is dihydrofolate<br>
reductase and the second gene is glutamine synthetase.<br>
] 3. The method as claimed in any of claims 1-12, wherein a third gene is inactivated in<br>
the cell, the method comprising:<br>
(a)	providing a cell in which first and second genes are inactivated; and<br>
(b)	expressing fifth and sixth fusion proteins in the cell, each of the fifth and sixth<br>
fusion proteins comprising a zinc finger DNA-binding domain and a nuclease, wherein<br>
the fifth and sixth fusion proteins bind to and cleave at a first site in the third gene.<br>
14. The method as claimed in claim 13, wherein cleavage in the third gene is followed<br>
by non-homologous end joining.<br><br>
15.	The method as claimed in claim 13, comprising expressing seventh and<br>
eighth fusion proteins in the cell, each of the seventh and eighth fusion proteins<br>
comprising a zinc finger protein and a nuclease, wherein the seventh and eighth fusion<br>
proteins bind to and cleave at a second site in the third gene.<br>
16.	The method as claimed in claim 13. comprising contacting the cell with an<br>
exogenous nucleic acid sequence;<br>
thereby resulting in integration of the exogenous nucleic acid sequence into the<br>
third gene.<br>
17.	The method as claimed in any of claims 13-16, wherein the genes that are<br>
inactivated are the bax, bac, and α-l,6-fucosyltransferase genes.<br><br>
Disclosed herein are methods and compositions for enhanced protein production<br>
and over expression from an exogenous sequence using engineered zinc finger proteins to<br>
inactivate first and second genes in a cell including the exogenous sequence.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-correspondence others 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1wY3QgcmVxdWVzdCBmb3JtLnBkZg==" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-pct request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1wcmlvcml0eSBkb2N1bWVudCAxLjEucGRm" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-priority document 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDEwMjMta29sbnAtMjAwNy1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">01023-kolnp-2007-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1BTUFOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04ucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-AMANDED PAGES OF SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1GT1JNIDEucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1GT1JNIDEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1GT1JNIDIucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LSgyOC0wOS0yMDExKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-(28-09-2011)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUFCU1RSQUNULnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUFNQU5ERUQgQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUFTU0lHTk1FTlQucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-ASSIGNMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUNBTkNFTExFRCBQQUdFUy5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-CANCELLED PAGES.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LURFU0NSSVBUSU9OIChDT01QTEVURSkucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LURSQVdJTkdTLnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-DRAWINGS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUZPUk0gMS0xLjEucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-FORM 1-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUZPUk0gMTMucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1rb2xucC0yMDA3LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">1023-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUZPUk0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUZPUk0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LUZPUk0tMjcucGRm" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-FORM-27.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LVBBLnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-PA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3LTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-PETITION UNDER RULE 137-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAyMy1LT0xOUC0yMDA3LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVC5wZGY=" target="_blank" style="word-wrap:break-word;">1023-KOLNP-2007-REPLY TO EXAMINATION REPORT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QtMDEwMjMta29sbnAtMjAwNy5qcGc=" target="_blank" style="word-wrap:break-word;">abstract-01023-kolnp-2007.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="250331-an-electromagnetic-actuator.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250333-method-for-the-biocompatible-coating-of-medical-products.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250332</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1023/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2011</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Dec-2011</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Mar-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SANGAMO BIOSCIENCES, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>POINT RICHMOND TECH CENTER, 501 CANAL BLVD., SUITE A100, RICHMOND, CA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LI XIAO-YONG</td>
											<td>1014 MASONIC AVENUE, ALBANY, CA 94706</td>
										</tr>
										<tr>
											<td>2</td>
											<td>BARTSEVICH VICTOR</td>
											<td>1009 KAINS AVENUE, APT. 5, ALBANY, CA 94706</td>
										</tr>
										<tr>
											<td>3</td>
											<td>COLLINGWOOD TREVOR</td>
											<td>1111 CAMBRIDGE STREET, NOVATO, CA 94947</td>
										</tr>
										<tr>
											<td>4</td>
											<td>JAMIESON ANDREW</td>
											<td>2528 SUTTER STREET, SAN FRANCISCO, CA 94115</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A61K 48/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/032157</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-09-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/610,853</td>
									<td>2004-09-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/661,841</td>
									<td>2005-03-15</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250332-an-in-vitro-method-for-enhancing-the-level-and-or-quality-of-protein-production by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:29:50 GMT -->
</html>
