<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250388-a-hybrid-enzyme-for-starch-processing by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:27:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250388:A HYBRID ENZYME FOR STARCH PROCESSING</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A HYBRID ENZYME FOR STARCH PROCESSING</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a hybrid enzyme comprising carbohydrate-binding module amino acid sequence and a fungal alpha-amylase amino acid sequence and to a variant of a fungal wild type enzyme comprising a carbohydrate-binding module and an alpha-amylase catalytic module. The invention also relates to the use of the hybrid enzyme or the variant in starch liquefaction.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
ENZYMES FOR STARCH PROCESSING<br>
CROSS-REFERENCE TO RELATED APPLICATIONS<br>
This application claims the benefit of U.S. Provisional application nos. 60/569,862, filed May 10, 2004, 60,490,751, filed July 29, 2003, 60/511,044, filed * October 14, 2003, 60/482,589, filed June 25, 2003, and 60/514,854, filed October 27, 2003 and priority from Danish application nos. PA 2003 00949, filed June 25, 2003, and PA 2003 01568, filed October 24, 2003, the contents of which are fully incorporated herein by reference.<br>
FIELD OF THE INVENTION<br>
The present invention relates, inter alia, to an enzyme comprising a carbohydrate-binding module (*CBMn) and an alpha-amylase catalytic domain. The enzyme may be a hybrid between a carbohydrate-binding module ("CBM") and an alpha-amylase or the enzyme may be a variant of a parent enzyme comprising a carbohydrate-binding module ("CBM") and an alpha-amylase catalytic domain. The invention also relates to the use of the enzyme in a starch liquefaction process in which starch is degraded to smaller oligo- and/or polysaccharide fragments.<br>
BACKGROUND OF THE INVENTION<br>
A large number of enzymes and processes have been described for converting starch to starch hydrolysates, such as maltose, glucose or specialty syrups, either for use as sweeteners or as precursors for other saccharides such as fructose. Glucose may also be fermented to ethanol or other fermentation products, such as citric acid, monosodium glutamate, gluconic acid, sodium gluconate, calcium gluconate, potassium gluconate, glucono delta lactone, or sodium erythorbate, itaconic acid, lactic acid, gluconic acid; ketones; amino acids, glutamic acid (sodium monoglutaminate), penicillin, tetracyclic enzymes; vitamins, such as riboflavin, B12, beta-carotene or hormones.<br>
Starch is a high molecular-weight polymer consisting of chains of glucose units. It usually consists of about 80% amylopectin and 20% amylose. Amylopectin is a branched polysaccharide in which linear chains of alpha-1,4 D-glucose residues are joined by alpha-1,6 glucosidic linkages.<br>
Amylose is a linear polysaccharide built up of D-glucopyranose units linked together by alpha-1,4 glucosidic linkages. In the case of converting starch into a soluble starch hydrolysate, the starch is depolymerized. The conventional depolymerization process<br><br>
consists of a gelatinization step and two consecutive process steps, namely a liquefaction process and a saccharification process.<br>
Granular starch consists of microscopic granules, which are insoluble in water at room temperature. When an aqueous starch slurry is heated, the granules swell and eventually burst, dispersing the starch molecules into the solution. During this "gelatinization' process there is a dramatic increase in viscosity. As the solids level is 30-40% in a typical industrial process, the starch has to be thinned or "liquefied" so that it can be handled. This reduction in viscosity is today mostly obtained by enzymatic degradation. During the liquefaction step, the long-chained starch is degraded into smaller branched and linear units (maltodextrins) by an alpha-amylase. The liquefaction process is typically carried out at about 105-110'C for about 5 to 10 minutes followed by about 1-2 hours at about 95"C. The temperature is then lowered to 60'C, a glucoamylase or a beta-amylase and optionally, a debranching enzyme, such as an isoamylase or a pullulanase are added, and the saccharification process proceeds for about 24 to 72 hours.<br>
It will be apparent from the above discussion that the conventional starch conversion process is very energy consuming due to the different requirements in terms of temperature during the various steps. It is thus desirable to be able to select and/or design the enzymes used in the process so that the overall process can be performed without having to gelatinize the starch. Such processes are the subject for the patents US4591560, US4727026 and US4009074 and EP0171218, and Danish patent application PA 2003 00949. The present invention discloses a new hybrid enzyme and a genetical modification of a wild type enzyme designed for such processes and comprising an amino acid sequence of a CBM and an amino acid sequence of a fungal starch degrading enzyme. Hybrid enzymes are the subject of WO9814601, WO0077165 and Danish patent application PA 2003 00949.<br>
SUMMARY OF THE INVENTION<br>
The invention provides in a first aspect a hybrid enzyme which comprises an amino acid sequence of a catalytic module having alpha-amylase activity and an amino acid sequence of a carbohydrate-binding module, wherein the catalytic module is of fungal origin.<br>
The invention provides in a second aspect a variant of the fungal alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41, which variant comprising an alteration at one or more of the positions: 13,15,18, 31, 32, 33, 34, 35, 36, 61, 63, 64, 68, 69, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 89, 117, 118, 119, 120, 121, 122, 123, 124, 125, 152, 153, 154, 155, 156, 157, 158,161,162, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 204, 205, 206, 207, 208, 209, 210, 211,<br><br>
216, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 242, 245, 250, 252, 253, 255, 256, 257,259, 260, 275, 292, 295, 296, 297, 298, 299, 304, 328, 339, 344, 348, 378, 383, 386,387,405,448 and 480 wherein; (a) the alteration(s) are independently, i) an insertion of an ammo acid downstream of the amino acid which occupies the position, ii) a deletion of the amino acid which occupies the position, or iii) a substitution of the amino acid which occupies the position with a different amino acid, (b) the variant has increased acid alpha-amylase activity and or improved enzyme stability relative to.the parent fungal alpha-amyiase and, (c) each position corresponds to a position of the amino acid sequence of the TAKA amylase shown in SEQ ID NO:43 and/or the Atowachii alpha-amylase shown as SEQlDNO:41.<br>
In further aspects the invention provides an isolated DNA sequence encoding the hybrid enzyme of the first aspect or the variant of the second aspect, a DNA construct comprising the DNA sequence encoding the hybrid enzyme of the first aspect or the variant of the second aspect, an expression vector comprising the DNA sequence encoding the hybrid enzyme of the first aspect or the variant of the second aspect, and a host cell transformed with a vector; which host cell is capable of expressing the DNA sequence encoding the hybrid enzyme of the first aspect or the variant of the second aspect.<br>
In an eighth aspect the invention provides a method for liquefying starch, wherein a gelatinized or granular starch substrate is treated in aqueous medium with the hybrid enzyme of the first aspect or the variant of the second aspect.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The term "granular starch" is understood as raw uncooked starch, i.e. starch that has not been subjected to a gelatinization. Starch is formed in plants as tiny granules insoluble in water. These granules are preserved in starches at temperatures below the initial gelatinization temperature. When put in cold water, the grains may absorb a small amount of the liquid. Up to 50°C to 70°C the swelling is reversible, the degree of reversibility being dependent upon the particular starch. With higher temperatures an irreversible swelling called gelatinization begins.<br>
The term "initial gelatinization temperature" is understood as the lowest temperature at which gelatinization of the starch commences. Starch heated in water begins to gelatinize between 50'C and 75'C; the exact temperature of gelatinization depends on the specific starch and can readily be determined by the skilled artisan. Thus, the initial gelatinization temperature may vary according to the plant species, to the particular variety of the plant species as well as with the growth conditions. In the context of this invention the initial gelatinization temperature of a given starch is the temperature at which birefringence is lost<br><br>
in 5% of the starch granules using the method described by Gorinstein. S. and Lii. C, Starch/StSrke, Vol. 44 (12) pp. 461-466 (1992).<br>
The term "soluble starch hydrolysate" is understood as the soluble products of the processes of the invention and may comprise mono- dh and oligosaccharides, such as glucose, maltose, maltodextrins, cyclodextrins and any mixture of these. Preferably at least 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97% or 98% of the dry solids of the granular starch is converted into a soluble starch hydrolysate.<br>
The term polypeptide "homology" is understood as the degree of identity between two<br>
sequences indicating a derivation of the first sequence from the second. The homology may<br>
suitably be determined by means of computer programs known in the art such as GAP	'<br>
provided in the GCG program package (Program Manual for the Wisconsin Package, Version 8, August 1994, Genetics Computer Group, 575 Science Drive, Madison, Wisconsin, USA 53711) (Needleman, S.B. and Wunsch, CD., (1970), Journal of Molecular Biology, 48, 443-453. The following settings for amino acid sequence comparison are used: GAP creation penalty of 3.0 and GAP extension penalty of 0.1. The relevant part of the amino acid sequence for the homology determination is the mature polypeptide, i.e. without the signal peptide.<br>
Hybrid enzymes<br>
Enzyme classification numbers (EC numbers) referred to in the present specification with claims are in accordance with the Recommendations (1992) of the Nomenclature Committee of the International Union of Biochemistry and Molecular Biology, Academic Press Inc, 1992.<br>
Hybrid enzymes or a genetically modified wild type enzymes as referred to herein include species comprising an amino acid sequence of an alpha-amyloiytic enzyme (EC 3.2.1.1) linked (i.e. covalently bound) to an amino acid sequence comprising a carbohydrate-binding module (CBM).<br>
CBM-containing hybrid enzymes, as well as detailed descriptions of the preparation and purification thereof, are known in the art [see, e.g. WO 90/00609, WO 94/24158 and WO 95/16782, as well as Greenwood et al. Biotechnology and Bioenaineering 44 (1994) pp. 1295-1305]. They may, e.g. be prepared by transforming into a host cell a DNA construct comprising at least a fragment of DNA encoding the carbohydrate-binding module ligated, with or without a linker, to a DNA sequence encoding the enzyme of interest, and growing the transformed host cell to express the fused gene. The resulting recombinant product (hybrid enzyme) - often referred to in the art as a "fusion protein - may be described by the following general formula:<br>
A-CBM-MR-X<br><br>
In the latter formula, A-CBM is the N-terminal or the C-terminal region of an amino acid sequence comprising at least the carbohydrate-binding module (CBM) perse. MR is the middle region (the Tinker")* and X is the sequence of amino acid residues of a polypeptide encoded by a DNA sequence encodng the enzyme (or other protein) to which the CBM is to be linked.<br>
The moiety A may either be absent (such that A-CBM is a CBM per se, i.e. comprises no amino acid residues other than those constituting the CBM) or may be a sequence of one or more amino acid residues (functioning as a terminal extension of the CBM per se). The linker (MR) may be a bond, or a short linking group comprising from about 2 to about 100 carbon atoms, in particular of from 2 to 40 carbon atoms. However, MR is preferably a sequence of from about 2 to about 100 amino acid residues, more preferably of from 2 to 40 amino acid residues, such as from 2 to 15 amino acid residues.<br>
The moiety X may constitute either the N-terminal or the C-terminal region of the overall hybrid enzyme.<br>
It will thus be apparent from the above that the CBM in a hybrid enzyme of the type in question may be positioned C-terminally, N-terminally or internally in the hybrid enzyme.<br>
Linker sequence<br>
The linker sequence may be any suitable linker sequence. In preferred embodiments the linker sequence is derived from the Athelia rolfsii AMG, the Aniger AMG or the A. kawachii alpha-amylase such as a linker sequence selected from the list consisting of A. niger AMG linker: T GGTTTTATPTGSGSVTSTSKTTATASKTSTSTSSTSA (SEQ ID NO:26), A. kawachii alpha-amylase linker: TTTTTTAAATSTSKATTSSSSSSA A A T T S S S (SEQ ID NO:27), Athelia rolfsii AMG linker: GATSPGGSSGS (SEQ ID NO:28), and the PEPT linker: PEPTPEPT (SEQ ID NO:29). In another preferred embodiment the hybrid enzymes has a linker sequence which differs from the amino acid sequence shown in SEQ ID NO:26, SEQ ID NO:27, SEQ ID NO:28, or SEQ ID NO:29 in no more than 10 positions, no more than 9 positions, no more than 8 positions, no more than 7 positions, no more than 6 positions, no more than 5 positions, no more than 4 positions, no more than 3 positions, no more than 2 positions, or even no more than 1 position.<br>
Carbohydrate-binding modules<br>
A carbohydrate-binding module (CBM), or as often referred to, a carbohydrate-binding<br>
domain (CBD), is a polypeptide amino acid sequence which binds preferentially to a poly- or<br>
oligosaccharide (carbohydrate), frequently - but not necessarily exclusively - to a water-<br>
insoiuble (including crystalline) form thereof.<br>
i	CBMs derived from starch degrading enzymes are often referred to as starch-binding<br>
modules or SBMs (CBMs which may occur in certain amylolytic enzymes, such as certain<br><br>
glucoamylases, or in enzymes such as cyclodextrin glucanotransferases, or in alpha-amylases). Likewise, other sub-classes of CBMs would embrace, e.g. cellulose-binding modules (CBMs from cellulolytic enzymes), chltin-binding modules (CBMs which typically occur in chitinases), xylan-binding modules (CBMs which typically occur in xylanases), mannarvbinding modules (CBMs which typically occur in mannanases). SBMs are often referred to as SBDs (Starch Binding Domains)<br>
CBMs are found as integral parts of large polypeptides or proteins consisting of two or<br>
more polypeptide amino acid sequence regions, especially in hydrolytic enzymes<br>
(hydrolases) which typically comprise a catalytic module containing the active site for<br>
substrate hydrolysis and a carbohydrate-binding module (CBM) for binding to the<br>
carbohydrate substrate in question. Such enzymes can comprise more than one catalytic<br>
module and one, two or three CBMs, and optionally further comprise one or more<br>
polypeptide amino acid sequence regions linking the CBM(s) with the catalytic module(s), a<br>
region of the latter type usually being denoted a "linker*. Examples of hydrolytic enzymes<br>
comprising a CBM - some of which have already been mentioned above - are cellulases,<br>
xylanases, mannanases, arabinofuranosidases, acetylesterases and chitinases. CBMs have<br>
also been found in algae, e.g. in the red alga Porphyra purpurea in the form of a non-<br>
hydrolytic polysaccharide-binding protein.	'<br>
In proteins/polypeptides in which CBMs occur (e.g. enzymes, typically hydrolytic enzymes), a CBM may be located at the N or C terminus or at an internal position.<br>
That part of a polypeptide or protein (e.g. hydrolytic enzyme) which constitutes a CBM per se typically consists of more than about 30 and less than about 250 amino acid residues. The "Carbohydrate-Binding Module of Family 20" or a CBM-20 module is in the context of this invention defined as a sequence of approximately 100 amino acids having at least 45% homology to the Carbohydrate-Binding Module (CBM) of the polypeptide disclosed in figure 1 by Joergensen et al (1997) in Biotechnol. Lett. 19:1027-1031. The CBM comprises the last 102 amino acids of the polypeptide, i.e. the subsequence from amino acid 582 to amino acid 683. The numbering of Glycoside Hydrolase Families applied in this disclosure follows the concept of Coutinho, P.M. &amp; Henrissat, B. (1999) CAZy - Carbohydrate-Active Enzymes server at URL: httD://afmb.cnrs-mrs.fr/-cazv/CAZY/index.html or alternatively Coutinho, P.M. &amp; Henrissat, B. 1999; The modular structure of cellulases and other carbohydrate-active enzymes: an integrated database approach. In "Genetics, Biochemistry and Ecology of Cellulose Degradation", K. Ohmiya, K. Hayashi, K. Sakka, Y. Kobayashi, S. Karita and T. Kimura eds., Uni Publishers Co., Tokyo, pp. 15-23, and Bourne, Y. &amp; Henrissat, B. 2001; Glycoside hydrolases and giycosyltransferases: families and functional modules, Current Opinion in Structural Biology 11:593-600.<br>
Examples of enzymes which comprise a CBM suitable for use in the context of the invention   are   alpha-amylases,   maltogenic   alpha-amylases,   cellulases,   xylanases,<br><br>
mannanases, arabinofuranosidases, acetylesterases and chitinases. Further CBMs of interest in relation to the present invention include CBMs deriving from glucoamylases (EC 3.2.1.3) or from CGTases (EC 2.4.1.19).<br>
CBMs deriving from fungal, bacterial or plant sources will generally be suitable for use in the context of the invention. Preferred are CBMs of fungal origin, more preferably from Aspergillus sp., bacillus sp., Klebsiella sp., or Rhizopus sp. In this connection, techniques suitable for isolating the relevant genes are well known in the art.<br>
Preferred for the invention is CBMs of Carbohydrate-Binding Module Family 20. CBMs of Carbohydrate-Binding Module Family 20 suitable for the invention may be derived from glucoamylases of Aspergillus awamori (SWISSPROT Q12537), Aspergillus kawachii (SWISSPROT P23176), Aspergillus niger (SWISSPROT P04064), Aspergillus oryzae (SWISSPROT P36914), from alpha-amylases of Aspergillus kawachii (EMBL#AB008370), Aspergillus nidulans (NCBI AAF17100.1), from beta-amylases of Bacillus cereus (SWISSPROT P36924), or from CGTases of Bacillus circulans (SWISSPROT P43379). Preferred is a CBM from the alpha-amyiase of Aspergillus kawachii (EMBL:#AB008370) as well as CBMs having at least 50%, 60%, 70%, 80% or even at least 90% homology to the CBM of the alpha-amyiase of Aspergillus kawachii (EMBL#AB008370), i.e. a CBM having at least 50%, 60%, 70%, 80% or even at least 90% homology to the amino acid sequence of SEQ ID NO:6. Also preferred for the invention are the CBMs of Carbohydrate-Binding Module Family 20 having the amino acid sequences shown in SEQ ID NQ:9, SEQ ID NO:10, and SEQ ID NO:11 and disclosed in Danish patent application PA 2003 00949 as SEQ ID NO:1, SEQ ID NO:2 and SEQ ID NO:3 respectively. Further preferred CBMs include the CBMs of the glucoamylase from Hormoconis sp. such as from Hormoconis resinae (Syn. Creosote fungus or Amorphotheca resinae) such as the CBM of SWISSPROT:Q03045 (SEQ ID NO:12), from Lentinula sp. such as from Lentinula edodes (shiitake mushroom) such as the CBM of SPTREMBLQ9P4C5 (SEQ ID NO:13), from Neurospora sp. such as from Neurospora crassa such as the CBM of SWISSPROT:P14804 (SEQ ID NO:14), from Talaromyces sp. such as from Talaromyces byssochlamydioides such as the CBM of NN005220 (SEQ ID NO:15), from Geosmithia sp. such as from Geosmithia cylindrospora, such as the CBM of NN48286 (SEQ ID NO:16), from Scorias sp. such as from Scorias spongiosa such as the CBM of NN007096 (SEQ ID NO:17), from Eupenicillium sp. such as from Eupenicillium ludwigii such as the CBM of NN005968 (SEQ ID NO:18), from Aspergillus sp. such as from Aspergillus japonicus such as the CBM of NN001136 (SEQ ID NO:19), from Penicillium sp, such as from Penicillium cf. miczynskii such as the CBM of NN48691 (SEQ ID NO:20), from Mz1 Penicillium sp. such as the CBM of NN48690 (SEQ ID NO:21), from Thysanophora sp. such as the CBM of NN48711 (SEQ ID NO:22), and from Humicola sp. such as from Humicola grisea var. thermoidea such as the CBM of SPTREMBLQ12623 (SEQ ID NO:23). Most preferred CBMs include the CBMs of the giucoamyiase from<br><br>
Aspergillus sp. such as from Aspergillus niger, such as SEQ ID NO:24, and Athelia sp. such as from Athelia raffs//, such as SEQ ID N0.25. Also preferred for the invention is any CBD having at least 50%, 60%, 70%, 80% or even at least 90% homology to any of the afore mentioned CBD amino acid sequences.<br>
Further suitable CBMs of Carbohydrate-Binding Module Family 20 may be found at URL: http://afmb.cnrs-mrs.fr/-cazv/CAZY/index.htmn.<br>
Once a nucleotide sequence encoding the substrate-binding (carbohydrate-binding) region has been identified, either as cDNA or chromosomal DNA, it may then be manipulated in a variety of ways to fuse it to a DNA sequence encoding the enzyme of interest. The DNA fragment encoding the carbohydrate-binding amino acid sequence, and the DNA encoding the enzyme of interest are then ligated with or without a linker. The resulting ligated DNA may then be manipulated in a variety of ways to achieve expression.<br>
Alpha-amyloiytic sequence<br>
Alpha-amylases (in particular acid stable alpha-amylases) which are appropriate as the basis for CBM/amylase hybrids of the types employed in the context of the present invention include those of fungal origin.<br>
Preferably the aipha-amylase is a wild type enzyme. More preferably the alpha-amylase is a variant alpha-amylases comprising amino acid modifications leading to increased activity, increased protein stability at low pH, and/or at high pH, increased stability towards calcium depletion, and/or increased stability at elevated temperature. Chemically or genetically modified mutants of such alpha-amylases are included in this connection.<br>
Relevant alpha-amylases include, for example, alpha-amylases obtainable from Aspergillus species, in particular from Aspergillus niger, such as an acid stable alpha-amylase (SWISSPROT P56271), described in more detail in WO 8901969 (example 3) and having the amino acid sequence shown in SEQ ID NO:8 and/or encode by the DNA sequence shown in SEQ ID NO:7. Also preferred are alpha-amylase sequences having more than 50%, such as 60%, or 70%, 80% or 90% homology to the amino acid sequence shown in SEQ ID NO:8 and/or encode by the DNA sequence shown in SEQ ID NO:7.<br>
In another preferred embodiment the alpha-amylolytic sequence is derived from the A.oryzae acid alpha-amylase (Fungamyl™). More preferably the alpha-amylolytic sequence has more than 50%, such as 60%, or 70%, 80% or 90% homology to the amino acid sequence shown in SEQ ID NO:30 and/or to the sequence shown as amino acids 21-498 of the amino acid sequence shown in SEQ ID NO:30.<br>
Even more preferred is an embodiment wherein the hybrid enzyme comprises an alpha-amyloiytic sequence derived from the A.oryzae acid alpha-amylase (Fungamyl™, SEQ ID NO:30), and/or a linker sequence derived from the A. kawachii alpha-amylase or the A. rolfsii AMG, and/or a CBM derived from the A. kawachii alpha-amylase (SEQ ID NO:5) or the A.<br><br>
rolfsii AMG (SEQ ID NO:25). In a particular such embodiment the hybrid enzyme has the amino acid sequence shown in SEQ ID NO:36 or in SEQ ID NO:40.<br>
Also preferred is an embodiment wherein the hybrid enzyme comprises an alpha-amyiolytic sequence derived from the Aniger acid alpha-amylase catalytic module having the sequence shown in SEQ ID NO:8, and/or a linker sequence derived from the A. kawachii alpha-amylase or the A rolfsii AMG, and/or the CBM is derived from the A. kawachii alpha-amylase, the A. rolfsii AMG or the A.niger AMG. In a particularly preferred embodiment the hybrid enzyme comprises the Aniger acid alpha-amylase catalytic module having the sequence shown in SEQ ID NO:8 and the A. kawachii alpha-amylase linker and CBM (SEQ ID NO:6).<br>
Preferably the hybrid enzyme comprises a CBD sequence having at least 50%, 60%, 70%, 80% or even at least 90% homology to any of the amino acid sequences shown in SEQ ID NO:9, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25. Even more preferred the hybrid enzyme comprises a CBD sequence having an amino acid sequence shown in SEQ ID NO:9, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19f SEQ ID NO;20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25. In yet another preferred embodiment the CBM sequence has an amino acid sequence which differs from the amino acid sequence amino acid sequence shown in SEQ ID NO:9, SEQ ID NO:10T SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25 in no more than 10 amino acid positions, no more than 9 positions, no more than 8 positions, no more than 7 positions, no more than 6 positions, no more than 5 positions, no more than 4 positions, no more than 3 positions, no more than 2 positions, or even no more than 1 position. In a most preferred embodiment the hybrid enzyme comprises a CBM derived from an AMG from A. rolfsii, such as the AMG from A. rolfsii AHU 9627 described in US patent 4,727,026.<br>
In a particular such embodiment the hybrid enzyme has the amino acid sequence shown in SEQ ID NO:32, SEQ ID NO;34 or SEQ ID NO:38 or the hybrid enzyme has an amino acid sequence having at least 50%, 60%, 70%, 80% or even at least 90% homology to any of the afore mentioned amino acid sequences.<br>
In yet another preferred embodiment the hybrid enzymes has an amino acid sequence which differs from the amino acid sequence amino acid sequence shown in SEQ ID NO:32, SEQ ID NO:34, SEQ ID N
 <br>
positions, no more than 5 positions, no more than 4 positions, no more than 3 positions, no more than 2 positions, or even no more than 1 position.<br>
In a preferred embodiment the hybrid enzymes is a variant comprises the catalytic domain shown as amino acids 21-498 of SEQ ID NO:40 (and/or amino acids 21-498 of SEQ ID NO:30) with one or more substitutions, more preferably a substitution in one or more positions selected from the list consisting of. 81,158,161,163,164,175,176,177, 253, 264, 266, 466, 468, 470, and most preferably one or more substitutions selected from the list consisting of: Q81R, K158D, K158V, S161D, S161N, Q163S, Q163A, D164S, Y175W, E176D, D177N, D253N, N264K, N264E, M266L, G466D, D468S, and N470D. In an even more preferred embodiment the hybrid enzymes is one of the variants listed in Table 6. Also preferred are variants comprising the catalytic domain shown as amino acids 21-498 of SEQ ID NO:40 (and/or amino acids 21-498 of SEQ ID NO:30) with one or more substations or combination of substitutions selected from the list comprising: Y175W + E176D, 276W + P277Q, K380T + L381W, G62N, G62F, Y95D, D106K+ S109D, A140P, K158D + D164S, F171L+E182Q, K200E + V202A, K200E + V202A, K200E + V202A, T227S + K233P, L252D + D255N, N264K + M266L, K283E, N359K+ D360V, N390K + Y391L, G466D + N470D, Y484L + S498S, Y484L + S498G.and Y484L + S498R.<br>
In a preferred embodiment the hybrid enzyme comprises the catalytic module according shown as amino acids 21-498 of SEQ ID NO:40 (and/or amino acids 21-498 of SEQ ID NO:30) with one or more substitutions selected from the list consisting of: K158V, S161N, Q163A, D164S, N264K, M266L, G466D, D468S and N470D.<br>
In another preferred embodiment the hybrid enzyme comprises the catalytic module shown as amino acids 21-498 of SEQ ID NO:40 (and/or amino acids 21-498 of SEQ ID NO:30) with one or more substitutions selected from the list consisting of: Q81R, K158V, S161N, Q163A, D164S, Y175W, E176D, D177N, N264K, M266L, G466D, D468S and N470D.<br>
Expression vectors<br>
The present invention also relates to recombinant expression vectors which may comprise a DNA sequence encoding the hybrid enzyme or a genetically modified wild type enzyme, a promoter, a signal peptide sequence, and transcriptional and translational stop signals. The various DNA and control sequences described above may be joined together to produce a recombinant expression vector which may include one or more convenient restriction sites to allow for insertion or substitution of the DNA sequence encoding the polypeptide at such sites. Alternatively, the DNA sequence of the present invention may be expressed by inserting the DNA sequence or a DNA construct comprising the sequence into an appropriate vector for expression. In creating the expression vector, the coding sequence is<br><br>
located in the vector so that the coding sequence is operably linked with the appropriate control sequences for expression, and possibly secretion.<br>
The recombinant expression vector may be any vector (e.g. a plasmid or vims), which can be conveniently subjected to recombinant DNA procedures and can bring about the expression of the DNA sequence. The choice of the vector will typically depend on the compatibility of the vector with the host cell into which the vector is to be introduced. The vectors may be linear or closed circular plasmids. The vector may be an autonomously replicating vector, i.e. a vector which exists as an extrachromosomal entity, the replication of which is independent of chromosomal replication, e.g. a plasmid, an extrachromosomal element, a minichromosome, a cosmid or an artificial chromosome. The vector may contain any means for assuring self-replication. Alternatively, the vector may be one which, when introduced into the host cell, is integrated into the genome and replicated together with the chromosome(s) into which it has been integrated. The vector system may be a single vector or plasmid or two or more vectors or plasmids which together contain the total DNA to be introduced into the genome of the host cell, or a transposon.<br>
Markers<br>
The vectors of the present invention preferably contain one or more selectable markers, which permit easy selection of transformed cells. A selectable marker is a gene the product of which provides for biocide or viral resistance, resistance to heavy metals, prototrophy to auxotrophs, and the like.<br>
Examples of selectable markers for use in a filamentous fungus host cell may be selected from the group including, but not limited to, amdS (acetamidase), argB (ornithine carbamoyltransferase), bar (phosphinothricin acetyltransferase), hygB (hygromycin phosphotransferase), niaD (nitrate reductase), pyrG (oroijdine-5'-phosphate decarboxylase), sC (sulfate adenyltransferase), trpC (anthranilate synthase), and glufosinate resistance markers, as well as equivalents from other species. Preferred for use in an Aspergillus cell are the amdS and pyrG markers of Aspergillus nidulans or Aspergillus oryzae and the bar marker of Streptomyces hygroscopicus. Furthermore, selection may be accomplished by co-transformation, e.g. as described in WO 91/17243, where the selectable marker is on a separate vector.<br>
The vectors of the present invention preferably contain an element(s) that permits stable integration of the vector into the host cell genome or autonomous replication of the vector in the cell independent of the genome of the cell.<br>
The vectors of the present invention may be integrated into the host cell genome when introduced into a host cell. For integration, the vector may rely on the DNA sequence encoding the polypeptide of interest or any other element of the vector for stable integration<br><br>
of the vector into the genome by homologous or none homologous recombination. Alternatively, the vector may contain additional DNA sequences for directing integration by homologous recombination into the genome of the host cell. The additional DNA sequences enable the vector to be integrated into the host cell genome at a precise location(s) in the chromosome(s). To increase the likelihood of integration at a precise location, the integrational elements should preferably contain a sufficient number of DNAs, such as 100 to 1,500 base pairs, preferably 400 to 1,500 base pairs, and most preferably 800 to 1,500 base pairs, which are highly homologous with the corresponding target sequence to enhance the probability of homologous recombination. The integrational elements may be any sequence that is homologous with the target sequence in the genome of the host cell. Furthermore, the integrational elements may be non-encoding or encoding DNA sequences. On the other hand, the vector may be integrated into the genome of the host cell by non-homologous recombination. These DNA sequences may be any sequence that is homologous with a target sequence in the genome of the host cell, and, furthermore, may be non-encoding or encoding sequences.<br>
For autonomous replication, the vector may further comprise an origin of replication enabling the vector to replicate autonomously in the host cell in question.<br>
The episomal replicating the AMA1 plasmid vector disclosed in WO 00/24883 may be used.<br>
More than one copy of a DNA sequence encoding a polypeptide of interest may be inserted into the host cell to amplify expression of the DNA sequence. Stable amplification of the DNA sequence can be obtained by integrating at least one additional copy of the sequence into the host cell genome using methods well known in the art and selecting for transfomnants.<br>
The procedures used to ligate the elements described above to construct the recombinant expression vectors of the present invention are well known to one skilled in the art (see, e.g. Sambrook et al, 1989, Molecular Cloning, A Laboratory Manual, 2nd edition, Cold Spring Harbor, New York).<br>
Host cells<br>
The host cell of the invention, either comprising a DNA construct or an expression vector comprising the DNA sequence encoding the hybrid enzyme or a genetically modified wild type enzyme, is advantageously used as a host cell in the recombinant production of the hybrid enzyme or a genetically modified wild type enzyme. The cell may be transformed with an expression vector. Alternatively, the cell may be transformed with the DNA construct of the invention encoding the hybrid enzyme or a genetically modified wild type enzyme,<br><br>
conveniently by integrating the DNA construct (in one or more copies] in ine nosi cnromu-some. Integration of the DNA construct into the host chromosome may be performed according to conventional methods, e.g. by homologous or heterologous recombination.<br>
The host cell may be any appropriate prokaryotic or eukaryotic cell, e.g. a bacterial cell, a filamentous fungus cell, a plant cell or a mammalian cell.<br>
In a preferred embodiment, the host cell is a filamentous fungus represented by the following groups of Ascomycota, include, e.g. Neurospora, Eupeniciltium (-Penicillium), Emericella (^Aspergillus), Eurotium {-Aspergillus).<br>
In a more preferred embodiment, the filamentous fungus include all filamentous forms of the subdivision Eumycota and Oomycota (as defined by Hawksworth et al. In,<br>
Ainsworth and Bisbfs Dictionary of The Fungi, 8th edition, 1995, CAB International,<br>
i<br>
University Press, Cambridge, UK. The filaimentous fungi are characterized by a vegetative mycelium composed of chitin, cellulose, glucan, chttosan, mannan, and other complex polysaccharides. Vegetative growth is by hyphal elongation and carbon catabolism is obligateiy aerobic.<br>
In an even more preferred embodiment, the filamentous fungus host cell is a cell of a species of, but not limited to a ceil selected from the group consisting of a strain belonging to a species of Aspergillus, preferably Aspergillus oryzae, Aspergillus niger, Aspergillus awamori, Aspergillus kawachii, or a strain of Bacillus, or a strain of Fusarium, such as a strain of Fusarium oxysporium, Fusarium graminearum (in the perfect state named Gribberella zeae, previously Sphaeria zeae, synonym with Gibberella roseum and Gibberella roseum 1 sp. cerealis), or Fusarium sulphureum (in the prefect state named Gibberella puricaris, synonym with Fusarium trichothecioides, Fusarium bactridioides, Fusarium sambucium, Fusarium roseum, and Fusarium roseum var. graminearum), Fusarium cerealis (synonym with Fusarium crookwellense), or Fusarium venenatum,<br>
In a most preferred embodiment, the filamentous fungus host cell is a cell of a strain belonging to a species of Aspergillus, preferably Aspergillus oryzae, or Aspergillus niger.<br>
The host cell may be a wild type filamentous fungus host cell or a variant, a mutant or a genetically modified filamentous fungus host cell. In a preferred embodiment of the invention the host cell is a protease deficient or protease minus strain. Also specifically contemplated is Aspergillus strains, such as Aspergillus niger strains, genetically modified to disrupt or reduce expression of glucoamylase, acid-stable alpha-amylase, alpha-1,6 transglucosidase, and protease activities.<br>
Transformation of filamentous fungus host cells<br>
Filamentous fungus host cells may be transformed by a process involving protoplast formation, transformation of the protoplasts, and regeneration of the cell wall in a manner<br><br>
known in the art. Suitable procedures for transformation of Aspergillus host cells are described in EP 238 023, EP 184 438, and Yelton et al, 1984, Proceedings of the National Academy of Sciences USA 81:1470-1474. A suitable method of transforming Fusarium species is described by Malardier et al. 1989, Gene 78:147-156 or US 6,060,305.<br>
Isolating and cloning a DNA sequence encoding a parent alpha-amylase<br>
The techniques used to isolate or clone a DNA sequence encoding a polypeptide of interest are known in the art and include isolation from genomic DNA, preparation from cDNA, or a combination thereof. The cloning of the DNA sequences of the present invention from such genomic DNA can be effected, e.g. by using the well known polymerase chain reaction (PCR) or antibody screening of expression libraries to detect cloned DNA fragments with shared structural features. See, e.g. Innis et al., 1990, PCR: A Guide to Methods and Application, Academic Press, New York. Other DNA amplification procedures such as ligase chain reaction (LCR), iigated activated transcription (LAT) and DNA sequence-based amplification (NASBA) may be used.<br>
The DNA sequence encoding a parent alpha-amylase may be isolated from any cell or microorganism producing the alpha-amylase in question, using various methods well known in the art. First, a genomic DNA and/or cDNA library should be constructed using chromosomal DNA or messenger RNA from the organism that produces the alpha-amylase to be studied. Then, if the amino acid sequence of the alpha-amylase is known, labeled oligonucleotide probes may be synthesized and used to identify alpha-amylase-encoding clones from a genomic library prepared from the organism in question. Alternatively, a labelled oligonucleotide probe containing sequences homologous to another known alpha-amylase gene could be used as a probe to identify alpha-amylase-encoding clones, using hybridization and washing conditions of very low to very high stringency.<br>
Yet another method for identifying alpha-amylase-encoding clones would involve inserting fragments of genomic DNA into an expression vector, such as a plasmid, transforming alpha-amylase-negative bacteria with the resulting genomic DNA library, and then plating the transformed bacteria onto agar containing a substrate for alpha-amylase {i.e. maltose), thereby allowing clones expressing the alpha-amylase to be identified.<br>
Alternatively, the DNA sequence encoding the enzyme may be prepared synthetically by established standard methods, e.g. the phosphoroamid'rte method described S.L Beaucage and M.H. Caruthers, (1981), Tetrahedron Letters 22, p. 1859-1869, or the method described by Matthes et al. (1984), EMBO J. 3, p. 801-805. In the phosphoroamidite method, oligonucleotides are synthesized, e.g. in an automatic DNA synthesizer, purified, annealed, Iigated and cloned in appropriate vectors.<br>
Finally, the DNA sequence may be of mixed genomic and synthetic origin, mixed synthetic and cDNA origin or mixed genomic and cDNA origin, prepared by ligating fragments of<br><br>
synthetic, genomic or cDNA origin (as appropriate, the fragments corresponding to various parts of the entire DNA sequence), in accordance with standard techniques. The DNA sequence may also be prepared by polymerase chain reaction (PCR) using specific primers, for instance as described in US 4,683,202 or R.K. Saiki et ai. (1988), Science 239,1988, pp. 487-491.<br>
isolated DNA sequence<br>
The present invention relates, inter alia, to an isolated DNA sequence comprising a DNA sequence encoding a hybrid enzyme or a genetically modified wild type enzyme comprising an amino acid sequence of a catalytic module having alpha-amylase activity and an amino acid sequence of a carbohydrate-binding module, wherein the catalytic module is of fungal origin.<br>
The term "isolated DNA sequence" as used herein refers to a DNA sequence, which is essentially free of other DNA sequences, e.g. at least about 20% pure, preferably at least about 40% pure, more preferably at least about 60% pure, even more preferably at least about 80% pure, and most preferably at least about 90% pure as determined by agarose electrophoresis.<br>
For example, an isolated DNA sequence can be obtained by standard cloning procedures used in genetic engineering to relocate the DNA sequence from its natural location to a different site where it will be reproduced. The cloning procedures may involve excision and isolation of a desired DNA fragment comprising the DNA sequence encoding the polypeptide of interest, insertion of the fragment into a vector molecule, and incorporation of the recombinant vector into a host cell where multiple copies or clones of the DNA sequence will be replicated. An isolated DNA sequence may be manipulated in a variety of ways to provide for expression of the polypeptide of interest. Manipulation of the DNA sequence prior to its insertion into a vector may be desirable or necessary depending on the expression vector. The techniques for modifying DNA sequences utilizing recombinant DNA methods are well known in the art.<br>
DNA construct<br>
The present invention relates, inter alia, to a DNA construct comprising a DNA sequence encoding a hybrid enzyme or a genetically modified wild type enzyme comprising an amino acid sequence of a catalytic module having alpha-amylase activity and an amino acid sequence of a carbohydrate-binding module, wherein the catalytic module is of fungal origin. BDNA construct" is defined herein as a DNA molecule, either single- or double-stranded, which is isolated from a naturally occurring gene or which has been modified to contain<br><br>
segments of DNA, which are combined and juxtaposed in a manner, which would not otherwise exist in nature. The term DNA construct is synonymous with the term expression cassette when the DNA construct contains all the control sequences required for expression of a coding sequence of the present invention.<br>
Site-directed mutagenesis<br>
Once a parent alpha-amylase-encoding DNA sequence has been isolated, and desirable sites for mutation identified, mutations may be introduced using synthetic oligonucleotides. These oligonucleotides contain nucleotide sequences flanking the desired mutation sites. In a specific method, a single-stranded gap of DNA, the alpha-amylase-encoding sequence, is created in a vector carrying the alpha-amylase gene. Then the synthetic nucleotide, bearing the desired mutation, is annealed to a homobgous portion of the single-stranded DNA. The remaining gap is then filled in with DNA polymerase I (Klenow fragment) and the construct is ligated using T4 iigase. A specific example of this method is described In Morinaga et al. (1984), Biotechnology 2, p. 646-639. US 4,760,025 disclose the introduction of oligonucleotides encoding multiple mutations by performing minor alterations of the cassette. However, an even greater variety of mutations can be introduced at any one time by the Morinaga method, because a multitude of oligonucleotides, of various lengths, can be introduced.<br>
Another method for introducing mutations into alpha-amylase-encoding DNA sequences is described in Nelson and Long, (1989), Analytical Biochemistry 180, p. 147-151. It involves the 3-step generation of a PCR fragment containing the desired mutation introduced by using a chemicaliy synthesized DNA strand as one of the primers in the PCR reactions. From the PCR-generated fragment, a DNA fragment carrying the mutation may be isolated by cleavage with restriction endonucleases and reinserted into an expression plasmid.<br>
Localized random mutagenesis<br>
The random mutagenesis may be advantageously localized to a part of the parent alpha-amylase in question. This may, e.g. be advantageous when certain regions of the enzyme have been identified to be of particular importance for a given property of the enzyme, and when modified are expected to result in a variant having improved properties. Such regions may normally be identified when the tertiary structure of the parent enzyme has been elucidated and related to the function of the enzyme.<br>
The localized or region-specific, random mutagenesis is conveniently performed by use of PCR generated mutagenesis techniques as described above or any other suitable technique known in the art. Alternatively, the DNA sequence encoding the part of the DNA sequence to be modified may be isolated, e.g. by insertion into a suitable vector, and said part<br><br>
may be subsequently subjected to mutagenesis by use of any of the mutagenesis methods discussed above.<br>
Variants of hybrid or wild type enzymes<br>
The performance in a starch degradation process of a wild type or hybrid enzyme comprising a carbohydrate-binding module ("CBM") and an alpha-amylase catalytic module may be improved through protein engineering, such as by site directed mutagenesis, by localized random mutagenesis, by synthetically preparing a new variant of the parent wild type enzyme or parent hybrid enzyme, or by any other suitable protein engineering techniques.<br>
Parent alpha-amylase contemplated for the invention include wild-type fungal alpha-amylase having a CBM, in particular fungal alpha-amylase obtainable from an Aspergillus strain, such as an Aspergillus kawachii acid alpha-amylase and variants or mutants thereof, homologous alpha-amylases, and further wild type or artificial alpha-amyiases being structurally and/or functionally similar to the acid alpha-amylase from Aspergillus kawachii showninSEQIDNO:41.<br>
While the acid alpha-amylase from Aspergillus kawachii is an example of a wild type alpha-amylase comprising a carbohydrate-binding module (uCBMn)f the Aspergillus kawachii acid alpha-amylase catalytic module has a limited specific activity, e.g. compared to the catalytic activity of the acid alpha-amylase from A. niger. Furthermore, the CBM of the Aspergillus kawachii alpha-amylase is easily cleaved off from the catalytic module during a conventional fermentation process thereby reducing the suitability of the enzyme for industrial application. Thus the Aspergillus kawachii alpha-amylase has several serious drawbacks as an industrial enzyme for raw starch hydrolysis. Protein engineering techniques can be applied to yield more suitable variants of a parent acid alpha-amylase wherein the catalytic activity and/or the enzyme stability has been improved.<br>
Preferred variants are variants of the fungal alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41, which variant comprising an alteration at one or more of the positions: 13,15,18, 31, 32, 33, 34, 35, 36, 61, 63, 64, 68, 69, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 89, 117, 118, 119, 120, 121, 122, 123, 124, 125, 152, 153, 154, 155, 156, 157, 158,161,162, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 204, 205, 206, 207, 208, 209, 210, 211, 216, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 242, 245, 250, 252, 253, 255, 256, 257,259, 260, 275, 292, 295, 296, 297, 298, 299, 304, 328, 339, 344, 348, 378, 383, 386, 387, 405, 448 and 480 and more preferred at one or more of the positions 31, 33, 36, 74, 75, 77, 84,120, 153, 154, 155, 156, 157, 158, 162, 166, 169, 170, 199, 232, 233, 235, 238, 239, 245, 256, 257, 331, 336, 339, 340, 342, 348, 378, 383, 386, 387, 405, 448 and 480 wherein (a) the<br><br>
alteration^) are independently (i) an insertion of an amino acid downstream of the amino acid which occupies the position, (ii) a deletion of the amino acid which occupies the position, or iii) a substitution of the amino acid which occupies the position with a different amino acid, (b) the variant has increased acid alpha-amyfase activity and or improved enzyme stability relative to the parent fungal alpha-amyiase and (c) each position corresponds to a position of the amino acid sequence of the TAKA amylase shown in SEQ ID NO:43 and/or the A.kawachii alpha-amylase shown as SEQ ID NO:41.<br>
More preferred variants are variants of the A.kawachii alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41 and comprising alterations in one or more of the following positions: 13,15,18, 31, 32, 33, 34, 35, 36, 61, 63, 64, 68, 69, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 89, 117, 118, 119, 120, 121, 122, 123, 124, 125, 152, 153, 154, 155, 156, 157, 158,161,162, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 204, 205, 206, 207, 208, 209, 210, 211, 216, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 242, 245, 250, 252, 253, 255, 256, 257,259, 260, 275, 292, 295, 296, 297, 298, 299, 304, 328, 339, 344, 348, 378, 383, 386, 387, 405, 448 and 480 wherein (a) the alteration(s) are independently (i) an insertion of an amino acid downstream of the amino acid which occupies the position, (ii) a deletion of the amino acid which occupies the position, or iii) a substitution of the amino acid which occupies the position with a different amino acid, (b) the variant has increased acid alpha-amyiase activity and or improved enzyme stability relative to the parent fungal alpha-amylase and (c) each position corresponds to a position of the amino acid sequence of the TAKA amylase shown in SEQ ID NO:43 and/or the A.kawachii alpha-amylase shown as SEQ ID NO:41. Even more preferred variants of the A.kawachii alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amyiase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41 and comprising alterations in one or more of the following positions: 31, 33, 36, 74, 75, 77, 84, 120,153, 154, 155, 156, 157, 158, 162, 166, 169, 170, 199, 232, 233, 235, 238, 239, 245, 256, 257, 331, 336, 339, 340, 342, 348, 378, 383, 386, 387,405,448 and 480.<br>
Yet more preferred are variants of the A.kawachii alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41 comprising one or more of the following amino acid substitution: G33A, I36K, S74A, D75Y, E77D, P120A, I153D, D154N, W155Y, D156E, N157D, L158Q, Q162E, E166L, T169N, I170T, E199K, E199L, D232L, N233D, N235D, L238Y, D239T, W256Y, Q257P, E331Q, S336A, D339K, D339N, V340D, and Y342A, and most preferred variants comprises one or more of the following amino acid substitution S74A, E166L, E199L, D339K, and D156E, such as the variant comprising the multiple amino acid substitution: S74A/E166L/E199L,<br><br>
wherein the variant has improve activity and/or enzyme stability relative to the parent alpha-amylase shown as SEQ ID NO:41.<br>
The variants may also be a variants of the A. kawachii alpha-amyiase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41 and comprising alterations in one or more of the following positions: 31, 74, 89, 209, 245, 348, 378, 383, 386, 387, 405, 448, and 480, wherein (a) the alteration(s) are independently (i) an insertion of an amino acid downstream of the amino acid which occupies the position, (ii) a deletion of the amino acid which occupies the position, or iii) a substitution of the amino acid which occupies the position with a different amino acid, (b) the variant has increased acid alpha-amyiase activity and or improved enzyme stability relative to the parent fungal alpha-amylase.<br>
Preferably the variants is a variants of the A.kawachii alpha-amylase shown as SEQ ID NO:41 or variants of an alpha-amylase having at least 60% homology, at least 70% homology, at least 80% homology, or even at least 90% homology to SEQ ID NO:41 and comprising one or more of the following substitutions: N31D, S74A, Y89D, E209L, Y245V, D348K, D378A, K383A, P386A, I387F, I405V, N448S, and N480R, wherein the variant has increased acid alpha-amyiase activity and or improved enzyme stability relative to the parent fungal alpha-amylase shown as SEQ ID NO:41.<br>
Most preferred is a variants of the A.kawachii alpha-amylase shown as SEQ ID NO:41 and comprising the following substitutions: N31D, S74A, Y89D, E209L, Y245V, D348K, D378A, K383A, P386A, I387F, I405V, N448S, and N480R.<br>
The variants may be produced using conventional protein engineering techniques<br>
Expression of the enzymes in plants<br>
A DNA sequence encoding an enzyme of interest, such as a hybrid enzyme or a variant of a wild type enzyme or a hybrid of the present invention, may be transformed and expressed in transgenic plants as described below.<br>
The transgenic plant can be dicotyledonous or monocotyledonous, for short a dicot or a monocot Examples of monocot plants are grasses, such as meadow grass (blue grass, Poa), forage grass such as Festuca, Lolium, temperate grass, such as Agrostis, and cereals, e.g. wheat, oats, rye, barley, rice, sorghum and maize (corn).<br>
Examples of dicot plants are tobacco, legumes, such as lupins, potato, sugar beet, pea, bean and soybean, and cruciferous plants (family Brassicaceae), such as cauliflower, oil seed rape and the closely related model organism Arabidopsis thaliana.<br>
Examples of plant parts are stem, callus, leaves, root, fruits, seeds, and tubers as well as the individual tissues comprising these parts, e.g. epidermis, mesophyll, parenchyme, vascular tissues, meristems. In the present context, also specific plant cell<br><br>
compartments, such as chloroplast, apoplast, mitochondria, vacuole, peroxisomes and cytoplasm are considered to be a plant part. Furthermore, any plant cell, whatever the tissue origin, is considered to be a plant part. Likewise, plant parts such as specific tissues and ceBs isolated to facilitate the utilisation of the invention are also considered plant parts e.g. embryos, endosperms, aleurone and seeds coats.<br>
Also included within the scope of the invention are the progeny of such plants, plant parts and plant cells.<br>
The transgenic plant or plant cell expressing the enzyme of interest may be constructed in accordance with methods known in the art. In short the plant or plant cell is constructed by incorporating one or more expression constructs encoding the enzyme of interest into the plant host genome and propagating the resulting modified plant or plant cell into a transgenic plant or plant cell.<br>
Conveniently, the expression construct is a DNA construct which comprises a gene encoding the enzyme of interest in operable association with appropriate regulatory sequences required for expression of the gene in the plant or plant part of choice. Furthermore, the expression construct may comprise a selectable marker useful for identifying host cells into which the expression construct has been integrated and DNA sequences necessary for introduction of the construct into the plant in question (the latter depends on the DNA introduction method to be used).<br>
The choice of regulatory sequences, such as promoter and terminator sequences and optionally signal or transit sequences is determined, e.g. on the basis of when, where and how the enzyme is desired to be expressed. For instance, the expression of the gene encoding the enzyme of the invention may be constitutive or inducible, or may be developmental, stage or tissue specific, and the gene product may be targeted to a specific cell compartment, tissue or plant part such as seeds or leaves. Regulatory sequences are, e.g. described by Tague et al, Plant, Phys., 86, 506,1988.<br>
For constitutive expression the 35S-CaMV, the maize ubiquitin 1 and the rice actin 1 promoter may be used (Franck et al. 1980. Cell 21: 285-294, Christensen AH, Sharrock RA and Quail 1992. Maize polyubiquitin genes: structure, thermal perturbation of expression and transcript splicing, and promoter activity following transfer to protoplasts by electroporation. Plant Mo. Biol. 18, 675-689.; Zhang W, McElroy D. and Wu R 1991, Analysis of rice Act1 5' region activity in transgenic rice plants. Plant Cell 3, 1155-1165). Organ-specific promoters may, e.g. be a promoter from storage sink tissues such as seeds, potato tubers, and fruits (Edwards &amp; Coruzzi, 1990. Annu. Rev. Genet. 24: 275-303), or from metabolic sink tissues such as meristems (Ito et al., 1994. Plant Mol. Biol. 24: 863-878), a seed specific promoter such as the glutelin, prolamin, globulin or albumin promoter from rice (Wu et al., Plant and Cell Physiology Vol. 39, No. 8 pp. 885-889 (1998)), a V7c/a faba promoter from the legumin B4 and the unknown seed protein gene from Vicia faba described by Conrad U. et al,<br><br>
Journal of Plant Physiology Vol. 152, No. 6 pp. 708-711 (1998), a promoter from a seed oil body protein (Chen et al., Plant and cell physiology vol. 39, No. 9 pp. 935-941 (1998), the storage protein napA promoter from Brassica napus, or any other seed specific promoter known in the art e.g. as described in WO 91/14772. Furthermore, the promoter may be a leaf specific promoter such as the rbcs promoter from rice or tomato (Kyozuka et al., Plant Physiology Vol. 102, No. 3 pp. 991-1000 (1993), the chlorella virus adenine methyttransferase gene promoter (Mitra, A. and Higgins, DW, Plant Molecular Biology Vol. 26, No. 1 pp. 85-93 (1994), or the aldP gene promoter from rice (Kagaya et al., Molecular and General Genetics Vol. 248, No. 6 pp. 668-674 (1995), or a wound inducible promoter such as the potato pin2 promoter (Xu et al, Plant Molecular Biology Vol. 22, No. 4 pp. 573-588 (1993). Likewise, the promoter may inducible by abiotic treatments such as temperature, drought or alterations in salinity or induced by exogenously applied substances that activate the promoter e.g. ethanol, oestrogens, plant hormones like ethylene, abscisic acid and gibberellic acid and heavy metals.<br>
A promoter enhancer element may be used to achieve higher expression of the enzyme in the plant. For instance, the promoter enhancer element may be an intron which is placed between the promoter and the nucleotide sequence encoding the enzyme. For instance, Xu et al. op cit disclose the use of the first intron of the rice actin 1 gene to enhance expression.<br>
The selectable marker gene and any other parts of the expression construct may be chosen from those available in the art.<br>
The DNA construct is incorporated into the plant genome according to conventional techniques known in the art, including Ag/r&gt;&amp;acfe/7£;/7WTiediated transformation, virus-mediated transformation, micro injection, particle bombardment, biolistic transformation, and electroporation (Gasser et al, Science, 244, 1293; Potrykus, Bio/Techn. 8, 535, 1990; Shimamoto et al, Nature, 338,274,1989).<br>
Presently, Agrobacterium tumefaciens mediated gene transfer is the method of choice for generating transgenic dicots (for review Hooykas &amp; Schilperoort, 1992. Plant Mol. Biol. 19: 15-38), and can also be used for transforming monocots, although other transformation methods often are used for these plants. Presently, the method of choice for generating transgenic monocots supplementing the Xgrobacterium approach is particle bombardment (microscopic gold or tungsten particles coated with the transforming DNA) of embryonic calli or developing embryos (Christou, 1992. Plant J. 2: 275-281; Shimamoto, 1994. Curr. Opin. Biotechnol. 5: 158-162; Vasil et al., 1992. Bio/Technology 10: 667-674). An alternative method for transformation of monocots is based on protoplast transformation as described by Omirulleh S, et al., Plant Molecular biology Vol. 21, No. 3 pp. 415-428 (1993).<br><br>
Following transformation, the transformants having incorporated the expression construct are selected and regenerated into whole plants according to methods well-known in the art Often the transformation procedure is designed for the selective elimination of selection genes either during regeneration or in the following generations by using e.g. co-transformation with two separate T-DNA constructs or site specific excision of the selection gene by a specific recombinase.<br>
Starch processing<br>
The hybrid enzyme of the first aspect of the invention or the variant of the second aspect of the invention may in an eighth aspect be used in a process for liquefying starch, wherein a gelatinized or granular starch substrate is treated in aqueous medium with the hybrid enzyme. Preferably the process comprising hydrolysis of a slurry of gelatinized or granular starch, in particular hydrolysis of granular starch into a soluble starch hydrolysate at a temperature below the initial gelatinization temperature of said granular starch.<br>
In a preferred embodiment the starch slurry in addition to being contacted with the hybrid enzyme of the first aspect or the variant of the second aspect of the invention is contacted with an enzyme selected from the list consisting of; a fungal alpha-amyiase (EC 3.2.1.1), a beta-amylase (E.G. 3.2.1.2), and a glucoamylase (E.C.3.2.1.3). In an embodiment further a Termamyl-like alpha-amylase or a debranching enzyme, such as an isoamylase (E.C. 3.2.1.68) or a pullulanases (E.C. 3.2.1.41) is added. In the context of the present invention a Termamyl-like alpha-amylase is an alpha-amylase as defined in WQ99/19467 on page 3, line 18 to page 6, line 27.<br>
The starch slurry to be subjected to the process of the seventh aspect of the invention may have 20-55% dry solids granular starch, preferably 25-40% dry solids granular starch, more preferably 30-35% dry solids granular starch.<br>
After being subjected to the process of the seventh aspect of the invention at least 85%, at least 86%, at least 87%, at least 88%, at least 89%, at least 90%, at least 91%, at least 92%, at least 93%, at least 94%, at least 95%, at least 96%, at least 97%, at least 98%, or preferably at least 99% of the dry solids of the granular starch is converted into a soluble stanch hydrolysate.<br>
According to the invention the process of the seventh aspect is conducted at a temperature below the initial gelatinization temperature. Preferably the temperature at which the processes are conducted is at least 30°C, at least 31 °C, at least 32°C, at least 33°C, at least 34°C, at least 35CC, at least 36°C, at least 37°C, at least 38°C, at least 39°C, at least 40°C, at least 41 °C, at least 42°C, at least 43°C, at least 44°C, at least 45DC, at least 46°C, at least 47°C, at least 48°C, at least 49°C, at least 50°C, at least 51 °C, at least 52°C, at least<br><br>
53°C, at least 54°C, at least 55°C, at least 56°C, at least 57°C, at least 58°C, at least 59°C, or preferably at least 60°C.<br>
The pH at which the process of the seventh aspect of the invention is conducted may in be in the range of 3.0 to 7.0, preferably from 3.5 to 6.0, or more preferably from 4.0-5.0.<br>
The granular starch to be processed in the process of the invention may in particular be obtained from tubers, roots, stems, legumes, cereals or whole grain. More specifically the granular starch may be obtained from corns, cobs, wheat, barley, rye, milo, sago, cassava, tapioca, sorghum, rice, peas, bean, banana or potatoes. Specially contemplated are both waxy and non-waxy types of com and barley. The granular starch to be processed may be a highly refined starch quality, preferably at least 90%, at least 95%, at least 97% or at least 99.5 % pure or it may be a more crude starch containing material comprising milled whole grain including non-starch fractions such as germ residues and fibres. The raw material, such as whole grain, is milled in order to open up the structure and allowing for further processing. Two milling processes are preferred according to the invention: wet and dry milling. In dry milling the whole kernel is milled and used. Wet milling gives a good separation of germ and meal (starch granules and protein) and is with a few exceptions applied at locations where the starch hydrolysate is used in production of syrups. Both dry and wet milling is well known in the art of starch processing and are equally contemplated for the process of the invention. Also corn grits, and preferably milled corn grits may be applied.<br>
In an embodiment of the process of the seventh aspect of the invention the hybrid enzyme is used in a process for production of fuel or potable ethanol comprising contacting the treated starch with a yeast. Preferably the process comprises fermentation with a yeast carried out simultaneously or separately/sequential to the hydrolysis of the granular starch slurry. When the fermentation is performed simultaneous to the hydrolysis the temperature is preferably between 30*0 and 35°C, and more preferably between 31 °C and 34°C. In another embodiment the granular starch slurry is being contacted with a polypeptide comprising a CBM, but no catalytic module, i.e. application of loose CBMs. The loose CBMs may be starch binding modules, cellulose-binding modules, chitin-binding modules, xylan-binding modules, mannan-binding modules, and other binding modules. Preferred CBMs in the present context are microbial CBMs, particularly bacterial or fungal CBMs. Particularly preferred are the starch binding modules disclosed in Danish patent application PA 2003 01568 as the polypeptide sequences SEQ ID NO:1, SEQ ID NO:2, and SEQ ID NO:3 or the starch binding modules shown in the present disclosure as SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15; SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18; SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23. Most preferred CBMs include the CBMs disclosed in the present disclosure as SEQ ID NO:24 and as SEQ ID NO:25. Also preferred for the invention is the application of any CBM having at least 50%, 60%, 70%, 80% or even at least 90% homology to any of the afore mentioned CBM amino<br><br>
acid sequences. The loose CBMs may be applied to the granular starch slurry in effective amounts.<br>
The glucose may also be fermented in to other fermentation products, such as citric acid, itaconic acid, lactic acid, gluconic acid; ketones; amino acids, such as glutamic acid (sodium monoglutaminate), but also more complex compounds such as antibiotics, such as penicillin, tetracyclic enzymes; vitamins, such as riboflavin, B12, beta-carotene; hormones, which are difficult to produce synthetically.<br>
Dough-based products<br>
The hybrid enzyme may be used for the preparation of a dough-based edible product, particularly a hybrid enzyme comprising an amyiolytic module derived from Aspergillus oryzae such as the amino acid sequence shown in SEQ ID NO: 30. The hybrid enzyme used for the preparation of a dough-based edible product is in particular a hybrid enzyme comprising the sequence shown as SEQ ID NO: 30 and/or as amino acids 21-498 of SEQ ID NO: 30 or any variant thereof such as a variant comprising one or more substitutions selected from the fist consisting of: Q81R, K158D, K158V, S161D, S161N, Q163S, Q163A, D164S, Y175W, E176D, D177N, N264K, N264E, M266L, G466D, D468S, and N470D. The hybrid enzyme used for the preparation of a dough-based edible product is in particular a hybrid enzyme comprising an amino acid sequence which has at least 50% homology, preferably at least 60%, 70%, 80%, 85% or at least 90%, e.g. at least 95%, 97%, 98%, or at least 99%, such as 100% homology to the sequences set forth in SEQ ID NO: 30 herein.<br>
The dough generally comprises flour (particularly wheat flour) and water. The dough is leavened e.g. by adding chemical leavening agents or yeast, usually Saccharomyces cerevisiae (baker's yeast).<br>
The dough-based product is made by leavening and heating the dough, e.g. by baking or steaming. Examples are steamed or baked bread (in particular white, whole-meal or rye bread), typically in the form of loaves or rolls,<br>
The dough may comprise one or more additional enzymes, e.g. a second amylase (e.g. a maltogenic alpha-amylase), a cyclodextrin glucanotransferase, a protease or peptidase, in particular an exopeptidase, a transglutaminase, a lipase, a phospholipase, a celiulase, a hemiceliulase (e.g. a pentopsanase or xyianase), a glycosyltransferase, a branching enzyme or an oxidase such as glucose oxidase or an oxidase with higher activity on maltose than on glucose.<br>
The hybrid enzyme may be used at a lower dosage than the catalytic module with alpha-amylase activity used alone (compared on a weight basis).<br><br>
MATERIALS AND METHODS<br>
Purchased materia!<br>
Enzymes for DNA manipulations (e.g. restriction endonucleases, ligases etc.) are obtainable from New England Biolabs, inc. and were used according to the manufacturer's instructions. Amplified plasmids were recovered with Qiagen® Plasmid Kit (Qiagen). Polymerase Chain Reaction (PCR) was carried out with Expand™ PCR system (Roche). QIAquick™ Gel Extraction Kit (Qiagen) was used for purification of PCR fragments and DNA fragments excised from agarose gels.<br>
Strains and plasmids<br>
Aspergillus kawachii strain 1FO4308 was used as donor of CBM and linker sequences. Aspergillus niger strain DSM 2761 donated the amylolytic sequence (WO89/01969). Aspergillus niger strain MBin120 described in US provisional patent application No. 60/459902 (10345.000-US) was used as host strain. The Aspergillus expression plasmid pMT2188 described in the patent WO200295014 was used as vector. The pyrF defective Escherichia coli strain DB6507 (ATCC 35673) was used for propagation of pHUda381 and pHUda387<br>
Plasmid pMT2188 consists of an expression cassette based on the Aspergillus niger neutral amylase II promoter fused to the Aspergillus niduians triose phosphate isomerase non translated leader sequence (Pna2/tpi) and the Aspergillus niger amyloglycosidase terminator (Tamg). Also present on the plasmid is the Aspergillus selective marker amdS from Aspergillus niduians enabling growth on acetamide as sole nitrogen source and the URA3 marker from Saccharomyces cerevisiae enabling growth of the pyrF defective Escherichia coli strain DB6507 (ATCC 35673), which propagates pHUda381 and pHUda387. Transformation into E. coli DB6507 using the S. cerevisiae URA 3 gene as selective marker was done in the following way:<br>
£ coli DB6507 was made competent by the method of Mandel and Higa (Mandel, M. and A. Higa (1970) J. Mol. Biol. 45,154). Transformants were selected on solid M9 medium (Sambrook et al (1989) Molecular cloning, a laboratory manual, 2. edition, Cold Spring Harbor Laboratory Press) supplemented with 1 g/l casaminoacids, 500 \xgl\ thiamine and 10 mg/l kanamycin.<br><br>
Media and substrates<br>
Cove was composed of 342.3 g/L Sucrose, 20 ml/L COVE salt solution, 10mM Acetamide and 30 g/L noble agar. Cove-2 was composed of 30 g/L Sucrose, 20 ml/L COVE salt solution, 10mM, Acetamide and 30 g/L noble agar. COVE salt solution was composed of 26 g/L KCI, 26 g/L MgS04-7H20, 76 g/L KH2P04 and 50ml/L Cove trace metals. COVE trace metals was composed of 0.04 g/L Na2B4O7-10 H20, 0.4 g/L CuS04-5H20, 1.2 g/L FeS04-7H20, 1.0 g/L MnS04-H20, 0.8 g/L Na2Mo02-2H20 and 10 g/L ZnS047H20. YPG was composed of 4 g/L yeast extract, 1 g/L K2HP04, 0.5 g/L MgS04-7H20 and 15 g/L glucose, pH 6.0. STC was composed of 0.8 M Sorbitol, 25 mM Tris pH 8 and 25 mM CaCI2. STPC was composed of 40 % PEG4000 in STC buffer. Cove top agarose was composed of 342.3 g/L Sucrose, 20 ml/L COVE salt solution, 10mM Acetamide and 10 g/L low melt agarose. MLC was composed of 40 g/L Glucose, 50 g/L Soybean powder, 4 g/L Citric acid, pH 5.0. GO-50 was composed of glucose 50 g/L, KH2P04 2 g/L, MgS04-7aq 2 g/L, K2S04 3 g, citric acid 3 g/L, oxalic acid 50 g/L, MAG trace metal solution 0.5 m/L and urea 3 g/L, pH 5.0. AMG trace metal solution was composed of 6.8 g/L ZnCI27H20, 2.5 g/L CuS04-5H20, 0.24 g/L NiCI2-6H20,13.9 g/L FeS04-7H20,13.5 g/L MnS04-H20 and 3 g/L citric acid.<br>
Acid stable alpha-amylase activity<br>
When used according to the present invention the activity of any acid stable alpha-amylase may be measured in AFAU (Acid Fungal Alpha-amylase Units), which are determined relative to an enzyme standard. 1 FAU is defined as the amount of enzyme which degrades 5.260 mg starch dry matter per hour under the below mentioned standard conditions.<br>
Acid stable alpha-amylase, an endo-alpha-amylase (1,4-alpha-D-glucan-glucano-hydrolase, E.C. 3.2.1.1) hydrolyzes alpha-1,4-glucosidic bonds in the inner regions of the starch molecule to form dextrins and oligosaccharides with different chain lengths. The intensity of color formed with iodine is directly proportional to the concentration of starch. Amylase activity is determined using reverse colorimetry as a reduction in the concentration of starch under the specified analytical conditions.<br><br><br>
Standard conditions/reaction conditions:<br>
Substrate:	Soluble starch, approx. 0.17 g/L<br>
Buffer.	Citrate, approx. 0.03 M<br>
Iodine (!2):	0.03 g/L<br>
CaC!2:	1.85 mM<br>
pH:	2.50 ± 0.05<br>
Incubation temperature:	40°C<br>
Reaction time:	23 seconds<br>
Wavelength:	590nm<br>
Enzyme concentration:	0.025 AFAU/mL<br>
Enzyme working range:	0.01-0.04 AFAU/mL<br>
A folder EB-SM-O259.02/01 describing this analytical method in more detail is available upon request to Novozymes A/S, Denmark, which folder is hereby included by reference.<br>
Glucoamylase activity<br>
Glucoamylase activity may be measured in AmyloGlucosidase Units (AGU). The AGU is defined as the amount of enzyme, which hydrolyzes 1 micromole maltose per minute under the standard conditions 37°Ct pH 4,3, substrate: maltose 23.2 mM, buffer acetate 0.1 M, reaction time 5 minutes.<br>
An autoanalyzer system may be used. Mutarotase is added to the glucose dehydrogenase reagent so that any alpha-D-glucose present is turned into beta-D-glucose. Glucose dehydrogenase reacts specifically with beta-D-glucose in the reaction mentioned above, forming NADH which is determined using a photometer at 340 nm as a measure of the original glucose concentration.<br><br>
AMG incubation:<br>
Substrate:	maltose 23.2 mM<br>
Buffer	acetate 0.1 M<br>
pH:	4.30 ± 0.05<br>
incubation	37°C ± 1<br>
temperature:<br>
Reaction time:	5 minutes<br>
Enzyme working range:	0.5-4.0 AGU/mL<br>
Color reaction:<br>
GlucDH:	430 U/L<br>
Mutarotase:	9 U/L<br>
NAD:	0.21 mM<br>
Buffer	phosphate 0.12 M; 0.15 M NaCI<br>
pH:	7.60 ± 0.05<br>
Incubation	37°C ± 1<br>
temperature:<br>
Reaction time:	5 minutes<br>
Wavelength:	340 nm<br>
A folder (EB-SM-0131.02/01) describing this analytical method in more detail is available on request from Novozymes A/S, Denmark, which folder is hereby included by reference.<br>
DNA manipulations<br>
Unless otherwise stated, DNA manipulations and transformations were performed using standard methods of molecular biology as described in Sambrook et al. (1989) Molecular cloning: A laboratory manual, Cold Spring Harbor lab. Cold Spring Harbor, NY; Ausubel, F. M. et al. (eds.) "Current protocols in Molecular Biology1', John Wiley and Sons, 1995; Harwood, C. R. and Cutting, S. M. (eds.)<br>
Example 1: Cloning of carbohydrate binding module (CBM) from Aspergillus kawachii<br>
In order to clone the carbohydrate binding module (CBM) with linker from A.kawachii, the primers CBM1 (SEQ ID NO:1) and CBM2 (SEQ ID NO:2) were designed based on the nucleotide sequences of Aspergillus kawachii acid stable alpha-amylase in the EMBL<br><br>
database (EMBL:#AB008370). CBM1 and CBM2 comprise a BamHI site and a Sail site, respectively.<br>
CBM1: ff- gaagggatccgatttttactagtacatccaaagccaccac- 3' CBM2:5'- tttgtcgacctacctccacgtatcaaccaccgtctcc- 3'<br>
PCR reaction was carried out with the primers CBM1 and CBM2 using genomic DNA from A. kawachii (IFO4308) as template. Reaction components (1 ng /microL of genomic DNA, 250 mM dNTP each, primer 250 nM each, 0.1 U/ microL in Taq polymerase in 1X buffer (Roche Diagnostics, Japan)) were mixed and submitted for PCR under the following<br>
conditions.<br>
Step	Temperature	Time<br>
1	94°C	2 min<br>
2	92°C	1 min<br>
3	55°C	1 min<br>
4	72°C	1 min<br>
5	72°C	10 min<br>
6	4°C	forever Steps 2 to 4 were repeated 30 times.<br>
A 0.4 kb fragment comprising the CBM with linker region was amplified. The amplified DNA was cut by Sal i and BamH I and ligated into pMT2188 digested by Xho I and BamH I to create pHUda381 having CBM with linker region from Aspergillus kawachii, Aspergillus niger neutral amylase promoter, Aspergillus nidulans TPI leader sequences, Aspergillus niger glucoamylase terminator and Aspergillus nidulans amdS gene as a marker. The pHUda381 was sequenced to confirm presence of correct CBM with linker sequence. The CBM with linker sequence is shown in SEQ ID NO:5.<br>
Example 2: Expression of the hybrid enzyme in Aspergillus niger<br>
The production of the cDNA sequence of Aspergillus niger acid stable alpha-amylase gene and the cDNA clone of Aspergillus niger acid stable alpha-amylase are described in WO 8901969 (example 1 and 3). A PCR reaction with cDNA clone of Aspergillus niger acid stable alpha-amylase as template was performed using the primers (SEQ ID NO:3) and (SEQ ID NO:4) to introduce a BamHI site and a Spei site, respectively.<br>
(SEQ ID NO:3): 5'- tttggatccaccatgagattatcgacttcgagtctcttc- 3'<br><br>
{SEQ ID N0:4): ff- tttactagtagcagcagcagttgtggtcgtggttgttc- 3'<br>
Reaction components (1 ng /microL of template DNA, 250 mM dNTP each, primer 250 nM each, 0.1 U/ microL in Taq polymerase in 1X buffer (Roche Diagnostics, Japan)) were mixed and submitted for PCR under the following conditions.<br>
Step	Temperature	Time<br>
1	94°C	2 min<br>
2	92°C	1 min<br>
3	55°C	1 min<br>
4	72°C	2 min<br>
5	72°C	10 min<br>
6	4°C	forever Steps 2 to 4 were repeated 30 times.<br>
The 1.5 kb amplified DNA fragment was cut by Spe I and BamH I and ligated into pHUda381 digested by Spe I and BamH I to create the expression piasmid pHUda387 comprising Aspergillus niger acid stable alpha-amylase cDNA fused with CBM from Aspergillus kawachii. The pHUda387 was sequenced to confirm that no changes had happen in the Aspergillus niger acid stable alpha-amylase cDNA sequences. The Aspergillus niger acid stable alpha-amylase cDNA sequence is shown in SEQ ID NO;7.<br>
The pHUda387 was transformed into Aspergillus niger MBin120. The host strain was propagated in 100 ml of non-selective YPG medium at 32 °C for 16 hrs on a rotary shaker at 120 rpm. Cells were collected by filtering, washed with 0.6 M KCI and resuspended in 20 ml of 0.6 M KCI containing a commercial beta-glucanase product (GLUCANEX™, Novozymes A/S) at final concentration of 600 microL /ml. The suspension was incubated at 32 °C at 80 rpm until protoplasts were formed, then washed twice with STC buffer. The protoplasts were counted with a hematometer and resuspended and adjusted in an 8:2:0.1 solution of STC:STPC:DMSO to a final concentration of 2.5x107 protoplasts/ml. About 3 microgram of piasmid DNA was added to 100 microL of protoplast suspension, mixed gently and incubated on ice for 20 min. One ml of SPTC was added and the protoplast suspension was incubated for 30 min at 37 °C. After the addition of 10 ml of 50 °C Cove top agarose, the reaction was poured onto Cove agar plates and the plates were incubated at 32 °C. After 5 days transformants were selected from the Cove medium.<br>
The selected transformants were inoculated in 100 ml of MLC media and cultivated at 30 °C for 2 days. 10 ml of MLC medium was inoculated to 100 ml of GO-50 medium and cultivated at 30 °C for 5 days. The supernatant was obtained by centrifugation.<br><br>
The acid stable alpha-amyiase activity in the supernatant was determined as decrease of btue color of starch-iodine complex measured in OD590nm. 25 microL of enzyme samples dissolved in sample buffer (51.4 mM calcium chloride in 2 mM citrate buffer [pH 2.51) was mixed with 135 microL of substrate solution (0.6 g/L of starch [Merck 1253] and 12 g/L of sodium acetate in 100 mM citrate buffer [pH 2.5]) and incubated at 37 °C for 325 sec. After 325 sec, 90 microL of iodine solution (1.2 g/L of potassium iodine [Merck 5043] and 0.12 g/L of iodine [Merck 4761]) was added to the reaction mixture and incubated at 37 °C for 25 sec. Activity was measured at 590 nm on a spectrophotometer. 25 microL of distilled water was used instead of enzyme samples in blank experiments.<br>
Example 3: Performance of the hybrid enzyme in SSF of non-gelatinized starch<br>
The relative performance of Aspergillus niger acid stable alpha-amylase and Aspergillus niger acid stable alpha-amylase with an attached carbohydrate-binding module was evaluated via mini-scale SSF (Simultaneous Saccharification and Fermentation) fermentations. Dosages used were 0.3, 0.5 and 1.0 AFAU/g DS of the respective alpha-amylase complemented with a 0.5 AGU/g DS dose of purified Aspergillus niger glucoamylase. Briefly, approximately 1.9 g of ground com (yellow #2 dent corn ground in a pilot scale hammer mill and passed through a 2 mm screen, 11.2% moisture content) was added to 16 ml polystyrene tubes (Falcon 352025). A solution of 0.02 N H2S04 with 3 mg/ml penicillin was added in an amount appropriate to bring the dry solids level (DS) to 34% and the pH to 5.0. Treatments were made in six replicates.<br>
After dosing the enzymes, the tubes were inoculated with 4.74 x 108 yeast cells/ml. Tubes were capped with a screw on lid which had been punctured with a very small needle to allow gas release and vortexed briefly before weighing and incubation at 32°C. Fermentation progress was followed by weighing the tubes over time. Tubes were vortexed briefly before weighing. The relationship used between amount of C02 loss and the weight of ethanol was: C02 loss (g) x 1.045 = EtOH (g). The results are shown in table 2.<br><br>
On an equal activity basis, Aspergillus niger acid stable alpha-amylase with an attached carbohydrate-binding module (hybrid) significantly outperformed Aspergillus niger<br><br>
acid stable aipha-amylase (native) at al! dosages tested. The hybrid enzyme was approximately three times more effective than the native enzyme (i.e. performance of Aspergillus niger acid stable alpha-amylase could be matched with about three times less Aspergillus niger acid stable aipha-amylase with an attached carbohydrate-binding module).<br>
Example 4: Performance of hybrid enzymes with different combinations of CBM, linker sequence and catalytic module<br>
Hybrid enzyme variants comprising the Aspergillus niger acid stable alpha-amylase (AA) and a CBM from A. kawachii alpha-amylase, A. niger AMG, T. emersonii AMG, Athelia rolfsii AMG, or Bacillus sp. maltogenic alpha-amylase (Novamyl) were constructed. The linker sequence from A. kawachii alpha-amylase (SEQ ID NO:27) was used in al! the variants.<br>
The performance of the variants was assessed using com starch (SIGMA S-9679) as substrate, Aspergillus niger glucoamylase G2 0.5 AGU/g DS, variant 1.0 AFAU/g DS and quantification of the liberated glucose with the Glucose B-test kit (Wako Pure Chemicals 271-3141). The results are shown in table 3.<br><br>
Hybrid enzyme variants comprising the Aspergillus niger acid stable alpha-amylase and a CBM from A. kawachii alpha-amylase, A. niger AMG, or Athelia rolfsii AMG, were constructed. Also variants with different linker sequence were constructed. The linker sequences used were A. niger AMG linker (SEQ ID NO:26), A. kawachii alpha-amylase linker (SEQ ID NO:27), Athelia rolfsii AMG linker (SEQ ID NO:28), and the PEPT linker (SEQ ID NO:29). A variant having two CBDs in tandem was also constructed (JA012).<br>
The performance of the variants was assessed as above, only with Aspergillus niger glucoamylase G1 and the variant was dosed as 0.3 AFAU/g DS. The results are shown in table 4.<br><br><br>
Hybrid enzyme variants comprising the linker and CBM from A. kawachii alpha-amylase with different catalytic modules were constructed. The catalytic modules applied were from Aspergillus niger acid stable alpha-amylase, A. kawachii alpha-amyiase or Aspergillus oryzae alpha-amylase (Fungamyl™). The performance of the variants was assessed as above, with Aspergillus niger glucoamylase G1 and the variant dosed as 0.3 AFAU/g DS. The results are shown in table 5.<br><br>
Hybrid enzyme variants comprising the linker and CBM from A. kawachii alpha-amylase or A. rolfsii AMG with different catalytic modules from Aspergillus niger acid stable alpha-amylase or Aspergillus oryzae alpha-amylase (Fungamyl™). The performance of the variants was assessed as above with Aspergillus niger glucoamylase G1 and the variant dosed as 0.3 AFAU/g DS. The results are shown in table 6.<br><br><br><br>
JA085 showed at least by 5 °C higher thermostability than JA017 at pH 4.5 was and stable even at pH 3.0.<br>
The new variants JA085 and JA074 showed a high specific activity in the glucose release test JA085 continued to release glucose even at pH 3.8 where JA017 was quickly inactivated. JA074 had the highest glucose yields at pH 4.0 was but was inactivated at pH 3.8.<br>
Tables 8-10 list the temperature stability as residual activity after 30 min incubation at 55°C and 60°C at pH 4.5, the specific activity, the pH stability and the activity in the glucose releasing test of a number of variants.<br><br><br><br>
Example 6: Variants of the Aspergillus kawachii acid alpha-amylase<br>
A variant of the Aspergillus kawachii acid alpha-amylase suitable for raw starch hydrolysis may be produced by conventional protein engineered of the sequence comprising the catalytic module. Specific position wherein alterations can improve specific activity and/or stability was predicted based on similarity to one of the following fungal alpha-amylases having the indicated amino acid sequence and a three-dimensional structure found under the indicated identifier in the RCSB Protein Data Bank (www.rcsb.ora): the A niger acid alpha-amylase (2aaa, SEQ ID NO:42 herein) and the alpha-amylase (Taka amylase) from Aspergillus oryzae (6taa or 7taa, SEQ ID NO:43 herein). The two 3D models were superimposed by aligning the amino acid residues of each catalytic triad. This was done by methods known in the art based on the deviations of the three pairs of C-aipha atoms, e.g. by minimizing the sum of squares of the three deviations or by aligning so as to keep each deviation below 0.8 A, e.g. below 0.6 A, below 0.4 A, below 0.3 A or below 0.2 A.<br><br>
Alternatively, the superimposition may be done by aligning the two amino acid sequences by a conventional method and minimizing the sum of squares of the deviations of all corresponding pairs of amino acid residues. The sequence alignment may be done by conventional methods, e.g. by use the software GAP from UWGCG Version 8.<br>
On the alignment of the three alpha-amylases from A kawachii, A niger (aaajiew) and A. oryzae (7taa) shown below are indicated the residues which are within 10A from the acarbose substrate in 7taa.pdb structure. These residues are targets for alterations conferring improved specific activity.<br><br><br><br>
The specific alterations for increased specific activity is one substitution or a combination of substitutions in the following positions within the "10A distance of the substrate": 13,15,18,32-36 (i.e. 32, 33, 34, 35, 36), 61,63-64, 68-69,73-84 (e.g. 73, 74, 75,<br>
76,	77, 78, 79, 80, 81, 82, 83, 84),117-125 (e.g. 117, 118, 119, 120, 121, 122, 123, 124, 125),152-158 (e.g. 152,153, 154,155,156, 157,158),161-162, 165-175 (e.g.165,166, 167, 168, 169, 170, 171, 172, 173, 174, 175), 204-211 (e.g. 204, 205, 206, 207, 208, 209, 210, 211), 216,229-239 (229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239), 242, 250, 252-253, 255-257 (e.g. 255, 256, 257)]259-260, 275, 292, 295-299 (e.g. 295, 296, 297, 298, 299), 304, 328, 339-344 according to the 7taa numbering in the alignment above where the N-termimal is ATPAD in 7taa (TAKA amylase). Most interesting positions are: 33, 36, 74, 75,<br>
77,	120, 153, 154, 155, 156, 157, 158, 162, 166, 169, 170, 199, 232, 233, 235, 238, 239, 256, 257, 331, 336, 339, 340, and 342. The substitutions may be to any amino acid residue.<br>
Variants comprising one of the following substitutions or combinations thereof may be produced: G33A, I36K, S74A, D75Y, E77D, P120A, I153D, D154N, W155Y, D156E, N157D, L158Q, Q162E, E166L, T169N, I170T, E199K, E199L, D232L, N233D, N235D, L238Y, D239T, W256Y, Q257P, E331Q, S336A, D339K, D339N, V340D, and Y342A. The most . interesting variants comprise one or more of the following substitutions; S74A, E166L, E199L, D339K, and D156E. Yet more interesting is the variant having the multiple substitutions S74A/E166L/E199L<br>
Based on similarity between Aspergillus kawachii and the A. niger acid alpha-amylases the following substitutions is predicted to improve the specific activity; N31D, S74A, Y89D, E209L, Y245V, D348K, D378A, K383A, P386A, I387F, I405V, N448S, and N480R, and a variant named WA01 comprising these substitutions can be produced.<br><br>
xample 7: Performance of a variant of the Aspergillus kawachii acid alpha-amyiase<br>
The sequences of the hybrids JA007 and JA001 described in example 4 are identical ) respectively the sequence of the A. kawachii acid alpha-amylase (SEQ ID N0:41) and the equence of the variant WA01 described in the previous example. As the two hybrids JA001 ind JA007 have been compared in several tests the improvement in specific activity onferred by the substitutions to the variant WA01 can be very accurately predicted. Specific ictivity of the variant WA01 will be 1.7 to 1.8 times higher than the specific activity of the A. tawachiiwM type amylase. The prediction is based on the data shown in table 12.<br><br>
Example 8: Raw starch hydrolysis with an acid fungal alpha-amylase with or without CBD<br>
This example illustrates the conversion of granular wheat starch or com grits into glucose using acid fungal amylase without a CBM (SEQ ID NO:8) or the corresponding hybrid with a CBM (JA001). A sluny with 33% dry solids (DS) granular starch was prepared by adding 247.5 g of wheat starch or maize grits under stirring to 502.5 ml of water. The pH was adjusted with HCl to 4.5. The granular starch slurry was distributed to 100 ml blue cap flasks with 75 g in each flask. The flasks were incubated with magnetic stirring in a 60°C water bath. At zero hours the enzyme activities were dosed to the flasks; glucoamylase (200 AGU/kg DS), an acid fungal amylase (50 AFAU/kg DS) and the wild type Aspergillus niger acid alpha-amylase (SEQ ID NO:8) or the Aspergillus niger acid alpha-amylase but fused to a linker and CBM derived from A. kawachii (SEQ ID NO:8 fused to SEQ ID NO:6). The Aspergillus niger acid alpha-amylase w/wo CBM was dosed as 100 KNU/kg DS. Samples were withdrawn after 24,48,72, and 96 hours.<br>
Total dry solids starch was determined using the following method. The starch was completely hydrolyzed by adding an excess amount of alpha-amylase (300 KNU/Kg dry solids) and placing the sample in an oil bath at 95 °C for 45 minutes. Subsequently the samples were cooled to 60°C and an excess amount of glucoamylase (600 AGU/kg DS) was added followed by incubatbn for 2 hours at 606C.<br><br><br><br>
CLAIMS<br>
1)	A hybrid enzyme which comprises an amino acid sequence of a catalytic module and an<br>
amino acid sequence of a carbohydrate binding module,<br>
a)	wherein the catalytic module is an amino acid sequence which has at least 70% homology to the amino acid sequence of SEQ ID NO:8 or an amino acid sequence which has at least 70% homology to the amino acid sequence of SEQ ID NO:30;<br>
b)	wherein the carbohydrate binding module is an amino acid sequence having at least 50% homology to an amino acid sequence selected from the group consisting of SEQ ID NO;9, SEQ ID NO:10, SEQ ID N0:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16f SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25.<br><br>
2)	The hybrid enzyme of claim 1, wherein the hybrid enzyme further comprises a linker amino acid sequence which connects the amino acid sequence of the catalytic module and the amino add sequence of the carbohydrate-binding module.<br>
3)	The hybrid enzyme of claim 2, wherein the linker sequence is a linker sequence selected from the group consisting of the linker sequence of SEQ ID NO:26, SEQ ID NO:27, SEQ ID NO:28, and SEQ ID NO:29 or a linker sequence which differs from the amino acid sequence shown in SEQ ID NO:26, SEQ ID NO:27, SEQ ID NO:28, or SEQ ID NO:29 by no more than 10 positions, no more than 9 positions, no more than 8 positions, no more than 7 positions, no more than 6 positions, no more than 5 positions, no more than 4 positions, no more than 3 positions, no more than 2 positions, or no more than 1 position.<br>
4)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 70% homology to the amino acid sequence of SEQ ID NO:8.<br>
5)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 80% homology to the amino acid sequence of SEQ ID NO:8.<br>
6)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 90% homology to the amino acid sequence of SEQ ID NO:8.<br><br>
7)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 95% homology to the amino acid sequence of SEQ ID NO:8.<br>
8)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 70% homology to the amino acid sequence of SEQ ID NO:30.<br>
9)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 80% homology to the amino acid sequence of SEQ ID NO:30.<br>
10)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 90% homology to the amino acid sequence of SEQ ID NO:30.<br>
11)	The hybrid enzyme of claim 1, wherein the catalytic module is an amino acid sequence which has at least 95% homology to the amino acid sequence of SEQ ID NO:30.<br>
12)	The hybrid enzyme of claim 1, wherein the carbohydrate binding module is an amino acid sequence having at least 60% homology to any of the amino acid sequences shown in SEQ ID NO:9, SEQ ID NO;10, SEQ ID NO:11, SEQ ID NO:12t SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17? SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25<br>
13)	The hybrid enzyme of claim 1, wherein the carbohydrate binding module is an amino acid sequence having at least 70% homology to any of the amino acid sequences shown in SEQ ID NO:9, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO;20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25<br>
14)	The hybrid enzyme of claim 1, wherein the carbohydrate binding module is an amino acid sequence having at least 80% homology to any of the amino acid sequences shown in SEQ ID NO;9, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25<br>
15)	The hybrid enzyme of claim 1, wherein the carbohydrate binding module is an amino<br><br>
acid sequence having at least 90% homology to any of the amino acid sequences shown in SEQ ID NO:9, SEQ ID NO:10, SEQ ID NO:11, SEQ ID NO:12, SEQ ID IMO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID UO-20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23, SEQ ID NO:24 or SEQ ID NO:25<br>
16)	The hybrid enzyme comprising the catalytic domain of the A. oryzae alpha amylase shown as amino acids 21-498 of the sequence SEQ ID NO:40 and an amino acid sequence having at least 70% homology amino acid sequence shown as amino acids 21-498 of the sequence SEQ ID NO:40.<br>
17)	A variant alpha-amylase having at least 80% homology to SEQ ID IMO:41, which variant comprises an alteration at one or more of the positions selected fro the group consisting of: 13,15,18, 31, 32, 33, 34, 35, 36, 61, 63, 64, 68, 69, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 89, 117, 118, 119, 120, 121, 122, 123, 124, 125, 152, 153, 154, 155, 156, 157,158,161,162,165,166,167,168,169,170,171,172,173,174,175, 204, 205, 206, 207, 208, 209, 210, 211, 216, 229, 230, 231,f232, 233, 234, 235, 236, 237, 238, 239, 242, 245, 250, 252, 253, 255, 256, 257,259, 260, 275, 292, 295, 296, 297, 298, 299, 304, 328, 339, 344, 348,378,383, 386, 387,405,448 and 480 wherein;<br>
a)	the alteration(s) are independently,<br>
i)   an insertion of an amino acid downstream of the amino acid which occupies the position,<br>
ii)  a deletion of the amino acid which occupies the position, or<br>
iii) a substitution of the amino acid which occupies the position with a different amino acid,<br>
b)	the variant has aipha-amylase activity.<br>
18)	The variant according to claim 17, comprising one or more of the following substitutions: G33A, I36K, S74A, D75Y, E77D, P120A, I153D, D154N, W155Y, D156E, N157D, L158Q, Q162E, E166L, T169N, 1170T, E199K, E199L, D232L, N233D, N235D, L238Y, D239T, W256Y, Q257P, E331Q, S336A, D339K, D339N, V340D, and Y342A.<br>
19)	The variant according to claim 17, comprising an alteration at one or more of the following positions: 31, 74, 89, 209, 245, 348, 378, 383, 386, 387,405,448 and 480.<br>
20)	The variant according to claim 17, comprising one or more of the following substitutions:<br><br>
N31D, S74A, Y89D, E209L, Y245V, D348K, D378A, K383A, P386A, I387F, I405V, N448S, N480R.<br>
21)	The variant according to claim 17? comprising the following substitutions: N31D, S74A, Y89D, E209L, Y245V, D348K, D378A, K383A, P386A, I387F, I405V, N448S, N480R.<br>
22)	An isolated DNA sequence encoding the hybrid enzyme or the variant according to any of the preceding claims.<br>
23)	A DNA construct comprising the DNA sequence according to claim 22.<br>
24)	An expression vector comprising the DNA sequence according to claim 22.<br>
25)	A host cell transformed with a vector according to claim 24 which host cell is capable of expressing the isolated DNA sequence of claim 18.<br>
26)	The host cell according to claim 25, which host cell is from a fungus, a bacteria, a plant or a mammal.<br>
27)	A method for liquefying starch, wherein a gelatinized or granular starch substrate is treated in aqueous medium with the hybrid enzyme or the variant according to any of claims 1-21.<br>
28)	The method according to claim 27 comprising contacting the treated starch with a yeast to produce fuel or potable ethanol.<br>
29)	The method according to any of claims 27-28 comprising fermenting the treated starch into a fermentation product, such as citric acid, monosodium glutamate, gluconic acid, sodium gluconate, calcium gluconate, potassium gluconate, glucono delta lactone, sodium erythorbate, rtaconic acid, lactic acid, gluconic acid; ketones; amino acids, glutamic acid (sodium monoglutaminate), penicillin, tetracyclic enzymes; vitamins, such as riboflavin, B12, beta-carotene or hormones.<br>
30)	The method according to any of claims 27-28 wherein the starch slurry is contacted with a polypeptide comprising a carbohydrate binding modcule, but no catalytic module.<br>
31)	A process for preparing a dough-based product, comprising adding the hybrid enzyme or<br><br>
the variant of any of claims 1-21 to a dough.<br>
32)	A hybrid enzyme comprising the catalytic domain of the A oryzae alpha amylase shown as amino acids 21-498 of the sequence SEQ ID NO:40 or a sequence having at least 50 % homology amino acid sequence shown as amino acids 21-498 of the sequence SEQ ID NO:40, and a carbohydrate binding domain.<br>
33)	The hybrid enzyme according to claim 32 comprising a substitution in one or more positions selected from the list comprising: of: 81, 158, 161, 163, 164, 175, 176, 177, 264, 266,466,468, and 470.<br>
34)	The hybrid enzyme according to claim 32 comprising one or more substitutions selected from the list consisting of: Q81R, K158D, K158V, S161D, S161N, Q163S, Q163A, D164S, Y175W, E176D, D177N, N264K, N264E, M266L, G466D, D468S, and N470D.<br>
35)	The hybrid enzyme according claim 32 comprising one or more substitutions selected from the list consisting of: K158V S161N Q163A D164S N264K M266L G466D D468S N470D<br>
36)	The hybrid enzyme according to claim 32 comprising one or more substitutions selected from the fist consisting of: Q81R K158V S161N Q163A D164S Y175W E176D D177N N?R4K M2R6I 0466D D468S N470D.<br>
7<br>
  *<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgIEFNRU5ERUQgQ0xBSU1TICAwOS0wOS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005    AMENDED CLAIMS  09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgIEFNRU5ERUQgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005    AMENDED PAGES OF SPECIFICATION  09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgIEZPUk0tMSAgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005    FORM-1  09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgIEZPUk0tMyAgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005    FORM-3  09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDExLTEwLTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   CORRESPONDENCE OTHERS 11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgRk9STS0xICAxMS0xMC0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   FORM-1  11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgRk9STS0xMyAgMTEtMTAtMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   FORM-13  11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgRk9STS0xMy0xICAgMTEtMTAtMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   FORM-13-1   11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgRk9STS0zICAxMS0xMC0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   FORM-3  11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgRk9STS01ICAxMS0xMC0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   FORM-5  11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAxMS0xMC0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005   OTHER PATENT DOCUMENT  11-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDI0LTExLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  CORRESPONDENCE OTHERS  24-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBDT1JSRVNQT05ERU5DRSBQTy5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  EXAMINATION REPORT REPLY RECEIVED 09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBGT1JNLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  FORM-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  OTHER PATENT DOCUMENT 09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1DSEVOUC0yMDA1ICBQT1dFUiBPRiBBVFRPUk5FWSAgMDktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-CHENP-2005  POWER OF ATTORNEY  09-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-pct.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUxNy1jaGVucC0yMDA1LXNlcXVlbmNlIGxpc3RpbmcucGRm" target="_blank" style="word-wrap:break-word;">3517-chenp-2005-sequence listing.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250387-method-and-system-for-the-transmission-of-service-data-between-telecommunication-devices.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250389-insectidal-proteins-secreted-from-bacillus-thuringiensis-and-uses-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250388</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3517/CHENP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES A/S and NOVOZYMES NORTH AMERICA,INC .</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TAIRA RIKAKO</td>
											<td>9-10, TAIHEI 4-CHOME, SUMIDA-KU, TOKYO 130-0012</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TKAGI RIKAKO,</td>
											<td>9-10, TAIHEI 4-CHOME, SUMIDA-KU, TOKYO 130-0012,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HJORT CARSTEN</td>
											<td>RAVNHUSPARKEN 6, DK-3500 VAERLOESE, DENMARK</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VIKSO-NIELSEN, ANDERS</td>
											<td>LINDVEJ 12, JORLUNDE, DK-3550 SLANGERUP</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ALLAIN, ERIC</td>
											<td>925 BARNFORD MILL ROAD WAKE FOREST, NC 27587, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>UDAGAWA HIROAKI</td>
											<td>1-2-5, ISEZAKI-CHO, NAKOHAMA-SHI, YOKOHAMA-SHI 231-0045, JAPAN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/30</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US04/20499</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-06-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/490,751</td>
									<td>2003-07-29</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250388-a-hybrid-enzyme-for-starch-processing by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:27:58 GMT -->
</html>
