<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250372-fc-variant-with-altered-binding-to-fcrn by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:29 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250372:FC VARIANT WITH ALTERED BINDING TO FCRN</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">FC VARIANT WITH ALTERED BINDING TO FCRN</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present application relates to optimized IgG immunoglobulin variants, engineering methods for their generation, and their application, particularly for therapeutic purposes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM-2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
Provisional /Complete specification<br>
[See Section 10 and rule 13]<br><br>
1	Title of the Invention<br>
	"FC VARIANTS WITH ALTERED BINDING TO FCRN"<br>
2	Applicant (s)<br>
	Applicant	XENCOR, INC<br>
	Nationality	United States of America<br>
	Address	111 West Lemon Avenue, Monrovia, California 91016, USA.<br>
The following specification particularly describes the invention and the manner in which it is to be performed.<br>
1<br><br>
Fc VARIANTS WITH ALTERED BINDING TO FcRn<br>
[1] This application claims benefit under 35 U.S.C. §119(e) to USSN 60/627,763, filed November 12, 2004; USSN, 60/642,886, filed January 11, 2005; USSN 60/649,508, filed February 2, 2005; USSN 60/662,468, filed March 15, 2005; 2005; USSN 60/669,311, filed April 6, 2005; USSN 60/681,607, filed May 16, 2005; USSN 60/690,200, filed June 13, 2005; USSN 60/696,609, filed July 5, 2005; USSN 60/703,018, filed July 27, 2005; and USSN 60/726,453, filed October 12, 2005, all entirely incorporated by reference.<br>
FIELD OF THE INVENTION<br>
[2] The present application relates to optimized IgG immunoglobulin variants, engineering methods for their generation, and their application, particularly for therapeutic purposes.<br>
BACKGROUND OF THE INVENTION<br>
[3] Antibodies are immunological proteins that bind a specific antigen. In most mammals, including humans and mice, antibodies are constructed from paired heavy and light polypeptide chains. Each chain is made up of individual immunoglobulin (Ig) domains, and thus the generic term immunoglobulin is used for such proteins. Each chain is made up of two distinct regions, referred to as the variable and constant regions. The light and heavy chain variable regions show significant sequence diversity between antibodies, and are responsible for binding the target antigen. The constant regions show less sequence diversity, and are responsible for binding a number of natural proteins to elicit important biochemical events. In humans there are five different classes of antibodies including IgA (which includes subclasses lgA1 and lgA2), IgD, IgE, IgG (which includes subclasses lgG1, lgG2, lgG3, and lgG4), and IgM. The distinguishing features between these antibody classes are their constant regions, although subtler differences may exist in the V region. Figure 1 shows an lgG1 antibody, used here as an example to describe the general structural features of immunoglobulins. IgG antibodies are tetrameric proteins composed of two heavy chains and two light chains. The IgG heavy chain is composed of four immunoglobulin domains linked from N- to C-terminus in the order VH-CH1-CH2-CH3, referring to the heavy chain variable domain, heavy chain constant domain 1, heavy chain constant domain 2, and heavy chain constant domain 3 respectively (also referred to as VH-Cy1-Cy2-Cy3, referring to the heavy chain variable domain, constant gamma 1 domain, constant gamma 2 domain, and constant gamma 3 domain respectively). The IgG light chain is composed of two immunoglobulin domains linked from N- to C-terminus in the order VL-CL, referring to the light chain variable domain and the light chain constant domain respectively.<br>
2<br><br>
[4] The variable region of an antibody contains the antigen binding determinants of the molecule, and thus determines the specificity of an antibody for its target antigen. The variable region is so named because it is the most distinct in sequence from other antibodies within the same class. The majority of sequence variability occurs in the complementarity determining regions (CDRs). There are 6 CDRs total, three each per heavy and light chain, designated VH CDR1, VH CDR2, VH CDR3, VL CDR1, VL CDR2, and VL CDR3. The variable region outside of the CDRs is referred to as the framework (FR) region. Although not as diverse as the CDRs, sequence variability does occur in the FR region between different antibodies. Overall, this characteristic architecture of antibodies provides a stable scaffold (the FR region) upon which substantial antigen binding diversity (the CDRs) can be explored by the immune system to obtain specificity for a broad array of antigens. A number of high-resolution structures are available for a variety of variable region fragments from different organisms, some unbound and some in complex with antigen. The sequence and structural features of antibody variable regions are well characterized (Morea et al., 1997, Biophys Chem 68:9-16; Morea ef al., 2000, Methods 20:267-279, entirely incorporated by reference), and the conserved features of antibodies have enabled the development of a wealth of antibody engineering techniques (Maynard ef al., 2000, Annu Rev Biomed Eng 2:339-376, entirely incorporated by reference). For example, it is possible to graft the CDRs from one antibody, for example a murine antibody, onto the framework region of another antibody, for example a human antibody. This process, referred to in the art as "humanization", enables generation of less immunogenic antibody therapeutics from nonhuman antibodies. Fragments including the variable region can exist in the absence of other regions of the antibody, including for example the antigen binding fragment (Fab) including VH-Cy1 and VH-CL, the variable fragment (Fv) including VH and VL, the single chain variable fragment (scFv) including VH and VL linked together in the same chain, as well as a variety of other variable region fragments (Little ef al., 2000, Immunol Today 21:364-370, entirely incorporated by reference).<br>
[5] The Fc region of an antibody interacts with a number of Fc receptors and ligands, imparting an array of important functional capabilities referred to as effector functions. For IgG the Fc region, as shown in Figures 1 and 2, comprises Ig domains Cy2 and Cy3 and the N-terminal hinge leading into Cy2. An important family of Fc receptors for the IgG class are the Fc gamma receptors (FcyRs). These receptors mediate communication between antibodies and the cellular arm of the immune system (Raghavan ef al., 1996, Annu Rev Cell Dev Biol 12:181-220; Ravetch ef al., 2001, Annu Rev Immunol 19:275-290, both entirely incorporated by reference). In humans this protein family includes FcyRI (CD64), including isoforms FcyRIa, FcyRIb, and FcyRIc; FcyRII (CD32), including isoforms FcyRlla (including allotypes H131 and R131), FcyRllb (including FcyRllb-1 and FcyRllb-2), and FcyRllca; and FcyRIII (CD16), including isoforms FcyRllla (including<br>
3<br><br>
allotypes V158 and F158) and FcyRlllb (including allotypes FcyRlllb-NA1 and FcyRIUb-NA2) (Jefferis et a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference). These receptors typically have an extracellular domain that mediates binding to Fc, a membrane spanning region, and an intracellular domain that may mediate some signaling event within the cell. These receptors are expressed in a variety of immune cells including monocytes, macrophages, neutrophils, dendritic cells, eosinophils, mast cells, platelets, B cells, large granular lymphocytes, Langerhans' cells, natural killer (NK) cells, and yy T cells Formation of the Fc/FcyR complex recruits these effector cells to sites of bound antigen, typically resulting in signaling events within the cells and important subsequent immune responses such as release of inflammation mediators, B cell activation, endocytosis, phagocytosis, and cytotoxic attack. The ability to mediate cytotoxic and phagocytic effector functions is a potential mechanism by which antibodies destroy targeted cells. The cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound antibody on a target cell and subsequently cause lysis of the target cell is referred to as antibody dependent cell-mediated cytotoxicity (ADCC) (Raghavan et al., 1996, Annu Rev Cell Dev Biol 12:181-220; Ghetie er al., 2000, Annu Rev Immunol 18:739-766; Ravetch et al., 2001, Annu Rev Immunol 19:275-290, all entirely incorporated by reference). The cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound antibody on a target cell and subsequently cause phagocytosis of the target cell is referred to as antibody dependent cell-mediated phagocytosis (ADCP). A number of structures have been solved of the extracellular domains of human FcyRs, including FcyRlla (pdb accession code 1H9V, entirely incorporated by reference)(Sondermann ef al., 2001, J Mol Biol 309:737-749, entirely incorporated by reference) (pdb accession code 1FCG, entirely incorporated by reference)(Maxwell ef al., 1999, Nat Struct Biol 6:437-442, entirely incorporated by reference), FcyRllb (pdb accession code 2FCB, entirely incorporated by reference)(Sondermann ef al., 1999, Embo J 18:1095-1103, entirely incorporated by reference); and FcyRlllb (pdb accession code 1E4J, entirely incorporated by reference)(Sondermann ef al., 2000, Nature 406:267-273, entirely incorporated by reference.). All FcyRs bind the same region on Fc, at the N-terminal end of the Cy2 domain and the preceding hinge, shown in Figure 1. This interaction is well characterized structurally (Sondermann ef al., 2001, J Mol Biol 309:737-749, entirely incorporated by reference), and several structures of the human Fc bound to the extracellular domain of human FcyRlllb have been solved (pdb accession code 1E4K, entirely incorporated by reference)(Sondermann ef al., 2000, Nature 406:267-273, entirely incorporated by reference) (pdb accession codes 11IS and 1IIX, entirely incorporated by reference)(Radaev et al., 2001, J Biol Chem 276:16469-16477, entirely incorporated by reference), as well as has the structure of the human IgE Fc/FceRIa complex (pdb accession code 1F6A, entirely incorporated by reference)(Garman ef al., 2000, Nature 406:259-266, entirely incorporated by reference).<br>
4<br><br>
[6] The different IgG subclasses have different affinities for the FcyRs, with lgG1 and lgG3 typically binding substantially better to the receptors than lgG2 and lgG4 (Jefferis et a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference). All FcyRs bind the same region on IgG Fc, yet with different affinities: the high affinity binder FcyRI has a Kd for lgG1 of 10"8 M'1, whereas the low affinity receptors FcyRII and FcyRIII generally bind at 10"6 and 10'5 respectively. The extracellular domains of FcyRllla and FcyRlllb are 96% identical; however FcyRlllb does not have a intracellular signaling domain. Furthermore, whereas FcyRI, FcyRlla/c, and FcyRllla are positive regulators of immune complex-triggered activation, characterized by having an intracellular domain that has an immunoreceptor tyrosine-based activation motif (ITAM), FcyRllb has an immunoreceptor tyrosine-based inhibition motif (ITIM) and is therefore inhibitory. Thus the former are referred to as activation receptors, and FcyRllb is referred to as an inhibitory receptor. The receptors also differ in expression pattern and levels on different immune cells. Yet another level of complexity is the existence of a number of FcyR polymorphisms in the human proteome. A particularly relevant polymorphism with clinical significance is V158/F158 FcyRllla. Human lgG1 binds with greater affinity to the V158 allotype than to the F158 allotype. This difference in affinity, and presumably its effect on ADCC and/or ADCP, has been shown to be a significant determinant of the efficacy of the anti-CD20 antibody rituximab (Rituxan®, Biogenldec). Patients with the V158 allotype respond favorably to rituximab treatment; however, patients with the lower affinity F158 allotype respond poorly (Cartron et a/., 2002, Blood 99:754-758, entirely incorporated by reference). Approximately 10-20% of humans are V158A/158 homozygous, 45% are V158/F158 heterozygous, and 35-45% of humans are F158/F158 homozygous (Lehrnbecher era/., 1999, Blood 94:4220-4232; Cartron et a/., 2002, Blood 99:754-758, all entirely incorporated by reference). Thus 80-90% of humans are poor responders, i.e., they have at least one allele of the F158 FcyRllla.<br>
[7] An overlapping but separate site on Fc, shown in Figure 1, serves as the interface for the complement protein C1q. In the same way that Fc/FcyR binding mediates ADCC, Fc/C1q binding mediates complement dependent cytotoxicity (CDC). C1q forms a complex with the serine proteases C1r and C1s to form the C1 complex. C1q is capable of binding six antibodies, although binding to two IgGs is sufficient to activate the complement cascade. Similar to Fc interaction with FcyRs, different IgG subclasses have different affinity for C1q, with lgG1 and lgG3 typically binding substantially better to the FcyRs than lgG2 and lgG4 (Jefferis et a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference).<br>
[8] In IgG, a site on Fc between the Cg2 and Cg3 domains (Figure 1) mediates interaction with the neonatal receptor FcRn, the binding of which recycles endocytosed antibody from the endosome back to the bloodstream (Raghavan et al., 1996, Annu Rev Cell Dev Biol 12:181-220;<br>
5<br><br>
Ghetie et al., 2000, Annu Rev Immunol 18:739-766, both entirely incorporated by reference). This process, coupled with preclusion of kidney filtration due to the large size of the full-length molecule, results in favorable antibody serum half-lives ranging from one to three weeks. Binding of Fc to FcRn also plays a key role in antibody transport. The binding site on Fc for FcRn is also the site at which the bacterial proteins A and G bind. The tight binding by these proteins is typically exploited as a means to purify antibodies by employing protein A or protein G affinity chromatography during protein purification. Thus the fidelity of this region on Fc is important for both the clinical properties of antibodies and their purification. Available structures of the rat Fc/FcRn complex (Burmeister et al., 1994, Nature, 372:379-383; Martin et al., 2001, Mol Cell 7:867-877, both entirely incorporated by reference), and of the complexes of Fc with proteins A and G (Deisenhofer, 1981, Biochemistry 20:2361-2370; Sauer-Eriksson et al., 1995, Structure 3:265-278; Tashiro et al., 1995, Curr Opin Struct Biol 5:471-481, all entirely incorporated by reference), provide insight into the interaction of Fc with these proteins. The FcRn receptor is also responsible for the transfer of IgG to the neo-natal gut and to the lumen of the intestinal epithelia in adults (Ghetie and Ward, Annu. Rev. Immunol., 2000, 18:739-766; Yoshida et al., Immunity, 2004, 20(6):769-783, both entirely incorporated by reference).<br>
[9] Studies of rat and human Fey domains have demonstrated the importance of some Fc residues to the binding of FcRn. The rat and human sequences have about 64% sequence identity in the Fc regions (residues 237-443 in the numbering of Kabat et al.). See figures 3, 4, and 5 for the rat/human alignments of Fc, FcRn heavy chain, and FcRn light chain (beta-2-microglobulin). A model of the human Fc/FcRn complex has been built from the existing structure of the rat Fc/FcRn complex (Martin et al., 2001, Mol Cell 7:867-877, entirely incorporated by reference). The rat and human sequences share some residues that are critical for FcRn binding, such as H310 and H435 (Medesan et al., 1997 J. Immunol. 158(5):221-7; Shields et al., 2001, J. Biol. Chem. 276(9):6591-6604, both entirely incorporated by reference). In many positions, however, the human and rat proteins have different amino acids, giving the residues in the human sequence different environments, and possibly a different identities, than in the rat sequence. This variability limits the ability to transfer characteristics from one homolog to the other homolog.<br>
[10] In the murine Fey, random mutation and phage display selection at the sites, T252, T254, and T256 lead to a triple mutant, T252LVT254Sfi"256F, that has a 3.5-fold increase in FcRn affinity and a 1.5-fold increase in serum half-life (Ghetie et al., 1997, Nat. Biotech. 15(7): 637-640, entirely incorporated by reference).<br>
[11] The crystal structures of the rat Fc/FcRn complex identified important Fc residues for FcRn binding (Burmeister et al. Nature. 372:379-383 (1994); Martin et al. Molecular Cell. 7:867-877 (2001), both entirely incorporated by reference). The original Fc/FcRn complex structure was<br>
6<br><br>
solved in 1994 to a resolution of 6 A (Table 2a. Burmeister et al. Nature. 372:379-383 (1994), entirely incorporated by reference) The higher resolution structure, solved in 2001 by Marin et al, showed a more detailed view of the side chains positions (Martin et al. Molecular Cell. 7:867-877 (2001), entirely incorporated by reference) This crystal structure of rat Fc bound to rat FcRn was solved using an Fc dimer with one monomer containing the mutations T252G/I253G/T254G/H310E/H433E/H435E, which disrupt FcRn binding, and one monomer containing a wild-type Fc monomer.<br>
[12] Mutational studies in human Fey have been done on some of the residues that are important for binding to FcRn and have demonstrated have demonstrated an increased serum half-life. In human Fcy1, Hinton et al. mutated three residues individually to the other 19 common amino acids. Hinton et al., found that some point mutants a double mutant increased the FcRn binding affinity (Hinton et al., 2004, J. Biol. Chem. 279(8): 6213-6216, entirely incorporated by reference). Two mutations had increased half-lives in monkeys. Shields et al. mutated residues, almost exclusively to Ala, and studied their binding to FcRn and the FcyR's (Shields et al., 2001, J. Biol. Chem., 276(9):6591-6604, entirely incorporated by reference).<br>
[13] Dall'Acqua et al. used phage display to select for Fc mutations that bound FcRn with increased affinity (Dall' Acqua et al. 2002, J. Immunol. 169:5171-5180, entirely incorporated by reference). The DNA sequences selected for were primarily double and triple mutants. The reference expressed the proteins encoded by many of their selected sequences and found some that bound to FcRn more tightly than the wild-type Fc.<br>
[14] The administration of antibodies and Fc fusion proteins as therapeutics requires injections with a prescribed frequency relating to the clearance and half-life characteristics of the protein. Longer in vivo half-lives allow more seldom injections or lower dosing, which is clearly advantageous. Although the past mutations in the Fc domain have lead to some proteins with increased FcRn binding affinity and in vivo half-lives, these mutations have not identified the optimal mutations and enhanced in vivo half-life.<br>
[15] One feature of the Fc region is the conserved N-linked glycosylation that occurs at N297, shown in Figure 1. This carbohydrate, or oligosaccharide as it is sometimes referred, plays a critical structural and functional role for the antibody, and is one of the principle reasons that antibodies must be produced using mammalian expression systems. Umafla et al., 1999, Nat Biotechnol 17:176-180; Davies etal., 2001, Biotechnol Bioeng 74:288-294; Mimura etal., 2001, J Biol Chem 276:45539-45547.; Radaev et al., 2001, J Biol Chem 276:16478-16483; Shields et al., 2001, J Biol Chem 276:6591-6604; Shields et al., 2002, J Biol Chem 277:26733-26740; Simmons<br>
7<br><br>
ef al., 2002, J Immunol Methods 263:133-147; Radaev et al., 2001, J Biol Chem 276:16469-16477; and Krapp et al., 2003, J Mol Biol 325:979-989, all entirely incorporated by reference) [16] Antibodies have been developed for therapeutic use. Representative publications related to such therapies include Chamow ef al., 1996, Trends Biotechnol 14:52-60; Ashkenazi ef al., 1997, Curr Opin Immunol 9:195-200, Cragg ef al., 1999, Curr Opin Immunol 11:541-547; Glennie et al. 2000, Immunol Today 21:403-410, McLaughlin ef al., 1998, J Clin Oncol 16:2825-2833, and Cobleigh ef al., 1999, J Clin Oncol 17:2639-2648, all entirely incorporated by reference. Currently for anticancer therapy, any small improvement in mortality rate defines success. Certain IgG variants disclosed herein enhance the capacity of antibodies to limit further growth or destroy at least partially, targeted cancer cells.<br>
[17] Anti-tumor potency of antibodies is via enhancement of their ability to mediate cytotoxic effector functions such as ADCC, ADCP, and CDC. Examples include Clynes et al., 1998, Proc Natl Acad Sci USA 95:652-656; Clynes ef al., 2000, Nat Med 6:443-446 and Cartron ef al., 2002, Blood 99:754-758, both entirely incorporated by reference.<br>
[18] Human lgG1 is the most commonly used antibody for therapeutic purposes, and the majority of engineering studies have been constructed in this context. The different isotypes of the IgG class however, including lgG1, lgG2, lgG3, and lgG4, have unique physical, biological, and clinical properties. There is a need in the art to design improved lgG1, lgG2, lgG3, and lgG4 variants. There is a further need to design such variants to improve binding to FcRn and/or increase in vivo half-life as compared to native IgG polypeptides. The present application meets these and other needs.<br>
 SUMMARY OF THE INVENTION<br>
[19] The present invention discloses the generation of novel variants of Fc domains, including those found in antibodies, Fc fusions, and immuno-adhesions, which have an increased binding to the FcRn receptor and longer serum retention in vivo. An additional aspect of the invention is the increase in FcRn binding over wild type specifically at lower pH, about pH 6.0, to facilitate Fc/FcRn binding in the endosome. An additional aspect of the present invention is the preferential binding of the designed variants at about pH 6 compared to their binding at about pH 7.4 to facilitate the re-release of Fc into blood following cellular recycling.<br>
[20] A further aspect of the present invention relates to the design of Fc variants with decreased binding to FcRn and decreased in vivo half-lives. Such proteins comprising mutations to reduce FcRn affinity and/or the in vivo half-lives are useful in many therapies and diagnostics, including the delivery and monitoring of radiotherapeutics wherein, ideally, the half-life of the radiolabel is about equal to the in vivo half-life of its protein conjugate.<br>
8<br><br>
[21] A further aspect of the invention relates to the alteration of the Fc domain binding to the FcR's, e.g. in humans, FcgRI, FcgRlla, FcgRllb, FcgRllla. These receptors are responsible for inducing the various effector functions of antibodies. Therefore, a further aspect of the invention relates to the alteration of the Fc domain effector functions, such as antibody-dependent cell-mediated toxicity (ADCC), complement-dependent cytotoxicity (CDC), and antibody dependent cell-mediated phagocytosis (ADCP).<br>
[22] A further aspect of the invention relates to Fc variants that contained both altered FcRn binding and altered Fcg binding to affect both the in vivo half life and the efffector functions of the Fc-comprising protein. For example, these variants may have increased half-life in vivo as well as improved ADCC. The variants, for example, may have increased half-life and decreased CDC.<br>
[23] In a further aspect, the invention provides recombinant nucleic acids encoding the variant Fc proteins, expression vectors, and host cells.<br>
[24] In an additional aspect, the invention provides methods of producing a variant Fc-comprising protein comprising culturing the host cells of the invention under conditions suitable for expression of the protein.<br>
[25] in a further aspect, the invention provides pharmaceutical compositions comprising a variant Fc protein of the invention and a pharmaceutical carrier.<br>
[26] In a further aspect, the invention provides methods for treating disorders comprising administering a protein comprising a variant Fc of the invention to a patient.<br>
[27] ln an additional aspect, the invention provides an Fc variant region of a parent Fc polypeptide comprising at least one modification in the Fc region of said parent polypeptide, wherein said variant protein exhibits altered binding to FcRn as compared to the parent polypeptide, and wherein said Fc variant comprises at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248H, 248P, 248Q, 248R, 248Y, 249T, 249W, 251D, 251E, 251H, 2511, 251K, 251M, 251N, 251T, 251V, 251Y, 252F, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254T, 254V, A254N, 255E, 255F, 255H, 255K, 255S, 255V, 256E, 256H, 256V, 257A, 257C, 257D, 257E, 257F, 257G, 257H, 257I, 257K, 257L, 257M, 257N, 257Q, 257R, 257S, 257T, 257V, 257W, 257Y, 258R, 258V, 279A, 279C, 279D, 279F, 279G, 279H, 279I, 279K, 279L, 279M, 279N, 279P, 279Q, 279R, 279S, 279T, 279W, 279Y, 280E, 280H, A281A, A281D, A281S, A281T, 282D, 282F, 282H, 282I, 282T, 283F, 283I, 283L, 283Y, 284H, 284K, 284P, 284Q, 284R, 284S, 284Y, 285S, 285V, 286#, 286L, 287H, 287S, 287V, 287Y, 288H, 288Q, 288R, 288S, 305H, 305T,<br>
9<br><br>
306F. 306H, 3061. 306N. 306T. 306V. 306Y. 307D. 307V. 307Y. 308A. 308C, 308D, 308E, 308F, 308G, 308H, 3081. 308K. 308L. 308M. 308N. 308P, 308Q. 308R. 308S. 308T, 308W, 308Y, 309F. 309H. 3091. 309N. 309P. 309Q, 309V, 309Y, 310K, 310N, 310T, 311H, 311L, 311S, 311T, 311V, 311W, 312H. 313Y. 315E. 315G. 315H, 315Q. 315S, 315T, 317H, 317S, 339P, 340P, 341S. 374H. 374S. 376H. 376L. 378H. 378N. 380T, 380Y. 382H, 383H, 383K, 383Q, 384E, 384G, 384H, 385A, 385C. 385D, 385E, 385F, 385H, 385I, 385K, 385L, 385M, 385N, 385P, 385Q, 385R, 385S, 385T, 385V, 385W. 385Y. 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389E, 389H, 426E, 426H, 426L, 426N, 426R, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T. 429Y, 430D, 430H, 430K, 430L, 430Q, 430Y, 431G, 431H, 4311, 431P, 431S, 432F, 432H, 432N. 432S. 432V. 433E. 433N. 433P, 433R, 433S, 434H, 434Q, 434S, 434Y, 435N, 436E, 436F, 436H. 436L, 436Q, 436V, 436W, 437E, 437V, 438E, 438H and 438K, wherein numbering is according to the EU Index and A is an insertion after the identified position and # is a deletion of that identified position.<br>
[28]ln a further aspect, the invention provides Fc variants comprising at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248H, 248P, 248Q, 248R, 248Y, 249T, 249W, 251D, 251E, 251H, 2511, 251K, 251M, 251N, 251T, 251V, 251Y, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254V, A254N, 255E, 255F, 255H, 255K, 255S, 255V, 256H, 256V, 257A, 257C, 257D, 257E. 257F, 257G, 257H, 257I, 257K, 257L, 257M, 257N, 257Q, 257R, 257S, 257T, 257V, 257W, 257Y, 258R, 258V, 279A, 279C, 279D, 279F, 279G, 279H, 279I, 279K, 279M, 279N, 279P, 279Q, 279R, 279S, 279T, 279W, 279Y, 280H, A281A, A281D, A281S, A281T, 282D, 282F, 282H, 282I, 282T, 283F, 283I, 283L, 283Y, 284H, 284K, 284P, 284Q, 284R, 284S, 284Y, 285S, 285V, 286#, 286L, 287H, 287S, 287V, 287Y, 288H, 288Q, 288S, 305H, 305T, 306F, 306H, 306I, 306N, 306T, 306V, 306Y, 307D, 307V, 307Y, 308C, 308E, 308F, 308G, 308H, 308I, 308K, 308L, 308M, 308N, 308P, 308Q, 308R, 308S, 308W, 308Y, 309F, 309H, 309N, 309Q, 309V, 309Y, 310K, 310N, 310T, 311L, 311T, 311V, 311W, 312H, 313Y, 315E, 315G, 315H, 315Q, 315S, 315T, 317H, 317S, 339P, 340P, 341S, 374H, 374S, 376H, 376L, 378H, 378N, 380T, 380Y, 382H, 383H, 383K, 383Q, 384E, 384G, 384H, 385A, 385C, 385F, 385H, 385I, 385K, 385L, 385M, 385N, 385P, 385Q, 385S, 385T, 385V, 385W, 385Y, 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389E, 389H, 426E, 426H, 426L, 426N, 426R, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T, 429Y, 430D, 430H, 430K, 430L, 430Q, 430Y, 431G, 431H, 4311, 431P, 431S, 432F, 432H, 432N, 432S, 432V, 433E, 433N, 433P, 433S, 434H, 434Q, 434S, 435N, 436E, 436F, 436L, 436V, 436W, 437E, 437V, 438H, and 438K. <br>
[29] In an additional aspect, the invention provides Fc variants comprising at least one modification selected from the group consisting of: 246H, 246S, 247D, 247T, 248P, 248Q, 248Y, 249T, 249W, 251D, 251E, 251H, 2511, 251T, 251V, 252L, 253L, 253T, 253V, 254H, 254L, 254N, 254V, A254N, 255E, 255H, 255K, 255V, 256H, 256V, 257A, 257C, 257F, 257G, 257I, 257L,<br>
10<br><br>
257M, 257N, 257Q, 257S, 257T, 257V, 257W, 257Y, 258V, 279A, 279C, 279F, 2791, 279P. 279S, 279T, 279W, 279Y, *281A, A281D, A281S. A281T, 282F, 282I, 282T, 283F, 283I. 283L. 283Y, 284P, 285V. 286#, 286L, 287V. 288Q, 288S, 305T, 306F, 306H, 306I. 306N, 306T. 306V, 306Y, 307V, 308C, 308F, 308G, 308L, 308M, 308N, 308P, 308Q, 308S, 308W, 308Y, 309F, 309N, 309Q, 309V, 309Y, 310T, 311L, 311T. 311V, 311W, 313Y, 315G, 315Q. 315S. 315T. 339P, 340P, 341S, 374H, 374S, 376L, 378H, 378N, 380T, 380Y, 382H, 383Q, 384E, 384G. 384H, 385A, 385C, 385F, 385I, 385L, 385M, 385N, 385P, 385Q, 385S, 385T, 385V, 385W, 385Y, 386E, 386H, 386K, 387#, 387A, 387H, 387K, 387Q, 389H, 426L, 426N, 426V, 426Y, 427I, 429D, 429F, 429K, 429N, 429Q, 429S, 429T, 429Y, 430L, 431G, 4311, 431P, 431S, 432F, 432H, 432V, 433E, 433N, 433P, 433S, 434H, 434Q, 434S, 435N, 436F, 436L, 436V, 436W, 437E, and 437V<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[30] Figure 1. Antibody structure and function. Shown is a model of a full length human lgG1<br>
antibody, modeled using a humanized Fab structure from pdb accession code 1CE1 (James ef<br>
a/., 1999, J Mol Biol 289:293-301, entirely incorporated by reference) and a human lgG1 Fc<br>
structure from pdb accession code 1DN2 (DeLano ef al., 2000, Science 287:1279-1283, entirely<br>
incorporated by reference). The flexible hinge that links the Fab and Fc regions is not shown.<br>
lgG1 is a homodimer of heterodimers, made up of two light chains and two heavy chains. The Ig<br>
domains that comprise the antibody are labeled, and include VL and CL for the light chain, and VH,<br>
Cgammal (Cy1), Cgamma2 (Cy2), and Cgamma3 (Cy3) for the heavy chain. The Fc region is<br>
labeled. Binding sites for relevant proteins are labeled, including the antigen binding site in the<br>
variable region, and the binding sites for FcyRs, FcRn, Clq, and proteins A and G in the Fc<br>
region.<br>
[31] Figure 2. Human IgG sequences used in the present invention with the EU numbering as in<br>
Kabat et al.<br>
[32] Figure 3. Example human and rodent IgG sequences used in the present invention with the<br>
EU numbering as in Kabat.<br>
[33] Figure 4.   Example human and rodent FcRn heavy chain sequences used in the present<br>
invention.<br>
[34] Figure 5.  Example human and rodent beta-2-microglobulin sequences used in the present<br>
invention.<br>
[35] Figure 6. A human Fc/FcRn complex model created from the rat structures (Burmeister et<br>
al., 1994, Nature, 372:379-383; Martin et al., 2001, Mol Cell 7:867-877, both entirely incorporated<br>
by reference). Some histidine residues are shown in space-filling atoms on the FcRn chains (light<br>
grey) and Fc polypeptide (dark grey).<br>
[36] Figure 7.  Illustration of some concepts used in the design of variants comprising insertions<br>
or deletions.<br>
11<br><br>
[37] Figure 8. Variants of the present invention.<br>
[38] Figure 9. Variants of the present invention.<br>
[39] Figure 10. Variants of the present invention.<br>
[40] Figure 11. Diagram of the vector pcDNA3.1 Zeo+, which may be used in the construct of Fc<br>
variants.<br>
[41] Figure 12. Competition FcRn binding data of wild-type Fc and Fc variants of the present<br>
invention. In each panel, the Fc variants of the present invention are shown as the left (red or<br>
dark grey) curve and the wild-type trastuzumab is shown as the right (blue or light grey) curve.<br>
[42] Figure 13. Summary of FcRn binding properties of the Fc variants. The columns from right<br>
to left show the FcRn binding modifications, the immunoglobulin used, other modifications, the<br>
relative FcRn affinity by AlphaScreen™ competition assays compared to wild type (median<br>
value), the number of assays performed, and a reference number of the protein.   Relative FcRn<br>
affinity numbers greater than 1.0 demonstrate increased binding over wild type.<br>
[43] Figure 14. FcRn binding data of Fc variants of the present invention. The Fc variants are in<br>
alemtuzumab or trastuzumab. The fold-increased binding compared to wild type are shown.<br>
[44] Figure 15.    Summary of surface plasmon resonance experiments of Fc variants with<br>
improved binding to FcRn. The bar graph shows the fold-increase in FcRn binding affinity of each<br>
variant relative to wild-type Fc domain.<br>
[45] Figure 16.   Surface plasmon resonance experiments of wild-type antibody and variants of<br>
the present invention. The traces shown are the association and dissociation of the Fc variant<br>
antibody to FcRn at pH6.0.<br>
[46] Figure 17,   Binding assays of Fc variants of the present invention to FcRn.   Shown are<br>
direct binding assays measured by AlphaScreen™ at pH 6.0 (a and b) and pH 7.0 (c).<br>
[47] Figure 18.  Binding assays of Fc variants of the present invention to FcRn.  Shown are the<br>
surface plasmon resonance units created upon binding of the variant Fc to surface-bound FcRn.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[48] The present invention discloses the generation of novel variants of Fc domains, including those found in antibodies, Fc fusions, and immuno-adhesions, which have an increased binding to the FcRn receptor. As noted herein, binding to RcRn results in longer serum retention in vivo.<br>
 [49] In order to increase the retention of the Fc proteins in vivo, the increase in binding affinity must be at around pH 6 without a concomitant increase in affinity at around pH 7.4. Although still under examination, Fc regions are believed to have a longer half-lives in vivo, because binding to FcRn at pH 6 in an endosome sequesters the Fc (Ghetie and Ward, 1997 Immunol Today. 18(12): 592-598, entirely incorporated by reference). The endosomal compartment then recycles the Fc to the cell surface. Once the compartment opens to the extracellular space, the higher pH, -7.4, induces the release of Fc back into the blood. Dall' Acqua et al. showed that Fc mutants<br>
12<br><br>
with increased FcRn binding at pH 6 and pH 7.4 actually had reduced serum concentrations and<br>
the same half life as wild-type Fc (Dair Acqua et al. 2002, J. Immunol. 169:5171-5180, entirely<br>
incorporated by reference). The increased affinity of Fc for FcRn at pH 7.4 is thought to forbid the<br>
release of the Fc back into the blood. Therefore, the Fc mutations that will increase Fc's half-life<br>
in vivo will ideally increase FcRn binding at the lower pH while still allowing release of Fc at<br>
higher pH    The amino acid histidine changes its charge state in the pH range of 6.0 to 7.4.<br>
Therefore, it is not surprising to find His residues at important positions in the Fc/FcRn complex<br>
(Figure 6.)<br>
[50]An additional aspect of the invention is the increase in FcRn binding over wild type<br>
specifically at lower pH, about pH 6.0, to facilitate Fc/FcRn binding in the endosome.   Also<br>
disclosed are Fc variants with altered FcRn binding and altered binding to another class of Fc<br>
receptors, the FcyRs, as differential binding to FcyRs, particularly increased binding to FcyRlllb<br>
and decreased binding to FcyRllb has been shown to result in increased efficacy.<br>
[51]Definitions<br>
[52]ln order that the application may be more completely understood, several definitions are set<br>
forth below. Such definitions are meant to encompass grammatical equivalents.<br>
[53]By "ADCC" or "antibody dependent cell-mediated cytotoxicity" as used herein is meant the<br>
cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound<br>
antibody on a target cell and subsequently cause lysis of the target cell.<br>
[54]By "ADCP" or antibody dependent cell-mediated phagocytosis as used herein is meant the<br>
cell-mediated reaction wherein nonspecific cytotoxic cells that express FcyRs recognize bound<br>
antibody on a target cell and subsequently cause phagocytosis of the target cell.<br>
[55]By "amino acid modification" herein is meant an amino acid substitution, insertion, and/or<br>
deletion in a polypeptide sequence.<br>
[56]By "amino acid substitution" or "substitution" herein is meant the replacement of an amino<br>
acid at a particular position in a parent polypeptide sequence with another amino acid.   For<br>
example, the substitution E272Y refers to a variant polypeptide, in this case an Fc variant, in<br>
which the glutamic acid at position 272 is replaced with tyrosine.<br>
[57]By "amino acid insertion" or "insertion" as used herein is meant the addition of an amino acid<br>
at a particular position in a parent polypeptide sequence.    For example, -233E or A233E<br>
designates an insertion of glutamic acid after position 233 and before position 234. Additionally, -<br>
233ADE or A233ADE designates an insertion of AlaAspGlu after position 233 and before position<br>
234.<br>
[58]By "amino acid deletion" or "deletion" as used herein is meant the removal of an amino acid<br>
at a particular position in a parent polypeptide sequence.    For example, E233- or E233#<br>
designates a deletion of glutamic acid at position 233.    Additionally, EDA233- or EDA233#<br>
designates a deletion of the sequence GluAspAla that begins at position 233.<br>
13<br><br>
[59]By "variant protein" or "protein variant". or "variant" as used herein is meant a protein that differs from that of a parent protein by virtue of at least one ammo acid modification. Protein variant may refer to the protein itself, a composition comprising the protein, or the amino sequence that encodes it. Preferably, the protein variant has at least one amino add modification compared to the parent protein, e.g. from about one to about ten amino acid modifications, and preferably from about one to about five amino acid modifications compared to the parent. The protein variant sequence herein will preferably possess at least about 80% homology with a parent protein sequence, and most preferably at least about 90% homology, more preferably at least about 95% homology. Variant protein can refer to the variant protein itself, compositions comprising the protein variant, or the amino acid sequence that encodes it. Accordingly, by "antibody variant" or "variant antibody" as used herein is meant an antibody that differs from a parent antibody by virtue of at least one amino acid modification, "IqG variant" or "variant IqG" as used herein is meant an antibody that differs from a parent IgG by virtue of at least one amino acid modification, and "immunoglobulin variant" or "variant immunoglobulin" as used herein is meant an immunoglobulin sequence that differs from that of a parent immunoglobulin sequence by virtue of at least one amino acid modification. Variants may comprise non-natural amino acids. Examples include US6586207; WO 98/48032; WO 03/073238; US2004-0214988A1; WO 05/35727A2; WO 05/74524A2; J. W. Chin et al., (2002), Journal of the American Chemical Society 124:9026-9027; J. W. Chin, &amp; P. G. Schultz, (2002), ChemBioChem 11:1135-1137; J. W. Chin, et al., (2002), PICAS United States of America 99:11020-11024; and, L Wang, &amp; P. G. Schultz, (2002), Chem. 1-10, all entirely incorporated by reference.<br>
[60] As used herein, "protein" herein is meant at least two covalently attached amino acids, which includes proteins, polypeptides, oligopeptides and peptides. The peptidyl group may comprise naturally occurring amino acids and peptide bonds, or synthetic peptidomimetica structures, i.e. "analogs", such as peptoids (see Simon et al., PNAS USA 89(20):9367 (1992), entirely incorporated by reference). The amino acids may either be naturally occurring or non-naturally occurring; as will be appreciated by those in the art. For example, homo-phenylalanine, citrulline, and noreleucine are considered amino acids for the purposes of the invention, and both D- and L- (R or S) configured amino acids may be utilized. The variants of the present invention may comprise modifications that include the use of unnatural amino acids incorporated using, for example, the technologies developed by Schultz and colleagues, including but not limited to methods described by Cropp &amp; Shultz, 2004, Trends Genet. 20(12):625-30, Anderson et al., 2004, Proc Natl Acad Sci USA 101(2):7566-71, Zhang et al., 2003, 303(5656):371-3, and Chin et al., 2003, Science 301(5635):964-7, all entirely incorporated by reference. In addition, polypeptides may include synthetic derivatization of one or more side chains or termini, glycosylation, PEGylation, circular permutation, cyclization, linkers to other molecules, fusion to proteins or protein domains, and addition of peptide tags or labels.<br>
14<br><br>
[61]By "residue" as used herein is meant a position in a protein and its associated amino acid<br>
identity. For example, Asparagine 297 (also referred to as Asn297, also referred to as N297) is a<br>
residue in the human antibody IgGl<br>
[62]By "Fab" or "Fab region" as used herein is meant the polypeptides that comprises the VH,<br>
CH1, VL, and CL immunoglobulin domains.   Fab may refer to this region in isolation, or this<br>
region in the context of a full length antibody or antibody fragment.<br>
[63]By "IgG subclass modification" as used herein is meant an amino acid modification that<br>
converts one amino acid of one IgG isotype to the corresponding amino acid in a different,<br>
aligned IgG isotype. For example, because lgG1 comprises a tyrosine and lgG2 a phenylalanine<br>
at EU position 296, a F296Y substitution in lgG2 is considered an IgG subclass modification.<br>
[64]By "non-naturallv occurring modification" as used herein is meant an amino acid modification<br>
that is not isotypic.  For example, because none of the IgGs comprise a glutamic acid at position<br>
332, the substitution I332E in lgG1, lgG2, lgG3, or lgG4 is considered a non-naturally occuring<br>
modification.<br>
[65]By "amino acid" and "amino acid identity" as used herein is meant one of the 20 naturally<br>
occurring amino acids or any non-natural analogues that may be present at a specific, defined<br>
position.<br>
[66]By "effector function" as used herein is meant a biochemical event that results from the<br>
interaction of an antibody Fc region with an Fc receptor or ligand.  Effector functions include but<br>
are not limited to ADCC, ADCP, and CDC.<br>
[67]By "effector cell" as used herein is meant a cell of the immune system that expresses one or<br>
more Fc receptors and mediates one or more effector functions. Effector cells include but are not<br>
limited to monocytes, macrophages, neutrophils, dendritic cells, eosinophils, mast cells, platelets,<br>
B cells, large granular lymphocytes, Langerhans' cells, natural killer (NK) cells, and y5T cells, and<br>
may be from any organism including but not limited to humans, mice, rats, rabbits, and monkeys.<br>
[68]By "IgG Fc ligand" as used herein is meant a molecule, preferably a polypeptide, from any<br>
organism that binds to the Fc region of an IgG antibody to form an Fc / Fc ligand complex.   Fc<br>
ligands include but are not limited to FcyRs, FcyRs, FcyRs, FcRn, C1q, C3, mannan binding<br>
lectin, mannose receptor, staphylococcal protein A, streptococcal protein G, and viral FcyR.  Fc<br>
ligands also include Fc receptor homologs (FcRH), which are a family of Fc receptors that are<br>
homologous to the FcyRs (Davis ef a/., 2002, Immunological Reviews 190:123-136, entirely<br>
incorporated by reference).    Fc ligands may include undiscovered molecules that bind Fc.<br>
Particular IgG Fc ligands are FcRn and Fc gamma receptors.   By "Fc ligand" as used herein is<br>
meant a molecule, preferably a polypeptide, from any organism that binds to the Fc region of an<br>
antibody to form an Fc / Fc ligand complex.<br>
[69]By "Fc gamma receptor" or "FcyR" as used herein is meant any member of the family of<br>
proteins that bind the IgG antibody Fc region and is encoded by an FcyR gene.  In humans this<br>
15<br><br>
family includes but is not limited to FcyRI (CD64), including isoforms FcyRIa, FcyRIb, and FcyRIc; FcyRII (CD32), including isoforms FcyRlla (including allotypes H131 and R131), FcyRllb (including FcyRllb-1 and FcyRllb-2), and FcyRllc; and FcyRill (CD16), including isoforms FcyRllla (including allotypes V158 and F158) and FcyRlllb (including allotypes FcyRlllb-NA1 and FcyRlllb-NA2) (Jefferis er a/., 2002, Immunol Lett 82:57-65, entirely incorporated by reference), as well as any undiscovered human FcyRs or FcyR isoforms or allotypes. An FcyR may be from any organism, including but not limited to humans, mice, rats, rabbits, and monkeys. Mouse FcyRs include but are not limited to FcyRI (CD64), FcyRII (CD32), FcyRill (CD16), and FcyRIII-2 (CD16-2), as well as any undiscovered mouse FcyRs or FcyR isoforms or allotypes.<br>
 [70]By "FcRn" or "neonatal Fc Receptor" as used herein is meant a protein that binds the IgG antibody Fc region and is encoded at least in part by an FcRn gene. The FcRn may be from any organism, including but not limited to humans, mice, rats, rabbits, and monkeys. As is known in the art, the functional FcRn protein comprises two polypeptides, often referred to as the heavy chain and light chain. The light chain is beta-2-microglobulin and the heavy chain is encoded by the FcRn gene. Unless other wise noted herein, FcRn or an FcRn protein refers to the complex of FcRn heavy chain with beta-2-microglobulin. Sequences of particular interest of FcRn are shown in the Figures, particularly the human species.<br>
[71]By "parent polypeptide" as used herein is meant an unmodified polypeptide that is subsequently modified to generate a variant. The parent polypeptide may be a naturally occurring polypeptide, or a variant or engineered version of a naturally occurring polypeptide. Parent polypeptide may refer to the polypeptide itself, compositions that comprise the parent polypeptide, or the amino acid sequence that encodes it. Accordingly, by "parent immunoglobulin" as used herein is meant an unmodified immunoglobulin polypeptide that is modified to generate a variant, and by "parent antibody" as used herein is meant an unmodified antibody that is modified to generate a variant antibody. It should be noted that "parent antibody" includes known commercial, recombinantly produced antibodies as outlined below. <br>
[72]By "position" as used herein is meant a location in the sequence of a protein. Positions may be numbered sequentially, or according to an established format, for example the EU index as in Kabat. For example, position 297 is a position in the human antibody IgGl <br>
[73]By "target antigen" as used herein is meant the molecule that is bound specifically by the variable region of a given antibody. A target antigen may be a protein, carbohydrate, lipid, or other chemical compound.<br>
[74]By "target cell" as used herein is meant a cell that expresses a target antigen. [75]By "variable region" as used herein is meant the region of an immunoglobulin that comprises one or more Ig domains substantially encoded by any of the Vic, VA., and/or VH genes that make up the kappa, lambda, and heavy chain immunoglobulin genetic loci respectively.<br>
16<br><br>
[76]By "wild type or WT" herein is meant an amino acid sequence or a nucleotide sequence that is found in nature, including allelic variations A WT protein has an amino acid sequence or a nucleotide sequence that has not been intentionally modified.<br>
[77] The present invention is directed to antibodies that exhibit moduluated binding to FcRn (modulation including increased as well as decreased binding) For example, in some instances, increased binding results in cellular recycling of the antibody and hence increased half-life, for example for therapeutic antibodies. Alternatively, decreased FcRn binding is desirable, for example for diagnostic antibodies or therapeutic antibodies that contain radiolabels. In addition, antibodies exhibiting increased binding to FcRn and altered binding to other Fc receptors, eg. FcyRs, find use in the present invention. Accordingly, the present invention provides antibodies. Antibodies<br>
[78]The present application is directed to antibodies that include amino acid modifications that modulate binding to FcRn. Of particular interest are antibodies that minimally comprise an Fc region, or functional variant thereof, that display increased binding affinity to FcRn at lowered pH, and do not exhibit substantially altered binding at higher pH.<br>
[79]Traditional antibody structural units typically comprise a tetramer. Each tetramer is typically composed of two identical pairs of polypeptide chains, each pair having one "light" (typically having a molecular weight of about 25 kDa) and one "heavy" chain (typically having a molecular weight of about 50-70 kDa). Human light chains are classified as kappa and lambda light chains. Heavy chains are classified as mu, delta, gamma, alpha, or epsilon, and define the antibody's isotype as IgM, IgD, IgG, IgA, and IgE, respectively. IgG has several subclasses, including, but not limited to lgG1, lgG2, lgG3, and lgG4. IgM has subclasses, including, but not limited to, lgM1 and lgM2. Thus, "isotype" as used herein is meant any of the subclasses of immunoglobulins defined by the chemical and antigenic characteristics of their constant regions. The known human immunoglobulin isotypes are lgG1, lgG2, lgG3, lgG4, lgA1, lgA2, lgM1, lgM2, IgD, and IgE. [80]The amino-terminal portion of each chain includes a variable region of about 100 to 110 or more amino acids primarily responsible for antigen recognition. In the variable region, three loops are gathered for each of the V domains of the heavy chain and light chain to form an antigen-binding site. Each of the loops is referred to as a complementarity-determining region (hereinafter referred to as a "CDR"), in which the variation in the amino acid sequence is most significant.<br>
[81]The carboxy-terminal portion of each chain defines a constant region primarily responsible for effector function. Kabat et al. collected numerous primary sequences of the variable regions of heavy chains and light chains. Based on the degree of conservation of the sequences, they classified individual primary sequences into the CDR and the framework and made a list thereof (see SEQUENCES OF IMMUNOLOGICAL INTEREST, 5th edition, NIH publication, No. 91-3242, E.A. Kabat et al., entirely incorporated by reference).<br>
17<br><br>
[82]In the IgG subclass of immunoglobulins, there are several immunoglobulin domains in the heavy chain. By "immunoglobulin (la) domain"herein is meant a region of an immunoglobulin having a distinct tertiary structure. Of interest in the present invention are the heavy chain domains, including, the constant heavy (CH) domains and the hinge domains. In the context of IgG antibodies, the IgG isotypes each have three CH regions. Accordingly, "CH" domains in the context of IgG are as follows: "CH1" refers to positions 118-220 according to the EU index as in Kabat. "CH2" refers to positions 237-340 according to the EU index as in Kabat, and "CH3" refers to positions 341-447 according to the EU index as in Kabat.<br>
[83]Another type of Ig domain of the heavy chain is the hinge region. By "hinge" or "hinge region" or "antibody hinge region" or "immunoglobulin hinge region" herein is meant the flexible polypeptide comprising the amino acids between the first and second constant domains of an antibody. Structurally, the IgG CH1 domain ends at EU position 220, and the IgG CH2 domain begins at residue EU position 237. Thus for IgG the antibody hinge is herein defined to include positions 221 (D221 in lgG1) to 236 (G236 in lgG1), wherein the numbering is according to the EU index as in Kabat. In some embodiments, for example in the context of an Fc region, the lower hinge is included, with the "lower hinge" generally referring to positions 226 or 230. <br>
[84]Of particular interest in the present invention are the Fc regions. By "Fc" or "Fc region", as used herein is meant the polypeptide comprising the constant region of an antibody excluding the first constant region immunoglobulin domain and in some cases, part of the hinge. Thus Fc refers to the last two constant region immunoglobulin domains of IgA, IgD, and IgG, and the last three constant region immunoglobulin domains of IgE and IgM, and the flexible hinge N-terminal to these domains. For IgA and IgM, Fc may include the J chain. For IgG, as illustrated in Figure 1, Fc comprises immunoglobulin domains Cgamma2 and Cgamma3 (Cg2 and Cg3) and the lower hinge region between Cgammal (Cg1) and Cgamma2 (Cg2). Although the boundaries of the Fc region may vary, the human IgG heavy chain Fc region is usually defined to include residues C226 or P230 to its carboxyl-terminus, wherein the numbering is according to the EU index as in Kabat. Fc may refer to this region in isolation, or this region in the context of an Fc polypeptide, as described below. By "Fc polypeptide" as used herein is meant a polypeptide that comprises all or part of an Fc region. Fc polypeptides include antibodies, Fc fusions, isolated Fes, and Fc fragments.<br>
[85]ln some embodiments, the antibodies are full length. By "full length antibody"herein is meant the structure that constitutes the natural biological form of an antibody, including variable and constant regions, including one or more modifications as outlined herein.<br>
[86]Altematively, the antibodies can be a variety of structures, including, but not limited to, antibody fragments, monoclonal antibodies, bispecific antibodies, minibodies, domain antibodies, synthetic antibodies (sometimes referred to herein as "antibody mimetics"), chimeric antibodies,<br>
18<br><br>
humanized antibodies, antibody fusions (sometimes referred to as "antibody conjugates"), and fragments of each, respectively. Antibody Fragments<br>
[87]ln one embodiment, the antibody is an antibody fragment.    Of particular interest are antibodies that comprise Fc regions, Fc fusions, and the constant region of the heavy chain (CH1-hinge-CH2-CH3), again also including constant heavy region fusions. <br>
[88]Specific antibody fragments include, but are not limited to, (i) the Fab fragment consisting of VL, VH, CL and CH1 domains, (ii) the Fd fragment consisting of the VH and CH1 domains, (iii) the Fv fragment consisting of the VL and VH domains of a single antibody; (iv) the dAb fragment (Ward et al., 1989, Nature 341:544-546, entirely incorporated by reference) which consists of a single variable, (v) isolated CDR regions, (vi) F(ab')2 fragments, a bivalent fragment comprising two linked Fab fragments (vii) single chain Fv molecules (scFv), wherein a VH domain and a VL domain are linked by a peptide linker which allows the two domains to associate to form an antigen binding site (Bird et al., 1988, Science 242:423-426, Huston et al., 1988, Proc. Natl. Acad. Sci. U.S.A. 85:5879-5883, entirely incorporated by reference), (viii) bispecific single chain Fv (WO 03/11161,  hereby incorporated by reference) and (ix) "diabodies" or "triabodies", multivalent or multispecific   fragments constructed by gene fusion (Tomlinson et. al., 2000, Methods Enzymol. 326:461-479; WO94/13804; Holliger et al., 1993, Proc. Natl. Acad. Sci. U.S.A. 90:6444-6448, all entirely incorporated by reference).  The antibody fragments may be modified. For example, the molecules may be stabilized by the incorporation of disulphide bridges linking the VH and VL domains (Reiter et al., 1996, Nature Biotech. 14:1239-1245, entirely incorporated by reference).<br>
Chimeric and Humanized Antibodies<br>
[89] In some embodiments, the scaffold components can be a mixture from different species. As such, if the antibody is an antibody, such antibody may be a chimeric antibody and/or a humanized antibody. In general, both "chimeric antibodies" and "humanized antibodies" refer to antibodies that combine regions from more than one species. For example, "chimeric antibodies" traditionally comprise variable region(s) from a mouse (or rat, in some cases) and the constant region(s) from a human. "Humanized antibodies" generally refer to non-human antibodies that have had the variable-domain framework regions swapped for sequences found in human antibodies. Generally, in a humanized antibody, the entire antibody, except the CDRs, is encoded by a polynucleotide of human origin or is identical to such an antibody except within its CDRs. The CDRs, some or all of which are encoded by nucleic acids originating in a non-human organism, are grafted into the beta-sheet framework of a human antibody variable region to create an antibody, the specificity of which is determined by the engrafted CDRs. The creation of such antibodies is described in, e.g., WO 92/11018, Jones, 1986, Nature 321:522-525, Verhoeyen  et  al.,   1988,   Science  239:1534-1536,   all  entirely  incorporated   by  reference.<br>
19<br><br>
"Backmutation" of selected acceptor framework residues to the corresponding donor residues is often required to regain affinity that is lost in the initial grafted construct (US 5530101; US 5585089, US 5693761; US 5693762; US 6180370; US 5859205; US 5821337; US 6054297; US 6407213  all entirely incorporated by reference).   The humanized antibody optimally also will comprise at least a portion of an immunoglobulin constant region, typically that of a human immunoglobulin, and thus will typically comprise a human Fc region.  Humanized antibodies can also be generated using mice with a genetically engineered immune system. Roque et al., 2004, Biotechnol  Prog  20:639-654, entirely incorporated by reference.   A variety of techniques and methods for humanizing and reshaping non-human antibodies are well known in the art (See Tsurushita &amp; Vasquez, 2004, Humanization of Monoclonal Antibodies, Molecular Biology of B Cells, 533-545, Elsevier Science (USA), and references cited therein, all entirely incorporated by reference).  Humanization methods include but are not limited to methods described in Jones ef al., 1986. Nature 321:522-525; Riechmann et a/.,1988; Nature 332:323-329; Verhoeyen et al., 1988, Science. 239:1534-1536; Queen et al., 1989, Proc Natl Acad Sci, USA 86:10029-33; He et al., 1998, J. Immunol. 160: 1029-1035; Carter ef al., 1992, Proc Natl Acad Sci USA 89:4285-9, Presta et al., 1997, Cancer Res.57(20):4593-9; Gorman et al., 1991, Proc. Natl. Acad. Sci. USA 88:4181-4185;  O'Connor ef al.,   1998,  Protein Eng  11:321-8,  all entirely incorporated  by reference.     Humanization or other methods of reducing the immunogenicity of nonhuman antibody variable regions may include resurfacing methods, as described for example in Roguska et al., 1994, Proc. Natl. Acad. Sci. USA 91:969-973, entirely incorporated by reference.   In one embodiment, the parent antibody has been affinity matured, as is known in the art.   Structure-based methods may be employed for humanization and affinity maturation, for example as described in USSN 11/004,590. Selection based methods may be employed to humanize and/or affinity mature antibody variable regions, including but not limited to methods described in Wu et al., 1999, J. Mol. Biol. 294:151-162; Baca et al., 1997, J. Biol. Chem. 272(16): 10678-10684; Rosok et al.. 1996, J. Biol. Chem. 271(37): 22611-22618; Rader et al., 1998, Proc. Natl. Acad. Sci. USA 95: 8910-8915; Krauss et al., 2003, Protein Engineering 16(10):753-759, all entirely incorporated by reference. Other humanization methods may involve the grafting of only parts of the CDRs, including but not limited to methods described in USSN 09/810,510; Tan et al., 2002, J. Immunol. 169:1119-1125; De Pascalis et al., 2002, J. Immunol. 169:3076-3084, all entirely incorporated by reference. Bispecific Antibodies<br>
[90]ln one embodiment, the antibodies of the invention multispecific antibody, and notably a bispecific antibody, also sometimes referred to as "diabodies". These are antibodies that bind to two (or more) different antigens. Diabodies can be manufactured in a variety of ways known in the art (Holliger and Winter, 1993, Current Opinion Biotechnol. 4:446-449, entirely incorporated by reference), e.g., prepared chemically or from hybrid hybridomas.<br>
20<br><br>
Minibodies<br>
[91]ln one embodiment, the antibody is a minibody    Minibodies are minimized antibody-like proteins comprising a scFv joined to a CH3 domain. Hu et at., 1996, Cancer Res 56:3055-3061, entirely incorporated by reference.  In some cases, the scFv can be joined to the Fc region, and may include some or all of the hinge region Human Antibodies<br>
[92]ln one embodiment, the antibody is a fully human antibody with at least one modification as outlined herein.    "Fully human antibody" or "complete human antibody" refers to a human antibody having the gene sequence of an antibody derived from a human chromosome with the modifications outlined herein. [93] Antibody Fusions<br>
[94] In one embodiment, the antibodies of the invention are antibody fusion proteins (sometimes referred to herein as an "antibody conjugate"). One type of antibody fusions comprises Fc fusions, which join the Fc region with a conjugate partner. By "Fc fusion" as used herein is meant a protein wherein one or more polypeptides is operably linked to an Fc region. Fc fusion is herein meant to be synonymous with the terms "immunoadhesin", "Ig fusion", "Ig chimera", and "receptor globulin" (sometimes with dashes) as used in the prior art (Chamow et al., 1996, Trends Biotechnol 14:52-60; Ashkenazi et al., 1997, Curr Opin Immunol 9:195-200, both entirely incorporated by reference). An Fc fusion combines the Fc region of an immunoglobulin with a fusion partner, which in general can be any protein or small molecule. Virtually any protein or small molecule may be linked to Fc to generate an Fc fusion. Protein fusion partners may include, but are not limited to, the variable region of any antibody, the target-binding region of a receptor, an adhesion molecule, a ligand, an enzyme, a cytokine, a chemokine, or some other protein or protein domain. Small molecule fusion partners may include any therapeutic agent that directs the Fc fusion to a therapeutic target. Such targets may be any molecule, preferably an extracellular receptor, which is implicated in disease. Thus, the IgG variants can be linked to one or more fusion partners. In one alternate embodiment, the IgG variant is conjugated or operably linked to another therapeutic compound. The therapeutic compound may be a cytotoxic agent, a chemotherapeutic agent, a toxin, a radioisotope, a cytokine, or other therapeutically active agent. The IgG may be linked to one of a variety of nonproteinaceous polymers, e.g., polyethylene glycol, polypropylene glycol, polyoxyalkylenes, or copolymers of polyethylene glycol and polypropylene glycol.<br>
[95]ln addition to Fc fusions, antibody fusions include the fusion of the constant region of the heavy chain with one or more fusion partners (again including the variable region of any antibody), while other antibody fusions are substantially or completely full length antibodies with fusion partners. In one embodiment, a role of the fusion partner is to mediate target binding, and thus it is functionally analogous to the variable regions of an antibody (and in fact can be).<br>
21<br><br>
Virtually any protein or small molecule may be linked to Fc to generate an Fc fusion (or antibody fusion). Protein fusion partners may include, but are not limited to, the target-binding region of a receptor, an adhesion molecule, a ligand, an enzyme, a cytokine, a chemokine, or some other protein or protein domain. Small molecule fusion partners may include any therapeutic agent that directs the Fc fusion to a therapeutic target. Such targets may be any molecule, preferably an extracellular receptor, which is implicated in disease.<br>
[96]The conjugate partner can be proteinaceous or non-proteinaceous; the latter generally being generated using functional groups on the antibody and on the conjugate partner. For example linkers are known in the art; for example, homo-or hetero-bifunctional linkers as are well known (see, 1994 Pierce Chemical Company catalog, technical section on cross-linkers, pages 155-200, incorporated herein by reference).<br>
[97]Suitable conjugates include, but are not limited to, labels as described below, drugs and cytotoxic agents including, but not limited to, cytotoxic drugs (e.g., chemotherapeutic agents) or toxins or active fragments of such toxins. Suitable toxins and their corresponding fragments include diptheria A chain, exotoxin A chain, ricin A chain, abrin A chain, curcin, crotin, phenomycin, enomycin and the like. Cytotoxic agents also include radiochemicals made by conjugating radioisotopes to antibodies, or binding of a radionuclide to a chelating agent that has been covalently attached to the antibody. Additional embodiments utilize calicheamicin, auristatinsa, geldanamycin, maytansine, and duocarmycins and analogs; for the latter, see U.S. 2003/0050331A1, hereby incorporated by reference in its entirety. Covalent modifications of Antibodies<br>
[98]Covalent modifications of antibodies are included within-the scope of this invention, and are generally, but not always, done post-translationally. For example, several types of covalent modifications of the antibody are introduced into the molecule by reacting specific amino acid residues of the antibody with an organic derivatizing agent that is capable of reacting with selected side chains or the N- or C-terminal residues.<br>
[99]Cysteinyl residues most commonly are reacted with a-haloacetates (and corresponding<br>
amines), such as chloroacetic acid or chloroacetamide, to give carboxymethyl or<br>
carboxyamidomethyl derivatives. Cysteinyl residues may also be derivatized by reaction with<br>
bromotrifluoroacetone, a-bromo-P-(5-imidozoyl)propionic acid, chloroacetyl phosphate, N-<br>
alkylmaleimides, 3-nitro-2-pyridyl disulfide, methyl 2-pyridyl disulfide, p-chloromercuribenzoate, 2-<br>
chloromercuri-4-nitrophenol, orchloro-7-nitrobenzo-2-oxa-1,3-diazole and the like.<br>
[100]	Histidyl residues are derivatized by reaction with diethylpyrocarbonate at pH 5.5-7.0<br>
because this agent is relatively specific for the histidyl side chain.  Para-bromophenacyl bromide<br>
also is useful; the reaction is preferably performed in 0.1 M sodium cacodylate at pH 6.0.<br>
[101]	Lysinyl and amino terminal residues are reacted with succinic or other carboxylic acid<br>
anhydrides.  Derivatization with these agents has the effect of reversing the charge of the lysinyl<br>
22<br><br>
residues Other suitable reagents for derivatizing alpha-amino-containing residues include imidoesters such as methyl picolinimidate; pyridoxal phosphate; pyridoxal; chloroborohydride; trmitrobenzenesulfonic acid; O-methylisourea; 2,4-pentanedione; and transaminase-catalyzed reaction with glyoxylate.<br>
[102]	Arginyl residues are modified by reaction with one or several conventional reagents,<br>
among    them    phenylglyoxal,    2,3-butanedione,     1,2-cyclohexanedione,    and    ninhydrin.<br>
Derivatization of arginine residues requires that the reaction be performed in alkaline conditions<br>
because of the high pKa of the guanidine functional group.   Furthermore, these reagents may<br>
react with the groups of lysine as well as the arginine epsilon-amino group.<br>
[103]	The specific modification of tyrosyl residues may be made, with particular interest in<br>
introducing spectral labels into tyrosyl residues by reaction with aromatic diazonium compounds or tetranitromethane. Most commonly, N-acetylimidizole and tetranitromethane are used to form O-acetyl tyrosyl species and 3-nitro derivatives, respectively. Tyrosyl residues are iodinated using 1251 or 1311 to prepare labeled proteins for use in radioimmunoassay, the chloramine T method described above being suitable.<br>
[104]	Carboxyl side groups (aspartyl or glutamyl) are selectively modified by reaction with<br>
carbodiimides (R'—N=C=N-R'), where R and R' are optionally different alkyl groups, such as 1-cyclohexyl-3-(2-morpholinyl-4-ethyl) carbodiimide or 1-ethyl-3-(4-azonia-4,4-dimethylpentyl) carbodiimide. Furthermore, aspartyl and glutamyl residues are converted to asparaginyl and glutaminyl residues by reaction with ammonium ions.<br>
[105]	Derivatization with bifunctional agents is useful for crosslinking antibodies to a water-<br>
insoluble support matrix or surface for use in a variety of methods, in addition to methods described below. Commonly used crosslinking agents include, e.g., 1,1-bis(diazoacetyl)-2-phenylethane, glutaraldehyde, N-hydroxysuccinimide esters, for example, esters with 4-azidosalicylic acid, homobifunctional imidoesters, including disuccinimidyl esters such as 3,3'-dithiobis (succinimidylpropionate), and bifunctional maleimides such as bis-N-maleimido-1,8-octane. Derivatizing agents such as methyl-3-[(p-azidophenyl)dithio]propioimidate yield photoactivatable intermediates that are capable of forming crosslinks in the presence of light. Alternatively, reactive water-insoluble matrices such as cyanogen bromide-activated carbohydrates and the reactive substrates described in U.S. Pat. Nos. 3,969,287; 3,691,016; 4,195,128; 4,247,642; 4,229,537; and 4,330,440, all entirely incorporated by reference, are employed for protein immobilization.<br>
[106]	Glutaminyl and asparaginyl residues are frequently deamidated to the corresponding<br>
glutamyl and aspartyl residues, respectively. Alternatively, these residues are deamidated under mildly acidic conditions. Either form of these residues falls within the scope of this invention. Other modifications include hydroxylation of proline and lysine, phosphorylation of hydroxyl groups of seryl or threonyl residues, methylation of the a-amino groups of lysine, arginine, and<br>
23<br><br>
histidine side chains (T. E   Creighton, Proteins: Structure and Molecular Properties, W. H. Freeman &amp; Co , San Francisco, pp 79-86 (1983), entirely incorporated by reference), acetylation of the N-terminal amine, and amidation of any C-termmal carboxyl group. Glycosylation<br>
[107]	Another type of covalent modification is glycosylation    In another embodiment, the<br>
IgG variants disclosed herein can be modified to include one or more engineered glycoforms. By<br>
"engineered glycoform" as used herein is meant a carbohydrate composition that is covalently<br>
attached to an IgG, wherein said carbohydrate composition differs chemically from that of a<br>
parent IgG Engineered glycoforms may be useful for a variety of purposes, including but not<br>
limited to enhancing or reducing effector function. Engineered glycoforms may be generated by a<br>
variety of methods known in the art (Umaria et al., 1999, Nat Biotechnol 17:176-180; Davies et<br>
al., 2001, Biotechnol Bioeng 74:288-294; Shields et al., 2002, J Biol Chem 277:26733-26740;<br>
Shinkawa et al., 2003, J Biol Chem 278:3466-3473; US 6,602,684; USSN 10/277,370; USSN<br>
10/113,929; PCT WO 00/61739A1; PCT WO 01/29246A1; PCT WO 02/31140A1; PCT WO<br>
02/30954A1, all entirely incorporated by reference; (Potelligent™ technology [Biowa, Inc.,<br>
Princeton, NJ]; GlycoMAb® glycosylation engineering technology [Glycart Biotechnology AG,<br>
Zurich, Switzerland]). Many of these techniques are based on controlling the level of fucosylated<br>
and/or bisecting oligosaccharides that are covalently attached to the Fc region, for example by<br>
expressing an IgG in various organisms or cell lines, engineered or otherwise (for example Lec-<br>
13 CHO cells or rat hybridoma YB2/0 cells), by regulating enzymes involved in the glycosylation<br>
pathway (for example FUT8 [a1,6-fucosyltranserase] and/or £1-4- N-<br>
acetylglucosaminyltransferase III [GnTlil]), or by modifying 'carbohydrate(s) after the IgG has<br>
been expressed. Engineered glycoform typically refers to the different carbohydrate or<br>
oligosaccharide; thus an IgG variant, for example an antibody or Fc fusion, can include an<br>
engineered glycoform. Alternatively, engineered glycoform may refer to the IgG variant that<br>
comprises the different carbohydrate or oligosaccharide. As is known in the art, glycosylation<br>
patterns can depend on both the sequence of the protein (e.g., the presence or absence of<br>
particular glycosylation amino acid residues, discussed below), or the host cell or organism in<br>
which the protein is produced. Particular expression systems are discussed below.<br>
[108]	Glycosylation of polypeptides is typically either N-linked or O-linked.  N-linked refers<br>
to the attachment of the carbohydrate moiety to the side chain of an asparagine residue. The tri-peptide sequences asparagine-X-serine and asparagine-X-threonine, where X is any amino acid except proline, are the recognition sequences for enzymatic attachment of the carbohydrate moiety to the asparagine side chain. Thus, the presence of either of these tri-peptide sequences in a polypeptide creates a potential glycosylation site. O-linked glycosylation refers to the attachment of one of the sugars N-acetylgalactosamine, galactose, or xylose, to a hydroxyamino<br>
24<br><br>
acid, most commonly serine or threonine, although 5-hydroxyproline or 5-hydroxy lysine may also<br>
be used.<br>
[109]	Addition of glycosylation sites to the antibody is conveniently accomplished by<br>
altering the amino acid sequence such that it contains one or more of the above-described tri-peptide sequences (for N-linked glycosylation sites). The alteration may also be made by the addition of, or substitution by, one or more serine or threonine residues to the starting sequence (for O-linked glycosylation sites). For ease, the antibody amino acid sequence is preferably altered through changes at the DNA level, particularly by mutating the DNA encoding the target polypeptide at preselected bases such that codons are generated that will translate into the desired amino acids.<br>
[110]	Another means of increasing the number of carbohydrate moieties on the antibody is<br>
by chemical or enzymatic coupling of glycosides to the protein. These procedures are advantageous in that they do not require production of the protein in a host cell that has glycosylation capabilities for N- and O-linked glycosylation. Depending on the coupling mode used, the sugar(s) may be attached to (a) arginine and histidine, (b) free carboxyl groups, (c) free sulfhydryl groups such as those of cysteine, (d) free hydroxyl groups such as those of serine, threonine, or hydroxyproline, (e) aromatic residues such as those of phenylalanine, tyrosine, or tryptophan, or (f) the amide group of glutamine. These methods are described in WO 87/05330 and in Aplin and Wriston, 1981, CRC Crit. Rev. Biochem., pp. 259-306, both entirely incorporated by reference..<br>
[111]	Removal of carbohydrate  moieties  present on  the  starting  antibody  may  be<br>
accomplished chemically or enzymatically. Chemical deglycosylation requires exposure of the protein to the compound trifluoromethanesulfonic acid, or an equivalent compound. This treatment results in the cleavage of most or all sugars except the linking sugar (N-acetylglucosamine or N-acetylgalactosamine), while leaving the polypeptide intact. Chemical deglycosylation is described by Hakimuddin et al., 1987, Arch. Biochem. Biophys. 259:52 and by Edge et al., 1981, Anal. Biochem. 118:131, both entirely incorporated by reference. Enzymatic cleavage of carbohydrate moieties on polypeptides can be achieved by the use of a variety of endo- and exo-glycosidases as described by Thotakura et al., 1987, Meth. Enzymol. 138:350, entirely incorporated by reference. Glycosylation at potential glycosylation sites may be prevented by the use of the compound tunicamycin as described by Duskin et al., 1982, J. Biol. Chem. 257:3105, entirely incorporated by reference. Tunicamycin blocks the formation of protein-N-glycoside linkages.<br>
[112]	Another type of covalent modification of the antibody comprises linking the antibody<br>
to various nonproteinaceous polymers, including, but not limited to, various polyols such as polyethylene glycol, polypropylene glycol or polyoxyalkylenes, in the manner set forth in, for example, 2005-2006 PEG Catalog from Nektar Therapeutics (available at the Nektar website) US<br>
25<br><br>
Patents  4,640.835;  4,496,689;  4.301,144;  4,670,417;  4.791.192 or 4,179,337.   all  entirely incorporated by reference.   In addition, as is known in the art, amino acid substitutions may be made in various positions within the antibody to facilitate the addition of polymers such as PEG. See for example, U.S. Publication No. 2005/0114037A1, entirely incorporated by reference. Labeled Antibodies<br>
[113]	In some embodiments, the covalent modification of the antibodies of the invention<br>
comprises the addition of one or more labels. In some cases, these are considered antibody fusions The term "labelling group" means any detectable label. In some embodiments, the labelling group is coupled to the antibody via spacer arms of various lengths to reduce potential steric hindrance. Various methods for labelling proteins are known in the art and may be used in performing the present invention.<br>
[114]	In general, labels fall into a variety of classes, depending on the assay in which they<br>
are to be detected: a) isotopic labels, which may be radioactive or heavy isotopes; b) magnetic<br>
labels (eg, magnetic particles); c) redox active moieties; d) optical dyes; enzymatic groups (e.g.<br>
horseradish peroxidase, B-galactosidase, luciferase, alkaline phosphatase); e) biotinylated<br>
groups; and f) predetermined polypeptide epitopes recognized by a secondary reporter (e.g.,<br>
leucine zipper pair sequences, binding sites for secondary antibodies, metal binding domains,<br>
epitope tags, etc.). In some embodiments, the labelling group is coupled to the antibody via<br>
spacer arms of various lengths to reduce potential steric hindrance. Various methods for labelling<br>
proteins are known in the art and may be used in performing the present invention.<br>
[115]	Specific labels include optical dyes, including, but not limited to, chromophores,<br>
phosphors and fluorophores, with the latter being specific inTnany instances. Fluorophores can be either "small molecule" fluores, or proteinaceous fluores.<br>
[116]	By "fluorescent label" is meant any molecule that may be detected via its inherent<br>
fluorescent properties. Suitable fluorescent labels include, but are not limited to, fluorescein, rhodamine, tetramethylrhodamine, eosin, erythrosin, coumarin, methyl-coumarins, pyrene, Malacite green, stilbene, Lucifer Yellow, Cascade BlueJ, Texas Red, IAEDANS, EDANS, BODIPY FL, LC Red 640, Cy 5, Cy 5.5, LC Red 705, Oregon green, the Alexa-Fluor dyes (Alexa Fluor 350, Alexa Fluor 430, Alexa Fluor 488, Alexa Fluor 546, Alexa Fluor 568, Alexa Fluor 594, Alexa Fluor 633, Alexa Fluor 660, Alexa Fluor 680), Cascade Blue, Cascade Yellow and R-phycoerythrin (PE) (Molecular Probes, Eugene, OR), FITC, Rhodamine, and Texas Red (Pierce, Rockford, IL), Cy5, Cy5.5, Cy7 (Amersham Life Science, Pittsburgh, PA). Suitable optical dyes, including fluorophores, are described in Molecular Probes Handbook by Richard P. Haugland, entirely incorporated by reference.<br>
[117]	Suitable proteinaceous fluorescent labels also include, but are not limited to, green<br>
fluorescent protein, including a Renilla, Ptilosarcus, or Aequorea species of GFP (Chalfie et al., 1994, Science 263:802-805), EGFP (Clontech Laboratories, Inc., Genbank Accession Number<br>
26<br><br>
U55762). blue fluorescent protein (BFP. Quantum Biotechnologies. Inc. 1801 de Maisonneuve Blvd West 8th Floor. Montreal. Quebec. Canada H3H 1J9; Stauber. 1998, Biotechniques 24462-471. Heim et al., 1996, Curr Biol 6:178-182), enhanced yellow fluorescent protein (EYFP. Clontech Laboratories, Inc). luciferase (Ichiki et al., 1993, J. Immunol. 150:5408-5417), p galactosidase (Nolan et al.. 1988. Proc Natl Acad Sci U.S.A. 85:2603-2607) and Renilla (W092/15673, WO95/07463, WO98/14605, W098/26277, WO99/49019, U.S. Patent Nos. 5292658. 5418155, 5683888, 5741668, 5777079, 5804387, 5874304, 5876995, 5925558). All of the above-cited references in this paragraph are expressly incorporated herein by reference. IgG Variants<br>
[118]	In one embodiment, the invention provides variant IgG proteins.  At a minimum, IgG<br>
variants comprise an antibody fragment comprising the CH2-CH3 region of the heavy chain.  In<br>
addition, suitable IgG variants comprise Fc domains (e.g. including the lower hinge region), as<br>
well as IgG variants comprising the constant region of the heavy chain (CH1-hinge-CH2-CH3)<br>
also being useful in the present invention, all of which can be fused to fusion partners.<br>
[119]	An IgG variant includes one or more amino acid modifications relative to a parent IgG<br>
polypeptide, in some cases relative to the wild type IgG. The IgG variant can have one or more<br>
optimized properties. An IgG variant differs in amino acid sequence from its parent IgG by virtue<br>
of at least one amino acid modification. Thus IgG variants have at least one amino acid<br>
modification compared to the parent. Alternatively, the IgG variants may have more than one<br>
amino acid modification as compared to the parent, for example from about one to fifty amino<br>
acid modifications, preferably from about one to ten amino acid modifications, and most<br>
preferably from about one to about five amino acid modifications compared to the parent.<br>
[120]	Thus the sequences of the IgG variants and those of the parent Fc polypeptide are<br>
substantially homologous. For example, the variant IgG variant sequences herein will possess about 80% homology with the parent IgG variant sequence, preferably at least about 90% homology, and most preferably at least about 95% homology.    Modifications may be made genetically using molecular biology, or may be made enzymatically or chemically. Target Antigens for Antibodies<br>
[121]	Virtually any antigen may be targeted by the IgG variants, including but not limited to<br>
proteins, subunits, domains, motifs, and/or epitopes belonging to the following list of target antigens, which includes both soluble factors such as cytokines and membrane-bound factors, including transmembrane receptors: 17-1 A, 4-1BB, 4Dc, 6-keto-PGF1a, 8-iso-PGF2a, 8-oxo-dG, A1 Adenosine Receptor, A33, ACE, ACE-2, Activin, Activin A, Activin AB, Activin B, Activin C, Activin RIA, Activin RIA ALK-2, Activin RIB ALK-4, Activin RIIA, Activin RUB, ADAM, ADAM10, ADAM12, ADAM15, ADAM17/TACE, ADAM8, ADAM9, ADAMTS, ADAMTS4, ADAMTS5, Addressins, aFGF, ALCAM, ALK, ALK-1, ALK-7, alpha-1-antitrypsin, alpha-V/beta-1 antagonist, ANG, Ang, APAF-1, APE, APJ, APP, APRIL, AR, ARC, ART, Artemin, anti-Id, ASPARTIC, Atrial<br>
27<br><br>
natriuretic factor, av/b3 integrin, Axl, b2M, B7-1, B7-2. B7-H, B-lymphocyte Stimulator (BryS), BACE, BACE-1, Bad, BAFF, BAFF-R, Bag-1, BAK. Bax. BCA-1. BCAM. Bel. BCMA, BDNF. b-ECGF, bFGF, BID, Bik, BIM, BLC, BL-CAM, BLK, BMP. BMP-2 BMP-2a. BMP-3 Osteogenin, BMP-4 BMP-2b, BMP-5, BMP-6 Vgr-1, BMP-7 (OP-1). BMP-8 (BMP-8a, OP-2), BMPR, BMPR-IA (ALK-3), BMPR-IB (ALK-6), BRK-2, RPK-1, BMPR-II (BRK-3). BMPs. b-NGF. BOK. Bombesin. Bone-derived neurotrophic factor, BPDE, BPDE-DNA, BTC, complement factor 3 (C3), C3a, C4, C5, C5a, C10, CA125, CAD-8, Calcitonin, cAMP, carcinoembryonic antigen (CEA), carcinoma-associated antigen, Cathepsin A, Cathepsin B, Cathepsin C/DPPI. Cathepsin D. Cathepsin E. Cathepsin H, Cathepsin L, Cathepsin O, Cathepsin S, Cathepsin V, Cathepsin X/Z/P, CBL, CCI, CCK2, CCL, CCL1, CCL11, CCL12, CCL13, CCL14, CCL15, CCL16, CCL17. CCL18, CCL19, CCL2, CCL20, CCL21, CCL22, CCL23, CCL24, CCL25, CCL26, CCL27, CCL28, CCL3, CCL4, CCL5, CCL6, CCL7, CCL8, CCL9/10, CCR, CCR1, CCR10, CCR10, CCR2, CCR3, CCR4, CCR5, CCR6, CCR7, CCR8, CCR9, CD1, CD2, CD3, CD3E, CD4, CD5, CD6, CD7, CD8, CD10, CD11a, CD11b, CD11c, CD13, CD14, CD15, CD16, CD18. CD19, CD20, CD21, CD22, CD23, CD25, CD27L, CD28, CD29, CD30, CD30L, CD32, CD33 (p67 proteins), CD34, CD38, CD40, CD40L, CD44, CD45, CD46, CD49a, CD52, CD54, CD55, CD56, CD61, CD64, CD66e, CD74, CD80 (B7-1), CD89, CD95, CD123, CD137, CD138, CD140a, CD146, CD147, CD148, CD152, CD164, CEACAM5, CFTR, cGMP, CINC, Clostridium botulinum toxin, Clostridium perfringens toxin, CKb8-1, CLC, CMV, CMV UL, CNTF, CNTN-1, COX, C-Ret, CRG-2, CT-1, CTACK, CTGF, CTLA-4, CX3CL1, CX3CR1, CXCL, CXCL1, CXCL2. CXCL3, CXCL4, CXCL5, CXCL6, CXCL7, CXCL8, CXCL9, CXCL10, CXCL11, CXCL12, CXCL13, CXCL14, CXCL15, CXCL16, CXCR, CXCR1, CXCR2, CXCR3, CXCR4, CXCR5, CXCR6, cytokeratin tumor-associated antigen, DAN, DCC, DcR3, DC-SIGN, Decay accelerating factor, des(1-3)-IGF-l (brain IGF-1), Dhh, digoxin, DNAM-1, Dnase, Dpp, DPPIV/CD26, Dtk, ECAD, EDA. EDA-A1, EDA-A2, EDAR, EGF, EGFR (ErbB-1), EMA, EMMPRIN, ENA, endothelin receptor, Enkephalinase, eNOS, Eot, eotaxinl, EpCAM, Ephrin B2/ EphB4, EPO, ERCC, E-selectin. ET-1, Factor Ha, Factor VII, Factor Vlllc, Factor IX, fibroblast activation protein (FAP), Fas, FcR1, FEN-1, Ferritin, FGF, FGF-19, FGF-2, FGF3, FGF-8, FGFR, FGFR-3, Fibrin, FL, FLIP, Flt-3, Flt-4, Follicle stimulating hormone, Fractalkine, FZD1, FZD2, FZD3, FZD4, FZD5, FZD6. FZD7, FZD8, FZD9, FZD10, G250, Gas 6, GCP-2, GCSF, GD2, GD3, GDF, GDF-1, GDF-3 (Vgr-2), GDF-5 (BMP-14, CDMP-1), GDF-6 (BMP-13, CDMP-2), GDF-7 (BMP-12, CDMP-3), GDF-8 (Myostatin), GDF-9, GDF-15 (MIC-1), GDNF, GDNF, GFAP, GFRa-1, GFR-alpha1, GFR-alpha2, GFR-alpha3, GITR, Glucagon, Glut 4, glycoprotein llb/llla (GP llb/llla), GM-CSF, gp130, gp72, GRO, Growth hormone releasing factor, Hapten (NP-cap or NIP-cap), HB-EGF, HCC, HCMV gB envelope glycoprotein, HCMV) gH envelope glycoprotein, HCMV UL, Hemopoietic growth factor (HGF), Hep B gp120, heparanase, Her2, Her2/neu (ErbB-2), Her3 (ErbB-3), Her4 (ErbB-4), herpes simplex virus (HSV) gB glycoprotein, HSV gD glycoprotein, HGFA, High molecular weight melanoma-associated antigen<br>
28<br><br>
(HMW-MAA), HIV gp120, HIV IIIB gp 120 V3 loop, HLA, HLA-DR, HM1.24, HMFG PEM, HRG, Hrk, human cardiac myosin, human cytomegalovirus (HCMV), human growth hormone (HGH), HVEM, I-309, IAP, ICAM, ICAM-1, ICAM-3, ICE, ICOS, IFNg, Ig, IgA receptor, IgE, IGF, IGF binding proteins, IGF-1R, IGFBP, IGF-I, IGF-II, IL, IL-1, IL-1R, IL-2, IL-2R, IL-4, IL-4R, IL-5, IL-5R, IL-6, IL-6R, IL-8, IL-9, IL-10, IL-12, IL-13, IL-15, IL-18, IL-18R, IL-23, interferon (INF)-alpha, INF-beta, INF-gamma, Inhibin, iNOS, Insulin A-chain, Insulin B-chain, Insulin-like growth factor 1, integrin alpha2, integrin alpha3, integrin alpha4, integrin alpha4/beta1, integrin alpha4/beta7, integrin alpha5 (alphaV), integrin alpha5/beta1, integrin alpha5/beta3, integrin alpha6, integrin betal, integrin beta2, interferon gamma, IP-10, l-TAC, JE, Kallikrein 2, Kallikrein 5, Kallikrein 6, , Kallikrein 11, Kallikrein 12, Kallikrein 14, Kallikrein 15, Kallikrein L1, Kallikrein L2, Kallikrein L3, Kallikrein L4 KC, KDR, Keratinocyte Growth Factor (KGF), laminin 5, LAMP, LAP, LAP (TGF- 1), Latent TGF-1, Latent TGF-1 bp1, LBP, LDGF, LECT2, Lefty, Lewis-Y antigen, Lewis-Y related antigen, LFA-1, LFA-3, Lfo, LIF, LIGHT, lipoproteins, LIX, LKN, Lptn, L-Selectin, LT-a, LT-b, LTB4, LTBP-1, Lung surfactant, Luteinizing hormone, Lymphotoxin Beta Receptor, Mac-1, MAdCAM, MAG, MAP2, MARC, MCAM, MCAM, MCK-2, MCP, M-CSF, MDC, Mer, METALLOPROTEASES, MGDF receptor, MGMT, MHC (HLA-DR), MIF, MIG, MIP, MIP-1-alpha, MK, MMAC1, MMP, MMP-1, MMP-10, MMP-11, MMP-12, MMP-13, MMP-14, MMP-15, MMP-2, MMP-24, MMP-3, MMP-7, MMP-8, MMP-9, MPIF, Mpo, MSK, MSP, mucin (Mud), MUC18, Muellerian-inhibitin substance, Mug, MuSK, NAIP, NAP, NCAD, N-Cadherin, NCA 90, NCAM, NCAM, Neprilysin, Neurotrophin-3,-4, or -6, Neurturin, Neuronal growth factor (NGF), NGFR, NGF-beta, nNOS, NO, NOS, Npn, NRG-3, NT, NTN, OB, OGG1, OPG, OPN, OSM, OX40L, OX40R, p150, p95, PADPr, Parathyroid hormone, PARC, PARP, PBR, PBSF, PCAD, P-Cadherin, PCNA, PDGF, PDGF, PDK-1, PECAM, PEM, PF4, PGE, PGF, PGI2, PGJ2, PIN, PLA2, placental alkaline phosphatase (PLAP), PIGF, PLP, PP14, Proinsulin, Prorelaxin, Protein C, PS, PSA, PSCA, prostate specific membrane antigen (PSMA), PTEN, PTHrp, Ptk, PTN, R51, RANK, RANKL, RANTES, RANTES, Relaxin A-chain, Relaxin B-chain, renin, respiratory syncytial virus (RSV) F, RSV Fgp, Ret, Rheumatoid factors, RLIP76, RPA2, RSK, S100, SCF/KL, SDF-1, SERINE, Serum albumin, sFRP-3, Shh, SIGIRR, SK-1, SLAM, SLPI, SMAC, SMDF, SMOH, SOD, SPARC, Stat, STEAP, STEAP-II, TACE, TACI, TAG-72 (tumor-associated glycoprotein-72), TARC, TCA-3, T-cell receptors (e.g., T-cell receptor alpha/beta), TdT, TECK, TEM1, TEM5. TEM7, TEM8, TERT, testicular PLAP-like alkaline phosphatase, TfR, TGF, TGF-alpha, TGF-beta, TGF-beta Pan Specific, TGF-beta Rl (ALK-5), TGF-beta Rll, TGF-beta Rllb, TGF-beta Rill, TGF-betal, TGF-beta2, TGF-beta3, TGF-beta4, TGF-beta5, Thrombin, Thymus Ck-1, Thyroid stimulating hormone, Tie, TIMP, TIQ, Tissue Factor, TMEFF2, Tmpo, TMPRSS2, TNF, TNF-alpha, TNF-alpha beta, TNF-beta2, TNFc, TNF-RI, TNF-RII, TNFRSF10A (TRAIL R1 Apo-2, DR4), TNFRSF10B (TRAIL R2 DR5, KILLER, TRICK-2A, TRICK-B), TNFRSF10C (TRAIL R3 DcR1, LIT, TRID), TNFRSF10D (TRAIL R4 DcR2, TRUNDD), TNFRSF11A (RANK ODF R,<br>
29<br><br>
TRANCE R), TNFRSF11B (OPG OCIF. TR1). TNFRSF12 (TWEAK R FN14), TNFRSF13B (TACI). TNFRSF13C (BAFF R). TNFRSF14 (HVEM ATAR, HveA, LIGHT R, TR2), TNFRSF16 (NGFR p75NTR), TNFRSF17 (BCMA), TNFRSF18 (GITR AITR), TNFRSF19 (TROY TAJ, TRADE) TNFRSF19L (REIT), TNFRSF1A (TNF Rl CD120a, p55-60), TNFRSF1B (TNF Rll CD120b p75-80). TNFRSF26 (TNFRH3). TNFRSF3 (LTbR TNF Rill, TNFC R), TNFRSF4 (OX40 ACT35. TXGP1 R), TNFRSF5 (CD40 p50). TNFRSF6 (Fas Apo-1, APT1, CD95), TNFRSF6B (DcR3 M68, TR6), TNFRSF7 (CD27), TNFRSF8 (CD30), TNFRSF9 (4-1BB CD137, ILA), TNFRSF21 (DR6), TNFRSF22 (DcTRAIL R2 TNFRH2), TNFRST23 (DcTRAIL R1 TNFRH1), TNFRSF25 (DR3 Apo-3. LARD. TR-3, TRAMP. WSL-1), TNFSF10 (TRAIL Apo-2 Ligand, TL2), TNFSF11 (TRANCE/RANK Ligand ODF, OPG Ligand), TNFSF12 (TWEAK Apo-3 Ligand, DR3 Ligand) TNFSF13 (APRIL TALL2), TNFSF13B (BAFF BLYS, TALL1, THANK, TNFSF20), TNFSF14 (LIGHT HVEM Ligand, LTg), TNFSF15 (TL1AA/EGI), TNFSF18 (GITR Ligand AITR Ligand. TL6) TNFSF1A (TNF-a Conectin. DIF. TNFSF2), TNFSF1B (TNF-b LTa, TNFSF1), TNFSF3 (LTb TNFC. p33), TNFSF4 (OX40 Ligand gp34, TXGP1), TNFSF5 (CD40 Ligand CD154, gp39, HIGM1, IMD3, TRAP), TNFSF6 (Fas Ligand Apo-1 Ligand, APT1 Ligand), TNFSF7 (CD27 Ligand CD70), TNFSF8 (CD30 Ligand CD153), TNFSF9 (4-1 BB Ligand CD137 Ligand), TP-1, t-PA, Tpo, TRAIL, TRAIL R, TRAIL-R1, TRAIL-R2, TRANCE, transferring receptor, TRF, Trk, TROP-2, TSG, TSLP, tumor-associated antigen CA 125, tumor-associated antigen expressing Lewis Y related carbohydrate, TWEAK, TXB2, Ung, uPAR, uPAR-1, Urokinase, VCAM, VCAM-1, VECAD, VE-Cadherin, VE-cadherin-2, VEFGR-1 (flt-1), VEGF, VEGFR, VEGFR-3 (flt-4), VEGI, VIM, Viral antigens, VLA, VLA-1, VLA-4, VNR integrin, von Willebrands factor, WIF-1, WNT1, WNT2, WNT2B/13, WNT3, WNT3A,'WNT4, WNT5A, WNT5B, WNT6, WNT7A, WNT7B, WNT8A, WNT8B, WNT9A, WNT9A, WNT9B, WNT10A, WNT10B, WNT11, WNT16, XCL1, XCL2, XCR1, XCR1, XEDAR, XIAP, XPD, and receptors for hormones and growth factors.<br>
[122]	One skilled in the art will appreciate that the aforementioned list of targets refers not<br>
only to specific proteins and biomolecules, but the biochemical pathway or pathways that comprise them. For example, reference to CTLA-4 as a target antigen implies that the ligands and receptors that make up the T cell co-stimulatory pathway, including CTLA-4, B7-1, B7-2, CD28, and any other undiscovered ligands or receptors that bind these proteins, are also targets. Thus target as used herein refers not only to a specific biomolecule, but the set of proteins that interact with said target and the members of the biochemical pathway to which said target belongs. One skilled in the art will further appreciate that any of the aforementioned target antigens, the ligands or receptors that bind them, or other members of their corresponding biochemical pathway, may be operably linked to the Fc variants of the present invention in order to generate an Fc fusion. Thus for example, an Fc fusion that targets EGFR could be constructed by operably linking an Fc variant to EGF, TGF-b, or any other ligand, discovered or undiscovered,<br>
30<br><br>
that binds EGFR Accordingly, an Fc variant of the present invention could be operably linked to EGFR in order to generate an Fc fusion that binds EGF, TGF-b. or any other ligand, discovered or undiscovered, that binds EGFR. Thus virtually any polypeptide, whether a ligand, receptor, or some other protein or protein domain, including but not limited to the aforementioned targets and the proteins that compose their corresponding biochemical pathways, may be operably linked to the Fc variants of the present invention to develop an Fc fusion.<br>
[123]	The choice of suitable antigen depends on the desired application.  For anti-cancer<br>
treatment it is desirable to have a target whose expression is restricted to the cancerous cells. Some targets that have proven especially amenable to antibody therapy are those with signaling functions. Other therapeutic antibodies exert their effects by blocking signaling of the receptor by inhibiting the binding between a receptor and its cognate ligand. Another mechanism of action of therapeutic antibodies is to cause receptor down regulation. Other antibodies do not work by signaling through their target antigen. In some cases, antibodies directed against infectious disease agents are used.<br>
[124]	In one embodiment, the Fc variants of the present invention are incorporated into an<br>
antibody against a cytokine. Alternatively, the Fc variants are fused or conjugated to a cytokine. By "cytokine" as used herein is meant a generic term for proteins released by one cell population that act on another cell as intercellular mediators. For example, as described in Penichet et al., 2001, J Immunol Methods 248:91-101, expressly incorporated by reference, cytokines may be fused to antibody to provide an array of desirable properties. Examples of such cytokines are lymphokines, monokines, and traditional polypeptide hormones. Included among the cytokines are growth hormone such as human growth hormone, N-mefhionyl human growth hormone, and bovine growth hormone; parathyroid hormone; thyroxine; insulin; proinsulin; relaxin; prorelaxin; glycoprotein hormones such as follicle stimulating hormone (FSH), thyroid stimulating hormone (TSH), and luteinizing hormone (LH); hepatic growth factor; fibroblast growth factor; prolactin; placental lactogen; tumor necrosis factor-alpha and -beta; mullerian-inhibiting substance; mouse gonadotropin-associated peptide; inhibin; activin; vascular endothelial growth factor; integrin; thrombopoietin (TPO); nerve growth factors such as NGF-beta; platelet-growth factor; transforming growth factors (TGFs) such as TGF-alpha and TGF-beta; insulin-like growth factor-l and -II; erythropoietin (EPO); osteoinductive factors; interferons such as interferon-alpha, beta, and -gamma; colony stimulating factors (CSFs) such as macrophage-CSF (M-CSF); granulocyte-macrophage-CSF (GM-CSF); and granulocyte-CSF (G-CSF); interleukins (ILs) such as IL-1, IL-lalpha, IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, IL-8, IL-9, IL-10, IL-11, IL-12; IL-15, a tumor necrosis factor such as TNF-alpha or TNF-beta; C5a; and other polypeptide factors including LIF and kit ligand (KL). As used herein, the term cytokine includes proteins from natural sources or from recombinant cell culture, and biologically active equivalents of the native sequence cytokines.<br>
31<br><br>
[125]	Cytokines and soluble targets, such as TNF superfamily members, are preferred<br>
targets for use with the variants of the present invention. For example, anti-VEGF, anti-CTLA-4, and anti-TNF antibodies, or fragments thereof, are particularly good antibodies for the use of Fc variants that increase the FcRn binding. Therapeutics against these targets are frequently involved in the treatment of autoimmune diseases and require multiple injections over long time periods. Therefore, longer serum half-lives and less frequent treatments, brought about from the variants of the present invention, are particularly preferred.<br>
[126]	A number of antibodies and Fc fusions that are approved for use, in clinical trials, or<br>
in development may benefit from the Fc variants of the present invention. These antibodies and Fc fusions are herein referred to as "clinical products and candidates". Thus in a preferred embodiment, the Fc polypeptides of the present invention may find use in a range of clinical products and candidates. For example, a number of antibodies that target CD20 may benefit from the Fc polypeptides of the present invention. For example the Fc polypeptides of the present invention may find use in an antibody that is substantially similar to rituximab (Rituxan®, IDEC/Genentech/Roche) (see for example US 5,736,137), a chimeric anti-CD20 antibody approved to treat Non-Hodgkin's lymphoma; HuMax-CD20, an anti-CD20 currently being developed by Genmab, an anti-CD20 antibody described in US 5,500,362, AME-133 (Applied Molecular Evolution), hA20 (Immunomedics, Inc.), HumaLYM (Intracel), and PRO70769 (PCT/US2003/040426, entitled "Immunoglobulin Variants and Uses Thereof). A number of antibodies that target members of the family of epidermal growth factor receptors, including EGFR (ErbB-1), Her2/neu (ErbB-2), Her3 (ErbB-3), Her4 (ErbB-4), may benefit from the Fc polypeptides of the present invention. For example the Fc polypeptides of the present invention may find use in an antibody that is substantially similar to trastuzumab (Herceptin®, Genentech) (see for example US 5,677,171), a humanized anti-Her2/neu antibody approved to treat breast cancer; pertuzumab (rhuMab-2C4, Omnitarg™), currently being developed by Genentech; an anti-Her2 antibody described in US 4,753,894; cetuximab (Erbitux®, Imclone) (US 4,943,533; PCT WO 96/40210), a chimeric anti-EGFR antibody in clinical trials for a variety of cancers; ABX-EGF (US 6,235,883), currently being developed by Abgenix-lmmunex-Amgen; HuMax-EGFr (USSN 10/172,317), currently being developed by Genmab; 425, EMD55900, EMD62000, and EMD72000 (Merck KGaA) (US 5,558,864; Murthy et al. 1987, Arch Biochem Biophys. 252(2):549-60; Rodeck et al., 1987, J Cell Biochem. 35(4):315-20; Kettleborough et al., 1991, Protein Eng 4(7):773-83); ICR62 (Institute of Cancer Research) (PCT WO 95/20045; Modjtahedi et al., 1993, J Cell Biophys. 1993, 22(1-3):129-46; Modjtahedi et al., 1993, Br J Cancer. 1993, 67(2):247-53; Modjtahedi et al, 1996, Br J Cancer, 73(2):228-35; Modjtahedi et al, 2003, Int J Cancer, 105(2)273-80); TheraCIM hR3 (YM Biosciences, Canada and Centra de Immunologia Molecular, Cuba (US 5,891,996; US 6, 506,883; Mateo et al, 1997, Immunotechnology, 3(1):71-81); mAb-806 (Ludwig Institue for Cancer Research, Memorial Sloan-Kettering) (Jungbluth et al.<br>
32<br><br>
2003, Proc Natl Acad Sci USA 100{2):639-44); KSB-102 (KS Biomedix); MR1-1 (IVAX, National Cancer institute\(PCT WO 0162931A2); and SC100 (Scancell) (PCT WO 01/88138). In another preferred embodiment, the Fc polypeptides of the present invention may find use in aiemtuzumab (Campath® Millenium), a humanized monoclonal antibody currently approved for treatment of B-cell chrome lymphocytic leukemia The Fc polypeptides of the present invention may find use in a variety of antibodies or Fc fusions that are substantially similar to other clinical products and candidates, including but not limited to muromonab-CD3 (Orthoclone OKT3®), an anti-CD3 antibody developed by Ortho Biotech/Johnson &amp; Johnson, ibritumomab tiuxetan (Zevalin®), an anti-CD20 antibody developed by IDEC/Schering AG, gemtuzumab ozogamicin (Mylotarg®), an anti-CD33 (pG7 protein) antibody developed by Celltech/Wyeth, alefacept (Amevive®), an anti-LFA-3 Fc fuson developed by Biogen), abciximab (ReoPro®), developed by Centocor/Lilly, basiliximab (Simulect®), developed by Novartis, palivizumab (Synagis®), developed by Medlmmune infliximab (Remicade®), an anti-TNFalpha antibody developed by Centocor, adalimumab (Humira®), an anti-TNFalpha antibody developed by Abbott, Humicade™, an anti-TNFalpha antibody developed by Celltech, etanercept (Enbrel®), an anti-TNFalpha Fc fusion developed by Immunex/Amgen, ABX-CBL, an anti-CD147 antibody being developed by Abgenix, ABX-IL8. an anti-IL8 antibody being developed by Abgenix, ABX-MA1, an anti-MUCl8 antibody being developed by Abgenix, Pemtumomab (R1549, 90Y-muHMFG1), an anti-MUC1 In development by Antisoma, Therex (R1550), an anti-MUC1 antibody being developed by Antisoma, AngioMab (AS1405), being developed by Antisoma, HuBC-1, being developed by Antisoma, Thioplatin (AS1407) being developed by Antisoma, Antegren® (natalizumab), an anti-alpha-4-beta-1 (VLA-4) and alpha-4-beta-7 antibody being developed by Biogen, VLA-1 mAb, an anti-VLA-1 integrin antibody being developed by Biogen, LTBR mAb, an anti-lymphotoxin beta receptor (LTBR) antibody being developed by Biogen, CAT-152, an anti-TGF-B2 antibody being developed by Cambridge Antibody Technology, J695, an anti-IL-12 antibody being developed by Cambridge Antibody Technology and Abbott, CAT-192, an anti-TGFB1 antibody being developed by Cambridge Antibody Technology and Genzyme, CAT-213, an anti-Eotaxinl antibody being developed by Cambridge Antibody Technology, LymphoStat-B™ an anti-Blys antibody being developed by Cambridge Antibody Technology and Human Genome Sciences Inc., TRAIL-R1mAb, an anti-TRAIL-R1 antibody being developed by Cambridge Antibody Technology and Human Genome Sciences, Inc., Avastin™ (bevacizumab, rhuMAb-VEGF), an anti-VEGF antibody being developed by Genentech, an anti-HER receptor family antibody being developed by Genentech Anti-Tissue Factor (ATF), an anti-Tissue Factor antibody being developed by Genentech, Xolair™ (Omalizumab), an anti-lgE antibody being developed by Genentech, Raptiva™ (Efalizumab). an anti-CD11a antibody being developed by Genentech and Xoma, MLN-02 Antibody (formerly LDP-02), being developed by Genentech and Millenium Pharmaceuticals, HuMax CD4, an anti-CD4 antibody being developed by Genmab, HuMax-IL15,<br>
33<br><br>
an anti-lL 15 antibody being developed by Genmab and Amgen, HuMax-lnflam. being developed<br>
by Genmab and Medarex, HuMax-Cancer, an anti-Heparanase I antibody being developed by<br>
Genmab and Medarex and Oxford GcoSciences. HuMax-Lymphoma. being developed by<br>
Genmab and Amgen, HuMax-TAC, being developed by Genmab. IDEC-131. and anti-CD40L<br>
antibody being developed by IDEC Pharmaceuticals. IDEC-151 (Clenoliximab), an anti-CD4<br>
antibody being developed by IDEC Pharmaceuticals, IDEC-114, an anti-CD80 antibody being<br>
developed by IDEC Pharmaceuticals, IDEC-152, an anti-CD23 being developed by IDEC<br>
Pharmaceuticals anti-macrophage migration factor (MIF) antibodies being developed by IDEC<br>
Pharmaceuticals, BEC2. an anti-idiotypic antibody being developed by Imclone, IMC-1C11, an<br>
anti-KDR antibody being developed by Imclone, DC101, an anti-flk-1 antibody being developed<br>
by Imclone, anti-VE cadherin antibodies being developed by Imclone, CEA-Cide™<br>
(labetuzumab). an anti-carcinoembryonic antigen (CEA) antibody being developed by<br>
Immunomedics. LymphoCide™ (Epratuzumab), an anti-CD22 antibody being developed by<br>
Immunomedics, AFP-Cide, being developed by Immunomedics, MyelomaCide, being developed<br>
by Immunomedics, LkoCide, being developed by Immunomedics, ProstaCide, being developed<br>
by Immunomedics, MDX-010, an anti-CTLA4 antibody being developed by Medarex, MDX-060,<br>
an anti-CD30 antibody being developed by Medarex, MDX-070 being developed by Medarex,<br>
MDX-018 being developed by Medarex, Osidem™ (IDM-1), and anti-Her2 antibody being<br>
developed by Medarex and Immuno-Designed Molecules, HuMax™-CD4, an anti-CD4 antibody<br>
being developed by Medarex and Genmab, HuMax-IL15, an anti-IL15 antibody being developed<br>
by Medarex and Genmab, CNTO 148, an anti-TNFa antibody being developed by Medarex and<br>
Centocor/J&amp;J, CNTO 1275, an anti-cytokine antibody being developed by Centocor/J&amp;J,<br>
MOR101 and MOR102, anti-intercellular adhesion molecule-1 (ICAM-1) (CD54) antibodies being<br>
developed by MorphoSys, MOR201, an anti-fibroblast growth factor receptor 3 (FGFR-3)<br>
antibody being developed by MorphoSys, Nuvion® (visilizumab), an anti-CD3 antibody being<br>
developed by Protein Design Labs, HuZAF™, an anti-gamma interferon antibody being<br>
developed by Protein Design Labs, Anti-a5pi Integrin, being developed by Protein Design Labs,<br>
anti-IL-12, being developed by Protein Design Labs, ING-1, an anti-Ep-CAM antibody being<br>
developed by Xoma, and MLN01, an anti-Beta2 integrin antibody being developed by Xoma, all of<br>
the above-cited references in this paragraph are expressly incorporated herein by reference.<br>
[127]	The  Fc  polypeptides  of the  present invention  may  be  incorporated  into the<br>
aforementioned clinical candidates and products, or into antibodies and Fc fusions that are substantially similar to them. The Fc polypeptides of the present invention may be incorporated into versions of the aforementioned clinical candidates and products that are humanized, affinity matured, engineered, or modified in some other way.<br>
[128]	In one embodiment, the Fc polypeptides of the present invention are used for the<br>
treatment of autoimmune, inflammatory, or transplant indications. Target antigens and clinical<br>
34<br><br>
products and candidates that are relevant for such diseases include but are not limited to anti-<br>
a4p7 integrin antibodies such as LDP-02, anti-beta2 integrin antibodies such as LDP-01, anti-<br>
complement (C5) antibodies such as 5G1.1, anti-CD2 antibodies such as BTI-322, MEDI-507,<br>
anti-CD3 antibodies such as OKT3, SMART anti-CD3, anti-CD4 antibodies such as IDEC-151,<br>
MDX-CD4, OKT4A, anti-CD11a antibodies, anti-CD14 antibodies such as IC14, anti-CD18<br>
antibodies anti-CD23 antibodies such as IDEC 152, anti-CD25 antibodies such as Zenapax, anti-<br>
CD40L antibodies such as 5c8, Antova, IDEC-131, anti-CD64 antibodies such as MDX-33, anti-<br>
CD80 antibodies such as IDEC-114, anti-CD147 antibodies such as ABX-CBL, anti-E-selectin<br>
antibodies such as CDP850, anti-gpllb/llla antibodies such as ReoPro/Abcixima, anti-ICAM-3<br>
antibodies such as ICM3, anti-ICE antibodies such as VX-740, anti-FcR1 antibodies such as<br>
MDX-33, anti-lgE antibodies such as rhuMab-E25, anti-IL-4 antibodies such as SB-240683, anti-<br>
IL-5 antibodies such as SB-240563, SCH55700, anti-IL-8 antibodies such as ABX-IL8, anti-<br>
interferon gamma antibodies, anti-TNF (TNF, TNFa, TNFa, TNF-alpha) antibodies such as<br>
CDP571. CDP870, D2E7, Infliximab, MAK-195F, and anti-VLA-4 antibodies such as Antegren.<br>
[129]	Fc variants of the present invention such as those with increased binding to FcRn<br>
may be utilized in TNF inhibitor molecules to provide enhanced properties. Useful TNF inhibitor molecules include any molecule that inhibits the action of TNF-alpha in a mammal. Suitable examples include the Fc fusion Enbrel® (etanercept) and the antibodies Humira® (adalimumab) and Remicade® (infliximab). Monoclonal antibodies (such as Remicade and Humira) engineered using the Fc variants of the present invention to increase FcFn binding, may translate to better efficacy through an increased half-life.<br>
[130] In some embodiments, antibodies against infectious diseases are used. Antibodies against eukaryotic cells include antibodies targeting yeast cells, including but not limited to Saccharomyces cerevisiae, Hansenula polymorpha, Kluyveromyces frag His and K. lactis, Pichia guillerimondii and P. pastoris, Schizosaccharomyces pombe, Plasmodium falciparium, and Yarrowia lipolytica.<br>
[131] Antibodies against additional fungal cells are also useful, including target antigens associated with Candida strains including Candida glabrata, Candida albicans, C. krusei, C. /us/fan/aeand C. maltosa, as well as species of Aspergillus, Cryptococcus, Histoplasma, Coccidioides, Blastomyces, and Penicillium, among others<br>
[132] Antibodies directed against target antigens associated with protozoa include, but are not limited to, antibodies associated with Trypanosoma , Leishmania species including Leishmania donovanii; , Plasmodium spp., Pneumocystis carinii, Cryptosporidium parvum, Giardia lamblia, Entamoeba histolytica, and Cyclospora cayetanensis.<br>
[133] Antibodies against prokaryotic antigens are also useful, including antibodies against suitable bacteria such as pathogenic and non-pathogenic prokaryotes including but not limited to Bacillus, including Bacillus anthracis;  Vibrio, e.g. V. cholerae; Escherichia, e.g. Enterotoxigenic<br>
35<br><br>
E coli Shigella, eg S. dysenteriae; Salmonella, e.g. S. typhi; Mycobacterium e.g. M. tuberculosis ,V leprae, Clostridium, e.g. C. botulinum, C. tetani, C. difficile, C.perfringens; Cornyebacter.um, e.g. C. diphtheriae; Streptococcus, S. pyogenes, S. pneumoniae; Staphylococcus e.g S. aureus, Haemophilus, e.g. H. influenzae; Neisseria, e.g. N. meningitidis, N. gonorrhoeae Yersinia, e.g. Y. lamblia, Y. pestis, Pseudomonas, e.g. P. aeruginosa, P. putida; Chlamydia e c C trachomatis; Bordetella, e.g. 6. pertussis; Treponema, e.g. T. palladium; B. anthracis Y testis, Brucella spp., F. tularensis, B. mallei, B .pseudomallei, B. mallei, B.pseudomai'f' C botulinum , Salmonella spp., SEB V. cholerae toxin B, E. coli 0157:H7, Listeria spp nchosporon beigelii, Rhodotorula species, Hansenula anomala, Enterobacter sp., Klebsiella sp   Listeria sp., Mycoplasma sp.and the like.<br>
[134] in sore aspects, the antibodies are directed against viral infections; these viruses include, but are not limited to, including orthomyxoviruses, (e.g. influenza virus), paramyxoviruses (eg respiratc/ syncytial virus, mumps virus, measles virus), adenoviruses, rhinoviruses, coronaviruses reoviruses, togaviruses (e.g. rubella virus), parvoviruses, poxviruses (e.g. variola virus, vaccinia virus), enteroviruses (e.g. poliovirus, coxsackievirus), hepatitis viruses (including A, B and C), herpesviruses (e.g. Herpes simplex virus, varicella-zoster virus, cytomegalovirus, Epstein-Barr virus), rotaviruses, Norwalk viruses, hantavirus, arenavirus, rhabdovirus (e.g. rabies virus), retroviruses (including HIV, HTLV-I and -II), papovaviruses (e.g. papillomavirus), polyomaviruses and picomaviruses, and the like.<br>
Optimized IgG Variant Properties<br>
[135]	The present application also provides IgG variants that are optimized for a variety of<br>
therapeutically relevant properties. An IgG variant that is engineered or predicted to display one or more optimized properties is herein referred to as an "optimized IgG variant". The most preferred properties that may be optimized include but are not limited to enhanced or reduced affinity for a FcRn and increased or decreased in vivo half-life. Suitable embodiments include antibodies that exhibit increased binding affinity to FcRn at lowered pH, such as the pH associated with endosomes, e.g. pH 6.0, while not displaying corresponding increased binding affinity at higher pH, such as 7.4., to allow increased uptake into endosomes but normal release rates. Similarly, these antibodies with modulated FcRn binding may optionally have other desirable properties, such as modulated FcyR binding, such as outlined in U.S.S.N.s U.S.S.N.s 11/174,287, 11/124,640, 10/822,231, 10/672,280, 10/379,392, and the patent application entitled IgG Immunoglobulin variants with optimized effector function filed on October 21, 2005 having<br>
application no 	.   That is, optimized properties also include but are not limited to<br>
enhanced or reduced affinity for an FcyR. In one optional embodiment, the IgG variants are optimized to possess enhanced affinity for a human activating FcyR, preferably FcyRllla in addition to the FcRn binding profile. In yet another optional alternate embodiment, the IgG variants are optimized to possess reduced affinity for the human inhibitory receptor FcyRllb. That<br>
36<br><br>
is. particular embodiments embrace the use of antibodies that show increased binding to FcRn, and increased binding to FcyRllla Other embodiments utilize use of antibodies that show increased bi	NDing to FcRn, and increased binding to FcyRllla. These embodiments are anticipated to provide IgG polypeptides with enhanced therapeutic properties in humans, for example enhanced effector function and greater anti-cancer potency In an alternate embodiment, lie IgG variants are optimized to have increased or reduced affinity for FcRn and increased or decreased affinity for a human FcyR, including but not limited to FcyRI, FcyRlla, FcyRllb FcyR:;-. FcyRllla, and FcyRlllb including their allelic variations. These embodiments are anticipated to provide IgG polypeptides with enhanced therapeutic properties in humans, for example mcro-.-sed serum half-life and reduced effector function. In other embodiments, IgG variants provr...-&gt; enhanced affinity for FcRn and enhanced affinity for one or more FcyRs, yet reduced affinity for one or more other FcyRs. For example, an IgG variant may have enhanced binding to Fc! ..n and FcyRllla, yet reduced binding to FcyRllb. Alternately, an IgG variant may have reduced binding to FcRn and to FcyR's. In another embodiment, an IgG variant may have reduced affinity for FcRn and enhanced affinity for FcyRllb, yet reduced affinity to one or more activating Fcy. In yet another embodiment, an IgG variant may have increased serum half-life and reduced effector functions.<br>
[136]	Preferred embodiments comprise optimization of binding to a human FcRn and FcyR,<br>
however in alternate embodiments the IgG variants possess enhanced or reduced affinity for FcRn and FcyR from nonhuman organisms, including but not limited to rodents and non-human primates. IgG variants that are optimized for binding to a nonhuman FcRn may find use in experimentation. For example, mouse models are available for a variety of diseases that enable testing of properties such as efficacy, toxicity, and pharmacokinetics for a given drug candidate. As is known in the art, cancer cells can be grafted or injected into mice to mimic a human cancer, a process referred to as xenografting. Testing of IgG variants that comprise IgG variants that are optimized for FcRn may provide valuable information with regard to the clearance characteristics of the protein, its mechanism of clearance, and the like. The IgG variants may also be optimized for enhanced functionality and/or solution properties in aglycosylated form. The Fc ligands include but are not limited to FcRn, FcyRs, C1q, and proteins A and G, and may be from any source including but not limited to human, mouse, rat, rabbit, or monkey, preferably human. In an alternately preferred embodiment, the IgG variants are optimized to be more stable and/or more soluble than the aglycosylated form of the parent IgG variant.<br>
[137]	IgG variants can include modifications that modulate interaction with Fc ligands other<br>
than FcRn and FcyRs, including but not limited to complement proteins, and Fc receptor homologs (FcRHs). FcRHs include but are not limited to FcRH1, FcRH2, FcRH3, FcRH4, FcRH5,<br>
37<br><br>
[180]	The concentration of the therapeutically active IgG variant in the formulation may<br>
vary from about 0.1 to 100% by weight. In a preferred embodiment, the concentration of the IgG is in the range of 0.003 to 1.0 molar. In order to treat a patient, a therapeutically effective dose of the IgG variant may be administered. By "therapeutically effective dose" herein is meant a dose that produces the effects for which it is administered. The exact dose will depend on the purpose of the treatment, and will be ascertainable by one skilled in the art using known techniques. Dosages may range from 0.01 to 100 mg/kg of body weight or greater, for example 0.01, 0.1,1.0, 10, or 50 mg/kg of body weight, with 1 to 10mg/kg being preferred. As is known in the art. adjustments for protein degradation, systemic versus localized delivery, and rate of new protease synthesis, as well as the age, body weight, general health, sex, diet, time of administration, drug interaction and the severity of the condition may be necessary, and wilt be ascertainable with routine experimentation by those skilled in the art.<br>
[181]	Administration   of the  pharmaceutical  composition  comprising  an   IgG  variant,<br>
preferably in the form of a sterile aqueous solution, may be done in a variety of ways, including, but not limited to, orally, subcutaneously, intravenously, parenterally, intranasally, intraotically, intraocularly, rectally, vaginally, transdermal^, topically (e.g., gels, salves, lotions, creams, etc.), intraperitoneally, intramuscularly, intrapulmonary (e.g., AERx® inhalable technology commercially available from Aradigm, or Inhance® pulmonary delivery system commercially available from Nektar Therapeutics, etc.. Therapeutic described herein may be administered with other therapeutics concomitantly, i.e., the therapeutics described herein may be co-administered with other therapies or therapeutics, including for example, small molecules, other biologicals, radiation therapy, surgery, etc. <br>
EXAMPLES<br>
[182]	Examples are provided below to illustrate the present invention. These examples are<br>
not meant to constrain the present invention to any particular application or theory of operation. For all positions discussed in the present invention, numbering is according to the EU index as in Kabat (Kabat et a/., 1991, Sequences of Proteins of Immunological Interest, 5th Ed., United States Public Health Service, National Institutes of Health, Bethesda, entirely incorporated by reference). Those skilled in the art of antibodies will appreciate that this convention consists of nonsequential numbering in specific regions of an immunoglobulin sequence, enabling a normalized reference to conserved positions in immunoglobulin families. Accordingly, the positions of any given immunoglobulin as defined by the EU index will not necessarily correspond to its sequential sequence.<br>
EXAMPLE 1: DNA construction, expression, and purification of Fc variants<br>
[183]	Fc variants were constructed using the human lgG1 Fc domain and the variable<br>
domain of trastuzumab (Herceptin®, Genentech). The Fc polypeptides were part of Alemtuzumab, Trastuzumab or AC10. Alemtuzumab (Campath®, a registered trademark of<br>
53<br><br>
Millenium) is a humanized monoclonal antibody currently approved for treatment of B-cell chronic lymphocytic leukemia (Hale et a/., 1990, Tissue Antigens 35:118-127, entirely incorporated by reference). Trastuzumab (Herceptin®, a registered trademark of Genentech) is an anti-HER2/neu antibody for treatment of metastatic breast cancer. AC10 is a anti-CD30 monoclonal antibody. The Herceptin variable region was assembled using recursive PCR. This variable region was then cloned with human lgG1 into the pcDNA3.1/Zeo(+) vector (Invitrogen), shown in Figure 11. Plasmids were propagated in One Shot TOP10 E. coli cells (Invitrogen) and purified using the Hi-Speed Plasmid Maxi Kit (Qiagen). Plasmids were sequenced to verify the presence of the cloned inserts.<br>
[184]	Site-directed mutagenesis was done using the Quikchange™ method (Stratagene).<br>
Plasmids containing the desired substitutions, insertions, and deletions were propagated in One Shot TOP10 E. coli cells (Invitrogen) and purified using the Hi-Speed Plasmid Maxi Kit (Qiagen). DNA was sequenced to confirm the fidelity of the sequences.<br>
[185]	Plasmids containing heavy chain gene (VH-CY1-Cy2-Cy3) (wild-type or variants) were<br>
co-transfected with plasmid containing light chain gene (VL-CK) into 293T cells. Media were harvested 5 days after transfection, and antibodies were purified from the supernatant using protein  A  affinity  chromatography  (Pierce).  Antibody  concentrations  were  determined  by bicinchoninic acid (BCA) assay (Pierce). EXAMPLE 2: Binding affinity measurements<br>
[186]	Binding of Fc polypeptides to Fc ligands was assayed with surface plasmon<br>
resonance measurements. Surface plasmon resonance (SPR) measurements were performed using a BIAcore 3000 instrument (BIAcore AB). Wild-type or Variant antibody was captured using immobilized protein L (Pierce Biotechnology, Rockford, IL), and binding to receptor analyte was measured. Protein L was covalently coupled to a CM5 sensor chip at a concentration of 1 uM in 10 mM sodium acetate, pH 4.5 on a CM5 sensor chip using N-hydroxysuccinimide/N-ethyl-N'-(-3-dimethylamino-propyl) carbodiimide (NHS/EDC) at a flow rate of 5 ul/min. Flow cell 1 of every sensor chip was mocked with NHS/EDC as a negative control of binding. Running buffer was 0.01 M HEPES pH 7.4, 0.15 M NaCI, 3 mM EDTA, 0.005% v/v Surfactant P20 (HBS-EP, Biacore, Uppsala, Sweden), and chip regeneration buffer was 10 mM glycine-HCI pH 1.5. 125 nM Wild-type or variant trastuzumab antibody was bound to the protein L CM5 chip in HBS-EP at 1 ul/min for 5 minutes. FcRn-His-GST analyte, a FcRn fused to a His-tag and glutathione S transferase, in serial dilutions between 1 and 250 nM, were injected for 20 minutes association, 10 minutes dissociation, in HBS-EP at 10 ul/min. Response, measured in resonance units (RU), was acquired at 1200 seconds after receptor injection, reflecting near steady state binding. A cycle with antibody and buffer only provided baseline response. RU versus 1 / log concentration plots were generated and fit to a sigmoidal dose response using nonlinear regression with GraphPad Prism.<br>
54<br><br>
[187]	Binding of Fc polypeptides to Fc ligands was also done with AlphaScreen™<br>
(Amplified Luminescent Proximity Homogeneous Assay). AlphaScreen™ is a bead-based nonradioactive luminescent proximity assay Laser excitation of a donor bead excites oxygen, which if sufficiently close to the acceptor bead will generate a cascade of chemilurninescent events, ultimately leading to fluorescence emission at 520-620 nm The principal advantage of the AlphaScreen™ is its sensitivity Because one donor bead emits up to 60,000 excited oxygen molecules per second, signal amplification is extremely high, allowing detection down to attomolar (10 ,e) levels Wild-type antibody was biotinylated by standard methods for attachment to streptavidin donor beads, and tagged Fc ligand, for example FcRn, was bound to glutathione chelate acceptor beads The AlphaScreen™ was applied as a direct binding assay in which the Fc/Fc ligand interactions bring together the donor and acceptor beads. Addtionally, the AlphaScreen™ was applied as a competition assay for screening designed Fc polypeptides. In the absence of competing Fc polypeptides, wild-type antibody and FcRn interact and produce a signal at 520-620 nm. Untagged Fc domains compete with wild-type Fc/FcRn interaction, reducing fluorescence quantitatively to enable determination of relative binding affinities. EXAMPLE 3: FcRn-binding properties of Fc variants.<br>
[188]	Binding affinity of lgG1 Fc to FcRn was measured with variant antibodies using<br>
AlphaScreen™. The Fc polypeptides were part of Alemtuzumab or Trastuzumab. Alemtuzumab (Campath®, Ilex) is a humanized monoclonal antibody currently approved for treatment of B-cell chronic lymphocytic leukemia (Hale et a/., 1990, Tissue Antigens 35:118-127, entirely incorporated by reference). Trastuzumab (Herceptin®, Genentech) is an anti-HER2/neu antibody for treatment of metastatic breast cancer.<br>
[189]	Competitive AlphaScreen™ data were collected to measure the relative binding of<br>
the Fc variants compared to the wild-type antibody at pH6.0. Examples of the AlphaScreen™<br>
signal as a function of competitor antibody are shown in Figure 12. The 12 variant curves shown,<br>
those of P257L, P257N, V279E, V279Q, V279Y, A281S, E283F. V284E, L306Y, T307V, V308F,<br>
and Q311V, demonstrate increased affinity as each variant curve is shifted to the left of the wild-<br>
type curve in their box. Competition AphaScreen™ data for Fc variants of the present invention<br>
are summarized in Figures 13 and 14. The relative FcRn binding of the variant compared to wild<br>
type are listed. Values greater than one demonstrate improved binding of the Fc variant to FcRn<br>
compared to the wild type. For example, the variant E283L and V284E have 9.5-fold and 26-fold<br>
stronger binding than the wild type, respectively. Surface plasmon resonance measurements of<br>
many variants also show increased binding to FcRn as shown in Figure 15 and 16.<br>
[190]	At position 257, all variants that remove the imino acid, proline, and substitute an<br>
amino acid without the backbone N to side chain covalent bond, allow the backbone more flexibility which allows more freedom for the Fc domain to better bind FcRn. In particular, variants at position 257 to L and N have strong FcRn binding at pH 6. demonstrating that the four atom<br>
55<br><br>
side chain and gamma branching pattern of the side chain helps the Fc domain make productive, ie strong, FcRn interactions. Position 308 interacts with position 257. Both of these positions in turn interact with H310, which is directly involved in the Fc/FcRn interactions (Table 2, Burmeister et al (1994) Nature 372:379-383, entirely incorporated by reference) The Fc variants V308F and V08Y have a 2.9-fold and 4.3-fold increase in FcRn affinity over wild type (Figure 13) Positions 279 and 385 interact with FcRn as variants V279E, V279Q and V279Y and G385H and G385N all have stronger FcRn interactions. These variants all are to amino acids that are capable of hydrogen bonding.<br>
[191]	The Fc variant N434Y has particularly strong binding to FcRn at pH 6 0 as shown in<br>
figure 13. The single variant N434Y has 16-fold increased binding. Combinations of this variant with other modifications led to even stronger binding. For example, P257L/N434Y, A281S/N434Y, and V308F/N434Y show 830-fold, 180-fold, and 350-fold increases in FcRn binding.<br>
EXAMPLE 4: Variants incorporating insertions and deletions.<br>
[192]	Insertions  and  deletions  that alter the  strength  of  Fc/FcRn   interactions  were<br>
constructed and their binding properties to various Fc ligands were measured. An Fc variant with<br>
an inserted Ser residue between residues 281 and 282, using the EU numbering of Kabat et al,<br>
was designed to increase the FcRn binding properties of the Fc domain. This variant is referred<br>
to as A281S with "A" meaning an insertion following the position given. The inserted sequence,<br>
which may be more than one residue, is given following the position number. This Fc variant was<br>
constructed in the kappa, lgG1 anitbody trastuzumab (Herceptin®, Genetech) using methods<br>
disclosed herein. An insertion at the site between residues 281 and 282 shifts the Fc loop<br>
residues C-terminal of residue 281 toward the C-terminus of the loop and alters the side chain<br>
positioning. Fc variants comprising substitutions at positions 282, 283, and 284 suggested that<br>
the C-terminal shift of this loop was beneficial (See Figure 14). Another variant, a deletion of<br>
N286, sometimes referred to as N286#, was also constructed to shift the position of this FcRn-<br>
binding loop. Both of these variants show an increased binding affinity for FcRn at pH 6.0.<br>
[193]	The AlphaScreen™ data shows the binding of the A281S variant and other variants to<br>
FcRn. This AlphaScreen™ data was collected as a direct binding assay. Higher levels of chemiluminescent signals demonstrate stronger binding. As the concentrations of the variants are raised in the assay, stronger signals are created. These data at pH 6.0, in Figures 17a and 17b, demonstrate the increased affinity of A281S, P257L, P257L/A281S (a combination substitution/insertion variant) and other variants over the wild-type Fc. Also shown is a double substitution, T250Q/M428L, shown previously to have an increased serum half in monkeys (Hinton et al., 2004, J. Biol. Chem. 279(8): 6213-6216, entirely incorporated by reference). The insertion, A281S, alone increases the Fc/FcRn binding. Additionally, A281S further increases the binding of P257L when the two modifications are combined in the variant P257L/A281S as shown<br>
56<br><br>
in the ~40nM data points. The data in figure 17c demonstrate that these variants do not show increased FcRn binding at pH 7.0. The reduced affinity at pH 7.0 is desired for increased half-life in vivo, because it allows the release of Fc polypeptides from FcRn into the extracellular space, an important step in Fc recycling.<br>
[194]	Surface plasmon resonance experiments also demonstrate the improved binding of<br>
A281S to FcRn. Figure 18 shows the response units created as various Fc variant binding to FcRn on the chip surface. After allowing the variant to fully bind to the chip, the response units are recorded and shown on the ordinate. The insertion, A281S shows binding properties comparable to other variants shown herein to have increased affinity for FcRn over the wild type (See figures 13, 14 and 15, for examples).<br>
[195]	The deletion variant comprising a deletion of N286, N286#, also shows increased<br>
affinity for FcRn over wild type. This variant has a 2.0-fold increase in FcRn affinity as shown in Figure 13. The data therein are also AlphaScreen™ data collected as a competition experiment at pH 6,0. The variants are used to inhibit the binding of wild-type Fc, linked to the donor bead, with FcRn, linked to the acceptor beads. Two-fold less free N286# was needed than free wild-type Fc to inhibit the binding of the donor/acceptor beads through the Fc/FcRn complex. This demonstrates the 2-fold tighter binding of N286# over the wild type.<br>
[196]	Other Fc variants comprising insertions or deletions have decreased affinity for FcRn.<br>
The insertion variant, A254N has greatly decreased FcRn binding as would be expected from the nature and positioning of the variant. This variant places the insertion, an Asn, in the middle of an FcRn binding loop. This insertion has only 1.1% of the binding of the binding affinity of the wild type (Figure 13).<br>
[197] Whereas particular embodiments of the invention have been described above for purposes of illustration, it will be appreciated by those skilled in the art that numerous variations of the details may be made without departing from the invention as described in the appended claims. All references cited herein are incorporated in their entirety.<br>
57<br><br>
We claim:<br>
1.	A polypeptide comprising an Fc variant of a parent Fc polypeptide comprising at least one modification in the Fc region corresponding to said parent Fc polypeptide, wherein said Fc variant exhibits altered binding to FcRn as compared to the parent Fc polypeptide, and wherein said Fc variant comprises at least one modification selected from the group consisting of: 246S. 257L, 257N. 257Q, 257Y, 279A, 279F, 279G, 279H, 279I, 279K, 279M, 279N, 279P, 279Q, 279S, 279T, 279W. 279Y, 284H, 284K, 284P, 284Q, 284R, 284S, 308C. 308E, 308F, 308G, 308H. 308L, 308M, 308N, 308P. 308Q, 308S, 308T, 308W, 308Y. 385C, 385F, 385H, 385I, 385L, 385M, 385N, 385P, 385Q. 385V, 385W, 385Y, 387K, and 387Q, wherein numbering is according to the EU Index.<br>
2.	A polypeptide according to claim 1, wherein said Fc variant comprises at least one substitution selected from the group consisting of: 257L, 257N, 257Y, 279Q, 279Y, 308F, and 308Y.<br>
3.	A polypeptide according to claim 1 or 2, wherein said Fc variant comprises G385H.<br>
4.	A polypeptide according to any one of claims 1-3, wherein said Fc variant comprises at least two modifications in the Fc region of said parent polypeptide.<br>
5.	A polypeptide according to any one of claims 1-4, wherein said Fc variant further<br>
comprises at least one substitution selected from the group consisting of: 248D, 248E,<br>
248K, 249H, 249K, 249R, 250A, 250C, 250D, 250E, 250F, 250G, 250H, 2501 , 250K,<br>
250L, 250M, 250N, 250P, 250Q, 250R, 250S, 250V, 250W. 250Y, 251 C, 251 G, 251 L,<br>
251 Q, 251 S, 252A, 252C, 252G, 252N, 252Q, 252S, 252T, 252V, 252W, 252Y, 253A,<br>
254A, 254S, 255C. 255G, 255N. 255Q, 255R, 255S, 255T, 255Y, 256A, 256D, 256F,<br>
256K, 256Q, 256R, 256S, 256T, 258C, 258D. 258E, 258G, 258H, 258K. 258N, 258Q,<br>
258S, 258T, 258Y, 277A, 277C. 277D, 277E, 277F, 277G, 277H, 277I, 277K, 277L,<br>
277M, 277N, 277P. 277Q, 277R, 277S. 277T. 277V, 277W, 277Y, 279E, 279R, 280K,<br>
280R, 281 D, 281 E, 281 H, 281 K, 281 Q, 281 R, 282E, 282K, 282R, 284C, 284D, 284E,<br>
284G, 284N, 284T, 285A, 285C, 285D, 285G, 285H, 285N, 285Q, 285R, 285T, 285Y,<br>
286E, 286M, 286T, 287C, 287D, 287E, 287G, 287H, 287K, 287N, 287Q, 288D, 288E,<br>
288H, 288K, 288R, 287R, 287S, 287T, 288A, 288C, 288D, 288E, 288F, 288G, 288H,<br>
58<br><br>
288K. 288L. 288M. 288N. 288P. 288T, 288V, 288W 288Y, 304C, 305A. 305C, 305D. 305E, 305G. 305K, 305N, 305Q. 305R. 305S. 305Y. 306A, 306C. 306D. 306E, 306G. 306K, 306L, 306M, 306N, 306P. 306Q 306R. 306R. 306S, 306T, 306V. 306W. 306Y, 309D, 309E, 309K, 309R, 310A, 310C, 310D. 310E. 310G. 310H, 311 A. 311 K. 311 R, 312A, 312C, 312D, 312G. 312K, 312N, 312Q, 312R. 312S. 312T, 312Y. 313D, 313E. 313H, 313K, 313R, 314A, 314C, 314 D. 314E, 314F, 314G, 314H. 3141, 314L. 314M, 314N, 314P, 314Q. 314R, 314S, 314T, 314V, 314W. 314Y. 315D. 315K. 315R. 316H. 316K, 316R, 317A, 317C, 317D. 317E, 317G, 317K, 317N, 317Q, 317R, 317T. 317Y, 340D, 340E, 340H, 340K, 340R, 343C. 343D, 343E, 343G. 343H, 343K. 343N. 343Q, 343R, 343S, 343T, 343Y, 344L, 345C, 345D, 345E, 345G, 345H, 345K, 345N, 345Q, 345R, 345S, 345T, 345Y, 360A, 362A, 376C, 376D, 376E, 376G, 376K, 376N, 376Q, 376R, 376S, 376T, 376Y, 378S, 380A, 382A, 383D, 383E, 383R, 386P, 386T, 387P, 387R, 389D, 389N, 389P, 389S, 415A, 424A, 424D, 424E, 424H, 424K, 424R, 426D, 426K, 428A, 428C, 428D, 428E, 428F, 428G, 428H, 428I, 428K, 428L, 428M, 428N, 428P, 428Q, 428R, 428S, 428T, 428V, 428W, 428Y, 430C, 430E, 430G, 430N, 430R, 430S, 430T, 431 D, 431 E, 431 K, 431 R, 432C, 432G, 432Q, 432T, 432Y, 433A, 433K, 434A, 434F, 434K, 434L, 434R, 435A, 436A, 436D, 436R, 436T, 436Y, 438D, 438E, and 438R.<br>
6.	A polypeptide according to any one of claims 1-5, wherein said Fc variant exhibits altered binding to FcRn and FcyR, as compared to said parent Fc polypeptide.<br>
7.	A polypeptide according to any one of claims 1-6, wherein said Fc variant comprises at least two amino acid modifications in the Fc region corresponding to the parent Fc polypeptide.<br>
8.	A polypeptide according to any one of claims 1-6, wherein said Fc variant comprises at least three amino acid modifications in the Fc region corresponding to the parent Fc polypeptide.<br>
9.	A polypeptide according to any one of claims 1-6, wherein said Fc variant comprises at least four amino acid modifications in the Fc region corresponding to the parent Fc polypeptide.<br>
59<br><br>
10.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits increased binding to FcRn as compared to said parent Fc polypeptide.<br>
11.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits decreased binding to FcRn as compared to said parent Fc polypeptide.<br>
12.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits increased binding to FcyR, as compared to said parent Fc polypeptide.<br>
13.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits increased binding to FcyR and FcRn.<br>
14.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits decreased binding to FcyR and FcRn.<br>
15.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits decreased binding to FcyR and increased binding to FcRn.<br>
16.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits increased binding to FcyR and decreased binding to FcRn.<br>
17.	A polypeptide according to any one of claims 1-9, wherein said Fc variant exhibits altered binding to FcRn as compared to the parent Fc polypeptide, and wherein said Fc variant is contained within a polypeptide with specificity for a target molecule that is selected from the group consisting of a cytokine, a soluble protein factor and a protein expressed on cancer cells.<br><br>
18.	An antibody comprising an Fc variant according to any one of claims 1-17.<br>
19.	An Fc fusion comprising an Fc variant according to any one of claims 1-17. Dated this 6th Day of June 2007.<br><br><br><br><br>
60 <br>
ABSTRACT<br>
The present application relates to optimized IgG immunoglobulin variants, engineering methods for their generation, and their application, particularly for therapeutic purposes.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQUJTVFJBQ1QoMTMtNS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-ABSTRACT(13-5-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQUJTVFJBQ1QoMi0zLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-ABSTRACT(2-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQUJTVFJBQ1QoMjMtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-ABSTRACT(23-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ0xBSU1TKEFNRU5ERUQpLSgxMy01LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CLAIMS(AMENDED)-(13-5-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ0xBSU1TKEFNRU5ERUQpLSgyLTMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CLAIMS(AMENDED)-(2-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ0xBSU1TKEFNRU5ERUQpLSgyMy0xMS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CLAIMS(AMENDED)-(23-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMTAtMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(10-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMTMtNS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(13-5-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMTUtNC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(15-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMi0zLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(2-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMjMtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(23-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMjMtNi0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(23-6-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctQ09SUkVTUE9OREVOQ0UoMjQtOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-CORRESPONDENCE(24-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQucGRm" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZGVzY3JpcGl0b24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-descripiton (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybSAxMygyMy02LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form 13(23-6-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctRk9STSAxOCgyNC05LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-FORM 18(24-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctRk9STSAzKDE4LTgtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-FORM 3(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctRk9STSBQQ1QtSUItMzA0KDE4LTgtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-FORM PCT-IB-304(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctRk9STSBQQ1QtSVBFQS00MDkoMTgtOC0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-FORM PCT-IPEA-409(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctRk9STSBQQ1QtSVNBLTIzNygxOC04LTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-FORM PCT-ISA-237(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS0yLmRvYw==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS1wY3QtaWItMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-pct-ib-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS1wY3QtaXBlYS00MDIucGRm" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-pct-ipea-402.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctZm9ybS1wY3QtaXNhLTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-form-pct-isa-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctT1RIRVIgRE9DVU1FTlQoMTgtOC0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-OTHER DOCUMENT(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctcGN0LXNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMTAtMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-PETITION UNDER RULE 137(10-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMTgtOC0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-PETITION UNDER RULE 137(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUE9XRVIgT0YgQVVUSE9SSVRZKDE4LTgtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-POWER OF AUTHORITY(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUE9XRVIgT0YgQVVUSE9SSVRZKDItMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-POWER OF AUTHORITY(2-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUFJPU0VDVVRJT04gREVUQUlMUyBPRiBDT1JSRVNQT05ESU5HIEVQIEFQUExJQ0FUSU9OKDItMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-PROSECUTION DETAILS OF CORRESPONDING EP APPLICATION(2-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDE4LTgtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-REPLY TO EXAMINATION REPORT(18-8-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLU1VTU5QLTIwMDctU1BFQ0lGSUNBVElPTihBTUVOREVEKS0oMTUtNC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">861-MUMNP-2007-SPECIFICATION(AMENDED)-(15-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODYxLW11bW5wLTIwMDctd28gaW50ZXJuYXRpb25hbCBwdWJsaWNhdGlvbiByZXBvcnQoOC02LTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">861-mumnp-2007-wo international publication report(8-6-2007).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250371-process-for-recovering-methanol.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250373-delivering-functional-nucleic-acids-to-mammalian-cells-via-bacterially-derived-intact-minicells.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250372</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>861/MUMNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2011</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Dec-2011</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>XENCOR INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>111 WEST LEMON AVENUE, MONROVIA, CALIFORNIA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHAMBERLAIN AARON KEITH</td>
											<td>2445 E. DEL MAR BLVD., #405, PASADENA, CALIFORNIA 91107</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DESJARLAIS JOHN R</td>
											<td>1030 NORTH MICHIGAN AVENUE, PASADENA,CALIFORNIA 91104</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LAZAR GREGORY ALAN</td>
											<td>750 ARCADIA AVENUE, #6 ARCADIA, CALIFORNIA 91007</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KARKI SHER BAHADUR</td>
											<td>2833 PROVIDENCE WAY, POMONA CALIFORNIA-91767.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>VIELMETTER JOST</td>
											<td>495 ALAMEDA STREET, ALTADENA CALIFORNIA 91001</td>
										</tr>
										<tr>
											<td>6</td>
											<td>YODER SEAN CHRISTOPHER</td>
											<td>4168-c ORONTES WAY, SIMI VALLEY, CALIFORNIA 93063</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K16/00 C07K16/32</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/041220</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-11-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/669,311</td>
									<td>2005-04-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/681,607</td>
									<td>2005-05-16</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/703,018</td>
									<td>2005-07-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/627,763</td>
									<td>2004-11-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/642,886</td>
									<td>2005-01-11</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250372-fc-variant-with-altered-binding-to-fcrn by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:30 GMT -->
</html>
