<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250370-a-method-for-increasing-total-oil-level-in-a-seed by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250370:&quot;A METHOD FOR INCREASING TOTAL OIL LEVEL IN A SEED&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD FOR INCREASING TOTAL OIL LEVEL IN A SEED&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for increasing total oil in a seed comprising, transforming a plant with a nucleic acid construct that comprises as operably linked components, a promoter, a structural nucleic acid sequence capable of modulating the level of FAD2 mRNA or FAD2 protein; and growing said plant.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD FOR INCREASING TOTAL OIL LEVEL IN A SEED<br>
This application claims priority to U.S. provisional application 60/402,527 filed on 8/12/2002, herein incorporated by reference in its entirety.<br>
FIELD OF THE INVENTION<br>
The present invention is in the field of plant genetics and biochemistry. More specifically, the present invention relates to the level of total oil in plants. In particular, the present invention is directed to methods for increasing the oil level and altering the oil composition in plants and seeds. Morebver, the present invention includes and provides methods for producing plants and obtaining seed with increased oil levels. Such plants and seeds can also exhibit essentially unaltered protein compositions.<br>
BACKGROUND<br>
Plant oils are utilized in a wide Variety of applications. For example, soybean oils have been used in applications as diverse as salad and cooking oils to biodiesel and biolube oils. Seed oils are composed almost entirely of triacylglycerols in which fatty acids are esterified to each of the three hydroxyl groups of glycerol. The use of triacylglycerols as a seed reserve maximizes the quantity of ptored energy within a limited volume, because the fatty acids are a highly reduced form of carbon (Miquel and Browse, in Seed Development and Germination. Galili et al. (eds.), Marcel Dekker, New York, pp. 169-193, 1994). A large variety of different fatty acid structures are found in nature (Gunstone et al., The Lipid Handbook. Chapman &amp; Hall, London, ^994; Hilditch and Williams, The Chemical Constituents of Natural Fats. Chapman &amp; Hall, London, 1964; Murphy, Designer Oil Crops. VCH, Weinheim, 1994; van de Loo etal,Proc. NatlAcad. Sci. USA, 92:6743-6747, 1993),<br>
but just five account for 90% of the commercial vegetable oil produced: palmitic (16:0), stearic (18:0), oleic (18:1), linoleic (18:2), and a-linolenic (18:3) acid.<br>
Factors governing the total oil levpl of a plant or plant part such as a seed are complex. As such, selection for increased total oil is often a laborious process often with the resulting plants exhibiting considerable plant-to-plant variation (Jensen, Plant Breeding Methodology, John Wiley &amp; Sons, Inc., USA, 1988). Moreover, selection for increased total oil often results in a decrease in the protein fraction of the seed. Thus, there remains a need for methods of producing plants with increased total oil, particularly a method that also produces plants with essentially unaltered protein levels.<br>
SUMMARY OF THE INVENTION<br>
The present invention includes arid provides a method for increasing total oil level in a seed comprising: (A) transforming a plant with a nucleic acid construct that comprises as operably linked components, a promoter; a structural nucleic acid sequence capable of modulating the level of FAD2 mRNA or) FAD2 protein; and (B) growing the plant.<br>
The present invention includes and provides a method for increasing total oil in a seed<br>
comprising: (A) transforming a plant with a nucleic acid construct that comprises as operably linked components, a promoter, a structural nucleic acid sequence capable of increasing the level of oleic acid; and (B) growing the plant.<br>
The present invention includes apd provides a method of obtaining a seed having increased total oil level comprising: (A) growing a plant having a modulated level of a FAD2 protein or a FAD2 mRNA;jand (B) obtaining the seed from the plant.<br>
The present invention includes and provides a method for increasing percentage of total oil in a seed comprising: (A) transforming a plant with a nucleic acid construct that<br>
comprises as operably linked components, a promoter.|a structural nucleic acid sequence<br>
capable of modulating the level of FADb mRNA or FAD2 protein and (B) growing the plant.<br>
The present invention includes and provides a method for the production of a plant having an increased percentage of total oil comprising: (A) crossing a first plant having a modified level of a FAD2 protein or a FfiD2 mRNA with a second plant to produce a<br>
segregating population; (B) screening the segregating population for a member having an<br>
increased percentage of total oil; and (C) selecting the member.<br>
The present invention includes and provides chimeric genes comprising an isolated nucleic acid fragment encoding a delta-12 desaturase or any functionally equivalent subfragment or the reverse complement of such fragment or subfragment that are operably linked and wherein expression of such combinations results in an increase in total oil.<br>
Also included in this invention are plants and plant parts thereof containing the various chimeric genes, seeds of such plants, oil obtained from the grain of such plants, animal feed derived from the processing of such grain, the use of the foregoing oil in food, animal feed, cooking oil or industrial applications, products made from the hydrogenation, fractionation, interesterification or hydrolysis of such oil and methods for improving the carcass quality of an animal.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 depicts the construct pNlON67563.<br>
Figure 2 depicts a correlation of percentage of total oil versus oleic acid (18:1) in pMON67563 and pCGN9979 control lines.<br>
Figure 3 depicts oleic acid (18:lj) level versus percentage of total oil in Arabidopsis seed.<br>
Figure 4 depicts mean (SEM) oil percentage in T3 seed from transgenic lines expressing the FAD2 dsRNAi suppression construct (right) versus control lines containing an empty vector (left).<br>
Figure 5 depicts the construct plvtON67589. ••   Figure 6 depicts the construct pMON67591.<br>
Figure 7 depicts the construct pMON67592.<br>
Figure 8 depicts the construct pMON68655.<br>
Figure 9 depicts the construct pMON68656.<br>
DETAILED DESCRIPTION OF THE INVENTION Definitions<br>
As used herein, "total oil level" refers to the total aggregate amount of fatty acid without regard to the type of fatty acid.<br>
As used herein, the term "gene" is used to refer to the nucleic acid sequence that encompasses the 5' promoter region associated with the expression of the gene product, any intron and exon regions and 3' untranslated regions associated with the expression of the gene product.<br>
As used herein, a "FAD2", "Δ12 desaturase" or "omega-6 desaturase" is an enzyme<br>
capable of catalyzing the insertion of a Double bond into a fatty acyl moiety at the twelfth position counted from the carboxyl ternjinus.<br>
The terms "subfragrnent that is functionally equivalent" and "functionally equivalent subfragment" are used interchangeably herein. These terms refer to a portion or subsequence of an isolated nucleic acid fragment in which the ability to alter gene expression or produce a certain phenotype is retained whether or not the fragment or subfragment encodes an active enzyme. For example, the fragment or Subfragment can be used in the design of chimeric<br>
genes to produce the desired phenotype in a transformed plant. Chimeric genes can be designed for use in cosuppression or antisense by linking a nucleic acid fragment or subfragment thereof, whether or not it encodes an active enzyme, in the appropriate orientation relative to a plant promoter sequence.<br>
The term "non-coding" refers to sequences of nucleic acid molecules that do not encode part or all of an expressed protein. Non-coding sequences include but are not limited to introns, promoter regions, 3' untranslated regions, and 5' untranslated regions.<br>
The term "intron" as used herein refers to the normal sense of the term as meaning a segment of nucleic acid molecules, usually DNA, that does not encode part of or all of an expressed protein, and which, in endogenous conditions, is transcribed into RNA molecules, but which is spliced out of the endogenqus RNA before the RNA is translated into a protein.<br>
The term "exon" as used herein refers to the normal sense of the term as meaning a segment of nucleic acid molecules, usually DNA, that encodes part of or all of an expressed protein.<br>
As used herein, when referring to proteins and nucleic acids herein, the use of plain capitals, e.g., "FAD2", indicates a reference to an enzyme, protein, polypeptide, or peptide, and the use of italicized capitals, e.g., "FAD2", is used to refer to nucleic acids, including without limitation genes, cDNAs, and mRNAs.<br>
As used herein, a promoter that is "operably linked" to one or more nucleic acid sequences is capable of driving expression of one or more nucleic acid sequences, including multiple coding or non-coding nucleic acid sequences arranged in a polycistronic configuration.<br>
As used herein, the term complement of a nucleic acid sequence refers to the complement of the sequence along its complete length.<br>
As used herein, any range set forth is inclusive of the end points of the range unless otherwise stated.<br>
One skilled in the art may refer to general reference texts for detailed descriptions of known techniques discussed herein or equivalent techniques. These texts include Ausubel et al., Current Protocols in Molecular Biology, John Wiley and Sons, Inc., 1995; Sambrook et al.. Molecular Cloning, A Laboratory Manual (2d ed.), Cold Spring Harbor Press, Cold Spring Harbor, New York, 1989; Birren et al., Genome Analysis: A Laboratory Manual, volumes 1 through 4, Cold Spring Harbor Press, Cold Spring Harbor, New York, 1997-1999; Plant Molecular Biology: A Laboratory1 Manual, Clark (ed.), Springer, New York, 1997; Richards et al., Plant Breeding Systems (2d ed.), Chapman &amp; Hall, The University Press, Cambridge, 1997; and Maliga et al., Methods in Plant Molecular Biology, Cold Spring Harbor Press, Cold Spring Harbor, New York, 1995. These texts can, of course, also be referred to in practicing an aspect of the invention.<br>
The present invention includes and provides a method for increasing total oil level in a seed comprising: (A) transforming a plant with a nucleic acid construct that comprises as operably linked components, a promoter, a structural nucleic acid sequence capable of modulating the level of FAffl. mRNA or FAD2 protein; and (B) growing the plant. The structural nucleic acid sequence can be Delected from the group of SEQ ID NOS: 1,4, 7-11, 14, 19, 22,25 or 26 or the reverse complement thereof, any functionally equivalent subfragment thereof or the reverse complement of said fragment or subfragment.<br>
The present invention provides aj method for increasing total oil level in a seed. An increase of total oil can be an increase of any amount. An increase of total oil may result from altering the level of any enzyme or transcript that increases oleic acid level (18:1). In a preferred aspect, an increase in total oil is the percentage increase between the total oil found in a seed or collection of seeds and the total oil measured in a second or subsequent seed or<br>
collection of seeds. As used herein, percentage increase is calculated as the difference between the total oil found in a seed or collection of seeds and the total oil measured in a second or subsequent seed or collection of seeds. In a particularly preferred aspect, the increase in total oil is measured relative to a seed from a plant with a similar genetic background but lacking a structural nucleic aeid sequence capable of affecting the level of oleic acid (18:1). In another particularly; preferred aspect, the increase in total oil is measured relative to a seed from a plant with a similar genetic background but lacking a structural nucleic acid sequence capable of modulating the level of FAD2 mRNA or FAD2 protein.<br>
When levels of an agent are compared, such a comparison is preferably carried out between organisms with a similar genetip background. In a preferred aspect, a similar genetic background is a background where the organisms being compared share 50% or greater of their nuclear genetic material, hi a moref preferred aspect a similar genetic background is a background where the organisms being Compared share 75% or greater, even more preferably 90% or greater of their nuclear genetic material. In another even more preferable aspect, a similar genetic background is a background where the organisms being compared are plants, and the plants are isogenic except for any genetic material originally introduced using plant transformation techniques.<br>
In another aspect, the increase is measured in a seed of a plant produced by crossing two plants and the increase in a seed of that plant is measured relative to one or more of the seeds of one or more of the plants utilized to generate the plant in question (i.e., parents).<br>
Total oil levels can be measured by any appropriate method. For example, without limitation, quantitation of oil content of seeds is often performed with conventional methods, such as near infrared analysis (NIR), nuclear magnetic resonance imaging (NMR), soxhlet extraction, accelerated solvent extraction (ASE), microwave extraction, and super critical fluid extraction. Near infrared (NIR) spectroscopy has become a standard method for<br>
screening seed samples whenever the sample of interest has been amenable to this technique. Samples studied include wheat, maize, soybean, canola, rice, alfalfa, oat, and others.<br>
NIR analysis of single seeds carj be used (see Velasco et al, "Estimation of Seed Weight, Oil Content and Fatty Acid Composition in Intact Single Seeds of Rapeseed (Brassica napus L.) by Near-Infrared Reflectance Spectroscopy," Euphytica, Vol. 106, 1999, pp. 79-85; Delwiche, "Single Wheat Kernel Analysis by Near-Infrared Transmittance: Protein Content," Analytical Techniques and Instrumentation, Vol. 72,1995, pp. 11-16; Dowell, "Automated Color Classificatidn of Single Wheat Kernels Using Visible and Near-Infrared Reflectance," Vol. 75(1), 1998, pp. 142-144; Dowell et al., "Automated Single Wheat Kernel Quality Measurement Usjng Near-Infrared Reflectance," ASAE Annual International Meeting, 1997, paper number 973022, all of which are herein incorporated by reference in their entirety). NMR has also been used to analyze oil content in seeds (see, for example, Robertson and Morrison, "Analysis of Oil Content of Sunflower Seed by Wide-Line NMR," Journal of the American OM Chemists Society, 1979, Vol. 56,1979, pp. 961-964, which is herein incorporated by reference in its entirety).<br>
Other techniques, including soxhlet extraction, accelerated solvent extraction (ASE), microwave extraction, and super critical fluid extraction, can be used to determine oil content. Some techniques use gravimetry as the final measurement step (see, for example, Taylor et al., "Determination of Oil Content in Oilseeds by Analytical Supercritical Fluid Extraction," Vol. 70 (No. 4), 1993, pp. 437-439, which is herein incorporated by reference in its entirety). Gravimetry, however, is not suitable for use with small samples, including small seeds and seed with little oil content, because oil levels in these samples can be below the level of minimum sensitivity for the technique. Furthermore, the use of gravimetry is time consuming and is not amenable to high-throughput automation.<br>
The methods of the present invention may be used to increase total oil level in any seed. In a preferred embodiment, a seed includes either endosperm or embryo. In another preferred embodiment, a seed includes both endosperm and embryo. The seeds can be from either dicots or monocots. In a preferred embodiment, the seed may be selected from the group consisting of Arabidopsis seed, Brassica seed, canola seed, com seed, oil palm seed, oilseed rape seed, peanut seed, rapeseed seed, safflower seed, soybean seed, and sunflower seed, with Arabidopsis seed, Brassica sded, canola seed, corn seed, and soybean seed particularly preferred.<br>
Transforming a plant may be effected by any means that results in the introduction of a construct into a plant. Various methods for the introduction of a desired polynucleotide sequence into plant cells are available and known to those of skill in the art and include, but are not limited to: (1) physical methods Such as microinjection, electroporation, and microprojectile mediated delivery (bioliitics or gene gun technology); (2) virus mediated delivery methods; and (3) Agrobacteriuin-mcdiated transformation methods.<br>
The most commonly used methods for transformation of plant cells are the Agrobacterium-media.ted DNA transfer process and the biolistics or microprojectile bombardment mediated process (i.e., the gene gun). Typically, nuclear transformation is desired but where it is desirable to specifically transform plastids, such as chloroplasts or amyloplasts, plant plastids may be transformed utilizing a microprojectile mediated delivery of the desired polynucleotide.<br>
Agrobacterium-mediated transfojroation is achieved through the use of a genetically engineered soil bacterium belonging to the genus Agrobacterium. A number of wild-type and disarmed strains of Agrobacterium tumefaciens and Agrobacterium rhizogenes harboring Ti or Ri plasmids can be used for gene transfer into plants.   Gene transfer is done via the<br>
transfer of a specific DNA known as "T-DNA", that can be genetically engineered to carry any desired piece of DNA into many plant species.<br>
Agrobacterium-mediated genetic transformation of plants involves several steps. The first step, in which the virulent Agrobacterium and plant cells are first brought into contact with each other, is generally called "inoculation". Following the inoculation, the Agrobacterium and plant cells/tissues ai]e permitted to be grown together for a period of : several hours to several days or more under conditions suitable for growth and T-DNA transfer. This step is termed "co-culture. Following co-culture and T-DNA delivery, the plant cells are treated with bactericidal or bacteriostatic agents to kill the Agrobacterium remaining in contact with the explant and/or in the vessel containing the explant If this is done in the absence of any selective agents to promote preferential growth of transgenic<br>
versus non-transgenic plant cells, then this is typically referred to as the "delay" step. If done in the presence of selective pressure favouring transgenic plant cells, then it is referred to as a "selection" step. When a "delay" is used, it is typically followed by one or more "selection" steps.<br>
With respect to microprojectile bombardment (U.S. Patent No. 5,550,318; U.S. Patent No. 5,538,880; U.S. Patent No. 5,610,04£; and PCT Publication WO 95/06128; each of which is specifically incorporated herein by reference in its entirety), particles are coated with nucleic acids and delivered into cells by a propelling force. Exemplary particles include those comprised of tungsten, platinum, and preferably, gold.<br>
An illustrative embodiment of a method for delivering DNA into plant cells by acceleration is the Biolistics Particle Delivery System (BioRad, Hercules, CA), which can be used to propel particles coated with DNA or cells through a screen, such as a stainless steel or Nytex screen, onto a filter surface covered with plant cells cultured in suspension.<br>
Microprojectile bombardment techniques are widely applicable and may be used to transform virtually any plant species. Examples of species that have been transformed by microprojectile bombardment include monocot species such as maize (PCT Publication WO 95/06128), barley, wheat (U.S. Patent No. 5,563,055, specifically incorporated herein by reference in its entirety), rice, oat, rye, sugarcane, and sorghum; as well as a number of dicots including tobacco, soybean (U.S. Patent No. 5,322,783, specifically incorporated herein by reference in its entirety), sunflower, peanut, cotton, tomato, and legumes in general (U.S. Patent No. 5,563,055, specifically incorporated herein by reference in its entirety).<br>
To select or score for transformed plant cells regardless of transformation methodology, the DNA introduced into the cell may contain a gene that functions in a regenerable plant tissue to produce a compound that confers upon the plant tissue resistance to an otherwise toxic compound. Genes of interest for use as a selectable, screenable, or scorable marker would include but are n0t limited to GUS, green fluorescent protein (GFP), luciferase (LUX), antibiotic or herbicide tolerance genes. Examples of antibiotic resistance genes include the penicillins, kanamycin (and neomycin, G418, bleomycin); methotrexate (and trimethoprim); chloramphenicol; kanamycin and tetracycline.<br>
The regeneration, development, and cultivation of plants from various transformed explants is well documented in the art. "This regeneration and growth process typically includes the steps of selecting transformed cells and culturing those individualized cells through the usual stages of embryonic development through the rooted plantlet stage. Transgenic embryos and seeds are similarly regenerated. The resulting transgenic rooted shoots are thereafter planted in an appropriate plant growth medium such as soil. Cells that survive the exposure to the selective agent, or cells that have been scored positive in a screening assay, may be cultured in media that supports regeneration of plants. Developing<br>
plantlets are transferred to soil-less plant growth mix, and hardened off, prior to transfer to a greenhouse or growth chamber for maturation.<br>
The present invention can be use)d with any transformable cell or tissue. By transformable as used herein is meant a cell or tissue that is capable of further propagation to give rise to a plant. Those of skill in the) art recognize that a number of plant cells or tissues are transformable in which after insertion of exogenous DNA and appropriate culture conditions the plant cells or tissues can form into a differentiated plant. Tissue, suitable for these purposes can include but is not limited to immature embryos, scutellar tissue, suspension cell cultures, immature inflorescence, shoot meristem, nodal explants, callus tissue, hypocotyl tissue, cotyledons, roots, and leaves.<br>
Any suitable plant culture medium can be used. Examples of suitable media would include but are not limited to MS-based media (Murashige and Skoog, Physiol. Plant, 15:473-497, 1962) orN6-based media (Chu et al., Scientia Sinica 18:659, 1975) supplemented with additional plant growth regulators including but not limited to auxins, cytokinins, ABA, and gibberellins. Those of skill in the art arei familiar with the variety of tissue culture media, which when supplemented appropriately, support plant tissue growth and development and are suitable for plant transformation and regeneration. These tissue culture media can either be purchased as a commercial preparation, or custom prepared and modified. Those of skill in the art are aware that media and media supplements such as nutrients and growth regulators for use in transformation and regeneration and other culture conditions such as light intensity during incubation, pH, and incubation temperatures that can be optimized for the particular variety of interest.<br>
A construct or vector may include a plant promoter to express the nucleic acid molecule of choice. In a preferred embodiment, any nucleic acid molecules described herein can be operably linked to a promoter region that functions in a plant cell to cause the<br>
production of an mRNA molecule. For example, any promoter that functions in a plant cell to cause the production of an mRNA molecule, such as those promoters described herein, without limitation, can be used. In a preferred embodiment, the promoter is a plant promoter. A number of promoters that are active in plant cells have been described in the<br>
literature. These include, but are not limited to, the nopaline synthase (NOS) promoter (Ebert et al., Proc. NatL Acad. Sci. (U.S.A.) 843745-5149, 1987), the octopine synthase (DCS) promoter (which is carried on tumor-indjucing plasmids of Agrobacterium tumefaciens), the caulimovirus promoters such as the cauliflower mosaic virus (CaMV) 19S promoter (Lawton etai, Plant Mol. Biol. 9:315-324, 1987) and the CaMV 35S promoter (Odell etal., Nature 313:810-812, 1985), the figwort mosaic virus 35S-promoter (U.S. Patent No. 5,378,619), the light-inducible promoter from the small subunit of ribulose-l,5-bis-phosphate carboxylase (ssRUBISCO), the Adh promoter (Walker et al., Proc. Natl. Acad. Sci. (U.S.A.) 84:6624-6628, 1987), the sucrose synthase promoter (Yang et al., Proc. Natl. Acad. Sci. (U.S.A.) 57:41444148, 1990), the R gene complex promoter (Chandler et al., The Plant Cell 1:1175-1183, 1989) and the chlorophyll a/b binding protein gene promoter. These promoters have been used to create DNA constructs that have been expressed in plants; see, e.g., PCT publication WO 84/02913. The CaMV 35S promoters are preferred for use in plants. Promoters known or found to cause transcription of DNA in plant cells can be used in the invention.<br>
Other promoters can also be used to express a polypeptide in specific tissues, such as seeds or fruits. Indeed, in a preferred embodiment, the promoter used is a seed-specific promoter. Examples of such promoters include the 5' regulatory regions from such genes as napin (Kridl et al., Seed Sci. Res. 1:209:219, 1991), phaseolin (Bustos et al, Plant Cell, l(9):839-853, 1989), soybean trypsin inhibitor (Riggs et al., Plant Cell 1(6):609-621,1989), ACP (Baerson etal.. Plant Mol. Biol., 22\(2):255-261, 1993), stearoyl-ACP desaturase<br>
(Slocorabe etal., PlantPhysiol. 104(4):167-176, 1994), soybean a' subunit of p-conglycinin (P-Gm7S, see for example, Chen etal, Proc. Natl. Acad. Sci. 83:8560-8564, 1986), Vicia faba USP (P-Vf.Usp, see for example, SpQ ID NO: 1, 2, and 3 in U.S. Patent Application 10/429,516) and Zea mays L3 oleosin promoter (P-Zrn.L3, see, for example, Hong et ai, Plant Mol. Biol., 34(3):549-555,1997). Also included are the zeins, which are a group of. storage proteins found in corn endosperm. Genomic clones for zein genes have been isolated (Pedersen etal., Cell 29:1015-1026, 1982; and Russell etal., Transgenlc Res. 6(2):157-168) and the promoters from these clones, including the 15 kD, 16 kD, 19 kD, 22 kD, 27 kD and genes, could also be used. Other promotjers known to function, for example, in corn include the promoters for the following genes: wpxy, Brittle, Shrunken 2, Branching enzymes I and n, starch synthases, debranching enzymes, oleosins, glutelins and sucrose synthases. A particularly preferred promoter for corn endosperm expression is the promoter for the glutelin gene from rice, more particularly the Osgt-1 promoter (Zheng et al., Mol. Cell Biol. 13:5829-5842,1993). Examples of promoters suitable for expression in wheat include those promoters for the ADPglucose pyrosynttyase (ADPGPP) subunits, the granule bound and<br>
other starch synthase, the branching and debranching enzymes, the embryogenesis-abundant proteins, the gliadins and the glutenins. Examples of such promoters in rice include those promoters for the ADPGPP subunits, the granule bound and other starch synthase, the branching enzymes, the debranching enzymes, sucrose synthases and the glutelins. A particularly preferred promoter is the prcjmoter for rice glutelin, Osgt-1. Examples of such promoters for barley include those for the ADPGPP subunits, the granule bound and other starch synthase, the branching enzymes, the debranching enzymes, sucrose synthases, the hordeins, the embryo globulins and the aleurone specific proteins. A preferred promoter for expression in the seed is a napin promoter, referred to herein as P-Br.Snap2. Another preferred promoter for expression is an Arcelin5 promoter (U.S. Patent Publication<br>
2003/0046727). Yet another preferred promoter is a soybean 7S promoter (P-Gm.7S) and the soybean 7Sct' betaconglycininpromoteri(P-Gm.Sphasl).<br>
Additional promoters that may bej utilized are described, for example, in U.S. Patents 5,378,619; 5,391,725; 5,428,147; 5,447458; 5,608,144; 5,608,144; 5,614,399; 5,633,441; 5,633,435; and 4,633,436. In addition, a tissue specific enhancer may be used.<br>
Constructs or vectors may also include, with the region of interest, a nucleic acid sequence that acts, in whole or in part, td terminate transcription of that region. A number of such sequences have been isolated, including the Tr7 3' sequence and the NOS 3* sequence (Ingelbrecht et al., The Plant Cell 7:671-J680,1989; Bevan et al, Nucleic Acids Res. 11:369-385, 1983). Regulatory transcript termination regions can be provided in plant expression constructs of this invention as well. Transcript termination regions can be provided by the DNA sequence encoding the gene of intejrest or a convenient transcription termination region derived from a different gene source, for [example, the transcript termination region that is naturally associated with the transcript initiation region. The skilled artisan will recognize that any convenient transcript terminatioiji region that is capable of terminating transcription in a plant cell can be employed in the constructs of the present invention.<br>
A vector or construct may also include regulatory elements. Examples of such include the Adh intron 1 (Callis et al., Genes and Develop. 1:1183-1200, 1987), the sucrose synthase intron (Vasil et al, Plant Physiol. 97:1575-1579,1989) and the TMV omega element (Gallic et al, The Plant Cell 7:301-311,1989). These and other regulatory elements may be included when appropriate.<br>
It is understood that two or more nucleic acid molecules of the present invention may be introduced into a plant using a single construct and that construct can contain one or more promoters. In embodiments where the construct is designed to express two nucleic acid molecules, it is preferred that the two promoters are (i) two constitutive promoters, (ii) two<br>
seed-specific promoters, or (iii) one constitutive promoter and one seed-specific promoter. Preferred seed-specific promoters are 7S, napin, and maize globulin-1 gene promoters. A preferred constitutive promoter is a CaMV promoter. It is further understood that two or more of the nucleic molecules may be physically linked and expressed utilizing a single promoter, preferably a seed-specific or constitutive promoter.<br>
In a preferred embodiment of the present invention, post-transcriptional gene silencing may be induced in plants by transforming them with antisense or co-suppression constructs. In particular, constructs constructed by the methods of Smith et al. (Nature 407; 319-320, 2000) may be used to good effect. Other methods of construction are well known to one of skill in the art and have been reviewed.<br>
Structural nucleic acid sequences capable of decreasing the level of FAD2 mRNA oi/ I Q FAD2 protein include any nucleic acid sequence with sufficient homology to FAD2 gene. Exemplary nucleic acids include those set forth in US 6,372,965, US 6,342,658, US 6,333,448, US 6,291,741, US 6,063,947, WO 01/14538 A3, US PAP 2002/20058340, and US \i PAP 2002/0045232.<br>
The present invention includes afid provides a method for the production of a plant having increased total oil level as compared to at least one of a first or a second plant comprising: (A) crossing a first plant having a modified level of a FAD2 protein or a FAD2 mRNA with a second plant to produce a segregating population; (B) screening the segregating population for a member having the modified level of a FAD2 protein or a FAD2 mRNA; and (C) selecting the member.<br>
The present invention includes and provides a method for the production of a plant having an increased percentage of total oil comprising: (A) crossing a first plant having a modified level of a FAD2 protein or a FAD2 mRNA with a second plant to produce a<br>
segregating population; (B) screening the segregating population for a member having an increase in total oil; and (C) selecting the member.<br>
The present invention includes and provides a method for the production of a plant having an increased percentage of total oil comprising: (A) crossing a first plant having an increased level of oleic acid and a decreased level of .linoleic acid with a second plant to produce a segregating population; (B) screening the segregating population for a member having the increased level of oleic acid and.the decreased level of linoleic acid; and (C) selecting the member.<br>
Plants of the present invention can be part of or generated from a breeding program. The choice of breeding method depends on the mode of plant reproduction, the heritability of the trait(s) being improved, and the type of cultivar used commercially (e.g., Fj hybrid cultivar, pureline cultivar, etc). Selected, non-limiting approaches, for breeding the plants of the present invention are set forth below. A breeding program can be increased using marker assisted selection of the progeny of any pross. It is further understood that any commercial and non-commercial cultivars can be utilized in a breeding program. Factors such as, for example, emergence vigor, vegetative vigor, stress tolerance, disease resistance, branching, flowering, seed set, seed size, seed density, standability, and threshability etc. will generally dictate the choice.<br>
For highly heritable traits, a choice of superior individual plants evaluated at a single location will be effective, whereas for traits with low heritability, selection should be based on mean values obtained from replicated evaluations of families of related plants. Popular selection methods commonly include pedigree selection, modified pedigree selection, mass selection, and recurrent selection. In a preferred embodiment, a backcross or recurrent breeding program is undertaken.<br>
The complexity of inheritance influences the choice of the breeding method. Backcross breeding can be used to transfer one or a few favorable genes for a highly heritable trait into a desirable cultivar. This approach has been used extensively for breeding disease-resistant cultivars. Various recurrent selection techniques are used to improve quantitatively inherited traits controlled by numerous genes. The use of recurrent selection in self-pollinating crops depends on the ease of pollination, the frequency of successful hybrids from each pollination, and the number of hybrid offspring from each successful cross.   ;<br>
Breeding lines can be tested and bompared to appropriate standards in environments representative of the commercial target area(s) for two or more generations. The best lines are candidates for new commercial cultivars; those still deficient in traits may be used as parents to produce new populations for further selection.<br>
One method of identifying a superior plant is to observe its performance relative to other experimental plants and to a widely grown standard cultivar. If a single observation is inconclusive, replicated observations can provide a better estimate of its genetic worth. A breeder can select and cross two or more parental lines, followed by repeated selfing and selection, producing many new genetic combinations.<br>
The development of new cultivars requires the development and selection of varieties, the crossing of these varieties and the selection of superior hybrid crosses. The hybrid seed can be produced by manual crosses between selected male-fertile parents or by using male sterility systems. Hybrids are selected for certain single gene traits such as pod color, flower color, seed yield, pubescence color, or herbicide resistance, which indicate that the seed is truly a hybrid. Additional data on parental lines, as well as the phenotype of the hybrid, influence the breeder's decision whether  to continue with the specific hybrid cross.<br>
Pedigree breeding and recurrent selection breeding methods can be used to develop cultivars from breeding populations. Breeding programs combine desirable traits from two or<br>
more cultivars or various broad-based sources into breeding pools from which cultivars are developed by selfing and selection of desired phenotypes. New cultivars can be evaluated to determine which have commercial potential.<br>
Pedigree breeding is used commonly for the improvement of self-pollinating crops. Two parents who possess favorable, complementary traits are crossed to produce an FI. An F2 population is produced by selfing one or several F1's. Selection of the best individuals from the best families is carried out. Replicated testing of families can begin in the F4 generation to improve the effectiveness of selection for traits with low heritability. At an advanced stage of inbreeding (i.e., Feanp F7), the best lines or mixtures of phenotypically similar lines are tested for potential release as new cultivars.<br>
Backcross breeding has been used to transfer genes for a simply inherited, highly heritable trait into a desirable homozygous cultivar or inbred line, which is the recurrent parent. The source of the trait to be transferred is called the donor parent. The resulting plant is expected to have the attributes of the recurrent parent (e.g., cultivar) and the desirable trait transferred from the donor parent. After the initial cross, individuals possessing the phenotype of the donor parent are selected and repeatedly crossed (backcrossed) to the recurrent parent. The resulting parent is expected to have the attributes of the recurrent parent (e.g., cultivar) and the desirable trait transferred from the donor parent.<br>
The single-seed descent procedure in the strict sense refers to planting a segregating population, harvesting a sample of one seed per plant, and using the one-seed sample to plant the next generation. When the population has been advanced from the F2 to the desired level of inbreeding, the plants from which lines are derived will each trace to different Y2 individuals. The number of plants in a population declines each generation due to failure of some seeds to germinate or some plants to produce at least one seed. As a result, not all of<br>
the F2 plants originally sampled in the population will be represented by a progeny when generation advance is completed.<br>
In a multiple-seed procedure, breeders commonly harvest one or more pods from each plant in a population and thresh them together to form a bulk. Pan of the bulk is used to plant the next generation and part is put in reserve. The procedure has been referred to as modified -single-seed descent or the pod-bulk technique. The multiple-seed procedure has been used to save labor at harvest. It is considerably faster to thresh pods with a machine than to remove one seed from each by hand for the single-seed procedure. The multiple-seed procedure also makes it possible to plant the same number of seed of a population each generation of inbreeding.<br>
Descriptions of other breeding methods that are commonly used for different traits and crops can be found in one of several reference books (e.g., Fehr, Principles ofCultivar Development, Vol. 1, 1987).<br>
A transgenic plant of the present invention may also be reproduced using apomixis. Apomixis is a genetically controlled method of reproduction in plants where the embryo is formed without union of an egg and a sperm. Apomixis is economically important, especially in transgenic plants, because it causes any genotype, no matter how heterozygous, to breed true. Thus, with apomictic reproduction, heterozygous transgenic plants can maintain their genetic fidelity throughout repeated life cycles. Methods for the production of<br>
apomictic plants are known in the art. See, e.g., U.S. Patent No. 5,811,636.<br>
All articles, patents, and patent Applications cited herein are incorporated by reference in their entirety.<br>
The following examples are illustrative and not intended to be limiting in any way.<br>
EXAMPLES<br>
Example 1<br>
A gene silencing construct is produced according to the method of Smith et al. in order to reduce FAD2 expression in Arabidopsis through post transcriptional gene silencing (PTGS). (Smith et al., Nature 407: 319-320,2000). A construct (pMON67563, Figure 1) is constructed using the napin promoter to drive expression of a hairpin RNA (hpRNA) containing 120 nucleotides of the 3'-untranslated region of FAD2 in sense (SEQ ED NO: 1) and antisense orientation flanking an intrpn. Arabidopsis plants are transformed with<br>
pMON67563 by Agrobacterium-mediated transformation. An empty napin vector (pCGN9979) is also transformed into Arabidopsis plants by Agrobacterium-mcdiated transformation as a control.<br>
Example 2<br>
Seed from transformed Arabidopsis plants is analyzed by gas chromatography (GC) and near infrared spectroscopy (NIR) for fatty acid profile and total oil content. GC analysis demonstrates that Arabidopsis plants transformed with pMON67563 have an increased proportion of oleic acid (18:1) and a decreased proportion of linoleic acid (18:2) relative to controls. Transformed strains 67563-1 through 67563-13 show an increased proportion of oleic acid (18:1) and a decreased proportion of linoleic acid (18:2) relative to untransformed control strains 9979-11 through 9979-15, The relative amounts of oleic acid and linoleic acid are measured in percent (w/w) with control strains 9979-11 through 9979-15 exhibiting an oleic acid level ranging between about 14 %(w/w) and about 18 %(w/w) and a linoleic acid level ranging between about 30 %(w/w) and about 32 %(w/w). Transformed strains 67563-1 through 67563-3 and 67563-5 through 67563-15 show an oleic acid level ranging between about 34 %(w/w) and about 50 %(w/w) and a linoleic: acid level ranging between about 7 %(w/w) and about 18 %(w/w). NIR analysis demonstrates that plants transformed with<br>
pMON67563 show an increase in total oil level and essentially the same protein level as compared with a control plant. Control strains 9979-11 through 9979-15 exhibit a total oil<br>
percentage ranging between about 33.5% and about 36.8%. Compared to the control strains, transformed strains 67563-1 through 67563-3 and 67563-5 through 67563-15 show an increased percentage of total oil and range from about 35.5% to about 38.9%. As illustrated by Figure 2, when control and transformed strains are plotted to compare % total oil (x-axis) versus % oleic acid (18:1), an increase in oleic acid content is correlated with an increase in total oil content.<br>
Example 3<br>
Arabidopsis plants transformed with pMON67563 (Figure 1) are grown to the T3 seed generation. T3 seed is harvested and analyzed. Gas chromatography (GC) and near infrared (NIR) analysis are used to determine fatty acid profile and total oil content, respectively. Results of GC analyses demonstrate that 100% of progeny of the transformed plants have an increased level of oleic acid (18:1) similar to that observed for parent plants.<br>
Progeny plants also exhibit an increase in total oil. A comparison of oleic acid (18:1) level versus percentage of total oil is provided in Figure 3.<br>
As illustrated in Figure 4, mean oil percentage in T2 and T3 seed from transgenic lines is increased as compared to control seed containing an empty vector. The correlation between increased percent oleic acid and increased percent total oil evident in TS generation seeds appears to be genetically heritable.<br>
As illustrated by Figure 3, when control and transformed strains are plotted to compare percent total oil (x-axis) versus percent oleic acid (18:1), an increase in oleic acid content is correlated with an increased total oil content in transgenic Arabidopsis T3 seed. Example 4 Canola FAD-2 construct<br>
A section of the Brassica napus FAD2 gene was isolated by PCR amplification. Primers 17942 5'- GCGGCCGCGCGTpCTAACCCrGCGTCTGGGTC -3' (SEQ ID NO: 2) and 17944 5'- CCATGGGAGACCGTAGCAGACGGCGAGG -3' (SEQ ID NO:3) were paired to amplify base pairs 284-781 of the FAD2 coding sequence from Brassica napus (cv. Ebony) genomic DNA. A NotI site was added to the 5' end an Ncol site was added to the 3' end of the fragment to facilitate cloningt The resulting PCR fragments were cloned into pCR2.1 Topo. The complete double strand sequence was obtained.<br>
A 444 bp fragment containing CR-BN.BnFad2-0 (SEQ ID NO:4), was removed by digestion with NotI and Ncol. The fragment was ligated in between the Brassica napus promoter and first intron of the Arabidopsis FAD2 gene (At3gl2120), which had been digested with NotI and Ncol. The resulting plasmid, was named pMON67589 (Figure 5). The nucleic acid sequence was determined using known methodology and confirmed the integrity of the cloning junctions.<br>
A section of the Brassica napus FAD2 gene was isolated by PCR amplification. Primers 17943 5'- CCCGGGGCGTCCTAACCGGCGTCTGGGTC -3' (SEQ ID NO:5) and 17945 5'- GGTACCGAGACCGTAGCf GACGGCGAGG -3' (SEQ ID NO:6) were paired to amplify base pairs 284-781 of the FAD2 coding sequence from Brassica napus (cv. Ebony) genomic DNA. A Kpnl site was added to the 3' end a Smal site was added to the 5' end of the fragment to facilitate cloning. The resulting PCR fragments were cloned into pCR2.1 Topo. The complete double strand sequence was obtained.<br>
A 455 bp fragment containing AS-BN.BnFad2-0 (SEQ ID NO:7), was removed by digestion with Kpnl and Smal. The fragment was ligated in between the first intron of the Arabidopsis FAD2 gene (At3gl2120) and napin 3' UTR in pMON67589, which had been digested with Smal and Kpnl. The resulting plasmid, was named pMON67591 (Figure 6).<br>
The nucleic acid sequence was determined using known methodology and confirmed the integrity of the cloning junctions.<br>
A 2030 bp fragment containing CR-BN.BnFad2-0 followed by the first intron of the<br>
Arabidopsis thaliana FAD2 gene (At3gl2120) and AS-BN.BnFad2-0, was removed from pMON67591 by digestion with NotI and Smal. The fragment was ligated into a plasmid that had been digested with NotI and HindM (the HindOI site was blunt ended prior to ligation). The resulting plasmid was named pMON67592 (Figure 7). The nucleic acid sequence was determined using known methodology and confirmed the integrity of the cloning junctions. This vector was used in the subsequent transformation of canola, which was done via Agrobacterium-medizted transformation.<br>
Example 5<br>
Seeds from R2 canola plants transformed with pMON67592 were analyzed to determine total oil, oleic acid content arid protein content. As can be seen in Table 1, differences between homozygous positive and null segregants ranged from 1.7-2.5% Total Oil and 20.4-25.6% oleic acid. Protein levels remained the same. Table 2 shows the combined results from all events.<br>
Table 1. Average Total Oil and Oleic Acids Levels in R2 Canola seed derived from five individual transformants.<br>
(Table Removed) <br>
The mean and standard error were calculated in IMP Version: 4.0.4 (SAS Institute). The differences between means of homozygous positive and null segregants for both total oil and oleic acid for each of the 5 events is statistically significant (p<.0001></.0001>
Table 2. Average Total Oil and Oleic Acid Levels in R2 Canola seed transformed with pMON65792<br><br>
(Table Removed) <br>
The mean and standard deviation were calculated in IMP Version: 4.0.4 (SAS Institute). Plants were derived from 5 independent transformants. The differences between means of homozygous positive and null segregants is statistically significant (p<.0001></.0001>
Example 6<br>
On the basis of sequence similarity to Arabidopsis, soy and maize delta-12 desaturases (FAD2), four genes were identified in a proprietary corn unigene data base. They have been designated FAD2-1, FAD2-2, FAD2-3 and FAD2-4. The full-length cDNA sequence of 7m. FAD2-1 is shown in SFQ ID NO:8. It encodes a polypeptide of 387 amino acids (translation frame: nucleotide 182f 1342). The full-length cDNA sequence of Zm. FAD2-2 is shown in SEQ ID NO:9. It eftcodes a polypeptide of 390 amino acids (translation frame: nucleotide 266-1435).   The full-length cDNA sequence of Zm. FAD2-3 is shown in SEQ ID NO: 10. It encodes a polypeptide of 382 amino acids (translation frame: nucleotide 170-1315). The partial sequence of Zm. FAD2-4 is shown in SEQ ID NO:11. It encodes a partial polypeptide of 252 amino acids (translation frame: nucleotide 1-256).<br>
The coding regions of the three genes share significant sequence identity. FAD2-1 shares 91% identity to FAD2-2at the nucleotide level and 88% identity at the amino acid level. FAD2-1 shares 85% identity to FAp2-3 at the nucleotide level and 68% identity at the amino acid level. FAD2-1 shares 82% identity to FAD2-4 at the nucleotide level and 68% identity at the amino acid level. FAD2-3 shares 80% identity to FAD2-4 at the nucleotide level and 65% identity at the amino acid level.<br>
A virtual northern was used to determine which of the 4 genes were present in the seed tissue of corn. Both FAD2-1 and FAD2-2 were present in whole seeds, germ tissue and embryo tissue collected at different times during seed development. Neither FAD2-3 nor FAD2-4 were present in the seed tissues but both were detected in leaf tissue. RNAi construct from a fusion of 3'UTR of FAD2-1 and FAD2-2<br>
An expression construct comprising a corn L3 promoter, a rice-actin intron 3' to the promoter and 5' to the RNAi element, an RNAi element followed by a globulin 3'end located 3' to the RNAi element was constructed. The RNAi element was composed of a fragment of the Zm. FAD2-1 3'UTR joined by a BamHl site to a fragment of the Zm. FAD2-2 3'UTR both in the sense orientation linked to the same two FAD2 3'UTR fragments in the antisense orientation by an HSP70 intron containing intron splice sites. The HSP70 intron is located such that it is in the sense orientation relative to the promoter. The order of sense and antisense of the 3'UTR fragments is not important as long as each fragment (FAD2-1 and FAD2-2) is sense on one side of the center intron and antisense on the other. The construct is suitable for transformation into corn either by microprojectile bombardment or by Agrobacterium-media.ted transformation.<br>
PCR was used to obtain the HSP70 intron with a Bspl20I site on the 5' end and a Stul site on the 3'end. Primers (SEQ ID NOS:12 and 13) specific for the HSP70 intron sequence were used to clone the intron.<br>
The Bspl 201 and Stul fragment off the 820 base pair PCR product (SEQ ED NO: 14) was cloned into the same sites of a turbo binary containing a cauliflower mosaic virus promoter driving nptll with a NOS 3' an(l a Zea mays L3 promoter followed by a rice actin intron and a globulin 3' to make an intermediate construct.<br>
The fragments of the Zm. FAD2-1 and FAD2-2 3'UTRs were obtained by PCR.  Monsanto library clones were used as templates with primers specific for FAD2-1 (SEQ ID NO: 15,<br>
containing added cloning sites Sse83871 and Sacl; and SEQ ID NO: 16, containing an added cloning site BamHl) or primers specific for FAD2-2 (SEQ ID NOS:17, containing an added cloning site BamHl; and SEQ ID NO:18L containing added sites Bspl20I and EcoRV).<br>
To link the two PCR products, they were each digested with BamHl, gel purified, ligated and the ligation product used as a template with primers SEQ ID NOS:15 and 18. The resulting 447 base pair fragment (SEQ ID NO: 19).<br>
The Sacl/Bspl20I fragment of SEQ ID NO:19 was cloned into the same sites and the Sse8387I/EcoRV fragment of SEQ ID NQ:19 is cloned into the Sse83871/Stul sites of the intermediate construct to produce pMON56855 (Figure 8). Example 7 RNAi construct from a fusion of introns of FAD2-1 and FAD2-2<br>
An expression construct comprising a corn L3 promoter, a corn rice-actin intron 3' to the promoter and 5' to the RNAi element, an RNAi element followed by a globulin 3'end located 3' to the RNAi element was constructed. The RNAi element was composed of a portion of the Zm. FAD2-1 intron joined by a BamHl site to a portion of the Zm. FAD2-2 intron both in the sense orientation linked to the same two FAD2 intron fragments in the antisense orientation by an HSP70 intron containing intron splice sites. The HSP70 intron is located such that it is in the sense orientation relative to the promoter. The order of sense and antisense of the intron fragments is not important as long as each fragment (FAJD2-1 and FAD2-2) is sense on one side of the center intron and antisense on the other. The construct is suitable for transformation into corn either by microprojectile bombardment or by Agrobacterium-mediated transformation.<br>
PCR was used to obtain the HSPTfO intron as described in the previous example.<br>
Fragments from introns from the Zm. FAD2-1 and FAD2-2 genes were obtained by PCR. Genomic DNA prepared from the leaves of Z. mays variety LH59 using the protocol of<br>
Dellaportaet al. (Dellaporta et al. (1983) A plant DNA minipreparation: version EL Plant MolBiolRepl: 19-21) was used as the template. For FAD2-1, specific primers (SEQ ID N0:20, with added cloning sites Sse83871 and Sacl; and SEQ ED NO:21) were used to produce a 267 base pair product (SEQ ID NO:22). For FAD2-2, specific primers (SEQ ED N0:23, which included 21 bases that overlap with the 3' sequence of SEQ ID NO:22; and SEQ ED NO:24, containing added sites Bsp 1201 and EcoRV) were used to produce a 260 base pair product (SEQ ED NO:25).<br>
To link the two PCR products (SEQ ID NOS:22 and 25), they were both used as templates in a PCR reaction using primers SEQ ID NO:20 and SEQ ED NO:24 to produce a 506 base pair fusion (SEQ ID NO:26). The Sacl and Bsp 1201 fragment from SEQ ID NO:26 was gel purified then cloned into the same sites to produce pMON68656 (Figure 9).<br><br><br><br><br><br>
WE CLAIM:<br>
1. A chimeric gene construct comprising the nucleic acid fragment selected from the group consisting of SEQ ID NOS: 1, 4, 7-11, 14, 19, 22, 25 and 26 or the reverse complement thereof, any functionally equivalent subfragment thereof or the reverse complement of said fragment or subfragment wherein said fragments are operably linked capable of modulating the level of FAD2 mRNA or FAD2 protein and oleic acid and is used for transforming the plant, further wherein expression of the chimeric gene results in an increase in total oil.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">765-delnp-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtY29tcGxldGUgc3BlY2lmaWNhdGlvbiAoYXMgZmlsZWQpLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-complete specification (as filed).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtY29tcGxldGUgc3BlY2lmaWNhdGlvbiAoZ3JhbnRlZCkucGRm" target="_blank" style="word-wrap:break-word;">765-delnp-2005-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtY29ycmVzcG9uZGVuY2UtcG8ucGRm" target="_blank" style="word-wrap:break-word;">765-delnp-2005-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">765-delnp-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTIyMC5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTQwMS5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-401.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGN0LTQwOC5wZGY=" target="_blank" style="word-wrap:break-word;">765-delnp-2005-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGV0aXRpb24tMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-petition-137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NzY1LWRlbG5wLTIwMDUtcGV0aXRpb24tMTM4LnBkZg==" target="_blank" style="word-wrap:break-word;">765-delnp-2005-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250369-a-novel-process-for-the-preparation-of-mycophenolate-mofetil-mmf-by-enzimatic-transesterification.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250371-process-for-recovering-methanol.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250370</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>765/DELNP/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>52/2011</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Dec-2011</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>25-Feb-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY,LLC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 N. LINDBERGH BLVD.,ST. LOUIS,MO 63167,U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHRISTINE K. SHEWMAKER</td>
											<td>1409 SPRINGCREEK DRIVE,WOODLAND,CA 95776,U.S.A.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>ALISON VAN EENENNAAM</td>
											<td>856 BURR STREET,DAVIS,CA 95616,U.S.A</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DEBRA J. HAWKINS</td>
											<td>230 GRANDE AVENUE,DAVIS CA 95616,U.S.A</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RICK SANDERS</td>
											<td>917 ARTHUR STREET DAVIS,CA 95616,U.S.A</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/025751</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-08-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/402,527</td>
									<td>2002-08-12</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250370-a-method-for-increasing-total-oil-level-in-a-seed by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:34 GMT -->
</html>
