<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250378-novel-recombinant-dengue-multi-epitope-r-dme-as-diagnostic-intermediates by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250378:&quot;NOVEL RECOMBINANT DENGUE MULTI-EPITOPE (R-DME) AS DIAGNOSTIC INTERMEDIATES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;NOVEL RECOMBINANT DENGUE MULTI-EPITOPE (R-DME) AS DIAGNOSTIC INTERMEDIATES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to two recombinant dengue multiepitope (r-DME) antigens. One is for use in the diagnosis and/or detection of any or all of dengue specific Immunoglobin M (IgM), comprising of immunodominal epitope from nonstructural protein-1 (NS1) of Dengue-virus type- 1, Dengue-virus type-2, Dengue-virus type-3 and Dengue-virus type-4, said epitopes being linked with a predetermined number of glycine linkers, and codon optimized for expression in £. coli. The other is for use in the diagnosis and/or detection of any or all of dengue specific Immunoglobin G (IgG) comprising of one isotope from core, one epitope from prM, eight epitopes from DEN-2 envelope, four epitopes from DEN-4 NS1, one epitope from DEN-1 NS1, one epitope from DEN-2 NS1 and one epitope form DEN-4 NS3, said being linked with predetermined number of glycine linkers, and codon optimized for expression in E. coli.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>RECOMBINANT DENGUE MULTI EPITOPE PROTEINS AS DIAGNOSTIC<br>
INTERMEDIATES<br>
Field of the invention<br>
The present invention relates to novel kits and reagents for diagnosis of Dengue viral infections. In particular, the present invention relates to novel kits for diagnosis of the four known closely related, antigenically distinct serotypes of Dengue virus. More particularly, the present invention relates to multiepitope recombinant proteins and their use in the diagnosis of dengue and other viral infections. Background of the invention<br>
Currently, dengue fever is the most important re-emerging mosquito-bome viral disease, with the major proportion of the target population residing in the developing countries of the world. This has prompted the need to develop inexpensive, simple and rapid diagnostic tests, without compromising on sensitivity and specificity.<br>
Dengue fever and its more severe manifestations, namely, dengue haemorrhagic fever and dengue shock syndrome, are caused by infection with the dengue viruses, which are transmitted by mosquitoes of the genus Aedes. These viruses can produce a spectrum of clinical symptoms in infected individuals, ranging from inapparent or mild febrile illness to severe and fatal haemorrhagic disease. Epidemiological and laboratory evidences suggest that both viral and host immunologic factors are involved in the pathogenesis of severe dengue disease. In recent decades, there has been a dramatic increase in the incidence and clinical severity of dengue infections. According to the World Health Organization's estimates, there may be currently as many as 100 million cases of dengue fever every year. About 2.5 billion people in over a hundred tropical and sub-tropical coimtries, representing ~40% of the world's population, are now at risk from dengue. The global resurgence of dengue has been attributed to several factors, such as lack of effective vector control measures, uncontrolled urbanization coupled to concurrent population growth and increased air travel. This, in conjunction with the unavailability of a vaccine, has led to the current emergence of dengue as a serious public health threat (Gubler, D. J. (1998) Dengue and dengue haemorrhagic fever. Clin. Microbiol Rev. 11: 480-496.).). There is no effective antiviral therapy for the treatment of dengue infections (Leyssen, P., De Clercq, E., and Neyts, J. (2000) Perspectives for the treatment and infections with Flaviviridae. Clin. Microb. Rev. 13: 67-82.). Early diagnosis, followed by supportive care, and symptomatic treatment through fluid replacement are the keys to survival in cases of severe dengue infection.<br>
There are four closely related, antigenically distinct, serotypes [(Gubler, D. J. (1998) Dengue and dengue haemorrhagic fever. Clin. Microbiol Rev. 11: 480-496.); (Leyssen, P., De Clercq, E., and Neyts, J. (2000) Perspectives for the treatment and infections with Flaviviridae. Clin. Microb. Rev. 13: 67-82.); (Lindenbach, B. D., and Rice, C. M. (2001) Flaviviridae: The viruses and their replication, pp.991-1041. In D. M. Knipe and P. M. Howley (eds.-in-chief), Fields Virology 4"^ ed. Lippincot Williams and Wilkins, Philadelphia.); (Kuhn, R J., Zhang, W., Rossman, M. G., Pletnev, S. V., Corver, J., Lenches, E.,)] of dengue viruses, each of which can cause disease. These viruses are members of the family Flaviviridae; they have a common morphology, genomic structure and antigenic determinants (Lindenbach, B. D., and Rice, C. M. (2001) Flaviviridae: The viruses and their replication, pp.991-1041. In D. M. Knipe and P. M. Howley (eds.-in-chief), Fields Virology 4th ed. Lippincot Williams and Wilkins, Philadelphia). A computer generated graphic representation of the dengue virion has been depicted in the art. {Kuhn, R J., Zhang, W., Rossman, M. G., Pletnev, S. v., Corver, J., Lenches, E, Jones, C.'T, Mukhopadhyay, S., Chipman, P. R, Strauss, E. G., Baker, T. S., and Strauss, J. H. (2002) Structure of dengue virus: implications for flavivirus organization, maturation, and fusion. Cell 108: 717-725.). The major structural protein covering almost the entire surface of the virion is the Envelope (E) protein. The E protein is critical in the process of viral invasion by virtue of its capacity to interact with host cell surface receptors (Chen, Y., Maguire, T, and Marks, R M. (1996) Demonstration of binding of dengue envelope protein to target cells. J. Virol 70: 8765-8772), and it is the major virus antigen capable of eliciting protective and long-lasting immune responses against infection. [(Men, R., Wyatt, L., Tokimatsu, I., Arakaki, S., Shameem, G., Elkins, R, Chanock, R, Moss, B., and Lai, C.-J. (2000) Immunization of rhesus monkeys with a recombinant of modified vaccinia virus Ankara expressing a truncated envelope glycoprotein of dengue type 2 virus induced resistance to dengue type 2 virus challenge. Vaccine 18: 31I3-3I22.); (Putnak, R, Feighny, R, Burrous, J., Cochran, M., Hackett, C, Smith, G, and Hoke, C. (1991) Dengue-1 virus envelope glycoprotein gene expressed in recombinant baculovirus elicits virus-neutralizing antibody in mice and protects them from virus challenge. Am. J. Trop. Med. Hyg. 45:159-167.); (Churdboonchart, V, Bhamarapravati, N.,Peampramprecha, S. and Srinavin, S. (1991) Antibodies against dengue viral proteins in primary and secondary dengue hemorrhagic fever. Am. J. Trop. Med. Hyg. 44: 481-493.)]<br>
The virion contains two other structural proteins, premembrane (prM), implicated in the maintenance of the structural integrity of E, and capsid (C), a highly basic protein, which interacts with the RNA genome. The -11 kilobase (Kb) RNA genome of the virus has positive polarity and serves as the viral mRNA. Its 5' end is capped, but lacks a 3' poly A tail. The 5' quarter of the genome encodes the structural proteins C, prM and E, mentioned above. The rest of the genome encodes seven nonstructural (NS) proteins. The genomic RNA contains a single open reading frame (ORF) of over 10 Kb, flanked by 5' and 3' non coding regions. The order of proteins encoded in the long ORF is 5' C-prM/M-E-NSl-NS2A-NS2B-NS3-NS4A-NS4B-NS5 3'. as depicted in Figure 2B. This ORF is translated into a single polyprotein precursor, which is co-translationally and post-translationally processed by both host-as well as virus-encoded proteolytic enzymes to give rise to 3 structural (C, prM and E) and 7 non-structural (NS) proteins. From the perspective of developing diagnostic antigens, E [(Innis B., L, Thirawuth, V. and Hemachudha, C. (1989) Identification of continuous epitopes of the envelope glycoprotein of dengue type 2 virus. Am. J Trop. Med Hyg. 40: 676-687.); (Trirawatanapong, T, Chandran, B., Putnak, R. and Padmanabhan, R (1992) Mapping of a region of dengue virus type-2 glycoprotein required for binding by a neutralizing monoclonal antibody. Gene 116: 139-150.),] NSl (50. Wu. H. C, Huang Y. L, Chao, T T, Jan, J. T, Huang J. L, Chiang H. Y., King, C. C. and Shaio, M. F. (2001) Identification of B-cell epitope of dengue virus type 1 and its application in diagnosis of patients. J. Clin. Microbiol 39: 977-982.); (Huang J. H, Wey, J. J., Sun, Y C. Chin, C, Chien, L. J. and Wu, Y. C (1999) Antibody responses to an immunodominant nonstructural 1 synthetic peptide in patients with dengue fever and dengue hemorrhagic fever. J. Med. Virol. 57: 1-8.); (Garcia, G., Vaughn, D. W. and Del Angel, R. M. (1997) Recognition of synthetic oligopeptide from nonstructural proteins NSl and NS3 of DEN-4 virus by sera from dengue infected children. Am. J. Trop. Med. Hyg. 56: 466-470.)' (Falconar, A. K., Young, P. R and Miles, M. A. (1994) Precise location of sequential dengue virus subcomplex and complex B cell epitopes on the nonstructural-1 glycoprotein. Arch. Virol. 137: 315-326.)] and NS3 (Garcia, G, Vaughn, D. W. and Del Angel, R. M. (1997) Recognition ofsyrtyitic oligopeptide from nonstructural proteins NSl andNS3 of DEN-4 virus by sera from dengue infected children. Am. J. Trop. Med. Hyg. 56: 466-470.) are important as they carry numerous immunodominant epitopes.<br>
Most often, the diagnosis of dengue infections in endemic regions is based on clinical presentation, and it can be confused with other viral diseases (such as rubella, enteroviruss and influenza) with similar clinical features. Further, clinical presentation of dengue can vary, making accurate diagnosis extremely difficult. This underscores the importance of laboratory-based diagnostic tests in providing timely medical attention {George, R. ondLum, L. C. S. (1997) Clinical spectrum of dengue infection, pp. 89-113. In D. J. Oubler and G. Kuno. G. (eds.). Dengue and Dengue Hemorrhagic Fever, CAB International, Wallingford.)<br>
Four major laboratory criteria are used to confirm dengue virus infection, namely, the isolation of infectious virus, the demonstration of elevated virus-specific antibody titers, the demonstration of dengue virus antigens and/or the detection of viral RNA [(Guzman, M. G. and Kouri, G (1996) Advances in dengue diagnosis. Clin. Diagn. Lab. Immunol 3: 621-627.); (Vorndam, V. and Kuno, G (1997) Laboratory diagnosis of dengue virus infections, pp. 313-333. In D. J. Gubler and G. Kuno. G. (eds.), Dengue and Dengue Hemorrhagic Fever, CAB International, Wallingford.)]. Isolation of virus from clinical samples can be achieved by intracerebral inoculation of 1-2 day old suckling mice, intrathoracic inoculation of mosquitoe larvae, or by using mammalian/insect cells in culture. Of these, the mosquito system is the most sensitive. All these methods are slow and cumbersome. Viral RNA can be detected using coupled reverse transcription and polymerase chain reaction (RTPCR) or direct molecular hybridisation. Viral antigens can be detected by immunohistochemistry or immunofluorescence. However, the complexity of these assays and their high cost preclude their routine use. Serodiagnosis of dengue viruses is complicated by the existence of cross-reactive antigenic determinants shared by members of the Flaviviridae family. Commercial kits available for dengue diagnosis through detection of virus-specific antibodies use virus lysates as the coating antigen for antibody capture and, consequently, suffer from poor sensitivity and specificity. Additionally, the production of viral antigen using the mouse, mosquito or tissue culture-based system is associated with high cost.<br>
There is thus, currently a need for developing cost-effective, simple and rapid diagnostics that combine sensitivity and specificity.<br>
Objects of the invention<br>
Accordingly, it is one of the important objects of the present invention to provide inexpensive, simple and rapid diagnostic kits and reagents for detecting dengue infections.<br>
It is another object of the present invention to provide inexpensive, simple and rapid diagnostic kits and reagents for detecting dengue infections without compromising on sensitivity and specificity.<br>
It is yet another object of the present invention to provide diagnostic kits and methods for detecting dengue infections, which obviates many of the disadvantages of the prior art.<br>
It is a further object of the present invention to provide diagnostic kits and methods capable of specifically detecting dengue-specific IgM (Immunoglobin M).<br>
It is a further object of the present invention to provide diagnostic kits and methods capable of specifically detecting dengue-specific IgG (Immunoglobin G).<br>
It is a further object of the present invention to provide novel recombinant protein antigens by assembling key immunodominant linear dengue-specific epitopes. Summary of the invention<br>
The above and other objects of the present invention are achieved by providing two dengue multiepitope proteins, one designed to detect IgM and the other to detect IgG antibodies in dengue patient sera. The present invention also discloses a simple and rapid dengue spot test that are useful under field conditions.<br>
An important aspect of the present invention resides in designing and expressing two novel recombinant protein antigens by assembling key immunodominant linear dengue-specific epitopes, chosen on the basis of pepscan analysis, phage display and computer predictions. One of these developed to specifically detect dengue-specific IgM and the other to detect IgG. These novel recombinant dengue multiepitope proteins were expressed in E. coli, purified in a single step and used as capture antigens in ELISA. The ELISA results, using a large panel of suspected dengue patient sera, were in excellent agreement with those obtained using the commercially available Dengue Duo IgM and IgG Rapid strip test (PanBio).<br>
The present invention also provides a simple and rapid spot test for dengue detection using these recombmant multiepitope proteins. The high epitope density, careful choice of epitopes and the use of E. coli system for expression, coupled to<br>
simple purification, jointly has resulted in inexpensive diagnostic tests kits and methods with a high degree of sensitivity and specificity. Detailed description<br>
The present invention will now be described in greater detail with reference to the accompanying drawings and the following examples.<br>
In the drawings:<br>
Figure 1 shows the global prevalence of dengue and its mosquito vector (WHO report 2000). The bar diagram depicts the remarkable rise in incidence of dengue infections in recent years.<br>
Figure 2A depicts the Dengue virus structure and genome organization, particularly, computer generated graphic representation of dengue virion structure.<br>
Figure 2B shows schematic representation of the -11 Kb RNA genome of dengue viruses. The asterisks indicate the locations of immunodominant epitopes identified by pepscan, phage display and computer predictions as mentioned in the text. NC denotes the non-coding regions of the genome.<br>
Figure 3A shows the design of the IgG-specific recombinant dengue multiepitope protein (rDME-G), particularly, schematic representation of the synthetic gene encoding hnear immunodominant epitopes (shown by the boxes) derived from E, NSl and NS3 interconnected by triglycyl linkers. The gene was designed with 5' Bam HI and 3' Bgl n sites to facilitate cloning.<br>
Figure 3B shows Map of the E. coli expression plasmid obtained by inserting the rDME-G gene into the vector pQE60. Expression of the recombinant gene is under the control of an IPTG inducible promoter. The vector provides a 6xHis Tag to facilitate one-step purification.<br>
Figure 3C shows a computer generated structure of the rDME-G protein based on homology modelling. The blue (dark) segments indicate the epitopes and the yellow segments indicate the triglycyl linkers.<br>
Figure 3D shows a schematic representation of the IgM multiepitope of the present invention.<br>
Figure 3E shows a schematic representation of the IgG multiepitope of the present invention<br>
Figure 4A depicts the design of the IgM-specific recombinant dengue multiepitope protein (rDME-M). The gene was designed with 5' Bam HI and 3' Bgl II sites to facilitate cloning. It schematic represents the synthetic gene encoding linear<br>
immunodominant epitopes (shown by the boxes) derived from NS 1 of all four dengue virus serotypes, linked by tetraglycyl linkers<br>
Figure 4B depicts the Map of the E. coli expression plasmid obtained by insertmg two copies of the rDME-M gene sequentially into the vector pMAL-c2x. Expression of the recombinant gene is under the control of an IPTG inducible promoter. A 6xHisTag was incorporated at the 3' end of the recombinant molecule (prior to insertion into pMAL-c2x) to facilitate one-step purification.<br>
Figure 4C shows a computer generated structure of the rDME-M protein based on homology modelling.<br>
Figure 5 A shows rapid spot tests for the detection of anti-dengue antibodies in sera. The assay utilizes purified rDME-G (or rDME-M) at spot "T" and normal (dengue-negative) human serum at spot "C". The presence of IgG antibodies is revealed by gold-conjugated protein G; IgM antibodies are detected with gold-labeled rDME-M, instead.<br>
Figure 5B depicts the design of the proposed dengue dual (IgM &amp; IgG) detection test.<br>
Figure 5C shows a computer generated structure of the rDME-M protein based on homology modelling.<br>
Figure 6 shows the regions of linear immunodominant epitopes of dengue proteins.<br>
Figure 7 A depicts a map of plasmid pQE-60-IgG-rDME.<br>
Figure 7B shows SDA-PAGE analysis of IgG-rDME protein expression.<br>
Figure 8 A shows map of plasmid [pMALc2x-rDME]<br>
Figure 8B shows SDA-PAGE Analysis IgG-rDME protein expression.<br>
Figure 8C shows SDA-PAGE analysis single copy IGM-rDME protein expression.<br>
Figure 9 shows localizatio of single copy IgM-rDME protein expression.<br>
Figures 10A, lOB and IOC show purofocation of rDME proteins by ni-NTA affinity Chromatograhy.<br>
Figures llA and IIB show the Western and Immunoblot analysis of rDME proteins.<br>
An important feature of the present invention resides in designing and expressing synthetic protein antigens by assembling key immunodominant linear<br>
dengue-specific epitopes. This multiepitope approach to designing antigens has several advantages as summarized in Table 1:<br>
Table 1: The advantages offered by the recombinant multiepitope protein strategy<br>
(Table Removed)<br><br>
The high density of the epitopes in the recombinant dengue multiepitope (rDME) protein and the careful choice of only dengue-specific epitopes as its components contribute to a high degree of sensitivity and specificity. Further, the applicants' novel approach of using a recombinant multiepitope protein completely obviates multiple peptide synthesis and multiple protein expression; it also avoids expensive and time-consuming virus culture (for antigen preparation) and the associated biohazard risk. The design of the rDME protein and the ease of its expression and purification makes this a highly cost-effective approach to dengue diagnosis. This approach has the potential for the simultaneous detection of multiple infectious diseases.<br>
According to the present invention, novel IgG- and IgM-specific dengue multiepitope proteins were developed and were expressed in E.coli as described below and purified to homogeneity by affinity chromatography.<br>
In brief, the novel IgM multiepitope and IgG multiepitope of the present invention were synthesised as follows: 1. IgM multiepitope:<br>
(a) The novel IgM multiepitope of the present invention, was synthesised having inununodominal epitope from NSl of DEN-1, DEN-2, DEN-3 and DEN-4. All these epitopes were linked with four glycine linkers, and codon optimized for expression m E.coli. The schematic representation of the IgM multiepitope of the present invention is shown in Figure 3D .The novel IgM multiepitope of the present invention has the nucleotide sequence as shown in Seq. ID 1. Similarly, it has the amino acid sequence as shown in Seq ID 2.<br>
Seq ID 1.<br>
(a) IgM multiepitope sequence<br>
Formula Removed<br>
Seq ID 2<br>
Formula Removed<br><br>
(b) The synthesised gene is cloned in-frame with maltose binding protein in pMAL-c2X vector with MBP fusion and expressed in E.coli (DH5-a).<br>
(c)	The protein is purified under denaturing conditions by Ni-NTA affinity<br>
chromatography.<br>
(d)	The purified protein was evaluated with dengue specific ELIS A.<br>
2. IgG multiepitope:<br>
(a) The IgG multiepitope of the present invention was designed, which is having one epitope from core, one epitope from prM,  eight epitopes from DEN-2<br>
envelope, four epitopes from DEN-4 NSl, one epitope from DEN-1 NSl, one epitope from DEN-2 NSl and one epitope from DEN-4 NS3. All these epitopes were linked with three glycine linkers, and codon optimized for expression in E.coli. The schematic representation of the IgG multiepitope of the present invention is shown in Figure 3E . It has a nucleotide sequence as shown in Seq ID 3 and amino acid sequence as shown in Seq ID 4.<br>
Formula Removed<br>
Seq ID 3<br>
IgG multiepitope sequence<br>
(b)	Formula Removed<br>
(c)	The  synthesised gene was  cloned in pQE-60  and  expressed  in E.coli (SG13009).<br>
(d)	The protein was purified under denaturing conditions by ni-NTA affinity chromatography. .<br>
(e)	The purified protein was evaluated with dengue specific ELIS A.<br>
The above procedures will be described in detail hereinafter: IgG-specific epitopes were carefully identified by one or more of the following techniques: pepscan, phage display or computer predictions. A synthetic gene (Figure 3A), that encodes a recombinant IgG-specific dengue multiepitopeprotein (rDME-G), was first generated by ligation of carefully designed oligonucleotides encoding the selected epitopes. The resultant gene encoded 15 linear epitopes, of which 8 were from the E protein and 7 fi-om the nonstructutal proteins NSl and NS3. These epitopes range from 6 to 20 amino acid (aa) residues in length. Adjacent epitopes are separated by gly-gly-gly tripeptide linkers. The rDME-G gene was inserted in-frame with the 6x histidine tag-encoding sequence of the bacterial expression vector pQE60, under the control of an inducible PT5 promoter. This expression vector is depicted in Figure 3B. The rDME-G (~25 kDa) protein was expressed in E. coli and purified to homogeneity, using Ni-NTA affinity chromatography under denaturing conditions. The rDME-G<br>
protein, modeled in Figure 3C, shows that all the epitopes are well displayed and freely accessible for antibody binding.<br>
An IgM-specific 15 aa epitope of the NSl protein of dengue virus serotype 2, identified on the basis of computer prediction and ELISA with patients sera was selected as the building unit for designing an IgM-specific dengue multiepitope (rDME-M) protein. Corresponding NSl epitopes from dengue serotypes 1, 3 and 4 were identified by sequence homology. Again, as described for rDME-G, a synthetic gene, encoding each of these four NSl epitopes joined by (gly)4 tetrapeptide linkers, was created. Two copies of this synthetic gene were fused in tandem as shown in Figure 4A. This gene was then inserted in-frame with the malE gene of the vector pMAL-c2x under the Ptac promoter, as shown in Figure 4B. As problems of protein insolubility during the expression and purification of rDME-G protein were encountered, the applicants chose to express the rDME-M protein as a fusion derivative of maltose binding protein (MBP) to enhance its solubility. The rDME-G/MBP fusion protein was purified to near homogeneity by Ni-NTA affinity chromatography. The rDME-M protein (without its fusion partner) is modeled in Figure 4C, showing once again that its epitopes are freely accessible for binding as in the rDME-G protein. Evaluation of the rDME proteins as diagnostic reagents<br>
Having created the rDME proteins, their utility as diagnostic reagents for dengue detection was tested. To determine if the rDME proteins could recognize and bind dengue virus-specific antibodies, the purified proteins were tested separately as capture antigens in ELISA, using dengue virus type 2 hyperimmune murine serum as the test sample. For comparison, a control experiment was performed in parallel, using dengue type-2 virus (instead of the rDME proteins) as the capture antigen. Antibody titers determined in the test and control experiments were comparable, indicating that our synthetic rD_ME proteins were capable of efficiently recognizing serum dengue antibodies. This suggested that these two rDME proteins might serve as potential diagnostic reagents for the detection of dengue antibodies in patient sera.<br>
In order to evaluate the feasibility of using these rDME proteins as diagnostic<br>
reagents to detect IgG and IgM anti-dengue antibodies, the applicants developed in-<br>
house ELISA protocols (ICGEB protocols). In these assays, either rDME-G or rDME-<br>
M protein was used separately to capture either IgG or IgM class of anti-dengue<br>
antibodies, respectively, from patient sera.   Captured IgG and IgM antibodies were<br>
revealed using horseradish peroxidase conjugated anti-human IgG and anti-human<br>
IgM, respectively. We analyzed a large panel (n=172) of suspected dengue patient sera, obtained from dengue endemic regions in Sri Lanka, for the presence of dengue antibodies, using our rDME proteins in the ELISA format described above. We then compared our results with those obtained using PanBio's Dengue Duo IgM and IgG rapid strip test. All samples were also tested for the presence of infectious virus and viral RNA. Based on the data obtained, the samples could be categorized into six groups as summarized in Table 2 below:<br>
Table 2: Evaluation of rDME-M and rDME-G proteins<br><br>
(Table Removed)<br>
Group 1 samples tested positive for virus; groups 2-6 tested negative for virus<br>
Presence and absence of IgM or IgG is indicated by    '+'    and '-' superscripts<br>
respectively<br>
Data generated using Dengue Duo IgM &amp; IgG Rapid Strip test purchased from PanBio<br>
Pty., AustraUa.<br>
The ICGEB data were obtained using rDME-M and rDME-G separately as capture<br>
antigens in ELISAs to detect IgM and IgG, respectively<br>
? All samples gave inconsistent results in both tests<br>
The first group (n=22) represents the infected sera, in which the viral RNA<br>
could be detected by RTPCR; further infectious virus could be isolated from all but one<br>
sample of this group. With regard to the serology, this group was heterogenous, as<br>
reflected by its division into subgroups a, b and c. A total of 6 samples (all 5 of lb + the<br>
single one from Ic) were found to contain IgM antibodies. Out of these 6, the PanBio<br>
strip test identified 3 samples (2 from lb and the single one from Ic) as IgM^. This<br>
observation suggests that the rDME-M based ICGEB IgM ELISA is more effective in<br>
the early detection of dengue infection in a slightly larger proportion of samples. With<br>
regard to IgG, only one sample (Ic) was found to be positive; this was corroborated by<br>
the PanBio test. The remaining samples (group la; n=16) were negative for both IgM<br>
and IgG. Once again, these results were borne out by the PanBio test. The samples<br>
represented by groups 2-6 (collective n=150), were all found to be virusVRNA". Of the<br>
129 samples, represented collectively by groups 2-5, we could detect the presence of<br>
IgM in 49 samples (12 samples of group 2 + 37 samples of group 3), and IgG in 33<br>
samples (12 samples of group 2 + 21 samples of group 4), using the ICGEB ELISAs.<br>
Samples in group 5 were all IgM"/IgG", and are presumably 'normal', as all of them<br>
tested negative in the virus isolation and RTPCR assays as well.    In regard to the<br>
PanBio test, 19 samples of group 1 (16 from group la, 2 from group lb and the single<br>
sample from group Ic) and all samples of groups 2-5 (n=129) yielded results that were<br>
identical to those obtained using the ICGEB IgM and IgG ELISAs.   For samples in<br>
Group 6 (n=21), the results of the ICGEB ELISAs did not agree with those of the<br>
PanBio test.   The reason for the observed discrepancy is unclear at present, but is<br>
presumably related to differences in the nature of the capture antigen. It must be<br>
pointed  out that neither virus nor viral RNA could  be  detected  in  all  these<br>
'indeterminate' samples. It is, therefore, essential at this juncture to establish the<br>
clinical histories of the patients, from whom these samples were obtained, in order to<br>
understand the reason for the observed discrepancy. The overall comparative analysis<br>
of our data with the conventional PanBio results suggests that there is an excellent<br>
agreement between the ICGEB and the PanBio tests, with -86% (148 out of 172) of the<br>
samples analyzed giving identical results. Further, the ICGEB IgM ELISA picked up<br>
three additional samples (group lb), which the PanBio test failed to identify. The fact,<br>
that these three samples were virus*/RNA^, suggests that the ICGEB IgM ELISA test is<br>
more effective in early diagnosis of dengue infection.<br>
Design of kits for rapid detection of dengue specific IgM and IgG<br>
The present invention particularly relates to the development of an inexpensive,<br>
simple and rapid test to detect dengue specific IgM and IgG antibodies that will have<br>
utility under field conditions. Therefore, having demonstrated that the applicants'<br>
recombinant multiepitope proteins can be used to identify dengue specific antibodies in<br>
the ELISA format, the applicants sought to adapt this to a spot test format, as shown in<br>
Figure 5 A.  The design of the test was simple. It consisted of a piece of nitrocellulose<br>
membrane with two spots C (control) and T (test).   Spot C contained normal human<br>
serum and spot T contained either rDME-G (IgG-specific test) or rDME-M (IgM-<br>
specific test). The nitrocellulose membrane with the two spots (invisible in an unused test module) was mounted on an adsorbent pad and enclosed in a shallow well of a plastic module. To perform the spot test, a drop of the serum sample was added to the well, washed and treated with gold conjugated protein-G for IgG-specific test, and gold conjugated rDME-M for IgM-specific test. A positive test was indicated by two pink spots (C and T) and a negative test shows a single pink spot at C with no visible spot at T. An invalid test was indicated by the absence of pink color in both C and T. The present example was performed as two separate tests to detect IgM and IgG anti-dengue antibodies but it is within the scope of the present invention to perform it as a single test, that can simultaneously score both classes of antibodies, as shown in Figure 5B. In this dual test, the rDME-G and rDME-M proteins are spotted separately on the nitrocellulose membrane. As a control, normal human serum was applied along the V-shaped hne. IgG and IgM antibodies, captured from a test serum sample, will then be revealed, using a mixture of gold-labeled anti-human IgG/IgM.<br>
Various tests were also carried out to compare the conventional PanBio Dengue IgM kit with the IgM kit of the present invention. The results, which are self-expanatory, are shown in the Tables 3 to 8 below:<br>
Comparison of Pan Bio Dengue IgM kit<br>
with ICGEB Dengue IgM kit<br>
Cut ofTvalue is 0.450<br>
Table 3. Virus isolation positive and PCR positive sera samples<br><br>
(Table Removed)<br><br>
Table 4. Virus isolation negative and PCR positive sera samples<br><br>
(Table Removed)<br>
Table 5. Virus isolation negative and PCR negative sera samples (A. Both Pan bio and ICGEB positives)<br><br>
(Table Removed)<br>
Table 6. Virus isolation negativee and PCR negative sera samples (B. panbio negative and ICGEB positive)<br><br>
(Table Removed)<br>
Table 7. Virus isolation negativee and PCR negative sera samples (C. panbio positive and ICGEB negative)<br><br>
(Table Removed)<br>
Table 8. Virus isolation negativee and PCR negative sera samples (D. panbio negative and ICGEB negative)<br><br>
(Table Removed)<br>
Various tests were also carried out to compare the conventional PanBio Dengue IgG kit with the IgG kit of the present invention. The results are shown in the Tables 9 to 14 below:<br>
Comparison of Pan Bio Dengue IgG kit with ICGEB Dengue IgG kit<br>
Cut ofTvalue is 0.450<br>
Table 9. Virus isolation positive and PCR positive sera samples<br><br>
(Table Removed)<br>
Table 10. Virus isolation negative and PCR positive sera samples<br><br>
Table 11. Virus isolation negative and PCR negative sera samples (A. Both Pan bio and ICGEb positives)<br><br>
(Table Removed)<br>
Table 12. Virus isolation negative and PCR negative sera samples (B. panbio negative and ICGEB positive)<br><br>
(Table Removed)<br>
Table 13. Virus isolation negative and PCR negative sera samples (C. panbio positive and ICGEB negative)<br><br>
 (Table Removed)<br>
Table 14. Virus isolation negative and PCR negative sera samples ( D. panbio negative and ICGEB negative)<br><br>
(Table Removed)<br>
The above results clearly establish the superiority of the dengue diagnosis kits present invention. Example<br>
Materials and methods Materials<br>
Escherichia coli strains, DH5a was purchased from invitrogen life technologies, USA and SGI3009 was purchased from Qiagen, Germany. The plasmid pQE-60, Ni-NTA superflow resin and anti-His mAb were purchased from Qiagen, Germany. The plasmid pMALc2x was purchased from New England Biolabs, USA. Urea was purchased from Serva Electrophoresis GmbH, Heidelberg. All other chemicals were from sigma Chemical, St. Louis, Missouri, USA. Designing of recombinant dengue multiepitope proteins (r-DME)<br>
Two recombinant dengue multiepitope proteins (r-DME), which are IgG and IgM specific, were designed from dengue virus- specific Unear immunodominant epitopes. IgG specific r-DME<br>
IgG specific r-DME was designed with eight epitopes from Envelope, six epitopes from NSl and one epitope from NS3. The location is shown in Figure 6. As can be seen from this figure, it depicts regions of linear immunodominant epitopes of dengue proteins. Linear immunodominant epitopes numbered as 1-8 from envelope, 9-14 from NSl and 15 from NS3 were included in IgG specific recombinant dengue multiepitope protein, and In IgM specific recombinant dengue multiepitope protein , epitope numbered as 10 from DEN-2 and the corresponding region of other three dengue serotypes were included. The details of the epitopes are shown in Table 15 below:<br>
Table. 15<br><br>
(Table Removed)<br>
Epitope numbered as 1 is spans from 235-255 amino acids in the DEN-2 envelope protein, which contains two hnear epitopes of amino acid 235-242 and 248-255. First linear epitope reacted with 4 out of 7 DEN-2 patients sera, 3 out of 8 DEN-1 patients sera, 1 out of 5 DEN-3 patients sera, 2 out of 6 DEN-4 patients sera, 0 out of 6 Japanese Encephalitis Virus (JEV) patients sera. Second linear epitope reacts 5 out of 7 DEN-2 patients sera, 3 out of 8 DEN-1 patients sera, 3 out of 5 DEN-3 patients sera, 2 out of 6 DEN-4 patients sera, 0 out of 6 JEV patients sera. Epitope numbered as 2, 6, 7 and 8 are spans from 276-281, 461-467, 472-477 and 485-491 amino acids respectively in the DEN-2 envelope protein. These are proposed DEN-2 specific epitopes. Epitope numbered as 3 is spans from 372-383 amino acids in the DEN-2 envelope protein. It reacted with 7 out of 7 DEN-2 patients sera, 6 out of 8 DEN-1 patients sera, 5 out of 5<br>
DEN-3 patients sera, 2 out of 6 DEN-4 patients sera, 2 out of 6 JEV patients sera. This showed strong reaction with DENl, 2, 3 and weak with DEN-4 and JEV. Epitope numbered as 4 is spans from 386-397 amino acids in the DEN-2 envelope protein. This binds specifically to the 3H5 mAb. Epitope numbered as 5 is spans from 418-433 amino acids in the DEN-2 envelope protein. It reacted with 6 out of 7 DEN-2 patients sera, 5 out of 8 DEN-1 patients sera, 4 out of 5 DEN-3 patients sera, 1 out of 6 DEN-4 patients sera, 1 out of 6 JEV patients sera. This showed strong reaction with DENl, 2, 3 and weak with DEN-4 and JEV. Epitope numbered as 9 is spans from 110-117 amino acids in the DEN-1 NSl protein. It reacted 20 out of 21 DEN-1 patients sera and did not react with DEN2, 3 and 4 patients sera. This epitope sequence is quite different from JEV, and JEV hyperimmune sera did not shown any reaction with this ephope. Epitope numbered as 10 is spans from 1-15 amino acids in the DEN-2 NSl protein. It showed significant reaction with dengue fever patients sera but not with JEV patients sera. It showed more IgM antibody response than IgG antibody response. Epitope numbered as 11, 13 and 14 are spans from 33-49, 133-149, 330-346 amino acids respectively in the DEN-4 NSl protein. These were reacted significantly with dengue patients sera. Epitope numbered as 12 is spans from 111-121 amino acids in the DEN-4 NS1 protein. This epitope was identified by using the mAb raised against DEN-2 NS1 and this was found to be a dengue complex epitope. Epitope numbered as 15 is spans from 572-591 amino acids in the DEN-4 NS3 protein. This epitope reacted significantly with dengue patients sera.<br>
All the 15 epitopes were linked with tri-glycyl linkers (if the glycine is coming from epitope, where only two glycyl linkers were added) for the flexibility of the each epitope . The nucleotide sequence of rDME gene was codon optimised to E. coli, and got synthesized with Bam HI at 3' site and Bgl II at 5' site firom Geneart, Regensburg, Germany. IgM specific r-DME<br>
IgM specific r-DME was designed with 15 amino acid long immunodominant<br>
epitopes from nonstructural protein 1 (NSl) of DEN-1, 2, 3 and 4. Epitope numbered as<br>
10 (Fig. 60 and Table. 15) is spans fi"om 1-15 amino acids in the DEN-2 NSl protein. It<br>
showed significant reaction with dengue fever patients sera but not with JEV patients<br>
sera. It showed more IgM antibody response than IgG antibody response. Due to its<br>
low IgG response it was included in IgG sfecific rDME. This sequence is quite similar<br>
with DENl, 3 and 4, because of that, sequence 1-15 amino acids of corresponding<br><br>
DEN-1, 2 and 3 NSl were taken. These four epitopes were linked with tetra-glycyl linkers for the flexibility of the each epitope . The nucleotide sequence of rDME gene was codon optimised to E. coli, and got synthesized with BamHl at 3'site and Bglll at 5'site from Geneart, Regensburg, Germany. Construction ofpQE-60-IgG andlgSd- rDMEplasmids<br>
Synthesized genes (IgG rDME is 708bp and IgM rDME is 240bp) got cloned in pCR-Script at Kpn\ and Sad sites with BamYH at 3' site and Bglil at 5'site. The genes retrieved from PCR-script by restriction digestion with BamHi. and Bglil and ligated into pQE-60 vector (in which starting codon and 6X His tag is coming), which was precut with the same enzymes and treated with calf intestine alkaline phosphatase to prevent self ligation. The ligated product was transformed into Ecoli DH5a cells and selected on kanamycin plates. Screened the clones by restriction digestion with BamHI and Bglll enzymes. One of the selected clone plasmid DNA was transformed into E.coli SGI3009 cells and selected on kanamycin and ampicillin plates and screened the clones by restriction digestion with BamEl and BglE enzymes. Construction qfpMALc2x- IgG and IgM- rDMEplasmids<br>
Genes got retrieved from pQE-60-IgG rDME and pQE-60-IgM rDME plasmids by restriction digestion with BamHl and Hindlll enzymes. Which releases the gene with 6X His tag and terminator codon (IgG rDME is 732bp and IgM rDME is 264bp) and ligated into pMALc2x vector that was precut with the same enzymes. The ligated product was transformed into E.coli DH5a cells and selected on ampicillin plates. Screened the clones by restriction digestion with BamHl and Hindlll enzymes. In pMALc2x rDME plasmid, maltose-binding protein (MBP) will expresse along with rDMEs. Construction qfpMALc2x- IgM rDME double copy plasmid<br>
Gene gotjetrieved from pQE-60-IgM rDME single copy plasmid by restriction digestion with BamHl and Bglll enzymes and ligated into pQE-60-IgM rDME single copy plasmid that had been precut with Bglll The ligated product was transformed into E.coli DH5a cells and selected on kanamycin plates. Screened the clones by restriction digestion with BamHl and BglU enzymes. Positive clone upon restriction digestion with BamHl and Bglll enzymes, releases double the size of the single copy (pQE-60-IgM rDME double copy plasmid, 474bp).<br>
Double copy of gene got retrieved from pQE-60-IgM rDME double copy plasmid by restriction digestion with BamHi and Hindlll enzymes and ligated into pMALc2x vector that had been precut with BamHl and HindlU.. The ligated product was transformed into E.coli DH5a cells and selected on ampicillin plates. Screened the clones by restriction digestion with BamHl and Hindill enzymes (releases 504bp fragment). Expression screening<br>
Positive clones confirmed by restriction digestion were inoculated into 3 ml test-tube cultures and allowed to grow at 37°C in a shaker at 200rpm. Cultures were induced with ImM isopropylthiogalactoside (EPTG) at logarithmic phase (at OD of -0.5 at 600nm) for 4 hours. After induction, equal number of cells from various induced cultures and respective un-induced cultures were lysed in sample buffer and analysed by SDS-PAGE . The clone, which expressed maximum levels of the expected recombinant protein was chosen for further experiments. Localization of rDME proteins in induced cells<br>
Test-tube cultures of 5ml were induced with ImM IPTG at log phase for 4 h. Pelletted 1 ml of culture and resuspended in 0.2ml Tris-EDTA pH8.0 (lOmM Tris and ImM EDTA) with lysozyme (Img/ml). Vortexed gently for Imin, and kept at 37°C for Ih. The lysate was centifiiged at 15, OOOg for 5 min and the supernatant was transferred into a fresh tube. The pellet was resuspended m 0.2ml SDS-PAGE sample buffer and solubilized by boiling for 10 min. Both pellets and supematants were analysed by SDS-PAGE .<br>
Purification of proteins by Ni-NTA affinity chromatogrhaphy<br>
Pre-culture was prepared by inoculating 10µ1 glycerol stock of pQE-60-IgG rDME into 20mLLB medium with lOOµg of ampicillin and 25 µg of kanamycin/ml, and 10|al glycerol stock of pMALc2x- rDME (IgG, IgM, IgM double copy) into 20ml LB medium with 100|ag of ampicillin. Cultures were grown overnight in a shaker at 37°C, at 200rpm. Cultures were inoculated into 1 L LB flasks with respective antibiotics and incubated at 37°C, at 200rpm for 2-3h. Cultures were induced with ImM isopropylthiogalactoside (IPTG) at logarithmic phase (at OD of-0.5 at 600nm) for 4 hours. Prior to purification small allquots of culture were analysed by SDA-PAGE.<br>
The induced cultures were harvested by centrifugation in a Sorvall GS3 rotar at 6000 rpm for 20min at 4°C. The cell pellet (~3.5g wet weight) was resuspended in 20ml buffer B (10mM NaH2PO4,10 mM Tris-Cl, 8 M urea, pH 8.0) and kept the cells on stirring for 60min at room temperature. Lysate was centrifuged at 10,000 rpm in Sorvall GSA rotar at room temperature for 30 min. The supernatant was removed and analysed by SDS-PAGE.<br>
The supernatant was bound to Ni-NTA Superflow matrix (~3ml packed volume) that is pre washed with buffer B for neutralization of matrix and kept on flip-flop shaker for Ih at room temperature (RT). The mix was packed into a column and flow-through was collected. The column was washed twice with 15 ml buffer C (l0mM NaH2PO4.10 mM Tris-Cl, 8M urea, pH 6.3) and eluted four times with 3ml of buffer D (lOmM NaH2PO4.10 mM Tris-Cl, 8 M urea, pH 5.9) and E (10mM NaH2PO4,10 mM Tris-Cl, 8 M urea, pH 4.5). All the fractions were analysed by SDS-PAGE and the pure eluted fractions were pooled and flash-frozen for storage at -80°C with 50|ig/ml gentamycin. Purified proteins were dialysed against different urea concentration (8-0 M) and different buffers with out urea (sodium acetate buffer pH 4 «&amp; 5, sodium phosphate buffer pH 11.5 &amp; 12, glycine buffer pH 10&amp; 11, sodium carbonate and bicarbonate buffer pH 10.8 &amp; 10). Western blotting<br>
Purified protein was run on 15% SDS-PAGE, along with pre-stained marker and transferred electrophoretically to nitrocellulose membrane. The membrane was blocked with 1% PVP made in IX PBS for 2h at RT. The membrane was washed three times with IX PBS-T (IX PBS containing 0.1% Tween 20) and incubated with anti-His mAbs at a dilution of 1: 2000 for 90min at RT. The membrane was washed three times with IX PBS-T an incubated with anti-mouse IgG-alkaline phosphatase at a dilution of 1:5000 for 90min at RT. The membrane was washed three times with IX PBS-T and incubated with substrate (5-bromo-4-chloro-3-indolyl phosphate with nitroblue tetrazolium) for 30 min at RT. Immunob lotting<br>
Purified protein was run with fused wells on 15% SDS-PAGE, along with pre-stained marker and transferred electrophoretically to nitrocellulose membrane. The membrane was cut into small vertical strips and was blocked with IX PBS containing 2% Tween 20 for Ih at RT. The strips were washed thrice with IX PBS-T (IX PBS<br>
containing 0.1% Tween 20) and incubated with dengue patients sera (Division of<br>
Virology, Defense Research and Development Establishment, Gwalior, INDIA) at 1:100 dilution, and for control, negative human sera was used with same conditions for 30min at RT. The strips were washed thrice with IX PBS-T and incubated with anti-human IgG-peroxidase at a dilution of 1:10,000 for 30min at RT. The strips were washed thrice with IX PBS-T and incubated with substrate (3,3', 5,5'-Tetramethylbenzidine) for 30 min at RT. Results<br>
Expression ofIgG andlgSd- rDME proteins B. Expression ofIgG- rDME inpQE-60 vector<br>
The protein IgG-rDME was expressed in-frame with the translation initiator codon and the 6x His tag of pQE-60 vector at BamHl and BglR sites, the vector map is shown in Figure 8A. The predicted protein is of 244 aa, of which methionine and glycine, and 6 histidines are coming from the vector sequence and molecular weight is -25 kDa. The protein was expressed at ~120mg/ml, under the control of IPTG inducible phage T7 promoter and the repressor is supplied by pREP-4 plasmid in the E.coli strain SG13009. Positive clones were screened by restriction digestion with BamHl and BglE enzymes. Selected positive clone was induced with IraM IPTG for 4h at 37'C, and the expected ~25kDa protein was observed by SDS-PAGE analysis (Fig.3B). The selected positive clone was used for further studies. B. Expression ofIgM- rDME in pMALc2x vector<br>
The gene IgM-rDME was cloned in pQE-60 vector and the protein did not express in the host. Then the gene was cloned in-frame with MBP in pMALc2x vector and expressed at ~60mg/ml (Fig. 8 A), under the control of the inducible tac promoter in E.coli DH5a. The expressed protein molecular weight is ~52kDa with 86aa of IgM-rDME with his tag and the complete MBP. Positive clones were screened by restriction digestion with BamHl and HindUll enzymes. Selected positive clone was induced with 0.4 mM IPTG for 4h at 37°C, and the expected ~52kDa protein was observed by SDS-PAGE analysis (Fig.8C). The selected positive clone was used for further studies.<br>
To increase the epitope density of IgM-rDME, two copies of the same gene was cloned in pQE-60 and sub-cloned in-frame with MBP in pMALc2x vector. The same procedure was followed as above for single copy to express the double copy and the expected ~60kDa protein was expressed at ~30mg/ml and observed by SDS-PAGE analysis (Fig. 8D).<br>
Localization rDME proteins<br>
To determine the solubility of proteins under native conditions (with the absence of denaturing reagents), all the three clones culture (pQE60-IgG-rDME and pMALc2x-IgM rDMEsingle copy and double copy clones) were induced with ImM IPTG for 4 h. Culture pellet was resuspended in Tris-EDTA pH8.0 containing lysozyme. Vortexed gently for lmin, and incubated at 3TC for Ih. The lysate was centifuged at 15, OOOg for 5 min and the supernatant was transferred into a fresh tube. The pellet was resuspended in 0.2ml SDS-PAGE sample buffer and solubilized by boiling for 10 min. Both pellets and supernatants were analysed by SDS-PAGE (7). In case of pQE-60-IgG protein was not found in supernatant fraction and observed in the pellet. It clearly indicates that the protein is associated with pellet, so it is not soluble under native condition. To overcome this insolubility of the protein, we expressed the protein at low IPTG concentrations (0.005, 0.06, 0.03 and 0.02 mM) and low temperatures (27 and 32 "C). These did not help in the solubility of the protein. Then the protein was expressed in-frame with MBP in pMALc2x vector (Fig. 9). Here expression levels are high but did not help in solubility.<br>
For further experiments, the IgG-rDME protein, which was expressed in pQE-60 was used. Due to insolubility of IgG-rDME protein, it was purified under denaturing condition in the presence of 8M urea.<br>
In case of IgM-rDME, single copy and double copy clones, which were expressed with MBP fusion, were checked for their sohibility under native condition. Single copy protein is slightly soluble under native condition, as it is shown in Figure 10, where single copy protein was observed both in supernatant and pellet. The same procedure was repeated for double copy, where the protean is associated with pellet, so it is not soluble imder native condition. Both single and double copy proteins were purified under denaturing condition in the presence of 8M urea, as urea do not interfere in ELIS A and immunoblots. Purification ofrDMEs<br>
Purification of rDMEs were done under denaturing conditions using 8M urea<br>
using Ni-NTA Superflow matrix. Induced culture was resuspended in buffer B of<br>
pH7.5 and kept the cells on stirring for 60min at RT. The centrifuged lysate was bound<br>
to using Ni-NTA Superflow matrix and incubated at RT on flip-flop shaker for Ih. The<br>
mix was packed into a column and flow-through was collected. The column was<br>
washed with buffer C of pH 6.3 to remove nonspecific bound proteins and eluted the<br>
protein with of buffer D of pH 5.9 and followed by buffer E of pH 4.5. The fractions were analysed by SDS-PAGE (Figure 11) and eluted fractions, which are pure enough, were pooled and flash-frozen for storage at -80°C with 50µg/ml gentamycin for further experiments.<br>
Purified proteins were subjected to electrophoresis on a denaturing gel and electro-transferred on to a nitrocellulose membrane and subjected to western analysis. The membrane was blocked with 1% PVP for 2h at RT. The membrane was washed with IX PBS-T and incubated with anti-His mAbs for 90 min at RT. The membrane was washed again and incubated with anti-mouse IgG-alkaline phosphatase for 90min at RT. The membrane was washed and protein bands were visualized by incubating in substrate (5-bromo-4-chloro-3-indolyl phosphate with nitroblue tetrazolium) for 30 min atRT.<br>
To check the solubility, purified proteins were dialysed against different urea concentrations and different buffers of pH ranging from 4.0 to 12. The results (not shown) indicate that IgG rDME protein is soluble in sodium acetate buffer at pH4.0 and 5.0 and the precipitation was least in comparison with other buffers tested and it need minimum of 3M urea to keep soluble protein of 2mg/ml.<br>
As IgM single copy is quite soluble under native conditions, the protein is quite soluble in all the above buffers and it need 3M urea to keep 5mg/ml. Preliminary analysis of IgG- rDME with dengue sera<br>
To check the reactivity of IgG-rDME to dengue patients sera, protein was electro- transferred on to nitrocellulose membrane and reacted with dengue patients sera for an hour at RT. Then the membrane was washed with IX PBST and incubated with anti-human IgG-peroxidase for 30min at RT. After washing the membranewas developed by incubating with substrate (3,3', 5,5'- Tetramethylbenzidine) for 30 min at RT. Control exp,eriment was done with dengue negative human sera, the results clearly indicates that the IgG-rDME is recognizing the antibodies in dengue patients sera .<br><br><br><br><br><br><br><br>
We claim:<br>
1.	A recombinant dengue multiepitope (r-DME) comprising:<br>
(a)	multiepitope of nucleotide sequence Seq ID 1 having immunodominal<br>
epitope from nonstructural protein-1 (NS1) of Dengue-virus type- 1, Dengue-<br>
virus type-2, Dengue-virus type-3 and Dengue-virus type-4, said epitopes<br>
being linked with a predetermined number of glycine linkers, and codon<br>
optimized for expression in E. coli having the following schematic structure:<br>
(Structure Removed)<br>
for the detection of any or all of dengue specific Immunoglobin M (IgM) and/or<br>
(b)	multiepitope of nucleotide sequence as shown Seq ID 3 having one<br>
epitope from core, one epitope from prM, eight epitopes from DEN-2<br>
envelope, four epitopes from DEN-4 NS1, one epitope from DEN-1 NS1, one<br>
epitope from DEN-2 NS 1 and one epitope from DEN-4 NS3, said being<br>
linked with a predetermined number of glycine linkers, and codon optimized<br>
for expression in E. coli having the following schematic representation:<br>
(Structure Removed)<br>
for the detection of any or all of dengue specific Immunoglobin G (IgG).<br>
2.	A recombinant dengue multiepitope as claimed in claim 1 wherein said<br>
multiepitope in paragraph (a) has a amino acid sequence as shown Seq ID 2.<br>
3.	A recombinant dengue multiepitope as claimed in claim 1 wherein said epitopes in paragraph (a) are linked with four glycine linkers.<br>
4.	A recombinant dengue multiepitope (r-DME) as claimed in claim 1 wherein said multiepitope in paragraph (b) has an amino acid sequence as shown in Seq ID 4.<br>
5.	A method for the manufacture of a recombinant dengue multiepitope (r-DME) as claimed in any of the preceding claims for use in the diagnosis and/ or detection of any or all of dengue specific Immunoglobin M (IgM), said method comprising:<br>
(a)	synthesising a gene comprising of immunodominal epitope from<br>
nonstructural protein-1 (NS1) of Dengue-virus type-1, Dengue-virus<br>
type-2, Dengue-virus type-3 and Dengue-virus type-4, said epitopes<br>
being linked with a predetermined number of glycine linkers;<br>
(b)	cloning said synthesised gene in-frame with maltose binding protein in pMAL-c2X vector with MBP fusion and expressed it in E. coli (DH5-a).<br>
(c)	purifying the protein so obtained under denaturing conditions.<br><br>
6.	A method as claimed in claim 5 wherein said E. coli comprises strain DH5-a.<br>
7.	A method as claimed in claim 5 or 6 wherein said purification is carried out under denaturing conditions by Ni-NTA affinity chromatography.<br>
8.	A method as claimed in claim any one of claims 5 to 7 wherein said epitopes are linked with four glycine linkers.<br>
9.	A method for the manufacture of a recombinant dengue multiepitope (r-DME) as claimed in claim 1 for use in the diagnosis and/or detection of any or all of dengue specific Immunoglobin G (IgG), which comprises:<br>
(a)	synthesising a gene comprising of one epitope from core, one epitope from prM, eight epitopes from DEN-2 envelope, four epitopes from DEN-4 NS1, one epitope from DEN-1 NS1, one epitope from DEN-2 NS1 and one epitope from DEN-4 NS3, linking said epitopes with glycine linkers;<br>
(b)	cloning said synthesised gene in pQE-60 and expressed it in E. coli.<br>
(c)	purifying the protein so obtained under denaturing conditions.<br><br>
10.	A method as claimed in claim 9 wherein said epiotpes are linked with three glycine linkers.<br>
11.	A method as claimed in claim 9 or 10 wherein said E. coli comprises strain SG13009.<br>
12.	A method as claimed in any one of claims 9 to 11 wherein said protein is purified under denaturing conditions by ni-NTA affinity chromatography.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUFic3RyYWN0LSgwNC0wNS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Abstract-(04-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLUFic3RyYWN0LSgyNS0wNC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-Abstract-(25-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">974-del-2003-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNsYWltcy0oMDMtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Claims-(03-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNsYWltcy0oMDMtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Claims-(03-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNsYWltcy0oMjItMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Claims-(22-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLUNsYWltcy0oMjUtMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-Claims-(25-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(03-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(03-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(04-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDgtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(08-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-Correspondence Others-(09-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(16-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(22-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjYtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence Others-(26-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTItMDUtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence-Others-(12-05-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Correspondence-Others-(17-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-Correspondence-Others-(25-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWNvcnJlc3BvbmRlbmNlLXBvLnBkZg==" target="_blank" style="word-wrap:break-word;">974-del-2003-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWRlc2NyaXB0aW9uIChwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">974-del-2003-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">974-del-2003-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUZvcm0tMy0oMTItMDUtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Form-3-(12-05-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUdQQS0oMDMtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-GPA-(03-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUdQQS0oMDQtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-GPA-(04-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUdQQS0oMTYtMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-GPA-(16-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLUdQQS0oMTctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-GPA-(17-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LURFTC0yMDAzLVBldGl0aW9uLTEzNy0oMTYtMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">974-DEL-2003-Petition-137-(16-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTc0LWRlbC0yMDAzLXBldGl0aW9uLTEzOC5wZGY=" target="_blank" style="word-wrap:break-word;">974-del-2003-petition-138.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250377-edible-product-containing-ginseng-polysaccharides-and-beneficial-bacteria.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250379-a-process-for-the-preparation-of-stable-form-of-ariprazole.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250378</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>974/DEL/2003</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>01/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Dec-2011</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Aug-2003</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INTERNATIONAL CENTRE FOR GENETIC ENGINEERING AND BIOTECNOLOGY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ARUNA ASAF ALI MARG, NEW DELHI-110067, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NAVIN KHANNA</td>
											<td>INTERNATIONAL CENTRE FOR GENETIC ENGINEERING AND BIOTECNOLOGY, ARUNA ASAF ALI MARG, NEW DELHI-110067, INDIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>A. M. JANA</td>
											<td>GWALIOR 474 002, MADHYA PRADESH, INDIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 7/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250378-novel-recombinant-dengue-multi-epitope-r-dme-as-diagnostic-intermediates by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:28:18 GMT -->
</html>
