<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250481-method-for-measuring-resistance-or-sensitivity-to-docetaxel by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:24:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250481:METHOD FOR MEASURING RESISTANCE OR SENSITIVITY TO DOCETAXEL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR MEASURING RESISTANCE OR SENSITIVITY TO DOCETAXEL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to novel and useful methods that predict or monitor a patient&#x27;s response to a molecule of the taxoid family by measuring the increase or decrease of specific genetic markers as compared to controls. The present invention also provides kits that predict or monitor patients&#x27; response to a molecule of the taxoid family by measuring nucleic acid or protein levels of particular genetic markers and comparing their levels to controls or reference markers.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
METHOD FOR MEASURING RESISTANCE OR SENSITIVITY TO DOCETAXEL<br>
FIELD OF THE INVENTION<br>
The present invention relates to novel, useful and heretofore unknown methods that may predict or monitor a patient's response to a molecule of the taxoid family by measuring the increase or decrease of specific genetic markers as compared to controls. The present invention also provides kits that predict or monitor patients' response to a molecule of the taxoid family by measuring nucleic acid or protein levels of particular genetic markers and comparing their levels to controls or reference markers.<br>
BACKGROUND OF THE INVENTON<br>
Docetaxel is an anti-mitotic drug widely used for the treatment of breast, lung and ovarian cancer, and to a lesser extent used for treating head and neck, gastric and prostate carcinomas (Hong, Oncology 16:9, 2002). Docetaxel inhibits microtubule dynamics by binding to beta-tubulin and blocking disassembly of alpha- and beta-tubulin heterodimers thus abrogating tumor growth (Ringel and Horwitz, J Natl Cancer Inst 83:288, 1991). The antitumor activities of docetaxel and a related taxane, paclitaxel arise from targeting microtubules of the mitotic spindle to impede chromosome alignment and segregation, block cell cycle progression and activate apoptosis pathways (Wang et al, Cancer 88:2619, 2000). The proapoptotic activities of paclitaxel have been linked with the phosphorylation and inactivation of Bcl-2 through cell signaling events that involve p53/p21 wafl/Cipl, raf/ras and mitogen-activated protein kinases (MAPKs) (Wang et al, Cancer 88:2619, 2000). Although docetaxel is a more potent anti-cancer agent than paclitaxel, the pathways involved in its cytotoxicity are less well defined (Katsumata, Br J Cancer 89:S9, 2003). Docetaxel-induced apoptosis has been observed in select cell lines through a mechanism involving Bcl-2 phosphorylation and caspase-3 activation (Kolfschoten et al.9 Biochem Pharmacol 63:733, 2002). Other proteins that modulate taxane-induced cell killing in different cultured lines include Aurora-A in HeLa cells (Anand etal, Cancer Cell 3:51, 2003), HER-2 in breast cancer eel Is (Tanabet, 2003), p21 wafl/Cipl in glioblastoma cells (Li<br><br><br>
There remains a need in the field to identify drugable proteins that mediate docetaxel resistance, which could lead to their utilization in drug discovery to identify reagents (small molecules, siRNA, etc.) that abrogate their in vivo function and potentially sensitize cells to anti-tumor drug chemotherapy. There also remains a need in the field for an assay which could confidently predict in advance of chemotherapy treatment which patients would and would not respond to anti-tumor docetaxel drug based chemotherapy. Applicants describe herein a novel assay that can predict resistance and or sensitivity to docetaxel and provide screening assays for the development of new therapies to abrogate drug resistance in cancer.<br>
SUMMARY OF THE INVENTION<br>
In accordance with the present invention, there is provided novel and useful methods to monitor and/or predict a patient's response to a molecule of the taxoid family by comparing the levels of activation and/or expression of specific genetic markers in patients and controls.<br>
In one embodiment, the present invention provides a method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	obtaining a control sample;<br>
c)	measuring the level of one or more genetic markers; and<br>
d)	comparing the measured levels of said one or more genetic markers in the test sample and the control sample;<br>
wherein a decrease in the level of said one or more genetic markers measured in the test sample as compared to the control sample indicates an increased resistance to a molecule of the taxoid family.<br>
Particular genetic markers provided in this aspect of the present invention include:<br><br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds (GenBank accession number: AF046079);<br>
Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number: AJ000186);<br>
Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number: NM_O03318);<br>
GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT), transcript variant 2, mRNA (GenBank accession number: NM_133483);<br>
Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast) (BUB1), mRNA (GenBank accession number: NM_004336);<br>
hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA (GenBank accession number: NM_012291);<br>
CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase) II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number: NM_001221);<br>
CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank accession number: NM_001259); and<br>
GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript variant 1, mRNA (GenBank accession number: NMJ502086).<br>
In another embodiment, the present invention relates to a method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	obtaining a control sample;<br>
c)	measuring the level of one or more genetic markers; and<br>
d)	comparing the measured levels of said one or more genetic markers in the test sample and the control sample;<br>
wherein a decrease in the level of said one or more genetic markers measured in the test sample as compared to the control sample indicates an increased sensitivity to a molecule of the taxoid family.<br>
In this particular aspect of the invention one or more genetic markers may be selected from the group consisting of:<br><br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl) (CDKN1 A), transcript variant 1, mRNA (GenBank accession number: NM_000389);<br>
Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number: NM_002648);<br>
GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa (GBP1), mRNA (GenBank accession number: NM_002053);<br>
RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank accession number: NM_002957);<br>
SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank accession number: NM_032905);<br>
Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession number: NM_006101);<br>
Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484);<br>
Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds (GenBank accession number: AF008551);<br>
TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3), mRNA (GenBank accession number: NM__006342);<br>
RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA (GenBank accession number: NM_006509);<br>
PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA (GenBank accession number: NM__006254);<br>
BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank accession number: NM_006339);<br>
HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1A), mRNA (GenBank accession number: NM_005345);<br>
STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11), mRNA (GenBank accession number: NM_000455); and<br>
MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete eds (GenBank accession number: L36719).<br><br>
In yet a further embodiment, the present invention relates to a method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	measuring the level of one or more genetic markers selected from the group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds (GenBank accession number: AF046079);<br>
Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number: AJ000186);<br>
Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number: NM_003318);<br>
GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT), transcript variant 2, mRNA (GenBank accession number: NM_133483);<br>
Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast) (BUB1), mRNA (GenBank accession number: NM_004336);<br>
hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA (GenBank accession number: NM_012291);<br>
CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase) II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number: NM_001221);<br>
CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank accession number: NM_001259); and<br>
GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript variant 1, mRNA (GenBank accession number: NMJ302086)<br>
c)	measuring the level of one or more reference genetic markers selected from the<br>
group consisting of:<br>
GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD), mRNA (GenBank accession number: NM_002046); and<br>
RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession number: BC071941);<br>
d)	comparing the measured levels of said one or more genetic markers and said one or<br>
more reference genetic markers in the test sample;<br><br>
wherein a decrease in the level of said one or more genetic markers as compared to the level of said one or more reference genetic markers indicates an increased resistance to a molecule of the taxoid family.<br>
Still yet a further embodiment of the present invention relates to a method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	measuring the level of one or more genetic markers selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl) (CDKN1 A), transcript variant I, mRNA (GenBank accession number: NM_000389);<br>
Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number: NM_002648);<br>
GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa (GBP1), mRNA (GenBank accession number: NM_002053);<br>
RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank accession number: NM_002957);<br>
SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank accession number: NM_032905);<br>
Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession number: NM_006101);<br>
Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484);<br>
Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds (GenBank accession number: AF008551);<br>
TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3), mRNA (GenBank accession number: NM_006342);<br>
RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA (GenBank accession number: NM_006509);<br>
PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA (GenBank accession number: NM_006254);<br><br>
BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank accession number: NM_006339);<br>
HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1 A), mRNA (GenBank accession number: NM_005345);<br>
STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11), mRNA (GenBank accession number: NM_000455); and<br>
MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete cds (GenBank accession number: L36719);<br>
c)	measuring the level of one or more reference genetic markers selected from the<br>
group consisting of:<br>
GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD), mRNA (GenBank accession number: NMJ302046); and<br>
RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession number: BC071941);<br>
d)	comparing the measured levels of said one or more genetic markers and said one or<br>
more reference genetic markers in the test sample;<br>
wherein a decrease in the level of said one or more genetic markers as compared to the level of said one or more reference genetic markers indicates an increased susceptibility to a molecule of the taxoid family.<br>
The present invention relates to methods for predicting or monitoring a cancer patient's response to a molecule of the taxoid family. This, a further embodiment of the present inventions includes the taxoid family molecules paclitaxel, docetaxel XRP9881 and XRP6258<br>
The genetic markers described in the present inventions may be evaluated by measuring the levels of RNA, DNA or protein using a variety of techniques that are described in further detail below.    Other embodiments of the present invention relate to kits for predicting or monitoring a patient's response to a molecule of the taxoid family.<br>
The above aspects and other aspects, features, and advantages of the present invention will be better understood from the following detailed description taken in conjunction with the accompanying figures.<br><br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figure 1 shows the characterization of genes whose downregulation confer docetaxel resistance. Docetaxel generated dose response curves shown for HGT116 cells transfected with RBI, BubRl, Mad2 and Mpsl siRNAs (solid squares) compared to control siRNA (open squares). RBI was an example of a siRNA that did not score in the screen. Cell viability was measured at an absorbance of 450 nM following WST-1 addition. The minimum ratio (MR) is the WST-1 value for a gene over the control at 40 nM docetaxel.<br>
Figure 2 shows dose response curves of BubRl and Mpsl shRNAs (solid squares) compared to control shRNA (open squares).<br>
Figure 3 shows TaqMan real-time PCR analysis of BubRl and Mpsl mRNA levels in cells containing BubRl and Mpsl shRNA, respectively.<br>
Figure 4 shows cell viability and morphology shown for HCT116 cells transfected with Mad2, BubRl and Mpsl siRNAs with or without docetaxel treatment. Twenty-four hours after transfection, cells were left untreated (-Docetaxel) or treated with 200 nM docetaxel (+Docetaxel) and stained with Calcein-AM to visualize live cells 16 and 72 hours after treatment.<br>
Figure 5 shows mitotic indices for HCT116 cells transfected with Mad2, BubRl, Mpsl and control siRNAs following docetaxel treatment. Twenty-four hours after transfection cells were treated with 200 nM docetaxel for 16 and 72 hours and processed. Mitotic cells were detected by a phosphorylated histone H3 antibody, shown here as red punctuate staining over the nucleus of cells (Mitotic Index Kit, Cellomics). Nuclei were stained blue with Hoechst dye.<br>
Figure 6 shows TaqMan real-time PCR analysis of Mpsl, BubRl and Mad2 mRNA levels in HCT116 cells transfected with Mpsl, BubRl and Mad2 siRNA, respectively.<br><br>
ngure / snows cell cycle analysis of HCT116 cells transfected with three mitotic checkpoint gene siRNAs. After transfection with Mpsl, BubRl and Mad2 siRNAs, cells were untreated (-Docetaxel) or treated with 200 nM docetaxel (+DocetaxeI) for 24 hours. Seventy-two hours after the addition of docetaxel, cells were harvested and analyzed for cell cycle. Numbers above peaks indicate DNA content such as 2N, 4N, 8N, 16N or 32N.<br>
Figure 8 shows clonogenic cell survival assay using stable knockdown cell lines. HCT116 cells containing BubRl or vector control shRNAs were plated on 10 cm dishes and maintained in 5 nM docetaxel for 10 days. Colonies were washed with PBS and stained with crystal violet.<br>
Figure 9 shows several dose response curves of genes whose down-regulation confer increased sensitivity to docetaxel. Doectaxel generated dose response curves shown for HCT116 cells transfected with RBI, Pim-1, p21, Aurora A and TACC3 siRNAs (solid squares) compared to control siRNA (open squares). The minimum ration (MR) is the ratio of WST-1 readout of a gene over that of the control at 40 nM docetaxel concentration. The IC50 (IR) ratio is the ratio of IC50 of a gene over that of the control.<br>
Figure 10 shows a dose response curves of Aurora A shRNA (solid squares) compared to vector control (open squares).<br>
Figure 11 shows TaqMan real-time PCR analysis of Aurora A mRNA levels in cells containing Aurora A shRNA.<br>
Figure 12 shows differences in proliferation of HCT116 cells transfected with Pim-1 and TACC3 siRNAs (solid squares) compared to control siRNA (open squares). Number of cells remaining in 6-well plates at different time points after transfection with Pim-1, TACC3 and control siRNAs.<br>
Figure 13 shows TaqMan analysis of Pim-1 and TACC3 mRNA levels after siRNA transfection.<br><br>
Figure 14 shows active caspase-3 levels in HCT116 cells transfected with Pim-1 and BubRl siRNAs. Active caspase-3 levels were shown as fluorescent intensity using the active caspase-3 beadmates kit (Upstate). Three docetaxel concentrations, 0, 5 and 40 nM, and three time points, 24, 48 and 72 hours after docetaxel addition were examined.<br>
Figure 15 shows AKT phosphorylation levels in HCT116 cells transfected with Pim-1 and BubRl siRNAs. The ratio of phosphorylated and total AKT levels was determined using a bead-based assay (Biosource). Twenty-four hours after transfection cells were left untreated or treated with 5 and 40 nM docetaxel. Forty-eight hours after docetaxel treatment cells lysates were analyzed. The phosphorylation level was calculated as the ratio of phosphorylated AKT (p-AKT) over total AKT (t-AKT).<br>
Figure 16 shows Western blots showing the reduced levels of Pim-1 and BubRl 48 hours after transfection.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The present invention is broadly based upon applicants' identification of gene markers that can predict whether a test subject is resistant or sensitive to drugs in the taxoid family. Using a cell-based RNA interference (RNAi) screen, applicants observed that drug resistance or sensitivity to docetaxel was associated with particular gene markers.<br>
It is noted that numerous terms and phases are used throughout the instant specification and claims. Definitions of these terms and phrases are provided below:<br>
As used herein, the term "prognosis" refers to the prediction or the likelihood of cancer-attributable death or progression, including drug resistance as well as recurrence, metastatic spread, and neoplastic disease.<br>
As used herein, the term "prediction" refers to the likelihood that a patient will respond either favorably or unfavorably to a drug or set of drugs, and also the extent of those<br><br>
responses Such as whether the patient will survive, following surgical removal of the primary tumor and/or chemotherapy for a certain period of time without cancer recurrence. Predictive methods envisioned by the present invention can be used clinically to make treatment decisions by choosing the most appropriate treatment modalities, in particular a chemotherapeutic of the taxoid family, for any particular patient. The predictive methods of the present invention are valuable tools in predicting if a patient is likely to respond favorably to a treatment regimen, such as chemotherapy with a given drug or drug combination, and/or radiation therapy, or whether long-term survival of the patient following termination of chemotherapy or other treatment modalities is likely.<br><br>
As used herein, the term "cancer" and "cancerous" refers to a physiological condition in mammals typically characterized by unregulated cell growth. Examples of cancer include but are not limited to the following: breast cancer; colon cancer; lung cancer; prostate cancer; hepatocellular cancer; gastric cancer; pancreatic cancer; cervical and ovarian cancer; liver cancer; bladder cancer; cancer of the urinary tract; carcinoma; melanoma; brain cancer including glioblastomas and medulloblastomas; biliary tract cancer; choriocarcinoma; esophageal cancer; gastric cancer; hematological neoplasms including acute lymphocytic and myelogenous leukemia; multiple myeloma; AIDS-associated leukemia and adult T-cell leukemia lymphoma; intraepithelial neoplasms including Bowen's disease and Paget's disease; lymphomas including Hodgkin's disease and lymnphocytic lymphomas; neuroblastomas; oral cancer including squamous cell carcinoma, sarcomas including leiomyosarcoma, rhabdomyosarcoma, liposarcoma, fibrosarcoma, and osteosarcoma; skin cancer including melanoma, Kaposi's sarcoma, basocellular cancer, and squamous cell cancer; testicular cancer including germinal tumors such as seminoma, non-seminoma (teratomas, choriocarcinomas), stromal tumors, and germ cell tumors; thyroid cancer including thyroid adenocarcinoma and modullar carcinoma; and renal cancer including adenocarcinoma and Wilms tumor..<br>
As used herein, the term "patient" refers preferably to a human but may also include a non-human primate, cow, horse, pig, sheep, goat, dog, cat or rodent. Preferably the subject is a human either suspected of having cancer, or having been diagnosed with cancer or in a high-<br><br>
risk category for developing cancer, for example, having a family history of cancer. In a preferred embodiment of the invention the cancer is breast cancer. Methods for identifying subjects suspected of having cancer may include manual examination, biopsy, subject's family medical history, subject's medical history, or a number of imaging technologies such as mammography, magnetic resonance imaging, magnetic resonance spectroscopy, or positron emission tomography. Diagnostic methods for cancer and the clinical characterizations of cancer diagnoses are well-known to those of skill in the medical arts.<br>
As used herein, the term "sample" is tissue obtained using methods well-known to those of ordinary skill in the related medical arts. Methods such as biopsy include gross apportioning of a mass, microdissection, laser-based microdissection, or other art-known cell-separation methods. Because of the variability of the cell types in diseased-tissue biopsy material, and the variability in sensitivity of the diagnostic methods used, the sample size required for analysis may range from 1,10, 50, 100, 200, 300, 500, 1000, 5000, 10,000, to 50,000 or more cells. The appropriate sample size may be determined based on the cellular composition and condition of the biopsy and the standard preparative steps for this determination and subsequent isolation of the nucleic acid for use in the invention are well known to one of ordinary skill in the art. For example, a sample from a biopsy may be sufficient for assessment of RNA expression without amplification. Conversely, the lack of a suitable number of cells in a small biopsy region may require use of RNA conversion and/or amplification methods or other methods to enhance resolution of the nucleic acid molecules. Such methods, which allow use of limited biopsy materials, are well known to those of ordinary skill in the art. Some examples include but are not limited to direct RNA amplification, reverse transcription of RNA to cDNA, amplification of cDNA, or the generation of radio-labeled nucleic acids.<br>
As used herein, the term kttest " with respect to a sample refers to a sample taken from a cancerous area of the body or from an area of the body exhibiting a stage of or characteristics of cancer.<br>
As used herein, the term "control" with respect to a sample refers to samples which are used for comparative purposes. Preferably, these samples are "control" in the sense that the samples do not exhibit any indication of, or are believed to have, any disease or condition that would affect gene expression, particularly in respect of the disease for which they are to be<br><br>
used as the standard. Alternatively, it may be appreciated that different stages of a disease or condition can be compared and in such cases, the "control" sample corresponds to the earlier stage of the disease or condition. For example, a control sample can be a sample taken from a non-cancerous but comparable area of the body. Additionally, a control sample may be a comparable area of the body from the same subject or a non-cancerous area of the body from a second subject substantially similar to the first subject (same or similar species, age, weight, sex, etc.). Finally, a control sample could also be from a cancerous area of a second subject who respond well to treatment using a molecule of the taxoid family.<br>
A "nucleic acid molecule11 refers to the phosphate ester polymeric form of ribonucleosides (adenosine, guanosine, uridine or cytidine; "RNA molecules") or deoxyribonucleosides (deoxyadenosine, deoxyguanosine, deoxythymidine, or deoxycytidine; "DNA molecules"), or any phosphoester analogs thereof, such as phosphorothioates and thioesters, in either single stranded form, or a double-stranded helix. Double stranded DNA-DNA, DNA-RNA and RNA-RNA helices are possible. The term nucleic acid molecule, and in particular DNA or RNA molecule, refers only to the primary and secondary structure of the molecule, and does not limit it to any particular tertiary forms. Thus, this term includes double-stranded DNA found, inter alia, in linear or circular DNA molecules {e.g., restriction fragments), plasrnids, and chromosomes. In discussing the structure of particular double-stranded DNA molecules, sequences may be described herein according to the normal convention of giving only the sequence in the 5' to 3' direction along the non-transcribed strand of DNA (i.e., the strand having a sequence homologous to the mRNA). A "recombinant DNA molecule" is a DNA molecule that has undergone a molecular biological manipulation.<br>
As used herein, the term "portion" of an isolated nucleic acid molecule that encodes for a particular protein, refers to a part or fragment of the isolated nucleic acid molecule that comprises a sufficient number of contiguous nucleotides that encode for a peptide or polypeptide. Naturally, a "portion" of an isolated nucleic acid molecule is greater than one nucleotide, and the peptide or polypeptide encoded by the portion contains numerous amino acid residues, as described in the definitions of peptide and polypeptide below.<br>
As used herein, the term "peptide" refers to two or more amino acids covalently joined by peptide binds. In a particular embodiment, a peptide comprises at least 10, preferably at least 20, more preferably at least 30, even more preferably at least 40, and most preferably 50 or more amino acids.<br><br>
As used herein, the term "polypeptide" refers to a linear polymer composed of multiple contiguous amino acids. In particular, a polypeptide may possess a molecular weight greater than 100 kD.<br>
As used herein, the term "genetic marker" refers to a physiological composition whose measured R"NA, D"NA or protein level within a sample serves to predict whether a test subject is resistant or sensitive to drugs in the taxoid family. Moreover, a genetic marker may encode the particular protein or alternatively, may serve as a "surrogate" marker for a protein whose activity is related to the level of the genetic marker in a bodily sample. This relationship may be direct, wherein a decrease in the level of protein activity corresponds to a decrease in the level of the genetic marker, or alternatively, the relationship may be inverse, wherein a decrease in the level of protein activity corresponds to an increase in the level of the genetic marker. Such physiological compositions include, but certainly are not limited to, cells (e.g., progenitor stem cells) proteins, polypeptides, DNA, RNA, carbohydrates, or fatty acids, to name only a few. In a particular embodiment of the present invention, the measured levels of certain gene markers can predict whether a test subject is resistant or sensitive to drugs in the taxoid family. Examples of such genetic markers include, but certainly are not limited to:<br>
BubRl, Homo sapiens similar to protein kinase (BUBRI) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number:<br>
NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: NM_133483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast)<br>
(BUB1), mRNA (GenBank accession number: NMJJ04336); hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase) II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number: NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank accession number: NM_001259); and<br><br>
GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086) P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21. Cipl)<br>
(CDKN1 A), transcript variant 1, mRNA (GenBank accession number: NMJ)00389); Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM_002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NM_032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession<br>
number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3),<br>
mRNA (GenBank accession number: NM_006342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA<br>
(GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA<br>
(GenBank accession number: NMJ306254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NMJ306339); HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1 A), mRNA (GenBank<br>
accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11),<br>
mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete eds (GenBank<br>
accession number: L36719)<br><br>
As used herein, the term "reference genetic marker" refers to a physiological composition whose measured RNA, DNA or protein levels remain unchanged before, during or after exposure to drugs in the taxoid family. "Reference genetic markers" are referred to as housekeeping genes. These are genes that are selected based on the relatively invariable levels of expression in the system which is being examined, for example, the in a particular disease such as cancer. Housekeeping genes are used to normalize results of expression. Examples of such genetic markers include, but certainly are not limited to:<br>
GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD),<br>
mRNA (GenBank accession number: NM_002046); and RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession number: BC071941).<br>
As used herein, the term "a molecule of the taxoid family" refers to a class of chemotherapeutic compounds to belonging to the taxane family. Specific members of the taxoid family include but are not limited to paclitaxel (taxol), docetaxel (taxotere) and analogs thereof (/.e.,XRP9881 and XRP6258; see Ojima and Geney, Curr Opin Investig Drugs 4:737, 2004). Since this class of molecules are beta-tubulin binders and stabilize the polymerized form of the microtubule similar to docetaxel, it is anticipated that the clinical expresson of the biomarker herein described will reflect similar response states to these drugs.<br>
As used herein, the terms "molecule", "compound" or "agent" refer to any composition presently known or subsequently discovered. Examples of compounds or agents having applications herein include organic compounds (e.g., man made, naturally occurring and optically active), peptides (man made, naturally occurring, and optically active, i.e., either D or L amino acids), carbohydrates, nucleic acid molecules, etc.<br>
As used herein, "stringency of hybridization" or 'hybridization under stringent conditions" refers to conditions readily determinable by one of ordinary skill in the art, and generally is an empirical calculation dependent upon probe length, washing temperature, and salt concentration. In general, longer probes require higher temperatures for proper annealing, while shorter probes need lower temperatures. Hybridization generally depends on the ability of denatured DNA to reanneal when complementary strands are present in an environment below their melting temperature. The higher the degree of desired homology between the<br><br>
probe and hybridizable sequence, the higher the relative temperature which can be used. As a result, it follows that higher relative temperatures would tend to make the reaction conditions more stringent, while lower temperatures less so. For additional details and explanation of stringency of hybridization reactions, see Ausubel et al., Current Protocols in Molecular Biology, Wiley lnterscience Publishers, (1995).<br>
As used herein, "stringent conditions" or "high stringency conditions", refers to those parameters that: (1) employ low ionic strength and high temperature for washing, for example 0.015 M sodium chloride/0.0015 M sodium citrate/0.1% sodium dodecyl sulfate at 50° C; (2) employ during hybridization a denaturing agent, such as formamide, for example, 50% (v/v) formamide with 0.1% bovine serum albumin/0.1% FicoIl/0.1% polyvinylpyrrolidone/50 mM sodium phosphate buffer at pH 6.5 with 750 mM sodium chloride, 75 mM sodium citrate at 42° C; or (3) overnight hybridization in a solution that employs 50% formamide, 5x SSC (0.75 M NaCl, 0.075 M sodium citrate), 50 mM sodium phosphate (pH 6.8), 0.1% sodium pyrophosphate, 5x Denhardt's solution, sonicated salmon sperm DNA (50 .mu.g/ml), 0.1% SDS, and 10% dextran sulfate at 42° C, with a 10 minute wash at 42°C in 0.2x SSC (sodium chloride/sodium citrate) followed by a 10 minute high-stringency wash consisting of 0.1 x SSC containing EDTA at 55° C. Moderately stringent conditions may be identified as described by Sambrook et al., Molecular Cloning: A Laboratory Manual, New York: Cold Spring Harbor Press, 1989, and include the use of washing solution and hybridization conditions (e.g., temperature, ionic strength and % SDS) less stringent that those described above. An example of moderately stringent conditions is overnight incubation at 37°C in a solution comprising: 20% formamide, 5x SSC (150 mMNaCl, 15 mM trisodium citrate), 50 mM sodium phosphate (pH 7.6), 5x Denhardt's solution, 10% dextran sulfate, and 20 mg/ml denatured sheared salmon sperm DNA, followed by washing the filters in lx SSC at about 37-50°C. The skilled artisan will recognize how to adjust the temperature, ionic strength, etc. as necessary to accommodate factors such as probe length and the like. The appropriate stringency for hybridizing nucleic acids depends on the length of the nucleic acids and the degree of complementation, variables well known in the art. The greater the degree of similarity or homology between two nucleotide sequences, the greater the value of Tm for hybrids of nucleic acids having those sequences. The relative stability (corresponding to higher Tm) of nucleic acid hybridizations decreases in the following order: RNA:RNA, DNA:RNA, DNA:DNA. Preferably a minimum length for a hybridizable nucleic acid is at least about 12<br><br>
nucleotides; preferably at least about 16 nucleotides; and more preferably the length is at least about 24 nucleotides; and most preferably at least 36 nucleotides.<br>
As used herein, "label" or "detectable-label" refers to a detectable label compound or composition which is conjugated directly or indirectly to an antibody, oligopeptide or other organic molecule so as to generate a "labeled" antibody, oligopeptide or other organic molecule. The label may be detectable by itself (e.g. radioisotope labels or fluorescent labels) or, in the case of an enzymatic label, may catalyze chemical alteration of a substrate compound or composition which is detectable.<br>
As used herein, a "direct labels" refers an entity, which in its natural state, is readily visible, either to the naked eye, or with the aid of an optical filter and/or applied stimulation, e.g. ultraviolet light to promote fluorescence. Examples include, but are not limited to, colored labels, metallic sol particles, dye sol particles dyed latex or dyes encapsulated liposomes (described by U.S. Pat. No. 4,313,734, U.S. Pat. No. 4,373,932, WO 88/08534), EP-A 0 280 559, 0 281 327, U.S. Pat. No. 4,703,017). Other direct labels include a radionucleotide, radiopaque substances, a fluorescent moiety or a luminescent moiety.<br>
As used herein, "indirect labels" refers to enzymes can also be used according to the present invention. Various types of enzyme linked immunoassays are well known in the art, for example, alkaline phosphatase and horseradish peroxidase, lysozyme, glucose-6-phosphate dehydrogenase, lactate dehydrogenase, urease, these and others have been discussed in detail by Eva Engvall in Enzyme Immunoassay ELISA and EMIT in Methods in Enzymology, 70:419-439 (1980) and in U.S. Pat. No. 4,857,453.<br>
As used herein, the term "kit" refers to an article of manufacture comprising one or more containers and a label or package insert on or associated with the containers. In a preferred embodiment, the containers may contain a detectable-labeled antibody, detectable-labeled antibody fragments or detectable-labeled oligonucleotides. In another embodiment, the containers provide the means to obtain total RNA from a bodily sample, reverse transcribe the total RNA to obtain cDNA and subject the cDNA to a polymerase chain reaction using a set of primers wherein one or both primers are detectable-labeled. Additional containers of the kit may be included that contain, e.g., diluents and buffers, control antibodies, oligopeptides or<br><br>
small organic molecules. The label or package insert may provide a description of the composition as well as instructions for storage and for the intended in vitro or diagnostic use.<br>
Furthermore, in accordance with the present invention there may be employed conventional molecular biology, microbiology, and recombinant DNA techniques within the skill of the art. Such techniques are explained fully in the literature. See, e.g., Sambrook, Fritsch &amp; Maniatis, Molecular Cloning: A Laboratory Manual, Second Edition (1989) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York (herein "Sambrook et a/., 1989"); DNA Cloning: A Practical Approach, Volumes I and II (D.N. Glover ed. 1985); Oligonucleotide Synthesis (M.J. Gait ed. 1984); Nucleic Acid Hybridization [B.D. Hames &amp; S.J. Higgins eds. (1985)]; Transcription And Translation [B.D. Hames &amp; S.J. Higgins, eds. {\9%A)}\ Animal Cell Culture [R.I. Freshney, ed. (1986)]; Immobilized Cells And Enzymes [IRL Press, (1986)]; B. Perbal, A PracticalI Guide To Molecular Cloning (1984); F.M. Ausubel et al. (eds.), Current Protocols in Molecular Biology, John Wiley &amp; Sons, Inc. (1994).<br>
In general, RNAi technology uses synthetic or vector-generated double stranded RNA to induce the degradation of mRNA containing homologous sequences (McManus and Sharp, Nat Rev Genet 3:131, 2002). RNAi screening has been used to elucidate gene function in many organisms such as Caenorhabditis elegans (Simmer e/A/., PloS Biol 1:E12, 2003), Drosophila (Lum et al, Science 299:2039, 2003) and mammalian cells (Aza-Blanc et al, Mol Cell 12:621,2003).<br>
In the present invention, HCT116 colon cancer cells were used to screen siRNAs directed against 101 cancer-related genes, mostly comprising the kinase family members, and the docetaxel killing effects were quantified by high-resolution dose-response curves. Using this approach, applicants have shown that under-expression of 9 genes, including BubRl, Bubl, Mad2, Mpsl and GEFT Rac/CDC, can inhibit killing by docetaxel, whereas under-expressing 15 others, including Piml, p21, TACC3 and Aurora-A, can potentiate docetaxel-induced cell death.<br><br>
Thus, broadly the present invention extends to diagnostic methods and kits that may be used to identify whether an individual who is either taking or considering taking docetaxel will be resistant or sensitive to the drug.<br>
Applicants have observed drug resistance to docetaxel, whereby drug resistance is associated with the loss of an expanded set of checkpoint control genes including BubRl, Bubl, Mad2, Mspl and GEFT Rac/CDC.<br>
Increased docetaxel sensitivity corresponds with the loss of any one of several genes, including GBP-1, STK11, RXRA, Heel, SPF45, Rafl, RELB, MKK3, PRKCD, HSPA1A, BRAF35, Aurora-A, Pim-1, TACC3 and p21wafl/Cipl, and inhibitors against these genes may serve as a valuable adjunct to docetaxel therapy.<br>
The mitotic checkpoint genes serve as a failsafe mechanism to delay anaphase by the inhibition of Cdc20-APC (anaphase-promoting complex), until the kinetochores of sister chromatids are properly attached to the mitotic spindle and aligned at the equator (Zhou et ah, J Cell Sci 115:3547, 2002), thus allowing cells to reliably exit mitosis and enter additional cell cycles with a precise complement of DNA. In the presence of microtubule inhibitors, cells exiting from mitosis can no longer properly segregate their chromosomes and typically undergo apoptosis immediately or after several cell cycles (Taylor and McKeon, Cell 89:727, 1997). The combination of microtubule inhibitors with loss of mitotic checkpoint genes such as BubRl allows cells to escape mitotic arrest without undergoing apoptosis and lead to chromosome instability (CIN) and aneuploidy (Shin et al, Cancer Cell 4:483, 2003). Recently, drug resistance data were generated when microtubule inhibitor paclitaxel was used to treat BubRl and Mad2 down regulated MCF-7 cells (Sudo et ai, Cancer Res 64:2502, 2004).<br>
The Pim-1 gene encodes a serine/threonine kinase that belongs to a small family of related kinases. Pim-1 function has been linked to proliferation, differentiation, apoptosis, tumorigenesis, hypoxia, angiogenesis and mitosis (Wang et al, Biochim Biophys Acta 1593:45, 2002). The microscopic imaging data, in conjunction with AKT hypo-phosphorylation data and elevated caspase-3 activity measurements, demonstrate that Pim-1 down regulation enhanced docetaxel-induced apoptosis by inactivating AKT signaling, which<br><br>
mediates cell survival. Pim-1 and p2I have very similar characteristics in sensitizing HCT116 cells to docetaxel; consistent with reports showing that p21 is a phosphorylation substrate of Pim-1 (Wang et al, Biochim Biophys Acta 1593:45, 2002). Pim-1 down regulation induced some apoptosis in the absence of docetaxel but was more effective in the presence of drug, whereas down regulation of transforming acidic coiled-coil protein TACC3 did not induced cell death in the absence of docetaxel, indicating the RNAi alone had na effect on cell proliferation. These results suggest that depletion of different proteins can evoke different mechanisms for sensitizing cells to docetaxel.<br>
Accordingly, in one aspect of the invention, activation of one or more of the genes described in Table I and II and protein levels of these genes may be utilized to select a therapeutic treatment strategy and to monitor effectiveness of the selected treatment strategy.<br>
In a particular embodiment, a method is provided that monitors the activation levels of one or more genes shown in Table I and II during the course of chemotherapy to assess and predict effectiveness of chemotherapy for the patient. Biopsies from tumors {e.g., breast, colon, non-small cell lung, and gastric tumors) or short-term culture of tumor biopsies from undiagnosed cancer patients or patients currently undergoing treatment is processed in order to isolate DNA (Hafner et al.Arch Pathol Lab Med 127:1221, 2003; Rodriguez et al, Clin Cancer Res 10:5785, 2004), messenger RNA (Chang et al., Lancet. 362:340, 2003) and/or proteins (Espina et al, J Immunol Methods 290:121, 2004) to assess the levels of gene activation or protein levels of the genes described in Table I and II or a subset thereof. Gene transcription and protein expression profiling has utility in diagnosis, prediction of therapeutic responses to potential treatment strategies and may be a useful tool to monitor a patient's response to therapy.<br>
According to the present invention, RNA may be isolated from tumor samples using any of the available methods commonly used in the art (Ullmann et al, J Biomol Screen 9:95, 2004; Badiee et al, BMCBiotechnol 3:23, 2003). One or more cells from test subjects are obtained and RNA is isolated from these cells. As examples, peripheral blood leukocytes (PBLs) cells may be obtained from the subject or it is also possible to obtain a cell sample and enrich the sample for a desired cell type. Cells may be isolated from a cell mixture using a variety of.techniques, such as cell isolation using antibodies that bind to an particular epitope<br><br>
on the desired cell type. Where the desired cells are in a solid tissue, particular cells may be dissected, for example, by microdissection or by laser capture microdissection (LCM) (Bonner etaL, Science 278:1481, 1997; Fend etal,AmJPath 154:61, 1999). RNA may be extracted from tissue or cell samples by a variety of methods, for example, guanidium thiocyanate lysis followed by CsCl centrifugation (Chirgwin et al9 Biochemistry 18:5294, 1979). RNA from single cells may be obtained as described in methods for preparing cDNA libraries from single cells ( Dulac, Curr Top Dev Biol. 36:245, 1998). The RNA sample can be further enriched for a particular species. In one embodiment, for example, poly(A)+RNA may be isolated from an RNA sample. In particular, poly-T oligonucleotides may be immobilized on a solid support to serve as affinity ligands for mRNA. Kits for this purpose are commercially available, for example, the MessageMaker kit (Life Technologies, Grand Island, N.Y.). Enrichment may be accomplished, for example, by primer-specific cDNA synthesis, or multiple rounds of linear amplification based on cDNA synthesis and template-directed in vitro transcription (Wang et aL, Proc Natl Acad Sci USA 86:9717, 1989; Dulac etal, supra).<br>
A variety of amplification methods are suitable for use in the methods of the present invention, including, for example, PCR; ligase chain reaction (LCR) (Wu and Wallace, Genomics 4:560, 1989); self-sustained sequence replication (SSR) (Guatelli et aL, Proc Natl Acad Sci USA 87:1874, 1990); nucleic acid based sequence amplification (NASBA) and transcription amplification (Kwoh et aL, Proc Natl Acad Sci USA 86:1173, 1989). Methods for PCR technology are well known in the art (see, e.g., PCR Technology: Principles and Applications for DNA Amplification (ed. H. A. Erlich, Freeman Press, N.Y., N.Y., 1992); PCR Protocols: A Guide to Methods and Applications (eds. Innis, et al., Academic Press, San Diego, Calif, 1990).<br>
The RNA may be labeled for detection by using any of the methods known to those persons skilled in the art. For example, using microarrays, such as those manufactured by Affymetrix. RNA is labeled and detected by hybridization using any of the methods commonly used, such as, direct RNA labeling with dyes such as Cyanine-3 or Cyanine-5 (Perkin Elmer MPS544001KT). Additional methods include direct cDNA labeling that is done by reverse transcription using nucleotides labeled with Cyanine-3 or Cyanine-5 (Badiee A 2003), fluorescein or Alexa dyes (Molecular Probes), or other fluorophores. Furthermore, indirect cDNA labeling such as FairPlay™/aminoallyl labeling (Stratagene Catalog number:<br><br>
252002), 3DNA dendrimer labeling (Genisphere Inc) or enzymatic signal amplification (MICROMAX TSA, Perkin Elmer) may be utilized. Alternatively, RNA may be quantified by using RNA probes that are detected by enzymatic, fluorescence, radioactive or light emitting methods.<br>
In one aspect of the present invention, RNA is obtained from the tumor sample and Taqman technology is employed in the detection of genetic markers. Primers are generated to detect the specific RNAs designated in Table I and II or subset thereof and amplified using reverse transcriptase. Detection of the specific fragment amplified by reverse transcription is accomplished by gel electrophoresis, polymer electrophoresis, direct DNA sequencing, light detection, loss of a fluorescence signal, enzymatic reaction, or detection by hybridization to a complementary RNA.<br>
In another aspect of the present invention, Real Time PCR is utilized to detect oligonucleotides. The mRNA obtained from a tumor sample is reverse translated into cDNA. Primers are generated to amplify the specific RNAs designated in Table I and II or subset thereof using polymerase chain reaction technology. Detection of mRNA level is achieved by gel electrophoresis and ethidium bromide or CYBR green staining, or by measuring the fluorescence intensity from fluorophore-labeled sequence specific probes released by polymerase.<br>
In a further aspect of the present invention, Northern blotting technology is utilized. RNA is obtained from the tumor sample and then separated by gel electrophoresis and transferred onto a membrane. RNA abundance is determined by hybridization using probes labeled with radioactive isotope such as P ~ or by enzymatic based chromogenic or luminescent assays.<br>
In yet another aspect of the present invention, gene dosage may be used as a surrogate indicator of transcript levels. The gene dosage of the transcripts in Table I and II or a subset thereof may be quantitatively determined and assessed for whether a tumor will be responsive or not to docetaxel therapy. (Rodriguez et ah, Clin Cancer Res 10:5785, 2004). Since this assessment may be drawn prior to neoadjuvant (treatment prior to surgery) therapy, the patient<br><br>
may be treated with docetaxel if they are classified as a responder or another chemotherapeutic agent may be administered if they are classified as a non-responder.<br>
DNA is extracted from patient tumor biopsies and purified to remove contaminants using methods as performed by those skilled in the art. Methods utilized to determine the gene dosage in the tumor DNA include, but are not limited to, quantitative PCR, genomic DNA-chips, in situ hybridization or Southern (Hafner et al, Arch Pathol Lab Med 127:1221, 2003; Rodriguez et ah, Clin Cancer Res 10:5785, 2004). Accordingly, these methods may be utilized to determine copy number of one or more genes in Table 1 and II and compared to a control sample or reference markers.<br>
In yet another aspect of the present invention, protein levels are used as a surrogate measure of transcript levels. Protein levels have been shown to correlate with suppression or elevation of the transcript levels. Thus, the present invention encompasses techniques to measure levels of proteins that correspond to the polypeptide products encoded by the transcripts in Table I and II. Proteins are detected and used as markers of a predictive response to docetaxel. Methods of protein detection are well known in the art. For example, immunoassays are methodologies that employ antibodies to detect the expression of a target protein.<br>
In a particular embodiment, an immunoassay is used to detect the protein products of one or more genes listed in Table I and II. Additionally, antibodies against any one of the proteins listed in Table I and II may be used in a number of other detection methods. These detection methods include, but are not limited to, the following: Western blots, ELISA assays, sandwich ELISA. Alternatively, other assays not employing antibodies are contemplated and include approaches that utilize DNA oligonucleotides or polypeptides that recognize proteins encoded by transcripts in Table I and II (e.g., aptamers).   Mass spectrometry analysis of biological samples may also be used that determine the composition of polypeptides by determining the precise molecular weight of peptide fragments after proteolytic cleavage.<br>
The present invention further encompasses suitable labels including enzymes, fluorophores (e.g., fluorescein isothiocyanate (FITC), phycoerythrin (PE), Texas red (TR), rhodamine, free or chelated lanthanide series salts, chromophores, radioisotopes, chelating<br><br>
agents, dyes, colloidal gold, latex particles, ligands (e.g., biotin), and chemiluminescent agents.<br>
In an aspect of the present invention, the radioactive label used may be isotopes such as3H, 14C,32P, 35S, 36C1, 51 Cr, 57Co, 58Co,59Fe, 90Y, 125 1, 1311, and 186Re. Quantification of radioactivity levels may be performed through currently known and available counting procedures. Conversely, the use of radiopaque substances may be used. In the embodiment where the label is an enzyme, detection may be accomplished by any of the presently utilized colorimetric, spectrophotometry, fluorospectrophotometric, amperometric or gasometric techniques known in the art.<br>
A further aspect of the invention is the use of peptide binding agents such as antibodies or fragments of antibodies. Antibodies include polyclonal and monoclonal antibodies, prepared according to conventional methodology. Only a small portion of an antibody molecule, the paratope, is involved in the binding of the antibody to its epitope (see, in general, Clark, W. R. (1986) The Experimental Foundations of Modem Immunology Wiley &amp; Sons, Inc., New York; Roitt, I. (1991) Essential Immunology, 7th Ed., Blackwell Scientific Publications, Oxford). The pFc' and Fc regions, for example, are effectors of the complement cascade but are not involved in antigen binding. An antibody from which the pFc' region has been enzymatically cleaved, or which has been produced without the pFc' region, designated an F(ab')2 fragment, retains both of the antigen binding sites of an intact antibody. Thus, an embodiment of the present invention utilizes a detectable-lableled antibody fragment such as the F(ab')2 fragment. Similarly, an antibody from which the Fc region has been enzymatically cleaved, or which has been produced without the Fc region, designated an Fab fragment, retains one of the antigen binding sites of an intact antibody molecule. Fab fragments consist of a covalently bound antibody light chain and a portion of the antibody heavy chain denoted Fd. The Fd fragments are the major determinant of antibody specificity (a single Fd fragment may be associated with up to ten different light chains without altering antibody specificity) and Fd fragments retain epitope-binding ability in isolation.<br>
As is well-known in the art, within the antigen-binding portion of an antibody, there are complementarity determining regions (CDRs), which directly interact with the epitope of the antigen, and framework regions (FRs), which maintain the tertiary structure of the<br><br>
paratope (see, in general, Clark, 1986; Roitt, 1991). In both the heavy chain Fd fragment and the light chain of IgG immunoglobulins, there are four framework regions (FR1 through FR4) separated respectively by three complementarity determining regions (CDR1 through CDR3). The CDRs, and in particular the CDR3 regions, and more particularly the heavy chain CDR3, are largely responsible for antibody specificity. It is now well-established in the art that the non-CDR regions of a mammalian antibody may be replaced with similar regions of conspecific or heterospecific antibodies while retaining the epitopic specificity of the original antibody. This is most clearly manifested in the development and use of "humanized" antibodies in which non-human CDRs are covalently joined to human FR and/or Fc/pFc' regions to produce a functional antibody (refer to U.S. Pat. Nos. 4,816,567, 5,225,539, 5,585,089, 5,693,762 and 5,859,205).<br>
Fully human monoclonal antibodies also can be prepared by immunizing mice transgenic for large portions of human immunoglobulin heavy and light chain loci. Following immunization of these mice (e.g., XenoMouse (Abgenix), HuMAb mice (Medarex/GenPharm)), monoclonal antibodies can be prepared according to standard hybridoma technology. These monoclonal antibodies will have human immunoglobulin amino acid sequences and therefore will not provoke human anti-mouse antibody (HAMA) responses when administered to humans.<br>
Therefore, it is readily apparent to one of ordinary skill in the art, the present invention also encompasses F(ab')2, Fab, Fv and Fd fragments; chimeric antibodies in which the Fc and/or FR and/or CDR1 and/or CDR2 and/or light chain CDR3 regions have been replaced by homologous human or non-human sequences; chimeric F(ab')2 fragment antibodies in which the FR and/or CDR1 and/or CDR2 and/or light chain CDR3 regions have been replaced by homologous human or non-human sequences; chimeric Fab fragment antibodies in which the FR and/or CDR1 and/or CDR2 and/or light chain CDR3 regions have been replaced by homologous human or non-human sequences; and chimeric Fd fragment antibodies in which the FR and/or CDR1 and/or CDR2 regions have been replaced by homologous human or non-human sequences. The present invention also includes so-called single chain antibodies.<br>
The present invention may be better understood by reference to the following non-limiting Examples, which are provided as exemplary of the invention. The following<br><br>
Examples are presented in order to more fully illustrate the preferred embodiments of the invention. They should in no way be construed, however, as limiting the broad scope of the invention.<br>
EXAMPLES<br>
EXAMPLE 1: siRNA Screening<br>
In order to identify genes that render cells more resistant or sensitive to docetaxel treatment, siRNAs against 101 genes were screened in HCT116 cells using a range of docetaxel concentrations for the generation of dose-response curves. Human colon cancer cell line HCT116 was obtained from ATCC and cultured in McCoy 5A supplemented with 100 units/ml penicillin, 100 jag/ml streptomycin, 4 mM L-glutamine and 10% fetal bovine serum at 37°C with 95% C02 and 5% 02. The siRNA list was comprised of cancer-related genes, including those that were over expressed in docetaxel-resistant breast tumors as detected by gene expression profiling experiments (Chang et ah, Lancet 362:362, 2003). Additional criteria were that the targets be amenable to drug development. The overall scheme is summarized as follows. On the first day, HCT116 cells were plated at 5,000 cells per well in a 96-well format and on the following day transfected with a pool of 3-4 siRNAs per gene. A representative sample of the siRNAs used is shown in Table 1 and II. Lipofectamine 2000 (Invitrogen) was used to transfect siRNAs (Dharmacon) into HCT 116 cells in 96 well plates. On the third day siRNA was removed and docetaxel was added at concentrations ranging from 0 to 40nM. On the sixth day cell viability was quantified using WST-1 assay. This is an assay that monitors the activity of mitochondrial dehydrogenases present in viable cells that results in the cleavage of tetrazolium salt WST-1 to form formazan. On the day of assay, media were taken off from 96 well plates. WST-1 reagent (Roche) was diluted 10 times in McCoy 5A medium and lOOul was added to each well. Plates were then incubated for 40 - 80 minutes at 37°C before reading on SpectroMax (Molecular Devices) at 450nm. The WST-1 values were clustered hierarchically by calculating the ratio of the experimental to the control siRNA and a total of 15 genes were characterized as conferring sensitivity, and 9 genes conferring resistance shown in Table III below. The sensitive and resistant populations could be further divided into additional groups: 1) where the siRNA effect was observed at lower<br><br>
concentrations of docetaxel (1-6 nM) to shift the IC50 or 2) where the major effect was observed at higher concentrations (&gt; 6nM) with a smaller shift in IC50.<br>
As compared to Mad2, BubRl and Mpsl siRNAs, the siRNAs targeting Grb2, CDK6, sepharase, and calcium/calmodulin-dependent protein kinase II delta (CamKIID) showed more pronounced effects at lower docetaxel concentrations (1-6 nM), with a characteristic shift in the IC50. Overall, this subgroup showed weaker resistance than the mitotic checkpoint genes and the CamKIID siRNA generated the greatest level of protection, which was two-fold over control cells. These data demonstrate that loss of four mitotic checkpoint genes along with several other genes can increase docetaxel resistance to varying degrees. The data in Figure 1 show several characteristic dose-response curves for cells transfected with siRNAs followed by exposure to various concentrations of docetaxel. The siRNAs against four mitotic checkpoint genes, including BubRl, Bubl, Mad2, and Mpsl (data not shown for Bubl) as well as guanine nucleotide exchange factor (GEFT) for Rac/Cdc42 demonstrated significant docetaxel resistance with the effect being more pronounced at higher docetaxel concentrations (&gt;6 nM). The greatest level of resistance was shown for Mad2 where cell viability increased five-fold over the control siRNA transfected cells. The majority of the siRNAs did not score in the screen and here we show human retinoblastoma 1 (RBI) as a representative example.<br>
Taken together, these results show that siRNA screening can provide insights about how down-regulating gene expression sensitizes or protects cells from docetaxel treatment and, depending on the concentrations, evokes different mechanisms of action to modulate docetaxel-mediated cell death.<br><br><br><br><br><br><br><br>
EXAMPLE 2: Cross Validation on a Subset of siRNAs that Confer Increased Docetaxel Resistance<br>
A potential drawback to siRNA screening approaches rests on the observation that siRNAs can anneal to transcripts with partial identity or initiate micro-RNA (miRNA) translational blocks, making it difficult to distinguish between target-specific effects and nonspecific (off-site) effects (Jackson et aln Nat Biotechnol 21:635, 2003). To confirm resistance using an independent approach with a second sequence, stable knock-down cells lines were generated for BubRl and Mpsl by infecting HCT116 cells with a retroviral vector carrying DMA oligonucleotides encoding a short hairpin (sh)RNA. The shRNA contained a different sequence than the siRNA, targeting a distinct region of the open reading frame. The retroviral packaging cell line GP2-293 was obtained from Clontech. The cells were maintained in DMEM supplemented with 100 units/ml penicillin, 100 jxg/ml streptomycin, 4 mM L-glutamine and 10% fetal bovine serum. Virus was generated by transiently transfecting GP2-293 cells. A total of 3.6 x 106 cells were seeded in a 10 cm dish 24 hours prior to transfection. The medium was replaced 4 hours prior to transfection with DMEM containing 10% FBS minus antibiotics. The cells were transfected with 6 (ig vector DNA, 6 j-ig of envelope plasmid VSV-G (Clontech) and 72 ^1 Lipfectamine-2000. The medium was replaced after 14-16 hours; the viral supernatant was harvested 24 hours later, filtered through a 0.45 (J.M filter and used to infect HCT116 cells at an M.O.I of 5 in the presence of 8 (ig/ml polybrene. Forty-eight hours post-infection the cells were selected in 0.5 |~ig/ml puromycin for 7 days or until the background cells died off. The dose response curves generated with the BubRl and Mpsl knock-down cell lines again showed enhanced resistance that was more pronounced at higher docetaxel concentrations (Figure 2), similar to the results obtained by siRNA transient transfections. To examine whether the shRNAs reduced mRNA levels we employed real-time PCR to detect endogenous transcripts and confirmed that the RNA levels were diminished in the BubRl and Mpsl stable knock-down cell lines as compared to the vector control stable line (Figure 3).<br>
Cells were lysed and RNA was extracted using RNAqueous -96 (Ambion). Taqman™ probes and forward and reverse primers were designed with Primer Express™ software (PE Applied Biosystems, UK). BLAST searches of the probe and primer sequences revealed no significant identity to other sequences other than the specific gene under test. For real time<br><br>
PCR, 20 ill of master TaqMan™ mix was made for each well: 4.825 jul Rnase-free water, 12.5 [i\ 2x Universal PCR master mix and 0.625 JJ.1 40x MultiscribeTM and Rnase Inhibitor Mix (Applied BioSystems), 0.9 jil forward and reverse primers (100 JIM), 0.25ul probe (100 JIM). To each well, 5ul sample RNA (about I ng/jul) was added. Plates were analyzed on a TaqMan™ ABI Prism 7700 Sequence Detector™ (Perkin-Elmer, UK). Cycling parameters were 48°C for 30 minutes, 95°C for 10 minutes, 40 cycles of 95°C for 15 seconds, and 60°C for 1 minute. Test gene mRNA values were extrapolated from the standard curve and presented as percent remaining.<br>
EXAMPLE 3: Downregulation of Mad2, BubRl and Mpsl Protects Cells from Docetaxel-Induced Death by Escaping Mitotic Arrest, Coupled with the Generation of Aneuploidy<br>
As determined by the WST-1 assay, loss of mitotic checkpoint genes in the presence of docetaxel correlated with increased cell viability to signify resistance (Figure 1). To further test the validity of WST-1 measurements, cytological experiments were performed to look at cell morphology and viability (Figure 4). The cells were tracked using confocal microscopy and Calcein-AM, a vital dye converted to green fluorescence by intracellular esterase serving as an indicator of metabolically active cells. Figure 4 demonstrates that 16 hours post docetaxel addition, the Mad2, BubRl and, to a lesser extent, Mpsl siRNA transfected cells prematurely exited mitosis into an apparent interphase state coincident with a flattened cell morphology, whereas the majority of the control siRNA transfected cells were arrested in mitosis with a rounded up cell morphology. The flat cell phenotype has been previously described for other microtubule inhibitors and refers to the ability of cells to exit from mitotic arrest into an apparent interphase state without actually completing mitosis (Kung et a!., Proc Natl AcadSci 87:9553, 1990; Lanni and Jacks, MoiWell Biol 18:1055, 1998). At 72 hours post treatment, the most striking observations for Mad2, BubRl or Mpsl siRNA down-regulated cells were the presence of many more cells and their very large size, as compared to the siRNA control transfected cells that were virtually eliminated by docetaxel treatment. In the absence of docetaxel, mitotic checkpoint down-regulated cells looked similar to control cells at the two different time points and the increased number of cells at 72 hours suggested the cells were actively growing in the presence of the siRNA.<br><br>
Mitotic checkpoint-impaired cells can bypass mitotic arrest and prematurely exit mitosis following treatment with microtubule inhibitors such as, colcemid, nocodazole and paclitaxel (Taylor and McKeon, Cell 89:727, 1997; Shin et al, Cancer Cell 4:483, 2003; Masuda et al,Am J Pathol 163:1109, 2003; Sudo et aU Cancer Res 64:2502, 2004). To determine if docetaxel treated HCT116 cells prematurely exit from mitosis following suppression of three specific checkpoint genes Mad2, BubRl or Mpsl, we measured mitotic indices of transfected cells. Accordingly, siRNA transfected Mad2, BubRl or Mpsl cells were harvested without (0) treatment or at 8, 16, 24, 36, 48 and 72 hours after docetaxel treatment, then fixed and incubated with a polyclonal antibody that detects phosphorylation of histone H3 as an indication of mitosis (Mitotic Index Kit, Cellomics). A graphic representation for all time points are shown in Figure 5. The mitotic index peaked between 16 and 24 hours post treatment indicating that in all cases there was mitotic arrest. The mitotic index reached the highest value for cells transfected with the control siRNA, whereas cells transfected with Mad2 and BubRl siRNA showed a significant decrease in mitotic index. The Mpsl siRNA effects were not as severe as Mad2 and BubRl. These results clearly show that mitotic checkpoint impaired cells can bypass mitotic arrest induced by docetaxel and that BubRl siRNA was the most effective followed by Mad2 and Mpsl.<br>
To rule out the possibility that changes in mitotic index were coincident with large differences in knockdown efficiencies, we employed real-time PCR to measure mitotic checkpoint transcript levels. The experiment shown in Figure 6 demonstrates that mRNA levels were diminished to comparable levels in Mad2, BubRl and Mpsl siRNA transfected cells, suggesting that the mitotic index differences were more reflective of gene function.<br>
From the microscopy data, it appeared that the very large cells may harbor abnormal DNA content. In order to further investigate whether these mitotic checkpoint impaired cells were capable of overriding docetaxel-induced cell cycle arrest and re-enter S-phase to generate aneuploid cells, we performed FACS analysis to measure DNA content. The, Mpsl, BubRl, Mad2 and control siRNA transfected cells were grown in the absence or presence of docetaxel, fixed and stained with propidium iodide and analyzed by FACS. The data were collected at several different time points post docetaxel addition (data not shown) and histograms for the 72 hour time point are shown in Figure 7. At 72 hours post docetaxel addition, transfections with Mpsl, BubRl, Mad2 and siRNAs caused an accumulation of cells<br><br>
with 8N, 16N and in the case of BubRl 32N DNA content. By just using the control siRNA. an 8N population of cells was detected in the presence of docetaxel, but the 8N numbers was far greater for the mitotic checkpoint down-regulated cells. At an earlier time point following docetaxel addition (data not shown), the majority of control cells were arrested at 4N, whereas cells transfected with Mpsl, BubRl and Mad2 siRNAs were advancing from 4N to 8N. These results indicate that knockdown of mitotic checkpoint genes allow cells to more rapidly accumulate abnormal DNA levels and cross a greater threshold of ploidy as compared to wild-type cells. Our results demonstrate roles for Mad2, BubRl and Mpsl in regulating mitotic index and cell cycle progression with a previously uncharacterized microtubule inhibitor docetaxel.<br>
EXAMPLE 4: Formation of Docetaxel Resistant Colonies in BubRl Downregulated Cells<br>
The apoptotic function of BubRl appears to be important for the regulation of chromosome fidelity given that underexpression of BubRl in combination with anti-microtubule agents was associated with aneuploidy, and restoration of BubRl associated with an activated checkpoint and apoptosis of aneuploid cells (Shin et ah, Cancer Cell 4:483, 2003). To determine if permanently downreguiating mitotic checkpoint genes enables cells to grow with a greater proliferative capacity, the BubRl and Mpsl stable knockdown lines (Figure 4) were tested for their ability to proliferate and establish colonies in the continuous presence of docetaxel. The Mad2 stable knockdown cell line could not be generated since impairment of Mad2 was lethal to cells (Michel et ah, Proc Natl Acad Sci 101:4459, 2004). The BubRl, Mpsl and control shRNA stable knockdown cells lines were subjected to 5 nM docetaxel for 10 days and after that stained with crystal violet. Figure 8 shows that the BubRl knockdown cell line produced many more large colonies as compared to the control cell line. The Mpsl knockdown cell line was not able to grow out colonies in the presence of 5 nM docetaxel. When the plates were incubated for as long as 17 days, colonies were observed on the control plates but were still fewer in number as compared to the BubRl knockdown line. These results link low levels of BubRl, but not Mpsl, with advanced cell growth in the presence of the chemotherapeutic drug docetaxel.<br><br>
EXAMPLE 5: Cross Validation of siRNA that Confers Increased Docetaxel Sensitivity<br>
In addition to identifying siRNAs that protect cells from docetaxel-induced killing, the screen identified siRNAs that sensitize cells to killing. In Table III, Pim-1, p21wat1/Cipl, GBP-1, RXRA, Heel, SPF45, Rafl represent a group of siRNAs that show pronounced killing at higher docetaxel concentrations (&gt;6 nM) with just a small shift in IC50. The siRNA dose-response curves for oncogenic serine/threonine kinase Pim-1 and cell cycle inhibitor p21wafi/cipi were yery sjmjjar (figure 9), which may reflect overlapping or cooperative roles within a signaling pathway; and previous reports have shown that p2]wan/Cipl is a substrate for Pim-1 (Wang et al, Biochim Biophys Acta 1593:45, 2002). Of further interest were data on Pim-1 overexpression in prostate epithelial cells, which lead to mitotic checkpoint interference and genetic instability (Roh et al, Cancer Res 63:8079, 2003). These results are in alignment with Mad2, BubRl and Mpsl underexpression data showing a comparable phenotype with opposing expression levels (see Figure 1). Another siRNA that conferred sensitivity to docetaxel was Heel.<br>
In Table III, TACC3, RELB, Aurora-A, PRKCD (PKC), HSPA1 A, and BRAF35 represent siRNAs that increase sensitivity at lower docetaxel concentrations (1-6 nM) with a characteristic shift in IC50. Aurora-A is a serine/threonine kinase that functions at the onset of mitosis and has been implicated in centrosome maturation and spindle assembly (reviewed in Meraldi et al, Curr Opin Genet Dev 14:29, 2004). Of great interest were published data showing that Aurora-A over expression in HeLa cells protected cells from paclitaxel, in alignment with our results where under expression of Aurora-A enhanced docetaxel killing (Anand et al, Cancer Cell 3:51, 2003). Admittedly this was a small decrease in IC50 for Aurora-A representing a 25% decrease in survival; however, the results were precisely duplicated by using retroviral delivery of shRNA generating a stable knock-down of Aurora-A in HCT116 cells (compare Figures 9 to Figures 10 and 11).<br>
EXAMPLE 6: Piml Inhibits Proliferation of HCT116 cells in the Absence of Docetaxel<br>
To further evaluate the positives from the screen, it was important to distinguish between siRNAs that affect cell viability in the absence of docetaxel, and those that act<br><br>
cooperatively with docetaxel to enhance killing. To distinguish between the possibilities we studied effects of siRNA transfection on proliferation using a trypan blue exclusion assay with cell counting to determine the number of viable cells in 6-well plates at 24, 48, 72 hours post-transfection. Our results show that Pim-1 siRNAs inhibit growth in HCT116 cells whereas TACC3 siRNAs have no effect on growth (Figure 12; knockdown quantitated in Figure 13), although both siRNAs enhance docetaxel-induced cell death (Figure 9).<br><br>
To visually compare siRNAs that confer docetaxel sensitivity and resistance, microscopic imaging is used to determine the ratio of live to dead cells where Calcein-AM is used to stain live cells green, and propidium iodide (PI) to stain the nucleus of dead cells red. Post docetaxel (72 hours) addition, siRNA transfected BubRl cells had the greatest number of viable cells, whereas siRNA transfected Pim-1 and TACC3 had the greatest number of dead cells and the control cells were in between (data not shown). Although the BubRl siRNA alone showed increased cell death by 72 hours, the cells ultimately become more resistant to docetaxel. Pim-1 siRNA alone induced cell death by 24 hours whereas the TACC3 siRNA did not cause death. These results demonstrate that BubRl down regulation was associated with increased cell viability and drug resistance, whereas Pim-1 and TACC3 down regulation was associated with increased cell death and drug sensitivity. Both Pim-1 and TACC3 showed more killing in the presence of docetaxel but this assay could not distinguish between an additive effect and cooperative killing.<br><br>
Docetaxel can cause cell death through induction of apoptosis (Kim et al, IntJMol Med 11:799, 2003). To determine whether Pim-1 and BubRl siRNAs could modulate a canonical apoptotic pathway in the absence and presence docetaxel, we studied activated caspase-3 levels in HCT116 cells (using a bioplex assay) (Figure 14). After 48 hours of 40 nM<br><br>
docetaxel, caspase-3 activity was induced in control cells and induction was further enhanced by down regulating Pim-1 and diminished by down regulating BubRl (middle and lower panels). In the absence of docetaxel, down regulation of Pim-1 caused a slight increase in caspase-3 activity at 24 hours, but returned to control level by 48 and 72 hours (upper panel), indicating that other apoptotic machinery were activated to maintain the enhanced killing observed at .72 hours by WST-1 and LIVE/DEAD assays. Down regulation of BubRl in the absence of docetaxel caused a very slight increase in caspase-3 activity, which is consistent with the observation that BubRl knockdown initially caused cell death at 72 hours in the LIVE/DEAD assay. These results further implicate Pim-1 and BubRl as effectors of docetaxel killing through the modulation of caspase-3 activity.<br><br>
The activities attributed to Pim-1 strongly suggest that it plays a central role in blocking signaling events that lead to cell death while promoting those that foster cell survival. In an attempt to define the molecular basis for enhanced cell death associated with a Pim-1 knockdown, we examined the activation state of a key cellular pathway involved in cell survival, AKT. HCT116 were seeded at 4 X 105 cells/well on 6-weIl plates and the next day transfected with siRNAs at 16 nM using Lipofectamine 2000 (Invitrogen). After 24 hours, cells were untreated or treated with 5 and 40 nM docetaxel. Cells were Iysed 24, 48 and 72 hours after docetaxel addition with ice-cold lysis buffer (50 mM HEPES beffer (PH7.4), 1% NP40, 2.5 mM EDTA, 100 mM sodium fluoride, 10 mM sodium PPj, protease inhibitor cocktail tablet (Roche), 2 mM sodium orthovanadate) and centrifuged at 12,000 g for 10 minutes. Lysates were then quantified for total protein levels using DC protein assay (BioRad). Active caspase-3 levels were determined using an active caspase-3 beadmates kit (Upstate) and the Luminex 100™ system with protocols suggested by the manufacture. Total and phosphorylated AKT levels were determined using a total AKT antibody bead kit and a phosphospecific AKT S473 antibody bead kit (Biosource) and the Luminex 100™ system with protocols suggested by manufacture.<br><br>
The experiment shown in Figure 15 demonstrates that Pirn-1 downregulation decreased the baseline phosphorylation of AKT, as compared to the control and the effect was significantly accentuated with increasing concentrations of docetaxel. By comparison, BubRl downregulation increased the baseline phosphorylation of AKT and the effect was accentuated at the lower but not higher concentration of docetaxel, indicating alternative-signaling pathways may be activated at higher doses.<br>
Applicants performed Western Blot analysis using antibodies against Pim-1 and BubRl to measure protein levels at the 48 hour time point. Equal amounts of proteins were loaded into NuPAGE Novex Bis-Tris gels (Invitrogen). Antibodies against Pim-1 (Santa Cruz biotechnology), BubRl (BD bioscience) and a-tubulin (Sigma) were used as suggested by the manufacture. Secondary antibodies were conjugated to HRP (BioRad) and detected by ECL Western Blot Detection System (Amhersham Biosciences) followed by exposure to Hyperfilm ECL (Amhersham Biosciences). The experiment shown in Figure 16 demonstrated that Pim-1 and BubRl protein levels were diminished as compared to the control, showing that reduced protein levels corresponded with a functional change (Figure 15). These data suggest that downregulating Pim-1 protein levels can block the phosphorylation and activation of AKT in order to compel the cells towards apoptosis. Pim-1 appears to mediate cell survival by activating pro-survival pathways.<br>
The present invention is not limited in scope by the specific embodiments described herein. Indeed, various modifications of the invention in addition to those described herein will become apparent to those skilled in the art from the foregoing description and the accompanying figures. Such modifications are intended to fall within the scope of the appended claims.<br><br><br>
Anand S, Penrhyn-Lowe S, Venkitaraman AR. (2003) AURORA-A amplification overrides the mitotic spindle assembly checkpoint, inducing resistance to Taxol. Cancer Cell. 3:51-62.<br><br>
Katsurnata N. (2003) Docetaxel: an alternative taxane in ovarian cancer. Br J Cancer. 89 Suppl3:S9-S15.<br>
Kim R, Tanabe K, Uchida Y, Emi M, Toge T. (2003) Effect of Bcl-2 antisense oligonucleotide on drug-sensitivity in association with apoptosis in undifferentiated thyroid carcinoma. Int J Mol Med. 11:799-804.<br>
Kolfschoten GM, Hulscher TM, Duyndam MC, Pinedo HM, Boven E. (2002) Variation in the kinetics of caspase-3 activation, Bcl-2 phosphorylation and apoptotic morphology in unselected human ovarian cancer cell lines as a response to docetaxel. Biochem Pharmacol. 63:733-43.<br><br>
Kung AL, Sherwood SW, Schimke RT. (1990) Cell line-specific differences in the control of cell cycle progression in the absence of mitosis. Proc Natl Acad Sci USA. 87:9553-7.<br>
Lanni JS, Jacks T. (1998) Characterization of the p53-dependent postmitotic checkpoint following spindle disruption. Mol Cell Biol. 18:1055-64.<br>
Lee LF, Li G, Templeton DJ, Ting JP. (1998) Paclitaxel (Taxol)-induced gene expression and cell death are both mediated by the activation of c-Jun NH2-terminaI kinase (JNK/SAPK). J Biol Chem. 273:28253-60.<br>
Li Y, Dowbenko D, Lasky LA. (2002) AKT/PKB phosphorylation of p21Cip/WAFl enhances protein stability of p21Cip/WAFl and promotes cell survival. J Biol Chem. 277:11352-61.<br>
Lum L, Yao S, Mozer B, Rovescalli A, Von Kessler D, Nirenberg M, Beachy PA. (2003) Identification of Hedgehog pathway components by RNAi in Drosophila cultured cells. Science. 299:2039-45.<br>
Masuda A, Maeno K, Nakagawa T, Saito H, Takahashi T. (2003) Association between mitotic spindle checkpoint impairment and susceptibility to the induction of apoptosis by anti-microtubule agents in human lung cancers. Am J Pathol. 163:1109-16.<br>
McManus MT, Sharp PA. (2002) Gene silencing in mammals by small interfering RNAs. Nat Rev Genet. 3:737-47.<br>
Meraldi P, Honda R, Nigg EA. (2004) Aurora kinases link chromosome segregation and cell division to cancer susceptibility. Curr Opin Genet Dev. 14:29-36.<br><br>
Michel L, Diaz-Rodriguez E, Narayan G, Hernando E, Murty VV, Benezra R. (2004) Complete loss of the tumor suppressor MAD2 causes premature cyclin B degradation and mitotic failure in human somatic cells. Proc Natl Acad Sci USA. 101:4459-64.<br>
Ojima, I. and Geney, R. 109881 Aventis (2004) Curr Opin Investig Drugs 4:737-40.<br>
Ringel I, Horwitz SB. (1991) Studies with RP 56976 (taxotere): a semisynthetic analogue of taxol. J Natl Cancer Inst. 83:288-91.<br>
Roh M, Gary B, Song C, Said-Al-Naief N, Tousson A, Kraft A, Eltoum IE, Abdulkadir SA. (2003) Overexpression of the oncogenic kinase Pim-1 leads to genomic instability. Cancer Res. 63:8079-84.<br>
Shin HJ, Baek KH, Jeon AH, Park MT, Lee SJ, Kang CM, Lee HS, Yoo SH, Chung DH, Sung YC, McKeon F, Lee CW. (2003) Dual roles of human BubRl, a mitotic checkpoint kinase, in the monitoring of chromosomal instability. Cancer Cell. 4:483-97.<br>
Simmer F, Moorman C, Van Der Linden AM, Kuijk E, Van Den Berghe PV, Kamath R, Fraser AG, Ahringer J, Plasterk RH. (2003) Genome-Wide RNAi of C. elegans Using the Hypersensitive rrf-3 Strain Reveals Novel Gene Functions. PLoS Biol. 1:E12.<br>
Sudo T, Nitta M, Saya H, Ueno NT. (2004) Dependence of paclitaxel sensitivity on a functional spindle assembly checkpoint. Cancer Res. 64:2502-8.<br>
Tanabe K, Kim R, Inoue H, Emi M, Uchida Y, Toge T. (2003) Antisense Bcl-2 and HER-2 oligonucleotide treatment of breast cancer cells enhances their sensitivity to anticancer drugs. IntJ Oncol. 22:875-81.<br>
Taylor SS, McKeon F. (1997) Kinetochore localization of murine Bubl is required for normal mitotic timing and checkpoint response to spindle damage.Cell. 89:727-35.<br><br>
Wang TH, Wang HS, Soong YK (2000) Paclitaxel-induced cell death: where the cell cycle and apoptosis come together. Cancer. 88:2619-28.<br>
Wang Z, Bhattacharya N, Mixter PF, Wei W, Sedivy J, Magnuson NS. (2002) Phosphorylation of the cell cycle inhibitor p21Cipl/WAFl by Pim-1 kinase. Biochim Biophys Acta. 1593:45-55.<br>
Zhou J, Yao J, Joshi HC. (2002) Attachment and tension in the spindle assembly checkpoint. J Ceil Sci. 115:3547-55.<br><br><br>
CLAIMS<br>
What is claimed is:<br>
1.	A method for predicting or monitoring a cancer patient's response to a molecule of the<br>
taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	obtaining a control sample;<br>
c)	measuring the level of one or more genetic markers; and<br>
d)	comparing the measured levels of said one or more genetic markers in the test sample and the control sample;<br>
wherein a decrease in the level of said one or more genetic markers measured in the test sample as compared to the control sample indicates an increased resistance to a molecule of the taxoid family.<br>
2.	The method of Claim 1, wherein said one or more genetic markers is selected from the<br>
group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number:<br>
NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: NM_133483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast)<br>
(BUB1), mRNA (GenBank accession number: NM_004336); hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM__012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase)<br>
II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number:<br>
NM_001221);<br><br>
CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); and GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086).<br>
3.	The method of Claim 1 wherein said molecule of the taxoid family is selected from the group consisting of paclitaxel, docetaxel XRP9881 and XRP6258.<br>
4.	The method of Claim 1 wherein said level of one or more genetic markers is measured by mRNA, DNA or protein.<br>
5.	The method of Claim 4 wherein said mRNA is measured by using a technique selected from the group consisting of in situ hybridization, reverse-transcriptase polymerase chain reaction, nucleic acid hybridization, electrophoresis, Northern blotting and mass spectrometry.<br>
6.	The method of Claim 4 wherein said DNA is measured by using a technique selected from the group consisting of quantitative polymerase chain reaction, genomic DNA-chips, in situ hybridization, electrophoresis, Southern blotting and mass spectrometry.<br>
7.	The method of Claim 4 wherein said protein is measured by using a technique selected from the group consisting of immunoassay, Western blots, ELISA, and mass spectrometry.<br>
8.	A method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br><br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	obtaining a control sample;<br>
c)	measuring the level of one or more genetic markers; and<br>
d)	comparing the measured levels of said one or more genetic markers in the test sample and the control sample;<br>
wherein a decrease in the level of said one or more genetic markers measured in the test sample as compared to the control sample indicates an increased sensitivity to a molecule of the taxoid family.<br><br>
9.   The method of Claim 8, wherein said one or more genetic markers is selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl)<br>
(CDKN1 A), transcript variant 1, mRNA (GenBank accession number: NM_000389); Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM_002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NM_032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession<br>
number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3),<br>
mRNA (GenBank accession number: NM_006342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA<br>
(GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA<br>
(GenBank accession number: NM_006254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NM_006339); HSPA1L, Homo sapiens heat shock 70kDa protein IA (HSPA1 A), mRNA (GenBank<br>
accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11),<br>
mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete eds (GenBank<br>
accession number: L36719).<br><br>
10.	The method of Claim 8 wherein said molecule of the taxoid family is selected from the group consisting of paclitaxel, docetaxel XRP9881 and XRP6258.<br>
11.	The method of Claim 8 wherein said level of one or more genetic markers is measured by mRNA, DNA or protein.<br>
12.	The method of Claim 11 wherein said mRNA is measured by using a technique selected from the group consisting of in situ hybridization, reverse-transcriptase polymerase chain reaction, nucleic acid hybridization, electrophoresis, Northern blotting and mass spectrometry.<br>
13.	The method of Claim 11 wherein said DNA is measured by using a technique selected from the group consisting of quantitative polymerase chain reaction, genomic DNA-chips, in situ hybridization, electrophoresis, Southern blotting and mass spectrometry.<br>
14.	The method of Claim 11 wherein said protein is measured by using a technique selected from the group consisting of immunoassay, Western blots, ELISA, and mass spectrometry.<br>
15.	A kit for predicting or monitoring a cancer patient's response to molecule of the taxoid family according to the method of Claim 1 wherein the kit comprises detectable-labeled antibodies, detectable-labeled antibody fragments or detectable-labeled oligonucleotides comprising nucleic acids capable of hybridizing under stringent conditions to said one or more genetic markers selected from the group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number:<br>
NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: NM_133483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast)<br>
(BUB1), mRNA (GenBank accession number: NM_004336);<br><br>
hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase)<br>
II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number:<br>
NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); and GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086).<br>
16. A kit for predicting or monitoring a cancer patient's response to molecule of the taxoid family according to the method of Claim 8 wherein the kit comprises detectable-labeled antibodies, detectable-labeled antibody fragments or detectable-labeled oligonucleotides comprising nucleic acids capable of hybridizing under stringent conditions to said one or more genetic markers selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl)<br>
(CDKN1A), transcript variant 1, mRNA (GenBank accession number: NM_000389); Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM_002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NM_032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession<br>
number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3), mRNA (GenBank accession number: NM_006342);<br><br>
RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA<br>
(GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA<br>
(GenBank accession number: NM_006254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NM_006339); HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1 A), mRNA (GenBank<br>
accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11),<br>
mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete cds (GenBank<br>
accession number: L36719).<br>
17. A kit for predicting or monitoring a cancer patient's response to molecule of the taxoid family according to the method of Claim 1 wherein the kit comprises the means of:<br>
a)	obtaining total RNA from a bodily sample;<br>
b)	reverse transcribing the total RNA to obtain cDNA; and<br>
c)	subjecting the cDNA to a polymerase chain reaction using a set of primers wherein one or both primers are detectably-labeled and both primers are derived from a nucleotide sequence of one or more genetic markers selected from the group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBRI) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number:<br>
NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: NM_133483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast)<br>
(BUB1), mRNA (GenBank accession number: NM_004336);<br><br>
hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase)<br>
II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number:<br>
NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); and GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086).<br>
18. A kit for predicting or monitoring a cancer patient's response to molecule of the taxoid family according to the method of Claim 8 wherein the kit comprises the means of:<br>
a)	obtaining total RNA from a bodily sample;<br>
b)	reverse transcribing the total RNA to obtain cDNA; and<br>
c)	subjecting the cDNA to a polymerase chain reaction using a set of primers wherein one or both primers are detectably-labeled and both primers are derived from a nucleotide sequence of one or more genetic markers selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl)<br>
(CDKN1A), transcript variant 1, mRNA (GenBank accession number: NM_000389); Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM_002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NM__032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession<br>
number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551);<br><br>
TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3),<br>
mRNA (GenBank accession number: NM_006342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA<br>
(GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA<br>
(GenBank accession number: NM_006254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NMJ306339); HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1A), mRNA (GenBank<br>
accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11),<br>
mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete cds (GenBank<br>
accession number: L36719).<br>
19.	The detectable label of any one of claims 15-18, wherein the detectable label is selected from the group consisting of an enzyme, a radioactive isotope, or a chemical which fluoresces, a chemiluminescent molecule, radiopaque substances, liposomes, and haptenic molecules.<br>
20.	A method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br><br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	measuring the level of one or more genetic markers selected from the group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession<br>
number: NMJ)03318);<br><br>
GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: NVM 33483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog<br>
(yeast) (BUB1), mRNA (GenBank accession number: NMJXJ4336); hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1),<br>
mRNA (GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM<br>
kinase) II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession<br>
number: NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); and GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086)<br>
c)	measuring the level of one or more reference genetic markers selected from the<br>
group consisting of:<br>
GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD), mRNA (GenBank accession number: NM_002046); and<br>
RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession number: BC071941);<br>
d)	comparing the measured levels of said one or more genetic markers and said one or<br>
more reference genetic markers in the test sample;<br>
wherein a decrease in the level of said one or more genetic markers as compared to the level of said one or more reference genetic markers indicates an increased resistance to a molecule of the taxoid family.<br>
21. A method for predicting or monitoring a cancer patient's response to a molecule of the taxoid family, comprising the steps of:<br>
a)	obtaining a test sample from a cancerous area of said patient;<br>
b)	measuring the level of one or more genetic markers selected from the group consisting of:<br>
P21(WafI), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl) (CDKN1 A), transcript variant 1, mRNA (GenBank accession number: NM_000389);<br><br>
Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession<br>
number: NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM_002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA<br>
(GenBank accession number: NM_032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank<br>
accession number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3<br>
(TACC3), mRNA (GenBank accession number: NM J)06342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B,<br>
nuclear factor of kappa light polypeptide gene enhancer in B-cells 3 (avian)<br>
(RELB), mRNA (GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1,<br>
mRNA (GenBank accession number: NMJ306254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA<br>
(GenBank accession number: NM_006339); HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1 A), mRNA<br>
(GenBank accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome)<br>
(STK11), mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete eds<br>
(GenBank accession number: L36719); c) measuring the level of one or more reference genetic markers selected from the group consisting of:<br>
GAPDH, Homo sapiens gIyceraIdehyde-3-phosphate dehydrogenase (GAPD),<br>
mRNA (GenBank accession number: NM_002046); and<br><br>
RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession number: BC071941); d) comparing the measured levels of said one or more genetic markers and said one or more reference genetic markers in the test sample;<br>
wherein a decrease in the level of said one or more genetic markers as compared to the level of said one or more reference genetic markers indicates an increased susceptibility to a molecule of the taxoid family.<br>
22.	The method of Claim 20 or 21 wherein said molecule of the taxoid family is selected from the group consisting of paclitaxel, docetaxel XRP9881 and XRP6258.<br>
23.	The method of Claim 20 or 21 wherein said level of one or more genetic markers is measured by mRNA, DNA or protein.<br>
24.	The method of Claim 23 wherein said mRNA is measured by using a technique selected from the group consisting of in situ hybridization, reverse-transcriptase polymerase chain reaction, nucleic acid hybridization, electrophoresis, Northern blotting and mass spectrometry.<br>
25.	The method of Claim 23 wherein said DNA is measured by using a technique selected from the group consisting of quantitative polymerase chain reaction, genomic DNA-chips, in situ hybridization, electrophoresis, Southern blotting and mass spectrometry.<br>
26.	The method of Claim 23 wherein said protein is measured by using a technique selected from the group consisting of immunoassay, Western blots, ELISA, and mass spectrometry.<br>
27.	A kit for predicting or monitoring a cancer patient's response to a molecule of the taxoid family according to the method of Claim 20 wherein the kit comprises detectable-labeled antibodies, detectable-labeled antibody fragments or detectable-labeled oligonucleotides comprising nucleic acids capable of hybridizing under stringent conditions to said one or more genetic markers and said one or more reference genetic markers selected from the group consisting of:<br><br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession number:<br>
NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number: N1VM 33483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog (yeast)<br>
(BUB1), mRNA (GenBank accession number: NMJ)04336); hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM kinase)<br>
II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession number:<br>
NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1, mRNA (GenBank accession number: NM_002086); GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD), mRNA<br>
(GenBank accession number: NM_002046); and RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank accession<br>
number: BC071941).<br>
28. A kit for predicting or monitoring a cancer patient's response to a molecule of the taxoid family according to the method of Claim 21 wherein the kit comprises detectable-labeled antibodies, detectable-labeled antibody fragments or detectable-labeled oligonucleotides comprising nucleic acids capable of hybridizing under stringent conditions to said one or more genetic markers and said one or more genetic markers selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl) (CDKN1 A), transcript variant 1, mRNA (GenBank accession number: NM_0O0389);<br><br>
Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NMJ302053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NM_032905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank accession<br>
number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3 (TACC3),<br>
mRNA (GenBank accession number: NM_006342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-cells 3 (avian) (RELB), mRNA<br>
(GenBank accession number: NM_006509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1, mRNA<br>
(GenBank accession number: NM_006254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NMJ306339); HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1A), mRNA (GenBank<br>
accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome) (STK11),<br>
mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete eds (GenBank<br>
accession number: L36719); GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD), mRNA<br>
(GenBank accession number: NM_002046); and RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial eds (GenBank accession<br>
number: BC071941).<br><br>
29. A kit for predicting or monitoring a cancer patient's response to a molecule of the taxoid family according to the method of Claim 20 wherein the kit comprises the means of:<br>
a)	obtaining total RNA from a bodily sample;<br>
b)	reverse transcribing the total RNA to obtain cDNA; and<br>
c)	subjecting the cDNA to a polymerase chain reaction using a set of primers wherein one or both primers are detectable-labeled and both primers are derived from a nucleotide sequence of one or more genetic markers selected from the group consisting of:<br>
BubRl, Homo sapiens similar to protein kinase (BUBR1) mRNA, complete cds<br>
(GenBank accession number: AF046079); Mad2, Homo sapiens mRNA for MAD2 protein (GenBank accession number:<br>
AJ000186); Mpsl, Homo sapiens TTK protein kinase (TTK), mRNA (GenBank accession<br>
number: NM_003318); GEFT for Racl/CDC42, Homo sapiens RAC/CDC42 exchange factor (GEFT),<br>
transcript variant 2, mRNA (GenBank accession number. NM_133483); Bubl, Homo sapiens BUB1 budding uninhibited by benzimidazoles 1 homolog<br>
(yeast) (BUB1), mRNA (GenBank accession number: NMJ)04336); hSepharase, Homo sapiens extra spindle poles like 1 (S. cerevisiae) (ESPL1), mRNA<br>
(GenBank accession number: NM_012291); CamKIId, Homo sapiens calcium/calmodulin-dependent protein kinase (CaM<br>
kinase) II delta (CAMK2D), transcript variant 3, mRNA (GenBank accession<br>
number: NM_001221); CDK6, Homo sapiens cyclin-dependent kinase 6 (CDK6), mRNA (GenBank<br>
accession number: NM_001259); and GRB2, Homo sapiens growth factor receptor-bound protein 2 (GRB2), transcript<br>
variant 1. mRNA (GenBank accession number: NM_002086); GAPDH, Homo sapiens gIyceraldehyde-3-phosphate dehydrogenase (GAPD),<br>
mRNA (GenBank accession number: NM_002046); and RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank<br>
accession number: BC071941).<br><br>
30. A kit for predicting or monitoring a cancer patient's response to a molecule of the taxoid family according to the method of claim 19 wherein the kit comprises the means of:<br>
a)	obtaining total RNA from a bodily sample;<br>
b)	reverse transcribing the total RNA to obtain cDNA; and<br>
c)	subjecting the cDNA to a polymerase chain reaction using a set of primers wherein one or both primers are detectable-labeled and both primers are derived from a nucleotide sequence of one or more genetic markers selected from the group consisting of:<br>
P21(Wafl), Homo sapiens cyclin-dependent kinase inhibitor 1A (p21, Cipl)<br>
(CDKN1 A), transcript variant 1, mRNA (GenBank accession number:<br>
NM_000389); Pim-1, Homo sapiens pim-1 oncogene (PIM1), mRNA (GenBank accession number:<br>
NM_002648); GBP-1, Homo sapiens guanylate binding protein 1, interferon-inducible, 67kDa<br>
(GBP1), mRNA (GenBank accession number: NM__002053); RXRA, Homo sapiens retinoid X receptor, alpha (RXRA), mRNA (GenBank<br>
accession number: NM_002957); SPF45, Homo sapiens RNA binding motif protein 17 (RBM17), mRNA (GenBank<br>
accession number: NMJ332905); Heel, Homo sapiens kinetochore associated 2 (KNTC2), mRNA (GenBank<br>
accession number: NM_006101); Rafl, Human mRNA for raf oncogene (GenBank accession number: X03484); Aurora A, Homo sapiens aurora-related kinase 1 (ARK1) mRNA, complete eds<br>
(GenBank accession number: AF008551); TACC3, Homo sapiens transforming, acidic coiled-coil containing protein 3<br>
(TACC3), mRNA (GenBank accession number: NM_006342); RelB, Homo sapiens v-rel reticuloendotheliosis viral oncogene homolog B, nuclear<br>
factor of kappa light polypeptide gene enhancer in B-celis 3 (avian) (RELB),<br>
mRNA (GenBank accession number: NMJ306509); PRKCD, Homo sapiens protein kinase C, delta (PRKCD), transcript variant 1,<br>
mRNA (GenBank accession number: NM_006254); BRAF35, Homo sapiens high-mobility group 20B (HMG20B), mRNA (GenBank<br>
accession number: NM_006339);<br><br>
HSPA1L, Homo sapiens heat shock 70kDa protein 1A (HSPA1A), mRNA<br>
(GenBank accession number: NM_005345); STK11, Homo sapiens serine/threonine kinase 11 (Peutz-Jeghers syndrome)<br>
(STK11), mRNA (GenBank accession number: NM_000455); and MKK3, Homo sapiens MAP kinase kinase 3 (MKK3) mRNA, complete cds<br>
(GenBank accession number: L36719); GAPDH, Homo sapiens glyceraldehyde-3-phosphate dehydrogenase (GAPD),<br>
mRNA (GenBank accession number: NM_002046); and RPS9, Homo sapiens cDNA clone IMAGE:6647283, partial cds (GenBank<br>
accession number: BC071941).<br>
31. The detectable label of any one of claims 24 - 27, wherein the detectable label comprises an enzyme, a radioactive isotope, or a chemical which fluoresces, a chemiluminescent molecule, radiopaque substances, liposomes, and haptenic molecules.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICAgQU1FTkRFRCBDTEFJTVMgICAxOS0wOS0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007   AMENDED CLAIMS   19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAgMTktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007   AMENDED PAGES OF SPECIFICATION   19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0lFVkVEICAgIDE5LTA5LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007   EXAMINATION REPORT REPLY RECIEVED    19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAgIDE5LTA5LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007   OTHER PATENT DOCUMENT    19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIzLTA5LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007   POWER OF ATTORNEY  23-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMDQtMTEtMjAxMC5wZGY=" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  CORRESPONDENCE OTHERS 04-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMjMtMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  CORRESPONDENCE OTHERS 23-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBGT1JNLTEgICAgMTktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  FORM-1    19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3ICBmb3JtLTEgIDIxLTEwLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">2444-chenp-2007  form-1  21-10-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBGT1JNLTEzICAyMS0xMC0yMDEwLnBkZg==" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  FORM-13  21-10-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBGT1JNLTIgIDIxLTEwLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  FORM-2  21-10-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBGT1JNLTMgICAgMTktMDktMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  FORM-3    19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1DSEVOUC0yMDA3ICBQT1dFUiBPRiBBVFRPUk5FWSAgIDE5LTA5LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">2444-CHENP-2007  POWER OF ATTORNEY   19-09-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ0NC1jaGVucC0yMDA3LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">2444-chenp-2007-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250480-a-process-for-making-solid-dosage-form-comprising-doxycycline-calcium-complex.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250482-equipment-and-method-for-measurement-and-control-of-load.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250481</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2444/CHENP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>05-Jan-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Jun-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AVENTIS PHARMACEUTICALS ,INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>55 CORPORATE DRIVE, BRIDGEWATER, NEW JERSEY 08807</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>AUGUST, PAUL,</td>
											<td>13 HOLLOW TREE ROAF , BOXFORD , MASSACHESTTS 01921, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GRUNENEBERG, DORRE</td>
											<td>58 OAK CLIFF ROAD, NEWTONVILLE ,MA 02460,USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>HUIANG, XI,</td>
											<td>47 LENNON ROAD, ARLINGTON , MASSACHUSETTS 02474,USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NATESAN, SRIDHARAN</td>
											<td>118 HERITAGE AVENUE ,ASHLAND ,MA 01721,US</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US05/43578</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-01</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/634,298</td>
									<td>2004-12-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250481-method-for-measuring-resistance-or-sensitivity-to-docetaxel by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:24:55 GMT -->
</html>
