<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250577-method-of-identifying-transmembrane-protein-interacting-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:21:42 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250577:&quot;METHOD OF IDENTIFYING TRANSMEMBRANE PROTEIN-INTERACTING COMPOUNDS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD OF IDENTIFYING TRANSMEMBRANE PROTEIN-INTERACTING COMPOUNDS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for screening compounds for their ability to interact with transmembrane is provided. Also provided is a method for determining whether proteins such as transmembrane proteins are able to oligomerise.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>METHOD OF IDENTIFYING TRANSMEMBRANE PROTEIN-INTERACTING<br>
COMPOUNDS<br>
FIELD OF THE INVENTION<br>
This invention relates to methods for screening compounds for their ability to interact with transmembrane proteins. The invention further relates to methods for screening transmembrane proteins for their ability to dimerise or oligomerise into groups of two or more proteins.<br>
BACKGROUND OF THE INVENTION<br>
In the description which follows, references are made to certain literature citations which are listed at the end of the specification and all of which are incorporated herein by reference.<br>
Transmembrane proteins have been classified in several major classes, including G protein coupled receptors, transporters, tyrosine kinase receptors, cytokine receptors and LDL receptors. G protein coupled receptors (GPCRs) can be grouped on the basis of structure and sequence homology into several families. Family 1 (also referred to as family A or the rhodopsin-like family) is by far the largest subgroup and contains receptors for small molecules such as the catecholamines, dopamine and noradrenaline, peptides such as the opioids, somatostatin and vasopressin, glycoprotein hormones such as thyrotropin stimulating hormone and the entire class of odorant molecules (George et al, 2002). Family 2 or family B contains the receptors such as for glucagon, parathyroid hormone and secretin. These GPCRs are characterised by a long amino terminus that contains several cysteines, which may form disulphide bridges. Family 3 or family C contains receptors such as the metabotropic glutamate, the Ca2+-sensing and the gamma-amino butyric acid (GABA)B receptors. These receptors are also characterised by a complex amino terminus. Although all GPCRs share the seven membrane-spanning helices, the various GPCR families show no sequence homology to one another.<br>
GPCRs are the largest known group of cell-surface mediators of signal transduction and are present in every cell in the body. GPCR action regulates<br>
the entire spectrum of physiological functions, such as those involving the brain, heart, kidney, lung, immune and endocrine systems. Extensive efforts during the past decade has identified a large number of novel GPCRs, including multiple receptor subtypes for previously known ligands, and numerous receptors for which the endogenous ligands are as yet unidentified, termed 'orphan' receptors or oGPCRs (Lee et al., 2001; Lee et al., 2002; Bailey et al., 2001).<br>
GPCRs have been the successful targets of numerous drugs for diverse disorders in clinical use today, with an estimated 50% of the current drug market targeting these molecules. Among the known GPCRs, -335 receptors are potential drug development targets, of which 195 have known ligands, and the remaining 140 being oGPCRs, awaiting identification of their ligands. Although various methodological advances have accelerated the pace of novel receptor discovery, the pace of ligand and drug discovery lags far behind. Conventional, small-scale pharmacological screening assay methods were initially used to discover the ligands and drugs for many of the GPCRs, but newer assay procedures are continually being sought.<br>
Since GPCRs form over 80% of cell surface receptors, they represent a substantial resource and constitute a highly relevant group of protein targets for novel drug discovery. Drugs interacting with GPCRs have the potential to be highly selective, as the interactions will be confined to the cell surface and to tissues bearing the receptors exclusively. The convergence of the discovery of GPCRs with the realisation that they are important drug targets, has led to intense pharmaceutical interest in devising better ways to detect and screen for compounds interacting with GPCRs. Therefore, creating improved assay methods is an urgent requirement towards the goal of more rapid drug screening and discovery. There is a need to optimise the ability to detect an interaction between test compounds and the receptors, which is the fundamental initial step in the process of drug development.<br>
Improved ligand-identification strategies to accelerate the characterisation of all GPCRs will define their physiological functions and realise their potential in discovering novel drugs. Even with the identified GPCRs, there is a paucity of highly selective subtype specific drugs being discovered and pharmaceutical<br>
houses are experiencing a dearth of promising lead compounds, in spite of the wealth of drug targets defined. The list of new drug product approvals by the top 20 pharmaceutical companies has declined considerably over the period 1999-2001, compared to the preceding three year period (Smith, 2002). Thus there is a real need to have improved, versatile assay systems, where not just endogenous ligands, but novel compounds interacting with receptors can be tested and identified in a quick and efficient manner that is amenable to automation.<br>
As the signal transduction pathway required to activate an oGPCR cannot be predicted, an assay system for interacting compounds which is independent of prior predictions of which effector system (such as adenylyl cyclase, PLC, cGMP, phosphodiesterase activity) is employed by the receptor is required. Assigning ligands to GPCRs and oGPCRs is an important task; however the diversity of both GPCR ligands and effector systems can limit the utility of some existing ligand-identificatjon assays, requiring novel approaches to drug discovery.<br>
Recently, several methods utilising refined assay systems testing tissue extracts, large ligand libraries and specific ligands of interest have successfully discovered the endogenous ligands for a number of these oGPCRs. Such methods have been collectively referred to as "Reverse Pharmacology" (Howard et al., 2001). Various methods have been used to assay induced cell activity in response to an agonist compound, including the Fluorescence Imaging Plate Reader assay (FLIPR, Molecular Devices Corp., Sunnyvale, Calif.) and Barak et al., (1997), and U.S. Patents Nos. 5,891,646 and 6,110,693 which disclose the use of a p-arrestin-green fluorescent fusion protein for imaging arrestin translocation to the cell surface upon stimulation of a GPCR.<br>
The potential disadvantages of such methods are as follows: 1) visualisation is not of the receptor; 2) the protein translocation requires complex computerised analytical technologies; 3) prior identification of agonist is necessary to screen for antagonists; and 4) specific G protein coupling is necessary to generate a signal.<br>
Mechanisms of ligand binding and signal transduction by GPCRs traditionally have been modelled on the assumption that monomeric receptors participate in the process, and a monomeric model for GPCRs has been generally accepted. Since the mid-1990s, however, numerous reports have demonstrated oligomerisation of many GPCRs (reviewed by George et al., 2002), and it is now realised that oligomerisation is an inherent aspect of GPCR structure and biology. Also certain receptor subtypes formed hetero-oligomers, and these receptors have functional characteristics that differ from homogeneous, receptor populations. At present, studies of GPCR oligomerisation do not make a distinction between dimers and larger complexes, and the term dimer is used interchangeably with the terms oligomer and multimer. There are no conclusive data to indicate how large the oligomers of functional GPCRs are. Importantly, generation of new properties through hetero-oligomerisation suggested a mechanism for generating diversity of function among GPCRs. Homooligomerisation of GPCRs is accepted as a universal occurrence and a number of GPCRs are known to assemble as heterooligomeric receptor complexes (George et al., 2002). For example, the GABA-B1 and GABA-B2 receptors are not functional individually and only form a functional receptor when co-expressed (White et al., 1998). The assembly of heterooligomer receptor complexes can result in novel receptor-ligand binding, signalling or intracellular trafficking properties. For example, co-transfection of the mu and delta opioid receptors resulted in the formation of oligomers with functional properties that were distinct from each of the receptors individually (George et al., (2000)). The interaction of mu and delta opioid receptors to form oligomers generated novel pharmacological and G protein coupling properties. When mu and delta opioid receptors were co-expressed, the highly selective agonists (DAMGO, DPDPE, and morphine) had reduced potency and altered rank order, whereas certain endogenous ligands endomorphin-1 and Leu-enkephalin had enhanced affinity, suggesting the formation of a novel ligand binding pocket (George et al., 2000). In contrast to the individually expressed mu and delta receptors, the coexpressed receptors showed pertussis toxin insensitive signal transduction, likely due to interaction with a different subtype of G protein. It would therefore be very<br>
useful, from the point of view of identification of potential drug targets, to have a means of determining whether a particular pair of GPCRs are able to form heterooiigomers.<br>
In many reports, heterooiigomers have been tentatively identified by the ability to co-immunoprecipitate. When two GPCRs are shown to co-immunoprecipitate, however, there are two possible interpretations; either the receptors are directly physically interacting, or both are interacting through contact with a common third protein (or proteins). An alternative approach to detecting receptor oligomers has been the development of energy transfer assays using bioluminescent resonance energy transfer (BRET) or fluorescence resonance energy transfer (FRET). Although these methods detect energy transfer between two receptor molecules labelled by fluorophores at proximities of less than 100 angstroms, it is unclear whether receptor conformational changes can be reliably distinguished from de novo oligomerisation.<br>
Transporters are protein pumps that move molecules, ions and other chemicals in and out of cells and exist in virtually all cells. The transporters can be grouped into families on the basis of structure, sequence homology and the molecules they transport. Separate transporters exist for monoamine neurotransmitters such as dopamine, serotonin, norepinephrine and GABA, for amino acids such as glycine, taurine, proline and glutamate, for vesicular monoamines, acetylcholine and GABA/glycine, for sugars such as glucose and disaccharides, for organic cations and organic anions, for oligopeptides and peptides, for fatty acids, bile acids, nucleosides, for water and for creatine. Pumps that export large molecules such as drugs, toxins and antibiotics from the cell are exemplified by the P-glycoprotein (multidrug resistance protein) family. There are also several related transporters the function of which remains unknown (Masson et al., 1999). These transporters are membrane proteins consisting of a polypeptide generally with 12 transmembrane domains. The glutamate and aspartate transporters belong to a separate family whose members have 6 to 10 TM domains and share no homology to the other transporters (Masson et al., 1999). Both the amino and carboxyl termini are located on the intracellular side of the membrane.<br>
A large number of neurological and psychiatric disorders including depression, Parkinson's disease, schizophrenia, drug addiction, Tourette's syndrome, and attention deficit disorders are considered to involve the monoamine transporters. The dopamine transporter (DAT) is the major target for psychostimulants such as cocaine and methylphenidate. The transporters have been the successful targets of numerous drugs for diverse disorders in clinical use today, particularly antidepressant drugs, including fluoxetine, sertraline and the other related serotonin selective reuptake inhibitors (SSRIs). Although methodological molecular advances have identified the known transporters, the pace of ligand and drug discovery lags behind. Conventional, pharmacological screening assay methods were used to discover the ligands and drugs for some of the transporters, but newer assay procedures are urgently being sought. Improved ligand-identification strategies to accelerate the characterisation of all the transporters will further define their physiological functions and realise their potential In discovering novel drugs. Even with the identified transporters, there is a paucity of highly selective specific drugs being discovered.<br>
The tyrosine kinase receptor family members are characterised by their structural similarity, with an extracellular ligand binding domain, a single transmembrane domain and an intracellular domain with tyrosine kinase activity for signal transduction. There are many subfamilies of receptor tyrosine kinases, exemplified by the epidermal growth factor (EGF) receptor (also called HER1 or erbB1), which is one of four members of such a subfamily, which also includes HER2, HER3 and HER4. The principal EGF-R ligands are EGF, TGF-a, heparin binding EGF, amphiregulin, betacellulin and epiregulin (Shawver et al., 2002). Activation of the EGF-R causes the receptor to dimerise with either another EGF-R monomer or another member of the HER subfamily. Marked diversity of ligand binding and signalling is generated by the formation of heterodimers among family members (Yarden and Sliwkowski, 2001). The EGF-R is widely expressed in a variety of tissues and mediates important functions such as cell growth and tissue repair. Overexpression of EGF-R occurs in many types of cancer, such as head and<br>
neck, lung, laryngeal, esophageal, gastric, pancreatic, colon, renal, bladder, breast, ovarian, cervical, prostate, thyroid, melanoma and glioma, and correlates with a poor outcome (Nicholson et al., 2001). Therefore there is great interest and need for developing drugs targeting the EGF-R and for methods which assist in identifying such drugs.<br>
Other subfamilies of receptor tyrosine kinases are exemplified by the receptors for vascular endothelial factor (four members) and fibroblast growth factor (four members). These have important roles in angiogenesis and also have significant roles in the uncontrolled proliferation of vessels characterizing carcinogenesis (Hanahan and Folkman. 1996).<br>
The cytokine receptors are proteins spanning the membrane with an extracellular ligarid binding domain and an intracellular domain with intrinsic kinase activity or adapter regions able to interact with intracellular kinases. The receptors are divided into subclasses based on their structural complexity. The 'simple' receptors are those including receptors for growth hormone, erythropoietin and interleukins, and the 'complex' receptors include the tumour necrosis factor receptor family, the 4-helical cytokine receptor family, the insulin/ insulin-like receptor family and granulocyte colony stimulating receptor (Grotzinger, 2002).<br>
The insulin and insulin-like growth factor (IGF) receptor family controls metabolism, reproduction and growth (Nakae et al., 2001). There are nine different insulin-like peptides known and there are three known receptors that interact with them, IR, IGF-1R and IGF-2R, and an orphan member IR-related receptor. Each receptor exists as homodimers on the cell surface or heterodimers. The IR subfamily is also related to the EGF-R family.<br>
IR, produced from a single mRNA, undergoes cleavage and dimerisation and translocation to the plasma membrane. Each monomer component contains a single transmembrane domain; the complete receptor comprised two a and two ß subunits, linked by disuiphide bridges. The ß subunit contains the single TM and the intracellular region. This receptor is a tyrosine kinase that catalyzes the phosphorylation of several Intracellular substrates.<br>
The low density lipoprotein (LDL)-receptor family act as cargo<br>
tranporters, regulating the levels of lipoproteins and proteases (Strickland et al., 2002). There are nine recognised members of the family, all of which share structural similarity, including an extracellular region, a single transmembrane domain region and a cytoplasmic tail. The LDL receptor plays a major role in the clearance of lipoproteins, and genetic defects in the LDL receptor can result in the accumulation of LDL in the bloodstream.<br>
The first characterised motif shown to be able to direct protein nuclear importation was exemplified by the amino acid sequence (PKKKRKV) contained in the SV40 large T antigen protein. The nuclear localisation sequence (NLS) motifs are recognised by ths importin a-p receptor complex, which binds the NLS (Gorlich et al., 1996). These are cytosolic proteins, which recognise NLS containing proteins and transport these proteins to dock at the nuclear pore. The entire complex subsequently docks at the nuclear pore complex (Weis et al., 1998, Schlenstedt et al., 1996), contained at the nuclear envelope. The nuclear envelope is a boundary containing pores that mediate the nuclear transport process (Weis et al., 1998).<br>
There have been very few and rare reports of GPCRs localising in the nucleus. One such example is the GPCR angiotensin type 1 (ATO receptor, which contains an endogenous NLS which serves to direct the GPCR into the nucleus (Lu et al., 1998), providing evidence that this NLS sequence was involved in the nuclear targeting of the AT1 receptor. These authors and Chen et al., (2000) reported that AT1 receptors increased in the nucleus in response to agonist. The nuclear localisation of the parathyroid hormone receptor has been reported (Watson et al, 2000). However very few of the superfamily of GPCRs contain an endogenous NLS mediating translocation of the receptor to the nucleus.<br>
There therefore remains a need for new, more convenient methods for identifying compounds which, interact with transmembrane proteins such as GPCRs. transporters, etc. There also remains a need for inproved, less ambiguous methods for detecting oligomerisation of transmembrane proteins.<br>
SUMMARY OF THE INVENTION<br>
The inventors have shown that the incorporation of a nuclear localisation sequence (NLS) into a transmembrane protein (not containing an endogenous functional NLS) routes the protein from the cell surface into the nucleus of a cell in a time-dependent and ligand-independent manner. In order to visualise this trafficking of transmembrane proteins from the cell surface, they carry a detectable moiety for visualisation by a variety of means. It has been demonstrated that membrane proteins from diverse protein families containing a synthetically incorporated NLS are redistributed under basal conditions from the cell surface to and towards the nucleus.<br>
This process can be exploited to identify compounds which interact with transmembrane proteins by determining whether candidate compounds are able to modulate this ligand-independent transfer of a transmembrane protein away from the cell membrane.<br>
It is also now possible, using methods based on this process, to determine whether protein molecules are able to oligomerise.<br>
In accordance with one embodiment, the invention provides a method for screening a candidate compound for its ability to interact with at least one transmembrane protein comprising:<br>
transfecting a cell with at least one nucleotide sequence encoding a protein comprising a transmembrane protein containing at least one nuclear localisation sequence (NLS) and a detectable moiety and permitting expression of the encoded protein in the cell;<br>
contacting the cell with a candidate compound; and<br>
determining the distribution of the expressed protein in the cell by detecting the distribution of the detectable moiety in the cell;<br>
wherein detection of an altered distribution of the detectable moiety in the cell relative to the distribution of the detectable moiety in a control cell not contacted with the candidate compound indicates that the compound interacts with the transmembrane protein.<br>
In accordance with a further embodiment of this method, the cell is contacted with a compound known to interact with the at least one transmembrane protein prior to contacting the cell with the candidate compound and<br>
wherein detection of an altered distribution of the detectable moiety in the cell relative to the distribution of the detectable moiety in a control cell contacted with the compound known to interact with the transmembrane protein but not contacted with the candidate compound indicates that the candidate compound interacts with the transmembrane protein.<br>
In accordance with a further embodiment, the invention provides a method for screening a candidate compound for its ability to interact with at least one transmembrane protein comprising:<br>
transfecting a cell with at least one nucleotide sequence encoding an NLS-containing transmembrane protein and permitting expression of the encoded protein in the cell;<br>
contacting the cell with a candidate compound; and<br>
determining the level of NLS-containing transmembrane protein remaining at the cell membrane by isolating the cell membrane fraction of the cell, contacting the fraction with a labelled ligand of the transmembrane protein and determining the level of binding of the ligand to the fraction;<br>
wherein detection of an altered level of the transmembrane protein at the cell membrane relative to the level at the cell membrane in a control cell not contacted with the candidate compound indicates that the compound interacts with the transmembrane protein.<br>
In accordance with a further embodiment, the invention provides an isolated cell transfected with at least one nucleotide sequence encoding a protein comprising a transmembrane protein containing at least one NLS and a detectable moiety.<br>
In accordance with a further embodiment, the invention provides a method for determining whether a first protein and a second protein are able to oligomerise comprising:<br>
transfecting a cell with a first nucleotide sequence encoding a first protein containing an NLS and a second nucleotide sequence encoding a second protein comprising a detectable moiety and permitting expression of the encoded first and second proteins in the cell; and<br>
determining the distribution of the detectable moiety in the cell;<br>
wherein detection of the detectable moiety In or adjacent to the nucleus of the cell or detection of a reduced level of the detectable moiety at the cell surface, relative to a control cell, indicates that the first and second proteins interact.<br>
SUMMARY OF THE DRAWINGS<br>
Certain embodiments of the invention are described, reference being made to the accompanying drawings, wherein:<br>
Figure 1 shows in diagrammatic form the structure of a typical GPCR, the dopamine D1 receptor, modified to contain an NLS.<br>
Figure 2 shows fluorescence (Relative fluorescence units) at surface of HEK cells transfected with dopamine D1 receptor-NLS and treated with various concentrations of butaclamol.<br>
Figure 3 shows cell surface fluorescence of HEK cells transfected with HA-dopamine D1 receptor-NLS and treated with various concentrations of SCH23390 alone or with SKF81297 (100 nM).<br>
Figure 4 shows cell surface fluorescence of HEK cells transfected with HA-dopamine D1 receptor-NLS and treated with 0.5 nM SCH23390 alone or together with various concentrations of SKF81297.<br>
Figure 5 shows the amount of 3H-SCH 23390 bound to the cell membrane fraction of HEK cells transfected with dopamine D1 receptor-NLS and treated with butaclamol (▲) or control untreated cells (■).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention provides, in one embodiment, a new and convenient method for screening candidate compounds for their ability to interact with a transmembrane protein.<br>
As used herein, when a candidate compound and a transmembrane protein "interact", this means that the compound is a ligand of the transmembrane protein and binds to the protein or is able to modulate the trafficking of the transmembrane protein away from the cell membrane described herein.<br>
Identification of compounds which interact with a transmembrane protein is the first important step in the process of identifying lead compounds for drug development.<br>
Working initially with GPCRs, the inventors have found that when a nucleated eukaryotic cell is transfected with a nucleotide sequence which encodes a GPCR containing a synthetically incorporated NLS, or a naturally occurring NLS, and the cell is permitted to express the nucleotide sequence, the expressed GPCR travels first to the cell membrane and then is transferred to the cell nucleus. This process is independent of ligand activation and takes from about 6 to about 72 hours, depending on the transmembrane protein involved, with an average of 24 to 48 hours. This is in contrast to the situation when a GPCR not containing an NLS is expressed in a cell, when the expressed GPCR remains predominantly at the cell surface, with small amounts occurring in the cytoplasm but no detectable amounts in the nucleus.<br>
The inventors have also found that the transfer or trafficking of the expressed NLS-containing GPCR from the cell membrane to or towards the nucleus can be modulated by treating the transfected cell with a compound which interacts with the GPCR. Screening of candidate compounds for their ability to interact with a GPCR can therefore be carried out by detecting this modulation of transfer of the expressed GPCR from the cell membrane to the nucleus.<br>
The inventors have further found that these observations are widely applicable to transmembrane proteins generally, and are not limited to GPCRs.<br>
Transmembrane protein" as used herein means a single chain protein found in the cell membrane and having at least one domain that spans the cell membrane.<br>
The inventors have shown that a wide variety of transmembrane proteins from several families of the GPCR group, from the transporter group, from the<br>
cytokine receptor group, from the tyrosine kinase group and from the low density lipoprotein receptor group, if expressed in a nucleated cell so that they contain an NLS group, all show initial accumulation of the expressed protein at the cell membrane, followed by iigand activation-independent transfer of the expressed protein away from the cell membrane and into the cell nucleus.<br>
The wide applicability of the methods of the invention is indicated by the immense variety of transmembrane protein structures represented by the exemplified transmembrane proteins used in the method; NLS insertion into a transmembrane protein resulting in translocation of the protein off the cell surface and to the nucleus has been shown to be effective with membrane proteins having one transmembrane (TM) domain, seven TM domains and twelve TM domains and sharing little or no sequence homology.<br>
It has been found that the method of the invention is widely applicable to identifying compounds which interact with transmembrane proteins.<br>
Compounds which interact with transmembrane proteins have been found to modulate their transfer from cell membrane to nucleus in different ways, including inhibition of the transfer, acceleration of the transfer and interference with the modulation produced by other compounds. Any interacting compound is of interest as a potential drug candidate.<br>
Modulation of the transfer of expressed transmembrane protein is determined by comparing the distribution of the transmembrane protein within the cell in control cells and cells treated with a candidate compound.<br>
In one embodiment, the method provides a convenient tool for screening candidate compounds for their ability to interact with a GPCR and modulate its trafficking. Compounds that specifically interact with the GPCR may inhibit or prevent transfer of the GPCR from the cell surface to the nucleus and may be antagonists to the GPCR, whereas other compounds can accelerate the transfer of GPCR to the nucleus, relative to a control cell and may be agonists to the GPCR.<br>
To allow determination of the distribution of the expressed transmembrane protein within the cell, with and without exposure to a test compound, the expressed transmembrane protein must carry a detectable moiety, which can be<br>
detected in the cell. The detectable moiety may be any moiety which will remain associated with transmembrane protein throughout its expression and trafficking within the cell and can be directly or indirectly detected to determine its distribution within the cell and to determine an altered distribution resulting from exposure to a candidate compound.<br>
In a first embodiment, the cell is transfected with a nucleotide sequence encoding a fusion protein comprising a transmembrane protein containing at least one NLS linked to a detectable moiety comprising a detectable peptide or polypeptide. As used herein, a peptide means a sequence of two to 20 amino acid residues, preferably a sequence of about 5 to about 15 amino acid residues, and a polypeptide means a sequence of more than 20 amino acid residues, including full proteins of any length. The detectable peptide or polypeptide may be directly detectable or may be reactable to give a detectable signal. The detectable peptide may be, for example, an antigenic peptide or epitope which is expressed, for example, at the amino terminus of the transmembrane protein. The distribution of the transmembrane protein within the cell is detected by detection of the epitope using a detectable antibody specific for the epitope. A number of suitable epitope antibody systems are available commercially. Examples are the HA (Roche Diagnostics), FLAG (Sigma Chemical Co.), c-myc (Santa Cruz), Histidine hexamer (BD Biosciences Clontech), GST (ABR Affinity BioReagents), V5 (Abeam) and Xpress (Invitrogen) epitope/antibody systems.<br>
Nucleotide sequences encoding these epitopes can be purchased, as well as antibodies specific for the epitopes. These antibodies may carry a detectable label (e.g. fluorescein isothiocyanate (FITC)) or may themselves be detected by use of a second antibody carrying a detectable label, as will be understood by those of skill in the art. This embodiment of the invention is particularly adaptable to an automated or semi-automated method, for example by examining antibody-treated plates of cells in an automated plate reader, allowing for high through put screening.<br>
The detectable polypeptide may be an optically detectable polypeptide such as green fluorescent protein (GFP), red fluorescent protein (RFP), yellow fluorescent protein (YFP) and or cyan fluorescent protein (CFP), or any of the<br>
modified variants of these proteins, which are commercially available. The detectable polypeptide may also be an enzyme such as luciferase or p-galactosidase, which can be reacted to give a detectable end point such as light emission or colour change. Nucleotide sequences encoding such polypeptides are readily available, for example from Clontech, and are linked to the nucleotide sequence encoding the NLS-containing transmembrane protein, preferably at the C-terminal end of that protein.<br>
In a further embodiment, the detectable moiety is an antigenic peptide comprising a portion of the amino acid sequence of the transmembrane protein itself, preferably a portion of an extracellular region of the protein. As described above, the distribution of the transmembrane protein within the cell is determined using a detectable antibody specific for the epitope. Suitable antibodies are available commercially, e.g. anti-D1 antibody directed to amino terminal amino acids 9-21 of the human D1 dopamine receptor, or may be prepared by conventional methods.<br>
In a further embodiment, applicable to transmembrane proteins with known ligands, the cell is transfected with a nucleotide sequence encoding a transmembrane protein containing at least one NLS. The cells are contacted with a candidate compound and incubated as described above. The cells are then harvested and the cell membrane fraction is isolated and contacted with a detectably labelled ligand of the transmembrane protein, for example a radio-labelled ligand. Determination of the amount of labelled ligand bound to the membrane fraction of treated cells, relative to the amount bound to the membrane fraction of control cells not contacted with the candidate compound, can be used to quantitate the transmembrane protein remaining at the cell surface and indicate interaction of the candidate compound with the transmembrane protein.<br>
Transmembrane protein-encoding nucleotide sequences can be obtained from public databases such as Genbank<br>
(http://www.ncbi.nlm.nih.gov:80/entrez) or from commercial databases. Suitable constructs may be synthesised by conventional methods, as described in the examples herein, or obtained commercially.<br>
An NLS-containing transmembrane protein" includes a transmembrane protein which contains an NLS in its wild type sequence and a transmembrane protein whose amino acid sequence has been modified to contain an NLS.<br>
Conventional NLSs are short peptide sequences that facilitate nuclear localisation of the proteins containing them (see for example, Table 1 which lists NLSs and Jans et al., 2000). There are three major classes of NLSs; two of these classes consist of basic amino acid residues, the monopartite NLSs, exemplified by the SV40 large tumor antigen, PKKKRKV, consisting of a single stretch of basic amino acids, and the bipartite NLSs which contain two stretches of basic amino acids separated by 10 to 22 (sometimes up to hundreds) amino acids.   Other types of NLSs are exemplified by those of the yeast protein Mata2 NLS where charged/polar residues are contained within the stretch of non-polar residues, or the protooncogene c-myc NLS, where proline and aspartic acid residues flanking the basic residues are required (PAAKRVKLD) for nuclear targeting.    All classes of NLS are recognized specifically by the a-p-importins.<br>
Any NLS may be employed in the methods of the invention. Nucleotide sequences encoding a selected NLS may be derived from the amino acid sequence of the NLS and are synthesised and incorporated into the nucleotide sequence encoding the transmembrane protein by conventional methods, as described herein. Many different locations within any of the intracellular loops or intracellular termini of the transmembrane protein are suitable for insertion of the NLS. Insertion of the NLS within an intracellular domain of the protein is preferred. For example, in a GPCR, the NLS could be placed in any of the intracellular loops or intracellular carboxyl tail. In a 12 TM transporter, the NLS could be placed in the intracellular amino or carboxyl termini or any of the intracellular loops.<br>
When an NLS is inserted into a transmembrane protein for use in the methods of the invention, the efficacy of the insertion can be screened by confirming that the NLS-containing transmembrane protein is substantially translocated from the cell membrane to the cell nucleus within 24 to 48 hours<br>
and that ligands of the transmembrane protein interfere with the translocation.<br>
Nucleotide sequences encoding NLS-containing transmembrane proteins are linked to sequences encoding detectable peptides or polypeptides by conventional methods.<br>
A nucleotide sequence encoding a selected NLS-containing transmembrane protein containing or linked to a detectable moiety, is transfected into a nucleated cell by cloning the sequence into a vector system containing a suitable promoter, using conventional techniques as described in the scientific literature, for example in Current Protocols in Molecular Biology, (1987). Suitable vectors include the pEGF-N1 (Clontech) which contains the human cytomegalovirus (CMV) promoter, and the vector pcDNA.<br>
Any cell may be used which is capable of expressing the transfected nucleotide sequences and in which an NLS facilitates transfer of a transmembrane protein away from the cell membrane. Suitable cells include prokaryotic cells, including bacterial cells, and eukaryotic cells. Suitable eukaryotic cells include isolated mammalian cells, yeast cells, plant cells, insect cells, nematode cells and fungal cells. Suitable mammalian cells include human cell lines, rodent cell lines, hamster cell lines, non-human primate cell lines.<br>
In one embodiment, the cell is transfected with a number of nucleotide sequences, each encoding a different NLS-containing transmembrane protein and a different detectable moiety. Interference with trafficking of the transmembrane protein away from the cell membrane by a test compound can be related to interaction of the compound with a particular transmembrane protein by the identity of the detectable moiety whose movement from the cell surface is interrupted.<br>
In a further embodiment, for higher throughput initial screening, the cell is transfected with a greater number of nucleotide sequences, each encoding a different NLS-containing transmembrane protein and a detectable moiety, some of the detectable moieties being common to more<br>
than one transmembrane protein. If initial screening indicates that a candidate compound is interacting with one or more of the transmembrane proteins, the compound is rescreened using a cell expressing fewer transmembrane proteins, or only one, until the specific interacting transmembrane protein is identified.<br>
In cells transfected with more than one transmembrane protein, there may be oligomerisation between pairs of proteins as discussed herein, and this may affect the interpretation of the effect of a candidate compound. Subsequent rescreening of the compound using cells transfected with only one transmembrane protein allows clarification of the interaction of the compound with a particular protein.<br>
Alternatively, for multiply transfected cells, transmembrane proteins may be selected which have been shown not to oligomerise with each other.<br>
Identification of interacting compounds<br>
In one embodiment of the invention, nucleated cells are transfected with a nucleotide sequence encoding a protein comprising a transmembrane protein containing an NLS and a detectable moiety and incubated for a suitable period of time to allow expression of the NLS-transmembrane protein and commencement of its accumulation at the cell membrane. For GPCRs and transporters, for example, a time period of about 6 to 24 hours is suitable. One of skill in the art can readily determine a suitable incubation time for other transmembrane proteins by. observation of the accumulation of the protein at the cell membrane. All of the expressed transmembrane protein need not have reached the cell membrane when the candidate compound is added. Test cells are then contacted with a candidate compound which is to be tested for interaction with the transmembrane protein for a period of time which is sufficient to allow translocation of a substantial portion of the NLS-transmembrane protein, preferably at least 20%, more preferably at least 50%, and still more preferably at least 90%, away from the cell membrane and into or towards the nucleus in a control cell not treated with compound.<br>
Depending on the transmembrane protein, this period of time may be from about 6 hours to about 72 hours; a time period of about 24 to about 48 hours is suitable for most transmembrane proteins examined. One of skill in the art can readily determine a suitable time by observation of control cells.<br>
Test compounds are initially tested generally at a concentration of about 1 to 10 micromolar.<br>
Test and control cells are then examined to determine the distribution of the detectable moiety and thereby the distribution of the NLS-transmembrane protein. The distribution of the detectable moiety may be determined by various methods. For example, when the detectable moiety is an optically detectable protein, the cells may be examined by direct microscopy and the amount of protein in the nucleus compared between test and control cells. In another embodiment, the amounts of detectable protein or peptide remaining in the membrane of control and test cells are compared. In several microscopic fields (5-10), each containing 30-100 cells, the location of the detectable moiety in these cells is determined and counted for each location. The percentage of cells having cell surface, or nuclear labelling and the sum of all the fields is then calculated for the treated and control cells.<br>
In a further example, when the detectable moiety is an antigenic epitope, the cells are contacted with a detectable antibody system containing an antibody specific for the epitope, as described above. For example, a first antibody specific for the epitope may be used, followed by a fluorescently labelled second antibody specific for the first antibody, the fluorescent signal being quantified by fluorometer.<br>
Where control cells show a substantial portion, preferably at least 50%, of the transmembrane protein translocated away from the cell membrane and test cells show retention of the transmembrane protein at the cell membrane, relative to control cells, this indicates interaction of the test compound with the transmembrane protein. In a preferred embodiment, interaction is indicated when the level of protein at the cell membrane is higher in the test cells by at least 10%, preferably by at least 15%, and more preferably by at least 20%.<br>
The proportion of the detectable moiety remaining at the cell membrane on exposure to the interacting compound is related to the concentration and potency of the compound. For example, the use of a known potent GPCR antagonist in micromolar concentration typically resulted in about 50 to 100% of the protein remaining at the cell surface, 0 to 15% in the nucleus and the remainder in the cytoplasm. Lower nanomolar concentrations of the same antagonist resulted in retention of 20 to 40% of the protein at the cell surface, with the rest of the protein in the cytoplasm and nucleus. In the untreated control cells, 0-15% of the protein was detectable at the cell surface, with the remainder in the cytoplasm and nucleus.<br>
In a variant of this method, used where the transmembrane protein has known ligands, an expressed NLS-containing transmembrane protein without a detectable moiety is used and distribution of the protein in the cell after treatment with a test compound is determined by isolation of the cell membrane fraction and determination of its transmembrane protein component using detectably labelled ligand, as discussed above.<br>
In a further embodiment of the invention, a similar method is used to identify compounds which interact with an NLS-transmembrane protein to promote its translocation away from the cell surface and into or towards the nucleus. Cells are transfected and incubated to permit expression of the NLS-transmembrane protein and its accumulation at the cell surface. Preferably, the cells are incubated until at least about 70 to 90% of the expressed transmembrane protein has accumulated at the cell surface. For many transmembrane proteins, a time period of about 12 to about 24 hours from transfection is suitable.<br>
Test cells are then contacted with a candidate compound, and individual test and control cells are immediately observed in real time for up to 4 hours to observe the distribution of the detectable moiety. An increased accumulation of detectable moiety in the nucleus of test cells compared with control cells indicates that the test compound has promoted translocation of the transmembrane protein. In a preferred embodiment, interaction is<br>
indicated when test cells show nuclear accumulation increased by at least 5%, preferably by at least 10%, and more preferably by at least 20%.<br>
A further embodiment of the invention is a method for identifying compounds which, although they do not themselves prevent translocation of an NLS-containing transmembrane protein away from the cell membrane, nevertheless can interfere with the interaction of the transmembrane protein with an interacting compound.<br>
Compounds which have proved negative in the first screening method described above may be tested by this further method for their ability to compete with a known interacting compound.<br>
In this method, cells are transfected as described above and incubated for a suitable period of time to allow expression and accumulation of the transmembrane protein at the cell surface, for example for about 24 to about 48 hours.<br>
Test cells and control cells are then contacted with a compound known to interact with the transmembrane protein, either a known ligand or an interacting compound identified by the method described above, for about 24 to about 48 hours. Test cells are then contacted with a candidate compound and test cells and control cells are observed after 1 hour, at one or more time points, up to 24 hours, to determine distribution of the NLS-transmembrane protein within the cells as described above. In control cells, the known interacting compound causes the transmembrane protein to be retained at the cell membrane. If the candidate compound competes with the interacting compound, test cells show a reduction of transmembrane protein at the cell surface and increased translocation of the protein away from the cell surface. In a preferred embodiment, interaction is indicated when test cells show a reduction of at least 10%, preferably 15%, and more preferably 20%.<br>
In a further embodiment, a cell which endogenously expresses an NLS-containing transmembrane protein may be employed, in conjunction with a first compound which has been demonstrated to interact with the protein and inhibit its transfer from the cell membrane, thus retaining the protein at the cell membrane. When such a system is contacted with a candidate<br>
compound, if that compound interacts with the transmembrane protein and competes with the first compound, an increased transfer of the protein away from the cell membrane is observed.<br>
Identifying transmembrane protein interactions with other proteins<br>
A number of transmembrane proteins, including GPCRs, transporters, tyrosine kinase receptors, the cytokine receptors for insulin, insulin-like growth factors, the epidermal growth factor and vascular endothelial growth factor, are capable of both homo- and heterooligomerisation (see, for example, review of GPCRs in George et al., 2002). As used herein, "oligomerisation" of a protein means association of two or more molecules of the protein.<br>
For hypothetical receptors A and B, the cell surface may contain dimers AA, BB and AB and it is believed that these may represent three different functional complexes and therefore three different drug targets. It is therefore important to identify which transmembrane proteins can interact with each other or with other proteins by oligomerisation.<br>
In further embodiments, the invention provides methods for determining whether two transmembrane proteins are capable of oligomerisation or whether a transmembrane protein and a non-transmembrane protein are capable of oligomerisation.<br>
In one embodiment, a nucleated cell is co-transfected with a first nucleotide sequence encoding a first transmembrane protein containing an NLS and a second nucleotide sequence encoding a second transmembrane protein lacking an NLS but carrying or linked to a detectable moiety. Creation of these nucleotide sequences is as described above. After a suitable time interval to allow for expression of the encoded proteins, accumulation at the cell membrane and subsequent translocation of the NLS-containing protein away from the cell membrane to or towards the nucleus, the distribution of the detectable moiety in the cell is determined, for example by determining an increase of detectable moiety in the nucleus or by a decrease of detectable moiety at the cell surface.<br>
It has been found that when cells are doubly transfected, and the first and second transmembrane protein are the same, except that one transmembrane protein contains an inserted NLS and the other does not, there is a slowing of the transfer of the NLS-containing transmembrane protein to the cell nucleus compared with transfer in a cell transfected only with the NLS-containing protein. The process of protein translocation to the nucleus now may take about 24 to 48 hours. In this method, therefore, the cells are incubated for about 24 to 48 hours before examination of the distribution of protein in the cell.<br>
Translocation of the detectable moiety from the cell surface to or towards the nucleus indicates that the first transmembrane protein has carried the second transmembrane protein away from the cell surface, indicating oligomerisation of the first and second proteins. Retention of the detectable moiety at the cell surface indicates a lack of interaction between the proteins.<br>
When the first and second transmembrane proteins are the same protein, the method allows the identification of the ability of the protein to homodimerise. When the first and second transmembrane proteins are different, the method allows the identification of the ability of two different proteins to heterodimerise and permits the determination of the specificity of interaction between two transmembrane proteins.<br>
The method may be carried out either in the absence of ligand activation or in the presence of a ligand of either protein.<br>
Using this method, oligomerisation has been demonstrated both within and between different classes of GPCRs and within and between other classes of transmembrane proteins.<br>
1 In addition, interactions have been detected between GPCRs and non-GPCR transmembrane proteins, for example between the D5 dopamine receptor and the GABA-A receptor, and between transmembrane proteins and non-transmembrane proteins.<br>
The invention therefore generally provides a method for detecting oligomerisation between two proteins by the method described above, where<br>
a cell is co-transfected with one of the proteins containing an NLS and the other protein carrying a detectable signal.<br>
Co-transfection of a cell with a first transmembrane protein containing an NLS and a second detectably labelled protein, such as a transmembrane protein from a different group, which has been shown by the method of the invention to oligomerise with the first protein, provides a cell which can be used to screen candidate compounds for interaction with either the first or second protein. A compound which interacts with either protein will influence oligomerisation or translocation of the oligomerised proteins away from the cell membrane. Compounds which interact with one member of the protein pair or with the oligomer to cause retention of the detectable protein at the cell surface or to cause accelerated translocation of the detectable protein away from the cell surface may be identified by this method.<br>
In a further embodiment, a cell which endogenously expresses an NLS-containing transmembrane protein is transfected with a nucleotide sequence encoding a second transmembrane protein carrying a detectable moiety but lacking an NLS. Oligomerisation of the two proteins is indicated by trafficking of the detectable moiety away from the cell membrane and into or towards the nucleus.<br>
In a further embodiment, a membrane protein containing an NLS may be used to identify novel interacting proteins. In this method, an NLS-containing transmembrane protein is expressed in a cell and is allowed to translocate to the nucleus. The nuclei are then harvested and assayed for newly appeared protein bands by Coomassie staining or silver staining and then identification by mass spectroscopy. The control will be nuclei from cells expressing the membrane protein without a NLS.<br>
Use of FRET for detection of nuclear translocation.<br>
In a further aspect of the invention, involving fluorescence resonance energy transfer (FRET) (Hailey et al., 2002), a nucleated cell is co-transfected with a first nucleotide sequence encoding a first NLS-containing transmembrane protein linked to a first optically detectable protein and a<br>
second nucleotide.sequence encoding a second non-NLS-containing transmembrane, protein linked to a second optically detectable protein, whose fluorescence can be activated by the emission of the first optically detectable protein when these are in close proximity. For example, the first protein may be linked to GFP and the second any other optically detectable moiety that can be activated by the laser activated emission spectrum of GFP. This second optically detectable moiety, after activation by the GFP, emits at a different wavelength. Where oligomers are formed between the two transmembrane proteins, the two labels are in close proximity to each other and their FRET interaction can be detected. The physical interaction is detected by selective fluorescence activation of the donor and detection of emission by the acceptor, using the FRET method or its variants such as photobleaching FRET, FRAP or FLIM. Lack of a FRET interaction indicates lack of oligomerisation.<br>
Confocal microscopy with FRET between two fluorescent molecules may be performed (e.g. the spectral pairs GFP and DsRed2, or CFP and YFP) to obtain a quantifiable signal indicating translocation to the nucleus. FRET requires an overlap between the emission and excitation spectra of donor and acceptor molecules and a proximity of under 100 angstroms (10-100), making FRET a highly suitable system to assay for specific close protein-protein interactions in cells. The fluorescent proteins listed above are excellent spectral partners. A resident fluorophore in the nucleus would enable FRET to occur when a transmembrane protein tagged with second fluorophore is translocated to the nucleus. This will facilitate an easy readout, using a FRET plate reader. This method is useful for detecting interactions between two transmembrane proteins or between a transmembrane protein and another protein and provides a signal readout more amenable to automation.<br>
This method can also be used in GPCR agonist and antagonist screening procedures. In the antagonist screening method, a reduction of a FRET signal between a GPCR-NLS-GFP trafficked to the nucleus with a fluorophore in the nucleus of treated cells compared to non treated cells<br>
would indicate an antagonist effect. In the agonist screening method, the increase in the FRET signal between a GPCR-NLS-GFP trafficked to the nucleus with a fluorophore in the nucleus of treated cells compared to non treated cells would indicate an agonist effect. In a further embodiment, the doubly transfected cells may be treated with an agonist before examination for evidence of oligomerisation, since this may be enhanced in the presence of agonist. In the measurement of receptonreceptor interactions, a GPCR-NLS-GFP is co-expressed with a second GPCR-DsRED. If these receptors interact with each other and traffic together to the nucleus a nuclear FRET signal will be detected. If the receptors do not interact then no FRET signal will be obtained in the nucleus. FRET may also be measured between two fluorophore-conjugated antibodies recognising incorporated ot native epitopes on the GPCRs.<br>
EXAMPLES<br>
The examples are described for the purposes of illustration and are not intended to limit the scope of the invention.<br>
Methods of chemistry, molecular biology, protein and peptide biochemistry and immunology referred to but not explicitly described in this disclosure and examples are reported in the scientific literature and are well known to those skilled in the art.<br>
Materials and Methods<br>
Green fluorescent protein: a DNA sequence encoding the Aequoria victoria green fluorescent protein (Prasher et al., 1992) was obtained from Clontech, U.S.A.<br>
Red fluorescent protein: a DNA sequence encoding the red fluorescent proteins (Matz et al., 1999) pDsRed2 and pDsRed2-nuc were obtained from Clontech, U.S.A. This construct encodes a protein derived from Discosoma sp.<br>
COS cells and HEK cells were obtained from American Type Culture Collection, Washington, D.C. The cell culture media were prepared by laboratory services at the University of Toronto.<br>
Antagonist and agonist compounds were obtained from various commercial sources such as Sigma Chemical Company U.S.A.<br>
Antibodies used for immunodetection of epitope tags were obtained from the following sources: Anti-HA monoclonal antibody was obtained from Roche Diagnostics, U.S.A. Anti-FLAG monoclonal antibody was obtained from Sigma Chemical Company, U.S.A. Anti-c-myc monoclonal antibody was obtained from Santa Cruz, U.S.A.<br>
Radioligand 3H-SCH 23390 used in the receptor binding assay was obtained from NEN Perkin Elmer, U.S.A.<br>
Creation of DNA Constructs<br>
Nucleotide sequences encoding GPCR's or transporters were obtained from the Genbank (http://www.ncbi.nlm.nih.gov:80/entrez) web site, established by the National Library of Science. A nucleotide sequence encoding a selected transmembrane protein was attached to a nucleotide sequence encoding a selected detectable signal protein. The constructs were cloned into the vector system, pEGFP (Clontech) or the pDsRed2-N1 vector or the vector pcDNA3.<br>
1a. Construction of the human D1 dopamine receptor with a NLS in the proximal carboxyl tail (helix 8) and fused to GFP (D1-GFP and D1-NLS-GFP).<br>
Using the PCR method with the following experimental conditions, DNA encoding the human D1 dopamine receptor in the vector pcDNA3, was subjected to PCR. The reaction mixture contained water (32 microlitres), 10x Pfu buffer (Stratagene) (5 microlitres), dNTP (2'-deoxynucleoside 5'-triphosphate, 10mM) (5 microlitres), DMSO (5 microlitres), oligonucleotide primers (100ng) (1 microlitre each), DNA template (100ng), Pfu enzyme (5units). Total volume was 50 microlitres. The following PCR conditions were used, one cycle at 94 °C for 2 mins, 30-35 cycles at 94 °C for 30 secs,<br>
55 °C for 30 sees, 72 °C for 1 min, per cycle, and then one cycle at 72 °C for 5 mins.<br>
Primer set for amplification of the DNA encoding the D1-dopamine receptor:<br>
(Sequence Removed)<br>
The restriction site EcoR1 was incorporated in the primer HD1-P1, and restriction site Kpn1 was incorporated into the primer HD1-P2. The PCR product, which contained no stop codon was unidirectionally subcloned into vector pEGFP (from Clontech) at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
The NLS sequence, KKFKR from the human AT1 receptor was inserted into DNA encoding the base of TM7 (helix 8) of the D1 dopamine receptor by PCR, replacing the natural sequence coding for DFRKA.<br>
The primer set for the construction of DNA encoding D1-NLS:<br>
(Sequence Removed)<br>
Using the DNA encoding D1-GFP as template, PCR with the primers HD1-P1 and HD1-NLSF resulted in a product of 1000bp (PCR#1). Using DNA encoding D1-GFP PCR with primers HD1-P2 and HD1-NLSR resulted in a product of 300bp (PCR#2). A subsequent PCR carried out with HD1-P1 and HD1-P2 primers resulted in a product of 1300 bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting DNA encoding D1-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
All the additional constructs described below were made using the same PCR method and experimental conditions as described above for the D1 dopamine receptor, but with specific primers as described below.<br>
1b. Constructing the human dopamine D1 receptor containing a NLS and fused to RFP (D1-NLS-RFP)<br>
The NLS sequence K K F K R was inserted into the helix 8 segment of the intracellular carboxyl tail of the human D1 receptor by PCR method as follows. Using the DNA encoding the human D1 in pcDNA3 vector as template, the first PCR was carried out with HD1-P1 and HD1-NLSR primers resulting in a 1kb product. A second PCR was done using HD1-P2 and HD1-NLSF primers resulting in a 300bp product,. Using PCR#1 and PCR#2 products as templates, the final PCR was done with HD1-P1 and HD1-P2 primers which generated a 1.3kp product.<br>
D1 NLS was subcloned into vector pDsRed (Clontech ) at EcoRI and Kpnl and fused to RFP. Primer sequences:<br>
(Sequence Removed)<br>
1c. Construction of the dopamine D1 receptor with a hemagglutinin (HA) epitope tag in the amino terminus<br>
The HA-Tag is as follows: Nucleotide sequence: TACCCTTACGACGTGCCGGATTACGCC HA amino acid sequence: YPYDVPDYA<br>
The HA epitope tag was inserted into the amino terminal of the human D1 receptor using D1-pcDNA3 as template with the following primers: P1HA-BamH:<br>
(Sequence Removed)<br>
1d. Construction of the human dopamine D1 receptor with a HA epitope and NLS in the proximal carboxyi tail (helix 8) (D1HA-NLS)<br>
Primer set for the PCR amplification of DNA encoding D1-NLS (helix 8), using DNA encoding D1-HA as template. Using DNA D1-HA as template with primers T7 and HD1-NLSR primers the resulting amplified DNA was 1000bp (PCR#1). Using DNA D1-HA as template with primers Sp6 and HD1-NLSR primers the resulting DNA was 300bp, (PCR#2). Using primers T7 and Sp6 primers and the product of PCR#1 and PCR# 2 as templates the resulting DNA was 1300 bp (PCR#3).<br>
(Sequence Removed)<br>
The result D1HA-NLS (helix 8) PCR was blunt-ended into pcDNA3 at EcorV. The correct orientation clone was sequenced.<br>
(Sequence Removed)<br>
1e. Construction the dopamine D1 receptor with a NLS in intracellular loop 3, fused to GFP (D1-NLS-IC3-GFP)<br>
Primer set for the construction of D1-NLS-IC3 EGFP:<br>
(Sequence Removed)<br>
Using D1 pcDNA3 template: PCR#1: HD1-P1 and D1NLSR-IC3 primers<br>
PCR#2: HD1-P2 and D1NLSF-IC3 primers (500bp)<br>
PCR#3: HD1-P1 and HD1-P2 primers using PCR#1 and PCR#2 as templates<br>
(1.3 kb)<br>
The resulting DNA fragment encoding D1-NLS-IC3 was subcloned into vector<br>
pEGFP at EcoR1 and Kpn1.<br>
(Sequence Removed)<br>
The NLS sequence KKFKR was inserted into the IC loop 3 segment of the D1 receptor replacing the sequence MFSKR, using D1 pcDNA3 as template.<br>
Using the DNA encoding D1 in pcDNA3 as template, PCR was carried out with the following primers HD1-P1 and D1-NLSR-IC3 resulting in a product of 800 bp (PCR#1). Using DNA encoding D1 in pcDNA3 with primers HD1-P2 and HD1-NLSF-IC3 resulted in a product of 500 bp (PCR#2). A subsequent PCR carried out with HD1-P1 and HD1-P2 primers resulted in a product of 1300 bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting construct encoding D1-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
1f. Construction of human D1 dopamine receptor with a NLS in intracellular loop 2 fused with GFP (D1-NLS-IC2-GFP)<br>
The primer set for the construction of DNA encoding D1NLS-IC2<br>
(Sequence Removed)<br>
Using DNA encoding D1 dopamine receptor in pcDNA3 as template, PCR with the primers HD1-P1 and D1NLSR-IC2 (PCR#1), resulted in a product of 500 bp. Using DNA encoding D1 dopamine receptor in pcDNA3 as a template with primers HD1-P2 and D1NLSF-IC2 (PCR#2) resulted in a product of800 bp. A subsequent PCR carried out with primers HD1-P1 and HD1-P2 using PCR#1 and PCR#2 as templates resulted in a product of 1300 bp.<br>
The resulting DNA encoding D1NLS-IC2 PCR was subcloned into vector EGFP at EcoR1 and Kpn1.<br>
(Sequence Removed)<br>
1g. Construction of the human D1 dopamine receptor with a NLS in intracellular loop 1 fused with GFP (D1-NLS-IC1-GFP)<br>
The primer set for the construction of DNA encoding D1-NLS-IC1.<br>
(Sequence Removed)<br>
Using the DNA encoding D1 dopamine receptor in pcDNA3 as template, PCR with the primers HD1-P1 and D1-NLSR-IC1 (PCR#1), resulted in a product of 300 bp. Using DNA encoding D1 dopamine receptor in pcDNA3 as template PCR with primers HD1-P2 and D1NLSF-IC1 resulted in a product of 1000 bp (PCR#2). A subsequent PCR carried out with primers HD1-P1 and HD1-P2 using PCR#1 and PCR#2 as templates resulted in a product of 1300 bp.<br>
The resulting DNA encoding D1-NLS-IC1 was subcloned into vector pEGFP at EcoR1 and Kpn1. C <br>
(Sequence Removed)<br>
1h. Construction of human dopamine D1 receptor with an alternate NLS in the proximal carboxyl tail and fused to GFP (D1-NLS2-GFP)<br>
The PCR method was used to introduce the NLS sequence PKKKRKV in replacement of the natural sequence ADFRKAF in the D1 receptor. The DNA encoding the D1 dopamine receptor in pcDNA3 was subjected to PCR with the primers HD1-P1 and HD1-NLS2R, resulting in a product of 1kb (PCR#1). Another PCR using D1 in pcDNA3 with primers HD1-P2 and HD1-NLS2F resulted in a product of 300bp (PCR#2). The third PCR using PCR#1 and PCR#2 as templates HD1-P1 and HD1-P2 primers resulted in a product of 1.3kb, and was subcloned into vector pEGFP at EcoR1 and Kpn1.<br>
(Sequence Removed)<br>
2. Construction the dopamine D2 and D2-NLS dopamine receptors fused to GFP (D2-GFP and D2-NLS-GFP)<br>
Primer set for amplification of the DNA in pcDNA3 encoding the D2-dopamine receptor.<br>
(Sequence Removed)<br>
The restriction site EcoR1 was incorporated into primer HD2-P1, and the restriction site Kpnlwas incorporated into primer HD2-P2. The D2-PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP (Clontech) at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
Primer set for the construction of D2-NLS-GFP<br>
(Sequence Removed)<br>
The NLS sequence KKFKR was inserted into the base of TM7 segment of the D2 receptor replacing the sequence IEFRK, using D2-GFP DNA construct as template.<br>
Using the DNA encoding D2-GFP as template, PCR was carried out with the following primers HD2-P1 and HD2-NLSR resulting in a product of 1300bp (PCR#1). Using DNA encoding D2-GFP PCR with primers HD2-P2 and HD1-NLSF resulted in a product of 100bp (PCR#2). A subsequent PCR carried out with HD2-P1 and HD2-P2 primers resulted in a product of<br>
1400bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting construct encoding D2-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
3.	Construction of DNA encoding the D3 and D5 dopamine<br>
receptors fused to GFP (D3-GFP and D5-GFP)<br>
Primer set for amplification of the DNA in pcDNA3 encoding the D3-dopamine receptor.<br>
(Sequence Removed)<br>
The restriction site Hindi 11 was incorporated into primer HD3-Hind, and the restriction site Kpnl and was incorporated into primer HD3-Kpn. The D3-PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at Hindlll and Kpnl and inframe with the S start codon of the GFP protein.<br>
Primer set for amplification of the DNA in pcDNA3 encoding the D5 dopamine receptor.<br>
(Sequence Removed)<br>
The restriction site Kpnl and was incorporated into primer HD5-Kpn. The D5-PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at EcoRI and Kpnl and inframe with the start codon of the GFP protein.<br>
4.	Construction of the Histaminel and Histamine1-NLS<br>
receptors fused to GFP (H1-GFP and H1-NLS-GFP)<br>
Primer set for amplification of the DNA, from human genomic DNA, encoding the encoding the H1 histamine receptor.<br>
(Sequence Removed)<br>
This H1-PCR product was used as a template for the subsequent PCR experiment.<br>
Primer set for amplification of the DNA encoding the H1-GFP construct.<br>
(Sequence Removed)<br>
The restriction site Pstl was incorporated into primer H1-PST, and the restriction site Apal was incorporated into primer H1-APA. This H1-PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at Pstl and Apal and inframe with the start codon of the GFP protein.<br>
Primer set for amplification of the DNA encoding the H1-NLS-GFP<br>
(Sequence Removed)<br>
The NLS sequence: KKFKR was inserted into the DNA encoding the TM7 segment of the H1 receptor by the PCR method, using the H1-GFP template, replacing the sequence ENFKK. PCR with H1-PST and H1-NLSR primers gave a product of 1500bp. The resulting fragment encoding H1-NLS was subcloned into vector pEGFP at Pstl and Apal restriction sites.<br>
5. Construction the cysteinyl leukotriene receptor 1 and CysLTI-NLS fused to GFP (CysLT1-GFP and CysLT1-NLS-GFP).<br>
Primer set for amplification of the DNA in pcDNA3 encoding the CysLTI receptor.<br>
(Sequence Removed)<br>
The restriction site EcoR1 was incorporated into primer LT1-EcoRI and the restriction site Kpnlwas incorporated into primer LT1-Kpnl. The CysLT1-PCR DNA product, which contained no stop codon, was unidirectionally subcloned into vector PGFP at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
Primer set for amplification of the DNA encoding the CysLTI -NLS -GFP<br>
(Sequence Removed)<br>
The NLS sequence KKFKR was inserted into the DNA encoding the TM7 segment of the CysLT1 by PCR, using DNA encoding the CysLT1-GFP as template, replacing the sequence GNFRK. Using the DNA encoding CysLT1-GFP as template, PCR with the following primers LT1-EcoRI and LT1-NLSR resulted in a fragment of 900bp (PCR#1). Using DNA encoding CysLTI -GFP PCR with primers LT1 -Kpnl and LT1 -NLSF resulted in a fragment of 100bp (PCR#2). A subsequent PCR carried out with LT1-EcoRI and LT1-Kpnl primers resulted in a product of 1000bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting DNA encoding CysLT1-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
6. Construction of the cysteinyl leukotriene receptor CysLT2 and CysLT2-NLS fused to GFP (CysLT2-GFP and CysLT2-NLS-GFP)<br>
Primer set for amplification of the DNA in pcDNA3 encoding the CysLT2 receptor.<br>
(Sequence Removed)<br>
The restriction site EcoR1 was incorporated into primer LT2-EcoRI, and the restriction site Kpn1 was incorporated into primer LT2-Kpnl. The CysLT2 product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
Primer set for the amplification of the CysLT2-NLS-GFP<br>
(Sequence Removed)<br>
The NLS sequence KKFKR was inserted into the TM7 segment of the CysLT2 by PCR method replacing the sequence ENFKD. Using the DNA encoding CysLT2-EGFP as template, a PCR with the following primers LT2-<br>
EcoR1 and LT2-NLSR primers resulted in a fragment of 900bp (PCR#1). Using DNA encoding LT2-Kpnl and LT2-NLSF primers a PCR resulted in a fragment of 200bp (PCR#2). A subsequent PCR carried out with LT2-EcoR1 and LT2-Kpnl primers using the product of PCR#1 and the product of PCR#2 as templates resulted in a product of 1100 bp. The resulting DNA encoding CysLT2-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
7. Construction of the M1 muscarinic receptor and the muscarinic NLS receptor fused to GFP (M1-GFP and M1-NLS-GFP)<br>
Primer set for amplification of the DNA encoding the muscarinic receptor (M1) from human genomic DNA.<br>
(Sequence Removed)<br>
The restriction site Pstl was incorporated into primer M1-PST, and the restriction site BamHIwas incorporated into primer M1-BAMH. The M1 PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at Pstl and BamHI and inframe with the start codon of the EGFP protein.<br>
(Sequence Removed)<br>
The NLS sequence KKFKR was inserted into the TM7 segment of the M1 by PCR, using the MR1 template, replacing the sequence KAFRD. Using the DNA encoding MR1 as template, PCR with the following primers using M1-PST and M1-NLSR resulted in a product of 1200bp (PCR#1). Using DNA encoding MR1 a PCR with primers reaction using M1-BAMH and M1 -NLSF primers resulted in a product of 10Obp (PCR#2). A subsequent PCR carried out with M1-PST and M1-BAMH primers resulted in a product of 1300bp using the product from PCR#1 and the product from PCR#2 as<br>
templates. This fragment encoding MR1-NLS was subcloned into vector pEGFP at Pstl and BamHI restriction sites.<br>
8. Construction of the serotonin (5HT1B) and the serotonin NLS receptors fused to GFP (5HT1B-GFP and 5HT1B-NLS-GFP)<br>
Primer set for amplification of the DNA encoding the 5HT1B receptor from the plasmid pcDNA3 encoding the 5HT1B receptor.<br>
(Sequence Removed)<br>
The restriction site EcoR1 was incorporated into primer 5HT1B-E1 and the restriction site Kpnlwas incorporated into primer 5HT1B-KPN. The 5HT1B -PCR product, which contained no stop codon, was unidirectionally subcloned into vector pEGFP at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
(Sequence Removed)<br>
The NLS sequence: KKFKR was inserted into the TM7 segment of the 5HT1B by PCR using 5HT1B-EGFP template, replacing the sequence EDFKQ. Using the DNA encoding 5HT1B-EGFP as template, a PCR with the following primers with 5HT1B-E1and HD1-NLSF primers gave a product of 1100bp (PCR#1). Using DNA encoding 5HT1B-EGFP with 5HT1B-KPN and HD1-NLSR primers resulted in a product of 100bp (PCR#2). A subsequent PCR carried out with 5HT1B-E1 and 5HT1 B-KPN primers resulted in a product of 1200bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting DNA encoding 5HT1B-NLS was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
9.	Construction of the beta2-adrenergic (beta2-AR) and the<br>
beta2-AR-NLS1 receptors fused to GFP (beta2-AR-GFP and beta2AR-<br>
NLS1-GFP)<br>
Primer set for amplification of the DNA encoding the beta2-AR receptor from pcDNA3.<br>
(Sequence Removed)<br>
The restriction site Kpnlwas incorporated into primer beta2-Kpn. The beta 2-AR product, which contained no stop codon, was unidirectionally subcioned into vector pEGFP at EcoR1 and Kpn1 and inframe with the start codon of the GFP protein.<br>
The NLS sequence KKFKR was inserted into the TM7 segment of the beta2-AR by PCR using beta2-AR-EGFP template, replacing the sequence PDFRI. Using the DNA encoding beta2-AR-EGFP as template, a PCR with the following primers with T7 and B2-NLSR primers resulted in a product of 1100bp (PCR#1). Using DNA encoding beta2-AR-EGFP with beta2-Kpn and B2-NLSF primers resulted in a product of 300bp (PCR#2). A subsequent PCR carried out with primers T7 and beta2-Kpn resulted in a product of 1300bp using the product from PCR#1 and the product from PCR#2 as templates. The resulting DNA encoding beta2-NLS was subcioned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
10.	Construction of the beta 2-adrenergic receptor with an<br>
alternate NLS and fused to GFP (beta2-NLS2-GFP)<br>
Primer set for amplification of the DNA encoding the beta2-NLS2 receptor from pcDNA3.<br>
(Sequence Removed)<br>
Using the DNA encoding beta2-AR-GFP as template, a PCR with the following primers with T7 and B2D1NLSR primers resulted in a product of 1000bp (PCR#1). Using DNA encoding beta2-AR-GFP with beta2-Kpn and B2D1NLSF primers resulted in a product of 300bp (PCR#2). A subsequent PCR carried out with primers T7 and beta2-Kpn primers using PCR#1 and PCR#2 as templates resulted in a product of 1300 bp. The resulting DNA encoding beta2-NLS2 was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
The NLS sequence AFSAKKFKR was inserted into the TM7 segment of the beta2-AR by PCR using beta2-GFP template, replacing the sequence CRSPDFRIA.<br>
The resulting DNA encoding beta2-NLS2 was subcloned into vector pEGFP at EcoR1 and Kpn1 restriction sites.<br>
11. Construction of the beta 2-adrenergic receptor with an alternate NLS and fused to GFP (beta2-NLS3-GFP)<br>
The NLS sequence K K F K R was inserted into another location of the proximal segment of the carboxyl tail of the beta2-AR. Using the DNA encoding the beta2AR in pcDNA3 vector as template, PCR was carried out with T7 and r B2-NLS3R primers resulting a 1000bp product. PCR using primers Beta2-Kpn and B2-NLS3F resulted in a 300bp product. Using PCR#1 and PCR#2 products as templates PCR with T7 and Beta2-Kpn primers generated a 1300bp product (beta2AR-NLS3) which was subcloned into vector pEGFP at EcoR1 and Kpn1.<br>
Primer set for Beta2-NLS3-GFP<br>
(Sequence Removed)<br>
12. Construction of the dopamine transporter fused to GFP (DAT-GFP)<br>
The full length cDNA encoding the human dopamine transporter (hDAT) was amplified using DAT in pcDNA3 as template by PCR with primer T7 and primer DT-1 (5' CGTCTCTGCTCCCTGGTACCGCCACCTTGAGCCAGTGG 3'). This PCR product contained no stop codon and was unidirectionally subcloned into vector pEGFP (from Clontech) at the EcoR1 and Kpn1 restriction sites and inframe with the start codon of the GFP protein.<br>
13a. Construction of the human dopamine transporter containing a NLS and fused to RFP (DAT-NLS-RFP)<br>
The cDNA encoding the human dopamine transporter (hDAT) was amplified by PCR with 1718 and hDAT-NLSF primers, producing a fragment of 100 bp. The cDNA encoding the human dopamine transporter (hDAT) was also amplified by PCR with T7 and hDAT-NLSR primers, producing a fragment of1.7 kB. These two PCR fragments were used as templates with T7 and 1718 primers, resulting in a fragment of 1.8 kB.<br>
(Sequence Removed)<br>
This PCR product was unidirectionally subcloned into vector pRFP at EcoR1 and Kpn1 and inframe with the start codon of the RFP protein.<br>
The resulting PCR fragment encoded the NLS sequence K K F K R after TM12 as follows:<br>
(Sequence Removed)<br>
13b. Construction of the human dopamine transporter with a NLS and fused to GFP (DAT-NLS-GFP)<br>
The NLS sequence K K F K R was inserted into the proximal carboxyl tail following the transmembrane 12 segment of the human DAT. Using the DNA encoding the human DAT-cDNA in pcDNA3, as template, the first PCR<br>
was carried out with T7 and hDAT-NLSR primers resulting a 1.7kb product. A second PCR was done using 1718 and hDAT-NLSF primers resulting in a 100bp product, then using PCR#1 and PCR#2 products as templates the final PCR was done with T7 and 1718 primers which generated a 1.8kp product (DAT-NLS) which was subcloned into vector pEGFP (Clontech ) at EcoR1 and Kpn1 and fused GFP. Sequences of the primers:<br>
(Sequence Removed)<br>
14.	Construction of the human serotonin transporter fused to<br>
GFP (SERT-GFP)<br>
The full length human SERT cDNA was isolated by PCR from pcDNA3 containing the SERT cDNA, using the two following primers:<br>
(Sequence Removed)<br>
This PCR product contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech ) at Hindlll and Kpnl and inframe with the start codon of the GFP protein.<br>
15.	Construction of the human Low Density Lipoprotein Receptor<br>
fused to GFP (LDL-R-GFP)<br>
The full length cDNA encoding LDL was subjected to PCR with LDLR-HIND and LDLR-KPN primers:<br>
(Sequence Removed)<br>
This PCR product (2600bp) contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech) at Hindlll and Kpnl and inframe with the start codon of the GFP protein.<br>
16.	Construction of the human Low Density Lipoprotein Receptor<br>
with a NLS and fused to GFP (LDLR-NLS-GFP)<br>
The NLS sequence KKFKR was inserted into DNA encoding the LDL receptor by PCR, replacing the natural sequence coding for RLKNI. The primer set for the construction of DNA encoding LDL-NLS:<br>
(Sequence Removed)<br>
Using the human DNA encoding LDL cDNA in pcDV1 as template, PCR with the primers LDLR-HIND and LDL-NLSR resulted in a product of 2450 bp (PCR#1). Using DNA encoding LDL as a template with primers LDLR-KPN and LDL-NLSF resulted in a product of 150 bp (PCR#2). A subsequent PCR carried out with primers LDLR-HIND and LDLR-KPN using the product of PCR#1 and PCR#2 as template resulted in a product of 2600 bp.<br>
The resulting PCR contained the NLS sequence KKFKR mutation as follows:<br>
(Sequence Removed)<br>
This PCR product contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech) at Hindlll and Kpnl and inframe with the start codon of the GFP protein.<br>
17.	Construction of the Epidermal Growth Factor Receptor fused<br>
to GFP (EGFR-GFP)<br>
The full length human EGFR cDNA in Prkf vector was isolated by PCR with the two following primers:<br>
(Sequence Removed)<br>
This PCR product (3600 bp) contained no stop codon and was<br>
unidirectionally subcloned into vector pEGFP (Clontech) at Xhol and Kpnl and<br>
inframe with the start codon of the GFP protein.	<br>
18.	Construction of the human serotonin transporter with a NLS<br>
and fused to GFP (SERT-NLS-GFP)<br>
The NLS sequence KKFKR was inserted into DNA encoding the SERT by PCR, replacing the natural sequence coding for GTFKE.<br>
The primer set for the amplification of the DNA encoding SERT-NLS.<br>
(Sequence Removed)<br>
Using the human SERT-cDNA in PcDNA3 as template, PCR with the primers SERT-HIND and SERT-NLSR resulted in a product of 1800 bp (PCR#1). Using DNA encoding SERT as a template with primers SERT-KPN and SERT-NLSF resulted in a product of 100 bp (PCR#2). A subsequent PCR carried out with primers SERT-HIND and SERT-KPN primers using the product of PCR#1 and PCR#2 as template resulted in a product of 1900 bp.<br>
The resulting PCR product encoded the NLS sequence KKFKR mutation after TM12 of the SERT as follows:<br>
(Sequence Removed)<br>
This PCR product contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech) at Hindlll and Kpnl and inframe with the start codon of the GFP protein.<br>
19.	Construction of the metabotropic glutamate-4-receptor fused<br>
to GFP, with and without NLS (mGluR4-GFP and mGluR4-NLS-GFP)<br>
The DNA encoding mGluR4 was isolated from a rat cDNA using the primer set<br>
(Sequence Removed)<br>
A restriction site Hindlll was incorporated into primer GLUR4-HIND, and a restriction site EcoRI was incorporated into primer GLUR4-ECORI. The mGluR4-PCR product, which contained no stop codon, was unidirectionally subcloned into vector EGFP (Clontech) at Hindlll and Ecorl and inframe with the start codon of the GFP protein.<br>
The NLS KKFKR was introduced into the DNA encoding the mGluR4 replacing the natural sequence KRKRS.<br>
Primer set for the amplification of the DNA to introduce the NLS into the Rat mGluR4-EGFP<br>
(Sequence Removed)<br>
Using the rat DNA encoding GluR4 as template PCR with the primers GLUR4-H1ND and GLUR4-NLSR resulted in a product of 2600 bp (PCR#1). Using DNA encoding GluR4 with primers GLUR4-ECORI and GLUR4-NLSF resulted in a product of 160 bp (PCR#2). A subsequent PCR carried out using the product of PCR#1 and PCR#2 as template, with primers: GLUR4 HIND and GLUR4-ECORI, resulted in a product of 2760 bp.   <br>
The resulting PCR contained the NLS sequence KKFKR mutation as follows:<br>
(Sequence Removed)<br>
This PCR product was unidirectionally subcloned into vector pEGFP (Clontech) at Hindlll and EcoRI and inframe with the start codon of the GFP protem.<br>
20. Construction of the Human Insulin Receptor fused to GFP (IR-GFP)<br>
The full length fR cDNA in plasmid pRK5 was isolated with the two PCR primers:<br>
(Sequence Removed)<br>
The PCR product (4.2kb) contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech ) at Hindlll and Apal and fused to the GFP protein.<br>
21.	Construction of the Human Insulin Receptor with a NLS and<br>
fused to GFP (IR-NLS-GFP)<br>
The NLS sequence KKFKR was introduced into the human insulin receptor to replace the sequence LYASS.<br>
Using the human insulin receptor cDNA in pRK5 vector as template, the first PCR#1 with HIR-HIND and HIR-NLSR primers generated a 2.9 kb product, the second PCR #2 with HIR-APA and HIR-NLSF primers generated a 1.3 kb product, and then using the products from PCR#1 and PCR #2 as templates, the third PCR #3 produced a fragment with HIR-HIND and HIR-APA primers (4.2 kb). This contained no stop codon and was unidirectionally subcloned into vector pEGFP at Hindlll and Apal and thus fused to the GFP protein. Primers for HIR-NLS:<br>
(Sequence Removed)<br>
22.	Construction of the human Erythropoietin receptor fused to<br>
GFP (EPO-GFP).<br>
Using PCR method and the cDNA in pc3.1 vector encoding the human Erythropoietin receptor (EPO) as template, the full length cDNA was isolated with the following primers:<br>
(Sequence Removed)<br>
This PCR product (1.6kb) contained no stop codon and was unidirectionally<br>
subcloned into vector pEGFP at Hindlll and Kpnl and fused to the GFP protein.<br>
23.	Construction of the human Erythropoietin receptor with a NLS<br>
and fused to GFP (EPO-NLS-GFP).<br>
The NLS sequence KKFKR was inserted into the DNA encoding the EPO receptor by PCR, replacing its natural sequence RRALK.<br>
Using the human EPO-cDNA in pc3.1 as template, the first PCR #1 with T7 and EPO-NLSR primers generated a 900bp product, the second PCR #2 with EPO-KPN and EPO-NLSF primers generated a 700bp product, and then using the products from PCR#i and PCR #2 as templates, the third PCR #3 with 77 and EPO-KPN primers produced a 1.6 kb fragment. This PCR product (1.6kb) contained no stop codon and was unidirectionally subcloned into vector pEGFP at Hindlll and Kpnl and thus fused to the GFP protein. Primer sequences:<br>
(Sequence Removed)<br>
24.	Construction of the human epidermal growth factor receptor<br>
fused to GFP (EGFR-GFP)<br>
• Using the human epidermal growth factor receptor cDNA in Prk5 vector as template, the full length cDNA was isolated by PCR with the two following primers:<br>
(Sequence Removed)<br>
This PCR product (3.6kb) contained no stop codon and was unidirectionally subcloned into vector pEGFP (Clontech ) at Xhol and Kpnl and fused to the GFP protein.<br>
25.	Construction of the human epidermal growth factor receptor<br>
with an NLS and fused to GFP (EGFR-NLS-GFP)<br>
The NLS sequence K K F K R was inserted into the sequence of the human epidermal growth factor receptor by PCR method as follows. Using the human EGFR cDNA in Prk5 vector as template, the first PCR was carried out with HER-XHO and EGF-NLSR primers resulting in a 2.1 kb product. A second PCR was done using HER-KPN and EGF-NLSF primers resulting a 1.5kb product, and then using PCR#1 and PCR#2 products as templates, the final PCR was done with HER-XHO and HER-KPN primers, which generated a 3.6kp product (EGFR-NLS) which was subcloned into vector pEGFP (Clontech) at Xho1 and Kpn1 and fused to GFP. Primer sequences:<br>
(Sequence Removed)<br>
26.	Construction of the human D1 dopamine receptor containing<br>
2 NLSs and fused to RFP (D1-NLS(Helix 8 and C-tail)-RFP)<br>
A second NLS sequence K K K R K was inserted into the carboxyl tail segment of the human D1-NLS-Helix 8 by PCR method as follows. Using the DNA encoding the human D1-NLS-Helix 8 in pDsRed vector as template, the first PCR was carried out with HD1-P1 and HD1-NLSCR primers resulting in a 1.2kb product, and a second PCR was done using HD1-P2 and HD1-NLSCF primers resulting in a 100bp product. Then using PCR#1 and PCR#2 products as templates, the final PCR was done with HD1-P1 and HD1-P2 primers which generated a 1.3kp product (D1-NLS-Helix 8 and C-tail) which was subcloned into pDsRed vector at EcoRI and Kpnl and fused to the DsRed protein.<br>
Primer sequences:<br>
(Sequence Removed)<br>
27.	Construction of the Mu opioid receptor fused to GFP (Mu-<br>
GFP)<br>
Using the DNA encoding the Mu opioid receptor in pcDNA3 vector as template, PCR was carried out with the following two primers.<br>
(Sequence Removed)<br>
Restriction site EcoRI was incorporated into primer RATMU-1. Restriction site Kpnl was incorporated into primer RATMU-2<br>
The PCR product (1.2 kb) which contained no stop codon, was then unidirectionally subcloned into vector pEGFP (Clontech) at EcoRI and Kpn1 and thus fused to GFP.<br>
28.	Construction of the Mu opioid receptor containing a NLS and<br>
fused to GFP (Mu-NLS-GFP)<br>
The NLS sequence K K F K R was inserted into the proximal carboxyl tail segment (helix 8) of the Mu opioid receptor by PCR as follows. Using the DNA encoding the Rat Mu in pcDNA3 as template, the first PCR was carried out with RATMU1 and MU-NLSR primers resulting a 1000bp product, another<br><br>
second PCR was done using RATMU-2 and MU-NLSF primers resulting a 200bp product. Using PCR#1 and PCR#2 products as templates the final PCR was done with RATMU1 and RATMU2 primers generated a 1200bp product (Mu-NLS) which was subcloned into vector pEGFP at EcoR1 and Kpn1 and fused to GFP. Primer sequences:<br>
(Sequence Removed)<br>
Cell culture and transfection<br>
COS-7 monkey kidney cells and HEK293T human embryonic kidney cells (American Type Culture Collection, Manassa, VA) were maintained as monolayer cultures at 37°C and 5% CO2 in minimal essential medium supplemented with 10% fetal bovine serum and antibiotics. For cell membrane harvesting, 100 mm plates of cells were transiently transfected at 70-80% confluency using lipofectamine reagent (Life Technologies, Rockville, MD). For confocal microscopy studies, 60 mm plates of cells were transiently transfected at 10-20% confluency using lipofectamine reagent. Six hours after transfection, the solution was removed and fresh media added and again replaced with fresh media 24 hours after transfection.<br>
Transfection medium was prepared by mixing 120 microlitres medium without antibiotics and/or fetal bovine serum (FBS) and 15 microlitress lipofectamine in a 14 ml tube. 2 micrograms DNA construct encoding the desired fusion protein and 120 microlitres medium were mixed and added to the 14 ml tube, which was mixed gently and incubated at room temperature for 25 minutes. A further 4 ml of medium was added and mixed. If multiple transmembrane proteins are being transfected, the cDNAs are mixed and transfected together. Growth medium was removed from a plate of cells and<br>
replaced with the transfection mixture from the 14 ml tube. Cells were incubated with the transfection mixture for 5 - 6 hours, after which the mixture was removed and replaced with regular growth medium containing FBS and antibiotics. Cells were incubated with a change of regular growth medium on the second day.<br>
Treatment with Test Compounds<br>
Protocol for determining retardation of translocation off cell surface<br>
Test compounds were prepared in a stock solution of 1 millimolar concentration and diluted in growth medium to achieve a final concentration ranging between 10 nanomolar and 10 micromolar when added to cell plates. Fresh compound-containing medium was added to cells at 6 hours, 22 hours, 30 hours and 42 hours after transfection.<br>
Protocol for determining promotion of translocation off cell surface<br>
Test compounds were prepared in a stock solution of 1 millimolar and diluted in growth medium at 37°C to achieve a final concentration of 10 micromolar when added to cells. Cell cultures were examined microscopically to focus on a single cell and to detect the presence of surface expression of the detectable label protein. Growth medium was replaced with compound-containing medium and the cells were examined microscopically in real time 5,10,15,20, 30 and 35 mins. after addition of compound for changes in distribution of the detectable moiety.<br>
Microscopy<br>
Cells were visualised using the LSM510 Zeiss confocal laser microscope. GFP was visualised following excitation with the argon laser at 488nm excitation wavelength and the DsRed was visualised following excitation with the helium neon laser at 543 nm wavelength for excitation. The confocal images were captured on disk and evaluated. In each<br>
experiment, multiple fields of cells (n=6-8 with 30-90 cells each) were counted and evaluated for localisation of the signal on the cell surface, in the cytoplasm and in the nucleus.<br>
Fluorocvtometrv<br>
A 96 well plate was coated with poly-L-ornithine (1/10 in PBS) and incubated for one hour. 50,000 cells were added to each well and transfected with cDNA encoding the epitope-tagged receptor, using Lipofectamine (Invitrogen, U.S.A.). The medium (MEM) was changed every 12 hours and contained test drug at varying concentrations or vehicle. 48 hours later, cells were washed and fixed with 4% paraformaldehyde and incubated for 30 min on ice. Cells were then incubated with the primary antibody directed against the epitope and then with the secondary antibody conjugated to FITC (fluorescein isothiocyanate) and kept shielded from light. Excess antibody was washed off and the signal detected by reading the piate in a Cytofluor 4000 (Perspective Biosystems, U.S.A.). The FITC was activated using light at 488 nm for excitation and the signal read at its emission wavelength 530 nm.<br>
Radioligand Binding<br>
Cells were transfected with DNA encoding D1-NLS and treated with varying concentrations of antagonist drug or left untreated. After 48 hours, the cells were washed, harvested, lysed and homogenised by polytron. The membrane fraction was collected by centrifugation and then layered over 35% sucrose solution and centrifuged at 30,000 rpm at 4 degrees C for 90 min to collect the heavy membrane fraction. The supernatant was again centrifuged at 35,000 rpm at 4 degrees C for 60 min to collect the light membrane fraction. The membranes were subjected to radioligand binding assay using [3H]-SCH23390 with (+)butaclamol 10 micromolar used to define specific binding. The incubation was at room temperature for 2 hours, followed by rapid filtration and quantitation by scintillation counting.<br>
Isolation of Nuclei from Cultured Cells<br>
Wash cells with PBS 3 times, using 10 ml and scrape gently off the culture dishes. Pool and spin the cells at 500g for 5 min, at 4 degrees C. Resuspend pelleted cells in lysis buffer (Tris HC110 mM, pH 7.4, NaC110 mM, MgCI2 3 mM) and inhibitor cocktail (0.5% leupeptin,1% soybean<br>
trypsin,1% benzamidine) at a density of 50 million cells per ml. Homogenize with sterile glass Teflon pestle B (tight clearance 20-50 mm; Bellco Glass) using 100 up and down strokes.<br>
Spin at 4 degrees C for 700 g fori 0 min, then sequentially centrifuge supernatant at 10 000 g, for 15 min at 4 C (to remove mitochondria) and 120 000g for 60 min, 4 degrees C (to remove plasma membrane).<br>
The nuclear pellet is resuspended in lysis buffer (with inhibitors) and 0.1 % of NP-40, kept on ice 5 min and then centrifuged at 700 g for 10 min at 4 degrees C. The supernatant is discarded and washing process repeated 3 x with 15 ml lysis buffer. Nuclear pellet is resuspended in 2 ml lysis buffer and loaded on top of discontinuous sucrose gradient made by successive layering of 4.5 ml of 2.0 and 1.6 M sucrose containing MgCI2 1 mM and spun at 100 000g for 60 min at 4 degrees C. The pellet at the bottom of the tube is collected and will contain pure nuclei.<br>
Example 1: Dopamine D1 receptor fused to the red fluorescent protein (D1-RFP) or containing a NLS and fused to the red fluorescent protein (D1-NLS-RFP)<br>
The sequence of the dopamine D1 receptor, which does not contain an NLS, was modified to replace the amino acids DFRKA at the base of TM7 domain with the NLS sequence KKFKR (which corresponds to the NLS of the human AT1 receptor), as described in the methods (see Figure 1). DNA constructs were created encoding the D1 dopamine receptor fusion proteins D1-RFP and D1-NLS-RFP. COS cells were transfected with DNA encoding D1-NLS-RFP or D1-RFP (2 micrograms) and incubated for 24 and 48 hours. The cells were examined by confocal microscopy at 100X magnification.<br>
Cells were counted manually in 8 to 10 microscopic fields and the percentage labelling in different subcellular compartments was calculated.<br>
At 24 and 48 hours, cells transfected with D1-RFP showed expression at the cell surface in the majority of cells, whereas cells transfected with D1-NLS-RFP showed little receptor expression at the cell surface, with nuclear localisation in 60% of the cells at 24 hours, and in 80% of the cells at 48 hours.<br>
Example 2:   Dopamine D1 receptor fusion protein containing a NLS (D1-NLS-RFP) treated with antagonist<br>
COS cells were transfected with a construct encoding D1-NLS-RFP and with wildtype D1 (2 micrograms) for 48 hours. At 6, 22, 30 and 42 hours after transfection, the cells were treated with the dopamine D1 receptor antagonist SCH23390 (final concentration 10 micromolar). Also at 6, 22, 30 and 42 hours after transfection, cells were treated with the antagonist (+)butaclamol (final concentration 10µM). Control cells received no antagonist treatment.<br>
At 48 hours, the majority of control cells had detectable D1-NLS-RFP in the nucleus. In contrast, the majority of antagonist-treated cells had fluorescence only on the cell surface, while 42% had fluorescence both on the surface and in the nucleus.<br>
Example 3: Dopamine D1 receptor (D1-GFP) co-expressed with D1 receptor containing a NLS (D1-NLS)<br>
HEK cells were transfected with DNA constructs encoding D1-NLS (3 micrograms) and/or D1-GFP (1.5 micrograms), and incubated for 48 hours. The cells were also transfected with a plasmid encoding DsRed-NUC to verify the localisation of the nucleus (1 microgram).<br>
Cells were also transfected with. DNA encoding D1-GFP (2 micrograms), incubated for 48 hours and examined by confocal microscopy.<br>
D1 -GFP expressed alone revealed that 90% of the cells demonstrated cell surface labelling and 10% showed both nuclear and cell surface labelling. With any DNA encoding a GPCR transfection, up to 10% of cells may be observed with a nuclear localisation.<br>
Cells expressing D1-GFP and D1-NLS showed 35% of cells with both nuclear and cell surface labelling and 70% with receptor expression on cell surface only. This experiment indicated that D1-GFP co-trafficked with D1-NLS resulting from oligomerisation of the D1-NLS and D1-GFP.<br>
Example 4: Dopamine D1 receptor containing a NLS (D1-NLS-GFP), was treated with an antagonist in a dose response study<br>
HEK cells were transfected with DNA encoding D1-NLS-GFP (2 micrograms), and D1-WT (6 micrograms) for 48 hours. These cells were treated 6 hrs after transfection with SCH- 23390 (10 micromolar), or (+)butaclamol (10 micromolar). The medium containing antagonist was changed at 6,22,30 and 42 hours after transfection. Control cells received no antagonist treatment.<br>
Following SCH-23390 treatment for 48 hours, 58% of the cells had cell surface expression of D1-NLS-GFP, less than 10% of the cells had receptor expression in the nucleus and 32% of the cells had receptor expression on both the cell surface and in the nucleus.<br>
Following (+)butaclamol treatment for 48 hours, 62% of the cells had cell surface expression of the D1-NLS-GFP receptor, 10% had receptor expression in the nucleus, and 28% of the cells had receptor expression on the cell surface and in the nucleus.<br>
Control cells at 48 hours showed approximately 65% with D1-NLS-GFP receptor expression in the nucleus, and 35% with receptor expression in the cytoplasm. No receptor D1-NLS-GFP expression was found on the cell surface of control cells.<br>
Incorporation of an NLS into the receptor sequence caused a very efficient removal of the D1-NLS-GFP receptor from the cell surface and localisation in the nucleus.<br>
Similar studies were carried out at various doses of SCH-23390 or (+)butaclamol. Results are shown in Tables 2 and 3. 32% to 35% of control cells showed receptor in cytoplasm.<br>
Table 2<br>
(Table Removed)<br>
Incorporation of an NLS Into the receptor sequence caused a very efficient removal of the D1 dopamine receptor from the cell surface and localisation in the nucleus. Treatment with D1 selective antagonists prevented this receptor translocation in a dose-responsive manner.<br>
Example 4a: Expression of the dopamine D1 receptor with an inserted NLS (D1-NLS-GFP) and treatment with agonists<br>
HEK cells were transfected with a DNA construct encoding D1-NLS-GFP (1.5 micrograms), and incubated with the D1 agonist SKF-81297 (10<br>
micromolar) for 48 hours. At 6,22,30 and 42 hours after transfection, the cells were treated with fresh medium containing SKF-81297 (final concentration 10 micromolar). The cells were examined by confocal microscopy.<br>
HEK cells were transfected with a DNA construct encoding D1-NLS-GFP (1.5 micrograms of DNA), and incubated with the agonist pergolide (10 micromolar) for 48 hours. At 6, 22, 30 and 42 hours after transfection, the cells were treated with fresh medium containing SKF-81297 (final concentration 10 micromolar). The cells were examined by confocal microscopy.<br>
Control HEK cells were transfected with a DNA construct encoding D1-NLS-GFP (1.5 micrograms of DNA) and left untreated.<br>
In the untreated cells after 48 hrs, there was no receptor detected at the cell surface. With cells treated with SKF-81297, 59% of cells had receptor expression at the cell surface. With cells treated with perglolide there was receptor surface expression in 59% of the cells. Thus long-term treatment with agonists prevented the modified D1 receptor from trafficking to the nucleus.<br>
Example 5: Dopamine D1 receptor with an incorporated NLS (D1-NLS-RFP) co-expressed with the wild type D1 receptor<br>
COS cells were cc-transfected with a DNA construct encoding Di-NLS-RFP (1 microgram) and a DNA sequence encoding the native dopamine D1 receptor (D1-WT, 7 microgram) and incubated for 24 or 48 hours.<br>
At 24 hours, D1-NLS-RFP was detected only at the cell surface, whereas at 48 hours, 80% of cells had D1-NLS-RFP in the nucleus. The wild type receptor retarded the movement of the D1-NLS-RFP to the nucleus by homo-oligomerisation.<br>
Example 6: D1 dopamine receptor with an incorporated NLS (D1-NLS-RFP) co-expressed with D1-GFP<br>
COS cells were transfected with a construct encoding D1-NLS-RFP (4 micrograms) and the dopamine D1-GFP (4 micrograms), and incubated for 48 hours. The cells were examined by confocal microscopy.<br>
D1 -GFP was detected at the cell surface and a yellow fluoresence was detected in the nuclei, the latter indicating co-localisation of both D1-NLS-RFP and D1-GFP in the nucleus, confirming oligomerisation of D1-NLS-RFP and D1-GFP, leading to importation of D1-GFP into the nucleus.<br>
Example 6a: Expression of the D1 dopamine receptor with an NLS inserted in the third intracellular cytoplasmic loop (D1-IC3-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding D1-IC3-NLS-GFP (2 micrograms), and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
In cells transfected with D1-IC3-NLS-GFP, the receptor was detected in the nucleus of 85% of cells. Thus insertion of a NLS into the third intracellular loop enabled receptor trafficking to the nucleus.<br>
Example 6b: Expression of the D1 dopamine receptor with an NLS inserted in the first intracellular cytoplasmic loop (D1-IC1-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding D1-IC1-NLS-GFP (2 micrograms), and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
In cells transfected with D1-IC1-NLS-GFP, the receptor was detected in the nucleus of 85% of cells. Thus insertion of a NLS into the first intracellular loop enabled receptor trafficking to the nucleus.<br>
Example 6c: Effect of the antagonist butaclamol or SCH-23390 on the trafficking of the D1 dopamine receptor with an NLS inserted in the first cytoplasmic loop (D1-IC1-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding D1-1C1-NLS-GFP (2 micrograms), and treated with either butaclamol (final concentration 1 micromolar or SCH-23390 (1 micromolar), for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
With the butaclamol treated cells, 82% had receptor on the cell surface or in the cytoplasrn. 18% of the cells had receptor in the nucleus. Thus treatment with butaclamol reduced D1-IC1-NLS-GFP trafficking to the nucleus.<br>
With the SCH-23390 treated cells, 77% of the cells had receptor on the, cells surface or in the cytoplasm. 23% of the cells had receptor in the nucleus. Thus treatment with SCH-23390 reduced receptor trafficking to the nucleus.<br>
With the untreated cells 76% had receptor expression in the nucleus and cytoplasm.<br>
Example 6d: Effect of the antagonist SCH-23390 on the trafficking of the D1 dopamine receptor with an NLS inserted in the third cytoplasmic loop (D1-IC3-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding D1-IC3-NLS-GFP (2 micrograms), and treated with four different concentrations of SCH-23390 (10 micromolar, 1 micromolar, 500 nanomolar and 100 nanomolar), for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
86% of the cells transfected with D1-IC3-NLS-GFP had the receptor in the nucleus, and 0% had receptor on the surface. With the SCH-23390 treated cells, 84% had receptor in the nucleus and 15% of the cells.had receptor on the surface. The insertion of an NLS at this position in the GPCR will translocate the receptor to the nucleus efficiently but does not respond to the drug.<br>
Example 6e: Expression of the D1 dopamine receptor with an NLS inserted in the second intracellular cytoplasmic loop (D1-IC2-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding D1-IC2-NLS-GFP (2 micrograms), and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
In cells transfected with D1-IC2-NLS-GFP, the receptor was detected in the nucleus of 51 % of cells.<br>
Example 6f: Ability of the dopamine D1 receptor to homodimerise, with staggered transfection<br>
HEK cells were transfected with a DNA construct encoding D1-RFP (2 micrograms) and after 24 hours incubation, the cells were transfected with a<br>
second DNA construct encoding D1-NLS-GFP (2 micrograms). Control cells were transfected with the D1-RFP (2 micrograms) construct alone. The cells were incubated for 48 hours following the second transfection and examined by confocal microscopy.<br>
90% of the cells transfected with D1-RFP alone expressed receptor on the cell surface, and 6% of the cells expressed receptor in the nucleus. In contrast, 97% of the cells expressing both forms of the receptor expressed both receptors (red plus green equals yellow fluorescence) in the nucleus. Thus, the D1 receptor without the NLS interacted with the D1 receptor with the NLS in order to traffic to the nucleus.<br>
Example 7: Dopamine D5 receptor (D5-GFP)<br>
A construct encoding the dopamine D5 receptor-GFP (D5-GFP) was prepared and used to transfect COS, cells (4 micrograms).<br>
Cells transfected with dopamine D5-GFP, at 48 hours, showed mainly a cytoplasmic localisation of receptor, with cell surface localisation in only a few cells and no instances of nuclear localisation.<br>
Example 8: Dopamine D1 with an incorporated NLS (D1-NLS) co-expressed with the D5 dopamine receptor (D5-GFP)<br>
HEK cells were transfected with two DNA constructs, one encoding D1-NLS (7 micrograms) and the other encoding D5-GFP (1.5 micrograms), and incubated for 48 hours.<br>
Approximately 70% of the cells transfected with D1-NLS and D5-GFP had cell surface expression of D5-GFP, 20% of the cells had both surface and cytoplasm expression of D5-GFP, and 10% had nuclear expression of D5-GFP. There was no nuclear translocation of the D5 dopamine receptor coexpressed with D1-NLS, indicating that the D1 and D5 receptors did not oligomerise.<br>
Example 9: D1 dopamine receptor containing two NLS motifs (D1-2NLS-RFP) and treated with antagonist<br>
By modifying the construct encoding D1-NLS-RFP, a DNA construct (D1-2NLS-RFP) was created to introduce a second NLS into the carboxyl tail<br>
of the dopamine D1 receptor by replacing the KKEEA sequence of the wild type D1 dopamine receptor with the NLS, KKKRK.<br>
HEK cells were transfected with DNA encoding this construct (D1-2NLS-RFP), and treated at intervals with the antagonist SCH-23390 (10µM) as previously described. At 6,22,30 and 42 hours after transfection, the culture medium containing antagonist was replaced. Control cells received no antagonist.<br>
In both COS and HEK cells transfected with D1-2NLS-RFP, the receptor was located in the nucleus in 100% of cells after 24 hours, indicating enhanced nuclear translocation when a second NLS was present.<br>
At 48 hours, 90% of cells not treated with antagonist showed fluorescence in the nucleus and 0% of cells had fluorescence on the cell surface. Antagonist-treated cells showed 51 % of cells with cell surface label and 49% with nuclear label.<br>
.   Incorporation of a second NLS resulted in a more efficient transport of the receptor to the nucleus, and this event was still retarded by antagonist treatment.<br>
Example 10: D2 dopamine receptor (D2-GFP)<br>
HEK cells were transfected with DNA constructs encoding D2-GFP (2 micrograms) and DsRed-NUC (1 microgram), and incubated for 48 hours. Cells were examined by confocal microscopy.<br>
Approximately 90% of the cells expressing D2-GFP had cell surface expression, and 10% had nuclear or cytoplasm expression. The D2 dopamine receptor, having no endogenous NLS, is predominantly expressed on the cell surface.<br>
Example 11a: Dopamine D1 receptor with an incorporated NLS (D1-NLS) and dopamine D2 (D2-GFP)<br>
HEK cells were transfected with DNA constructs encoding D1-NLS (7 micrograms), and D2-GFP (1.5 micrograms) and incubated for 48 hours. The cells were also transfected with Ds-Red-NUC to verify the localisation of the nucleus (1 microgram). The cells were examined by confocal microscopy.<br>
In cells transfected with D1-NLS and D2-GFP, 33% of the cells had D2-GFP expression in the nucleus, indicating transport of both D1-NLS and D2-GFP to the nucleus, due to oligomerisation between the D1 and D2 receptors. 67% of the cells had D2-GFP receptors on the cell surface only or on surface and cytoplasm.<br>
Example 11b: Ability of D2 dopamine receptor D2 short (D2S) to dimerise with dopamine receptor D2 long (D2L)<br>
HEK cells were transfected with DNA constructs encoding D2S-GFP (2 micrograms) and D2L-NLS (2 micrograms) and were incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
D2S-GFP receptor was visualised in the nuclei of 29% of cells. This indicated that D2S dimerised with D2L and was transported to the nucleus.<br>
Example 11c: Ability of dopamine receptor D2S to dimerise with dopamine receptor D2L.<br>
HEK cells were transfected with DNA constructs encoding D2S-RFP (2 micrograms) and D2L-NLS-GFP (2 micrograms) and incubated for 48 hours. The cells were examined by confocal microscopy.<br>
40% of the cells had a yellow colour (red plus green overlay) in the nucleus, indicating that D2L-NLS dimerised with D2S-RFP and transported it to the nucleus.<br>
Example 12: D2 dopamine receptor with an incorporated NLS (D2-NLS-GFP) treated with antagonists<br>
HEK cells were transfected with DNA encoding D2-NLS-GFP and the cells were treated with the D2 dopamine receptor antagonists, (+)butaclamol (10 micromolar) or raclopride (10 micromolar). At 6,22, 30 and 42 hours after transfection, the cells were treated with the antagonists. Cells were incubated 48 hrs after drug treatment and examined by confocal microscopy.<br>
In the absence of antagonist, cells expressing D2-NLS-GFP showed nuclear label in 70% of cells, and cytoplasmic labelling in 20% and cytoplasmic and surface labelling in 10% of cells. With (+)butaclamol treatment, nuclear labelling appeared in only 5% of cells, 5% of cells had<br>
cytoplasmic label and 90% of the cells had cell surface labelling. With raclopride treatment, 5% of cells showed nuclear labelling, 15% of cells had cytoplasmic labelling and 80% of cells had cell surface labelling. Both antagonists of the D2 receptor prevented the translocation of the receptor off the cell surface and to the nucleus.<br>
Example 13: Beta2-adrenergic receptor-GFP (beta2-AR-GFP)<br>
A DNA construct was created encoding a fusion protein comprising the human beta2-adrenergic receptor and GFP (beta2-AR-GFP). Cells were transfected with the DNA construct encoding beta2-AR-GFP (2 micrograms), and incubated for 24 hours and examined by confocal microscopy.<br>
In cells expressing beta2-AR-GFP, 42% of cells had receptor expression in cytoplasm only, and 58% of the cells had receptor expression in the cytoplasm and on the cell surface. No nuclear localisation of the receptor was observed.<br>
Example 14: Beta2-adrenergic receptor with an incorporated NLS (beta2-AR-NLS3-GFP)<br>
A DNA construct was created encoding a fusion protein comprising the human beta2-AR-NLS3-GFP. HEK cells were transfected with DNA encoding beta2-AR-NLS3-GFP-3 (2 microgram), and Ds-Red-NUC (1 microgram) and the cells were incubated for 48 hours.<br>
45% of the cells transfected with beta2-AR-NLS3-GFP had nuclear localisation of receptor and 55% of the ceils had surface and cytoplasmic expression. Incorporation of a NLS into the beta2-AR induced receptor translocation to the nucleus.<br>
Example 15: Beta2-adrenergic receptor with an incorporated NLS (beta2-AR-NLS3-GFP), treated with antagonist<br>
HEK cells were transfected with DNA encoding beta2-AR-NLS3-GFP-3 (1 microgram), and Ds-Red-NUC (1 microgram) and incubated for 48 hours. Cells were treated at intervals with atenolol (10 micromolar), an adrenergic receptor antagonist At 6,22, 30 and 42 hours after transfection, the culture medium containing the antagonist was replaced.<br>
Control cells received no antagonist. In control cells, 60% had receptor expression in the nucleus, 21% had receptor expression on the cell surface, 19% had receptor expression in the cytoplasm.<br>
In antagonist atenolol-treated cells, 70% had receptor expression on the cell surface, 14% had receptor expression in the nucleus, and 16% had receptor expression in the cytoplasm. Treatment with the antagonist atenolol prevented beta2-AR-NLS3-GFP trafficking to the nucleus and retained the receptor on the cell surface.<br>
Example 16: Beta2-adrenergic receptor (beta2-AR-GFP) coexpressed with dopamine D1 receptor with an incorporated NLS (D1-NLS)<br>
HEK cells were transfected with DNA constructs encoding the beta2-AR-GFP (1.5 microgram) and D1-NLS (3 microgram) for 48 hours.<br>
Approximately 40% of cells showed beta2-AR-GFP receptor expression in the nucleus, demonstrating that the beta2-AR not containing an NLS had trafficked to the nucleus. This indicated that oligomerisation had occurred between the beta2-AR receptor and D1 dopamine receptor, containing the NLS. 45% of the cells showed beta2-AR-GFP in the cytoplasm and 15% in cytoplasm and on cell surface.<br>
Example 17: Beta2-adrenergic receptor (beta2-AR-GFP) and dopamine D1 receptor with an incorporated NLS (D1-NLS) treated with antagonist<br>
HEK cells were transfected with DNA constructs encoding beta2-AR-GFP (1.5 micrograms) and D1-NLS (3 micrograms) and incubated for 48 hours. These cells were treated with the adrenergic antagonist, propranolol (5 micromolar). At 6,22,30 and 42 hours after transfection the culture medium containing the antagonist was replaced. Control cells received no antagonist. The cells were examined by confocal microscopy.<br>
25% of control cells showed beta2-AR-GFP nuclear expression and 75% of cells showed label in cytoplasm and on cell surface.<br>
In propranolol-treated cells, beta2-AR-GFP nuclear expression was 10%, and 90% of the cells showed cytoplasmic and surface label. The formation of a heterooligomer with between beta2-AR-GFP and D1-NLS<br>
resulted in the trafficking of the beta2-AR-GFP to the nucleus. This trafficking was attenuated by the presence of the antagonist to the adrenergic receptor.<br>
Example 18: Beta2-adrenergic receptor with an incorporated NLS (beta2-AR-NLS3-GFP)<br>
HEK cells were transfected with a DNA construct containing an NLS encoding beta2-AR-NLS3-GFP (8 micrograms) for 48 hours. The cells were also transfected Ds-Red-NUC to verify the localisation of the nucleus (1 microgram).<br>
80% of cells showed beta2-AR-NLS3-GFP receptor in the nucleus. The efficiency of the NLS was improved, resulting in a greater localisation of receptor in the nucleus.<br>
Example 19: Serotonin 1B receptor with an incorporated NLS (5HT1B-NLS-GFP) and treatment with antagonist<br>
HEK cells were transfected with a DNA construct encoding the serotonin 5HT1B-NLS-GFP (2 micrograms). The cells were transfected with Ds-red-NUC to verify the localisation of the nucleus (1 microgram). Cells were treated with methysergide (10 micromolar), a serotonin receptor antagonist. At 6,22, 30 and 42 hours after transfection, the culture medium containing antagonist was replaced. Control cells received no antagonist. The cells were examined by confocal microscopy.<br>
Control cells, not treated with antagonist, showed 55% with receptor localised in the nucleus, and 20% with receptor localised on the cell surface. At 48 hours, methysergide-treated cells showed 25% of the cells had receptor in the nucleus and 62% of the cells had cell surface localisation. The serotonin 5HT1B receptor was efficiently translocated from the cell surface to the nucleus by insertion of the NLS. Treatment with the serotonin antagonist methysergide prevented the translocation of the receptor.<br>
Example 20: Cysteinyl leukotriene receptor-2 with an incorporated NLS (CysLT2-NLS-GFP)<br>
HEK cells were transfected for 48 hours with a DNA construct encoding CysLT2-NLS-GFP (8 micrograms). The cells were also transfected with Ds-RED-NUC to verify the localisation of the nucleus (1 microgram). The cells were examined by confocal microscopy.<br>
83% of cells expressing Cys-LT2-NLS-GFP showed receptor expression in the nucleus and 0% of cells had receptor expression on the cell surface, indicating Cys-LT2-NLS-GFP receptor localisation in the nucleus.<br>
Example 21: Cysteinyl leukotriene receptor-2 with an incorporated NLS (Cys-LT2-NLS-GFP) treated with antagonist<br>
The DNA encoding Cys-LT2-NLS-GFP (3 micrograms) was used to transfect HEK cells. These cells were treated with the cysteinyl leukotriene receptor antagonist, montelukast (10 micromolar). At 6,22,30 and 42 hours after transfection the culture medium containing antagonist was replaced. Control cells received no antagonist. The cells were examined by confocal microscopy.<br>
In the absence of antagonist, 70% of cells expressing Cys-LT2-NLS-GFP had localisation of receptor in the nucleus and 30% of cells showed a cytoplasmic localisation with 0% of cells showing receptor on the cell surface. For the antagonist-treated cells, only 10% showed nuclear localisation of the receptor, while 90% showed cell surface expression of receptor. Thus the cysteinyl leukotriene receptor antagonist montelukast prevented the transport of the Cys-LT2-NLS-GFP receptor off the cell surface and into the nucleus.<br>
Example 22: Mu opioid receptor with an incorporated NLS (mu opioid-NLS-GFP)<br>
HEK cells were transfected for 48 hours with a DNA construct encoding the mu opioid-NLS-GFP (2 micrograms). The cells were also transfected with Ds-Red-NUC (1 microgram) to verify the localisation of the nucleus. The cells were examined by confocal microscopy.<br>
65% of the mu opioid-NLS-GFP transfected cells showed receptor expression in the nucleus. 15% of the cells showed cell surface localisation<br>
of receptor and 20% receptor of cells had cytoplasmic labelling. Thus the insertion of the NLS permitted the mu opioid receptor to traffic to the nucleus.<br>
Example 23: Mu opioid receptor with an incorporated NLS (mu-NLS GFP) treated with antagonists<br>
HEK cells were transfected with a DNA construct encoding the mu. opioid-NLS-GFP (2 micrograms). The transfected cells were treated with the mu opioid antagonists, naloxone (10 micromolar) or naltrexone (10 micromolar). At 6, 22,30 and 42 hours after transfection the culture medium containing the antagonist was replaced. Control cells received no antagonist. The cells were examined by confocal microscopy.<br>
When untreated, 62% of cells had Mu-NLS-GFP in the nucleus and 20% of cells had receptor detectable on the cell surface. With naloxone treatment, 21 % of cells had receptor expression in the nucleus and 66% of cells had receptor on the cell surface. With naltrexone treatment, 22% of cells had receptor expression in the nucleus and 58% of cells had receptor on the cell surface. Thus the mu opioid antagonists naloxone and naltrexone reduced receptor translocation off the cell surface and to the nucleus.<br>
Example 24: Muscarinic M1 receptor with an incorporated NLS (M1-NLS-GFP) treated with antagonist<br>
HEK cells were transfected with DNA encoding M1-NLS-GFP (1 microgram), and Ds-Red-NUC (1 microgram) for 48 hours. These cells were treated with ipratropium bromide (10 micromolar). The medium containing antagonist was replaced at 6,22,30 and 42 hours after transfection. Control cells received no antagonist treatment. The cells were examined by confocal microscopy.<br>
Following ipratropium bromide treatment, 72% of the cells had receptor expression on the cell surface, 17% had receptor expression in the cytoplasm only, 11 % of the cells had receptor expression in the nucleus.<br>
For control cells, 64% had receptor expression in the nucleus, 23% had receptor expression on the cell surface and 13% of the cells had receptor expression in cytoplasm.<br>
Treatment with a muscarinic antagonist prevented the M1-NLS-GFP from translocating off the cell surface and trafficking to the nucleus.<br>
Example 25: Histamine H1 receptor with an incorporated NLS (H1-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding the histamine H1-NLS-GFP receptor (2 micrograms), and a construct encoding Ds-Red-NUC (1 microgram) for and incubated for 48 hours.<br>
Approximately 65% of the cells had receptor expression in the nucleus, and 35% of the cells had receptor expression on both surface and cytoplasm. Insertion of the NLS into the H1 histamine receptor resulted in translocation of the receptor off the surface and to the nucleus.<br>
Example 26: Effect of the antagonist promethazine on the trafficking of the H1 histamine receptor with an NLS inserted (H1-NLS-GFP)<br>
HEK cells were transfected with H1-NLS-GFP (2 micrograms) and DsRED- Nuc (2 micrograms) and incubated for 48 hours. The cells were treated with promethazine (10 micromolar) for 48 hours. Nucleii were visualised with DsRED- Nuc. The cells were examined by confocal microscopy.<br>
With the promethazine treated cells 88% of the cells had receptor on the cell surface, 10% of the cells had receptor in the nucleus. With the untreated cells 85% had receptor expression in the nucleus and cytoplasm. Thus treatment with promethazine reduced H1-NLS-GFP trafficking to the nucleus.<br>
Example 26: Angiotensin AT1 receptor (AT1R)<br>
A DNA construct (AT1R-RFP) was created encoding a fusion protein comprising the NLS-containing human angiotensin AT1 receptor and DsRed2 (RFP).<br>
COS cells were transfected with the DNA construct AT1 R-RFP (4 micrograms) and incubated for 48 hours at 37°C.<br>
Cells were examined by confocal microscopy and the receptor was found to be located exclusively within the nuclei of the cells, indicating a basal agonist-independent translocation of the AT1R into the nucleus.<br>
Example 27: Dopamine receptor (D1-NLS-GFP) treated with agonist for a short term<br>
HEK cells were transfected with the DNA constructs encoding D1-NLS-GFP (2 micrograms), and D1-WT (4 micrograms), and the cells incubated for 24 hours. The cells were treated with the dopamine D1 agonist, SKF 81297 (10 micromolar) for 35 mins. A single group of cells were visualised by confocal microscopy in real time.<br>
An increased expression of the receptor in the nucleus was demonstrated, with a maximum increase occurring at 20 minutes, indicating short term agonist effect.<br>
Example 28: Dopamine transporter with a NLS, fused to GFP and RFP (DAT-NLS-GFP and DAT-NLS-RFP)<br>
HEK cells were transfected with a DNA construct encoding DAT-GFP (2 micrograms) for 48 hours. Nuclei were visualised with DsRED-nuc (2 micrograms) using confocal microscopy.<br>
At 48 hours, DAT-GFP was detected on the cell surface or in the cytoplasm in 86% of the cells. In 14% of the cells, the transporter was in the nucleus.<br>
HEK cells were transfected with a construct encoding DAT-NLS-RFP (2 micrograms) and visualised by confocal microscopy at 48 hours. DAT-NLS-RFP was detected in the nuclei in 85% of the cells. In 18% of the cells, the transporter was either at the surface or in the cytoplasm.<br>
' HEK cells were then transfected with DNA encoding DAT-NLS-GFP (2 micrograms) and visualised by confocal microscopy at 48 hours. Nuclei were visualised with DsRED-nuc (2 micrograms). DAT-NLS-GFP was detected in the nucleus of 77% of cells.<br>
Example 29: Co-trafficking of DAT-GFP with DAT-NLS-RFP<br>
HEK cells were transfected with DAW constructs encoding DAT-NLS-RFP (2 micrograms) and DAT-GFP (2 micrograms), and incubated for 48 hours. The cells were examined by confocal microscopy.<br>
A yellow fluorescence was detected in the nuclei in 56% of the cells, indicating co-localisation of DAT-NLS-RFP and DAT-GFP in the nucleus, confirming oligomerisation of DAT-NLS-RFP and DAT-GFP.<br>
Example 30: Effect of cocaine on DAT-NLS-RFP trafficking to the nucleus<br>
HEK cells were transfected with a DNA construct encoding DAT-NLS-RFP (2 micrograms) and incubated for 48 hours. At 6, 22, 30 and 42 hours after transfection, the cells were treated with cocaine or amphetamine (final concentration 10 micromolar), or left untreated. The cells were examined by confocal microscopy.<br>
In the non-treated HEK cells, 77% of the cells had DAT-NLS-RFP expression in the nucleus.<br>
Following cocaine treatment, 75% of the cells had cell surface or cytoplasmic expression of DAT-NLS-RFP, whereas 25% of the cells had transporter expression in the nucleus and cytoplasm. Treatment with cocaine reduced the trafficking of the DAT-NLS-RFP to the nucleus.<br>
Following amphetamine treatment, 34% of the cells had cell surface/cytoplasm expression, and 66% of the cells had transporter expression in the nucleus/cytoplasm. Treatment with an amphetamine (which does not target DAT but targets the vesicular monoamine transporter, VMAT) had no inhibitory effect on the trafficking of the DAT-NLS-RFP to the nucleus.<br>
Example 31: Expression of the dopamine transporter with a NLS (DAT-NLS-GFP) and treatment with antagonists<br>
HEK cells were transfected with a DNA construct encoding DAT-NLS-GFP (2 micrograms) and incubated for 48 hours. At 6, 22, 30 and 42 hours after transfection, the cells were treated with GBR-12909 (final concentration 1 micromolar). The cells were examined by confocal microscopy.<br>
HEK cells were transfected with a construct encoding DAT-NLS-GFP (2 micrograms) and incubated for 48 hours. At 6, 22, 30 and 42 hours after<br>
transfection cells were treated with mazindol (final concentration 1 micromolar). The cells were examined by confocal microscopy.<br>
Control HEK cells were transfected with DAT-NLS-GFP incubated for 48 hours and not treated with drug.<br>
In the untreated HEK cells transfected with DAT-NLS-GFP, 77% of the cells had transporter expression in the nucleus and cytoplasm, 23% in the cytoplasm only, and 0% on the cell surface.<br>
Following GBR-12909 treatment, 62% of the cells had transporter expression on the cell surface and in cytoplasm, and 38% of the cells had transporter expression in the nucleus and cytoplasm. Treatment with GBR-12909 reduced DAT-NLS-GFP translocation off the cell surface and trafficking to the nucleus.<br>
Following mazindol treatment 61% of the cells had cell surface and cytoplasm expression of transporter, and 39% of the cells had transporter expression in the nucleus and cytoplasm. Treatment with mazindol reduced the DAT-NLS-GFP translocation of the cell surface and trafficking to the nucleus.<br>
Example 32: Ability of dopamine transporter to homooligomerise, using staggered expression of DAT-GFP and DAT-NLS-RFP<br>
HEK cells were transfected with the DNA construct encoding DAT-GFP (2 micrograms) and 24 hrs later with the DNA construct encoding DAT-NLS-RFP (0.5, 1, and 2 micrograms) and incubated for 48 hours. Cells were also transfected with DAT-GFP alone as control. The cells were incubated 48 hours after the second transfection. Total incubation period was 72 hours.<br>
85% of the cells transfected with DAT-GFP alone contained transporter in the cytoplasm, and 7% in the nucleus. In the staggered experiment (ratio 1:0.5), 97% of the cells had yellow (= red + green) fluorescence in the nucleus. In the staggered experiment (ratio 1:1), 94% of the cells had yellow fluorescence in the nucleus. In the staggered experiment (ratio 1:2), 94% of the cells had yellow fluorescence In the nucleus. Therefore the DAT-GFP interacted with and dimerised with DAT-NLS-RFP in order to traffic to the nucleus.<br>
Example 33: Expression of the metabotropic glutamate-4-receptor (mGluR4-GFP)<br>
HEK cells were transfected with a DNA construct encoding mGluR4-GFP (2 micrograms) and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
89% of the receptors were expressed at the cell surface. Thus the mGluR4 receptor was largely located at the cell surface.<br>
Example 34: Expression of the metabotropic glutamate-4 receptor with an inserted NLS (mGluR4-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding mGluR4-NLS-GFP (2 micrograms), and incubated for 48 hours. The cells were also transfected with Ds-RED-NUC (2 micrograms) to verify the localisation of the nucleus. The cells were examined by confocal microscopy.<br>
60% of ceils expressing mGluR4-NLS-GFP showed expression of receptor in the nucleus.<br>
Thus the insertion of an NLS into the mGluR4 receptor increased the nuclear localisation of the receptor.<br>
Example 35: Expression of the muscarinic M1 receptor with or without NLS incorporation (M1 -GFP and M1 -NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding the M1-GFP (2 micrograms) or with a construct encoding the M1-NLS-GFP (2 micrograms) and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
After transfection with M1-GFP, 67% of the cells had receptor expressed on the cell surface or in the cytoplasm.<br>
Transfection with M1-NLS-GFP showed 92% of the cells with nuclear expression of the receptor, indicating that the NLS directed the receptor to the nucleus.<br>
Example 36: Expression of the H1 histamine receptor (H1-GFP)<br>
HEK cells were transfected with a DNA construct encoding H1-GFP (1.5 micrograms) and incubated for 48 hours. Nuclei were visualised with DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
97% of the cells expressed receptor at the cell surface. Thus, the unmodified receptor did not traffic to the nucleus.<br>
Example 37: Expression of the Cysteinyl leukotriene receptor with NLS inserted (CysLT1-NLS-GFP)<br>
HEK cells were transfected with a DNA construct encoding CysLT1-NLS-GFP (2 micrograms) and incubated for 48 hours. Nuclei were visualised with^ DsRED-NUC (2 micrograms). The cells were examined by confocal microscopy.<br>
With the control untreated cells, 0% of the cells had receptor expression on the cell surface, and 100% of the cells had nuclear expression, indicating robust removal of the receptor off the cell surface.<br>
Example 38: Expression of the serotonin transporter fused to GFP (SERT-GFP)<br>
HEK cells were transfected with a DNA construct encoding SERT-GFP (2 micrograms) and incubated for 48 hours. 91% of the cells expressed transporter on the cell surface and cytoplasm.<br>
Example 39: Expression of the serotonin transporter with an inserted NLS and treatment with fluoxetine (SERT-NLS-GFP)<br>
HEK cells were transfected with DNA encoding SERT-NLS-GFP (2 micrograms of DNA) and treated with fluoxetine (final concentrationl micromolar) for 48 hours. At 6,22,30 and 42 hours after transfection, the cells were treated with fluoxetine (final concentration 1 micromolar). The cells were examined by confocal microscopy.<br>
In the untreated cells expressing SERT-NLS-GFP, 0% of the cells had transporter expression on the cell surface, 26% had transporter expression in the cytoplasm, and 60% of the cells had transporter expression in the nucleus and cytoplasm.<br>
Following fluoxetine treatment, 68% of the cells had SERT-NLS-GFP transporter expression on the cell surface and cytoplasm, and 27% of the cells had transporter expression in the nucleus and cytoplasm. Thus treatment with fluoxetine inhibited the SERT-NLS-GFP from translocating off the cell surface and trafficking to the nucleus.<br>
Example 40: Evaluation of the ability of two different cell surface membrane proteins to interact with each other (D2-GFP and DAT-NLS-RFP)<br>
HEK cells were cotransfected with DNA constructs encoding the D2-GFP (2 micrograms) and DAT-NLS-RFP (2 micrograms) and incubated for 48 hours. Cells were also transfected separately with D2-GFP and DAT-NLS-RFP alone as controls. The cells were examined by confocal microscopy.<br>
85% of the cells transfected with DAT-NLS-RFP contained transporter in the nucleus. 97% of the cells transfected with D2-GFP contained the receptor on the cell surface, and 4% of the cells contained receptor in the nucleus. 86% of the cotransfected cells contained yellow (red plus green) fluorescence in the nucleus, . indicating the presence of both D2 and DAT proteins in the nucleus and confirming dimerisation of the co-expressed proteins.<br>
Example 41: Evaluation of the ability of a membrane protein and a non-membrane protein, to associate in a complex and interact with each other (D1-NLS and beta-arrestin1-GFP)<br>
HEK cells were co-transfected with DNA constructs encoding D1-NLS (2 micrograms) and beta-arrestin1-GFP (2 micrograms) and incubated for 48 hours. Cells were also transfected with beta-arrestin1-GFP alone. The cells were checked by confocal microscopy.<br>
100% of cells transfected with beta-arrestinl-GFP alone expressed fluorescent protein in the cytoplasm. Of these cells 15% also had fluorescence in the nucleus. 89% of cells co-transfected with both proteins expressed fluorescent protein in the nucleus and of these 16% had expression in the cytoplasm. Thus, the interaction between the GPCR and the non-membrane protein enabled the trafficking of the non-NLS containing beta-arrestin protein to the nucleus.<br>
Example 42: Expression of the dopamine D1 receptor with an inserted NLS (HA-D1 -NLS) treatment with antagonist and detection with fluorometry<br>
Wells in a multi-well plate were coated with poly-L-ornithine and then plated with 50,000 cells per well. The cells were transfected with DNA encoding an HA epitope tagged D1-NLS receptor and treated with (+)butaclamol (10 nanomolar to 10 micromolar) over 48 hours. Following this, the cells were fixed with paraformaldehyde, and cell surface receptors were detected with a rat anti-HA antibody and then a goat anti-rat antibody conjugated to FITC. The fluorescent signal was detected by fluorometry (Cytofluor). The results are the average of five<br>
wells per experimental condition and are shown in Figure 2. There was a dose-dependent effect of butaclamol to retain receptor on the cell surface, indicating that this antagonist reduced receptor trafficking from the cell surface. Thus fluorometry can be utilised to detect receptor retained at the cell surface.<br>
Example 43: Expression of the dopamine D1 receptor with an inserted NLS (HA-D1-NLS), and blockade of antagonist dose-response effect by agonist and detection with fluorometry<br>
Wells in a multi-well plate were coated with poly-L-ornithine and then plated with 50,000 cells per well. The cells were transfected with DNA encoding an HA epitope tagged D1-NLS receptor and treated with the antagonist SCH 23390 (1 nanomolar to 1 micromolar) with or without the agonist SKF 81297 (1 micromolar) over 48 hours. Following this, the cells were fixed with paraformaldehyde, and cell surface receptors were detected with a rat anti-HA antibody and then a goat anti-rat antibody conjugated to FITC. The fluorescent signal was detected by fluorometry (Cytofluor). The results are the average of five wells per experimental condition. There was a dose-dependent effect of SCH 23390 to retain receptor on the cell surface, indicating that this antagonist reduced receptor trafficking from the cell surface. The concomitant addition of agonist reduced the antagonist effect (Figure 3). Thus agonist action can be detected by blockade of antagonist effect and fluorometry can be utilised to quantify the agonist effect.<br>
Example 43b: Expression of the dopamine D1 receptor with an inserted NLS (HA-D1 -NLS), and blockade of antagonist effect by agonist dose-response and detection with fluorometry.<br>
HEK cells were transfected with HA-D1-NLS in a multi-well plate were coated with poly-L-ornithine at a concentration of 50,000 cells per well. The cells were treated with the antagonist SCH 23390 (0.5 micromolar) for 48 hrs. The agonist SKF 81297 (100 nanomolar to 1 micromolar) together with SCH 23390 was added for the last hour of incubation. Following this, the cells were fixed with paraformaldehyde, and cell surface receptors were detected with a rat anti-HA antibody and then a goat anti-rat antibody conjugated to FITC. The fluorescent signal was detected by fluorometry (Cytofluor). The results are the average of five wells per experimental condition.<br>
Treatment with SCH 23390 retained HA-D1-NLS on the cell surface (Figure 4, Bar 1 vs. Bar 6). Short-term addition of the agonist resulted in a dose-dependent blockade of SCH 23390 effect. Removal of SCH 23390 from the cells for the last hour of incubation (and in the absence of agonist) resulted in a 33% loss of HA-D1-NLS receptors from the cell surface (Figure 4, Bar 5 vs. Bar 6), whereas addition of agonist SKF 81297 100 nanomolar in the continued presence of SCH 23390 resulted in a 66% loss of receptors from the cell surface (Figure 4, Bar 4 vs. Bar 6), up to a 78% loss of receptors with addition of SKF 81297 1 micromolar (Figure 4, Bar 2 vs. Bar 6).<br>
The effect of the antagonist SCH 23390 resulted in retention of receptor on the cell surface, indicating that this antagonist reduced receptor trafficking from the cell surface. The concomitant addition of agonist reduced the antagonist effect and accelerated the removal of the receptor from the cell surface in a dose-responsive manner. Thus interacting compounds can be detected by blockade of the effect of compounds that retain the NLS-containing receptor at the cell surface and fluorometry can be utilized to quantify the effect.<br>
Example 44: Expression of the dopamine D1 receptor with an inserted (D1-NLS), treatment with (+)butaclamol 10 micromolar and detection with radioligand binding<br>
HEK cells were transfected with DNA encoding D1-NLS and treated with (+)butaclamol (10 micromolar) or left untreated. After 48 hours, the cells were washed, harvested, lysed and homogenised by polytron. The membrane fraction was collected by centrifugation and then layered over 35% sucrose solution and centrifuged at 30,000 rpm at 4 degrees C for 90 min to collect the heavy membrane fraction.<br>
The membrane fractions were subjected to radioligand binding assay using [3H]-SCH23390 with (+)butaclamol (10 micromolar) used to define specific binding. The incubation was at room temperature for 2 hours, followed by rapid filtration and quantitation by scintillation counting.<br>
Antagonist treatment of D1-NLS prevented its translocation off the cell surface and to the nucleus and the receptor retained on the cell surface was quantified by radioligand binding assay (Figure 5).<br><br>
Example 45: Expression of the dopamine D1 receptor with an inserted NLS (D1-NLS), treatment with (+)butaclamol 500 nanomolar and detection with radioligand binding<br>
HEK cells were transfected with DNA encoding D1-NLS and treated with (+)butaclamol 500 nanomoiar or left untreated. After 48 hours, the cells were washed, harvested, lysed and homogenised by polytron. The membrane fraction was collected by centrifugation and then layered over 35% sucrose solution and centrifuged at 30,000 rpm at 4 degrees C for 90 min to collect the heavy membrane fraction.<br>
The membranes were subjected to radioligand binding assay using [3H]-SCH23390 with (+)butaclamol 10 micromolar used to define specific binding. The incubation was at room temperature for 2 hours, followed by rapid filtration and quantitation by scintillation counting. Results are shown in Tables 4 and 5.<br>
Table 4: Control plasma membrane fraction<br>
(Table Removed)<br>
Table 5: Butaclamol treated plasma membrane fraction<br>
(Table Removed)<br>
Antagonist treatment with (+)butaclamol of D1-NLS prevented its translocation to the nucleus and the receptor retained on the cell surface was quantified by radioligand binding assay.<br>
Example 46: Expression of the dopamine D1 receptor with an inserted NLS (D1-NLS), treatment with (+)butaclamol 100 nanomolar and detection with radioligand binding<br>
HEK cells were transfected with DNA encoding D1-NLS and treated with (+)butaclamol 100 nanomolar or left untreated. After 48 hours, the cells were washed, harvested, lysed and homogenised by polytron. The membrane fraction was collected by centrifugation and then layered over 35% sucrose solution and centrifuged at 30,000 rpm at 4 degrees C for 90 min to collect the heavy membrane fraction.<br>
The membranes were subjected to radioligand binding assay using [3H]-SCH23390 with (+)butaclamol (10 micromolar) to define specific binding. The incubation was at room temperature for 2 hours, followed by rapid filtration and quantitation by scintillation counting.<br>
Antagonist treatment with (+)butaclamol (100 nanomolar) prevented D1-NLS translocation to the nucleus and the receptor retained on the cell surface was quantified by radioligand binding assay. In the absence of butaclamol treatment, 0.03 pmol/mg protein of receptor was detected in the cell surface membranes, and with butaclamol treatment, 0.09 pmol/mg protein of receptor was detected in the cell surface membranes.<br>
Example 47: Expression of the epidermal growth factor receptor (tyrosine kinase receptor) EGFR-GFP and EGFR-NLS-GFP.<br>
HEK cells were transfected with DNA encoding EGFR-NLS-GFP (2 micrograms). HEK cells were also transfected with DNA encoding EGFR-GFP (2 micrograms) and incubated for 24 hours.<br>
EGFR-GFP was expressed on the cell surface in 73% of cells and 12% of cells had receptor in the nucleus. EGFR-NLS-GFP was expressed in the nucleus in 91% of cells and 0% of cells had receptor on the cell surface. The incorporation of a NLS into the sequence of the EGF receptor induced robust translocation off the cell surface and into the nucleus.<br>
Example 48: Expression of the low density lipoprotein receptor (LDL-GFP)<br>
HEK cells were transfected with DNA encoding the LDL-GFP (2 micrograms) and incubated for 24 hours. The receptor was expressed on the cell surface in 67% of cells and in the nucleus in 8% of cells.<br>
The LDL receptor is expressed on the ceJJ surface in the majority of cells with not many cells containing receptor in the nucleus.<br>
Example 49: Expression of the LDL receptor with a NLS (LDL-NLS-GFP)<br>
HEK cells were transfected with DNA encoding LDL-NLS-GFP (2 micrograms), and DsRED-NUC (2 micrograms), and incubated for 48 hours. Cells were examined by confocal microscopy.<br>
LDL-NLS-GFP was expressed in the nucleus in 22% of cells, and on the cell surface in 67%. The incorporation of a NLS into the LDL receptor induced receptor translocation into the nucleus.<br>
Example 50: Expression of the erythropoietin receptor (cytokine receptor) EPO-GFP and EPO-NLS-GFP<br>
HEK cells were transfected with DNA encoding EPO-NLS-GFP (2 micrograms). HEK cells were transfected with EPO-GPF (2 micrograms). The cells were also transfected with DsRed-NUC (2 micrograms). The cells were incubated for 48 hours and were examined by confocal microscopy.<br>
The EPO-NLS-GFP was located in the nucleus of 72% of cells and on the cell surface in 0% of cells. The EPO-GFP was located on the cell surface in 79%} of cells and 28% of cells had receptor expression in the nucleus. The incorporation of a NLS into the sequence of the EPO receptor induced translocation off the cell surface and into the nucleus.<br>
Example 51: Expression of the serotonin transporter with a NLS (SERT-NLS-GFP) and treatment with sertraline<br>
HEK cells were transfected with DNA encoding SERT-NLS-GFP (2 micrograms of DNA) and treated with sertraline (final concentration 500 nanomolar) for 48 hours. At 6,22,30 and 42 hours after transfection, the cells were treated with sertraline. The cells were examined by confocal microscopy.<br>
In the untreated cells expressing SERT-NLS-GFP, 0% of the cells had transporter expression on the cell surface and 75% of the cells had transporter expression in the nucleus and cytoplasm.<br>
Following sertraline treatment, 69% of the cells had SERT-NLS-GFP transporter expression on the cell surface and cytoplasm, and 21% of the cells had transporter expression in the nucleus and cytoplasm. Thus treatment with sertraline inhibited the SERT-NLS-GFP from translocating off the cell surface and trafficking to the nucleus.<br>
Example 52: Expression of D1 dopamine receptor with an alternate NLS (D1-NLS2-GFP) and treatment with antagonists<br>
HEK cells were transfected with DNA encoding D1-NLS2-GFP (2 micrograms) and treated with (+)butaclamol or SCH 23390 (1 micromolar) for 48 hrs. Nuclei were visualised with DsRed-nuc (2 micrograms). Cells were examined by confocal microscopy.<br>
With butaclamol treatment, 81% of cells had receptor on the cell surface or in cytoplasm and 19% of cells had receptor expression in the nucleus. With SCH 23390 treatment, 78% of cells had receptor on the cell surface or in cytoplasm and 22% of cells had receptor expression in the nucleus<br>
In the untreated cells, 89% of cells had receptor expression in the nucleus and cytoplasm.<br>
Thus treatment with the dopamine D1 antagonists prevented D1-NLS2-GFP receptor translocation off the surface and trafficking to or toward the nucleus.<br>
The present invention is not limited to the features of the embodiments described herein, but includes all variations and modifications within the scope of the claims.<br>
REFERENCES:<br>
Bailey et al., (2001), Expert Opinion in Therapeutics, v.11, p. 1861-1887;.<br>
Barak et al., (1997), J. Biol. Chem., v.272, p. 27497-27500;<br>
Chen et al., (2000), Am. J. Physiol. Renal Physiol., v.279, F440-F448;<br>
George et al., (2000), J. Biol. Chem., v.275, p. 26128-26135;<br>
George et al., (2002), Nature Reviews (2002), v.1, p. 808-820;<br>
Gorlich et al., (1996), Science, v.271, p. 1513-1518;<br>
Grotzinger, (2002), Biochim. Biophys. Acta, v. 1592, p. 215-223;<br>
Hailey et al., (2002), Methods in Enzymology, v. 351, p.34-41;<br>
Hanahan et al., (1996), Cell, v. 86, p. 353-364;<br>
Howard et al., (2001), Trends in Pharmacol. Sci., v.22, p.132-140;<br>
Jans et al, (2000), Bioessays v. 22:532-544;<br>
Lee et al., (2002), Expert Opinion in Therapeutic Targets, v. 6, p. 185-202;<br>
Lu et al., (1998), Endocrinol., v.139, p. 365-375;<br>
Masson et al., (1999), Pharmacol Reviews, 51:439-464;<br>
Matz et al., (1999), Nature Biotech., v.17, p. 969-973;<br>
Nakae et al., (2001), Endo. Reviews., v. 22, p 818-835;<br>
Nicholson et al., (2001), Eur. J. Cancer, v. 37, Suppl. 4, p. S9-S15;<br>
Prasher et al., (1992), Gene, v.111, p.129;<br>
Schlenstedt et al., (1996), FEBS Lett., v.389, p. 75-79;<br>
Shawver et al., (2002), Cancer Cell, v. 1, p. 117-123;<br>
Smith, (2002), Nature v. 418, p. 453-459;<br>
Strickland et al., (2002) Trends Endo. Metab., v. 13, p 66-74;<br>
Watson et al, (2000), Bone v. 26, p. 221-225;<br>
Weis et al., (1998), Trends in Biochem., v. 23, p. 185-189;<br>
White et al., Nature, v. 396, p. 679-682 (1998);<br>
TABLE 1    EXAMPLES OF NUCLEAR LOCALISATION SEQUENCES<br>
(adapted from Jans et al. 2002)<br>
(Table Removed)<br>
RAG-1, recombination activating protein 1; RCP, red cell protein; RB, Retinoblastoma protein; STAT, signal transducer and activator of transcription (TF); CBP80, Cap-binding protein; LEF, Lymphocyte enhancer factor; EBNA, Epstein-Barr virus nuclear antigen; IN, HIV-1 integrase; tTG, tissue transglutaminase; ICP, Infected cell protein.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
We claim:<br>
1.	An in vitro method for screening a candidate compound for its ability to<br>
interact with at least one transmembrane protein comprising:<br>
contacting a cell with a candidate compound wherein the cell comprises at least one nucleotide sequence encoding a protein comprising a transmembrane protein containing at least one nuclear localisation sequence (NLS) wherein the wild type transmembrane protein lacks an NLS and the nucleotide sequence encoding the transmembrane protein is modified to encode an NLS and the encoded protein is expressed in the cell; and<br>
determining the distribution of the expressed protein in the cell;<br>
wherein detection of an altered distribution of the transmembrane protein in the cell relative to the distribution of the transmembrane protein in a control cell not contacted with the candidate compound indicates that the compound interacts with the transmembrane protein.<br>
2.	The method as claimed in claim 1, wherein the protein comprises a detectable moiety and the distribution of the expressed protein is determined by detecting the distribution of the detectable moiety in the cell wherein detection of an altered distribution of the detectable moiety in the cell relative to the distribution of the detectable moiety in a control cell not contacted with the candidate compound indicates that the compound interacts with the transmembrane protein.<br>
3.	The method as claimed in claim 2, wherein the detectable moiety is a detectable peptide comprising an antigenic portion of the amino acid sequence of the transmembrane protein.<br>
4.	The method as claimed in claim 2, wherein the nucleotide sequence encodes a fusion protein comprising a transmembrane protein containing at least one NLS and a detectable moiety.<br><br>
5.	The method as claimed in any one of claims 1 to 4, wherein the nucleotide<br>
sequence is modified to encode an NLS selected from PKKKRKV, RRR-(11 aa)-<br>
KRRK, KK-(15 aa)-KKRK, KRKRRP, PKKNRLRRK, KRQR-(20 aa)-KKSKK,<br>
PAAKRVKLD, QRKRQK, HRIEEKRKRTYETFKSI, KKKYKLK, KSKKKAQ,<br>
PKKKRKV, QRKRQK, KK-(15 aa)-KKRK, KKKKRKREK, LKRPRSPSS, KRK-(22<br>
aa)-KELQKQITK, GKKKYKLKH, KKKYKLK, KSKKKAQ, EED-(350 aa)-<br>
KKKRERLD, PKKKRKV, CYFQKKAANMLQQSGSKNTGAKKRK, DILRR-(323<br>
aa)-PKQKRK, PKKKRKV, KKKKRKREK, SSDDEATADSQHSTPPKKKRKV,<br>
PKKKRKV, RKKRK-(9 aa)-KAKKSK, KR-(11 aa)-KKLR, RRPS-(22 aa)-RRKRQK,<br>
RRR-(11 aa)-KRRK, KK-(15 aa)-KKRK, KKKKRKREK,<br>
SSDDEATADSQHSTPPKKKRKV, PKKKRKV, RKKRK-(9 aa)-KAKKSK, KR-(11<br>
aa)-KKLR, RRPS-(22 aa)-RRKRQK, RRR-(11 aa)-KRRK, KK-(15 aa)-KKRK,<br>
KKKKRKREK,	PKKKRKV,	KR-(10	aa)-KKKL,	PKKKRKV,<br>
SSDDEATADSQHSTPPKKKRKV,	RKKRK-(9  aa)-KAKKSK,   KRK-(22  aa)-<br>
KELQKQITK, PKKKRKV, SSDDEATADSQHSTPPKKKRKV, RKRK-(7 aa)-RRSRYRK, MISEALRKA, RKKRK-(9 aa)-KAKKSK, recombination activating protein 1, red cell protein, Retinoblastoma protein, signal transducer and activator of transcription (TF), Cap-binding protein, Lymphocyte enhancer factor, Epstein-Barr virus nuclear antigen, HIV-1 integrase, tissue transglutaminase, and Infected cell protein.<br>
6.	The method as claimed in any one of claims 1 to 4, wherein the nucleotide sequence is modified to encode an amino acid sequence selected from the group consisting of KKFKR, PKKKRKV and AFSAKKFKR.<br>
7.	The method as claimed in any one of claims 1 to 6, wherein the cell is a eukaryotic cell or a prokaryotic cell.<br>
8.	The method as claimed in claim 7, wherein the cell is a eukaryotic cell selected from the group consisting of a mammalian cell, a yeast cell, an insect cell, a nematode cell, a plant cell and a fungal cell.<br>
9.	The method as claimed in claim 8, wherein the mammalian cell selected from the group consisting of HEK, COS and CHO cells.<br><br>
10.	The method as claimed in claim 2, wherein the detectable moiety is an antigenic peptide and the distribution of the antigenic peptide in the cell is determined by allowing it to bind to an antibody-based detection system comprising an antibody specific for the antigenic peptide.<br>
11.	The method as claimed in claim 10, wherein the antibody-based detection system comprises a first antibody specific for the antigenic peptide and a second antibody carrying a detectable label and specific for the first antibody.<br>
12.	The method as claimed in claim 10, wherein the antibody-based detection system comprises a first antibody specific for the antigenic peptide and carrying a detectable label.<br>
13.	The method as claimed in claim 11 or 12, wherein the detectable label is an optically detectable label.<br>
14.	The method as claimed in claim 13, wherein the detectable label is a luminescent label or a fluorescent label.<br>
15.	The method as claimed in claim 2, wherein the detectable moiety is a polypeptide selected from the group consisting of green fluorescent protein, red fluorescent protein and modified variants thereof.<br>
16.	The method as claimed in any one of claims 1 to 15, wherein the transmembrane protein is selected from the group consisting of a G protein coupled receptor (GPCR), a transporter, a cytokine receptor, a tyrosine kinase receptor and a low density lipoprotein (LDL) receptor.<br>
17.	The method as claimed in claim 16, wherein the transmembrane protein is a GPCR.<br>
18.	The method as claimed in claim 17, wherein the GPCR is selected from the group consisting of dopamine D1 receptor, dopamine D2 receptor, dopamine D3 receptor, dopamine D5 receptor, histamine 1 receptor, cysteinyl leukotriene receptor 1, cysteinyl leukotriene receptor 2, opioid receptor, muscarinic receptor,<br><br>
serotonin receptor, beta2-adrenergic receptor and metabotropic glutamate 4 receptor.<br>
19.	The method as claimed in claim 16, wherein the transmembrane protein is a transporter.<br>
20.	The method as claimed in claim 19, wherein the transporter is selected from the group consisting of dopamine transporter and serotonin transporter.<br>
21.	The method as claimed in claim 16, wherein the transmembrane protein is a cytokine receptor.<br>
22.	The method as claimed in claim 21, wherein the cytokine receptor is selected from the group consisting of erythropoietin receptor and insulin receptor.<br>
23.	The method as claimed in claim 16, wherein the transmembrane protein is a tyrosine kinase receptor.<br>
24.	The method as claimed in claim 23, wherein the tyrosine kinase receptor is selected from the group consisting of epidermal growth factor receptor and insulin receptor.<br>
25.	The method as claimed in claim 16, wherein the transmembrane protein is a low density lipoprotein receptor.<br>
26.	The method as claimed in claim 2, wherein the cell comprises a plurality of nucleotide sequences, each of said sequences encoding a protein comprising a different transmembrane protein containing at least one NLS.<br>
27.	The method as claimed in claim 26, wherein each of said nucleotide sequences encodes a protein comprising a different detectable moiety.<br>
28.	The method as claimed in claim 26, wherein at least one detectable moiety is common to at least two encoded proteins.<br><br>
29.	The method as claimed in claim 1, wherein the cell is contacted with a compound known to interact with the at least one transmembrane protein prior to contacting the cell with the candidate compound and wherein detection of an altered distribution of the transmembrane protein in the cell relative to the distribution of the transmembrane protein in a control cell contacted with the compound known to interact with the transmembrane protein but not contacted with the candidate compound indicates that the candidate compound interacts with the transmembrane protein.<br>
30.	The method as claimed in any one of claims 1 to 29, wherein detection of an altered distribution of the transmembrane protein in the cell comprises detection of an altered level of the transmembrane protein associated with the cell membrane.<br>
31.	The method as claimed in claim 30, wherein detection of an altered distribution of the transmembrane protein comprises detection of a reduced level of the transmembrane protein associated with the cell membrane.<br>
32.	The method as claimed in claim 30, wherein detection of an altered distribution of the transmembrane protein comprises detection of an increased level of the transmembrane protein associated with the cell membrane.<br>
33.	The method as claimed in any one of claims 1 to 30, wherein detection of an altered distribution of the transmembrane protein in the cell comprises detection of an altered level of the transmembrane protein in the nucleus of the cell.<br>
34.	The method as claimed in claim 33, wherein detection of an altered distribution of the transmembrane protein comprises detection of a reduced level of the transmembrane protein in the nucleus of the cell.<br>
35.	The method as claimed in claim 33, wherein detection of an altered distribution of the transmembrane protein comprises detection of an increased level of the transmembrane protein in the nucleus of the cell.<br><br>
36.	The method as claimed in claim 1, wherein the distribution of the expressed<br>
protein is determined by determining the level of NLS-containing transmembrane<br>
protein remaining at the cell membrane by isolating the cell membrane fraction of<br>
the cell, contacting the fraction with a labelled ligand of the transmembrane<br>
protein and determining the level of binding of the ligand to the fraction;<br>
wherein detection of an altered level of the transmembrane protein at the cell membrane relative to the level at the cell membrane in a control cell not contacted with the candidate compound indicates that the compound interacts with the transmembrane protein.<br>
37.	The method as claimed in claim 36, wherein the labelled ligand is a radio-labelled ligand.<br>
38.	The method as claimed in claim 36 or 37, wherein the nucleotide sequence is modified to encode an NLS selected from Table 1.<br>
39.	The method as claimed in claim 36 or 37, wherein the nucleotide sequence is modified to encode an amino acid sequence selected from the group consisting of KKFKR, PKKKRKV and AFSAKKFKR.<br>
40.	The method as claimed in any one of claims 36 to 39, wherein the cell is a eukaryotic cell or a prokaryotic cell.<br>
41.	The method as claimed in claim 40, wherein the cell is a eukaryotic cell selected from the group consisting of a mammalian cell, a yeast cell, an insect cell, a nematode cell, a plant cell and a fungal cell.<br>
42.	The method as claimed in claim 41, wherein the cell is a mammalian cell selected from the group consisting of HEK, COS and CHO cells.<br>
43.	The method as claimed in any one of claims 36 to 42, wherein the transmembrane protein is selected from the group consisting of a G protein coupled receptor (GPCR), a transporter, a cytokine receptor, a tyrosine kinase receptor and a low density lipoprotein (LDL) receptor.<br><br>
44.	The method as claimed in claim 43, wherein the transmembrane protein is a GPCR.<br>
45.	The method as claimed in claim 44, wherein the GPCR is selected from the group consisting of dopamine D1 receptor, dopamine D2 receptor, dopamine D3 receptor, dopamine D5 receptor, histamine 1 receptor, cysteinyl leukotriene receptor 1, cysteinyl leukotriene receptor 2, opioid receptor, muscarinic receptor, serotonin receptor, beta2-adrenergic receptor, and metabotropic glutamate 4 receptor.<br>
46.	The method as claimed in claim 43, wherein the transmembrane protein is a transporter.<br>
47.	The method as claimed in claim 46, wherein the transporter is selected from the group consisting of dopamine transporter and serotonin transporter.<br>
48.	The method as claimed in claim 43, wherein the transmembrane protein is a cytokine receptor.<br>
49.	The method as claimed in claim 48, wherein the cytokine receptor is selected from the group consisting of erythropoietin receptor and insulin receptor.<br>
50.	The method as claimed in claim 43, wherein the transmembrane protein is a tyrosine kinase receptor.<br>
51.	The method as claimed in claim 50, wherein the tyrosine kinase receptor is selected from the group consisting of epidermal growth factor receptor and insulin receptor.<br>
52.	The method as claimed in claim 43, wherein the transmembrane protein is a low density lipoprotein receptor.<br>
53.	The method as claimed in any one of claims 36 to 52, wherein the cell comprises a plurality of nucleotide sequences, each of said sequences encoding a protein comprising a different transmembrane protein containing at least one NLS.<br><br>
54.	The method as claimed in claim 53, wherein each of said nucleotide sequences encodes a protein having a different labelled ligand.<br>
55.	The method as claimed in claim 53, wherein at least one labelled ligand is common to at least two encoded proteins.<br>
56.	The method as claimed in claims 36 to 55, wherein detection of an altered distribution of the transmembrance protein comprises detection of a reduced level of the labelled ligand associated with the cell membrane.<br>
57.	The method as claimed in claims 36 to 55, wherein detection of an altered distribution of the transmembrane protein comprises detection of an increased level of the labelled ligand associated with the cell membrane.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LUFzc2lnbm1lbnQtKDEzLTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-Assignment-(13-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUFzc2lnbm1lbnQtKDE1LTEwLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Assignment-(15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNsYWltcy0oMDctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Claims-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNsYWltcy0oMTMtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Claims-(13-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNsYWltcy0oMTQtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Claims-(14-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-Correspondence Others-(13-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Correspondence Others-(13-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Correspondence-Others (15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Correspondence-Others-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTQtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Correspondence-Others-(14-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LURyYXdpbmdzLSgwNy0wMS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Drawings-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LUZvcm0tMS0oMTMtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-Form-1-(13-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUZvcm0tMS0oMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Form-1-(15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUZvcm0tMTMtKDA3LTAxLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Form-13-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUZvcm0tMi0oMDctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Form-2-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LUZvcm0tMi0oMTMtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-Form-2-(13-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LUdQQS0oMTMtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-GPA-(13-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LUdQQS0oMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-GPA-(15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">3528-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzUyOC1ERUxOUC0yMDA0LVBldGl0aW9uIDEzNy0oMDctMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3528-DELNP-2004-Petition 137-(07-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="250576-design-and-development-of-lubricity-additive-dosing-system-for-operation-of-bhel-ge-gas-turbines-on-naphtha-fuel-with-flexibility-to-operate-on-various-lubricity-additives.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250578-a-process-for-the-preparation-of-nanomaterial.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250577</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3528/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jan-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>10-Nov-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>OMEROS CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1420, FIFTH AVENUE, SUITE 2600, SEATTLE, WASHINGTON 98101, UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>O&#x27;DOWD, BRAIN F</td>
											<td>229 CATALINA DRIVE, SCARBOROUGH, ONTARIO M1E 1B8, CANADA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GEORGE, SUSAN, R</td>
											<td>229 CATALINA DRIVE, SCARBOROUGH, ONTARIO M1E 1B8, CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G01N 33/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CA2003/00542</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-04-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/371,704</td>
									<td>2002-04-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/387,570</td>
									<td>2002-06-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/442,556</td>
									<td>2003-01-27</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/379,419</td>
									<td>2002-04-12</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/422,891</td>
									<td>2002-11-01</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250577-method-of-identifying-transmembrane-protein-interacting-compounds by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:21:43 GMT -->
</html>
