<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250570-a-method-of-making-a-type-ii-restriction-endonuclease-having-altered-specificity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:21:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250570:A METHOD OF MAKING A TYPE II RESTRICTION ENDONUCLEASE HAVING ALTERED SPECIFICITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF MAKING A TYPE II RESTRICTION ENDONUCLEASE HAVING ALTERED SPECIFICITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A novel restriction endonuclease and methods of making the same are obtainable from either Citrobacter species 2144 or the recombinant stain Escherichia coli which leaves at nt sequence 5’=CAANNNNNGTGG-3’ (SEQ ID NO: 14) in double-stranded DNA molecules. The novel restriction endonuclease is a modular protein in which the specificity moiety is an independent module from the restriction-modification module.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp;<br>
The Patents Rules, 2003<br>
COMPLETE SPECIFICATION<br>
(See section 10 and rule 13)<br>
1. TITLE OF THE INVENTION :<br>
"A NOVEL MODULAR TYPE II RESTRICTION ENDONUCLEASE, CSPCI,<br>
AND THE USE OF MODULAR ENDONUCLEASES FOR GENERATING<br>
ENDONUCLEASES WITH NEW SPECIFICITIES"<br>
2.  APPLICANT (S)<br>
(a)	NAME	:	NEW ENGLAND BIOLABS, INC.<br>
(b)	NATIONALITY	:	USA<br>
(c)	ADDRESS	:	240 County Road, Ipswich, MA 01938, USA<br><br>
WO 2005/094516	2	PCT/US2005/009824<br>
A NOVEL MODULAR TYPE II RESTRICTION<br>
ENDONUCLEASE, CspCI, AND THE USE OF MODULAR<br>
ENDONUCLEASES FOR GENERATING ENDONUCLEASES WITH<br>
5	NEW SPECIFICITIES<br>
BACKGROUND OF THE INVENTION<br>
Restriction endonucleases are enzymes that occur naturally<br>
10	in certain unicellular microbes—mainly bacteria and archaea—and<br>
that function to protect those organisms from infections by viruses and other parasitic DNA elements. Restriction endonucleases bind to specific sequences of nucleotides ('recognition sequence') in double-stranded DNA molecules (dsDNA) and cleave the DNA,<br>
15	usually within or close to these sequences, disrupting the DNA and<br>
triggering its destruction.  Restriction endonucleases usually occur with one or more companion enzymes termed modification methyltransferases.  Methyltransferases bind to the same sequences in dsDNA as the restriction endonucleases they<br>
20	accompany, but instead of cleaving the DNA, they alter it by the<br>
addition of a methyl group to one of the bases within the sequence. This modification ('methylation') prevents the restriction endonuclease from productively recognizing that site thereafter, rendering the site resistant to cleavage. Methyltransferases<br>
25	function as cellular antagonists to the restriction endonucleases<br>
they accompany, protecting the cell's own DNA from destruction by its restriction endonucleases. Together, a restriction endonuclease and its companion modification methyltransferase(s) form a<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-2-<br>
restriction-modification (R-M) system, an enzymatic partnership that accomplishes for microbes what the immune system accomplishes, in some respects, for multicellular organisms.<br>
5	A large and varied class of restriction endonucleases has<br>
been classified as 'Type II' class of restriction endonucleases. These enzymes cleave DNA at defined positions, and when purified can be used to cut DNA molecules into precise fragments for gene cloning and analysis. The biochemical precision of Type II restriction<br>
10	endonucleases far exceeds anything achievable by chemical<br>
methods, making these enzymes the reagents sine qua non of molecular biology laboratories. In this capacity as molecular tools for gene dissection Type II restriction endonucleases have had a profound impact on the life sciences and medicine in the past 25<br>
15	years, transforming the academic and commercial arenas, alike.<br>
Their utility has spurred a continuous search for new restriction endonucleases, and a large number have been found: today more than 250 Type II endonucleases are known, each possessing different DNA cleavage characteristics (Roberts, RJ. et al., Nucl.<br>
20	Acids. Res. 33:D230-D232 (2005)). (Rebase,<br>
http://rebase.neb.com/rebase). The production and purification of these enzymes have also been improved by the cloning and overexpression of the genes that encode them, usually in the context of non-native host cells such as E. coli.<br>
25<br>
Since the various restriction enzymes appear to perform similar biological roles, and share the biochemistry of causing dsDNA breaks, it might be thought that they would resemble one<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-3-<br>
another in amino acid sequence closely. Experience shows this not<br>
to be true, however. Surprisingly, far from sharing significant<br>
amino acid similarity with one another, most enzymes appear<br>
unique, with their amino acid sequences resembling neither other<br>
5	restriction enzymes nor any other known kind of protein. Type II<br>
restriction endonucleases seem to have arisen independently of each other during evolution, for the most part, and to have done so hundreds of times, so that today's enzymes represent a heterogeneous collection rather than a discrete family descended<br>
10	from a common ancester. Restriction endonucleases are<br>
biochemically diverse in organization and action: some act as homodimers, some as monomers, others as heterodimers. Some bind symmetric sequences, others asymmetric sequences; some bind continuous sequences, others discontinuous sequences; some<br>
15	bind unique sequences, others multiple sequences. Some are<br>
accompanied by a single methyltransferase, others by two, and yet others by none at all. When two methyltransferases are present, sometimes they are separate proteins and at other times they are fused. The orders and orientations of restriction and modification<br>
20	genes vary, with all possible organizations occurring. Several kinds<br>
of methyltransferases exist, some methylating adenines, others methylating cytosines at the N-4 position, or at the 5 position). Usually there is no way of predicting, a priori, which modifications will block a particular restriction endonuclease, which kind(s) of<br>
25	methyltransferases(s) will accompany that restriction endonuclease<br>
in any specific instance, nor what their gene orders or orientations will be.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-4-<br>
From the point of view of cloning a Type II restriction<br>
endonuclease, the great variability that exists among R-M systems<br>
means that, for experimental purposes, each is unique. Each<br>
enzyme is unique in amino acid sequence and catalytic behavior;<br>
5	each occurs in unique enzymatic association, adapted to unique<br>
	microbial circumstances; and each presents the experimenter with<br>
a unique challenge. Sometimes a restriction endonuclease can be<br>
cloned and over-expressed in a straightforward manner but very<br>
often it cannot, and what works well for one enzyme may fail<br>
10	altogether for the next. Success with one is no guarantee of<br>
success with another.<br>
Novel endonucleases provide opportunities for innovative genetic engineering.<br>
15<br>
SUMMARY OF THE INVENTION<br>
			In an embodiment: of the invention, a substantially pure<br>
            Type IIG restriction endonuclease and an isolated DNA<br>
20	obtainable from Citrobacter species 2144 (NEB#1398) (ATCC<br>
Patent Accession No. PTA-5846) have been obtained. The recombinant DNA of the enzyme from the Citrobacter species and cloned product thereof from Escherichia coli NEB#1554 (ATCC Patent Accession No. PTA-5887) is provided.<br>
25<br>
A further characteristic of the above-described restriction endonuclease is that it recognizes the following base sequence in double-stranded deoxyribonucleic acid molecules:<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-5-<br><br><br>
5 1.0<br><br>
and cleaves the DNA on both sides of the recognition sequence<br>
15	at the alternative positions shown by the arrows.<br>
			The DNA encoding the restriction endonuclease described<br>
                   above may include a first DNA segment expressing endonuclease<br>
                    and methyltransferase catalytic functions and a second DNA<br>
20	segment encoding a sequence specificity function of the<br>
restriction endonuclease wherein the first and second DNA segments are contained in one or more DNA molecules.<br>
The above-described DNA may be inserted into a vector.<br>
25	The vector may include at least one of a first DNA segment<br>
coding for the restriction and modification domains of CspCI<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-6-<br>
restriction endonuclease and a second segment coding for the specificity domain of the restriction endonuclease.<br>
In an embodiment of the invention, a host cell is provided<br>
5	which is transformed by a first DNA segment coding for the<br>
                   restriction and modification domains of CspCI restriction<br>
                    endonuclease and a second segment coding for the specificity<br>
                    domain of the restriction endonuclease. The first DNA segment<br>
                    and the second DNA segment may be contained within one or<br>
10	 more DNA vectors.<br>
			In an embodiment of the invention, a method is provided<br>
                  for obtaining the restriction endonuclease which includes the<br>
                  steps of cultivating a sample of Citrobacter species 2144<br>
15	(NEB#1398) or a host cell as described above under conditions<br>
favoring the production of the endonuclease; and purifying the endonuclease therefrom.<br>
In an embodiment of the invention, a method of making a<br>
20	Type II restriction endonuclease having an altered specificity<br>
                    includes: (a) selecting a restriction endonuclease from a set of<br>
                   enzymes wherein each enzyme in the set is characterized by a<br>
                    modular structure having a specificity subunit and a catalytic<br>
                  subunit. The specificity subunit further includes an N-terminal<br>
25	domain for binding one half site of a bipartite recognition<br>
sequence and a C-terminal domain for binding a remaining half site of the bipartite recognition sequence;   (b) modifying the<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-7-<br>
specificity subunit; and (c) obtaining the restriction endonuclease with altered specificity.<br>
Where the restriction endonuclease is CspCI, one half site<br>
5	is CAA and the other half site is GTGG.<br>
In this method, the step of modifying the specificity subunit may further include (a) substituting the N-terminal domain with a second copy of the C-terminal domain or<br>
10	substituting the C-terminal domain with a second copy of the N-<br>
terminal domain (b) substituting the N-terminal domain or the C-terminal domain or both N-terminal and C-terminal domain with a DNA-binding domain from a second restriction endonuclease or methylase, or (c) mutating the N-terminal<br>
15	domain, the C-terminal domain or both domains to alter the<br>
binding specificity. In any of these modifications or without   . these modifications, an additional modification can be added, namely changing the length of the spacer amino acid sequence between the N-terminal and C-terminal domains of the<br>
20	specificity subunit.  In any of the above, the specificity subunit<br>
and the catalytic subunit may be encoded by separate and distinct genes.<br>
In an embodiment of the invention, DNA-binding domain<br>
25	from the second restriction endonuclease or methylate may<br>
derive from a Type I restriction endonuclease, another Type IIG restriction endonuclease, or from a y-type m6A methyltransferase. Additionally, it is envisioned that the N-<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
terminal cleavage domains can be grafted onto other DNA-binding proteins.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
5<br>
Figure 1 is an agarose gel showing CspCI-cleavage of phage lambda, T7, PhiX174, pBR322 and pUC19 DNAs.  Lanes are as follows:<br>
lanes 1,  10,  15:  lambda-HindIII,  PhiX174-HaeIII size<br>
10	standards;<br>
lane 2:  lambda DNA + CspCI;<br>
lane 3:  T7 DNA + CspCI;<br>
lane 4:  PhiX174 DNA;<br>
lane 5:  PhiX174 DNA + CspCI;<br>
15	lane 6:   PhiX174 DNA + CspCI + PstI;<br>
lane 7:   PhiX174 DNA + CspCI + Sspl;<br>
lane 8:   PhiX174 DNA + CspCI + Neil;<br>
lane 9:   PhiX174 DNA + CspCI + StuI;<br>
lane 11: pBR322 DNA;<br>
20	lane 12: pBR322 DNA + CspCI;<br>
lane 13: pUC19 DNA;<br>
lane 14: pUC19 DNA + CspCI.<br>
Figure 2 is a high-concentration agarose gel of CspCI-<br>
25	cleaved pUC2CspC DNA showing 35±1 bp internal 'mini-<br>
fragment' (arrows).<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-9-<br>
Figure 3 is a high-resolution agarose gel showing partial-digestion doublet fragments. DNA: BglI-cleaved pUC2CspC re-digested with increasing amounts of CspCI. Transient CspCI-BglI fragment doublets are show by the arrows. <br>
5<br>
			Figures 4a and 4b show a determination of the CspCI<br>
               cleavage sites by primed synthesis. Two experiments were<br>
               performed using the same M13mpl8 template and primer<br>
                combination.  (-) is CspCI-cleaved DNA only; (+) is Klenow-<br>
10	treatment of the CspCI-cleaved DNA.<br>
Figure 5 shows a determination of the CspCI cleavage sites by run-off automated sequencing.<br>
15	Figure 5a:  pUClCspC-4 template; forward primer<br>
(SEQIDNO.-l)<br>
              Figure 5b:  pUClCspC-4 template; reverse primer            (SEQID NO:2)                                                                              <br>
20<br>
Figure 5c:   pUClCspC-1 template; forward primer (SEQID NO:3)<br>
Figure 5d:  pUClCspC-1 template; reverse primer<br>
25	(SEQID NO:4<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-10-<br>
A-anomalies, signifying template cleavage, are shown as triangles (A) below the tracings.<br>
5	Figure 6 shows the complete nucleotide sequence of the<br>
DNA cloned from Citrobacter species 2144 (NEB#1398, New England Biolabs, Inc., Beverly, MA) (SEQ ID NO:5).<br>
Figure 7a shows the nucleotide sequence of the CspCI-R-M<br>
10	gene (SEQ ID NO:6).<br>
Figure 7b shows the nucleotide sequence of the CspCI-S gene (SEQ ID NO:7).<br>
15	Figure 8a shows the gene organization of the CspCI<br>
restriction-modification system.<br>
			Figure 8b shows the gene organization of the plasmid clone<br>
                pUC19-CspCI-R-M-S Apol #3 carrying the CspCI genes inserted<br>
20	into the EcoRI site of pUC19<br>
        Figure 9a shows the predicted amino acid sequences of the R-M-   CspCI endonuclease-methyltransferase subunit (SEQ ID NO:8).<br>
 25<br>
Figure 9b shows the predicted amino acid sequences of the CspCI specificity subunit (SEQ ID NO:9).<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-11-<br>
DETAILEP DESCRIPTION OF THE INVENTION<br>
In most restriction enzymes, the parts of the protein<br>
5	responsible for binding to the recognition sequence<br>
('specificity':S) and for cleaving it ('catalysis') are interlinked. Experience has taught that altering either of these functions frequently impairs the other, and renders the enzyme inactive. A new class of enzymes has been identified in which the functions<br>
10	of specificity and catalysis are largely separated. These<br>
members of the Type IIG class of restriction endonucleases are large enzymes in which the twin activities of restriction and modification are combined in a single polypeptide chain while specificity resides with a different polypeptide chain. Examples<br>
15	of restriction endonucleases in this class are CspGI, Bcgl and<br>
Bael.  While not wishing to be limited by theory, CspCI is believed to act as a dimer of one R-M-subunit and one S-subunit, while Bcgl acts as a trimer of two R-M subunits and one S-subunit.<br>
20<br>
The separated functional organization of this class of enzymes provides unusual opportunities for protein engineering because the functional modules can be independently manipulated to generate novel specificities of choice as<br>
25	described in more detail in Example V.<br>
This new class of endonucleases is characterized by a DNA encoding the specificity subunit that is distinct from the R-M<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-12-<br>
genes. The genes for these occur side by side, naturally, and<br>
are expressed in cis. These genes can also be separated into<br>
different replicons, and expressed in trans, without loss of<br>
activity. The separate location of these genes in different<br>
5	amplicons permits the S and the R-M genes to be altered<br>
                  individually, and allows the endonuclease, or variants of it, to be<br>
                  reconstituted easily in vivo, simply by introducing the two<br>
                  replicons into the same cell, rather than rejoining the genes into<br>
                  the same DNA molecule. Reconstitute can be performed<br>
10	individually, or in bulk by transforming libraries of one altered<br>
gene into cells harboring the other. Both genes may alternatively be co-transformed, together in a mixture.<br>
Alternatively, the R-M and S genes can be separated to<br>
15	allow them to be expressed individually in different host cells. It<br>
                  will be appreciated that since neither protein alone exhibits toxic<br>
                  activity, the cells producing either subunit will be viable.<br>
                   Expressing the subunits separately allows them to be purified<br>
                   individually, and enables the enzyme, or variants of it, to be<br>
20	reconstituted easily in vitro, simply by mixing together<br>
preparations of the two subunits. High-throughput screening, and/or multiplexing can be achieved using extracts of cells instead of purified proteins.<br>
25	The presence of DIMA-methyltransferase motifs within this<br>
class of endonuclease suggests that the endonucleases have intrinsic methylation activity, in addition to endonuclease<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-13-<br>
activity.  For example, CspCI is dependent on S-adenosyl-L-<br>
methionine (AdoMet).  By mutating the catalytic sites for these<br>
activities, variants of these endonucleases can be isolated. DNA-<br>
cleavage activity, DNA-methylation activity, or both, may be<br>
5	abolished in these mutants.<br>
			Typically, the specificity subunit of endonucleases in the<br>
                  Type IIG class determines which target sequence in a DNA<br>
                  molecule will undergo cleavage by means of the R-M subunit.<br>
10	The R-M subunit has a distinct N-terminal domain for DNA-<br>
cleavage, and a distinct C-terminal domain for DNA-methylation. The S subunit has a distinct N-terminal domain for binding one-half of the bipartite recognition sequence, and a distinct C-terminal domain, for binding the other half.<br>
15<br>
			Other modular enzymes exist which characteristically<br>
                   cleave DNA at a sequence that is distant to the recognition site.<br>
                   However, these enzymes are monomers (Cjel and Alol) or<br>
                    homodimers (HaelV) both types being single proteins with a<br>
20	composition of R-M-S.,<br>
			For any unknown restriction endonuclease that is observed<br>
                   to have a modular structure, the recognition sequence of the<br>
25	endonuclease of the class may be determined by mapping the<br>
locations of the cleavage sites in a target DNA of known sequence. The DNA sequences of these regions are compared for similarity and common features.  Candidate recognition sequences are compared with the observed restriction fragments<br><br>
WO 2005/094516<br><br>
PCT7US2005/009824<br><br>
-14-<br>
produced by endonuclease-cleavage of a variety of DNAs. The<br>
approximate size of DNA fragments produced by endonuclease<br>
digestion can be entered into the program REBPredictor, which<br>
can be accessed at<br>
5	http://taa.neb.com/~vincze/REBpredictor/index.php.   Example<br>
III describes how REBPredictor was used to predict potential recognition sites for CspCI.<br>
A modular endonuclease of the type described above can<br>
10	be obtained as a product of recombination in a host cell or by<br>
culturing the native strain. Host cells are grown in suitable<br>
media supplemented with lOOmg/ml ampicillin and incubated<br>
aerobically at 37°C. Cells in the late logarithmic stage of growth<br>
are collected by centrifugation and either disrupted immediately<br>
15	or stored frozen at -70°C.<br>
Conventional protein purification techniques can be used to isolate the endonuclease from lysed cells. Cell paste is suspended in a buffer solution and ruptured by sonication, high-<br>
20	pressure dispersion or enzymatic digestion to allow extraction of<br>
the endonuclease by the buffer solution. Intact cells and cellular debris are then removed by centrifugation to produce a cell-free extract containing the endonuclease. The endonuclease is then purified from the cell-free extract by ion-exchange<br>
25	chromatography, affinity chromatography, molecular sieve<br>
chromatography, or a combination of these methods.<br><br>
WO 2005/094516	PCTVUS2005/009824<br>
- 15 -<br>
Alteration of the specificity domains in Type I restriction enzymes has been achieved to generate novel enzymes that recognize symmetric DNA sequences, and hybrid DNA sequences (Mac Williams et al. Journal of Cell Biochemistry 18cl36 (1994); Mac Williams et <br>
5	aI.EMBO Journal 15: 4775-4783 (1996)). Example VI describes how the         <br>
         specificity domain in a modular Type II restriction enzyme can be                     <br>
         manipulated to alter the specificity of the enzyme.<br>
Present embodiments of the invention are further illustrated <br>
10          by the following Examples. These Examples are provided to aid in        the understanding of embodiments of the invention and are not construed as a limitation thereof.<br>
The references cited above and below as well as provisional <br>
15          application number 60/555,795 are herein incorporated by reference.<br>
EXAMPLES<br>
20	Example I: Isolation of CspCI<br>
CspCI was obtained  by culturing either (i) Citrobacter species 2144 (NEB#1398) or (ii) the transformed host,E. coli NEB#1554, and recovering theendonuclease from the cells. A <br>
25            sample of Citrobacter species 2144 (NEB#1398) has been<br>
deposited under the terms and conditions of the Budapest Treaty with the American Type Culture Collection (ATCC) on March 4,<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-16-<br>
2004 and bears the Patent Accession No. PTA-5846. A sample of<br>
a recombinant strain expressing CspCI, E. coli (NEB#1554), has<br>
also been deposited under the terms and conditions of the<br>
Budapest Treaty with the American Type Culture Collection<br>
5	(ATCC) on March 24, 2004 and bears the Patent Accession No.<br>
PTA-5887.<br>
			Citrobacter species 2144 (NEB#1398) or E. coli<br>
(NEB#1554) were incubated aerobically at 37°C. Cells in the late<br>
10	logarithmic stage of growth are collected by centrifugation and<br>
either disrupted immediately or stored frozen at -70°C.<br>
The CspCI endonuclease was isolated from Citrobacter species 2144 (NEB#1398) or Escherichia coli (NEB#1554) by<br>
15	conventional protein purification techniques. The cell paste was<br>
suspended in a buffer solution and ruptured by sonication, high-pressure dispersion or enzymatic digestion to allow extraction of the endonuclease by the buffer solution. Intact cells and cellular debris are then removed by centrifugation to produce a cell-free<br>
20	extract containing CspCI. The CspCI endonuclease was then<br>
purified from the cell-free extract by ion-exchange chromatography, affinity chromatography, molecular sieve chromatography, or a combination of these methods to produce the endonuclease.<br>
25<br>
Example II: Production of Native or Recombinant CspCI Endonuclease<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-17-<br>
277 grams of E. coli NEB#1554 CspCI cell pellet or<br>
Citrobacter species 2144 (NEB#1398) (New England Biolabs,<br>
Inc., Beverly, MA) were suspended in 1 liter of Buffer A (20mM<br>
Tris-HCI (pH 7.4), 1.0mM DTT, 0.1mM EDTA, 5% Gycerol)<br>
5	containing 300mM NaCI, and passed through a Gaulin<br>
homogenizer at ~12,000 psig. The lysate was centrifuged at "13,000 x G for 40 minutes and the supernatant collected.<br>
The supernatant solution was applied to a 400 ml DEAE<br>
10	Fast-Flow column (GE Healthcare, formerly Amersham<br>
Biosciences, Piscataway NJ) column equilibrated in buffer A plus 300mM NaCI, and the flow-through, containing the CspCI endonuclease activity, was diluted 1:1 with buffer A.<br>
15	The diluted enzyme was applied to a 375 ml Heparin<br>
Hyper-D column (Biosepra, Marlborough MA), which had been equilibrated in buffer B. (20mM Tris-HCI (pH 7.4), 150mM NaCI, 1.0mM DTT, 0.1mM EDTA, 5% Gycerol). A 2.5 L wash of buffer B was applied, then a 2 L gradient of NaCI from 0.15M to 1M in<br>
20	buffer B was applied and fractions were collected. Fractions<br>
were assayed for CspCI endonuclease activity by incubating with 1 microgram of phage lambda DNA (NEB) in 50 microliter NEBuffer 2, supplemented with 20 microMolar (AdoMet) for 15 minutes at 37° C. CspCI activity eluted at 0.3M to 0.35M NaCI.<br>
25<br>
The Heparin Hyper-D column fractions containing the CspCI activity were pooled and load directly onto a 200 ml Ceramic htp column (Biosepra, Marlborough MA) equilibrated in<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-18-<br>
Buffer B. A 1 L wash of buffer B was applied, then a 1 L gradient of KHPO4 (pH 7.5) from 0M to 0.6M in buffer B was applied and fractions were collected.  Fractions were assayed for CspCI endonuclease activity by incubating with 1 microgram of phage lambda DNA in 50 microliter NEBuffer 2, supplemented with 20 microMolar AdoMet for 15 minutes at 37° C.  CspCI activity eluted at 0.4M to 0.5M KHP04.<br>
The Ceramic HTP column fractions containing the CspCI activity were pooled and dialyzed into Buffer C (20mM Tris-HCI (pH 7.4), 100mM NaCI, 1.0mM DTT, 0.1mM EDTA, 5% Gycerol).<br>
This pool was flowed through a 50 ml Source Q column (GE Healthcare, formerly Amersham Biosciences, Piscataway NJ.) equilibrated in buffer C and directly onto a Heparin TSK equilibrated in buffer C. A 250 ml wash of buffer C was applied, then a 400 ml gradient of NaCI from 0.1M to 0.8 M in buffer C was applied and fractions were collected.  Fractions were assayed for CspCI endonuclease activity by incubating with 1 microgram of phage lambda DNA (New England Biolabs, Inc., Beverly, MA) in 50 microliter NEBuffer 2, supplemented with 20 microMolar AdoMet for 15 minutes at 37° C. CspCI activity eluted at 0.3M to 0.35M NaCI.<br>
The pool was dialyzed into Storage Buffer (10mM Tris-HCI (pH 7.4), 100mM NaCI, 1.0mM DTT, 0.1mM EDTA, 50% Gycerol). One million units of CspCI were obtained from this procedure. The CspCI endonuclease thus produced was<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-19-<br>
substantially pure and free of contaminating nucleases. SDS<br>
polyacrylamide gel electrophoresis of a sample of this<br>
preparation showed it comprised two principal proteins of<br>
approximately 70 kDa and 35 kDa in the approximate ratio by<br>
5	mass of 2:1.<br>
Activity determination<br>
CspCI activity: Samples of from 1 to 10 microliter were<br>
10	added to 50 microliter of substrate solution consisting of IX<br>
NEBuffer 2 (New England Biolabs, Inc., Beverly, MA) containing<br>
1	microgram of phage lambda phage DNA supplemented with 20<br>
microMolar AdoMet. The reaction was incubated at 37°C for 60<br>
minutes. The reaction was terminated by adding 20 microliter of<br>
15	stop solution (50% glycerol, 50 mM EDTA pH 8.0, and 0.02%<br>
Bromophenola Blue.) The reaction mixture was applied to a 1% agarose gel and electrophoresed. The bands obtained were identified by comparison with DNA size standards.<br>
20	Unit Definition: One unit of CspCI is defined as the amount<br>
of CspCI required to completely cleave one microgram of phage lambda DNA in a reaction volume of 50 microliter of IX NEBuffer<br>
2	(New England Biolabs, Inc., Beverly, MA) supplemented with<br>
20 microMolar AdoMet, within one hour at 37°C.<br>
25<br>
Properties of CSPCI:<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-20-<br>
AdoMet: Supplementing the CspCI reaction with 20 mM<br>
AdoMet greatly enhanced the activity of the enzyme. In<br>
reactions where AdoMet was omitted, the enzyme exhibited less<br>
than 5% of the cutting activity it exhibited in the AdoMet-<br>
5	supplemented reactions, indicating that AdoMet is a necessary<br>
cofactor for this enzyme.<br>
			Activity in various reaction buffers: CspCI was found to be<br>
                  most active in NEBuffer 2 + AdoMet, relative to other standard<br>
10	NEBuffers (New England Biolabs, Inc, Beverly, MA).<br>
			Digestion at 37°C for one hour in the following NEBuffers<br>
                yielded the following approximate percentage cleavage activities<br>
                 relative to NEBuffer 2 (New England Biolabs, Inc, Beverly, MA)+<br>
15	20mM AdoMet:<br>
NEBuffer 1 + 20mM AdoMet: 10%<br>
                                                 NEBuffer 2 + 20mM AdoMet: 100%<br>
                                                 NEBuffer 3 + 20mM AdoMet: 10%<br>
20	NEBuffer 4 + 20mM AdoMet: 75%<br>
NEBuffer 2 - (No AdoMet): 
			Activity in a 16-hour reaction: 0.5 units of CspCI are<br>
                  required to cut one microgram of phage lambda DNA in a 16-<br>
25	hour digest, compared to one unit that is required to cut one<br>
microgram of phage lambda DNA in a one-hour digest.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-21-<br>
Temperature: The CspCI unit titer was determined at 37°C<br>
by a one-hour incubation in IX NEBuffer 2 plus 20 microMolar<br>
AdoMet. Incubation of CspCI at 70°C for 20 minutes prior to<br>
performing a reaction at 37°C does not inactivate the enzyme.<br>
5	After heat treatment at 70°C for 20 minutes, CspCI retains<br>
nearly full activity.<br>
Bilateral cleavage: CspCI cleaves DNA on both sides of its recognition sequence. As a result, in addition to producing<br>
10	regular restriction fragments, CspCI cleavage generates small,<br>
internal, 'mini-fragments' of 35±1 bp, one from each recognition site. These mini-fragments, which can be visualized by gel electrophoresis (Figure 2), comprise the recognition sequence and the flanking DNA on each side up to the cut sites. The two<br>
15	cleavage events that produce the mini-fragments appear to<br>
proceed separately: cleavage occurs first on one side of the recognition sequence and then later on the other side, rather than on both sides simultaneously. As a result, when partially digested samples of DNA are examined by gel electrophoresis,<br>
20	the DNA fragments appear as doublets or triplets depending on<br>
whether the min-fragrnents have been trimmed yet from their termini (Figure 3).<br><br>
WO 2005/094516<br><br>
PCT7US2005/009824<br><br>
-22-<br>
Example III: Determination of the CspCI Cleavage Site<br>
5	The location of CspCI-induced cleavage relative to the<br>
recognition sequence was determined by two methods, primed synthesis and run-off automated sequencing.<br>
A: Primed synthesis method<br>
10<br>
The locations of CspCI cleavages relative to the recognition sequence was determined by cleavage of a primer extension product, which was then electrophoresed alongside a set of standard dideoxy sequencing reactions produced from the<br>
15	same primer and template.  M13mpl8 DNA was employed as<br>
the template with a primer near the recognition sequence position at 3009. Readable sequence for this primer template combination begins at position 3069 and continues through the CspCI site.<br>
20<br>
Sequencing Reactions<br>
The sequencing reactions were performed using   the<br>
25	Sequaenase version 2.0 DNA sequencing kit (GE Healthcare,<br>
formerly Amersham Life Science) with modifications for the cleavage site determination. The template and primer were assembled in a 0.5 ml Eppendorf tube by combining 2.5<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-23-<br>
microliter dH20, 3 microliter 5X sequencing buffer (200 mM Tris<br>
pH 7.5, 250 mM NaCI, 100 mM MgCI2), 8 microliter M13mpl8<br>
single-stranded DNA (1.6 microgram) and 1.5 microliter of<br>
primer at 3.2 mM concentration. The primer-template solutions<br>
5	were incubated at 65°C for 2 minutes, then cooled to 37°C over<br>
	20 minutes in a beaker of 65°C water on the bench top to<br>
                   anneal the primer. The labeling mix (diluted 1:20) and T7<br>
                    Sequenase polymerase were diluted according to manufacturer's<br>
                   instructions. The annealed primer and template tube was placed<br>
10	on ice. To this tube were added 1.5 microliter 100mM DTT, 3<br>
                 microliter diluted dGTP labeling mix, 1 microliter [α- 33P] dATP<br>
                (2000Ci/mM, 10mCi/ml) and 3 microliter diluted T7 Sequenase<br>
                polymerase (GE Healthcare, formerly Amersham, Piscataway,<br>
                 NJ). The reaction was mixed and incubated at room temperature<br>
15	for 4 minutes.<br>
3.5 microliter of this reaction was then transferred into each of four tubes containing 2.5 microliter termination mix for the A, C, G and T sequencing termination reactions. To the<br>
20	remaining reaction was added to 10 microliter of Sequence<br>
Extending Mix (GE Healthcare, formerly Amersham Biosciences, Piscataway, NJ), which is a mixture of dNTPs (no ddNTPs) to allow extension of the primer through and well beyond the CspCI site with no terminations to create a labeled strand of DNA<br>
25	extending through the CspCI recognition site for subsequent<br>
cleavage. The reactions were incubated 5 minutes at 37°C. To the A, C, G and T reactions were added 4 microliter of stop' solution and the samples were stored on ice. The extension<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-24-<br>
reaction was then incubated at 70°C for 20 minutes to inactivate the DNA polymerase (Sequenase) (GE Healthcare, formerly Amersham, Piscataway, NJ), then cooled on ice.<br>
5	10 microliter of the extension reaction was then placed in<br>
zone 0.5 ml Eppendorf tube and 7 microliter was placed in a second tube. To the first tube was added 1 microliter (approximately 0.5 unit) of CspCI endonuclease, The reaction was mixed, and then 2 microliter was transferred to the second<br>
10	tube. These enzyme digest reactions were mixed and then<br>
incubated at 37°C for 1 hour, following which the reactions were divided in half. To one half, 4 microliter of stop solution was added and mixed (the 'minus' polymerase reaction). To the second half, 0.4 microliter Klenow DNA polymerase (NEB#210)<br>
15	(New England Biolabs, Inc., Beverly, MA) containing 80 mM<br>
dNTPs was added (the 'plus' reaction), and the reaction was incubated at room temperature for 15 minutes, following which 4 microliter of stop solution was added.<br>
20	The sequencing reaction products were electrophoresed on<br>
an 6% Bis-Acrylamide sequencing gel (Stratagene Corporation, La Jolla, CA), with the CspCI digestions of the extension reaction next to the set of sequencing reactions produced from the same primer and template combination.<br>
25<br>
Results<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-25-<br>
Digestion of the extension reaction product (the 'minus' reaction) produced a band which co-migrated with the C residue 12 bases 5' to the CspCI recognition sequence, 5'-CAGAGAGATAACCCACAAGAATTG-3', (SEQ ID NO: 10) indicating<br>
5	cleavage between the 12th and 11th bases 5' of the recognition<br>
sequence on this strand. A second band was produced which co-migrated with the A residue 12 bases 3' to the CspCI recognition site on this strand, CCACAAGAATTGAGTTAAGCCCAA (SEQ ID<br>
NO: 11), indicating cleavage between the 12th and 13th bases 3'<br>
10	to the recognition site. There was also a faint band one base<br>
farther from the recognition site, indicating that a small portion<br>
                of the molecules were cut between the 13th and 14th bases 3' to<br>
                the recognition sequence. Treatment of the cleaved extension<br>
                reaction product with Klenow DNA polymerase (the 'plus'<br>
15	reaction) produced a band two bases shorter than the first band<br>
described above, which co-migrated with the A residue 14 bases 5' to the recognition sequence; 5'-ATCGAGAGATAACCCACAAGAATTG-3' (SEQ ID NO: 12), indicating<br>
cleavage between the 13th and 14th bases 3' to the recognition<br>
20	sequence on the opposite strand of the DNA (5'-<br>
                  CAANNNNNGTGG(N13) (SEQ ID NO: 13).  Several additional<br>
                   bands were observed in the 'plus' lane as well, corresponding to<br>
                   the original band, 12 bases 3' to the site, and bands one and<br>
                   two bases shorter, produced from cuts on the opposite strand of<br>
25	DNA closer to the recognition sequence (Figure 4).<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-26-<br>
These results, when combined with those obtained by the<br>
second method described below, indicate that CspCI cleaves<br>
DNA on both sides of its recognition sequence, and can do so at<br>
either N11/N13 or N10/N12 5' to the sequence 5'-<br>
5	CAANNNNNGTGG-3' (SEQ ID NO: 14) and at N13/N11 or<br>
N12/N10 3' to the sequence, to produce DNA fragments with 2-base 3'-extensions, and an excised fragment of 34, 35 or 36 bases that contains the recognition site.<br>
10	B: Run-off sequencing method<br>
15<br>
The second approach employed automated sequencing of CspCI-partially cleaved template DNA with forward and reverse                    <br>
primers to produce sequencing traces that extended through the sites of cleavage. Two plasmids served as templates, pUClCspC-1            <br>
              and pUClCspC-4, constructed by inserting an oligonucleotide containing the CspCI recognition sequence into the AatII site at nt 2617 of pUC19 in both orientations (described in Example III, section 2, below).<br>
20<br>
                CspCI-cleavaae of pUClCspC-1 and pUClCspC-4<br>
Sequencing reactions were carried out on partial digests of<br>
pUClCspC-1 and pUClCspC-4, in order to determine the sites of<br>
25	cleavage on both sides of the recognition site.<br>
The digests were performed as follows:<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-27-<br>
a.	Combine:<br>
25 microgram pUClCspC-1 or pUClCspC-4<br>
100 microliter NEBuffer2<br>
1 microliter 32 mM AdoMet<br>
5	dH20 to 1000 microliter<br>
b.	Distribute the mixture: 200 microliter in one reaction<br>
tube, 100 microliter in 8 subsequent tubes.<br>
10	c. Add 160 units CspCI endonuclease to the first tube, mix,<br>
remove 100 microliter and add it to the second tube, mix, remove 100 microliter and add it to the third tube, etc. until the 9th tube is reached.<br>
15	d. Incubate all 9 reactions at 37°C for 60 minutes, then<br>
place on ice.<br>
e.	Analyze a sample of each reaction on agarose gel;<br>
select completely cleaved and partially cleaved plasmids.<br>
20<br>
f.	Purify the cleaved plasmids for sequencing using Zymo<br>
DNA Clean and Concentrator-5 spin-columns according to the<br>
manufacturer's recommendations (Zymo Research, Orange, CA).<br>
25	Sequencing Reactions<br>
The reactions were performed with an ABI377 DNA sequencer using CspCI-cleaved pUClCspC-1 and -4 plasmid<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-28-<br>
templates, and a pair of primers that initiate synthesis<br>
approximately 250 nt away from the CspCI site on one side,<br>
(forward-primer), and 160 nt away from the CspCI site on the<br>
other side (reverse primer). The sequences of these two primers<br>
5	are:<br>
5'- CAGTTCGATGTAACCCACTCG -3' (5EQ ID NO: 15) forward primer; corresponds to pUC19 nt 2346-2366; interrogates the minus-strand of the vector.<br>
10<br>
5'- CCCGCTGACGCGCCCTGACGGGC -3' (SEQ ID NO: 16) reverse primer; corresponds to pUC19 nt 96-118 complement; interrogates the plus-strand of the vector.<br>
15	When sequencing reactions encounter the 5' end of a<br>
template strand, they frequently add a final, non-templated A to the synthesized strand. If the template DNA comprises a mixture of intact and truncated strands, such as occurs in incompletely cleaved DNA samples, the position of cleavage<br>
20	reveals itself in the sequencing trace by an anomalous A peak<br>
superimposed on the normal peak, and by an overall reduction in the heights of the following peaks. If the base normally present at the position of the anomaly is something other than A — G, for example — then a mixed signal is seen, in this example<br>
25	G plus A. However, if the base normally present at this position<br>
is also A, then a single A peak is seen, perhaps higher than normal, and this confounds unambiguous identification.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-29-<br>
Results<br>
	Unambiguous results were obtained for the positions of<br>
                 cleavage on the 5' sides of the recognition sequence, but the<br>
5	data was poorer regarding cleavage on the 3' sides. As a whole,<br>
however, they were consistent with the endonuclease cleaving to produce fragments with 2-base 3'-overhangs at. Sequence traces from representative reactions are shown in Figure 5.<br>
10	The reaction of partially cleaved pUClCspC-4 with the<br>
forward primer displayed a strong anomalous A superimposed on the G 13 nt before the recognition sequence, and a stronger-than-expected A peak 11 nt after it:<br>
5'...AAGTGccacctgacgtgcaacctaggtggcacgtctaagaaac...<br>
15	(SEQIDNO:17)<br>
	         (Notation. Underlined: CspCI recognition site; bold:<br>
                   normal base over which anomalous A superimposed; UPPER<br>
                   CASE: peaks of normal height; lower case: peaks of reduced<br>
20	height)<br>
These results suggest that cleavage of the complementary strand (indicated |) occurs:<br>
5'-GTTTICTTAGACGTGCCACCTAGGTTGCACGTCAGGTGGC<br>
25	lACTT... (SEQIDNO:18)<br>
The reaction of partially cleaved pUClCspC-4 with the reverse primer displayed a strong A-anomaly on the T 12 nt<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-30-<br>
before the recognition sequence, and a suggestion of two anomalous A's under the two G's 11 and 12 nt after the sequence:<br>
5'...TGGTTtcttagacgtgccacctaggttgcacgtcaggtggcact...<br>
5	(SEQIDNO:19)<br>
Ignoring momentarily the G-ll anomaly, these results suggests that cleavage of the complementary strand occurs:<br>
5'.. TGC | CACCTGACGTGCAACCTAGGTGGCACGTCTAAGAA |<br>
10	ACCA...<br>
(SEQ ID NO:20)<br>
Combining these results, CspCI-cleavage at the site in pUClCspC-4 appears to be:<br>
15	5'... AGTGC | CACCTG ACGTGCAACCTAGGTGGCACGTCTAAG A<br>
A|ACC...<br>
(SEQIDNO-.21) 3’…TCA|CGGTGGACTGCACGTTGGATCCACCGTGCAGATTC|<br>
TTTGG...<br>
20	(SEQ ID NO:22)<br>
That is to say: 11/13 CAA N5 GTGG 12/10 (SEQ ID NO: 14)<br>
The same G-13 and A-ll A-anomalies were seen when<br>
25	partially-cleaved pUClCspC-1 was interrogated the forward<br>
primer, and the same T-12 A-anomaly was seen when it was interrogated with the reverse primer. Consequently, cleavage at the site in pUClCspC-1 appears to be:<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-31-<br>
5'...AGTGC|CACCTGACGTGCCACCCGGGTTGCACGTCTAAGA A|ACC...<br>
(SEQIDNO:23)<br>
5	3/...TCAICGGTGGACTGCACGGTGGGCCCAACGTGCAGATTC|<br>
TTTGG...<br>
(SEQ ID NO:24)<br>
               That is to say: 10/12 CAA N5 GTGG 13/11 (SEQ ID NO: 14) 10<br>
                  This numerical reversal in cleavage distances indicates<br>
                that the positions of DNA cleavage are independent of<br>
                recognition-sequence orientation, and dependent on nature of<br>
                 flanking sequence. The sequence to the left (counter-clockwise)<br>
15	of the recognition site is the same in both plasmids, as also is<br>
                  the sequence to the right (clockwise). The latter, which is<br>
                  somewhat A:T-rich, would seem to be more extended,<br>
                  physically, than the G:C-rich DNA to the left, such that the<br>
                   endonuclease, as it 'measures' out from its binding site, cleaves<br>
20	12/10 on either side if the DNA is extended, and 13/11 on either<br>
 side if the DNA is compact.<br>
                      Returning to the G-ll anomaly momentarily ignored,<br>
                  above, its presence in the pUClCspC-4/reverse primer reaction<br>
25	suggests that the otherwise compact leftward DNA can become<br>
more extended, perhaps due to torsional relaxation that accompanies supercoil-release during digestion, leading to<br><br>
WO 2005/094516<br><br>
PCT7US2005/009824<br><br>
-32-<br>
10/12 cleavage at this location, also. This confirms to a degree that CspCI can also cleave:<br>
                10/12 CAA Ns GTGG 12/10 (SEQ ID NO: 14), and by<br>
                  extension,<br>
5	11/13 CAA N5 GTGG 13/11 (SEQ ID NO: 14).<br>
Example IV: Cloning of the CspCI Restriction-Modification Genes<br>
10	1. Preparation of genomic DNA<br>
Genomic DNA was prepared from 2.5g of Citrobacter species 2144, by the following steps:<br>
15	a. Cell wall digestion by addition of lysozyme (2 mg/ml<br>
final), sucrose (1% final), and 50 mM Tris-HCI, pH 8.0.<br>
b.	Cell lysis by addition of 24 ml of Lysis mixture:<br>
(50mM Tris-HCI pH 8.0, 62.5mM EDTA, 1% Triton.<br>
20<br>
c.	Removal of proteins by phenol-CHCI3 extraction of<br>
DNA 2 times (equal volume).<br>
d.	Dialysis in 4 liters of TE buffer, buffer change four<br>
25                         times.<br>
e. RNase A treatment to remove RNA.<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-33-<br>
f. Genomic DNA precipitation in 0.4M NaCI and 0.55 volume of 100% isopropanol, spooled, dried and resuspended in TE buffer.<br>
5	2. Preparation of plasmid vector pUC2CspC<br>
			Plasmid cloning vector pUC2CspC was constructed from<br>
                  E.coli cloning vector pUC19 by inserting two CspCI recognition<br>
                  sites, one at the unique AatII site at nt 2617, and another at the<br>
10	Dral site at nt 1563.<br>
            a. Two pairs of complementary oligonucleotides<br>
                   were synthesized. Annealing of each pair produces a CspCI<br>
                   recognition site, and double-stranded ends that can be ligated to<br>
15	either AatII or Dral DNA fragments such that the ligation<br>
product no longer contains the AatII or Dral site.<br>
The oligonucleotide sequences, shown below in annealed double-strand format, were:<br>
20<br>
AatII-site linker:<br><br><br>
25<br><br>
Dral-site linker:<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-34-<br><br><br>
5<br><br>
b.	For the AatII site linker, 1 microgram pUC19<br>
was digested in a small volume with AatII.<br>
10<br>
c.	Annealed oligonucleotide linker was added to<br>
the reaction, along with T4 DNA ligase and ligase buffer, and the<br>
reaction incubated at room temperature for two hours.<br>
15	d. Reaction products were transformed into E.coli,<br>
and grown in the presence of ampicillin.<br>
e.	ApR transformants were isolated, their plasmids<br>
prepared using a FastPlasmid® Mini Kit (Eppendorf, Hamburg,<br>
20	Germany), and analyzed by digesting with restriction enzymes<br>
AatII and CspCI.<br>
f.	Two plasmids were identified, pUClCspC-1 and<br>
pUClCspC-4, each lacking an AatII site but containing one<br>
25	CspCI recognition site in either of the two possible, opposite<br>
orientations. One of these, pUClCspC-4, was purified on a larger scale, using a Qiagen Plasmid Midi Kit (Qiagen, Valencia, CA) according to the manufacturer's recommendations, for linker insertion at the Dral site.<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-35-<br>
g. For the Dral site linker, only partial digestion products were  desired, therefore digestion, ligation, and Dral site linker components were all added simultaneously. <br>
5<br>
h. Samples of the reaction were removed and placed on ice after incubation times of 2, 5, 10, 20, 40, and 100 minutes.<br>
10	i. Reaction samples were transformed into E.coli,<br>
plasmids prepared and analyzed as in d. and e. above, digesting with restriction enzymes Dral and CspCI.<br>
j. One plasmid, pUC2CspC, containing two CspCI<br>
15	sites was identified and prepared on a large scale using a<br>
Qiagen Plasmid Mega Kit according to the manufacturer's recommendations (Qiagen, Valencia, CA).<br>
Plasmid pUC2CspC was used as the plasmid selection<br>
20	vector for cloning the genes for the CspCI restriction-<br>
modification system. Plasmids pUClCspC-1 and -4 were used as substrates for analysis of the CspCI-cleavage reactions (Example II section b, above).<br>
25	3. Genomic DNA digestion and library construction<br>
Restriction enzymes Apol, BamHI, Bglll, and Sau3AI were used to individually digest ~10 microgram quantities of Citrobacter<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-36-<br>
sp. 2144 genomic DNA to achieve complete and partial digestions.<br>
Following heat-inactivation of the restriction enzymes at 65°C for<br>
15 minutes, the Apol-digests were ligated to EcoRI-cleaved, CIP-<br>
dephosphorylated pUC2CspC vector, and the BamHI-, Bglll-, and<br>
5	Sau3AI-digests were ligated to BamHI-cleaved, CIP-<br>
	dephosphorylated pCspIx2. The ligations, performed overnight with<br>
                   T4 DNA ligase, were then used to transform the endA' E. coli host,<br>
                    ER2683 (New England Biolabs, Inc., Beverly, MA), made competent<br>
                    by the CaCI2 method. Several thousand Ampicillin-resistant (ApR)<br>
10	transformants were obtained from each ligation. These colonies<br>
from each ligation were pooled and amplified in 500ml LB + Ap overnight, and plasmid DNA was prepared from them by CsCI gradient purification to make primary plasmid libraries.<br>
15	4. Cloning the CspCI genes by methylase-selection<br>
One microgram of each of the primary plasmid libraries was challenged by digestion with ~8 units of CspCI at 37°C for 1 hr. The digestions were transformed back into ER2683 and plated for<br>
20	survivors. Approximately 500 ApR survivors arose from the Bglll-<br>
library, and 5, 29, and 20 from the BamHI-, Sau3AI-, and Apol-libraries, respectively. Plasmids from BamHI, Sau3AI and Apol survivors was prepared individually using the Compass Mini Plasmid Kit method, and subjected to CspCI-digestion. 3 of the 20<br>
25	clones from the Apol-library were found to be resistant to CspCI,<br>
but all those from BamHI- and Sau3AI-libraries were found to be sensitive. The survivors from the Bglll-library were pooled and used to prepare a secondary plasmid library. This was challenged<br><br>
WO 2005/094516<br><br>
PCT7US2005/009824<br><br>
-37-<br>
again with CspCI and plated, and among the survivors several additional CspCI-resistant clones were found.<br>
5. Identification of the cspCI-R-M endonuclease-<br>
5	methyltransferase gene, and the cspCI-S specificity gene<br>
The nt sequence of the inserted DNA in the CspCI-resistant plasmid clones was determined by dideoxy automated sequencing. Transposon-insertion into clone Apol #3, using the GPS-1 System<br>
10	(New England Biolabs, Inc., Beverly, MA), provided the initial<br>
substrates for sequencing, and primer-walking was used subsequently, on clones Apol #3 and #12, and Bglll #2 and #17, to finalize the sequence. A total of 4616 bp was determined (Figure 6), within which two complete open reading frames (ORFs) of 1899<br>
15	bp (nt 1604-3502), and 960 bp (nt 3489-4448) were found (Figure<br>
7). The two ORFs have the same orientation and overlap by 14 bp (Figure 8). Analysis of the ORFs indicated that the larger, termed cspCI-R-M, encodes a combined restriction-and-modification enzyme, R-M-CspCI, and the smaller, termed cspCI-S, encodes a<br>
20	DNA-sequence-specificity protein, S-CspCI (Figure 9). R-M-CspCI is<br>
predicated to be 632 aa in length and to have a molecular mass of 70,712 Daltons (or 631 aa and 70,580 Daltons, without the N-terminal fMet). S-CspCI is predicted to be 319 aa in length and to have a molecular mass of 35,267 Daltons (318 aa and 35,136<br>
25	Daltons without the fMet). Both proteins are necessary for CspCI<br>
restriction endonuclease activity.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-38-<br>
R-M-CspCI appears to comprise a DNA-cleavage catalytic<br>
moiety joined to a DNA-methylation catalytic moiety. Amino acids<br>
2-300, the N-terminal half of R-M-CspCI, more-or-less, are<br>
believed to form an endonuclease domain, and to be responsible,<br>
5	primarily, for DNA strand-cleavage activity of CspCI. This section<br>
includes the aa sequence motif ...PE-X15-ECK... (aa 57-76), a motif found at the catalytic site of numerous DNA-endonucleases, and likely therefore to be the endonuclease catalytic site of CspCI. Amino acids 301-632 of R-M-CspCI, the C-terminal half of the<br>
10	protein, are believed to form a methyltransferase domain, and to<br>
be responsible, primarily, for DNA-modification. This section includes several aa sequence motifs characteristic of the gamma-class of DNA-adenine methyltransferases including ...VLTP... (aa 325-328), ...VLDICAGTGGF...  (SEQ ID NO:26) (aa 347-357), and<br>
15	...NPPY... (aa 435-438). On the basis of this, CspCI is predicted to<br>
accomplish modification by methylating adenine residues within its recognition sequence. Symmetry considerations suggest that the bases modified are the second A in the top strand (left subsequence), and the only A in the bottom strand (right sub-<br>
20	sequence), thus:<br>
5'...CAA N5 GTGG...3'    -&gt;     5'...CAA N5 GTGG...3' (SEQ ID NO: 14)<br>
3'...GTT N5 CACC...5'	3'...GTT N5 CACC...5'<br>
25<br>
R-M-CspCI displays substantial homology to the fused R-M subunit of the Bcgl restriction enzyme, and to several similar putative R-M-subunits in Genbank.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-39-<br>
S-CspCI also appears to be a fusion protein. In this case, the<br>
two sections are similar in sequence and function, and are believed<br>
to confer upon CspCI the ability to bind to the two specific<br>
5	components of its recognition sequence. S-CspCI is analogous to,<br>
and indeed weakly homologous to, the specificity subunits of type I R-M systems. Amino acids 2-168, the N-terminal half of S-CspCI, more or less, are believed to form one target-recognition domain (TRD), likely the one responsible for binding to the left, 5'-CAA-3',<br>
10	component of the recognition sequence. Amino acids 169-319 are<br>
believed to form the other TRD, and likely binds the other, 5'-CCAC-3' component. These two TRDs display considerable homology to each other, and consequently S-CspCI contains several internal repeated sequences. Among these is the proximal<br>
15	repeat INDLF (aa 4-8) and LQDLF (aa 172-176), and the distal<br>
repeat PDAYQGVRS (aa 144-152) and POWDFMEKY (aa 300-308). Similar repeats occur within other specificity proteins, and perhaps mediate in the binding between the S-subunit and R-M-subunit. S-CspCI displays substantial homology to the specificity subunit of<br>
20	Bcgl, and to several similar putative specificity subunits in<br>
Genbank.<br>
6. Characterization of the cloned CspCI endonuclease<br>
25	CspCI restriction endonuclease purified according to example<br>
1, above, was subjected to SDS-polyacrylamide gel electrophoresis and found to comprise two proteins of approximately 70 kDa and 35 kDa. High-pressure liquid chromatography of the same sample<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-40-<br>
demonstrated that the 70kDa and 35kDa proteins occurred in the<br>
mass ratio of 1:0.47, implying a molar ratio of 1:1.06. We take this<br>
to indicate that CspCI purifies as, and likely is active as, a<br>
heterodimer comprising one large subunit (R-M-CspCI) and one<br>
5	small subunit (S-CspCI).<br>
N-terminal sequence analysis of the isolated large subunit indicated that it began with the probable amino acid sequence, ANERKTEELV (SEQ ID NO:27). The initial codons of the CspCI-R-M<br>
10	ORF specify almost the same sequence: MANERKTESLV<br>
(SEQ ID NO:28). This result confirms that the large subunit is encoded by the CspCI-R-M ORF; that its translation begins at the predicted ATG at nt 1604; and that the initiating fMet is likely absent in the mature protein. N-terminal analysis of the isolated<br>
15	small subunit indicated that it began with the probable amino acid<br>
sequence, PKINDLFHL.E (SEQ ID NO:29). The initial codons of the cspCIS ORF specify almost the same sequence: MPKINDLFHLE (SEQ ID NO:30). This result confirms that the small subunit is encoded by the CspCI-S ORF; that its translation begins at the<br>
20	predicted ATG at nt 3489; and that its initiating fMet is also likely<br>
absent from the mature protein.<br>
7. Establishing the cleavage site of CspCI<br>
25	The endonuclease CspCI was found to cleave PhiX174 DNA<br>
twice, producing fragments of approximately 3300 bp and 2050 bp. The locations of the cut sites were mapped to approximate positions of nt 1575 and nt 4875 by simultaneously digesting<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-41-<br>
PhiX174 DNA with CspCI and with additional restriction<br>
endonucleases which cleave at known positions, such as PstI,<br>
Sspl, Neil, and StuI (Figure 1). CspCI did not cut pBR322 DNA<br>
or pUC19 DNA. The approximate size of the DNA fragments<br>
5	produced by CspCI digestion of phage lambda DNA (18 kb, 11<br>
kb, 8.3 kb, 5.1 kb, 4.3 kb and 1.8 kb) were entered into the program REBPredictor, which can be accessed at http://taQ.neb.com/~vincze/REBpredictor/index.php<br>
10	REBPredictor uses the algorithm of Gingeras, et al. Nucl.<br>
Acids Res. 5:4105 (1978), to predict potential recognition sequences by comparing observed fragment sizes with those produced by cleaving the DNA in silico at any given recognition pattern. One predicted potential pattern computed was 5'-<br>
15	CCACNNNNNTTG-3' [SEQ ID NO:31] (or 5'-CAANNNNNGTGG-3'<br>
[SEQ ID NO: 14] on the complementary strand), which occurs in PhiX174 DNA at positions consistent with the mapping data obtained, i.e. at positions 1563 and 4866. This sequence does not occur in pBR322 or pUC19 DNA. The size of fragments<br>
20	predicted from cleavage at 5'-CAANNNNNGTGG-3'<br>
(SEQ ID NO: 14) sites in PhiX174, T7 and phage lambda DNAs matched the observed size of fragments from the actual cleavage of these DNAs with CspCI. From these results we conclude that CspCI recognizes the sequence 5'-<br>
25	CAANNNNNGTGG-3'(SEQID NO:14).<br>
The positions of cleavage at the CspCI recognition sequence were determined by dideoxy sequencing analysis of<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-42-<br>
the terminal base sequence obtained from CspCI-cleavage of a<br>
suitable DNA substrate, and by comparing the lengths of the<br>
CspCI-cleavage products of a labeled DNA to a sequence ladder<br>
made from the same primer-template pair (Sanger, et al., PNAS<br>
5	74:5463-5467 (1977); Brown, et al., J. Mol. Biol. 140:143-148<br>
(1980)). By the above referenced methods, it was found that CspCI, like several other endonucleases including Bcgl, BsaXI, Cjel and HaelV, cleaves on both sides of its recognition sequence. Our observations suggest that the position of<br>
10	cleavage can vary by one base-pair on either side, being either<br>
5'-Nll/N13-CAANNNNNGTGG-N13/Nll-3' (SEQ ID NO:32), or 5'-N10/N12-CAANNNNNGTGG-N12/N10-3' (SEQ ID NO:33) or 5'-N10/N12-CAANNNNNGTGG-N13/Nll-3' (SEQ ID NO:34) or 5'-Nll/N13-CAANNNNNGTGG-N12/N10-3' (SEQ ID NO:35).<br>
15	While not wishing to be limited, by theory, we believe the<br>
enzyme cuts at a certain distance from the recognition sequence, and that it is the degree of compactness of the DNA within this span that determines whether this results in cutting at 11/13 or 10/12 base pairs.<br>
20<br>
Example V:  Expression of CspCI endonuclease in E. coli<br>
The plasmid [pUC19-CspCI-R-M-S Apol #3] was transferred<br>
25	into ER2683,and plated on ApR plates at 37°C overnight. Several<br>
individual colonies were inoculated into 50 ml LB+ApR and grown at 37°C overnight. All clones expressed CspCI endonuclease activity at &gt;105 u/g per gram of wet E. coll cells. While the<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-43-<br>
pUC19-CspCI-R-M-S Apol contains all three domains (cleavage,<br>
methylase and specificity moieties) of the endonuclease on a single<br>
plasmid for transforming a host cell, it is within the skill of one of<br>
ordinary skill in the art to place the cleavage moiety, methylase<br>
5	moiety and specificity moiety on separate plasmids or on a plurality<br>
of plasmids in which 2 out of 3 of the domains are present on a single plasmid and the third domain is on a second plasmid.<br>
The strain NEB#1554, ER2683 [pUC19-CspCI-R-M-S Apol<br>
10	#3] has been deposited under the terms and conditions of the<br>
Budapest Treaty with the American Type Culture Collection on March 24, 2004 and received ATCC Accession No. PTA-5887.<br>
15	Example VI:  Engineering Variants of CSPCI<br>
CspCI offers a variety of engineering opportunities stemming from its modular organization.<br>
20	The specificity subunit of CspCI has a duplicated<br>
organization that includes a pair of autonomous sequence-selection domains. The domains occur as direct repeats within the linear amino acid sequence, but they adopt reverse orientations in the folded protein to match the anti-parallel<br>
25	organization of double-strand DNA. One domain of S-CspCI is<br>
selective for 5'-CAA in dsDNA, and the other for 5'-CCAC; the two domains are separated by about 15 angstroms in the<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-44-<br>
subunit so that as a whole it recognizes 5'-CAANNNNNGTGG<br>
(SEQ ID NO: 14) in dsDNA. While not wishing to be limited by<br>
theory, it is proposed that actual binding to this sequence<br>
involves cooperation between the S-CspCI and the<br>
5	methyltransferase domain of R-M-CspCI, the one sequence-<br>
specific, the other non-specific. Alterations introduced into S-CspCI can change the sequence it recognizes in the same ways they have been shown to do in type I R-M systems:<br>
10	The separation between sequence selection domains and<br>
alteration in the length of the non-specific interval in the recognition sequence can be achieved by introducing changes in the 'spacer' region.  Examples of such changes include insertions such as small duplications (e.g. to CAA N6 GTGG [SEQ ID<br>
15	NO:36]) for increased length or deletions to reduce length (e.g.<br>
to CAA N4 GTGG [SEQ ID NO:37]).<br>
Various approaches exemplified below are used to alter the specificity of CspCI.<br>
20<br>
			(a) The recognition sequence of the endonuclease can be<br>
                  altered by tandemly duplicating one of the two specificity<br>
                  domains. In this way, the specificity domain is transformed from<br>
                  recognizing an asymmetric recognition site to recognizing a<br>
25	symmetrical recognition site (e.g. CAA Ns TTG<br>
[SEQ ID NO:38] or CCAC N5 GTGG [SEQ ID NO:39]). This is<br><br>
WO 2005/094516<br><br>
PCT7US2005/009824<br><br>
-45-<br>
accomplished without physically joining the domains in a single polypeptide chain where dimerization of the tandem repeat can occur spontaneously.<br>
5	(b) Amino acid changes can be introduced within either<br>
domain to alter the sequence selected by that domain, resulting in altered specificity and causing nucleotide discrimination to be diminished (e.g. CAA N5 GTGR [SEQ ID NO.-40]), or lost (e.g. CAA N5 GTG [SEQ ID NO:41]).<br>
10	Amino acid changes in the S-subunit within the regions flanking<br>
the sequence-selection domains are expected to abolish cleavage on both sides of its recognition sequence. The ability of the R-M-subunit to bind to the S-subunit in either orientation can be modified to limit its binding to a single orientation.<br>
15	Accordingly, CspCI, or a variant, may be transformed into an<br>
endonuclease that cleaves unilaterally, on only one side of its recognition sequence.<br>
(C) Swaps between the sequence-selection domains of S-<br>
20	CspCI and those of other type IIG enzymes is expected to<br>
                   generate chimeric S-subunits with hybrid specificities. A protein<br>
                  comprising the N-terminus of S-CspCI (recognition sequence<br>
                  CAA N5 GTGG) (SEQ ID NO: 14) and the C-terminus of, for<br>
                   example, S-Bcgl (recognition sequence CGA N5TGC)<br>
25	(SEQ ID NO:42), when combined with R-M-CspCI may result in<br>
an endonuclease that recognizes CAA N5 TGC (SEQ ID NO:43). For example, N- and C-terminal domains are expected to be<br><br>
WO 2005/094516	PCT/US2005/009824<br>
-46-<br>
interchangeable to create combinations of two C-terminal<br>
domains or two N-terminal domains. In this way, the C-terminal<br>
domains of S-CspCI and S-Bcgl, together will recognize<br>
GCA N5 GTGG (SEQ ID NO:44). In some Type IIG enzymes,<br>
5	such as HaelV, Alol, and Cjel, the specificity domain(s) are<br>
fused at the C-terminus of the combined R-M-S protein. These can also be swapped into S-CspCI.<br>
Sequence-specificity modules are abundant in nature,<br>
10	occurring both as individual proteins and as domains within<br>
composite proteins. Coupling these specificity modules to an endonuclease catalytic: site will create endonucleases with new specificities.<br>
15	Examples of specificity domains from class IIG restriction<br>
enzymes that may be used to replace the N- and the C-terminal domains of S-CspCI are as follows:<br>
Bcgl (New England Biolabs, Inc., Beverly, MA)<br>
20	CGANNNNNNTGC (SEQ ID NO:45)<br>
Bael (New England Biolabs, Inc., Beverly, MA) ACNNNNGTAYC (SEQ ID NO:46) <br>
25<br>
BglI  (Fermentas GmbH, Vilnius, Lithuania) GAGNNNNNCTC (SEQ ID NO:47)<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-47-<br>
Cjel, CCANNNNNNGT (SEQ ID N0:48) from Camylobacter jejuni    (Vitor, J.M.B., Morgan, R.D. Gene 157: 109-110 (1995)).<br>
5<br>
Alol (Fermentas GmbH, Vilnius, Lithuania) GAACNNNNNNTCC (SEQ ID NO:49)<br>
HaelV (Piekarowicz, A., et al. J. Mol. Biol. 293: 1055-1065<br>
10	(1999)) GAYNNNNNRTC (SEQ ID NO:50)<br>
BsaXI (New England Biolabs, Inc., Beverly, MA) ACNNNNNCTCC (SEQ ID NO:51)<br>
15<br>
In addition to the above, Type I specificity proteins are a<br>
rich potential source of specificity-domains for domain-swaps<br>
with S-CspCI. The sequence-selection domains of S-CspCI bear<br>
some homology to those of the specificity subunits of Type I R-M<br>
20	systems. Hundreds of generally uncharacterized type I S-<br>
subunits can be found in Genbank. These proteins interact<br>
naturally with Type I modification subunits, which belong to the<br>
same gamma-class, of DNA-adenine methyltransferases as R-M-<br>
CspCI and can be used as specificity domains for domain swaps.<br>
25<br>
The C-terminal section of stand-alone gamma-class DNA-adenine methyltransferases is thought to act as a sequence-selection domain, conveying to the otherwise indiscriminate<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-48-<br>
catalytic site a particular nt sequence to be methylated. These<br>
methyltransferases, some solitary, others from Type II and Type<br>
IIS R-M systems, abound in nature. Over one hundred have<br>
been characterized and many more uncharacterized examples<br>
5	can be found in Genbank. In general, these enzymes recognize<br>
                  continuous nt sequences. Most recognize symmetric sequences 4<br>
                  to 6 nt in length; others recognize asymmetric sequences of up<br>
                  to 7 nt. These stand-alone methyltransferases also represent a<br>
                 rich potential source of specificity-domains for domain-swaps<br>
10	with S-CspCI. CspCI endonuclease variants with recognition<br>
sequences of considerable length could be assembled from these enzymes.<br>
Type I S-proteins interact naturally with Type I<br>
15	modification (M) subunits, forming trimers of composition<br>
                 2M:1S. These trimers binds specifically to the sequences<br>
                 selected by the S-subunits and subsequently catalyze their<br>
                  methylation. Type I M-subunits are homologous to the C-<br>
                  terminal, methyltransferase, domain of R-M-CspCI, but they lack<br>
20	the N-terminal portion of this protein that forms the<br>
                  endonuclease domain. CspCI can be used to endow<br>
                  endonuclease activity on type I modification enzymes by<br>
                 transferring an endonuclease domain from R-M-CspCI to a type I<br>
                 M-subunit-a 'domain graft'. This will cause the Type I<br>
25	methyltransferase to cleave DNA as well as to modify it.<br><br>
WO 2005/094516<br><br>
PCT/US2005/009824<br><br>
-49-<br>
This experimental approach of grafting the endonuclease<br>
domain of R-M-CspCI to the front of a Type I methyltransferase<br>
can be applied to other stand-alone methyltransferases to cleave<br>
at sequences that originally were only modified. For example,<br>
5	the N-terminus cleavage domain of R-M-CspCI which is a<br>
gamma-class DNA adenine methyltransferase can be transferred to other gamma-class DNA adenine methyltransferases.<br><br>
50WE CLAIM:<br>
1.	A method of making a Type II restriction endonuclease having an<br>
altered specificity; comprising:<br>
(a)	selecting a restriction endonuclease characterized by a modular structure having a specificity subunit and a catalytic subunit, the specificity subunit further comprising an N-terminal domain for binding one half site of a bipartite recognition sequence and a C-terminal domain for binding a second half site of the bipartite recognition sequence;<br>
(b)	modifying the specificity subunit; and<br>
(c)	obtaining the Type II restriction endonuclease with altered specificity.<br><br>
2.	A method according to claim 1, wherein the restriction endonuclease is selected from a set of enzymes having a modular structure comprising a specificity subunit and a catalytic subunit, the specificity subunit further comprising an N-terminal domain for binding one half site of a bipartite recognition sequence and a C-terminal domain for binding a second half site of the bipartite recognition sequence.<br>
3.	A method according to claim 1, wherein modifying the specificity subunit in step (b) further comprises substituting the N-terminal domain with a second C-terminal domain or substituting the C-terminal domain with a second N-terminal domain.<br>
4.	A method according to claim 1, wherein modifying the specificity subunit further comprises substituting the N-terminal domain or the C-terminal domain or both N-terminal and C-terminal domain with a binding domain from a second restriction endonuclease or methyltransferase.<br>
5.	A method according to claim 1, wherein modifying the specificity subunit further comprises mutating the N-terminal domain, the C-terminal domain or both domains to alter the binding specificity.<br><br>
51<br>
6.	A method according to claims 1-5, wherein modifying the specificity subunit further comprises changing the length of the spacer amino acid sequence between the N-terminal and C-terminal domains of the specificity module.<br>
7.	A method according to claim 4, wherein the second restriction endonuclease or methyltransferase is selected from a group consisting of a Type I restriction endonuclease, a Type IIG restriction endonuclease and a γ-type m6A methyltransferase.<br>
8.	A method according to claim 1, wherein the specificity subunit and the catalytic subunit are encoded by different genes.<br>
9.	A substantially pure Type IIG restriction endonuclease obtainable from Citrobacter species 2144 (NEB#1398) (ATCC Patent Accession No. PTA-5846) or from Escherichia coli NEB#1554 (ATCC Patent Accession No. PTA-5887) capable of recognizing at least one sequence selected from the group consisting of SEQ ID NO:32, SEQ ID NO:33, SEQ ID NO:34 and SEQ ID NO:35, and cleaving the DNA on both sides of the recognition sequence.<br>
10.	An isolated DNA encoding CstMI restriction endonuclease obtainable from Escherichia coli NEB#1554 (ATCC Patent Accession No. PTA-5887) or from Citrobacter species 2144 (NEB#1398) (ATCC Patent Accession No. PTA-5846).<br>
11.	Isolated DNA encoding the restriction endonuclease of claim 1, wherein the DNA comprises a first DNA segment encoding an endonuclease and methyl transferase catalytic function and a second DNA segment encoding a sequence specificity function of the restriction<br><br>
endonuclease wherein the first and second DNA segments comprise one or more DNA molecules.<br>
12.	A recombinant DNA vector, comprising: at least one of a first DNA segment coding for the restriction and modification domains of CspCI restriction endonuclease and a second segment coding for the specificity domain of the restriction endonuclease.<br>
13.	A host cell transformed with a first DNA segment coding for the restriction and modification domains of CspCI restriction endonuclease and a second segment coding for the specificity domain of the restriction endonuclease wherein the first DNA segment and the second DNA segment are contained within one or more DNA vectors.<br>
14.	A method for obtaining the endonuclease of claim 9, comprising cultivating a sample of Citrobacter species 2144 (NEB#1398) or a host cell according to claim 6 under conditions favoring the production of the endonuclease; and purifying the endonuclease therefrom.<br>
15.	A method of making a Type II restriction endonuclease, a substantially pure Type IIG restriction endonuclease, an isolated DNA encoding, a recombinant DNA vector, a host cell and a method, substantially as hereindescribed with reference to the accompanying examples and figures.<br>
Dated this   SK  day of October, 2006.<br>
[RAJESHWARI H.]<br>
OF K &amp; S PARTNERS<br>
ATTORNEY FOR THE APPLICANT<br><br>
ABSTRACT<br>
Title: "A novel modular type II restriction endonuclease, CspCI, and the use of modular endonucleases for generating endonucleases with new specificities"<br>
A novel restriction endonuclease and methods of making the same are obtainable from either Citrobacter species 2144 or the recombinant stain Escherichia coli which leaves at nt sequence 5'=CAANNNNNGTGG-3' (SEQ ID NO: 14) in double-stranded DNA molecules. The novel restriction endonuclease is a modular protein in which the specificity moiety is an independent module from the restriction-modification module.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUFCU1RSQUNUKDI5LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-ABSTRACT(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUFCU1RSQUNUKDYtMTAtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-ABSTRACT(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUFCU1RSQUNUKEdSQU5URUQpLSgxMC0xLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-ABSTRACT(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWFic3RyYWN0LmRvYw==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNBTkNFTExFRCBQQUdFUygyOS0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CANCELLED PAGES(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyg2LTEwLTIwMDYpLTEucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(6-10-2006)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyg2LTEwLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyhBTUVOREVEKS0oMjMtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(AMENDED)-(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyhBTUVOREVEKS0oMjktMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(AMENDED)-(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyhHUkFOVEVEKS0oMTAtMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyhNQVJLRUQgQ09QWSktKDIzLTMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(MARKED COPY)-(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNMQUlNUyhNQVJLRUQgQ09QWSktKDI5LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CLAIMS(MARKED COPY)-(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWNvcnJlc3BvbmRlbmNlIHJlY2VpdmVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-correspondence received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNPUlJFU1BPTkRFTkNFKDIxLTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CORRESPONDENCE(21-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWNvcnJlc3BvbmRlbmNlKDIyLTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-correspondence(22-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNPUlJFU1BPTkRFTkNFKDQtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CORRESPONDENCE(4-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUNPUlJFU1BPTkRFTkNFKElQTyktKDEwLTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-CORRESPONDENCE(IPO)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LURFU0NSSVBUSU9OKENPTVBMRVRFKS0oNi0xMC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-DESCRIPTION(COMPLETE)-(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LURFU0NSSVBUSU9OKEdSQU5URUQpLSgxMC0xLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-DESCRIPTION(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LURSQVdJTkcoNi0xMC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-DRAWING(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LURSQVdJTkcoR1JBTlRFRCktKDEwLTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-DRAWING(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0gMSgyMi0zLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form 1(22-3-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMSgyOS0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 1(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0gMTMoMjMtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form 13(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMTMoMjktMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 13(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0gMTgoMjItMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form 18(22-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMihDT01QTEVURSktKDYtMTAtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 2(COMPLETE)-(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMihHUkFOVEVEKS0oMTAtMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 2(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMihUSVRMRSBQQUdFKS0oMjktMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 2(TITLE PAGE)-(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMihUSVRMRSBQQUdFKS0oNi0xMC0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 2(TITLE PAGE)-(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDEwLTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 2(TITLE PAGE)-(GRANTED)-(10-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMjYoMjEtMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 26(21-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMjYoMjMtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 26(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0gMygyMi0zLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form 3(22-3-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LUZPUk0gMygyOS0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-FORM 3(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0tMi5kb2M=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXBjdCByZXF1ZXN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-pct request.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXBjdCBzZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-pct search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXBjdC1pc2EtMjIwLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-pct-isa-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXBjdC1pc2EtMjM3LnBkZg==" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-pct-isa-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3ICgyOS0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-PETITION UNDER RULE 137 (29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgyMy0zLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-REPLY TO EXAMINATION REPORT(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgyOS0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-REPLY TO EXAMINATION REPORT(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXNlcXVlbmNlIGxpc3RpbmcoMjMtMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-sequence listing(23-3-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LVdPIElOVEVSTkFUSU9OQUwgUFVCTElDQVRJT04gUkVQT1JUIEEzKDI5LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-WO INTERNATIONAL PUBLICATION REPORT A3(29-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1tdW1ucC0yMDA2LXdvIGludGVybmF0aW9uYWwgcHVibGljYXRpb24gcmVwb3J0KDIyLTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1195-mumnp-2006-wo international publication report(22-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTE5NS1NVU1OUC0yMDA2LVdPIElOVEVSTkFUSU9OQUwgUFVCTElDQVRJT04gUkVQT1JUKDYtMTAtMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">1195-MUMNP-2006-WO INTERNATIONAL PUBLICATION REPORT(6-10-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="250569-compositions-containing-flourine-substituted-olefins.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250571-method-for-recovering-molybdenum-and-method-for-producing-catalyst.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250570</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1195/MUMNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jan-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Oct-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NEW ENGLAND BIOLABS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>240 County Road, Ipswich, MA 01938,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>Celine Nguefue Nkenfou</td>
											<td>UNIVERSITY OF YAOUNDE 1, MOLECULAR BIOLOGY CENTER, P.O.BOX 14495, YAOUNDE, CAMEROON</td>
										</tr>
										<tr>
											<td>2</td>
											<td>Stephen Picone</td>
											<td>8 Princeton Avenue, Beverly, MA 01915,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>RICHARD MORGAN</td>
											<td>31 Donovan&#x27;s Way, Middleton, MA 01949,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>Geoffrey Wilson</td>
											<td>23 Cutler Road, South Hamilton, MA 01982,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>Keith Lunnen</td>
											<td>20 Winthrop Street, Essex, MA 01929,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>Daniel Heiter</td>
											<td>87 Gardner Street, Groveland, MA 01834,</td>
										</tr>
										<tr>
											<td>7</td>
											<td>Jack Benner</td>
											<td>101 Sagamore Street, South Hamilton, MA 01982,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N9/22 C12N15/55</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/029824</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/555,796</td>
									<td>2004-03-24</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250570-a-method-of-making-a-type-ii-restriction-endonuclease-having-altered-specificity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:21:57 GMT -->
</html>
