<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250662-integrated-nucleic-acid-analysis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:18:53 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250662:INTEGRATED NUCLEIC ACID ANALYSIS.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">INTEGRATED NUCLEIC ACID ANALYSIS.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an integrated method of nucleic acid analysis, and more particularly to a simplified sample pre-treatment, which renders the method more easily automated, where the sample is provided on or applied onto a solid matrix and the subsequent amplification and detection steps are performed in one single, sealed reaction vial without removing the matrix.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>WO 2006/000648	PCT/FI2005/050245<br>
Integrated Nucleic Acid Analysis<br>
Field of the Invention<br>
The present invention relates to an integrated method of nucleic acid analysis, and more particularly to a simplified sample pre-treatment, which renders the method more easily automated. The method of the present invention provides minimized risks related to contamination due to minimal handling of the samples.<br>
Background of the Invention<br>
As a result of the rapidly developing genomic testing, whole nucleic acid analysis is an increasing task in many genetic laboratories. The poly-merase chain reaction (PCR) is a well established method for amplifying nucleic acid sequences, and the method is routinely used in numerous application areas, such as microbiological testing, expression studies, determination of genetic variation in population, and genetic testing, forensics and food and environmental testing. Testing of nucleic acids using PCR generally involves three steps: sample preparation, amplification and detection. However, the processes for performing the tests used today are often laborious. The current trend is towards simplified assays allowing automation for nucleic acid analysis.<br>
The increased number of tests to be run raises a need for cost-effective operations based on integration and automation of the assay procedures, and the automation of the whole DNA analysis is an increasing task in many genetic laboratories. Although several attempts to perform DNA-analysis as high throughput assays has been described, the logistics of sample handling from sample preparation and pre-treatment to the ultimate analysis still requires manual handling and physical transportation of the samples. In addition, the need to physically transfer amplified DNA samples within the lab poses a serious contamination risk.<br>
One type of a simplified assay that reduces sample handling is provided by so-called closed-tube assays. In closed-tube assays, the PCR prod-uct is analyzed in the amplification tube by a homogeneous method, such as TaqMan® (US Patent No. 5210015) or by molecular beacons (US Patent No. 5118 801). These approaches allow integration of amplification and detection. However, they lack integration of sample preparation with amplification and detection.<br><br>
WO 2006/000648	PCT/F12005/050245<br>
2<br>
Pre-treatment of the sample is required to remove common inhibi-tors to nucleic acid amplification that may be present in samples from biologi-cal sources. Inhibitors to amplification include, e.g. naturally occurring chelat-ing agents, enzymes and/or proteins that can damage either nucleic acid tem-plates or PCR polymerases used in the amplification reactions. In addition, the common anticoagulants that are used to treat whole blood samples can inter-fere with nucleic acid amplification reactions.<br>
Numerous technologies have been developed to purify nucleic acids from biological samples but all available procedures are time-consuming and labour intensive. There are several automated stations for sample preparation available on the market, based on silica-chaotrop extraction (US Patent No. 5234809) columns, such as QIAamp®, or the like, and various magnetic bead systems. A different approach is the FTA® Technology (US Patent No. 5496 562) that lyses cell membranes as soon as the sample is applied onto a coated filtration matrix allowing immobilization of nucleic acids onto the matrix. After washing, the nucleic acids can be released in a manner that enables them to be amplified by PCR.<br>
Another approach is represented by the development of different solid matrices for collecting, transporting, storing and purifying biological sam-ples, such as clinical whole blood, saliva or faecal samples, for nucleic acid analysis. US Patent No. 5 807 527 describes a solid medium for long term stor-age of blood DNA, which comprises a composition, which protects against degradation of DNA, a protein denaturing agent and a free radical trap.<br>
Another example of this approach is the described in US Patent No. 5939259, which discloses an absorbent material, which does not bind nucleic acids irreversibly which is impregnated with a chaotropic salt.<br>
In these known technologies, the DNA of stored blood samples is extracted from the medium before performing PCR, or the DNA is used in PCR in situ on the solid medium after extensive purification, as described in US Patent No. 5 807 527. Extraction or elution of DNA from the medium requires a multi-step procedure with special solutions and incubations.<br>
Known technologies using solid media for collecting, storing and pu-rifying DNA thus involve multi-step procedures, some using several separate vials and solutions, for generating a sample useful for amplification and detec-tion. Moreover, certain of these methods unnecessarily produce waste.<br><br>
WO 2006/000648	PCT/FI2OO5/O5O245<br>
3<br>
Brief Description of the Drawings<br>
Figure 1 is a graphical presentation of the result of the comparative test described in Example 1, analyzing the effect of pre-washing dried samples on collection paper.<br>
Figure 2 is a graphical presentation of the result of the test de-scribed in Example 2, where the release of DNA from dried samples, amplifica-tion and detection of amplified DNA is performed in a single reaction vial.<br>
Figure 3 is a graphical presentation of the result of the test de-scribed in Example 3, where the release of DNA from dried samples, amplifica-tion and detection of amplified DNA is performed in a single reaction vial, and where all necessary reagents were added simultaneously.<br>
Figure 4 is a graphical presentation of the result of the test de-scribed in Example 4, where a whole blood sample is applied to an absorbent matrix in the reaction vial, and amplification and detection of amplified DNA is performed in the same reaction vial.<br>
Figure 5 is a graphical presentation of the result of the test de-scribed in Example 5, where all necessary reagents were provided in dried form in the reaction vial.<br>
Brief Description of the Invention<br>
The present invention relates to a method for analyzing a target nu-cleic acid in a biological sample, comprising the following steps of i) providing an absorbent matrix in a reaction vial; ii) providing a biological sample contain-ing said target nucleic acids on or applying it onto said absorbent matrix; iii) providing in said reaction vial a reagent mixture for amplifying said nucleic ac-ids; iv) providing in said reaction vial a detection reagent mixture; v) performing an amplification reaction; and vi) detecting the amplified nucleic acids, wherein steps v) and vi) are performed in said vial without removing said matrix from the vial.<br>
More specifically the present invention relates to a method for ana-lyzing a target nucleic acid in a biological sample, wherein the amplification and detection steps are performed in sealed reaction vials without adding or removing any reagents or other components after the amplification step.<br>
The present invention further relates to a method wherein said sam-ple is applied onto the matrix immediately prior to providing the matrix into said reaction vial, without drying and/or storage.<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
4<br>
The present invention further relates to a method wherein said ma-trix is provided in the reaction vial prior to applying the sample directly onto the matrix.<br>
The present invention further relates to a method wherein at least part of said amplification reagents and detection mixtures are provided in dried form prior to the addition of the sample, and said method including an optional further step of adding water prior to performing the amplification step.<br>
Detailed Description of the Invention<br>
The present invention provides an integrated method for analyzing nucleic acids contained in a biological sample, comprising a simplified sample pre-treatment, amplification of the target nucleic acid to be analyzed, and de-tection of said target sequence. Due to the simplified sample pre-treatment, the method allows automation and integration of sample preparation, amplifica-tion and analysis of nucleic acids.<br>
The present invention is based on the surprising finding that the amplification and detection steps may be performed on samples collected and, optionally, stored on solid media without prior extraction, washing or elution of the nucleic acids. Only a short incubation to inactive inhibitors and to release the nucleic acids is needed.<br>
Inactivation of inhibitors and denaturation of disturbing proteins is achieved by use of solid matrices based on any porous or absorbing material, e.g. cellulose based filter paper or wadding, cotton wad or fabric, or synthetic plastic material. Suitable matrices for use in the method according to the pre-sent invention include absorbent materials which bind proteinaceous material, but do not bind nucleic acids irreversibly, and which are impregnated with e.g., a base, an acid, reducing agents, chaotropic salts, detergents, or other agents to denature natural inhibitors within the sample. Solid matrix material suitable for use in the present invention may be provided as flat sheets, swabs, tablets, pellets or beads, or as a mesh or lattice.<br>
Such solid matrix materials are readily available, and include e.g. the paper matrix provided by Schleicher&amp;Schuell under the trade marks 903™ Specimen Collection paper or IsoCode® for collecting, storing, transporting and purifying DNA.<br>
In a preferred embodiment of the present invention the biological sample is provided on such an absorbent matrix. At least a portion of the ma-trix containing the sample is transferred to the reaction vial, together with re-<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
5<br>
agents for amplification and detection. The reaction vial is then sealed and moved to a device for performing the amplification reaction. The DNA of the sample is released by a brief heat-treatment. Optionally, if the reagents are not heat stable, e.g. in case of isothermal amplification, the DNA release may be performed at ambient temperature. Surprisingly, after amplification the detec-tion may be performed without opening the reaction vial or removing said ma-trix.<br>
It has also surprisingly been found, that the sample may be applied onto the matrix immediately prior to use. Thus, in a preferred embodiment of the present invention, the sample is applied onto a suitable solid matrix, whereafter at least a portion of the matrix is transferred to the reaction vial, without prior drying of the matrix. After addition of necessary reagents, the amplification and detection steps are performed.<br>
It has further, surprisingly, been found, that the solid matrix may be provided in the reaction vial and that the sample may be applied directly onto the matrix, and necessary reagents added, once the sample has been applied. Thereafter the amplification and detection steps are performed in the sealed vial.<br>
The term "amplification" is meant to include any method for amplify-ing nucleic acids known in the art, either thermal cycling methods such as po-lymerase chain reaction (PCR; US Patent 4683202), reverse transcriptase PCR, (US Patent No. 5310652), and ligase chain reaction (LCR; U.S. Patent No. 5185243), and any variations thereof, or isothermal methods, such as Q-Beta replicase technology (U.S. Patent No. 4786600), nucleic acid sequence based amplification (NASBA; U.S. Patent No. 5409818), transcription medi-ated amplification (TMA; U.S. Patent Nos. 5399491), strand displacement amplification (SDA; US Patent No. 5455166) and multiple displacement ampli -fication (MDA; US Patent No. 6124120). One preferred method of amplification is asymmetric PCR, described by Innis et al., PNAS 85(24), 1988: 9436-40, which generates single stranded amplification products.<br>
Other known amplification methods may also be useful in the inte-grated analysis method according to the present invention. The amplification reagents and the amplification procedure may thus vary according to the method of choice.<br>
The detection step is performed by any suitable detection method of choice. Thus the detection reagents may vary according to the method of<br><br>
WO 2006/000648	PCT/FI20O5/05O245<br>
6<br>
choice. Suitable detection methods are based on e.g. intercalating reagents to detect the accumulation of amplification product or to perform melting tempera-ture analysis, or labelled primers, or specific primer extension in which directly or indirectly labelled nucleotides are incorporated to the amplification product, or methods in which the directly or indirectly labelled detection probes are de-graded during extension, or hybridization with directly or indirectly labelled probes. The detection may be based on e.g. fluorescence resonance energy transfer, fluorescence quenching or environmentally sensitive labels. Suitable detection methods may also include imaging of array of spots or beads by con-focal scanning or evanescent wave. The detection step is preferably performed in a closed-tube format without removing the matrix or unreacted components, in order to avoid post-amplification handling.<br>
One preferred detection method is based on hybridization using la-belled probes that recognize the amplified target nucleic acids. In such a method, the hybridization, and thus labelling of the target nucleic acids, is achieved simply by lowering the temperature after amplification to hybridization temperature, if necessary, whereafter the un-reacted labelled probe is quenched by a complementary probe. Detection of formed labelled hybrids may be performed on the reaction vial, without removing the matrix or other unreacted components. Thus the reaction vials remain unopened after per-forming the amplification reaction.<br>
In a preferred embodiment of the present invention, the method is further simplified in a way that at least part of the reagents necessary for per-forming the amplification and detection steps, is provided in the reagent vial in dried form prior to the addition of the sample to be analyzed. Optionally, the absorbent matrix may also be provided in the reaction vial. Such a simplified, integrated assay format minimizes the manual handling, is easily automated and decreases the risk of contamination, as the reaction vials may be sealed immediately after providing the sample and, if necessary, adding water.<br>
A variety of samples can be analyzed using the methods of the in-vention for preparing a sample for molecular analysis. Such samples include bodily fluids, such as whole blood, saliva, sputum, urine, faecal, peritoneal and pleural fluids; lavations, such as bronchoalveolar, nasal, cervical and intestinal samples; aspiration or biopsy samples; cell cultures or microbial cultures. Spe-cific non-limiting examples include whole blood and saliva samples. The bio-logical samples may also be samples taken from food or environmental sam-<br><br>
WO 2006/000648	PCT/FI200S/050245<br>
7<br>
pies. In addition to complex biological samples, the method of the present in-vention is equally suitable for amplifying and analysing pre-treated, purified samples, for example samples where DNA and/ or RNA has already been ex-tracted and thereafter stored on matrix.<br>
In a preferred embodiment, the method of the present invention is carried out as follows. The biological sample, such as whole blood, is provided dried on an absorbent matrix sheet or stick, or, optionally, applied to such a matrix by the user, and is allowed to dry onto the matrix. After drying, at least a portion is transferred to the reaction vial. The amplification reagents, com-prising primers, nucleotides, polymerase and necessary buffers are added, as well as the detection mixture, comprising directly or indirectly labelled nucleo-tides or probes, and optionally secondary labels, or intercalators. The vial is then sealed and moved to a thermal cycler. The DNA of the samples is re-leased by a brief heat-treatment. The amplification proceeds after the pre-heating. After amplification, the presence and/or composition of the nucleic acid is detected through the seal on the vial without removing the matrix or even opening the sealed vials at any stage.<br>
In another, highly preferred embodiment, the method of the present invention is carried out as a multi-vial assay. The method is performed using a kit containing dried reagents for amplification and analysis, as well as an ab-sorbent matrix, either as a disk at the bottom of the vials, or as a mesh at least partly filling the vials. The sample to be analyzed is applied, without pre-treatment, directly onto the matrix in the reaction vial. Water is then dispensed into the vials and the vials are sealed and moved to a thermal cycler. The DNA of the samples is released by a brief heat-treatment. The amplification pro-ceeds after the pre-heating. After amplification, the presence and/or composi-tion of the nucleic acid is then detected through the seal of the vials, without removing the matrix. A suitable, readily available format for a kit according to the present invention is e.g. a multi-well plate.<br>
The following examples are given to further illustrate preferred em-bodiments of the present invention, but are not intended to limit the scope of the invention. It will be obvious to a person skilled in the art that, as the tech-nology advances, the inventive concept can be implemented in various ways. The invention and its embodiments are not limited to the examples described above but may vary within the scope of the claims.<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
8<br>
Examples Example 1<br>
Release of nucleic acids from dried sample disks<br>
This example shows how the elution of dried blood and saliva disks recommended by manufacturer can be simplified. The recommended pre-wash is omitted and it is shown that the release of nucleic acids can be performed effectively also at room temperature. The release is automated by use of filter plates and automated filtration.<br>
First, whole blood and saliva was collected from a volunteer. The samples were applied onto IsoCode® (Schleicher &amp; Schuell) collection paper and allowed to dry. For analysis, eight 1.5-mm disks were punched and the nucleic acids were eluted according to manufacturer's instructions, i.e. pre-wash with 500 pi of distilled water followed by incubation at + 95 °C for 30 min in 135 ul of distilled water. Another eight disks were incubated without pre-wash at + 95 °C for 30 min. After incubation, the supernatant was analysed. The automated release was demonstrated by punching a 1.5-mm disk together with 40 pi of distilled water into a vial in a 96-well filtration plate, and the sam -ples were incubated at room temperature for 30 min. After incubation, the filter plate was aligned with reaction plate and the supernatant was transferred into the reaction plate by pressure from top of the filter plate.<br>
The closed-tube PCR amplification and detection reaction was per-formed in a total volume of 50 ul and the final concentration in the reaction mixture was following: 1 x DyNAzyme™ buffer (Finnzymes), 0.2 mM dNTP's, 2.5 mM MgCI2, 0.1 uM CFTR exon 10 forward primer (51 AAG CAC AGT GGA AGA ATT TC 3"), 0.1 uM CFTR exon 10 reverse primer (5' CTC TTC TAG TTG GCA TGC T 3'), 0.02 U/ul DyNAzyme™ enzyme (Finnzymes), 20 nM analyte specific oligonucleotide probe (51 TAA AGA AAA TAT CAT CTT TGG TGT TTC CTA TAA 3') labelled at its 5' end with a stable fluorescent W14054 Tb chelate (Wallac) and 0.4 uM quencher probe complement to the Tb probe (5' ATG ATA TTT TCT TTA 3') labelled at its 3' with BHQ1 (Biosearch Tech-nologies), 10 ul of supernatant from the 95°C incubation, and as a control 0.4 ng/ul genomic DNA sample purified from a volunteer's whole blood. The su-pernatant from the automated release was analysed by adding 10 µl of the re-action mixture to the final concentration described above.<br><br>
WO 2006/000648	PCT/F12005/050245<br>
9<br>
The PCR program was as follows: Pre-heating +95 °C 1 min; 35 cy-cles of following +95 °C 30 s, ramping from +95 °C to +84 C 0.2 °C/s, ramping from +84 °C to +56 °C 2.5 °C/s, +56 °C 1 min, +66 °C 1 min and final exten-sion 7 min. After amplification, the reaction was incubated at +30 °C for 5 min, whereafter the time-resolved fluorescence was measured with Victor2 1420 Multilabel Counter (Wallac).<br>
The results of this experiment are shown in Figure 1 by the average and standard deviation of time-resolved Tb fluorescence of each reaction in duplicates. Negative and positive PCR controls are drawn by bars A and B, re-spectively. When the simplified release (D and E) is compared to elution by manufacturer's recommendation (C), it is clearly seen that the analysis can be performed with blood (black bars) and saliva (grey bars) disks without pre-wash both at room temperature (D) and at +95 C (E). The result from the su-pernatants is also comparable to the estimated amount of genomic DNA per blood disk.<br>
Example 2<br>
DNA analysis from whole blood and saliva<br>
This example shows how the pre-treatment of blood and saliva samples can be simplified to enable simple automation. The sample is col-lected onto protein denaturing matrix, and the release of nucleic acids from the matrix, amplification and detection are all performed in a single reaction vial. After release, the amplification and detection reagents are added and the reac-tion vial is sealed, and after amplification and short analytical incubation, the fluorescence is measured without opening the reaction vial, i.e., without re-moval of the matrix.<br>
First, whole blood and saliva was collected from two volunteers. The samples were applied onto IsoCode® (Schleicher &amp; Schuell) collection paper and allowed to dry. For analysis, a 1.2-mm blood disk or two 1.2-mm saliva disks were punched and the nucleic acids were released at room temperature for 30 min in 40 µl of distilled water. The release was performed in PCR vials and as a reference in separate micro tubes. After release, the PCR/detection reaction mixture was added into the PCR vial comprising the disk. As refer-ence, the 40-µl supernatant from the separate tubes was analysed.<br>
The closed-tube PCR amplification and detection reaction was per-formed in a total volume of 50 µl. A 10-µl portion of following PCR/detection<br><br>
WO 20(16/000648	PCT/F12005/050245<br>
10<br>
reaction mixture was added into the PCR: 5 x DyNAzyme™ buffer (Finn-zymes), 1 mM dNTP's, 12.5 mM MgCI2, 1 uM CFTR exon 10 forward primer (5' AAG CAC AGT GGA AGA ATT TC 3'), 0.25 uM CFTR exon 10 reverse primer (5' CTC TTC TAG TTG GCA TGC T 31), 0.1 U/ul DyNAzyme™ enzyme (Finnzymes), 83 nM analyte specific oligonucleotide probe (5' ACC AAA GAT-GAT ATT TAA A 3') labelled at its 5' end with a stable fluorescent W8184 Eu chelate (Wallac) and 166 nM quencher probe complement to the Eu probe (5' TCA TTG GTG TTT 3') labelled at its 3' with BHQ1 (Biosearch Technologies). As a control, 20 ng/reaction of genomic DNA sample purified from a volun-teer's whole blood was analysed.<br>
The PCR program was as follows: Pre-heating 95 °C 1 min; 35 cy-cles of following +95 °C 30 s, +56 °C 1 min, +66 °C 1 min, final extension 7 min and final denaturation 8 min. After amplification, the reaction was incu-bated at +40 °C for 20 min and at +22 °C for 15 min, whereafter the time-resolved fluorescence was measured with Victor2 1420 Multilabel Counter (Wallac) directly from the unopened vial.<br>
The results of this experiment are shown in Figure 2 by the average and standard deviation of time-resolved Eu fluorescence of each reaction in triplicates. Negative and positive PCR controls are drawn by bars A and B, re-spectively. C-D represent results from blood and E-F from saliva samples ob-tained from 2 volunteers. When the integrated release in the reaction vial with-out removal of matrix at any point (hatched bars) is compared to release in a separate via! (grey bars), it is clearly seen that the whole analysis can be per-formed with blood disk and saliva disks without removal of disk. The result shows ihat the release from the matrix can be performed at room temperature in the reaction vial and after amplification the detection can be performed with-out removal of matrix enabling simple automation of whole analysis.<br>
Example 3<br>
Integrated DNA analysis from whole blood and saliva<br>
This example shows how the release of nucleic acids from matrix can be further simplified to enable simple automation. The sample is collected onto protein denaturing matrix, and the release of nucleic acids, amplification and detection are all performed in a single reaction vial even in the presence of amplification and detection reagents. The matrix comprising the sample is added together with the amplification/detection reagents into a reaction vial,<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
11<br>
the vial is sealed, and after amplification and short analytical incubation, the fluorescence is measured without opening the reaction vial, i.e. without re-moval of the matrix.<br>
First, whole blood and saliva was collected from two volunteers. The samples were applied onto IsoCode® (Schleicher &amp; Schuell) collection paper and allowed to dry.<br>
For analysis, a 1.2-mm blood disk or two 1.2-mm saliva disks were punched into PCR vials. The closed-tube PCR amplification and detection re-action was performed in a total volume of 50 ul in following PCR/detection re-action mixture: 1 x HotStarTaq® buffer (Qiagen), 0.2 mM dNTP's, 2.5 mM MgCI2, 0.05 % BSA, 0.2 uM CFTR exon 10 forward primer (51 AAG CAC AGT GGA AGA ATT TC 3'), 0.05 uM CFTR exon 10 reverse primer (51 CTC TTC TAG TTG GCA TGC T 3'), 0.02 U/ul DyNAzyme™ enzyme (Finnzymes), 17 nM analyte specific oligonucleotide probe (5' ACC AAA GAT GAT ATT TAA A 3') labelled at its 5' end with a stable fluorescent W8184 Eu chelate (Wallac) and 33 nM quencher probe complement to the Eu probe (5' TCA TTG GTG TTT 3J) labelled at its 3' with BHQ1 (Biosearch Technologies). As a control, 20 ng/reaction of a volunteer's genomic DNA sample on disk was analysed. As a reference, the release of nucleic acids was performed at room temperature for 30 min in 40 ul of distilled water, and analysed in the closed-tube PCR/detection after addition of a 10-ul portion of reaction mixture as 5 x con-centrate.<br>
The PCR program was as follows: Pre-heating +95 °C 15 min; 35 cycles of following +95 °C 30 s, +56 °C 1 min, +66 °C 1 min, final extension 7 min and fina! denaturation 8 mih. After amplification, the reaction was incu-bated at +40 °C for 20 min and at +22 °C for 15 min, whereafter the time-resolved fluorescence v.as measured with Victor2 1420 Multilabe! Counter (Wallac).<br>
The results of this experiment are shown in Figure 3 by the average and standard deviation of time-resolved Eu fluorescence of each reaction in triplicates. Negative and positive PCR controls are dawn by bars A and B, re-spectively. C-D represent results from blood and E-F from saliva samples ob-tained from 2 volunteers. When the release in amplification/detection reagents (grey bars) is compared to release in distilled water (white bars), the result is nearly equal with both methods. The result shows that release from the matrix can be performed even in the amplification/detection mixture, and the whole<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
12<br>
process can be performed in a single vial in a closed-tube format enabling simple automation of whole analysis.<br>
Example 4<br>
Integrated sample pre-treatment and DNA analysis from whole blood<br>
This example shows how the matrix comprising the sample can be used in analysis right after the application of the sample enabling development of amplification/detection devices comprising the matrix in the reaction vial. The sample is applied onto the protein denaturing absorbent matrix in the reac-tion vial, the amplification/detection reagents are added, the vial is sealed, and after amplification and short analytical incubation, the fluorescence is meas-ured without opening the reaction vial.<br>
The closed-tube PCR amplification and detection reaction was per-formed in a total volume of 50 µl. First, a 1.2-mm disk from IsoCode® (Schlei-cher &amp; Schuell) collection paper was punched into PCR vials. Then, a 0.35-µl sample from a volunteer's whole blood was applied into the vial. After various time points, a 50-pl portion of PCR/detection reaction mixture (as follows) was added: 1 x Phusion™ HF buffer (Finnzymes), 0.2 mM dNTP's, 2.5 mM MgCI2, 0.02 % BSA, 0.2 pM CFTR exon 10 forward primer (5' AAG CAC AGT GGA AGA ATT TC 3'), 0.05 pM CFTR exon 10 reverse primer (5' CTC TTC TAG TTG GCA TGC T 3'), 0.02 U/µl DyNAzyme™ enzyme (Finnzymes), 17 nM analyte specific oligonucleotide probe (5' ACC AAA GAT GAT ATT TAA A 3') labelled at its 5' end with a stable fluorescent W8184 Eu chelate (Wallac) and 33 nM quencher probe complement to the Eu probe (5' TCA TTG GTG TTT 3') labelled at its 3' with BHQ1 (Biosearch Technologies). As a control, 20 ng/reaction of a volunteer's genomic DNA sample on disk was analysed.<br>
The PCR program was as follows: Pre-heating +95 C 15 min; 35 cycles of following +95 °C 30 s, +56 °C 1 min, +66 °C 1 min, final extension 7 min and final denaturation 8 min. After amplification, the reaction was incu-bated at +40 °C for 20 min and at +22 °C for 15 min, whereafter the time-resolved fluorescence was measured with Victor2 1420 Multilabel Counter (Wallac).<br>
The results of this experiment are shown in Figure 4 by the average and standard deviation of time-resolved Eu fluorescence of each reaction in 4 replicates. Negative and positive PCR controls are drawn by bars A and B, re-spectively. C-F represent results from blood samples analysed. The signal<br><br>
WO 2006/000648	PC T/FI2005/050245<br>
13<br>
from the analysis which has been performed right after applying the sample into the amplification/detection vial comprising the matrix (F) is equal to those which have been let dry 1, 2 and 16 h (C-E, respectively). The result shows that the whole process can be performed in a single vial avoiding multiple steps from sample pre-treatment to detection allowing very simple automation.<br>
Example 5<br>
Integrated DNA analysis from whole blood using dried reagents<br>
This example shows the extreme simplification of the nucleic acid analysis process from sample pre-treatment to detection. The reagents are dried into the reaction vial, and the sample is either applied onto the protein denaturing absorbent matrix in the vial, or the matrix containing sample is pun-ched into the vial together with water. Then the vial is sealed, and after amplifi-cation and short analytical incubation, the fluorescence is measured without opening the vial.<br>
First, a 12.5-|jl portion of following amplification/detection reaction mixture was allowed to dry in the reaction vial at room temperature for 2 h: 4 x Phusion™ HF buffer (Finnzymes), 0.8 mM dNTP's, 10 mM MgCI2, 0.08 % BSA, 0.8 pM CFTR exon 10 forward primer (5' AAG CAC AGT GGA AGA ATT TC 3'), 0.2 uM CFTR exon 10 reverse primer (51 CTC TTC TAG TTG GCA TGC T 3J), 0.08 U/µl DyNAzyme™ enzyme (Finnzymes), 66 nM analyte spe -cific oligonucleotide probe (5' ACC AAA GAT GAT ATT TAA A 3') labelled at its 5' end with a stable fluorescent W8184 Eu chelate (Wallac) and 133 nM quencher probe complement to the Eu probe (5' TCA TTG GTG TTT 3') la-belled at its 3' with BHQ1 (Biosearch Technologies).<br>
Then, the reaction vial comprising the dried reagents was used in the closed-tube PCR amplification and detection reaction in a total volume of 50 µl. A 1.2-mm blood disk of previously collected volunteer's sample on Iso-Code® (Schleicher &amp; Schuell) was punched into the reaction vial with simulta-neous dispensing of 50 pi of distilled water. Or, a 1.2-mm disk from IsoCode® (Schleicher &amp; Schuell) collection paper was punched into the vial, and then, a 0.35-µl sample from a volunteer's whole blood was applied into the vial, and 50 µl of distilled water was added. Also, supernatant after release of nucleic acids in 50 µl of distilled water in a separate vial at room temperature for 30 min was used as sample in analysis. As a positive control, 20 ng/reaction of a volun-teer's genomic DNA sample on disk was analysed.<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
14<br>
The PCR program was as follows: Pre-heating +95 °C 15 min; 35 cycles of following +95 °C 30 s, +56 °C 1 min, +66 C 1 min, final extension 7 min and final denaturation 8 min. After amplification, the reaction was incu-bated at +40 °C for 20 min and at +22 °C for 15 min, whereafter the time-resolved fluorescence was measured with Victor2 1420 Multilabel Counter (Wallac).<br>
The results of this experiment are shown in Figure 5 by the average and standard deviation of time-resolved Eu fluorescence of each reaction in 4 replicates. Negative and positive PCR controls are drawn by bars A and B, re-spectively. C-E represent the results from blood samples obtained from 3 vol-unteers, respectively. When the integrated release in the reaction vial (hatched bars) is compared to the release in a separate vial (grey bars), it is clearly seen that the whole analysis can be performed using dried reagents even without removal of disk. Also, E shows that the matrix can be integrated into the reac-tion vial comprising the dried reagents, and the sample can be applied right be/-fore the analysis. The result shows that the whole process from sample pre-treatment to detection can be performed in a single reaction vial comprising all reagents dried enabling extremely simple automation.<br><br>
WO 2006/000648	PCT/FI2005/050245<br>
15<br>
Claims<br>
1.	A method for analyzing a target nucleic acid in a biological sample,<br>
comprising the following steps<br>
i) providing an absorbent matrix in a reaction vial;<br>
ii) providing a biological sample containing said target nucleic acids on, or ap-plying it onto, said absorbent matrix;<br>
iii) providing in said reaction vial a reagent mixture for amplifying said nucleic acids;<br>
iv) providing in said reaction vial a detection reagent mixture; v) sealing said reaction vial; vi) performing an amplification reaction; and vii) detecting the amplified nucleic acids, wherein<br>
steps vi) and vii) are performed in said vial without removing said matrix from the vial, and wherein the detection step is performed in closed-tubes, without opening the seals.<br>
2.	The method according to claim 1, wherein the sample is provided<br>
on the absorbent matrix, and at least a portion of said matrix is provided into<br>
the vial together with said sample.<br>
3.	The method according to claim 2, wherein the sample is applied<br>
onto the matrix immediately prior to providing said matrix into the reaction vial,<br>
without drying and/or storage.<br>
4.	The method of claims 1, wherein the matrix is provided in the reac-<br>
tion vial prior to step ii) and the sample is applied onto said matrix contained in<br>
said vial.<br>
5.	The method according to claims 1-4, wherein the matrix is an ab-<br>
sorbent material that does not irreversibly bind nucleic acids.<br>
6.	The method according to claim 5, wherein the matrix comprises a<br>
protein binding and/or a protein denaturing agent.<br>
7.	The method according to claim 5, wherein the matrix comprises a<br>
chaotropic agent.<br><br>
WO 2006/00064S	PCT/FI2005/050245<br>
16<br>
8.	The method according to claims 1-7, wherein at least part of the<br>
amplification and detection reagent mixtures are contained in the reaction vial<br>
in dried form, said method including an optional further step of adding water<br>
prior to step v).<br>
9.	The method according to claims 1-8, wherein the amplification is<br>
performed by PCR.<br>
10.	The method according to claims 1-9, wherein the detection step is<br>
performed by hybridization of the amplified sample to a fluorescently labelled<br>
probe.<br>
11.	A kit for use in a method according to claims 1 -10, comprising<br>
at least one reaction vial comprising an absorbent matrix; a reagent mixture for amplifying said nucleic acids; and a detection reagent mixture for detecting said amplified nucleic acids; wherein at least part of said reagent and detection mixtures are provided in dried form.<br>
12.	A kit according to claim 11, wherein all reagents are provided in<br>
dried form in said reaction vial.<br>
The present invention relates to an integrated method of nucleic acid analysis, and more particularly to a simplified sample pre-treatment, which renders the method more easily automated, where the sample is provided on or applied onto a solid matrix and the subsequent amplification and detection steps are performed in one single, sealed reaction vial without removing the matrix.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBhYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBjbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBjb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBkZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBkcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBmb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBmb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBmb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBncGEucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBpbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBpbnRlcm5hdGlvbmFsIHNlYXJjaCBhdXRob3JpdHkgcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 international search authority report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBwY3Qgb3RoZXIgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 pct other document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNiBwcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006 priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNi1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNi1jb3JyZXNwb25kZW5jZS0xLjEucGRm" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006-correspondence-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDM5MjQta29sbnAtMjAwNi1mb3JtLTMtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">03924-kolnp-2006-form-3-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUFCU1RSQUNUIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-ABSTRACT 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUFCU1RSQUNULTEuMi5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-ABSTRACT-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUFNQU5ERUQgQ0xBSU1TLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-AMANDED CLAIMS-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUFNQU5ERUQgQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUNPUlJFU1BPTkRFTkNFLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-CORRESPONDENCE-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LURFU0NSSVBUSU9OIChDT01QTEVURSkgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-DESCRIPTION (COMPLETE) 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LURFU0NSSVBUSU9OIChDT01QTEVURSktMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-DESCRIPTION (COMPLETE)-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LURSQVdJTkdTIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-DRAWINGS 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSBSRUNJRVZFRC5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-EXAMINATION REPORT REPLY RECIEVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMS0xLjEucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 1-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMS0xLjIucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 1-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMTMucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1rb2xucC0yMDA2LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">3924-kolnp-2006-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMi0xLjEucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 2-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LUZPUk0gMy0xLjEucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-FORM 3-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LU9USEVSUyAxLjEucGRm" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-OTHERS 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzkyNC1LT0xOUC0yMDA2LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">3924-KOLNP-2006-PETITION UNDER RULE 137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250661-a-method-of-sample-preparation-for-sub-surface-phenomenon-and-alloying-segregation-of-hot-rolled-steel-sheets-through-transmision-electron-mictroscopy.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250663-a-process-for-the-preparation-of-tomato-extracts-with-high-content-in-lycopene.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250662</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3924/KOLNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>03/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Jan-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Dec-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>WALLAC OY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>MUSTIONKATU 6, FI-20750 TURKU,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OLLIKKA, PIA</td>
											<td>KOOPPIKUJA 4, FI-20660, LITTOINEN</td>
										</tr>
										<tr>
											<td>2</td>
											<td>YLIKOSKI, ALICE</td>
											<td>SAHANTIE 38, FI-21500, PIIKKIO,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/FI2005/050245</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-06-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>20045248</td>
									<td>2004-06-29</td>
								    <td>Finland</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250662-integrated-nucleic-acid-analysis by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:18:54 GMT -->
</html>
