<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250775-an-expression-cassette-comprising-a-dna-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:15:07 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250775:&quot;AN EXPRESSION CASSETTE COMPRISING A DNA MOLECULE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN EXPRESSION CASSETTE COMPRISING A DNA MOLECULE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An expression cassette comprising a DNA molecule operably linked to plant expression sequences including the transcriptional and translational regulatory signals necessary for expression of the associated DNA constructs in a host organism and optionally further regulatory sequences, wherein said DNA molecule encodes a vegetative insecticidal protein isolatable from liquid culture media during the vegetative growth phase of Bacillus spp., and wherein said protein is encoded by a nucleotide sequence that hybridizes to a nucleotide sequence of SEQ ID NOs: 28, 30 or 31 at 65 Â°C in a buffer comprising 7 % SDS and 0.5 M sodium phosphate.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>NOVEL PEST1CIDAL PROTEINS AND STRAINS<br>
The present invention is drawn to methods and compositions for controlling plant and non-plant pests. Particularly, new pesticidal proteins are disclosed which are isolatable from the vegetative growth stage of Bacillus. Bacillus strains, proteins, and genes encoding the proteins are provided. The methods and compositions of the invention may be used in a variety of systems for controlling plant and non-plant pests.<br>
Insect pests are a major factor in the loss of the world's commercially important agricultural crops. Broad spectrum chemical pesticides have been used extensively to control or eradicate pests of agricultural importance. There is, however, substantial interest in developing effective alternative pesticides.<br>
Microbial pesticides have played an important role as alternatives to chemical pest control. The most extensively used microbial product is based on the bacterium Bacillus thuringiensis (Bt). Bt is a gram-positive spore forming Bacillus which produces an insecticidal crystal protein (ICP) during sporulation.<br>
Numerous varieties of Bt are known that produce more than 25 different but related ICP's. The majority of ICP's made by Bt are toxic to larvae of certain insects in the orders Lepidoptera, Diptera and Coleoptera. In general, when an ICP is ingested by a susceptible insect the crystal is solubilized and transformed into a toxic moiety by the insect gut proteases. None of the ICP's active against coleopteran larvae such as Colorado potato beetle (Leptinotarsa decemlineata) or Yellow mealworm (Tenebrio molitor) have demonstrated significant effects on members of the genus Diabrotica particularly Diabrotica virgifera virgifera  the western corn rootworm (WCRW) or Diabrotica longicornis barberi, the northern corn rootworm.<br>
Bacillus cereus (Be) is closely related to Bt. A major distinguishing characteristic is the absence of a parasporal crystal in Be. Be is a widely distributed bacterium that is commonly found in soil and has been isolated from a variety of foods and drugs. The organism has been implicated in the spoilage of food.<br>
Although Bt has been very useful in controlling insect pests, there is a need to expand the number of potential biological control agents.<br>
Within the present invention compositions and methods for controlling plant pests are provided. In particular, novel pesticidal proteins are provided which are produced during vegetative growth of Bacillus strains. The proteins are useful as pesticidal agents.<br>
More specifically, the present invention relates to a substantially purified Bacillus strain which produces a pesticidal protein during vegetative growth wherein said Bacillus is not B. sphaericus SSI 1-1. Preferred are a Bacillus cereus strain having Accession No. NRRL B-21058 and Bacillus thuringiensis strain having Accession No. NRRL B-21060. Also preferred is a Bacillus strain selected from Accession Numbers NRRL B-21224, NRRL B-21225, NRRL B-21226, NRRL B-21227, NRRL B-21228, NRRL B-21229, NRRL B-21230, and NRRL B-21439.<br>
The invention further relates to an insect-specific protein isolatable during the vegetative growth phase of Bacillus spp, but preferably of a Bacillus thuringiensis and B. cereus strain, and components thereof, wherein said protein is not the mosquitocidal toxin from B. sphaericus SSII-1. The insect-specific protein of the invention is preferably toxic to Coleoptera or Lepidoptera insects and has a molecular weight of about 30 kDa or greater, preferably of about 60 to about 100 kDa, and more preferably of about 80 kDa.<br>
More particularly, the insect-specific protein of the invention has a spectrum of insecticidal activity that includes an activity against Agrotis and/or Spodoptera species, but preferably a black cutworm [Agrotis ipsilon; BCW] and/or fall armyworm [Spodoptera frugiperda] and/or beet armyworm [Spodoptera exigua ] and/or tobacco budworm and/or corn earworm [Helicoverpa zea] activity.<br>
The insect-specific protein of the invention can preferably be isolated, for example, from Bacillus cereus having Accession No. NRRL B-21058, or from Bacillus thuringiensis having Accession No. NRRL B-21060.<br>
The insect-specific protein of the invention can also preferably be isolated from a Bacillus spp strain selected from Accession Numbers NRRL B-21224, NRRL B-21225, NRRL B-21226, NRRL B-21227, NRRL B-21228, NRRL B-21229, NRRL B-21230, and NRRL B-21439.<br>
The present invention especially encompasses an insect-specific protein that has the amino acid sequence selected from the group consisting of SEQ ID NO:5 and<br>
SEQ ID NO:7, including any proteins that are structurally and/or functionally homologous thereto.<br>
Further preferred is an insect-specific protein, wherein said protein has the sequence selected from the group consisting of SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:29 SEQ ID N0:32 and SEQ ID NO:2, including any proteins that are structurally and/or functionally homologous thereto.<br>
Especially preferred is an insect-specific protein, wherein said protein has the sequence selected from the group consisting of SEQ ID NO:29 and SEQ ID NO:32, including any proteins that are structurally and/or functionally homologous thereto.<br>
A further preferred embodiment of the invention comprises an insect-specific protein of the invention, wherein the sequences representing the secretion signal have been removed or inactivated.<br>
The present invention further encompasses auxiliary proteins which enhance the insect-specific activity of an insect-specific protein. The said auxiliary proteins preferably have a molecular weight of about 50 kDa and can be isolated, for example, from the vegetative growth phase of a Bacillus cereus strain, but especially of Bacillus cereus strain AB78.<br>
A preferred embodiment of the invention relates to an auxiliary protein, wherein the sequences representing the secretion signal have been removed or inactivated.<br>
The present invention further relates to multimeric pesticidal proteins, which comprise more than one polypeptide chain and wherein at least one of the said polypeptide chains represents an insect-specific protein of the invention and at least one of the said polypeptide chains represents an auxiliary protein of the invention, which activates or enhances the pesticidal activity of the said insect-specific protein.<br>
The multimeric pesticidal proteins according to the invention preferably have a molecular weight of about 50 kDa to about 200 kDa.<br>
The invention especially encompasses a multimeric pesticidal protein, which comprises an insect-specific protein of the invention and an auxiliary protein according to the invention, which activates or enhances the pesticidal activity of the said insect-specific protein.<br>
The present invention further relates to fusion proteins comprising several protein domains including at least an insect-specific protein of the invention and/or an auxiliary protein according to the invention produced by in frame genetic fusions,<br>
which, when translated by ribosomes, produce a fusion protein with at least the combined attributes of the insect-specific protein of the invention and/or an auxiliary protein according to the invention and, optionally, of the other components used in the fusion.<br>
A specific embodiment of the invention relates to a fusion protein comprising a ribo-nuclease S-protein, an insect-specific protein of the invention and an auxiliary protein according to the invention.<br>
A further specific embodiment of the invention relates to a fusion protein comprising an insect-specific protein according to the invention and an auxiliary protein according to the invention having either the insect-specific protein or the auxiliary protein at the N-terminal end of the said fusion protein.<br>
Preferred is a fusion protein, which comprises an insect-specific protein as given in SEQ ID N0:5 and an auxiliary protein as given in SEQ ID NO: 2 resulting in the protein given in SEQ ID NO: 23, including any proteins that are structurally and/or functionally homologous thereto.<br>
Also preferred is a fusion protein, which comprises an insect-specific protein as given in SEQ ID NO:35 and an auxiliary protein as given in SEQ ID NO: 27 resulting in the protein given in SEQ ID NO: 50, including any proteins that are structurally and/or functionally homologous thereto.<br>
The invention further relates to a fusion protein comprising an insect-specific protein of the invention and/or an auxiliary protein according to the invention fused to a signal sequence, preferably a secretion signal sequence or a targeting sequence that directs the transgene product to a specific organelle or cell compartment, which signal sequence is of herterologous origin with respect to the recipient protein.<br>
Especially preferred within this invention is a fusion protein wherein the said protein has a sequence as given in SEQ ID NO: 43, or in SEQ ID NO: 46, including any proteins that are structurally and/or functionally homologous thereto.<br>
As used in the present application, substantial sequence homology means close structural relationship between sequences of amino acids. For example, substantially homologous proteins may be 40% homologous, preferably 50% and most preferably 60% or 80% homologous, or more. Homology also includes a relationship wherein one or several subsequences of amino acids are missing, or subsequences with additional amino acids are interdispersed.<br>
A further aspect of the invention relates to a DNA molecule comprising a nucleotide sequence which encodes an insect-specific protein isolatable during the vegetative growth phase of Bacillus spp. and components thereof, wherein said protein is not the mosquitocidal toxin from B. sphaericus SSII-1. In particular, the present invention relates to a DNA molecule comprising a nucleotide sequence which encodes an insect-specific protein wherein the spectrum of insecticidal activity includes an activity against Agrotis and/or Spodoptera species, but preferably a black cutworm [Agrotis ipsilon ; BCW] and/or fall armyworm [Spodoptera frugiperda] and/or beet armyworm [Spodoptera exigua ] and/or tobacco budworm and/or corn earworm [Helicoverpa zea] activity.<br>
Preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ, ID NO: 4, or SEQ ID NO: 6, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
Also preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given SEQ ID NO:19, SEQ ID NO:28, SEQ ID NO:31. or SEQ ID NO:1, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
The invention further relates to a DNA molecule comprising a nucleotide sequence which encodes an auxiliary protein according to the invention which enhances the insect-specific activity of an insect-specific protein.<br>
Preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given SEQ ID N0:19, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
A further embodiment of the invention relates to a DNA molecule comprising a nucleotide sequence which encodes an insect-specific protein isolatable during the vegetative growth phase of Bacillus spp. and components thereof, wherein said protein is not the mosquitocidal toxin from B. sphaericus SSII-1, which nucleotide sequence has been optimized for expression in a microorganism or a plant.<br>
Preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ ID N0:17 or SEQ ID N0:18, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
Also preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ ID NO:24, SEQ ID NO:26, SEQ ID NO:27, or<br>
SEQ ID NO:30, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
The invention further relates to a DNA molecule which comprises a nucleotide sequence encoding a multimeric pesticidal protein, which comprises more than one polypeptide chains and wherein at least one of the said polypeptide chains represents an insect-specific protein of the invention and at least one of the said polypeptide chains represents an auxiliary protein according to the invention, which activates or enhances the pesticidal activity of the said insect-specific protein.<br>
Preferred is a DNA molecule comprising a nucleotide sequence encoding an insect-specific protein of the invention and an auxiliary protein according to the invention, which activates or enhances the pesticidal activity of the said insect-specific protein.<br>
Especially preferred is a DNA molecule, wherein said molecule comprises a nucleotide sequence as given in SEQ ID NO:1 or SEQ ID NO:19, including any nucleotide sequences that are structurally and/or functionally homologous thereto. A further embodiment of the invention relates to a DNA molecule which comprises a nucleotide sequence encoding a fusion protein comprising several protein domains including at least an insect-specific protein of the invention and/or an auxiliary protein according to the invention produced by in frame genetic fusions, which, when translated by ribosomes, produce a fusion protein with at least the combined attributes of the insect-specific protein of the invention and/or an auxiliary protein according to the invention and, optionally, of the other components used in the fusion.<br>
Preferred within the invention is a DNA molecule which comprises a nucleotide sequence encoding a fusion protein comprising an insect-specific protein according to the invention and an auxiliary protein according to the invention having either the insect-specific protein or the auxiliary protein at the N-terminal end of the said fusion protein. Especially preferred is a DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ ID NO:22, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
The invention further relates to a DNA molecule which comprises a nucleotide sequence encoding a fusion protein comprising an insect-specific protein of the invention and/or an auxiliary protein of the invention fused to a signal sequence, preferably a secretion signal sequence or a targeting sequence that directs the<br>
transgene product to a specific organelle or cell compartment, which signal sequence is of herterologous origin with respect to the recipient DNA.<br>
The present invention further encompasses a DNA molecule comprising a nucleotide sequence encoding a fusion protein or a mulitmeric protein according to the invention that has been optimized for expression in a microorganism or plant.<br>
Preferred is an optimized DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ ID NO:42, SEQ ID NO:45, or SEQ ID NO:49, including any DNA molecules that are structurally and/or functionally homologous thereto.<br>
The invention further relates to an optimized DNA molecule, wherein the sequences encoding the secretion signal have been removed from its 5' end, but especially to an optimized DNA molecule, wherein the said molecule comprises a nucleotide sequence as given in SEQ ID NO: 35 or SEQ ID NO:39, including any DNA molecules that are structurally and/or functionally homologous thereto. As used in the present application, substantial sequence homology means close structural relationship between sequences of nucleotides. For example, substantially homologous DNA molecules may be 60% homologous, preferably 80% and most preferably 90% or 95% homologous, or more. Homology also includes a relationship wherein one or several subsequences of nucleotides or amino acids are missing, or subsequences with additional nucleotides or amino acids are interdispersed.<br>
Also comprised by the present invention are DNA molecules which hybridizes to a DNA molecule according to the invention as defined hereinbefore, but preferably to an oligonucleotide probe obtainable from said DNA molecule comprising a contiguous portion of the coding sequence for the said insect-specific protein at least 10 nucleotides in length, under moderately stringent conditions and which molecules have insect-specific activity and also the insect-specific proteins being encoded by the said DNA molecules.<br>
Preferred are DNA molecules, wherein hybridization occurs at 65Â°C in a buffer comprising 7% SDS and 0.5 M sodium phosphate.<br>
Especially preferred is a DNA molecule comprising a nucleotide sequence which encodes an insect-specific protein according to the invention obtainable by a process comprising<br>
obtaining a DNA molecule comprising a nucleotide sequence encoding an insect-<br>
specific protein; and<br>
hybridizing said DNA molecule with an oligonucleotide probe acording to claim<br>
107 obtained from a DNA molecule comprising a nucleotide sequence as given in<br>
SEQ ID NO: 28, SEQ ID NO: 30, or SEQ ID NO: 31; and<br>
(c) isolating said hybridized DNA.<br>
The invention further relates to an insect-specific protein, wherein the said protein is encoded by a DNA molecule according to the invention.<br>
Also encompassed by the invention is an expression cassette comprising a DNA molecule according to the invention operably linked to expression sequences including the transcriptional and translational regulatory signals necessary for expression of the associated DNA constructs in a host organism, preferably a microorganism or a plant, and optionally further regulatory sequences.<br>
The invention further relates to a vector molecule comprising an expression cassette according to the invention.<br>
The expression cassette and/or the vector molecule according to the invention are preferably part of the plant genome.<br>
A further embodiment of the invention relates to a host organism, preferably a host organism selected from the group consisting of plant and insect cells, bacteria, yeast, baculoviruses, protozoa, nematodes and algae, comprising a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette, preferably stably incorporated into the genome of the host organism.<br>
The invention further relates to a transgenic plant, but preferably a maize plant, including parts as well as progeny and seed thereof comprising a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette, preferably stably incorporated into the plant genome.<br>
Preferred is a transgenic plant including parts as well as progeny and seed thereof which has been stably transformed with a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette.<br>
Also preferred is a transgenic plant including parts as well as progeny and seed thereof which expresses an insect-specific protein according to the invention.<br>
The invention further relates to a transgenic plant, preferably a maize plant, according to the invention as defined hereinbefore, which further expresses a second distinct insect control principle, but preferably a Bt Î´-endotoxin. The said plant is preferably a hybrid plant.<br>
Parts of transgenic plants are to be understood within the scope of the invention to comprise, for example, plant cells, protoplasts, tissues, callus, embryos as well as flowers, stems, fruits, leaves, roots originating in transgenic plants or their progeny previously transformed with a DNA molecule according to the invention and therefore consisting at least in part of transgenic cells, are also an object of the present invention.<br>
The invention further relates to plant propagating material of a plant according to the invention, which is treated with a seed protectant coating.<br>
The invention further encompasses a microorganism transformed with a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette, wherein the said microorganism is preferably a microorganism that multiply on plants and more preferably a root colonizing bacterium.<br>
A further embodiment of the invention relates to an encapsulated insect-specific protein which comprises a microorganism comprising an insect specific protein according to the invention.<br>
The invention also relates to an entomocidal composition comprising a host organism of the invention, but preferably a purified Bacillus strain, in an insecticidally-effective amount together with a suitable carrier.<br>
Further comprised by the invention is an entomocidal composition comprising an isolated protein molecule according to the invention, alone or in combination with a host organism of the invention and/or an encapsulated insect-specific protein according to the invention, in an insecticidally-effective amount, together with a suitable carrier.<br>
A further embodiment of the invention relates to a method of obtaining a purified insect-specific protein according to the invention, said method comprising applying a<br>
solution comprising said insect-specific protein to a NAD column and eluting bound protein.<br>
Also comprised is a method for identifying insect activity of an insect-specific protein according to the invention, said method comprising: growing a Bacillus strain in a culture; obtaining supernatant from said culture; allowing insect larvae to feed on diet with said supernatant; and, determining mortality.<br>
Another aspect of the invention relates to a method for isolating an insect-specific protein according to the invention, said method comprising: growing a Bacillus strain in a culture; obtaining supernatant from said culture; and, isolating said insect-specific protein from said supernatant. The invention also encompasses a method for isolating a DNA molecule comprising a nucleotide sequence encoding an insect-specific protein exhibiting the insecticidal activity of the proteins according to the invention, said method comprising: obtaining a DNA molecule comprising a nucleotide sequence encoding an<br>
insect-specific protein; and hybridizing said DNA molecule with DNA obtained from a Bacillus species;<br>
and<br>
isolating said hybridized DNA.<br>
The invention further relates to a method of increasing insect target range by using an insect specific protein according to the invention in combination with at least one second insecticidal protein that is different from the insect specific protein according to the invention, but preferably with an insecticidal protein selected from the group consisting of Bt Î´-endotoxins, protease inhibitors, lectins, Î±-amylases and peroxidases.<br>
Preferred is a method for increasing insect target range within a plant by expressing within the said plant a insect specific protein according to the invention in combination with at least one second insecticidal protein that is different from the insect specific protein according to the invention, but preferably with an insecticidal protein selected from the group consisting of Bt 5-endotoxins, protease inhibitors, lectins, Î±-amylases and peroxidases.<br>
Also comprised is a method of protecting plants against damage caused by an insect pest, but preferably by Spodoptera and/or Agrotis species, and more preferably by an insect pest selected from the group consisting of black cutworm [Agrotis ipsilon BCW], fall armyworm [Spodoptera frugiperda], beet armyworm [Spodoptera exigua ], tobacco budworm and corn earworm [Helicoverpa zea] comprising applying to the plant or the growing area of the said plant an entomocidal composition or a toxin protein according to the invention.<br>
The invention further relates to method of protecting plants against damage caused by an insect pest, but preferably by Spodoptera and/or Agrotis species, and more preferably by an insect pest selected from the group consisting of black cutworm [Agrotis ipsilon ; BCW], fall armyworm [Spodoptera frugiperda], beet armyworm [Spodoptera exigua ], tobacco budworm and corn earworm [Helicoverpa zea] comprising planting a transgenic plant expressing a insect-specific protein according to the invention within an area where the said insect pest may occur.<br>
The invention also encompasses a method of producing a host organism which comprises stably integrated into its genome a DNA molecule according to the invention and preferably expresses an insect-specific protein according to the invention comprising transforming the said host organism with a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette.<br>
A further embodiment of the invention relates to a method of producing a transgenic plant or plant cell which comprises stably integrated into the plant genome a DNA molecule according to the invention and preferably expresses an insect-specific protein according to the invention comprising transforming the said plant and plant cell, respectively, with a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette.<br>
The invention also relates to a method of producing an entomocidal composition comprising mixing an isolated Bacillus strain and/or a host organism and/or an isolated protein molecule, and/or an encapsulated protein according to the invention in an insecticidally-effective amount with a suitable carrier.<br>
The invention also encompasses a method of producing transgenic progeny of a transgenic parent plant comprising stably incorporated into the plant genome a DNA<br>
molecule comprising a nucleotide sequence encoding an insect-specific protein according to the invention comprising transforming the said parent plant with a DNA molecule according to the invention, an expression cassette comprising the said DNA molecule or a vector molecule comprising the said expression cassette and transferring the pesticidal trait to the progeny of the said transgenic parent plant involving known plant breeding techniques.<br>
Also encompassed by the invention is oligonucleotide probe capable of specifically hybridizing to a nucleotide sequence encoding a insect-specific protein isolatable during the vegetative growth phase of Bacillus spp. and components thereof, wherein said protein is not the mosquitocidal toxin from B. sphaericus SSII-1, wherein said probe comprises a contiguous portion of the coding sequence for the said insect-specific protein at least 10 nucleotides in length and the use of the said oligonucleotide probe for screening of any Bacillus strain or other organisms to determine whether the insect-specific protein is naturally present or whether a particular transformed organism includes the said gene<br>
The present invention recognizes that pesticidal proteins are produced during vegetative growth of Bacillus strains. Having recognized that such a class exists, the present invention embraces all vegetative insecticidal proteins, hereinafter referred to as VIPs, except for the mosquitocidal toxin from B. sphaericus.<br>
The present VIPs are not abundant after sporulation and are particularly expressed during log phase growth before stationary phase. For the purpose of the present invention vegetative growth is defined as that period of time before the onset of sporulation. Genes encoding such VIPs can be isolated, cloned and transformed into various delivery vehicles for use in pest management programs.<br>
For purposes of the present invention, pests include but are not limited to insects, fungi, bacteria, nematodes, mites, ticks, protozoan pathogens, animal-parasitic liver flukes, and the like. Insect pests include insects selected from the orders Coleoptera, Diptera, Hymenoptera, Lepidoptera, Mallophaga, Homoptera, Hemiptera, Orthroptera, Thysanoptera, Dermaptera, Isoptera, Anoplura, Siphonaptera, Trichoptera, etc., particularly Coleoptera and Lepidoptera.<br>
Tables 1-10 gives a list of pests associated with major crop plants and pests of human and veterinary importance. Such pests are included within the scope of the present invention.<br>
TABLE 1<br>
Lepidoptera (Butterflies and Moth)<br>
Maize<br>
Ostrinia nubilalis, European corn borer Agrotis ipsilon, black cutworm Helicoverpa zea, corn earworm Spodoptera frugiperda, fall armyworm Diatraea grandiosella, southwestern corn borer Elasmopalpus lignosellus, lesser cornstalk borer Diatraea saccharalis, sugarcane borer<br>
Sorghum<br>
Chilo partellus, sorghum borer Spodoptera frugiperda, fall armyworm Helicoverpa zea, corn earworm Elasmopalpus lignosellus, lesser cornstalk borer Feltia subterranea, granulate cutworm<br>
Wheat<br>
Pseudaletia unipunctata, army worm Spodoptera frugiperda, fall armyworm Elasmopalpus lignosellus, lesser cornstalk borer Agrotis orthogonia, pale western cutworm Elasmopalpus lignosellus, lesser cornstalk borer<br>
Sunflower<br>
Suleima helianthana, sunflower bud moth Homoeosoma electellum, sunflower moth<br>
Cotton<br>
Heliothis virescens, cotton boll worm Helicoverpa zea, cotton bollworm Spodoptera exigua, beet armyworm Pectinophora gossypiella, pink bollworm<br>
Rice<br>
Diatraea saccharalis, sugarcane borer Spodoptera frugiperda, fall armyworm Helicoverpa zea, corn earworm<br>
Soybean<br>
Pseudoplusia includens, soybean looper Anticarsia gemmatalis, velvetbean caterpillar Plathypena scabra, green cloverworm Ostrinia nubilalis, European corn borer Agrotis ipsilon, black cutworm Spodoptera exigua, beet armyworm Heliothis virescens, cotton boll worm Helicoverpa zea, cotton bollworm<br>
Barley<br>
Ostrinia nubilalis, European corn borer Agrotis ipsilon, black cutworm<br>
TABLE 2 Coleoptera (Beetles)<br>
Maize<br>
Diabrotica virgifera virgifera, western corn rootworm Diabrotica longicornis barberi, northern corn rootworm Diabrotica undecimpunctata howardi, southern corn rootworm Melanotus spp., wireworms<br>
Cyclocephala borealis, northern masked chafer (white grub) Cyclocephala irnmaculata, southern masked chafer (white grub) Popillia japonica, Japanese beetle Chaetocnema pulicaria, corn flea beetle Sphenophorus maidis, maize billbug<br>
Sorghum<br>
Phyllophaga crinita, white grub Eleodes, Conoderus, and Aeolus spp., wireworms Oulema melanopus, cereal leaf beetle Chaetocnema pulicaria, corn flea beetle Sphenophorus maidis, maize billbug<br>
Wheat<br>
Oulema melanopus, cereal leaf beetle<br>
Hypera punctata, clover leaf weevil<br>
Diabrotica undecimpunctata howardi, southern corn rootworm<br>
Sunflower<br>
Zygogramma exclamationis, sunflower beetle Bothyrus gibbosus, carrot beetle<br>
Cotton<br>
Anthonomus grandis, boll weevil<br>
Rice<br>
Colaspis brunnea, grape colaspis Lissorhoptrus oryzophilus, rice water weevil Sitophilus oryzae, rice weevil<br>
Soybean<br>
Epilachna varivestis, Mexican bean beetle<br>
TABLE 3<br>
Homoptera (Whiteflies. Aphids etc..)<br>
Maize<br>
Rhopalosiphum maidis, corn leaf aphid Anuraphis maidiradicis, corn root aphid<br>
Sorghum<br>
Rhopalosiphum maidis, corn leaf aphid Sipha flava, yellow sugarcane aphid<br>
Wheat<br>
Russian wheat aphid Schizaphis graminum, greenbug Macrosiphum avenae, English grain aphid<br>
Cotton<br>
Aphis gossypii, cotton aphid Pseudatomoscelis seriatus, cotton fleahopper Trialeurodes abutilonea, bandedwinged whitefly<br>
Rice<br>
Nephotettix nigropictus, rice leafhopper<br>
Soybean<br>
Myzus persicae, green peach aphid Empoasca fabae, potato leafhopper<br>
Barley<br>
Schizaphis graminum, greenbug<br>
Oil Seed Rape<br>
Brevicoryne brassicae, cabbage aphid<br>
TABLE 4 Hemiptera (Bugs)<br>
Maize<br>
Blissus leucopterus leucopterus, chinch bug<br>
Sorghum<br>
Blissus leucopterus leucopterus, chinch bug<br>
Cotton<br>
Lygus lineolaris, tarnished plant bug<br>
Rice<br>
Blissus leucopterus leucopterus, chinch bug Acrosternum hi/are, green stink bug<br>
Soybean<br>
Acrosternum hilare, green stink bug<br>
Barley<br>
Blissus leucopterus leucopterus, chinch bug Acrosternum hilare, green stink bug Euschistus servus, brown stink bug<br>
TABLE 5<br>
Orthoptera (Grasshoppers. Crickets, and Cockroaches)<br>
Maize<br>
Melanoplus femurrubrum, redlegged grasshopper Melanoplus sanguinipes, migratory grasshopper<br>
Wheat<br>
Melanoplus femurrubrum, redlegged grasshopper Melanoplus differentialis, differential grasshopper Melanoplus sanguinipes, migratory grasshopper<br>
Cotton<br>
Melanoplus femurrubrum, redlegged grasshopper Melanoplus differentialis, differential grasshopper<br>
Soybean<br>
Melanoplus femurrubrum, redlegged grasshopper Melanoplus differentialis, differential grasshopper<br>
Structural/Household<br>
Periplaneta americana, American cockroach Blattella germanica, German cockroach Blatta orientalis, oriental cockroach<br>
TABLE 6<br>
Diptera (Flies and Mosquitoes)<br>
Maize<br>
Hylemya platura, seedcorn maggot Agromyza parvicornis, corn blotch leafminer<br>
Sorghum<br>
Contarinia sorghicola, sorghum midge<br>
Wheat<br>
Mayetiola destructor, Hessian fly Sitodiplosis mosellana, wheat midge Meromyza americana, wheat stem maggot Hylemya coarctata, wheat bulb fly<br>
Sunflower<br>
Neolasioptera murtfeldtiana, sunflower seed midge<br>
Soybean<br>
Hylemya platura, seedcorn maggot<br>
Barley<br>
Hylemya platura, seedcorn maggot Mayetiola destructor, Hessian fly<br>
Insects attacking humans and animals and disease carriers<br>
Aedes aegypti, yellowfever mosquito Aedes albopictus, forest day mosquito Phlebotomus papatasii, sand fly Musca domestica, house fly Tabanus atratus, black horse fly Cochliomyia hominivorax, screwworm fly<br>
TABLE 7 Thvsanoptera (Thrips)<br>
Maize<br>
Anaphothrips obscurus, grass thrips<br>
Wheat<br>
Frankliniella fusca, tobacco thrips<br>
Cotton<br>
Thrips (abaci, onion thrips Frankliniella fusca, tobacco thrips<br>
Soybean<br>
Sericothrips variabilis, soybean thrips Thrips tabaci, onion thrips<br>
TABLE 8<br>
Hymenoptera (Sawflies. Ants. Wasps, etc.)<br>
Maize<br>
Solenopsis milesta, thief ant<br>
Wheat<br>
Cephus cinctus, wheat stem sawfly<br>
TABLE 9<br>
Other Orders and Representative Species<br>
Dermaptera (Earwigs)<br>
Forficula auricularia, European earwig<br>
Isoptera (Termites)<br>
Reticulitermes flavipes, eastern subterranean termite<br>
Mallophaga (Chewing Lice)<br>
Cuclotogaster heterographa, chicken head louse Bovicola bovis, cattle biting louse<br>
Anoplura (Sucking Lice)<br>
Pediculus humanus, head and body louse<br>
Siphonaptera (Fleas)<br>
Ctenocephalides felis, cat flea<br>
TABLE 10<br>
Acari (Mites and Ticks)<br>
Maize<br>
Tetranychus urticae, twospotted spider mite<br>
Sorghum<br>
Tetranychus cinnabarinus, carmine spider mite Tetranychus urticae, twospotted spider mite<br>
Wheat<br>
Aceria tulipae, wheat curl mite<br>
Cotton<br>
Tetranychus cinnabarinus, carmine spider mite Tetranychus urticae, twospotted spider mite<br>
Soybean<br>
Tetranychus turkestani, strawberry spider mite Tetranychus urticae, twospotted spider mite<br>
Barley<br>
Petrobia latens, brown wheat mite<br>
Important human and animal Acari<br>
Demacentor variabilis, American dog tick<br>
Argas persicus, fowl tick<br>
Dermatophagoides farinae, American house dust mite<br>
Dermatophagoides pteronyssinus, European house dust mite<br>
Now that it has been recognized that pesticidal proteins can be isolated from the vegetative growth phase of Bacillus, other strains can be isolated by standard techniques and tested for activity against particular plant and non-plant pests. Generally Bacillus strains can be isolated from any environmental sample, including soil, plant, insect, grain elevator dust, and other sample material, etc., by methods<br>
known in the art. See, for example, Travers ef al. (1987) Appl. Environ. Microbiol. 53:1263-1266; Saleh et al. (1969) Can J. Microbiol. 15:1101-1104; DeLucca et al. (1981) Can. J. Microbiol. 27:865-870; and Norris, era/. (1981) "The genera Bacillus and Sporolactobacillus,  In Starr et al. (eds.), The Prokaryotes: A Handbook on Habitats, Isolation, and Identification of Bacteria, Vol. II, Springer-Verlog Berlin Heidelberg. After isolation, strains can be tested for pesticidal activity during vegetative growth. In this manner, new pesticidal proteins and strains can be identified.<br>
Such Bacillus microorganisms which find use in the invention include Bacillus cereus and Bacillus thuringiensis, as well as those Bacillus species listed in Table 11,<br>
TABLE 11 List of Bacillus species<br>
Morphological Group 1<br>
B. megaterium<br>
B. cereus*<br>
B. cereus var. mycoides<br>
B. thuringiensis*<br>
B. licheniformis<br>
B. subtilis*<br>
B. pumilus<br>
B. firmus*<br>
B. coagulans<br>
Morphological Group 2<br>
B. polymyxa<br>
B. macerans<br>
B. circulans<br>
B. stearothermophilus<br>
B. alvei'<br>
B. laterosporus1<br>
B. brevis<br>
B. pulvifaciens<br>
B. popilliae*<br>
B. lentimorbus*<br>
B. larvae*<br>
Morphological Group 3<br>
B. sphaericus* B. pasteurii<br>
Unassigned Strains Subgroup A<br>
B. apiarus* B. filicolonicus B. thiaminolyticus B. alcalophilus<br>
Subgroup B<br>
6. cirroflagellosus B. chitinosporus B. lentus<br>
Subgroup C<br>
B. badius B. aneurinolyticus B. macroides B. freundenreichii<br>
Subgroup D<br>
B, pantothenticus B. epiphytus<br>
Subgroup E1<br>
B. aminovorans B. globisporus B. insolitus B. psychrophilus<br>
Subgroup E2<br>
B. psychrosaccharolyticus B. macquariensis<br>
*=Those Bacillus strains that have been previously found associated with insects Grouping according to Parry, J.M. etal. (1983) Color Atlas of Bacillus species, Wolfe Medical Publications, London.<br>
In accordance with the present invention, the pesticidal proteins produced during vegetative growth can be isolated from Bacillus. In one embodiment, insecticidal proteins produced during vegetative growth, can be isolated. Methods for protein isolation are known in the art. Generally, proteins can be purified by conventional chromatography, including gel-filtration, ion-exchange, and immunoaffinity chromatography, by high-performance liquid chromatography, such as reversed-phase high-performance liquid chromatography, ion-exchange high-performance liquid chromatography, size-exclusion high-performance liquid chromatography, high-performance chromatofocusing and hydrophobic interaction chromatography, etc., by electrophoretic separation, such as one-dimensional gel electrophoresis, two-dimensional gel electrophoresis, etc. Such methods are known in the art. See for example Current Protocols in Molecular Biology. Vols. 1 and 2, Ausubel et a/, (eds.), John Wiley &amp; Sons, NY (1988). Additionally, antibodies can be prepared against substantially pure preparations of the protein. See, for example, Radka et a/. (1983) J. Immunol. 128:2804; and Radka et at. (1984) Immunoaenetics 19:63. Any combination of methods may be utilized to purify protein having pesticidal properties. As the protocol is being formulated, pesticidal activity is determined after each purification step.<br>
Such purification steps will result in a substantially purified protein fraction. By "substantially purified" or "substantially pure" is intended protein which is substantially free of any compound normally associated with the protein in its natural state. "Substantially pure" preparations of protein can be assessed by the absence of other detectable protein bands following SDS-PAGE as determined visually or by densitometry scanning. Alternatively, the absence of other amino-terminal sequences or N-terminal residues in a purified preparation can indicate the level of purity. Purity can be verified by rechromatography of "pure" preparations showing the absence of other peaks by ion exchange, reverse phase or capillary electrophoresis. The terms "substantially pure" or "substantially purified" are not meant to exclude artificial or synthetic mixtures of the proteins with other compounds. The terms are also not meant to exclude the presence of minor impurities which do not interfere with the biological activity of the protein, and which may be present, for example, due to incomplete purification.<br>
Once purified protein is isolated, the protein, or the polypeptides of which it is comprised, can be characterized and sequenced by standard methods known in the art.   For example, the purified protein, or the polypeptides of which it is comprised, may be fragmented as with cyanogen bromide, or with proteases such as papain, chymotrypsin, trypsin, lysyl-C endopeptidase, etc. (Oike et al. (1982) J. Biol. Chem. 257:9751 -9758; Liu et al. (1983) ]nt J. Pept. Protein Res. 21:209-215). The resulting peptides are separated, preferably by HPLC, or by resolution of gels and electroblotting onto PVDF membranes, and subjected to amino acid sequencing. To accomplish this task, the peptides are preferably analyzed by automated sequenators. It is recognized that N-terminal, C-terminal, or internal amino acid sequences can be determined. From the amino acid sequence of the purified protein, a nucleotide sequence can be synthesized which can be used as a probe to aid in the isolation of the gene encoding the pesticidal protein.<br>
It is recognized that the pesticidal proteins may be oligomeric and will vary in molecular weight, number of protomers, component peptides, activity against particular pests, and in other characteristics. However, by the methods set forth herein, proteins active against a variety of pests may be isolated and characterized.<br>
Once the purified protein has been isolated and characterized it is recognized that it may be altered in various ways including amino acid substitutions, deletions, truncations, and insertions. Methods for such manipulations are generally known in the art. For example, amino acid sequence variants of the pesticidal proteins can be prepared by mutations in the DNA. Such variants will possess the desired pesticidal activity. Obviously, the mutations that will be made in the DNA encoding the variant must not place the sequence out of reading frame and preferably will not create complementary regions that could produce secondary mRNA structure. See, EP Patent Application Publication No. 75,444.<br>
In this manner, the present invention encompasses the pesticidal proteins as well as components and fragments thereof. That is, it is recognized that component protomers, polypeptides or fragments of the proteins may be produced which retain pesticidal activity. These fragments include truncated sequences, as well as N-terminal, C-terminal, internal and internally deleted amino acid sequences of the proteins.<br>
Most deletions, insertions, and substitutions of the protein sequence are not expected to produce radical changes in the characteristics of the pesticidal protein. However, when it is difficult to predict the exact effect of the substitution, deletion, or insertion in advance of doing so, one skilled in the art will appreciate that the effect will be evaluated by routine screening assays.<br>
The proteins or other component polypeptides described herein may be used alone or in combination. That is, several proteins may be used to control different insect pests.<br>
Some proteins are single polypeptide chains while many proteins consist of more than one polypeptide chain, i.e., they are oligomeric. Additionally, some VIPs are pesticidally active as oligomers. In these instances, additional protomers are utilized to enhance the pesticidal activity or to activate pesticidal proteins. Those protomers which enhance or activate are referred to as auxiliary proteins. Auxiliary proteins activate or enhance a pesticidal protein by interacting with the pesticidal protein to form an oligomeric protein having increased pesticidal activity compared to that observed in the absence of the auxiliary protein.<br>
Auxiliary proteins activate or increase the activity of pesticidal proteins such as the VIP1 protein from AB78. Such auxiliary proteins are exemplified by, but not limited to, the VIP2 protein from AB78. As demonstrated in the Experimental section of the application, auxiliary proteins can activate a number of pesticidal proteins. Thus, in one embodiment of the invention, a plant, Parent 1, can be transformed with an auxiliary protein. This Parent 1 can be crossed with a number of Parent 2 plants transformed with one or more pesticidal proteins whose pesticidal activities are activated by the auxiliary protein.<br>
Amongst the pesticidal proteins of the invention a new class of insect-specific proteins could be surprisingly identified within the scope of the present invention. The said proteins, which are designated throughout this application as VIP3, can be obtained from Bacillus spp strains, but preferably from Bacillus thuringiensis strains and most preferably from Bacillus thuringiensis strains AB88 and AB424. The said VIPs are present mostly in the supernatants of Bacillus cultures amounting to at least 75% of the total in strain AB88. The VIP3 proteins are further characterized by their unique spectrum of insectical acitivity, which includes an activity against Agrotis and/or Spodoptera species, but especially a black cutworm [BCW] and/or fall<br>
army worm and/or beet armyworm and/or tobacco budworm and/or corn earworm activity.<br>
Black cutworm is an agronomically important insect quite resistant to 5-endotoxins. Macintosh et al (1990) J Invertebr Pathol 56, 258-266 report that the 5-endotoxins CrylA(b) and CrylA(c) possesses insecticidal properties against BCW with LCso of more than 80 ng and 18 ng/ml of diet respectively. The vipSA insecticidal proteins according to the invenition provide &gt;50% mortality when added in an amount of protein at least 10 to 500, preferably 50 to 350, and more preferably 200 to 300 fold lower than the amount of CrylA proteins needed to achieve just 50% mortality. Especially preferred within the invention are vip3A insecticidal proteins which provide 100% mortality when added in an amount of protein at least 260 fold lower than the amount of CrylA proteins needed to achieve just 50% mortality.<br>
The vip3 insecticidal proteins according to the invention are present mostly in the supernatants of the cultures and are therefore are to be classified as secreted proteins. They preferably contain in the N-terminal sequence a number of positively charged residues followed by a hydrophobic core region and are not N-terminally processed during export.<br>
As the other pesticidal proteins reported hereto within the scope of the invention, the VIP3 proteins can be detected in growth stages prior to sporulation establishing a further clear distinction from other proteins that belong to the 5-endotoxin family. Preferably, expression of the insect-specific protein starts during mid-log phase and continues during sporulation. Owing to the specific expression pattern in combination with the high stability of the VIP3 proteins, large amounts of the VIP3 proteins can be found in supernatants of sporulating cultures. Especially preferred are the VIP3 proteins identified in SEQ ID NO.29 and SEQ ID N0:32 and the corresponding DMA molecules comprising nucleotide sequences encoding the said proteins, but especially those DNA molecules comprising the nucleotide sequences given in SEQ ID N0:28, SEQ ID NO:30 and SEQ ID NO:31.<br>
The pesticidal proteins of the invention can be used in combination with Bt endotoxins or other insecticidal proteins to increase insect target range. Furthermore, the use of the VIPs of the present invention in combination with Bt 5-endotoxins or other insecticidal principles of a distinct nature has particular utility for the prevention and/or management of insect resistance. Other insecticidal principles include<br>
protease inhibitors (both serine and cysteine types), lectins, Î±-amylase and peroxidase. In one preferred embodiment, expression of VIPs in a transgenic plant is accompanied by the expression of one or more Bt Î´-endotoxins. This co-expression of more than one insecticidal principle in the same transgenic plant can be achieved by genetically engineering a plant to contain and express all the genes necessary. Alternatively, a plant, Parent 1, can be genetically engineered for the expression of VIPs. A second plant, Parent 2, can be genetically engineered for the expression of Bt Î´-endotoxin. By crossing Parent 1 with Parent 2, progeny plants are obtained which express all the genes introduced into Parents 1 and 2. Particularly preferred Bt 8-endotoxins are those disclosed in EP-A 0618976, herein incorporated by reference.<br>
A substantial number of cytotoxic proteins, though not all, are binary in action. Binary toxins typically consist of two protein domains, one called the A domain and the other called the B domain (see Sourcebook of Bacterial Protein Toxins. J. E. Alouf and J. H. Freer eds.(1991) Academic Press). The A domain possesses a potent cytotoxic activity. The B domain binds an external cell surface receptor before being internalized. Typically, the cytotoxic A domain must be escorted to the cytoplasm by a translocation domain. Often the A and B domains are separate polypeptides or protomers, which are associated by a protein-protein interaction or a di-sulfide bond. However, the toxin can be a single polypeptide which is proteolytically processed within the cell into two domains as in the case for Pseudomonas exotoxin A. In summary binary toxins typically have three important domains, a cytotoxic A domain, a receptor binding B domain and a translocation domain. The A and B domain are often associated by protein-protein interacting domains.<br>
The receptor binding domains of the present invention are useful for delivering any protein, toxin, enzyme, transcription factor, nucleic acid, chemical or any other factor into target insects having a receptor recognized by the receptor binding domain of the binary toxins described in this patent. Similarly, since binary toxins have translocation domains which penetrate phosopholipid bilayer membranes and escort cytotoxins across those membranes, such translocation domains may be useful in escorting any protein, toxin, enzyme, transcription factor, nucleic acid, chemical or any other factor across a phospholipid bilayer such as the plasma membrane or a vesicle membrane. The translocation domain may itself perforate membranes, thus having toxic or insecticidal properties. Further, all binary toxins have cytotoxic domains; such a<br>
cytotoxic domain may be useful as a lethal protein, either alone or when delivered into any target cell(s) by any means.<br>
Finally, since binary toxins comprised of two polypeptides often form a complex, it is likely that there are protein-protein interacting regions within the components of the binary toxins of the invention. These protein-protein interacting domains may be useful in forming associations between any combination of toxins, enzymes, transcription factors, nucleic acids, antibodies, cell binding moieties, or any other chemicals, factors, proteins or protein domains.<br>
Toxins, enzymes, transcription factors, antibodies, cell binding moieties or other protein domains can be fused to pesticidal or auxiliary proteins by producing in frame genetic fusions which, when translated by ribosomes, would produce a fusion protein with the combined attributes of the VIP and the other component used in the fusion. Furthermore, if the protein domain fused to the VIP has an affinity for another protein, nucleic acid, carbohydrate, lipid, or other chemical or factor, then a three-component complex can be formed. This complex will have the attributes of all of its components. A similar rationale can be used for producing four or more component complexes. These complexes are useful as insecticidal toxins, Pharmaceuticals, laboratory reagents, and diagnostic reagents, etc. Examples where such complexes are currently used are fusion toxins for potential cancer therapies, reagents in ELISA assays and immunoblot analysis.<br>
One strategy of altering pesticidal or auxiliary proteins is to fuse a 15-amino-acid "S-tag" to the protein without destroying the insect cell binding domain(s), translocation domains or protein-protein interacting domains of the proteins. The S-tag has a high affinity (Kd = 10-9 M) for a ribonuclease S-protein, which, when bound to the S-tag, forms an active ribonuclease (See F. M. Richards and H. W. Wyckoff (1971) in "The Enzymes", Vol. IV (Boyer, P.O. ed.). pp. 647-806. Academic Press, New York). The fusion can be made in such a way as to destroy or remove the cytotoxic activity of the pesticidal or auxiliary protein, thereby replacing the VIP cytotoxic activity with a new cytotoxic ribonuclease activity. The final toxin would be comprised of the S-protein, a pesticidal protein and an auxiliary protein, where either the pesticidal protein or the auxiliary protein is produced as translational fusions with the S-tag. Similar strategies can be used to fuse other potential cytotoxins to pesticidal or auxiliary proteins including (but not limited to) ribosome inactivating<br>
proteins, insect hormones, hormone receptors, transcription factors, proteases, phosphatases, Pseudomonas exotoxin A, or any other protein or chemical factor that is lethal when delivered into cells. Similarly, proteins can be delivered into cells which are not lethal, but might alter cellular biochemistry or physiology.<br>
The spectrum of toxicity toward different species can be altered by fusing domains to pesticidal or auxiliary proteins which recognize cell surface receptors from other species. Such domains might include (but are not limited to) antibodies, transferrin, hormones, or peptide sequences isolated from phage displayed affinity selectable libraries. Also, peptide sequences which are bound to nutrients, vitamins, hormones, or other chemicals that are transported into cells could be used to alter the spectrum of toxicity. Similarly, any other protein or chemical which binds a cell surface receptor or the membrane and could be internalized might be used to alter the spectrum of activity of VIP 1 and VIP2.<br>
The pesticidal proteins of the present invention are those proteins which confer a specific pesticidal property. Such proteins may vary in molecular weight, having component polypeptides at least a molecular weight of 30 kDa or greater, preferably about 50 kDa or greater.<br>
The auxiliary proteins of the invention may vary in molecular weight, having at least a molecular weight of about 15 kDa or greater, preferably about 20 kDa or greater; more preferably, about 30 kDa or greater. The auxiliary proteins themselves may have component polypeptides.<br>
It is possible that the pesticidal protein and the auxiliary protein may be components of a rnultimeric, pesticidal protein. Such a pesticidal protein which includes the auxiliary proteins as one or more of its component polypeptides may vary in molecular weight, having at least a molecular weight of 50 kDa up to at least 200 kDa, preferably about 100 kDa to 150 kDa.<br>
An auxiliary protein may be used in combination with the pesticidal proteins of the invention to enhance activity or to activate the pesticidal protein. To determine whether the auxiliary protein will affect activity, the pesticidal protein can be expressed alone and in combination with the auxiliary protein and the respective activities compared in feeding assays for pesticidal activity.<br>
It may be beneficial to screen strains for potential pesticidal activity by testing activity of the strain alone and in combination with the auxiliary protein. In some<br>
instances an auxiliary protein in combination with the native proteins of the strains yields pesticidal activity where none is seen in the absence of an auxiliary protein.<br>
The auxiliary protein can be modified, as described above, by various methods known in the art. Therefore, for purposes of the invention, the term "Vegetative Insecticidal Protein" (VIP) encompasses those proteins produced during vegetative growth which alone or in combination can be used for pesticidal activity. This includes pesticidal proteins, auxiliary proteins and those proteins which demonstrate activity only in the presence of the auxiliary protein or the polypeptide components of these proteins.<br>
It is recognized that there are alternative methods available to obtain the nucleotide and amino acid sequences of the present proteins. For example, to obtain the nucleotide sequence encoding the pesticidal protein, cosmid clones, which express the pesticidal protein, can be isolated from a genomic library. From larger active cosmid clones, smaller subclones can be made and tested for activity. In this manner, clones which express an active pesticidal protein can be sequenced to determine the nucleotide sequence of the gene. Then, an amino acid sequence can be deduced for the protein. For general molecular methods, see, for example, Molecular Cloning, A Laboratory Manual, Second Edition, Vols. 1-3, Sambrook er a/, (eds.) Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY (1989), and the references cited therein.<br>
The present invention also encompasses nucleotide sequences from organisms other than Bacillus, where the nucleotide sequences are isolatable by hybridization with the Bacillus nucleotide sequences of the invention. Proteins encoded by such nucleotide sequences can be tested for pesticidal activity. The invention also encompasses the proteins encoded by the nucleotide sequences. Furthermore, the invention encompasses proteins obtained from organisms other than Bacillus wherein the protein cross-reacts with antibodies raised against the proteins of the invention. Again the isolated proteins can be assayed for pesticidal activity by the methods disclosed herein or others well-known in the art.<br>
Once the nucleotide sequences encoding the pesticidal proteins of the invention have been isolated, they can be manipulated and used to express the protein in a variety of hosts including other organisms, including microorganisms and plants.<br>
The pesticidal genes of the invention can be optimized for enhanced expression in plants. See, for example EP-A 0618976; EP-A 0359472; EP-A 0385962; WO 91/16432; Perlak etal. (1991) Proc. Natl. Acad. Sci. USA 88:3324-3328; and Murray et al. (1989) Nucleic Acids Research 17: 477-498. In this manner, the genes can be synthesized utilizing plant preferred codons. That is the preferred codon for a particular host is the single codon which most frequently encodes that amino acid in that host. The maize preferred codon, for example, for a particular amino acid may be derived from known gene sequences from maize. Maize codon usage for 28 genes from maize plants is found in Murray et al. (1989), Nucleic Acids Research 17:477-498, the disclosure of which is incorporated herein by reference. Synthetic genes can also be made based on the distribution of codons a particular host uses for a particular amino acid.<br>
In this manner, the nucleotide sequences can be optimized for expression in any plant. It is recognized that all or any part of the gene sequence may be optimized or synthetic. That is, synthetic or partially optimized sequences may also be used.<br>
In like manner, the nucleotide sequences can be optimized for expression in any microorganism. For Bacillus preferred codon usage, see, for example US Patent No. 5,024,837 and Johansen etal. (1988) Gene 65:293-304.<br>
Methodologies for the construction of plant expression cassettes as well as the introduction of foreign DNA into plants are described in the art. Such expression cassettes may include promoters, terminators, enhancers, leader sequences, introns and other regulatory sequences operably linked to the pesticidal protein coding sequence. It is further recognized that promoters or terminators of the VIP genes can be used in expression cassettes.<br>
Generally, for the introduction of foreign DNA into plants Ti plasmid vectors have been utilized for the delivery of foreign DNA as well as direct DNA uptake, liposomes, electroporation, micro-injection, and the use of microprojectiles. Such methods had been published in the art. See, for example, Guerche etal., (1987) Plant Science 52:111 -116; Neuhause et al., (1987) Theor. Appl. Genet. 75:30-36; Klein et al., (1987) Nature 327: 70-73; Howell etal., (1980) Science 208:1265; Horsch etal., (1985) Science 227: 1229-1231; DeBlock et al., (1989) Plant Physiology 91:694-701; Methods for Plant Molecular Biology (Weissbach and Weissbach, eds.) Academic Press, Inc. (1988); and Methods in Plant Molecular Biology (Schuler and Zielinski,<br>
eds.) Academic Press, Inc. (1989). See also US patent application serial no. 08/008,374 herein incorporated by reference. See also, EP-A 0193259 and EP-A 0451878. It is understood that the method of transformation will depend upon the plant cell to be transformed.<br>
It is further recognized that the components of the expression cassette may be modified to increase expression. For example, truncated sequences, nucleotide substitutions or other modifications may be employed. See, for example Perlak et al. (1991) Proa Natl. Acad. Sci. USA 88:3324-3328: Murray et al., (1989) Nucleic Acids Research 17:477-498: and WO 91/16432.<br>
The construct may also include any other necessary regulators such as terminators, (Guerineau et al., (1991). Mol. Gen. Genet. 226:141-144; Proudfoot, (1991), Cell. 64:671-674; Sanfacon et al., (1991). Genes Dev.. 5:141-149; Mogen et al., (1990). Plant Cell. 2:1261-1272; Munroe et al., (1990), Gene. 91:151-158; Ballas etaletal., (1989). Nucleic Acids Res.. 17:7891-7903; Joshi et al., (1987). Nucleic Acid Res.. 15:9627-9639); plant translational consensus sequences (Joshi, C.P., (1987), Nucleic Acids Research. 15:6643-6653), introns (Luehrsen and Walbot, (1991), Mol. Gen. Genet. 225:81-93) and the like, operably linked to the nucleotide sequence. It may be beneficial to include 5' leader sequences in the expression cassette construct. Such leader sequences can act to enhance translation. Translational leaders are known in the art and include:<br>
Picornavirus leaders, for example, EMCV leader (encephalomyocarditis 5' noncoding region) (Elroy-Stein, O., Fuerst, T.R., and Moss, B. (1989) PNAS USA 86:6126-6130);<br>
Potyvirus leaders, for example, TEV leader (Tobacco Etch Virus) (Allison et al., (1986); MDMV leader (Maize Dwarf Mosaic Virus); Virology. 154:9-20), and<br>
Human immunoglobulin heavy-chain binding protein (BiP), (Macejak, D.G., and Sarnow, P., (1991), Nature. 353:90-94;<br>
Untranslated leader from the coat protein mRNA of alfalfa mosaic virus (AMV RNA 4), (Jobling, S.A., and Gehrke, L, (1987), Nature. 325:622-625;<br>
Tobacco mosaic virus leader (TMV), (Gallie, D.R. et al., (1989), Molecular Biology of RNA. pages 237-256; and<br>
Maize Chlorotic Mottle Virus leader (MCMV) (Lommel, S.A. et al., (1991), Virology. 81:382-385. See also, Della-Cioppa era/., (1987). Plant Physiology. 84:965-968.<br>
A plant terminator may be utilized in the expression cassette. See, Rosenberg et al., (1987), Gene. 56:125; Guerineau et al., (1991), Mol. Gen. Genet.. 226:141-144; Proudfoot, (1991), Cell. 64:671-674; Sanfacon et al., (1991). Genes Dev.. 5:141-149; Mogen et al., (1990). Plant Cell. 2:1261-1272; Munroe etal., (1990), Gene, 91:151-158: Ballas et al., (1989). Nucleic Acids Res.. 17:7891-7903; Joshi etal., (1987). Nucleic Acid Res.. 15:9627-9639.<br>
For tissue specific expression, the nucleotide sequences of the invention can be operably linked to tissue specific promoters. See, for example, EP-A 0618976, herein incorporated by reference.<br>
Further comprised within the scope of the present invention are transgenic plants, in particular transgenic fertile plants transformed by means of the aforedescribed processes and their asexual and/or sexual progeny, which comprise and preferably also express the pesticidal protein according to the invention. Especially preferred are hybrid plants.<br>
The transgenic plant according to the invention may be a dicotyledonous or a monocotyledonous plant. Preferred are monocotyledonous plants of the Graminaceae family involving Lolium, Zea, Triticum. Triticale. Sorghum, Saccharum. Bromus, Oryzae, Avena, Hordeum. Secale and Setaria plants.<br>
Especially preferred are transgenic maize, wheat, barley, sorghum, rye, oats, turf grasses and rice.<br>
Among the dicotyledonous plants soybean, cotton, tobacco, sugar beet, oilseed rape, and sunflower are especially preferred herein.<br>
The expression 'progeny' is understood to embrace both, "asexually" and "sexually" generated progeny of transgenic plants. This definition is also meant to include all mutants and variants obtainable by means of known processes, such as for example cell fusion or mutant selection and which still exhibit the characteristic properties of the initially transformed parent plant, together with all crossing and fusion products of the transformed plant material.<br>
Another object of the invention concerns the proliferation material of transgenic plants.<br>
The proliferation material of transgenic plants is defined relative to the invention as any plant material that may be propagated sexually or asexually in vivo or in vitro. Particularly preferred within the scope of the present invention are protoplasts, cells,<br>
calli, tissues, organs, seeds, embryos, pollen, egg cells, zygotes, together with any other propagating material obtained from transgenic plants.<br>
Parts of plants, such as for example flowers, stems, fruits, leaves, roots originating in transgenic plants or their progeny previously transformed by means of the process of the invention and therefore consisting at least in part of transgenic cells, are also an object of the present invention.<br>
Before the plant propagation material [fruit, tuber, grains, seed], but expecially seed is sold as a commerical product, it is customarily treated with a protectant coating comprising herbicides, insecticides, fungicides, bactericides, nematicides, molluscicides or mixtures of several of these preparations, if desired together with further carriers, surfactants or application-promoting adjuvants customarily employed in the art of formulation to provide protection against damage caused by bacterial, fungal or animal pests.<br>
In order to treat the seed, the protectant coating may be applied to the seeds either by impregnating the tubers or grains with a liquid formulation or by coating them with a combined wet or dry formulation. In addition, in special cases, other methods of application to plants are possible, eg treatment directed at the buds or the fruit.<br>
The plant seed according to the invention comprising a DMA molecule comprising a nucleotide sequence encoding a pesticidal protein according to the invention may be treated with a seed protectant coating comprising a seed treatment compound, such as, for example, captan, carboxin, thiram (TMTDÂ®), methalaxyl (ApronÂ®) and pirimiphos-methyl (ActellicÂ®) and others that are commonly used in seed treatment. Preferred within the scope of the invention are seed protectant coatings comprising an entomocidal composition according to the invention alone or in combination with one of the a seed protectant coating customarily used in seed treatment.<br>
It is thus a further object of the present invention to provide plant propagation material for cultivated plants, but especially plant seed that is treated with a seed protectant coating as defined hereinbefore.<br>
It is recognized that the genes encoding the pesticidal proteins can be used to transform insect pathogenic organisms. Such organisms include Baculoviruses, fungi, protozoa, bacteria and nematodes.<br>
The Bacillus strains of the invention may be used for protecting agricultural crops and products from pests. Alternatively, a gene encoding the pesticide may be<br>
introduced via a suitable vector into a microbial host, and said host applied to the environment or plants or animals. Microorganism hosts may be selected which are known to occupy the "phytosphere" (phylloplane, phyllosphere, rhizosphere, and/or rhizoplana) of one or more crops of interest. These microorganisms are selected so as to be capable of successfully competing in the particular environment with the wild-type microorganisms, provide for stable maintenance and expression of the gene expressing the polypeptide pesticide, and, desirably, provide for improved protection of the pesticide from environmental degradation and inactivation.<br>
Such microorganisms include bacteria, algae, and fungi. Of particular interest are microorganisms, such as bacteria, e.g., Pseudomonas, Erwinia, Serratia, Klebsiella, Xanthomonas, Streptomyces, Rhizobium, Rhodopseudomonas, Methylius, Agrobacterium, Acetobacter, Lactobacillus, Arthrobacter, Azotobacter, Leuconostoc, and Alcaligenes; fungi, particularly yeast, e.g., Saccharomyces, Cryptococcus, Kluyveromyces, Sporobolomyces, Rhodotorula, and Aureobasidium. Of particular interest are such phytosphere bacterial species as Pseudomonas syringae, Pseudomonas fluorescens, Serratia rnarcescens, Acetobacter xylinum, Agrobacteria, Rhodopseudomonas spheroides, Xanthomonas campestris, Rhizobium melioti, Alcaligenes entrophus, Clavibacter xyli and Azotobacter vinlandir, and phytosphere yeast species such as Rhodotorula rubra, R. glutinis, R. marina, R. aurantiaca, Cryptococcus albidus, C. diffluens, C. laurentii, Saccharomyces rosei, S. pretoriensis, S. cerevisiae, Sporobolomyces rosues, S. odorus, Kluyveromyces veronae, and Aureobasidium pollulans. Of particular interest are the pigmented microorganisms.<br>
A number of ways are available for introducing a gene expressing the pesticidal protein into the microorganism host under conditions which allow for stable maintenance and expression of the gene. For example, expression cassettes can be constructed which include the DNA constructs of interest operably linked with the transcriptional and translational regulatory signals for expression of the DNA constructs, and a DNA sequence homologous with a sequence in the host organism, whereby integration will occur, and/or a replication system which is functional in the host, whereby integration or stable maintenance will occur.<br>
Transcriptional and translational regulatory signals include but are not limited to promoter, transcriptional initiation start site, operators, activators, enhancers, other regulatory elements, ribosomal binding sites, an initiation codon, termination signals,<br>
and the like. See, for example, US Patent 5,039,523; US Patent No. 4,853,331; EPO 0480762A2; Sambrook etal. supra: Molecular Cloning, a Laboratory Manual, Maniatis et at. (eds) Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1982); Advanced Bacterial Genetics, Davis et al. (eds.) Cold Spring Harbor Laboratory, Cold Spring Harbor, NY (1980); and the references cited therein.<br>
Suitable host cells, where the pesticide-containing cells will be treated to prolong the activity of the toxin in the cell when the then treated cell is applied to the environment of the target pest(s), may include either prokaryotes or eukaryotes, normally being limited to those cells which do not produce substances toxic to higher organisms, such as mammals. However, organisms which produce substances toxic to higher organisms could be used, where the toxin is unstable or the level of application sufficiently low as to avoid any possibility of toxicity to a mammalian host. As hosts, of particular interest will be the prokaryotes and the lower eukaryotes, such as fungi. Illustrative prokaryotes, both Gram-negative and -positive, include Enterobacteriaceae, such as Escherichia, Erwinia, Shigella, Salmonella, and Proteus; Bacillaceae; Rhizobiceae, such as Rhizobium; Spirillaceae, such as photobacterium, Zymomonas, Serratia, Aeromonas, Vibrio, Desulfovibrio, Spirillum; Lactobacillaceae; Pseudomonadaceae, such as Pseudomonas and Acetobacter; Azotobacteraceae and Nitrobacteraceae. Among eukaryotes are fungi, such as Phycomycetes and Ascomycetes, which includes yeast, such a Saccharomyces and Schizosaccharromyces', and Basidiomycetes yeast, such as Rhodotorula, Aureobasidium, Sporobolomyces, and the like.<br>
Characteristics of particular interest in selecting a host cell for purposes of production include ease of introducing the protein gene into the host, availability of expression systems, efficiency of expression, stability of the protein in the host, and the presence of auxiliary genetic capabilities. Characteristics of interest for use as a pesticide microcapsule include protective qualities for the pesticide, such as thick cell walls, pigmentation, and intracellular packaging or formation of inclusion bodies; leaf affinity; lack of mammalian toxicity; attractiveness to pests for ingestion; ease of killing and fixing without damage to the toxin; and the like. Other considerations include ease of formulation and handling, economics, storage stability, and the like.<br>
Host organisms of particular interest include yeast, such as Rhodotorula sp., Aureobasidium sp., Saccharomyces sp., and Sporobolomyces sp.; phylloplane<br>
organisms such as Pseudomonas sp., Erwinia sp. and Flavobacterium sp.', or such other organisms as Escherichia, LactoBacillus sp., Bacillus sp., and the like. Specific organisms include Pseudomonas aeurginosa, Pseudomonas fluorescens, Saccharomyces cerevisiae, Bacillus thuringiensis, Escherichia coli, Bacillus subtilis, and the like.<br>
VIP genes can be introduced into micro-organisms that multiply on plants (epiphytes) to deliver VIP proteins to potential target pests. Epiphytes can be gram-positive or gram-negative bacteria for example.<br>
Root colonizing bacteria, for example, can be isolated from the plant of interest by methods known in the art. Specifically, a Bacillus cereus strain which colonizes roots could be isolated from roots of a plant (for example see J. Handelsman, S. Raffel, E. Mester, L Wunderlich and C. Grau, add!. Environ. Microbiol. 56:713-718, (1990)). VIP1 and/or VIP2 and/or VIP3 could be introduced into a root colonizing Bacillus cereus by standard methods known in the art.<br>
Specifically, VIP1 and/or VIP2 derived from Bacillus cereus strain AB78 can be introduced into a root colonizing Bacillus cereus by means of conjugation using standard methods (J. Gonzalez, B. Brown and B. Carlton, Proc. Natl. Acad. Sci. 79:6951-6955, (1982)).<br>
Also, VIP1 and/or VIP2 and/or VIP3 or other VIPs of the invention can be introduced into the root colonizing Bacillus by means of electro-transformation. Specifically, VIPs can be cloned into a shuttle vector, for example, pHT3101 (D. Lereclus et a/., FEMS Microbiol. Letts.. 60:211-218 (1989)) as described in Example 10. The shuttle vector pHT3101 containing the coding sequence for the particular VIP can then be transformed into the root colonizing Bacillus by means of electroporation (D. Lereclus et al. 1989. FEMS Microbiol. Letts. 60:211-218).<br>
Expression systems can be designed so that VIP proteins are secreted outside the cytoplasm of gram negative bacteria, E. coli, for example. Advantages of having VIP proteins secreted are (1) it avoids potential toxic effects of VIP proteins expressed within the cytoplasm and (2) it can increase the level of VIP protein expressed and (3) can aid in efficient purification of VIP protein.<br>
VIP proteins can be made to be secreted in E. coli, for example, by fusing an appropriate E. coli signal peptide to the amino-terminal end of the VIP signal peptide or replacing the VIP signal peptide with the E. coli signal peptide. Signal peptides<br>
recognized by E. co//can be found in proteins already known to be secreted in E. coli, for example the OmpA protein (J. Ghrayeb, H. Kimura, M. Takahara, Y. Masui and M. Inouye, EMBO J.. 3:2437-2442 (1984)).   OmpA is a major protein of the E. co//outer membrane and thus its signal peptide is thought to be efficient in the translocation process. Also, the OmpA signal peptide does not need to be modified before processing as may be the case for other signal peptides, for example lipoprotein signal peptide (G. Duffaud, P. March and M. Inouye, Methods in Enzvmoloav. 153:492 (1987)).<br>
Specifically, unique BamHI restriction sites can be introduced at the amino-terminal and carboxy-terminal ends of the VIP coding sequences using standard methods known in the art. These BamHI fragments can be cloned, in frame, into the vector pIN-lll-ompAl, A2 or A3 (J. Ghrayeb, H. Kimura, M. Takahara, H. Hsiung, Y. Masui and M. Inouye, EMBO J.. 3:2437-2442 (1984)) thereby creating ompA:VIP fusion gene which is secreted into the periplasmic space. The other restriction sites in the polylinker of pIN-lll-ompA can be eliminated by standard methods known in the art so that the VIP amino-terminal amino acid coding sequence is directly after the ompA signal peptide cleavage site. Thus, the secreted VIP sequence in E. coli would then be identical to the native VIP sequence.<br>
When the VIP native signal peptide is not needed for proper folding of the mature protein, such signal sequences can be removed and replaced with the ompA signal sequence. Unique BamHI restriction sites can be introduced at the amino-termini of the proprotein coding sequences directly after the signal peptide coding sequences of VIP and at the carboxy-termini of VIP coding sequence. These BamHI fragments can then be cloned into the pIN-lll-ompA vectors as described above.<br>
General methods for employing the strains of the invention in pesticide control or in engineering other organisms as pesticidal agents are known in the art. See, for example US Patent No. 5,039,523 and EP 0480762A2.<br>
VIPs can be fermented in a bacterial host and the resulting bacteria processed and used as a microbial spray in the same manner that Bacillus thuringiensis strains have been used as insecticidal sprays. In the case of a VIP(s) which is secreted from Bacillus, the secretion signal is removed or mutated using procedures known in the art. Such mutations and/or deletions prevent secretion of the VIP protein(s) into the growth medium during the fermentation process. The VIPs are retained within the cell<br>
and the cells are then processed to yield the encapsulated VIPs. Any suitable microorganism can be used for this purpose. Psuedomonas has been used to express Bacillus thuringiensis endotoxins as encapsulated proteins and the resulting cells processed and sprayed as an insecticide. (H. Gaertner etal. 1993, In Advanced Engineered Pesticides, L. Kim ed.)<br>
Various strains of Bacillus thuringiensis are used in this manner. Such Bt strains produce endotoxin protein(s) as well as VIPs. Alternatively, such strains can produce only VIPs. A sporulation deficient strain of Bacillus subtilis has been shown to produce high levels of the CrylllA endotoxin from Bacillus thuringiensis (Agaisse, H. and Lereclus, D., "Expression in Bacillus subtilis of the Bacillus thuringiensis CrylllA toxin gene is not dependent on a sporulation-specific sigma factor and is increased in a spoOA mutant", J. BacterioL 176:4734-4741 (1994)). A similar spoOA mutant can be prepared in Bacillus thuringiensis and used to produce encapsulated VIPs which are not secreted into the medium but are retained within the cell.<br>
To have VIPs maintained within the Bacillus cell the signal peptide can be disarmed so that it no longer functions as a secretion signal. Specifically, the putative signal peptide for VIP1 encompasses the first 31 amino acids of the protein with the putative consensus cleavage site, Ala-X-Ala, at the C-terminal portion of this sequence (G. von Heijne , J. Mol. Biol. 184:99-105 (1989)) and the putative signal peptide for VIP2 encompasses the first 40 amino acids of the protein with the putative cleavage site after Ala40. The cleavage sites in either VIP1 or VIP2 can be mutated with methods known in the art to replace the cleavage site consensus sequence with alternative amino acids that are not recognized by the signal peptidases.<br>
Alternatively, the signal peptides of VIP1, VIP2 and/or other VIPs of the invention can be eliminated from the sequence thereby making them unrecognizable as secretion proteins in Bacillus. Specifically, a methionine start site can be engineered in front of the proprotein sequence in VIP1, starting at Asp32, or the proprotein sequence in VIP2, starting at Glu41 using methods known in the art.<br>
VIP genes can be introduced into micro-organisms that mutiply on plants (epiphytes) to deliver VIP proteins to potential target pests. Epiphytes can be gram-positive or gram-negative bacteria for example.<br>
The Bacillus strains of the invention or the microorganisms which have been genetically altered to contain the pesticidal gene and protein may be used for<br>
protecting agricultural crops and products from pests. In one aspect of the invention, whole, i.e., unlysed, cells of a toxin (pesticide)-producing organism are treated with reagents that prolong the activity of the toxin produced in the cell when the cell is applied to the environment of target pest(s).<br>
Alternatively, the pesticides are produced by introducing a heterologous gene into a cellular host. Expression of the heterologous gene results, directly or indirectly, in the intracellular production and maintenance of the pesticide. These cells are then treated under conditions that prolong the activity of the toxin produced in the cell when the cell is applied to the environment of target pest(s). The resulting product retains the toxicity of the toxin. These naturally encapsulated pesticides may then be formulated in accordance with conventional techniques for application to the environment hosting a target pest, e.g., soil, water, and foliage of plants. See, for example EPA 0192319, and the references cited therein.<br>
The active ingredients of the present invention are normally applied in the form of compositions and can be applied to the crop area or plant to be treated, simultaneously or in succession, with other compounds. These compounds can be both fertilizers or micronutrient donors or other preparations that influence plant growth. They can also be selective herbicides, insecticides, fungicides, bactericides, nematicides, mollusicides or mixtures of several of these preparations, if desired, together with further agriculturally acceptable carriers, surfactants or application-promoting adjuvants customarily employed in the art of formulation. Suitable carriers and adjuvants can be solid or liquid and correspond to the substances ordinarily employed in formulation technology, e.g. natural or regenerated mineral substances, solvents, dispersants, wetting agents, tackifiers, binders or fertilizers.<br>
Preferred methods of applying an active ingredient of the present invention or an agrochemical composition of the present invention which contains at least one of the insect-specific proteins produced by the bacterial strains of the present invention are leaf application, seed coating and soil application. The number of applications and the rate of application depend on the intensity of infestation by the corresponding pest.<br>
The present invention thus further provides an entomocidal composition comprising as an active ingrdient at least one of the novel insect-specific proteins<br>
according to the invention and/or a recombinant microorganism containing at least one DMA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, but especially a recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis, containing at least one one DMA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, or a derivative or mutant thereof, together with an agricultural adjuvant such as a carrier, diluent, surfactant or application-promoting adjuvant. The composition may also contain a further biologically active compound. The said compound can be both a fertilizer or micronutrient donor or other preparations that influence plant growth. It can also be a selective herbicide, insecticide, fungicide, bactericide, nematicide, molluscide or mixtures of several of these preparations, if desired, together with further agriculturally acceptable carriers, surfactants or application-promoting adjuvants customarily employed in the art of formulation. Suitable carriers and adjuvants can be solid or liquid and correspond to the substances ordinarily employed in formulation technology, e.g. natural or regenerated mineral substances, solvents, dispersants, wetting agents, tackifiers, binders or fertilizers<br>
The composition may comprise from 0.1 to 99% by weight of the active ingredient, from 1 to 99.9% by weight of a solid or liquid adjuvant, and from 0 to 25% by weight of a surfactant. The acitve ingredient comprising at least one of the novel insect-specific proteins according to the invention or a recombinant microorganism containing at least one DNA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, but especially a recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis strain containing at least one DNA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, or a derivative or mutant thereof, or the composition containing the said acitve ingredient, may be administered to the plants or crops to be protected together with certain other insecticides or chemicals (1993 Crop Protection Chemicals Reference, Chemical and Pharmaceutical Press, Canada) without loss of potency. It is compatible with most other commonly used agricultural spray materials but should not be used in extremely alkaline spray solutions. It may be administered as a dust, a suspension, a wettable powder or in any other material form suitable for agricultural application.<br>
The invention further provides methods for for controlling or inhibiting of insect pests by applying an active ingredient comprising at least one of the novel insect-specific proteins according to the invention or a recombinant microorganism containing at least one ONA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form or a composition comprising the said active ingredient to (a) an environment in which the insect pest may occur, (b) a plant or plant part in order to protect said plant or plant part from damage caused by an insect pest, or (c) seed in order to protect a plant which develops from said seed from damage caused by an insect pest.<br>
A preferred method of application in the area of plant protection is application to the foliage of the plants (foliar application), with the number of applications and the rate of application depending on the plant to be protected and the risk of infestation by the pest in question. However, the active ingredient may also penetrate the plants through the roots (systemic action) if the locus of the plants is impregnated with a liquid formulation or if the active ingredient is incorporated in solid form into the locus of the plants, for example into the soil, e.g. in granular form (soil application). In paddy rice crops, such granules may be applied in metered amounts to the flooded rice field.<br>
The compositions according to the invention are also suitable for protecting plant propagating material, e.g. seed, such as fruit, tubers or grains, or plant cuttings, from insect pests. The propagation material can be treated with the formulation before planting: seed, for example, can be dressed before being sown. The acitve ingredient of the invention can also be applied to grains (coating), either by impregnating the grains with a liquid formulation or by coating them with a solid formulation. The formulation can also be applied to the planting site when the propagating material is being planted, for example to the seed furrow during sowing. The invention relates also to those methods of treating plant propagation material and to the plant propagation material thus treated.<br>
The compositions according to the invention comprising as an active ingredient a recombinant microorganism containing at least one of the novel toxin genes in recombinant form, but especially a recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis strain containing at least one DMA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, or a derivative or mutant thereof may be applied in any method<br>
known for treatment of seed or soil with bacterial strains. For example, see US Patent No.4,863,866. The strains are effective for biocontrol even if the microorganism is not living. Preferred is, however, the application of the living microorganism.<br>
Target crops to be protected within the scope of the present invention comprise, e.g., the following species of plants:<br>
cereals (wheat, barley, rye, oats, rice, sorghum and related crops), beet (sugar beet and fodder beet), forage grasses (orchardgrass, fescue, and the like), drupes, pomes and soft fruit (apples, pears, plums, peaches, almonds, cherries, strawberries, raspberries and blackberries), leguminous plants (beans, lentils, peas, soybeans), oil plants (rape, mustard, poppy, olives, sunflowers, coconuts, castor oil plants, cocoa beans, groundnuts), cucumber plants (cucumber, marrows, melons) fiber plants (cotton, flax, hemp, jute), citrus fruit (oranges, lemons, grapefruit, mandarins), vegetables (spinach, lettuce, asparagus, cabbages and other Brassicae, onions, tomatoes, potatoes, paprika), lauraceae (avocados, carrots, cinnamon, camphor), deciduous trees and conifers (e.g. linden-trees, yew-trees, oak-trees, alders, poplars, birch-trees, firs, larches, pines), or plants such as maize, tobacco, nuts, coffee, sugar cane, tea, vines, hops, bananas and natural rubber plants, as well as ornamentals (including composites).<br>
A recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis strain, containing at least one DNA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form is normally applied in the form of entomocidal compositions and can be applied to the crop area or plant to be treated, simultaneously or in succession, with further biologically active compounds. These compounds may be both fertilizers or micronutrient donors or other preparations that influence plant growth. They may also be selective herbicides, insecticides, fungicides, bactericides, nematicides, molluscicides or mixtures of several of these preparations, if desired together with further carriers, surfactants or application-promoting adjuvants customarily employed in the art of formulation.<br>
The active ingredient according to the invention may be used in unmodified form or together with any suitable agriculturally acceptable carrier. Such carriers are adjuvants conventionally employed in the art of agricultural formulation, and are therefore formulated in known manner to emulsifiable concentrates, coatable pastes, directly sprayable or dilutable solutions, dilute emulsions, wettable powders, soluble powders,<br>
dusts, granulates, and also encapsulations, for example, in polymer substances. Like the nature of the compositions, the methods of application, such as spraying, atomizing, dusting, scattering or pouring, are chosen in accordance with the intended objective and the prevailing circumstances. Advantageous rates of application are normally from about 50 g to about 5 kg of active ingredient (a.i.) per hectare ("ha", approximately 2.471 acres), preferably from about 100 g to about 2kg a.i./ha. Important rates of application are about 200 g to about 1kg a.i./ha and 200g to 500g a.i./ha.<br>
For seed dressing advantageous application rates are 0.5 g to 1000 g a.i.per 100 kg seed, preferably 3 g to 100 g a.i. per 100 kg seed or 10 g to 50 g a.i.per 100 kg seed.<br>
Suitable carriers and adjuvants can be solid or liquid and correspond to the substances ordinarily employed in formulation technology, e.g. natural or regenerated mineral substances, solvents, dispersants, wetting agents, tackifiers, binders or fertilizers. The formulations, i.e. the entomocidal compositions, preparations or mixtures containing the recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis strain containing at least one DMA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form as an active ingredient or combinations thereof with other active ingredients, and, where appropriate, a solid or liquid adjuvant, are prepared in known manner, e.g., by homogeneously mixing and/or grinding the active ingredients with extenders, e.g., solvents, solid carriers, and in some cases surface-active compounds (surfactants).<br>
Suitable solvents are: aromatic hydrocarbons, preferably the fractions containing 8 to 12 carbon atoms, e.g. xylene mixtures or substituted naphthalenes, phthalates such as dibutyl phthalate or dioctyl phthalate, aliphatic hydrocarbons such as cyclohexane or paraffins, alcohols and glycols and their ethers and esters, such as ethanol, ethylene glycol monomethyl or monoethyl ether, ketones such as cyclohexanone, strongly polar solvents such as N-methyl-2-pyrrolidone, dimethylsulfoxide or dimethylformamide, as well as vegetable oils or epoxidised vegetable oils such as epoxidised coconut oil or soybean oil; or water.<br>
The solid carriers used, e.g., for dusts and dispersible powders, are normally natural mineral fillers such as calcite, talcum, kaolin, montmorillonite or attapulgite. In order to improve the physical properties it is also possible to add highly dispersed silicic acid or highly dispersed absorbent polymers. Suitable granulated adsorptive<br>
carriers are porous types, for example pumice, broken brick, sepiolite or bentonite; and suitable nonsorbent carriers are materials such as calcite or sand. In addition, a great number of pregranulated materials of inorganic or organic nature can be used, e.g. especially dolomite or pulverized plant residues.<br>
Depending on the nature of the active ingredients to be formulated, suitable surface-active compounds are non-ionic, cationic and/or anionic surfactants having good emulsifying, dispersing and wetting properties. The term "surfactants" will also be understood as comprising mixtures of surfactants. Suitable anionic surfactants can be both water-soluble soaps and<br>
water-soluble synthetic surface-active compounds. Suitable soaps are the alkali metal salts, alkaline earth metal salts or unsubstituted or substituted ammonium salts of higher fatty acids (Ci0 -Cza), e.g. the sodium or potassium salts of oleic or stearic acid, or of natural fatty acid mixtures which can be obtained, e.g. from coconut oil or tallow oil. Further suitable surfactants are also the fatty acid methyltaurin salts as well as modified and unmodified phospholipids.<br>
More frequently, however, so-called synthetic surfactants are used, especially fatty sulfonates, fatty sulfates, sulfonated benzimidazole derivatives or alkylarylsulfonates. The fatty sulfonates or sulfates are usually in the forms of alkali metal salts, alkaline earth metal salts or unsubstituted or substituted ammonium salts and generally contain a C8 -C22 alkyl radical which also includes the alkyl moiety of acyl radicals, e.g. the sodium or calcium salt of lignosulfonic acid, of dodecylsulfate, or of a mixture of fatty alcohol sulfates obtained from natural fatty acids. These compounds also comprise the salts of sulfuric acid esters and sulfonic acids of fatty alcohol/ethylene oxide adducts. The sulfonated benzimidazole derivatives preferably contain 2 sulfonic acid groups and one fatty acid radical containing about 8 to 22 carbon atoms. Examples of alkylarylsulfonates are the sodium, calcium or triethanolamine salts of dodecylbenzenesulfonic acid, dibutylnaphthalenesulfonic acid, or of a naphthalenesulfonic acid/formaldehyde condensation product. Also suitable are corresponding phosphates, e.g. salts of the phosphoric acid ester of an adduct of p-nonylphenol with 4 to 14 moles of ethylene oxide.<br>
Non-ionic surfactant are preferably polyglycol ether derivatives of aliphatic or cycloaliphatic alcohols, or saturated or unsaturated fatty acids and alkylphenols, said derivatives containing 3 to 30 glycol ether groups and 8 to 20 carbon atoms in the<br>
(aliphatic) hydrocarbon moiety and 6 to 18 carbon atoms in the alkyl moiety of the alkylphenols.<br>
Further suitable non-ionic surfactants are the water-soluble adducts of polyethylene oxide with polypropylene glycol, ethylenediaminopolypropylene glycol and alkylpolypropylene glycol containing 1 to 10 carbon atoms in the alkyl chain, which adducts contain 20 to 250 ethylene glycol ether groups and 10 to 100 propylene glycol ether groups. These compounds usually contain 1 to 5 ethylene glycol units per propylene glycol unit. Representative examples of non-ionic surfactants are nonylphenolpolyethoxyethanols, castor oil polyglycol ethers, polypropylene/polyethylene oxide adducts, tributylphenoxypolyethoxyethanol, polyethylene glycol and octylphenoxypolyethoxyethanol. Fatty acid esters of polyoxyethylene sorbitan, such as polyoxyethylene sorbitan trioleate, are also suitable non-ionic surfactants.<br>
Cationic surfactants are preferably quaternary ammonium salts which contain, as N-substituent, at least one C8 -C22 alkyl radical and, as further substituents, lower unsubstituted or halogenated alkyl, benzyl or hydroxyl-lower alkyl radicals. The salts are preferably in the form of halides, methylsulfates or ethylsulfates, e.g., stearyltrimethylammonium chloride or benzyldi-(2-chloroethyl)ethylammonium bromide.<br>
The surfactants customarily employed in the art of formulation are described, e.g., in "McCutcheon's Detergents and Emulsifiers Annual", MC Publishing Corp. Ridgewood, N.J., 1979; Dr. Helmut Stache, "Tensid Taschenbuch" (Handbook of Surfactants), Carl Hanser Verlag, Munich/Vienna.<br>
Another particularly preferred characteristic of an entomocidal composition of the present invention is the persistence of the active ingredient when applied to plants and soil. Possible causes for loss of activity include inactivation by ultra-violet light, heat, leaf exudates and pH. For example, at high pH, particularly in the presence of reductant, 5-endotoxin crystals are solubilized and thus become more accessible to proteolytic inactivation. High leaf pH might also be important, particularly where the leaf surface can be in the range of pH 8-10. Formulation of an entomocidal composition of the present invention can address these problems by either including additives to help prevent loss of the active ingredient or encapsulating the material in such a way that the active ingredient is protected from inactivation. Encapsulation<br>
can be accomplished chemically (McGuire and Shasha, J Econ Entomol 85: 1425-1433, 1992) or biologically (Barnes and Cummings, 1986; EP-A 0 192 319). Chemical encapsulation involves a process in which the active ingredient is coated with a polymer while biological encapsulation involves the expression of the 5-endotoxin genes in a microbe. For biological encapsulation, the intact microbe containing at least one DNA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form is used as the active ingredient in the formulation. The addition of UV protectants might effectively reduce irradiation damage. Inactivation due to heat could also be controlled by including an appropriate additive.<br>
Preferred within the present application are formulations comprising living microorganisms as active ingredient either in form of the vegetative cell or more preferable in form of spores, if available. Suitable formulations may consist, for example, of polymer gels which are crosslinked with polyvalent cations and comprise these microorganisms. This is described, for example, by D.R. Fravel et al. in Phytopathology, Vol. 75, No. 7, 774-777, 1985 for alginate as the polymer material. It is also known from this publication that carrier materials can be co-used. These formulations are as a rule prepared by mixing solutions of naturally occurring or synthetic gel-forming polymers, for example alginates, and aqueous salt solutions of polyvalent metal ions such that individual droplets form, it being possible for the microorganisms to be suspended in one of the two or in both reaction solutions. Gel formation starts with the mixing in drop form. Subsequent drying of these gel particles is possible. This process is called ionotropic gelling. Depending on the degree of drying, compact and hard particles of polymers which are structurally crosslinked via polyvalent cations and comprise the microorganisms and a carrier present predominantly uniformly distributed are formed. The size of the particles can be up to 5 mm.<br>
Compositions based on partly crosslinked polysaccharides which, in addition to a microorganism, for example, can also comprise finely divided silicic acid as the carrier material, crosslinking taking place, for example, via Ca++ ions, are described in EP-A1-0 097 571. The compositions have a water activity of not more than 0.3. W.J. Cornick et al. describe in a review article [New Directions in Biological Control: Alternatives for Suppressing Agricultural Pests and Diseases, pages 345-372, Alan R.<br>
Liss, Inc. (1990)] various formulation systems, granules with vermiculite as the carrier and compact alginate beads prepared by the ionotropic gelling process being mentioned. Such compositions are also disclosed by D.R.Fravel in Pesticide Formulations and Application Systems: 11th Volume, ASTM STP 1112 American Society for Testing and Materials, Philadelphia, 1992, pages 173 to 179 and can be used to formulate the recombinant microorganisms according to the invention.<br>
The entomocidal compositions of the invention usually contain from about 0.1 to about 99%, preferably about 0.1 to about 95%, and most preferably from about 3 to about 90% of the active ingredient, from about 1 to about 99.9%, preferably from about 1 to about 99%, and most preferably from about 5 to about 95% of a solid or liquid adjuvant, and from about 0 to about 25%, preferably about 0.1 to about 25%, and most preferably from about 0.1 to about 20% of a surfactant.<br>
In a preferred embodiment of the invention the entomocidal compositions usually contain 0.1 to 99%, preferably 0.1 to 95%, of a recombinant Bacillus spp strain, such as Bacillus cereus or Bacillus thuringiensis strain containing at least one DNA molecule comprising a nucleotide sequence encoding the novel insect-specific proteins in recombinant form, or combination thereof with other active ingredients, 1 to 99.9% of a solid or liquid adjuvant, and 0 to 25%, preferably 0.1 to 20%, of a surfactant.<br>
Whereas commercial products are preferably formulated as concentrates, the end user will normally employ dilute formulations of substantially lower concentration. The entomocidal compositions may also contain further ingredients, such as stabilizers, antifoams, viscosity regulators, binders, tackifiers as well as fertilizers or other active ingredients in order to obtain special effects.<br>
In one embodiment of the invention a Bacillus cereus microorganism has been isolated which is capable of killing Diabrotica virgifera virgifera, and Diabrotica longicornis barberi. The novel B. cereus strain AB78 has been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, IL 61604. USA and given Accession No. NRRL B-21058.<br>
A fraction protein has been substantially purified from the B. cereus strain. This purification of the protein has been verified by SDS-PAGE and biological activity. The protein has a molecular weight of about 60 to about 100 KDa, particularly about 70 to about 90 kDa, more particularly about 00 kDa, hereinafter VIP.<br>
Amino-terminal sequencing has revealed the N-terminal amino-acid sequence to be:<br>
NH2-Lys-Arg-Glu-lle-Asp-Glu-Asp-Thr-Asp-Thr-AsX'Gly-Asp-Ser-lle-Pro-(SEQ ID NO:8) where Asx represents either Asp or Asn. The entire amino acid sequence is given in SEQ ID NO:7. The DNA sequence which encodes the amino acid sequence of SEQ ID NO:7 is disclosed in SEQ ID NO:6.<br>
An oligonuleotide probe for the region of the gene encoding amino acids 3-9 of the Nf^-terminus has bean generated. The probe was synthesized based on the codon usage of a Bacillus thuringiensis (Bt) 8-endotoxin gene. The nucleotide sequence of the oligonucleotide probe used for Southern hybridizations was as follows:<br>
(Sequence Removed)<br>
where N represents any base.<br>
In addition, the DNA probe for the Bc AB78 VIP1 gene described herein, permits the screening of any Bacillus strain or other organisms to determine whether the VIP1 gene (or related gene) is naturally present or whether a particular transformed organism includes the VIP1 gene. The present invention relates to an expression cassette comprising a DNA molecule operably linked to plant expression sequences including the transcriptional and translational regulatory signals necessary for expression of the associated DNA constructs in a host organism and optionally further regulatory sequences, wherein said DNA molecule encodes a vegetative insecticidal protein isolatable from liquid culture media during the vegetative growth phase of Bacillus spp., and wherein said protein is encoded by a nucleotide sequence that hybridizes to a nucleotide sequence of SEQ ID NOs: 28, 30 or 31 at 65 Â°C in a buffer comprising 7 % SDS and 0.5 M sodium phosphate.<br>
Further, the present invention also relates to an insect-specific protein substantially as herein described with reference to the foregoing examples<br>
The invention now being generally described, the same will be better understood by reference to the following detailed examples that are provided for the purpose of illustration and are not to be considered limiting of the invention unless so specified.<br>
A standard nomenclature has been developed based on the sequence identity of the proteins encompassed by the present invention. The gene and protein names for the detailed examples which follow and their relationship to the names used in the parent application (US application serial no 314594/08] are shown below. <br>
Gene I Protein Name under<br>
Standard Nomenclature<br>
VIP1A(a)<br>
Gene/<br>
Protein<br>
Name in<br>
Parent<br>
VIP1<br>
Description of Protein<br>
VIP1 from strain AB78 as disclosed in SEQ ID NO:5.<br><br>
VIP2A(a)<br>
VIP2<br>
VIP2 from strain AB78 as disclosed in SEQ ID NO:2.<br><br>
VIP1A(b)<br>
VIP1 homolog<br>
VIP1 from Bacillus thuringiensis var. tenebrionis as disclosed in SEQ ID NO:21.<br><br>
VIP2A(b)<br>
VIP2 homolog<br>
VIP2 from Bacillus thuringiensis var. tenebrionis as disclosed in SEQ ID IMO:20.<br><br>
VIP3A(a)<br>
VIP from strain AB88 as disclosed in SEQ ID NO:28 of the present application<br><br>
VIP3A(b)<br>
VIP from strain AB424 as disclosed in SEQ ID NO:31 of the present application<br>
EXPERIMENTAL<br>
Formulation Examples<br>
The active ingredient used in the following formulation examples are Bacillus cereus strain AB78 having Accession No. NRRL B-21058; Bacillus thuringiensis strains having Accession Nos. NRRL B-21060, NRRL B-21224, NRRL B-21225, NRRL B-21226, NRRL B-21227, and NRRL B-21439; and Bacillus spp strains having Accession Nos NRRL B-21228, NRRL B-21229, and NRRL B-21230. All the mentioned strains are natural isolates comprising the insect-specific proteins according to the invention.<br>
Alternatively, the isolated insect-specific proteins are used as the active ingredient alone or in combination with the above-mentioned Bacillus strains.<br>
A1.       Wettable powders<br>
a)	b)	c)<br>
Bacillus thuringiensis spores	25%	50%      75%<br>
sodium lignosufonate	5%	5%<br>
sodium laurylsulfate	3%	-	5%<br>
sodium diisobutylnaphthalenesulfonate        -	6%      10%<br>
octylphenol polyethylene glycol ether	--	2%<br>
(7-8 moles of ethylene oxid)<br>
highly dispersed silicid acid	5%	10%      10%<br>
kaolin	62%	27%<br>
The spores are thoroughly mixed with the adjuvants and the mixture is thoroughly ground in a suitable mill, affording wettable powders which can be diluted with water to give suspensions of the desired concentrations.<br>
A2.        Emulsifiable concentrate<br>
Bacillus thuringiensis spores	10%<br>
octylphenol polyethylene glycol ether (4-5 moles ethylene oxide)	3%<br>
clacium dodecylbenzensulfonate	3%<br>
castor oil polyglycol ether (36 moles of ethylene oxide)	4%<br>
ro<br>
cyclohexanone	30%<br>
ro<br>
xylene mixture	50%<br>
Emulsions of any required concentration can be obtained from this concentrate by dilution with water.<br>
A3.	Dusts<br>
a)	b)<br>
Bacillus thuringiensis spores                                     5%	8%<br>
talcum                                                                    95%<br>
kaolin                                                                     --	92%<br>
Ready for use dusts are obtained by mixing the active ingredient with the carriers and grinding the mixture in a suitable mill.<br>
A4.	Extruder Granulate<br>
Bacillus thuringiensis spores	10%<br>
'o<br>
sodium lignosulfonate	2%<br>
'o<br>
carboxymethylcellulose	1%<br>
kaolin	87%<br>
The active ingredient or combination is mixed and ground with the adjuvants and the mixture is subsequently moistened with water. The mixture is extruded, granulated and the dried in a stream of air.<br>
A5.	Coated Granule<br>
Bacillus thuringiensis spores	3%<br>
polyethylene glycol (mol wt 200)	3%<br>
kaolin	94%<br>
'o<br>
The active ingredient or combination is uniformly applied in a mixer to the kaolin moistened with polyethylene glycol. Non-dusty coated granulates are obtained in this manner.<br>
A6.	Suspension Concentrate<br>
Bacillus thuringiensis spores	40%<br>
ethylene glycol	10%<br>
nonylphenol polyethylene glycol ether (15 moles of ethylene oxide)         6%<br>
sodium lignosulfonate	10%<br>
carboxymethylcellulose	1 %<br>
37% aqueous formaldehyde solution	0.2%<br>
silicone oil in the form of a 75% aqueous solution	0.8%<br>
water	32%<br>
The active ingredient or combination is intimately mixed with the adjuvants giving a suspension concentrate from which suspensions of any desired concentration can be obtained by dilution with water.<br>
EXAMPLE 1. AB78 ISOLATION AND CHARACTERIZATION<br>
Bacillus cereus strain AB78 was isolated as a plate contaminant in the laboratory<br>
on T3 media (per liter: 3 g tryptone, 2 g tryptose, 1.5 g yeast extract, 0.05 M sodium phosphate (pH 6.8), and 0.005 g MnCI2; Travers, R.S. 1983). During log phase growth, AB78 gave significant activity against western corn rootworm. Antibiotic activity against gram-positive Bacillus spp. was also demonstrated (Table 12).<br>
TABLE 12<br>
Antibiotic activity of AB78 culture supernatant<br>
Zone of inhibition(cm) (Table Remove) Morphological characteristics of AB78 are as follows:<br>
Vegetative rods straight, 3.1-5.0 mm long and 0.5-2.0 mm wide. Cells with rounded ends, single in short chains. Single subterminal, cylindrical-oval, endospore formed per cell. No parasporal crystal formed. Colonies opaque, erose, lobate and flat. No pigments produced. Cells motile. Flagella present.<br>
Growth characteristics of AB78 are as follows:<br>
Facultative anaerobe with optimum growth temperature of 21-30Â°C. Will grow at 15, 20, 25, 30 and 37Â°C. Will not grow above 40Â°C. Grows in 5-7% NaCI.<br>
Table 13 provides the biochemical profile of AB78.<br>
(Table Remove) <br>
EXAMPLE 2. BACTERIAL CULTURE<br>
A subculture of Be strain AB78 was used to inoculate the following medium, known as TB broth:<br>
Tryptone Yeast Extract Glycerol KH2PO4 K2HPO4 pH 7.4<br>
(Table Remove)<br>
The potassium phosphate was added to the autoclaved broth after cooling. Flasks were incubated at 30Â°C on a rotary shaker at 250 rpm for 24 h-36 h, which represents an early to mid-log growth phase.<br>
The above procedure can be readily scaled up to large fermentors by procedures well known in the art.<br>
During vegetative growth, usually 24-36 h. after starting the culture, which represents an early to mid-log growth phase, AB78 bacteria were centrifuged from the culture supernatant. The culture supernatant containing the active protein was used in bioassays.<br>
EXAMPLE 3.         INSECT BIOASSAYS<br>
B. cereus strain AB78 was tested against various insects as described below.<br>
Western, Northern and Southern corn rootworm, Diabrotica virgifera virgifera, D. longcornis barberiand D. undecempunctata howardi, respectively: dilutions were made of AB78 culture supernatant grown 24-36 h., mixed with molten artificial diet (Marrone et al. (1985) J^ of Economic Entomology 78:290-293) and allowed to solidify. Solidified diet was cut and placed in dishes. Neonate larvae were placed on the diet and held at 30 C. Mortality was recorded after 6 days.<br>
E. coli clone bioassav: E. co//cells were grown overnight in broth containing 100 ng/ml ampicillin at 37Â°C. Ten ml culture was sonicated 3X for 20 sec each. 500 |il of sonicated culture was added to molten western corn rootworm diet.<br>
Colorado potato beetle, Leptinotarsa decemlineata: dilutions in Triton X-100 (to give final concentration of 0.1% TX-100) were made of AB78 culture supernatant grown 24-36 h. Five cm2 potato leaf pieces were dipped into these dilutions, air dried, and placed on moistened filter paper in plastic dishes. Neonate larvae were placed on the leaf pieces and held at 30Â°C. Mortality was recorded after 3-5 days.<br>
Yellow mealworm, Tenebrio molitor. dilutions were made of AB78 culture supernatant grown 24-36 h., mixed with molten artificial diet (Bioserv #F9240) and allowed to solidify. Solidified diet was cut and placed in plastic dishes. Neonate larvae were placed on the diet and held at 30Â°C. Mortality was recorded after 6-8 days.<br>
European corn borer, black cutworm, tobacco budworm, tobacco hornworm and beet armyworm; Ostrinia nubilalis, Agrotis ipsilon, Heliothis virescens, Manduca sexta and Spodoptera exigua, respectively: dilutions, in TX-100 (to give final concentration<br>
of 0.1 % TX-100), were made of AB78 culture supernatant grown 24-36 hrs. 100 u.l<br>
2 was pipetted onto the surface of 18 cm  of solidified artificial diet (Bioserv #F9240)<br>
and allowed to air dry.   Neonate larvae were then placed onto the surface of the diet and held at 30Â°C. Mortality was recorded after 3-6 days. Northern house mosquito, Ct//exp/p/ens:-dilutions were made of AB78 culture supernatant grown 24-36 h. 100 m.! was pipetted into 10 ml water in a 30 ml plastic cup. Third instar larvae were added to the water and held at room temperature. Mortality was recorded after 24-48 hours. The spectrum of entomocidal activity of AB78 is given in Table 14.<br>
(Table Remove) The newly discovered B. cereus strain AB78 showed a significantly different spectrum of insecticidal activity as compared to known coleopteran active 5-endotoxins from Bt. In particular, AB78 showed more selective activity against beetles than known coleopteran-active Bt strains in that it was specifically active against Diabrotica spp. More specifically, it was most active against D. virgifera virgifera and D. longicornis barberibut not D. undecimpunctata howardi.<br>
A number of Bacillus strains were bioassayed for activity during vegetative growth (Table 15) against western corn rootworm. The results demonstrate that AB78 is unique in that activity against western corn rootworm is not a general phenomenon.TABLE 15<br>
Activity of culture supernatants from various Bacillus spp. against western corn rootworm(Table Remove)<br>
Specific activity of AB78 against western corn rootworm is provided in Table 16.<br>
 (Table Remove) he LCso was calculated to be 6.2 |j.l of culture supernatant per ml of western corn rootworm diet.<br>
The cell pellet was also bioassayed and had no activity against WCRW. Thus, the presence of activity only in the supernatant indicates that this VIP is an exotoxin.<br>
EXAMPLE 4. ISOLATION AND PURIFICATION OF CORN ROOTWORM ACTIVE PROTEINS FROM AB78.<br>
Culture media free of cells and debris was made to 70% saturation by the addition of solid ammonium sulfate (472 g/L). Dissolution was at room temperature followed by cooling in an ice bath and centrifugation at 10,000 X g for thirty minutes to pellet the precipitated proteins. The supernatant was discarded and the pellet was dissolved in 1/10 the original volume of 20 mM TRIS-HCI at pH 7.5. The dissolved pellet was desalted either by dialysis in 20 mM TRIS-HCI pH 7.5, or passing through a desalting column.<br>
The desalted material was titrated to pH 3.5 using 20 mM sodium citrate pH 2.5. Following a thirty minute room temperature incubation the solution was centrifuged at<br>
3000 X g for ten minutes. The supernatant at this stage contained the greatest amount of active protein.<br>
Following neutralization of the pH to 7.0 the supernatant was applied to a Mono-Q, anion exchange, column equilibrated with 20 mM TRIS pH 7.5 at a flow rate of 300 mL/min. The column was developed with a stepwise and linear gradient employing 400 mM NaCI in 20 mM TRIS pH 7.5.<br>
Bioassay of the column fractions and SDS-PAGE analysis were used to confirm the active fractions. SDS-PAGE analysis identified the biologically active protein as having components of a molecular weight in the range of about 80 kDa and 50 kDa.<br>
EXAMPLE 5. SEQUENCE ANALYSIS OF THE CORN ROOTWORM ACTIVE PROTEIN<br>
The 80 kDa component isolated by SDS-PAGE was transferred to PVDF membrane and was subjected to amino-terminal sequencing as performed by repetitive Edman cycles on an ABI 470 pulsed-liquid sequencer. Transfer was carried out in 10 mM CAPS buffer with 10% methanol pH 11.0 as follows:<br>
Incubation of the gel following electrophoresis was done in transfer buffer for five minutes. ProBlott PVDF membrane was wetted with 100% MeOH briefly then equilibrated in transfer buffer. The sandwich was arranged between foam sponges and filter paper squares with the configuration of cathode-gel-membrane-anode.<br>
Transfer was performed at 70 V constant voltage for 1 hour.<br>
Following transfer, the membrane was rinsed with water and stained for two minutes with 0.25% Coomassie Blue R-250 in 50% MeOH.<br>
Destaining was done with several rinses with 50% MeOH 40% water 10% acetic acid.<br>
Following destaining the membrane was air dried prior to excision of the bands for sequence analysis. A BlottCartridge and appropriate cycles were utilized to achieve maximum efficiency and yield. Data analysis was performed using model 610 Sequence Analysis software for identifying and quantifying the PTH-amino acid derivatives for each sequential cycle.<br>
The N-terminal sequence was determined to be:<br>
NH2-Lys-Arg-Glu-lle-Asp-Glu-Asp-Thr-Asp-Thr-Asx-Gly-Asp-Ser-lle-Pro-<br>
(SEQ ID NO:8) where Asx represents Asp or Asn. The complete amino acid sequence for the 80 kDa component is disclosed in SEQ ID NO:7. The DMA sequence which encodes SEQ ID NO:7 is disclosed in SEQ ID NO:6.<br>
EXAMPLE 6. CONSTRUCTION OF DMA PROBE<br>
An oligonucleotide probe for the region of the gene encoding amino acids 3-9 of the N-terminal sequence (Example 5) was generated. The probe was synthesized based on the codon usage of a Bacillus thuringiensis (Bt) 8-endotoxin gene. The nucleotide sequence<br>
5'- GAA ATT GAT CAA GAT ACN GAT -3'     (SEQ ID NO:9) was used as a probe in Southern hybridizations. The oligonucleotide was synthesized using standard procedures and equipment.<br>
EXAMPLE 7. ISOELECTRIC POINT DETERMINATION OF THE CORN ROOTWORM ACTIVE PROTEIN<br>
Purified protein from step 5 of the purification process was analyzed on a 3-9 pi isoelectric focusing gel using the Phastgel electrophoresis system (Pharmacia). Standard operating procedures for the unit were followed for both the separation and silver staining development procedures. The pi was approximated at about 4.9.<br>
EXAMPLE 8. PCR DATA ON AB78<br>
PCR analysis (See, for example US patent application serial no. 08/008,006; and, Carozzi et al. (1991) Appl. Environ. Microbiol. 57(11 ):3057-3061, herein incorporated by reference.) was used to verify that the B. cereus strain AB78 did not contain any insecticidal crystal protein genes of B. thuringiensis or B. sphaericus (Table 17).<br>
(Table Remove) <br>
EXAMPLE 9. COSMID CLONING OF TOTAL DNA FROM B. CEREUS STRAIN AB78<br>
The VIP1 A(a) gene was cloned from total DNA prepared from strain AB78 as follows:<br>
Isolation of AB78 DNA was as follows:<br>
Grow bacteria in 10 ml L-broth overnight. (Use 50 ml sterile centrifuge tube)<br>
Add 25 ml of fresh L-broth and ampicillin (30 ng/ml).<br>
Grow cells 2-6 h. at 30Â°C with shaking.<br>
Spin cells in a 50 ml polypropylene orange cap tube in IEC benchtop clinical<br>
centrifuge at 3/4 speed.<br>
Resuspend cell pellet in 10 ml TES (TES = 50 mM TRIS pH 8.0, 100 mM EDTA, 15<br>
mM NaCI).<br>
Add 30 mg lysozyme and incubate 2 hrs at 37Â°C.<br>
7.	Add 200 |il 20% SDS and 400 (il Proteinase K stock (20 mg/ml). Incubate at 37Â°C.<br>
Add 200 |il fresh Proteinase K. Incubate 1 hr. at 55Â°C. Add 5 ml TES to make 15<br>
ml final volume.<br>
Phenol extract twice (10 ml phenol, spin at room temperature at 3/4 speed in an<br>
(EC benchtop clinical centrifuge). Transfer supernatant (upper phase) to a clean tube<br>
using a wide bore pipette.<br><br>
Extract once with 1:1 vol. phenolrchloroform/isoamyl alcohol (24:1 ratio).<br>
Precipitate DNA with an equal volume of cold isopropanol; Centrifuge to<br>
pellet DNA.<br>
Resuspend pellet in 5 ml TE.<br>
Precipitate DNA with 0.5 ml 3M NaOAc pH 5.2 and 11 ml 95% ethanol. Place<br>
at-20Â°Cfor2h.<br>
"Hook" DNA from tube with a plastic loop, transfer to a microfuge tube, spin,<br>
pipette off excess ethanol, dry in vacuo.<br>
Resuspend in 0.5 ml TE. Incubate 90 min. at 65Â°C to help get DNA back into<br>
solution.<br>
Determine concentration using standard procedures.<br>
Cosmid Cloning of AB78<br>
All procedures, unless indicated otherwise, were performed according to Stratagene Protocol, Supercos 1 Instruction Manual, Cat. No. 251301.<br>
Generally, the steps were as follows:<br>
A.	Sau 3A partial digestion of the AB78 DNA.<br>
B.	Preparation of vector DNA<br>
C.	Ligation and packaging of DNA<br>
D.	Tittering the cosmid library<br>
Start a culture of HB101 cells by placing 50 ml of an overnight culture in<br>
5 mis of TB with 0.2% maltose. Incubate 3.5 hrs. at 37Â°C.<br>
Spin out cells and resuspend in 0.5 ml 10 mM MgSO4.<br>
3.	Add together:<br>
100 I cells<br>
100 I diluted packaging mixture 100 I 10 mM MgSO4<br>
30   I TB<br>
Adsorb at room temperature for 30 minutes with no shaking.<br>
Add 1 ml TB and mix gently. Incubate 30 minutes at 37Â°C.<br>
Plate 200 I onto L-amp plates. Incubate at 37Â°C overnight.<br>
At least 400 cosmid clones were selected at random and screened for activity against western corn rootworm as described in Example 3. DMA from 5 active clones and 5 non-active clones were used in Southern hybridizations. Results demonstrated that hybridization using the above described oligonucleotide probe correlated with western corn rootworm activity (Table 18).<br>
Cosmid clones P3-12 and P5-4 have been deposited with the Agricultural Research Service Patent Culture Collection (NRRL) and given Accession Nos. NRRL B-21061 and NRRL B-21059 respectively.<br>
TABLE 18 Activity of AB78 cosmid clones against western corn rootworm. (Table Remove)<br>
EXAMPLE 10.       IDENTIFICATION OF A 6 KB REGION ACTIVE AGAINST WESTERN CORN ROOTWORM.<br>
DMA from P3-12 was partially digested with restriction enzyme Sau 3A, and ligated into the E. co//'vector pUC19 and transformed into E. coli. A DMA probe specific for the 80 kDa VIP1 A(a) protein was synthesized by PCR amplification of a portion of P3-12 DMA. Oligonucleotides MK113 and MK117, which hybridize to portions of VIP1 A(a), were synthesized using the partial amino acid sequence of the 80 kDa protein. Plasmid subclones were identified by colony hybridization to the PCR-generated probe, and tested for activity against western corn rootworm. One such clone, PL2, hybridized to the PCR-generated fragment, and was active against western corn rootworm in the assay previously described.<br>
A 6 kb Cla I restriction fragment from pL2 was cloned into the Sma I site of the E. coli-_Bacillus shuttle vector pHT 3101 (Lereclus, D. et a/., FEMS Microbiology Letters 60:211-218 (1989)) to yield pCIB6201. This construct confers anti-western corn rootworm activity upon both Bacillus and E.constrains, in either orientation. pCIB6022 contains this same 6 kb Cla I fragment in pBluescript SK(+) (Stratagene), produces equivalent VIP1 A{a) protein (by western blot), and is also active against western corn rootworm.<br>
The nucleotide sequence of pCIB6022 was determined by the dideoxy termination method of Sanger et a/., Proc. Natl. Acad. Sci. USA, 74:5463-5467 (1977), using PRISM Ready Reaction Dye Deoxy Terminator Cycle Sequencing Kits and PRISM SequenaseÂ® Terminator Double-Stranded DNA Sequencing Kit and analyzed on an ABI 373 automatic sequencer. The sequence is given in SEQ ID NO:1. The 6 kb fragment encodes both VIP1A(a) and VIP2A(a), as indicated by the open reading frames described in SEQ ID NO:1. The sequence encoding VIP2A(a) is further disclosed in SEQ ID NO:4. The relationship between VIP1 A(a) and VIP2A(a) within the 6 kb fragment found in pCIB6022 is depicted in Table 19. pCIB6022 was<br>
deposited with the Agricultural Research Service, Patent Culture Collection, (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA, and given the Accession No. NRRL B-21222.<br>
EXAMPLE 11.       FUNCTIONAL DISSECTION OF THE VIP1 Ate) DMA REGION.<br>
To confirm that the VIP1 A(a) open reading frame (ORF) is necessary for insecticidal activity a translational frameshift mutation was created in the gene. The restriction enzyme Bgl II recognizes a unique site located 857 bp into the coding region of VIP1 A(a). pCIB6201 was digested with Bgl II, and the single-stranded ends filled-in with DNA polymerase (Klenow fragment) and dNTPS. The plasmid was re-ligated and transformed into E. coll. The resulting plasmid, pCIB6203, contains a four nucleotide insertion in the coding region of VIP1 A(a). pCIB6203 does not confer WCRW insecticidal activity, confirming that VIP1 A(a) is an essential component of western corn rootworm activity.<br>
To further define the region necessary to encode VIP1 A(a), subclones of the VIP1 A(a) and VIP2A(a) (auxiliary protein) region were constructed and tested for their ability to complement the mutation in pCIB6203. pCIB6023 contains the 3.7kb Xba I-EcoRV fragment in pBluescript SK(+) (Stratagene). Western blot analysis indicates that pCIB6023 produces VIP1 A(a) protein of equal size and quantity as clones PL2 and pCIB6022. pCIB6023 contains the entire gene encoding the 80 kD protein. pCIB6023 was deposited with the Agricultural Research Service, Patent Culture Collection, (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA, and given the Accession No. NRRL B-21223N. pCIB6206 contains the 4.3 kb Xba l-CIa I fragment from pCIB6022 in pBluescript SK(+) (Stratagene). pCIB6206 was also deposited with the Agricultural Research Service, Patent Culture Collection, (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA, and given the Accession No. NRRL B-21321.<br>
pCIB6023, pCIB6206, and pCIB6203 do not produce detectable western corn rootworm activity when tested individually. However, a mixture of cells containing pCIB6203 (VIPIA(a)-mutated, plus VIP2A(a)) and cells containing pCIB6023 (only<br>
v.. .A(a)) shows high activity against western corn rootworm. Similarly, a mixture of cells containing pCIB6206 and cells containing pCIB6203 shows high activity against western corn rootworm.<br>
To further define the limits of VIP2A(a), we constructed pCIB6024, which contains the entirety of VIP2A(a), but lacks most of the VIP1 A(a) coding region. pCIB6024 was constructed by gel purifying the 2.2 kb Cla l-Sca I restriction fragment from pCIB6022, filling in the single-stranded ends with DNA polymerase (Klenow fragment) and dNTPs, and ligating this fragment into pBluescript SK(+) vector (Stratagene) digested with the enzyme Eco RV. Cells containing pCIB6024 exhibit no activity against western corn rootworm. However, a mixture of cells containing pCIB6024 and cells containing pCIB6023 shows high activity against western corn rootworm .(See Table 19).<br>
Thus, pCIB6023 and pCIB6206 must produce a functional VIP1 A(a) gene product, while pCIB6203 and pCIB6024 must produce a functional VIP2A(a) gene product. These results suggest a requirement for a gene product(s) from the VIP2A(a) region, in combination with VIP1A(a), to confer maximal western corn rootworm activity. (See Table 19.)<br>
(Table Remove) Boxed regions represent the extent of VIP1 A(a) and VIP2A(a). White box represents the portion of VIP1 encoding the 80 kDa peptide observed in Bacillus. Dark box represents the N-terminal 'propeptide' of VIP1 A(a) predicted by DMA sequence analysis. Stippled box represents the VIP2A(a) coding region. Large 'X1 represents the location of the frameshift mutation introduced into VIP1 A(a). Arrows represent constructs transcribed by the beta-galactosidase<br>
EXAMPLE 12. AB78 ANTIBODY PRODUCTION<br>
Antibody production was initiated in 2 Lewis rats to allow for both the possibility of moving to production of hybridoma cell lines and also to produce enough serum for limited screening of genomic DNA library. Another factor was the very limited amount of antigen available and the fact that it could only be produced to purity by PAGE and subsequent electrotransfer to nitrocellulose.<br>
Due to the limited availability of antigen on nitrocellulose, the nitrocellulose was emulsified in DMSO and injected into the hind footpads of the animals to elicit B-cell production in the popliteal lymph nodes just upstream. A strong reacting serum was produced as judged by western blot analysis with the first production bleed. Several subsequent injections and bleeds produced enough serum to accomplish all of the screening required.<br>
Hybridoma production with one of the rats was then initiated. The popliteal lymph node was excised, macerated, and the resulting cells fused with mouse myeloma P3x63Ag8.653. Subsequent cell screening was accomplished as described below. Four initial wells were selected which gave the highest emulsified antigen reaction to be moved to limited dilution cloning. An additional 10 wells were chosen for expansion and cryoperservation.<br>
Procedure to Emulsify AB78 on nitrocellulose in DMSO for ELISA screening:<br>
After electrotransfer of AB78 samples run on PAGE to nitrocellulose, the reversible strain Ponceau S is used to visualize all protein transferred. The band corresponding to AB78 toxin, previously identified and N-terminal sequenced, was identified and excised from nitrocellulose. Each band is approximately 1 mm x 5 mm in size to minimize the amount of nitrocellulose emulsified. A single band is placed in a microfuge tube with 250 nl of DMSO and macerated using a plastic pestle (Kontes, Vineland, NJ). To aid in emulsification, the DMSO mixture is heated for 2-3 minutes at 37 C-45 C. Some further maceration might be necessary following heating; however, all of the nitrocellulose should be emulsified. Once the AB78 sample is emulsified, it is placed on ice. In preparation for microtiter plate coating with the emulsified antigen, the sample must be diluted in borate buffered saline as follows: 1:5, 1:10, 1:15, 1:20, 1:30, 1:50, 1:100, and 0. The coating antigen must be prepared fresh immediately prior to use.<br>
ELISA protocol:<br>
Coat with AB78/DMSO in BBS. Incubate overnight at 4Â°C.<br>
Wash plate 3X with 1X ELISA wash buffer.<br>
Block (1% BSA &amp; 0.05% Tween 20 in PBS) for 30 minutes at Room<br>
Temperature.<br>
Wash plate 3X with 1X ELISA wash buffer.<br>
Add rat serum. Incubate 1.5 hours at 37Â°C.<br>
Wash plate 3X with 1X ELISA wash buffer.<br>
Add goat anti-rat at a concentration of 2 ng/ml in ELISA diluent. Incubate 1<br>
hr. at 37Â°C.<br>
Wash plate 3X with 1X ELISA wash buffer.<br>
Add rabbit anti-goat alkaline phosphatase at 2 ng/ml in ELISA diluent.<br>
Incubate 1 hr. at 37Â°C.<br>
Wash 3X with 1X ELISA wash buffer.<br>
Add Substrate. Incubate 30 minutes at room temperature.<br>
Stop with 3N NaOH after 30 minutes.<br>
Preparation of VIP2A(a) Antisera<br>
A partially purified AB78 culture supernatant was separated by discontinuous SDS PAGE (Novex) following manufacturer's instructions. Separated proteins were electrophoresed to nitrocellulose (S&amp;S #21640) as described by Towbin etal., (1979). The nitrocellulose was stained with Ponceau S and the VIP2A(a) band identified. The VIP2A(a) band was excised and emulsified in DMSO immediately prior to injection. A rabbit was initially immunized with emulsified VIP2A(a) mixed approximately 1:1 with Freund's Complete adjuvant by intramuscular injection at four different sites. Subsequent immunizations occurred at four week intervals and were identical to the first, except for the use of Freund1 Incomplete adjuvant. The first serum harvested following immunization reacted with VIP2A(a) protein. Western blot analysis of AB78 culture supernatant using this antisera identifies predominately full length VIP2A(a) protein.<br>
EXAMPLE 1 3.         ACTIVATION OF INSECTICIDAL ACTIVITY OF NON-ACTIVE BT STRAINS WITH AB78 VIP CLONES.<br>
Adding pCIB6203 together with a 24 h culture (early to mid-log phase) supernatant from Bt strain GC91 produces 100% mortality in Diabrotica virgifera virgifera. Neither pCIB6203 nor GC91 is active on Diabrotica virgifera virgifera by itself. Data are shown below:<br>
Test material	Percent Diabrotica mortality<br>
GC91	16<br>
pCIB6203 + GC91	100<br>
Control	0<br>
EXAMPLE 14. ISOLATION AND BIOLOGICAL ACTIVITY OF B. CEREUS AB81.<br>
A second B. cereus strain, designated AB81, was isolated from grain bin dust samples by standard methodologies. A subculture of AB81 was grown and prepared for bioassay as described in Example 2. Biological activity was evaluated as described in Example 3. The results are as follows:<br>
(Table Remove) <br>
EXAMPLE 15. ISOLATION AND BIOLOGICAL ACTIVITY OF B. THURINGIENSIS AB6.<br>
A B. thuringiensis strain, designated AB6, was isolated from grain bin dust samples by standard methods known in the art. A subculture of AB6 was grown and prepared for bioassay as described in Example 2. Half of the sample was autoclaved 15 minutes to test for the presence of p-exotoxin.<br>
Biological activity was evaluated as described in Example 3. The results are as follows:<br>
Insect species	Percent<br>
tested	Mortality<br>
Ostrinia nubilalis	0<br>
Agrotis ipsilon	100<br>
Agrotis ipsilon (autoclaved sample)	0<br>
Diabrotica virgifera virgifera	0<br>
The reduction of insecticidal acitivity of the culture supernatant to insignificant levels by autoclaving indicates that the active principle is not p-exotoxin.<br>
Strain AB6 has been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA, and given Accession No. NRRL B-21060.<br>
EXAMPLE 16. ISOLATION AND BIOLOGICAL CHARACTERIZATION OF B. THURINGIENSIS AB88.<br>
A Bt strain, designated AB88, was isolated from grain bin dust samples by standard methodologies. A subculture of AB88 was grown and prepared for bioassay as described in Example 2. Half of the sample was autoclaved 15 minutes to test for the presence of p-exotoxin. Biological activity was evaluated against a number of insect species as described in Example 3. The results are as follows: <br>
(Table Remove)  The reduction of insecticidal acitivity of the culture supernatant to insignificant levels by autoclaving indicates that the active principle is not p-exotoxin.<br>
Delta-endotoxin crystals were purified from strain AB88 by standard methodologies. No activity from pure crystals was observed when bioassayed against Agrotis ipsilon.<br>
EXAMPLE 17. PURIFICATION OF VIPS FROM STRAIN AB88:<br>
Bacterial liquid culture was grown overnight [for 12h] at 30Â°C in TB media. Cells were centrifuged at 5000 x g for 20 minutes and the supernatant retained. Proteins present in the supernatant were precipitated with ammonium sulfate (70% saturation),centrifuged [at 5000 x g for 15 minutes] and the pellet retained. The pellet was resuspended in the original volume of 20 mM Tris pH 7.5 and dialyzed overnight against the same buffer at 4Â°C. AB88 dialysate was more turbid than comparable material from AB78. The dialysate was titrated to pH 4.5 using 20 mM sodium citrate (pH 2.5) and, after 30 min incubation at room temperature, the solution was centrifuged at 3000 x g for 10 min. The protein pellet was redissolved in 20 mM Bis-Tris-Propane pH 9.0.<br>
AB88 proteins have been separated by several different methods following clarification including isoelectric focusing (Rotofor, BioRad, Hercules, CA), precipitation at pH 4.5, ion-exchange chromotography, size exclusion chromatography and ultrafiltration.<br>
Proteins were separated on a Poros HQ/N anion exchange column (PerSeptive Biosystems, Cambridge, MA) using a linear gradient from 0 to 500 mM NaCI in 20 mM Bis-Tris-Propane pH 9.0 at a flow rate of 4 ml/min. The insecticidal protein eluted at 250 mM NaCI.<br>
European corn borer (ECB)-active protein remained in the pellet obtained by pH 4.5 precipitation of dialysate. When preparative IEF was done on the dialysate using pH 3-10 ampholytes, ECB insecticidal activity was found in all fractions with pH of 7 or greater. SDS-PAGE analysis of these fractions showed protein bands of MW -60 kDa and ~80 kDa. The 60 kDa and 80 kDa bands were separated by anion exchange HPLC on a Poros-Q column (PerSeptive Biosystems, Cambridge, MA). N-terminal sequence was obtained from two fractions containing proteins of slightly differing MW, but both of approximately 60 kDa in size. The sequences obtained were similar to each other and to some 6-endotoxins.<br>
anion exchange fraction 23 (smaller):	xEPFVSAxxxQxxx (SEQIDNO:10)<br>
anion exchange fraction 28 (larger):	xEYENVEPFVSAx (SEQ ID NO:11)<br>
When the ECB-active pH 4.5 pellet was further separated by anion exchange on a Poros-Q column, activity was found only in fractions containing a major band of ~60 kDa.<br>
Black cutworm-active protein also remained in the pellet when AB88 dialysate was brought down to pH 4.5. In preparative IEF using pH 3-10 ampholytes, activity was not found in the ECB-active IEF fractions; instead, it was highest in a fraction of pH 4.5-5.0. Its major components have molecular weights of ~35 and ~80 kDa.<br>
The pH 4.5 pellet was separated by anion exchange HPLC to yield fractions containing only the 35 kDa material and fractions containing both 35 kDa and 80 kDa bands.<br>
EXAMPLE 18. CHARACTERIZATION OF AB88 VIP.<br>
Fractions containing the various lepidopteran active vegetative proteins were generated as described in Example 17. Fractions with insecticidal acitivity were separated in 8 to 16% SDS-polyacrylamide gels and transferred to PVDF membranes [LeGendre et al, (1989) in: A Practical Guide to Protein and Peptide Purification for Microsequencing, ed Matsudaria PT (Academic Press Inc, New Yorkl]. Biological analysis of fractions demonstrated that different VIPs were responsible for the different lepidopteran species activity.<br>
The Agrotis ipsilon activity is due to an 80 kDa and/or a 35 kDa protein, either delivered singly or in combination.   These proteins are not related to any 6-endotoxins from Bt as evidenced by the lack of sequence homology of known Bt 8-endotoxin sequences. The vip3A(a) insecticidal protein from strain AB88 is present mostly (at least 75% of the total) in supernatants of AB88 cultures.<br>
Also, these proteins are not found in the AB88 5-endotoxin crystal. N-terminal sequences of the major 8-endotoxin proteins were compared with the N-terminal sequences of the 80 kDa and 35 kDa VIP and revealed no sequence homology. The N-terminal sequence of the vip3A(a) insecticidal protein posses a number of positively charged residues (from Asn2 to Asn7) followed by a hydrophobic core region (from Thr8 to Ile34). Unlike most of the known secretion proteins, the vip3A(a) insecticidal protein from strain AB88 is not N-terminally processed during export.<br>
A summary of the results follows: (Table Remove)<br>
The Ostrinia nubilalis activity is due to a 60 kDa VIP and the Spodoptera frugiperda activity is due to a VIP of unknown size.<br>
Bacillus thuringiensis strain AB88 has been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA and given the Accession No. NRRL B-21225.<br>
EXAMPLE 18A.      ISOLATION AND BIOLOGICAL ACTIVITY OF B. THURINGIENSIS AB424<br>
A 6. thuringiensis strain, designated AB424, was isolated from a moss covered pine cone sample by standard methods known in the art. A subculture of AB424 was grown and prepared for bioassay as described in Example 2.<br>
Biological activity was evaluated as described in Example 3. The results are as follows:<br>
(Table Remove) Strain AB424 has been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA, and given Accession No. NRRL B-21439.<br>
EXAMPLE 18B.      CLONING OF THE VIP3A(a) and VIP3A(b) GENES WHICH ENCODE PROTONS ACTIVE AGAINST BLACK CUTWORM.<br>
Total DNA from isolates AB88 and AB424 was isolated [Ausubel et al (1988), in: Current Protocols in Molecular Biology (John Wiley &amp; Sons, NY)] and digested with the restriction enzymes Xba\ [library of 4.0 to 5.0 Kb size-fractionated Xbal fragments of B thuringiensis AB88 DNA] and EcoR\ [library of 4.5 to 6.0 Kb size-fractionated EcoRI fragments B thuringiensis AB424 DNA] respectively, ligated into pBluescript vector previously linearized with the same enzymes and dephosphorylated, and transformed into E. co//DH5cc strain. Recombinant clones were blotted onto nitrocellulose filters which were subsequently probed with a3Z P labeled 33-bases long oligonucleotide corresponding to the 11 -N terminal amino acids of the 80 kDa protein active against Agrotis ipsilon (black cutworm). Hybridization was carried out at 42Â°C in 2 x SSC/0.1% SDS (1 x SSC = 0.15 m NaCI/0.015 M sodium citrate, pH 7.4) for 5 min and twice at 50Â°C in 1 x SSC/0.1 SDS for 10 min. Four out of 400 recombinant clones were positive. Insect bioassays of the positive recombinants exhibited toxicity to black cutworm larvae comparable to that of AB88 or AB424 supernantants.<br>
Plasmid pCIB7104 contains a 4.5 Kb Xbalfragment of AB88 DNA. Subclones were constructed to define the coding region of the insecticidal protein.<br>
Eco//pCIB7105 was constructed by cloning the 3.5 Kb Xbal-Acclfragment of pCIB7104 into pBluescript.<br>
Plasmid pCIB7106 contained a 5.0 Kb EcoRI fragment of AB424 DNA. This fragment was further digested with Hindi to render a 2.8 kb EcoRI-Hincll insert (pCIB7107), which still encoded a functional insecticidal protein.<br>
The nucleotide sequence of pCIB7104, a positive recombinant clone from AB88, and of pCIB7107, a positive recombinant clone from AB424, was determined by the dideoxy termination method of Sanger et al., Proc. Natl. Acad. Sci. USA, 74:5463-5467 (1977), using PRISM Ready Reaction Dye Deoxy Terminator Cycle Sequencing Kits and PRISM SequenaseÂ® Terminator Double-Stranded DNA Sequencing Kit and analysed on an ABI 373 automatic sequencer.<br>
The clone pCIB7104 contains the VIP3A(a) gene whose coding region is disclosed in SEQ ID NO:28 and the encoded protein sequence is disclosed in SEQ ID NO:29. A synthetic version of the coding region designed to be highly expressed in maize is given in SEQ ID NO:30. Any number of synthetic genes can be designed based on the amino acid sequence given in SEQ ID NO:29.<br>
The clone pCIB7107 contains the VIP3A(b) gene whose coding region is disclosed in SEQ ID NO:31 and the encoded protein is disclosed in SEQ ID N0:32.   Both pCIB7104 and pCIB7107 have been deposited with the Agricultural Research Service Patent Culture Collection (NRRL) and given Accession Nos. NRRL B-21422 and B-21423, respectively.<br>
The VIP3A(a) gene contains an open reading frame (ORF) that extends form nucleotide 732 to 3105. This ORF encodes a peptide of 791 amino acids corresponding to a molecular mass of 88,500 daltons. A Shine-Dalgarno (SD) sequence is located 6 bases before the first methionine and its sequence identifies a strong SD for Bacillus.<br>
The VIP3A(b) gene is 98% identical to VIP3A(a).<br>
When blost of total DNA isolated from AB88 B thuringiensis cells were probed with a 33.base fragment that spans the N-terminal region of the VIPSA-insecticidal protein, single bands could be observed in different restriction digests. This result was<br>
confirmed by using larger probes spanning the coding region of the gene. A search of the GenBank data base revealed no homology to known proteins.<br>
EXAMPLE 18C.      EXPRESSION OF THE VIP3A INSECTICIDAL PROTEINS<br>
The time course for expression of the VIP3A(a) insecticidal protein was analyzed by western blot. Samples from Bacillus thuringiensis Ab88 clutures were taken throughout ist growth curve and sporulation. The VIP3A(a) insecticidal protein can be detected in the supernatants of AB88 cultures during logarithmic phase, as early as 15 h after initiating the culture. It reached its maximum level during early stages of stationary phase and remained at high levels during and after sporulation. Similar results were obtained when supernatants of AB424 Bacillus cereus cultures were used. The levels of VIP3A(a) insecticidal protein reflected the expression of the VIP3A(a) gene as determined by Northern blot. The initiation of the sporulation was determined by direct microscopic observations and by analyzing the presence of S-endotoxins in cell pellets. Cry-l type prtoeins could be detected late in the stationary phase , during and after sporulation.<br>
EXAMPLE 18D.      IDENTIFICATION OF NOVEL VIP3-LIKE GENES BY HYBRIDIZATION<br>
To identify Bacillus containing genes related to the VIP3A(a) from isolate AB88, a collection of Bacillus isolates was screened by hybridization. Cultures of 463 Bacillus strains were grown in microtiter wells until sporulation. A 96-pin colony stampel was used to transfer the cultures to 150 mm plates containing L-agar. Inoculated plates were kept at 30Â°C for 10 hours, then at 4Â°C overnight. Colonies were blotted onto nylon filters and probed with a 1.2Kb H/ndlll VIP3A(a) derived fragment. Hybridization was performed overnight at 62Â°C using hybridization conditions of Maniatis etal. Molecular Cloning: A Laboratory Manual (1982). Filters were washed with 2xSSC/0.1% SDS at 62Â°C and exposed to X-ray film.<br>
Of the 463 Bacillus strains screened, 60 contain VIP3-like genes that could detected by hybridization. Further characterization of some of them (AB6 and AB426)<br>
showed that their supernatants contain a BCW insecticidal protein similar to the Vip3 protein that are active against black cutworm.<br>
EXAMPLE 18E. CHARACTERIZATION OF A fl. thurinaiensis STRAIN M2194 CONTAINING A CRYPTIC VIP3-LIKE GENE<br>
A B. thuringiensis strain, designated M2194, was shown to contain VIP3-like gene(s) by colony hybridization as described in Example 18C. The M2194 VIP3 like gene is considered cryptic since no expression can be detected throughout the bacterial growth phases either by immunoblot analysis using polyclonal antibodies raised against the VIP3A(a) protein isolated from AB88 or by bioassay as described in Example 3.<br>
Antiserum against purified VIP3A(a) insecticidal protein was produced in rabbits. Nictrocellulose-bound protein {50 mj) was dissolved in DMSO and emulsified with Freund's complete adjuvant (Difco). Two rabbits were given subcutaneous injections each month for three month. They were bled 10 days after the second and third injection and the serum was recovered from the blood sample [Harlow et al (1988) in : Antibodies: A Laboratory Manual (Cold Spring Harbor Lab Press, Plainview, NY)].<br>
The M2194 VIP3-like gene was cloned into pKS by following the protocol described in Example 9, which created pCIB7108. Â£ co//containing pCIB7108 which comprises the M2194 VIP3 gene were active against black cutworm demonstrating that the gene encodes a functional protein with insecticidal activity. The plasm id pCIB7108 has been deposited with the Agricultural Research Service Patent Culture Collection (NRRL) and given Accession No. NRRL B-21438.<br>
EXAMPLE 18F.	INSECTICIDAL ACITIVITY OF VIP3A PROTEINS<br>
The activity spectrum of VIP3A insecticidal proteins was qualitatively determined in insect bioassays in which recombinant E coli carrying the VIP*A genes were fed to larvae. In these assays, cells carrying the VIP3A(a) and VIP3A(b) genes were insecticidal to Agrotis ipsilon, Spodoptera frugiperda, Spodoptera exigua, Heliothis virescens and Helicoverpa zea. Under the same expermimental conditions, bacterial extracts containing VIP3A proteins did not show any activity against Ostrinia nubilalis.<br>
Effect of VIP*A insecticidal proteins on Agrotis ipsilon larvae<br>
(Table Remove<br>
EXAMPLE 19. ISOLATION AND BIOLOGICAL ACTIVITY OF OTHER BACILLUS SP.<br>
Other Bacillus species have been isolated which produce proteins with insecticidal activity during vegetative growth. These strains were isolated from environmental samples by standard methodologies. Isolates were prepared for bioassay and assayed as described in Examples 2 and 3 respectively. Isolates which produced insecticidal proteins during vegetative growth with activity against Agrotis ipsilon in the bioassay are tabulated below. No correlation was observed between the presence of a 5-endotoxin crystal and vegetative insecticidal protein production. (Table Remove)  Isolates AB289, AB294 and AB359 have been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria II 61604, USA and given the Accession Numbers NRRL B-21227, NRRL B-21229, and NRRL B-21226 respectively.<br>
Bacillus isolates which produce insecticidal proteins during vegetative growth with activity against Diabrotica virgifera virgifera are tabulated below.<br>
(Table Remove) Isolates AB59 and AB256 have been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria Illinois 61604, USA, and given the Accession Numbers NRRL B-21228 and NRRL B-21230, respectively.<br>
EXAMPLE 20.         IDENTIFICATION OF NOVEL VIP1/VIP2 LIKE GENES BY HYBRIDIZATION<br>
To identify strains containing genes related to those found in the VIP1 A(a)/VIP2A(a) region of AB78, a collection of Bacillus strains was screened by hybridization. Independent cultures of 463 Bacillus strains were grown in wells of 96 well microtiter dishes (five plates total) until the cultures sporulated. Of the strains tested, 288 were categorized as Bacillus thuringiensis, and 175 were categorized as other Bacillus species based on the presence or absence of 5-endotoxin crystals. For each microtiter dish, a 96-pin colony stamper was used to transfer approximately 10 |xl of spore culture to two 150 mm plates containing L-agar. Inoculated plates were grown 4-8 hours at 30 Â°C, then chilled to 4 Â°C. Colonies were transferred to nylon filters, and the cells lysed by standard methods known in the art. The filters were hybridized to a DNA probe generated from DNA fragments containing both VIP1 A(a) and VIP2A(a) DNA sequences. Hybridization was performed overnight at 65 Â°C using the hybridization conditions of Church and Gilbert (Church, G.M., and W. Gilbert,<br>
PNAS, 81:1991-1995 (1984)). Filters were washed with 2x SSC containing 0.1% SDS at 65 Â°C and exposed to X-Ray film.<br>
Of the 463 Bacillus strains screened, 55 strains were identified that hybridized to the VIP1 A(a)/VIP2A(a) probe. DNA was isolated from 22 of these strains, and analyzed using a Southern blot with VIP1 A(a)/VIP2A(a) DNA as probes. These strains were grouped into 8 classes based on their Southern blot pattern. Each class differed in Southern blot pattern from AB78. One class had a pattern identical to that of the VIP1 A(a)/VIP2A(a) homologs from Bacillus thuringiensis var tenebrionis (see below). Each of the 22 strains was tested for activity against western corn rootworm (WCRW). Three strains, AB433, AB434, and AB435 were found to be active on WCRW. Western blot analysis using VIP2A(a) antisera revealed that strains AB6, AB433, AB434, AB435, AB444, and AB445 produce a protein(s) of equivalent size to VIP2A(a).<br>
Notable among the strains identified was Bacillus thuringiensis strain AB6, (NRRL B-21060) which produced a VIP active against black cutworm (Agrotis ipsilon) as described in Example 15. Western blot analysis with polyclonal antisera to VIP2A(a) and polyclonal antisera to VIP1 A(a) suggests that AB6 produces proteins similar to VIP2A(a) and VIP1A(a). Thus, AB6 may contain VIPs similar to VIP1A(a) and VIP2A(a), but with a different spectrum of insecticidal activity.<br>
EXAMPLE 21.        CLONING OF A VIP1 AteWIP2A(a) HOMOLOG FROM BACILLUS THURINGIENSIS MM. TENEBRIONIS.<br>
Several previously characterized Bacillus strains were tested for presence of DNA similar to VIP1 A(a)/VIP2A(a) by Southern blot analysis. DNA from Bacillus strains AB78, AB88, GC91, HD-1 and ATCC 10876 was analyzed for presence of VIP1 A(a)/VIP2A(a) like sequences. DNA from Bt strains GC91 and HD-1, and the Be strain ATCC 10876 did not hybridize to VIP2A(a)/VIP1A(a) DNA, indicating they lack DNA sequences similar to VIP1 A(a)/VIP2A(a) genes. Similarly, DNA from the insecticidal strain AB88 (Example 16) did not hybridize to VIP1A(a)/VIP2A(a) DNA region, suggesting that the VIP activity produced by this strain does not result from VIP1A(a)/VIP2A(a) homologs. In contrast, Bacillus thuringiensis var. tenebrionis (Btt)<br>
contained sequences that hybridized to the VIP1 A(a)/VIP2A(a) region. Further analysis confirmed that Btt contains VIP1A(a)/VIP2A(a) like sequences.<br>
To characterize the Btt homologs of VIP2A(a) and VIP1 A(a), the genes encoding these proteins were cloned. Southern blot analysis identified a 9.5 kb Eco Rl restriction fragment likely to contain the coding regions for the homologs. Genomic DNA was digested with Eco Rl, and DNA fragments of approximately 9.5 kb in length were gel-purified. This DNA was ligated into pBluescript SK(+) digested with Eco Rl, and transformed into E. coli to generate a plasmid library. Approximately 10,000 colonies were screened by colony hybridization for the presence of VIP2A(a) homologous sequences. Twenty eight positive colonies were identified. All twenty eight clones are identical, and contain VIP1A(a)/VIP2A(a) homologs. Clone pCIB7100 has been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria Illinois 61604, USA, and given the Accession Number B-21322. Several subclones were constructed from pCIB7100. A 3.8 kb Xba I fragment from pCIB7100 was cloned into pBluescript SK(+) to yield pCIB7101. A 1.8 kb Hind III fragment and a 1.4 kb Hind III fragment from pCIB7100 were cloned into pBluescript SK(+) to yield pCIB7102 and pCIB7103, respectively. Subclones pCIB7101, pCIB7102 and pCIB7103 have been deposited in the Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria Illinois 61604, USA, and given the Accession Numbers B-21323, B-21324 and B-21325 respectively.<br>
The DNA sequence of the region of pCIB7100 containing the VIP2A(a)/VIP1 A(a) homologs was determined by the dideoxy chain termination method (Sanger etal., 1977, Proc. Natl. Acad. Sci. USA 74:5463-5467). Reactions were performed using PRISM Ready Reaction Dye Deoxy Terminator Cycle Sequencing Kits and PRISM SequenaseÂ® Terminator Double-Stranded DNA Sequencing Kits, and analyzed on an ABI model 373 automated sequencer. Custom oligonucleotides were used as primers to determine the DNA sequence in certain regions. The DNA sequence of this region is shown in SEQ ID NO:19.<br>
The 4 kb region shown in SEQ ID NO:19 contains two open readings frames (ORFs), which encode proteins with a high degree of similarity to VIP1 A(a) and VIP2A(a) proteins from strain AB78. The amino acid sequence of the VIP2A(a)<br>
homolog, designated as VIP2A(b) using the standardized nomenclature, is found at SEQ ID NO:20 and the amino acid sequence of the VIP1 A(a) homolog, designated as VIP1 A(b) using the standardized nomenclature, is disclosed at SEQ ID NO:21. The VIP2A(b) protein exhibits 91% amino acid identity to VIP2A(a) from AB78. An alignment of the amino acid sequences of the two VIP2 proteins is provided in Table 20. The VIP1A(b) protein exhibits 77 % amino acid identity to VIP1 A(a) from AB78. An alignment of these two VIP1 proteins is provided in Table 21. The alignment shown in Table 21 discloses the similarity between VIP1A(b) and VIP1 A(a) from AB78. This alignment reveals that the amino terminal regions of the two VIP1 proteins share higher amino acid identity in the amino-terminal region than in the carboxy terminal region. In fact, the amino terminal two thirds (up to aa 618 of the VIP1 A(b) sequence shown in Table 21 ) of the two proteins exhibit 91% identity, while the carboxy-terminal third (from aa 619-833 of VIP1 A(b)) exhibit only 35% identity.<br>
Western blot analysis indicated that Bacillus thuringiensis var. tenebrionis (Btt) produces both VIP1 A(a) like and VIP2A(a) like proteins. However, these proteins do not appear to have activity against western corn rootworm. Bioassay for activity against western corn rootworm was performed using either a 24 h culture supernatant from Btt or E. coll clone pCIB7100 (which contains the entire region of the VIP1A(a)/VIP2A(a) homologs). No activity against western corn rootworm was detected in either case.<br>
Given the similarity between the VIP2 proteins from Btt and AB78, the ability of VIP2A(b) from Btt to substitute for VIP2A(a) from AB78 was tested. Cells containing pCIB6206 (which produces AB78 VIP1 A(a) but not VIP2A(a) protein) were mixed with Btt culture supernatant, and tested for activity against western corn rootworm. While neither Btt culture supernatant nor cells containing pCIB6206 had activity on WCRW, the mixture of Btt and pCIB6206 gave high activity against WCRW. Furthermore, additional bioassay showed that the Btt clone pCIB7100, which contains the Btt VIP1 A(b)/VIP2A(b) genes in E. coli, also confers activity against WCRW when mixed with pCIB6206. Thus, the VIP2A(b) protein produced by Btt is functionally equivalent to the VIP2A(a) protein produced by AB78.<br>
Thus, the ability to identify new strains with insecticidal activity by using VIP DNA as hybridization probes has been demonstrated. Furthermore, Bacillus strains that contain VIP1A(a)/VIP2A(a) like sequences, produce VIP1 A(a)/VIP2A(a) like protein,<br>
yet demonstrate toxicity toward different insect pests. Similar methods can identify many more members of the VIP1/VIP2 family. Furthermore, use of similar methods can identify homologs of other varieties of VIPs (for example, the VIPs from AB88).<br>
(Table Remove) <br>
EXAMPLE 22.         FUSION OF VIP PROTEINS TO MAKE A SINGLE POLYPEPTIDE<br>
VIP proteins may occur in nature as single polypeptides, or as two or more interacting polypeptides. When an active VIP is comprised of two or more interacting protein chains, these protein chains can be produced as a single polypeptide chain from a gene resulting from the fusion of the two (or more) VIP coding regions. The genes encoding the two chains are fused by merging the coding regions of the genes to produce a single open reading frame encoding both VIP polypeptides. The composite polypeptides can be fused to produce the smaller polypeptide as the NH2 terminus of the fusion protein, or they can be fused to produce the larger of thepolypeptides as the NHZ terminus of the fusion protein. A linker region can optionally be used between the two polypeptide domains. Such linkers are known in the art. This linker can optionally be designed to contain protease cleavage sites such that once the single fused polypeptide is ingested by the target insect it is cleaved in the linker region to liberate the two polypeptide components of the active VIP molecule.<br>
VIP1 A(a) and VIP2A(a) from B. cereus strain AB78 are fused to make a single polypeptide by fusing their coding regions. The resulting DNA comprises a sequence given in SEQ ID NO:22 with the encoded protein given in SEQ ID NO:23. In like manner, other fusion proteins may be produced.<br>
The fusion of the genes encoding VIP1 A(a) and VIP2A(a) is accomplished using standard techniques of molecular biology. The nucleotides deleted between the VIP1 A(a) and VIP2A(a) coding regions are deleted using known mutagenesis techniques or, alternatively, the coding regions are fused using PCR techniques.<br>
The fused VIP polypeptides can be expressed in other organisms using a synthetic gene, or partially synthetic gene, optimized for expression in the alternative host. For instance, to express the fused VIP polypeptide from above in maize, one makes a synthetic gene using the maize preferred codons for each amino acid, see for example EP-A 0618976, herein incorporated by reference. Synthetic DNA sequences created according to these methods are disclosed in SEQ ID N0:17 (maize optimized version of the 100 kDa VIP1A(a) coding sequence), SEQ ID NO:18 (maize optimized version of the 80 kDa VIP1 A(a) coding sequence) and SEQ ID NO:24 (maize optimized version of the VIP2A(a) coding sequence).<br>
Synthetic VIP1 and VIP2 genes optimized for expression in maize can be fused using PCR techniques, or the synthetic genes can be designed to be fused at a common restriction site. Alternatively, the synthetic fusion gene can be designed to encode a single polypeptide comprised of both VIP1 and VIP2 domains.<br>
Addition of a peptide linker between the VIP1 and VIP2 domains of the fusion protein can be accomplished by PCR mutagenesis, use of a synthetic DNA linker encoding the linker peptide, or other methods known in the art.<br>
The fused VIP polypeptides can be comprised of one or more binding domains. If more than one binding domain is used in the fusion, multiple target pests are controlled using such a fusion. The other binding domains can be obtained by using all or part of other VIPs; Bacillus thuringiensis endotoxins, or parts thereof; or other<br>
proteins capable of binding to the target pest or appropriate biding domains derived from such binding proteins.<br>
One example of a fusion construction comprising a maize optimized DNA sequence encoding a single polypeptide chain fusion having VIP2A(a) at the N-terminal end and VIP1A(a) at the C-terminal end is provided by pCIB5531. A DNA sequence encoding a linker with the peptide sequence PSTPPTPSPSTPPTPS (SEQ ID NO:47) has been inserted between the two coding regions. The sequence encoding this linker and relevant cloning sites is 5'- CCC GGG CCT TCT ACT CCC CCA ACT CCC TCT CCT AGC ACG CCT CCG ACA CCT AGC GAT ATC GGA TC C -3' (SEQ ID NO:48). Oligonucleotides were synthesized to represent both the upper and lower strands and cloned into a pUC vector following hybridization and phosphorylation using standard procedures. The stop codon in VIP2A(a) was removed using PCR and replaced by the Bglll restriction site with a Smal site. A translation fusion was made by ligating the Bam HI / Pstl fragment of the VIP2A(a) gene from pCIB5522 (see Example 24), a PCR fragment containing the Pstl-end fragment of the VIP2A(a) gene (identical to that used to construct pCIB5522), a synthetic linker having ends that would ligate with a blunt site at the 5' end and with BamHI at the 3' end and the modified synthetic VIP1 A(a) gene from pCIB5526 described below (See SEQ ID NO:35). The fusion was obtained by a four way ligation that resulted in a plasmid containing the VIP2A(a) gene without a translation stop codon, with a linker and the VIP1 A(a) coding region without the Bacillus secretion signal. The DNA sequence for this construction is disclosed in SEQ ID N0:49, which encodes the fusion protein disclosed in SEQ ID NO:50. A single polypeptide fusion where VIP1 A(a) is at the N-terminal end and VIP2A(a) is at the C-terminal end can be made in a similar fashion.   Furthermore, either one or both genes can be linked in a translation fusion with or without a linker at either the 5' or the 3' end to other molecules like toxin encoding genes or reporter genes.<br>
EXAMPLE 23. TARGETING OF VIP2 TO PLANT ORGANELLES<br>
Various mechanisms for targeting gene products are known to exist in plants and the sequences controlling the functioning of these mechanisms have been characterized in some detail. For example, the targeting of gene products to thechloroplast is controlled by a signal sequence found at the amino-terminal end of various proteins. This signal is cleaved during chloroplast import, yielding the mature protein (e.g. Comai etal. J. Biol. Chem. 263:15104-15109 (1988)). These signal sequences can be fused to heterologous gene products such as VIP2 to effect the import of those products into the chloroplast (van den Broeck etal. Nature 313:358-363 (1985)). DNA encoding for appropriate signal sequences can be isolated from the 5' end of the cDNAs encoding the RUBISCO protein, the CAB protein, the EPSP synthase enzyme, the GS2 protein and many other proteins which are known to be chloroplast localized.<br>
Other gene products are localized to other organelles such as the mitochondrion and the peroxisome (e.g. Linger etal. Plant Molec. Biol. 13:411-418 (1989)). The cDNAs encoding these products can also be manipulated to effect the targeting of heterologous gene products such as VIP2 to these organelles. Examples of such sequences are the nuclear-encoded ATPases and specific aspartate amino transferase isoforms for mitochondria. Similarly, targeting to cellular protein bodies has been described by Rogers etal. (Proc. Natl. Acad. Sci. USA 82: 6512-6516 (1985)).<br>
By the fusion of the appropriate targeting sequences described above to coding sequences of interest such as VIP2 it is possible to direct the transgene product to any organelle or cell compartment. For chloroplast targeting, for example, the chloroplast signal sequence from the RUBISCO gene, the CAB gene, the EPSP synthase gene, or the GS2 gene is fused in frame to the amino-terminal ATG of the transgene. The signal sequence selected should include the known cleavage site and the fusion constructed should take into account any amino acids after the cleavage site which are required for cleavage. In some cases this requirement may be fulfilled by the addition of a small number of amino acids between the cleavage site and the start codon ATG, or alternatively replacement of some amino acids within the coding sequence. Fusions constructed for chloroplast import can be tested for efficacy of chloroplast uptake by in vitro translation of in vitro transcribed constructions followed by in vitro chloroplast uptake using techniques described by (Bartlett etal. In: Edelmann etal. (Eds.) Methods in Chloroplast Molecular Biology, Elsevier. pp 1081-1091 (1982); Wasmann etal. Mol. Gen. Genet. 205: 446-453 (1986)). Theseconstruction techniques are well known in the art and are equally applicable to mitochondria and peroxisomes.<br>
The above described mechanisms for cellular targeting can be utilized not only in conjunction with their cognate promoters, but also in conjunction with heterologous promoters so as to effect a specific cell targeting goal under the transcriptional regulation of a promoter which has an expression pattern different to that of the promoter from which the targeting signal derives.<br>
A DNA sequence encoding a secretion signal is present in the native Bacillus VIP2 gene. This signal is not present in the mature protein which has the N-terminal sequence of LKITDKVEDF (amino acid residues 57 to 66 of SEQ ID NO:2). It is possible to engineer VIP2 to be secreted out of the plant cell or to be targeted to subcellular organelles such as the endoplasmic reticulum, vacuole, mitochondria or plastids including chloroplasts. Hybrid proteins made by fusion of a secretion signal peptide to a marker gene have been successfully targeted into the secretion pathway. (Itirriaga G. era/., The Plant Cell. 1:381-390 (1989), Denecke etal., The Plant Cell. 2:51-59 (1990). Amino-terminal sequences have been identified that are responsible for targeting to the ER, the apoplast, and extracellular secretion from aleurone cells (Koehler &amp; Ho, Plant Cell 2: 769-783 (1990)).<br>
The presence of additional signals are required for the protein to be retained in the endoplasmic reticulum or the vacuole. The peptide sequence KDEL/HDEL at the carboxy-terminal of a protein is required for its retention in the endoplasmic reticulum (reviewed by Pelham, Annual Review Cell BioL 5:1-23 (1989). The signals for retention of proteins in the vacuole have also been characterized. Vacuolar targeting signals may be present either at the amino-terminal portion, (Holwerda et a/., The Plant Cell. 4:307-318 (1992), Nakamura etal., Plant Phvsiol.. 101:1-5 (1993)), carboxy- terminal portion, or in the internal sequence of the targeted protein. (Tague etal., The Plant Cell. 4:307-318 (1992), Saalbach etal.. The Plant Cell. 3:695-708 (1991)). Additionally, amino-terminal sequences in conjunction with carboxy-terminal sequences are responsible for vacuolar targeting of gene products (Shinshi et at. Plant Molec. Biol. 14:357-368 (1990)). Similarly, proteins may be targeted to the mitochondria or plastids using specific carboxy terminal signal peptide fusions (Heijne etal., Eur. J. Biochem.. 180:535-545 (1989), Archer and Keegstra, Plant Molecular Biology. 23:1105-1115(1993)).<br>
In order to target VIP2, either for secretion or to the various subcellular organelles, a maize optimized DNA sequence encoding a known signal peptide(s) may be designed to be at the 5' or the 3' end of the gene as required. To secrete VIP2 out of the cell, a DNA sequence encoding the eukaryotic secretion signal peptide MGWSWIFLFLLSGAAGVHCL (SEQ ID NO:25) from PCT application No. IB95/00497 or any other described in the literature (Itirriaga et a/., The Plant Cell. 1:381-390 (1989), Denecke, et a/., The Plant Cell. 2:51-59 (1990)) may be added to the 5' end of either the complete VIP2 gene sequence or to the sequence truncated to encode the mature protein or the gene truncated to nucleotide 286 or encoding a protein to start at amino acid residue 94 (methionine). To target VIP2 to be retained in the endoplasmic reticulum, a DNA sequence encoding the ER signal peptide KDEL /HDEL, in addition to the secretion signal, can be added to the 3' end of the gene. For vacuolar targeting a DNA sequence encoding the signal peptide SSSSFADSNPIRVTDRAAST (SEQ ID N0:3; Holwerda etal., The Plant Cell. 4:307-318 (1992)) can be designed to be adjacent to the secretion signal or a sequence encoding a carboxyl signal peptide as described by Dombrowski et ai, The Plant Cell. 5:587-596 (1993) or a functional variation may be inserted at the 3' end of the gene. Similarly, VIP2 can be designed to be targeted to either the mitochondria or the plastids, including the chloroplasts, by inserting sequences in the VIP2 sequence described that would encode the required targeting signals. The bacterial secretion signal present in VIP2 may be retained or removed from the final construction.<br>
One example of a construction which incorporates a eukaryotic secretion signal fused to a coding sequence for a VIP is provided by pCIB5528. Oligonucleotides corresponding to both the upper and lower strand of sequences encoding the secretion signal peptide of SEQ ID N0:25 was synthesized and has the sequence 5'-GGATCCACC ATG GGC TGG AGC TGG ATC TTC CTG TTC CTG CTG AGC GGC GCC GCG GGC GTG CAC TGC CTGCAG-31 (SEQ ID NO:41). When hybridized, the 5' end of the secretion signal resembled "sticky-ends" corresponding to restriction sites BamHI and Pstl. The oligonucleotide was hybridized and phosphorylated and ligated into pCIB5527 (construction described in Example 23A) which had been digested with BamHI/ Pstl using standard procedures. The resulting maize optimized coding sequence is disclosed in SEQ ID NO:42 which encodes the protein disclosed<br>
in SEQ ID NO:43. This encoded protein comprises the eukaryotic secretion signal in place of the Bacillus secretion signal.<br>
One example of a construction which incorporates a vacuolar targetting signal fused to a coding sequence for a VIP is provided by pCIB5533. Oligonucleotides corresponding to both the upper and lower strand of sequences encoding the vacuolar targetting peptide of SEQ ID N0:3 was synthesized and has the sequence 5'-CCG CGG GCG TGC ACT GCC TCA GCA GCA GCA GCT TCG CCG ACA GCA ACC CCA TCC GCG TGA CCG ACC GCG CCG CCA GCA CCCTGCAG-31 (SEQ ID NO:44). When hybridized, the 5' end of the vacuolar targetting signal resembled "sticky-ends" corresponding to restriction sites Sacll and Pstl. The oligonucleotide was hybridized and phosphorylated and ligated into pCIB5528 (construction described above) which had been digested with Sacll / Pstl using standard procedures. The resulting maize optimized coding sequence is disclosed in SEQ ID NO:45 which encodes the protein disclosed in SEQ ID NO:46. This encoded protein comprises the vacuolar targetting peptide in addition to the eukaryotic secretion signal.<br>
The VIP1 gene can also be designed to be secreted or targeted to subcellular organelles by similar procedures.<br>
EXAMPLE 23A.      REMOVAL OF BACILLUS SECRETION SIGNAL FROM VIP1A(a)ANDVIP2A(a)<br>
VIP1 A(a) and VIP2A(a) are secreted during the growth of strain AB78. The nature of peptide sequences that act as secretion signals has been described in the literature (Simonen and Palva, Microbiological reviews, pg. 109-137 (1993)). Following the information in the above publication, the putative secretion signal was identified in both genes. In VIP1 A(a) this signal is composed of amino acids 1-33 (See SEQ ID N0:5). Processing of the secretion signal probably occurs after the serine at amino acid 33. The secretion signal in VIP2A(a) was identified as amino acids 1-49 (See SEQ ID NO:2). N-terminal peptide analysis of the secreted mature VIP2A(a) protein revealed the N-terminal sequence LKITDKVEDFKEDK. This sequence is found beginning at amino acid 57 in SEQ ID NO:2. The genes encoding these proteins have been modified by removal of the Bacillus secretion signals.<br>
A maize optimized VIP1A(a) coding region was constructed which had the sequences encoding the first 33 amino acids, i.e., the secretion signal, removed from its 5' end. This modification was obtained by PCR using an forward primer that<br>
contained the sequence 5'-GGA TCC ACC ATG AAG ACC AAC GAG ATC AGC-3' (SEQ ID NO:33), which hybridizes with the maize optimized gene (SEQ ID NO:26) at nucleotide position 100, and added a BamHI restriction site and a eukaryotic translation start site consensus including a start codon. The reverse primer that contained the sequence 5'-AAG CTT GAG CTC CTT G-3' (SEQ ID N0:34) hybridizes on the complementary strand at nucelotide position 507. A 527 bp amplification product was obtained containing the restriction sites BamHI at the 5' end and Hindlll site at the 3' end. The amplification product was cloned into a T- vector (described in Example 24, below) and sequenced to ensure the correct DNA sequence. The BamHI / Hindlll fragment was then obtained by restriction digest and used to replace the BamHI/Hindlll fragment of the maize optimized VIP1A(a) gene cloned in the root-preferred promoter cassette. The construct obtained was designated pCIB5526. The maize optimized coding region for VIP1A(a) with the Bacillus secretion signal removed is disclosed as SEQ ID NO:35 and the encoded protein is disclosed as SEQ ID NO:36.<br>
The gene encoding the processed form of VIP2A(a), i.e., a coding region with the secretion signal removed, was constructed by a procedure similar to that described for that used to construct the processed form of VIP1 A(a), above. The modification was obtained by PCR using the forward primer 5'-GGA TCC ACC ATG CTG GAG AAC CTG AAG ATC AC -3' (SEQ ID NO:37). This primer hybridizes at nucleotide position 150 of the maize optimized VIP2A(a) gene (SEQ ID NO:27). A silent mutation has been inserted at nucleotide position 15 of this primer to obtain a Pstl restriction site. The reverse primer has the sequence 5'-AAG CTT CCA CTC CTT CTC-31 (SEQ ID N0:38). A 259 bp product was obtained with Hindlll restriction site at the 3' end. The amplification product was cloned into a T- vector, sequenced and ligated to a BamHI /Hindlll digested root-preferred promoter cassette containing the maize optimized VIP2A(a). The construct obtained was designated pCIB5527. The maize optimized coding region for VIP2A(a) with the Bacillus secretion signal removed is disclosed as SEQ ID NO:39 and the encoded protein is disclosed as SEQ ID NO:40.<br>
EXAMPLE 24. CONSTRUCTION AND CLONING OF THE VIP1 Ate) AND VIP2A(a) MAIZE OPTIMIZED GENES<br>
Design: The maize optimized genes were designed by reverse translation of the native VIP1 A(a) and VIP2A(a) protein sequences using codons that are used most often in maize (Murray et a/., Nucleic Acid Research. 17:477-498 (1989)). To facilitate cloning, the DNA sequence was further modified to incorporate unique restriction sites at intervals of every 200-360 nucleotides. VIP1A(a) was designed to be cloned in 11 such fragments and VIP2A(a) was cloned in 5 fragments. Following cloning of the individual fragments, adjacent fragments were joined using the restriction sites common to both fragments, to obtain the complete gene. To clone each fragment, oligonucleotides (50-85 nucleotides) were designed to represent both the upper and the lower strand of the DNA. The upper oligo of the first oligo pair was designed to have a 15 bp single stranded region at the 3' end which was homologous to a similar single stranded region of the lower strand of the next oligo pair to direct the orientation and sequence of the various oligo pairs within a given fragment. The oligos are also designed such that when the all the oligos representing a fragment are hybridized, the ends have single stranded regions corresponding to the particular restriction site to be formed. The structure of each oligomer was examined for stable secondary structures such as hairpin loops using the OLIGO program from NBI Inc. Whenever neccesary, nucleotides were changed to decrease the stability of the secondary structure without changing the amino acid sequence of the protein. A plant ribosomal binding site consensus sequence, TAAACAATG (Joshi et a/., Nucleic Acid Res.. 15:6643-6653 (1987)) or eukaryotic ribosomal binding site concensus sequence CCACCATG (Kozak, Nucleic Acid Research. 12:857-872 (1984)) was inserted at the translational start codon of the gene.<br>
Cloning: Oligos were synthesized by IDT Inc., and were supplied as lyophilized powders. They were resuspended at a concentration of 200 m.m. To 30 pJ of each oligo formamide was added a final concentration of 25-50% and the sample was boiled for two minutes before separation on a premade 10% polyacryamide / urea gel obtained from Novex. After electrophoresis, the oligo was detected by UV shadowing by placing the gel on a TLC plate containing a fluorescent indicator and exposing it to UV light. The region containing DNA of the correct size was excised and extracted<br>
from the polyacryamide by an overnight incubation of the minced gel fragment in a buffer containing 0.4 M LiCI, 0.1 mM EDTA. The DNA was separated from the gel residue by centrifugation through a Millipore UFMC filter. The extracted DNA was ethanol precipitated by the addition of 2 volumes of absolute alcohol. After centrifugation, the precipitate was resuspended in dH20 at a concentration of 2.5 m.m. Fragments were cloned either by hybridization of the oligos and ligation with the appropriate vector or by amplification of the hybridized fragment using a equimolar mixture of all the oligos for a particular fragment as a template and end-specific PCR primers.<br>
Cloning bv hybridization and liaation: Homologous double stranded oligo pairs were obtained by mixing 5 pJ of the upper and of the lower oligo for each oligo pair with buffer containing 1X polynucleotide kinase (PNK) buffer (70 mM Tris-HCI (pH 7.6), 10 mM MgCI2,5 mM dithiothreitol (DTT)), 50 mM KCI, and 5 % formamide in a final volume of 50 jj.l. The oligos were boiled for 10 minutes and slow cooled to 37Â° C or room temperature. 10 \i\ was removed for analysis on a 4% agarose in a TAE buffer system (MetaphoreÂ®; FMC). Each hybridized oligo pair was kinased by the addition of ATP at a final concentration of 1 mM, BSA at a final concentration of 100 jxg per ml and 200 units of polynucleotide kinase and 1 u.l of 10X PNK buffer in a volume of 10 jil. Following hybridization and phosphorylation, the reaction was incubated at 37Â° C for 2 hours to overnight. 10 |oJ of each of the oligo pairs for a particular fragment, were mixed in a final volume of 50 jil. The oligo pairs were hybridized by heating at 80Â° C for 10 minutes and slow cooling to 37Â° C. 2 |il of oligos was mixed with about 100 ng of an appropriate vector and ligated using a buffer containing 50 mM Tris-HCI (pH 7.8), 10 mM MgCI2, 10 mM DTT, 1 mM ATP. The reaction was incubated at room temp, for 2 hours to overnight and transformed into DH5a strain of E.coli, plated on L- plates containing ampicillin at a concentration of 100 |o.g/ml using standard procedures. Positive clones were further characterized and confirmed by PCR miniscreen described in detail in EP-A 0618976 using the universal primers "Reverse" and M13 "-20" as primers. Positive clones were identified by digestion of DNA with appropriate enzymes followed by sequencing. Recombinants that had the expected DNA sequence were then selected for further work.<br>
PCR Amplification and cloning into T- vector:<br>
PCR amplification was carried out by using a mixture of all the oligomers that represented the upper and the lower strand of a particular fragment (final concentration 5 mM each) as template, specific end primers for the particular fragment (final concentration 2 m.m) 200 ^M of each dATP, dTTP, dCTP and dGTP, 10 mM Tris-HCI (pH 8.3), 50 mM KCI, 1.5 mM MgCI2,0.01% gelatin and 5 units of Tap polymerase in a final reaction volume of 50 nl. The amplification reaction was carried out in a Perkin Elmer thermocycler 9600 by incubation at 95Â° C for 1 min (1 cycle ), followed by 20 cycles of 95 Â°C for 45 sec., 50 Â°C for 45 sec., 72 Â°C for 30 sec. Finally the reaction was incubated for 5 min at 72Â°C before analyzing the product. 10 nl of the reaction was analyzed on a 2.5% Nusieve (FMC) agarose gel in a TAE buffer system. The correct size fragment was gel purified and used for cloning into a PCR cloning vector or T-vector. T-vector construction was as described by Marchuk et a/., Nucleic Acid Research. 19:1154 (1991).   pBluescriptsk+ (StratageneÂ®, Ca.) was used as the parent vector. Transformation and identification of the correct clone was carried out as described above.<br>
Fragments 1, 3, 4, 5, 6, 8, and 9 of VIP1 A(a) and fragments 2 and 4 of VIP2A(a) were obtained by cloning of PCR amplification products; whereas, fragments 2, 7,10 and 11 of VIP1 A(a) and fragments 1, 3, and 5 of VIP2A(a) were obtained by hybridization/ ligation.<br>
Once fragments with the desired sequence were obtained, the complete gene was assembled by cloning together adjacent fragments. The complete gene was resequenced and tested for activity against WCRW before moving it into plant expression vectors containing the root preferred promoter (disclosed in U.S. patent application serial no. 08/017,209, herein incorporated by reference) and the rice actin promoter.<br>
One such plant expression vector is pCIB5521. The maize optimized VIP1 A(a) coding region (SEQ ID NO:26) was cloned in a plant expression vector containing the root preferred promoter at the 5' of the gene with the PEP Carboxylase intron #9 followed by the 35S terminator at the 3' end. The plasmid also contains sequences for ampicillin resistance from the plasmid pUC19. Another plant expression vector is pCIB5522, which contains the maize optimized VIP2A(a) coding region (SEQ ID<br>
N0:27) fused to the root preferred promoter at the 5' of the gene with the PEP Carboxylase intron #9 followed by the 35S terminator at the 3' end.<br>
EXAMPLE 25. NAP AFFINITY CHROMATOGRAPHY<br>
A purification strategy was used based on the affinity of VIP2 for the substrate NAD. The supernatant from the pH 3.5 sodium citrate buffer treatment described in Example 4 was dialyzed in 20 mM TRIS pH 7.5 overnight. The neutralized supernatant was added to an equal volume of washed NAD agarose and incubated with gentle rocking at 4Â° C overnight. The resin and protein solution were added to a 10 ml disposable polypropylene column and the protein solution allowed to flow out. The column was washed with 5 column volumes of 20 mM TRIS pH 7.5 then washed with 2-5 column volumes of 20 mM TRIS pH 7.5, 100 mM NaCI, followed by 2-5 column volumes of 20 mM TRIS 7.5. The VIP proteins were eluted in 20 mM TRIS pH 7.5 supplemented with 5 mM NAD. Approximately 3 column volumes of the effluent were collected and concentrated in a Centricon -10. Yield is typically about 7-15 ng of protein per ml of resin.<br>
When the purified proteins were analyzed by SDS-PAGE followed by silver staining, two polypeptides were visible, one with Mr of approximately 80,000 and one with Mr of approximately 45,000. N-terminal sequencing revealed that the Mr 80,000 protein corresponded to a proteolytically processed form of VIP1 A(A) and the Mr 45,000 form corresponded to a proteolytically processed form of VIP2A(a). The co-purification of VIP1 A(a) with VIP2A(a) indicates that the two proteins probably form a complex and have protein-protein interacting regions. VIP1A(a) and VIP2A(a) proteins purified in this manner were biologically active against western corn rootworm.<br>
EXAMPLE 26.         EXPRESSION OF MAIZE OPTIMIZED VIP1 Ate) AND VIP2A(a)<br>
E. coli strains containing different plasmids comprising VIP genes were assayed for expression of VIPs. E. constrains harboring the individual plasmids were grown overnight in L-broth and expressed protein was extracted from the culture as described in Example 3, above. Protein expression was assayed by Western Blot analysis using antibodies developed using standard methods known in the art, similar<br>
to those described in Example 12, above. Also, insecticidal activity of the expressed proteins were tested against Western corn rootworm according to the method in Example 3, above. The results of the E. co//expression assays are described below.<br>
(Table Remove) <br>
The DNA from these plasmids was used to transiently express the VIPs in a maize protoplast expression system.   Protoplasts were isolated from maize 2717 Line 6 suspension cultures by digestion of the cell walls using Cellulase RS and Macerase R10 in appropriate buffer. Protoplasts were recovered by sieving and centrifugation. Protoplasts were transformed by a standard direct gene transfer method using approximately 75 g plasmid DNA and PEG-40. Treated protoplasts were incubated overnight in the dark at room temperature. Analysis of VIP expression was<br>
accomplished on protoplast explants by Western blot analysis and insecticidal activity against Western corn rootworm as described above for the expression in E. coll. The results of the maize protoplast expression assays are described below.<br>
Expression of VIPs in Plant Protoplasts(Table Remove) <br>
 = extract of E. constrain harboring indicated plasmid<br>
The expression data obtained with both E. co//and maize protoplasts show that the maize optimized VIP1A(a) and VIP2A(a) genes make the same protein as the native VIP1 A(a) and VIP2A(a) genes, respectively, and that the proteins encoded by the maize optimized genes are functionally equivalent to the proteins encoded by the native genes.<br>
All publications and patent applications mentioned in this specification are indicative of the level of skill of those skilled in the art to which this invention pertains. All publications and patent applications are herein incorporated by reference to the same extent as if each individual publication or patent application was specifically and individually indicated to be incorporated by reference.<br>
The following deposits have been made at Agricultural Research Service, Patent Culture Collection (NRRL), Northern Regional Research Center, 1815 North University Street, Peoria, Illinois 61604, USA:<br>
(Table Remove) Although the foregoing invention has been described in some detail by way of illustration and example for purposes of clarity of understanding, it will be obvious that certain changes and modifications may be practiced within the scope of the appended claims.<br>
SEQUENCE LISTING<br>
(1)	GENERAL INFORMATION:<br>
NAME: CIBA-GEIGY AG<br>
STREET: Klybeckstr. 141<br>
CITY: Basel<br>
(E)	COUNTRY: Switzerland<br>
(F)	POSTAL CODE (ZIP): 4002<br>
(G)	TELEPHONE: +41 61 69 11 11<br>
(H)	TELEFAX: + 41 61 696 79 76<br>
(I)	TELEX: 962 991<br>
(ii) TITLE OF INVENTION: Novel Pesticidal Proteins and Strains (iii) NUMBER OF SEQUENCES: 50<br>
(iv) COMPUTER READABLE FORM:<br>
MEDIUM TYPE: Floppy disk<br>
COMPUTER: IBM PC conpatible<br>
OPERATING SYSTEM: PC-DOS/MS-DOS<br>
SOFTWARE: Patentln Release #1.0, Version #1.30B<br>
(2)	INFORMATION FOR SEQ ID N0:l:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 6049 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(vi) ORIGINAL SOURCE:<br>
ORGANISM: Bacillus cereus<br>
STRAIN: AB78<br>
INDIVIDUAL ISOLATE: NRRL B-21058<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1082..2467<br>
(D)	OTHER INFORMATION: /product^ "VIP2A(a)"<br>
(ix) FEATURE:<br>
NAME/KEY: misc__feature<br>
LOCATION: 2475..5126<br>
(D) OTHER INFORMATION: /note= "Coding sequence for the 100 kd VIPlA(a) protein. This coding sequence is repeated in SEQ ID NO:4 and translated separately."<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID N0:l:<br>
ATCGATACAA TGTTGTTTTA CTTAGACCGG TAGTCTCTGT AATTTGTTTA ATGCTATATT	60<br>
CTTTACTTTG ATACATTTTA ATAGCCATTT CAACCTTATC AGTATGTTTT TGTGGTCTTC	120<br>
CTCCTTTTTT TCCACGAGCT CTAGCTGCGT TTAATCCTGT TTTGGTACGT TCGCTAATAA	180<br>
TATCTCTTTC TAATTCTGCA ATACTTGCCA TCATTCGAAA GAAGAATTTC CCCATAGCAT	240<br>
TAGAGGTATC AATGTTGTCA TGAATAGAAA TAAAATCTAC ACCTAGCTCT TTGAATTTTT	300<br>
CACTTAACTC AATTAGGTGT TTTGTAGAGC GAGAAATTCG ATCAAGTTTG TAAACAACTA	360<br>
TCTTATCGCC TTTACGTAAT ACTTTTAGCA ACTCTTCGAG TTGAGGGCGC TCTTTTTTTA	420<br>
TTCCTGTTAT TTTCTCCTGA TATAGCCTTT CTACACCATA TTGTTGCAAA GCATCTATTT	480<br>
GCATATCGAG ATTTTGTTCT TCTGTGCTGA CACGAGCATA ACCAAAAATC AAATTGGTTT	540<br>
CACTTCCTAT CTAAATATAT CTATTAAAAT AGCACCAAAA ACCTTATTAA ATTAAAATAA	600<br>
GGAACTTTGT TTTTGGATAT GGATTTTGGT ACTCAATATG GATGAGTTTT TAACGCTTTT	660<br>
GTTAAAAAAC AAACAAGTGC CATAAACGGT CGTTTTTGGG ATGACATAAT AAATAATCTG	720<br>
TTTGATTAAC CTAACCTTGT ATCCTTACAG CCCAGTTTTA TTTGTACTTC AACTGACTGA	780<br>
ATATGAAAAC AACATGAAGG TTTCATAAAA TTTATATATT TTCCATAACG GATGCTCTAT	840<br>
CTTTAGGTTA TAGTTAAATT ATAAGAAAAA AACAAACGGA GGGAGTGAAA AAAAGCATCT	900<br>
TCTCTATAAT TTTACAGGCT CTTTAATAAG AAGGGGGGAG ATTAGATAAT AAATATGAAT	960<br>
ATCTATCTAT AATTGTTTGC TTCTACAATA ACTTATCTAA CTTTCATATA CAACAACAAA	1020<br>
ACAGACTAAA TCCAGATTGT ATATTCATTT TCAGTTGTTC CTTTATAAAA TAATTTCATA	1080<br>
A ATG AAA AGA ATG GAG GGA AAG TTG TTT ATG GTG TCA AAA AAA TTA	1126<br>
Met Lys Arg Met Glu Gly Lys Leu Phe Met Val Ser Lys Lys Leu<br>
15               10               15<br>
CAA GTA GTT ACT AAA ACT GTA TTG CTT ACT ACA GTT TTC TCT ATA TCT	1174<br>
Gin Val Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser He Ser<br>
20               25               30<br>
TTA TTA AAT AAT GAA GTG ATA AAA GCT GAA CAA TTA AAT ATA AAT TCT	1222<br>
Leu Leu Asn Asn Glu Val lie Lys Ala Glu Gin Leu Asn He Asn Ser<br>
35               40               45<br>
CAA AGT AAA TAT ACT AAC TTG CAA AAT CTA AAA ATC ACT GAC AAG GTA	1270<br>
Gin Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys He Thr Asp Lys Val<br>
50                55                60<br>
GAG GAT TTT AAA GAA GAT AAG GAA AAA GCG AAA GAA TGG GGG AAA GAA	1318<br>
Glu Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu<br>
65	70	75<br>
AAA GAA AAA GAG TGG AAA CTA ACT GCT ACT GAA AAA GGA AAA ATG AAT     1366<br>
Lys Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn<br>
80	85	90	95<br>
AAT TTT TTA GAT AAT AAA AAT GAT ATA AAG ACA AAT TAT AAA GAA ATT     1414<br>
Asn Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie<br>
100	105	110<br>
ACT TTT TCT ATG GCA GGC TCA TTT GAA GAT GAA ATA AAA GAT TTA AAA     1462<br>
Thr Phe Ser Met Ala Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys<br>
115	120	125<br>
GAA ATT GAT AAG ATG TTT GAT AAA ACC AAT CTA TCA AAT TCT ATT ATC     1510<br>
Glu lie Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser lie lie<br>
130	135	140<br>
ACC TAT AAA AAT GTG GAA CCG ACA ACA ATT GGA TTT AAT AAA TCT TTA     1558 Thr Tyr Lys Asn Val Glu Pro Thr Thr lie Gly Phe Asn Lys Ser Leu<br>
145	150	155<br>
ACA GAA GGT AAT ACG ATT AAT TCT GAT GCA ATG GCA CAG TTT AAA GAA     1606<br>
Thr Glu Gly Asn Thr lie Asn Ser Asp Ala Met Ala Gin Phe Lys Glu<br>
160	165	170	175<br>
CAA TTT TTA GAT AGG GAT ATT AAG TTT GAT ACT TAT CTA GAT ACG CAT     1654<br>
Gin Phe Leu Asp Arg Asp lie Lys Phe Asp Ser Tyr Leu Asp Thr His<br>
180	185	190<br>
TTA ACT GCT CAA CAA GTT TCC AGT AAA GAA AGA GTT ATT TTG AAG GTT     1702<br>
Leu Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val<br>
195	200	205<br>
ACG GTT CCG AGT GGG AAA GGT TCT ACT ACT CCA ACA AAA GCA GGT GTC     1750<br>
Thr Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val<br>
210	215	220<br>
ATT TTA AAT AAT AGT GAA TAG AAA ATG CTC ATT GAT AAT GGG TAT ATG     1798<br>
lie Leu Asn Asn Ser Glu Tyr Lys Met Leu lie Asp Asn Gly Tyr Met<br>
225	230	235<br>
GTC CAT GTA GAT AAG GTA TCA AAA GTG GTG AAA AAA GGG GTG GAG TGC     1846<br>
Val His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys<br>
240	245	250	255<br>
TTA CAA ATT GAA GGG ACT TTA AAA AAG AGT CTT GAG TTT AAA AAT GAT     1894<br>
Leu Gin lie Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp<br>
260	265	270<br>
ATA AAT GCT GAA GCG CAT AGC TGG GGT ATG AAG AAT TAT GAA GAG TGG     1942<br>
He Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp<br>
275	280	285<br>
GCT AAA GAT TTA ACC GAT TCG CAA AGG GAA GCT TTA GAT GGG TAT GCT     1990<br>
Ala Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala<br>
290	295	300<br>
AGG CAA GAT TAT AAA GAA ATC AAT AAT TAT TTA AGA AAT CAA GGC GGA     2038<br>
Arg Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly<br>
305	310	315<br>
AGT GGA AAT GAA AAA CTA GAT GCT CAA ATA AAA AAT ATT TCT GAT GCT     2086<br>
Ser Gly Asn Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala<br>
320	325	330	335<br>
TTA GGG AAG AAA CCA ATA CCG GAA AAT ATT ACT GTG TAT AGA TGG TGT     2134<br>
Leu Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys<br>
340	345	350<br>
GGC ATG CCG GAA TTT GGT TAT CAA ATT AGT GAT CCG TTA CCT TCT TTA     2182<br>
Gly Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu<br>
355	360	365<br>
AAA GAT TTT GAA GAA CAA TTT TTA AAT ACA ATC AAA GAA GAG AAA GGA     2230<br>
Lys Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp Lys Gly<br>
370	375	380<br>
TAT ATG AGT ACA AGC TTA TCG AGT GAA CGT CTT GCA GCT TTT GGA TCT     2278<br>
Tyr Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser<br>
385	390	395<br>
AGA AAA ATT ATA TTA CGA TTA CAA GTT CCG AAA GGA AGT ACG GGT GCG     2326<br>
Arg Lys lie lie Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala<br>
400	405	410	415<br>
TAT TTA AGT GCC ATT GGT GGA TTT GCA AGT GAA AAA GAG ATC CTA CTT     2374<br>
Tyr Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu<br>
420	425	430<br>
GAT AAA GAT AGT AAA TAT CAT ATT GAT AAA GTA ACA GAG GTA ATT ATT     2422<br>
Asp Lys Asp Ser Lys Tyr His He Asp Lys Val Thr Glu Val He He<br>
435	440	445<br>
AAA GGT GTT AAG CGA TAT GTA GTG GAT GCA ACA TTA TTA ACA AAT        2467<br>
Lys Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
450	455	460<br>
TAAGGAGATG AAAAATATGA AGAAAAAGTT AGCAAGTGTT GTAACGTGTA CGTTATTAGC	2527<br>
TCCTATGTTT TTGAATGGAA ATGTGAATGC TGTTTACGCA GACAGCAAAA CAAATCAAAT	2587<br>
TTCTACAACA CAGAAAAATC AACAGAAAGA GATGGACCGA AAAGGATTAC TTGGGTATTA	2647<br>
TTTCAAAGGA AAAGATTTTA GTAATCTTAC TATGTTTGCA CCGACACGTG ATAGTACTCT	2707<br>
TATTTATGAT CAACAAACAG CAAATAAACT ATTAGATAAA AAACAACAAG AATATCAGTC	2767<br>
TATTCGTTGG ATTGGTTTGA TTCAGAGTAA AGAAACGGGA GATTTCACAT TTAACTTATC	2827<br>
TGAGGATGAA CAGGCAATTA TAGAAATCAA TGGGAAAATT ATTTCTAATA AAGGGAAAGA	2887<br>
AAAGCAAGTT GTCCATTTAG AAAAAGGAAA ATTAGTTCCA ATCAAAATAG AGTATCAATC	2947<br>
AGATACAAAA TTTAATATTG ACAGTAAAAC ATTTAAAGAA CTTAAATTAT TTAAAATAGA	3007<br>
TAGTCAAAAC CAACCCCAGC AAGTCCAGCA AGATGAACTG AGAAATCCTG AATTTAACAA	3067<br>
GAAAGAATCA CAGGAATTCT TAGCGAAACC ATCGAAAATA AATCTTTTCA CTCAAAAAAT	3127<br>
GAAAAGGGAA ATTGATGAAG ACACGGATAC GGATGGGGAC TCTATTCCTG ACCTTTGGGA	3187<br>
AGAAAATGGG TATACGATTC ACAATAGAAT CGCTGTAAAG TGGGACGATT CTCTAGCAAG	3247<br>
TAAAGGGTAT ACGAAATTTG TTTCAAATCC ACTAGAAAGT CACACAGTTG GTGATCCTTA	3307<br>
TACAGATTAT GAAAAGGCAG CAAGAGATCT AGATTTGTCA AATGCAAAGG AAACGTTTAA	3367<br>
CCCATTGGTA GCTGCTTTTC CAAGTGTGAA TGTTAGTATG GAAAAGGTGA TATTATCACC	3427<br>
AAATGAAAAT TTATCCAATA GTGTAGAGTC TCATTCATCC ACGAATTGGT CTTATACAAA	3487<br>
TACAGAAGGT GCTTCTGTTG AAGCGGGGAT TGGACCAAAA GGTATTTCGT TCGGAGTTAG	3547<br>
CGTAAACTAT CAACACTCTG AAACAGTTGC ACAAGAATGG GGAACATCTA CAGGAAATAC	3607<br>
TTCGCAATTC AATACGGCTT CAGCGGGATA TTTAAATGCA AATGTTCGAT ATAACAATGT	3667<br>
AGGAACTGGT GCCATCTACG ATGTAAAACC TACAACAAGT TTTGTATTAA ATAACGATAC	3727<br>
TATCGCAACT ATTACGGCGA AATCTAATTC TACAGCCTTA AATATATCTC CTGGAGAAAG	3787<br>
TTACCCGAAA AAAGGACAAA ATGGAATCGC AATAACATCA ATGGATGATT TTAATTCCCA	3847<br>
TCCGATTACA TTAAATAAAA AACAAGTAGA TAATCTGCTA AATAATAAAC CTATGATGTT	3907<br>
GGAAACAAAC CAAACAGATG GTGTTTATAA GATAAAAGAT ACACATGGAA ATATAGTAAC	3967<br>
TGGCGGAGAA TGGAATGGTG TCATACAACA AATCAAGGCT AAAACAGCGT CTATTATTGT	4027<br>
GGATGATGGG GAACGTGTAG CAGAAAAACG TGTAGCGGCA AAAGATTATG AAAATCCAGA	4087<br>
AGATAAAACA CCGTCTTTAA CTTTAAAAGA TGCCCTGAAG CTTTCATATC CAGATGAAAT	4147<br>
AAAAGAAATA GAGGGATTAT TATATTATAA AAACAAACCG ATATACGAAT CGAGCGTTAT	4207<br>
GACTTACTTA GATGAAAATA CAGCAAAAGA AGTGACCAAA CAATTAAATG ATACCACTGG	4267<br>
GAAATTTAAA GATGTAAGTC ATTTATATGA TGTAAAACTG ACTCCAAAAA TGAATGTTAC	4327<br>
AATCAAATTG TCTATACTTT ATGATAATGC TGAGTCTAAT GATAACTCAA TTGGTAAATG	4387<br>
GACAAACACA AATATTGTTT CAGGTGGAAA TAACGGAAAA AAACAATATT CTTCTAATAA	4447<br>
TCCGGATGCT AATTTGACAT TAAATACAGA TGCTCAAGAA AAATTAAATA AAAATCGTGA	4507<br>
CTATTATATA AGTTTATATA TGAAGTCAGA AAAAAACACA CAATGTGAGA TTACTATAGA	4567<br>
TGGGGAGATT TATCCGATCA CTACAAAAAC AGTGAATGTG AATAAAGACA ATTACAAAAG	4627<br>
ATTAGATATT ATAGCTCATA ATATAAAAAG TAATCCAATT TCTTCACTTC ATATTAAAAC	4687<br>
GAATGATGAA ATAACTTTAT TTTGGGATGA TATTTCTATA ACAGATGTAG CATCAATAAA	4747<br>
ACCGGAAAAT TTAACAGATT CAGAAATTAA ACAGATTTAT AGTAGGTATG GTATTAAGTT	4807<br>
AGAAGATGGA ATCCTTATTG ATAAAAAAGG TGGGATTCAT TATGGTGAAT TTATTAATGA	4867<br>
AGCTAGTTTT AATATTGAAC CATTGCAAAA TTATGTGACC AAATATGAAG TTACTTATAG	4927<br>
TAGTGAGTTA GGACCAAACG TGAGTGACAC ACTTGAAAGT GATAAAATTT ACAAGGATGG	4987<br>
GACAATTAAA TTTGATTTTA CCAAATATAG TAAAAATGAA CAAGGATTAT TTTATGACAG	5047<br>
TGGATTAAAT TGGGACTTTA AAATTAATGC TATTACTTAT GATGGTAAAG AGATGAATGT	5107<br>
TTTTCATAGA TATAATAAAT AGTTATTATA TCTATGAAGC TGGTGCTAAA GATAGTGTAA	5167<br>
AAGTTAATAT ACTGTAGGAT TGTAATAAAA GTAATGGAAT TGATATCGTA CTTTGGAGTG	5227<br>
GGGGATACTT TGTAAATAGT TCTATCAGAA ACATTAGACT AAGAAAAGTT ACTACCCCCA	5287<br>
CTTGAAAATG AAGATTCAAC TGATTACAAA CAACCTGTTA AATATTATAA GGTTTTAACA	5347<br>
AAATATTAAA CTCTTTATGT TAATACTGTA ATATAAAGAG TTTAATTGTA TTCAAATGAA	5407<br>
GCTTTCCCAC AAAATTAGAC TGATTATCTA ATGAAATAAT CAGTCTAATT TTGTAGAACA	5467<br>
GGTCTGGTAT TATTGTACGT GGTCACTAAA AGATATCTAA TATTATTGGG CAAGGCGTTC	5527<br>
CATGATTGAA TCCTCGAATG TCTTGCCCTT TTCATTTATT TAAGAAGGAT TGTGGAGAAA	5587<br>
TTATGGTTTA GATAATGAAG AAAGACTTCA CTTCTAATTT TTGATGTTAA ATAAATCAAA	5647<br>
ATTTGGCGAT TCACATTGTT TAATCCACTG ATAAAACATA CTGGAGTGTT CTTAAAAAAT	5707<br>
CAGCTTTTTT CTTTATAAAA TTTTGCTTAG CGTACGAAAT TCGTGTTTTG TTGGTGGGAC	5767<br>
CCCATGCCCA TCAACTTAAG AGTAAATTAG TAATGAACTT TCGTTCATCT GGATTAAAAT	5827<br>
AACCTCAAAT TAGGACATGT TTTTAAAAAT AAGCAGACCA AATAAGCCTA GAATAGGTAT	5887<br>
CATTTTTAAA AATTATGCTG CTTTCTTTTG TTTTCCAAAT CCATTATACT CATAAGCAAC	5947<br>
ACCCATAATG TCAAAGACTG TTTTTGTCTC ATATCGATAA GCTTGATATC GAATTCCTGC	6007<br>
AGCCCGGGGG ATCCACTAGT TCTAGAGCGG CCGCCACCGC GG	6049<br>
(2) INFORMATION FOR SEQ ID NO:2:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 462 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:2:<br>
Met Lys Arg Met Glu Gly Lys Leu Phe Met Val Ser Lys Lys Leu Gin<br>
15	10	15<br>
Val Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser lie Ser Leu<br>
20	25	30<br>
Leu Asn Asn Glu Val lie Lys Ala Glu Gin Leu Asn lie Asn Ser Gin<br>
35	40	45<br>
Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu<br>
50	55	60<br>
Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
65	70	75	80<br>
Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn<br>
85	90	95<br>
Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr<br>
100	105	110<br>
Phe Ser Met Ala Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu<br>
115	120	125<br>
lie Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser lie lie Thr<br>
130	135	140<br>
Tyr Lys Asn Val Glu Pro Thr Thr lie Gly Phe Asn Lys Ser Leu Thr<br>
145	150	155	160<br>
Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
165	170	175<br>
Phe Leu Asp Arg Asp He Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
180	185	190<br>
Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr<br>
195	200	205<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He<br>
210	215	220<br>
Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val<br>
225	230	235	240<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu<br>
245	250	255<br>
Gin lie Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp lie<br>
260	265	270<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala<br>
275	280	285<br>
Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
290	295	300<br>
Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
305	310	315	320<br>
Gly Asn Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala Leu<br>
325	330	335<br>
Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly<br>
340	345	350<br>
Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys<br>
355	360	365<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp Lys Gly Tyr<br>
370	375	380<br>
Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
385	390	395	400<br>
Lys lie He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
405	410	415<br>
Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu Asp<br>
420	425	430<br>
Lys Asp Ser Lys Tyr His He Asp Lys Val Thr Glu Val lie He Lys<br>
435	440	445<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
450	455	460<br>
(2) INFORMATION FOR SEQ ID NO:3:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 20 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..20<br>
(D) OTHER INFORMATION: /note= "Signal peptide for vacuolar<br>
targetting"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:3:<br>
Ser Ser Ser Ser Phe Ala Asp Ser Asn Pro lie Arg Val Thr Asp Arg<br>
15	10	15<br>
Ala Ala Ser Thr 20<br>
(2) INFORMATION FOR SEQ ID NO:4:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2655 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br><ii molecule type: dna hypothetical: no anti-sense:></ii>
(vi) ORIGINAL SOURCE:<br>
ORGANISM: Bacillus cereus<br>
STRAIN: AB78<br>
INDIVIDUAL ISOLATE: NRRL B-21058<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1..2652<br>
(D)	OTHER INFORMATION: /product= "100 kDa protein VIPlA(a)"<br>
/note= "This sequence is identical to the portion of SEQ ID N0:l<br>
between and including nucleotide 2475 to 5126."<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:4:<br>
ATG AAA AAT ATG AAG AAA AAG TTA GCA ACT GTT GTA ACG TGT ACG TTA       48<br>
Met Lys Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys Thr Leu<br>
465	470	475<br>
TTA GCT CCT ATG TTT TTG AAT GGA AAT GTG AAT GCT GTT TAG GCA GAC       96<br>
Leu Ala Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Tyr Ala Asp<br>
480	485	490<br>
AGC AAA ACA AAT CAA ATT TCT ACA ACA CAG AAA AAT CAA CAG AAA GAG      144<br>
Ser Lys Thr Asn Gin He Ser Thr Thr Gin Lys Asn Gin Gin Lys Glu<br>
495	500	505	510<br>
ATG GAC CGA AAA GGA TTA CTT GGG TAT TAT TTC AAA GGA AAA GAT TTT<br>
Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe<br>
515	520	525<br>
192<br><br>
ACT AAT CTT ACT ATG TTT GCA CCG ACA CGT GAT ACT ACT CTT ATT TAT<br>
Ser Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu lie Tyr<br>
530	535	540<br>
240<br><br>
GAT CAA CAA ACA GCA AAT AAA CTA TTA GAT AAA AAA CAA CAA GAA TAT<br>
Asp Gin Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin Glu Tyr<br>
545	550	555<br>
288<br>
CAG TCT ATT CGT TGG ATT GGT TTG ATT CAG AGT AAA GAA ACG GGA GAT      336<br>
Gin Ser lie Arg Trp lie Gly Leu lie Gin Ser Lys Glu Thr Gly Asp<br>
560	565	570<br>
TTC ACA TTT AAC TTA TCT GAG GAT GAA CAG GCA ATT ATA GAA ATC AAT      384<br>
Phe Thr Phe Asn Leu Ser Glu Asp Glu Gin Ala He He Glu He Asn<br>
575	580	585	590<br>
GGG AAA ATT ATT TCT AAT AAA GGG AAA GAA AAG CAA GTT GTC CAT TTA      432<br>
Gly Lys He He Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu<br>
595	600	605<br>
GAA AAA GGA AAA TTA GTT CCA ATC AAA ATA GAG TAT CAA TCA GAT ACA      480<br>
Glu Lys Gly Lys Leu Val Pro He Lys He Glu Tyr Gin Ser Asp Thr<br>
610	615	620<br>
AAA TTT AAT ATT GAC AGT AAA ACA TTT AAA GAA CTT AAA TTA TTT AAA      528<br>
Lys Phe Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys<br>
625	630	635<br>
ATA GAT AGT CAA AAC CAA CCC CAG CAA GTC CAG CAA GAT GAA CTG AGA      576<br>
He Asp Ser Gin Asn Gin Pro Gin Gin Val Gin Gin Asp Glu Leu Arg<br>
640	645	650<br>
AAT CCT GAA TTT AAC AAG AAA GAA TCA CAG GAA TTC TTA GCG AAA CCA      624<br>
Asn Pro Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Pro<br>
655	660	665	670<br>
TCG AAA ATA AAT CTT TTC ACT CAA AAA ATG AAA AGG GAA ATT GAT GAA      672<br>
Ser Lys He Asn Leu Phe Thr Gin Lys Met Lys Arg Glu He Asp Glu<br>
675	680	685<br>
GAC ACG GAT ACG GAT GGG GAC TCT ATT CCT GAC CTT TGG GAA GAA AAT      720<br>
Asp Thr Asp Thr Asp Gly Asp Ser He Pro Asp Leu Trp Glu Glu Asn<br>
690	695	700<br>
GGG TAT ACG ATT CAA AAT AGA ATC GCT GTA AAG TGG GAC GAT TCT CTA<br>
Gly Tyr Thr He Gin Asn Arg He Ala Val Lys Trp Asp Asp Ser Leu<br>
705	710	715<br>
768<br><br>
GCA AGT AAA GGG TAT ACG AAA TTT GTT TCA AAT CCA CTA GAA AGT CAC<br>
Ala Ser Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu Ser His<br>
720	725	730<br>
816<br>
ACA GTT GGT GAT CCT TAT ACA GAT TAT GAA AAG GCA GCA AGA GAT CTA      864<br>
Thr Val Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu<br>
735	740	745	750<br>
GAT TTG TCA AAT GCA AAG GAA ACG TTT AAC CCA TTG GTA GCT GCT TTT      912<br>
Asp Leu Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe<br>
755	760	765<br>
CCA ACT GTG AAT GTT AGT ATG GAA AAG GTG ATA TTA TCA CCA AAT GAA<br>
Pro Ser Val Asn Val Ser Met Glu Lys Val He Leu Ser Pro Asn Glu<br>
770	775	780<br>
960<br><br>
AAT TTA TCC AAT AGT GTA GAG TCT CAT TCA TCC ACG AAT TGG TCT TAT<br>
Asn Leu Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr<br>
785	790	795<br>
1008<br>
ACA AAT ACA GAA GGT GCT TCT GTT GAA GCG GGG ATT GGA CCA AAA GGT     1056<br>
Thr Asn Thr Glu Gly Ala Ser Val Glu Ala Gly He Gly Pro Lys Gly<br>
800	805	810<br>
ATT TCG TTC GGA GTT AGC GTA AAC TAT CAA CAC TCT GAA ACA GTT GCA     1104<br>
He Ser Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr Val Ala<br>
815	820	825	830<br>
CAA GAA TGG GGA ACA TCT ACA GGA AAT ACT TCG CAA TTC AAT ACG GCT     1152<br>
Gin Glu Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala<br>
835	840	845<br>
TCA GCG GGA TAT TTA AAT GCA AAT GTT CGA TAT AAC AAT GTA GGA ACT     1200<br>
Ser Ala Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr<br>
850	855	860<br>
GGT GCC ATC TAG GAT GTA AAA CCT ACA ACA AGT TTT GTA TTA AAT AAC     1248<br>
Gly Ala He Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn<br>
865	870	875<br>
GAT ACT ATC GCA ACT ATT ACG GCG AAA TCT AAT TCT ACA GCC TTA AAT     1296<br>
Asp Thr He Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Asn<br>
880	885	890<br>
ATA TCT CCT GGA GAA AGT TAG CCG AAA AAA GGA CAA AAT GGA ATC GCA<br>
He Ser Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly He Ala<br>
895	900	905	910<br>
1344<br><br>
ATA ACA TCA ATG GAT GAT TTT AAT TCC CAT CCG ATT ACA TTA AAT AAA<br>
He Thr Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys<br>
915	920	925<br>
1392<br>
AAA CAA GTA GAT AAT CTG CTA AAT AAT AAA CCT ATG ATG TTG GAA ACA     1440<br>
Lys Gin Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu Glu Thr<br>
930	935	940<br>
AAC CAA ACA GAT GGT GTT TAT AAG ATA AAA GAT ACA CAT GGA AAT ATA     1488 Asn Gin Thr Asp Gly Val Tyr Lys He Lys Asp Thr His Gly Asn He<br>
945	950	955<br>
GTA ACT GGC GGA GAA TGG AAT GGT GTC ATA CAA CAA ATC AAG GCT AAA     1536<br>
Val Thr Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys Ala Lys<br>
960	965	970<br>
ACA GCG TCT ATT ATT GTG GAT GAT GGG GAA CGT GTA GCA GAA AAA CGT     1584<br>
Thr Ala Ser He He Val Asp Asp Gly Glu Arg Val Ala Glu Lys Arg<br>
975	980	985	990<br>
GTA GCG GCA AAA GAT TAT GAA AAT CCA GAA GAT AAA ACA CCG TCT TTA     1632<br>
Val Ala Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro Ser Leu<br>
995	1000	1005<br>
ACT TTA AAA GAT GCC CTG AAG CTT TCA TAT CCA GAT GAA ATA AAA GAA     1680<br>
Thr Leu Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu<br>
1010	1015	1020<br>
ATA GAG GGA TTA TTA TAT TAT AAA AAC AAA CCG ATA TAG GAA TCG AGC     1728<br>
He Glu Gly Leu Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu Ser Ser<br>
1025	1030	1035<br>
GTT ATG ACT TAG TTA GAT GAA AAT ACA GCA AAA GAA GTG ACC AAA CAA     1776<br>
Val Met Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr Lys Gin<br>
1040	1045	1050<br>
TTA AAT GAT ACC ACT GGG AAA TTT AAA GAT GTA AGT CAT TTA TAT GAT     1824<br>
Leu Asn Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu Tyr Asp<br>
1055	1060	1065	1070<br>
GTA AAA CTG ACT CCA AAA ATG AAT GTT ACA ATC AAA TTG TCT ATA CTT     1872<br>
Val Lys Leu Thr Pro Lys Met Asn Val Thr He Lys Leu Ser He Leu<br>
1075	1080	1085<br>
TAT GAT AAT GCT GAG TCT AAT GAT AAC TCA ATT GGT AAA TGG ACA AAC     1920<br>
Tyr Asp Asn Ala Glu Ser Asn Asp Asn Ser He Gly Lys Trp Thr Asn<br>
1090	1095	1100<br>
ACA AAT ATT GTT TCA GGT GGA AAT AAC GGA AAA AAA CAA TAT TCT TCT     1968<br>
Thr Asn He Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr Ser Ser<br>
1105	1110	1115<br>
AAT AAT CCG GAT GCT AAT TTG ACA TTA AAT ACA GAT GCT CAA GAA AAA     2016<br>
Asn Asn Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin Glu Lys<br>
1120	1125	1130<br>
TTA AAT AAA AAT CGT GAC TAT TAT ATA AGT TTA TAT ATG AAG TCA GAA     2064<br>
Leu Asn Lys Asn Arg Asp Tyr Tyr He Ser Leu Tyr Met Lys Ser Glu<br>
1135	1140	1145	1150<br>
AAA AAC ACA CAA TGT GAG ATT ACT ATA GAT GGG GAG ATT TAT CCG ATC     2112<br>
Lys Asn Thr Gin Cys Glu He Thr He Asp Gly Glu He Tyr Pro He<br>
1155	1160	1165<br>
ACT ACA AAA ACA GTG AAT GTG AAT AAA GAC AAT TAG AAA AGA TTA GAT     2160<br>
Thr Thr Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg Leu Asp<br>
1170	1175	1180<br>
ATT ATA GCT CAT AAT ATA AAA AGT AAT CCA ATT TCT TCA CTT CAT ATT     2208<br>
lie lie Ala His Asn lie Lys Ser Asn Pro lie Ser Ser Leu His lie<br>
1185	1190	1195<br>
AAA ACG AAT GAT GAA ATA ACT TTA TTT TGG GAT GAT ATT TCT ATA ACA     2256<br>
Lys Thr Asn Asp Glu lie Thr Leu Phe Trp Asp Asp lie Ser lie Thr<br>
1200	1205	1210<br>
GAT GTA GCA TCA ATA AAA CCG GAA AAT TTA ACA GAT TCA GAA ATT AAA     2304<br>
Asp Val Ala Ser lie Lys Pro Glu Asn Leu Thr Asp Ser Glu lie Lys<br>
1215	1220	1225	1230<br>
CAG ATT TAT AGT AGG TAT GGT ATT AAG TTA GAA GAT GGA ATC CTT ATT     2352<br>
Gin lie Tyr Ser Arg Tyr Gly lie Lys Leu Glu Asp Gly lie Leu lie<br>
1235	1240	1245<br>
GAT AAA AAA GGT GGG ATT CAT TAT GGT GAA TTT ATT AAT GAA GCT AGT     2400<br>
Asp Lys Lys Gly Gly lie His Tyr Gly Glu Phe lie Asn Glu Ala Ser<br>
1250	1255	1260<br>
TTT AAT ATT GAA CCA TTG CAA AAT TAT GTG ACC AAA TAT GAA GTT ACT     2448<br>
Phe Asn lie Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu Val Thr<br>
1265	1270	1275<br>
TAT AGT AGT GAG TTA GGA CCA AAC GTG AGT GAC ACA CTT GAA AGT GAT     2496<br>
Tyr Ser Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu Ser Asp<br>
1280	1285	1290<br>
AAA ATT TAG AAG GAT GGG ACA ATT AAA TTT GAT TTT ACC AAA TAT AGT     2544<br>
Lys lie Tyr Lys Asp Gly Thr He Lys Phe Asp Phe Thr Lys Tyr Ser<br>
1295	1300	1305	1310<br>
AAA AAT GAA CAA GGA TTA TTT TAT GAC AGT GGA TTA AAT TGG GAC TTT     2592<br>
Lys Asn Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp Asp Phe<br>
1315	1320	1325<br>
AAA ATT AAT GCT ATT ACT TAT GAT GGT AAA GAG ATG AAT GTT TTT CAT     2640<br>
Lys He Asn Ala He Thr Tyr Asp Gly Lys Glu Met Asn Val Phe His<br>
1330	1335	1340<br>
AGA TAT AAT AAA TAG	2655<br>
Arg Tyr Asn Lys 1345<br>
(2) INFORMATION FOR SEQ ID NO:5:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 884 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:5:<br>
Met Lys Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys Thr Leu<br>
15	10	15<br>
Leu Ala Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Tyr Ala Asp<br>
20	25	30<br>
Ser Lys Thr Asn Gin lie Ser Thr Thr Gin Lys Asn Gin Gin Lys Glu<br>
35	40	45<br>
Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe<br>
50	55	60<br>
Ser Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu lie Tyr<br>
65	70	75	80<br>
Asp Gin Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin Glu Tyr<br>
85	90	95<br>
Gin Ser lie Arg Trp lie Gly Leu He Gin Ser Lys Glu Thr Gly Asp<br>
100	105	110<br>
Phe Thr Phe Asn Leu Ser Glu Asp Glu Gin Ala lie lie Glu lie Asn<br>
115	120	125<br>
Gly Lys lie lie Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu<br>
130	135	140<br>
Glu Lys Gly Lys Leu Val Pro lie Lys lie Glu Tyr Gin Ser Asp Thr<br>
145	150	155	160<br>
Lys Phe Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys<br>
165	170	175<br>
He Asp Ser Gin Asn Gin Pro Gin Gin Val Gin Gin Asp Glu Leu Arg<br>
180	185	190<br>
Asn Pro Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Pro<br>
195	200	205<br>
Ser Lys He Asn Leu Phe Thr Gin Lys Met Lys Arg Glu He Asp Glu<br>
210	215	220<br>
Asp Thr Asp Thr Asp Gly Asp Ser He Pro Asp Leu Trp Glu Glu Asn<br>
225	230	235	240<br>
Gly Tyr Thr He Gin Asn Arg He Ala Val Lys Trp Asp Asp Ser Leu<br>
245	250	255<br>
Ala Ser Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu Ser His<br>
260	265	270<br>
Thr Val Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu<br>
275	280	285<br>
Asp Leu Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe<br>
290	295	300<br>
Pro Ser Val Asn Val Ser Met Glu Lys Val lie Leu Ser Pro Asn Glu<br>
305	310	315	320<br>
Asn Leu Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr<br>
325	330	335<br>
Thr Asn Thr Glu Gly Ala Ser Val Glu Ala Gly lie Gly Pro Lys Gly<br>
340	345	350<br>
lie Ser Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr Val Ala<br>
355	360	365<br>
Gin Glu Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala<br>
370	375	380<br>
Ser Ala Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr<br>
385	390	395	400<br>
Gly Ala He Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn<br>
405	410	415<br>
Asp Thr He Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Asn<br>
420	425	430<br>
He Ser Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly He Ala<br>
435	440	445<br>
He Thr Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys<br>
450	455	460<br>
Lys Gin Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu Glu Thr<br>
465	470	475	480<br>
Asn Gin Thr Asp Gly Val Tyr Lys He Lys Asp Thr His Gly Asn He<br>
485	490	495<br>
Val Thr Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys Ala Lys<br>
500	505	510<br>
Thr Ala Ser He He Val Asp Asp Gly Glu Arg Val Ala Glu Lys Arg<br>
515	520	525<br>
Val Ala Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro Ser Leu<br>
530	535 .	540<br>
Thr Leu Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu<br>
545	550	555	560<br>
He Glu Gly Leu Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu Ser Ser<br>
565<br>
570<br>
575<br>
Val Met Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr Lys Gin<br>
580	585	590<br>
Leu Asn Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu Tyr Asp<br>
595	600	605<br>
Val Lys Leu Thr Pro Lys Met Asn Val Thr lie Lys Leu Ser lie Leu<br>
610	615	620<br>
Tyr Asp Asn Ala Glu Ser Asn Asp Asn Ser lie Gly Lys Trp Thr Asn<br>
625	630	635	640<br>
Thr Asn He Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr Ser Ser<br>
645	650	655<br>
Asn Asn Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin Glu Lys<br>
660	665	670<br>
Leu Asn Lys Asn Arg Asp Tyr Tyr lie Ser Leu Tyr Met Lys Ser Glu<br>
675	680	685<br>
Lys Asn Thr Gin Cys Glu lie Thr He Asp Gly Glu He Tyr Pro He<br>
690	695	700<br>
Thr Thr Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg Leu Asp<br>
705	710	715	720<br>
He He Ala His Asn He Lys Ser Asn Pro He Ser Ser Leu His lie<br>
725	730	735<br>
Lys Thr Asn Asp Glu He Thr Leu Phe Trp Asp Asp He Ser He Thr<br>
740	745	750<br>
Asp Val Ala Ser He Lys Pro Glu Asn Leu Thr Asp Ser Glu He Lys<br>
755	760	765<br>
Gin He Tyr Ser Arg Tyr Gly He Lys Leu Glu Asp Gly He Leu He<br>
770	775	780<br>
Asp Lys Lys Gly Gly He His Tyr Gly Glu Phe He Asn Glu Ala Ser<br>
785	790	795	800<br>
Phe Asn He Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu Val Thr<br>
805	810	815<br>
Tyr Ser Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu Ser Asp<br>
820	825	830<br>
Lys He Tyr Lys Asp Gly Thr He Lys Phe Asp Phe Thr Lys Tyr Ser<br>
835	840	845<br>
Lys Asn Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp Asp Phe<br>
850	855	860<br>
Lys lie Asn Ala He Thr Tyr Asp Gly Lys Glu Met Asn Val Phe His<br>
865	870	875	880<br>
Arg Tyr Asn Lys<br>
(2) INFORMATION FOR SEQ ID NO:6:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2004 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic) (iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO<br>
(vi) ORIGINAL SOURCE:<br>
ORGANISM: Bacillus cereus<br>
STRAIN: AB78<br>
INDIVIDUAL ISOLATE: NRRL B-21058<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1. .2001<br>
(D)	OTHER INFORMATION: /product" "80 kDa protein VIPlA(a)"<br>
/note= "This sequence is identical to that found in SEQ ID N0:l<br>
between and including nucleotide positions 3126 and 5126"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:6:<br>
ATG AAA AGG GAA ATT GAT GAA GAC ACG GAT ACG GAT GGG GAC TCT ATT       48<br>
Met Lys Arg Glu He Asp Glu Asp Thr Asp Thr Asp Gly Asp Ser He<br>
885	890	895	900<br>
CCT GAC CTT TGG GAA GAA AAT GGG TAT ACG ATT CAA AAT AGA ATC GCT       96<br>
Pro Asp Leu Trp Glu Glu Asn Gly Tyr Thr He Gin Asn Arg He Ala<br>
905	910	915<br>
GTA AAG TGG GAC GAT TCT CTA GCA ACT AAA GGG TAT ACG AAA TTT GTT      144<br>
Val Lys Trp Asp Asp Ser Leu Ala Ser Lys Gly Tyr Thr Lys Phe Val<br>
920	925	930<br>
TCA AAT CCA CTA GAA ACT CAC ACA GTT GGT GAT CCT TAT ACA GAT TAT      192<br>
Ser Asn Pro Leu Glu Ser His Thr Val Gly Asp Pro Tyr Thr Asp Tyr<br>
935	940	945<br>
GAA AAG GCA GCA AGA GAT CTA GAT TTG TCA AAT GCA AAG GAA ACG TTT      240<br>
Glu Lys Ala Ala Arg Asp Leu Asp Leu Ser Asn Ala Lys Glu Thr Phe<br>
950	955	960<br>
AAC CCA TTG GTA GCT GCT TTT CCA ACT GTG AAT GTT AGT ATG GAA AAG      288<br>
Asn Pro Leu Val Ala Ala Phe Pro Ser Val Asn Val Ser Met Glu Lys<br>
965	970	975	980<br>
GTG ATA TTA TCA CCA AAT GAA AAT TTA TCC AAT AGT GTA GAG TCT CAT      336<br>
Val lie Leu Ser Pro Asn Glu Asn Leu Ser Asn Ser Val Glu Ser His<br>
985	990	995<br>
TCA TCC ACG AAT TGG TCT TAT ACA AAT ACA GAA GGT GCT TCT GTT GAA      384<br>
Ser Ser Thr Asn Trp Ser Tyr Thr Asn Thr Glu Gly Ala Ser Val Glu<br>
1000	1005	1010<br>
GCG GGG ATT GGA CCA AAA GGT ATT TCG TTC GGA GTT AGC GTA AAC TAT      432<br>
Ala Gly He Gly Pro Lys Gly He Ser Phe Gly Val Ser Val Asn Tyr<br>
1015	1020	1025<br>
CAA CAC TCT GAA ACA GTT GCA CAA GAA TGG GGA ACA TCT ACA GGA AAT      480<br>
Gin His Ser Glu Thr Val Ala Gin Glu Trp Gly Thr Ser Thr Gly Asn<br>
1030	1035	1040<br>
ACT TCG CAA TTC AAT ACG GCT TCA GCG GGA TAT TTA AAT GCA AAT GTT      528<br>
Thr Ser Gin Phe Asn Thr Ala Ser Ala Gly Tyr Leu Asn Ala Asn Val<br>
1045	1050	1055	1060<br>
CGA TAT AAC AAT GTA GGA ACT GGT GCC ATC TAG GAT GTA AAA CCT ACA      576<br>
Arg Tyr Asn Asn Val Gly Thr Gly Ala He Tyr Asp Val Lys Pro Thr<br>
1065	1070	1075<br>
ACA AGT TTT GTA TTA AAT AAC GAT ACT ATC GCA ACT ATT ACG GCG AAA      624<br>
Thr Ser Phe Val Leu Asn Asn Asp Thr He Ala Thr He Thr Ala Lys<br>
1080	1085	1090<br>
TCT AAT TCT ACA GCC TTA AAT ATA TCT CCT GGA GAA AGT TAG CCG AAA      672<br>
Ser Asn Ser Thr Ala Leu Asn He Ser Pro Gly Glu Ser Tyr Pro Lys<br>
1095	1100	1105<br>
AAA GGA CAA AAT GGA ATC GCA ATA ACA TCA ATG GAT GAT TTT AAT TCC      720<br>
Lys Gly Gin Asn Gly He Ala He Thr Ser Met Asp Asp Phe Asn Ser<br>
1110	1115	1120<br>
CAT CCG ATT ACA TTA AAT AAA AAA CAA GTA GAT AAT CTG CTA AAT AAT      768<br>
His Pro He Thr Leu Asn Lys Lys Gin Val Asp Asn Leu Leu Asn Asn<br>
1125	1130	1135	1140<br>
AAA CCT ATG ATG TTG GAA ACA AAC CAA ACA GAT GGT GTT TAT AAG ATA      816<br>
Lys Pro Met Met Leu Glu Thr Asn Gin Thr Asp Gly Val Tyr Lys He<br>
1145	1150	1155<br>
AAA GAT ACA CAT GGA AAT ATA GTA ACT GGC GGA GAA TGG AAT GGT GTC      864<br>
Lys Asp Thr His Gly Asn He Val Thr Gly Gly Glu Trp Asn Gly Val<br>
1160	1165	1170<br>
ATA CAA CAA ATC AAG GCT AAA ACA GCG TCT ATT ATT GTG GAT GAT GGG      912 He Gin Gin He Lys Ala Lys Thr Ala Ser He He Val Asp Asp Gly<br>
1175<br>
1180<br>
1185<br><br>
GAA CGT GTA GCA GAA AAA CGT GTA GCG GCA AAA GAT TAT GAA AAT CCA<br>
Glu Arg Val Ala Glu Lys Arg Val Ala Ala Lys Asp Tyr Glu Asn Pro<br>
1190	1195	1200<br>
960<br>
GAA GAT AAA ACA CCG TCT TTA ACT TTA AAA GAT GCC CTG AAG CTT TCA     1008<br>
Glu Asp Lys Thr Pro Ser Leu Thr Leu Lys Asp Ala Leu Lys Leu Ser<br>
1205	1210	1215	1220<br>
TAT CCA GAT GAA ATA AAA GAA ATA GAG GGA TTA TTA TAT TAT AAA AAC     1056<br>
Tyr Pro Asp Glu lie Lys Glu lie Glu Gly Leu Leu Tyr Tyr Lys Asn<br>
1225	1230	1235<br>
AAA CCG ATA TAG GAA TCG AGC GTT ATG ACT TAG TTA GAT GAA AAT ACA<br>
Lys Pro He Tyr Glu Ser Ser Val Met Thr Tyr Leu Asp Glu Asn Thr<br>
1240	1245	1250<br>
1104<br>
GCA AAA GAA GTG ACC AAA CAA TTA AAT GAT ACC ACT GGG AAA TTT AAA     1152<br>
Ala Lys Glu Val Thr Lys Gin Leu Asn Asp Thr Thr Gly Lys Phe Lys<br>
1255	1260	1265<br>
GAT GTA ACT CAT TTA TAT GAT GTA AAA CTG ACT CCA AAA ATG AAT GTT     1200<br>
Asp Val Ser His Leu Tyr Asp Val Lys Leu Thr Pro Lys Met Asn Val<br>
1270	1275	1280<br>
ACA ATC AAA TTG TCT ATA CTT TAT GAT AAT GCT GAG TCT AAT GAT AAC     1248<br>
Thr lie Lys Leu Ser He Leu Tyr Asp Asn Ala Glu Ser Asn Asp Asn<br>
1285	1290	1295	1300<br>
TCA ATT GGT AAA TGG ACA AAC ACA AAT ATT GTT TCA GGT GGA AAT AAC     1296<br>
Ser He Gly Lys Trp Thr Asn Thr Asn He Val Ser Gly Gly Asn Asn<br>
1305	1310	1315<br>
GGA AAA AAA CAA TAT TCT TCT AAT AAT CCG GAT GCT AAT TTG ACA TTA     1344<br>
Gly Lys Lys Gin Tyr Ser Ser Asn Asn Pro Asp Ala Asn Leu Thr Leu<br>
1320	1325	1330<br>
AAT ACA GAT GCT CAA GAA AAA TTA AAT AAA AAT CGT GAG TAT TAT ATA     1392<br>
Asn Thr Asp Ala Gin Glu Lys Leu Asn Lys Asn Arg Asp Tyr Tyr He<br>
1335	1340	1345<br>
AGT TTA TAT ATG AAG TCA GAA AAA AAC ACA CAA TGT GAG ATT ACT ATA     1440<br>
Ser Leu Tyr Met Lys Ser Glu Lys Asn Thr Gin Cys Glu He Thr He<br>
1350	1355	1360<br>
GAT GGG GAG ATT TAT CCG ATC ACT ACA AAA ACA GTG AAT GTG AAT AAA<br>
Asp Gly Glu He Tyr Pro He Thr Thr Lys Thr Val Asn Val Asn Lys<br>
1365	1370	1375	1380<br>
1488<br><br>
GAC AAT TAG AAA AGA TTA GAT ATT ATA GCT CAT AAT ATA AAA AGT AAT<br>
Asp Asn Tyr Lys Arg Leu Asp He He Ala His Asn He Lys Ser Asn<br>
1385	1390	1395<br>
1536<br><br>
CCA ATT TCT TCA CTT CAT ATT AAA ACG AAT GAT GAA ATA ACT TTA TTT<br>
1584<br>
Pro lie Ser Ser Leu His lie Lys Thr Asn Asp Glu lie Thr Leu Phe<br>
1400	1405	1410<br>
TGG GAT GAT ATT TCT ATA ACA GAT GTA GCA TCA ATA AAA CCG GAA AAT     1632<br>
Trp Asp Asp He Ser He Thr Asp Val Ala Ser He Lys Pro Glu Asn<br>
1415	1420	1425<br>
TTA ACA GAT TCA GAA ATT AAA CAG ATT TAT AGT AGG TAT GGT ATT AAG     1680<br>
Leu Thr Asp Ser Glu He Lys Gin He Tyr Ser Arg Tyr Gly He Lys<br>
1430	1435	1440<br>
TTA GAA GAT GGA ATC CTT ATT GAT AAA AAA GGT GGG ATT CAT TAT GGT     1728<br>
Leu Glu Asp Gly He Leu lie Asp Lys Lys Gly Gly He His Tyr Gly<br>
1445	1450	1455	1460<br>
GAA TTT ATT AAT GAA GCT AGT TTT AAT ATT GAA CCA TTG CCA AAT TAT     1776<br>
Glu Phe He Asn Glu Ala Ser Phe Asn He Glu Pro Leu Pro Asn Tyr<br>
1465	1470	1475<br>
GTG ACC AAA TAT GAA GTT ACT TAT AGT AGT GAG TTA GGA CCA AAC GTG     1824<br>
Val Thr Lys Tyr Glu Val Thr Tyr Ser Ser Glu Leu Gly Pro Asn Val<br>
1480	1485	1490<br>
AGT GAC ACA CTT GAA AGT GAT AAA ATT TAG AAG GAT GGG ACA ATT AAA     1872<br>
Ser Asp Thr Leu Glu Ser Asp Lys He Tyr Lys Asp Gly Thr He Lys<br>
1495	1500	1505<br>
TTT GAT TTT ACC AAA TAT AGT AAA AAT GAA CAA GGA TTA TTT TAT GAC     1920<br>
Phe Asp Phe Thr Lys Tyr Ser Lys Asn Glu Gin Gly Leu Phe Tyr Asp<br>
1510	1515	1520<br>
AGT GGA TTA AAT TGG GAC TTT AAA ATT AAT GCT ATT ACT TAT GAT GGT     1968<br>
Ser Gly Leu Asn Trp Asp Phe Lys He Asn Ala He Thr Tyr Asp Gly<br>
1525	1530	1535	1540<br>
AM GAG ATG AAT GTT TTT CAT AGA TAT AAT AAA TAG	2004<br>
Lys Glu Met Asn Val Phe His Arg Tyr Asn Lys<br>
1545	1550<br>
(2) INFORMATION FOR SEQ ID NO:7:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 667 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:7:<br>
Met Lys Arg Glu He Asp Glu Asp Thr Asp Thr Asp Gly Asp Ser He<br>
15	10	15<br>
Pro Asp Leu Trp Glu Glu Asn Gly Tyr Thr He Gin Asn Arg He Ala<br>
20<br>
25<br>
30<br>
Val Lys Trp Asp Asp Ser Leu Ala Ser Lys Gly Tyr Thr Lys Phe Val<br>
35	40	45<br>
Ser Asn Pro Leu Glu Ser His Thr Val Gly Asp Pro Tyr Thr Asp Tyr<br>
50	55	60<br>
Glu Lys Ala Ala Arg Asp Leu Asp Leu Ser Asn Ala Lys Glu Thr Phe<br>
65	70	75	80<br>
Asn Pro Leu Val Ala Ala Phe Pro Ser Val Asn Val Ser Met Glu Lys<br>
85	90	95<br>
Val lie Leu Ser Pro Asn Glu Asn Leu Ser Asn Ser Val Glu Ser His<br>
100	105	110<br>
Ser Ser Thr Asn Trp Ser Tyr Thr Asn Thr Glu Gly Ala Ser Val Glu<br>
115	120	125<br>
Ala Gly He Gly Pro Lys Gly He Ser Phe Gly Val Ser Val Asn Tyr<br>
130	135	140<br>
Gin His Ser Glu Thr Val Ala Gin Glu Trp Gly Thr Ser Thr Gly Asn<br>
145	150	155	160<br>
Thr Ser Gin Phe Asn Thr Ala Ser Ala Gly Tyr Leu Asn Ala Asn Val<br>
165	170	175<br>
Arg Tyr Asn Asn Val Gly Thr Gly Ala He Tyr Asp Val Lys Pro Thr<br>
180	185	190<br>
Thr Ser Phe Val Leu Asn Asn Asp Thr He Ala Thr He Thr Ala Lys<br>
195	200	205<br>
Ser Asn Ser Thr Ala Leu Asn He Ser Pro Gly Glu Ser Tyr Pro Lys<br>
210	215	220<br>
Lys Gly Gin Asn Gly He Ala He Thr Ser Met Asp Asp Phe Asn Ser<br>
225	230	235	240<br>
His Pro He Thr Leu Asn Lys Lys Gin Val Asp Asn Leu Leu Asn Asn<br>
245	250	255<br>
Lys Pro Met Met Leu Glu Thr Asn Gin Thr Asp Gly Val Tyr Lys He<br>
260	265	270<br>
Lys Asp Thr His Gly Asn He Val Thr Gly Gly Glu Trp Asn Gly Val<br>
275	280	285<br>
He Gin Gin He Lys Ala Lys Thr Ala Ser He He Val Asp Asp Gly<br>
290	295	300<br>
Glu Arg Val Ala Glu Lys Arg Val Ala Ala Lys Asp Tyr Glu Asn Pro<br>
305	310	315	320<br>
Glu Asp Lys Thr Pro Ser Leu Thr Leu Lys Asp Ala Leu Lys Leu Ser<br>
325	330	335<br>
Tyr Pro Asp Glu lie Lys Glu lie Glu Gly Leu Leu Tyr Tyr Lys Asn<br>
340	345	350<br>
Lys Pro He Tyr Glu Ser Ser Val Met Thr Tyr Leu Asp Glu Asn Thr<br>
355	360	365<br>
Ala Lys Glu Val Thr Lys Gin Leu Asn Asp Thr Thr Gly Lys Phe Lys<br>
370	375	380<br>
Asp Val Ser His Leu Tyr Asp Val Lys Leu Thr Pro Lys Met Asn Val<br>
385	390	395	400<br>
Thr He Lys Leu Ser He Leu Tyr Asp Asn Ala Glu Ser Asn Asp Asn<br>
405	410	415<br>
Ser He Gly Lys Trp Thr Asn Thr Asn He Val Ser Gly Gly Asn Asn<br>
420	425	430<br>
Gly Lys Lys Gin Tyr Ser Ser Asn Asn Pro Asp Ala Asn Leu Thr Leu<br>
435	440	445<br>
Asn Thr Asp Ala Gin Glu Lys Leu Asn Lys Asn Arg Asp Tyr Tyr He<br>
450	455	460<br>
Ser Leu Tyr Met Lys Ser Glu Lys Asn Thr Gin Cys Glu He Thr He<br>
465	470	475	480<br>
Asp Gly Glu He Tyr Pro He Thr Thr Lys Thr Val Asn Val Asn Lys<br>
485	490	495<br>
Asp Asn Tyr Lys Arg Leu Asp He He Ala His Asn He Lys Ser Asn<br>
500	505	510<br>
Pro He Ser Ser Leu His He Lys Thr Asn Asp Glu He Thr Leu Phe<br>
515	520	525<br>
Trp Asp Asp He Ser He Thr Asp Val Ala Ser He Lys Pro Glu Asn<br>
530	535	540<br>
Leu Thr Asp Ser Glu He Lys Gin He Tyr Ser Arg Tyr Gly He Lys<br>
545	550	555	560<br>
Leu Glu Asp Gly He Leu He Asp Lys Lys Gly Gly He His Tyr Gly<br>
565	570	575<br>
Glu Phe He Asn Glu Ala Ser Phe Asn He Glu Pro Leu Pro Asn Tyr<br>
580	585	590<br>
Val Thr Lys Tyr Glu Val Thr Tyr Ser Ser Glu Leu Gly Pro Asn Val<br>
595	600	605<br>
Ser Asp Thr Leu Glu Ser Asp Lys lie Tyr Lys Asp Gly Thr lie Lys<br>
610	615	620<br>
Phe Asp Phe Thr Lys Tyr Ser Lys Asn Glu Gin Gly Leu Phe Tyr Asp<br>
625	630	635	640<br>
Ser Gly Leu Asn Trp Asp Phe Lys lie Asn Ala lie Thr Tyr Asp Gly<br>
645	650	655<br>
Lys Glu Met Asn Val Phe His Arg Tyr Asn Lys<br>
660	665<br>
(2) INFORMATION FOR SEQ ID NO:8:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 16 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(v) FRAGMENT TYPE: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
ORGANISM: Bacillus cereus<br>
STRAIN: AB78<br>
INDIVIDUAL ISOLATE: NRRL B-21058<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..16<br>
(D)	OTHER INFORMATION: /note= "N-terminal sequence of<br>
protein purified from strain AB78"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO: 8:<br>
Lys Arg Glu lie Asp Glu Asp Thr Asp Thr Asx Gly Asp Ser lie Pro<br>
15	10	15<br>
(2) INFORMATION FOR SEQ ID NO:9:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 21 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic) (iii) HYPOTHETICAL: NO<br>
(iv) ANTI-SENSE: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: 1..21<br>
(D) OTHER INFORMATION: /note= "Oligonucleotide probe based on amino acids 3 to 9 of SEQ ID NO:8, using codon usage of Bacillus thuringiensis"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:9:<br>
GAAATTGATC AAGATACNGA T	21<br>
(2) INFORMATION FOR SEQ ID NO:10:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 14 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(v) FRAGMENT TYPE: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
ORGANISM: Bacillus thuringiensis<br>
STRAIN: AB88<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..14<br>
(D) OTHER INFORMATION: /note= "N-terminal amino acid sequence of protein known as anion exchange fraction 23 (smaller)"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:10:<br>
Xaa Glu Pro Phe Val Ser Ala Xaa Xaa Xaa Gin Xaa Xaa Xaa<br>
15	10<br>
(2) INFORMATION FOR SEQ ID NO:11:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 13 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Bacillus thuringiensis<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:11:<br>
Xaa Glu Tyr Glu Asn Val Glu Pro Phe Val Ser Ala Xaa<br>
15	10<br>
(2) INFORMATION FOR SEQ ID NO:12:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 14 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Bacillus thurigiensis<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:12:<br>
Met Asn Lys Asn Asn Thr Lys Leu Pro Thr Arg Ala Leu Pro<br>
15	10<br>
(2) INFORMATION FOR SEQ ID NO:13:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 15 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(v) FRAGMENT TYPE: N-terminal<br>
(vi) ORIGfNAL SOURCE:<br>
ORGANISM: Bacillus thuringiensis<br>
STRAIN: AB88<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..15<br>
(D) OTHER INFORMATION: /note= "N-terminal amino acid sequence of 35 kDa VIP active against Agrotis ipsilon"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:13:<br>
Ala Leu Ser Glu Asn Thr Gly Lys Asp Gly Gly Tyr lie Val Pro<br>
15	10	15<br>
(2) INFORMATION FOR SEQ ID NO:14:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 9 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Bacillus thuringiensis<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:14:<br>
Met Asp Asn Asn Pro Asn lie Asn Glu<br>
1	5<br>
(2) INFORMATION FOR SEQ ID NO:15:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 9 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(v) FRAGMENT TYPE: N-terminal<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..9<br>
(D) OTHER INFORMATION: /note= "N-terminal sequence of 80 kDa delta-endotoxin"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:15:<br>
Met Asp Asn Asn Pro Asn lie Asn Glu<br>
1	5<br>
(2) INFORMATION FOR SEQ ID NO:16:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 11 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(v) FRAGMENT TYPE: N-terminal<br>
(vi) ORIGINAL SOURCE:<br>
(A) ORGANISM: Bacillus thuringiensis<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: I..11<br>
(D) OTHER INFORMATION: /note= "N-terminal sequence from 60 kDa delta-endotoxin"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:16:<br>
Met Asn Val Leu Asn Ser Gly Arg Thr Thr lie<br>
15	10<br>
(2) INFORMATION FOR SEQ ID NO:17:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2655 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: 1..2652<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence for 100 kd VIPlA(a) protein from AB78"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:17:<br>
ATGAAGAACA TGAAGAAGAA GCTGGCCAGC GTGGTGACCT GCACCCTGCT GGCCCCCATG	60<br>
TTCCTGAACG GCAACGTGAA CGCCGTGTAC GCCGACAGCA AGACCAACCA GATCAGCACC	120<br>
ACCCAGAAGA ACCAGCAGAA GGAGATGGAC CGCAAGGGCC TGCTGGGCTA CTACTTCAAG	180<br>
GGCAAGGACT TCAGCAACCT GACCATGTTC GCCCCCACGC GTGACAGCAC CCTGATCTAC	240<br>
GACCAGCAGA CCGCCAACAA GCTGCTGGAC AAGAAGCAGC AGGAGTACCA GAGCATCCGC	300<br>
TGGATCGGCC TGATCCAGAG CAAGGAGACC GGCGACTTCA CCTTCAACCT GAGCGAGGAC	360<br>
GAGCAGGCCA TCATCGAGAT CAACGGCAAG ATCATCAGCA ACAAGGGCAA GGAGAAGCAG	420<br>
GTGGTGCACC TGGAGAAGGG CAAGCTGGTG CCCATCAAGA TCGAGTACCA GAGCGACACC	480<br>
AAGTTCAACA TCGACAGCAA GACCTTCAAG GAGCTGAAGC TTTTCAAGAT CGACAGCCAG	540<br>
AACCAGCCCC AGCAGGTGCA GCAGGACGAG CTGCGCAACC CCGAGTTCAA CAAGAAGGAG	600<br>
AGCCAGGAGT TCCTGGCCAA GCCCAGCAAG ATCAACCTGT TCACCCAGCA GATGAAGCGC	660<br>
GAGATCGACG AGGACACCGA CACCGACGGC GACAGCATCC CCGACCTGTG GGAGGAGAAC	720<br>
GGCTACACCA TCCAGAACCG CATCGCCGTG AAGTGGGACG ACAGCCTGGC TAGCAAGGGC	780<br>
TACACCAAGT TCGTGAGCAA CCCCCTGGAG AGCCACACCG TGGGCGACCC CTACACCGAC	840<br>
TACGAGAAGG CCGCCCGCGA CCTGGACCTG AGCAACGCCA AGGAGACCTT CAACCCCCTG	900<br>
GTGGCCGCCT TCCCCAGCGT GAACGTGAGC ATGGAGAAGG TGATCCTGAG CCCCAACGAG	960<br>
AACCTGAGCA ACAGCGTGGA GAGCCACTCG AGCACCAACT GGAGCTACAC CAACACCGAG	1020<br>
GGCGCCAGCG TGGAGGCCGG CATCGGTCCC AAGGGCATCA GCTTCGGCGT GAGCGTGAAC	1080<br>
TACCAGCACA GCGAGACCGT GGCCCAGGAG TGGGGCACCA GCACCGGCAA CACCAGCCAG	1140<br>
TTCAACACCG CCAGCGCCGG CTACCTGAAC GCCAACGTGC GCTACAACAA CGTGGGCACC	1200<br>
GGCGCCATCT ACGACGTGAA GCCCACCACC AGCTTCGTGC TGAACAACGA CACCATCGCC	1260<br>
ACCATCACCG CCAAGTCGAA TTCCACCGCC CTGAACATCA GCCCCGGCGA GAGCTACCCC	1320<br>
AAGAAGGGCC AGAACGGCAT CGCCATCACC AGCATGGACG ACTTCAACAG CCACCCCATC	1380<br>
ACCCTGAACA AGAAGCAGGT GGACAACCTG CTGAACAACA AGCCCATGAT GCTGGAGACC	1440<br>
AACCAGACCG ACGGCGTCTA CAAGATCAAG GACACCCACG GCAACATCGT GACCGGCGGC	1500<br>
GAGTGGAACG GCGTGATCCA GCAGATCAAG GCCAAGACCG CCAGCATCAT CGTCGACGAC	1560<br>
GGCGAGCGCG TGGCCGAGAA GCGCGTGGCC GCCAAGGACT ACGAGAACCC CGAGGACAAG	1620<br>
ACCCCCAGCC TGACCCTGAA GGACGCCCTG AAGCTGAGCT ACCCCGACGA GATCAAGGAG	1680<br>
ATCGAGGGCC TGCTGTACTA CAAGAACAAG CCCATCTACG AGAGCAGCGT GATGACCTAT	1740<br>
CTAGACGAGA ACACCGCCAA GGAGGTGACC AAGCAGCTGA ACGACACCAC CGGCAAGTTC	1800<br>
AAGGACGTGA GCCACCTGTA CGACGTGAAG CTGACCCCCA AGATGAACGT GACCATCAAG	1860<br>
CTGAGCATCC TGTACGACAA CGCCGAGAGC AACGACAACA GCATCGGCAA GTGGACCAAC	1920<br>
ACCAACATCG TGAGCGGCGG CAACAACGGC AAGAAGCAGT ACAGCAGCAA CAACCCCGAC	1980<br>
GCCAACCTGA CCCTGAACAC CGACGCCCAG GAGAAGCTGA ACAAGAACCG CGACTACTAC	2040<br>
ATCAGCCTGT ACATGAAGAG CGAGAAGAAC ACCCAGTGCG AGATCACCAT CGACGGCGAG	2100<br>
ATATACCCCA TCACCACCAA GACCGTGAAC GTGAACAAGG ACAACTACAA GCGCCTGGAC	2160<br>
ATCATCGCCC ACAACATCAA GAGCAACCCC ATCAGCAGCC TGCACATCAA GACCAACGAC	2220<br>
GAGATCACCC TGTTCTGGGA CGACATATCG ATTACCGACG TCGCCAGCAT CAAGCCCGAG	2280<br>
AACCTGACCG ACAGCGAGAT CAAGCAGATA TACAGTCGCT ACGGCATCAA GCTGGAGGAC	2340<br>
GGCATCCTGA TCGACAAGAA GGGCGGCATC CACTACGGCG AGTTCATCAA CGAGGCCAGC	2400<br>
TTCAACATCG AGCCCCTGCA GAACTACGTG ACCAAGTACG AGGTGACCTA CAGCAGCGAG	2460<br>
CTGGGCCCCA ACGTGAGCGA CACCCTGGAG AGCGACAAGA TTTACAAGGA CGGCACCATC	2520<br>
AAGTTCGACT TCACCAAGTA CAGCAAGAAC GAGCAGGGCC TGTTCTACGA CAGCGGCCTG	2580<br>
AACTGGGACT TCAAGATCAA CGCCATCACC TACGACGGCA AGGAGATGAA CGTGTTCCAC	2640<br>
CGCTACAACA AGTAG	2655<br>
(2) INFORMATION FOR SEQ ID NO:18:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2004 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic) (iii) HYPOTHETICAL: NO (iv) ANTI-SENSE: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: 1..2004<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence for VIPlA(a) 80 kd protein from AB78"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:18: ATGAAGCGCG AGATCGACGA GGACACCGAC ACCGACGGCG ACAGCATCCC CGACCTGTGG     60<br>
GAGGAGAACG GCTACACCAT CCAGAACCGC ATCGCCGTGA AGTGGGACGA CAGCCTGGCT	120<br>
AGCAAGGGCT ACACCAAGTT CGTGAGCAAC CCCCTGGAGA GCCACACCGT GGGCGACCCC	180<br>
TACACCGACT ACGAGAAGGC CGCCCGCGAC CTGGACCTGA GCAACGCCAA GGAGACCTTC	240<br>
AACCCCCTGG TGGCCGCCTT CCCCAGCGTG AACGTGAGCA TGGAGAAGGT GATCCTGAGC	300<br>
CCCAACGAGA ACCTGAGCAA CAGCGTGGAG AGCCACTCGA GCACCAACTG GAGCTACACC	360<br>
AACACCGAGG GCGCCAGCGT GGAGGCCGGC ATCGGTCCCA AGGGCATCAG CTTCGGCGTG	420<br>
AGCGTGAACT ACCAGCACAG CGAGACCGTG GCCCAGGAGT GGGGCACCAG CACCGGCAAC	480<br>
ACCAGCCAGT TCAACACCGC CAGCGCCGGC TACCTGAACG CCAACGTGCG CTACAACAAC	540<br>
GTGGGCACCG GCGCCATCTA CGACGTGAAG CCCACCACCA GCTTCGTGCT GAACAACGAC	600<br>
ACCATCGCCA CCATCACCGC CAAGTCGAAT TCCACCGCCC TGAACATCAG CCCCGGCGAG	660<br>
AGCTACCCCA AGAAGGGCCA GAACGGCATC GCCATCACCA GCATGGACGA CTTCAACAGC	720<br>
CACCCCATCA CCCTGAACAA GAAGCAGGTG GACAACCTGC TGAACAACAA GCCCATGATG	780<br>
CTGGAGACCA ACCAGACCGA CGGCGTCTAC AAGATCAAGG ACACCCACGG CAACATCGTG	840<br>
ACCGGCGGCG AGTGGAACGG CGTGATCCAG CAGATCAAGG CCAAGACCGC CAGCATCATC	900<br>
GTCGACGACG GCGAGCGCGT GGCCGAGAAG CGCGTGGCCG CCAAGGACTA CGAGAACCCC	960<br>
GAGGACAAGA CCCCCAGCCT GACCCTGAAG GACGCCCTGA AGCTGAGCTA CCCCGACGAG	1020<br>
ATCAAGGAGA TCGAGGGCCT GCTGTACTAC AAGAACAAGC CCATCTACGA GAGCAGCGTG	1080<br>
ATGACCTATC TAGACGAGAA CACCGCCAAG GAGGTGACCA AGCAGCTGAA CGACACCACC	1140<br>
GGCAAGTTCA AGGACGTGAG CCACCTGTAC GACGTGAAGC TGACCCCCAA GATGAACGTG	1200<br>
ACCATCAAGC TGAGCATCCT GTACGACAAC GCCGAGAGCA ACGACAACAG CATCGGCAAG	1260<br>
TGGACCAACA CCAACATCGT GAGCGGCGGC AACAACGGCA AGAAGCAGTA CAGCAGCAAC	1320<br>
AACCCCGACG CCAACCTGAC CCTGAACACC GACGCCCAGG AGAAGCTGAA CAAGAACCGC	1380<br>
GACTACTACA TCAGCCTGTA CATGAAGAGC GAGAAGAACA CCCAGTGCGA GATCACCATC	1440<br>
GACGGCGAGA TATACCCCAT CACCACCAAG ACCGTGAACG TGAACAAGGA CAACTACAAG	1500<br>
CGCCTGGACA TCATCGCCCA CAACATCAAG AGCAACCCCA TCAGCAGCCT GCACATCAAG	1560<br>
ACCAACGACG AGATCACCCT GTTCTGGGAC GACATATCGA TTACCGACGT CGCCAGCATC	1620<br>
AAGCCCGAGA ACCTGACCGA CAGCGAGATC AAGCAGATAT ACAGTCGCTA CGGCATCAAG	1680<br>
CTGGAGGACG GCATCCTGAT CGACAAGAAG GGCGGCATCC ACTACGGCGA GTTCATCAAC	1740<br>
3AGGCCAGCT TCAACATCGA GCCCCTGCAG AACTACGTGA CCAAGTACGA GGTGACCTAC	1800<br>
AGCAGCGAGC TGGGCCCCAA CGTGAGCGAC ACCCTGGAGA GCGACAAGAT TTACAAGGAC	1860<br>
GGCACCATCA AGTTCGACTT CACCAAGTAC AGCAAGAACG AGCAGGGCCT GTTCTACGAC	1920<br>
AGCGGCCTGA ACTGGGACTT CAAGATCAAC GCCATCACCT ACGACGGCAA GGAGATGAAC	1980<br>
GTGTTCCACC GCTACAACAA GTAG	2004<br>
(2) INFORMATION FOR SEQ ID NO:19:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 4074 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DMA (genomic)<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1..1386<br>
(D) OTHER INFORMATION: /product= "VIP2A(b) from Btt"<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1394..3895<br>
(D) OTHER INFORMATION: /product^ "VIPlA(b) from Btt"<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: 1..4074<br>
(D) OTHER INFORMATION: /note= "Cloned DNA sequence from Btt which contains the genes for both VIPlA(b) and VIP2A(b)"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:19:<br>
ATG CAA AGA ATG GAG GGA AAG TTG TTT GTG GTG TCA AAA ACA TTA CAA       48<br>
Met Gin Arg Met Glu Gly Lys Leu Phe Val Val Ser Lys Thr Leu Gin<br>
670	675	680<br>
GTA GTT ACT AGA ACT GTA TTG CTT AGT ACA GTT TAG TCT ATA ACT TTA       96<br>
Val Val Thr Arg Thr Val Leu Leu Ser Thr Val Tyr Ser He Thr Leu<br>
685	690	695<br>
TTA AAT AAT GTA GTG ATA AAA GCT GAC CAA TTA AAT ATA AAT TCT CAA      144<br>
Leu Asn Asn Val Val He Lys Ala Asp Gin Leu Asn He Asn Ser Gin<br>
700	705	710	715<br>
AGT AAA TAT ACT AAC TTG CAA AAT CTA AAA ATC CCT GAT AAT GCA GAG      192 Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys He Pro Asp Asn Ala Glu<br>
720	725	730<br>
GAT TTT AAA GAA GAT AAG GGG AAA GCG AAA GAA TGG GGG AAA GAG AAA      240<br>
Asp Phe Lys Glu Asp Lys Gly Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
735	740	745<br>
GGG GAA GAG TGG AGG CCT CCT GCT ACT GAG AAA GGA GAA ATG AAT AAT      288<br>
Gly Glu Glu Trp Arg Pro Pro Ala Thr Glu Lys Gly Glu Met Asn Asn<br>
750	755	760<br>
TTT TTA GAT AAT AAA AAT GAT ATA AAG ACC AAT TAT AAA GAA ATT ACT      336<br>
Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr<br>
765	770	775<br>
TTT TCT ATG GCA GGT TCA TGT GAA GAT GAA ATA AAA GAT TTA GAA GAA      384<br>
Phe Ser Met Ala Gly Ser Cys Glu Asp Glu lie Lys Asp Leu Glu Glu<br>
780	785	790	795<br>
ATT GAT AAG ATC TTT GAT AAA GCC AAT CTC TCG AGT TCT ATT ATC ACC      432<br>
lie Asp Lys lie Phe Asp Lys Ala Asn Leu Ser Ser Ser lie lie Thr<br>
800	805	810<br>
TAT AAA AAT GTG GAA CCA GCA ACA ATT GGA TTT AAT AAA TCT TTA ACA      480<br>
Tyr Lys Asn Val Glu Pro Ala Thr lie Gly Phe Asn Lys Ser Leu Thr<br>
815	820	825<br>
GAA GGT AAT ACG ATT AAT TCT GAT GCA ATG GCA CAG TTT AAA GAA CAA      528<br>
Glu Gly Asn Thr lie Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
830	835	840<br>
TTT TTA GGT AAG GAT ATG AAG TTT GAT AGT TAT CTA GAT ACT CAT TTA      576<br>
Phe Leu Gly Lys Asp Met Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
845	850	855<br>
ACT GCT CAA CAA GTT TCC AGT AAA AAA AGA GTT ATT TTG AAG GTT ACG      624<br>
Thr Ala Gin Gin Val Ser Ser Lys Lys Arg Val lie Leu Lys Val Thr<br>
860	865	870	875<br>
GTT CCG AGT GGG AAA GGT TCT ACT ACT CCA ACA AAA GCA GGT GTC ATT      672<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val lie<br>
880	885	890<br>
TTA AAC AAT AAT GAA TAG AAA ATG CTC ATT GAT AAT GGG TAT GTG CTC      720<br>
Leu Asn Asn Asn Glu Tyr Lys Met Leu lie Asp Asn Gly Tyr Val Leu<br>
895	900	905<br>
CAT GTA GAT AAG GTA TCA AAA GTA GTA AAA AAA GGG ATG GAG TGC TTA      768<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Met Glu Cys Leu<br>
910	915	920<br>
CAA GTT GAA GGG ACT TTA AAA AAG AGT CTC GAC TTT AAA AAT GAT ATA      816<br>
Gin Val Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp lie<br>
925	930	935<br>
AAT GCT GAA GCG CAT AGC TGG GGG ATG AAA ATT TAT GAA GAC TGG GCT      864<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys lie Tyr Glu Asp Trp Ala<br>
940	945	950	955<br>
AAA AAT TTA ACC GCT TCG CAA AGG GAA GCT TTA GAT GGG TAT GOT AGG      912<br>
Lys Asn Leu Thr Ala Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
960	965	970<br>
CAA GAT TAT AAA GAA ATC AAT AAT TAT TTG CGC AAT CAA GGC GGG AGT      960<br>
Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
975	980	985<br>
GGA AAT GAA AAG CTG GAT GCC CAA TTA AAA AAT ATT TCT GAT GCT TTA     1008<br>
Gly Asn Glu Lys Leu Asp Ala Gin Leu Lys Asn lie Ser Asp Ala Leu<br>
990	995	1000<br>
GGG AAG AAA CCC ATA CCA GAA AAT ATT ACC GTG TAT AGA TGG TGT GGC     1056<br>
Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly<br>
1005	1010	1015<br>
ATG CCG GAA TTT GGT TAT CAA ATT AGT GAT CCG TTA CCT TCT TTA AAA     1104<br>
Met Pro Glu Phe Gly Tyr Gin He Ser Asp Pro Leu Pro Ser Leu Lys<br>
1020	1025	1030	1035<br>
GAT TTT GAA GAA CAA TTT TTA AAT ACA ATT AAA GAA GAC AAA GGG TAT     1152<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr<br>
1040	1045	1050<br>
ATG AGT ACA AGC TTA TCG AGT GAA CGT CTT GCA GCT TTT GGA TCT AGA     1200 Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
1055	1060	1065<br>
AAA ATT ATA TTA CGC TTA CAA GTT CCG AAA GGA AGT ACG GGG GCG TAT     1248<br>
Lys He He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
1070	1075	1080<br>
TTA AGT GCC ATT GGT GGA TTT GCA AGT GAA AAA GAG ATC CTA CTT GAT     1296<br>
Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu Asp<br>
1085	1090	1095<br>
AAA GAT AGT AAA TAT CAT ATT GAT AAA GCA ACA GAG GTA ATC ATT AAA     1344<br>
Lys Asp Ser Lys Tyr His He Asp Lys Ala Thr Glu Val He He Lys<br>
1100	1105	1110	1115<br>
GGT GTT AAG CGA TAT GTA GTG GAT GCA ACA TTA TTA ACA AAT	1386<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
1120	1125<br>
TAAGGAG ATG AAA AAT ATG AAG AAA AAG TTA GCA AGT GTT GTA ACC TGT     1435<br>
Met Lys Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys<br>
15	10<br>
ATG TTA TTA GCT CCT ATG TTT TTG AAT GGA AAT GTG AAT GCT GTT AAC     1483<br>
Met Leu Leu Ala Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Asn<br>
15	20	25	30<br>
GCG GAT ACT AAA ATA AAT GAG ATT TCT ACA ACG GAG GAA AAC CAA GAG     1531<br>
Ala Asp Ser Lys lie Asn Gin lie Ser Thr Thr Gin Glu Asn Gin Gin<br>
35	40	45<br>
AAA GAG ATG GAC CGA AAG GGA TTA TTG GGA TAT TAT TTC AAA GGA AAA     1579<br>
Lys Glu Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys<br>
50	55	60<br>
GAT TTT AAT AAT CTT ACT ATG TTT GCA CCG ACA CGT GAT AAT ACC CTT     1627<br>
Asp Phe Asn Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Asn Thr Leu<br>
65	70	75<br>
ATG TAT GAC CAA CAA ACA GCG AAT GCA TTA TTA GAT AAA AAA CAA CAA     1675<br>
Met Tyr Asp Gin Gin Thr Ala Asn Ala Leu Leu Asp Lys Lys Gin Gin<br>
80	85	90<br>
GAA TAT CAG TCC ATT CGT TGG ATT GGT TTG ATT CAG CGT AAA GAA ACG     1723<br>
Glu Tyr Gin Ser lie Arg Trp lie Gly Leu lie Gin Arg Lys Glu Thr<br>
95	100	105	110<br>
GGC GAT TTC ACA TTT AAC TTA TCA AAG GAT GAA CAG GCA ATT ATA GAA     1771<br>
Gly Asp Phe Thr Phe Asn Leu Ser Lys Asp Glu Gin Ala lie lie Glu<br>
115	120	125<br>
ATC GAT GGG AAA ATC ATT TCT AAT AAA GGG AAA GAA AAG CAA GTT GTC     1819<br>
lie Asp Gly Lys lie lie Ser Asn Lys Gly Lys Glu Lys Gin Val Val<br>
130	135	140<br>
CAT TTA GAA AAA GAA AAA TTA GTT CCA ATC AAA ATA GAG TAT CAA TCA     1867<br>
His Leu Glu Lys Glu Lys Leu Val Pro lie Lys lie Glu Tyr Gin Ser<br>
145	150	155<br>
GAT ACG AAA TTT AAT ATT GAT AGT AAA ACA TTT AAA GAA CTT AAA TTA     1915<br>
Asp Thr Lys Phe Asn lie Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu<br>
160	165	170<br>
TTT AAA ATA GAT AGT CAA AAC CAA TCT CAA CAA GTT CAA CTG AGA AAC     1963<br>
Phe Lys lie Asp Ser Gin Asn Gin Ser Gin Gin Val Gin Leu Arg Asn<br>
175	180	185	190<br>
CCT GAA TTT AAC AAA AAA GAA TCA CAG GAA TTT TTA GCA AAA GCA TCA     2011<br>
Pro Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Ala Ser<br>
195	200	205<br>
AAA ACA AAC CTT TTT AAG CAA AAA ATG AAA AGA GAT ATT GAT GAA GAT     2059<br>
Lys Thr Asn Leu Phe Lys Gin Lys Met Lys Arg Asp lie Asp Glu Asp<br>
210	215	220<br>
ACG GAT ACA GAT GGA GAC TCC ATT CCT GAT CTT TGG GAA GAA AAT GGG     2107<br>
Thr Asp Thr Asp Gly Asp Ser lie Pro Asp Leu Trp Glu Glu Asn Gly<br>
225	230	235<br>
TAG ACG ATT CAA AAT AAA GTT GCT GTC AAA TGG GAT GAT TCG CTA GCA     2155<br>
Tyr Thr He Gin Asn Lys Val Ala Val Lys Trp Asp Asp Ser Leu Ala<br>
240	245	250<br>
ACT AAG GGA TAT ACA AAA TTT GTT TCG AAT CCA TTA GAC AGC CAC ACA     2203<br>
Ser Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Asp Ser His Thr<br>
255	260	265	270<br>
GTT GGC GAT CCC TAT ACT GAT TAT GAA AAG GCC GCA AGG GAT TTA GAT     2251<br>
Val Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu Asp<br>
275	280	285<br>
TTA TCA AAT GCA AAG GAA ACG TTC AAC CCA TTG GTA GCT GCT TTT CCA     2299<br>
Leu Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe Pro<br>
290	295	300<br>
AGT GTG AAT GTT AGT ATG GAA AAG GTG ATA TTA TCA CCA AAT GAA AAT     2347<br>
Ser Val Asn Val Ser Met Glu Lys Val lie Leu Ser Pro Asn Glu Asn<br>
305	310	315<br>
TTA TCC AAT AGT GTA GAG TCT CAT TCA TCC ACG AAT TGG TCT TAT ACG     2395<br>
Leu Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr Thr<br>
320	325	330<br>
AAT ACA GAA GGA GCT TCC ATT GAA GCT GGT GGC GGT CCA TTA GGC CTT     2443<br>
Asn Thr Glu Gly Ala Ser He Glu Ala Gly Gly Gly Pro Leu Gly Leu<br>
335	340	345	350<br>
TCT TTT GGC GTG AGT GTT ACT TAT CAA CAC TCT GAA ACA GTT GCA CAA     2491<br>
Ser Phe Gly Val Ser Val Thr Tyr Gin His Ser Glu Thr Val Ala Gin<br>
355	360	365<br>
GAA TGG GGA ACA TCT ACA GGA AAT ACT TCA CAA TTC AAT ACG GCT TCA     2539<br>
Glu Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala Ser<br>
370	375	380<br>
GCG GGA TAT TTA AAT GCA AAT GTT CGG TAT AAC AAT GTA GGG ACT GGT     2587<br>
Ala Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr Gly<br>
385	390	395<br>
GCC ATC TAT GAT GTA AAA CCT ACA ACA AGT TTT GTA TTA AAT AAC AAT     2635<br>
Ala lie Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn Asn<br>
400	405	410<br>
ACC ATC GCA ACG ATT ACA GCA AAA TCA AAT TCA ACA GCT TTA CGT ATA     2683<br>
Thr lie Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Arg He<br>
415	420	425	430<br>
TCT CCG GGG GAT AGT TAT CCA GAA ATA GGA GAA AAC GCT ATT GCG ATT     2731<br>
Ser Pro Gly Asp Ser Tyr Pro Glu He Gly Glu Asn Ala He Ala He<br>
435	440	445<br>
ACA TCT ATG GAT GAT TTT AAT TCT CAT CCA ATT ACA TTA AAT AAA CAA     2779<br>
Thr Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys Gin<br>
450	455	460<br>
CAG GTA AAT CAA TTG ATA AAT AAT AAG CCA ATT ATG CTA GAG ACA GAC     2827 Gin Val Asn Gin Leu He Asn Asn Lys Pro He Met Leu Glu Thr Asp<br>
465	470	475<br>
CAA ACA GAT GGT GTT TAT AAA ATA AGA GAT ACA CAT GGA AAT ATT GTA     2875<br>
Gin Thr Asp Gly Val Tyr Lys He Arg Asp Thr His Gly Asn He Val<br>
480	485	490<br>
ACT GGT GGA GAA TGG AAT GGT GTA ACA CAA CAA ATT AAA GCA AAA ACA     2923<br>
Thr Gly Gly Glu Trp Asn Gly Val Thr Gin Gin He Lys Ala Lys Thr<br>
495	500	505	510<br>
GCG TCT ATT ATT GTG GAT GAC GGG AAA CAG GTA GCA GAA AAA CGT GTG     2971<br>
Ala Ser He He Val Asp Asp Gly Lys Gin Val Ala Glu Lys Arg Val<br>
515	520	525<br>
GCG GCA AAA GAT TAT GGT CAT CCA GAA GAT AAA ACA CCA CCT TTA ACT     3019<br>
Ala Ala Lys Asp Tyr Gly His Pro Glu Asp Lys Thr Pro Pro Leu Thr<br>
530	535	540<br>
TTA AAA GAT ACC CTG AAG CTT TCA TAG CCA GAT GAA ATA AAA GAA ACT     3067<br>
Leu Lys Asp Thr Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu Thr<br>
545	550	555<br>
AAT GGA TTG TTG TAC TAT GAT GAC AAA CCA ATC TAT GAA TCG ACT GTC     3115<br>
Asn Gly Leu Leu Tyr Tyr Asp Asp Lys Pro He Tyr Glu Ser Ser Val<br>
560	565	570<br>
ATG ACT TAT CTG GAT GAA AAT ACG GCA AAA GAA GTC AAA AAA CAA ATA     3163<br>
Met Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Lys Lys Gin He<br>
575	580	585	590<br>
AAT GAT ACA ACC GGA AAA TTT AAG GAT GTA AAT CAC TTA TAT GAT GTA     3211<br>
Asn Asp Thr Thr Gly Lys Phe Lys Asp Val Asn His Leu Tyr Asp Val<br>
595	600	605<br>
AAA CTG ACT CCA AAA ATG AAT TTT ACG ATT AAA ATG GCT TCC TTG TAT     3259<br>
Lys Leu Thr Pro Lys Met Asn Phe Thr He Lys Met Ala Ser Leu Tyr<br>
610	615	620<br>
GAT GGG GCT GAA AAT AAT CAT AAC TCT TTA GGA ACC TGG TAT TTA ACA     3307<br>
Asp Gly Ala Glu Asn Asn His Asn Ser Leu Gly Thr Trp Tyr Leu Thr<br>
625	630	635<br>
TAT AAT GTT GCT GGT GGA AAT ACT GGG AAG AGA CAA TAT CGT TCA GCT     3355<br>
Tyr Asn Val Ala Gly Gly Asn Thr Gly Lys Arg Gin Tyr Arg Ser Ala<br>
640	645	650<br>
CAT TCT TGT GCA CAT GTA GCT CTA TCT TCA GAA GCG AAA AAG AAA CTA     3403<br>
His Ser Cys Ala His Val Ala Leu Ser Ser Glu Ala Lys Lys Lys Leu<br>
655	660	665	670<br>
AAT CAA AAT GCG AAT TAC TAT CTT AGC ATG TAT ATG AAG GCT GAT TCT     3451<br>
Asn Gin Asn Ala Asn Tyr Tyr Leu Ser Met Tyr Met Lys Ala Asp Ser<br>
675	680	685<br>
ACT ACG GAA CCT ACA ATA GAA GTA GCT GGG GAA AAA TCT GCA ATA ACA     3499<br>
Thr Thr Glu Pro Thr He Glu Val Ala Gly Glu Lys Ser Ala He Thr<br>
690	695	700<br>
ACT AAA AAA GTA AAA TTA AAT AAT CAA AAT TAT CAA AGA GTT GAT ATT     3547<br>
Ser Lys Lys Val Lys Leu Asn Asn Gin Asn Tyr Gin Arg Val Asp He<br>
705	710	715<br>
TTA GTG AAA AAT TCT GAA AGA AAT CCA ATG GAT AAA ATA TAT ATA AGA     3595<br>
Leu Val Lys Asn Ser Glu Arg Asn Pro Met Asp Lys He Tyr He Arg<br>
720	725	730<br>
GGA AAT GGC ACG ACA AAT GTT TAT GGG GAT GAT GTT ACT ATC CCA GAG     3643<br>
Gly Asn Gly Thr Thr Asn Val Tyr Gly Asp Asp Val Thr He Pro Glu<br>
735	740	745	750<br>
GTA TCA GCT ATA AAT CCG GCT ACT CTA TCA GAT GAA GAA ATT CAA GAA     3691<br>
Val Ser Ala He Asn Pro Ala Ser Leu Ser Asp Glu Glu He Gin Glu<br>
755	760	765<br>
ATA TTT AAA GAC TCA ACT ATT GAA TAT GGA AAT CCT AGT TTC GTT GCT     3739 He Phe Lys Asp Ser Thr He Glu Tyr Gly Asn Pro Ser Phe Val Ala<br>
770	775	780<br>
GAT GCC GTA ACA TTT AAA AAT ATA AAA CCT TTA CAA AAT TAT GTA AAG     3787<br>
Asp Ala Val Thr Phe Lys Asn He Lys Pro Leu Gin Asn Tyr Val Lys<br>
785	790	795<br>
GAA TAT GAA ATA TAT CAT AAA TCT CAT CGA TAT GAA AAG AAA ACG GTC     3835<br>
Glu Tyr Glu He Tyr His Lys Ser His Arg Tyr Glu Lys Lys Thr Val<br>
800	805	810<br>
TTT GAT ATC ATG GGT GTT CAT TAT GAG TAT AGT ATA GCT AGG GAA CAA     3883<br>
Phe Asp He Met Gly Val His Tyr Glu Tyr Ser He Ala Arg Glu Gin<br>
815	820	825	830<br>
AAG AAA GCC GCA TAATTTTAAA AATAAAACTC GTTAGAGTTT ATTTAGCATG        3935 Lys Lys Ala Ala<br>
GTATTTTTAA GAATAATCAA TATGTTGAAC CGTTTGTAGC TGTTTTGGAA GGGAATTTCA 3995<br>
TTTTATTTGG TCTCTTAAGT TGATGGGCAT GGGATATGTT CAGCATCCAA GCGTTTNGGG 4055<br>
GGTTANAAAA TCCAATTTT	4074<br>
(2) INFORMATION FOR SEQ ID NO:20:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 462 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID N0:20:<br>
Met Gin Arg Met Glu Gly Lys Leu Phe Val Val Ser Lys Thr Leu Gin<br>
15	10	15<br>
Val Val Thr Arg Thr Val Leu Leu Ser Thr Val Tyr Ser lie Thr Leu<br>
20	25	30<br>
Leu Asn Asn Val Val lie Lys Ala Asp Gin Leu Asn lie Asn Ser Gin<br>
35	40	45<br>
Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys lie Pro Asp Asn Ala Glu<br>
50	55	60<br>
Asp Phe Lys Glu Asp Lys Gly Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
65	70	75	80<br>
Gly Glu Glu Trp Arg Pro Pro Ala Thr Glu Lys Gly Glu Met Asn Asn<br>
85	90	95<br>
Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr<br>
100	105	110<br>
Phe Ser Met Ala Gly Ser Cys Glu Asp Glu lie Lys Asp Leu Glu Glu<br>
115	120	125<br>
He Asp Lys He Phe Asp Lys Ala Asn Leu Ser Ser Ser He He Thr<br>
130	135	140<br>
Tyr Lys Asn Val Glu Pro Ala Thr He Gly Phe Asn Lys Ser Leu Thr<br>
145	150	155	160<br>
Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
165	170	175<br>
Phe Leu Gly Lys Asp Met Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
180	185	190<br>
Thr Ala Gin Gin Val Ser Ser Lys Lys Arg Val He Leu Lys Val Thr<br>
195	200	205<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He<br>
210	215	220<br>
Leu Asn Asn Asn Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Val Leu<br>
225	230	235	240<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Met Glu Cys Leu<br>
245	250	255<br>
Gin Val Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He<br>
260	265	270<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys He Tyr Glu Asp Trp Ala<br>
275	280	285<br>
Lys Asn Leu Thr Ala Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
290	295	300<br>
Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
305	310	315	320<br>
Gly Asn Glu Lys Leu Asp Ala Gin Leu Lys Asn lie Ser Asp Ala Leu<br>
325	330	335<br>
Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly<br>
340	345	350<br>
Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys<br>
355	360	365<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp Lys Gly Tyr<br>
370	375	380<br>
Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
385	390	395	400<br>
Lys lie lie Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
405	410	415<br>
Leu Ser Ala lie Gly Gly Phe Ala Ser Glu Lys Glu lie Leu Leu Asp<br>
420	425	430<br>
Lys Asp Ser Lys Tyr His lie Asp Lys Ala Thr Glu Val lie lie Lys<br>
435	440	445<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
450	455	460<br>
(2) INFORMATION FOR SEQ ID NO:21:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 834 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:21:<br>
Met Lys Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys Met Leu<br>
15	10	15<br>
Leu Ala Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Asn Ala Asp<br>
20	25	30<br>
Ser Lys lie Asn Gin lie Ser Thr Thr Gin Glu Asn Gin Gin Lys Glu<br>
35	40	45<br>
Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe<br>
50	55	60<br>
Asn Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Asn Thr Leu Met Tyr<br>
65	70	75	80<br>
Asp Gin Gin Thr Ala Asn Ala Leu Leu Asp Lys Lys Gin Gin Glu Tyr<br>
85	90	95<br>
Gin Ser lie Arg Trp lie Gly Leu lie Gin Arg Lys Glu Thr Gly Asp<br>
100	105	110<br>
Phe Thr Phe Asn Leu Ser Lys Asp Glu Gin Ala lie lie Glu lie Asp<br>
115	120	125<br>
Gly Lys He He Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu<br>
130	135	140<br>
Glu Lys Glu Lys Leu Val Pro He Lys He Glu Tyr Gin Ser Asp Thr<br>
145	150	155	160<br>
Lys Phe Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys<br>
165	170	175<br>
He Asp Ser Gin Asn Gin Ser Gin Gin Val Gin Leu Arg Asn Pro Glu<br>
180	185	190<br>
Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Ala Ser Lys Thr<br>
195	200	205<br>
Asn Leu Phe Lys Gin Lys Met Lys Arg Asp He Asp Glu Asp Thr Asp<br>
210	215	220<br>
Thr Asp Gly Asp Ser He Pro Asp Leu Trp Glu Glu Asn Gly Tyr Thr<br>
225	230	235	240<br>
He Gin Asn Lys Val Ala Val Lys Trp Asp Asp Ser Leu Ala Ser Lys<br>
245	250	255<br>
Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Asp Ser His Thr Val Gly<br>
260	265	270<br>
Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu Asp Leu Ser<br>
275	280	285<br>
Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe Pro Ser Val<br>
290	295	300<br>
Asn Val Ser Met Glu Lys Val He Leu Ser Pro Asn Glu Asn Leu Ser<br>
305	310	315	320<br>
Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr Thr Asn Thr<br>
325	330	335<br>
Glu Gly Ala Ser He Glu Ala Gly Gly Gly Pro Leu Gly Leu Ser Phe<br>
340<br>
345<br>
350<br>
Gly Val Ser Val Thr Tyr Gin His Ser Glu Thr Val Ala Gin Glu Trp<br>
355	360	365<br>
Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala Ser Ala Gly<br>
370	375	380<br>
Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr Gly Ala lie<br>
385	390	395	400<br>
Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn Asn Thr lie<br>
405	410	415<br>
Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Arg He Ser Pro<br>
420	425	430<br>
Gly Asp Ser Tyr Pro Glu He Gly Glu Asn Ala He Ala He Thr Ser<br>
435	440	445<br>
Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys Gin Gin Val<br>
450	455	460<br>
Asn Gin Leu He Asn Asn Lys Pro He Met Leu Glu Thr Asp Gin Thr<br>
465	470	475	480<br>
Asp Gly Val Tyr Lys He Arg Asp Thr His Gly Asn He Val Thr Gly<br>
485	490	495<br>
Gly Glu Trp Asn Gly Val Thr Gin Gin He Lys Ala Lys Thr Ala Ser<br>
500	505	510<br>
He He Val Asp Asp Gly Lys Gin Val Ala Glu Lys Arg Val Ala Ala<br>
515	520	525<br>
Lys Asp Tyr Gly His Pro Glu Asp Lys Thr Pro Pro Leu Thr Leu Lys<br>
530	535	540<br>
Asp Thr Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu Thr Asn Gly<br>
545	550	555	560<br>
Leu Leu Tyr Tyr Asp Asp Lys Pro He Tyr Glu Ser Ser Val Met Thr<br>
565	570	575<br>
Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Lys Lys Gin He Asn Asp<br>
580	585	590<br>
Thr Thr Gly Lys Phe Lys Asp Val Asn His Leu Tyr Asp Val Lys Leu<br>
595	600	605<br>
Thr Pro Lys Met Asn Phe Thr He Lys Met Ala Ser Leu Tyr Asp Gly<br>
610	615	620<br>
Ala Glu Asn Asn His Asn Ser Leu Gly Thr Trp Tyr Leu Thr Tyr Asn<br>
625	630	635	640<br>
Val Ala Gly Gly Asn Thr Gly Lys Arg Gin Tyr Arg Ser Ala His Ser<br>
645	650	655<br>
Cys Ala His Val Ala Leu Ser Ser Glu Ala Lys Lys Lys Leu Asn Gin<br>
660	665	670<br>
Asn Ala Asn Tyr Tyr Leu Ser Met Tyr Met Lys Ala Asp Ser Thr Thr<br>
675	680	685<br>
Glu Pro Thr lie Glu Val Ala Gly Glu Lys Ser Ala He Thr Ser Lys<br>
690	695	700<br>
Lys Val Lys Leu Asn Asn Gin Asn Tyr Gin Arg Val Asp He Leu Val<br>
705	710	715	720<br>
Lys Asn Ser Glu Arg Asn Pro Met Asp Lys He Tyr He Arg Gly Asn<br>
725	730	735<br>
Gly Thr Thr Asn Val Tyr Gly Asp Asp Val Thr He Pro Glu Val Ser<br>
740	745	750<br>
Ala He Asn Pro Ala Ser Leu Ser Asp Glu Glu He Gin Glu He Phe<br>
755	760	765<br>
Lys Asp Ser Thr He Glu Tyr Gly Asn Pro Ser Phe Val Ala Asp Ala<br>
770	775	780<br>
Val Thr Phe Lys Asn He Lys Pro Leu Gin Asn Tyr Val Lys Glu Tyr<br>
785	790	795	800<br>
Glu He Tyr His Lys Ser His Arg Tyr Glu Lys Lys Thr Val Phe Asp<br>
805	810	815<br>
He Met Gly Val His Tyr Glu Tyr Ser He Ala Arg Glu Gin Lys Lys<br>
820	825	830<br>
Ala Ala<br>
(2) INFORMATION FOR SEQ ID NO:22:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 4041 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 1..4038<br>
(D) OTHER INFORMATION: /product= "VIPlA(a)/VIP2A(a) fusion<br>
product"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:22:<br>
ATG AAA AGA ATG GAG GGA AAG TTG TTT ATG GTG TCA AAA AAA TTA CAA       48<br>
Met Lys Arg Met Glu Gly Lys Leu Phe Met Val Ser Lys Lys Leu Gin<br>
835	840	845	850<br>
GTA GTT ACT AAA ACT GTA TTG CTT ACT ACA GTT TTC TCT ATA TCT TTA       96<br>
Val Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser lie Ser Leu<br>
855	860	865<br>
TTA AAT AAT GAA GTG ATA AAA GCT GAA CAA TTA AAT ATA AAT TCT CAA      144<br>
Leu Asn Asn Glu Val lie Lys Ala Glu Gin Leu Asn lie Asn Ser Gin<br>
870	875	880<br>
AGT AAA TAT ACT AAC TTG CAA AAT CTA AAA ATC ACT GAG AAG GTA GAG      192<br>
Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu<br>
885	890	895<br>
GAT TTT AAA GAA GAT AAG GAA AAA GCG AAA GAA TGG GGG AAA GAA AAA      240<br>
Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
900	905	910<br>
GAA AAA GAG TGG AAA CTA ACT GCT ACT GAA AAA GGA AAA ATG AAT AAT      288<br>
Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn<br>
915	920	925	930<br>
TTT TTA GAT AAT AAA AAT GAT ATA AAG ACA AAT TAT AAA GAA ATT ACT      336<br>
Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr<br>
935	940	945<br>
TTT TCT ATG GCA GGC TCA TTT GAA GAT GAA ATA AAA GAT TTA AAA GAA      384<br>
Phe Ser Met Ala Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu<br>
950	955	960<br>
ATT GAT AAG ATG TTT GAT AAA ACC AAT CTA TCA AAT TCT ATT ATC ACC      432<br>
lie Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser lie lie Thr<br>
965	970	975<br>
TAT AAA AAT GTG GAA CCG ACA ACA ATT GGA TTT AAT AAA TCT TTA ACA      480<br>
Tyr Lys Asn Val Glu Pro Thr Thr He Gly Phe Asn Lys Ser Leu Thr<br>
980	985	990<br>
GAA GGT AAT ACG ATT AAT TCT GAT GCA ATG GCA CAG TTT AAA GAA CAA      528<br>
Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
995	1000	1005	1010<br>
TTT TTA GAT AGG GAT ATT AAG TTT GAT AGT TAT CTA GAT ACG CAT TTA      576<br>
Phe Leu Asp Arg Asp He Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
1015	1020	1025<br>
ACT GCT CAA CAA GTT TCC AGT AAA GAA AGA GTT ATT TTG AAG GTT ACG      624 Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr<br>
1030	1035	1040<br>
GTT CCG ACT GGG AAA GGT TCT ACT ACT CCA ACA AAA GCA GGT GTC ATT      672<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val lie<br>
1045	1050	1055<br>
TTA AAT AAT AGT GAA TAG AAA ATG CTC ATT GAT AAT GGG TAT ATG GTC      720<br>
Leu Asn Asn Ser Glu Tyr Lys Met Leu lie Asp Asn Gly Tyr Met Val<br>
1060	1065	1070<br>
CAT GTA GAT AAG GTA TCA AAA GTG GTG AAA AAA GGG GTG GAG TGC TTA      768<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu<br>
1075	1080	1085	1090<br>
CAA ATT GAA GGG ACT TTA AAA AAG AGT CTT GAC TTT AAA AAT GAT ATA      816<br>
Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp lie<br>
1095	1100	1105<br>
AAT GCT GAA GCG CAT AGC TGG GGT ATG AAG AAT TAT GAA GAG TGG GCT      864<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala<br>
1110	1115	1120<br>
AAA GAT TTA ACC GAT TCG CAA AGG GAA GCT TTA GAT GGG TAT GCT AGG      912<br>
Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
1125	1130	1135<br>
CAA GAT TAT AAA GAA ATC AAT AAT TAT TTA AGA AAT CAA GGC GGA AGT      960<br>
Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
1140	1145	1150<br>
GGA AAT GAA AAA CTA GAT GCT CAA ATA AAA AAT ATT TCT GAT GCT TTA â¢   1008<br>
Gly Asn Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala Leu<br>
1155	1160	1165	1170<br>
GGG AAG AAA CCA ATA CCG GAA AAT ATT ACT GTG TAT AGA TGG TGT GGC     1056<br>
Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly<br>
1175	1180	1185<br>
ATG CCG GAA TTT GGT TAT CAA ATT AGT GAT CCG TTA CCT TCT TTA AAA     1104<br>
Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys<br>
1190	1195	1200<br>
GAT TTT GAA GAA CAA TTT TTA AAT ACA ATC AAA GAA GAC AAA GGA TAT     1152<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr<br>
1205	1210	1215<br>
ATG AGT ACA AGC TTA TCG AGT GAA CGT CTT GCA GCT TTT GGA TCT AGA     1200<br>
Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
1220	1225	1230<br>
AAA ATT ATA TTA CGA TTA CAA GTT CCG AAA GGA AGT ACG GGT GCG TAT     1248<br>
Lys He He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
1235	1240	1245	1250<br>
TTA AGT GCC ATT GGT GGA TTT GCA AGT GAA AAA GAG ATC CTA CTT GAT     1296<br>
Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu lie Leu Leu Asp<br>
1255	1260	1265<br>
AAA GAT ACT AAA TAT CAT ATT GAT AAA GTA ACA GAG GTA ATT ATT AAA     1344<br>
Lys Asp Ser Lys Tyr His lie Asp Lys Val Thr Glu Val lie lie Lys<br>
1270	1275	1280<br>
GGT GTT AAG CGA TAT GTA GTG GAT GCA ACA TTA TTA ACA AAT ATG AAA     1392<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn Met Lys<br>
1285	1290	1295<br>
AAT ATG AAG AAA AAG TTA GCA AGT GTT GTA ACG TGT ACG TTA TTA GCT     1440<br>
Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys Thr Leu Leu Ala<br>
1300	1305	1310<br>
CCT ATG TTT TTG AAT GGA AAT GTG AAT GCT GTT TAG GCA GAC AGC AAA     1488<br>
Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Tyr Ala Asp Ser Lys<br>
1315	1320	1325	1330<br>
ACA AAT CAA ATT TCT ACA ACA CAG AAA AAT CAA CAG AAA GAG ATG GAC     1536<br>
Thr Asn Gin He Ser Thr Thr Gin Lys Asn Gin Gin Lys Glu Met Asp<br>
1335	1340	1345<br>
CGA AAA GGA TTA CTT GGG TAT TAT TTC AAA GGA AAA GAT TTT AGT AAT     1584<br>
Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe Ser Asn<br>
1350	1355	1360<br>
CTT ACT ATG TTT GCA CCG ACA CGT GAT AGT ACT CTT ATT TAT GAT CAA     1632<br>
Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu lie Tyr Asp Gin<br>
1365	1370	1375<br>
CAA ACA GCA AAT AAA CTA TTA GAT AAA AAA CAA CAA GAA TAT CAG TCT     1680<br>
Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin Glu Tyr Gin Ser<br>
1380	1385	1390<br>
ATT CGT TGG ATT GGT TTG ATT CAG AGT AAA GAA ACG GGA GAT TTC ACA     1728<br>
He Arg Trp He Gly Leu He Gin Ser Lys Glu Thr Gly Asp Phe Thr<br>
1395	1400	1405	1410<br>
TTT AAC TTA TCT GAG GAT GAA CAG GCA ATT ATA GAA ATC AAT GGG AAA     1776<br>
Phe Asn Leu Ser Glu Asp Glu Gin Ala He He Glu He Asn Gly Lys<br>
1415	1420	1425<br>
ATT ATT TCT AAT AAA GGG AAA GAA AAG CAA GTT GTC CAT TTA GAA AAA     1824<br>
He He Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu Glu Lys<br>
1430	1435	1440<br>
GGA AAA TTA GTT CCA ATC AAA ATA GAG TAT CAA TCA GAT ACA AAA TTT     1872<br>
Gly Lys Leu Val Pro He Lys He Glu Tyr Gin Ser Asp Thr Lys Phe<br>
1445	1450	1455<br>
AAT ATT GAC AGT AAA ACA TTT AAA GAA CTT AAA TTA TTT AAA ATA GAT     1920<br>
Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys He Asp<br>
1460	1465	1470<br>
ACT CAA AAC CAA CCC CAG CAA GTC GAG CAA GAT GAA CTG AGA AAT CCT     1968<br>
Ser Gin Asn Gin Pro Gin Gin Val Gin Gin Asp Glu Leu Arg Asn Pro<br>
1475	1480	1485	1490<br>
GAA TTT AAC AAG AAA GAA TCA CAG GAA TTC TTA GCG AAA CCA TCG AAA     2016<br>
Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Pro Ser Lys<br>
1495	1500	1505<br>
ATA AAT CTT TTC ACT CAA AAA ATG AAA AGG GAA ATT GAT GAA GAC ACG     2064<br>
He Asn Leu Phe Thr Gin Lys Met Lys Arg Glu He Asp Glu Asp Thr<br>
1510	1515	1520<br>
GAT ACG GAT GGG GAC TCT ATT CCT GAC CTT TGG GAA GAA AAT GGG TAT     2112<br>
Asp Thr Asp Gly Asp Ser He Pro Asp Leu Trp Glu Glu Asn Gly Tyr<br>
1525	1530	1535<br>
ACG ATT CAA AAT AGA ATC GCT GTA AAG TGG GAC GAT TCT CTA GCA AGT     2160<br>
Thr lie Gin Asn Arg He Ala Val Lys Trp Asp Asp Ser Leu Ala Ser<br>
1540	1545	1550<br>
AAA GGG TAT ACG AAA TTT GTT TCA AAT CCA CTA GAA AGT CAC ACA GTT     2208<br>
Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu Ser His Thr Val<br>
1555	1560	1565	1570<br>
GGT GAT CCT TAT ACA GAT TAT GAA AAG GCA GCA AGA GAT CTA GAT TTG     2256<br>
Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu Asp Leu<br>
1575	1580	1585<br>
TCA AAT GCA AAG GAA ACG TTT AAC CCA TTG GTA GCT GCT TTT CCA AGT     2304<br>
Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe Pro Ser<br>
1590	1595	1600<br>
GTG AAT GTT AGT ATG GAA AAG GTG ATA TTA TCA CCA AAT GAA AAT TTA     2352<br>
Val Asn Val Ser Met Glu Lys Val He Leu Ser Pro Asn Glu Asn Leu<br>
1605	1610	1615<br>
TCC AAT AGT GTA GAG TCT CAT TCA TCC ACG AAT TGG TCT TAT ACA AAT     2400<br>
Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr Thr Asn<br>
1620	1625	1630<br>
ACA GAA GGT GCT TCT GTT GAA GCG GGG ATT GGA CCA AAA GGT ATT TCG     2448<br>
Thr Glu Gly Ala Ser Val Glu Ala Gly He Gly Pro Lys Gly He Ser<br>
1635	1640	1645	1650<br>
TTC GGA GTT AGC GTA AAC TAT CAA CAC TCT GAA ACA GTT GCA CAA GAA     2496<br>
Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr Val Ala Gin Glu<br>
1655	1660	1665<br>
TGG GGA ACA TCT ACA GGA AAT ACT TCG CAA TTC AAT ACG GCT TCA GCG     2544<br>
Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala Ser Ala<br>
1670	1675	1680<br>
GGA TAT TTA AAT GCA AAT GTT CGA TAT AAC AAT GTA GGA ACT GGT GCC     2592<br>
Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr Gly Ala<br>
1685	1690	1695<br>
ATC TAG GAT GTA AAA CCT ACA ACA ACT TTT GTA TTA AAT AAC GAT ACT     2640<br>
lie Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn Asp Thr<br>
1700	1705	1710<br>
ATC GCA ACT ATT ACG GCG AAA TCT AAT TCT ACA GCC TTA AAT ATA TCT     2688<br>
lie Ala Thr lie Thr Ala Lys Ser Asn Ser Thr Ala Leu Asn lie Ser<br>
1715	1720	1725	1730<br>
CCT GGA GAA ACT TAG CCG AAA AAA GGA CAA AAT GGA ATC GCA ATA ACA     2736<br>
Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly lie Ala lie Thr<br>
1735	1740	1745<br>
TCA ATG GAT GAT TTT AAT TCC CAT CCG ATT ACA TTA AAT AAA AAA CAA     2784<br>
Ser Met Asp Asp Phe Asn Ser His Pro lie Thr Leu Asn Lys Lys Gin<br>
1750	1755	1760<br>
GTA GAT AAT CTG CTA AAT AAT AAA CCT ATG ATG TTG GAA ACA AAC CAA     2832<br>
Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu Glu Thr Asn Gin<br>
1765	1770	1775<br>
ACA GAT GGT GTT TAT AAG ATA AAA GAT ACA CAT GGA AAT ATA GTA ACT     2880<br>
Thr Asp Gly Val Tyr Lys lie Lys Asp Thr His Gly Asn lie Val Thr<br>
1780	1785	1790<br>
GGC GGA GAA TGG AAT GGT GTC ATA CAA CAA ATC AAG GCT AAA ACA GCG     2928<br>
Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys Ala Lys Thr Ala<br>
1795	1800	1805	1810<br>
TCT ATT ATT GTG GAT GAT GGG GAA CGT GTA GCA GAA AAA CGT GTA GCG     2976<br>
Ser He He Val Asp Asp Gly Glu Arg Val Ala Glu Lys Arg Val Ala<br>
1815	1820	1825<br>
GCA AAA GAT TAT GAA AAT CCA GAA GAT AAA ACA CCG TCT TTA ACT TTA     3024<br>
Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro Ser Leu Thr Leu<br>
1830	1835	1840<br>
AAA GAT GCC CTG AAG CTT TCA TAT CCA GAT GAA ATA AAA GAA ATA GAG     3072<br>
Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu He Glu<br>
1845	1850	1855<br>
GGA TTA TTA TAT TAT AAA AAC AAA CCG ATA TAG GAA TCG AGC GTT ATG     3120<br>
Gly Leu Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu Ser Ser Val Met<br>
1860	1865	1870<br>
ACT TAG TTA GAT GAA AAT ACA GCA AAA GAA GTG ACC AAA CAA TTA AAT     3168<br>
Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr Lys Gin Leu Asn<br>
1875	1880	1885	1890<br>
GAT ACC ACT GGG AAA TTT AAA GAT GTA ACT CAT TTA TAT GAT GTA AAA     3216<br>
Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu Tyr Asp Val Lys<br>
1895	1900	1905<br>
CTG ACT CCA AAA ATG AAT GTT ACA ATC AAA TTG TCT ATA CTT TAT GAT     3264 Leu Thr Pro Lys Met Asn Val Thr He Lys Leu Ser He Leu Tyr Asp<br>
1910	1915	1920<br>
AAT GCT GAG TCT AAT GAT AAC TCA ATT GGT AAA TGG ACA AAC ACA AAT     3312<br>
Asn Ala Glu Ser Asn Asp Asn Ser lie Gly Lys Trp Thr Asn Thr Asn<br>
1925	1930	1935<br>
ATT GTT TCA GGT GGA AAT AAC GGA AAA AAA CAA TAT TCT TCT AAT AAT     3360<br>
lie Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr Ser Ser Asn Asn<br>
1940	1945	1950<br>
CCG GAT GCT AAT TTG ACA TTA AAT ACA GAT GCT CAA GAA AAA TTA AAT     3408<br>
Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin Glu Lys Leu Asn<br>
1955	1960	1965	1970<br>
AAA AAT CGT GAC TAT TAT ATA AGT TTA TAT ATG AAG TCA GAA AAA AAC     3456<br>
Lys Asn Arg Asp Tyr Tyr lie Ser Leu Tyr Met Lys Ser Glu Lys Asn<br>
1975	1980	1985<br>
ACA CAA TGT GAG ATT ACT ATA GAT GGG GAG ATT TAT CCG ATC ACT ACA     3504<br>
Thr Gin Cys Glu He Thr He Asp Gly Glu He Tyr Pro He Thr Thr<br>
1990	1995	2000<br>
AAA ACA GTG AAT GTG AAT AAA GAC AAT TAG AAA AGA TTA GAT ATT ATA     3552<br>
Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg Leu Asp He He<br>
2005	2010	2015<br>
GCT CAT AAT ATA AAA AGT AAT CCA ATT TCT TCA CTT CAT ATT AAA ACG     3600<br>
Ala His Asn He Lys Ser Asn Pro He Ser Ser Leu His He Lys Thr<br>
2020	2025	2030<br>
AAT GAT GAA ATA ACT TTA TTT TGG GAT GAT ATT TCT ATA ACA GAT GTA     3648<br>
Asn Asp Glu He Thr Leu Phe Trp Asp Asp He Ser He Thr Asp Val<br>
2035	2040	2045	2050<br>
GCA TCA ATA AAA CCG GAA AAT TTA ACA GAT TCA GAA ATT AAA CAG ATT     3696<br>
Ala Ser He Lys Pro Glu Asn Leu Thr Asp Ser Glu He Lys Gin He<br>
2055	2060	2065<br>
TAT AGT AGG TAT GGT ATT AAG TTA GAA GAT GGA ATC CTT ATT GAT AAA     3744<br>
Tyr Ser Arg Tyr Gly He Lys Leu Glu Asp Gly He Leu He Asp Lys<br>
2070	2075	2080<br>
AAA GGT GGG ATT CAT TAT GGT GAA TTT ATT AAT GAA GCT AGT TTT AAT     3792<br>
Lys Gly Gly He His Tyr Gly Glu Phe He Asn Glu Ala Ser Phe Asn<br>
2085	2090	2095<br>
ATT GAA CCA TTG CAA AAT TAT GTG ACC AAA TAT GAA GTT ACT TAT AGT     3840<br>
He Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu Val Thr Tyr Ser<br>
2100	2105	2110<br>
AGT GAG TTA GGA CCA AAC GTG AGT GAC ACA CTT GAA AGT GAT AAA ATT     3888<br>
Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu Ser Asp Lys He<br>
2115	2120	2125	2130<br>
TAG AAG GAT GGG ACA ATT AAA TTT GAT TTT ACC AAA TAT AGT AAA AAT     3936<br>
Tyr Lys Asp Gly Thr lie Lys Phe Asp Phe Thr Lys Tyr Ser Lys Asn<br>
2135	2140	2145<br>
GAA CAA GGA TTA TTT TAT GAC AGT GGA TTA AAT TGG GAG TTT AAA ATT     3984<br>
Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp Asp Phe Lys lie<br>
2150	2155	2160<br>
AAT GCT ATT ACT TAT GAT GGT AAA GAG ATG AAT GTT TTT CAT AGA TAT     4032<br>
Asn Ala He Thr Tyr Asp Gly Lys Glu Met Asn Val Phe His Arg Tyr<br>
2165	2170	2175<br>
AAT AAA TAG	4041<br>
Asn Lys<br>
2180<br>
(2) INFORMATION FOR SEQ ID NO:23:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1346 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:23:<br>
Met Lys Arg Met; Glu Gly Lys Leu Phe Met Val Ser Lys Lys Leu Gin<br>
15	10	15<br>
Val Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser He Ser Leu<br>
20	25	30<br>
Leu Asn Asn Glu Val He Lys Ala Glu Gin Leu Asn lie Asn Ser Gin<br>
35	40	45<br>
Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys He Thr Asp Lys Val Glu<br>
50	55	60<br>
Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
65	70	75	80<br>
Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn<br>
85	90	95<br>
Phe Leu Asp Asn Lys Asn Asp He Lys Thr Asn Tyr Lys Glu He Thr<br>
100	105	110<br>
Phe Ser Met Ala Gly Ser Phe Glu Asp Glu He Lys Asp Leu Lys Glu<br>
115	120	125<br>
He Asp Lys Met. Phe Asp Lys Thr Asn Leu Ser Asn Ser He He Thr<br>
130	135	140<br>
Tyr Lys Asn Val Glu Pro Thr Thr He Gly Phe Asn Lys Ser Leu Thr<br>
145<br>
150<br>
155<br>
160<br>
Glu Gly Asn Thr lie Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
165	170	175<br>
Phe Leu Asp Arg Asp lie Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
180	185	190<br>
Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val lie Leu Lys Val Thr<br>
195	200	205<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He<br>
210	215	220<br>
Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val<br>
225	230	235	240<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu<br>
245	250	255<br>
Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He<br>
260	265	270<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala<br>
275	280	285<br>
Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
290	295	300<br>
Gin Asp Tyr Lys Glu He Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
305	310	315	320<br>
Gly Asn Glu Lys Leu Asp Ala Gin He Lys Asn He Ser Asp Ala Leu<br>
325	330	335<br>
Gly Lys Lys Pro He Pro Glu Asn He Thr Val Tyr Arg Trp Cys Gly<br>
340	345	350<br>
Met Pro Glu Phe Gly Tyr Gin He Ser Asp Pro Leu Pro Ser Leu Lys<br>
355	360	365<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr<br>
370	375	380<br>
Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
385	390	395	400<br>
Lys He He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
405	410	415<br>
Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu Asp<br>
420	425	430<br>
Lys Asp Ser Lys Tyr His He Asp Lys Val Thr Glu Val He He Lys<br>
435	440	445<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn Met Lys<br>
450	455	460<br>
Asn Met Lys Lys Lys Leu Ala Ser Val Val Thr Cys Thr Leu Leu Ala<br>
465	470	475	480<br>
Pro Met Phe Leu Asn Gly Asn Val Asn Ala Val Tyr Ala Asp Ser Lys<br>
485	490	495<br>
Thr Asn Gin lie Ser Thr Thr Gin Lys Asn Gin Gin Lys Glu Met Asp<br>
500	505	510<br>
Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe Ser Asn<br>
515	520	525<br>
Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu lie Tyr Asp Gin<br>
530	535	540<br>
Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin Glu Tyr Gin Ser<br>
545	550	555	560<br>
lie Arg Trp lie Gly Leu He Gin Ser Lys Glu Thr Gly Asp Phe Thr<br>
565	570	575<br>
Phe Asn Leu Ser Glu Asp Glu Gin Ala He He Glu He Asn Gly Lys<br>
580	585	590<br>
He He Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu Glu Lys<br>
595	600	605<br>
Gly Lys Leu Val Pro He Lys He Glu Tyr Gin Ser Asp Thr Lys Phe<br>
610	615	620<br>
Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys He Asp<br>
625	630	635	640<br>
Ser Gin Asn Gin Pro Gin Gin Val Gin Gin Asp Glu Leu Arg Asn Pro<br>
645	650	655<br>
Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Pro Ser Lys<br>
660	665	670<br>
He Asn Leu Phe Thr Gin Lys Met Lys Arg Glu He Asp Glu Asp Thr<br>
675	680	685<br>
Asp Thr Asp Gly Asp Ser He Pro Asp Leu Trp Glu Glu Asn Gly Tyr<br>
690	695	700<br>
Thr He Gin Asn Arg He Ala Val Lys Trp Asp Asp Ser Leu Ala Ser<br>
705	710	715	720<br>
Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu Ser His Thr Val<br>
725	730	735<br>
Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu Asp Leu<br>
740	745	750<br>
Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe Pro Ser<br>
755	760	765<br>
Val Asn Val Ser Met Glu Lys Val lie Leu Ser Pro Asn Glu Asn Leu<br>
770	775	780<br>
Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr Thr Asn<br>
785	790	795	800<br>
Thr Glu Gly Ala Ser Val Glu Ala Gly He Gly Pro Lys Gly He Ser<br>
805	810	815<br>
Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr Val Ala Gin Glu<br>
820	825	830<br>
Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala Ser Ala<br>
835	840	845<br>
Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr Gly Ala<br>
850	855	860<br>
He Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn Asp Thr<br>
865	870	875	880<br>
He Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Asn He Ser<br>
885	890	895<br>
Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly He Ala He Thr<br>
900	905	910<br>
Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys Lys Gin<br>
915	920	925<br>
Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu Glu Thr Asn Gin<br>
930	935	940<br>
Thr Asp Gly Val Tyr Lys He Lys Asp Thr His Gly Asn He Val Thr<br>
945	950	955	960<br>
Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys Ala Lys Thr Ala<br>
965	970	975<br>
Ser He He Val Asp Asp Gly Glu Arg Val Ala Glu Lys Arg Val Ala<br>
980	985	990<br>
Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro Ser Leu Thr Leu<br>
995	1000	1005<br>
Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu He Glu<br>
1010	1015	1020<br>
Gly Leu Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu Ser Ser Val Met<br>
1025	1030	1035	1040<br>
Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr Lys Gin Leu Asn<br>
1045	1050	1055<br>
Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu Tyr Asp Val Lys<br>
1060	1065	1070<br>
Leu Thr Pro Lys Met Asn Val Thr lie Lys Leu Ser lie Leu Tyr Asp<br>
1075	1080	1085<br>
Asn Ala Glu Ser Asn Asp Asn Ser lie Gly Lys Trp Thr Asn Thr Asn<br>
1090	1095	1100<br>
lie Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr Ser Ser Asn Asn<br>
1105	1110	1115	1120<br>
Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin Glu Lys Leu Asn<br>
1125	1130	1135<br>
Lys Asn Arg Asp Tyr Tyr lie Ser Leu Tyr Met Lys Ser Glu Lys Asn<br>
1140	1145	1150<br>
Thr Gin Cys Glu He Thr He Asp Gly Glu He Tyr Pro He Thr Thr<br>
1155	1160	1165<br>
Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg Leu Asp He He<br>
1170	1175	1180<br>
Ala His Asn He Lys Ser Asn Pro He Ser Ser Leu His He Lys Thr<br>
1185	1190	1195	1200<br>
Asn Asp Glu He Thr Leu Phe Trp Asp Asp He Ser He Thr Asp Val<br>
1205	1210	1215<br>
Ala Ser He Lys Pro Glu Asn Leu Thr Asp Ser Glu He Lys Gin He<br>
1220	1225	1230<br>
Tyr Ser Arg Tyr Gly He Lys Leu Glu Asp Gly He Leu He Asp Lys<br>
1235	1240	1245<br>
Lys Gly Gly He His Tyr Gly Glu Phe He Asn Glu Ala Ser Phe Asn<br>
1250	1255	1260<br>
He Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu Val Thr Tyr Ser<br>
1265	1270	1275	1280<br>
Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu Ser Asp Lys He<br>
1285	1290	1295<br>
Tyr Lys Asp Gly Thr He Lys Phe Asp Phe Thr Lys Tyr Ser Lys Asn<br>
1300	1305	1310<br>
Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp Asp Phe Lys He<br>
1315	1320	1325<br>
Asn Ala lie Thr Tyr Asp Gly Lys Glu Met Asn Val Phe His Arg Tyr<br>
1330	1335	1340<br>
Asn Lys 1345<br>
(2) INFORMATION FOR SEQ ID NO:24:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1399 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DNA (genomic)<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: 1..1386<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence for VIP2A(a) protein from AB78"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:24:<br>
ATGAAGCGCA TGGAGGGCAA GCTGTTCATG GTGAGCAAGA AGCTCCAGGT GGTGACCAAG	60<br>
ACCGTGCTGC TGAGCACCGT GTTCAGCATC AGCCTGCTGA ACAACGAGGT GATCAAGGCC	120<br>
GAGCAGCTGA ACATCAACAG CCAGAGCAAG TACACCAACC TCCAGAACCT GAAGATCACC	180<br>
GACAAGGTGG AGGACTTCAA GGAGGACAAG GAGAAGGCCA AGGAGTGGGG CAAGGAGAAG	240<br>
GAGAAGGAGT GGAAGCTTAC CGCCACCGAG AAGGGCAAGA TGAACAACTT CCTGGACAAC	300<br>
AAGAACGACA TCAAGACCAA CTACAAGGAG ATCACCTTCA GCATGGCCGG CAGCTTCGAG	360<br>
GACGAGATCA AGGACCTGAA GGAGATCGAC AAGATGTTCG ACAAGACCAA CCTGAGCAAC	420<br>
AGCATCATCA CCTACAAGAA CGTGGAGCCC ACCACCATCG GCTTCAACAA GAGCCTGACC	480<br>
GAGGGCAACA CCATCAACAG CGACGCCATG GCCCAGTTCA AGGAGCAGTT CCTGGACCGC	540<br>
GACATCAAGT TCGACAGCTA CCTGGACACC CACCTGACCG CCCAGCAGGT GAGCAGCAAG	600<br>
GAGCGCGTGA TCCTGAAGGT GACCGTCCCC AGCGGCAAGG GCAGCACCAC CCCCACCAAG	660<br>
GCCGGCGTGA TCCTGAACAA CAGCGAGTAC AAGATGCTGA TCGACAACGG CTACATGGTG	720<br>
CACGTGGACA AGGTGAGCAA GGTGGTGAAG AAGGGCGTGG AGTGCCTCCA GATCGAGGGC	780<br>
ACCCTGAAGA AGAGTCTAGA CTTCAAGAAC GACATCAACG CCGAGGCCCA CAGCTGGGGC	840<br>
ATGAAGAACT ACGAGGAGTG GGCCAAGGAC CTGACCGACA GCCAGCGCGA GGCCCTGGAC	900<br>
GGCTACGCCC GCCAGGACTA CAAGGAGATC AACAACTACC TGCGCAACCA GGGCGGCAGC	960<br>
GGCAACGAGA AGCTGGACGC CCAGATCAAG AACATCAGCG ACGCCCTGGG CAAGAAGCCC	1020<br>
ATCCCCGAGA ACATCACCGT GTACCGCTGG TGCGGCATGC CCGAGTTCGG CTACCAGATC	1080<br>
AGCGACCCCC TGCCCAGCCT GAAGGACTTC GAGGAGCAGT TCCTGAACAC CATCAAGGAG	1140<br>
GACAAGGGCT ACATGAGCAC CAGCCTGAGC AGCGAGCGCC TGGCCGCCTT CGGCAGCCGC	1200<br>
AAGATCATCC TGCGCCTGCA GGTGCCCAAG GGCAGCACCG GCGCCTACCT GAGCGCCATC	1260<br>
GGCGGCTTCG CCAGCGAGAA GGAGATCCTG CTGGACAAGG ACAGCAAGTA CCACATCGAC	1320<br>
AAGGTGACCG AGGTGATCAT CAAGGGCGTG AAGCGCTACG TGGTGGACGC CACCCTGCTG	1380<br>
ACCAACTAGA TCTGAGCTC	1399<br>
(2) INFORMATION FOR SEQ ID NO:25:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 19 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..19<br>
(D) OTHER INFORMATION: /note= "Secretion signal peptide to secrete VIP2 out of a cell"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:25:<br>
Gly Trp Ser Trp lie Phe Leu Phe Leu Leu Ser Gly Ala Ala Gly Val<br>
15	10	15<br>
His Cys Leu<br>
(2) INFORMATION FOR SEQ ID NO:26:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2655 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA" (ill) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: I..2655<br>
(D) OTHER INFORMATION: /note= "maize optimized DNA sequence encoding VIPlA(a)"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:26:<br>
ATGAAGAACA TGAAGAAGAA GCTGGCCAGC GTGGTGACCT GCACCCTGCT GGCCCCCATG	60<br>
TTCCTGAACG GCAACGTGAA CGCCGTGTAC GCCGACAGCA AGACCAACCA GATCAGCACC	120<br>
ACCCAGAAGA ACCAGCAGAA GGAGATGGAC CGCAAGGGCC TGCTGGGCTA CTACTTCAAG	180<br>
GGCAAGGACT TCAGCAACCT GACCATGTTC GCCCCCACGC GTGACAGCAC CCTGATCTAC	240<br>
GACCAGCAGA CCGCCAACAA GCTGCTGGAC AAGAAGCAGC AGGAGTACCA GAGCATCCGC	300<br>
TGGATCGGCC TGATCCAGAG CAAGGAGACC GGCGACTTCA CCTTCAACCT GAGCGAGGAC	360<br>
GAGCAGGCCA TCATCGAGAT CAACGGCAAG ATCATCAGCA ACAAGGGCAA GGAGAAGCAG	420<br>
GTGGTGCACC TGGAGAAGGG CAAGCTGGTG CCCATCAAGA TCGAGTACCA GAGCGACACC	480<br>
AAGTTCAACA TCGACAGCAA GACCTTCAAG GAGCTGAAGC TTTTCAAGAT CGACAGCCAG	540<br>
AACCAGCCCC AGCAGGTGCA GCAGGACGAG CTGCGCAACC CCGAGTTCAA CAAGAAGGAG	600<br>
AGCCAGGAGT TCCTGGCCAA GCCCAGCAAG ATCAACCTGT TCACCCAGCA GATGAAGCGC	660<br>
GAGATCGACG AGGACACCGA CACCGACGGC GACAGCATCC CCGACCTGTG GGAGGAGAAC	720<br>
GGCTACACCA TCCAGAACCG CATCGCCGTG AAGTGGGACG ACAGCCTGGC TAGCAAGGGC	780<br>
TACACCAAGT TCGTGAGCAA CCCCCTGGAG AGCCACACCG TGGGCGACCC CTACACCGAC	840<br>
TACGAGAAGG CCGCCCGCGA CCTGGACCTG AGCAACGCCA AGGAGACCTT CAACCCCCTG	900<br>
GTGGCCGCCT TCCCCAGCGT GAACGTGAGC ATGGAGAAGG TGATCCTGAG CCCCAACGAG	960<br>
AACCTGAGCA ACAGCGTGGA GAGCCACTCG AGCACCAACT GGAGCTACAC CAACACCGAG	1020<br>
GGCGCCAGCG TGGAGGCCGG CATCGGTCCC AAGGGCATCA GCTTCGGCGT GAGCGTGAAC	1080<br>
TACCAGCACA GCGAGACCGT GGCCCAGGAG TGGGGCACCA GCACCGGCAA CACCAGCCAG	1140<br>
TTCAACACCG CCAGCGCCGG CTACCTGAAC GCCAACGTGC GCTACAACAA CGTGGGCACC	1200<br>
GGCGCCATCT ACGACGTGAA GCCCACCACC AGCTTCGTGC TGAACAACGA CACCATCGCC	1260<br>
ACCATCACCG CCAAGTCGAA TTCCACCGCC CTGAACATCA GCCCCGGCGA GAGCTACCCC	1320<br>
AAGAAGGGCC AGAACGGCAT CGCCATCACC AGCATGGACG ACTTCAACAG CCACCCCATC	1380<br>
ACCCTGAACA AGAAGCAGGT GGACAACCTG CTGAACAACA AGCCCATGAT GCTGGAGACC	1440<br>
AACCAGACCG ACGGCGTCTA CAAGATCAAG GACACCCACG GCAACATCGT GACGGGCGGC	1500<br>
GAGTGGAACG GCGTGATCCA GCAGATCAAG GCCAAGACCG CCAGCATCAT CGTCGACGAC	1560<br>
GGCGAGCGCG TGGCCGAGAA GCGCGTGGCC GCCAAGGACT ACGAGAACCC CGAGGACAAG	1620<br>
ACCCCCAGCC TGACCCTGAA GGACGCCCTG AAGCTGAGCT ACCCCGACGA GATCAAGGAG	1680<br>
ATCGAGGGCT TGCTGTACTA CAAGAACAAG CCCATCTACG AGAGCAGCGT GATGACCTAT	1740<br>
CTAGACGAGA ACACCGCCAA GGAGGTGACC AAGCAGCTGA ACGACACCAC CGGCAAGTTC	1800<br>
AAGGACGTGA GCCACCTGTA CGACGTGAAG CTGACCCCCA AGATGAACGT GACCATCAAG	1860<br>
CTGAGCATCC TGTACGACAA CGCCGAGAGC AACGACAACA GCATCGGCAA GTGGACCAAC	1920<br>
ACCAACATCG TGAGCGGCGG CAACAACGGC AAGAAGCAGT ACAGCAGCAA CAACCCCGAC	1980<br>
GCCAACCTGA CCCTGAACAC CGACGCCCAG GAGAAGCTGA ACAAGAACCG CGACTACTAC	2040<br>
ATCAGCCTGT ACATGAAGAG CGAGAAGAAC ACCCAGTGCG AGATCACCAT CGACGGCGAG	2100<br>
ATATACCCCA TCACCACCAA GACCGTGAAC GTGAACAAGG ACAACTACAA GCGCCTGGAC	2160<br>
ATCATCGCCC ACAACATCAA GAGCAACCCC ATCAGCAGCC TGCACATCAA GACCAACGAC	2220<br>
GAGATCACCC TGTTCTGGGA CGACATATCG ATTACCGACG TCGCCAGCAT CAAGCCCGAG	2280<br>
AACCTGACCG ACAGCGAGAT CAAGCAGATA TACAGTCGCT ACGGCATCAA GCTGGAGGAC	2340<br>
GGCATCCTGA TCGACAAGAA AGGCGGCATC CACTACGGCG AGTTCATCAA CGAGGCCAGC	2400<br>
TTCAACATCG AGCCCCTGCA GAACTACGTG ACCAAGTACG AGGTGACCTA CAGCAGCGAG	2460<br>
CTGGGCCCCA ACGTGAGCGA CACCCTGGAG AGCGACAAGA TTTACAAGGA CGGCACCATC	2520<br>
AAGTTCGACT TCACCAAGTA CAGCAAGAAC GAGCAGGGCC TGTTCTACGA CAGCGGCCTG	2580<br>
AACTGGGACT TCAAGATCAA CGCCATCACC TACGACGGCA AGGAGATGAA CGTGTTCCAC	2640<br>
CGCTACAACA AGTAG	2655<br>
(2) INFORMATION FOR SEQ ID NO:27:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1389 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc_feature<br>
LOCATION: I..1389<br>
(D) OTHER INFORMATION: /note= "maize optmized DNA sequence encoding VIP2A(a)"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:27:<br>
ATGAAGCGCA TGGAGGGCAA GCTGTTCATG GTGAGCAAGA AGCTCCAGGT GGTGACCAAG	60<br>
ACCGTGCTGC TGAGCACCGT GTTCAGCATC AGCCTGCTGA ACAACGAGGT GATCAAGGCC	120<br>
GAGCAGCTGA ACATCAACAG CCAGAGCAAG TACACCAACC TCCAGAACCT GAAGATCACC	180<br>
GACAAGGTGG AGGACTTCAA GGAGGACAAG GAGAAGGCCA AGGAGTGGGG CAAGGAGAAG	240<br>
GAGAAGGAGT GGAAGCTTAC CGCCACCGAG AAGGGCAAGA TGAACAACTT CCTGGACAAC	300<br>
AAGAACGACA TCAAGACCAA CTACAAGGAG ATCACCTTCA GCATAGCCGG CAGCTTCGAG	360<br>
GACGAGATCA AGGACCTGAA GGAGATCGAC AAGATGTTCG ACAAGACCAA CCTGAGCAAC	420<br>
AGCATCATCA CCTACAAGAA CGTGGAGCCC ACCACCATCG GCTTCAACAA GAGCCTGACC	480<br>
GAGGGCAACA CCATCAACAG CGACGCCATG GCCCAGTTCA AGGAGCAGTT CCTGGACCGC	540<br>
GACATCAAGT TCGACAGCTA CCTGGACACC CACCTGACCG CCCAGCAGGT GAGCAGCAAG	600<br>
GAGCGCGTGA TCCTGAAGGT GACCGTCCCC AGCGGCAAGG GCAGCACCAC CCCCACCAAG	660<br>
GCCGGCGTGA TCCTGAACAA CAGCGAGTAC AAGATGCTGA TCGACAACGG CTACATGGTG	720<br>
CACGTGGACA AGGTGAGCAA GGTGGTGAAG AAGGGCGTGG AGTGCCTCCA GATCGAGGGC	780<br>
ACCCTGAAGA AGAGTCTAGA CTTCAAGAAC GACATCAACG CCGAGGCCCA CAGCTGGGGC	840<br>
ATGAAGAACT ACGAGGAGTG GGCCAAGGAC CTGACCGACA GCCAGCGCGA GGCCCTGGAC	900<br>
GGCTACGCCC GCCAGGACTA CAAGGAGATC AACAACTACC TGCGCAACCA GGGCGGCAGC	960<br>
GGCAACGAGA AGCTGGACGC CCAGATCAAG AACATCAGCG ACGCCCTGGG CAAGAAGCCC	1020<br>
ATCCCCGAGA ACATCACCGT GTACCGCTGG TGCGGCATGC CCGAGTTCGG CTACCAGATC	1080<br>
AGCGACCCCC TGCCCAGCCT GAAGGACTTC GAGGAGCAGT TCCTGAACAC CATCAAGGAG	1140<br>
GACAAGGGCT ACATGAGCAC CAGCCTGAGC AGCGAGCGCC TGGCCGCCTT CGGCAGCCGC	1200<br>
MGATCATCC TGCGCCTGCA GSTGCCCAA^^GCAGCACTG GTGCCTACCT GAGCGCCATC	1260<br>
GGCGGCTTCG CCAGCGAGM GGAGATCCTG CTGGATAAGG ACAGCAAGTA CCACATCGAC	1320<br>
AAGGTGACCG AGGTGATCAT GAAGGGCGTG AAGCGCTACG TQGTGGACOC CKCCCTGCTG	1380<br>
ACCAACTAG	1389<br>
(2) INFORMATION FOR SEQ ID NO:28:<br>
(1) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2378 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
 (iii) HYPOTHETICALs NO<br>
fix} FEATURE:<br>
NAME/KEY: CDS<br>
LOCATIONi 9..2375<br>
(D) OTHER INFORMATION: /note* "Native DMA sequence encoding viP3A(a) protein from AB88 as contained in pCIB7:.04"<br>
(xi) SEQUENCE DESCRIPTION; SEQ ID NO:28<br>
AGATGAAC ATG AAC AAG AAT AAT ACT AAA TTA AGC ACA AGA GCC TTA CCA     50<br>
Met Asn Lya Asn Asn Thr Lya Leu Ser Thr Arg Ala Leu Pro<br>
15	10<br>
AGT TTT ATT GAT TAT TTT AAT GGC ATT TAT GGA TTT GCC ACT GOT ATC      98<br>
Ser Phe lie Asp Tyr Phe Aan Gly lie Tyr Gly Phe Ala Thr Gly lie<br>
15	20	25	30<br>
AM GAC ATT ATG AAC ATG ATT TTT AM ACG GAT ACA GGT GGT GAT CTA     146<br>
Lye Asp He Met Asn Met He Phe Lys Thr Asp Thr Gly Gly Asp lieu<br>
35	40	45<br>
ACC CTA GAC GAA ATT TTA AAG MT GAG CAG TTA CTA AAT GAT ATT TCT     194<br>
Thr Leu Asp Glu He Leu Lys Asn Gin Gin Leu Leu Asn Asp He Ser<br>
50	55	60<br>
GGT AAA TTG GAT GGG GTG MT GGA AGC TTA MT GAT CTT ATC GCA CAG     242<br>
Gly Lys Leu Asp Gly Val Asn Gly Ser Leu Asn Asp Leu He Ala Gin<br>
65	70	75<br>
GGA MC TTA MT ACA GM TTA TCI1 MG GM ATA TTA MA ATT GCA MT     290<br>
Gly Aan Leu Asn Thr Glu T^u Ser Lys Glu He Leu Lys He Ala Asn<br>
80	85	90<br>
95<br>
100<br>
GAA CAA AAT C^ GTT m AAT GAT GTT MT AAC AAA CTC GAT GCG ATA Glu Gin Asn Gin Val Leu Asn Asp Val Asn Asn Lys Lau Asp Ala lie<br>
105	110<br>
AAT AOG IBG.CTT CGG GfA TAT CTA CCT AAA ATT ACC TCT ATQ TTG ACT<br>
Asn Thr Met "Leu Arg Val Tyr Leu Pro Lys He Thr Ser Met Leu Ser<br>
115	120	125<br>
GAT GTA ATG AAA CAA AAT TAT GOG CTA AST CTG CAA ATA QAA TAG TTA<br>
Asp Val Met Lya Gin Asn Tyr Ala Leu Ser Leu Gin lie Glu Tyr Leu<br>
130	135	140<br>
AGT AAA CAA TTG CAA GAG ATT TCT GAT AAG TTG GAT ATT ATT AAT GTA<br>
Ser Lys Gin Lou Gin Glu lie Ser Asp Lys Leu Asp lie lie Asn Val<br>
145	150	155<br>
AAT GTA CTT ATT AAC TCT ACA CTT ACT GAA ATT ACA CCT GCG TAT CAA<br>
Asn Val Leu lie Asn Ser Thr Leu Thr Glu He Thr Pro Ala Tyr Gin<br>
160	165	170<br>
AGG ATT AAA TAT GTG AAC GAA AAA TTT GAG GAA TTA ACT TTT GOT ACA<br>
Arg He Lys Tyr Val Asn Glu Lys Phe Glu Glu Leu Thr Phe Ala Thr<br>
175	180	185	190<br>
GAA ACT AGT TCA AAA GTA AAA AAG GAT GGC TCT CCT OCA GAT ATT CTT<br>
Glu Thr Ser Ser Lys Val Lys Lys Asp Gly Ser Pro Ala Asp He Leu<br>
195	200	205<br>
GAT GAG TTA ACT GAG TTA ACT GAA CTA GCG AAA AGT GTA ACA AAA AAT<br>
Asp Glu Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val Thr Lys Asn<br>
210	215	220<br>
GAT GTG GAT GGT TTT GAA TTT TAC CTT AAT ACA TTC GAG GAT GTA ATG<br>
Asp Val Asp Gly Phe Qlu Phe Tyr Leu Asn Thr Phe His Asp Val Net<br>
225	230	235<br>
GTA GGA AAT AAT TTA TTC GGG CGT TCA GCT TTA AAA ACT GCA TCG GAA Val Gly Asn Asn Leu Phe Gly Arg Ser Ala Leu Lys Thr Ala Ser Glu<br>
240	245	250<br>
TTA ATT ACT AAA GAA AAT GTG AAA ACA AGT GGC AGT GAG GTC GGA MT<br>
Leu He Thr Lys Glu Aan Val Lys Thr Ser Gly Ser Glu Val Gly Asn<br>
255	260	265	270<br>
GTT TAT AAC TTC TTA ATT GTA TTA ACA GCT CTG CAA GCC CAA GCT TTT<br>
Val Tyr Asn Phe Leu He Val Leu Thr Ala Leu Gin Ala Gin Ala i?he<br>
275	280	285<br>
CTT ACT TTA ACA ACA TGC CGA AAA TTA TTA GGC TTA GCA GAT ATT (SAT<br>
Leu Thr Leu Thr Thr Cys Arg Lys Leu Leu Gly Leu Ala Asp He Asp<br>
290	295	300<br>
TAT ACT TCT ATT ATG AAT GAA CAT TTA AAT AAG GAA AAA GAG GAA TTT Tyr Thr Ser He Met Asn Glu His Leu Asn Lya Glu Lya Glu Glu Phe<br>
305	310	315<br>
AGA GTA AAC ATC CTC CCT ACA CTT TCT AAT ACT TTT TCT AAT CC7 AAT    1010<br>
Arg Val Asn lie Leu Pro Thr Lew Ser Aan Thr Phe Ser Asn Pro Asn<br>
320	325	330<br>
TAT GCA AM GTT AAA GGA AGT GAT GAA GAT GCA AAG ATG ATT GTG GAA    1058 Tyr Ala Lys Val Lys Gly Ser Asp Glu Asp Ala Lys Met He Val Glu<br>
gÂ£Â£ /uÂ«i u^i W*A UU' 368 TTO ATT GGG TTT 
Ala Lys Pro Gly His Ala Leu lie Gly Phe Glu lie Ser Asn Asp Ser<br>
355	360	36Â£<br>
ATT ACA GTA TTA AM GTA TAT GAG GOT AAG CTA AAA CAA AAT TAT CAA     1154<br>
lie Thr Val Leu Lys Val Tyr Glu Ala Lys Leu Lys Gin Asn Tyr Gin<br>
370	375	380<br>
GTC GAT AAG GAT TCC TTA TCG GAA GTT ATT TAT GGT GAT ATG GAT AAA    1202<br>
Val Asp Lys Asp Ser Leu Ser Glu Val lie Tyr Gly Asp Met Asp Lys<br>
385	390	395<br>
TTA TTG TGC CCA GAT CM TCT GAA CAA ATC TAT TAT AGA AAT AAC ATA    1250<br>
Leu Leu Cys Pro Asp Gin Ser Glu Gin lie Tyr Tyr Thr Asn Asn lie<br>
400	405	410<br>
GTA TTT CCA AAT GAA TAT GTA ATT ACT AAA ATT GAT TTC ACT AAA AAA    1298<br>
Val Phe Pro Asn Glu Tyr Val lie Thr Lys lie Asp Phe Thr Lys Lys<br>
415	420	425	430<br>
ATG AAA ACT TTA AGA TAT GAG GTA ACA GCG AAT TTT TAT GAT TCT TCT    1346<br>
Met Lys Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr Asp Ser Ser<br>
435	440	445<br>
ACA GGA GM AIT GAC TTA AAT AAG AAA MA GTA GAA TCA AGT GAA GCG    1394<br>
Thr Gly Glu lie Aap Leu Asn Lys Lys Lys Val Glu Ser Ser Glu Ala<br>
450	455	460<br>
GAG TAT AGA AOG TTA AGT GCT AAT GAT GAT GGG GTG TAT ATG COS TTA    1442<br>
Glu Tyr Arg Thr Leu Ser Ala Asn Aap Asp Gly Val Tyr Met Pro Leu<br>
465	470	475<br>
1490<br>
Gly Val lie Ser Qlu Thr Phe Leu Thr Pro lie Asn Gly Phe Gly Leu<br>
480	485	490<br>
CAA GCT GAT GAA AAT TCA AGA TTA ATT ACT TTA ACA TGT AAA TCA TAT    1538<br>
Gin Ala Asp Glu Asn Ser Arg Leu lie Thr Leu Thr Cys Lys Ser Tyr<br>
495	500	505	510<br>
TTA AGA GAA CTA CTG CTA OCA ACA GAC TTA AGO AAT AAA â¬AA ACT AAff    1586<br>
Leu Arg Glu Leu Leu Leu Ala Thr Asp Leu Ser Asn Lya Glu Thr Lys<br>
515	520	525<br>
TTG ATC GTC CCG CCA AGT GGT TTT ATT AGC AAT ATT GTA GAG AAC GGG    1634<br>
Leu lie Val Pro Pro Ser Gly Phe lie Ser Asn lie Val Glu Aan Gly<br>
530	535	540<br>
TCC ATA SAA GAG GAC'AAT TTA GAG CCG TGG AAA GCA AAT AAT AAG AAT    1682<br>
Ser lie Glu -Glu Asp Asn Leu Glu Pro Trp Lys Ala Asn Asn Lys Asn<br>
545	550	555<br>
GCG TAT GTA GAT CAT ACA GGC GGA GTG AAT GGA ACT AAA GOT TTA TAT    1730<br>
Ala Tyr Val Asp His Thr Gly Gly Val Asn Gly Thr Lys Ala Leu Tyr<br>
560	565	570<br>
GTT CAT AAG GAC C3GA GGA ATT TCA CAA TTT ATT GGA GAT AAG TTA AAA    1778<br>
Val His Lys Asp Gly Gly lift Ser Gin Phe lie Gly Asp Lys Leu Lys<br>
575	580	585	590<br>
CCG AAA ACT GAG TAT GTA ATC CAA TAT ACT GTT AAA GGA AAA CCT TCT    1826<br>
Pro Lys Thr Glu Tyr Val lie Gin Tyr Thr Val Lys Gly Lys Pro Ser<br>
595	600	605<br>
ATT CAT TTA AAA GAT GAA AAT ACT GGA TAT ATT CAT TAT GAA GAT ACA     1874<br>
lie His Leu Lys Asp Glu Asn Thr Gly Tyr lie His Tyr Glu Asp Thr<br>
610	615	620<br>
AAT AAT AAT TTA GAA GAT TAT CAA ACT ATT AAT AAA CGT TTT ACT' ACA    1922<br>
Asn Asn Asn Leu Glu Asp Tyr Gin Thr lie Asn Lys Arg Phe Thr Thr<br>
625	630	635<br>
GGA ACT GAT TTA AAG QGA GTG TAT TTA ATT TTA AAA AST CAA AAT GGA    1970<br>
Gly Thr Asp Leu Lys Gly Val Tyr Leu lie Leu Lys Ser Gin Asn Gly<br>
640	645	650<br>
GAT GAA GCT TGG GGA GAT AAC TTT ATT ATT TTG GAA ATT ACT CCT TCT    2018<br>
Asp Glu Ala Trp Gly Asp Asn Phe lie lie Leu Glu He Ser Pro Ser<br>
655	660	665	670<br>
GAA AAG TTA TTA AST OCA GAA TTA ATT AAT ACA AAT AAT TGG ACG AGT    2066<br>
Glu Lys Leu Leu Ser Pro Glu Leu He Asn Thr Asn Asn Trp Thr Ser<br>
675	680	685<br>
ACG GGA TCA ACT AAT ATT AGC GOT AAT ACA CTC ACT CTT TAT GAG GGA    2114<br>
Thr Gly Ser Thr Asn He Ser Gly Asn Thr Leu Thr Leu Tyr Gin Gly<br>
690	695	700<br>
GGA CGA GGG ATT CTA AAA CAA AAC CTT CAA TTA GAT AGT TTT TCA ACT     2162<br>
Gly Arg Gly He Leu Lys Gin Asn Leu Gin Leu Asp Ser Phe Ser Thr<br>
705	710	715<br>
TAT AGA GTG TAT TTT TCT GTG TCC GGA GAT XT AAT GTA AGG ATT AGA    2210<br>
Tyr Arg Val Tyr Phe Ser Val Ser Gly Asp Ala AÂ«n Val Arg He Arg<br>
720	725	730<br>
AAT TCT AGG GAA GTG TTA TTT GAA AAA AGA TAT ATG AGC GGT GCT AAA    2258<br>
Asn Ser Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser Gly Ala Lys<br>
735	740	745	750<br>
GAT GTT TCT 6AA ATG TTC ACT ACA MA TTT GAS AAA GAT AAC TTT TAT    2306<br>
Asp Val Ser Glu Met Pha Thr Thr Lya Phe filu i.ys Asp AÂ»n Pho Tyr<br>
756	760	765<br>
ATA GAG CTT TCT 
lie Glu Leu S0r Gin Gly Asn Asn Leu Tyr Gly Gly Pro He Val His<br>
770	775	780<br>
TTT TAG GAT GTC TCT ATT AAG TAA	2378<br>
Phe Tyr Asp Val Ser He Lys 785<br>
(2) INFORMATION FOR SBQ ID NO:29:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 789 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY* linear<br>
(ii) MOLECULE TYPEs protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:29s<br>
Met Asn Lys Asn Asn Thr Lys Leu SÂ«r Thr Arg Ala Leu Pro Oer Phe<br>
15	10	15<br>
He Asp Tyr Phe Aan Gly He Tyr Gly Phe Ala Thr Gly He Lys Asp<br>
20	25	30<br>
He Met Asn Mit He Phe Lys Thr Asp Thr Gly Gly Asp Leu Thr Leu<br>
35	40	45<br>
Asp Glu He Leu Lys Asn Gin Gin Leu Leu Asn Asp He Ser Gly Lys<br>
50	55	60<br>
Leu Asp Gly Val Asn Gly Ser LÂ»u Asn Asp Leu He Ala Gin Gly Asn<br>
65	70	75	80<br>
Leu Asn Thr Glu Leu Ser Lys Glu He Leu Lys He Ala Asn Glu Gin<br>
85	90	95<br>
Asn Gin Val Leu Asn Asp Val Asn Asn Lys Leu Asp Ala He Asn Thr<br>
100	105	HO<br>
Met Leu Arg Val Tyr Leu Pro Lys He Thr Ser Met Leu Ser Asp Val<br>
115	120	125<br>
Met Lys Gin Asn Tyr Ala Leu Ser Leu Gin He Glu Tyr Leu Ser Lys<br>
130	135	140<br>
Gin Leu Gin Glu He Ser Asp Lys Leu Asp He He Asn Val Asn Val<br>
145	150	155	160<br>
Leu He Asn Ser Thr Leu Thr Glu He Thr Pro Ala Tyr Gin Arg He<br>
165	170	175<br>
Tyr Val Asn Glu Ty* Ph Gin Glu Leu Thr Ph Aim Thr Glu Thr<br>
180	185	190<br>
Ser Ser Jys^Val Lya Lys Asp Gly Ser Pxo Ala Aap lie Leu Asp Glu<br>
195	200	205<br>
Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val Thr Lys Asn Asp Val<br>
210	215	220<br>
Asp Gly Phe Glu Phe Tyr Leu Asn Thr Phe His Asp Val Met Val Gly<br>
225	230	235	240<br>
ash Asn Leu Phe Gly Arg Sec Ala Leu Lys Thr Ala Ser Glu Leu He<br>
245	250	255<br>
Thr Lys Glu Asn Val Lys Thr Ser Gly Ser Glu Val Gly Asn Val Tyr<br>
260	265	270<br>
Asn Phe Leu He Val Leu Thr Ala Leu Gin Ala Gin Ala Phe Leu Thr<br>
275	280	285<br>
Leu Thr Thr Cys Arg Lys Leu Ltu Gly Leu Ala Asp He Asp Tyr Thr<br>
290	295	300<br>
Ser He Met Asn Glu His Leu Aan Lys Glu Lys Glu Glu Phe Arg Val<br>
305	310	315	320<br>
Asn He Leu Pro Thr Leu Ser Asn Thr Phe Ser Asn Pro Asn Tyr Ala<br>
325	330	335<br>
Lys Val Lys Gly Ser Asp Glu Asp Ala Lys Met He Val Glu Ala Lys<br>
340	345	350<br>
Pro Gly His Ala Leu He Gly Phe Glu He Ser Asn Asp Ser He Thr<br>
355	360	365<br>
Val Leu Lys Val Tyr Glu Ala Lys Leu Lys Gin Asn Tyr Gin Val Asp<br>
370	375	380<br>
Lys Asp Ser Leu Ser Glu Val lie Tyr Gly Asp Met Asp Lys Leu Leu<br>
385	390	395	400<br>
Cys Pro Asp Gin Ser Glu Gin He Tyr Tyr Thr Asn Asn He Val Phe<br>
405	410	415<br>
Pro Asn Glu Tyz Val He Thr Lys He Asp Phe Thr Lys-tys Met Lys<br>
420	425	430<br>
Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr Asp Ser Ser Thr Gly<br>
435	440	445<br>
Glu He Asp Leu Asn Lys Lys Lys Val Glu Ser Ser Glu Ala Glu Tyr<br>
450	455	460<br>
Arg Thr I ah S*r Ala AÂ»n Asp AÂ»p Giy Val Tyr Met Pxx&gt; Leu Gly Val<br>
465	470	475	480<br>
He Ser -Glu Thr Phe Leu Thr Pro lie Asn Gly Phe Gly Leu Gin Ala<br>
485	490	495<br>
Asp Glu Asn Sec Arg Leu He The Leu Thr Cys Lye Ser Tyr Leu Arg<br>
500	505	510<br>
Glu Leu Leu Leu Ala Thr Asp Leu Sar Aen Lys Glu Thr Lys Leu lie<br>
515	520	525<br>
Val Fro Pro Ser Gly Phe lie Ser Asn lie Val Glu Asn Gly Ser lie<br>
530	535	540<br>
Glu Glu Asp Asn Leu Glu Pro Trp Lye Ala Asn Asn Lys Asn Ala Tyr<br>
545	550	555	560<br>
Val Asp His Thr Gly Gly Val Aan Gly Thr Lys Ala Leu Tyr Val His<br>
565	570	575<br>
Lys Asp Gly Gly He Ser Gin Phe He Gly Asp Lys Leu Lys Pro Lys<br>
580	585	590<br>
Thr Glu Tyr Val lie Gin Tyr Thr Val Lys Gly Lys Pro Ser He His<br>
595	600	605<br>
Leu Lys Asp Glu Asn Thr Gly Tyr He His Tyr Glu Asp Thr Asn Asn<br>
610	615	620<br>
Asn Leu Glu Asp Tyr Gin Thr He Asn Lys Arg<br>
625	630	635<br>
Gly Thr 640<br>
Asp Leu Lys Gly Val Tyr Leu He Leu Lys Ser Gin Asn Gly Asp Glu<br>
645	650	655<br>
Ala Trp Gly Asp Asn Phe He He Leu Glu He Ser Pro Ser Glu Lys<br>
660	665	670<br>
Leu Leu Ser Pro Glu Leu He Asn Thr Asn Asn Trp Thr Ser Thr Gly<br>
675	680	685<br>
Ser Thr Asn He Ser Gly Asn Thr Leu Thr Leu Tyr Gin Gly Gly Arg<br>
690	695	' 700<br>
Gly He Leu Lys Gin Asn Leu Gin Leu Asp Ser Phe Ser Thr Tyr Arg<br>
705	710	715	720<br>
Val Tyr Phe Ser Val Ser Gly Asp Ala Asn Val Arg He Arg Asn Ser<br>
725	730	735<br>
Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser Gly Ala Lys Asp Val<br>
740	745	750<br>
Olu Mat Plie Thr Thr Lys the (ilu Lys Asp Asn Phe Tyr lie Glu<br>
755	760	765<br>
Leu Ser, Gin (Sly AsÂ« Asn Leu Tyr Gly Gly Pro He Val His Phe Tyr<br>
770^ _         775	780<br>
Aap Val Ser lie Lys 785<br>
(2) INFORMATION FOR SEQ ID NO:30:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENOTH: 2403 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
0} TOPOLOGY: linear<br>
(ii) MOLECULE TYPE; other nucleic acid<br>
(A) DESCRIPTION: /desc - "Synthetic DMA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: misc feature<br>
LOCATION: 11..2389<br>
(D) OTHER INFORMATION: /note- "maize optimized DNA sequence encoding VlP3A(a)"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:30:<br>
GGATCCACCA ATGAACATGA ACAAGAACAA CACCAAGCTG AGCACCCGCG CCCTGCCGAG	60<br>
CTTCATCGAC TACTTCAACG GCATCTACGG CTTCGCCACC GQCATCAAGG ACATCATGAA	120<br>
GCAGCTGCTG AACGACATCA GCGGCAAGCT GGACGGCGTG AACGGCAGCC TGAA(2GACCT	240<br>
GATCGCCCAG GGCAACCTGA ACACCGAGCT GAGCAAGGAG ATCCTTAAGA TCGCCAACGA	300<br>
GCAGAACCAG GTQCTGAACG ACOTGAACAA CWW3CTGGAC GCCATCAACA CCAT(3CTGCG	360<br>
CGTGTACCTG CCGAAGATCA CCAGCATGCT GAGCGACGTG ATGAAGCAGA ACTA(^3CCCT	420<br>
GAGCX7TGCAG ATCGAGTACC TOAGCAAGCA GCTGCAGGAG ATCAGCGACA AGCTGGACAT	480<br>
CATCAACGTG AACGTCClKjA TCAACAGCAC CCTGACCGAG ATCACCCCGG CCTACCAGCG	540<br>
CATCAAGTAC GTGAACGAGA AGTTCGAAGA GCTGACCTTC GCCACCGAGA CCAGCAGCAA	600<br>
GGTGAAGAAG GACGGCAGCC CGGCCGACAT CCTGGACGAG CTGACCGAGC TGACCGAGCT	660<br>
GGCCAAGAGC GTGACCAAGA ACGACGTG3A CGGCTTCGAG TTCTACCTGA ACACCOTCCA	720<br>
CGACGTGATG CTGGGCAACA ACCTGTTCGG CCGCAGCGCC CTGAAGACCG CCAGCGAGCT	780<br>
GATCACCAAG GAGAACGfGA AGACCAGCGG CAGCGAGGTG GGCAACGTGT ACAftCTTCCT	840<br>
GATCGTGCTG ACCGCCCTGC AGSCCCAGGC CTTCCTQACC CTGACCACCT GTCQCAAGCT	900<br>
GCTGQGCCTG GCCGACATCG ACTACACCAG CATCATQAAC GAGCACTTGA ACAAGGAGAA	960<br>
GGAG6AGTTC OGCGTGAACA TCCTGCCGAC CCTGAGCAAC AOCTTCAGCA ACCCGAACTA	1020<br>
CGCCAAGGTG AAGGGCAQCG AC6AGGACGC CMGATGATC GTGGAGGCTA AGCCGGGCCA	1080<br>
CGCGTTGATC GGCTTCGAGA TCAGCAACGA CAGCATCACC GTGCTGAAGG TGTACGAGGC	1140<br>
CAAGCTGAAG CAGAACTACC AGGTGGACAA GGACAGCTTG AGCGAGGTGA TCTACGGCQA	1200<br>
CATGGACAAG CTGCTGTGTC CGGACCA6AG CGAGCAAATC TACTACACCA ACAACATOGT	1260<br>
GTTCCC6AAC GAGTACGTGA TCACCAAGAT CGACTTCACC AAGAAGATGA AGACCCTGCG	1320<br>
CTACGAGGTG ACCGCCAACT TCTACGACAG CAGCACCGGC GAGATCGACC TGAACAAGAA	1380<br>
GAAGGTGGAG AQCAGCGAGG CCGAGTACCG CACCCTGAGC GCGAACGACG ACGGCGTCTA	1440<br>
CATGCCACTG GGCGTGATCA GCGAGACCTT CCTGACCCCG ATCAACGGCT TTGGCCTGCA	1500<br>
GGCCGACGAG AACAGCCGCC TGATCACCCT GACCTGTAAG AGCTACCTGC GCGAGCTGCT	1560<br>
GCTAGCCACC GACCTQAGCA ACAAQGAGAC CAAGCT3ATC GTGCCACCGA GCGGCTTCAT	1620<br>
CAGCAACATC GTGGAGAACG GCAGCATCGA GGAGSACAAC CTGGAGCCGT GGAAGGCCAA	1680<br>
CAACAAGAAC GCCTAO5TGG ACCACACCQG CGGCGTGAAC QGCACCAAGG CCCT'3TACGT	1740<br>
GCACAAGGAC GGCGGCATCA GCCAGTTCAT CGGCGACAAG CTGAAGCCGA AGA033AGTA	1800<br>
CGTGATCCAG TACACCGTGA AGGGCAAGCC ATCGATTCAC CTGAAGGACG AGAAi^CCGG	1860<br>
CTACATCCAC TACGAGGACA CCAACAACAA CCTGGAGGAC TACCAGACCA TCAAI^AGCG	1920<br>
CTTCACCACC GGCACCGACC TÂ»3AAGGGCGT GTACCTGATC CTGAAGAGCC AGAA(^GCGA	1980<br>
CGAGGCCTGG GGCGAOAAfrr TCATCATO?T GGRGATCAGC CCXSAGCGAGA AGC?K3CTGAG	2040<br>
CCCGGAGCTG ATCAACACCA ACAACTGGAC CAGCACCGGC AGCACCAACA TCAGCGGCAA	2100<br>
CACCCTGACC ClXSTACCAGG GCGGCCGCGG CATCCTGAAG CAGAACCTGC AGCTG1GACAG	2160<br>
CTTCAGCACC TACCGCGTGT ACTTCAGCGT GAGCGGCGAC GCCAACGTGC GCATCCGCAA	2220<br>
CAGCCGOGAG GTGCTGTTCG AGAAGAQ6TA CATGAGCGGC GCCAAGGACG TGAGCGAGAT	2280<br>
GTTCACCACC AAGTTCGAGA AGGACAACTT CTACATCGAG CTGAGCCAGG GCAAC^ACCT	2340<br>
GTACGGOGGC CCGATCGTGC ACTTCTAC6A CGTGAGCATC AAGTTAACGT AGAGCTCAGA   2400<br>
TCT	2403<br>
*<br>
(2) INFORMATION FOR SEQ 3D NO;31;<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2612 base pairs<br>
TYPBt nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: DMA (gencmic) (iii) HYPOTHETICAL: NO<br>
(ix) FEATURE!<br>
NAME/KEY; CDS<br>
LOCATION: 118,.2484<br>
(0) OTHER INFORMATION: /note- "Native DNA sequence encoding VIP3A(b) from AB424"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:31:<br>
ATTGAAATTG ATAAAAAGTT ATGAGTGTTT AATAATCAGT MTTACCAAT AAAGMTTAA    60 GAATACAAGT TTACAAGAAA TAAGTGTTAC AAAAAATAGC TGAAAAGGAA GATGAAC      117<br>
ATG AAC AAG AAT AAT ACT AAA TTA AGC ACA AGA GCC TTA OCA AGTTTT     165<br>
Met Asn Lys Asn Asn Thr Lys Leu Ser Thr Arg Ala Leu Pro Ser Phe<br>
790	795	800	805<br>
ATT GAT TAT TTC AAT QGC ATT TAT GGA TTT GCC ACT GGT ATC AAA GAC     213<br>
lie Asp Tyr Phe Asn Gly He Tyr Gly Phe Ala Thr Gly He Lys Asp<br>
810	815	820<br>
ATT ATG AAC ATG ATT TTT AAA ACG GAT ACA GGT GGT GAT CTA ACC CTA     261<br>
He Met Asn Met He Phe Lys Thr Asp Thr Gly Gly Asp Leu Thr Leu<br>
825	830	835<br>
GAC GAA ATT TTA AAG AAT GAG CAG CTA CTA AAT GAT ATT TCT GGT AAA     309<br>
Asp Glu He Leu Lys Asn Gin Gin Leu Leu Asn Asp He Ser Gly Lys<br>
840	845	850<br>
TTG GAT GGG GTG AAT GGA AGC TTA AAT GAT CTT ATC GCA CAG GGA AAC     357<br>
Leu Asp Gly Val Asn Gly Ser Leu Asn Asp Leu He Ala Gin Gly Asn<br>
855	860	865<br>
TTA AAT ACA GAA TTA TCT AAG GAA ATA TTA AAA ATT GCA AAT GAA CAA     405<br>
Leu Asn Thr Glu Leu Ser Lys Glu He Leu Lys He Ala Asn Glu Gin<br>
870	875	880	885<br>
AAT CAA GTT TTA AAT GAT GTT AAT AAC AAA CTC GAT GCG ATA AAT ACG     453<br>
Asn Gin Val Leu Asn Asp Val Asn Asn Lys Leu Asp Ala lie Asn Thr<br>
890	895	900<br>
ATG CTT CGG GTA TAT CTA CCT AAA ATT ACC TCT ATG TTG AGT GAT GTA<br>
Met Leu Arg Val Tyr Leu Pro Lys He Thr Ser Met Leu Ser Asp Val<br>
905	910	915<br>
501<br>
ATG AAA CAA AAT TAT GCG CTA AGT CTG CAA ATA GAA TAG TTA AGT AAA      549<br>
Met Lys Gin Asn Tyr Ala Leu Ser Leu Gin He Glu Tyr Leu Ser Lys<br>
920	925	930<br>
CAA TTG CAA GAG ATT TCT GAT AAG TTG GAT ATT ATT AAT GTA AAT GTA      597<br>
Gin Leu Gin Glu He Ser Asp Lys Leu Asp He He Asn Val Asn Val<br>
935	940	945<br>
CTT ATT AAC TCT ACA CTT ACT GAA ATT ACA CCT GCG TAT CAA AGG ATT<br>
Leu He Asn Ser Thr Leu Thr Glu He Thr Pro Ala Tyr Gin Arg He<br>
950	955	960	965<br>
645<br><br>
AAA TAT GTG AAC GAA AAA TTT GAG GAA TTA ACT TTT GCT ACA GAA ACT<br>
Lys Tyr Val Asn Glu Lys Phe Glu Glu Leu Thr Phe Ala Thr Glu Thr<br>
970	975	980<br>
693<br><br>
AGT TCA AAA GTA AAA AAG GAT GGC TCT CCT GCA GAT ATT CGT GAT GAG<br>
Ser Ser Lys Val Lys Lys Asp Gly Ser Pro Ala Asp He Arg Asp Glu<br>
985	990	995<br>
741<br><br>
TTA ACT GAG TTA ACT GAA CTA GCG AAA AGT GTA ACA AAA AAT GAT GTG<br>
Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val Thr Lys Asn Asp Val<br>
1000	1005	1010<br>
789<br><br>
GAT GGT TTT GAA TTT TAG CTT AAT ACA TTC CAC GAT GTA ATG GTA GGA<br>
Asp Gly Phe Glu Phe Tyr Leu Asn Thr Phe His Asp Val Met Val Gly<br>
1015	1020	1025<br>
837<br>
AAT AAT TTA TTC GGG CGT TCA GCT TTA AAA ACT GCA TCG GAA TTA ATT      885<br>
Asn Asn Leu Phe Gly Arg Ser Ala Leu Lys Thr Ala Ser Glu Leu He<br>
1030	1035	1040	1045<br>
ACT AAA GAA AAT GTG AAA ACA AGT GGC AGT GAG GTC GGA AAT GTT TAT      933<br>
Thr Lys Glu Asn Val Lys Thr Ser Gly Ser Glu Val Gly Asn Val Tyr<br>
1050	1055	1060<br>
AAC TTC CTA ATT GTA TTA ACA GCT CTG CAA GCA AAA GCT TTT CTT ACT      981<br>
Asn Phe Leu He Val Leu Thr Ala Leu Gin Ala Lys Ala Phe Leu Thr<br>
1065	1070	1075<br>
TTA ACA CCA TGC CGA AAA TTA TTA GGC TTA GCA GAT ATT GAT TAT ACT     1029<br>
Leu Thr Pro Cys Arg Lys Leu Leu Gly Leu Ala Asp He Asp Tyr Thr<br>
1080	1085	1090<br>
TCT ATT ATG AAT GAA CAT TTA AAT AAG GAA AAA GAG GAA TTT AGA GTA<br>
Ser He Met Asn Glu His Leu Asn Lys Glu Lys Glu Glu Phe Arg Val<br>
1095	1100	1105<br>
1077<br>
AAC ATC CTC CCT ACA CTT TCT AAT ACT TTT TCT AAT CCT AAT TAT GCA     1125<br>
Asn lie Leu Pro Thr Leu Ser Asn Thr Phe Ser Asn Pro Asn Tyr Ala<br>
1110	1115	1120	1125<br>
AAA GTT AAA GGA ACT GAT GAA GAT GCA AAG ATG ATT GTG GAA GCT AAA     1173<br>
Lys Val Lys Gly Ser Asp Glu Asp Ala Lys Met lie Val Glu Ala Lys<br>
1130	1135	1140<br>
CCA GGA CAT GCA TTG ATT GGG TTT GAA ATT ACT AAT GAT TCA ATT ACA     1221<br>
Pro Gly His Ala Leu lie Gly Phe Glu lie Ser Asn Asp Ser lie Thr<br>
1145	1150	1155<br>
GTA TTA AAA GTA TAT GAG GCT AAG CTA AAA CAA AAT TAT CAA GTC GAT     1269<br>
Val Leu Lys Val Tyr Glu Ala Lys Leu Lys Gin Asn Tyr Gin Val Asp<br>
1160	1165	1170<br>
AAG GAT TCC TTA TCG GAA GTT ATT TAT GGC GAT ATG GAT AAA TTA TTG     1317<br>
Lys Asp Ser Leu Ser Glu Val lie Tyr Gly Asp Met Asp Lys Leu Leu<br>
1175	1180	1185<br>
TGC CCA GAT CAA TCT GGA CAA ATC TAT TAT ACA AAT AAC ATA GTA TTT     1365<br>
Cys Pro Asp Gin Ser Gly Gin He Tyr Tyr Thr Asn Asn He Val Phe<br>
1190	1195	1200	1205<br>
CCA AAT GAA TAT GTA ATT ACT AAA ATT GAT TTC ACT AAA AAA ATG AAA     1413<br>
Pro Asn Glu Tyr Val lie Thr Lys He Asp Phe Thr Lys Lys Met Lys<br>
1210	1215	1220<br>
ACT TTA AGA TAT GAG GTA ACA GCG AAT TTT TAT GAT TCT TCT ACA GGA     1461 Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr Asp Ser Ser Thr Gly<br>
1225	1230	1235<br>
GAA ATT GAC TTA AAT AAG AAA AAA GTA GAA TCA AGT GAA GCG GAG TAT     1509<br>
Glu He Asp Leu Asn Lys Lys Lys Val Glu Ser Ser Glu Ala Glu Tyr<br>
1240	1245	1250<br>
AGA ACG TTA AGT GCT AAT GAT GAT GGG GTG TAT ATG CCG TTA GGT GTC     1557<br>
Arg Thr Leu Ser Ala Asn Asp Asp Gly Val Tyr Met Pro Leu Gly Val<br>
1255	1260	1265<br>
ATC AGT GAA ACA TTT TTG ACT CCG ATT AAT GGG TTT GGC CTC CAA GCT     1605<br>
He Ser Glu Thr Phe Leu Thr Pro He Asn Gly Phe Gly Leu Gin Ala<br>
1270	1275	1280	1285<br>
GAT GAA AAT TCA AGA TTA ATT ACT TTA ACA TGT AAA TCA TAT TTA AGA     1653<br>
Asp Glu Asn Ser Arg Leu He Thr Leu Thr Cys Lys Ser Tyr Leu Arg<br>
1290	1295	1300<br>
GAA CTA CTG CTA GCA ACA GAC TTA AGC AAT AAA GAA ACT AAA TTG ATC     1701<br>
Glu Leu Leu Leu Ala Thr Asp Leu Ser Asn Lys Glu Thr Lys Leu He<br>
1305	1310	1315<br>
GTC CCG CCA AGT GGT TTT ATT AGC AAT ATT GTA GAG AAC GGG TCC ATA     1749<br>
Val Pro Pro Ser Gly Phe He Ser Asn He Val Glu Asn Gly Ser He<br>
1320	1325	1330<br>
GAA GAG GAG AAT TTA GAG CCG TGG AAA GCA AAT AAT AAG AAT GCG TAT     1797<br>
Glu Glu Asp Asn Leu Glu Pro Trp Lys Ala Asn Asn Lys Asn Ala Tyr<br>
1335	1340	1345<br>
GTA GAT CAT ACA GGC GGA GTG AAT GGA ACT AAA GCT TTA TAT GTT CAT     1845<br>
Val Asp His Thr Gly Gly Val Asn Gly Thr Lys Ala Leu Tyr Val His<br>
1350	1355	1360	1365<br>
AAG GAC GGA GGA ATT TCA CAA TTT ATT GGA GAT AAG TTA AAA CCG AAA     1893<br>
Lys Asp Gly Gly lie Ser Gin Phe lie Gly Asp Lys Leu Lys Pro Lys<br>
1370	1375	1380<br>
ACT GAG TAT GTA ATC CAA TAT ACT GTT AAA GGA AAA CCT TCT ATT CAT     1941<br>
Thr Glu Tyr Val He Gin Tyr Thr Val Lys Gly Lys Pro Ser He His<br>
1385	1390	1395<br>
TTA AAA GAT GAA AAT ACT GGA TAT ATT CAT TAT GAA GAT ACA AAT AAT     1989<br>
Leu Lys Asp Glu Asn Thr Gly Tyr He His Tyr Glu Asp Thr Asn Asn<br>
1400	1405	1410<br>
AAT TTA GAA GAT TAT CAA ACT ATT AAT AAA CGT TTT ACT ACA GGA ACT     2037<br>
Asn Leu Glu Asp Tyr Gin Thr He Asn Lys Arg Phe Thr Thr Gly Thr<br>
1415	1420	1425<br>
GAT TTA AAG GGA GTG TAT TTA ATT TTA AAA AGT CAA AAT GGA GAT GAA     2085<br>
Asp Leu Lys Gly Val Tyr Leu He Leu Lys Ser Gin Asn Gly Asp Glu<br>
1430	1435	1440	1445<br>
GCT TGG GGA GAT AAC TTT ATT ATT TTG GAA ATT AGT CCT TCT GAA AAG     2133<br>
Ala Trp Gly Asp Asn Phe He He Leu Glu He Ser Pro Ser Glu Lys<br>
1450	1455	1460<br>
TTA TTA AGT CCA GAA TTA ATT AAT ACA AAT AAT TGG ACG AGT ACG GGA     2181<br>
Leu Leu Ser Pro Glu Leu He Asn Thr Asn Asn Trp Thr Ser Thr Gly<br>
1465	1470	1475<br>
TCA ACT AAT ATT AGC GGT AAT ACA CTC ACT CTT TAT CAG GGA GGA CGA     2229<br>
Ser Thr Asn He Ser Gly Asn Thr Leu Thr Leu Tyr Gin Gly Gly Arg<br>
1480	1485	1490<br>
GGG ATT CTA AAA CAA AAC CTT CAA TTA GAT AGT TTT TCA ACT TAT AGA     2277<br>
Gly He Leu Lys Gin Asn Leu Gin Leu Asp Ser Phe Ser Thr Tyr Arg<br>
1495	1500	1505<br>
GTG TAT TTC TCT GTG TCC GGA GAT GCT AAT GTA AGG ATT AGA AAT TCT     2325<br>
Val Tyr Phe Ser Val Ser Gly Asp Ala Asn Val Arg He Arg Asn Ser<br>
1510	1515	1520	1525<br>
AGG GAA GTG TTA TTT GAA AAA AGA TAT ATG AGC GGT GCT AAA GAT GTT     2373<br>
Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser Gly Ala Lys Asp Val<br>
1530	1535	1540<br>
TCT GAA ATG TTC ACT ACA AAA TTT GAG AAA GAT AAC TTC TAT ATA GAG     2421 Ser Glu Met Phe Thr Thr Lys Phe Glu Lys Asp Asn Phe Tyr He Glu<br>
1545	1550	1555<br>
CTT TCT CAA GGG AAT AAT TTA TAT GGT GGT CCT ATT GTA CAT TTT TAG     2469<br>
Leu Ser Gin Gly Asn Asn Leu Tyr Gly Gly Pro lie Val His Phe Tyr<br>
1560	1565	1570<br>
GAT GTC TCT ATT AAG TAAGATCGGG ATCTAATATT AACAGTTTTT AGAAGCTAAT     2524 Asp Val Ser He Lys<br>
1575<br>
TCTTGTATAA TGTCCTTGAT TATGGAAAAA CACAATTTTG TTTGCTAAGA TGTATATATA   2584<br>
GCTCACTCAT TAAAAGGCAA TCAAGCTT	2612<br>
(2) INFORMATION FOR SEQ ID NO:32:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 789 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:32:<br>
Met Asn Lys Asn Asn Thr Lys Leu Ser Thr Arg Ala Leu Pro Ser Phe<br>
15	10	15<br>
He Asp Tyr Phe Asn Gly He Tyr Gly Phe Ala Thr Gly He Lys Asp<br>
20	25	30<br>
He Met Asn Met He Phe Lys Thr Asp Thr Gly Gly Asp Leu Thr Leu<br>
35	40	45<br>
Asp Glu He Leu Lys Asn Gin Gin Leu Leu Asn Asp He Ser Gly Lys<br>
50	55	60<br>
Leu Asp Gly Val Asn Gly Ser Leu Asn Asp Leu He Ala Gin Gly Asn<br>
65	70	75	80<br>
Leu Asn Thr Glu Leu Ser Lys Glu He Leu Lys He Ala Asn Glu Gin<br>
85	90	95<br>
Asn Gin Val Leu Asn Asp Val Asn Asn Lys Leu Asp Ala He Asn Thr<br>
100	105	110<br>
Met Leu Arg Val Tyr Leu Pro Lys He Thr Ser Met Leu Ser Asp Val<br>
115	120	125<br>
Met Lys Gin Asn Tyr Ala Leu Ser Leu Gin He Glu Tyr Leu Ser Lys<br>
130	135	140<br>
Gin Leu Gin Glu He Ser Asp Lys Leu Asp He He Asn Val Asn Val<br>
145	150	155	160<br>
Leu lie Asn Ser Thr Leu Thr Glu He Thr Pro Ala Tyr Gin Arg lie<br>
165	170	175<br>
Lys Tyr Val Asn Glu Lys Phe Glu Glu Leu Thr Phe Ala Thr Glu Thr<br>
180	185	190<br>
Ser Ser Lys Val Lys Lys Asp Gly Ser Pro Ala Asp lie Arg Asp Glu<br>
195	200	205<br>
Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val Thr Lys Asn Asp Val<br>
210	215	220<br>
Asp Gly Phe Glu Phe Tyr Leu Asn Thr Phe His Asp Val Met Val Gly<br>
225	230	235	240<br>
Asn Asn Leu Phe Gly Arg Ser Ala Leu Lys Thr Ala Ser Glu Leu He<br>
245	250	255<br>
Thr Lys Glu Asn Val Lys Thr Ser Gly Ser Glu Val Gly Asn Val Tyr<br>
260	265	270<br>
Asn Phe Leu He Val Leu Thr Ala Leu Gin Ala Lys Ala Phe Leu Thr<br>
275	280	285<br>
Leu Thr Pro Cys Arg Lys Leu Leu Gly Leu Ala Asp He Asp Tyr Thr<br>
290	295	300<br>
Ser He Met Asn Glu His Leu Asn Lys Glu Lys Glu Glu Phe Arg Val<br>
305	310	315	320<br>
Asn He Leu Pro Thr Leu Ser Asn Thr Phe Ser Asn Pro Asn Tyr Ala<br>
325	330	335<br>
Lys Val Lys Gly Ser Asp Glu Asp Ala Lys Met He Val Glu Ala Lys<br>
340	345	350<br>
Pro Gly His Ala Leu He Gly Phe Glu He Ser Asn Asp Ser He Thr<br>
355	360	365<br>
Val Leu Lys Val Tyr Glu Ala Lys Leu Lys Gin Asn Tyr Gin Val Asp<br>
370	375	380<br>
Lys Asp Ser Leu Ser Glu Val He Tyr Gly Asp Met Asp Lys Leu Leu<br>
385	390	395	400<br>
Cys Pro Asp Gin Ser Gly Gin He Tyr Tyr Thr Asn Asn He Val Phe<br>
405	410	415<br>
Pro Asn Glu Tyr Val He Thr Lys He Asp Phe Thr Lys Lys Met Lys<br>
420	425	430<br>
Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr Asp Ser Ser Thr Gly<br>
435	440	445<br>
Glu lie Asp Leu Asn Lys Lys Lys Val Glu Ser Ser Glu Ala Glu Tyr<br>
450	455	460<br>
Arg Thr Leu Ser Ala Asn Asp Asp Gly Val Tyr Met Pro Leu Gly Val<br>
465	470	475	480<br>
He Ser Glu Thr Phe Leu Thr Pro He Asn Gly Phe Gly Leu Gin Ala<br>
485	490	495<br>
Asp Glu Asn Ser Arg Leu He Thr Leu Thr Cys Lys Ser Tyr Leu Arg<br>
500	505	510<br>
Glu Leu Leu Leu Ala Thr Asp Leu Ser Asn Lys Glu Thr Lys Leu He<br>
515	520	525<br>
Val Pro Pro Ser Gly Phe He Ser Asn He Val Glu Asn Gly Ser He<br>
530	535	540<br>
Glu Glu Asp Asn Leu Glu Pro Trp Lys Ala Asn Asn Lys Asn Ma Tyr<br>
545	550	555	560<br>
Val Asp His Thr Gly Gly Val Asn Gly Thr Lys Ala Leu Tyr Val His<br>
565	570	575<br>
Lys Asp Gly Gly He Ser Gin Phe He Gly Asp Lys Leu Lys Pro Lys<br>
580	585	590<br>
Thr Glu Tyr Val He Gin Tyr Thr Val Lys Gly Lys Pro Ser He His<br>
595	600	605<br>
Leu Lys Asp Glu Asn Thr Gly Tyr He His Tyr Glu Asp Thr Asn Asn<br>
610	615	620<br>
Asn Leu Glu Asp Tyr Gin Thr He Asn Lys Arg Phe Thr Thr Gly Thr<br>
625	630	635	640<br>
Asp Leu Lys Gly Val Tyr Leu He Leu Lys Ser Gin Asn Gly Asp Glu<br>
645	650	655<br>
Ala Trp Gly Asp Asn Phe He He Leu Glu He Ser Pro Ser Glu Lys<br>
660	665	670<br>
Leu Leu Ser Pro Glu Leu He Asn Thr Asn Asn Trp Thr Ser Thr Gly<br>
675	680	685<br>
Ser Thr Asn He Ser Gly Asn Thr Leu Thr Leu Tyr Gin Gly Gly Arg<br>
690	695	700<br>
Gly He Leu Lys Gin Asn Leu Gin Leu Asp Ser Phe Ser Thr Tyr Arg<br>
705	710	715	720<br>
Val Tyr Phe Ser Val Ser Gly Asp Ala Asn Val Arg He Arg Asn Ser<br>
725	730	735<br>
Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser Gly Ala Lys Asp Val<br>
740	745	750<br>
Ser Glu Met Phe Thr Thr Lys Phe Glu Lys Asp Asn Phe Tyr lie Glu<br>
755	760	765<br>
Leu Ser Gin Gly Asn Asn Leu Tyr Gly Gly Pro lie Val His Phe Tyr<br>
770	775	780<br>
Asp Val Ser lie Lys 785<br>
(2) INFORMATION FOR SEQ ID NO:33:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 30 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "forward primer used to make pCIB5526"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:33:<br>
GGATCCACCA TGAAGACCAA CCAGATCAGC	30<br>
(2) INFORMATION FOR SEQ ID NO:34:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 15 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "reverse primer used to make pCIB5526"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:34:<br>
AAGCTTCAGC TCCTT	15<br>
(2) INFORMATION FOR SEQ ID NO:35:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 2576 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 9..2564<br>
(D) OTHER INFORMATION: /note= "Maize optimized sequence encoding VIPlA(a) with the Bacillus secretion signal removed as contained in pCIB5526"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:35:<br>
GATCCACC ATG AAG ACC AAC CAG ATC AGC ACC ACC CAG AAG AAC CAG CAG      50<br>
Met Lys Thr Asn Gin lie Ser Thr Thr Gin Lys Asn Gin Gin<br>
825	830	835<br>
AAG GAG ATG GAC CGC AAG GGC CTG CTG GGC TAG TAG TTC AAG GGC AAG       98<br>
Lys Glu Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys<br>
840	845	850<br>
GAC TTC AGC AAC CTG ACC ATG TTC GCC CCC ACG CGT GAC AGC ACC CTG      146<br>
Asp Phe Ser Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu<br>
855	860	865<br>
ATC TAG GAC CAG CAG ACC GCC AAC AAG CTG CTG GAC AAG AAG CAG CAG      194<br>
lie Tyr Asp Gin Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin<br>
870	875	880<br>
GAG TAG CAG AGC ATC CGC TGG ATC GGC CTG ATC CAG AGC AAG GAG ACC      242<br>
Glu Tyr Gin Ser lie Arg Trp lie Gly Leu lie Gin Ser Lys Glu Thr<br>
885	890	895<br>
GGC GAC TTC ACC TTC AAC CTG AGC GAG GAC GAG CAG GCC ATC ATC GAG      290<br>
Gly Asp Phe Thr Phe Asn Leu Ser Glu Asp Glu Gin Ala lie lie Glu<br>
900	905	910	915<br>
ATC AAC GGC AAG ATC ATC AGC AAC AAG GGC AAG GAG AAG CAG GTG GTG      338<br>
lie Asn Gly Lys lie lie Ser Asn Lys Gly Lys Glu Lys Gin Val Val<br>
920	925	930<br>
CAC CTG GAG AAG GGC AAG CTG GTG CCC ATC AAG ATC GAG TAG CAG AGC      386<br>
His Leu Glu Lys Gly Lys Leu Val Pro lie Lys He Glu Tyr Gin Ser<br>
935	940	945<br>
GAC ACC AAG TTC AAC ATC GAC AGC AAG ACC TTC AAG GAG CTG AAG CTT      434<br>
Asp Thr Lys Phe Asn lie Asp Ser Lys Thr Phe Lys Glu Leu	Lys Leu<br>
950              955              960<br>
TTC AAG ATC GAC AGC CAG AAC GAG CCC CAG GAG GTG GAG GAG	GAG GAG<br>
Phe Lys He Asp Ser Gin Asn Gin Pro Gin Gin Val Gin Gin	Asp Glu<br>
965              970              975<br>
CTG CGC AAC CCC GAG TTC AAC AAG AAG GAG AGC CAG GAG TTC	CTG GCC<br>
Leu Arg Asn Pro Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe	Leu Ala<br>
980              985              990	995<br>
482<br>
530<br>
AAG CCC AGC AAG ATC AAC CTG TTC ACC CAG CAG ATG AAG CGC GAG ATC      578<br>
Lys Pro Ser Lys lie Asn Leu Phe Thr Gin Gin Met Lys Arg Glu lie<br>
1000	1005	1010<br>
GAC GAG GAC ACC GAC ACC GAC GGC GAC AGC ATC CCC GAC CTG TGG GAG      626<br>
Asp Glu Asp Thr Asp Thr Asp Gly Asp Ser lie Pro Asp Leu Trp Glu<br>
1015	1020	1025<br>
GAG AAC GGC TAC ACC ATC CAG AAC CGC ATC GCC GTG AAG TGG GAC GAC      674<br>
Glu Asn Gly Tyr Thr lie Gin Asn Arg lie Ala Val Lys Trp Asp Asp<br>
1030	1035	1040<br>
AGC CTG GCT AGC AAG GGC TAC ACC AAG TTC GTG AGC AAC CCC CTG GAG      722<br>
Ser Leu Ala Ser Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu<br>
1045	1050	1055<br>
AGC CAC ACC GTG GGC GAC CCC TAC ACC GAC TAC GAG AAG GCC GCC CGC      770<br>
Ser His Thr Val Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg<br>
1060	1065	1070	1075<br>
GAC CTG GAC CTG AGC AAC GCC AAG GAG ACC TTC AAC CCC CTG GTG GCC      818<br>
Asp Leu Asp Leu Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala<br>
1080	1085	1090<br>
GCC TTC CCC AGC GTG AAC GTG AGC ATG GAG AAG GTG ATC CTG AGC CCC     866<br>
Ala Phe Pro Ser Val Asn Val Ser Met Glu Lys Val lie Leu Ser Pro<br>
1095	1100	1105<br>
AAC GAG AAC CTG AGC AAC AGC GTG GAG AGC CAC TCG AGC ACC AAC TGG      914<br>
Asn Glu Asn Leu Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp<br>
1110	1115	1120<br>
AGC TAC ACC AAC ACC GAG GGC GCC AGC GTG GAG GCC GGC ATC GGT CCC      962<br>
Ser Tyr Thr Asn Thr Glu Gly Ala Ser Val Glu Ala Gly He Gly Pro<br>
1125	1130	1135<br>
AAG GGC ATC AGC TTC GGC GTG AGC GTG AAC TAC CAG CAC AGC GAG ACC     1010<br>
Lys Gly He Ser Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr<br>
1140	1145	1150	1155<br>
GTG GCC CAG GAG TGG GGC ACC AGC ACC GGC AAC ACC AGC CAG TTC AAC     1058<br>
Val Ala Gin Glu Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn<br>
1160	1165	1170<br>
ACC GCC AGC GCC GGC TAG CTG AAC GCC AAC GTG CGC TAG AAC AAC GTG     1106<br>
Thr Ala Ser Ala Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val<br>
1175	1180	1185<br>
GGC ACC GGC GCC ATC TAG GAC GTG AAG CCC ACC ACC AGC TTC GTG CTG     1154<br>
Gly Thr Gly Ala He Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu<br>
1190	1195	1200<br>
AAC AAC GAC ACC ATC GCC ACC ATC ACC GCC AAG TCG AAT TCC ACC GCC     1202<br>
Asn Asn Asp Thr He Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala<br>
1205	1210	1215<br>
CTG AAC ATC AGC CCC GGC GAG AGC TAG CCC AAG AAG GGC CAG AAC GGC     1250<br>
Leu Asn He Ser Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly<br>
1220	1225	1230	1235<br>
ATC GCC ATC ACC AGC ATG GAC GAC TTC AAC AGC CAC CCC ATC ACC CTG     1298<br>
He Ala He Thr Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu<br>
1240	1245	1250<br>
AAC AAG AAG CAG GTG GAC AAC CTG CTG AAC AAC AAG CCC ATG ATG CTG     1346<br>
Asn Lys Lys Gin Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu<br>
1255	1260	1265<br>
GAG ACC AAC CAG ACC GAC GGC GTC TAG AAG ATC AAG GAC ACC CAC GGC     1394<br>
Glu Thr Asn Gin Thr Asp Gly Val Tyr Lys He Lys Asp Thr His Gly<br>
1270	1275	1280<br>
AAC ATC GTG ACG GGC GGC GAG TGG AAC GGC GTG ATC CAG CAG ATC AAG     1442<br>
Asn He Val Thr Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys<br>
1285	1290	1295<br>
GCC AAG ACC GCC AGC ATC ATC GTC GAC GAC GGC GAG CGC GTG GCC GAG     1490<br>
Ala Lys Thr Ala Ser He He Val Asp Asp Gly Glu Arg Val Ala Glu<br>
1300	1305	1310	1315<br>
AAG CGC GTG GCC GCC AAG GAC TAG GAG AAC CCC GAG GAC AAG ACC CCC     1538<br>
Lys Arg Val Ala Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro<br>
1320	1325	1330<br>
AGC CTG ACC CTG AAG GAC GCC CTG AAG CTG AGC TAG CCC GAC GAG ATC     1586<br>
Ser Leu Thr Leu Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He<br>
1335	1340	1345<br>
AAG GAG ATC GAG GGC TTG CTG TAG TAG AAG AAC AAG CCC ATC TAG GAG     1634<br>
Lys Glu He Glu Gly Leu Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu<br>
1350	1355	1360<br>
AGC AGC GTG ATG ACC TAT CTA GAC GAG AAC ACC GCC AAG GAG GTG ACC     1682<br>
Ser Ser Val Met Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr<br>
1365	1370	1375<br>
AAG CAG CTG AAC GAC ACC ACC GGC AAG TTC AAG GAC GTG AGC CAC CTG     1730<br>
Lys Gin Leu Asn Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu<br>
1380	1385	1390	1395<br>
TAG GAC GTG AAG CTG ACC CCC AAG ATG AAC GTG ACC ATC AAG CTG AGC     1778<br>
Tyr Asp Val Lys Leu Thr Pro Lys Met Asn Val Thr lie Lys Leu Ser<br>
1400	1405	1410<br>
ATC CTG TAG GAC AAC GCC GAG AGC AAC GAC AAC AGC ATC GGC AAG TGG     1826<br>
lie Leu Tyr Asp Asn Ala Glu Ser Asn Asp Asn Ser lie Gly Lys Trp<br>
1415	1420	1425<br>
ACC AAC ACC AAC ATC GTG AGC GGC GGC AAC AAC GGC AAG AAG CAG TAG     1874<br>
Thr Asn Thr Asn He Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr<br>
1430	1435	1440<br>
AGC AGC AAC AAC CCC GAC GCC AAC CTG ACC CTG AAC ACC GAC GCC CAG     1922<br>
Ser Ser Asn Asn Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin<br>
1445	1450	1455<br>
GAG AAG CTG AAC AAG AAC CGC GAC TAG TAG ATC AGC CTG TAG ATG AAG     1970<br>
Glu Lys Leu Asn Lys Asn Arg Asp Tyr Tyr He Ser Leu Tyr Met Lys<br>
1460	1465	1470	1475<br>
AGC GAG AAG AAC ACC CAG TGC GAG ATC ACC ATC GAC GGC GAG ATA TAG     2018<br>
Ser Glu Lys Asn Thr Gin Cys Glu He Thr He Asp Gly Glu He Tyr<br>
1480	1485	1490<br>
CCC ATC ACC ACC AAG ACC GTG AAC GTG AAC AAG GAC AAC TAG AAG CGC     2066<br>
Pro He Thr Thr Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg<br>
1495	1500	1505<br>
CTG GAC ATC ATC GCC CAC AAC ATC AAG AGC AAC CCC ATC AGC AGC CTG     2114<br>
Leu Asp He He Ala His Asn He Lys Ser Asn Pro He Ser Ser Leu<br>
1510	1515	1520<br>
CAC ATC AAG ACC AAC GAC GAG ATC ACC CTG TTC TGG GAC GAC ATA TCG     2162<br>
His He Lys Thr Asn Asp Glu He Thr Leu Phe Trp Asp Asp He Ser<br>
1525	1530	1535<br>
ATT ACC GAC GTC GCC AGC ATC AAG CCC GAG AAC CTG ACC GAC AGC GAG     2210<br>
He Thr Asp Val Ala Ser He Lys Pro Glu Asn Leu Thr Asp Ser Glu<br>
1540	1545	1550	1555<br>
ATC AAG CAG ATA TAC ACT CGC TAG GGC ATC AAG CTG GAG GAC GGC ATC     2258<br>
He Lys Gin He Tyr Ser Arg Tyr Gly He Lys Leu Glu Asp Gly He<br>
1560	1565	1570<br>
CTG ATC GAC AAG AAA GGC GGC ATC CAC TAC GGC GAG TTC ATC AAC GAG     2306<br>
Leu He Asp Lys Lys Gly Gly He His Tyr Gly Glu Phe He Asn Glu<br>
1575	1580	1585<br>
GCC AGC TTC AAC ATC GAG CCC CTG CAG AAC TAC GTG ACC AAG TAC GAG     2354<br>
Ala Ser Phe Asn He Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu<br>
1590	1595	1600<br>
GTG ACC TAC AGC AGC GAG CTG GGC CCC AAC GTG AGC GAC ACC CTG GAG     2402 Val Thr Tyr Ser Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu<br>
1605	1610	1615<br>
AGC GAC AAG ATT TAG AAG GAC GGC ACC ATC AAG TTC GAC TTC ACC AAG     2450<br>
Ser Asp Lys lie Tyr Lys Asp Gly Thr lie Lys Phe Asp Phe Thr Lys<br>
1620	1625	1630	1635<br>
TAG AGC AAG AAC GAG GAG GGC CTG TTC TAG GAC AGC GGC CTG AAC TGG     2498<br>
Tyr Ser Lys Asn Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp<br>
1640	1645	1650<br>
GAC TTC AAG ATC AAC GCC ATC ACC TAG GAC GGC AAG GAG ATG AAC GTG     2546<br>
Asp Phe Lys lie Asn Ala lie Thr Tyr Asp Gly Lys Glu Met Asn Val<br>
1655	1660	1665<br>
TTC CAC CGC TAG AAC AAG TAGATCTGAG CT	2576<br>
Phe His Arg Tyr Asn Lys 1670<br>
(2) INFORMATION FOR SEQ ID NO:36:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 852 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:36:<br>
Met Lys Thr Asn Gin He Ser Thr Thr Gin Lys Asn Gin Gin Lys Glu<br>
15	10	15<br>
Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr Phe Lys Gly Lys Asp Phe<br>
20	25	30<br>
Ser Asn Leu Thr Met Phe Ala Pro Thr Arg Asp Ser Thr Leu lie Tyr<br>
35	40	45<br>
Asp Gin Gin Thr Ala Asn Lys Leu Leu Asp Lys Lys Gin Gin Glu Tyr<br>
50	55	60<br>
Gin Ser He Arg Trp He Gly Leu He Gin Ser Lys Glu Thr Gly Asp<br>
65	70	75	80<br>
Phe Thr Phe Asn Leu Ser Glu Asp Glu Gin Ala He He Glu He Asn<br>
85	90	95<br>
Gly Lys He He Ser Asn Lys Gly Lys Glu Lys Gin Val Val His Leu<br>
100	105	110<br>
Glu Lys Gly Lys Leu Val Pro He Lys He Glu Tyr Gin Ser Asp Thr<br>
115	120	125<br>
Lys Phe Asn He Asp Ser Lys Thr Phe Lys Glu Leu Lys Leu Phe Lys<br>
130<br>
135<br>
140<br>
lie Asp Ser Gin Asn Gin Pro Gin Gin Val Gin Gin Asp Glu Leu Arg<br>
145	150	155	160<br>
Asn Pro Glu Phe Asn Lys Lys Glu Ser Gin Glu Phe Leu Ala Lys Pro<br>
165	170	175<br>
Ser Lys lie Asn Leu Phe Thr Gin Gin Met Lys Arg Glu lie Asp Glu<br>
180	185	190<br>
Asp Thr Asp Thr Asp Gly Asp Ser lie Pro Asp Leu Trp Glu Glu Asn<br>
195	200	205<br>
Gly Tyr Thr lie Gin Asn Arg lie Ala Val Lys Trp Asp Asp Ser Leu<br>
210	215	220<br>
Ala Ser Lys Gly Tyr Thr Lys Phe Val Ser Asn Pro Leu Glu Ser His<br>
225	230	235	240<br>
Thr Val Gly Asp Pro Tyr Thr Asp Tyr Glu Lys Ala Ala Arg Asp Leu<br>
245	250	255<br>
Asp Leu Ser Asn Ala Lys Glu Thr Phe Asn Pro Leu Val Ala Ala Phe<br>
260	265	270<br>
Pro Ser Val Asn Val Ser Met Glu Lys Val lie Leu Ser Pro Asn Glu<br>
275	280	285<br>
Asn Leu Ser Asn Ser Val Glu Ser His Ser Ser Thr Asn Trp Ser Tyr<br>
290	295	300<br>
Thr Asn Thr Glu Gly Ala Ser Val Glu Ala Gly lie Gly Pro Lys Gly<br>
305	310	315	320<br>
He Ser Phe Gly Val Ser Val Asn Tyr Gin His Ser Glu Thr Val Ala<br>
325	330	335<br>
Gin Glu Trp Gly Thr Ser Thr Gly Asn Thr Ser Gin Phe Asn Thr Ala<br>
340	345	350<br>
Ser Ala Gly Tyr Leu Asn Ala Asn Val Arg Tyr Asn Asn Val Gly Thr<br>
355	360	365<br>
Gly Ala He Tyr Asp Val Lys Pro Thr Thr Ser Phe Val Leu Asn Asn<br>
370	375	380<br>
Asp Thr He Ala Thr He Thr Ala Lys Ser Asn Ser Thr Ala Leu Asn<br>
385	390	395	400<br>
He Ser Pro Gly Glu Ser Tyr Pro Lys Lys Gly Gin Asn Gly He Ala<br>
405	410	415<br>
He Thr Ser Met Asp Asp Phe Asn Ser His Pro He Thr Leu Asn Lys<br>
420	425	430<br>
Lys Gin Val Asp Asn Leu Leu Asn Asn Lys Pro Met Met Leu Glu Thr<br>
435	440	445<br>
Asn Gin Thr Asp Gly Val Tyr Lys He Lys Asp Thr His Gly Asn He<br>
450	455	460<br>
Val Thr Gly Gly Glu Trp Asn Gly Val He Gin Gin He Lys Ala Lys<br>
465	470	475	480<br>
Thr Ala Ser He lie Val Asp Asp Gly Glu Arg Val Ala Glu Lys Arg<br>
485	490	495<br>
Val Ala Ala Lys Asp Tyr Glu Asn Pro Glu Asp Lys Thr Pro Ser Leu<br>
500	505	510<br>
Thr Leu Lys Asp Ala Leu Lys Leu Ser Tyr Pro Asp Glu He Lys Glu<br>
515	520	525<br>
He Glu Gly Leu .Leu Tyr Tyr Lys Asn Lys Pro He Tyr Glu Ser Ser<br>
530	535	540<br>
Val Met Thr Tyr Leu Asp Glu Asn Thr Ala Lys Glu Val Thr Lys Gin<br>
545	550	555	560<br>
Leu Asn Asp Thr Thr Gly Lys Phe Lys Asp Val Ser His Leu Tyr Asp<br>
565	570	575<br>
Val Lys Leu Thr Pro Lys Met Asn Val Thr He Lys Leu Ser He Leu<br>
580	585	590<br>
Tyr Asp Asn Ala Glu Ser Asn Asp Asn Ser He Gly Lys Trp Thr Asn<br>
595	600	605<br>
Thr Asn He Val Ser Gly Gly Asn Asn Gly Lys Lys Gin Tyr Ser Ser<br>
610	615	620<br>
Asn Asn Pro Asp Ala Asn Leu Thr Leu Asn Thr Asp Ala Gin Glu Lys<br>
625	630	635	640<br>
Leu Asn Lys Asn Arg Asp Tyr Tyr He Ser Leu Tyr Met Lys Ser Glu<br>
645	650	655<br>
Lys Asn Thr Gin Cys Glu He Thr He Asp Gly Glu He Tyr Pro He<br>
660	665	670<br>
Thr Thr Lys Thr Val Asn Val Asn Lys Asp Asn Tyr Lys Arg Leu Asp<br>
675	680	685<br>
He He Ala His Asn He Lys Ser Asn Pro He Ser Ser Leu His He<br>
690	695	700<br>
Lys Thr Asn Asp Glu He Thr Leu Phe Trp Asp Asp He Ser He Thr<br>
705	710	715	720<br>
Asp Val Ala Ser He Lys Pro Glu Asn Leu Thr Asp Ser Glu He Lys<br>
725	730	735<br>
Gin He Tyr Ser Arg Tyr Gly He Lys Leu Glu Asp Gly He Leu He<br>
740	745	750<br>
Asp Lys Lys Gly Gly He His Tyr Gly Glu Phe He Asn Glu Ala Ser<br>
755	760	765<br>
Phe Asn He Glu Pro Leu Gin Asn Tyr Val Thr Lys Tyr Glu Val Thr<br>
770	775	780<br>
Tyr Ser Ser Glu Leu Gly Pro Asn Val Ser Asp Thr Leu Glu Ser Asp<br>
785	790	795	800<br>
Lys He Tyr Lys Asp Gly Thr He Lys Phe Asp Phe Thr Lys Tyr Ser<br>
805	810	815<br>
Lys Asn Glu Gin Gly Leu Phe Tyr Asp Ser Gly Leu Asn Trp Asp Phe<br>
820	825	830<br>
Lys He Asn Ala He Thr Tyr Asp Gly Lys Glu Met Asn Val Phe His<br>
835	840	845<br>
Arg Tyr Asn Lys 850<br>
(2) INFORMATION FOR SEQ ID NO:37:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 32 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "forward primer used to make pCIB5527"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:37:<br>
GGATCCACCA TGCTGCAGAA CCTGAAGATC AC	32<br>
(2) INFORMATION FOR SEQ ID NO:38:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 18 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "reverse primer used to make pCIB5527"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:38:<br>
AAGCTTCCAC TCCTTCTC	18<br>
(2) INFORMATION FOR SEQ ID NO:39:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1241 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 9..1238<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence encoding VIP2A(a) with the Bacillus secretion signal removed as contained in pCIB5527"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:39:<br>
GATCCACC ATG CTG GAG AAC CTG AAG ATC ACC GAG AAG GTG GAG GAC TTC      50<br>
Met Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu Asp Phe<br>
855	860	865<br>
AAG GAG GAC AAG GAG AAG GCC AAG GAG TGG GGC AAG GAG AAG GAG AAG       98<br>
Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys Glu Lys<br>
870	875	880<br>
GAG TGG AAG CTT ACC GCC ACC GAG AAG GGC AAG ATG AAC AAC TTC CTG      146<br>
Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn Phe Leu<br>
885	890	895<br>
GAC AAC AAG AAC GAC ATC AAG ACC AAC TAG AAG GAG ATC ACC TTC AGC      194<br>
Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr Phe Ser<br>
900	905	910<br>
ATA GCC GGC AGC TTC GAG GAC GAG ATC AAG GAC CTG AAG GAG ATC GAC      242<br>
lie Ala Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu lie Asp<br>
915              920	925              930<br>
AAG ATG TTC GAG AAG ACC AAC	CTG AGC AAC AGC ATC ATC ACC TAG AAG<br>
Lys Met Phe Asp Lys Thr Asn	Leu Ser Asn Ser lie lie Thr Tyr Lys<br>
935	940              945<br>
AAC GTG GAG CCC ACC ACC ATC	GGC TTC AAC AAG AGC CTG ACC GAG GGC<br>
Asn Val Glu Pro Thr Thr lie	Gly Phe Asn Lys Ser Leu Thr Glu Gly<br>
950	955              960<br>
290<br>
338<br><br>
AAC ACC ATC AAC AGC GAC GCC ATG GCC CAG TTC AAG GAG CAG TTC CTG<br>
Asn Thr lie Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin Phe Leu<br>
965	970	975<br>
386<br>
GAC CGC GAC ATC AAG TTC GAC AGC TAG CTG GAC ACC CAC CTG ACC GCC      434<br>
Asp Arg Asp lie Lys Phe Asp Ser Tyr Leu Asp Thr His Leu Thr Ala<br>
980	985	990<br>
CAG CAG GTG AGC AGC AAG GAG CGC GTG ATC CTG AAG GTG ACC GTC CCC      482<br>
Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr Val Pro<br>
995	1000	1005	1010<br>
AGC GGC AAG GGC AGC ACC ACC CCC ACC AAG GCC GGC GTG ATC CTG AAC      530<br>
Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He Leu Asn<br>
1015	1020	1025<br>
AAC AGC GAG TAG AAG ATG CTG ATC GAC AAC GGC TAG ATG GTG CAC GTG      578<br>
Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val His Val<br>
1030	1035	1040<br>
GAC AAG GTG AGC AAG GTG GTG AAG AAG GGC GTG GAG TGC CTC CAG ATC      626<br>
Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu Gin He<br>
1045	1050	1055<br>
GAG GGC ACC CTG AAG AAG AGT CTA GAC TTC AAG AAC GAC ATC AAC GCC      674<br>
Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He Asn Ala<br>
1060	1065	1070<br>
GAG GCC CAC AGC TGG GGC ATG AAG AAC TAG GAG GAG TGG GCC AAG GAC      722<br>
Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala Lys Asp<br>
1075	1080	1085	1090<br>
CTG ACC GAC AGC CAG CGC GAG GCC CTG GAC GGC TAG GCC CGC CAG GAC      770<br>
Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg Gin Asp<br>
1095	1100	1105<br>
TAG AAG GAG ATC AAC AAC TAG CTG CGC AAC CAG GGC GGC AGC GGC AAC      818<br>
Tyr Lys Glu He Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser Gly Asn<br>
1110	1115	1120<br>
GAG AAG CTG GAC GCC CAG ATC AAG AAC ATC AGC GAC GCC CTG GGC AAG      866<br>
Glu Lys Leu Asp Ala Gin He Lys Asn He Ser Asp Ala Leu Gly Lys<br>
1125	1130	1135<br>
AAG CCC ATC CCC GAG AAC ATC ACC GTG TAG CGC TGG TGC GGC ATG CCC      914<br>
Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly Met Pro<br>
1140	1145	1150<br>
GAG TTC GGC TAG CAG ATC AGC GAC CCC CTG CCC AGC CTG AAG GAC TTC      962<br>
Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys Asp Phe<br>
1155	1160	1165	1170<br>
GAG GAG CAG TTC CTG AAC ACC ATC AAG GAG GAC AAG GGC TAG ATG AGC     1010<br>
Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp Lys Gly Tyr Met Ser<br>
1175	1180	1185<br>
ACC AGC CTG AGC AGC GAG CGC CTG GCC GCC TTC GGC AGC CGC AAG ATC     1058<br>
Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg Lys lie<br>
1190	1195	1200<br>
ATC CTG CGC CTG CAG GTG CCC AAG GGC AGC ACT GGT GCC TAG CTG AGC     1106<br>
lie Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr Leu Ser<br>
1205	1210	1215<br>
GCC ATC GGC GGC TTC GCC AGC GAG AAG GAG ATC CTG CTG GAT AAG GAC     1154<br>
Ala lie Gly Gly Phe Ala Ser Glu Lys Glu lie Leu Leu Asp Lys Asp<br>
1220	1225	1230<br>
hGC AAG TAG CAC ATC GAC AAG GTG ACC GAG GTG ATC ATC AAG GGC GTG     1202<br>
Ser Lys Tyr His He Asp Lys Val Thr Glu Val He He Lys Gly Val<br>
1235	1240	1245	1250<br>
AAG CGC TAG GTG GTG GAC GCC ACC CTG CTG ACC AAC TAG	1241<br>
Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
1255	1260<br>
(2) INFORMATION FOR SEQ ID NO:40:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 410 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:40:<br>
Met Leu Gin Asn Leu Lys He Thr Asp Lys Val Glu Asp Phe Lys Glu<br>
15	10	15<br>
Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys Glu Lys Glu Trp<br>
20	25	30<br>
Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn Phe Leu Asp Asn<br>
35	40	45<br>
Lys Asn Asp He Lys Thr Asn Tyr Lys Glu He Thr Phe Ser He Ala<br>
50	55	60<br>
Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu lie Asp Lys Met<br>
65	70	75	80<br>
Phe Asp Lys Thr Asn Leu Ser Asn Ser lie lie Thr Tyr Lys Asn Val<br>
85	90	95<br>
Glu Pro Thr Thr He Gly Phe Asn Lys Ser Leu Thr Glu Gly Asn Thr<br>
100	105	110<br>
He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin Phe Leu Asp Arg<br>
115	120	125<br>
Asp He Lys Phe Asp Ser Tyr Leu Asp Thr His Leu Thr Ala Gin Gin<br>
130	135	140<br>
Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr Val Pro Ser Gly<br>
145	150	155	160<br>
Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He Leu Asn Asn Ser<br>
165	170	175<br>
Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val His Val Asp Lys<br>
180	185	190<br>
Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu Gin He Glu Gly<br>
195	200	205<br>
Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He Asn Ala Glu Ala<br>
210	215	220<br>
His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala Lys Asp Leu Thr<br>
225	230	235	240<br>
Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg Gin Asp Tyr Lys<br>
245	250	255<br>
Glu He Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser Gly Asn Glu Lys<br>
260	265	270<br>
Leu Asp Ala Gin He Lys Asn He Ser Asp Ala Leu Gly Lys Lys Pro<br>
275	280	285<br>
He Pro Glu Asn He Thr Val Tyr Arg Trp Cys Gly Met Pro Glu Phe<br>
290	295	300<br>
Gly Tyr Gin He Ser Asp Pro Leu Pro Ser Leu Lys Asp Phe Glu Glu<br>
305	310	315	320<br>
Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr Met Ser Thr Ser<br>
325	330	335<br>
Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg Lys He He Leu<br>
340	345	350<br>
Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr Leu Ser Ala lie<br>
355	360	365<br>
Gly Gly Phe Ala Ser Glu Lys Glu lie Leu Leu Asp Lys Asp Ser Lys<br>
370	375	380<br>
Tyr His He Asp Lys Val Thr Glu Val He He Lys Gly Val Lys Arg<br>
385	390	395	400<br>
Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
405	410<br>
(2) INFORMATION FOR SEQ ID NO:41:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 72 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "oligonucleotide encoding eukaryotic secretion signal used to construct pCIB5527"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:41:<br>
GGATCCACCA TGGGCTGGAG CTGGATCTTC CTGTTCCTGC TGAGCGGCGC CGCGGGCGTG     60<br>
CACTGCCTGC AG	72<br>
(2) INFORMATION FOR SEQ ID NO:42:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1241 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 9..1238<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence encoding VIP2A(a) with the Bacillus secretion signal removed and the eukaryotic secretion signal inserted as<br>
contained in pCIB5528"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:42:<br>
GATCCACC ATG CTG CAG AAC CTG AAG ATC ACC GAC AAG GTG GAG GAC TTC      50<br>
Met Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu Asp Phe<br>
415	420<br>
AAG GAG GAC AAG GAG AAG GCC AAG GAG TGG GGC AAG GAG AAG GAG AAG       98<br>
Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys Glu Lys<br>
425	430	435	440<br>
GAG TGG AAG CTT ACC GCC ACC GAG AAG GGC AAG ATG AAC AAC TTC CTG      146<br>
Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn Phe Leu<br>
445	450	455<br>
GAC AAC AAG AAC GAC ATC AAG ACC AAC TAG AAG GAG ATC ACC TTC AGC      194<br>
Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr Phe Ser<br>
460	465	470<br>
ATA GCC GGC AGC TTC GAG GAC GAG ATC AAG GAC CTG AAG GAG ATC GAC      242<br>
He Ala Gly Ser Phe Glu Asp Glu He Lys Asp Leu Lys Glu He Asp<br>
475	480	485<br>
AAG ATG TTC GAC AAG ACC AAC CTG AGC AAC AGC ATC ATC ACC TAG AAG      290<br>
Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser He He Thr Tyr Lys<br>
490	495	500<br>
AAC GTG GAG CCC ACC ACC ATC GGC TTC AAC AAG AGC CTG ACC GAG GGC      338<br>
Asn Val Glu Pro Thr Thr He Gly Phe Asn Lys Ser Leu Thr Glu Gly<br>
505	510	515	520<br>
AAC ACC ATC AAC AGC GAC GCC ATG GCC CAG TTC AAG GAG CAG TTC CTG      386<br>
Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin Phe Leu<br>
525	530	535<br>
GAC CGC GAC ATC AAG TTC GAC AGC TAG CTG GAC ACC CAC CTG ACC GCC      434<br>
Asp Arg Asp He Lys Phe Asp Ser Tyr Leu Asp Thr His Leu Thr Ala<br>
540	545	550<br>
CAG CAG GTG AGC AGC AAG GAG CGC GTG ATC CTG AAG GTG ACC GTC CCC      482<br>
Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr Val Pro<br>
555	560	565<br>
AGC GGC AAG GGC AGC ACC ACC CCC ACC AAG GCC GGC GTG ATC CTG AAC      530<br>
Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He Leu Asn<br>
570	575	580<br>
AAC AGC GAG TAG AAG ATG CTG ATC GAC AAC GGC TAG ATG GTG CAC GTG      578<br>
Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val His Val<br>
585	590	595	600<br>
GAC AAG GTG AGC AAG GTG GTG AAG AAG GGC GTG GAG TGC CTC CAG ATC      626 Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu Gin He<br>
605	610	615<br>
GAG GGC ACC CTG AAG AAG ACT CTA GAG TTC AAG AAC GAG ATC AAC GCC      674<br>
Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp lie Asn Ala<br>
620	625	630<br>
GAG GCC CAC AGC TGG GGC ATG AAG AAC TAG GAG GAG TGG GCC AAG GAG      722<br>
Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala Lys Asp<br>
635	640	645<br>
CTG ACC GAG AGC CAG CGC GAG GCC CTG GAG GGC TAG GCC CGC CAG GAC      770<br>
Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg Gin Asp<br>
650	655	660<br>
TAG AAG GAG ATC AAC AAC TAG CTG CGC AAC CAG GGC GGC AGC GGC AAC      818<br>
Tyr Lys Glu He Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser Gly Asn<br>
665	670	675	680<br>
GAG AAG CTG GAC GCC CAG ATC AAG AAC ATC AGC GAC GCC CTG GGC AAG      866<br>
Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala Leu Gly Lys<br>
685	690	695<br>
AAG CCC ATC CCC GAG AAC ATC ACC GTG TAG CGC TGG TGC GGC ATG CCC      914<br>
Lys Pro He Pro Glu Asn He Thr Val Tyr Arg Trp Cys Gly Met Pro<br>
700	705	710<br>
GAG TTC GGC TAG CAG ATC AGC GAC CCC CTG CCC AGC CTG AAG GAC TTC      962<br>
Glu Phe Gly Tyr Gin He Ser Asp Pro Leu Pro Ser Leu Lys Asp Phe<br>
715	720	725<br>
GAG GAG CAG TTC CTG AAC ACC ATC AAG GAG GAC AAG GGC TAG ATG AGC     1010<br>
Glu Glu Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr Met Ser<br>
730	735	740<br>
ACC AGC CTG AGC AGC GAG CGC CTG GCC GCC TTC GGC AGC CGC AAG ATC     1058<br>
Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg Lys He<br>
745	750	755	760<br>
ATC CTG CGC CTG CAG GTG CCC AAG GGC AGC ACT GGT GCC TAG CTG AGC     1106<br>
He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr Leu Ser<br>
765	770	775<br>
GCC ATC GGC GGC TTC GCC AGC GAG AAG GAG ATC CTG CTG GAT AAG GAC     1154<br>
Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu Asp Lys Asp<br>
780	785	790<br>
AGC AAG TAG CAC ATC GAC AAG GTG ACC GAG GTG ATC ATC AAG GGC GTG     1202<br>
Ser Lys Tyr His He Asp Lys Val Thr Glu Val He He Lys Gly Val<br>
795	800	805<br>
AAG CGC TAG GTG GTG GAC GCC ACC CTG CTG ACC AAC TAG	1241<br>
Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
810	815	820<br>
(2) INFORMATION FOR SEQ ID NO:43:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 410 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:43:<br>
Met Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu Asp Phe Lys Glu<br>
15	10	15<br>
Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys Glu Lys Glu Trp<br>
20	25	30<br>
Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn Phe Leu Asp Asn<br>
35	40	45<br>
Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr Phe Ser lie Ala<br>
50	55	60<br>
Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu lie Asp Lys Met<br>
65	70	75	80<br>
Phe Asp Lys Thr Asn Leu Ser Asn Ser lie lie Thr Tyr Lys Asn Val<br>
85	90	95<br>
Glu Pro Thr Thr lie Gly Phe Asn Lys Ser Leu Thr Glu Gly Asn Thr<br>
100	105	110<br>
lie Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin Phe Leu Asp Arg<br>
115	120	125<br>
Asp lie Lys Phe Asp Ser Tyr Leu Asp Thr His Leu Thr Ala Gin Gin<br>
130	135	140<br>
Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr Val Pro Ser Gly<br>
145	150	155	160<br>
Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val lie Leu Asn Asn Ser<br>
165	170	175<br>
Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val His Val Asp Lys<br>
180	185	190<br>
Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu Gin He Glu Gly<br>
195	200	205<br>
Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He Asn Ala Glu Ala<br>
210	215	220<br>
His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala Lys Asp Leu Thr<br>
225	230	235	240<br>
Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg Gin Asp Tyr Lys<br>
245	250	255<br>
Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser Gly Asn Glu Lys<br>
260	265	270<br>
Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala Leu Gly Lys Lys Pro<br>
275	280	285<br>
lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly Met Pro Glu Phe<br>
290	295	300<br>
Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys Asp Phe Glu Glu<br>
305	310	315	320<br>
Gin Phe Leu Asn Thr He Lys Glu Asp Lys Gly Tyr Met Ser Thr Ser<br>
325	330	335<br>
Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg Lys He He Leu<br>
340	345	350<br>
Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr Leu Ser Ala He<br>
355	360	365<br>
Gly Gly Phe Ala Ser Glu Lys Glu He Leu Leu Asp Lys Asp Ser Lys<br>
370	375	380<br>
Tyr His He Asp Lys Val Thr Glu Val He He Lys Gly Val Lys Arg<br>
385	390	395	400<br>
Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
405	410<br>
(2) INFORMATION FOR SEQ ID NO:44:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 86 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "oligonucleotide encoding vacuolar targetting peptide used to construct pCIB5533"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:44: CCGCGGGCGT GCACTGCCTC AGCAGCAGCA GCTTCGCCGA CAGCAACCCC ATCCGCGTGA     60<br>
CCGACCGCGC CGCCAGCACC CTGCAG<br>
(2) INFORMATION FOR SEQ ID NO:45:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1358 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 9..1355<br>
(D) OTHER INFORMATION: /note= "Maize optimized VIP2A(a) with the Bacillus secretion signal removed and the vacuolar targetting signal inserted as contained in pCIB5533"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:45:<br>
GATCCACC ATG GGC TGG AGC TGG ATC TTC CTG TTC CTG CTG AGC GGC GCC      50<br>
Met Gly Trp Ser Trp lie Phe Leu Phe Leu Leu Ser Gly Ala<br>
415	420<br>
GCG GGC GTG CAC TGC CTC AGC AGC AGC AGC TTC GCC GAC AGC AAC CCC       98<br>
Ala Gly Val His Cys Leu Ser Ser Ser Ser Phe Ala Asp Ser Asn Pro<br>
425	430	435	440<br>
ATC CGC GTG ACC GAC CGC GCC GCC AGC ACC CTG CAG AAC CTG AAG ATC      146<br>
lie Arg Val Thr Asp Arg Ala Ala Ser Thr Leu Gin Asn Leu Lys lie<br>
445	450	455<br>
ACC GAC AAG GTG GAG GAC TTC AAG GAG GAC AAG GAG AAG GCC AAG GAG      194<br>
Thr Asp Lys Val Glu Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu<br>
460	465	470<br>
TGG GGC AAG GAG AAG GAG AAG GAG TGG AAG CTT ACC GCC ACC GAG AAG      242<br>
Trp Gly Lys Glu Lys Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys<br>
475	480	485<br>
GGC AAG ATG AAC AAC TTC CTG GAC AAC AAG AAC GAC ATC AAG ACC AAC      290<br>
Gly Lys Met Asn Asn Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn<br>
490	495	500<br>
TAG AAG GAG ATC ACC TTC AGC ATA GCC GGC AGC TTC GAG GAC GAG ATC      338<br>
Tyr Lys Glu He Thr Phe Ser He Ala Gly Ser Phe Glu Asp Glu He<br>
505	510	515	520<br>
AAG GAC CTG AAG GAG ATC GAC AAG ATG TTC GAC AAG ACC AAC CTG AGC      386<br>
Lys Asp Leu Lys Glu lie Asp Lys Met Phe Asp Lys Thr Asn Leu Ser<br>
525	530	535<br>
AAC AGC ATC ATC ACC TAG AAG AAC GTG GAG CCC ACC ACC ATC GGC TTC      434<br>
Asn Ser He He Thr Tyr Lys Asn Val Glu Pro Thr Thr He Gly Phe<br>
540	545	550<br>
AAC AAG AGC CTG ACC GAG GGC AAC ACC ATC AAC AGC GAG GCC ATG GCC      482<br>
Asn Lys Ser Leu Thr Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala<br>
555	560	565<br>
GAG TTC AAG GAG GAG TTC CTG GAC CGC GAC ATC AAG TTC GAC AGC TAG      530<br>
Gin Phe Lys Glu Gin Phe Leu Asp Arg Asp He Lys Phe Asp Ser Tyr<br>
570	575	580<br>
CTG GAC ACC CAC CTG ACC GCC CAG CAG GTG AGC AGC AAG GAG CGC GTG      578<br>
Leu Asp Thr His Leu Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val<br>
585	590	595	600<br>
ATC CTG AAG GTG ACC GTC CCC AGC GGC AAG GGC AGC ACC ACC CCC ACC      626<br>
He Leu Lys Val Thr Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr<br>
605	610	615<br>
AAG GCC GGC GTG ATC CTG AAC AAC AGC GAG TAG AAG ATG CTG ATC GAC      674<br>
Lys Ala Gly Val He Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp<br>
620	625	630<br>
AAC GGC TAG ATG GTG CAC GTG GAC AAG GTG AGC AAG GTG GTG AAG AAG      722<br>
Asn Gly Tyr Met Val His Val Asp Lys Val Ser Lys Val Val Lys Lys<br>
635	640	645<br>
GGC GTG GAG TGC CTC CAG ATC GAG GGC ACC CTG AAG AAG ACT CTA GAC      770<br>
Gly Val Glu Cys Leu Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp<br>
650	655	660<br>
TTC AAG AAC GAC ATC AAC GCC GAG GCC CAC AGC TGG GGC ATG AAG AAC      818<br>
Phe Lys Asn Asp He Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn<br>
665	670	675	680<br>
TAC GAG GAG TGG GCC AAG GAC CTG ACC GAC AGC CAG CGC GAG GCC CTG      866<br>
Tyr Glu Glu Trp Ala Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu<br>
685	690	695<br>
GAC GGC TAC GCC CGC CAG GAC TAC AAG GAG ATC AAC AAC TAC CTG CGC      914<br>
Asp Gly Tyr Ala Arg Gin Asp Tyr Lys Glu He Asn Asn Tyr Leu Arg<br>
700	705	710<br>
AAC CAG GGC GGC AGC GGC AAC GAG AAG CTG GAC GCC CAG ATC AAG AAC      962<br>
Asn Gin Gly Gly Ser Gly Asn Glu Lys Leu Asp Ala Gin He Lys Asn<br>
715	720	725<br>
ATC AGC GAC GCC CTG GGC AAG AAG CCC ATC CCC GAG AAC ATC ACC GTG     1010<br>
He Ser Asp Ala Leu Gly Lys Lys Pro He Pro Glu Asn He Thr Val<br>
730	735	740<br>
TAG CGC TGG TGC GGC ATG CCC GAG TTC GGC TAG GAG ATC AGC GAG CCC     1058<br>
Tyr Arg Trp Cys Gly Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro<br>
745	750	755	760<br>
CTG CCC AGC CTG AAG GAC TTC GAG GAG CAG TTC CTG AAC ACC ATC AAG     1106<br>
Leu Pro Ser Leu Lys Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys<br>
765	770	775<br>
GAG GAC AAG GGC TAG ATG AGC ACC AGC CTG AGC AGC GAG CGC CTG GCC     1154<br>
Glu Asp Lys Gly Tyr Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala<br>
780	785	790<br>
GCC TTC GGC AGC CGC AAG ATC ATC CTG CGC CTG CAG GTG CCC AAG GGC     1202<br>
Ala Phe Gly Ser Arg Lys lie lie Leu Arg Leu Gin Val Pro Lys Gly<br>
795	800	805<br>
AGC ACT GGT GCC TAG CTG AGC GCC ATC GGC GGC TTC GCC AGC GAG AAG     1250<br>
Ser Thr Gly Ala Tyr Leu Ser Ala lie Gly Gly Phe Ala Ser Glu Lys<br>
810	815	820<br>
GAG ATC CTG CTG GAT AAG GAC AGC AAG TAG CAC ATC GAC AAG GTG ACC     1298<br>
Glu lie Leu Leu Asp Lys Asp Ser Lys Tyr His lie Asp Lys Val Thr<br>
825	830	835	840<br>
GAG GTG ATC ATC AAG GGC GTG AAG CGC TAG GTG GTG GAC GCC ACC CTG     1346<br>
Glu Val lie lie Lys Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu<br>
845	850	855<br>
CTG ACC AAC TAG	1358<br>
Leu Thr Asn<br>
(2) INFORMATION FOR SEQ ID NO:46:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 449 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:46:<br>
Met Gly Trp Ser Trp He Phe Leu Phe Leu Leu Ser Gly Ala Ala Gly<br>
15	10	15<br>
Val His Cys Leu Ser Ser Ser Ser Phe Ala Asp Ser Asn Pro lie Arg<br>
20	25	30<br>
Val Thr Asp Arg Ala Ala Ser Thr Leu Gin Asn Leu Lys He Thr Asp<br>
35	40	45<br>
Lys Val Glu Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly<br>
50	55	60<br>
Lys Glu Lys Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys<br>
65	70	75	80<br>
Met Asn Asn Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys<br>
85	90	95<br>
Glu lie Thr Phe Ser lie Ala Gly Ser Phe Glu Asp Glu lie Lys Asp<br>
100	105	110<br>
Leu Lys Glu lie Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser<br>
115	120	125<br>
lie lie Thr Tyr Lys Asn Val Glu Pro Thr Thr lie Gly Phe Asn Lys<br>
130	135	140<br>
Ser Leu Thr Glu Gly Asn Thr lie Asn Ser Asp Ala Met Ala Gin Phe<br>
145	150	155	160<br>
Lys Glu Gin Phe Leu Asp Arg Asp lie Lys Phe Asp Ser Tyr Leu Asp<br>
165	170	175<br>
Thr His Leu Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu<br>
180	185	190<br>
Lys Val Thr Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala<br>
195	200	205<br>
Gly Val He Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly<br>
210	215	220<br>
Tyr Met Val His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val<br>
225	230	235	240<br>
Glu Cys Leu Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys<br>
245	250	255<br>
Asn Asp He Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu<br>
260	265	270<br>
Glu Trp Ala Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly<br>
275	280	285<br>
Tyr Ala Arg Gin Asp Tyr Lys Glu He Asn Asn Tyr Leu Arg Asn Gin<br>
290	295	300<br>
Gly Gly Ser Gly Asn Glu Lys Leu Asp Ala Gin He Lys Asn He Ser<br>
305	310	315	320<br>
Asp Ala Leu Gly Lys Lys Pro He Pro Glu Asn He Thr Val Tyr Arg<br>
325	330	335<br>
Trp Cys Gly Met Pro Glu Phe Gly Tyr Gin He Ser Asp Pro Leu Pro<br>
340	345	350<br>
Ser Leu Lys Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp<br>
355	360	365<br>
Lys Gly Tyr Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe<br>
370	375	380<br>
Gly Ser Arg Lys lie lie Leu Arg Leu Gin Val Pro Lys Gly Ser Thr<br>
385	390	395	400<br>
Gly Ala Tyr Leu Ser Ala lie Gly Gly Phe Ala Ser Glu Lys Glu lie<br>
405	410	415<br>
Leu Leu Asp Lys Asp Ser Lys Tyr His lie Asp Lys Val Thr Glu Val<br>
420	425	430<br>
He lie Lys Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr<br>
435	440	445<br>
Asn<br>
(2) INFORMATION FOR SEQ ID NO:47:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 16 amino acids<br>
TYPE: amino acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: peptide (iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: Peptide<br>
LOCATION: 1..16<br>
(D) OTHER INFORMATION: /note= "linker peptide for fusion of VIPlA(a) and VIP2A(a) used to construct pCIB5533"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:47:<br>
Pro Ser Thr Pro Pro Thr Pro Ser Pro Ser Thr Pro Pro Thr Pro Ser<br>
15	10	15<br>
(2) INFORMATION FOR SEQ ID NO:48:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 66 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "DMA encoding linker peptide used to construct pCIB5533"<br>
(iii) HYPOTHETICAL: NO<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:48:<br>
CCCGGGCCTT CTACTCCCCC AACTCCCTCT CCTAGCACGC CTCCGACACC TAGCGATATC     60<br>
GGATCC	66<br>
(2) INFORMATION FOR SEQ ID NO:49:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 4031 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: other nucleic acid<br>
(A) DESCRIPTION: /desc = "Synthetic DNA"<br>
(iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY: CDS<br>
LOCATION: 6..4019<br>
(D) OTHER INFORMATION: /note= "Maize optimized DNA sequence encoding a VIP2A(a) - VIPlA(a) fusion protein as contained in pCIB5531"<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:49:<br>
GATCC ATG AAG CGC ATG GAG GGC AAG CTG TTC ATG GTG AGC AAG AAG        47<br>
Met Lys Arg Met Glu Gly Lys Leu Phe Met Val Ser Lys Lys<br>
450	455	460<br>
CTC CAG GTG GTG ACC AAG ACC GTG CTG CTG AGC ACC GTG TTC AGC ATC       95<br>
Leu Gin Val Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser He<br>
465	470	475<br>
AGC CTG CTG AAC AAC GAG GTG ATC AAG GCC GAG CAG CTG AAC ATC AAC      143<br>
Ser Leu Leu Asn Asn Glu Val He Lys Ala Glu Gin Leu Asn He Asn<br>
480	485	490	495<br>
AGC CAG AGC AAG TAG ACC AAC CTC CAG AAC CTG AAG ATC ACC GAG AAG      191<br>
Ser Gin Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys He Thr Asp Lys<br>
500	505	510<br>
GTG GAG GAG TTC AAG GAG GAG AAG GAG AAG GCC AAG GAG TGG GGC AAG      239<br>
Val Glu Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys<br>
515	520	525<br>
GAG AAG GAG AAG GAG TGG AAG CTT ACC GCC ACC GAG AAG GGC AAG ATG      287<br>
Glu Lys Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met<br>
530	535	540<br>
AAC AAC TTC CTG GAC AAC AAG AAC GAC ATC AAG ACC AAC TAG AAG GAG      335 Asn Asn Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu<br>
545	550	555<br>
ATC ACC TTC AGC ATA GCC GGC AGC TTC GAG GAC GAG ATC AAG GAC CTG      383<br>
He Thr Phe Ser He Ala Gly Ser Phe Glu Asp Glu He Lys Asp Leu<br>
560	565	570	575<br>
AAG GAG ATC GAC AAG ATG TTC GAC AAG ACC AAC CTG AGC AAC AGC ATC      431<br>
Lys Glu He Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser He<br>
580	585	590<br>
ATC ACC TAG AAG AAC GTG GAG CCC ACC ACC ATC GGC TTC AAC AAG AGC      479<br>
He Thr Tyr Lys Asn Val Glu Pro Thr Thr He Gly Phe Asn Lys Ser<br>
595	600	605<br>
CTG ACC GAG GGC AAC ACC ATC AAC AGC GAC GCC ATG GCC CAG TTC AAG      527<br>
Leu Thr Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys<br>
610	615	620<br>
GAG CAG TTC CTG GAC CGC GAC ATC AAG TTC GAC AGC TAG CTG GAC ACC      575<br>
Glu Gin Phe Leu Asp Arg Asp He Lys Phe Asp Ser Tyr Leu Asp Thr<br>
625	630	635<br>
CAC CTG ACC GCC CAG CAG GTG AGC AGC AAG GAG CGC GTG ATC CTG AAG      623<br>
His Leu Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys<br>
640	645	650	655<br>
GTG ACC GTC CCC AGC GGC AAG GGC AGC ACC ACC CCC ACC AAG GCC GGC     671<br>
Val Thr Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly<br>
660	665	670<br>
GTG ATC CTG AAC AAC AGC GAG TAG AAG ATG CTG ATC GAC AAC GGC TAG      719<br>
Val He Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr<br>
675	680	685<br>
ATG GTG CAC GTG GAC AAG GTG AGC AAG GTG GTG AAG AAG GGC GTG GAG      767<br>
Met Val His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu<br>
690	695	700<br>
TGC CTC CAG ATC GAG GGC ACC CTG AAG AAG ACT CTA GAC TTC AAG AAC      815<br>
Cys Leu Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn<br>
705	710	715<br>
GAC ATC AAC GCC GAG GCC CAC AGC TGG GGC ATG AAG AAC TAG GAG GAG      863<br>
Asp He Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu<br>
720	725	730	735<br>
TGG GCC AAG GAG CTG ACC GAG AGC GAG CGC GAG GCC CTG GAG GGC TAG      911<br>
Trp Ala Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr<br>
740	745	750<br>
GCC CGC GAG GAG TAG AAG GAG ATC AAC AAC TAG CTG CGC AAC GAG GGC      959<br>
Ala Arg Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly<br>
755	760	765<br>
GGC AGC GGC AAC GAG AAG CTG GAC GCC GAG ATC AAG AAC ATC AGC GAG     1007<br>
Gly Ser Gly Asn Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp<br>
770	775	780<br>
GCC CTG GGC AAG AAG CCC ATC CCC GAG AAC ATC ACC GTG TAG CGC TGG<br>
Ala Leu Gly Lys Lys Pro He Pro Glu Asn He Thr Val Tyr Arg Trp<br>
785	790	795<br>
1055<br><br>
TGC GGC ATG CCC GAG TTC GGC TAG GAG ATC AGC GAC CCC CTG CCC AGC<br>
Cys Gly Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser<br>
800	805	810	815<br>
1103<br><br>
CTG AAG GAC TTC GAG GAG CAG TTC CTG AAC ACC ATC AAG GAG GAC AAG<br>
Leu Lys Asp Phe Glu Glu Gin Phe Leu Asn Thr He Lys Glu Asp Lys<br>
820	825	830<br>
1151<br>
GGC TAG ATG AGC ACC AGC CTG AGC AGC GAG CGC CTG GCC GCC TTC GGC     1199<br>
Gly Tyr Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly<br>
835	840	845<br>
AGC CGC AAG ATC ATC CTG CGC CTG CAG GTG CCC AAG GGC AGC ACT GGT     1247<br>
Ser Arg Lys He He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly<br>
850	855	860<br>
GCC TAG CTG AGC GCC ATC GGC GGC TTC GCC AGC GAG AAG GAG ATC CTG     1295<br>
Ala Tyr Leu Ser Ala He Gly Gly Phe Ala Ser Glu Lys Glu He Leu<br>
865	870	875<br>
CTG GAT AAG GAC AGC AAG TAG CAC ATC GAC AAG GTG ACC GAG GTG ATC     1343<br>
Leu Asp Lys Asp Ser Lys Tyr His He Asp Lys Val Thr Glu Val He<br>
880	885	890	895<br>
ATC AAG GGC GTG AAG CGC TAG GTG GTG GAC GCC ACC CTG CTG ACC AAC     1391<br>
He Lys Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn<br>
900	905	910<br>
TCC CGG GGG CCT TCT ACT CCC CCA ACT CCC TCT CCT AGC ACG CCT CCG     1439<br>
Ser Arg Gly Pro Ser Thr Pro Pro Thr Pro Ser Pro Ser Thr Pro Pro<br>
915	920	925<br>
ACA CCT AGC GAT ATC GGA TCC ACC ATG AAG ACC AAC CAG ATC AGC ACC     1487<br>
Thr Pro Ser Asp He Gly Ser Thr Met Lys Thr Asn Gin He Ser Thr<br>
930	935	940<br>
ACC CAG AAG AAC CAG CAG AAG GAG ATG GAC CGC AAG GGC CTG CTG GGC     1535<br>
Thr Gin Lys Asn Gin Gin Lys Glu Met Asp Arg Lys Gly Leu Leu Gly<br>
945	950	955<br>
TAG TAG TTC AAG GGC AAG GAG TTC AGC AAC CTG ACC ATG TTC GCC CCC     1583<br>
Tyr Tyr Phe Lys Gly Lys Asp Phe Ser Asn Leu Thr Met Phe Ala Pro<br>
960	965	970	975<br>
ACG CGT GAG AGC ACC CTG ATC TAG GAC GAG CAG ACC GCC AAC AAG CTG     1631<br>
Thr Arg Asp Ser Thr Leu lie Tyr Asp Gin Gin Thr Ala Asn Lys Leu<br>
980	985	990<br>
CTG GAC AAG AAG CAG CAG GAG TAG CAG AGC ATC CGC TGG ATC GGC CTG     1679<br>
Leu Asp Lys Lys Gin Gin Glu Tyr Gin Ser lie Arg Trp lie Gly Leu<br>
995	1000	1005<br>
ATC CAG AGC AAG GAG ACC GGC GAC TTC ACC TTC AAC CTG AGC GAG GAC     1727<br>
lie Gin Ser Lys Glu Thr Gly Asp Phe Thr Phe Asn Leu Ser Glu Asp<br>
1010	1015	1020<br>
GAG CAG GCC ATC ATC GAG ATC AAC GGC AAG ATC ATC AGC AAC AAG GGC     1775<br>
Glu Gin Ala lie lie Glu He Asn Gly Lys He He Ser Asn Lys Gly<br>
1025	1030	1035<br>
AAG GAG AAG CAG GTG GTG CAC CTG GAG AAG GGC AAG CTG GTG CCC ATC     1823<br>
Lys Glu Lys Gin Val Val His Leu Glu Lys Gly Lys Leu Val Pro He<br>
1040	1045	1050	1055<br>
AAG ATC GAG TAG CAG AGC GAC ACC AAG TTC AAC ATC GAC AGC AAG ACC     1871<br>
Lys He Glu Tyr Gin Ser Asp Thr Lys Phe Asn He Asp Ser Lys Thr<br>
1060	1065	1070<br>
TTC AAG GAG CTG AAG CTT TTC AAG ATC GAC AGC CAG AAC CAG CCC CAG     1919<br>
Phe Lys Glu Leu Lys Leu Phe Lys He Asp Ser Gin Asn Gin Pro Gin<br>
1075	1080	1085<br>
CAG GTG CAG CAG GAC GAG CTG CGC AAC CCC GAG TTC AAC AAG AAG GAG     1967<br>
Gin Val Gin Gin Asp Glu Leu Arg Asn Pro Glu Phe Asn Lys Lys Glu<br>
1090	1095	1100<br>
AGC CAG GAG TTC CTG GCC AAG CCC AGC AAG ATC AAC CTG TTC ACC CAG     2015<br>
Ser Gin Glu Phe Leu Ala Lys Pro Ser Lys He Asn Leu Phe Thr Gin<br>
1105	1110	1115<br>
CAG ATG AAG CGC GAG ATC GAC GAG GAC ACC GAC ACC GAC GGC GAC AGC     2063<br>
Gin Met Lys Arg Glu He Asp Glu Asp Thr Asp Thr Asp Gly Asp Ser<br>
1120	1125	1130	1135<br>
ATC CCC GAC CTG TGG GAG GAG AAC GGC TAG ACC ATC CAG AAC CGC ATC     2111<br>
He Pro Asp Leu Trp Glu Glu Asn Gly Tyr Thr He Gin Asn Arg He<br>
1140	1145	1150<br>
GCC GTG AAG TGG GAC GAC AGC CTG GCT AGC AAG GGC TAG ACC AAG TTC     2159<br>
Ala Val Lys Trp Asp Asp Ser Leu Ala Ser Lys Gly Tyr Thr Lys Phe<br>
1155	1160	1165<br>
GTG AGC AAC CCC CTG GAG AGC CAC ACC GTG GGC GAC CCC TAG ACC GAC     2207 Val Ser Asn Pro Leu Glu Ser His Thr Val Gly Asp Pro Tyr Thr Asp<br>
1170	1175	1180<br>
TAG GAG AAG GCC GCC CGC GAG CTG GAG CTG AGC AAC GCC AAG GAG ACC     2255<br>
Tyr Glu Lys Ala Ala Arg Asp Leu Asp Leu Ser Asn Ala Lys Glu Thr<br>
1185	1190	1195<br>
TTC AAC CCC CTG GTG GCC GCC TTC CCC AGC GTG AAC GTG AGC ATG GAG     2303<br>
Phe Asn Pro Leu Val Ala Ala Phe Pro Ser Val Asn Val Ser Met Glu<br>
1200	1205	1210	1215<br>
AAG GTG ATC CTG AGC CCC AAC GAG AAC CTG AGC AAC AGC GTG GAG AGC     2351<br>
Lys Val lie Leu Ser Pro Asn Glu Asn Leu Ser Asn Ser Val Glu Ser<br>
1220	1225	1230<br>
CAC TCG AGC ACC AAC TGG AGC TAG ACC AAC ACC GAG GGC GCC AGC GTG     2399<br>
His Ser Ser Thr Asn Trp Ser Tyr Thr Asn Thr Glu Gly Ala Ser Val<br>
1235	1240	1245<br>
GAG GCC GGC ATC GGT CCC AAG GGC ATC AGC TTC GGC GTG AGC GTG AAC     2447<br>
Glu Ala Gly lie Gly Pro Lys Gly lie Ser Phe Gly Val Ser Val Asn<br>
1250	1255	1260<br>
TAG GAG CAC AGC GAG ACC GTG GCC CAG GAG TGG GGC ACC AGC ACC GGC     2495<br>
Tyr Gin His Ser Glu Thr Val Ala Gin Glu Trp Gly Thr Ser Thr Gly<br>
1265	1270	1275<br>
AAC ACC AGC CAG TTC AAC ACC GCC AGC GCC GGC TAG CTG AAC GCC AAC     2543<br>
Asn Thr Ser Gin Phe Asn Thr Ala Ser Ala Gly Tyr Leu Asn Ala Asn<br>
1280	1285	1290	1295<br>
GTG CGC TAG AAC AAC GTG GGC ACC GGC GCC ATC TAG GAC GTG AAG CCC     2591<br>
Val Arg Tyr Asn Asn Val Gly Thr Gly Ala lie Tyr Asp Val Lys Pro<br>
1300	1305	1310<br>
ACC ACC AGC TTC GTG CTG AAC AAC GAC ACC ATC GCC ACC ATC ACC GCC     2639<br>
Thr Thr Ser Phe Val Leu Asn Asn Asp Thr lie Ala Thr lie Thr Ala<br>
1315	1320	1325<br>
AAG TCG AAT TCC ACC GCC CTG AAC ATC AGC CCC GGC GAG AGC TAG CCC     2687<br>
Lys Ser Asn Ser Thr Ala Leu Asn lie Ser Pro Gly Glu Ser Tyr Pro<br>
1330	1335	1340<br>
AAG AAG GGC CAG AAC GGC ATC GCC ATC ACC AGC ATG GAC GAC TTC AAC     2735<br>
Lys Lys Gly Gin Asn Gly lie Ala lie Thr Ser Met Asp Asp Phe Asn<br>
1345	1350	1355<br>
AGC CAC CCC ATC ACC CTG AAC AAG AAG CAG GTG GAC AAC CTG CTG AAC     2783<br>
Ser His Pro lie Thr Leu Asn Lys Lys Gin Val Asp Asn Leu Leu Asn<br>
1360	1365	1370	1375<br>
AAC AAG CCC ATG ATG CTG GAG ACC AAC CAG ACC GAC GGC GTC TAG AAG     2831<br>
Asn Lys Pro Met Met Leu Glu Thr Asn Gin Thr Asp Gly Val Tyr Lys<br>
1380	1385	1390<br>
ATC AAG GAC ACC CAC GGC AAC ATC GTG ACG GGC GGC GAG TGG AAC GGC     2879<br>
He Lys Asp Thr His Gly Asn He Val Thr Gly Gly Glu Trp Asn Gly<br>
1395	1400	1405<br>
GTG ATC CAG GAG ATC AAG GCC AAG ACC GCC AGC ATC ATC GTC GAC GAG     2927<br>
Val He Gin Gin He Lys Ala Lys Thr Ala Ser He He Val Asp Asp<br>
1410	1415	1420<br>
GGC GAG CGC GTG GCC GAG AAG CGC GTG GCC GCC AAG GAC TAG GAG AAC     2975<br>
Gly Glu Arg Val Ala Glu Lys Arg Val Ala Ala Lys Asp Tyr Glu Asn<br>
1425	1430	1435<br>
CCC GAG GAC AAG ACC CCC AGC CTG ACC CTG AAG GAC GCC CTG AAG CTG     3023<br>
Pro Glu Asp Lys Thr Pro Ser Leu Thr Leu Lys Asp Ala Leu Lys Leu<br>
1440	1445	1450	1455<br>
AGC TAG CCC GAC GAG ATC AAG GAG ATC GAG GGC TTG CTG TAG TAG AAG     3071<br>
Ser Tyr Pro Asp Glu He Lys Glu He Glu Gly Leu Leu Tyr Tyr Lys<br>
1460	1465	1470<br>
AAC AAG CCC ATC TAG GAG AGC AGC GTG ATG ACC TAT CTA GAC GAG AAC     3119<br>
Asn Lys Pro He Tyr Glu Ser Ser Val Met Thr Tyr Leu Asp Glu Asn<br>
1475	1480	1485<br>
ACC GCC AAG GAG GTG ACC AAG CAG CTG AAC GAC ACC ACC GGC AAG TTC     3167<br>
Thr Ala Lys Glu Val Thr Lys Gin Leu Asn Asp Thr Thr Gly Lys Phe<br>
1490	1495	1500<br>
AAG GAC GTG AGC CAC CTG TAG GAC GTG AAG CTG ACC CCC AAG ATG AAC     3215<br>
Lys Asp Val Ser His Leu Tyr Asp Val Lys Leu Thr Pro Lys Met Asn<br>
1505	1510	1515<br>
GTG ACC ATC AAG CTG AGC ATC CTG TAG GAC AAC GCC GAG AGC AAC GAC     3263<br>
Val Thr He Lys Leu Ser He Leu Tyr Asp Asn Ala Glu Ser Asn Asp<br>
1520	1525	1530	1535<br>
AAC AGC ATC GGC AAG TGG ACC AAC ACC AAC ATC GTG AGC GGC GGC AAC     3311<br>
Asn Ser He Gly Lys Trp Thr Asn Thr Asn He Val Ser Gly Gly Asn<br>
1540	1545	1550<br>
AAC GGC AAG AAG CAG TAG AGC AGC AAC AAC CCC GAC GCC AAC CTG ACC     3359<br>
Asn Gly Lys Lys Gin Tyr Ser Ser Asn Asn Pro Asp Ala Asn Leu Thr<br>
1555	1560	1565<br>
CTG AAC ACC GAC GCC CAG GAG AAG CTG AAC AAG AAC CGC GAC TAG TAG     3407<br>
Leu Asn Thr Asp Ala Gin Glu Lys Leu Asn Lys Asn Arg Asp Tyr Tyr<br>
1570	1575	1580<br>
ATC AGC CTG TAG ATG AAG AGC GAG AAG AAC ACC CAG TGC GAG ATC ACC     3455<br>
He Ser Leu Tyr Met Lys Ser Glu Lys Asn Thr Gin Cys Glu He Thr<br>
1585	1590	1595<br>
ATC GAC GGC GAG ATA TAG CCC ATC ACC ACC AAG ACC GTG AAC GTG AAC     3503<br>
He Asp Gly Glu He Tyr Pro He Thr Thr Lys Thr Val Asn Val Asn<br>
1600	1605	1610	1615<br>
AAG GAG AAC TAG AAG CGC CTG GAC ATC ATC GCC CAC AAC ATC AAG AGC     3551<br>
Lys Asp Asn Tyr Lys Arg Leu Asp lie lie Ala His Asn lie Lys Ser<br>
1620	1625	1630<br>
AAC CCC ATC AGC AGC CTG CAC ATC AAG ACC AAC GAC GAG ATC ACC CTG     3599<br>
Asn Pro lie Ser Ser Leu His lie Lys Thr Asn Asp Glu lie Thr Leu<br>
1635	1640	1645<br>
TTC TGG GAC GAC ATA TCG ATT ACC GAC GTC GCC AGC ATC AAG CCC GAG     3647<br>
Phe Trp Asp Asp lie Ser lie Thr Asp Val Ala Ser lie Lys Pro Glu<br>
1650	1655	1660<br>
AAC CTG ACC GAC AGC GAG ATC AAG CAG ATA TAG ACT CGC TAG GGC ATC     3695<br>
Asn Leu Thr Asp Ser Glu lie Lys Gin lie Tyr Ser Arg Tyr Gly lie<br>
1665	1670	1675<br>
AAG CTG GAG GAC GGC ATC CTG ATC GAC AAG AAA GGC GGC ATC CAC TAG     3743<br>
Lys Leu Glu Asp Gly lie Leu lie Asp Lys Lys Gly Gly lie His Tyr<br>
1680	1685	1690	1695<br>
GGC GAG TTC ATC AAC GAG GCC AGC TTC AAC ATC GAG CCC CTG CAG AAC     3791<br>
Gly Glu Phe lie Asn Glu Ala Ser Phe Asn lie Glu Pro Leu Gin Asn<br>
1700	1705	1710<br>
TAG GTG ACC AAG TAG GAG GTG ACC TAG AGC AGC GAG CTG GGC CCC AAC     3839 Tyr Val Thr Lys Tyr Glu Val Thr Tyr Ser Ser Glu Leu Gly Pro Asn<br>
1715	1720	1725<br>
GTG AGC GAC ACC CTG GAG AGC GAC AAG ATT TAG AAG GAC GGC ACC ATC     3887<br>
Val Ser Asp Thr Leu Glu Ser Asp Lys lie Tyr Lys Asp Gly Thr lie<br>
1730	1735	1740<br>
AAG TTC GAC TTC ACC AAG TAG AGC AAG AAC GAG CAG GGC CTG TTC TAG     3935<br>
Lys Phe Asp Phe Thr Lys Tyr Ser Lys Asn Glu Gin Gly Leu Phe Tyr<br>
1745	1750	1755<br>
GAC AGC GGC CTG AAC TGG GAC TTC AAG ATC AAC GCC ATC ACC TAG GAC     3983<br>
Asp Ser Gly Leu Asn Trp Asp Phe Lys lie Asn Ala lie Thr Tyr Asp<br>
1760	1765	1770	1775<br>
GGC AAG GAG ATG AAC GTG TTC CAC CGC TAG AAC AAG TAGATCTGAG         4029<br>
Gly Lys Glu Met Asn Val Phe His Arg Tyr Asn Lys<br>
1780	1785<br>
CT	4031<br>
(2) INFORMATION FOR SEQ ID NO:50:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
LENGTH: 1338 amino acids<br>
TYPE: amino acid<br>
(D) TOPOLOGY: linear<br>
(ii) MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID NO:50:<br>
4et Lys Arg Met Glu Gly Lys Leu Phe Met Val Ser Lys Lys Leu Gin<br>
15	10	15<br>
Jal Val Thr Lys Thr Val Leu Leu Ser Thr Val Phe Ser lie Ser Leu<br>
20	25	30<br>
'jeu Asn Asn Glu Val lie Lys Ala Glu Gin Leu Asn lie Asn Ser Gin<br>
35	40	45<br>
Ser Lys Tyr Thr Asn Leu Gin Asn Leu Lys lie Thr Asp Lys Val Glu<br>
50	55	60<br>
Asp Phe Lys Glu Asp Lys Glu Lys Ala Lys Glu Trp Gly Lys Glu Lys<br>
65	70	75	80<br>
Glu Lys Glu Trp Lys Leu Thr Ala Thr Glu Lys Gly Lys Met Asn Asn<br>
85	90	95<br>
Phe Leu Asp Asn Lys Asn Asp lie Lys Thr Asn Tyr Lys Glu lie Thr<br>
100	105	110<br>
Phe Ser lie Ala Gly Ser Phe Glu Asp Glu lie Lys Asp Leu Lys Glu<br>
115	120	125<br>
He Asp Lys Met Phe Asp Lys Thr Asn Leu Ser Asn Ser He He Thr<br>
130	135	140<br>
Tyr Lys Asn Val Glu Pro Thr Thr He Gly Phe Asn Lys Ser Leu Thr<br>
145	150	155	160<br>
Glu Gly Asn Thr He Asn Ser Asp Ala Met Ala Gin Phe Lys Glu Gin<br>
165	170	175<br>
Phe Leu Asp Arg Asp He Lys Phe Asp Ser Tyr Leu Asp Thr His Leu<br>
180	185	190<br>
Thr Ala Gin Gin Val Ser Ser Lys Glu Arg Val He Leu Lys Val Thr<br>
195	200	205<br>
Val Pro Ser Gly Lys Gly Ser Thr Thr Pro Thr Lys Ala Gly Val He<br>
210	215	220<br>
Leu Asn Asn Ser Glu Tyr Lys Met Leu He Asp Asn Gly Tyr Met Val<br>
225	230	235	240<br>
His Val Asp Lys Val Ser Lys Val Val Lys Lys Gly Val Glu Cys Leu<br>
245	250	255<br>
Gin He Glu Gly Thr Leu Lys Lys Ser Leu Asp Phe Lys Asn Asp He<br>
260	265	270<br>
Asn Ala Glu Ala His Ser Trp Gly Met Lys Asn Tyr Glu Glu Trp Ala<br>
275<br>
280<br>
285<br>
Lys Asp Leu Thr Asp Ser Gin Arg Glu Ala Leu Asp Gly Tyr Ala Arg<br>
290	295	300<br>
Gin Asp Tyr Lys Glu lie Asn Asn Tyr Leu Arg Asn Gin Gly Gly Ser<br>
305	310	315	320<br>
Gly Asn Glu Lys Leu Asp Ala Gin lie Lys Asn lie Ser Asp Ala Leu<br>
325	330	335<br>
Gly Lys Lys Pro lie Pro Glu Asn lie Thr Val Tyr Arg Trp Cys Gly<br>
340	345	350<br>
Met Pro Glu Phe Gly Tyr Gin lie Ser Asp Pro Leu Pro Ser Leu Lys<br>
355	360	365<br>
Asp Phe Glu Glu Gin Phe Leu Asn Thr lie Lys Glu Asp Lys Gly Tyr<br>
370	375	380<br>
Met Ser Thr Ser Leu Ser Ser Glu Arg Leu Ala Ala Phe Gly Ser Arg<br>
385	390	395	400<br>
Lys lie He Leu Arg Leu Gin Val Pro Lys Gly Ser Thr Gly Ala Tyr<br>
405	410	415<br>
Leu Ser Ala lie Gly Gly Phe Ala Ser Glu Lys Glu lie Leu Leu Asp<br>
420	425	430<br>
Lys Asp Ser Lys Tyr His lie Asp Lys Val Thr Glu Val lie lie Lys<br>
435	440	445<br>
Gly Val Lys Arg Tyr Val Val Asp Ala Thr Leu Leu Thr Asn Ser Arg<br>
450	455	460<br>
Gly Pro Ser Thr Pro Pro Thr Pro Ser Pro Ser Thr Pro Pro Thr Pro<br>
465	470	475	480<br>
Ser Asp He Gly Ser Thr Met Lys Thr Asn Gin He Ser Thr Thr Gin<br>
485	490	495<br>
Lys Asn Gin Gin Lys Glu Met Asp Arg Lys Gly Leu Leu Gly Tyr Tyr<br>
500	505	510<br>
Phe Lys Gly Lys Asp Phe Ser Asn Leu Thr Met Phe Ala Pro Thr Arg<br>
515	520	525<br>
Asp Ser Thr Leu He Tyr Asp Gin Gin Thr Ala Asn Lys Leu Leu Asp<br>
530	535	540<br>
Lys Lys Gin Gin Glu Tyr Gin Ser He Arg Trp He Gly Leu He Gin<br>
545	550	555	560<br>
Ser Lys Glu Thr Gly Asp Phe Thr Phe Asn Leu Ser Glu Asp Glu Gin<br>
565	570	575<br>
Ala lie lie Glu He Asn Gly Lys He He Ser Asn Lys Gly Lys Glu<br>
580	585	590<br>
Lys Gin Val Val His Leu Glu Lys Gly Lys Leu Val Pro He Lys He<br>
595	600	605<br>
Glu Tyr Gin Ser Asp Thr Lys Phe Asn He Asp Ser Lys Thr Phe Lys<br>
610	615	620<br>
Glu Leu Lys Leu Phe Lys He Asp Ser Gin Asn Gin Pro Gin Gin Val<br>
625	630	635	640<br>
Gin Gin Asp Glu Leu Arg Asn Pro Glu Phe Asn Lys Lys Glu Ser Gin<br>
645	650	655<br>
Glu Phe Leu Ala Lys Pro Ser Lys He Asn Leu Phe Thr Gin Gin Met<br>
660	665	670<br>
Lys Arg Glu He Asp Glu Asp Thr Asp Thr Asp Gly Asp Ser He Pro<br>
675	680	685<br>
Asp Leu Trp Glu Glu Asn Gly Tyr Thr He Gin Asn Arg He Ala Val<br>
690	695	700<br>
Lys Trp Asp Asp Ser Leu Ala Ser Lys Gly Tyr Thr Lys Phe Val Ser<br>
705	710	715	720<br>
Asn Pro Leu Glu Ser His Thr Val Gly Asp Pro Tyr Thr Asp Tyr Glu<br>
725	730	735<br>
Lys Ala Ala Arg Asp Leu Asp Leu Ser Asn Ala Lys Glu Thr Phe Asn<br>
740	745	750<br>
Pro Leu Val Ala Ala Phe Pro Ser Val Asn Val Ser Met Glu Lys Val<br>
755	760	765<br>
He Leu Ser Pro Asn Glu Asn Leu Ser Asn Ser Val Glu Ser His Ser<br>
770	775	780<br>
Ser Thr Asn Trp Ser Tyr Thr Asn Thr Glu Gly Ala Ser Val Glu Ala<br>
785	790	795	800<br>
Gly He Gly Pro Lys Gly He Ser Phe Gly Val Ser Val Asn Tyr Gin<br>
805	810	815<br>
His Ser Glu Thr Val Ala Gin Glu Trp Gly Thr Ser Thr Gly Asn Thr<br>
820	825	830<br>
Ser Gin Phe Asn Thr Ala Ser Ala Gly Tyr Leu Asn Ala Asn Val Arg<br>
835	840	845<br>
Tyr Asn Asn Val Gly Thr Gly Ala He Tyr Asp Val Lys Pro Thr Thr<br>
850	855	860<br>
Ser Phe Val Leu Asn Asn Asp Thr He Ala Thr lie Thr Ala Lys Ser<br>
865	870	875	880<br>
Asn Ser Thr Ala Leu Asn lie Ser Pro Gly Glu Ser Tyr Pro Lys Lys<br>
885	890	895<br>
Gly Gin Asn Gly He Ala He Thr Ser Met Asp Asp Phe Asn Ser His<br>
900	905	910<br>
Pro lie Thr Leu Asn Lys Lys Gin Val Asp Asn Leu Leu Asn Asn Lys<br>
915	920	925<br>
Pro Met Met Leu Glu Thr Asn Gin Thr Asp Gly Val Tyr Lys He Lys<br>
930	935	940<br>
Asp Thr His Gly Asn He Val Thr Gly Gly Glu Trp Asn Gly Val He<br>
945	950	955	960<br>
Gin Gin He Lys Ala Lys Thr Ala Ser He He Val Asp Asp Gly Glu<br>
965	970	975<br>
Arg Val Ala Glu Lys Arg Val Ala Ala Lys Asp Tyr Glu Asn Pro Glu<br>
980	985	990<br>
Asp Lys Thr Pro Ser Leu Thr Leu Lys Asp Ala Leu Lys Leu Ser Tyr<br>
995	1000	1005<br>
Pro Asp Glu He Lys Glu He Glu Gly Leu Leu Tyr Tyr Lys Asn Lys<br>
1010	1015	1020<br>
Pro He Tyr Glu Ser Ser Val Met Thr Tyr Leu Asp Glu Asn Thr Ala<br>
1025	1030	1035	1040<br>
Lys Glu Val Thr Lys Gin Leu Asn Asp Thr Thr Gly Lys Phe Lys Asp<br>
1045	1050	1055<br>
Val Ser His Leu Tyr Asp Val Lys Leu Thr Pro Lys Met Asn Val Thr<br>
1060	1065	1070<br>
He Lys Leu Ser He Leu Tyr Asp Asn Ala Glu Ser Asn Asp Asn Ser<br>
1075	1080	1085<br>
He Gly Lys Trp Thr Asn Thr Asn He Val Ser Gly Gly Asn Asn Gly<br>
1090	1095	1100<br>
Lys Lys Gin Tyr Ser Ser Asn Asn Pro Asp Ala Asn Leu Thr Leu Asn<br>
1105	1110	1115	1120<br>
Thr Asp Ala Gin Glu Lys Leu Asn Lys Asn Arg Asp Tyr Tyr He Ser<br>
1125	1130	1135<br>
Leu Tyr Met Lys Ser Glu Lys Asn Thr Gin Cys Glu He Thr He Asp<br>
1140	1145	1150<br>
Gly Glu lie Tyr Pro He Thr Thr Lys Thr Val Asn Val Asn Lys Asp<br>
Aan Tyr Lys Arg Leu Asp lie He Ala His Asn lie Lys Ser Asn Pro<br>
1170	1175	1180<br>
lie Server Leu His lie Lys Thr Asn Asp Glu lie Thr Leu Phe Tip<br>
1185    -     1190	1195	1200<br>
Asp Asp lie Ser lie Thr Asp Val Ala Sex lie Lye Pro Glu Asn Leu<br>
1205	1210	1215<br>
Thr Asp Ser Glu lie Lys Gin lie Tyr Ser Arg Tyr Qly lie Lys Leu<br>
1220	1225       _   1230<br>
Glu Asp Gly lie Leu lie Asp Lys Lya Gly Gly lie His Tyr Gly Glu<br>
1235	1240	1245<br>
Phe lie Asn Glu Ala Ser Phe Asn lie Glu Pro Leu Gin Asn Tyr Val<br>
1250	1255	1260<br>
Thr Lys Tyr Glu Val Thr Tyr Ser Ser Glu Leu Gly Pro Asn Val Ser<br>
1265	1270	1275	1280<br>
Asp Thr Leu Glu Ser Asp Lys He Tyr Lys Asp Gly Thr He Lys Phe<br>
1285	1290	1295<br>
Asp Phe Thr Lys Tyr Ser Lys Asn Glu Gin Gly Leu Phe Tyr Asp Ser<br>
1300	1305	1310<br>
Gly Leu Asn Trp Asp Phe Lys He Asn Ala He Thr Tyr Asp Gly Lys<br>
1315	1320	1325<br>
Glu Met Asn Via Phe His Arg Tyr Asn Lys<br>
1330	1335<br>
(2) INFORMATION FOR SEQ ID NO:51:<br>
(i) SEQUENCE CHARACTERISTICS;<br>
LENGTH: 2444 base pairs<br>
TYPE: nucleic acid<br>
STRANDEDNESS: single<br>
TOPOLOGY; linear<br>
(ii) MOLECULE TYPE: DMA (genomic) (iii) HYPOTHETICAL: NO<br>
(ix) FEATURE:<br>
NAME/KEY; CDS<br>
LOCATION: 17..2444<br>
(D) OTHER INFORMATION: /product- "3A(a) synthetic .-native fusion"<br>
(xi) SEQUENCE DESCRIPTION: SEQ IP NO:51:<br>
QGATOCACCA ATGAAG ATG AAC AAG AAC AAC ACC AAG CTG AGC ACC CGC        49 Met Asn Lys Asn Asn Thr Lys Leu Ser Thr Arg<br>
* 1	5	10<br>
GCC CTG COf AGC TTCATCG^TACTTCAACGQCATCTACQGCTTCGCC	97<br>
Ala Leu Pro Ser Phe He Asp Tyr Phe Asn Gly lie Tyr Gly Phe Ala<br>
15	20	25<br>
ACC GGC ATC AAG GAC ATC ATG AAC ATG ATC TTC AAG ACC GAC ACC GGC	145<br>
Thr Sly lie Lys Asp He Met Asn Net He Phe Lys Thr Asp Thr Gly<br>
30	35	40<br>
GGCGACCTGACCCTGGACGAGATCCTGAAGAACGWGAGCrGCTGAAC	193<br>
Gly Asp Leu Thr Leu Asp Glu He Leu Lys Asn Gin Gin Leu Leu Asn<br>
45	50	55<br>
GAC ATC AQC GGC AAG CTG GAC G3C GTC AAC GGC AGC CTG MC GAC CTG	241<br>
Asp He Ser Gly Lys Leu Asp Gly Val Asn Gly Ser Leu Asn Asp Leu<br>
60	65	70	75<br>
ATC Gâ¬C C^G GGC AAC CTG MC ACC GAG CTG AGC AAG GAG ATC CTT AAG	289<br>
He Ala Gin Gly Asn Leu Asn Thr Glu Leu Ser Lys Glu He Leu Lys<br>
80	85	90<br>
ATC GCC AAC GAG CAR AAC CAG GTC CTG AAC GAC GTG AAC AAC AAG CTG	337<br>
He Ala Asn Glu Gin Asn Gin Val Leu Asn Asp Val Asn Asn Lys Leu<br>
95	100	105<br>
GAC GCC ATC AAC ACC ATG CTG CGC GTG TAG CTG COG AAG ATC ACC AGC	385<br>
Asp Ala He Asn Thr Met Leu Arg Val Tyr Leu Pro Lys He Thr Ser<br>
110	115	120<br>
ATG CTG AGC GAC GTG ATG AAG CAG AAC TAC GCC CTG AGC CTG GAG ATC     433<br>
Met Leu Ser Asp Val Met Lys Gin Asn Tyr Ala Leu Ser Leu Gin He<br>
125	130	135<br>
GAG TAC CTC AGC AAG CAG CTG CAG GAG ATC AGC GAC AAG CTG GAC ATC     481<br>
Glu Tyr Leu Ser Lys Gin Leu Gin Glu He Ser Asp Lys Leu Asp He<br>
140	145	150	155<br>
ATC AAC GTG AAC GTC CTG ATC AAC AGC ACC CTG ACC GAG ATC ACC CCG     529<br>
He Asn Val Asn Val Leu He Asn Ser Thr Leu Thr Glu He Thr Pro<br>
160	165	170<br>
GCC TAC CAG CGC ATC AAG TAC GTG AAC GAG AAG TTC GAA GAG CTG ACC     577<br>
Ala Tyr Gin Arg He Lys Tyr Val Asn Glu Lys Phe Glu Glu Leu Thr<br>
175	180	185<br>
TTC GCC ACC GAG ACC AGC AGC AAG GTG AAG AAG GAC GGC AGC CCG GCC     625<br>
Phe Ala Thr Glu Thr Ser Ser Lys Val Lys Lys Asp Gly Ser Pro Ala<br>
190	195	200<br>
GAC ATC CTG GAC GAG CTG ACC GAG CTG ACC GAG CTG GCC AAG AGC GTG     673 Asp He Leu Asp Glu Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val<br>
205<br>
210<br>
215<br><br>
ACC AAG AAC GAC GTG GAC CX5C TTC GAG TTC TAG CTG MC ACX: TTC CAC<br>
Thr Lys. Asn Asp Val Asp Gly Phe Glu Phe Tyr Leu Asn Thr Phe His<br>
220	â	225	230	235<br>
GACGTGATGGTGGGCAACMCCTGTTCGGCCGCAGCQCCCTGAAGACC<br>
Asp Val Met Val Gly Asn Asn Leu Phe Gly Arg Sar Ala Leu Lys Thr<br>
240	245	250<br>
go:agcgagctgatcaccaaggagaacgtgaagaccagcggcag<:></:>
Ala Ser Glu Leu lie Thr Lya (MÂ« Asn Val LyÂ« Thr Ser Gly Ser Glu<br>
255	260	265<br>
721<br>
769<br>
si?<br><br>
GTG QGC MC GTG TAG MC TTC CTG ATC GTG CTG A(X GCC CTG GAG! GCC<br>
Val Gly Aan Val Tyr Asn Phe Leu lie Val Leu Thr Ala Leu Gin Ala<br>
270	275	280<br>
CAG GCC TTC CIG ACC CTG ACC ACC TGT
Gin Ala Phe Leu Thr Leu Thr Thr Cys Arg Lys Leu Leu Gly Leu Ala<br>
285	290	295<br>
GAG ATC GAG TAG AX AGC ATC ATG AAC GAG CAC TTG AAC AAG GAG AAG<br>
Asp lie Asp Tyr Thr Ser lie Mat Asn Glu His Leu Asn Lys Glu Lys<br>
300	305	310	315<br>
GAGGAGTTCCGCGTGAACATCCTGGCGACCC^AGCAACA
Glu Glu Phe Arg Val Asn He Leu Pro Thr Leu Ser Asn Thr Phe Ser<br>
320	325	330<br>
AAC COS AAC TAG GCC AAG GTC AAG GGC AGC GAG GAG GAC GCC AAG ATG<br>
Asn Pro Asn Tyr Ala Lys Val Lys Gly Ser Asp Glu Asp Ala Lys Met<br>
335	340	345<br>
ATC GTG GAG GCT AAG CXX5 OQC CAC GO3 TTG ATC GGC TTC GAG ATC AGC<br>
lie Val Glu Ala Lys Pro Gly His Ala Leu lie Gly Phe Glu He Ser<br>
350	355	360<br>
AACGACAGCATCACCGTGCTGAAGGTGTACGAGGCCAAGCTGAAGCAG<br>
Asn Asp Ser He Thr Val Leu Lys Val Tyr Glu Ala Lys Leu Lys Gin<br>
365	370	375<br>
AAC TAC CAG GTG GAG AAG GAG AGC TTG AGC GAG GTG ATC TAG GGC GAG<br>
Aen Tyr Gin Val Asp Lys Asp Ser Leu Ser Glu Val He Tyr Gly Asp<br>
380	385	390	395<br>
ATG GAC AAG CTG CT6 TGT CCG GAC CAG AGC GAG CAA ATC TAC TAC ACC<br>
Met Asp Lys Leu Leu Cys Pro Asp Gin Ser Glu Gin He Tyr Tyr Thr<br>
400	405	410<br>
AAC AAC ATC GTG TTC CCG AAC GAG TAC GTG ATC ACC AAG ATC GAC TTC<br>
Asn Asn He Val Phe Pro Asn Gly Tyr Val He Thr Lys He Asp Phe<br>
415	420	425<br>
865<br>
913<br>
961<br>
1009<br>
1057<br>
1105<br>
1153<br>
1201<br>
1249<br>
1297<br><br>
ACC AAG AAG ATG AAG ACC CTG CGC TAC GAG GTG ACC GCC AAC TTC TAC<br>
1345<br>
Thr lys Lys Met Lys Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr<br>
430	435	440<br>
Afif\ AGO ACC GGC GAG ATC GAC CTC AAC AAG AA6 AAG GTG GAG AGC     1393 Asp Ser 'Ser Thr Gly Glu lie Asp Leu Asn Lys Lys Lys Val Glu Ser<br>
445  ~         450	455<br>
AGC GAG GCC GAG TAG CGC ACC CTG AGC GCG AAC GAC GAC GGC GTC TAG    1441<br>
Ser Glu Ala Glu Tyr Arg Thr Leu Ser Ala Asn Asp Asp Gly Val Tyr<br>
460	465	470	475<br>
ATG CCA CTG GGC GTG ATC AGC GAG ACC TTC CTG ACC CCG ATC AAC GGC    1489<br>
Met Pro Leu Gly Val He Ser Glu Thr Phe Leu Thr Pro He Asn Gly<br>
480	485	490<br>
Phe Gly Leu Gin Ala Asp Glu Asn Ser Arg Leu lie Thr Leu Thr Cys<br>
495	500	505<br>
AAGAGCTACCTOCGCGAGCTGCTGCTAGCXIACCGACCreAGCAACM<br>
Lys Ser Tyr Leu Arg Glu Leu Leu Leu Ala Thr Asp Leu Ser Asn Lys<br>
510	515	520<br>
1537<br>
1585<br>
GAGACCAAGCTXSATCGTtSCO^OSSAGCGGCTTC ATC AGC AAC ATC GTG	1633<br>
Glu Thr Lys Leu He Val Pro Pro Ser Gly Phe He Ser Asn He Val<br>
525	530	535<br>
GAG AAC GGC AGC ATC GAG GAG GAC AAC CTC GAG CCG TGG AAG GCC AAC	1681<br>
Glu Asn Gly Ser He Glu Glu Asp Asn Leu Glu Pro Tip Lys Ala Asn<br>
540	545	550	555<br>
AACAAGAACGCCTACGTGGACCACACCGGCGXGTGAACGGCACCAAG	1729<br>
Asn Lys Asn Ala Tyr Val Asp His Thr Gly Gly Val Asn Gly Thr Lys<br>
560	565	570<br>
CTG TAC GTG CAC AAG GAC G^ GGC ATC AGC C^ TTC ATC GGC GAC	1777<br>
Ala Leu Tyr Val His Lys Asp Gly Gly He Ser Gin Phe He Gly Asp<br>
575	580	585<br>
AAG CTG AAG CCG AAG ACC GAG TAC GTG ATC GAG TAC ACC GTG AAG GGC	1825<br>
Lys Leu Lys Pro Lys Thr Glu Tyr Val He Gin Tyr Thr Val Lys Gly<br>
590	595	600<br>
AAG CCA TCG ATT CAC CTG AAG GAC GAG AAC AX GGC TAC ATC CAC TAC	1873<br>
Lys Pro Ser He His Leu Lys Asp Glu Asn Thr Gly Tyr He His Tyr<br>
605	610	615<br>
GAG GAC AX AAC AAC AAC CTG GAG GAC TAC GAG ACC ATC AAC AAG CGC	1921<br>
Glu Asp Thr Asn Asn Asn Leu Glu Asp Tyr Gin Thr lie Asn Lys Arg<br>
620	625	630	635<br>
TTC ACC ACC GGC AX GAC CTG AAG GGC GTG TAC CTG ATC CTG AAG AGC<br>
Phe Thr Thr Gly Thr Asp Leu Lys Gly Val Tyr Leu He Leu Lys Ser<br>
640	645	650<br>
1969<br>
AAC GGC C^^ GAG GCC TGG GGC GAC AAC TTC ATC ATC CTG GAG ATC    2017<br>
Gin Aan Gly Asp Glu Ala Trp Gly Asp Asn Phe lie lie Leu Glu lie<br>
655	660	665<br>
AGC CCG 1\GQ GJ^ AAG CTG CTG AGC CCG GAG CTX5 ATC AAC ACC AA(: AAC    2065<br>
Ser Pro Ser Glu Lys Leu Leu Ser Pro Glu Leu lie Asn Thr Asn Asn<br>
670	675	680<br>
TGG ACC AGC AO: GGC AO: ACC MC ATC AGC GGC AAC ACC CTC ACC: CTG    2113<br>
Trp Thr Ser Thr Gly Ser Thr Asn lie Ser Gly Asn Thr Leu Thr Leu<br>
685	690	695<br>
TAG C^ GÂ«^ GGC CO5 GOG ATT CTA AM CAA AAC CTT CAA TTA GAT AGT    2161<br>
Tyr Gin Gly Gly Arg Gly lie Leu Lys Gin Asn Leu Gin Leu Asp Ser<br>
700	705	710	715<br>
TTT TCA ACT TAT AGA GTG TAT TTT TCT GTG TCC GGA GAT GCT AAT GTA    2209<br>
Phe Ser Thr Tyr Arg Val Tyr Phe Ser Val Ser Gly Asp Ala Asn Val<br>
720	725	730<br>
AGG ATT AGA AAT TCT AGG GAA GTG TTA TTT GAA AAA AGA TAT ATG AGC    2257<br>
Arg lie Arg Asn Ser Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser<br>
735	740	745<br>
GGTGCTAAAGATGTTTCTGAAATGTTCACTACAAAATTTGAGAARGA^    2305<br>
Gly Ala Lys Asp Val Ser Glu Met Phe Thr Thr Lys Phe Glu Lys Asp<br>
750	755	760<br>
AAC TTT TAT ATA GAG CTT TCT CAA QGG AAT AAT OTA TAT GOT GGT OCT    2353<br>
Asn Phe Tyr lie Glu Leu Ser Gin Gly Asn Asn Leu Tyr Gly Gly Pro<br>
765	770	775<br>
ATT GTA CAT TTT TAG GAT GTC TCT ATT AAG NAA GAT CGG GAT CTA ATA    2401<br>
He Val His Phe Tyr Asp Val Ser lie Lys Xaa Asp Arg Asp Leu lie<br>
780	785	790	795<br>
'iU'A AWUTTTTTAAAAGCI^TTCTTGTATMTGTCCTTGAT T        2444<br>
Leu Thr Val Phe Lys Ser Xaa Phe Leu Tyr Asn Val Leu Asp<br>
800	805<br>
(2) INFOPMATICN FOR SEQ ID NO;52:<br>
(i) SEQUENCE CHARACTERISTICS:<br>
(A) LENGTH: 809 anuno acids (Â») TWtii amtno acid (D) TOPOLOGY: linear<br>
(ii&gt; MOLECULE TYPE: protein<br>
(xi) SEQUENCE DESCRIPTION: SEQ ID N0s52:<br>
Met Asn Lys Asn Asn Thr Lys Leu Ser Thr Arg Ala Leu Pro Ser Phe<br>
15	10	15<br>
He Asp Tyr Phe Asn Gly He Tyr Gly Phe Ala Thr Gly He Lya Asp<br>
20	25	30<br>
He Met Asn Met He Phe Lys Thr Asp Thr Gly Gly Asp L*u Thr ten<br>
, 35     *        40	45<br>
â¢f<br>
Asp Glu Ile'Leu Lys Asn Gin Gin Leu Leu Asn Asp He Ser Gly Lys<br>
50	55	60<br>
Leu Asp Gly Val Asn Gly Ser Leu Asn Asp Leu He Ala Gin Gly Asn<br>
65	70	75	80<br>
Leu Asn Thr Glu Leu Ser Lys Glu He Leu Lys He Ala Asn Glu Gin<br>
85	90	95<br>
Asn Gin Val Leu Asn Asp Val Asn Asn Lys Leu Asp Ala He Asn Thr<br>
100	105	110<br>
Met Leu Arg Val Tyr Leu Pro Lys He Thr Ser Met Leu Ser Asp Val<br>
115	120	125<br>
Met Lys Gin Asn Tyr Ala Leu Ser Leu Gin He Glu Tyr Leu Ser Lys<br>
130	135	140<br>
Gin Leu Gin Glu He Ser Asp Lys Leu Asp He He Asn Val Asn Val<br>
145	150	155	160<br>
Leu He Asn Ser Thr Leu Thr Glu He Thr Pro Ala Tyr Gin Arg He<br>
165	170	175<br>
Lys Tyr Val Asn Glu Lys Phe Glu Glu Leu Thr Phe Ala Thr Glu Thr<br>
180	185	190<br>
Ser Ser Lys Val Lys Lys Asp Gly Ser Pro Ala Asp He Leu Asp Glu<br>
195	200	205<br>
Leu Thr Glu Leu Thr Glu Leu Ala Lys Ser Val Thr Lys Asn Asp Val<br>
210	215	220<br>
Asp Gly Phe Glu Phe Tyr Leu Asn Thr Phe His Asp Val Met Val Gly<br>
225	230	235	240<br>
Asn Asn Leu Phe Gly Arg Ser Ala Leu Lys Thr Ala Ser Glu Leu He<br>
245	250	255<br>
Thr Lys Glu Aen Val Lyo Thr Car Gly Oer Glu Val Gly Asn Val Tyr<br>
260	265	270<br>
Asn Phe Leu He Val Leu Thr Ala Leu Gin Ala Gin Ala Phe Leu Thr<br>
275	280	285<br>
Leu Thr Thr Cys Arg Lys Leu Lsu Gly Leu Ala Asp He Asp Tyr Thr<br>
290	295	300<br>
Ser He Met Asn Glu His Leu Asn Lys Glu Lys Glu Glu Phe Arg Val<br>
305<br>
310<br>
315<br>
320<br>
Asn lie Leu Pro Thr Leu Ser Asn Thr Phe Ser Asn Pro Asn Tyr Ala<br>
326	330	335<br>
â¢t<br>
Lys Val Lys' Gly Ser Asp Glu Asp Ala Lys Met He Val Glu Ala Lys<br>
340	345	350<br>
Pro Gly His Ala Leu lie Gly Phe Glu lie Ser Asn Asp Ser lie Thr<br>
355	360	365<br>
Val Leu Lys Val Tyc Glu Ala Lys Leu Lys Gin Asn Tyr Gin Val Asp<br>
375<br>
Lys Asp Ser Leu Ser Glu Val lie Tyr Gly Asp Met Asp Lys Leu Leu<br>
385	390	395	400<br>
Cys Pro Asp Gin Ser Glu Gin lie Tyr Tyr Thr Asn Asn He Val Phe<br>
405	410	415<br>
Pro Asn Glu Tyr Val lie Thr Lys lie Asp Phe Thr Lys Lys Met Lys<br>
420	425	430<br>
Thr Leu Arg Tyr Glu Val Thr Ala Asn Phe Tyr Asp Ser Ser Thr Gly<br>
435	440	445<br>
Glu He Asp Leu Asn Lya Lys Lys Val Glu Ser Ser Glu Ala Glu Tyr<br>
450	455	460<br>
Arg Thr Leu Ser Ala Asn Asp Asp Gly Val Tyr Met Pro Leu Gly Val<br>
465	470	475	480<br>
He Ser Glu Thr Phe Leu Thr Pro He Asn Gly Phe Gly Leu Gin Ala<br>
485	490	495<br>
Asp Glu Asn Ser Arg Leu He Thr Leu Thr Cys Lys Ser Tyr Leu Arg<br>
500	505	510<br>
Glu Leu Leu Leu Ala Thr Asp Leu Ser Asn Lys Glu Thr Lys Leu He<br>
515	520	525<br>
Val Pro Pro Ser Gly Phe He Ser Asn He Val Glu Asn Gly Ser He<br>
530	535	540<br>
Glu Glu Asp Asn Leu Glu Pro Trp Lys Ala Aen Asn Lys Aan Ala Tyr<br>
545	550	555	560<br>
Val Asp His Thr Gly Gly Val Asn Gly Thr Lya Ala Leu Tyr Val His<br>
565	570	575<br>
Lys Asp Gly Gly He Ser Gin Phe He Gly Asp Lys Leu Lys Pro Lys<br>
580	585	590<br>
Thr Glu Tyr Val He Gin Tyr Thr Val Lys Gly Lys Pro Ser He His<br>
595	600	605<br>
Leu Lys Asp Glu Asn Thr Gly Tyr He His Tyr Glu Asp Thr Asn Asn<br>
610	615	620<br>
*<br>
Aen Leulau Asp Tyr Gin Thr He Asn Lys Arg Phe Thr Thr Gly Thr<br>
625	630	635	640<br>
Asp Leu Lys Gly Val Tyr Leu lie Leu Lys Ser Gin Asn Gly Asp Glu<br>
645	650	655<br>
Ala Trp Gly Asp Asn Phe He He Leu Glu He Ser Pro Ser Glu Lys<br>
660	665	670<br>
Leu Leu Ser Pro Glu Leu He Asn Thr Asn Asn Trp Thr Ser Thr Gly<br>
675	680	685<br>
Ser Thr Asn He Ser Gly Asn Thr Leu Thr Leu Tyr Gin Gly Gly Arg<br>
690	695	700<br>
Gly He Leu Lys Gin Asn Leu Gin Leu Asp Ser Phe Ser Thr Tyr Arg<br>
705	710	715	720<br>
Val Tyr Phe Ser Val Ser Gly Asp Ala Asn Val Arg He Arg Asn Ser<br>
725	730	735<br>
Arg Glu Val Leu Phe Glu Lys Arg Tyr Met Ser Gly Ala Lys Asp Val<br>
740	745	750<br>
Ser Glu Mat Phe Thr Thr Lys Phe Glu Lys Asp Asn Phe Tyr He Glu<br>
755	760	765<br>
Leu Ser Gin Gly Asn Asn Leu Tyr Gly Gly Pro He Val His Phe Tyr<br>
770	775	780<br>
Asp Val Ser He Lys Xaa Asp Arg Asp Leu He Leu Thr Val Phe Lys<br>
785	790	795	800<br>
Ser xaa Phe Leu Tyr Asn Val Leu Asp 805<br><br><br><br><br><br><br><br><br>
We claim:<br>
1.	An expression cassette comprising a DNA molecule operably linked to plant expression sequences including the transcriptional and translational regulatory signals necessary for expression of the associated DNA constructs in a host organism and optionally further regulatory sequences, wherein said DNA molecule encodes a vegetative insecticidal protein isolatable from liquid culture media during the vegetative growth phase of Bacillus spp., and wherein said protein is encoded by a nucleotide sequence that hybridizes to a nucleotide sequence of SEQ ID NOs: 28, 30 or 31 at 65 Â°C in a buffer comprising 7 % SDS and 0.5 M sodium phosphate.<br>
2.	The expression cassette as claimed in claim 1, wherein said DNA molecule encodes a protein as defined by SEQ ID NO: 29 or SEQ ID NO: 32.<br>
3.	The expression cassette as claimed in claim 2, wherein said DNA molecule has the nucleotide sequence given in SEQ ID NO: 28, SEQ ID NO: 30, or SEQ ID NO: 31.<br>
4.	The expression cassette as claimed in claim 1, wherein said DNA molecule comprises a nucleotide sequence that has been wholly or partially optimized for expression in a plant by utilizing plant preferred codons.<br>
5.	The expression cassette as claimed in claim 4, wherein said DNA molecule has the nucleotide sequence given in SEQ ID NO: 30.<br>
6.	The expression cassette as claimed in claim 1, wherein said DNA molecule comprises a nucleotide sequence that has been wholly or partially optimized for expression in a microorganism by utilizing host preferred codons.<br><br>
7.	The expression cassette as claimed in claim 1, wherein said DNA molecule is obtainable<br>
by a process comprising<br>
a)	obtaining a DNA molecule comprising a nucleotide sequence encoding a vegetative<br>
insecticidal protein; and<br>
b)	hybridizing said DNA molecule with an oligonucleotide probe comprising a contiguous<br>
portion of the coding sequence for said insecticidal protein at least 10 nucleotides in<br>
length obtainable from a DNA molecule defined in SEQ ID NO: 28, SEQ ID NO: 30, or<br>
SEQ ID NO: 31; and<br>
c)	isolating said hybridized DNA.<br>
8.	The expression cassette as claimed in any one of claims 1 to 7, wherein the host organism<br>
is a plant.<br>
9.	A vector molecule comprising an expression cassette as claimed in any one of claims 1 to<br>
7.<br>
10.	The expression cassette substantially as herein described with reference to the foregoing examples.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1BYnN0cmFjdC0oMDItMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Abstract-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1BYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Bc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1DbGFpbXMtKDAxLTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Claims-(01-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1DbGFpbXMtKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Claims-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1DbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1kZWwtMTk5NS1jb21wbGV0ZSBzcGVjaWZpY2F0aW9uIChncmFudGVkKS5wZGY=" target="_blank" style="word-wrap:break-word;">1772-del-1995-complete specification (granted).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1kZWwtMTk5NS1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDAxLTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-del-1995-Correspondence Others-(01-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1kZWwtMTk5NS1Db3JyZXNwb25kZW5jZS1PdGhlcnMoMTYtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1772-del-1995-Correspondence-Others(16-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Correspondence-Others-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Db3JyZXNwb25kZW5jZS1PdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLSgwMi0xMS0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Description (Complete)-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Description (Complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTEtKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-1-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTEwLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-10.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1kZWwtMTk5NS1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">1772-del-1995-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTItKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-2-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTMtKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-3-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1Gb3JtLTYucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Form-6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1HUEEtKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-GPA-(02-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1HUEEucGRm" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-GPA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc3Mi1ERUwtMTk5NS1QZXRpdGlvbiAxMzctKDAyLTExLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1772-DEL-1995-Petition 137-(02-11-2010).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250774-a-process-for-preparing-a-gel-element.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250776-flooring-system-having-multiple-alignment-points.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250775</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1772/DEL/1995</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jan-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Jan-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Sep-1995</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SYNGENTA PARTICIPATIONS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>SCHWARZWALDALLEE 215, CH-4058,BASEL</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KRISTY KOSTICHKA</td>
											<td>5017 WINEBERRY DRIVE, DURHAM, NC 27713, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NICHOLAS BRENDAN DUCK</td>
											<td>1215 GATEHOUSE DRIVE, CARY, NC 27511, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JEAN JOSE ESTRUCH</td>
											<td>2911-E BAINBRIDGE DRIVE, DURHAM NC 27713, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NALINI MANO DESAI</td>
											<td>107 SILVERWOOD LANE CARY WINEBERRY DRIVE, DURHAM NC 27511, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>MARTHA ALICE MULLINS</td>
											<td>104 COUNTRYBROOK LANE, YOUNGSVILLE, NC 27596, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MICHAEL GENE KOZIEL</td>
											<td>509 CAROLYN COURT, CARY, NC 27511, USA</td>
										</tr>
										<tr>
											<td>7</td>
											<td>GREGORY WAYNE WARREN</td>
											<td>324 BOND LAKE DRIVE, CARY, NC 27513, U.S.A.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>GORDON JAMES NYE</td>
											<td>1001 BARY COURT, APEX NC 27502 USA</td>
										</tr>
										<tr>
											<td>9</td>
											<td>BRIAN CARR</td>
											<td>1100D LADY S&#x27; SLIPPER COURT, RALEIGH, NC 27606, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250775-an-expression-cassette-comprising-a-dna-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:15:08 GMT -->
</html>
