<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250899-affinity-polypeptide-for-purification-of-recombinant-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:11:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250899:AFFINITY POLYPEPTIDE FOR PURIFICATION OF RECOMBINANT PROTEINS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AFFINITY POLYPEPTIDE FOR PURIFICATION OF RECOMBINANT PROTEINS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present disclosure provides an affinity polypeptide for the purification of a recombinant biologically active protein or polypeptide. Further, the present disclosure provides a fusion recombinant protein or polypeptide wherein the fusion recombinant protein comprises of at least two components, a biologically active polypeptide or protein or protein of interest and the affinity polypeptide. The biologically active polypeptide may be linked directly or indirectly to the affinity polypeptide by covalent binding. The present disclosure provides a recombinant expression vector for the producing said fusion recombinant protein in host cells. Further, the present disclosure provides an improved method of purification of recombinant protein from the host cells. Further, the disclosure provides a method of purification of the recombinant biologically active polypeptide or protein by immobilized metal ion chelating chromatography.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>                                                   FORM 2<br>
THE PATENTS ACT 1970<br>
        (39 of 1970)                 &amp;The Patent Rules 2005<br>
PROVISIONAL SPECIFICATION<br>
(see sections 10 &amp; rule 13)<br>
1.   TITLE OF THE INVENTION<br>
"NOVEL AFFINITY PEPTIDES USEFUL IN PURIFICATION OF<br>
RECOMBINANT PROTEINS"<br>
2.   APPLICANT (S)<br>
NAME	NATIONALITY	ADDRESS<br>
USV LIMITED	INDIAN	USV Limited, B.S.D. Marg, Station Road, Govandi, Mumbai 400088, Maharashtra, India.<br>
3.   PREAMBLE TO THE DESCRIPTION<br>
PROVISIONAL SPECIFICATION<br>
The following specification describes the invention.<br><br>
NOVEL    AFFINITY    PEPTIDES    USEFUL    IN    PURIFICATION    OF RECOMBINANT PROTEINS<br>
TECHNICAL FIELD<br>
The present invention relates to a novel affinity peptide handle for the purification of a recombinant biologically active protein/ polypeptide. The present invention also relates to the production of fusion proteins comprising biologically active polypeptide or protein and a novel affinity handle (DAC), wherein the affinity handle is suitable for purification of recombinant proteins using affinity chromatography on metal chelate resins with immobilized metal ions. BACKGROUND OF THE INVENTION<br>
Genes for the desired protein may be isolated from organisms, which contain the gene in nature or alternately they may be chemically synthesized. The isolated or chemically synthesized gene may be inserted in vectors and expressed in host systems, which produce the desired protein at high levels. Suitable purification procedures are indispensable for the establishment of an efficient process to obtain recombinant or synthetic proteins. It can be useful to know beforehand some physical properties such as hydrophobicity, ionic charge etc of the protein to facilitate the development of a suitable purification protocol from the recombinant source. On the other hand, there are now several ways of preparing fusion proteins, which can be purified by affinity techniques without any knowledge of the properties of the target protein. Using affinity handles as fusion partners, efficient purification schemes may be used which allows rapid recovery of expressed foreign gene products from crude extracts. Hybrid products require a cleavage process for the liberation of the protein of interest-from-the precursor molecule for preparation of pharmaceutical grade recombinants. The cleavage could be either chemical or enzymatic.<br>
The desired proteins may be isolated from complex mixtures by methods based on differences in size, solubility, charge etc. However none of these methods are capable of purifying proteins beyond a moderate level. The most common problem is low yield of purified protein in the expected fraction. This can be caused by insufficient protein loaded on the column, protein not binding to the column or protein not eluting from the column or nonspecific binding of other proteins to the column. Affinity chromatography based on the ability of proteins to bind non-covalently but specifically with an immobilized ligand is often preferred as it can purify proteins from complex mixtures without significant losses. The<br><br>
disadvantage is that it requires the availability of the corresponding ligand for the desired protein, which is not always possible since such specific ligands do not exist for all proteins.<br>
To overcome this problem, antibodies to a linker peptide may be used as an immunoaffinity ligand. The fusion peptide is passed through a column containing immobilized antibodies which bind to the antigenic linker peptide and thus the fusion peptide is isolated (disclosed in US Patent No 4,782,137). The disadvantage of this method is that the desired polypeptide may get denatured by either the buffer conditions which are necessary to allow immunogenic complexing or the buffer conditions which must be present to terminate such complexes, leading to low yields of biological active protein.<br>
Due to the difficulties often experienced in purifying recombinant proteins, a variety of vector systems (Sassenfeld, 1990) have been developed in which the expressed protein is a fusion protein containing an N-terminal polypeptide that simplifies purification. Such "tags" can be subsequently removed using a specific protease or by chemical cleavage. Tags used include proteins and polypeptides for which there is a specific antibody, binding proteins that will interact with columns containing a specific ligand, polyhistidine tags with affinity to immobilized metal columns and sequences that result in biotinylation by the host and enable purification on an avidin column.<br>
Protein and peptide affinity tags have become highly popular tools for purifying recombinant proteins. They can provide hundred or even thousand-fold purification from crude extracts without prior steps to remove nucleic acid or other cellular material. J. Porath introduced immobilized metal ion affinity chromatography (IMAC) for fractionating proteins (Porath et al, 1975). IMAC consists of derivatizing a resin with iminodiacetic acid (IDA) and chelating metal ions to the IDA-derivatized resin. Proteins bind to the metal ions through amino acid residues capable of donating electrons and are immobilized on the column. Once binding has occurred, the protein can be released by protonation of its associated metal ion-binding ligand. Dissociation is achieved by lowering the pH of the surrounding buffer medium or using competitive counter ligands such as imidazole. Histidines containing di- or tri-peptides in proteins have been used to show that IMAC is a specific and selective purification procedure (US Patent No 4,569,794 and US Patent No 5,310,663).<br>
BRIEF DESCRIPTION OF ACCOMPANYING DRAWINGS<br>
Figure 1: Map of pDAC-lacZ<br>
Figure 2: SDS-PAGE gel showing binding ratios of expression of  b-galactosidase gene Lane 1: HMW marker<br><br>
Lane 2: Total protein <br>
Lane 3: Supernatant <br>
Lane 4: Ni-IDA FT (-imidazole) <br>
Lane 5: Ni-IDA wash (-imidazole) <br>
Lane 6: Ni-IDA elution (-imidazole) Lane 7: Ni-IDA FT (+imidazole) <br>
Lane 8: Ni-IDA wash (+imidazole)<br>
 Lane 9: Ni-IDA elution (+imidazole)     Lane 10: Cu-IDA FT (-imidazole) <br>
Lane 11: Cu-IDA wash (-imidazole) Lane 12: Cu-IDA elution (-imidazole) Lane 13: Cu-IDA FT (+imidazole)<br>
 Lane 14: Cu-IDA wash (+imidazole) Lane 15: Cu-IDA elution (+imidazole)<br>
DESCRIPTION OF THE INVENTION<br>
The present invention provides a novel affinity peptide handle for the purification of a recombinant biologically active protein/ polypeptide. The invention relates to a recombinant expression vector for the production of fusion protein in host cells. The present invention provides an improved method of purification of fusion protein from the host cells wherein the fusion protein comprises of at least two components, a biologically active polypeptide/protein and an affinity peptide handle. The biologically active polypeptide may be linked directly or indirectly to the affinity peptide handle by covalent binding. Further, the invention provides a method of purification of the biologically active polypeptide/ protein.<br>
In one aspect of the invention, the affinity peptide handle has an amino acid sequence having the general formula R-supl- (peptide T)-R.sup2 where R.sup.l is either hydrogen or a peptide comprising of 1-30 amino acids ; peptide T has an amino acid sequence (Xn- His--Xn-Yn-Pro.Hisn) 2 -60 - Xn- Pron- Hisn, where X is selected from the group consisting of Gly, Ala, Ser, Thr, Asp, Glu, His, Val, and Leu; Y is selected from the group consisting of Gly, Ser, Glu, Asp, Lys, Val, Arg, and Leu; and n ranges from 1 to 4 and R.sup 2 is Q, or Q-Asp-Asp-Asp-Asp-Lys- or Q-Ile- Glu-Gly-Arg-, where Q is a peptide ranging from 1 to 30 amino acid(s) in length (s).<br>
■ In one embodiment, the invention provides a novel affinity peptide handle having an amino acid sequence selected from the group consisting of amino acid sequences having SEQ ID NOs: 1,2, 3 and 4.<br><br>
method of purification of fusion protein from a mixture containing the recombinant fusion protein and impurities. The method of purification of the fusion protein consists of contacting the fusion protein with a resin containing immobilized metal ions for a sufficient amount of time, selectively eluting the fusion protein from the resin and then optionally cleaving to separate the desired protein from the affinity peptide tag.<br>
When the fusion protein according to the present invention is contacted with an immobilized metal ion containing resin, the fusion protein will be immobilized which will allow it to be separated from impurities. The conditions needed for purification described in the present invention does not denature the fusion protein. Thus, in few steps, it is possible to purify the final protein product in high yield.<br>
The purification is based on the ability of certain amino acids acting as electron donors on the surface of proteins (histidine, tryptophan, tyrosine or phenylalanine) to bind reversibly to transition-metal ions that have been immobilized by a chelating group covalently bound to a solid support. Of these amino acids, histidine is quantitatively the most important in mediating the binding of most proteins to immobilized metal ions. Copper and Nickel ions have the greatest affinity for histidine (Nagakawa &amp; Porath, 1989). The chelating groups that are commonly used are either iminodiacetic acid (IDA) or nitrilotriacetic acid (NTA). Also, the presence of proline residue before His residue in the peptide handle gives rise to steric properties of the affinity handle.<br>
An embodiment of the present invention relates to a novel affinity peptide handle for the purification of recombinant fusion proteins. Further, the invention provides the amino acid sequence of the novel affinity peptide handle having an amino acid sequence of general formula R-supl- (peptide T)- R sup2, wherein R.sup.l is either hydrogen or a peptide comprising of 1-30 amino acids ; "peptide T having an amino acid sequence (Xn- His-Xn-Yn-Pro.Hisrl) 2 -60 -■ Xn- Pron- Hisn&gt; where X is selected from the group consisting of Gly, Ala, Ser, Thr, Asp, Glu, His, Val, and Leu; Y is selected from the group the consisting of Gly, Ser, Glu, Asp, Lys, Vaf, Arg, and Leu; and n ranges from l to 4 and R.sup 2 is Q, or Q-Asp-Asp-Asp-Asp-Lys- or Q-Ile- Glu-Gly-Arg-, where Q is a peptide ranging from 1 to 30 amino acid(s) in length (s).<br>
In one embodiment, the invention provides a novel affinity peptide handle having an amino acid sequence selected from the group consisting of amino acid sequences having SEQ ID NOs: 1,2, 3 and 4.<br><br>
In another embodiment, the invention provides the peptide T' of the affinity peptide handle having an amino acid sequence selected from the group consisting of amino acid sequences SEQ ID NOs: 5, 6, 7 and 8.<br>
In yet another embodiment, the invention provides a polynucleotide sequence which encodes the affinity peptide handle having the general formula as shown above. Further, the polynucleotide sequence encoding the affinity peptide handle is selected from the group consisting of polynucleotide sequences of SEQ ID NOs: 9, 10, 11 and 12.<br>
In another embodiment, the invention provides the polynucleotide sequence of the peptide T' of the affinity peptide handle is selected, from the group consisting of polynucleotide sequence SEQ ED NOs: 13, 14, 15 and 16.<br>
In another embodiment, the invention teaches a method of construction of a recombinant plasmid vector comprising of the polynucleotide sequence coding for the affinity peptide handle. This is carried out by inserting the polynucleotide sequence coding for the affinity peptide handle into a plasmid vector. The plasmid vectors used for the construction are selected from the group consisting of pRA, pET series, pcDNA3.1, pLex, PCR 3.1 and pBAD. Further, the polynucleotide sequence coding for the affinity peptide handle is selected from the group consisting of polynucleotide sequences having SEQ ID NOs: 9, 10, 11 and 12.<br>
In another embodiment, the invention provides a method of production of novel affinity peptide handle by amplification of the desired fragments. The amplification was carried out by using the bacterial genomic DNA (Top 10, HB101 and JM109) as a template DNA using the primer sequences as shown in SEQ ID No: 17 and 18. Subsequent amplifications using different template DNA were carried out to obtain the desired nucleotide sequence of the affinity peptide handle using the primer sequences as shown in the SEQ ID NOs: 19-32.<br>
In another embodiment, the invention describes a method of construction of recombinant expression vector for the production of fusion" protein^ The expression vector comprises of a promoter, a polynucleotide sequence coding for a novel affinity peptide handle, a protease recognition site or a chemical cleavage site, and a heterologous gene/DNA of interest.<br>
In another embodiment, the invention provides an expression vector for production of fusion protein, the said vector comprising of<br>
i)       a promoter which is selected from the group consisting of AraB, trp, tac, lac, osmB, CMV, EF-la,SV 40 and T7,<br><br>
ii)	a nucleotide sequence coding for the affinity peptide handle,  <br>
iii)	protease recognition site which is selected from the group consisting of Carboxy peptidase, Factor Xa, Trypsin, V8, and Chymotrypsin and Enterokinase; or a chemical cleavage site recognized by a reagent selected from a group consisting of cyanogen halide, hydroxyl amine, formic acid, acetic acid, hydrochloric acid and trifluoroacetic acid,<br>
 iv)      the heterologous gene of interest which is selected from the group consisting of genes coding for b galactosidase, Granulocyte Colony Stimulating Factor, human Interleukin, Interleukin-2,   Platelet-derived   growth   factor,   Granulocyte   Macrophage   Colony Stimulating Factor, Insulins, bovine Enterokinase, Vascular Endothelial Growth Factor, Nerve Growth Factor, Interferons, and Tissue Plasminogen Activators,<br>
                                    In yet another embodiment, the invention discloses recombination expression vector for production of fusion protein which is selected from the group of vectors consisting of pDAC-LacZ, pDAC-IL2, pDAC-PDGF, pDAC-GCSF, pDAC-EK, pDAC-hGH, pDAC-IFN and pDAC-EPO.<br>
Another embodiment of the invention describes transformation of host cells with the recombination expression vector. Further, the present invention discloses a host cell selected from a group of cells consisting of Pseudomanas, Yeast, Saccharomyces, Pichia, Hanseneula, CHO, BHK and COS and E. coli.<br>
Another embodiment of the present invention discloses a host cell selected from the ' group of E. coli strains such as JM109, DH5a, Top 10, BL21, HB101, XLl-Blue and LMG19.<br>
Another, embodiment of the present invention relates to a process for the production and purification of recombinant polypeptide/protein wherein the process comprises the steps of:<br>
i)   obtaining a suitable plasmid vector;<br>
ii) inserting a promoter, a polynucleotide sequence of the affinity peptide handle used for the purification of recombinant fusion proteins, a protease recognition site or a chemical cleavage site, a heterologous gene/DNA encoding a polypeptide of interest and a polynucleotide sequence of the terminator into expression vector to produce a recombinant expression vector, iii) transforming the recombinant expression vector into host cell to produce recombinant cells,<br><br>
iv) growing the recombinant cells in a suitable medium for production of the recombinant<br>
fusion protein, <br>
v)  recovering recombinant fusion protein using metal ion chelating chromatography and, vi) purifying the desired biologically active protein by protease or chemical cleavage.<br>
In another embodiment, the present invention provides the process of purification of recombinant polypeptide/protein, wherein the metal ion for immobilized metal ion chelating chromatography is selected from a group consisting of Ni, Cu, Zn and Co.<br>
In another embodiment, the invention discloses a method for the construction of recombinant expression vectors which are selected from a group consisting of pDAC-LacZ, pDAC-IL2, pDAC-PDGF, pDAC-GCSF, pDAC-EK, pDAC-hGH, pDAC-IFN and pDAC-EPO.<br>
According to one aspect of the invention, the recombinant expression vector pDAC-LacZ was constructed by inserting the polynucleotide sequence having SEQ ID NO: 9 encoding for the affinity tag peptide (SEQ ED NO: 1) into a plasmid vector.<br>
The nucleotide sequence coding for the affinity peptide handle in pDAC-LacZ is linked to a protease cleavage site which is Enterokinase (EK) recognition site. Other protease cleavage sites, such as Carboxy peptidase, Factor Xa, Trypsin, V8, and Chymotrypsin can also be used. Further, a chemical cleavage site can be inserted which is linked to the affinity peptide handle sequence wherein the chemical cleavage site is recognized by a reagent selected from a group consisting of cyanogen halide, hydroxyl amine, formic acid, acetic acid, hydrochloric acid and trifluoroacetic acid.<br>
The nucleotide sequence coding for affinity peptide handle in the pDAC-LacZ vector is further linked to the LacZ gene under the control of the AraB promoter. The LacZ gene can also be driven by other promoter, which is "selected from the group consisting of trp, lac, lac, osmB, CMV, EF-la, SV 40 and T7. The vector pDAC-LacZ gene has the rrnB transcription terminator. The pDAC-LacZ vector also comprises of ampicillin ORF, pMBl (pUC-derived) origin and AraC ORF. The detailed procedure for the construction of the pDAC-LacZ is provided in Example 4. The schematic map of the pDAC-LacZ vector (7.16 kb) is depicted in Figure 1. Nucleotide sequence of the pDAC-LacZ vector from nucleotide position 1 to 3781 is provided in SEQ ED NO: 33 (ref to Table No: 6) The nucleotide sequence coding for an affinity peptide handle is located at 322 to 444 bp and is followed by a EK recognition site located at the nucleotide position of 448 to 462 bp. This is followed by LacZ gene which starts from nucleotide position of 469 to 3428 bp. The LacZ gene is terminated by rrnB terminator located at 3516 to 3674 bp.<br><br>
The invention discloses different methods of constructing the recombinant expression vectors for production of the fusion protein. The nucleotide sequence (SEQ ID NO: 9) coding for the affinity peptide handle (SEQ ID NO: 1) is first synthetically produced using the methods known in the art and inserted into the plasmid vectors to produce recombinant expression vector such as pDAC-LacZ. Other nucleotide sequences (SEQ ID NO: 10, SEQ ID NO: 11 and SEQ ID NO: 12) coding for affinity peptide handle (SEQ ID NO: 2, SEQ ID NO: 3 and SEQ ID NO: 4) can also be inserted into the plasmid vector using the methods known in the art to produce recombinant expression vector such as pDAC-LacZ.<br>
The recombinant vector pDAC-LacZ was used to transform strain of E. coli cells such as JM109, DH5oc, Top 10, BL21, HB101, XLl-Blue and LMG19 to produce recombinant E. coli strain. Transformation of E. coli strains with the desired recombinant expression vector such as pDAC-LacZ is given in Example 7. This vector can also be transformed into other host cells such as Pseudomanas, Yeast, Saccharomyces, Pichia, Hanseneula, CHO, BHK, COS-and other mammalian cells.<br>
The recombinant pDAC-LacZ vector is responsible for the production of a fusion protein which comprises of at least two components, the affinity peptide handle and /3 galactosidase protein produced by the recombinant cells containing this vector.<br>
The recombinant cells containing the pDAC-LacZ vector were grown in a suitable medium under conditions well known in the art to produce the fusion protein. Example 8 gives details of growing the recombinant cells. The fusion protein was purified by using immobilized metal ion chelating chromatography. Details of the purification are given in the Example 9. Pure /? galactosidase protein was obtained from the fusion protein using protease or chemical cleavage method.<br>
Plasmid DNA was isolated and.analyzed by restriction with various enzymes to confirm the vector construction. Details are given in Example 7. SDS-PAGE gel electrophoresis of P-galactosidase protein obtained from recombinant vector pDAC-LacZ was carried out: Details are given in Example 8. SDS-PAGE analysis (Figure 3) shows binding ability P-galactosidase protein obtained from pDAC-LacZ to Ni-IDA, Cu-IDA columns and presence of p-galactosidase. proteins in, flow-through and wash fractions. However, in case of Cu-IDA, less P-galactosidase protein was seen in flow-through and wash fractions showing that Cu-IDA has higher binding efficiency than Ni-IDA column.<br>
Similarly, the recombinant expression vector such as pDAC-IL2 and pDAC-PDGFB were constructed as described above. These vectors can also be constructed by methods well known in the art. The recombinant vector pDAC-IL2 has the nucleotide sequence coding for<br><br>
affinity tag peptide linked to EK site or chemical cleavage site at position 322 to 444 which is further linked to IL-2 gene controlled by AraB promoter. The recombinant pDAC-PDGFB vector has the nucleotide sequence coding for the affinity peptide handle linked to EK site or chemical cleavage at position 322 to 444 which is further linked to PDGFB gene controlled by AraB promoter. These recombinant expression vectors were used for transformation into strains of E. coli cells such as JM109, DH5a, Top 10, BL21, HB101, XLl-Blue and LMG19 to produce recombinant E. coli strain. The recombinant cells containing these vectors were grown in the suitable medium under conditions as known in the art to produce fusion protein. The fusion protein was purified by using immobilized metal ion chelating chromatography. Further the desired protein was purified by chemical and or protease cleavage. The expression and binding analysis of pDAC-IL2, pDAC-PDGFB was carried out and details are explained in the Example 8-12. Genomic DNA extraction from E. coli<br>
Genomic DNA from E. coli (Top 10, HB101, DH5 alpha and JM109 strains) was isolated.  The details of the isolation of the genomic DNA from E. coli are given in Example 1. Construction of the plasmid vector containing the affinity tag sequence<br>
Amplification was carried out using specially designed primers (SEQ ID NO: 17 &amp; 18 (25 uM)), bacterial genomic DNA (Top 10, HB101 and JM109 strains ) as template and DNA polymerase enzyme (1-2.5 units//xl) under following amplification conditions: 25-35 cycles of denaturation at 95°C for 1-5 min, annealing at 54-63°C for 1-5 min and extension at 72°C for 1-10 min. The purified PCR product (designated as Fragment A) was used as a template for amplification with specifically designed primers having nucleotide sequence as shown in SEQ ID NO: 19 and SEQ ID NO: 20 ( See Example 1 for details ). The amplification methods are as described in Example 1.<br>
The amplified fragment (designated as FRAGMENT B) and a prokaryotic expression vector such as pBAD/His, pET series, pRA, pcDNA3.1 Myc/His were digested with restriction enzymes Agel and Sail in IX buffer at 25-55°C overnight. The digested fragment (designated as FRAGMENT 1) was purified using Qiaquick PCR purification kit (Qiagen) followed by ligation of "the digested products with the restricted vector (designated as FRAGMENT 2) to create an intemiediate vector (pDAC-I) containing partial affinity handle sequence (see Example 2 for details).<br>
The intermediate tag vector (pDAC-I) was used as a template DNA for amplification with specific primers (SEQ ID NO: 21 and 22). The amplified fragment was digested with<br><br>
restriction enzymes Ncol and Pvul to create a FRAGMENT 3 and a prokaryotic expression vector such as pRA was digested with the same enzymes i.e. Ncol and Pvul to create FRAGMENT 4, in IX buffer at 25-55°C overnight. Restricted FRAGMENT 3 and FRAGMENT 4 were extracted and purified using the Qiaquick PCR purification kit (Qiagen) followed by ligation to produce a second intermediate vector (pRA-DACI) which was used as a template for another round of amplification (see Example 3 for details).<br>
The above steps were repeated twice and both the PCR products were purified and used as template DNA for amplification with specific primers and resultant purified PCR product was digested over night with a restriction enzymes Age I and Hind III in respective IX buffer at 25-55°C. This was ligated to the digested expression vector such as pBAD/His, pET series, prokaryotic expression vector pRA, pcDNA3.1 Myc/His or other vectors known in the art to create the desired expression vector which has the affinity peptide handle sequence linked to a protease cleavage site. A DNA sequence of interest (coding for sequences such as p-galactosidase, human growth hormone, human GCSF, human PDGFB, bovine enterokinase, human insulin, human interleukin, interferon and other clinically therapeutic protein or peptide) was linked to the protease cleavage site. The gene sequences coding for biologically active polypeptide can be isolated from such vectors known in the art as pRA-LacZ, pRA-hGH, pRA-GCSF, pRA-PDGFB, pRA-EK, pRA-Insulin, and pRA-IL2.<br>
To have more copies of the affinity peptide handle in tandem, the above steps can be repeated as many times (it is possible to have 1-10 copies of the affinity peptide handle in tandem).<br>
The present invention provides construction of recombinant expression vector pDAC-LacZ vector which has the affinity peptide handle linked to EK site which is linked to LacZ gene controlled by AraB promoter (See Example 4 for details).<br>
Similarly, the expression vector pDAC-IL2 which comprises of the DNA sequences affinity peptide handle linked to EK site/ Chemical cleavage which are further linked to gene sequences coding for IL2 protein. The gene sequences in the vector pDAC-IL2 are under the control of AraB promoter. Other promoters known in the art may be employed for expression of the protein. The details for producing the expression vector are provided in Example 5.<br>
pDAC-PDGFB which has the affinity peptide handle linked to EK site/Chemical cleavage which is linked to PDGFB gene controlled by AraB promoter was constructed (see Example 6 for details).<br>
All of the above 3 constructs have the Ara B promoter with an Enterokinase protease cut  /  chemical   cleavage   site   followed  by  the   heterologous   gene,   rmB   transcription<br><br>
termination region, gene for ampicillin resistance, pMBl(pUC-derived) origin and the Am C gene. These constructs (pDAC-LacZ, pDAC-IL2, pDAC-PDGFB) were transformed in competent cells of E. coil (DH5a, Top 10, LMG19, JM109 strains). Plasmid DNA was isolated and analyzed by restriction with various enzymes to confirm the vector construction. Details are given in Example 7. Expression analysis of pDAC-LacZ, pDAC-lL2, pDAC-PDGFB was carried out. The binding of the fusion proteins was carried out. Details are explained in the Example 8-12.<br>
Enhanced genome annotation of the affinity peptide handle sequence was done using structural profiles in the programme 3D-PSSM (Kelley et al, 2000). The data was generated by the 3D-PSSM web server, Structural Bioinformatics group, Imperial College of Science, Technology and Medicine, UK. The predicted secondary structures that involve at least 3 amino acid residues binding to metal (e.g. Nickel) are selected from the E_score-predicted secondary structure of the sequence. E_scores are the probable structure predictors. From the Chou-Fasman analysis it is clear that two amino acid residues involved in metal binding belong to a seemingly alpha helical domain and one residue belongs to a turn.<br>
Analysis of physicochemical properties of the affinity peptide handle was done using Protean module of Lasergene software. From the SDS-PAGE Simulation Analysis of protease cut sites of the affinity peptide handle using Protean module (Lasergene software, DNASTAR), it is seen that the affinity peptide handle is not cut by proteases like Enterokinase, Factor Xa, Trypsin and endopeptidase and hence the affinity peptide handle can be used in a genetic construct with either upstream or downstream cut sites of such proteases. Details are given in Example 13.<br>
EXAMPLES<br>
It should be understood that the following examples described herein  are  for illustrative purposes only and that various modifications or changes in light will be suggested to persons skilled in the art and are to be included within the spirit and purview of this application and the scope of the appended claims.<br>
 Example 1<br>
 Isolation of genomic DNA and PCR amplification<br>
Genomic DNA was isolated from E. coli strain JM109. 1.5 ml o/n bacterial culture (Top 10, HBlOl, DH5-a and JM109) was transferred into a micro centrifuge tube and centrifuged at 6000 rpm, 5 min. Supernatant was decanted and the pellet was resuspended in TE buffer. SDS (Final 0.3%) and proteinase K were added, mixed well and incubated for 1 hr at 37 C.  An equal volume of phenol/chloroform was added and mixed well by inverting the<br><br>
tube until the phases were completely mixed. DNA/phenol mixture was carefully transferred into a microcentrifuge tube and centrifuged for 2 min at 14 K rpm at room temperature. The upper aqueous phase was transferred to a new tube and an equal volume of phenol/chloroform was added, mixed well and centrifuged for 2 min at 14 K rpm at RT. One tenth volume of sodium acetate was added to the upper aqueous phase which was transferred to a fresh microcentrifuge tube. DNA was precipitated by adding 0.6 volumes of isopropanol on ice for 5 to 8 minutes. DNA was pelleted by centrifugation at 14 K rpm for 15 min at room temperature and washed with 70% ethanol. DNA was dissolved in 100-200 ul TE buffer and kept at 37°C for 10-15 minutes.<br>
Bacterial genomic DNA from JM109 strain of E. coli was used as the template for amplification using specially designed primers U-l and L-l (SEQ ID No: 17 and 18), and PAfu polymerase enzyme (1- 2.5 units//xl, MBI) in IX PCR buffer (MBI) containing 20mM Tris-HCl, pH 8.8, 10 mM each of (NH4)2S04 and KCl, 0.1% Triton X-100, O.lmg/ml BSA and 2 mM MgS04 under following cycling conditions : 4 cycles of denaturation at 95°C for 1 min, annealing at 60°C for 2 min and extension at 72°C for lmin followed by 30 cycles of denaturation at 95°C for 1 min, annealing at 63°C for 2 min and extension at 72°C for 1 min. For analysis of the PCR products, 5-10 /xl of sample was mixed with 1-10 ml of IX loading . dye and run on 1.0 % agarose gel.<br>
U-l: 5' - GGTGTCGGCGATTTTTAACCGTGACTATC -3'	SEQ ID NO: 17<br>
L-l: 5' - CTGCCGCTACCGCCGACTAACG -3'	SEQ ID NO: 18<br>
Purified the 1099 bp PCR product (hereafter designated as Fragment A), using Qiaquick PCR purification kit (Qiagen). This purified fragment was used as the template for further amplifications. <br>
Example 2 <br>
Preparation of pDAC intermediate vector<br>
Restriction enzyme site(s) was introduced during synthesis of the primers thus making possible the introduction of restriction enzyme sites at the 5' and 3' end of the amplified product. Fragment A was used as template for amplification with primers U-2 (has Age I site, SEQ ID No: 19) and L-2 (has Sal I site, SEQ ID No: 20). Amplification was carried out under the condition described in Example 1.<br>
U-2: 5'-CTGGCGCTGACCGGTATCGCCCTGCTGGCGCTGATT -3' SEQ ID NO: 19<br>
L-2:5'-CGTAGTTGTCGACATGAGGGTCCAGGTGTGCGTGCTCCGTCA-3' SEQ ID NO: 20<br><br>
Amplified product of 820 bp was purified using Qiaquick PCR purification kit (Qiagen) and digested with restriction enzymes, Age I and Sal I, in buffer containing 50 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 100 mM NaCl and O.lmg/ml BSA at 37° C overnight. Fragment of 802 bp (hereafter designated as FRAGMENT 1) was purified using Qiaquick PCR purification kit (Qiagen). DNA sample was mixed with 3-5 volumes of the Buffer PB provided in the kit and applied to a Qiaquick column. This was centrifuged for 30-60 sec at 14 K rpm and the flow through was discarded. The membrane was washed with the wash buffer provided and the DNA was eluted with nuclease free distilled water.<br>
pRA vector DNA was digested with Age I and Sal I in buffer containing 50 mM Tris-HCl, pH 7.5, 10 mM MgC12, 100 mM NaCl and O.lmg/ml BSA at 37°C overnight. The 3784 bp fragment (hereafter designated as FRAGMENT 2) and not the smaller 275 bp fragment, was purified as mentioned in Example 1. Ligation of FRAGMENT 1 and FRAGMENT 2 was carried out using 2 units of T4 DNA ligase in presence of buffer containing 40 mM Tris HC1, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP at 37° C for 2 hours and overnight at 4°C resulting in pDAC-Intermediate vector (pDAC-I) , which was purified using Qiaquick PCR purification kit (Qiagen). Example 3 Preparation of pRA-DAC-I vector<br>
PDAC-Intermediate (pDAC-I) vector was used as the template for amplification with primers U-3 (has Nco I site SEQ ID No: 21) and L-3 (has Pvu I site, SEQ ID No: 22) in a thermal cycler machine.<br>
U-3: 5' - GGTGGCCATGGCACTGCACGCACATCTGGACCCTCAT -3'  SEQ ID NO: 21<br>
L-3: 5' - GTTCCCAACGATCAAGGCGAGTTACATGATC-3'	SEQ ID NO: 22<br>
This amplified product of 1160 bp was purified using Qiaquick PCR purification kit (Qiagen) and digested with Nco I and Pvu I in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM Potassium acetate and O.lmg/ml BSA at 37° C overnight. The 1080 bp fragment (hereafter designated as FRAGMENT 3) was purified using Qiaquick PCR purification kit (Qiagen).<br>
pRA vector DNA was digested with Nco I and Pvu I in buffer containing 33mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66mM Potassium acetate and O.lmg/ml BSA at 37°C overnight and 3012bp fragment (hereafter designated as FRAGMENT 4) was purified using Qiaquick PCR purification kit (Qiagen) and not the smaller 1047 bp fragment. Ligation of FRAGMENT 3 with FRAGMENT 4 was set up using 2 units of T4 DNA ligase in presence<br><br>
of buffer containing 40 mM Tris HCl, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP<br>
at 37° C for 2 hours and then overnight at 4°C to produce pRA-DAC-I vector.<br>
Example 4<br>
Preparation of pDAC-LacZ vector<br>
pRA-DAC-I vector DNA was used as the template for amplification with primers U-4 (has Age I site, SEQ ID No: 23) and L-4 (has 6 more codons of the tag, SEQ ID No: 24) in a thermal cycler machine. Amplified product of 302 bp fragment (hereafter designated as FRAGMENT 5) was purified using Qiaquick PCR purification kit (Qiagen).<br>
U-4: 5' - GCTAACCAAACCGGTAACCCCGCTTATTAA - 3'	SEQ ID NO: 23<br>
L-4: 5' - GGCGTGCTCCGTCACCAGATGAGGGTCCAGATGT- 3'   SEQ ID NO: 24<br>
pRA-DAC-I vector DNA was used as the template for amplification with primers U-5 (has 11 codons of the tag, SEQ ID No: 25) and L-5 (SEQ ID No: 26). Amplified product of 870 bp fragment (hereafter designated as FRAGMENT 6) was purified using Qiaquick PCR purification kit (Qiagen).<br>
U-5:5'-CTGGTGACGGAGCACGCCCACCTCGATCCGCACGTCGACGA-3'SEQ ID NO: 25 L-5:5'-GGATAATACCGCGCCACATAGCAGAACTTTAAAAGTGCT-3' SEQ ID NO: 26<br>
Purified FRAGMENT 5 and FRAGMENT 6 were used as the templates for amplification with primers U-6 having Age I site (SEQ ID No: 27) and L-6 having Hind III site (SEQ ID No: 28). Amplified product of 630 bp was purified and digested with Age I and Hind III in buffer containing 10 mM Tris-HCl, pH 7.5, 10 mM MgCL2, 100 mM NaCl and O.lmg / ml BSA at 37° C overnight. The digested 382 bp fragment (hereafter named as FRAGMENT 7) was purified using Qiaquick PCR purification kit (Qiagen) and not the smaller 248 bp fragment.<br>
U-6: .5'-CTAACCAAACCGGTAAACCCCGCTTATTAAA- 3'	SEQ ID NO: 27<br>
L-6: 5'-CCAGTCTTTCGACTGAGCCTTTCGTTTTATT- 3'	SEQ ID NO: 28<br>
pRA vector DNA was digested with Age I and Hind III in buffer containing 10 mM Tris-HCl, pH 7.5, 10 mM MgCL2, 100 mM NaCl and O.lmg / ml BSA at 37° C overnight and 3710 bp fragment was purified using Qiaquick PCR purification kit (Qiagen) (hereafter designated as FRAGMENT 8) and not the smaller 349 bp fragment. Ligation of FRAGMENT 7 with FRAGMENT 8 was carried out using 2 units of T4 DNA ligase in presence of buffer containing 40 mM Tris HCl, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5mM ATP at 37°C for 2 hours and overnight at 4°C to produce pRA-T vector of size 4092 bp.<br>
pRA-T vector DNA was digested with Age I and Sal I in buffer containing 50 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 100 mM NaCl and O.lmg/ml BSA at 37°C overnight and<br><br>
308 bp fragment (hereafter designated as FRAGMENT 9) was purified using Qiaquick PCR purification kit (Qiagen) and not the larger 3784 bp fragment. pRA-LacZ vector DNA was digested with Age I and Sal I in buffer containing 50 mM Tris-HCl, pH 7.5, 10 mM MgCl2, 100 mM NaCl and O.lmg/ml BSA at 37°C overnight and 6791 bp fragment was purified using Qiaquick PCR purification kit (Qiagen) (hereafter designated as FRAGMENT 10) and not the smaller 275 bp fragment. FRAGMENT 9 was ligated with FRAGMENT 10 using 2 units of T4 DNA ligase in presence of buffer containing 4 mM Tris HC1, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP at 37°C for 2 hours and overnight at 4°C to produce pT-LacZ vector (7099 bp). The purified ligated product was used as a template for amplifications using specifically designed primers.<br>
pT-LacZ vector DNA was used as the template, for amplification using primers U-7 (has Age I site, SEQ ID No: 29) and L-7 (has Nhe I site, SEQ ID No: 30) in a thermal cycler machine.<br>
U-7: 5' - GCTCTTCTCGCTAACCAAACCGGTAAC -3'	SEQ ID NO: 29<br>
L-7: 5' - CGTCGTCGCTAGCGTGCGGATCGAGGT -3'	•	SEQ ID NO: 3 0<br>
Amplified product of 339 bp was purified Qiaquick PCR purification kit (Qiagen) and digested with Age I and Nhe I in buffer containing 33mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37° C overnight. Fragment of 311 bp was purified using Qiaquick PCR purification kit (Qiagen) (hereafter named as FRAGMENT 11).<br>
pT-LacZ vector DNA was used as the template for amplification using primers U-8<br>
(has Nhe I site, SEQ ID No: 31) and L-8 (has Aat II site, SEQ ID No: 32).<br>
U-8: 5'-GGAGGAATTAACCGCTAGCCTGCACGCACATCTGGA-3'      SEQ ID NO: 31<br>
L-8: 5'-GCAGCAACGAGACGTCACGGAAAAT-3'	SEQ ID NO: 32<br>
Amplified product of 733 bp was purified using Qiaquick PCR purification kit (Qiagen) and digested with Nhe I and Aat II in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and 704 bp fragment was purified (fragmenthereafter designated as FRAGMENT 12).<br>
FRAGMENT 11 with FRAGMENT 12 were ligated using 2 units of T4 DNA ligase in presence of buffer containing 40 mM Tris HC1, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP at 37°C for 2 hours and overnight at 4°C and 1015 bp ligated fragment was purified using Qiaquick PCR purification kit (Qiagen) (hereafter designated as FRAGMENT 13).<br>
pT-LacZ vector DNA was digested with Age I and Aat II in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and 6146 bp fragment was purified using Qiaquick PCR purification kit (Qiagen)<br><br>
(hereafter designated as FRAGMENT 14) and not the smaller 953 bp fragment. FRAGMENT<br>
13 and FRAGMENT 14 were ligated using 2 units of T4 DNA ligase in presence of buffer<br>
containing 40 mM Tris HCl, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP (pH 7.8) at 37°C<br>
for 2 hours and overnight at 4°C to produce pDAC-LacZ vector (7162 bp) which has the<br>
affinity tag peptide linked to EK site which is further linked to LacZ gene controlled by AraB<br>
promoter.<br>
Example 5<br>
Preparation of pDAC-IL2 vector<br>
pDAC-LacZ vector DNA was digested with Sal I and Hind III in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and the 4081 bp fragment (hereafter designated as FRAGMENT 15) was purified using Qiaquick PCR purification kit (Qiagen) and not the smaller 3081 bp fragment. The recombinant vector pRA-IL2 was digested with Sal I and Hind III in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and the smaller 419 bp fragment (hereafter designated as FRAGMENT 16) was purified using Qiaquick PCR purification kit (Qiagen) and not the larger 3984 bp fragment. Ligation of FRAGMENT 15 with FRAGMENT 16 was carried out using 2 units of T4 DNA ligase in presence of buffer containing 40 mM Tris HCl, pH 7.8, 10 mM MgCl2, 10 mM DTT and 0.5 mM ATP at 37°C for 2 hours and overnight at 4°C to produce pDAC-IL2 vector (4499 bp) which has the affinity tag peptide linked to chemical cleavage site which is further linked to IL2 gene controlled by AraB promoter.<br>
 Example 6 <br>
Preparation of pDAC-PDGF vector<br>
pDAC-LacZ vector DNA was digested with Sal I and Hind III in buffer containing 33-mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and the 4081 bp fragment (hereafter designated as FRAGMENT 15) was purified using Qiaquick PCR purification kit (Qiagen) and not the smaller 3081 bp fragment: Recombinant vector pRAZ-1-PDGFB DNA was digested with Sal I and Hind III in buffer containing 33 mM Tris-acetate, pH 7.9, 10 mM Mg-acetate, 66 mM K-acetate and O.lmg/ml BSA at 37°C overnight and the smaller 402 bp (fragment hereafter designated as FRAGMENT 17) was purified using Qiaquick PCR purification kit (Qiagen) and not the larger 4110 bp fragment. FRAGMENT 15 was ligated with FRAGMENT 17 using 2 units of T4 DNA ligase in presence of buffer containing 40 mM Tris HCl, pH 7.8, 10 mM MgCl2] 10 mM DTT and 0.5 mM ATP at 37°C for 2 hours and overnight at 4°C to produce pDAC-<br><br>
PDGFB vector (4483 bp) which has the affinity tag peptide linked to EK site which is further linked to PDGFB gene controlled by AraB promoter. <br>
Example 7 <br>
Transformation of cells<br>
The constructs pDAC-LacZ, pDAC-IL2 and pDAC-PDGFB were transformed in competent cells of E. coil strains such as DH5-a, Top 10, LMG19, JM109. Required amount of competent cells were thawed on ice. Ligation mix was transferred into the tube containing the competent cells, mixed gently without pipetting or vortexing and incubated on ice for 30 min. Cells were subjected to heat shock at 42°C / 2 min and incubated on ice for 5 min. One ml of appropriate medium without antibiotic was added and cells were grown at 37°C/lhr with shaking. Cells were pelleted at 3000 rpm/5mins, resuspended in 100 ml appropriate medium without antibiotic and spread on an agar plate containing appropriate medium with antibiotic. Plates were incubated at 37°C in incubator for 12-18 hrs. Colonies obtained on the agar plate were inoculated in 3 ml liquid media containing antibiotic and allowed to grow at 37°C o/n with shaking. <br>
Plasmid isolation and analysis<br>
Plasmid DNA was isolated by alkaline lysis method from 1.5ml o/n grown cultures by standard methods in prior art. DNA was subjected to restriction enzyme digestions to confirm the vector construction. The DNA of interest was cleaved with a variety of restriction endonucleases, either individually or in combination and the resulting products were separated by agarose gel electrophoresis. By determining the sizes of DNA fragments produced by the action of the endonucleases, the restriction map was deduced progressively from simple situations where enzymes cleave the DNA once or twice to more complex situations where cleavage occurs more frequently.<br>
Positive clones were sequenced using the automated DNA sequencer (ABI Prism 310 Genetic Analyzer). The plasmid was purified either through columns or by PEG precipitation (Kraft et al, 1988). To the purified DNA (50 ng to 500 ng), 4-8 m1 of the terminator ready reaction mix was added. This ready reaction mix is composed of pre-mixed dNTPs, dye terminator, ampliTaq DNA polymerase, MgCl2 and buffer. On addition of 1 m1 of primer (5 pmolesml), samples underwent cycle sequencing in a thermal cycler (25 cycles of 94°C for 10 sec, 50°C for 5 sec and 60°C for 4 min). The resulting products were precipitated with 2.7 M sodium acetate (pH 4.6) and ethanol and washed twice with 70% ethanol. DNA pellet was dissolved in formamide. Samples were analyzed in the automated sequencer (ABI Prism 310 Genetic Analyzer).<br><br>
Example 8<br>
Expression analysis of pDAC-LacZ, pDAC-IL2, pDAC-PDGF<br>
The bacterial cells transformed with the recombinant expression vector (pDAC-LacZ, pDAC-IL2, pDAC-PDGF, pDAC-GCSF, pDAC-EK, pDAC-hGH, pDAC-IFN, pDAC-EPO) containing the affinity handle (affinity tag peptide) and DNA sequence of interest (e.g. (3-galactosidase, IL2, PDGF) were grown in liquid media in presence of appropriate antibiotic. The cells were grown either till log (O.D 60onm ~ 0.5) or at a stationary phase (16 hrs growth) at 37°C in an incubator shaker in 1 liter fermentor. Then an appropriate amount (0-7%) of the inducer was added to the culture and incubated further for required time (0-72 hrs) at 37°C with shaking. The bacterial cells were harvested by centrifuging at 4,000 rpm for 10 min at 4°C. The bacterial pellet was washed three times with ice-cold IX PBS and resuspended in 200 /xl of IX PBS. Bacterial cell extracts were prepared by subjecting the cells to four cycles of rapid freezing in liquid nitrogen, followed by thawing at 37°C. Cells were vortexed vigorously for 5 min and centrifuged at 14,000 rpm for 10 min at 4°C. The supernatant was transferred to a fresh 1.5ml eppendorff tube. Total protein of the cell extract was estimated by Bradford Method. Colorimetric estimation of (b-galactosidase protein was done using O-nitrophenyl b-D-Galacto-Pyranoside (ONPG, Sigma) as the substrate at 405 nm O.D. (Table: 2).<br>
Table 2: Expression levels of the reporter gene by enzymatic assay<br><br>
Sample	b-galactosidase / ug protein<br>
pDAC-LacZ, Uninduced, 4hrs	0.7<br>
pDAC-LacZ, Induced with 0.2% L-Arabinose, 4hrs	26.5<br>
To determine expression levels of the reporter gene by SDS-PAGE, cell pellets were sonicated in lysis buffer and centrifuged to get clear cell-lysate.- After protein content was estimated by Bradford method, the required amount of sample was mixed with the sample buffer and heated at 90°C for 5 min. The samples were pulse spun and loaded on a SDS-PAGE gel (10- 20%) immediately. After the electrophoretic run, the gel was stained with Coomassie blue. <br>
Example 9<br>
 Binding Studies of b-galactosidase protein<br>
Cell pellets were resuspended in 25 ml lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl) and sonicated for 10 min (30 sec on/off pulse mode). Cell lysates were centrifuged for<br><br>
l5 min at 15,000 rpm and the clear supernatant was used for affinity chromatography. Twenty-five mg of clear supernatant was used for studying the binding of LacZ protein with the affinity tag peptide to metal affinity beads. Chelating Sepharose beads (Amersham Biosciences) were packed in a 1ml PD-10 column. The beads were allowed to settle and washed with 10 ml distilled water, followed by 10 ml of 100 mM NiS04 or CuS04, and then again with 10 ml of distilled water and finally with 10 ml of equilibration buffer (50 mM Tris, pH 8.0, 150 mM NaCl) with and without 20 mM imidazole.<br>
Sample (with and without 20 mM imidazole) was loaded onto the column at 1 ml/min flow rate and washed with 5 ml of respective buffers (with and without 20 mM imidazole) to remove unbound proteins. Bound protein in all experiments was eluted with 2 ml of respective buffer containing 250 mM Imidazole. Protein estimations of all relevant fractions were done using Bradford's protein assay kit from Pierce. Samples were analyzed on 12% SDS-PAGE gel followed by Coomassie staining (Table: 3). Table: 3 Protein estimations for pDAC-LacZ sample<br><br>
Exptal. conditions	Cone.          in mg/ml	Total   vol   in ml	Total in mg	protein<br>
Total Protein loaded onto the column	8.07	3.09	25.0	<br>
Ni-IDA elution(- imidazole in eq. buffer)	1.54	2	3.08	<br>
Ni-IDA elution(+ imidazole in eq. buffer)	1.66	2	3.32	<br>
Cu-IDA elution (-imidazole in eq. buffer)	2.49	2	4.98	<br>
Cu-IDA elution(+ imidazole in eq. buffer)	1.94	2	3.88	<br>
Binding efficiency of expressed P-galactosidase protein to Ni-IDA was found to be better when bound in presence of equilibrating buffer which does not contain Imidazole. SDS-PAGE analysis (Fig: 2) showed the presence of P-galactosidase proteins in flow-through and wash fractions. However, in case of Cu-IDA, less p-Galactosidase protein was seen in flow-through and wash fractions showing that Cu-IDA has higher binding efficiency than Ni-IDA although column. <br>
Example 10<br>
 Analysis of pDAC-LacZ expression and binding<br>
Cell pellets were resuspended in 25 ml lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl) and sonicated for 10 min (30 sec on/off pulse mode). Cell lysates were centrifuged for 15 min at 15,000 rpm and the clear supernatant was used for affinity chromatography. Clear<br><br>
supernatant of pDAC-LacZ (55 mg) was used for binding analysis. A 3 ml column of<br>
chelating Sepharose (Amersham Biosciences) was packed into an empty PD-10 column. The<br>
column was charged with 100 mM of CuS04 and equilibrated with 50 mM Tris, pH 8.0, 150<br>
mM NaCl, 20 mM imidazole. Imidazole was added to the sample, to give 20 mM final<br>
concentration. Sample was loaded onto the column at 1 ml/min flow rate. After complete<br>
loading, unbound proteins were washed off with the same buffer. Bound protein was eluted<br>
with above buffer containing 250 mM imidazole. Protein estimations of all relevant fractions<br>
were done using Bradford's protein assay kit (Pierce). The results are given in Table 4.<br>
Samples were analyzed on 12% SDS-PAGE electrophoresis. The gel was stained with<br>
Coomassie blue.<br>
Table: 4 Protein estimations for pDAC-LacZ clone sample	<br><br>
Sample	Cone, (mg/ml)	Total vol (ml)	Total protein (mg)<br>
Total protein loaded onto the column	5.5	10	55<br>
Cu-IDA elution	1.12	6	6.72<br>
Example 11<br>
Expression analysis of pDAC-IL2<br>
The bacterial cells transformed with the expression vector pDAC-IL2 were grown in liquid media containing ampicillin (50 ug/ml). The cells were grown to stationary phase (16 hrs growth) at 37°C in an incubator shaker. An appropriate amount (0.2% final concentration) of L-Arabinose was added to the culture and incubated further for 7 hrs at 37°C with shaking. The bacterial cells were harvested by centrifuged at 4000 rpm for 10 min at 4°C. Cell pellets were resuspended in 1.5 ml lysis buffer (50 mM Tris, pH 8.0, 150 mM NaCl). Samples were sonicated to disrupt cells and total lysate was spun at 15,000 rpm for 10 min. Protein was estimated from total lysate, supernatant and pellet. Ten ug of protein was analyzed on 15% SDS-PAGE and gel was stained with Coomassie stain. <br>
Example 12 .<br>
 Binding studies of expressed DAC-IL2 protein<br>
Cell pellets were solubilized in buffer containing 50 mM Tris, pH 8.0, 150 mM NaCl, 8M urea and 20 mM imidazole for 3 hrs. 1ml Ni-IDA beads were packed and charged with 5 column volumes of 0.1 M Ni-S04. The column was equilibrated with equilibration buffer (50 mM Tris, pH 8.0, 150 mM NaCl, 8 M urea, and 20 mM imidazole) and -5.5 mg protein was loaded into the column. Flow through was collected. Column was washed with 2 column volumes of equilibration buffer and protein was eluted with elution buffer containing 250 mM imidazole. Amount of protein in the different fractions was estimated and analyzed by SDS-PAGE electrophoresis (Table: 5).<br><br>
Table: 5 Protein estimations for pDAC-IL2 sample<br><br>
Sample	Concentration (mg/ml)	Volume in ml	Total   protein (mg)<br>
Ni-IDA Starting material	3.05	1.8	5.49<br>
Flow through/ Wash	0.77	2	1.54<br>
Elution	3.5	1	3.5<br>
The expressed DAC-IL2 protein binds to Ni-IDA affinity matrix. Non-specific bound<br>
protein is removed mostly by a 40 mM imidazole wash while the desired protein is eluted<br>
with buffer containing 80 and 250 mM imidazole.<br>
Example 13<br>
Enhanced genome annotation of the tag sequence was done using structural profiles in the<br>
programme 3D-PSSM (Kelley et al, 2000). The data was generated by the 3D-PSSM web<br>
server, Structural Bioinformatics group, Imperial College of Science, Technology and<br>
Medicine, UK. The predicted secondary structures that involve at least 3 amino acid residues<br>
binding to metal (eg Nickel) are selected from the E_score-predicted secondary structure of<br>
the sequence. E_scores are the probable structure predictors. From the Chou-Fasman analysis<br>
it is clear that two amino acid residues involved in metal binding belong to a seemingly alpha<br>
helical domain and one residue belongs to a turn. Physicochemical properties of the novel<br>
affinity tag peptide  was analyzed using Protean software (Lasergene) and is given below -<br>
Predicted Structural Class of the Whole Protein: R (Deleage &amp; Roux Modification of<br>
Nishikawa &amp; Ooi 1987).<br>
Molecular Weight: 4589.06 dalton Length: 41 amino acids Strongly basic (+) amino acids (K, R): 0 Strongly acidic (-) amino acids (D, E): 6 Hydrophobic amino acids (A, I, L, F, W, and V): 16 Polar amino acids (N, C, Q, S, T, Y):3 1 microgram =217.910 pmoles Molar Extinction coefficient: 0 + 5% 1 A(280):        No CYWs Isoelectric Point: 6.30 Charge at pH 7: -4.09 Results   from  Predictprotein   (www.predictprotein.org)   gave   the   following  MAXHOM<br>
alignment.<br>
Identities computed with respect to: (1) predict_h2630<br>
Colored by: consensus / 70% and property<br>
1	predict_h2630   100.0%        ALHAHLDPHLVTEHAHLDPHASLHAHLDPHLVTEHAHLDPH<br>
2	nikc_ecoli	87.5%	ALRDHLDPHLVTEHAH	-	-	-<br>
Consensus 7100%	AL+S HLDPHLVTEHAH	<br><br>
Consensus /90%	AL+S HLDPHLVTEHAH	<br>
Consensus /80%	AL+S HLDPHLVTEHAH...	<br>
Consensus /70%	AL+S HLDPHLVTEHAH	<br>
The above results indicate that a portion of the affinity peptide handle (affinity tag peptide) has 87.5% homology with E. coli nik c protein. A portion of the sequence information was used to build Fragment A.<br>
Table No. 6 - the following table provides SEQ ID's with Sequence listings mentioned in the specification.<br><br>
SEQ ID NO	PROTEIN/NUCLEOTIDE SEQUENCE	<br>
1	MALHAHLDPH LVTEHAHLDP HASLHAHLDP HLVTEHAHLD PHVDDDDK<br>
2	MVAHLHASPH AADTHVHASP HVTAHVHASP HAADTHVHAS PHLVDDDDK	<br>
3 '	MLVHLHVEPH VLSDHLHVGP HLDVHLHVGP HVLSDHLHVE PHLSVDDDDK	<br>
4	MGLHVHGTPH GLESHGHGTP HGEGHLHGTP HGLESHGHVT PHGVASLDDD DK	<br>
5	ALHAHLDPHL VTEHAHLDPH ASLHAHLDPH LVTEHAHLDP H	<br>
6	ALHAHLDPHL VTEHAHLDPH ASLHAHLDPH LVTEHAHLDP H	<br>
'   7   •	LVHLHVGPAV LSDHLHVGPH LDVHLHVEPH VLSDHLHVEP H	<br>
8	GLHVHGTPHG LESHGHGTPH GEEHLHGTPH GLESHGHVTP H	<br>
9	atggcactgc acgcacatct ggaccctcat ctggtgacgg agcacgccca cctcgatccg cacgctagcc tgcacgcaca tctggaccct catctggtga cggaacacgc ccacctcgat ccgcacgtcg acgacgacga caag	60120 144<br>
10	atggtggcac atctgcatgc aagcccgcat gcagcagata cccatgttca tgcaagcccg catgtgaccg cacatgttca tgcaagcccg catgcagcag atacccatgt tcatgcaagc ccgcatttag tcgacgacga cgacaag	60 120 147<br>
11	atgctggtgc atctgcatgt tgaaccgcat gttctgagcg atcatctgca tgttgaaccg catctggatg tgcatctgca tgtggaaccg catgtgctga gcgatcatct gcatgtggaa ccgcatctgt ctgttgacga cgacgacaag	60 120 150<br>
•   12	atgggtctgc atgtgcatgg taccccgcat ggtctggaaa gccatggtca tggtaccccg catggtgaag gtcatctgca tggtaccccg catggtctgg aaagccatgg tcatgtgacc ccgcatggtg ttgcaagctt agacgacgac gacaag	60 120 156<br>
13	gcactgcacg cacatctgga ccctcatctg gtgacggagc acgcccacct cgatccgcac gctagcctgc acgcacatct ggaccctcat ctggtgacgg aacacgccca cctcgatccg cac	60120123<br>
14	gtggcacatc tgcatgcaag cccgcatgca gcagataccc atgttcatgc aagcccgcat gtgaccgcac atgttcatgc aagcccgcat gcagcagata cccatgttca tgcaagcccg cat	60 120 123<br>
15	ctggtgcatc tgcatgttga accgcatgtt ctgagcgatc atctgcatgt tgaaccgcat ctggatgtgc atctgcatgt ggaaccgcat gtgctgagcg atcatctgca tgtggaaccg cat	60 120 123<br>
16	ggtctgcatg tgcatggtac cccgcatggt ctggaaagcc atggtcatgg taccccgcat ggtgaaggtc atctgcatgg taccccgcat ggtctggaaa gccatggtca tgtgaccccg cat	60 120 123<br>
17	ggtgtcggcg atttttaacc gtgactatc	29<br><br>
18	ctgccgctac cgccgactaa eg	22<br>
19	ctggcgctga ccggtatcgc cctgctggcg ctgatt	36<br>
20	cgtagttgtc gacatgaggg tccaggtgtg cgtgctccgt ca	42<br>
21	ggtggccatg gcactgcacg cacatctgga ccctcat	37<br>
22	gttcccaacg ateaaggega gttacatgat c	31<br>
23	gctaaccaaa ccggtaaccc cgcttattaa	30<br>
24	ggcgtgctcc gtcaccagat gagggtccag atgt	34<br>
25	ctggtgacgg agcacgccca cctcgatccg cacgtcgacg a	41<br>
26	ctaaccaaac cggtaacccc gcttattaaa	30<br>
27	ctaaccaaac cggtaacccc gcttattaaa     '	30<br>
28	ccagtctttc gaetgagect ttcgttttat t	31<br>
29	gctcttctcg ctaaccaaac eggtaac	27<br>
30	cgtcgtcgct agcgtgcgga tcgaggt	27<br>
31	ggaggaatta accgctagcc tgcacgcaca tctgga	36<br>
32	gcagcaacga gaegtcaegg aaaat	25<br>
33	aagaaaccaa ttgtccatat tgeatcagae attgeegtea ctgcgtcttt tactggctct	60<br>
	tctcgctaac caaaceggta accccgctta ttaaaagcat tctgtaacaa agegggacca	120<br>
	aagecatgae aaaaacgcgt aacaaaagtg tctataatca eggcagaaaa gtccacattg	180<br>
	attatttgea cggcgtcaca etttgetatg ccatagcatt tttatccata agattagegg	240<br>
	atcctacctg aegcttttta tcgcaactct ctactgtttc tccatacccg ttttttgggc	300<br>
	taacaggagg aattaaccat ggcactgcac gcacatctgg accctcatct ggtgacggag	360<br>
	cacgcccacc tcgatccgca cgctagcctg cacgcacatc tggaccctca tctggtgacg	420<br>
	gaacacgccc acctcgatcc gcacgtcgac gacgacgaca aggatccaat gatagatccc	480<br>
	gtcgttttac aacgtcgtga ctgggaaaac cctggcgtta cccaacttaa tcgccttgca	540<br>
	gcacatcccc ctttcgccag ctggcgtaat agegaagagg cccgcaccga tcgcccttcc	600<br>
	caacagttgc geagectgaa tggcgaatgg cgctttgcct ggtttccggc accagaagcg	660<br>
	gtgccggaaa gctggctgga gtgegatett cctgaggccg atactgtcgt cgtcccctca	720<br>
	aactggcaga tgcacggtta cgatgcgccc atctacacca aegtgaccta tcccattacg	780<br>
	gtcaatccgc cgtttgttcc caeggagaat ccgacgggtt gttactcget cacatttaat	840<br>
	gttgatgaaa gctggctaca ggaaggccag aegegaatta tttttgatgg cgttaactcg	900<br>
	gcgtttcatc tgtggtgcaa egggegctgg gtcggttacg gecaggacag tcgtttgccg	960<br>
	ctgaatttga cctgagcgca tttttacgcg ccggagaaaa ccgcctcgcg gtgatggtgc	1020<br>
	tgcgttggag tgacggcagt tatctggaag atcaggatat gtggcggatg ageggcattt	1080<br>
	tccgtgacgt ctcgttgctg cataaaccga ctacacaaat cagegattte cagttgccac	1140<br>
	tegctttaafgatgatttea gccgcgctgt actggaggct gaagttcaga tgtgcggcga	1200<br>
	gttgcgtgac tacctaeggg taacagtttc tttatggcag ggtgaaacgc aggtcgecag	1260<br>
	cggcaccgcg cctttcggcg gtgaaattat egatgagegt ggtggttatg ccgatcgcgt	1320<br>
	cacactacgt ctgaaegteg aaaacccgaa actgtggagc gccgaaatcc cgaatctcta	1380<br>
	tcgtgcggtggttgaactgcacaccgecga cggcacgctg attgaagcag aagectgega	1440<br>
	tgtcggtttc cgcgaggtgc ggattgaaaa tggtctgctg ctgctgaacg geaagcegtt	1500<br>
	gctgattcga ggcgttaacc gtcacgagca tcatcctctg catggtcagg tcatggatga	1560<br>
	gcagacgatg gtgeaggata tectgetgat gaagcagaac aactttaacg ccgtgcgctg	1620<br>
	ttegcattat ccgaaccatc cgctgtggta.cacgctgtgc gaccgctacg gcctgtatgt	1680<br>
	ggtggatgaa gecaatattg aaacccacgg catggtgcca atgaatcgtc tgaccgatga	1740<br>
	tccgcgctgg ctaccggcga tgagcgaacg egtaacgega atggtgcagc gcgatcgtaa	1800<br>
	tcacccgagt gtgatcatct ggtcgctggg gaatgaatca ggccacggcg ctaatcacga	1860<br>
	cgcgctgtat cgctggatca aatctgtcga tccttcccgc ccggtgcagt atgaaggegg	1920<br>
	cggagccgac accacggcca ccgatattat ttgcccgatg tacgcgcgcg tggatgaaga	1980<br>
	ccagcccttc ccggctgtgc cgaaatggtc catcaaaaaa tggctttege tacctggaga	2040<br>
	gacgcgcccg ctgatccttt gegaataege ccacgcgatg ggtaacagtc ttggcggttt	2100<br><br>
	cgctaaatac tggcaggcgt ttcgtcagta tceccgttta cagggcggct tcgtctggga	2160<br>
	ctgggtggat cagtcgctga ttaaatatga tgaaaacggc aacccgtggt cggcttacgg	2220<br>
	cggtgatttt ggcgatacgc cgaacgatcg ccagttctgt atgaacggtc tggtctttgc	2280<br>
	cgaccgcacg ccgcatccag cgctgacgga agcaaaacac cagcagcagt ttttccagtt	2340<br>
	ccgtttatcc gggcaaacca tcgaagtgac cagcgaatac ctgttccgtc atagcgataa	2400<br>
	cgagctcctg cactggatgg tggcgctgga tggtaagccg ctggcaagcg gtgaagtgcc	2460<br>
	tctggatgtc gctccacaag gtaaacagtt gattgaactg cctgaactac cgcagccgga	2520<br>
	gagcgccggg caactctggc tcacagtacg cgtagtgcaa ccgaacgcga ccgcatggtc	2580<br>
	agaagccggg cacatcagcg cctggcagca gtggcgtctg gcggaaaacc tcagtgtgac	2640<br>
	gctccccgcc gcgtcccacg ccatcccgca tctgaccacc agcgaaatgg atttttgcat	2700<br>
	cgagctgggt aataagcgtt ggcaatttaa ccgccagtca ggctttcttt cacagatgtg	2760<br>
	gattggcgat aaaaaacaac tgctgacgcc gctgcgcgat cagttcaccc gtgcaccgct	2820<br>
	ggataacgac attggcgtaa gtgaagcgac ccgcattgac cctaacgcct gggtcgaacg	2880<br>
	ctggaaggcg gcgggccatt accaggccga agcagcgttg ttgcagtgca cggcagatac	2940<br>
	acttgctgat gcggtgctga ttacgaccgc tcacgcgtgg cagcatcagg ggaaaacctt	3000<br>
	atttatcagc cggaaaacct accggattga tggtagtggt caaatggcga ttaccgttga	3060<br>
	tgttgaagtg gcgagcgata caccgcatcc ggcgcggatt ggcctgaact gccagctggc	3120<br>
	gcaggtagca gagcgggtaa actggctcgg attagggccg caagaaaact atcccgaccg	3180<br>
	ccttactgcc gcctgttttg accgctggga tctgccattg tcagacatgt ataccccgta	3240<br>
	cgtcttcccg agcgaaaacg gtctgcgctg cgggacgcgc gaattgaatt atggcccaca	3300<br>
	ccagtggcgc ggcgacttcc agttcaacat cagccgctac agtcaacagc aactgatgga	3360<br>
	aaccagccat cgccatctgc tgcacgcgga agaaggcaca tggctgaata tcgacggttt	3420<br>
	ccatatgggg attggtggcg acgactcctg gagcccgtca gtatcggcgg aattccagct	3480<br>
	gagcgccggt cgctaccatt accagttggt ctggtgtcaa aaataagctt ggctgttttg	3540<br>
	gcggatgaga gaagattttc agcctgatac agattaaatc agaacgcaga agcggtctga	3600<br>
	taaaacagaa tttgcctggc ggcagtagcg cggtggtcca cctgacccca tgccgaactc	3660<br>
	agaagtgaaa cgccgtagcg ccgatggtag tgtggggtct ccccatgcga gagtagggaa	3720<br>
	ctgccaggca tcaaataaaa cgaaaggctc agtcgaaaga ctgggccttt cgttttatct	3780<br>
	gttgtttgtc ggtgaacgct ctcctgagta ggacaaatcc gccgggagcg gatttgaacg	3840<br>
	ttgcgaagca acggcccgga gggtggcggg caggacgccc gccataaact gccaggcat	3899<br>
References:<br>
Kelley, L.A., MacCallum, R.M. and Sternberg, M. J. E, (2000), J. Mol. Biol. 299 (2): 499 -<br>
520.<br>
Kraft et.al, Biotechniques (1988) 6: 544-547<br>
Porath, J., Carlsson, J., Olsson, I. and Belfrage, G., (1975), Nature, 258: 598-599.<br>
Sassenfeld, H.M., (1990), Trends Biotechnol. 8: 88-93.<br>
Uhlen, M. and Moks, T. (1991) Methods in Enzymology 185: 129 - 143 .<br>
Yip, T. T., Nagakawa, Y. and Porath, J., (1989), Anal. Biochem., 183: 159-171.<br><br><br>
Abstract<br>
The present invention discloses a novel affinity peptide handle for production and purification of the recombinant fusion protein. The recombinant fusion protein of the present invention is composed of at least two components, an affinity peptide handle and a biologically active polypeptide or protein. The affinity peptide handle is an immobilized metal ion chelating peptide linked directly or indirectly by covalent binding. The invention also provides amino acid sequence of the novel affinity peptide handle and nucleotide sequences coding for the novel affinity peptide handle. The invention further describes construction of a expression vector comprises of a promoter, a polynucleotide sequence coding for the affinity peptide handle, a protease cleavage site or a chemical cleavage site and a heterologous gene/DNA of interest coding for polypeptide/protein of interest. The present invention further provides an improved method of purification of recombinant polypeptides and/or proteins in form of a fusion protein from a mixture containing the fusion protein and impurities. The method of purification comprises contacting the fusion protein with a resin containing immobilized metal ions for a sufficient amount of time, selectively eluting recombinant fusion protein from the resin and optionally cleaving to separate the desired protein from the linker affinity peptide handle.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWFic3RyYWN0KDctMi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-abstract(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUFCU1RSQUNUKEdSQU5URUQpLSg2LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-ABSTRACT(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWFic3RyYWN0LmRvYw==" target="_blank" style="word-wrap:break-word;">186-mum-2006-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNBTkNFTExFRCBQQUdFUygxMS0xLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CANCELLED PAGES(11-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWNsYWltcyg3LTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-claims(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhBTUVOREVEKSgxMS0xLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(AMENDED)(11-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhBTUVOREVEKS0oMS00LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(AMENDED)-(1-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhBTUVOREVEKS0oMjEtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(AMENDED)-(21-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhHUkFOVEVEKS0oNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhNQVJLIENPUFkpKDExLTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(MARK COPY)(11-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNMQUlNUyhNQVJLRUQgQ09QWSktKDIxLTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CLAIMS(MARKED COPY)-(21-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNPUlJFU1BPTkRFTkNFKDEwLTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CORRESPONDENCE(10-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNPUlJFU1BPTkRFTkNFKDExLTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CORRESPONDENCE(11-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNPUlJFU1BPTkRFTkNFKDMwLTMtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CORRESPONDENCE(30-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWNvcnJlc3BvbmRlbmNlKDctMi0yMDA3KS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-correspondence(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUNPUlJFU1BPTkRFTkNFKElQTyktKDYtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-MUM-2006-CORRESPONDENCE(IPO)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLXZlci0wODAyMDYucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-correspondence-received-ver-080206.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLXZlci0xMzAzMDYucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-correspondence-received-ver-130306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWRlc2NyaXBpdG9uIChwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-descripiton (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS0oNy0yLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-description(complete)-(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LURFU0NSSVBUSU9OKEdSQU5URUQpLSg2LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-DESCRIPTION(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWRyYXdpbmcoNy0yLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-drawing(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LURSQVdJTkcoR1JBTlRFRCktKDYtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-MUM-2006-DRAWING(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMSgxMy0zLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 1(13-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMSg3LTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 1(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMTMoMzAtMy0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 13(30-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUZPUk0gMTgoMTAtMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-FORM 18(10-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMig3LTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 2(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUZPUk0gMihHUkFOVEVEKS0oNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-FORM 2(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMih0aXRsZSBwYWdlKS0oNy0yLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 2(title page)-(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDYtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-MUM-2006-FORM 2(TITLE PAGE)-(GRANTED)-(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUZPUk0gMihUSVRMRSBQQUdFKS0oUFJPVklTSU9OQUwpLSg4LTItMjAwNikucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-FORM 2(TITLE PAGE)-(PROVISIONAL)-(8-2-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMjYoMTMtMy0yMDA2KS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 26(13-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LUZPUk0gMygxLTQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-FORM 3(1-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMygxNC0zLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 3(14-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gMyg3LTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 3(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gNSgxNC0zLTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 5(14-3-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0gNSg3LTItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-form 5(7-2-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tMi5kb2M=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">186-mum-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LU9USEVSIERPQ1VNRU5UKDEtNC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-MUM-2006-OTHER DOCUMENT(1-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3KDEtNC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">186-MUM-2006-PETITION UNDER RULE 137(1-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgxLTQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-REPLY TO EXAMINATION REPORT(1-4-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LVJFUExZIFRPIEhFQVJJTkcoMTEtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-REPLY TO HEARING(11-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LVJFUExZIFRPIEhFQVJJTkcoMjEtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">186-MUM-2006-REPLY TO HEARING(21-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LU1VTS0yMDA2LVNFUVVFTkNFIExJU1RJTkcoNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-MUM-2006-SEQUENCE LISTING(6-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTg2LW11bS0yMDA2LXNlcXVlbmNlIGxpc3RpbmcoNy0yLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">186-mum-2006-sequence listing(7-2-2007).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250898-a-transport-device-adapted-to-be-transported-by-air-or-surface-transportation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250900-device-for-controlling-supply-of-current-and-static-capacitance-to-compressor.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250899</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>186/MUM/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Feb-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>USV LIMITED</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>B.S.D. MARG, STATION ROAD, GOVANDI, MUMBAI-400088,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>RAO, LAXMI S</td>
											<td>B.S.D. MARG,STATION ROAD, GOVAND,MUMBAI-400 088</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MISHRA, SHRIKANT</td>
											<td>B.D.S. MARG,station road, GOVANDI,MUMBAI-400 088.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BORBHUYIA, MONSUR AHMED</td>
											<td>B.D.S. MARG,station road, GOVANDI,MUMBAI-400 088.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MHATRE, DEEPA</td>
											<td>B.D.S. MARG,station road, GOVANDI,MUMBAI-400 088.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>THAKUR, PRITI</td>
											<td>B.D.S. MARG,station road, GOVANDI,MUMBAI-400 088.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/62</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250899-affinity-polypeptide-for-purification-of-recombinant-proteins by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:11:06 GMT -->
</html>
