<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250916-repair-of-nucleic-acids-for-improved-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250916:REPAIR OF NUCLEIC ACIDS FOR IMPROVED AMPLIFICATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">REPAIR OF NUCLEIC ACIDS FOR IMPROVED AMPLIFICATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods and compositions are provided for repairing apolynucleotide so that it can be synthesized efficiently with improved fidelity and yield in, for example, an amplification reaction. This involves the use of a reaction mixture that includes a ligase and a cofactor selected from NAD + or ATP and incubating the polynucleotide with the reaction mixture in the absence of Endonuclease VI. The reaction mixture may further contain an AP endonuclease and a polymerase. These enzymes are optionally selected according to their ability to withstand high temperatures so they can be included in an amplification mixture. The reaction mixture may used prior to a polynucleotide synthesis reaction in which case enzymes that are not thermophilic may be used. The repair reaction is not time sensitive with respect to seconds, minutes or hours of incubation in the enzyme mixture.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Repair of Nucleic Acids for Improved Amplification<br>
BACKGROUND<br>
Various approaches have been reported to repair DNA using base excision enzymes.   Unfortunately, these approaches in different ways cause further damage to the DNA.  Conventional PCR techniques have been modified to improve amplification in some aspects.   U.S. Patent No. 5,035,996 describes a process for controlling contamination of nucleic acid amplification reactions that uses the modified nucleotide, dUTP, in the amplification reaction. This process uses uracil glycosylase to eliminate those PCR products containing uracil to prevent contaminating subsequent PCR reactions.  U.S. patent publication no. 2004-0067559 Al also relies on modified bases in primer DNA prior to amplification and uses, for example, dUTP for incorporation into the amplicon. The amplicon can then be fragmented by adding, for example, Uracil-DNA Glycosylase (UDG) and Endonucleaese (Endo) IV.<br>
Hot start nucleic acid amplification has been used to lower mis-priming during PCR. One type of hot start amplification relies on the presence of a PCR primer with a blocked 3' terminus to prevent extension by the polymerase present in the PCR reaction (see for example US 2003-0119150). The primer is unblocked by a thermostable 3'-5' exonuclease that is active at &gt;37°C. Therefore, the polymerase will only extend the PCR primers once the exonuclease unblocks the 3' end at &gt;37°C. Alternatively the Taq polymerase is blocked and then activated at amplification temperatures.<br><br>
Barnes, W. M. Proc. Natl. Acad. Sci. USA 91:2216-2220 (1994) describes the use of vent polymerase and Taq polymerase as an improvement over the use of Taq polymerase only in amplification.  Ghadessy et al. reported a mutant Taq polymerase that is not halted by damaged or abasic sites (Ghadessy et al. Nature Biotechnol. 22(6):755-9 (2004)).<br>
It has been reported that conventional amplification techniques are compromised if the DNA is substantially damaged (DiBernardo et al. Nucl. Acids Res.30:el6 (2002)).   Degradation and/or fragmentation of DNA resulting from exposure to the environment and microorganisms which contain DNA endonucleases is a frequent problem in forensics, diagnostic tests and routine amplification and affects fidelity and yield of the amplification product.  In addition, the problem of degraded DNA is also faced by researchers who are analyzing the DNA obtained from frozen, extinct or extremely rare organisms.<br>
Fromenty, B., et al. Nucl. Acids Res. 28(ll):e50 (2000) and International Publication No. WO/0151656 reported that Exonuclease (Exo) III improved yields of long PCR.   Fromenty also reported decreased yields of amplicon for DNA
Di Benardo et al. Nucl. Acids Res. 30(4):el6 (2002) described the use of T4 DNA ligase (T4 ligase) and an E. coli polymerase to amplify short regions of single-stranded DNA between cross-linked regions of double-stranded DNA.<br><br>
Another approach to amplification of damaged DNA has been described in U.S. Publication No. 2003-0077581.   Degraded nucleic acid was hybridized to undegraded nucleic acid having a sequence homologous to the degraded nucleic acid. Regions of the degraded nucleic acid were then filled in with nucleotide precursors. The fragmented strands were then covalently linked using a polymerizing and/or ligating enzyme.<br>
Preparations for improving amplification of damaged DNA can be obtained commercially from Sigma, St. Louis, MO and Qbiogene, now MP Biomedicals, Irvine, CA. Although the compositions of these preparations are not provided, it is assumed that Exo III is contained in the preparation. The preparations are not recommended for DNA templates less than 500 base pairs in length.<br>
Others report the use of a combination of E. coli DNA Poll and T4 ligase for pre-amplification repair (Pusch, et al., NucJ. Acids Res. 26:857 (1998)). However, according to Pusch et al. the preamplification product is purified before initiation of amplification.<br>
SUMMARY<br>
In an embodiment of the invention, a method is provided for enhancing at least one of fidelity and yield of an amplification product of a damaged polynucleotide, that includes the steps of: (a) incubating the polynucleotide in a reaction mixture comprising a ligase and a cofactor selected from NAD+ or ATP and excluding Endo VI; (b) permitting amplification of the polynucleotide to occur in the reaction mixture by the addition of amplification reagents to the reaction mixture during or after step (a); and (c) enhancing at least one of fidelity or yield of the amplification product in the presence of step (a) compared to in the absence of step(a).<br><br>
The above method is not particularly time sensitive in respect to whether the incubation occurs in seconds, minutes or hours. The ligase used in embodiments of the method may be mesophilic or thermophilic and does not exclude cryophilic ligases, which might be useful under particular circumstances. The choice of ligase with respect to temperature sensitivity depends on what is best suited for a particular set of reaction conditions. For example, if the amplification reagents are added during the incubation step Qa), then it may be desirable to employ a thermophilic ligase to withstand temperatures utilized during amplification. Examples of thermophilic ligases are Taq DNA ligase {Taq ligase) and 9°N ligase. Taq ligase is more effective with a NAD+ cofactor while 9°N DNA ligase (9°N ligase) is more effective with an ATP cofactor. Examples of a mesophilic ligase are T4 ligase (using an ATP cofactor) a nd E. coli DNA ligase (E coli ligase) (using an NAD+ cofactor).<br>
The reaction mixture may further include an AP endonuclease such as Type II endonuclease, T7 Endonuclease (Endo) I or mutant thereof or Endo IV. The reaction mixture may alternatively or also include a polymerase for example Taq polymerase, an E. coli polymerase, a Thermomicrobium sp. polymerase or an archa eal polymerase or mutant thereof such as Pfu, Vent®, Deep Vent®, 9°N or GBD polymerase.<br>
In embodiments of the invention, enzymes that may be additionally added to the reagent mixture include T4 pyrimidine dimer glycosylase, [fapy]-DNA glycosylase (Fpg), at least one of UvrA, UvrB, UvrC, UvrD, Cho, UDG, Aag, Endo III and Endo V in various combinations depending on the type of damage sustained by the polynucleotide.<br><br>
In an embodiment of the invention, a reaction mixture is used containing about 1-100 units of endonuclease, about 0.05-0.25 units of polymerase and about 5-500 units of ligase optionally added to 1-1000 ng DNA.<br>
Types of damage that may affect a polynucleotide include apurinic/apyrimidinic (AP) sites, mutagenized nucleotides, modified nucleotides, nicks, gaps and DNA-DNA or DNA~protein cross-links.<br>
The damaged polynucleotide may be obtained from natural sources, preserved biological material, forensic evidence, ancient polynucleotides, a tissue biopsy or routine biological manipulation.<br>
According to embodiments of the method, amplification of DNA is achieved by any of PCR amplification, helicase-dependent amplification, transcription-mediated amplification, strand-displacement amplification, rolling circle amplification and whole genome amplification.<br>
Where the polynucleotide is a single-stranded RNA, the amplification may be a reverse transcriptase dependent amplification.<br>
In an embodiment of the method, the polynucleotide is capable of producing an amplicon in a size range of 50 nucleotides to 100,000 nucleotides for PCR amplification.<br>
In an embodiment of the invention, an amplification kit is provided that includes instructions for use and one or more enzymes wherein at least one of the enzymes is a ligase, the one or<br><br>
more enzyme being formulated for addition to an amplification mixture to enhance amplification or for use prior to addition of the amplification mixture to enhance amplification.<br>
In another embodiment of the invention, a composition is provided that contains an effective amount of a ligase, a polymerase, and an AP endonuclease not including Endo VI, the mixture being capable of enhancing at least one of yield and fidelity of amplification of a polynucleotide compared with amplification of the polynucleotide in the absence of the composition. For example, concentrations of reagents in the composition include: an AP endonuclease at a concentration of 1-100 units of endonuclease, a polymerase at 0.05-0.25 units, and 5-500 units of ligase contained for example in a reaction volume of 10-100 |xl-   This formulation<br>
may be applied to 1-1000 ng DNA for repairing the DNA.  For larger concentrations of DNA, the amounts of enzymes should be increased proportionally. In embodiments of the invention, additional enzymes may be included in the composition including one or more of T4 pyrimidine dimer glycosylase, [fapy]-DNA glycosylase (Fpg), UvrA, UvrB, UvrC, UvrD, Cho, UDG, Aag, Endo III and Endo V in various combinations depending on the type of damage sustained by the polynucleotide.<br>
LIST OF FIGURES<br>
Figure 1 shows enhanced amplicon yield from heat-damaged lambda DNA after preincubation with specified enzymes.<br>
Figure 1A shows DNA template damaged by heat to differing extents and the effect of this damage on amplification of a 5 kb segment of lambda DNA where 5 ng, 2 ng and 1 ng of heat-treated<br><br>
lambda DNA was amplified after prior damage by 99°C heat treatment for 0 sec, 30 sec, 60 sec, 90 sec, 120 sec or 180 sec. The damaged DNA was not subjected to enzyme treatment prior to amplification. The amount of amplification was determined after electrophoresis and was found to be substantially reduced by 120 sec heat treatment. The first and last lanes on the gel contain 1 ug of a 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Figure IB shows increased amplicon yields from heat-damaged lambda DNA using Taq ligase, E. coli Endo IV and E. coli Poll on amplification of a 5 kb segment of lambda DNA.   DNA was heat damaged as described in Figure 1A but the damaged DNA was subjected to enzyme treatment prior to amplification. The results of amplification are shown after a 10-minute pretreatment reaction with Taq ligase, E. coli Endo IV and E. coli Poll. The amplicon yield was increased throughout but was especially noticeable with 120 sec and 180 sec heat damaged DNA.<br>
Figure 1C shows increased amplicon yields from heat-damaged lambda DNA using Taq ligase, Tth Endo IV and E. coli Poll. The amplification was performed according to Figure IB but the enzyme treatment prior to amplification contained Therrnus thermophilus (Tth) Endo IV in place of E. coli Endo IV.  The results of amplification are shown after a 10-minute pretreatment reaction with Therrnus aquaticus (Taq) ligase, Tth Endo IV and E.  coli Poll. The amplicon yield was increased throughout but was especially noticeable with 120 sec and 180 sec heat-damaged DNA. Only the first lane contains the size ladder.<br>
Figure ID shows increased amplicon yields from heat-<br><br>
damaged lambda DNA using E. coli ligase, E. coli Endo IV and E. coli DNA poll. The amplification was performed according to Figure IB but the enzyme treatment prior to amplification contained E. coli ligase in place of Taq ligase.  The lambda DNA subjected to 99°C for 180 sec was used as a template. The amount of template DNA used is indicated above each lane. The yield of amplicon is increased for each of the template amounts by enzyme pretreatment.<br>
Figure 2 shows the effect of citrate treatment of template DNA on amplicon yield.<br>
Figure 2A shows the results of amplification of a 5 kb segment of lambda DNA where lambda DNA was heated to 70°C in citrate buffer for 0, 20, 40, 80, 120, and 160 minutes. 50 ng, 10 ng and 5 ng of each heat-treated sample were amplified and the resulting products were visualized on a gel to determine the extent of amplification. The DNA was not treated with selected enzymes prior to amplification.  The last lane on the right contains 1 ug of 2-log ladder.<br>
Figure 2B shows the increase in yield of a 5 kb amplicon of lambda DNA regardless of which polymerase was used in the enzyme mixture. 120-minute heat/citrate-damaged lambda DNA was treated with various enzymes prior to amplification.<br>
Lane 1:   lug 2-log ladder (NEB# N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 2: no pretreatment.<br>
Lane 3:   Pretreatment with Taq ligase, Taq DNA polymerase and E. coli Endo IV.<br>
Lane 4:   Pretreatment with Taq ligase, E. coli Poll, and<br><br>
E. coli Endo IV.<br>
Lane 5: Pretreatment with Taq ligase, Taq:Vent® DNA polymerase mix, and E. coli Endo IV.<br>
Figure 3 shows the results of amplification of a 200 bp segment of krill genome that has been extracted from an ethanol stored sample of krill and pretreated with an enzyme mixture containing one of various polymerases, a ligase and an AP endonuclease that enhances amplification yields.<br>
Lane 1:   No pretreatment of krill DNA with enzymes. Lane 2: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and Taq polymerase.<br>
Lane 3: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and Vent® polymerase.<br>
Lane 4: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and 50:1 Taq:Vent® polymerase.<br>
Figure 4 shows an increase in yield of a 10 kb amplicon from heat-damaged DNA. 180 sec heat-damaged DNA was pretreated with an enzyme mixture and then amplified.<br>
Lane 1: 1 ug of a 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 2:   Pre-treatment with Taq ligase, E. coli Endo IV, and E. coli Poll.<br>
Lane 3:   Pre-treatment with Taq ligase and E. coli Endo IV.<br>
Lane 4:   Pretreatment with Taq ligase.<br>
Lane 5:   Control - untreated DNA.<br>
Figure 5 shows that ligase pretreatment increases amplicon yield from environmental DNA (soil sample extract).<br><br>
Lane 1: A 2-!og ladder size standard (NEB#  N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 1: No enzyme p- ' eatment. Lane 2: Pre-treatment with T4 ligase. Lane 3: No enzyme pre-treatment. Lane 4: Pretreatment with Taq ligase.<br>
Figure 6: Genbank search revealing proteins with sequence homology with T4 ligase.<br>
Figure 7: DNA sequence of Tth Endo IV (SEQ ID NO: 11).<br>
Figure 8 shows the effect of UV light on amplicon yield using lambda DNA as a template by gel electrophoresis.<br>
A: Lambda DNA is subjected to UV irradiation for up to 50 sec and a slight reduction in yield of a 2 Kb amplicon produced is shown.<br>
B: Lambda DNA is subjected to UV irradiation for up to 50 seconds and the reduction in yield of a 5kb amplicon is shown.<br>
C: The effect of various reaction mixtures added to lambda DNA on yield of a 5kb amplicon after UV irradiation is shown.<br>
Lanes 2-7 are controls in the absence of a reaction mixture.<br>
Lanes 8-13 show the increased beneficial effect of adding ligase, polymerase and AP endonuclease plus 10 Units of T4 PDG.<br>
Lanes 14-19 show the increased beneficial effect of adding ligase, polymerase and AP endonuclease plus 80 units of T4 PDG.<br>
Lanes 1 and 20: A 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Figure 9 shows that adding ligase to T7 Endo I expands the useful range of the enzyme:DNA ratio to facilitate the removal of<br><br>
heteroduplexes from the amplification mixture so as to increase the ratio of correct sequences. Taq ligase and T7 Endo I were added to supercoiled DNA in varying amounts as indicated for each lane.<br>
Figure 9a is the control in which no Taq ligase has been added but increasing amounts of T7 Endo I are used. The supercoiled DNA is predominantly cleaved into fragments of various sizes with 12.5-25 units of T7 Endo I.<br>
Figure 9b shows how the addition of 100 units of Taq ligase protects DNA from non-specific cleavage in the presence of T7 Endo I such that even at 200 units of T7 Endo I, there is a clear band corresponding to linear DNA not present in the absence of ligase.<br>
Figure 10 shows the effect of repair enzyme treatment on amplicon yield from oxidatively damaged DNA or undamaged template.<br>
Figure 10A shows that the addition of repair enzymes to an undamaged template, pWB407 has no effect on amplicon yield.<br>
Figure 10B shows that the addition of Fpg to a damaged template, plasmid pWB407, incubated in the presence of methylene blue, gives inconsistent effects on yield. The addition of Taq ligase, E. coli DNA polymerase, and E. coli Endo IV in the presence or absence of Fpg increases amplicon yield consistently.<br>
Figure 11 shows increased PCR reaction accuracy from damaged DNA after treatment with repair enzymes. Repair enzyme treatment of undamaged template, plasmid pWB407, prior to PCR<br><br>
has no significant effect on reaction accuracy. Treatment of a damaged template, plasmid pWB407 incubated with methylene blue, with Fpg alone or also with Taq ligase, E coli DMA polymerase I, and E coli endonuclease increases the accuracy of PCR.  The measure of accuracy is the number of white colonies verses the number of blue colonies after cloning a lacZ-containing amplicon as discussed below. The higher the percentage of white colonies the greater the error rate.<br>
Figure 12 shows a flow diagram for treating DNA with unknown damage to increase at least of one of fidelity and yield.<br>
DESCRIPTION<br>
Embodiments of the invention describe methods for improving at least one of yield or fidelity for synthesis of damaged polynucleotides. Where polynucleotide synthesis leads to polymerase-dependent amplification, short amplicons that are less than about 500 bases in length (as short as 100 nt) or long amplicons that are greater than 500 bases or as much as about 100 kb may be amplified (for PCR amplification).  Other types of polynucleotide synthesis include primer extension reactions such as amplification (for example PCR, RT-PCR, and QPCR), genome amplification, rolling circle amplification (RCA) and helicase-dependent amplification-(HDA); and DNA sequencing  reactions. Embodiments of the methods'have wide utility in molecular biology research and in solving problems in applied biology, for example, in forensics, in biological archeology in which it is desirable to analyze DNA from ancient sources, for taxonomy where it is desirable to analyze DNA from environmental samples such as required for the Barcode of Life Project, for diagnostic assays including tissue<br><br>
biopsies to determine a disease susceptibility or status and for molecular biology research.<br>
Source and Extent of Damage<br>
Damage sustained by polynucleotide molecules is common even in "normal" polynucleotides although damage is more severe in preserved tissues, dried specimens or polynucleotides that are exposed to the environment. Damage can occur as a result of the age of the sample or its length, its source or its preparation. In addition, damage can occur during the application of a methodology for polynucleotide synthesis such as occurs during PCR amplification, which involves a high temperature step.<br>
Polynucleotides can sustain damage in a variety of ways. Various types of damage include: (a) apurinic or apyrimidinic damage caused for example by heat, storage of polynucleotides in ethanol, and exposure to factors in the environment such as H20, pH etc; (b) modification of individual nucleotides, caused for example by deamination, alkylation, oxidation and dimerization; (c) nicks and gaps caused for example by heat, storage of polynucleotide in ethanol, and exposure to factors in the environment such as H20, pH etc; (d) cross-linking caused for example, by formaldehyde, environmental factors, and ethanol storage; and (e) mismatched DNA caused by for example misincorporation of a nucleotide by a polymerase.<br>
Different polynucleotide preparations will experience different types of damage resulting from, for example, storage or handling of the polynucleotide preparation in vitro, and may depend on how prokaryotic cells, archaea or eukaryotic cells containing the<br><br>
polynucleotides are stored and the characteristics of the cells from which the polynucleotide is extracted.<br>
Definitions<br>
The term "polynucleotide" refers to double-stranded DNA, double-stranded RNA, hybrid DNA/RNA duplex, single-stranded DNA and single-stranded RNA.<br>
A "repair enzyme" refers to a cryophilic, mesophilic or thermophilic enzyme that participates in the process of repair of a polynucleotide. For example, a repair enzyme may induce breakage of the polynucleotide at a bond, thereby facilitating removal of damaged regions of the polynucleotide. Enzymes with a synthetic role such as ligases and polymerases are also repair enzymes.<br>
DNA repair enzymes are described in the scientific literature, for example, see Wood, R.D., et al. Mutat. Res. 577(l-2):275-83 (2005) and Eisen, J.A. and Hanawalt, P.C. Mutat. Res. 435(3): 171-213 (1999).  A list of human repair enzymes is provided in Table 1. Although not described in Table 1, the homo logs of the listed enzymes and other functionally related enzymes are included in the definition of repair enzymes. Any of the above enzymes may be naturally occurring, recombinant or synthetic. Any of the enzymes may be a native or in wtro-created chimera with several activities. In addition to the enzymes described abover it is known to a person of ordinary skill in the art how to search the databases to identify other related enzymes that share conserved  sequence motifs and have similar enzyme activity. For example, the NCBI web site (www.ncbi.com) provides a conserved domain database. If, for<br><br>
example, the database is searched for homologs of Endo IV, 74 sequence matches are recovered. (Also see Figure 6 for ligases).<br>
A "polynucleotide cleavage enzyme" used in enzyme mixtures for repairing damaged DNA is a class of repair enzymes and includes AP endonucleases, glycosyiases and lyases responsible for base excision repair.<br>
A damaged base can be removed by a DNA glycosylase enzyme which hydrolyses an N-glycosylic bond between the deoxyribose sugar moiety and the base.   The product of this reaction is an apurinic or apyrimidinic site (AP site) that must be correctly filled. This can be achieved by an endonuclease, which nicks the sugar phosphate backbone adjacent to the AP site. The abasic sugar is removed and a new nucleotide is inserted by polymerase/ligase activity. Some enzymes having applicability herein have glycosylase and AP endonuclease activity in one molecule. These repair enzymes are found in prokaryotic and eucaryotic cells. Abasic sites can be recognized and cleaved by AP endonucleases and/or AP lyases. Class II AP endonucleases cleave at AP sites to leave a 3' OH that can be used in polynucleotide polymerization.  Furthermore, AP endonucleases can remove moieties attached to the 3' OH that inhibit polynucleotide polymerization.  For example a 3' phosphate can be converted to a 3' OH by E. coli Endo IV.  AP endonucleases can work in conjunction ,with glycosyiases.<br>
Examples of glycosylase specificities include Uracil, Hypoxanthine, 3-methyladenine (3-mAde), Formamidopyrimidine and Hydroxymethyluracil.  The presence of Uracil in DNA occurs due to mis-incorporation or deamination of<br><br>
cytosine by bisuifate, nitrous acids, or spontaneous deamination. Hypoxanthine occurs due to deamination of adenine by nitrous acids or spontaneous deamination.  3-mAde is a product of alkylating agents. E. coli has two 3-mAde glycosylase called TagI and Tagil. Formamidopyrimidine (FAPY) (7 -mGua) is the most common product of methylating agents of DNA. Gamma radiation produces 4.6-diamino-5-FAPY. An £ coli glycosylase that repairs this lesion is Fpg endonuclease. Hydroxymethyuricil is created by ionizing radiation or oxidative damage to thymidine.<br>
Lyases break the phosphodiester bond in a polynucleotide.<br>
Examples of AP endonudeases belong to 4 classes.<br>
(I)	cleaves 3'—&gt; 3'-OH + 5'-P - and has associated glycosylase activity.<br>
(II)	cleaves 5'   -&gt; 3'-OH + 5'-P<br><br>
(III)	cleaves 3' --&gt; 3'-P + 5'-OH<br>
(IV)	cleaves 5' --&gt; 3'-P + 5'-OH<br>
Several enzymes have been isolated that appear to have AP endonuclease or lyase and glycosylase activities that are coordinated in a concerted manner (i.e., without causing AP site formation) or sequentially.<br>
Examples of polynucleotide cleavage enzymes for use in enhancing at least one of yield and fidelity in an amplification reaction include: 1) AP endonudeases, such as E. coli Endo IV, Tth Endo IV, and human AP endonuclease; 2) glycosylases, such as UDG, E. coli AlkA and human Aag; and 3) glycosylase/lyases, such as E. coli Endo III, E. coli Endo V, E. coli Endo VIII, E. coli Fpg,<br><br>
human 0GG1, T4 pyrimidine dimer glycosylase (T4 pdg) and human AP endonuclease.<br>
Endo VI (also termed Exo III) is capable of degrading a substantial portion of a polynucleotide outside the damaged regions in a polynucleotide under normal reaction conditions in a few hours and is not included in enzyme mixtures for treating damaged polynucleotides.<br>
A "polymerase" as used in enzyme mixtures herein refers to an enzyme that contains polymerase activity. The repair and amplifying polymerases can be the same or different.<br>
Examples of polymerases include thermostable bacterial polymerases such as Taq polymerase and archeal polymerases such as Vent®, deep Vent® and Pfu; and thermolabile enzymes such as Bst polymerase, E. coli Poll, thermomicrobiurn roseum polymerase and thermomicrobiurn thermophilus, phage polymerases such as phi29 polymerase, T7 polymerase and T4 polymerase etc., or mutants, derivatives or modifications therefrom.  Examples of derivatives include Pfusion™ enzyme (Finnzynnes, Espoo, Finland) and other polymerases that combine a double strand binding protein with polymerase sequences from one or several sources.<br>
A "ligase" as used in the enzyme mixtures described here refers to an enzyme that joins a 5! end of a single strand of a polynucleotide to a 3' end of another single strand of a polynucleotide. Such ligases are found in substantially all eukaryotic cells as well as prokaryotic cells, viruses and archaea. Any of these ligases can be used in repair as described herein.  Examples of ligases include 9°N ligase, E. coli ligase, T4 ligase and Taq ligase.<br><br>
Other ligases include LIGA (NP-416906.1), TthDNALGS (AAA27486.1), LIG3 (NM-013975) and LIG4 (NM-002312).<br>
Other ligases or ligase-like proteins that may have utility herein are revealed by a Genbank search using T4 ligase or E. coli ligase to search the database (see Figure 6) in which any enzyme sharing at least 6 contiguous amino acids with these known ligases may be included in a repair mixture according to embodiments of the invention.<br>
Contrary to a published use of ligase in combination with Exo III in the absence of any cofactors (U.S. Publication No. 2005-0026147), it has been found here that NAD+ or ATP is required in enzyme mixtures that include ligase. More specifically, Taq ligase and E. coli ligase require NAD+ while T4 ligase and 9°N ligase require ATP.<br>
Exemplified ligases, polymerases and endonucleases are available from New England Biolabs Inc. where pages 107-117 of the 2005-2006 catalog are incorporated by reference (pp. 102-108 for ligases), U.S. Provisional Application No. 60/717,296 and International Publication No. WO 2005/052124. In addition, thermostable repair enzymes can be used interchangeably with thermolabile repair enzymes in a preamplification mixture. Thermostable enzymes are active at above 40 °C or more particularly 65°C or above.<br>
Embodiments of present methods improve the yield or fidelity of products resulting from polynucleotide amplification or other synthesis reaction. This can be achieved, for example, when<br><br>
a damaged polynucleotide is treated with a preparation of enzyme(s) in a pre-incubation mixture and/or during amplification.<br>
Amplification protocols that may benefit from the above described pre-incubation include polymerase chain reaction (PCR), Strand-Displacement Amplifcation (SDA) (U.S. Patent Nos. 5,455,166 and 5,470,723); HDA(U.S. Publication No. 2004-0058378-A1); Transcription-Mediated Amplification (TMA) (Guatelli era/., Proc. Natl. Acad. Sci. USA 87 :1874-1878 (1990)); Rolling Circle Amplification (RCA) which generates multiple copies of a sequence for the use in in vitro DNA amplification adapted from in vivo rolling circle DNA replication (see, for example, Fire and Xu, Proc. Natl. Acad Sci. USA 92:4641-4645 (1995); Lui, et al., J. Am. Chem. Soc. 118:1587-1594 (1996); Lizardi, etal., Nature Genetics 19:225-232 (1998)) and whole genome amplification methods.<br>
A universal enzyme mixture has been found to be useful in a reaction mixture for repairing damaged polynucleotides prior to or during amplification regardless of the type of damage to the polynucleotide. The mixture repairs damaged DNA without causing further damage.<br>
The universal enzyme mixture contains a ligase and a cofactor such as NAD+ or ATP. The mixture preferably additionally includes a polymerase and an AP endonuclease as defined above within a suitable buffer such asThermopol (New England Biolabs, Inc., Ipswich, MA), AccuTaq LA DNA polymerase buffer (Takara Bio Inc., Shiga, Japan) or any other standard Taq buffer.  In various embodiments, the universal enzyme mixture contains E. coli Poll or Taq polymerase and an AP endonuclease such as a mesophilic Endo IV, e.g., E. coli Endo IV or a thermophilic Endo IV, e.g., Tth Endo IV<br><br>
and a ligase selected from E. coli ligase, Taq ligase or an archaeal ligase such as 9°N ligase.  In a particular embodiment, the enzyme mixture contains 1-100 units Endo IV, 0.05-0.25 units £ coli Poll, and 5-500 units of a ligase suitable for repairing 1-1000 ng DNA prior to or during amplification. It will be understood that the concentration range for endonucleases and polymerases other than those specified in the universal mixture above may vary with the enzyme used and the temperature of the reaction. However, the concentration range can be readily ascertained using the assays described in the Examples. For example, a standard preparation of lambda DNA can be heat treated according to Example 1.   The DNA can then be subjected to a series of enzyme mixtures containing ligase and cofactors.  An additional enzyme is titrated to determine a preferred concentration for that enzyme in the mixture.   In this way, DNA repair can be optimized. After amplification of each sample, the amount of the amplified DNA can be determined by gel electrophoresis revealing the preferred concentration range for the test enzyme.<br>
The universal enzyme mixture can be used prior to or during polynucleotide amplification or other synthesis.<br>
As demonstrated in the Examples, depending on the type of damage, it may be desirable to supplement the universal enzyme mixture with additional repair enzymes defending on the nature of the DNA damage. The utility of individual repair enzymes or mixtures of repair enzymes can be determined using the assays described in the Examples and in the Figures to determine their suitability for repairing a particular polynucleotide.<br><br>
Repair of general or specific damage to polynucleotides<br>
(a) General damage<br>
Determining the nature of damage in a polynucleotide is time-consuming. If some form of damage to a polynucleotide is suspected, for example, the polynucleotide is poorly amplified, it is preferable not to have to identify the lesion or lesions. In these circumstances, a universal mix of enzymes such as described above may be utilized to determine whether improved amplification is obtained. If the improvement is sufficient using the universal mixture then no further action is required. If the improvement is not sufficient, additional enzymes can be added to the mixture as described herein until the preferred result is obtained. The entire assay may be achieved in a single reaction vessel such as a 96 well dish. Each micro-well in the dish is available for a different enzyme mixture including the universal mixture plus enzymes selected to address each class of damage outlined below.<br>
The protocol for selecting enzymes for repair of general damage or unknown damage of DNA is provided in Figure 12 (flow chart) and in the assays described in the Examples.<br>
(b) Specific damage<br>
In some circumstances, the nature of the damage to a polynucleotide might be known. In these circumstances, a mixture of enzymes can be selected without undertaking the analysis of Figure 12.<br><br>
(i) AP sites<br>
The loss of a base is the most common spontaneous form of DNA damage.   Polymerases and polymerase-based techniques are adversely affected by the presence of these abasic sites. The effectiveness of primer extension reactions is enhanced by repairing any abasic sites found in a polynucleotide.  This is achieved in one embodiment by Endo IV activity that cleaves the phosphate backbone at the abasic site. This leaves an extendable 3' OH on the DNA fragment 5' to the cleaved abasic site.   It also leaves a deoxyribose-5'-phosphate (dR5P) on the DNA fragment 3' to the cleaved abasic site.  A polymerase can extend from the free 3' OH replacing the cleaved abasic site with a correct nucleotide.  The dR5P may be removed by an enzyme that specifically targets dR5Ps by a flap endonuciease activity present in certain polymerases such as E. coli DNA polymerase I or a separate flap endonuciease such as FENI. The removal of dR5P can also occur by cleavage downstream of this group by the flap endonuciease activity.  After removal of the dR5P and the generation of a 5' phosphate adjacent to the 3' OH, a ligase can seal this nick finishing the repair (see Examples 1-3 and<br>
7).<br>
(ii) Modified nucleotides<br>
(a) Thymidine dimers<br>
Light can damage DNA by inducing the formation of pyrimidine dimers.   Pyrimidine dimers block the DNA extension reaction catalyzed by DNA polymerases such asTaq DNA polymerase and hence inhibit DNA amplification (Wellinger, et al. Nucleic Acids Res. 24(8):1578-79 (1996)). Consequently it is<br><br>
desirable to repair pyrimidine primers prior to or during amplification. This can be achieved by adding a pyrimidine dimer glycosylase/lyase (Vande Berg, et al. J. Biol. Chem. 273(32):20276-20284 (1998)) to the universal enzyme mixture. The DNA backbone is cleaved 5' to the pyrimidine dimer and leaves a 3' hydroxyl moiety that is extendable by a DNA polymerase.  In certain embodiments, extension at the 3' hydroxyl and subsequent formation and then cleavage of the lesion-containing flap generated during DNA extension results in a nick that is sealed by an enzyme capable of sealing the nick.  Cleavage of the flap can be achieved by the extending polymerase for example, E. coli polymerase I or by the action of a flap endonuclease ((Xu, Y., et al. X Biol. Chem. 275(27):20949-20955 (2000), Liu, Y., et al., Annu. Rev. Biochem. 73:589-615 (2004)).<br>
(b) Oxidative damage<br>
Inaccuracies can be introduced into the products of DNA amplification reactions because of undesired nucleotide incorporation opposite a damaged base (Gilbert, et al. Am. J.  Hum. Gen. 72:48-61 (2003); Hofreiter et al. NucL Acids Res. 29:4793-9 (2001)).   These inaccuracies can be discovered by amplifying, cloning and sequencing the same sample many times.   Inaccuracies due to base damage can also be identified by comparing sequence data before and after sample treatment with an enzyme such as UDG, which removes one of the common types of mutagenic DNA lesions (Hofreiter, et al. NucL Acids Res 29:4793-9 (2001)). However, treatment with UDG creates an abasic site within the DNA that inhibits DNA amplification by primer extension.  This creates problems for rare DNA samples that may be made refractory to amplification by UDG treatment.<br><br>
Modified nucleotides that are the product of oxidative damage can be removed from the polynucleotide by Fpg or hOGG to leave a blocked polynucleotide where the blocked polynucleotide is repairable by an AP endonuclease such as Endo IV.<br>
The effectiveness of enzyme pretreatment to repair oxidative damage to a polynucleotide prior to amplification is illustrated in Example 9 where the universal enzyme mixture is supplemented with Fpg in the pre-incubation mixture.<br>
Other modified nucleotides such as alkylated bases or deaminated bases where cytosine is converted to uracil, guanine to xanthine or adenine to hypoxanthine give rise to miscoding . Removal of these modified nucleotides is desirable.  These modified bases can be removed by any of AlkA, UDG or Aag as described in Example 10, leaving an AP site. This AP site can then be repaired by a reaction mixture containing a ligase and preferably also an AP endonuclease and a polymerase.  Removal of a uracil enables a polymerase in an amplification reaction that would normally be stopped at this site to continue amplifying the DNA. For example, Vent® polymerase activity is inhibited by an incorrect uracil inserted into the DNA. The ability to remove the uracil permits the polymerase to have enhanced effectiveness.<br>
(iii) Nicks and gaps<br>
Nicks and gaps in the DNA backbone can lead to truncated primer extension products and inhibit amplification reactions. The concerted action of a ligase and a polymerase in the universal<br><br>
enzyme mixture repairs nicks and gaps in the DNA thus enhancing DNA amplification reactions.<br>
(iv) Cross-links<br>
Additional nucleotide excision repair (NER) proteins (Minko et al. Biochemistry 44:3000-3009 (2005); Costa et al. Biochirnie 85(11):1083-1099 (2003); Sancar Ann. Rev. Biochem 65:4-3-81 (1996)) can be added to the Universal enzyme mixture to repair damage resulting from exposure of polynucleotides to formaldehyde and bulky adducts as well as damage that results in chemically-modified bases that form DNA-protein cross-links.  At least one of E. coli UvrA, UvrB, mutant UvrB, UvrC, UvrD or Cho (Moolenar et al. Proc. Natl Acad. Sci USA 99:1467-72 (2002)) can be used to make incisions at the 5! end and optionally the 3' end around a damaged site. Details about the properties and purification protocols of these enzymes can be obtained from (Zou, Y., et al. Biochemistry 43:4196-4205 (2004)). The repair process can be completed by means of a DNA polymerase, a DNA ligase and optionally a flap endonuclease.<br>
The generation of a 3' hydroxyl at a 5' incision site can be useful if the NER enzyme(s) cleave the DNA but leave a blocked 3' end on the DNA that inhibits primer extension. An example would be if the NER enzyme(s) cleaved the DNA and left a 3' phosphate. This would not be extendable by known DNA polymerases unless the 3* phosphate was removed by for example, E. coli Endo IV.<br>
If the NER enzyme or enzymes cleaves 5' and 3' to the DNA lesion then the damage is removed when the newly released oligonucleotide dissociates from the DNA.  A polymerase can simply<br><br>
fill in the excised region of DNA leaving a nick which ligase then seals to complete the repair.  In certain cases the polymerase may fill in the DNA and then proceeds to displace the remaining DNA strand. In these circumstances, an enzyme with flapase activity permits a nick to be formed that a ligase can seal. In cases in which the NER enzyme or enzymes only cleaves 5' to the damage, the polymerase preferably displaces the original DNA strand until it is past the damage at which point a flapase cleaves the DNA flap to create a ligatable nick. The flapase may be active before and after the DNA lesion is reached.   Preferably, the polymerase and flapase activities work to eventually displace and remove the DNA lesion leaving a ligatable nick, thus repairing the DNA template. An example of the effectiveness of the above approach is provided in Example 7.<br>
(v) Mismatched polynucleotides<br>
Heteroduplex DNA can be a problem in multi-template PCR and in homogeneous template PCR (Lowell, J. L &amp; Klein, D. A. Biotechniques 28: 676-681 (2000); Thompson, J. R., et al. Nucl. Acids Res. 30(9):2083-2088 (2002); Smith, J. &amp; Modrich, P. Proa Natl. Acad. Sci. USA 94:6847-6850 (1997)). T7 Endo I or mutant thereof can be used together with a ligase to remove mismatch regions. This approach does not require quantitation of DNA and avoids the extra steps after the PCR reaction required by Lowell, et al. Biotechniques 28:676-681 (2000); and Smith, et al. Proc. NatL Acad. Sci. USA 94:6847-6850 (1997).  An example of the use of these enzymes is provided in Example 8. The useful range of the T7 endonudease or mutant:DNA ratio can be expanded by including a DNA ligase activity to minimize non-specific cleavage in the heteroduplex cleavage reaction.<br><br>
Discussion of the Examples and Figures<br>
Example 1  and Figure 1 show that amplicon yields obtained from PCR amplification are substantially negatively affected when the template DNA is damaged beyond a certain threshold of damage (e.g., about 90 seconds heat treatment) (see Figure 1A). The effect of this damage on amplification can be reversed and amplicon yields enhanced by incubating the DNA with a mixture of enzymes before amplification (see Figures IB, 1C and ID). In addition, amplicon yields of "undamaged" DNA can be enhanced by adding the enzyme mixture described.<br>
Example 1 shows that the effect of the enzyme mixture on amplification of DNA is not dependent on a single type of AP endonuclease or ligase, but instead endonucleases or ligases from multiple alternative sources can be used. For example, thermostable Tth Endo IV was found to be as effective as E. coli Endo IV and E coli ligase was as effective as the thermostable Taq ligase.<br>
Example 2 and Figure 2 show the negative effect on amplification yields of another type of DNA damage - depurination, which is induced in the presence of heat and citrate. Moreover, the example shows that the effect of a mixture of enzymes on amplification of DNA is not dependent on a single type of polymerase but rather polymerases from multiple alternative sources can be used.   For example, E. coli Poll can be substituted by Taq DNA polymerase or a mixture of Taq and Vent® DNA polymerases to produce enhanced yields.<br><br>
Example 3 and Figure 3 show that the enhancement of amplification yields can be observed with short (200 bp) fragments. In fact, enhancement of amplification yields are observed for a wide range of sizes of DNA templates from as short as 100 bases to as long as 100 kb and it is believed that amplification yields for DNA even larger than 100 kb can be achieved. The upper limit of size is limited only by the polymerase in the amplification mixture.<br>
Figure 3 also shows that even when the DNA has been damaged through storage in a crude form (for example, within the cells of an organism that has itself been stored), amplification yields are significantly enhanced by the addition of a mixture of enzymes prior to amplification. Although the mixture of enzymes was added to template DNA prior to amplification, a similar yield effect can be seen when the template DNA is incubated with the mixture ofenzymes that are thermostable equivalents during amplification or during a pre-amplification step.<br>
Example 4 and Figure 4 show that ligase alone can enhance amplicon yield, but adding an AP endonuclease helps more. The best result was observed in this example when a ligase, an AP endonuclease, and a DNA polymerase were used prior to amplification.  Furthermore, this example demonstrates that repair is not DNA size dependent.   For example, similar results were obtained with 5 kb and 10 kb amplicons.<br>
Example 5 and Figure 5 show that an enhanced yield from amplification can be achieved using a ligase and that this effect can be achieved without limitation to a single source of ligase. Figure 5 shows that Taq ligase and T4- ligase are both effective in enhancing amplification yield even when used without additional enzymes in a<br><br>
pre-incubation mix. This effect is also believed to occur if the ligase is added to the amplification mix (if thermostable).   Figure 5 also shows the benefit of this approach to amplifying environmental DNA obtained directly from soil samples that has been exposed in nature to a variety of damaging agents.<br>
All references cited herein as well as U.S. provisional applications serial numbers 60/620,896 filed October 21, 2004, 60/646,728 filed January 24, 2005 and 60/673,925 filed April 22, 2005 are incorporated by reference.<br>
EXAMPLES<br>
Example 1: Enhancing amplification yields for DNA with various extents of damage<br>
An assay was developed for optimizing the use of selected reagents to repair DNA prior to amplification.<br>
Generation of various extents of heat damage<br>
Various amounts of DNA damage were induced by heat treatment. This was achieved as follows: 100 pL lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) at 0.5 mg/ml was aliquoted into separate tubes for heat treatment at 990 for 30 sec, 60 sec, 90 sec, 120 sec, and 180 sec, respectively in a PE2700 thermal cycler. A sample was used as a template for amplification without pretreatment.<br><br>
The remaining damaged DNA was pretreated by the mixture of enzymes as follows: The damaged DNA templates were incubated at room temperature in the following mixture for 10 minutes:<br>
DNA (5ng, 2ng and Ing);<br>
100 uM dNTPs (NEB#M0447, New England Biolabs, Ipswich, MA);<br>
1 mM NAD+ (Sigma#N-7004, Sigma, St. Louis, MO);<br>
80 units Taq ligase (NEB#M0208, New England Biolabs, Ipswich, MA) or 40-100 units of E. coli ligase;<br>
0.1 units E. coli DNA polymerase I (E. coli pPolI) NEB#M0209, New England Biolabs, Inc., Ipswich, MA);<br>
10 units E. coli Endo IV (NEB#M0304, New England Biolabs, Inc., Ipswich, MA) or 10 units of Tth Endo IV;<br>
IX thermopol buffer (NEB#B9004, New England Biolabs, Inc., Ipswich, MA) to a final volume of 96 uL.<br>
At the end of the reaction, the samples were transferred to ice and then amplified.<br>
DNA Amplification Reaction<br>
DNA amplification of lambda was performed using the following primers: CGAACGTCGCGCAGAGAAACAGG (L72-5R) (SEQ ID NO:l) and CCTGCTCTGCCGCTTCACGC (L30350F) (SEQ ID NO:2) according to the method of Wang et al. Nucl. Acids Res. 32: 1197-1207(2004).<br>
4 pi of amplification mixture was added to 96 ^.1 of the above repair mixture. The amplification mixture contained 100 pM dNTPs, 5 units Taq DNA polymerase, 0.1 units Vent® (exo+) DNA<br><br>
polymerase, 5x10-7 M primer L72-5R and 5x10-7 M primer L30350F in IX thermopol buffer.<br>
To correct for any enzyme storage buffer effects, when a repair enzyme was omitted from a reaction, the appropriate volume of its storage buffer was added to the reaction.    In all cases, the amplification reactions were placed into a thermal cycler using the following parameters: 20 sec at 95°C for 1 cycle followed by 5 sec at 94°C, then 5 min at 72°C for 25 cycles. The size of the amplicon being amplified was 5 kb.<br>
The results of amplification of DNA (5 kb) were determined by 1% agarose gel elecrophoresis. 6X loading dye (Molecular Cloning:A Laboratory Manual, 3rd ed., eds. Sambrook and Russell, Cold Spring Harbor Press, Cold Spring Harbor, NY, 2001, pp. 5.4-5.17) was added to the 100 pi amplification reactions.  20 pi of this solution was then loaded onto the agarose gel along with 1 pg of 2-log ladder (NEB#N3200, New England Biolabs, Inc., Ipswich, MA) as a size standard.<br>
The amount of amplified DNA for each sample was compared by gel electrophoresis and the results are shown in Figure 1A. When the samples were treated with a mixture of enzymes after heat treatment but prior to amplification, significant enhancement of amplification yields were achieved (Figures IB, 1C and ID).<br>
Example 2: Increased amplicon yields from DNA with induced abasic sites (after citrate treatments following pretreatment with an enzyme mixture<br>
Generation of various extents of damage resulting from abasic site<br><br>
To assay the extent of repair of damaged DNA, various amounts of DNA damage was first induced by citrate treatment. This was achieved as follows:<br>
DNA was depurinated as described by Ide, H., et al. Biochemistry 32(32):8276-83 (1993). Lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) was ethanol precipitated. The DNA was resuspended in depurination buffer (100 mM NaCI, 10 mM citrate, pH 5.0) at a concentration of 0.5mg/ml and incubated at 70°C for 0, 20, 40, 80, 120, and 160 minutes. The sample was then ethanol precipitated and resuspended in EB buffer (Qiagen, Inc., Valencia, CA). The DNA concentration was determined by measuring the A260 of the DNA containing solutions.<br>
Pretreatment of DNA with a mixture of enzymes<br>
The damaged DNA was incubated at room temperature for 10 minutes in the following mixture:<br>
DNA (2.5ng/120 minute damage);<br>
100 uM dNTPs;<br>
1 mM NAD+;<br>
80 units Taq ligase;<br>
0.1 units Taq DNA polymerase or 0.1 units E. coli Poll (NEB#M0209, New England Biolabs, Inc., Ipswich, MA)) or 0 .1 units 7ag:0.002 units of Vent® Pol, (NEB#M0254, New England Biolabs, Inc., Ipswich, MA));<br>
10 units E. coli Endo IV;<br>
IX thermopol buffer to a final volume of 96 ^1.<br><br>
The above mixture was incubated at room temperature for 10 minutes and then transferred to ice prior to amplification.<br>
DNA amplification reaction<br>
Amplification was performed as described in Example 1 to generate a 5 kb amplicon. Amplicon yields were increased as compared with negative controls (Figure 2A) by treating lambda DNA containing abasic sites with the mixture of enzymes. The results are shown in Figure 2B for a series of pretreatments using different enzyme mixtures. The enzyme mixtures were varied with respect to the polymerase (E. coli Poll or 7ag:Vent®).<br>
Example 3: Improved amplification yield of DNA extracted from an intact organism after storage in a preservative<br>
Genomic DNA was isolated from Meganyctiphanes norvegica (Krill) as described in Bucklin, A. &amp; Allen, L. D. MoL Phylogenet. Evol. 30(3):879-882 (2004). The Krill had been stored in ethanol since 1999.<br>
Pretreatment of the Krill DNA by a mixture of enzymes was carried out as follows:<br>
50 ng of M. norvegica genomic DNA;<br>
100 pM dNTPs;<br>
1 mM NAD+;<br>
40 units of Taq ligase;<br>
0.5 units Taq DNA polymerase, 0.2 units Vent® (exo+) DNA polymerase, or a Tag:Vent® (exo+) mix containing 0.05 units of Taq DNA polymerase and 0.001 units of Vent® (exo+);<br>
10 units E. coli Endo IV;<br><br>
IX Therrnopol buffer to a final volume of 96 \x\. This reaction was incubated 15 minutes at room temperature before proceeding to the amplification step.<br>
DNA amplification reaction<br>
The amplification primers corresponded to 52F and 233R as described in Bucklin, A. &amp; Allen, L D. Mol. Phylogenet EvoL 30(3):879-82 (2004) generating a 200 bp amplicon.<br>
52F: I I I I IAGCAATACACTACACAGCAA (SEQ ID NO:3) 233R: ATTACGCCAATCGATCACG (SEQ ID NO:4)<br>
Primers were added to a final concentration of 0.5 \x\At and each dNTP to a final concentration of 200 pM. 1 pi of the 50:1 Taq:Vent® mix (5 units Taq DNA polymerase and 0.1 units Vent® (exo+) DNA polymerase added to the reaction) was then added to each reaction to a final volume of 100 pL<br>
For the control reaction (lane 1), no Endo IV, Taq ligase or pretreatment polymerase was added. Volumes were adjusted accordingly.  In reactions in which repair enzymes were omitted, the appropriate volume of enzyme storage buffer was added to control for buffer effects.<br>
Cycling conditions were as follows: 30 sec at 94°C, 30 sec at 52CC and 1 min 40 sec at 72°C for 40 cycles. 25vtL (one quarter of<br>
the reaction) was loaded on a 1% agarose gel, prepared, electrophoresed and visualized as described above.<br><br>
Increased amplicon yield from krill genomic DNA was observed after preincubation of the samples using the enzyme mixtures described above (Figure 3).<br>
Example 4: Increased yields of 10 kb amplicon using heat-damaged DNA<br>
Heat-damaged DNA was prepared as described in Example 1. Lambda DNA was heated to 99°C for 180 sec.<br>
Pretreatment of damaged DNA by a mixture of enzymes was carried out as follows:<br>
Lambda DNA (1 ng of 180 sec heat-treated DNA);<br>
100 uM dNTPs;<br>
1 mM NAD+;<br>
80 units of Taq ligase;<br>
0.1 unit of 5. co//Poll;<br>
100 units of E. coli Endo IV;<br>
IX therrnopol buffer to a volume of 96 uL.<br>
The mixture was incubated for 10 minutes prior to amplification.<br>
DNA amplification was performed as described in Example 1 except where specified below.  Primers were added to the above 96 \i\ of preincubation mixture. Primer L71-10R (sequence GCACAGAAGCTATTATGCGTCCCCAGG) (SEQ ID NO:5) replaced L72-5R in Example 1. The icycler thermal cycler program was: Z0 sec at 95°C for 1 cycle, 5 sec at 95°C, 10 min at 72°C for 25 cycles and then 10 min at 72°C for 1 cycle. Amplicon size was 10 kb.<br><br>
The DNA was visualized as described in Example 1 with the following exceptions.  20 ul of 6X loading buffer was added to the 100 ul amplification reaction.   10 ul of this solution was diluted to 100 ul with H2O and IX loading buffer.  20 pi of this was loaded into each lane. The gel was a 0.8% agarose gel. The results a re shown in Figure 4.<br>
Example 5: Improved amplification yield of DNA extracted from soil samples<br>
Environmental DNA was isolated from the soil using an UltraClean Soil DNA Kit from MoBio Laboratories, Inc., Carlsbad, CA (catalog # 12800-50).<br>
Pretreatment of DNA with a ligase<br>
A final volume of 100 pi containing 0.6 ^g of environmental<br>
DNA isolated from soil and one of the two ligases described below in (a) and (b) formed the reaction mixture. This reaction mixture was then incubated at room temperature for 15 min.<br>
(a) IX Taq ligase buffer (New England Biolabs, Inc., Ipswich, MA) and 80 units of Taq ligase.<br>
(b) IX T4 ligase buffer (New England Biolabs, Inc., Ipswich, MA) and 800 units of T4 ligase (Nr3#M0202, New England Biolabs, Inc., Ipswich, MA).<br>
1 fxl of reaction mixture was used in the amplification reaction described below.<br><br>
DNA Amplification Reaction<br>
DNA amplification was performed using primers: GGGGGXAGAGTTTGATCMTGGCTCA (SEQ ID NO:6) and GGGGGXTACGGYTACCTTGTTACGACTT (SEQ ID N0:7) (M = C or A, Y = C or T, X = 8-oxo-Guanine). These primers target 16S rDNA having an amplicon size of 1.6Kb.<br>
The 50 pi reaction contained 10 pmol of each of the primers, 1 pi of the repaired environmental DNA, 200 pM dNTPs, IX thermopol buffer, and 1.25 units Taq DNA polymerase. The amplification was performed using the following cycling parameters: 5 min at 94°C for 1 cycle, 30 sec at 94°C, 1 min at 55°C, 1 min 40 sec at 72°C for 32 cycles, then 5 min at 72°C for 1 cycle.<br>
Gel electrophoresis was performed as described in Example 1. The results are shown in Figure 5.<br>
Example 6:  Improved amplification yield of ultraviolet liqfit-damaged DNA<br>
To determine conditions for assaying the effectiveness of DNA repair, 50 \xg lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) was diluted in TE buffer (10 mM Tris-HCI, 1 mM EDTA, pH 7.5) to a concentration of 50 ng/ml and irradiated with 36 3/rn2<br>
UV light for 0, 10, 20, 30, 40 and 50 sec.<br>
Pretreatment of damaged DNA by a mixture of enzymes was carried out as follows:<br><br>
The damaged DNA was incubated at room temperature for 15 minutes in the following mixture:<br>
DNA (50 ng of lambda DNA-damaged for 0, 10, 20, 3D, 40, or 50 seconds);<br>
200 uM dNTPs;<br>
ImM NAD+;<br>
400 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
80 units or 10 units T4 pdg (also referred to as T4 Endo V). (Trevigen, Gaithersburg, MD);<br>
Thermopol buffer to a volume of 50 \i\.<br>
After the 15 minutes incubation, the 50 n.1 reaction mixture was added to 50 pi of an amplification solution. The amplifi cation solution consisted of 40 pmol of each primer (L72-5R and L30350F as described in Example 1 or L72-2R (the DNA sequence was CCATGATTCAGTGTGCCCGTCTGG) (SEQ ID NO:8), IX Thermopol buffer, 1 mM NAD+, 200 uM dNTPs, 2.5 units Taq DNA polymerase (NEB#M0267, New England Biolabs, Inc., Ipswich, MA), and H20 to a final volume of 50 uL  Combining the 50 uL repair reaction with the 50 pi amplification solution gave a final volume of 100 \J\.<br>
The 100 pi solutions were placed into a thermal cycler. For the L72-5R and L30350F primer combination: 5 min at 94°C for 1 cycle; 30 sec at 94°C, 60 sec at 58°C, a nd 4 min at 72°C for 30 cycles; 5 min at 72°C for 1 cycle. For the L72-2R and L30350F primer combination: 5 min at 94°C for 1 cycle; 30 sec at 94°C, 60 sec at 58°C, and 2 min at 72°C for 30 cycles; 5 min at 72°C for 1 cycle.<br><br>
The presence of amplification product was visualized on a 1.8% agarose gel using ethidium bromide. The size of any band was compared against a lane containing the 2-Iog ladder (NEB#N3200S, New England Biolabs, Inc., Ipswich, MA) size standards. The results are shown in Figure 8.<br>
Example 7: Improved Amplification yield of DNA using the nucleotide excision repair proteins, UvrA, UvrB and UvrC<br>
Increased amplicon yield from krill genomic DNA is determined after preincubation of the samples using an enzyme mixture containing proteins involved in nucleotide excision repair.<br>
Pretreatment of stored DNA by a mixture of enzymes is carried out as follows:<br>
Stored DNA is incubated for 1-180 minutes at 4~37°C in the following mixture:<br>
DNA:   50 ng of M. norvegica genomic DNA; 1O0 pM dNTPs;<br>
1 mM ATP;<br>
4O0 units of Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 nM E. coli UvrA, 250 nM E. coli UvrB (or mutant UvrB^), plus or minus 50 nM E. coli UvrC<br>
IX Thermopol buffer to a final volume of 96 \i\.<br>
* for mutant UvrB, see Zou, Y., et al. Biochemistry 43:4196-4205 (2004).<br>
DNA amplification reactions are conducted as described in Example 3.<br><br>
Example 8: Increasing sequence accuracy of a DNA amplification reaction by enzyme cleavage of heteroduplexes<br>
Experimental Conditions<br>
A.  Adding Taq ligase to T7 Endo I was demonstrated to increase the T7 Endo P.DNA ration in a reaction mixture without randomly degrading the DNA. This approach makes it possible to reduce unwanted heteroduplexes resulting from mismatches in an amplification reaction.<br>
The assay relies on treating a supercoiled DNA containing a cruciform structure with increasing amounts of T7 Endo I.<br>
0, 1.6, 3.1, 6.2, 12.5, 25, 50, 100, 200, or 400 units of T7 Endo I (NEB#M0302, New England Biolabs, Inc., Ipswich, MA) was added to 50 IJI reactions composed of 1 pg of pUC(AT) (Guan, C, et.al. Biochemistry 43:4313-4322 (2004)) and IX NEBuffer 2 (NEB#B7002S, New England Biolabs, Inc., Ipswich, MA).   Plasmid pAT25tetA can be used in place of pUC(AT) (Parkinson, M. J . &amp; Lilley, D. M. J. MoL Biol, 270:169-178 (1997))  and Bowater, R. P., et. al. Biochemistry 33:9266-9275 (1994)).  Another set of reactions were set up simultaneously and used the same components as described above with the addition of 1 mM NAD+ (Sigma catalog#N-7004, Sigma, St. Louis, MO) and 100 units of Taq ligase (using a stock of NEB#M0208 at a concentration of 100 u/pl).  All reactions were incubated at 37°C for 60 minutes.<br>
The results were analyzed by running the reactions on a 0.9% TBE agarose gel, stained with ethidium bromide, and visualized<br><br>
using UV light (see Figure 9). With no T7 Endo I present the pUC(AT) plasmid produced 2 bands on the gel corresponding to the supercoiled form (lower band) and the relaxed circular form (upper band).<br>
T7 Endo I resolved the supercoiled pUC(AT) into the relaxed circular form and a linear form that ran intermediate to the supercoiled and relaxed circular forms.  At certain T7 Endo I:DNA ratios, a smear was produced indicating that the T7 Endo I had degraded the DNA by non-specific enzymatic activity.   The presence of Taq ligase significantly increased the usable T7 Endo I to DNA ratio.  This ratio is further improved by substituting T7 Endo I with the mutant T7 Endo I described in International Publication No. WO 2005/052124.<br>
B.   Experimental conditions for determining the effectiveness of the T7 Endo I and ligase mix for removing heterduplexes from PCR reactions.<br>
Isolation of DNA from soil and amplification of the purified DNA is performed as described in Example 5 with the optional addition of 5 units T7 Endo I or mutant thereof. When T7 Endo 1 or mutant thereof is added, an additional amplification cycle is added (37°C for 15 minutes for 1 cycle). The last step is to allow the AP endonuclease to cleave any heteroduplexes formed.<br>
Gel electrophoresis is performed as described in Example 1. Heteroduplex DNA is visualized on the gel as described in Lowell, 3. L &amp; Klein, D. A. Biotechniques 28:676-681 (2000)).   Absence of heteroduplex DNA in the presence of T7 Endo I or mutant thereof shows the effectiveness of T7 Endo I or mutant thereof with ligase.<br><br>
Unit definitions are described with the product description for each of the enzymes recited herein in the NEB catalog, New England Biolabs, Inc., Ipswich, MA. For example, unit definition for T7 Endo I or mutant thereof is the amount of enzyme required to convert greater than 90% of 1 \iq of supercoiled plasmid into greater than 90% linear DNA in a reaction volume of 50 \i\ in 1 hour at 37°C.<br>
The T7 Endo I:DNA ratio can be increased without increasing non-specific cleavage of DNA in the presence of ligase.<br>
Example 9: Increasing the sequence accuracy of a DNA amplication reaction after oxidative damage<br>
Generating DNA with oxidative damage<br>
The DNA subject to oxidative damage was pWB407 (Kermekchiev, M. B. et al. Nucl. Acids Res. 31:6139-47 (2003)). The damage was incurred using a combination of methylene blue (MB) and visible light as described previously (Sattler, et al. Arch. Biochem Biophys. 376(1):26-3 (2000)).   Plasmid DNA (200 ng/ml in distilled water) was spotted on parafilm stretches (50 ^,1 drops). MB was added to the drops to a final concentration ranging from 0 to 50 (0, 3, 6, 12.5, 25 and 50) ug/ml (100 \i\ final volume). Plates<br>
with these parafilm stretches were placed on ice and illuminated for 8 min. with a 1 x 100-W lamp. The MB-light-treated DNA was precipitated, dried, and then resuspended in 50 pi of TE buffer (pH 8.0). Final DNA concentration was determined by the absorbance of light at 260 nm.<br><br>
DNA amplification conditions<br>
A portion of pWB407 that contained the lacZ gene was amplified using primers 316-138,<br>
TGTCGATCAGGATGATCTGGACGAAGAGC (SEQ ID NO:9), and 316-137, CGAAAGCTTTCAAGGATCTTACCGCTGTTGAGA (SEQ ID NO:10). Primers 316-138 and 316-137 were based on the previously-described primers Kfd-29 and H3Bla34, respectively (Kermekchiev, M. B. et al. NucL Acids Res. 31:6139-47 (2003)). The 100 pL PCR reactions contained either 10 or 50  ng of template DNA, indicated where appropriate, and 40 picomoles of each primer. The cycling conditions utilized varied with the thermal stable polymerase used for amplification.<br>
Cycling conditions when using Taq DNA polymerase (NEB cat#M0267S, New England Biolabs, Inc., Ipswich, MA) had an initial denaturation step of 5 min at 94°C for 1 cycle, then 30 sec at 94°C, 60 sec at 58°C, and 3 min 30 sec at 72°C for 30 cycles, and finally 5 min at 72°C.<br>
Cycling conditions when using Phusion DNA polymerase (NEB cat#F-530S, New England Biolabs, Inc., Ipswich, MA) had an initial denaturation step of 30 sec at 98°C for 1 cycle, then 10 sec at 98°C, 30 sec at 62°C, and 1 min 30 sec at 72°C for 30 cycles, and finally 5 min at 72°C.<br>
The reaction outcomes were analyzed by loading 25 i^L of the reaction on a 1.6% agarose gel, prepared, electrop horesed and visualized as described above.  The marker used was the 2-log DNA ladder (NEB cat#N3200S, New England Biolabs, Inc., Ipswich, MA).<br><br>
Amplification accuracy determination<br>
The accuracy of DNA amplification from the pWB-407 template was determined as described by Barnes, et al. Gene lL2:29-35 (1992) and Kermekchiev, et al. Nucl. Acids Res. 31:6139-47 (2003).   Amplicons containing the lacZ gene were generated from plasmids pWB407 that had been subjected to differing amounts of oxidative damage. The oxidative damage was performed using methylene blue as described above.  Tne PCR reactions were performed using 50 ng of template as described above.    After cycling, 10 units of the restriction endonuclease Dpnl was added to each 100 pL PCR reaction and incubated for 2 hours at 37°C.  This step eliminated the original template plasmid.   Next, the resulting amplification products were extracted with phenol/chloroform precipitation using isopropanol (Molecular Cloning: A Laboratory Manual, 3rd ed., eds. Sambrook and Russell, Cold Spring Harbor Press, Cold Spring Harbor, NY, 2001, pp. 6.25, A8.12-A8.24). Precipitated products were resuspended in H2O and cut with the restriction endonucleases Styl and Hindlll using conditions recommended by the manufacturer (New England Biolabs, Inc., Ipswich, MA).  The DNA digestion reactions were stopped by inactivating the Hindlll and Styl enzymes by heating to 65°C for 20 min. The restriction digestion products were purified using a microcon YM-100 column (Miilipore, Billerica, MA) to eliminate short DNA fragments.<br>
The repair reaction mixtures in a total of 50 pi contained 10 or 50 ng of pWB407 amplicons +/- methylene blue incubation.  The repair reactions contained 20 mM Tris-HCI (pH 8.8 at 25°C), 10 mM KCI, 10 mM (NH4)2S04/ 2 mM MgS04/ 0.1% Triton X-L00, l mM<br><br>
NAD+, 200 pM dNTPs (dATP, dTTP, dCTP, and dGTP), and various repair enzyme mixtures.<br>
The repair enzyme mixtures used separately or in various combinations in a total volume of 50ul were:<br>
0.4 units Fpg, NEB cat#M0240S, New England Biolabs, Inc., Ipswich, MA);<br>
200 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
1 mM NAD+;<br>
100 [xM dNTPs;<br>
IX Thermopol buffer.<br>
The reactions were incubated at 25°C for 15 minutes.  After the incubation, 50 pL of a PCR mix (20 mM Tris-HCI (pH 8.8 at 25°C), 10 mM KCI, 10 mM (NH4)2S04/ 2 mM MgS04, 0.1% Triton X-100, 1 mM NAD+, 200 pM dNTPs (dATP, dTTP, dCTP, and dGTP), 2.5 units Taq DNA polymerase (NEB cat#M0267S, INew England Biolabs, Inc., Ipswich, MA) was added to the 50 pL repair reaction and this new solution was subjected to thermal cycling conditions for PCR. The ampiicons from these reactions were purified and restriction enzyme digested as described for other ampiicons above.<br>
The ampiicons were cloned into the pWB407 plasmid. Plasmid pWB407 was prepared by digestion with ttie restriction endonucleases Styl and Hindlll followed by a 30-minute incubation at 37°C with 1 unit/pg DNA of antarctic phosphatase (NEB cat#M0289S, New England Biolabs, Inc., Ipswich, MA).  The dephosphorylated pWB407 vector backbone was purified by agarose<br><br>
gel electrophoresis.  Gel extraction was performed with a QIAquick Gel Extraction Kit (Qiagen, Valencia, CA).<br>
The digested amplicons were ligated into the prepared pWB407 plasmids in 30 pL reactions using approximately 0.1 pg vector DNA and about 0.5 pg amplicon.  T4 ligase was used to perform the ligation following the manufacturers recommended conditions (New England Biolabs, Inc., Beverly, MA).   Ligation products were electroporated into E. coli strain WB441 (Barnes, W. Gene 112:29-35 (1992)). The selective indicator plates used were LB plates containing 50 txg/ml ampicilin and 80 ug/ml Xgal. Before<br>
plating, the bacteria were incubated in rich broth for 1 hour at 37°C to allow expression of the ampicilin resistance. Control transformations lacking ligase treatment resulted in zero colonies. Colonies were scored for blue color after one day at 37°C, and one or two days at 25°C. The results are shown in Figures 10 and 11.<br>
Example 10: Increasing the sequence accuracy of a DNA amplification reaction after deamination damage<br>
Generating deaminated DNA<br>
The DNA subject to deamination was pWB407 (Kermekchiev, et al. Nucleic Acids Research, 2003, Vol. 31, 6139-6147 ).  The damage was incurred using random mutagenesis with nitrous acid as described in Yan, W. etal. J Virol. 2003 Feb;77(4):2640-50. Nitrous acid can deaminate guanine in DNA to xanthine, cytosine to uracil, and adenine to hypoxanthine.<br><br>
Plasmid DNA (2mg) was treated with 0.7 M Na No2 in 1M acetate buffer, pH 4.6. The reaction was terminated at various time points by addition of 4 volumes of ice-cold 1 M Tris-CI (pH 7.9. The plasmid DNA was precipitated, dried and then resuspended in 100 ml of TE buffer.<br>
Pretreatment reaction to repair deaminated bases<br>
The repair enzyme mixtures used separately or in various combinations in total volume of 50 ml were:<br>
(a)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo III (NEB cat # M0268S), New England Biolabs, Inc., Ipswich, MA;<br>
2 units Endo V (NEB cat # M0305S), New Eng land Biolabs, Inc., Ipswich, MA;<br>
2 units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units E. coli Endo IV;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(b)<br>
2 units Endo V (NEB cat # M0305S), New England Biolabs,<br>
Inc., Ipswich, MA;<br>
2 units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br><br>
200 units E. coli Endo IV;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(c)<br>
2 units Endo V (NEB cat # M0305c), New England Biolabs, Inc., Ipswich, MA;<br>
200 units E. coli Endo IV;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
.(d)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo III (NEB cat # M0268S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units E. coli Endo IV;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(e)<br>
1 unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br><br>
2 units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units E. coli Endo IV;<br>
0.1 units £ coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
1 mM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(0<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo V (NEB cat # M0305S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units £ coli Endo IV;<br>
0.1 unit £ coli DNA polymerase I;<br>
10 units £ coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
The amplification reaction conditions and amplification accuracy determination are performed as described in Example 9.<br>
Example 11: Unit Definitions<br>
Thermophilic ligase unit<br>
One unit is defined as the amount of enzyme required to give 50% ligation of 1 ug of BstE Il-digested lambda DNA in a total reaction volume of 50 pi in 15 minutes at 45°C.<br><br>
Mesophilic ligase unit<br>
One unit is defined as the amount of enzyme required to give 50% ligation of Hind III digested lambda DNA (5' DNA termini concentration of 0.12 pM, 300 pg/ml) in a total reaction volume of 20 pi in 30 minutes at 16°C.<br>
AP endonuclease unit<br>
One unit is defined as the amount of enzyme required to cleave 1 pmol of a 34-mer oligonucleotide duplex containing a single AP site in a total reaction volume of 10 pi in 1 hour at 37°C.<br>
Mesophilic polymerase unit<br>
One unit is defined as the amount of enzyme that will incorporate 10 nmol of dNTP into acid-insoluble material in a total reaction volume of 50 pi in 30 minutes at 37°C with 33 pM dNTPs including [3H]-dTTP and 70 pg/ml denatured herring sperm DNA.<br>
Thermophilic polymeraseuUnit<br>
One unit is defined as the amount of enzyme that will incorporate 10 nmol of dNTP into acid-insoluble material in a total reaction volume of 50 pi in 30 minutes at 75°C with 200 pM dNTPs including [3H]-dTTP and 200 pg/ml activated Calf Thymus DNA.<br>
For unit definitions for UDG and Fpg, (see NEB catalog, New England Biolabs, Inc., Ipswich, MA).<br><br><br><br><br><br><br><br><br><br>
IN THE CLAIMS<br>
L. A method for enhancing at least one of fidelity and yield of an amplification product of a damaged polynucleotide, comprising:<br>
(a)	incubating the polynucleotide in the absence of Endonuclease (Endo) VI in a reaction mixture comprising a ligase and a cofactor selected from NAD+ or ATP;<br>
(b)	permitting amplification of the polynucleotide to occur in the reaction mixture by the addition of amplification reagents to the reaction mixture during or after step (a); and<br>
(c)	enhancing at least one of fidelity or yield of the amplification product in the presence of step (a) compared to in the absence of step(a).<br><br>
2.	A method according to claim 1, wherein step (c) is independent of a time of incubation in step (a).<br>
3.	A method according to claim 1, wherein the ligase is a thermostable ligase.<br>
4.	A method according to claim 1, wherein the damage is selected from apurinic/apyrimidinic (AP) sites, mutagenized nucleotides, modified nucleotides, nicks, gaps and DNA-DNA or DNA-protein cross-links.<br>
5.	A method according to claim 1, wherein the polynucleotide is obtained from a natural source.<br>
6.	A method according to claim 1, wherein the polynucleotide is<br>
obtained from preserved biological material.<br><br>
7.	A method according to claim 1, wherein the polynucleotide is obtained from forensic evidence.<br>
8.	A method according to claim 1, wherein the polynucleotide is ancient.<br>
9.	A method according to claim 1, wherein the polynucleotide is obtained from a tissue biopsy.<br><br>
10.	A method according to daim 1, wherein the reaction mixture further comprises a T7 Endo I or mutant thereof.<br>
11.	A method according to claim 1, wherein the reaction mixture further comprises a polymerase and an AP endonudease.<br>
12.	A method according to claim 2, wherein the reaction mixture further comprises a polymerase and a class II AP endonudease.<br>
13.	A method according to claim 3, wherein the ligase is selected from a Taq ligase and an E. coli ligase and the cofactor is NAD+.<br>
14.	A method according to claim 11, wherein the AP endonudease comprises a T4 endonudease or an E. coli endonudease.<br><br>
15.	A method according to claim 14, wherein the AP endonudease is E coli Endo IV.<br>
16.	A method according to claim 11, wherein the polymerase comprises a Taq polymerase, an E. coli polymerase or an archaeal polymerase or mutant thereof.<br><br>
17.	A method according to claim 16, wherein the archael polymerase is selected from Pfu, Vent®, Deep Vent®, 9°North, and GBD polymerase.<br>
18.	A method according to claim 11, wherein the polynucleotide is DNA and the reaction mixture comprises 1-100 units of E. coli Endo IV, 0.05-0.25 units of E coli Poll and 5-500 units of ligase added to 1-1000 ng DNA.<br>
19.	A method according to claim 11, 12 or 13, wherein the reaction mixture further comprises T4 pyrimidine dimer glycosylase.<br>
20.	A method according to claim 11, 12 or 18, wherein the reaction mixture further comprises [fapy]-DNA glycosylase (Fpg).<br>
21.	A method according to claim 11, 12 or 18, wherein the reaction mixture further comprises at least one of UvrA, UvrB, UvrC, UvrD and Cho.<br><br>
22.	A method according to claim 11, 12 or 18, wherein the reaction mixture further comprises Endo V or Endo III.<br>
23.	A method according to claim 22, wherein the reaction mixtu re further comprises Endo V and at least one of UDG and Aag or Endo III and at least one of UDG and Aag.<br>
24.	A method according to claim 11, 12 or 18 wherein the mixture further comprises UDG and Aag.<br><br>
25.	A method according to claim 1, wherein the amplification is PCR amplification, helicase-dependent amplification, transcription-mediated amplification, strand-displacement amplification, roiling circle amplification or whole genome amplification.<br>
26.	A method according to claim 1, wherein the polynucleotide is a single-stranded RNA and the amplification is RT-ampIification.<br>
27.	A method according to claim 1, wherein amplification of the polynucleotide is capable of producing an amplicon in a size range of 50 nucleotides to 100,000 nucleotides in a polymerase chain reaction.<br><br>
28.	An amplification kit, comprising: instructions for use and o ne or more enzymes wherein at least one of the enzymes is a ligase,  the one or more enzyme being formulated for addition to an amplification mixture to enhance amplification or for use prior to addition of the amplification mixture to enhance amplification.<br>
29.	A composition, comprising:<br>
an effective amount of a ligase, a polymerase, and an AF&gt; endonuclease not including Endo VI, the mixture being capable of enhancing at least one of yield and fidelity of amplification of a polynucleotide compared with amplification of the polynucleotide in the absence of the composition.<br>
30.	A composition according to claim 29, wherein the AP<br>
endonuclease is present at a concentration of 1-100 units, the<br>
polymerase is present at 0.05-0.25 units, and the ligase is present<br>
at 5-500 units in a volume 10-100 \i\ for use in a pre-amplification<br>
mixture.<br><br>
31.	A composition according to claim 30, suitable for repairing<br>
1-1000 ng DNA.<br>
32.	A composition according to claim 31, wherei n the AP endonuclease is a Type II AP Endonuclease.<br>
33.	A composition according to claim 32, wherei n the AP endonuclease is E. coli Endo IV.<br>
34.	A composition according to claim 29, wherei n the polymerase is<br>
E. coli poll.<br>
315 A composition according to claim 29, further comprising T4 pyrimidine dimer glycosylase.<br>
36.	A composition according to claim 29, further comprising [fapy]-<br>
DNA glycosylase (Fpg).<br>
37.	A composition according to claim 29, further comprising at least<br>
one of UvrA, UvrB, UvrC, UvrD and Cho.<br>
38.	A composition according to claim 29, further comprising Endo V<br>
or Endo III.<br>
39.	A composition according to claim 33, further comprising Endo V<br>
and at least one of UDG and Aag or Endo III and at least one of<br>
UDG and Aag.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3ICAgIGZvcm0tMyAgICAyOS0wNy0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">1635-chenp-2007    form-3    29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgIE9USEVSIFBBVEVOVCBET0NVTUVOVCAgIDI5LTA3LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007    OTHER PATENT DOCUMENT   29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMDctMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   AMENDED  PAGES OF SPECIFICATION  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgQU1FTkRFRCBDTEFJTVMgIDA3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   AMENDED CLAIMS  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAzMC0wMS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   CORRESPONDENCE OTHERS  30-01-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVEICAgIDI5LTA3LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   EXAMINATION REPORT REPLY RECEIVED    29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgRk9STS0xICAwNy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   FORM-1  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgRk9STS0xMyAgMDktMDUtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   FORM-13  09-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAwNy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   OTHER PATENT DOCUMENT  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICAgUE9XRVIgT0YgQVRUT1JORVkgIDA3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007   POWER OF ATTORNEY  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1DSEVOUC0yMDA3ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDA3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">1635-CHENP-2007  CORRESPONDENCE OTHERS  07-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTYzNS1jaGVucC0yMDA3LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1635-chenp-2007-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250915-a-process-for-treating-anaerobically-decomposed-organic-material.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250917-method-for-producing-directionally-solidified-silicone-ingots.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250916</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1635/CHENP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>06/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>07-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Apr-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NEW ENGLAND BIOLABS, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>240 COUNTY ROAD, IPSWICH, MA 01938, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>EVANS, THOMAS, C</td>
											<td>45 FOX RUN ROAD, TOPSFIELD, MA 01983, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SLATKO, BARTON</td>
											<td>11 BRENTWOOD WAY, IPSWICH, MA 01938, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHEN, LIXEN</td>
											<td>8 MORRISON AVENUE, BEVERLY, MA 01915, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VAISVILA, ROMALDAS</td>
											<td>UNIT 8, 100 CENTRAL STREET, IPSWICH, MA 01938, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GUAN, CHUDI</td>
											<td>10 GREAT POND ROAD, WENHAM, MA 01984, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US05/038281</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/646,728</td>
									<td>2005-01-24</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/620,896</td>
									<td>2004-10-21</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/673,925</td>
									<td>2005-04-21</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250916-repair-of-nucleic-acids-for-improved-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:10:34 GMT -->
</html>
