<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/250960-process-for-generating-and-detecting-recombinant-dna-sequences-in-saccharomyces-cerevisiae by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:09:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 250960:&quot;PROCESS FOR GENERATING AND DETECTING RECOMBINANT DNA SEQUENCES IN SACCHAROMYCES CEREVISIAE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;PROCESS FOR GENERATING AND DETECTING RECOMBINANT DNA SEQUENCES IN SACCHAROMYCES CEREVISIAE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to methods for generating and detecting recombinant DNA sequences in Saccharomyces cerevisiae and plasmids and Saccharomyces cerevisiae cells used for conducting the inventive methods.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to process for generating and detecting recombinant dna sequences in saccharomyces cerevisiae.<br>
Description<br>
The present invention relates in general to methods for generating and detecting recombinant DNA sequences in Saccharomyces cerevisiae and plasmids and S. cerevisiae cells used for conducting the inventive methods.<br>
DNA sequences for which these methods are relevant include protein-encoding and non-coding sequences; they may also consist of larger continuous stretches that contain more than a single coding sequence with intervening non-coding sequences, such as those that as may belong to a biosynthetic pathway.<br>
The microbiaf and enzymatic production of substances such as enzymes and other proteins is an important economical topic. Enzymes are bioeatalytically active proteins not only responsible for the metabolism of natural compounds and organisms, but also utilized for the industrial production of natural and non-natural compounds. Enzymes or those compounds produced by the help of enzymes can be used for the production of drugs, cosmetics, foodstuffs, etc. However, the industrial use of enzymes has been greatly hindered by their target specifity and the specific conditions under which they can function. Other, proteins have therapeutic applications in the fields of human and animal health. Important classes of medically important proteins include cytokines and growth factors.<br>
Proteins, enzymes, and pathways with novel or improved 'functions and properties can be obtained either by searching among largely unknown natural species or by improving upon currently known natu-<br><br>
proteins or enzymes. The latter approach may be more suitable<br>
for creating properties for which natural evolutionary processes are<br>
unlikely to have been selected.<br>
One promising strategy to create such novel desirable properties<br>
and to redesign enzymes other proteins non-coding sequences or<br>
pathways is by directed molecular evolution. Conventionally as direct<br>
evolution of DMA sequences has been achieved with such techniques<br>
as site-directed mutagenesis multi-site or cassette<br>
mutagenesis random mutagenesis and error prone PCR. Recently<br>
gene shuffling approaches to optimize or fine-tune the properties of<br>
enzymes or proteins have attracted much attention. These directed<br>
evolutionary techniques can produce enzymes that can improve existing<br>
technology produce novel products and expand the capabilities<br>
of synthetic chemistry.<br>
A number of different mutagenesis methods exist such as random<br>
mutagenesis site-directed mutagenesis oligonucleotide cassette<br>
mutagenesis or point mutagenesis by error-prone PCR. Random<br>
mutagenesis for example entails the generation of a large number<br>
of randomly distributed nucleotide substitution mutations in cloned<br>
DNA fragments by treatment with chemicals such as nitrous acid<br>
hydrazine etc. Error-prone PCR has been developed to introduce<br>
random point mutations into cloned genes. Modifications that decrease<br>
the fidelity of the PCR reaction include increasing the concentration<br>
of MgCI2l adding MnCI2ll or altering the relative concentrations<br>
of the four dNTPs.<br>
These traditional mutagenesis methods focus on the optimization of<br>
individual genes having discrete and selectable phenotypes. The<br>
general strategy is to clone a gene identify a discrete function for<br>
the gene establish an assay by which it can be monitored mutate<br>
selected positions in the gene and select variants of the gene for<br>
improvement in the known function of the gene. A variant having improved<br>
function can then be expressed in a desired cell type.<br>
Repetitive cycles of mutagenesis methods can be carried out to obtain<br>
desirable enzyme properties.<br>
Each of these conventional approaches has an implicit sequence<br>
search strategy. The strategies employed in the above techniques of<br>
sequence searching are very different. Performing a saturating sitedirected<br>
mutagenesis search involves a process of installing every<br>
possible permutation at a site of interest. For a protein this procedure<br>
consists of replacing an amino acid at a site of interest with all<br>
19 other amino acids and searching the resultant library for improved<br>
mutants. In sequence space terms this means that a very small region<br>
has been searched very thoroughly. In comparison cassette- .<br>
mutagenesis inserts a random peptide sequence in a specific region<br>
of a protein giving a less thorough sampling of a larger defined region<br>
of sequence space. Error-prone PCR involves repeated copying<br>
of a sequence with the introduction of a low but significant number<br>
of errors. In this case a sparse sampling of a less defined region of<br>
sequence space is achieved. In each of these strategies the best<br>
mutant obtained in each round of selection is used to initiate the next<br>
round.<br>
However traditional mutagenesis approaches for evolving new properties<br>
in enzymes have a number of limitations. First they are only<br>
applicable to genes or sequences that have been cloned and funcfunctionally<br>
characterized. Second these approaches are usually<br>
applicable only to genes that have a discrete function. Therefore<br>
multiple genes that cooperatively confer a single phenotype usually<br>
cannot be optimized in this manner. Finally these approaches can<br>
only explore a very limited number of the total number of permutations<br>
even for a single gene. In view of these limitations conventional<br>
mutagenesis approaches are inadequate for improving cellular<br>
genomes with respect to many useful properties. For example<br>
improvements in the capacity of a cell to express a protein might require<br>
alterations in transcriptional efficiency translation and posttranslational<br>
modifications secretion or proteolytic degradation of a<br>
gene product. It therefore might be necessary to modify additional<br>
genes having a role in one or more of these cellular mechanisms in<br>
order to express a protein with new properties. Attempting to individually<br>
optimize all of the genes having such function would be a<br>
virtually impossible task.<br>
Most of the problems associated with conventional mutagenesis approaches<br>
can be overcome by gene shuffling approaches. Gene<br>
shuffling entails randomly recombining different sequences of functional<br>
genes enabling the molecular mixing of naturally similar or<br>
randomly mutated genes. DNA or gene shuffling or variations of<br>
these techniques have been used to improve the activity stability<br>
folding and substrate recognition properties of enzymes. In comparison<br>
to conventional mutagenesis approaches with gene shuffling<br>
the probability of obtaining mutants with improved phenotype is<br>
significantly higher. Gene shuffling is fundamentally different from<br>
conventional strategies in that it recombines favorable mutations in a<br>
combinatorial fashion. It therefore will search much larger regions of<br>
sequence space much more sparsely and with a bias towards producing<br>
functional sequences. It also allows more beneficial mutations<br>
from each round of selection to be retained in the next round<br>
because it allows sequence information to be contributed from more<br>
than one source. Whereas conventional strategies also allow for the<br>
fixation of negative mutations this is not the case for gene shuffling<br>
approaches. Therefore it is not surprising that gene shuffling strategies<br>
have yielded much more dramatic results.<br>
DMA or gene shuffling approaches are based on recombination<br>
events between regions with a certain homology or between<br>
stretches of identity. A key organism used in experiments to examine<br>
genetic recombination in eukaryotes has been the budding yeast<br>
Saccharomyces cerevisiae. The study of these processes in a simple<br>
unicellular organism has the obvious advantage of the ease of<br>
manipulation of DMA sequences and the possibility of studying specific<br>
recombination events induced synchronously in a large proportion<br>
of cells. Furthermore over the last few decades a wealth of expertise<br>
has been accumulated both in the fermentation technology<br>
and the basic genetics of this organism which is at present the best<br>
studied eukaryote at the molecular level. Due to its non-pathogenic<br>
character its secretion proficiency and its glycosylation potential S.<br>
cerevisiae is a preferred host organism for gene cloning and gene<br>
expression. Therefore the technical problem underlying the present<br>
invention is to provide methods and means for the generation of recombinant<br>
mosaic genes in Saccharomyces cerevisiae.<br>
The present invention solves this underlying technical problem by<br>
providing a process for generating and detecting recombinant DNA<br>
sequences in Saccharomyces cerevisiae comprising the steps of:<br>
a) generating first diploid S. cerevisiae cells bearing in a defined<br>
locus of their genome a first recombination cassette comprising<br>
a first DNA sequence to be recombined which is flanked<br>
by at least a first and a second marker sequence in an allelic<br>
position a second recombination cassette comprising a second<br>
DNA sequence to be recombined which is flanked by at<br>
least a third and a fourth marker sequence<br>
b) inducing the sporulation of the first diploid cells obtained in a)<br>
and<br>
c) isolating haploid cells containing recombination cassettes in<br>
which first recombined DNA sequences are flanked by the<br>
first and fourth marker sequences and haploid cells containing<br>
recombination cassettes in which second recombined<br>
DNA sequences are flanked by the second and the third<br>
marker sequences.<br>
The present invention provides a yeast-based system to screen for<br>
recombination events between at least two diverging DNA sequences.<br>
The system is based on the sexual reproductive cycle of S.<br>
cerevisiae which alternates between a haploid phase and a diploid<br>
phase. In the first step of the inventive process diploid S. cereivisiae<br>
cells are generated which are heterozygous for these recombination<br>
substrates. The DNA sequences to be recombined are integrated in<br>
the genome of the diploid S. cerevisiae cells at allelic positions.<br>
Each DNA sequence to be recombined is integrated in the form of a<br>
recombination cassette which comprises besides this DNA sequence<br>
at least two marker sequences that flank the DNA sequence<br>
whereby the two recombination cassettes comprise at least four different<br>
marker sequences.<br>
The heterozygous diploid cells thus obtained are then grown under<br>
conditions which induce the processes of meiosis and spore formation.<br>
Meiosis is generally characterized by elevated frequencies of<br>
genetic recombination which is initiated via the formation and subsequent<br>
repair of double-strand breaks (DSBs) induced early in<br>
meiosis I prophase. Yeast meiotic cells are therefore of particular<br>
interest because they experience high levels of recombination as a<br>
result of the genome-wide induction of DSBs. Thus the products of a<br>
first round of meiosis which are haploid cells or spores for each<br>
meiosis event four produced by a parental diploid cell can contain<br>
recombined DNA sequences due to recombination between the two<br>
diverged DNA sequences.<br>
Recombination between the two diverging DNA sequences during<br>
meiosis can also lead to an exchange of the flanking marker sequences.<br>
Therefore the present process allows a rapid and simple<br>
identification of recombined DNA sequences by the selection of individual<br>
cells or molecules in which an exchange of marker sequences<br>
flanking a recombination substrate has taken place. The recombinants<br>
obtained after the first round of meiosis are therefore characterized<br>
in that they contain and/or express at least one marker sequence<br>
of the first recombination cassette and at least one marker<br>
sequence of the second recombination cassette. In particular recombinant<br>
spores can contain the first marker sequence of the first<br>
recombination cassette and the fourth marker sequence of the second<br>
recombination cassette or the second marker sequence of the<br>
first recombination cassette and the third marker sequence of the<br>
second recombination cassette whereby both types of recombinant<br>
spores contain besides this different marker combination also different<br>
recombinant DNA sequences. Both types of spores containing<br>
recombinant sequences can easily be selected and distinguished<br>
under conditions that permit selection for the new recombinant<br>
marker configurations produced by recombination during meiosis.<br>
The inventive process can be conducted either in wild-type or mismatch<br>
repair-defective S. cerevisiae cells. The processes by which<br>
damaged DNA is repaired and the mechanisms of genetic recombination<br>
are intimately related and it is known that the mismatch repair<br>
machinery has inhibitory effects on the recombination frequency between<br>
divergent sequences i.e. homeologous recombination. Mutations<br>
of the mismatch repair system therefore greatly enhance the<br>
overall frequency of recombination events in yeast. On the other<br>
hand it is known that wild-type S. cerevisiae cells have a mismatch<br>
repair-dependent recombination mechanism which is based on distantly<br>
spaced mismatches in two recombination substrates. Depending<br>
on the DNA sequences to be recombined either wild-type or<br>
mismatch repair-defective S. cerevisiae cells can be used to obtain<br>
recombined sequences.<br>
The inventive process has the advantage that it is iterative i.e. it<br>
allows further rounds of recombination. The products of the first<br>
round of meiosis i.e. haploid cells of opposite mating types which<br>
comprise different recombined DNA sequences are mated again to<br>
obtain diploid cells which are heterozygous for recombined DNA sequences.<br>
In the diploid cells thus obtained meiosis is again induced<br>
whereby the recombined DNA sequences are once again recombined<br>
leading again to an exchange of the two markers flanking<br>
each recombination substrate. The new haploid recombinants obtained<br>
after the second meiosis can now be easily identified by the<br>
joint expression of either those marker genes which flanked the first<br>
DNA sequence in the original first recombination cassette or those<br>
marker genes which flanked the second DNA sequence in the original<br>
second recombination cassette.<br>
In a preferred embodiment of the invention therefore haploid cells<br>
containing a recombination cassette with the first recombined DNA<br>
sequences obtained in the first round of the inventive process are<br>
mated with haploid cells containing recombination cassettes with the<br>
second recombined DNA sequences obtained in the first round of<br>
the inventive process in order to generate second diploid cells. In<br>
the thus obtained second diploid cell sporulation is induced resulting<br>
in the generation of haploid cells. In the next steps haploid cells<br>
containing recombination cassettes in which third recombined DNA<br>
sequences are flanked by at least the first and second marker sequences<br>
and haploid cells containing recombination cassettes in<br>
which fourth recombined DNA sequences are flanked by at least the<br>
third and fourth marker sequences are isolated.<br>
Further recombined DNA sequences can be generated by subjecting<br>
the haploid cells containing third and fourth recombined DNA sequences<br>
to one or more further cycles of mating and meio10<br>
sis/sporulation. After each round of recombination recombinants are<br>
either identified by the joint presence of at least one marker sequence<br>
that flanked the first recombination substrate and at least<br>
one marker sequence that flanked the second recombination substrate<br>
or by the joint presence of the two markers that flanked the<br>
first or the second DNA sequence in the starting recombination substrates.<br>
Therefore an advantageous feature of the present process is that it<br>
is iterative: recombinant haploid progeny is selected individually or<br>
en masse and mated to one another the resulting diploids are<br>
sporulated anew and their progeny spores are subjected to appropriate<br>
selection conditions to identify new recombination events.<br>
With the inventive process a large library of recombined mutated<br>
sequences can be easily generated and variants that have acquired<br>
a desired function can then be identified by using an appropriate<br>
selection or screening system.<br>
In a preferred embodiment of the invention the first diploid S. cerevisiae<br>
cell is generated by simultaneously or sequentially transforming<br>
a diploid S. cerevisiae cell with a DNA molecule comprising the<br>
first recombination cassette and a DNA molecule comprising the<br>
second recombination cassette and optionally allowing the integration<br>
of the two recombination cassettes into allelic positions on natural<br>
chromosomes of the S. cerevisiae genome. The DNA molecules<br>
used can also be for example yeast artificial chromosomes (YAC).<br>
YACs are characterized in that they are linear DNA molecules that<br>
contain all the sequences necessary for stable maintenance in the<br>
yeast cell such as a centromere DNA replication origin and te11<br>
lomeres as well as yeast selectable markers. Upon introduction into<br>
a yeast cell YACs behave similar to natural chromosomes and therefore<br>
can be considered as part of the yeast genome. In the context of<br>
the present invention the term "genome" includes the whole of all<br>
hereditary components present within a cell which are stably maintained<br>
and inherited. In case YACs are used as DMA molecules for<br>
introduction the first and second recombination cassettes into diploid<br>
S. cerevisiae cells it is not necessary to integrate the two recombination<br>
cassettes into allelic positions in natural chromosomes. In the<br>
case in which recombination cassettes are introduced into natural<br>
chromosomes it is possible to use a cloning vehicle for example a<br>
plasmid from which a fragment bearing the recombination cassettes<br>
can be liberated. Preferably the two respective marker sequences of<br>
the two recombination cassettes are flanked by targeting sequences<br>
which are homologous to a defined locus of the S. cerevisiae genome.<br>
Alternatively a DMA molecule can be used which does not<br>
contain a replication origin. In this case the DNA molecules must be<br>
able to integrate into a component of the genome and therefore contain<br>
targeting sequences which are homologous to a defined locus of<br>
the S. cerevisiae genome.<br>
In another preferred embodiment of the invention the first diploid S.<br>
cerevisiae cells are generated by fusing haploid cells bearing in a<br>
locus of their genome the first recombination cassette with S. cerevisiae<br>
haploid cells bearing in an allelic position of their genome the<br>
second recombination cassette.<br>
In still another preferred embodiment of the invention the first diploid<br>
S. cerevisiae cells are generated by mating haploid cells bearing in<br>
a locus of their genome the first recombination cassette with S. cerevisiae<br>
haploid cells of opposite mating type bearing in an allelic position<br>
of their genome the second recombination cassette.<br>
In the context of the present invention the terms "mating" and "fusing"<br>
denote either the purposeful or the random combination of two<br>
haploid cells containing different recombination cassettes. A purposeful<br>
mating or fusing of two haploid cells occurs when two selected<br>
and//or isolated haploid cells of opposite mating type with desired<br>
properties are brought into contact under conditions stimulating<br>
mating and fusing respectively. The two haploid cells can be derived<br>
from the same library of cells which for example contain DMA<br>
sequences to be recombined or already recombined DMA sequences<br>
or from different libraries of cells which for example contain<br>
DNA sequences to be recombined or already recombined DNA<br>
sequences.<br>
A random mating or fusing of two haploid cells can occur when a<br>
plurality of different haploid cells are brought into contact under conditions<br>
stimulating mating and fusing respectively. The plurality of<br>
haploid cells can be derived from the same library of cells which for<br>
example contain DNA sequences to be recombined or already recombined<br>
DNA sequences or from different libraries of cells which<br>
for example contain DNA sequences to be recombined or already<br>
recombined DNA sequences.<br>
The inventive process for generating and detecting recombined DNA<br>
sequences has the advantage that more than two diverging sequences<br>
can be recombined. If for example four diverging DNA sequences<br>
shall be recombined then in the first step of the present<br>
process two different sets of diploid S. cerevisiae cells can be generated.<br>
For example a first set of diploid cells can be generated by<br>
mating or fusing haploid cells comprising a first and a second DNA<br>
sequence to be recombined and a second set of diploid cells can be<br>
generated by mating or fusing haploid cells comprising a third and a<br>
fourth DNA sequence to be recombined. After sporulation of the two<br>
sets of diploid cells haploid cells obtained from the first diploid cell<br>
set that contain recombined DNA sequences due to recombination<br>
between the first and the second DNA sequence are mated with<br>
appropriate haploid cells obtained from the second diploid cell set<br>
that contain recombined DNA sequences due to recombination between<br>
the third and the fourth DNA sequences. The products of this<br>
mating are diploid cells which after sporulation give rise to haploid<br>
cells bearing recombined DNA sequences which comprise regions of<br>
the first DNA sequence the second DNA sequence the third DNA<br>
sequence and the fourth DNA sequence. If for example three diverging<br>
DNA sequences shall be recombined in the first step of the<br>
present process diploid S. cerevisiae cells are generated by for example<br>
mating or fusing haploid cells comprising a first and a second<br>
DNA sequence to be recombined. After sporulation of these diploid<br>
cells the haploid cells thus obtained which contain recombined<br>
DNA sequences due to recombination between the first and the second<br>
DNA sequences can be fused or mated with haploid cells comprising<br>
a third DNA sequence to be recombined. The products of this<br>
mating are diploid cells which after sporulation give rise to haploid<br>
cells bearing recombined DNA sequences which comprise regions of<br>
the first DNA sequence the second DNA sequence and the third<br>
DNA sequence. In this way five six or more diverging DNA sequences<br>
can also be recombined.<br>
In a preferred embodiment haploid S. cerevisiae cells bearing the<br>
first or second recombination cassette are generated by:<br>
a) inserting the first DNA sequence to be recombined between<br>
the first and the second marker sequence located adjacently<br>
on a first cloning vehicle and inserting the second DNA sequence<br>
to be recombined between the third and the fourth<br>
marker sequence located adjacently on a second cloning vehicle<br>
whereby the respective two marker sequences are<br>
flanked by targeting sequences which are homologous to a<br>
defined locus of the S. cerevisiae genome<br>
b) excising from the cloning vehicles obtained in a) the first recombination<br>
cassette and the second recombination cassette<br>
with flanking targeting sequences respectively whereby each<br>
excised fragment comprises the DNA sequence to be recombined<br>
which is flanked by the respective two marker<br>
sequences and by targeting sequences<br>
c) transforming the excised fragments obtained in b) separately<br>
into S. cerevisiae diploid cells whereby the targeting sequences<br>
direct the integration of the cassettes into that locus<br>
to which they are homologous in order to obtain diploid cells<br>
heterozygous for the first cassette or the second cassette<br>
d) inducing separately the sporulation of the heterozygous diploid<br>
cells obtained in c) and<br>
e) isolating haploid cells containing the first cassette flanked by<br>
the first and second marker sequences and separately hap15<br>
loid cells containing the second cassette flanked by the third<br>
and the fourth marker sequences.<br>
In a preferred embodiment of the invention the respective two marker<br>
sequences in the first or second cloning vehicle are flanked by targeting<br>
sequences which are homologous to the BUD31-HCM1 locus<br>
on chromosome III of the S. cerevisiae genome and which direct the<br>
integration of the excised cassettes into that locus.<br>
In a preferred embodiment of the invention the cloning vehicle used<br>
for cloning the recombination cassettes is a plasmid. "Plasmid"<br>
means an extrachromosomal element which can autonomously replicate.<br>
The plasmid is physically unlinked to the genome of the cell<br>
wherein it is contained. Most plasmids are double-stranded circular<br>
DNA molecules. In another embodiment the cloning vehicle is an<br>
YAC.<br>
In particular it is preferred to use as the first cloning vehicle in which<br>
the first recombination cassette is cloned plasmid pMXY9. Plasmid<br>
pMXYS comprises the URA3 marker gene and the CAN1 marker<br>
gene. In this plasmid the two marker genes are adjacently located.<br>
Between the two marker genes are arranged several restriction sites<br>
in particular recognition sites for the restriction enzymes Smal Xbal<br>
Bgl\\ and Pad for inserting a DNA sequence to be recombined. The<br>
two marker sequences are flanked by targeting sequences homologous<br>
to the BUD31-HCM1 locus on chromosome III of the S. cerevisiae<br>
genome.<br>
Furthermore it is preferred to use as the second cloning vehicle in<br>
which the second recombination cassette is cloned plasmid<br>
16<br>
pMXY12. Plasmid pMXY12 comprises the TRP1 marker gene and<br>
the CYH2 marker gene. In this plasmid the two marker genes are<br>
adjacently located. Between the two genes are arranged several restriction<br>
sites in particular recognition sites for the restriction enzymes<br>
Spel Smal and Pad for inserting a DNA sequence to be recombined.<br>
The two marker sequences are flanked] by targeting sequences<br>
homologous to the BUD31-HCM1 locus on chromosome ill<br>
of the S. cerevisiae genome.<br>
In a preferred embodiment of the invention the diploid cells used for<br>
transformation of the excised recombination cassette are auxotrophic<br>
for at least two nutritional factors and resistant to at least two<br>
antibiotics. Preferably the diploid cells are homozygous for the ura3-<br>
1 allele and the trp1-1 allele which renders the cells auxotrophic for<br>
uracil and tryptophan respectively. Furthermore it is preferred that<br>
the diploid cells used for transformation are homozygous for the<br>
can1-100 allele and the cyh2R allele which renders them resistant<br>
to canavanine and cycloheximide respectively.<br>
In particular it is preferred that diploid cells of the S. cerevisiae<br>
strain MXY47 are used for transformation which are homozygous for<br>
the alleles ura3-1 trp1-1 can1-100 and cyh2R and heterozygous for<br>
the msh2::KanMX mutation. When diploid cells of the strain MXY47<br>
are used for the transformation with the excised first or second fragments<br>
bearing recombination cassettes and their flanking targeting<br>
sequences then transformants obtained can be sporulated to yield<br>
haploid wild type or msh2 segregants that bear the respective recombination<br>
cassette.<br>
According to the invention it may be preferred to use S. cerevisiae<br>
cells which have a functional mismatch repair system for the inventive<br>
process. The mismatch repair system belongs to the largest contributors<br>
to avoidance of mutations due to DNA polymerise errors in<br>
replication. Mismatch repair also promotes genetic stability by editing<br>
the fidelity of genetic recombination. It is known that therefore<br>
the mismatch repair machinery has a somewhat inhibitory effect on<br>
recombination between diverged sequence. However in a normal S.<br>
cerevisiae diploid another aspect of mismatch repair termed mismatch<br>
repair-dependent recombination was detected (Borts and<br>
Haber Science 237 (1987) 1459-1465). It is thought that the mismatch<br>
repair of widely spaced mismatches such as in diverged sequences<br>
leads to new double-strand breaks that can in turn stimulate<br>
a second round of (mismatch repair-dependent) recombination.<br>
In certain circumstances in particular when it is known that the two<br>
recombination substrates used have widely spaced base differences<br>
it is therefore useful to employ S. cerevisiae cells with a functional<br>
mismatch repair system for conducting the inventive process.<br>
In another preferred embodiment of the invention S. cerevisiae cells<br>
that are deficient in the mismatch repair system are used. In S. cerevisiae<br>
several genes have been identified whose products share<br>
homology with bacterial mismatch repair proteins including six<br>
homologues of the MutS protein i.e. Msh1 Msh2p MshSp Msh4<br>
Msh5 and MshGp and four homologues of the MutL protein i.e.<br>
Mlhlp Mlh2p MlhSp and Pmsl It is known that in particular the<br>
PMS1 and MSH2 genes set up a barrier to the recombination of diverged<br>
sequences. Therefore in msh2 and pmsl .mutants meiotic<br>
recombination between diverged sequences is increased relative to<br>
the frequency of recombination in wild type cells.<br>
In the context of the present invention the term "deficient in the mismatch<br>
repair system" means that the mismatch repair system (MMR)<br>
of a cell is transiently or permanently impaired. MMR deficiency of a<br>
cell or an organism can be achieved by any strategy that transiently<br>
or permanently impairs the mismatch repair including but not limited<br>
to a mutation of one or more genes involved in mismatch repair<br>
treatment with an agent like UV light which results in a global impairment<br>
of MMR treatment with an agent like 2-aminopurin or a<br>
heteroduplex containing an excessive amount of mismatches to<br>
transiently saturate and inactivate the MMR system and inducible<br>
expression or repression of one or more genes involved in the mismatch<br>
repair for example via regulatable promoters which would<br>
allow for transient inactivation i.e. during meiosis but not during<br>
vegetative growth.<br>
In a preferred embodiment of the invention the mismatch repair deficiency<br>
of the S. cerevisiae cell is due to a mutation of at least one<br>
gene involved in the MMR. In a preferred embodiment the S. cerevisiae<br>
cells are deficient in the MSH2 gene. Preferably diploid cells<br>
are homozygous for the msh2 allele in which the MSH2 coding sequences<br>
are replaced by the KanMX construct.<br>
In the context of the present invention the term "recombfnation cassette"<br>
refers to a DMA sequence comprising at least one recombination<br>
substrate or one DNA sequence to be recombined which is<br>
flanked by at least two different marker sequences. The first and the<br>
second recombination cassette differ in the DNA sequences to be<br>
recombined and in the flanking marker sequences such that any<br>
pair of recombination cassettes comprises two different DMA sequences<br>
to be recombined and at least four different flanking marker<br>
sequences.<br>
In a preferred embodiment of the invention both the first and the<br>
second recombination cassettes are generated by inserting the respective<br>
DMA sequences to be recombined between two marker sequences<br>
that are closely located on a cloning vehicle and which in<br>
turn are surrounded by targeting sequences that are homologous to<br>
a defined locus of the S. cerevisiae genome. The targeting sequences<br>
therefore can direct the integration of an excised fragment<br>
containing a recombination cassette into this defined locus. The insertion<br>
of the DMA to be recombined between the two marker sequences<br>
is preferably effected by genetic engineering methods. In a<br>
preferred embodiment of the invention the two marker sequences in<br>
the cloning vehicle are flanked by targeting sequences which are<br>
homologous to the BUD31-HCM1 locus on chromosome II! of the S.<br>
cerevisiae genome. Therefore the targeting sequences direct the<br>
integration of the excised fragments containing a recombination cassettes<br>
into that locus.<br>
In the context of the present invention the terms "DNA sequences to<br>
be recombined" and "recombination substrate" mean any two DNA<br>
sequences that can be recombined as a result of meiotic recombination<br>
processes whereby recombination between these sequences<br>
can be due to homologous or non-homologous recombination.<br>
Homologous recombination events of several types are characterized<br>
by the base pairing of a damaged DNA strand with a homolo20<br>
gous partner where the extent of interaction can involve hundreds of<br>
nearly perfectly matched base pairs. The term "homology" denotes<br>
the degree of identity existing between the sequence of two nucleic<br>
acid molecules. In contrast illegitimate or non-homologous recombination<br>
is characterized by the pining of ends of DMA that share no<br>
or only a few complementary base pairs. In yeast non-homologous<br>
repair and recombination events occur at significantly lower frequencies<br>
than homologous recombination events.<br>
The first and second DMA sequences to be recombined are diverging<br>
sequences i.e. sequences which are not identical but show a<br>
certain degree of homology. This means that the DNA sequences to<br>
be recombined diverge by at least one nucleotide. Preferably the<br>
DNA sequences to be recombined are sequences that share at least<br>
one or more homologous regions which can be very short. The homologous<br>
regions should comprise at least 5-10 nucleotides preferably<br>
more than 20-30 nucleotides more preferred more than 30-40<br>
nucleotides and most preferred more than 50 nucleotides. In a preferred<br>
embodiment of the invention the first and the second DNA<br>
sequences to be recombined diverge by at least one nucleotide in<br>
particular more than 01 % preferably more than 5 % to more than<br>
50 %. This means that the first and second DNA sequences to be<br>
recombined can also diverge by 55% 60% 65 % or even more.<br>
Recombination substrates or DNA sequences to be recombined can<br>
have a natural or synthetic origin. DNA sequences to be recombined<br>
therefore can be derived from any natural source including viruses<br>
bacteria fungi including S. cerevisiae animals plants and humans.<br>
In a preferred embodiment of the invention the first and the second<br>
DNA sequences to be recombined are derived from organisms other<br>
than S. cerevisiae.<br>
In a preferred embodiment of the i nvention DNA sequences to t&gt;e<br>
recombined are protein-encoding sequences for example sequences<br>
encoding enzymes which can be utilized for the industrial<br>
production of natural and non-natural compounds. Enzymes or those<br>
compounds produced by the help of enzymes can be used for the<br>
production of drugs cosmetics foodstuffs etc. Protein-encoding sequences<br>
can also be sequences which encode proteins that have<br>
therapeutic applications in the fields of human and animal health.<br>
Important classes of medically important proteins include cytokines<br>
and growth factors. The recombination of protein coding sequences<br>
allows for the generation of new mutated sequences which code for<br>
proteins with altered preferably improved functions and/or newly<br>
acquired functions. In this way it is possible for example to achieve<br>
improvements in the thermostability of a protein to change the substrate<br>
specificity of a protein to improve its activity to evolve new<br>
catalytic sites and/or to fuse domains from two different enzymes.<br>
Protein coding DNA sequences to be recombined can include sequences<br>
from different species which code for the same or similar<br>
proteins that have in their natural context similar or identical functions.<br>
Protein coding DNA sequences to be recombined can include<br>
sequences from the same protein o»r enzyme family. Protein coding<br>
sequences to be recombined can also be sequences which code for<br>
proteins with different functions - for example sequences that cod©<br>
for enzymes which catalyse different steps of a given metabolic<br>
pathway. In a preferred embodiment of the invention the first and the<br>
22<br>
second DMA sequences to be recombined are selected from the<br>
group of gene sequences of the Oxa superfamily of B-lactamases.<br>
In another preferred embodiment of the invention DMA sequences to<br>
be recombined are non-coding sequences such as sequences<br>
which for example are involved within their natural cellular context<br>
in the regulation of the expression of a protein-coding sequence.<br>
Examples for non-coding sequences include but are not limited to<br>
promoter sequences sequences containing ribosome binding sites<br>
intron sequences polyadenylation sequences etc. By recombining<br>
such non-coding sequences it is possible to evolve mutated sequences<br>
which in a cellular environment result in an altered regulation<br>
of a cellular process - for example an altered expression of a<br>
gene.<br>
According to the invention a recombination substrate or DMA sequence<br>
to be recombined can of course comprise more than one<br>
protein coding sequence and/or more than one non-coding sequence.<br>
For example a recombination substrate can comprise one<br>
protein coding sequence plus one non-coding sequence or a combination<br>
of different protein coding sequences and different noncoding<br>
sequences. In another embodiment of the invention DNA sequences<br>
to be recombined therefore can consist of one or more<br>
stretches of coding sequences with intervening and/or flanking noncoding<br>
sequences. That means the DNA sequence to be recombined<br>
can be for example a gene sequence with regulatory sequences<br>
at its 5'-terminus and/or an untranslated S'-region or an<br>
mammalian gene sequence with an exon/intron structure. In still another<br>
embodiment of the invention DNA sequences to be recom23<br>
bined can consist of larger continuous stretches that contain more<br>
than a single coding sequence with intervening non-coding sequences<br>
such as those that as may belong to a biosynthetic pathway<br>
or an operon. DNA sequences to be recombined can be sequences<br>
which have already experienced one or more recombination<br>
events for example homologous and/or non-homologous recombination<br>
events.<br>
The recombination substrates can comprise non-mutated wild-type<br>
DNA sequences and/or mutated DNA sequences in a preferred embodiment<br>
therefore it is possible to recombine wild-type sequences<br>
with already existing mutated sequences in order to evolve new mutated<br>
sequences.<br>
In the context of the present invention the term "marker sequences"<br>
refers to unique DNA sequences that are positioned upstream or<br>
downstream of a recombination substrate or an already recombined<br>
DNA sequence in Saccharomyces cerevisiae cells. The presence of<br>
a marker sequence on the same molecule of DNA as the recombination<br>
substrate or already recombined DNA sequence preferably in<br>
combination with another marker sequence positioned on the other<br>
side of the recombination substrate allows that recombination substrate<br>
or already recombined DNA sequence to be recognized and<br>
selected for whether by molecular or genetic methods. Therefore in<br>
one preferred embodiment of the invention there must be one or<br>
more marker sequences upstream of each recombination substrate<br>
and one or more marker sequences downstream of each recombination<br>
substrate such that in a cell heterozygous for two different recombination<br>
substrates there are at least four different marker se24<br>
quences altogether. This arrangement allows for the selection of<br>
crossovers involving recombination substrates. It also allows further<br>
rounds of recombination to be carried out in a iterative fashion. In<br>
another preferred embodiment of the invention more than one<br>
marker can be situated on each side of the recombination substrate.<br>
For example additional markers can be introduced to increase the<br>
stringency of selection.<br>
Marker sequences may comprise protein-encoding or non-coding<br>
DMA sequences. In a preferred embodiment of the invention the protein-<br>
encoding marker sequences are selected from the group consisting<br>
of nutritional markers pigment markers antibiotic resistance<br>
markers antibiotic sensitivity markers and sequences that encode<br>
different subunits of an enzyme which functions only if both or more<br>
subunits are expressed in the same cell. In a further preferred embodiment<br>
of the invention the molecular non-coding marker sequences<br>
include but are not limited to primer recognition sites i.e.<br>
sequences to which PCR primers anneal and which allow an amplification<br>
of recombinants intron/exon boundaries promoter sequences<br>
downstream regulated gene sequences or restriction enzyme<br>
sites.<br>
A "nutritional marker" is a marker sequence that encodes a gene<br>
product that can compensate an auxotrophy of an organism or cell<br>
and thus can confer prototrophy on that auxotrophic organism or<br>
cell. In the context of the present invention the term "auxotrophy"<br>
means that an organism or cell must be grown in a medium containing<br>
an essential nutrient which cannot be synthesized by the<br>
auxotrophic organism itself. The gene product of the nutritional<br>
25<br>
marker gene promotes the synthesis of this essential nutrient missing<br>
in the auxotrophic cell. Therefore upon expression of the nutritional<br>
marker gene it is not necessary to add this essential nutrient to<br>
the medium in which the organism or cell is grown since the organism<br>
or cell has acquired prototrophy.<br>
A "pigment marker" is a marker gene wherein the gene product is<br>
involved in the synthesis of a pigment which upon expression will<br>
stain that cell in which the pigment marker is expressed. A cell without<br>
the pigment marker does not synthesize the pigment and is<br>
therefore not stained. The pigment marker therefore allows a rapid<br>
phenotypical detection of that cell containing the pigment marker.<br>
An "antibiotic resistance marker" is a marker gene wherein the gene<br>
product confers upon expression to a cell in which the expression of<br>
the antibiotic marker gene takes place the ability to grow in the<br>
presence of a given antibiotic at a given concentration whereas a<br>
cell without the antibiotic resistance marker cannot.<br>
An "antibiotic sensitivity marker" is a marker gene wherein the gene<br>
product destroys upon expression the ability of a cell to grow in the<br>
presence of a given antibiotic at a given concentration.<br>
In a preferred embodiment of the invention each of the gene products<br>
of the first and third marker sequences can compensate an<br>
auxotrophy of a S. cerevisiae cell. Preferably the first marker sequence<br>
is URA3 the gene product of which can confer uracil prototrophy<br>
to a uracil auxotrophic S. cerevisiae cell. Preferably the third<br>
marker sequence is TRP1 the gene product of which can confer<br>
tryptophan prototrophy to an tryptophan auxotrophic S. cerevisiae<br>
cell.<br>
In another preferred embodiment of the invention the gene products<br>
of the second and fourth marker sequences confer sensitivity to an<br>
antibiotic to a S. cerevisiae cell which is resistant to that antibiotic.<br>
Preferably the second marker sequence is CAN1 the gene product<br>
of which can confer to a canavanine resistant S. cerevisiae cell sensitivity<br>
to canavanine. Preferably the fourth marker sequence is<br>
CYH2 the gene product of which can confer to a cycloheximide resistant<br>
S. cerevisiae cell sensitivity to cycloheximide.<br>
In another preferred embodiment of the invention the marker sequences<br>
comprise annealing sites for PCR primers. Preferably the<br>
first second third and fourth marker sequences are recognized by<br>
the primers KNS11 KNS28 KNS16 and KNS29.<br>
In a preferred embodiment of the inventive process haploid cells<br>
containing recombination cassettes with either first second third or<br>
fourth recombined DNA sequences can be identified by PCR processes<br>
in order to detect the presence of the respective marker combination.<br>
In another preferred embodiment of the inventive process haploid<br>
cells containing recombination cassettes with either first second<br>
third or fourth recombined DNA sequences are identified by plating<br>
the haploid cells on media that select for the presence on the same<br>
DNA molecule of the respective marker combination. This means<br>
that tiaploid cells containing first recombined DNA sequences are<br>
plated on a medium that selects for the presence of the first and the<br>
27<br>
fourth marker sequences. Haploid cells containing second recombined<br>
DMA sequences are plated on a medium that selects for the<br>
presence of the second and the third marker sequences. Haploid<br>
cells containing third recombined DMA sequences are plated on a<br>
medium that selects for the presence of the first and the second<br>
marker sequences. Haploid cells containing fourth recombined DMA<br>
sequences are plated on a medium that selects for the presence of<br>
the third and the fourth marker sequences.<br>
Another aspect of the present invention relates to a process of generating<br>
novel proteins enzymes pathways and non-coding sequences<br>
with novel or improved functions and properties whereby<br>
known protein-coding sequences or known non-coding sequences<br>
are subjected one or more recombination rounds by using the inventive<br>
process for generating and detecting recombinant DNA sequences<br>
in S. cerevisiae.<br>
Another aspect of the present invention relates to plasmid pMXY9.<br>
Plasmid pMXY9 comprises the URA3 marker gene and the CAN1<br>
marker gene which are located adjacently. Between the two marker<br>
gene a polylinker sequence comprising several restriction sites for<br>
inserting a DNA sequence to be recombined is arranged. The two<br>
markers are flanked by targeting sequences homologous to the<br>
BUD31-HCM1 locus on chromosome III of the S. cerevisiae genome.<br>
The polylinker sequence between the two marker genes comprises<br>
restriction sites for the restriction enzymes Smal Xba\ Bgl\\ and<br>
Pad.<br>
Another aspect of the present invention relates to plasmid pMXY12.<br>
Plasmid pMXY12 comprises the TRP1 marker gene and the CYH2<br>
marker gene. Between the two marker genes a polylinker sequence<br>
comprising several restriction sites for inserting a DMA sequence to<br>
be recombined is arranged. The two markers are flanked by targeting<br>
sequences homologous to the BUD31-HCM1 locus on chromosome<br>
III of the S. cerevisiae genome. The polylinker sequence comprises<br>
restriction sites for the restriction enzymes Spel Smal and<br>
Pacl.<br>
The present invention relates also to the S. cerevisiae strain MXY47<br>
characterized in that diploid cells thereof are homozygous for the<br>
alleles ura3-1 trp1-1 can1-100 and cyh2R and heterozygous for the<br>
msh2::KanMX mutation.<br>
The present invention also relates to the E. coli strain JM101 containing<br>
plasmid pMXY9 and to E. coli strainDHSa containing plasmid<br>
pMXY12.<br>
Plasmids pMXYQ and pMXY12 and the Saccharomyces cerevisiae<br>
strain MXY47 were deposited on the 3rd of January 2005 at the<br>
DSMZ (Deutsche Sammlung fur Mikroorganismen und Zellkulturen<br>
GmbH Mascheroderweg 1b 38124 Braunschweig Germany) under<br>
accession numbers DSM 1701O DSM 17011 and DSM 17026 respectively.<br>
Another aspect of the present invention relates to a kit which can be<br>
used for conducting the inventive process for generating and detecting<br>
recombined DMA sequences in Saccharomyces cerevisiae. In a<br>
first embodiment the kit comprises at least a first container which<br>
contains cells of S. cerevisiae strain MXY47 a second container<br>
which contains cells of E. coli strain JM101 bearing plasmid pMXY9<br>
and a third container containing cells of E. coli strain DH5a bearing<br>
plasmid pMXY12.<br>
In a second embodiment the kit comprises at least a first container<br>
containing cells of S. cerevisiae strain MXY47 a second container<br>
containing DMA of plasmid pMXY9 and a third container containing<br>
DMA of plasmid pMXY12.<br>
29<br>
The present invention is illustrated by the following sequence listing<br>
figures and example.<br>
Figure 1 shows a schematic of the selection system for the selection<br>
of recombinants on defined media. Diploid parental cells heterozygous<br>
for recombination cassettes - here recombination substrate A<br>
flanked by the URA3 and CAN1 genes and recombination substrate<br>
B flanked by the TRP1 and CYH2 genes - are induced to undergo<br>
meiosis. Spores are plated on medium lacking uracil and containing<br>
canavanine (-Ura+Can) and on medium lacking tryptophan and containing<br>
cycloheximide (-Trp+Cyh) to select for recombinant cells 3<br>
and 4 in which a crossover involving the recombination substrates A<br>
and B has taken place as indicated by (+). Parental diploids and<br>
non-recombinant haploids 1 and 2 cannot grow on either of these<br>
media as indicated by (-}. A subsequent round of meiosis may use<br>
recombinants 3 and 4 to construct a new diploid which when<br>
sporulated yields new recombinant cells bearing the same flanking<br>
marker configurations as those shown in cells 1 and 2. Recombinant<br>
spore colonies with these configurations can be selected on medium<br>
lacking uracil and containing cycloheximide (-Ura+Cyh) and on medium<br>
lacking tryptophan and containing canavanine(-Trp+Can) respectively.<br>
Figure 2 shows the plasmids pMXYS and pMXY12 (above) which<br>
are vectors used for the targeting of recombination cassettes to the<br>
BUD31-HCM1 locus on chromosome ill of the yeast genome. Both<br>
plasmids bear sequences homologous to this locus (indicated as 5'<br>
and 3') which flank the URA3 and CAN1 markers (pMXY9) or TRP1<br>
and CYH2 (pMXY12) markers. A short sequence bearing restriction<br>
sites that allow for cloning recombination substrates is located between<br>
each pair of marker sequences. Below integration of recombination<br>
cassettes into the BUD31-HCM1 locus. A pMXY9 derivative<br>
bearing recombination substrate A is digested with Not\ to liberate<br>
the recombination cassette flanked by 5' and 3' targeting sequences<br>
and digestion products are transformed into MXY47 cells. Ura+ derivatives<br>
that contain a correctly targeted insert are identified for<br>
subsequent use in constructing strains heterozygous for recombination<br>
cassettes. Recombination cassettes bearing the TRP1 and<br>
CYH2 markers are similarly constructed in pMXY12 and transformed<br>
into MXY47 followed by selection for tryptophan prototrophy.<br>
Figure 3 shows the frequency of recombination between Oxa genes<br>
as a function of sequence identity in wild type and msh2 strains.<br>
Above the mean ± standard deviation of (n) independent experiments<br>
is provided. Below graphical representation of these data.<br>
The following strains were used: MXY60 MXY62 MXY64 MXY66<br>
MXY99andMXY102.<br>
Figure 4 shows the msh2 hyper-recombination effect. A msh2Nvt ratio<br>
was calculated for each independent experiment (total number =<br>
n) for pairs of strains with the given percent of shared Oxa homology<br>
and for each selection condition and the the mean ± standard deviation<br>
of these summed values are shown. The data are represented<br>
graphically below. The following pairs of strains were used: MXY60<br>
and MXY62 MXY64 and MXY66 MXY99 and MXY102.<br>
Figure 5 shows a PCR analysis of recombination between Oxa sequences<br>
sharing 78% homology. Spore colonies were derived from<br>
31<br>
wild type (MXY99) and msh2 (MXY102) diploids by selection on medium<br>
lacking uracil and containing canavanine or on medium lacking<br>
tryptophan and containing cycloheximide. Colony PCR was performed<br>
on selected spore colonies that exhibited phenotypes consistent<br>
with those expected for crossover recombinants. Above two<br>
reactions were carried out for each wild type and msh2 Ura+CanR<br>
candidate one with a parental-specific primer pair (KNS16 + KNS28<br>
products shown in the first of each pair of lanes for each candidate)<br>
and the other with a recombinant-specific- primer pair (KNS16 +<br>
KNS29 second lane). Below similar reactions were carried out for<br>
each wild type and msh2 Trp+CyhR candidate one with a parentalspecific<br>
primer pair (KNS11 + KNS29 first lane) and the other with a<br>
recornbinant-specific primer pair (KNS11 + KNS28 second lane).<br>
Control reactions were carried out on appropriate genomic DNA<br>
templates containing known configurations of flanking marker sequences<br>
either parental (P) or recombinant (R). (-) no DNA control.<br>
Figure 6 shows the frequencies of recombination for second-round<br>
recombination. Wild type and msh2 haploids obtained after a first<br>
round of recombination with MXY64 and MXCY66 were mated to produce<br>
wild type (MXY81 MXY82 and MXY83) and msh2 (MXY86<br>
MXY87 and MXY88) diploids with mosaic Oxa7-0xa11 recombination<br>
cassettes. Wild type (MXY90) and msh2 (MXY92) diploids homozygous<br>
for the Oxa11 recombination substrate were also constructed<br>
from recombinant progeny of MXY60 and MXY62. All diploids<br>
were sporulated and spores were plated on media to select for<br>
Ura+CanR and Trp+CyhR recombinants.<br>
32<br>
The sequence listing comprises the following sequences:<br>
SEQ ID No. 1 and 2 show the sequences of the primers MSH2UP<br>
and MSH2DN respectively for the amplification of MSH2.<br>
SEQ ID No. 3 to SEQ ID No. 6 show the sequences of the primers<br>
MSH2A1 MSH2A2 MSH2A3 and MSH2A4 respectively which are<br>
MSW2-specific analytical primers.<br>
SEQ ID No. 7 and SEQ ID No. 8 show the sequences of the primers<br>
K2KANMX and K3KANMX respectively which are KanMX-specific<br>
analytical primers.<br>
SEQ ID No. 9 and SEQ ID No. 10 show the sequences of the primers<br>
LEU2UP and LEU2DN respectively which are used for the amplification<br>
of LEU2.<br>
SEQ ID No. 11 and SEQ ID No. 12 show the sequences of the primers<br>
HIS3UP and HIS3DN respectively which are used for the amplification<br>
of HIS3.<br>
SEQ ID No. 13 and SEQ ID No. 14 show the sequences of the primers<br>
KNS1 and KNS2 respectively which are used for the amplification<br>
of the 3' targeting sequence.<br>
SEQ ID No. 15 to SEQ ID No. 17 show the sequences of the primers<br>
KNS3 KNS4 and KNS6 respectively which are used for the amplification<br>
of a 5' targeting sequence.<br>
SEQ ID No. 18 and SEQ ID No. 19 show the sequences of the primers<br>
KNS7 and KNS8 respectively which are used for the amplification<br>
of Oxa7.<br>
SEQ ID No. 20 and SEQ ID No. 21 show the sequences of the primers<br>
KNS9 and KNS10 respectively which are used for the amplification<br>
of Oxa11.<br>
SEQ ID No. 22 shows the sequence of the primer KNS12 which is a<br>
BUD31 downstream analytical primer.<br>
SEQ ID No. 23 shows the sequence of the primer KNS13 which is a<br>
BUD31 upstream analytical primer.<br>
SEQ ID No. 24 shows the sequence of the primer KNS14 which is a<br>
TRPf-specific analytical primer.<br>
SEQ ID No. 25 shows the sequence of the primer KNS15 which is a<br>
l/R&gt;43-specific analytical primer.<br>
SEQ ID No. 26 and SEQ ID No. 27 show the sequences of the primers<br>
KNS17 and KNS18 respectively which are used for the amplification<br>
of CYH2.<br>
SEQ ID No. 28 shows the sequence of the primer KNS30 which is a<br>
TRP1 -specific forward primer used as sequencing primer.<br>
SEQ ID No. 29 shows the sequence of the primer KNS31 which is a<br>
CyA/Vf-specific reverse primer used as sequencing primer.<br>
34<br>
SEQ ID No. 30 shows the sequence of the primer KNS33 which is a<br>
CYH2-specific reverse primer used as sequencing primer.<br>
SEQ ID No. 31 and SEQ ID No. 32 show the sequences of the primers<br>
KNS36 and KNS37 respectively which are used for the amplification<br>
of OxaS.<br>
SEQ ID No. 33 shows the sequence of the primer KNS38 which is a<br>
L/R/43-specific forward primer used as sequencing primer.<br>
Example - Generation of mosaic genes in Saccharomyces cerevisiae<br>
mismatch repair mutants<br>
1. Materials and methods<br>
1.1 Media<br>
Standard rich medium YPD (Bio101) was used for routine growth<br>
and synthetic dropout media (Bio101) were used to monitor genetic<br>
markers and for selection of recombinants. For sporulation cells were<br>
precultured overnight in SPS (50 mM potassium phthalate pH<br>
5.0 0.5% yeast extract (Difco) 1% Bactopeptone (Difco) 0.17%<br>
yeast nitrogen base 1% potassium acetate 0.5% ammonium sulfate)<br>
plus required nutritional supplements washed resuspended in<br>
1 % potassium acetate plus supplements and incubated with shaking<br>
for two days. All manipulations were carried out at 30°C. For tetrad<br>
analysis asci were digested with Helix pomatia B-glucuronidase<br>
(Sigma) and dissected using a Nikon Eclipse E400 microscope fitted<br>
with a TDM400 micromanipulator (Micro Video Instruments Inc.).<br>
Other genetic methods were conducted as described by Ausubel et<br>
al. Current Protocols in Molecular Biology (1998) John Wiley and<br>
Sons Inc. New York. All yeast transformations were performed using<br>
the LiAc method according to Agatep et al. Technical Tips Online<br>
(http://tto.trends.com).<br>
1.2. Yeast strains<br>
All yeast strains used or created in this study are listed in Table 1<br>
and Table 2. All yeast strains are isogenic derivatives of the readily<br>
sporulating W303 background. The diploid MXY47 which serves as<br>
a host for transformation with recombination cassettes was constructed<br>
by transformation and genetic crosses as follows. The haploid<br>
D184-1B (a gift of S. Gangloff CEA France) was transformed<br>
with a LEU2 fragment (obtained by preparatory PCR of the W303<br>
strain U474 with the primer pair LEU2UP/LEU2DN which are listed in<br>
the sequence listing to yield the Leu+ haploid MXY13. The haploid<br>
D184-1C (a gift of S. Gangloff) was transformed with a HI S3 fragment<br>
(obtained by preparatory PCR of ORD4369-25D with the<br>
primer pair HIS3UP/HIS3DN) to yield the His+ haploid MXY25. The<br>
haploids MXY18 and MXY22 are recessive cycloheximide-resistant<br>
(cyh2R) derivatives of D184-1B and D184-1C respectively selected<br>
on 10 jj.g/ml cycloheximide; the presence of mutations mapping to<br>
the CYH2 locus that confer cycloheximide resistance was confirmed<br>
by sequencing (two different nucleotide alterations resulting in a<br>
change of glutamine 38 to lysine) and segregation analysis. MX7Y18<br>
and MXY25 were crossed to obtain the diploid MXY29; MXY13 and<br>
MXY22 were crossed to obtain the diploid MXY33. The haploid segregants<br>
MXY29-6D and MXY33-8C were crossed to obtain MXY38<br>
which is heterozygous for the Ieu2-3112 and his3-1115 markers<br>
and homozygous for the cyh2Fi mutation. MXY38 was transformed<br>
with the msh2::KanMX cassette ampli-fied by PCR from RBT348 (a<br>
gift of R. Sorts University of Leicester) with the primers MSH2UP<br>
and MSH2DN to yield MXY47. Transformants were selected on 200<br>
jig/ml G418 (Invitrogen) and confirmed by colony PCR (see below)<br>
with the primers MSH2A1 MSH2A2 MSH2A3 and MSH2A4 and by<br>
tetrad analysis i.e. analysis of the four spores to confirm marker<br>
segregation.<br>
1.3 Plasmid construction<br>
The bacteria! strains XL1-B!ue MRF(AmcrA)183(mcrCB-hsdSMRmrr)<br>
173 endA1 supE44 thi-1 recA1 gyrA96 relA1 lac /F proAB<br>
IacPZkM15 Tn10 (Tef)]) and JM110 (rpsL [Stf] thr leu thi-1 lacY<br>
galK galT era tonA tsx dam dcm supE44 k[lac-proAB] IF' traD36<br>
proAB laclqZA.M15]) were used as hosts for cloning. Standard methods<br>
were used for plasmid construction (Ausubel et al.). All plasmids<br>
used or created in this study are listed in Table 3. Restriction enzymes<br>
T4 DMA ligase and other enzymes used in cloning were purchased<br>
from New England BioLabs. DNA fragments and plasmids<br>
were purified using kits supplied by Qiagen and Macherey-Nagel.<br>
Upstream ("51 target") sequences corresponding to the BUD31 locus<br>
were amplified by preparatory PCR from W303 genomic DNA with<br>
the primer pair KNS3/KNS4 and cloned as a Kpn1/X/7o1 fragment<br>
into Kp/?1/X/7o1 -digested pKSII(+) (Stratagene) to create pMXY1;<br>
downstream ("31 target") targeting sequences were similarly amplified<br>
with the primer pair KNS1/KNS2 and cloned as a XbaMNott<br>
fragment into X6a1/A/ott -digested pKSII(+) (Stratagene) to create<br>
pMXY2. The TRP1 marker was excised from pJH53 (a gift of R.<br>
Sorts) as a Bglll/EcoRl fragment and ligated to Ba/nHI/EcoRIdigested<br>
pMXY1 to create pMXY3 and the URA3 marker was excised<br>
from X/70l/H//?DIII-digested pRED316 (a gift of R. Sorts) and<br>
ligated to Xhol/HinDIll-digested pMXY1 to create pMXY4. The CAM<br>
marker was isolated from pRED316 as a Sma1 fragment and ligated<br>
to Hpa1-digested pMXY2 to create pMXYS. The 5' targeting sequences<br>
in pMXYS and pMXY4 were replaced with sequences reamplified<br>
from genomic DNA with the primer pair KNS4/KNS6 and<br>
ligated as Kp/?1/X/?o1 fragments into the respective KpnVXholdigested<br>
plasmids to produce pMXY7 and pMXYG. This step was<br>
undertaken to correct the absence from the primer KNS3 of restriction<br>
sites required in later phases of cloning. The Kpnl-Smal ment of pMXYS containing the 5' target and the URA3 marker were<br>
I [gated to Kpnl/Smal-digested pMXYS to produce the URA3-CAN1<br>
recombination cassette vector pMXY9. The Kpnl/Spel fragment of<br>
pMXYT containing the 5' target and TRP1 marker were ligated to<br>
Kpnl/Spel-digested pMXY2 to produce pMXY11. Finally the CYH2<br>
marker was amplified by preparatory PCR from W303 genomia DNA<br>
with the primer pair KNS17/KNS18 digested with SamHI and Pvul<br>
and ligated to Bg/ll/Pad-digested pMXY11 to create the TRP1-CYH2<br>
recombination cassette vector pMXY12. All plasmid constructs were<br>
introduced into bacterial hosts by electroporation and verified by restriction<br>
analysis and pMXYS and pMXY12 were further verified by<br>
sequencing of all cloning junctions.<br>
B-lactamase recombination substrates were amplified by preparatory<br>
PCR from host plasmids (provided by W. Schoenfeld) using the<br>
primer pairs KNS36/KNS37 for Oxa5 (accession X58272)<br>
KNS7/KNS8 for Oxa7 (accession X75562) and KNS9/KNS1 O for<br>
Oxa11 (accession Z22590). Oxa7 and Oxa11 PCR products were<br>
digested with Pad and ligated to S/wal/Pacl-digested pMXY9 to create<br>
pMXY13 and pMXY14 respectively and the Oxa11 PCR product<br>
was also digested with Spel and Pad and ligated to Spelf/^acldigested<br>
pMXY12 to create pMXY22. The OxaS PCR products were<br>
digested with SamHI and Pad and iigated to Sg/ll/Pacl-digested<br>
pMXY9 to create pMXY24. All constructs were verified by restriction<br>
analysis.<br>
1.4 Recombinant selection and characterization<br>
For first round recombination plasmids bearing recombination cassettes<br>
were digested with A/of1 and total digestion products were<br>
used to transform MXY47. Uracil (for pMXY9 derivatives) or tryptophan<br>
(for pMXY12 derivatives) prototrophs were selected and targeting<br>
of one of the two chromosomal copies of the BUD31-HCM1<br>
locus by the introduced construct was confirmed by colony PCR us43<br>
ing the primers KNS12/KNS13/KNS15 for URA3-CAN1 derivatives<br>
and the primers KNS12/KNS13/KNS14 for TRP1-CYH2 derivatives<br>
which allow fragments from the intact and from the disrupted BUD31-<br>
HCM1 loci to be amplified. Transformed heterozygotes were<br>
speculated and tetrad analysis was carried out to identify wild type or<br>
msh2 haploids bearing recombination cassettes. Appropriate haploids<br>
of opposite mating type were patched on YPD plates allowed<br>
to grow overnight mixed together on the same YPD plate and allowed<br>
to mate overnight. The mating plate was replica plated to -<br>
Ura-Trp medium to select for diploids which were inoculated the following<br>
day in bulk into SPS plus supplements and cultured overnight.<br>
The preculture was spun down and washed and the cells<br>
were resuspended in 1 % K acetate plus supplements and incubated<br>
for two days.<br>
Sporulated cells were harvested quantified and in some cases dissected<br>
to confirm appropriate segregation of ail markers. Asci were<br>
digested with zymolyase-20T (ICN Biomedicals) to liberate spores<br>
the spore suspension was sonicated (Branson Model 250 Digital<br>
Sonifier) and appropriate dilutions were plated on YPD to determine<br>
cell viability on uracil dropout media containing 60 ug/ml canavanine<br>
(Sigma) to select Ura+CanR recombinants and on tryptophan<br>
dropout media containing 3 ug/ml cycloheximide (Sigma) to<br>
select Trp+CyhR recombinants. Spore colonies arising on each medium<br>
were counted and subjected to phenotypic and molecular tests<br>
to determine whether they represented true recombinants. For<br>
phenotypic analysis a representative number of candidate<br>
recombinants was restreaked to the same medium used for selection<br>
and then replica plated to -Ura -Trp cycloheximide (10 ^g/ml)<br>
canavanine (60 pg/ml) and mating type tester plates.<br>
Spores were also plated on -<br>
Ura-Trp media to determine the frequency of diploids for each spore<br>
preparation which in all cases was lower than 4% of total viable<br>
cells. For molecular analysis total genomic DMA was subjected to<br>
analytical PCR (see below) using appropriate primer pairs that specifically<br>
amplify parental or recombinant fragments. The frequencies<br>
of recombination for a given selection are expressed as the frequency<br>
of viable cells on a given selection medium corrected for the<br>
presence of non-recombinants exhibiting a false positive phenotype.<br>
In most cases such false positives arose by mutational inactivation<br>
of the CAN1 or CYH2 marker as suggested by analytical PCR.<br>
For second round recombination appropriate recombinants derived<br>
from the first round of recombination were mated and Ura+Trp+ diploids<br>
were selected. The same sporulation procedure as for first<br>
round recombination was followed except that spores were plated<br>
on YPD on uracil dropout media containing cycloheximide to select<br>
Ura+CyhR recombinants and on tryptophan dropout media containing<br>
canavanine to select Trp+CanR recombinants. Candidate recombinants<br>
were similarly subjected to phenotypic and molecular<br>
analysis.<br>
1.5 Molecular methods<br>
Genomic DNA used as a template for preparatory or analytical PCR<br>
was prepared from overnight YPD cultures by a standard miniprep<br>
procedure according to Ausubel et al. Preparatory PCR of fragments<br>
used in cloning or for sequencing was performed with Oxa plasmid<br>
DNA (approximately 50 pg) or yeast genomic DNA (approximately<br>
0.5 fig) as a template in 50 pi reactions containing 2.5 U Herculase<br>
polymerase (Stratagene) 1x Herculase reaction buffer 0.2 mM each<br>
dNTP and 100 ng each primer. Amplification was carried out as follows:<br>
94° C 2 min; 30 cycles of 94°C 10s 55°C 30 s 72°C 30s; 68°C<br>
10 min. A modified colony PCR procedure was employed to confirm<br>
integration of recombination cassettes at the BUD31 locus<br>
(http://www.fhcrc.ora/labs/hahn/methods/mol bio meth/pcr veast c<br>
olonv.html). with the following amplification conditions: 95°C 5 min;<br>
35 cycles of 95°C 1 min 55°C 1 min 68°C 1 min; 72°C 10 min. Analytical<br>
PCR to characterize Oxa inserts was carried out in 100 \i\ reaction<br>
volumes containing approximately 0.5 ng genomic DNA prepared<br>
from candidate recombinants and control strains 1.5 U Taq<br>
polymerase (Roche) 1 x reaction buffer 0.2 mM each dNTP and<br>
100 ng each primer with the same amplification conditions as for<br>
colony PCR except that extension was carried out at 68°C for 2 min.<br>
All amplification reactions were performed with a Mastercycler gradient<br>
5331 (Eppendorf).<br>
For sequence analysis of recombinant Oxa inserts preparatory PCR<br>
was carried out with the primer pairs KNS16/KNS29 (for Ura+CanR<br>
recombinants) or KNS11/KNS28 (for Trp+CyhR) followed by purification<br>
with the Qiaquick PCR kit (Qiagen). PCR products were sequenced<br>
by Genome Express (Meylan FR) with the primers KNS30<br>
KNS31 KNS33 or KNS38 as appropriate. Recombinant sequences<br>
were aligned and analyzed using Clone Manager software (Sci Ed<br>
Central). Oligonucleotides used in PCR and sequencing (Table 3)<br>
were purchased from Proligo France.<br>
2. Results<br>
2.1 Development of a yeast meiotic homeologous recombination<br>
system<br>
A strategy that makes use of the yeast Saccharomyces cerevisiae to<br>
promote in vivo recombination between diverged DNA sequences<br>
has been developed. Critical features of the strategy include the use<br>
of meiotic cells in which high levels of genome-wide recombination<br>
take place and inactivation of the mismatch repair (MMR) system<br>
which normally restricts recombination between diverged sequences.<br>
Sequences to be recombined i.e. the recombination substrates are<br>
introduced into one of two vectors that also bears flanking marker<br>
sequences so as to create recombination cassettes. The recombination<br>
cassettes are introduced into the yeast genome at a locus on<br>
chromosome III (the BUD31-HCM1 interval) which is in a region<br>
known to be recombinationally active in meiosis. Diploids heterozygous<br>
for recombination cassettes are sporulated and spores are<br>
plated on media that select for cells with specific configurations of<br>
flanking markers thereby allowing for the selection of recombinants<br>
in which a crossover involving recombination substrates has taken<br>
place (Figure 1).<br>
Two general recombination cassette vectors were constructed<br>
pMXY9 and pMXY12 which contain the URA3 and CAN1 and the<br>
TRP1 and CYH2 markers respectively flanking restriction sites that<br>
can be used for the introduction of recombination substrates (Figure<br>
2). The URA3 marker confers uracil prototrophy and the CAN1<br>
marker confers canavanine sensitivity. In the absence of this marker<br>
cells are resistant to the drug. The TRP1 marker confers tryptophan<br>
prototrophy and the CYH2 marker confers cycloheximide sensitivity.<br>
In the absence of this marker cells are resistant to the drug. Each of<br>
the two recombination cassettes is in turn flanked by sequences that<br>
allow targeting of the entire insert to the BUD31-HCM1 locus by<br>
transformation of competent cells (Figure 2). A strain that serves as<br>
a primary host for transformation MXY47 was also constructed (Table<br>
2). This diploid is heterozygous for the msh2::KanMX mutation<br>
and is phenotypically wild type with respect to MMR. It is also homozygous<br>
for the ura3-1 trp1-1 can1-100 and cyh2R markers which<br>
allows the presence of recombination cassette markers to be monitored<br>
and heterozygous for the his3-1115 and Ieu2-3112 markers.<br>
MXY47 is transformed with fragments bearing recombination cassettes<br>
primary transformants are selected as Ura+ or Trp+ prototrophs<br>
(for MXY9 and MXY12 derivatives respectively) and targeting<br>
Is confirmed by analytic PCR using primers that recognize sequences<br>
within and external to the introduced construct. Primary<br>
transformants are sporulated and tetrads are dissected and replica<br>
plated to identify wild type or msh2 segregants that bear the recombination<br>
cassette. Suitable haploids are mated to one another to<br>
generate MSH2/MSH2 (wild type) and msh2/msh2 diploids heterozygous<br>
for recombination substrates. In a first round of meiotic<br>
recombination to generate recombinants these diploids are<br>
sporulated and free spores are plated on media lacking uracil and<br>
containing canavanine to select for recombinants with the URA3-<br>
CYH2 configuration of flanking markers (Ura+CanR spore colonies)<br>
or on media lacking tryptophan and containing cycloheximide to select<br>
for the TRP1-CAN1 configuration (Trp+CyhR spore colonies).<br>
Parental diploids and non-recombinant haploid progeny cannot grow<br>
on these media. The frequency of spore colonies arising on selective<br>
media is determined candidate recombinant spore colonies are<br>
characterized phenotypically by replica plating on test media and<br>
molecularly by PCR with appropriate primer pairs and a sample of<br>
confirmed recombinants is selected for sequencing.<br>
The strategy is iterative in that cells bearing recombinant inserts can<br>
be identified and subjected to further rounds of meiotic recombination<br>
to increase diversity. In a second round Ura+CanR and<br>
Trp+CyhR haploids are mated and the speculation and selection<br>
process is repeated except that new recombinants are selected on<br>
media lacking uracil and containing cycloheximide to select for recombinants<br>
with the URA3-CAN1 configuration of flanking markers<br>
(Ura+CyhR spore colonies) or on media lacking tryptophan and containing<br>
canavanine to select for the TRP1-CYH2 configuration<br>
(Trp+CanR spore colonies). The strategy detailed here can also be<br>
modified to include additional markers to increase the stringency of<br>
selection. Furthermore recombinants can also be directly selected<br>
by PCR using primers specific to flanking sequences.<br>
2.2 Phenotvpic selection for recombination between Oxa gene<br>
pairs of varying sequence divergence<br>
Genes belonging to the Oxa superfamily of beta-lactamases were<br>
chosen as substrates to test the feasibility of the system for the selection<br>
of recombinants. Recombination between the following Oxa<br>
pairs was assessed in the wild type and msh2 backgrounds: Oxa11-<br>
Oxa11 which share 100% homology throughout the 800 bp ORF;<br>
Oxa7-0xa11 95%; Oxa5-Oxa11 78%. Diploids generated by<br>
crosses between appropriate haploids were induced to enter meiosis.<br>
Spores were prepared from meiotic cultures and serial dilutions<br>
were plated on YPD to determine cell viability and on medium lacking<br>
uracil and containing canavanine (-Ura+Can) and on medium<br>
lacking tryptophan and containing cycloheximide (-Trp+Cyh) to select<br>
for recombinants.<br>
2.3 Frequencies of recombination between Oxa genes of varying<br>
sequence homoloav<br>
The data shown in Figure 3 demonstrate that in the wild type background<br>
increased sequence heterology has a strong inhibitory effect<br>
on crossover recombination and that this effect is relieved but not<br>
abolished by the msh2 mutation. In general the msh2 mutation<br>
causes an increase in the frequency of recombination of about one<br>
order of magnitude above that observed for wild type strains at the<br>
two levels of divergence tested. However inactivation of MSH2<br>
alone does not fully compensate for the inhibition of recombination<br>
between recombination substrates with higher degrees of heterology.<br>
For example the frequencies of recombination for a msh2 strain<br>
with Oxa inserts sharing 78% homology (MXY102) are at least 10-<br>
fold (Ura+CanR) and 25-fold (Trp+CyhR) below those found for a<br>
wild type strain with Oxa inserts of 100% homology (MXY60) indicating<br>
that factors other than MSH2-dependent mismatch repair prevent<br>
crossover recombination between more diverged sequences. It<br>
is noteworthy that the appearance of msh2 recombinants at the 78%<br>
divergence level at frequencies of roughly 2 x 10"4 indicates that<br>
recombination may be achieved between even more divergent substrates.<br>
2.4 The msh2 hyper-recombination effect<br>
The effect of msh2 on homologous and homeologous recombination<br>
was quantified by first calculating the ratio of msh2 to wild type<br>
recombinants for a given percent of homology for a given selection<br>
for each experiment and then calculating the means and standard<br>
deviations of the ensemble of ratios thus determined. The data are<br>
shown in in Figure 4. The presence of the msh2 mutation increases<br>
the frequency of homeologous recombination for sequences of 95%<br>
and 78% homology and there is a less pronounced but still quantifiable<br>
enhancement of recombination between 100% identical sequences.<br>
Furthermore the extent of the msh2 enhancement of homeologous<br>
recombination differs for the two selections: for strains<br>
with homeologous Oxa inserts inactivation of MSH2 increases the<br>
frequency of Trp+CyhR recombinants to a greater extent than it increases<br>
the frequency of Ura+CanR recombinants. In principle the<br>
frequencies of both types of recombinants (Ura+CanR and<br>
Trp+CyhR) should be equivalent but these numbers indicate that<br>
there are biases in the system that are provoked or enhanced by the<br>
msh2 mutation in conjunction with variations in the extent of sequence<br>
divergence. Experiments to test the relative influences of<br>
inserts and flanking marker sequences on the types of recombinants<br>
obtained indicate that this bias is a property of the flanking markers<br>
but the influence of the recombination substrates in directing the outcomes<br>
of meiotic recombination events cannot yet be accounted for<br>
(data not shown).<br>
2.5 PCR analysis of selected recombinants<br>
An example of PCR analysis as applied to Ura+CanR and<br>
Trp+CyhR spore colonies derived from wild type and msh2 diploids<br>
containing Oxa genes of 22% divergence (MXY99 and MXY102<br>
respectively) is shown in Figure 5. For each strain ten spore colonies<br>
that exhibited each recombinant phenotype were analyzed.<br>
Extracts from each colony were used as templates for amplification<br>
with primer pairs that specifically amplify parental molecules and<br>
with primer pairs that specifically amplify recombinant molecules.<br>
In every case(s) only the predicted recombinant insert was amplified<br>
indicating that the selected spore colonies contained sequences<br>
produced by recombination between the parental Oxa recombination<br>
substrates. These results also demonstrate that recombinant<br>
molecules can be directly recovered from sporulated cultures<br>
even without the imposition of a genetic selection step. Here the<br>
primer recognition sites located in the URA3 CAN1 TRP1 and<br>
CYH2 genes represent molecular marker sequences that flank each<br>
recombination substrate.<br>
2.6 Sequence analysis of first round rneiotic homeologous recombinants:<br>
5% and 22% divergence<br>
Oxa7-Oxa11 meiotic recombinants derived from wild type and msh2<br>
diploids (MXY64 and MXY66 respectively) which contain recombination<br>
substrates sharing 95% homology that satisfied phenotypic<br>
and molecular (PCR) tests were subjected to sequence analysis.<br>
Recombinant fragments were amplified with primers specific to flanking<br>
markers and sequenced using primers close to the translational<br>
start and stop sites. Overall 55 recombinant sequences derived from<br>
haploid progeny of Oxa7-Oxa 11 diploids were analyzed: 14<br>
Ura+CanR and 13 Trp+CyhR recombinants from MXY64 and 14<br>
Ura+CanR and 14 Trp+CyhR recombinants from MXY66. The sequenced<br>
sample size allows several observations to be made. 1) For<br>
both wild type and msh2 recombinants the position at which the<br>
crossover took place ranged throughout the full coding region with<br>
no apparent preference for a specific interval. Crossovers that occurred<br>
in the 5' region were as likely as those in the 3' region. Also<br>
for a given strain there was no apparent difference in the distributions<br>
of crossover sites for spore colonies obtained by -Ura+Can or<br>
by -Trp+Cyh selection. 2) The length of uninterrupted homology in<br>
the crossover interval was also unimportant: crossovers were detected<br>
between two closely spaced polymorphisms (positions 543-<br>
552 for MXY66 Trp+CyhR #7 #8 and #13 where position 1 represents<br>
the adenosine residue of the ATG translational start site) as<br>
well as between the two most widely spaced polymorphisms (positions<br>
163-265 eg MXY66 Trp+CyhR #15). 3) The recombinant Oxa<br>
inserts isolated from both wild type and msh2 backgrounds contained<br>
full-length recombinant sequences potentially capable of encoding<br>
new functional Oxa proteins. That is all crossovers occurred<br>
in such a manner as to preserve an intact ORF without a net insertion<br>
or deletion of nucleotides in the crossover interval or in any<br>
other interval. 4) Although the structures of most recombinarit sequences<br>
are consistent with a simple crossover between the two<br>
Oxa sequences in local regions of homology several recombinants<br>
isolated in the msh2 background exhibited greater complexity. Sequences<br>
derived from four recombinants (MXY66 Ura+CanR #16<br>
and #31 and MXY66 Trp+CyhR #5 and #9) exhibited a higher degree<br>
of mosaicism as if they were produced by more than one<br>
crossover event. Indeed analysis of two of these recombinants was<br>
complicated because inspection of electropherograms revealed the<br>
presence of two overlapping peaks at multiple sites within the sequenced<br>
region each site corresponding to an Oxa 7-Oxa11 polymorphism.<br>
This observation indicates that the population of molecules<br>
that was sequenced was heterogeneous for which the most<br>
likely explanation is the presence of unrepaired or partially repaired<br>
heteroduplex DMA present in msh2 recombinant spores. This<br>
interpretation is consistent with the known increased frequency of<br>
post-meiotic segregation (PMS) caused by the msh2 mutation. In<br>
these two cases MXY66 Ura+CanR #16 and #31 one or more<br>
repaired sites was flanked by stretches of unrepaired heteroduplex<br>
consistent with the unmasking of a short patch mismatch repair<br>
activity in the msh2 background as suggested by CoTc Gluck and<br>
Fabre (EMBO J. 19:3408). Several other cases of PMS unassociated<br>
with short-patch mismatch repair were also observed for msh2<br>
recombinant sequences indicating that this alternative mismatch<br>
repair system may not be highly efficient at correcting mismatches in<br>
heteroduplex DNA. Judging from sequence electropherograms the<br>
extent of uncorrected heteroduplex varied from a short region of<br>
about 50 nt to a region almost covering the entire ORF. No evidence<br>
for PMS or short-patch mismatch repair was found for wild type<br>
recombinant sequences. Overall these findings suggest that the<br>
extent of diversity created is greater in msh2 meiosis than in wild<br>
type meiosis.<br>
Meiotic recombinants were also derived from wild type and msh2<br>
diploids that contain recombination substrates sharing 78% homology<br>
(MXY99 and MXY102 respectively). In total 24 recombinant<br>
sequences derived from recombinant progeny of Oxa5-Oxa 11 diploids<br>
were analyzed: five Ura+CanR and three Trp+CyhR recombinants<br>
from MXY99 and nine Ura+CanR and seven Trp+CyhR recombinants<br>
from MXY102. Inspection of these sequences suggests<br>
several trends. 1) Recombinant Oxa sequences obtained In both<br>
wild type and msh2 strains by selection on -Trp+CyhR exhibited<br>
crossovers at different positions throughout the ORF with perhaps a<br>
overall shared homology. In contrast recombinants obtained in both<br>
wild type and msh2 strains by selection on -Ura+CanR exhibited a<br>
pronounced bias in the positions of crossovers: in 3 of 5 wild type<br>
and 8 of 9 msh2 sequences crossovers occurred within the last 80<br>
nt of the region of shared homology ie the last 10% of the ORF. 2)<br>
The intervals of absolute homology in which crossovers were identified<br>
ranged from 11 to 20 nt for the -Trp+Cyh selection indicating a<br>
preference for these relatively larger regions of sequence identity. In<br>
contrast crossover intervals were shorter for the -Ura+Can selection<br>
ranging from 3 to 17 nt (13/14 of these involved intervals 13 nt<br>
and shorter). 3) As for recombination involving sequences sharing<br>
95% homology the new sequences obtained also consisted of intact<br>
ORFs and potentially encode novel Oxa proteins. 4) No cases of<br>
PMS as judged by inspection of electropherograms were found for<br>
wild type recombinants but very short patches of unrepaired heteroduplex<br>
were found for a few msh2 recombinants including 3 of the 7<br>
Trp+CyhR recombinants. These regions included at most 67 nt<br>
shorter than some of the tracts observed for Oxa7-Oxa11 recombinants.<br>
In sum these observations indicate that recombinant sequences<br>
can be selected from input recombination substrates varying<br>
by at least 22% and that these sequences encode novel proteins.<br>
2.7 Sequence analysis of Oxa7-0xa11 second-round recombi-<br>
' nants<br>
The ability of the yeast system to increase sequence diversity in an<br>
iterative manner was tested by. constructing diploids from Oxa7-<br>
Oxa11 recombinant haploids generated in a first round of meiosis<br>
and subjecting these new diploids to a second round of meiosis.<br>
Among the sequenced Ura+CanR and Trp+CyhR progeny of MXY64<br>
and MXY66 pairs of appropriate recombinants with crossovers in<br>
the same interval were selected to construct new diploids in which<br>
the overall level of sequence homology was again 95%. Three wild<br>
type (MXY81 MXY82 and MXY83) and three msh2 (MXY86 MXY87<br>
and MXY88) diploids were created. Control wild type and msh2 diploids<br>
containing only Oxa11 sequence inserts were also constructed<br>
from appropriate recombinant progeny of MXY60 and MXY62 yielding<br>
MXY90 and MXY92. These diploids were sporulated and spores<br>
were plated on medium lacking uracil and containing cycloheximide<br>
(-Ura+Cyh) and on medium lacking tryptophan and containing canavanine<br>
(-Trp+Can) to select for second-round recombinants. As<br>
shown in Figure 6 the frequencies of -Ura+CyhR and Trp+CanR<br>
spore colonies observed for all of these strains is consistent with the<br>
anti-recombination effect of the MSH2 gene. Both types of colonies<br>
were found among progeny of the wild type homozygote MXY90 at<br>
frequencies above 10"3 whereas these frequencies were decreased<br>
5- to 10-fold among progeny of wild type diploids with Oxa insert<br>
heterology (MXY81 MXY82 and MXY83). Inactivation of the MSH2<br>
gene in diploids with diverged Oxa inserts (MXY 86 MXY 87 and<br>
MXY 88) led to a 2 to 6-fold increase in the frequency of Ura+CyhR<br>
and Trp+CanR spore colonies similar to the levels seen for a msh2<br>
diploid bearing identical Oxa inserts (MXY92). Although the media<br>
used differ from those used for selection of first-round recombinants<br>
the frequencies at which wild type and msh2 second-round recombinants<br>
were selected are comparable to those for first-round recombinants.<br>
Ura+CyhR and Trp+CanR spore colonies in both the wild type<br>
(MXY81 and MXY83) and msh2 (MXY86) backgrounds were selected<br>
for sequencing. In all 14 wild type and 7 msh2 Oxa inserts<br>
were sequenced. In most cases a crossover occurred in a novel interval<br>
during second round recombination again without apparent<br>
bias with respect to position or interval size: crossovers involving<br>
different intervals were found throughout the Oxa ORF and they occurred<br>
in intervals as large as 101 nt and as small as 5 nt. In one<br>
case (a MXY83 Trp+CyhR haploid) a second round crossover occurred<br>
in the first round crossover interval thereby restoring a full<br>
Oxa11 sequence. Recombinants recovered from msh2 diploids were<br>
more diverse than those recovered from wild type strains. For the<br>
msh2 diploid MXY86 several spore colonies exhibiting extensive<br>
PMS and sequence mosaicism were observed consistent with the<br>
formation of long tracts of heteroduplex in the recombinational intermediate.<br>
Furthermore some mismatches were repaired in the heteroduplex<br>
tract again consistent with a short-patch mismatch repair<br>
activity. In sum second-round recombination in the msh2 background<br>
is as efficient as first-round recombination both qualitatively<br>
with respect to generating sequence diversity (eg. crossover interval<br>
distribution and incidence of PMS) and quantitatively with respect<br>
to increasing the overall frequency of homeologous (5% divergence)<br>
recombination.<br><br><br><br><br><br>
WE CLAIM<br>
1.	A process for generating and detecting recombinant DNA sequences in Saccharomyces<br>
cerevisiae comprising the steps of:<br>
a)	inserting a first DNA sequence to be recombined between first and second marker sequences located adjacently on a first cloning vehicle;<br>
b)	excising from the first cloning vehicles obtained in a) a first fragments bearing the first recombination cassette, whereby the first recombination cassette comprises the first DNA sequence flanked by the first and second marker sequences, and the first recombination cassette in turn is flanked by targeting sequences,<br>
c)	inserting a second DNA sequence to be recombined between third and fourth marker sequences located adjacently on a second cloning vehicle;<br>
d)	excising from the second cloning vehicles obtained in c) a second fragments bearing the second recombination cassette, whereby the second recombination cassette comprises the second DNA sequence flanked by the third and fourth marker sequences, and the second recombination cassette in turn is flanked by targeting sequences,<br>
e)	transforming the first and second fragments bearing the recombination cassettes with flanking targeting sequences obtained in b) and d) separately into S. cerevisiae haploid cells, whereby the targeting sequences direct the integration of the cassettes into that locus to which they are homologous, in order to obtain a first haploid cell comprising the first cassette and a second haploid cell comprising the second cassette;<br>
f)	fusing the first haploid cell bearing in a locus of its genome the first recombination cassette with the second haploid cell bearing in an allelic position the second recombination cassette in order to produce a heterozygous diploid cell;<br>
g)	inducing the sporulation of the heterozygous diploid cell obtained in f) and<br>
h) isolating haploid cells containing recombination cassettes in which first recombined DNA sequences are flanked by at least the first and fourth marker sequences, and haploid cells containing recombination cassettes in which second recombined DNA sequences are flanked by at least the second and the third marker sequences.<br>
2.	The process as claimed in claim 1, wherein the step after transformation optionally comprises:<br><br>
i)   mating the first haploid cell bearing in a locus of its genome the first recombination<br>
cassette with the second haploid cell bearing in an allelic position the second recombination<br>
cassette in order to produce a heterozygous diploid cell;<br>
ii) inducing the sporulation of the heterozygous diploid cell obtained i) and<br>
iii) isolating haploid cells containing recombination cassettes in which first recombined DNA<br>
sequences are flanked by at least the first and fourth marker sequences, and haploid cells<br>
containing recombination cassettes in which second recombined DNA sequences are flanked<br>
by at least the second and the third marker sequences.<br>
3.	The process as claimed in claim 1, wherein the step after transformation excising optionally<br>
comprises :<br>
i) transforming the first and second fragments bearing the recombination cassettes with flanking targeting sequences obtained in b) and d) into S. cerevisiae diploid cells, whereby the targeting sequences direct the integration of the cassettes into that locus to which they are homologous, in order to obtain diploid cells heterozygous for the first cassette, or the second cassette ;<br>
ii)inducing the sporulation of the heterozygous diploid cells obtained in c) and iii)isolating haploid cells containing the first cassette and expressing the first and fourth marker sequences and separately haploid cells containing the second cassette and expressing the second and the third marker sequences.<br>
4.	The process as claimed in any one of claims 1 to 3, wherein the first cloning vehicle is plasmid pMXY9 and the second cloning vehicle is plasmid pMXY12.<br>
5.	The process as claimed in claim 3, wherein the diploid S. cerevisiae cells used for transformation are auxotrophic for at least two nutritional factors.<br>
6.	The process as claimed in claim 5, wherein the diploid cells are homozygous for the ura3-l allele and the trpl-1 allele, which render them autotrophic for uracil and tryptophan, respectively.<br><br>
7.	The process as claimed in any one of claims 3 to 6, wherein the diploid cells used for transformation are resistant to at least two antibiotics.<br>
8.	The process according to claim 7, wherein the diploid cells are homozygous for the canl-100 allele and the cyh2R allele, which render them resistant to canavanine and cycloheximide, respectively.<br>
9.	The process as claimed in any one of claims 3 to 8, wherein* diploid cells of the S. cerevisiae strain MXY47 are used for transformation, which are homozygous for the alleles ura3-l, trpl-1, canl-100 and cyh2R and heterozygous for the msh2: :KanMXmutation.<br>
10.	The process as claimed in any one of claims 3 to 9, wherein the S. cerevisiae cells have a functional mismatch repair system.<br>
11.	The process as claimed in any one of claims 3 to 10, wherein the first and the second recombination cassettes are integrated in the BUD31-HCM1 locus on chromosome III of the S. cerevisiae genome.<br>
12.	The process as claimed in any one of claims 3 to 9 and 11, wherein the first and the second DNA sequences to be recombined diverge by at least 1 nucleotide.<br>
13.	The process as claimed in claim 1, wherein the first and the second DNA sequences to be recombined are derived from organisms other than and including S. cerevisiae.<br>
14.	The process as claimed in claim 1, wherein the first and the second DNA sequences to be recombined comprise one or more non-coding sequences and/or one or more protein-coding sequences.<br>
15.	The process as claimed in claim 1, wherein the marker sequences are selected from the group consisting of nutritional markers, pigment markers, antibiotic resistance markers, antibiotic sensitivity markers, primer recognition sites, intron/exon boundaries, sequences encoding a<br><br>
particular subunit of an enzyme, promoter sequences, downstream regulated gene sequences and restriction enzyme sites.<br>
16.	The process as claimed in claim 1, wherein the first and third marker sequences are nutritional markers, the gene products of which can compensate an auxotrophy of a S. cerevisiae cell.<br>
17.	The process as claimed in claim 1, wherein the first marker sequence is URA3, the gene product of which can confer uracil prototrophy to an uracil auxotrophic S. cerevisiae cell.<br>
18.	The process as claimed in claim 1, wherein the third marker sequence is TRP1, the gene product of which can confer tryptophan prototrophy to an tryptophan auxotrophic S. cerevisiae cell.<br>
19.	The process as claimed in claim 1, wherein the second and fourth marker sequences are antibiotic sensitivity markers, the gene products of which can confer sensitivity to an antibiotic to a S. cerevisiae cell which is resistant to that antibiotic.<br>
20.	The process as claimed in claim 1, wherein the second marker sequence is CAN1, the gene product of which can confer sensitivity to canavanine to a canavanine-resistant S. cerevisiae cell.<br>
21.	The process as claimed in claim 1, wherein the fourth marker sequence is CYH2, the gene product of which can confer sensitivity to cycloheximide to a cycloheximide-resistant S. cerevisiae cell.<br>
22.	The process as claimed in claim 1, wherein haploid cells containing recombination cassettes with either first, second, third or fourth recombined DNA sequences are identified by PCR processes in order to detect the presence of the respective marker combination.<br><br>
23.	The process as claimed in claim 1, wherein haploid cells containing recombination cassettes with either first, second, third or fourth recombined DNA sequences are identified by plating the haploid cells on media that select for the molecular linkage on the same DNA molecule of the respective marker combination.<br>
24.	The process as claimed in claim 1, wherein haploid cells containing first recombined DNA sequences are plated on a medium that selects for molecular linkage on the same DNA molecule of the first and the fourth marker sequences.<br>
25.	The process as claimed in claim 1, wherein haploid cells containing second recombined DNA sequences are plated on a medium that selects for molecular linkage on the same DNA molecule of the second and the third marker sequences.<br>
26.	The process as claimed in claim 1, wherein haploid cells containing third recombined DNA sequences are plated on a medium that selects for molecular linkage on the same DNA molecule of the first and the second marker sequences.<br>
27.	The process as claimed in claim 1, wherein haploid cells containing fourth recombined DNA sequences are plated on a medium that selects for molecular linkage on the same DNA molecule of the third and the fourth marker sequences.<br>
28.	Plasmid pMXY9 used in claim 1, deposited as DSM 17010, comprising adjacently the URA3 marker gene and the CAN1 marker gene, whereby the two marker sequences flank a polylinker sequence for inserting a DNA sequence to be recombined and whereby the two markers are flanked by targeting sequences homologous to the BUD31-HCM1 locus on chromosome III of the S. cerevisiae genome.<br>
29.	Plasmid pMXY9 as claimed in claim 28, wherein the polylinker sequence comprises restriction sites for the restriction enzymes Smal, Xbal, Pad and Bg111.<br><br>
30.	Plasmid pMXY12 used in claim 1, deposited as DSM 17011, comprising adjacently the TRPI marker gene and the CYH2 marker gene, whereby the two marker sequences flank a polylinker sequence for inserting a DNA sequence to be recombined and whereby the two markers are flanked by targeting sequences homologous to the BUD31-HCM1 locus on chromosome III of the S. cerevisiae genome.<br>
31.	Plasmid pMXY12 as claimed in claim 30, wherein the polylinker sequence comprises restriction sites for the restriction enzymes Smal, Spel and Pacl.<br><br>
32.	S. cerevisiae strain MXY47 as used in claim 1, deposited as DSM 17026, wherein diploid cells thereof are homozygous for the alleles ura3-l, trpl-1, canl-100 and cyh2R and heterozygous for the msh2::KanMXmutation.<br>
33.	E. coli strain JM101, containing plasmid pMXY9 as claimed in claim 28.<br>
34.	E. coli strain DH5a, containing plasmid pMXY12 as claimed in claim 30.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUFic3RyYWN0LSgxMS0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Abstract-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUNsYWltcy0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Claims-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUNsYWltcy0oMzEtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Claims-(31-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzEtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Correspondence Others-(31-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycygxMS0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Correspondence-Others(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Correspondence-Others-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDExLTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Description (Complete)-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LURyYXdpbmdzLSgxMS0wNi0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Drawings-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUZvcm0tMS0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Form-1-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUZvcm0tMi0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Form-2-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUZvcm0tMy0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Form-3-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LUdQQS0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-GPA-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LU90aGVycy1Eb2N1bWVudHMtKDExLTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Others-Documents-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LVBDVC0yMzctKDExLTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-PCT-237-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">4057-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LVBDVC0zNzMtKDExLTA2LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-PCT-373-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Petition-137-(11-06-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDA1Ny1ERUxOUC0yMDA2LVBldGl0aW9uLTEzOC0oMTEtMDYtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">4057-DELNP-2006-Petition-138-(11-06-2009).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="250959-child-resistant-closure-having-a-non-child-resistant-mode-of-operation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="250961-a-composite-patterning-device-for-generating-a-pattern-on-a-sbstrate-surface.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>250960</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4057/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>07/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>17-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>14-Jul-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MIXIS FRANCE S.A.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>166, BOULEVARD DU MONTPARNASSE, F-75014 PARIS, FRANCE.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KATHLEEN SMITH</td>
											<td>1, AVE DE LA PORTE BRANCION, F-75015 PARIS, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RHONA BORTS</td>
											<td>RODHILL BARN, MAIN STREET, TILTON ON THE HILL, LEICESTER LE7 9LN, ENGLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/81</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/000841</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-01-28</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04360008.9</td>
									<td>2004-01-30</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/250960-process-for-generating-and-detecting-recombinant-dna-sequences-in-saccharomyces-cerevisiae by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:09:06 GMT -->
</html>
