<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251089-soluble-analyte-detection-and-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251089:SOLUBLE ANALYTE DETECTION AND AMPLIFICATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">SOLUBLE ANALYTE DETECTION AND AMPLIFICATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a method of detection of a compound of interest that is present at low level in a sample. In particular, the present invention relates to a method of detection of a compound of interest in solution by a nucleic acid-labelled binding construct separation of the unbound nucleic acid-labelled binding construct.and the XXX of the bound nucleic acid-labelled binding construct in the solution phase.The present invention particularly XXX to be used in XXX with a nucleic acid amplification reaction for detecting the presence or absence of the nucleic acid portion of binding construct in a sample indicating the presence or absence of the compound of interest.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>WO 2005/021731	PCT/US2004/028267<br>
1<br>
SOLUBLE ANALYTE DETECTION AND AMPLIFICATION<br>
FIELD OF INVENTION<br>
	The present invention relates generally to the field of compound detection, and<br>
particularly to methods for compound detection and a kit for compound detection<br>
BACKGROUND OF THE INVENTION<br>
	The detection of ummune responses to pathological organisms or the detection of<br>
pathogen-related proteins or other artigens in the serum of patients has benefited greatly from<br>
immunoassay development over the last 15 years. In one form of immunoassay, monoclonal<br>
or polyclonal antibodies that recognize the immunoglobulins of another species are used.<br>
Those reagents, known as anti-species antibodies are labelled, typically with a fluorochrome<br>
or enzyme, and used to detect antigen binding by immunoglobulins found in immune serum.<br>
In another form of immunoassay, known as the sandwich assay, antibodies directed against<br>
pathogen proteins are used to capture antigen from, for example, patients' serum or<br>
cerebrospinal fluid (CSF), that is then detected by the binding of another labelled antibody<br>
directed against the same antigen. All of these assays, however, are limited by the sensitivity<br>
of detection of the bound immunoglobulin and require fairly large concentrations, on a molar<br>
basis, of labelled reagent<br>
	More recent assays utilize nucleic acid amplification methods, such as the polymerase<br>
chain reaction (PCR) for DNA amplification to detect very low levels of nucleic acid.<br>
Nucleic acid amplification methods can allow the detection of pathological agents in serum<br>
or in the environment at levels well below that of immune assay detection. Such techniques,<br>
however, are often very sensitive to contamination from the environment and require prior<br>
knowledge of the nucleic acid sequence of interest in order to identify a portion of the nucleic<br>
acid and amplify it for detection. Nucleic acid amplification requires that there be nucleic<br>
acid present to amplify and detect and is, therefore, of no or limited utility when the<br>
compound of interest is a protein, carbohydrate, or other non-nucleic acid molecule. Further<br>
improvements in the effectiveness and sensitivity of compound detection in a sample are<br>
desirable, and the present invention addresses the existing problems and pvovides related<br>
benefits<br><br>
WO 2005/021731	PCT/US2004/028267<br>
2<br>
SUMMARY OF THE INVENTION<br>
	Throughout this application various publications are referenced The disclosures of<br>
these publications are hereby incorporated by reference, in their entirely, in this application.<br>
Citations of these documents are not intended as an admission that any of them are pertinent<br>
prior art. All statements as to the date or representation as to the contents of these documents<br>
is based on the information available to the applicant and does not constitute any admission<br>
as to the correctness of the dates or contents of these documents.<br>
	The present invention relates to a method of detection of a compound of interest that<br>
can be present at low levels in a sample. In particular, the present invention relates to a<br>
method of detection of a compound of interest in solution by a nucleic acid-labelled binding<br>
construct that binds the compound of interest, separation of the unbound nucleic acid-labelled<br>
binding construct, and the detection of the bound nucleic acid-labelled binding construct in<br>
the solution phase. The present invention is particularly adaptable to be used in conjunction<br>
with a nucleic acid amplification method for detecting the presence or absence of the nucleic<br>
acid portion of the binding construct in a sample, thus indicating the presence or absence of<br>
the compound of interest<br>
	The present invention recognizes that compound detection methods can be made more<br>
sensitive and thereby more effective by separation from the solution any excess or unbound<br>
binding construct that is not bound to the compound of interest, thus increasing the<br>
proportion of true 'positive" signal to false "positive" signal. The present invention further<br>
provides a versatile detection system that does not require solid-phase capture or detection of<br>
the compound of interest. Where the compound of interest includes a non-nucleic acid<br>
molecule such as a peptide or protein a method of the present invention further provides the<br>
advantage of not requiring prior knowledge of the nucleic acid sequence encoding the peptide<br>
or protein of interest in order to identify it and to use DNA amplification for its detection.<br>
	A first aspect of the present invention is a method for detecting a compound of<br>
interest in a sample, including the use of a binding construct. The binding construct includes<br>
a recognition portion which recognizes and binds the compound of interest and a nucleic<br>
acid portion. When the binding construct is mixed with the sample, the recognition portion<br>
binds with the compound of interest and forms construct-compound complexes. The present<br>
invention also includes surfaces bearing one or more accessible binding targets capable of<br>
binding to the recognition portion of the binding construct. When the surfaces are contacted<br>
with the mixture of the sample and binding construct, the accessible binding targets of the<br><br>
WO 2005/021731	PCT/US2004/028267<br>
3<br>
surfaces bind with any excess or unbound binding construct to form cortstruct-surface<br>
complexes. After sufficient incubation, the construct-surface complexes and any unbound or<br>
excess surfaces are separated from the mixture leaving behind the construct-compound<br>
complexes in solution. After separation, the solution is analysed in order to detect the<br>
presence or absence of the nucleic acid portion of the binding construct, wherein the presence<br>
of the nucleic acid portion of the binding construct indicates the presence of the compound of<br>
interest in the sample.<br>
	A second aspect of the present invention is a method for increasing the sensitivity of<br>
solution-phase detection of a compound of interest in a sample. The method includes<br>
providing a sample suspected of containing the compound of interest; providing a binding<br>
construct including a recognition portion capable of binding the compound of interest, and a<br>
nucleic acid portion and contacting the sample with the binding construct for a period of<br>
time sufficient to permit the recognition portion to bind any compound of interest in the<br>
sample, thus forming construct-compound complexes in solution. The method further<br>
includes providing one or more surfaces that beam one or more accessible binding target<br>
capable of binding to the recognition portion, and contacting the surface with the solution for<br>
a period of time sufficient for the accessibte binding target to bind the recognition portion of<br>
any binding construct not bound to the compound of interest, thereby forming construct-<br>
surface complexes. The construct-surface complexes are separated from the solution, leaving<br>
the construct-compound complexes in the solution. The presence or absence of the nucleic<br>
acid portion of the binding construct is detected in the solution. The separation of the<br>
construct -surface complexes from the solution results in a separation of substantially all<br>
binding constructs not bound to a compound of interest and in an increased sensitivity of<br>
detection of the compound of interest and the presence of the nucleic acid portion of the<br>
binding construct indicates the presence of the compound of interest in the sample.<br>
	A third aspect of the present invention is a kit for detecting a compound of interest<br>
including a binding construct comprising a recognition portion which recognizes and binds<br>
the compound of interest, and a nucleic acid portion. The kit of the present invention also<br>
includes one or more surfaces bearing one or more accessible binding targets capable of<br>
binding to the recognition portion of the binding construct. The kit of the present invention<br>
can also optionally include a nucleic acid amplification primer pair, wherein eacb primer of<br>
the primer pair hybridizes to its complementary sequence at the 3' end of a target nucleic acid<br>
sequence of the nucleic acid portion of the binding construct<br><br>
WO 2005/021731	PCT/US2004/028267<br>
4<br>
BRIEF DESCRIPTION OF THE DRAWINGS0<br>
	FlGURE 1 depicts a plurality of binding constructs, each including a recognition<br>
portion and a nucleic acid portion (see Example 1)<br>
	FIGURE 2 depicts a plurality of binding constructs, some of which have recognized<br>
and bound a compound of interest to their recognition portion to form a corcstruct-compound<br>
complex (see Example 1)<br>
	FIGURE 3 depicts the surfaces bearing accessible binding targets, which are<br>
recognized by and are bound to the recognition portion of the binding constructs forming<br>
construct-surface complexes (see Example 1)<br>
	FIGURE 4 depicts separation of the construct-surface complexes from the construct-<br>
compound complexes (see Example 1)<br>
	FIGURE 5 depicts results of Example 11; The Mopep2 particles were capable of<br>
binding to and separating monoclonal antibody 12D5 (12D.5 Mab) horsendish peroxidase<br>
(HRP0) conjugates from solution, thus preventing the 12D.5 Mab HRPO conjugate from<br>
binding to Mopep2-coated wells in a dose-dependent manner. Particles coated with bovine<br>
serum albumin (BSA) were unable to inhibit 12D.5 Mab HRPO conjugate from binding to<br>
Mopep2-coated wells. OD. optical density<br>
	FIGURE 6 depicts the results obtained in Example IV, using a method of the<br>
invention to detect a compound of interest wherein the nucleic acid portion of the binding<br>
construct is amplified by PCR. This experiment demonstrated the ability of the binding<br>
construct Fab-DNA (Mab 12D.5/pUC19 construct) to bind to the compound of interest, the<br>
free antigen (bacterial recombinant fragment of OMPE or rOMPE), thus forming construct-<br>
compound complexes in solution. Surfaces (magnetic particles) bearing accessible binding<br>
targets (Mopep2 peptides) bound any Fab-DNA not bound to the compound of interest, and a<br>
magnet was used to separate the surfaces, leaving the construct-compound complexes (Fab-<br>
DNA bound to antigen) in solution with the nucleic acid portion (pUCl9) of the binding<br>
construct available for nucleic acid amplification. Polymerise chain reaction amplification<br>
resulted in a detectable 2,6 kilobase DNA fragment in the following samples: 790 picogram,<br>
395 picogram, 198 picogram, 98 picogram, and 12 picogram. Very fine 2.6 kilobase bands<br>
were also observed in samples of 49 picograrn and 24 picogram.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
5<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
INTRODUCTION<br>
	The present invention recognizes that compound detection methods can be made more<br>
sensitive and thereby more effective by increasing the proportion of true "positive" signal to<br>
false "positive signal" The present invention further recognizes the versatility of a detection<br>
system that does not require solid-phase detection of the compound of interest, and further<br>
recognizes the desirability of benefiting from signal amplification methods (such as nucleic<br>
acid amplification) even in the detection of a compound of interest that does not include a<br>
nucleic acid.<br>
	As a non-limiting introduction to the breadth of the present invention, the present<br>
invention includes several general and useful aspects, including:<br>
	1)	A method for detecting a compound of interest in a sample, including the steps<br>
		of providing a binding construct including a recognition portion which<br>
		recognizes and binds the compound of interest, and a nucleic acid portion;<br>
		mixing the binding construct with the sample to form construct-compound<br>
		complexes providing one or more surface, wherein the surface bears one or<br>
		more accessible binding targets capable of recognising and binding to the<br>
		recognition portion of the binding construct introducing the surface to the<br>
		mixture of the binding construct and the sample in order for the surface to<br>
		form construct-surface complexes with any unbound binding constructs;<br>
		separating the construct-surface complexes from the mixture leaving behind<br>
		the construc-compound complexes; detecting the presence or absence of the<br>
		nucleic acid portion of the binding construct, wherein the presence of the<br>
		nucleic acid portion of the binding construct indicates the presence of the<br>
		compound of interest at the sample.<br>
	2)	A method for increasing the sensitivity of solution-phase detection of a<br>
		compound of interest, including the steps of providing a sample suspected of<br>
		containing the compound of interest providing a binding construct that<br>
		includes a recognition portion capable of binding the compound of interest,<br>
		and a nucleic acid portion contacting the sample with the binding construct<br>
		for a period of time sufficient to permit the recognition portion to bind any<br><br>
WO 2005/021731	PCT/US2004/028267<br>
6<br>
		compound of interest in the sample, thus forming construct-compound<br>
		complexes in solution providing one or more surfaces that bears one or more<br>
		accessible binding target capable of binding to the recognition portion;<br>
		contacting the surface with the solution for a period of time sufficient for the<br>
		accesible binding target to bind the recognition portion of any binding<br>
		construct not bound to the compound of interest thereby forming construct-<br>
		surface complexes: separating the construet-surface complexes from the<br>
		solution, leaving the construct-compound complexes in the solution; and<br>
		detecting the presence or absence of the nucteic acid portion of the binding<br>
		construct in the solution, wherein the separation of the construct-surface<br>
		complexes from the solution results in a separation of substantially all binding<br>
		constructs not bound to a compound of interest and in an increased sensitivity<br>
		of detection of the compound of interest, and wherein the presence of the<br>
		nucleic acid portion of the binding construct indicates the presence of the<br>
		compound of interest in the sample<br>
	3)	A kit for detecting a compound of interest comprising a binding construct<br>
		comprising a recognition portion which recognizes and binds the compound of<br>
		interest, and a nucleic acid portion; and one or more surfaces, wherein the<br>
		surfaces bear one or more accessible binding targets known to be capable of<br>
		binding to the recognition portion of the binding construct. The kit includes,<br>
		optionally, a nucleic acid primer pair, wherein the wherein each primer of the<br>
		primer pair hybridizes to its complementary sequence at the 3' end of a target<br>
		nucleic acid sequence of the nucleic arid portion of the binding construct.<br>
	The present invention relates to a sensitive method for the detection of a compound of<br>
interest present in a sample, such as a biological fluid, a biological extract, or an<br>
environmental sample, by means of a nucleic acid-labelled binding construct which is<br>
capable of recognizing and binding the compound of interest. This compound detection<br>
method is called soluble analyte detection and amplification. The present invention provides<br>
a method for detection of a compound of interest that is particularly adaptable for use with<br>
samples wherein the compound of interest is present in the sample in low levels.<br>
"Sensitivity" can be defined as the proportion of true positives detected by a system designed<br>
to discriminate between two categories, known conventionally as positive and negative. The<br><br>
WO 2005/021731	PCT/US2004/028267<br>
7<br>
present invention provides enhanced detection sensitivity because it is possible to provide<br>
even excess amounts of the binding construct, and then separate from the sample any<br>
unbound (that is to say, not bound to a compound of interest) or excess binding construct<br>
(detection of which would lead is a false "positive" signal), thus leaving in solution only the<br>
binding construct that is bound to a compound of interest in a construct-compound complex<br>
available for defection as a true "positive" signal. Thus, detection sensitivity is proportional<br>
to the efficiency of separation of unbound or excess binding construct from the solution. The<br>
present invention is also generally more versatile because detection occurs in the solution<br>
phase and is not restricted to solid phase detection. Furthermore, only one binding entity (the<br>
binding construct) is required to recognize and bind the compound of interest, thus avoiding<br>
the problems associated with methods using two binding entities, for example, the problems<br>
of steric hindrance or potential loss of sensitivity from conformational changes induced by<br>
the binding of a first antibody, such as may occur for example, in a two-antibody sandwich<br>
assay.<br>
Further objectives and advantages of the present invention will become apparent as<br>
the description proceeds and when taken in conjunction with the accompanying drawings. To<br>
gain a full appreciation of the scope of the present invention, it will be further recognized that<br>
various aspects of the present invention can be combined to make desirable embodiments of<br>
the invention.<br>
Unless defined otherwise, all technical and scientific terms used herein have the same<br>
meaning as commonly understood by one of ordinary skill in the art to which this invention<br>
belongs. Generally the nomenclature used herein and the manufacture or laboratory<br>
procedures described below are well known and commonly employed in the art. The<br>
technical terms used herein have their ordinary meaning in the art that they are used, as<br>
exemplified by a variety of technical dictionaries. Where a term is provided in the singular,<br>
the inventor also contemplates the plural of that term. The nomenclature used herein and the<br>
procedure described below are those well known and commonly employed in the art. Where<br>
there are discrepancies in terms and definitions used in references that are incorporated by<br>
reference, the, terms used in this application shall have the definitions given herein. Other<br>
technical terms used herein have their ordinary meaning in the art that they are used, as<br>
exemplified by a variety of technical dictionaries (For example Chambers Dictionary of<br>
Science and Technology, Peter M. B. Walker (editor). Chambers Harrap Publishers, Ltd.,<br>
Edinburgh, UK, 1999, 1325 pp.). The inventors do not intend to be limited to a mechanism<br>
or mode of action. Reference thereto is provided for illustrative purposes only.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
8<br>
1.	METHOD FOR DETECTING A COMPOUND OF INTEREST IN A SAMPLE<br>
	The first method of the present invention includes a sensitive method for detection of<br>
a compound of interest in a sample that is capable of detecting the presence of very low<br>
quantities of the compound of interest in the sample. The first method of the present<br>
invention involves the recognition of a compound of interest by a nucleic acid-labelled<br>
binding construct, separation of the unbound nucleic acid-labelled binding construct by<br>
means of one or more surfaces bearing one or more binding targets, and detecting in solution<br>
the presence or absence of the nucleic acid portion of the binding construct wherein the<br>
presence of the nucleic acid portion of the binding construct indicates the presence of the<br>
compound of interest in the sample. The sample suspected of containing the compound of<br>
interest may be, for example, of entirely natural origin of entirely non-natural origin (such as<br>
of synthetic origin), or a combination of natural and non-natural origins. The sample may<br>
include whole cells, tissues,, organs, biological fluids, extracts, or environmental samples.<br>
	The first method of the present invention includes use of a binding construct. The<br>
binding construct are mixed with the sample, allowing the recognition portion of the binding<br>
construct to bind to the compound of interest to form construct-compound complexes in<br>
solution.<br>
	The binding construct includes a recognition portion that can recognize and bind to<br>
the compound of interest, and a nucleic acid portion. The recognition portion can include<br>
virtually any molecule or combination of molecules capable of recognizing and binding the<br>
compound of interest. Such recognition portions can include, without limitation. peptides,<br>
polypeptides, antibodies. Fab fragments, nucleic acids, nucleic acid mimics, cetl surface<br>
antigens, carbohydrates, or combination thereof .In one embodiment, the recognition portion<br>
includes an antibody (natural, modified, or recombinant) or an antibody fragment (such as an<br>
Fab fragment or single-chain antibody variable region fragment). In other embodiments the<br>
recognition portion can be an antigen that binds an antibody, an aptamer that binds a target<br>
such as a peptide or small molecule, of a receptor that binds a ligand. ln one preferred<br>
embodiment, the recognition portion binds monovalently to the compound of interest. In<br>
another preferred embodiment, the recognition portion binds multivalently, for example<br>
bivalently and optionally bispecifically, to the compound of interest<br>
	The nucleic acid portion of the binding construct can include any nucleic acid or<br>
nucleic acid mimic that is capable of being detected. The nucleic acid that is used as the<br>
nucleic acid portion of the binding construct can include any type of nucleic acid, for<br>
example DNA or RNA, or a nucleic acid mimic (such as, but not limited to a peptide nucleic<br><br>
WO 2005/021731	PCT/US2004/028267<br>
9<br>
acid), or a combination thereof. The nucleic acid of the invention can be single-stranded or<br>
double-stranded. In one preferred embodiment, the binding construct of the present invention<br>
includes a nucleic acid portion, wherein the sequence of the nucleic acid portion does not<br>
include a sequence that is expected to be found in the sample, and can thereby be reasonably<br>
expected to be less easily contaminated by nucleic acid sequences in the sample, but is,<br>
however, designed so as to be easily detectable. The nucleic acid portion is preferably<br>
sufficiently long enough to be easily detectable by the detection method chosen.<br>
	The recognition portion can be attached to the nucleic acid portion by any method,<br>
covalently or non-covalently directly or indirectly, and it will depend on the nature of a<br>
given recognition portion. Non-covalem attachment methods include but are not limited to<br>
physical adsorption, electrostatic forces, ionic interactions, hydrogen bonding, hydrophilic-<br>
hydrophobic interaction, van der Waals forces, and magnetic forces. Where desired, for<br>
example, when increased flexibility is needed, a recognition portion may be indirectly affixed<br>
to the nucleic acid portion using a spacer arm. Preferably, the recognition portion is attached<br>
to the nucleic acid portion via covalent linkage, or via a high affinity non-covalent interaction<br>
such as that of biotin and avidin.<br>
	The first method of the present invention includes the use of one or more surfaces that<br>
bears one or more accessible binding targets that are known to be recognized by and bind to<br>
the recognition portion of the binding construct. Such surfaces can be any particle capable of<br>
being separated from a liquid sample, or such surfaces may be non-particulate surfaces such<br>
as planar or non-planar surfaces, or a combination thereof. The surfaces can be optionally<br>
enclosed, for example, in a chamber. The binding targets can include any target capable of<br>
being recognized by and binding to the recognition portion of the binding construct, for<br>
example, peptide mimotopes or epitope mimetics for antibody variable regions, whole or<br>
partial antigen, nucleic acids, or sugar moietics. The accessible binding target can be<br>
attached to the surface by any method, and it will depend on the nature of a given surface and<br>
binding target<br>
	When the surfaces are introduced to the mixture of the sample and binding construct,<br>
after sufficient incubation, the accessible binding targets of the surfaces bind any unbound<br>
recognition portions of the binding construct forming construct-surface complexes. The<br>
construct-surface and any excess unbound surfaces are then separated from the construct-<br>
compound complexes in solution in the sample, by methods suitable to the type of surface<br>
used, for example, a magnet to separate magnetic particles, decanting from a planar surface,<br>
or separation by pressure or vacuum, cenrifugation or filtration. Alternatively, any binding<br><br>
WO 2005/021731	PCT/US2004/028267<br>
10<br>
construct not bound to a compound of interest in a construct-compound complex can be<br>
separated from solution by methods such as, but not limited to precipitation, "salting-out",<br>
size-exclusion or filtration, extraction, or phase-separation.<br>
	After the separation step, by detecting the presence or absence of the nucleic acid<br>
portion of the binding construct,the presence or absence of the compound of interest in the<br>
sample is determined. The presence or absence of the nucleic acid portion of the binding<br>
construct can be detected by any method that is capable of detecting the presence or absence<br>
of nucleic acid, for example, enzymatic amplification, hybridisation, or detection of a label.<br>
In one embodiment of the present invention, the method can be preferably adapted to detect<br>
the presence or absence of the nucleic acid portion of the binding construct by amplification<br>
of the nucleic acid portion, for example, by means of a polymerase chain reaction with<br>
appropriate primers. In this embodiment, only a few, and at least only one construct-<br>
compound complex needs to serve as a template for nucleic acid amplification. The<br>
amplified nucleic acid can be measured or detected by any suitable method, for example,<br>
using labelled oligonucleotides or by detecting appropriate bands on polyacrylamide gel<br>
electrophoresis.<br>
	In one embodiment of the present invention the sample to be analyzed may contain<br>
two or more compounds of interest. In order to detect two or more different compounds of<br>
interest in sample, two or rnore differem types of binding constructs, each having a different<br>
recognition portion capable of recognizing a different compound, are provided. Each type of<br>
binding construct includes a nucleic acid portion unique to the type, in order to detect the<br>
presence or absence of two or more compounds of interest in the sample.<br>
ll.	METHOD FOR INCREASING THE SENSITIVITY OF SOLUTION -PHASE DETECTION OF A<br>
COMPOUND OF INTEREST<br>
	The second method of the present invention includes a method for increasing the<br>
sensitivity of solution-phase detection of a compound of interest The method is especially<br>
suitable for samples that are suspected to contain a low concentration or low quantities of the<br>
compound of interest<br>
	The second method of the present invention includes the steps of providing a sample<br>
suspected of containing the compound of interest providing a binding construct that includes<br>
a recognition portion capable of binding the compound of interest and a nucleic acid portion;<br>
contacting the sample with the binding construct for a period of time sufficient to permit the<br>
recognition portion to bind any compound of interest in the sample, thus forming construct-<br><br>
WO 2005/021731	PCT/US2004/028267<br>
11<br>
compound complexes in solution; providing one or more surfaces that bears one of more<br>
accessible binding target capable of binding to the recognition portion; contacting the surface<br>
with the solution for a period of time sufficient for the accessible binding target to bind the<br>
recognition portion of any binding construct not bound to the compound of interest, thereby<br>
forming construct-surface complexes separating the construct-surface complexes from the<br>
solution, leaving the construct-compound complexes in the solution; and detecting the<br>
presence or absence of the nucleic acid portion of the binding construct in the solution<br>
wherein the separation of the construct-surface complexes from the solution results in a<br>
separation of substantially all binding constructs not bound to a compound of interest and in<br>
an increased sensitivity of detection of the compound of interest, and wherein the presence of<br>
the nucleic acid portion of the binding construct indicate the presence of the compound of<br>
interest in the sample.<br>
	The second method of the present invention include the steps of providing a sample<br>
suspected of containing a compound of interest, and providing a binding construct. The<br>
sample suspected of containing the compound of interest may be for example of entirely<br>
natural origin of entirely non-natural origin (such as of synthetic origin), or a combination of<br>
natural and non-natural origins. The sample may include whole cells, tissues, organs,<br>
biological fluids, extracts, or environmental samples. The binding construct includes a<br>
recognition portion that can recognize and bind to the compound of interest, and a nucleic<br>
acid portion. The sample is contacted with the binding construct for a period of time<br>
sufficient to permit the recognition portion of the binding construct to bind the compound of<br>
interest and form a construct-compound complex in solution.<br>
	The recognition portion of the binding construct can include virtually any molecule or<br>
combination of molecules capable of recognizing and binding the compound of interest.<br>
Such recognition portions can include, without limitation, peptides, polypeptides, antibodies,<br>
Fab fragments, nucleic acids, nucleic acid mimics., cell surface antigens, carbohydrates, or<br>
combination thereof. In one embodiment, the recognition portion includes an antibody<br>
(natural, modified, or recombinant) or an antibody fragment (such as an Fab fragment or<br>
single-chain antibody variable region fragment). In other embodiments, the recognition<br>
portion can be an antigen that binds an antibody, an aptamer that binds a target such as a<br>
peptide or small molecule, or a receptor that binds a ligand. In one preferred embodiment,<br>
the recognition portion binds monovalently to the compound of interest. In another preferred<br>
embodiment the recognitioa portion binds multivalently, for example bivalently and<br>
optionally bispecificalliy, to the compound of interest.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
12<br>
	The nucleic acid portion of the binding construct can include any nucleic acid or<br>
nucleic acid mimic that is capable of being detected. The nucleic acid that is used as the<br>
nucleic acid portion of the binding construct can include any type of nucleic acid, for<br>
example DNA or RNA, or a nucleic acid mimic (such as but not limited to a peptide nucleic<br>
acid), or a combination thereof. The nucleic acid of the invention can be single stranded or<br>
double-strandad. In one preferred embodiment, the binding construct of the present invention<br>
includes a nudeic acid portion, wherein the sequence of the nucleic acid portion does not<br>
include a sequence that is expected to be found in the sample, and can thereby be reasonably<br>
expected to be less easily contaminated by nucleic acid sequences in the sample, but is,<br>
however, designed so as to be easily detectable. The nucleic acid portion is preferably<br>
sufficiently long enough to easily dirtectable by the detection method chosen.<br>
	The recognition portion can be attached to the nucleic acid portion by any method,<br>
covalently or non-covalently, directly or indirectly, and it will depend on the nature of a<br>
gjven recognition portion Non-covalent attachment methods include but are not limited to<br>
physical adsorption, electrostaiic forces, ionic interactions, hydrogen bonding, hydrophilic-<br>
hydrophobic interactions, van der Walls forces, and magnetic forces. Where desired, for<br>
example, when increased flexibility is needed, a recognition portion may be indirectly affixed<br>
to the nucleic acid portion using a spacer arm. Preferably, the recognition portion is attached<br>
to the nucleic acid portion via covalent linkage, or via a high affinity non-covalent interaction<br>
such as that of biotin and avidin.<br>
	The second method of the present invention includes providing one or more surfaces<br>
bearing one or more accessible binding targets that are known to be recognized by and bind<br>
to the recognition portion of the binding construct. Such surfaces can be any particle capable<br>
of being separated from a liquid sample, or such surfaces may be non-particulate surfaces<br>
such as planar or non-planar surfaces, or a combination thereof. The surfaces can be<br>
optionally enclosed, for example, enclosed in a chamber. The binding targets can include any<br>
target capable of being recognized by and binding to the recognition portion of the binding<br>
construct, for example, peptide mimotopes or epitope mimetics for antibody variable regions,<br>
whole or partial antigen, nucleic acids, or sugar moietics. The accessible binding target can<br>
be attached to the surface by any method and it will depend on the nature of a given surface<br>
and binding target.<br>
	The surfaces are contacted with the solution for a period of time sufficieni for the<br>
accessible binding target to bind the recognition portion of any binding construct not bound<br>
to a compound of interest, thus forming construct-surface complexes. The construct-surface<br><br>
WO 2005/021731	PCT/US2004/028267<br>
13<br>
complexes and any excess unbound surfaces are separated from the solution, leaving behind<br>
the construct-Compound complexes. Separation may be by any method suitable to the type of<br>
surface used, for example, a magnet to separate magnetic particles, decanting from a planar<br>
surface, or separation by pressure Or vacuum, centrifugation, or filtration. Alternatively, any<br>
binding construct not bound to a compound of interest in a construct-compound complex can<br>
be separated from solution by methods such as, but not limited to precipitation, "salting-out",<br>
size-exclusion or filtration, extraction, or phase-separation. Separation of the construct-<br>
surface complexes and any excess unbound surfaces results in a separation of substantially all<br>
binding constructs not bound to a compound of interest. Most preferably, all binding<br>
constructs not bound to a compound of interest are separated from solution. Separation of the<br>
construct -surface complexes and any excess unbound surfaces results in a decrease in "false"<br>
passive signal generated by binding, complexes not bound to a compound of interest, and<br>
thus in an increased sensitivity of detection of the compound of interest relative to an assay<br>
wherein unbound or excess binding constructs is not separated<br>
	After the separation step, by detecting the presence or absence of the nucleic acid<br>
portion of the binding construct, the presence or absence of the compound of interest in the<br>
sample is determined. The presence or absence of the nucleic acid portion of the binding<br>
construct can be detected by any method that is capable of detecting the presence or absence<br>
of nucleic acid, for example, enzymatic amplification, hybridization, or detection of a label.<br>
In one embodiment of the present invention, the method can be preferably adapted to detect<br>
the presence or absence of the nucleic acid portion of the binding construct by amplification<br>
of the nucleic acid portion, for example, by means of a polymerase chain reaction with<br>
appropriate primers. In this embodiment, only a few, and at least only one construct-<br>
compound complex needs to serve as a template for nucleic acid amplification. The<br>
amplified, nucleic acid can be measured or detected by any suitable method, for exampfe,<br>
using labelled oligonucleotides or by detecting appropriate bands on polyacrylamide gel<br>
electrophorasis-<br>
	In one embodiment of the present invention, the sample to be analyzed may contain<br>
two or more compounds of interest. In order to detect two or more different compounds of<br>
interest in a sample, two or more different types of binding constructs, each having a different<br>
recognition portion capable of recognizing a different compound, are provided. Each type of<br>
binding construct includes a nucleic acid portion unique to the type, in order to detect the<br>
presence or absence of two or more compounds of interest in the sample.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
14<br>
III.	KIT FOR DETECTION OF A COMPOUND OF INTEREST<br>
	The present invention also includes a kit used for the detection of a compound of<br>
interest in a sample that is capable of detecting the presence of very tow quantities of the<br>
compound of interest in the sample.<br>
		The kit of the present invention includes a binding construct. The binding<br>
construct includes a recognition portion that can recognize and bind to be the compound of<br>
interest, and a nucleic acid portion. The recognition portion can include virtually any<br>
molecule or combination of molecules capable of recognizing and binding the compound of<br>
interest. Such recognition portions can include without limitation, peptides, polypeptides,<br>
antibodies. Fab fragments, nucleic acids, nucleic acid mimics, cell surface antigens,<br>
carbohydrates, or combination thereof. In one embodiment, the recognition portion includes<br>
an antibody (natural, modified, of recombinani) or an antibody fragment (such as an Fab<br>
fragment or single-chain antibody variable region fragment). In other embodiments, the<br>
recognition portion can be an antigen that binds an antibody, an aptamer that binds a target<br>
such as a peptide or small molecule, or a receptor that binds a ligand. In one preferred<br>
embodiment, the recognition portion binds monovalently to the compound of interest. In<br>
another preferred embodiment, the recognition portion binds multivalently, for example<br>
bivalently and optionally bispecifically, to the compound of interest<br>
	The nucleic acid portion of the binding construct can include any nucleic acid or<br>
nucleic acid mimic that is capable of being detected. The nucleic acid that is used as the<br>
nucleic acid portion of the binding construct can include any type of nucleic acid, for<br>
example DNA or RNA, or a nucleic acid mimic (such as, but not limited to a peptide nucleic<br>
acid), or a combination thereof. The nucleic acid of the invention can be single-stranded or<br>
double-stranded. In one preferred embodiment, the binding construct of the present invention<br>
includes a nucleic-acid portion, wherein the sequence of the nucleic acid portion does not<br>
include a sequence that is expected to be found in the sample, and can thereby be reasonably<br>
expected to be less easily contaminated by nucleic acid sequences in the sample, but is,<br>
however, designed so as to be easily detectable. The nucleic acid portion is preferably<br>
sufficiently long enough to be easily detectable by the detection method chosen.<br>
	The recognition portion can be attached to the nucleic acid portion by any method,<br>
covalently or non-covalently, directly or indirectly, and it will depend on the nature of a<br>
given recognition portion. Non-covalentt attachment methods include but are not limited to<br>
physical adsorption etectrostatic force ionic interaction, hydrogen bonding, hydrophilic-<br><br>
WO 2005/021731	PCT/US2004/028267<br>
15<br>
hydrophobic interactions, van der Walls forces, and magnetic forces. Where desired, for<br>
example, when increased flexibility is needed, a recognition portion: may be indirectly affixed<br>
to the nucleic acid portion using a spacer arm. Preferably, the recognition portion is attached<br>
to the nucleic acid portion via covalem linkage, or via a high affinity non-covlaent interaction<br>
such as that of biotin and avidin.<br>
	The kit of the present invention also includes one of more surfaces that bears one or<br>
more accessible toning targets that are known to be recognised by and bind to the<br>
recognition portion of the binding construct. Such surfaces can be any particle capable of<br>
being separated from a liquid sample, or such surfaces may be non-particulate surfaces such<br>
as planar or non-planar surfaces, or a combination thereof. The surfaces can be optionally<br>
enclosed, for example, in a chamber. The binding targets can include any target capable of<br>
being recognized by and binding to the recognition portion of the binding construct, for<br>
example, peptide mimotopes or epitope mimetics for antibody variable regions, whole or<br>
partial antigen, nucleic acids, or sugar moieties. The accessible binding target can be<br>
attached to the surface by any method, and it will depend on the nature of a given surface and<br>
	In embodiments wherein the nucleic acid portion of the binding construct is detected<br>
by nucleic acid amplificaiton, the kit can optionally include a nucleic acid amplification<br>
primer pair for example, a PCR primer pair wherein each primer of the primer pair<br>
hybridizes to its complementary sequence at the 3' end of a target nucleic acid sequence of<br>
the nucleic acid portion. The kit can further optionally include enzymes for nucleic acid<br>
amplification, such as Taq polymerase or RNA reverse transcriptase. In embodiments<br>
wherein the nucleic acid portion of the binding construct is detected by nucleic acid<br>
hybridizatian. the kit may contain one or more hybridization probes, such as oligonucleotides<br>
labelled with a detectable label. In embodiments including amplification of a signal, the kit<br>
may optionally include the reagents needed for signal amplification, such as enzymes or<br>
substrates. Kits of the invention can optionally include reagents for direct detection of the<br>
nucleic acid portion of the binding construct, such as molecular beacons or an appropriate<br>
antibody. The kit can optionally include means for separation of construct-surface complexes<br>
and any unbound surfaces, for example a magnet for embodiments wherein the surface is a<br>
magnetic particle, a filler for embodiments wherein the surface is a filterable particulate, or a<br>
pipette for embodiments wherein the surface is the walls of a tube.<br>
	Optionally, the kit can include instructions for the use of the kit Such instructions<br>
may be in any suitable form, such as a brochure, leaflet, pamphlel, booklet, or audiovisual<br><br>
WO 2005/021731	PCT/US2004/028267<br>
16<br>
materials. Preferably the instructions are sufficiently detailed to permit a user of the kit to<br>
successfully use the kit to detect a compound of interest in a sample. Such instructioos may<br>
include, for example, instructions for mixing reagents, manipulating components of the kit,<br>
proper handling of a sample, guidance in safety measures and in interpreting results, and<br>
trouble-shooting instructions.<br>
The Compound of Interest<br>
	The methods and kits of the present invention may be used to detect various classes of<br>
compounds of interest. The methods are particularly suitable to the detection of compounds<br>
of interest that are suspected to be present in low amounts or low concentrations in a sample.<br>
Compounds of interest can include, but are not limited to, nucleic acids, peptides, proteins,<br>
glycoproteins, Lipoproteins, lectins, antibodies, enzymes, and receptors; carbohydrates<br>
(monosaccharides, oligosaccharides. and polysaccharides) and glycosylated molecules; lipids,<br>
fats and lipidated molecules and whole or partial antigens. Compounds of interest may be<br>
small molecules (for example, ligands for a receptor, drugs of abuse, inorganic ions, metals,<br>
or chelates, metabolites, chemical intermediates, or natural products). Compounds of interest<br>
may be monomers, olieorners, or polymers; they can also be multi-molecular assemblies (for<br>
example; amyloid beta pratofibrils, the dystrophin-glycopnatein assembly, prateosomes,<br>
chapeione proteins, or fragments of cell walls or cell membranes). Compounds of interest<br>
may be of completely natural origin, completely artificial origin, or a combination of both<br>
(such as a compound of natural origin that is chemically or physically modified)<br>
The Sample<br>
	The methods of the invention may be applied to any suitable sample that is suspected<br>
of containing the compound of interest. The sample may be of entirely natural origin, of<br>
entirely non-natural origin (such as of synthetic origin), or a combination of natural and non-<br>
natura) origins. A sample may include whole cells (such as prokaryotic cells, bacterial cells,<br>
eukaryotic cells, plant cells, fungal cells, of cells from multi-cellular organisms including<br>
invertebrate, vertebrate, mammals, and humans), tissues, organ, or bioligical fluids (suth<br>
as, but not limited to, blood, serum, plasma, urine, semen, and cerebrospinal fluid) A sample<br>
may be an extract made from biological materials, such as from prokaryotes, bacteria,<br>
eukaryotes, plants, fungi, multi-cellular organisms or animals, invertebrates, vertebrates,<br>
mammals, non-human mammals, and humans, A sample may be an extract made from whole<br>
organisms or portions of organisms, cells, organs, tissues, fluids, whole cultures or portions<br><br>
WO 2005/021731	PCT/US2004/028267<br>
17<br>
of cultures, or environmental samples or portions thereof. A sample may need minimal<br>
preparation (for example, collection into a suitable container) for use in a method of the<br>
present invention, or more extensive preparation (such as, but not limited to removal,<br>
inactivation, or blocking of undesirable material of contaminants, filtration, size selection,<br>
affinity purification, cell lysis or tissue, digestion, concentration, or dilution). The sample<br>
may be in any phase (solid, liquid, or gaseous), in solution or in suspension, as long as it may<br>
be treated (suth as by mechanical disruption, lysis, heating, addition of a solvent, or<br>
suspension agent) to permit detection, of the construct-compound complex in solution. For<br>
example, a sample may be a soil sample, which may be suspended in an aqueous buffer and<br>
optionally filtered lo remove undesired particulates before introduction of the binding<br>
construct.<br>
The Binding Construct<br>
	The present invention includes a binding construct. The binding construct includes a<br>
recognition portion that can recognize and bind to the compound of interest, and a nucleic<br>
acid portion. Tht recognition portion can be virtually any molecule or combination of<br>
molecules capable of recognising and binding the compound of interest. Such recognition<br>
portions can include, without limitation, peptides, polypeptides, mimotopes. antibodies. Fab<br>
fragments, nucleic acids, nucleic acid mimics, aptamers, cell surface antigen, carbohydrates,<br>
small molecules (such as a small molecule antigen or a ligand for a receptor), inorganic ions<br>
or chelates, or a combination thereof. Particularly preferred is a recognition portion capable<br>
or monovalent binding to the compound of interest.<br>
	In one embodiment, the recognition portion is an antibody, for example, human or<br>
other mammalian lgG. IgG1, IgG2, IgG3, IgG4, IgM, IgA, SigA. or IgE, or avian IgY. and in<br>
one preferable embodiment, the recognition portion is an antibody ftagment. preparation of<br>
antibodies and antibody fragments against a compound of interest is well known in the art.<br>
These techniques are described in, for example. Antibodies, A Laboratory Manual, (Harlow<br>
and Lane) Cold Spring Harbor Laboratory Press (1988), and updated in Using Antibodies, A<br>
Laboratory Manual, (Hartow and Lane) Cold Spring Harbor Laboratory Press (1999). Where<br>
the recognition portion is an antibody or antibody fragment, it can be natural (such as an<br>
immunoglobulin isolated from serum) modified (such as a reduced or deglycosylated<br>
antibody), recombinant (such as an antibody produced by phage display), or a combination<br>
thereof.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
18<br>
	The recognition portion can be a natural, modified, or recombinant antibody binding<br>
fragment, such as an Fab fragment, of such as a single-chain antibody variable region<br>
fragment or ScFv in which the recombinant variable regions of an immunoglobulin's light<br>
and heavy chain domains are connected by a linker sequence (Pantoliano et al., (1991)<br>
Biochemistry, 30:10117-10125). In one embodiment, the recognition portion of the binding<br>
construct includes an Fab fragment, capable of monovalent binding to the compound of<br>
interest Fab fragments against the compound of interest can be produced when antibodies<br>
against the compound of interest are cleaved by reduction or enzymatic cleavage of the<br>
disulfide bridge that holds the heavy chains together, converting an antibody into two<br>
separate Fab fragments, each capable of recognizing and binding the compound of interest,<br>
and bearing a reactive sulfhydryl or thiol group that can be attached to other molecules, for<br>
example, the nucleic acid portion of the binding construct. In One embodiment, a Fab<br>
fragment can be directly covalently attached to a nucleic acid portion through the free<br>
sulfhydsyl of the Fab fragment, thus forming the binding construct. In another embodiment,<br>
the Fab fragment can be indirectly covalently attched to the nudeic acid portion of the<br>
binding construct through a bi-functional linker capable of specifically binding both said Fab<br>
fragment and said nucleic acid portion forming the binding construct. Alternatively, the<br>
recognition portion of the present invention can also be attached to the nucleic acid portion<br>
via non-covalenr means. for example, via an avidin-biotin interaction, a zinc-polyhistidine<br>
interaction, an antibody-antigen interaction, an aptamer- peptide interaction, or via other<br>
polypeptides capable of binding nucleic acids such as zinc-binding polypeptide domain.<br>
	The recognition portion (for example, peptdes, mimotopes, antibodies, or aptamers)<br>
can include natural molecule, artificial molecules, fusion or chimeric molecules, molecules<br>
developed by random or non-random combinatorial synthesis (Dooley and Houghten (1993)<br>
LifeSci.. 52: 1509-1517; Kramer etaL (1993) peptide Res., 6314-319; Folgori et al, (1994)<br>
EMBOJ, 13:2236-2243, Smith &amp;Petrenko (1997) Chem. Rev., 97:391-410) or by directed<br>
evolution methods such as yeast two-hybrid systems, protein fragment complementation<br>
assay, phage display, ribosome display, yeast surface display, and bacterial surface display<br>
techniques (Crameri and Suter (1993) Gene, 137:69-75, Meola et al (1995) J. Immurtol<br>
154:3l62-3l72:Georgiou et al (1997) Nature Biotechnol 15:29-34; Mossner and<br>
Pluckthun (2001) Chimia, 55:324, B. K. Kay, J. Winter, and J McCaffurty (editors) "Phage<br>
Display of Peptides and proteins: A Laboratory Manual", Academic Press, Inc., San Diego.<br>
1996, 344 pp), or a combination thereof. The recognition portion can be selected for its<br>
ability to recognize and bind the compound of interest by any suitable means known in the<br><br>
WO 2005/021731	PCT/US2004/028267<br>
19<br>
art, for example, by affinity selection, affinity purification, iterative panning, or surface<br>
plasmon resonance technology (Fagerstam et al (1991) J. Mol- Recognition. 3:208-214;<br>
Housbmand et al (1999) Anal Biochetn,, 268:363-370).<br>
	The recognition portion of the binding construct may bind to the compound of interest<br>
in a monovalenl or in a multivalent fashion. In one preferred embodiment, the binding of the<br>
recognition portion of the binding construct to the compound of interest is monovalent<br>
Examples of monovalent binding include, but are not limited to, a Fab fragment that<br>
monovalently binds to an antigen, a receptor molecule that monovalently binds its ligand, and<br>
an aptamer that monovalently binds a peptide. In certain other embodiments, the binding of<br>
the recognition portion of the binding construct to the compound of interest may preferably<br>
be multivalent, for example bivalent or trivalent. Multivalency may be desirable, for<br>
example, to increase the avidity of the binding between the binding construct and the<br>
compound of interest. In some embodiments, the binding of the recognition portion of the<br>
binding construct to the compound of interest may be both bivalent and bispecific (that is the<br>
recognition portion may bind to and recognize two separate specific binding sites of the<br>
compound of interest. An example of a bivalent, monospecific binding is a dimeric antibody<br>
fragment or diabody (Holigen et al. (1993) Proc. Natt. Aaad Sci. USA. 90:6444-6448) that<br>
is designed to bind monospecifically. An example of a bivalent, bispecific antibody is a<br>
dtabody that is designed to bind to two distinct binding sites of the compound of interest<br>
	The nucleic acid portion of the binding construct can include any nucleic acid or<br>
nucleic acid mimic that is capable of being detected. The nucleic acid that is used as the<br>
nucleic acid portion of the binding construct can include any type of nucleic acid, for<br>
example DNA or RNA, or a nucleic acid mimic (such as, but not limited to, a peptide nucleic<br>
acid), or a combination thereof. In this respect, the nucleic acid portion of the binding<br>
construct acts as a marker for detection, for example, for detection through nucleic acid<br>
amplification, nucleic acid hybridisation, enzymatic signal amplification, detection of a 1abel<br>
or a combination of these detection methods. The nucleic acid of the invention can be single-<br>
stranded or double-stranded.<br>
	In one preferred embodiment, the binding construct of the present invention includes<br>
a nucleic acid portion, wherein the sequence of the nucleic acid portion does not include a<br>
sequence that is expected to be found in the sample, and can thereby be reasonably expected<br>
to be less easily contaminated by nucleic acid sequences in the sample, but is, however,<br>
designed so as to be easily detectable. For example, where the sample is a serum sample<br>
from a mammal, the sequence of the nucleic acid portion can include a nucleic acid sequence<br><br>
WO 2005/021731	PCT/US2004/028267<br>
20<br>
believed to occur only in higher plants, and thus unlikely to be found in mammalian serum.<br>
In another preferred embodiment, the binding construct of the present invention includes a<br>
nucleic acid portion, wherein the sequence of the nucleic acid portion does not include a<br>
sequence that is believed to occur in nature, In other embodiments, the nucleic acid portion<br>
of the invention can include artificially derived sequences (such as sequences arrived at by<br>
random or non-random combinatorial approaches) or repetitive sequences, for example,<br>
repetitive sequences that may be complementary to a single hybridization primer or probe. In<br>
other embodiments., the nucleic acid portion of the invention can include multiple, repetitive<br>
nucleic acid portions attached in series of in parallel to the recognition portion.<br>
	The nucleic acid portion is preferably sufficiently long enough to be easily detectable<br>
by the detection method chosen. Where detection of the nucleic acid portion includes nucleic<br>
acid amplification the nucleic acid portion preferably includes a single-strand length of<br>
between about 50 to about 5000 nucleotides, or between about 100 to about 4000 nucleoudes,<br>
or between about 200 and about 3000 nudeotides However, the nucleic acid portion may<br>
include any number of nucleotides that is suitable to the chosen method of amplification of<br>
the nucleic acid portion, for example PCR or reverse-transcriptase PCR. Where detection of<br>
the nucleic acid portion does not include nucleic add amplification, the nucleic acid portion<br>
preferably includes a single strand lenght of between about 4 nutiectides to about 5000<br>
nucleotides, or between about 20 nudeotides to about 4OOO nucleotides, or between about<br>
100 nudeotides to about 3000 nudeotides. However, the nucleic acid portion may include<br>
any number of nucleotides that is suitable to the chosen method of detection of the nucleic<br>
acid portion.<br>
	The recognition portion can be attached to the nucleic acid portion by any method,<br>
covalently or non-covalently, directly or indirectly, depending on the nature of a given<br>
recognition portion and nucleic acid portion. Such attaching methods can be, for example,<br>
covalent cross-linking as well as non-covalent linking methods such as are known in the art<br>
(see, for example, R P. Haugland, "Handbook of Fluorescent Probes and Research<br>
Products". 9th edition, J. Gregory (editor), Molecular Probes. Inc, Eugene, OR, USA, 2002,<br>
966 pp.; Seitz and Kohler (2001), Chemistry. 7-3911-3925; and Pierce Technical Handbook,<br>
Pierce Biotechnology, Inc., 1994, Rockford, IL,). Where desired, for example, when<br>
increased flexibility is needed, a recognition portion may be affixed to the nucleic acid<br>
portion using a spacer arm. (Keyes et al. (1997) Biophys. J., 72:282-90; Hustedt et al. (1995)<br>
Biochemistry. 34:4369-4375; and Pieree Technical Handbook, Pierce Biotechnology. Inc..<br>
1994, Rodford, IL). In one embodiment, the recognnion portion is attached to the nucleic<br><br>
WO 2005/021731	PCT/US2004/028267<br>
21<br>
acid portion via covalent linkage, Covalent means are well known in the art and may include,<br>
for example, the use of reactive groups, chemical modification or activation, photoactivated<br>
cross-linking, or bifunctional or trifunctional cross-linking agents (Pierce Technical<br>
Handbook, Pierce Biotechnology, Inc., 1994, Rockford, IL). In another embodiment, the<br>
recognition potion is attached to the nucleic acid portion via non-covalent means- Non-<br>
covalent means include, but are not limited, to physical adsorption, electrostatic forces, ionic<br>
interactions, hydrogen bonding, hydrophilic-hydrophobic interactions, van der Waals forces,<br>
and magnetic forces. A combination of covalent and non-covalent attachment means may be<br>
used. For example, the nudeic acid portion (or repetitive multiples of the nucleic acid<br>
portion) may be biotinylated and non-covalently attached to a multivalent avidin moiety that<br>
is covelently cross-linked to the recognition portion.<br>
The Surfaced and Accessible Binding Targets<br>
	The present invention also includes one or more surfaces that bears one or more<br>
accessible binding targets that are known to be recognized by and bind to the recognition<br>
portion of the binding construct. Such surfaces can be particulate surfaces or non-particulate<br>
surfaces, and can be made of any suitable material, such as, but not limited to, plastics,<br>
polymers, ceramics, glass, silica compounds, modified silica compounds, fluorocarbons,<br>
metals or metal oxides, sorberuts. resins, biological materials (for example, potypeptides and<br>
carbohydrates) or a combination thereof. Particulate surfaces can be any particle capable of<br>
being separated from a liquid sample, for example, magnetic particles, polymeric particles,<br>
glass particles, silica particles, ceramic particles, and the like. Particulate surfaces can be any<br>
shape including spherical, non-spherical, symmetric, asymmetric, or irregular, they can be of<br>
uniform or non-uniform size. Particulate surfaces can take any suitable form, for example,<br>
powders, beads, fibers, macromolecular aggregates, nanoparticles, or nanotubes. Particulate<br>
surfaces may be optionally enclosed in a chamber, such as in a reusable on a disposable<br>
cartridge, cassette, or insert, Non-particulate surfaces include but are not limited to planer or<br>
non-planar surfaces (for example, the sides of a tube or a well), non-porous films or<br>
membranes, porous films or membranes, fibers, fillers, meshes, grids, fillers, matrices, gels,<br>
or a combination thereof<br>
	The accessible binding targets of the present invention can be any binding target<br>
capable of being recognised by and binding to the recognition portion of the binding<br>
construct, for example, nucleic acids, peptide mimotopes or epitope mimetics for antibody<br>
variable regions (Geysen et al. (1986) Mol. Immunol, 23:709-715), proteins, glycoproteins.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
22<br>
lipoproteins, lectins, antibodies, enzymes, receptors; carbohyd rates (monosaccharides,<br>
oligosaccharides, and polysaccharides) and glycosylated molecules; lipids, fats, and tipidated<br>
molecules; and whole or partial antigens; small molecules (for example, ligands for a<br>
receptor, drugs of abuse, inorganic ions or chelates, metabolites, chemical intermediates, or<br>
natural products). Accessible binding targets may be monomers, oligomers, polymers; they<br>
can also be rnulti -molecular assemblies. Accessible binding targets may be of completely<br>
natural origin, completely artificial origin, or a combination of both (such as a compound of<br>
natural origin that is chemically modified).<br>
	The accessible binding target can be attached to the surface by any method, covalent<br>
or non-covalent, directly or indirectly, depending on the nature of a given surface and<br>
accessible binding target. The accessible binding target can be attached to the surfaces by<br>
covalent bonds, for example, through the reaction of a chemically reactive group of the<br>
accessible binding target with a chemically reactive group of the surface, using appropriate<br>
reagents. For example, binding targets bearing primary amines can be linked to amine-<br>
binding supports to form a secondary amine using sodium cyanoborohydride; binding targets<br>
bearing carbohydrates can be linked to surfaces bearing a free hydrazide group to form a<br>
stable hydrazone bond. In another example, l-ethyl-3[3-dimethylaminopropyl]-carbodiamide<br>
hydrochloride (EDC) can be used to link carbokyl groups to surfaces bearing primary amines<br>
The accessible binding target can be attached to the surface by non-covalent means,<br>
including, but not limited, to physical adsorption, electrostatic forces, ionic interactions,<br>
hydrogen bonding, hydrophi1ic-hydrophobic interactions, van der Waals forces, and magnetic<br>
forces Non-limiting examples of non-covalent means include biotin-avidin interactions,<br>
pratcin A or protein G interactions with immnnoglobulins, and antigen-antibody interactions.<br>
	The accessible binding target may be covalently attached to surface, such as, but not<br>
limited to, polymeric, ceramic, glass, silica, or organic surfaces, through a variety of surface<br>
chemistries. Surfaces, both particulate and non-particulate, are readily available with<br>
carboxyl, amines, hydroxyl, hydrazide, or chloromethyl, functional groups. In one<br>
embodiment, where the surface is a magnetic particle, commercial magnetic stocks may be<br>
used, and may be selected for properties allowing the desired accessible binding target to be<br>
attached to the magnetic particles. For example, the commercially available magnetic<br>
particle stock Micromod. Nanomag Silica, NH250 (catalogue number 13-01-252,<br>
Micromod Fartikeltechnologie GmbH. Rostock-Warremuaide. Germany) contains on its<br>
surface a free amino group, which can be reacted with cross linkers, for example, N-<br>
succinimidyl(4-iodoacetyl)aminobenzoate (SIAB) or sulfosuccinimidyl-4-(N-<br><br>
WO 2005/021731	PCT/US2004/028267<br>
23<br>
maleimindomethyl)Gyclohexane-l-carboxylate(Sulfo-SMCC), each of which has an amine-<br>
reactive end and a sulfhydryl-reactivie end, The NSH ester of SLAB can couple to primary<br>
amine containing molecules, and the ensuing reaction is a stable amide linkage.<br>
Separation<br>
	The construct-surface and any excess unbound surfaces are separated from the<br>
construct-compound complexes in the sample solution using any suitable method of<br>
separation, which can depend on the type of surface used. Such methods can include, without<br>
limitation, separation by pressure or vacuum, centrifugation, size-exclusion, filtration, or a<br>
combination thereof. The following are non-limiting examples of separation. Where the<br>
surfaces are particulates (such as powders, beads, fibers, macromolecular aggregates,<br>
nanoparticles, or nanotubes), the construct-surface complexes can be separated by<br>
sedimentation, centrifugation, filtration, size-exclusion, or non-covalent attraction (for<br>
example, charge or hydrophobicity interaction). Where the surfaces are pfanar or non-planar<br>
non-particulate (such as the wall of a microliter well or tube or vessel). the construct-<br>
surface complexes can be separated by decanting or aspiration. Where the surfaces are<br>
magnetic particles, separation of the construct-surface complexes from the mixture may be<br>
accomplished by application of magnetic force, such as by exposure to a magnet.<br>
	In one embodiment, the surfaces can be enclosed, permanently or temporarily, in a<br>
chamber (such as in a tube, cartridge, column, or cassette), which can facilate separation of<br>
the excess or unbound binding construct. For example, the mixture containing the sample<br>
and binding construct can be passed through a cartridge or column containing accessible<br>
binding targets bound to surfaces (such as beads, a matrix a gel, or a filter), whereby the<br>
solution that is cluted or that passes through the cartridge or column is stripped of excess of<br>
unbound binding construct.<br>
	In some embodiments, "separation" of the construct-surface complexes from the<br>
mixture need not require physical removal of the construct-surface complexes from the<br>
solution, where it is sufficient to isolate the construct-surface complexes from the solution-<br>
phase detection step. For example, where the surfaces are magnetic particles, adequate<br>
separation of the construct-surface complexes from the mixture may be accomplished by<br>
application of magnetic force, for example, to the side of a vessel containing the mixture, thus<br>
attracting the magnetic particles to the side of the vessel and isolating them from the solution,<br>
allowing an aliquot of the solution to be sampled for the detection step.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
24<br>
	In an alternative embodiment, any binding construct not bound to a compound of<br>
interest in a construct-compound complex can be separated from the solution by methods<br>
such as, but not limited to precipitation, "salting-out", size-exclusion or filtration, extraction,<br>
or phase-separation. In this alternative embodiment the principles of the methods of the<br>
invention remain the same, that is to say. (1) the use of only a single binding entity (the<br>
binding construct); (2) the separation of substantially all binding constructs not bound to a<br>
compound of interest, resulting in an increased proportion of true "positive" signal to false<br>
"positive" signal, and therefore an increased sensitivity of detection of the compound of<br>
interest; (3) detection in solution phase, and no requirement for solid -phase detection; and (4)<br>
no requirement for prior knowledge of a nucleic acid sequence of the compound of interest.<br>
Detection<br>
	After the separation step, by detecting the presence or absence of the nucleic acid<br>
portion of the binding construct, the presence or absence of the compound of interest in the<br>
sample is indicated. The presence or absence of the nucleic acid portion of the binding<br>
construct can be detected by any method. not necessarily involving nucleic acid<br>
amplification, that is capable of detecting the presence or absence of nucleic acid, for<br>
example, enzymatic amplification, hybridisation, or detection of a label. The defection of the<br>
nucleic acid portion of the binding construct can be accomplished by any method suitable for<br>
such purpose. These methods are well known in the art and they include. without limitation.<br>
amplification of the nucleic acid portion, hybridization of the nucleic acid portion,<br>
amplification of a signal, detection of a label, of a combination thereof<br>
	Methods of the present invention can include nucleic acid amplification of the nucleic<br>
acid portion. Preferred embodiments of the present invention are capable of detecting low<br>
amounts or concentrations of the compound of interest, and are capable of amplifying and<br>
delecting a few, and a minimum of one construct-compouud complexes. In such preferred<br>
embodiments, only a few, and a minimum of one, construct-compound complexes need be<br>
reman in sototion to serve as amplification templates after the separation of the construct-<br>
surface complexes. The amplified nucleic acid can be measured or detected by any suitable<br>
method, for example, using labelled oligonucleotides as hybridization probes or by detecting<br>
amplified fragments of the appropriate size by polyacrylamide gel electrophoresis.<br>
Amplification of the nucleic acid portion may use any suitable amplification method, such as<br>
polymerase chain reaction amplification or reverse transcriptase amplification (Molecular<br>
Cloning: A Laboratory Manual, Joseph Sambrook et al, Cold Spring Harbor Laboratory,<br><br>
WO 2005/021731	PCT/US2004/028267<br>
25<br>
2001, 999 pp.; Short Protocols in Molecular Biology, Frederick M. Ausubel et al (editors),<br>
John Wiley &amp; Sons, 2002, 1548 pp ), rolling circle amplification (Liu et al. (1996), J. Am.<br>
Chem, Sac., 118:1587-1594), antisense RNA amplification (Phillips and Eberwine(1996)<br>
Methods, 10:283-288), strand displacement amplification (Walker et a/. (1592), Nucleic<br>
Acids Res., 20:1691-1696), composite primer/strand displacement amplification (United<br>
States Patent Number 6.251,639 to Kurn, "Methods and compositions for linear isothermal<br>
amplificalion of polynucleotide sequence, using a RNA-DNA composite primer", issued<br>
June 26, 2001) Q-beta repIicase-mediated amplification (Lomeli et al. (1989) Clin. Chem.,<br>
35:1826-1831). linked linear amplification (Reyes et al. (20O1) Clin. Chem, 47:31-40), self-<br>
sustained sequence replication (3SR) (Fahy et al (1991) Genome Res-, 1:25-33), or other<br>
nucleic acid amplification methods known in the art (Andras et al. (2001) Mo/. Biotechnol.<br>
19:29-44). Another method of detection of the nucleic acid portion of the binding construct<br>
can be, for example, by primer extension and detection of the extended nucleic acid.<br>
	Hybridisation of the nucleic acid portion may be achieved by hybridization of a probe<br>
to the nucleic acid portion of the binding construct, followed by detection of the hybridized<br>
structure seconding to methods known in the art. The probe may include DNA, RNA, a<br>
nucleic acid mimic (such as, but not limited to, a peptide nucleic acid), or a combination<br>
thereof. The probe may include a suitable detectable label, such as, but not limited to,<br>
radioisotopes, spirt labels, fluorophores (including organic dyes and lanthanide chelates),<br>
chromophores, one or both member of a resonance energy transfer pair. haptens, antigens,<br>
antibodies, or enzymes. A label, for example, a fluorophore or a hapten, can also be<br>
incorporated directly into the nucleic acid by methods known in the art followed by the<br>
detection of the label. Detection of the nucleic acid may include enzymatic amplification of a<br>
signal (such as a signal from a label on a probe), for example by using peroxidase-tyramide<br>
signal amplification (R. P. Haugland, "Handbook of Fluorescent Probes and Research<br>
Products". 9th edition, J. Gregory (editor), Molecular Probes, Inc. Eugene, OR, USA, 2002.<br>
966 pp) or alkaline phosphatase-anti-alkaline phosphatase signal amplification. The nucleic<br>
acid portion of the buiding construct can also be detected, directly or after amplification or<br>
hybridization, by other methods, including but not limited to molecular beacons (optionally<br>
combined with real time detection) or other resonance energy transfer methods, or<br>
immunological detection (for example, using antibodies that recognize and capture an<br>
amplified DNA sequence in an ELISA-type assay).<br>
	When detection of the nucleic acid portion of the binding construct involves<br>
amplification a preferred method is amplification of the nucleic acid portion by polymerase<br><br>
WO 2005/021731	PCT/US2004/028267<br>
26<br>
chain reaction, according to methods known in the art (Molecular Cloning: A Laboratory<br>
Manual, Joseph Sambrook et al, Cold Spring Harbor Laboratory, 2001; Short Protocols in<br>
Molecular Biology, Frederick M. Ausubri et al. (editors), John Wiley &amp; Sons, 2002). The<br>
capability to detect a compound of interest in a sample can be significantly increased and<br>
broadened by coupling to polymerase chain reason PCR allows for enormous<br>
amplification capability and is a process Where as specific sequence of nucleic acid can be<br>
amplified millions of times. This enormous amplification capability is based on the ability to<br>
amplify a specific target sequence of nucleic acid flanked by a set of primers. Once the<br>
nucleic acid is amplified, the result can be detected using any method suitable for such<br>
purpose, for example using an agarose gel<br>
EXAMPLES<br>
EXAMPLE I: A NON-LIMITING EMBODIMENT OF A METHOD FOR DETECTION OF A<br>
COMPOUND OF INTEREST IN A SAMPLE<br>
	FIGURE 1 depicts a plurality of binding constructs for use in a method of the present<br>
invention, as described in Example III. Each binding construct 101 includes a recognition<br>
portion 102, which recognize and binds the compound of interest, and a nucleic acid portion<br>
103. In this embodiment, the recognition portion 102 preferably includes a Fab fragment,<br>
such as a Fab fragment made by cleaving the disulfide bridge of the heavy chains of a<br>
monoclonal antibody against the compound of interest. In this embodiment, the nucleic acid<br>
portion 103 preferably includes DNA. The recognition portion 102 can be attached in the<br>
nucleic acid portion 103 thruogh a covalent bond 104, for example, through a covalent bond<br>
between the nucleic acid portion 103 and the free sulfhydryl of the Fab fragment of the<br>
recognition portion 102.<br>
	FIGURE 2 shows a number of binding constructs 101 that have been mixed with a<br>
sample coatainiog a compound of interest 201, as described in Example IV. The recognition<br>
portion 102 of the binding construct 101 has recognized a compound of interest 201 forming<br>
a construct-compound complex 202. Also shown in the lowest part of the figure is a binding<br>
construct 101, including a recognition portion 102, which has not bound the compound of<br>
interest 201, and a nucleic acid portion 103<br>
	FIGURE 3 shows surfaces 301, bearing accessible binding targets 302. In this<br>
embodiment, the surfaces 301 are magnetic particles and the accessible binding targets 302<br><br>
WO 2005/021731	PCT/US2004/028267<br>
27<br>
are peptide mimotopes, as described in Example II. The surfaces are introduced in the<br>
sample mixture, which can contain binding constructs 101, which may be bound to a<br>
compound of interest in a construct-compound complex 201, or not bound to a compound of<br>
interest. The accessible binding target 301 binds to the recognition portion 101 of any<br>
binding construct 101 not bound to a compound of interest, forming a construct-surface<br>
complex 303.<br>
	FIGURE. 4 shows a magnet 401 being used to separate the construct-surface<br>
complexes 303, as described in Example IV. The construct-surface complexes 303 and any<br>
excess or unbound surfaces 301 are separated. leaving the construct-compound complexes<br>
202 in solution. After separation, the nucleic acid portion 103 of the binding construct 101<br>
included in a construct-compound complex 202 can be detected by any method suitable for<br>
that purpose. Detecting the presence of the nucleic acid portion 103 of the binding construct<br>
101 indicates the presence of the compound of interest 201.<br>
EXAMPLE II: PREPARATION OF THE MAGNETIC PARTICLES BEARING MIMOTOPES<br>
	This example provides one non-limiting embodiment of the present invention wherein<br>
the surfaces are magnetic particles bearing peptide mimotopes as the accessible binding<br>
targets. In this model system, an antibody that recognises a bacterial cell wall protein, the<br>
Moroxella catarrhalis OMPE proiein, was used. A bacterial recombinant fragment of M.<br>
catarrhalis OMPE protein was provided by Dr. Timothy Murphy at the University of Buffalo<br>
and used as the immunogen for preparing monoclonal antibody 12D.5 (Mab 12D.5). Using<br>
overlapping peptide mapping, the antigenic epitope was shown to reside within amino acids<br>
187 -220 of the recombinant fragment. A peptide corresponding to this epitope, designated<br>
Mopep2= was synthesized with an N-terminal cysteine for post-synthesis attachment to free<br>
sulfhydryl-binding structures. Mab 12D.5 was cleaved into two Fab fragments using 2-<br>
mercaptoethylamins HCL (Pierce) leaving a reactive sulfhydryl group that can be attached to<br>
another molecule.<br>
	Thn milligrams per mililiter of magnetic particle stock Micromod, Nanomag<br>
Silica, NH250, (lot number 210213T, catalogue number 13-01 -252, Micromod<br>
Partikeltechnologie GmbH, Rostock-Warrenmuende, Germany), which contains a free amino<br>
group on its surface, was reacted with 140 micrograms SIAB in DMSO and rotated in the<br>
dark for 2 hours. The particles were separated by means of a magnet and subsequently<br>
incubated in 50 millimolar sodium borate buffer pH 9.6 containing 5 millimolar EDTA with<br>
100 microliter of 1.7 milligrams per milliliter Mopep2 in DMSO and incubated overnight.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
28<br>
The free sulfhydryl group on the Mopep2 peptide was reacted with the SIAB-treated<br>
magnetic particles creating a stable thioethar bond. Unreacted sites were blocked with free<br>
cysteine and the Mopep2-labelled particles were washed and resuspended in 1 milliliter PBS<br>
for analysis.<br>
	To test the ability of the Mopep2 particles to bind to free 12D. 5 Mab in solution, an<br>
inhibition ELISA was performed. Ten rnicroliters of either Mopep2-particles or control<br>
particles similarly labelled with bovine serum albumin (BSA) were incubated with a 2-fold<br>
serial dilution of horseradish peroxidase (HRPO)-conjugated 12D.5 in PBS (total volume 100<br>
microliter) starting at 2 microgram/millilitre in a 96-well microliter plate and incubated for<br>
30 minutes. Particles were separated to the bottom qf the wells by means of a magnet and 50<br>
micro liters of each dilution set were added to Mopep2-coated plates and incubated for an<br>
additional 30 minutes. A standard ELISA was then performed with the addition of 3.3'5,5'-<br>
tetramethylbenzidine substrate, allowed to develop for 10 minutes and abssorbance read an<br>
630 nanometers with no stop solution added. Data depicted in FIGURE 5 show that the<br>
Mopep2 particles were capable of binding to and separating monoclonal antibody 12D5<br>
(12D5 Mab) horseradish peroxidase (HRPO) conjugates from solution, thus preventing the<br>
12D.5 Mab HRPO conjugate from binding to Mopep2-coated wells in a dose-dependant<br>
manner. Particles coated with bovine serum albumin (BSA) were unable to inhibit 12D.5<br>
Mab HRPO conjugate from binding to Mopep2-coated wells.<br>
EXAMPLE III: PREPARATION OF THE BINDING CONSTRUCT<br>
	This example provide one embodiment of the binding construct in which on Fab<br>
fragment is used as the recognition portion of the binding construct. The plasmid pUC 19<br>
was purchased from Invitrogen and plasmid DNA was linearized by EcoRl. The linearized<br>
pUC19 DNA was extracted and purified from agarose gel. This DNA was attached to the<br>
Fab fragrnent of Mab 12D.5 and served as the nucleic acid portion of the binding construct.<br>
Commercially available primers were subsequently used to amplify a 1 kilobase fragment<br>
from the linearized pUC 19 template. To generate a DNA fragment that would be attached to<br>
the 12D.5 Fab fragment through the free solfhydryls (-SH) and serve as the nucleic acid<br>
portion of the binding construct, a 5'-psoralen, 3'-amino oligonucleotide was conjugated to a<br>
SIAB linker through the available amine. The pUC 1 kilobase DNA was denatured in the<br>
presence of excess oligonucleotide and XXX annealed to the psoralen using long wave<br>
ultraviolet light. The excess oligonucleiotide vas removed with an molecular weight cut-off<br>
(MWCO) filter and the SIAB/pUC template vas then attached to the Fab fragment using the<br><br>
WO 2005/021731	PCT/US2004/028267<br>
29<br>
free sulfhydryl-binding end of the attached linker. The entire binding construct, designated<br>
Fab-DNA, was not further purified.<br>
EXAMPLE IV<br>
	This example provides one embodiment for detection of a compound of interest<br>
wherein the nucleic acid portion of the binding construct is amplified by PCR. This<br>
experiment demonstrated the ability of the Fab-DNA binding construct (Mab 12D.5/pUCl9<br>
construct) to bind to the compound of interest, the free antigen (bacterial recombinant<br>
fragment of OMPE or rOMPE), thus forming construc-compound complexes in solution.<br>
Surfaces (magnetic particles) bearing accessible binding targets (Mopep2 peptides) bound<br>
any Fab-DNA not bound to the compound of interest, and a magnet was used to separate the<br>
surfaces, leaving the construct-compound complexes (Fab-DNA bound to antigen) in solution<br>
with the nucleic acid portion (pUC19) of the binding construct available for nucleic acid<br>
amplificaiion<br>
	rOMPE was titered in two-fold dilution steps in 1 milliliter PBS from 100<br>
nanogram/milliliter to 1.56 nanogram/milliliter. Each serial dilution was incubated with 20<br>
microliter (8 micrograms of Fab-DNA) for 30 minutes, and then 10 microliters of Mopep2<br>
particles was added to each well and incubated for an additional 30 minutes. Control wells<br>
included: (1) 1.1 milliliter PBS and 0.2 milliliter Fab-DNA no rOMPE (positive control), and<br>
(2) 1 milliliter PBS, 0.2 milliliter Fab-DNA, and 0.1 milliliter particles (negative control).<br>
Wells were exposed to a rare earth magnet for 15 minutes to separate the magnetic particles<br>
from the solution, and 0.2 milliliter of the solution removed from each well for amplification<br>
	Samples were amplified by PCR as follows. A 50 microliter PCR reaction mix<br>
contained 5 microliters 1OX PCR. buffer without MgCl2 1 microliter of 50 millimolar MgCl2.<br>
0.5 microliter of 50 millimolar dNTPs, 1 microliter of each sense or antisense primer to<br>
pUC19, 1 microliter Taq polymerase, 2 microliters of sample (solution), and 38.5 microliters<br>
ddH2O. The sense primer had the sequence CCTCTAGAGTCGACCTGCAGGCATGC<br>
(SEQ ID NO. 1). The anti-sense primer had the sequence<br>
CACTGGCCGTCGTTTTACAACGTCGTG (SEQ ID NO. 2). All sequences are given in<br>
the 5' to 3' direction.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
30<br>
Samples were amplified using the following cycling parameters;<br><br>
	PCR samples were resolved on 1% agarose gel and visualized by ethidium bromide.<br>
10 microliters of each concentration of PCR samples was loaded on the gel. PCR primers<br>
amplified an approximate 2650 bp pUC19 fragment. A 1 kilobase extension DNA ladder<br>
(Invitrogen) was used as the molecular weight standard As shown in FIGURE 6. PCR,<br>
primers amplified a 2.6 kilobase DNA fragment in the following samples: 790 picogram, 395<br>
picogram, 198 picogram, 98 picogram and 12 picogram. Very fine bands were observed in<br>
samples of 49 picogram and 24 picogram.<br>
EXAMPLE V: DETECTION OF COMPOUNDS OF INTEREST<br>
	Non-limiting examples of detection of different types of compounds of interest<br>
follow.<br>
Beta-Amyloid protofibrils: In this example the compound of interest is an oligomer of<br>
amyloid beta protein, called a protofibril Detection of amyloid beta protofibrils can be<br>
useful in the early detection of Alzheimer's disease. The sample is scrum or cerebral spinal<br>
fluid (CSF) from a patient suspected of having or being at risk for Alzheimer's disease. The<br>
binding construct includes (1) a recognition portion, which is an antibody or an antibody<br>
fragment that recognizes the protofibril, find (2) a nucleic acid portion that does not exist in<br>
nature. The nucleic acid portion is a DNA template that can be recognized by synthetic<br>
primers designed to bind to the template at low stringency for amplification using PCR. The<br>
binding construct is added to a concentration of 10 nanograms permilliliter with a 100<br>
microliter sample of serum, and incubated for a sufficient time 10 allow the antibody or<br>
antibody fragment to bind to protofibrils present in the sample, thus forming construct-<br>
compound (construct-protofibril) complexes in solution. The binding surfaces are magnetic<br>
particles labelled with an accessible binding target, the same peptide sequence that was used<br>
as the immunogen to generate the anti-protofibril antibody (the recognition portion of the<br>
binding construct). The magnetic partides bear primary amine reactive groups, allowing the<br>
immunogenic peptide to be conjugated to the particles through its carboxy-terminus carboxyl<br><br>
WO 2005/021731	PCT/US2004/028267<br>
31<br>
using a water-soluble carbodiamide such as 1-ethyl-3-(3-dimethylaminopropyl)carbodiamide<br>
(EDC). Ten microliter of a 30% particle suspension is added to the solution containing<br>
construct-compound (construct-protofibril) complexes, allowing the particles to form<br>
contruct-surface complexes with the unbound anti-protofibril/DNA binding constructs. A<br>
magnet is applied to the solution and the magnetic particle are separated, thus separating<br>
unbound binding constructs and leaving behind the construct-compound complexes in the<br>
solution. Two microliters of the resulting solution (containing construct-compound<br>
complexes) are added to a PCR reaction solution containing primers, free nucleotides, and<br>
Tag DNA polymerase. The DNA portion of any binding constructs present as construct-<br>
compound complexes in solution is amplified. The PCR reaction is resolved on a 1% agarose<br>
gel and visualized by ethidium bromide staining. The presence or absence of amyloid beta<br>
protofibrils in the sample is indicated by the presence or absence, respectively, in the gel of<br>
bands corresponding to the correct molecular weight for the amplified DNA fragment.<br>
Legionella pneuntophilia Antigen:In this example, the compound of interest is a Legionella<br>
pneuntophilia serogroup 1 carbohydrate antigen L pneuntophilia is the causative agent of<br>
the community acquired pneumonia known as Legionnaire's disease, and of pontiac fever.<br>
Serogroup 1 is responsible for the majority of cases of Legionnaire's disease. The sample is<br>
preferably a urine sample from a patient suspected of having a L- pneuntophilia infection.<br>
The binding construct includes (1) a recognition portion, which is an antibody or an antibody<br>
fragment that recognises the L pneuntophilia serogroup 1 carbohydrate antigen ("antigen"),<br>
and (2) a nucleic acid portion that does not exist in nature. The nucleic acid portion is a DNA<br>
template that can be recognized by synthetic primers designed to bind to the template at low<br>
stringency for amplification using PCR. The binding construct is added to a concentration of<br>
10 nanograms per milliliter with a 100 microliter sample of urine, and incubated for a<br>
sufficient time to allow the antibody or antibody fragment to bind to L. pneuntophilia<br>
serogroup 1 carbohydrate antigen present in the sampte, thus forming construct-compound<br>
(construct-antigen) complexes in solution. The binding surfaces are magnetic particles<br>
labelled with an accessible binding target, a peptide mimotope that is capable of binding to<br>
the variable regions of the antibody or antibody fragment (the recognition portion of the<br>
bidding construct). A suitable peptide mimotope can be obtained by methods known in the<br>
art, such as phage display of a random or non-random peptide library or combinatorial<br>
peptide synthesis, followed, by affinity selection (Smith &amp; Petrenko (1997) Chem. Rev.,<br>
97:391-410; Kramer et al. (1993) PeptideRes., 6:314-319). The magnetic particles bear<br><br>
WO 2005/021731	PCT/US2004/028267<br>
32<br>
primary amine reactive groups, allowing the peptide mimotope to be conjugated to the<br>
particles through its carboxy-terminus carboxyl using a water-soluble carbodiamide such as 1-<br>
ethyl-3-(3-dimethylaminopropyl)carbodiamide (EDC). Ten microliters of a 30% particle<br>
suspension is added to the solution containing construct-compound (construct-L.<br>
pneuntophilia serogroup 1 carbohydrate antigen) complexes, allowing the particles to form<br>
construct-surface complexes with the unbound anti-antigen/DNA binding constructs. A<br>
magnet is applied to the solution and the magnetic particles are separated, thus separating<br>
unbound binding constructs and leaving behind the construct-compound complexes in the<br>
solution. Two microliner of the resulting solution (containing construct -compound<br>
complexes) are added to a PCR reaction solution containing primers free nucleotides, and<br>
Taq DNA polymerase. The DNA portion of any binding constructs, present as construct*<br>
compound complexes in solution is amplified. The PCR reaction is resolved on a 1% agarose<br>
gel and visualized by ethidium bromide staining. The presence or absence of L<br>
pneuntophilia serogroup 1 carbohydrate antigen in the sample is indicated by the presence or<br>
absence, respectively, to the gel of bands corresponding to the correct molecular weight for<br>
the amplified DNA fragment.<br>
Mycobacterium tuberculosis-specific Human Antibodies: In this example, the Compound<br>
of interest is Mycobacterium tuberculosis-specific human IgG specific for a 38 kilodalton<br>
extracellular protein from tuberculosis (anti-MTb IgG) M. tuberculosis is the causative<br>
agent of pulmonary tuberculosis in humans. The sample is preferably serum from a patient<br>
suspected of having a M. tuberculosis infection. The binding construct includes: (1) a<br>
recognition portion, which is a peptide epitope or a mimotope that is recognized and bound<br>
by the variable region of the anti-MTb IgG, and (2) a nucleic acid portion, most preferably a<br>
nucleic acid portion wherein the sequence of the nucleic acid portion does not include a<br>
sequence that is expected to be found in the sample, for example a nucleic acid sequence<br>
from higher plants that is not found in either mammals or bacteria. The recognition portion<br>
can be a mimotope for the anti-MTb IgG or a native peptide sequence derived from the<br>
bacterial protein or peptide that is recognized by the anti-MTb IgG; in either case, the peptide<br>
is capable of binding to a subclass of M .tuberclusis specific human immunoglobulin G and<br>
forming construct-compound (construct-anti-MTb IgG) complexes. The nucleic acid portion<br>
is a DNA Template that can be recognized by synthetic primers designed to bind to the<br>
template at low stringency for amplification using PCR. The binding construct is added to a<br>
concentration of 10 nanograms per milliliter with a 100 microliter sample of serum and<br><br>
WO 2005/021731	PCT/US2004/028267<br>
33<br>
incubated for a sufficient time to allow the recognition portion of the binding construct to<br>
bind to anti-MTb IgG present in the sample, thus forming construct-compound (construct-<br>
anti-MTb IgG) complexes in solution. The binding surfaces are agarose beads to which are<br>
attached an antibody or antibody fragment that is capable of recognizing and binding to the<br>
recognition portion of the binding construct. The agarose beads bear aldehyde reactive<br>
groups which may be crosslinked to an amino group on the antibody's heavy chain using<br>
sodium cyanoborohydride (NaBH3CN). Twenty microliters of a 30% particle suspension is<br>
added to the solution containing construct-compound (construct-anti-MTb IgG) complexes,<br>
allowing the beads to form construct-surface complexes with the unbound peptide DNA<br>
binding constructs. The mixture is placed in an Eppendorf tube and centrifuged to pellet the<br>
agarose beads out of suspension, leaving the construct-compound (construct anti-MTb IgG)<br>
complexes remaining free in solution in the supernatant. Two microliters of the supernatant<br>
solution are added to a PCR reaction solution containing primers, free nucleotides, and Taq<br>
DNA polymerase. The DNA portion of any binding constructs present as construct-<br>
compound complexes in solution is amplified. The PCR reaction is resolved on a 1% agarose<br>
gel and visualized by ethidiuin bromide staining. The presence or absence of Mycobacterium<br>
tuberclusis-specific human immunoglobulin G in the sample is indicated by the presence or<br>
absence, respectively, in the gel of bands corresponding to the correct molecular weight for<br>
the amplified DNA fragment.<br>
	Al1 heading are for the convenience of the reader and should not be used to limit the<br>
meaning of the text that follows the headings unless so specified. Various changes and<br>
departures may be made to the present invention without departing from the spirit and scope<br>
thereof. Accordingly, it is not intended that the invention be limited to that specifically<br>
described in the specification or as illustrated in the drawing, but only as set forth in the<br><br>
WO 2005/021731	PCT/US2004/028267<br>
34<br>
SEQUENCE LISTING<br> Lawton, Robert L.<br> SOLUBLE ANALYTE DETECTION AND APPLICATION<br> 2<br> Patentin version 3-2<br> 1<br> 26<br> DNA<br> artificial sequence<br><br> synthetic construct<br> 1<br>
cetotagagt cqacctgcag gcatgc								26<br> 2<br> 27<br> DNA<br> artificial sequencc<br> synthetic construct<br><br> synthetic construct<br> 2<br>
cattggccgt sgttttacaa cgtcgtg 								27<br><br>
WO 2005/021731	PCT/US2004/028267<br>
35<br>
What is claimed is:<br>
1.	A method for detecting a compound of interest in a sample comprising the steps of:<br>
	a)	providing a binding construct comprising a recognition portion which<br>
		and binds said compound of interest, and a nucleic acid portion;<br>
	b)	mixing said binding construct with said sample to form construct-Compound<br>
		complexs.<br>
	c)	providing one or more surfaces, wherein said surface bears one or more accessible<br>
		binding targets capable of recognizing and binding to said recognition portion of<br>
		said binding construct;<br>
	d)	introducing said one or more surfaces to said mixture of said binding construct<br>
		and said sample in order for said one or more surfaces to form construct-surface<br>
		complexes with any unbound binding constructs;<br>
	e)	separating said construct-surface complexes from said mixture leaving behind said<br>
		consrtruct-compound complexes,<br>
	f)	detecting the presence or absence of said nucleic acid portion of said binding<br>
		construct.<br>
		Wherein the presence of said nucleic acid portion of said binding construct indicates<br>
		the presence of said compound of interest in said sample.<br>
2.	The method of claim 1, wherein said one or more surfaces is selected from the group<br>
consisting of particles, powders, beads, planar surfaces, non-planar surfaces, a tube, a well,<br>
non-porous films, non-porous membranes, porous films, porous membrunes, fibers, filters,<br>
meshes, grids, filters, matrices. gels and combinations thereof<br>
3.	The method of claim 1, wherein said one or more surfaces comprises particles.<br>
4.	The method of claim 3, wherein said particles comprise magnetic particles.<br>
5.	Tte method of claim 4, wherein said step (e) comprises separating said constant-surface<br>
complexes out of said mixture by means of a magnet<br><br>
WO 2005/021731	PCT/US2004/028267<br>
36<br>
6.	The method of cfaim 1. wherein in step (f), said detecting the presence or absence of said<br>
nucleic acid portion of said binding construct comprises amplification of said nucleic acid<br>
portion, hybridization of said nucleic acid portion, enzymatic amplification, detection of a<br>
label, or a combination thereof.<br>
7.	The method of claim 1, wherein, in step (f). said detecting the presence or absence of said<br>
nucleic acid portion of said binding construct comprises amplification of said nucleic acid<br>
7.	The method of claim 7, wherein said amplificalion of said nucleic acid portion comprises<br>
a polymerase chain reaction.<br>
8.	The method of claim 5, wherein, in step (f), said detecting the presence or absence of said<br>
nucleic acid portion of said binding construct comprises amplification of said nucleic acid<br>
portion, hybridization of said nucleic acid portion, enzymatic amplification, detection of a<br>
label or a combination thereof.<br>
9.	The method of claim 5, wherein, in step (f), said detecting the presence or absence of said<br>
nucleic acid portion of said binding construct comprises amplification of said nucleic acid<br>
portion.<br>
10.	The method of claim 9, Wherein said amplification of said nucleic acid portion comprises<br>
a polymerase chain reaction.<br>
11.	The method of claim 1. wherein said recognition portion comprises a receptor<br>
12.	The method of claim 1, wherein said recognition portion comprises an antigen.<br>
13.	The method of claim 1, wherein said recognition portion comprises an antibody or an<br>
antibody fragment<br>
14.	The method of claim 1, wherein said recognition portion comprises a single chain<br>
antibody variable region fragment<br><br>
WO 2005/021731	PCT/US2004/028267<br>
37<br>
I5.	The method of claim 1, wherein said recognition portion comprises a Fab fragment.<br>
16.	The method of claim 15, wherein said Fab fragment is attached to said nucleic acid<br>
portion through the free Sulfhydryl of the Fab fragment.<br>
17.	Th method of dairo 12 whereio said compound of interest comprises an entUwdy<br>
&amp;otfy fragmsm, said recognition -potties of said bwdiiig coretriict tqtiipiisea an<br>
 is recognised bv said compound or interest, nd Said accssibte binding tjjrgds comprise<br>
on antibody or antibody fragment that is capable *&gt;f recognizing and bindmg to said<br>
nion portion of said binding construct<br>
18.	The meitiod of claim I. vtfierctn said nudeic acid portion comprises DNA.<br>
19.	Th? method of claim I, wherein said nucleic odd portion comprises RNA,<br>
20.	The method of claim I r wherein said nucleic flcid porpon comprises $. nucleic sequence<br>
that docs not include a sequence thai is expected to be found in the sample.<br>
21.	The (method of claim. 1., wberdn said step (*1 comprises providing (wo or rnore differcnr<br>
typts <lf bindtftg cqbsw wheteta each afraid two oc toitc difisreni binding constructs has></lf>
a different TecognJ^iort portion m&amp; a djffeiert tmdeac add<br><br>
WO 2005/021731	PCT/US2004/028267<br>
38<br>
22.	A method for increasing the sensitivity of solution-phase detection of a compound of<br>
interest comprising the steps of:<br>
	a)	providing a sample suspected of containing said compound of interest,<br>
	b)	providing a binding construct comprising<br>
		i)	a recognition portion capable of binding said contpound of interest,<br>
			and<br>
		ii)	a nucleic acid portion<br>
	c)	contacting said sample with said binding construct for a period of time sufficient<br>
		to permit said recognition portion to bind said compound of interest present in<br>
		said sample, thereby forming construct-compound complexes in solution;<br>
	d)	Providing one or more surface, wherein said one or more surfaces bears one or<br>
		 more accessible binding target capable of binding to said recognition portion;<br>
	e)	contracting said one or more surfaces with said solution for a period of time<br>
		sufficient for said one or more accessible binding target to bind said recognition<br>
		portion of any binding construct not bound to said compound of interest, thereby<br>
		forming; construct-surface complexes;<br>
	f)	separating said construct-surface complexes from said solution, leaving said<br>
		construct-compound complexes in said solution; and<br>
	g)	detecting the presence or absence of said nucleic acid portion of said binding<br>
		construct in said solution,<br>
		wherein said separation of said construct-surface complexes from said solution<br>
		 results in a separation of substantially all binding constructs not bound to a<br>
		 compound of interest and in an increased sensitivity of detection of said<br>
		compound of interest, and<br>
		wherein the presence of said nucleic acid portion of said binding construct<br>
		indicates the presence of said compound of interest in said sample.<br>
23.	The method of claim 22, wherein said one of more surfaces is selected from the group<br>
consisting of: particles, powders, beads, planar surfaces, non-planar surfaces, a tube, a well,<br>
non-porous films, non-porous membranes, porous films, porous membranes, fibers, fillers,<br>
 meshes, grids, filters, matrices, gels, and combinations thereof.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
39<br>
24.	The method of claim 22, wherein said one or more surfaces comprises particles.<br>
25.	The method of claim 24, wherein said particles comprise magnetic particles.<br>
26.	The method of claim 25, wherein said step (f) comprises separating substantially all said<br>
construct-surface complexes from said solution by means of a magnet<br>
27.	The method of claim 22, wherein, in step (g), said detecting the presence or absence of<br>
said nucleic acid portion of said binding construct comprises amplification of said nucleic<br>
acid portion, hybridization of said nucleic acid portion, enzymatic amplification, detection of<br>
a label, or a combination thereof<br>
28.	The method of claim 22, wherein, in step (g), said detecting the presence or absence of<br>
said nucleic acid portion of said binding construct comprises amplification of said nucleic<br>
acid portion.<br>
29.	The method of claim 23, wherein said amplification of said nucleic acid portion<br>
comprises a polymerase chain reaction<br>
30.	The method of claim 26, wherein, in step (g), said detecting the presence or absence of<br>
said nucleic acid portion of said binding construct comprises amplification of said nucleic<br>
acid portion, hybridization of said nucleic acid portion, enzymatic amplification, detection of<br>
a label, or a combination thereof.<br>
31.	The method of daim 26. wherein, in step (g), said detecting the presmefl or absence of<br>
said nucleic acid portion of sad binding construct comprises amplification of said nucleic<br>
acid portion<br>
32.	The method of claim 31, wherein said amplification of said nucleic acid portion<br>
comprises a polymerase chain reaction.<br>
33.	The method of claim 22, wherein said recognition portion comprises a receptor<br>
34.	The method of claim 22, wherein said recognition portion comprises an antigen.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
40<br>
35.	The method of claim 22, wherein said recognition portion comprises an antibody of<br>
antibody fragment.<br>
36.	The method of claim 22. wherein said recognition portion comprises a single chain<br>
antibody variable region fragment.<br>
37.	The method of claim 22, wherein said recognition portion comprises a Fab fragment<br>
38.	The method of claim 37, wherein said Fab fragment is attached to said nucleic acid<br>
portion through the free sulfhydryl of the Fab fragment.<br>
39.	The method of claim 34, wherein said compound of interest comprises an antibody or<br>
antibody fragment, said recognition portion of said binding construct comprises an antigen<br>
that is recognized by said compound of interest, and said accessible binding targets comprise<br>
an antibody or antibody fragment that is capable of recognizing and binding to said<br>
recognition portion of said binding construct<br>
38.	The method of claim 21, wherein said nucleic acid portion comprises DNA.<br>
39.	The method of claim 21, wherein said nucleic acid portion comprises RNA<br>
40.	The method of claim 21, wherein said nucleic acid portion comprises a nucleic sequence<br>
that does not include a sequence that is expected to be found in the sample.<br>
41.	The method of claim 21, wherein said step (b) comprises providing two or more different<br>
types of binding constructs, wherein each of said two or more different binding constructs has<br>
a different recognition portion and a different nucleic acid portion.<br><br>
WO 2005/021731	PCT/US2004/028267<br>
41<br>
42.	A kit for detecting a compound of interest in a sample suspected of containing said<br>
compound of interest comprising:<br>
	a)	a binding construct comprising<br>
		(i)	a recognition portion which recognizes and binds said compound of interest,<br>
				and<br>
		(ii)	a nucleic acid portion;<br>
	and<br>
	b)	one or more surfaces bearing one or more accessible binding targets capable of<br>
		binding to said recognition portion of said binding construct.<br>
43.	The kit of claim 42, further comprising a nucleic atid amplification primer pair, wherein<br>
each primer of said primer pair is capable of hybridizing to its complementary sequence at<br>
the 3' end of a target nucleic acid sequencc of said nucleic acid portion.<br><br>
The present invention relates to a method of detection of a compound of interest that is present at low level in<br>
a sample. In particular, the present invention relates to a method of detection of a compound of interest in solution by a nucleic<br>
acid-labelled binding construct separation of the unbound nucleic acid-labelled binding construct.and the XXX of the bound<br>
nucleic acid-labelled binding construct in the solution phase.The present invention particularly XXX to be used in XXX<br>
with a nucleic acid amplification reaction for detecting the presence or absence of the nucleic acid portion of binding construct in a<br>
sample indicating the presence or absence of the compound of interest.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1wY3QgcmVxdWVzdC5wZGY=" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-pct request.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA1NTcta29sbnAtMjAwNi1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">00557-kolnp-2006-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQUJTVFJBQ1QuMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-ABSTRACT.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQU1BTkRFRCBQQUdFIE9GIFNQRUNJRklDQVJUSU9OLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-AMANDED PAGE OF SPECIFICARTION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQVNTSUdOTUVOVC5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-ASSIGNMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ0xBSU1TLjEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CLAIMS.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ09SUkVTUE9OREVOQ0UgMS4yLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CORRESPONDENCE 1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ09SUkVTUE9OREVOQ0UtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CORRESPONDENCE-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ09SUkVTUE9OREVOQ0UuMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CORRESPONDENCE.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtREVTQ1JJUFRJT04gKENPTVBMRVRFKS4xLjEucGRm" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-DESCRIPTION (COMPLETE).1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRFJBV0lOR1MuMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-DRAWINGS.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0lFVkVELnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-EXAMINATION REPORT REPLY RECIEVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAxIDEuMi5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 1 1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAxLjEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 1.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAyLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 2-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAzLjEuMi5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 3.1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSAzLjEuMy5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 3.1.3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSA1LjEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 5.1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSA2LTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 6-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtRk9STSA2LTEuMi5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-FORM 6-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LWtvbG5wLTIwMDYtZm9ybSA2LnBkZg==" target="_blank" style="word-wrap:break-word;">557-kolnp-2006-form 6.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtT1RIRVJTIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-OTHERS 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtT1RIRVJTIERPQ1VNRU5UUy5wZGY=" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-OTHERS DOCUMENTS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtUEEtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-PA-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtUEEucGRm" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-PA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzctMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-PETITION UNDER RULE 137-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTU3LUtPTE5QLTIwMDYtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcucGRm" target="_blank" style="word-wrap:break-word;">557-KOLNP-2006-PETITION UNDER RULE 137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251088-method-for-the-manufacture-of-a-green-tea-product.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251090-compositin-for-making-breathable-films.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251089</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>557/KOLNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Mar-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NEW ENGLAND RARE REAGENTS, LLC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>3 BURNHAM ROAD, GORHAM, MAINE 04038 U.S.A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LAWTON, ROBERT, L.</td>
											<td>3 BURNHAM ROAD, GORHAM, ME 04038 U.S.A.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/028267</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-08-31</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/653,321</td>
									<td>2003-09-02</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251089-soluble-analyte-detection-and-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:48 GMT -->
</html>
