<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251012-nucleic-acids-and-methods-for-producing-seeds-having-a-full-diploid-complement-of-the-maternal-genome-in-he-embryo by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:07:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251012:NUCLEIC ACIDS AND METHODS FOR PRODUCING SEEDS HAVING A FULL DIPLOID COMPLEMENT OF THE MATERNAL GENOME IN HE EMBRYO</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">NUCLEIC ACIDS AND METHODS FOR PRODUCING SEEDS HAVING A FULL DIPLOID COMPLEMENT OF THE MATERNAL GENOME IN HE EMBRYO</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to DYAD genes, mutants thereof, and use of them for making plants that retain heterozygosity of the female parent plant. The invention also encompasses plants, plant tissues, and seeds of plants that have a dyad phenotype and so retain heterozygosity of the female parent, either constitutively or conditionally. The invention is useful for propagating desired hybrid phenotypes in a manner of an apomictic plant and for increasing the ploidy of a plant genotype, which may result in plants having increased biomass.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention:<br>
The present invention relates to the use of alleles of the DYAD gene and gene product of Arabidopsis, Boechera, rice and other plants to manipulate gametogenesis and seed development for the purpose of producing seeds that carry a full diploid complement of the maternal genome in the embryo. The present invention also relates to use of an altered DYAD gene for producing an unreduced female gametophyte without substantial effect on pollen development.<br>
Background and prior art of the invention:<br>
The plant life cycle alternates between a diploid sporophyte generation and a haploid gametophyte generation. Meiosis represents the transition between the diploid sporophyte and haploid gametophyte phases of the plant life cycle. Meiosis leads to the formation of haploid spores. In plants, unlike animals, the meiotic products undergo additional divisions to form a multicellular haploid gametophyte. Differentiation of the gametes occurs towards the later stages of gametophyte development, following division of the meiotic products. The sexual process prior to fertilization therefore comprises two distinct stages: sporogenesis which includes meiosis and the formation of haploid spores; and gametogenesis which refers to the development of the spores into a gametophyte, comprising the gamete and associated cells required for fertilization and for supporting growth of the embryo.<br>
Most plant species undergo sexual reproduction; however some plant species are capable of asexual reproduction. The term apomixis is generally accepted as the replacement of sexual reproduction by any of certain forms of asexual reproduction (Koltunow A. and Grossniklauss U. Annu. Rev. Plant Biol. Vol. 54: 547-74, 2003). Apomixis is a genetically controlled method of reproduction in plants, involving seed formation in which the embryo is formed without union of an egg and a sperm. There are three basic types of apomictic reproduction: 1) apospory, in which the embryo develops parthenogenetically from a chromosomally unreduced egg in an embryo sac derived from the nucellus, 2) diplospory, in which an embryo develops parthenogenetically from an unreduced egg in an embryo sac derived from the megaspore mother cell, and 3) adventitious embryony, in which an embryo develops directly from a somatic cell. The first two types of apomixis are together classified under gametophytic apomixis because in both cases the embryo develops from a female gametophyte or embryo sac, whereas in adventitious embryony the embryo develops directly from a somatic cell without an intermediate female gametophyte stage. Gametophytic apomixis therefore involves two components: i) apomeiosis, or the production of an unreduced female gametophyte (embryo sac) that retains the parental genotype, and ii)<br>
parthenogenetic development of the embryo, with or without fertilization of the central cell which develops into the endosperm.<br>
Apomixis is thus a reproductive process that bypasses female meiosis and syngamy to produce embryos genetically identical to the maternal parent. The three types of apomixis have economic potential because they can cause any genotype, regardless of how heterozygous, to breed true. With apomictic reproduction, progeny of especially adaptive or hybrid genotypes would maintain their genotype throughout repeated life cycles. In addition to fixing hybrid vigour, apomixis can make possible commercial hybrid production in crops where efficient male sterility or fertility restoration systems for producing hybrids are not known or developed. Apomixis can therefore make hybrid development more efficient. Apomixis also simplifies hybrid production and increases<br>
genetic diversity in plant species with good male sterility systems. It would be highly desirable to introduce genes controlling obligate or a high level of apomixis into cultivated species and to be able to readily hybridize cross-compatible sexual and apomictic genotypes to produce true-breeding Fl hybrids. The transfer of apomixis to important crops would make possible development of true-breeding hybrids and commercial production of hybrids without a need for cytoplasmic-nuclear male sterility and high cost, labor-intensive production processes. An obligately apomictic Fl hybrid would breed true through the seed indefinitely and could be considered to provide a vegetative or clonal method of reproduction through the seed. The development of apomictically reproducing cultivated crops would also provide a major contribution toward the food security in developing nations (Spillane C, Steimer A, and Grossniklaus U, Sex. Plant Reprod.  14:179-187,2001).<br>
In reality, most known genes controlling apomixis are found in the wild species, which are distantly related to the cultivated species. Although interspecific crosses may be possible between the cultivated and wild species, chromosome pairing between genomes is usually low or nonexistent, leading to failure of this approach.<br>
Brief description of the drawings:<br>
Figure 1 represents reduced seed set in dyad mutants plants. The modal range is 1-10 seed per plant.<br>
Figure 2 represents normal pollen viability in dyad mutant plants using Alexander staining. (Figure 2A) Wild type. (Figure 2B) dyad.<br>
Figure 3 represents male and female meiosis in wild type and the dyad mutant. (Figure 3A-C) Wild type. (Figure 3D-F) dyad. (Figure 3A, D) Male meiocytes at the end of meiosis 1 (telophase). (Figure 3B, E) Male meiocyte at the tetrad stage. (Figure 3C, F) Female meiocyte at anaphase 1. dyad undergoes an equational female meiosis.<br>
Figure 4 represents chromosome ploidy of representative progeny of a diploid dyad mutant plant. (Figure 4A) Somatic cell of a triploid progeny plant showing 15 chromosomes. (Figure 4B) Male meiosis 1 in a triploid progeny plant carrying 15 chromosomes showing 9:6 segregation. (Figure 4C) Somatic cell of a diploid progeny plant showing 10 chromosomes.<br>
Figure 5 represents complementation of the dyad mutant by the Boechera holboelli DYAD homologue: (Figure 5A) dyad mutant showing unelongated siliques. (Figure 5B) dyad mutant transformed with the BhDYAD gene showing elongated siliques containing seeds. (Figure 5C) Comparison of siliques from a dyad mutant plant (1), a complemented plant (2) and a wild type plant (3). (Figure 5D) Dissected silique from a complemented plant, showing full seed set. (Figure 5E) Dissected silique from a wild type plant.<br>
Figure 6 is a diagram showing the pB1101.3::Dyad::(A)GR cassette used to construct a DYAD conditional complementation line.<br>
Figure 7 is a polyacralymide gel showing CAPS polymorphism for genotyping the endogenous locus of DYAD as described in Example 6. Figure 7A: Resolved HinF1<br>
digested fragments from KNEF/KNER primers amplified products. Figure 7B: Resolved HinFldigested fragments from KKF/KKR primers amplified products.<br>
Figure 8 illustrates the conditional complementation of the dyad phenotype in Example 6.<br>
Figure 8A: Inflorescence showing non-elongated silique (dyad phenotype) before and after<br>
dexamethasone treatment. The arrow indicates the position of the youngest open flower at the<br>
start of treatment. 5-7 days after the start of treatment siliques showed elongation (wild type<br>
phenotype). Figure 8B: Isolated siliques showing sterile (dyad) phenotype before dexamethasone<br>
treatment. Figure 8C: shows restored wild type phenotype after conditional complementation by<br>
dexamethasone  treatment.	Figure   8D:   Split  open   silique   showing  full   seed   set  after<br>
dexamethasone treatment.<br>
Figure 9 shows the morphology of the ovule after conditional complementation of dyad phenotype in Example 6.<br>
Figure 9A: Cleared ovule showing dyad phenotype and absence of embryo sac at the mature ovule stage before dexamethasone treatment. Figure 9B: Embryo sac restored after dexamethasone treatment.<br>
Figure 10 shows the variation in size of seeds produced by the dyad mutant and differences in size of seeds obtained from reciprocal crosses between diploid and tetraploid Arabidopsis strains. Figure 10A: Seeds from selfed wild type diploid Col-0 plants are uniformly normal in size. Figure 10B: Seeds from a tetraploid plant. Figure IOC: Size of seeds from selfed dyad plants varies between large (L), normal (N), and shrunken (S). Figure 10D: Maternal excess — seeds from a tetraploid female crossed to a diploid male are shrunken. Figure 10E: Paternal excess — seeds from a tetraploid male crossed to a diploid female are larger in size when compared to seeds from a maternal excess cross.<br>
Figure 11 shows an alignment of the protein sequences of the DYAD protein from Arabidopsis (SEQ ID NO: 5), Boechera (SEQ ID NO: 18), rice (SEQ ID NO: 51), and from poplar (Populus trichocarpa) (SEQ ID NO: 26), using Clustal W as in http://www.ebi.ac.uk/clustalw with default parameters.<br>
Figure 12 shows alignment of the rice DYAD polypeptide sequences (SEQ ID NO: 51) with putative maize DYAD polypeptide sequences (SEQ ID NOS: 55 and 54) using Clustal W (1.82). Figure 12A: Alignment of rice DYAD amino acids 1-147. Figure 12B: Alignment of rice DYAD amino acids 317-803.<br>
Figure 13 shows mapping of the DYAD polypeptide sequence from rice onto two Zea mays contigs identified as comprising DYAD-encoding sequences.<br>
Detailed description of the invention:<br>
There are two general strategies that may be considered in order to introduce apomixis into cultivated crops. The first is by introgression from wild relatives into cultivated species. The second is by identification of genes from sexual species that can confer aspects of apomixis, followed by pyramiding these genes to produce the full repertoire of apomixis. These genes could then be introduced into cultivated crops using transgenic methods. Thus for instance, expression of one or more genes could be used to engineer apomeiosis, and these genes could be combined<br>
with another set of genes or other treatments to induce parthenogenetic embryo development. Methods for inducing parthenogenesis in plants are known in the art (See, e.g. US Pat. No.5,840,567). A preferred method for inducing parthenogenetic development for use with the present invention is to pollinate a plant using pollen that has been irradiated, thereby inactivating it for fertilization. (Pandey K.K.. and Phung M, Theoret. Appl. Genet., Vol. 62:295-300, 1982; Lofti M. et al., Plant Cell Reprod., Vol. 21:1121-1128, 2003).<br>
This method is preferred in that it has been used in a number of plant species and appears to be generally applicable, most easily to plants having incomplete flowers (monoecious and dioecious). However, it can be applied to hermaphroditic plants having complete flowers that have been made male-sterile or from which the fertile pollen has been mechanically removed or segregated.<br>
The specific dose of radiation for sterilizing the pollen will vary depending upon the particulars of the species. In general, a dose of about 10 to 2000 Gray is sufficient. Preferably, the dose is about 100 to 500 Gray, more preferably from 200 to 250 Gray.<br>
Successful induction of parthenogenesis can be detected by screening of seeds for the presence of embryos, for instance by dissection or by observation of the seeds on a light box after culture in liquid medium as described by Lofti M. et al., Plant Cell Reprod., Vol. 21: 1121-1128, 2003. Introducing the apomictic trait into normally sexual crops has been attempted. Asker S. (Hereditas, Vol. 91: 231-241, 1979) reports that attempts have been unsuccessful with wheat, sugar beets, and maize. PCT publication WO 89/00810 (Maxon et al, 1989) discloses inducing an apomictic form of reproduction in cultivated plants using extracts from nondomesticated sterile alfalfa plants. When induction of male sterility was evaluated in sorghum, sunflower, pearl millet, and tomato it was reported that there was reduced seed set in sorghum, pearl millet, and sunflower and reduced fruit set in tomato.<br>
Although apomixis is effectively used in Citrus to produce uniform and disease-and virus-free rootstock (Parlevliet J. E. et al.. in Citrus. Proc. Am. Soc. Hort. Sci., Vol. 74: 252-260, 1959) and in buffelgrass (Bashaw, Crop Science, Vol. 20: 112, 1980) and Poa (Pepin et al., Crop Science, Vol. 11: 445-448, 1971) to produce improved cultivars, it has not been successfully transferred to a cultivated crop plant.<br>
The second approach towards engineering apomixis involves the identification and manipulation of apomixis related genes from sexual species. A developmental view of apomixis has suggested that apomixis is related to sexual reproduction and involves the action of genes that also play a role in the sexual pathway (Tucker M.R. et al., Plant Cell, Vol. 15(7): 1524-1537, 2003). In sexual reproduction, usually a megaspore mother cell arising from the hypodermal layer towards the<br>
apex of the developing ovule enlarges and goes through meiosis and two cell divisions to form a linear tetrad of megaspores each with a haploid chromosome number. Most commonly among different plant species, the three most apical spores degenerate while the functional chalazal spore undergoes three rounds of nuclear division accompanied by cell expansion to form an embryo sac with an egg, two polar nuclei, two synergids, and three antipodal cells. Apomixis is a process that requires multiple steps and the control of the complete pathway of apomixis as has been shown in certain species to require the action of multiple genes (van Dijk et al., Heredity, Vol. 83: 715-721,1999; Matzk F., et al., Plant Cell, 17(1): 13-24, 2005). It has been considered that individual component steps controlled by one or a subset of genes in the pathway operating in isolation would have a negative effect on fertility (Spillane , C, Steimer A. and Grossniklaus U., Sex. Plant Reprod. Vol. 14: 179-87, 2001), and that it is only the concerted action of the complete set of genes comprising the entire pathway that is able to efficiently promote apomixis. Genetic and molecular analysis of Arabidopsis mutants has led to the identification of a number of genes that play a role in stages of sporogenesis and gametogenesis (Yang W. C. and Sundaresan V., Curr. Opin. Plant Biol. Vol. 3(1): 53-57, 2000). The dyad mutant of Arabidopsis was identified as causing female sterility (Siddiqi I. et al., Development, Vol. 127(1): 197-207, 2000) and its analysis showed that dyad mutant plants are defective in female meiosis. The majority of female meiocytes in the dyad mutant undergo single division meiosis to give two cells instead of four, followed by an arrest in further stages of development including gametogenesis. Male meiosis, pollen development, and male fertility in the dyad mutant was found to be normal (Siddiqi I. et al., Development, Vol. 127(1): 197-207, 2000; Reddy T. V., et al., Development, Vol. 130 (24):5975-5987, 2003). Analysis of meiotic chromosomes during female meiosis indicated that homologous chromosomes do not undergo synapsis and that the reductional meiosis 1 division is replaced by an equational one (Agashe B., Prasad C. K ., and Siddiqi I., Development, Vol. 129(16), 3935-3943, 2002). An independent study has led to identification of the SWI1 gene (Motamayor J. C, et al., Sex. Plant Reprod. Vol. 12:209-218, 2000; Mercier R., et al., Genes and Dev. Vol. 15: 1859-1871, 2001), which is identical to DYAD. The gene identified by these studies is hereafter referred to as the DYAD gene. The wild type DYAD gene from Arabidopsis encodes a protein of 639 amino acids (SEQ ID NO:5). Three alleles of the DYAD gene in Arabidopsis have been described. These are: i) dyad, having a truncation at amino acid 508; the resulting protein is therefore missing the C-terminal 130 amino acids present in the wild type protein; ii) swil.l which results in production of reduced amounts of the wild type protein causing some female meiocytes to undergo an equational meiosis 1 division whereas others undergo a reductional division: and iii) swil.2 which creates a stop codon at position 394 and<br>
causes a female phenotype similar to dyad but in addition also causes defects in male meiosis resulting in male sterility. The position corresponding to the dyad allele in Boechera would be a mutation that causes a frameshift at position 508 of the amino acid sequence and results in a stop codon after ten additional codons (i.e. position 518). The corresponding positions in rice are at 563 and 572, respectively.<br>
Without being bound by any theory of the invention, the inventors suggest that a reduction in the amount of DYAD protein having the portion of the polypeptide carboxy-terminal to position 394 (in Arabidopsis, and corresponding positions in other species) produces a phenotype in which female meiocytes undergo an equational meiosis 1 division, resulting in retention of the female genotype (and hence heterozygosity) in female gametes. Retention of a normal (or approximately so) amount of the DYAD protein having the domain from position 394 to position 508 (in Arabidopsis and corresponding positions in other species) provides for normal pollen development, whereas elimination of this domain in the plant produces a male sterile phenotype. Prior to the making of the present invention, plants homozygous for the dyad or swil.2 alleles had not been reported to show seed set. Plants carrying the swil. 1 allele have been reported to show reduced seed set when homozygous but the seeds that are produced have been analyzed with respect to their chromosomal constitution and found to be diploid, thereby showing that the seeds arise from a normal megasporogenesis and megagametogenesis (Motamayor J. C, et al., Sex. Plant Reprod. Vol. 12:209-218. 2000). As described previously, the spores produced as a result of the equational, single division meiosis in dyad, swil.l, and swil.2 remain arrested and until the making of the present invention, it was not known whether any of these had the potential to develop into female gametes. It was also not known until the making of the present invention whether the chromosomes experienced recombination during the equational single division female meiosis and as a result the products of division lost parental heterozygosity. The plausibility of recombination accompanying an equational division is supported by studies in yeast which demonstrate that diploid cells can enter meiosis, experience meiotic recombination, then withdraw from meiosis upon transfer to growth medium and divide mitotically. Such a mitotic division can lead to loss of heterozygosity for a genetic marker if recombination has taken place between the gene and the centromere (Esposito R. E. and Esposito M. S., Proc. Natl. Acad. Sci. USA Vol. 71(8): 3172-3176 1974). The present invention relates to the finding that the products of the equational meiosis 1 division seen in different dyad homozygous mutant plants are capable of giving rise to a functional unreduced embryo sac, which has the characteristic features of apomeiosis, an important component of apomixis.<br>
Accordingly the present invention relates to the use of the DYAD gene, especially mutant alleles thereof, and their gene products, of Arabidopsis, Boechera, Rice, Populus and other plants to manipulate gametogenesis and seed development to produce seeds whose embryonic genotype contains a full diploid complement of the maternal genome.<br>
In an embodiment of the invention triploid seeds are produced in Arabidopsis and other plant types.<br>
In another embodiment of the invention, a method for the production of a heterotic plant using mutant alleles of the DYAD gene and gene product is provided by invention.<br>
In yet another embodiment invention, the plants and seed contain a full diploid complement of the maternal genome, and no contribution from the paternal genome, and thus represent true apomicts.<br>
In still another embodiment of the invention, the plant contributing the maternal genome is a hybrid having an assortment of alleles having a desirable phenotype, and the method of the invention allows for fixation and easy propagation of that combination of alleles.<br>
In an embodiment of the invention ,the use of the DYAD gene and its gene product which leads to<br>
the formation of seeds containing a full diploid complement of the maternal genome.<br>
In another embodiment of the invention, DYAD gene is used in making triploid plants which can<br>
be used for producing seedless fruit, for constructing trisomic lines for mapping studies, and for<br>
maintainance of heterozygosity of the parent plant and apomixis. The alleles of DYAD used in the<br>
present invention cause formation of an unreduced (diploid) embryo sac.<br>
In yet another embodiment of the invention, the use of the DYAD gene for causing formation of<br>
an unreduced embryo sac without substantially affecting pollen development.<br>
In still another embodiment of the invention, the use of the DYAD gene for producing higher<br>
order polyploids by selfing of triploids, which would be useful for the purpose of generating<br>
plants with increased biomass.<br>
It should be understood that various embodiments of the invention will exhibit different aspects<br>
of the invention, and may provide different advantages of the invention.  Not every embodiment<br>
will enjoy all of the advantages of the invention<br>
Definitions:<br>
genetic code. In such cases, the nucleic acids typically hybridize under moderately stringent<br>
hybridization conditions.<br>
In the present invention, genomic DNA or cDNA comprising DYAD nucleic acids to be used in<br>
the invention can be identified in standard Southern blots under stringent conditions using the<br>
nucleic acid sequences disclosed here. For the purposes of this disclosure, suitable stringent<br>
conditions for such hybridizations are those which include a hybridization in a buffer of 40%<br>
formamide, 1M NaCl, 1% SDS at 37 °C, and at least one wash in 0.1X to IX SSC, preferably<br>
0.5X SSC, more preferably 0.2X SSC at a temperature of at least about 50 °C, usually about 55<br>
°C, up to about 60 °C, for 20 minutes, or equivalent conditions. A positive hybridization is at<br>
least  twice   background.   Those   of ordinary   skill   will   readily   recognize   that   alternative<br>
hybridization and wash conditions can be utilized to provide conditions of similar stringency.<br>
A further indication that two polynucleotides are substantially identical is if the reference<br>
sequence, amplified by a pair of oligonucleotide primers, can then be used as a probe under<br>
stringent hybridization conditions to isolate the test sequence from a cDNA or genomic library, or<br>
to identify the test sequence in, e.g., a northern or Southern blot.<br>
A "plant hybrid" is defined as a plant obtained by crossing two cultivars of the same plant<br>
species.<br>
An "interspecific hybrid" is defined as a plant obtained by crossing two plants of different<br>
species.<br>
A "female parent" in a reproductive event is defined as the plant which bears the seed.<br>
The present invention provides the DYAD gene and its product and methods involving the application of molecular genetic approaches for the control of seed development and apomixis. The invention further relates to mutant alleles of the DYAD gene that express a truncated form of the DYAD polypeptide lacking the C-terminal portion of the native protein, and causes the development of an unreduced female gametophyte while at the same time leaving pollen development substantially unaltered as determined by pollen viability assays and microscopic examination of chromosome segregation in male meiosis. It also relates to nucleotide sequences for a female specific mutant allele of the DYAD gene, that encodes a DYAD polypeptide lacking a C-terminal portion of the native DYAD polypeptide, and such that expression of the mutant polypeptide in plants specifically leads to unreduced female gametophyte development but does not substantially affect pollen development. Such a mutant allele would express a DYAD polypeptide that, for example in the instance of a mutant allele from Arabidopsis, lacks all or part<br>
of the portion of the native polypeptide sequences between amino acid 509 and amino acid 639 in SEQ ID NO:5 but does contain all the region encoding polypeptide sequences up to amino acid 394. Further it also provides the nucleotide sequences that hybridize under stringent conditions to the sequence given in SEQ ID NO: 4 and which encode C-terminal deletion derivatives of native DYAD polypeptides wherein the deletion corresponds to a region between amino acid 509 and 639 in SEQ ID NO:5 as determined by comparison with SEQ ID NO:5 using a comparison window. Corresponding portions of Boechera, Rice, and Populus DYAD proteins can be identified by reference to Figure 11. Compositions of the invention also comprise C-terminal deletion derivatives of native DYAD polypeptide sequences, and fusion proteins and the nucleic acids that encode them, formed from the above DYAD polypeptides and protein sequences, such as glucocorticoid hormone receptor proteins, that conditionally transport the fusion protein into the nucleus of a plant cell.<br>
The methods of the invention comprise expression of DYAD polynucleotide sequences in plants to produce unreduced female gametes that retain the genotype of the parent. Production of such unreduced female gametes is useful for engineering apomixis and for fixing heterosis, as well as for production of triploid plants. In one embodiment of the invention a DYAD polynucleotide sequence may be introduced into the genome of a plant by any of several well known methods for transformation wherein it is expressed in the plant as antisense or as double-stranded RNA thereby leading to the inhibition of the endogenous DYAD gene and causing production of unreduced female gametes. In another embodiment of the invention a C-terminal deletion of DYAD polynucleotide sequences is introduced into the genome of a plant as part of an expression cassette and leads to the formation of unreduced female gametophytes, while at the same time leaving the development of pollen substantially unaffected. The expression of DYAD polynucleotide sequences in plants leading to unreduced female gametophyte formation can then be used to generate apomictic seeds by parthenogenetic development of the egg cell into an embryo. The expression of such DYAD polynucleotide sequences in plant hybrids leads to the formation of unreduced female gametes that retain the genotype of the parent thereby leading to the fixation of heterosis in the next generation. Fixation of heterosis is very useful as it would allow the multiplication of hybrid seeds by selfing without having to resort to crosses between two parent cultivars of differing genotype.<br>
Still another embodiment of the invention is the expression of DYAD polynucleotide sequences in interspecific hybrids of plant species leading to the formation of an unreduced female gamete, which can be used for generating apomictic seed. The generation of such apomictic seeds is useful for introgressing agronomically useful genes from one plant species into another species.<br>
Yet another embodiment of the invention involves conditional or controlled expression of DYAD polynucleotide sequences or DYAD polypeptide sequences and/or the activities thereof. Such conditional expression may be used to promote the generation of unreduced female gametes and hence apomictic seeds only when desired. Methods for effecting conditional expression or activity of polynucleotide and polypeptide sequences in plants are well known in the art and include but are not limited to ethanol inducible gene expression (Devaux et al., Plant J., Vol. 36(6): 918-930 ,2003), steroid hormone inducible control of activity (Schena M., Lloyd A. M. and Davis R. W., Proc. Natl. Acad. Sci. USA Vol. 88(23): 10421-10425, 1991), and Tetracycline mediated control of expression (Bohner S. et al., Plant J. Vol. 9(1): 87-95, 1999). Example 6 below describes one embodiment of the invention wherein a homogenous population of plants showing the dyad mutant phenotype may be developed. The same may be accomplished by employing conditional DYAD RNAi or antisense in which the DYAD RNAi or antisense construct is expressed under control of a conditional promoter. Another manifestation of the invention is one in which a complementing copy of the DYAD gene is expressed in a plant under control of a conditional promoter, in a genetic background that is homozygous for a mutant allele of dyad. Still another manifestation of the invention would employ crossing a first plant carrying a DYAD RNAi or antisense construct expressed under control of a promoter that is expressed under control of a transactivator and wherein the first plant lacks the transactivator, to a second plant that expresses the transactivator.<br>
The isolated sequences prepared as described herein can be used in a number of techniques, for example, to suppress or alter endogenous DYAD gene expression. Modulation of DYAD gene expression or DYAD activity in plants is particularly useful, for example as part of a system to generate apomictic seed.<br>
Isolation of DYAD Nucleic Acids<br>
Generally, the nomenclature and the laboratory procedures in recombinant DNA technology<br>
described below are those well known and commonly employed in the art. Standard techniques<br>
are used for cloning, DNA and RNA isolation, amplification and purification. Generally<br>
enzymatic reactions involving DNA ligase, DNA polymerase, restriction endonucleases and the<br>
like are performed according to the manufacturer's specifications. These techniques and various<br>
other techniques are generally performed according to Sambrook et al., Molecular Cloning—A<br>
Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y., (1989).<br>
The isolation of DYAD nucleic acids may be accomplished by a number of techniques. For<br>
instance, oligonucleotide probes based on the sequences disclosed here can be used to identify the<br>
desired gene in a cDNA or genomic DNA library. To construct genomic libraries, large segments of genomic DNA are generated by random fragmentation, e.g. using restriction endonucleases, and are ligated with vector DNA to form concatemers that can be packaged into the appropriate vector. To prepare a cDNA library, mRNA is isolated from the desired organ, such as ovules, and a cDNA library, which contains the DYAD gene transcript, is prepared from the mRNA. Alternatively, cDNA may be prepared from mRNA extracted from other tissues in which DYAD genes or homologs are expressed.<br>
The cDNA or genomic library can then be screened using a probe based upon the sequence of a cloned DYAD gene disclosed here. Probes may be used to hybridize with genomic DNA or cDNA sequences to isolate homologous genes in the same or different plant species. Alternatively, antibodies raised against a DYAD polypeptide can be used to screen an mRNA expression library.<br>
Alternatively, the nucleic acids of interest can be amplified from nucleic acid samples using amplification techniques. For instance, polymerase chain reaction (PCR) technology can be used to amplify the sequences of the DYAD genes directly from genomic DNA, from cDNA, from genomic libraries or cDNA libraries. PCR and other in vitro amplification methods may also be useful, for example, to clone nucleic acid sequences that code for proteins to be expressed, to make nucleic acids to use as probes for detecting the presence of the desired mRNA in samples, for nucleic acid sequencing, or for other purposes. For a general overview of PCR see PCR Protocols: A Guide to Methods and Applications. (Innis, M, Gelfand, D., Sninsky, J. and White, T., eds.), Academic Press, San Diego (1990).<br>
Appropriate primers and probes for identifying DYAD sequences from plant tissues are generated from comparisons of the sequences provided here with other DYAD related genes or the proteins they encode. For instance, Boechera holboelli DYAD can be compared to the closely related gene from rice (Genbank ID No. 50917243). Using these techniques, one of skill can identify conserved regions in the genes or polypeptides disclosed here to prepare the appropriate primer and probe sequences. Primers that specifically hybridize to conserved regions in DYAD related genes can be used to amplify sequences from widely divergent plant species. Standard nucleic acid hybridization techniques using the conditions disclosed above can then be used to identify full length cDNA or genomic clones.<br>
Control of DYAD Activity or Gene Expression<br>
Since DYAD genes are involved in controlling meiosis and ploidy of the female gametophyte ,<br>
inhibition of endogenous DYAD activity or gene expression is useful in a number of contexts.<br>
For instance, inhibition of expression or modification of DYAD activity by use of an allele carrying a C-terminal deletion as described above can be used for production of fruit with absent or small/degraded seed (referred to here as "seedless fruit"). In most plant species the creation of triploids causes defects in the formation of germ cells due to unbalanced segregation of chromosomes in meiosis and leads to absence of seeds or the formation of small/degraded seeds. Inhibition of endogenous DYAD expression or activity can allow control of ploidy. Thus, in some embodiments of plants of the invention in which DYAD activity is inhibited or modified, seeds are absent or degraded and seedless fruit are produced.<br>
Another use of nucleic acids of the invention is in the development of apomictic plant lines (i.e.,<br>
plants in which asexual reproductive processes occur in the ovule, see, KoltunowA., Plant Cell,<br>
Vol.5: 1425-1437 (1993) for a discussion of apomixis). Apomixis provides a novel means to<br>
select and fix complex heterozygous genotypes that cannot be easily maintained by traditional<br>
breeding. Thus, for instance, new hybrid lines with desired traits (e.g., hybrid vigor) can be<br>
obtained	and	readily	maintained.<br>
One of skill will recognize that a number of methods can be used to modulate DYAD activity or gene expression. DYAD activity can be modulated in the plant cell at the gene, transcriptional, posttranscriptional, translational, or posttranslational, levels. Techniques for modulating DYAD activity at each of these levels are generally well known to one of skill and some are discussed briefly below.<br>
Methods for introducing genetic mutations into plant genes are well known. For instance, seeds or other plant material can be treated with a mutagenic chemical substance, according to standard techniques. Such chemical substances include, but are not limited to, the following: diethyl sulfate, ethylene imine, ethyl methanesulfonate and N-nitroso-N-ethylurea. Alternatively, ionizing radiation from sources such as, for example, X-rays, gamma rays, or fast neutrons can be used. Plants carrying mutations in DYAD gene sequences can be identified by molecular screening of pooled populations of mutagenized plants using PCR primers to amplify DYAD nucleotide sequences followed by analysis of PCR products to identify plants carrying genetic mutations in DYAD polynucleotide sequences. Methods for screening and identifying plants carrying mutations in specific gene sequences have been described (Henikoff S., Bradley T. J. and Comai L., Plant Physiol. Vol. 135(2): 630-636, 2004).<br>
Alternatively, homologous recombination can be used to induce targeted gene disruptions by specifically deleting or altering the DYAD gene in vivo (see, generally Grewal and Klar, Genetics 146: 1221-1238 (1997) and Xu et al., Genes Dev. 10: 2411-2422 (1996)). Homologous recombination has been demonstrated in plants (Puchta et al., Experientia 50: 277-284 (1994),<br>
Swoboda et al., EMBO J. 13: 484-489 (1994); Offringa et al., Proc. Natl. Acad. Sci. USA 90: 7346-7350 (1993); and Kempin et al. Nature 389:802-803 (1997)).<br>
In applying homologous recombination technology to the genes of the invention, mutations in selected portions of DYAD gene sequences (including 5' upstream, 3' downstream, and intragenic regions) such as those disclosed here are made in vitro and then introduced into the desired plant using standard techniques. Since the efficiency of homologous recombination is known to be dependent on the vectors used, use of dicistronic gene targeting vectors as described by Mountford et al. Proc. Natl. Acad. Sci. USA 91: 4303-4307 (1994); and Vaulont et al. Transgenic Res. 4: 247-255 (1995) are conveniently used to increase the efficiency of selecting for altered DYAD gene expression in transgenic plants. The mutated gene will interact with the target wild-type gene in such a way that homologous recombination and targeted replacement of the wild-type gene will occur in transgenic plant cells, resulting in suppression of DYAD activity. Alternatively, oligonucleotides composed of a contiguous stretch of RNA and DNA residues in a duplex conformation with double hairpin caps on the ends can be used. The RNA/DNA sequence is designed to align with the sequence of the target DYAD gene and to contain the desired nucleotide change. Introduction of the chimeric oligonucleotide on an extrachromosomal T-DNA plasmid results in efficient and specific DYAD gene conversion directed by chimeric molecules in a small number of transformed plant cells. This method is described in Cole-Strauss et al. Science 273:1386-1389 (1996) and Yoon et al. Proc. Natl. Acad. Sci. USA 93: 2071-2076 (1996). Gene expression can be inactivated using recombinant DNA techniques by transforming plant cells with constructs comprising transposons or T-DNA sequences. DYAD mutants prepared by these methods are identified according to standard techniques. For instance, mutants can be detected by PCR or by detecting the presence or absence of DYAD mRNA, e.g., by Northern blots or Reverse Transcription followed by PCR (RT-PCR). Mutants can also be selected by assaying for alterations in fertility, female meiosis, and megaspore development. The isolated nucleic acid sequences prepared as described herein can also be used in a number of techniques to control endogenous DYAD gene expression at various levels. Subsequences from the sequences disclosed here can be used to control transcription, RNA accumulation, translation, and the like.<br>
A number of methods can be used to inhibit gene expression in plants. For instance, RNA interference (RNAi) technology can be conveniently used. To achieve this, a nucleic acid segment from the desired gene is cloned as an inverted repeat in which the two copies are separated by a spacer which may be commonly between 5 and 2000 nucleotides in length, preferably between 30 and 500 nucleotides, and more preferably between 50 and 200 nucleotides.<br>
The inverted repeat is operably linked to a promoter followed by a terminator such that both copies will be transcribed and give rise to an RNA species that is self-complementary along all or part of its length. The construct is then transformed into plants and double stranded RNA is produced.<br>
As another instance, antisense technology can be conveniently used to inhibit DYAD gene expression. To accomplish this, a nucleic acid segment from the desired gene is cloned and operably linked to a promoter such that the antisense strand of RNA will be transcribed. The construct is then transformed into plants and the antisense strand of RNA is produced. In plant cells, it has been suggested that antisense suppression can act at all levels of gene regulation including suppression of RNA translation (see, Bourque Plant Sci. (Limerick) 105: 125-149 (1995); Pantopoulos In Progress in Nucleic Acid Research and Molecular Biology, Vol. 48. Cohn, W. E. and K. Moldave (Ed.). Academic Press, Inc.: San Diego, Calif., USA; London, England, UK. p. 181-238; Heiser et al. Plant Sci. (Shannon) 127: 61-69 (1997)) and by preventing the accumulation of mRNA which encodes the protein of interest, (see, Baulcombe Plant Mol. Bio. 32:79-88 (1996); Prins and Goldbach Arch. Virol. 141: 2259-2276 (1996); Metzlaff et al. Cell 88: 845-854 (1997), Sheehy et al., Proc. Nat. Acad. Sci. USA, 85:8805-8809 (1988), and Hiatt et al., U.S. Pat. No. 4,801,340).<br>
The nucleic acid segment to be introduced generally will be substantially identical to at least a portion of the endogenous DYAD gene or genes to be repressed. The sequence, however, need not be perfectly identical to inhibit expression. The vectors of the present invention can be designed such that the inhibitory effect applies to other genes within a family of genes exhibiting homology or substantial homology to the target gene.<br>
For antisense suppression, the introduced sequence also need not be full length relative to either the primary transcription product or fully processed mRNA. Generally, higher homology can be used to compensate for the use of a shorter sequence. Furthermore, the introduced sequence need not have the same intron or exon pattern, and homology of non-coding segments may be equally effective. Normally, a sequence of between about 30 or 40 nucleotides and about full-length nucleotides should be used, though a sequence of at least about 100 nucleotides is preferred, a sequence of at least about 200 nucleotides is more preferred, and a sequence of about 500 to about 1700 nucleotides is especially preferred.<br>
A number of gene regions can be targeted to suppress DYAD gene expression. The targets can include, for instance, the coding regions, introns, sequences from exon/intron junctions, 5' or 3' untranslated regions, and the like. In some embodiments, the constructs can be designed to eliminate the ability of regulatory proteins to bind to DYAD gene sequences that are required for<br>
its cell- and/or tissue-specific expression. Such transcriptional regulatory sequences can be located either 5'-, 3'-, or within the coding region of the gene and can be either promote (positive regulatory element) or repress (negative regulatory element) gene transcription. These sequences can be identified using standard deletion analysis, well known to those of skill in the art. Once the sequences are identified, an antisense construct targeting these sequences is introduced into plants to control gene transcription in particular tissue, for instance, in developing ovules and/or seed. Oligonucleotide-based triple-helix formation can be used to disrupt DYAD gene expression. Triplex DNA can inhibit DNA transcription and replication, generate site-specific mutations, cleave DNA, and induce homologous recombination (see, e.g., Havre and Glazer J. Virology 67:7324-7331 (1993); Scanlon et al. FASEB J. 9:1288-1296 (1995); Giovannangeli et al. Biochemistry 35:10539-10548 (1996); Chan and Glazer J. Mol. Medicine (Berlin) 75: 267-282 (1997)). Triple helix DNAs can be used to target the same sequences identified for antisense regulation.<br>
Catalytic RNA molecules or ribozymes can also be used to inhibit expression of DYAD genes. It<br>
is possible to design ribozymes that specifically pair with virtually any target RNA and cleave the<br>
phosphodiester backbone at a specific location, thereby functionally inactivating the target RNA.<br>
In carrying out this cleavage, the ribozyme is not itself altered, and is thus capable of recycling<br>
and cleaving other molecules, making it a true enzyme. The inclusion of ribozyme sequences<br>
within antisense RNAs confers RNA-cleaving activity upon them, thereby increasing the activity<br>
of the constructs. Thus, ribozymes can be used to target the same sequences identified for<br>
antisense	regulation.<br>
A number of classes of ribozymes have been identified. One class of ribozymes is derived from a number of small circular RNAs which are capable of self-cleavage and replication in plants. The RNAs replicate either alone (viroid RNAs) or with a helper virus (satellite RNAs). Examples include RNAs from avocado sunblotch viroid and the satellite RNAs from tobacco ringspot virus, lucerne transient streak virus, velvet tobacco mottle virus, solanum nodiflorum mottle virus and subterranean clover mottle virus. The design and use of target RNA-specific ribozymes is described in Zhao and Pick Nature 365:448-451 (1993); Eastham and Ahlering J. Urology 156:1186-1188 (1996); Sokol and Murray Transgenic Res. 5:363-371 (1996); Sun et al. Mol. Biotechnology 7:241-251 (1997); and Haseloff et al. Nature, 334:585-591 (1988). Another method of suppression is sense cosuppression. Introduction of nucleic acid configured in the sense orientation has been shown to be an effective means by which to block the transcription of target genes. For an example of the use of this method to modulate expression of endogenous genes (see, Assaad et al. Plant Mol. Bio. 22: 1067-1085 (1993); Flavell Proc. Natl. Acad. Sci.<br>
USA 91: 3490-3496 (1994); Stam et al. Annals Bot. 79: 3-12 (1997); Napoli et al., The Plant Cell 2:279-289 (1990); and U.S. Pat. Nos. 5,034,323, 5,231,020, and 5,283,184). The suppressive effect may occur where the introduced sequence contains no coding sequence per se, but only intron or untranslated sequences homologous to sequences present in the primary transcript of the endogenous sequence. The introduced sequence generally will be substantially identical to the endogenous sequence intended to be repressed. This minimal identity will typically be greater than about 65%, but a higher identity might exert a more effective repression of expression of the endogenous sequences. Substantially greater identity of more than about 80% is preferred, though about 95% to absolute identity would be most preferred. As with antisense regulation, the effect should apply to any other proteins within a similar family of genes exhibiting homology or substantial homology.<br>
For sense suppression, the introduced sequence, needing less than absolute identity, also need not be full length, relative to either the primary transcription product or fully processed mRNA. This may be preferred to avoid concurrent production of some plants which are overexpressers. A higher identity in a shorter than full length sequence compensates for a longer, less identical sequence. Furthermore, the introduced sequence need not have the same intron or exon pattern, and identity of non-coding segments will be equally effective. Normally, a sequence of the size ranges noted above for antisense regulation is used. In addition, the same gene regions noted for antisense regulation can be targetted using cosuppression technologies.<br>
Alternatively, eliminating the proteins that are required for DYAD cell-specific gene expression may modulate DYAD activity. Thus, expression of regulatory proteins and/or the sequences that control DYAD gene expression can be modulated using the methods described here. Another method is use of engineered tRNA suppression of DYAD mRNA translation. This method involves the use of suppressor tRNAs to transactivate target genes containing premature stop codons (see, Betzner et al. Plant J.l 1:587-595 (1997); and Choisne et al. Plant J.l 1: 597-604 (1997). A plant line containing a constitutively expressed DYAD gene that contains an amber stop codon is fir plants, each containing tRNA suppressor gene constructs under the direction of cell-type specific promoters are also generated. The tRNA gene construct is then crossed into the DYAD line to activate DYAD activity in a targeted manner. These tRNA suppressor lines could also be used to target the expression of any type of gene to the same cell or tissue types. The production of dominant-negative forms of DYAD polypeptides that are defective in their abilities to bind to other proteins is a convenient means to inhibit endogenous DYAD activity. This approach involves transformation of plants with constructs encoding mutant DYAD polypeptides that form defective complexes with endogenous proteins and thereby prevent the<br>
complex from forming properly. The mutant polypeptide may vary from the naturally occurring sequence at the primary structure level by amino acid substitutions, additions, deletions, and the like. These modifications can be used in a number of combinations to produce the final modified protein chain. Use of dominant negative mutants to inactivate target genes is described in Mizukami et al. Plant Cell 8:831-845 (1996).<br>
Another strategy to affect the ability of a DYAD protein to interact with itself or with other proteins involves the use of antibodies specific to DYAD. In this method cell-specific expression of DYAD-specific Abs is used inactivate functional domains through antibody:antigen recognition (see, Hupp et al. Cell 83:237-245 (1995)).<br>
Use of Nucleic Acids of the Invention to Enhance DYAD Gene Expression<br>
Isolated sequences prepared as described herein can also be used to introduce expression of a<br>
particular DYAD nucleic acid to enhance or increase endogenous gene expression. Enhanced<br>
expression can also be used, for instance, to increase vegetative growth by preventing the plant<br>
from setting seed. Where overexpression of a gene is desired, the desired gene from a different<br>
species may be used to decrease potential sense suppression effects.<br>
One of skill will recognize that the polypeptides encoded by the genes of the invention, like other<br>
proteins, have different domains that perform different functions. Thus, the gene sequences need<br>
not be full length, so long as the desired functional domain of the protein is expressed.<br>
Modified protein chains can also be readily designed utilizing various recombinant DNA<br>
techniques well known to those skilled in the art and described in detail, below. For example, the<br>
chains can vary from the naturally occurring sequence at the primary structure level by amino<br>
acid substitutions, additions, deletions, and the like. These modifications can be used in a number<br>
of combinations to produce the final modified protein chain.<br>
Preparation of Recombinant Vectors<br>
To use isolated sequences in the above techniques, recombinant DNA vectors suitable for transformation of plant cells are prepared. Techniques for transforming a wide variety of higher plant species are well known and described in the technical and scientific literature. See, for example, Weising et al. Ann. Rev. Genet. 22:421-477 (1988). A DNA sequence coding for the desired polypeptide, for example a cDNA sequence encoding a full-length protein, or a fusion protein of DYAD to an intracellular localization sequence, or a truncated DYAD protein, will preferably be combined with transcriptional and translational initiation regulatory sequences<br><br><br><br><br><br><br>
which will direct the transcription of the sequence from the gene in the intended tissues of the transformed plant.<br>
st created. Multiple lines of For example, for overexpression, a plant promoter fragment may be employed which will direct expression of the gene in all tissues of a regenerated plant. Such promoters are referred to herein as "constitutive" promoters and are active under most environmental conditions and states of development or cell differentiation. Examples of constitutive promoters include the cauliflower mosaic virus (CaMV) 35S transcription initiation region. The I'- or 2'- promoter derived from T-DNA of Agrobacterium tumafaciens, and other transcription initiation regions from various plant genes known to those of skill. Such genes include for example, ACT11 from Arabidopsis (Huang et al. Plant Mol. Biol. 33:125-139 (1996)), Cat3 from Arabidopsis (GenBank No. U43147, Zhong et al., Mol. Gen. Genet. 251:196-203 (1996)), the gene encoding stearoyl-acyl carrier protein desaturase from Brassica napus (Genbank No. X74782, Solocombe et al. Plant Physiol. 104:1167-1176 (1994)), GPcl from maize (GenBank No. X15596, Martinez et al. J. Mol. Biol 208:551-565 (1989)), and Gpc2 from maize (GenBank No. U45855, Manjunath et al., Plant Mol. Biol. 33:97-112 (1997)). Alternatively, the plant promoter may direct expression of the DYAD nucleic acid in a specific tissue or may be otherwise under more precise environmental or developmental control. Examples of environmental conditions that may effect transcription by inducible promoters include anaerobic conditions, elevated temperature, or the presence of light. Such promoters are referred to here as "inducible" or "tissue-specific" promoters. One of skill will recognize that a tissue-specific promoter may drive expression of operably linked sequences in tissues other than the target tissue. Thus, as used herein a tissue-specific promoter is one that drives expression preferentially in the target tissue, but may also lead to some expression in other tissues as well. Examples of promoters under developmental control include promoters that initiate transcription only (or primarily only) in certain tissues, such as fruit, seeds, or flowers. Promoters that direct expression of nucleic acids in ovules, flowers, or seeds are particularly useful in the present invention. As used herein a seed-specific promoter is one that directs expression in seed tissues. Such promoters may be, for example, ovule-specific (which includes promoters which direct expression in maternal tissues or the female gametophyte, such as egg cells or the central cell), embryo-specific, endosperm-specific, integument-specific, seed coat-specific, or some combination thereof. Examples include a promoter from the ovule-specific BEL1 gene described in Reiser et al. Cell 83:735-742 (1995) (GenBank No. U39944), and the promoter from the male meiocyte specific DUET gene (Reddy T. V., et al., Development, Vol. 130 (24):5975-5987, 2003). Other suitable seed specific promoters are derived from the following genes: MAC1 from<br>
maize (Sheridan et al. Genetics 142:1009-1020 (1996), Cat3 from maize (GenBank No. L05934, Abler et al. Plant Mol. Biol. 22:10131-1038 (1993), the gene encoding oleosin 18kD from maize (GenBank No. J05212, Lee et al. Plant Mol. Biol. 26:1981-1987 (1994)), viviparous-1 from Arabidopsis (Genbank No. U93215), the gene encoding oleosin from Arabidopsis (Genbank No. Z17657), Atmycl from Arabidopsis (Urao et al. Plant Mol. Biol. 32:571-576 (1996), the 2S seed storage protein gene family from Arabidopsis (Conceicao et al. Plant J. 5:493-505 (1994)) the gene encoding oleosin 20kD from Brassica napus (GenBank No. M63985), napA from Brassica napus (GenBank No. J02798, Josefsson et al. JBL 26:12196-1301 (1987), the napin gene family from Brassica napus (Sjodahl et al. Planta 197:264-271 (1995), the gene encoding the 2S storage protein from Brassica napus (Dasgupta et al. Gene 133:301-302 (1993)), the genes encoding oleosin A (Genbank No. U09118) and oleosin Band the gene encoding low molecular weight sulphur rich protein from soybean (Choi et al. Mol. Gen., Genet. 246:266-268 (1995)). In addition, the promoter sequences from the DYAD genes disclosed here can be used to drive expression of the DYAD polynucleotides of the invention or heterologous sequences. If proper polypeptide expression is desired, a polyadenylation region at the 3'-end of the coding region should be included. The polyadenylation region can be derived from the natural gene, from a variety of other plant genes, or from T-DNA.<br>
The vector comprising the sequences (e.g., promoters or coding regions) from genes of the invention will typically comprise a marker gene, which confers a selectable phenotype on plant cells. For example, the marker may encode biocide resistance, particularly antibiotic resistance, such as resistance to kanamycin, G418, bleomycin, hygromycin, or herbicide resistance, such as resistance to chlorosulfuron or Basta.<br>
Production of Transgenic Plants<br>
DNA constructs of the invention may be introduced into the genome of the desired plant host by a<br>
variety of conventional techniques. For example, the DNA construct may be introduced directly<br>
into the genomic  DNA  of the  plant cell  using techniques  such  as  electroporation and<br>
microinjection of plant cell protoplasts, or the DNA constructs can be introduced directly to plant<br>
tissue using ballistic methods, such as DNA particle bombardment.<br>
Microinjection techniques are known in the art and well described in the scientific and patent<br>
literature.  The introduction of DNA constructs using polyethylene glycol  precipitation is<br>
described in Paszkowski et al. Embo J. 3:2717-2722 (1984). Electroporation techniques are<br>
described in Fromm et al. Proc. Natl. Acad. Sci. USA 82:5824 (1985). Ballistic transformation<br>
techniques are described in Klein et al. Nature 327:70-73 (1987).<br>
Alternatively, the DNA constructs may be combined with suitable T-DNA flanking regions and introduced into a conventional Agrobacterium tumefaciens host vector. The virulence functions of the Agrobacterium tumefaciens host will direct the insertion of the construct and adjacent marker into the plant cell DNA when the cell is infected by the bacteria. Agrobacterium tumefaciens-mediated transformation techniques, including disarming and use of binary vectors, are well described in the scientific literature. See, for example Horsch et al. Science 233:496-498 (1984), and Fraley et al. Proc. Natl. Acad. Sci. USA 80:4803 (1983).<br>
Transformed plant cells which are derived by any of the above transformation techniques can be cultured to regenerate a whole plant which possesses the transformed genotype and thus the desired phenotype such as increased seed mass. Such regeneration techniques rely on manipulation of certain phytohormones in a tissue culture growth medium, typically relying on a biocide and/or herbicide marker which has been introduced together with the desired nucleotide sequences. Plant regeneration from cultured protoplasts is described in Evans et al., Protoplasts Isolation and Culture, Handbook of Plant Cell Culture, pp. 124-176, MacMillilan Publishing Company, New York, 1983; and Binding, Regeneration of Plants, Plant Protoplasts, pp. 21-73, CRC Press, Boca Raton, 1985. Regeneration can also be obtained from plant callus, explants, organs, or parts thereof. Such regeneration techniques are described generally in Klee et al. Ann. Rev. of Plant Phys. 38:467-486 (1987).<br>
The nucleic acids of the invention can be used to confer desired traits on essentially any plant. Thus, the invention has use over a broad range of plants, including (Genbank No. U09119) from soybean species from the genera Anacardium, Arachis, Asparagus, Atropa, Avena, Brassica, Citrus. Citrullus, Capsicum, Carthamus, Cocos, Coffea, Cucumis, Cucurbita, Daucus, Elaeis, Fragaria, Glycine, Gossypium, Helianthus, Heterocallis, Hordeum, Hyoscyamus, Lactuca, Linum, Lolium, Lupinus, Lycopersicon, Malus, Manihot, Majorana, Medicago, Nicotiana, Olea, Oryza, Panieum, Pannesetum, Persea, Phaseolus, Pistachia, Pisum, Pyrus, Prunus, Raphanus, Ricinus, Secale, Senecio, Sinapis, Solanum, Sorghum, Theobromus, Trigonella, Triticum, Vicia, Vitis, Vigna, and Zea.<br>
One of skill will recognize that after the expression cassette is stably incorporated in transgenic plants and confirmed to be operable, it can be introduced into other plants by sexual crossing. Any of a number of standard breeding techniques can be used, depending upon the species to be crossed.<br>
Seed obtained from plants of the present invention can be analyzed according to well known procedures to identify plants with the desired trait. If antisense or other techniques are used to control DYAD gene expression, RT-PCR or Northern blot analysis can be used to screen for<br>
desired plants. In addition, the presence of fertilization independent reproductive development can be detected. Plants can be screened, for instance, for the ability to form embryo-less seed, form seed that abort after fertilization, or set fruit in the absence of fertilization. These procedures will depend in part on the particular plant species being used, but will be carried out according to methods well known to those of skill.<br>
The following examples are given by way of illustration of the present invention and should not be construed to limit the scope of present invention.<br>
EXAMPLE 1: The dyad mutant shows defective female fertility and reduced seed set<br>
The dyad mutant was isolated in a screen for sterile mutants of Arabidopsis among a population of EMS mutagenized M2 plants (Siddiqi I. et. al. Development Vol. 127(1): 197-207 (2000)) Analysis of fertility by reciprocal crosses indicated that the mutant was female sterile but male fertile. Analysis of female sporogenesis and ovule development indicated that dyad underwent a defective female meiosis resulting in a single meiotic division due to defective progression through the meiotic cell cycle, followed by arrest and failure to develop female gametes in the majority of ovules. Analysis of female meiosis by observations of chromosome spreads of meiocytes indicated that female meiosis was abnormal in the dyad mutant: chromosomes failed to synapse and underwent an equational division instead of a reductional division, which would normally take place at meiosis l(Agashe B., Prasad C. K., and Siddiqi I., Development Vol. 129(16): 3935-3943 (2002)).<br>
As shown in Figure 1, seed set in the dyad mutant is highly reduced when compared to wild type and variation was observed in the degree of seed set among different dyad mutant plants. The seed set was sporadic and random such that no uniformity in terms of number was observed among the plants in the population. The mode for seed set was 1-10 per plant but ranged upto a maximum of about 275 that was observed rarely (1 in 500 plants).<br>
EXAMPLE 2: Male meiosis and fertility are normal in the dyad mutant<br>
Pollen viability was examined using Alexander staining and pollen were found to be fully viable and comparable to wild type (Figure 2). Examination of male meiosis by analysis of chromosome spreads of meiocytes indicated that male meiosis was normal and resulted in the production of a tetrad of haploid spores (Figure 3). Male meiosis, male fertility, and pollen development as well as function were therefore normal in the dyad mutant. On the other hand female meiosis is abnormal in dyad. Synapsis of homologous chromosomes is not seen to occur and the reductional<br>
meiosis 1 division of wild type female meiosis (Figure 3C) is replaced by an equational one in dyad (Figure 3E).<br>
EXAMPLE 3: Seeds obtained from the dyad mutant germinate to give triploid plants<br>
It is possible that the seeds produced in the dyad mutant arise from a normal meiosis in a small minority of female meiocytes, which go on to give rise to a normal functional embryo sac that is then fertilized by haploid pollen to develop into seed. If this was the case, these seeds would represent escapees from the abnormal female meiosis which takes place in the dyad mutant. To examine this possibility, seeds (n=169) from dyad plants were germinated and found to germinate with high efficiency (&gt;90%) and produce morphologically normal seedlings except a few that gave abnormal seedlings (10%). No instances of variations in shape, symmetry and number of cotyledons were observed in the germinating seedlings. This is in contrast to seedlings derived from other meiotic mutants such as AtSpol 1-1 and AtDmcl which undergo random segregation of chromosomes in meiosis 1, resulting in higher proportion of aneuploid progeny that show a range of developmental abnormalities at the seedling stage (Grelon M. et al., The EMBO J., Vol 20: 589-600, 2001, Couteau F. et al., Plant Cell, Vol. 11(9): 1623-1634 , 1999). Subsequent vegetative growth of seedlings on transferring to soil also was normal and gave rise to plants in which vegetative growth was similar to wild type as well as the parent dyad mutant plants. The main difference observed was in flower size when the plants started bolting. In a majority of the plants (n=41/52) a comparative increase in flower size was observed as to wild type. The increased flower size could possibly be attributed to increase in vigour or favourable environmental influences. Since the plants are grown in controlled environment we ruled out the latter possibility. The other possible reason for increase in floral organ size might be increase in ploidy. Increase in ploidy is manifested by the increase in size of vegetative and floral structures, particularly the pollen grains (Altmann T., et al., Plant Cell Reports. Vol. 13: 652 - 656, 1994). Flower buds from randomly picked plants were examined for their ploidy level by analysis of chromosomes in somatic cells and in male meiocytes. It was found by examination of meiotic chromosome spreads that in 17/19 cases the plants were triploid and the remaining 2 were found to be diploid (Figure 4). Since pollen development and male meiosis are normal in the dyad mutant whereas a reductional female meiosis is replaced by an equational division, these results suggest that the majority of the seeds which are triploid arise from fertilization of an unreduced (diploid) egg cell by a normal haploid sperm and do not arise from a normal female meiosis in a minority of ovules, i.e. the majority of seeds do not represent escapees from the abnormal meiosis.<br>
EXAMPLE 4: Triploid plants derived from dyad show retention of all heterozygous markers<br>
The triploid seeds formed in the dyad mutant could be the product of fertilization of an unreduced embryo sac by a normal haploid pollen which would be consistent with the equational female meiosis that takes place in dyad. If such an unreduced embryo sac is formed from an unreduced megaspore that arises from the product of an equational, division of the megaspore mother cell wherein chromosomes remain as univalents and fail to undergo recombination, then the genotype of the unreduced embryo sac would be identical to that of the diploid parent plant. Hence if the parent plant is heterozygous for a molecular marker then the triploid progeny will also be heterozygous for that marker. If a marker unlinked to the centromere is considered in a heterozygous condition, then in the complete absence of recombination 100% transmission of parental heterozygosity will be achieved in the resultant female gamete and the triploid progeny. If recombination and crossing over take place then 100% heterozygosity will not be maintained in the resultant triploid progenies. For a marker that is unlinked to the centromere, one can expect homozygosity in the unreduced embryo sac at a frequency of 33% and in the triploid progeny at a frequency of 16.7% whereas in the complete absence of recombination there will be no homozygotes. The formation of unreduced embryo sacs without loss of heterozygosity is highly desirable for engineering apomixis and fixation of heterosis. We used microsatellite to measure loss of heterozygosity among the triploid progeny of dyad mutant plants. The dyad mutant plants were identified in a segregating F2 population of a cross between wild type Nossen (No-O) and dyad mutant Columbia (Col) ecotypes. Candidate markers distributed across each of the five Arabidopsis chromosomes and unlinked to the centromere (&gt;35 cM) were obtained from the TAIR database (www.arabidopsis.org). The parent plants used to generate the F2 population were examined to ascertain the polymorphism and based on the results we choose 5 different markers (Table 1) on 4 different chromosomes to genotype 50 F2 dyad mutant plants and identify those markers for which each plant was heterozygous. Selfed seeds were collected from the 50 F2 plants individually and grown as 50 different families consisting of a variable number of siblings. This gave a total of 196 plants distributed across 50 families. All members of each family were genotyped with respect to those markers for which the parent plant was heterozygous so as to give between 74-119 plants distributed across all the 50 F2 families for each marker.<br>
Table 1: Marker analysis of progeny of dyad plants to measure loss of heterozygosity and recombination.<br>
a Figure in brackets in the column 6 represents the percentage homozygotes of the total plants analysed for that marker.<br><br>
(Table Removed)<br>
Out of 196 plants screened we obtained 35 plants that were homozygous for at least one marker for which the parent plant was heterozygous. The ploidy of 22 of these 35 plants was determined by carrying out meiotic chromosome spreads. It was found that 21 were diploid and another a hyperdiploid having 13 chromosomes. Hence according to the analysis loss of heterozygosity was found almost exclusively only in diploids. Of the plants that did not show loss of heterozygosity, 15 plants were chosen at random from separate F2 families and examined for their ploidy. All 15 were found to be triploid.<br>
The results therefore indicate that there is no loss of heterozygosity in triploids which make up the majority class of progeny from a diploid dyad mutant plant. The failure to find loss of heterozygosity in triploids also rules out an alternative possible mechanism for their formation, namely polyspermy, i.e. fertilization of a haploid female gamete by two separate male gametes, which would also predict loss of heterozygosity. Our findings show that the triploid progeny of dyad mutant plants arise from fertilization of an unreduced embryo sac that retains the genotype of the parent plant. The formation of an unreduced embryo sac is a key aspect of apomixis.<br>
EXAMPLE 5: Isolation and functional characterization of the DYAD homologue from Boechera holboelli<br>
The 3 kB genomic coding region of the DYAD homolog from the facultatively apomictic Boechera holboellii accessions Diploid Greenland and Triploid Colorado (Naumova T. N., et al., Sex. Plant Reprod. Vol. 14: 195-200, 2001) were cloned using Bho5Bam (SEQ ID NO:39) and Bho3Bam (SEQ ID NO:40) primers. The BhDYAD genomic clone (SEQ ID NO: 16) was<br>
operably linked to the Arabidopsis DYAD promoter and used to transform dyad mutant plants to test for complementation. The BhDYAD cDNA was also amplified and sequenced (SEQ ID NO: 17). Agrobacterium mediated in planta vacuum infiltration transformation mobilized the expression construct to Fl plants that were heterozygous for dyad. We obtained 42 transformants out of which 9 transformants were homozygous for the dyad mutant allele as determined by the CAPS and microsatellite markers that flank the dyad locus (Agashe B, Prasad C. K., and Siddiqi I., Development Vol. 129(16): 3935-3943 (2002)). Of the 9 transformants 4 transformants showed complementation of the dyad mutant phenotype, which can be judged by the well elongated siliques (Figure 5) which were found to contain full seed set. The remaining 5 plants were sterile possibly due to cosuppression.<br>
Growth of Arabidopsis Plants<br>
The Arabidopsis strain harboring the dyad mutant was as described earlier (Siddiqi I., et. al., Development. Vol. 127(1): 197-207 (2000)). F2 population used for microsatellite marker analysis was derived from a cross between the strain No-0 (Nossen ecotype) and dyad mutant in the Col-0 ecotype background as described (Siddiqi I., et. al., Development. Vol. 127(1): 197-207 (2000)). Plants were grown in a controlled environment as described (Siddiqi 1., et. al., Development. Vol. 127(1): 197-207(2000)).<br>
For germinating seeds in Petri plates, the seeds were surface sterilized with ethanol for 10 min followed by treating them with 0.025% mercuric chloride for 5 min. Further the seeds were washed three times with sterile water to remove any traces of mercuric chloride. The seeds were resuspended in lukewarm 0.5% top agar and evenly spread on MS agar plates (0.7%) supplemented with 2% Sucrose. The plates were allowed to dry for an hour in a laminar flow hood and the plates were sealed with parafilm and kept in a cold room at 4°C for stratification for 3 days. After that the plates were shifted to a growth chamber. Germination frequencies were counted after two weeks thereafter.<br>
For growing seeds in the pots, the synthetic medium used for growing plants was prepared by mixing an equal proportion of Soilrite: Perlite: Vermiculite (Keltech Energies Ltd., Karnataka 574 108, India). The pot mixture was evenly applied to the pots perforated at the bottom allowing capilla Solution containg Major Salts: CaC12 (4mM), MgSo4 (1.5mM), KN03 (18.8mM), NH4N03 (20.6mM), KH2Po4 (1.25mM pH 5.6), Fe-EDTA (20mM) to which 1 ml (1000X) Minor Salts: (H3B03 (70mM), MnC12 (14mM), CuS04 (0.5mM), ZnS04 (ImM), NaMo04 (0.2mM), NaCl (10mM), CoC12 (0.01mM)) was added per litre. The seeds were evenly spread on<br>
the surface of the pot and covered with Saran wrap and kept at 4-8 °C for 3 days for stratification and then shifted to a growth chamber. In case of transplantation, the pots were covered with Saran wrap after the seedlings were transferred to the soil medium and directly placed in the growth chamber. The Saran wrap was removed once the plants were established in the potting mix. Watering was done at regular intervals using distilled water.<br>
Seed set analysis<br>
The F2 segregating population harbouring a dyad mutation in the Col-0 ecotype background was used for scoring the frequency of seed set in the dyad homozygous plants, dyad mutant plants were allowed to grow till their final stage when the plant ceased to flower. After this stage watering was witheld to allow the siliques to reach harvest maturity. Meanwhile the lowest siliques that turn yellow and were about to shatter were individually split open and the seeds if any were harvested on a single plant basis. Likewise necessary seeds were harvested at regular intervals to avoid possible seed loss. Finally the seeds collected were pooled on a single plant basis to count for the total number of seeds per plant.<br>
Pollen viability<br>
Vital staining for microspores in the anther was done as described (Alexander M. P., Stain Technol. Vol. 44(3): 117-122, 1971).<br>
Meiotic Spreads<br>
Analyses of male and female meiotic spreads are as described (Agashe B, Prasad C. K., and Siddiqi I., Development Vol. 129(16): 3935-3943 (2002))<br>
Plant DNA isolation<br>
Genomic DNA for microsatellite marker analysis was isolated according to the method described by Dellaporta S. L., et al., Plant Mol. Bio. Rep., Vol. 1: 19-21 (1983) with minor modifications. About 500 mg of leaf tissue was collected from an individual plant in 1.5 ml eppendorf tubes and snap frozen in liquid nitrogen. Then the tissue was ground to a fine powder using a micropestle. To this powder was added 200 µl of freshly prepared extraction buffer (100 mM Tris (pH 8), 50 mM EDTA, 500 mM NaCl, 1.4% SDS, and lOmM p-mercaptoethanol) and was finely homogenized with the micropestle. Then equal volume of 2X CTAB was added and the mixture was gently vortexed. Then the mixture was incubated at 65°C for 5 minutes in a shaking water bath. After that the sample was allowed to cool and an equal volume of 24:1 chlorofom: isoamyl<br>
alcohol was added and mixed gently and centrifuged for 10 min at 13000 rpm. The aqueous phase containing the DNA was transferred to a fresh eppendorf tube and 2/3 volumes of ice-cold isopropanol was added to precipitate the DNA. The DNA was pelleted down by centrifugation at 4°C at 13000 rpm for 20 min. The DNA pellet was given a 70% ethanol wash and the pellet was air dried for 30 minutes and suspended in 50 µl of sterile water or TE buffer (pH 8.0) containing DNAse free RNAse (20 µg/ml).<br>
Marker Analysis<br>
Based on the parental survey of Col-0 and No-0 ecotypes 5 microsatellite markers   from 4<br>
different chromosomes that are reasonably unlinked to the centromere were   chosen.   These<br>
markers were used on a F2 (No-0 x Col-0 (dyad)) segregating population to choose dyad plants<br>
that are heterozygous for a given marker.     Seeds from these dyad plants were collected and<br>
germinated in individual petri plates such that each progeny constitutes a sib of the particular<br>
mother dyad plant. Likewise data on sibs from various plants that were heterozygous for a given<br>
marker was considered together for marker analysis.<br>
The list of microsatellite markers and their location are as described in the Tablel. The primer<br>
sequences    used    for    amplifying    the    microsatellites    are    from    the    TAIR    website<br>
(www.arabidopsis.org):<br>
nga 162<br>
nga l62F SEQ ID NO:6<br>
ngal62R SEQ ID NO:7<br>
nga225<br>
nga225F SEQ ID NO:8<br>
nga225R SEQ ID NO:9<br>
nga168<br>
ngal68F SEQ ID NO: 10<br>
ngal68R SEQ ID NO: 11<br>
nga1107<br>
nga1107F SEQ ID NO:12<br>
ngall07R SEQ ID NO:13<br>
nga6<br>
nga6F SEQ ID NO:14<br>
nga6R SEQ ID NO:15<br>
PCR was performed in IX PCR buffer (Perkin Elmer) containing 2 mM MgCl., 0.2 mM each dNTP, 1 unit of Taq DNA polymerase (Perkin-Elmer/Cetus), and 5 pmoles of forward and reverse flanking primers at an annealing temperature of 55 °C with an extension at 72 °C for 20 seconds. The PCR products were resolved on a 8% polyacrylamide gel at 150V for 3hrs and stained with ethidium bromide and captured using Syngene gel documentation system (Synoptics Inc. UK).<br>
Plant materials<br>
The facultatively apomictic diploid Greenland and triploid Colorado accessions of Boechera holboellii were a kind gift from Kim Boutilier (Naumova T. N., et al., Sex. Plant Reprod. Vol. 14: 195-200, 2001). The plants were grown on pots containing the medium as described for Arabidopsis and grown under conditions identical to those for Arabidopsis.<br>
Cloning of DYAD promoter<br>
A 1.8 kb DYAD promoter region was amplified from Col-0 ecotype using the primers pg2r4 (SEQ ID NO: 48) and PDYBAM (SEQ ID NO: 47) and the product was cloned into a pGEMT vector (Promega) as per manufacturer's instructions.<br>
Cloning of DYAD homolog from Boechera holboellii<br>
The genomic coding region of the Arabidopsis DYAD homolog from Boechera holboellii (BhDYAD) was amplified with primers harboring a BamHI site on the 5' end: Bho5BAM (SEQ ID NO: 39) and Bho3BAM. (SEQ ID NO: 40) The resultant 3kb fragment was cloned into pGEMT.<br>
Construction of binary vector pCAMBIA1300 driving BhDYAD under Arabidopsis DYAD promoter<br>
The Bh DYAD was released from pGEMT as a 3 kb BamHI fragment and cloned into a pCAMBIA1300 vector carrying a plant selectable marker hygromycin. The orientation was checked using the primers BDY3 (SEQ ID NO: 36) and OCSR (SEQ ID NO: 38). The 1.6 kb DYAD promoter region (SEQ ID NO: 22) was released as a Sacl fragment from the pGEMT vector and inserted upstream of a BhDYAD in pCAMBIA1300 vector. The orientation of the promoter with respect to the BhDYAD genomic sequence was confirmed using primers ismr4 (SEQ ID NO: 37 ) and bdyl (SEQ ID NO:35)<br>
Triparental mating<br>
The transfer of the above constructed binary vector pCAMBIA into Agrobacterium (AGL1) was by triparental mating as described (Agashe B, Prasad C. K., and Siddiqi I , Development, Vol. 129(16): 3935-3943 (2002)).<br>
Transformation of Arabidopsis plants<br>
For complementation analysis of BhDYAD, Fl plants of Col-0 x dyad were transformed with the construct carrying BhDYAD driven by the Arabidopsis DYAD promoter. Agrobacterium mediated in planta vacuum infiltration transformation was carried out according to Bechtold N. and Pelletier G., Methods Mol. Biol., Vol. 82: 259-66 (1998).<br>
Selection of transformants<br>
TO seeds from vacuum infiltrated Fl plants were plated onto a petri plate containing 0.8% Bacto Agar, ImM K.N03 and 1% Sucrose with 20 u.g/ml hygromycin. After cold stratification for 3 days the plates were transferred to a growth chamber .The transformants that are resistant to hygromycin can be identified as early as 5 days post transfer by virtue of well elongated root, erect hypocotyl and well spread cotyledonary leaves. The selected transformants were further transferred to MS plates containing hygromycin and after resistance is established they were finally transferred to soil medium. Furthermore the plants were checked for the present of insert using bdy3 and OCSR primer as described earlier.<br>
Genotyping for zygosity at dyad locus<br>
The three genotypes from the segregating dyad F2 population were identified by the codominant CAPS markers (Konieczny A. and Ausubel F. M. , Plant J., Vol. 4(2): 403-410, 1993) and variable microsatellites. The flanking sequences of the dyad mutant allele are derived from Landsberg erecta ecotype and those from the wild type allele have Colombia ecotype sequence. Thus the SNPs in these flanking sequences were utilized to develop CAPs marker that are closely linked to and flanking either side of the dyad locus (KNEF (SEQ ID NO:31) and KNER(SEQ ID NO:32), KKF(SEQ ID NO:33) and KKR(SEQ ID NO:34)) and microsatellite marker primers (KMF (SEQ ID NO:29 and KMR (SEQ ID NO:30)) that are closely linked to DYAD (Agashe B, Prasad C. K., and Siddiqi I., Development Vol. 129(16): 3935-3943 (2002)). The genotyping at the dyad locus using the above markers was as described (Agashe B, Prasad C. K., and Siddiqi I., Development Vol. 129(16): 3935-3943 (2002)).<br>
RNA isolation and cDNAs synthesis<br>
Well-developed single buds from a diploid Greenland plant were used for total RNA isolation by TriZol reagent (Invitrogen) as per manufacturer's instructions. 4 (xg of total RNA was used for first strand cDNA synthesis using the Superscript™ Choice system for cDNA synthesis (GIBCO BRL). The cDNA was further amplified for cloning by using primers 5RF3(SEQ ID NO: 41) and Bho3BAM (SEQ ID NO:40). The resultant 1.9 KB fragment was cloned into pGEMT and sequenced. Results are presented in the Sequence Listing as SEQ ID NO: 17. The amino acid sequence of the corresponding DYAD protein is shown in SEQ ID NO: 18.<br>
EXAMPLE 6: Construction of a conditional allele of DYAD and development of a homogenous population of transgenic plants showing the dyad mutant phenotype<br>
The strategy used to construct a conditional allele of the DYAD gene was based on fusing the hormone binding domain of the rat glucocorticoid receptor (GR) (SEQ ID NO: 27) to the C-terminus of DYAD and integrating the fusion construct into the genome of plants that were homozygous for the dyad mutant allele (dy/dy). The DYAD-GR fusion protein on its own is not capable of complementing the dyad mutant because the GR domain confers cytoplasmic localization in the absence of steroid hormone, whereas the site of action of DYAD is in the nucleus. However in presence of the steroid hormone, the fusion protein is released from the cytoplasmic binding site and becomes capable of translocating to the nucleus where it can complement the dyad mutant. The steps in the construction were the following: the plant binary vector pBI101.3 was digested with BamHI plus Sad to remove the GUS reporter gene and replace it with a BamHl-Sacl fragment comprising the GR domain (A.M. Lloyd et al., Science 266,436-^39(1994)).<br>
The resultant plasmid was named pBI101.3::GR. Next the primers DyCF (SEQ ID NO: 43) and DyPB (SEQ ID NO: 42) (which contains sequence to modify the termination codon and introduce restriction sites for BamHI and PstI) were used to PCR amplify a 304 bp C-terminal region of the DYAD gene. The modified sequence was cloned as a 216 bp PstI fragment into the pBS (KS)::Dyad plasmid which carried a 5.8 kb genomic clone (SEQ ID NO:28) that contained the entire DYAD gene corresponding to coordinates 9684 to 3878 of the PI clone MFG13 (Ace No. AB025621) to give pBS(KS)::Dyad*. The resulting plasmid contained a DYAD gene whose termination codon TGA had been replaced by GGG and which also carried a BamHI site along with the replaced codon. The 269 bp Sall-BamHI fragment from pBSII(KS)::Dyad* which contained nucleotides 9684 to 9416 of MFG13 was cloned into pBHOl .3::GR following digestion with Sail plus BamHI. The remaining portion of DYAD from 9417-5335 was then cloned as a<br>
BamHI- BamHI fragment from pBS(KS)::Dyad* into the product of the previous step which resulted in an in frame fusion of the GR domain to the C-terminus of DYAD. The final plasmid named pBI101.3::DyadAGR is represented in Figure 6.<br>
The construct was introduced into the Agrobacterium strain AGL1 by triparental mating using the helper E. coli strain HB101[pRK2013]. The T-DNA region was transformed into Arabidopsis plants (TO) that were heterozygous for the dyad mutant allele (+/dy) by in-planta transformation (Bechtold N. and Pelletier G., Methods Mol. Biol., Vol. 82: 259-66, 1998). Kanamycin resistant Tl seedlings were selected by plating the seeds on MS agar plates containing kanamycin (50 mg/litre) and transferred to MS + kanamycin plates to confirm the resistant phenotype. Transformants were further identified by PCR using DyCF (SEQ ID NO:43) and GRrev (SEQ ID NO:44) primers. Confirmed kanamycin resistant seedlings were transferred to soil and grown to the adult stage. Following bolting and development of the first 8-10 siliques, plants were watered every three days with 10 uM dexamethasone in addition to being sprayed daily with 10 uM dexamethasone + 0.015% Silwet L-77. It was noted that several plants that showed sterility prior to dexamethasone treatment developed fertile siliques 5-7 days after the start of dexamethasone treatment. Part of the plant material was used for Southern analysis to determine copy number of the insertion and also genotyped with respect to the dyad locus using PCR based CAPS markers closely linked to and flanking the dyad locus. The dyad mutant was originally isolated in the Ler background and then introgressed into the Col strain. Hence the Ler allele of the CAPS markers is diagnostic for the dyad mutant whereas the Col allele is indicative of wild type (Figure 7). Single copy insertions were identified among plants that had at least one copy of the dyad mutant allele and seeds from these plants were plated on MS + kanamycin plates. Kanamycin resistant seedlings were transferred to soil and genotyped with respect to the dyad locus. Plants that were homozygous for the dyad mutant allele were identified and grown to adulthood. Following bolting all the plants were fed with water during the intial phase upto the opening of the first 8-10 flowers followed by watering with a solution containing dexamethasone as described above. As an example, one line No. 33 shown in Figure 8 gave dyad mutant plants (dy/dy) all of which showed sterility during the initial phase of reproductive growth and which became fertile following dexamethasone treatment. Ovules from buds isolated prior to dexamethasone treatment showed the dyad mutant phenotype, whereas those isolated after dexamethasone treatment showed the wild type phenotype (Figure 9). Seeds were collected from homozygous dyad mutant plants to give T3 families and T3 families which were homozygous for the DYAD-GR insertion were identified by screening for families, which gave all kanamycin resistant seedlings. These results exemplify construction of a conditional allele of DYAD and its introduction into plants<br>
thereby giving plants that show the dyad mutant phenotype under one set of conditions (the absence of dexamethasone) and the wild type phenotype when fed (or sprayed) with dexamethasone. These results also enable development of a homogenous population of plants all of which show the dyad mutant phenotype.<br>
Glucocorticoid receptor domain sequence used in this study (914 bp) (SEQ ID NO: 27)<br>
(Sequence Removed)<br>
EXAMPLE 7: Selfed seed of the dyad mutant that are triploid (3n) contain a diploid (2n)<br>
contribution from the female gamete<br>
Reciprocal  crosses were carried out between tetraploid (4n) and diploid (2n) wild type<br>
Arabidopsis plants. In both cases the seeds that are produced are triploid. However, when the<br>
male parent was tetraploid and the female parent was diploid, the seeds that were produced were<br>
large, whereas when the male parent was diploid and the female parent was tetraploid the seeds<br>
were shrunken. These results are depicted in Figure 10 and the 100-seed weights for each category of seed are shown in Table 2.<br>
Table 2: Weight of 100 seeds obtained from plants of various crosses<br>
(Table Removed)<br>
These findings reproduce what is known in the prior art (Scott RJ et al., Development 125, 3329-3341, 1998). Without being bound by any theory of mechanism, the nonequivalence of the paternal and maternal genomes in the regulation of seed development has been explained according to the parent-offspring conflict theory (Haig D. and Westoby M., Am. Nat. Vol. 134: 147-155, 1989) as arising from competition for resource allocation between the maternal parent which limits growth of the embryo by favouring equitable distribution of resources among all the seeds, and each embryo whose fitness is increased by garnering of greater resources. According to Haig D. and Westoby M., Am. Nat. Vol. 134: 147-155, 1989 imprinted genes that are maternally expressed in the embryo would act to limit growth of the embryo whereas paternally expressed genes would favour increased growth of the embryo. Thus seeds that contain an extra paternal genome equivalent would be larger than normal due to an excess dosage of gene products that promote growth of the embryo whereas seeds that contain an extra maternal genome equivalent would be smaller than normal due to an excess dosage of gene products that limit growth of the embryo.<br>
To address the maternal and paternal contributions in selfed seeds of the dyad mutant the seeds were analyzed with respect to size. The selfed seeds obtained from dyad mutant plants were heterogenous in size and classified in either of three categories: large, normal, and shrunken as depicted in FigurelO. The size class distribution from 7 individual dyad mutant plants is shown below:<br>
Table 3: Size Class Distribution for seeds from dyad mutant plants:<br>
(Table Removed)<br>
Seeds from each class were sampled from multiple plants, germinated and grown into plants. The ploidy of each plant was determined by chromosomal counts in meiotic spreads. The results are indicated in Table 4 below:<br>
Table 4: Ploidy of plants from each seed class in selfed dyad mutants<br><br>
(Table Removed)<br>
Numbers in brackets indicate percentage of total plants examined in each category<br>
These data show that most triploids are shrunken in size and make up the major portion of the shrunken category of seed. The observation that most triploids are shrunken indicates that they arise from an excess maternal contribution (2n) and not from an excess paternal contribution which would therefore be In in the triploids. Together with the finding of Example 4 that all triploids retain parental heterozygosity, these results indicate that the retention of heterozygosity is obtained from the female parent, and hence that the triploids arise from an unreduced female gamete that retains parental heterozygosity.<br>
To confirm that triploids in dyad arise from a 2n female contribution, we crossed a dyad mutant as a female to the line ETC60 (wild type for DYAD) as a male to give Fl seed. The ETC60 line (described in US Pat. Appl. No. 10/857,539) carries a single copy of a Ds transposon harbouring a kanamycin resistance gene. By following the segregation of kanamycin resistance following further crossing of the Fl to wild type diploid plants, it is possible to determine the ploidy contribution from the male gamete in the Fl plant. Seeds from the first cross were germinated and seedlings were transferred to soil. Six Fl plants were tested for the presence of the kanamycin resistance gene using kanamycin resistance gene-specific primers (KanF SEQ ID NO: 49 and KanR SEQ ID NO: 50) as well as for a copy of the transposon in ETC60 using a transposon specific Ds5-2 primer (SEQ ID NO: 45) in combination with a gene-specific primer GLTF (SEQ ID NO:46). All six plants were positive for the Ds element carrying kanamycin resistance and were also fertile as would be expected for crossed plants containing a wild type copy of DYAD. The ploidy of the six plants was examined using spreads of meiotic chromosomes. It was found that 3 plants were triploid with 15 chromosomes, 2 plants had 16 chromosomes, and 1 had 17 chromosomes. These results suggest the likelihood that female gametes arose from unreduced/hyperdiploid spores. Fertilization of the unreduced female gametes by a haploid pollen would give (near) triploids which be simplex for the kanamycin resistance gene (Kkk). Alternatively the triploids could arise from fertilization of a haploid female gamete by an unreduced male gamete or two reduced male gametes in which case the triploids would be duplex for the kanamycin resistance gene (KKk).<br>
If a simplex condition plant is crossed to a wild type plant that does not carry kanamycin resistance then the segregation ratio for kanamycin resistance to susceptibility in the resulting plants will be 1:1. If however a duplex condition plant is crossed to a wild type plant then the segregation ratio would be expected to be 5:1. Crosses were carried out for two of the triploid plants obtained above to wild type and the seeds obtained were scored for segregation of kanamycin resistance. The results shown in Table 5 indicate 1:1  segregation for kanamycin<br>
resistance ruling out polyspermy, and show that the triploids arise from unreduced female gametes.<br>
Table 5: Segregation of KanR phenotype in crosses<br><br>
(Table Removed)<br>
* Since the seeds are result of a cross of a triploid parent to a diploid parent, a few seeds are<br>
not expected to germinate due to aneuploidy.<br>
** Test of signficance for goodness of fit for 1:1 ratio is calculated excluding ungerminated<br>
seeds<br>
*** Test of significance for goodness of fit for 5:1 ratio is calculated by including the ungerminated seeds in the KanR category. Theoretically only 50% of the ungerminated seeds should be included in either category (based on the ratio of KanR and Kans seedlings obtained) but in order to increase the level of significance we have included the entire ungerminated lot into KanR category. This rules out that even though we include the entire ungerminated lot in KanR category the goodness of fit for 5:1 ratio is not significant and thus strongly support a condition favouring only 1:1 ratio.<br>
S     Significant for x2 test indicating that it does not follow the given ratio NS Non significant for x2 test indicating that it follows the given ratio<br>
EXAMPLE 8: The DYAD gene and coding sequence from poplar {Populus trichocarpa)<br>
An additional example of a DYAD gene from poplar is found at http://www.ornl.gov/sci/ipgc. Translation of the coding portion of the cDNA sequence provides an amino acid sequence that is compared to the amino acid sequence of the wild-type DYAD protein from Arabidopsis thaliana using the Clustal W program in Figurel 1.<br>
At Dyad homologue Populus trichocarpa as in http://www.ornl.gov/sci/ipgc<br>
Genomic region (SEQ ID NO: 24)<br>
EXON<br>
INTRON<br>
(Sequence Removed)<br>
EXAMPLE 9: Identification of Maize DYAD polynucleotides and polypeptides<br>
A search of the maize genome using TBLASTN and the rice DYAD protein (SEQ ID NO: 51) as query at the website (www.plantgdb.org) revealed the presence of a putative DYAD gene within a region of the maize genome corresponding to the contigs ZmGSStucl 1-12-04.1016.1 (SEQ ID NO:52) and ZmGSStucl 1-12-04.1016.2 (SEQ ID NO:53). Annotation of the region using GENSCAN (http://genes.mit.edu) in combination with manual editing led to the identification of putative maize polypeptide sequences that could be aligned with rice DYAD polypeptide sequences (Figure 12). The present invention encompasses the use of the said maize polypeptide sequences and polynucleotide sequences encoding said polypeptides.<br>
The polypeptide sequences obtained from Z. mays are mapped to the contig nucleotide sequences as shown by nucleotide coordinates below.    The assembled partial Zm DYAD polypeptide sequences encoded by the contig sequences are also shown. ZmGSStucl 1-12-04.1016.1 (SEQ ID NO: 52) Coordinates and conceptual translation<br>
(Sequence Removed)<br>
EXAMPLE 10: A General procedure for parthenogenesis<br>
Determination of optimum irradiation dose:<br>
1.	Collect anthers from a male parent plant of the same species or related species as the female parent plant to be used and irradiate with ionizing radiation in a dose range comprising 1, 5, 10, 20, 30, 50, 70, 100, 150, 200 krad.<br>
2.	Pollinate emasculated flowers or female flowers from the female plant that differs from the irradiated pollen parent in carrying one or more recessive phenotypic markers or else with respect to DNA markers (microsatellite, CAPS, or RAPD). Preferably use 10-50 flowers for pollination at each dose of ionizing radiation.<br>
3.	Collect seeds from pollinated flowers and pool seeds from flowers that were pollinated with pollen that received the same radiation dose.<br>
4.	Germinate seeds and grow into plants so as to give about 20-100 plants for each dose of irradiation.<br>
5.	Score the genotype of plants with respect to the phenotypic marker or DNA markers and calculate the proportion of plants that resemble the maternal parent.<br>
6.	Choose a dosage that gives an optimum combination of both a high percentage of<br>
viable plants as well as a high proportion of plants that resemble the maternal parent.<br>
Induction of parthenogenesis in a dyad mutant plant:<br>
1.	Pollinate a dyad mutant plant with pollen irradiated using an appropriate dose of ionizing radiation determined as described above.<br>
2.	Collect seeds.<br>
3.	Germinate seeds and grow into plants.<br>
Identification of parthenogenetic plants:<br>
1.	Score plants with respect to a recessive phenotypic marker carried by the female parent. Plants that show the recessive phenotype are classified as parthenogenetic. In addition the plants may be scored for DNA markers by isolating DNA from plant tissue followed by analysis of DNA with respect to polymorphic markers. Plants showing marker patterns that are characteristic of the female parent and are lacking the marker bands for the male parent are classified as parthenogenetic. The percentage of parthenogenetic plants from a pollination experiment may thus be calculated.<br>
2.	Parthenogenetic plants can be examined for markers for which the female parent was heterozygous. Those plants that retain heterozygosity for all markers for which the female dyad mutant parent was heterozygous are apomictic plants.<br>
References for possible molecular markers that may be used for different crop species are listed<br>
below:<br>
Wheat:<br>
www.gramene.org<br>
1.	Torada et al. (2006). SSR-based linkage map with new markers using an intraspecific population of common wheat. Theor Appl Genet. 2006 Apr; 112(6): 1042-51.<br>
2.	Song et al. (2005). Development and mapping of microsatellite (SSR) markers in wheat. Theor Appl Genet. 2005 Feb;l 10(3):550-60.<br>
Rice:<br>
www.gramene.org<br>
1.	Harushima et al. (1998). A high-density rice genetic linkage map with 2275 markers..." Genetics 148: 479-494.<br>
2.	Causse et al.  (1994).  Saturated  molecular map  of the  rice  genome  based on  an interspecific backcross population. Genetics. 1994 Dec; 138(4): 1251-74.<br>
Maize: Coe et al. (2002). "Access to the maize genome: an integrated physical and genetic map".<br>
Plant Physiol. 128:9-12.<br>
www.gramene.org<br>
Barley:	www.gramene.org<br>
Wenzl et al. (2006). A high-density consensus map of barley linking DArT markers to SSR,<br>
RFLP and STS loci and agricultural traits. BMC Genomics. 2006 Aug 12;7(1):206<br>
Oats:	www.gramene.org<br>
De Koeyer et al. (2004). A molecular linkage map with associated QTLs from a hulless x covered<br>
spring oat population. Theor Appl Genet. 2004 May; 108(7): 1285-98.<br>
Pearl millet:      www.gramene.org<br>
An integrated genetic map and a new set of simple sequence repeat markers for pearl millet,<br>
Pennisetum glaucum. Theor Appl Genet. 2004 Nov; 109(7): 1485-93.<br>
Sorghum: Chittenden et al. (1994). "A detailed RFLP map of Sorghum bicolor ...". Theor. Appl. Genet. 87: 925-933.<br>
Brassica oleracea: Bohuon et al. (1998). "Comparison of a Brassica oleracea genetic map with the genome of Arabidopsis thaliana". Genetics 150: 393-401.<br>
Brassica juncea : Pradhan et al. (2003). A high-density linkage map in Brassica juncea (Indian mustard) using AFLP and RFLP markers. Theor Appl Genet. 2003 Feb;106(4):607-14.<br>
Brassica napus: Piquemal et al. (2005). Construction of an oilseed rape (Brassica napus L.) genetic map with SSR markers. Theor Appl Genet. 2005 Nov;l 11(8): 1514-23.<br>
Brassica rapa: Kole et al. (1997). Genetic linkage map of a Brassica rapa recombinant inbred population. J. Hered. 88:553-557<br>
Cotton: Rong et al. (2004). "A 3347-locus genetic recombination map ..." Genetics 166: 389-417.<br>
Tomato: Zhang et al. (2002). A molecular linkage map of tomato displaying chromosomal locations of resistance gene analogs based on a Lycopersicon esculentum x Lycopersicon hirsutum cross. Genome 2002 Feb;45(l): 133-46.<br>
Eggplant: Doganlar et al. (2002)A comparative genetic linkage map of eggplant (Solanum melongena) and its implications for genome evolution in the solanaceae. Genetics 161(4): 1697-711<br>
Capsicum: Genome mapping in capsicum and the evolution of genome structure in the solanaceae. Genetics 152(3): 1183-202.<br>
Potato: Tanksley et al. (1992). High density molecular linkage maps of the tomato and potato genomes. Genetics 132(4): 1141-1160.<br>
Soybean: Ferreira et al. (2000). Soybean genetic map of RAPD markers assigned to an existing scaffold RFLP map. J. Hered. 91(5): 392-396.<br>
Populus: Yin et al. (2001). Preliminary interspecific genetic maps of the populus genome constructed from RAPD markers. Genome 2001 Aug;44(4):602-9.<br>
Tuskan et al. (2004). Characterization of microsatellites revealed by genomic sequencing of Populus trichocarpa. Canadian J. Forest Res. 34(1): 85-93.<br>
Various articles of the scientific periodical and patent literature are cited herein. Each such article is hereby incorporated by reference in its entirety and for all purposes by such citation.<br><br><br><br><br><br>
We Claim:-<br>
1. An expression construct useful for conditional expression of a wild-type DYAD protein in the nucleus of a plant megaspore mother cell, said construct comprising an isolated polynucleotide of SEQ ID NO: 4, or of SEQ ID NO: 17, or of SEQ ID NO: 23, or of SEQ ID NO: 25, encoding a wild type DYAD protein, or comprising an isolated polynucleotide that hybridizes to the complement of SEQ ID NO: 1, or SEQ ID NO: 23, or SEQ ID NO: 25, said polynucleotide being operably linked to a promoter and terminator that confer expression in plants and the isolated coding region of the DYAD gene is operably linked to a an isolated polynucleotide of SEQ ID NO: 27 encoding a glucocorticoid receptor ligand binding domain so as to provide for expression of a fusion protein .</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1BYnN0cmFjdC0oMDMtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-Abstract-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1DbGFpbXMtKDAzLTAxLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Claims-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1DbGFpbXMtKDA1LTA1LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Claims-(05-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDAzLTAxLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Correspondence Others-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDA1LTA1LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Correspondence Others-(05-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1ERUwtMjAwNS1Db3JyZXNwb25kZW5jZS1PdGhlcnMtKDA3LTA4LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-DEL-2005-Correspondence-Others-(07-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1jb3JyZXNwb25kZW5jZS1wby5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-correspondence-po.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1EZXNjcmlwdGlvbiAoQ29tcGxldGUpLSgwMy0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-Description (Complete)-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1kZXNjcmlwdGlvbiAocHJvdmlzaW9uYWwpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-description (provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTEzLSgwMy0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-13-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTEzLSgwNS0wNS0yMDExKS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-13-(05-05-2011)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTEzLSgwNS0wNS0yMDExKS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-13-(05-05-2011)-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTEzLSgwNS0wNS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-13-(05-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTItKDAzLTAxLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-2-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTItKDA1LTA1LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-2-(05-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTI2LnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTMtKDAzLTAxLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-3-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1Gb3JtLTMtKDA1LTA1LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Form-3-(05-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1ERUwtMjAwNS1Gb3JtLTMtKDA3LTA4LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-DEL-2005-Form-3-(07-08-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1QZXRpdGlvbi0xMzctKDAzLTAxLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3337-del-2005-Petition-137-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzMzNy1kZWwtMjAwNS1wZXRpdGlvbi0xMzcucGRm" target="_blank" style="word-wrap:break-word;">3337-del-2005-petition-137.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251011-method-and-apparatus-for-determining-abortion-of-reception-of-message-in-a-cdma-communication-system.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251013-an-ifnar2-mutant-polypeptide-dna-encoding-the-same-and-vector-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251012</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3337/DEL/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Dec-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC AND INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ANUSANDHAN BHAWAN, RAFI MARG, NEW DELHI-110 001, INDIA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SIDDIQI IMRAN</td>
											<td>CCMB, UPPAL ROAD, HYDERABAD-500007, INDIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MARUTHACHALAM RAVI</td>
											<td>CCMB, UPPAL ROAD, HYDERABAD-500007, INDIA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MARIMATHU MOHAN PREM ANAND</td>
											<td>CCMB, UPPAL ROAD, HYDERABAD-500007, INDIA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251012-nucleic-acids-and-methods-for-producing-seeds-having-a-full-diploid-complement-of-the-maternal-genome-in-he-embryo by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:07:22 GMT -->
</html>
