<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251093-repair-of-nucleic-acids-for-improved-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:39 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251093:REPAIR OF NUCLEIC ACIDS FOR IMPROVED AMPLIFICATION</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">REPAIR OF NUCLEIC ACIDS FOR IMPROVED AMPLIFICATION</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods and compositions are provided for repairing a polynucleotide so that it can be copied with improved fidelity and/or yield in, for example, an amplification reaction. This involves the use of a reaction mixture that includes a ligase and a cofactor selected from NAD+ or ATP and incubating the polynucleotide with the reaction mixture in the absence of Endonuclease VI. The reaction mixture may further contain an AP endonuclease and a polymerase. If used, these enzymes may be selected according to their ability to withstand high temperatures. For example, the reaction mixture may be used prior to a polynucleotide synthesis reaction in which case enzymes that are not thermophilic may be used. The repair reaction is not time sensitive with respect to seconds, minutes or hours of incubation in the enzyme mixture in as much as the repair is effected rapidly and prolonged incubation is not generally adverse.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 <br>
(39 of 1970)<br><br>
&amp; <br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
REPAIR      OF      NUCLEIC      ACIDS      FOR IMPROVED AMPLIFICATION;<br>
NEW ENGLAND BIOLABS, INC., A CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF UNITED STATES OF AMERICA, WHOSE ADDRESS IS 240 COUNTY ROAD, IPSWICH, MA 01938, U.S.A.<br>
THE	FOLLOWING	SPECIFICATION<br>
PARTICULARLY	DESCRIBES	THE<br>
INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED.  <br><br>
BACKGROUND<br>
Copying of polynucleotides, more particularly amplification, is commonly used in molecular biology for studying, for example, the properties of genes. Problems in copying arise when the polynucleotide is damaged in some way.<br>
By way of illustration, U.S. Patent No. 5,035,996 describes a process for controlling contamination of polymerase chain reaction amplification reactions that uses the modified nucleotide, dUTP, in the amplification reaction. This process uses uracil DNA glycosylase (UDG) to eliminate those PCR products containing uracil to prevent contaminating subsequent PCR reactions*  U.S. patent publication No. 2004-0067559 Al also relies on modified bases in primer DNA prior to amplification and uses, for example, dUTP for incorporation into the amplicon. The amplicon can then be fragmented by adding, for example, UDG.and Endonuclease (Endo) IV.<br>
One amplification methodology referred to as hot start nucleic acid amplification has been used to lower mis-priming during polymerase chain reaction (PCR). In one type of hot start amplification, prevention of extension by the polymerase relies on the presence of a PCR primer with a blocked 3' terminus in the PCR reaction (see for example U.S. Publication No. 2003-0119150). The primer is unblocked by a thermostable 3'-5' exonuclease that is active at &gt;37°C. Therefore, the polymerase will only extend the PCR primers once the exonuclease unblocks the 3' end at &gt;37°C. Alternatively the Taq polymerase is blocked and then activated at amplification temperatures.<br><br><br>
Barnes, W. M. Proc. Natl. Acad. Sci. USA 91:2216-2220 (1994) describes the use of Vent® polymerase and Taq polymerase as an improvement over the use of Taq polymerase only in amplification.  Ghadessy et al. reported a mutant Taq polymerase that is not halted by damaged or abasic sites (Ghadessy et al. Nature Blotechnol. 22(6):755-9 (2004)).<br>
It has been reported that conventional amplification techniques are compromised if the DNA is substantially damaged (Di Bernardo et al. Nucl. Acids Res.30:el6 (2002)).  Degradation and/or fragmentation of DNA resulting from exposure to the environment and microorganisms which contain DNA nucleases is a frequent problem in forensics, diagnostic tests and routine amplification and affects fidelity and yield of the amplification product.  In addition, the problem of degraded DNA is also faced by researchers who are analyzing the DNA obtained from extinct or extremely rare organisms that have been stored, frozen or fossilized.<br>
Fromenty, B., et al. Nucl. Acids Res. 28(ll):e50 (2000) and International Publication No. WO/0151656 reported that treatment with Exonuclease (Exo) III improved yields of long PCR.  However, Fromenty also reported decreased yields of amplicon for DNA
Di Benardo et al. Nucl. Acids Res. 30(4):el6 (2002) described the use of T4 DNA ligase (T4 ligase) and an E coli polymerase as a pretreatment to amplify short regions of single-stranded DNA between cross-linked regions of double-stranded DNA.<br><br><br>
Another approach to amplification of damaged DNA has been described in U.S. Publication No. 2003-0077581.  Degraded nucleic acid was hybridized to non-degraded nucleic acid having a sequence homologous to the degraded nucleic acid. Regions of the degraded nucleic acid were then filled in with nucleotide precursors. The fragmented strands were then covalently linked using a polymerizing and/or ligating enzyme.<br>
Preparations for improving amplification of damaged DNA can be obtained commercially from Sigma, St. Louis, MO and Qbiogene, now MP Biomedicals, Irvine, CA. Although the compositions of these preparations are not stated, it is assumed that Exo III is contained in the preparation.<br>
Others report the use of a combination of E. coli DNA Poll and T4 ligase for pre-amplification repair (Pusch, et al., Nucl. Acids Res. 26:857 (1998)). However, according to Pusch et al., the pre-amplification product must be purified before initiation of amplification.<br>
SUMMARY<br>
In an embodiment of the invention, a method is provided for enhancing at least one of fidelity and yield of a copied or amplified product by repairing a damaged polynucleotide where the method may include: incubating the polynucleotide in the absence of Endonuclease VI (Endo VI) in a reaction mixture comprising a ligase and at least one of NAD+ or ATP as a cofactor. The polynucleotide may be derived from any natural or synthetic source including from preserved biological material, forensic evidence, ancient material of biological origin, and tissue biopsy material. Examples of damage to<br><br><br>
the polynucleotide can include at least one of a apurinic/apyrimidinic (AP) site, a chemically modified nucleotide, a nick, a gap and DNA-DNA, DNA-protein cross-links, DNA-RNA crosslinks or RNA-RNA cross-links.<br>
In one example of the method, the ligase is a thermostable ligase. Taq DNA ligase is an example of a thermostable ligase that uses NAD+ as a cofactor. E. coli DNA ligase is another example of a ligase where the cofactor is NAD+. Other components of the reaction mixture containing the ligase may include one or more of a T7 Endo I or mutant thereof, a polymerase and an AP endonuclease. Examples of a polymerase for use in the reaction mixture include a Taq polymerase, an E. coli polymerase, or an archaeal polymerase or mutant thereof.  Examples of archeal polymerases or chimeric polymerases derived therefrom include one or more of a Pfu, Vent®, Deep Vent®, 9°North, KOD and Phusion™ polymerases and Pwo.  Examples of other polymerases for use in the reaction mixture include members of the Y family of polymerases such as E. coli polIV, E. coli polV, human pol kappa, human pol eta, Sso Dpo4, Sac Dbh, See pol zeta, and human pol iota.<br>
Examples of an endonuclease for use in the reaction mixture may include at least one of: at least one endonuclease from any of Phage T4, E. coli, human and Thermus species. The reaction mixture may further include one or more enzymes selected from the following: T4 pyrimidine dimer glycosylase, [fapy]-DNA glycosylase (Fpg), UvrA, UvrB, UvrC, UvrD, Cho, Endo V, Endo III, Endo VIII, UDG, lambda beta protein, RecA and Aag.<br><br><br>
In an embodiment of the invention, improved yield is achieved after copying the polynucleotide by means selected from the group consisting of: PCR amplification, helicase-dependent amplification, transcription-mediated amplification, strand-displacement amplification, rolling circle amplification and whole genome amplification. Where the polynucleotide is an RNA, the amplification may be RT-amplification. Where the polynucleotide is DNA, the reaction mixture may comprise 1-100 units of E. coli Endo IV, 0.05-0.25 units of E. coli Poll and 5-500 units of Taq ligase added to 1-1000 ng DNA in a reaction volume of 10-1000 fxL<br>
Improved or enhanced yield is determined where less than about 50% of the concentration of the starting polynucleotide preparation provides at least a similar amount of copied product obtainable absent the reaction mixture. Preferably, the improvement should be reproducible meaning that about 75% or more of replicate samples will demonstrate the improvement. Moreover, optionally, an additional feature of the method is that it is capable of achieving an improved result at a single temperature of incubation.<br>
In an embodiment of the invention, a method is provided where amplification of the polynucleotide is capable of producing an amplicon in a size range of 50 nucleotides to 100,000 nucleotides in a polymerase chain reaction.<br>
In an embodiment of the invention, a kit is provided that includes one or more enzymes wherein at least one of the enzymes is a ligase, the one or more enzyme being formulated for addition to a polynucleotide preparation to enhance (as described above) copying of the polynucleotide and instructions for their use.<br><br><br>
In an embodiment of the invention, a composition is provided that includes an effective amount of a ligase, a polymerase, and an AP endonuclease, not including Endo VI, for providing enhancement of at least one of yield and fidelity of a copied polynucleotide compared with a copied polynucleotide in the absence of the composition. In an example of the composition, the concentration of certain enzymes is as follows: 1-100 units of AP endonuclease, 0.05-5 units of the polymerase, and 5-500 units of the ligase in a volume 10-1000 mL for use in a pre-amplification mixture.<br>
Examples of particular enzymes in the composition include an AP endonuclease, more particularly Type II AP endonuclease, more particularly, E. coli Endo IV. The polymerase is exemplified by E. coli poll. In addition, at least one of the following enzymes may be further included in the composition: T4 pyrimidine dimer glycosylase, [fapy]-DNA glycosylase (Fpg), UvrA, UvrB, UvrC, UvrD, Cho, Endo V, Endo III, Endo VIII, UDG, RecA, lambda beta protein and Aag.<br>
In an embodiment of the invention, a method is provide for enhancing the yield of a copied or amplified polynucleotide, which includes: (a) obtaining at least a first pair and a second pair of primers wherein the second pair of primers is nested within the first set of primers when hybridized to the polynucleotide; (b) subjecting the polynucleotide to a composition described above; (c) amplifying the polynucleotide with the first set of primers; (d) amplifying the product of (c) with the second set of primers; and (e) obtaining an enhanced yield of amplified polynucleotide.<br><br><br>
In an additional embodiment of the invention, a method is provided for cloning a polynucleotide fragment that includes: repairing sequence errors in the polynucleotide fragment by means of a composition such as described above. This composition may have the advantageous property of blunt-ending the polynucleotide fragment, which eliminates a step in cloning a fragment into a vector.<br>
In an additional embodiment of the invention, a method is provided for sequencing a polynucleotide, which includes contacting the polynucleotide with the composition described above prior to sequencing. The quality of the sequence results is generally improved with respect to sensitivity and to accuracy.<br>
In an additional embodiment of the invention, a method for copying or amplifying a fragmented DNA is provided which involves (a) contacting the fragmented DNA with the composition above; (b) optionally adding a recombination competent protein; and c) amplifying or copying the fragmented DNA. Examples of recombination competent proteins are RecA and beta protein from, for example, lambda phage.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Figures 1A-1D show enhanced amplicon yield from heat-damaged lambda DNA after pre-incubation with specified enzymes.<br>
Figure 1A shows DNA template damaged to differing extents by heat and the effect of this damage on amplification of a 5 kb segment of lambda DNA where 5 ng, 2 ng and 1 ng of heat-treated lambda DNA were amplified after prior damage by 99°C heat<br><br><br>
treatment for 0 sec, 30 sec, 60 sec, 90 sec, 120 sec or 180 sec. The damaged DNA was not subjected to enzyme treatment prior to amplification. The amount of amplification was determined after electrophoresis and was found to be substantially reduced by 120 sec heat treatment. The first lane on the gel contains 1 mg of a 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Figure IB shows enhanced amplicon yields from heat-damaged lambda DNA using Taq ligase, E. coli Endo IV and E. coli Poll on amplification of a 5 kb segment of lambda DNA.  DNA was heat damaged as described in Figure 1A but the damaged DNA was subjected to enzyme treatment prior to amplification. The results of amplification are shown after a 10-minute pretreatment reaction with Taq ligase, E. coli Endo IV and E coli Poll. The amplicon yield was increased throughout but was especially noticeable with 120 sec and 180 sec heat damaged DNA. First and last lanes on the gel contain 1 mg of a 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA.<br>
Figure 1C shows enhanced amplicon yields from heat-damaged lambda DNA using Taq ligase, Tth Endo IV and E. coli Poll. The amplification was performed according to Figure IB but the enzyme treatment prior to amplification contained Thermus thermophilus (Tth) Endo IV in place of E coli Endo IV. The results of amplification are shown after a 10-minute pretreatment reaction with Thermus aquaticus (Taq) ligase, Tth Endo IV and E coli Poll. The amplicon yield was increased throughout but was especially noticeable with 120 sec and 180 sec heat-damaged DNA. Only the first lane contains the molecular weight marker ladder.<br><br><br>
Figure ID shows enhanced amplicon yields from heat-damaged lambda DNA using E coli ligase, E coli Endo IV and E coli DNA poll. The amplification was performed according to Figure IB but the enzyme treatment prior to amplification contained E coli ligase in place of Taq ligase. The lambda DNA subjected to 99°C for 180 sec was used as a template. The amount of template DNA used is indicated above each lane. The yield of 5kb amplicon is enhanced for each of the template amounts by enzyme pretreatment.<br>
Figures 2A and 2B show the effect of citrate buffer pH 5 treatment of template DNA on amplicon yield.<br>
Figure 2A shows the results of amplification of a 5 kb segment of lambda DNA where lambda DNA was heated to 70°C in citrate buffer pH 5 for 0, 20, 40, 80, 120, and 160 minutes. 50 ng, 10 ng and 5 ng of each citrate treated sample were amplified and the resulting products were visualized on a gel to determine the extent of amplification. The DNA was not treated with selected enzymes prior to amplification. The last lane on the right contains 1 mg of 2-log ladder.<br>
Figure 2B shows the increase in yield of a 5 kb amplicon of lambda DNA regardless of which polymerase was used in the enzyme mixture. 120-minute citrate-damaged lambda DNA was treated with various enzymes prior to amplification.<br>
Lane 1:   mg 2-log ladder (NEB# N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 2: no pretreatment.<br>
Lane 3:  Pretreatment with Taq ligase, Taq DNA polymerase and E coli Endo IV.<br><br><br>
Lane 4:  Pretreatment with Taq ligase, E. coli Poll, and E. coli Endo IV.<br>
Lane 5: Pretreatment with Taq ligase, Taqr:Vent® DNA polymerase mix, and E. coli Endo IV.<br>
Figure 3 shows the results of amplification of a 200 bp segment of krill genome that has been extracted from an ethanol stored sample of krill and pretreated with an enzyme mixture containing one of various polymerases, a ligase and an AP endonuclease that enhances amplification yields.<br>
Lane 1:  No pretreatment of krill DNA with enzymes. Lane 2: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and Taq polymerase.<br>
Lane 3: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and Vent® polymerase.<br>
Lane 4: Pretreatment of krill DNA with Taq ligase, E. coli Endo IV, and 50:1 Taq:Vent® polymerase.<br>
Figure 4 shows an increase in yield of a 10 kb amplicon from heat-damaged DNA. 180 sec heat-damaged DNA was pretreated with an enzyme mixture and then amplified.<br>
Lane 1: 1 mg of a 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 2:   Pre-treatment with Taq ligase, E. coli Endo IV, and E. coli Poll.<br>
Lane 3:  Pre-treatment with Taq ligase and E. coli Endo IV.<br>
Lane 4:   Pretreatment with Taq ligase.<br>
Lane 5:  Control - untreated DNA.<br><br><br>
Figure 5 shows that ligase pretreatment increases amplicon yield from environmental DNA (soil sample extract).<br>
Lane 1: A 2-log ladder size standard (NEB# N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 1: No enzyme pretreatment.<br>
Lane 2: Pre-treatment with T4 ligase.<br>
Lane 3: No enzyme pre-treatment.<br>
Lane 4: Pretreatment with Taq ligase.<br>
Figures 6A-1 - 6A-9 and 6B-1 - 6B-2:   Blast P search at NCBI using E. coli DNA ligase (A) and T4 DNA ligase (B).<br>
Figure 7: DNA sequence of Tth Endo IV (SEQ ID NO:ll).<br>
Figures 8A, 8B and 8C show the effect of UV light on amplicon yield using lambda DNA.<br>
Figure 8A: Lambda DNA is subjected to UV irradiation for up to 50 sec and a slight reduction in yield of a 2 kb amplicon produced is shown.<br>
Figure 8B: Lambda DNA is subjected to UV irradiation for up to 50 seconds and the reduction in yield of a 5kb amplicon is shown.<br>
Figure 8C: The effect of various reaction mixtures added to lambda DNA on yield of a 5 kb amplicon after UV irradiation is shown.<br>
Lanes 2-7 are controls in the absence of a reaction mixture.<br>
Lanes 8-13 show the increased beneficial effect of adding ligase, polymerase and AP endonudease plus 10 units of T4 pdg.<br>
Lanes 14-19 show the increased beneficial effect of adding ligase, polymerase and AP endonudease plus 80 units of T4 pdg.<br><br><br>
Lanes 1 and 20: A 2-log ladder size standard (NEB#N3200, New England Biolabs, Inc., Ipswich, MA).<br>
Figures 9A and 9B show that adding ligase to T7 Endo I expands the useful range of the EndoI:DNA ratio in which the product is not degraded.  Taq ligase and T7 Endo I were added to supercoiled DNA in varying amounts as indicated for each lane.<br>
Figure 9A is the control in which no Taq ligase has been added but increasing amounts of T7 Endo I are used. The supercoiled DNA is predominantly cleaved into fragments of various sizes with 12.5-25 units of T7 Endo I.<br>
Figure 9B shows how the addition of 100 units of Taq ligase protects DNA from non-specific cleavage in the presence of T7 Endo I such that even at 200 units of T7 Endo I, there is a clear band corresponding to linear DNA not present in the absence of ligase.<br>
Figures 10A and 10B show the effect of repair enzyme treatment on amplicon yield from oxidatively damaged DNA or undamaged template.<br>
Figure 10A shows that the addition of repair enzymes to an undamaged template, pWB407 has no effect on amplicon yield.<br>
Figure 10B shows that the addition of Fpg to a damaged template, plasmid pWB407, which was previously incubated in the presence of methylene blue, gives inconsistent effects on yield. The addition of Taq, ligase, E. coli DNA polymerase, and E. coli Endo IV<br><br><br>
in the presence or absence of Fpg consistently increases amplicon yield.<br>
Figure 11 shows increased PCR reaction fidelity from damaged DNA after treatment with repair enzymes. Repair enzyme treatment of undamaged template, plasmid pWB407, prior to PCR has no significant effect on fidelity. Treatment of a damaged template, plasmid pWB407 incubated with methylene blue, with Fpg alone or also with Taq ligase, E. coli DNA polymerase I, and E coli endonuclease increases the fidelity of PCR. The measure of fidelity is the number of white colonies verses the number of blue colonies after cloning a lacZ-containing amplicon as discussed below. The higher the percentage of white colonies the greater the error rate.<br>
Figure 12 shows a flow diagram for treating damaged DNA or to increase at least of one of fidelity or yield.<br>
Figures 13A and 13B show how yield of amplicon is increased for a 5 kb fragment of 30s UV damaged DNA incubated for 15 minutes at room temperature or at 4°C overnight with a multi-enzyme repair mix.<br>
Figure 13A:  Room temperature incubation: Lane 1: 2-log ladder DNA molecular weight standard. Lanes 2 and 3: the two reactions incubated without the multi-enzyme repair mix at room temperature for 15 minutes. Lanes 4 and 5: the reactions incubated with the repair mix at room temperature for 15 minutes have the expected 5 kb amplicon.<br>
Figure 13B:  4°C incubation:<br>
Lane 1: 2-log ladder DNA molecular weight standard.<br><br><br>
Lanes 2 and 3: the two reactions incubated without the multi-enzyme repair mix overnight at 4°C. Lanes 4 and 5: the reactions incubated with the repair mix overnight at 4°C have the expected 5 kb amplicon.<br>
Figure 14 shows enhanced amplicon yield from a uracil-containing plasmid after treatment with a repair enzyme mix for 15 mins at room temperature and PCR amplification using an archaeal polymerase.  Lanes 1 and 2 shows the product of PCR amplification of pl\IEB0.92U using vent DNA polymerase. There is a weakly visible band at 920 bp. Lanes 3 and 4 show the product of PCR amplification from pNEB0.92U treated with a repair enzyme mix.<br>
Figure 15 shows an agarose gel on which a band corresponding to an amplified DNA of 620 base pairs is identified. The 620 bp amplicon was obtained from 20 overlapping single strand oligonucleotides of 48 nucleotides or smaller.<br>
Lane 1: 2-log DNA molecular weight standards (NEB#N3200S, New England Biolabs, Inc., Ipswich, MA).<br>
Lane 2: 20 oligonucleotides incubated with 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, and 20 units EndoIV during the assembly step.<br>
Lane 3: 20 oligonucleotides incubated with 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, 20 units Endo IV, and lambda beta protein during the assembly step.<br>
Lane 4: 20 oligonucleotides incubated with 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, 20 units Endo IV, and E. coli RecA during the assembly step.<br>
Lane 5: 20 oligonucleotides incubated with 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, 20 units Endo IV, lambda beta protein and RecA during the assembly step.<br><br><br>
Lane 6: the control, 20 oligonucleotides with no added repair enzymes during the assembly step.<br>
Figure 16 shows the effect of DNA repair treatment on non-irradiated (-) and irradiated DNA (+) as determined by the number of colonies obtained when the DNA containing a selection marker is used to transform cells.<br>
Figure 17 shows improved yields of amplicon from ancient cave bear DNA after 2 sets of amplification reactions using different nested primer pairs. The gene map shows the location of primer pairs Fl-Rl, F1-R2 and F1-R4. Above the gels, a set of numbers is provided (88, 79, 10, 11, 1868 and 1314) that represent the estimated amount of mitochondrial DNA in each sample. Lanes 3A and 3B contain the most DNA. +/- indicates whether a repair mix was used prior to the first amplification using Fl-Rl. In Lane 3B, a sharp band corresponding to repaired cave bear DNA was observed that was not present in the absence of repair.<br>
Figure 18 shows the DNA sequence for plasmid pNEB0.92U (SEQ ID NO:42).<br>
DETAILED DESCRIPTION OF EMBODIMENTS<br>
Embodiments of the invention provide improvements in the method for copying or more specifically amplifying polynucleotides. Enhancing at least one of yield and fidelity of copied or amplified polynucleotides is associated with obtaining a detectable yield of polynucleotides after copying or amplification of 50% of the polynucleotides needed in the absence of a reaction mixture containing selected repair enzymes as exemplified in present<br><br><br>
embodiments. 'Fidelity' relates to copying in which the error rate is less than would otherwise be the case in the absence of a reaction mixture containing selected repair enzymes as exemplified in present embodiments.<br>
The inventive method utilizes a reaction mixture containing enzymes.  If multiple types of damage are suspected, then a combination of enzymes can be constructed as illustrated by the specific cocktails used in the examples.  If the damage to the polynucleotide is known, then a preferred cocktail of enzymes may be selected as illustrated in the examples. If the nature of the damage is unrecognized, a universal mixture of enzymes can be employed. In general, adding enzymes together in one step does not preclude adding enzymes sequentially.<br>
Where polynucleotide copying leads to polymerase-dependent amplification, short amplicons that are less than about 500 bases in length (as short as 100 base pairs) or long amplicons that are greater than 500 bases or as much as about 50 kb may be amplified (for PCR, RT-PCR and qPCR amplification).  Other types of amplification can produce amplicons having a wide range of sizes. For example, polynucleotides having a size as small as 100 bases or as large as a whole genome (3 billion bases for humans). Examples of such amplification methodologies include: whole genome amplification, rolling circle amplification (RCA) and helicase-dependent amplification (HDA). Other types of polynucleotide synthesis include primer extension reactions such as sequencing reactions. Embodiments of the methods have wide utility in molecular biology research and in solving problems in applied biology, including for example, analyzing fragmented and damaged DNA such as found in forensic analysis, in biological archeology in<br><br><br>
which it is desirable to analyze DNA from ancient sources, for taxonomy where it is desirable to analyze DNA from environmental samples such as required for the Barcode of Life Project, and for diagnostic assays including tissue biopsies to determine a disease susceptibility or status. Other uses include: high-fidelity sequencing, gene assembly, fragment analysis and one step copying and ligation for cloning.<br>
Source and Extent of Damage<br>
Most polynucleotides that are isolated or in vitro replicated polynucleotides are damaged to some extent. Damage of a polynucleotide may result from chemical modification of individual nucleotides or disruption of the polynucleotide backbone. Polynucleotides experience damage from diverse sources such as chemicals including formaldehyde and ethanol, environmental factors, temperature extremes, oxidation, dessication and ultra violet light. Various types of damage include: (a) apurinic or apyrimidinic damage caused for example by heat, storage of polynucleotides in ethanol, and exposure to factors in the environment such as H2O or extremes of pH; (b) modification of individual nucleotides, caused for example by deamination, alkylation, and oxidation; (c) nicks and gaps caused for example by heat, storage of polynucleotide in ethanol, and exposure to factors in the environment such as H2O or extremes of pH; (d) cross-linking caused for example, by formaldehyde, light or environmental factors; (e) mismatched DNA caused by for example misincorporation of a nucleotide by a polymerase; and (f) fragmentation of DNA.<br><br><br>
Damage is more severe in preserved tissues, dried specimens or polynucleotides that are exposed to the environment. Damage can occur as a result of the age of the sample or its source or its preparation. In addition, damage can occur during the application of a methodology for polynucleotide synthesis such as occurs during PCR amplification, which involves a high temperature step. Hence most polynucleotides are damaged to some extent. This damage has a greater influence when longer amplicons are analyzed since the likelihood of encountering damage during amplification is increased.<br>
Polynucleotides can sustain damage in a variety of ways. Different polynucleotide preparations experience different types of damage depending upon, for example, the storage or handling of the polynucleotide preparation in vitro, how prokaryotic cells, archaea or eukaryotic cells containing the polynucleotides are stored and the characteristics of the cells from which the polynucleotides are extracted. Synthetic polynucleotides can sustain damage during chemical synthesis.<br>
The term "polynucleotide" refers in particular to double-stranded DIMA, double-stranded RNA, hybrid DNA/RNA duplex, single-stranded DNA and single-stranded RNA.<br>
A "repair enzyme" refers in particular to a psychrophilic, mesophilic or thermophilic enzyme that participates in the process of repair of a polynucleotide. For example, a repair enzyme may induce breakage of the polynucleotide at a bond, thereby facilitating removal of damaged regions of the polynucleotide or removal of single nucleotides. Enzymes with a synthetic role such as ligases and polymerases are also repair enzymes. In embodiments of the<br><br><br>
invention, one or more selected repair enzymes are provided in a reaction mixture. The damaged DNA is subjected to the reaction mixture so as to enhance copying and/or amplification of DNA. "Enhancing" refers to obtaining an improved ratio of copied or amplified product to starting material compared to the ratio observed in the absence of the repair mixture.<br>
DNA repair enzymes are described in the scientific literature, for example, see Wood, R.D., et al. Mutat. Res. 577(1-2):275-83 (2005) and Eisen, J.A. and Hanawalt, P.C. Mutat. Res. 435(3): 171-213 (1999). A list of human repair enzymes is provided in Table 1 below. Although not described in Table 1, the homologs of the listed enzymes and other functionally related enzymes are included in the description of repair enzymes. Any of the above enzymes may be naturally occurring, recombinant or synthetic. Any of the enzymes may be a native or in Wtro-created chimeric protein with several activities. The methods of searching the databases to identify related enzymes that share conserved sequence motifs and have similar enzyme activity are known to a person of ordinary skill in the art. For example, the NCBI web site (www.ncbi.com) provides a conserved domain database. If, for example, the database is searched for homologs of Endo IV, 74 sequence matches are recovered. (Also see Figures 6A-1 - 6A-9 and 6B-1 - 6B-2 for ligases.)<br>
A "polynucleotide cleavage enzyme" used in enzyme mixtures for repairing damaged DNA refers in particular to a class of repair enzymes and includes AP endonucleases, glycosylases and lyases responsible for base excision repair.<br><br><br>
A damaged base can be removed by a DNA glycosylase enzyme which hydrolyses an N-glycosylic bond between the deoxyribose sugar moiety and the base.  For example, an E. coli glycosylase, UDG endonuclease, repairs deaminated cytosine while two 3-mAde glycosylases from E. coli (TagI and Tagil) repair damage from alkylating agents.<br>
The product of removal of a damaged base by a glycosylase is an apurinic or apyrimidinic site (AP site) that must be correctly replaced. This can be achieved by an endonuclease, which nicks the sugar phosphate backbone adjacent to the AP site. The abasic sugar is removed and a new nucleotide is inserted by polymerase/ligase activity. These repair enzymes are found in prokaryotic and eukaryotic cells. Some enzymes having applicability herein have glycosylase and AP endonuclease activity in one molecule. Abasic sites can be recognized and cleaved by AP endonucleases and/or AP lyases. Class II AP endonucleases cleave at AP sites to leave a 3' OH that can be used in polynucleotide polymerization. Furthermore, AP endonucleases can remove moieties attached to the 3' OH that inhibit polynucleotide polymerization.  For example a 3' phosphate can be converted to a 3' OH by E. coli Endo IV. AP endonucleases can work in conjunction with glycosylases.<br>
Examples of glycosylase substrates include Uracil, Hypoxanthine, 3-methyladenine (3-mAde), Formamidopyrimidine, 7,8 dihydro-8-oxyguanine and Hydroxymethyluracil. The presence of uracil in DNA may occur due to mis-incorporation or deamination of cytosine by bisulfate, nitrous acids, or spontaneous deamination. Hypoxanthine generally occurs due to deamination of adenine by nitrous acids or spontaneous deamination.  In this context, 3-mAde is a product of alkylating agents. Formamidopyrimidine (FAPY)<br><br><br>
(7 -mGua) is a common product of methylating agents of DNA. 7,8-dihydro-8 oxoguanine is a mutagenic oxidation product of guanine. Gamma radiation produces 4,6-diamino-5-FAPY. Hydroxymethyuracil is created by ionizing radiation or oxidative damage to thymidine.<br>
These different types of damage may be repaired using glycosylases of the sort described above and in Table 1.<br>
Another type of repair enzyme is a lyase. This enzyme can break the phosphodiester bond in a polynucleotide.<br>
Examples of AP endonucleases belong to 4 classes.<br>
(I)	cleaves 3'--&gt; 3'-OH + 5'-P - and has associated glycosylase activity.<br>
(II)	cleaves 5' --&gt; 3'-OH + 5'-P<br><br>
(III)	cleaves 3' --&gt; 3'-P + 5'-OH<br>
(IV)	cleaves 5' --&gt; 3'-P + 5'-OH<br>
Several enzymes have been isolated that appear to have AP endonudease or lyase and glycosylase activities that are coordinated either in a concerted manner or sequentially.<br>
Examples of polynucleotide cleavage enzymes now found to be suitable for use in enhancing at least one of yield or fidelity in a copying or amplification reaction include the following types of enzymes derived from but not limited to any particular organism or virus: 1) AP endonucleases, such as E. coli Endo IV, Tth Endo IV, and human AP endonudease; 2) glycosylases, such as UDG, E. coli AlkA and human Aag; 3) glycosylase/lyases, such as E. coli Endo<br><br><br>
Ill, E coli Endo VIII, E. coli Fpg, human OGG1, and T4 pyrimidine dimer glycosylase (T4 pdg); and 4) lyases such as E coli Endo V.<br>
Endo VI (also termed Exo III) is known to be capable of degrading a substantial portion of a polynucleotide outside the damaged regions of interest in a polynucleotide in a few hours under normal reaction conditions, hence is not to be included in the present enzyme mixtures for treating damaged polynucleotides.<br>
A "polymerase" for present purposes refers to an enzyme that has polymerase activity even though it may have other activities. The same polymerase or polymerases may be used throughout the repair and copying reactions or different polymerases may be employed at different stages of the present methods.<br>
Examples of polymerases include thermostable bacterial polymerases such as Taq DNA and Thermus thermophilus polymerases and archeal polymerases such as Vent®, Deep Vent® and Pfu; less thermostable enzymes such as Bst polymerase, thermomicrobium roseum polymerases and mesophilic polymerases such as phage polymerases (such as phi29 polymerase, T7 polymerase and T4 polymerase), E coli poll and E. coli polll Y family polymerases such as E. coli pol IV, E coli pol V, human pol kappa, human pol eta, Sso Dpo 4, Sac Dbh, See pol zeta, human pol iota (MacDonald et al. Nucleic Acids Res. 34:1102-1111 (2006); Vaisman et al. DNA Repair 5:210 (2006); Ohmori et al. Mol. Cell. 8:7-8 (2001); Goodman Ann. Rev. Biochem. 71:17-50 (2002)) or mutants, derivatives or modifications therefrom. Examples of derivatives include Phusion™ enzyme (Finnzymes, Espoo, Finland) and other polymerases that combine a double<br><br><br>
strand binding protein with polymerase sequences from one or several sources.<br>
A "ligase" as used in the enzyme mixtures described here refers to an enzyme that joins a 5' end of a single strand of a polynucleotide to a 3' end of another single strand of a polynucleotide. Such ligases are found in substantially all eukaryotic, prokaryotic, and archaeal cells, and can also be found in some bacteriophages and viruses. Examples of suitable ligases include 9°N ligase, E. coli ligase, T4 ligase and Taq ligase. Other ligases include LIGA (NP-416906.1), TthDNALGS (AAA27486.1), LIG3 (NM-013975) and LIG4 (NM-002312).<br>
Other ligases or ligase-like proteins that may have utility herein are revealed by a Blast search using T4 ligase or E. coli ligase to search the database (see Figures 6A-1 - 6A-9 and Figures 6B-1 and 6B-2) in which any enzyme sharing at least 6 contiguous amino acids with these known ligases may be included in a repair mixture according to embodiments of the invention.<br>
Contrary to a published use of ligase in combination with Exo III in the absence of any cofactors (U.S. Publication No. 2005-0026147), it has been found here that NAD+ or ATP is required in enzyme mixtures that include ligase. More specifically, Taq ligase and E. coli ligase require NAD+ while T4 ligase and 9°N ligase require ATP.<br>
Certain ligases, polymerases and endonucleases are available from New England Biolabs Inc., Ipswich, MA where pages 107-117 of the 2005-2006 catalog are incorporated by reference (pp. 102-108 for ligases) and described in U.S. Provisional Application No.<br><br><br>
60/717,296 and International Publication No. WO 2005/052124. In addition, thermostable repair enzymes can be used interchangeably with thermolabile repair enzymes in a preamplification mixture. Thermostable enzymes retain activity at above 40°C or more particularly 65°C or above.<br>
Embodiments of present methods improve the yield or fidelity of products resulting from copying polynucleotide sequences, amplification or other synthesis reaction. This can be achieved, for example, when a damaged polynucleotide is treated with a preparation of enzyme(s) in a pre-incubation mixture and/or during amplification.<br>
Amplification protocols that may benefit from the above described pre-incubation include polymerase chain reaction (PCR), Strand-Displacement Amplification (SDA) (U.S. Patent Nos. 5,455,166 and 5,470,723); Helicase-Dependent Amplification (HAD) (U.S. Publication No. 2004-0058378-A1); Transcription-Mediated Amplification (TMA) (Guatelli etal., Proc. Natl. Acad. Sci. USA 87:1874-1878 (1990)); Rolling Circle Amplification (RCA) which generates multiple copies of a sequence for the use in in vitro DNA amplification adapted from in vivo rolling circle DNA replication (see, for example, Fire and Xu, Proc. Natl. Acad Sci. USA 92:4641-4645 (1995); Lui, etal., J. Am. Chem. Soc. 118:1587-1594 (1996); Lizardi, etal., Nature Genetics 19:225-232 (1998)); and whole genome amplification methods. (Hawkins et al. Current Opinions in Biotechnology 13:65-67 (2002)).<br>
A "universal" enzyme mixture has now been found to be useful in a reaction mixture for generally repairing damaged polynucleotides prior to or during copying or amplification. In<br><br><br>
accordance with the present embodiments, there are also specific advantageous combinations in given situations.<br>
The universal enzyme mixture contains a ligase and a cofactor such as NAD+ or ATP. The mixture preferably additionally includes a polymerase and an AP endonuclease as described above within a suitable buffer such as Thermopol (New England Biolabs, Inc., Ipswich, MA), AccuTaq LA DNA polymerase buffer (Takara Bio Inc., Shiga, Japan) or any other conventional Taq buffer.  In various illustrative embodiments, the universal enzyme mixture contains E. coli Poll or Taq polymerase and an AP endonuclease such as a mesophilic Endo IV, e.g., E. coli Endo IV or a thermophilic Endo IV, e.g., Tth Endo IV and a ligase selected from E. coli ligase, Taq ligase or an archaeal ligase such as 9°N ligase.  In a particular embodiment, the enzyme mixture contains 1-100 units Endo IV, 0.05-0.25 units E. coli Poll, and 5-500 units of a Taq DNA ligase suitable for repairing 1-1000 ng DNA in a reaction volume of 10-1000 mL prior to or during amplification. It will be understood that the concentration range for endonucleases and polymerases other than those specified in the universal mixture above may vary with the enzyme used and the temperature of the reaction. However, the concentration range can be readily ascertained using the assays described in the Examples. For example, a standard preparation of lambda DNA can be heat-treated according to Example 1. The DNA can then be subjected to a series of enzyme mixtures containing ligase and cofactors. An additional enzyme is titrated to determine a preferred concentration for that enzyme in the mixture.  In this way, DNA repair can be optimized. After amplification of each sample, the amount of the amplified DNA can be determined by gel electrophoresis revealing the preferred concentration range for the test enzyme.<br><br><br>
The universal enzyme mixture can be used prior to or during polynucleotide amplification or other synthesis.<br>
As illustrated in the Examples, depending on the type of damage, it may be desirable to supplement the universal enzyme mixture with additional repair enzymes depending on the nature of the DNA damage. The utility of individual repair enzymes or mixtures of repair enzymes can be determined using the assays described in the Examples and in the Figures to determine their suitability for repairing a particular polynucleotide.<br>
Repair of general or specific damage to polynucleotides<br>
(a) General damage<br>
Determining the nature of damage in a polynucleotide is time-consuming. If some form of damage to a polynucleotide is suspected, for example, the polynucleotide is poorly amplified, it is preferable not to have to identify the lesion or lesions. In these circumstances, a universal mix of enzymes such as described above may be utilized to determine whether improved amplification is obtained. If the improvement is sufficient using the universal mixture then no further action is required. If the improvement is not sufficient, additional enzymes can be added to the mixture as described herein until the preferred result is obtained. The entire assay may be achieved in a single reaction vessel such as a 96 well dish. Each micro-well in the dish is available for a different enzyme mixture including the universal mixture plus enzymes selected to address each class of damage outlined below.<br><br><br>
The protocol for selecting enzymes for repair of general damage or unknown damage of DNA is provided in Figure 12 (flow chart) and in the assays described in the Examples.<br>
(b) Specific damage<br>
(i) AP sites<br>
The loss of a base is the most common form of spontaneous DNA damage.  Polymerases and polymerase-based techniques are adversely affected by the presence of these a basic sites. The effectiveness of primer extension reactions is enhanced by repairing any abasic sites found in a polynucleotide. This is achieved in one embodiment by Endo IV activity that cleaves the phosphate backbone at the abasic site. This leaves an extendable 3' OH on the DNA fragment 5' to the cleaved abasic site. It also leaves a deoxyribose-5'-phosphate (dR5P) on the DNA fragment 3' to the cleaved abasic site. A polymerase can extend from the free 3' OH replacing the cleaved abasic site with a correct nucleotide. The dR5P may be removed by an enzyme that specifically targets dR5Ps such as mammalian pol beta or the 8Kd N-terminal portion of mammalian pol beta (Deterding J Biol Chem 275:10463-71 (2000)), by a flap endonuclease activity present in certain polymerases such as E. coli DNA polymerase I or by a separate flap endonuclease such as FENI. The removal of dR5P can also occur by cleavage downstream of this group by the flap endonuclease activity. After removal of the dR5P and the generation of a 5' phosphate adjacent to the 3' OH, a ligase can seal this nick finishing the repair (see Examples 1-3).<br>
(ii) Modified nucleotides<br><br><br>
(a)	Thymidine dimers<br>
Light can damage DNA by inducing the formation of pyrimidine dimers.  Pyrimidine dimers block the DNA extension reaction catalyzed by DNA polymerases such as Taq DNA polymerase and hence inhibit DNA amplification (Wellinger, et al. Nucleic Acids Res. 24(8):1578-79 (1996)). Consequently it is desirable to repair pyrimidine dimers prior to or during amplification. This can be achieved by adding a pyrimidine dimer glycosylase/lyase (Vande Berg, et al. J. Biol. Chem. 273(32):20276-20284 (1998)) to the universal enzyme mixture. The DNA backbone is cleaved 5' to the pyrimidine dimer and leaves a 3' hydroxyl moiety that is extendable by a DNA polymerase. In certain embodiments, extension at the 3' hydroxyl and subsequent formation and then cleavage of the lesion-containing flap generated during DNA extension results in a nick that is sealed by an enzyme capable of sealing the nick. Cleavage of the flap can be achieved by the extending polymerase, for example, E. coli polymerase I or by the action of a flap endonuclease ((Xu, Y., et al. J. Biol. Chem. 275(27):20949-20955 (2000), Liu, Y., et al., Annu. Rev. Biochem. 73:589-615 (2004)).<br>
(b)	Oxidative damage, alkylation and deamination<br>
Inaccuracies can be introduced into the products of DNA amplification reactions because of undesired nucleotide incorporation opposite a damaged base (Gilbert, et al. Am. J. Hum. Gen. 72:48-61 (2003); Hofreiter et al. Nucl. Acids Res. 29:4793-9 (2001)).   These inaccuracies can be discovered after amplifying, cloning and sequencing the same sample many times.  Inaccuracies<br><br><br>
due to base damage can also be identified by comparing sequence data before and after sample treatment with an enzyme such as UDG, which removes one of the common types of mutagenic DNA lesions (Hofreiter, et al. Nuci Acids Res 29:4793-9 (2001)). However, treatment with UDG creates an abasic site within the DNA that inhibits DNA amplification by primer extension. This may cause DNA samples to be refractory to amplification after UDG treatment. This AP site can then be repaired by a reaction mixture containing a ligase and preferably also an AP endonuclease and a polymerase.  Removal of a uracil enables a polymerase in an amplification reaction that would normally be stopped at this site to continue amplifying the DNA. For example, Vent® polymerase activity is inhibited on DNA templates containing uracil. The ability to remove the uracil permits the polymerase to have enhanced effectiveness.<br>
In contrast, it is here shown that including UDG with an enzyme mixture that includes a ligase and a polymerase can be successfully used to enhance the yield and fidelity of the product of polynucleotide copying or amplification. Examples 12-14 provide descriptions of various beneficial enzyme mixes that include UDG.<br>
Modified nucleotides that are the product of oxidative damage can also be removed from the polynucleotide by Fpg or hOGG to leave a blocked polynucleotide where the blocked polynucleotide is repairable by an AP endonuclease such as Endo IV.<br>
The effectiveness of enzyme pretreatment to repair oxidative damage to a polynucleotide prior to copying or amplification is illustrated in Example 9 in which improved fidelity of the copied<br><br><br>
polynucleotide product is demonstrated using an enzyme mixture containing a ligase, a polymerase, Endo IV and Fpg.<br>
Other modified nucleotides such as alkylated bases or deaminated bases where cytosine is converted to uracil, guanine to xanthine or adenine to hypoxanthine give rise to miscoding. Removal of these modified nucleotides is desirable. These modified bases can be removed by UDG as discussed above or by AlkA or Aag as described in Example 10.<br>
(hi) Cross-links<br>
Additional nucleotide excision repair (NER) proteins (Minko et al. Biochemistry 44:3000-3009 (2005); Costa et al. Biochimie 85(11): 1083-1099 (2003); Sancar Ann. Rev. Biochem 65:43-81 (1996)) can be used to repair damage resulting from formaldehyde and bulky adducts as well as damage that results in chemically-modified bases that form DNA-protein cross-links. At least one of E. coli UvrA, UvrB, mutant UvrB, UvrC, UvrD or Cho (Moolenar et al. Proc. Natl Acad. Sci USA 99:1467-72 (2002)) can be used to make incisions at the 5' end and optionally the 3' end around a damaged site.  Details about the properties and purification protocols of these enzymes can be obtained from Zou, Y., et al. Biochemistry 43:4196-4205 (2004). The repair process can be completed by means of a DNA polymerase, a DNA ligase and optionally a flap endonuclease.<br>
The generation of a 31 hydroxyl at a 5' incision site can be useful if the NER enzyme(s) cleave the DNA but leave a blocked 3' end on the DNA that inhibits primer extension. An example would be if the NER enzyme(s) cleaved the DNA and left a 3' phosphate.<br><br>
This would not be extendable by known DNA polymerases unless the 3' phosphate was removed by, for example, E. coli Endo IV.<br>
If the NER enzyme or enzymes cleaves 5' and 3' to the DNA lesion then the damage is removed when the newly released oligonucleotide dissociates from the DNA. A polymerase can simply fill in the excised region of DNA leaving a nick, which ligase then seals to complete the repair. In certain cases, the polymerase may fill in the DNA and then proceed to displace the remaining DNA strand. In these circumstances, an enzyme with flapase activity permits a nick to be formed that a ligase can seal. In cases in which the NER enzyme or enzymes only cleaves 5' to the damage, the polymerase preferably displaces the original DNA strand until it is past the damage, at which point a flapase cleaves the DNA flap to create a ligatable nick. The flapase may be active before and after the DNA lesion is reached.  Preferably, the polymerase and flapase activities work to eventually displace and remove the DNA lesion leaving a ligatable nick, thus repairing the DNA template. An example of the effectiveness of the above approach is provided in Example 7.<br>
(iv) Nicks, Gaps and Mismatched Polynucleotides<br>
Nicks and gaps in the DNA backbone can lead to truncated primer extension products and formation of chimera by undesirable hybridization of single strand regions. Heteroduplex DNA can be a problem in multi-template PCR and in homogeneous template PCR (Lowell, J. L &amp; Klein, D. A. Biotechniques 28:676-681 (2000); Thompson, J. R., et al. Nucl. Acids Res. 30(9):2083-2088 (2002); Smith, J. &amp; Modrich, P. Proc. Natl. Acad. Sci. USA 94:6847-6850<br><br><br>
(1997)). For example, chimera can be formed at the mismatch sites.<br>
The combined effect of a ligase and a polymerase together optionally with an enzyme that recognizes and cleaves at heteroduplex sites (T7 Endol and mutants thereof) contained within a universal enzyme mixture results in repairs of nicks, gaps, heteroduplexes and chimera in the DNA thus enhancing yield and fidelity of polynucleotide copying and amplification reactions. Example 8 demonstrates the beneficial effects of using T7 Endo I and a ligase as illustrative of the above. The useful range of the T7 endonuclease or mutant:DNA ratio can be expanded by including a DNA ligase activity to minimize the detrimental effects of nonspecific cleavage in the heteroduplex cleavage reaction. This approach does not require quantitation of DNA and avoids the extra steps after the PCR reaction required by Lowell, et al. Biotechniques 28:676-681 (2000); and Smith, et al. Proc. Natl. Acad. Sci. USA 94:6847-6850 (1997).<br>
For some polynucleotides, the nature of the damage might be known. By way of illustration, a mixture of enzymes can be selected according to section (b) above for repairing the specific damage. Where the damage is unknown or the sources are mixed, the universal mix described herein including in the examples may be employed.<br>
DNA microarrays<br>
DNA microarrays are a powerful methodology used to analyze DNA samples (Lipshutz et al. Curr Opinion in Structural Biology 4:376-380 (1994); Kozal, et al. Nat Med 2(7):753-9 (1996)). The<br><br>
amount and quality of information from microarray analysis of damaged DNA would benefit from first repairing the damaged DNA.<br>
Discussion of the Examples and Figures<br>
The enhancement in yields of product after amplification is shown both generally and for specific sources of damage: Figures 1A-1D, and 4 (heat damage), 2A-2B (citrate), 8A-8C (UV irradiation), 12, 3, 5 and 17 (general damage to DNA, environmental DNA, stored krill DNA and cave bear DNA) and 9A-9B (subject to undesirable heteroduplex formation) as well as the impact of different incubation times at different temperatures (Figures 13 and 14). The enhancement of fidelity is demonstrated in Figures 10A-10B and 11. Figures 6A-1 to 6A-9, 6B-1 and 6B-2 provide details of sequences that are homologous with Taq ligase and T4 ligase sequences.  Figure 7 describes a DNA sequence for Tth Endo IV and Figures 18-1 to 18-2 describe a sequence for plasmid pNEBO.920. Figure 15 shows that multiple fragmented DNA can be repaired to form a template suitable for amplification to produce an amplicon.  Benefits of repair mixes for cloning and transformation are illustrated in Figures 15 and 16.<br>
Example 1 and Figures 1A-1D show that amplicon yields obtained from PCR amplification are substantially negatively affected when the template DNA is damaged beyond a certain threshold of damage (e.g., about 90 seconds heat treatment) (see Figure 1A). The effect of this damage on amplification can be reversed and amplicon yields enhanced by incubating the DNA with a mixture of enzymes before amplification (see Figures IB, 1C and ID). In addition, if amplicon yields of assumed "undamaged" DNA<br><br><br>
can be enhanced by adding the enzyme mixture described then in fact, the DNA is deemed to have been damaged.<br>
Example 1 shows that the effect of the enzyme mixture on amplification of DNA is not dependent on a single type of AP endonuclease or ligase, but instead endonucleases or ligases from multiple alternative sources can be used.  For example, thermostable Tth Endo IV was found to be as effective as E. coli Endo IV and E. coli ligase was as effective as the thermostable Taq ligase.<br>
Example 2 and Figures 2A-2B show the negative effect on amplification yields of another type of DNA damage - depurination, which is induced in the presence of low pH. Moreover, the example shows that the effect of a mixture of enzymes on amplification of DNA is not dependent on a single type of polymerase but rather polymerases from multiple alternative sources can be used.  For example, E. coli Poll can be substituted by Taq DNA polymerase or a mixture of Taq and Vent® DNA polymerases to produce enhanced yields.<br>
Example 3 and Figure 3 show that the enhancement of amplification yields can be observed with short (200 bp) fragments. In fact, enhancement of amplification yields should be observable for a wide range of sizes of DNA templates from as short as 100 bases to as long as 100 kb and it is believed that amplification yields for DNA even larger than 100 kb can be achieved.<br>
Figure 3 also shows that even when the DNA has been damaged through storage in a crude form (for example, within the cells of an organism that has itself been stored), amplification yields<br><br><br>
are significantly enhanced by the addition of a mixture of enzymes prior to amplification. Although the mixture of enzymes was added to template DNA prior to amplification, a similar yield effect can be seen when the template DNA is incubated with the mixture of enzymes that are thermostable equivalents during amplification or during a pre-amplification step.<br>
Example 4 and Figure 4 show that ligase alone can enhance amplicon yield, but adding an AP endonudease further enhances the yield. The best result was observed in this example when a ligase, an AP endonudease, and a DNA polymerase were used prior to amplification.  Furthermore, this example demonstrates that repair is not DNA size dependent.  For example, similar results were obtained with 5 kb and 10 kb amplicons.<br>
Example 5 and Figure 5 show that an enhanced yield from amplification can be achieved using a ligase, which may be an NAD+ dependent or ATP dependent enzyme and that this effect can be achieved without limitation to a single source of ligase. Figure 5 shows that Taq ligase and T4 ligase are both effective in enhancing amplification yield even when used without additional enzymes in a pre-incubation mix. This effect is also believed to occur if the ligase is added to the amplification mix (if thermostable).  Figure 5 also shows the benefit of this approach when amplifying environmental DNA obtained directly from soil samples that have been exposed in nature to a variety of damaging agents.<br>
Figures 6A-1 to 6A-9, 6B-1 and 6B-2 and 7 are descriptions of enzymes or the sequence encoding enzymes used in the universal mixture of enzymes, which is so named because it can be broadly applied to damaged polynucleotides without specific knowledge<br><br><br>
concerning the damage itself. While the examples refer to DNA, amplification or copying of polynucleotides is more generally envisaged. In general, the conditions under which amplification is found to be improved are also applicable to enhancing copying of polynucleotides generally even if this is not explicitly stated in all cases.<br>
All references cited herein, as well as U.S. application serial number 11/255,290 filed October 20, 2005 and U.S. provisional applications serial numbers 60/620,896 filed October 21, 2004, 60/646,728 filed January 24, 2005 and 60/673,925 filed April 21, 2005, are incorporated by reference.<br>
EXAMPLES<br>
Example 1: Enhancing amplicon yields from DNA damaged bv I .eat treatment<br>
An assay was developed for optimizing the use of selected reagents to repair DNA prior to amplification.<br>
Generation of various extents of heat damage<br>
Various amounts of DNA damage were induced by heat treatment. This was achieved as follows: 100 mL lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) at 0.5 mg/ml was aliquoted into separate tubes for heat treatment at 99°C for 30 sec, 60 sec, 90 sec, 120 sec, and 180 sec, respectively in a PE2700 thermal cycler. A sample was used as a template for amplification without pretreatment.<br><br><br>
The remaining damaged DNA was pretreated by the mixture of enzymes as follows: The damaged DNA templates were incubated at room temperature in the following mixture for 10 minutes:<br>
DNA (5ng, 2ng and lng);<br>
100 mM dNTPs (NEB#M0447, New England Biolabs, Ipswich, MA);<br>
1 mM NAD+ (Sigma#N-7004, Sigma, St. Louis, MO);<br>
80 units Taq ligase (NEB#M0208, New England Biolabs, Ipswich, MA) or 40-100 units of E. coli ligase (NEB#M0205S);<br>
0.1 units E. coli DNA polymerase I (E coli poll) NEB#M0209, New England Biolabs, Inc., Ipswich, MA);<br>
10 units E. coli Endo IV (NEB#M0304, New England Biolabs, Inc., Ipswich, MA) or 10 units of Tth Endo IV;<br>
lXThermopol buffer (NEB#B9004, New England Biolabs, Inc., Ipswich, MA) to a final volume of 96 mL<br>
At the end of the reaction, the samples were transferred to ice and then amplified.<br>
DNA Amplification Reaction<br>
DNA amplification of lambda was performed using the following primers: CGAACGTCGCGCAGAGAAACAGG (L72-5R) (SEQ ID NO:l) and CCTGCTCTGCCGCTTCACGC (L30350F) (SEQ ID NO:2) according to the method of Wang et al. Nucl. Acids Res. 32: 1197-1207(2004).<br>
4 pi of amplification mixture was added to 96 ml of the above repair mixture. The amplification mixture contained 100 mM dNTPs, 5 units Taq DNA polymerase, 0.1 units Vent® (exo+) DNA<br><br><br>
polymerase, 5x10-11 M primer L72-5R and 5x10-11 M primer L30350F in IX Thermopol buffer.<br>
To correct for any enzyme storage buffer effects, when a repair enzyme was omitted from a reaction, the appropriate volume of its storage buffer was added to the reaction.   In all cases, the amplification reactions were processed in a thermal cycler using the following parameters: 20 sec at 95°C for 1 cycle followed by 5 sec at 94°C, then 5 min at 72°C for 25 cycles. The size of the amplicon being amplified was 5 kb.<br>
The results of amplification of DNA (5 kb) were determined by 1% agarose gel elecrophoresis. 6X loading dye (Molecular Cloning:A Laboratory Manual, 3rd ed., eds. Sambrook and Russell, Cold Spring Harbor Press, Cold Spring Harbor, NY, 2001, pp. 5.4-5.17) was added to the 100 ul amplification reactions. 20 ml of this solution was then loaded onto the agarose gel along with 1 mg of 2-log ladder (NEB#N3200, New England Biolabs, Inc., Ipswich, MA) as a size standard.<br>
The amount of amplified DNA for each sample was compared by gel electrophoresis and the results are shown in Figures 1A-D. When the samples were treated with a mixture of enzymes after heat treatment but prior to amplification, significant enhancement of amplification yields was achieved (Figures IB, 1C and ID).<br>
Example 2: Enhanced amplicon yields from DNA with low pH-ind^ced abasic sites following pretreatmsnt with ail enzyme mixture<br>
Generation of various extents of damage resulting from a basic sites<br><br><br>
To assay the extent of repair of damaged DNA, various amounts of DNA damage was first induced by acidic pH. This was achieved as follows:<br>
DNA was depurinated as described by Ide, H., et al. Biochemistry 32(32):8276-83 (1993). Lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) was ethanol precipitated. The DNA was resuspended in depurination buffer (100 mM NaCI, 10 mM citrate, pH 5.0) at a concentration of 0.5 mg/ml and incubated at 70°C for 0, 20, 40, 80, 120, and 160 minutes. The sample was then ethanol precipitated and resuspended in 0.01 M Tris, 0.001 M EDTA, pH 8.0. The DNA concentration was determined by measuring the A260 of the DNA-containing solutions after calibrating with a buffer control.<br>
Pretreatment of DNA with a mixture of enzymes<br>
The damaged DNA was incubated at room temperature for 10 minutes in the following mixture:<br>
DNA (2.5 ng of damaged DNA after 120 minute of low pH treatment);<br>
100 mM dNTPs;<br>
1 mM NAD+;<br>
80 units Taq ligase;<br>
0.1 units Taq DNA polymerase or 0.1 units E. coli Poll (NEB#M0209, New England Biolabs, Inc., Ipswich, MA)) or 0.1 units ra
10 units E. coli Endo IV;<br><br><br>
IX Thermopol buffer to a final volume of 96 mL<br>
The above mixture was incubated at room temperature for 10 minutes and then transferred to ice prior to amplification.<br>
DNA amplification reaction<br>
Amplification was performed as described in Example 1 to generate a 5 kb amplicon. Amplicon yields were increased as compared with negative controls (Figure 2A) by treating lambda DNA containing abasic sites with the mixture of enzymes. The results are shown in Figure 2B for a series of pretreatments using different enzyme mixtures. The enzyme mixtures were varied with respect to the polymerase (E. coli Poll or Tag:Vent®).<br>
Example 3: Enhanced amplicon yields of DNA extracted from an intact organism after storage in a preservative<br>
Genomic DNA was isolated from Meganyctiphanes norvegica (Krill) as described in Bucklin, A. &amp; Allen, L. D. Mol. Phylogenet. Evol. 30(3):879-882 (2004). The Krill had been stored in ethanol for about 5 years.<br>
Pretreatment of the Krill DNA by a mixture of enzymes was carried out as follows:<br>
50 mg of M. norvegica genomic DNA;<br>
100 mM dNTPs;<br>
1 mM NAD+;<br>
40 units of Tag ligase;<br><br><br>
0.5 units Taq DNA polymerase, 0.2 units Vent® (exo+) DNA polymerase, or a TagiVent® (exo+) mix containing 0.05 units of Taq DNA polymerase and 0.001 units of Vent® (exo+);<br>
10 units E. coli Endo IV;<br>
IX Thermopol buffer to a final volume of 96 ml. This reaction was incubated 15 minutes at room temperature before proceeding to the amplification step.<br>
DNA amplification reaction<br>
The amplification primers corresponded to 52F and 233R as described in Bucklin, A. &amp; Allen, L. D. Mol. Phylogenet. Evol. 30(3):879-82 (2004) generating a 200 bp amplicon.<br>
52F: TTTTTAGCAATACACTACACAGCAA (SEQ ID NO:3) 233R: ATTACGCCAATCGATCACG (SEQ ID NO:4)<br>
Primers were added to a final concentration of 0.5 mM, and each dNTP to a final concentration of 200 mM. 1 ml of the 50:1 Tag:Vent® mix (5 units Taq DNA polymerase and 0.1 units Vent® (exo+) DNA polymerase added to the reaction) was then added to each reaction to a final volume of 100 mL<br>
For the control reaction (Lane 1), no Endo IV, Taq ligase or pretreatment polymerase was added. Volumes were adjusted accordingly. In reactions in which repair enzymes were omitted, the appropriate volume of enzyme storage buffer was added to control for buffer effects.<br>
Cycling conditions were as follows: 30 sec at 94°C, 30 sec at 52°C and 1 min 40 sec at 72°C for 40 cycles. 25^L (one quarter of<br><br><br>
the reaction) was prepared, loaded on a 1% agarose gel, electrophoresed, and visualized as described above.<br>
Increased amplicon yield from krill genomic DNA was observed after preincubation of the samples using the enzyme mixtures described above (Figure 3).<br>
Example 4: Enhanced yields of a large f 10 kbi amplicon from heat-damaged DNA<br>
Heat-damaged DNA was prepared as described in Example 1. Lambda DNA was heated to 99°C for 180 sec.<br>
Pretreatment of damaged DNA by a mixture of enzymes was carried out as follows:<br>
Lambda DNA (1 ng of 180 sec heat-treated DNA);<br>
100 mM dNTPs;<br>
1 mM NAD+;<br>
80 units of Taq ligase;<br>
0.1 unit of E. coli Poll;<br>
100 units of E. coli Endo IV;<br>
IX Thermopol buffer to a volume of 96 mL.<br>
The mixture was incubated for 10 minutes prior to amplification.<br>
DNA amplification was performed as described in Example 1, except where specified below.  Primers were added to the above 96 ml of preincubation mixture. Primer L71-10R (sequence GCACAGAAGCTATTATGCGTCCCCAGG) (SEQ ID NO:5) replaced L72-5R in Example 1. The iCycler thermal cycler program (Bio-Rad, Hercules, CA) was: 20 sec at 95°C for 1 cycle, 5 sec at 95°C, 10 min<br><br><br>
at 72°C for 25 cycles and then 10 min at 72°C for 1 cycle. Amplicon size was 10 kb.<br>
The DNA was visualized as described in Example 1 with the following exceptions.  20 ml of 6X loading buffer was added to the 100 pi amplification reaction.  10 ml of this solution was diluted to 100 ml with H2O and IX loading buffer.  20 ml of this was loaded into each lane. The gel was a 0.8% agarose gel. The results are shown in Figure 4.<br>
Example 5: Improved amplification yield of DNA from environmental DNA (extracted from soil samples<br>
Environmental DNA was isolated from the soil using an UltraClean Soil DNA Kit from MoBio Laboratories, Inc., Carlsbad, CA (catalog # 12800-50).<br>
Pretreatment of DNA with a ligase<br>
A final volume of 100 ml containing 0.6 mg of environmental DNA isolated from soil and one of the two ligases described below in (a) and (b) formed the reaction mixture. This reaction mixture was then incubated at room temperature for 15 min.<br>
(a) IX Taq ligase buffer (New England Biolabs, Inc., Ipswich, MA) and 80 units of Taq ligase.<br>
(b) IX T4 ligase buffer (New England Biolabs, Inc., Ipswich, MA) and 800 units of T4 ligase (NEB#M0202, New England Biolabs, Inc., Ipswich, MA).<br><br><br>
1 ml of reaction mixture was used in the amplification reaction described below.<br>
DNA Amplification Reaction<br>
DNA amplification was performed using primers: GGGGGXAGAGTTTGATCMTGGCTCA (SEQ ID NO:6) and GGGGGXTACGGYTACCTTGTTACGACTT (SEQ ID NO:7) (M = C or A, Y = C or T, X = 8-oxo-Guanine). These primers target 16S rDNA having an amplicon size of 1.6Kb.<br>
The 50 ul reaction contained 10 pmol of each of the primers, 1 ml of the repaired environmental DNA, 200 mM dNTPs, IX Thermopol buffer, and 1.25 units Taq DNA polymerase. The amplification was performed using the following cycling parameters: 5 min at 94°C for 1 cycle, 30 sec at 94°C, 1 min at 55°C, 1 min 40 sec at 72°C for 32 cycles, then 5 min at 72°C for 1 cycle.<br>
Gel electrophoresis was performed as described in Example 1. The results are shown in Figure 5.<br>
Example 6:  Enhanced amplicon yield of ultraviolet light-damaged DNA<br>
To determine conditions for assaying the effectiveness of DNA repair, 50 mg lambda DNA (NEB#N3011, New England Biolabs, Inc., Ipswich, MA) was diluted in TE buffer (10 mM Tris-HCI, 1 mM EDTA, pH 7.5) to a concentration of 50 ng/ml and irradiated with 36 J/m2 UV light for 0, 10, 20, 30, 40 and 50 sec.<br><br><br>
Pretreatment of damaged DNA by a mixture of enzymes was carried out as follows:<br>
The damaged DNA was incubated at room temperature for 15 minutes in the following mixture:<br>
DNA (50 ng of lambda DNA-damaged for 0, 10, 20, 30, 40, or 50 seconds);<br>
200 mM dNTPs;<br>
1 mM NAD+;<br>
400 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
80 units or 10 units T4 PDG (also referred to as T4 Endo V). (Trevigen, Gaithersburg, MD);<br>
lXThermopol buffer<br>
Adjust volume with water to 50 \i\.<br>
After the 15 minutes incubation, the 50 ml reaction mixture was added to 50 ml of an amplification solution. The amplification solution consisted of 40 pmol of each primer (L72-5R and L30350F as described in Example 1 or L72-2R (the DNA sequence was CCATGATTCAGTGTGCCCGTCTGG) (SEQ ID NO:8), IX Thermopol buffer, 1 mM NAD+, 200 mM dNTPs, 2.5 units Taq DNA polymerase (NEB#M0267, New England Biolabs, Inc., Ipswich, MA), and H20 to a final volume of 50 mL  Combining the 50 ml_ repair reaction with the 50 pi amplification solution gave a final volume of 100 ml.<br>
The 100 pi solutions were placed into a thermal cycler. For the L72-5R and L30350F primer combination: 5 min at 94°C for 1 cycle; 30 sec at 94°C, 60 sec at 58°C, and 4 min at 72°C for 30 cycles; 5 min at 72°C for 1 cycle.<br><br><br>
For the L72-2R and L30350F primer combination:<br>
5 min at 94°C for 1 cycle; 30 sec at 94°C, 60 sec at 58°C, and 2 min<br>
at 72°C for 30 cycles; 5 min at 72°C for 1 cycle.<br>
The presence of amplification product was visualized on a 1.8% agarose gel using ethidium bromide. The size of any band was compared against a lane containing the 2-log ladder (NEB#N3200S, New England Biolabs, Inc., Ipswich, MA) size standards. The results are shown in Figure 8.<br>
Example 7: Enhanced amplicon yield of DNA using the nucleotide excision repair proteins. UvrA. UvrB and UvrC<br>
Increased amplicon yield from krill genomic DNA is determined after pre-incubation of the samples using an enzyme mixture containing proteins involved in nucleotide excision repair.<br>
Pretreatment of stored DNA by a mixture of enzymes is carried out as follows:<br>
Stored DNA is incubated for 1-180 minutes at 4-37°C in the following mixture:<br>
DNA:   50 ng of M. norvegica genomic DNA;<br>
100 MM dNTPs;<br>
1 mM ATP;<br>
400 units of Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 nM E. coli UvrA, 250 nM E coli UvrB (or mutant UvrB*), plus or minus 50 nM E. coli UvrC;<br>
IX Thermopol buffer to a final volume of 96 |xl.<br><br><br>
* for mutant UvrB, see Zou, Y., et al. Biochemistry 43:4196-4205 (2004).<br>
DNA amplification reactions are conducted as described in Example 3.<br>
Example 8: Increasing sequence accuracy of a DNA after removal of incorrect nucleotides on at least one strand by means of enzyme cleavage of heteroduolexes<br>
Experimental Conditions<br>
A. Adding Taq ligase to T7 Endo I was demonstrated to permit the use of an increased concentration of T7 Endo I in a DNA preparation without randomly degrading the DNA.<br>
The assay relies on treating a supercoiled DNA containing a cruciform structure with increasing amounts of T7 Endo I.<br>
0, 1.6, 3.1, 6.2, 12.5, 25, 50, 100, 200, or 400 units of T7 Endo I (NEB#M0302, New England Biolabs, Inc., Ipswich, MA) was added to 50 ul reactions composed of 1 mg of pUC(AT) (Guan, C, et.al. Biochemistry 43:4313-4322 (2004)) and IX NEBuffer2 (NEB#B7002S, New England Biolabs, Inc., Ipswich, MA).  Plasmid pAT25tetA can be used in place of pUC(AT) (Parkinson, M. J. &amp; Lilley, D. M. J. Mol. Biol. 270:169-178 (1997) and Bowater, R. P., et. al. Biochemistry 33:9266-9275 (1994)). Another set of reactions were set up simultaneously and used the same components as described above with the addition of 1 mM NAD+ (Sigma catalog#N-7004, Sigma, St. Louis, MO) and 100 units of Taq ligase (using a stock of NEB#M0208 at a concentration of 100<br><br><br>
m/ml, New England Biolabs, Inc., Ipswich, MA). All reactions were incubated at 37°C for 60 minutes.<br>
The results were analyzed by running the reactions on a 0.9% TBE agarose gel, stained with ethidium bromide, and visualized using UV light (see Figures 9A and 9B). With no T7 Endo I present the pUC(AT) plasmid produced 2 bands on the gel corresponding to the supercoiled form (lower band) and the relaxed circular form (upper band).<br>
T7 Endo I resolved the supercoiled pUC(AT) into the relaxed circular form and a linear form that ran intermediate to the supercoiled and relaxed circular forms. At certain T7 Endo I:DNA ratios, a smear was produced indicating that the T7 Endo I had degraded the DNA by non-specific enzymatic activity. The presence of Taq ligase significantly increased the usable T7 Endo I to DNA ratio. This ratio is further improved by substituting T7 Endo I with the mutant T7 Endo I described in International Publication No. WO 2005/052124.<br>
B. The use of Method A to remove heterduplexes from PCR reactions.<br>
Isolation of DNA from soil and amplification of the purified DNA is performed as described in Example 5 with the optional addition of 5 units T7 Endo I or mutant thereof. When T7 Endo I or mutant thereof is added, an additional amplification cycle is added (37°C for 15 minutes for 1 cycle). The last step is to allow the T7 endonuclease to cleave any heteroduplexes formed.<br><br><br>
Gel electrophoresis is performed as described in Example 1. Heteroduplex DNA is visualized on the gel as described in Lowell, 3. L. &amp; Klein, D. A. Biotechniques 28:676-681 (2000). Absence of heteroduplex DNA in the presence of T7 Endo I or mutants thereof shows the effectiveness of T7 Endo I or mutants thereof with a ligase.<br>
Unit definitions are described with the product description for each of the enzymes recited herein in the NEB catalog, New England Biolabs, Inc., Ipswich, MA. For example, the unit definition for T7 Endo I or mutant thereof is the amount of enzyme required to convert greater than 90% of 1 mL of supercoiled plasmid into greater than 90% linear DNA in a reaction volume of 50 ml in 1 hour at 37°C.<br>
The T7 Endo I:DNA ratio can be increased without increasing non-specific cleavage of DNA in the presence of ligase.<br>
Example 9: Enhancing the sequence accuracy of a DNA amplication reaction after oxidative damage<br>
Generating DNA with oxidative damage<br>
The DNA subject to oxidative damage was pWB407 (Kermekchiev, M. B. et al. Nucl. Acids Res. 31:6139-47 (2003)). The damage was incurred using a combination of methylene blue (MB) and visible light as described previously (Sattler, et al. Arch. Biochem Biophys. 376(l):26-3 (2000)).  Plasmid DNA (200 ng/ml in distilled water) was spotted on parafilm stretches (50 ml drops). MB was added to the drops to a final concentration ranging from 0 to 50 (0, 3, 6, 12.5, 25 and 50) mg/ml (100 ml final volume). Plates<br><br><br>
with these parafilm stretches were placed on ice and illuminated for 8 min. with a 1 x 100-W lamp. The MB-light-treated DNA was precipitated, dried, and then re-suspended in 50 ul of TE buffer (pH 8.0). Final DNA concentration was determined by the absorbance of light at 260 nm.<br>
DNA amplification conditions<br>
A portion of pWB407 that contained the lacZ gene was amplified using primers 316-138,<br>
TGTCGATCAGGATGATCTGGACGAAGAGC (SEQ ID NO:9), and 316-137, CGAAAGCTTTCAAGGATCTTACCGCTGTTGAGA (SEQ ID NO: 10). Primers 316-138 and 316-137 were based on the previously-described primers Kfd-29 and H3Bla34, respectively (Kermekchiev, M. B. et al. Nucl. Acids Res. 31:6139-47 (2003)). The 100 ul_ PCR reactions contained either 10 or 50 ng of template DNA, indicated where appropriate, and 40 picomoles of each primer. The cycling conditions utilized varied with the thermal stable polymerase used for amplification.<br>
Cycling conditions when using Taq DNA polymerase (NEB cat#M0267S, New England Biolabs, Inc., Ipswich, MA) had an initial denaturation step of 5 min at 94°C for 1 cycle, then 30 sec at 94°C, 60 sec at 58°C, and 3 min 30 sec at 72°C for 30 cycles, and finally 5 minutes at 72°C.<br>
Cycling conditions when using Phusion™ DNA polymerase (NEB cat#F-530S, New England Biolabs, Inc., Ipswich, MA) had an initial denaturation step of 30 sec at 98°C for 1 cycle, then 10 sec at 98°C, 30 sec at 62°C, and 1 min 30 sec at 72°C for 30 cycles, and finally 5 min at 72°C.<br><br><br>
The reaction outcomes were analyzed by loading 25 ml of the reaction on a 1.6% agarose gel, prepared, electrophoresed and visualized as described above. The marker used was the 2-log DNA ladder (NEB cat#N3200S, New England Biolabs, Inc., Ipswich, MA).<br>
Amplification accuracy determination<br>
The accuracy of DNA amplification from the pWB407 template was determined as described by Barnes, et al. Gene 112:29-35 (1992) and Kermekchiev, et al. Nucl. Acids Res. 31:6139-47 (2003). Amplicons containing the lacZ gene were generated from plasmids pWB407 that had been subjected to differing amounts of oxidative damage. The oxidative damage was performed using methylene blue as described above. The PCR reactions were performed using 50 ng of template as described above.   After cycling, 10 units of the restriction endonuclease, Dpnl, was added to each 100 mL PCR reaction and incubated for 2 hours at 37°C. This step eliminated the original template plasmid.  Next, the resulting amplification products were extracted with phenol/chloroform and precipitated using isopropanol (Molecular Cloning: A Laboratory Manual, 3rd ed., eds. Sambrook and Russell, Cold Spring Harbor Press, Cold Spring Harbor, NY, 2001, pp. 6.25, A8.12-A8.24).  Precipitated products were re-suspended in H2O and cut with the restriction endonucleases Styl and Hindlll using conditions recommended by the manufacturer (New England Biolabs, Inc., Ipswich, MA). The DNA digestion reactions were stopped by inactivating the Hindlll and Styl enzymes by heating to 65°C for 20 min. The restriction digestion products were purified using a microcon YM-100 column (Millipore, Billerica, MA) to eliminate short DNA fragments.<br><br><br>
The repair reaction mixtures in a total of 50 ul contained 10 or 50 ng of pWB407 amplicons +/- methylene blue incubation. The repair reactions contained 20 mM Tris-HCI (pH 8.8 at 25°C), 10 mM KCI, 10 mM (NH4)2S04, 2 mM MgS04/ 0.1% Triton X-100, 1 mM NAD+, 200 mM dNTPs (dATP, dTTP, dCTP, and dGTP), and various repair enzyme mixtures.<br>
The repair enzyme mixtures used separately or in various combinations in a total volume of 50 mL were:<br>
0.4 units Fpg, NEB cat#M0240S, New England Biolabs, Inc., Ipswich, MA);<br>
200 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
1 mM NAD+;<br>
100 mM dNTPs;<br>
IX Thermopol buffer.<br>
The reactions were incubated at 25°C for 15 minutes. After the incubation, 50 mL of a PCR mix (20 mM Tris-HCI (pH 8.8 at 25°C), 10 mM KCI, 10 mM (NH4)2S04, 2 mM MgS04, 0.1% Triton X-100, 1 mM NAD+, 200 mM dNTPs (dATP, dTTP, dCTP, and dGTP), and either 2.5 units Taq DNA polymerase (NEB cat#M0267S, New England Biolabs, Inc., Ipswich, MA) or 1 unit of Phusion™ DNA polymerase was added to the 50 mL repair reaction and this new solution was subjected to thermal cycling conditions for PCR. The amplicons from these reactions were purified and restriction enzyme digested as described for other amplicons above.<br><br><br>
The amplicons were cloned into the pWB407 plasmid. Plasmid pWB407 was prepared by digestion with the restriction endonucleases Styl and Hindlll followed by a 30-minute incubation at 37°C with 1 unit/ug DNA of antarctic phosphatase (NEB cat#M0289S, New England Biolabs, Inc., Ipswich, MA). The dephosphorylated pWB407 vector backbone was purified by agarose gel electrophoresis.  Gel extraction was performed with a QIAquick Gel Extraction Kit (Qiagen, Valencia, CA).<br>
The digested amplicons were ligated into the prepared pWB407 plasmids in 30 mL reactions using approximately 0.1 mg vector DNA and about 0.5 mg amplicon. T4 ligase was used to perform the ligation following the manufacturers recommended conditions (New England Biolabs, Inc., Beverly, MA).  Ligation products were electroporated into E. coli strain WB441 (Barnes, W. Gene 112:29-35 (1992)). The selective indicator plates used were LB plates containing 50 ng/ml ampicilin and 80 ng/ml Xgal. Before plating, the bacteria were incubated in rich broth for 1 hour at 37°C to allow expression of the ampicilin resistance. Control transformations lacking ligase treatment resulted in zero colonies. Colonies were scored for blue color after one day at 37°C, and one or two days at 25°C. The results are shown in Figures 10A-10B and 11.<br>
Example 10: Enhancing the sequence accuracy of a DNA amplification reaction after deamination damage<br>
Generating deaminated DNA<br>
The DNA subjected to deamination is pWB407 (Kermekchiev, et al. Nucl Acids Res 31: 6139-6147 (2003)). The damage is<br><br><br>
incurred using random mutagenesis with nitrous acid as described in Yan, W. et al. 3 Virol. 77(4):2640-50 (2003). Nitrous acid can deaminate guanine in DNA to xanthine, cytosine to uracil, and adenine to hypoxanthine.<br>
Plasmid DNA (2mg) is treated with 0.7 M NaN02 in 1M acetate buffer, pH 4.6. The reaction is terminated at various time points by addition of 4 volumes of ice-cold 1 M Tris-HCI (pH 7.9). The plasmid DNA is alcohol precipitated, dried and then resuspended in 100 mL of TE buffer.<br>
Pretreatment reaction to repair deaminated bases<br>
The repair enzyme mixtures used separately or in various combinations in a total volume of 50 mL are:<br>
(a)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo III (NEB cat # M0268S), New England Biolabs, Inc., Ipswich, MA;<br>
2 units Endo V (NEB cat # M0305S), New England Biolabs, Inc., Ipswich, MA;<br>
2 units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br><br><br>
(b)<br>
2 units Endo V (NEB cat # M0305S), New England Biolabs, Inc., Ipswich, MA;<br>
2 units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq DNA ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(c)<br>
2 units Endo V (NEB cat # M0305S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq Ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(d)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo III (NEB cat # M0268S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq Ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E coli Endo IV;<br>
ImM NAD+;<br>
100 mM dNTPs;<br><br>
lx Thermopol buffer.<br>
(e)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units UDG (NEB cat # M0280S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq ligase;<br>
0.1 units E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
1 mM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
(f)<br>
1	unit Human Aag, New England Biolabs, Inc., Ipswich, MA;<br>
2	units Endo V (NEB cat # M0305S), New England Biolabs, Inc., Ipswich, MA;<br>
200 units Taq ligase;<br>
0.1 unit E. coli DNA polymerase I;<br>
10 units E. coli Endo IV;<br>
1 mM NAD+;<br>
100 mM dNTPs;<br>
lx Thermopol buffer.<br>
The amplification reaction conditions and amplification accuracy determination are performed as described in Example 9.<br>
Example 11: Unit Definitions<br>
Thermophilic ligase unit<br><br><br>
One unit is defined as the amount of enzyme required to give 50% ligation of 1 pg of BstE II-digested lambda DNA in a total reaction volume of 50 pi in 15 minutes at 45°C. Taq ligase is available from New England Biolabs, Inc., Ipswich, MA.<br>
Mesophilic ligase unit<br>
One unit is defined as the amount of enzyme required to give 50% ligation of Hind III digested lambda DNA (5' DNA termini concentration of 0.12 pM, 300 pg/ml) in a total reaction volume of 20 pi in 30 minutes at 16°C. E. coli ligase is available from New England Biolabs, Inc., Ipswich, MA.<br>
AP endonuclease unit<br>
One unit is defined as the amount of enzyme required to cleave 1 pmol of a 34-mer oligonucleotide duplex containing a single AP site in a total reaction volume of 10 ml in 1 hour at 37°C.<br>
Mesophilic polymerase unit<br>
One unit is defined as the amount of enzyme that will incorporate 10 nmol of dNTP into acid-insoluble material in a total reaction volume of 50 pi in 30 minutes at 37°C with 33 pM dNTPs including [3H]-dTTP and 70 pg/ml denatured herring sperm DNA.<br>
Thermophilic polymerase Unit<br>
One unit is defined as the amount of enzyme that will incorporate 10 nmol of dNTP into acid-insoluble material in a total reaction volume of 50 ml in 30 minutes at 75°C with 200 pM dNTPs<br><br><br>
including [3H]-dTTP and 200 mg/ml activated Calf Thymus DNA. Thermophilic polymerases-Taq polymerase and archaeal polymerases are available from New England Biolabs, Inc., Ipswich, MA.<br>
The unit definitions for thermophilic UDG, Fpg , Endo III and Endo VIII are the same as those for the mesophilic equivalents listed (NEB catalog, New England Biolabs, Inc., Ipswich, MA).<br><br><br>
Gene Name	Activity	Accession Number<br>
UNG	Uracil-DNA glycosylase	NM_080911<br>
SMUG1	Uracil-DNA glycosylase	NM_014311<br>
MBD4	Removes U or T opposite G at CpG sequences	NM_003925<br>
TDG	Removes U, T or ethenoC opposite G	NM_003211<br>
0GG1	Removes 8-oxoG opposite C	NM_016821<br>
MUTYH (MYH)	Removes A opposite 8-oxoG	NM_012222<br>
NTHL1 (NTH1)	Removes Ring-saturated or fragmented pyrimidines	NM_002528<br>
MPG	Removes 3-meA, ethenoA, hypoxanthine	NM_002434<br>
NEIL1	Removes thymine glycol	NM_024608<br>
NEIL2	Removes oxidative products of pyrimidines	NM_145043<br>
XPC	Binds damaged DNA as complexXPC, RAD23B, CETN2	NM_004628<br>
RAD23B (HR23B)		NM_002874<br>
CETN2   '		NM_004344<br>
RAD23A (HR23A)	Substitutes for HR23B	NM_005053<br>
XPA	Binds damaged DNA in preincision complex	NM_000380<br>
RPA1	Binds DNA in preincision complex RPA1, RPA2, RPA3	NM_002945<br>
RPA2		NM_002946<br>
RPA3		NM_002947<br>
ERCC5 (XPG)	3' incision	NM_000123<br><br><br>
Gene Name	Activity	Accession Number<br>
ERCC1	5' incision subunit	NM_001983<br>
ERCC4 (XPF)	5' incision subunit	NM_005236<br>
LIG1	DNA joining	NM_000234<br>
CKN1 (CSA)	Cockayne syndrome; Needed fortranscription-coupled NER CKN1, ERCC6, XAB2	NM_000082<br>
ERCC6 (CSB)		NM_000124<br>
XAB2 (HCNP)		NM_020196<br>
DDB1	Complex defective in XP group E DDB1, DDB2	NM_001923<br>
DDB2		NM_000107<br>
MMS19L (MMS19)	Transcription and NER	NM_022362<br>
FEN1 (DNase IV)	Flap endonuclease	NM_004111<br>
SP011	endonuclease	NM_012444<br>
FU35220 CENDOV)	incision 3' of hypoxanthine and uracil	NM_173627<br>
FANCA	Involved in tolerance or repair of DNA crosslinks FANCA, FANCB, FANCC,FANCD2, FANCE,FANCF, FANCG, FANCL	NM_000135<br>
FANCB		NM_152633<br>
FANCC		NM_000136<br>
FANCD2		NM_033084<br>
FANCE		NM_021922<br>
FANCF		NM_022725<br>
FANCG (XRCC9)		NM_004629<br>
FANCL		NM_018062<br>
DCLRE1A (SNM1)	DNA crosslink repair	NM_014881<br>
DCLRE1B (SNM1B)	Related to SNM1	NM_022836<br><br><br>
Gene Name	Activity	Accession Number<br>
NEIL3	Resembles NEIL1 and NEIL2	NM_018248<br>
ATRIP (TREX1)	ATR-interacting protein 5' alternative ORF of the TREX1/ATRIP gene	NM_130384<br>
NTH	Removes damaged pyrimidines	NP 416150.1<br>
NEI	Removes damaged pyrimidines	NP_415242.1<br>
NFI	Deoxyinosine 3' endonuclease	NP_418426.1<br>
MUTM	Formamidopyrimidine DNA glycosylase	NP_418092.1<br>
UNG	Uracil-DNA glycosylase	NP_417075.1<br>
UVRA	DNA excision repair enzymecomplexUVRA, UVRB, UVRC	NP_418482.1<br>
UVRB		NP_415300.1<br>
UVRC		NP_416423.3<br>
DENV	Pyrimidine dimer glycosylase	NP_049733.1<br>
Example 12: Repair of DNA prior to use in DNA sequencing reactions to increase the sensitivity of the sequencing 5    reactions<br>
The sensitivity of the sequencing reaction is intended to mean that the amount of template DNA having a correct sequence prior to sequencing results in reduced background noise and increased<br>
       signal. This makes possible longer and/or more complete sequence reads. The improved fidelity of the sequence read is an additional benefit. The beneficial use of a repair mix such as described below can be observed for sequencing methods in general. For example, sequencing methods include 454 sequencing, single molecule<br>
       sequencing, Sanger sequencing and Maxam-Gilbert sequencing.<br><br><br>
Two DNA samples are subjected to DNA sequencing before and after DNA repair. The two DNA samples are UV treated for 40 seconds (see Example 6) and lambda DNA is exposed to light in the presence of 25 ug/mL methylene blue.<br>
Prior to use in the DNA sequencing reaction the DNA to be sequenced is contacted with one or more repair enzymes under conditions that permit activity of the repair enzymes.  For example, 0.5 mg template DNA for sequencing is mixed with NEB Thermopol buffer to IX concentration (New England Biolabs, Inc., Ipswich, MA) (IX concentration of Thermopol buffer contains 20 mM Tris-HCI, pH 8.8 at 25°C, 10 mM KG, 10 mM (NH4)2S04, 2 mM MgS04, 0.1% Triton X-100) and incubated for 15 minutes at room temperature with a DNA repair mixture (200 units Taq DNA ligase, 0.1 units E. coli Poll, 1 unit T4 pdg, 15 units Endo IV, 2.5 units Endo VIII, 0.1 unit Fpg, and optionally 0.5 unit E. coli UDG). The repaired DNA is either used immediately for sequencing or is purified and concentrated using a commercial kit (a Qiagen, Inc., Valencia, CA kit, for example) prior to DNA sequencing. The sequencing reaction may be performed by the classical Sanger sequencing reactions or by methods described in U.S. Publication No. 2005/0100932, U.S. Patent No. 6,897,023, or Margulies, et al. Nature 437(7057):376-80 (2005).<br>
The sensitivity of the sequencing reaction and fidelity of the results are improved as a result of the pre-incubation with the repair mixture.<br>
Example 13: A multi-enzvme repair mix for repairing damaged DNA is effective at a single temperature<br><br><br>
Lambda DNA was treated by 30s irradiation with UV (see Example 6). L72-5R (SEQ ID NO:l) and L30350F (SEQ ID NO:2) primers were selected for amplifying a 5 kilobase amplicon from the UV-treated lambda DNA either with or without prior repair. The DNA repair mix contained 200 units/mLTaq DNA ligase, 0.1 units/mL E. coli Poll, 1 unit/mLT4 pdg, 15 units/mL Endo IV, 0.5 unit/mL E coli UDG, 2.5 units/mL Endo VIII, and 0.1 unit/mL Fpg stored in 20 mM Tris-HCI, pH 7.5 at 25°C, 100 mM NaCI, and 50% glycerol. Fifty ng of the 30 s UV-treated lambda DNA was added to thermocycler tubes each containing, IX Thermopol buffer, 100 mM dNTPs, and 0.5 mM NAD+. 1 mL of the repair enzyme mix was added to 4 out of the 8 tubes and all were brought to a final volume of 47 mL with H2O. Two tubes containing repair enzymes and 2 tubes lacking the enzymes were incubated at room temperature for 15 minutes. The remaining solutions were incubated overnight at 4°C. After the indicated incubation times, the primers (1 mM), dNTPs (100 uM), and 2.5 units Taq DNA polymerase were added to each thermocycler tube and the solutions placed into a MyCycler thermocycler running the program (Bio-Rad, Hercules, CA): 95°C for 2 min, one cycle; 95°C for 10 sec, 60°C for 30 sec and 72°C for 5 min, 25 cycles; 72°C for 5 min, one cycle; and a 4°C hold.  25 mL of each reaction was analyzed on a 1% agarose gel.<br>
In contrast to the findings of others (U.S. Publication No. 2006/0014154), which required the use of multiple different temperatures to achieve repair, incubation of UV-damaged DNA with the above repair mixture at room temperature for 15 minutes or 4°C overnight produced an amplification product of the correct size (Figures 13A and 13B).<br><br><br>
Example 14:  Repair of plasmid DNA containing multiple uracils and amplification using Vent® DNA polymerase<br>
Plasmid pNEB0.92U was purified from E coli CJ236 (NEB# E4141S, New England Biolabs, Inc., Ipswich, MA). The sequence is shown in Figure 18.  Because E. coli CJ236 lacks dUTPase and uracil-N-glycosylase, this plasmid contains uracils randomly distributed throughout its sequence. The archaeal polymerase Vent® DNA polymerase is inhibited by uracil containing templates. Amplification of a 920 base amplicon from pNEB0.92U DNA was examined using primers S1224S (CGCCAGGGTTTTCCCAGTCACGAC) (SEQ ID NO: 12 and S1233S (AGCGGATAACAATTTCACACAGGA) (SEQ ID NO: 13) either with or without prior repair. The DNA repair mix contained 200 units/mLTaq DNA ligase, 0.1 units/mLE coli Poll, 1 unit/ml_T4 pdg, 15 units/mL Endo IV, 0.5 unit/mLE coli UDG, 2.5 units/mL Endo VIII, and 0.1 unit/mL Fpg and stored in 20 mM Tris-HCI, pH 7.5 at 25°C, 100 mM NaCI, and 50% glycerol.  One mL of the repair enzyme mix was added to 2 of 4 thermocycler tubes each containing 0.5 ng pNEB0.92U, IX Thermopol buffer, 100 mM dNTPs, and 0.5 mM NAD+. All were brought to a final volume of45 mL with H2O. The reaction solutions were incubated at room temperature for 15 minutes after which the primers (final concentration: 0.4 mM), dNTPs (final concentration: 100 mM), and 1 unit Vent® DNA polymerase were added to each tube. The solutions were placed into a MyCycler thermocycler running the program: 95°C for 2 min, one cycle; 95°C for 10 sec, 65°C for 30 sec and 72°C for 1 min, 25 cycles; 72°C for 5 min, one cycle; then a 4°C hold.  25 mL of each reaction was examined by electrophoresis on a 1% agarose gel.<br>
Amplification from pNEB0.92U without removal of uracils<br><br><br>
using Vent® DNA polymerase produced a barely detectable product of the desired size. Treatment of the pNEB0.92U with the repair mix significantly increased the amount of amplicon produced from this plasmid using Vent® DNA polymerase (Figure 14).<br>
Example 15:  Enhanced amplicon yield from DNA fragments<br>
The template in this reaction was a set of 20 overlapping synthetic single strand oligonucleotides with an average size of approximately 45 nucleotides. The oligonucleotide sequences are shown below:<br>
NEB oligo No. 316-219 (New England Biolabs, Inc., Ipswich, MA):<br>
GGCGGCCTCGAGGCGAAACGCCGCAACTGCTTTCCGGGCGATACC (SEQID NO: 14)<br>
NEB oligo No. 316-220 (New England Biolabs, Inc., Ipswich, MA):<br>
CGGCACGCCATCAATCTGCACCAGAATGCGGGTATCGCCCGGAAA (SEQID NO: 15)<br>
NEB oligo No. 316-221 (New England Biolabs, Inc., Ipswich, MA):<br>
ATTGATGGCGTGCCGCAGAAAATTACCCTGCGCGAACTGTATGAA (SEQID NO: 16)<br>
NEB oligo No. 316-222 (New England Biolabs, Inc., Ipswich, MA):<br>
CATGTTTTCATAGCGTTCATCTTCAAACAGTTCATACAGTTCGCG (SEQID NO: 17)<br><br><br>
NEB oligo No. 316-223 (New England Biolabs, Inc., Ipswich, MA):<br>
CGCTATGAAAACATGGTGTATGTGCGCAAAAAACCGAAACGCGAA (SEQID NO: 18)<br>
NEB oligo No. 316-224 (New England Biolabs, Inc., Ipswich, MA):<br>
GGTTTCCAGGTCAATGCTATACACTTTAATTTCGCGTTTCGGTTT (SEQID NO: 19)<br>
NEB oligo No. 316-227 (New England Biolabs, Inc., Ipswich, MA):<br>
ATTGACCTGGAAACCGGCAAAGTGGTGCTGACCGATATTGAAGAT (SEQID NO:20)<br>
NEB oligo No. 316-228 (New England Biolabs, Inc., Ipswich, MA):<br>
CAGATGATCGGTCGCCGGCGCTTTAATCACATCTTCAATATCGGT (SEQID NO:21)<br>
NEB oligo No. 316-229 (New England Biolabs, Inc., Ipswich, MA):<br>
GCGACCGATCATCTGATTCGCTTTGAACTGGAAGATGGCCGCAGC (SEQID NO:22)<br>
NEB oligo No. 316-230 (New England Biolabs, Inc., Ipswich, MA):<br>
CAGCACCGGATGATCCACGGTGGTTTCAAAGCTGCGGCCATCTTC (SEQIDNO:23)<br><br><br>
NEB oligo No. 316-233 (New England Biolabs, Inc., Ipswich, MA):<br>
GATCATCCGGTGCTGGTGTATGAAAACGGCCGCTTTATTGAAAAA (SEQ ID NO:24)<br>
NEB oligo No. 316-234 (New England Biolabs, Inc., Ipswich, MA):<br>
TTTATCGCCTTCTrTCACTTCAAACGCGCG I I I I ICAATAAAGCG (SEQ ID NO:25)<br>
NEB oligo No. 316-237 (New England Biolabs, Inc., Ipswich, MA):<br>
AAAGAAGGCGATAAAGTGCTGGTGAGCGAACTGGAACTGGTGGAA (SEQ ID NO:26)<br>
NEB oligo No. 316-238 (New England Biolabs, Inc., Ipswich, MA):<br>
TTTCGGGTTATCCTGGCTGCTGCTGCTCTG7TCCACCAGTTCCAG (SEQ ID NO:27)<br>
NEB oligo No. 316-239 (New England Biolabs, Inc., Ipswich, MA):<br>
CAGGATAACCCGAAAAACGAAAACCTGGGCAGCCCGGAACATGAT (SEQ ID NO:28)<br>
NEB oligo No. 316-240 (New England Biolabs, Inc., Ipswich, MA):<br>
ATATTTAATGTTTTTAATTTCCAGCAGCTGATCATGTTCCGGGCT (SEQIDNO:29)<br><br>
NEB oligo No. 316-247 (New England Biolabs, Inc., Ipswich, MA):<br>
AAAAACATTAAATATGTGCGCGCGAACGATGATTTTGTGTTTAGCCTG (SEQID NO:30)<br>
NEB oligo No. 316-248 (New England Biolabs, Inc., Ipswich, MA):<br>
TTAATAATCACGTTATGATAI I I I I ICGCGTTCAGGCTAAACACAAA (SEQID NO:31)<br>
NEB oligo No. 316-265 (New England Biolabs, Inc., Ipswich, MA):<br>
TAACGTGATTATTAACGAAAACATTGTGACCCATGCGTGCGATG (SEQID NO:32)<br>
NEB oligo No. 316-266(New England Biolabs, Inc., Ipswich, MA):<br>
GCCGCCCTGCAGACCGGTCAGATCTTCATCGCCATCGCACGCATGGG (SEQIDNO:33)<br>
The assembly reaction consisted of two parts: an assembly step and an amplification step.  For the assembly step the standard reaction was 50 mL and contained 70 mM of each oligo, 100 mM dNTP, 0.5 mM NAD+, 10 mM Tris-HCI, pH 7.5 at 25°C, 2 mM MgCI2, and 50 mM NaCI.  No enzymes were added to the control reaction. The first experimental reaction also contained 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, and 20 units Endo IV. The second set of reactions contained enzymes used in the first reaction and lambda beta protein (Kmiec, et al. J. Biol. Chem. 256:12636-12639 (1981); Rybalchenko, N., et al, Proc. Natl. Acad. Sci. USA 101:17056-17060 (2004)) added at a 1:1 beta<br><br><br>
protein:nucleotide mole ratio. The third reaction set contained 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, 20 units Endo IV, 0.3 mM ATP, and a 3:1 nucleotide to RecA mole ratio. The RecA was from E. coli (NEB catalog #M0249L, New England Biolabs, Inc., Ipswich, MA). The fourth reaction contained 400 units Taq DNA ligase, 0.1 units E. coli Poll, 5 units T4 pdg, 20 units Endo IV, 0.3 mM ATP, a 3:1 nucleotide to RecA mole ratio, and a 1:1 beta protein nucleotide mole ratio. Glycerol content in each reaction was controlled for. The reaction mixtures were incubated for 30 minutes at room temperature.<br>
After room temperature incubation, 5 mL of the assembly reaction was amplified using 200 muM dNTPs, 500 nM in oligonucleotides 316-219 and 316-266, 6 mM in MgS04, 1 unit of Vent® DNA Polymerase, and IX Thermopol buffer final volume of 100 mL The reactions were mixed and placed in a MyCycler (Bio-Rad, Hercules, CA) and the following thermal cycler touchdown program was used: 94°C for 2 minutes (1 cycle); 94°C for 30 seconds, 72°C - 62°C (decreasing 1°C per cycle) for 30 seconds, 72°C for 45 seconds (10 cycles); 94°C for 30 seconds, 62°C for 30 seconds, 72°C for 45 seconds (20 cycles); 72°C for 5 minutes (1 cycle), and a 4°C hold. Each reaction was performed in duplicate. 11 mL of 10X sample buffer was added to each sample and 25 uL was loaded onto a 1% agarose gel for electrophoresis.<br>
The results are shown in Figure 15. When the assembly reaction contained no added repair proteins, no amplification product was detected after the amplification step.  However, when repair proteins were added in the assembly reactions, the correct 620 bp amplicon was obtained from the amplification step. Inclusion of lambda beta protein and/or E. coli RecA further<br><br><br>
increased the yield of amplicon. It was concluded that in a system in which a DNA template is composed of fragments, the inclusion of DNA repair proteins facilitates the ability to produce an amplicon. Furthermore, this effect is enhanced when some of those DNA repair proteins are known to be involved in DNA recombination.<br>
Example 16;  Enhanced transformation efficiency with damaged plasmid DNA for E. coli<br>
The plasmid pUC19 (GenBank Accession #L09137) was applied to a 1% agarose gel and electrophoresed in the presence of ethidium bromide until the plasmid had moved into the gel. The DNA in the gel was subjected to 254 nm UV light for 60 seconds. During the UV exposure a gel plug containing the pUC19 plasmid was excised. The plasmid was extracted from the gel plug using a Qiagen gel extraction kit (Qiagen, Valencia, CA).  30 ng of UV irradiated DNA or non-irradiated DNA in a final volume of 25 mL was treated with 50 units E. coli DNA ligase (NEB#M0205S, New England Biolabs, Inc., Ipswich, MA), 0.1 units E coli Poll, 5 units T4 pdg, and 20 units Endo IV in a buffer of IX Thermopol buffer (NEB#B9004S, New England Biolabs, Inc., Ipswich, MA) with added NAD+ (Sigma product #N-7004, Sigma-Aldrich, St. Louis, MO) and dNTPs (NEB#N0447S, New England Biolabs, Inc., Ipswich, MA) to 0.5 mM and 100 uM, respectively. The reaction was incubated 15 minutes at room temperature before using the DNA to transform E coli DH-5 alpha (NEB#C2991H, New England Biolabs, Inc., Ipswich, MA). As a control, both UV irradiated and non-irradiated DNA were treated as above in the absence of added enzymes. The DH5 alpha cells were transformed with UV irradiated and non-irradiated plasmid DNA that had been treated with repair enzymes or not so treated. The transformation was performed by heat<br><br><br>
shocking the E coli in the presence of plasmid DNA.  50 mL of E coli and plasmid were incubated on ice for 30 minutes before a 30 second incubation at 42°C. The transformation reaction was then placed on ice for 2 minutes before plating the cells on LB agar plates containing 100 mg/mL ampicillin.  LB agar plates with differing dilutions of each transformation were placed in a 37°C incubator overnight to determine the transformation efficiency of the plasmid.<br>
Plasmid pUC19 that had been subjected to UV irradiation and not repaired had a significantly reduced transformation efficiency when compared to undamaged pUC19 plasmid that had been treated with a repair enzyme mix (see Figure 16).<br>
Example 17: Simultaneous repair and blunting of DNA for subsequent ligation required for PCR, cloning or immobilization<br>
DNA libraries are commonly made from the environment, tissues, or cell cultures (Brady, S. F., et al. Applied and Environmental Microbiology, 70(ll):6865-6870 (2004); Current Protocols in Molecular Biology, Vol. 1, Ausubel, F., et al (editors), John Wiley &amp; Sons, Inc., Hoboken, NJ; Chapter 5: "Construction of Recombinant DNA Libraries" (2004); Courtois, S., et al., Applied and Environmental Microbiology, 69(l):49-55 (2003); US Patent No. 6,444,426). These libraries are routinely created by shearing the DNA from the desired source by sonication, enzymatic treatment, or nebulization, preparing the DNA ends and ligating the mixture to oligonucleotides or plasmid DNA (Weinmann, A. S., Molecular and Cellular Biology, 21(20):6820-6832 (2001)).  Ligation to oligonucleotides permits subsequent PCR or immobilization on<br><br><br>
arrays containing DNA sequences complimentary to the ligated oligo.  Ligation to a plasmid permits the propagation in a heterologous host. A recent use of libraries is in chromatin immunoprecipitation (Guenther, M. G., et al. Proc. Natl. Acad. Sci. USA 102(24):8603-8608 (2005); Ren, B., et al. Genes Dev. 16:245-256 (2002); and Odom, D. T., et al. Science 303:1378-1381 (2004)). As part of preparing DNA ends for blunt end ligation, researchers often use a polymerase such as T4 DNA polymerase. However, an enzyme mix is provided here that can not only repair damage that the DNA may have acquired during purification, preparation and storage, but can also create blunt ends. This enzyme mix includes a ligase and a proof-reading polymerase and any cofactors necessary to allow enzyme activity.  Preferably, the mix is composed of a ligase, a proof-reading polymerase, an apurinic/apyrimidinic endonuclease, UDG, FPG, and T4 pdg.<br>
As an example, Chromatin IP (ChIP) is performed on HeLa cell DNA using antibodies to E2F1, E2F2, E2F3, E2F4, E2F5, or E2F6 as described previously (Weinmann, A. S. Molecular and Cellular Biology 21(20):6820-6832 (2001)). The cloning of ChIP enriched DNA is as described previously (Weinmann, A. S. Molecular and Cellular Biology 21(20):6820-6832 (2001)); http://mcardle.oncology.wisc.edu/farnham). The use of T4 DNA polymerase alone to blunt the DNA is replaced by an enzyme mixture containing at least a combination of a ligase and a proofreading polymerase.  For example, the DNA is incubated with 400 units Taq DNA ligase, 0.1 units E coli polymerase I, 20 units E coli endonuclease IV, 5 units T4 PDG in IX Thermopol buffer supplemented with 0.5 mM NAD+ and 100 uM dNTPs at room temperature for 15 minutes.  Prior to the ligation step the blunted and repaired DNA can be incubated at 75°C for 20 minutes to<br><br><br>
inactivate the E. coli polymerase I.<br>
The mix of a proof-reading polymerase and at least a ligase is able to blunt the DNA ends for subsequent ligation to either primers or a plasmid.<br>
Example 18:  Enhanced amplicon yield from fragmented DNA. Production of larger DNA pieces from fragmented DNA for downstream processes such as amplification, DNA sequencing, microarray analysis, and hybridization analysis<br>
Fragmented DNA, from 0.1-1000 ng, is incubated with a recombination/DNA annealing proficient protein, such as E. coli RecA (NEB# M0249S, New England Biolabs, Inc., Ipswich, MA; West, S.C. Ann. Rev. Biochem. 61, 603-640 (1992)) and/or lambda beta protein (Rybalchenko, N., et al. Proc. Natl. Acad. Sci. USA, 101(49):17056-17060 (2004); Kmeic, E., &amp; Holloman, W. K., J. Biol. Chem. 256(24): 12636-12639 (1981)) in a standard reaction buffer and any required cofactors in a final volume of 5-1000 mL An example of a standard reaction buffer is 10 mM Tris-HCI, pH 7.5 at 25°C, 2 mM MgCI2, and 50 mM NaCI.  When using RecA, 1 mM ATP is included in the standard reaction. Either simultaneous with or subsequent to the incubation with the RecA and/or beta protein the DNA is also contacted with a repair mix composed of at least a DNA ligase activity a DNA polymerase activity and any required cofactors, i.e., ATP, NAD+ and dNTPs. The repair mix contains 400 units Taq DNA ligase and 0.1 units E. coli Poll and, in addition, 5 units T4 pdg, 20 units Endo IV, 0.5 units E. coli UDG, 2.5 units/mL Endo VIII, and/or 0.1 unit Fpg are added.  Prior to incubation with the repair proteins the DNA fragments may be heat-denatured and the temperature reduced to less than 39°C.  For example, the DNA<br><br><br>
in reaction buffer may be heated to 98°C for 5 minutes then cooled down to less than 39°C. A standard reaction volume is 5 to 1000 mL and the incubation time is 1 to 60 minutes at 4-37°C. Typically, the RecA or beta protein is used at a 0.5:1 to 5:1 nucleotide to protein mole ratio.<br>
Modifications to the above method include substituting RecA and/or beta protein with thermostable equivalents. Some examples of these proteins are ttRecA (Kato R, &amp; Kuramitsu S., Eur J Biochem. 259(3):592-601 (1999)), Taq RecA, Tma RecA, and Apy RecA (Wetmur, J. G., et al. J Biol Chem. 269(41):25928-35 (1994)). The use of thermostable proteins means that thermostable RecA or beta-like protein can be mixed with the DNA during the denaturation step. Any co-factors required for the protein activity are also included. In addition, repair enzymes as described above are added prior to or after denaturation. Note that for thermostable recombination proteins (RecA or beta-like protein) the proteins can be added to the reaction mixture for 1-60 minutes at 45-75°C to permit the optimal activity before the addition of non-thermostable repair proteins at temperatures of less than 39°C.<br>
The repaired DNA can then be used in a subsequent process, for example PCR.  For example, as a test system human genomic DNA is fragmented using sonication and size fractionated to give average fragment sizes clustered around 200 bp.  500 ng of the size-fractionated material is treated as described above. A titration of 5-100 ng of this repaired material is used in PCR reactions using primers that reliably generate 1, 2, and 4 kb amplicons from undamaged human genomic DNA.  Examples of a primer set are DNMT-R: GGGGCACCTTCTCCAACTCATACT (SEQ ID NO:34),<br><br><br>
DNMT-lFb:<br>
cctcatttggggaggggttatct (SEQ ID NO:35),<br>
DNMT-2Fc:<br>
cctgaaacaaggttgtggcatagc (SEQ ID NO:36), and<br>
DNMT-4Fb:<br>
gagtgagttgaaagtgctccataca (SEQ ID NO:37).<br>
The same template titration is performed with the fragmented DNA.<br>
When the non-repaired and repaired DNA are compared, the<br>
repaired templates permit a visible amplicon on an agarose gel,<br>
visualized with UV light and ethidium bromide, to be generated with<br>
at least two fold lower amounts of template DNA.<br>
The use of RecA and/or beta protein-like activities in conjunction with at least ligase and polymerase activities results in the detection of PCR amplicons at lower template amounts as compared to unrepaired DNA.<br>
Example 19: Amplification of DNA from stored ancient cave bear tissue samples after repair of DNA damage<br>
In contrast to modern material, the DNA extracted from ancient bones shows a variety of types of damage. The most common type of damage is fragmentation caused by single stand breaks, which lead to a reduced average molecule length of the extracted DNA, as well as non-enzymatic attacks such as irradiation and reactive oxygen species. (See:  Hoss, et al. Nucleic Acids Res. 24(7):1304-7 (1996)). Repairing ancient DNA (aDNA) damage is important to improve the utility of the extracted DNA.<br>
An ancient DNA was extracted as described in Paabo Proc Natl AcadSci USA 86(6):1939-43 (1989). Amplification of a 330 bp<br><br><br>
cave bear DNA (~44,000 years old) was performed using primers CB Fl (CTATTTAAACTATTCCCTGGTACATAC) (SEQ ID NO:38) and CB Rl (GGAGCGAGAGGTACACGT) (SEQ ID NO:39) either with or without prior repair. The DNA repair mix was composed of 200 units/mL Taq DNA ligase, 0.1 units/mL E. coli Poll, 1 unit/uLT4 pdg, 15 units/ML Endo IV, 0.5 unit/mL E. coli UDG, 2.5 units/mL Endo VIII, and 0.1 unit/mL Fpg and stored in 20 mM Tris-HCI, pH 7.5 at 25°C, 100 mM NaCI, and 50% gycerol. To 2 of 4 thermocycler tubes each containing 2 ml aDNA, IX Phusion™ polymerase buffer, 100 mM dNTPs, and 0.5 mM NAD+ was added 1 mL of the repair enzyme mix. All were brought to a final volume of 45 mL with H2O. The reaction solutions were incubated at room temperature for 15 minutes after which the primers (to 0.4 mM), dNTPs (to 100 mM), and 1 unit Phusion™ DNA polymerase were added to each. The solutions were placed into a MyCycler thermocycler (Bio-Rad, Hercules, CA) running the program: 98°C for 30 s, one cycle; 98°C for 10 sec, 58°C for 20 s and 72°C for 20 s, 30 cycles; 72°C for 5 min, one cycle; then a 4°C hold. The repaired PCR amplified DNA and control PCR amplified DNA (no repair) was used immediately in a second PCR amplification using nested primers. The amplification reaction with the nested primers used 1X Taq Master Mix (Catalog # M0270S, New England Biolabs, Inc., Ipswich, MA), 2 mL of the previous amplification, and primers CB Fl (CTATTTAAACTATTCCCTGGTACATAC) (SEQ ID NO:40) and CB F3 (GCCCCATGCATATAAGCATG) (SEQ ID NO:41) at a final concentration of 0.2 mM. The total reaction volume was 50 m/L. The reaction was analyzed by applying 5 ul of each reaction to a 1% agarose gel, prepared, electrophoresed and visualized as described above.<br><br><br>
The amount of mitochondrial DNA in the cave bear bone samples was estimated with the TaqMan® assay (Applied Biosystems, Foster City, CA) using primers 5'-AAAATGCCCTTTGGATCTTAAA-3' (SEQ ID NO:43) and 5'-ACTGCTGTATCCCGTGGG-3' (SEQ ID NO:44).<br>
The amplified DNA is either used immediately in the DNA sequencing methodology or subjected to DNA purification and concentration. After purification the DNA is subjected to DNA sequencing (see Example 12).<br>
Amplification from cave bear DNA using Phusion DNA polymerase and Taq polymerase in nested PCR produced a detectable product of the desired amplicon size in one sample (CB3A). Treatment with the repair mix produced another amplicon from sample CB3B.  Sequence analysis of amplicons from treated and untreated cave bear DNAs (CB3B sample) will reveal whether treatment with the repair mix significantly helped to remove PCR amplification errors associated with DNA modifications described in Hoss, et al. Nucleic Acids Res. 24(7): 1304-7 (1996).<br>
Treatment of the cave bear DNA template with the repair enzyme mix permitted Phusion™ DNA polymerase to more effectively produce the desired amplicon and to remove PCR amplification errors (Figure 17).<br><br><br>
WE CLAIM :<br>
1.	A method for enhancing at least one of fidelity and yield of a<br>
copied or amplified product by repairing a damaged polynucleotide,<br>
comprising:<br>
incubating the polynucleotide in the absence of Endonuclease (Endo) VI in a reaction mixture comprising a ligase and at least one of NAD+ or ATP as a cofactor; and<br>
enhancing at least one of fidelity and yield of the copied or amplified product.<br>
2.	A method according to claim 1, wherein the polynucleotide is obtained from a source selected from the group consisting of: a natural source, preserved biological material, forensic evidence, ancient material of biological origin, a tissue biopsy and chemical synthesis.<br>
3.	A method according to claim 1, wherein the damage is selected from apurinic/apyrimidinic (AP) sites, mutagenized nucleotides, modified nucleotides, nicks, gaps,  DNA-DNA or DNA-protein crosslinks, DNA-RNA crosslinks and RNA-RNA cross links.<br>
4.	A method according to claim 1, wherein the ligase is a thermostable ligase.<br>
5.	A method according to claim 1, wherein the ligase is selected from a Taq ligase and an E. coli ligase and the cofactor is NAD+.<br>
6.	A method according to claim 1, wherein the reaction mixture further comprises a T7 Endo I or mutant thereof.<br><br><br>
7.	A method according to claim 1, wherein the reaction mixture further comprises a polymerase and at least one AP endonuclease.<br>
8.	A method according to claim 7, wherein the at least one AP endonuclease comprises an endonuclease obtainable from E. coli, human or Thermus species.<br>
9.	A method according to claim 8, wherein the at least one AP endonuclease is E. coli Endo IV.<br><br>
10.	A method according to claim 1 or 7, wherein the polymerase is selected from the group consisting of: a Taq polymerase, an E. coli polymerase and an archaeal polymerase or mutant thereof.<br>
11.	A method according to claim 10, wherein the archael polymerase is selected from the group consisting of Pfu, Vent®, Deep Vent®, 9°North, KOD or Pwo polymerases.<br>
12.	A method according to claim 1 or 7 wherein the polymerase is a member of the Y family of polymerases.<br>
13.	A method according to claim 12, wherein the polymerase is selected from E. coli polIV, E. coli polV, human pol kappa, human pol eta, Sso Dpo4, Sac Dbh, See pol zeta and human pol iota.<br>
14.	A method according to claim 1 or 7, wherein the reaction mixture further comprises T4 pyrimidine dimer glycosylase.<br>
15.	A method according to claim 1 or 7, wherein the reaction mixture further comprises [fapy]-DIMA glycosylase (Fpg).<br><br><br>
16.	A method according to claim 1 or 7, wherein the reaction mixture further comprises at least one of UvrA, UvrB, UvrC, UvrD and Cho.<br>
17.	A method according to claim 1 or 7, wherein the reaction mixture further comprises at least one of Endo VIII, Endo V or Endo III.<br>
18.	A method according to claim 1 or 7, wherein the reaction mixture further comprises UDG and Aag.<br>
19.	A method according to claim 1, wherein amplification occurs by means selected from the group consisting of: PCR amplification, helicase-dependent amplification, strand-displacement amplification, rolling circle amplification and whole genome amplification.<br>
20.	A method according to claim 1, wherein the polynucleotide is an RNA and the amplification is RT-amplification.<br>
21.	A method according to claim 7, wherein the polynucleotide is DNA and the reaction mixture comprises 1-100 units of E. coli Endo IV, 0.05-0.25 units of E coli Poll and 5-500 units of Taq ligase added to 1-1000 ng DNA in 10-1000 \iL.<br>
22.	A method according to claim 1, wherein amplification of the polynucleotide is capable of producing an amplicon in a size range of 50 nucleotides to 100,000 nucleotides in a polymerase chain reaction.<br><br><br>
23.	A method according to claim 1, wherein incubating the polynucleotide in the reaction mixture is accomplished at a single temperature so as to enhance at least one of yield or fidelity.<br>
24.	A method according to claim 23, wherein the single temperature is room temperature and the incubation time is less than 1 hour.<br>
25.	A method according to claim 23, wherein the single temperature is 4°C and the incubation time is approximately 8-12 hours.<br>
26.	A kit, comprising: one or more enzymes wherein at least one of<br>
the enzymes is a ligase, the one or more enzyme being formulated<br>
for addition to a polynucleotide preparation to enhance copying of<br>
the polynucleotide and instructions for their use.<br>
27.	A composition, comprising:<br>
an effective amount of a ligase, a polymerase, and an AP endonuclease not including Endo VI for incubating with a polynucleotide at a single temperature so as to provide enhancement of at least one of yield and fidelity of a copied polynucleotide compared with a copied polynucleotide in the absence of the composition.<br>
28.	A composition according to claim 27, wherein the AP endonuclease is present at a concentration of 1-100 units, the polymerase is present at 0.05-5 units, and the ligase is present at 5-500 units in a volume 10-500 \i\ for use in a pre-amplification mixture.<br>
29.	A composition according to claim 27, suitable for enhanced copying of 1-1000 ng DNA.<br><br><br>
30.	A composition according to claim 27, wherein the AP endonuclease is a Type II AP Endonuclease.<br>
31.	A composition according to claim 30, wherein the AP endonuclease is E. coil Endo IV.<br>
32.	A composition according to claim 27, wherein the polymerase is E. coli poll.<br>
33.	A composition according to claim 27, further comprising T4 pyrimidine dimer glycosylase.<br>
34.	A composition according to claim 33, further comprising Fpg.<br>
35.	A composition according to claim 27, further comprising at least one of UvrA, UvrB, UvrC, UvrD and Cho.<br>
36.	A composition according to claim 27, further comprising at least one of Endo VIII, Endo V or Endo III.<br>
37.	A composition according to claim 27, at least one of UDG and Aag.<br>
38.	A method for cloning or sequencing a polynucleotide fragment, comprising:<br>
repairing sequence errors in the polynucleotide fragment by means of a composition according to claim 27; and<br>
cloning or sequencing the polynucleotide fragment.<br><br><br>
39.	A method according to claim 38, wherein the composition is capable of blunt-ending the polynucleotide fragment for cloning into a vector.<br>
40.	A method for enhancing the yield of a copied or amplified polynucleotide, comprising:<br><br>
(a)	obtaining at least a first pair and a second pair of primers wherein the second pair of primers is nested within the first set of primers when hybridized to the polynucleotide;<br>
(b)	subjecting the polynucleotide to a composition according to claim 27;<br>
(c)	amplifying the polynucleotide with the first set of primers;<br>
(d)	amplifying the product of (c) with the second set of primers; and<br>
(e)	obtaining an enhanced yield of amplified polynucleotide.<br><br>
41.	A method according to claim 38 or 40, wherein the composition comprises: a ligase, a polymerase, a pdg, an endonuclease IV, an Endonuclease VIII, Fpg and optionally UDG.<br>
42.	A method according to claim 41, wherein the ligase is Taq DNA ligase, the polymerase is E. coli Poll, the PDG is T4 pdg, and the Endo IV, Endo VIII, Fpg and optionally UDG are obtained from<br>
E. coli.<br>
43.	A method for sequencing a polynucleotide, comprising:<br>
(a)	contacting the polynucleotide with the composition of claim 27; and<br>
(b)	sequencing the polynucleotide.<br><br><br>
44.	A method for copying or amplifying a fragmented DNA,<br>
comprising:<br>
(a)	contacting the fragmented DNA with the composition of claim 27;<br>
(b)	optionally adding a recombination competent protein; and<br>
(c)	amplifying or copying the fragmented DNA.<br>
45.	A method according to claim 44, wherein the recombination<br>
competent protein is a RecA or beta protein.<br><br><br><br>
ABSTRACT<br>
Methods and compositions are provided for repairing a polynucleotide so that it can be copied with improved fidelity and/or yield in, for example, an amplification reaction. This involves the use of a reaction mixture that includes a ligase and a cofactor selected from NAD+ or ATP and incubating the polynucleotide with the reaction mixture in the absence of Endonuclease VI. The reaction mixture may further contain an AP endonuclease and a polymerase. If used, these enzymes may be selected according to their ability to withstand high temperatures. For example, the reaction mixture may be used prior to a polynucleotide synthesis reaction in which case enzymes that are not thermophilic may be used. The repair reaction is not time sensitive with respect to seconds, minutes or hours of incubation in the enzyme mixture in as much as the repair is effected rapidly and prolonged incubation is not generally adverse.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQUJTVFJBQ1QoR1JBTlRFRCktKDIyLTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-ABSTRACT(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0FOQ0VMTEVEIFBBR0VTKDExLTEwLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CANCELLED PAGES(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKEFNRU5ERUQpLSgxMS0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(AMENDED)-(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKEFNRU5ERUQpLSgxMS0yLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(AMENDED)-(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKEdSQU5URUQpLSgyMi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMTEtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(MARKED COPY)-(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMTEtMi0yMDExKS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(MARKED COPY)-(11-2-2011)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMTEtMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(MARKED COPY)-(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ0xBSU1TKE1BUktFRCBDT1BZKS0oNi05LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CLAIMS(MARKED COPY)-(6-9-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMS03LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(1-7-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMTQtMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(14-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMi0yLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(2-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMjEtMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(21-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoOS02LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(9-6-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoSVBPKS0oMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-CORRESPONDENCE(IPO)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQucGRm" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtREVTQ1JJUFRJT04oR1JBTlRFRCktKDIyLTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-DESCRIPTION(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRFJBV0lORyhHUkFOVEVEKS0oMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-DRAWING(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAxKDExLTEwLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 1(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAxKDExLTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 1(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAxKDE0LTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 1(14-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAxKDIxLTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 1(21-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAxOCg4LTktMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 18(8-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAyKEdSQU5URUQpLSgyMi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 2(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAyKFRJVExFIFBBR0UpLSgyMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 2(TITLE PAGE)-(21-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAyKFRJVExFIFBBR0UpLShHUkFOVEVEKS0oMjItMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 2(TITLE PAGE)-(GRANTED)-(22-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAzKDExLTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 3(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAzKDE0LTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 3(14-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAzKDIxLTQtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 3(21-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtRk9STSAzKDctNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-FORM 3(7-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS0yLmRvYw==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS1wY3QtaXNhLTIyMC5wZGY=" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-pct-isa-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS1wY3QtaXNhLTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-pct-isa-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS1wY3Qtcm8tMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-pct-ro-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtZm9ybS1wY3Qtc2VwYXJhdGUgc2hlZXQtMjM3LnBkZg==" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-form-pct-separate sheet-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtT1RIRVIgRE9DVU1FTlQoMS0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-OTHER DOCUMENT(1-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LW11bW5wLTIwMDgtcGN0LXNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">779-mumnp-2008-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMS0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-PETITION UNDER RULE 137(1-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMTEtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-PETITION UNDER RULE 137(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUE9XRVIgT0YgQVRUT1JORVkoMTMtNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-POWER OF ATTORNEY(13-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDExLTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-REPLY TO EXAMINATION REPORT(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUkVQTFkgVE8gSEVBUklORygxLTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-REPLY TO HEARING(1-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUkVQTFkgVE8gSEVBUklORygxMS0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-REPLY TO HEARING(11-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUkVQTFkgVE8gSEVBUklORyg1LTEwLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-REPLY TO HEARING(5-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtUkVQTFkgVE8gSEVBUklORyg1LTktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-REPLY TO HEARING(5-9-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtU0VRVUVOQ0UgTElTVElORygyMS00LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-SEQUENCE LISTING(21-4-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtVVMgQVNTSUdOTUVOVCgxMS0yLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-US ASSIGNMENT(11-2-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Nzc5LU1VTU5QLTIwMDgtV08gSU5URVJOQVRJT05BTCBQVUJMSUNBVElPTiBSRVBPUlQoMjEtNC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">779-MUMNP-2008-WO INTERNATIONAL PUBLICATION REPORT(21-4-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251092-a-composition-of-at-least-two-amide-based-compounds-represented-by-general-formula-i.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251094-a-process-for-a-single-stage-melt-polymerization-for-the-production-of-a-polybenzimidazole.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251093</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>779/MUMNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Apr-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NEW ENGLAND BIOLABS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>240 COUNTY ROAD, IPSWICH, MA 01938,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>EVANS, THOMAS C.</td>
											<td>45 FOX RUN ROAD, TOPSFIELD, MA 01983</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SLATKO, BARTON</td>
											<td>11 BRENTWOOD WAY, IPSWICH, MA 01938</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHEN, LIXEN</td>
											<td>8 MORRISON AVENUE, BEVERLY, MA 01915</td>
										</tr>
										<tr>
											<td>4</td>
											<td>VAISVILA, ROMALDAS</td>
											<td>100 CENTRAL STREET, UNIT 8, IPSWICH, MA 01938</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GUAN, CHUDI</td>
											<td>10 GREAT POND ROAD, WEMHAM, MA 01984</td>
										</tr>
										<tr>
											<td>6</td>
											<td>KUCERA, REBECCA</td>
											<td>160 WOODLNAD MEAD, HAMILTON, MA 01982</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/013513</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-11</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/US2005/038281</td>
									<td>2005-10-20</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251093-repair-of-nucleic-acids-for-improved-amplification by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:04:40 GMT -->
</html>
