<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251014-method-for-the-detection-and-characterization-of-microorganisms-on-a-filter by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:07:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251014:&quot;METHOD FOR THE DETECTION AND CHARACTERIZATION OF MICROORGANISMS ON A FILTER&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR THE DETECTION AND CHARACTERIZATION OF MICROORGANISMS ON A FILTER&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method for the specific detection on a membrane of one or more microorganisms present in a liquid, a gas or on a surface, characterized in that it comprises the following steps: (i) the liquid, the gas or the surface is put in contact with the membrane, having a pore diameter of between 0.1 and 1.5 microns so as to trap the microorganism or microorganisms on the membrane; (ii) said microorganisms are treated on said membrane by contacting the membrane with a fixation solution containing a cross-linker selected from glutaraldehyde, formaldehyde and/or paraformaldehyde; (iii)nucleic acids present in the microorganism or microorganisms are specifically isothermally amplified, at a temperature of between 35 and 90°C, on said membrane in order to obtain amplification products; (iv)the obtained amplification products are detected, using standard detection methods, directly on said membrane where the microorganisms have been trapped.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a method for the specific detection on a filter of one or more microorganisms present in a fluid, comprising the step of specific isothermal amplification of the nucleic acids of said microorganisms, as well as to a detection kit and sequences for the implementation of this method.<br>
Nowadays the monitoring of the microbiological quality of water and air as well as used or disposed-of tools and materials, within the scope of industrial or medical activities, is subject to increasingly strict standards.<br>
For this reason, industrial and health authorities must be able to have tools allowing them to detect microbiological contamination as early as possible in order to be able to remedy it as soon as possible and at low cost.<br>
Traditionally, microbiological analysis is carried out on agar culture media. These types of cultures, easy to implement, allow germs to be counted with the naked eye and the microorganisms to be kept alive, also allowing characterization tests to be carried out, if appropriate.<br>
These characterization tests generally consist of extracting the nucleic acids contained in the microorganisms and specifically amplifying said nucleic acids, by one of a number of chain reaction techniques (PCR: polymerase chain reaction or LCR: ligase chain reaction) known to a person skilled in the art.<br>
However, although the characterization tests by amplification of nucleic acids are very reliable, the steps consisting of culturing the microorganisms, extracting the nucleic acids there from, and implementing one of the chain reaction techniques described above are time consuming.<br>
To be visible to the naked eye, the microorganisms must be left in culture for at least 24 hours. It is sometimes longer for microorganisms that grow more slowly, such as mycobacteria or for microorganisms that have been stressed by environmental conditions.<br>
Furthermore, the extraction of nucleic acids, which cannot be carried out directly on the agar medium, requires the microorganisms to be sampled prior to their characterization.<br>
PCR, which consists of numerous amplification cycles at different temperatures, has eventually the drawback that it must be carried out in solution in tubes of small dimension which are placed in a thermocycler, which entails the additional work of transferring the nucleic acids before and after amplification.<br>
The recent development of lab-on-a-chip facilities, using different techniques of hybridization with nucleic acid probes fixed on a solid support, could allow time to be saved at the step of characterizing the microorganisms. However, these techniques have an even higher production cost and do not avoid the steps of extraction and transfer of the nucleic acids prior to their characterization.<br>
Under these conditions, more than 24 hours must still be allowed for the microorganisms to be counted and characterized, which is far too long in an industrial context where a permanent monitoring of the quality of the products must be ensured.<br>
In the particular field of the quality control of industrial water intended for the composition of injectable pharmaceutical solutions, for instance, information on the quality of the water introduced into the production chain must be able to be obtained virtually in real time.<br>
The detection of the microorganisms must take place in situ and, if possible, with a minimum of equipment and staff.<br>
Implementing detection is all the more difficult in this context that the desired level of detection is of the order of only a few microorganisms per litre of water, which requires a prior step of concentrating the microorganisms by filtration.<br>
To respond to these limitations, an approach for the rapid determination of the presence or absence of microorganisms has been first proposed by the applicant in the international application WO 01/59157.<br>
This approach consists of reducing culture duration by basing the detection of the microroganisms on their metabolic activity.<br><br>
A universal metabolic marker, which for example can be ATP contained in the microorganisms, or also the emission of CO2, is measured, thus establishing the presence of the microorganisms before colonies are visible to the naked eye on the agar medium.<br>
In the example of the microbiological monitoring of water, a certain volume of water (comprised between 100 ml and 11) is filtered through a filtering membrane so as to trap the microorganisms contained in the water. The filtering membrane is then placed on an agar medium, which is exploited by the microorganisms for initiating growth. Then, the membrane is recovered to detect the ATP of the microorganisms present on its surface, by reacting ATP with an enzyme (Luciferase) producing photons. The luminescence is measured using an image analysis system.<br>
This system, marketed by the applicant under the name Milliflex® Rapid system, has already reduced the time necessary to establish the presence or absence of microorganisms in a liquid by several hours.<br>
This system produces quantitative, but not qualitative, information. When the nature of the microorganisms present has to be established, or a given microorganism is specifically sought, a step of characterization, generally carried out by PCR, must be added which, as previously mentioned, significantly increases the duration of the test.<br>
Meanwhile, a technique for the specific amplification of nucleic acids has been developed under the name LAMP (loop-mediated isothermal amplification) the principle of which is described in the International Applications WO 00/28082, WO 01/077317 and WO 02/024902.<br>
Unlike PCR, this method has the feature of being carried out at constant temperature (isothermal condition) while maintaining a high level of specificity.<br>
It is an object of the present invention to provide a method obviating the drawbacks of earlier detection methods consisting in implementing the detection directly on a filter by amplification of their nucleic acids.<br><br>
In a surprising manner, an isothermal amplification technique, such as the LAMP technique, usually implemented in solution after culture and extraction of nucleic acids, can be performed on a filter where the microorganisms have been trapped, without necessarily resorting to the step of culturing the microorganisms and transferring their nucleic acids to a tube for amplification.<br>
The method of the invention therefore allows the microorganisms present in a volume of water, gas or on the surface of a solid, to be detected over a reduced time period in a specific manner with a detection threshold of only a few microorganisms.<br>
In a preferred aspect, the invention allows individual counting in situ on a membrane, i.e. in a given part of the membrane, of the microorganisms trapped thereon.<br>
The present invention therefore relates to a method for the specific detection on a filter of one or more microorganisms present in a fluid, characterized in that it comprises the following steps:<br>
a)	contacting the microorganisms present in the fluid on the filter;<br>
b)	amplifying specifically the nucleic acids from the microorganism or<br>
microorganisms present on the filter, in an isothermal manner, in order to<br>
obtain amplification products;<br>
c)	detecting the amplification products, preferably on said filter.<br>
More specifically, the present invention relates to a method for the specific detection on a filter of one or more microorganisms present in a liquid, a gas or on a surface, characterized in that it comprises the following steps:<br>
a)	the liquid, the gas or the surface is put in contact with the filter so as<br>
to trap the microorganism or microorganisms on the filter;<br>
b)	the nucleic acids present in the microorganism or microorganisms are<br>
specifically isothermally amplified on the filter in order to obtain amplification<br>
products;<br>
c)	the obtained amplification products are detected.<br><br>
The invention also relates to a device, a kit and primers specifically suitable for the implementation of the method according to the invention.<br>
The aim of the figures below is to illustrate certain features of the invention:<br>
FIGURE 1: Principle of the LAMP amplification technique suitable for the detection of messenger RNAs contained in the microorganism or microorganisms, preferably used in step b) of the detection method according to the invention.<br>
FIGURE 2: DNA sequence originating from the oprL gene (5' -&gt; 3') showing the primers used for the specific detection of Pseudomonas aeruginosa by the LAMP technique according to the method of the invention. The sequences corresponding to the primers used according to the invention are shaded and their orientation is indicated by the arrows.<br>
FIGURE 3: Steps of the method according to the invention, implemented in Example 1.<br>
FIGURE 4: Illustrations of steps b) and c) of the method according to the invention, described in Example 1<br>
FIGURE 5: Steps of the detection method according to the invention, implemented in Example 2.<br>
FIGURE 6: Steps b) and c) of the detection method according to the invention, described in Example 2.<br>
FIGURE 7: Steps of the detection method according to the invention, implemented in Example 3.<br><br>
FIGURE 8:  Perspective view of a device for the amplification of nucleic acids intended to implement the method according to the invention.<br>
FIGURE 9:   Cross-section of the device in Figure 8.<br>
FIGURE 10:   Cross-section of the device in Figure 8 without its flat wall and its cover.<br>
FIGURE 11:   Top view of the figure of the device in Figure 10. FIGURE 12:   Perspective exploded view of the device in Figure 8.<br>
FIGURE 13:   Perspective view of the device in Figure 8 after its reservoir has been removed.<br>
FIGURE 14:   Transverse cutaway view of the device in Figure 13. FIGURE 15:    Perspective exploded view of the device in Figure 13. Detailed description of the invention:<br>
The detection method according to the invention allows the monitoring of the microbiological quality of a fluid, such as a liquid or a gas, but also of a surface.<br>
This method is thus applicable to a large number of both industrial and health applications, such as for example the monitoring of water, during production and retreatment in agri-food or pharmaceutical contexts, of air emitted from air-conditioning systems or inside clean rooms, or the monitoring of the asepsis level of work surfaces in laboratories, factories and operating theatres.<br>
This method can afso be used for diagnostic purposes, for example for the analysis of urine in the case of urinary infection, cerebrospinal fluid in the case of meningitis, amniotic fluid in the case of fcetal infections or<br><br>
abnormalities, plasma for viral infection research, or exhaled air in the case of bronchial or pulmonary infection.<br>
More generally, it constitutes an alternative to the lab-on-a-chip for the characterization of nucleic acids, regardless of the envisaged application.<br>
The microorganisms aimed at by the method of the invention are in general chosen from the pathogenic bacteria of the genera Pseudomonas, Escherichia, Legionella, Salmonella, Listeria, Bacillus, Streptococcus, Vibrio, Yersinia, Staphylococcus, Mycobacterium, Shigella, Clostridium, Campylobacter, or Aeromonas; protozoa of the genera Giardia, Entamoeba, Cryptosporidium, Cyclospora; mycoplasmas of the genera Mycoplasma and Ureaplasma, fungi of the genus Aspergillus, Candida or Peniciflium, the hepatitis A and E viruses, which are pathogenic contaminants present in the environment. This list is not meant to be limiting but merely to identify the organisms of most interest in the industry. Other organisms may also be of interest and it is meant to include them within the present invention.<br>
In the sense of the present application, the term "microorganism" is not restricted to bacteria, fungi and yeast. It also applies to other microscopic living forms like unicellular algae such as cyanobacteria, unicellular forms of parasitic organisms, such as amoeba or nematode eggs, or also plant or animal gametes such as pollens or spermatozoa, and even microscopic pluricellular organisms like trematodes or ascaris.<br>
According to one aspect of the invention the microorganisms are initially present in a liquid, a gas or on a surface.<br>
A liquid in the meaning of the invention may be a solution which can comprise a solid phase in suspension, more preferentially a solution in aqueous medium. A gas is preferably constituted by a mixture of gas such as air, but can also by extension consist of a mixture of solid, liquid and gas particles, such as an aerosol.<br>
A surface is preferably that of a solid such as for example the floor or the walls of a clean room.<br><br>
The microorganisms are trapped, i.e. retained, on a filter or in the depth of said filter, either by filtration or by contacting said filter with a contaminated surface.<br>
By "filter" is meant, in the sense of the present invention, a medium of which the main constituent is a material, which, by its nature, its size and/or its arrangement, retains the microorganisms contained in suspension in a liquid or gaseous flow passing through it.<br>
The term "filter" thus designates various types of filtering items such as filtration pads, membranes or microcolumns filled with chromatography media such as microbeads, gels or resins.<br>
According to a preferred embodiment, the material which is the main constituent of the filter is coated with antibodies or chemical groups, which interact with the outer membrane of the microorganisms or with the nucleic acids therefrom to help trap or affix the organisms to the filter.<br>
The filter can be mono- or multi-layered. In genera! it is constituted by one or more materials chosen from glass, latex, potytetrafluoroethylene, po!y{vinylidene) fluoride, polycarbonate, polystyrene, polyamide, poiysulphone, polyethersulfone, acetyl cellulose, mixed celluloses and nitrocellulose, or any materials known by one skilled in the art in the field of filtration items.<br>
In a preferred embodiment the filter is a membrane. The pores of the membrane in general have an average diameter comprised between 0.1 and 1.5 microns, preferably between 0.15 and 0.8 microns, more preferentially between 0.2 and 0.6 microns.<br>
The materials are preferably chosen to be compatible with the solvents used during steps b) and c) of the method, in particular alcohols and aldehydes capable of being used to fix the microorganisms on the filter and release the nucleic acids contained in these microorganisms.<br>
By "membrane" is meant in the present Application a filter constituted by a synthetic support having two surfaces, the pores of which have an average known diameter. This average known diameter is calculated so as to prevent at least 99 % of the microorganisms having the dimension of<br><br>
said average diameter from passing through the membrane. The membrane used within the scope of the present invention in general has a high surface/volume ratio and a constant thickness preferably comprised between 90 and 200 microns.<br>
A membrane according to the invention is preferably a PVDF filtering membrane, such as for example the membrane marketed by Millipore Corporation under the brand name MXHVWP124.<br>
According to a preferred aspect of the invention, the microorganism or microorganisms are trapped on the membrane by filtration, which is carried out by passing a liquid or a gas through the pores of the membrane by applying a pressure difference between the two surfaces of the membrane. The microorganism or microorganisms are then retained by the surface and/or pores of the membrane.<br>
According to a preferred feature of the invention, step a) of the method consists of a filtration which aims to concentrate the microorganisms present in the liquid or the gas on the membrane.<br>
When attempting to retain yeast- and mold-type microorganisms, it is advisable to choose an average pore diameter comprised between 0.5 and 1.5 microns, preferably between 0.6 and 1 micron. For bacterial microorganisms, an average diameter comprised between 0.2 and 0.8 micron, preferably between 0.3 and 0.5 micron, is chosen. For mycoplasma-type microorganisms and viruses, a diameter of less than 0.2 micron, preferably between 0.1 and 0.2 micron, is chosen. The pore size is obtained by statistical calculation since in general the pores result from the superposition of different layers of the same material or different materials each with an irregular porosity.<br>
It can be advantageous during filtration to use a filtering membrane comprising a hydrophobic-type mesh forming a network of a solution of hydrocarbonated, non-wettable waxes, vaseline, silicone waxes or oils, epoxy resins, potyfluoroethylene or polystyrene, so as to better redistribute the microorganisms over the whole membrane and to be able to count them better in step c).<br><br>
In the absence of filtration, the microorganisms are trapped by simply putting the microorganism in contact with the surface of the filter. This contact can be produced by active projection of a liquid or a gas in the direction of a filter (air flow, water current) or passively when the microorganism is deposited by gravity on the filter. Conversely, the filter can be applied to a solid surface on which the microorganism is located. For this trapping method, it can be advantageous for the surface of the filter to be covered with a coating facilitating the adhesion of the microorganisms. This coating can be made of glycerol for example.<br>
If desired, a step of culturing the microorganism or microorganisms immobilized on the filter can be carried out before amplification step b). Such a culturing can take place directly on the filter Ifke a membrane by putting this one in contact with a nutritive medium, in particular an agar medium, which if appropriate allows the number of microorganisms present to be increased, while respecting the initial location of the microorganism or microorganisms on the filter. The colonies obtained can serve as biological materials for step c), or can be sampled with a view to other analyses, such as for example quantification by ATP bioluminescence.<br>
The nucleic acids contained in the microorganism or microorganisms trapped on or in the depth of the filter consist essentially of double- or single-stranded DNA and RNA. These can consist of in particular chromosomic DNA, chloroplastic DNA, mitochondrial DNA, ribosomic RNAs, transfer RNAs or messenger RNAs.<br>
According to a preferred mode of the invention, the external wall of the microorganisms is lysed before amplification step c) in order to release the nucleic acids contained in the microorganisms. The lysis step can be carried out directly on the filter, for example, by impregnating the latter with a lysis buffer. It can be completed if appropriate by a purification step consisting, for example, of precipitating the nucleic acids with alcohol on the surface of or inside the filter.<br>
According to a preferred mode of the invention, the microorganisms trapped on the fitter with their nucleic acids are fixed on the<br><br>
filter by an appropriate treatment. This fixation step prevents the migration and mixing of nucleic acids originating from the different microorganisms. This treatment can consist of the impregnation of the filter by a fixing solution comprising, for example, a cross-linker such as glutaraldehyde, formaldehyde or paraformaldehyde. Glutaraldehyde is particularly advantageous when a polyamide-based membrane is used.<br>
Another mode of fixing according to the invention consists of irradiating the filter with ultraviolet (UV) rays in order to obtain a cross-linking reaction between the materials of which the filter is composed and the nucleic acids.<br>
According to step b) of the method according to the invention, the nucleic acids present in the microorganism(s) are specifically amplified on the filter in isothermal manner in order to obtain amplification products.<br>
By specific amplification is meant here a polymerization reaction of nucleic acids initiated by the hybridization of a specific primer complementary to a nucleic acid serving as template, with a view to reproducing all or part of the sequences comprising this nucleic acid serving as template.<br>
Unlike PCR techniques, the amplification step b) according to the invention does not require cyclic changes of temperature, and therefore requires neither the use of a thermocycler, nor consumables of small dimension which are associated with it, designed to ensure an effective heat transfer to the reaction medium.<br>
According to a preferred feature of the invention, the amplification step is carried out on a filter, i.e. on the surface of or inside the latter, and more preferentially in an enclosure at controlled temperature. In general the temperature is maintained between 35 and 90°C, preferentially between 50 and 70°C, and more preferentially between 60 and 65°C.<br>
Different methods for the isothermal amplification of the nucleic acids can be applied to the method of the invention such as, for example, the NASBA technique (nucleic acid sequence based amplification), or TMA<br><br>
 (transcription mediated  amplification), which  allows  single-stranded  RNA amplicons to be obtained [Nature, 1991, 350:91-92].<br>
A preferred amplification technique according to the invention is the LAMP technique by which a DNA template is reproduced via amplicons forming stem loops.<br>
The principle of the LAMP technique is described in the International Application WO 00/28082.<br>
A preferred adaptation of this technique according to the invention is illustrated in Figure 1.<br>
By LAMP-type amplification technique is meant any isothermal amplification technique derived from the LAMP technique, comprising at least the step by which a DNA polymerase, having a strand-displacement activity, synthesizes from a template of nucleic acids a strand of DNA complementary to the latter via a specific "FIP" primer (Forward Inner Primer). The FIP primer is characterized in that it hybridizes to the template at its 5' part, but not at its 3' part. In fact its 3' part has a sequence which can subsequently hybridize with one of the sequences of the complementary strand formed. The result is that, once the complementary strand has synthesized, the 3' part of FIP will favour hybridization on itself of the synthesized strand, creating a stem loop structure. This stem loop structure is then reproduced in the subsequent steps of the amplification.<br>
The LAMP-type amplification preferentially requires a second "BIP" primer (Backward Inner Primer) which has the same function as FIP, but on the opposite strand. More preferentially, the complementary primers called F1, F2, F3 (F: Forward) and B1, B2, B3 (B: Backward) are used. The F1, F2, F3 primers correspond to sequences situated on the coding strand and the B1, B2, B3 primers to sequences on the complementary strand. F3 and B3 correspond to the 5' ends of the amplified fragments, F2 and B2 correspond to the sequences situated at the 5' end of FIP and BIP, F1 and B1 correspond to the complementary sequences of the sequences situated at the 3' end of FIP and BIP (called F1c and B1c).<br><br>
According to the type of nucleic acids to be amplified, a phase of denaturation of the DNA by heating to between 60° and 80°C aimed at separating the DNA strands of the double-stranded DNA molecules may be necessary at the start of the amplification procedure.<br>
In general, a LAMP-type amplification technique uses a single- or double-stranded DNA as starting template.<br>
According to a preferred aspect of the method according to the invention, the starting template is a cDNA.<br>
According to a more preferred feature of the invention, the RNAs of the microorganisms are retro-transcribed in advance with the help of a reverse transcriptase using a BIP primer, as indicated in Figure 1, which allows a cDNA comprising a stem loop structure to be obtained, allowing the LAMP-type amplification to be started directly.<br>
According to an even more preferred feature of the invention, the RNAs originating from the microorganisms used for the previous retro-transcription step are messenger RNAs. In fact, messenger RNAs have the advantage of being present in much greater numbers than the DNA molecules serving for their transcription. In this way it is possible to lower the detection threshold of the method according to the invention.<br>
The LAMP-type amplification technique advantageously comprises a phase during which additional primers loop F and loop B hybridize to the amplification products at the level of the sequences constituting the loops. These primers promote the synthesis of new complementary strands starting from the loops that comprise the amplification products, which allows the speed of formation of the amplification products to be increased.<br>
The amplification products obtained according to a LAMP-type technique are generally double-stranded DNA molecules comprising several stem loops. The size of the amplification products varies as a function of the number of loops synthesized. In general, several sizes of fragments co-exist at the end of the amplification method, which results in an electrophoretic profile of the type represented in Figure 4.<br><br>
According to a preferred feature of the invention, step c) of the detection of the amplification products is carried out by methods directly or indirectly using the marked molecules, allowing detection by fluorescence, colorimetry or chemoluminescence of the amplification products.<br>
An image analyzer can then be used to record the emission spectrum of the marked molecules and the position of the latter on the filter. By appropriate treatment of the signal, aimed in particular at eliminating the parasitic signals linked to the non-specific amplifications, it is possible to view the amplification products in situ. This is particularly useful when a membrane is used, because the large surface of the membrane allows quantitative and qualitative precision.<br>
The means for detecting the amplification products by colorimetry or fluorescence are known to a person skilled in the art.<br>
Preferably, the detection means used according to the method of the invention are specific and allow the individual identification of the microorganism or microorganisms the nucleic acids of which have been amplified.<br>
A detection means can consist, for example, of the incorporation of modified purine or pyrimidine bases into the amplification products during step b) of the method, as substrate of the amplification reaction. Preferably, the marked purine or pyrimidine bases are dNTPs, more preferentially dUTPs. Another detection means can consist of hybridizing at the end of step b) the amplification products obtained with one or more marked specific probes. Different types of probes can be used, such as, for example, PNA-type probes (Peptide Nucleic Acid), i.e. probes constituted by a polypeptide chain substituted by purine and pyrimidine bases, the spatial structure of which mimics that of a DNA [Nielsen, P.E. et al. (1991) Sequence selective recognition of DNA by strand displacement with a thymine-substituted polyamide, Science 254:1497-1500].<br>
The probes used are advantageously of the "molecular beacon" type, i.e. they have a fluorophore, a quencher group and a stem loop structure<br><br>
allowing a fluorescence of the probe only when it is hybridized to the amplification products.<br>
When the amplification products are detected using hybridization probes, it may prove necessary to proceed to a preliminary step of denaturation of the amplification products, aimed in particular at separating the DNA strands.<br>
Another detection means according to the method of the invention consists of the use of specific ligands, preferably marked antigens and antibodies. In this case, modified purine or pyrimidine bases, or even specific probes, are marked using one or more molecules serving as antigens, such as dioxygenin, being able to be specifically recognized by an antibody such as for example an antidioxygenin antibody. Alternatively, the detection probes can be coupled to an antibody which is then put in contact with a marked antigen. The antibodies or antigens can be conjugated to a fluorophore, such as FITC (a particular form of isothiocyanate).<br>
The marking of the modified purine or pyrimidine bases, or of the specific probes can be carried out by radioactive isotopes (H3, P32, P33, S35...) or by non-radioactive products such as, for example, blotin, which is detected by adding a solution containing conjugated avidin or streptavidin to an enzyme such as Raifort peroxidase or soya seed or alkaline phosphatase. When a substrate of these enzymes is spread, for instance on the surface of the membrane, these enzymes release photons, which causes a detectable luminescent reaction.<br>
A preferred method of marking according to the invention consists of coupling the hybridization or the ligands mentioned above to fluorescent molecules or fluorophores. Fluorophores having different emission spectra can advantageously be used during the detection step c), coupled to the ligands or probes defined to recognize the amplification products originating from different microorganisms. To this end, fluorophores such as those marketed under the name ALEXA FLUORINE® by Molecular Probes or DY-® by Dyomics are advantageously used, since they have different emission spectra. A parallel detection carried out on different species of<br><br>
microorganisms is consequently possible. After appropriate image processing, a color code can be attributed to each of them which allows the different microorganisms or colonies of each sought species to be determined. This provides a particularly good image when it is performed on a membrane as a filter medium.<br>
It is also possible to mark the probes or the ligands used in step c) with very fine particles known as "quantum dots" and marketed under the name Qdots®. These are inorganic fluorophores composed of nanocrystals the fluorescence of which is much more stable over time. Their use is particularly suited to the detection of less abundant targets requiring a longer time for collection of the data by an image analyzer. The "quantum dots" are also compatible with a multi-coloured marking; they therefore allow the simultaneous detection of different species of microorganisms.<br>
Within the scope of the present invention, when using a membrane as a filter, it is preferable to use dyes which have a fluorescence in the near infrared, such as IRdyes marketed by LiCor® under the names IRdye 700 and IRDye 800 or marketed by Dyomics® for example under the names DY-700, DY-730, DY-750 and DY-780, in order to avoid the fluorescence due to the synthetic materials which compose the membrane. It was in fact observed that the membrane generally emitted less fluorescence in the near infrared.<br>
The method according to the invention aims at detecting in a gas, a liquid or on a surface fewer than 1000 microorganisms of the same species, preferentially fewer than 50 microorganisms of the same species, more preferentially, lower than 5 microorganisms from the same species and down to just one microorganism of a given species.<br>
In order to avoid the displacement of the microorganisms or their nucleic acids on the surface of the filter during one of the steps of the method described above, especially when the filter is a membrane, it is advantageous to use means which limit the exchanges, which can operate via the reaction solutions, between the different zones on said surface<br><br>
One means of limiting these exchanges consists for example of putting the reaction solution in contact with the microorganisms or nucleic acids via a pad impregnated with the reaction solution. The impregnated pad, when it is plated, for instance, on the surface of a membrane, delivers locally a volume of reaction solution sufficient for the reaction to be able to take place, without overflowing towards the adjacent zones.<br>
Another means of limiting the exchanges can consist of formulating the reaction solution in the form of a gel, preferably an acrylamide gel.<br>
Another means of limiting the exchanges according to the invention consists of a vertical partitioning positioned on the filter delimiting different zones of the latter. Advantageously, this partitioning is designed to provide a certain volume on the surface of a membrane so as to accommodate the reaction solution. Preferably, the partitioning is presented in the form of a grid forming a plurality of cells which is kept in contact with the filter.<br>
According to a preferred feature of the invention, the present method applies to the specific detection on a filter of one or more bacteria of the species Pseudomonas aeruginosa contained in a liquid, characterized in that it comprises the following steps:<br>
a)	the liquid is filtered through a filter, preferably a membrane;<br>
b)	the bacteria contained in the liquid are trapped on said filter;<br>
c)	the nucleic acids of Pseudomonas aeruginosa contained in these<br>
bacteria are specifically amplified in isothermal manner on the surface or<br>
inside the filter using a LAMP-type amplification requiring, in particular, at least<br>
one primer comprising a sequence taken from SEQ ID N°2, SEQ ID N°3, SEQ<br>
ID N°4, SEQ ID N°5, SEQ ID N°6, and SEQ ID N°7;<br>
d)	the amplification products obtained on the filter are detected.<br>
According to a particularly preferred feature of the invention, the primers used for the amplification of the nucleic acids of Pseudomonas aeruginosa by the LAMP-type technique are as follows: - F3      : CCTCCAAGGGCGGCGATGC<br><br>
 (SEQ ID No.2)<br>
-	B3     : TGCCTTTCAGGTCTTTCGCGTGT<br>
(SEQ ID NO.3)<br>
-	FIP    : CTGCCGTCAACGGCACCGCTTTTCCGGTGAAGGTGCCAATGGC<br>
(SEQ ID N0.4)<br>
-	BIP    : GCGTGCGATCACCACCTTCTACTTTTCAGAGCGCGCATGGCTTCC<br>
(SEQ ID No.5)<br>
-	LOOP F: CCTGCGTTCGGGTCGACGC<br>
(SEQ ID N0.6)<br>
-	LOOP B: CGAGTACGACAGCTCCGACC<br>
(SEQ ID No.7)<br>
These primers allow the amplification of the region of Pseudomonas aeruginosa represented by SEQ ID No.1 (region of the oprL gene).<br>
The messenger RNAs contained in Pseudomonas aeruginosa are preferably retro-transcribed in Step b) in order to obtain a cDNA comprising the sequence SEQ ID N°1, which is then amplified according to the LAMP-type amplification technique.<br>
Another feature of the invention relates to a device for the amplification of the nucleic acids, suitable for implementing the method according to the invention described previously. This device, which contains a filter (1) for receiving the nucleic acids to be amplified is characterized in that it contains a body (2) delimiting an internal volume and provided with means for keeping the filter (1) in contact, in this internal volume, with a flat impermeable wall (3) and with a grid (4) forming a plurality of cells on the surface of the filter<br>
(D-<br>
Preferably, the cells, which can be of variable size and geometry,<br>
are designed such that they make it possible to contain a volume, generally comprised between 1 and 100 \j\, preferentially between 1 and 50 yl and more preferentially between 3 and 10 jil, of a reaction solution.<br>
The body (2) of the device can have a shoulder (5) suitable for cooperating with the periphery of the filter (1), such that the latter can be fixed by its periphery against said shoulder (5).<br><br>
The flat wall (3) can also serve to secure the periphery of the porous filter (1) against said shoulder (5). It can be detachable vis-a-vis the body and, moreover, have a skirt (6) extending transversely. In this case, the body (2) advantageously has a sleeve (7) intended to cooperate with the skirt (6), which can have a shape opening out away from the flat wall (3).<br>
The grid (4) can be independent of the body or in a single piece with the body (2). When it is independent of the body, it can be recessed into the body, such that the partitions of the cetls are uniformly in contact with the surface of the filter. The cells are preferably of equal size and regular square or parallelepiped shape in order to be able to be filled with an equal volume of a reaction solution. When the grid and the body form just one piece, the filter can be then advantageously sealed on the grid in order to avoid cross-contaminations between cells.<br>
According to another feature of the invention, the device has a reservoir (9) attached to the body (2). This reservoir is intended to be filled with a liquid that is filtered by passage through the filter (1) (which is then a filtering filter), for example with the help of a vacuum pump which is fitted to the sleeve (7) in place of the flat wall (3). The microorganisms contained in the liquid are then trapped on the admission or upstream side of the filter. Once this operation has finished, the flat wall (3) is replaced in hermetic manner against the lower surface or downstream face of the filter.<br>
To avoid contamination of the liquid before filtration, it is envisaged that the reservoir can be provided with a detachable cover (8).<br>
The reservoir (9) can advantageously be attached to the body (2) by a frangible zone (10) such that once the filtration operation is finished, the reservoir can be detached by exerting from top to bottom a strong pressure on the upper horizontal edge of the reservoir.<br>
According to a preferred feature of the invention, the detachable cover (8) is suitable for alternately closing the reservoir (9) and said internal volume at the level of the frangible zone (10).<br>
Once the cover has closed on the frangible zone, the detachable cover (8) is then located opposite the grid (4) to close said internal volume, as<br><br>
is shown in Figures 13 and 14. The device in this shape allows the filter to be confined in a volume delimited by the flat wall (3), the body (2) and the cover (8), and vertically partitioned by the grid (4). This volume is intended to receive the reaction solutions for the amplification and optionally other steps of the method, such as the purification, fixing, and washing steps mentioned above. This volume is preferably sealed so as not to lose solution.<br>
The confinement of the filter in a reduced reaction volume is important to be able to realize the amplification reactions in satisfactory conditions. In fact, on the one hand contamination originating from the external environment must be avoided, and on the other hand the enzymatic reactions must not take place in too large volumes. This is necessary because the enzymatic reagents are expensive and the temperature must be regulated in uniform manner. Finally, too great an evaporation of the reaction solutions must be avoided.<br>
The deposition of said solutions is preferably carried out by temporarily removing the cover, applying the solution(s) over the grid, using a micropipette or a vaporizer. The volume of the solution can exceed the height of the grid or not.<br>
According to a preferred aspect of the device according to the invention, the flat wall (3) has a drainage orifice (11), and if appropriate a stopper (12) suitable for closing the drainage orifice (11). It is then possible, after the amplification or marking step according to the method of the invention, for example, to carefully remove the solution without the risk of excessively putting into suspension the nucleic acids located on the filter. Once the amplification and marking steps have been carried out, it is possible to retain only the body of the device provided with the filter, or, if appropriate, the filter alone, in order to analyze the latter according to the chosen technology, such as for example analysis by bioluminescence using an image analyzer.<br>
Another feature  of the  invention  relates  to   a  kit for the implementation of the detection method according to the invention. This kit comprises at least:<br><br>
- a filter, preferably a membrane;<br>
-one or more specific primers allowing amplification of the nucleic acids in isothermal manner.<br>
It may also comprise an amplification solution comprising a polymerase enzyme, a lysis solution, a fixing solution containing preferably glutaraldehyde, formaldehyde or paraformaldehyde, a blocking solution to stop the reactions and a detection solution comprising one of the detecting agents referred to in the present application.<br>
By "specific primers allowing amplification of the nucleic acids in isothermal manner" is meant molecules capable of hybridizing and initiating a polymerization reaction in the presence of a DNA polymerase, such as for example the FIP and BIP oligonucleotides in the case of a LAMP-type amplification, or in the presence of an RNA polymerase as in the case of an NASBA-type amplification.<br>
A kit according to the invention intended more particularly for the detection of Pseudomonas aeruginosa bacteria, comprises at least one oligonucleotide comprising a DNA sequence chosen from SEQ ID No.2, SEQ ID No.3, SEQ ID No.4, SEQ ID No.5, SEQ ID No.6 and SEQ ID No.7.<br>
Another feature of the invention relates to an oligonucieotide allowing the specific detection of Pseudomonas aeruginosa by amplification of the nucleic acids contained in this bacterium, characterized in that its nucleotide sequence comprises a sequence chosen from SEQ ID No.2, SEQ ID No.3, SEQ ID No.4, SEQ ID No.5, SEQ ID No.6 and SEQ ID No.7.<br>
The examples below aim to illustrate the invention in a more detailed manner.<br><br>
Example 1<br>
A test illustrated in Figure 4 of the present Application consisted of amplifying according to the LAMP technique illustrated in Figure 1 the messenger RNAs of only 50 Pseudomonas aeruginosa cells. The primers used for this purpose are those described in Figure 2 of the present Application.<br>
In order to be able to be precise about the number of cells treated, the nucleic acids were extracted from the bacteria and the amplification carried out in 25pl of the solution (2 pmol/ul of FIP &amp; BIP oligonucleotides; 1 pmoi/ul of loop F &amp; loop B oligonucleotides; 0.2 pmol/yl of F3 &amp; B3 oiigonucleotides in 40 mM Tris HCl pH 8.8, 20 mM KCI, 0.6 M betaine, 1.5 mM each of dNTP + 0.2 mM UTP biotin, 20 mM (NH4)2SO4, 8 mM MgSO4, 0.1% Triton® X100 surfactant, 0.005 U/ul AMV RTase, 0.32 U/ul B. st DNA polymerase) in a tube left in a water bath at 63 °C for 3 hours.<br>
The amplification was carried out under the following conditions:<br>
-	tube No. 1: treatment of 50 P. aeruginosa cells;<br>
-	tube No. 2: control without cells;<br>
-	tube No, 3: control without cells + dUTP biotin;<br>
-	tube No. 4: treatment of 50 cells + dUTP biotin<br>
The amplification products were observed on agarose get to ascertain the presence of amplification products in reaction mixtures 1 and 4 and the absence of amplification in mixtures 2 and 3.<br>
The products of the 4 reactions were applied to a nylon membrane.<br>
The membrane was then incubated with a blocking solution (Pierce blocking buffer + 1% SDS) for 45 minutes, then put in contact with a solution of streptavidin conjugated to Raifort peroxidase (1 mg/mi), which was diluted 1:750 in the blocking solution; after 45 minutes of incubation, the membrane was rinsed using a washing solution (PBS with Tween® surfactant, 0.2 % v/v) for 30 minutes. A luminol-based substrate, marketed by Pierce Chemical, of black radish peroxidase was sprayed onto the surface of the membrane in order to allow the luminous reaction to occur.<br><br>
Point C/ of Figure 4 shows the image obtained after processing of the image by an analyzer (CCD camera). The reaction obtained between the amplification products labelled by the biotin and the peroxidase can easily be seen for reaction mixture No.4.<br>
This experiment shows that amplification products obtained from a quantity of nucleic acids equivalent to 50 cells of a microorganism such as Pseudomonas aeruginosa can be easily detected on membrane.<br>
Example2<br>
Another test illustrated in Figure 4 consisted of applying -130 Pseudomonas aeruginosa cells to a membrane (MXHVWP124),<br>
After this application, the membrane was placed on a cellulose pad impregnated with 1.5 ml of the fixation solution (0.25% glutaraldehyde in 95% of denatured alcohol) for 5 minutes at ambient temperature then dried for 5 minutes at ambient temperature.<br>
The membrane was placed in a hermetic Petri dish then 0.5 ml of the amplification solution (2 pmol/ul of FIP &amp; BIP oligonucleotides; 1 pmol/ul of loop F &amp; loop B oligonucleotides; 0.2 pmol/(j| of F3 &amp; B3 oligonucleotides in 40 mM Tris HCI pH 8.8, 20 mM KCl, 0.6 M betaine, 1.5 mM each of dNTP + 0.2 mM UTP biotin, 20 mM (NH4)2SO4, 8 mM MgSO4, 0.1% Triton® X100 surfactant, 0.005 U/ul AMV RTase, 0.32 U/ul DNA polymerase) was added before incubation at 63°C for 150 minutes.<br>
The membrane was then incubated with a blocking solution (Pierce blocking buffer + 1% SDS) for 45 minutes, then put in contact with a solution of streptavidin conjugated to Raifort peroxidase (1 mg/ml), which was diluted 1:750 in the blocking solution; after 45 minutes of incubation, the membrane was rinsed using a washing solution (PBS with Tween® surfactant, 0.2 % v/v) for 30 minutes. A luminol-based substrate, marketed by Pierce, of black radish peroxidase was sprayed onto the surface of the membrane in order to allow the luminous reaction to occur. The signals were recorded using the Milliflex® Rapid system.<br><br>
The results are shown in parts B &amp; C of Figure 6. In part C of Figure 6, two amplification peaks corresponding to the initial applications of the cells to the membrane can easily be seen.<br>
In parallel, the solution, having served for amplification, was recovered and analyzed by agarose gel electrophoresis. The result of this is shown in part B of Figure 6. It can easily be seen that part of the amplification products is found in the amplification solution.<br>
This experiment shows that amplification products obtained from a quantity of nucleic acids equivalent to 130 cells of a microorganism such as Pseudomonas aeruginosa can be easily detected on membrane. Nevertheless, part of these amplification products is eluted by the membrane and can be detected by electrophoresis.<br>
Example 3<br>
Another test illustrated in Figure 6 can consist of filtering 100 ml of a solution using the device shown in Figure 5.<br>
After filtration, the device is placed on a cellulose pad impregnated with 1.5 ml of the fixation solution (0.25% glutaraldehyde in 95% of denatured alcohol) for 5 minutes at ambient temperature.<br>
The membrane is dried for 5 minutes at ambient temperature, then the lower part of the device is closed using the closure cassette in order to avoid leaks during incubation.<br>
1.5 ml of the amplification solution (2 pmol/µl of FIP &amp; BIP oligonucleotides; 1 pmol/µl of loop F &amp; loop B oligonucleotides; 0.2 pmol/µl of F3 &amp; B3 oligonucleotides in 40 mM Tris HCI pH 8.8, 20 mM KCI, 0.6 M betaine, 1.5 mM each of dNTP + 0.2 mM UTP biotin, 20 mM (NH4)2SO4, 8 mM MgSO4, 0.1% Triton® X100 surfactant, 0.005 U/µl AMV RTase, 0.32 U/µl DNA polymerase) is distributed over the membrane of the device then its upper part is closed again with the cover in order to avoid evaporation during incubation at 63°C for 150 minutes.<br>
The membrane is then incubated with a blocking solution for 45 minutes, then put in contact with a streptavidin solution conjugated to Raifort<br><br>
peroxidase (1 mg/ml), which is diluted 1:750 in the blocking solution; After 45 minutes of incubation, the membrane was rinsed using a solution of PBS with Tween® surfactant.0.2 % v/v for 30 minutes. A luminol-based substrate of black radish peroxidase, marketed by Pierce, is sprayed onto the surface of the membrane in order to allow the luminous reaction to occur. The signals are recorded using the Milliflex® Rapid system.<br><br><br><br><br>
We Claim:<br>
1.	Method for the specific detection on a membrane of one or more microorganisms present<br>
in a liquid, a gas or on a surface, characterized in that it comprises the following steps:<br>
(i) the liquid, the gas or the surface is put in contact with the membrane, having a pore<br>
diameter of between 0.1  and 1.5 microns so as to trap the microorganism or<br>
microorganisms on the membrane; (ii) said microorganisms are treated on said membrane by contacting the membrane with<br>
a   fixation   solution   containing   a   cross-linker   selected   from   glutaraldehyde,<br>
formaldehyde and/or paraformaldehyde; (iii)nucleic acids present in the microorganism or microorganisms are specifically<br>
isothermally amplified, at a temperature of between 35 and 90°C, on said membrane<br>
in order to obtain amplification products; (iv)the obtained amplification products are detected, using standard detection methods,<br>
directly on said membrane where the microorganisms have been trapped.<br>
2.	Method as claimed in claim 1, wherein the microorganism or microorganisms, or their nucleic acids, are fixed on a PVDF or polyamide-based membrane.<br>
3.	Method as claimed in claim 1 or 2, wherein said method allows individual counting of the microorganisms trapped on said membrane.<br>
4.	Method as claimed in any one of claims 1 to 3, wherein in step i) the microorganism or microorganisms are trapped by passing a liquid or a gas through the membrane.<br>
5.	Method as claimed in any one of claims 1 to 3, wherein in step a) the microorganism or microorganisms present in a gas or on a surface are trapped by impaction of the microorganisms on the membrane.<br>
6.	Method as claimed in any one of claims 1 or 5, wherein the microorganism or microorganisms to be detected are contained in an aqueous medium.<br>
7.	Method as claimed in any one of claims 1 to 6, wherein the microorganism or microorganisms to be detected are contained in air.<br>
8.	Method as claimed in any one of claims 1 to 7, wherein the nucleic acids amplified in step iii) consist of DNAs.<br>
9.	Method as claimed in claim 8, wherein the nucleic acids amplified in step iii) consist of cDNAs obtained by a specific additional reverse transcription step carried out starting from the RNAs contained in the microorganism or microorganisms.<br>
10.	Method as claimed in claim 9, wherein the RNAs starting from which the reverse transcription step is carried out are the messenger RNAs contained in the microorganism or microorganisms.<br>
11.	Method as claimed in any one of claims 8 to 10, wherein the nucleic acid, or the cDNA obtained by reverse transcription, are amplified in step iii) by a LAMP-type amplification technique.<br>
12.	Method as claimed in claim 11, wherein the LAMP-type amplification technique comprises a phase in which loop F and loop B primers are hybridized at the level of the loops present in the amplification products, in order to increase the speed of formation of said amplification products.<br>
13.	Method as claimed in any one of claims 1 to 12, wherein the amplification products which are detected in step iv) consist of DNAs.<br>
14.	Method as claimed in any one of claim 8 to 10, wherein the nucleic acids, or the cDNA obtained by reverse transcription, are amplified in step iv) by NASBA (nucleic acid sequence based amplification), or TMA (transcription mediated amplification) technique.<br>
15.	Method as claimed in any one of claims 1 to 14, wherein a marker is incorporated into the amplification products obtained in step iii) allowing their detection in step iv).<br>
16.	Method as claimed in claim 15, wherein the marker incorporated into the amplification products is a marked purine or pyrimidine base.<br><br>
17.	Method as claimed in claim 15, wherein the marker incorporated into the amplification products consists of primers which are marked.<br>
18.	Method as claimed in claim 17, wherein the marked primers, incorporated into the amplification products, consist of loop F and/or loop B primers as defined in claim 12.<br>
19.	Method as claimed in any one of the claims 15 to 18, wherein the marker incorporated into the amplification products is coupled with a ligand which is then reacted with a marked molecule capable of interacting with this ligand.<br>
20.	Method as claimed in claim 19, wherein the ligand is a biotin.<br>
21.	Method as claimed in claim 20, wherein in step iv) the amplification products comprising the biotin are put in contact with the streptavidin coupled to a molecule allowing it to be detected.<br>
22.	Method as claimed in claim 19, wherein the ligand consists of an antigen or an antibody which is then reacted with the marked antibodies or antigens.<br>
23.	Method as claimed in claim 22, wherein the marker incorporated into the amplification products in step iii) is coupled to a dioxygenin.<br>
24.	Method as claimed in claim 23, wherein in step iv) the amplification products comprising the dioxygenin are put in contact with an antidioxygenin antibody coupled to a molecule allowing it to be detected.<br>
25.	Method as claimed in one of claims 19 and 24, wherein the marked molecule capable of interacting with the ligand is conjugated with black radish peroxidase or the soya seed peroxidase.<br>
26.	Method as claimed in one of claims 19 and 24, wherein the marked molecule capable of interacting with this ligand is conjugated with alkaline phosphatase.<br>
27.	Method as claimed in any one of claims 15 to 26, wherein the marker incorporated into the amplification products allows detection by fluorescence.<br>
28.	Method as claimed in claim 27, wherein the molecules allowing the detection by fluorescence of the amplification products have an emission spectrum in the near infrared.<br>
29.	Method as claimed in any one of claims 1 to 10, wherein in step iv) the amplification products obtained in step iii) are hybridized with marked specific hybridization probes.<br>
30.	Method as claimed in claim 29, in which the hybridization probes are marked using a fluorescent molecule, a ligand, an antibody or an antigen as defined in any one of claims 19 to 29.<br>
31.	Method as claimed in one of claims 29 and 30, wherein the marked specific hybridization probe is of PNA type.<br>
32.	Method as claimed in any one of claims 1 to 31, wherein the membrane used has pores of an average diameter comprised between 0.1 and 1.5 microns, preferably between 0.15 and 0.8 microns, more preferentially between 0.2 and 0.6 microns.<br>
33.	Method as claimed in any one of claims 1 to 32, wherein the membrane is constituted by one or more materials chosen from polytetrafluoroethylene, poly(vinylidene) fluoride, polycarbonate, polyamide, polysulphone, polyethersulfone, acetyl cellulose, mixed cellulosic esters and nitrocellulose.<br>
34.	Method as claimed in any one of claims 1 to 33, wherein it additionally comprises the step of culturing the microorganism or microorganisms trapped on the membrane in step ii) by putting said membrane in contact with a nutritive medium.<br>
35.	Method as claimed in any one of claims 1 to 34, wherein it additionally comprises the step of lysis of the wall of the microorganism or microorganisms trapped on the membrane in order to release the nucleic acids contained in the microorganism or microorganisms.<br>
36.	Method as claimed in any one of claims 1 to 35, wherein it additionally comprises the step of detecting the presence of the living microorganism or microorganisms by ATP bioluminescence.<br>
37.	Method as claimed in any one of claims 1 to 36, wherein several types of microorganisms originating from the same fluid can be detected in parallel in a specific manner on the same membrane.<br>
38.	Method as claimed in any one of claims 1 to 37, wherein the microorganism or microorganisms to be detected are chosen from bacteria, viruses, protozoa, mycoplasmas, molds and yeasts.<br>
39.	Method as claimed in any one of claims 1 to 38, wherein the microorganism or microorganisms to be detected are present in a liquid which is water.<br>
40.	Method as claimed in any one of claims 1 to 39, wherein the microorganism is the Pseudomonas aeruginosa bacteria.<br>
41.	Method for the specific detection on a membrane of one or more bacteria of the species Pseudomonas aeruginosa contained in a liquid, as claimed in claim 1 wherein it comprises the following steps:<br>
(i) the liquid is passed through a membrane;<br>
(ii) the bacteria contained in the liquid are trapped on the membrane;<br>
(iii)said microorganisms are treated on said membrane with a fixation solution containing a cross-linker selected from glutaraldehyde, formaldehyde and/or paraformaldehyde;<br>
(iv)nucleic acids of Pseudomonas aeruginosa contained in these bacteria are specifically isothermally amplified on said membrane with the help of a LAMP-type amplification by using at least one primer comprising a sequence taken from SEQ ID No.2, SEQ ID No.3, SEQ ID No.4, SEQ ID No.5, SEQ ID No.6, and SEQ ID No.7;<br>
(v) the amplification products obtained on the membrane are detected.<br>
42.	Method as claimed in claim 41, wherein a reverse transcription step on the RNAs<br>
contained in the bacteria in order to obtain a cDNA comprising the sequence SEQ ID<br>
No.l, the LAMP-type amplification being carried out starting from said cDNA.<br>
43.	Method as claimed in any one of claims 1 to 42, wherein the membrane is a PVDF membrane.<br>
44.	A method for the specific detection on a membrane of one or more microorganisms as claimed in any of the preceding claims and when used as a kit comprising:<br><br>
-	a PVDF membrane ;<br>
-	a fixing solution containing glutaraldehyde, formaldehyde or paraformaldehyde; and<br>
-	at least one specific primer allowing isothermal amplification of the nucleic acids from the microorganisms.<br><br>
45.	Device for the amplification of nucleic acids suitable for implementing the method as claimed in claims 1 to 41, comprising a filter (1) for receiving the nucleic acids to be amplified, this device being wherein it contains a body (2) delimiting an internal volume, a flat impermeable wall (3) and a grid (4) forming a plurality of cells on the surface of the filter (1), the body (2) being provided with means for keeping the filter (1) in contact, in this internal volume, with said flat impermeable wall (3).<br>
46.	Device as claimed in claim 45, wherein said cells make it possible to contain a volume comprised between 1 and 100 µl, preferably between 3 and 10 µl, of a reaction solution.<br>
47.	Device as claimed in one of claims 45 and 46, wherein the body (2) has a shoulder (5) suitable for cooperating with the periphery of the filter (1).<br>
48.	Device as claimed in claim 47, wherein the filter (1) is fixed by its periphery against said shoulder (5).<br>
49.	Device as claimed in any one of claims 47 and 48, wherein the flat wall (3) secures the periphery of the filter (1) against said shoulder (5).<br>
50.	Device as claimed in any one of claims 45 to 49, wherein the grid (4) is made of a single piece with the body (2).<br><br>
51.	Device as claimed in any one of claims 45 to 50, wherein the flat wall (3) is detachable vis-a-vis the body (2).<br>
52.	Device as claimed in claim 51, wherein the flat wall (3) has a skirt (6) extending transversely and in that the body (2) has a sleeve (7) intended to cooperate with the skirt (6).<br>
53.	Device as claimed in any -one of claims 45 to 52, wherein it also has a detachable cover (8) which is suitable for being arranged opposite the grid (4) to close said internal volume.<br>
54.	Device as claimed in any one of claims 45 to 53, wherein it has a reservoir (9) attached to the body (2).<br>
55.	Device as claimed in claim 54, wherein the reservoir (9) is attached to the body (2) by a frangible zone (10).<br>
56.	Device as claimed in claim 54, wherein it can also have a detachable cover suitable for alternately closing the reservoir (9) and said internal volume at the level of the frangible zone (10).<br>
57.	Device as claimed in any one of claims 45 to 56, wherein the flat wall (3) has a drainage orifice (11).<br>
58.	Device as claimed in any one of claims 45 to 57, wherein the filter (1) is a membrane.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUFic3RyYWN0LSgxNy0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Abstract-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUFic3RyYWN0LSgyMC0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Abstract-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUNsYWltcy0oMTctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Claims-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUNsYWltcy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Claims-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-Correspondence Others-(03-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Correspondence Others-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Correspondence Others-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMzEtMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-Correspondence-Others-(31-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE3LTA2LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Description (Complete)-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LURyYXdpbmdzLSgxNy0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Drawings-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUZvcm0tMy0oMjAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Form-3-(20-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LUZvcm0tMy0oMzEtMDEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-Form-3-(31-01-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LUdQQS0oMTctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-GPA-(17-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">8182-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=ODE4Mi1ERUxOUC0yMDA3LVBldGl0aW9uLTEzNy0oMTctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">8182-DELNP-2007-Petition-137-(17-06-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251013-an-ifnar2-mutant-polypeptide-dna-encoding-the-same-and-vector-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251015-an-isolated-polynucleotide-and-vector-comprising-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251014</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>8182/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>24-Feb-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>16-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-Oct-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MILLIPORE CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>290 CONCORD ROAD, BILLERICA, MA 01821, USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FREDERIC MARC</td>
											<td>61, ROUTE DU VIN, F-67140 ITTERSWILLER, FRANCE.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SERGE OHRESSER</td>
											<td>9 RUE, DES GERANIUMS, F-67190 STILL, FRANCE.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/04</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2006/001230</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-04-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0504379</td>
									<td>2005-04-29</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251014-method-for-the-detection-and-characterization-of-microorganisms-on-a-filter by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:07:18 GMT -->
</html>
