<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251200-recombinant-poxvirus-expressing-homologous-genes-inserted-into-the-poxviral-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:01:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251200:&quot;RECOMBINANT POXVIRUS EXPRESSING HOMOLOGOUS GENES INSERTED INTO THE POXVIRAL GENOME&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;RECOMBINANT POXVIRUS EXPRESSING HOMOLOGOUS GENES INSERTED INTO THE POXVIRAL GENOME&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a recombinant poxvirus vector capable of expressing two or more homologous, foreign sequences, which derive from different variants of a microorganism, and which have a homology of 50% or above. The invention further relates to a method for preparing such recombinant poxvirus and the use of such recombinant poxvirus as medicament or vaccine. Additionally, a method for affecting, preferably inducing, an immune response in a living animal, including a human, is provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a recombinant poxvirus capable of expressing two or more homologous foreign genes. Said genes are heterologous to the viral genome, but homologous in comparison to each other. The genes are especially derived from closely related variants or subtypes of a microorganism. The invention further relates to a method for preparing such recombinant poxvirus and to the use of such recombinant poxvirus as medicament or vaccine. Additionally, a method for affecting, preferably inducing, an immune response in a living animal, including a human, is provided.<br>
Background of the invention<br>
Every living organism is constantly challenged by infectious or pathogenous agents such as bacteria, viruses, fungi or parasites. The so-called immune system prevents the organism from permanent infections, diseases or intoxination caused by such agents.<br>
The immune system of a mammal can be divided into a specific and an unspecific part although both parts are closely cross-linked. The unspecific immune response enables an immediate defense against a wide variety of pathogenic substances or infectious agents. The specific immune response is raised after a lag phase, when the organism is challenged with a substance for the first time. This specific immune response is mainly based on the production of antigen-specific antibodies and the generation of macrophages and lymphocytes, e.g. cytotoxic T-cells (CTL). The specific immune response is responsible for the fact that an individual who recovers from a specific infection is protected against this specific infection but still is susceptible for other infectious diseases. In general, a second infection with the same or a very similar infectious agent causes much milder symptoms or no symptoms at all. This so-called immunity persists for a long time, in some cases even lifelong. The underlying effect is often referred to as immunological memory, which can be used for vaccination proposes.<br>
With the term vaccination a method is described, where an individual is challenged with a harmless, partial or inactivated form of the infectious agent to affect,<br>
preferably induce, an immunological response in said individual, which leads to long lasting - if not lifelong - immunity against the specific infectious agent.<br>
The human smallpox disease is caused by Variola virus. Variola virus belongs to the family of Poxviridae, a large family of complex DNA viruses that replicate in the cytoplasma of vertebrate and invertebrate cells.<br>
The family of Poxviridae can be divided into the two subfamilies Chordopoxvirinae and Entomopoxvirinae based on vertebrate and insect host range. The Chordopoxvirinae comprise beside others the genera of Orthopoxviruses and Avipoxviruses (Fields Virology, ed. by Fields B.N., Lippincott-Raven Publishers, 3rd edition 1996, ISBN: 0-7817-0253-4, Chapter 83).<br>
The genera of Orthopoxviruses comprises variola virus, the causative agent of human smallpox, and also other viruses with economical importance, e.g. camelpox, cowpox, sheeppox, goatpox, monkeypox and Vaccinia virus. All members of this genus are genetically related and have similar morphology or host range. Restriction endonuclease maps have even shown high sequence identity from up to 90% between different members of the Orthopoxviruses (Mackett &amp; Archard, [1979], J Gen Virol, 45: 683-701).<br>
Vaccinia virus (VV) is the name given to the agent that was used at least the last 100 years for the vaccination against smallpox. It is not known whether VV is a new species derived from cowpox or variola virus by prolonged serial passages, the living representative of a now extinct virus or maybe a product of genetic recombination. Additionally, in course of the VV history, many strains of Vaccinia have arisen. These different strains demonstrate varying immunogenicity and are implicated to varying degrees with potential complications, the most serious of which is post-vaccinial encephalitis. However, many of these strains were used for the vaccination against smallpox. For example the strains NYCBOH, Western Reserve or Wyeth were used primarily in US, while the strain Ankara, Bern, Copenhagen, Lister and MVA were used for vaccination in Europe. As a result of the worldwide vaccination program with these different strains of VV in 1980 the WHO finally declared the successful eradication of variola virus.<br>
Today, VV is mainly used as a laboratory strain, but beside this it is still considered as the prototype of Orthopoxviruses, which is also the reason why VV became one of the most intensively characterized viruses (Fields Virology, ed. by Fields B.N., Lippincott-Raven Publishers, 3rd edition 1996, ISBN: 0-7817-0253-4, Chapter 83 and 84).<br>
VV and more recently other poxviruses have been used for the insertion and expression of foreign genes. The basic technique for inserting foreign genes into live infectious poxvirus involves recombination between pox DNA sequences flanking a foreign genetic element in a donor plasmid and homologous sequences present in the rescuing poxvirus. Genetic recombination is, in general, the exchange of homologous sections of DNA between two strands of DNA. In certain viruses RNA may replace DNA. Homologous sections of nucleic acid are sections of nucleic acid (DNA or RNA), which have the same sequence of nucleotide bases. Genetic recombination may take place naturally during the replication or manufacture of new viral genomes within an infected host cell. Thus, genetic recombination between viral genes may occur during the viral replication cycle that takes place in a host cell, which is co-infected with two or more different viruses or other genetic constructs. A section of DNA from a first genome is used interchangeably in constructing the section of the genome of a second co-infecting virus in which the DNA is homologous with that of the first viral genome.<br>
Successful expression of the inserted DNA genetic sequence by the modified infectious virus requires two conditions. First, the insertion should be into a nonessential region of the virus in order that the modified virus remains viable. The second condition for expression of inserted DNA is the presence of a promoter in the proper relationship to the inserted DNA. Regularly, the promoter is located upstream from the DNA sequence to be expressed.<br>
The usefulness of recombinant VV expressing, e.g., Hepatitis B virus surface antigen (HBsAg), Influenza virus hemagglutinin (InfHA) or Plasmodium knowlesi sporozoite antigen, as live vaccines for the prophylaxis of infectious diseases has been demonstrated and reviewed (Smith, et al. [1984] Biotechnology and Genetic Engineering Reviews 2, 383-407).<br>
A further advantage of VV is the capacity to take up multiple foreign sequences, genes or antigens within a single VV genome (Smith &amp; Moss [1983], Gene, 25(1): 21-28). Furthermore, it has been reported that it is possible to elicit immunity to a number of heterologous infectious diseases with a single inoculation of a polyvalent vaccine (Perkus et al, [1985], Science, Vol. 229, 981-984).<br>
One example of the expression of various antigens by a single VV is described by Bray et al. It was shown that a recombinant VV, which is capable to express three different structural proteins of Dengue virus serotype 4, namely the capsid (C), pre-membrane (prM), envelope (E) protein, and two non-structural proteins of Dengue virus serotype 4, namely NSl and NS2a, had the ability to protect mice against a homologous Dengue virus serotype 4 challenge (Bray et al, [1989], Virology 2853-2856).<br>
The Dengue virus with its four serotypes, Dengue virus serotype 1 (Den-1) through Dengue virus serotype 4 (Den-4), is one important member of the Flavivirus genus with respect to infections of humans. Dengue virus infection produces diseases that range from flu-like symptoms to severe or fatal illness, Dengue haemorrhagic fever (DHF) with shock syndrome (DSS). Dengue outbreaks continue to be a major public health problem in densely populated areas of the tropical and subtropical regions, where mosquito vectors are abundant.<br>
The concern over the spread of Dengue infection and other diseases induced by mosquito-borne Flaviviruses in many parts of the world has resulted in more efforts being made towards the development of Dengue vaccines, which could prevent both Dengue fever (DF), and Dengue haemorrhagic fever (DHF) and in vaccines useful to protect the vaccinated individual against infections induced by some or all mosquito-borne flaviviruses.<br>
While most cases of DF are manifested after the first infection by any of the four serotypes, a large percentage of DHF cases occur in subjects who are infected for the second time by a serotype, which is different from the first infecting serotype of Dengue virus. These observations give rise to the hypothesis that sequential infection of an individual having antibodies against one Dengue serotype by a<br>
different virus serotype at an appropriate interval may result in DHF in a certain number of cases.<br>
Accordingly, vaccination against one serotype does not result in a complete protection against Dengue virus infection, but only against infection with the same Dengue virus strain. Even more important, a person vaccinated against one serotype, has an increased risk of developing severe complications such as Dengue haemorrhagic fever when said person is infected from a Dengue virus strain of a different serotype.<br>
Thus, a multivalent vaccine that contains antigens from all four Dengue virus serotypes is desired.<br>
So far it had been suggested to prepare multivalent vaccines by mixing a panel of recombinant VV, each VV encoding sequences of a different viruses (Moss, [1990] Immunology, 2, 317-327). However, such a multivalent vaccine comprises several disadvantages. Firstly, it is cumbersome to generate several independent recombinant VV. Beside the separated production processes, also quality control and quality assurance is highly time consuming. Secondly, an infection with a mixture of recombinant viruses expressing different sequences always bears the risk that the infection event is not particularly well balanced. The main risk is that only individual recombinants, but not all different recombinants comprised in the multivalent vaccine will infect target cells. One reason might be an uneven distribution of recombinant viruses. Another reason might be interferences between the different recombinant viruses while infecting single cells. Such interferences are known as phenomenon of superinfection. In this case, only some antigens, but not all different antigens of the multivalent vaccine will finally be expressed from infected cells and, thus, presented to the immune system of a patient. As a consequence, immune protection will be obtained only against some of the antigens, but is far from providing a complete immune protection against the various antigens presented or presentable by the multivalent vaccine.<br>
In the context of a vaccine against Dengue virus infection the approach of a multivalent vaccine has the disadvantage that if the  different  sequences are<br>
expressed in different amounts or in an unpredictable manner, as it had been shown for the envelope protein of Dengue virus 2 (Deuble et ai, [1988], J. Virol 65: 2853), then such a vaccination is highly risky for a patient. An incomplete vaccination using a panel of recombinant Vaccinia viruses will only provide an immune protection against some, but not against all serotypes of Dengue virus. Unfortunately, in case of Dengue infection an incomplete vaccination is extremely unacceptable, since it increases the risk of lethal complications such as Dengue hemorrhagic fever.<br>
Object of the Invention<br>
It is, therefore, an object of the present invention to provide a stable, effective and reliable vaccine against infectious diseases, which can be caused by more than one strain, clade, variant, subtype or serotype of said infectious disease causing microorganism.<br>
It is a further object of the present invention to provide a stable, effective and reliable vaccine against Dengue virus infections, which allows reliable vaccination against all Dengue virus serotypes.<br>
Detailed description of the Invention<br>
The present invention is based on the idea to include into a poxvirus homologous genes derived from different strains, clades, variants, subtypes or serotypes of an infectious disease causing microorganism. As already mentioned above, there are, for example, 4 groups, subtypes or serotypes of Dengue virus existing which are all comprising the same types of genes as, e.g., the gene encoding the capsid (C) protein, the gene encoding the pre-membrane (PrM) or envelope (E) protein. However, the nucleic acid sequence of the same type of gene is not completely identical and not perfectly homologous, respectively, in all 4 serotypes: For example, sequence comparison (with Lasergene 4.05 Magalign, Macintosh) between the PrM genes of Dengue virus serotype 1, 2, 3 and 4 (PrM 1-4) revealed a sequence identity of 66.5-72.9%, i.e., a homology of appr. 65-75%. It is assumed that differences and variations, respectively, in the genes of different subtypes of infectious disease causing microorganisms are the reasons why vaccination against<br>
one subtype does not automatically result in protection against infections with other variants of the same microorganism. It was, therefore, the idea to generate a recombinant virus including closely related or homologous genes derived from different strains, clades, variants, subtypes or serotypes of an infectious disease causing microorganism. However, as also already indicated above, homologous recombination between homologous sequences occur during the viral life cycle and even takes place between sections of DNA that are not perfectly homologous. It was, thus, expected that the insertion of homologous genes in a single viral genome would result in homologous recombination and, thus, in deletions of the inserted homologous genes.<br>
However, when generating a recombinant poxvirus comprising in its genome at least two foreign genes with a homology of at least 60%, it was unexpectedly found that said homologous genes remain stably inserted into the viral genome.<br>
Even if homologous genes, preferably with a homology of at least 50%, are inserted into different insertion sites of the viral genome, the genes also remain stably inserted into the viral genome: In this case, it was expected that recombination events between said homologous genes would additionally result in loss of viral genes important for amplification of the virus and for the viral life cycle, respectively, i.e., it was expected that the viral life cycle would seriously be damaged. Since, additionally, the frequency of recombination is proportional to the distance between two linked genes, it was expected that frequency of recombination events between two or more homologous genes located in different insertion sites would be high and, thus, result in deletions of said genes and/or in severe interferences. Accordingly, it was extremely surprising that no recombination events occurred, but that the homologous genes remained stably inserted into the different insertion sites of the viral genome.<br>
According to the prior art, a recombinant poxvirus containing foreign DNA from flavivirus, such as Japanese Encaphalitis Virus (JEV), Yellow Fever Virus (YFV) and Dengue Virus, are known (US Patent No. 5,514,375). However, each gene derived from said flaviviruses were inserted only a single time and into the same insertion site. Additionally, sequence comparison with a suitable computer software<br>
(Lasergene 4.05 Megalign, Macintosh) revealed a homology of the genes inserted into the poxviral genome and derived from JEV of 20.2%-29.6%, from YFV of 29.2%-45.3%, and from Dengue Virus of 22.8%-29.5%.<br>
Similar disclosure applies to Indian Patent No: 212567 describing insertion of Dengue virus antigens into the same insertion site of Modified Vaccinia Virus Ankara (MVA), especially into deletion site II.<br>
US Patent No. 5,338,683 discribes insertion of gp 13 and 14 of herpesvirus glycoprotein genes into two different insertion sites of a single recombinant poxvirus; however, both genes have a homology of 25.2% only.<br>
Sequence comparison between influenza virus haemagglutinin and nucleoprotein gene inserted into the same insertion site (deletion site III) of a Modified Vaccinia Virus Ankara (MVA) resulted in a homology of 49.1% (US Patent No. 5,676,950; Sutter et al, [1994], Vaccine 12: 1032).<br>
US Patent No. 5,891,442 discloses a recombinant poxvirus containing the coding sequence for the polyprotein VP2, VP3 and VP4 of infectious bursal disease. Said genes were fused and, thus, inserted into a single insertion site and have a homology of 41.9%-50.3%.<br>
Finally, US Patent No. 6,217,882 describes a recombinant swinepox virus vector containing pseudorabies antigens gp50 and gp63 with a homology of 52.7% inserted into the same insertion site.<br>
In summary, it can be stated that according to the prior art homologous genes or sequences having a homology of at least 50% are all inserted into the same or a single insertion site within the viral genome.<br>
According to the present invention, homologous genes or sequences have a homology of at least 50%, i.e., a homology of 50% - 100%, i.e., at least 50% identical nucleotide bases. Genes or sequences having a homology below 50% can be considered as being heterologous. In the context of the present invention, the term "homologous" or "homology" is used when genes or sequences are compared to each other, whereas the terms "foreign" gene, "exogenous" or "heterologous"<br>
sequence are used when genes or sequences are compared to the poxviral genome; i.e., said terms refer to a DNA sequence which is in nature not normally found associated with a poxvirus as used according to the invention. Accordingly, the present invention relates to a recombinant poxvirus comprising at least two genes which are heterologous in comparison to the viral genome, but which are homologous among each other. The term "genes" refers to coding sequences, which encode, e.g., proteins, polypeptides, peptides, antigens and the like. Proteins, polypeptides or peptides translated from homologous genes fulfill the same tasks and show the same functional properties. Homologous genes are regularly derived from different, but related sources or organisms. According to one embodiment of the present invention, the homology in the coding sequences is preferably 70% to 80%, more preferably 80% to 90% or 90% to 100%. Most preferred is a homology of 65% to 75%.<br>
Since the recombinant poxvirus according to the present invention comprises the relevant genetic information in one single infectious unit or in one virus particle only, there is no risk of uneven infection and unbalanced expression of the different homologous sequences. Thus, the recombinant poxvirus according to the present invention comprising and capable of expressing several closely or even closest related genes or almost identical sequences in one infected cell is particularly advantageous for the generation of multivalent vaccines.<br>
This advantage is particularly interesting for the development of vaccines against diseases, which can be caused by several closely related strains or serotypes of a virus, like e.g. Dengue virus. Recombinant poxviruses comprising homologous genes of different Dengue virus serotypes are described in the Examples.<br>
The homologous genes or sequences according to the present invention can be derived from any microorganism, such as any virus except the vector virus, any bacterium, any fungus or parasite. Preferably, the homologous genes or sequences are derived from an infectious or pathogenic microorganism and most preferably from different strains or clades, variants, subtypes or serotypes of said microorganism.<br>
The terms "strain" or "clade" are technical terms, well known to the practitioner, referring to the taxonomy of microorganisms. The taxonomic system classifies all so far characterised microorganisms into the hierarchic order of Families, Genera, species, strains (Fields Virology, ed. by Fields B.N., Lippincott-Raven Publishers, 4th edition 2001). While the criteria for the members of Family is their phylogenetic relationship, a Genera comprises all members which share common characteristics, and a species is defined as a polythetic class that constitutes a replicating lineage and occupies a particular ecological niche. The term "strain" or "clade" describes a microorganism, i.e. virus, which shares the common characteristics, like basic morphology or genome structure and organisation, but varies in biological properties, like host range, tissue tropism, geographic distribution, attenuation or pathogenicity. The term "variants" or "serotypes" further distinguishes between members of the same strain, also called subtypes, which show individual infection spectra or antigenic properties due to minor genomic variations.<br>
According to a further embodiment of the present invention the homologous genes or sequences are selected from viruses, preferably viruses, which belong to the genera of Flaviviruses, such as preferable - but not limited to - Dengue virus, West Nile virus or Japanese encephalitis virus; which belong to the genera of Retroviruses, such as preferable - but not limited to - Human Immunodeficiency Virus (HIV); which belong to the genera of Enteroviruses, such as preferable - but not limited to - Hand, Foot and Mouth disease, EV71; which belong to the genera of Rotaviruses or which belong to the genera of Orthomyxoviruses, such as preferable -but not limited to- Influenza virus. Most preferred are homologous genes derived from a Flavivirus.<br>
According to a further preferred embodiment, the homologous genes are selected from Dengue virus genes, preferably C, NS1 and/or NS2, or preferably E, more preferably PrM. Most preferred are homologous genes derived from different serotypes of the virus, wherein said genes may be derived from one, two, three or from all of the 4 Dengue virus serotypes.<br>
According to still a further embodiment the homologous genes are selected from different HIV stains or clades. Preferably the homologous genes are selected from<br>
the gag/pol coding sequence, more preferably from the env coding sequence or further preferably from a combination of structural and/or regulatory HIV coding sequences.<br>
The vector virus suitable for the present invention is selected from the group of poxviruses, which can be easily cultured in selected host cells as, e.g., avian host cells, but which are highly replication deficient or actually not replicating in humans or human cells.<br>
According to some preferred embodiments the poxvirus according to the present invention is selected from the group comprising canarypox viruses (Plotkin et al. [1995] Dev Biol Stand, vol 84: pp 165-170. Taylor et al. [1995] Vaccine, Vol 13. No. 6: pp 539-549), fowlpox viruses (Afonso et al. [2000] J Virol, pp 3815-3831. Fields Virology, ed. by Fields B.N., Lippincott-Raven Publishers, 4th edition 2001, Chapter 85: page 2916), penguin pox viruses (Stannard et al. [1998] J Gen Virol, 79, pp 1637-1649) or derivatives thereof. Since these viruses belong to the genera of Avipoxviruses they can be easily cultured and amplified in avian cells. However, in mammalian or human cells they are replication defective, which means that essentially no or nearly no infectious progeny viruses are produced.<br>
According to a further embodiment of the present invention a Vaccinia viruse, preferrably an attenuated Vaccinia virus, is used for the generation of recombinant poxviruses comprising two or more homologous genes.<br>
Although it is known that Vaccinia viruses (VV) may undergo homologous recombination of short homologous sequences and thereby may delete homologous sequences (Howley et al., [1996], Gene 172: 233-237) the inventors provide a recombinant Vaccinia virus including homologous sequences or genes stably inserted into its genome. This finding is particularly unexpected, since according to Howely et al. already short sequences from up to 300 base pairs (bp) were sufficient to induce genomic rearrangement and deletion of homologous sequences in Vaccinia virus. The practitioner would, thus, expect that longer sequences would induce recombination events with an even higher probability. However, according to the<br>
present invention even sequences comprising complete homologous genes can be stably inserted into the genome of Vaccinia virus.<br>
One - not limiting - example of a vaccinia virus is the highly attenuated and host range restricted Vaccinia strain, Modified Vaccinia Ankara (MVA) (Sutter, G. et al. [1994], Vaccine 12: 1032-40). MVA has been generated by about 570 serial passages on chicken embryo fibroblasts of the Ankara strain of Vaccinia virus (CVA) (for review see Mayr, A., et al. [1975], Infection 3, 6-14). As a consequence of these long-term passages CVA deleted about 31 kilobases of its genomic sequence. The resulting virus strain, MVA, was described as highly host cell restricted (Meyer, H. et al, J. Gen. Virol. 72, 1031-1038 [1991]). A typical MVA strain is MVA 575 that has been deposited at the European Collection of Animal Cell Cultures under the deposition number ECACC V00120707.<br>
In another embodiment the MVA-Vero strain or a derivative thereof can be used according to the present invention. The strain MVA-Vero has been deposited at the European Collection of Animal Cell Cultures under the deposition number ECACC V99101431 and ECACC 01021411. The safety of the MVA-Vero is reflected by biological, chemical and physical characteristics as described in the Indian Patent No: 228892. In comparison to normal MVA, MVA-Vero has one additional genomic deletion.<br>
The term "derivatives" of a virus according to the present invention refers to progeny viruses showing the same characteristic features as the parent virus but showing differences in one or more parts of its genome.<br>
Still another embodiment according to the present invention uses MVA-BN. MVA-BN has been deposited at the European Collection of Animal Cell Cultures with the deposition number ECACC V00083008. By using MVA-BN or a derivative thereof a particular safe virus vaccine is generated, since it has been shown that the MVA-BN virus is an extremely high attenuated virus, derived from Modified Vaccinia Ankara virus. Therefore, in the most preferred embodiment, MVA-BN or derivatives thereof containing two or more homologous genes according to the present invention is used as viral vector. The term "derivative of MVA-BN" describes a virus, which has the same functional characteristics compared to<br>
MVA-BN. The features of MVA-BN, the description of biological assays allowing the evaluation whether an MVA is MVA-BN and a derivative thereof and methods allowing the generation of MVA-BN or derivatives thereof are described in Indian Patent No: 203475 (incorporated herein by reference). One easy way to examine a functional characteristic of MVA-BN or derivatives thereof is its attenuation and lack of replication in human HaCat cells.<br>
In a recombinant poxvirus according to the present invention the expression of the exogenous sequences is controlled preferably by a poxviral transcriptional control element, more preferably by an MVA, canary pox, fowl pox, or penguin pox transcriptional control element or most preferably a Vaccinia virus promoter. Poxviral transcriptional control elements according to the present invention comprise furthermore every transcriptional control element functional in a poxviral system.<br>
The insertion of the exogenous sequences according to the present invention is preferably directed into a non-essential region of the virus genome. Non-essential regions are e.g. loci or open reading frames (ORF) of poxviral genes, which are non-essential for the poxviral life cycle. Also intergenic regions, which describe the space in between two ORF, are considered as non-essential regions according to the present invention. In another embodiment of the invention, the exogenous sequences are inserted at a naturally occurring deletion site of the MVA genome (disclosed in Indian Patent No: 212562 and incorporated herein by reference).<br>
The orientation of the inserted DNA does not have an influence on the functionality or stability of the recombinant virus according the present invention.<br>
Since the recombinant poxvirus according to the invention is highly growth restricted and, thus, highly attenuated, it is an ideal candidate for the treatment of a wide range of mammals including humans and even immune-compromised humans. Hence, the present invention also provides a pharmaceutical composition and also a vaccine for inducing an immune response in a living animal body, including a human.<br>
The pharmaceutical composition may generally include one or more pharmaceutically acceptable and/or approved carriers, additives, antibiotics, preservatives, adjuvants, diluents and/or stabilizers. Such auxiliary substances can be water, saline, glycerol, ethanol, wetting or emulsifying agents, pH buffering substances, or the like. Suitable carriers are typically large, slowly metabolized molecules such as proteins, polysaccharides, polylactic acids, polyglycollic acids, polymeric amino acids, amino acid copolymers, lipid aggregates, or the like.<br>
For the preparation of vaccines, the recombinant poxvirus according to the invention is converted into a physiologically acceptable form. This can be done based on the experience in the preparation of poxvirus vaccines used for vaccination against smallpox (as described by Stickl, H. et al. [1974] Dtsch. med. Wschr. 99, 2386-2392). For example, the purified virus is stored at -80°C with a titre of 5x10E8 TCID50/ml formulated in about 10mM Tris, 140 mM NaCl pH 7.4. For the preparation of vaccine shots, e.g., 10E2-10E8 particles of the virus are lyophilized in 100 ml of phosphate-buffered saline (PBS) in the presence of 2% peptone and 1% human albumin in an ampoule, preferably a glass ampoule. Alternatively, the vaccine shots can be produced by stepwise freeze-drying of the virus in a formulation. This formulation can contain additional additives such as mannitol, dextran, sugar, glycine, lactose or polyvinylpyrrolidone or other aids such as antioxidants or inert gas, stabilizers or recombinant proteins (e.g. human serum albumin) suitable for in vivo administration. The glass ampoule is then sealed and can be stored between 4°C and room temperature for several months. However, as long as no need exists the ampoule is stored preferably at temperatures below -20°C.<br>
For vaccination or therapy the lyophilisate can be dissolved in 0.1 to 0.5 ml of an aqueous solution, preferably physiological saline or Tris buffer, and administered either systemically or locally, i.e. parenteral, subcutaneous, intravenous, intramuscular, by scarification or any other path of administration know to the skilled practitioner. The mode of administration, the dose and the number of administrations can be optimized by those skilled in the art in a known manner. However, most commonly a patient is vaccinated with a second shot about one month to six weeks after the first vaccination shot.<br>
The recombinant virus according to the present invention is used for the introduction of the exogenous coding sequences into a target cell. The introduction of an exogenous coding sequence into a target cell may be used to produce in vitro proteins, polypeptides, peptides, antigens and epitopes, respectively. Furthermore, the method for introduction of a homologous or of a heterologous sequence into cells may be applied for in vitro and in vivo therapy. For in vitro therapy, isolated cells that have been previously (ex vivo) infected with the recombinant poxvirus according to the invention are administered to the living animal body for inducing an immune response. For in vivo therapy, the recombinant poxvirus according to the invention is directly administered to the living animal body for inducing an immune response. In this case, the cells surrounding the site of inoculation are directly infected in vivo by the virus or its recombinant according to the invention. After infection, the cells synthesize the proteins, polypeptides, peptides or antigens, which are encoded in the exogenous coding sequences and, subsequently, present them or parts thereof on the cellular surface. Specialized cells of the immune system recognize the presentation of such foreign proteins, polypeptides, peptides, antigens and epitopes and launch a specific immune response.<br>
Methods to obtain recombinant poxviruses or to insert exogenous coding sequences into a poxviral genome are well known to the person skilled in the art. Additionally, the method is described in the examples and can also be deduced or completed from the following references:<br>
Molecular Cloning, A laboratory Manual. Second Edition. By J. Sambrook,<br>
E.F. Fritsch and T. Maniatis. Cold Spring Harbor Laboratory Press. 1989: describes<br>
techniques and know how for standard molecular biology techniques such as cloning<br>
 of DNA, DNA and RNA isolation, western blot analysis, RT-PCR and PCR<br>
amplification techniques.<br>
Virology Methods Manual. Edited by Brian WJ Mahy and Hillar O Kangro. Academic Press. 1996: describes techniques for the handling and manipulation of viruses.<br>
Molecular Virology: A Practical Approach. Edited by AJ Davison and RM Elliott. The Practical Approach Series. IRL Press at Oxford University Press. Oxford<br>
1993. Chapter 9: Expression of genes by Vaccinia virus vectors. Current Protocols in Molecular Biology. Publisher: John Wiley and Son Inc. 1998. Chapter 16, section IV: Expression of proteins in mammalian cells using Vaccinia viral vector: describes techniques and know-how for the handling, manipulation and genetic engineering of MVA.<br>
For the generation of recombinant poxviruses according to the present invention different methods may be applicable: The DNA sequence to be inserted into the virus may be placed into an E. coli plasmid construct into which DNA homologous to a section of DNA of the poxvirus has been inserted. Seperately, the DNA sequence to be inserted is ligated to a promoter. The promoter-gene linkage is positioned in the plasmid construct so that the promoter-gene linkage is flanked on both ends by DNA homologous to a DNA sequence flanking a region of pox DNA containing a non-essential locus. The resulting plasmid construct is amplified by growth within E. coli bacteria and isolated. The isolated plasmid containing the DNA gene sequence to be inserted is transfected into a cell culture, e.g., chicken embryo fibroblasts (CEFs), along with the poxvirus. Recombination between homologous pox DNA in the plasmid and the viral genome, respectively, gives a poxvirus modified by the presence of foreign DNA sequences.<br>
According to a more preferred embodiment, a cell of a suitable cell culture as, e.g., CEF cells, is infected with a poxvirus. The infected cell is, subsequently, transfected with a first plasmid vector comprising the foreign gene, preferably under the transcriptional control of a poxvirus expression control element. As explained above, the plasmid vector also comprises sequences capable of directing the insertion of the exogenous sequence into a selected part of the poxviral genome. Optionally, the plasmid vector contains also a cassette comprising a marker and/or selection gene operably linked to a poxviral promoter. Suitable marker or selection genes are, e.g., the genes encoding the Green Fluorescent Protein, ß-Galactosidase, neomycin, phosphoribosyltransferase or other markers. The use of selection or marker cassettes simplifies the identification and isolation of the generated recombinant poxvirus. However, a recombinant poxvirus can also be identified by PCR technology. Subsequently, a further cell is infected with the recombinant<br>
poxvirus obtained as described above and transfected with a second vector comprising a gene, which is homologous to the gene included in the first vector. In case, this gene shall be included into a different insertion site of the poxviral genome, the second vector also differs in the sequence directing the integration of the homologous gene into the genome of the poxvirus. After homologous recombination occurred, the recombinant virus comprising two homologous genes can be isolated. For introducing more than two homologous genes into the recombinant virus, the steps of infection and transfection are repeated by using the recombinant virus isolated in previous steps for infection and by using a further vector comprising a further homologous gene for transfection.<br>
Alternatively, the steps of infection and transfection as described above are interchangeable, i.e., a suitable cell may at first be transfected by the plasmid vector comprising the foreign gene and, then, infected with the poxvirus.<br>
As a further alternative, it is also possible to introduce each homologous gene into different viruses, coinfect a cell with all the obtained recombinant viruses and screen for a recombinant including all homologous genes.<br>
The invention further provides a kit comprising two or more plasmid vector constructs capable of directing the integration of expressable, homologous genes into the poxvirus genome. Beside a suitable cloning site such plasmid vectors comprise sequences capable of directing the insertion of the exogenous sequence to selected parts in the poxviral genome. Optionally, such vectors comprise selection or marker gene cassettes. The kit further comprises means and instructions to select viruses, which are recombinant for one or several of the homologous genes and optionally a selection or marker gene, inserted via said vector constructs.<br>
According to another further embodiment the invention includes DNA sequences or parts thereof derived from or homologous to the recombinant poxvirus of the present invention. Such sequences comprise at least part of the exogenous sequence comprising at least a fragment of one of the homologous gene according to the present invention and at least a fragment of the genomic poxvirus sequence<br>
according to the present invention, said genomic poxvirus sequences preferably flanking the exogenous sequence.<br>
Such DNA sequences can be used to identify or isolate the virus or its derivatives, e.g. by using them to generate PCR-primers, hybridization probes or in array technologies.<br>
Short Description of the Figures<br>
Figure 1: Schematic presentation of the insertion sites of the four PrM (= pre-membrane) genes (serotype 1-4) in the MVA genome according to Example 1.<br>
Figures 2-9 and 12-17: Insertion plasmid vector constructs indicating the name of the vector, its size and the localization of the sequences of interest such as: AmpR = ampicillin resistance gene, bfp = blue florescence protein gene, dA = deletion A, dE = deletion E, d2 = deletion 2, Ecogpt = E. coli guanosinphosphoribosyl-transferase gene, EGFP = enhanced green florescence protein gene, Fl = flanking sequence 1, F2 = flanking sequence 2,14L = intergenic region I4L, IGR = intergenic region, NPT II = neomycin resistance gene, P = poxvirus promoter, pr7.5 = Vaccinia promoter 7.5, PrM = pre-membrane gene of Dengue virus, number indicating from which of the four serotypes it derives, rpt = repetition of flanking sequence.<br>
Figure 10: PCR verification of the vector cloning strategies of four different insertion vectors (pBN49, pBN50, pBN40, pBN39). Each of the plasmids was tested with 4 different PCR primer combinations. Each combination is specific for one distinct PrM sequence integrated into one distinct insertion site.<br>
Figure 11: PCR verification of the recombinant poxvirus including four homologous Dengue virus PrM genes (Example 1). While in the upper part of the gel the different PCR results of the recombinant virus are shown, the lower part provides the results of the same PCR reactions of the control plasmids as indicated. The plasmids containing the homologous sequences are named pBN39, pBN49 or pBN50. PrM stands for the inserted pre-membrane genes of Dengue virus, wherein the numbers indicate from which of the four serotypes it derives. dA = deletion A,<br>
dE = deletion E, d2 = deletion 2,14L = intergenic region I4L describes the insertion site of the exogenous DNA.<br>
Figure 18: Schematic presentation of the insertion sites of three PrM genes (serotype 2-4) in the MVA genome according to Example 2.<br>
Figure 19: PCR verification of the recombinant poxvirus including three homologous Dengue virus PrM genes inserted into intergenic regions (Example 2). The upper panel shows the results of the PCR reactions specific for PrM2; the middle panel shows the results of the PCR reactions specific for PrM3, and the lower panel shows the results of the PCR reactions specific for PrM4. Lane 8 shows the same PCR reactions of the control plasmids. Lane 2 shows the empty vector control MVA. PrM stands for the inserted pre-membrane genes of Dengue virus, wherein the numbers indicate from which of the four serotypes it derives. M = Molecular weight marker.<br>
The following examples will further illustrate the present invention. It will be well understood by a person skilled in the art that the provided examples in no way may be interpreted in a way that limits the applicability of the technology provided by the present invention to this examples.<br>
Example 1<br>
Insertion vectors<br>
Insertion vector for deletion A<br>
For the insertion of exogenous sequences into the MVA genome at the so-called deletion A or deletion 1 respectively, corresponding to the genome position 7608-7609, a plasmid vector was constructed, which comprises about 600 bp of the flanking sequences adjacent to the deletion site A. To isolate the flanking sequences from the genomic MVA-BN DNA, suitable PCR primers can be designed with suitable computer software (DNAsis, Hitashi software engeneering, San Bruno, USA). Such primers comprise extensions with restriction enzyme sites, which will be used to clone the flanking sequences into a vector plasmid. In between these flanking sequences, a selection gene cassette is introduced, e.g a NPT II gene<br>
(neomycin resistance) under the transcriptional control of a poxviral promoter. Additionally, there is a cloning site for the insertion of additional genes or exogenous sequences to be inserted into deletion site A. One such vector construct according to the present invention is disclosed in Figure 2 (pBNX10).<br>
Insertion vector for deletion E<br>
For the insertion of exogenous sequences into the MVA genome at the so-called deletion E or deletion 4, respectively, corresponding with the genome position 170480-170481, a vector was constructed, which comprises about 600 bp of the flanking sequences adjacent to the deletion site E. The vector is designed and constructed like described above. In between the flanking sequences is located an EGFP gene (green fluorescence protein, Clonetech) under the transcriptional control of a poxviral promoter. Additionally, there is a cloning site for the insertion of additional genes or sequences to be inserted into deletion site A. One such vector construct according to the present invention is disclosed in Figure 3 (pBNX32).<br>
Insertion vector for deletion 2<br>
For the insertion of exogenous sequences into the MVA genome at the so-called deletion 2, corresponding with the genome position 20718-20719, a vector was constructed, which comprises about 600 bp of the flanking sequences adjacent to the deletion site 2. The vector is designed and constructed like described above. In between the flanking sequences is located an hbfp gene (humanized blue fluorescing protein, Pavalkis GN et al.) under the transcriptional control of a poxviral promoter. Additionally, there is a cloning site for the insertion of additional genes or sequences to be inserted into deletion site 2. One such vector construct according to the present invention is disclosed in Figure 4 (pBNX36).<br>
Insertion vector for intergenetic region, I4L<br>
For the insertion of exogenous sequences in the intergenetic region, between the ORF I3L and I4L, corresponding to the genome position 56760, a vector was constructed, which comprises about 600 bp of the flanking sequences adjacent to the intergenetic region at the I4L locus. The vector is designed and constructed like described above. In between the flanking sequences is located an Ecogpt gene (or<br>
gpt stands for phosphoribosyl-transferase gene isolated from E.coli) under the transcriptional control of a poxviral promoter. Additionally, there is a cloning site for the insertion of additional genes or sequences to be inserted into the intergenetic region after the I4L ORF. One such vector construct according to the present invention is disclosed in Figure 5 (pBNX39).<br>
Construction of recombinant poxvirus comprising several homologous genes integrated in its genome.<br>
Insertion vectors<br>
For the insertion of the four PrM genes of the four serotypes of Dengue virus in the MVA genome four independent recombination vectors were used.<br>
These vectors contain - as described in details above - sequences homologous to the MVA genome for targeting insertion by homologous recombination. Additionally each vector contains a selection- or reporter gene cassette.<br>
The PrM sequences of the four Dengue virus serotypes were synthetically made by oligo annealing and PCR amplification. The PrM sequences were cloned downstream of poxvirus promoter elements to form an expression cassette. This expression cassette was, then, cloned into the cloning site of the relevant insertion vector constructs.<br>
As result, the insertion vector construct for deletion A contained the PrM gene of Dengue virus serotype 2 (Figure 6 - pBN39). The insertion vector construct for deletion 2 contained the PrM gene of Dengue virus serotype 1 (Figure 7 - pBN49). The insertion vector construct for intergenic region I4L contained the PrM gene of Dengue virus serotype 3 (Figure 8 - pBN50). The insertion vector construct for deletion E contained the PrM gene of Dengue virus serotype 4 (Figure 9 - pBN40).<br>
PCR verification of the insertion vectors<br>
For verification of the cloning strategies, PCR assays were performed. For these PCR assays the selected primer pairs are a combination of a primer specifically binding to the specific flanking sequence relative to the insertion site and a second<br>
primer specifically binding to one of the highly homologous Dengue virus PrM genes.<br>
The insertion vector for deletion A containing the PrM gene of Dengue virus serotype        2        was         screened        with        the         Primers        oBN93<br>
(CGCGGATCCATGCTGAACATCTTGAACAGGAGACGCAGA. SEQ ID NO.: 1) and oBN477 (CATGATAAGAGATTGTATCAG. SEQ ID NO.: 2).<br>
The insertion vector for deletion 2 containing the PrM gene of Dengue virus serotype 1 was screened with the Primers oBN194 (ATGTTGAACATAATGAACAGGAGGAAAAGATCTGTGACCATGCTCCTCA TGCTGCTGCCCACAGCCCTGGCGTTCCATCT. SEQ ID NO.: 3) and oBN476 (GATTTTGCTATTCAGTGGACTGGATG. SEQ ID NO.: 4).<br>
The insertion vector for intergenic region I4L containing the PrM gene of Dengue virus      serotype      3      was      screened      with      the      Primers      oBN255 (CCTTAATCGAATTCTCATGTCATGGATGGGGTAACCAGCATTAATAGT. SEQ ID NO.: 5) and oBN479 (GCTCCCATTCAATTCACATTGG. SEQ ID NO.: 6).<br>
The insertion vector for deletion E containing the PrM gene of Dengue virus serotype        4        was        screened        with        the        Primers        oBN210 (ATCCCATTCCTGAATGTGGTGTTAAAGCTACTGAGCGCTTCTCTCGTCTC CGTTCTCCGCTCTGGGTGCATGTCCCATAC. SEQ ID NO.: 7) and oBN478 (GTACATGGATGATATAGATATG. SEQ ID NO.: 8).<br>
PCR experiments are performed in a Thermal cycler GeneAmp 9700 (Perkin Elmer) using the Taq DNA Polymerase Kit containing 10x PCR buffer, MgCl2 buffer and Taq DNA polymerase (Roche, Cat.no. 201205) or equivalent. In general the PCR reactions were prepared with a total reaction volume of 50 µl containing 45 µl mastermix, the sample DNA and ddH2O as required. The mastermix should be prepared with 30.75 µl DdH20, 5 µl 10x buffer, 1 µl dNTP-mix (10 mM each), 2.5 µl of each primer (5 pmol/Dl), 3 µl MgCl2 (25 mM) and 0.25 µl Taq-polymerase (5 U/µl).<br>
The amplification was performed using the following programme:<br>
1)           Denaturation:                       4min            94°C<br>
2)           30 Cycles:<br>
             Denaturation:                       30 sec           94°C<br>
             Annealing:                           30 sec           55°C<br>
              Elongation:                         1-3 min         72°C<br>
3)           Elongation                          7 min            72°C<br>
4)           Store                                                         4°C<br>
Based on the size of the inserted gene the elongation time should at least be 1min/kb.<br>
The PCR results shown in Figure 10 demonstrate the specifity of the primer combinations used for the single insertions.<br>
The primer combination oBN194/oBN476 is specific for deletion 2 and PrMl as insert. The expected PCR fragment of plasmid pBN49 has a size of 678 bp (shown in lane 3, upper part of the gel).<br>
The primer combination oBN255/oBN479 is specific for intergenic region I4L and PrM3 as insert. The expected PCR fragment of plasmid pBN50 has a size of 825 bp (shown in lane 9, upper part of the gel).<br>
The primer combination oBN210/oBN478 is specific for deletion E and PrM4 as insert. The expected PCR fragment of plasmid pBN40 has a size of 607 bp (shown in lane 5, lower part of the gel).<br>
The primer combination oBN93/oBN477 is specific for deletion A and PrM2 as insert. The expected PCR fragment of plasmid pBN39 has a size of 636 bp (shown in lane 11, lower part of the gel).<br>
Generation of the recombinant MVA via homologous recombination<br>
For expression of foreign genes by a recombinant MVA, these genes have to be inserted into the viral genome by a process called homologous recombination. For that purpose, the foreign gene of interest had been cloned into an insertion vector, as described above. This vector has to be transfected after infection of cells with MVA-<br>
BN. The recombination will take place in the cellular cytoplasm of the infected and transfected cells. With help of the selection and/or reporter cassette, which is also contained in the insertion vector, cells comprising recombinant viruses are identified and isolated.<br>
Homologous recombination<br>
For homologous recombination, BHK (Baby hamster kidney) cells or CEF (primary chicken embryo fibroblasts) cells are seeded in 6 well plates using DMEM (Dulbecco's Modified Eagles Medium, Gibco BRL) + 10% fetal calf serum (FCS) or VP-SFM (Gibco BRL) + 4mmol/l L-Glutamine for a serum free production process.<br>
Cells need to be still in the growing phase and, therefore, should reach 60-80% confluence on the day of transfection. Cells were counted before seeding, as the number of cells has to be known for determination of the multiplicity of infection (moi) for infection.<br>
For infection, the MVA stock is diluted in DMEM/FCS or VP-SFM/L-Glutamine so that 500 ul dilution contain an appropriate amount of virus that will give a moi of 0.01. Cells are assumed to be divided once after seeding. The medium is removed from cells and cells are infected with 500ul of diluted virus for 1 hour rocking at room temperature. The inoculum is removed and cells are washed with DMEM/VP-SFM. Infected cells are left in 1.6ml DMEM/FCS and VP-SFM/L- Glutamine respectively while setting up transfection reaction (Qiagen Effectene Kit).<br>
For transfection, the "Effectene" transfection kit (Qiagen) is used. A transfection mix is prepared of 1-5 µg of linearized insertion vector (total amount for multiple transfection) with buffer EC to give a final volume of 150 µl. Add 8.0 µl Enhancer per µg DNA, vortex and incubate at room temperature for 5 min. Then, 25 µl of Effectene are added per ug DNA after vortexing stock tube and the solution is mixed thoroughly by vortexing and incubated at room temperature for 10 min. 600 µl of DMEM/FCS and VP-SFM/L-Glutamine respectively, are added, mixed and subsequently, the whole transfection mix is added to the cells, which are already covered with medium. Gently the dish is rocked to mix the transfection reaction.<br>
Incubation takes place at 37°C with 5%CO2 over night. The next day the medium is removed and replaced with fresh DMEM/FCS or VP-SFM/L-Glutamine. Incubation is continued until day 3.<br>
For harvesting, the cells are scraped into medium, then the cell suspension is transferred to an adequate tube and frozen at -20°C for short-term storage or at -80°C for long term storage.<br>
Insertion of PrM4 into MVA<br>
In a first round, cells were infected with MVA-BN according to the above-described protocol and were additionally transfected with insertion vector pBN40 containing the PrM gene of Dengue virus serotype 4 and as reporter gene the EGPF gene. Since the transfected vector contains a reporter gene, EGFP, the synthesized protein is detectable latest on day three in cells infected with a recombinant virus. Resulting recombinant viruses have to be purified by plaque purification.<br>
For plaque purification infected cells (fluorescing or stained) are isolated with a pipet tip, resuspended and aspirated in 200µl PBS or medium. Then a fresh culture dish containing about 10E6 cells is infected with 100µl of the resuspended plaques. After 48h cells are taken up in 300µl PBS. DNA is extracted from suspension and screened with PCR analysis. A clone that shows the expected bands is chosen and fresh 6-well plates are infected with different amounts of this virus. Overlaying the wells with 1% agarose avoids further spreading of virus. After 48h infected cells comprising a recombinant virus clone are isolated.<br>
This procedure is repeated until no wild-type MVA-BN can be detected in the PCR analysis.<br>
After 4 rounds of plaque purification recombinant viruses, MVA-PrM4, were identified by PCR assays using a primer pair selectively amplifying the expected insertion (oBN210 and oBN478, as described above) and as control a primer pair specifically recognizing the insertion site deletion E (oBN453: GTTGAAGGATTCACTTCCGTGGA, SEQ ID NO.: 9 and oBN454: GCATTCACAGATTCTATTGTGAGTC, SEQ ID NO.: 10)<br>
Insertion of PrM2 into MVA-PrM4<br>
Cells were infected with MVA-PrM4 according to the above described protocol and were additionally transfected with insertion vector pBN39 containing the PrM gene of Dengue virus serotype 2 and as selection gene the NPT II, a neomycin resistance gene. For purification of recombinant MVA expressing an antibiotic resistance gene three rounds of virus amplification under selective conditions before plaque purification are recommended. For neomycinphosphotransferase selection G418 is added to the medium. G418 is a derivative of neomycin and inhibits the protein-biosynthesis by interference with the action of the ribosomes. NPT gene activity inactivates G418 by phosphorylation.<br>
After 16 rounds of plaque purification under neomycin selection recombinant viruses, MVA-PrM4/PrM2, were identified by PCR assays using a primer pair selectively amplifying the expected insertion (oBN93 and oBN477, as described above) and as control a primer pair specifically recognizing the insertion site deletion A (oBN477: as described above) and oBN452: GTTTCATCAGAAATGACTCCATGAAA, SEQ ID NO.: 11). Additionally also insertion of PrM4 into deletion E is verified with the primer pairs: oBN210-oBN478 and oBN453 - oBN454.<br>
Insertion of PrM1 into MVA<br>
In a first round, cells were infected with MVA-BN according to the above described protocol and were additionally transfected with insertion vector pBN49 containing the PrM gene of Dengue virus serotype 1 and as reporter gene the hbfjp, the gene for humanized blue fluorescing protein. The synthesized hbfp protein is detectable on day three in cells infected with a recombinant virus. Resulting recombinant viruses were purified by plaque purification.<br>
After 10 rounds of plaque purification recombinant viruses, MVA-PrMl, were identified by PCR assays using a primer pair selectively amplifying the expected insertion (oBN194 and oBN476, as described above) and as control a primer pair specifically recognizing the insertion site deletion 2 (oBN54: CGGGGTACCCGACGAACAAGGAACTGTAGCAGAGGCATC, SEQ ID NO.:<br>
12 and oBN56: AACTGCAGTTGTTCGTATGTCATAAATTCTTTAATTAT, SEQIDN0.:13)<br>
Insertion of PrMS into MVA<br>
In a first round, cells were infected with MVA-BN according to the above described protocol and were additionally transfected with insertion vector pBN50 containing the PrM gene of Dengue virus serotype 3 and as reporter gene the Ecogpt gene (Ecogpt or shortened to gpt stands for phosphoribosyltransferase gene). Resulting recombinant viruses were purified by 3 rounds of plaque purification under phosphribosyltransferase metabolism selection by addition of mycophenolc acid, xanthin and hypoxanthin. Mycophenolic acid (MP A) inhibits inosine monophosphate dehydrogenase and results in blockage of purine synthesis and inhibition of viral replication in most cell lines. This blockage can be overcome by expressing Ecogpt from a constitutive promoter and providing the substrates xanthine and hypoxanthine.<br>
Resulting recombinant viruses, MVA-PrM3, were identified by PCR assays using a primer pair selectively amplifying the expected insertion (oBN255 and oBN479, as described above) and as control a primer pair specifically recognizing the insertion site I4L (oBN499: CAACTCTCTTCTTGATTACC, SEQ ID NO.: 14 and oBN500: CGATCAAAGTCAATCTATG, SEQ ID NO.: 15).<br>
Coinfection ofMVA-PrM1 andMVA-PrM3<br>
The cells were infected with equal amounts of MVA-PrMl and MVA-PrM3 according to the above protocol. After 3 rounds of plaque purification under phosphribosyltransferase metabolism selection of blue fluorescing clones of recombinant viruses were analyzed by PCR using the primer pairs (oBN255 and oBN479. oBN499 and oBN500. oBN194 and oBN476. oBN54 and oBN56 as described above). Resulting recombinant viruses were designated MVA-PrMl/PrM3.<br>
Coinfection of MVA-PrMl/PrM3 and MVA-PrM2/PrM4<br>
The cells were infected with equal amounts of MVA-PrMl/PrM3 and MVA-PrM2/PrM4 according to the above protocol. Plaque purification was performed under phosphnbosyltransferase metabolism and neomycin selection. Recombinant viruses inducing a green and blue flourescence were isolated and analyzed by PCR using the primer pairs (oBN255 and oBN479. oBN499 and oBN500. oBN194 and oBN476. oBN54 and oBN56. oBN93 and oBN477. oBN477 and oBN452. oBN210 and oBN478. oBN453 and oBN454 as described above).<br>
The PCR analysis of the recombinant virus (Clone 20) comprising all four PrM genes is shown in Figure 11. While in the upper part of the gel the different PCR results of the recombinant virus are shown, the lower part provides the results of the same PCR reactions of the control plasmids (as indicated). Lane 1,10 and 11 show a lkb and a lOObp molecular marker.<br>
The primer combination oBN210/oBN478 is specific for deletion E and PrM4 as insert. The expected PCR fragment of the recombinant virus and the plasmid pBN40 has a size of 607 bp (shown in lane 2).<br>
The primer combination oBN453/oBN454 is specific for deletion E. The expected PCR fragment of the recombinant virus is 2.7 kb, the expected band of the wild-type virus 2.3 kb (shown in lane 3). Also in the upper part of the gel a band specific for a wild-type virus can be identified. This means that the recombinant virus preparation is not yet completely free of wild-type virus. Further plaque purification is necessary.<br>
The primer combination oBN93/oBN477 is specific for deletion A and PrM2 as insert. The expected PCR fragment of the recombinant virus and the plasmid pBN39 has a size of 636 bp (shown in lane 4).<br>
The primer combination oBN477/oBN452 is specific for deletion A. The expected PCR fragment of the recombinant virus is 4.1 kb, the expected band of the wild-type virus 2.7 kb (shown in lane 5). In the upper part of the gel a band specific for a wild-type virus can be identified.<br>
The primer combination oBN255/oBN479 is specific for intergenic region I4L and PrM3 as insert. The expected PCR fragment of the recombinant virus and the plasmid pBN50 has a size of 825 bp (shown in lane 6).<br>
The primer combination oBN499/oBN500 is specific for the intergenic region of I4L. The expected PCR fragment of the recombinant virus is 1.0 kb, the expected band of the wild-type virus 0.3 kb (shown in lane 7).<br>
The primer combination oBN194/oBN476 is specific for deletion 2 and PrMl as insert. The expected PCR fragment of the recombinant virus and the plasmid pBN49 has a size of 678 bp (shown in lane 8).<br>
The primer combination oBN54/oBN56 is specific for deletion 2. The expected PCR fragment of the recombinant virus is 1.6 kb, the expected band of the wild-type virus 0.9 kb (shown in lane 9). In the upper part of the gel a band specific for a wild-type virus can be identified.<br>
Alternatively, one can produce 4 different viruses, coinfect cells with all four viruses and screen for a recombinant.<br>
Improvements can also be achieved with recombination vectors, which contain further selection- or resistance markers.<br>
Example 2<br>
Insertion vectors<br>
Recombination vector for intergenic region 136-137 (IGR 136-137)<br>
For the insertion of exogenous sequences into the MVA genome at the so-called intergenic region (IGR) 136-137 corresponding to the genome position 129.940 a plasmid vector was constructed which comprises about 600 bp of the flanking sequences adjacent to the insertion site. To isolate the flanking sequences from the genomic MVA-BN DNA suitable PCR primers can be designed. Such primers comprise extensions with restriction enzyme sites, which will be used to clone the flanking sequences into a vector plasmid. In between this flanking sequences, a selection gene cassette is introduced, e.g. NPT II gene (neomycine resistance) under the transcriptional control of a poxviral promoter (P). Additionally there is a cloning<br>
site for the insertion of additional genes or exogenous sequences to be inserted into IGR 136-137 (Pad). One such vector construct according to the present invention is disclosed in Fig. 12 (pBNX67).<br>
Recombination vector for intergenic region 07-08 (IGR 07-08)<br>
For the insertion of exogenous sequences into the MVA genome in the intergenic region (IGR) 07-08 corresponding to the genome position 12.800, a plasmid vector was constructed which comprises about 600 bp of the flanking sequences adjacent to the insertion site. To isolate the flanking sequences from the genomic MVA-BN DNA suitable PCR primers can be designed. Such primers comprise extensions with restriction enzyme sites, which will be used to clone the flanking sequences into a vector plasmid. In between this flanking sequences, a selection gene cassette is introduced, e.g., Ecogpt gene (Guanin-Phosphoribosyltransferase) under the transcriptional control of a poxviral promoter (P). Additionally, there is a cloning site for the insertion of additional genes or exogenous sequences to be inserted into IGR 07-08 (PacI). One such vector construct according to the present invention is disclosed in Fig. 13 (pBNX88).<br>
Recombination vector for intergenic region 44-45 (IGR 44-45)<br>
For the insertion of exogenous sequences into the MVA genome at the intergenic region (IGR) 44-45 corresponding to the genome position 37.330, a plasmid vector was constructed which comprises about 600 bp of the flanking sequences adjacent to the insertion site. To isolate the flanking sequences from the genomic MVA-BN DNA suitable PCR primers can be designed. Such primers comprise extensions with restriction enzyme sites, which will be used to clone the flanking sequences into a vector plasmid. In between this flanking sequences, a selection gene cassette is introduced, e.g., NPT II gene (neomycine resistance) under the transcriptional control of a poxviral promoter (P). Additionally there is a cloning site for the insertion of additional genes or exogenous sequences to be inserted into IGR 44-45 (Pad). One such vector construct according to the present invention is disclosed in Fig. 14 (pBNX87).<br>
Construction of recombinant poxvirus comprising several homologous genes integrated in its genome<br>
Insertion vectors<br>
For the insertion of the three PrM genes of serotype 2, 3 and 4 of Dengue virus in the MVA genome, three independent recombination vectors were used.<br>
These vectors contain - as described in detail above - sequences homologous to the MVA genome for targeting insertion by homologous recombination. Additionally each vector contains a selection and reporter gene cassette.<br>
The PrM sequences of three Dengue virus serotypes were synthetically made as described in Example 1.<br>
As result, the insertion vector construct for IGR136-137 contained the PrM of Dengue virus serotype 4 (Fig. 15 - pBN27). The insertion vector construct for IGR 07-08 contained the PrM of Dengue virus serotype 2 (Fig. 16 - pBN34), and the insertion vector construct for IGR 44-45 contained the PrM of Dengue virus serotype 3 (Fig. 17 - pBN47).<br>
Generation of the recombinant MVA via homologous recombination<br>
The generation of recombinant MVA by homologous recombination was performed as described in Example 1.<br>
The insertion sites for the PrM4, PrM3 and PrM 2 in the MVA genome are depicted in Fig. 18.<br>
Insertion of PrM 4 into MVA<br>
In a first round, cells were infected with MVA-BN according to the above-described protocol and were additionally transfected with the insertion vector pBN27 containing the PrM gene of Dengue virus serotype 4 and as a reporter gene the EGFP gene. Since the transfected vector contains a reporter gene, EGFP, the synthesized protein is detectable latest on day three in cells infected with a recombinant virus. Resulting recombinant viruses have to be purified by plaque purification as described in Example 1. After four rounds of plaque purification recombinant viruses, MVA-PrM4, were identified by PCR assays using a primer<br>
pair selectively amplifying the insertion site IGR136-137 (0BN1008: gataccgatcacgttcta. SEQ ID NO.: 16; and oBN1009 ggatatgattatgtagag. SEQ ID NO.: 17).<br>
Insertion ofPrM2 into MVA<br>
Cells were infected with MVA-PrM4 according to the above-described protocol and were additionally transfected with the insertion vector pBN34 containing the PrM gene of Dengue virus serotype 2 and as a reporter gene the BFP gene. Since the transfected vector contains a reporter gene, BFP, the synthesized protein is detectable latest on day three in cells infected with a recombinant virus. Resulting recombinant viruses have to be purified by plaque purification as described in Example 1. After six rounds of plaque purification recombinant virus MVA-PrM4+PrM2 was further passaged and amplified and a crude stock was prepared. The recombinant was identified by PCR assays using a primer pair selectively amplifying the insertion site IGR07-08 (oBN 903: ctggataaatacgaggacgtg. SEQ ID NO.: 18; and oBN904: gacaattatccgacgcaccg; SEQ ID NO.: 19).<br>
Insertion of PrM 3 into MVA<br>
Cells were infected with MVA-PrM2+4 according to the above-described protocol and were additionally transfected with the insertion vector pBN47 containing the PrM gene of Dengue virus serotype 3 and as a reporter gene the EGFP gene. Since the transfected vector contains a reporter gene, EGFP, the synthesized protein is detectable latest on day three in cells infected with a recombinant virus. Resulting recombinant viruses have to be purified by plaque purification as described in Example 1. After three rounds of plaque purification recombinant viruses, MVA-PrM4+3+2, were identified by PCR assays using a primer pair selectively amplifying the insertion site IGR44-45 (oBN904: cgttagacaacacaccgacgatgg. SEQ ID NO.: 20; and oBN905 cggatgaaaaatttttggaag. SEQ ID NO.: 21).<br>
The PCR analysis of the recombinant virus comprising the three Dengue virus PrM genes is shown in Fig. 19. PCR experiments are performed as described in Example 1. The primer combination 0BN1008 and oBN1009 is specific for IGR136-137, which contains the PrM4 insertion (Fig.  19, lower panel). The expected PCR<br>
fragment of the recombinant virus has a size of lkb (shown in lane 4, 5 and 6) as the plasmid positive control (lane 8). The empty vector control, devoid of PrM 4 shows the expected fragment of 190 bp (lane 2). Lane M shows the molecular weight marker and lanes 1, 3 and 7 are empty. The primer combination oBN902 and oBN903 is specific for IGR07-08, which contains the PrM2 insertion (Fig. 19, upper panel). The expected PCR fragment of the recombinant virus has a size of 960bp (shown in lane 4-6) as the plasmid positive control (lane 8). The empty vector control, devoid of PrM 2 shows the expected fragment of 190 bp (lane 2). The primer combination oBN904 and oBN905 is specific for IGR44-45, which contains the PrM3 insertion (Fig. 19, middle panel). The expected PCR fragment of the recombinant virus has a size of 932bp (shown in lane 4-6) as the plasmid positive control (lane 8). The empty vector control, devoid of PrM 2 shows the expected fragment of 185 bp (lane 2).<br><br><br><br><br><br><br><br><br><br><br>
I/Wc Claim:<br>
1.	A recombinant Modified Vaccinia Ankara (MVA) virus comprising at least two homologous sequences or genes with an identity of at least 50% wherein each of said sequences or genes is stably inserted into a different insertion site of the viral genome or each of the said sequences is stably inserted into a naturally occurring deletion site and/or each of the said sequences is stably inserted into an intergenic region of the poxviral genome.<br>
2.	A recombinant MVA virus comprising at least two homologous sequences or genes, with an identity of at least 60%.<br>
3.	The recombinant MVA virus as claimed in claim 1 or 2, wherein the sequences or genes have an identity of 65-75%.<br>
4.	The recombinant MVA virus as claimed in any one of claims 1 to 3, wherein the<br>
homologous genes are derived from a flavivirus.<br>
5.	The recombinant MVA virus as claimed in claim 4, wherein the flavivirus is a Dengue virus.<br>
6.	The recombinant MVA virus as claimed in claim 4 or 5, wherein the genes arc at least two homologous genes derived from at least two different serotypes of the virus.<br>
7.	The recombinant MVA virus as claimed in any one of claims 4 to 6, wherein the genes are at least two PrM genes.<br>
8.	The recombinant MVA virus as claimed in anyone of claims 4 to 7, wherein the genes arc 4 PrM genes.<br>
9.	The recombinant MVA virus according to 1 or 2, wherein the homologous sequences or genes are identical.<br>
10.	The recombinant MVA virus according to claim 9, wherein said homologous<br>
sequences are promoters.<br>
11.	The recombinant MVA virus according to claim 9 or 10, wherein the sequences are<br>
the vaccinia virus early/late promoter pr7.5.<br><br>
12.	The recombinant MVA according to any one of claims 4 to 8, wherein the homologous genes are each under transcriptional control of the same promoter.<br>
13.	The recombinant MVA virus as claimed in claim 4 to 9 or 12, wherein the homologous genes are each under transcriptional control of the vaccinia virus early/late promoter pr7.5.<br><br>
14.	The recombinant MVA virus as claimed in any one of claims 1 to 13, wherein the MVA used for generating the recombinant virus is an MVA-BN as deposited at the Europcan Collection of Animal Cell Cultures (ECACC) under number V00083008.<br>
15.	The recombinant MVA virus as claimed in anyone of claims 1 to 14, wherein the MVA is replication deficient or replication incompetent in mammalian cells, including human cells.<br>
16.	A vaccine comprising the recombinant MVA virus as claimed in any of the claims 1 to 15.<br>
17.	A pharmaceutical composition comprising the recombinant MVA virus as claimed in any of the claims 1 to 15 and a pharmaceutically acceptable carrier, diluent, adjuvant and/or additive.<br>
18.	An in-vitro or ex-vivo method for generating a recombinant MVA virus as claimed in any one of claims 1 to 15 comprising the steps of<br><br>
-	infecting a cell with an MVA virus;<br>
-	transfecting the infected cell with a first vector construct comprising a sequence to be introduced into the genome of MVA virus, and a genomic MVA virus sequence capable of directing the integration of the sequence to be inserted into an insertion site of the MVA viral genome;<br>
-	identifying, isolating and, optionally, purifying the generated recombinant poxvirus;<br>
- repeating the above steps by using the recombinant MVA virus obtained from previous steps for infecting the cell and an additional vector construct comprising a further sequence to be introduced into the MVA viral genome, said sequence being homologous to the sequence of the first vector construct.<br><br>
19.	A recombinant DNA sequence comprising the viral genome of the recombinant MVA virus as claimed in anyone of claims 1 to 15, wherein said DNA sequence comprises (i) at least two homologous sequences with an identity of atleast 50% and which are inserted into a different insertion site and (ii) at least part of the sequences of the MVA viral genome, wherein said part of the sequences of the MVA viral genome flank said homologous sequences.<br>
20.	A plasmid vector comprising the DNA sequence as claimed in Claim 19.<br>
21.	An in vitro or ex vivo method for detecting cells infected with the recombinant MVA virus as claimed in anyone of claims 1 to 15 and/or for identifying the recombinant MVA virus as claimed in any of claims 1 to 15, said method comprising contacting the DNA sequence as claimed in claim 19 to said cells.<br>
22.	An in vitro or ex vivo method for detecting cells infected with the recombinant MVA virus as claimed in any one of claims 1 to 15 and/or for identifying the recombinant MVA virus as claimed in any of claims 1 to 15, said method comprising contacting the cell with DNA primers selectively amplifying the homologous sequences and/or the flanking sequence related to the insertion site of the sequence.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUFic3RyYWN0LSgwOS0wMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Abstract-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNsYWltcy0oMDktMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Claims-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNsYWltcy0oMjQtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Claims-(24-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvZGVuY2UgT3RoZXJzLSgwOC0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspodence Others-(08-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDEtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(01-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(02-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDktMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(09-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(13-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(24-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence Others-(24-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMTUtMDItMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-others (15-02-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-Others-(01-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDktMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-Others-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDctMjAxMCktLnBkZg==" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-Others-(19-07-2010)-.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-Others-(19-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjYtMTAtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Correspondence-Others-(26-10-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUNvcnJlc3BvbmVuY2UtT3RoZXJzLSgxMC0wNy0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Corresponence-Others-(10-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDA5LTAzLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Description (Complete)-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LURyYXdpbmdzLSgwOS0wMy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Drawings-(09-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMS0oMTktMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-1-(19-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMDEtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(01-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMDItMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(02-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMDgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(08-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMDktMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(09-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMTMtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(13-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMTktMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(19-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUZvcm0tMy0oMjYtMTAtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Form-3-(26-10-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUdQQS0oMTAtMDctMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-GPA-(10-07-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUdQQS0oMTUtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-GPA-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUdQQS0oMTktMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-GPA-(19-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LUdQQS0oMjQtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-GPA-(24-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0xMzQucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-134.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC00MDgucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-408.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">2955-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk1NS1ERUxOUC0yMDA0LVBldGl0aW9uIDEzNy0oMDgtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2955-DELNP-2004-Petition 137-(08-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="251199-a-tool-for-securing-a-flurality-of-workpieces-with-multi-piece-swage-type-fastener.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251201-a-crystalline-material-and-method-of-synthesizing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251200</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2955/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Mar-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Feb-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>29-Sep-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BAVARIAN NORDIC A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BOGESKOVVEJ 9, DK-3490 KVISTGAARD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PAUL HOWLEY</td>
											<td>60 ROSE AVENUE, GLEN WAVERLY, VIC 3150, AUSTRALIA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SONJA LEYRER</td>
											<td>FURSTENRIEDERSTRASSE 272, 81377 MUNCHEN, GERMANY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/863</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2003/05047</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-05-14</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PA 2002 00752</td>
									<td>2002-05-16</td>
								    <td>Denmark</td>
								</tr>
								<tr>
									<td>2</td>
									<td>PA 2002 00753</td>
									<td>2002-05-16</td>
								    <td>Denmark</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251200-recombinant-poxvirus-expressing-homologous-genes-inserted-into-the-poxviral-genome by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 14:01:22 GMT -->
</html>
