<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251650-an-isolated-monoclonal-antibody by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:48:14 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251650:AN ISOLATED MONOCLONAL ANTIBODY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN ISOLATED MONOCLONAL ANTIBODY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>&quot;AN 0772P POLYPEPTIDE&quot; An 0772P polypeptide having the structure: Xn-Y wherein X comprises a sequence having at least 50% identity with the consensus 0772P repeat sequence set forth in SEQ ID NO: 596; Y comprises a sequence having at least 80% identity with the 0772P constant region sequence set forth in SEQ ID NO: 594; n is an integer from 1 to 35;</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT 1970<br>
[39 OF 1970]<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See Section 10; rule 13]<br>
"AN 0772P POLYPEPTIDE" <br>
 CORIXA CORPORATION, A corporation of the State of Delaware, of 1124  Columbia Street, Suite 200, Seattle, Washington 98104, United States of    America<br><br><br>
The following specification particularly describes the invention and the manner in which it is to be performed:<br><br><br>
COMPOSITIONS AND METHODS FOR THE THERAPY AND DIAGNOSIS OF<br>
OVAPJAN CANCER<br>
Technical Field	<br>
The present invention relates generally to ovaian cancer therapy. The invention is more specifically related to polypeptides comprising at least a portion of an ovarian carcinoma protein, and to polynucleotides encoding such polypeptides, as well as antibodies and immune, system cells that specifically recognize such polypeptides. Such polypeptides, polynucleotides, antibodies mid cells may be used in vaccines and pharmaceutical compositions for treatment of ovarian cancer.<br>
Background of the Invention<br>
Ovarian cancer is a significant health problem for women in the United States and throughout the world. Although advances have been made in detection and therapy of this cancer, no vaccine or other universally successful method for prevention<br>
or treatment is currently available. Management of the disease currently relies on a combination of early diagnosis and aggressive treatment, which may include one or more of a variety of treatments such as surgery, radiotherapy, chemotherapy and hormone therapy. The course of treatment for a particular cancer is often selected based on a variety of prognostic parameters, including an analysis of specific tumor markers. However, the use of established markers often leads to a result that is difficult to interpret, and high mortality continues to be observed in many cancer patients. â€¢<br>
Immunotherapies have the potential  to  substantially improve cancer treatment and survival.   Such therapies may involve the generation or enhancement of an immune response to an ovarian carcinoma antigen.  However, to date, relatively few ovarian carcinoma antigens are known and the generation of an immune response against such antigens has not been shown to be therapeutically beneficial.<br>
Accordingly, there is a need in the art for improved methods for identifying ovarian tumor antigens and for using such antigens in the therapy of ovarian cancer. The present invention fulfills these needs and further provides other related advantages.<br><br>
SUMMARY OF THE INVENTION<br>
Briefly stated, this invention provides compositions and methods for the therapy of cancer, such as ovarian cancer. In one aspect, the present invention provides polypeptides comprising an immunogenic portion of an ovarian carcinoma protein, or a variant thereof that differs in one or more substitutions, deletions, additions and/or insertions such that the ability of the variant to react with ovarian carcinoma protein-specific antisera is not substantially diminished. Within certain embodiments, the ovarian carcinoma protein comprises a sequence that is encoded by a polynucleotide sequence selected from the group consisting of SEQ ID NO:456-457, 460-477 and 512-570 and complements of such polynucleotides.<br>
The present invention further provides polynucleotides that encode a poJypeptide as described above or a portion thereof, expression vectors comprising such polynucleotides and host cells transformed or transfected with such expression vectors.<br>
The present invention further provides polypeptide compositions comprising an amino acid sequence selected from the group consisting of sequences recited in SEQ ID Nos:394-455, 458-459, 478-511, and 571-596.<br>
Within other aspects, the present invention provides pharmaceutical compositions    and    vaccines.       -Pharmaceutical    compositions   may    comprise   a physiologically acceptable carrier or excipient in combination with one or more of: (i) a polypeptide comprising an immunogenic portion of an ovarian carcinoma protein, or a variant thereof that differs in one or more substitutions, deletions, additions and/or insertions such that the ability of the variant to react with ovarian carcinoma protein-specific antisera is not substantially diminished, wherein the ovarian carcinoma protein comprises an amino acid sequence encoded by a polynucleotide that comprises a sequence recited in any one of SEQ ID NO: 456-457, 460-477 and 512-570  or (if) a polynucleotide encoding such a polypeptide; (iii) an antibody that specifically binds to such a polypeptide; (iv) an antigen-presenting cell that expresses such a poJypeptide and/or (v) a T cell that specifically reacts with such a polypeptide.    Vaccines may comprise a non-specific immune response enhancer in combination with one or more of: (i) a polypeptide comprising an immunogenic portion of an ovarian carcinoma protein, or a variant thereof that differs in one or more substitutions, deletions, additions<br><br>
and/or insertions such that the ability of the variant to react with ovarian carcinoms protein-specific antisera is not substantially diminished, wherein the ovarian carcinoma protein comprises an amino acid sequence set forth in SEQ ID Nos:394-455, 458-459 478-511, and 571-596 or an amino acid sequence encoded by a polynucleotide that comprises a sequence recited in any one of SEQ ID NO: 456-457, 460-477 and 512-570 or (ii) a polynucleotide encoding such a polypeptide (Hi) an anti-idiotypic antibody that is specifically bound by an antibody that specifically binds to such a polypeptide; (iv) an antigen-presenting cell that expresses such a polypeptide and/or (v) a T cell that specifically reacts with such a polypeptide.<br>
The present invention further provides, in other aspects, fusion proteins that comprise at least one polypeptide as described above, as well as polynucleotides encoding such fusion proteins.<br>
Within related aspects, pharmaceutical compositions comprising a fusion protein or polynucleotide encoding. a fusion protein in combination with a physiologically acceptable carrier are provided.<br>
Vaccines are further provided, within other aspects, comprising a fusion protein or polynucleotide encoding a fusion protein in combination with a non-specific immune response enhancer.<br>
Within further aspects, the present invention provides methods for inhibiting the development of a cancer in a patient, comprising administering to a patient a pharmaceutical composition or vaccine as recited above.<br>
The present invention further provides, within other aspects, methods for stimulating  and/or  expanding T  cells,  comprising   contacting  T  cells  with (a)  a polypeptide comprising an immunogenic portion of an ovarian carcinoma protein, or a variant thereof that differs in one or more substitutions, deletions, additions and/or insertions such that the ability of the variant to react with ovarian carcinoma protein-specific antisera is not substantially diminished, wherein the ovarian carcinoma protein comprises an amino acid sequence set forth in SEQ ID Nos:394-455, 458-459, 478-511, and 571-596 or an amino acid sequence encoded by a polynucleotide that comprises a sequence recited in any one of SEQ ID NO: 456-457, 460-477 and 512-570; (b) a polynucleotide encoding such a polypeptide and/or (c) an antigen presenting cell that<br><br>
expresses such a polypeptide under conditions and for a time sufficient to permit the stimulation and/or expansion of T cells. Such polypeptide, polynucleotide and/or antigen presenting cell(s) may be present within a pharmaceutical composition or vaccine; for use in stimulating and/or expanding T cells in a mammal.<br>
Within other aspects, the present invention provides methods for inhibiting the development of ovarian cancer in a patient, comprising administering to a patient T cells prepared as described above.<br>
Within  further aspects,  the present invention provides methods for inhibiting the development of ovarian cancer in a patient, comprising the steps of:  (a) incubating CD4+ and/or CD8+ T cells isolated from a patient with one or more of: (i) a polypeptide comprising an immunogenic portion of an ovarian carcinoma protein, or a variant thereof that differs in one or more substitutions, deletions, additions and/or insertions such that the ability of the variant to react with ovarian carcinoma protein-specific antisera is not substantially diminished, wherein the ovarian carcinoma protein comprises  an amino acid sequence encoded by a polynucleotide that comprises a sequence recited in any one of SEQ ID NO: 456-457, 460-477 and 512-570; (ii) a polynucleotide encoding such a polypeptide; or (iii) an antigen-presenting cell that expresses such a polypeptide; such that T cells proliferate; and (b) administering to the patient an effective amount of the proliferated T cells, and thereby inhibiting the development of ovarian cancer in the patient The proliferated cells may be cloned prior to administration to the patient.<br>
The present invention also provides, within other aspects, methods for identifying  secreted tumor  antigens.     Such methods  comprise the  steps  of:     (a) implanting tumor cells in an immunodeficient mammal; (b) obtaining serum from the immunodeficient mammal after a time sufficient to permit secretion of tumor antigens into the serum; (c) immunizing an immunocompetent mammal with the serum; (d) obtaining antiserum from the immunocompetent mammal; and (e) screening a tumor expression library with the antiserum, and therefrom identifying a secreted tumor antigen.    A preferred method for identifying a secreted ovarian carcinoma antigen ;omprises the steps of:   (a) implanting ovarian carcinoma cells in a SCID mouse; (b) Dbtainine serum from the SCID mouse after a time sufficient to permit secretion of<br><br>
ovarian carcinoma antigens into the  serum;  (c) immunizing an immunocompetent<br>
mouse with the serum; (d) obtaining antiserum fom the immunocompetent mouse; and<br>
(e) screening an ovarian carcinoma expression library with the antiserum, and therefrom<br>
identifying a secreted ovarian carcinoma antigen.<br>
5	The present invention also discloses antibody epitopes recognized by the<br>
08E polyclonal anti-sera which epitopes are preserved herein as SEQ ID NO: 394-415.<br>
Further disclosed by the present invention are 10-mer and 9-mer peptides<br>
predicted to bind HLA-0201 which peptides are disclosed herein as SEQ ID N0:416-<br>
435 and SEQ ID N0:436-455? respectively.<br>
10	These and other aspects of the present invention will become apparent<br>
upon reference to the following detailed description and attached drawings. All references disclosed herein are hereby incorporated by reference in their entirety as if each was incorporated individually.<br>
In   another   aspect   of  the   present   invention,   the   applicants   have<br>
15     unexpectedly identified a series of novel repeating sequence elements in the 5' end of<br>
the   gene   encoding   0772P.      Therefore,   the   present   invention   provides   0772P<br>
polypeptides having structures represented by Xn-Y, wherein X comprises a sequence<br>
having at least 50% identity, preferably at least 70% identity, and more preferably at<br>
least 90% identity with an 0772P repeat sequence set forth in SEQ ID NO: 596. Y will<br>
20     typically comprise a sequence having at least  80Â°A identity, preferably at least 90%<br>
identity and more preferably at least 95% identity with the 0772P constant region<br>
sequence set forth in SEQ ID NO: 594. According to this embodiment, n will generally<br>
be an integer from 1 to 35, preferably an integer from 15 to 25, and X can be the same<br>
or different.<br>
25	In one preferred embodiment, X comprises a sequence selected from the<br>
group consisting of any one of SEQ ID NOs: 574-593 and Y comprises the sequence set forth in SEQ ID NO: 594.<br>
In another preferred embodiment,  an illustrative 0772P polypeptide<br>
comprises the sequence set forth in SEQ ID NO: 595, containing 20 repeating sequence<br>
30     elements (i.e., X2o) wherein the X elements are arranged in the following order (moving<br>
from N-terminal to C-terminal in the 0772P repeat region): SEQ ID NO: 574 - SEQ ID<br><br>
NO: 575 - SEQ ID NO: 576 - SEQ ID NO: 577 - SEQ ID NO: 578 - SEQ ID NO: 579 -SEQ ID NO: 580 - SEQ ID NO: 581 - SEQ ID NO: 582 - SEQ ID NO: 583 - SEQ ID NO: 584 - SEQ ID NO: 585 - SEQ ID NO: 586 - SEQ ID NO: 587 - SEQ ID NO: 588 -SEQ ID NO: 589 - SEQ ID NO: 590 - SEQ ID NO: 591 - SEQ ID NO: 592 - SEQ ID 5     NO: 593.<br>
According to another aspect of the present invention, an 01129 polynucleotide is provided having the structure 3fn-Y, wherein X comprises an 0772P repeat sequence element selected from the group consisting of any one of SEQ ID NOs: 512-540, 542-546 and 548-567. Y will generally comprise a sequence having at least 10 80% identity, preferably at least 90% identity, and more preferably at least 95% identity with the 0772P constant region sequence set forth in SEQ ID NO: 568. In this embodiment, n is typically an integer from 1 to 35, preferably from 15 to 25 and X can be the same or different<br>
In another embodiment, an illustrative 0772P polynucleotide comprises 15     the sequence set forth in SEQ ID NO: 569, containing 20 repeating sequence elements (i.e.,X20).<br>
According to another aspect of the present invention, 0772 polypeptides<br>
are provided comprising at least an antibody epitope sequence set forth in any one of<br>
SEQ ID NOs: 490-511.<br>
20	According to another aspect of the present invention, 08E polypeptides<br>
are provided comprising at least an antibody epitope sequence set forth in any one of SEQ ID NOs: 394-415.<br>
BRIEF DESCRIPTION OF THE SEQUENCE IDENTIFIERS AND DRAWINGS<br>
SEQ ID NO:l-71 are ovarian carcinoma antigen polynucleotides shown 25     in Figures 1A-IS.<br>
SEQ ID NO:72-74 are ovarian carcinoma antigen polynucleotides shown in Figures 2A-2C.<br>
SEQ ID NO:75 is the ovarian carcinoma polynucleotide 3g (Figure 4).<br>
SEQ ID NO:76 is the ovarian carcinoma polynucleotide 3f (Figure 5).<br>
30	SEQ ID NO:77 is the ovarian carcinoma polynucleotide 6b (Figure 6).<br><br>
SEQ ID NO:78 is the ovarian carcinoma polynucleotide 8e (Figure 7A).<br>
SEQ ID NO:79 is the ovarian carcinoma polynucleotide 8h (Figure 7B).<br>
SEQ ID NO:80 is the ovarian carcinoma polynucleotide 12e (Figure 8).<br>
SEQ ID NO:81 is the ovarian carcinoma polynucleotide 12h (Figure 9).<br>
5	SEQ  ID NO:82-310  are  ovarian  carcinoma  antigen polynucleotides<br>
shown in Figures 15A-15EEE.<br>
SEQ  ID  NO:311   is  a  full   length   sequence   of ovarian  carcinoma polynucleotide 0112V.<br>
SEQ ID NO:312 is the 0772P amino acid sequence.<br>
10	SEQ IDNO:313-384 are ovarian carcinoma antigen polynucleotides.<br>
SEQ ID NO:385 represents the cDHA sequence of a form of the clone 0772P, designated 21-013.<br>
SEQ ID NO:386 represents the cDNA sequence of a form of the clone<br>
0772P, designated 21003.<br>
15	SEQ ID NO:387 represents the cDNA sequence of a form of the clone<br>
0772P, designated 21008.<br>
SEQ ID NOs:388 is the amino acid sequence corresponding to SEQ ID NO:385.<br>
SEQ ID NOs:389 is the amino acid sequence corresponding to SEQ ID 20    NO:386.SEQ ID NOs:390 is the amino acid sequence corresponding to  SEQ ID<br>
NO:387.<br>
SEQ  ID  NO:391   is  a full  length   sequence   of ovarian  carcinoma polynucleotide 08E.<br>
SEQ ID NO:392-393 are protein sequences encoded by 08E.<br>
25	SEQ ID NO:394-415 are peptide sequences corresponding to the OE8<br>
antibody epitopes.<br>
SEQ ID NO:416-435 are potential HLA-A2 10-mer binding peptides predicted using the full length open-reading frame from OE8.<br>
SEQ ID NO:436-455  are potential HLA-A2 9-mer binding peptides 30    predicted using the full length open-reading frame irom OE8.<br><br>
SEQ ID NO:456 is a truncated nucleotide sequence of the full length Genbank sequence showing homology to 0772p<br>
SEQ ID NO:457 is the full length Genbank sequence showing significant homology to 0112?<br>
5	SEQ ID NO:458 is a protein encoding a truncated version of the full<br>
length Genbank sequence showing homology to O772P<br>
SEQ IDNO:459 is the full length protein sequence from Genbank showing significant homology to the protein sequence for 0772p<br>
SEQ   ID  NO:460  encodes  a  unique N-terminal  portion  of 0772P 10     contained in residues 1-70.<br>
SEQ ID NO':461 contains  unique sequence and encodes residues 1-313 ofSEQ ID NO:456.<br>
SEQ ID NO:46Z is the nypotneticai sequence for clone 0772P.<br>
SEQ ID NO:463 is the cDNA sequence for clone FLJ14303.<br>
15	SEQ ID NO:464 is a partial cDNA sequence for clone 0772P.<br>
SEQ ID NO:465 is a partial cDNA Sequence for clone 0772P.<br>
SEQ ID NO:466 is a partial cDNA Sequence for clone 0772P.<br>
SEQ ID NO:467 is a partial cDNA sequence for clone 0772P.<br>
SEQ ID NO:468 is a partial cDNA sequence for clone 0112?.<br>
20	SEQ ID NO:469 is a partial cDNA sequence for clone 0112?.<br>
SEQ ID NO:470 is a partial cDNA sequence for done 0772P.<br>
SEQ ID NO:471 is a partial cDNA sequence for clone 0772P.<br>
SEQ ID NO:472 is a partial cDNA sequence for clone 0772P.<br>
SEQ ID NO:473 is a partial cDNA sequence for clone 0772P.<br>
25	SEQ ID NO:474 is a partial cDNA sequence for clone 0772P.<br>
SEQ ID NO:475 is a partial cDNA sequence for clone 0772P. SEQ ID NO:476 is a partial cDNA sequence for clone 0772P. SEQ ID NO:477 represents the novel .5'-end of the ovarian tumor antigen 0772P.<br>
30	SEQ ID NO:478  is the amino acid sequence encoded by SEQ ID<br>
NO:462.<br><br>
SEQ ID NO:479 is the amino acid sequence encoded by SEQ ID NO:463.<br>
SEQ ID NO:480 is a partial amino acid sequence encoded by SEQ ID<br>
NO:472.<br>
5	SEQ ID NO:481 is a partial amino acid sequence encoded by a possible<br>
open reading frame of SEQ ID NO:471.<br>
SEQ ID NO:482 is a partial amino acid sequence encoded by a second possible open reading frame of SEQ ID NO;471.<br>
SEQ ID NO:483 is a partial amino acid sequence encoded by SEQ ID 10    NO:467.<br>
SEQ ID NO:484 is a partial amino acid sequence encoded by a possible open reading frame of SEQ ID NO:466.<br>
SEQ ID NO:485 is a partial amino acid sequence encoded by a second<br>
possible open reading frame of SEQ ID NO:466.<br>
15	SEQ ID NO:486 is a partial amino acid sequence encoded by SEQ ID<br>
NO:465.<br>
SEQ ID NO:487 is a partial amino acid sequence encoded by SEQ ID NO:464.<br>
SEQ   ID   NO:488   represents   the   extracellular,   transmembrane   and 20     cytoplasmic regions of 0772P.<br>
SEQ ID NO:489 represents the predicted extracellular domain of 0772P. SEQ ID NO:490 represents the amino acid sequence of peptide #2 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:491 represents the amino acid sequence of peptide #6 which 25     corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:492 represents the amino acid sequence of peptide #7 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:493 represents the amino acid sequence of peptide #8 which<br>
corresponds to an 0772P specific antibody epitope.<br>
30	SEQ ID NO:494 represents the amino acid sequence of peptide #9 which<br>
corresponds to an 0772P specific antibody epitope.<br><br>
SEQ ID NO:495 represents the amino acid sequence of peptide #11 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:496 represents  the amino acid sequence of peptide #13<br>
which corresponds to an 0772P specific antibody epitope.<br>
5	SEQ ID NO:497 represents the amino acid sequence of peptide #22<br>
which corresponds to an 0772P specific antibody epitope.<br>
  SEQ ID NO:498 represents the amino acid sequence of peptide #24 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:499 represents the amino acid sequence of peptide #27 10     which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO.500 represents the amino acid sequence of peptide #40 which corresponds to an 0772P specific antibody epitope.<br>
,    SEQ ID NO:501 represents the amino acid sequence of peptide #41<br>
which corresponds to an 0112V specific antibody epitope.<br>
15	SEQ ID NO:502 represents the amino acid sequence of peptide #47<br>
which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:503 represents the amino acid sequence of peptide #50 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:504 represents tire amino acid sequence of peptide #51 20     which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:505 represents the amino acid sequence of peptide #52 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:506 represents the amino acid sequence of peptide #53<br>
which corresponds to an 0112? specific antibody epitope.<br>
25	SEQ ID NO:507 represents the amino acid sequence of peptide #58<br>
which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:5Q8 represents the amino acid sequence of peptide #59 which corresponds to an 0772P specific antibody epitope.<br>
SEQ ID NO:509 represents the amino acid sequence of peptide #60 30     which corresponds to an 0112? specific antibody epitope.<br><br>
SEQ ID NO:510 represents the amino acid sequence of peptide #61 which corresponds to an 0117? specific antibody epitope.<br>
SEQ ID NO:511 represents the amino acid sequence of peptide #71<br>
which corresponds to an 0772P specific antibody epitope.<br>
5	SEQ ID NO:512 (0772P repeatl) represents an example of a cDNA<br>
sequence corresponding to repeat number 21 from the 5' variable region of 0772P.<br>
SEQ ID NO:513 (0772P repeat?.) represents an example of a cDNA sequence corresponding to repeat number 20 from the 5' variable region of 0772P.<br>
SEQ ID NO:514 (0772P repeats) represents an example of a cDNA 10     sequence corresponding to repeat number 19 from the 5' variable region of 0772P.<br>
SEQ ID NO:515 (0772P repeats) represents an example of a cDNA sequence corresponding to repeat number 18 from the 5' variable region of 0772P.<br>
SEQ ID NO:5I6 (0772P repeats') represents an example of a cDNA<br>
sequence corresponding to repeat number 17 from the 5' variable region of 0772P.<br>
15      -	SEQ ID NO:5I7  (HB  repeatl) represents  an  example  of a  cDNA<br>
sequence corresponding to repeat number 21 from the 5' variable region of 0772P.<br>
SEQ ID NO:518  (HB repeat2) represents' an  example  of a  cDNA sequence corresponding to repeat number 20 from the 5' variable region of 0772P.<br>
SEQ ID NO:519  (HB  repeat3) represents  an  example  of a cDNA 20     sequence corresponding to repeat number 19 from the 5' variable region of 0772P.<br>
SEQ ID NO:520  (HB  repeat4) represents  an  example of a cDNA sequence corresponding to repeat number 18 from the 5' variable region of 0772P.<br>
SEQ ID NO:521   (HB repeat5) represents  an  example of a  cDNA<br>
sequence corresponding to repeat number 17 from the 5' variable region of 0772P.<br>
25	SEQ ID NO:522 (HB repeat6 5'-end) represents an example of a cDNA<br>
sequence corresponding to repeat number 1'5 from the 5' variable region of 0772P.<br>
SEQ ID NO:523 (1043400.1 repeatl) represents.an example of a cDNA sequence corresponding to repeat number 9 from the 5' variable region of 0772P.<br>
SEQ ID NO:524 (1043400.1 repeat2) represents an example of a cDNA 30     sequence corresponding to repeat number 10 from the 5' variable region of 0772P.<br><br>
SEQ ID NO:525 (1043400.1 repeat3) represents an example of a cDNA sequence corresponding to repeat number 10/11 from the 5' variable region of 0772P.<br>
SEQ ID NO:526 (1043400.1 repeat4) represents an example of a cDNA<br>
sequence corresponding to repeat number 11 from the 5' variable region of 0112V.<br>
5	SEQ ID NO:527 (1043400.1 repeat5) represents an example of a cDNA<br>
sequence corresponding to repeat number 14 from the 5' variable region of 0772P.<br>
SEQ ID NO:528 (1043400.1 repeat6) represents an example of a cDNA sequence corresponding to repeat number 17 from the 5' variable region of 0772P.<br>
SEQ ID NO:529 (1043400.3 repeatl) represents an example of a cDNA 10     sequence corresponding to repeat number 20 from the 5' variable region of 0772P.<br>
SEQ ID NO:530 (1043400.3 repeat2) represents an example of a cDNA sequence corresponding to repeat number 21 from the 5' variable region of 0772P.<br>
SEQ IDNO:531 (1043400.5 repeatl) represents an example of a cDNA<br>
sequence corresponding to repeat number 8 from the 5' variable region of 0772P.<br>
15	SEQ ID NO:532 (1043400.5 repeat2) represents an example of a cDNA<br>
sequence corresponding to repeat number 9 from the 5' variable region of 0772P, in addition containing intron sequence.<br>
SEQ ID NO:533 (1043400.5 repeat2) represents an example of a cDNA<br>
sequence corresponding to repeat number 9 from the 5' variable region of 0112?.<br>
20	SEQ ID NO:534 (1043400.8 repeatl) represents an example of a cDNA<br>
sequence corresponding to repeat number 17 from the 5' variable region of 0772P.<br>
SEQ ID NO:535 (1043400.8 repeat2) represents an example of a cDNA sequence corresponding to repeat number 18 from the 5' variable region of 0772P.<br>
SEQ ID NO:536 (1043400.8 repeat3) represents an example of a cDNA 25     sequence corresponding to repeat number 19 from the 5' variable region of 0772P.<br>
SEQ ID NO:537 (1043400.9 repeatl) represents an example of a cDNA sequence corresponding to repeat number 4 from the 5' variable region of 0772P.<br>
SEQ ID NO:538 (1043400.9 repeat2) represents an example of a cDNA<br>
sequence corresponding to repeat number 5 from the 5' variable region of 0772P.<br>
30	SEQ ID NO:539 (1043400.9 repeat3) represents an example of a cDNA<br>
sequence corresponding to repeat number 7 from the 5' variable region of 0772P.<br><br>
SEQ ID NO:540 (1043400.9 repeat4) represents an example of a cDNA sequence corresponding to repeat number 8 from the 5' variable region of 0772P.<br>
SEQ ID NO:541 (1043400.11 repeatl) represents an example of a cDNA<br>
sequence corresponding to repeat number 1 from the 5' variable region of 01127.<br>
5	SEQ ID NO:542 (1043400.11 repeat2) represents an example of a cDNA<br>
sequence corresponding to repeat number 2 from the 5' variable region of 0772P.<br>
	SEQ ID NO:543 (1043400.11 repeat3) represents an example of a cDNA<br>
sequence corresponding to repeat number 3 from the 5' variable region of 0722P.<br>
SEQ ID NO:544 (1043400.11 repeat4) represents an example of a cDNA 10     sequence corresponding to repeat number 11 from the 5' variable region of 0772P.<br>
:.	SEQ ID NO:545 (1043400.11 repeat5) represents an example of a cDNA<br>
sequence corresponding to repeat number 12 from the 5' variable region of 0112V.<br>
SEQ ID 1<546 repeatl represents an example of a cdna></546>
sequence corresponding to repeat number 20 from the 5' variable region of 0772P.<br>
15	SEQ ID NO:547  (PB repeatA) represents an  example of a  cDNA<br>
sequence corresponding to repeat number 1 from the 5' variable region of 0772P.<br>
SEQ ID NO:548  (PB repeatB) represents an  example of a cDNA sequence corresponding to repeat number 2 from the 5' variable region of 0772P.<br>
SEQ ID NO:549  (PB repeatE) represents  an  example  of a cDNA 20     sequence corresponding to repeat number 3 from the 5' variable region of 0772P.<br>
SEQ ID NO:550  (PB repeatG) represents an  example  of a  cDNA sequence corresponding to repeat number 4 from the 5' variable region of 0772P.<br>
SEQ ID NO:551   (PB  repeatC) represents an  example  of a cDNA<br>
sequence corresponding to'repeat number 4 from the 5' variable region of 0112V.<br>
25	SEQ ID 140:552  (PB repeatH) represents an  example  of a  cDNA<br>
sequence corresponding to repeat number 6 from the 55 variable region of 0772P.<br>
SEQ ID  NO:553   (PB  repeatJ)  represents  an  example  of a  cDNA sequence corresponding to repeat number 7 from the 5' variable region of 0772P.<br>
SEQ ID NO:554  (PB repeatK) represents  an  example  of a  cDNA 30     sequence corresponding to repeat number 8 from the 5' variable region of 0772P.<br><br>
SEQ ID NO:555  (PB repeat) represents  an example  of a cDNA sequence corresponding to repeat number 9 front the 5' variable region of 0772P.<br>
SEQ ID NO:556 (PB repeat!) represents  an example of a cDNA<br>
sequence corresponding to repeat number 10 from the 5' variable region of 0772P.<br>
5	SEQ ID NO:557  (PB repeatM) represents  an example of a cDNA<br>
sequence corresponding to repeat number 11 from the 5' variable region of 0772P.<br>
SEQ ID NO:558 (PB repeat!?) represents an example of a cDNA sequence corresponding to repeat number 12 from the 5' variable region of 0772P.<br>
SEQ ID NO:559 (PB repeatS.5) represents an example of a cDNA 10     sequence corresponding to repeat number 13 from the 5' variable region of 0772P.<br>
SEQ  ID NO:560  (PB  repeats)  represents  an  example  of a  cDNA sequence corresponding to repeat number 14 from the 5' variable region of 0772P.<br>
SEQ ID NO:S61  (PB repeat7} represents  an  examp/e  of a cDNA<br>
sequence corresponding to repeat number 15 from the 5' variable region of 0772P.<br>
15	SEQ  ID NO:562  (PB repeat6)  represents  an  example  of a  cDNA<br>
Sequence corresponding to repeat number 16 from the 5' variable region of 0772P.<br>
SEQ  ID NO:563   (PB  repeat5) represents  an  example  of a cDNA Sequence corresponding to repeat number 17 from the 5' variable region of 0772P.<br>
SEQ  ID  NO:564  (PB  repeat4) represents  an  example  of a  cDNA 20     Sequence corresponding to repeat number 18 from the 5' variable region of 0772P.<br>
SEQ  ID NO:565  (PB repeat.3)  represents  an  example  of a  cDNA sequence corresponding to repeat number 19 from the 5' variable region of 0772P.<br>
SEQ  ID NO:566  (PB repeat2) represents  an  example  of a  cDNA<br>
Sequence corresponding to repeat number 20 from the 5' variable region of 0772P.<br>
25	SEQ  ID NO:567  (PB  repeatl) represents  an  example  of a cDNA<br>
sequence corresponding to repeat number 21 from the 5' variable region of 0772P.<br>
SEQ ID NO:568 represents the cDNA sequence form the 3' constant region.<br>
SEQ ID NO:569 represents a cDNA sequence containing the consensus 30    sequences of the 21 repeats, the 3' constant region and the 3' untranslated region.<br><br>
SEQ ID NO:570 represents the cDNA sequence of the consensus repeat sequence.<br>
SEQ ID NO:571 represents the consensus amino acid sequence of one<br>
potential open reading frame of repeat number 1 from the 5' variable region of 0772P.<br>
5	SEQ ID NO:572 represents the consensus amino acid sequence of a<br>
second potential open reading frame of repeat number 1 from the 5' variable region of 0772P.<br>
SEQ ID NO:573 represents the consensus amino acid sequence of a third<br>
potential open reading frame of repeat number 1 from the 5' variable region of 0772P.<br>
10	SEQ ID NO:574 represents the consensus amino acid sequence of repeat<br>
number 2 from the.5' variable region of 0772P.<br>
SEQ: ID NO:575 represents the consensus amino acid sequence of repeat number 3 from the 5' variable region of 0772P.<br>
SEQ ID NO:576 represents the consensus amino acid sequence of repeat 15     number 4 from the 5' variable region of 0772P.<br>
SEQ ID NO:577 represents the consensus amino acid sequence of repeat number 5 from the 5' variable region of 0772P.<br>
.	SEQ ID NO:578 represents the consensus amino acid sequence of repeat<br>
number 6 from the 5' variable region of 0772P.<br>
20	SEQ ID N.O:579 represents the consensus amino acid sequence of repeat<br>
number 7 from the 5' variable region of 0772P.<br>
SEQ ID NO:580 represents the consensus amino acid sequence of repeat number 8 from the 5' variable region of 0772P.<br>
SEQ ID NO:581 represents the consensus amino acid sequence of repeat 25     number 9 from the 5' variable region of 0772P.<br>
SEQ ID NO:582 represents the consensus amino acid sequence of repeat number 10 from the 5' variable region of 0772P.<br>
SEQ ID NO:583 represents the consensus amino acid sequence of repeat<br>
number 11 from the 5' variable region of 0772P.<br>
30	SEQ ID NO:584 represents the consensus amino acid sequence of repeat<br>
number 12 from the 5' variable region of 0772P.<br><br>
SEQ ID NO:585 represents the consensus amino acid sequence of repeat number 13 from the 5' variable region of 0772P.<br>
SEQ ID NO:586 represents the consensus amino acid sequence of repeat<br>
number 14 from the 5' variable region of 0772P.<br>
5	SEQ ID NO:5S7 represents the consensus amino,acid sequence of repeat<br>
number 15 from the 5' variable region of 0772P<br>
SEQ ID NO:58S represents the consensus amino acid sequence of repeat number 16 from the 5' variable region of 0772P.<br>
SEQ ID NO:589 represents the consensus amino acid sequence of repeat 10     number 17 from the 5' variable region of 0112?.<br>
SEQ ID NO:590 represents the consensus amino ,acid sequence of repeat number 18 from the 5' variable region of 0112?.<br>
SEQ ID NO-591 represents the amino acid sequence of repeat<br>
number 19 from the 5' variable region of 0772P.<br>
15	SEQ ID NO:592 represents the consensus amino acid sequence of repeat<br>
number 20 from the 5' variable region of 0772P.<br>
SEQ ID NO:593 represents the consensus amino acid sequence of repeat number 21 from the 5' variable region of 0772P.<br>
SEQ ID NO:594 represents the amino acid sequence of the 3 constant 20     region.<br>
SEQ ID NO:595 represents an amino acid sequence containing the consensus sequences of the 21 repeats and the 3' constant region.<br>
SEQ ID NO:596 represents the amino acid sequence of the consensus<br>
repeat sequence.<br>
25	Figures    1A-1S   (SEQ   ID   NO:l-71)   depict   partial   sequences   of<br>
polynucleotides encoding representative secreted ovarian carcinoma antigens.<br>
Figures 2A-2C depict full insert sequences for three of the clones of Figure 1.   Figure 2A shows the sequence designated 07E (11731; SEQ ID NO:72), Figure 2B shows the sequence designated 09E (11785; SEQ ID NO:73) and Figure 2C 30     shows the sequence designated 08E (13695; SEQ IP NO:74).<br><br>
Figure 3 presents results of microarray expression analysis of the ovarian carcinoma Sequence designated OSE.<br>
Figure 4 presents a partial sequence of a polynucleotide (designated 3g; SEQ ID No:75) encoding an ovarian carcinoma sequence that is a splice fusion 5     between the human T-cell leukemia virus type I oncoprotein TAX and osteonectin.<br>
Figure 5 presents the ovarian carcinoma polynucleotide designated 3f (SEQIDNO:76).<br>
Figure 6 presents the ovarian carcinoma polynucleotide designated 6b (SEQ IQ NO:77)<br>
^	Figures  7 A  and   7B  present the  ovarian  carcinoma polynucleotides<br>
designated 8 (SEQ ID NO:78) and 8h (SEQ ID NO:79).<br>
Figure 8 presents the ovarian carcinoma polynucleotide designated 12c (SEQ ID NO:80)<br>
Figure 9 presents the ovarian carcinoma polynucleotide designated 12h   -15     (SEQ ID NO:81)<br>
Figure 10 depicts results of microarray expression analysis of the ovarian carcinoma sequence designated 3f.<br>
Figure j 1 depicts results of microarray expression analysis of the ovarian carcinoma sequence designated 6b<br>
Figure 12 depicts results of microarray expression analysis of the ovarian carcinoma sequence designated 8e.<br>
Figure 13 depicts results of microarray expression analysis of the ovarian carcinoma sequence designated 12c.<br>
Figure 14 depicts results of microarray expression analysis of the ovarian 25     carcinoma seqiaence designated 12h.<br>
Figures 15A-15EEE depict partial sequences of additional polynucleotides ecoding representative secreted ovarian carcinoma antigens (SEQ ID NO:82-310).<br>
figure 16 is a diagram illustrating the location of various partial OSE 30     sequences within the full length sequnence.<br><br>
Figure 17 is a graph illustrating the results of epitope mapping studies on 08E protein.<br>
Figure  18  is graph  of a fluorescence activated cell sorting (FACS)<br>
analysis of 08E cell surface expression.<br>
5	Figure 19 is graph of a FACS analysis of 08E cell surface expression.<br>
Figure 20 shows FACS analysis results for OSE transfected HEK293 cells demonstrating cell surface expression of 08E<br>
Figure 21 shows FACS analysis results for SKBR3 breast tumor cells<br>
demonstrating cell surface expression of OSE.<br>
10	Figure 22 shows OSE expression in HEK 293 cells.    The cells were<br>
probed with anti-08E rabbit polyclonal antisera #2333L.<br>
Figure 23 shows the ELISA analysis 0f anti-OSE rabbit sera.<br>
Figure 24shows theLISA of affinity purified rabbit anti-OSE<br>
polyclonal antibody.<br>
5	Figure 25 is a graph determining antibody internalization of anti-OSE<br>
mAb showing that mAbs against amino acids 61-8() induces ligand internalization.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
As   noted   above,   the   present   invention   is   generally   directed   to compositions and methods for the therapy of cancer, such as ovarian cancer.    The" 20     compositions described herein may include immunogenic polypeptides, polynucleotides encoding   such  polypeptides,   binding   agents   su,ch   as   antibodies   that   bind   to   a polypeptide, antigen presenting cells (APCs) and/or immune system cells {e.g., T cells). Polypeptides of the present invention generally comprise at least an immunogenic portion of an ovarian carcinoma protein or a variant thereof   Certain 25     ovarian carcinoma proteins have been identified using an immunoassay technique, and are referred to herein as ovarian carcinoma antigens. An "ovarian carcinoma antigen" is a protein that is expressed by ovarian rumor cells (preferably human cells) at a level that is at least two fold higher than the level in normal ovarian cells.    Certain ovarian carcinoma antigens react detectably (within an immunoassay, such as an ELISA or 30     Western blot) with antisera generated against serum, from an immunodeficient animal<br><br>
implanted with a human ovarian tumor. Such ovarian carcinoma antigens are shed or secreted from an ovarian tumor into the sera of the immunodeficient animal. Accordingly, certain ovarian .carcinoma antigens provided herein are secreted antigens. Certain nucleic acid sequences of the subject invention generally comprise a DNA or 5 RNA sequence that encodes all or a portion of such a polypeptide, or that is complementary to such a sequence.<br>
The present invention further provides ovarian carcinoma sequences that are identified using techniques to evaluate altered expression within an ovarian tumor. Such sequences may be poiynucleotide or protein sequences.    Ovarian carcinoma 10     sequences are generally expressed in an ovarian tumor at a level that is at least two fold, and preferably at least five fold, greater than the level of expression in normal ovarian tissue, as determined using a representative assay provided herein.    Certain partial ovarian carcinoma polynucleotide sequences are. presented herein. Proteins encoded by genes comprising such polynucleotide sequences (or complements thereof) are also 15     considered ovarian carcinoma proteins.<br>
Antibodies are generally immune system proteins, or antigen-binding fragments thereof, that are capable of binding to at least a portion of an ovarian carcinoma polypeptide as described herein. T cells that may be employed within the compositions provided herein are generally T cells (e.g., CD4+ and/or CD8+) that are 20 specific for such a polypeptide. Certain methods described herein further employ antigen-presenting cells (such as dendritic cells or macrophages) that express an ovarian carcinoma polypeptide as provided herein.<br>
.Ovarian Carcinoma Polynucleotides<br>
Any polynucleotide that encodes an ovarian carcinoma protein  or a<br>
25 portion or other variant thereof as described herein is encompassed by the present invention. Preferred polynucleotides comprise at least 15 consecutive nucleotides, preferably at least 30 consecutive nucleotides, and more preferably at least 45 consecutive nucleotides, that encode a portion of an ovarian carcinoma protein. More preferably, a polynucleotide encodes an immunogenic portion of an ovarian carcinoma<br>
30    protein, such as an ovarian carcinoma antigen.   Polynucleotides complementary to any<br><br>
such sequences are also encompassed by the present invention. Polynucleotides may be single-stranded (coding or antisense) or double-stranded, and may be DNA (genomic, cDNA or synthetic) or RA molecules.   Additional coding or non-coding sequences may, but need not, be present within a polynucleotide of the present invention, and a 5     polynucleotide may, but need not, be linked to other molecules and/or support materials. Polynucleotides may comprise a native sequence {i.e., an endogenous sequence that encodes an ovarian carcinoma protein or a portion thereof) or may comprise a variant of such a sequence.   Polynucleotide variants may contain one or more substitutions, additions, deletions and/or insertions such that the immunogenicity 10     of the encoded polypeptide is not diminished, relative to a native ovarian carcinoma protein. The effect on the immunogenicity of the encoded polypeptide may generally be assessed as described herein.   Variants preferably exhibit at least about 70% identity, more preferably at least about 80% identity and most preferably at least about 90% identity to a polynucleotide sequence that encodes a native ovarian carcinoma protein or 15     a portion thereof.<br>
The percent identity for two polynucleotide or polypeptide sequences may be readily determined by comparing sequences using computer algorithms well known to those of ordinary skill in the art, such as Megalign, using default parameters. Comparisons   between  two  sequences  are  typically performed   by  comparing  the 20     sequences over a comparison window to identify and compare local regions of sequence similarity. A "comparison window" as used herein, refers to a segment of at least about 20 contiguous positions, usually 30 to about 75, or 40 to about 50, in which a sequence may be compared to a reference sequence of the same number of contiguous positions after the two sequences are optimally aligned.    Optimal alignment of sequences for 25    omparison may be conducted,  for example,  using the Megalign program in the Lasergene suite of bioinformatics software (DNASTAR, Inc., Madison, WI), using default parameters.   Preferably, the percentage of sequence identity is determined by comparing two optimally aligned sequences over a window of comparison of at least 20 positions, wherein the portion of the polynucleotide or polypeptide sequence in the<br>
30     window may comprise additions or deletions (i.e., gaps) of 20 % or.less, usually 5 to 15 %, or 10 to 12%, relative to the reference sequence (which does not contain additions or<br><br>
deletions). The percent identity may be calculated by determining the number of positions at which the identical nucleic acid bases or amino acid residue occurs in both sequences to yield the number of matched positions, dividing the-number of matched positions by the total number of positions in the reference sequence (i.e., the window 5    size) and multiplying the results by 100 to yield the percentage of sequence identity.<br>
Variants may also, or alternatively, be substantially homologous to a  native gene, or a portion or complement thereof,    Such polynucleotide variants are capable of hybridizing under moderately stringent conditions to a naturally occurring DNA sequence encoding a native ovarian carcinoma protein (or a complementary 0     sequence).  Suitable moderately stringent conditions include prewashing in a solution of 5 X SSC, 0.5% SDS3. 1.0 mM EDTA (pH 8.0); hybridizing at 50Â°C-65Â°C, 5 X SSC, overnight; followed by washing twice at 65Â°C for 20 minutes with each of 2X, 0.5X and :0.2X SSC containing 0. J % SDS.<br>
It will be appreciated by those of ordinary skill in the art that, as a result i     of the degeneracy of the genetic Code, there are many nucleotide sequences that encode a polypeptide as described herein.     Some of these polynucleotides bear minimal homology to the nucleotide sequence of any native gene.  Nonetheless, polynucleotides that vary due to differences in codon usage are specifically contemplated by the present ,   invention.     Further, alleles of the genes comprising the polynucleotide sequences provided herein are within the scope of the present invention.   Allelesare endogenous genes that are altered as a result of one or more mutations, such as deletions, additions and/or substitutions of nucleotides. The resulting raRNA and protein may, but need not, have an altered  structure or function.     Alleles  may be identified using  standard techniques (such as hybridization, amplification and/or database sequence comparison). Polynucleotides may be prepared using any of a variety of techniques. For example, an ovarian carcinoma polynucleotide may be identified, as described in more detail below, by screening a late passage ovarian tumor expression library with antisera generated against sera of immunocompetent mice after injection of such mice with sera from SCID mice implanted with late passage ovarian tumors.    Ovarian carcinoma polynucleotides may also be identified using any of a variety of techniques designed to evaluate differential gene expression.   Alternatively, polynucleotides may<br><br>
be amplified from cDNA prepared from ovarian tumor cells.  Such polynucleotides may<br>
be amplified via polymerase chain reaction (PCR). For this approach, sequence-specific<br>
primers  may be  designed  based  on the  sequences provided herein,  and may be<br>
purchased or synthesized.<br>
5	An amplified portion may be used to isolate a full length gene from a<br>
suitable library (e.g., an ovarian carcinoma cDMA library) using well known techniques. Within such techniques, a library (cDNAor genomic) is screened using one or more polynucleotide probes or primers suitable for amplification. Preferably, a library is size-selected to include larger molecules. Random primed libraries may also be preferred for 10 identifying 5' and upstream regions of genes. Genomic libraries are preferred for obtaining introns and extending 5' sequences.<br>
For hybridization techniques, a partial sequence may be labeled (e.g., by nick-translation or end-labeling with 32P) using well known techniques.  A bacterial or bacteriophage library is then screened by hybridizing filters containing denatured 15     bacterial colonies (or lawns containing phage plaques) with the labeled probe (see Sambrook et ah, Molecular Cloning:    A Laboratory Manual,  Cold Spring Harbor Laboratories, Cold Spring Harbor, NY, 1989).    Hybridizing colonies or plaques are selected and expanded, and the DNA is isolated for further analysis. cDNA clones may be analyzed to determine the amount of additional sequence by, for example, PCR using 20     a primer from the partial sequence arid a primer from the vector. Restriction maps and partial sequences may be generated to identify one or more overlapping clones.   The complete sequence may then be determined using standard techniques, which may involve generating a series of deletion clones.  The resulting overlapping sequences are then assembled into a single contiguous sequence. A frill length cDNA molecule can be 25     generated by ligating suitable fragments, using well known techniques.<br>
Alternatively, there are numerous amplification techniques for obtaining a frill length coding sequence from a partial cDNA sequence. Within such techniques, amplification is generally performed via PCR. Any of a variety of commercially available kits may be used to perform the amplification step. Primers may be designed 30 using, for example, software well known in the art. Primers are preferably 22-30 nucleotides in length, have a GC content of at least 50% and anneal to the target<br><br>
sequence at temperatures of about 68Â°C to 72Â°C. The amplified region may be sequenced as described above, and overlapping sequences assembled into a contiguous sequence.<br>
One such amplification technique is inverse PCR (see Triglia et al., Nucl. 5    Acids Res. 75:81=86, 1988), which uses restriction enzymes to generate a fragment in the known region of the gene. The fragment is then circularized by intramolecular ligation and used as a template for PCR with divergent primers derived from the known region. Within an alternative approach,  sequences adjacent to a partial  sequence may be retrieved by amplification with a primer to a linker sequence and a primer specific to a 10    known region.   The amplified sequences are typically subjected to a second round of amplification with the same linker primer and a second primer specific to the known region.     A variation on this procedure, which employs two primers that initiate extension  in  opposite  directions from the known sequence,  is  described in  WO 96/38591. Additional techniques include capture PCR (Lagerstrom et al., PCR Methods 15    Applic. 7:111-19, 1991) and walking PCR (Parker et al., Nucl. Acids. Res. 7.9:3055-60, 1991).  Other methods employing amplification may also be employed to obtain a full length cDNA sequence.<br>
In certain instances, it is possible to obtain a full length cDNA sequence<br>
by analysis of sequences provided in an expressed sequence tag (EST) database, such:as<br>
20    that available from  GenBank.     Searches for overlapping ESTs may generally be<br>
performed using well known programs (e.g., NCBI BLAST searches), and such ESTs<br>
may be used to generate a contiguous full length sequence.<br>
Certain nucleic acid sequences of cDNA molecules encoding portions of ovarian carcinoma antigens are provided in Figures 1A-1S (SEQ ID NOT to 71) and 15    Figures 15Ato 15EEE (SEQ IDNO:82 to 310). The sequences provided in Figures IA-1S appear to be novel.    For sequences in Figures  15A-15EEE,  database searches revealed matches having substantial identity.   These polynucleotides were isolated rby serological screening of an ovarian tumor cDNA expression library, using a technique designed to identify secreted tumor antigens.   Briefly, a late passage ovarian tumor 0    expression library was prepared from a SCID-derived human ovarian tumor (OV9334) in the vector X-screen (Novagen).    The sera used for screening were obtained by<br><br>
injecting immunocompetent mice with sera from SCID mice implanted with one late passage ovarian tumors. This technique permits the identification of cDNA molecules that encode immunogenic portions of secreted tumor antigens.<br>
The polynucleotides recited herein, as well as full length polynucleotides<br>
5 comprising such sequences, other portions of such full length polynucleotides, and<br>
sequences complementary to all or a portion of such full length molecules, are<br>
specifically encompassed by the present invention. It will be apparent to those of<br>
ordinary skill in the art that this technique can also be applied to the identification of<br>
antigens that are secreted from other types of tumors.<br>
10	Other nucleic acid sequences of cDNA molecules encoding portions of<br>
ovarian carcinoma proteins are provided in Figures 4-9 (SEQ ID NO:75-81), as well as<br>
SEQ ID NO:313-384.   These sequences were identified by screening a microarray of<br>
cDNAs for tumor-associated expression (i. e., expression that is at least five fold greater<br>
in an ovarian tumor than in normal ovarian tissue, as determined using a representative<br>
5     assay provided herein).  Such screens were performed using a Synteni microarray (Palo<br>
Alto, CA) according to the manufacturer's instructions (and essentially as described by<br>
Schena et al., Proc. Natl: Acad Sci. USA P3:10614-10619, 1996 and Heller et al., Proa<br>
Natl. Acad. Sci.  USA 94:2150-2155, 1997).    SEQ ID NO:311 and 391 provide full<br>
length sequences incorporating certain of these nucleic acid sequences.<br>
0	Any of a variety of well known techniques may be used to evaluate<br>
tumor-associated expression of a cDNA.   For example, hybridization techniques using labeled polynucleotide  probes  may  be  employed.     Alternatively,   or in   addition, amplification techniques such as real-time PCR may be used (see Gibson et al., Genome Research 6:995-1001, 1996; Heid et :al., Genome Research 6:986-994, 1996).   Real-     time PCR is a technique that evaluates the level of PCR product accumulation during amplification.    This technique permits' quantitative evaluation of mRNA levels in multiple samples. Briefly, mRNA is extracted from rumor and normal tissue and cDNA is prepared using standard techniques.  Real-time PCR may be performed, for example, using a Perkin Elmer/Applied Biosystems (Foster City, CA) 7700 Prism instrument. Matching primers and fluorescent probes may be designed for genes of interest using, for example, the primer express program provided by Perkin Elmer/Applied Biosystems<br><br>
(Foster City, CA).    Optimal concentrations of primers and probes may be initially determined by those of ordinary skill in the art, .and control (e.g., (3-actin) primers and probes may  be  obtained  commercially from,  for  example,  Perkin Elmer/Applied Biosystems (Foster City, CA). To quantitate the amount of specific RNA in a sample, a 5     standard curve is generated alongside using a plasmid containing the gene of interest. Standard curves may be generated using the Ct values determined in the real-time PCR, which are related to the initial cDNA concentration used in the assay.    Standard dilutions ranging from 10-106 copies of the gene of interest are generally sufficient.  In addition,   a  standard  curve  is  generated for the control  sequence.     Tins permits 10     standardization of initial RNA content of a tissue sample to the amount of control for comparison purposes.<br>
Polynucleotide variants may generally be prepared by any method known in the art, including chemical synthesis by, for example, solid phase phosphoramidite chemical synthesis. Modifications in a polynucleotide sequence may also be introduced 15     using standard mutagenesis techniques, such as oligonucleotide-directed site-specific mutagenesis (see Adelman et al., DNA 2:183, 1983).   Alternatively, RNA molecules may be generated by in vitro or in vivo transcription of DNA sequences encoding an ovarian carcinoma antigen, or portion thereof, provided that the DNA is incorporated into a vector with a suitable RNA polymerase promoter (such as T7 or SP6).   Certain 20     portions may be used to prepare an encoded polypeptide, as described herein.    In addition, or alternatively, a portion may be administered to a patient such that the encoded polypeptide is generated invivo.<br>
A portion of a sequence complementary to a coding sequence (i.e., an<br>
antisense polynucleotide) may also be used as a probe or to modulate gene expression.<br>
25     cDNA constructs that can be transcribed into anlisense RNA may also be introduced<br>
into cells or tissues to facilitate the production of antisense RNA.    An antisense<br>
polynucleotide may be used, as described herein, to inhibit expression of an ovarian<br>
carcinoma protein.    Antisense technology can be used to control gene expression<br>
through triple-helix formation, which compromises the ability of the double helix to<br>
30     open sufficiently for the binding of polymerase:;, transcription factors or regulatory<br>
molecules (see Gee et al., In Huber and Can", Molecular and Immunologic Approaches,<br><br>
Futura Publishing Co. (Mt. Kisco, NY; 1994).   Alternatively, an antisense molecule<br>
may be designed to hybridize with a control region of a gene (e.g., promoter, enhancer<br>
or transcription  initiation  site),  and  block transcription  of the  gene;  or to  block;<br>
translation by inhibiting binding of a transcript to ribosomes.<br>
5	Any polynucleotide may be further modified to increase stability in vivo.<br>
Possible modifications include, but are not limited to, the addition of flanking sequences at the 5' and/or 3' ends; the use of phosphorothioate or 2' O-methyl rather than phosphodiesterase linkages in the backbone; and/or the inclusion of nontraditional bases such as inosine, queosine and wybutosine, as well as acetyl- methyl-, thio- and 10     other modified forms of adenine, cytidine, guanine, thymine and uridine.<br>
Nucleotide sequences as described herein may be joined to a variety of<br>
other, nucleotide sequences  using established recombinant DNA  techniques.     For<br>
example, a polynucletide may be cloned into any of a variety of cloning vectors,<br>
including plasmids, phagemids, lambda phage derivatives and cosmids.    Vectors of<br>
15     particular intent include expression vectors, replication vectors, probe generation<br>
vectors and sequencing vectors. In general, a vector will contain an origin of replication<br>
functional in at jeast one organism, convenient restriction endonuclease sites and one or<br>
more selectable markers.  Other elements will depend upon the desired use, and will be<br>
apparent to those of ordinary skill in the art.<br>
20	Within certain embodiments, polynucleotides may be formulated so as to<br>
permit entry into a cell of a mammal, and expression therein.   Such formulations are particularly useful fortherapeutic purposes, as described below. Those of ordinary skill in the art  will  appreciate  that there are  many ways  to  achieve  expression  of a polynucleotide  a target cell  amdany suitable method may be employed.    For 25     example, a polynucleotide may be incorporated into a viral vector such as, but not limited to, adenovirus, adeno-associated virus, retrovirus, or vaccinia or other pox virus (e.g., avian pox virus).   Techniques for incorporating DNA into such vectors are well known to those (if ordinary skill in the art. A retroviral vector may additionally transfer or incorporate a gene for a selectable marker (to aid in the identification or selection of 30     transduced cells) and/or a targeting moiety, such as a gene that encodes a ligand for a receptor on a specjfic target cell, to render the vector target specific. Targeting may also<br><br>
be accomplished using an antibody, by methods known to those of ordinary skill in the art.<br>
Other formulations for therapeutic purposes include colloidal dispersion systems, such as macromolecule complexes, nanocapsules, microspheres, beads, and 5 lipid-based systems including oil-in-water emulsions, micelles, mixed micelles, and liposomes. A preferred colloidal system for use as a delivery vehicle in vitro and in vivo is a liposome {i.e., an artificial membrane vesicle). The preparation and use of such systems is well known in the art.<br>
Ovarian Carcinoma Polypeptides<br>
10	Within the context of the present invention, polypeptides may comprise<br>
at least an immunogenic portion of an ovarian carcinoma protein or a variant thereof, as<br>
described herein.    As noted above, certain ovarian carcinoma proteins are ovarian<br>
carcinoma antigens that are expressed by ovarian tumor cells and react detectably within<br>
an immunoassay (such as an ELISA) with antisera generated against serum from an<br>
15     immunodeiicient animal implanted with an ovarian tumor.   Other ovarian carcinoma<br>
proteins are encoded by ovarian carcinoma polynucleotides recited herein. Polypeptides<br>
as described herein may be of any length. Additional sequences derived from the native<br>
protein and/or heterologous sequences may be present, and such sequences may (but<br>
need not) possess further immunogenic or antigenic properties.<br>
20	An "immunogenic portion," as used herein is a portion of an antigen that<br>
is recognized (i. e., specifically bound) by a B-cell and/or T-cell surface antigen receptor. Such immunogenic portions generally comprise at least 5 amino acid residues, more preferably at least 10, and still more preferably  least 20 amino acid residues of an ovarian carcinoma protein or a variant thereof ' Preferred immunogenic portions are 25     encoded by cDNA molecules isolated as  described herein.    Further immunogenic portions may generally be identified using well known techniques, such as those summarized in Paul, Fundamental Immunology, 3rd ed., 243-247 (Raven Press, 1993) and references cited therein.'   Such techniques include screening polypeptides for the ability to react with ovarian carcinoma protein-specific antibodies, antisera and/or T-cell 30     lines or clones.  As used herein, antisera and antibodies are "ovarian carcinoma protein-<br><br>
specific" if they specifically bind to an ovarian carcinoma protein (i.e., they react with the ovarian carcinoma protein in an ELISA 01 other immunoassay, and do not react detectably with unrelated proteins).    Such antisera, antibodies and T cells may be prepared as described herein, and using well known techniques.    An immunogenic 5     portion of a native ovarian carcinoma protein is a portion that reacts with such antisera, antibodies and/or T-cells at a level mat is not substantially less than the reactivity of the full  length polypeptide (e.g.,  in an ELISA and/or T-cell reactivity assay).     Such immunogenic portions may react within such assays at a level that is similar to or greater than the reactivity of the full length protein.   Such screens may generally be 10    performed using methods well known to those of ordinary skill in the art, such as those described in Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988.  For example, a polypeptide may be immobilized on a solid support and contacted withpatient sera to allow binding of antibodies within the sera to the immobilized polypeptide.   Unbound sera may men be removed and bound antibodies 15     detected using, for example, ]25I-labeled Protein A.<br>
As noted above, a composition may comprise a variant of a native ovarian carcinoma protein.   A polypeptide "variant/' as used herein, is a polypeptide that differs from a native ovarian carcinoma protein in one or more substitutions, deletions, additions and/or insertions, such that the immunogenicity of the polypeptide 20     is not substantially diminished.   In other words, the ability of a variant to react with  , ovarian carcinoma protein-specific antisera may be enhanced or unchanged, relative to the native ovarian carcinoma protein, or may be diminished by less than 50%, and preferably less than 20%, relative to the native ovarian carcinoma protein.    Such variants may  generally be identified by modifying one of the  above polypeptide 25     sequences and  evaluating the reactivity of the modified polypeptide with ovarian carcinoma protein-specific antibodies or antisera as described herein. Preferred variants include those in which one or more portions, such as an N-terminal leader sbquence or transmembrane domain, have been removed.   Other preferred variants include variants in which a small portion (e.g., 1-30 amino acids, preferably 5-15 amino acids) has been 30    removed from the N- and/or C-terminal of the mature protein.<br><br>
Polypeptide  variants   preferably   exhibit   at  least  about   70%,   more<br>
preferably at least about 90% and most preferably at least about 95% identity to the<br>
native polypeptide.     Preferably,  a variant  contains conservative  substitutions.    A<br>
"conservative substitution" is one in which an amino acid is substituted for another<br>
5     amino acid that has similar properties, such that one skilled in the art of peptide<br>
chemistry   would   expect  the  secondary  structure  and hydropathic nature  of the<br>
polypeptide to be substantially unchanged.  Amino acid substitutions may generally be<br>
made  on   the   basis   of similarity   in  polarity,   charge,   solubility,   hydrophobicity,<br>
hydrophilicity and/or the amphipathic nature of 1he residues.   For example, negatively<br>
10     charged amino acids include aspartic acid and glutamic acid; positively charged amino<br>
acids include lysine and arginine; and amino acids with uncharged polar head groups<br>
having similar hydrophilicity values include leucine, isoleucine and valine; glycine and<br>
alanine; asparagine and glutamine; and serine, threonine, phenylalanine and tyrosine.<br>
Other groups of amino acids that may represent conservative changes include:  (1) ala,<br>
15     pro, gly, glu, asp, gin, asn, ser, thr; (2) cys, ser, tyr, thr; (3) val, ile, leu, met, ala, phe;<br>
(4) lys, arg, his; and (5) phe, tyr, trp, his.  A variant may also, or alternatively, contain<br>
nonconservative changes.    Variants may also (or alternatively) be modified by, for<br>
example, the deletion or addition of amino acids that have minimal influence on the<br>
immunogenicity, secondary structure and hydropathic nature of the polypeptide.<br>
10	As noted above, polypeptides may comprise a signal (or leader) sequence<br>
at the N-terminal end of the protein which co-translationally or post-translationaliy directs transfer of the protein. The polypeptide may also be conjugated to a linker or other sequence for ease of synthesis, purification or identification of the polypeptide {e.g., poly-His), or to enhance binding of the polypeptide to a solid support. For 5     example, a polypeptide may be conjugated to an immunoglobulin Fc region.<br>
Polypeptides may be prepared using any of a variety of. well known techniques. Recombinant polypeptides encoded by DNA sequences as described above may be readily prepared from the DNA sequences using any of a variety of expression vectors known to those of ordinary skill in the art. Expression may be achieved in any ) appropriate host cell that has been transformed or transfected with an expression vector containing a DNA molecule that encodes a recombinant polypeptide.  Suitable host cells<br><br>
include prokaryotes, yeast and higher eukaryotic cells. Preferably, the host cells employed are E. coli, yeast or a mammalian cell line such as COS or CHO. Supematants from suitable host/vector systems which secrete recombinant protein or polypeptide into culture media may be first concentrated using a commercially available 5 filter. Following concentration, the concentrate may be applied to a suitable purification matrix such as an affinity matrix or an ion exchange resin. Finally, one or more reverse phase HPLC steps can be employed to .further purify a recombinant polypeptide.<br>
Portions and other variants having fewer than about 100 amino acids, and generally fewer than about 50 amino acids, may also be generated by synthetic 10     means, using techniques well known to those of ordinary skill in the art. For example, such polypeptides may be synthesized using any of the commercially available solid-phase techniques, such as the Merrifield solid-phase synthesis method, where amino acids are sequentially added to a growing amino acid.chain.   See Merrified, J. Am. Chem. Soc. 85:2149-2146, 1963.  Equipment for automated synthesis of polypeptides is 15     commercially available from suppliers such as Applied BioSystems, Inc. (Foster City, CA), and may be operated according to the manufacturer's instructions.<br>
Within certain specific embodiments, a polypeptide may be a fusion protein that comprises multiple polypeptides as described herein, or that comprises one polypeptide as described herein and a known tumor antigen, such as an ovarian 20     carcinoma protein or a variant of such a protein.   A fusion partner may, for example, assist in providing T helper epitopes (an immunological fusion partner), preferably T helper epitopes recognized by humans, or may assist in expressing the protein (an expression enhancer) at higher yields than the native recombinant protein.    Certain preferred fusion partners are both immunological and expression enhancing fusion 25     partners.   Other fusion partners may be selected so as to;increase the solubility of the protein or to enable the protein to be targeted to desired intracellular compartments. Still further fusion partners include affinity-tags, which facilitate purification of the protein.<br>
Fusion proteins may generally be prepared using standard techniques,<br>
recombinant protein, allowing the production of increased levels, relative to a non-fused<br><br>
protein, in an expression system. Briefly, DHA sequences encoding the polypeptide components may be assembled separately, and ligated into an appropriate expression vector. The 3' end of the DNA sequence encoding one polypeptide component is ligated, with or without a peptide linker, to the 5' end of a DNA sequence encoding the 5 second polypeptide component so that the reading frames of the sequences are in phase. This permits'translation into a single fusion protein that retains the biological activity of both component polypeptides.<br>
A peptide linker sequence may be employed to separate the first and the<br>
second polypeptide components by a distance sufficient to ensure that each polypeptide<br>
JO    folds into its secondary and tertiary structures.    Such a peptide linker sequence is<br>
incorporated into the fusion protein using standard techniques well known in the art-<br>
Suitable peptide linker sequences may be chosen based on the following factors:<br>
(1) their ability to adopt a flexible extended conformation; (2) their inability to adopt a<br>
secondary structure that could interact with functional epitopes on the first and second<br>
15     polypeptides; and (3) the lack of hydrophobic or charged residues that might react with<br>
the polypeptide functional epitopes.   Preferred peptide linker sequences contain Gly,<br>
Asn and Ser residues.  Other near neutral amino acids, such as Thr and Ala may also be,<br>
used in the linker sequence. Amino acid sequences which may be usefully employed as<br>
linkers include those disclosed in Maratea et al, Gene 40:39-46, 1985 Murphy et al.,'<br>
20    Proc. Natl Acad Sci. USA 53:8258-8262, 1986; U.S. Patent No. 4,935,233 and U.S."<br>
Patent No. 4,751,180.  The linker sequence may generally be from 1 to about 50 amino<br>
acids  in  length.     Linker sequences  are not required when the  first  and  second<br>
polypeptides have non-essential N-terminal amino acid regions that can be used to<br>
separate the functional domains and prevent steric interference.<br>
25	The    ligated    DNA    sequences    are    operably    linked    to    suitable <br>
transcriptional   or   translational   regulatory   elements.       The   regulatory   elements' responsible for expression of DNA are located only 5' to the DNA sequence encoding' the   first  polypeptides.      Similarly,   stop  codons  required  to  end  translation   and transcription termination signals are only present 3' to the DNA sequence encoding the 30     second polypeptide.<br><br>
Fusion proteins are also provided that comprise a polypeptide of the<br>
present invention together with an unrelated immunogenic protein.    Preferably the<br>
immunogenic protein is capable of eliciting a recall response.    Examples of such<br>
proteins include tetanus, tuberculosis and hepatitis proteins (see, for example, Stoute<br>
5     et al. New Engl. J. Med., 33(5:86-91, 1997).<br>
Within preferred  embodiments,  an immunological  fusion partner is derived from protein D, a surface protein of the gram-negative bacterium Haemophilus influenza   B   (WO   91/18926).       Preferably,   a   protein   D   derivative- comprises approximately the first third of the protein (e.g., the first N-terminal 100-110 amino 10     acids),  and  a protein  D   derivative  may  be  lipidated.     Within  certain  preferred embodiments, the first 109 residues of a Lipoprotein D fusion partner is included on the N-terminus to provide the polypeptide with additional, exogenous T-cell epitopes and to increase the expression level in E. coli (thus -functioning as an expression enhancer). The lipid tail ensures optimal presentation of the antigen to antigen present cells.  Other 15     fusion  partners   include   the   non-structural   protein   from   influenzae   virus,   NS1 (hemaglutmin).   Typically, the N-terminal 81 amino acids are used, although different fragments that include T-helper epitopes may be used.<br>
In another embodiment, -the immunological fusion partner is the protein known as LYTA, or a portion thereof (preferably a C-terminal portion).   LYTA is 20     derived from Streptococcus pneumoniae,  which  synthesizes  an N-acetyJ-L-alanine amidase known as amidase LYTA (encoded by the LytA gene; Gene 43:265-292, 1986). LYTA is an autolysin that specifically degrades certain bonds in the peptidoglycan backbone. The C-terminal domain of the LYTA protein is responsible for the affinity to the choline or to some choline analogues such as DEAE.    This property has been 25     exploited for the development of E.  coli C-LYTA expressing plasmids useful for expression of fusion proteins.   Purification of hybrid proteins containing the C-LYTA fragment at the amino terminus has been described (see Biotechnology 10:195-19%, 1992). Within a preferred embodiment, a repeat portion of LYTA may be incorporated into a fusion protein.   A repeat portion is found in the C-terminal region starting at 30    residue 178. A particularly preferred repeat porticn incorporates residues 188-305.<br><br>
 generaljoojypeptides (including fusion proteins) and polynucleotides         as-described herein are isolated An 'isoIated" polypeptide or polynucleotide is one that<br>
is removed frorn ts original environment. For example, a naturally-occurring protein is isolated if it is separated from some or all of the coexisting materials in the natural 5 system. Preferably., such polypeptides are at least about 90% pure, more preferably at least about 95% pure and most preferably at least about 99% pure. A polynucleotide is considered to be isolated if for example, it is cloned into a vector that is not a part of the natural environment.<br>
Binding Agents<br>
10	The present invention further provides agents, such as antibodies and<br>
antigen-binding  fragments thereof, that specifically bind to  an  ovarian carcinoma<br>
protein.   As used herein, an antibody, or antigen-binding fragment thereof, is said to<br>
"specifically bind" to an ovarian carcinoma protein if it reacts at a detectable level<br>
(within, for example, an ELISA) with an ovarian carcinoma protein, and does not react<br>
15     detectably with unrelated proteins under similar conditions.  As used herein, "binding"<br>
refers  to  a noncovalent  association  between two separate molecules such  that a<br>
"complex"  is  formed.     The  ability to  bind  may be  evaluated  by, for example,<br>
determining a binding constant for the formation of the complex. The binding constant<br>
is the value obtained when the concentration of the complex is divided by the product of<br>
.20    the component concentrations.   In general, two compounds are said to "bind," in the<br>
context of the present invention, when the binding constant for complex formation<br>
exceeds about 103 L/mol.  The binding constant maybe determined using methods well<br>
known in the art.	,;<br>
Binding agents may be further capable of differentiating between patients 26 with and without a cancer, such as ovarian cancer, using the representative assays provided herein. In other words, antibodies or other binding agents that bind to a ovarian carcinoma antigen will generate a signal indicating the presence of a cancer in at least about 20% of patients with the disease, and will generate a negative signal indicating the absence of the disease in at least about 90% of individuals without the 30    cancer.   To determine whether a binding agent satisfies this requirement, biological<br><br>
samples (e.g., Hood, sera, leukophoresis, urine and/or tumor biopsies) from patients<br>
with and without a cancer (as determined using standard clinical tests) may be assayed<br>
as described herein for the presence of polypeptides that bind to the binding agent. It<br>
will be apparent that a statistically significant number of samples with and without the<br>
5 disease should be assayed. Each binding agent should satisfy the above criteria;<br>
however, those of ordinary skill in the art will recognize that binding agents may be<br>
Used in combination to improve sensitivity.	<br>
Any agent that satisfies the above requirements may be a binding agent. For example, a binding agent may be a ribosome, with or without a peptide component, 10     an RNA molecule or a polypeptide.  In a preferred embodiment, a binding agent is an antibody or an antigen-binding fragment thereof. Antibodies may be prepared by any of a variety of techniques known.to those of ordinary skill in the art. See, e.g.; Harlow and Lrane, Antibodies:   A Laboratory Manual, Cold Spring Harbor Laboratory, 1988.   In general, antibodies can be produced by cell culture techniques, including the generation 15     of monoclonal antibodies as described herein, or via transfection of antibody genes into suitable bacterial or mammalian cell hosts, in order to allow for the production of recombinant antibodies. In one technique, an immunogen comprising the polypeptide is initially injected into any of a wide variety of mammals {e.g., mice, rats, rabbits, sheep or goats).  In this step, the polypeptides of this invention may serve as the immunogen 20     without modification.    Alternatively, particularly for relatively short polypeptides, a superior immune response may be elicited if the polypeptide is joined to a carrier protein, such as bovine serum albumin or keyhole limpet hemocyanin. The immunogen is injected into the animal host, preferably according to a predetermined schedule incorporating one or more booster immunizations, and the animals are bled periodically. 25     Polyclonal antibodies specific for the polypeptide may then be purified from such antisera by, for example, affinity chromatography using the polypeptide coupled to a suitable solid support. <br>
Monoclonal antibodies specific for an antigenic polypeptide of interest<br>
may be prepared, for example, using the technique of Kohler and Milstein, Eur. J.<br>
30     Immunol. 6:511-519, 1976, and improvements thereto.  Briefly, these methods involve<br>
the preparation of immortal cell lines capable of producing antibodies having the<br><br>
desired specificity (i.e., reactivity with the polypeptide of interest). Such cell lines may be produced, for example, from spleen cells obtained from an animal immunized as described above.  The spleen cells are then immortalized by, for example, fusion with a myeloma cell fusion partner, preferably one that is syngeneic with the immunized 5     animal. A variety of fusion techniques may be employed. For example, the spleen cells and myeloma cells may be combined with a nonionic detergent for a few minutes and then plated at low density on a selective medium that supports the growth of hybrid cells, but not myeloma cells. A preferred selection technique uses HAT (hypoxanthine, aminopterin, thymidine) selection.  After a sufficient time, usually about 1 to 2 weeks', 10    colonies of hybrids are observed.    Singie colonies are selected and their culture supematants tested for binding activity against the polypeptide.   Hybridomas having high reactivity and specificity are preferred.<br>
Monoclonal antibodies may be isolated from the supematants of growing hybridoma colonies, In addition, various techniques may be employed to enhance the 15 yield, such as injection of the hybridoma cell line into the peritoneal cavity of a suitable vertebrate host, such as a mouse. Monoclonal antibodies may then be harvested from the ascites fluid or the blood. Contaminants may be removed from the antibodies by conventional techniques, such as chromatography, gel filtration, precipitation, and extraction. The polypeptides of this invention may be used in the purification process 20    in, for example, an affinity chromatography step.<br>
Within certain embodiments, the use of antigen-binding fragments of antibodies may be preferred. Such fragments include Fab fragments, which may be prepared using standard techniques. Briefly, immunoglobulins may be purified from rabbit serum by affinity chromatography on Protein A bead columns (Harlow and Lane, 25 Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988) and digested by papain to yield Fab and Fc fragments. The Fab and Fc fragments may be separated by affinity chromatography on protein A bead columns.<br>
Monoclonal antibodies of the present invention may be coupled to one or<br>
more  therapeutic   agents.     Suitable  agents  in   this  regard  include radionuclides,<br>
30     differentiation inducers, drugs, toxins, and derivatives thereof.  Preferred radionuclides<br>
include 90Y,  ,123I, 1 125I,  I3IJ, 86Re,  188Re, 2IIAt, -and 212Bi.    Preferred drugs include<br><br>
methotrexate, and pyrimidine and purine analogs. Preferred differentiation inducers include phorbol esters and butyric acid. Preferred toxins include ricin, abrin, diptheria toxin, cholera toxin, gelonin, Pseudoraonas exotoxin, Shigella toxin, and pokeweed<br>
antiviral protein.      ,<br>
5	A therapeutic agent may be coupled (e.g.,  covalently bonded) to a<br>
suitable monoclonal antibody either directly or indirectly (e.g., via a linker group). A direct reaction between an agent and an antibody is possible when each possesses a substituent capable of reacting with the other. For example, a nucleophilic group, such as an amino or sulfhydryl group, on one may be capable of reacting with a carbonyl-10 containing group, such as an anhydride or an acid halide, or with an alkyl group containing a good leaving group {e.g., a halide) on the other.<br>
Alternatively, it may be desirable to couple a therapeutic agent and an antibody via a linker group. A linker group can function as a spacer to distance an antibody from an agent in orderto avoid interference with binding capabilities. A linker 15 group can also serve to increase the chemical reactivity of a substituent on an agent or an antibody, and thus increase the coupling efficiency. An increase in chemical reactivity may also facilitate the use of agents, or functional groups on agents, which otherwise would not be possible.<br>
It will be evident to those skilled in the art that a variety of bifunctional<br>
20. or poly/functional reagents, both homo- and hetero-functional (such as those described in<br>
the catalog of the Pierce Chemical Co., Rockford, IL), may be employed as the linker<br>
group. "Coupling may be effected, for example, tlirough amino groups, carboxyl groups,<br>
sulfhydryl groups or oxidized carbohydrate residues. There are numerous references<br>
describing such methodology, e.g., U.S. Patent No. 4,671,95.8, to Rodwell et al.<br>
25	Where a therapeutic agent is more potent when free from the antibody<br>
portion of the immunoconjugates of the present invention, it may be desirable to use a linker group which is cleavable during or upon internalization into a cell. A number of different cieavable linker groups have been described. The mechanisms for the intracellular release of an agent from these linker groups include cleavage by reduction 30 of a disulfide bond (e.g., U.S. Patent No. 4,489,710, to Spitler), by irradiation of a photolabile bond (e.g., U.S. Patent No. 4,625,014, to Senter etal.), by hydrolysis of<br><br>
derivatized amino acid side chains (e.g., U.S. Patent No. 4,638,045, to Kohn et a].), by serum complement-mediated hydrolysis (e.g., U,S. Patent No. 4,671,958, to Rodwell et al.), and acid-catalyzed hydrolysis (e.g., U.S. Patent No. 4,569,789, to Blattler et al.). It may be desirable to couple more than one agent to an antibody. In one 5     embodiment, multiple molecules of an agent are coupled to one antibody molecule.  In another embodiment, more than one type of agent may be coupled to one antibody. Regardless of the particular embodiment, immunoconjugates with more man one agent may be prepared in a variety of ways.   For example, more than one agent may be coupled directly to an antibody molecule, or linkers which provide multiple sites for 10     attachment can be used. Alternatively, a carrier can be used.<br>
A carrier may bear the agents in a variety of ways, including covalent bonding either directly or via a linker group.  Suitable carriers include proteins such as albumins (e.g., U.S. Patent No. 4,507,234, to Kato et al.), peptides; and polysaccharides such as aminodextran (e.g., U.S. Patent No. 4,699,784, to Shih ei: al.).  A carrier may 15     also bear an agent by noncovalent bonding or by encapsulation, such as within a liposome vesicle (e.g., U.S. Patent Nos. 4,429,008 and 4,873,088). Carriers specific for radionuclide    agents    include    radiohalogenated    small   molecules   and    chelating compounds.      For 'example,   U.S.   Patent  No.   4,735,792   discloses  representative radiohalogenated small molecules and their synthesis.   A radionuclide chelate may be 20     formed from chelating compounds that include those containing nitrogen and sulfur atoms as the donor atoms for binding the metal, or metal oxide, radionuclide.   For example, U.S. Patent No. 4,673,562, to Davison et al. discloses representative chelating compounds and their synthesis.<br>
A   variety   of  routes   of   administration   for   the   antibodies   and<br>
25     immunoconjugates  may  be  used:    Typically,  administration  will  be  intravenous,<br>
intramuscular, subcutaneous or in the bed of a resected tumor. It will be evident that the<br>
precise dose of the antibody/immunoconjugate will vary depending upon the antibody<br>
used, the antigen density on the tumor, and the rate of clearance of the antibody.<br>
Also   provided   herein   are   anti-idiotypic   antibodies  that   mimic   an<br>
30    immunogenic portion of an ovarian carcinoma protein.   Such antibodies may be raised<br>
against an antibody, or antigen-binding fragment thereof, that specifically binds to an<br><br><br>
immunogenic portion of an ovarian carcinoma protein, using well known techniques. Anti-idiotypic antibodies that mimic an immunogenic portion of an ovarian carcinoma protein are those antibodies that bind to an antibody, or antigen-binding fragment thereof, that specifically binds to an immunogenic portion of an ovarian carcinoma 5    protein, as described herein.<br>
T Cells<br>
Imrnunotherapeutic compositions may also, or alternatively, comprise T cells specific for an ovarian carcinoma protein. Such cells may generally be prepared in vitrp-o or ex vivo, using standard procedures. For example., T cells may be present within 10     (or isolated from) bone marrow, peripheral blood or a fraction of bone marrow or peripheral blood of a mammal, such as a patient, using a commercially available cell separation system,  such  as the CBPRATEâ„¢ system,  available from CellPro Inc., Bothell WA (see also U.S. Patent No. 5,240,856; U.S. Patent No. 5,215,926; WO" 89/06280; WO 91/16116 and WO 92/07243).   Alternatively, T cells may be derived 15     from related or unrelated humans, non-human animals, cell lines or cultures.<br>
T cells may be stimulated with an ovarian carcinoma polypeptide, polynucleotide encoding an ovarian carcinoma polypeptide and/or an antigen presenting cell (APC) that expresses such a polypeptide. Such stimulation is performed under conditions and for a time sufficient to permit the generation of T cells that are specific 20 for the polypeptide. Preferably, an ovarian carcinoma polypeptide or polynucleotide is present within a delivery vehicle, such as a microsphere, to facilitate the generation of specific T cells.<br>
T cells are considered to be specific for an- ovarian carcinoma polypeptide if the T cells kill target cells coated with an ovarian carcinoma polypeptide 25 or expressing a gene encoding such a polypeptide. T cell specificity may be evaluated using any of a variety of standard techniques. For example, within a chromium release assay or proliferation assay, a stimulation index of more than two fold increase in lysis and/or proliferation, compared to negative controls, indicates T cell specificity. Such assays may be performed, for example, as described in Chen et al., Cancer Res. 30     54:1065-1070, 1994.   Alternatively, detection of the proliferation of T cells may be<br><br>
accomplished by a variety of known techniques.   For example, T cell proliferation can be detected by measuring an increased rate of DNA synthesis (e.g., by pulse-labeling cultures of T cells with tritiated thymidine and measuring the amount of tritiated thymidine'incorporated into DNA).   Contact with an ovarian carcinoma polypeptide 5     (200 ng/nil - 100 jig/ml, preferably 100 ng/ml - 25 p.g/ml) for 3 - 7 days should result.in at least a-two fold increase in proliferation of the T ceils and/or contact as described above for 2-3 hours should result in activation of the T cells, as measured using standard cytokine assays in which a two fold increase in the level of cytokine release (e.g., TOF or IFN-Î») is indicative of T cell activation (see Coligan et al., Current 10     Protocols in Immunology, vol. 1, Wiley Interscience (Greene 1998).   T ceils that have been activated in response to an ovarian carcinoma polypeptide, polynucleotide or ovarian carcinoma polypeptide-expressing APC may be CD4+ and/or CD8+.   Ovarian carcinoma polypeptide-specific T ceils may be expanded using standard techniques. Within preferred embodiments, the T cells are derived from a patient or a related or 15     unrelated   donor  and   are   administered' to  the  patient   following  stimulation  and expansion.<br>
For therapeutic purposes,  CD4+ or CD8+ T cells that proliferate in response to an ovarian carcinoma polypeptide, polynucleotide or APC can be expanded in number either in vitro or in vivo.    Proliferation of such T cells in vitro may be 20     accomplished in a variety of ways.   For example, the T cells can be re-exposed to an ovarian carcinoma polypeptide, with or without the addition of T cell growth factors, such as interleukin-2, and/or stimulator cells that synthesize an ovarian carcinoma polypeptide.   Alternatively, one or more T cells that proliferate in the presence of an. ovarian carcinoma polypeptide can be expanded in number by cloning.   Methods for 25     cloning cells are well known in the art, and include limiting dilution.    Following expansion, the cells may be administered back to the patient as described, for example, by Chang et al., Crit. JRev. Oncol. Hematol. 22:213,1996.<br>
Pharmaceutical Compositions and Vaccines<br>
Within certain aspects, polypeptides, polynucleotides, binding agents 30     and/or   immune   system   cells   as   described   herein   may   be   incorporated   into<br><br>
pharmaceutical compositions or vaccines.   Pharmaceutical compositions comprise one<br>
or more such compounds or cells and a physiologically acceptable carrier.   Vaccines<br>
may comprise one or more such compounds or cells and a non-specific immune<br>
response enhancer.   A non-specific immune response enhancer may be any substance<br>
5     that enhances an immune response to an exogenous antigen.  Examples of non-specific<br>
immune response  enhancers include  adjuvants,  biodegradabJe microspheres  (e.g.,<br>
polylactic galactide) and liposomes (into which the compound is incorporated; see e.g.,<br>
Fullerton, U.S. Patent No. 4,235,877).   Vaccine preparation is generally described in,<br>
for example, M.F. Powell and M.J. Newman, eds., "Vaccine Design (the subunit and<br>
10     adjuvant approach)," Plenum Press (NY,  1995).    Pharmaceutical compositions and<br>
vaccines within the scope of the present invention may also contain other compounds,<br>
which may be biologically active or inactive.  For example, one or more immunogenic<br>
portions of other tumor antigens may be present, either incorporated into a fusion<br>
polypeptide or as a separate compound within the composition or vaccine.<br>
15	A pharmaceutical composition or vaccine may contain DNA encoding<br>
one or more of the polypeptides as described above, such that the polypeptide is generated in situ.  As noted above, the DNA may be present within any of a variety of delivery systems known to those of ordinary skill in the art, including nucleic acid expression systems, bacteria and viral expression systems.   Appropriate nucleic acid 20     expression systems contain the necessary DNA sequences for expression in the patient (such as a suitable promoter and terminating signal). Bacterial delivery systems involve the administration of a bacterium (such as Bacilus-Cdlmette-Guerrin) that expresses an immunogenic portion of the polypeptide on its cell surface.  In a preferred embodiment, the DNA may be introduced using a viral expression system {e.g., vaccinia or other pox 25     virus, retrovirus,  or adenovirus), which may involve the use of a non-pathogenic (defective), replication competent virus.  Suitable systems are disclosed, for example, in Fisher-Hoch et ah, PNAS 86:311-32},  1989; Flexner et ah, Ann. N.Y. Acad Sci. . 569:86-103, 1989; Flexner et al. Vaccine 8:11-21, 199Q; U.S. Patent Nos. 4,603,112, 4,769,330, and 5,017,487; WO 89/01973; U.S. Patent No. 4,777,127; GB 2,200,651; JO    EP 0,345,242; WO 91/02805; Berlcner, Biotechniques 5:616-627, 1988; Rosenfeld et al., Science 252:431-434, 1991; Kolls et al., PNAS 91:215-219, 1994; Kass-Eisler et al.,<br><br><br>
PNAS 90:11498-11502, 1993;'Guzman et al., Circulation 55:2838-2848, 1993; and Guzman et al., Cir. Res. 73:1202-1207; 1993. Techniques for incorporating DNA into such expression systems are well known to those of ordinary skill in the art. The DNA may also be "naked," as described, for example, in Ulmer et al, Science 259:1745-1749, 5 1993 and reviewed by Cohen, Science 259:\691-1692, 1993. The uptake of naked DNA may be increased by coating the DNA onto biodegradable beads, which are efficiently transported into the cells.<br>
While any suitable carrier known to those of ordinary skill in the art may be employed in the pharmaceutical compositions of this invention, the type of carrier 10     will vary depending on the mode of administration.    Compositions of the present invention may be formulated for any appropriate manner of administration, including for example, topical, oral, nasal, intravenous, intracranial, intraperitoneal, subcutaneous or intramuscular administration.   For parenteral administration, such as subcutaneous injection, the carrier preferably comprises water, saline, alcohol, a fat, a wax or a buffer. 5     For oral administration, any of the above carriers or a solid carrier, such as mannitol, lactose, starch, magnesium stearate, sodium saccharine, talcum, cellulose, glucose, sucrose, and magnesium carbonate, may be employed.    Biodegradable microspheres (e.g., polylactate polyglycolate) may also be employed as carriers for the pharmaceutical compositions of this invention.  Suitable biodegradable microspheres are disclosed, for<br>
0	example, in U.S. Patent Nos. 4,897,268 and 5,075,109.<br>
Such compositions may also comprise buffers (e.g., neutral buffered saline or phosphate buffered saline), carbohydrates (e.g., glucose, mannose, sucrose or dextrans), mannitol, proteins, polypeptides or amino acids such as glycine, antioxidants, chelating agents such as EDTA or glutathione, adjuvants (e.g., aluminum hydroxide)   and/or preservatives. Alternatively, compositions of the present invention may be formulated as a iyophilizate. Compounds may also be encapsulated within liposomes  using well known technology.<br>
Any of a variety of non-specific immune response enhancers may be employed in the vaccines of this invention.  For example, an adjuvant may be included.<br>
	Most adjuvants  contain  a  substance designed to protect the  antigen from rapid<br>
catabolism, such as aluminum hydroxide or mineral oil, and a stimulator of immune<br><br>
responses, such as lipid A, Bortadella pertussis or Mycobacterium tuberculosis derived proteins. Suitable adjuvants are commercially available as, for example, Freunc Incomplete Adjuvant and Complete Adjuvant (Difco Laboratories, Detroit, MI), Merc  Adjuvant 65 (Merck and Company, Inc., Rahway, NJ), alum, biodegradab 5  microspheres, monophosphoryl lipid A and quil A. Cytokines, such as GM-CSF for interleukin-2, -7, or -12, may also be used as adjuvants.<br>
Within the  vaccines  provided  herein,  the  adjuvant  composition i<br>
preferably designed to induce an immune response predominantly of the Thl type<br>
High levels of Thl-type cytokines (e.g.,  EFN-Î³, IL-2 and IL-12) tend to favor the<br>
     induction of cell mediated immune responses to an administered antigen.   In contrasl<br>
  high levels of Th2-type cytokines (e.g., IL-4, IL-5, IL-6, IL-10 and TNF~Î²) tend to favo<br>
 the induction of humoral immune responses.   Following application of a vaccine a:<br>
provided herein, a patient will support an immune response that includes Thl- and Th2-<br>
type responses.   Within a preferred embodiment, in which a response is-predominantly<br>
5     Thl-type, the level of Thl-type cytokines will increase to a greater extent than the level<br>
of Th2-type cytokines.   The levels of these cytokines may be readily assessed using<br>
standard assays.  For a review of the families of cytokines, see Mosmann and Coffman,<br>
Ann. Rev. Immunol. 7:145-173, 1989.<br>
Preferred   adjuvants  for  use  in   eliciting  a  predominantly  Thl-type     response include, for example, a combination of monophosphoryl lipid A, preferably 3-de-O-acylated monophosphoryl lipid A (3D-MPL), together with an aluminum salt. MPL adjuvants are available from Ribi ImmunoChem Research Inc. (Hamilton, MT; see US Patent Nos. 4,436,727; 4,877,611; 4,866,034 and 4,912,094). Also preferred is AS-2 (SmithKIine Beecham).    CpG-containing oligonucleotides (in which the CpG dinucleotide  is unmethylated)  also  induce  a predominantly Th]   response.     Such oligonucleotides are well known and are described, for example, in WO 96/02555. Another preferred adjuvant is a saponin, preferably QS21, which may be used alone or in combination with other adjuvants.   For example, an enhanced system involves the combination   of a  monophosphoryl  lipid A  and  saponin  derivative,, such  as  the combination of QS21 and 3D-MPL as described in WO 94/00153, or a less reactogenic composition  where the  QS21 is quenched  with cholesterol,  as  described  in  WO<br><br>
96/33739. Other preferred formulations comprises an oil-in-water emulsion and tocopherol. A particularly potent adjuvant formulation involving QS21, 3D-MPL and tocopherol in an oil-in-water emulsion is described in WO 95/17210. Any vaccine provided herein may be prepared using well known methods that result in a combination 5     of antigen, immune response enhancer and a suitable carrier or excipient<br>
The compositions described herein may be administered as part of a sustained release formulation (i.e., a formulation such as a capsule or sponge that effects a  slow release  of compound following administration).     Such  formulations may generally be prepared using well known technology and administered by, for example, 10     oral, rectal or subcutaneous implantation, or by implantation at the desired target site. Sustained-release formulations may contain a polypeptide, polynucleotide or antibody dispersed in a carrier matrix and/or contained within a reservoir surrounded by a rate controlling membrane.   Carriers for use within such formulations are biocompatible, and may also be biodegradable; preferably the formulation provides a relatively constant 15     level of active component release.  The amount of active compound contained within a sustained  release formulation depends upon the site of implantation, the rate and expected duration of release and the nature of the condition to be treated or prevented.<br>
Any   of  a  variety  of delivery  vehicles  may   be   employed   within pharmaceutical compositions and vaccines to facilitate production of an antigen-specific 20    immune response that targets tumor cells. Delivery vehicles include antigen presenting cells (APCs), such as dendritic cells, macrophages, B cells, monocytes and other cells that may be engineered to be efficient APCs.    Such cells may, but need not, be genetically modified to increase the capacity for presenting the antigen, to improve activation and/or maintenance of the T cell response, to have anti-tumor effects per se 25     and/or to  be  immunologically  compatible  with  me  receiver  {i.e.,   matched  HLA haplotype).   APCs may generally he isolated from any of a variety of biological fluids and organs, including tumor and peritumoral tissues, and may be autologous, allogeneic, syngeneic or xenogeneic cells.<br>
Certain preferred embodiments of the present invention use dendritic<br>
30     cells or progenitors thereof as antigen-presenting cells. Dendritic cells are highly potent<br>
.   APCs (Banchereau and Steinman, Nature 392:245-25], 1998) and have been shown to<br><br>
be  effective  as  a physiological  adjuvant for  eliciting prophylactic . or therapeutic antitumor immunity (see Timmerman and Levy, Ann. Rev. Med. 50:507-529, 1999). In general, dendritic cells may be identified based on their typical shape (stellate in situ, with marked cytoplasmic processes (dendrites) visible in vitro) and based on the lack of 5    differentiation markers of B cells (CD19 and CD20), T cells (CD3), monocytes (CD14) and natural killer cells (CD56), as determined using standard assays.   Dendritic cells may, of course, be engineered to express specific cell-surface receptors or ligands that are not commonly found on dendritic cells in vivo or ex vivo, and such modified dendritic cells are contemplated by the present invention.  As an alternative to dendritic 10    cells, secreted vesicles antigen-loaded dendritic cells (called exosomes) may be used within a vaccine (see Zitvogel et al., Nature Med. 4:594-600, 1998).<br>
Dendritic cells and progenitors may be obtained from peripheral blood,   bone  marrow,  tumor-infiltrating  cells, peritumoral  tissues-infiltrating  cells,  lymph nodes, spleen, skin, umbilical cord blood or any other suitable tissue or fluid.   For 15     example, dendritic cells may be differentiated ex vivo by adding a combination of cytokines  such  as  GM-CSF,  1L-4,  IL-13  and/or TNFÎ±  to  cultures  of monocytes harvested from peripheral blood.   Alternatively, CD34 positive cells harvested from peripheral blood, umbilical cord blood or bone marrow may be differentiated, into dendritic cells by adding to the culture medium combinations of GM-CSF, IL-3, TNFÎ±, 20     CD40 ligand, LPS, flt3 ligand and/or other compound(s) that induce maturation and proliferation of dendritic cells.<br>
Dendritic cells are conveniently categorized as "immature" and "mature" cells, which allows a simple way to discriminate between two well characterized phenotypes.    However,  this nomenclature should not be construed to  exclude all 25     possible   intermediate   stages   of  differentiation.       Immature   dendritic   cells  are characterized as APC with a high capacity for antigen uptake and processing, which correlates with the high expression of FcÎ³  receptor, mannose receptor and DBC-205 marker. The mature phenotype is typically characterized by a lower expression of these markers, but  a high expression  of cell  surface  molecules responsible for T cell 30     activation such as class I and class II MHC, adhesion molecules (e.g., CD54 and CD 11) and costimulatory molecules (e.g., CD40, CDSO and CD86).<br><br>
APCs may generally be transfected with a polynucleotide encoding a<br>
ovarian carcinoma antigen (or portion or other variant thereof) such that the antigen, or<br>
an immunogenic portion thereof, is expressed on the cell surface.   Such transfection.<br>
may take place ex vivo, and a composition or vaccine comprising such transfected cells<br>
5     may then be used for therapeutic purposes, as described herein.   Alternatively, a gene<br>
delivery vehicle that targets a dendritic  or other antigen presenting cell  may be<br>
administered to a patient, resulting in transfection that occurs in vivo.   In vivo and ex<br>
vivo transfection of dendritic cells, for example, may generally be performed using any<br>
methods known in the art, such as those described in WO 97/24447, or the gene gur<br>
10     approach described by Mahvi et al, Immunology and cell Biology 75:456-460, 1997.<br>
Antigen loading of dendritic cells may be achieved by incubating dendritic cells or<br>
.:     progenitor cells with the polypeptide, DNA (naked or within a plasmid vector) or RNA;<br>
or with antigen-expressing recombinant bacterium or viruses (e.g., vaccinia, fowlpox,<br>
adenovirus or lentivirus vectors).   Prior to loading, the polypeptide may be covalenily<br>
15     conjugated to an immunological partner that provides T cell help (e.g.,  a carrier<br>
molecule).     Alternatively,  a dendritic  cell  may be pulsed with a non-conjugated<br>
immunological partner, separately or in the presence of the polypeptide.<br>
Cancer Therapy<br>
In further aspects of the present invention, the compositions described<br>
20 herein may be used for immunotherapy of cancer, such as ovarian cancer. Within such methods, pharmaceutical compositions and vaccines are typically administered to a patient. As used herein, a "patient" refers to any warm-blooded animal, preferably a human. A patient may or may not be afflicted with cancer. Accordingly, the above pharmaceutical compositions and vaccines may be used to prevent the development of a<br>
25     cancer  or  to  treat  a  patient  afflicted  with  a  cancer.   Within  certain  preferred<br>
embodiments, a patient is afflicted with ovarian cancer.  Such cancer may be diagnosed<br>
using criteria generally accepted in the art, including the presence of a malignant tumor.<br>
- Pharmaceutical compositions and vaccines may be administered either prior to or<br>
following surgical removal of primary tumors and/or treatment such as administration<br>
30     of radiotherapy or conventional chemotherapeutic drugs.<br><br>
Within    certain     embodiments,    immunotherapy    may    be     active<br>
immunotherapy, in which treatment relies on the in vivo stimulation of the endogenous<br>
host immune system to react against tumors with the administration of immuno<br>
response-modifying   agents   (such   as   tumor   vaccines,   bacterial   adjuvants   and/or<br>
5     cytokines).<br>
Within    other    embodiments,    immunotherapy    may    be    passive immunotherapy, in which treatment involves the delivery of agents with establishesd tumor-immune reactivity (such as effector cells or antibodies) that can directly or indirectly mediate antitumor effects and does not necessarily depend on an intact hos 10     immune system.    Examples of effector cells include T lymphocytes (such as CD8+ cytotoxic T lymphocytes and CD4+ T-helper tumor-infiltrating lymphocytes), killer cells (such as Natural Killer cells and lymphokine-activated killer cells), B cells and antigen-presenting ce)ls (such as dendritic cells and macrophages) expressing a polypeptide . rprovided herein.   T cell receptors and antibody receptors specific for the polypeptides 15     recited herein may be cloned, expressed and transferred into other vectors or effector cells for adoptive immunotherapy.  The polypeptides provided herein may also be used to generate antibodies or anti-idiotypic antibodies (as described above and in U.S. Patent No. 4,918,164) for passive immunotherapy.<br>
Effector cells may generally be obtained in sufficient quantities for 20     adoptive immunotherapy by growth in vitro, as described herein. Culture conditions-for expanding single antigen-specific effector cells to several billion in number with retention of antigen recognition in vivo are well known in the art.  Such in vitro culture conditions typically use intermittent stimulation with antigen, often in the presence of cytokines   (such   as   IL-2)   and   non-dividing   feeder   cells,    As   noted   above, 25     immunoreactive polypeptides  as provided herein  may be  used  to rapidly expand anti gen-specific T cell cultures in order to generate a sufficient number of cells for immunotherapy.   In particular, antigen-presenting cells, such as ;dendritic, macrophage or B cells, may be pulsed with immunoreactive polypeptides or transfected with one or more polynucleotides using standard techniques well known in the art.   For example,        anti gen-presenting cells can be transfected with a polynucleotide having a promoter appropriate for increasing expression in a recombinant virus or other expression system.<br><br>
Cultured effector cells for use in therapy must be able to grow and distribute widely and to survive long term in vivo. Studies have shown that cultured effector cells can be induced to grow in vivo and to survive long term in substantial numbers by repeated stimulation with antigen supplemented with IL-2 {see, for example, Cheever et al.,     Immunological Reviews 157:111, 1997).<br>
Alternatively, a vector expressing a polypeptide recited herein may be introduced into stem cells taken from a patient and clonally propagated in vitro for autologous transplant back into the same patient.<br>
Routes and frequency of administration, as well as dosage, will vary 10    from individual to individual, and may be readily established using standard techniques. In general, the pharmaceutical compositions and vaccines may be administered by injection {e.g., intracutaneous, intramuscular, intravenous or subcutaneous), intranasally (e.g., by aspiration), orally or in the bed of a resected rumor. Preferably, between 1 and 10 doses may be administered  over a 52  week period.    Preferably, 6 doses are 15     administered,   at   intervals   of   1   month,   and   booster   vaccinations   may   be   given periodically thereafter.   Alternate protocols may be appropriate for individual patients. A suitable dose is an amount of a compound that, when administered as described above, is capable of promoting an anti-tumor immune response, and is at least 10-50% above the basal {i.e., untreated) level..   Such response can be monitored by measuring 20    the anti-tumor antibodies in a patient or by vaccine-dependent generation of cytolytic effector cells capable of killing the patient's tumor cells in vitro.  Such vaccines should also be capable of causing an immune response that leads to an improved clinical outcome (e.g., more frequent remissions, complete or partial or longer disease-free survival) in vaccinated patients as compared to non-vaccinated patients. In general, for 25    pharmaceutical compositions and vaccines comprising one or more polypeptides, the amount of each polypeptide present in a dose ranges from about 100 Âµg to 5 mg per kg of host.   Suitable dose sizes will vary with the size of the patient, but will typically range from about 0.1 mL to about 5 mL.<br>
In general, an appropriate dosage and treatment regimen provides the<br>
30     active compound(s) in an amount sufficient to provide therapeutic and/or prophylactic<br>
benefit.     Such a response can be monitored by establishing an improved clinical<br><br>
outcome (e.g., more frequent remissions, complete or partial, or longer disease-free survival) in treated patients as compared to non-treated patients. Increases in preexisting immune responses to an ovarian carcinoma antigen generally correlate with an improved clinical outcome. Such immune responses may generally be evaluated 5 using standard proliferation, cytotoxicity or cytokine assays, which may be performed using samples obtained from a patient before and after treatment.<br>
Screens for Identifying Secreted Ovarian Carcinoma Antigens<br>
The present invention provides methods for identifying secreted tumor<br>
antigens.   Within such methods, tumors are implanted into immunodeficient animals<br>
10     such as SCID mice and maintained for a time sufficient to permit secretion of tumor<br>
antigens into serum.  In general, tumors may be implanted subcutaneously or within the<br>
gonadal  fat pad  of an  immunodeficient  animal  and maintained  for  1-9 months,<br>
preferably 1-4 months.  Implantation may generally be performed as described in WO<br>
97/18300.   The serum containing secreted antigens is then used to prepare antisera in<br>
15     immunocompetent mice, using standard techniques and as described herein.   Briefly,<br>
50-100 pL of sera (pooled from three sets of immunodeficient mice, each set bearing a<br>
different SCID-derived human ovarian tumor) may be mixed 1:1 (vol:vol) with an<br>
appropriate adjuvant, such as RIBI-MPL or MPL + TDM (Sigma Chemical Co., St.<br>
Louis, MO) and injected intraperitoneally into syngeneic immunocompetent animals at<br>
20     monthly intervals for a total of 5 months.  Antisera from animals immunized in such a<br>
manner  may   be   obtained   by   drawing   blood   after   the   third,   fourth   and   fifth<br>
immunizations.   The resulting antiserum is generally pre-cleared  of E. coli and phage<br>
antigens  and  used  (generally  following dilution, such as   1:200)  in a  serological<br>
expression screen.<br>
25	The library is typically an expression library containing cDNAs from one<br>
or more tumors of the type that was implanted into SCID mice. This expression library may be prepared in any suitable vector, such as Î»-screen (Novagen). cDNAs that encode a polypeptide that reacts with the antiserum may be identified using standard techniques, and sequenced.    Such cDNA molecules may be further characterized to<br><br>
evaluate expression in tumor and norma] tissue, and to evaluate antigen secretion ir.<br>
patients.<br>
The methods provided herein have advantages over other methods for<br>
tumor antigen discovery.   In particular, all antigens identified by such methods should<br>
5     be secreted or released through necrosis of the tumor cells.    Such antigens may be<br>
present on the surface of tumor cells for an amount of time sufficient to permit targeting<br>
and killing by the immune system, following vaccination.<br>
Methods for Detecting Cancer<br>
In general, a cancer may be detected in a patient based on the presence of<br>
10     one or more ovarian carcinoma proteins and/or polynucleotides encoding such proteins<br>
in a biological sample (such as.blood, sera, urine and/or tumor biopsies) obtained from<br>
the patient.    In other words, such proteins may be used as markers to indicate the<br>
presence or absence of a cancer such as ovarian cancer.  In addition, such proteins may<br>
be useful for the detection of other cancers.    The binding agents provided herein<br>
15     generally permit detection of the level of protein that binds to the agent in the biological<br>
sample.  Polynucleotide primers and probes may be used to detect the level of mRNA<br>
encoding a tumor protein, which is also indicative of the presence or absence of a<br>
cancer.   In general, an ovarian carcinoma-associated sequence should be present at a<br>
level that is at least three fold higher in tumor tissue than in normal tissue<br>
20	There are a variety of assay formats known to those of ordinary skill in<br>
the art for using a binding agent to detect polypeptide markers in a sample. See, e.g., Harlow and Lane, Antibodies: A Laboratoiy Manual, Cold Spring Harbor Laboratory, 1988. In general, the presence or absence of a cancer in a patient may be determined by (a) contacting a biological sample obtained from a patient with a binding agent; (b) 5  defecting in the sample a level of polypeptide that binds to the-binding agent; and (c) comparing the level of polypeptide with a predetermined cut-off value.<br>
In a preferred embodiment, the assay involves the use of binding agent<br>
immobilized on a solid support to bind to and remove the polypeptide from the<br>
remainder of the sample. The bound polypeptide may then be detected using a detection<br>
0    reagent   that   contains   a   reporter   group   and   specifically   binds   to   the   binding<br><br>
agent/polypeptide complex.    Such detection reagents may comprise, for example, a binding agent that specifically binds to the polypeptide or an antibody or other agent that specifically binds to the binding agent, such as an anti-immunoglobulin, protein G, protein A or a lectin.   Alternatively, a competitive assay may be utilized, in which a 5     polypeptide is labeled with a reporter group and allowed to bind to the immobilized binding agent after incubation of the binding agent with the sample.   The extent to which components of the sample inhibit the binding of the labeled polypeptide to the binding agent is indicative of the reactivity of the sample with the immobilized binding agent.   Suitable polypeptides for use within such assays include full length ovarian 10     carcinoma proteins and portions thereof to which the binding agent binds, as described above.<br>
The solid support may be any material known to those of ordinary skill in the art to which the tumor protein may be attached.   For example, the solid support may be a test well in a microliter plate or a nitrocellulose or other suitable membrane. 15     Alternatively, the support may be a bead or disc, such as glass, fiberglass, latex or a plastic material such as polystyrene or polyvinylchloride.   The support may also be a magnetic particle or a fiber optic sensor, such as those disclosed, for example, in U.S. Patent No. 5,359,681.   The binding agent may be immobilized on the solid support using a variety of techniques known to those of skill in the art, which are amply 20     described in the patent and scientific literature.  In the context of the present invention, the term "immobilization" refers to both noncovalent association, such as adsorption, and covalent attachment (which may be a direct linkage between  the agent and functional groups on the support or may be a linkage by way of a cross-linking agent). Immobilization by adsorption to a well in a microliter plate or to a membrane is 25    preferred. In such cases, adso.rption may be achieved by contacting the binding agent, in a suitable buffer, with the solid support for a suitable amount of time. The contact time varies with temperature, but is typically between about 1 hour and about 1 day.   In general,  contacting  a  well  of a plastic  microliter plate  (such  as polystyrene  or polyvinyichloride) with an amount of binding agent ranging from about 10 ng to about 30     10 Âµg, and preferably about 100 ng to about i pg, is sufficient to immobilize an adequate amount of binding agent.<br><br>
Covalent attachment of binding agent to a solid support may generally be achieved by first reacting the support with a bifunctional reagent that will react with both the support and a functional group, such a:; a hydroxyl or amino group, on the binding agent.   For example, the binding agent may be covalently attached to supports 5     having an appropriate polymer coating using benzoquinone or by condensation of an aldehyde group on the support with an amine and an active hydrogen on the binding partner   (see,   e.g.,   Pierce   Immunotechnology   Catalog   and   Handbook,   1991,   at A12-A13).<br>
In certain embodiments, the assay is a two-antibody sandwich assay.<br>
10     This assay may be performed by first contacting an antibody that has been immobilized<br>
on a solid support, commonly the well of a microriter plate, with the sample, such that<br>
polypeptides within the sample are allowed to bind to the immobilized antibody.<br>
Unbound   sample   is   then   removed   from   the   immobilized   polypepti de-antibody<br>
complexes and a detection reagent (preferably a second antibody capable of binding to a<br>
15     different site on the polypeptide) containing a reporter group is added.   The amount of<br>
detection reagent that remains bound to the solid support is then detennined using a<br>
method appropriate for the specific reporter group.<br>
More specifically, once the antibody is immobilized on the support as described above, the remaining protein binding sites OM the support are typically 20     blocked. Any suitable blocking agent known to those of ordinary skill in the art, such as bovine serum albumin or Tween 20â„¢ (Sigma Chemical Co., St. Louis, MO).   The immobilized antibody is then incubated with the sample, and polypeptide is allowed to bind to the antibody.    The sample may be diluted with a suitable diluent, such as phosphate-buffered saline (PBS) prior to incubation.  In general, an appropriate contact 25     time (i.e., incubation time) is a period of time that is sufficient to detect the presence of polypeptide   within  a  sample  obtained  from   an   individual  with   ovarian  cancer. Preferably, the contact time is sufficient to achieve a level of binding that is at least about 95% of that achieved at equilibrium between bound and unbound polypeptide. Those of ordinary skill in the art will recognize that the time necessary to achieve 30     equilibrium may be readily determined by assaying the level of binding that occurs over<br><br>
a period of time.   At room temperature, an incubation time of about 30 minutes is generally sufficient.<br>
Unbound sample may then be removed by washing the solid support   with an. appropriate -buffer, such as PBS containing 0.1% Tween 20â„¢.   The second 5     antibody, which contains a reporter group, may then be added to the solid support, Preferred reporter groups include those groups recited above.<br>
The detection reagent is then incubated with the immobilized antibody-polypeptide complex for an amount of time sufficient to detect the bound polypeptide. An appropriate amount of time may generally be determined by assaying the level of     binding that occurs over a period of time.  Unbound detection reagent is then removed and bound detection reagent is detected using the reporter group. The method employed for detecting the reporter group depends upon the nature of the reporter group.   For radioactive groups, scomtollation counting or autoradiogratphic methods are generally appropriate.   Spectroscopic methods may be used to detect dyes, luminescent groups  and fluorescent groups.   Biotin may be detected using avidin, coupled to a different reporter group (commonly a radioactive or fluorescent group or an enzyme).  Enzyme reporter groups may generally be detected by the addition of substrate (generally for a specific period of time), followed by spectroscopic or other analysis of the reaction products.<br>
To determine the presence or absence of a cancer, such as ovarian cancer, the signal detected from the reporter group that remains bound to the solid support is generally compared to a signal that corresponds to a predetermined cut-off value.   In one preferred embodiment, the cut-off value for the detection of a cancer is the average mean signal obtained when the immobilized antibody is incubated with samples from patients without the cancer,  In general, a sample generating a signal that is three standard deviations above the predetermined cut-off value is considered positive for the cancer.   In an alternate preferred embodiment, the cut-off value is determined using a Receiver Operator Curve, according to the method of Sackett et al., Clinical Epidemiology: A Basic Science for Clinical Medicine, Little Brown and Co., 1985, p. 106-7. Briefly, in this embodiment, the cut-off value may be determined from a plot of pairs of true positive rates {i.e., sensitivity) and false positive rates (100%-specificity)<br><br>
that correspond to each possible cut-off value for the diagnostic test result The cut-off value on the plot that is the closest to the upper left-hand corner (i.e., the value that encloses the largest area) is the most accurate cut-off value, and a sample generating a signal that is higher than the cut-off value determined by this method may be considered 5 positive. Alternatively, the cut-off value may be shifted to the left along the plot, to minimize the false positive rate, or to the right, to minimize the false negative rate. In general, a sample generating a signal that is higher than the cut-off value determined by this method is considered positive for a cancer.<br>
In a related embodiment, the assay is performed in a flow-through or . 10     strip test format, wherein the binding agent is immobilized on a membrane, such as nitrocellulose.   In the flow-through test, polypeptides within the sample bind to the immobilized binding agent as the sample passes through the membrane.   A second, labeled binding agent then binds to the binding agent-polypeptide complex as a. solution containing the second binding agent flows through the membrane.   The detection of 15     bound second binding agent may then be performed as described above. In the strip test format, one end of the membrane to which binding agent is bound is immersed in a solution containing the sample.   The sample migrates along the membrane through a region containing second binding agent and to the area of immobilized binding agent. Concentration of second binding agent at the area of immobilized antibody indicates the 20     presence of a cancer.   Typically, the concentration of second binding agent at that site generates a pattern, such as a line, that can be read visually.   The absence of such a pattern indicates a negative result. In general, the amount of binding agent immobilized on the membrane is selected to generate a visually discernible pattern when the biological sample contains a level of polypeptide that would be sufficient to generate a 25     positive signal in the two-antibody sandwich assay, in the format discussed above. Preferred binding agents for use in such assays are antibodies and antigen-binding fragments thereof.   Preferably, the amount of antibody immobilized on the membrane ranges from about 25 ng to about lug, and more preferably from about 50 ng to about 500 ng.   Such tests can typically be performed with a very small amount of biological 30     sample.<br><br>
Of course, numerous other assay protocols exist that are suitable for use with the tumor proteins or binding agents of the present invention. The above descriptions are intended to be exemplary only. For example, it will be apparent to those of ordinary skill in the art that the above protocols may be readily modified to use 5 ovarian carcinoma polypeptides to detect antibodies that bind to such polypeptides in a biological sample. The detection of such ovarian carcinoma protein specific antibodies    may correlate with, the presence of a cancer.<br>
A cancer may also, or alternatively, be detected based on the presence of<br>
T cells that specifically react with an ovarian carcinoma protein in a biological sample.<br>
10     Within certain methods, a biological sample comprising CD4+ and/or CD8+ T cells<br>
isolated from a patient is incubated with an ovarian carcinoma protein, a polynucleotide<br>
encoding such a polypeptide and/or an APC that expresses at least an immunogenic<br>
portion of such a polypeptide, and the presence or absence of specific activation of the<br>
T cells is detected.  Suitable biological samples include, but are not limited to, isolated    .<br>
15     T cells. For example, T cells may be isolated from a patient by routine techniques (such<br>
as by Ficoll/Hypaque density gradient centrifugntion of peripheral blood lymphocytes).<br>
T cells may be incubated in vitro for 2-9 days (typically 4 days) at 37Â°C with an ovarian<br>
carcinoma protein {e.g., 5-25 p.g/ml).  It may be desirable to incubate another aliquot<br>
of a T cell sample in the absence of ovarian carcinoma protein to serve as a control. For<br>
,20     CD4+ T cells, activation is preferably detected by evaluating proliferation of the T cells.<br>
For CD8+ T cells, activation is preferably detected by evaluating cytolytic activity.  A<br>
level of proliferation that is at least two fold greater and/or a level of cytolytic activity<br>
that is at least 20% greater than in disease-free patients indicates the presence of a<br>
 Cancer in the patient.<br>
25	As noted above, a cancer may also, or alternatively, be detected based on<br>
the level of mRNA encoding an ovarian carcinoma protein in a biological sample. For example, at least two oligonucleotide primers may be employed in.a polymerase chain reaction (PCR) based assay to amplify a portion of an ovarian carcinoma protein cDNA derived from a biological sample, wherein at least one of the oligonucleotide primers is 30 specific for (i.e., hybridizes to) a polynucleotide encoding the ovarian carcinoma protein.    The amplified cDNA is then separated and detected using techniques well<br><br>
known in the art, such as gel electrophoresis.   Similarly, oligonucleotide probes that<br>
specifically hybridize to a polynucleotide encoding an ovarian carcinoma protein may<br>
be used in a hybridization assay to detect the presence of polynucleotide encoding the<br>
tumor protein in a biological sample.<br>
5	To permit hybridization under assey conditions, oligonucleotide primers<br>
and probes should comprise an oligonucleotide sequence that has at least about 60%,<br>
preferably at least about 75% and more preferably at least about 90%, identity to a<br>
portion of a polynucleotide encoding an ovarian carcinoma protein that is ,at least 10<br>
nucleotides,   and   preferably   at   least   20   nucleotides,   in   length.       Preferably,<br>
10     oligonucleotide  primers   and/or  probes  hybridize  to  a  polynucleotide  encoding  a<br>
polypeptide described herein under moderately stringent conditions, as defined above.<br>
Oligonucleotide  primers   and/or probes  which  may  be  usefully  employed  in  the<br>
diagnostic methods described herein preferably are at least 10-40 nucleotides in length.<br>
In a preferred embodiment, the oligonucleotide primers comprise at least 10 contiguous<br>
15     nucleotides, more preferably at least 15 contiguous nucleotides, of a DNA molecule<br>
having a  sequence provided herein.     Techniques for both PCR based  assays  and<br>
hybridization assays are well known in the art {see, for example, Mullis et al., Cold<br>
Spring Harbor Symp. Quant. Biol., 51:263, 1987; Brlich ed., PCR Technology, Stockton<br>
Press, NY, 1989).<br>
20	One preferred assay employs RT-PCR, in which PCR is applied in<br>
conjunction with reverse transcription.   Typically, RNA is extracted from a biological sample such as a biopsy tissue and is reverse transcribed to produce cDNA molecules. PCR amplification using at least one specific primer generates a cDNA molecule, which may be separated and visualized using, for example, gel electrophoresis. Amplification 25     may be performed  on biological samples taken from  a test patient and from an individual who is not afflicted with a cancer.    The amplification reaction may be performed on several dilutions of cDNA spanning two orders of magnitude. A two-fold or greater increase in expression in several dilutions of the test patient sample as compared to the same dilutions of the non-cancerous sample is typically considered 30    positive.<br><br>
In. another embodiment, ovarian carcinoma proteins and polynucleotides<br>
encoding such proteins may be used as markers for monitoring the progression of<br>
cancer.   In this embodiment, assays as described above for the diagnosis of a cancer<br>
may be performed over time, and the change in the level of reactive polypeptide(s)<br>
5     evaluated. For example, the assays may be performed every 24-72 hours for a period of<br>
6 months to  1  year, and thereafter performed as needed.    In general, a cancer is<br>
progressing in those patients in whom the level of polypeptide detected by the binding<br>
agent increases over time.  In contrast, the cancer is not progressing when the level of<br>
reactive polypeptide either remains constant or decreases with time.<br>
10	Certain in vivo diagnostic assays may be performed directly on a tumor.<br>
One such assay involves contacting tumor cells with a binding agent.    The bound<br>
binding agent may then be detected directly or indirectly via a reporter group.   Such<br>
binding   agents   may   also   be   used   in   histological   applications:      Alternatively,<br>
polynucleotide probes may be used within such applications.<br>
15	As noted above, to improve sensitivity, multiple ovarian carcinoma<br>
protein markers may be assayed within a given sample. It will be apparent that binding agents specific for different proteins provided herein may be combined within a single assay. Further, multiple primers or probes may be used concurrently. The selection of tumor protein markers may be based on routine experiments to determine combinations 20 that results in optimal sensitivity. In addition, or alternatively, assays for tumor proteins provided herein may be combined with assays for other known tumor antigens.<br>
Diagnostic Kits<br>
The present invention further provides kits for use within any of the above diagnostic methods.    Such kits typically .comprise two or more components<br>
25 necessary for performing a diagnostic assay. Components may be compounds, reagents, containers and/or equipment. For example, one container within a kit may contain a monoclonal antibody or fragment thereof that specifically binds to an ovarian carcinoma protein. Such antibodies or fragments may be provided attached to a support material, as described above.   One or more additional containers may enclose elements, such as<br>
30     reagents or buffers, to be used in the assay.  Such kits may also, or alternatively, contain<br><br>
a detection reagent as described above that contains a reporter group suitable for direct<br>
or indirect detection of antibody binding.<br>
Alternatively, a kit may be designed to  detect the level of mRNA<br>
encoding an ovarian carcinoma protein in a biological sample.    Such kits generally<br>
5     comprise at least  one  oligonucleotide probe  or primer,  as described  above,  the<br>
hybridizes to a polynucleotide encoding an ovarian carcinoma protein.     Such %<br>
oligonucleotide may be used, for example, within a PCR or hybridization assay<br>
Additional   components  that  may  be present  within  such  kits  include  a second<br>
oligonuclectide and/or a diagnostic reagent or container to facilitate the detection of a<br>
10     polynucleotide encoding an ovarian carcinoma protein.<br>
The following Examples are offered by way of illustration and not by way of limitation   <br><br>
EXAMPLE 1 IDENTIFICATION OF REPRESENTATIVE OVAFJAN CARCINOMA PROTEIN CDNAS<br>
This Example illustrates the identification of cDNA molecules encoding 5     ovarian carcinoma proteins.<br>
Anti-SCID mouse sera (generated against sera from SOD mice carrying late passage ovarian carcinoma) was pre-cleared of E. coli and phage antigens and used at a 1:200 dilution in a serological expression screen. The library screened was made from a SCID-derived human ovarian tumor (OV9334) using a directional RH bligo(dT) 10 priming cDNA library construction kit and the XScreen vector (Novagen). A bacteriophage lambda screen was employed. Approximately 400,000 pfu of the amplified OV9334 library were screened.<br>
196 positive clones were isolated. Certain sequences that appear to be novel are provided in Figures 1A-1S and SEQ ID NOT to 71. Three complete insert 15 sequences are shown in Figures 2A-2C (SEQ ID NO:72 to 74). Other clones having known sequences are presented in Figures 15A-15EEE (SEQ ID NO:82 to 310). Database searches identified the following sequences that were substantially identical to the sequences presented in Figures 15A-15EEE..<br>
These clones were farther characterized using microarray technology to 20     determine mRNA expression levels in a variety of tumor and normal tissues.    Such analyses were performed using a Synteni (Palo Alto, CA) microarray, according to the manufacturer's instructions.   PCR amplification products were arrayed on slides, with each product occupying a unique location in the array. mRNA was extracted from the tissue sample to be tested, reverse transcribed and fluorescent-labeled cDNA probes 25     were generated.   The microarrays were probed with the labeled cDNA probes and the slides were scanned to measure fluorescence intensity.    Data was analyzed using Synteni's provided GEMtools software. The results for one clone (13695, also referred to as 08E) are shown in Figure 3.<br><br>
EXAMPLE 2 IDENTIC CATION OF OVARIAN CARCINOMA CDNAS USING MICROARRAY TECHNOLOGY<br>
This Example illustrates the identification of ovarian carcinoma polynucleotides by PCR subtraction and microarray analysis. Microarrays of cDNAs 5 were analyzed for ovarian tumor-specific expression using a Synteni (Palo Alto, CA) microarray according to the manufacturer's instructions (and essentially as described by Schena et al, Proc. Natl. Acad. Set USA 25:10614-10619, 1996 and Heller et ah, Proc Natl Acad Sci. USA 94:2150-2155, 1997).<br>
A PCR subtraction was performed using a tester comprising cDNA of 10     four ovarian tumors (three of which were metastatic tumors) and a driver of cDNA form five normal tissues (adrenal gland, lung, pancreas, spleen and brain).  cDNA fragments recovered from this subtraction were subjected to DNA microarray analysis where the fragments were PCR amplified, adhered to chips and hybridized with fluorescently labeled prcbes derived from mRNAs of human ovarian tumors and a variety of normal 15     human tissues   In  this analysis, (he slides were scanned and the fluorescence intensity was measured, and the data were analyzed using Synteni's GEMtools software.    In general, seQuences showing at least a 5-fold increase in expression in tumor cells (relative to normal cells) were considered ovarian tumor antigens.    The fluorescent results were analyzed and clones that displayed increased expression in ovarian tumors 20     were further characterized by DNA sequencing and database searches to determine the novelty of the sequences.<br>
Using such assays, an ovarian tumor antigen was identified that is splice fusion between the human T-cell leukemia virus type I oncoprotein TAX (see Jin et al, Cell 93:81l-91, 1998) and an extracellular matrix protein called osteonectin. A 25 splice junction sequence exists at the fusion point. The sequence of this clone is presented in Figure 4 and SEQ ID NO:75. Osteonectin, unspliced and unaltered, was also identified from such assays independently.<br>
Further clones identified by this method are referred to herein as 3f, 6b,<br>
8e, Sh, 12c and 12h. Sequences of these clones are shown in Figures 5 to 9 and SEQ ID<br>
30     NO:76 to  81.   Microarray analyses  were performed  as described above, and are<br>
presented in Figures 10 to 14.   A full length sequence encompassing clones 3f, 6b, 8e<br><br><br>
and 12h was obtained by screening an ovarian tumor (SCID-derived) cDNA library.<br>
This 2996 base pair sequence (designated 0772P) is presented in SEQ ID NO:311, and<br>
the encoded 914 amino acid protein sequence is shown in SEQ ID NO:312.   PSORT<br>
analysis indicates a Type 1 a transmembrane protein localized to the plasma membrane.<br>
5	In addition to certain of the sequences described above, tin's screen<br>
identified the following sequences which are described in detail in Table 1;<br><br><br><br><br><br>
This  screen   further identified  multiple  forms  of the  clone  0772P, referred to herein as 21013, 21003 and 21008.   PSORT analysis indicates that 21003<br>
	(SEQ ID NO:3S6;  translated as SEQ ID NO:3S9) and 2100S (SEQ ID NO:387;<br>
translated as SEQ ID NO:390) represent Type  la transmembrane protein forms of<br>
5 '   0772P.    21013 (SEQ ID NO:385; translated as SEQ ID NO:388) appeats to be a<br>
truncated form of the protein and is predicted by PSORT analysis TO be a secreted protein.<br>
Additional sequence analysis resulted in a full length clone for 08E<br>
(2627 bp, which agrees with the message size observed by Northern analysis; SEQ ID<br>
0     NO:39l). This nucleotide sequence was obtained as follows: the original 08E sequence<br>
   (Orig08Econs) was found to overlap by 33 nucleotides with a sequence from an EST<br>
clone (IMA GE#1987589).   This clone provided 1042 additional nucleotides upstream<br>
of the original 08E sequence.   The link: between the EST and 08E was confirmed by<br>
sequencing multiple PCR fragments generated from an ovary primary tumor library<br>
5     using primers to the unique EST and the OSE sequence (ESTxOSEPCR).   Pull length<br>
status was further indicated when anchored PCR from the ovary tumor library gave<br><br>
several clones (AnchoredPCR cons) that all terminated upstream of the putative start<br>
methionine, but failed to yield any additional sequence information. Figure 16 presents<br>
a diagram that illustrates the location of each partial sequence within the foil length<br>
08E sequence.<br>
	Two protein sequences may be translated from the full length 08E.  For<br>
"a" (SEQ ID NO:393) begins with a putative start methionine. A second form "b" (SE-Q ID NO:392) includes 27 additional upstream residues to the 5' end of the nucleotide sequence.<br>
EXAMPLE 3<br>
	This   example   discloses   the   identification   and   characterization   of<br>
antibody epitopes recognized by the 08E polyclonal anti-sera.<br>
Rabbit anti-sera was raised against E. coli derived 08E recombinant protein and tested for antibody epitope recognition against 20 or 21 mer peptides the correspond to the 08E amino acid sequence. Peptides spanning amino acid regions 31 15 to 65, 76 to 110, 136 to 200 and 226 to 245 of the full length 08E protein were recognized by an acid eluted peak and/or a salt eluted peak from affinity purified anti-08E sera. Thus, the corresponding amino acid sequences of the above peptides constitute the antibody epitopes recognized by affinity purified anti-08E antibodies.<br>
ELISA analysis of anti-08E rabbit sera is shown in Figure 23, and ELISA 20     analysis of affinity purified rabbit anti-OSE polyclonal antibody is shown in Figure 24. For epitope mapping, 20 or 21 mer peptides corresponding to the 08E protein were synthesized.   For antibody affinity purification, rabbit anti-O8E sera was, run over an 08E-sepharose column, then antibody was eluted with a salt buffer containing 0,5 M NaCI and 20 mM P04, followed by an acid elution step using 0.2 M 25     Glycine, pH 2.3.   Purified antibody was neutralized by the addition of 1M Tris, pH 8 and buffer Exchanged into phosphate buffered saline (PBS).    For enzyme linked immunosorbent assay (ELISA) analysis, O8E peptides and 08E recombinant protein were coated (into 96 well flat bottom plates at 2 Âµg/ml for 2.hours at room temperature (RT).  Plates were then washed 5 times with PBS + 0.1 % Tween 20 and blocked with 30     PBS + 1  % bovine serum albumin (BSA) for 1 hour.    Affinity purified anti-OSE antibody, eitlher an acid or salt eluted fraction, was then added to the wells at 1 Âµg/ml<br><br>
and incubated at RT for 1 hr.   Plates were again washed, followed by the addition of donkey anti-rabbit-Ig-horseradish peroxidase (HRP) antibody for 1 hour at RT.   Plates were washed, then developed by the addition of the chromagenfc substrate 3, 3', 5, 5'-tetramethylbenzidine (TMB) (described by Bos ei at, J. of Immunoassay 2:187-204 5.   (1981); available from. Sigma (St. Louis, MO)). The reaction was incubated 15 minutes at RT and then stopped by the addition of 1 N H2SO4.  Plates were read at an optical density of 450 (OD450) in an automated plate reader.    The sequences of peptides corresponding to the OE8 antibody epitopes are disclosed herein as SEQ ID NO: 394-415. Antibody epitopes recognized by the 08JE polyclonal anti-sera are disclosed herein 10     in Figure 17.<br>
EXAMPLE 4 This example discloses IHC analysis of O8E expression in ovarian cancer tissue samples.<br>
For immunohistochemistry studies, paraffin-embedded formalin fixed 15     ovarian cancer tissue was sliced into 8 micron sections.   Steam heat induced epitope retrieval (SHIER) in 0.1 M sodium citrate buffer (pH 6.0) was used for optimal staining conditions.    Sections were incubated with 10% serum/PBS for 5 minutes. Primary antibody (anti-08E rabbit affinity purified polyclonal antibody) was added to each section for 25 min followed by a 25 min incubation with an anti-rabbit biotinylated 20     antibody.   Endogenous peroxidase activity was blocked by three 1.5 min incubations with hydrogen peroxidase.  The avidin biotin complex/horse radish peroxidase system "was used along wit;; DAB-chromogen to visualize antigen expression.   Slides were counterstained with hematoxylin.    One (papillary serous carcinoma) of six ovarian cancer tissue sections displayed O8E immunoreactivity.    Upon optimization of the 25     staining  conditions,  4/5   ovarian   cancer  samples  stained  positive  using  the   08E polyclonal antibody. 08E expression was localized to the plasma membrane.<br>
Six ovarian cancer tissues were analyzed with the anti-OSE rabbit polyclonal antibody.    One (papillary serous carcinoma) of six ovarian cancer tissue samples stained positive for 08E expression.    08E expression was localized to the 30     surface membrane.<br><br>
EXAMPLE 5 This example discloses 08E peptides that.are predicted to bind HLA-A2 and to be immunogenic for CDS T cell responses in humans.<br>
Potential HLA-A2 binding peptides of 08E were predicted by using the 5 full-length open-reading frame (ORF) from 08E and running it through "Episeek," a program used to predict MHC binding peptides. The program used is based on the algorithm published by Parker, K.C. el al, J. Immunol. 152(1): 163-175 (1994) (incorporated by reference herein in its entirety). ] 0-mer and 9-mer peptides predicted to bind HLA-0201 are disclosed herein as SEQ ID NO: 416-435 and SEQ ID NO: 436-10    455, respectively.<br>
EXAMPLE 6 This  example  discloses  08E  cell   surface  expression  measured  by fluoresence activated cell sorting.<br>
For FACS analysis, cells were v/ashed with ice cold staining buffer 15     (PBS/1% BSA/azide).   Next, the cells were incubated for 30 minutes on ice with 10 micrograms/ml of affinity purified rabbit anti-B305D polyclonal antibody.   The cells were washed 3 times with staining buffer and then incubated with a 1:100 dilution of a goat anti-rabbit Ig (H+L)-F1TC reagent (Southern Biotechnology) for 30 minutes on ice. Following 3 washes, the cells were resuspended in staining buffer containing prodium 20     iodide, a vital stain mat allows for identification of permeable cells, and analyzed by FACS.    08E surface expression was confirmed on SKBR3 breast cancer cells and HEK293 cells that stably overexpress the cDNA for 08E.   Neither MB415 cells nor HEK293 cells stably transacted with a control irrelevant plasmid DNA showed surface expression of O8E (Figures I8 and 19).<br>
25	EXAMPLE 7<br>
This example further evaluates the Expression and surface localization of 08E.<br>
For expression and purification of antigen used for immunization, OSE<br>
Expressed in an E. coli recombinant expression system was grown overnight in LB<br>
30     Broth with the appropriate antibiotics at 37Â°C in a shaking incubator. The next morning,<br><br>
10 ml of the overnight culture was added to. 500 ml of 2x- YT plus appropriate antibiotics  in  a 2L-baffled  ErJenmeyer  flask.   When the  Optical  Density (at 56C nanometers) of the culture reached 0.4-0.6 the cells were induced with PTG (1 mM). 4 hours after induction with IPTG the cells were harvested by centrifugation. The cells 5     were   then   washed   with   phosphate   buffered   saline   and   centrifuged   again.   The supernatant was discarded and the cells were .either frozen for future use or immediately processed. Twenty milliliters of lysis buffer was added to the cell pellets and vortexed. To break open the E. coli cells, this mixture was then run through the French Press at a pressure of 16,000 psi. The cells were then centrifuged again and the supernatant and 10    pellet were checked by SDS-PAGE for the partitioning of the recombinant protein. For protein that localized to the cell pellet, the pellet was resuspended in 10 mM Tris pH 8.0 , 1% CHAPS and the inclusion body pellet was washed and centrifuged again. This procedure was repeated twice more. The washed inclusion body pellet was solubilized with either 8 M urea or 6 M guanidine HCl containing 10 mM Tris pH 8.0 plus 10 mM 15     imidazole. The solubilized protein was added to 5 ml of nickel-chelate resin (Qiagen) and incubated for 45 min to 1 hour at room temperature with continuous agitation. After incubation, the resin and protein mixture were poured through a disposable column and the flow through was collected. The column was then washed with 10-20 column volumes of the solubilization buffer. The antigen was then eluted from the column using 20     8M urea, 10 mM tris pH 8.0 and 300 mM imidazole and collected in 3 ml fractions. A SDS-PAGE gel was run to determine which fractions to pool for further purification. As a final purification step, a strong anion exchange resin such as Hi-Prep Q (Biorad) was equilibrated with the appropriate buffer and the pooled fractions from above were loaded onto the column. Each antigen was eluted off of the column with an increasing 25     salt gradient. Fractious were collected as the column was run and another SDS-PAGE gel was run to determine which fractions from the column to pool.,The pooled fractions were dialyzed against  10 mM Tris pH 8.0. This material was then evaluated for acceptable purity as determined by SDS-PAGE or HPLC, concentration as determined by Lowry assay or Amino Acid Analysis, identity as determined by amino terminal 30    protein sequence, and endotoxin level as determined by the Limulus (LAV) assay. The<br><br>
proteins were then vialed after filtration through a 0.22 micron filter and the antigens were frozen until needed for immunization.<br>
For generation of polyclonal anti-sera, 400 micrograms of each prostate antigen was combined with 100 micrograms" of muramyldipeptide (MDP). Equal 5 volume of Incomplete Freund's Adjuvant (IFA) was added and then mixed. Every four weeks animals were boosted with 100 micrograms of antigen mixed with an equal volume of IF A. Seven days following each boost the animal was bled. Sera was generated by incubating the blood at 4Â°C for 1244 hours followed by centrifugation.<br>
For characterization of polyclonal antisera, 96 well plates were coated 10     with antigen by incubating with 50 microliters (typically I microgram) at 40C for 20 hrs. 250 microliters of BSA blocking buffer was added to the wells and incubated at RT for 2 hrs. Plates were washed 6 times with PBS/0.01% tween. Anti-08E rabbit sera or affinity purified anti-08e antibody was diluted in PBS. "Fifty microliters of diluted antibody was added to each well and incubated at RT for 30 min. Plates were washed as " 15     described above before 50 microliters of goat anti-rabbit horse radish peroxidase (HRP) at a 1:1 0000 dilation was added and incubated at RT for 30 mm. Plates were washed as described above and 100 microliters of TMB microwell Peroxidase Substrate was added to each well. Following a 15 minute incubation in the dark at room temperature the colorimetric  reaction  was  stopped  with  100  microliters  of IN H2S04  and read 20     immediately at 450 run. All polyclonal antibodies showed immunoreactivity to the 08E antigen.<br>
For recombinant expression in mammalian HEK293 cells, full length 08E cDNA was subcloned into the mammalian expression vectors pcDNA3.1+ and pCEP4 (Invitrogen) which were modified to contain His and FLAG epitope tags, 25     respectively.  These constructs were transfected  into HEK293  cells (ATCC) using -Fugene 6 reagent (Roche). Briefly, HEK293 cells were plated at a density of 100,000 cells/ml in DMEM (Gibco) containing 10% FBS (Hyclone) and grown overnight. The following day, 2 ul of Fugene6 was added to 100 Âµl of DMEM containing no FBS and incubated for 15 minutes at room temperature. The Fugene6/DMEM mixture was then       added to lug of 08E/pCEP4 or 08E/pcDNA3.1 plasmid DNA and incubated for 15 minutes at room temperature. The Fugene/DNA mix was then added to the HEK293<br><br>
cells and  incubated for 48-72 hrs at 37oC with 7% C02. Cells were rinsed with PBS<br>
then collected and pelleted by centrifugation. For "Western blot analysis, whole c
lysates were generated by incubating the cells in Triton-Xl 00 containing lysis buffer f<br>
30 minutes on ice. Lysates were then cleared by centrifugation at   10,000rpm for<br>
5     minutes at 4 C. Samples were diluted with SDS-PAGE loading buffer containing bet<br>
mercaptoethanol, then boiled for 10 minutes prior to loading the SDS-PAGE ge<br>
Protein was transferred to nitrocellulose and probed using anti-08E rabbit polyclon;<br>
sera #2333L at a dilution of 1:750. The blot was revealed with a goat anti-rabbit 1<br>
coupled to HRP followed by incubation in ECL substrate.<br>
10	For FACS analysis, cells were washed further with ice cold stainin<br>
buffer (PBS+I%BSA+Azide). Next, the cells were incubated for 30 minutes on ice wit<br>
lOug/ml of Protein A purified anti-08E polyclonal sera. The cells were washed 3 time<br>
with staining buffer and then incubated with a 1:100 dilution of a goat anti-rabbi<br>
Ig(H+L)-FITC reagent (Southern Biotechnology) for 30 minutes on ice. Following :<br>
15     washes, the cells were resuspended in staining buffer containing Propidium Iodide (PI).<br>
a vital stain that allows for the identification of permeable cells, and analyzed by FACS.<br>
From these experiments, the results of which are illustrated in Figures<br>
20-21, OSE expression was detected on the surface of transfected HEK293 cells and<br>
SKBR3  cells by FACS analysis using rabbit anti-08E sera.    Expression was also<br>
20     detected in transfected HEK293 cell lysates by Western blot analysis (Figure 22).<br>
EXAMPLE 8 GENERATION AND CHARACTERIZATION OF ANTJ-08E MABS.<br>
Mouse monoclonal antibodies were raised against E. coli derived 08E proteins as follows.   A/J mice were immunized intraperi tone ally (IP) with Complete 25     Freund's Adjuvant (CFA) containing 50 Âµg recombinant 08E, followed by a subsequent IP boost with Incomplete Freund's Adjuvant (IFA) containing lOÂµg recombinant 08E  protein.     Three days, prior to removal of the spleens, the mice were immunized intravenously with approximately 50ug of soluble 08E recombinant protein.    The spleen  of a mouse with a positive titer to OSE was removed,  and a single-cell 30     suspension made and used for fusion to SP2/0 myeloma cells to generate B cell<br><br><br>
hybridomas. The supematants from the hybrid clones were tested by ELISA for specificity to recombinant OSE, and epitope inapped using peptides that spanned the entire. OSE sequence. The mAbs were also tested by flow cytometry for their ability to detect 08E on the surface of cells stably transfected with OSE and on the surface of a 5     breast tumor cell line.<br>
For ELISA analysis, 96 well plates were coated with either recombinant<br>
08E protein or overlapping 20-mer peptides Spanning the entire OSE molecule at a<br>
concentration of either l-2pÂµg/ml or 10Âµg/ml, respectively.   After coating, the plates<br>
were washed 5 times with washing buffer (PBS + 0.1% Tween-20) and blocked with<br>
10     PBS containing 0.5% BSA, 0.4% Tween-20.   Hybrid supernatants or purified mAbs<br>
were then added and the plates incubated for 60 minutes at room temperature.   The<br>
plates were washed 5 times with washing buffer and the secondary antibody, donkey-<br>
anti mouse 1glinked to  horseradish peroxidase- (HRP)(Jackson Immuno Research), was<br>
added for 60 minutes.    The plates were again washed 5 times in washing buffer,<br>
15     followed  by the addition  of the peroxidase  Substrate.     Of the hybridoma clones<br>
generated, 15 secreted mAbs that recognized the entire OSE protein.   Epitope mapping<br>
revealed that of these IS clones, 14 secreted mAbs that recognized the 08E amino acid<br>
residues 61-80 and one clone secreted a mAbs that recognized amino acid residues 151-<br>
170.<br>
20	For flow cytometric analysis, HEK293  cells which had been stably<br>
transfected with 08E and SKBR3 cells which express OSE mRNA,- were harvested and washed in flow staining buffer (PBS+1%BSA+Azide). The cells were incubated with the supernatant from the mAb hybrids for 30 minutes on ice followed by 3 washes with staining buffer. The cells were incubated with goat-anti mouse Ig-FTTC for 30 minutes 15 on ice, followed by three washes with staining buffer before being resuspended in wash buffer containing propidium iodide. Flow cytometric analysis revealed that 15/15 mAbs were able to detect OSE protein expressed on the surface of OSE-transfected HEK293 cells. 6/6 rnAbs tested on SKBR3 cells were able to recognize surface expressed 08E.<br><br>
EXAMPLE 9 EXTENDED DNA AND PROTEIN SEQUENCE ANALYSIS OF SEQUENCE 0112?<br>
A full-length sequence encompassing clones 3f, 6b, 8e, and 12 WE<br>
obtained by screening an ovarian tumor (SCID-derived) cDNA library described i<br>
5     detail in Example 2.   This 2996 base pair sequence, designated 0772P, is presented i:<br>
SEQ ID NO: 311, and the encoded 914 amino acid protein sequence is shown in SE(<br>
ID NO:  312.    The DNA  sequence  0772P was searched against public database.<br>
including  Genbank  and  showed  a  significant hit  to   Genbank Accession number<br>
AK024365 (SEQ ID NO: 457).   This Genbank sequence was found to be 3557 base<br>
10    pairs in length and encodes a protein 1156 amino acids in length (SEQ ID NO: 459). A<br>
truncated version of tin's sequence, residues 25-3471, in which residue 25 corresponds<br>
to the first ATG initiation codon in the GenbaruV sequence, (SEQ ID NO: 456), encodes<br>
a protein that is 1148 amino acids in length (SEQ ID NO: 458).  The published DNA<br>
sequence (SEQ ID NO: 457) differs from 0772P in that it has a 5 base pair insertion<br>
15     corresponding to bases 958-962 of SEQ ID NO: 457.  This insertion results in a frame<br>
shift such that SEQ ID NO: 457 encodes an additional N-terminal protein sequence<br>
relative to 0772P (SEQ ID NO: 312).  In addition, 0772P encodes a unique N-terminal<br>
portion contained in residues 1-79 (SEQ ID NO: 460).  The N-terminal portion of SEQ<br>
ID NO: 456, residues 1-313, also contains unique sequence and is listed as SEQ ID NO:<br>
20    461.<br>
EXAMPLE 10<br>
THE GENERATION OF POLYCLONAL ANTIBODIES FOR IMMUNOHISTOCHEMISTRY<br>
AND FLOW CYTOMETRIC ANALYSIS OF THE CELL ASSOCIATED EXPRESSION<br>
PATTERN OF MOLECULE 0772P<br>
25	The  0117? molecule  was  identified in Examples 2 and  9 of this<br>
application. To evaluate the subcellular localization and specificity of antigen expression in various tissues, polyclonal antibodies were generated against 0772P. To produce these antibodies, 0772P-1 (amino acids 44-772 of SEQ ID N0:312) and 0772P-2 (477-914 of SEQ ID NO:312) were expressed in an E. coli recombinant<br>
30     expression system and grown overnight at 37Â°C in LB Broth.  The following day, 10ml<br><br>
of the overnight culture was added to 500ml of 2xYT containing the appropriate antibiotics.  When the optical density of the cultures (560 nanometers) reached 0.4-0.6 the cells were induced with IPTG.    Following induction, the cells were harvested, washed, lysed and run through a French Press at a pressure of 16000 psi. The cells were then centrifuged and the pellet checked by SDS-PAGE for the partitioning of the recombinant protein.    For proteins that locali2;e to the cell pellet, the pellet was resuspended in lOmM Tris, pH 8.0, 1% CHAPS and the inclusion body pellet washed and centrifuged. The washed inclusion body was solubilized with either 8M urea or 6M guanidine HCL containing lOmM Tris, pH 8.0, plus lOmM imidazole.  The solubilized protein was then added to 5ml of nickel-chelate resin (Qiagen) and incubated for 45 minutes at room temperature.<br>
Following the incubation, the resin and protein mixture was poured through a column and the flow through collected.  The column was washed with 10-20 column volumes of buffer and the antigen eluted using 8M. urea, lOmM Tris, pH 8.0, and 300  mM imidazole  and  collected  in  3ml  fractions.     SDS-PAGE  was run to determine which fractions to pool for further purification. As a final purification step, a strong anion exchange resin was equilibrated with the appropriate buffer and the pooled fractions were loaded onto the column.  Each antigen was eluted from the column with an increasing salt gradient  Fractions were collected and analyzed by a SDS-PAGE to determine which fractions from the column to pool. The pooled fractions were dialyzed against lOmM Tris, pH 8.0, and the resulting protein was submitted for quality control for final release.  The release criteria were: (a) purity as determined by SDS-PAGE or HPLC, (b) concentration as determined by Lowry assay or Amino Acid Analysis, (c) identity  as  determined  by  amino  terminal  protein,  and   (d)  endotoxin   levels  as determined by the Limulus (LAL) assay.    The proteins were then filtered through a 0.22pM filter and frozen until needed for immunizations.<br>
To generate polyclonal antisera, 400Âµg of 0772P-1 or 0772P-2 was combined with lOOÂµg of muramyldipeptide (MDP). The rabbits were immunized every 4 weeks with lOOÂµg of antigen mixed with an equal volume of Incomplete Freund's Adjuvant (IFA). Seven days following each boost, the animals were bled and sera was generated by incubating the blood at 4Â°C for 12-24 hours followed by centrifugation.<br><br>
Injection of Insulin 2 may reduce the incidence of diabetes to about 30% when given at weeks 8 and 10 (Group 4). When given at a later time point, group 5, no significant reduction in the incidence of diabetes is expected. When the veto vector is given early (Group 6), a significantly more pronounced reduction in the incidence of diabetes is expected as compared to Group 2. Less than 20% of animals are expected to develop diabetes. A delay in diabetes delivery is also expected. No more than 30% of animals in Group 7 (compared to more than 70% in Group 5) are expected to develop diabetes. A similar delay in diabetes onset is expected.<br>
In another experiment, female NOD mice are injected with a CD8 a-chain / insulin vector prior to the disease onset starting at two-weeks of age to determine whether ~ the overt onset of diabetes is delayed, if not prevented in these animals. The peripheral blood glucose levels are measured. In addition, mice are sacrificed at different periods of time, and the pancreatic islets undergo histological examination for evidence of insulitis The results are compared with studies in which the respective control vectors (insulin only, CD8 a-chain only) are injected.<br>
The vector treatment is given at later stages of the disease to determine whether vectors can interfere with later stages of the disease development, such as insulttis. Finally, animals with evidence of overt diabetes are treated.<br>
If the development of diabetes is prevented, studies are undertaken to investigate whether a concomitant inhibition of T lymphocytes will be observed. For this purpose T lymphocytes are harvested from these mice and studied for their ability to respond to insulin. Adoptive transfer experiments are included to look at any evidence of regulatory cell induction.<br><br>
For  Western  Blot   analysis,   -whole  cell   lysates  were  generated  by<br>
incubating the cells in lysis buffer followed by clarification by centrifugation.    The<br>
samples were  diluted  and run on SDS-PAGE.    The gel was then transferred to<br>
nitrocellulose and probed using purified anti-0772P-2 rabbit polyclonal antibody.  The<br>
     blot was revealed with a goat anti-rabbit Ig coupled to HRP followed by incubation in<br>
ECL substrate.  Western Blot analysis revealed that 0772P-21008 could be detected in<br>
HEK293 cells that had been transfected with 0772P.<br>
To determine the cell expression profile of 0772P in cells, primary ovarian tumor cells were grown in SCID mice.  The cells were retrieved from the mice and  analyzed  by flow cytometry.     Briefly,  cells  washed  in  cold staining buffer containing PBS, 1% BSA, and Na Azide. The cells were incubated for 30 minutes with lOÂµg/ml of purified  anti-0772P-l   and 0772P-2 polyclonal  sera.     Following this incubation, the cell's were washed three times in staining buffer and incubated with goat anti-rabbit Ig (H+L) conjugated to FITC (Southern Biotechnology).   The cells were washed and resuspended in staining buffer containing Propidium Iodide (PI), a vital stain that identifies non-viable cells.   The cells were then analyzed using Fluorescence Activated Cell Sorting (FACS). FACS analysis revealed that 0772P was present on the cells surface.  Surface expression of 0772P on tumor cells allows for immune targeting by therapeutic antibodies.<br>
EXAMPLE 12<br>
FUNCTIONAL CHARACTERIZATION OF ANTI-QSE MONOCLONAL ANTIBODIES<br>
Mouse monoclonal antibodies (mAb) raised against E. coli derived 08E, as described in Example 8, were tested for their ability to promote 08E antigen internalization. Internalization of the antibody was determined using an in vitro cytotoxicity assay. Briefly, HEK293 and O8E/HEK transfected cells were plated into 96 well plates containing DME plus 10% heat-inactivated FBS in the presence of 50Âµg/well of purified anti-08E or control antibodies. The isotype of the anti-08E mAbs are as follows: llA6-IgGl/kappa, 15C6-IgG2b/kappa, 18A8-IgG2b/kappa, and 14Fl-IgG2a/lcappa. W6/32 is a pan anti-human MHC class I mouse monoclonal antibody that serves as a positive control, and two irrelevant mAbs, Ir-Pharm and Ir-<br><br>
Crxa were included as negative controls.  Following incubation with the OSE specify antibodies or the relevant controls antibodies, the mAb-zap,  a goat anti-mouse Ig saporin conjugated secondary antibody (Advanced Targeting Systems) was added at a concentration of lOOng/ml to half of the wells, and the plates were incubated for 48 tc 5     72.hours at 37Â°C in a 7% CO2 incubator. This assay takes' advantage of the toxic nature of saporin, a ribozyme inactivating protein, which when internalized has a cytotoxic effect.   Following incubation with the mAb-zap, internalization was quantitated by the addition of MTS reagent followed by reading the OD490 of the plate on a microplate ELISA reader. Figure 25 depicts the results from these assays. The top panel represents      HEK cells that have not been transfected with 08E and therefore OSE antibody should not bind and be internalized.   Levels of proliferation were the same in.all samples whether they were incubated with or without the rnAb-zap, with the exception of the positive control Ab, W6/32. The lower panel represents cells that have been transfected with OSE and therefore should bind OSE specific antibodies.    Antibodies from the hybridomas 11H6, 14F1, and 1SC6, which recognize the amino acids 61-80 of OSE were able to promote internalization of the  08E surface protein as measured by decreased levels of proliferation due to the toxic nature of the mAb-zap (See Figure 25). The antibody generated by the hybridoma 1SA8, which recognizes amino acids 151-170 of 08E, was unable to promote internalization as determined by normal levels of proliferation either in the absence or presence of the mAb-zap.<br>
EXAMPLE 13 CHARACTERIZATION OF THE OVARIAN TUMOR ANTIGEN, 0772P<br>
The cDNA and protein, sequences for multiple forms, of the ovarian tumor antigen 0772P have been described in the above (e.g., Examples 2 and 9). A Genbanlc search indicated that 0772P.has a high degree of similarity with FLJ14303 (Accession # AK024365; SEQ ID NO:457 and 463). Protein sequences corresponding to 0772P and FLJ14303 are disclosed in SEQ ID NO:478 and 479, respectively. FLJ14303 was identical to the majority of 0772P, with much of the 3-end showing 100% homology. However, the 5'-end of FLJ14303 was found to extend further 5' uhan 0772P.  In addition, FLJ14303 contained a 5 bp insert (SEQ ID NO:457) resulting in a<br><br><br>
frame shift of the ami no-terminus protein sequence such that FLJ14303 utilizes  a<br>
different starting methionine than 0772P and therefore encodes a different protein<br>
This insertion was present in the genomic sequence and seen in all EST clones thai<br>
showed identity to this region, suggesting that FLJ14303 (SEQ ID NO:457) represents a<br>
splice variant of 0117?, with an ORE that contains an extended and different amino-<br>
terminus.   The additional 5'-nucleotide sequence included repeat-sequences that were<br>
identified during the genomic mapping of 0772P.    The 5'-end of 0112? and the<br>
corresponding region  of FLJ14303   showed  between  90-100% homology.    Taken<br>
together, this suggests that 0772P and FLJ14303 are different splice variants of the<br>
same gene, with different unique repeat sequences being spliced into the 5'-end of the<br>
gene.<br>
The identification of an additional ten or more repeat sequences within the same region of chromosome 19, indicates that there may be many forms of 0772P, each with a different 5'-end, due to differential splicing of different repeat sequences. Northern blot analysis of 0112? demonstrated multiple 0772P-hybridizing transcripts of different sizes, some in excess lOkb.<br>
Upon  further analysis,   13  additional  0772P-related  sequences  were identified, the cDNA and amino acid sequences of which are described in Table 2.<br>
Table 2<br><br><br><br>
nd=not determined<br>
Initially it appeared that these sequences represented overlapping and/ discrete sequences of 0112V splice forms that were capable of encoding polypeptid unique to the specific splice forms of 0772P. However, nucleotide alignment of the sequences failed to identify any identical regions within the repeat elements. This. indicates that the sequences may represent different specific regions of a single 772P gene, one that contains 16 or more repeat domains, all of which form a single line transcript. The 5'-end of sequence LS #1043400.10 (Table 2; SEQ ID NO:465) unique to both 0112V and FLJ14303 and contains no repeat elements, indicating th this sequence may represent the 5'-end of 0112V.<br>
Previously, transmembrane prediction analysis had indicated that 0772 contained between 1 and 3 transmembrane spanning domains.  This was verified by th<br><br>
use of immunohis to chemistry and flow cytometry, which demonstrated the existence c<br>
a    plasma    membrane-associated    molecule    representing    0112?.	Howeven<br>
immunohistochemistry also indicated the presence of secreted' form(s) of 0112V possibly resulting from an alternative splice form of 0112V or from a post-translationa cleavage event. Analysis of several of the sequences presented in Table 2 showed the sequences 1043400B.12, 1043400.8B, and 1043400.1 IB all contained transmembrane regions, while 10434.00,8A 1043400.10, 1043400.1, 1043400.11A, and 1043400.9 were all lacking transmembrane sequences, suggesting that these proteins may be secreted.<br>
Analysis indicates a part of 0112V is expressed and/or retained on the plasma membrane, making 0112? an attractive target for directing specific immunotherapies, e.g., therapeutic antibodies, against this protein. The predicted extracellular domain of 0772P is disclosed in SEQ ID NO:489 and secretion of 0772P is likely to occur as a result of a cleavage event within the sequence:<br>
SLVEQVFLDATLNASFHWLGSTYQLVDIHVTEMESSVYQP. Proteolytic cleavage is most likely to occur at the Lysine (K) at position 10 of SEQ ID NO;489.   The extracellular, transmembrane, and cytoplasmic regions of 0772P are all disclosed in SEQ ID NO:488: Extracellular:<br>
SLVEQVFLDKTLNASFHWLGSTYQLVDIHVTEMESSVYQPTSSSS TQHFYLNFTITNLPYSQDKAQPGTTNYQRNKRNIEDALNQLFRNSSIKSYFSDCQ VSTFRSVPNRHHTGVDSLCNFSPLAJRRVDRVAIYEEFLRMTKNGTQLQNFTLDR SSVLVDGYFPNRNEPLTGNSDLPF Transmembrane:<br>
WAVILIGLAGLLGLITCLICGVL VTT Cytoplasmic: RRRIKKEGEYNVQQQCPGYYQSHLDLEDLQ<br><br>
EXAMPLE 14 IMMUNOHIST0CHEM1STRY (IHC) ANALYSIS OF 08E EXPRESSION IN OVARIAN CANCER<br>
AND NORMAL TISSUES<br>
In order to determine which tissues express the ovarian cancer antigen 08E, IHC analysis was performed on a diverse range of tissue sections using both polyclonal and monoclonal antibodies specific for 08E. The generation of 08E specific polyclonal antibodies is described in detail in Example 8. The monoclonal antibodies used for staining were 11A6 and 14F1, both of which are specific for amino acids 61-80 of O8E and 18AS, which recognizes amino acids 151-170 of 08E (see Example 12 for details on generation).<br>
To perform staining, tissue samples were fixed in formalin solution for 12-24 hours and embedded in paraffin before being sliced into 8 micron sections. Steam heat induced epitope retrieval (SHEIR) in 0.1M sodium citrate buffer (pH 6.0) was  used   for  optimal   staining  conditions.      Sections  were  incubated  with   10% serum/PBS for 5 minutes.   Primary antibody was then added to each section for 25 minutes followed by 25 minutes of incubation with either anti-rabbit or anti-mouse biotinylated antibody. Endogenous peroxidase activity was blocked by three 1.5 minute incubations  with  hydrogen peroxidase.     The  avidin  biotin  complex/horse radish peroxidase (ABC/HRP) system was used along with DAB chromogen to visualize the antigen expression.   Slides were counterstained with hematoxylin to visualize the cell nuclei.<br>
Results using rabbit affinity purified polyclonal antibody to 08E (a.a. 29-283; for details on the generation of this Ab, see Example 3) are presented in Table 3. Results using the three monoclonal antibodies are presented in Table 4.<br>
Table 3 Immunohitochemistry analysis of 08E using polyclonal antibodies -<br>
Tissue	08E Expression	<br>
	Ovarian Cancer	Positive	<br>
Breast Cancer		Positive	<br><br><br><br><br>
EXAMPLE 15 EPITOPE MAPPING OF 0772P POLYCLONAL ANTIBODIES<br>
To perform epitope mapping of 0772P, peptides were generated, the sequences of which were derived from the sequence of 0772P. These peptides were 15      mers that overlapped by 5 amino acids and were generated via chemical synthesis on membrane supports. The peptides were covalently bound to Whatman 50 cellulose support by their C-terminus with the N-terminus unbound.    In order to determine epitope specificity, the membranes were wet with ]00% ethanol for 1 minute, and then blocked for 16 hours in IBS/Tween/Triton buffer (50mM Tris, 137 mM NaCl, 2.7 mM      KC1, 0.5% BSA, 0.05% Tween 20, 0.05% Triten-X-100, pH 7.5),   The peptides were then probed with 2 0772P specific antibodies, 0772P-I (amino acids 44-772 of SEQ ID NO;312) and 0772P-2 (477-914 of SEQ ID N'0:312; see Example 10 for details of antibody generation), as well as irrelevant rabbit antibodies for controls. The antibodies were diluted  to  lÂµg/ml  and  incubated with the membranes for 2 hours at room i     temperature.   The membranes were then washed for 30 minutes in TBS/Tween/Triton buffer, prior to being incubated with a 1:10,000 dilution of HRP-conjugated anti-rabbit secondary antibody for 2 hours.  The membrane; were again washed for 30 minutes in TBS/Tween/Triton and anti-peptide reactivity was visualized using ECL.    Specific epitope binding specificity for each of the O772P, -polyclonal antibodies is described in Table 5.<br><br><br><br>
EXAMPLE 16 IDENTIFICATION OF A NOVELN-TERMIMAL REPEAT STRUCTURE ASSOCIATED WITH<br>
0772P<br>
Various 0112? cDNA and protein forms have been identified and characterized as detailed above (e.g., Examples 1, 2, 9, and 14),   Importantly, 0772P RNA and protein have been demonstrated to be over-expressed in ovarian cancer tissue relative to normal tissues and thus represents an attractive target for ovarian cancer." diagnostic and therapeutic applications.<br>
Using bioinforrnatic analysis of open reading frames (OP-Ps) from genomic nucleotide sequence identified previously as having homology with 0772?, multiple nucleotide repeat sequences were identified in the 5' region of the gene encoding the 0772P protein. A number of these repeat sequences were confirmed by , RT-PCR using primers specific for the individual repeats. Fragments which contained multiple repeats were amplified from cDNA, thus confirming the presence of specific repeats and allowing an order of these repeats to be established.<br>
Unexpectedly, -when various sets of 0772P sequences derived from different database and laboratory sources were analyzed, at least 20 different repeat structures, each having substantial levels of identity with each other (see Table 6), were identified in the 5' region of the 0772P gene and the corresponding N-terminal region of the 0772P protein. Each repeat comprises a contiguous open reading frame encoding a polypeptide unit that is capable of being spliced to one or more other repeats such that concatomers of the repeats are formed in differing numbers and orders. Interestingly, other molecules have been described in the scientific literature that have repeating structural domains analogous to those described herein for 0772P.   Tor example, the<br><br>
mucin family of proteins, which are the major glycoprotein component of the mucous which coats the surfaces of cells lining the respiratory, digestive and urogenital tracts have been shown to be composed of tandemly repeated sequences that vary in number. length and amino acid sequence from one mucin to another (Perez-Vilar and Hill, J. Biol Chem. 274(45):31751-31754, 1999).<br>
The various identified repeat structures set forth herein are expected to give rise to multiple forms of 0772P, most likely by alternative splicing. The cDNA sequences of the identified repeats are set forth in SEQ ID NOs:513-540, 542-546, and 548-567. The encoded amino acid sequences of the repeats are set forth in SEQ ID NOs:574-593. In many instances these amino acid sequences represent consensus sequences that were derived from the alignment of more than one experimentally derived sequence.<br>
Each of these splice forms is capable of encoding a unique 0772P protein with multiple repeat domains attached to a constant carboxy terminal protein" portion of 0172P that contains a trans membrane region.    The cDNA sequence of the 0112? constant region is set forth in SEQ ID NO:568 and the encoded amino acid sequence is set forth in SEQ ID NO:594.<br>
All of the available 0112V sequences that were obtained were broken down into their identifiable repeats and these sequences were-compared using the Clustal   method  with   weighted   residue, weight  table   (MegAlign   software  within DNASTAR sequence analysis package) to identify the relationship between the repeat sequences.   Using this information, the ordering data provided by the RT-PCR, and sequence   alignments   (automatic and  manual)   using   SeqMan   (DNASTAR),   one illustrative consensus full length 0112V contig was identified comprising 20 distinct repeat units.   The cDNA for this 0772P c-DNA contig is set forth in SEQ ID NO:569 and the encoded amino acid sequence is set forth in SEQ ID NO:595.    This form of the 0112V protein includes the following consensus repeat structures in the following order:<br>
SEQ ID NO:572- SEQ ID NO:574- SEQ ID NO:575-SEQ ID NO:576-SEQ ID NO:577- SEQ ID NO:578- SEQ ID NO:579- SEQ ID NO:580- SEQ ID NO:58l- SEQ ID NO:582- SEQ ID NO:583- SEQ ID NO:584- SEQ ID NO:585- SEQ<br><br>
ID NO:586- SEQ ID NO:587- SEQ ID NO:5S8- SEQ ID NO:589- SEQ ID NO:590-SEQ ID NO:591- SEQ ID NO:592- SEQ ID NO:593.<br>
SEQ   ID   NO:595.   therefore,   represents   one   illustrative   full-length consensus sequence for the 0772P protein.   As discussed above, however, based on current knowledge  of this protein  and  based upon scientific literature  describing proteins containing analogous repeating structures, many other forms of 0772P are expected to exist with either more or Jess repeats,   in addition, many forms of 0772P are expected to have differing arrangements, e.g., different orders, of these N-terminal repeat structures.   The existence of multiple forms of 0772P having differing numbers of repeats is supported by Northern  analysis of 0112P,.    In this study, Northern hybridization of a 0772P-specific probe resulted in a smear of multiple 0772P-hybridizing transcripts, some in excess lOkb.<br>
Thus, the variable repeat region of the 0772 protein can be illustratively represented by the structure Xn - Y, wherein X comprises a repeat structure having at least 50% identity with the consensus repeat sequence set forth in SEQ ID NO:596; n is the number of repeats present in the protein and is expected to typically be a integer from ] to about 35; Y comprise the 0772P constant region sequence set forth in SEQ ID NO:594 or sequences having at least 80% identity with SEQ ID NO:594. Each X present in the Xn repeat region of the 0772 molecule is different.<br>
To determine the consensus sequences of each of the 20 repeat regions, sequences thatwere experimentally determined for a discrete repeat region were aligned and a consensus sequence determined. In addition to determining me consensus sequences for individual repeat regions, a consensus, repeat sequence was also determined. This sequence was obtained by aligning the 20 individual consensus sequences. Variability of the repeats was determined by aligning the consensus amino acid sequences from each of the individual repeat regions with the over all repeat consensus sequence. Identity data is presented in Table 6.<br><br><br><br>
various modifications may be made without deviating from the spirit and scope of the invention. Accordingly, the invention is not limited except as by the appended claims.<br><br>
WE CLAIM:<br>
1.	An isolated antibody, or antigen binding fragment thereof, which<br>
specifically binds to an 08E polypeptide at an amino acid sequence selected from the<br>
group consisting of SEQ ID NOs: 396-400, 403-406, 409 and 413-415.<br>
2.	The isolated antibody or antigen binding fragment of claim 1, wherein the<br>
antibody or antigen binding fragment is conjugated to a toxin.<br>
3.	The isolated antibody, or antigen binding fragment of claim 2, wherein the toxin is selected from the group consisting of a ricin toxin, abrin toxin, diptheria toxin, cholera- toxin, gelonin toxin, Pseudomonas exotoxin, Shigella toxin, and pokeweed antiviral protein.	<br>
4.	The isolated antibody, or antigen binding fragment of claim 1, wherein the antibody or antigen binding fragment is conjugated to a radionuclide.<br>
5.	The isolated antibody, or antigen binding fragment of claim 4, wherein the radionuclide is selected from the group consisting of 90Y, ,231,125I, l3,I, 186Re, l88Re, 211At and 212Bi.<br>
6.	The isolated antibody, or antigen binding fragment of claim 1, wherein the antibody or antigen binding fragment binds to a polypeptide encoded by SEQ ID NO: 391, and wherein said polypeptide is expressed on the cell surface of a ovarian tumor cell.<br>
7.	The isolated antibody, or antigen binding fragment of claim 1, wherein the antibody or antigen binding fragment induces internalization in an in vitro cytotoxicity assay upon binding a cell surface-expressed polypeptide encoded by SEQ ID NO: 391.<br><br>
1-7, wherein the antibody or antigen.binding fragment is a human isolated antibody, or antigen binding fragment.<br>
9.	A pharmaceutical composition comprising at least one antibody or antigen<br>
binding fragment as in any one of claims 1 -7, and a physiologically acceptable carrier, for<br>
use in treating ovarian cancer.<br>
10.	A diagnostic kit comprising at least one antibody or antigen binding<br>
fragment as in any one of claims 1-7, and a detection reagent, for use in diagnosing<br>
ovarian cancer.<br>
11.	The diagnostic kit of claim 10, wherein the detection reagent comprises a reporter group.<br>
12.	A pharmaceutical composition comprising a sufficient amount of an isolated antibody, or antigen binding fragment as in any one of claims 1-7, for killing or inhibiting the growth of a tumor cell which expresses a polypeptide encoded by SEQ ID NO: 391.<br>
13.	A pharmaceutical composition comprising a therapeutically effective amount of an isolated antibody, or antigen binding fragment as in any one of claims 1-7, for treating ovarian cancer in a patient, wherein said cancer expresses a polypeptide encoded by SEQ ID NO; 391.<br>
14.	A method for making an isolated antibody, comprising immunizing a mouse with a peptide having an amino acid sequence selected from any one of SEQ ID NOs: 396-400, 403-406, 409, and 413-415.<br><br>
15.	An isolated polypeptide consisting of an amino acid sequence selected from the group consisting of: SEQ ID NOs: 396-400, 403-406, 409, and 413-415.<br>
16.	An isolated antibody, or antigen binding fragment thereof that specifically binds a polypeptide of claim 15.<br>
17.	An isolated polynucleotide sequence encoding a polypeptide of claim 15.<br>
18.	A pharmaceutical composition comprising,<br>
(a)	a polypeptide of claim 15, or<br>
(b)	an isolated antibody or antigen binding fragment as in any one of<br>
claims 1-7; or<br>
(c)	a polynucleotide of claim 17; and<br>
a physiologically acceptable carrier, for use in treating ovarian cancer.<br>
19.       A vaccine comprising,<br>
(a)	a polypeptide of claim 15, or<br>
(b)	an isolated antibody or antigen binding fragment as in any one of<br>
claims 1-7; or<br>
(c)	a polynucleotide of claim 17; and<br>
a non-specific immune response enhancer, for use in preventing ovarian cancer.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQUJTVFJBQ1QoMTAtOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-ABSTRACT(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtYWJzdHJhY3QoMTEtMi0yMDA4KS5kb2M=" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-abstract(11-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtYWJzdHJhY3QoMTEtMi0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-abstract(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQUJTVFJBQ1QoMTYtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-ABSTRACT(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQUJTVFJBQ1QoR1JBTlRFRCktKDI3LTMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-ABSTRACT(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ0FOQ0VMTEVEIFBBR0VTKDE2LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CANCELLED PAGES(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY2xhaW1zKDExLTItMjAwOCkuZG9j" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-claims(11-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY2xhaW1zKDExLTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-claims(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ0xBSU1TKDI5LTA3LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CLAIMS(29-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ0xBSU1TKEFNRU5ERUQpLSgxMC04LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CLAIMS(AMENDED)-(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ0xBSU1TKEFNRU5ERUQpLSgxNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CLAIMS(AMENDED)-(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ0xBSU1TKEdSQU5URUQpLSgyNy0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CLAIMS(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMS0xMC0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(1-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMTAtMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(10-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMTgtOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(18-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMjEtNC0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(21-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMjQtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(24-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoMjktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(29-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtQ09SUkVTUE9OREVOQ0UoSVBPKS0oMjctMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-CORRESPONDENCE(IPO)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY29ycmVzcG9uZGVuY2Utb3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtY29ycmVzcG9uZGVuY2UtcmVjZWl2ZWQucGRm" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZGVzY3JpcHRpb24oY29tcGxldGUpLSgxMS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-description(complete)-(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtREVTQ1JJUFRJT04oR1JBTlRFRCktKDI3LTMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-DESCRIPTION(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRFJBV0lORygxMS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-DRAWING(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRFJBV0lORyhHUkFOVEVEKS0oMjctMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-DRAWING(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAxKDEwLTgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 1(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAxKDE2LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 1(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAxMygxNC01LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 13(14-5-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAxMygyMS00LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 13(21-4-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAxMygyOS03LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 13(29-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAxOCgyOS0wNy0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 18(29-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAyKDExLTItMjAwOCkuZG9j" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 2(11-2-2008).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAyKDExLTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 2(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAyKEdSQU5URUQpLSgyNy0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 2(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAyKFRJVExFIFBBR0UpLSgxMC04LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 2(TITLE PAGE)-(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybSAyKHRpdGxlIHBhZ2UpLSgxMS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form 2(title page)-(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAyKFRJVExFIFBBR0UpLSgxNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 2(TITLE PAGE)-(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtRk9STSAyKFRJVExFIFBBR0UpLShHUkFOVEVEKS0oMjctMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-FORM 2(TITLE PAGE)-(GRANTED)-(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybS0yLmRvYw==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtZm9ybS1wY3QtaWItMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-form-pct-ib-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtT1RIRVIgRE9DVU1FTlQoMjktMDctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-OTHER DOCUMENT(29-07-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtUENULUlCLTMwNCgxLTEwLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-PCT-IB-304(1-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMTAtMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-PETITION UNDER RULE 137(10-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtcG93ZXIgb2YgYXV0aG9yaXR5KDEtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-power of authority(1-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtUE9XRVIgT0YgQVVUSE9SSVRZKDEwLTgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-POWER OF AUTHORITY(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDEwLTgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-REPLY TO EXAMINATION REPORT(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtUkVQTFkgVE8gSEVBUklORygxNi0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-REPLY TO HEARING(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtU0VRVUVOQ0UgTElTVElORygxMS0yLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-SEQUENCE LISTING(11-2-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtU0VRVUVOQ0UgTElTVElORygyNy0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-SEQUENCE LISTING(27-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLW11bW5wLTIwMDgtc2VxdWVuY2UgbGlzdGluZygyOC03LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">241-mumnp-2008-sequence listing(28-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtU1BFQ0lGSUNBVElPTihBTUVOREVEKS0oMTAtOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-SPECIFICATION(AMENDED)-(10-8-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQxLU1VTU5QLTIwMDgtU1BFQ0lGSUNBVElPTihBTUVOREVEKS0oMTYtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">241-MUMNP-2008-SPECIFICATION(AMENDED)-(16-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="251649-method-and-mould-of-manufacturing-a-ring-shaped-friction-lining.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251651-multilayer-polymer-sheets.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251650</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>241/MUMNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>13/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Mar-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Mar-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Feb-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CORIXA CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1124 COLUMBIA STREET, SUITE 200, SEATTLE, WASHINGTON 98104,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JENNIFER L. MITCHAM</td>
											<td>16677 N.W.88TH STREET, REDMOND, WA 98052.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>GORDON E. KING</td>
											<td>15716 FIRST AVENUE N.W., SHORELINE, WA 98177.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PAUL A. ALGATE</td>
											<td>580 KALMIA PLACE N.W.ISSAQUAH, WA 98027.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>STEVEN P. FLING</td>
											<td>11414 PINYON AVENUE N.E., BAINBRIDGE ISLAND, WA 98110.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>GARY RICHARD FANGER</td>
											<td>15906 29TH DRIVE S.E. MILL CREEK, WA 98102.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>STEVEN G. REED</td>
											<td>2843 122ND PLACE N.E. BELLEVUE, WA 98005.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>THOMAS S. VEDVICK</td>
											<td>124 S. 300TH PLACE, FEDERAL WAY, WA 98003.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>DARRICK CARTER</td>
											<td>321 SUMMIT AVENUE E., SEATTLE, WA 98102.</td>
										</tr>
										<tr>
											<td>9</td>
											<td>PAUL HILL</td>
											<td>4917 WEST VIEW DRIVE, EVERETT, WA 98201.</td>
										</tr>
										<tr>
											<td>10</td>
											<td>EARL ALBONE</td>
											<td>509 LAUNFALL ROAD, PLYMOUTH MEETING, PA 19462.</td>
										</tr>
										<tr>
											<td>11</td>
											<td>MARC W. RETTER</td>
											<td>33402 N.E. 43RD PLACE, CARNATION, WA 98104.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/47</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2001/22635</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2001-07-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>09/667,857</td>
									<td>2000-09-20</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>09/884,441</td>
									<td>2001-06-18</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>09/636,801</td>
									<td>2000-08-10</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>09/827/271</td>
									<td>2001-04-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>09/617,747</td>
									<td>2000-07-17</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251650-an-isolated-monoclonal-antibody by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:48:15 GMT -->
</html>
