<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251825-single-tube-multiplex-assay-for-detection-of-adulterants-in-basmati-rice-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:42:24 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251825:SINGLE TUBE MULTIPLEX ASSAY FOR DETECTION OF ADULTERANTS IN BASMATI RICE SAMPLES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">SINGLE TUBE MULTIPLEX ASSAY FOR DETECTION OF ADULTERANTS IN BASMATI RICE SAMPLES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to a single tube multiplex assay for distinguishing basmati from non-basmati rice varieties and thereby the adulteration, said assay comprising steps of: a) running multiplex PCR with sample using one or more loci of table 2, and b) distinguishing the basmati from non-basmati rice varieties and thereby the adulteration on the basis of varietal specific multiplex allele profile.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention is a Patent of Addition of pending Indian Patent Application No.260/MAS/2002 dated 08 April 2002<br>
Field of the Present Invention<br>
The present invention relates to the assays for detection and quantification of adulterants in basmati rice varieties.<br>
Background and prior arts of the Present Invention<br>
Traditional basmati varieties command a considerable price advantage in the<br>
international market over others. For instance, in European market, Indian traditional<br>
Basmati like Dehradun Basmati commands $850 per tonne where as evolved basmati<br>
cultivars like Pusa Basmati and Super Basmati get $480 and $500 per tonne<br>
respectively, and non-basmati long-grain rice fetch a meagre $160 per tonne.<br>
Additionally, some overseas markets encourage varieties that are more authentic by<br>
granting duty exemption. For example, in European market, a tariff of $78 per tonne is<br>
imposed on husked rice; whereas for nine Basmati varieties, the import duty is<br>
completely exempted (European Commission regulation 1549/2004).<br>
Considering the price differences in the light of the total volume of international<br>
basmati rice trade (~ 1.5 million MT), it is obvious that unscrupulous practices such as<br>
adulteration of traditional basmati offer cost advantage to the traders. Since it is not<br>
quite easy to differentiate between traditional basmati and other long grain rice<br>
varieties, and a label of traditional basmati brings along duty advantage, fraudulent<br>
traders make a substantial profit by adulterating traditional basmati with either evolved<br>
basmati or non-basmati varieties and exploit the gullible consumer. Such practices have<br>
been shown to be existing and rampant by a food survey conducted by the Food<br>
Standards	Agency	of	the	United	Kingdom<br>
(http://www.food.gov,uk/science/surveillance/fsis2004branch/fsis4704basmati:). The adulteration of traditional basmati grains affects the exporting countries too in terms of the tarnished image and diminished interest in the brands.<br>
Hence, to protect the interests of consumers and trade, identification of genuine basmati rice samples and devaluation of adulterated samples becomes vital. Differentiation of traditional basmati varieties from other long grain varieties based on aroma, chemical composition and grain elongation are impracticable for large-scale<br><br>
applications. Microsatellite profiles can be used for cultivar identification and detection<br>
of adulteration. We have already designated microsatellite profiles of traditional<br>
basmati, evolved basmati and non-basmati rice varieties (Nagaraju et al 2002). In fact,<br>
importers like European Union, have now stipulated that all Basmati imports carry a<br>
certificate of purity based on a DNA test.<br>
Objects of the Present Invention<br>
The main object of the present invention relates to develop a single tube multiplex<br>
assay for distinguishing basmati from non-basmati rice varieties and thereby the<br>
adulteration.<br>
Yet another object of the present invention is to develop a method of quantifying<br>
adulteration in basmati rice varieties.<br>
One more object of the present invention relates to a kit to perform a single tube<br>
multiplex assay for distinguishing basmati from non-basmati rice varieties and thereby<br>
the adulteration, said kit comprises:<br>
a)	primers of table 3;<br>
b)	appropriate reagents for PCR; and<br>
c)	optionally, package insert to conduct the assay. Statement of the Present Invention<br>
The present invention relates to a single tube multiplex assay for distinguishing basmati from non-basmati rice varieties and thereby the adulteration, said assay comprising steps of:<br>
a)	running multiplex PCR with sample using one or more loci of table 2, and<br>
b)	distinguishing the basmati from non-basmati rice varieties and thereby the adulteration on the basis of varietal specific multiplex allele profile;<br>
also, a method of quantifying adulteration in basmati rice varieties, said method comprising steps of:<br>
a)	constructing a standard curve on the basis of ratio of quantity of amplified products of the alleles of adulterant and the basmati rice against the progressive proportion of adulteration, and<br>
b)	quantifying the adulteration in basmati rice variety on the basis of peak area of the alleles corresponding to basmati and that of the adulterant; and<br><br>
also a kit to perform a single tube multiplex assay for distinguishing basmati from non-basmati rice varieties and thereby the adulteration, said kit comprises:<br>
a)	primers of table 3;<br>
b)	appropriate reagents for PCR; and<br>
c)	optionally, package insert to conduct the assay. Detailed description of the present Invention<br>
Accordingly, the present invention relates to a single tube multiplex assay for distinguishing basmati from ndn-basmati rice varieties and thereby the adulteration, said assay comprising steps of running multiplex PCR with sample using one or more loci of Table 3, and distinguishing the basmati from non-basmati rice varieties and thereby the adulteration on the basis of varietal specific multiplex allele profile; and also, a method of quantifying adulteration in basmati rice varieties, said method comprising steps of constructing a standard curve on the basis of ratio of quantity of amplified products of the alleles of adulterant and the basmati rice against the progressive proportion of adulteration, and quantifying the adulteration in basmati rice variety on the basis of peak area of the alleles corresponding to basmati and that of the adulterant.<br>
In an embodiment of the present invention, a kit to perform a single tube multiplex assay for distinguishing basmati from non-basmati rice varieties and thereby the adulteration, said kit comprises:<br>
a)	primers of table 3;<br>
b)	appropriate reagents for PCR; and<br>
c)	optionally, package insert to conduct the assay.<br>
A set of ten SSR loci has been identified and the competence its allele profiles to genotype various basmati varieties has been demonstrated. Further, a multiplex system to make use of allele size information for the identification of adulterants in commercial samples of basmati rice has been designed. It was also demonstrated that the multiplex system could be used to quantify the adulterant. Here, a high throughput "single tube assay" method based on multiplexing all or a combination of the ten microsatellite markers is described as a tool to certify genuineness of Basmati rice samples as shown in figure 7.<br><br>
1.	Identification of the adulterant<br>
Primary step in the identification of an adulterant is to make unequivocal identification possible by generating varietyspecific microsatellite profiles of the basmati varieties designated for trade and possible adulterants (Table 1). 350 primers were screened on the varieties (sequence source: www.gramene.org). Sixteen primers were selected based on amplification of a single and clear band and discrimination power (Table 2). A panel of ten informative microsatellite loci was developed that differentiate various traditional basmati, evolved basmati varieties and others as well as amenable for multiplexing (Table 3). Upon PCR, a genuine sample of a traditional basmati variety yields a single allele of the size listed in the panel.<br>
However, any admixture of traditional basmati with either evolved basmati or non-basmati would be detected at least at one of the microsatellite loci because of different allele sizes. Subsequently, we arranged these primers based on allele sizes in such a way that using 3 fluorescent ligands in the PCR primers we could run a single genotyping assay. The above two steps resulted in a methodology where, (a). Pure samples of all varieties could be unequivocally identified and (b). Allele pattern could also identify the varietal mixtures.<br>
2.	Construction of standard curve and Quantitation of adulterant<br>
It is possible that some basmati rice samples may contain adventitious mixture as a result of inadvertent mixing in the field/storage. If we can measure the actual amount of the adulterant, such samples having admixture within limits allowed by the importing countries (for instance, 7% recommended by The Grain and Feed Trade Association, GAFTA Code of Practice for Rice) could be certified as practically genuine. Therefore, we went a step ahead in our effort and designed experiments to actually quantify the adulterant in basmati rice samples.<br>
Given the differentiating alleles between the traditional basmati (major component) and evolved basmati or non-basmati (adulterant), the quantitation procedure was based on the premise that if we can quantify the amplified allelic products of a "common locus", the ratio between quantities of the amplicons can reveal the ratio of the quantities of competing DNA templates in a PCR mixture. The procedure involved preparation of a series of standards of traditional basmati rice samples with a progressive proportion of adulteration. The approach was to generate a "standard curve" by plotting the ratio of<br><br>
the quantity of amplified products of the alleles of adulterant and the traditional<br>
Basmati against the progressive proportion of adulteration. Quantity of the amplified<br>
allele  was  calculated  based  on  the  peak  area  of the  allele  obtained  on  the<br>
electropherogram.<br>
Brief description of the accompanying drawings<br>
Figure 1 shows top panel with allelic profile of Basmati 370 obtained by single assay<br>
multiplex reaction. Three colours represent three groups of primers labeled with<br>
specific fluorescent ligands (blue is FAM, Green is JOE and black is TAMRA). Locus<br>
name and allelic size in base pairs are given below the peaks; bottom panel shows<br>
allelic profiles of pure basmati 370, pure adulterant sharbati and an adulterant sample,<br>
by using only two primers from the multiplex panel.<br>
Figure 2 shows sequence alignment of alleles of RM55 locus from different basmati<br>
rice varieties showing variation in length.<br>
Figure 3 shows standard curve generated for a combination of basmati 370 adulterated<br>
with sharbati using allele differences at RM 348 locus.<br>
Figure 4 shows mixing experiments in different combinations and the peaks obtained<br>
thereby at particular combinations.<br>
Figure 5 shows photographs of Agarose Gel run to establish amplification for various<br>
PCRs.<br>
Figure 6 shows sequence alignment data of various loci.<br>
Figure 7 shows multiplex mixing combinations.<br>
Brief description of the tables of the present Invention<br>
Table 1 shows list of varieties used for standardization of multiplex<br>
Table 2 shows SSR loci (including those added in the CIP) that are selected to<br>
distinguish basmati from non-basmati subsequent to large-scale screening.<br>
Table 3 shows the panel often informative SSR loci selected for multiplex assay<br>
Table 4 shows allele sizes (in base pairs) of various basmati rice varieties obtained by<br>
multiplex single assay method<br>
Table 5 shows Genotype codes of various basmati rice varieties based on single assay<br>
multiplex method. The order of codes from left to right correspond to loci 1 to 8 given<br>
in Table 5.<br>
Table 6 shows Allele sizes in base pairs for corresponding codes of Table 4.<br><br>
Table 7 Shows how these 10 primers were arranged in a particular manner to facilitate single genotyping assay. It is clear from the table that loci were grouped so as to avoid overlapping allele sizes in the same fluorescence label (read as 'same coloured peaks in the electrophoresis') as shown in figure 7.<br>
The loci could be employed to distinguish basmati and non-basmati in a 'single tube assay' is the result of the present research. The number of markers would vary from case to case and thus, the requirement can vary from 1 to all the 10 markers. The assay can differentiate any two known varieties using only one locus. However, a combination of the markers is employed in a multiplex single tube reaction to identify the main variety and any combination of adulterants in the genuine basmati grains. The web link for the rice microsatellite primer list is http://www.gramene.org/microsat/ssr.html. This site had only 350 loci when the study was initiated, but now contains nearly two thousand microsatellite loci. Experimental data on the basis of which 10 markers were selected is provided below.<br>
1.	Preliminary screening of the loci was done for the amplification of a clear and single amplicon. Those loci, at which a) no amplification b) non-specific amplification c) stutter problem and, d) inconsistent amplification were obtained were eliminated.<br>
2.	In the second step of screening only those loci for which primer pairs have annealing temperature of at least 55° C were selected to ensure stringent PCR conditions in the assay.<br>
3.	Ideally such loci were selected that generated more than two alleles and could be easily differentiated from stutters if any.<br>
4.	Loci generating private alleles specific to particular variety were given preference.<br>
5.	Among the most distinguishing loci, those with high reproducibility of the allele size were selected for further analysis.<br>
6.	The loci were then tested for existence of polymorphism among and between basmati genotypes especially a set of the varieties that are commercially important.<br>
Comprehensive details of the experimental data to arrive at the "Standard Curve" are provided as given below. In addition, shown are standard curve experiments for other combinations also, apart from Basmati 370 and Sharbati using locus RM348. Here,<br><br>
calculations are also provided to arrive at Peak Area and also, the percentage adulteration determined in such cases.<br>
Construction of standard curve and Quantitation of the adulterant It is possible that some basmati rice samples contain adventitious mixture because of inadvertent mixing in the field/storage. If we can measure the actual amount of the adulterant, such samples having admixture within limits allowed by the importing countries (for instance, 7% recommended by The Grain and Feed Trade Association, GAFTA Code of Practice for Rice) could be certified as practically genuine. Therefore, we designed experiments to actually quantify the adulterant in basmati rice samples. Given the differentiating alleles between the traditional basmati (major component) and evolved basmati or non-basmati (adulterant), the quantitation procedure is based on the premise that if we can quantify the amplified allelic products of a "common locus", the ratio between quantities of the amplicons can reveal the ratio of the quantities of competing DNA templates in the PCR mixture. The procedure involved preparation of a series of standards of traditional basmati rice samples with a progressive proportion of adulteration. The approach was to generate a "standard curve" by plotting the ratio of the quantities of amplified products of adulterant and the traditional Basmati alleles against the degree of adulteration. Quantity of the amplified allele was calculated based on the peak area of the allele obtained on the electropherogram.<br>
Standard curves were constructed for a combination of Basmati370:Sharbati mixtures at two discriminating loci, RM72 and RM348. Standard samples were prepared by mixing the grains of the Basmati370 with Sharbati at progressive ratio of 1%, 3%, 5%, 7%, 10%, 15%, 17%, 20%, 25%, 30%, 40% and 60% to generate data at 12 score points. Subsequent to genotyping, peak areas were determined for each score point and were plotted against the percent adulterant to develop a standard curve based on logistic model (y=a/l+be-cx) by using CurveExpert 1.38 (http://curveexpert.webhop.net). A standard curve was also generated by mixing DNA isolated from the milled grains of Sharbati, a common adulterant, in various ratio at 5%, 10%, 20%, 30%, 40%, 50% and 60% to Basmati370 DNA to generate seven score points on the curve. Systematic bias associated with the employment of standard curves was calculated. The differences were averaged over three independent runs to compute the bias (b) at each score point. Bias (B) introduced by using standard curve was computed as, <br><br>
For illustrating mixing experiments in different combinations, peaks obtained at<br>
particular combinations are given as figure 4. Further, photographs of Agarose Gel run<br>
to establish amplification for various PCRs is provided as figure 5.<br>
Bi-directional sequencing of PCR products was carried out thrice on ABI 3100<br>
sequencer using ABI PRISM BigDye Primer Cycle Sequencing Kit according to the<br>
manufacturer's instructions. Sequence alignment data of various loci, as provided for<br>
locus RM 55 in figure 2 is provided in as given as figure 6.<br>
The invention is further elaborated with the help of following examples. However, the<br>
examples should not be construed to limit the scope of the invention.<br>
Example 1. Multiplex PCR<br>
PCR amplification was carried with the following reaction mixture composition. lOng<br>
of DNA template, 80 μM dNTPs, 2 mM MgC12, primer-mix providing 0.1 μM of each<br>
primer pair to the reaction, 0.5 unit Ampli Taq Gold DNA polymerase (Applied<br>
Biosystems), were mixed in a reaction volume of I0ΜL 5' ends of forward primers were<br>
labeled with any one of the following fluorescent ligands: TAMRA, JOE or FAM<br>
(Sigma). After an initial denaturation of 15 min at 95°C, the PCR mix was cycled 30<br>
times at 94°, 55° and 72°C for 30, 90 and 60 seconds respectively. This was followed by<br>
a final extension step at 60°C at 30 min. Amplification was carried out on a PE9700<br>
thermal cycler.<br>
Example 2. Genotyping<br>
Amplification was confirmed on 1.5% agarose gel before running genotyping assays on<br>
the capillary-based ABI 3100 genetic analyser according to manufacturer's instructions.<br>
0.2μl PCR product was mixed with ROX-500 size standard and Hi-dye before loading.<br>
Subsequent to electrophoresis, lanes were extracted and analysed using GeneScan<br>
version 3.1 and allele sizes of the true peaks were determined by Genotyper version<br>
2.1. Bi-directional sequencing of PCR products was carried out thrice on ABI 3100<br>
sequencer to obtain accurate sequences of the repeat regions.<br>
Example 3. Quantification of adulterant<br>
Standard curves were constructed for a combination of Basmati370:Sharbati mixtures<br>
at two discriminating loci, RM72 and RM348. Standard samples were prepared by<br>
mixing the grains of the Basmati370 with Sharbati at progressive ratio of 1, 3, 5, 7, 10,<br>
15, 17, 20, 25, 30, 40 and 60% to generate data at 12 score points. Triplicate lg<br>
samples at each score point were used for DNA isolation. Subsequent to genotyping,<br><br>
peak areas were determined for each score point and were plotted against the percent adulterant to develop a standard curve based on logistic model (y=a/l+be-cx). A standard curve was also generated by mixing DNA isolated from the milled grains of Sharbati in various ratio at 5%, 10%, 20%, 30%, 40%, 50% and 60% to Basmati370 DNA to generate seven score points on the curve. Systematic bias associated with the employment of standard curves was calculated. The differences were averaged over three independent runs to compute the bias (b) at each score point. Bias (B) introduced by using standard curve was computed as, RESULTS <br>
1.	Variety specific profiles and identification<br>
Excellent quality peaks were obtained in the single assay multiplex reactions to obtain allele sizes for all the rice varieties tested (Table 3). Figure 1 top panel shows the multiplex profile (8 loci) for Basmati370, Figure 1 bottom panel shows the allele profile (2 loci) of pure and adulterated Basmati370 samples. All varieties were assigned specific profiles (Table 4).<br>
The multiplex single assay can identify all the listed basmati varieties. RM171 alone can clearly separate traditional basmati from others. Confirmation of allele sizes<br>
Microsatellite alleles may produce stutters even under best of the conditions. Determination of the allele sizes can therefore be prone to errors, which is not acceptable for sensitive assays such as determination of adulterants. We confirmed the allele sizes in twelve varieties by Bi-directional sequencing of the alleles and actual counting the number of repeat units in each allele at all the loci. Sequencing also helps discover reasons for the size differences between alleles. Sequencing of PCR products was carried out thrice on ABI 3100 sequencer. In RM55, the size differences between alleles were due to disparate repeat numbers as well as indel events in the flanking sequences (Figure 2). In all other loci, differences in the allele sizes were entirely due to differences in the number of repeat units. We therefore have confirmed sizes of all the alleles at all loci.<br>
2.	Quantification of the adulterant<br>
Sample standard curve obtained at RM348 is shown in Figure 3. Systematic bias associated with the employment of standard curves was calculated to be ±4.95% for RM72 based curve and ±5.2% for RM348, based curve in the region of 1-15%<br><br>
adulteration. The standard curves were validated by quantifying the adulteration in blind samples. Three blind samples with 4%, 8% and 12% adulteration were genotyped and the peak-area ratios were plotted on the standard curves. The per cent adulteration was estimated with an error of ±2.6% and ±2.3% respectively for RM348 and RM72 based curves. Therefore, our protocol quantifies the adulterant with an accuracy of at least ±3% adulteration.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgIEFNRU5ERUQgIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gICAyOS0wNy0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">662-CHE-2006    AMENDED  PAGES OF SPECIFICATION   29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2ICAgIGFtZW5kZWQgY2xhaW1zICAgMjktMDctMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006    amended claims   29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgIEZPUk0tMTMgICAyOS0wNy0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">662-CHE-2006    FORM-13   29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMjktMDctMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">662-CHE-2006   CORRESPONDENCE OTHERS   29-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAxNC0wNi0yMDExLnBkZg==" target="_blank" style="word-wrap:break-word;">662-CHE-2006   CORRESPONDENCE OTHERS  14-06-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAyMC0wMS0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">662-CHE-2006   CORRESPONDENCE OTHERS  20-01-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgICAgIDA4LTA3LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">662-CHE-2006   POWER OF ATTORNEY     08-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICBBTUVOREVEIENMQUlNUyAgMzAtMTEtMjAxMC5wZGY=" target="_blank" style="word-wrap:break-word;">662-CHE-2006  AMENDED CLAIMS  30-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICBDT1JSRVNQT05ERU5DRSAgT1RIRVJTICAgIDA4LTA3LTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">662-CHE-2006  CORRESPONDENCE  OTHERS    08-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDSUVWRUQgIDMwLTExLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">662-CHE-2006  EXAMINATION REPORT REPLY RECIEVED  30-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2ICBmb3JtLTMgIDMwLTExLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">662-che-2006  form-3  30-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLUNIRS0yMDA2ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgIDMwLTExLTIwMTAucGRm" target="_blank" style="word-wrap:break-word;">662-CHE-2006  OTHER PATENT DOCUMENT  30-11-2010.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">662-che-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">662-che-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">662-che-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjYyLWNoZS0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">662-che-2006-form 5.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251824-minimizing-voltatile-organic-sulphur-byproducts-in-dimethyl-sulfate-quaternization-of-amines-made-with-hypophorous-acid.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251826-indol-3-carbonyl-spiro-piperidine-derivatives-as-via-receptor-antagonists.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251825</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>662/CHE/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>15/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Apr-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Apr-2002</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CENTER FOR DNA FINGERPRINTING AND DIAGNOSTICS (CDFD)</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>DEPARTMENT OF BIOTECHNOLOGY MINISTRY OF SCIENCE AND TECHNOLOGY GOVERNMENT OF INDIA, 4-87/1, ECIL ROAD, NACHARAM, HYDERABAD 500 076</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>JAVARE GOWDA NAGARAJU</td>
											<td>CENTER FOR DNA FINGERPRINTING AND DIAGNOSTICS (CDFD)4-87/1, ECIL ROAD, NACHARAM, HYDERABAD 500 076</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SUNIL ARCHAK</td>
											<td>CENTER FOR DMA DOMGERPRINTING AND DIAGNOSTICS (CDFD)4-87/1, ECIL ROAD, NACHARAM, HYDERABAD 500 076</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251825-single-tube-multiplex-assay-for-detection-of-adulterants-in-basmati-rice-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:42:25 GMT -->
</html>
