<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/251967-method-for-the-humanization-of-antibodies-and-humanized-antibodies-thereby-obtained by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:37:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 251967:&quot;METHOD FOR THE HUMANIZATION OF ANTIBODIES AND HUMANIZED ANTIBODIES THEREBY OBTAINED&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;METHOD FOR THE HUMANIZATION OF ANTIBODIES AND HUMANIZED ANTIBODIES THEREBY OBTAINED&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Method for the humanization of the VH and VL variable regions of an animal antibody of known sequence, comprising the steps of (a) if not available, obtaining the crystallographic structure of the VH and VL regions of the animal antibody; (b) pre-selecting a series of 0 to n possible frameworks acceptors of human origin or humanized antibodies, whose structure was determined experimentally with a resolution no greater than 2.5 A, based on the highest level of homology and identity with the primary sequence of the framework of the animal antibody; (c) conducting a structural comparison between the VH and VL variable regions of the animal antibody and the regions VH and VL obtained in b), respectively and calculating for each comparison the root mean square deviation (RMS), to identify the region VH and the region VL of human origin with the smaller RMS; (d) inserting in appropriate position the sequences of the CDR regions of the animal antibody in the human sequences identified in c); (e) if necessary, retromutate one or more amino acid residues of the human VH and VL regions identified in c).</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>BACKGROUND<br>
The present invention relates to a method for the humanization of antibodies, by means of determining and comparing three-dimensional structures, humanized antibodies thereby obtained and their uses in therapy and diagnostics in vivo. The therapeutic and diagnostic application of monoclonal antibodies of animal origins in humans has fundamental contraindications especially for therapeutic regimes which necessitate for repeated administrations. In particular, murine monoclonal antibodies have a relatively short half-life and, when used in humans, lack some fundamental functional characteristics of immunoglobulins, such as complement-dependent cytotoxicity and cell-mediated cytotoxicity.<br>
Moreover, monoclonal antibodies of non-human origin contain immunogenic amino acid sequences if injected into patients. Numerous studies have shown that after the injection of an exogenous antibody, subjects develop a rather strong immune reaction against the antibody itself (known as HAMA - human anti-mouse antibodies -reaction), completely eliminating its therapeutic usefulness, with the formation of immunocomplexes, alteration of pharmacokinetics, production of allergic reactions, etc. Moreover, considering the growing number of different monoclonal antibodies developed in mice or in other mammals (and thus antigenic for humans) for the therapy of different pathologies, treatments, also for non correlated therapies can be ineffective or even dangerous due to cross-reactivity. Although the production of so-called chimeric antibodies (variable murine regions joined to constant regions of human origin) has yielded some positive result, a significant immunogenicity problem still remains.<br>
Humanized antibodies have at least three potential advantages with respect to antibodies of animal origin in the field of therapeutic use in humans. In the first place, the effector region, being human, can better interact with the other parts of the human immune system, destroying target cells more efficiently by means of complement-dependent cytotoxicity, or cell-mediated, antibody dependent cytotoxicity. Moreover, the human immune system does not recognize the framework or the constant region (C) of the <br>
humanized antibody as exogenous, and hence the antibody response against the humanized antibody is minimized, both relative to that against a murine antibody (totally extraneous) and relative to the response induced by a chimeric antibody (partially extraneous).<br>
It has been reported that murine antibodies injected into humans have a much shorter half-life time than normal antibodies (Shaw et al., 1987). Humanized antibodies have a very similar half life to that of natural human antibodies, allowing less frequent administration and lower doses.<br>
The basic principle of humanization is configured in transferring the specificity of antigen recognition, i.e. the CDR domains, in the context of a human immunoglobulin ("CDR  grafting",  Winter and Milstein,  1991).   Several  examples  of humanized antibodies, produced in the attempt to solve the problem of immunogenicity, have been reported (Maeda et al., 1991; Singer et al.,1993; Tempest et al, 1994; Kettleborough et al, 1991; Hsiao et al, 1994; Baca et al., 1997; Leger et al.,1997; Ellis et al., 1995; Sato et a/.,1994; Jones et a/.,1986; Benhar et a/.,1994; Sha and Xiang,1994; Shearman et o/.,1991; Rosok et a/.,1996; Gussow &amp; Seeniann,1991; Couto et al.,1994; Kashmiri et al, 1995; Baker et al.,1994; Riechmann et a/.,1988; Gorman et al., 1991; Verhoeyen et a/.,1988; Foote &amp; Winter,1992; Lewis &amp; Crowe,1991; Co et al, 1991; Co et al, 1991; Verhoeyen et a/.,1991; Eigenbrot et al, 1994; Hamilton et al.,1997; Tempest et al,l995; Verhoeyen et al.,1993; Cook et a/.,,1996; Poul et a/.,,1995; Co et al.,1992; Graziano et al.,1995; Presta et al.,1993; HaMmi et al.,1993; Roguska et al, 1996; Adair et a/.,1994; Sato et al, 1993; Tempest et a/.,1991; Sato et al.,1996; Kolbinger et al, 1993; Zhu and Carter,1995; Sims et al.,1993; Routledge et al., 1991; Roguska et al.,1994; Queen et al, 1989; Carter et al.,1992}.<br>
The transcription of an antibody from animal (generally murine) to humanized entails the compromise between opposite requirements, whose solution varies case by case. To minimize immunogenicity, immunoglobuun shall maintain as much of the accepting human sequence as possible, hi any case, to preserve the original binding properties, the immunoglobulin framework should contain a sufficient number of mutations in the accepting human sequence to guarantee that the conformation of the CDR regions is as similar as possible to that in the donor murine immunoglobulin. As a consequence of these opposite considerations, for many humanized antibodies a significant loss in binding affinity with respect to the corresponding murine antibodies has been reported<br><br>
(Jones et al, 1986; Shearman et al., 1991; Kettleborough, 1991; Gorman et al., 1991; Riechmann et al, 1988).<br>
Currently, the most common method for the production of humanized immunoglobulin is based on the use of appropriate genomic, synthetic sequences, as well as cDNA (Reichmann et al, 1988).<br>
The patent application EP  592106 discloses  a method for the humanization  of antibodies from rodents. The method is based on the identification of the amino acid residues exposed at the surface of the three-dimensional structure of the antibody to be humanized, on the identification of the amino acid residues in the same positions on the corresponding human antibody, and on the replacement of the residues identified in the sequence of the rodent antibody with those identified in the human antibody. DESCRIPTION OF THE INVENTION<br>
The authors of the present invention set up a method to obtain optimized humanized forms of immunoglobulins which are substantially not immunogenic in humans, with an approach that is consistently based on structural data, obtained experimentally, deriving from crystallographic studies. The method of the invention allows to obtain antibodies in a form adapted to therapeutic formulation and to other medical and diagnostic applications.<br>
The invention relates to a method fully based on structural data to conduct the first design stages (generally more subject to error) of humanization. Humanized immunoglobulins have two pairs of heterodimers between light and heavy chain, with at least one of the chains bearing one or more CDRs of animal origin, functionally bound to segments of regions of the framework of human origin. For example, CDRs of animal origin, together with amino acid residues, naturally associated, also of animal origins, are introduced in framework regions of human origin, to produce humanized immunoglobulins able to bind the respective antigens, with affinities comparable to the affinities of the original immunoglobulins of animal origin.<br>
The method of the invention led to obtain humanized antibodies suitable for therapeutic and diagnostic applications. In particular, humanized immunoglobuh'ns have been obtained, derived from anti-TrkA antibodies (Patent EP 1181318) and from anti-NGF antibodies able to bind with high specificity respectively TrkA and NGF, neutralizing the interaction between ligand and receptors. Such molecules are useful for the treatment of tumors which depend on NGF/TrkA, of chronic pain and of inflammatory<br><br>
forms, and for diagnostic purposes, for in vivo imaging, e.g. on TrkA positive tumors, or on basal forebrain as a precocious marker of Alzheimer's Disease. In particular, humanized anti-TrkA antibodies find specific therapeutic and diagnostic application in the inflammatory forms of the urinary tract and of the pelvic region. In particular, humanized anti-NGF antibodies find specific therapeutic and diagnostic application in pathologies induced by HIV virus, to induce apoptosis of immune cells, such as HIV infected, NGF dependent macrophages.<br>
Therefore, an object of the present invention is to provide a method for the humanization of the VH and VL variable regions of a animal antibody of known sequence, comprising the steps of:<br>
a)	if not available, obtaining the crystallographic structure of the VH and VL regions of<br>
the animal antibody;<br>
b)	pre-selecting a series of 0 to n possible frameworks acceptors of human origin or<br>
humanized antibodies, whose structure was determined experimentally with a resolution<br>
of no less than 3 A, based on the highest level of homology and identity with the<br>
primary sequence of the framework of the animal antibody;<br>
c)	conducting a structural comparison between the VH and VL variable regions of the<br>
animal antibody and the regions VH and VL obtained in b), respectively and calculating<br>
for each comparison the RMS, to identify the region VH and the region VL of human<br>
origin with the smaller RMS;<br>
d)	inserting in appropriate position the sequences of the regions CDR of the animal<br>
antibody in the human sequences identified in c);<br>
e)	if necessary, retromutate one or more ammo acid residues of the human VH and VL<br>
regions identified in c).<br>
Preferably, the modifications of the antibody take place with recombining DNA<br>
techniques.<br>
In a preferred embodiment, the animal antibody is an anti-NGF antibody, preferably it is<br>
the alpha Dll antibody, and the humanized sequences essentially have the following<br>
VH sequences: Hum alpha Dll VH,<br>
EVQLVESGGGLVQPGGSLRLSCAASGFSLTNNNVNWVRQAPGKGLEWVGGV<br>
WAGGATDYNSALKSRFTISRDNSKNTAYLQMNSLRAEDTAVYYCARDGGYSS<br>
STLYAMDAWGQGTLVTVSS, (SEQ ID No. 17)<br>
and VL: Hum alpha Dl 1 Vk,<br><br>
DIQMTQSPSSLSASVGDRVTITCRASEDIYNALAWYQQKPGKAPKLLIYNTDTL<br>
HTGVPSRFSGSGSGTDYTLTISSLQPEDFATYFCQHYFHYPRTFGQGTKVEIK<br>
(SEQ ID No, 18).<br>
In an alternative embodiment, the animal antibody is an anti-TrkA antibody, preferably<br>
it is the alpha MNAC13 antibody, and the humanized sequences essentially have the<br>
following sequences: VH: HumMNAC13VH,<br>
EVQLLESGGGLVQPGGSLRLSCAASGFTFSTYTMSWARQAPGKGLEWVAYISK<br>
GGGSTYYPDTVKGRFTISRDNSKNTLYLQMNSLRAEDSAVYYCARGAMFGND<br>
FFFPMDRWGQGTLVTVSSA, (SEQ ID No. 37)<br>
and VL: Hum MNACBVk,<br>
DIVLTQSPSSLSASVGDRVTITCSASSSVSYMHWYQQKPGQAPKLLIYTTSNLAS<br>
GVPSRFSGSGSGTDYTLTISSLQPEDVATYYCHQWSSYPWTFGGGTKVEIK<br>
(SEQ ED No. 38).<br>
The humanized immunoglobulins of the present invention (or derived fragments which maintain binding activities and other compounds which can be derived) can be produced by means of known recombining DNA techniques. As a function of the subsequent use of the humanized immunoglobulins, transgenic animals or transfected cells can be used for their expression, preferably immortalized eukaryotic cells (such as myeloma or hybridoma cells), but also prokaryotic hosts, insect or vegetable cells. The coding polynucleotides for the resulting sequences of the humanized immunoglobulins can also be obtained by synthesis.<br>
The humanized immunoglobulins of the present invention can be used alone or in<br>
combination with other therapeutic agents. In case of use as anti-tumor agents, a<br>
chemotherapeutic  agent  will be preferred,  which may vary  depending  on  the<br>
pharmacological application (such as anthracyclin, paclitaxel, cisplatin, gemcytabin,<br>
non steroidal and corticosteroid anti-inflammatory drugs, or immunosuppressants), as<br>
well as with all drugs currently applied in the therapy of each specific pathology.<br>
Humanized immunoglobulins or their complexes can be prepared in the form of<br>
pharmacologically  acceptable  dosages,  which  vary  depending   on  the   type   of<br>
administration.<br>
Definitions<br>
The term "substantially identical" within the context of two polynucleotides  or polypeptides (respectively sequences of coding DNA for humanized immunoglobulins<br><br>
or amino acid sequences of humanized immunoglobulins, or portions thereof) refers to two or more sequences which have a rrrinirnurn of 80% (preferably 90-95% or more) of identity in the nucleotide or amino acid residues, when compared and aligned with maximum correspondence. Generally, the "substantial identity" is verified in regions that are at least 50 residues long, more preferably on a region of at least 100 residues or, in optimal conditions, on over 150 residues or on the complete sequences. As described below, any two sequences of antibodies can be aligned in only one way, using Kabat's numbering scheme. Consequently, for antibodies the percentage of identity has a unique and well defined meaning. The amino acids of the variable regions of the heavy and light chains of mature immunoglobulins are designated Hx and Lx, with x being the number that designates the position of the amino acid according to Kabat's numbering scheme, Sequences of Proteins of Immunological Interest (National Institutes of Health, Bethesda MD, 1987, 1991). Kabat has determined a list of amino acid sequences of antibodies for each subgroup as well as a list of the most frequent amino acids in each position in each subgroup to generate a consensus sequence. Kabat uses a method to assign a number to each amino acid of each sequence in the list and this method for assigning the number of each residue has become a standard method in the field. Kabat's scheme can be extended to other antibodies not present in his study, aligning the antibody in question with one of the consensus sequences identified by Kabat, basing on the preserved amino acids. Use of Kabat's numbering scheme allows easily to identify the amino acids in equivalent positions in different antibodies. For example, an amino acid in L10 position in an antibody of human origin occupies the equivalent position of an amino acid in LI 0 position in an antibody of murine origin. It is well known that the basic structural unit of an antibody comprises a tetramer. Each tetramer is constituted by two identical pairs of polypeptide chains, each of which is composed by a light chain (25 KDa) and by a heavy chain (50-75 KDa). The amino-terminal region of each chain includes a variable region of about 100-110 or more amino acids, which is involved in antigen recognition. The carboxy-terminal region of each chain comprises the constant region that mediates the effector function. The variable regions of each pair of light and heavy chains form the binding site of the antibody. Therefore, an intact antibody has two binding sites.<br>
Light chains are classified as K; or A. Heavy chains are classified as 7, p,, a, e and they define the isotype of the antibody as respectively IgG, IgM, IgA, IgD e IgE. Inside both<br>
the light and the heavy chain, the variable and constant regions are joined by a "J" region of about 12 amino acids or more, whilst only the heavy chains include a "D" region of about 10 amino acids (Paul, 1993).<br>
The variable regions of each pair of light and heavy chains form the binding site of the antibody. They are characterized by the same general structure constituted by relatively preserved regions called frameworks (FR) joined by three hyper-variable regions called complementarity determining regions (CDR) (Kabat et al, 1987; Chothia and Lesk, 1987). The CDRs of the two chains of each pair are aligned by the framework regions, acquiring the function of binding a specific epitope. Starting from the amino-terminal region towards the carboxy-terminal region, the variable domains both of the light chain and of the heavy chain comprise and alternation of FR and CDR regions: FR, CDR, FR, CDR, FR, CDR, FR; consequently, both the heavy chain and the light chain are characterized by three CDRs, respectively CDRH1, CDRH2, CDRH3 and CDRL1, CDRL2, CDRL3. Amino acid assignment to each region was conducted according to the definitions by Kabat (1987 and 1991) and/or Chothia &amp; Lesk (1987), Chothia et al. (1989).<br>
Preferably, the analogs of the exemplified humanized immunoglobulins differ from the original immunoglobulins due to conservative amino acid substitutions. In order to classify the amino acid substitutions as conservative or non conservative, amino acids can be grouped as follows:<br>
Group I (hydrophobic lateral chains): M, A, V, L, I; Group II (neutral hydrophilic lateral chains): C, S, T, N, Q;<br>
Group III (acid lateral chains): D, E;<br>
Group IV (basic lateral chains): K, R;<br>
Group V (residues that influence the orientation of the main chain): G, P;<br>
Group VI (aromatic lateral chains): F, Y, W.<br>
Conservative amino acid substitutions regard substitutions between amino acid of the<br>
same class, whilst non conservative amino acid substitutions entail an exchange<br>
between members of different classes.<br>
The term "epitope" includes every protein determinant able to bind an immunoglobulin<br>
in specific fashion. Generally, epitopes are formed by sets of chemically active surfaces<br>
of macromolecules, such as lateral chains of amino acid or sugars and they generally<br>
have specific chemical-physical and conformational characteristics.<br>
The term "imrnunoglobulins" refers to proteins which consist of one or more polypeptides coded by genes of the imrnunoglobulins. Immunoglobuhns can exist in a variety of forms, in addition to the tetramer antibody form: for example, they include fragments Fv, Fab e F(ab') as well as bifunctional hybrid antibodies (Lanzavecchia et al, 1987) and single chain Fv fragments (Hood et al, 1984; Harlow and Lane, 1988; Hunkapiller and Hood, 1986).<br>
Chimeric antibodies are antibodies whose genes for the light and heavy chains have been engineering starting from gene regions of immunoglobulins belonging to different species. For example, variable segments (V) of the genes of a monoclonal mouse antibody can be joined to constant segments (C) of an antibody of human origin. A therapeutic chimeric antibody, therefore, is a hybrid protein which consists of the domain V which recognizes the antigen deriving from a mouse antibody and in the effector domain C deriving from a human antibody (although other combinations of mammal species can be used).<br>
The term "framework" refers to those portions of the variable regions of the light and heavy chain of the immunoglobulins that are relatively preserved (not belonging to the CDRs) between different immunoglobulins within a species, according to Rabat's definition. Hence, a human framework is a framework region that is substantially identical (at least 85% or more) to the framework that is naturally found in human antibodies.<br>
The term "humanized immunoglobulin" refers to an immunoglobulin which comprises a human framework and at least one CDR deriving from a non human antibody and in which each constant region present is substantially identical to a region of human immunoglobulin (at least 85%, preferably at least 90-95% identical). Hence, all the parts of a humanized immunoglobulin except the CDR are substantially identical to the corresponding regions of one or more sequences of natural human immunoglobulins. For example, the chimeric antibodies, constituted by variable mouse regions and constant regions of human origin, are not included among the humanized immunoglobulins.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The method is based on the high resolution structural comparison for the humanization of antibodies of in vivo therapeutic and diagnostic interest. Moreover, humanized immunoglobulins are provided, able to be reactive specifically against the respective<br><br>
antigens (i.e. NGF neurotrophin and its TrkA receptor). Humanized irnmunoglobuHns have a framework of human origin and they have one or more complementarity determining regions (CDRs) deriving from each original rmmunoglobulin (i.e. oDll, a rat immunoglobulin, reactive specifically against NGF and MNAC13, a murine imrnunoglobulin, which specifically recognizes TrkA). Therefore, the immunoglobulins of the present invention, which may be easily produced on a large scale, find therapeutic application not only in the therapy of NGF/TrkA dependent tumor forms, but also in the treatment of chronic pain and inflammatory forms. Moreover, the specific humanized immunoglobulin for the receptor has an additional diagnostic application for in vivo imaging both on TrkA positive tumors and on cells of the basal forebrain (as a precocious marker of Alzheimer's disease).<br>
The present invention uses the recombinant segments of DNA coding the CDR regions of the light and/or heavy chain, able to bind an epitope of interest both on NGF and on TrkA, as in the case of the monoclonal antibodies oDll and MNAC13 (respectively rat and mouse). The coding DNA segments for these regions are joined to the DNA segments coding appropriate framework regions of human origin. The DNA sequences that code for the polypeptide chains comprising the CDRs of the light and heavy chain of the monoclonal antibodies MNAC13 and oDll are included in Figure 7A, 7B and 8A, SB respectively. Because of the degeneration of the genetic code and of the substitutions of non critical amino acids, the DNA sequences can easily be modified. Moreover, DNA segments typically include an additional control sequence for the expression, operatively bound to the coding sequences for humanized immunoglobulins and comprising regions of heterologous or naturally associated promoters. Preferably, the expression control sequences are systems with eukaryotic promoters in vectors able to transform or transfect eukaryotic host cells, but prokaryotic control sequences can be used as well. Once the vector is incorporated in the appropriate host, the host is maintained in suitable conditions to assure a high level of expression. A further purification follows of the light and heavy chains individually in the form of dimers, of intact antibodies or of other forms of immunoglobulins.<br>
The sequences of coding DNA for the human constant region can be isolated by means of well known procedures from a variety of human cells, but preferably starting from immortalized B cells. The CDRs in the immunoglobulins of the present invention are similarly derived from the monoclonal antibodies oDll and MNAC13 able to bind<br>
respectively NGF and TrkA and products respectively in rat and mouse. Host cells<br>
suitable for the expression and the secretion irnmunoglobulins can be obtained from<br>
many sources such as the American Type Culture Collection (Catalogue of Cell Lines<br>
and Hybridomas, Fifth edition (1985) Rockville, Maryland, USA). Preferably, the<br>
CDRs incorporated in humanized antibodies have sequences corresponding to those of<br>
the CDRs of oDll and MNAC13 and can include degenerated nucleotide sequences<br>
coding the corresponding amino acid sequences of the antibodies themselves.<br>
Generally, the humanization design procedure is cyclical and iterative and it comprises:<br>
The analysis of the amino acid sequence of the murine antibody;<br>
The modeling of the corresponding Fv region;<br>
The analysis and selection of the amino acid sequence of the acceptor framework of the<br>
human antibody;<br>
The identification of putative retro-mutations in the selected framework;<br>
The design and the actual construction of the humanized antibody;<br>
The verification, by means of in vitro and/or in vivo assays, of the maintained affinity<br>
and specificity of the binding.<br>
If these activities are negatively influenced by the human framework, it will be<br>
necessary to change the selection of the framework of the acceptor human antibodies, or<br>
to introduce compensating mutations.<br>
Even if the choice of the human framework is configured as the most critical phase of<br>
the cycle, no general rules have been established to date. This depends on the fact that<br>
the advantages of the various choices (in terms of immunogenicity in the patient) have<br>
not been accurately studied from the clinical viewpoint. Therefore, to operate the correct<br>
choice of the framework, only a series of approaches are available, which must be<br>
combined with the results obtained previously.<br>
In particular, it is possible to use fixed frameworks (usually NEW for the heavy chain<br>
and REI for the light chain, since their structures have been available for a long time).<br>
Another approach provides for the use of the frameworks found to be the most<br>
homologous in teims of sequence with the antibody to be humanized. There are many<br>
databases to search for homologous human antibodies: the choice generally takes into<br>
account the length of the CDRs, the identity at the level of canonical residues and of the<br>
residues at the interface level, in addition to a higher percentage of identify between the<br>
sequences of the donor and of the acceptor. For a comparison between these two methods, see Graziano et al. (1995).<br>
Moreover, according to a variant of the second approach the light chain and the heavy chain can be chosen from two different human antibodies characterized by a higher sequence homology. This approach was proposed by Riechmann et al. (1988) and by Shearman et al. (1991). In this regard, in general, light and heavy chains deriving from the same antibody have a higher probability of associating correctly, forming a functional binding site, with respect to light and heavy chains deriving from different antibodies, although the fact that the interface between the two chains is quite preserved can equally assure a correct interaction. For a comparison between these two methods, see Roguska et al. (1996 and!996)<br>
Limiting the approach to a framework deriving from a particular human antibody can entail the risk of incurring in somatic mutations which produce immunogenic epitopes even if the frameworks are of human origin. An alternative approach is to use frameworks based on human consensus sequences, where idiosyncratic somatic mutations have been eliminated. The two approaches have been compared: in one case, no difference in binding avidity was noted (Kolbinger et al., 1993), in another one instead the binding proved superior in the case of individual frameworks (Sato et al., 1994).<br>
hi any case, the consensus sequences themselves are artificial and therefore, even if they have no idiosyncratic residues, they can create non natural motives which are imrnunogenic. The alternative (Rosok et al., 1996) is to use germline human sequences collected in the V-B ASE database.<br>
The non natural juxtaposition of the murine CDR regions with the variable regions of the framework of human origin can give rise to conformational limits not represented in nature which, unless they are corrected by the substitution of particular amino acid residues, determine the loss of binding affinity. The selection of the amino acid residues to be substituted is partially determined by means of computer modeling. Hardware and software are available to produce three-dimensional images of immunoglobulin molecules, hi general, molecular models are produced starting from already resolved crystallographic structures of immunoglobulin domains or chains. The chains to be modeled are compared based on the amino acid resemblance with chains or domains of resolved three-dimensional structures and the chains or the domains, which show the<br>
highest resemblance in terms of sequence, are selected as starting points in the construction of the molecular model. However, the prediction of the antibody structure is not always accurate. In particular, the third CDR region is difficult to model and it always represents a point of uncertainty in the structural prediction of an antibody (Chothia et al., 1987). For this reason, as a rule humanized antibodies, as a first approximation, have far less binding affinity and/or specificity towards the antigen than the starting monoclonal antibody. This requires many successive cycles of point mutations in the attempt to reconstitute the properties of the starting antibody, with a trial and error procedure that cannot be completely rationalized.<br>
Considering the growing number of high resolution X-ray structures both of available human and humanized antibodies, the intent was to avoid the uncertainties and ambiguities  deriving from use of computer modeling,  obtaining high resolution structural data for the Fab fragments of both the antibodies of the invention by means of X-ray crystallography. For this purpose, both antibodies were purified from hybridoma, treated proteolytically with papaine (a protease that cuts at the level of the junction between CHI and CH2 domain of the heavy chain) which gives origin to the Fab fragments.  As  a result of the additional purification, both Fab fragments were crystallized and from two databases (low and high resolution), it was possible to solve the structures with the Molecular Substitution method and subsequently to refine them. The   approach   proposed   by  the   invention,   based   on   structural   data   obtained experimentally, provides a much more solid and rational starting point, both in the critical phase of the selection of the framework of the acceptor human antibody, and for the identification of putative retro-mutations in the framework selected within the humanization process of both neutralizing antibodies.<br>
Amongst the various reported criteria which can guide the selection of the human antibody framework, the one used was the degree of identity between the antibody of murine and human origin at the primary sequence, to extend and complete its results with an analysis based on structural alignment. A compared analysis of the corresponding structures associated to the original criterion assures a much more accurate comparison and consequently a greater probability that the resulting humanized antibody can preserve the characteristics of affinity and specificity of the original murine antibody. Consequently, the strategy employed combines the information deriving from the analysis and comparison of amino acid sequences, both in terms of<br>
degree of identity and of level of homology, with the comparison of the respective three-dimensional structures.<br>
hi particular, the information deriving from the optimal alignment of the primary structures has a dual role, hi the first place, this analysis allows to reduce the number of possible tertiary structures to be compared, limiting itself to those characterized by a high degree of homology and identity. Among these sequences characterized by an optimal alignment at the primary structure level and for which structural data are available, a further selection was conducted, concentrating only on the resolved structures with high resolution or otherwise with resolution comparable to that of the structures obtained by us (i.e. no greater than 2.5A). This approach assures a much more accurate alignment of the tertiary structures and a much more significant estimates of the structural differences, expressed in RMS (root mean square deviation: square root of the mean square deviation; Carugo and Pongor, 2001 and 2003). Low resolution data provide rather indicative, and definitely less precise information on the actual relative position of each individual atom in space.<br>
To assess the degree of superposition of each individual structure, of human origin or engineered, the RMS was calculated between atoms of alpha carbon constituting the respective amino acid skeletons, not considering atom pairs with an RMS exceeding 2A. From this analysis, an information is obtained which must therefore take into account not only the diversity between the structures (expressed by the value of RMS), but also the percentage of atoms of alpha carbon actually employed in calculating each RMS.<br>
These tertiary structure level resemblance data were associated to the comparative analysis of the primary sequences both in terms of identity and of homology. It is hence deduced that the selection of the optimal framework for humanization is configured as a three-variable problem, which can thus be represented in space, both when associating the homology level and the degree of identity to the structural alignment. This type of analysis was then conducted also reducing the regions in question in the two types of alignment to the regions of the respective frameworks. Comparing the distributions of the antibodies considered in the space of the three analyzed variables (respectively, value of RMS, percentages of atoms on which RMS was calculated and a similitude index between primary structures, i.e. percentage of overall identity, of overall homology, of identity at the framework level, of homology at<br>
the framework level) with the optimal position in the space of the three variables that each antibody would occupy if it were of human origin, it is possibly clearly to identify the human origin antibody that most approaches this ideal position at the level of primary and tertiary structure. To rationalize this result, in each of the four analyses the deviations from the hypothetical optimal position are calculated for each position of the humanized or human origin antibodies considered.<br>
On the basis of this method of selection, it is possible to choose the acceptor framework in the subsequent process of CDR grafting for the humanization of a given antibody, hi general, it is necessary to minimize the substitutions of amino acid residues of human origin with residues of murine origin, for the introduction of murine residues increases the risk that the antibody will induce a HAMA response in the human patient. On the other hand, the complementarity determining regions (CDRs) contain the residues with the greater probability of interacting with the antigen and for this reason they must be maintained in the humanized antibody. They are defined by means of the sequence according to Kabat or by means of the structure according to Chothia. The advantage of using the second system to define them is that in general the CDRs are shorter and hence the humanized antibody is constituted by a lesser fraction of xenogenic fragments. In any case it has been demonstrated that generally following Rabat's definitions it is possible drastically to reduce the number of cycles required for humanization. Once the CDRs are defined, it is necessary to identify the canonical classes (defined by Chothia and Lesk) to which they belong and subsequently maintain the canonical residues in the humanized antibodies.<br>
It is also essential to analyze the residues that mediate the interaction between the light chain and the heavy chain of the variable domains (Table 1), maintaining any unusual residues in the humanized antibody (Singer et al, 1993; Daugherty et al.', 1991; De Martinoe/a/., 1991).<br>
Moreover, further amino acids to be maintained are selected based on their possible influence on the conformation of the CDRs and/or on the interaction with the antigen. When the amino acid differs between the framework of animal origin and the equivalent acceptor framework of human original, the amino acid of the acceptor framework should be substituted by the equivalent murine residue, if it is reasonable to expect that the amino acid is in direct non covalent contact with the antigen, or is adjacent to a<br>
CDR region, or in any case interacts with a CDR region (it is situated within 4-6 A from<br>
a CDR region).<br>
TABLE 1<br>
Residues that mediate the interaction between the light chain and the heavy chain of the<br>
variable domains<br>
LIGHT VARIABLE CHAIN L	HEAVY VARIABLE CHAIN H<br>
(Table Removed)<br>
In particular, a further analysis involves other residues which define the so-called Vernier zone, a zone that stabilizes the structure of the CDRs; it is important to maintain the characteristics of this region.<br>
Other residues candidate for mutation are amino acids of the acceptor framework which are unusual for a human immunoglobulin in that position. These residues can be substituted with amino acids deriving from the equivalent position of more typical human immunoglobuHns or alternatively residues originating from the equivalent position of the donor framework can be introduced into the acceptor framework when said amino acids are typical for the human immunoglobulins in those particular positions.<br>
Moreover, again on the basis of the consensus sequences of human immunoglobulins, mutations are introduced in the humanized form which insert residues preserved in the human instead of the unusual residues present both in the donor and in the acceptor framework.<br>
The respective pairs of crystallographic structures are then modified, first effecting the grafting of the CDRs of animal origin in the human frameworks. Then, all the mutations and retro-mutations described above are introduced. The modified structures are then assembled in composite immunoglobulins. The resulting models are refined by minimizing mechanical energy (in terms of torsion angles and binding angles and distances) using the force field.<br>
For all other regions, different from the specific ammo acid substitutions discussed above,  the  framework regions  of the  humanized immunoglobulins   are  usually substantially identical to the framework regions of the human antibodies from which they were derived. In any case in these engineered proteins obtained by grafting, the framework regions can vary relative to the native sequence at the primary structure level due to many ammo acid substitutions, deletions or insertions, terminal or intermediate, and other changes. Naturally, most of the residues in the framework region brings a very small or even non-existent contribution to the specificity or affinity of an antibody. Therefore, many individual conservative substitutions in the residues of the framework can be tolerated without appreciable variations of the specificity or affinity in the resulting humanized immunoglobulin. hi general, nevertheless, such substitutions are not desirable. It is possible to obtain modifications in the nucleotide sequence with a variety of widely employed techniques, such as site-specific mutagenesis (Gillman &amp; Smith, 1979; Roberts et a/.,1987).<br>
Alternative, polypeptide fragments can be produced, comprising only a part of the primary structure of the antibody, which fragments retain one or more peculiar activities of the immunoglobulins (e.g., the binding activity). These polypeptide fragments can be produced by means of proteolytic digestion starting from intact antibodies or inserting stop codons in the desired positions in the carriers bearing the coding DNA sequences for the variable regions of the heavy and light chain by means of site specific mutagenesis (in particular after the CHI region to produce Fab fragments or after the hinge region to produce (Fab')2 fragments. Antibodies in the form of scFv can be obtained by joining the variable regions of the heavy chain and of the light chain by<br>
means of a linker (Huston et al, 1988; Bird et al, 1988). The Fv or Fab fragments can be expressed in E. coli (Buchner and Rudolph, 1991; Skerra et al, 1991) or also in eukaryotic cells, preferably mammal derived. Considering that like many other genes, the genes of the immunoglobulm super family contain distinct functional regions, each characterized by one or more specific biological activities, the genes can be fused to functional regions deriving from other genes (e.g. enzymes) to produce fusion proteins (e.g. immunotoxins) provided with new properties.<br>
The expression of humanized imrnunoglobulin sequences in bacteria can be used to select   humanized   immunoglobulin   sequences,   characterized   by  higher   affinity mutagenizing the CDR regions and producing phage libraries for phage display. Using these libraries, it is possible to perform a screening in the search for variants at the level of the CDRs of the humanized immunoglobulins that have a higher affinity and/or binding specificity for the antigens. Methods to obtain phage-display libraries bearing sequences of the variable regions of immunoglobulins have been amply reported (Cesareni, 1992; Swimmer et al, 1992; Gram et al, 1992; Clackson et al, 1991; Scott &amp; Smith, 1990; Garrard et al, 1991). The sequences resulting from the variants of humanized immunoglobulins, whose CDRs were thus remodeled, are subsequently expressed in a host that is suitable to assure a high expression thereof. As stated above, the DNA sequences are expressed in the host cells after being operatively bound (i.e. positioned in such a way as to assure their functionality) to expression control sequences. These carriers can typically be replicated in the host organism as episomes or as an integral part of the chromosome DNA. Commonly, the expression carriers contain a selectable marker to allow to identify the cells that have been transformed with the DNA sequences of interest.<br>
For the production of the humanized immunoglobulins of the invention in recombinant form of scFv or in Fab form, prokaryotic systems are preferred. E. coli is one of the prokaryotic hosts that is particularly useful for cloning the DNA sequences of the present invention. Moreover, a great number of well characterized promoters is available, e.g. lac or trp operon or /3~lactamase or X phage. Typically, these promoters control expression and bear binding site for the ribosome, for the correct start and finish of transcription and translation. It is possible to increase the half-life of the humanized immunoglobulins of the invention produced in prokaryotic systems by conjugation with polyethylene glycol (PEG).<br>
Other single-cell organisms, such as yeasts, can be used for expression. The host of choice is Saccharomyces, using suitable carriers provided with expression control, replication termination and origin sequences.<br>
Insect cell cultures can also be used to produce the humanized immunoglobulins of the invention, typically using cells of S2 Drosophila transfected in stable fashion or cells of Spodoptera frugiperda with the expression system based on the Baculovirus ( Putlitz et al, 1990).<br>
Plants and cultures of vegetable cells can be used for the expression of the humanized immunoglobulins of the invention. (Larrick &amp; Fry, 1991; Benvenuto et al, 1991; Durin et al, 1990; Hiatt et al, 1989).<br>
However, in all these cases it is impossible to obtain the correct type of glycosylation * necessary to assure the effector function in the activation of the human immune system. For this purpose, it is possible to use tissue cultures of mammal cells to express the polypeptides of the present invention in integral form of IgGl, which have proven to be the most effective isotype among seric irnmunoglobulins in the induction of the immune response (Winnacker, 1987). It should be stressed that, considering that the isotype determines the lytic potential of an antibody, generally the IgGl isotype is used for therapeutic purposes (since it induces the immune response, both cell-mediated and mediated by the system of the complement), whilst the IgG4 is used for diagnostic applications (Riechmann et al, 1988). In particular, mammal cell are preferred, considering the great number of host cell lines developed for the secretion of intact immunoglobulins, among them the CHO cell lines, several lines of COS, the HeLa cells, myeloma cell lines (NSO, SP/2, YB/0 e P3X63.Ag8.653), transformed B cells or<br>
\.<br>
hybridomas. Expression carriers for these cells can include expression control sequences, such as a replication origin, a promoter, an enhancer (Queen et al, 1986), and the sequences required for ribosome binding, RNA splicing and polyadenylation, and sequences for transcription termination. The expression control sequences of choice are promoters deriving from immunoglobulin genes and from viruses, such as SV40, Adenovirus, Bovine Papilloma Virus, Cytomegalovirus and the like. Generally, the expression vector includes a selectable marker, such as the resistance to neomycin. For the expression of humanized antibodies, it is preferable to cultivate the mammal cell lines with a serum-free medium. For example, the HUDREG-55 cell line can easily be<br>
grown in Serum-Free and Protein-Free Hybridoma Medium Cat. No. S-2897 from Sigma (St. Louis, Mo.).<br>
The genes coding the humanized immunoglobulins of the invention can be used to generate    non    human    transgenic    animals,    which    express    the    humanized immunoglobulins of interest, typically in a retrievable body fluid such as milk or serum. Such   transgenes   comprise   the  polynucleotide   sequence   coding   the   humanized immunoglobulins operatively bound to a promoter, usually with an enhancer sequence, such as that of the rodent immunoglobulin or the promoter/enhancer of the casein gene (Buhler et al., 1990; Meade et al., 1990). The transgenes can be transferred into the cells or embryos by means of homologous recombination constructs. Among non human animals used: mouse, rat, sheep, bovine and goat (WO91/08216). Once they are expressed as intact antibodies, their dimers, the individual light and heavy chains, or in other forms the immunoglobulins of the present invention can be purified following standard procedures, such as precipitation with ammonium sulfate, affinity columns, chromatography on column (Scopes, 1982). For pharmaceutical applications, substantially pure  immunoglobulins  are  necessary,  with  minimum  homogeneity between 90 and 95%, but preferably between 98 and 99% or even higher. Once purified, partially or to the desired homogeneity, proteins can be used for therapeutic use (also in extra-body fashion), for diagnostic use (imaging for the diagnostics of tumors or of Alzheimer's    Disease)    or    to    develop    and    perform    biochemical    assays, immunofluorescent colorings and the like (see, in general, Lefkovits and Pernis, 1979 and 1981).<br>
A pharmaceutical application of the present invention pertains to the use of humanized immunoglobulin MNAC13 in the form of immunotoxin to eliminate TrkA-expressing cells (in the case of pancreas and prostate tumors). The immunotoxins are characterized by two components and are particularly suitable to kill particular cells both in vitro and in vivo. One component of the cytotoxic agent that is generally lethal for a cell is absorbed or if it interacts with the cell surface. The second component provides the means to address the toxic agent to a specific target cell type, such as the cells that express the epitope of the TrkA receptor. The two components are chemically bound to each other by means of any one of the great variety of chemical procedures available. For example, when the cytotoxic agent is a protein and the second component is an intact immunoglobulin the link can be mediated by cross-binding and heterobifunctional<br>
agents (SPDP, carbodiimide, glutaraldehyde). Alternatively, the two components can be bound genetically (Chaudhary et al, 1989). The production of various immunotoxiiis is reported by Thorpe et al. (1982).<br>
A great number of cytotoxic agents are suitable for application as immunotoxins. Cytotoxic agents can include radionuclides such as iodine 131 or other isotopes of iodine, yttrium 90, rhenium 188 and bismuth 212 or other isotopes that emit alpha particles, a great number of chemotherapeutic drugs such as vindesin, methotrexate, adriamycin and cisplatin; and cytotoxic proteins, such as proteins that inhibit and ribosomes (such as the pokeweed antiviral protein, the Pseudomonas exotoxin A and the diphteric toxin, ricin A and clavin of vegetable origin) or agents active at the cell surface level (such as phospholipase enzymes such as phospholipase C) - eds. Baldwin and Byers,1985; U.S. Ser. No. 07/290,968; Olsnes and Phil, 1982. It should be stressed that the cytoxic region of the immunotoxin can itself be immunogenic and consequently limit the clinical usefulness of the fusion protein in case of chronic or long term therapy. An alternative to avoid the problem of the immunogenicity of the toxin is to express in fusion with the binding domain of the antibody a protein able to interact with the DNA and bind to this fusion protein the expression carrier that contains the toxin expression cassette. The numerous positive charges of protamin, a human protein that binds the DNA, can interact in stable fashion with the negative charges of the DNA, generating a fusion partner for the neutral charge antibody, much more stable and less immunogenic than the toxin itself. After internalization of the antibody-plasmide complex via receptor mediated endocytosis, the expression of the toxin causes the death of the cell. Moreover, selectivity towards the target cell to be eliminated can be further enhanced by inserting inducible or cell-specific promoters into the toxin expression cassette. This approach is aimed at maximizing the selective elimination of tumor cells while minimizing toxicity side effects (Chen et al, 1995).<br>
The component that addresses the immunotoxin to the correct target includes the MNAC13 humanized immunoglobulin of the present invention in the form of intact immunoglobulin or of the binding fragment or as Fab or Fv fragment. Typically, the antibodies in the immunotoxins are of the human isotype IgM or IgG, but other constant regions can be used as well.<br>
The antibodies and the pharmaceutical compositions of this invention are particularly useful for administration, following any effective methodology to address the antibodies<br>
at the level of the tissue involved in the pathology. This includes (but is not limited to): intraperitoneal, intramuscular, intravenous, subcutaneous, intratracheal, oral, enteral, parenteral, intranasal or dermal administration. The antibodies of the present invention can typically be administered for local application by injection (intraperitoneal or intracranial - typically in a cerebral ventricle - or inrrapericardiac or intrabursal) of liquid formulations or by ingestion of solid formulations (in the form of pills, tablets, capsules)  or  of liquid formulations  (in the  form  of emulsions  and  solutions). Compositions   for  parenteral   administration   commonly  comprise   a   solution   of immunoglobulin   dissolved   in   a  compatible,   preferably   aqueous   solution.   The concentration of the antibody in these formulations can vary from less than 0.005% to 15-20% and it is selected mainly according to the volumes of the liquid, its viscosity, etc., and according to the particular administration mode selected. Alternatively, the antibodies can be prepared for administration in solid form. The antibodies can be combined with different inert or excipient substances, which can include ligands such as microcrystalline cellulose, gelatin or Arabic rubber; recipients such lactose or starch; agents such as alginic acid, Primogel or corn starch; lubricants such as magnesium stearate, colloidal silicon dioxide; sweeteners such as saccharose or saccharin; or flavors, such as mint and methyl salicylate. Other pharmaceutical administration    systems    include   hydrogel,    hydroxymethylcellulose,    liposomes, microcapsules, microemulsions, microspheres, etc. Local injections directly in the tissues affected by illness such as rumors is a preferential method for the administration of the antibodies of the present invention.<br>
The antibodies of the invention can be frozen or lyophilized and reconstituted immediately before use in a suitable buffer. Considering that lyophilization and reconstitution can determine a variable loss in the activity of the antibody (for conventional imniunoglobulins, class IgM antibodies tend to have a greater loss of activities than class IgG antibodies), administration levels must be calibrated to compensate for this fact.<br>
Thanks to their high blocking capacity, the compositions containing the antibodies of the present invention can be administered for prophylactic and/or therapeutic treatments to prevent or reduce the inflammatory component associated to pathological situations or chronic pain, in particular chronic visceral pain (associated to physiological<br>
disorders, such as dysmenorrhea, dyspepsia, gastrointestinal reflux, pancreatitis, visceralgia or irritable intestine syndrome).<br>
In prophylactic applications, compositions containing antibodies of the present invention are administered to patients who do not yet suffer from a particular pathology to enhance their resistance.<br>
The antibodies of the present invention also provide a method for reducing the volume of prostate or pancreas tumors and for preventing further tumor growth or reduce the rate of growth of the tumor. This effect can be mediated by both the humanized antibodies of the present invention because they are extremely effective in the neutralization of the interaction between NGF and TrkA, necessary to sustain tumor growth and progression in autocrine or paracrine fashion. Moreover the humanized form of MNAC13 interacts with a membrane receptor and hence can also be used for the direct elimination of neoplastic cells because they are able to activate the host's immune response (if administered in the form of IgGl) or to convey a cytotoxic agent localizing it at the level of the cancerous mass (if administered in the form of immunotoxin). Their administration in the tumor site preferably takes place through direct and localized injection into the tissue or near the tumor site. For systemic administration, doses vary from 0.05 mg/kg per day to 500 mg/kg per day, although dosages in the lower region of the range are preferred because they are easier to administer. Dosages can be calibrated for example to guarantee a particular level in the plasma of the antibody (in the range of about 5-30 mg/ml, preferably between 10-15 mg/ml) and maintain this level for a given period of time until the clinical results are achieved. Humanized antibodies should be eliminated much more slowly and require lower dosages to maintain an effective level in the plasma; moreover, considering the high affinity, administration is less frequent and less sizable than with antibodies having lower affinity. The therapeutically effective dosage of each antibody can be determined during the treatment, based on the reduction in the volume of the tumor or on the rate of growth of the tumor or ideally on the total disappearance of the cancerous pathological state. Effective methods for measuring or assessing the stage of pancreatic or prostatic tumors are based on the measurement of the prostate specific antigen (PSA) in blood, on the measurement of the survival time for pancreas tumors, on the measurement of the slowing or inhibition of diffusion for metastases in the case of both tumor.<br>
For direct injection at the level of the tumor site, dosage depends on different factors including the type, stage and volume of the tumor, along with many other variables. Depending on tumor volume, typical therapeutic doses may vary from 0.01 mg/mm and 10 mg/mm injections which can be administered with the necessary frequency. Another method to assess the effectiveness of a particular treatment is to evaluate the inhibition of the TrkA receptor, e.g. by measuring its activity by means of ELISA assay (Angeles etal.,1996),<br>
It is important to stress that, TrkA is configured not only as a therapeutic target but also as a diagnostic target for in vivo imaging, e.g. for imaging of TrkA positive tumors (as a positive or negative marker, depending on tumor type and origin) and imaging on cells of the basal forebrain (as a precocious marker of insurgence of Alzheimer's disease). The MNAC13 humanized antibody of the present invention can also find a wide variety of in vitro applications (ELISA, IRMA, RIA, immunohistochemistry). For diagnostic purposes, the antibodies can be both marked and unmarked. Unmarked antibodies can be used in combination with other marked antibodies (secondary antibodies), which are reactive against humanized, or human antibodies (e.g. specific antibodies  for the  constant regions  of human immunoglobulins).  Alternatively, antibodies can be marked directly. A wide variety of markings can be used, e.g. radionuclides,  fluorophores,   colorings,   enzymes,  enzymatic  substrates,  enzymatic factors, enzymatic inhibitors, ligands (in particular aptenic), etc. Numerous types of immunologic assays are available in the sector.<br>
In particular, for imaging diagnostic applications, to the antibody is conjugated an agent that is detectable or marked in isotopic manner (using radioisotopes of iodine, indium, technetium) or in paramagnetic manner (paramagnetic atoms or ions, such as transition elements, actinides and rare earths; in particular, manganese JH, copper n and cobalt II) as described by Goding (1986) and Paik et al. (1982). Imaging procedures entail the intravenous, intraperitoneal or subcutaneous injection (in lymphatic drainage regions to identify lymph node metastases) and they use detectors of radionuclide emissions (such as scintillation /3 counters) in the case of immunoscintigraphy; if a paramagnetic marking is used instead, an NMR (Nuclear Magnetic Resonance) spectrometer is used. The invention shall now be described in its non h'miting embodiments, with reference to the following figures:<br>
Figure 1: A) Analysis by means of polyacrylamide in denaturing conditions (SDS-PAGE 12%) and coloring with Coomassie Blue of the result of the purification of the Fab fragment of the MNAC13 antibody (well 1: sample of MNAC13 antibody digested proteolytically with papaine; well 2: fraction bound to the DEAE Sephacell ionic exchange resin and eluted with NaCl 250mM; well 3: molecular weights; well 4: Fab fragment of the purified and concentrated MNAC13 antibody); B) typical crystal of the Fab fragment of the MNAC13 antibody C) High resolution diffraction spectrum obtained with a crystal of the Fab fragment of the MNAC13 antibody; D) Ramachandran chart of the torsion angles of the main chain of the heavy and light domains of the Fab fragment of the MNAC13 antibody.<br>
Figure 2: A) Analysis by means of polyacrylamide in denaturing conditions (SDS-PAGE 12%) and coloring with Coomassie Blue of the result of the purification of the Fab fragment of the aDll antibody (well 1: sample of aDll antibody digested proteolytically with papaine; well 2: Fab fragment of the purified and concentrated aDll antibody; well 3: molecular weights); B) typical crystal of the Fab fragment of the aDll antibody C) High resolution diffraction spectrum obtained with a crystal of the Fab fragment of the aDll antibody; D) Ramachandran chart of the torsion angles of the main chain of the heavy and light domains of the Fab fragment of the aDll antibody.<br>
Figure 3: A) B) C) D) Distributions of the humanized or human origin antibodies (named using the PDB codes of their crystallograpnic structures) according to the three analyzed variables; E) F) Deviations of the humanized or human origin antibodies from the hypothetical optimal value of MNAC13 (calculated both considering the degree of overall identity and of homology - in blue - and framework level - in magenta-) G) Structural alignment with the Fv fragment of MNAC13 of the respective regions of the humanized or human origin antibodies, selected according to the degree of identity and homology with the murine antibodies and to the degree of resolution of available structural data; H) I) Structural alignment with the Fv fragment of MNAC13 (shown in cyan) of the respective region of the selected humanized antibody 1ADO (shown in red) in H); of the model of the antibodies resulting after CDR grafting (shown in yellow at the framework level, in white at the CDR level) in I); L) Model of the Fv fragment of the MNAC13 humanized antibody obtained as a result of the identification of putative<br>
retro-mutations in the chosen framework (human origin residues are shown in green and murine origin residues are shown in magenta).<br>
Figure 4: A) B) C) D) Distributions of the humanized or human origin antibodies (named using the PDB codes of their crystallographic structures) according to the three analyzed variables; E) F) Deviations of the humanized or human origin antibodies from the hypothetical optimal value of aDll (calculated both considering the degree of overall identity and of homology - in blue - and framework level -in magenta-) G) Structural alignment with the Fv fragment of aDll of the respective regions of the humanized or human origin antibodies, selected according to the degree of identity and homology with the murine antibodies and to the degree of resolution of available structural data; H) I) Structural alignment with the Fv fragment of aDll (shown in cyan) of the respective region of the selected humanized antibody 1JPS (shown in red) in H); of the model of the antibodies resulting after CDR grafting (shown in yellow at the framework level, in white at the CDR level) in I); L) Model of the Fv fragment of the aDl 1 humanized antibody obtained as a result of the identification of putative retro-mutations in the chosen framework (human origin residues are shown in cyan and murine origin residues are shown in purple).<br>
Figure 5: Alignment of the primary structures of the variable regions of the heavy chain (A) and of the light chain (B) respectively of MNAC13 (SEQ ID No. 22, SEQ ID No. 24), of the humanized antibody selected for humanization (1ADO; SEQ ID No. 39, SEQ ID No. 40), of the humanized form of MNAC13 after CDR grafting on the framework of 1ADO and of the described retro-mutations and mutations (Hum MNAC13: SEQ ID No. 37, SEQ ID No. 38). CDRs are highlighted in the sequence of the humanized form of the two chains of MNAC13 by underlined character.<br>
Figure 6: Alignment of the primary structures of the variable regions of the heavy chain (A) and of the light chain (B) respectively of aDll (SEQ ID No. 2, SEQ ID No. 4), of the humanized antibody selected for humanization (UPS; SEQ ID No. 19, SEQ ID No. 20), of the humanized form of aDl 1 after CDR grafting on the framework of 1 ADO and of the described retro-mutations and mutations (Hum aDll; SEQ ID No. 17, SEQ ID No. 18). CDRs are highlighted in the sequence of the humanized form of the two chains of aDl 1 by underlined character.<br>
Figure 7: A) nucleotide sequence of the cDNA of the variable region of the light chain of the murine form of MNAC13 (SEQ ID No. 23); B) nucleotide sequence of the cDNA of the variable region of the heavy chain of the murine form of MNAC 13 (SEQ ID No. 21); C) and E) sequence of the oligonucleotides drawn to obtain the humanized form of the variable region of the light chain of MNAC13 (SEQ ID No. 38): L1S: SEQ ID No. 31; L2AS: SEQ ID No. 32; L3S: SEQ ID No. 33; L4AS: SEQ ID No. 34; L5S: SEQ ID No. 35; L6AS: SEQ ID No. 36, by means of the overlap-assembly PCR technique, shown together with the corresponding translation into amino acid sequence; D and F) sequence of the oligonucleotides drawn to obtain the humanized form of the variable region of the heavy chain of MNAC13 (SEQ ID No. 37): HIS: SEQ ID No. 25; H2AS: SEQ ID No. 26; H3S: SEQ ID No. 27; H4AS: SEQ ID No. 28; H5S: SEQ ID No. 29; H6AS: SEQ ID No. 30, by means of the overlap-assembly PCR technique, shown together with the corresponding translation into amino acid sequence. Figure 8: A) nucleotide sequence of the cDNA of the variable region of the light chain of the rat form of aDll (SEQ ID No. 3); B) nucleotide sequence of the cDNA of the variable region of the heavy chain of the murine form of aDll (SEQ ID No.l); C) and E) sequence of the oligonucleotides drawn to obtain the humanized form of the variable region of the light chain of aDll (SEQ ID No. 18): L1S: SEQ ID No. 11; L2AS: SEQ ID No. 12; L3S: SEQ ID No. 13; L4AS: SEQ ID No. 14; L5S: SEQ ID No. 15; L6AS: SEQ ID No. 16, by means of the overlap-assembly PCR technique, shown together with the corresponding translation into amino acid sequence; D and F) sequence of the oligonucleotides drawn to obtain the humanized form of the variable region of the heavy chain of aDll (SEQ ID No. 17): HIS: SEQ ID No. 5; H2AS: SEQ ID No, 6; H3S: SEQ ID No. 7; H4AS: SEQ ID No. 8; H5S: SEQ ID No. 9; H6AS: SEQ ID No. 10, by means of the overlap-assembly PCR technique, shown together with the corresponding translation into amino acid sequence.<br>
Figure 9: Maps of the plasmids used to clone the sequences of the humanized variable regions of both antibodies obtained by overlap-assembly PCR. A) pVLexpress for the variable domain of the light chain, B) pVHexpress for the variable domain of the heavy chain, C) plasmid resulting from cloning in pVLexpress the variable region of the light chain of each humanized antibody, D) alternative constructs obtained as a result of cloning in pJ^H express the variable region of the heavy chain of each humanized<br>
antibody: 1) for the expression in intact immunoglobulin form IgGl, 2) for expression in Fab fragment form, 3) for expression in immunotoxin form.<br>
Figure 10: Comparison of the binding activity of the MNAC13 antibody in chimeric form and in humanized form by means of ELISA assay, conducted immobilizing on plastic TrkA in immunoadhesin form: A) comparison between serial dilutions of supernatants of transfected COS cells, subsequently concentrated; B) comparison between serial dilutions of supernatants of transfected COS cells purified by means of G sepharose protein.<br>
Figure 11: Assay of the binding activity of the aDll antibody in humanized form by means of ELISA assay, conducted immobilizing on plastic NGF. RESULTS<br>
X-RAY STRUCTURES OF THE Fab FRAGMENT OF THE MNAC13 AND oDll MONOCLONAL ANTIBODIES<br>
Both monoclonal antibodies were obtained and purified according to standard procedures. The MNAC13 IgGl and aDll IgG2a immunoglobulins were expressed in the supernatant by means of culture of hybridoma cells and concentrated by precipitation with 29% ammonium sulfate followed by dialysis in PBS. Both immunoglobulins were purified by affinity chromatography using a column of Protein G Sepharose (Pharmacia).<br>
Following dialysis in phosphate buffer 10 mM pH 7, EDTA 20 mM using Spectra-Por 12/14K membranes (Spectrum) at 4° C, each sample was concentrated by means of Centricon SOKDa ultrafiltration units (Amicon) and incubated with 13mM Cys and treated with immobilized papaine (Pierce) (with an enzyme : substrate ratio of 1:15) for 5 h at 37°C. The procedure for purifying the respective Fab fragments is diversified, although it is always based on ionic exchange chromatography.<br>
In the case of MNAC13, after dialysis against Tris HC1 lOOmM pH 8.0, it was possible to eliminate the Fc fragments through a DEAE-Sephacel column (Pharmacia) balanced with the same buffer. FabMNACIS was collected in the excluded volume whilst the Fc fragments and a fraction of undigested IgGl were eluted with 250mM NaCl. The Fab fragment was separated from undigested IgGl by gel filtration on a Superdex G75 (Pharmacia) column balanced with Tris HC1 lOOmM pH 8.0, NaCl 150mM . The homogeneity and purity of the fractions was controlled by electrophoretic separation on 12% polyacrylamide gel followed by coloring with Coomassie (Figure 1A). The<br><br>
concentration of the purified protein was determined by means of Lowry assay (Bio-Rad). From 11 of hybridoma sumatant, it was possible to obtain up to 3 mg of MNAC13 Fab (with purity exceeding 99%).<br>
In terms of the purification of the Fab fragment of the gDll antibody, the sample treated with papaine was dialyzed against 10 mM pH 7.8 phosphate buffer: the Fc fragments were eliminated through a DEAE-Sephacel column (Pharmacia) balanced with this same buffer. The Fab fragment of aDl 1 was collected in the excluded volume, whilst the Fc fragments and a fraction of undigested IgG2a were eluted with 250mM pH 6.8 phosphate buffer. The Fab fragment was separated from the undigested IgG2a by means of filtration gel on a Superdex G75 column (Pharmacia) balanced with 10 mM pH 7.8 phosphate buffer, NaCl 150mM. The homogeneity and purity of the fractions was controlled by electrophoretic separation on 12% polyacrylamide gel followed by coloring with Coomassie (Figure 2A). The concentration of the purified protein was determined by means of Lowry assay (Bio-Rad). From 11 of hybridoma surnatant, it was possible to obtain up to 6 mg of aDl 1 Fab (with purity exceeding 99%). Both the Fab fragment of the MNAC13 antibody purified in lOmM Tris pH 8.0, 50mM NaCl, and the Fab fragment of the aDll antibody purified in lOmM Na phosphate pH 7.8 and 50mM NaCl were concentrated to 5-10 mg/ml by means of Centricon 30KDa ultrafiltration unit (Amicon). The crystallization experiments were conducted following the hanging-drop method at 16°C following a factorial combination approach (Jancarik &amp; Kim, 1991) using Crystal Screen I and n (Hampton Research -Laguna Niguel, CA, USA-) and Screening Kit (Jena BioSciences).<br>
Drops of 2 jul of the concentrated proteic sample were added to an equal volume of the solution containing the precipitant agent and balanced by diffusion with a solution in the reservoir (0.7 ml) in 24 well Linbro plates.<br>
hi terms of the Fab fragment of the MNAC13 antibody, the most promising initial result, obtained with equal volumes of protein and precipitant containing 2M ammonium sulfate, 5% v/v isopropanol (Crystal Screen II, Reactant #5), was optimized until obtaining crystals that grow in about one week, similar to what is shown in Figure IB.<br>
In terms of the Fab fragment of the aDll antibody, the most promising initial result, obtained using equal volumes of protein and precipitant containing 20% PEG4000, 0.6M NaCl, lOOmM MES pH 6.5 (Kit number 4, solution C2), required a long<br><br>
optimization process, modifying the composition of the precipitant agent to PEG4000) 0.6M NaCl, lOOmM BTP pH 5.5 and the ratios between protein and precipitant solution (1.5 : 1) until obtaining crystals that grow in about one week, similar to what is shown in Figure 2B.<br>
In both cases, an initial set of low resolution data was collected on the XRD1 diffraction<br>
line of the ELETTRA synchrotron (Trieste, Italy), and then a second, more complete set<br>
of data at higher resolution was collected on the ED14-EH1 diffraction line of the ESRF<br>
synchrotron (Grenoble, France) The crystals were frozen under liquid nitrogen flow<br>
with the cooling system by Oxford Cryosystems (Oxford, UK), using in the case of the<br>
Fab fragment of the MNAC13 antibody a solution containing 2.2 M ammonium sulfate,<br>
6% v/v isopropanol and 20% v/v glycerol as cryoprotector. A representative high<br>
resolution diffraction spectrum for each protein is shown in Figures IB and 2B.<br>
All four sets of X-ray diffraction data were processed, indexed, integrated and<br>
subsequently scaled using the DENZO and SCALEPACK programs (Otwinowski &amp;<br>
Minor,  1997) respectively, while the CCP4 (Collaborative Computational Project,<br>
Number 4, 1994) package was used for data reduction. The statistics for the collection<br>
and processing of high and low resolution data of the crystals of the Fab fragment of the<br>
MNAC13 antibody are set out in the following table:<br>
X-ray source	ELETTRA	ESRF<br>
Wavelength (A)	1.000	0.934<br>
Detector	mar345	marCCD<br>
Spatial group	~?2\2\2l	P2i2]2i<br>
Parameters of the unitary cell<br>
a (A)	52.78	52.73<br>
b(A)	67.53	67.55<br>
c(A)	111.51	111.43<br>
Mosaicity (°)	0.40	0.47<br>
Resolution interval (A)	12.0 - 2.50	17.0-1.80<br>
(2.59-2.50)	(1.83-1.80)<br>
No. measures	98688	414115<br>
No. of reflexes observed with I &gt;0   56918	227914<br>
No. of unique reflexes with I &gt;0       14203 (1371)	38392 (1893)<br>
Completeness (%)	99.5 (99.3)	99.5 (99.6)<br><br><br>
Redundancy	4.0 (4.0)<br> of the measured data 9.4 (4.7)<br>
R<br>
sym<br>
5.7(15.2)<br><br>
5.9 (4.9) 8.2(1.1) 6.3 (39.8)<br><br>
Similarly, the following table summarizes the statistics for the collection and processing<br>
of high and low resolution data of the crystals of the Fab fragment of the DD1 1<br>
antibody:<br>
X-ray source<br>
Wavelength (A)<br>
Detector<br>
Spatial group<br>
Parameters of the unitary cell<br>
a (A)<br>
b(A)<br>
c(A)<br>
TO Mosaicity (°)<br>
Resolution interval (A)<br>
No. measures	124456	492594<br>
No. of reflexes observed with I &gt;0    74241	399184<br>
No. of unique reflexes with I &gt;0       23413 (2162) 47951 (3198)<br>
Completeness (%)	98.2 (92.4)	97.2 (78.4)<br>
Redundancy	5.7 (5.2)	6.7 (7.5)<br><i> of the measured data 29.6 (6.7)	9.5 (2.1)<br>
Rsym(%)	11.0(33.5)	5.8(27.8)<br>
Where the values in parenthesis refer to the shell with the highest resolution.<br>
Considering the high number of available structures of Fab fragments, the most<br>
convenient  method  to  determine  the  structure  of both proteins  was  Molecular<br>
Substitution.  In a research in the Protein Data Bank (Herman et a!., 2000)  for<br>
homologous structures, the selection criteria gave priority to the combination between comparable resolution and highest level of sequence identity. On these bases, respectively, were selected<br>
for MNAC13: 1BM3: the structure of the complex between Fab fragment of the Opg2 Fab immunoglobulin and the peptide recognized by it (Kodandapani et al., 1999), resolved at a resolution 2.0A and provided with a sequence identity respectively of 70 and 88% for the heavy and light chain.<br>
for otDll: 1CIC: the structure of the complex of idiotype-anti-idiotype Fab fragments FabD1.3-FabE225 (Bentley et al, 1990), resolved at a resolution 2.5A and provided with a sequence identity respectively of 82 and 82.65% for the heavy and light chain. The determination of both structures was obtained by the Molecular Substitution method using the AMoRe program (Navaza, 1994), with the respective models using separately the constant domains and variable domains considering the extreme variability of the angle formed by the axis of binary pseudosymmetry between the variable and constant regions. The solution obtained in the determination of the structure, of the Fab fragment of MNAC13 following refined with rigid body is shown in the following table:<br>
Similarly, the solution obtained in the determination of the structure of the Fab fragment of ccDll following refinement with rigid body for the spatial group C2 is shown in the following table:<br>
(Table Removed)<br>
Where V= variable domain<br>
C= constant domain<br>
a, P, 7 = Eurelian angles (°).<br>
x,y,z = Translation (fractionary).<br>
Cf = Correlation of the amplitudes (xlOO).<br><br>
Rf = Crystallographic R factor (xlOO). Q = Correlation of the intensities (xlOO). Cp~ Correlation of the truncated Patterson function (xlOO).<br>
The subsequent refinement of the two structures was obtained by means of a cyclic procedures, comprising two alternated phase: manual construction of the model using the interactive software for computer graphics "O" (Kleywegt and Jones, 1994); positional refinement and refinement of B isotropic thermal factors using automatic protocols of the CNS suite, Crystallography and NMR System (Briinger et al, 1998). The procedure after some phases of refinement with rigid body, contemplated different refinement cycles. Once the insertion of all mutations and deletion is completed to complete the models, the localization of the water molecules and any ions and ligands was conducted. At the end, maintaining the model as close as possible to the ideal values in terms of stereochemical, the positional weight wa and the weight of the thermal factor B r-weight were optimized. The statistics and the final parameters that describe the quality of the model obtained for the Fab fragment of the MNAC13 antibody are summarized in the following table:<br>
Number of protein atoms	3244<br>
Number of solvent atoms	351<br>
Number of sulfate ions	4<br>
Number of Tris molecules	1<br>
Number of isopropanol molecules	1<br>
Resolution interval (A)	3 9-1.778<br>
Final R factor	19.35%<br>
Final Rfree factor (calculated on 10% of the data)     23.22%<br>
Rms Deviations<br>
Binding distances (A)	0.008<br>
Binding angles (°)	1.456<br>
Dihedral angles (°)	27.29<br>
Improper angles (°)	0.928<br>
Mean Isotrope Thermal Factor (A2)<br>
Complete protein	23.55<br>
Light chain	24.14<br>
Heavy chain	22.99<br><br>
Water molecules	31.95<br>
Ions (sulfate)	55.94<br>
Tris	46.06<br>
Isopropanol	32.60<br>
Similarly, the statistics and the final parameters that describe the quality of the model<br>
obtained for the Fab fragment of the aDll antibody are summarized in the following<br>
table:<br>
Number of protein atoms	3229<br>
Number of solvent atoms	403<br>
Number of chloride ions	1<br>
- Resolution interval (A)	30-1.70<br>
Final R factor	19.54%<br>
Final Rftee factor (calculated on 10% of the data)     24.22%<br>
Rms Deviations<br>
Binding distances (A)	0.0096<br>
Angoli di legame (°)	1.6571Binding angles (°)<br>
1.6571<br>
Angoli dieDihedral angle (°)	27.40<br>
Improper angles (°)	1.048<br>
Mean Isotrope Thermal Factor (A2)<br>
Complete protein	25.58<br>
Light chain	24.14<br>
Heavy chain	22.99<br>
Water molecules	38.80<br>
Ions (chloride)	20.58<br>
Moreover,   both  models were   examined  by  final   geometry  analysis   with  the<br>
PROCHECK suite (Laskowski et al, 1993) as shown in the respective tables and in the<br>
respective Ramachandran charts (Figures ID and 2D).<br>
USE OF THE X-RAY STRUCTURES OF THE Fab FRAGMENT OF THE MNAC13<br>
and   aDll    MONOCLONAL   ANTIBODIES   IN   THE   SELECTION    OF   A<br>
FRAMEWORK OF HUMAN ORIGIN<br>
In the selection of human antibody framework, the approach described above was follows, which combines on the degree of identity between the antibody of murine and human origin at the primary sequence level to the degree of structural similarity of the polypeptide skeletons.<br>
ID particular, a series of possible acceptor frameworks of human origin or humanized antibodies was selected on the basis of the highest level both of homology and of identity of the primary structures by a search in the BLAST database. This selection was conducted for both blocking antibodies both considering the entire variable regions of the antibodies and narrowing the search to the framework regions. Within each group of selected antibodies, only those for which structural data with high resolution or otherwise with resolution comparable to that of the structures obtained by us (i.e. no greater than 2.5A) are available were considered, conducting a search in PDB (Protein Data Bank). The respective amino acid skeleton were then superimposed using the "superimpose" software (Diederichs, 1995).<br>
Figures 3G and 4G show the result of the alignment between the Fv region (respectively of MNAC13 and aDll) and the tertiary structures of the alpha carbon atom skeletons of the humanized or human origin antibodies, selected on the basis of the optimal alignment of the primary structures with the antibody to be humanized and of the high resolution of the available structural data.<br>
To assess the degree of superposition of each individual structure, of human origin or engineered, both with MNAC13 and with aDl 1 the RMS was calculated between atoms of alpha carbon constituting the respective amino acid skeletons, not considering atom pairs with an RMS exceeding 2A.<br>
The selection of the optimal framework for humanization is configured as a three-variable problem, which can thus be represented in space, both when associating the homology level and the degree of identity to the structural alignment. This type of analysis was then conducted also reducing the regions in question in the two types of alignment to the regions of the respective frameworks.<br>
As shown in Figures 3 and 4, the distributions of the antibodies considered in the space of the three analyzed variables (respectively, value of RMS, percentages of atoms on which RMS was calculated and a similitude index between primary structures, i.e. percentage of overall identity -A-, of overall homology -C-, of identity at the<br>
framework level -B-, of homology at the framework level -D-) are mutually coherent and consistent for both cases considered.<br>
Moreover, comparing these distributions with the optimal position in the space of the three variables which each antibody would occupy if it were of human origin, it is possible clearly to identify the human origin antibody that most approximates this ideal position at the primary and tertiary structure level. To rationalize, in the case of both antibodies, this result, in each of the four analyses the deviations from the hypothetical optimal position were calculated for each position of the humanized or human origin antibodies considered (Figure 3E and 3F for MNAC13 and Figure 4E and 4F for aDll). hi this case, too, the results are consistent and confirm the previous indications. On the basis of this method of selection, two different humanized antibodies were "   selected as acceptor framework in the subsequent process of CDR grafting for the humanization of the two  antibodies  neutralizing the NGF/TrkA interaction.   In particular, Figures 3H and 4H show the structural alignment at the level of the Fv region of the two blocking antibodies with the respective selected humanized antibody, i.e. using the PDB, IJPS codes for aDll and 1ADO codes for MNAC13. Figures 31 and 41 compare the same region of the murine antibody with the model of the same antibody following CDR grafting.<br>
Once the CDRs are defined, the canonical classes (defined by Chothia and Lesk) to which they belong were identified and subsequently the canonical residues in the humanized antibody were maintained: for each antibody, they were highlighted with underlined character in Figure 5 and Figure 6.<br>
hi regard to the subsequent analysis of the retro-mutations to be introduced, to maintain the residues that mediate the interaction between the light chain and the heavy chain of the variable domains, the following retro-mutations were inserted to maintain title interface between the two domains: L46 and H35, H37 for MNAC13 L34, L46, L92 and H35 for ccDl 1.<br>
Moreover, to maintain the characteristics of the Vernier zone, the following retro-mutations were made: L98forMNAC13<br>
H71 for aDll  (which in any case regard substitutions for amino acid residues represented in human consensus sequences).<br>
Subsequently, following the comparison with the main consensus sequences of human<br>
inimunoglobulins, the following retro-mutations were made:<br>
LI, L2, L13, L50, L73, L104 and H24, H48, H49, H73, H76, H82B, H87, H90, H93 for<br>
MNAC13<br>
L56forccDll<br>
Moreover, again on the basis of the consensus sequences of human irnmunoglobulins, in<br>
the humanized form of MNAC13 the following mutations were introduced which insert<br>
residues preserved in the human instead of the unusual residues present both in the<br>
donor and in the acceptor framework.<br>
L42 (L-&gt; Q), L83 (I-&gt; V) and H83 (Q-* R), H89 (I-* V).<br>
For the same reason, the following mutation was introduced in the humanized form of<br>
oDll.H67(V-&gt;F).<br>
The respective pairs of crystallographic structures were modified, first effecting the grafting of the CDRs of animal origin in the humanized frameworks. Then, all the mutations and retro-mutations described above were introduced. The modified structures were then assembled in composite immunoglobulins. The resulting models were refined by minimizing mechanical energy (in terms of torsion angles and binding angles and distances) using the force field.<br>
HUMANKATION OF THE MNAC13 AND oDl 1 MONOCLONAL ANTIBODIES<br>
After selecting the donor humanized antibody of the framework to achieve the CDR<br>
grafting of MNAC13, the respectively variable regions are designed which combine the<br>
murine CDRs of MNAC13 with the framework of the humanized antibody modified<br>
according to the mutations set out above. A similar procedure was followed for oD. 11.<br>
Substantially, the two humanized variable regions can be obtained by a procedure based<br>
on the overlap assembly PCR method, using oligonucleotides of about 80 bases, which<br>
alternate the sense and anti-sense filament with consecutive superposed for a length of<br>
20 bases in such a way as to allow the formation of partially dual filament molecules as<br>
a result of hybridation (Figures 7B and 8B). After filling discontinuities by means of<br>
Vent polymerase, the dual filament is amplified for PCR using as primers two short<br>
oligonucleotides bearing the sequences at the 5' of the dual filament itself together with<br>
restriction   sites   suitable   for   the   subsequent   directional   cloning   (respectively<br>
ApaLI/BglTI for the cloning of the variable domain of the light chain and BssHH/BstEII<br>
for the cloning of the variable domain of the heavy chain), after digestion with<br>
respective restriction enzymes, in the pVLexpress plasmid for the variable domain of the<br>
light chain (Figure 9A) and in the pVHexpress plasmid for the variable domain of the<br>
heavy chain (Figure 9B). These carriers allow to express in fusion with the cloned<br>
sequences the constant domains of human origin respectively CK and CHI, CH2 and<br>
CHS. Using these vectors, it is therefore possible to express both antibodies in the form<br>
of IgGl molecules (Figure 9C and 9D1) in human cell lines like those listed above.<br>
To obtain both humanized antibodies in the form of Fab fragments, it is sufficient to act<br>
solely on the carrier in which the heavy chain is cloned. In particular, it is possible to<br>
substitute the entire constant part with the sole CHI domain amplified for PCR using<br>
specific primers provided with restriction sites at the extremes for the SacII-Xbal<br>
directional cloning (as shown in Figure 9D2).<br>
Lastly, to obtain the MNAC13 humanized antibody in the form of immunotoxin, it is<br>
possible to express at the carboxy-terminal of the constant domain CHI the basic<br>
protamin protein amplified for PCR using specific primers provided with the restriction<br>
site at the extreme for non direction Xbal cloning (as shown in Figure 9D3).<br>
EXPRESSION AND BINDING ASSAY OF THE MNAC AND oDll HUMANIZED<br>
ANTIBODIES<br>
250 thousand COS cells were co-transfected with 1 jig of coding plasmidic DNA for<br>
VH at VK of each humanized antibody (a total of 2 ug) by means of FuGENE<br>
according to recommended protocol (Roche). The constructs were used to obtain the<br>
humanized antibodies in the form of IgGl.<br>
hi parallel to the co-transfections of the constructs described above, the corresponding<br>
chimeric forms were co-transfected for each antibody, i.e.:<br>
the murine VH of MNAC 13 cloned in CMV pVH express and the murine Vk of<br>
MNAC 13 cloned in CMV pVk express;<br>
in regard to oDll the rat VH is cloned in fusion with the Cj of human origin in<br>
pcDNAl and the rat Vk is cloned in fusion with the Oy of human origin pcDNAl.<br>
After 72 hours from the transfection, the supernatant containing the immunoglobuUns<br>
expressed by the host cells was collected, and concentrated using Centriprep 50<br>
(Amicon).<br>
The ability to recognize the respective ligands of the two humanized antibodies was<br>
verified by means of ELISA assay and compared with respective chimeric forms. The<br>
results are shown in Figure 10 and 11.<br><br>
For immobilization on plastic, 96 well Maxi sorb plates were incubated at 4°C<br>
overnight with a solution containing the respective ligands of the two antibodies (the<br>
purified recombinant knmunoadhesin TrkA Camel and the murine NGF purified from<br>
submandibular glands) at a concentration of 10 Jig/ml in 0.1M pH 9.6 sodium carbonate<br>
buffer.<br>
After one hour of blocking with PBS containing 3% milk (MPBS) at ambient<br>
temperature, concentrated supematants were incubated with serial dilutions (1:2, 1.20;<br>
1:200) and in parallel also with the supernatant of non transfected COS cells (negative<br>
control).<br>
After incubation with the primary antibody (which recognizes the constant Cy region of<br>
human  origin)   and  with  the  secondary   antibody   (anti-rabbit  conjugated  with<br>
peroxidase), it is possible to detect the binding activity as optical density at 450 run<br>
(OD450) by means of incubation with the TMB substrate (TECNA). The respective<br>
monoclonal antibodies at a concentration of 500 ng/ml were included as positive<br>
controls.<br>
Figure 10B shows the results of a similar ELISA assay conducted after purification by<br>
means of affinity chromatography of the expressed irnrnunoglobulins.<br>
In detail, the supematants of the transfected cells were collected and, after removing<br>
cellular detritus by centrifuging, they were incubated with  LOO /il of Protein G<br>
Sepharose and after extensive washings with PBS each protein was eluted with 1 mM<br>
HC1 and the pH was neutralized immediately after elution with 1 M Tris pH 8.8. The<br>
respective concentrations were estimated with Lowry assay. BIBLIOGRAPHY<br>
Adair et al. Hum.Antibod.Hybridomas 5:41(1994)<br>
Angeles et al. Anal Biochem; 236,49 (1996)<br>
Bacaef a/. J.Biol.Chem. 272:10678 (1997)<br>
Baker etal. "Antigen and Antibody Molecular Engineermg":61 (1994)<br>
Barinaga, Science; 264,272 (1994)<br>
Benhar et al. P.N.A.S. 91:12051 (1994)<br>
Bentley et al Nature; 348,254 (1990)<br>
Benvenuto et al. Plant Mol Biol; 17 865 (1991)<br>
Berardi et al. PNAS; 91, 684 (1994)<br>
Berman et al. Nucleic Acids Res.; 28, 235 (2000).<br><br>
Bird et al. Science; 242, 423 (1988)<br>
Bold etal. J. Neurochem.; 64, 2622 (1995)<br>
Buhler et al. Bio/Technology; 8,140 (1990)<br>
Buchner and Rudolph, Bio/Technology; 9, 157 (1991)<br>
Briinger^cta Cryst.; D54, 905 (1998)<br>
Carter et al. P.N.A.S. 89:4285 (1992)<br>
Canigo and Pongor Protein Science 10, 1470 (2001)<br>
Carugo, J. Appl. Cryst.; 36,125 (2003)<br>
Cattaneo et al. J. NeuroscL; 19, 9687 (1999)<br>
Cesareni, FEES Lett; 307, 66 (1992)<br>
Clackson et al. Nature; 352, 624 (1991) ' Chaudhary et al, Nature; 339, 394 (1989)<br>
Chen et al, Gene Ther; 2,116 (1995)<br>
Chothia &amp; Lesk,./. Mol Biol; 196, 901 (1987)<br>
Chothia et al Nature; 342, 878 (1989).<br>
Co et al. PNAS; 88, 2869 (1991)<br>
Co et al JJmmunol 148:1149-1154 (1992)<br>
Cook et al Prot.Engng. 9:623-628 (1996)<br>
Couto et al. "Antigen and Antibody Molecular Engineering" 55 (1994)<br>
Daugherty et al Nucleic Acid Res. 19:2471 (1991)<br>
De Martino et al. Antibod. Immunoconj. Radiopharmaceut. 4:829 (1991).<br>
Delsite et al. J. Androl; 17,481 (1996)<br>
De Schryver-Kecskemeti etal. Arch. Pathol; 111, 833 (1987)<br>
Diederichs, Proteins : 23 187 (1995)<br>
Djakiew et al. Cancer Res.; 51, 3304 (1991)<br>
Domenici etal. Vis NeuroscL; 11,1093 (1994)<br>
Durin et al. Plant Mol. Biol; 15,281 (1990)<br>
Eigenbrot et al Proteins 18: 49 (1994)<br>
Ellis et al. JJmmunol 155:925 (1995)<br>
Foote &amp; Winter J.Mol.Biol 224:487 (1992)<br>
Garrard et al Bio/Techniques; 9,1373 (1991)<br>
Gillman &amp; Smith, Gene; 8, 81 (1979)<br>
Goretzki et al. Siirgery; 102, 1035 (1987)<br>
Gorman et al PNAS; 88, 4181 (1991)<br>
Graziano et al JJmmunol. 155:4996 (1995)<br>
Gram et al PNAS; 89, 3576 (1992)<br>
Geldof et al. J. Cancer Res. Clin. Oncol.; 123,107 (1997)<br>
George et al. Tlie Prostrate; 36,172 (1998)<br>
Goding, Monoclonal antibodies: Principle and practise, 2nd edition, Academic Press<br>
(1986)<br>
Gussow &amp; SeemawiMeth.Enz)moL 203:99 (1991) Hakimi et al JJmmunol. 151:1075 (1993) Hamilton et al. JJnfect.Diseases 176:59 (1997)<br>
Harlow and Lane, Antibodies. A Laboratory Manual, Cold Spring Harbor Laboratory ' (1988)<br>
Hiatt et al Nature; 342, 76 (1989) Hood et al Immunology, Benjamin, N. Y., 2nj ed. (1984) Hsiao et al Prot.Engng. 7:815 (1994) Hunkapiller and Hood, Nature; 323,15 (1986) Huston et al PNAS; 85, 5879 (1988) Jancarik &amp; Kim, Appl Cryst.; 24,409 (1991) Jones et al Nature; 321, 522 (1986) Kashmiri et al Hybridoma 14:461 (1995) Kettleborough, Protein Engineering; 4, 773 (1991)<br>
Kabat, Sequences of Proteins of Immunological Merest (National Institutes of Health, Bethesda MD, 1987 e 1991) Kleywegt &amp; Jones, Structure; 2,1241 (1994). Kolbinger et al Prot.Engng. 6:971-980 (1993) Kodandapani et al. Biochem. Biophys., Res. Comm.; 251,61 (1998) Koizumi et al. Pathol Int.; 48, 93 (1998) Lachyankar et al Cancer Res; 57, 532 (1997) Lanzavecchia et al Eur. J. Immunol; 17,105 (1987) Larrick &amp; Fry, Hum. Antibodies Hybridomas; 2,172 (1991) Laskowski et al J. Appl Cryst.; 26, 283 (1993) Lefkovits e Perm's, Immunological Methods, Vols. I and JJ (Academic Press, NY, 1979<br>
1<br>
and 1981).<br>
Leger etal. Hum.Antibod. 8:3-16 (1997)<br>
Lewin &amp; Mendell, Trends Neurosci; 16, 353 (1993)<br>
Lewis &amp; Crowe Gene 101:297 (1991)<br>
Lindsay, Ciba Foundation Symposium; 196, 39, (1996)<br>
MacGrogan et al., J. Neurochem.; 59, 1381 (1992)<br>
Maeda et al. Hum.Antibod.Hybridomas 2:124 (1991)<br>
Maffei et al. J Neurosci; 12, 4651 (1992)<br>
Marchetti et al. Cancer Res.; 56, 2856 (1996)<br>
Matsusliima &amp; Bogenmann, Mol CellBiol; 13, 7447 (1993)<br>
Meade et al. Bio/Technology; 8, 443 (1990)<br>
McGregor et al. PNAS; 96,4540 (1999) "   Miknyoczki et al. Int. J. Cancer, 81, 417 (1999)<br>
Miknyoczki etal. Crit. Rev. Oncogenesis; 7, 89 (1996).<br>
Miralles etal J. Endocrinology; 156, 431 (1998)<br>
Molnar etal Eur J Neurosci; 10, 3127 (1998)<br>
Molnar et al. Neuroreport; 8, 575 (1997)<br>
Monoclonal Antibodies for Cancer Detection and Therapy (eds. Baldwin and Byers,<br>
Academic Press, 1985), 159,224<br>
Muragaki et al. J Neurosci; 17,530(1997)<br>
Nakagawara et al. NEnglJMed.; 328, 847 (1993)<br>
Navaza/lrtfl Cryst.; A50,157 (1994).<br>
Oermann et al. Cancer Res.; 55,2212 (1995)<br>
Ohta et al. J. Pathol; 181,405 (1997)<br>
Oikawa, et al. Int. J. Pancreat.; 18,15, (1995)<br>
Olsnes and Phil, Pharmac. There', 25, 355 (1982)<br>
Otwinowski &amp; Minor, Methods Enzymol; 276, 307 (1997)<br>
Paik et al. J. Nucl. Med.; 23, 37 (1982)<br>
Passaniti et al. Int. J. Cancer; 51,318 (1992)<br>
Paul, Fundamental Immunology, 3rd ed. Raven Press, N.Y., (1993)<br>
Pflug et al. Mol. Carcin.; 12, 106 (1998)<br>
Pflug et al. Endocrinology; 136, 262 (1995)<br>
Pflug et al. Cancer Res.; 52, 5403 (1992)<br>
Poul et al MoUmmunol. 32:101 (1995)<br>
Prestae/ al. J.lmmimol. 151:2623 (1993)<br>
Putlitz, et al. Bio/Technology; 8, 651 (1990)<br>
Queen et al. P.N.A.S. 86:10029 (1989)<br>
Remington's Pharmaceutical Science (15th ed., Mack Publishing Company, Easton, Pa.,<br>
1980)<br>
Revoltella &amp; Butler, J. Cell. PhysioL; 104,27 (1980)<br>
Riechmann et al, Nature; 332, 323 (1988)<br>
Roberts et al, Nature; 328, 731 (1987)<br>
Roguska et al Prot.Engng. 9:895(1996)<br>
Roguska et al. Prot.Engng. 9:895 (1996)<br>
Roguska et al PNAS, 91:969 (1994)<br>
Rosok et al J.Biol.Chem. 271:22611 (1996)<br>
Routledge et al Eur J.lmmimol. 21:2717 (1991)<br>
Ruberti et al JNeurosd.; 20, 2589 (2000)<br>
Ruggeri et al. Current Medicinal Chemistry; 6, 845 (1999)<br>
Sztoetal, Canc.Res. 53:851 (1993)<br>
Sato et al Mol.Immunol 31:371 (1994)<br>
Sato et al. Hum.Antibod.Hybridomas 7:175 (1996)<br>
Scopes, Protein Purification (Springer-Verlag, N.Y., 1982).<br>
Scott &amp; Smith, Science; 249,386 (1990)<br>
Sha and Xiang Canc.Biother. 9:341 (1994)<br>
Shearman et al J. Immunol; 147,4366 (1991)<br>
Skerra et al Bio/Technology; 9, 273 (1991)<br>
Sijmons et al. Bio/Technology; 8,217 (1990)<br>
Sims et al. J.Immunol. 151:2296 (1993)<br>
Singer et al J.Immunol 150:2844 (1993)<br>
Swimmer et al PNAS; 89, 3756 (1992)<br>
Tagliabue et al J. Biol Chem.; 275, 5388 (2000)<br>
Tempest et al. Prot.Engng. 7:1501 (1994)<br>
Tempest et al. Int.J.Biol.Macromol. 17:37 (1995)<br>
Tempest et al Biotechnology; 9,266 (1991)<br>
Thorpe et al. Monoclonal Antibodies in Clinical Medicine, Academic Press, 168 (1982)<br>
U.S. Ser. No. 07/290,968<br>
Verhoeyen et al Science 239:1534 (1988)<br>
Verhoeyen et al. "Monoclonal Antibodies" :37 (1991)<br>
Verhoeyen et al. Immunol 78:364 (1993)<br>
Winnacker, From Genes to Clones (VCH Publishers, NY, 1987)<br>
Winter and Milstein, Nature; 349, 293 (1991)<br>
Woolf et al, Neuroscience; 62, 327 (1994).<br>
WO91/08216, 1991<br>
Zhuetal.,J. din. Oncol, 17,2419(1999)<br>
Zhu and Carter J.Immunol. 155:1903 (1995)<br><br><br><br><br><br><br><br>
We Claim:<br>
1.	Method for the humanization of the VH and VL variable regions of an animal antibody of<br>
known sequence, comprising the steps of:<br>
a)	if not available, obtaining the crystallographic structure of the VH and VL regions of the animal antibody;<br>
b)	pre-selecting a series of 0 to n possible frameworks acceptors of human origin or humanized antibodies, whose structure was determined experimentally with a resolution no greater than 2.5 A, based on the highest level of homology and identity with the primary sequence of the framework of the animal antibody;<br>
c)	conducting a structural comparison between the VH and VL variable regions of the animal antibody and the regions VH and VL obtained in b), respectively and calculating for each comparison the root mean square deviation (RMS), to identify the region VH and the region VL of human origin with the smaller RMS;<br>
d)	inserting in appropriate position the sequences of the CDR regions of the animal antibody in the human sequences identified in c);<br>
e)	if necessary, retromutate one or more amino acid residues of the human VH and VL regions identified in c).<br><br>
2.	Method as claimed in claim 1, wherein the animal antibody is an anti-TrkA antibody.<br>
3.	Method as claimed in claim 2, wherein the animal anti-TrkA antibody is the MNAC13 antibody.<br>
4.	Method as claimed in claim 3, wherein the humanized VH and VL variable sequences have the following sequences: VH: HumMNAC13VH (SEQ ID No. 37), and VL: Hum MNAC13Vk (SEQ ID No. 38).<br>
5.	An anti-TrkA humanized animal antibody obtainable according to the method as claimed in claims 2 to 4.<br>
6.	The anti-TrkA humanized animal antibody as claimed in claim 2 comprising the sequences SEQ ID No. 37 and SEQ ID No. 38.</i></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUFic3RyYWN0LSgxMS0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Abstract-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1ERUxOUC0yMDA2LUFic3RyYWN0LSgxOC0xMS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-DELNP-2006-Abstract-(18-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUNsYWltcy0oMTEtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Claims-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1ERUxOUC0yMDA2LUNsYWltcy0oMTgtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-DELNP-2006-Claims-(18-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTEtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Correspondence Others-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTgtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-DELNP-2006-Correspondence Others-(18-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Correspondence Others-(24-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDExLTA3LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Description (Complete)-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LURyYXdpbmdzLSgxMS0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Drawings-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUZvcm0tMy0oMTEtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Form-3-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUZvcm0tMy0oMjQtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Form-3-(24-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LUdQQS0oMTEtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-GPA-(11-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwMi1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMTEtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3602-delnp-2006-Petition-137-(11-07-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="251966-a-computer-implemented-method-of-increasing-glue-logic-distribution-efficiency.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="251968-a-fused-heterocycle-derivative-and-pharmaceutical-composition-comprising-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>251967</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3602/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>16/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>20-Apr-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>LAY LINE GENOMICS S.P.A.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>VIA DI CASTEL ROMANO, 100 00128 ROMA, ITALY</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ANTONINO CATTANEO</td>
											<td>C/O LAY LINE GENOMICS S.P.A., VIA DI CASTEL ROMANO, 100 00128 ROMA, ITALY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DORIANO LAMBA</td>
											<td>C/O LAY LINE GENOMICS S.P.A. VIA DI CASTEL ROMANO, 100 00128, ITALY</td>
										</tr>
										<tr>
											<td>3</td>
											<td>SONIA COVASEUSZACH</td>
											<td>C/O LAY LINE GENOMICS S.P.A. VIA DI CASTEL ROMANO, 100 00128, ITALY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IT2004/000722</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-12-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>RM2003000601</td>
									<td>2003-12-24</td>
								    <td>Italy</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/251967-method-for-the-humanization-of-antibodies-and-humanized-antibodies-thereby-obtained by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:37:45 GMT -->
</html>
