<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252151-method-of-making-plants-having-increased-seed-yield-and-growth-rate by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252151:METHOD OF MAKING PLANTS HAVING INCREASED SEED YIELD AND GROWTH RATE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD OF MAKING PLANTS HAVING INCREASED SEED YIELD AND GROWTH RATE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention concerns a method for improving the growth characteristics of plants by introducing and expressing in a plant a nucleic acid encoding an S3a protein. The invention also relates to transgenic plants having introduced therein a nucleic acid encoding an S3a protein, which plants have improved growth characteristics relative to corresponding wild type plants. The present invention also concerns constructs useful in the methods of the invention.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Plants having improved growth characteristics and method for<br>
making the same<br>
The present invention relates generally to the field of molecular biology and concerns a method for improving plant growth characteristics. More specifically, the present invention concerns a method for improving plant growth characteristics, particularly yield, by introducing and expressing in a plant a nucleic acid encoding a small subunit ribosomal protein (S3a). The present invention also concerns plants having introduced therein an S3a-encoding nucleic acid, which plants have improved growth characteristics relative to corresponding wild type plants.<br>
The ever-increasing world population and the dwindling supply of arable land available for agriculture fuel agricultural research towards improving the efficiency of agriculture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogeneous genetic components that may not always result in the desirable trait being passed on from parent plants. Advances in molecular biology have allowed mankind to modify the germplasm of animals and plants. Genetic engineering of plants entails the isolation and manipulation of genetic material (typically in the form of DNA or RNA) and the subsequent introduction of that genetic material into a plant Such technology has the capacity to deliver crops or plants having various improved economic, agronomic or horticultural traits. A trait of particular economic interest is yield. Yield is normally defined as the measurable produce of economic value from a crop. This may be defined in terms of quantity and/or quality. Yield is directly dependent on several factors, for example, the number and size of the organs, plant architecture (for example, the number of branches), seed production and more. Root development, nutrient uptake and stress tolerance are also important factors in determining yield. Crop yield may be increased by optimizing one of the abovementioned factors.<br>
An outline of the ribosome assembly pathway emerged from sucrose gradient analyses in the early 1970s. These studies identified three major pre-ribosomal particles. An early 90S particle ('S1 being the rate of sedimentation) is subsequently processed into 66S and 43S pre-ribosomes, the precursors to the mature 60S and 40S subunits, respectively. The 60S subunit, after processing, is composed of 25S, 5.8S and 5S ribosomal RIMA. A number of proteins will have interacted along the maturation process to yield the final structure. The 4OS subunit is composed of an 18S ribosome RNA, and again a number of proteins will have<br><br>
interacted before the mature complex is ready. These two subunits assemble into a large complex in which the small subunit 40S binds the mRNA and the tRNAs, while the large subunit catalyzes peptide bond formation. About 2/3 of the mass of the ribosome consists of RNA and 1/3 of protein. The proteins are named in accordance with the subunit of the ribosome to which they belong: the small subunit (S1 to S31) and the large subunit (L1 to L44). Most of the proteins interact with multiple RNA elements, often from different domains, to organize and stabilize the rRNA tertiary structure. While the crucial activities of decoding and peptide transfer are RNA based processes, proteins play an active role in functions that may have evolved to streamline the process of protein synthesis. In addition to their typical ribosomal functions, many ribosomal proteins also have non-ribosomal functions, such asDNA repair.<br>
S3a (cycO7) was first cloned in 1991. It encodes a 40S ribosomal protein (small subunit) and is homologous to the v-fos transformation effector (encoded by the fte-1 gene in humans and rats), the TNF-alpha-induced TU-11 gene in mice, and is similar to the yeast PLC1 and PLC2. The expressions of mammalian fte-1, plant cyc-07 and yeast PLC2 have all been shown to be cell-cycle-regulated and involved in protein synthesis at the level of the ribosome.<br>
The ability to improve various growth characteristics of a plant, would have many applications in areas such as crop enhancement, plant breeding, in the production of ornamental plants, agriculture, horticulture, forestry, the production of algae for use in bioreactors (for the biotechnological production of substances such as Pharmaceuticals, antibodies, or vaccines, or for the by conversion of organic waste) and other such areas.<br>
It has now been found that introduction and expression in a plant of a nucleic acid encoding a small subunit ribosomal protein (S3a) gives plants having improved growth characteristics relative to corresponding wild type plants. Therefore according to one embodiment of the present invention there is provided a method for improving the growth characteristics of a plant, comprising introduction and expression in a plant of a nucleic acid encoding an S3a.<br>
Advantageously, performance of the methods according to the present invention result in plants having a variety of improved growth characteristics, especially increased yield, particularly seed yield.<br>
The term "increased yield" as defined herein is taken to mean an increase in any one or more of the following, each relative to corresponding wild type plants: (i) increased biomass (weight) of one or more parts of a plant, particularly aboveground (harvestable) parts, increased root<br><br>
biomass or increased biomass of any other harvestable part; (ii) increased seed yield, which may result from an increase in seed biomass (seed weight) and which may be an increase in the seed weight per plant or on an individual seed basis, and which increase in seed weight may be due to altered seed dimensions, such as seed length and/or seed width and/or seed area; (iii) increased number of (filled) seeds; (iv) increased seed size, which may also influence the composition of seeds; (v) increased seed volume, which may also influence the composition of seeds; (vi) increased harvest index, which is expressed as a ratio of the yield of harvestable parts, such as seeds, over the total biomass; and (vii) increased thousand kernel weight (TKW), which is extrapolated from the number of filled seeds counted and their total weight. An increased TKW may result from an increased seed size and/or seed weight.<br>
According to a preferred embodiment of the invention, the increase in yield encompasses an increase in yield on a seed level as defined in any one or more of (ii) to (vii) above.<br>
Taking corn as an example, a yield increase may be manifested as one or more of the following: increase in the number of plants per hectare or acre, an increase in the number of ears per plant, an increase in the number of rows, number of kernels per row, kernel weight, thousand kernel weight, ear length/diameter, among others. Taking rice as an example, a yield increase may be manifested by an increase in one or more of the following: number of plants per hectare or acre, number of panicles per plant, number of spikelets per panicle, number of flowers per panicle, increase in the seed filling rate, increase in thousand kernel weight, among others. An increase in yield may also result in modified architecture, or may occur as a result of modified architecture.<br>
According to a preferred feature of the present invention, performance of the methods of the invention result in plants having increased yield which is manifested by at least one of: increased TKW, increased number of (filled) seeds, increased seed weight and increased harvest index, each relative to control plants. Therefore, according to the present invention, there is provided a method for increasing plant yield, which method comprises introduction and expression in a plant of a nucleic acid encoding an S3a polypeptide.<br>
Since the transgenic plants according to the present invention have increased yield, it is likely that these plants exhibit an increased growth rate (during at least part of their life cycle), relative to the growth rate of corresponding wild type plants at a corresponding stage in their life cycle. The increased growth rate may be specific to one or more parts of a plant (including seeds), or may be throughout substantially the whole plant. A plant having an increased growth rate may even exhibit early flowering. The increase in growth rate may take place at<br><br>
one or more stages in the life cycle of a plant or during substantially the whole plant life cycle. Increased growth rate during the early stages in the life cycle of a plant may reflect enhanced vigour. The increase in growth rate may alter the harvest cycle of a plant allowing plants to be sown later and/or harvested sooner than would otherwise be possible. If the growth rate is sufficiently increased, it may allow for the sowing of further seeds of the same plant species (for example sowing and harvesting of rice plants followed by sowing and harvesting of further rice plants all within one conventional growing period). Similarly, if the growth rate is sufficiently increased, it may allow for the sowing of further seeds of different plants species (for example the sowing and harvesting of rice plants followed by, for example, the sowing and optional harvesting of soy bean, potatoes or any other suitable plant). Harvesting additional times from the same rootstock in the case of some plants may also be possible. Altering the harvest cycle of a plant may lead to an increase in annual biomass production per acre (due to an increase in the number of times (say in a year) that any particular plant may be grown and harvested). An increase in growth rate may also allow for the cultivation of transgenic plants in a wider geographical area than their wild-type counterparts, since the territorial limitations for growing a crop are often determined by adverse environmental conditions either at the time of planting (early season) or at the time of harvesting (late season). Such adverse conditions may be avoided if the harvest cycle is shortened. The growth rate may be determined by deriving various parameters from growth curves plotting growth experiments, such parameters may be: T-Mid (the time taken for plants to reach 50% of their maximal size) and T-90 (time taken for plants to reach 90% of their maximal size), amongst others.<br>
Performance of the methods of the invention gives plants having an increased growth rate. Therefore, according to the present invention, there is provided a method for increasing the growth rate of plants, which method comprises introduction and expression in a plant of a nucleic acid encoding an S3a polypeptide. An increase in growth rate is exemplified herein by an increase in TKW, increased number of (filled) seeds, increased seed weight and increased harvest index, each relative to control plants/corresponding wild-type plants.<br>
An increase in yield and/or growth rate occurs whether the plant is under non-stress conditions or whether the plant is exposed to various stresses compared to control plants. Plants typically respond to exposure to stress by growing more slowly. In conditions of severe stress, the plant may even stop growing altogether. Mild stress on the other hand is defined herein as being any stress to which a plant is exposed which does not result in the plant ceasing to grow altogether. Due to advances in agricultural practices (irrigation, fertilization, pesticide treatments) severe stresses are not often encountered in cultivated crop plants. As a consequence, the compromised growth induced by mild stress is often an undesirable feature<br><br>
for agriculture. Mild stresses are the typical stresses to which a plant may be exposed. These stresses may be the everyday biotic and/or abiotic (environmental) stresses to which a plant is exposed. Typical abiotic or environmental stresses include temperature stresses caused by atypical hot or cold/freezing temperatures; salt stress; water stress (drought or excess water). Abiotic stresses may also be caused by chemicals. Biotic stresses are typically those stresses caused by pathogens, such as bacteria, viruses, fungi and insects.<br>
The abovementioned growth characteristics may advantageously be modified in any plant.<br>
The term "plant" as used herein encompasses whole plants, ancestors and progeny of the plants and plant parts, including seeds, shoots, stems, leaves, roots (including tubers), and tissues and organs, wherein each of the aforementioned comprise the gene of interest. The term "plant" also encompasses embryos, meristematic regions, gametophytes, sporophytes, pollen, and microspores, again wherein each of the aforementioned comprise the gene of interest.<br>
Plants that are particularly useful in the methods of the invention include all plants which belong to the superfamily Viridiplantae, in particular monocotyledonous and dicotyledonous plants including fodder or forage legumes, ornamental plants, food crops, trees or shrubs selected from the list comprising Acacia spp.,Acer spp., Actinidia spp,, Aesculus spp., Agathis australis, Albizia amara, Alsophila tricolor, Andropogon spp., Arachis spp, Areca catechu, Astelia fragrans, Astragalus cicer, Baikiaea plurijuga, Betula spp., Brassica spp., Bruguiera gymnorrhiza, Burkea africana, Butea frondosa, Cadaba farinosa, Calliandra spp, Camellia sinensis, Canna indica, Capsicum spp., Cassia spp., Centroema pubescens, Chaenomeles spp., Cinnamomum cassia, Coffea arabica, Colophospermum mopane, Coronillia varia, Cotoneaster serotina, Crataegus spp., Cucumis spp., Cupressus spp., Cyathea dealbata, Cydonia oblonga, Cryptomeria japonica, Cymbopogon spp., Cynthea dealbata, Cydonia oblonga, Dalbergia monetaria, Davallia divaricate, Desmodium spp., Dicksonia squarosa, Diheteropogon amplectens, Dioclea spp, Dolichos spp., Dorycnium rectum, Echinochloa pyramidalis, Ehrartia spp., Eleusine coracana, Eragrestis spp., Erythrina spp., Eucalyptus spp., Euclea schimperi, Eulalia villosa, Fagopyrum spp., Feijoa sellowiana, Fragana spp., Flemingia spp, Freycinetia banksii, Geranium thunbergii, Ginkgo biloba, Glycine javanica, Gliricidia spp, Gossypium hirsutum, Grevillea spp., GuibourOa coleosperma, Hedysarum spp., Hemarthia altissima, Heteropogon contortus, Hordeum vulgare, Hyparrhenia rufa, Hypericum erectum, Hyperthelia dissolute, Indigo incamata, Iris spp., Leptarrhena pyrolifolia, Lespediza spp., Lettuca spp., Leucaena leucocephala, Loudetia simplex, Lotonus bainesii, Lotus spp., Macrotyloma   axillare,   Malus  spp.,   Manihot  esculenta,   Medicago  sativa,   Metasequoia<br><br>
glyptostmboides, Musa sapientum, Nicotianum spp., Onobrychis spp., Omithopus spp., Oryza spp.f Peltophorum africanum, Pennisetum spp., Persea gratissima, Petunia spp., Phaseolus spp-, Phoenix canariensis, Phormium cookianum, Photinia spp., Picea glauca, Pinus spp., Pisum sativum, Podocarpus totara, Pogonarthria fleckii, Pogonarthria squarrosa, Populus spp., Prosopis cineraria, Pseudotsuga menziesii, Ptemiobium stellatum, Pyrus communis, Quercus spp., Rhaphiolepsis umbellata, Rhopalostylis sapida, Rhus natalensis, Ribes gmssularia, Ribes spp., Robinia pseudoacacia, Rosa spp., Rubus spp., Salix spp., Schyzachyrium sanguineum, Sciadopitys verticillata, Sequoia sempervirens, Sequoiadendron giganteum, Sorghum bicolor, Spinacia spp., Sporobolus fimbriatus, Stiburus alopecurvides, Stylosanthos humilis, Tadehagi spp, Taxodium distichum, Themeda triandra, Trifolium spp., Triticum spp., Tsuga heterophylla, Vaccinium spp., Vicia spp., Vitis vinitera, Watsonia pyramidata, Zantedeschia aethiopica, Zea mays, amaranth, artichoke, asparagus, broccoli, Brussels sprouts, cabbage, canola, carrot, cauliflower, celery, collard greens, flax, kale, lentil, oilseed rape, okra, onion, potato, rice, soybean, strawberry, sugar beet, sugar cane, sunflower, tomato, squash, tea and algae, amongst others. According to a preferred embodiment of the present invention, the plant is a crop plant such as soybean, sunflower, canola, alfalfa, rapeseed, cotton, tomato, potato or tobacco. Further preferably, the plant is a monocotyledonous plant, such as sugarcane. More preferably the plant is a cereal, such as rice, maize, wheat, barley, millet, rye, sorghum or oats.<br>
The term S3a is well known in the art Alternative names for S3a are given in Table 1 below taken from Naora and Naora (Immunology and Cell Biology (1999) 77,197-205).<br>
Table 1: Homologues of S3a and Nomenclature<br><br><br>
An S3a may easily be identified by aligning a query sequence (preferably a protein sequence) with known S3a protein sequences (see for example the alignments shown in Figures 1 and 2). The query sequence may be aligned (with known S3a sequences) using, for example, the VNTI AlignX multiple alignment program (InforMax, Bethesda, MD), with default settings for gap opening penalty of 10 and a gap extension of 0.05. Since S3a sequences are highly conserved, a person skilled in the art would readily be able to identify other S3a sequences by comparing any conserved regions of the query sequence against those of the known S3a sequences. These regions of high conservation are illustrated in Figure 2 by the three boxed regions. Therefore, a query sequence having corresponding regions of conservation would be identifiable as an S3a. S3as have also been shown to bind to eukaryotic initiation factors elF-2 and elF-3 (Westermann etal. (FEBS Letters Vol. 97, No. 1 (January 1979)).<br>
The term "S3a" as defined herein is taken to mean a protein, which when used in an alignment, such as the ones shown in Figures 1 or 2, aligns with known S3a protein sequences and comprises regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2. Reference herein to an S3a-encoding nucleic acid is to a nucleic acid encoding a protein, which when used in an alignment, such as the one shown in Figures 1 or 2, aligns with known S3a protein sequences and comprises regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2. S3as and S3a-encoding nucleic acids as defined above are useful in the methods of the invention.<br>
The nucleic acid to be introduced into a plant may be any S3a-encoding sequence as hereinbefore defined. Preferably the nucleic acid is as represented by SEQ ID NO: 1. The nucleic acid to be introduced into a plant is preferably operably linked to a constitutive promoter for over expression in a plant. The constitutive promoter is preferably a GOS2 promoter, further preferably a GOS2 promoter from rice. It should be clear that the applicability of the present invention is not restricted to the S3a represented by SEQ ID NO: 1, nor is the applicability of the invention restricted to expression of an S3a gene/nucleic acid when driven by a GOS2 promoter.<br>
According to a preferred aspect of the present invention, enhanced or increased expression of the S3a nucleic acid is envisaged. Methods for obtaining enhanced or increased expression of genes or gene products are well documented in the art and include, for example, over expression driven by a strong promoter, the use of transcription enhancers or translation enhancers.<br><br>
The nucleic acid encoding an S3a may be derived from any source. The nucleic acid/gene encoding an S3a may be isolated from a microbial source, such as bacteria, yeast or fungi, or from a plant, algae or animal (including human) source. This nucleic acid may be modified from its native form in composition and/or genomic environment through deliberate human manipulation. The nucleic acid is preferably a homologous nucleic add, i.e. a nucleic acid obtained from a plant, whether from the same plant species in which it is to be introduced or whether from a different plant species. The nucleic acid may be isolated from a dicotyledonous species, preferably from the family Brassicaceae, further preferably from Arabidopsis thaliana. More preferably, the S3a isolated from Arabidopsis thaliana is represented by SEQ ID NO: 1 and the amino acid sequence as represented by SEQ ID NO: 2.<br>
The sequence represented by SEQ ID NO: 1 depicts an S3a from Arabidopsis thaliana, with SEQ ID NO: 2 being the corresponding amino acid sequence. Advantageously, the applicability of the present invention is not restricted to use of an S3a from Arabidopsis thaliana, as represented by SEQ ID NO: 1. The methods according to the present invention may also be practised using any S3a or S3a-encoding sequence as defined hereinabove.<br>
The S3a or S3a-encoding nucleic acids useful in practising the method according to the invention may be:<br>
(i)        Portions of an S3a-encoding nucleic acid, preferably portions of an S3a-encoding<br>
nucleic acid as represented by SEQ ID NO: 1; (ii)       Sequences capable of hybridising to an S3a-encoding nucleic acid, preferably<br>
sequences capable of hybridising to an S3a-encoding nucleic acid as represented<br>
by SEQ ID NO: 1; (iii)      Alternative splice variants of an S3a-encoding nucleic acid, preferably alternative<br>
splice variants of an S3a-encoding nucleic acid as represented by SEQ ID NO: 1; (iv)      Allelic variants of an S3a-encoding nucleic acid, preferably allelic variants of an<br>
S3a-encoding nucleic acid represented by SEQ ID NO: 1; and (v)       Homologues and derivatives of S3a proteins/amino acids, preferably homologues<br>
and derivatives of an S3a protein represented by SEQ ID NO: 2.<br>
The aforementioned portions, hybridising sequences, splice variants and allelic variants, homologues and derivatives are those falling under the definition of an S3a or S3a-encoding nucleic acid as defined hereinabove, i.e. in the case of a protein being a sequence that aligns with known S3a protein sequences and that comprises regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2, and in the case of a nucleic acid<br><br>
encodes a protein that aligns with known S3a protein sequences and comprises regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
It will be apparent to a person skilled in the art that use of the full length S3a DNA sequence would not be a prerequisite to carrying out the methods according to the invention. The methods according to the invention may advantageously be practised using portions of the S3a-encoding DNA/nucleic acid, such as represented by SEQ ID NO: 1. A portion refers to a piece of DNA derived or prepared from an original (larger) DNA molecule. A portion may be prepared, for example, by making one or more deletions to the nucleic acid sequence of SEQ ID NO: 1 using techniques well known in the art. Portions suitable for use in performing the methods of the invention are those encoding S3a proteins that align with known S3a protein sequences and that comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
Therefore according to the invention, there is provided, a method for improving the growth characteristics of plants, comprising introduction and expression in a plant of a portion of an S3a-encoding nucleic acid, preferably a portion of a nucleic acid as represented by SEQ ID NO:1.<br>
Also useful in performing methods of the invention are nucleic acids capable of hybridising with an S3a-encoding nucleic acid, preferably nucleic acids capable of hybridising to an S3a-encoding nucleic acid sequence as represented by SEQ ID NO: 1. Such hybridising sequences are those encoding S3a proteins that align with known S3a protein sequences and that comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2<br>
The term "hybridisation" as defined herein is a process wherein substantially homologous complementary nucleotide sequences anneal to each other. The hybridisation process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, nuclease S1 mapping, primer extension, reverse transcription, cDNA synthesis, differential display of RNAs, and DNA sequence determination. The hybridisation process can also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process indude the isolation of poly (A+) mRNA. The hybridisation process can furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a nitro-cellulose or nylon<br><br>
membrane or immobilised by e.g. photolithography to, for example, a siliceous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process include RNA and DNA gel blot analysis, colony hybridisation, plaque hybridisation, in situ hybridisation and microarray hybridisation, in order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands and/or to remove hairpins or other secondary structures from single stranded nucleic acids. The stringency of hybridisation is influenced by conditions such as temperature, salt concentration and hybridisation buffer composition. Hybridisation preferably occurs under stringent conditions. Stringent conditions are those that (1) employ low ionic strength and high temperature for washing, for example, 0.5M sodium phosphate buffer pH 7.2,1mM EDTA pH 8.0 in 7% SDS at either 65°C or 55°C, or (2) employ during hybridization a denaturing agent such as formamide, for example, 50% (vol/vol) formamide with 0.1% bovine serum albumin, 0.1% Ficoll, 0.1% polyvinylpolypyrrolidone, 0.05 M sodium phosphate buffer at pH 6.5 with 0.75 M NaCI, 0.075 M sodium citrate at 42°C. A specific example includes the use of 50% formamide, 5XSSC (0.75 M NaCI, 0.075 M sodium citrate), 50 mM sodium phosphate (pH 6.8), 0.1% sodium pyrophosphate, 5X Denhard's solution, sonicated salmon sperm DNA (50nm/ml), 0.1% SDS and 10% dextran sulfate at 55°C, with washes at 55°C in 0.2XSSC and 0.1% SDS. A skilled person can readily determine and vary the stringency conditions appropriately to obtain a clear and detectable hybridization signal.<br>
Therefore according to the invention, there is provided, a method for improving the growth characteristics of plants, comprising introduction and expression in a plant of a sequence capable of hybridising preferably under stringent conditions to an S3a-encoding nucleic acid sequence, preferably to an S3a-encoding nucleic acid as represented by SEQ ID NO: 1.<br>
Also useful in performing the methods of the invention are alternative splice variants of an S3a~ encoding nucleic acid, preferably an alternative splice variant of an S3a-encoding nucleic acid as represented by the sequence of SEQ ID NO: 1. Splice variants suitable for use in performing the methods of the invention are those encoding S3a proteins that align with known S3a protein sequences and that comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2. The term "alternative splice variant" as used herein encompasses variants of a nucleic acid sequence in which selected introns and/or exons have been excised, replaced or added. Such variants will be ones in which the biological activity of the protein remains unaffected, which can be achieved by selectively retaining functional segments of the protein. Such splice variants may be found in nature or may be manmade. Methods for making such splice variants are well known in the art<br><br>
Therefore, the invention also provides a method for improving the growth characteristics of plants, comprising introduction and expression in a plant of an alternative splice variant of an S3a-encoding nucleic acid, preferably an alternative splice variant of an S3a-encoding nucleic acid as represented by SEQ ID NO: 1.<br>
Also useful in performing the methods of the invention are allelic variants of an S3a-encoding nucleic acid, preferably an allelic variant of an S3a-encoding nucleic acid as represented by SEQ ID NO: 1. Allelic variants suitable for use in performing the methods of the invention are those encoding S3a proteins that align with known S3a protein sequences and that comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
Allelic variants exist in nature and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants encompass Single Nucleotide Polymorphisms (SNPs), as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest set of sequence variants in naturally occurring polymorphic strains of most organisms.<br>
Therefore, the invention also provides a method for improving the growth characteristics of plants, comprising introduction and expression in a plant of an allelic variant of an S3a-encoding nucleic acid, preferably an allelic variant of an S3a-encoding nucleic acid as represented by SEQ ID NO: 1.<br>
Further advantageously, the methods according to the present invention may also be practised using homologues, derivatives or active fragments of an S3a, such as an S3a represented by SEQ ID NO: 2. Nucleic acids encoding homologues or derivatives of S3as may readily be determined using routine techniques well known to persons skilled in the art.<br>
"Homologues" of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological and functional activity to the unmodified protein from which they are derived. To produce such homologues, amino acids of the protein may be replaced by other amino acids having similar properties (such as similar hydrophobicity, hydrophilicity, antigenicity, propensity to form or break α.-helical structures or 15-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company).<br><br>
The homologues useful in the method according to the invention have in increasing order of preference at least 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95% sequence identity to the amino acid sequence as represented by SEQ ID NO: 2. Furthermore, the homologues useful in the methods of the invention are those that align with known S3a protein sequences and that comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
Also encompassed by the term "homologues" are two special forms of homology, which include orthologous sequences and paralogous sequences, which encompass evolutionary concepts used to describe ancestral relationships of genes. The term "paralogous" relates to gene-duplications within the genome of a species leading to paralogous genes. The term "orthologous" relates to homologous genes in different organisms due to ancestral relationship.<br>
Othoiogues in, for example, monocot plant species may easily be found by performing a so-called reciprocal blast search. This may be done by a first blast involving blasting the sequence in question (for example, SEQ ID NO: 1 or SEQ ID NO: 2) against any sequence database, such as the publicly available NCBI database which may be found at: http://www.ncbi.nlm.nih.gov. If orthologues in rice were sought, the sequence in question would be blasted against, for example, the 28,469 lull-length cDNA clones from Oryza sativa Nipponbare available at NCBI. BLASTn may be used when starting from nucleotides or TBLASTX when starting from the protein, with standard default values (expectation 10, alignment 50). The blast results may be filtered. The full-length sequences of either the filtered results or the non-filtered results are then blasted back (second blast) against the sequence in question (SEQ ID NO: 1 or 2). The results of the first and second blasts are then compared. In the case of large families, ClustalW is used followed by a neighbour joining tree to help visualize the clustering.<br>
A homologue may be in the form of a "substitutional variant" of a protein, i.e. where at least one residue in an amino acid sequence has been removed and a different residue inserted in its place. Amino acid substitutions are typically of single residues, but may be clustered depending upon functional constraints placed upon the polypeptide; insertions will usually be of the order of about 1 to 10 amino acid residues, and deletions will range from about 1 to 20 residues. Preferably, amino acid substitutions comprise conservative amino acid substitutions.<br>
A homologue may also be in the form of an "insertional varianf of a protein, i.e. where one or more amino acid residues are introduced into a predetermined site in a protein. Insertions may<br><br>
comprise amino-terminal and/or carboxy-terminal fusions as well as intra-sequence insertions of single or multiple amino acids. Generally, insertions within the amino acid sequence will be smaller than amino- or carboxy-terminal fusions, of the order of about 1 to 10 residues. Examples of amino- or carboxy-terminal fusion proteins or peptides include the binding domain or activation domain of a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins, (histidine)6-tag, glutathione S-transferase4ag, protein A, maltose-binding protein, dihydrofolate reductase, Tag-100 epitope, c-myc epitope, FLAG®-eprtope, lacZ, CMP (calmodulin-binding peptide), HA epitope, protein C epitope and VSV epitope.<br>
Homologues in the form of "deletion variants" of a protein are characterised by the removal of one or more amino acids from a protein.<br>
Amino acid variants of a protein may readily be made using peptide synthetic techniques well known in the art, such as solid phase peptide synthesis and the like, or by recornbinant DNA manipulations. Methods for the manipulation of DNA sequences to produce substitution, insertion or deletion variants of a protein are well known in the art. For example, techniques for making substitution mutations at predetermined sites in DNA are well known to those skilled in the art and include M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis or other site-directed mutagenesis protocols.<br>
"Derivatives" include peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise substitutions, deletions or additions of naturally and non-naturally occurring amino add residues compared to the amino acid sequence of a naturally-occurring form of the S3a protein, for example, as presented in SEQ ID NO: 2. "Derivatives" of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise naturally occurring altered, glycosylated, acylated or non-naturally occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of the polypeptide. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived, for example a reporter molecule or other ligand, covalently or non-covalently bound to the amino acid sequence, such as a reporter molecule which is bound to facilitate its detection, and non-naturally occurring amino acid residues relative to the amino acid sequence of a naturally-occurring protein.<br>
Methods for the search and identification of S3a homologues would be well within the realm of a person skilled in the art. Methods for the alignment of sequences for comparison are well known in the art, such methods include GAP, BESTFIT, BLAST, FASTA and TFASTA.  GAP<br><br>
uses the algorithm of Needleman and Wunsch (J. Mol. Biol. 48: 443-453, 1970) to find the alignment of two complete sequences that maximises the number of matches and minimises the number of gaps. The BLAST algorithm calculates percent sequence identity and performs a statistical analysis of the similarity between the two sequences. The software for performing BLAST analysis is publicly available through the National Centre for Biotechnology Information. Homologues suitable for use in the methods of the invention, i.e. those having at least 55% sequence identity to the amino acid sequence represented by SEQ ID NO: 2, may be identified by taking full length protein sequences and aligning them using the VNTI AlignX multiple alignment program (InforMax, Bethesda, MD), with default settings for gap opening penalty of 10 and a gap extension of 0.05. See for example Figure 1.<br>
Therefore, the invention also provides a method for improving the growth characteristics of plants, comprising introducing and expressing in a plant a nucleic acid encoding a homologue or derivative of an S3a as represented by SEQ ID NO: 2, which homologue, derivative or active fragment has in increasing order of preference at least 55%, 60%, 65%, 70%, 75%, 80%, 85%, 90%, 95% sequence identity to an amino acid sequence as represented by SEQ ID NO: 2 and which homologue or derivative aligns with known S3a protein sequences and comprises regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
The invention also provides genetic constructs and vectors to facilitate introduction and/or expression of the nudeotide sequences useful in the methods according to the invention.<br>
Therefore, there is provided a gene construct comprising:<br>
(i)       An S3a-encoding nucleic acid, preferably as represented by SEQ ID NO: 1; (ii)       one or more control sequences capable of driving expression of the nucleic acid sequence of (i); and optionally a transcription termination sequence.<br>
Constructs useful in the methods according to the present invention may be constructed using recombinant DNA technology well known to persons skilled in the art. The gene constructs may be inserted into vectors, which may be commercially available, suitable for transforming into plants and suitable for expression of the gene of interest in the transformed cells.<br>
Plants are transformed with a gene construct comprising the sequence of interest (i.e., an S3a-encoding nucleic acid as defined hereinabove. The sequence of interest is operabty linked to one or more control sequences (at least to a promoter).   The terms Regulatory element",<br><br>
"control sequence" and "promoter" are all used interchangeably herein and are to be taken in a broad context to refer to regulatory nucleic acid sequences capable of effecting expression of the sequences to which they are legated. Encompassed by the aforementioned terms are transcriptional regulatory sequences derived from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific manner. Also included within the term is a transcriptional regulatory sequence of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. The term "regulatory element" also encompasses a synthetic fusion molecule or derivative which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ. The term "operably linked” as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiate transcription of the gene of interest.<br>
Advantageously, any type of promoter may be used to drive expression of the S3a-encoding nucleic acid.<br>
Preferably, the nucleic acid encoding an S3a is operably linked to a constitutive promoter. The term "constitutive" as defined herein refers to a promoter that is expressed predominantly in more than one tissue or organ and predominantly at any stage in the life cycle of a plant. Preferably, the constitutive promoter is expressed predominantly throughout the plant. Preferably, the constitutive promoter is the GOS2 promoter from rice.<br>
Examples of other constitutive promoters suitable for use in the methods of the invention are listed in Table A below.<br><br><br><br>
Optionally, one or more terminator sequences may also be used in the construct introduced into a plant The term "terminator"  encompasses a control sequence which is a DNA sequence at the end of a transcriptional unit which signals 3' processing and polyadenylation  of a primary transcript and termination of transcription. Additional regulatory elements may include transcriptional as well as translational enhancers. Those skilled in the art will be aware of terminator and enhancer sequences which may be suitable for use in performing the invention. Such sequences would be known or may readily be obtained by a person skilled in<br>
the art.<br>
i<br>
The genetic constructs of the invention may further include an origin of replication sequence which is required for maintenance and/or replication in a specific cell type. One example is when a genetic construct is required to be maintained in a bacterial cell as an episomal genetic element (e.g. plasmid or cosmid molecule). Preferred origins of replication include, but are not limited to, the f1-ori and colEl<br>
The genetic construct may optionally comprise a selectable marker gene. As used herein, the term "selectable marker gene" includes any gene which confers a phenotype on a cell in which it is expressed to facilitate the identification and/or selection of cells which are transfected or transformed with a nucleic acid construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance, that introduce a new metabolic tract or that allow visual selection. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as nptll that phosphorylates neomycin and kanamycin, or hpt, phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as man A that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example p-glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof).<br><br>
The present invention also encompasses plants obtainable by the methods according to the present invention. The present invention therefore provides plants obtainable by the method according to the present invention, which plants have introduced and expressed therein a nucleic acid encoding an S3a protein, as defined hereinabove.<br>
The invention also provides a method for the production of transgenic plants having improved growth characteristics, comprising introduction and expression in a plant of an S3a-encoding nucleic acid, preferably as represented by SEQ ID NO: 1.<br>
More specifically, the present invention provides a method for the production of transgenic plants having improved growth characteristics, which method comprises:<br>
(i)        introducing into a plant or plant cell an S3a-encoding nucleic acid, preferably as<br>
represented by SEQ ID NO: 1;<br>
(ii)       cultivating the plant cell under conditions promoting regeneration and mature plant growth.<br>
The nucleic acid may be introduced directly into a plant cell or into the plant itself (including introduction into a tissue, organ or any other part of a plant). According to a preferred feature of the present invention, the nucleic acid is preferably introduced into a plant by transformation.<br>
The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucleotide into a host cell, irrespective of the method used for transfer. Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention and a whole plant regenerated therefrom. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem). The polynucleotide may be transiently or stably introduced into a host cell and may be maintained non-integrated, for example, as a plasmid- Alternatively, it may be integrated into the host genome. The resulting transformed plant cell may then be used to regenerate a transformed plant in a manner known to persons skilled in the art.<br>
Transformation of plant species is now a fairly routine technique. Advantageously, any of several transformation methods may be used to introduce the gene of interest into a suitable<br><br>
ancestor cell. Transformation methods include the use of liposomes, electroporation, chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle gun bombardment, transformation using viruses or pollen and microprojection. Methods may be selected from the calcium/polyethylene glycol method for protoplasts (Krens, FA. et al., 1882, Nature 296, 72-74; Negrutiu I. et al, June 1987, Plant Mol. Biol. 8, 363-373); electroporation of protoplasts (Shillito R.D. et al., 1985 Bio/Technol 3, 1099-1102); microinjection into plant material (Crossway A. et al., 1986, Mol. Gen Genet 202, 179-185); DNA or RNA-coated particle bombardment (Klein T.M. et al., 1987, Nature 327, 70) infection with (non-integrative) viruses and the like. Transgenic rice plants expressing an S3a are preferably produced via Agrobacterium-medlated transformation using any of the well known methods for rice transformation, such as described in any of the following: published European patent application EP 1198985 A1, Aldemita and Hodges (Planta, 199, 612-617, 1996); Chan et aL (Plant Mol. Biol. 22 (3) 491-506, 1993), Hiei et aL (Plant J. 6 (2) 271-282, 1994), which disclosures are incorporated by reference herein as if fully set forth. In the case of corn transformation, the preferred method is as described in either Ishida et al. (Nat Biotechnol. 1996 Jun; 14(6): 745-50) or Frame et aL (Plant Physiol. 2002 May; 129(1): 13-22), which disclosures are incorporated by reference herein as if fully set forth.<br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers which are encoded by plant-expressible genes co-transferred with the gene of interest, following which the transformed material is regenerated into a whole plant.<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organisation. Alternatively or additionally, expression levels of the newly introduced DNA may be monitored using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the art.<br>
The generated transformed plants may be propagated by a variety of means, such as by donal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give homozygous second generation (or T2) transformants, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transformants (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootstock grafted to an untransformed scion).<br><br>
The present invention clearly extends to any plant cell or plant produced by any of the methods described herein, and to all plant parts and propagules thereof. The present invention extends further to encompass the progeny of a primary transformed or transfected cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the only requirement being that progeny exhibit the same genotypic and/or phenotypic characteristic(s) as those produced in the parent by the methods according to the invention. The invention also includes host cells containing an isolated S3a-encoding nucleic acid. Preferred host cells according to the invention are plant cells. The invention also extends to harvestable parts of a plant such as but not limited to seeds, leaves, fruits, flowers, stem cultures, rhizomes, tubers and bulbs.<br>
The methods according to the invention may also be performed without introducing a nucleic acid encoding an S3a into a plant. This may be achieved by introducing a genetic modification (preferably in the locus of an S3a-encoding gene). The locus of a gene as defined herein is taken to mean a genomic region, which includes the gene of interest and 10KB up- or down stream of the coding region.<br>
The genetic modification may be introduced, for example, by any one (or more) of the following methods: TDNA activation, TILLING, site-directed mutagenesis, homologous recombination, directed evolution, or as discussed hereinabove, by introducing and expressing in a plant (cell) an S3a-encoding nucleic add.<br>
T-DNA activation tagging (Hayashi et al Science (1992) 1350-1353) involves insertion of T-DNA usually containing a promoter (may also be a translation enhancer or an intron), in the genomic region of the gene of interest or 10KB up- or down stream of the coding region of a gene in a configuration such that the promoter directs expression of the targeted gene. Typically, regulation of expression of the targeted gene by its natural promoter is disrupted and the gene falls under the control of the newly introduced promoter. The promoter is typically embedded in a T-DNA. This T-DNA is randomly inserted into the plant genome, for example, through Agrobacterium infection and leads to overexpression of genes near to the inserted T-DNA. The resulting transgenic plants show dominant phenotypes due to overexpression of genes close to the introduced promoter. The promoter to be introduced may be any promoter capable of directing expression of a gene in the desired organism, in this case a plant. For example, constitutive, tissue-preferred, cell type-preferred and inducible promoters are all suitable for use in T-DNA activation.<br><br>
A genetic modification may also be introduced in the locus of an S3a-encoding nucleic acid/gene using the technique of TILLING (Targeted Induced Local Lesions IN Genomes). This is a mutagenesis technology useful to generate and/or identify, and to eventually isolate mutagenised variants of an S3a-encoding nucleic acid capable of exhibiting S3a biological activity. TILLING also allows selection of plants carrying such mutant variants. These mutant variants may even exhibit higher S3a activity than that exhibited by the gene in its natural form. TILLNG combines high-density mutagenesis with high-throughput screening methods. The steps typically followed in TILLING are: (a) EMS mutagenesis (Redei and Koncz, 1992; Feldmann et a/., 1994; Lightner and Caspar, 1998); (b) DNA preparation and pooling of individuals; (c) PCR amplification of a region of interest; (d) denaturation and annealing to allow formation of heteroduplexes; (e) DHPLC, where the presence of a heteroduplex in a pool is detected as an extra peak in the chromatogram; (f) identification of the mutant individual; and (g) sequencing of the mutant PCR product. Methods for TILLING are well known in the art (McCallum Nat Biotechnol. 2000 Apr; 18(4):455-7, reviewed by Stemple 2004 (TILLlNG-a high-throughput harvest for functional genomics. Nat Rev Genet 2004 Feb;5(2):145-50.)).<br>
Site directed mutagenesis may be used to generate variants of S3a-encoding nucleic acids or portions thereof. Several methods are available to achieve site directed mutagenesis, the most common being PCR based methods (current protocols in molecular biology. Wiley Eds. httpi/www.4ulr.com/products/currentprotocols/index.html).<br>
Directed evolution may also be used to generate variants of S3a-encoding nucleic acids. This consists of iterations of DNA shuffling followed by appropriate screening and/or selection to generate variants of S3a-encoding nucleic acids (Castle et a/., (2004) Science 304(5674): 1151-4; US patents 5,811,238 and 6,395,547).<br>
T-DNA activation, TILLING, site-directed mutagenesis and directed evolution are examples of technologies that enable the generation of novel alleles and SYT variants.<br>
Homologous recombination allows introduction in a genome of a selected nucleic acid at a defined selected position. Homologous recombination is a standard technology used routinely in biological sciences for lower organisms such as yeast or the moss physcomitrella. Methods for performing homologous recombination in plants have been described not only for model plants (Offringa et a/. Extrachromosomal homologous recombination and gene targeting in plant cells after Agrobacterium-mediated transformation. 1990 EMBO J. 1990 Oct; 9(10):3077-84) but also for crop plants, for example rice (Terada R, Urawa H, Inagaki Y, Tsugane K, lida S. Efficient gene targeting by homologous recombination in rice. Nat Biotechnol. 2002. lida and<br><br>
Terada: A tale of two integrations, transgene and T-DNA: gene targeting by homologous recombination in rice. Curr Opin Biotechnol. 2004 Apr; 15(2): 132-8). The nucleic acid to be targeted (which may be an S3a-encoding nucleic acid as hereinbefore defined) need not be targeted to the locus of an S3a -encoding gene, but may be introduced in, for example, regions of high expression. The nucleic acid to be targeted may be an improved allele used to replace the endogenous gene or may be introduced in addition to the endogenous gene.<br>
The present invention also encompasses the use of nucleic acids encoding S3as and the use of S3a polypeptides.<br>
One such use of course relates to the use of an S3a as defined hereinabove in improving the growth characteristics of plants, in particular in improving yield, especially seed yield. The seed yield may include one or more of the following: increased number of (filled) seeds, increased seed weight, increased harvest index and increased TKW, among others. The S3a-encoding nucleic acid may be as represented by SEQ ID NO: 1 and the S3a protein may be an amino acid as represented by SEQ ID NO: 2.<br>
Nucleic acids encoding S3as, and S3a polypeptides, as hereinbefore defined may also find use in breeding programmes. The S3a-encoding nucleic acid may be as represented by SEQ ID NO: 1; or the S3a may be an amino acid as represented by SEQ ID NO: 2. For example, the S3a-encoding nucleic acid or a part thereof may be on a chromosome (or a part thereof), preferably together with one or more related family members. In an example of such a breeding programme, a DNA marker is identified which may be genetically linked to a gene capable of modulating expression of a nucleic acid encoding an S3a protein in a plant, which gene may be a gene encoding the S3a protein itself or any other gene which may directly or indirectly influence expression of a gene encoding an S3a protein and/or activity of the S3a protein itself. This DNA marker may then used in breeding programs to select plants having improved growth characteristics.<br>
Alielic variants of an S3a may also be used in conventional breeding programmes, such* as in marker-assisted breeding. Such breeding programmes sometimes require the introduction of alielic variations in the plants by mutagenic treatment of a plant One suitable mutagenic method is EMS mutagenesis. Identification of allelic variants then takes place by, for example, PCR. This is followed by a selection step for selection of superior allelic variants of the sequence in question and which give improved growth characteristics in a plant. Selection is typically carried out by monitoring growth performance of plants containing different alielic variants of the sequence in question, for example, different alleiic variants of SEQ ID MO: 1.<br><br>
Monitoring growth performance can be done in a greenhouse or in the field. Further optional steps include crossing plants, in which the superior allelic variant was identified, with another plant. This could be used, for example, to make a combination of interesting phenotypic features.<br>
An S3a-encoding nucleic acid may also be used as probes for genetically and physically mapping the genes that they are a part of, and as markers for traits linked to those genes. Such information may be useful in plant breeding in order to develop lines with desired phenotypes. Such use of an S3a-encoding nucleic acid requires only a nucleic acid sequence of at least 15 nucleotides in length. The S3a-encoding nucleic acid may be used as restriction fragment length polymorphism (RFLP) markers. Southern blots (Maniatis) of restriction-digested plant genomic DNA may be probed with the S3a-encoding nucleic acid. The resulting banding patterns may then be subjected to genetic analyses using computer programs such as MapMaker (Lander et al. (1987) Genomics 1:174-181) in order to construct a genetic map. In addition, the nucleic acids may be used to probe Southern blots containing restriction endonuclease-treated genomic DNAs of a set of individuals representing parent and progeny of a defined genetic cross. Segregation of the DNA polymorphisms is noted and used to calculate the position of the S3a-encoding nucleic acid in the genetic map previously obtained using this population (Botstein et al. (1980) Am. J. Hum. Genet. 32:314-331).<br>
The production and use of plant gene-derived probes for use in genetic mapping is described in Bematzky and Tanksley (1986) Plant Mol. Biol. Reporter 4:37-41. Numerous publications describe genetic mapping of specific cDNA clones using the methodology outlined above or variations thereof. For example, F2 intercross populations, backcross populations, randomly mated populations, near isogenic lines, and other sets of individuals may be used for mapping. Such methodologies are well known to those skilled in the art.<br>
The nucleic acid probes may also be used for physical mapping (i.e., placement of sequences on physical maps; see Hoheisel et a/. In: Non-mammalian Genomic Analysis: A Practical Guide, Academic press 1996, pp. 319-346, and references cited therein).<br>
In another embodiment, the nucleic acid probes may be used in direct fluorescence in situ hybridization (FISH) mapping (Trask (1991) Trends Genet. 7:149-154). Although current methods of FISH mapping favor use of large clones (several to several hundred KB; see Laan et aL (1995) Genome Res. 5:13-20), improvements in sensitivity may allow performance of FISH mapping using shorter probes.<br><br>
A variety of nucleic acid amplification-based methods for genetic and physical mapping may be carried out using the nucleic acids. Examples include allele-specific amplification (Kazazian (1989) J. Lab. Clin. Med 11:95-96), polymorphism of PCR-amplified fragments (CAPS; Sheffield et al. (1993) Genomics 16:325-332), allele-specific ligation (Landegren et al. (1988) Science 241:1077-1080), nucleotide extension reactions (Sokolov (1990) Nucleic Acid Res. 18:3671), Radiation Hybrid Mapping (Walter et al. (1997) Nat Genet. 7:22-28) and Happy Mapping (Dear and Cook (1989) Nucleic Acid Res. 17:6795-6807). For these methods, the sequence of a nucleic acid is used to design and produce primer pairs for use in the amplification reaction or in primer extension reactions. The design of such primers is well known to those skilled in the art. In methods employing PCR-based genetic mapping, it may be necessary to identify DNA sequence differences between the parents of the mapping cross in the region corresponding to the instant nucleic acid sequence. This, however, is generally not necessary for mapping methods.<br>
Nucleic acids encoding S3as and S3a polypeptides may also find use as growth regulators. The S3a-encoding nucleic acid may be a nucleic acid as represented by SEQ ID NO: 1 and the S3a protein/polypeptide may be an amino acid as represented by SEQ ID NO: 2. Since these S3as are useful in improving the growth characteristics of plants, the S3as would also be useful growth regulators, such as herbicides or growth stimulators. The present invention therefore provides a composition comprising an S3a or S3a-encoding nucleic acid, together with a suitable carrier, diluent or excipient, for use as a growth regulator.<br>
The methods according to the present invention result in plants having improved growth characteristics, as described hereinbefore. These improved growth characteristics may also be combined with other economically advantageous traits, such as further yield-enhancing traits, tolerance to various stresses, traits modifying various architectural features and/or biochemical and/or physiological features.<br>
Description of figures<br>
The present invention will now be described with reference to the following figures in which:<br>
Fig. 1 shows an alignment of S3a protein sequences made using the VNTI AlignX multiple alignment program (InforMax, Bethesda, MD), with default settings for gap opening penalty of 10 and a gap extension of 0.05. Residues appearing in white against a black background are identical; residues appearing in white against a gray background are either conservative or block of similar residues, as defined by VNTI options.<br><br>
Fig. 2 shows an alignment taken from Lyamouri etal. (Gene 294 (2002) 147-156) which shows the high degree of conservation in S3a proteins from various species: H. sapiens (Hs), M. musculus (Mm), 7. rubripes (Tr), X laevisQQ), D. melanogaster (Dm), D. virilis(Dv), C. elegans (Ce), S. cerevisiae (Sc), A. thaliana (At), O. safoVa (Os) and H. holobium (Hh). Black shading represents identical amino acids, while light grey represents conserved amino acid substitutions. Gaps are indicated by dashes. Regions of highest conservation are boxed.<br>
Fig. 3 shows a binary vector for expression in Oryza sativa of the Arabidopsis thaliana cycO7putative S-phase-specific 40S S3a ribosomal protein gene (internal reference CDS0730) under the control of the rice GOS2 promoter (internal reference PROO 129).<br>
Fig. 4 details examples of sequences useful in performing the methods according to the present invention.<br>
Examples<br>
The present invention will now be described with reference to the following examples, which are by way of illustration alone.<br>
DNA manipulation: unless otherwise stated, recombinant DNA techniques are performed according to standard protocols described in (Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York) or in Volumes 1 and 2 of Ausubel ef a/. (1994), Current Protocols in Molecular Biology, Current Protocols. Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfase (1993) by R.D.D. Cray, published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications (UK).<br>
Example 1: Gene Cloning<br>
The Arabidopsis S3a (S-phase specific 40S cycO7 was amplified by PCR using as template an Arabidopsis thaliana seedling cDNA library (Invitrogen, Paisley, UK). After reverse transcription of RNA extracted from seedlings, the cDNAs were cloned into pCMV Sport 6.0. Average insert size of the bank was 1.5 kb, and original number of do nes was of 1.59x107 cfu. The original titer was determined to be 9.6x105 cfu/ml, and became after a first amplification 6x1011 cfu/ml. After plasmid extraction, 200 ng of template was used in a 50 yl PCR mix. Primers prmO2255 (sense, start codon in bold, AttB1 site in italic: 5' GGGGACAAGTTTGTACAAAAAAGCAGGCTTCACAATGGCTGTCG&amp;GAAGAA 31) and prmO2256   (reverse,   complementary,   stop   codon   in   bold,   At*B2   site   in   italic:   5f<br><br>
GGGGACCACTrrGTACAAGAAAGCTGGGmCTAAGCTCCGATGATTTCT 3'), which include the AttB sites for Gateway recombination, were used for PCR amplification. PCR was performed using Hifi Taq DNA polymerase under standard conditions. A PCR fragment of 789 bp was amplified and purified also using standard methods. The first step of the Gateway procedure, the BP reaction, was then performed, during which the PCR fragment recombines in vivo with the pDONR201 plasmid to produce, according to the Gateway terminology, an "entry clone", p2782. Plasmid pDONR201 was purchased from Invitrogen, as part of the Gateway®  technology.<br>
Example 2: Vector Construction<br>
The entry clone p2782 was subsequently used in an LR reaction with p0640, a destination vector used for Oryza sativa transformation. This vector contains as functional elements within the T-DNA borders: a plant selectable marker; a plant screenable marker; and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the entry clone. A rice GOS2 promoter for constitutive expression (PRO0129) is located upstream of this Gateway cassette. After the LR recombination step, the resulting expression vector as shown in Figure 3 was transformed into Agrobacterium and subsequently into Oryza sativa plants. Transformed rice plants were allowed to grow and were then examined for the parameters described in Example 3.<br>
Example 3; Evaluation and Results<br>
Approximately 15 to 20 independent TO rice transformants were generated. The primary transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. 6 events, of which the T1 progeny segregated 3:1 for presence/absence of the transgene, were retained. For each of these events, approximately 10 T1 seedlings containing the transgene (hetero- and homo-zygotes), and approximately 10 T1 seedlings lacking the transgene (nullizygotes), were selected by monitoring visual marker expression. Some T1 events were further evaluated in the T2 generation following the same evaluation procedure as for the T1 generation.<br>
Statistical analysis: F-test<br>
A two factor ANOVA (analysis of variants) was used as a statistical model for the overall evaluation of plant phenotypic characteristics. An F-test was carried out on all the parameters measured of all the plants of all the events transformed with the gene of the present invention. The F-test was carried out to check for an effect of the gene over all the transformation events and to verify for an overall effect of the gene, also known as a global gene effect. The threshold for significance for a true global gene effect was set at a 5% probability level for the<br><br>
F-test. A significant F-test value points to a gene effect, meaning that it is not only the presence or position of the gene that is causing the differences in phenotype.<br>
3.1 Seed-related parameter measurements<br>
The mature primary panicles were harvested, bagged, barcode-labelled and then dried for three days in the oven at 37°C. The panicles were then threshed and all the seeds were collected and counted. The filled husks were separated from the empty ones using an air-blowing device. The empty husks were discarded and the remaining fraction was counted again. The filled husks were weighed on an analytical balance. This procedure resulted in the set of seed-related parameters described below.<br>
The Table of results below show the p values from the F test for the T1 evaluations, the T2 evaluations and the combined p values from the F tests for the T1 and T2 evaluations. A combined analysis may be considered when two experiments have been carried out on the same events. This may be useful to check for consistency of the effects over the two experiments and to increase confidence in the conclusion. The method used is a mixed-model approach that takes into account the multilevel structure of the data (i.e. experiment - event -segregants). P-values are obtained by comparing likelihood ratio test to chi square distributions. Each of the tables also gives the % difference between the transgenics and the corresponding nullizygotes for each generation.<br>
3.1.1 Number of filled seeds<br>
The number of filled seeds was determined by counting the number of filled husks that remained after the separation step. As shown in Table 1 below, the p value from the F test for the T1 and T2 evaluation combined was significant (with a p value of 0.0071) indicating that the presence of the construct in the plants has a significant effect on the number of filled seeds of transgenic plants.<br>
Table 1<br><br><br>
3.1.2	Total seed yield per plant<br>
The total seed yield was measured by weighing all filled husks harvested from a plant. As shown in Table 1 below, the p value from the F test for the T1 and T2 evaluation combined was significant (with a p value of 0.0016) indicating that the presence of the construct in the plants has a significant effect on the total seed weight of transgenic plants.<br>
Table 2<br><br>
3.1.3	Harvest index of plants<br>
The harvest index in the present invention is defined as the ratio between the total seed yield and the above ground area (mm2), multiplied by a factor 106. As shown in Table 3 below, the p value from the F test for the T1 and T2 evaluation combined (and individually) was significant (with a p value of 0.0005) indicating that the presence of the construct in the plants has a significant effect on the harvest index of transgenic plants.<br>
Table 3<br><br>
3.1 A Thousand Kernel Weight (TKW)<br>
This parameter is extrapolated from the number of filled seeds counted, and their total weight As shown in Table 4 below, the p value from the F test for the T1 and T2 evaluation combined was significant (with a p value of 0.0405) indicating that the presence of the construct in the plants has a significant effect on the TKW of transgenic plants.<br><br>
Table 4  	<br>
Claims<br>
1.	Method  for  improving plant  growth characteristics,  comprising introducing  and<br>
2.	expressing in a plant a nucleic acid encoding an small subunit ribosomal protein S3a<br>
3.	polypeptide.<br>
4.	Method according to claim 1, comprising introducing and expressing in a plant:<br>
5.	(i)  A portion of an S3a-encoding nucleic acid;<br>
(ii) A sequence capable of hybridising to an S3a-encoding nucleic acid;<br>
(iii)  An alternative splice variant of an S3a-encoding nucleic acid;<br>
(iv)  An allelic variant of an S3a-encoding nucleic acid; and<br>
(v) A nucleic acid encoding a homologue or derivative of an S3a amino acid<br>
sequence,<br>
wherein proteins encoded by (i) to (v) align with known S3a protein sequences and comprise regions of conservation corresponding to the boxed regions shown in the alignment of Figure 2.<br>
3.	Method according to claim 1 or 2, wherein said improved plant growth characteristic is<br>
4.	increased yield relative to corresponding wild type plants.<br>
5.	Method according to claim 3, wherein said increased yield is increased seed yield.<br>
6.	Method according to claim 3 or 4, wherein said increased yield is selected from (i)<br>
7.	increased seed biomass; (ii) increased number of (filled) seeds; (iii) increased seed<br>
8.	size; (iv) increased seed volume; (v) increased harvest index; and (vi) increased<br>
9.	thousand kernel weight (TKW).<br>
10.	Method according to any of one of claims 1 to 5, wherein said improved plant growth<br>
11.	characteristics is increased growth rate.<br>
12.	Method according to any one of claims 1 to 6, wherein said nucleic acid encoding an<br>
13.	S3a is derived from a plant, preferably from a dicotyledonous plant, further pre-ferably<br>
14.	from the family Brassicaceae, more preferably the nucleic acid is from ArabJdopsis<br>
15.	thaliana.<br>
16.	Method according to any one of claims 1 to 7, wherein said nucleic acid sequence<br>
17.	encoding an S3a is overexpressed in a plant.<br><br>
9. Method according to any one of claims 1 to 8, wherein expression of said nucleic acid encoding an S3a is driven by a constitutive promoter, particularly the GOS2 promoter.<br>
10. Method for improving plant growth characteristics, particularly yield, further particularly seed yield, comprising introducing a genetic modification into a plant in the locus of a gene encoding an S3a polypeptide or a variant thereof.<br>
11.	Method according to claim 10, wherein said genetic modification is effected by one of:<br>
site-directed mutagenesis, homologous recombination, TILLING, directed evolution<br>
and T-DNA activation.<br>
12.	Plants obtainable by a method according to any of claims 1 to 11.<br>
13.	Construct comprising:<br>
(i)  An S3a-encoding nucleic acid;<br>
(ii) one or more control sequence capable of driving expression of the nucleic acid<br>
sequence of (i); and optionally (iii)  a transcription termination sequence.<br>
14.	Construct according to claim 13, wherein said control sequences comprise at least a<br>
15.	constitutive promoter, preferably a GOS2 promoter.<br>
16.	Plant transformed with a construct according to claim 13 or 14.<br>
17.	Method for the production of a transgenic plant having improved growth characteristics,<br>
18.	which method comprises:<br>
(i)  introducing into a plant or plant cell an S3a-encoding nucleic acid; (ii) cultivating the plant cell under conditions promoting regeneration and mature plant growth.<br>
17.	Transgenic plant having improved growth characteristics, characterised in that said<br>
18.	plant has introduced and expressed therein a nucleic acid sequence encoding an S3a<br>
19.	protein.<br>
20.	Transgenic plant according to claim 12, 15 or claim 17, wherein said plant is a<br>
21.	monocotyledonous plant such as sugarcane or a cereal selected from rice, maize,<br><br>
wheat, barley, soybean, sorghum, sunflower, canola, sugarcane, alfalfa, millet, barley, rapeseed and cotton.<br>
19.	Harvestable parts of a plant according to any one of claims 12,15,17 or 18.<br>
20.	Harvestable parts of a plant according to claim 19, wherein said harvestable parts are<br>
21.	seeds.<br>
22.	Use of an  S3a or of an S3a-encoding  nucleic acid  in  improving the growth<br>
23.	characteristics of plants, in particular in improving yield, especially seed yield.<br>
24.	Use according to claim 21, wherein said seed yield includes one or more of the<br>
25.	following: increased number of (filled) seeds, increased seed weight, increased harvest<br>
26.	index and increased TKW.<br>
27.	Use according to claim 21 or 22, wherein said S3a-encoding nucleic acid is as<br>
28.	represented by SEQ ID NO: 1 and wherein said S3a is an amino acid as represented<br>
29.	bySEQIDNO:2.<br>
30.	Use of an S3a or of an S3a-encoding nucleic acid as a molecular marker.<br>
Dated this 21 day of September 2006<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjMtMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   AMENDED  PAGES OF SPECIFICATION  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgIDIzLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   AMENDED CLAIMS  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgMDktMDQtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   AMENDED CLAIMS 09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAwOS0wNC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   AMENDED PAGES OF SPECIFICATION  09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMTctMDgtMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   CORRESPONDENCE OTHERS   17-08-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDA5LTA0LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   CORRESPONDENCE OTHERS 09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgRk9STS0xICAyMy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   FORM-1  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgRk9STS0zICAyMy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   FORM-3  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgRk9STS0zIDA5LTA0LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   FORM-3 09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyMy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   OTHER PATENT DOCUMENT  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIzLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006   POWER OF ATTORNEY  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1DSEVOUC0yMDA2ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgMjMtMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3462-CHENP-2006  EXAMINATION REPORT REPLY RECEIVED 23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzQ2Mi1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3462-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252150-pyrrole-compounds-as-modulators-of-metabotropic-glutamate-receptor-5.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252152-a-pneumatic-flotation-column-for-treatment-of-mineral-resources.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252151</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3462/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-May-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Sep-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3, B-9052 ZWIJNAARDE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FRANKARD, VALERIE</td>
											<td>BIERENBERG 47, B-1640 SINT-GENESIUS-RODE, BELGIUM</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DUDITS, DENES</td>
											<td>BERKERT UTCA 36/3, H-6726 SZEGED, HUNGARY</td>
										</tr>
										<tr>
											<td>3</td>
											<td>FEHER, ATTILA</td>
											<td>JACINT UTCA 29, H-6771 SZEGED, HUNGARY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/415</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/51281</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/556,847</td>
									<td>2004-03-26</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>04101179.2</td>
									<td>2004-03-22</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252151-method-of-making-plants-having-increased-seed-yield-and-growth-rate by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:41 GMT -->
</html>
