<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252136-a-method-of-obtaining-a-disease-resistant-plant-by-down-regulating-a-target-sequence-of-a-plant-virus-or-plant-viroid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:32:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252136:A METHOD OF OBTAINING A DISEASE RESISTANT PLANT BY DOWN REGULATING A TARGET SEQUENCE OF A PLANT VIRUS OR PLANT VIROID</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF OBTAINING A DISEASE RESISTANT PLANT BY DOWN REGULATING A TARGET SEQUENCE OF A PLANT VIRUS OR PLANT VIROID</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A METHOD OF OBTAINING A DISEASE RESISTANT PLANT BY DOWN REGULATING A TARGET SEQUENCE OF A PLANT VIRUS OR PLANT VIROID A method of obtaining a disease resistant plant by down regulating a target sequence of a plant virus or plant viroid in a plant consisting of: (a) transforming a plant cell susceptible to infection by the plant virus or plant viroid with a nucleic acid construct comprising a promoter functional in a plant cell operably linked to a polynucleotide, wherein the polynucleotide encodes one or more modified miRNA precursors, wherein each modified miRNA precursor comprises a fragment of a plant miRNA primary transcript that is modified in the miRNA sequence and the miRNA complementary sequence, further wherein each modified miRNA precursor is capable of forming a first double-stranded RNA or a hairpin, wherein each modified miRNA is a miRNA modified to be (i) fully complementary to a target sequence, (ii) fully complementary to a target sequence except for GU base pairing or (iii) fully complementary to a target sequence in the first ten nucleotides counting from the 5&#x27; end of the miRNA, wherein the target sequence of each modified miRNA is the same or different, wherein at least one target sequence encodes a gene silencing suppressor of a plant virus or plant viroid and wherein the nucleic acid construct is stably integrated into the genome of the transformed plant cell; (b) obtaining a transgenic plant from the transformed plant cell of (a); (c) expressing the nucleic acid construct in the transgenic plant of (b) for a time sufficient to produce each modified miRNA, wherein each modified miRNA binds to its target sequence to form a second double-stranded RNA which is cleaved thereby down regulating the target sequence; and (d) selecting a transgenic plant of (c) that is resistant to a plant virus or plant viroid, wherein the plant virus or plant viroid comprises the at least one target sequence that encodes the gene silencing suppressor, wherein the nucleic acid construct is stably integrated into the genome of the resistant transgenic plant and wherein the resistant transgenic plant shows no symptoms of infection.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A METHOD OF OBTAINING A DISEASE RESISTANT PLANT BY DOWN REGULATING A TARGET SEQUENCE OF A PLANT VIRUS OR PLANT VIROID<br><br>
WO 2006/044322	PCT/US2005/036373<br>
FIELD OF THE INVENTION<br>
[0001] The field of the present invention relates generally to plant molecular biology and<br>
plant biotechnology. More specifically it relates to constructs and methods to suppress the<br>
expression of targeted genes or to down regulate targeted genes.<br>
BACKGROUND OF THE INVENTION<br>
[0002] RNA interference refers to the process of sequence-specific post-transcriptional gene<br>
silencing in animals mediated by short interfering RNAs (siRNAs) (Fire et al. (1998) Nature<br>
391:806-811). The corresponding process in plants is commonly referred to as post-<br>
transcriptional gene silencing (PTGS) or RNA silencing and is also referred to as quelling in<br>
fungi. The process of post-transcriptional gene silencing is thought to be an evolutionarily-<br>
conserved cellular defense mechanism used to prevent the expression of foreign genes and is<br>
commonly shared by diverse flora and phyla (Fire (1999) Trends Genet. 15:358-363). Such<br>
protection from foreign gene expression may have evolved in response to the production of<br>
double-stranded RNAs (dsRNAs) derived from viral infection or from the random integration of<br>
transposon elements into a host genome via a cellular response that specifically destroys<br>
homologous single-stranded RNA of viral genomic RNA. The presence of dsRNA in cells<br>
triggers the RNAi response through a mechanism that has yet to be fully characterized.<br>
[0003] A new class of small RNA molecules is involved in regulating gene expression in a<br>
number of eukaryotic organisms ranging from animals to plants. These short RNAs or<br>
microRNAs (miRNAs; miRs) are 20-22 nucleotide-long molecules that specifically base-pair to<br>
target messenger-RNAs to repress their translation or to induce their degradation. Recent<br>
reports have identified numerous miRNAs from vertebrates, Caenorhabditis elegans,<br>
Drosophila and Arabidopsis thaliana (Bartel (2004) Cell 116:281-297; He and Harmon (2004)<br>
Nature Reviews Genetics 5:522-531).<br>
[0004] Viruses such as Turnip Mosaic Virus (TuMV) and Turnip Yellow Mosaic Virus<br>
(TYMV) cause considerable crop loss world-wide and have serious economic impact on<br>
agriculture (Morch et al. (1998) Nucleic Acids Res 16:6157-6173; Skotnicki et al. (1992) Arch<br>
Virol 127:25-35; Tomlinson (1987) Ann Appl Biol 110:661-681). Most if not all plant viruses<br>
encode one or more proteins that are able to suppress the host's post-transcriptional gene<br>
silencing (PTGS) mechanism so as to ensure their successful replication in host cells.   The<br><br>
WO 2006/044322	PCT/US2005/036373<br>
PTGS is a mechanism that a plant host uses to defend against viruses by triggering breakdown<br>
of double stranded RNAs which are produced as intermediates in viral genome replication<br>
(Bernstein et al. (2001) Nature 409:363-366; Hamilton and Baulcombe (1999) Science 286:950-<br>
952; Zamore et al. (2000) Cell 31:25-33).<br>
[0005] Reduction of the activity of specific genes (also known as gene silencing, or gene<br>
suppression), including virus genes, is desirable for several aspects of genetic engineering in<br>
plants. There is still a need for methods and constructs that induce gene suppression against a<br>
wide selection of target genes, and that result in effective silencing of the target gene at high<br>
efficiency.<br>
SUMMARY OF THE INVENTION<br>
[0006] According to one aspect, the present invention provides a method of down regulating<br>
a target sequence in a cell and a nucleic acid construct for use in this method, as well as a<br>
polynucleotide for use in the nucleic acid construct. The method comprises introducing into the<br>
cell a nucleic acid construct capable of producing miRNA and expressing the nucleic acid<br>
construct for a time sufficient to produce the miRNA, wherein the miRNA inhibits expression of<br>
the target sequence. The nucleic acid construct comprises a polynucleotide encoding a modified<br>
miRNA precursor capable of forming a double-stranded RNA or a hairpin, wherein the modified<br>
miRNA precursor comprises a modified miRNA and a sequence complementary to the modified<br>
miRNA, wherein the modified miRNA is a miRNA modified to be (i) fully complementary to<br>
the target sequence or (ii) fully complementary to the target sequence except for GU base<br>
pairing. As is well known in the art, the pre-miRNA forms a hairpin which in some cases the<br>
double-stranded region may be very short, e.g., not exceeding 21-25 bp in length. The nucleic<br>
acid construct may further comprise a promoter operably linked to the polynucleotide. The cell<br>
may be a plant cell, either monocot or dicot, including, but not limited to, corn, wheat, rice,<br>
barley, oats, sorghum, millet, sunflower, safflower, cotton, soy, canola, alfalfa, Arabidopsis, and<br>
tobacco. The promoter may be a pathogen-inducible promoter or other inducible promoters.<br>
The binding of the modified miRNA to the target RNA leads to cleavage of the target RNA.<br>
The target sequence of a target RNA may be a coding sequence, an intron or a splice site.<br>
[0007] According to another aspect, the present invention provides an isolated polynucleotide<br>
encoding a modified plant miRNA precursor, the modified precursor is capable of forming a<br>
double-stranded RNA or a hairpin and comprises a modified miRNA and a sequence<br>
complementary to the modified miRNA, wherein the modified miRNA is a miRNA modified to<br><br>
WO 2006/044322	PCT/US2005/036373<br>
be (i) fully complementary to the target sequence or (ii) fully complementary to the target<br>
sequence except for GU base pairing. Expression of the polynucleotide produces a miRNA<br>
precursor which is processed in a host cell to provide a mature miRNA which inhibits<br>
expression of-the target sequence. The polynucleotide may be a nucleic acid construct or may<br>
be the modified plant miRNA precursor. The nucleic acid construct may further comprise a<br>
promoter operably linked to the polynucleotide. The promoter may be a pathogen-inducible<br>
promoter or other inducible promoter. The binding of the modified miRNA to the target RNA<br>
leads to cleavage of the target RNA. The target sequence of a target RNA may be a coding<br>
sequence, a non-coding sequence or a splice site.<br>
[0008] According to another aspect, the present invention provides a nucleic acid construct<br>
for suppressing a multiple number of target sequences. The nucleic acid construct comprises at<br>
least two and up to 45 or more polynucleotides, each of which encodes a miRNA precursor<br>
capable of forming a double-stranded RNA or a hairpin. Each miRNA is substantially<br>
complementary to a target or is modified to be complementary to a target as described herein. In<br>
some embodiments, each of the polynucleotides encoding precursor miRNAs in the construct is<br>
individually placed under control of a single promoter. In some embodiments, the multiple<br>
polynucleotides encoding precursor miRNAs are operably linked together such that they can be<br>
placed under the control of a single promoter. The promoter may be operably linked to the<br>
construct of multiple miRNAs or the construct of multiple miRNAs may be inserted into a host<br>
genome such that it is operably linked to a single promoter. The promoter may be a pathogen-<br>
inducible promoter or other inducible promoter. In some embodiments, the multiple<br>
polynucleotides are linked one to another so as to form a single transcript when expressed.<br>
Expression of the polynucleotides in the nucleic acid construct produces multiple miRNA<br>
precursors which are processed in a host cell to provide multiple mature miRNAs, each of which<br>
inhibits expression of a target sequence. In one embodiment, the binding of each of the mature<br>
miRNA to each of the target RNA leads to cleavage of each of the target RNA. The target<br>
sequence of a target RNA may be a coding sequence, a non-coding sequence or a splice site.<br>
[0009] According to another aspect, the present invention provides a method of down<br>
regulating a multiple number of target sequences in a cell. The method comprises introducing<br>
into the cell a nucleic acid construct capable of producing multiple miRNAs and expressing the<br>
nucleic acid construct for a time sufficient to produce the multiple miRNAs, wherein each of the<br>
miRNAs inhibits expression of a target sequence. The nucleic acid construct comprises at least<br>
two and up to 45 or more polynucleotides, each of which encodes a miRNA precursor capable of<br><br>
WO 2006/044322	PCT/US2005/036373<br>
forming a double-stranded RNA or a hairpin. Each miRNA is substantially complementary to a<br>
target or is modified to be complementary to a target as described herein. In some<br>
embodiments, each of the polynucleotides encoding precursor miRNAs in the construct is<br>
individually placed under control of a single promoter. In some embodiments, the multiple<br>
polynucleotides encoding precursor miRNAs are linked together such that they can be under the<br>
control of a single promoter as described herein. In some embodiments, the multiple<br>
polynucleotides are linked one to another so as to form a single transcript when expressed. In<br>
some embodiments, the construct may be a hetero-polymeric pre-miRNA or a homo-polymeric<br>
pre-miRNA. Expression of the polynucleotides in the nucleic acid construct produces multiple<br>
miRNA precursors which are processed in a host cell to provide multiple mature miRNAs, each<br>
of which inhibits expression of a target sequence. In one embodiment, the binding of each of the<br>
mature miRNA to each of the target RNA leads to cleavage of each of the target RNA. The<br>
target sequence of a target RNA may be a coding sequence, a non-coding sequence or a splice<br>
site.<br>
[0010] According to a further aspect, the present invention provides a cell comprising the<br>
isolated polynucleotide or nucleic acid construct of the present invention. In some<br>
embodiments, the isolated polynucleotide or nucleic acid construct of the present invention may<br>
be inserted into an intron of a gene or a transgene of the cell. The cell may be a plant cell, either<br>
a monocot or a dicot, including, but not limited to, corn, wheat, rice, barley, oats, sorghum,<br>
millet, sunflower, safflower, cotton, soy, canola, alfalfa, Arabidopsis, and tobacco.<br>
[0011] According to another aspect, the present invention provides a transgenic plant<br>
comprising the isolated polynucleotide or nucleic acid construct. In some embodiments, the<br>
isolated polynucleotide or nucleic acid construct of the present invention may be inserted into an<br>
intron of a gene or a transgene of the transgenic plant. The transgenic plant may be either a<br>
monocot or a dicot, including, but not limited to, corn, wheat, rice, barley, oats, sorghum, millet,<br>
sunflower, safflower, cotton, soy, canola, alfalfa, Arabidopsis, and tobacco.<br>
[0012] According to a further aspect, the present invention provides a method of inhibiting<br>
expression of a target sequence in a cell comprising: (a) introducing into the cell a nucleic acid<br>
construct comprising a modified plant miRNA precursor comprising a first and a second<br>
oligonucleotide, wherein at least one of the first or the second oligonucleotides is heterologous<br>
to the precursor, wherein the first oligonucleotide encodes an RNA sequence substantially<br>
identical to the target sequence, and the second oligonucleotide encodes a miRNA substantially<br>
complementary to the target sequence, whereby the precursor encodes  a miRNA;   and<br><br>
WO 2006/044322	PCT/US2005/036373<br>
(b) expressing the nucleic acid construct for a time sufficient to produce the miRNA, wherein<br>
the miRNA inhibits expression of the target sequence.<br>
[0013] According to another aspect, the present invention provides an isolated polynucleotide<br>
comprising a modified plant miRNA precursor, the modified precursor comprising a first and a<br>
second oligonucleotide, wherein at least one of the first or the second oligonucleotides is<br>
heterologous to the precursor, wherein the first oligonucleotide encodes an RNA sequence<br>
substantially identical to a target sequence, and the second oligonucleotide comprises a miRNA<br>
substantially complementary to the target sequence, wherein expression of the polynucleotide<br>
produces the miRNA which inhibits expression of the target sequence. The present invention<br>
also relates to a cell comprising this isolated polynucleotide. The cell may be a plant cell, either<br>
monocot or dicot, including, but not limited to, corn, wheat, rice, barley, oats, sorghum, millet,<br>
sunflower, safflower, cotton, soy, canola, alfalfa, Arabidopsis, and tobacco.<br>
[0014] According to a further aspect, the present invention provides for a method of<br>
inhibiting expression of a target sequence in a cell, such as any of those herein described that<br>
further comprises producing a transformed plant, wherein the plant comprises the nucleic acid<br>
construct which encodes the miRNA. The present invention also relates to a plant produced by<br>
such methods. The plant may a monocot or a dicot, including, but not limited to, corn, wheat,<br>
rice, barley, oats, sorghum, millet, sunflower, safflower, cotton, soy, canola, alfalfa,<br>
Arabidopsis, and tobacco.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
[0015]      Figure 1 shows the predicted hairpin structure formed by the sequence surrounding<br>
miRl72a-2, The mature microRNA is indicated by a grey box.<br>
[0016]      Figure 2 shows the miR172a-2 overexpression phenotype.   a, Wild type (Columbia<br>
ecotype) plant, 3.5 weeks old. b, EAT-D plant, 3.5 weeks old. c, Wild type flower,  d, EAT-D<br>
flower. Note absence of second whorl organs (petals). Arrow indicates sepal with ovules along<br>
the margins and stigmatic papillae at the tip.   e, Cauline leaf margin from a 35S-EAT plant.<br>
Arrows indicate bundles of stigmatic papillae projecting from the margin, f, Solitary gynoecium<br>
(arrow) emerging from the axil of a cauline leaf of a 35S-EAT plant.<br>
[0017]      Figure 3  shows the EAT  gene contains a miRNA that is complementary to<br>
AJPETALA2 (AP2). a, Location of the EAT gene on chromosome 5. The T-DNA insertion and<br>
orientation of the 35S enhancers is indicated. The 21-nt sequence corresponding to miR172a-2<br>
is shown below the EAT gene (SEQ ID NO:86). b, Putative 21-nt miRl72a-2/AP2 RNA duplex<br><br>
WO 2006/044322	PCT/US2005/036373<br>
is shown below the gene structure of AP2.   The GU wobble in the duplex is underlined,   c,<br>
Alignment  of AP2  21-nt region (black bar) and  surrounding sequence with  three other<br>
Arabidopsis AP2 family members, and with two maize AP2 genes (IDS1  and GL15).    d,<br>
Alignment of miR172a-2 miRNA (black bar) and surrounding sequence with rm'R172-like<br>
sequences from Arabidopsis, tomato, soybean, potato and rice.<br>
[0018]      Figure 4 shows the miR172a-2 miRNA expression,   a, Northern blot of total RNA<br>
from wild type (lanes 3 and 7) and EAT-D (lanes 4 and 8). Blots were probed with sense (lanes<br>
1-4) or antisense (lanes 5-8) oligo to miR172a-2 miRNA.   100 pg of sense oligo (lanes 2 and 6)<br>
and antisense oligo (lanes 1 and 5) were loaded as hybridization controls.   Nucleotide size<br>
markers are indicated on the left,   b, SI nuclease mapping of miR172a-2 miRNA.   A 5'-end-<br>
labeled probe undigested (lane 1) or digested after hybridization to total RNA from wild-type<br>
(lane 2), EAT-D (lane 3), or tRNA (lane 4).<br>
[0019]      Figure 5 shows the developmental expression pattern of miR172 family members,  a,<br>
RT-PCR of total RNA from wild type seedlings harvested at 2, 5, 12, and 21  days after<br>
germination (lanes 1-4, respectively), or from mature leaves (lane 5) and floral buds (lane 6).<br>
Primers for PCR are indicated on the left,   b, Northern analysis of mirR172 expression in the<br>
indicated mutants, relative to wild type (Col).  Blot was probed with an oligo to miR172a-2;<br>
however, all miRl 72 members should cross hybridize.<br>
[0020]      Figure 6 shows the expression analysis of putative EAT target genes, a, Northern blot<br>
analysis of polyA+ RNA isolated from wild type (Col) or EAT-D floral buds.   Probes for<br>
hybridization are indicated on the right,  b, Western blot of proteins from wild type or EAT-D<br>
floral buds, probed with AP2 antibody.    RbcL, large subunit of ribulose 1,5-bisphosphate<br>
carboxylase as loading control.<br>
[0021]      Figure 7 shows the identification of LAT-D.   a, Location of the T-DNA insert in<br>
LAT-D, in between At2g28550 and At2g28560. The 4X 35S enhancers are approximately 5 kb<br>
from At2g28550.   b, RT-PCR analysis of At2g28550 expression in wild type versus LAT-D<br>
plants.<br>
[0022]      Figure 8 shows that EAT-D is epistatic to LAT-D. Genetic cross between EAT-D and<br>
LAT-D plants, with the resultant Fl plants shown, along with their flowering time (measured as<br>
rosette leaf number).<br>
[0023]      Figure 9 shows the loss-of-function At2g28550 (2-28550) and At5g60120 (6-60120)<br>
mutants. Location of T-DNA in each line is indicated, along with intron/exon structure.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
7<br>
[0024] Figure 10 shows the potential function of the miR172 miRNA family, a, Temporal<br>
expression of miRl 72a-2 and its relatives may cause temporal downregulation of AP2 targets<br>
(e.g. At2g28550 and At5g60120), which may trigger flowering once the target proteins drop<br>
below a critical threshold (dotted line), b, Dicer cleavage at various positions may generate at<br>
least four distinct miRNAs from the miRl 72 family (indicated as a single haiipin with a miRNA<br>
consensus sequence). Sequences at the 5' and 3' ends of each miRNA are indicated, with the<br>
invariant middle 15 nt shown as ellipses. The putative targets recognized by the individual<br>
miRNAs are in parentheses below each.<br>
[0025] Figures 11A-11C show an artificial microRNA (miRNA) designed to cleave the<br>
phytoene desaturase (PDS) mRNAs of Nicotiana benthamiana. Fig. 11A shows the structure of<br>
the pre-miR159a sequence construct under the control of the CaMV 35S promoter (35S) and<br>
NOS terminator (Tnos). The orientation and position of the mature miRNA is indicated by an<br>
aiTow. Fig. 11B shows that point mutations in miRl 59a (SEQ ID NO: 141) (indicated by<br>
arrows) turn it into miR-PDSl59a (SEQ ID NO: 142) to become fully complementary to a region<br>
in N. benthamiana PDS mRNA. Fig. 11C shows that Northern blot analysis of Agrobacterium<br>
infiltrated TV. benthamiana leaves shows expression of miR-PAS"159a, miR-PDSl59a* and miRl59<br>
in samples infiltrated with an empty vector (vector) or the artificial miRNA (xniR-PDS]59a) 1, 2,<br>
3 days post infiltration (d.p.i).<br>
[0026] Figures 12A-12B show that miR-PDS]59a (SEQ ID NO: 142) causes PDS mRNA (SEQ<br>
ID NO: 143) cleavage. Fig. 12 A shows Northern blot analysis of PDS mRNA from samples<br>
infiltrated with empty vector or miR-PDSl59a after 1 or 2 days post infiltration (d.p.i.) (upper<br>
panel). The bottom panel shows the EtBr-stained agarose gel from the same samples. Fig. 12B<br>
shows the site of cleavage of the miRNA. 5'RACE analysis was conducted on samples<br>
infiltrated with rniR-PDS159aconstructs and the 5'-end sequence of 5 out of 6 clones indicated<br>
the site of cleavage of the miRNA as indicated by an arrow.<br>
[0027] Figures 13A-13E show that the expression of miR-PDS169g results in cleavage of the<br>
PDS mRNA. Fig. 13A shows the point mutations (underlined nucleotides) in miR169g that it<br>
turn it into miR-PDSa169g(SEQ ID NO:145) or miR-PDSb]69g (SEQ ID NO:146) to become fully<br>
complementary to two different regions in N. benthamiana PDS mRNA. Fig. 13B shows<br>
Northern blot analysis of two different miR169g expression constructs. Total RNA was<br>
extracted from non-infiltrated leaves (C) or from leaves infiltrated with Agrobacterium<br>
containing the pre-miR169g sequence in the context of a 0.3kb (0.3kb) or 2.0kb (2.0kb)<br>
fragment, or from control Arabidopsis leaves (+).   The arrow indicates the position of the<br><br>
WO 2006/044322	PCT/US2005/036373<br>
miR169 signal. Fig. 13C shows Northern blot showing the expression of miR-PDSal69g (a) and<br>
miR-PDSb169g (b) in infiltrated leaves containing the 0.3kb construct but not in control using the<br>
empty plasmid (vector). Fig. 13D shows the sites of cleavage of the miRNA. 5'RA.CE analysis<br>
was conducted on samples infiltrated with miR-PDSl69g a (SEQ ID NO: 145) and b (SEQ ID<br>
NO:146) constructs and the 5'-end sequence identified from independent clones is indicated by<br>
an arrow together with the number of clones analyzed. The PDS mRNAs are SEQ ID NO: 147<br>
and SEQ ID NO: 148. Fig. 13E shows a Northern blot analysis to detect PDS mRNA levels in<br>
plants infiltrated with Agrobacterium strains carrying the empty vector (C) or constructs<br>
expressing miR-PDSa169g (a) or miR-PDSbl69g (b).<br>
[0028] Figures 14A-14C show the microRNA-directed cleavage of Nicotiana benthamiana<br>
rbcS mRNAs. Fig. 14A shows that point mutations in miR159a (SEQ ID NO:141) (Indicated by<br>
arrows) turn it into miR-rbcSl59a-A (SEQ ID NO: 149) to become complementary to a region<br>
common to all N. benthamiana rbcS mRNAs (shown as rbcS mRNA; SEQ ID NO: 150).<br>
miRNA:mRNA base-pairs are indicated by vertical lines and G:U wobble base-paiis by colons.<br>
Fig. 14B shows that Northern blot analysis of Agrobacterium infiltrated N. benthar?iiana leaves<br>
shows expression of miR-rbcSl59a-A in samples infiltrated with an empty vector (C) or the<br>
artificial miRNA (A) 2 days post infiltration (d.p.i). Fig. 14C shows that RT-PCR analysis was<br>
used to detect rbcS mRNA abundance for all six genes in the same samples shown in B.<br>
Amplification of EFla mRNA served as a loading control.<br>
[0029] Figures 15A-15B show the schematic representation of the genes and relevant<br>
sequences used in the work shown in Figures 11-14. Fig. 15A shows the PDS gene from<br>
Lycopersicum esculetum that was used as reference sequence since the complete PDS gene from<br>
N. benthamiana is not known (segments missing are shown as a dashed line). Large grey arrows<br>
indicate positions targeted by the miR-PDS constructs described in the text. Small arrowheads<br>
indicate primers used for 5'RACE analysis. Known N. benthamiana PDS fragments are<br>
indicated along with the origin of the sequences. Fig. 15B shows the different reported<br>
sequences that were used to assemble the rbcS gene sequence schematized here. Thie grey arrow<br>
indicates the position of the sequence targeted by miR-rbcS159a-A, the arrowheads indicate the<br>
position of primers used in RT-PCR experiments shown in Figures 14A-14C.<br>
[0030] Figures 16A-16B show a summary of changes introduced to Arabidopsis miR159a<br>
and miR169g. Fig. 16A shows sequences of miR-PDS159a (SEQ ID NO: 142) and miR-rbcSl59a-<br>
A (SEQ ID NO:149) as compared to miR159a (SEQ ID NO:141). The base-changes in each<br>
case are underlined while unmodified positions are marked with an asterisk.   Fig. 16B shows<br><br>
WO 2006/044322	PCT/US2005/036373<br>
sequences  of miR-PDSal69g (SEQ ID NO:145) and miR-PDSb169g (SEQ ID NO:146) as<br>
compared to miR169g (SEQ ID NO:144).  The base-changes in each case are underlined while<br>
unmodified positions are marked with an asterisk.<br>
[0031]      Figure 17 shows development of Arabidopsis root hairs in wildtype, mutant and<br>
transgenic plants. Panel A: Wild type root shows many root hair structures. Panel B: Very few<br>
root hair in cpc mutant. Panel C: 35S::CPC plants show more root hairs. Panel D: More root<br>
hair in gl2 mutant. This figure is taken from Wada et al. ((2002) Development 129:5409-5419).<br>
[0032]      Figure 18 shows Arabidopsis root hair development in transgenic plants.   Panel a:<br>
XVE::pre-miRCPCl159a without inducer (estradiol).    Panel b: XVE::pre-miRCPCl159" with<br>
inducer (estradiol). Panel c: XVE::pre-miRl59a without inducer (estradiol). Panel d: XVE::pre-<br>
miR159a with inducer (estradiol).<br>
[0033]      Figure 19 shows Arabidopsis root hair development in transgenic plants.   Pane] a:<br>
35S::pre-miR159. Panel b: 35S::pre-miR.CPClI59a. Panel c: 35S::pre-miRP
[0034]      Figure 20 shows Arabidopsis root hair development in transgenic plants.   Panel a:<br>
35S::pre-miR159. Panel b: 35S::pre-miRCPClI59a.<br>
[0035]      Figures 21A-21E represent a diagram for a process for designing a polymeric pre-<br>
miRNA.   Fig. 21A: The products of amplification of three different pre-miRNAs (pre-miR A,<br>
pre-miR B  and pre-miR C) in which Avrll, Spel and Xhol  sites have been added by<br>
amplification. Fig. 21B: Pre-miR A is digested with Spel and Xhol and pre-miR B is digested<br>
with Avrll and Xhol.  Fig. 21C: The digested pre-miR A and pre-miR B are ligated to form a<br>
dimeric pre-miRNA. Fig. 21D: Pre-miR A-B is digested with Spel and Xhol and pre-miR C is<br>
digested with Avrll and Xhol. Fig. 21E: The digested pre-miR A-B and pre-miR C are ligated<br>
to form a trirneric pre-rmiRNA.<br>
[0036]      Figure 22 is a diagram of a dimeric construct containing pre-miRPDSl169g and pre-<br>
miRCPC3159a.<br>
[0037]      Figures 23A and 23B show that mature miRPDSl169g (Fig. 23A) and miRCPC3!59a<br>
(Fig. 23 B) was successfully produced from the dimeric construct.    Lane 1 is 35S::pre-<br>
miRPDSl169g, lane 2 is 35S::CPC3159a and lane 3 is 35S::pre-}niRPDSl169g-CPC3l59a.<br>
[0038]      Figure 24 shows the structure of the miR159a precursor (SEQ ID NO:161).<br>
[0039]      Figure 25 shows Northern blot analysis of miR-HC-Pro!59a were performed with three<br>
different treatments: (1) Agrobacterial cells with 35S::pre-miR-HC-Prol59a, (2) Agrobacterial<br>
cells  with  35S::HC-Pro,  and  (3) Agrobacterial cells  with  35S::pre-miR-HC-Pro159a  and<br>
35S::HC-Pro.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
10<br>
[0040] Figure 26 shows shows Northern blot analysis of miR-P69, 4 different treatments<br>
were performed : (1) Agrobacterial cells carrying 35S:: pre-miR-P69159a, (2) Agrobacterial<br>
cells XVE::pre-miR-P69159a, (3) Agrobacterial cells carrying 35S::P69, and (4) Agrobacterial<br>
cells carrying 35S::pre-miR-P69l59a and 35S::P-69.<br>
[0041] Figure 27 shows Northern blot analysis of mature artificial miRNA levels for<br>
randomly picked T2 35S::pre-miRHC-Pro!59a transgenic lines (plants). The T2 plants are known<br>
to be transgenic because they were first selected on Kan-containing medium to remove WT.<br>
The T-2 plants are either heterozygous (one copy) or homozygous (two copies), and the ratio<br>
should be about 2:1.<br>
[0042] Figure 28 shows Northern blot analysis of mature artificial miRNA levels for<br>
randomly picked T2 35S::pre-miR-P69I59a transgenic lines (plants).<br>
[0043] Figure 29 shows that T2 transgenic plants expressing miR-HC-ProI59a artificial<br>
miRNA are resistant to TuMV infection. Photographs were taken 2 weeks (14 days after<br>
infection) after inoculation. T2 transgenic plants expressing miR-HC-Pro159a (line #11; Fig.<br>
33B) developed normal influorescences whereas WT plants and T2 transgenic plants expressing<br>
miR-P69!59a (line #1; Fig 33B) showed viral infection symptoms. The bar represents 3 cm.<br>
[0044] Figure 30 shows symptoms of influorescences caused by TuMV infection. (Top<br>
panel) Forteen days after TuMV infection, T2 transgenic miR-P69!59a plants (line #1) and col-0<br>
plants showed shorter internodes between flowers in influoresences, whereas T2 miR-HC-Pro159a<br>
transgenic plant (line #11) displayed normal influoresences development. The bar represents 1<br>
cm. (Bottom panel) Close-up views of influoresences on TuMV-infected Axabidopsis plants. T2<br>
transgenic miR-P69159a plants (line #1) and col-0 plants showed senescence and pollination<br>
defects whereas T2 transgenic miR-HC-Pro159a plants (line #11) showed normal flower and<br>
silique development. For mock-infection, plants were inoculated with buffer only. The bar<br>
represents 0.2 cm.<br>
[0045] Figure 31 shows symptoms of siliques caused by TuMV infection. In TuMV-<br>
infected T2 transgenic miR-P69I59a plants (line #1) and WT (col-0) plants, siliques were small<br>
and mal-developed. T2 transgenic miR-HC-Pro159a plants (line #11) were resistant to TuMV<br>
infection and showed normal silique development. Buffer-inoculated plants (mock-inculated)<br>
were used as controls. The bar represents 0.5 cm.<br>
[0046] Figure 32 shows Western blot analysis of TuMV coat protein (CP) levels in leaves of<br>
different transgenic and WT plants.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
11<br>
[0047]      Figure 33 shows (A) Western blot analysis of representative plants of 35S:: miR-HC-<br>
Pro159a, 35S:: miR-P69!59a, and WT (Col-o) and (B) Northern blot analysis of miRNAs<br>
produced by the transgenic plants.<br>
[0048]      Figure  34 shows  ELISA detection of TuMV in different transgenic  and non-<br>
transgenic Arabidopsis.<br>
[0049]      Figure 35 shows Northern blot analysis of prt-miR-P69l59a, pre-miR-HC-Pro159a and<br>
prt-miR-P69159a-HC-Pro159a  demonstrating that homo-dimeric  miRNA precursor, pre-miR-<br>
P69l59a-HC-Pro!59a, can produce mature miR-P69I59a and miR-HC-ProlS9a.<br>
[0050]      Figure 36 shows constructs in which miR-HC-Pro159a is placed in either intron 1 or<br>
intron 2 of the CPC gene.<br>
[0051]      Figure  37   shows  Northern blot  analysis  of the  constructs  of Figure   36  and<br>
demonstrates that intron 1 and intron 2 of the CPC transcript can be used to produce artificial<br>
miRNAs.<br>
[0052]      Figure 38 shows constructs in which pre-miR-HC-Pro159a is placed in either intron 1<br>
or intron 2 and pre-miR-P69l59a is placed in either intron 2 or intron 1 of the CPC gene.<br>
[0053]      Figure  39  shows  Northern blot  analysis  of the  constructs  of Figure  38  and<br>
demonstrates that it is possible to use CPC introns to produce two different artificial miRNAs<br>
simultaneously in one transcript.<br>
DETAILED DESCRIPTION<br>
[0054] Recently discovered small RNAs play an important role in controlling gene<br>
expression. Regulation of many developmental processes including flowering is controlled by<br>
small RNAs. It is now possible to engineer changes in gene expression of plant genes by using<br>
transgenic constructs which produce small RNAs in the plant.<br>
[0055] The invention provides methods and compositions useful for suppressing targeted<br>
sequences. The compositions can be employed in any type of plant cell, and in other cells which<br>
comprise the appropriate processing components (e.g., RNA interference components),<br>
including invertebrate and vertebrate animal cells. The compositions and methods are based on<br>
an endogenous miRNA silencing process discovered in Arabidopsis, a similar strategy can be<br>
used to extend the number of compositions and the organisms in which the methods are used.<br>
The methods can be adapted to work in any eukaryotic cell system. Additionally, the<br>
compositions and methods described herein can be used in individual cells, cells or tissue in<br>
culture, or in vivo in organisms, or in organs or other portions of organisms.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
12<br>
[0056] The compositions selectively suppress the target sequence by encoding a miRNA<br>
having substantial complementarity to a region of the target sequence. The miRNA is provided<br>
in a nucleic acid construct which, when transcribed into RNA, is predicted to form a hairpin<br>
structure which is processed by the cell to generate the miRNA, which then suppresses<br>
expression of the target sequence.<br>
[0057] A nucleic acid construct is provided to encode the miRNA for any specific target<br>
sequence. Any miRNA can be inserted into the construct, such that the encoded miRNA<br>
selectively targets and suppresses the target sequence.<br>
[0058] A method for suppressing a target sequence is provided. The method employs the<br>
constructs above, in which a miRNA is designed to a region of the target sequence, and inserted<br>
into the construct. Upon introduction into a cell, the miRNA produced suppresses expression of<br>
the targeted sequence. The target sequence can be an endogenous plant sequence, or a<br>
heterologous transgene in the plant. The target gene may also be a gene from a plant pathogen,<br>
such as a pathogenic virus, nematode, insect, or mold or fungus.<br>
[0059] A plant, cell, and seed comprising the construct and/or the miRNA is provided.<br>
Typically, the cell will be a cell from a plant, but other eukaryotic cells are also contemplated,<br>
including but not limited to yeast, insect, nematode, or animal cells. Plant cells include cells<br>
from monocots and dicots. The invention also provides plants and seeds comprising the<br>
construct and/or the miRNA. Viruses and prokaryotic cells comprising the construct are also<br>
provided.<br>
[0060] Units, prefixes, and symbols may be denoted in their SI accepted form. Unless<br>
otherwise indicated, nucleic acids are written left to right in 5' to 3' orientation; amino acid<br>
sequences are written left to right in amino to carboxyl orientation, respectively. Numeric<br>
ranges recited within the specification are inclusive of the numbers defining the range and<br>
include each integer within the defined range. Amino acids may be referred to herein by either<br>
commonly known three letter symbols or by the one-letter symbols recommended by the<br>
IUPAC-IUB Biochemical Nomenclature Commission. Nucleotides, likewise, may be referred to<br>
by their commonly accepted single-letter codes. Unless otherwise provided for, software,<br>
electrical, and electronics terms as used herein are as defined in The New IEEE Standard<br>
Dictionary of Electrical and Electronics Terms (5th edition, 1993). The terms defined below are<br>
more fully defined by reference to the specification as a whole.<br>
[0061] As used herein, "nucleic acid construct" or "construct" refers to an isolated<br>
polynucleotide which is introduced into a host cell.    This construct may comprise any<br><br>
WO 2006/044322	PCT/US2005/036373<br>
13<br>
combination of deoxyribonucleotides, ribonucleotides, and/or modified nucleotides. The<br>
construct may be transcribed to form an RNA, wherein the RNA may be capable of forming a<br>
double-stranded RNA and/or hairpin structure. This construct may be expressed in the cell, or<br>
isolated or synthetically produced. The construct may further comprise a promoter, or other<br>
sequences which facilitate manipulation or expression of the construct.<br>
[0062] As used here "suppression" or "silencing" or "inhibition" are used interchangeably to<br>
denote the down-regulation of the expression of the product of a target sequence relative to its<br>
normal expression level in a wild type organism. Suppression includes expression that is<br>
decreased by about 10%, 15%, 20%, 25%, 30%, 35%, 40%, 45%, 50%, 55%, 60%, 65%, 70%,<br>
75%, 80%, 85%, 90%, 95%, or 100% relative to the wild type expression level.<br>
[0063] As used herein, "encodes" or "encoding" refers to a DNA sequence which can be<br>
processed to generate an RNA and/or polypeptide.<br>
[0064] As used herein, "expression" or "expressing" refers to the generation of an RNA<br>
transcript from an introduced construct, an endogenous DNA sequence, or a stably incorporated<br>
heterologous DNA sequence. The term may also refer to a polypeptide produced from an<br>
mRNA generated from any of the above DNA precursors.<br>
[0065] As used herein, "heterologous" in reference to a nucleic acid is a nucleic acid that<br>
originates from a foreign species, or is synthetically designed, or, if from the same species, is<br>
substantially modified from its native form in composition and/or genomic locus by deliberate<br>
human intervention. A heterologous protein may originate from a foreign species or, if from the<br>
same species, is substantially modified from its original form by deliberate human intervention.<br>
[0066] By "host cell" is meant a cell which contains an introduced nucleic acid construct and<br>
supports the replication and/or expression of the construct. Host cells may be prokaryotic cells<br>
such as E. coli, or eukaryotic cells such as fungi, yeast, insect, amphibian, nematode, or<br>
mammalian cells. Alternatively, the host cells are monocotyledonous or dicotyledonous plant<br>
cells. An example of a monocotyledonous host cell is a maize host cell.<br>
[0067] The term "introduced" means providing a nucleic acid or protein into a cell.<br>
Introduced includes reference to the incorporation of a nucleic acid into a eukaryotic or<br>
prokaryotic cell where the nucleic acid may be incorporated into the genome of the cell, and<br>
includes reference to the transient provision of a nucleic acid or protein to the cell. Introduced<br>
includes reference to stable or transient transformation methods, as well as sexually crossing.<br>
[0068] The term "isolated" refers to material, such as a nucleic acid or a protein, which is: (1)<br>
substantially or essentially free from components which normally accompany or interact with<br><br>
WO 2006/044322	PCT/US2005/036373<br>
14<br>
the material as found in its naturally occurring environment or (2) if the material is in its natural<br>
environment, the material has been altered by deliberate human intervention to a composition<br>
and/or placed at a locus in the cell other than the locus native to the material.<br>
[0069] As used herein, "miRNA" refers to an oligoribonucleic acid, which suppresses<br>
expression of a polynucleotide comprising the target sequence transcript or down regulates a<br>
target RNA. A "miRNA precursor" refers to a larger polynucleotide which is processed to<br>
produce a mature miRNA, and includes a DNA which encodes an RNA precursor, and an RNA<br>
transcript comprising the miRNA. A "mature miRNA" refers to the miRNA generated from the<br>
processing of a miRNA precursor. A "miRNA template" is an oligonucleotide region, or<br>
regions, in a nucleic acid construct which encodes the miRNA. The "backside" region of a<br>
miRNA is a portion of a polynucleotide construct which is substantially complementary to the<br>
miRNA template and is predicted to base pair with the miRNA template. The miRNA template<br>
and backside may form a double-stranded polynucleotide, including a hairpin structure. As is<br>
known for natural miRNAs, the mature miRNA and its complements may contain mismatches<br>
and form bulges and thus do not need to be fully complementary.<br>
[0070] As used herein, the phrases "target sequence" and "sequence of interest" are used<br>
interchangeably. Target sequence is used to mean the nucleic acid sequence that is selected for<br>
suppression of expression, and is not limited to polynucleotides encoding polypeptides. The<br>
target sequence comprises a sequence that is substantially or completely complementary to the<br>
miRNA. The target sequence can be RNA or DNA, and may also refer to a polynucleotide<br>
comprising the target sequence.<br>
[0071] As used herein, "nucleic acid" means a polynucleotide and includes single or double-<br>
stranded polymer of deoxyribonucleotide or ribonucleotide bases. Nucleic acids may also<br>
include fragments and modified nucleotides.<br>
[0072] By "nucleic acid library" is meant a collection of isolated DNA or RNA molecules<br>
which comprise and substantially represent the entire transcribed fraction of a genome of a<br>
specified organism or of a tissue from that organism. Construction of exemplary nucleic acid<br>
libraries, such as genomic and cDNA libraries, is taught in standard molecular biology<br>
references such as Berger and Kimmel, Guide to Molecular Cloning Techniques, Methods in<br>
Erizymology, Vol. 152, Academic Press, Inc., San Diego, CA (Berger); Sambrook et al.,<br>
Molecular Cloning - A Laboratory Manual, 2nd ed., Vol. 1-3 (1989); and Current Protocols in<br>
Molecular Biology, F.M. Ausubel et al., Eds., Current Protocols, a joint venture between Greene<br>
Publishing Associates, Inc. and John Wiley &amp; Sons, Inc. (1994).<br><br>
WO 2006/044322	PCT/US2005/036373<br>
15<br>
[0073] As used herein "operably linked" includes reference to a functional linkage of at least<br>
two sequences. Operably linked includes linkage between a promoter and a second sequence,<br>
wherein the promoter sequence initiates and mediates transcription of the DNA sequence<br>
corresponding to the second sequence.<br>
[0074] As used herein, "plant" includes plants and plant parts including but not limited to<br>
plant cells, plant tissue such as leaves, stems, roots, flowers, and seeds.<br>
[0075] As used herein, "polypeptide" means proteins, protein fragments, modified proteins,<br>
amino acid sequences and synthetic amino acid sequences. The polypeptide can be glycosylated<br>
or not.<br>
[0076] As used herein, "promoter" includes reference to a region of DNA that is involved in<br>
recognition and binding of an RNA polymerase and other proteins to initiate transcription.<br>
[0077] The term "selectively hybridizes" includes reference to hybridization, under stringent<br>
hybridization conditions, of a nucleic acid sequence to a specified nucleic acid target sequence<br>
to a detectably greater degree (e.g., at least 2-fold over background) than its hybridization to<br>
non-target nucleic acid sequences and to the substantial exclusion of non-target nucleic acids.<br>
Selectively hybridizing sequences typically have about at least 80% sequence identity, or 90%<br>
sequence identity, up to and including 100% sequence identity (i.e., fully complementary) with<br>
each other.<br>
[0078] The term "stringent conditions" or "stringent hybridization conditions" includes<br>
reference to conditions under which a probe will selectively hybridize to its target sequence.<br>
Stringent conditions are sequence-dependent and will be different in different circumstances.<br>
By controlling the stringency of the hybridization and/or washing conditions, target sequences<br>
can be identified which are 100% complementary to the probe (homologous probing).<br>
Alternatively, stringency conditions can be adjusted to allow some mismatching in sequences so<br>
that lower degrees of similarity are detected (heterologous probing). Generally, a probe is less<br>
than about 1000 nucleotides in length, optionally less than 500 nucleotides in length.<br>
[0079] Typically, stringent conditions will be those in which the salt concentration is less<br>
than about 1.5 M Na ion, typically about 0.01 to 1.0 M Na ion concentration (or other salts) at<br>
pH 7.0 to 8.3 and the temperature is at least about 30°C for short probes (e.g., 10 to 50<br>
nucleotides) and at least about 60°C for long probes (e.g., greater than 50 nucleotides).<br>
Stringent conditions may also be achieved with the addition of destabilizing agents such as<br>
formamide. Exemplary low stringency conditions include hybridization with a buffer solution<br>
of 30 to 35% formamide, 1 M NaCl, 1% SDS (sodium dodecyl sulphate) at 37°C, and a wash in<br><br>
WO 2006/044322	PCT/US2005/036373<br>
16<br>
IX to 2X SSC (20X SSC = 3.0 M NaCl/0.3 M trisodium citrate) at 50 to 55°C. Exemplary<br>
moderate stringency conditions include hybridization in 40 to 45% formamide, 1 M NaCl, 1%<br>
SDS at 37°C, and a wash in 0.5X to IX SSC at 55 to 60°C. Exemplary high stringency<br>
conditions include hybridization in 50% formamide, 1 M NaCl, 1% SDS at 37°C, and a wash in<br>
0.1X SSC at 60 to 65°C.<br>
[0080] Specificity is typically the function of post-hybridization washes, the critical factors<br>
being the ionic strength and temperature of the final wash solution. For DNA-DNA hybrids, the<br>
Tm can be approximated from the equation of Meinkoth and Wahl ((1984) Anal Biochem<br>
138:267-284): Tm = 81.5 °C + 16.6 (log M) + 0.41 (%GC) - 0.61 (% form) - 500/L; where M is<br>
the molarity of monovalent cations, %GC is the percentage of guanosine and cytosine<br>
nucleotides in the DNA, % form is the percentage of formamide in the hybridization solution,<br>
and L is the length of the hybrid in base pairs. The Tm is the temperature (under defined ionic<br>
strength and pH) at which 50% of a complementary target sequence hybridizes to a perfectly<br>
matched probe. Tm is reduced by about 1°C for each 1% of mismatching; thus, Tm,<br>
hybridization and/or wash conditions can be adjusted to hybridize to sequences of the desired<br>
identity. For example, if sequences with &gt;90% identity are sought, the Tm can be decreased<br>
10°C. Generally, stringent conditions are selected to be about 5°C lower than the thermal<br>
melting point (Tm) for the specific sequence and its complement at a defined ionic strength and<br>
pH. However, severely stringent conditions can utilize a hybridization and/or wash at 1, 2, 3, or<br>
4 °C lower than the thermal melting point (Tm); moderately stringent conditions can utilize a<br>
hybridization and/or wash at 6, 7, 8, 9, or 10 °C lower than the thermal melting point (Tm); low<br>
stringency conditions can utilize a hybridization and/or wash at 11, 12, 13, 14, 15, or 20 °C<br>
lower than the thermal melting point (Tm). Using the equation, hybridization and wash<br>
compositions, and desired Tm, those of ordinary skill will understand that variations in the<br>
stringency of hybridization and/or wash solutions are inherently described. If the desired degree<br>
of mismatching results in a Tm of less than 45 °C (aqueous solution) or 32 °C (formamide<br>
solution) it is preferred to increase the SSC concentration so that a higher temperature can be<br>
used. An extensive guide to the hybridization of nucleic acids is found in Tijssen, Laboratory<br>
Techniques in Biochemistry and Molecular Biology—Hybridization with Nucleic Acid Probes,<br>
Part I, Chapter 2 "Overview of principles of hybridization and the strategy of nucleic acid probe<br>
assays", Elsevier, New York (1993); and Current Protocols in Molecular Biology, Chapter 2,<br>
Ausubel   et   al.,   Eds.,   Greene   Publishing   and   Wiley-Interscience,   New   York   (1995).<br><br>
WO 2006/044322	PCT/US2005/036373<br>
17<br>
Hybridization and/or wash conditions can be applied for at least 10, 30, 60, 90, 120, or 240<br>
minutes.<br>
[0081) As used herein, "transgenic" includes reference to a plant or a cell which comprises a<br>
heterologous polynucleotide. Generally, the heterologoas polynucleotide is stably integrated<br>
within the genome such that the polynucleotide is passed on to successive generations.<br>
Transgenic is used herein to include any cell, cell line, callus, tissue, plant part or plant, the<br>
genotype of which has been altered by the presence of heterologous nucleic acid including those<br>
transgenics initially so altered as well as those created by sexual crosses or asexual propagation<br>
from the initial transgenic. The term "transgenic" as used herein does not encompass the<br>
alteration of the genome (chromosomal or extra-chromosomal) by conventional plant breeding<br>
methods or by naturally occurring events such as random cross-fertilization, non-recombinant<br>
viral infection, non-recombinant bacterial transformation, non-recombinant transposition, or<br>
spontaneous mutation.<br>
[0082] As used herein, "vector" includes reference to a nucleic acid used in introduction of a<br>
polynucleotide of the invention into a host cell. Expression vectors permit transcription of a<br>
nucleic acid inserted therein.<br>
[0083] Polynucleotide sequences may have substantial identity, substantial homology, or<br>
substantial complementarity to the selected region of the target gene. As used herein<br>
"substantial identity" and "substantial homology" indicate sequences that have sequence identity<br>
or homology to each other. Generally, sequences that are substantially identical or substantially<br>
homologous will have about 75%, 80%, 85%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%,<br>
98%, 99%, or 100% sequence identity wherein the percent sequence identity is based on the<br>
entire sequence and is determined by GAP alignment ixsing default parameters (GCG, GAP<br>
version 10, Accelrys, San Diego, CA). GAP uses the algorithm of Needleman and Wunsch<br>
((1970) J Mol Biol 48:443-453) to find the alignment of two complete sequences that maximizes<br>
the number of matches and minimizes the number of sequence gaps. Sequences which have<br>
100% identity are identical. "Substantial complementarity" refers to sequences that are<br>
complementary to each other, and are able to base pair with each other. In describing<br>
complementary sequences, if all the nucleotides in the first sequence will base pair to the second<br>
sequence, these sequences are fully complementary.<br>
[0084] Through a forward genetics approach, a microRNA that confers a developmental<br>
phenotype in Arabidopsis was identified. This miRNA, miR l72a-2 (Park et al. (2002) Curr Biol<br>
12:1484-1495), causes early flowering and defects in floral organ identity when overexpressed.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
18<br>
The predicted target of miR172a-2 is a small subfamily of APETALA2-Iike transcription factors<br>
(Okamuro et al. (1997) Proc Natl Acad Sci USA 94:7076-7081). Overexpre ssion of miR172a-2<br>
downregulates at least one member of this family. In addition, overexpTe ssion of one of the<br>
AP2-like target genes, At2g28550, causes late flowering. This result, in conjunction with loss-<br>
of-function analyses of At2g28550 and another target gene, At5g60120, indicates that at least<br>
some of the AP2-Iike genes targeted by miR172a-2 normally function as floral repressors. The<br>
EAT-D line overexpressing miRl 72-a2 has a wild-type response to photoperiod. The genomic<br>
region encoding the miRNA was also identified (SEQ ID NO:1) and used to produce a cassette<br>
into which other miRNAs to target sequences can be inserted (SEQ ID NO:3 ), and to produce an<br>
expression vector (SEQ ID NO:44) useful for cloning the cassettes and expxessing the miRNA.<br>
The expression vector comprises the 1.4kb region encoding the miRNA. Expression of this<br>
region is processed in the cell to produce the miRNA which suppresses expression of the target<br>
gene. Alternatively, the miRNA may be synthetically produced and introduced to the cell<br>
directly.<br>
[0085] In one embodiment, there is provided a method for the suppression of a target<br>
sequence comprising introducing into a cell a nucleic acid construct encoding a miRNA<br>
substantially complementary to the taTget. In some embodiments the miRNA comprises about<br>
10-200 nucleotides, about 10-15, 15-20, 19, 20, 21, 22, 23, 24, 25, 26, 27, 25-30, 30-50, 50-100,<br>
100-150, or about 150-200 nucleotides. In some embodiments the nucleic acid construct<br>
encodes the miRNA. In some embodiments the nucleic acid construct encodes a polynucleotide<br>
precursor which may form a double-stranded RNA, or hairpin structure coirxprising the miRNA.<br>
In some embodiments, nucleotides 39-59 and 107-127 of SEQ ID NO:3 are replaced by the<br>
backside of the miRNA template and the miRNA template respectively. In some embodiments,<br>
this new sequence replaces the equivalent region of SEQ ID NO: 1. In further embodiments, this<br>
new sequence replaces the equivalent region of SEQ ID NO:44.<br>
[0086] In some embodiments, the nucleic acid construct comprises a modified endogenous<br>
plant miRNA precursor, wherein the precursor has been modified to replace the endogenous<br>
miRNA encoding regions with sequences designed to produce a miRNA directed to the target<br>
sequence. In some embodiments the miRNA precursor template is a miR l72a miRNA<br>
precursor. In some embodiments, the miRl72a precursor is from a dicot or a monocot. In some<br>
embodiments the miR l72a precursor is from Arabidopsis thaliana, tomato, soybean, rice, or<br>
corn. In some embodiments the miRNA precursor is SEQ ID NO:1, SEQ YD NO:3, or SEQ ID<br>
NO:44.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
19<br>
(0087]      In another embodiment the method comprises:<br>
[0088]      A method of inhibiting expression of a target sequence in a cell comprising:<br>
(a)	introducing into the cell a nucleic acid construct comprising a promoter operably<br>
linked to a polynucleotide, wherein the polynucleotide comprises in the following order:<br>
(i) at least about 20 and up to 38 contiguous nucleotides in the region of<br>
nucleotides 1-38 of SEQ ID NO:3,<br>
(ii) a first oligonucleotide of 10 to about 50 contiguous nucleotides, wherein<br>
the first oligonucleotide is substantially complementary to a second oligonucleotide<br>
(iii) at least about 20 and up to 47 contiguous nucleotides in the region of<br>
nucleotides 60-106 of SEQ ID NO:3,<br>
(iv) the second oligonucleotide of about 10 to about 50 contiguous<br>
nucleotides, wherein the second oligonucleotide encodes a miRNA, and the second<br>
oligonucleotide is substantially complementary to the target sequence, and<br>
(v)       at least about 20 and up to 32 contiguous nucleotides in the region of<br>
nucleotides 128-159 of SEQ ID NO:3;<br>
wherein the polynucleotide encodes an RNA precursor capable of forming a hairpin, and<br>
(b)	expressing the nucleic acid construct for a time sufficient to produce the miRNA,<br>
wherein the miRNA inhibits expression of the target sequence.<br>
[0089] In another embodiment, the method comprises selecting a target sequence of a gene,<br>
and designing a nucleic acid construct comprising polynucleotide encoding a miKNA<br>
substantially complementary to the target sequence. In some embodiments, the target sequence<br>
is selected from any region of the gene. In some embodiments, the target sequence is selected<br>
from an untranslated region. In some embodiments, the target sequence is selected from a<br>
coding region of the gene. In some embodiments, the target sequence is selected from a region<br>
about 50 to about 200 nucleotides upstream from the stop codon, including regions from about<br>
50-75, 75-100, 100-125, 125-150, or 150-200 upstream from the stop codon. In further<br>
embodiments, the target sequence and/or the miRNA is based on the polynucleotides and<br>
process of EAT suppression of Apetela2-like genes in Arabidopsis thaliana. In some<br>
embodiments, nucleotides 39-59 and 107-127 of SEQ ID NO.3 are replaced by the backside of<br>
the miRNA template (first oligonucleotide) and the miRNA template (second oligonucleotide)<br>
respectively. In some embodiments, this new sequence replaces the equivalent region of SEQ<br>
ID NO:1. In further embodiments, this new sequence replaces the equivalent region of SEQ ID<br>
NO:44.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
20<br>
[0090] In some embodiments, the miRNA template, (i.e. the polynucleotide encoding the<br>
miRNA), and thereby the miRNA, may comprise some mismatches relative to the target<br>
sequence. In some embodiments the miRNA template has &gt; 1 nucleotide mismatch as compared<br>
to the target sequence, for example, the miRNA template can have 1, 2, 3, 4, 5, or more<br>
mismatches as compared to the target sequence. This degree of mismatch may also be<br>
described by determining the percent identity of the miRNA template to the complement of the<br>
target sequence. For example, the miRNA template may have a percent identity including about<br>
at least 70%, 75%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%,<br>
90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or 100% as compared to the<br>
complement of the target sequence.<br>
[0091] In some embodiments, the miRNA template, {i.e. the polynucleotide encoding the<br>
miRNA) and thereby the miRNA, may comprise some mismatches relative to the miRNA<br>
backside. In some embodiments the miRNA template has &gt; 1 nucleotide mismatch as compared<br>
to the miRNA backside, for example, the miRNA template can have 1, 2, 3, 4, 5, or more<br>
mismatches as compared to the miRNA backside. This degree of mismatch may also be<br>
described by determining the percent identity of the miRNA template to the complement of the<br>
miRNA backside. For example, the miRNA template may have a percent identity including<br>
about at least 70%, 75%, 77%, 78%, 79%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%,<br>
89%, 90%, 91%, 92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or 100% as compared to the<br>
complement of the miRNA backside.<br>
[0092] In some embodiments, the target sequence is selected from a plant pathogen. Plants<br>
or cells comprising a miRNA directed to the target sequence of the pathogen are expected to<br>
have decreased sensitivity and/or increased resistance to the pathogen. In some embodiments,<br>
the miRNA is encoded by a nucleic acid construct further comprising an operably linked<br>
promoter. In some embodiments, the promoter is a pathogen-inducible promoter.<br>
[0093] In another embodiment, the method comprises replacing the miRNA encoding<br>
sequence in the polynucleotide of SEQ ID NO: 3 with a sequence encoding a miRNA<br>
substantially complementary to the target region of the target gene.<br>
[0094] In another embodiment a method is provided comprising a method of inhibiting<br>
expression of a target sequence in a cell comprising:<br>
(a) introducing into the cell a nucleic acid construct comprising a promoter operably<br>
linked to a polynucleotide encoding a modified plant miRNA precursor comprising a first and a<br>
second oligonucleoride, wherein at least one of the first or the second oligonucleotides is<br><br>
WO 2006/044322	PCT/US2005/036373<br>
21<br>
heterologous to the precursor, wherein the first oligonucleotide is substantially complementary<br>
to the second oligonucleotide, and the second oligonucleotide encodes a miRNA substantially<br>
complementary to the target sequence, wherein the precursor is capable of forming a hairpin;<br>
and<br>
(b)       expressing the nucleic acid construct for a time sufficient to produce the miKNA,<br>
wherein the miRNA inhibits expression of the target sequence.<br>
[0095] In another embodiment a method is provided comprising a method of inhibiting<br>
expression of a target sequence in a cell comprising:<br>
(a)	introducing into the cell a nucleic acid construct comprising a promoter operably<br>
linked to a polynucleotide encoding a modified plant miR172 miRNA precursor comprising a<br>
first  and   a   second   oligonucleotide,   wherein  at   least   one   of the   first   or  the   second<br>
oligonucleotides   is  heterologous   to  the  precursor,   wherein  the   first   oligonucleotide   is<br>
substantially complementary to the second oligonucleotide, and the second oligonucleotide<br>
encodes a miRNA substantially complementary to the target sequence, wherein the precursor is<br>
capable of forming a hairpin; and<br>
(b)	expressing the nucleic acid construct for a time sufficient to produce the miRNA,<br>
wherein the miRNA inhibits expression of the target sequence.<br>
[0096] In some embodiments, the modified plant miR172 miRNA precursor is a modified<br>
Arabidopsis miR172 miRNA precursor, or a modified corn miR172 miRNA precursor or the<br>
like.<br>
[0097] In another embodiment, there is provided a nucleic acid construct for suppressing a<br>
target sequence. The nucleic acid construct encodes a miRNA substantially complementary to<br>
the target. In some embodiments, the nucleic acid construct further comprises a promoter<br>
operably linked to the polynucleotide encoding the miRNA. In some embodiments, the nucleic<br>
acid construct lacking a promoter is designed and introduced in such a way that it becomes<br>
operably linked to a promoter upon integration in the host genome. In some embodiments, the<br>
nucleic acid construct is integrated using recombination, including site-specific recombination.<br>
See, for example, PCT International published application No. WO 99/25821, herein<br>
incorporated by reference. In some embodiments, the nucleic acid construct is an RNA. In<br>
some embodiments, the nucleic acid construct comprises at least one recombination site,<br>
including site-specific recombination sites. In some embodiments the nucleic acid construct<br>
comprises at least one recombination site in order to facilitate integration, modification, or<br>
cloning of the construct.  In some embodiments the nucleic acid construct comprises two site-<br><br>
WO 2006/044322	PCT/US2005/036373<br>
22<br>
specific recombination sites flanking the miRNA precursor. In some embodiments the site-<br>
specific recombination sites include FRT sites, lox sites, or art sites, including attB, attL, attP or<br>
attR sites. See, for example, PCT International published application No. WO 99/25821, and<br>
U.S. Patents 5,888,732, 6,143,557, 6,171,861, 6,270,969, and 6,277,608, herein incorporated by<br>
reference.<br>
[0098] In some embodiments, the nucleic acid construct comprises a modified endogenous<br>
plant miRNA precursor, wherein the precursor has been modified to replace, the miRNA<br>
encoding region with a sequence designed to produce a miRNA directed to the target sequence.<br>
In some embodiments the miRNA precursor template is a miR172a miRNA precursor. In some<br>
embodiments, the miRl 72a precursor is from a dicot or a monocot. In some embodiments the<br>
miR172a precursor is from Arabidopsis thaliana, tomato, soybean, rice, or com. In some<br>
embodiments the miRNA precursor is SEQ ID NO:1, SEQ ID NO:3, or SEQ ID NO:44.<br>
[0099] In another embodiment, the nucleic acid construct comprises an isolated<br>
polynucleotide comprising a polynucleotide which encodes a modified plant miRNA precursor,<br>
the modified precursor comprising a first and a second oligonucleotide, wherein at least one of<br>
the first or the second oligonucleotides is heterologous to the precursor, wherein the first<br>
oligonucleotide i is substantially complementary to the second oligonucleotide, and the second<br>
oligonucleotide comprises a miRNA substantially complementary to the target sequence,<br>
wherein the precursor is capable of forming a hairpin.<br>
[0100] In another embodiment, the nucleic acid construct comprises an isolated<br>
polynucleotide comprising a polynucleotide which encodes a modified plant miRl 72 miRNA<br>
precursor, the modified precursor comprising a first and a second oligonucleotide, wherein at<br>
least one of the first or the second oligonucleotides is heterologous to the precursor, wherein the<br>
first oligonucleotide is substantially complementary to the second oligonucleotide, and the<br>
second oligonucleotide comprises a miRNA substantially complementary to the target sequence,<br>
wherein the precursor is capable of forming a hairpin. In some embodiments, the modified plant<br>
miR l72 miRNA precursor is a modified Arabidopsis miRl 72 miRNA precursor, or a modified<br>
corn miR l72 miRNA precursor, or the like.<br>
[0101] In some embodiments the miRNA comprises about 10-200 nucleotides, about 10-15,<br>
15-20, 19, 20, 21, 22, 23, 24, 25, 26, 27, 25-30, 30-50, 50-100, 100-150, or about 150-200<br>
nucleotides. In some embodiments the nucleic acid construct encodes the miRNA. In some<br>
embodiments the nucleic acid construct encodes a polynucleotide precursor which may form a<br>
double-stranded RNA, or hairpin structure comprising the miRNA.   In some embodiments,<br><br>
WO 2006/044322	PCT/US2005/036373<br>
23<br>
nucleotides 39-59 and/or 107-127 of SEQ ID NO:3 are replaced by the backside of the miRNA<br>
template and the miRNA template respectively. In some embodiments, this new sequence<br>
replaces the equivalent region of SEQ ID NO: 1. In further embodiments, this new sequence<br>
replaces the equivalent region of SEQ ID NO:44. In some embodiments, the target region is<br>
selected from any region of the target sequence. In some embodiments, the target region is<br>
selected from a untranslated region. In some embodiments, the target region is selected from a<br>
coding region of the target sequence. In some embodiments, the target region is selected from a<br>
region about 50 to about 200 nucleotides upstream from the stop codon, including regions from<br>
about 50-75, 75-100, 100-125, 125-150, or 150-200 upstream from the stop codon. In further<br>
embodiments, the target region and/or the miRNA is based on the polynucleotides and process<br>
of EAT suppression of Apetela2-Iike sequences in Arabidopsis thaliana.<br>
[0102] In another embodiment the nucleic acid construct comprises an isolated<br>
polynucleotide comprising in the following order at least 20 and up to 38 contiguous nucleotides<br>
in the region from nucleotides 1-38 of SEQ ID NO.3, a first oligonucleotide of about 10 to about<br>
50 contiguous nucleotides, wherein the first oligonucleotide is substantially complementary to a<br>
second oligonucleotide, at least about 20 and up to 47 contiguous nucleotides in the region from<br>
nucleotides 60-106 of SEQ ID NO:3, a second oligonucleotide of about 10 to about 50<br>
contiguous nucleotides, wherein the second oligonucleotide encodes a miRNA., and the second<br>
oligonucleotide is substantially complementary to the target sequence, and at least about 20 and<br>
up to 32 contiguous nucleotides in the region from nucleotides 128-159 of SEQ ID NO:3,<br>
wherein the polynucleotide encodes an RNA precursor capable of forming a hairpin structure.<br>
[0103] In some embodiments there are provided cells, plants, and seeds comprising the<br>
introduced polynucleotides, and/or produced by the methods of the invention. The cells include<br>
prokaryotic and eulcaryotic cells, including but not limited to bacteria, yeast, fungi, viral,<br>
invertebrate, vertebrate, and plant cells. Plants, plant cells, and seeds of the invention include<br>
gynosperms, monocots and dicots, including but not limited to, for example, rice, wheat, oats,<br>
barley, millet, sorghum, soy, sunflower, safflower, canola, alfalfa, cotton, Arabidopsis, and<br>
tobacco.<br>
[0104] In some embodiments, the cells, plants, and/or seeds comprise a nucleic acid construct<br>
comprising a modified plant miRNA precursor, wherein the precursor has "been modified to<br>
replace the endogenous miRNA encoding regions with sequences designed to produce a miRNA<br>
directed to the target sequence. In some embodiments the miRNA precursor template is a<br>
miR172a miRNA precursor. In some embodiments, the miRl72a precursor is from a dicot or a<br><br>
WO 2006/044322	PCT/US2005/036373<br>
24<br>
monocot. In some embodiments the miRl 72a precursor is from Arabidopsis thaliana, tomato,<br>
soybean, rice, or corn. In some embodiments the miRNA precursor is SEQ ID NO:1, SEQ ID<br>
NO:3, or SEQ ID NO:44. In some embodiments the miRNA precursor is encoded by SEQ ID<br>
NO:1, SEQ ID NO:3, or SEQ ID NO:44. In some embodiments, the nucleic acid construct<br>
comprises at least one recombination site, including site-specific recombination sites. In some<br>
embodiments the nucleic acid construct comprises at least one recombination site in order to<br>
facilitate modification or cloning of the construct. In some embodiments the nucleic acid<br>
construct comprises two site-specific recombination sites flanking the miRNA precursor. In<br>
some embodiments the site-specific recombination sites include FRT sites, lox sites, or art sites,<br>
including attB, attL, attP or attR sites. See, for example, PCT International published application<br>
No. WO 99/25821, and U.S. Patents 5,888,732, 6,143,557, 6,171,861, 6,270,969, and 6,277,608,<br>
herein incorporated by reference.<br>
[0105] In a further embodiment, there is provided a method for down regulating a target<br>
RNA comprising introducing into a cell a nucleic acid construct that encodes a miRNA that is<br>
complementary to a region of the target RNA. In some embodiments, the miRNA is fully<br>
complementary to the region of the target RNA. In some embodiments, the miRNA is<br>
complementary and includes the use of G-U base pairing, i.e. the GU wobble, to otherwise be<br>
fully complementary. In some embodiments, the first ten nucleotides of the miRNA (counting<br>
from the 5' end of the miRNA) are fully complementary to a region of the target RNA and the<br>
remaining nucleotides may include mismatches and/or bulges with the target RNA. In some<br>
embodiments the miRNA comprises about 10-200 nucleotides, about 10-15, 15-20, 19, 20, 21,<br>
22, 23, 24, 25, 26, 27, 25-30, 30-50, 50-100, 100-150, or about 150-200 nucleotides. The<br>
binding of the miRNA to the complementary sequence in the target RNA results in cleavage of<br>
the target RNA. In some embodiments, the miRNA is a miRNA that has been modified such<br>
that the miRNA is fully complementary to the target sequence of the target RNA. In some<br>
embodiments, the miRNA is an endogenous plant miRNA that has been modified such that the<br>
miRNA is fully complementary to the target sequence of the target RNA. In some<br>
embodiments, the polynucleotide encoding the miRNA is operably linked to a promoter. In<br>
some embodiments, the nucleic acid construct comprises a promoter operably linked to the<br>
miRNA.<br>
[0106] In some embodiments, the nucleic acid construct encodes the miRNA. In some<br>
embodiments, the nucleic acid construct comprises a promoter operably linked to the miRNA.<br>
In some embodiments, the nucleic acid construct encodes a polynucleotide which may form a<br><br>
WO 2006/044322	PCT/US2005/036373<br>
25<br>
double-stranded RNA, or hairpin structure comprising the miRNA. In some embodiments, the<br>
nucleic acid construct comprises a promoter operably linked to the polynucleotide which may<br>
form a double-stranded RNA, or hairpin structure comprising the miRNA. In some<br>
embodiments, the nucleic acid construct comprises an endogenous plant miRNA precursor that<br>
has been modified such that the miRNA is fully complementary to the target sequence of the<br>
target RNA. In some embodiments, the nucleic acid construct comprises a promoter operably<br>
linked to the miRNA precursor. In some embodiments, the nucleic acid construct comprises<br>
about 50 nucleotides to about 3000 nucleotides, about 50-100, 100-150, 150-200, 200-250, 250-<br>
300, 300-350, 350-400, 400-450, 450-500, 500-600, 600-700, 700-800, 800-900, 900-1000,<br>
1000-1100, 1100-1200, 1200-1300, 1300-1400, 1400-1500, 1500-1600, 1600-1700, 1700-1800,<br>
1800-1900, 1900-2000, 2000-2100, 2100-2200, 2200-2300, 2300-2400, 2400-2500, 2500-2600,<br>
2600-2700, 2700-2800, 2800-2900 or about 2900-3000 nucleotides.<br>
[0107] In some embodiments, the nucleic acid construct lacking a promoter is designed and<br>
introduced in such a way that it becomes operably linked to a promoter upon integration in the<br>
host genome. In some embodiments, the nucleic acid construct is integrated using<br>
recombination, including site-specific recombination. In some embodiments, the nucleic acid<br>
construct is an RNA. In some embodiments, the nucleic acid construct comprises at least one<br>
recombination site, including site-specific recombination sites. In some embodiments the<br>
nucleic acid construct comprises at least one recombination site in order to facilitate integration,<br>
modification, or cloning of the construct. In some embodiments the nucleic acid construct<br>
comprises two site-specific recombination sites flanking the miRNA precursor.<br>
[0108] In another embodiment, the method comprises a method for down regulating a target<br>
RNA in a cell comprising introducing into the cell a nucleic acid construct that encodes a<br>
miRNA that is complementary to a region of the target RNA arid expressing the nucleic acid<br>
construct for a time sufficient to produce miRNA, wherein the miRNA down regulates the target<br>
RNA. In some embodiments, the miRNA is fully complementary to the region of the target<br>
RNA. In some embodiments, the miRNA is complementary and includes the use of G-U base<br>
pairing, i.e. the GU wobble, to otherwise be fully complementary.<br>
[0109] In another embodiment, the method comprises selecting a target RNA, selecting a<br>
miRNA, comparing the sequence of the target RNA (or its DNA) with the sequence of the<br>
miRNA, identifying a region of the target RNA (or its DNA) in which the nucleotide sequence is<br>
similar to the nucleotide sequence of the miRNA, modifying the nucleotide sequence of the<br>
miRNA so that it is complementary to the nucleotide sequence of the identified region of the<br><br>
WO 2006/044322	PCT/US2005/036373<br>
26<br>
target RNA and preparing a nucleic acid construct comprising the modified miRNA. In some<br>
embodiments, the miRNA is fully complementary to the identified region of the target RNA. In<br>
some embodiments, the miRNA is complementary and includes the use of G-U base pairing, i.e.<br>
the GU wobble, to otherwise be fully complementary. In some embodiments, a nucleic acid<br>
construct encodes a polynucleotide which may form a double-stranded RNA, or hairpin structure<br>
comprising the miRNA. In some embodiments, a nucleic acid construct comprises a precursor<br>
of the miRNA, i.e., a pre-miRNA that has been modified in accordance with this embodiment.<br>
[0110] In another embodiment, the method comprises selecting a target RNA, selecting a<br>
nucleotide sequence within the target RNA, selecting a miRNA, modifying the sequence of the<br>
miRNA so that it is complementary to the nucleotide sequence of the identified region of the<br>
target RNA and preparing a nucleic acid construct comprising the modified miRNA. In some<br>
embodiments, the miRNA is fully complementary to the identified region of the target RNA. In<br>
some embodiments, the miRNA is complementary and includes the use of G-U base pairing, i.e.<br>
the GU wobble, to otherwise be fully complementary. In some embodiments, a nucleic acid<br>
construct encodes a polynucleotide which may form a double-stranded RNA, or hairpin structure<br>
comprising the miRNA. In some embodiments, a nucleic acid construct comprises a precursor<br>
of the miRNA, i.e., a pre-miRNA that has been modified in accordance with this embodiment.<br>
[0111] In some embodiments, the miRNA is a miRNA disclosed in the microRNA registry,<br>
now also known as the miRBase Sequence Database (Griffiths-Jones (2004) Nucl Acids Res 32,<br>
Database issue:D109-Dlll; http://microrna.sanger.ac.uk/). In some embodiments, the miRNA<br>
is ath-MIR156a, ath-MIR156b, ath-MIR156c, ath-MIR156d, ath-MIR156e3 ath-MDR.156f, ath-<br>
MIR156g, ath-MIR156h, ath-MER157a, ath-MIR157b, ath-MIR157c, ath-MIR157d, ath-<br>
MIR158a, ath-MIR158b, ath-MIR159a, ath-MIR159b, ath-MIR159c, ath-MIR160a, ath-<br>
MIR160b, ath-MER160c, ath-MIR161, ath-MIR162a, ath-MIRl 62b, ath-MDR.163, ath-MIR164a,<br>
ath-MIR164b, ath-MIR164c, ath-MIR165a, ath-MIR165b, ath-MIR166a, ath-MIR166b, ath-<br>
MIRl 66c, ath-MIRl 66d, ath-MIRl 66e, ath-MIRl 66f, ath-MIRl 66g, ath-MIRl 67a, ath-<br>
MIRl 67b, ath-MIRl 67c, ath-MIRl 67d, ath-MIRl68a, ath-MIRl 68b, ath-MIRl 69a, ath-<br>
MIR169b, ath-MIRl 69c, ath-MIRl 69d, ath-MIRl 69e, ath-MIRl 69f, ath-MIRl 69g, ath-<br>
MIRl 69h, ath-MIRl 69i, ath-MIRl 69j, ath-MIRl69k, ath-MIRl691, ath-MIRl69m, ath-<br>
MIR169n, ath-MIR.170, ath-MIR171a, ath-MIR171b, ath-MIR171c, ath-MIR172a, ath-<br>
MIR172b, ath-MIR172c, ath-MIR172d, ath-MIR172e, ath-MIR173, ath-MIR319a, ath-<br>
MIR319b, ath-MIR319c, ath-MIR390a, ath-MIR390b, ath-MIR393a, ath-MIR393b, ath-<br>
MIR394a,   ath-MIR394b,   ath-MIR395a,   ath-MIR395b,   ath-MIR395c,   ath-MIR395d,   ath-<br><br>
WO 2006/044322	PCT/US2005/036373<br>
27<br>
MIR395e, ath-MIR395f, ath-MIR396a, ath-MIR3 96b&gt;, ath-MIR397a, ath-MIR397b, ath-<br>
MIR398a, ath-MIR398b, ath-MIR398c, ath-MIR399a, ath-MIR399b, ath-MIR399c, ath-<br>
MIR399d, ath-MIR399e, ath-MIR399f, ath-MIR400, ath-MIR401, ath-MIR402, ath-MIR403,<br>
ath-MIR404, ath-MIR405a, ath-MIR405b, ath-MIR4O5d, ath-MIR406, ath-MIR407, ath-<br>
MIR408, ath-MIR413, ath-MIR414, ath-MIR415, ath-MIR416, ath-MIR417, ath-MIR418, ath-<br>
MIR419, ath-MR420, ath-MIR426, ath-MIR447a1 ath-MIR447b, ath-MIR447c, osa-MIR156a,<br>
osa-MIR156b, osa-MIR156c, osa-MIR156d, osa-MIR156e, osa-MIR156f, osa-MIR156g, osa-<br>
MIR156h, osa-MIRl 56i, osa-MIR156j5 osa-MIRl 56k, osa-MIRl 561, osa-MIRl 59a, osa-<br>
MIR159b, osa-MIR159c, osa-MIR159d, osa-MIRl 59&gt;e, osa-MIR159f, osa-MIRl 60a, osa-<br>
MIRl 60b, osa-MIR160c, osa-MIR160d, osa-MIRl 60 e, osa-MIRl 60f, osa-MIR162a, osa-<br>
MIR162b, osa-MIR164a, osa-MIR164b, osa-MIR164c, osa-MIR164d, osa-MIR164e, osa-<br>
MIR166a, osa-MIR166b, osa-MIR166c, osa-MIR166d, osa-MIR166e, osa-MIR166f, osa-<br>
MIR166J, osa-MIR166k, osa-MIR1661, osa-MIR166g, osa-MIR166h, osa-MIRl66i, osa-<br>
MIR166TI1, osa-MIRl 66n, osa-MIRl67a, osa-MIRl 67b, osa-MIRl67c, osa-MIRl 67d, osa-<br>
MIR167e, osa-MIR167f, osa-MIR167g, osa-MIR167h, osa-MIR167i, osa-MIR167j, osa-<br>
MIRl 68a, osa-MIRl 68b, osa-MIRl 69a, osa-MIRl 69"b, osa-MIRl69c, osa-MIRl 69d, osa-<br>
MIR169e, osa-MIR169f, osa-MIR169gs osa-MIR169h, osa-MIR169i, osa-MIR169j, osa-<br>
MIRl 69k, osa-MIRl 691, osa-MIRl 69m, osa-MIRl 69n, osa-MIRl 69o, osa-MIRl 69p, osa-<br>
MIR169q, osa-IR171a, osa-MIR171b, osa-MIR171c, osa-MTR171d, osa-MIR171e, osa-<br>
MER171f, osa-MIR171g, osa-MIR171h, osa-MIR171 i, osa-MIR172a, osa-MIR172b) osa-<br>
MIR172c, osa-MIR173d, osa-MIR390, osa-MIR319&gt;a, osa-MIR319b, osa-MIR393, osa-<br>
MIR393b, osa-MIR394, osa-MIR395b, osa-MIR395c, osa-MIR395d, osa-MIR395e, osa-<br>
MIR395g, osa-MIR395h, osa-MIR395i, osa-MIR395j, osa-MIR395k, osa-MIR3951, osa-<br>
MrR395m, osa-MIR395n, osa-MrR395o, osa-MIR395r, osa-MIR395q, osa-MIR395c5 osa-<br>
MIR395a, ,osa-MIR395f, osa-MIR395p, osa-MIR396a, osa-MIR396b, osa-MIR396c, osa-<br>
MIR397a, osa-MIR397b, osa-MIR398a, osa-MIR398T3, osa-MIR399a, osa-MTR399b, osa-<br>
MIR399c, osa-MIR399d, osa-MIR399e, osa-MIR399f, osa-MER399g, osa-MIR399h, osa-<br>
MIR399i, osa-MIR399j, osa-MIR399k, osa-MIR408, osa-MIR413, osa-MIR414, osa-MER.415,<br>
osa-MIR416, osa-MIR 417, osa-MIR418, osa-MIR419, osa-MIR426, osa-MIR437, osa-<br>
MIR439, osa-MIR439c, osa-MIR439d, osa-MIR438e, osa-MIR439f, osa-MIR439g, osa-<br>
MIR439h, osa-MIR440, osa-MIR441a, osa-MIR441c, osa-MIR442, osa-MIR443, osa-<br>
MIR445d, osa-MIR446, zma-MIR156a, zma-MIR156b, zma-MIRl 56c, zma-MIR156d, zma-<br>
MIR156e, 2ma-MIR156f, zma-MER.156g, zma-MIR156h, zma-MIR156i, zma-MIR156j, zma-<br><br>
WO 2006/044322	PCT/US2005/036373<br>
28<br>
MIR156k, zma-MIR159a, zma-MIR159b, zma-MIR159c, zma-MIR159d, zma-MIR160a, ana-<br>
MIR160b, zma-MIR160c, zma-MIR160d, zma-MIR160e, zma-MIR160f, zma-MIR 1611, znna-<br>
MIR162, zma-MIRl64a, zma-MIR164b, zrna-MIRl64c, zma-MIRl 64d, zma-MIR166a, zina-<br>
MIR166b, zma-MIR166c, zma-MIR166d, zma-MIR166e, zma-MIR166e, zma-MIR166f, zrna-<br>
MIR166g, zma-MIR166h, zma-MIR166i, zma-MIR166j, zma-MIR166k, zma-MIR166m, zrna-<br>
MIR167a, zma-MIR167b, zma-MIRl 67c, zma-MIRl 67 d, zma-MIR 167e, zma-MIR167f, zrna-<br>
MIR167g, zma-MIRl 67h, zma-MIRl 68a, zma-MIRl 68b, zma-MIRl 69a, zma-MIRl 69b, zrna-<br>
MIR169c, zma-MIRl 69d, zma-MIRl 69e, zma-MIRl 69f, zma-MIRl 69g, zma-MIRl 69i, zma-<br>
MIRl 69j, zma-MIR169k, zma-MIR171a, zma-MIR171b, zma-MIR171c, zma-MIRHld, zma-<br>
MIR171e, zma-MIR171f, zma-MIR171g, zma-MIR171h, zma-MIR171i, zma-MIR171j, zma-<br>
MIR171k, zma-MIR172a, zma-MIR172b, zma-MIR172c or zma-MIR172d, zma-MIR172e,<br>
zma-MIR319a, zma-MIR319b, zma-MIR319d, zma-MIR393, zma-MIR394a, zma-MIR39-4b,<br>
zma-MIR395a, zma-MIR395b, zma-MIR395c, zma-MIR395d, zma-MIR396a, zma-MIR39 6b,<br>
zma-MIR399a, zma-MIR399b, zma-MIR399c, zma-MIR399d, zma-MIR399e, zma-MIR35»9f,<br>
zma-MIR408.<br>
[0112] In some embodiments, the miRNA is a miRNA disclosed in Genbank (USA), EMIBL<br>
(Europe) or DDBJ (Japan). In some embodiments, the miRNA is selected from one of the<br>
following Genbank accession numbers: AJ5O5OO3, AJ505002, AJ505001, AJ4968O5,<br>
AJ496804, AJ496803, AJ496802, AJ496801, AJ505004, AJ493656, AJ493655, AJ493654,<br>
AJ493653, AJ493652, AJ493651, AJ493650, AJ493649, AJ493648, AJ493647, AJ493646,<br>
AJ493645, AJ493644, AJ493643, AJ493642, AJ493641, AJ493640, AJ493639, AJ493638,<br>
AJ493637, AJ493636, AJ493635, AJ493634, AJ493633, AJ493632, AJ493631, AJ493S30,<br>
AJ493629, AJ493628, AJ493627, AJ493626, AJ493625, AJ493624, AJ493623, AJ493S22,<br>
AJ493621, AJ493620, AY615374, AY615373, AY730704, AY730703, AY730702, AY730701,<br>
AY730700, AY730699, AY730698, AY599420, AY551259, AY551258, AY551Z57,<br>
AY551256, AY551255, AY551254, AY551253, AY551252, AY551251, AY551Z5O,<br>
AY551249, AY551248, AY551247, AY551246, AY551245, AY551244, AY551Z43,<br>
AY551242, AY551241, AY551240, AY551239, AY551238, AY551237, AY551Z36,<br>
AY551235, AY551234, AY551233, AY551232, AY551231, AY551230, AY5512.29,<br>
AY551228, AY551227, AY551226, AY551225, AY551224, AY551223, AY551222,<br>
AY551221, AY551220, AY551219, AY551218, AY551217, AY551216, AY551Z15,<br>
AY551214, AY551213, AY551212, AY551211, AY551210, AY551209, AY551208,<br>
AY551207,    AY551206,   AY551205,    AY551204,    AY551203,    AY551202,    AY551201,<br><br>
WO 2006/044322	PCT/US2005/036373<br>
29<br>
AY551200, AY551199, AY551198, AY551197, AY551196, AY551195, AY551194,<br>
AY551193, AY551192, AY551191, AY551190, AY5511S9, AY551188, AY501434,<br>
AY501433, AY501432, AY501431, AY498859, AY376459, AY376458 AY88423 3,<br>
AY884232, AY884231, AY884230, AY884229, AY884228, AY884227, AY884226,<br>
AY884225, AY884224, AY884223, AY884222, AY884221, AY884220, AY88421 9,<br>
AY884218, AY884217, AY884216, AY728475, AY728474, AY728473, AY728472,<br>
AY728471, AY728470, AY728469, AY728468, AY728467, AY728466, AY728465,<br>
AY728464, AY728463, AY728462, AY728461, AY728460, AY728459, AY728458,<br>
AY728457, AY728456, AY728455, AY728454, AY728453, AY728452, AY728451,<br>
AY728450, AY728449, AY728448, AY728447, AY728446, AY728445, AY72844-4,<br>
AY728443, AY728442, AY728441, AY728440, AY728439, AY728438, AY728437,<br>
AY728436, AY728435 AY728434, AY728433, AY728432, AY728431, AY728430,<br>
AY728429, AY728428, AY728427, AY728426, AY728425, AY728424, AY7284Z3,<br>
AY728422, AY728421, AY728420, AY728419, AY728418, AY728417, AY728416,<br>
AY728415, AY728414, AY728413, AY728412, AY728411, AY728410, AY7284O9,<br>
AY728408, AY728407, AY728406, AY728405, AY728404, AY728403, AY7284O2,<br>
AY728401, AY728400, AY728399, AY728398, AY728397, AY728396, AY728395,<br>
AY728394, AY72S393, AY728392, AY728391, AY728390, AY728389, AY728388,<br>
AY851149, AY851148, AY851147, AY851146, AY851145, AY851144 or AY599420.<br>
[0113] In some embodiments, the miRNA is selected from one of the sequences disclosed in<br>
U.S. published patent application No. 2005/0144669 Al, incorporated herein by reference.<br>
[0114] In some embodiments, the above miRNAs, as well as those disclosed herein, have<br>
been modified to be directed to a specific target as described herein.<br>
[0115] In some embodiments the target RNA is an RNA of a plant pathogen, such as a plant<br>
virus or plant viroid. In some embodiments, the miRNA directed against the plant pathogen<br>
RNA is operably linked to a pathogen-inducible promoter. In some embodiments, the target<br>
RNA is an tnRNA. The target sequence in an mRNA may be a non-coding sequence (such as an<br>
intron sequence, 5' untranslated region and 3' untranslated regeion), a coding sequence ox a<br>
sequence involved in mRNA splicing. Targeting the miRNA to an intron sequence<br>
compromises the maturation of the mRNA. Targeting the miRNA to a sequence involved in<br>
mRNA splicing influences the maturation of alternative splice forms providing different protein<br>
isoforms.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
30<br>
[0116] In some embodiments there are provided cells, plants, and seeds comprising the<br>
polynucleotides of the invention, and/or produced by the methods of the invention. In some<br>
embodiments, the cells, plants, and/or seeds comprise a nucleic acid construct comprising a<br>
modified plant miRNA precursor, as described herein. In some embodiments, the modified<br>
plant miRNA precursor in the nucleic acid construct is operably linked to a promoter. The<br>
promoter may be any well known promoter, including constitutive promoters, inducible<br>
promoters, derepressible promoters, and the like, such as described below. The cells include<br>
prokaryotic and eukaryotic cells, including but not limited to bacteria, yeast, fungi, viral,<br>
invertebrate, vertebrate, and plant cells. Plants, plant cells, and seeds of the invention include<br>
gynosperms, monocots and dicots, including but not limited to, rice, wheat, oats, barley, millet,<br>
sorghum, soy, sunflower, safflower, canola, alfalfa, cotton, Arabidopsis, and tobacco.<br>
[0117] In another embodiment, there is provided a method for down regulating multiple<br>
target RNAs comprising introducing into a cell a nucleic acid construct encoding a multiple<br>
number of miRNAs. One miRNA in the multiple miRNAs is complementary to a region of one<br>
of the target RNAs. In some embodiments, a miRNA is fully complementary to the region of<br>
the target RNA. In some embodiments, a miRNA is complementary and includes the use of G-<br>
U base pairing, i.e. the GU wobble, to otherwise be fully complementary. In some<br>
embodiments, the first ten nucleotides of the miRNA (counting from the 5' end of the miRNA)<br>
are fully complementary to a region of the target RNA and the remaining nucleotides may<br>
include mismatches and/or bulges with the target RNA. In some embodiments a miRNA<br>
comprises about 10-200 nucleotides, about 10-15, 15-20, 19, 20, 21, 22, 23, 24, 25, 26, 27, 25-<br>
30, 30-50, 50-100, 100-150, or about 150-200 nucleotides. The binding of a miRNA to its<br>
complementary sequence in the target RNA results in cleavage of the target RNA. In some<br>
embodiments, the miRNA is a miRNA that has been modified such that the miRNA is fully<br>
complementary to the target sequence of the target RNA. In some embodiments, the miRNA is<br>
an endogenous plant miRNA that has been modified such that the miRNA is fully<br>
complementary to the target sequence of the target RNA. In some embodiments, the miRNA is<br>
operably linked to a promoter. In some embodiments, the multiple miRNAs are linked one to<br>
another so as to form a single transcript when expressed. In some embodiments, the nucleic acid<br>
construct comprises a promoter operably linked to the miRNA.<br>
[0118] In some embodiments, the nucleic acid construct encodes miRNAs for suppressing a<br>
multiple number of target sequences. The nucleic acid construct encodes at least two miRNAs.<br>
In some embodiments, each miRNA is substantially complementary to a target or which is<br><br>
WO 2006/044322	PCT/US2005/036373<br>
31<br>
modified to be complementary to a target as described herein. In some embodiments, the<br>
nucleic acid construct encodes for 2-30 or more miRNAs, for example 3-40 or more miRNAs,<br>
for example 3-45 or more miRNAs, and for further example, multimers of 2, 3, 4, 5, 6, 7, 8, 9,<br>
10, 11, 12, 13, 14, 15, 16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,<br>
36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, or more miRNAs. In some embodiments,<br>
the multiple miRNAs are linked one to another so as to form a single transcript when expressed.<br>
[0119] In some embodiments, polymeric pre-miRNAs that are artificial miRNA precursors<br>
consisting of more than one miRNA precursor units are provided. The polymeric pre-miRNAs<br>
can either be hetero-polymeric with different miRNA precursors, or homo-polymeric containing<br>
several units of the same miRNA precursor. The Examples demonstrate that hetero-polymeric<br>
pre-miRNAs are able to produce different mature artificial miRNAs. For example, pre-miR-<br>
PDSl169g-CPC3159a, which is a dimer comprising of pre-miR-CPC3I59a and pre-miR-PDSl169g<br>
can produce mature miR-PDSl!69g and miR-CPC3l39a when expressed in plant cells. The<br>
Examples also demonstrate that homo-polymeric miRNA precursors are able to produce<br>
different mature artificial miRNAs. For example, pre-miR-P69159a-HC-Pro159a, which is a dimer<br>
comprising pre-miR-P69159a and pre-miR-HC-Pro159a, can produce mature miR-P69I59a and<br>
miR-HC-Pro!59a. In a similar manner, hetero- or homo-polymeric pre-miRNAs are produced<br>
that contain any number of monomer units, such as described herein. An exemplary method for<br>
preparing a nucleic acid construct comprising multiple pre-miRNAs under the control of a single<br>
promoter is shown in Examples 21 and 27. Each mature miRNA is properly processed from the<br>
nucleic acid construct as demonstrated in Examples 22 and 27.<br>
[0120] In some embodiments, the nucleic acid construct comprises multiple polynucleotides,<br>
each polynucleotide encoding a separate miRNA precursor, i.e., a separate pre-miRNA. The<br>
polynucleotides are operably linked one to another such that they may be placed under the<br>
control of a single promoter. In some embodiments, the multiple polynucleotides are linked one<br>
to another so as to form a single transcript containing the multiple pre-miRNAs when expressed.<br>
The single transcript is processed in the host cells to produce multiple mature miRNAs, each<br>
capable of downregulating its target gene. As many polynucleotides encoding the pre-miRNAs<br>
as desired can be linked together, with the only limitation being the ultimate size of the<br>
transcript. It is well known that transcripts of 8-10 kb can be produced in plants. Thus, it is<br>
possible to form a nucleic acid construct comprising multimeric polynucleotides encoding 2-30<br>
or more pre-miRNAs, for example 3-40 or more pre-miRNAs, for example 3-45 or more pre-<br>
miRNAs, and for further example, multimers of 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16,<br><br>
WO 2006/044322	PCT/US2005/036373<br>
32<br>
17, 18, 19, 20, 21, 22, 23, 24, 25,26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42,<br>
43, 44, 45, 46, 47, 48, 49, or more pre-miRNAs.<br>
[0121] In some embodiments, the nucleic acid construct further comprises a promoter<br>
operably linked to the polynucleotide encoding the multiple number of miRNAs. In some<br>
embodiments, the nucleic acid construct lacking a promoter is designed and introduced in such a<br>
way that it becomes operably linked to a promoter upon integration in the host genome. In some<br>
embodiments, the nucleic acid construct is integrated using recombination, including site-<br>
specific recombination. See, for example, PCT International published application No. WO<br>
99/25821, herein incorporated by reference. In some embodiments, the nucleic acid construct is<br>
an RNA. In some embodiments, the nucleic acid construct comprises at least one recombination<br>
site, including site-specific recombination sites. In some embodiments the nucleic acid<br>
construct comprises at least one recombination site in order to facilitate integration,<br>
modification, or cloning of the construct. In some embodiments the nucleic acid construct<br>
comprises two site-specific recombination sites flanking the miRNA precursor. In some<br>
embodiments the site-specific recombination sites include FRT sites, lox sites, or att sites,<br>
including attB, attL, attP or attR sites. See, for example, PCT International published application<br>
No. WO 99/25821, and U.S. Patents 5,888,732, 6,143,557, 6,171,861, 6,270,969, and 6,277,608,<br>
herein incorporated by reference.<br>
[0122] In some embodiments, the pre-miRNA is inserted into an intron in a gene or a<br>
transgene of a cell or plant. If the gene has multiple introns, a pre-miRNA, which can be the<br>
same or different, can be inserted into each intron. In some embodiments the pre-miRNA<br>
inserted into an intron is a polymeric pre-miRNA, such as described herein. During RNA<br>
splicing, introns are released from primary RNA transcripts and therefore, as illustrated herein,<br>
can serve as precursors for miRNAs. Most introns contain a splicing donor site at the 5' end,<br>
splicing acceptor site at the 3' end and a branch site within the intron. The branch site is<br>
important for intron maturation — without it, an intron can not be excised and released from the<br>
primary RNA transcript. A branch site is usually located 20-50 nt upstream of the splicing<br>
acceptor site, whereas distances between the splice donor site and the branch site are largely<br>
variable among different introns. Thus, in some embodiments, the pre-miRNA is inserted into<br>
an intron between the splicing donor site and the branch site.<br>
[0123] In some embodiments the target RNA is an RNA of a plant pathogen, such as a plant<br>
virus or plant viroid. In some embodiments, the miRNA directed against the plant pathogen<br>
RNA is opeTably linked to a pathogen-inducible promoter.   In some embodiments, the target<br><br>
WO 2006/044322	PCT/US2005/036373<br>
33<br>
RNA is an mRNA. The target sequence in an mRNA may be an intron sequence, a coding<br>
sequence or a sequence involved in mRNA splicing. Targeting the miRNA to an intron<br>
sequence compromises the maturation of the mKNA. Targeting the miRNA to a sequence<br>
involved in mRNA splicing influences the maturation of alternative splice forms providing<br>
different protein isoforms. In some embodiments, the target includes genes affecting agronomic<br>
traits, insect resistance, disease resistance, herbicide resistance, sterility, grain cliaracteristics,<br>
and commercial products.<br>
[0124] In some embodiments there are provided cells, plants, and seeds comprising the<br>
nucleic acid construct encoding multiple miRNAs of the invention, and/or produced by the<br>
methods of the invention. In some embodiments, the cells, plants, and/or seeds comprise a<br>
nucleic acid construct comprising multiple polynucleotides, each encoding a plant miRNA<br>
precursor, as described herein. In some embodiments, the multiple polynucleotides are operably<br>
linked to a promoter. The promoter may be any well known promoter, includiag constitutive<br>
promoters, inducible promoters, derepressible promoters, and the like, such as described below.<br>
The polynucleotides encoding the miRNA precursors are linked together. In some<br>
embodiments, the multiple polynucleotides are linked one to another so as to form a single<br>
transcript containing the multiple pre-miRNAs when expressed in the cells, plants or seeds. The<br>
cells include prokaryotic and eukaryotic cells, including but not limited to bacteria., yeast, fungi,<br>
viral, invertebrate, vertebrate, and plant cells. Plants, plant cells, and seeds of the invention<br>
include gynosperms, monocots and dicots, including but not limited to, rice, wheat, oats, barley,<br>
millet, sorghum, soy, sunflower, safflower, canola, alfalfa, cotton, Arabidopsis, and tobacco.<br>
[0125] The present invention concerns methods and compositions useful in suppression of a<br>
target sequence and/or validation of function. The invention also relates to a method for using<br>
microRNA (miRNA) mediated RNA interference (RNAi) to silence or suppress a target<br>
sequence to evaluate function, or to validate a target sequence for phenotypic effect and/or trait<br>
development. Specifically, the invention relates to constructs comprising small nucleic acid<br>
molecules, miRNAs, capable of inducing silencing, and methods of using these miRNAs to<br>
selectively silence target sequences.<br>
[0126] RNA interference refers to the process of sequence-specific post-transcriptional gene<br>
silencing in animals mediated by short interfering RNAs (siRNAs) (Fire et al. (1998) Nature<br>
391:806-810). The corresponding process in plants is commonly referred, to as post-<br>
transcriptional gene silencing (PTGS) or RNA silencing and is also referred to  as quelling in<br><br>
WO 2006/044322	PCT/US2005/036373<br>
34<br>
fungi. The process of post-transcriptional gene silencing is thought to be an evolutionarily-<br>
conserved cellular defense mechanism used to prevent the expression of foreign genes and is<br>
commonly shared by diverse flora and phyla (Fire et al. (1999) Trends Genet. 15:358-363).<br>
Such protection from foreign gene expression may have evolved in response to the production of<br>
double-stranded RNAs (dsRNAs) derived from viral infection or from the random integration of<br>
transposon elements into a host genome via a cellular response that specifically destroys<br>
homologous single-stranded RNA of viral genomic RNA. The presence of dsRNA in cells<br>
triggers the RNAi response through a mechanism that has yet to be fully characterized.<br>
[0127] The presence of long dsRNAs in cells stimulates the activity of a ribonuclease UJ<br>
enzyme referred to as dicer. Dicer is involved in the processing of the dsRNA into short pieces<br>
of dsRNA known as short interfering RNAs (siRNAs) (Bernstein et al. (2001) Nature 409:363-<br>
366). Short interfering RNAs derived from dicer activity are typically about 21 to about 23<br>
nucleotides in length and comprise about 19 base pair duplexes (Elbashir et al. (2001) Genes<br>
Dev 15:188-200). Dicer has also been implicated in the excision of 21- and 22-nucleotide small<br>
temporal RNAs (stRNAs) from precursor RNA of conserved structure that are implicated in<br>
translational control (Hutvagner et al. (2001) Science 293:834-838). The RNAi response also<br>
features an endonuclease complex, commonly referred to as an RNA-induced silencing complex<br>
(RISC), which mediates cleavage of single-stranded RNA having sequence complementarity to<br>
the anrisense strand of the siRNA duplex. Cleavage of the target RNA takes place in the middle<br>
of the region complementary to the antisense strand of the siRNA duplex (Elbashir et al. (2001)<br>
Genes Dev 15:188-200). In addition, RNA interference can also involve small RNA (e.g.,<br>
microRNA, or miRNA) mediated gene silencing, presumably through cellular mechanisms that<br>
regulate chromatin structure and thereby prevent transcription of target gene sequences (see,<br>
e.g., Allshire, Science 297:1818-1819 2002; Volpe et al. (2002) Science 297:1833-1837;<br>
Jenuwein (2002) Science 297:2215-2218; Hall et al. (2002) Science 297:2232-2237). As such,<br>
miRNA molecules of the invention can be used to mediate gene silencing via interaction with<br>
RNA transcripts or alternately by interaction with particular gene sequences, wherein such<br>
interaction results in gene silencing either at the transcriptional or post-transcriptional level.<br>
[0128] RNAi has been studied in a variety of systems. Fire et al. ((1998) Nature 391:806-<br>
811) were the first to observe RNAi in C. elegans. Wianny and Goetz ((1999) Nature Cell Biol<br>
2:70) describe RNAi mediated by dsRNA in mouse embryos. Hammond et al. ((2000) Nature<br>
404:293-296) describe RNAi in Drosophila cells transfected with dsRNA. Elbashir et al.<br>
((2001) Nature 411:494-498) describe RNAi induced by introduction of duplexes of synthetic<br><br>
WO 2006/044322	PCT/US2005/036373<br>
35<br>
21-nucleotide RNAs in cultured mammalian cells including human embryonic kidney and HeLa<br>
cells.<br>
[0129] Small RNAs play an important role in controlling gene expression. Regulation of<br>
many developmental processes, including flowering, is controlled by small RNAs. It is now<br>
possible to engineer changes in gene expression of plant genes by using transgenic constructs<br>
which produce small RNAs in the plant.<br>
[0130] Small RNAs appear to function by base-pairing to complementary RNA or DNA<br>
target sequences. When bound to RNA, small RNAs trigger either RNA cleavage or<br>
translational inhibition of the target sequence. When bound to DNA target sequences, it is<br>
thought that small RNAs can mediate DNA methylation of the target sequence. The<br>
consequence of these events, regardless of the specific mechanism, is that gene expression is<br>
inhibited.<br>
[0131] It is thought that sequence complementarity between small RNA.S and their RNA<br>
targets helps to determine which mechanism, RNA cleavage or translational inhibition, is<br>
employed. It is believed that siRNAs, which are perfectly complementary with their targets,<br>
work by RNA cleavage. Some miRNAs have perfect or near-perfect complementarity with their<br>
targets, and RNA cleavage has been demonstrated for at least a few of these miRNAs. Other<br>
miRNAs have several mismatches with their targets, and apparently inhibit their targets at the<br>
translational level. Again, without being held to a particular theory on the mechanism of action,<br>
a general rule is emerging that perfect or near-perfect complementarity favors RNA cleavage,<br>
especially within the first ten nucleotides (counting from the 5'end of the miRNA), whereas<br>
translational inhibition is favored when the miRNA/target duplex contains many mismatches.<br>
The apparent exception to this is microRNA 172 (miR172) in plants. One of the targets of<br>
miR172 is APETALA2 (AP2), and although rm'R172 shares near-perfect complementarity with<br>
AP2 it appears to cause translational inhibition of AP2 rather than RNA cleavage.<br>
[0132] MicroRNAs (miRNAs) are noncoding RNAs of about 19 to about 24 nucleotides (nt)<br>
in length that have been identified in both animals and plants (Lagos-Quintana et al. (2001)<br>
Science 294:853-858, Lagos-Quintana et al. (2002) Curr Biol 12:735-739; Lau et al. (2002)<br>
Science 294:858-862; Lee and Ambros (2001) Science 294:862-864; Llave &amp;t al. (2002) Plant<br>
Cell 14:1605-1619; Mourelatos et al. (2002) Genes Dev 16:720-728; Park et al. (2002) Curr<br>
Biol 12:1484-1495; Reinhart et al. (2002) Genes Dev 16:1616-1626). They are processed from<br>
longer precursor transcripts that range in size from approximately 70 to 200 nt, and these<br>
precursor transcripts have the ability to form stable hairpin structures. In animals, the enzyme<br><br>
WO 2006/044322	PCT/US2005/036373<br>
36<br>
involved in processing miRNA precursors is called Dicer, an RNAse Ill-like protein (Grishok et<br>
al. (2001) Cell 106:23-34; Hutvagner et al. (2001) Science 293:834-838; Ketting et al. (2001)<br>
Genes Dev 15:2654-2659). Plants also have a Dicer-like enzyme, DCL1 (previously named<br>
CARPEL FACTORY/SHORT INTEGUMENTS1/ SUSPENSOR1), and recent evidence<br>
indicates that it, like Dicer, is involved in processing the hairpin precursors to generate mature<br>
miRNAs (Park et al. (2002) Curr Biol 12:1484-1495; Reinhart et al. (2002) Genes Dev 16:1616-<br>
1626). Furthermore, it is becoming clear from recent work that at least some miRNA hairpin<br>
precursors originate as longer polyadenylated transcripts, and several different miRNAs and<br>
associated hairpins can be present in a single transcript (Lagos-Quintana et al. (2001) Science<br>
294:853-858; Lee et al. (2002) EMBO J 21:4663-4670). Recent work has also examined the<br>
selection of the miRNA strand from the dsRNA product arising from processing of the hairpin<br>
by DICER (Schwartz et al. (2003) Cell 115:199-208). It appears that the stability {i.e. G:C vs.<br>
A:U content, and/or mismatches) of the two ends of the processed dsRNA affects the strand<br>
selection, with the low stability end being easier to unwind by a helicase activity. The 5' end<br>
strand at the low stability end is incorporated into the RISC complex, while the other strand is<br>
degraded.<br>
[0133] In animals, there is direct evidence indicating a role for specific miRNAs in<br>
development. The lin-4 and let-7 miRNAs in C. elegans have been found to control temporal<br>
development, based on the phenotypes generated when the genes producing the lin-4 and let-7<br>
miRNAs are mutated (Lee et al. (1993) Cell 75:843-854; Reinhart et al. (2000) Nature 403-901-<br>
906). In addition, both miRNAs display a temporal expression pattern consistent with their roles<br>
in developmental timing. Other animal miRNAs display developmentally regulated patterns of<br>
expression, both temporal and tissue-specific (Lagos-Quintana et al. (2001) Science 294:853-<br>
853, Lagos-Quintana et al. (2002) Curr Biol 12:735-739; Lau et al. (2001) Science 294:858-862;<br>
Lee and Ambros (2001) Science 294:862-864), leading to the hypothesis that miRNAs may, in<br>
many cases, be involved in the regulation of important developmental processes. Likewise, in<br>
plants, the differential expression patterns of many miRNAs suggests a role in development<br>
(Llave et al. (2002) Plant Cell 14:1605-1619; Park et al. (2002) Curr Biol 12:1484-1495;<br>
Reinhart et al. (2002) Genes Dev 16:1616-1626), which has now been shown (e.g., Guo et al.<br>
(2005) Plant Cell 17:1376-1386).<br>
[0134] MicroRNAs appear to regulate target genes by binding to complementary sequences<br>
located in the transcripts produced by these genes. In the case of lin-4 and let-7, the target sites<br>
are located in the 3' UTRs of the target mRNAs (Lee et al. (1993) Cell 75:843-854; Wightman<br><br>
WO 2006/044322	PCT/US2005/036373<br>
37<br>
et al. (1993) Cell 75:855-862; Reinhart et al. (2000) Nature 403:901-906; Slack et al. (2000)<br>
Mol Cell 5:659-669), and there are several mismatches between the lin-4 and let-7 miRNAs and<br>
their target sites. Binding of the lin-4 or let-7 miRNA appears to cause downregulation of<br>
steady-state levels of the protein encoded by the target mRNA without affecting the transcript<br>
itself (Olsen and Ambros (1999) Dev Biol 216:671-680). On the other hand, recent evidence<br>
suggests that miRNAs can, in some cases, cause specific RNA cleavage of the target transcript<br>
within the target site, and this cleavage step appears to require 100% complementarity between<br>
the miRNA and the target transcript (Hutvagner and Zamore (2002) Science 297:2056-2060;<br>
Llave et al. (2002) Plant Cell 14:1605-1619), especially within the first ten nucleotides<br>
(counting from the 5' end of the miRNA). It seems likely that miRNAs can enter at least two<br>
pathways of target gene regulation. Protein downregulation when target complementarity is<br>
RNA cleavage pathway are analogous to the 21-25 nt short interfering RNAs (siRNAs)<br>
generated during RNA interference (RNAi) in animals and posttranscriptional gene silencing<br>
(PTGS) in plants (Hamilton and Baulcombe (1999) Science 286:950-952; Hammond et al,<br>
(2000) Nature 404:293-296; Zamore et al, (2000) Cell 31:25-33; Elbashir et al, (2001) Nature<br>
411:494-498), and likely are incorporated into an RNA-rnduced silencing complex (RISC) that<br>
is similar or identical to that seen for RNAi.<br>
[0135] Identifying the targets of miRNAs with bioinformatics has not been successful in<br>
animals, and this is probably due to the fact that animal miRNAs have a low degree of<br>
complementarity with their targets. On the other hand, bioinformatic approaches have been<br>
successfully used to predict targets for plant miRNAs (Llave et al. (2002) Plant Cell 14:1605-<br>
1619; Park et al. (2002) Curr Biol 12:1484-1495; Rhoades et al. (2002) Cell 110:513-520), and<br>
thus it appears that plant miRNAs have higher overall complementarity with their putative<br>
targets than do animal miRNAs. Most of these predicted target transcripts of plant miRNAs<br>
encode members of transcription factor families implicated in plant developmental patterning or<br>
cell differentiation. Nonetheless, biological function has not been directly demonstrated for any<br>
plant miRNA. Although Llave et al. ((2002) Science 297:2053-2056) have shown that a<br>
transcript for a SCARECROW-like transcription factor is a target of the Arabidopsis miRNA<br>
mirl71, these studies were performed in a heterologous species and no plant phenotype<br>
associated with mid 71 was reported.<br>
[0136] The methods provided can be practiced in any organism in which a method of<br>
transformation is available, and for which there is at least some sequence information for the<br><br>
WO 2006/044322	PCT/US2005/036373<br>
38<br>
target sequence, or for a region flanking the target sequence of interest. It is also understood that<br>
two or move sequences could be targeted by sequential transformation, co-transformation with<br>
more than one targeting vector, or the construction of a DNA construct comprising more than<br>
one miRNA sequence. The methods of the invention may also be implemented by a<br>
combinatorial nucleic acid library construction in order to generate a library of miRNAs directed<br>
to random target sequences. The library of miRNAs could be used for high-throughput<br>
screening for gene function validation.<br>
[0137] General categories of sequences of interest include, for example, those genes involved<br>
in regulation or information, such as zinc fingers, transcription factors, homeotic genes, or cell<br>
cycle and cell death modulators, those involved in communication, such as kinases, and those<br>
involved in housekeeping, such as heat shock proteins.<br>
[0138] Target sequences further include coding regions and non-coding regions such as<br>
promoters, enhancers, terminators, introns and the like, which may be modified in order to alter<br>
the expression of a gene of interest. For example, an intron sequence can be added to the 5'<br>
region to increase the amount of mature message that accumulates (see for example Buchman<br>
and Berg (1988) MoI Cell Biol 8:4395-4405); and Callis et al. (1987) Genes Dev 1:1183-1200).<br>
[0139] The target sequence may be an endogenous sequence, or may be an introduced<br>
heterologous sequence, or transgene. For example, the methods may be used to alter the<br>
regulation or expression of a transgene, or to remove a transgene or other introduced sequence<br>
such as an introduced site-specific recombination site. The target sequence may also be a<br>
sequence from a pathogen, for example, the target sequence may be from a plant pathogen such<br>
as a virus, a mold or fungus, an insect, or a nematode. A miRNA can be expressed in a plant<br>
which, upon infection or infestation, would target the pathogen and confer some degree of<br>
resistance to the plant. The Examples herein demonstrate the techniques to design artificial<br>
miRNAs to confer virus resistance/tolerance to plants. In some embodiments, two or more<br>
artificial miRNA sequences directed against different seqeuences of the virus can be used to<br>
prevent the target virus from mutating and thus evading the resistance mechanism. In some<br>
embodiments, sequences of artifical miRNAs can be selected so that they target a critical region<br>
of the viral RNA (e.g. active site of a silencing gene suppressor). In this case, mutation of the<br>
virus in this selected region may render the encoded protein inactive, thus preventing mutation<br>
of the virus as a way to escape the resistance mechanism. In some embodiments, an artifical<br>
miRNA directed towards a conserved sequence of a family of viruses would confer resistance to<br>
members of the entire family.   In some embodiments, an artifical miRNA directed towards a<br><br>
WO 2006/044322	PCT/US2005/036373<br>
39<br>
sequence conserved amongst members of would confer resistance to members of the differem<br>
/iral families (e.g, see Table 1).<br>
TABLE 1<br>
Conserved Viral Genome Sequence of TuMV for Artificial miRNA Design<br><br>
				TuMV CY5	<br>
No	Regiona	Gene		Sequenceb (SEQ ID NO:)	length<br>
1	3207 to 3229	P3	5	'-cgatttaggcggcagatacagcg-3' (167)	23<br>
2	9151 to 9185	CP	5	'-attctcaatggtttaatggtctggtgcattgagaa-3' (168)	35<br>
3	9222 to 9227	CP	5	' -ataaacggaatgtgggtgatgatgga-3 '(169)	26<br>
4	9235 to 9255	CP	5	'-gatcaggtggaattcccgatc-3' (170)	21<br>
5	9270 to 9302	CP	5	'-cacgccaaacccacatttaggcaaataatggc-3' (171)	32<br>
6	9319 to 9386	CP	5	'-gctgaagcgtacattgaaaagcgtaaccaagaccgaccatac	68<br>
				atgccacgatatggtcttcagcgcaa-3' (172)	<br>
7	9430 to 9509	CP	5	'-gaaatgacttctagaactccaatacgtgcgagagaagcacac	80<br>
				atccagatgaaagcagcagcactgcgtggcgcaaataa-3'	<br>
				(173)	<br>
8	9541 to 9566	CP	5	'-acaacggtagagaacacggagaggca-3' (174)	26<br>
a The region of genome sequence is according to TuMV CY5 strain (AF530055).<br>
b The highly conserved of TuMV sequence from 21  different TuMV strains was<br>
alignment by Vector NTI Advance 10.0.1 software (Invitrogen Corp) .<br>
The full-length sequence of 21  different TuMV strains were obtained from the<br>
10	GenBank database under the following accession numbers  including AB093596,<br>
AB093598, AB093599, AB093600, AB093615, AB093616, AB093617, AB093618,<br>
AB093619, AB093611, AB093612, AY227024, AB093609, AF394601, AF169561,<br>
AF530055, AJF394602, AB093623, AB093624, AY090660, D83184.<br>
15 [0140] In plants, other categories of target sequences include genes affecting agronomic<br>
traits, insect resistance, disease resistance, herbicide resistance, sterility, grain characteristics,<br>
and commercial products. Genes of interest also include those involved in oil, starch,<br>
carbohydrate, or nutrient metabolism as well as those affecting, for example, kernel size, sucrose<br>
loading, and the like. The quality of grain is reflected in traits such as levels and types of oils,<br>
20 saturated and unsaturated, quality and quantity of essential amino acids, and levels of cellulose.<br>
For example, genes of the phytic acid biosynthetic pathway could be suppressed to generate a<br>
high available phosphorous phenotype. See, for example, phytic acid biosynthetic enzymes<br>
including inositol polyphosphate kinase-2 polynucleotides, disclosed in WO 02/059324, inositol<br>
1,3,4-trisphosphate 5/6-kinase polynucleotides, disclosed in WO 03/027243, and myo-inositol 1-<br>
25 phosphate synthase and other phytate biosynthetic polynucleotides, disclosed in WO 99/05298,<br>
all of which are herein incorporated by reference.  Genes in the lignification pathway could be<br><br>
WO 2006/044322	PCT/US2005/036373<br>
40<br>
suppressed to enhance digestibility or energy availability. Genes affecting cell cycle or cell<br>
death could be suppressed to affect growth or stress response. Genes affecting DNA repair<br>
and/or recombination could be suppressed to increase generic variability. Genes affecting<br>
flowering time could be suppressed, as well as genes affecting fertility. Any target sequence<br>
could be suppressed in order to evaluate or confirm its role in a particular trait or phienotype, or<br>
to dissect a molecular, regulatory, biochemical, or proteomic pathway or network.<br>
[0141] A number of promoters can be used, these promoters can be selected based on the<br>
desired outcome. It is recognized that different applications will be enhanced by the use of<br>
different promoters in plant expression cassettes to modulate the timing, location and/or level of<br>
expression of the miRNA. Such plant expression cassettes may also contain, if desired, a<br>
promoter regulatory region (e.g., one conferring inducible, constitutive, environmentally- or<br>
developmentally-regulated, or cell- or tissue-specific/selective expression), a transcription<br>
initiation start site, a ribosome binding site, an RNA processing signal, a transcription<br>
termination site, and/or a polyadenylation signal.<br>
[0142] Constitutive, tissue-preferred or inducible promoters can be employed. Examples of<br>
constitutive promoters include the cauliflower mosaic virus (CaMV) 35S transcription initiation<br>
region, the 1'- or 2'- promoter derived from T-DNA of Agrobacterium tumefaciens, the ubiquitin<br>
1 promoter, the Smas promoter, the cinnamyl alcohol dehydrogenase promoter (U.S. Patent No.<br>
5,683,439), the Nos promoter, the pEmu promoter, the rubisco promoter, the GRP1 -8 promoter<br>
and other transcription initiation regions from various plant genes known to those of skill. If<br>
low level expression is desired, weak promoter(s) may be used. Weak constitutive promoters<br>
include, for example, the core promoter of the Rsyn7 promoter (WO 99/43838 and U.S. Patent<br>
No. 6,072,050), the core 35S CaMV promoter, and the Like. Other constitutive promoters<br>
include, for example, U.S. Patent Nos. 5,608,149; 5,608,144; 5,604,121; 5,569,597; 5,466,785;<br>
5,399,680; 5,268,463; and 5,608,142. See also, U.S. Patent No. 6,177,611, herein incorporated<br>
by reference.<br>
[0143] Examples of inducible promoters are the Adhl promoter which is inducible by<br>
hypoxia or cold stress, the Hsp70 promoter which is inducible by heat stress, the PPDK<br>
promoter and the PEP (phophoenol pyruvate) carboxylase promoter which are both inducible by<br>
light. Also useful are promoters which are chemically inducible, such as the In2-2 promoter<br>
which is safener induced (U.S. Patent No. 5,364,780), the ERE promoter which, is estrogen<br>
induced, and the Axigl promoter which is auxin induced and tapetum specific but also active in<br>
callus (PCT International published application No. WO 02/04699).    Other examples of<br><br>
WO 2006/044322	PCT/US2005/036373<br>
41<br>
inducible promoters include the GVG and XVE promoters, which are induced by<br>
glucocorticoids and estrogen, respectively (U.S. Patent No. 6,452,068).<br>
[0144] Examples of promoters under developmental control include promoters that initiate<br>
transcription preferentially in certain tissues, such as leaves, roots, fruit, seeds, or flowers. An<br>
exemplary promoter is the anther specific promoter 5126 (U.S. Patent Nos. 5,689,049 and<br>
5,689,051). Examples of seed-preferred promoters include, but are not limited to, 27 kD gamma<br>
zein promoter and waxy promoter ( Boronat et al. (1986) Plant Sci 47:95-102; Reina et al.<br>
(1990) Nucl Acids Res 18(21):6426; Kloesgen et al. (1986) Mol. Gen. Genet. 203:237-244).<br>
Promoters that express in the embryo, pericarp, and endosperm are disclosed in U.S. Patent No.<br>
6,225,529 and PCT International published application No. WO 00/12733. The disclosures of<br>
each of these are incorporated herein by reference in their entirety.<br>
[0145] In some embodiments it will be beneficial to express the gene from an inducible<br>
promoter, particularly from a pathogen-inducible promoter. Such promoters include those from<br>
pathogenesis-related proteins (PR proteins), which are induced following infection by a<br>
pathogen; e.g., PR proteins, SAR proteins, beta-l,3-glucanase, chitinase, etc. See, for example,<br>
Redolfi et al. (1983) Neth. J. Plant Pathol. 89:245-254; Uknes et al. (1992) Plant Cell 4:645-<br>
656; and Van Loon (1985) Plant Mol. Virol. 4:111-116. See also PCT International published<br>
application No. WO 99/43819, herein incorporated by reference.<br>
[0146] Of interest are promoters that are expressed locally at or neaT the site of pathogen<br>
infection. See, for example, Marineau et al. (1987) Plant Mol Biol 9:335-342; Matton et al.<br>
(1989) Molecular Plant-Microbe Interactions 2:325-331; Somsisch et al. (1986) Proc NatlAcad<br>
Sci USA 83:2427-2430; Somsisch et al. (1988) Mol Gen Genet 2:93-98; and Yang (1996) Proc<br>
Natl Acad Sci USA 93:14972-14977. See also, Chen et al. (1996) Plant J 10:955-966; Zhang et<br>
al. (1994) Proc Natl Acad Sci USA 91:2507-2511; Warner et al. (1993) Plant J 3:191-201;<br>
Siebertz et al. (1989) Plant Cell 1:961-968; U.S. Patent No. 5,750,386 (nematode-inducible);<br>
and the references cited therein. Of particular interest is the inducible promoter for the maize<br>
PRms gene, whose expression is induced by the pathogen Fusarium moniliforme (see, for<br>
example, Cordero et al. (1992) Physiol Mol Plant Path 41:189-200).<br>
[0147] Additionally, as pathogens find entry into plants through wounds or insect damage, a<br>
wound-inducible promoter may be used in the constructions of the polynucleotides. Such<br>
wound-inducible promoters include potato proteinase inhibitor (pin II) gene (Ryan (1990) Ann<br>
Rev Phytopath 28:425-449; Duan et al. (1996) Nature Biotech 14:494-498); wunl and wun2,<br>
U.S. Patent No. 5,428,148; winl and win2 (Stanford ei al. (1989) Mol Gen Genet 215:200-208);<br><br>
WO 2006/044322	PCT/US2005/036373<br>
42<br>
systemin (McGurl et al. (1992) Science 225:1570-1573); WIP1 (Rohmeier et al. (1993) Plant<br>
Mol Biol 22:783-792; Eckelkamp et al. (1993) FEBS Lett 323:73-76); MPI gene (Corderok et al.<br>
(1994) Plant J 6(2): 141-150); and the like, herein incorporated by reference.<br>
[0148] Chemical-regulated promoters can be used to modulate the expression of a gene in a<br>
plant through the application of an exogenous chemical regulator. Depending upon the<br>
objective, the promoter may be a chemical-inducible promoter, where application of the<br>
chemical induces gene expression, or a chemical-repressible promoter, where application of the<br>
chemical represses gene expression. Chemical-inducible promoters are known in the art and<br>
include, but are not limited to, the maize In2-2 promoter, which is activated by<br>
benzenesulfonamide herbicide safeners, the maize GST promoter, which is activated by<br>
hydrophobic electrophilic compounds that are used as pre-emergent herbicides, and the tobacco<br>
PR-la promoter, which is activated by salicylic acid. Other chemical-regulated promoters of<br>
interest include steroid-responsive promoters (see, for example, the glucocorticoid-inducible<br>
promoter in Schena et al. (1991) Proc Natl Acad Sci USA 88:10421-10425 and McNellis et al.<br>
(1998) Plant J 14(2):247-257) and tetracycline-inducible and tetracycline-repressible promoters<br>
(see, for example, Gatz et al. (1991) Mol Gen Genet 227:229-237, and U.S. Patent Nos.<br>
5,814,618 and 5,789,156), herein incorporated by reference.<br>
[0149] Tissue-preferred promoters can be utilized to target enhanced expression of a<br>
sequence of interest within a particular plant tissue. Tissue-preferred promoters include<br>
Yamamoto et al. (1997) Plant J 12(2):255-265; Kawamata et al. (1997) Plant Cell Physiol<br>
38(7):792-803; Hansen et al. (1997) Mol Gen Genet 254(3):337-343; Russell et al. (1997)<br>
Transgenic Res 6(2): 157-168; Rinehart et al. (1996) Plant Physiol 112(3):1331-1341; Van<br>
Camp et al. (1996) Plant Physiol 112(2):525-535; Canevascini et al. (1996) Plant Physiol<br>
112(2):513-524; Yamamoto et al. (1994) Plant Cell Physiol 35(5):773-778; Lam (1994) Results<br>
Probl Cell Differ 20:181-196; Orozco et al. (1993) Plant Mol Biol 23(6):1129-1138; Matsuoka<br>
et al. (1993) Proc Natl Acad Sci USA 90(20):9586-9590; and Guevara-Garcia et al. (1993) Plant<br>
J 4(3):495-505. Such promoters can be modified, if necessary, for weak expression.<br>
[0150] Leaf-preferred promoters are known in the art. See, for example, Yamamoto et al.<br>
(1997) Plant J 12(2):255-265; Kwon et al. (1994) Plant Physiol 105:357-67; Yamamoto et al.<br>
(1994) Plant Cell Physiol 35(5):773-778; Gotor et al. (1993) Plant J 3:509-18; Orozco et al.<br>
(1993) Plant Mol Biol 23(6): 1129-1138; and Matsuoka et al. (1993) Proc Natl Acad Sci USA<br>
90(20):9586-9590. In addition, the promoters of cab and RUBISCO can also be used. See, for<br><br>
WO 2006/044322	PCT/US2005/036373<br>
43<br>
example, Simpson et al. (1958) EMBO J 4:2723-2729 and Timko et al. (1988) Nature 318:57-<br>
58.<br>
[0151] Root-preferred promoters are known and can be selected from the many available<br>
from the literature or isolated de novo from various compatible species. See, for example, Hire<br>
et al. (1992) Plant Mol Biol 20(2): 207-218 (soybean root-specific glutamine synthetase gene);<br>
Keller and Baumgartner (1991) Plant Cell 3(10):1051-1061 (root-specific control element in the<br>
GRP 1.8 gene of French bean); Sanger et al. (1990) Plant Mol Biol 14(3):433-443 (root-specific<br>
promoter of the mannopine synthase (MAS) gene of Agrobacterium tumefaciens); and Miao et<br>
al. (1991) Plant Cell 3(1): 11-22 (full-length cDNA clone encoding cytosolic glutamine<br>
synthetase (GS), which is expressed in roots and root nodules of soybean). See also Bogusz et<br>
al. (1990) Plant CW/2(7):633-641, where two root-specific promoters isolated from hemoglobin<br>
genes from the nitrogen-fixing nonlegume Parasponia andersonii and the related non-nitrogen-<br>
fixing nonlegume Trema tomentosa are described. The promoters of these genes were linked to<br>
a p-glucuronidase reporter gene and introduced into both the nonlegume Nicotiana tabacum and<br>
the legume Lotus corniculatus, and in both instances root-specific promoter activity was<br>
preserved. Leach and Aoyagi ((1991) Plant Science (Limerick) 79(l):69-76) describe their<br>
analysis of the promoters of the highly expressed rolC and rolD root-inducing genes of<br>
Agrobacterium rhizogenes. They concluded that enhancer and tissue-preferred DNA<br>
determinants are dissociated in those promoters. Teeri et al. ((1989) EMBO J 8(2):343-350)<br>
used gene fusion to lacZ to show that the Agrobacterium T-DNA gene encoding octopine<br>
synthase is especially active in the epidermis of the root tip and that the TR21 gene is root<br>
specific in the intact plant and stimulated by wounding in leaf tissue, an especially desirable<br>
combination of characteristics for use with an insecticidal or larvicidal gene. The TR11 gene,<br>
fused to nptll (neomycin phosphotransferase II) showed similar characteristics. Additional root-<br>
preferred promoters include the VfENOD-GRP3 gene promoter (Kuster et al. (1995) Plant Mol<br>
Biol 29(4):759-772); and rolB promoter (Capana et al. (1994) Plant Mol Biol 25(4):681-691.<br>
See also U.S. Patent Nos. 5,837,876; 5,750,386; 5,633,363; 5,459,252; 5,401,836; 5,110,732;<br>
and 5,023,179. The phaseolin gene (Murai et al. (1983) Science 23:476-482 and Sengopta-<br>
Gopalen et al. (1988) Proc Natl Acad Sci USA 82:3320-3324.<br>
[0152] Transformation protocols as well as protocols for introducing nucleotide sequences<br>
into plants may vary depending on the type of plant or plant cell, i.e., monocot or dicot, targeted<br>
for transformation. Suitable methods of introducing the DNA construct include microinjection<br>
(Crossway et al. (1986) Biotechniques 4:320-334; and U.S. Patent No. 6,300,543), sexual<br><br>
WO 2006/044322	PCT/US2005/036373<br>
44<br>
crossing, electroporation (Riggs et al. (1986) Proc Natl Acad Sci USA 83:5602-5606),<br>
Agrobacterium-mediated transformation (Townsend et al., U.S. Pat No. 5,563,055; and U.S.<br>
Patent No. 5,981,840), direct gene transfer (Paszkowski et al. (1984) EMBO J 3:2717-2722),<br>
and ballistic particle acceleration (see, for example, U.S. Patent No. 4,945,050; U.S. Patent No.<br>
5,879,918; U.S. Patent No. 5,886,244; U.S. Patent No. 5,932,782; Tomes et al. (1995) "Direct<br>
DNA Transfer into Intact Plant Cells via Microprojectile Bombardment," in Plant Cell, Tissue,<br>
and Organ Culture: Fundamental Methods, ed. Gamborg and Phillips (Springer-Verlag, Berlin);<br>
and McCabe et al. (1988) Biotechnology 6:923-926). Also see Weissinger et al. (1988) Ann Rev<br>
Genet 22:421-477; Sanford et al. (1987) Paniculate Science and Technology 5:27-37 (onion);<br>
Christou et al. (1988) Plant Physiol 87:671-674 (soybean); Finer and McMullen (1991) In Vitro<br>
Cell Dev Biol 27P:175-182 (soybean); Singh et al. (1998) Theor Appl Genet 96:319-324<br>
(soybean); Datta et al. (1990) Biotechnology 8:736-740 (rice); Klein et al. (1988) Proc Natl<br>
Acad Sci USA 85:4305-4309 (maize); Klein et al. (1988) Biotechnology 6:559-563 (maize); U.S.<br>
Patent No. 5,240,855; U.S. Patent No. 5,322,783; U.S. Patent No. 5,324,646; Klein et al. (1988)<br>
Plant Physiol 91:440-444 (maize); Fromm et al. (1990) Biotechnology 8:833-839 (maize);<br>
Hooykaas-Van Slogteren et al. (1984) Nature 311:763-764; U.S. Patent No. 5,736,369 (cereals);<br>
Bytebier et al. (1987) Proc Natl Acad Sci USA 84:5345-5349 (Liliaceae); De Wet et al. (1985)<br>
in The Experimental Manipulation of Ovule Tissues, ed. Chapman et al. (Longman, New York),<br>
pp. 197-209 (pollen); Kaeppler et al. (1990) Plant Cell Reports 9:415-418 and Kaeppler et al.<br>
(1992) Theor Appl Genet 84:560-566 (whisker-mediated transformation); D'Halluin et al. (1992)<br>
Plant Cell 4:1495-1505 (electroporation); Li et al (1993) Plant Cell Reports 12:250-255 and<br>
Christou and Ford (1995) Annals of Botany 75:407-413 (rice); Osjoda et al. (1996) Nature<br>
Biotechnology 14:745-750 (maize via Agrobacterium tumefaciens); and U.S. Patent No.<br>
5,736,369 (meristem transformation), all of which are herein incorporated by reference.<br>
[0153] The nucleotide constructs may be introduced into plants by contacting plants with a<br>
virus or viral nucleic acids. Generally, such methods involve incorporating a nucleotide<br>
construct of the invention within a viral DNA or RNA molecule. Further, it is recognized that<br>
useful promoters encompass promoters utilized for transcription by viral RNA polymerases.<br>
Methods for introducing nucleotide constructs into plants and expressing a protein encoded,<br>
therein, involving viral DNA or RNA molecules, are known in the art. See, for example, U.S.<br>
Patent Nos. 5,889,191, 5,889,190, 5,866,785, 5,589,367 and 5,316,931; herein incorporated by-<br>
reference.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
45<br>
[0154] In some embodiments, transient expression may be desired. In those cases, standard<br>
transient transformation techniques may be used. Such methods include, but are not limited to<br>
viral transformation methods, and microinjection of DNA or RNA, as well other methods well<br>
known in the art.<br>
[0155] The cells from the plants that have stably incorporated the nucleotide sequence may<br>
be grown into plants in accordance with conventional ways. See, for example, McCormick et al.<br>
(1986) Plant Cell Reports 5:81-84. These plants may then be grown, and either pollinated with<br>
the same transformed strain or different strains, and the resulting hybrid having constitutive<br>
expression of the desired phenotypic characteristic imparted by the nucleotide sequence of<br>
interest and/or the genetic markers contained within the target site or transfer cassette. Two or<br>
more generations may be grown to ensure that expression of the desired phenotypic<br>
characteristic is stably maintained and inherited and then seeds harvested to ensure expression of<br>
the desired phenotypic characteristic has been achieved.<br>
[0156] Initial identification and selection of cells and/or plants comprising the DNA<br>
constructs may be facilitated by the use of marker genes. Gene targeting can be performed<br>
without selection if there is a sensitive method for identifying recombinants, for example if the<br>
targeted gene modification can be easily detected by PCR analysis, or if it results in a certain<br>
phenotype. However, in most cases, identification of gene targeting events will be facilitated by<br>
the use of markers. Useful markers include positive and negative selectable markers as well as<br>
markers that facilitate screening, such as visual markers. Selectable markers include genes<br>
carrying resistance to an antibiotic such as spectinomycin (e.g. the aada gene, Svab et al. (1990)<br>
Plant Mol Biol 14:197-205), streptomycin (e.g., aada, or SPT, Svab et al. (1990) Plant Mol Biol<br>
14:197-205; Jones et al. (1987) Mol Gen Genet 210:86), kanamycin (e.g., nptn, Fraley et al.<br>
(1983) Proc Natl Acad Sci USA 80:4803-4807), hygromycin (e.g., HPT, Vanden Elzen et al.<br>
(1985) Plant Mol Biol 5:299), gentamycin (Hayford et al. (1988) Plant Physiol 86:1216),<br>
phleomycin, zeocin, or bleomycin (Hille et al. (1986) Plant Mol Biol 7:171), or resistance to a<br>
herbicide such as phosphinothricin (bar gene), or sulfonylurea (acetolactate synthase (ALS))<br>
(Charest et al. (1990) Plant Cell Rep 8:643), genes that fulfill a growth requirement on an<br>
incomplete media such as HIS3, LEU2, URA3, LYS2, and TRP1 genes in yeast, and other such<br>
genes known in the art. Negative selectable markers include cytosine deaminase (codA)<br>
(Stougaard (1993) Plant J. 3:755-761), tms2 (DePicker et al. (1988) Plant Cell Rep 7:63-66),<br>
nitrate reductase (Nussame et al. (1991) Plant J 1:267-274), SU1 (O'Keefe et al. (1994) Plant<br>
Physiol. 105:473-482), aux-2 from the Ti plasmid of Agrobacterium, and thymidine kinase.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
46<br>
Screenable markers include fluorescent proteins such as green fluorescent protein (GFP)<br>
(Chalfie et al. (1994) Science 263:802; US 6,146,826; US 5,491,084; and WO 97/41228),<br>
reporter enzymes such as |3-glucuronidase (GUS) (Jefferson (1987) Plant Mol Biol Rep 5:387;<br>
U.S. Patent No. 5,599,670; U.S. Patent No. 5,432,081), P-gakctosidase (lacZ), alkaline<br>
phosphatase (AP), glutathione S-transferase (GST) and luciferase (U.S. Patent No. 5,674,713;<br>
Ow et al. (1986) Science 234:856-859), visual markers like anthocyanins such as CRC (Ludwig<br>
et al. (1990) Science 247:449-450) R gene family (e.g. Lc, P, S), A, C, R-nj, body and/or eye<br>
color genes in Drosophila, coat color genes in mammalian systems, and others known in the art.<br>
[0157] One or more markers may be used in order to select and screen for gene targeting<br>
events. One common strategy for gene disruption involves using a target modifying<br>
polynucleotide in which the target is disrupted by a promoterless selectable marker. Since the<br>
selectable marker lacks a promoter, random integration events are unlikely to lead to<br>
transcription of the gene. Gene targeting events will put the selectable marker under control of<br>
the promoter for the target gene. Gene targeting events are identified "by selection for expression<br>
of the selectable marker. Another common strategy utilizes a positive-negative selection<br>
scheme. This scheme utilizes two selectable markers, one that confers resistance (R+) coupled<br>
with one that confers a sensitivity (S+), each with a promoter. When this polynucleotide is<br>
randomly inserted, the resulting phenotype is R+/S+. When a gene targeting event is generated,<br>
the two markers are uncoupled and the resulting phenotype is R-+/S-. Examples of using<br>
positive-negative selection are found in Thykjaer et al. (1997) Plant Mol Biol 35:523-530; and<br>
PCT International published application No. WO 01/66717, which are herein incorporated by<br>
reference.<br>
[0158] The practice of the present invention employs, unless otherwise indicated,<br>
conventional techniques of chemistry, molecular biology, microbiology, recombinant DNA,<br>
genetics, immunology, cell biology, cell culture and transgenic biology, which are within the<br>
skill of the art. See, e.g., Maniatis et al. (1982) Molecular Cloning (Cold Spring Harbor<br>
Laboratory Press, Cold Spring Harbor, New York); Sambrook et al. (1989) Molecular Cloning,<br>
2nd Ed. (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, INew York); Sambrook and<br>
Russell (2001) Molecular Cloning, 3rd Ed. (Cold Spring Harbor Laboratory Press, Cold Spring<br>
Harbor, New York); Ausubel et al. (1992) Current Protocols in Molecular Biology (John Wiley<br>
&amp; Sons, including periodic updates); Glover (1985) DNA Cloning (IIRL Press, Oxford); Anand<br>
(1992) Techniques for the Analysis of Complex Genomes (Academic Press).; Guthrie and Fink<br><br>
WO 2006/044322	PCT/US2005/036373<br>
47<br>
(1991) Guide to Yeast Genetics and Molecular Biology (Academic Press).; Harlow and Lane<br>
(1988) Antibodies, (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York);<br>
Jakoby and Pastan, eds. (1979) "Cell Culture" Methods in Enzymology Vol. 58 (Academic Press,<br>
Inc., Harcourt Brace Jovanovich (NY)).; Nucleic Acid Hybridization (B. D. Harnes &amp; S. J.<br>
Higgins eds. 1984); Transcription And Translation (B. D. Hames &amp; S. J. Higgins eds. 1984);<br>
Culture Of Animal Cells (R. I. Freshney, Alan R. Liss, Inc., 1987); Immobilized Cells And<br>
Enzymes (IRL Press, 1986); B. Perbal, A Practical Guide To Molecular Cloning: (1984); the<br>
treatise, Methods In Enzymology (Academic Press, Inc., N.Y.); Gene Transfer Vectors For<br>
Mammalian Cells (J. H. Miller and M. P. Calos eds., 1987, Cold Spring Harbor Laboratory);<br>
Methods In Enzymology, Vols. 154 and 155 (Wu et al. eds.), Immunochemical Methods In Cell<br>
And Molecular Biology (Mayer and Walker, eds., Academic Press, London, 1987); Handbook<br>
Of Experimental Immunology, Volumes I-IV (D. M. Weir and C. C. Blackwell, eds., 1986);<br>
Riott, Essential Immunology, 6th Edition, Blackwell Scientific Publications, Oxford, 1988;<br>
Hogan et al., Manipulating the Mouse Embryo, (Cold Spring Harbor Laboratory Press, Cold<br>
Spring Harbor, N.Y., 1986); Westerfield, M., The zebrafish book. A guide for the laboratory use<br>
of zebrafish {Danio rerio), (4th Ed., Univ. of Oregon Press, Eugene, 2000); Methods in<br>
Arabidopsis Research (C. Koncz et al., eds, World Scientific Press, Co., Inc., River Edge,<br>
Minnesota, 1992); Arabidopsis: A Laboratory Manual (D. Weigel and J. Glazebrook, eds., Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, New York, 2002).<br>
EXAMPLES<br>
[0159] The following are non-limiting examples intended to illustrate th.e invention.<br>
Although the present invention has been described in some detail by way of illustration and<br>
example for purposes of clarity of understanding, it will be obvious that certain changes and<br>
modifications may be practiced within the scope of the appended claims. For example, any of<br>
the pre-miRNAs and miRNAs described herein can be used in place of the pre-miRNAs and<br>
miRNAs used in the examples. Examples 1-15 are derived from PCT International published<br>
application Nos. WO 2005/052170 and WO 2005/035769 and from U.S. published application<br>
Nos. US 2005/0138689 and US 2005/0120415, each incorporated herein by reference.<br>
EXAMPLE 1<br>
[0160]      The example describes the identification of a microRNA<br><br>
WO 2006/044322	PCT/US2005/036373<br>
48<br>
[0161] The following experiments are carried out on the Arabidopsis thaliana Col-0 ecotype.<br>
Plants are grown in long days (16 h light, 8 h dark) under cool white light at 22° C.<br>
[0162] Arabidopsis plants are transformed by a modified version of the floral dip method, in<br>
which Agrobacterium cell suspension is applied to plants by direct watering from above. The T-<br>
DNA vector used, pHSbarENDs, contained four copies of the CAMV 35S enhancer adjacent to<br>
the right border, an arrangement similar to that described by Weigel et al. {Plant Physiol.<br>
122:1003-1013, 2000). Transformed plants are selected with glufosinate (BAST.A) and screened<br>
for flowering time, which resulted in the identification of the early-flowering EA.T-D mutant. A<br>
single T-DNA cosegregating with early flowering is identified in EAT-D, and TAIL-PCR is<br>
performed to amplify sequences adjacent to the left and right borders of the T-DISf A. To identify<br>
transcripts upregulated in the EAT-D mutant, Northern blots containing RNA extracted from<br>
wild type (Col-0) and EAT-D plants is probed. Probes for the genes At5g04270 and At5g04280<br>
(GenBank NC_003076) do not detect any difference between wild type and EA.T-D, whereas a<br>
probe from the intergenic region identifies an ~1.4 kb transcript that is expressed at significantly<br>
higher levels in EAT-D than in wild type.<br>
[0163] To isolate the full-length EAT cDNA, 5'- and 3'-RACE-PCR is performed with a<br>
GeneRacer kit (Invitrogen) that selects for 5'-capped mRNAs. Reverse transcription is carried<br>
out using an oligo-dT primer, and PCR utilized a gene-specific primer (SEQ ID NO:45 5'-<br>
CTGTGCTCACGATCTTGTTGTTCTTGATC-3') paired with the 5' kit primer, or a second<br>
gene-specific primer (SEQ ID NO:46 5'- GTCGGCGGATCCATGGAAGAAA.GCTCATC-5')<br>
paired with the 3' kit primer.<br>
[0164] The Arabidopsis EAT-D (Early Activation Tagged - Dominant) mutant is identified<br>
in an activation tagging population (Weigel et al. (2000) Plant Physiol 122:1003-1013). As<br>
evidenced by visual inspection and by measuring rosette leaf number (Table 2), the EAT-D<br>
mutant flowers extremely early. In addition, EAT-D displays floral defects that are virtually<br>
identical to those observed for strong apetala2 (ap2) mutant alleles (Bowman et al. (1991)<br>
Development 112:1-20), including the complete absence of petals and the transformation of<br>
sepals to carpels. This ap2-like phenotype is only observed in EAT-D homozygotes, whereas<br>
both EAT-D heterozygotes and homozygotes are early flowering, indicating tixat the flowering<br>
time phenotype is more sensitive to EAT-D dosage than the ap2-like floral phenotype.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
49<br>
TABLE 2<br>
Rosette leaf numbers for Arabidopsis lines<br><br>
Genotype	rosette leaf no.	floral phenotype<br>
Col-0	11.4+/-1.2	wild type<br>
EAT-D	3.1+/-0.8	ap2<br>
EAT-OX	2.0 +/- 0.2	ap2 + additional<br>
eatdel	11.1+/-1.1	wild type<br>
miR172al-OX	2.1+/-0.3	ap2 + additional<br>
LAT-D	22.5+/-2.1	wild type<br>
At2g28550-OX	28.6 +/- 3.6	wild type<br>
5-60120	10.2+/-1.4	wild type<br>
2-28550	8.7 +/- 0.6	wild type<br>
5-60120; 2-28550	6.0 +/- 0.8	wild type<br>
[0165] The activation-tagged T-DNA insert in EAT-D is mapped to chromosome 5, in<br>
between the annotated genes At5g04270 and At5g04280. 5'- and 3'-RACE PCR is then used<br>
with primers located within this region to identify a 1.4 kb transcript (SEQ ID NO:1), which is<br>
named EAT, that is upregulated in EAT-D. When the 1.4 kb EAT cDNA is fused to the<br>
constitutive CAMV 35S promoter and the resultant 35S::EAT construct is introduced into wild<br>
type (Col-0) plants by Agrobacterium-mediated transformation (Clough and Bent (1998) Plant J<br>
16:735-743), the 35S::EAT transformants display the identical early-flowering and ap2-like<br>
phenotypes seen for EAT-D (Table 1). Many of the 35S::EAT transformants occasionally<br>
display additional defects, including stigmatic papillae on cauline leaf margins and the formation<br>
of a complete or partial flower rather than a secondary inflorescence in the axils of cauline<br>
leaves. Ectopic expression of the EAT gene in 35S::EAT plants, therefore, affects both<br>
flowering time and the specification of floral organ identity.<br>
[0166] The EAT gene produces a 1417-nucleotide noncoding RNA that is predicted to be 5' -<br>
capped and polyadenylated, based on the RACE-PCR methodology. BLASTN and BLASTX<br>
searches of several databases with the EAT cDNA do not reveal extensive nucleotide oi<br>
predicted amino acid sequence identity between EAT and any other gene. However, a 21-<br>
nucleotide (nt) (SEQ ID NO:4) stretch in the middle of the EAT transcript is identified that is<br>
identical to miR172a-2, a recently identified miRNA (Park et ah (2002) Curr Biol 12:1484-<br>
1495). To confirm the functional importance of the miR172a-2 sequence within the EAT<br>
cDNA, a mutant form of EAT is generated in which the miR172a-2 sequence is deleted, and a<br>
construct consisting of this mutant EAT cDNA, eatdel, is made driven by the 35S promoter:.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
50<br>
Transgenic plants carrying this 35S::eatdel construct flower with the same number of leaves as<br>
wild-type and had normal flowers (Table 1), indicating that the miRl72a-2 sequence is<br>
necessary to confer both the flowering time and floral organ identity phenotypes seen in EAT-<br>
overexpressing lines.<br>
[0167] As noted by Park et al. (2002) Curr Biol 12:1484-1495), the 21-nt miR172a-2<br>
miRNA has the potential to form an RNA duplex with a sequence near the 3' end of the coding<br>
region ofAP2 (Table 3).<br>
TABLE 3<br>
Putative 21-nt miR172a-2/AP2 RNA duplex<br><br>
Sequence	Duplex	SEQ ID NO:<br>
AP2KNA	5 '-CUGCAGCAUCAUCAGGAUUCU-3'	47<br>
EAT miRNA	3 '-U ACGUCGUAGUAGUUCUAAGA-5'	48<br>
The GU wobble in the duplex is underlined.<br>
[0168] This particular region of the AP2 gene is poorly conserved at the nucleotide level<br>
among the AP2 family; nevertheless, the AP2 sequence (SEQ ID NO:49) that is complementary<br>
to miRl72a-2 is found in a similar location in three other Arabidopsis AP2 family members,<br>
At5g60120 (SEQ ID NO:50), At2g28550 (SEQ ID NO:51), At5g67180 (SEQ ID NO:52). In<br>
addition, the sequence can be found at the corresponding positions of the maize AP2 genes<br>
indeterminate spikeletl (Chuck et al. (1998) Genes Dev 12:1145-1154) (IDS I (SEQ ID NO:53))<br>
and glossyl5 (Moose and Sisco (1996) Genes Dev 10:3018-3027) (GL15 (SEQ ID NO:54)), and<br>
in AP2 family members from many other plant species, including soybean, rice, wheat, tomato<br>
and pea (not shown). The alignment of three Arabidopsis and two maize AP2 family members<br>
is shown in Table 4 below.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
51<br>
[0169] There is an additional copy of the imR172a-2 miRNA in the Arabidopsis genome on<br>
chromosome 2 (miR172a-l, Fig. 2d), and miR172a-2 is highly similar to three other Arabidopsis<br>
loci. Like the miR172a-2 miRNA, all four reiterations of the sequence are in intergenic regions,<br>
i.e. in between the Arabidopsis genes currently annotated in GenBank. In addition, the sequence<br>
is found in ESTs from tomato, potato and soybean, and four copies were found in the genomic<br>
sequence of rice.<br>
EXAMPLE 2<br>
[0170]      This example describes the construction of expression vectors<br>
[0171] To overexpress the EAT gene, primers containing Xhol sites (SEQ ID NO:55 5'-<br>
GACTACTCGAGCACCTCTCACTCCCTTTCTCTAAC-3' and SEQ ID NO:56- 5'-<br>
GACTACTCGAGGTTCTCAAGTTGAGCACTTGAAAAC-3') are designed to amplify the<br>
entire EAT gene from Col-0 DNA. The PCR product is digested with Xhol and inserted into a<br>
modified pBluescriptSK+ vector (Stratagene, La Jolla, CA) that lacked BamHI and Hindlll sites,<br>
to generate EATX4 (SEQ ID NO:44). To generate the 35S::EAT transformants', the Xhol-cut<br>
EAT gene is inserted into the binary vector pBE851 in between a CAMV 35S promoter and b-<br>
phaseolin terminator, and Col-0 was transformed by floral dip. To generate the eatdel construct,<br>
two oligonucleotides are synthesized (SEQ ID NO:57 5'-GATCCATGGAAGAAAGCTCAT<br>
CTGTCGTTGTTTGTAGGCGCAGCACCATTAAGATTCACATGGAAATTGATAAATAC-<br>
3' and SEQ ID NO:58 5'-CCTAAATTAGGGTTTTGATATGTATATTCAACAATCGACG<br>
GCTACAAATACCTAA-3') that completely recreated the BamHI/Hindlll fragment of the EAT<br>
cDNA except that it lacked the 21 nt rm'R172a-2 sequence located within the fragment. These<br>
two oligos are annealed to their synthesized complementary strands (SEQ ID NO:59 5'-TAG<br>
GGTATTTATCAATTTCCATGTGAATCTTAATGGTGCTGCGCCTACAAACAACGACAG<br>
ATGAGCTTTCTTCCATG-3' and SEQ ID NO.60 5'-AGCTTTAGGTATTTGTAGCCGTC<br>
GATTGTTGAATATACATATCAAAACCCTAATT-3') and Kgated to EATX4 that had been<br>
digested with BamHI and Hindlll, in a trimolecular ligation reaction. This resulted in the<br>
replacement of 159 bp of wild-type EAT sequence with the 138 bp mutant sequence. The eatdel<br>
cDNA is then subcloned into pBE851 and transformed as described above. BASTA is used to<br>
select in plants for both the EAT and eatdel overexpression constructs.<br>
[0172] To test whether another member of the miR172 family, miR172a-l, would confer a<br>
phenotype similar to that of miR172a-2, a construct containing the 35S promoter fused to the<br>
genomic region surrounding miR172a-l is generated.   Plants containing the 35S::miR172a-l<br><br>
WO 2006/044322	PCT/US2005/036373<br>
52<br>
construct flower early and display an ap2 phenotype (Table 1), indicating that miR172a-l<br>
behaves in an identical manner to miR172a-2 when overexpressed.<br>
[0173] All of the miR172 miRNA family members are located within a sequence context that<br>
allows an RNA hairpin to form (Figure 1). Presumably this hairpin is the substrate which is<br>
subsequently cleaved by a plant Dicer homolog to generate the mature miRNA. The location of<br>
the miRNA within the hairpin, i. e. on the 3' side of the stem, is conserved amongst all the<br>
members of the miR172 family, and this may reflect a structural requirement for processing of<br>
this particular miRNA family. The 21-nt miR172a-2 miRNA, therefore, is predicted to be a<br>
member of a family of miRNAs that have the capacity to regulate a subset of AP2 genes by<br>
forming an RNA duplex with a 21-nt cognate sequence in these genes.<br>
EXAMPLE 3<br>
[0174] Th&amp;example describes the analysis of microRNA expression and AP2 expression<br>
[0175] Total RNA is isolated from wild type and EAT-D whole plants that had already<br>
flowered, using TRIZOL reagent (Sigma). 50 mg of each RNA is subjected to electrophoresis<br>
on a 15% TBE-Urea Criterion gel (BioRad), electroblotted onto Hybond-N+ filter paper<br>
(Amersham) using a TransBlot-SD apparatus (BioRad). The filter is then hybridized at 37° C<br>
overnight in UltraHyb-Oligo buffer (Ambion) with 32P-labeled oligos. The oligos are 30-mers<br>
that corresponded to either the sense or antisense strands of the miR172a-2 miRNA, with 4-5 nt<br>
of flanking sequence on each side. The filter is washed twice at 37°C, in buffer containing 2X<br>
SSC and 0.5% SDS. For SI analysis, probe is made by end-labeling an oligo (SEQ ID NO:61)<br>
(5'-ATGCAGCATCATCAAGATTCTCATATACAT-35) with T4 polynucleotide kinase and<br>
32P. Hybridization and processing of SI reactions are carried out using standard protocols. For<br>
developmental analysis of miR172a-2 and miR172a-l, total RNA is isolated from plants at the<br>
various stages and tissues indicated in Example 4, using an Rneasy kit (Qiagen). RT-PCR is<br>
carried out using standard protocols, and utilized oligos specific for sequences adjacent to<br>
miR172a-2 (SEQ ID NO:62) (5'-GTCGGCGGATCCATGG AAGAAAGCTCATC-3' and<br>
(SEQ ID NO:63) 5'-CAAAGATCGATCCAGACTTCAATCAA TATC-3') or sequences<br>
adjacent to miR172a-l (SEQ ID NO:64) (5'-TAATTTCCGGAGCCAC GGTCGTTGTTG-3'<br>
and (SEQ ID NO:65) 5'-AATAGTCGTTGATTGCCGATGCAGCATC-3'). Oligos used to<br>
amplify the ACT11 (Actin) transcript were: (SEQ ID NO:66) 5'-<br>
ATGGCAGATGGTGAAGACATTCAG-3', and (SEQ ID NO:67) 5'-GAAGCACTTCCTGTG<br>
GACTATTGATG-3'.   RT-PCR analysis of AP2 is performed on RNA from floral buds, and<br><br>
WO 2006/044322	PCT/US2005/036373<br>
53<br>
utilized the following oligos: (SEQ ID NO:68) 5'-TTTCCGGGCAGCAGCAACATTGGTAG-<br>
3', and (SEQ ID NO:69) 5'-GTTCGCCTAAGTTAACAAGAGGATTTAGG-3'. Oligos used to<br>
amplify the ANT transcript are: (SEQ ID NO:70) 5'-GATCAACTTCAATGACTAACTCTG<br>
GTTTTC-3', and (SEQ ID NO:71) 5'-GTTATAGAGAGATTCATTCTGTTTCACATG-3'.<br>
[0176] Immunoblot analysis of AP2 is performed on proteins extracted from floral buds.<br>
Following electrophoresis on a 10% SDS-PAGE gel, proteins are transferred to a Hybond-P<br>
membrane (Amersham) and incubated with an antibody specific for A.P2 protein (aA-20, Santa<br>
Cruz Biotechnology). The blot is processed using an ECL-plus kit (Amersham).<br>
[0177] Northern analysis using probes both sense and antisense to the miR172a-2 miRNA<br>
identifies a small single-stranded RNA of 21-25 nucleotides accumulating to much higher levels<br>
in EAT-D mutant plants relative to wild type. The small amount of transcript seen in wild type<br>
presumably represents endogenous levels of not only the miR172a-2 rniRNA but also its family<br>
members, which are similar enough to cross-hybridize with the probe. The predicted miR172a-2<br>
hairpin is 117 nt in length (Fig. 1), a small amount of an -100 nt transcript accumulating is<br>
detected in EAT-D, this likely represents partially processed miR172a-2 hairpin precursor. SI<br>
nuclease mapping of the miR172a-2 miRNA provides independent confirmation of the 5' end of<br>
miR172a-2 reported by Park et al. ((2002) CurrBiol 12:1484-1495).<br>
EXAMPLE 4<br>
[0178] The example describes the developmental pattern of EAT miRNA expression.<br>
[0179] To address the wild-type expression pattern of miR172a-2 separate from its other<br>
Arabidopsis family members, RT-PCR is used to specifically detect a fragment of the 1.4 kb<br>
EAT full-length precursor transcript containing miR172a-2. EAT precursor transcript<br>
expression is temporally regulated, with little or no transcript detected two days after<br>
germination, and progressively more steady-state transcript accumulation seen as the plant<br>
approaches flowering. The precursor transcript of miR172a-l shows a similar temporal pattern<br>
of expression. Both miR172a-2 and miR172a-l precursor transcripts continue to be expressed<br>
after flowering has occurred, and accumulate in both leaves and floral buds. Expression of the<br>
precursors for the other rniR172 family members is not detected, perhaps due to their exclusive<br>
expression in tissue types not included in this analysis, or because their precursor transcripts are<br>
too transient to detect. The temporal expression pattern seen for miR172a-2 and miR172a-l is<br>
reminiscent of that observed for let-7 and lin-4, two miRNAs that control developmental timing<br><br>
WO 2006/044322	PCT/US2005/036373<br>
54<br>
in C. elegans (Feinbaum and Ambros (1999) Dev Biol 210:87-95; Reinhart et al. (2000) Nature<br>
403:901-906).<br>
EXAMPLE 5<br>
[0180] The levels of miR172 in various flowering time mutants are assessed, in an attempt to<br>
position miR172 within the known flowering time pathways. The levels of miR172 are not<br>
altered in any of the mutants tested, and the levels of the EAT transcript are identical in plants<br>
grown in long days versus plants grown in short days.<br>
EXAMPLE 6<br>
[0181]      The example describes evaluation of protein expression<br>
[0182] Immunoblot analysis indicates that AP2 protein is reduced 3.5-fold in the EAT-D<br>
mutant relative to wild type, whereas the AP2 transcript is unaffected. This data suggests tha.t<br>
the miR172a-2 miRNA negatively regulates AP2 by translational inhibition. The predicted<br>
near-perfect complementarity between the miR172a-2 miRNA and the AP2 target site would be<br>
predicted to trigger AP2 mRNA cleavage by the RNA interference (RNAi) pathway (Llave et aZ.<br>
(2002) Plant Cell 14:1605-1619; Hutvagner and Zamore (2002) Science 297:2056-2060).<br>
Indeed, others have proposed that many plant miRNAs enter the RNAi pathway exclusively due<br>
to their near-perfect complementarity to putative targets (Rhoades et al. (2002) Cell 110:513-<br>
520). While there is no evidence regarding the GU wobble base pair in the predicted miR172a.-<br>
2/AP2 RNA duplex, it is conserved in all predicted duplexes between miR172 family members<br>
and their AP2 targets. Regardless of the mechanism, it is apparent from the AP2 expression data<br>
and the observed phenotype of EAT-D that AP2 is a target of negative regulation by miR172a-2,<br>
at least when miR172a-2 is overexpressed.<br>
EXAMPLE 7<br>
[0183] In the same genetic screen that identified the early-flowering EAT-D mutant, an<br>
activation-tagged late-flowering mutant, called LAT-D, is identified. The LAT-D mutant<br>
displays no additional phenotypes besides late flowering (Table 1), and the late-flowerin.g<br>
phenotype cosegregates with a single T-DNA insertion. Sequence analysis of the T-DNA inse:rt<br>
in LAT-D indicates that the 4X 35S enhancer is located approximately 5 kb upstream of<br>
At2g28550, which is one of the AP2-like target genes that are potentially regulated by miR172.<br>
RT-PCR   analysis   using   primers   specific   for   At2g28550   indicates   that   the   transcript<br><br>
WO 2006/044322	PCT/US2005/036373<br>
55<br>
corresponding to this gene is indeed expressed at higher levels in the LAT-D mutant relative to<br>
wild type. To confirm that overexpression of At2g28550 causes late flowering, a genomic<br>
region containing the entire At2g28550 coding region (from start to stop codon) is fused to the<br>
35S promoter, and transgenic plants containing this construct are created. Transgenic<br>
35S::At2g28550 plants flower later than wild type plants, and are slightly later than the LAT-D<br>
mutant (Table 1). This late flowering phenotype is observed in multiple independent<br>
transformants.<br>
[0184] The fact that overexpression of At2g28550 causes late flowering suggests that<br>
miR172 promotes flowering in part by downregulating At2g28550. However, because miR172<br>
appears to affect protein rather than transcript accumulation of its target genes, and because<br>
there is not an antibody to the At2g28550 gene product, this regulation is tested indirectly via a<br>
genetic cross. A plant heterozygous for LAT-D is crossed to a plant homozygous for EAT-D,<br>
such that all Fl progeny would contain one copy of EAT-D and 50%&gt; of the Fl progeny would<br>
also have one copy of LAT-D. Fl progeny are scored for the presence or absence of the LAT-D<br>
allele by PCR, and also are scored for flowering time. All of the Fl plants are early flowering,<br>
regardless of whether or not they contained a copy of the LAT-D allele, indicating that EAT-D<br>
is epistatic to LAT-D. This result is consistent with the idea that miR172a-2, which is<br>
overexpressed in EAT-D, directly downregulates At2g28550, which is overexpressed in LAT-D.<br>
EXAMPLE 8<br>
[0185] To assess the effects of reducing At2g28550 function, plants containing a T-DNA<br>
insertion in the At2g28550 gene are identified. In addition, a T-DNA mutant for At2g60120, a<br>
closely related AP2-like gene that also contains the miR172 target sequence, is identified.<br>
Plants homozygous for either the At2g28550 insert or the At5g60120 insert are slightly early<br>
flowering relative to wild type (Table 1). The two mutants are crossed, and the double mutant is<br>
isolated by PCR genotyping. The At2g28550/At5g60120 double mutant is earlier flowering<br>
than either individual mutant (Table 1), suggesting that the genes have overlapping function.<br>
The early flowering phenotype of the At2g28550/At5g60120 double mutant is consistent with<br>
the idea that the early flowering phenotype of miR172-overexpressing lines is due to<br>
downregulation of several AP2-like genes, including At2g28550 and At5g60120. Interestingly,<br>
the At2g28550/At5g60120 double mutant is not as early as miR172-overexpressing lines (c.f.<br>
EAT-OX, Table 1), which suggests that other AP2-like targets of miR172, for example AP2<br>
itself or At5g67180, also contribute to flowering time control.   Because ap2 mutants are not<br><br>
WO 2006/044322	PCT/US2005/036373<br>
56<br>
early flowering, any potential negative regulation of flowering by AP2 must be normally<br>
masked by genetic redundancy.<br>
EXAMPLE 9<br>
[0186] This example describes a method of target selection and method to design DNA<br>
constructs to generate miRNAs using the constructs of SEQ ID NOS: 3 and 44. Any sequence<br>
of interest can be selected for silencing by miRNA generated using the following method:<br>
[0187] 1. Choose a region from the coding strand in a gene of interest to be the target<br>
sequence. Typically, choose a region of about 10-50 nucleotides found in a similar location to<br>
the region targeted by EAT in AP2-like genes, which are regions about 100 nt upstream of the<br>
stop codon. The exact location of the target, however, does not appear to be critical. It is<br>
recommended to choose a region that has -50% GC and is of high sequence complexity, i.e. no<br>
repeats or long polynucleotide tracts. It is also recommended that the chosen region ends with a<br>
T or A, such that the complementary miRNA will start with an A or U. This is to help ensure a<br>
lower stability at the 5' end of the miRNA in its double-stranded Dicer product form (Schwartz,<br>
et al. (2003) Cell 115:199-208). For example, in the miR172a-2 precursor, the miRNA<br>
sequence starts with an A, and many other miRNAs start with a U.<br>
[0188] 2. To use the construct of SEQ ID NO:3, create a 21 nucleotide sequence<br>
complementary to the 21 nt target region (miRNA). Optionally, change a C in the miRNA to a<br>
T, which will generate a GU wobble with the target sequence, which mimics the GU wobble<br>
seen in EAT.<br>
[0189] 3. Create the 21 nucleotide "backside" sequence of the hairpin. This will be<br>
substantially complementary to the miRNA from step 2. Note, this backside sequence will also<br>
be substantially identical to the target sequence. Typically, introduce a few mismatches to make<br>
some bulges in the stem of the hairpin that are similar to the bulges in the original EAT hairpin.<br>
Optionally, introduce an A at the 3' end of the backside, to create mismatch at the 5' end of the<br>
miRNA. This last step may help ensure lower stability at the 5' end of the miRNA in its double-<br>
stranded Dicer product form (Schwartz et al. (2003) Cell 115:199-208).<br>
[0190] 4. Replace the 21 nucleotide miRNA sequence and the 21 nucleotide "backside"<br>
sequence in the EAT BamHI/Hindlll DNA construct (SEQ ID NO:3) with the new miRNA and<br>
"backside" sequences from steps 2 and 3.<br>
[0191] 5. Use MFOLD (GCG, Accelrys, San Diego, CA), or an equivalent program, to<br>
compare the new hairpin from Step 4 with the original hairpin.    Generally, the sequence<br><br>
WO 2006/044322	PCT/US2005/036373<br>
57<br>
substantially replicate the structure of the original hairpin (Figure 1).   It is predicted that the<br>
introduced bulges need not be exactly identical in length, sequence or position to the original.<br>
Examine the miRNA sequence in the hairpin for the relative stability of the 5' and 3' ends of the<br>
predicted dsRNA product of Dicer.<br>
[0192]      6.  Generate four synthetic oligonucleotides of 16-11 nucleotides in length to produce<br>
two double-stranded fragments which comprise the BamHI and Hindffl restriction sites, and a 4<br>
nucleotide overhang to facilitate directional ligation which will recreate the BamHI/Hindlll<br>
fragment.   Design of the overhang can be done by one of skill in the art, the current example<br>
uses the 4 nucleotide region of positions 79-82 (CCTA) of SEQ ID NO.3. Hence, for example:<br>
[0193]      Oligo 1 will have an unpaired BamHI site at the 5' end, and will end with the<br>
nucleotide at position 78 of SEQ ID NO:3.<br>
[0194]      Oligo 2 will have the nucleotides of position 79-82 (CCTA) unpaired at the 5' end,<br>
and will terminate just before the Hindlll site (or positions 151-154 in SEQ ID NO: 3).<br>
[0195]      Oligo 3 will be essentially complementary to Oligo 1, (nucleotides 5-78 of SEQ ID<br>
NO:3), and will terminate with 4 nucleotides complementary to nucleotides 1-4 (CCTA) of<br>
Oligo 2.<br>
[0196]      Oligo 4 will be essentially complementary to Oligo 2 beginning at the nucleotide of<br>
position 5, and will terminate with the Hindlll site at the 3' end.<br>
[0197]      Anneal the oligonucleotides to generate two fragments to be used in a subsequence<br>
ligation reaction with the plasmid sequence.<br>
[0198]      Optionally,   two   synthetic   oligonucleotides  comprising   attB   sequences  can  be<br>
synthesized and annealed to create an attB-flanked miRNA precursor that is then integrated into<br>
a vector using recombinational cloning (GATEWAY, InVitrogen Corp., Carlsbad, CA).<br>
[0199]      7. Ligate the two DNA fragments from Step 6 in a trimolecular ligation reaction with<br>
a plasmid cut with BamHI/Hindlll.   The current example uses the modified pBluescript SK+<br>
plasmid of SEQ ID NO:44, which comprises the 1.4kb EAT sequence of SEQ ID NO:1,<br>
digested with BamHI/Hindlll and gel purified away from the small fragment using standard<br>
molecular biological techniques. The new designed miRNA to the gene of interest has replaced<br>
the previous miRNA.<br>
[0200]      If an attB-flanked sequence is used from Step 6, the BP and LR recombination<br>
reactions (GATEWAY, InVitrogen Corp., Carlsbad, CA) can be used to insert the modified<br>
hairpin into a destination vector comprising the full-length miR172a-2 precursor.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
58<br>
[0201] 8. The plasmid from Step 7, subject to any other preparations or modifications as<br>
needed, is used to transform the target organism using techniques appropriate for the target.<br>
[0202] 9. Silencing of the target gene can be assessed using techniques well-known in the<br>
art, for example, Northern blot analysis, imrnunoblot analysis if the target gene of interest<br>
encodes a polypeptide, and any phenotypic screens relevant to the target gene, for example<br>
flowering time, or floral morphology.<br>
EXAMPLE 10<br>
[0203]      Described  in  this  example  are methods  one  may use   for  introduction  of a<br>
polynucleotide or polypeptide into a plant cell.<br>
[0204]      A. Maize particle-mediated DNA delivery<br>
[0205]      A DNA construct can be introduced into maize cells capable of growth on suitable<br>
maize culture medium.   Such competent cells can be from maize suspension culture, callus<br>
culture on solid medium, freshly isolated immature embryos or meristem cells.    Immature<br>
embryos of the Hi-II genotype can be used as the target cells.     Ears are harvested  at<br>
approximately 10 days post-pollination, and 1.2-1.5mm immature embryos are isolated from the<br>
kernels, and placed scutellum-side down on maize culture medium.<br>
[0206]      The immature embryos are bombarded from 18-72 hours after being harvested from<br>
the ear.   Between 6 and 18 hours prior to bombardment, the immature embryos are placed on<br>
medium with additional osmoticum (MS basal medium, Musashige and Skoog (1962) Physiol<br>
Plant 15:473-497, with 0.25 M sorbitol). The embryos on the high-osmotic medium are used as<br>
the bombardment target,  and are left on this medium for an additional  18 hours after<br>
bombardment.<br>
[0207]      For particle bombardment, plasmid DNA (described above) is precipitated onto 1.8<br>
mm tungsten particles using standard CaC12- spermidine chemistry (see, for example, Klein et<br>
al. (1987) Nature 327:70-73). Each plate is bombarded once at 600 PSI, using a DuPont Helium<br>
Gun (Lowe et al. (1995) Bio/Technol 13:677-682).   For typical media formulations used for<br>
maize immature embryo isolation, callus initiation, callus proliferation and regeneration of<br>
plants, see Armstrong (1994) In The Maize Handbook, M. Freeling and V. Walbot, eds. Springer<br>
Verlag, NY, pp 663-671.<br>
[0208]      Within 1-7 days after particle bombardment, the embryos are moved onto N6-based<br>
culture medium containing 3 mg/1 of the selective agent bialaphos.  Embryos, and later callus,<br><br>
WO 2006/044322	PCT/US2005/036373<br>
59<br>
are transferred to fresh selection plates every 2 weeks. The calli developing from the immature<br>
embryos are screened for the desired phenotype. After 6-8 weeks, transformed calli are<br>
recovered.<br>
[0209]      B. Soybean transformation<br>
[0210] Soybean embryogenic suspension cultures are maintained in 35 ml liquid media<br>
SB 196 or SB 172 in 250 ml Erlenmeyer flasks on a rotary shaker, 150 rpm, 26C with cool white<br>
fluorescent lights on 16:8 hr day/night photoperiod at light intensity of 30-35 uE/m2s. Cultures<br>
are subcultured every two weeks by inoculating approximately 35 mg of tissue into 35 ml of<br>
fresh liquid media. Alternatively, cultures are initiated and maintained in 6-well Costar plates.<br>
[0211] SB 172 media is prepared as follows: (per liter), 1 bottle Murashige and Skoog<br>
Medium (Duchefa # M 0240), 1 ml B5 vitamins 1000X stock, 1 ml 2,4-D stock (Gibco 11215-<br>
019), 60 g sucrose, 2 g MES, 0.667 g L-Asparagine anhydrous (GibcoBKL 11013-026), pH 5.7.<br>
SB 196 media is prepared as follows: (per liter) 10ml MS FeEDTA, 10ml MS Sulfate, 10ml FN-<br>
Lite Halides, 1 Oml FN-Lite P,B,Mo, lml B5 vitamins 1000X stock, 1 ml 2,4-D, (Gibco 11215-<br>
019), 2.83g KNO3 , 0.463g (NH4)2SO4, 2g MES, lg Asparagine Anhydrous, Powder (Gibco<br>
11013-026), lOg Sucrose, pH 5.8. 2,4-D stock concentration 10 mg/ml is prepared as follows:<br>
2,4-D is solubilized in 0.1 N NaOH, filter-sterilized, and stored at -20°C. B5 vitamins 1000X<br>
stock is prepared as follows: (per 100 ml) - store aliquots at -20°C, 10 g myo-inositol, 100 mg<br>
nicotinic acid, 100 mg pyridoxine HC1, 1 g thiamin.<br>
[0212] Soybean embryogenic suspension cultures are transformed with various plasmids by<br>
the method of particle gun bombardment (Klein et al. (1987) Nature 327:70). To prepare tissue<br>
for bombardment, approximately two flasks of suspension culture tissue that has had<br>
approximately 1 to 2 weeks to recover since its most recent subculture is placed in a sterile 60 x<br>
20 mm petri dish containing 1 sterile filter paper in the bottom to help absorb moisture. Tissue<br>
(i.e. suspension clusters approximately 3-5 mm in size) is spread evenly across each petri plate.<br>
Residual liquid is removed from the tissue with a pipette, or allowed to evaporate to remove<br>
excess moisture prior to bombardment. Per experiment, 4-6 plates of tissue are bombarded.<br>
Each plate is made from two flasks.<br>
[0213] To prepare gold particles for bombardment, 30 mg gold is washed in ethanol,<br>
centrifuged and resuspended in 0.5 ml of sterile water. For each plasmid combination<br>
(treatments) to be used for bombardment, a separate micro-centrifuge tube is prepared, starting<br>
with 50 ml of the gold particles prepared above.  Into each tube, the following are also added;<br><br>
WO 2006/044322	PCT/US2005/036373<br>
60<br>
5 ml of plasmid DNA (at l mg/ml), 50ml CaC12, and 20ml 0.1 M spermidine. This mixture is<br>
agitated on a vortex shaker for 3 minutes, and then centrifuged using a microcentrifuge set at<br>
14,000 RPM for 10 seconds. The supernatant is decanted and the gold particles with attached,<br>
precipitated DNA are washed twice with 400 ul aliquots of ethanol (with a brief centrifugation<br>
as above between each washing). The final volume of 100% ethanol per each tube is adjusted to<br>
40ul, and this particle/DNA suspension is kept on ice until being used for bombardment.<br>
[0214] Immediately before applying the particle/DNA suspension, the tube is briefly dipped<br>
into a sonicator bath to disperse the particles, and then 5 mL of DNA prep is pipetted onto each<br>
flying disk and allowed to dry. The flying disk is then placed into the DuPont Biolistics<br>
PDS1000/HE. Using the DuPont Biolistic PDS1000/HE instrument for particle-mediated DNA<br>
delivery into soybean suspension clusters, the following settings are used. The membrane<br>
rupture pressure is 1100 psi. The chamber is evacuated to a vacuum of 27-28 inches of mercury.<br>
The tissue is placed approximately 3.5 inches from the retaining/stopping screen (3rd shelf from<br>
the bottom). Each plate is bombarded twice, and the tissue clusters are rearranged using a sterile<br>
spatula between shots.<br>
[0215] Following bombardment, the tissue is re-suspended in liquid culture medium, each<br>
plate being divided between 2 flasks with fresh SB196 or SB172 media and cultured as<br>
described above. Four to seven days post-bombardment, the medium is replaced with fresh<br>
medium containing a selection agent. The selection media is refreshed weekly for 4 weeks and<br>
once again at 6 weeks. Weekly replacement after 4 weeks may be necessary if cell density and<br>
media turbidity is high.<br>
[0216] Four to eight weeks post-bombardment, green, transformed tissue may be observed<br>
growing from untransformed, necrotic embryogenic clusters. Isolated, green tissue is removed<br>
and inoculated into 6-well microtiter plates with liquid medium to generate clonally-propagated,<br>
transformed embryogenic suspension cultures.<br>
[0217] Each embryogenic cluster is placed into one well of a Costar 6-well plate with 5mls<br>
fresh SB 196 media with selection agent. Cultures are maintained for 2-6 weeks with fresh<br>
media changes every 2 weeks. When enough tissue is available, a portion of surviving<br>
transformed clones are subcultured to a second 6-well plate as a back-up to protect against<br>
contamination.<br>
[0218] To promote in vitro maturation, transformed embryogenic clusters are removed from<br>
liquid SB 196 and placed on solid agar media, SB 166, for 2 weeks. Tissue clumps of 2 - 4 mm<br>
size are plated at a tissue density of 10 to 15 clusters per plate. Plates are incubated in diffuse,<br><br>
WO 2006/044322	PCT/US2005/036373<br>
61<br>
low light (
for 3 - 4 weeks.<br>
[0219| SB 166 is prepared as follows: (per liter), 1 pkg. MS salts (Gibco/ BRL - Cat# 11117-<br>
017), 1 ml B5 vitamins 1000X stock, 60 g maltose, 750 mg MgC12 hexahydrate, 5 g activated<br>
charcoal, pH 5.7, 2 g gelrite. SB 103 media is prepared as follows: (per liter), 1 pkg. MS salts<br>
(Gibco/BRL - Cat# 11117-017), 1 ml B5 vitamins 100OX stock, 60 g maltose, 750 mg MgC12<br>
hexahydrate, pH 5.7, 2 g gelrite. After 5-6 week maturation, individual embryos are desiccated<br>
by placing embryos into a 100 X 15 petri dish with a lcm2 portion of the SB 103 media to create<br>
a chamber with enough humidity to promote partial desiccation, but not death.<br>
[0220] Approximately 25 embryos are desiccated per plate. Plates are sealed with several<br>
layers of parafilm and again are placed in a lower light condition. The duration of the<br>
desiccation step is best determined empirically, and depends on size and quantity of embryos<br>
placed per plate. For example, small embryos or few embryos/plate require a shorter drying<br>
period, while large embryos or many embryos/plate require a longer drying period. It is best to<br>
check on the embryos after about 3 days, but proper desiccation will most likely take 5 to 7<br>
days. Embryos will decrease in size during this process.<br>
[0221] Desiccated embryos are planted in SB 71-1 or MSO medium where they are left to<br>
germinate under the same culture conditions described for the suspension cultures. When the<br>
plantlets have two fully-expanded trifoliate leaves, germinated and rooted embryos are<br>
transferred to sterile soil and watered with MS fertilizer. Plants are grown to maturity for seed<br>
collection and analysis. Healthy, fertile transgenic plants are grown in the greenhouse.<br>
[0222] SB 71-1 is prepared as follows: 1 bottle Gamborg's B5 salts w/ sucrose (Gibco/BRL -<br>
Cat# 21153-036), 10 g sucrose, 750 mg MgC12 hexahydiate, pH 5.7, 2 g gelrite. MSO media is<br>
prepared as follows: 1 pkg Murashige and Skoog salts (Gibco 11117-066), 1 ml B5 vitamins<br>
1000X stock, 30 g sucrose, pH 5.8, 2g Gelrite.<br>
EXAMPLE 11<br>
[0223] This example describes the design and synthesis of miRNA targets and hairpins<br>
directed to various gene targets found in maize, soy, and/or Arabidopsis, using the method<br>
described in Example 9.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
62<br>
[0224]      A. Targeting Arabidopsis AGAMOUS, At4gl 8960<br>
[0225] The miRNA sequence of SEQ ID NO:4 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 12-15, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br>
[0226] Arabidopsis thaliana Col-0 is transformed and grown as described in Example 1.<br>
After transformation with a vector comprising the miRNA of SEQ ID NO:4, 88% of the<br>
transformants exhibit a mutant AGAMOUS {ag) floral phenotype, characterized by the<br>
conversion of stamens to petals in whorl 3, and carpels to another ag flower in whorl 4<br>
(Bowman, et al. (1991) The Plant Cell 3:749-758). The mutant phenotype varies between<br>
transformants, with approximately 1/3 exhibiting a strong ag phenotype, 1/3 exhibiting an<br>
intermediate ag phenotype, and 1/3 exhibiting a weak ag phenotype. Gel electrophbresis and<br>
Northern Blot analysis of small RNAs isolated from the transformants demonstrates that the<br>
degree of the mutant ag phenotype is directly related to the level of antiAG miRNA, with the<br>
strongest phenotype having the highest accumulation of the processed miRNA (~ 21 nt).<br>
[0227]      B. Targeting Arabidopsis Apetela3 (AP3), At3g54340<br>
[0228]      Two miRNA targets from AP3 are selected and oligonucleotides designed.<br>
[0229]      The miRNA sequence of SEQ ID NO:5 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 16-19, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br>
[0230]      The miRNA sequence of SEQ ID NO: 6 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 20-23, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br>
[0231]      Arabidopsis thaliana Col-0 is transformed and grown as described in Example 1.<br>
After transformation with a vector comprising the miRNA of SEQ ID NO: 5, the transformants<br>
have novel leaf and floral phenotypes, but do not exhibit any mutant AP3 phenotype.   Gel<br>
electrophoresis and Northern analysis of RNA isolated from 2 week old. rosette leaf tissue from<br>
the transformants demonstrates that the highest accumulation of the processed miRNA (~ 21 nt)<br>
corresponds to the "backside" strand of the precursor, which evidently silences a different target<br>
sequence to produce the novel leaf and floral phenotypes.<br>
[0232] A new target sequence is selected, with the correct asymmetry in order for the<br>
miRNA target strand to be selected during incorporation into RISC (Schwartz et al. (2003) Cell<br>
115:199-208). The miRNA sequence of SEQ ID NO:6 is selected and designed. The sequence<br><br>
WO 2006/044322	PCT/US2005/036373<br>
63<br>
is put into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of<br>
SEQ ID NOS: 20-23, and ligating them into the BamHI/Hindlll backbone fragment of SEQ ID<br>
NO.44. Greater than 90% of the transformants show silencing for the AP3 gene, as<br>
demonstrated by floral phenotype and electrophoretic analysis. An approximately 21 nt miRNA<br>
(antiAP3b) is detected at high levels in the transgenic plants, and not in wild type control plants.<br>
RT-PCR analysis confirmed that the amount of AP3 transcript is reduced in the transformants,<br>
as compared to wild type control plants.<br>
[0233]      C. Targeting Maize Phytoene Desaturase<br>
[0234]      Two    miRNA   targets    from   phytoene    desaturase    (PDS)   are   selected   and<br>
oligonucleotides designed.<br>
[0235]      The miRNA sequence of SEQ ID NO:7 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 24-27, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br>
[0236]      The miRNA sequence of SEQ ID NO: 8 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 28-31, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br>
[0237]      D. Targeting Maize Phytic Acid biosynthetic enzymes<br>
[0238] Three maize phytic acid biosynthetic enzyme gene targets are selected and miRNA<br>
and oligonucleotides designed. Inositol polyphosphate kinase-2 polynucleotides are disclosed in<br>
PCT International published application No. WO 02/059324, herein incorporated by reference.<br>
Inositol 1,3,4-trisphosphate 5/6-kinase polynucleotides are disclosed in PCT International<br>
published application No. WO 03/027243, herein incorporated by reference. Myo-inositol 1-<br>
phosphate synthase polynucleotides are disclosed in PCT International published application No.<br>
WO 99/05298, herein incorporated by reference.<br>
[0239]      Inositol polyphosphate kinase-2 (IPPK2)<br>
[0240] The miRNA sequence of SEQ ID NO:9 is selected and designed. The sequence is put<br>
into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID<br>
NOS: 32-35, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID NO:44.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
64<br>
[0241]      Inositol 1,3,4-trisphosphate 5/6-kinase-5 (ITPK5)<br>
[0242] The miRNA sequence of SEQ ID NO: 10 is selected and designed. The sequence is<br>
put into the BamHI/Hindlll hairpin cassette by annealing the synthetic oligonucleotides of SEQ<br>
ID NOS: 36-39, and ligating them into the BamHI/Hindlll backbone fragment of SEQ ID<br>
NO: 44.<br>
[0243]      Myo-inositol 1-phosphate synthase (milps)<br>
[0244]      The miRNA sequence of SEQ ID NO: 11 is selected and designed.   The sequence is<br>
put into the BamHI/HindIII hairpin cassette by annealing the synthetic oligonucleotides of SEQ<br>
ID NOS: 40-43, and ligating them into the BamHI/HindIII backbone fragment of SEQ ID<br>
NO:44.<br>
[0245]      E. Targeting Soy Apetela2-like sequences (AP2)<br>
[0246] The same EAT (miR172a-2) construct, comprising SEQ ID NO:1, used for<br>
Arabidopsis transformation is used to transform soybean. This construct has a miRNA template<br>
sequence which encodes the miRNA of SEQ ID NO:48. The construct is created using a PCR<br>
amplification of miR172a-2 precursor sequence from Arabidopsis, restriction digestion, and<br>
ligation as described in Example 2.<br>
[0247] Soybean tissue is transformed and grown essentially as described in Example 10.<br>
After transformation, 42% of the transformants exhibit a mutant phexiotype, characterized by the<br>
conversion of sepals to leaves. Plants exhibiting the strongest phenotypes are sterile, and<br>
produce no seed. Both the homeotic conversion of the organs and the effects on fertility are<br>
similar to that seen for ap2 mutant alleles in Arabidopsis. Small RNA gel electrophoresis and<br>
Northern analysis, probed with an oligonucleotide probe antisense to miR172, shows<br>
accumulation of miR172 in the transgenic lines. A small amount of endogenous soy miR172 is<br>
also detected in the soy control line. The degree of the mutant phenotype is directly related to<br>
the level of miRNA, with the strongest phenotype having the highest accumulation of the<br>
processed miRNA (~ 21 nt).<br>
[0248]      F. Targeting Arabidopsis AP2-like genes<br>
[0249] The miRNA sequence of SEQ ID NO:72 is selected and designed. The sequence is<br>
put into the attB hairpin cassette by annealing the synthetic oligonucleotides of SEQ ID NOS:<br>
73-74, and performing the BP recombination reaction (GATEWAY) to generate the attL<br><br>
WO 2006/044322	PCT/US2005/036373<br>
65<br>
intermediate. This intermediate is used in the LR reaction to recombine with the destination<br>
vector, generally described in Example 12, comprising the EAT full-length precursor containing<br>
attR sites, and negative selection markers in place of the hairpin. The product of this reaction<br>
comprises the miR172a-2 precursor hairpin cassette flanked by attR sites (i.e., the hairpin<br>
replaces the marker cassette).<br>
[0250]      G. Targeting Arabidopsis Fatty Acid Desaturase (FAD2)<br>
[0251] The miRNA sequence of SEQ ID NO:75 is selected and designed based on the<br>
sequence of NM_112047 (At3gl2120). The sequence is put into the attB hairpin cassette by<br>
annealing the synthetic oligonucleotides of SEQ ID NOS: 76-77, and performing the BP<br>
recombination reaction (GATEWAY) to generate the attL intermediate. This intermediate is<br>
used in the LR reaction to recombine with the destination vector, generally described in<br>
Example 12, comprising the EAT full-length precursor containing attR sites, and negative<br>
selection markers in place of the hairpin. The product of this reaction comprises the FAD2<br>
miRNA precursor hairpin cassette flanked by attR sites (i.e., the hairpin replaces the marker<br>
cassette). The effect of the anti-FAD2 miRNA can be determined by fatty acid analysis to<br>
determine the change in the fatty acid profile, for example, see Wu et al. (1997) Plant Physiol.<br>
113:347-356, herein incorporated by reference.<br>
[0252]      H. Targeting Arabidopsis Phytoene Desaturase (PDS)<br>
[0253] The miRNA sequence of SEQ ID NO:78 is selected and designed based on the<br>
sequence of NM_202816 (At4gl4210). The sequence is put into the attB hairpin cassette by<br>
annealing the synthetic oligonucleotides of SEQ ID NOS: 79-80, and performing the BP<br>
recombination reaction (GATEWAY) to generate the attL intermediate. This intermediate is<br>
used in the LR reaction to recombine with the destination vector, generally described in<br>
Example 12, comprising the EAT full-length precursor containing attR sites, and negative<br>
selection markers in place of the hairpin. The product of this reaction comprises the PDS<br>
miRNA precursor hairpin cassette flanked by attR sites (i.e., the hairpin replaces the marker<br>
cassette). Transgenic plants containing the antiPDS construct are photobleached upon<br>
germination in greater than about 90% of the lines, indicating silencing of PDS.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
66<br>
EXAMPLE 12<br>
[0254] This example describes the construction of expression vectors using recombinational<br>
cloning technology.<br>
[0255] The vector described in Example 2 (SEQ ID NO:44) is modified to incorporate art<br>
recombination sites to facilitate recombinational cloning using GATEWAY technology<br>
(InVitrogen, Carlsbad, CA). The BamHI/HindIII segment is replaced with a sequence<br>
comprising in the following order: attRl - CAM - ccdB - attR2. Upon recombination (BP +<br>
LR) with oligos containing attB sites flanking the miRNA hairpin precursor construct, the<br>
selectable markers axe replaced by the miRNA hairpin precursor.<br>
EXAMPLE 13<br>
[0256] This example, particularly Table 5, summarizes the target sequences and oligos used<br>
for miRNA silencing constructs as described in the examples.<br><br>
		TABLE	5	<br>
Organism	Target gene	mIRNAname	miRNA template	Precursor oligosSEQ ID NOS<br>
Arabidopsis	AP2-like	miR172-a2	SEQ ID NO:86	55-56 (PCR)<br>
	none	EATdel	none	57-60<br>
	AGAMOUS	antiAG	SEQ ID NO: 4	12-15<br>
	APETELA3 (a)	antiAP3a	SEQIDNO:5	16-19<br>
	APETELA3 (b)	antiAP3b	SEQ ID NO: 6	20-23<br>
Corn	PDS1	antiPDSl	SEQ ID NO:7	24-27<br>
	PDS2	antiPDSl	SEQ ID NO:8	28-31<br>
	IPPK2	antiIPPK2	SEQ ID NO:9	32-35<br>
	ITPK5	antiITPK5	SEQ ID NO: 10	36-39<br>
	MIPS	antiMIlPS	SEQ ID NO: 11	40-43<br>
Soybean	AP2-like	miR172a-2	SEQ ID NO: 86	55-56 (PCR)<br>
Arabidopsis	AP2-like	miR172a-2	SEQIDNO:72	73-74<br>
	FAD2	antiFAD2	SEQ ID NO:75	76-77<br>
	PDS	antiAtPDS	SEQ ID NO:78	79-80<br>
Com	miR172b	miR172	SEQ ID NO:92	91<br>
	PDS	antiZinPDS	SEQ ID NO:95	94<br><br>
WO 2006/044322	PCT/US2005/036373<br>
67<br>
EXAMPLE 14<br>
[0257] This example describes the identification and isolation of genomic corn miR172<br>
precursors.<br>
[0258] The Genome Survey Sequence (GSS) database of the National Center for<br>
Biotechnology Information (NCBI) is searched using the 21nt miR172a-2 sequence in order to<br>
identify genomic com sequences containing miR172 precursor sequence. Several corn miR172<br>
precursors are identified, and named miR172a -miR172e (SEQ ID NOS: 81-85) as summarized<br>
in Table 6. Each sequence is imported into Vector NTI (InVirrogen, Carlsbad, CA) and contig<br>
analyses done. The analysis identifies four distinct loci, each with a unique consensus sequence.<br>
A region of about 200 nucleotides surrounding the miRNA sequence from each locus is<br>
examined for secondary structure folding using RNA Structure software (Mathews et al. (2004)<br>
Proc Natl Acad Sci USA 101:7287-7292, herein incorporated by reference). The results of this<br>
analysis identifies the hairpin precursors of each of the com sequences miR172a-e.<br><br>
Corn miR172 precursors and	TABLE 6positions of hairpin, &amp;	miRNA duplex components<br>
Precursor	NCBIED	CornLine	SEQ IDNO:	Length	Hairpin	Backside	miRNA<br>
miR172a	CG090465	B73	81	907	L508-598	L112-532	574-594<br>
miR172b	BZ401521andBZ4011525	B73(both)	82	1128	551-654	567-587	620-640<br>
miR172c	CG247934	B73	83	912	230-400	250-270	364-384<br>
miR172d	CGO9786OandBZ972414	B73	84	1063	351-520	361-381	466-486<br>
miR172e	CG065885andCC334589	B73(both)	85	1738	913-1072	931-951	1033-1053<br>
[0259] Oligonucleotides are designed in order amplify miR172a or miR172b from a B73<br>
20 genomic com library, these primers also add restriction enzyme recognition sites in order to<br>
facilitate cloning (BamHI or EcoRV). Alternatively, PCR primers designed to create art sites for<br>
recombinational cloning could be used. After PCR amplification, the products are isolated,<br>
purified, and confirmed by sequence analysis. Once confirmed, these sequences are inserted<br>
into a construct comprising the com ubiquitin (UBI) promoter.  This construct can be used for<br><br>
WO 2006/044322	PCT/US2005/036373<br>
68<br>
further transformation vector construction, for example, with the addition of art sites, the<br>
GATEWAY system can be used.<br>
[0260]      The following PCR primers are used to amplify a sequence comprising the hairpin<br>
precursor of corn miR172a<br>
Forward primer (SEQ ID NO:87): 5' GGATCCTCTGCACTAGTGGGGTTATT 3'<br>
Reverse primer (SEQ ID NO:88): 5'GATATCTGCAACAGTTTACAGGCGTT 3'<br>
[0261]      The following PCR primers are used to amplify a sequence comprising the hairpin<br>
precursor of corn miR172b<br>
Forward primer (SEQ ID NO.89): 5' GGATCCCATGATATAGATGATGCTTG 3'<br>
Reverse primer (SEQ ID NO.90): 5' GATATCAAGAGCTGAGGACAAGTTTT 3'<br>
EXAMPLE 15<br>
[0262] This example describes the design and synthesis of miRNA targets and hairpins<br>
directed to various gene targets found in maize, for use with the com miR172b miRNA<br>
precursor.<br>
[0263]      A. miR172b target in corn<br>
[0264] Similar to the Arabidopsis EAT examples, the corn miR172b hairpin precursor will be<br>
tested by overexpression in com. The precursor sequence comprising the miRNA template is<br>
shown in SEQ ID NO.91. The miRNA is shown in SEQ ID NO:92, and the backside of the<br>
miRNA duplex is shown in SEQ ID NO:93. A double-stranded DNA molecule comprising the<br>
miRNA precursor and restriction enzyme overhangs, for BamHI and Kpnl, is created by<br>
annealing the oligonucleotides of SEQ ID NOS: 97 and 98.<br>
[0265]      B. Phytoene Desaturase (PDS)<br>
[0266] An oligonucleotide comprising the the miRNA template is shown in SEQ ID NO:94.<br>
The miRNA directed to PDS is shown in SEQ ID NO:92, and the backside of the miRNA<br>
duplex is shown in SEQ ID NO.93. A double-stranded DNA molecule comprising the miRNA<br>
precursor and restriction enzyme overhangs, for BamHI and Kpnl, is created by annealing the<br>
oligonucleotides of SEQ ID NOS: 99 and 100.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
69<br>
[0267] The oligonucleotides of this example can be inserted into vectors for transformation<br>
of corn using standard cloning techniques, including restriction digestion and ligation, and/or<br>
recombinational cloning such as GATEWAY.<br>
EXAMPLE 16<br>
[0268]      This example describes the materials and methods used for Examples 17-19.<br>
[0269]      Plasmid constructs<br>
[0270] A fragment of 276 base pairs containing the entire sequence of Arabidopsis miR159a<br>
(see below) was cloned by PCR amplification using primers CACC-miR159a-prec: 5'<br>
CACCACAGTTTGCTTATGTCGGATCC 3' (SEQ ID NO: 101) and miR159a-Xma: 5'<br>
TGACCCGGGATGTAGAGCTCCCTTCAATCC 3' (SEQ ID NO: 102). The miR159a-Xma<br>
contains 18 of 21 nucleotides of the mature miR159a (bold) and an introduced Xrnal site (italic).<br>
The PCR fragment was cloned in the pENTR/SD/D-TOPO vector (Invitrogen) according to<br>
manufacturers directions to obtain pENTR-miR159a-prec.<br>
[0271]      The Gateway recombination system was used to transfer the pre-miR159a sequence to<br>
the plant binary vector pK2GW7, which contains two copies of the 35S promoter and a NOS<br>
terminator to generate pK2-pre-miRl59a.<br>
[0272]      Mutagenesis   of pre-miR159a   was   performed   by   PCR   with   the   following<br>
oligonucleotides.<br>
[0273]      5'-mzi?-PDS159a: 5' AT^G^rCTTGATCTGACGATGGAAGAAGAGATCCTAAC<br>
T TTTCAAA 3' (SEQ ID NO:103; This oligonucleotide contains a natural Bgl II site (italic)<br>
and the miR-PDS159a* sequence (bold)).<br>
[0274]      3'-miR-PDS159a: 5' TGACCCGGGATGAAGAGATCCCATATTTCCAAA 3' SEQ<br>
ID NO: 104; This oligonucleotide contains point mutations in the miR159a sequence (bold) to<br>
increase its complementarity to the PDS mRNA sequence, based on available N. benthamiana<br>
PDS mRNA partial sequence (Genbank AJ571700, see below)).<br>
[0275]      PCR amplification of the miR159a precursor using the above primers and pENTR-<br>
miRl 59a-prec DNA as template generated a DNA fragment that was digested with Bgin and<br>
Xmal  to  be  re-inserted into pENTR.-pre-miR159a,  to  generate pENTR.-pre-miR-PDS159a.<br>
Gateway system procedures were used again to transfer the miR-PDS159a precursor to pK2GW7<br>
and generate pK2-pre-miR-PDS159a.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
70<br>
[0276]      The miR-PDSl69g was cloned as follows. An Arabidopsis genomic fragment of 222<br>
base pairs containing the miR169g sequence (see below) was amplified using primers miR169g-<br>
For 5' CACCAATGATGATTACGATGATGAGAGTC 3' (SEQ ID NO: 105), and miR169g-<br>
Rev  5'  CAAAGTTTGATCACGATTCATGA  3'   (SEQ  ID NO: 106).  The resulting PCR<br>
fragment was introduced into pENTR/D-TOPO vector (Invitrogen) to obtain pENTR-^re-<br>
rniR169g. The pre-miR169g sequence was then transferred into binary vectors pBADC and<br>
pB2GW7 using the Gateway system to generate pBA-pre-tniRl 69g and pB2-pre-miRl 69g.<br>
[0277]      Two miR-PDSJ69g precursors were  created using -pENTR-pre-miRl 69g as template<br>
and the Quick-change Mutagenesis kit from Stratagene. pENTR-pre-7)miR-PD5a159g was made by<br>
using the following oligonucleotides:<br>
[0278]      miR169PDSa: 5' GAGAATGAGGTrGAGTTTAGTCTGACTTGGCCAGTTTTTTTA<br>
CCAATG 3' SEQ ID NO: 107), and<br>
[0279]      miR169PDSa* : 5' CTGATTCTGGTGTTGGCCAAGTCAGACTAAACTCTGTTTCC<br>
TTCTC 3' (SEQ ID NO: 108).<br>
[0280]      pENTR-pre-miR-PDSb169g was prodxiced by using the oligonucleotides:<br>
[0281]      miR169PDSb: 5' GAGAATGAGGTTGATCTCTrrCCAGTCTTCAGGGTTTTTTTA<br>
CCAATG 3' (SEQ ID NO: 109), and<br>
[0282]      miR169PDSb*: 5' GATTCTGGTGTCCTGAAGACTGGAAAGAGATCTGTTTCCTT<br>
CTCTTC 3' (SEQ ID NO: 110).<br>
[0283]      The two mutagenized miR-PDSl69g precursors above were then transferred into plant<br>
binary vectors pBADC  and pB2GW7 to   generate  pBA-pre-miR-PDSa169g, pB2-pre-miR-<br>
PDSa169g; pBA-pre-miR-PDSb169g, and pB2-pre-miR-PDSb169g.<br>
[0284]      Precursors for artificial miRNAs that target N. benthamiana rbcS transcripts (pENTR-<br>
pre-miR-rbcSl59a-A) were produced using similar procedures as those described for pENTR-<br>
miR-PDS159a using the following primers and cloned into pK2GW7:<br>
[0285]      MrbcSA-S: 5' TCTGACGATGGAA.GTTCCTCGCCCGACATTCGAAAATGAGTT<br>
GA 3' (SEQ ID NO: 111), and<br>
[0286]      MrbcSA-R: 5' AAACCCGGGATG-TTCCTCGCCCGGAATTCGAAAGAGAGTAA<br>
AAG 3'(SEQ ID NO: 112).<br>
[0287]      All cloned sequences were confirmed by DNA sequencing.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
71<br>
[0288]      Precursor sequences used<br>
[0289]      miRl 59a precursor template sequence (276bp)<br>
ACAGTTTGCTTATGTCGGATCCATAATATATTTGACAAGATACTTTGTTTTTCGATAG<br>
ATCTTGATCTGACGATGGAAGTAGAGCTCCTTAAAGTTCAAACATGAGTTGAGCA<br>
GGGTAAAGAAAAGCTGCTAAGCTATGGATCCCATAAGCCCTAATCCTTGTAAAGTA<br>
AAAAAGGATTTGGTTATATGGATTGCATATCTCAGGAGrCTTTAACTTGCCCTTTAAT<br>
GGCTTTTACICTYCTTTGGATTGAAGGGAGCTCTACATCCCGGGTC (SEQ ID NO:113)<br>
(Sequence of the pre-miR159a cloned. Sequences of miRl 59a* and miRl59a (italic) are shown<br>
in bold. Nucleotides changed in miR-PDS159a are underlined.)<br>
[0290]      miR-159a mature template<br>
5' TTTGGATTGAAGGGAGCTCTA 3' (SEQ ID NO: 114)<br>
[0291]      miR-PDS159a mature template<br>
5' TTTGGA a a t A t GGGAt CTCT13' (SEQ ID NO: 115)<br>
[0292]      miR l69g precursor template sequence 0.3 kb (222bp)<br>
AATGATGATTACGATGATGAGAGTCTCTAGTTGTATCAGAGGGTCTTGCATGGAAG<br>
AATAGAGAATGAGGTTGAGCCAA GGA TGA CTTGCCGGGTTTTTTTACCAATGAATCT<br>
AATTAACTGATTCTGGTGTCCGGCAAGTTGACCTTGGCTCTGTTTCCTTCTCTTCTT<br>
TTGGATGTCAGACTCCAAGATATCTATCATCATGAATC GTGATCAAACTTTG (SEQ<br>
ID NO: 116) (Sequence of the pre-miR169g fragment (0.3kb) cloned. Sequences of rm'R169g<br>
(italic) and miR169g* are shown in bold. Nucleotides changed in vniR-PDS ®8 are underlined.)<br>
[0293]      miRl 69g mature template<br>
5' GAGCCAAGGATGACTTGCCGG 3' (SEQ IDNO:117)<br>
[0294]      miK-PDSa169g mature template<br>
5' GAG 111AG t c TGACTTG gC c a 3' (SEQ ID NO: 118)<br>
[0295]      rm'R169g mature template<br>
5' GAGCCAAGGATGACTTGCCGG 3' (SEQ ID NO: 119)<br>
[0296]      miR.-PDSb169g mature template<br>
5' GA tC t c t t t c c a g t c T t C aGG 3' (SEQ ID NO:120)<br>
[0297]      miRl69g precursor template sequence 2.0kb (2474bp)<br>
AAGCTTTGATCTITAGCTCTTTGCCAAAGCTTCTTTTGA.TTTTTCTATTTCTCTAATCT<br>
ATCCATTGACCATTTGGGGTGATGATATTCTTCAATTTATGTTGTTGTTTATTGCCCA<br>
TCCACAGACCCACGTTTGATTTGTTTAATCAAAATATATAAACTGACAGTTGTGCCA<br>
CTAGTCACTTGCCAATTAAGCATTCCAAAGCTCCTTCCTTTACATTAGTATCAAGTG<br>
AGACTAGCACAAGCTTTTAAGTCCAGATAAAAAGCCCCATGGAAGGGAAGCTTTCA<br>
AGAACGAGATTTAACCGTAAAACCCAATTTCGATTTCCGCTAATAATTTGGATCCAA<br>
AAATCTAGACAAAATCTGATAAAATTAGACAAAGAAATGGATAAAACCCCAAAAC<br>
CCATAATCGTCGTTGTTCTTGTTTGCTTCAATATCACTCTTTCCCCTCCAACGAGTTA<br>
GTTAGAGTGACGTGGCAGCTGAACTAGATTTGGAGTAACGGGATAGATTACCCATA<br>
AAGCCCAATAATGATCATTACGTGAGACATAACTTGCTTAGATAACCTCATTTTATG<br><br>
WO 2006/044322	PCT/US2005/036373<br>
72<br>
GGCTTAGATGGGGTCTCTAGTGTTAGTCATAAGCTCTTAAATACCATTTCTAGTTAT<br>
ATATCAATCTTTAGCTTGGAATTGGATCGTTGTCCTATAGTAAAAAAACTTTTACTA<br>
TTTTATGTTAGCAATCCCACTTAACATTCAATATGTTTAAAATGAAAGAGTTTACCA<br>
AAAGGAAAGAAAAAAAGGTTGGTAATGAATTTATCTAATCGGATACGATATTTCAT<br>
AATCTAATGATGGGATCTATCAATAAATAGAATCAAAGTTAACTTTAACGCTTTTGT<br>
TACCTGTTTTCTTTCTTTAGCAATTAATATTAAACGAGTTTTAGTAATATAAATATGT<br>
TTCCAGTTATATACCAAACTTTATGTAATATTCATAAGCTTGCCAAAATTTACA^GA<br>
GTTTTTGGAACGCGCACAAAATTCTCATATATTTCTTACCCAAAAATAAATTTTTTTT<br>
TTTTTTTTACTTGTTTATAATCCTATATGAACATTGCTCATCTTCCCCATTTGATG-GTA<br>
ATTTTTCTATTCCTATATGTAATTAAATCCTAACTAATGAAATTGAAAACATAATTT<br>
GAAGATAATCAATCCTAATATCTCCCGTCTTAGATCTATTTAAATGGTCTTATTTAAT<br>
TTCCTATATTTTGGCCTAATTATTTATTTGATATAGTGAATTTATGGAAGCTTCATGT<br>
TGATGGAATAAAACCGGCTTATCCCAATTAATCGATCGGGAGCTATAACACAAATC<br>
GAAACTCTAGTAGCTATAAAGAGTGTGTAATAGCTTTGGATCACATGTATTACTATT<br>
TATTTACTAGCTCGTGCAACAATTGGCTTTGGGAAAAAATTTATTTACTAGTACXCC<br>
CCCTTCACAATGTGATGAGTCTCCAAATGATATATTCTCAACCCAAAGGACAArCTG<br>
AAATTTTCAATATATATTCCATTTTATCCGCAACATTTGAAATTTGTGGCAATGrTTT<br>
TAAAAAGACTATTTATAAAGAATCTTTCTAAATTGTTTCTACGACAATCGATAACAC<br>
CTTTTGTTGATCAACCCCACACAAGACTATGATTCCAATCCTAAGAAACATACG-ACA<br>
CGTGGATTTTTATGTCACACTAGTACGATGCGTCGATGCCTTCAGAGTACGAATATT<br>
ATTCACATAAAATTCTTATTCGAATTTGATAATATAAGGTCAGCCAATCTTTTAAAG<br>
TAATTATATTCTTCAATATACGGTTGTGGTCAAAATTCCATTTTATTTTGTAGCT'TGC<br>
ATGCACTACTAGTTTAAAACCATGCATGGATTTATTGCATATAATAACATTATA'TGA<br>
ATTTTCAATTAAATTAATCCACACATTTCCCATTTCAATATGCCTATAAATACCrTCA<br>
TCACGAGTATGACAAGATCACAAGACAAGAAAAGAAAGGTAGAGAAAACATG.ATA<br>
ATGATGATTACGATGATGAGAGTCTCTAGTTGTATCAGAGGGTCTTGCATGGAAGA<br>
ATAGAGAATGAGGTTGAGCCAAGGATGACTTGCCGGGTTTTTTTACCAATGAArCTA<br>
ATTAACTGATTCTGGTGTCCGGCAAGTTGACCTTGGCTCTGTTTCCTTCTCTTCTTT<br>
TGGATGTCAGACTCCAAGATATCTATCATCATGAATCGTGATCAAACTTTGTAA.TTT<br>
CATTGAAATGTGTTTTTCTTGATGCGAATTTTTTGGCTTACGGTTTTTCGATTTGAAT<br>
GATCAGATTTTTGTTTITGCACTCAAACTATAGTTTCACTTAGGTTCTATTTTTTTCA<br>
GGTTTATGAATGATAAAACAAGTAAGATTTTATGCTAGTTTTAGTTCATTTTTCGATT<br>
CAAATTCAAACATCTTGGTTTTGGTTTAGTTAAGTTTGATTTTTCAAGTCAAATGCTA<br>
TGTTTTCTTGT (SEQ ID NO:121)   (Sequence of the pre-miR169g fragment (2.0kb) cloned.<br>
Sequences of miR169g (italic) and miR169g* are shown in bold.)<br>
[0298]      Target gene sequences used:<br>
[0299]      Nicotiana benthamiana PDS sequences:<br>
[0300]      5'end probe sequence (corresponding to Le-PDS pos.1-268, see Fig. 15A):<br>
ATGCCTCAAATTGGACTTGTTTCTGCTGTTAACTTGAGAGTCCAAGGTAGTTCAGCT<br>
TATCTTTGGAGCTCGAGGTCGTCTTCTTTGGGAACTGAAAGTCGAGATGGTTGCTTG<br>
CAAAGGAATTCGTTATGTTTTGCTGGTAGCGAATCAATGGGTCATAAGTTAAAGATT<br>
CGTACTCCCCATGCCACGACCAGAAGATTGGTTAAGGACTTGGGGCCTTTAAAGGT<br>
CGTATGCATTGATTATCCAAGACCAGAGCTGGACAATACAG (SEQ ID NO: 122)<br><br>
WO 2006/044322	PCT/US2005/036373<br>
73<br>
[0301] Partial+5'RACE fragment. Assembled sequence from partial Nicotiana benthamiana<br>
PDS sequence (Genbank AJ571700) and 5'RACE experiments (corresponding to LQ-PDS pos.<br>
853-1514, see Fig. 15A).<br>
GGCACTCAACTTTATAAACCCTGACGAGCTTTCGATGCAGTGCATTTTGATTGCTTT<br>
GAACAGATTTCTTCAGGAGAAACATGGTTCAAAAATGGCCTTTTTAGATGGTAACC<br>
CTCCTGAGAGACTTTGCATGCCGATTGTGGAACATATTGAGTCAAAAGGTGGCCAA<br>
GTCAGACTAAACTCACGAATAAAAAAGATCGAGCTGAATGAGGATGGAAGTGTCA<br>
AATGTTTTATACTGAATAATGGCAGTACAATTAAAGGAGATGCTTTTGTGTTTGCCA<br>
CTCCAGTGG ATATCTTGAAGCTTCTTTTGCCTGAAGA CTGGAAA GA GA TCCCATATTT<br>
CCAAAAGTTGGAGAAGCTAGTGGGAGTTCCTGTGATAAATGTCCATATATGGTTTG<br>
ACAGAAAACTGAAGAACACATCTGATAATCTGCTCTTCAGCAGAAGCCCGTTGCTC<br>
AGTGTGTACGCTGACATGTCTGTTACATGTAAGGAATATTACAACCCCAATCAGTCT<br>
ATGTTGGAATTGGTATTTGCACCCGCAGAAGAGTGGATAAATCGTAGTGACTCAGA<br>
AATTATTGATGCTACAATGAAGGAACTAGCGAAGCTTTTCCCTGATGAAATTTCGGC<br>
AGATCAGAGCAAAGCAAAAATATTGAAGTACCATGT (SEQ ID NO: 123)    (Sequences<br>
targeted by miR-PDS&amp;169g (bold), miR-PDSb169g (bold and italic) and miR-PDS159a (underlined)<br>
are indicated.)<br>
[0302]      Nicotiana benthamiana rbcS sequences (Bolded nucleotides in all six rbcS gene<br>
sequences correspond to the sequence targeted by miR-rbcSl59a-A.):<br>
[0303]      rbcSl (Genbank accessions: CN748904: 56-633bp, CN748069:419-end)<br>
GGAGAAAGAGAAACTTTCTGTCTTAAGAGTAATTAGCAATGGCTTCCTCAGTTCTTT<br>
CCTCAGCAGCAGTTGCCACCCGCAGCAATGTTGCTCAAGCTAACATGGTTGCACCTT<br>
TCACAGGTCTTAAGTCTGCTGCCTCATTCCCTGTTTCAAGAAAGCAAAACCTTGACA<br>
TCACTTCCATTGCCAGCAACGGCGGAAGAGTGCAATGCATGCAGGTGTGGCCACCA<br>
ATTAACATGAAGAAGTATGAGACTCTCTCATACCTTCCCGATTTGAGCCAGGAGCA<br>
ATTGCTCTCCGAAATTGAGTACCTTTTGAAAAATGGATGGGTTCCTTGCTTGGAAT<br>
TCGAGACTGAGAAAGGATTTGTCTACCGTGAACACCACAAGTCACCAGGATACTAT<br>
GATGGCAGATACTGGACCATGTGGAAGCTACCTATGTTCGGATGCACTGATGCCAC<br>
CCAAGTGTTGGCTGAGGTGGGAGAGGCGAAGAAGGAATACCCACAGGCCTGGGTC<br>
CGTATCATTGGATTTGACAACGTGCGTCAAGTGCAGTGCATCAGTTTCATTGCCTCC<br>
AAGCCTGACGGCTACTGAGTTTCATATTAGGACAACTTACCCTATTGTCTGTCTTTA<br>
GGGGCAGTTTGTTTGAAATGTTACTTAGCTTCTTTTTTTTCCTTCCCATAAAAACTGT<br>
TTATGTTCCTTCTTTTTATTCGGTGTATGTTTTGGATTCCTACCAAGTTATGAGACCT<br>
AATAATTATGATTTTGTGCTTTGTTTGTAAAAAAAAAAAAAAAAA (SEQ ID NO: 124)<br>
[0304]      rbcS2 (Genbank accessions: CN748495: 3-552 b, CN748945: 364-575 b)<br>
TCTTTCTGTCTTAAGTGTAATTAACAATGGCTTCCTCAGTTCTTTCCTCAGCAGCAGT<br>
TGCCACCCGCAGCAATGTTGCTCAAGCTAACATGGTTGCACCTTTCACTGGTCTTAA<br>
GTCAGCTGCCTCGTTCCCTGTTTCAAGGAAGCAAAACCTTGACATCACTTCCATTGC<br>
CAGCAACGGCGGAAGAGTGCAATGCATGCAGGTGTGGCCACCAATTAACAAGAAG<br>
AAGTACGAGACTCTCTCATACCTTCCTGATCTGAGCGTGGAGCAATTGCTTAGCGAA<br>
ATTGAGTACCTCTTGAAAAATGGATGGGTTCCTTGCTTGGAATTCGAGACTGAGC<br>
GCGGATTTGTCTACCGTGAACACCACAAGTCACCGGGATACTATGACGGCAGATAC<br>
TGGACCATGTGGAAGTTGCCTATGTTCGGATGCACTGATGCCACCCAAGTGTTGGCC<br>
GAGGTGGAAGAGGCGAAGAAGGCATACCCACAGGCCTGGATCCGTATTATTGGATT<br>
CGACAACGTGCGTCAAGTGCAGTGCATCAGTTTCATTGCCTACAAGCCAGAAGGCT<br><br>
WO 2006/044322	PCT/US2005/036373<br>
74<br>
ACTAAGTTTCATATTAGGACAACTTACCCTATTGTCCGACTTTAGGGGCAATTTGTT<br>
TGAAATGTTACTTGGCTTCTTTTTTTTTTAATTTTCCCACAAAAACTGTTTATGTTTCC<br>
TACTTTCTATTCGGTGTATGTTTTTGCATTCCTACCAAGTTATGAGACCTAATAACTA<br>
TGATTTGGTGCTTTGTTTGTAAAT (SEQ ID NO: 125)<br>
[0305]      rbcS3   (Genbank   accessions:   CN746374:   22-108   b,   CN748757:   156-175   b,<br>
CN748929: 158-309 b, CN748913: 319-489 b, CN748777: 485-603 b,CN748188: 453-529 b)<br>
TAGCAATAGCTTTAAGCTTAGAAATTATTTTCAGAAATGGCTTCCTCAGTTATGTCC<br>
TCAGCAGCTGCTGTTGCGACCGGCGCCAATGCTGCTCAAGCCAACATGGTTGCACC<br>
CTTCACTGGCCTCAAGTCCGCCTCCTCCTTCCCTGTTACCAGGAAACAAAACCTTGA<br>
CATTACCTCCATTGCTAGCAATGGTGGAAGAGTTCAATGCATGCAGGTGTGGCCAC<br>
CAATTAACATGAAGAAGTACGAGACACTCTCATACCTTCCTGATTTGAGCCAGGAG<br>
CAATTGCTTAGTGAAGTTGAGTACCTTTTGAAAAATGGATGGGTTCCTTGCTTGGA<br>
ATTCGAGACTGAGCGTGGATTCGTCTACCGTGAACACCACAACTCACCAGGATACT<br>
ACGATGGCAGATACTGGACCATGTGGAAGTTGCCCATGTTCGGGTGCACTGATGCC<br>
ACTCAGGTGTTGGCTGAGGTCGAGGAGGCAAAGAAGGCTTACCCACAAGCCTGGGT<br>
TAGAATCATTGGATTCGACAACGTCCGTCAAGTGCAATGCATCAGTTTTATCGCCTC<br>
CAAGCCAGAAGGCTACTAAAATCTCCATTTTTAAGGCAACTTATCGTATGTGTTCCC<br>
CGGAGAAACTGTTTTGGTTTTCCTGCTTCCTTATATTATTCAATGTATGTTTTTGAAT<br>
TCCAA(SEQ IDNO:126)<br>
[0306]      rbcS4 (Genbank accessions CN748906: 9-607 b, CN747257: 629-709b)<br>
AATGGCTTCCTCAGTTATGTCCTCAGCTGCCGCTGTTGCCACCGGCGCCAATGCTGC<br>
TCAAGCCAGTATGGTTGCACCTTTCACTGGCCTCAAGTCCGCAACCTCCTTCCCTGT<br>
TTCCAGAAAACAAAACCTTGACATTACTTCCATTGCTAGCAACGGCGGAAGAGTTC<br>
AATGCATGCAGGTGTGGCCACCAATTAACAAGAAGAAGTACGAGACACTCTCATAC<br>
CTTCCCGATTTGAGCCAGGAGCAATTGCTTAGTGAAGTTGAGTACCTGTTGAAAAAT<br>
GGATGGGTTCCTTGCTTGGAATTCGAGACTGAGCGTGGATTCGTCTACCGTGAAC<br>
ACCACAGCTCACCAGGATATTATGATGGCAGATACTGGACCATGTGGAAGTTGCCC<br>
ATGTTCGGGTGCACTGATGCCACTCAGGTGTTGGCTGAGGTCGAGGAGGCAAAGAA<br>
GGCTTACCCACAAGCCTGGGTTAGAATCATTGGATTCGACAATGTCCGTCAAGTGC<br>
AATGCATCAGTTTCATCGCCTACAAGCCAGAAGGCTACTAGAATCTCCATTTTTAAG<br>
GCAACTTATCGTATGTGTTCCCCGGAGAAACTGTTTTGGTTTTTCCTGCTTCATTATA<br>
TTATTCAATGTATGTTTTTGAATTCCAATCAAGGTTATGAGAACTAATAATGACATT<br>
TAATTTGTTTCTTTTCTATATA (SEQ ID NO: 127)<br>
[0307]      rbcSS (Genbank accession: CN744712: 16-713 b)<br>
TAAATAATTAATTGCAACAATGGCTTCCTCTGTGATTTCCTCAGCTGCTGCCGTTGC<br>
CACCGGCGCTAATGCTGCTCAAGCCAGCATGGTTGCACCCTTCACTGGCCTCAAATC<br>
TGCTTCCTCCTTCCCTGTTACCAGAAAACAAAACCTTGACATTACATCCATTGCTAG<br>
CAATGGTGGAAGAGTCCAATGCATGCAGGTGTGGCCACCAATTAACATGAAGAAGT<br>
ACGAGACACTCTCATACCTTCCTGATTTGAGCCAGGAGCAATTGCTTAGTGAAGTTG<br>
AGTATCTTTTGAAAAATGGATGGGTTCCTTGCTTGGAATTCGAGACTGAGCGTGG<br>
ATTTGTCTACCGTGAACATCACAGCTCACCAGGATACTACGATGGCAGATACTGGA<br>
CCATGTGGAAGTTGCCCATGTTCGGGTGCACTGATGCCACTCAGGTGTTGGCTGAGG<br>
TCGAGGAGGCAAAGAAGGCTTACCCACAAGCCTGGGTTAGAATCATTGGATTCGAC<br>
AACGTCCGTCAAGTGCAATGCATCAGTTTTATCGCCTCCAAGCCAGAAGGCTACTA<br>
AAATCTCCATTTTTAAGGCAACTTATCGTATGTGTTCCCCGGAGAAACTGTTTTGGT<br><br>
WO 2006/044322	PCT/US2005/036373<br>
75<br>
TTTCCTGCTTCATTATATTATTCAATGTATGTTTTTGAATTCCAATCAAGGTTATGAG<br>
AACTAATAATGACATTTAA (SEQ ID NO: 128)<br>
[0308]      rbcS6 (Genbank accessions: CN745030: 14-123 b, CN748077: 1-523 b)<br>
GCACGAGGCTTCCTCAGTTATGTCCTCAGCTGCCGCTGTTTCCACCGGCGCCAATGC<br>
TGTTCAAGCCAGCATGGTCGCACCCTTCACTGGCCTCAAGGCCGCCTCCTCCTTCCC<br>
GGTTTCCAGGAAACAAAACCTTGACATTACTTCCATTGCTAGAAATGGTGGAAGAG<br>
TCCAATGCATGCAGGTGTGGCCGCCAATTAACAAGAAGAAGTACGAGACACTCTCA<br>
TACCTTCCTGATTTGAGCGTGGAGCAATTGCTTAGCGAAATTGAGTACCTTTTGAAA<br>
AATGGATGGGTTCCTTGCTTGGAATTCGAGACTGAGCATGGATTCGTCTACCGTG<br>
AACACCACCACTCACCAGGATACTACGATGGCAGATACTGGACGATGTGGAAGTTG<br>
CCCATGTTCGGGTGCACCGATGCCACTCAGGTCTTGGCTGAGGTAGAGGAGGCCAA<br>
GAAGGCTTACCCACAAGCCTGGGTCAGAATCATTGGATTCGACAACGTCCGTCAAG<br>
TGCAATGCATCAGTTTCATCGCCTACAAGCCCGAAGGCTATTAAAATCTCCATTTTT<br>
AGGACAGCTTACCCTATGTATTCAGGGGAAGTTTGTTTGAATTCTCCTGGAGAAA.CT<br>
GTTTTGGTTTTCCTTTGTTTTAATCTTCTTTCTATTATATTTTTGGATTTTACTCAAGT<br>
TTATAAGAACTAATAATAATCATTTGTTTCGTTACTAAAAAAAAAAAA     (SEQ       ID<br>
NO: 129)<br>
[0309]      Infiltration of N. benthamiana with Agrobacterium tumefaciens<br>
[0310] Infiltration with A. tumefaciens carrying appropriate plasmids was carried out as<br>
follows. Cells were grown to exponential phase in the presence of appropriate antibiotics and<br>
40 (JM acetosyringone. They were harvested by centrifugarion, resuspended in 10 mM MgCk<br>
containing 150(iM acetosyringone and incubated at room temperature for 2 hrs without<br>
agitation. Infiltration was performed by using a syringe without needle applied to the abaxial<br>
side of leaves. After 1, 2, or 3 days leaf tissue was collected, frozen and ground in li<quid></quid>
nitrogen before RNA extraction.<br>
[0311]      Northern blot hybridizations<br>
[0312] Leaves from Nicotiana benthamiana were used to extract total RNA using the Trizol<br>
reagent (Invitrogen). 10-20 p.g total RNA were resolved in a 15% polyacrylamide/ 1 x TBE (8.9<br>
mM Tris, 8.9 mM Boric Acid, 20 mM EDTA)/ 8 M urea gel and blotted to a Hybond-N+<br>
membrane (Amersham). DNA oligonucleotides with the exact reverse-complementary sequence<br>
to miRNAs were end-labeled with P-y-ATP and T4 polynucleotide kinase (New England<br>
Biolabs) to generate high specific activity probes. Hybridization was carried out using the<br>
ULTRAHyb-Oligo solution according to the manufacturer's directions (Ambion, TX), and<br>
signals were detected by autoradiography. In each case, the probe contained the exact antisense<br>
sequence of the expected miRNA to be detected.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
76<br>
[0313] Northern blot hybridizations to detect PDS tnRNA abundance were performed<br>
according to standard procedures. The 5' end probe corresponded to a fragment of N.<br>
benthamiana PDS gene reported before (Guo et al. (2003) Plant J 34:383-392) equivalent to the<br>
tomato PDS gene sequence positions 1-268 (Genbank X59948, see above). The 3'end probe<br>
corresponded to a fragment obtained by 5'RACE and equivalent to the tomato PDS gene<br>
sequence positions 1192-1514.<br>
[0314]      5' RACE<br>
[0315]      To identify the products of miRNA-directed cleavage the First Choice RLM-RACE<br>
Kit (Ambion) was used in 5' RACE experiments, except that total RNA (2/xg) was used for<br>
direct ligation to the RNA adapter without further processing of the RNA sample.  Subsequent<br>
steps were according to the manufacturer's directions.   Oligonucleotide sequences for nested<br>
PCR amplification of PDS cleavage fragment were:<br>
3'Nb-PDSl     5' CCACTCTTCTGCAGGTGCAAAAACC 3' (SEQ ID NO:130)<br>
3 'Nb-PDS2     5' ACATGGTACTTCAATATTTTTGCTTTGC 3' (SEQ ID NO: 131)<br>
3 'Nb-PDS3     5' GATCTTTGTAAAGGCCGACAGGGTTCAC 3' (SEQ ID NO: 132)<br>
All three primers were designed based on available sequence information for the tomato PDS<br>
gene since the complete N. bethamiana PDS gene sequence has not been published.<br>
[0316]      PCR fragments obtained from 5'RACE experiments were cloned in the pCR4 vector<br>
(Invitrogen) and analyzed by DNA sequencing of individual clones.<br>
[0317]      RT-PCR<br>
[0318]     First strand cDNA was synthesized form 5 mg total RNA using an oligo-dT primer<br>
(Sigma) and Ready-To-Go You-Prime First-strand beads (Amersham Biosciences). Amounts of<br>
first strand cDNA were normalized by PCR using primers for EFla (Nishihama et al. (2002)<br>
Cell 109:87-99). To amplify DNA fragments of rbcS cDNAs, the following primers were used.<br>
NBrbcs5:l/2-F: 5' TTCCTCAGTTCTTTCCTCAGCAGCAGTTG 3' (SEQ ID NO:133)<br>
rbcS3-F: 5' CTCAGTTATGTCCTCAGCAGCTGC 3' (SEQ ID NO: 134)<br>
rbcS4/6-F: 5' TCCTCAGTTATGTCCTCAGCTGCC 3' (SEQ ID NO:135)<br>
NBrbcS5-F: 5' TGTGATTTCCTCAGCTGCTGCC 3' (SEQ ID NO: 136)<br>
NBrbcsl rev2: 5' AACTCAGTAGCCGTCAGGCTTGG 3' (SEQ ID NO: 137)<br>
NBrbcs2 rev2: 5' AATATGAAACTTAGTAGCCTTCTGGCTTGT 3' (SEQ ID NO: 138)<br>
NBrbcs3/4/5 revl: 5' GTTTCTCCGGGGAACACATACGA 3' (SEQ ID NO: 139)<br><br>
WO 2006/044322	PCT/US2005/036373<br>
77<br>
NBrbcs6 revl: 5' AAACAAACTTCCCCTGAATACATAGGG 3' (SEQ ID NO: 140)<br>
EXAMPLE 17<br>
[0319] This example describes the design of an artificial microRNA to cleave the phytoene<br>
desaturase (PDS) mRNAs of Nicotiana benthamiana.<br>
[0320] Arabidopsis miRNAs identified so far have been shown to target different mRNAs,<br>
and a significant number encodes transcription factors (Bartel (2004) Cell 116:281-297; Wang et<br>
al. (2004) Genome Biol 5:R65; Rhoades et al. (2002) Cell 110:513-520). Base-pairing of plant<br>
miRNAs to their target mRNAs is almost perfect and results in cleavage of the RNA molecule<br>
as has been shown for several examples (Jones-Rhoades and Bartel (2004) Mol Cell 14:787-<br>
799), resulting in silencing of gene expression. Alternatively, miRNA interaction with the target<br>
mRNA can result in inhibition of translation rather than mRNA cleavage as shown for miR172<br>
of Arabidopsis (Aukerman and Sakai (2003) Plant Cell 15:2730-2741; Chen (2004) Science<br>
303:2022-2025).<br>
[0321] In an effort to design artificial miRNAs that can inhibit the expression of particular<br>
genes, we sought to modify the sequence of a known miRNA to target an mRNA of choice.<br>
[0322] The Arabidopsis miR159 has been shown to target a set of MYB transcription factors.<br>
Base-pairing of miR159 to its target mRNAs is almost perfect and results in cleavage of the<br>
RNA molecule (Achard et al. (2004) Development 131:3357-3365; Palatnik et al. (2003) Nature<br>
425:257-263). There are three genomic sequences (MR159a, MTR159b, MIR159c) with the<br>
potential to encode miR159. The natural promoter and precise precursor sequence of miR159<br>
are not known, nor is it known whether microRNA genes are transcribed by DNA polymerase II<br>
or III. We decided to use as precursor sequence a DNA fragment of 276 bp that contains the<br>
Arabidopsis miR.159a. This precursor sequence, which is called pre-miR159a was placed<br>
downstream of a 35S promoter and flanked at the 3' end by a polyA addition sequence of the<br>
nopaline synthase gene (Fig. 11 A). We decided to use the N. benthamiana phytoene desaturase<br>
(PDS) gene as a target to see whether we can design an artificial microRNA to cleave its mRNA<br>
and thereby compromise its expression. We compared the sequence of At-miR.\ 59a to that of<br>
PDS to find the best match between the two sequences. For one particular region of the PDS<br>
mRNA we found that only 6 base changes are sufficient to convert miR159a into a miRNA<br>
capable to perfectly base-pair to PDS mRNA (Fig. 1 IB). We called this sequence miR-PDS159".<br>
[0323] To generate pre-miR-PDSl59a, PCR techniques were used to introduce point mutations<br>
in both miR159a and the miR159a* sequence (the RNA sequence located in the opposite arm to<br><br>
WO 2006/044322	PCT/US2005/036373<br>
78<br>
the miRNA within the precursor sequence) in the context of the Arabidopsis pre-miR159a. The<br>
resulting precursor was placed under the control of the strong cauliflower mosaic virus (CaMV)<br>
35S promoter and expressed in N. benthamiana by infiltration of Agrobacterium tumefaciens<br>
containing the appropriate constructs.<br>
[0324] Expression of the Arabidopsis pre-miR-PDS159a in N. benthamiana was first analyzed<br>
to confirm that the mutations introduced in its sequence did not affect its processing and<br>
maturation of miR-PDS159a. Northern blot analysis showed that 2 to 3 days after infiltration<br>
miR-PDS159a is clearly expressed (Fig. 11C), accumulating to levels comparable to endogenous<br>
miR159. Biogenesis of known miRNAs includes the generation of the almost complementary<br>
miRNA* which is short-lived and accumulates to very low levels when compared to those of the<br>
actual miRNA. Consistently, the presence of miR-PDS159** was detected but its abundance was<br>
significantly lower than that of miR-PDS159* (Fig. 11C, middle panel). Expression of<br>
endogenous miR159 was unchanged under these conditions and served as both a loading and<br>
probe-specificity control (Fig. 11B, bottom panel). In addition, this result indicates that<br>
expression of an artificial miRNA based on the Arabidopsis miRNA precursor does not affect<br>
expression of the endogenous N. benthamiana miR159. Finally, these findings imply that the<br>
enzymatic machinery for processing of natural microRNA precursors is not rate limiting and can<br>
process artificial precursors with great efficiency.<br>
[0325] We next determined whether expression of miR-PDSJ59a resulted in the expected<br>
cleavage of the endogenous PDSmRNA. Northern blot hybridization of the samples expressing<br>
miR-PDS159a showed a clear reduction in PDS mRNA levels (Fig. 12A). To further establish the<br>
mechanism of PDS mRNA reduction we set to define: (1) whether the PDS mRNA is cleaved by<br>
miR-PDS159a and contains a diagnostic 5' phosphate, and (2) whether the cleavage point<br>
corresponds to the predicted site, based on the PDS mRNA:miR-PD5159a base-pairing<br>
interaction. To this end, 5'RACE experiments were performed. We found that the 5'-end<br>
sequence of 5 out of 6 independent clones mapped the site of cleavage after the tenth nucleotide<br>
counting from the 5' end of miR-PDSl59a. The location of the cleavage site correlates perfectly<br>
with published work with other miRNA targets (Jones-Rhoades and Bartel (2004) Mol Cell<br>
14:787-799).<br>
[0326] The results demonstrate that the reduction of PDS mRNA levels was caused by<br>
accurate cleavage directed by miR-PDS'59a.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
79<br>
EXAMPLE 18<br>
[0327]      This example demonstrates that microRNA-directed cleavage of PDS mRNA can be<br>
produced from a different microRNA precursor.<br>
[0328] To show that expression of artificial miRNAs is not restricted to the use of pre-<br>
miR159a we have designed a different miR-PDS based on a putative precursor sequence<br>
containing the Arabidopsis miR169g to generate two different miR.-PDS169g (Fig. 13 A). During<br>
the design of the expression vector for miR169g, we noticed that a construct containing only the<br>
stem-loop precursor of 222 bp resulted in higher accumulation of the mature miRNA than a<br>
construct containing the entire 2.0 kb intergenic region including the miR169g gene (Fig. 13B).<br>
Based on this result we decided to continue our mutagenesis of miRNA sequences using<br>
exclusively short precursor vectors. Examination of the PDS mRNA with the miR169g<br>
sequence revealed a region in the mRNA sequence susceptible for miRNA cleavage, different<br>
from that found for miR-PDS159a. Seven point mutations turn miR169g into a microRNA<br>
capable of base-pairing perfectly to the PDS mRNA (miR-PDSa169g, Fig. 13A). As shown<br>
before for the miRl 59a-based miR-PDS, transient expression of miR-PDSa1698 in N.<br>
benthamiana is easily detected (Fig. 13C). In addition, to test whether the entire miRNA can be<br>
changed independently of its original sequence, we have generated miR-PDSb169g (Fig. 13A and<br>
Fig. 13C), which targets a different region in the PDS mRNA selected irrespective of its<br>
homology to the original miR169g. Using both rmR-PDSa169g and miR-PDSb1698 we could<br>
detect cleavage of the PDS mRNA, as determined by 5'RACE analysis (Fig. 13D) and a<br>
reduction in PDS mRNA levels as determined by Northern blot analysis (Fig. 13E).<br>
[0329] These results show that a different miRNA precursor can be used to target<br>
degradation of PDS mRNA and importantly, that the sequence of the original miRNA can be<br>
extensively changed to design an artificial one.<br>
EXAMPLE 19<br>
[0330] This example demonstrates microRNA-directed specific cleavage of Nicotiana<br>
benthamiana rbcS mRNAs.<br>
[0331]- To show that this approach can be used to target other genes different from PDS, we<br>
have introduced point mutations in miRl 59a to target the different members of the Rubisco<br>
small subunit (rbcS) gene family of N. benthamiana. We searched for rbcS EST transcripts<br>
present in publicly available databases and found that at least 6 different rbcS transcripts are<br>
expressed in N.  benthamiana. Nucleotide sequences of the coding region of these rbcS<br><br>
WO 2006/044322	PCT/US2005/036373<br>
80<br>
transcripts were over 90% identical to each other, and allowed us to design miR.-rbsSl59a-A,<br>
which targets all members of the gene family. Here, the sequence introduced in miR159a was<br>
not guided by the minimal number of changes that would target rbsS but reflected the need to<br>
target a specific region common to all rbsS mRNAs and thus included several changes. In this<br>
way, we have generated one miRNA that targets all six rbcS mRNAs (miR.-rbcSl59a-A, Fig.<br>
14A).<br>
[0332] As in the previous examples, we have detected efficient expression of the miR-<br>
rbsS159a-A (Fig. 14B), but due to the high degree of homology among the members of this<br>
family, distinct rbsS mRNAs have been difficult to detect by Northern blot analysis. Instead, we<br>
have used semi-quantitative RT-PCR to determine the levels of mRNAs in plants infiltrated with<br>
Agrobacterium strains containing the miR-rbcSl59a-A construct Compared to leaves infiltrated<br>
with the empty binary vector (C in Fig. 14C), mRNA accumulation for rbcS genes 1, 2 and 3<br>
was reduced while for rbcS genes 4, 5 and 6 it could not be detected in samples infiltrated with a<br>
miR-rbcSl59a-A construct (A in Fig. 14C). These results indicate that the artificial miRNA<br>
targeted all the rbcS mRNAs it was intended for, although the efficiency in each case varied.<br>
Finally, the presence of the artificial miRNA did not interfere with expression of other plant<br>
genes such as EFla (Fig. 14C, bottom panel).<br>
[0333] The artificial miRNAs presented here are distributed along three different locations in<br>
PDS mRNA (summarized in Fig. 15A), and have been used to target 2 different genes (PDS and<br>
rbcS, Fig. 15A and Fig. 15B). This range of use is also reflected in the flexibility of the miRNA<br>
sequences, as the artificial miRNAs show that almost every nucleotide position can be changed<br>
(Fig. 16). Changes in miR159a to create two artificial miRNAs retained only 8 positions<br>
unchanged (Fig. 16A). In the case of miR169g this number was reduced to only three positions<br>
(Fig. 16B). Moreover, when the mutations in both miRNAs are analyzed together, only the first<br>
two nucleotide positions remain untouched. This suggests that every position along the miRNA<br>
sequence can be changed, adding to the advantages of using artificial miRNAs for gene<br>
silencing.<br>
EXAMPLE 20<br>
[0334] This example demonstrates that artificial miRNACPC159" inhibits root hair<br>
development in Arabidopsis.<br>
[0335] Root epidermal cells differentiate root-hair cells and hairless cells. Only root-hair<br>
epidermal cells are able to develop into root hair. In Arabidopsis roots, among a total of 16-22<br><br>
WO 2006/044322	PCT/US2005/036373<br>
81<br>
cell files, 8 symmetrically positioned cell files are root-hair cells and all others are hairless cell<br>
files. CAPRICE (CPC), a MYB like protein, positively regulates root hair development by<br>
negatively regulating GLABRA2 (GL2), which promotes root epidermal cells differentiation into<br>
hairless cells. In cpc mutant, GL2 causes most epidermal cells to differentiate into root hairless<br>
cells, and consequently, very few cells are able develop root hair. Roots of the gl2 mutant or<br>
wild type transgenic plants over-expressing CPC, produce more root hairs compared to wild<br>
type roots (Fig. 17; Wada et al. (2002) Development 129:5409-5419).<br>
[0336] CPC is a good candidate for investigations on the utility of artificial miRNAs to<br>
silence or suppress gene function because the loss-of-function phenotype of CPC appears at a<br>
very early stage during seedling development, does not cause lethality and is easy to observe.<br>
Using pre-miRNA159 as a backbone two artificial pre-miRNAs, pre-miRCPCl159a and pre-<br>
miRCPC3159a were designed to target different regions of the CPC mRNA. Mature<br>
miRCPCl159a and miRCPC3I59a are complementary to the sequences located in nt 233-253 and<br>
nt 310-330, respectively, of the CPC messenger RNA. The nucleotide sequences for the<br>
precursor and mature miRNAs are as follows.<br>
[0337]      miRCPCl159a precursor template:<br>
5'acagtttgctiatgtcggatccataatatatttgacaagatactttgtttttcgatagatcttgatctgacgatggaagaagaggtgagtaatgt<br>
tgaaacatgagttgagcagggtaaagaaaagctgctaagctatggatcccataagccctaatccttgtaaagtaaaaaaggatttggttatat<br>
ggattgcatatctcaggagctttaacttgcccrttaatggcttttactcttctttcgatactactcacctcttcatcccgggtca 3' (SEQ ID<br>
NO:151).<br>
[0338]      miRCPCl'59a mature template: 5' tttcgatactactcacctctt 3' (SEQ ID NO: 152).<br>
[0339]      miR CPC3IS9a precursor template:<br>
5'acagtttgcttatgtcggatccataatatatttgacaagatacrrtgtttttcgatagatcttgatctgacgatggaagctcgttggcgacaggt<br>
gggagcatgagttgagcagggtaaagaaaagctgctaagctatggatcccataagccctaatccttgtaaagtaaaaaaggatttggttatat<br>
ggattgcaiatctcaggagctttaacttgccctttaatggcttttactcttcctcccacctgacgccaacgagcatcccgggtca 3' (SEQ<br>
ID NO: 153).<br>
[0340] miRCPC3159a mature template: 5' ctcccacctgacgccaacgag 3' (SEQ ID NO: 154).<br>
[0341] These two artificial pre-miRNAs were cloned into a vector which contains a<br>
constitutive 35S promoter for expression of these precursors. Northern blot analysis of<br>
Nicotinana benthamiana leaves infiltrated by Agrobacteria carrying 35S::per-miRCPCl159a or<br>
35S:: pre-miRCPC3l59a constructs indicated successful production of mature miRCPCl159a and<br>
miRCPC3159a.<br>
[0342] Arabidopsis thaliana plants were transformed by Agrobacteria carrying XVE::pre-<br>
miRCPCl159" or 35S::pre-miRCPClI59a, and many transgenic lines were obtained. Ti seeds of<br>
XVE::pre-miRCPCl'59a plants were geminated on antibiotic selection medium containing<br><br>
WO 2006/044322	PCT/US2005/036373<br>
82<br>
kanamycin and resistant trasngenic seedlings were transferred to MS medium with or without p-<br>
estradiol, an inducer of the XVE system. T1 transgenic lines carrying XVE::pre-miR159 were<br>
used as a control. Pre-miR159 is the backbone used to construct the artificialpre-miRCPCl159a.<br>
[0343] No difference in root hair development between XVE::pre-miR159 seedlings grown<br>
on medium with or without inducer (Figure 18, panels c and d) was seen. By contrast,<br>
XVE::pre-miRCPCll59a seedlings grown on medium wi"th (3-estradiol clearly developed fewer<br>
root hairs (Figure 18, panel b) than those grown without inducer (Figure 18, panel a).<br>
[0344] T1 seedlings of transgenic Arabidopsis seedlings carrying 35S::pre-miRCPCl159a<br>
35S::pre-miR159 and 35S::pre-miRP69159a were investigated and similar results were obtained<br>
as the XVE inducible lines. T| seeds of transgenic lines were geminated on a BASTA-selective<br>
medium and two-week old seedlings were transferred to MS medium plates placed vertically in<br>
a tissue culture room. In this experiment, two negative controls were used: transgenic lines<br>
carrying 35S::pre-miR159 and those carrying 35S::pre-miRP69J59a. The latter was designed<br>
using pre-miR159 as a backbone to produce an artificial pre-miRP69159a targeting nt 214-234 of<br>
the P69 mENA of turnip yellow mosaic virus (TYMV; Bozarth et al. (1992) Virology 187:124-<br>
130). The nucleotide sequences for the precursor and mature miRNAs are as follows.<br>
[0345]      miRP691S9a precursor template:<br>
5'acagtttgcttatgtcggatccataatatatttgacaagatactttgtttttcgatagatcttgatctgacgatggaagccacaagacaatcga<br>
gactttcatgagttgagcagggtaaagaaaagctgctaagctatggatcccataa.gccctaatccttgtaaagtaaaaaaggatttggttatat<br>
ggartgcatatctcaggagctttaacttgccctttaatggcttttactcttcaaagtctcgattgtcttgtggcatcccgggtca 3' (SEQ ID<br>
NO: 155)<br>
[0346]      miRP69l59a mature template: 5' aaagtctcgattgtcttgtgg 3' (SEQ ID NO: 156).<br>
[0347]      Seedlings of both types of transgenic plants developed abundant root hair as wild type<br>
plants (Figure 19, panels a and c).  By contrast, among  30 independent 35S::pre-miRCPCll59a<br>
lines, 18 lines showed clearly fewer root hair (Figure 19, panel b) compared to negative control<br>
plants (Figure 19 panels a and c).<br>
[0348] In negative control transgenic plants {35S::pre-miRl 59 and pre-miRP69159a\ all root-<br>
hair file cells in the epidermis of the root tip region were able to develop root hairs (Figure 20,<br>
panel a; see arrows). However, in transgenic lines carrying 35S::pre-miRCPCl159a many cells in<br>
root-hair files were unable to produce root hairs (Figure 20, panel b; see arrows). These results<br>
indicate that the artificial miRCPCl159" is able to indiice cleavage of the endogenous CPC<br>
mJRNA to cause a loss function of the CPC gene function, and inhibit root hair development.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
83<br>
EXAMPLE 21<br>
[0349] This example describes one embodiment of a process for the designing a polymeric<br>
pre-miRNA.<br>
[0350] Step 1: Different pre-miRNAs are amplified by PCR to include an Avrll site in the 5'<br>
end and to include an Spel site and an Xhol site in the 3' end Each pre-miRNA is then cloned<br>
into a vector, such as pENTR/SD/D-TOPO (Invitrogen) to produce, for example, pENTR/pre-<br>
miRA, pENTR/pre-miRB and pENTR/pre-miRC (Fig. 21A).<br>
[0351] Step 2: The pENTR/pre-rm'RA is digested with the restriction enzyirxes Spel and<br>
Xhol. The restriction enzymes Avrll and Xhol are used to digest the pENTR/pre-miRB vector<br>
(Fig. 21B). Opened vector pENTR/pre-miRA and DNA fragment of pre-miRB are collected and<br>
purified for further steps.<br>
[0352] Step 3: The opened vector pENTR/pre-miRA and DNA fragment of pre-miRB from<br>
step 2 are ligated to generate dimeric pre-miRA-B (Fig. 21C). Because of compatible cohesive<br>
ends of Avrll and Spel, the pre-miRB fragment can be inserted into the opened pENTR/pre-<br>
miRA and both Avrll and Spel sites will disappear after ligation (Fig. 21C).<br>
[0353] Step 4: The pENTR/pre-miRA-B is digested by with the restriction enzymes Spel and<br>
Xhol, and pENTR/pre-miRC is digested with the restriction enzymes Avrll and Xhol (Fig.<br>
21D). Opened vector pENTR/pre-miRA-B and DNA fragment of pre-miRC are collected and<br>
purified for further steps.<br>
[0354] Step 5: The opened vector pENTR/pre-miRA-B and DNA fragment of pre-miRC<br>
from step 4 are ligated to generated triple pre-miRNA-B-C (Fig. 21E).<br>
[0355] In this manner, or using functionally equivalent restriction enzymes polymeric pre-<br>
miRNAs containing more pre-miRNA units can be prepared. As many pre-miKN/\s as desired<br>
can be linked together in this fashion, with the only limitation being the ultimate size of the<br>
transcript. It is well known that transcripts of 8-10 kb can be produced in plants. Thus, it is<br>
possible to form a multimeric pre-miRNA molecule containing from 2-30 or more, for example<br>
from 3-40 or more, for example from 3-45 and more, and for further example, multimers of 3, 4,<br>
5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31,<br>
32, 33, 34, 35, 36, 37, 38, 39, 40,41,42, 43,44,45,46,47,48,49, or more pre-miRNAs.<br>
EXAMPLE 22<br>
[0356] This example demonstrates the successful processing of a dimeric pre-iRNA to two<br>
mature miRNAs.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
[0357] Artificial pre-miRPDSll69g and pre-miRCPC3159a were linked to form dimeric<br>
precursor, pre-miRPDS169g-CPC3159a as described in Example 21. This dimeric miRNA<br>
precursor was cloned into a vector in which 35S promoter drives expression of the pre-<br>
miRPDSl g-CPC3159a (Fig. 22). The nucleotide sequences for the precursor and mature<br>
miRNAs are as follows.<br>
[0358]      miRPDSl169g precursor template:<br>
5'aatgatgattacgatgatgagagtctctagttgtatcagagggtcttgcatggaagaatagagaatgaggttgagtttagtctgacttggcc<br>
agtttttttaccaatgaatctaattaactgattctggtgttggccaagtcagactaaactctgtttccttctcttcttttggatgtcagactccaagata<br>
tctatcatcatgaatcgtgatcaaactttg 3' (SEQ ID NO: 157).<br>
[0359]      rniRPDSl1698 mature template: 5' gagtttagtctgacttggcca 3' (SEQ ID NO: 158).<br>
[0360]      miRPDSl169g-CPC3I59a precursor template:<br>
5'cacctaggaatgatgattacgatgatgagagtctctagttgtatcagagggtcttgcatggaagaatagagaatgaggttgagtttagtctg<br>
acttggccagtttttttaccaatgaatctaattaactgattctggtgttggccaagtcagactaaactctgtttccttctcttcttttggatgtcagact<br>
ccaagatatctatcatcatgaatcgtgatcaaactttgaagggtgggcgactaggacagtttgcttatgtcggatccataatatatttgacaaga<br>
tactttgtttttcgatagatcttgatctgacgatggaagctcgttggcgacaggtgggagcatgagttgagcagggtaaagaaaagctgctaa<br>
gctatggatcccataagccctaatccttgtaaagtaaaaaaggatttggttatatggattgcatatctcaggagctttaacttgccctttaatggct<br>
tttactcttcctcccacctgacgccaacgagcatcccgggtcaaagggtgggcgactagtctagactcgagtart 3' (SEQ ID<br>
NO: 159).<br>
[0361] Northern blotting analysis of tobacco Nicotiana benthamiana levies, infiltrated by<br>
Agrobacteria carrying different constructs of 35S::pre-miRPDSl'69g, 35S::pre-miRCPC3I59a and<br>
35S::pre-miRPDSJ69g-CPC3I59a, indicates that mature miRPDS1698 and CPC3lS9a were<br>
successfully produced from the dimeric miRNA precursor (Figs. 23A and 23B). In this<br>
experiment, treatment 1 is 35S::pre-miRPDSl'69s, treatment 2 is 35S::miRCPC3J59a and<br>
treatment 3 is 35S:: pre-miRPDS169g-CPC3159a. When miRPDSlI69g anti sense DNA oligo as<br>
probe, both land 3 treatments showed signals that proved the dimeric precursor was able to<br>
produce matured miRPDSlJ69g. When the probe is miRCPC3I59a anti sense DNA oligo, signal in<br>
treatment 3 confirmed the ability of pre-miRPDS'69s-CPC3159a to generate mature miRCPC3159a.<br>
EXAMPLE 23<br>
Design of Anti-viral miRNAs<br>
[0362] Since viral gene silencing suppressors are used to counteract host defense, we<br>
reasoned that compromising the production of these suppressors by the expression of specific<br>
miRNAs would be an effective mechanism to confer resistance or tolerance to plant viruses<br>
(Roth et al. (2004) Virus Res 102:97-108). This principle is demonstrated by using TuMV as an<br>
example.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
85<br>
[0363] HC-Pro and P69 are plant PTGS suppressors encoded by TuMV and TYMV,<br>
respectively (Anandalakshmi et al. (1998) Proc Nad Acad Sci USA 95:13079-13084; Chen et al.<br>
(2004) Plant Cell 16:1302-1313; Kasschau and Carrington (1998) Cell 95:461-470). Using<br>
these two viral suppressor genes as targets, artificial miRNAs were designed with sequence<br>
complementarity to their coding sequences.<br>
[0364] At-miR159a is strongly expressed in most Arabidopsis organs and at high levels.<br>
Similar high level expression was also found in other plants species such as corn and tobacco.<br>
For these reasons, the miR159a precursor (pre-miR159a) was used as a backbone to generate<br>
artificial miRNAs. Pre-miR159a, a 184nt stem-loop RNA, produces mature miR159a (5'-<br>
uuuggauugaagggagcucua-3'; SEQ ID NO: 160) from the base of its stem near the 3'end. This<br>
base stem sequence is the miR159a sequence and the complementary strand is called miR159a*<br>
sequence (Fig. 24; SEQ I DNO:161). To design artificial miRNA, the miR159a sequence was<br>
replaced by a sequence 5'-acuugcucacgcacucgacug-3' (SEQ ID NO:162), which is<br>
complementary to the viral sequence encoding HC-P from 2045 to 2065 of the TuMV genome<br>
sequence. The miR159a* sequence was also altered to maintain the stem structure. For more<br>
efficient miRNA processing and convenient manipulation of the artificial miRNA precursor, a<br>
78bp sequence cloned from the genome sequence upstream of pre-miR159 was added to the<br>
5'end of this artificial miRNA precursor. This primary miRNA-like artificial miRNA precursor<br>
was called pre-miR//C-f&gt;/59a. Its DNA sequence follows.<br>
[0365]      Pre-rniRHC-P!59a<br>
5' CAGTTTGCTTATGTCGGATCCATAATATATTTGACAAGATACTTTGTTTTTCGATA<br>
GATCTTGATCTGACGATGGAAGCAGTCGAGTGCGTGAGCAAGTCATGAGTTGAGCA<br>
GGGTAAAGAAAAGCTGCTAAGCTATGGATCCCATAAGCCCTAATCCTTGTAAAGTA<br>
AAAAAGGATTTGGTTATATGGATTGCATATCTCAGGAGCTTTAACTTGCCCTTTAAT<br>
GGCTTTTACTCTTCACTTGCTCACGCACTCGACTGC 3' (SEQ ID NO: 163)<br>
[0366]     Using the same method, pre-miRP69159a was also constructed. Pre- miRP69159a was<br>
predicted to generate mature artificial miRNA P69159a, 5'-aaagucucgauugucuugugg-3' (SEQ ID<br>
NO: 164), to target the P69 gene of TYMV. Its DNA sequence follows.<br>
[0367]      Pre-miR P69l59a<br>
5'CAGTTTGCTrATGTCGGATCCATAATATATTTGACAAGATACTTTGTTTTTCGATA<br>
GATCTTGATCTGACGATGGAAGCCACAAGACAATCGAGACTTTCATGAGTTGAGCA<br>
GGGTAAAGAAAAGCTGCTAAGCTATGGATCCCATAAGCCCTAATCCTTGTAAAGTA<br>
AAAAAGGATTrGGTrATATGGATTGCATATCTCAGGAGCTTTAACTTGCCCTTTAAT<br>
GGCTTTTACTCTTCAAAGTCTCGATTGTCTTGTGGC 3' (SEQ ID NO:165)<br><br>
WO 2006/044322	PCT/US2005/036373<br>
86<br>
EXAMPLE 24<br>
Expression of pre-miRHC-P159a and pre-miRP69IS9a va.Nicotia.na benthamiana<br>
[0368] Replacement of the miR159 and miR159* sequences in the pre-miR159 may possibly<br>
effect RNA folding structure which is believed to be important for miRNA biosynthesis. A<br>
tobacco transient expression system was used to check whether these two artificial miRNA<br>
precursors can produce the desired miRNAs. Agrobacterial cells containing plasmids with<br>
35S::pre-miR-HC-Pro!59a, 35::pre-miR-P69159a, 35S::HC-Pro, 35S::P69, and XVE::pre-miR-<br>
P69l59a were used to infiltrate N. benthamiana leaves (Llave et al. (2000) Proc Natl Acad Sci<br>
USA 97:13401-13406; Voinnet etal. (2000) Cell 103:157-167).<br>
[0369] One ml of stationary phase growth culture of Agrobacteria tumefaciens carrying<br>
different constructs were cultured overnight in 50 ml LB medium containing 100 mg/1<br>
spectinomycin and 50 mg/1 kanamycin and cells were collected by centrifugation at 4,000 rpm<br>
for 10 minutes. Bacterial pellets were re-suspended in 50 ml 10 mM MgCIj solution with 75 p.1<br>
of 100 mM acetosyringone. After incubation at room temperature for 3 hr without shaking, the<br>
Agrobacterial suspensions were infiltrated into leaves of N. bethamiana by a syringe. Two days<br>
later, total RNA was extracted from the infiltrated leaves using the trizol reagent (Invitogen) and<br>
analyzed by northern blot hybridizations (Guo et al. (2005) Plant Cell 17:1376-1386; Wang et<br>
al. (2004) Genome Biol 5:R65). Samples of 20 ug total RNA were analyzed by electrophoresis<br>
on a 15% polyacrylamide gel and blotted to a Hybond-N+ membrane (Amersham). IXNA<br>
oligonucleotides with exact complementary sequence to miR-HC-Pro159a or pre-miR-P6P;59fl<br>
were end-labeled with [y-32P]-ATP and T4 polynucleotide kinase to generate high specific<br>
activity probe. Hybridization was carried out using the ULTRA-Hyb Oligo solution according<br>
to the manufacturer's directions (Ambton) and signals were detected by autoradiography.<br>
[0370] Northern blot analyses of miR-HC-Pro159a were performed with three diffexent<br>
treatments: (1) Agrobacterial cells with 35S::pre-miR-HC-Pro159a, (2) Agrobacterial cells with<br>
35S::HC-Pro, and (3) Agrobacterial cells with 35S::pre-miR-HC-Pro159a and 35S::HC-pro.<br>
The results are shown in Fig. 25.<br>
[0371] Note that mature miR-HC-Pro159a signals were detected in all treatments with<br>
35S::pre-miR-HC-Pro159a (column 1, 2, 5, 6 of Fig. 25). No signal was detected when leaves<br>
were infiltrated with the 35S::HC-Pro construct only (column 3 and 4 of Fig. 25). This result<br>
indicates that the artificial pre-miR-HC-Pro159a can generate mature miR-HC-Pro159a in the plant<br>
cell.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
87<br>
[0372] In the case of miR-P69, 4 different treatments were performed : (1) Agrobaeterial<br>
cells carrying 35S:: pre-miR-P69159a, (2) Agrobaeterial cells carrying XVE::pre-miR-P69l59a, (3)<br>
Agrobaeterial cells carrying 35S::P69, and (4) Agrobaeterial cells carrying 35S::pre-miR-<br>
P69I59a and 35S::P-69. Note that the XVE system is a transcriptional inducible system<br>
responsive to p-estradiol (Zuo et al. (2001) Nature Biotechnol 19(2): 157-61).<br>
[0373] The Northern blot results (Fig. 26) showed that mature miR-P69l59a was detectable<br>
only in leaves infiltrated with 35S::-pre-miR.-P69I59a and XVE::pre-miR-P69!59a plus inducer<br>
(column 1, 2, 4, 6, 8, and 9 of Fig. 26). Leaves infiltrated with 35S::P69 and XVEr.pre-<br>
miRP69159a without inducer can not produce miR-P69!59a (column 3, 5, and 7 of Fig. 26).<br>
Together, these results indicate that artificial pre-miR-P69159a can be successfully used to<br>
generate mature miR-P69159a.<br>
EXAMPLE 25<br>
Stable Arabidopsis Transgenic Lines<br>
with High Artificial miRNAs Expression Levels<br>
[0374] Constructs containing 35S::pre-miR-HC-Pro159a or 35S::pre-miR-P69159a was<br>
transformed into Arabidopsis Col-0 ecotype mediated by Agrobacteria using the floral dip<br>
method (Clough and Bent (1998) Plant J 16:735-43).<br>
[0375] Transgenic seedlings were selected on selection medium (MS salts 4.3 g/1 + Sucrose<br>
10 g/1 + Basta 10 mg/1 + Carbenicilin 200 mg/1 + Agar 8 g/1). Twelve different 35S::pre-miR-<br>
HC-Pro'59a or 35S::pre-miR-P69l59a T2 transgenic lines were randomly picked and used to<br>
analyze mature artificial miRNA levels by northern blots. Among 12 transgenic 35S::pre-<br>
miRHC-Pro159a lines, 11 lines showed high levels of expression of miRHC-Pro159a (Fig. 27). In<br>
Arabidopsis transgenic 35S::pre-miR-P69159a plants, all T2 lines tested showed miR-P69159a<br>
signals and 10 lines showed high expression levels (Fig. 28).<br>
EXAMPLE 26<br>
TuMV Virus Challenge of WT.and Transgenic Plants<br>
[0376]      Inoculation of WT and transgenic Arabidopsis lines with the TuMV<br>
[0377]     N. benthamiana leaves were inoculated with Turnip mosaic virus (TuMV) (Chen et<br>
al. (2003) Plant Dis 87:901-905) and two weeks later tissues were extracted in 1:20 (wt/vol)<br>
dilution in 0.05 M potassium phosphate buffer (pH 7.0).   This extract was used as a viral<br>
inoculum. T2 plants of 35S::miR-HC-Pro159a transgenic Arabidopsis lines were grown in a<br><br>
WO 2006/044322	PCT/US2005/036373<br>
greenhouse for 4 weeks (5 to 6 leaves stage) before inoculation. Plants were dusted with 600-<br>
mesh Carborundum on the first to fourth leaf and gently rubbed with 200 JJ.1 inoculum. Wild<br>
type Arabidopsis thaliana (col-0) plants and transgenic plants expressing 35S::miR-P69159a were<br>
used as controls. Inoculated plants were kept in a temperature-controlled greenhouse (23° C to<br>
28° C) and symptom development was monitored daily for 2 weeks.<br>
[0378]      Enzyme-linked immunosorbent assay (ELISA)<br>
[0379] Leaf disks (a total of 0.01 g) from different systemic leaves of each plant infected<br>
with TuMV were taken 14 dpi (days post infection), and assayed by indirect enzyme-linked<br>
immunosorbent assay (ELISA) using a polyclonal antiserum to TuMV coat protein (CP) (Chen<br>
et al. (2003) Plant Dis 87:901-905) and goat anti-rabbit immunoglobulin G conjugated with<br>
alkaline phosphatase. The substrate p-nitrophenyl phosphate was used for color development.<br>
Results were recorded by measuring absorbance at 405 nm using Tunable Microplate Reader<br>
(VersaM&amp;x, Molecular Devices Co., CA).<br>
[0380]      Western blot analysis<br>
[0381] Western blot analysis was conducted using the rabbit antiserum to TuMV CP (Chen et<br>
al. (2003) Plant Dis 87:901-905) and goat anti-rabbit immunoglobulin G conjugated with<br>
alkaline phosphatase. Systemic leaves from Arabidopsis plants were homogenized in 20<br>
volumes (wt/vol) of denaturation buffer (50 mM Tris-HCl, pH 6.8, 4% SDS, 2% 2-<br>
mercaptoethanol, 10% glycerol, and 0.001% bromophenol blue). Extracts were heated at 100° C<br>
for 5 min and centrifuged at 8,000xg for 3 mm to pellet plant debris. Total protein of each<br>
sample (15 ul) was loaded on 12% polyacryamide gels, separated by SDS-polyacrylamide gel<br>
electrophoresis, and subsequently transferred onto PVDF membrane (immobilon-P, Millipore,<br>
Bedford, MA) using an electro transfer apparatus (BioRad). The membranes were incubated<br>
with polyclonal rabbit antiserum to TuMV CP as primary antibodies and peroxidase-conjugated<br>
secondary antibodies (Amersham Biosciences) before visualization of imrnunoreactive proteins<br>
using ECL kits (Amersham Biosciences). Gels were stained with coomassie-blue R250 and<br>
levels of the large subunit of RUBISCO (55 kd) were used as loading controls.<br>
[0382] It was found that transgenic plants expressing miR-HC-Prol59a artificial miRNA are<br>
resistant to TuMV infection (Fig. 29). Photographs were taken 2 weeks (14 days after infection)<br>
after inoculation. Plants expressing miR-HC-Pro159a (line #11; Fig 33B) developed normal<br>
influorescences whereas WT plants and transgenic plants expressing miR-P69159a (line #1; Fig<br>
33B) showed viral infection symptoms.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
89<br>
[0383] Forteen days after TuMV infection, miR-P69159a (line #1) and col-0 plants showed<br>
shorter intemodes between flowers in influoresences, whereas miR-HC-Pro159a transgenic plant<br>
(line #11) displayed normal influoresences development (Fig. 30, upper panel). Close-up views<br>
of influoresences on TuMV-infected Arabidopsis plants. miR-P69159a (line #1) and col-0 plants<br>
showed senescence and pollination defects whereas miR-HC-ProI59a plants (line #11) showed<br>
normal flower and silique development (Fig. 30, bottom panel). For mock-infection, plants were<br>
inoculated with buffer only.<br>
[0384] In TuMV-infected miR-P69159a (line #1) and WT (col-0) plants, siliques were small<br>
and mal-developed. miR-HC-Pro159a plants (line #11) were resistant to TuMV infection and<br>
showed normal silique development (Fig. 31). Buffer-inoculated plants (mock-inculated) were<br>
used as controls.<br>
[0385] Two independent experiments were performed to examine the resistance of various<br>
transgenic miR-HC-Pro159a and WT plants to TuMV infection. Experiment 1: Sixteen individual<br>
plants of a T2 transgenic line (line # 11 of miR-HC-Pro159a plant and line #1 of miR-P69I59a<br>
plant) were used. Twelve individual plants were inoculated with virus whereas 4 individual<br>
plants were inoculated with buffer as control (MOCK). After 2 weeks system leaves were<br>
collected for western blot analyses using an antibody against TuMV CP. No TuMV CP was<br>
detected in miR-HC-Prol59a transgenic plants, whereas, TuMV CP was highly expressed in miR-<br>
P69159a and WT col-0 plants (Fig. 32). The large subunit (55kd) of RUBISCO was used as a<br>
loading control. Note that no CP was detected in lane 6 (top panel) and lane 4 (middle panel)<br>
likely due to failed virus inoculation. These plants had no symptoms. The results of the<br>
infectivity assay are shown in Table 7.<br><br>
TABLE 7Infectivity Assay of Transgenic Arabidopsis of themiR-HC-Prol59a and miR-P69159a Challenged with TuMV Inocula<br>
	Number of seedlings	<br>
Transgenic line	Resistant     Susceptible     Total	Resistant rate (%)<br>
miR-HC-Pro159a #11miR-P69I59a#lcol-0	12                 0               121                  11               121                   11               12	100■      8.38.3<br>
[0386]      Experiment 2: The following transgenic lines and WT plants were used. (1) 35S::miR-<br>
HC-Prol59a plants: line #10 (12 plants inoculated with TuMV and 4 with buffer;); line #11 (12<br><br>
WO 2006/044322	PCT/US2005/036373<br>
90<br>
plants inoculated with TuMV and 4 with buffer); line #12 (9 plants inoculated with TuMV and 4<br>
with buffer); line #13 (10 plants inoculated with TuMV and 4 with buffer). (2) 35S::miR-P69l59a<br>
plants: line #1 (8 plants inoculated with TuMV and 4 with buffer); line #2 (7 plants inoculated<br>
with TuMV and 4 with buffer); line #3 (9 plants inoculated with TuMV and 4 with buffer); line<br>
7 (5 plants inoculated with TuMV and 4 with buffer).<br>
[0387] Western blot results of a representative plant from each transgenic line are shown in<br>
Figure 33, panel A. Levels of the large subunit (55kd) of RUBISCO were used as loading<br>
controls. All plants expressing 35S::miR-HC-Prol59a were resistant to the virus and did not<br>
show any visible symptoms nor expressed any TuMV CP. All WT plants and 35S::miR-P69IS9a<br>
plants showed TuMV infection symptoms and expressed high levels of TuMV CP. All mock-<br>
infected plants were normal and did not express any TuMV CP. Expression of artificial miRNA<br>
in miR-HC-Pro'59a and miR-P69J59a transgenic Arabidopsis is shown in Figure 33, panel B. The<br>
results of the infectivity assay are shown in Table 8.<br><br>
	TABLE 8		<br>
Infectivity Assay of Transgenic Arabidopsis	of the<br>
miR-HC-Pro159a and miR-P69l59a Challenged with TuMV Inocula<br>
	Number of seedlings		<br>
Transgenic line	Resistant     Susceptible	Total	Resistant rate (%)<br>
miR-HC-Pro'™a #10	12                 0	12	100<br>
miR-HC-Prol59a #11	12                 0	12	100<br>
miR-HC-Pro!59a #12	9                  0	9	100<br>
miR-HC-Pro159a #13	10                 0	10	100<br>
miR-P69159a#\	0                   8	8	0<br>
col-0	1                   11	12	8.3<br>
[0388] Fourteen days after infection with TuMV, samples of systemic leaves were collected<br>
and extracts assayed by ELISA. The results are means of ELISA readings of 9 or 12 plants from<br>
two different experiments. The results (Fig. 34) show that the miR-HC-Pro159a plants were<br>
completely resistant to TuMV infection. The readings were taken after 30 min of substrate<br>
hydrolysis.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
91<br>
EXAMPLE 27<br>
Production of More Than One Synthetic miRNAs<br>
from Same Transcript Using Homo-Polymeric pre-miRNAs<br>
[0389] Polymeric pre-miRNAs are artificial miRNA precursors consisting of more than one<br>
miRNA precursor units. They can either be hetero-polymeric with different miRNA precursors,<br>
or homo-polymeric containing several units of the same miRNA precursor. In previous<br>
Examples, it has been demonstrated that hetero-polymeric pre-miRNAs are able to produce<br>
different mature artificial miRNAs. For example, pre-miR-PDSlI69g-CPC3IS9a, which is a dimer<br>
comprising of pre-miR-CPC3l59a andpre-miR-PDSl'69s can produce mature miR-PDSl'69g and<br>
miR-CPC3159" when expressed in plant cells. Here, the use of homo-polymeric miRNA<br>
precursors to produce different mature artificial miRNAs is described.<br>
[0390]      Pre-miR-P69159a and pre-miR-HC-Pro159a were generated from the pre-miR159a<br>
backbone. They are derived from the same miRNA precursor.   They were linked together to<br>
form a homo-dimeric pre-miRNA, pre-miR-P69I59tt-HC-Pro159a. The DNA sequence follows.<br>
[0391]      Pre-miRP69159a-HC-P159a<br>
5'cagtrtgcttatgtcggatccataatatatttgacaagatactttgtttttcgatagatcttgatctgacgatggaagccacaagacaatcgag<br>
actttcatgagttgagcagggtaaagaaaagctgctaagctatggatcccataagccctaatccttgtaaagtaaaaaaggatttggttatatg<br>
gattgcatatctcaggagctttaacttgccctttaatggcttttactcttcAAAGTCTCGATTGTCTTGTGGa4TCCC<br>
atcttgatctgacgatggaagcagtcgagtgcgtgagcaagtcatgagttgagcagggtaaagaaaagctgctaagctatggatcccataa<br>
gccctaatccttgtaaagtaaaaaaggatttggttatatggattgcatatctcaggagctteacttgccctttaatggcttttactcttC/4C2TG<br>
CTCACGCACTCGACTGc 3' (SEQ ID NO: 166)<br>
[0392] The sequences in lower case text are At-miR159 backbone. The sequence in bold text<br>
is miR-P69159a. The sequence in italic text is miR-HC-Pro159a. The sequence in bold italic text<br>
is the linker sequence.<br>
[0393] A tobacco transient expression system was used to check whether this homo-dimeric<br>
miRNA precursor can produce the desired mature miR-P69159a and miR-HC-Pro159". rn this<br>
experiment, three treatments were performed: (1) Agrobacteria with 35S::pre-miR-P69J59a, (2)<br>
Agrobacteria with 35S::pre-miR-HC-Pro159a, and (3) Agrobacteria with 35::pre-miR-P69I59a-<br>
HC'Pro159". Northern analysis indicated that homo-dimeric miRNA precursor, Tpxe-miR-P69l59a-<br>
HC-Pro159a, can produce mature miR-P69l59a and miR-HC-Pro159a (Fig. 35).<br><br>
WO 2006/044322	PCT/US2005/036373<br>
92<br>
EXAMPLE 28<br>
Expression of miRNAs from pre-miRNAs Inserted in Intronic Sequences<br>
[0394] During RNA splicing, introns are released from primary RNA transcripts and<br>
therefore can potentially serve as precursors for miRNAs. In this example, the insertion of pre-<br>
miRNAs into such intronic sequences to produce artificial miRNAs is described.<br>
[0395] Most introns begin with the sequence 5'-GU-3' and end with the sequence 5VA.G-3'.<br>
These sequences are referred to as the splicing donor and splicing acceptor site, respectively. In<br>
addition to these sequences, the branch site which is located within introns is also important for<br>
intron maturation. Without the branch, site, an intron can not be excised and released from the<br>
primary RNA transcript. A branch site is located 20-50 nt upstream of the splicing acceptor site.<br>
Distances between the splice donor site and the branch site are largely variable among different<br>
introns. For this reason, it was decided to insert artificial pre-miRNAs in between these two<br>
sites, i.e., the splice donor site and the branch site, of introns.<br>
[0396] The Arabidopsis CARPRICE (CPC) gene contains three exons and two introns.<br>
Following the consensus sequence of the branch site 5'-CU(A7G)A(C/U)-3', where A is<br>
conserved in all transcripts, two branch sites located in 128 tol32 nt (intron 1) and 722 to 726 nt<br>
(intron 2) downstream of the start codon are predicted. Sequences from 111 to 114 nt and from<br>
272 to 697 nt, located in intron 1 and in intron 2, respectively, were replaced by artificial<br>
miRNA precursors containing the miR159a backbone. The DNA sequence follows.<br>
[0397]      CPC genome sequence<br>
atgtttcgttcagacaaggcggaaaaaatggataaacgacgacggagacagagcaaagccaaggcttcttgttccgaagGTCTGAT<br>
TTCTCTTTGTTTCTCTCTATATCITTTTGATCGGTTTGAGTCTGATTTTGTATGTTGT<br>
TTCGCAGaggtgagtagtatcgaatgggaagctgtgaagatgtcagaagaagaagaagatctcatttctcggatgtataaactcgttg<br>
gcgacagGTTAGAGACTCTTTCTCTCTCGATCCATCTTGTTGCTTTCTCTTTTTTTTGG<br>
TCTTTCATGTTTTGTCGAATCTGCTTAGATTTTGATCTCAAAGTCGGTCGTTTA<br>
TTTATGCATTTTCTTGGTTTTTCTATTATATTATTGGGTCTAACTTACCGAGCTG<br>
TCAATGACTGTGTTCAGCCTGATTTTTGATCTTGTTATTATTCTCTGTTTTTTGT<br>
TTTAGTTGTTCAAATAGCAAAACCTAATCAAGATTTCGTTTTCAGTTTCTTTTTT<br>
TATATATGATTCTTTAGCAAAACATATTCTTAATTTATGTCAGAACTCACTTTGG<br>
CTAGTTTGGTTCAATTTTGATTACAGCATGTTTGTATGAAGTCAAAGTGTAAAT<br>
TACGATTTTGGTTCGGTTCCATAGAATTTTAACCGAATTACAAACTTTATGCGG<br>
TTTTTATCGGAATAAAAGGTATTTGGTTAAGTGTAAGTTCCTCAACACTGACTGTT<br>
AGCCTATCCTACGTGGCGCGTAGgtgggagttgatcgccggaaggatcccgggacggacgccggaggagataga<br>
gagatattggcttatgaaacacggcgtcgtttttgccaacagacgaagagacttttttaggaaatga (SEQ ID NO: 167)<br>
[0398] The sequences in lower case are exons. The sequences in bold italic text are branch<br>
sites. The sequences in bold were replaced by artificial pre-miRNAs. Intron sequences include<br>
sequences in normal text, bold text and bold italic text.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
93<br>
[0399] Constructs 35S::CPC-A and 35S::CPC-£ were generated to check whether intron 1 or<br>
intron 2 of the unspliced CPC transcript can be used to insert artificial pre-miRNA for the<br>
production of artificial miRNAs. In the CPC-A construct, pre-miR-HC-Pro159a was inserted into<br>
intron I with no change in intron 2. In CPC-B, pre-miR-HC-Pro159a was inserted into intron 2<br>
with no change in intron 1 (Fig. 36). Agrobacterial cells carrying 35S::CPC-A, 35S::CPC-B,<br>
35S::pre-miR-HC-Pro159a, and 35S::pre-miR159a were infiltrated into N. benthamiana leaves<br>
for transient expression. Northern blot hybridizations using a probe complementary to miR-HC-<br>
Pro!59a showed that in 4 separate experiments leaf samples infiltrated with CPC-A and CPC-B<br>
expressed miR-HC-Pro159a (Fig. 37). This result demonstrates that both intron 1 and intron 2 of<br>
the CPC transcript can be used to produce artificial miRNAs.<br>
[0400] Constructs 35S::CPC-C and 35S::CPC-D were generated to determine the possibility<br>
of producing miRNAs in both introns. In CPC-C, pre-miR-HC-Pro159a was inserted into intron<br>
1 and pre-miR-P69l59a into intron 2. In CPC-D, pre-miR-P69159a was inserted into intron 1 and<br>
pre-miR-HC-Pro'59a into intron 2 (Fig. 38). Agrobacterial cells carrying 35S::CPC-C,<br>
35S::CPC-D, 35S::pre-miR-HC-PJ59a, and 35S.- :pre-miR-P69I59a were infiltrated into N.<br>
benthamiana leaves for transient expression. Figure 39 shows northern blot results of four<br>
independent experiments. Note that all of the four samples show signals corresponding to miR-<br>
HC-Pro159a miRNA and miR-P69159a, although the signal in sample 1 is weak (Fig. 39, 1 of<br>
35S::CPC-C). This weak signal could be due to a lower transient expression efficiency in this<br>
particular sample. A similar situation was encountered in sample 4 of the 35S..CPC-D<br>
experiment. These results demonstrate that it is possible to use CPC introns to produce two<br>
different artificial miRNAs simultaneously in one transcript.<br>
[0401] The above examples are provided to illustrate the invention but not to limit its scope.<br>
Other variants of the invention will be readily apparent to one of ordinary skill in the art and are<br>
encompassed by the appended claims. For example, in the Examples described above, pre-<br>
miR159a and pre-miR169g were used to generate artifical pre-miRNAs. However, other pre-<br>
miRNAs, such as described herein, could be used in place of pre-miR159a and pre-miR169g.<br>
All publications, patents, patent applications, and computer programs cited herein are hereby<br>
incorporated by reference. It will also be appreciated that in this specification and the appended<br>
claims, the singular forms of "a," "an" and "the" include plural reference unless the context<br>
clearly dictates otherwise.    It will further be appreciated that in this specification and the<br><br>
WO 2006/044322	PCT/US2005/036373<br>
94<br>
appended claims, The term "comprising" or "comprises" is intended to be open-ended, including<br>
not only the cited elements or steps, but further encompassing any additional elements or steps.<br><br>
WO 2006/044322	PCT/US2005/036373<br>
95<br><br>
We Claim:<br>
1.	A method of obtaining a disease resistant plant by down regulating a target sequence of a<br>
plant virus or plant viroid in a plant consisting of:<br>
(a)	transforming a plant cell susceptible to infection by the plant virus or plant<br>
viroid with a nucleic acid construct comprising a promoter functional in a plant cell<br>
operably linked to a polynucleotide, wherein the polynucleotide encodes one or more<br>
modified miRNA precursors, wherein each modified miRNA precursor comprises a<br>
fragment of a plant miRNA primary transcript that is modified in the miRNA sequence<br>
and the miRNA complementary sequence, further wherein each modified miRNA<br>
precursor is capable of forming a first double-stranded RNA or a hairpin, wherein each<br>
modified miRNA is a miRNA modified to be (i) fully complementary to a target<br>
sequence, (ii) fully complementary to a target sequence except for GU base pairing or<br>
(iii) fully complementary to a target sequence in the first ten nucleotides counting from<br>
the 5' end of the miRNA, wherein the target sequence of each modified miRNA is the<br>
same or different, wherein at least one target sequence encodes a gene silencing<br>
suppressor of a plant virus or plant viroid and wherein the nucleic acid construct is stably<br>
integrated into the genome of the transformed plant cell;<br>
(b)	obtaining a transgenic plant from the transformed plant cell of (a);<br>
(c)	expressing the nucleic acid construct in the transgenic plant of (b) for a time<br>
sufficient to produce each modified miRNA, wherein each modified miRNA binds to its<br>
target sequence to form a second double-stranded RNA which is cleaved thereby down<br>
regulating the target sequence; and<br>
(d)	selecting a transgenic plant of (c) that is resistant to a plant virus or plant<br>
viroid, wherein the plant virus or plant viroid comprises the at least one target sequence<br>
that encodes the gene silencing suppressor, wherein the nucleic acid construct is stably<br>
integrated into the genome of the resistant transgenic plant and wherein the resistant<br>
transgenic plant shows no symptoms of infection.<br>
2.	The method as claimed in claim 1, wherein the cell is selected from the group consisting<br>
of corn, wheat, rice, barley, oats, sorghum, millet, sunflower, safflower, cotton, soy,<br>
canola, alfalfa, Arabidopsis, and tobacco.<br><br>
3.	The method as claimed in claims 1 or 2, wherein the modified miRNA is a plant miRNA<br>
modified to be fully complementary to the target sequence.<br>
4.	The method as claimed in claims 1 or 2, wherein the modified miRNA is a plant miRNA<br>
modified to be fully complementary to the target sequence except for the use of GU base<br>
pairing.<br>
5.	The method as claimed in claim 4 or 5, wherein the plant miRNA is from a plant selected<br>
from the group consisting of Arabidopsis, tomato, soybean, rice, and corn.<br>
6.	The method as claimed in any one of claims 1 to 5, wherein the promoter is a pathogen-<br>
inducible promoter.<br>
7.	The method as claimed in claim 6, wherein the nucleic acid construct encodes for two or<br>
more modified miRNA precursor sequences.<br>
8.	The method as claimed in claim 7, wherein the nucleic acid construct is a hetero-<br>
polymeric precursor miRNA or a homo-polymeric precursor miRNA.<br>
9.	The method as claimed in any one of claims 1 to 5, wherein the nucleic acid construct is<br>
inserted into an intron of a gene or transgene of the cell.<br>
10.	The method as claimed in any one of claims 1 to 6, wherein the gene silencing<br>
suppressor of the plant virus or plant viroid is selected from the group consisting of HC-<br>
Pro and P69.<br>
11.	The method as claimed in any one of claims 1 to 10, wherein the miRNA is a plant<br>
miRNA selected from the group consisting of miR-159, miR-169 and miR-172.<br>
12.	The method as claimed in claims 11, wherein the miRNA is a plant miR-159.<br>
Dated this 8th day of May, 2007 	<br><br><br>
A METHOD OF OBTAINING A DISEASE RESISTANT PLANT BY DOWN REGULATING<br>
A TARGET SEQUENCE OF A PLANT VIRUS OR PLANT VIROID<br>
A method of obtaining a disease resistant plant by down regulating a target sequence of a plant<br>
virus or plant viroid in a plant consisting of: (a) transforming a plant cell susceptible to infection<br>
by the plant virus or plant viroid with a nucleic acid construct comprising a promoter functional<br>
in a plant cell operably linked to a polynucleotide, wherein the polynucleotide encodes one or<br>
more modified miRNA precursors, wherein each modified miRNA precursor comprises a<br>
fragment of a plant miRNA primary transcript that is modified in the miRNA sequence and the<br>
miRNA complementary sequence, further wherein each modified miRNA precursor is capable<br>
of forming a first double-stranded RNA or a hairpin, wherein each modified miRNA is a miRNA<br>
modified to be (i) fully complementary to a target sequence, (ii) fully complementary to a target<br>
sequence except for GU base pairing or (iii) fully complementary to a target sequence in the first<br>
ten nucleotides counting from the 5' end of the miRNA, wherein the target sequence of each<br>
modified miRNA is the same or different, wherein at least one target sequence encodes a gene<br>
silencing suppressor of a plant virus or plant viroid and wherein the nucleic acid construct is<br>
stably integrated into the genome of the transformed plant cell; (b) obtaining a transgenic plant<br>
from the transformed plant cell of (a); (c) expressing the nucleic acid construct in the transgenic<br>
plant of (b) for a time sufficient to produce each modified miRNA, wherein each modified<br>
miRNA binds to its target sequence to form a second double-stranded RNA which is cleaved<br>
thereby down regulating the target sequence; and (d) selecting a transgenic plant of (c) that is<br>
resistant to a plant virus or plant viroid, wherein the plant virus or plant viroid comprises the at<br>
least one target sequence that encodes the gene silencing suppressor, wherein the nucleic acid<br>
construct is stably integrated into the genome of the resistant transgenic plant and wherein the<br>
resistant transgenic plant shows no symptoms of infection.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-correspondence others 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMgMS4yLnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-correspondence others 1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1mb3JtIDMgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-form 3 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1wY3QgcmVxdWVzdCBmb3JtLnBkZg==" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-pct request form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDE2NDAta29sbnAtMjAwNy1wcmlvcml0eSBkb2N1bWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">01640-kolnp-2007-priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1BTUFOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04ucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-AMANDED PAGES OF SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1GT1JNLTEzLnBkZg==" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-FORM-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1GT1JNLTIucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-FORM-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LSgyOC0xMS0yMDExKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-(28-11-2011)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LUFCU1RSQUNULTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-ABSTRACT-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LUFNQU5ERUQgQ0xBSU1TLnBkZg==" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1rb2xucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSktMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">1640-kolnp-2007-description (complete)-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LURSQVdJTkdTLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-DRAWINGS-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LUVYQU1JTkFUSU9OIFJFUE9SVCBSRVBMWSBSRUNJRVZFRC5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-EXAMINATION REPORT REPLY RECIEVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LUZPUk0gMS0xLjEucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-FORM 1-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1rb2xucC0yMDA3LWZvcm0gMTgucGRm" target="_blank" style="word-wrap:break-word;">1640-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LUZPUk0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-FORM 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1rb2xucC0yMDA3LWZvcm0gMy0xLjIucGRm" target="_blank" style="word-wrap:break-word;">1640-kolnp-2007-form 3-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LU9USEVSUy0xLjEucGRm" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-OTHERS-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTY0MC1LT0xOUC0yMDA3LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3LnBkZg==" target="_blank" style="word-wrap:break-word;">1640-KOLNP-2007-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QtMDE2NDAta29sbnAtMjAwNy5qcGc=" target="_blank" style="word-wrap:break-word;">abstract-01640-kolnp-2007.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="252135-multistep-separation-of-plastics.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252137-thioxanthine-derivatives-as-myeloperoxidase-inhibitors.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252136</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1640/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-May-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>27-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE ROCKEFELLER UNIVERSITY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1230, YORK AVENUE, NEW YORK, NEW YORK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TOBODA, JOSE, REYES</td>
											<td>SUR 81 NO. 409 COL. LORENZO BOTURINI, 15820, MEXICO CITY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SOYANO, TAKASHI</td>
											<td>320, E. 53RD STREET, APT. 10 H, NEW YORK, NEW YORK 10022</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHUA,NAM-HAI</td>
											<td>422, E. 72ND STREET, APT. 33C, NEW YORK, NEW YORK 10021</td>
										</tr>
										<tr>
											<td>4</td>
											<td>NIU, QI-WEN</td>
											<td>100, DELAFIELD PLACE, STATEN ISLAND, NEW YORK 10310</td>
										</tr>
										<tr>
											<td>5</td>
											<td>LIN, SHIH-SHUN</td>
											<td>500, EAST 63RD STREET, APT. 21F, NEW YORK, NEW YORK 10021</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ZHANG, XIUREN</td>
											<td>504, E. 63RD STREET, APT. 7S, NEW YORK, NEW YORK 10021</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 1/00</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/036373</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>PCT/US04/33379</td>
									<td>2004-10-12</td>
								    <td>IB</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/671,089</td>
									<td>2005-04-14</td>
								    <td>IB</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252136-a-method-of-obtaining-a-disease-resistant-plant-by-down-regulating-a-target-sequence-of-a-plant-virus-or-plant-viroid by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:32:11 GMT -->
</html>
