<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252153-a-method-of-making-plant-with-increased-seed-yield by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252153:A METHOD OF MAKING PLANT WITH INCREASED SEED YIELD</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A METHOD OF MAKING PLANT WITH INCREASED SEED YIELD</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention concerns a method for improving growth characteristics of plants by modulating expression of a nucleic acid sequence encoding a NAP 1-like protein. The invention also relates to transgenic plants having improved growth characteristics, which plants have modulated expression of a nucleic acid encoding a NAP 1-like protein.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
Plants having improved growth characteristics and method for<br>
making the same<br>
The present invention concerns a method for improving growth characteristics, and in particular for increasing yield of a plant. More specifically, the present invention concerns a method for increasing yield by modulating expression in a plant of a nucleic add encoding a protein homologous to the Nudeosome Assembly Protein 1 (NAP1-like protein). The present invention also concerns plants having modulated expression of a nucleic acid encoding a NAP1-Iike protein, which plants have increased yield relative to corresponding wild type plants.<br>
Given the ever-increasing world population, and the dwindling area of land available for agriculture, it remains a major goal to improve the efficiency of agriculture and to increase the diversity of plants in horticulture. Conventional means for crop and horticultural improvements utilise selective breeding techniques to identify plants having desirable characteristics. However, such selective breeding techniques have several drawbacks, namely that these techniques are typically labour intensive and result in plants that often contain heterogeneous genetic complements that may not always result in the desirable trait being passed on from parent plants. Advances in molecular biology have allowed mankind to manipulate the germplasm of animals and plants. Genetic engineering of plants entails the isolation and manipulation of genetic material (typically in the form of DNA or RNA) and the subsequent introduction of that genetic material Into a plant. Such technology has led to the development of plants having various improved economic, agronomic or horticultural traits. Traits of particular economic interest are growth characteristics such as high yield.<br>
NAP proteins form a family of related proteins that are known in animals and are reported to be involved in chromatin-related activities. The family of NAP proteins is characterised by the presence of a conserved sequence known as the NAP domain. The NAP domain fe described in the Pfam (accession PF00956) and Interpro databases (accession IPR002164). NAP is a component of a multifactor complex that mediates DNA packaging into nucleosomes (Krude, T. and Keller, C. (2001) Cell. Mol. Life Sci. 58, 665-672). During the S phase of the eukaryotic cell division cycle, newly replicated DNA is rapidly assembled into chromatin. This process requires the coordinated action of several factors. In the initial stages, CAF1 (chromatin assembly factor 1) binds histone proteins H3 and H4 and directs them to the replication fork via PCNA binding- Subsequent deposition of histone proteins H2A and H2B is mediated by NAP1 proteins. NAP1 was first described in HeLA cells (von Lindern et al. (1992) Mol. Cell. Bio!. 12, 3346-3355) and was later found conserved in all eukaryotes.   In addition, NAP<br><br>
proteins are thought to regulate gene transcription and may influence cell differentiation and development.<br>
SET proteins are highly related to NAP proteins and play a role in various cellular processes in humans. In human cells, SET has been shown to be associated with various CDK-cyc!in complexes during the regulation of the cell cycle, such as G2/M transition. SET is a potent inhibitor of Protein Phosphatase 2A (PP2A) that is involved in several signalling pathways. The inhibitory activity of SET could be attributed to an acidic C-terminal domain (Canela et al. (2003) J. Biol. Chem. 278, 1158-1164). Other reports show the involvement of SET in DNA repair and transcription. SET is part of a complex that has DNA binding and bending activities mediated by the chromatin-associated protein HMG2. HMG2 facilitates the assembly of nucleoprotein higher-order structures by bending and looping DNA or by stabilizing underwound DNA. HMG2 co-precipitates with SET (Fan et al. (2002) Mol. Cell. Biol. 22, 2810-2820). SET is also reported to inhibit active DNA demethyiation (Cervoni et al. (2002) J. Biol. Chem. 277, 25026-25031). The oncoprotein Set/TAF-I, involved in the inhibition of histone acetylation, also inhibits demethyiation of ectopically methylated DNA resulting in gene silencing. Set/TAF-I is suggested to play a role in integrating epigenetic states of histones and DNA in gene regulation.<br>
The activity of NAP1 proteins is in part regulated by phosphoryiation. It was shown that subcellular localization of NAP1 in Drosophila is dependent on its phosphorylation state, which may be controlled by Casein Kinase II (Rodriguez at al (2000) J. Mol. Biol. 298, 225-238). Mammals are reported to possess several NAP1 proteins, while in yeast there is only one known NAP1 protein.<br>
Plant NAP1 orthologues remain largely unknown, although NAP1 proteins were reported from soybean (Yoon et al (1995) Mol. Gen. Genet. 249, 465-473), Arabidopsis, tobacco, maize and rice (Dong et al. (2003) Planta 216, 561-570). Phylogenetic analysis of plant MAPI-like genes has revealed that there are two subgroups, one related to NAP1 and the other to the SET protein (Figure 1). Most likely, later sequence divergence may have occurred since the two Arabidopsis, the two maize and the two tobacco sequences cluster together pointing to a more recent gene duplication effect. The Saccharomyces cerevisiae genome contains only one NAP-encoding gene, combining the functional properties of both the NAP1 and SET subgroups. Similarly, Template Activating Factor 1 (TAF-I), a homologue of NAP 1, combines both PP2a inhibiting activity (Saito etaL, Biochem. Biophys. Res. Comm. 259,471-475,1999) and chromatin remodelling activity (Kawase et al., Genes Cells 1, 1045-1056, 1996). It is therefore likely that the plant proteins of the NAP/SET family are largely redundant in function,<br><br>
particularly in the group of SET proteins where a lower degree of divergence is observed compared to the NAP group. Furthermore, there is structural evidence that NAP and SET proteins belong to the same family since they share the NAP domain which is followed by a C-terminal acidic region.<br>
Little is known about the function of NAPHike proteins in plants, although a role in mitosis and cytokinesis has been proposed {Dong et al 2003). The plant orthologues of the NAP1 protein most likely play a different role than their animal counterparts. Based on its nuclear localisation and on sequence similarities with the mammalian SET protein, a role in chromatin remodelling may be expected for the plant proteins. Furthermore, the plant NAP/SET group of proteins could be involved in the regulation of PP2A in plants. PP2A is one of the major phosphatases in plants, acting to a large extent on transcription factors and protein kinases, and proposed to regulate activity of proteins involved in a variety of cellular processes, including cell cycle (Ayaydin et al. (2000) Plant J. 23, 85-96), hormonal actions such as ABA mediated stomatal movement, germination (Kwak et al. (2002) Plant Ceil 14, 2849-2861), or auxin transport and root development (Garters et al 1996 EMBO J. 15,2115-2124). PP2A is furthermore reported to be involved in photosynthesis and light signalling (Sheen (1993) EMBO J. 12, 3497-3505) and in nitrogen assimilation (Hirose and Yamaya (1999) Plant Physiology 121,805-812).<br>
To date, no effects on agronomic traits have been described upon modulating the expression of NAP 1-like proteins in plants. It has now surprisingly been found that modulating expression of a nucleic acid encoding a NAP 1-1 ike protein in a plant gives plants having improved growth and development, in particular increased yield, more particularly increased seed yield when compared to corresponding wild type plants. Therefore according to a first embodiment of the present invention there is provided a method for improving growth and development of a plant, comprising introducing a genetic modification in a plant and selecting for modulated expression in this plant of a nucleic acid sequence encoding a NAP1-like protein. In particular, improved growth and development is increased yield of a plant, more particularly increased seed yield; and the modulated expression is increased expression.<br>
The term NAP1-like protein, as defined herein, refers to any protein comprising a NAP domain and an acidic C-terminal region and having PP2a phosphatase inhibiting activity. The term "NAP domain" as used herein is as defined by the Pfam database by accession number PF00956 (http^/www.sanger.ac.uk/Software/Pfam/; Bateman et aL, Nucleic Acids Research 30(1):276-280 (2002), see for example Table 1). Preferably, NAPHike protein sequences useful in the present invention have a NAP domain comprising a (T/S)FF(T/N/S/E/D)(W/F)(L/F)<br><br>
signature and/or the conserved amino acid sequence as given in SEQ ID NO:33. More preferably the NAP domain is as represented by SEQ ID NO: 32. The term "acidic C-terminal region* or "acidic C-terminus" as used herein refers to the carboxy-terminal end of the protein, which carboxy-terminal end is about 20 to 25 amino acids long, of which at least 13 residues are glutamic and/or aspartic acid.<br>
Table 1: Examples of Arabidopsis proteins comprising a NAP1 domain<br><br>
Optionally, the NAP 1-like protein useful in the methods of the present invention has, besides being an inhibitor of PP2a phosphatases, also chromatin remodelling activities. Methods for measuring inhibition of PP2a phosphatases are known in the art and comprise, for example, assays based on commercially available dye-labelled or fluorochrome-iabetled substrates or assays based on measurement of radioactivity in TCA soluble fractions after treatment of [^P]-labelled histone H1 with PP2a phosphatase (Saito et al.(1999) Biochem. Biophys. Res. Comm. 259, 471-475) or of radioactivity released from labelled myelin basic protein (Li et al, J. Bio!. Chem. 271, 11059-11062, 1996). An alternative method for measuring NAP1-like protein activity is given in Example 6, which is based on the method described by Ulloa et al. (FE8S Letters 330, 85-89, 1993). Chromatin remodelling activities may be assayed in several ways, such as measurement of DNA-binding activity in a gel retardation assay (Fan et al., 2002) or as measurement of histone-binding activity using ELISA (Rodriguez et al. (1997) Genomics 44, 253-265). DNA bending activity may be determined in a ligase-mediated circularization assay (Fan et al.f 2002) or in a supercoiling assay (FujB-Nakata et al.(1992) J. Biol. Chem. 267, 20980-20986; Yoon et al. (1995), Mol. Gen. Gen. 249,465-473).<br>
Preferably, the NAPHike protein, comprising a NAP domain and an acidic C-terminal region as described above, is of plant origin. The NAPHike protein is preferably from a dicotyledonous plant,. preferably from the family of Brassicaceae, more preferably from Arabidopsis thatiana, most preferably the NAP1-like protein is a protein as represented by SEQ ID NO: 2 or is a homologue, derivative or active fragment thereof, which hamologues, derivatives or active fragments comprise a NAP domain and the acidic C-terminus as<br><br>
described above, and which homologues, derivatives or active fragments furthermore have PP2A inhibiting activity. Preferably the NAP1-like proteins are encoded by a nucleic acid as represented by SEQ ID NO: 1 or a nucleic acid capable of hybridising therewith.<br>
The term *NAP1-Iike protein" includes proteins homologous to the protein presented in SEQ ID NO: 2. Preferred homologues to be used in the methods of the present invention comprise a NAP domain and an acidic C-terminus as described above. Preferably the homologues have a NAP domain comprising a (T/S)FF(T/N/S/E/D)(W/F)(L/F) signature and/or the conserved sequence of SEQ ID NO:33, and an acidic C-terminus of 20 to 25 residues comprising at least 13 aspartic and/or glutamic acid residues. Additionally, the NAP1 homologues have PP2a inhibiting activity and possess optionally also chromatin-remodelling activities, which can be measured as described above.<br>
Homologues of SEQ ID NO: 2 may be found in various eukaryotic organisms. The closest homologues are generally found in ffte plant kingdom. Homologues of SEQ ID NO: 2 suitable in the methods of the present invention include two tobacco proteins (SEQ ID NO: 7 and 9), a tomato protein (SEQ ID NO: 23), an alfalfa protein (SEQ ID NO: 11), the Arabidopsis protein represented by SEQ ID NO: 21. Other homologues suitable for practising the method according to the invention include for example the Zea mays homologues nfa104 (Accession No. AF384036, SEQ ID NO: 13) and nfa103 (Accession No. AF384035, SEQ ID NO: 19); or the Oryza sativa homologues (SEQ ID NO: 15 and 17).<br>
Methods for the search and identification of NAP1-like homologues would be well within the realm of persons skilled in the art. Such methods comprise comparison of the sequences represented by SEQ ID NO: 1 or 2, in a computer readable format, with sequences that are available in public databases such as MIPS (http://mips.gsf.de/), GenBank (http://www.ncbi.nlm.nih.gov/Genbank/index.html) or EMBL Nucleotide Sequence Database (http://www.ebi.ac.uk/embl/index.html), using algorithms well known in the art for the alignment or comparison of sequences, such as GAP (Needleman and Wunsch, J. Mol. Biol. 48; 443-453 (1970)), BESTFIT (using the local homology algorithm of Smith and Waterman (Advances in Applied Mathematics 2; 482-489 (1981))), BLAST (Altschul, S.F., Gish, W., Miller, W., Myers, E.W. &amp; Lipman, D.J., J. Mol. Biol. 215:403-410 (1990)), FASTA and TFASTA (W. R. Pearson and D. J. Ljpman Proc.Natl.Acad.Sci. USA 85:2444- 2448 (1988)). The software for performing BLAST analysis is publicly available through the National Centre for Biotechnology Information. The abovementioned homologues were identified using Wast default parameters (BLOSUM62 matrix, gap opening penalty 11 and gap extension penalty 1) and preferably the<br><br>
full length sequences are used for analysis. Alternatively, only the JMAP1 domain may be used for comparison, since this domain comprises the major part of the protein.<br>
"Homologues" of a NAPUike protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes having amino acid substitutions, deletions and/or insertions relative to the unmodified protein in question and having similar biological and functional activity as the unmodified protein from which they are derived. To produce such homologues, amino acids of the protein may be replaced by other amino adds having similar properties (such as similar hydrophobicity, hydrophiiicsty, antigenfcity, propensity to form or break a-helical structures or p-sheet structures). Conservative substitution tables are well known in the art (see for example Creighton (1984) Proteins. W.H. Freeman and Company).<br>
The homologues useful in the method according to the invention have at least 50% sequence identity or similarity (functional identity) to the protein sequence as represented in SEQ ID NO: 2 (GenBank accession NP_177596), alternatively at least 60% or 70% sequence identity or similarity to SEQ ID NO:2. Typically, the homologues have at least 80% sequence identity or simDarity, preferably at least 85% sequence identify or similarity, more preferably at least 90% sequence identity or similarity, most preferably at least 95%, 96%, 97%, 98% or 99% sequence identity or similarity to SEQ ID NO:2.<br>
Alternatively, the homologues useful in the method according to the invention have at least 40% sequence identity or similarity (functional identity) to the protein sequence as represented in GenBank accession NP_568844 (SEQ ID NO: 29), alternatively at least 50%, 60% or 70% sequence identity or similarity to SEQ ID NO:29. Typically, the homologues have at least 80% sequence identity or similarity to SEQ ID NO:29, preferably at least 85% sequence identity or similarity, further preferably at least 90% sequence identity or similarity, most preferably at least 95%, 96%, 97%, 98% or 99% sequence identity or similarity to SEQ ID NO:29.<br>
The percentage of identity may be calculated starting from full-length protein sequences or with certain (preferably conserved) regions of such a sequence, using alignment programs based on the Needleman and Wunsch algorithm (such as GAP), using the BLOSUM62 matrix, a gap opening penalty of 10 and a gap extension penalty of 0.5. For example the NAP domain as given in SEQ ID NO:32 may be used as query. The identification of such domains in a protein sequence would be well within the realm of the person skilled in the art and involve a computer readable format of the nucleic acids used in the methods of the present invention, the use of alignment software programs and the use of publicly available information on protein domains, conserved motife and boxes- An integrated search may be done using the INTERPRO    database    (Mulder   et   al.,    (2003)    Nucl.    Acids    Res.    31,    315-318,<br><br>
http://www.ebi.ac.uk/interpro/scan.html) which combines several databases on protein families, domains and functional sites, such as the PRODOM (Servant et al., (2002) Briefings in Bioirrfbrmatics 3, 246-251, http^/prodes.toulouse.inra.fr/prodom/2002.1/html/home.php), PIR (Huang et al. (2003) Nucl. Adds Res. 31, 390-392, http://pir.georgetown.edu/) or Pfam (Bateman et al. (2002) Nucl. Acids Res. 30, 276-280, http://pfam.wustl.edu/) databases. Sequence analysis programs designed for motif searching may be used for identification of conserved fragments, regions and domains as mentioned above. Suitable computer programs include for example MEME (Bailey and Elkan (1994) Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology, pp. 28-36, AAAI Press, Menlo Park, California, http://meme.sdsc.edu/meme/website/intro.html).<br>
Homologous proteins may be grouped in "protein families". A protein family may be defined by functional and sequence similarity analysis, such as, for example, Clustal W. A neighbour-joining tree of the proteins homologous to NAP1-like may be generated by the Clustal W program and gives a good overview of its structural and ancestral relationship. In a particular embodiment of the present invention, the NAP 1-like homologue(s) belong(s) to the same protein family as the protein corresponding to SEQ ID NO: 2.<br>
In the Arabidopsis genome, two family members of the NAPHike protein were identified (NWM77596 (SEQ ID NO: 2), NP_564063 (SEQ ID NO: 21)). Family members of the NAP1-like protein may also be identified in other plants such as rice or other monocotyledonous plants. Advantageously these family members are also useful in the methods of the present invention.<br>
Two special forms of homology, orthologous and paralogous, are evolutionary concepts used to describe ancestral relationships of genes. The term "paralogous11 relates to homologous genes that result from one or more gene duplications within the genome of a species. The term "orthologous" relates to homologous genes in different organisms due to ancestral relationship of these genes. The term "homologues" as used herein also encompasses paralogues and orthologues of the proteins useful in the methods according to the invention.<br>
Orthologous genes may be identified by querying one or more gene databases with a query gene of interest using for example the BLAST program. The highest-ranking subject genes that result from the search are then again subjected to a BLAST analysis, and only those subject genes that match again with the query gene are retained as true orthologous genes. For example, to find a rice orthologue of an Arabidopsis thaliana gene, one may perform a BLASTN or TBLASTX analysis on a rice database (such as (but not limited to) the Oryza sativa  Nipponbare database available at the NCBI  (http'7/www.ncbi.nlm.nih.gov) or the<br><br>
genomlc sequences of rice (cultivars indica or japonica)). In a next step, the obtained rice sequences are used in a reverse BLAST analysis using an Arabidopsis database. The results may be further refined when the resulting sequences are analysed with ClustalW and visualised in a neighbour joining tree. This method may be used to identify orthologues from many different species.<br>
"Homologues" of a NAP1-like encompass proteins having amino acid substitutions; insertions<br>
and/or deletions relative to the unmodified protein in question.  'Substitutions! variants" of a<br>
protein are those in which at least one residue in an amino acid sequence has been removed<br>
and a different residue inserted in its place.   Amino acid substitutions are typically of single<br>
residues, but may be clustered depending upon functional constraints placed upon the<br>
polypeptide; insertions will usually be of the order of about 1 to 10 amino acid residues, and<br>
deletions will range from about 1 to 20 residues. Preferably, amino acid substitutions comprise<br>
conservative amino acid substitutions.   "Inserttonal variants" of a protein are those in which<br>
one or more amino acid residues are introduced into a predetermined site in a protein.<br>
Insertions may comprise amino-terminal and/or carboxy-terminal fusions as well as intra-<br>
sequence insertions of single or multiple amino acids. Generally, insertions within the amino<br>
acid sequence will be smaller than amino- or carboxy-terminal fusions. Examples of amino- or<br>
carboxy-terminal fusion proteins or peptides include the binding domain or activation domain of<br>
a transcriptional activator as used in the yeast two-hybrid system, phage coat proteins,<br>
(histidineje-tag, glutathione S-transferase-tag, protein A, maltose-binding protein, dihydrofolate<br>
reductase, Tag-100 epitope, c-myc epitope, FLAG®-ep'rtope, lacZ, CMP (calmodulin-binding<br>
peptide), HA epitope, protein C epitope and VSV epitope. "Deletion variants" of a protein are<br>
characterised by the removal of one or more amino acids from the protein.   Amino acid<br>
variants of a protein may readily be made using peptide synthetic techniques well known in the<br>
art, such as solid phase peptide synthesis and the like, or by recombinant DNA manipulations.<br>
Methods for the manipulation of DNA sequences to produce substitution, insertion or deletion<br>
variants of a protein are well known in the art. For example, techniques for making substitution<br>
mutations at predetermined sites in DNA are well known to those skilled in the art and include<br>
M13 mutagenesis, T7-Gen in vitro mutagenesis (USB, Cleveland, OH), QuickChange Site<br>
Directed mutagenesis (Stratagene, San Diego, CA), PCR-mediated site-directed mutagenesis<br>
or other site-directed mutagenesis protocols.<br>
The term "derivatives" of a NAP1-like protein refers to peptides, oligopeptides, pdypeptides, proteins and enzymes which may comprise substftutions, deletions or additions of naturally and non-naturally occurring amino acid residues compared to the amino acid sequence of the naturally-occurring form of the NAPI-like protein (such as the protein presented in SEQ ID NO:<br><br>
2). "Derivatives" of a protein encompass peptides, oligopeptides, polypeptides, proteins and enzymes which may comprise naturally occurring altered, glycosylated, acylated or non-naturally occurring amino acid residues compared to the amino acid sequence of a naturally-occurring form of the polypeptide. A derivative may also comprise one or more non-amino acid substituents compared to the amino acid sequence from which it is derived, for example a reporter molecule or other Hgand, covalently or non-covalently bound to the amino acid sequence such as, for example, a reporter molecule which is bound to facilitate its detection, and non-naturally occurring amino acid residues relative to the amino acid sequence of a naturally-occurring protein.<br>
"Active fragments" of a NAP 1-like protein encompasses at least the amount of amino acid residues, sufficient to retain similar biological and/or functional activity compared to the naturally occurring protein. A preferred active fragment of a NAP1-like protein comprises at least a NAP domain (Pfam 00956) and an acidic C-terrninal domain enriched in D and/or E residues as described above, a more preferred active fragment furthermore comprises the conserved sequence of SEQ ID NO:33.<br>
The term NAP1-Uke nucleic acid/gene, as defined herein, refers to any nucleic acid encoding a<br>
NAP1-like protein as defined above, or the complement thereof. The nucleic acid may be<br>
derived (either directly or indirectly (if subsequently modified)) from any source provided that<br>
the nucleic add, when expressed in a plant, leads to modulated expression of a NAP1-like<br>
nucleic acid/gene or modulated activity and/or levels of a NAP1-like protein. The nucleic acid<br>
may be isolated from a eukaryotic source, such as yeast or fungi, plants (including algae) or<br>
animals (including humans).  This nucleic acid may be substantially modified from its native<br>
form in composition and/or genomic environment through deliberate human manipulation. The<br>
nucleic acid sequence is preferably a homologous nucleic acid sequence, i.e. a nucleic acid<br>
sequence encoding a protein structurally and/or functionally related to SEQ ID NO:2,<br>
preferably obtained from a plant, whether from the same plant species or different. Preferably,<br>
the nucleic acid is as represented by SEQ ID NO: 1 or a portion thereof or a nucleic acid<br>
sequence capable of hybridising therewith, which hybridising sequence encodes proteins<br>
having NAP 1-like protein activity as described above; or it is a nucleic add encoding an amino<br>
acid represented by SEQ ID NO: 2 or encoding a homologue, derivative or active fragment<br>
thereof. This term also encompasses variants of the nucleic acid encoding a NAP1-iike protein<br>
due to the degeneracy of the genetic code; allelic variants; and different splice variants of the<br>
nudeic acid encoding a NAP1-like protein, induding variants that are interrupted by one ov<br>
more intervening sequences.<br><br>
Advantageously, the method according to the present invention may also be practised using portions of a nucleic acid sequence encoding a NAP1-like protein as defined above, such as the NAP1-!ike protein encoded by SEQ ID NO: 1, or by using sequences that hybridise to a nucleic acid sequence encoding a NAP1-like protein as defined above, such as SEQ ID NO: 1, preferably under stringent c nditions, (which hybridising sequences encode proteins having NAP1-like activity), or by using nucleic acids encoding homologues, derivatives or active fragments of a NAPHike protein, such as the one represented by SEQ ID NO:2.<br>
Portions of a DNA sequence refer to a piece of DNA derived or prepared from an original (larger) DNA molecule, which DNM portion, when expressed in a plant, gives rise to plants having improved growth characteristics. Preferably, the improved growth characteristics are increased yield, more preferably increased seed yield, most preferably the increased seed yield comprises one or more cf increased harvest index, increased total weight of seeds and increased number of filled s .ds. The portion may comprise many genes, with or without additional control elements, or may contain just spacer sequences etc.<br>
The present invention also encompasses nucleic acid sequences capable of hybridising with a nucleic acid sequence encoding a NAP 1-like protein, which nucleic acid sequences may also be useful in practising the methods according to the invention. The term "hybridisation11 as defined herein is a process wherein substantially homologous complementary nudeotide sequences anneal to each other. The hybridisation process can occur entirely in solution, i.e. both complementary nucleic acids are in solution. Tools in molecular biology relying on such a process include the polymerase chain reaction (PCR; and all methods based thereon), subtractive hybridisation, random primer extension, nuclease S1 mapping, primer extension, reverse transcription, cDNA synthesis, differential display of RNAs, and DNA sequence determination. The hybridisation process may also occur with one of the complementary nucleic acids immobilised to a matrix such as magnetic beads, Sepharose beads or any other resin. Tools in molecular biology relying on such a process include the isolation of poly (A+) mRNA. The hybridisation process may furthermore occur with one of the complementary nucleic acids immobilised to a solid support such as a n'rtro-cellulose or nylon membrane or immobilised by e.g. photolithography to, for example, a silioeous glass support (the latter known as nucleic acid arrays or microarrays or as nucleic acid chips). Tools in molecular biology relying on such a process include RNA and DNA gel blot analysis, colony hybridisation, plaque hybridisation, in situ hybridisation and micro array hybridisation. In order to allow hybridisation to occur, the nucleic acid molecules are generally thermally or chemically denatured to melt a double strand into two single strands andfor to remove hairpins or other secondary structures from single stranded nucleic acids.   The stringency of hybridisation is<br><br>
influenced by conditions such as temperature, salt concentration and hybridisation buffer composition.<br>
For applications requiring high selectivity, one will typically desire to employ relatively stringent conditions to form the hybrids, e.g., one will select relatively low salt and/or high temperature conditions, such as provided by about 0.02 M to about 0.15 M NaCI at temperatures of about 50°C to about 70°C.<br>
High stringency conditions for hybridisation thus include high temperature and/or low salt concentration (salts include NaCI and Narcitrate) but may also be influenced by the inclusion of formamide in the hybridisation buffer and/or lowering the concentration of compounds such as SDS (sodium dodecyl sulphate) in the hybridisation buffer and/or exclusion of compounds such as dextran sulphate or polyethylene glycoi (promoting molecular crowding) from the hybridisation buffer. Sufficiently low stringency hybridisation conditions are particularly preferred for the isolation of nucleic acids homologous to the DNA sequences useful in the methods of the invention defined supra. Elements contributing to homology include allelism, degeneration of the genetic code and differences in preferred codon usage. The presence of monovalent cations in the hybridisation solution reduce the electrostatic repulsion between the two nucleic acid strands thereby promoting hybrid formation; this effect is visible for sodium concentrations of up to 0.4M. Formamide reduces the melting temperature of DNA-DNA and DNA-RNA duplexes with 0.6 to 0.7°C for each percent formamide, and addition of 50% formamide allows hybridisation to be performed at 30 to 45°C, though the rate of hybridisation will be lowered. Base pair mismatches reduce the hybridisation rate and the thermal stability of the duplexes. On average and for large probes, the Tm (temperature under defined ionic strength and pH, at which 50% of the target sequence hybridises to a perfectly matched probe) decreases about 1°C per % base mismatch.<br>
"Stringent hybridisation conditions" and "stringent hybridisation wash conditions" in the context of nucleic acid hybridisation experiments such as Southern and Northern hybridisations are sequence dependent and are different under different environmental parameters. For example, longer sequences hybridise specifically at higher temperatures. Specificity is typically the function of post-hybridisation washes. Critical factors of such washes include the ionic strength and temperature of the final wash solution.<br>
Generally, stringent conditions are selected to be about 50°C lower than the thermal melting point (Tm) for the specific sequence at a defined ionic strength and pH. The Tm is dependent upon the solution conditions and the base composition of the probe, and may be calculated using the following equation:<br><br>
Tm= 79.8°C + (18.5xlog[Na*]) + (58.4°Cx%[G+C]) - (820x(#bp in duplex)"1) - (0.5x% formamicle)<br>
Alternative formula's, depending on the types of hybrids, are known in the art, for example:<br>
•	DNA-DNA hybrids (Meinkoth and Wahl, Anal. Biochem., 138:267-284,1984):<br>
Tm= 81.5°C + 16.6xlog[NaT + OAU%[G/C^\ - 500x[LT1 - 0.61x% formamide .   DNA-RNA or RNA-RNA hybrids:<br>
Tm= 79.8 + 18.5 (logi0[Na7) + 0.58 (%G/Cb) + 118 (%G/Cb)2 - 820/Lc<br>
•	oligo-DNA or oligo-RNAd hybrids:<br>
For 
For 20-35 nudeotides:	Tm= 22 + 1.46 (/„)<br>
3 or for other monovalent cation, but only accurate in the 0.01-0.4 M range.<br>
5 only accurate for %GC in the 30% to 75% range.<br>
c L == length of duplex in base pairs.<br>
d Oligo, oligonudeotkte; /n, effective length of primer = 2x(no. of G/C)+(no. of A/T).<br>
Afofe; for each 1% formamide, the Tm is reduced by about 0.6 to 0.7°C, while the presence of 6M urea reduces the Tm by about 30°C<br>
More preferred stringent conditions are when the temperature is 20°C below Tm, and the most preferred stringent conditions are when the temperature is 10°C below Tm. Non-specific binding may also be controlled using any one of a number of known techniques such as, for example, blocking the membrane with protein containing solutions, additions of heterologous RNA, DNA, and SDS to the hybridisation buffer, and treatment with Rnase.<br>
Wash conditions are typically performed at or below hybridisation stringency. Generally, suitable stringent conditions for nucleic acid hybridisation assays or gene amplification detection procedures are as set forth above. More or less stringent conditions may also be selected.<br>
For the purposes of defining the level of stringency, reference can conveniently be made to Sambrook et al. (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York or to Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y. (1989). An example of low stringency conditions is 4-6x SSC / 0.1-0.5% w/v SDS at 37-45°C for 2-3 hours. Depending on the source and concentration of the nucleic acid involved<br><br>
in the hybridisation, alternative conditions of stringency may be employed such as medium stringent conditions. Examples of medium stringent conditions include 1-4x SSC / 0.25% w/v SDS at £ 45°C for 2-3 hours. An example of high stringency conditions includes 0.1-1x SSC / 0.1% w/v SDS at 60°C for 1-3 hours. The skilled artisan is aware of various parameters which may be altered during hybridisation and washing and which will either maintain or change the stringency conditions. For example, another stringent hybridisation condition is hybridisation at 4x SSC at 65°C, followed by a washing in 0.1 x SSC, at 65°C for about one hour. An alternative example of stringent hybridisation conditions is in 50% formamide, 4x SSC at 42°C. Still another example of stringent conditions include hybridisation at 62°C in 6x SSC, 0.05x BLOTTO and washing at 2x SSC, 0.1% w/v SDS at 62°C.<br>
The methods according to the present invention may also be practised using an alternative splice variant of a nucleic acid sequence encoding a NAP 1-like protein. The term "alternative splice varianf as used herein encompasses variants of a nucleic acid sequence in which selected introns and/or exons have been excised, replaced or added. Such variants will be ones in which the biological activity of the protein remains unaffected, which can be achieved by selectively retaining functional segments of the protein. Such splice variants may be found in nature or can be manmade. Methods for making such splice variants are well known in the art Therefore according to another aspect of the present invention, there is provided, a method for improving the growth characteristics of plants, comprising modulating expression in a plant of an alternative splice variant of a nucleic acid sequence encoding a NAP1 -like protein and/or by modulating activity and/or levels of a NAP1-like protein encoded by the alternative splice variant. Preferably, the splice variant is a splice variant of the sequence represented by SEQ ID NO: 1. Preferably the improved growth characteristics are increased yield, more preferably increased seed yield, most preferably increased seed yield comprises increased harvest index, increased number of filled seeds and/or increased total weight of seeds.<br>
Advantageously, the methods according to the present invention may also be practised using allelic variants of a nucleic acid sequence encoding a NAP1-like protein, preferably an allelic variant of a sequence represented by SEQ ID NO: 1. Allelic variants exist in nature and encompassed within the methods of the present invention is the use of these natural alleles. Allelic variants encompass Single Nudeotide Polymorphisms (SNPs), as well as Small Insertion/Deletion Polymorphisms (INDELs). The size of INDELs is usually less than 100 bp. SNPs and INDELs form the largest NAPI-like of sequence variants in naturally occurring polymorphic strains of most organisms. They are helpful in mapping genes and discovery of genes and gene functions. They are furthermore helpftjl in identification of genetic loci, e.g.<br><br>
plant genes, involved in determining processes such as growth rate, plant size and plant yield, plant vigor, disease resistance, stress tolerance etc.<br>
The activity of a NAP1-like protein or a homologue thereof may also be modulated {increased or decreased) by introducing a genetic modification (preferably in the locus of a NAP1-like gene). The locus of a gene as defined herein is taken to mean a genomic region, which includes the gene of interest and 10 KB up- or downstream of the coding region. The term "genetic modification" refers to a change by human intervention in the genetic content of a cell compared to a wild type cell and includes techniques like genetic engineering, breeding or mutagenesis. The change in genetic content comprises modifications of the genome and includes addition, deletion and substitution of genetic material in the chromosomes of a plant cell as well as in episornes. The term also encompasses the addition of extrachromosomal information to a plant cell. Preferably, the genetic modification results in modulated expression of a NAP1-tike nucleic acid, more preferably in increased expression of a NAP1-tike nucleic acid.<br>
The genetic modification may be introduced, for example, by any one (or more) of the following methods: TDNA activation, TILLING, site-directed mutagenesis, homologous recombination or by introducing and expressing in a plant cell a nucleic acid encoding a NAP 1-like protein or a homologue thereof. Following introduction of the genetic modification, there follows a step of selecting for moduated activity of a NAP 1-like protein. Preferably the modulated activity of a NAP1-fike protein is increased activity, which increase in activity gives plants having improved growth characteristics such as increased seed yield. The selection step may be based on monitoring the presence or absence of improved growth characteristics, or on monitoring the presence or absence of selectable or screenable marker genes linked an introduced nucleic acid of interest.<br>
T-DNA activation tagging (Hayashi et al. Science (1992) 1350-1353) involves insertion of T-DNA usually containing a promoter (may also be a translation enhancer or an intron), in the genomic region of the gene of interest or 10KB up- or down stream of the coding region of a gene in a configuration such that the promoter directs expression of the targeted gene. Typically, regulation of expression of the targeted gene by its natural promoter is disrupted and the gene falls under the control of the newly introduced promoter. The promoter is typically embedded in a T-DNA. This T-DNA is randomly inserted into the plant genome, for example, through Agrobacterium infection and leads to overexpression of genes near to the inserted T-DNA. The resulting transgenic plants show dominant phenotypes due to overexpression of genes close to the introduced promoter. The promoter to be introduced may be any promoter<br><br>
capable of directing expression of a gene in the desired organism, in this case a plant. For example, constitutive, tissue-preferred, cell type-preferred and inducible promoters are all suitable for use in T-DNA activation.<br>
A genetic modification may also be introduced in the locus of a NAPI-Bte gene using the technique of TILLING {Targeted Induced Local Lesions IN Genomes). This is a mutagenesis technology useful to generate and/or identify, and to eventually isolate mutagenised variants of a NAP1-Iike nucleic acid encoding a polypeptide capable of exhibiting NAP1-like activity. TILLING also allows selection of plants carrying such mutant variants. These mutant variants may even exhibit higher NAP1-like activity than that exhibited by the gene in its natura\ form. TILLING combines high-density mutagenesis with high-throughput screening methods. The steps typically followed in TILLING are: (a) EMS mutagenesis (Redei and Koncz, 1992; Feldmann et aL, 1994; Lightner and Caspar, 1998); (b) DNA preparation and pooling of individuals; (c) PCR amplification of a region of interest; (d) denaturation and annealing to allow formation of heteroduplexes; (e) DHPLC, where the presence of a heteroduplex in a pool is detected as an extra peak in the chromatogram; (f) identification of the mutant individual; and (g) sequencing of the mutant PCR product. Methods for TILLING are well known in the art (McCallum Nat Biotechnol. 2000 Apr; 18(4):455-7, reviewed by Stemple 2004 (TILLING-a high-throughput harvest for functional genomics. Nat Rev Genet 2004 Feb;5(2):145-50.)).<br>
Site-directed mutagenesis may be used to generate variants of NAP1-Hke nucleic acids or portions thereof. Several methods are available to achieve site-directed mutagenesis, the most common being PCR based methods (current protocols in molecular biology. Wiley Eds. http://www.4ulr.com/products/currentprotocols/index.html).<br>
TDNA activation, TILLING and site-directed mutagenesis are examples of technologies that enable the generation of novel alleles and NAPt-ttke variants.<br>
Homologous recombination allows introduction in a genome of a selected nucleic acid at a defined selected position. Homologous recombination is a standard technology used routinely in biological sciences for lower organisms such as yeast or the moss Physcomitrella. Methods for performing homologous recombination in plants have been described not only for model plants (Offringa et ak Extrachromosomal homologous recombination and gene targeting in plant cells after Agrobacterium-mediatedi transformation. 1990 EMBO J. 1990 Oct; 9(10):3077-84) but also for crop plants, for example rice (Terada R, Urawa H, Inagaki Y, Tsugane K, lida S. Efficient gene targeting by homologous recombination in rice. Nat Biotechnd. 2002. lida and Terada: A tale of two integrations, transgene and T-DNA: gene targeting by homologous<br><br>
recombination in rice. Curr Opin Biotechnol. 2004 Apr; 15(2):132-8). The nucleic acid to be targeted (which may be a NAP1-like nucleic acid or variant thereof as hereinbefore defined) need not be targeted to the locus of a NAP1-like gene, but may be introduced in, for example, regions of high expression. The nucleic acid to be targeted may be an improved allele used to replace the endogenous gene or may be introduced in addition to the endogenous gene.<br>
IMAP1-like proteins have a typical domain organisation, consisting of a NAP domain followed by an acidic C-terminal region. Therefore, it is envisaged that engineering of the domains of the NAP1-like protein in such a way that the activity of the NAP 1
Accordingly, as another aspect of the invention, there is provided a method for improving plant growth and development when compared to corresponding wild type plants, preferably for increasing plant yield, more preferably for increasing seed yield of a plant, comprising modulating expression, preferably increasing expression in a plant of a nucleic acid sequence encoding a NAP1-like protein and/or modulating activity and/or levels in a plant of a NAP 1-like protein, preferably increasing activity and/or levels of a NAP1-like protein, wherein the nucleic add sequence and the proteins include variants chosen from:<br>
(i)     a nucleic acid encoding a NAPI-like protein, wherein the NAP1-Jike protein is preferably as represented by SEQ ID NO: 1 or encodes a NAP1-Iike protein as represented by SEQ ID NO: 2; (ii)    an alternative splice variant of a nucleic acid sequence encoding a NAP1-like<br>
protein or wherein said NAP1-like protein is encoded by a splice variant; (iii)   an alleiic variant of a nucleic acid sequence encoding a NAP1-Iike protein or<br>
wherein said NAP1-Iike protein is encoded by an alleiic variant; (iv)   sequence capable of hybridising to a NAPHike encoding nucleic add, preferably<br>
under stringent conditions; (v)    a NAP1-Iike protein<br><br>
(vi)   a NAPHike protein as represented by SEQ ID NO: 2<br>
(vii)   homologues and derivatives of a NAPI-like protein, preferably of the NAPUike protein presented in SEQ ID NO:2,<br>
and wherein said NAP1-Iike protein comprises a NAP domain and an acidic C-terminal,<br>
and has PP2a phosphatase inhibiting activity..<br>
Preferably, the increased seed yield comprises one or more of increased harvest index, increased number of filled seeds and/or increased total weight of seeds.<br>
In the methods of the present invention modulated expression, and in particular increased expression, of a nucleic acid is envisaged. Modulating expression (increasing or decreasing expression) of a nucleic acid encoding a NAP 1-like protein or modulation of the activity and/or levels of the NAP1-like protein itself encompasses altered expression of a gene and/or altered activity and/or levels of a gene product, namely a polypeptide, in specific cells or tissues. Altered (increased or decreased) expression of a gene and/or altered (increased or decreased) activity and/or levels of a gene product may be effected, for example by chemical means and/or recombinant means. Modulating expression of a gene and/or levels of a gene product and/or modulating activity of a gene product may be effected directly through the modulation of expression of a NAP1-like-encoding gene and/or directly through the modulation of the activity and/or levels of a NAP1-Iike protein. The modulated expression may result from altered expression levels of an endogenous NAP1-like gene and/or may result from modulated expression of a NAP 1-like encoding nucleic acid that was previously introduced into a plant. Similarly, modulated levels and/or activity of a NAP 1-like protein may be the result of altered expression levels of an endogenous NAP1-like gene and/or may result from altered expression of a NAP 1-like encoding nucleic acid that was previously introduced into a plant. Additionally or alternatively, the modulation of expression as mentioned above is effected in an indirect way, for example it may be effected as a result of decreased or increased levels and/or activity of factors that control the expression of a NAP1-like gene or that influence the activity and/or levels of the NAP 1-like protein.<br>
According to a preferred embodiment of the present invention, modulation of expression of a nucleic acid encoding a NAP1-like protein and/or modulation of activity and/or levels of the NAP 1-like protein itself is effected by recombinant means. Such recombinant means may comprise a direct and/or indirect approach for modulation of expression of a nucleic acid and/or for modulation of the activity and/or levels of a protein.<br><br>
A direct and preferred approach for modulating expression of a NAP1-like gene or modulating the activity and/or levels of a NAP1-iike protein, comprises introducing into a plant an isolated nucleic acid sequence encoding a NAP 1-like protein or a homologue, derivative or active fragment thereof. The nucleic add may be introduced into a plant by, for example, transformation. Therefore, according to a preferred aspect of the present invention, there is provided a method for improving growth and development of a plant, in particular for increasing yield of a plant comprising a genetic modification of the plant, which genetic modification comprises introducing a NAP1-like encoding nucleic acid into a plant. Preferably the increased plant yield is increased seed yield, more preferably comprises one or more of increased harvest index, increased number of filled seeds or increased total weight of seeds.<br>
According to a preferred aspect of the present invention, enhanced or increased expression of<br>
a nucleic acid is envisaged.   Methods for obtaining enhanced or increased expression of<br>
genes or gene  products  are well  documented  in the art and  include, for example,<br>
overexpressbn driven by a suitable (preferably strong) promoter, the use of transcription<br>
enhancers or translation enhancers. The term ovenexpression as used herein means any form<br>
of expression that is additional to the original wild-type expression level. Preferably the nucleic<br>
acid to be introduced into the plant and/or the nucleic add that is to be overexpressed in the<br>
plants is in sense direction with respect to the promoter to which it is operably linked.<br>
Preferably, the nucleic add to be overexpressed encodes a NAP1-like protein, further<br>
preferably the nucleic acid sequence encoding the NAP1-like protein is isolated from a<br>
dicotyledonous plant, preferably of the family Brassicaceae, further preferably the sequence is<br>
isolated from Arabidopsis thaliana,   most  preferably the  nudeic acid sequence  is  as<br>
represented by SEQ ID NO: 1 or a portion thereof, or encodes an amino acid sequence as<br>
represented by SEQ ID NO: 2 or a homologue, derivative or active fragment thereof.<br>
Alternatively, the nucleic acid sequence encoding the NAP 1-like protein is as represented in<br>
SEQ ID NO: 20 (GenBank Accession Number NMJI01738) or is a portion thereof, or encodes<br>
an amino add sequence as represented in SEQ ID NO: 21 (GenBank Accession Number<br>
NP_564063) or encodes a homologue, derivative or active fragment thereof.   It should be<br>
noted that the applicability of the invention does not rest upon the use of the nucleic acid<br>
represented by SEQ ID NO: 1, nor upon the nudeic acid sequence encoding the amino acid<br>
sequence of SEQ ID NO: 2, but that other nudeic add sequences encoding homoiogues,<br>
derivatives or active fragments of SEQ ID NO: 2, or portions of SEQ ID NO: 1, or sequences<br>
hybridising with SEQ ID NO: 1 may be used in the methods of the present invention,   in<br>
particular, homologues from other species such as tobacco (SEQ ID NO: 7 or 9), maize (SEQ<br>
ID NO: 13 or 19), Medicago sativa {SEQ ID NO: 11), tomato (SEQ ID NO: 23) or rice (SEQ ID<br>
NO: 15 or 17) are also useful in the methods of the present invention.<br><br>
The present invention relates to methods to improve growth characteristics of a plant or to methods to produce plants with improved growth characteristics, wherein the growth characteristics include increased yield, comprising any one or more of: increased number of tillers, increased number of first panicles (being the tallest panicle and all the panicles that overlap with the tallest panicle when aligned vertically), increased number of second panicles, increased total number of seeds, increased number of filled seeds, increased total seed weight per plant, increased harvest index, increased thousand kernel weight. The present invention also provides methods to improve one of the above mentioned growth characteristics, without causing a penalty on one of the other growth characteristics, for example increase of the number of filled seeds while retaining the same number of spikelets per panicle.<br>
The term "increased yield" encompasses an increase in biomass in one or more parts of a plant relative to the corresponding part(s) of wild-type plants. The term encompasses an increase in seed yield, which includes an increase in the biomass of the seed (seed weight) and/or an increase in the number of (filled) seeds and/or in the size of the seeds and/or an increase in seed volume, each relative to corresponding wild-type plants. Depending on the crop, the plant parts in question may be above-ground biomass (e.g. com, when used as silage, sugarcane), roots (e.g. sugar beet), fruit (e.g. tomato), cotton fibres, or any other part of the plant which is of economic value. Taking rice as an example, a yield increase may be manifested by an increase in one or more of the following: number of panicles per plant, number of spikelets per panicle, number of flowers per panicle, increase in the seed filling rate, increase in thousand kernel weight, among others. For maize, the increase of seed yield may be reflected in for example an increase of rows (of seeds) per ear and/or an increased number of kernels per row. An increase in seed size and/or volume may also influence the composition of seeds. An increase in seed yield could be due to an increase in the number and/or size of flowers. An increase in yield might also increase the harvest index, which is expressed as a ratio of the yield of harvestable parts, such as seeds, over the total biomass; or thousand kernel weight. Increased yield also encompasses the capacity for planting at higher density (number of plants per hectare or acre).<br>
Also modified cell division may contribute to yield increase. The term "modified cell division" encompasses an increase or decrease in cell division or an abnormal cell division/cytokinesis, altered plane of division, altered cell polarity, altered cell differentiation. The term also comprises phenomena such as endomrtosis, acytokinesis, polyploidy, polyteny and endoreduplication.<br><br>
It can be envisaged that plants having increased yield also exhibit a modified growth rate when compared to corresponding wild-type plants. The term "modified growth rate" as used herein encompasses, but is not limited to, a faster rate of growth in one or more parts of a plant (including including seeds), at one or more stages in the life cycle of a plant. Plants with improved growth may show a modified growth curve and may have modified values for their Tmid or T90 (respectively the time needed to reach half of their maximal size or 90% of their maximal size, each relative to corresponding wild-type plants). The term "improved growth" encompasses enhanced vigour, earlier flowering, modified cycling time.<br>
According to a preferred feature of the present invention, performance of the methods according to the present invention results in plants having increased yield, in particular plants having increased seed yield. Preferably, the increased seed yield includes at least an increase in any one or more of number of filled seeds, total seed weight, and harvest index, each relative to control plants. Therefore, according to the present invention, there is provided a method for increasing yield of plants, which method comprises modulating expression of a nucleic acid sequence encoding a NAP 1-like protein and/or modulating activity of the NAP1-like protein itself in a plant, preferably wherein the NAP 1-like protein is encoded by a nucleic acid sequence represented by SEQ ID NO: 1 or a portion thereof or by sequences capable of hybridising therewith or wherein the NAP1-like protein is represented by SEQ ID NO: 2 or is a homologue, derivative or active fragment thereof. Alternatively, the NAP1-like protein may be encoded by nucleic acid sequences as represented in SEQ ID NO: 20 (GenBank Accession Number NM_101738), SEQ ID NO: 6, 8,10,12,14,16, 18 or 22, or by a portion thereof or by sequences capable of hybridising therewith, or the NAP 1-like protein may be as represented in SEQ ID NO: 21 (GenBank Access on NumberNP_564063), SEQ ID NO: 7, 9, 11, 13, 15, 17, 19,23 or may be a homologue, derivative or active fragment of any thereof.<br>
The methods of the present invention are favourable to apply to crop plants because the methods of the present invention are used to increase one or more of the total seed weight, number of filed seeds and harvest index of a plant. Therefore, the methods of the present invention are particularly useful for crop plants cultivated for their seeds, such as cereals, sunflower, soybean, pea, flax, lupines ^nola. Accordingly, a particular embodiment of the present invention relates to a meth to increase seed yield (increased total seed weight, increased number of filled seeds and/or to increase harvest index) of a cereal.<br>
According to a further embodiment of the present invention, genetic constructs and vectors to facilitate introduction and/or expression of the nudeotide sequences useful in the methods<br><br>
according to the invention are provided. Therefore, according to a second embodiment of the present invention, there is provided a gene construct for expression in a plant, comprising:<br>
(i)     a nucleic acid sequence encoding a NAP 1-like protein;<br>
(ii) one or more control sequences capable of driving expression in a plant of the nucleic acid sequence of (i); and optionally<br>
(iii)    a transcription termination sequence.<br>
Constructs useful in the methods according to the present invention may be created using recombinant DNA technology well known to persons skilled in the art. The gene constructs may be inserted into vectors, which may be commercially available, suitable for transforming into plants and suitable for expression of the gene of interest in the transformed cells. The genetic construct may be an expression vector wherein the nucleic acid sequence is operably linked to one or more control sequences allowing expression in prokaryotic and/or eukaryotic host cells.<br>
According to a preferred embodiment of the invention, the genetic construct is an expression vector designed to overexpress the nucleic acid sequence. The nucleic acid sequence capable of modulating expression of a nucleic acid encoding a NAP1-Iike protein and/or activity of the NAP1-like protein itself may be a nucleic acid sequence encoding a NAP1-like protein or a homologue, derivative or active fragment thereof, such as any of the nucleic acid sequences described hereinbefore. A preferred nucleic acid sequence is the sequence represented by SEQ ID NO: 1 or a portion thereof or sequences capable of hybridising therewith or a nucleic add sequence encoding a protein represented by SEQ ID NO: 2 or a homologue, derivative or active fragment thereof. Preferably, this nucleic acid is cloned in sense orientation relative to the control sequence to which it is operably linked.<br>
Plants are transformed with a vector comprising the sequence of interest (i.e., the nucleic acid sequence capable of modulating expression of nucleic acid encoding a NAP1-like protein), which sequence is operably linked to one or more control sequences (at least a promoter). The terms "regulatory element", "control sequence" and "promoter" are all used herein interchangeably and are to be taken in a broad context to refer to regulatory nucleic acid sequences capable of effecting expression of the sequences to which they are ligated. Encompassed by the aforementioned terms are transcriptional regulatory sequences derived from a classical eukaryotic genomic gene (including the TATA box which is required for accurate transcription initiation, with or without a CCAAT box sequence) and additional regulatory elements (i.e. upstream activating sequences, enhancers and silencers) which alter gene expression in response to developmental and/or external stimuli, or in a tissue-specific<br><br>
manner. Also included within the term is a transcriptiona! regulatory sequence of a classical prokaryotic gene, in which case it may include a -35 box sequence and/or -10 box transcriptional regulatory sequences. The term "regulatory elemenf also encompasses a synthetic fusion molecule or derivative which confers, activates or enhances expression of a nucleic acid molecule in a cell, tissue or organ. The term "operably linked" as used herein refers to a functional linkage between the promoter sequence and the gene of interest, such that the promoter sequence is able to initiate transcription of the gene of interest<br>
Advantageously, any type of promoter may be used to drive expression of the nucleic acid sequence depending on the desired outcome. Preferably, the nucleic acid sequence encoding a NAP1-like protein is operably linked to a constitutive promoter. The term "constitutive" as defined herein refers to a promoter that is expressed predominantly in at least one tissue or organ and predominantly at any life stage of the plant Preferably the promoter is expressed predominantly throughout the plant Preferably, the constitutive promoter is the GOS2 promoter from rice, or a promoter of similar strength and/or a promoter with a similar expression pattern. Alternatively, tissue specific promoters may be used. For example, in cases where increased seed yield is envisaged, the use of seed preferred, flower preferred, meristem preferred promoters or promoters active in dividing cells can be contemplated. Promoter strength and/or expression pattern may be analysed for example by coupling the promoter to a reporter gene and assay the expression of the reporter gene in various tissues of the plant One suitable reporter gene well known to a person skilled in the art is beta-glucuronidase.<br>
Examples of alternative promoters with their respective expression pattern are presented in Table 2, and these promoters or derivatives thereof may be useful for the methods of the present invention.<br>
Table 2: Examples of promoters for use in the performance of the present invention<br><br><br><br><br><br>
Optionally, one or more terminator sequences may also be used in the construct introduced into a plant. The term "terminator" encompasses a control sequence which is a DNA sequence at the end of a transcriptional unit which signals 3' processing and polyadenylation of a primary transcript and termination of transcription. Additional regulatory elements may include transcriptional as well as translational enhancers. Those skilled in the art will be aware of terminator and enhancer sequences which may be suitable for use in performing the invention. Such sequences would be known or may readily be obtained by a person skilled in the art<br>
The genetic constructs of the invention may further include an origin of replication sequence which is required for maintenance and/or replication in a specific cell type. One example is when a genetic construct is required to be maintained in a bacterial cell as an episomal genetic element (e.g. plasmld or cosmid molecule). Preferred origins of replication include, but are not limited to, the f1 -on and colE1.<br>
The genetic construct may optionally comprise a selectable marker gene. As used herein, the term "selectable marker gene" includes any gene which confers a phenotype on a cell in which<br><br>
it is expressed to facilitate the identification and/or selection of cells which are transfectecl or transformed with a nucleic acid construct of the invention. Suitable markers may be selected from markers that confer antibiotic or herbicide resistance, that introduce a new metabolic trait or that allow visual selection. Examples of selectable marker genes include genes conferring resistance to antibiotics (such as npfll that phosphorylates neomycin and kanamycin, or hpt, phosphorylating hygromycin), to herbicides (for example bar which provides resistance to Basta; aroA or gox providing resistance against glyphosate), or genes that provide a metabolic trait (such as manA that allows plants to use mannose as sole carbon source). Visual marker genes result in the formation of colour (for example {J-glucuronidase, GUS), luminescence (such as luciferase) or fluorescence (Green Fluorescent Protein, GFP, and derivatives thereof).<br>
In a preferred embodiment, the genetic construct as mentioned above, comprises a NAPUIike gene in sense orientation coupled to a promoter that is preferably a constitutive promoter, such as the rice GOS2 promoter. Therefore, another aspect of the present invention is a vector construct carrying an expression cassette essentially similar to SEQ ID NO: 3, comprising the rice GOS2 promoter, the Arabidopsis NAP1-like gene and the T-zein + T-rubisco deltaGA transcription terminator sequence. A sequence essentially similar to SEQ ID NO: 3 encompasses a nucleic add encoding a protein homologous to SEQ ID NO: 2 or hybridising to SEQ ID NO: 1, which nucleic acid is operably linked to a rice GOS2 promoter or a promoter with a similar expression pattern and/or which nucleic add is linked to a transcription termination sequence.<br>
Therefore according to another aspect of the invention, there is provided a gene construct, comprising an expression cassette in which is located a nucleic acid sequence encoding an NAP1-like protein, chosen from the group comprising:<br>
(i)     a nucleic acid sequence represented by SEQ ID NO: 1 or the complement strand<br>
thereof (ii)    a nudeic acid sequence encoding an amino acid sequence represented by SEQ<br>
ID NO: 2 or homologues, derivatives or active fragments thereof, (in)    a nucleic acid sequence capable of hybridising (preferably under stringent conditions) with a nucleic acid sequence of (i) or (ii) above, which hybridising sequence preferably encodes a protein having NAPHike protein activity; (iv)   a nudeic add sequence according to (i) to (iii) above which is degenerate as a<br>
results of the genetic code;<br>
(v)    nudeic add sequence which is an allelic variant to the nudeic add sequences according to (i) to (iii);<br><br>
(vi)   nucleic acid sequence which is an alternative splice variant to the nucleic acid sequences according to (i) to (Hi);<br>
The present invention also encompasses plants obtainable by the methods according to the present invention. The present invention therefore provides plants obtainable by the methods of the present invention, which plants have increased yield and which plants have modulated NAP 1-1 ike protein activity and/or levels and/or modulated expression of a nucleic acid encoding a NAP1-like protein. Preferably, the plants are transgenic plants comprising an isolated nucleic acid sequence encoding a NAP 1-like protein, characterized in that the transgenic plant has been selected for having increased yield. Further preferably, the transgenic plant has been selected for modulateo expression of a nucleic acid encoding a NAP1-like protein. Furthermore preferably, the transgenic plant has been selected for modulated expression of a nucleic acid encoding a NAP 1-like protein as represented by SEQ ID NO:2<br>
According to a third embodiment of the present invention, there is provided a method for the production of transgenic plants having improved growth characteristics, comprising introduction and expression in a plant of a nucleic acid encoding a NAP 1-like protein as described above. Preferably, the improved growth characteristics comprise increased yield, more preferably increased seed yield, most preferably comprising one or more of increased number of filled seeds, increased harvest index or increased total weight of seeds.<br>
More specifically, the present invention provides a method for the production of transgenic plants having increased yield, which method comprises:<br>
(i) introducing into a plant cell a nucleic acid sequence encoding a NAP1-Iike protein;<br>
(ii) regenerating and/or growing a plant from a transgenic plant cell.<br>
The NAP1-like protein itself and/or the NAP14ike nucleic acid itself may be introduced directly into a plant cell or into the plant itself (including introduction into a tissue, organ or any other part of the plant). According to a preferred feature of the present invention, the nucleic acid is preferably introduced into a plant by transformation. The nucleic acid is preferably as represented by SEQ ID NO: 1 or a portion thereof or sequences capable of hybridising therewith, or is a nucleic acid encoding an amino acid sequence represented by SEQ ID NO: 2 or a homologue, derivative or active fragment thereof. Alternatively, the nucleic add sequence is as represented in SEQ ID NO: 20 (GenBank Accession Number NM_101738), SEQ ID NO: 6,8,10,12,14,16, 18, 22 or a portion thereof or sequences capable of hybridising with any of<br><br>
the aforementioned sequences. The amino acid sequence may alternatively be a sequence as represented in SEQ ID NO: 21 (GenBank Accession NumberNP_564063), SEQ ID NO: 7, 9, 11,13,15,17,19,23 or homologues, derivatives or active fragments thereof.<br>
The term "transformation" as referred to herein encompasses the transfer of an exogenous polynucieotide into a host cell, irrespective of the method used for transfer. Plant tissue capable of subsequent clonal propagation, whether by organogenesis or embryogenesis, may be transformed with a genetic construct of the present invention and a whole plant regenerated therefrom. The particular tissue chosen will vary depending on the clonal propagation systems available for, and best suited to, the particular species being transformed. Exemplary tissue targets include leaf disks, pollen, embryos, cotyledons, hypocotyls, megagametophytes, callus tissue, existing meristematic tissue (e.g., apical meristem, axillary buds, and root meristems), and induced meristem tissue (e.g., cotyledon meristem and hypocotyl meristem). The polynucieotide may be transiently or stably introduced into a host cell and may be maintained non-integrated, for example, as a plasmid. Alternatively, it may be integrated into the host genome. The resulting transformed plant ceil can then be used to regenerate a transformed plant in a manner known to persons stalled in the art.<br>
Transformation of a plant species is now a fairly routine technique.  Advantageously, any of<br>
several transformation methods may be used to introduce the gene of interest into a suitable<br>
ancestor cell.     Transformation  methods include the use of liposomes, electroporation,<br>
chemicals that increase free DNA uptake, injection of the DNA directly into the plant, particle<br>
gun bombardment, transformation using viruses or pollen and microprojection.  Methods may<br>
be selected from the calcium/polyethylene glycol method for protoplasts (Krens, F.A. et alM<br>
1882, Nature 296,  72-74;  Negrutiu I. et al., June  1987, Plant Mol.  Bid. 8, 363-373);<br>
electroporation   of  protoplasts  (Shillito  R.D.   et  al.,   1985   Bio/Technol  3,   1099-1102);<br>
microinjection into plant material (Crossway A. et al., 1986, Mol. Gen Genet 202, 179-185);<br>
DNA or RNA-coated particle bombardment (Klein T-M. et al., 1987, Nature 327, 70) infection<br>
with (non-integrative) viruses and the like.   Transgenic rice plants expressing an NAP1-Iike<br>
gene are preferably produced via Agro&amp;acferiurn-mediated transformation using any of the well<br>
known methods for rice transformation, such as described in any of the following: European<br>
patent application EP 1198985 A1, Aldemita and Hodges (Planta, 199, 612-617, 1996); Chan<br>
et al. (Plant Mol. Biol. 22 (3) 491-506,1993), Hiei et al. (Plant J. 6 (2) 271-282, 1994), which<br>
disclosures are incorporated by reference herein as if fully set forth.   In the case of corn<br>
transformation, the preferred method is as described in either lshida et al. (Nat Biotechnol<br>
1996 Jun; 14(6): 745-50) or Frame et al. (Plant Physiol. 2002 May; 129(1): 13-22), which<br>
disclosures are incorporated by reference herein as if fully set forth.<br><br>
Generally after transformation, plant cells or cell groupings are selected for the presence of one or more markers which are encoded by plant-expressible genes co-transferred with the gene of interest, following which the transformed material is regenerated into a whole plant.<br>
Following DNA transfer and regeneration, putatively transformed plants may be evaluated, for instance using Southern analysis, for the presence of the gene of interest, copy number and/or genomic organisation. Alternatively or additionally, expression levels of the newly introduced DNA may be monitored using Northern and/or Western analysis, both techniques being well known to persons having ordinary skill in the ait<br>
The generated transformed plants may be propagated by a variety of means, such as by clonal propagation or classical breeding techniques. For example, a first generation (or T1) transformed plant may be selfed to give homozygous second generation (or T2) transformants, and the T2 plants further propagated through classical breeding techniques.<br>
The generated transformed organisms may take a variety of forms. For example, they may be chimeras of transformed cells and non-transformed cells; clonal transfonmants (e.g., all cells transformed to contain the expression cassette); grafts of transformed and untransformed tissues (e.g., in plants, a transformed rootstock grafted to an untransformed scion).<br>
The present invention dearly extends to any plant cell or plant produced by any of the methods described herein, and to all plant parts and propagules thereof. The present invention extends further to encompass the progeny of a primary transformed or transfected cell, tissue, organ or whole plant that has been produced by any of the aforementioned methods, the only requirement being that progeny exhibit the same genotypic and/or phenotypic characteristic(s) as those produced in the parent by the methods according to the invention. The invention also includes host cells containing an isolated nucleic acid molecule encoding a protein capable of modulating levels and/or activity of a NAP1-like protein, preferably wherein the protein is a NAP 1-like protein. Preferred host cells according to the invention are plant cells. Therefore, the invention also encompasses host cells or transgenic plants having altered growth characteristics, characterized in that the host ce or transgenic plant has modulated expression of a nucleic acid sequence encoding a NAP1-like protein and/or modulated activity and/or level of a NAP1-fike protein. Preferably, the altered growth characteristics comprise increased yield, more preferably increased seed yield.<br><br>
The invention also extends to harvestable parts of a plant such as, but not limited to, seeds, leaves, fruits, flowers, stems or stem cultures, rhizomes, roots, tubers and bulbs. The invention furthermore relates to products directly derived from a harvestable part of such a plant such as dry pellets or powders, oil, fat and fatty acids, starch or proteins.<br>
The term "plant" as used herein encompasses whole plants, ancestors and progeny of the<br>
plants and plant parts, plant cells, tissues and organs.   The term "plant" also therefore<br>
encompasses suspension cultures, embryos, meristematic regions, callus tissue, leaves,<br>
flowers,   fruits,   seeds,   roots  (including   rhizomes   and tubers),   shoots,   bulbs,   stems,<br>
gametophytes, sporophytes, pollen, and microspores. Plants that are particularly useful in the<br>
methods of the invention include algae, ferns, and all plants which belong to the superfamily<br>
Viridiplantae, in particular monocotyledonous and dicotyledonous plants, including fodder or<br>
forage legumes, ornamental plants, food crops, trees, or shrubs selected from the list<br>
comprising  Abelmoschus spp., Acer spp.,  Actinidia  spp., Agropyron spp.,  Aliium spp.,<br>
Amaranthus spp., Ananas comosus, Annona spp., Apium graveolens, Arabidopsis thaliana,<br>
Arachis spp, Artocarpus spp., Asparagus offidnalis, Avena saffva, Averrhoa carambola,<br>
Bonincasa hispida, Bertholletia excelsea, Beta vulgaris, Brassica spp., Cadaba farinosa,<br>
Camellia sinensis,  Canna indica,  Capsicum spp.,  Carica papaya,  Carissa macrocarpa,<br>
Carthamus tinctorius, Carya spp., Castanea spp., Cichorium endhria, Cinnamornum spp.f<br>
Citrullus lanatus, Citrus spp., Cocos spp., Coffea spp., Cola spp., Colocasia esculenta, Corylus<br>
spp., Crataegus spp., Cucumis spp., Cucurbita spp., Cynara spp., Dauous carota, Desmodium<br>
spp.,  Dimocarpus longan,  Dioscorea spp.,  Diospyros spp., Echinochloa spp.f Eleusine<br>
coracana, Eriobotrya japonica, Eugenia un'rflora, Fagopyrum spp., Fagus spp., Ficus carica,<br>
Fortunella spp., Fragaria spp., Ginkgo biloba, Glydne spp., Gossypium hirsutum, Helianthus<br>
spp., Hibiscus spp., Hordeum spp., Ipomoea batatas, Juglans spp., Lactuca sativa, Lathyrus<br>
spp., Lemna spp., Lens culinaris, Linum usitatissimum, Utchi chinensis, Lotus spp., Luffa<br>
acutangula, Lupinus spp., Macrotyioma spp., Malpighia emarginata, Malus spp., Mammea<br>
americana, Mangifera indica, Manihot spp., Manilkara zapota, Medicago sativa, Meiilotus spp.,<br>
Mentha spp., Momordica spp., Morus nigra, Musa spp., Nicotiana spp., Olea spp., Opuntia<br>
spp., Omithopus spp., Oryza spp., Panicum miliaceum, Passiflora edulis, Pastinaca sativa,<br>
Persea spp., Petroselinum crispum, Phaseolus spp., Phoenix spp., Physalis spp., Pinus spp.,<br>
Pistacia vera, Pisum spp., Poa spp., Populus spp., Prosopis spp., Pnmus spp., Psidium spp.,<br>
Punica granatum, Pyrus communis, Quercus spp., Raphanus sativus, Rheum rhabarbarum,<br>
Ribes spp., Rubus spp., Saccharum spp., Sambucus spp., Secale cereale, Sesamum spp.,<br>
Solarium spp., Sorghum bicolor, Spinacia spp., Syzygium spp., Tamarindus indica, Theobroma<br>
cacao, Trifblium spp., TriScosecale rimpaui, Trib'cum spp., Vaccinium spp., Vida spp., Vigna<br>
spp., Vitis spp., Zea mays, Zizania palustris, liziphus spp., amongst others.<br><br>
According to a preferred feature of the present invention, the plant is a crop plant comprising soybean, sunflower, canola, alfalfa, rapeseed or cotton. Further preferably, the plant according to the present invention is a monocotyledonous plant such as sugarcane, most preferably a cereal, such as rice, maize, wheat, millet, barley, oats, sorghum.<br>
The present invention also encompasses the use of nucleic acids encoding a NAP1-lfce protein, portions or variants thereof and the use of NAP1-like polypeptides, homologues or derivatives thereof.<br>
One such use relates to improving the growth characteristics of plants, in particular in improving yield, especially seed yield. The seed yield may include one or more of the following: increased number of seeds, increased number of filled seeds, increased total seed weight increased harvest index, increased thousand kennel weight, seed filling rate, among others. Preferably, the NAP1-like protein or the nucleic acid encoding a NAP1-like protein is of plant origin, more preferably from a dicotyledonous plant, furthermore preferably from the family of Brassicaceae, most preferably, the NAP1-like protein is encoded by SEQ ID NO:1 or is as represented by SEQ ID NO:2.<br>
NAP1~iike nucleic acids or variants thereof or NAPHike polypeptides or homologues thereo* may find use in breeding programmes in which a DNA marker is identified that may be genetically linked to a NAP1-likB gene or variant thereof. The NAP1-like gene or variants thereof or NAP1-like protein or homologues thereof may be used to define a molecular marker. This DNA or protein marker may then t &amp;ed in breeding programmes to select plants having improved growth characteristics. The NAP1-like gene or variant thereof may, for example, be a nucleic acid as represented by SEQ ID NO: 1, or a nucleic acid encoding any of the above mentioned homologues.<br>
Allelic variants of a gene encoding a NAP 1-like protein may also find use in marker-assisted breeding programmes. Such breeding programmes sometimes require introduction of allelic variation by mutagenic treatment of the plants, using for example EMS mutagenesis; alternatively, the programme may start with a collection of allelic variants of so called "natural" origin caused unintentionally. IdentificatL - of allelic variants then takes place by, for example, PCR. This is followed by a selection step for selection of superior allelic variants of the sequence in question and which give rise to improved growth characteristics in a plant, such as increased seed yield. Selection is typically carried out by monitoring growth performance of plants containing different allelic variants of the sequence in question, for example, different allelic variants of SEQ ID NO: 1, or of nucleic acids encoding any of the above mentioned plant<br><br>
homologues. Growth performance may be monitored in a greenhouse or in the field. Further optional steps include crossing plants, in which the superior alleiic variant resulting in increased seed yield was identified, with another plant. This could be used, for example, to make a combination of interesting phenotypic features.<br>
A NAPUIike nucleic acid or variant thereof may also be used as probes for genetically and physically mapping the genes that they are a part of, and as markers for traits linked to those genes. Such information may be useful in plant breeding in order to develop lines with desired phenotypes. Such use of NAP1-Iike nucleic acids or variants thereof requires only a nucleic add sequence of at least 10 nucleotides in length. The NAP1-like nucleic acids or variants thereof may be used as restriction fragment length polymorphism (RFLP) markers. Southern blots of restriction-digested plant genomic DNA may be probed with the NAP1-like nucleic acids or variants thereof. The resulting banding patterns may then be subjected to genetic analyses using computer programs such as MapMaker (Lander et al. (1987) Genomics 1, 174-181) in order to construct a genetic map. In addition, the nucleic acids may be used to probe Southern blots containing restriction endonuclease-treated genomic DNAs of a set of individuals representing parent and progeny of a defined genetic cross. Segregation of the DNA polymorphisms is noted and used to calculate the position of the NAP1-tike nucleic acid or variant thereof in the genetic map previously obtained using this population (Botstein et al. (1980) Am. J. Hum. Genet 32, 314-331).<br>
The production and use of plant gene-derived probes for use in genetic mapping is described in Bematzky and Tanksley (Plant Mol. Bid. Reporter 4, 37-41, 1986). Numerous publications describe genetic mapping of specific cDNA clones using the methodology outlined above or variations thereof. For example, F2 intercross populations, backcross populations, randomly mated populations, near isogenic lines, and other sets of individuals may be used for mapping. Such methodologies are well known to those skilled in the art<br>
The nudeic acid probes may also be used for physical mapping (i.e., placement of sequences on physical maps; see Hoheisel et al. In: Nonmammalian Genomic Analysis: A Practical Guide, Academic press 1996, pp. 319-346, and references cited therein).<br>
In another embodiment, the nucleic acid probes may be used in direct fluorescence in situ hybridization (FISH) mapping (Trask (1991) Trends Genet. 7, 149-154). Although current methods of FISH mapping favour use of large clones (several to several hundred kb; see taan et al. (1995) Genome Res. 5, 13-20), improvements in sensitivity may allow performance of FISH mapping using shorter probes.<br><br>
A variety of nucleic acid amplification-based methods of genetic and physical mapping may be carried out using the nucleic acids. Examples include allele-specific amplification (Kazazian (1989) J. Lab. Clin. Med. 11, 95-96), polymorphism of PCR-ampIified fragments (CAPS; Sheffield et al. (1993) Genomics 16, 325-332), allele-specrfic ligation (Landegren et a!. (1966) Science 241, 1077-1080), nucleotide extension reactions (Sokolov (1990) Nucleic Add Res. 18, 3671), Radiation Hybrid Mapping (Walter et al. (1997) Nat Genet. 7, 22-28) and Happy Mapping (Dear and Cook (1989) Nucleic Add Res. 17, 6795-6807). For these methods, the sequence of a nucleic add is used to design and produce primer pairs for use in the amplification reaction or in primer extension reactions. The design of such primers is well known to those skilled in the art In methods employing PCR-based genetic mapping, it may be necessary to identify DNA sequence differences between the parents of the mapping cross in the region corresponding to the instant nudeic acid sequence. This, however, is generally not necessary for mapping methods.<br>
In this way, generation, identification and/or isolation of modified plants with altered NAP1-Hke activity displaying improved growth characteristics can be performed.<br>
NAP1-like nudeic acids or variants thereof or NAP 1-1 ike polypeptides or homologues thereof may also find use as growth regulators. Since these molecules have been shown to be useful in improving the growth characteristics of plants, they would also be useful growth regulators, such as herbicides or growth stimulators. The present invention therefore provides a composition comprising a NAP1-4ike nudeic add or variant thereof or a NAP1-like polypeptide or homologue thereof, together with a suitable carrier, diluent or excipient, for use as a growth regulator, preferably as a growth promoter, more preferably for increasing yield.<br>
The methods according to the present invention result in plants having improved growth characteristics, as described hereinbefore. These advantageous growth characteristics may also be combined with other economically advantageous traits, such as further yield-enhancing traits, tolerance to various stresses, traits modifying various architectural features and/or biochemical and/or physiological features. Accordingly, the methods of the present invention may also be used in so-called "gene stacking" procedures.<br>
Description of figures<br>
The present invention will now be described with reference to the following figures in which:<br><br>
Fig. 1 Phylogenetic tree representing the relationships among NAP and SET proteins from yeast, man and plants. The tree was established by the AlignX program of VNTl Suite 5.5 (Infbrmax, http://www.informaxinc.com/). The matrix used to generate the multiple alignment is Blosum62 and the alingment parameters used were: Gap Opening penalty, 10; Gap Extension penalty, 0.5; Gap separation penalty range, 8; % identity for alignment delay, 40. The Phylogenetic tree is built using the Neighbor Joining method of Saitou and Nei. GenBank and MIPS (for Arabidopsis thaliana) accession numbers of the sequences used in the alignment are indicated in the tree- At Arabidopsis thaliana, Gm: Glycine max, Nt: Nicotiana tabacum (sequences derived from WO 03/085115), Os: Oryza saliva, Ps: Pisum sativum, Zm: Zea mays, Hs: Homo sapiens, Sc: Saccharomyces cerevisiae.<br>
Fig- 2 Schematic presentation of the entry clone p68, containing CDS0406 within the AttL1 and AttL2 sites for Gateway® cloning in the pDONR201 backbone. CDS0406 is the internal code for the Arabidopsis NAP1 -like coding sequence. This vector contains also a bacterial kanamycine-resistance cassette and a bacterial origin of replication.<br>
Fig. 3 Binary vector for the expression in Oryza sativa of the Arabidopsis NAP1-like gene (CDS0406) under the control of the Gos2 promoter (PRO0129). This vector contains a T-DNA derived from the Ti Plasmid, limited by a left border (LB repeat, LB Ti C58) and a right border (RB repeat, RB Ti C58)). From the left border to the right border, this T-DNA contains: a cassette for antibiotic selection of transformed plants; a selectable marker cassette for visual screening of transformed plants and the PRO0129 - CDS0406 -zein and rtcS-deltaGA double terminator cassette for expression of the Arabidopsis NAP1-like gene- This vector also contains an origin of replication from pBR322 for bacterial replication and a selectable marker (Spe/SmeR) for bacterial selection with spectinomycin and streptomycin.<br>
Fig. 4 The Medicago NAP1 -like protein was expressed in E. coli and purified from crude cell extract by affinity chromatography via the 6XHIS-tag, Elution of the 34 kD protein at different imidazol concentrations from the nickel-agarose resin is visualised by Western blotting using anti-6xHIS antibody (Sigma, St Louis, USA).<br>
Fig. 5 Alignment of the Arabidopsis thaliana NAP1-like protein (AtNAP1-like) and the Medicago sativa NAPI-like protein (MsNAPHike) using the Needleman and Wunsch algorithm. The gap opening penalty was set on 10, the gap extension penalty was 0.5. With these settings, the sequence identity was 71.2% and sequence homology 84.5%,<br><br>
Fig- 6 The NAP1-like protein has a nuclear localisation in plants. A) The Medicago NAP1-!ike protein has been shown to be localised in the nucleus of cultured alfalfa cells by indirect immunofluorescence using an antibody raised against the purified protein (left picture of panel A). To confirm the nuclear localisation, the nuclei were stained in parallel with the fluorescent dye DAPI, (right picture of panel A). In the insert the arrow points to a metaphase cell. A faint fluorescence indicates low abundance of the NAP1-like protein around the chromosomes in metaphase cells without a nuclear compartment. B) The transiently expressed Arabidopsis NAP1-like protein, fused to GFP, is localised to the nucleus in Arabidopsis cells following a PEG-mediated uptake of the gene construct into protoplasts.<br>
Fig. 7 The purified Medicago NAP1 -like protein inhibits in vitro phospho -histone H2B dephosphorylation activity of PP2A (purified from rabbit skeletal muscle), but has no influence on the dephosphorylation of the glycogen phosphorylase by the same enzyme.<br>
Fig. 8 Examples of sequences useful in performing the methods according to the present invention.<br>
Examples<br>
The present invention will now be described with reference to the following examples, which are by way of illustration alone.<br>
Unless otherwise stated, recombinant DNA techniques are performed according to standard protocols described in (Sambrook (2001) Molecular Cloning: a laboratory manual, 3rd Edition Cold Spring Harbor Laboratory Press, CSH, New York) or in Volumes 1 and 2 of Ausubel et al. (Current Protocols in Molecular Biology. New York: John Wiley and Sons, 1998). Standard materials and methods for plant molecular work are described in Plant Molecular Biology Labfase (1993) by R.D.D. Croy, published by BIOS Scientific Publications Ltd (UK) and Blackwell Scientific Publications (UK).<br>
Example 1: Gene Cloning<br>
The Arabidopsis NAP1-!ike (internal reference CDS0406) was amplified by PCR using as template an Arabidopsis thaliana seedling cDNA library (Inv'rtrogen, Paisley, UK). After reverse transcription of RNA extracted from seedlings, the cDNAs were cloned into pCMV Sport 6.0. Average insert size of the bank was 1.5 kb, and original number of clones was of 1.59x107 cfu. Original titer was determined to be 9.6x105 cfu/ml, after first amplification of 6x1011 cfu/ml. After plasmid extraction, 200 ng of template was used in a 50 pi PCR mix. Primers prm1505 (SEQ<br><br>
ID NO: 4) and prm1506 (SEQ ID NO: 5), which include the AttB sites for Gateway recombination, were used for PCR amplification. PCR was performed using Hifi Taq DNA polymerase in standard conditions. A PCR fragment of 771 bp was amplified and purified also using standard methods. The first step of the Gateway procedure, the BP reaction, was then performed, during which the PCR fragment recombines in vivo with the pDONR201 plasmid to produce, according to the Gateway terminology, an "entry clone", p68 (Figure 2). Plasmid pDONR201 was purchased from Invitrogen, as part of the Gateway® technology.<br>
Example 2: Vector Construction<br>
The entry clone p68 was subsequently used in an LR reaction with p0640, a destination vector used for Oryza sativa transformation. This vector contains as functional elements within the T-DNA borders: a plant selectable marker; a visual marker expression cassette; and a Gateway cassette intended for LR in vivo recombination with the sequence of interest already cloned in the entry clone. A GOS2 promoter for constitutive expression (PRO0129) is located upstream of this Gateway cassette. After the LR recombination step, the resulting expression vector p73 (Figure 3) can be transformed into the Agrobacterium strain LBA4404 and subsequently to Oryza sativa plants.<br>
Example 3: Rice Transformation<br>
Mature dry seeds of Oryza sativa japonica cultivar Nipponbare were dehusked. Sterilization was done by incubating the seeds for one minute in 70% ethanol, followed by 30 minutes in 0.2% HgCI2 and by 6 washes of 15 minutes with sterile distilled water. The sterile seeds were then germinated on a medium containing 2,4-D (callus induction medium). After a 4-week incubation in the dark, embryogenic, scutellum-derived call! were excised and propagated on the same medium. Two weeks later, the calli were multiplied or propagated by subculture on the same medium for another 2 weeks. 3 days before co-cultivation, embryogenic callus pieces were sub-cultured on fresh medium to boost cell division activity. The Agrobacterium strain LBA4404 harbouring the binary vector p73 was used for co-cultivation. The Agrobacterium strain was cultured for 3 days at 28°C on AB medium with the appropriate antibiotics. The bacteria were then collected and suspended in liquid co-cultivation medium at an ODBOO of about 1. The suspension was transferred to a petri dish and the calli were immersed in the suspension during 15 minutes. Next, the callus tissues were Wotted dry on a filter paper, transferred to solidified co-cultivation medium and incubated for 3 days in the dark at 25°C. Thereafter, co-cultivated callus was grown on 2,4-D-containing medium for 4 weeks in the dark at 28°C in the presence of a selective agent at a suitable concentration. During this<br><br>
period, rapidlv growing resistant callus islands developed. Upon transfer of this material to a regeneration medium and incubation in the light, the embryogenic potential was released and shoots developed in the next four to five weeks. Shoots were excised from the callus and incubated for 2 to 3 weeks on an auxin-containing medium from which they were transferred to soil. Hardened shoots were grown under high humidity and short days in a greenhouse. Finally seeds were harvested three to five months after transplanting. The method yielded single locus transformants at a rate of over 50 % (Aldemita and Hodges, Planta 199, 612-617, 1996; Chan et a!., Plant Mot. BioL 22(3), 491-506, 1993; Hiei et ctf., Plant J. 6(2), 271282, 1994).<br>
Example 4: Evaluation of Transformants: Vegetative growth measurements<br>
Approximately 15 to 20 independent TO transformants were generated. The primary transformants were transferred from tissue culture chambers to a greenhouse for growing and harvest of T1 seed. Five events of which the T1 progeny segregated 3:1 for presence/absence of the transgene were retained. For each of these events, 10 T1 seedlings containing the transgene (hetero- and homo-zygotes), and 10 T1 seedlings lacking the transgene (nullizygotes), were selected by visual marker screening. The selected T1 plants were transferred to a greenhouse. Each plant received a unique barcode label to link unambiguously the phenotyping data to the corresponding plant. The selected T1 plants were grown on soil in 10 cm diameter pots under the following environmental settings: photoperiod= 11.5 h, daylight intensity= 30,000 lux or more, daytime temperature= 28°C or higher, nighttime temperature= 22°C, relative humidity= 60-70%. Transgenic plants and the corresponding nullizygotes were grown side-by-side at random positions. From the stage of sowing until the stage of maturity the plants were passed several times through a digital imaging cabinet. At each time point digital images (2048x1536 pixels, 16 million colours) were taken of each plant from at least 6 different angles.<br>
The mature primary panicles were harvested, bagged, barcode-labelled and then dried for three days in the oven at 37°C. The panicles were then threshed and all the seeds collected. The filled husks were separated from the empty ones using an air-blowing device. After separation, both seed lots were then counted using a commercially avalable counting machine. The empty husks were discarded. The filled husks were weighed on an analytical balance and the cross-sectional area of the seeds was measured using digital imaging. This procedure resulted in the set of seed-related parameters described below.<br>
These parameters were derived in an automated way from the digital images using image analysis software and were analysed statistically. A two factor ANOVA (analyses of variance) corrected for the unbalanced design was used as statistical model for the overall evaluation of<br><br>
plant phenotypic characteristics. An F-test was carried out on all the parameters measured of all the plants of all the events transformed with that gene. The F-test was carried out to check for an effect of the gene over all the transformation events and to verify for an overall effect of the gene, also named herein "global gene effect". If the value of the F test shows that the data are significant, than it is concluded that there is a "gene" effect, meaning that not only presence or the position of the gene is causing the effect. The threshold for significance for a true global gene effect is set at 5%. probability level for the F test<br>
To check for an effect of the genes within an event, i.e., for a line-specific effect, a t-test was performed within each event using data sets from the transgenic plants and the corresponding null plants. "Null plants" or "Null segregants" or "Nullizygotes" are the plants treated in the same way as the transgenic plant, but from which the transgene has segregated. Null plants can also be described as the homozygous negative transformed plants. The threshold for significance for the t-test is set at 10% probability level. The results for some events can be above or below this threshold. This is based on the hypothesis that a gene might only have an effect in certain positions in the genome, and that the occurrence of this position-dependent effect is not uncommon. This land of gene effect is also named herein a "line effect of the gene". The p-value is obtained by comparing the t-value to the t-distribution or alternatively, by comparing the F-value to the F-distribution. The p-value then gives the probability that the null hypothesis (i.e., that there is no effect of the transgene) is correct.<br>
Vegetative growth and seed yield were measured according to the methods as described above. The inventors surprisingly found that the total weight of seeds, the number of filled seeds and the harvest index were increased in the rice plants transformed with the NAP 1-like gene when compared the control plants without the NAP1-like gene.<br>
The data obtained in the first experiment were confirmed in a second experiment with T2 plants. Four lines that had the correct expression pattern were selected for further analysis. Seed batches from the positive plants (both hetero- and homozygotes) in T1, were screened by monitoring marker expression. For each chosen event, the heterozygote seed batches were then retained for T2 evaluation. Within each seed batch an equal number of positive and negative plants were grown in the greenhouse for evaluation.<br>
A total number of 160 NAP1-tike transformed plants were evaluated in the T2 generation, that is 40 plants per event of which 20 positives for the transgene, and 20 negatives.<br><br>
Example   5:   Evaluation   of   transformants:   measurement   of  seed-related parameters<br>
Upon analysis of the seeds as described above, the inventors found that plants transformed with the NAP1-like gene construct had a higher number of filled seeds, a higher total weight of seeds and an increased harvest index compared to plants lacking the NAP1-like transgene. Positive results obtained for plants in the T1 generation were again obtained in the T2 generation. Not only individual transgenic lines scored significantly better than the corresponding nullizygous control lines, but there was also a significant positive overall effect when all plants of all tested T2 events were evaluated, strongly indicating a global gene effect. An overview of the data is given in Table 3.<br>
Table 3<br><br>
The % increase presents the average increase for all tested events. The p-value stands for the p-value derived from the F-test<br>
Number of filled seeds<br>
The number of filled seeds was determined by counting the number of filled husks that remained after the separation step. 4 of the 5 tested lines showed an increase in filled seed number, mounting up to 37%. There was an overall increase of 17% in the number of filled seeds produced by transgenic plants relative to corresponding null segregants, which increase is statistically significant (p-value 0-0352). In the T2 generation, there was increase for all tested lines, ranging between 14 and 46%. The mean increase for the T2 lines was 28%, this mean increase was also statistically significant (p-value of 0.0013). The combined analysis of T1 and T2 data also confirmed that the global gene effect was highly significant (p-value of 0-0003).<br>
Total seed yield<br>
The total seed yield (total weight of seeds) per plant was measured by weighing all filled husks harvested from a plant All transgenic T1 lines showed an increase in total seed weight, which varied between 8 and 43%. On average, the increase in seed yield was 24% and this overall effect from the presence of the transgene on seed yield was significant, as evidenced by a P-<br><br>
value for the F test of 0.0076. These results were also observed in the T2 generation. The 4 tested lines had a yield increase between 14 and 48% with an average of 28%, This mean increase was statistically significant (p-value of 0.0013) and also the combined analysis of the T1 and T2 plants showed there was a global gene effect (p-value of 0.0006).<br>
Harvest Index<br>
The harvest index in the present invention is defined as the ratio between the total seed yield and the above ground area (mm2), multiplied by a factor 106. 4 of the 5 tested lines showed an increased harvest index, ranging between 9 and 48%. There was a significant overall gene effect (an effect associated with of the presence of the transgene) on harvest index (an overall increase of 15%), with a statistically significant p-value for the F test of 0.085. Similar results were obtained for T2 plants. The harvest index for the individual lines was increased between 12 and 34% with a significant mean of 17% (p-value of 0.0025). Here too, the combined analysis of the T1 and T2 data showed a global gene effect (p-value 0.003).<br>
It is known to persons skilled in the art that the expression of transgenes in plants, and hence also the phenotypic effect due to expression of such transgene, can differ among different independently obtained transgenic lines and progeny thereof. The transgenes present in different independently obtained transgenic plants differ from each other by the chromosomal insertion locus as well as by the number of transgene copies inserted in that locus and the configuration of those transgene copies in that locus. Differences in expression levels can be ascribed to influence from the chromosomal context of the transgene (the so-called position effect) or from silencing mechanisms triggered by certain transgene configurations (e.g. inwards facing tandem insertions of transgenes are prone to silencing at the transcriptional or post-transcriptional level). Notwithstanding these possible causes of variation, the data show that transgenic plants expressing the NAP1-like gene consistently gave a higher number of filled seeds, a higher total weight of seeds, as well as an increased Harvest Index, each relative to corresponding non-transgenic plants. The observed increases were significant in both T1 and T2 generation, which is a strong argument for a global gene effect as evidenced by the p-values of the combined analysis.<br>
Example 6: Characterisation of a Medicago sativa NAP1-like protein:<br>
Materials and methods<br>
Isolation of the full-length cDNA clone of the putative alfalfa PP2A-inhibitor<br>
An isolated cDNA fragment coding for a part of an alfalfa (Medicago sativa) putative NAP1 -Ike<br>
protein has been used to isolate the fuINength clone from an alfalfa root-nodule A-ZAP phage<br><br>
cDNA library (Savoure et al. Plant Mol. Bid. 27, 1059-1070; 1995) using standard screening procedures as described by the manufacturer (Stratagene). 400 000 plaques were screened, 20 clones were retained, of which 18 were positive in the second hybridization screen. 8 of these clones were selected for further work and converted into phagemids from individual phages. Four clones were sequenced and two of them proved to be the full-length cDNA clones of the putative NAP1 -like protein. One of the clones (Ms10.1) was used for further work (SEQ ID NO: 10, encoding the protein of SEQ ID NO: 11).<br>
Production and purification of the Medicaao NAPHike protein<br>
The cDNA sequence coding for the Medicago sativa NAP1 -like protein was inserted into the NcoVXho\ site of the pENTRY4 GATEWAY*1 vector (Invitrogen) and subsequently introduced into the pDEST17 bacterial expression vector The pDEST17 vector allowed the expression of the NAP1-like protein in BL21 E. coli cells as a 6xHlS-tagged protein. The 34 kDa NAP1-like protein was purified by affinity chromatography using a nickel agarose resin (Sigma) (Figure<br>
4).<br>
Phosphatase activity measurements<br>
Potential phosphatase-inhibtting activity of the Medicago saliva NAP1-like protein was tested in vitro on Protein Phosphatase 2A (PP2A) catalytic subunits purified from rabbit skeletal muscle using ^P-isotope-labelled glycogen phosphorylase and histone H2A proteins as substrates according to Ulloa et al. (1993).<br>
Intracellular localization of the MsNAP1-like and AtNAP1-like proteins<br>
Polyclonal anti-MsNAP1-like antibodies were raised in rabbits against the purified 6xHIS-<br>
tagged protein using a standard immunization protocol.<br>
Protoplasts were isolated from suspension cultured alfalfa {Medicago saliva) cells and fixed by<br>
6% formaldehyde. The cells were than attached to poly-L-lysine coated glass slides and<br>
exposed to the anti-MsNAP1-like antiserum (200x diluted in PBS), washed and exposed to<br>
FITC-conjugated goat antinrabbit secondary antibody (SIGMA, 100x dilution). Nuclei were<br>
stained with DAPI (0,02 mg/ml) in parallel and photographed with a Nikon TE300 fluorescent<br>
microscope and a SPOT II colour CCD camera.<br>
The coding region of the Arabidopsis thaliana orthologue of the /Ifecf/cago sativa NAP1 -like<br>
protein (SEQ ID NO: 1), was inserted in frame with the green fluorescent protein (GFP) into the<br>
GATEWAY^-compatible plant expression vector (pK7WGF2). Protoplasts were isolated and<br>
transfected with the purified plasmid DNA using standard procedures. Transient expression<br>
was recorded one or two days after transfection by fluorescence microscopy.<br><br>
Results<br>
Arabidopsis and Medicaao NAP1-like proteins are localised In the nucleus<br>
The NAP1-like protein of Medicago sativa was highly homologous to Arabidopsis NAP1-like<br>
protein: the sequence identity was 71.2% and sequence similarity was 84.5% (Figure 5).<br>
Using the anti-MsNAP1-like antibodies,  indirect immunofluorescence revealed that the<br>
antibodies recognised a protein that was localized to the nuclei of suspension cultured alfalfa<br>
cells. This localisation was verified by the nuclear stain, DAPI. Faint fluorescence was<br>
associated with the chromosomes in metaphase cells (Figure 6A insert).<br>
The GFP-tagged Arabidopsis NAP1-Iikej&gt;rotein was also exclusively localised to the nuclei of<br>
suspension cultured Arabidopsis cells (Figure 6.B).<br>
2) The alfalfa NAPHIke protein inhibits in vitro PP2A phosphatase activity on a phospho-histone substrate<br>
Purified alfalfa NAP1-likej)fotein was added at various concentrations to reaction mixtures containing the catalytic subunits of rabbit skeletal muscle PP2A and phosphorylated histone H2A, or glycogen phosphorylase as substrate. It was observed that the NAP1-ltkej&gt;rotein had no influence on the dephosphorylation of the glycogen phosphorylase even at 500 mM concentration, but already 2,5 mM concentration of the NAPt-likej&gt;rotein efficiently inhibited PP2A activity on the phospho-histonH2A substrate (50% decrease in activity) (Figure 7).<br>
Conclusion<br>
The Medicago sativa and Arabidopsis thaliana NAP1 -like proteins show both structurally and functionally resemblance. Plant NAP 1-like.proteins inhibit in vitro phosphatase (PP2A) activity on histone substrates, indicating a possible in vivo ro le on chromatin organisation and gene transcription.<br>
Claims<br>
1.	Method for improving growth characteristics of a plant relative to corresponding wild type<br>
plants, comprising introducing a genetic modification in said plant and selecting for<br>
modulated expression in said plant of a nucleic acid sequence encoding a NAPl-like<br>
protein.<br>
2.	Method according to claim 1, wherein said modulated expression is increased expression.<br>
3.	Method according claim 1 or 2, wherein said genetic modification is effected by one of site-<br>
directed mutagenesis, homologous recombination, TILLING, T-DNA activation and DNA<br>
shuffling.<br>
4.	Method according to claim 1 or 2, wherein said genetic modification comprises introducing<br>
an isolated nucleic acid sequence encoding a NAP 1-like protein into a plant<br>
5.	Method according to claim 4, wherein said nucleic acid is derived from a plant, preferably<br>
from a dicotyledonous plant, more preferably from the family Brassicaceae, most preferably<br>
from Arabidopsis thaliana.<br>
6.	Method according to claim 4 or 5, wherein said nucleic acid sequence and said proteins<br>
include variants chosen from:<br>
(i)        a nucleic acid encoding a NAP1-like protein, wherein the NAPHike protein is<br>
preferably as represented by SEQ ID NO: 1 or encodes a NAP 1-like protein as<br>
represented by SEQ ID NO: 2; (ii)        an alternative splice variant of a nucleic acid sequence encoding a NAP1-like<br>
protein or wherein said NAP 1-1 ike protein is encoded by a splice variant; (iii)      an allelic variant of a nucleic acid sequence encoding a NAP 1-like protein or<br>
wherein said NAP 1-like protein is encoded by an allelic variant; (iv)      sequence capable of hybridising to  a  NAPHike encoding nucleic acid,<br>
preferably under stringent conditions; (v)       a NAP1-like protein;<br>
(vi)      a NAP1-like protein as represented by SEQ ID NO: 2; (vii)      homologues and derivatives of a NAP1-Iike protein, preferably of the NAPHike<br>
protein presented in SEQ ID NO:2,<br>
and wherein said NAPl-like protein comprises a NAP domain and an acidic C-terminal, and has PP2a phosphatase inhibiting activity.<br><br>
7.	Method according to any of claims 1 to 6, wherein said improved growth characteristics is<br>
increased yield, preferably increased seed yield.<br>
8.	Method according to claim 7, wherein said increased seed yield comprises one or more of<br>
increased number of filled seeds, increased total weight of seeds or increased Harvest<br>
index, each relative to corresponding wild type plants.<br>
9.	Method according to any of claims 4 to 8, wherein said nucleic acid encoding a NAP1 -like<br>
protein is operably linked to a constitutive promoter, preferably a G0S2 promoter.<br>
10.	Method for production of a transgenic plant having increased yield, which method<br>
comprises:<br>
(i)     introducing into a plant cell a nucleic acid sequence encoding a WAP 1-like<br>
protein; (ii)    regenerating and/or growing a plant from a transgenic plant cell.<br>
11.	Plants with increased yield obtainable by a method according to any of claims 1 to 10,<br>
including harvestable parts, propagules, seeds or progeny thereof, and including products<br>
directly derived thereof.<br>
12.	Gene construct for expression in a plant, comprising:<br>
(i)     a nucleic acid sequence encoding a NAP1-like protein;<br>
(ii)    one or more control sequences capable of driving expression in a plant of the<br>
nucleic add sequence of (t); and optionally (iii)   a transcription termination sequence.<br>
13.	Construct according to claim 12, wherein said nucleic acid encoding a NAPHike protein is<br>
chosen from the group comprising:<br>
(i)     a nucleic acid sequence represented by SEQ ID NO: 1 or the complement strand<br>
thereof; (ii)    a nucleic acid sequence encoding an amino acid sequence represented by SEQ<br>
ID NO: 2 or homologues, derivatives or active fragments thereof; (iii)    a nucleic acid sequence capable of hybridising (preferably under stringent<br>
conditions) with a nucleic add sequence of (i) or (ii) above, which hybridising<br>
sequence preferably encodes a protein having NAP1-Iike protein activity; (iv)   a nucleic add sequence according to (i) to (iii) above which is degenerate as a<br>
results of the genetic code;<br><br>
(v)    nucleic acid sequence which is an allelic variant to the nucleic acid sequences<br>
according to (i) to (iii); (vi)    nucleic acid sequence which is an alternative splice variant to the nucleic acid<br>
sequences according to (i) to (iii),<br>
and wherein said NAPI-like protein comprises a NAP domain and an acidic C-terminal, and has PP2a phosphatase inhibiting activity.<br>
14.	Construct according to claim 12 or 13, wherein said control sequences comprise at least a<br>
constitutive promoter, preferably a GOS2 promoter.<br>
15.	Construct according to any of claims 12 to 14, wherein said nucleic acid sequence<br>
encoding a NAP 1-like protein is oriented in sense direction relative to said control<br>
sequence.<br>
16.	Construct comprising an expression cassette essentially similar to SEQ ID NO: 3.<br>
17.	Transgenic plant comprising an isolated nucleic acid sequence encoding a NAPI-like<br>
protein, characterized in that said plant has been selected for having increased yield.<br>
18.	Transgenic plant of claim 17, selected for increased expression of a nucleic acid encoding<br>
a NAP1-like protein.<br>
19.	Transgenic plant of claim 17 or 18, wherein said isolated nucleic acid encoding a NAPHike<br>
protein encodes a protein as represented in SEQ (0 N0:2<br>
20.	Transgenic plant according to any of claims 17 to 19, wherein said plant is a crop plant<br>
comprising soybean, sunflower, canola, alfalfa, rapeseed or cotton, preferably, the plant is<br>
a monocotyledonous plant such as sugarcane, most preferably a cereal, such as rice,<br>
maize, wheat, millet, barley, oats, sorghum.<br>
21.	Transgenic plant cells, transgenic plants or transgenic plant parts, including harvestable<br>
parts, propagules, seeds or transgenic progeny, and products directly derived thereof, of a<br>
plant according to any of claims 17 to 20.<br>
22.	Use of a nucleic acid sequence encoding a NAP 1-like protein, or a portion thereof or a<br>
sequence hybridising therewith, for increasing yield of a plant<br><br>
23.	Use of a NAPI-like protein, and/or homologues, derivatives or active fragments thereof for<br>
increasing yield of a plant<br>
24.	Use of claim 22 or 23, wherein said increased yield is increased seed yield.<br>
25.	Use of claim 24, wherein said increased seed yield comprises one or more of total weight<br>
of seeds, number of filled seeds and harvest index.<br>
26.	Use of any of claims 22 to 25, wherein the sequence of said nucleic acid encoding a NAP1 -<br>
like protein or of the NAP1-iike protein itself is as represented in SEQ ID NO: 1 or 2.<br>
27.	A composition for increasing yield of a plant comprising a NAP1-like protein and/or a<br>
nucleic acid encoding a NAP1-like protein.<br>
28.	A composition for increasing yield of plants comprising a protein represented by SEQ ID<br>
NO: 2 or a homologue, derivative or active fragment thereof.<br>
29.	A composition for increasing yield of plants comprising a nucleic acid as represented by<br>
SEQ ID NO: 1 or a portion thereof or a sequence hybridising therewith, or encoding a<br>
protein as represented in SEQ ID NO:2.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjMtMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   AMENDED  PAGES OF SPECIFICATION  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBDTEFJTVMgIDIzLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   AMENDED CLAIMS  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgQU1FTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAyNy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   AMENDED PAGES OF SPECIFICATION  27-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMTctMDgtMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   CORRESPONDENCE OTHERS   17-08-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDA5LTA0LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   CORRESPONDENCE OTHERS 09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgRk9STS0zICAwOS0wNC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   FORM-3  09-04-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgRk9STS0zICAyMy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   FORM-3  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyMy0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   OTHER PATENT DOCUMENT  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIzLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   POWER OF ATTORNEY  23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICAgUE9XRVIgT0YgQVRUT1JORVkgIDI3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006   POWER OF ATTORNEY  27-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgIDI3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  CORRESPONDENCE OTHERS  27-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  CORRESPONDENCE OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBQTy5wZGY=" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  CORRESPONDENCE PO.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgMjMtMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  EXAMINATION REPORT REPLY RECEIVED 23-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBGT1JNLTEgIDI3LTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  FORM-1  27-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1DSEVOUC0yMDA2ICBGT1JNLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">3624-CHENP-2006  FORM-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYyNC1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3624-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252152-a-pneumatic-flotation-column-for-treatment-of-mineral-resources.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252154-a-system-and-a-method-of-rough-road-detection-for-an-engine.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252153</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3624/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>18/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>04-May-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>30-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>28-Sep-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CROPDESIGN N.V.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGIEPARK 3, B-9052 ZWIJNAARDE, BELGIUM</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>FEHER, ATTILA</td>
											<td>JACINT UTCA 29, H-6771 SZEGED, HUNGARY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>FRANKARD, VALERIE</td>
											<td>BIERENBERG 47, B-1640 SINT-GENESIUS-RODE, BELGIUM</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DUDITS, DENES</td>
											<td>BERKERT UTCA 36/3, H-6726 SZEGED, HUNGARY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K14/415</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/51403</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-03-25</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>04101388.9</td>
									<td>2004-04-02</td>
								    <td>EUROPEAN UNION</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/563,847</td>
									<td>2004-04-20</td>
								    <td>EUROPEAN UNION</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252153-a-method-of-making-plant-with-increased-seed-yield by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:31:37 GMT -->
</html>
