<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252104-a-method-of-identifying-a-target-protein-specific-translational-fusion-partner-tfp by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:33:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252104:&quot;A METHOD OF IDENTIFYING A TARGET PROTEIN SPECIFIC TRANSLATIONAL FUSION PARTNER (TFP)&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A METHOD OF IDENTIFYING A TARGET PROTEIN SPECIFIC TRANSLATIONAL FUSION PARTNER (TFP)&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of identifying a target protein specific translational fusion partner (TFP), said method comprising: (i) co-transforming a plurality of reporter protein deficient host cells with (1)a plurality of linear vectors and (2) a nucleic acid encoding a target protein to produce a plurality of transformed host cells, wherein each of said linear vectors comprises a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terminal amino acid-deleted reporter protein, and wherein said nucleic acid encoding a target protein comprises, at the 3&#x27; end, a nucleotide sequence encoding the N-terminal amino acids deleted from said reporter protein in said linear vector, and at the 5&#x27; end, a linker DNA, and wherein said target protein is a protein that is difficult to produce using conventional recombinant production methods.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>LIBRARY OF TRANSLATIQNAL FUSION PARTNERS FOR<br>
PRODUCING RECOMBINANT PROTEINS AND TRANSLATIONAL<br>
FUSION PARTNERS SCREENED THEREFROM<br>
BACKGROUND OF THE INVENTION<br>
Field of the Invention<br>
[0001]	This invention is in the field of recombinant protein expression.   In<br>
particular, the invention relates to techniques for the rapid screening of suitable translational fusion partners (TFPs) capable of inducing secretory production of recombinant proteins, especially proteins that are difficult to produce using conventional recombinant production methods.<br>
Related Art<br>
[0002]	The recombinant expression of proteins of interest is a widely used;<br>
procedure to produce large quantities of proteins for research purposes or for therapeutic and other commercial uses. A variety of recombinant expression systems are known in the art, including bacterial, yeast, and mammalian host cell systems, and many different proteins have been successfully produced in these systems. However, there are also many proteins that are not easily produced using available expression systems, resulting in little or no protein expression and secretion. Methods for improving the secretion of recombinantly expressed proteins, such as overexpressing secretory factors in the host cells,, using fusion proteins comprising the protein of interest fused to a well-secreted protein, and adding synthetic linker sequences, have had some success with particular proteins of interest. However, no general technique has been identified that is effective for the secretory production of all proteins.<br>
[0003]	In an effort to identify secreted proteins and novel signal sequences,<br>
several signal sequence trap systems have been developed. U.S. Patent No. 6,228,590 describes a technique for screening for mammalian signal sequences by transforming reporter protein-deficient yeast with nucleic acids comprising mammalian coding sequences fused to a reporter protein and detecting cells that secrete the reporter protein. A similar system using<br>
invertase-deficient yeast and an invertase reporter protein is disclosed'in EP0907727. Yeast-based signal sequence traps have been used to identify secreted proteins from human DNA (Klein et al, Proc. Natl. Acad. Sci. USA 95:7108 (1996); Jacobs et al, Gene 198:289 (1997)), mouse DNA (Gallicioti et al, J. Membrane Biol 183:175 (2001)), zebrafish DNA (Crosier et al., Dev. Dynamics 222:637 (2001)), Arabidopsis DNA (Goo et al, Plant Mol Biol 41A15 (1999)), potato DNA (Surpili et al, Anais de Academia Brasileira de Ciencias 74:599 (2002)), and Candida albicans DNA (Monteoh'va et al, Eukaryotic Cell 1:514 (2002)). Similar trap systems have been developed using mammalian host cells (Gallicioti et al, J. Membrane Biol 183:175 (2001)) and bacterial host cells (Ferguson et al., Cancer Res. (55:8209 (2000). •'••, Reporter proteins that have been used in signal sequence traps include . • invertase (Klein et al, Proc. Natl Acad. Sci. USA P3:7108 (1996)), alpha.--' amylase (U.S. Patent No. 6,228,590), acid phosphatase (PHQ5) (Surpili et al., Anais de Academia Brasileira de Ciencias 74:599 (2002)), and j3-lactamase Ferguson etal, Cancer Res. 65:8209(2000).<br>
[0004]	A method for identifying translational fusion partners (TFPs) useful for<br>
secretion of a target protein is disclosed in WO 2005/068658.. The method, comprises (i) obtaining a plurality of host cells transformed with a variety of vectors comprising a library of nucleic acid fragments and a target protein-encoding nucleotide sequence fused with a reporter protein-encoding nucleotide sequence, wherein the host cells are deficient in the reporter protein, and (ii) identifying a TFP library from the host cells, wherein the TFP library comprises nucleic acid fragments which individually induce the secretion of the target protein.<br>
SUMMARY OF THE INVENTION<br>
[0005]	The present invention relates to  a rapid and efficient automatic<br>
screening method for the identification of TFPs that are effective for inducing secretion of a target protein. The invention allows any target protein to be<br>
secreted from a host cell, including target proteins that are not expressed or<br>
expressed only at low levels using traditional recombinant expression systems.<br>
[0006]	In one embodiment, the invention relates to a method of identifying a<br>
target protein specific TFP, said method comprising:<br>
(i) co-transforming a plurality of reporter protein deficient host cells with a plurality of linear vectors and a nucleotide sequence encoding a target protein to produce a plurality of transformed host cells,<br>
wherein each of said linear vectors comprises a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terminal amino acid-deleted reporter protein, and<br>
wherein said micleotide sequence encoding a target protein comprises, . at the 3' end, a nucleotide sequence encoding the N-terminal amiho acids-. deleted from said reporter protein in said linear vector, and at the 5' end, a.- linker DNA;<br>
(ii) incubating said plurality of transformed host cells under conditions  effective. to allow in vivo recombination of said linear vectors and said nucleotide. sequence encoding a target protein;<br>
(iii) identifying a cell showing an activity of the reporter protein from    -5 the plurality of transformed host cells of (ii); and<br>
(iv) identifying a TFP from the cell identified in (iii);<br>
wherein said TFP comprises a nucleic acid fragment which induces the secretion of said target protein,<br>
[0007]	Another  embodiment  of the  invention  relates  to  a method of<br>
identifying a target protein specific TFP library, said method comprising:<br>
(i) co-transforming a plurality of reporter protein deficient host cells with a plurality of linear vectors and a nucleotide sequence encoding a target protein to produce a plurality of transformed host cells,<br>
wherein each of said linear vectors comprises a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terminal amino acid-deleted reporter protein, and<br>
wherein said nucleotide sequence encoding a target protein comprises,<br>
at the 3' end, a nucleotide sequence encoding the N-terminal amino acids<br>
deleted from said reporter protein in said linear vector, and at the 5' end, a<br>
linker DNA;<br>
(ii) incubating said plurality of transformed host cells under conditions<br>
effective to allow in vivo recombination of said linear vectors and said<br>
nucleotide sequence encoding a target protein;.<br>
(iii) identifying cells showing an activity of the reporter protein from<br>
the plurality of transformed host cells of (ii); and<br>
(iv) identifying a TFP library from the cells identified in (iii);<br>
wherein said TFP library comprises nucleic acid fragments which individually<br>
induce the secretion of said target protein.<br>
[0008]	The invention further relates to a TFP or a library of TFPs identified by^<br>
the methods of the invention.<br>
[0009]	The invention further comprises a nucleic acid fragment encoding a<br>
TFP or a library of nucleic acid fragments encoding TFPs.<br>
[0010]	The invention also includes a nucleic acid comprising a nucleotide<br>
sequence encoding a TFP and a nucleotide sequence encoding a target protein.,.<br>
[0011]	The invention further relates to a method of producing, a: target protein<br>
using a TFP of the invention.<br>
[0012]	The invention additionally relates to a linear vector comprising a<br>
nucleic acid fragment from a library of nucleic acid fragments and a<br>
nucleotide sequence encoding a N-terminal amino  acid-deleted reporter<br>
protein.<br>
[0013]	The invention also comprises a plurality of reporter protein-deficient <br>
host cells transformed with the library of linear vectors and a nucleotide<br>
sequence encoding a target protein of the invention.<br>
BRIEF DESCRIPTION OF THE DRAWINGS/FIGURES<br>
[0014]	The above and other objects, features and advantages of the present<br>
invention will be more clearly understood from the following detailed<br>
description taken in conjunction with the accompanying drawings.<br>
[0015]	FIG. 1 shows a process for deleting the invertase gene and &amp; pop-out<br>
process of a selectable marker.<br>
[0016]	FIG. 2 shows zymogram analysis for invertase activity (lanes 1,2 and<br>
3: wild-type Saccharomyces cerevisiae Y2805; and lanes 4, 5 and 61. invertase-<br>
deficient strain (S. cerevisiae Y2805Aswc2).<br>
[0017]	FIG. 3 photographically shows the growth of yeast cells according to<br>
carbon sources (SUC2: wild-type S. cerevisiae Y2805; and &amp;suc2\ invertase-<br>
deficient strain (S. cerevisiae Y28Q5Asuc2).<br>
[0018]	FIG. 4 shows the results of Southern blotting for the deletion of the<br>
invertase gene (lanes 1 and 2: S. cerevisiae Y2805 (ura3 SUC2)\ lanes 3 and<br>
4: S. cerevisiae ~Y2&amp;Q5Asuc2U(URA3&amp;suc2'); and lanes 5 and 6: S. cerevisiae<br>
Y28Q5&amp;SUC2 (ura3ksuc2).<br>
[0019]	FIG. 5 photographically shows the growth of yeast cells containing;<br>
plasmids pYGAP-SNS-SUC2, pYGAP-HSA-SUC2, and pYGAP-hIL2-SUC2,<br>
on glucose and sucrose media, respectively.<br>
[0020]	FIG. 6 shows a map of plasmid YGalNV containing multiple cloning<br>
sites for the insertion of a cDNA library between the GALIO promoter and the<br>
mature invertase gene.<br>
[0021]	FIG; 7 shows a map of plasmids YGaFQINV, YGaFlINV and<br>
YGaF21NV containing multiple cloning sites for the insertion of a genomic<br>
DNA library between the GALIO promoter and the mature invertase gene with<br>
three different reading frames.<br>
[0022]	FIG. 8 shows the process of synthesis of a cDNA library with random<br>
primer and construction of a cDNA library in the TFP selection • vector<br>
YGalNV.<br>
[0023]	FIG. 9 shows the process of construction of a genomic DNA library in<br>
the TFP selection vectors YGaFOINV, YGaFlINV, and YGaF2INV.<br>
[0024]	FIG. 10 shows the plasmid map of YGadV45 containing a defective<br>
SUC2 and subcloning of a TFP library into YGadV45.<br>
[0025]	FIG. 11 shows the procedure of TFP selection for a target gene using<br>
an invertase as a reporter from a TFP library through in vivo recombination.<br>
[0026]	FIG. 12 shows the procedure of TFP selection for a target gene using a<br>
double reporter, lipase and invertase, as a reporter from a TFP library through in vivo recombination.<br>
[0027]	FIG. 13 shows tributyrin plates containing halo forming transfonnants<br>
(A) A halo forming plate (YPSGA with tributyrin) directly from  transformation, (B) Selected transfonnants showing different halo sizes in-,-tributyrin plate.<br>
[0028]	FIG. 14 shows the procedure for the construction of 9 human IL2<br>
expression vectors with 9 selected TFPs.	<br>
[0029]	FIG. 15 shows the maps of human IL2 expression vectors (A) pYGT9-<br>
IL2, (B) pYGT13-IL2, and (C) pYGT17-IL2.<br>
[0030]	FIG.  16 shows the maps of human IL2 expression  vectors (A);<br>
pYGTl 8-IL2, (B) pYGT19-IL2, and (C) pYGT20-IL2.   ..<br>
[0031]	FIG.  17 shows the maps of human IL2 expression vectors (A)<br>
pYGT21-IL2, (B) pYGT25-IL2, and (C) pYGT27-IL2.<br>
[0032]	FIG. 18 shows the results of SDS-PAGE of culture supernatants of<br>
yeast cells secreting human IL2 (lane M: protein size marker; lane 1: culture supernatant of yeast cells containing pYIL-KRTl-4 (WO 2005/068658) as a control for IL2 secretion; lane 2: culture supernatant of yeast cells containing  pYGT9-IL2; lane 3: culture supernatant of yeast cells containing pYGT21-EL2; lane 4: culture supernatant of yeast cells containing pYGT13-IL2; lane 5: culture supernatant of yeast cells containing pYGT17-IL2; lane 6: culture supernatant of yeast cells containing pYGT25-IL2; lane 7: culture supernatant of yeast cells containing pYGT19-EL2; lane 8: culture supernatant of yeast<br>
cells containing pYGT18-EL2; lane 9: culture supernatant of yeast cells containing pYGT27-IL2).<br>
[0033]	FIG. 19 shows the result of SDS-PAGE of culture supematants of 38<br>
yeast transformants obtained from the TFP selection process for human IL32cc (lane M: protein size marker; lane N: untransformea' cell as a negative control; lane 1 to 38: yeast transformants).<br>
[0034]	FIG, 20 shows the results of SDS-PAGE and Western blotting of<br>
culture supematants of yeast cells secreting human IL32a (lane M: protein size marker; lane 1: culture supernatant of yeast cells containing pYGT3-IL32a; lane 2: culture supernatant of yeast cells containing pYGT21-IL32a; lane 3: culture supernatant of yeast cells containing pYGT13-IL32a; lane 4: culture supernatant of yeast cells containing pYGT25-EL32oc; lane 5: culture. . supernatant of yeast cells containing pYGT22-IL32a and lane 6: culture supernatant of yeast cells containing pYGTl l-IL32a).<br>
[0035]	FIG.   21   shows  (A)   a profile  for  fed-batch  fermentation  of a<br>
recombinant yeast strain containing pYGT3-bIL32a and (B) the results of SDS-PAGE for analyzing proteins secreted into the medium according to fermentation time.<br>
[0036]	FIG. 22 shows the results of SDS-PAGE of culture supematants of<br>
yeast cells secreting human growth hormone (lane M: protein size marker; lane N: culture supernatant of untransformed yeast cells as a negative control; lane 1: culture supernatant of yeast cells containing pYGTl-hGH, lane 2: pYGT2-hGH; lane 3: pYGT3-hGH; lane 4: pYGT4-hGH; lane 5: pYGTS-hGH; lane 6: pYGT6-hGH; lane 7: pYGT7-hGH; lane 8: pYGT8-hGH; lane 9: pYGT9-hGH; lane 10: pYGT21-hGH; lane 11: pYGT13-hGH; lane 12: pYGT25-hGH; lane 13: pYGT17-hGH; lane 14: pYGT22-hGH; lane 15: pYGT32-hGH; lane 16: pYGT19-hGH; lane 17: pYGT27-nGH; lane 18: pYGTll-hGH; lane 19: pYGT40-hGH; lane 20: pYGT43-hGH; lane 21: pYGT44-hGH.<br>
[0037]	FIG.   23   shows  (A)  a profile  for  fed-batch  fermentation of a<br>
recombinant yeast strain containing pYGT18-hGH and (B) the results of SDS-PAGE for analyzing proteins secreted into the medium according to fermentation time.<br>
[0038]	FIG. 24 shows a procedure for the construction of a TFP library from<br>
selected ORFs using a unidirectional deletion method.<br>
[0039]	FIG. 25 shows the results of SDS-PAGE of culture supernatants of<br>
randomly selected yeast transformants transformed with the unidirectional-deleted TFP library constructed from the ORFs selected by BLAST search.<br>
[0040]	FIG. 26 shows the results of SDS-PAGE of culture supernatants of<br>
randomly selected yeast transformants transformed with the unidirectional- -.. deletion TFP library constructed from 35 selected ORFs.<br>
[0041]	FIG. 27 shows the results of SDS-PAGE and Western blotting (anti-,;<br>
hIGF) of culture supernatants of yeast cells secreting human insulin-like growth factor (Lane M; protein size marker; lane 1: culture supernatant of yeast cells containing pYGa-MFa-hlGF; lane 2: pYGa-Tlcc-IGF; lane 3: pYGa-T2a-IGF; lane 4: pYGa-T3oc-IGF; lane 5: pYGa-T4oc-IGF). .<br>
[0042]	FIG- 28 shows the results of SDS-PAGE of culture supernatants of<br>
yeast cells transformed with TFP vectors for the secretion of human caspase-1 • subunit P10 (lane M: protein size marker; lane 1: culture supernatant of yeast cells with pYGTI-hPIO; lane 2: pYGT2-hP10; lane 3: pYGT3-hP10; lane 4: pYGT4-hP10; lane 5: pYGT5-hP10; lane 6: pYGT6-hP10; lane 7: pYGT7-hPlO; lane 8: pYGTS-hPlQ; lane 9: pYGT9-hP10; lane 10: pYGT21-hP10; lane 11: pYGT13-hP10; lane 12: pYGT25-hP10; lane 13: pYGT17-hP10; lane 16: PYGT22-hP10; lane 18: pYGT18-hP10; lane 19: pYGT33-hP10; lane 20: pYGT19-hP10; lane 21: pYGT27-hP10; lane 22: pYGTll-hPlQ; lane 24: pYGT39-hP10; lane 25: pYGT40-hP10; lane 28: pYGT43-hP10; lane 29: pYGT44-hP10; lane 32: negative control).<br>
[0043]	FIG. 29 shows the results of SDS-PAGE and Western blotting (anti-<br>
IL32) of culture supernatants of yeast cells secreting human interleukin 32 gamma (lane M: protein size marker; lane C: culture supernatant of<br>
untransformed yeast cells as a negative control; lane 1: pYGTl-IL32y; lane 2: pYGT2-IL32y; lane 3: pYGT3-IL32y; lane 4: pYGT4-IL32y; lane 5: pYGTS-IL32y; lane 6: pYGT6-IL32y; lane 7: pYGT7- IL32y; lane 8: pYGT8-IL32y; lane 9: pYGT9-IL32y; lane 10: pYGT21-IL32y; lane 11: pYGT13-IL32y; lane 12: pYGT25-IL32y; lane 13: pYGT17-IL32y; lane 16: pYGT22-IL32y; lane 18: pYGT18-IL32y; lane 19: pYGT33-IL32y; lane 20: pYGT19-IL32y; lane 21: pYGT27-IL32y; lane 22: pYGTll-IL32y; lane 24: pYGT39-IL32y; lane 25: pYGT40-JL32y; lane 28: pYGT43-IL32y; lane 29: pYGT44-JCL32y; lane 33: pYGT48-IL32y; lane 35: pYGT50-IL32y; lane 36: pYGT51-IL32y; lane 37: pYGT52-EL32y; lane 39: pYGT54-IL32y).<br>
[0044]	FIG. 30 shows the results of SDS-PAGE of culture supernatants of<br>
yeast cells secreting human interleukin-2(lane M: protein size marker; lane 1: culture supernatant of yeast cells containing 'YGaSW-pSUN-IL2; lane 2: YGaSW-pSED-IL2; lane 3: YGaSW-pUNK-IL2; lane 4: YGaSW-pMUC-IL2).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0045J	The present invention addresses the need for a rapid and efficient<br>
screening technique for identification of a TFP specifically applicable to a target protein for maximal secretion of the target protein. While the invention is useful to optimize the recombinant expression of any protein, it is particularly useful to enable the production of proteins that cannot be produced on a large scale and/or at low cost due to their low level of expression in known expression systems.<br>
[0046]	In one embodiment, the invention relates to a method of identifying a<br>
target protein specific TFP, said method comprising:<br>
(i) co-transforming a plurality of reporter protein deficient host cells with a plurality of linear vectors and a nucleotide sequence encoding a target protein to produce a plurality of transformed host cells,<br>
wherein each of said linear vectors comprises a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terminal amino acid-deleted reporter protein, and<br>
wherein said nucleotide sequence encoding a target protein comprises, at the 3' end, a nucleotide sequence encoding the N-terminal amino acids deleted from said reporter protein in said linear vector, and at the 5' end, a linker DNA;<br>
(ii) incubating said plurality of transformed host cells under conditions effective to allow in vivo recombination of said linear vectors and said nucleotide sequence encoding a target protein;<br>
(iii) identifying a cell showing an activity of the reporter protein from the plurality of transformed host cells of (ii); and<br>
(iv) identifying a TFP from the cell identified in (iii);<br>
wherein said TFP comprises a nucleic acid fragment which induces the secretion of said target protein.<br>
[0047]	Another   embodiment   of the  invention  relates   to   a  method  of<br>
identifying a target protein specific TFP library, said method comprising:<br>
(i) co-transforming a plurality of reporter protein deficient host cells with a plurality of linear vectors and a nucleotide sequence encoding a target protein to produce a plurality of transformed host cells,<br>
wherein each of said linear vectors comprises a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terminal amino acid-deleted reporter protein, and<br>
wherein said nucleotide sequence encoding a target protein comprises, at the 3' end, a nucleotide sequence encoding the N-terminal amino acids deleted from said reporter protein in said linear vector, and at the 5' end, a linker DNA;<br>
(ii) incubating said plurality of transformed host cells under conditions effective to allow in vivo recombination of said linear vectors and said nucleotide sequence encoding a target protein;<br>
(iii) identifying cells showing an activity of the reporter protein from the plurality of transformed host cells of (ii); and<br>
(iv) identifying a TFP library ,from the cells identified in (iii); wherein said TFP library comprises nucleic acid fragments which individually induce the secretion of said target protein.<br>
|0048]	The library of nucleic acid fragments may be obtained from DNA of<br>
any type, including genomic DNA, cDNA, synthetic DNA, and recombinant DNA. Nucleic acids other than DNA may also be used, including RNA and non-naturally occurring nucleic acids.<br>
[0049]	TFPs may be identified from the DNA  of any eukaryotic - or<br>
prokaryotic organism, including bacteria, fungi (e.g., yeast), plants, and' animals (e.g., mammals).   Suitable bacteria include, but are not limited tot ,-Escherichia and Bacillus species. Suitable yeast include, but are not limited to.,', Candida,       Debaryomyces,       Hansenula,       Kluyveromyces,       Pichia, Schizosaccharomyces,   Yarrovria,   Saccharomyces,   Schwanniomyces,   and Arxula species. Examples of specific species include Candida utilis, Candida boidinii, Candida albicans, Kluyveromyces lactis, Pichia pastoris, Pichia stipitis, Schizosaccharomyces .pombe, Saccharomyces cerevisiae, Hansenula polymorpha, Yarrowia lipolytica, Schwanniomyces occidentalis, and Arxula, adeninivorans.  Other fungi that may serve as a source of DNA include, but are not limited to Aspergillus, Penicillium, Rhizopus,  and Trichodema species. Plants that may serve as a source of DNA include, but are not limited to Arabidopsis, maize, tobacco, and potato. Suitable animals include, but are not limited to humans, mice, rats, rabbits, dogs, cats, and monkeys.<br>
[0050]	The nucleic acid fragments may be derived from the entire genome of<br>
an organism, e.g., an entire genomic or cDNA library. The fragments may also be derived from any subset of the entire genome, e.g., a subtracted library or a sized library.<br>
[0051]	hi one embodiment, the nucleic acid fragments are derived from a<br>
library of pre-selected candidate TFPs, e.g., a library comprising TFPs that have been identified in previous screens, hi a particular embodiment, the<br>
library of pre-selected candidate TFPs is a library of core TFPs that have been identified as effective TFPs for one or more target proteins.<br>
[0052]	In another embodiment, the library of pre-selected candidate TFPs is<br>
obtained by transforming a plurality of reporter protein-deficient host cells with a variety of vectors comprising a library of nucleic acid fragments and a reporter protein-encoding nucleic acid sequence, collecting cells that grow, isolating vectors from the cells, and isolating nucleic acid fragments from the vectors, thereby obtaining a TFP library comprising the nucleic acid fragments which individually induce secretion of the reporter protein.<br>
(0053]	In a further embodiment, the library of pre-selected candidate TFPs is .<br>
derived from sequences identified in a genome database by searching for (i) .( genes containing a pre-secretion signal homologous with those of one or more:-  previously identified TFPs; (ii) genes comprising a secretion signal sequence;1;<br>
or (iii) genes encoding proteins passing through endoplasmic reticulum (e.g., cell wall proteins, excretory proteins, plasma membrane proteins, vacuolar proteins, bud proteins).<br>
[0054]	In another embodiment, the library of pre-selected candidate TFPs is.<br>
obtained by diversifying previously identified TFPs, e.g., by unidirectional ;&gt; .,. deletion, mutation, addition of functional sequences (e.g., glycosylation sites) or swapping of pre- and pro-signal sequences between TFPs.<br>
[0055]	hi one embodiment, the nucleic acid fragments have a size of less than<br>
1000 base pairs, e.g., less than 700, 500, or 300 base pairs. In a further embodiment, the library of nucleic acid fragments is constructed by enzymatic cleavage of the DNA, by cDNA synthesis, or by recombinant DNA technology (e.g., unidirectional deletion, mutagenesis).<br>
[0056]	The linear vectors of the present invention may be any vector that is<br>
functional in the selected host cell. As used herein, the term "vector" refers to a nucleic acid molecule capable of transporting another nucleic acid to which it has been linked. One type of vector is a "plasmid", which refers to a circular double stranded DNA loop into which additional DNA segments can be ligated. Another type of vector is 'a viral vector, wherein additional DNA<br>
segments can be ligated into the viral genome.  Certain vectors are capable of autonomous replication in a host cell into which they are introduced (e.g., bacterial vectors having a bacterial origin of replication and episomal mammalian vectors).   Other vectors (e.g., non-episornal mammalian vectors) are integrated into the genome of a host cell upon introduction into the host cell, and thereby are replicated along with the host genome. The vectors of the present invention are capable of directing the expression of genes encoding target proteins to which they are operatively linked. Such vectors are referred to herein as "expression vectors".  In general, expression vectors of utility in recombinant DNA techniques are often in the form of plasmids. In the present . specification, "plasmid" and "vector" can be used interchangeably as the plasmid is the most commonly used form of vector. However, the invention is:--'.-. , intended to include such other forms of expression vectors, such as virafc . vectors (e.g., replication defective retroviruses, adenoviruses and adeno-associated viruses), that serve equivalent functions.<br>
[0057]	Expression of proteins in prokaryotes may be carried out with vectors<br>
containing constitutive or inducible promoters directing the expression of the target protein-reporter protein fusion. Examples of suitable E. coli expression • vectors include pTrc (Amrann et al, Gene 69:301-315 (198'8)) and pET   • (Studier et al, GENE EXPRESSION TECHNOLOGY:  METHODS IN ENZYMOLOGY 185, Academic Press, San Diego, Calif. (1990) 60-89).<br>
[0058]	For expression in yeast cells, suitable yeast expression vectors include,<br>
but are not limited to pYepSecl (Baldari et al, EMBO J. 5:229-234 (1987)), pMFa (Kurjan et al., Cell 30:933-943 (1982)), pJRY88 (Schultz et al, Gene 54:113-123 (1987)), pYES2 (mvitrogen Corporation, San Diego, Calif.), and picZ (Invitrogen Corp, San Diego, CaL).<br>
[0059]	For expression in insect cells, baculovirus expression vectors may be<br>
used. Baculovirus vectors available for expression of proteins in cultured insect cells (e.g., SF9 cells) include the pAc series (Smith et al, Mb/.. Cell Biol. 3:2156-2165 (1983)) and the pVL series (Lucklow et al, Virology ./70:31-39 (1989)).<br>
 [0060]	In another embodiment, the host cells are mammalian cells and the<br>
vector is a mammalian expression vector. Examples of mammalian expression vectors include pCDMS (Seed, Nature 329:840 (1987)) and pMT2PC (Kaufrnan et al, EMBO J. 6: 187-195 (1987)). When used in mammalian cells, the expression vector's control functions are often'provided by viral regulatory elements. For example, commonly used promoters are derived from polyoma, adenovirus 2, cytomegalovirus and Simian Virus 40. For other suitable expression systems for both prokaryotic and eukaryotic cells, see, e.g., Chapters 16 and 17 of Sambrook et al, MOLECULAR CLONING: A LABORATORY MANUAL. 2nd ed., Cold Spring Harbor -Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989.<br>
[0061]	Preferred vectors include, but are not limited to, plasmids, phages'j ;••<br>
cosmids, episomes, viral particles or viruses, and integratable DNA fragments (i.e., fragments integratable into the host genome by homologous recombination). Preferred viral particles include, but are not limited to, adenoviruses, baculo viruses, parvoviruses, herpesviruses, poxviruses,--adenoassociated viruses,-, Semliki Forest viruses, vaccinia, .viruses, and retroviruses. Preferred expression vectors include, but are -not limited to, pcDNAS (Invitrogen) and pSVL (Pharmacia Biotech). Other expression vectors include, but are not limited to, pSPORTTM vectors, pGEMTM vectors (Promega), pPROEXvectorsTM (LTI, Bethesda, MD), BluescriptTM vectors (Stratagene), pQETM vectors (Qiagen), pSE420TM (Invitrogen), and pYES2TM (Invitrogen).<br>
[0062]	In one embodiment, expression vectors are replicable DNA constructs<br>
in which a DNA sequence encoding the target protein is operably linked or connected to suitable control sequences capable of effecting the expression of the target protein in a suitable host. DNA regions are operably linked or connected when they are functionally related to each other. For example, a promoter is operably linked or connected to a coding sequence if it controls the transcription of the sequence. Amplification vectors do not require<br>
expression control domains, but rather need only the ability to replicate in a host, usually conferred by an origin of replication, and a selection gene to facilitate recognition of transformants. The need for control sequences in the expression vector will vary depending upon the host selected and the transformation method chosen. Generally, control sequences include, but are not limited to a transcriptional promoter, enhancers, an optional operator sequence to control transcription, polyadenylation signals, a sequence encoding suitable mRNA ribosomal binding and sequences which control the termination of transcription and translation. Such regulatory sequences are described, for example, in Goeddel; GENE EXPRESSION TECHNOLOGY: METHODS IN ENZYMOLQGY 185, Academic Press, San Diego, Calif. (1990). Regulatory sequences include those that direct constitutive expression,. of a nucleotide sequence in many types of host cell and those that direct5', expression of the nucleotide sequence only in certain host cells (e.g., tissue-specific regulatory sequences). It will be appreciated by those skilled in the art that the design of the expression vector can depend on such factors as the choice of the host cell to be transformed, the level of expression of protein1 desired, etc.<br>
[0063]	The expression vectors of the invention can be introduced into host<br>
cells to thereby produce proteins or peptides, including fusion proteins or peptides, encoded by nucleic acids as described herein. Preferred vectors preferably contain a promoter that is recognized by the host organism. The promoter sequences of the present invention maybe prokaryotic, eukaryotic or viral. Examples of suitable prokaryotic sequences include the PR and PL promoters of bacteriophage lambda (The bacteriophage Lambda, Hershey, A. D., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY (1973), which is. incorporated herein by reference in its entirety; Lambda n, Hendrix, R. W., Ed., Cold Spring Harbor Press, Cold Spring Harbor, NY (1980), which is incorporated herein by reference in its entirety); the trp, recA, heat shock, and lacZ promoters of E. colt and the SV40 early promoter (Benoist et al, Nature; 2903Q4-310 (1981)), which is incorporated herein by reference in its entirety).<br>
For yeast, examples of suitable promoters include, but are not limited to GAPDH, PGK, ADH, PH05, GAL1, and GAL10. Additional promoters include, but axe not limited to, mouse mammary tumor virus, long terminal repeat of human immunodeficiency virus, maloney virus, cytomegalovirus immediate early promoter, Epstein Barr virus, Rous sarcoma virus, human actin, human myosin, human hemoglobin, human muscle creatine, and human metallothionein.<br>
[0064]	Additional regulatory sequences can also be included in preferred<br>
vectors.   Examples of suitable regulatory sequences are represented by the Shine-Dalgarno sequence of the replicase gene of the phage MS-2 and of the • gene ell of bacteriophage lambda.<br>
[0065]	Moreover, suitable expression vectors can include an appropriate/<br>
marker that allows  the screening of the transformed host  cells.     The-, transformation of me selected host is carried out using any one of the various techniques well known to the expert in the art and described in Sambrook et al., supra.<br>
 [0066]	An origin of replication can also be provided either by construction of<br>
the vector to include an exogenous origin or may be provided hyi-the host cell chromosomal replication mechanism. If the vector is integrated into the host •/ • cell chromosome, the latter may be sufficient. Alternatively, rather than using vectors which contain viral origins of replication, one skilled in the art can transform mammalian cells by the method of co-transformation with a selectable marker and target protein DNA. An example of a suitable marker is dihydrofolate reductase (DHFR) or thymidine kinase (see, U.S. Patent No. 4,399,216).<br>
[0067]	Nucleotide sequences encoding the target protein may be recombined<br>
with vector DNA in accordance with conventional techniques, including blunt-ended or staggered-ended termini for ligation, restriction enzyme digestion to provide appropriate termini, filling in of cohesive ends as appropriate, alkaline phosphatase treatment to avoid undesirable joining, and ligation with appropriate ligases. Techniques for such manipulation are disclosed by<br>
Sambrook et al, supra and are well known in the art. Methods for construction of mammalian expression vectors are disclosed in, for example,' Okayama et al, Mol Cell Biol 5:280 (1983), Cosman et al., Mol. Immunol. 25:935 (1986), Cosman et al, Nature 572:768 (1984), EP-A-0367566, and WO 91/18982, each of which is incorporated herein by reference in its entirety.<br>
[0068]	The host cells used in the present invention may be any host cells<br>
know to those of skill in the art.  Suitable host cells include bacterial, fungal, (e.g., yeast), plant, or animal (e.g., mammalian or insect) cells. Suitable yeast cells include Candida, Debaryomyces, Hansenula, Kluyveromyces, Pichia, Schizosaccharomyces,   Yarroma,   Saccharomyces,   Schwanniomyces,   and , Arxula species.  Specific examples include Candida utilis, Candida boidiniv,: Candida albicans, Kluyveromyces lactis, Pichia pastoris, Pichia stipitis;.  Schizosaccharomyces    pombe,     Saccharomyces    cerevisiae,    Hansenula polymorpha, Yarrowia lipolytica, Schwanniomyces occidentalis, and Arxula adeninivorans.      Other   suitable   fungi   include  Aspergillus,   Penicillium, Rhizopus, and Trichoderma species.  Bacteria that may be used as host cells', include Escherichia, Pseudomonas, and Bacillus species.  Suitable plant host  cells include Arabidopsis, maize, tobacco, and potato.   Animal, cells include cells from humans, mice, rats, rabbits, dogs, cats, monkeys, and insects. Examples include CHO, COS 1, COS 7, BSC 1, BSC 40, BMT 10, and Sf9 cells.<br>
[0069]	hi a particular embodiment, the host cells are yeast cells, and the<br>
nucleic acid fragments are isolated from the genome or cDNA of a yeast.<br>
[0070]	Polynucleotides of the invention may be introduced into the host cell<br>
as part of a circular plasmid, or as linear DNA comprising an isolated protein coding region or a viral vector.  Methods for introducing DNA into the host cell  that  are  well  known  and routinely practiced in the   art  include   . transformation, transfection, electroporatkm, nuclear injection, or fusion with carriers such as liposomes, micelles, ghost cells, and protoplasts.<br>
[0071]	Any reporter protein that is rapidly and efficiently detectable may be<br>
used in the present invention. In one embodiment, the reporter protein has an activity that can be positively selected for in order to automate the screening process. In an additional embodiment, the reporter protein is a protein that is secreted into the extracellular space,  e.g., invertase,  sucrase, cellulase, xylanase,   maltase,   amylase,   glucoamylase,   galactosidase   (e.g.,   alpha-galactosidase beta-galactosidase, melibiase), phosphatase (e.g., PH05), beta-lactamase,' lipase or protease. In a particular embodiment, the secreted protein permits a cell to grow on a particular substrate.   As an example of reporter system in mammalian cell, CD2/neoniycin-phosphotransferase (Ceo) gene can • be used as a secretion reporter in the media containing antibiotics G418 to trap , the secretion pathway genes in mouse embryonic stem cells  (De-Zolt et ah,,. . Nucleic Acid Res. 34:e25 (2006)).<br>
'[0072]	In one embodiment, the host cells are yeast, the reporter protein is<br>
invertase and the transformed yeast cells are selected for their ability to grow on sucrose or raffmose. In another embodiment, the host cells are yeast, the reporter protein is melibiase and the transformed yeast cells are selected for/ their ability to grow on melibiose. _In a further embodiment, the.iiost cells are yeast, the reporter protein is amylase (e.g., an endoamylase,.,exoamylase, |3-&gt;. amylase, or glucoamylase), the yeast cells are non-amylolytic, and the transformed cells are screened for their ability to degrade starch. In an additional embodiment, the step of identifying cells showing an activity of the reporter protein occurs by using a reporter protein which provides resistance to a growth inhibitor, e.g., an antibiotic. In another embodiment, the reporter protein is a protein that can be detected visually, e.g., green fluorescent protein or luciferase. In one embodiment, the step of identifying cells showing an activity of the reporter protein occurs by using two or more reporter proteins, e.g., lipase and invertase.<br>
[0073]	The host cells of the present invention do not exhibit reporter protein<br>
activity.    In one embodiment, the host cells naturally do not express the reporter protein.   In other embodiments, the gene(s) encoding the reporter<br>
protein have been deleted in whole or in part or have been mutated such that the reporter protein" is not expressed or is expressed in an inactive form. Methods for rendering a cell deficient in a particular protein are well known in the art and any such method may be used to prepare the host cells of the present invention (Sambrook et al, supra). For yeast, a reporter gene deficiency can be introduced using well known gene replacement techniques (Rothstein, Metk Enzymol 194:281 (1991)).<br>
[0074]	The linear vector of the invention comprises a nucleic acid fragment<br>
and a nucleotide sequence encoding a N-terniinal amino acid-deleted reporter protein.  The N-terminal amino acid deletion may encompass any number of amino acids as long as the deletion is sufficient to substantially eliminate reporter protein activity.  For example, the deletion may encompass about 5,', 10, 15, 20, 25, 30, 35, 40, 45, or 50 or more amino acids from the N-terminus.i' of the reporter protein.<br>
[0075]	The methods of the present invention may be used with any target<br>
protein for which there is a desire for high level recombinant expression. The target protein may be one that is being studied for research purposes or one. that is being produced for commercial purposes, e.g., therapeutic or industrial use. The target protein may be from any plant, animal, or microorganism, and may be naturally occurring or modified in any way, as long as it can be encoded by a nucleic acid. In one embodiment the target protein is a human protein. In another embodiment, the target protein is a cytokine, serum protein, colony stimulating factor, growth factor, hormone, or enzyme. For example, the target protein may be selected from an interleukin, coagulation factor, interferon-a, -|3 or -y, granulocyte-colony stimulating factor, granulocyte macrophage-colony stimulating factor, tissue growth factor, epithelial growth factor, TGFcc, TGFp, epidermal growth factor, platelet-derived growth factor, fibroblast growth factor, follicle stimulating hormone, thyroid stimulating hormone, antidiuretic hormone, pigmentary hormone, parathyroid hormone, luteinizing hormone-releasing hormone, carbohydrate-specific enzymes, proteolytic enzymes, lipases, oxidoreductases, transferases,<br>
hydrolases, lyases, isomerases, ligases, immuno globulins, cytokine receptors, lactoferrin, phospholipase A2-activating protein,  insulin, tumor necrosis factor, calcitonin, calcitonin gene related peptide, enkephalin, somatomedin, erythropoietin,    hypothalamic    releasing    factor,    prolactin,    chorionic gonadotropin,   tissue   plasminogen   activator,   growth  hormone' releasing peptide, thymic humoral factor, anticancer peptides, or antibiotic peptides. Specific examples include, but are not limited to human uiterleukin-2, human interleukin-lp,     human     interleukin-6,     human     mterleukin-32a, 32(3 or -32y, Factor VH, Factor VET, Factor DC, human serum albumin, human interferon-ct, -J3 or -y, human granulocyte-colony stimulating factor, human • granulocyte macrophage-colony stimulating factor, human growth hormone, . human platelet-derived growth factor, human basic fibroblast growth factor,'1 human epidermal growth factor, human insulin-like growth factor, human-nerve growth factor, human transforming growth factor (3-1, human follicle stimulating     hormone,     glucose     oxidase,     glucodase,     galactosidase,1 glucocerebrosidase,     glucuronidase,     asparaginase,     arginase,     argfnfne deaminase, peroxide dismutase, endotoxinase, catalase, chymotrypsin, uricase, adenosrne diphosphatase, tyrosinase, bilirubin oxidase, bovine galactose-1- , phosphate uridyltransferase, jellyfish green fluorescent protein,  Candida antarctica lipase B,  Candida rugosa lipase, fungal chloroperoxidase, P-galactosidase, resolvase, a-galactosidase, (3-glucosidase, trehalose synthase, cyclodextrin  glycosyl  transferase,  xylanase,  phytase,  human lactoferrin, human erythropoietin, human paraoxonase, human growth differentiation factor 15, human galectin-3 binding protein, human serine protease inhibitor, Kunitz type 2, human Janus kinase 2, human mis-like tyrosine kinase 3 ligand, human YM1 &amp; 2, human CEMI, human diacylglycerol acyltransferase, human leptin, human mL259, human proteinase 3, human lysozyme, human DEAD box protein 41, human etoposide .induced protein 24, mouse caspasel, bovine angiogenin, and earthworm lumbrokinase.<br>
[0076]	In one embodiment, the target protein is a protein that is difficult to<br>
produce using conventional recombinant production methods, that is, a protein<br>
that is not produced at all or is only produced at low levels. In another embodiment, the target protein is one that is readily produced using known expression systems, but for which there is a desire to achieve higher levels of expression.<br>
[0077]	Nucleic acids encoding a target protein may be obtained from any<br>
source using routine techniques well known in the art, including isolation from a genomic or cDNA library, amplification by PCR, or chemical synthesis.<br>
[0078]	The nucleotide sequence encoding a target protein used in the methods<br>
of the present invention comprises at the 5' end a linker DNA that is used for in vivo recombination with the linear vectors of the invention and further comprises at the 3' end a nucleotids sequence encoding a portion of the N-terminus of the reporter protein, including the N-terminal ammo acids deleted.;."'. in the linear vector and sufficient additional amino acids to allow in vivo:-' recombination between the nucleotide sequence encoding a target protein and the linear vector when they are co-transformed into the host cell.   In ons embodiment, the sequence encoding a portion of the N-terminus of the reporter protein comprises at least 20 base pairs that overlap with the reporter; protein-encoding sequence of the linear vector, e.g., at least 30 or 40 base pairs. The addition of the 5' linker and the 3' reporter protein sequence to the nucleotide sequence encoding a target protein may be carried out using routine recombinant DNA techniques, e.g., PCR and/or restriction enzyme cleavage and ligation.<br>
[0079]	The linker DNA of the invention must be of sufficient length and have<br>
sufficient sequence identity to a portion of the nucleotide sequence of the linear vector to allow in vivo recombination between the target protein-encoding nucleotide sequence and the linear vector when they are co-transformed into a host cell. In one embodiment, the linker DNA is more than 20 base pairs in length, e.g., more than 30 or 40 base pairs in length. In a further embodiment, the linker DNA is at least 80% identical to the corresponding sequence on the linear vector, e.g., at least 85%, 90%, 95%, or 99% identical.<br>
[0080]	In one embodiment, the linker DNA encodes a protease recognition<br>
sequence thereby allowing cleavage at the junction of the TFP and the target protein.  For example, the linker DNA may encode a yeast kex2p- or Kex2-like protease recognition sequence (e.g., an amino acid sequence comprising Lys-Arg, Arg-Arg, or Leu-Asp-Lys-Arg (SEQ ID N0:214)), a mammalian furin-recognition sequence (e.g., an amino acid sequence comprising Arg-X-X-Arg), a factor Xa-recognition sequence (e.g., an amino acid sequence comprising Ile-Glu-Gly-Arg (SEQ ID NQ:215)), an enterokinase-recogm'tion sequence   (e.g.,   an  amino   acid  sequence  comprising  Asp-Asp-Lys),   a subtilisin-recognition sequence (e.g., an amino acid sequence comprising Ala- .• Ala-His-Tyr (SEQ ID NO:216)), a tobacco etch virus protease-recognition sequence (e.g., an amino acid sequence comprising Glu-Asn-Leu-Tyr-Phe'- • Gln-Gly (SEQ ID NO:217)), a ubiquitin hydrolase-recognition sequence (e.g.,.. an amino acid sequence comprising Arg-Gly-Gly) or a thrombin-recognition sequence (e.g., an amino acid sequence comprising Arg-Gly-Pro-Arg (SEQ ID NO:218)).<br>
[0081]	In another embodiment, the linker DNA encodes an affinity tag, e.gv;.<br>
GST, MBP, NusA, thioredoxin, ubiquitin, FLAG, BAP, 6HIS, STREP, CBP, . CBD, or S-tag.<br>
[0082]	In a further embodiment, the linker DNA encodes a restriction enzyme<br>
recognition site, e.g., a Sfil site, hi another embodiment, the linker DNA encodes a restriction enzyme recognition site and a protease recognition sequence (e.g., kex2p-like protease- or kex-2p-recognition sequence).<br>
[0083]	The present invention relates to a TFP identified by the methods of the<br>
invention or a derivative or fragment thereof. In one embodiment, the TFP is selected from the group consisting of TFP-9 (SEQ ID NO:29), TFP-13 (SEQ ID NO:31), TFP-17 (SEQ ID NQ:33), TFP-18 (SEQ ID N0:35), TFP-19 (SEQ ID N0:37), TFP-20 (SEQ ID NO:39), TFP-21 (SEQ ID N0:41), TFP-25 (SEQ ID NO:43), TFP-27 (SEQ ID N0:45), TFP-11 (SEQ ID NO:61), TFP-22 (SEQ ID NO:63), TFP-29 (SEQ ID NO:65), TFP-34 (SEQ ID N0:67), TFP-38 (SEQ ID N0:69), TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ<br>
ID NO-.131), TFP-44 (SEQ ID NO: 133), TFP-48 (SEQ ID NO:135), TFP-52 (SEQ ID NO:137), TFP-54 (SEQ ID N0:139), TFP-40 (SEQ ID NO: 175), TFP-50 (SEQ ID NO: 177), TFP-51 (SEQ ID NO:179), TFP-57 (SEQ ID NO:181), TFP-58 (SEQ ID N0:183), TFP-59 (SEQ ID N0:185), TFP-5 (SEQ ID NO:200), TFP-6 (SEQ ID NO:202), TFP-7 (SEQ ID N0:204), TFP-8 (SEQ ID N0:206), PpTFP-1 (SEQ ID N0:84), PpTFP-2 (SEQ ID N0:86), PpTFP-3 (SEQ ID NO:88), and PpTFP-4 (SEQ ID N0:90) or a derivative or fragment thereof.<br>
[0084]	The invention further relates to a TFP library comprising two or more<br>
of the TFPs identified by the methods of the invention or a fragment or derivative thereof. In one embodiment, the TFP library comprises TFPs identified as effective for a particular target protein. In another embodiment^., the TFP library comprises TFPs identified as effective for more than one target.1-. protein. In a particular embodiment, the TFP library comprises two or more (e.g., 4, 6, 8, 10, or 12 or more) TFPs selected from the group consisting of TFP-9 (SEQ ID NO:29);;.TFP-13 (SEQ ID N0:31), TFP-17 (SEQ ID NO:33), TFP-18 (SEQ ID NO:3:5), TFP-19 (SEQ ID N0:37), TFP-20 (SEQ ID NO:39), TFP-21 (SEQ ID N0:41), TFP-25 (SEQ ID NO:43),. TFP-27 (SEQ . ID NO:45), TFP-11 (SEQ ID NO:61), TFP-22 (SEQ ID NO:63), TFP-29 ' (SEQ ID NO:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ID N0:69), TFP-39 (SEQ ID NO:129), TFP-43 (SEQ ID NO:131), TFP-44 (SEQ ID NO: 133), TFP-4S (SEQ ID NO:135), TFP-52 (SEQ ID NO:137), TFP-54 (SEQ ID NO: 139), TFP-40 (SEQ ID N0:175), TFP-50 (SEQ ID NO: 177), TFP-51 (SEQ ID NO: 179), TFP-57 (SEQ ID NO: 181), TFP-58 (SEQ ID NO: 183), TFP-59 (SEQ ID NO: 185), TFP-5 (SEQ ID N0:200), TFP-6 (SEQ ID N0:202), TFP-7 (SEQ ID N0:204), TFP-8 (SEQ ID N0:206), PpTFP-1 (SEQ ID NO:84), PpTFP-2 (SEQ ID NO:86), PpTFP-3 (SEQ ID NO:88), and PpTFP-4 (SEQ ID NO:90) or a derivative or fragment thereof.<br>
[0085]	In a further embodiment, the TFP library comprises six or more (e.g.,<br>
8, 10, 12, or 15 or more) TFPs selected from the group consisting of TFP-9 (SEQ ID N0:29), TFP-13 (SEQ ID N0:31), TFP-17 (SEQ ID NO:33), TFP-<br>
18 (SEQ ID NO:35), TFP-19 (SEQ ID N0:37), TFP-20 (SEQ ID N0:39), TFP-21 (SEQ ID N0:41), TFP-25 (SEQ ID N0:43)5 TFP-27 (SEQ ID N0:45), TFP-11 (SEQ ID N0:61), TFP-22 (SEQ ID NO:63), TFP-29 (SEQ ID N0:65), TFP-34 (SEQ ID N0:67), TFP-38 (SEQ ID NO:69), TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ ID NO: 131), TFP-44 (SEQ ID NO: 133), TFP-48 (SEQ ID NO:135), TFP-52 (SEQ ID N0:137), TFP-54 (SEQ ID NO:139), TFP-40 (SEQ ID NO:175)5 TFP-50 (SEQ ID NO:177), TFP-51 (SEQ ID NO:179), TFP-57 (SEQ ID NO:181), TFP-58 (SEQ ID N0:183)5 TFP-59 (SEQ ID NO: 185), TFP-5 (SEQ ED NO:200), TFP-6 (SEQ ID . N0:202), TFP-7 (SEQ ID N0:204), TFP-8 (SEQ ID NO:206), PpTFP-1 (SEQ ID NO:84), PpTFP-2 (SEQ ID NO:86), PpTFP-3 (SEQ ID NO:88), PpTFP-4 (SEQ ID NO:90), TFP-1 (SEQ ID NO:219), TFP-2 (SEQ ID NO:221), TFP-3; 
[0086]	The present invention further relates to a nucleic acid encoding a TFP<br>
identified by the methods of the invention or a derivative or fragment thereof. In one embodiment, the nucleic acid encodes a TFP selected from the group, consisting of TFP-9 (SEQ ID NO:29), TFP-13 (SEQ ID N0:31), TFP-17 •'., (SEQ ID NO:33), TFP-18 (SEQ ID NO:35), TFP-19 (SEQ D&gt;NO:37), TFP- V 20 (SEQ ID NO:39), TFP-21 (SEQ ID NO:41), TFP-25 (SEQ ID N0:43), TFP-27 (SEQ ID NO:45), TFP-11 (SEQ ID NO:61), TFP-22 (SEQ ID N0:63), TFP-29 (SEQ ID NO:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ID N0:69), TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ ID NO:131), TFP-44 (SEQ ID NO: 133), TFP-48 (SEQ ID N0:135), TFP-52 (SEQ ID NO: 137), TFP-54 (SEQ ID NO:139), TFP-40 (SEQ ID NO:175), TFP-50 (SEQ ID N0:177), TFP-51 (SEQ ID NO: 179), TFP-57 (SEQ ID NO:181), TFP-58 (SEQ ID NO: 183), TFP-59 (SEQ ID NO: 185), TFP-5 (SEQ ID N0:200), TFP-6 (SEQ ID N0:202), TFP-7 (SEQ ED N0:204), TFP-8 (SEQ ED N0:206), PpTFP-1 (SEQ ID NO-.84), PpTFP-2 (SEQ ED NO:86), PpTFP-3 (SEQ ID N0:88), and PpTFP-4 (SEQ ED NO:90) or a derivative or fragment thereof. In one embodiment, the nucleic acid comprises a nucleotide sequence selected<br>
from the group consisting of SEQ ID NOS:30, 32, 34, 36, 38, 40, 42, 44, 46, 62, 64, 66, 68, 70, 85, 87, 89, 91, 130,132, 134, 136, 138, 140, 176, 178, 180, 182,184, 186, 201, 203,205, or 207 or a derivative or fragment thereof.<br>
[0087]	The invention further relates to a library of nucleic acids encoding two<br>
or more TFPs identified by the methods of the invention or a derivative or fragment thereof. In one embodiment, the library of nucleic acids encodes TFPs identified as effective for a particular target protein. In another embodiment, the library of nucleic acids encodes TFPs identified as effective for more than one target protein. In a particular embodiment, the library of nucleic acids encodes two or more (e.g., 4, 6, 8, 10, or 12 or more) TFPs selected from the group consisting of TFP-9 (SEQ ID NO:29), TFP-13 (SEQ ID N0:31), TFP-17 (SEQ ID NO:33), TFP-18 (SEQ ID N0:35), TFP-19T' " (SEQ ID N0:37), TFP-20 (SEQ ID N0:39), TFP-21 (SEQ ID N0:41), TFP*-25 (SEQ ID N0:43), TFP-27 (SEQ ID NO:45), TFP-11 (SEQ ID N0:61), TFP-22 (SEQ ID NO:63), TFP-29 (SEQ ID NO:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ID NO:69), TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ ID NO: 131), TFP-44 (SEQ ID NO: 133), TFP-48 (SEQ ID NO:135), TFP-52 (SEQ ID N0:137), TFP-54 (SEQ ID NO:139), TFP-40 (SEQ ID NO: 175), / TFP-50 (SEQ ID NO: 177), TFP-51 (SEQ ID NO: 179), TFP.-57 (SEQ EK -NO:181), TFP-58 (SEQ ID NO:183), TFP-59 (SEQ ID NO:185), TFP-5 (SEQ ID N0:200), TFP-6 (SEQ ED NO:202), TFP-7 (SEQ ID NO:204), TFP-8 (SEQ ID NO:206), PpTFP-1 (SEQ ID NO:84), PpTFP-2 (SEQ ID NO:86), PpTFP-3 (SEQ ID NO:88), and PpTFP-4 (SEQ ID N0:90) or a derivative or fragment thereof. In one embodiment, the library of nucleic acids comprises two or more (e.g., 4, 6, 8, 10, or 12 or more) of the nucleotide sequences of SEQ ID NOS:30, 32, 34, 36, 38, 40, 42,44, 46, 62, 64, 66, 68, 70, 85, 87, 89, 91,130,132,134,136, 138, 140, 176, 178, 180,182,184, 186,201, 203, 205, or 207 or a derivative or fragment thereof.<br>
[0088]	In a further embodiment, the library of nucleic acids encodes six or<br>
more (e.g., 8, 10, 12, or 15 or more) TFPs selected from the group consisting of TFP-9 (SEQ ID NO:29), TFP-13 (SEQ ID N0:3l), TFP-17 (SEQ ID<br>
N0:33), TFP-18 (SEQ ID N0:35), TFP-19 (SEQ ID NO:37), TFP-20 (SEQ ID N0:39), TFP-21 (SEQ ID N0:41), TFP-25 (SEQ ID N0:43), TFP-27 (SEQ ID N0:45), TFP-11 (SEQ ID N0:61), TFP-22 (SEQ ID NO:63), TFP-29 (SEQ ID N0:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ED NO:69), TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ ED NO: 131), TFP-44 (SEQ ED NO:133), TFP-48 (SEQ ID N0:135), TFP-52 (SEQ ID N0:137), TFP-54 (SEQ ID NO: 139), TFP-40 (SEQ ID NO: 175), TFP-50 (SEQ ID NO: 177), TFP-51 (SEQ 'ED NO:179), TFP-57 (SEQ ED N0:181), TFP-58 (SEQ ED NO: 183), TFP-59 (SEQ ED NO: 185), TFP-5 (SEQ ID NO:200), TFP-6 (SEQ ED N0:202), TFP-7 (SEQ ED NO:204), TFP-8 (SEQ ED N0:206), PpTFP-1 . (SEQ ID NO:84), PpTFP-2 (SEQ ED NO:86)} PpTFP-3 (SEQ ED NO:88), 'PpTFP-4 (SEQ ID NO:90), TFP-1  (SEQ ED N0:219), TFP-2 (SEQ ED'.'"" NO:221), TFP-3 (SEQ ED NO:223), TFP-4 (SEQ ID NO:225), and TFP 32n.' ' (SEQ ID N0:208) or a derivative or fragment thereof.  In one embodiment, the library of nucleic acids comprises six or more (e.g., 8, 10, 12, or 15 or more) of the nucleotide sequences of SEQ ED NOS:30, 32, 34, 36, 38, 40, 42, 44, 46, 62, 64, 66, 68, 70, 85, 87, 89, 91, 130, 132, 134, 136, 138, 140, 176, , 178, 180, 182, 184, 186, 201, 203, 205, 207, 209, 220, 222, 224, or 226 or a ,!:.. derivative or fragment thereof.<br>
[0089]	The term "fragment thereof," as applied to a TFP, refers to a<br>
polypeptide comprising of any portion of the amino acid sequence of the TFP, wherein the fragment substantially retains the ability, to induce the secretion of a target protein to which it is fused.<br>
[0090]	The term "derivative thereof," as applied to a TFP, refers to a<br>
polypeptide consisting of an amino acid sequence that is at least 70% identical to the amino acid sequence of the TFP, wherein the polypeptide substantially retains the ability to induce the secretion of a target protein to which it is fused. In some embodiments, the derivative comprises an amino acid sequence that is at least 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical to the amino acid sequence of the TFP. The derivative may comprise additions, deletions, substitutions, or a combination thereof to the amino acid<br>
sequence of the TFP. Additions or substitutions also include the use of non-naturally occurring amino acids.<br>
[0091]	Preferably, any substitutions are conservative amino acid substitutions.<br>
A "conservative amino acid substitution" is one in which the amino acid residue is replaced with an amino acid residue having a similar side chain. Families of amino acid residues having similar side chains have been defined within the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., glycine, asparagine, glutamine, serine, threonine, tyrosine, cysteine), nonpolar side chains (e.g., alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine, tryptophan), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side;. chains (e.g., tyrosine, phenylalanine, tryptophan, histidine).<br>
[0092]	The term "derivative thereof," as applied to a nucleic acid encoding a<br>
TFP, refers to a nucleic acid consisting of a nucleotide sequence that is at least 70% identical to the nucleotide sequence of the nucleic acid encoding the TFP, wherein the polypeptide encoded by the derivative substantially retains the: ability to induce the secretion of a target protein to which it is fused. In some1 embodiments, the derivative comprises a nucleotide sequence that is at least ; 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, or 99% identical to the nucleotide sequence of the nucleic acid encoding the TFP. The derivative may comprise additions, deletions, substitutions, or a combination thereof to the nucleotide sequence of the nucleic acid encoding the TFP.<br>
[0093]	Sequence identity is calculated by comparing two optimally aligned<br>
sequences over that region of comparison, determining the number of positions at which the identical amino acid residue or nucleotide occurs in both sequences to yield the number of matched positions, dividing the number of matched positions by the total number of positions in the region of comparison (i.e., the window size), and multiplying the result by 100 to yield the percentage of sequence identity. In one aspect, percent identity is calculated as the percentage of amino acid residues or nucleotides in the smaller<br>
of two sequences which align with an identical amino acid residue or nucleotide in the sequence being compared, when four gaps in a length of 100 amino acids or nucleotides may be introduced to maximize alignment (Dayhoff, in Atlas of Protein Sequence and Structure, Vol. 5, p. 124, National Biochemical Research Foundation, Washington, D.C. (1972), incorporated herein by reference). A determination of identity is typically made by a computer homology program known in the art. An exemplary program is the Gap program (Wisconsin Sequence Analysis Package, Version 8 for UNIX, Genetics Computer Group, University Research Park, Madison, WI) using the default settings, which uses the algorithm of Smith and Waterman (Adv. Appl. • Math., 1981, 2: 482-489, which in incorporated herein by reference in its • entirety).<br>
[0094]	Examples of derivatives include, but are not limited to deletion .v<br>
•mutants (e.g., unidirectional deletion), addition of functional sequences (e.g., glycosylation sites, restriction enzyme sites), and deletion or addition (e.g., swapping) of pro-sequences or pre-sequences identified within TFPs.  One of skill in the art can prepare derivatives of TFPs or nucleic acids encoding TFPs.. using routine  mutagenesis  techniques,  such  as  those  described in the" references cited above, and identify derivatives that substantially retain the •. ability to induce the secretion of a target protein to which it is fused.<br>
[0095]	The term " substantially retains the ability to induce the secretion of a<br>
target protein to which it is fused," as used herein, refers to a fragment or derivative of a TFP which retains at least 50% of the ability of the parent TFP to induce secretion of a target protein to which it is fused. In some embodiments, at least 60, 65, 70, 75, 80, 85, 90, or 95% of the ability to induce the secretion of a target protein to which it is fused is retained. The ability to induce the secretion of a target protein may be determined by routine techniques well known in the art and described above.<br>
[0096]	One embodiment of the present invention relates to a library of nucleic<br>
acid fragments encoding TFPs, comprising 10 or more nucleic acid fragments (e.g., 50, 100, 500, 100, or 2000 or more) identified by the methods of the<br>
invention, wherein a library of pre-selected candidate TFPs was used in the screening.<br>
[0097]	Another embodiment of the present invention relates to a library of<br>
nucleic acid fragments encoding TFPs, comprising 10 or more nucleic acid fragments (e.g., 50 or 100 or more) identified by the methods of the invention, wherein a library of pre-selected candidate TFPs obtained by transforming a plurality of reporter protein-deficient host cells with a variety of vectors comprising a library of nucleic acid fragments and a nucleotide sequence encoding a reporter protein, collecting cells that grow, isolating vectors from the cells, and isolating nucleic acid fragments from the vectors, thereby . obtaining a TFP library comprising nucleic acid fragments which individually induce secretion of the reporter protein, was used in the screening.<br>
[0098]	A further embodiment of the present invention relates to a library of-;<br>
•nucleic acid fragments encoding TFPs, comprising 10 or more nucleic acid fragments (e.g., 50, 100, 500, or 1000 or more) identified by the methods of the invention, wherein a library of pre-selected candidate TFPs derived from sequences identified in a genome database by searching for (i) genes' containing a pre-secretion signal homologous with those of one or more previously identified TFPs; (ii) genes comprising a secretion signal sequence, •• or (iii) genes encoding proteins passing through endoplasmic reticulum, was used in the screening.<br>
[0099]	A further embodiment of the present invention relates to a library of<br>
nucleic acid fragments encoding TFPs, comprising 10 or more nucleic acid fragments (e.g., 50, 100, or 500 or more) identified by the methods of the invention, wherein a library of pre-selected candidate TFPs obtained by diversifying previously identified TFPs, was used in the screening.<br>
[0100]	The present invention further relates to a nucleic acid comprising a<br>
nucleotide sequence encoding a TFP identified by the methods of the invention and a nucleotide sequence encoding a target protein. In' one embodiment, the TFP is selected from, the group consisting of TFP-9, TFP-13, TFP-17, TFP-18, TFP-19, TFP-20, TFP-21, TFP-25, TFP-27, TFP-11, TFP-<br>
22, TFP-29, TFP-34, TFP-38, TFP-39, TFP-43, TFP-44, TFP-48, TFP-52, TFP-54, TFP-40, TFP-50, TFP-51, TFP-57, TFP-58, TFP-59, TFP-5, TFP-6, TFP-7 and TFP-8 or a derivative or fragment thereof. In another embodiment, the target protein is selected from IL-2, IL-32, human growth hormone and human caspase-1 subunit P10. In a particular embodiment, the TFP is TFP-9, TFP-13, TFP-17, TFP-18, TFP-19, TFP-20, TFP-21, TFP-25, TFP-27, PpTFP-1, PpTFP-2, PpTFP-35 PpTFP-4 or a derivative or fragment thereof, and the target protein is IL-2. In another embodiment, the TFP is TFP-11, TFP-22, TFP-29, TFP-34 or TFP-38 or a derivative or fragment thereof, and the target protein is IL-32 alpha, m a further embodiment, the TFP is TFP-9, TFP-13,,: TFP-17, TFP-18, TFP-19, TFP-20, TFP-21, TFP-25, TFP-27, TFP-11, TFP-22, TFP-29, TFP-34 or TFP-38 or a derivative or fragment thereof, and tha-v . target protein is growth hormone.<br>
[0101]	The present invention further relates to methods of recombinantly<br>
producing a target protein using the TFPs of the invention. In one embodiment, the method comprises preparing a vector comprising a nucleotide sequence encoding a target protein operably linked to a nucleotide sequence encoding a TFP or a derivative or fragment thereof, transforming a host cell with the vector, and culturing the host cell under conditions in which the target protein is produced and secreted from the host cell. In one embodiment, the TFP is selected from the group consisting of TFP-9 (SEQ ID N0:29), TFP-13 (SEQ TD NO:31), TFP-17 (SEQ ID NO.33), TFP-18 (SEQ ID NO:35), TFP-19 (SEQ TD NO:37), TFP-20 (SEQ ID N0:39), TFP-21 (SEQ ID N0:41), TFP-25 (SEQ ID N0:43), TFP-27 (SEQ ID N0:45), TFP-11 (SEQ ID NO-.61), TFP-22 (SEQ E) N0:63), TFP-29 (SEQ ID NO:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ID N0:69), TFP-39 (SEQ 3D N0:129), TFP-43 (SEQ ID NO:131), TFP-44 (SEQ ID NO:133), TFP-48 (SEQ TD NO-.135), TFP-52 (SEQ ID NO:137), TFP-54 (SEQ ID NO:139), TFP-40 (SEQ ID N0:175)&gt; TFP-50 (SEQ ID NO:177), TFP-51 (SEQ ID N0:179), TFP-57 (SEQ ID NO:181), TFP-58 (SEQ ID NO:183), TFP-59 (SEQ ID N0:185), TFP-5 (SEQ 3D NO:200), TFP-6 (SEQ ID NO:202), TFP-<br><br>
7 (SEQ ID NO:204), TFP-8 (SEQ ID NO:206), PpTFP-1 (SEQ ID N0:84), PpTFP-2 (SEQ ID NO:86), PpTFP-3 (SEQ ID NO:S8), and PpTFP-4 (SEQ ID N0:90) or a derivative or fragment thereof. In a further embodiment, the target protein is selected from IL-2, IL-32, human growth hormone and human caspase-1 subunitPlO.<br>
[0102]	The  target protein may be  recombinantly produced  using  any<br>
expression system known in the art.    Preferably, the target protein is recombinantly expressed, e.g., in bacterial, yeast, or mammalian cell cultures. Recombinant   expression   involves   preparing   a   vector   comprising   a polynucleotide encoding the target protein, delivering the vector into a host • cell, culturing the host cell under conditions in which the target protein is expressed, and separating the target protein.    Methods and materials fopv preparing recombinant vectors and transforming host cells using the same1,,, replicating the vectors in host cells and expressing biologically active foreign polypeptides and proteins are discussed above and described in Sambrook et al, Molecular Cloning, 3rd edition, Cold Spring Harbor Laboratory, 2001 and Ausubel et al, Current Protocols in Molecular Biology, John Wiley &amp; Sons, New York 3rd edition, (2000), each incorporated herein by reference.<br>
[0103]	Vector DNA can be introduced into prokaryotic or eukaryotic cells via<br>
conventional transformation or transfection techniques. As used herein, the terms "transformation" and "transfection" are intended to refer to a variety of art-recognized techniques for introducing foreign nucleic acid (e.g., DNA) into a host cell, including calcium phosphate or calcium chloride co-precipitation, DEAE-dextran-mediated transfection, lipofection, or electroporation. Suitable methods for transforming or transfecting host cells can be found in Sambrook, et al. (MOLECULAR CLONING: A LABORATORY MANUAL. 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989), and other laboratory manuals.<br>
[0104]	For stable transfection of mammalian cells, it is known that, depending<br>
upon the expression vector and transfection technique used, only a small fraction of cells may integrate the foreign DNA into their genome. In order to<br>
identify and select these integrants, a gene that encodes a selectable marker (e.g., resistance to antibiotics) is generally introduced into the host cells along with the gene of interest. Various selectable markers include those that confer resistance to drugs, such as G418, hygromycin and methotrexate. Nucleic acid encoding a selectable marker can be introduced into a host cell on the same vector as that encoding the target protein or can be introduced on a separate vector. Cells stably transfected with the introduced nucleic acid can be identified by drug selection (e.g., cells that have incorporated the selectable marker gene will survive, while the other cells die).<br>
[0105]	The target protein may be isolated from the medium in which the host •<br>
cells, are grown, by purification methods known in the art, e.g., conventional . chromatographic methods including immunoaffinity chromatography, receptor •, affinity chromatography, hydrophobic interaction chromatography, lectin*. affinity chromatography, size exclusion filtration, cation or anion exchange chromatography, high pressure liquid chromatography (HPLC), reverse phase HPLC, and the like. Still other methods of purification include those methods wherein the desired protein- is expressed and purified as a fusion protein having a specific tag, label, or chelating moiety that is recognized by a specific binding partner or agent. The purified protein can be cleaved to yield the desired protein, or can be left as an intact fusion protein. Cleavage of the fusion component may produce a form of the desired protein having additional amino acid residues as a result of the cleavage process.<br>
[0106]	If the isolated target protein is not biologically active following the<br>
isolation procedure employed, various methods for "refolding" or converting the polypeptide to its tertiary structure and generating disulfide linkages, can be used to restore biological activity. Methods known to one of ordinary skill in the art include adjusting the pH of the solubilized polypeptide to a pH usually above 7 and in the presence of a particular concentration of a chaotrope. The selection of chaotrope is very similar to the choices used for inclusion body solubilization but usually at a lower concentration and is not necessarily the same chaotrope as used for the solubilization. It may be<br>
required to employ a reducing agent or the reducing agent plus its oxidized form in a specific ratio, to generate a particular redox potential allowing for disulfide shuffling to occur in the formation of the protein's cysteine bridge(s). Some of the commonly used redox couples include cysteme/cystamiae, glutathione (GSH)/dithiobis GSH, cupric chloride, dithiothreitol (DTT)/dithiane DTT, 2-mercaptoethanol (bME)/dithio-b(ME). To increase the efficiency of the refolding, it may be necessary to employ a co solvent, such as glycerol, polyethylene glycol of various molecular weights, and arginine.<br>
[0107]	In one embodiment, the present invention relates to a linear vector<br>
comprising a nucleic acid fragment from a library of nucleic acid fragments and a nucleotide sequence encoding a N-terrninal amino acid-deleted reporter." protein.   la another embodiment, the linear vector further comprises a target'-protein-encoding nucleotide sequence.<br>
[0108]	The present invention further relates to a plurality of reporter protein-<br>
deficient host cells transformed with a library of linear vectors of the invention. In one embodiment, the host cells are further transformed with a-nucleic acid encoding a target protein.<br>
[0109]	The following examples are illustrative, but not limiting, of the method<br>
and compositions of the present invention. Other suitable modifications and adaptations of the variety of conditions and parameters normally encountered in clinical therapy and which are obvious to those skilled in the art are within the spirit and scope of the invention.<br>
EXAMPLE 1 PREPARATION OF INVERTASE-DEFICIENT YEAST MUTANT<br>
[0110]	For rapid screening of the translational fusion partners (TFP) of non-<br>
producible proteins, an automatic screening system was established through<br>
the evaluation of cell growth in a sucrose medium using yeast invertase as a reporter.<br>
[0111]	A yeast strain having no invertase activity was required to use an<br>
invertase gene as a reporter for the positive screening of useful TFP. Thus, the chromosomal SUC2 gene of wild type yeast was deleted. In order to prepare a SUC2 deletion cassette, a plasmid pRB58 (Carlson et al., Cell 20:145 (1982)) was digested with EcoKL and Xliol, and a SUC2 coding gene was recovered and introduced into EcoRI-XhoI sites of pBluescript KS+ (Stratagene, USA), thus generating pBIABX. As shown in FIG. 1, an URA3 gene having a repeat sequence of 190 bp (Tcl90) (Bae et al., Yeast 21:437 (2004)) at both ends was inserted into Hin&amp;Sl-Xbal sites of the SUC2 gene contained in pBIABX, thus ... generating pBIU. The pBIU was digested with EcoRI and Xhol, and was •-transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GALF canl) and Y2805Aga/J (Mat a ura3 SUC2 pep4::HIS3 gall canl) strain (SK Rhee, Korea Research Institute of Bioscience and Biotechnology) according to a lithium acetate method (Hill et al, Nucleic Acids Res. 19:5791 (1991)). The transformants, Y28Q5Asuc2U (Mat a suc2::URA3 pep4::HIS3 GAL1 canl), Y2805Ag-a/lAs-wc2L7 (Mat a suc2::URA3 pep4::HJS3 gall canl), were selected in a selection medium lacking uracil.<br>
[0112]	To evaluate the invertase activity of the transformed cells, a single<br>
colony was cultured in two media containing glucose and sucrose, respectively, as the sole carbon source. As a result, the colonies grew normally in the glucose medium, but grew very slowly in the sucrose medium compared to a control. In order to investigate the amount of invertase secreted into the culture medium, the SUC2+ strain and the Asuc2 strain were cultured on YPD media (1% yeast extract, 2% Bacto-peptone and 2% glucose). Proteins contained in the culture supernatants were separated by SDS-PAGE, and the gel was incubated in a sucrose solution for 30 rnin and subjected to zymogram analysis using a dye, TTC (2, 3, 5-triphenyl-tetrazoh'um chloride). As shown in FIG. 2, the Lsuc2 strain was found to lose most of its invertase activity. However, the mutant strain had a problem of growing even at very<br>
slow rates in the sucrose medium. This is believed to be because cells<br>
partially grow by gluconeogenesis • through the function of mitochondria.<br>
Thus, to solve this problem, antimycin A, an inhibitor of mitochondrial<br>
electron transport, was added to the medium to block cell growth. As a result,<br>
the growth of the mutant strain was completely inhibited in the YPSA (1%<br>
yeast extract, 2% Bacto-peptone, 2% sucrose, 1 [ig/ml antimycin A, and 2%<br>
agar) or YPSGA (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 1 [j,g/ml<br>
antimycin A, and 2% agar) medium containing antimycin A (FIG. 3).<br>
[0113]	In order to recover uracil auxotrophy of the selected strain,<br>
Y2805As«c2[7 (Mat a suc2::URA3 pep4::HIS3 GAL1 canl) and Y2805AgaWAswc2£/(Mrt a suc2::URA3pep4::HIS3 gall canl), with a URA3 vector containing a TFP library, it was necessary to remove the URA3 gene;, which was used for the deletion of the SUC2 gene. To do this, cells were, cultured in a medium containing 5-fluoroorotic acid (5-FOA) and selected for loss of the URA3 gene, thus obtaining URA3 pop-out strains, Y2805Asue2 (Mat a ura3 suc2::Tc!90 pep4::HIS3 GAL1 canl) and Y2805Agc/7Ajwc2 (Mat a ura3 suc2::Tc!90 pep4::HIS3 gall canl) (FIG. 1). Southern blotting was carried out to confirm the deletion of the ££702 gene on the chromosome, as expected, and the URA3 gene was deleted (popped-out) from the integration locus (FIG. 4). When chromosomal DNA from S. cerevisiae Y2805 was treated with EcdBI and analyzed by Southern blotting using a SUC2 gene as a probe, a fragment of about 4.3 kb was detected. This size increased to about 5.0 kb when a URA3 gene was inserted (Y2805Aga/IAs«c2lOs and decreased to about 3.7 kb when the URA3 gene was popped-out (Y2805AgflWAs«c2). As shown in FIG. 4, as expected, the SUC2 gene was obviously deleted, and the URA3 gene was lost (popped-out).<br>
EXAMPLE 2<br>
DEVELOPMENT OF AUTOMATIC SCREENING SYSTEM USING AN INVERTASE AS A SECRETION REPORTER<br>
[0114]	The invertase deficient strain was evaluated for the possibility of being<br>
automatically screened in a sucrose medium through the expression of a protein fused to invertase, using two human therapeutic proteins, a human serum albumin (HSA) which is well secreted in yeast, and a human interleukin-2 (IL-2) which is hardly secretable in yeast.<br>
[0115]	Three   plasmids,    pYGAP-SNS-SUC2,    pYGAP-HSA-SUC2    and<br>
pYGAP-hIL2-SUC2, were constructed to test for automatic selection on-sucrose media. For the construction of pYGAP-SUC2 containing an invertase,-gene (SUC2, YIL162W) expression cassette under the control of the yeast GAPDH promoter, pST-SUC2 was constructed first by subcloning a PCR product containing SUC2 gene amplified from pBIABX (FIG. 1) using primers SUC-F (SEQ ID NO. 1) and SUC-R (SEQ ID NO. 2) into pST-Blue-1 (Novagen, USA). PCR was carried out with Pfu polyrnerase (Stratagene, USA) or Ex-Taq DNA polyrnerase (TaKaRa Korea Biomedical Inc., Seoul, Korea). PCR conditions included one cycle of 94°C for 5 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 min, followed by one final cycle of 72°C for 7 min. The EcoRI-Sali fragment containing SUC2 from pST-SUC2 was subcloned into EcoSI-SalL digested YGAPa-HTR harboring a GAPDH promoter instead of the GAL10 promoter of YEGa-HTR525 (Sohn et al., Process Biochem. 30:653 (1995)), and the resulting plasrnid was named pYGAP-SUC2. To facilitate the fusion of foreign genes with SUC2 and induce in vivo cleavage of the fused proteins by yeast dipeptidyl protease Kex2p (Mizuno K et al, Biochem. Biophys. Res. Commun. 156:246 (1988)) during secretion, an artificial sequence for two Sfil and &amp;NotI recognition sites and a sequence coding for Kex2p cleavage site (Leu-Asp-Lys-Arg (SEQ ID NO:214)) were in-frame added between a secretion signal sequence (19 amino<br>
acids) and a SUC2 mature sequence (513 amino acids) ofSUC2 by PCR. Two<br>
PCR fragments, PCR-A containing a GAPDH promoter and a SUC2 secretion<br>
signal sequence amplified using primers GAP-F (SEQ E)'NO:3) and SUCSS-<br>
R (SEQ ID NO:4) and PCR-B containing a mature part of SUC2 amplified<br>
from pYGAP-SUC2 using primers SUCM-F (SEQ ID N0:5) and SUC-R<br>
(SEQ ID NO:2) were amplified from pYGAP-SUC2, respectively. Both<br>
fragments were subcloned into pST-Blue-1 and recovered by SacI-NotI<br>
digestion for PCR-A and Notl-SaE digestion for PCR-B. Enzyme digested<br>
PCR-A and PCR-B were co-ligated into Sacl-SaH digested pYGAP-SUC2 and<br>
the resulting plasmid was named pYGAP-SNS-SUC2. For the construction of<br>
a plasmid, pYGAP-HSA-SUC2, containing an in-frame fused gene between .<br>
human serum albumin (HSA) with SUC2, the HSA gene was amplified from. *<br>
pYHSAS (Kang et al., J. Microbiol Biotechnol 8:42 (1998)) using primers<br>
HSA-F (SEQ ID NO:6) and HSA-R (SEQ ID NO:7) and subcloned in pST-<br>
Blue-1. A Sfii digested DNA containing the HSA gene was subcloned into the<br>
SfiL digested pYGAP-SNS-SUC2 vector. The resulting plasmid was named<br>
pYGAP-HSA-SUC2. For the construction of a plasmid, pYGAP-hIL2-SUC25<br>
containing an in-frame fused gene between human interleukin-2 (hTL2) with<br>
SUC2, the hIL2 gene was amplified from pT7-hIL2 (JK Jung, Korea Research<br>
Institute of Bioscience and Biotechnology) using primers 3L2-F (SEQ ID<br>
N0:8) and IL2-R (SEQ ID N0:9) and subcloned into pST-Blue-1. Then a<br>
plasmid pYGAP-ML2-SUC2 was constructed by the subcloning of a SfiL<br>
digested hTL2 fragment into the SfiL digested pYGAP-SNS-SUC2 vector.<br>
[0116]	The pYGAP-HSA-SUC2 vector expressing a fusion protein of human<br>
serum albumin and invertase, the pYGAP-hIL2-SUC2 expressing a fusion protein of IL-2 and invertase, and the pYGAP-SNS-SUC2 expressing only invertase were individually transformed into a yeast strain (Y2805As'Wc2) which is deleted for its endogenous invertase gene and thus unable to grow in a sucrose medium. The transformed cells were spread onto a UD plate (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) containing glucose as a sole carbon source and YPSA<br>
media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 1 ng/ml antimycin A, and 2% agar) containing sucrose as a sole carbon source and cell growth of each transformation was observed (FIG. 5). When cells were transformed with pYGAP-SNS-SUC2 expressing invertase, they normally grew in both carbon sources. Similarly, when cells were transformed with pYGAP-HSA-SUC2 having a fusion of HS A at the N-terminus of invertase, they grew well using both carbon sources. In contrast, when cells were transformed with pYGAP-hIL2-SUC2 having a fusion of IL2 instead of HSA, they grew normally on the glucose medium but hardly grew on the sucrose medium. This inability of the pYGAP-hIL2-SUC2-transformed cells to grow in the sucrose medium was believed to be caused by the IL-2 being unable to be secreted from the cells and leading to a block of the secretion of invertase" fused thereto. These results suggested a positive selection system using an1 invertase as a reporter for a secretion signals and a fusion partner (a translational fusion partner, TFP) from any sources of DNA enhancing the secretion of non- or hardly-secretable proteins such as human EL2.<br>
EXAMPLES<br>
PREPARATION OF VECTORS FOR THE CONSTRUCTION OF TRANSLATIONAL FUSION PARTNER (TFP) LIBRARY<br>
[0117]	Several vectors were designed for the construction of a TFP library<br>
from genomic DNA or a cDNA library from any source. For the construction of a TFP library from cDNA, a plasmid YGaMV was constructed (FIG. 6). A PCR was carried out to amplify a DNA fragment encoding invertase from pYGAP-hIL2-SUC2 using two PCR primers, Sfil-SUC-F (SEQ ID NO: 10) and SUC-Xho-R (SEQ ID NO: 11). PCR conditions included one cycle of 94°C for 5 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 min, followed by one final cycle of 72°C for 7 min. Then an EcoRI-Satt digested PCR fragment was ligated to EooBI-SaR digested YEGa-HTR525<br>
and the resulting plasmid was named YGalNV (FIG. 6). For the construction of a TFP library from partially digested genomic DNA, three vectors, YGaFOINV, YGaFlINV and YGaF2JNV, each containing one of three different reading frames of the SUC2 gene were constructed (FIG. 7). Three different PCR amplifications were performed from YGalNV as a template using a common forward primer GallOO-F (SEQ ID NO:12) and three reverse primers with different reading frames, Xho-FO-R (SEQ ID NO: 13), Xho-Fl-R (SEQ ID N0:14), and Xho-F2-R (SEQ ID NO: 15). PCR was done using a Pfu polymerase (Stratagene, USA). PCR conditions included one cycle of 94°C for 5 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 rain, followed by one final cycle of 72°C for 7 min. Three PCR fragments. were eluted from agarose gel and digested with SfiL. Then they were,. subcloned into Sfil digested YGalNV, respectively. Three resulting plasmids were named YGaFOINV, YGaFlINV and YGaF2INV (FIG. 7).<br>
EXAMPLE 4 CONSTRUCTION OF CDNA LIBRARY FUSED TO YEAST INVERTASE<br>
[0118]	For the construction of a cDNA library, total RNA was isolated from<br>
yeast S. cere\&gt;isiae Y2805 (Mat a ura3 his3 pep4::HIS3 canl). Yeast cells were cultivated to mid-exponential phase in YPD media (2% yeast extract, 1% Bacto-peptone and 2% glucose). Total RNA was isolated by a method described in Elion et al. (Elion et at., Cell 39:663 (1984)). Purification of poly(A)+ mRNA from the total RNA was carried out using an Oligotex mRNA kit (Qiagen, Germany). cDNA was synthesized from the isolated mRNA using a SMART cDNA synthesis kit (BD Bioscience, USA). A specially designed primer ASA24N6 (SEQ ID NO: 16) was used for the synthesis of the first strand cDNA instead of a primer included in the SMART kit. Because the primer ASA24N6 was designed to contain a Sfil recognition site and a random hexameric sequence, it was used for the synthesis of the first strand<br>
cDNA from mRNA by reverse transcription as in the method described in the instruction manual of SMART kit (FIG. 8). Primer ASA24N6 could randomly bind to any position of mRNA due to its random hexameric sequence. Thus, most of the first stranded cDNA amplified by using this method contained the 5' partial sequence encoding the N-terminal part of yeast genes. The first stranded cDNA library with 5' partial sequences was used as a PCR template for double stranded cDNA synthesis with the 5' PCR primer of SMART kit (BD Bioscience, USA) and the primer ASA24 (SEQ ID NO: 17). The resulting PCR products contained numerous 5' partial fragments of cDNA with Sfil sites at both ends. PCR conditions included one cycle of 95°C for 20 sec, and 20 cycles of 95°C for 30 sec, 68°C for 6 min as recommended in the • kit. Amplified cDNA was treated with phenol/chloroform/isoamyl alcohol (25:24:1) and precipitated with 2 volumes of ethanol and 0.1 volume of 3 M sodium acetate (pH 5.0).. Recovered cDNA was digested with Sfil at 50°C for 2 hours and then fractionated using agarose gel electrophoresis. 0.5 to 1 kb DNA was isolated from the gel using a gel extraction kit (Bioneer, Korea). Extracted DNA was ligated into Sfil digested YGalNV vector (FIG. 6) and<br>
i<br>
transformed into E. coli DH5a. Transformed E. coli was plated on LB media containing ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 ug/ml ampicillin) and incubated at 37°C overnight. About 5&gt;
EXAMPLES<br>
CONSTRUCTION OF GENOMIC DNA LIBRARY FUSED TO YEAST<br>
INVERTASE<br>
[0119]	The TFP library constructed in Example 4 was obtained from a cDNA<br>
library which were synthesized from a pool of mRNA. Because the mRNA of<br>
a highly expressed gene is usually abundant compared to that of a poorly expressed gene, a TFP library could be biased with those from highly expressed genes. Furthermore, some genes are completely repressed at a point of the growth phase and thus, they could not be amplified in a TFP library even though they were good candidates for a TFP. To solve such problems, genomic DNA was also used for the construction of a TFP library. As shown in FIG. 9, genomic DNA of S. cerevisiae Y2805 was partially digested with Sau3AI and incubated at 70°C for 10 min to inactivate the enzyme. The DNA was 2 bases filled with Klenow fragment and 0.2 mM of dTTP and dCTP at 25° for 1 hour and then 0.5 to 1 kb DNA was isolated from an agarose gel. In • addition, vectors YGaFOINV, YGaFlINV and YGaF2INV (FIG. 7) were; digested with XJiol. After inactivation of the enzyme at 70°C for 10 min, the' vectors were also 2 bases filled with Klenow fragment and 0.2 mM of dTTP and dCTP and purified from an agarose gel. Each vector was ligated with the partially digested genomic DNA and transformed into E. coif DH5a, respectively. Transformed E. coli was plated on LB media containing ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 u-g/ml ampicillin) and incubated at 37°C overnight. About 2&gt;
EXAMPLE 6 CONSTRUCTION OF TFP LIBRARY SECRETING INVERTASE<br>
[0120]	For the first selection of a TFP library secreting invertase from the<br>
genomic and cDNA libraries constructed in Example 4 and 5, library DNA was transformed into S. cerevisiae ~Y28Q5A.gall&amp;suc2(Mat a ura3 suc2::Tc!90 pep4::HIS3 gall canl) according to a lithium acetate method (Hill et al, Nucleic Acids Res. 19:5791 (1991)). Y28Q5AgallAsuc2 cannot<br>
use sucrose and galactose as carbon sources due to the deletion of both, genes. Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 pig/ml antimycin A, and 2% agar) and incubated at 30°C for 4 to 6 . days.  Around 3,000 and 1,000 transformants were obtained from the cDNA and genomic DNA library, respectively. All transformants grown on YPSGA media were transferred to a UD plate with a toothpick and incubated at 30°C for 2 days. Total DNA was isolated from the pooled cells using glass beads and" then the DNA was precipitated with ethanol.   To recover the plasmid containing TFP library, total DNA was retransformed into E. coli DH5a. •. Transformed E. coli was plated on LB media containing ampicillin (1% Bacto-peptone,  0.5% yeast extract,  1% NaCl with 50  fig/ml ampicillin) and incubated at 37°C overnight.    Around 2*104 E. coli transformants were obtained and collected with sterile distilled water for the isolation of total plasmids using a plasmid isolation kit (Bioneer, Korea).   Thus, a TFP pool containing up to 4,000 TFPs which individually induce the secretion of invertase was constructed.   Nucleotide sequencing of the raridomly selected plasmids from the library revealed that all of the TFPs were originated from yeast genes individually encoding different secretory proteins.<br>
EXAMPLE 7<br>
CONSTRUCTION OF TFP LIBRARY VECTOR APPLICABLE TO MANY TARGET PROTEINS THROUGH IN VIVO RECOMBINATION<br>
[0121]	Around 4,000 TFPs having a potential to secrete invertase were<br>
collected in Example 6. For the development of a TFP library vector which can be easily applicable to any target gene, a simple in vivo recombination system was designed. A vector, YGadV45 (FIG. 10), was first constructed for the in-frame insertion of any target protein gene between the TFP library and<br>
the SUC2 gene through in vivo recombination. YGadV45 contains a defective SUC2 (dSUC2) which is an N-terminal 45 ammo acid deleted SUC2 and thus, having no invertase activity. The vector was also designed to contain a Notl and two Sfil recognition sequences, a linker sequence as a recombination target and a Swal recognition sequence in front of the dSUC2 for the simple insertion of a TFP library and target gene through in vivo recombination. A PCR was carried out from a template YGaBSTV using a forward primer TNV45-F (SEQ ID NO: 18) and a reverse primer SUC-Xho-R (SEQ ID NO: 11) and P/u polymerase (Stratagene, USA). PCR conditions included one cycle of 94°C for 3 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 90 sec, followed by one final cycle of 72°C for 7 min. From the PCR, an ., N-terminal modified defective SUC2 gene fragment was obtained. A Notl-SalL digested PCR fragment was subcloned into a Notl-SaH digested vector YGalNV (FIG. 6) and the resulting plasmid was named YGadV45 (FIG. 10). For the construction of a TFP library in YGadV45, the TFP library obtained in Example 6 was digested with SfiL and fractionated in an agarose gel. Around 0.5 to 1 kb DNA fragments were isolated from the gel using a gel extraction kit (Bioneer, Korea). Purified DNA was subcloned into Sfil digested YGadV45 (FIG. 10) and transformed into E. coli DH5a. Transformed E. coli was plated on LB media with ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 fig/ml ampicillin) and incubated at 37°C overnight. Around 5x10 E. coli transformants were collected with sterile distilled water for the isolation of total plasmids. Total plasmids were isolated using a plasmid isolation kit (Bioneer, Korea). The isolated vectors contained TFPs selected in Example 6 fused to a defective SUC2. Thus, transformation of this TFP library vector into S. cerevisiae Y2805AgaWAs«c2 (Mat a uraS suc2::Tc!90pep4::HIS3 gall canT) gave thousands of transformants on aUD plate (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) but no transformants on YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 ug/ml antimycin A, and 2% agar). Thus, it could greatly reduce the background<br>
level of selection on YPSGA media. Only the cells with a vector harboring an in-frame inserted target gene between TFP and SUC2 could grow on YPSGA after correct in vivo recombination. The TFP library vectors contained a rare cutting restriction enzyme Swal site and a linker sequence between the TFP library and the dSUC2 for linearization and for homologous recombination, respectively.<br>
EXAMPLES<br>
AUTOSELECTION OF AN OPTIMAL TFP SECRETING A TARGET PROTEIN FROM TFP LIBRARY<br>
[0122]	For   the   in-frame   fusion   of  target   proteins   through   in   vivo<br>
recombination to the TFP library vectors developed'in Example 7, a target gene must have a linker DNA in the 5'-end and a N-terminal part ofSUC2 in the 3'-end. To add such a sequence to the end of a target gene, overlap , extension PCR was used. A first step PCR was carried out for the amplification of a target gene encoding a mature protein using a target specific forward primer KR-target-F (SEQ ID NO: 19) and a target specific reverse primer Target-DsTV-R (SEQ ID NO:20) from a plasmid containing target gene. Separately, another PCR for the amplification of a N-terminal part of SUC2 which will be fused to the 3'-end of a target gene was also carried out using a forward primer KR-lhv-F (SEQ TD N0:21) and a reverse primer Inv500-R (SEQ ID N0:22) from YGalNV (FIG. 6). PCR was performed with PJu polymerase (Stratagene, USA) and PCR conditions included one cycle of 94°C for 3 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 90 sec, followed by one final cycle of 72°C for 7 min. Then a second PCR was done from the two DNA fragments amplified in the first step using a forward primer LNK40 (SEQ ID NO:23) and a reverse primer lhv500-R (SEQ ID NO:22). The resulting fragment (insert fragment) harbored 40 nucleotides of linker DNA in the 5'-end and 500 bp of DNA encoding the Kex2p<br>
recognition site (Leu-Asp-Lys-Arg (SEQ ID N0:214)) and a N-terminal part of invertase in the 3'-end, respectively. For in vivo recombination, the insert fragment was mixed at a 2:1 ratio with Sfwal digested TFP library vectors constructed in Example 7 and used for transformation into S. cerevisiae Y2805kgallhsuc2 (Mat a ura3 suc2::Tc!90 pep4::HIS3 gall canl) (FIG. 11). Transformed cells were spread on YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 ug/ml antimycin A, and 2% agar) and incubated for 5 days. Only an in-frame fusion of the insert fragment with a vector containing a proper TFP through in vivo recombination could support cell growth on YPSGA media. Thus, using this method, an optimal TFP for any target protein could be retrieved by simple selection of growing cells on YPSGA media.<br>
EXAMPLE 9<br>
'    AUTOSELECTION OF AN OPTIMAL TFP USING A DUAL REPORTER SYSTEM WITH LIPASE AND INVERTASE<br>
[0123]	An autoselection system using a reporter, invertase, as described in<br>
Example 8 was very useful for the screening of an optimal TFP for a target protein such as IL2 which blocks the secretion of invertase completely as found in Example 2. As dozens of colonies could grow on sucrose media, it was easy to select an optimal TFP from the TFP library. Fusion of some target proteins, however, could not completely block the secretion of invertase even though a weak TFP was connected. Such leaky colonies could also grow on sucrose media. Thus, considerable number of colonies should be tested for their secretion level to select an optimal TFP. To solve such a time-consuming problem, a simple selection method was developed to identify a colony having a high protein secretion level with a halo-forming reporter, lipase, on a tributyrin-containing plate. A gene encoding lipase (CalB, lipase B of Candida antarctica) was in-frame fused to the 5'end of invertase. Using<br>
this dual reporter system, transformants could be selected with both invertase and lipase activity on YPSGA media containing tributyrin, simultaneously. Colonies secreting protein at a high level could be simply determined with the size of halo formed around the colonies. As shown in FIG. 12, construction of a dual reporter was done by three steps of PCR. A 1 kb PCR fragment containing CalB was first amplified using a CalB forward primer KR-CalB-F (SEQ ID NO:24) and a reverse primer CalB-Inv-R (SEQ ID N0:25) from a plasmid pLGK-Lip!4* containing a mutant CalB gene (SY Kim, Ph.D. thesis, Yonsei University, Korea, 2001). Separately, a 0.5 kb PCR fragment containing a 5' partial SUC2 gene was amplified from YGalNV (FIG. 6) using a forward primer KR-Inv-F (SEQ ID NO.21) and a reverse primer Inv500-R (SEQ ID NO:22). PCR was performed with Pfu polymerase (Stratagene, USA) and PCR conditions included one cycle of 94°C for 3 rnin, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 90 sec, followed by one final cycle of 72°C for 7 min. Then, a second PCR was done from the two DNA iragments amplified in the first step using a forward primer KR-CalB-F (SEQ ID NO:24) and a reverse primer Inv500-R (SEQ ID NO:22). Separately, a PCR for a target gene was amplified using primers KR-Target-F (SEQ ID NO: 19) and Target-CalB-R (SEQ ID NO.26) from a plasmid containing a target gene as described in Example 8. The third PCR was done using a forward primer LNK40 (SEQ ID NO.23) and a reverse primer Ihv500-R (SEQ ID NO:22) from a template rnixture of a target gene and CalB fused with a partial SUC2 gene. The resulting DNA fragment (insert fragment) consisted of 40 nucleotides of linker, a target gene, Kex2p cleavage site (Leu-Asp-Lys-Arg (SEQ ID N0:214)), CalB, Kex2p cleavage site (Leu-Asp-Lys-Arg (SEQ ID N0:214)) and 500 bp of 5' partial SUC2 gene in order. For in vivo recombination, PCR amplified insert fragment was mixed at a 2:1 ratio with Swal digested TFP library vectors constructed in Example 7 and used for transformation into S. cerevisiae Y2805Aga/7Asrwc2 (Mat a ura.3 suc2::Tc!90 pep4::HIS3 gall canT). Transformed cells were spread on YPSGA (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 ug/ml antimycin A,<br>
and 2% agar) for selection with invertase activity and YPSGAT (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 jig/ml antimycin A, 1% tributyrin, and 2% agar) for selection with invertase and lipase activities, respectively. The transforming plates were incubated at 30°C for 5 days. Colonies secreting a target protein, lipase and invertase were formed on both YPSGA and YPSGAT plates. As expected, different sizes of halo were formed around colonies. The size of the halo was comparatively proportional . to the secreted lipase activity. Thus, it was easy to select a colony with high secretion level directly from the transforming plate (FIG. 13).<br>
EXAMPLE 10<br>
NOVEL TFP SELECTED FROM TFP LIBRARY FOR THE SECRETION<br>
OF HUMAN INTERLEUKIN-2<br>
[0124]	As an example for identifying optimal TFPs using a method developed<br>
in this invention, .a hardly secretable protein, human mterleukm-2 (hTL2) was tried. An insert fragment containing the human IL2 gene and a 500 bp N-terminal part of SUC2 was amplified using PCR as described in,Example 8 (FIG. 11). A PCR was carried out using a forward primer KR-IL2-F (SEQ ID NO:27) and a reverse primer IL2-INV-R (SEQ ID NO:2S) from pT7-hIL-2 (IK Jung, Korea Research Institute of Bioscience and Biotechnology) as a template. Separately, another PCR for the amplification of an N-terminal part ofSUC2 to be fused to the 3'-end of the IL2 gene was also carried out using a forward primer KR-Inv-F (SEQ ID NO:21) and a reverse primer Inv500-R (SEQ ID N0:22) from YGalNV (FIG. 6). Then the second PCR was done from the two DNA fragments amplified in the first step using a forward primer LNK40 (SEQ ID N0:23) and Inv500-R (SEQ ID N0:22). The resulting fragment (insert fragment) harbored a 40 nucleotide linker DNA containing a Kex2p recognition sequence (Leu-Asp-Lys-Arg (SEQ ID N0:214)), IL2, an additional Kex2p recognition sequence, and an N-terminal part of invertase in<br>
order. This fragment was co-transformed with the Swal digested TFP library vector constructed in Example 7 into S. cerevisiae Y2805Ag-a/7A1ywc2 (Mat a ura3 suc2::Tc!90pep4::HIS3 gall canT). Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 ug/ml antimycin A, and 2% agar) and incubated at 30°C for 5 days. Around 2&gt;
<br>
(Table Removed)<br>
EXAMPLE 11<br>
SECRETION OF HUMAN IL2 USING SELECTED TFPS<br>
[0125]	To confirm the secretion of human IL2 using selected TFPs, 9<br>
plasmids were constructed using PCR to remove the 5'-UTR of each TFP and • SUC2 of selected plasmids in Example 10 (FIG. 14). Nine forward primers, BamH-YGR-F (SEQ ID NO:47), BamH-SIM-F (SEQ ID N0:48), BamH-YNL-F (SEQ ID NO:49), BamH-ECM-F (SEQ ID N0:50), BamH-ATG-F (SEQ ID NO:51), BamH-GAS-F (SEQ TD NO:52), BamH-YOR-F (SEQ ID NO:53), BamH-OST-F (SEQ ID NO:54), BamH-UTH-F (SEQ ID NO:55) and a common reverse primer IL2-TGA-R (SEQ ID N0:56) were used for PCR from plasmids PYHTS-TFP9, pYHTS-TFP13, pYHTS-TFP17, pYHTS-TFP18, pYHTS-TFP19, pYHTS-TFP20, pYHTS-TFP21, pYHTS-TFP25, and pYHTS-TFP27, respectively. The nine PCR amplified fragments were digested with BamUl and SalL and each fractionated from an agarose gel. Separately, another PCR to amplify the GAL promoter was done using a forward primer Sac-GAL-F (SEQ ID N0:57) and a reverse primer GAL-BamH-R (SEQ ID N0:58) from YEGa-HIR525 (Sohn et al, Process Biochem. 30:653 (1995)). Sacl-SamHI digested GAL promoter and the nine BarnHL-SaH digested fragments were co-ligated into Sacl-Sali digested YEGa-HIR525. The resulting plasmids were named pYGT9-IL2 (FIG. ISA), pYGT13-IL2 (FIG. 15B), pYGTl7-IL2 (FIG. 15C), pYGT18-IL2 (FIG. 16A), pYGT19-IL2 (FIG. 16B), pYGT20-IL2 (FIG. 16C), pYGT21-IL2 (FIG. 17A),<br>
pYGT25-IL2 (FIG. 17B), and pYGT27-EL2 (FIG. 17C), respectively. Human IL2 expression vectors, pYGT9-IL2 (E. coli DH5a/pYGT9-IL2, FIG. ISA) andpYGT17-IL2 (E. coli DH5a/pYGT17-IL2, FIG. 15C) were deposited at an international depository authority, KCTC (Korea Collection for Type Cultures; 52, Oun-dong, Yusong-ku, Taejon, Korea) on July 21, 2005, and assigned accession numbers KCTC 10828BP and KCTC 10829BP, respectively. Nucleotide sequences of all constructed vectors were confirmed to have a correct in-frame fusion between TFP and IL2 and each vector was transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canT). Transforming cells were plated on UD media (0,67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days. A single colony of each transformation was inoculated into YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000 xg. The pellet was freeze-dried and resuspended ha IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed in 12% SDS-PAGE. The gel was stained with gel staining reagent (PhastGel® Blue R, Pharmacia Biotech, USA). As shown in FIG. 18, the levels of secreted IL2 were considerably different between the TFPs but all could secrete human IL2 into culture supernatant. A plasmid pYTL-KRTl-4 (WO 2005/068658) containing a TFPl-human IL2 gene was used as a control. TFP9, 13, 21 and 27 were found to be useful for the secretion of human IL2 (FIG. 18).<br>
EXAMPLE 12<br>
NOVEL TFP SELECTED FROM TFP LIBRARY FOR THE SECRETION OF HUMAN INTERLEUKIN-32<br>
[0126]	As an example for identifying optimal TFPs using a method developed<br>
in this invention, a rarely secreting protein, a novel human cytokine, interleukin-32a (hIL32) (Kim et al, Immunity 22:131 (2005)) was tested. An insert fragment containing the human IL32a gene and a 500 bp N-terminal part of SUC2 was amplified using PCR as described in Example 8 (FIG. 11). A PCR was carried out using a forward primer KR-IL32a-F (SEQ ID NO:59) and a reverse primer IL32a-INV-R (SEQ ID N0:60) from pProExHTa-IL32a (DY Yoon, Konkuk University, Korea) as a template.   Separately, another. PCR for the amplification of an N-terminal part ofSUC2 to be fused to the 3'-end of the IL32a gens was also carried out using a forward primer KR-Inv-F (SEQ ID NO:21) and a reverse primer Inv500-R (SEQ ID NO:22) from YGalNV (FIG. 6).    Then, a second PCR was done from the two DNA fragments amplified in the first step using a forward primer LNK40 (SEQ ID NO:23) and a reverse primer Inv500-R (SEQ ID NO:22).- . The resulting fragment (insert fragment) harbored a 40 nucleotide linker DNA containing a Kex2p recognition sequence (Leu-Asp-Lys-Arg (SEQ ID N0:214)), IL32a, an additional Kex2p recognition sequence, and an N-terminal part of invertase in order.   This fragment was co-transformed with the Swal digested TFP library vector constructed in Example 7 into S. cerevisiae Y28Q5AgalJAsuc2 (Mat a ura3 suc2::Tc!90 pep4::HIS3 gall canl).   Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 fig/ml antimycin A, and 2% agar) and incubated at 30°C for 5 days. Around 2&gt;
 <br>
cultivated on YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 105000*g. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-PAGE (FIG. 19). Most of the transformants could secrete human IL32cx judging from the protein bands appeared around 20 kDa. Among them, 17 transformants showing dark IL32a bands were further analyzed. Each transformant was cultivated on YPD broth and total DNA was isolated and retransformed into E. coli DH5a. Transformed E. coli was plated on LB media containing ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl,. with 50 jug/ml ampicillin) and incubated at 37°C overnight. Plasmids were isolated from each E. coli transformant using a plasmid extraction kit (Bioneer, Korea). To analyze the sequence of each plasmid, a sequencing primer GAL100-F (SEQ ID NO: 12) binding to the GAL10 promoter was used for all plasmids containing TFPs. Nucleotide sequences were determined by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA). The sequences were analyzed by a BLAST search of the Saccharomyces Genome Database (www.yeastgenome.org). As a result, nine different TFPs were identified from plasmids isolated from 17 selected yeast strains. The isolated plasmids were named pYHTS-IL32-TFP3, pYHTS-IL32-TFPll, pYHTS-IL32-TFP13, pYHTS-IL32-TFP21, pYHTS-IL32-TFP22, pYHTS-IL32-TFP25, pYHTS-IL32-TFP29 pYHTS-IL32-TFP34, and pYHTS-IL32-TFP38. Among them, TFPS, TFP13, TFP21 and TFP25 were commonly obtained as optimal TFPs for human IL2 (WO 2005/068658) and in Example 10 (Table 1). Five novel TFPs isolated for IL32cc are summarized in Table 2. Table 2. Novel TFPs for the secretion of human interleukin-32cc<br>
(Table Removed)<br>
EXAMPLE 13<br>
SECRETION OF HUMAN IL32a USING SELECTED TFPS<br>
[0127]	To confirm the secretion of human EL32cc using selected TFPs, several<br>
plasmids were constructed using PCR to remove the 5'-UTR of each TFP and SUC2 of selected plasmids in Example 12. Six forward primers, BamH-CIS-F , (SEQ ID NO:71)3 BamH-SED-F (SEQ ID NO:72), BamH-SIM-F (SEQ ID NO:73), BamH-YOR247W-F (SEQ ID NO:74), BamH-HSP-F (SEQ ID NO:75), BamH-OST-F (SEQ ID NO:76), and a common reverse primer IL32-TGA-R (SEQ ID NO:77) were used for PCR from- plasmids pYHTS-IL32-TFP3, pYHTS-IL32-TFPll, pYHTS-IL32-TFP13, pYHTS-IL32-TFP21, pYHTS-IL32-TFP22, and pYHTS-TL32-TFP255 respectively. The six PCR amplified fragments were digested with Ba?nHL and Sail and each fractionated from an agarose gel. Separately, another PCR to amplify the GAL promoter was done using a forward primer Sac-GAL-F (SEQ ID NO:57) and a reverse primer GAL-BamH-R (SEQ E&gt; N0:58) from YEGa-Hffi.525 (Sohn et at, Process Biochem. 30:653 (1995)). Sacl-BamHl digested GAL promoter and the six BaniSL-SalL digested fragments were co-ligated into Sacl-SaH. digested YEGa-HIR525. The resulting plasmids were named pYGT3-rL32a5 pYGTll-IL32a, pYGT13-IL32a, pYGT21-IL32a5 pYGT22-IL32a, and pYGT25-IL32a, respectively. Nucleotide sequences of all constructed vectors were confirmed to have a correct in-frame fusion between TFP and IL32a and<br><br>
each, vector was transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4;:HIS3 GAL1 can!}. Transformed cells were plated on UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 ammo acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days.  A single colony of each transformation was inoculated into YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours.  Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000xg.   The pellet was freeze-dried and resuspended in 1*SDS-PAGE sample buffer (Bio-Rad, USA) and analyzed on 12% SDS-PAGE.   The gel was stained with gel staining reagent (PhastGel® Blue R, Pharmacia Biotech, USA).  Secreted IL32a was. further analyzed by Western blotting using a monoclonal antibody of hIL32a. Proteins were transferred to PVDF membranes (Millipore, USA) in CAPS buffer (2.2 g per liter CAPS, MeOH 10%, pH 11 adjusted withNaOH) using a Mighty small tank transfer (Hoefer, USA) at 300 mA for 90 min.   Proteins were then detected with human IL32 antibody (DY Yoon, Konkuk University, Korea).  Membranes were blocked overnight at 4°C in PBS (137 mM NaCl, 2.7 mM KC1, 10 mM Na2HPO4, 2 mM KH2PO4, pH 7.4 adjusted with HC1) containing 5% skim milk.    Membranes were washed 3 times with PBS containing 0.05% Tween-20 and then incubated with primary antibodies diluted in PBS containing 3% skim milk at room temperature for 1 hour. Membranes were then washed 3 times and incubated with the anti-mouse secondary antibody (Sigma Chemical Co., USA) diluted in PBS containing 3% skim milk at room temperature for 1 hour.  Membranes were washed as above and developed with Sigma Fast NBT/BCIP (Sigma Chemical Co., USA).  As shown in FIG. 20, all selected TFPs could secrete human IL32a into the culture supernatant. Among them, TFPS, 13, 21 and 22 were found to be optimal for the secretion of human IL32a.<br>
EXAMPLE 14<br>
FED-BATCH FERMENTATION FOR THE PRODUCTION OF HUMAN<br>
IL32oc<br>
[0128] . A recombinant yeast strain transformed with pYGT3-IL32oc was cultured in a 5-L jar ferm.en.tor by fed-batch culture for the evaluation of the secretory productivity of human IL32a. A 200 ml seed culture was cultured in a 1 liter flask using a niinirnal medium (0.67% yeast nitrogen base without amino acids, 0.5% casamino acids and 2% glucose). When the culture using a fermentation culture medium (4% yeast extract, 1% peptone, 2% glucose) as an initial fermentation medium reached an OD600 of about 15, a fed-batch medium (15% yeast extract, 30% glucose) was supplied with different feeding., rates according to cell growth rates. After the culture reached an OD600 of about 130, galactose (30% galactose) was additionally supplied with different • feeding rates according to cell growth rates. After a culture period of about 72 hrs, the culture reached an OD600 of about 220 (FIG. 21A). 15 jal of the medium was collected at the given time points and assessed for secreted proteins by SDS-PAGE (FIG. 21B). Over 300 rag/L of hIL32a;.was found to be secreted into the culture medium as determined by the direct measurement of proteins with BCA protein assay reagent (Pierce, USA) and with a densitometer (GS700, Bio-Rad, USA).<br>
EXAMPLE 15<br>
SEQUENCE-BASED SELECTION OF TFPS USING BLAST SEARCH FROM YEAST GENOMIC DATABASE<br>
[0129]	For the sequence-based selection of TFPs from the yeast genome,<br>
amino acid sequences of pre-secretion signals of 18 selected TFPs (4 from WO 2005/068658, 9 from example 10 and 5 from example 12) were used as a query sequence for a BLAST search of the Saccharomyces Genome Database<br>
(www.yeastgenome.org). Using a low expect threshold (100 or 1000) in the BLASTP search, several hundred ORFs having over 70% homology were identified. Of those, the ORFs with sequence homology near the N-terminus were        selected,        and        further        subjected        to        SignalP (www.cbs.dta. dk/services/SignalP-2.0/) analysis for the selection of ORFs with secretion signal.  As a result, 18 ORFs were randomly selected as TFP candidates. Eighteen selected ORFs identified by the search were YGR279C (SCW4, cell wall protein), YLR037C (DAN2, cell wall mannoprotein), YLR110C   (CCW12,   cell   wall   protein),   YOR383C' (FITS,   cell   wall mannoprotein),   YIL011W   (T1R3,   cell  wall  mannoprotein),   YHR214W (putative membrane protein), YNL160W (YGP1, cell wall-related secretory glycoprotein), YGR296C-A (dubious open reading frame), YOL154W (ZPS1, . putative GPI-anchored protein), YPL187W (MFot, mating pheromone alpha--. factor), YHR214W (putative membrane protein), YKR.013W (PRY2, protein of unknown function), YHR139C (SPS100, protein required for spore wall maturation), YIL169C (putative protein of unknown function), YOL155C (uncharacterized    ORF),    YMR325W    (PAU19,    hypothetical    protein), YDR134W (hypothetical protein) and YLR300W (EXGl, major exo-l,3-beta-glucanase of the cell wall). Each ORF was amplified from the genornic DNA of £ cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canT) using PCR primer pairs YGR279C-F(SEQ ID NO:92) and YGR279C-R (SEQ ID NO:93) for YGR279C, YLR037C-F (SEQ ID NO:94) and YLR037C-R (SEQ ID N0:95) for YLR037C, YLR110C-F (SEQ ID N0:96) and YLR110C-R (SEQ ID N0:97) for YLR110C, YOR383C-F (SEQ ID N0:98) and YOR383C-R (SEQ ID NO:99) for YOR383C,  Y3L011W-F  (SEQ ID N0:100)  and YIL011W-R (SEQ ID NO:101) for YIL011W,  YHR214W-F (SEQ ID N0:102) and YHR214W-R (SEQ ID NO:103) for YHR214W, YNL160W-F (SEQ ID NO: 104) and YNL160W-R (SEQ ID NO:105) for YNL160W, YGR296C-A-F (SEQ ID N0:106) and YGR296C-A-R (SEQ ID NO:107) for YGR296C-A, YOL154W-F (SEQ ID N0:108) and YOL154W-R (SEQ ID NO:109) for YOL154W, YPL187W-F (SEQ ID NO: 110) and YPL187W-R<br>
(SEQ ID NO:111) for YPL187Wf YHR214W-F (SEQ 3D N0:112) and YHR214W-R (SEQ ID N0:113) for YHR214W, YKR013W-F (SEQ ID NO:114) and YKR013W-R (SEQ ID NO:115) for YKR013W, YHR139C-F (SEQ ID N0:116) and YHR139C-R (SEQ ID NO-.117) for YHR139C, YIL169C-F (SEQ ED NO: 118) and YTL169C-R (SEQ ID NO:119) for YTL169C, YOL155C-F (SEQ ID N0:120) and YOL155C-R (SEQ ID N0:121) for YOL155C, YMR325W-F (SEQ ID NO:122) and YMR325W-R (SEQ ID N0:123) for YMR325W, YDR134W-F (SEQ ID NO:124) and YDR134W-R (SEQ ID NO: 125) for YDR134W and YLR300W-F (SEQ ID NO: 126) and YLR300W-R (SEQ ID NO: 127) for YLR300W, respectively. PCR was performed with Pfit polymerase (Stratagene, USA) and PCR . conditions included one cycle of 94°C for 3 min, and 25 cycles of 94°C for 30 • sec, 55°C for 30 sec and 72°C for 2 min, followed by one final cycle of 72°C-' for 7 min. Each amplified PCR fragment was confirmed by nucleotide sequencing by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA).<br>
[0130]	For the screening of TFPs from the selected 18 ORFs, unidirectional<br>
deletion of the mixture of 18 PCR fragments was carried out and-used for the'. construction of a TFP library in YGadV45 (FIG. 24).- Single stranded template was obtained by unidirectional PCR using a primer SfiA-F (SEQ ID NO: 128) from the template consisting of the 18 ORFs. PCR was performed with ExTaq (Takara Korea, Korea) and PCR conditions included one cycle of 94°C for 3 min, and 30 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 min, followed by one final cycle of 72°C for 7 min. PCR product containing the single stranded DNA was purified using a PCR purification kit (Bioneer, Korea). Then, the regeneration of double stranded DNA was carried out using E, coli DNA polymerase I (NEB, England) and a random hexameric primer, ASA24N6 (SEQ ID NO: 16). A reaction mixture containing 20 ul of template DNA, 1 ul of ASA24N6 primer, 3 ul of lOx E. coli DNA poll buffer, 5 ul of 2.5 mM dNTP, and 1 jil of E. coli DNA poll was incubated at 37°C for 1 hour. The DNA was column purified using a PCR purification kit (Bioneer,<br>
Korea) and PCR amplified using primers SfiA-F (SEQ ID NO: 128) and ASA24 (SEQ ID NO: 17). The amplified DNA was column purified again, digested with Sfil and fractionated by agarose gel electrophoresis. 0.5-1.0 kb ofSfil digested DNA was subcloned into the SfiL treated YGadV45 containing a defective SUC2 (dSUC2). The ligated DNA was transformed into E. coli DH5a. Transformed E. coli was plated on LB media with ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 ug/rnl ampicillin) and incubated at 37°C overnight About IxlO4 E. coli colonies were pooled with sterile distilled water and the total plasmids containing the unidirectional-deleted DNA fragment library of 18 ORFs in YGadV45, were isolated by using a plasmid isolation kit (Bioneer, Korea).<br>
[0131]	To screen proper TFPs from the unidirectional-deleted DNA fragment-<br>
library of 18 ORFs, a gene encoding human interleukin-2 (hIL2) was inserted' . • between the library and dSUC2. An insert fragment containing the hIL2 gene and a 500 bp N-terminal part of SUC2 was amplified using PCR as described in Example 8 (FIG. 11). This fragment was co-transformed with Swai digested vector containing the unidirectional-deleted DNA fragment library of 18 ORFs into S. cerevisiae Y2%Q5&amp;gaJl&amp;suc2 (Mat a uraS-. suc2::Tcl90 , . pep4::HIS3 gall canl). Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 jj.g/rnl antimycin A, and 2% agar) and incubated at 30°C for 5 days. Around 2xl04 transformants were obtained on UD plates but about several hundred transformants were obtained in YPSGA. A random selection of 29 transformants growing on YPSGA was cultivated on YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000xg. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-<br>
PAGE (FIG. 25).   Several transformants were found to secrete human IL-2 into the culture supernatant. Total DNA was isolated from each cell secreting human IL-2 and retransformed into E. coli DH5a.   Plasmids were isolated from, each E. coli transformant using a plasmid extraction kit (Bioneer, Korea). To analyze the sequence of each TFP, a sequencing primer GAL10Q-F (SEQ ID NO: 12) binding to the GAL10 promoter was used for all plasmids containing TFPs.   Nucleotide sequences were determined by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA).   The sequences were analyzed by a BLAST      search      of     the      Saccharomyces      Genome      Database (www.yeastgenorne.org).   As a result, six novel TFPs were identified from , plasmids isolated from the 12 transformants secreting human IL-2.    The. isolated plasmids were named pYTL-TFP39, pYEL-TEP41, PYIL-TFP431, pYTL-TFP44, pYIL-TFP52, and pYTL-TFP54, respectively.   The six novel TFPs are summarized in Table 3.<br>
Table 3. TFPs from sequence-based selected ORFs for the secretion of human IL-2<br><br>
(Table Removed)<br>
EXAMPLE 16 DIVERSIFICATION OF CORE-TFPS BY UNIDIRECTIONAL DELETION<br>
[0132]	To diversify the usefulness of 14 TFPs (core-TFPs) selected by using<br>
IL-2 and IL-32a in Examples 10 and 11, and 3 TFPs previously identified in WO 2005/068658, seventeen genornic ORFs, YAR066W for TFP-1, YFR026C for TFP-2, YJL158C for TFPS, YGR106C for TFP-9, YDR077W for TFP-11, YIL123W for TFP13, YNL190W for TFP-17, YBR078W for TFP18, YJL178C for TFP-19, YMR307W for TFP-20, YOR247W for TFP-21, YJL159W for TFP-22, YOR085W for TFP-25, YKR042W for TFP-27, YEL060C for TFP29, YLR390W-A for TFP-34, and YMR251W-A for TFP-38, were PCR amplified and unidirectionally deleted as described in Example" 15. Each ORF was amplified from the genornic DNA of S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canl) using PCR primer pairs YAR066W-F (SEQ ID NO: 141) and YAR066W-R (SEQ ID NO: 142) for YAR066W, YFR026C-F (SEQ ID NO: 143) and YFR026C-R (SEQ ID NO:144) for YFR026C, YJL158C-F (SEQ ID NO:145) and YJL158C-R (SEQ ID N0:146) for YJL158C, YGR106C-F (SEQ ID N0:147) and YGR106C-R (SEQ ID NO: 148) for YGR106C, YDR077W-F (SEQ ID NO: 149) and YDR077W-R (SEQ ID NO: 150) for YDR077W, YDL123W-F (SEQ ID N0:151) and YIL123W-R (SEQ ID NO:152) for YTL123W, YNL190W-F (SEQ ID NO: 153) and YNL190W-R (SEQ ID NO:154) for YNL190W, YBR078W-F (SEQ ID NO:155) and YBR078W-R (SEQ ID N0:156) for YBR078W, YJL178C-F (SEQ ID NO:157) and YJL178C-R (SEQ ID NO.-158) for YJL178C, YMR307W-F (SEQ ID N0:159) and YMR307W-R (SEQ ID NO:160) for YMR307W, YOR247W-F (SEQ ID N0:161) and YOR247W-R (SEQ ID NO: 162) for YOR247W, YJL159W-F (SEQ ID N0:163) and YJL159W-R (SEQ ED NO: 164) for YJL159W, YOR085W-F (SEQ ID NO:165) and YOR085W-R (SEQ ID NO:166) for YOR085W, YKR042W-F (SEQ ID N0:167) and YKR042W-R (SEQ ID N0:168) for<br>
YKR042W, YEL060C-F (SEQ ID NO: 169) and YEL060C-R (SEQ ID N0:170) for YEL060C, YLR390W-A-F (SEQ ID N0:171) and YLR390W-A-R (SEQ ID N0:172) for YLR390W-A, YMR251W-A-F (SEQ ID N0:173) and YMR251W-A-R (SEQ ID NO: 174) for YMR251W-A, respectively. PCR was performed with Pfu polymerase (Stratagene, USA) and PCR conditions included one cycle of 94°C for 3 min, and 25 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 min, followed by one final cycle of 72°C for 7 min. Each amplified PCR fragment was confirmed by nucleotide sequencing by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA).<br>
[0133]	For the screening of diversified TFPs from the 17 ORFs from which 17<br>
core-TFPs were obtained, unidirectional deletion of the mixture of 17 PCR., .fragments was carried out and used for the construction of a TFP library in YGadV45 (FIG. 24). Single stranded template was obtained by unidirectional PCR using a primer SnA-F (SEQ ID NO: 128) from the template consisting of 117 ORFs. PCR was performed with ExTaq (Talcara Korea, Korea) and PCR conditions included one cycle of 94°C for 3 min, and 30 cycles of 94°C for 30 sec, 55°C for 30 sec and 72°C for 2 min, followed by one final" cycle of 72°C for 7 min. PCR product containing the single stranded DNA was purified using a PCR purification kit (Bioneer, Korea). Then, the regeneration of double stranded DNA was carried out using E. coli DNA polymerase I (NEB, England) and a random hexameric primer, ASA24N6 (SEQ ID NO:16). A reaction mixture containing 20 ul of template DNA, 1 ul of ASA24N6 primer, 3 ul of lOx E. coli DNA poll buffer, 5 ul of 2.5 mM dNTP, and 1 ul of E. coli DNA poll was incubated at 37°C for 1 hour. The DNA was column purified using a PCR purification kit (Bioneer, Korea) and PCR amplified using primers SfiA-F (SEQ ID NO: 128) and ASA24 (SEQ ID NO:17)/ The amplified DNA was column purified again, digested with Sfil and fractionated by agarose gel electrophoresis. 0.5-1.0 kb of Sfil digested DNA was subcloned into Sfil treated YGadV45 containing a defective SUC2 (dSUC2). The ligated DNA was transformed into E. coli DH5a. Transformed E. coli<br>
was plated on LB media with ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 u-g/ml ampicillin) and incubated at 37°C overnight. About IxlO4 E. coli colonies were pooled with sterile distilled water and the total plasmids containing the unidirectional-deleted DNA fragment library of 17 ORFs in YGadV45 were isolated by using a plasmid isolation kit (Bioneer, Korea). Two unidirectional-deleted library DNAs from 17 ORFs for core-TFPs and 18 ORFs prepared in Example 15 were combined for further application.<br>
[0134]	To screen proper TFPs from the unidirectional-deleted DNA fragment<br>
library from 35 ORFs, a gene encoding human interleukin-2 (hTL2) was inserted between the library and dSUC2. An insert fragment containing the human IL2 gene and a 500 bp N-terminal part of SUC2 was amplified using, PCR as described in Example 8 (FIG. 11). This fragment was co-transformed" with Swal digested vector containing the unidirectional-deleted DNA fragment library of 35 ORFs, into S. cerevisias Y2805Aga/JAswc2 (Mat a ura.3 suc2::Tc!90 pep4::HIS3 gall canT). Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (l%,yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 ug/ml antimycin A, and 2% agar) and incubated at 30°C for 5 days. Around 2*104 transformants were obtained on UD plates but about several hundred transformants were obtained in YPSGA. A random selection of 24 transformants growing on YPSGA was cultivated on YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 103000xg. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by , 12% SDS-PAGE (FIG. 26). Most of the transformants could secrete human IL-2 into the culture supernatant but with different levels between them. Total DNA was isolated from each transformant secreting human IL-2 and<br>
retransformed into E. coli DH5a. Plasmids were isolated from E. coll using a plasmid extraction kit (Bioneer, Korea). To analyze the sequence of each TFP, a sequencing primer GAL100-F (SEQ ID NO: 12) binding to the GAL10 promoter were used for all plasmids containing TFPs. Nucleotide sequences were determined by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA). The sequences were analyzed by a BLAST search of the Saccharomyces Genome Database (www.yeastgenome.org). As a result, six novel TFPs were identified from plasmids isolated from the 18 transforrnanis secreting human IL-2. The isolated plasmids were named pYTL-TFP40s pYTL-TFP5Q, pYTL-TFP51, pYIL-TFP57, pYIL-TFP58, and pYIL-TFP59, respectively. The six novel , TFPs are summarized in Table 4.<br>
Table 4. TFPs from sequence-based selected ORFs for the secretion of humatf IL-2<br><br>
(Table Removed)<br>
EXAMPLE 17<br>
ARTIFICIAL TFPS USING SWAPPING OF PRE AND PRO SIGNAL SEQUENCE BETWEEN CORE-TFPS<br>
[0135]	To date, a yeast secretion signal from mating factor alpha (MFa) has<br>
been the most widely used for the secretion of various recombinant proteins in yeast (Romanos et al, Yeast 5:423 (1992)). The secretion signal comprises 19 ammo acids of pre-signal and 66 amino acids of pro-signal. The exact<br>
function of pro-signal is uncertain but it has been known to be essential for the correct folding and secretion of some proteins. The fact was also investigated in the secretion of some recombinant proteins in yeast (Chaudhuri et al, Eur. J. Biochem. 206:193 (1992)). In this invention, two secretion signals, TFP-3 and TFP-22, were identified as pre-pro type. For the expansion of the usefulness of TFPs selected in this invention, artificial TFPs were designed to have a different origin of the pre and pro signals. Four artificial TFPs were constructed using the pre-sigaal of TFP-l,2,3 and 4 and a common pro-signal of mating factor alpha and the resulting TFPs were named as TFP-5, 6, 7, and 8. For the fusion between 4 different pre-signals and a common pro signal, overlap extension PCR was used.<br>
[0136]	A first step PCR was carried out for the amplification of four different<br>
pre-signals of 4 TFPs using primer pairs Tl-F (SEQ ID NO:187) and Tl-R (SEQ ID NO:188), T2-F (SEQ ID NO:189) and T2-R (SEQ ID NO: 190), T3-F (SEQ ID NO: 191) and T3-R (SEQ ID NO:192), T4-F (SEQ ID NO:193) and T4-R (SEQ ID NO: 194) from plasmids pYTL-KRTFPl, 2, 3, and 4 (WO 2005/068658), respectively. Separately, another PCR for the amplification of about 190 bp of mating factor alpha pro-signal was also carried out using primers MF-Pro-F (SEQ ID NO: 195) and MF-R (SEQ ID,NO: 196) from plasmid YEGa-HTR525. Then second PCRs for the 4 different pre-pro signals were done from 4 sets of two DNA fragments, 4 pre-signals and a MFa pro-signal amplified in the first step using 4 different forward primers, Tl-F (SEQ ID N0:187), T2-F (SEQ ID NO: 189), T3-F (SEQ ID NO:191) and T4-F (SEQ ID NO: 193) and a common reverse primer, MF-R (SEQ ID NO: 196), respectively. To compare the efficiency of each artificial pre-pro signal sequence with that of mating factor alpha, pre-pro signal of mating factor alpha was also PCR amplified using primers MF-Pre-F (SEQ ID NO: 197) and MF-R (SEQ ID NO: 196) from YEGa-HTR525.<br>
[0137]	A target protein, human insulin-like growth factor (hIGF) was selected<br>
to test the five pre-pro signal sequences. It has been reported that the pro signal of mating factor alpha was necessary for the secretion of human insulin-<br>
like growth factor in yeast (Chaudhuri et al, Eur. J. Biochem. 206:193 (1992)). Human IGF gene was first PCR amplified using primers KR-IGF-F (SEQ ID NO:198) and IGF-R (SEQ ID NO:199) from a human cDNA library (ES Choi, Korea Research Institute of Bioscience and Biotechnology, Korea) and then a second PCR was done using LNK40 (SEQ ID N0:23) and IGF-R (SEQ ID NO: 199).   The DNA fragment containing IGF was fused to the previously amplified 5 PCR fragments containing pre-pro signals using 5 forward primers, Tl-F (SEQ ID N0:187), T2-F (SEQ ID NO.189), T3-F (SEQ ID N0:191), T4-F (SEQ ID NO: 193), MF-Pre-F (SEQ ID NO:197) and a common reverse primer IGF-R (SEQ ID NO: 199).  All fused PCR products were digested with Sfil and Sail and then subcloned into the Sfil-Sali digested vector YGallNV (FIG. 6).   The resulting plasmids were named pYGa-Tlcc-IGF, pYGa-T2a-IGF pYGa-T3a-IGF pYGa-T4a-IGF and pYGa-MFoc-IGF,. respectively. Five plasmids were transformed into S. cerevisiae Y2805 (Mat a ura.3 pep4::HIS3 gall ca.nl).   Transformed cells were spread on UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar).    A single colony of each transformation was isolated and cultivated in YPDG (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000xg.   The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-PAGE.   Secreted IGF was further analyzed by Western blotting using an antibody for hIGF (FIG 27). All tested pre-pro secretion signals could secrete human IGF into the culture supernatant but with different efficiencies. Among 5 pre-pro signals, T3a (pre-signal from TFP-3 and pro-signal from MFa) and T4a (pre-signal from TFP-4 and pro-signal from MFa) were found to be effective for the secretion of human IGF. The four artificial TFPs and a novel TFP are summarized in Table 5.<br>
Table 5. Novel TFPs for the secretion of human IGF<br><br>
(Table Removed)<br>
EXAMPLE 18<br>
CONSTRUCTION OF THE SELECTED TFP VECTORS APPLICABLE TO MANY TARGET GENES THROUGH IN VWO RECOMBINATION<br>
[0138]	Thirty five TFPs (core-TFPs) selected in this invention (4 TFPs from<br>
WO 2005/068658, 14 TFPs selected using two reporter proteins, human IL2 and IL32a in Example 10 and 11, 6 TFPs from ORFs selected by BLAST search in Example 15, 6 TFPs from unidirectional deletion of ORFs encoding the pre-selected TFPs in Example 16, 5 TFPs from artificial design of TFPs in Example 17) might be also useful for the secretion of other proteins. To apply such vectors to large numbers of target genes, the core-TFP vectors were reconstructed for in vivo recombination with target genes. For the construction of plasmid YGaSW, a PCR for the amplification of 170 bp fragment containing an EcoRl, 2 Sfil, Notl, a linker DNA containing a Kex2p recognition site, Swal and Sali site was carried out using primers GAL100-F (SEQ ID N0:12) and H77-1-R (SEQ ID NO:78) firom YGadV45 (FIG. 10). An EcoRL-SalL digested PCR fragment was subcloned into EcdKL-SalL digested YGadV45 and the resulting plasmid was named YGaSW. The<br>
plasmid harbors restriction sites for EcoBl, Sfil, Notl, Sfil, a 40 bp linker and<br>
^ restriction sites Swal and Sail between the GAL10 promoter and the GAL7<br>
terminator.   Thirty five core-TFPs were obtained by the Sfil digestion of plasmids containing each TFP. Each core-TFP was gel purified and subcloned<br><br>
into Sfil digested YGaSW and the resulting 35 plasmids were named YGaSW-TFP1, YGaSW-TFP2, YGaSW-TFP3, YGaSW-TFP4, YGaSW-TFP5, YGaSW-TFP6, YGaSW-TFP7, YGaSW-TFPS, YGaSW-TFP9, YGaSW-TFP11, YGaSW-TFPIS, YGaSW-TFP17, YGaSW-TFPIS, YGaSW-TFP19, YGaSW-TFP20, YGaSW-TFP21, YGaSW-TFP22, YGaSW-TFP25, YGaSW-TFP27, YGaSW-TFP29, YGaSW-TFP32 YGaSW-TFP34, YGaSW-TFP38, YGaSW-TFP39,'YGaSW-TFP40, YGaSW-TFP43, YGaSW-TFP44, YGaSW-TFP48, YGaSW-TFPSO, YGaSW-TFPSl, YGaSW-TFP52, YGaSW-TFP54, YGaSW-TFP57, YGaSW-TFP58, and YGaSW-TFP59, respectively.<br>
EXAMPLE 19<br>
EVALUATION OF SELECTED CORE-TFPS FOR THE SECRETION OF HUMAN GROWTH HORMONE<br>
[0139]	Core-TFPs selected in this invention were tested for the secretion of<br>
human growth hormone (hGH). The human GH gene was PCR amplified from a human cDNA library (ES Choi, Korea Research Institute of Bioscience and Biotechnology, Korea) using primers hGH-F (SEQ ID N0:79) and hGH-R (SEQ ID NO: 80) and subcloned into pST-Bluel (Novagen, USA). The resulting plasmid was named pST-hGH. A second PCR was carried out using primers KR-hGH-F (SEQ ID NO:81) and hGH-Sal-R (SEQ ID NO:82) from pST-hGH. The PCR product containing the hGH gene was used for a third PCR using primers LNK40 (SEQ ID NO:23) and GT70-R (SEQ ID NO:83) to add homologous sequences with YGaSW-TFP vectors constructed in Example 18. The amplified PCR fragment was mixed 2:1 with Swal digested YGaSW-TFP vectors and transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canl) through in vivo recombination. Transformed cells were plated on UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days. A single colony of each transformation was inoculated into YPDG<br>
broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) . and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000xg. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-PAGE. As shown in FIG. 22, most TFPs could secrete human growth hormone into the culture supernatant. Among them, a strain with pYGT21-hGH was tested for the secretion level during fed-batch fermentation. Ten microliters of culture supernatant sampled at the indicated time points were analyzed by SDS-PAGE (FIG. 23). Around 500 mg/liter of human growth hormone was secreted into f, the culture supernatant.<br>
EXAMPLE 20<br>
EVALUATION OF SELECTED CORE-TFPS FOR THE SECRETION OF HUMAN CASPASE-1 SUBUNIT P10<br>
[0140]	Core-TFPs selected in this invention were tested for the secretion of<br>
human caspase-1 subunit plO(hPlO). The human plO gene was PCR amplified from a human cDNA library (ES Choi, Korea Research Institute of Bioscience and Biotechnology, Korea) using primers KR-hPlO-F (SEQ ID NO:210) and hPlO-Sal-R (SEQ ID NO:211). The PCR product containing the hPIO gene was used for a second PCR using primers LNK40 (SEQ ID NO:23) and GT70-R (SEQ ID N0:83) to add homologous sequences with YGaSW-TFP vectors constructed in Example 18. The amplified PCR fragment was mixed 2:1 with Swal digested YGaSW-TFP vectors and transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canT) through in vivo recombination. Transformed cells were plated on UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days. A single colony of each<br>
transformation .was inoculated into YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000*g. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-PAGE. As shown in FIG. 28, only 4 artificial TFPs containing pre-pro signals could secrete hPIO protein into the culture supernatant. As found in the case of MGF, pro-signal was necessary for the proper secretion of human caspase-1 subunit PI 0 in yeast.<br>
EXAMPLE 21<br>
EVALUATION OF SELECTED CORE-TFPS FOR THE SECRETION OF HUMAN INTERLEUKIN-32y<br>
[0141]	Core-TFPs selected in this invention were tested for the secretion of<br>
human interleukin-32y (hIL32y). A gene coding for human interleukin 32 splicing variant gamma was PCR amplified from pGMT-IL32y (DY Yoon, Konkuk University, Korea) using primers KR-hIL32g-F (SEQ ID NO:212) and hTL32g-Sal-R (SEQ E) NQ:213). The PCR product containing the hIL32y gene was used for a second PCR using primers LNK40 (SEQ ID NO:23) and GT70-R (SEQ ID N0:83) to add homologous sequences with YGaSW-TFP vectors constructed in Example 18. The amplified PCR fragment was mixed 2:1 with Swal digested YGaSW-TFP vectors and transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 can!) through in vivo recombination. Transformed cells were plated on UD media (0.67% yeast nitrogen base without arnino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days. A single colony of each transformation was inoculated into YPDG broth (1% yeast extract, 2% Bacto-peptone, 1% glucose and 1% galactose) and cultivated at<br>
. 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000xg. The pellet was freeze-dried and resuspended in IxSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed by 12% SDS-PAGE. Among the tested TFPs, TFP3 and TFP27 were identified to be effective for the secretion of human IL-32y (FIG. 29).<br>
EXAMPLE 22<br>
TFP LIBRARY FROM PICHIA PASTORIS SELECTED IN SACCHAROMYCES CEREVISIAE<br>
[0142]	The TFP selection method of this invention could also be applied to<br>
other sources of the genomic or cDNA library. As an example of mRNA sources, the yeast P. pastons was tested. Total RNA was isolated from yeast P. pastons GS115 (Invitrogen, USA) for the construction of a cDNA library. Yeast was cultivated to mid-exponential phase in YPD media (2% yeast extract, 1% Bacto-peptone and 2% glucose). Total RNA was isolated from P. pastons by a method described in Elion et al. (Elion et at, Cell 39'663 (1984)). Purification of Poly(A)+ mRNA from total RNA was carried out using an Oligotex mRNA kit (Qiagen, Germany). cDNA was synthesized from the isolated mRNA using a SMART cDNA synthesis kit (BD Bioscience, USA). A specially designed primer ASA24N6 (SEQ ID NO:16) was used for the synthesis of the first strand cDNA instead of a primer included hi the SMART kit as described in Example 4 (FIG. 8). Primer ASA24N6 could randomly bind to any position of mRNA due to its random hexameric sequence. Thus, most of first stranded cDNA amplified using this method contained the 5' partial sequence encoding the N-terminal part of 3'east genes. The first stranded cDNA library with 5' partial sequence was used as a PCR template for double stranded cDNA synthesis with the 5' PCR<br>
primer of the SMART Kit (BD Bioscience, USA) and primer ASA24 (SEQ ID NO: 17).   PCR products produced using this method contain numerous 5' partial fragments of cDNA with Sfil sites at both ends.   PCR conditions included one cycle of 95°C for 20 sec, and 20 cycles of 95°C for 30 sec, 68°C for 6 min as recommended in the kit.   Amplified cDNA was treated with phenol/chloroform/isoamyl alcohol (25:24:1) and precipitated with 2 volumes of ethanol and 0.1 volume of 3 M sodium acetate (pH 5.0). Recovered cDNA was digested with Sfil at 50°C for 2 hours and then fractionated using agarose gel electrophoresis.  0.5 to 1 kb DNA was isolated from the gel^using a gel extraction kit (Bioneer, Korea).    Extracted DNA was ligated into a Sfil digested YGalNV vector and transformed into E. coli DH5oc. Transformed E. coli was plated on LB media with ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with 50 ug/ml ampicillin) and incubated at 37°C overnight About 4*104 E. coli colonies were pooled with sterile distilled water and the total plasmids containing the cDNA library synthesized by random primer fused to the SUC2 gene were isolated by using a plasmid isolation kit (Bioneer, Korea). For the selection of a TFP library secreting invertase from yeast P. pastoris, library DNA was transformed into S. cerevisiae Y2805 AgallAsuc2(Mat a ura3 suc2::Tc!90 pep4::HIS3 gall canl) according to a lithium acetate method (Hill et a/., Nucleic Acids Res. 19:5191 (1991)). Transformed cells were spread on both UD media (0.67% yeast nitrogen base without amino acids, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and YPSGA media (1% yeast extract, 2% Bacto-peptone, 2% sucrose, 0.3% galactose, 1 fig/ml antintycin A, and 2% agar) and incubated at 30°C for 4 to 6 days. Around 1,000 transformants were obtained from the cDNA library of P. pastoris.     Five different transformants grown on YPSGA media were randomly selected and total DNA was isolated from cultured cells of each colony using glass beads.   Then the DNA was precipitated with ethanol. Isolated DNA was retransformed into E. coli DH5cc. E. coli was plated on LB media with ampicillin (1% Bacto-peptone, 0.5% yeast extract, 1% NaCl with<br>
50 µg/ml ampicillin) and incubated at 37°C overnight. Plasmids were isolated from transformed E. coli using a plasrnid isolation kit (Bioneer, Korea).  To analyze the sequence of each TFP obtained from the cDNA of P. pastoris, a • sequencing primer GAL100-F (SEQ ID NO: 12) binding to the GAL10 promoter was used. Nucleotide sequences were determined by Genotech Co. (Taejon, Korea) using an automated sequencing unit (ABI Prism 377; PE Biosystems, Foster City, CA, USA). The sequences were tested on a BLAST search of the National Center for Biotechnology Information (NCBT) sequence database (www.ncbi.nlm.nih.gov).   As a result, four different TFPs of P. pastoris were identified from plasrnids isolated from 5 selected strains.  The isolated plasmids were named pYHTS-PpTFPl, pYHTS-PpTFP2, pYHTS-- . PpTFPS, and pYHTS-PpTFP4. The four TFPs isolated from P. pastoris are., summarized in Table 6. Table 6. Isolated TFPs from Pichia pastoris<br><br>
(Table Removed)<br>
EXAMPLE 23<br>
EVALUATION OF TFPS FROM PICHIA PASTORIS USING HUMAN IL2<br>
[0143]	Four Pichia pastoris TFPs summarized in Table 6 were tested for their<br>
secretion efficiency in S. cerevisiae using human IL-2. Each PpTFP was PCR amplified using primer pairs, PpTFPl-F (SEQ ID NO:227) and PpTFPl-R (SEQ ID NO.-228), PpTFP2-F (SEQ ID N0:229) and PpTFP2-R (SEQ ID N0:230), PpTFP3-F (SEQ ID NO:231) and PpTFP3-R (SEQ TD N0:232), PpTFP4-F(SEQ ID NO:233) and PpTFP4-R (SEQ ID NO:234) from plasmids,<br><br>
pYHTS-PpTFPl, pYHTS-PpTFP25 pYHTS-PpTFP3, and pYHTS-PpTFP4, respectively. Gel-purified PCR fragments were digested with Sfil and subcloned into Sfil digested YGaSW vector (FIG. 10) and the resulting plasmids were named as YGaSW-PpTFPl, YGaSW-PpTFP2, YGaSW-PpTFPS, and YGaSW-PpTFP4, respectively.<br>
[0144]	The amplified PCR fragment containing human IL-2 gene harboring<br>
homologous sequences with YGaSW-PpTFP vectors, was 2:1 mixed with Swal digested YGaSW-PpTFP vectors and transformed into S. cerevisiae Y2805 (Mat a ura3 SUC2 pep4::HIS3 GAL1 canl) through in vivo recombination. Transforming cells were plated on UD media (0.67% yeast nitrogen base without amino acid, 0.77 g/1 amino acid mixture, 2% glucose and 2% agar) and incubated at 30°C for 3 days. A single colony of each transformation was inoculated into YPDG broth (1% yeast extract, 2% bacto-peptone, 1% glucose and 1% galactose) and cultivated at 30°C for 40 hours. Culture supernatant (0.6 ml) was mixed with cold acetone for a final acetone concentration of 40%. After incubation at -20°C for 2 hours, proteins were precipitated by centrifugation for 15 min at 10,000 xg. The pellet was freeze-dried and resuspended on 1 xSDS-PAGE sample buffer (Bio-Rad, USA) and analyzed in 12% of SDS-PAGE. As shown in FIG. 30, all PpTFPs secreted human interleuldn-2 into culture supernatant, suggesting the compatibility of TFP between two yeasts.<br>
[0145]	Having now folly described the invention, it will be understood by<br>
those of skill in the art that the same can be performed within a wide and equivalent range of conditions, formulations, and other parameters without affecting the scope of the invention or any embodiment thereof. All patents, patent applications and publications cited herein are fully incorporated by reference herein in their entirety.<br>
(Sequence Removed)<br><br><br><br><br><br>
We Claim:<br>
1.        A in-vitro method of identifying a target protein specific TFP library, said method comprising:<br>
(i) co-transforming a plurality of reporter protein deficient host cells with (1) a<br>
plurality of linear vectors and (2) a nucleic acid encoding a target protein to<br>
produce a plurality of transformed host cells, wherein the linear vector and<br>
nucleic acid encoding a target protein are introduced into their genome;<br>
wherein each of said linear vectors comprises a nucleic acid fragment from a<br>
library of nucleic acid fragments and a nucleotide sequence encoding a N-<br>
terminal  amino  acid-deleted  reporter protein,  wherein  said  nucleic  acid<br>
encoding a target protein comprises, at the 3' end, a nucleotide sequence<br>
encoding the N-terminal amino acids deleted from said reporter protein in said<br>
linear vector, and at the 5' end, a linker DNA, and wherein said target protein<br>
is a protein that is difficult to produce using conventional  recombinant<br>
production methods;<br>
(ii) incubating said plurality of transformed host cells of step (i) to allow in<br>
vivo recombination of said linear vectors and said nucleic acid encoding a<br>
target protein;<br>
(iii) identifying cells showing an activity of the reporter protein from the<br>
plurality of transformed host cells of (ii); and<br>
(iv) identifying a TFP library from the cells identified in (iii); wherein said<br>
TFP library comprises nucleic acid fragments which individually induce the<br>
secretion of said target protein, and<br>
wherein the transformed host cells are selected from plant, bacterial, fungal.<br>
yeast, or animal cells.<br>
2. The method as claimed in claim 1. wherein said library of nucleic acid fragments is from genomic DNA or cDNA of a plant, bacteria, yeast, fungus, or animal or a recombinant DNA.<br><br>
3.	The method as claimed in claim 1. wherein said library of nucleic acid fragments is a<br>
library of pre-selected candidate TFPs.<br>
4.	The method as claimed in claim 3. wherein said library of pre-selected candidate<br>
TFPs is obtained by transforming a plurality of reporter protein-deficient host cells<br>
with a variety of vectors comprising a library of nucleic acid fragments and a nucleic<br>
acid encoding a reporter protein, collecting cells that grow, isolating vectors from the<br>
cells, and isolating nucleic acid fragments from the vectors, thereby obtaining a TFP<br>
library comprising nucleic acid fragments which individually induce secretion of the<br>
reporter protein.<br>
5.	The method as claimed in claim 3. wherein said library of pre-selected candidate<br>
TFPs is derived from sequences identified in a genome database by searching for (i)<br>
genes containing a pre-secretion signal homologous with those of one or more<br>
previously identified TFPs; (ii) genes comprising a secretion signal sequence, or (iii)<br>
genes encoding proteins passing through the endoplasmic reticulum.<br>
6.	The method as claimed in claim 3. wherein said library of pre-selected candidate<br>
TFPs is obtained by diversifying previously identified TFPs.<br>
7.	The method as claimed in claim 3. wherein said library of pre-selected candidate<br>
             TFPs is obtained by artificially designing nucleic acid fragments to have the pre and<br>
pro signal sequence swapped between previously identified TFPs,<br>
8.	The method as claimed in claim 3. wherein said library of pre-selected candidate<br>
TFPs is a library of core TFPs wherein the core TFPs are a collection of previously<br>
identified TFPs that are effective for one or more target proteins.<br>
9.	The method as claimed in claim 1, wherein said nucleic acid fragments have a size of<br>
fewer than 1000 base pairs.<br>
10.	The method as claimed in claim 1, wherein said library of nucleic acid fragments is<br>
constructed by enzymatic cleavage of the DNA.<br><br>
11.	The method as claimed in claim 1, wherein said library of nucleic acid fragments is constructed by cDNA synthesis.<br>
12.	The method as claimed in claim 1. wherein said library of nucleic acid fragments is constructed by recombinant DNA technology.<br>
13.	The method as claimed in claim 12, wherein said recombinant DNA technology comprises unidirectional deletion.<br>
14.	The method as claimed in claim 1. wherein said host cells are selected from plant, bacterial, fungal, yeast, or animal cells.<br>
15.	The method as claimed in claim 1, wherein said reporter protein is a protein that is secreted into the extracellular space.<br>
16.	The method as claimed in claim 15, wherein said reporter protein is selected from invertase, sucrase, cellulase, xylanase. maltase, amylase, glucoamylase, galactosidase, phosphatase, beta-lactamase, lipase or protease.<br>
17.	The method as claimed in claim 1. wherein said host cells are yeast, said reporter protein is invertase and the transformed yeast cells are selected for their ability to grow on sucrose or raffinose.<br>
18.	The method as claimed in claim 1. wherein said host cells are yeast, said reporter protein is amylase, the yeast cells are non-amylolytic, and the transformed cells are screened for their ability to degrade starch.<br>
19.	The method as claimed in claim 1. wherein said step of identifying cells showing an activity of the reporter protein occurs by using a reporter protein which provides resistance to a growth inhibitor.<br>
20.	The method as claimed in claim 1, wherein said step of identifying cells showing an<br><br>
 activity of the reporter protein occurs by using two or more reporter proteins.<br>
21.	The method as claimed in claim 1, wherein said target protein is from a plant, animal, or microorganism.<br>
22.	The method as claimed in claim 21. wherein said target protein is a human protein.<br>
23.	The method as claimed in claim 21, wherein said target protein is a cytokine, serum protein, colony stimulating factor, growth factor, hormone, or enzyme.<br>
24.	The method as claimed in claim 1. wherein said linker DNA is more than 20 base pairs in length.<br>
25.	The method as claimed in claim 1, wherein said linker DNA encodes a protease recognition sequence thereby allowing cleavage at the junction of the TFP and the target protein.<br>
26.	The method as claimed in claim 1. wherein said linker DNA encodes a recognition sequence for yeast kex2p, mammalian furin, factor Xa, enterokinase, subtilisin, tobacco etch virus protease, thrombin, or ubiquitin hydrolase,<br>
27.	The method as claimed in claim 1, wherein said linker DNA encodes an affinity tag.<br>
28.	The method as claimed in claim 1, wherein said linker DNA encodes a restriction enzyme recognition site.<br>
29.	The method as claimed in claim 30, wherein said linker DNA further encodes kex2p-like protease- or kex2p-recognition sequence.<br>
30.	A method of expression of a target protein using a TFP identified by the method as claimed in claim 1, wherein said TFP can be one, two, six or more and is selected from the group consisting of TFP-9 (SEQ ID NO: 29), TFP-13 (SEQ ID NO:31), TFP-17 (SEQ ID NO:33), TFP-18 (SEQ ID NO:35). TFP-19 (SEQ ID NO:37). TFP-<br><br>
20 (SEQ ID NO:39). TFP-21 (SEQ ID NO:41), TFP-25 (SEQ ID NO:43), TFP-27 (SEQ ID NO:45), TFP-11 (SEQ ID NO:61). TFP-22 (SEQ ID NO:63), TFP-29 (SEQ ID NO:65), TFP-34 (SEQ ID NO:67), TFP-38 (SEQ ID NO:69). TFP-39 (SEQ ID NO: 129), TFP-43 (SEQ ID NO: 131), TFP-44 (SEQ ID NO: 133). TFP-48 (SEQ ID NO: 135), TFP-52 (SEQ ID NO: 137), TFP-54 (SEQ ID NO: 139), TFP-40 (SEQ ID NO: 175), TFP-50 (SEQ ID NO: 177), TFP-51 (SEQ ID NO: 179), TFP-57 (SEQ ID NO: 181), TFP-58 (SEQ ID NO: 183), TFP-59 (SEQ ID NO: 185), TFP-5 (SEQ ID NO:200), TFP-6 (SEQ ID NO:202), TFP-7 (SEQ ID NO:204), TFP-8 (SEQ ID NO:206), PpTFP-1 (SEQ ID NO:84), PpTFP-2 (SEQ ID NO:86), PpTFP-3 (SEQ ID NO:88), and PpTFP-4 (SEQ ID NO:90).</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUFic3RyYWN0LSgyNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Abstract-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNsYWltcy0oMjMtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Claims-(23-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNsYWltcy0oMjQtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Claims-(24-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNsYWltcy0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Claims-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDUtMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-Correspondence Others-(05-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjMtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Correspondence Others-(23-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Correspondence Others-(24-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Correspondence Others-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMDYtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Correspondence-Others-(22-06-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWRlY3NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-decsription (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LURyYXdpbmdzLSgyNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Drawings-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUZvcm0tMS0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Form-1-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LUZvcm0tMTMgKDIwLTEyLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-Form-13 (20-12-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUZvcm0tMi0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Form-2-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LUZvcm0tMy0oMDUtMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-Form-3-(05-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LUZvcm0tMy0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Form-3-(27-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">9929-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LVBDVC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-PCT-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LVBDVC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-PCT-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=OTkyOS1ERUxOUC0yMDA3LVBldGl0aW9uLTEzNy0oMjctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">9929-DELNP-2007-Petition-137-(27-12-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252103-device-for-continuously-filtering-material-mixtures.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252105-method-and-devices-for-supporting-a-flexible-handling-of-connections-towards-a-dual-mode-mobile-station.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252104</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>9929/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>17/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Apr-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-Apr-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Dec-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>KOREA RESEARCH INSTITUTE OF BIOSCIENCE AND BIOTECHNOLOGY</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>52,EOEUN-DONG,YUSEONG-GU, DAEJEON, 305-806, REPUBLIC OF KOREA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>CHOI, EUI-SUNG</td>
											<td>307-1703,SONG RIM, MAEUL APT., HAKI-KONG, DAEJEON, 305-328, REPUBLIC OF KOREA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>SHIN, MI-KYUNG</td>
											<td>504-7, NOEUN-DONG, YUSEONG-GU, DAEJEON, 305-768, REPUBLIC OF KOREA.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BAE,JUNG-HOON</td>
											<td>539-3, NOEUN-DONG,YUSEONG-GU, DAEJEON, 305-325, REPUBLIC OF KOREA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>YOON, SUNG-SOOK</td>
											<td>319-401, JONGGUNKYO APT., SINBONG-DONG, YUSEONG-GU, DAEJEON, 305-154, REPUBLIC OF KOREA.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>SOHN, JUNG-HOON</td>
											<td>103-506, NURI APT., WOLPYEONG-DONG, SEO-GU,DAEJEON, 302-791, REPUBLIC OF KOREA.</td>
										</tr>
										<tr>
											<td>6</td>
											<td>CHUN, CHANG-SOO</td>
											<td>7-63, MIA 4-DONG, GANGBUK-GU, SEOUL, 142-104, REPUBLIC OF KOREA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/IB2006/003102</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-07-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10-2005-0064402</td>
									<td>2005-07-15</td>
								    <td>Republic of Korea</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252104-a-method-of-identifying-a-target-protein-specific-translational-fusion-partner-tfp by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:33:13 GMT -->
</html>
