<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252327-fgf18-production-in-prokaryotic-hosts by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:25:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252327:FGF18 PRODUCTION IN PROKARYOTIC HOSTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">FGF18 PRODUCTION IN PROKARYOTIC HOSTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The expression vectors and methods using an E. coli expression system for the large scale production of FGF18 are described. The vectors utilize the FGF18 coding sequence with specific changes in nucleotides in order to optimize codons and mRNA secondary structure for translation in E. coli. Using the expression vectors, the FGF18 gene was produced in E. coli to a level of greater than 1 g/L in fed batch fermentation. Also included are OmpT deficient E. coli strains, as well as OmpT and fhuA negative strains transformed with an FGF18 expression vector.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
FGF18 PRODUCTION IN PROKARYOT1C HOSTS<br>
BACKGROUND OF THE INVENTION<br>
The increased availability and identification of genes from human and other genomes has led to an increased need for efficient expression and purification of recombinant proteins. The expression of proteins in bacteria is by far the most widely used approach for the production of cloned genes. For many reasons, expression in bacteria is preferred to expression in eukaryotic cells. For example, bacteria are much easier to grow than eukaryotic cells. More specifically, the availability of a wealth of sophisticated molecular genetic tools and thousands of mutants make E. colU as an expression host, extremely useful for protein production. However, the high-level production of functional proteins in E. colt, especially those from eukaryotic sources has often been difficult<br>
FGF18 is a member of the fibroblast growth factor family that shares significant sequence homology with FGF8 and FGF17. These three factors are thought to comprise a subfamily of the FGF proteins. As with all members of the FGF family, FGF18 has discrete effects on both developing and adult tissues and is thought to play a part in embroyonic development and wound healing (see, e.g., Ornitz and Marie, Genes &amp; Development 16:1446-1465 (discussing the role of all FGFs, including FGF18 in bone development)). In particular, FGF18 has been shown to have proliferative effects on cartilage and neural tissues, among others (Ellsworth et aL, Osteoarthritis and Cartilage (2002) 10,308-320; Ellsworth et al.Stroke (2003) 34(6): 1507-12).<br>
Recombinant FGF18 has been produced in prokaryotic cells, in particular E. coli. The resulting bacterial produced protein is not glycosylated and is produced in an aggregated state. Initial experiments indicated that FGF18 produced in bacterial resulted in truncation of the protein, producing truncated FGF18 (trFGF18). As the truncated version appeared to have very similar, if not identical, biological properties as the full-length "protein, constructs were then made that produced solely the truncated version. Production of FGF18 or trFGF18 from £. coli requires that the aggregated proteins be solubilized from the insoluble inclusion bodies and renatured or refolded. Without renaturation, the specific activity of the recombinant protein will be significantly reduced.<br>
Despite advances in the expression of recombinant proteins in bacterial hosts, there exists a need for improved methods for producing biologically active and<br><br>
purified recombinant FGF18 and trFGF18 proteins in prokaryotic systems that result in higher yields for protein production. These and other aspects of the invention will become evident upon reference to the following detailed description. In addition, various references are identified below and are incorporated by reference in their entirety.<br>
SUMMARY OF THE INVENTION<br>
In one aspect, the present invention provides an expression vector for producing FGF18 or trFGF18 proteins comprising the operably linked elements of a prokaryotic origin of replication, a transcriptional initiation DNA element, and polynucleotide sequence and a transcriptional terminator. In another aspect, the expression vector is the vector pSDH170 (SEQ ID NO:l) that can be used to produce FGF18. In another aspect, the expression vector is pSDH174 (SEQ ID NO:2) that can be used to produce trFGF18. Further embodiments provide the expression vector can include a selectable marker.<br>
hi another aspect, the present invention provides prokaryotic host cells transformed the expression vectors described as comprising SEQ ID NO: 3:, a polynucleotide sequence encoding the polypeptide of SEQ ID NO:4, or vector pSDH170 (SEQ ID NO:l). In another aspect, the present invention provides prokaryotic host cells transformed the expression vectors described as comprising SEQ ID NO:5, a polynucleotide sequence encoding the polypeptide of SEQ ID NO:6, or vector pSDH174 (SEQ ID NO:2). In other embodiments, the host strain is E. colt strain W3110 or the strain zGOLD 1 or zGOLD5.<br>
hi another aspect, the present invention provides methods for producing FGF18 or trFGF18 proteins under conditions wherein the FGF18 or trFGF18 protein is expressed. In one embodiment, the method comprises culturing a host cell expressing FGF18 after being transformed with pSDH170. In a second embodiment, the method comprises culturing a host cell expressing trFGF18 after being transformed with pSDH174. In other embodiments, the method comprising culturing a host cell transformed with an expression vector comprising SEQ ID NO:l. In further embodiments, the method comprising culturing a host cell transformed with an expression vector comprising SEQ ID NO:2. The method also comprises recovering the host cells from the growth medium, and then isolating the FGF18 or trFGF18 protein from the host cells.<br>
In other aspects, the present invention provides methods for producing FGF18 or trFGF18 comprising the steps as described above, in a fed batch fermentation process or a batch fermentation process.<br><br>
In another aspect, the present invention provides methods for producing an FGF18 or trFG18 protein comprising culturing a host cell as described above in a shake flask to an ODeoo of 5 to 20 in a growth medium, inoculating a fermentation vessel with 1 to 12% volume to volume (v/v) of shake flask medium containing host cells, culturing the host cells in a growth medium at a pH of 6-2 to 7.2, where a feed solution is fed into the fermentation vessel before 15 hours elapsed fermentation time (EFT), adding an inducing agent to the fermentation vessel at 20 to 30 hours EFT, and harvesting the host cells at 48 to 56 hours EFT. In one embodiment, the inducing agent is isopropyl 8-D thiogalactopyranoside (IPTG) at 0.5 to 2 mM. In another embodiment, the feed solution comprises a carbohydrate selected from the group consisting of glycerol and glucose and the feed is 5 to 15 grams of carbohydrate per hour. In another embodiment, the glycerol in the feed solution is 40 to 70% v/v glycerol or the glucose is 40 to 70% w/v glucose. In further embodiments, the glycerol is about 70% v/v or the glucose is about 60% w/v.<br>
In one aspect, the present invention provides methods of producing FGF18 or trFGF18 comprising seeding a flask with an inoculum comprising an E. coli W3110 host cells expressing an FGF18 or trFGF18 polypeptide as shown in SEQ ID NO:4 or SEQ ID NO:6, or an E. coli W3110 host cell comprising pSDH170 or pSDH174 vector, wherein an FGF18 or trFGF18 polypeptide is expressed, and with growth medium comprising about 5 g/1 glycerol, culturing the inoculum in a growth medium for 16 to 20 hours at about 30°C, transferring the cultured inoculum in growth medium to a batch fermentator at a concentration 0.5 to 5% v/v inoculum, fermenting the batch fermentation at about 37°C and about pH 6.8 with about 2% glycerol, introducing a glucose feed at about 8 hours EFT of about 9.5 g glucose/liter/hour and continuing until end of a fermentation run, adding IPTG at about 24 hours EFT to final concentration of 0.5 to 2 mM, fermenting about 28 hours of IPTG, harvesting fermentation broth from the fermentor, adding an equal volume of water to the fermentation broth, and homogenizing and centrifuging to collect a cell pellet or cell slurry comprising FGF18 or tdPGF18 protein material.<br>
In another aspect, the present invention provides methods for isolating insoluble FGF18 or trFGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:4 or SEQ ID NO:6 comprising separating water insoluble FGF18 or trFGF18 protein from a cell pellet or slurry, dissolving the insoluble FGF18 or trFGF18 material in a chaotropic solvent, diluting the chaotropic solvent and refolding the FGF18 or trFGF18 protein; and isolating the FGF18 or trFGF18 protein, wherein the isolated protein is capable of being biologically active. In one embodiment of the invention, the isolated FGF18 or trFGF18 protein is at least 90% pure.   In another<br><br>
embodiment, the isolated FGF18 or trFGF18 protein is at least 90% pure and has an endotoxin level of less that 10 endotoxin units per mg FGF18 or trFGF18 protein.<br>
In another aspect, the present invention provides methods for isolating insoluble FGF18 or trFGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:4 or SEQ ID NO:6 comprising separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble FGF18 or trFGF18 protein material, homogenizing the cell pellet or cell slurry to collect inclusion bodies, dissolving the insoluble FGF18 or trFGF18 protein material in a chaoptropic solvent comprising a guanidine salt, diluting the chaotropic solvent by addition of a refolding buffer, isolating the FGF18 or trFGF18 protein by removing unfolded and aggregated proteins by filtering, and purifying the FGF18 or trFGF18 refolded protein on a cation exchange column, wherein the isolated and purified FGF18 or trFGF18 protein is capable of being biologically active.<br>
In another aspect, the present invention provides a method for isolating insoluble FGF18 or trFGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:4 or SEQ ID NO: 6 comprising separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble FGF18 or trFGF18 material, homogenizing the cell pellet or cell slurry to collect inclusion bodies, dissolving the insoluble FGF18 or trFGF18 protein material in a chaotxopic solvent comprising a guanidine salt, diluting the chaotropic solvent by addition of a refolding buffer, isolating the FGF18 or trFGF18 protein by removing unfolded and aggregated proteins by filtering, purifying the FGF18 or HFGF18 refolded protein on a cation exchange column, and purifying the FGF18 or trFGFlS eluate on a hydrophobic interaction column, wherein the isolated and purified FGF18 or trFGF18 protein is capable of being biologically active.<br>
In other embodiments, the above methods for isolating insoluble FGF18 or trFGFlS protein comprise measuring biological activity using an FGF18 receptor binding assays,<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is illustration of expression plasmid pSDH170, which comprises the codon optimized nucleotide sequence for FGF18.<br>
Figure 2 is illustration of expression plasmid pSDH174, which comprises the codon optimized nucleotide sequence for trFGFl8.<br><br>
DESCRIPTION OF THE INVENTION<br>
The following definitions are provided to facilitate understanding of the invention.<br>
As used herein, "nucleic acid" or "nucleic acid molecule" refers to polynucleotides, such as deoxyribonucleic acid (DNA) or ribonucleic acid (RNA), oligonucleotides, fragments generated by the polymerase chain reaction (PCR), and fragments generated by any of ligation, scission, endonuclease action, and exonuclease action. Nucleic acid molecules can be composed of monomers that are naturally-occurring nucleotides (such as DNA and RNA), or analogs of naturally-occurring nucleotides (e.g., a-enantiomeric forms of naturally-occurring nucleotides), or a combination of both. Modified nucleotides can have alterations in sugar moieties and/or in pyrimidine or purine base moieties. Sugar modifications include, for example, replacement of one or more hydroxyl groups with halogens, alkyl groups, amines, and azido groups, or sugars can be functionalized as ethers or esters. Moreover, the entire sugar moiety can be replaced with sterically and electronically similar structures,-such as aza-sugars and carbocyclic sugar analogs. Examples of modifications in a base moiety include alkylated purines and pyrimidines, acylated purines or pyrimi dines, or other well-known heterocyclic substitutes. Nucleic acid monomers can be linked by phosphodiester bonds or analogs of such linkages. Analogs of phosphodiester linkages include phosphorothioate, phosphorodithioate, phosphoroselenoate, phosphorodiselenoate, phosphoroanilothioate, phosphoranilidate, phosphoramidate, and the like. The term "nucleic acid molecule" also includes so-called "peptide nucleic acids," which comprise naturally-occurring or modified nucleic acid bases attached to a polyamide backbone. Nucleic acids can be either single stranded or double stranded.<br>
The term "complement of a nucleic acid molecule" refers to a nucleic acid molecule having a complementary nucleotide sequence and reverse orientation as compared to a reference nucleotide sequence.<br>
An "enhancer" is a type of regulatory element that can increase the efficiency of transcription, regardless of the distance or orientation of the enhancer relative to the start site of transcription.<br>
"Heterologous DNA" refers to a DNA molecule, or a population of DNA molecules, that does not exist naturally within a given host cell. DNA molecules heterologous to a particular host cell may contain DNA derived from the host cell species {te., endogenous DNA) so long as that host DNA is combined with non-host DNA {i.e., exogenous DNA). For example, a DNA molecule containing a non-host DNA segment encoding a polypeptide operably linked to a host DNA segment comprising a transcription promoter is considered to be a heterologous DNA molecule.<br><br>
Conversely, a heterologous DNA molecule can comprise an endogenous gene operably linked with an exogenous promoter. As another illustration, a DNA molecule comprising a gene derived from a wild-type cell is considered to be heterologous DNA if that DNA molecule is introduced into a mutant cell that lacks the wild-type gene.<br>
The term "contig" denotes a nucleic acid molecule that has a contiguous stretch of identical or complementary sequence to another nucleic acid molecule. Contiguous sequences are said to "overlap" a given stretch of a nucleic acid molecule either in their entirety or along a partial stretch of the nucleic acid molecule.<br>
"Complementary DNA (cDNA)" is a single-stranded DNA molecule that is formed from an mRNA template by the enzyme reverse transcriptase. Typically, a primer complementary to portions of mRNA is employed for the initiation of reverse transcription. Those skilled in the ait also use the term "cDNA" to refer to a double-stranded DNA molecule consisting of such a single-stranded DNA molecule and its complementary DNA strand. The term "cDNA" also refers to a clone of a cDNA molecule synthesized from an RNA template.<br>
An "isolated nucleic acid molecule" is a nucleic acid molecule that is not integrated in the genomic DNA of an organism. For example, a DNA molecule that encodes a growth factor that has been separated from the genomic DNA of a cell is an isolated DNA molecule. Another example of an isolated nucleic acid molecule is a chemically-synthesized nucleic acid molecule that is not integrated in the genome of an organism- A nucleic acid molecule that has been isolated from a particular species is smaller than the complete DNA molecule of a chromosome from that species.<br>
'linear DNA" denotes non-circular DNA molecules with free 5r and 3* ends. Linear DNA can be prepared from closed circular DNA molecules, such as plasmids, by enzymatic digestion or physical disruption.<br>
A "promoter" is a nucleotide sequence that directs the transcription of a structural gene. Typically, a promoter is located in the 5! non-coding region of a gene, proximal to the transcriptional start site of a structural gene. Sequence elements within promoters that function in the initiation of transcription are often characterized by consensus nucleotide sequences. These promoters include, for example, but are not limited to, IPTG-inducible promoters, bacteriophage T7 promoters and bacteriophage' XpL. See Sambrook et al., Molecular Cloning: A Laboratory Manual, 3rd edL, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 2001. A typical promoter will have three components, consisting of consensus sequences at -35 and -10 with a sequence of between 16 and 19 nucleotides between them (Lisset, S. and Margalit, H., Nucleic Acids Res. 21: 1512, 1993). Promoters of this sort include the lac, trp, trp-lac (tac) and trp-ladfrc) promoters. If a promoter is an inducible promoter, then the rate of<br><br>
transcription increases in response to an inducing agent In contrast, the rate of transcription is not regulated by an inducing agent if the promoter is a constitutive promoter. R'epressible promoters are also known.<br>
A "core promoter" contains essential nucleotide sequences for promoter function, including the start of transcription. By this definition, a core promoter may or may not have detectable activity in the absence of specific sequences that may enhance the activity or confer tissue specific activity.<br>
A "regulatory element" is a nucleotide sequence that modulates the activity of a core promoter. For example, a eukaryotic regulatory element may contain a nucleotide sequence that binds with cellular factors enabling transcription exclusively or preferentially in particular cells, tissues, or organelles. These types of regulatory elements are normally associated with genes that are expressed in a "cell-specific," '"tissue-specific," or "organelle-specific" manner. Bacterial promoters have regulatory elements that bind and modulate the activity of the core promoter, such as operator sequences that bind activator or repressor molecules.<br>
A "cloning vector7' is a nucleic acid molecule, such as a plasinid, cosmid, or bacteriophage, which has the capability of replicating autonomously in a host cell. Cloning vectors typically contain one or a small number of restriction endonuclease recognition sites that allow insertion of a nucleic acid molecule in a determinable fashion without loss of an essential biological function of the vector, as well as nucleotide sequences encoding a marker gene that is suitable for use in the identification and selection of cells transformed with the cloning vector. Marker genes typically include genes that provide resistance to antibiotic.<br>
An "expression vector" is a nucleic acid molecule encoding a gene that is expressed in a host cell. Typically, an expression vector comprises a transcriptional promote", a gene, an origin of replication, a selectable marker, and a transcriptional terminator. Gene expression is usually placed under the control of a promoter, and such a gene is said to be "operably linked to" the promoter. Similarly, a regulatory element and a core promoter are operably linked if the regulatory element modulates the activity of the core promoter. An expression vector may also be known as an expression construct.<br>
A "recombinant host" is a cell that contains a heterologous nucleic acid molecule, such as a cloning vector or expression vector.<br>
The term "expression" refers to the biosynthesis of a gene product. For example, in the case of a structural gene, expression involves transcription of the structural gene into mRNA and the translation of mRNA into one or more polypeptides.<br>
The term "secretory signal sequence" denotes a DNA sequence that encodes a peptide (a "secretory peptide") that, as a component of a larger polypeptide,<br><br>
directs the larger polypeptide through a secretory pathway of a cell in which it is synthesized. The larger polypeptide is commonly cleaved to remove the secretory peptide during transit through the secretory pathway.<br>
A "polypeptide" is a polymer of amino acid residues joined by peptide bonds, whether produced naturally or synthetically. Polypeptides of less than about 10 amino acid residues are commonly referred to as "peptides."<br>
A "protein" is a macromolecule comprising one or more polypeptide chains. A protein may also comprise non-peptidic components, such as carbohydrate groups. Carbohydrates and other non-peptidic substituents may be added to a protein. Proteins are defined herein in terms of their amino acid backbone structures; substituents such as carbohydrate groups and non-peptidic groups are generally not specified, but may be present nonetheless.<br>
A peptide or polypeptide encoded by a non-host DNA molecule is a "heterologous" peptide or polypeptide.<br>
An "isolated polypeptide" is a polypeptide that is essentially free from contaminating cellular components, such as carbohydrate, lipid, or other proteinaceous impurities associated with the polypeptide in nature. Typically, a preparation of isolated polypeptide contains the polypeptide in a highly purified form, i.e.t at least about 80% pure, at least about 90% pure, at least about 95% pure, greater than 95% pure, or greater ) than 99% pure. One way to show that a particular protein preparation contains an isolated polypeptide is by the appearance of a single band following sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis of the protein preparation and Coomassie Brilliant Blue staining of the gel. However, the term "isolated" does not exclude the presence of the same polypeptide in alternative physical forms, such as i       dimers or alternatively glycosylated or derivatized forms.<br>
The terms "amino-terminal" or "N-terminal" and "carboxyl-terminaT or "C-terminal" are used herein to denote positions within polypeptides. Where the context allows, these terms are used with reference to a particular sequence or portion of a polypeptide to denote proximity or relative position. For example, a certain sequence ) positioned carboxyl-terminal to a reference sequence within a polypeptide is located proximal to the carboxyl terminus of the reference sequence, but is not necessarily at the carboxyl terminus of the complete polypeptide.<br>
A "fusion protein" is a hybrid protein expressed by a nucleic acid<br>
molecule comprising nucleotide sequences of at least two genes.<br>
5	The term "affinity tag" is used herein to denote a polypeptide segment<br>
that can be attached to a second polypeptide to provide for purification or detection of<br>
the second polypeptide or provide sites for attachment of the second polypeptide to a<br><br>
substrate. In principal, any peptide or protein for which an antibody or other specific binding agent is available can be used as an affinity tag. Affinity tags include a poly-histidine tract, protein A (Nilsson et al, EMBO J. 4:1075 (1985); Nilsson et al., Methods Enzvmol. 198:3 (1991)), glutathione S transferase (Smith and Johnson, Gene 67:31 (1988)), Glu-Glu affinity tag (Grussenmeyer et al, Proc, Natl. Acad. Sci. USA 82:7952 (1985)), substance P, FLAG peptide (Hopp et al, Biotechnology 6:1204 (1988)), streptavidin binding peptide, or other antigenic epitope or binding domain. See, in general, Ford et alt Protein Expression and Purification 2:95 (1991). DNA molecules encoding affinity tags are available from commercial suppliers (e.g., Pharmacia Biotech,<br>
Piscataway, NJ).<br>
The term "isotonic" is used herein for its conventional meaning, that is a tonicity equal to that of blood, equivalent to a 0.9% solution of NaCl. "An isotonic amount" of a salt is that amount required to make a solution isotonic or to produce an isotonic solution upon reconstitution of a lyophilized preparation.<br>
Concentrations are specified herein in units of molarity or % w/v of liquid compositions. When the composition is in the form of a lyophilized powder, the concentrations of the respective components will be such as to provide the specified concentration on reconstitution of the powder.<br>
Due to the imprecision of standard analytical methods, molecular weights and lengths of polymers are understood to be approximate values. When such a value is expressed as "about" X or "approximately" X, the stated value of X will be understood to be accurate to ±10%.<br>
EXPRESSION OF RECOMBINANT FGF18 OR TRFGF18<br>
The present invention provides expression vectors and methods for producing recombinant FGF18 protein from a prokaryotic host. FGF18 was previously designated zfgf5, and is fully described in commonly assigned U.S. Patents 5,989,866 6,518,236 and 6,352,971, incorporated herein by reference. In particular, the expression vectors and methods of the present invention comprise an E. coli expression system for the large scale production of FGF18 or trFGF18 utilizing the FGF18 or trFGF18 coding sequence with specific changes in nucleotides in order to optimize codons and mRNA secondary structure for translation in E. coli. Using the expression vectors and methods of the present invention, the FGF18 or trFGF18 gene was produced in E. coli to a level of greater than 1 g/L in fed batch fermentation. The present inventors found that use of the E. coli OmpT protease deficient strains like, for example, UT5600, as a production host overcame stability problems with FGF18 ox trFGF18. In addition to full length FGF18, a different polypeptide can be produced, truncated FGF18 (trFGF18) that is the<br><br>
FGF18 coding sequence with a codon encoding an N-terminal Met added at the 5' end of the polynucleotide sequence at amino acid 27 (Glu) and truncated at amino acid 196 (Lys). The trFGF18 sequence is SEQ ID NO: 5. Using the expression vectors described herein significantly improved the yield of recombinant protein recovered from the bacteria. In another embodiment, to facilitate the development of high cell density fed-batch fermentation, another E. coli strain, W3110, was selected as a host for the large-scale production of FGF18 or trFGF18. This host strain is non-pathogenic and can grow to high cell density in minimally defined fermentation media. The productivity of FGF18 or trFGF18 in E. coli strain W3110 was comparable to that obtained in E. coli strain UT5600 when produced in shaker flask and batch fermentations.<br>
The present invention also provides methods for recovering recombinant FGF18 or trFGF18 protein from a prokaryotic host when the protein is expressed by the host and found within the host cell as an unglycosyiated, insoluble inclusion body. When the prokaiyotic cell is lysed to isolate the inclusion bodies (also called refractile bodies), the inclusion bodies are aggregates of FGF18 or trFGF18. Therefore, the inclusion bodies must be disassociated and dissolved to isolate the FGF18 or trFGF18 protein, and generally this requires the use of a denaturing chaotropic solvent, resulting in recovering a polypeptide that must be refolded to have significant biological activity. Once the FGF18 or trFGF18 protein is refolded, the protein must be captured and purified. Thus, the present invention provides for methods for isolating insoluble FGF18 or trFGF18 protein from prokaryotic cells, dissolving the insoluble protein material in a chaotropic solvent, diluting the chaotropic solvent in such a manner that the FGF18 or trFGF18 protein is refolded and isolated. The present invention also includes methods for capturing the renatured FGF18 or trFGF18 from the dilute refold buffer using cation exchange chromatography, and purifying the refolded FGF18 or trFGF18 protein using hydrophobic interaction chromatography. Further purification can be achieved using anion exchange in binding assays using an FGF18 receptor and the like.<br>
The human FGF18 gene encodes a polypeptide of 207 amino acids. The full length sequence includes a signal peptide of 27 amino acids, as shown in SEQ ID NOS: 3 and 4, and a mature protein of 180 amino acids comprising residue 28 (Glu) to residue 207 (Ala). Truncated FGF18 (trFGF18) comprises the fragment from residue 28 (Glu) to 196 (Lys) with an added N-terminal Met. The nucleotide sequence of SEQ ID NO: 3 (SEQ BD NO:l nucleotides 1495-2040) and SEQ ID NO:5 (SEQ ID NO: 2 nucleotides 1494-2007) show codon optimized sequences that fall within the scope of the present invention.<br>
Production of recombinant human FGF18 or trFGF18 which utilized a mammalian expression system produced approximately 20 mg/L of protein. Therefore, a<br><br>
more cost effective expression system was desirable for large-scale production of FGF18 or trFGF18. The E. coli system was found to be a better alternative for large-scale production. Expression in E. coli offers numerous advantages over other expression systems, particularly low development costs and high production yields.<br>
Recombinant FGF18 or trFGF18 expressed in E. coli was isolated as insoluble inclusion bodies after cell breakage. In most cases inclusion bodies needed to be solublized in denaturing chaotropic solvent and the protein refolded by dilution of the chaotropic agent followed by purification. Proteins vary a great deal with respect to their optimal refolding environment. Factors that can affect the recovery of properly folded and biologically active material include: initial protein concentration, oxidative state, pH, excipients, salts, detergents, termperature, mode of refolding buffer addition and the like.<br>
Examination of the codons used in the human FGF18 cDNA indicated that it contained an excess of the least frequently used codons in E. colu Genes with a high content of rarely used codons tend to be expressed at a low level in E. coli (Kane, Curr Opin BiotechnoL 6(51:494-500, 1995). An additional concern relating to the expression of human FGF18 in E. coli was the occurrence of eight potential OmpT cleavage sites located in the FGF18 sequence. OmpT is an endopeptidase that specifically cleaves between two consecutive basic residues and the enzyme is active under denaturing conditions such as 8M urea and 6M guanidine-HCl (White et ah, J Biol Chem. 270(221:12990-4, 1995; Dekker et al., Biochemistry, 40(61:1694-701, 2001). It appeared that the site closest to the C-terminal end of the protein was particularly susceptible to cleavage. This raises concerns for the stability of full length FGF18 in a cell extract from E. coli due to the proteolytic activity of OmpT.<br>
Several laboratories have shown that the expression level of proteins whose genes contain rare codons can be dramatically improved when the level of certain rare tRNAs is increased within the host (Zdanovsky et al., Appl Environ Microbiol. 66(8^:3166-73, 2000; Calderone et al., J Mol Biol. 262(41:407-12; Kleber-Janke et al., Protein Exnr Purif. 19(31:419-24, 2000; You et al,. Biotechniques, 27(51:950-4, 1999.) An alternative approach is engineering the nucleotide sequence to take advantage of the redundancy of the genetic code and no longer require the utilization of the rare codons. Because these rare codons were plentiful in FGF18, the present inventors re-synthesized both the FGF18 gene and the trFGF18 gene with more appropriate codons.<br>
The present invention provides an expression vector comprising the coding sequence of FGF18 with codons optimized for translation in E. coli. It also provides an expression vector comprising the coding sequence of a truncated FGF18 with codons optimized for translation in E. coll The synthetic gene encoding trFGF18 was obtained by overlap PCR. The final PCR product was introduced into an expression vector for<br><br>
expression under the' control of the Tac promoter. An examination of the secondary structure of the trFGFlS cDNA revealed an exceptionally stable two hairpin structure. It was suspected that these hairpin loops were a structural element that would prevent efficient expression from the fully optimized sequence. Therefore, the hairpin structures were eliminated during the codon optimization process. The resulting gene was expressed in E. coli at high levels. Expression levels with the new expression construct increased to around 20% of total cell protein or 100 mg/L.<br>
Expression vectors that are suitable for production of a desired protein in prokaryotic cells typically comprise (1) prokaryotic DNA elements coding for a bacterial origin for the maintenance of the expression vector in a bacterial host; (2) DNA elements that control initiation of transcription, such as a promoter; (3) DNA elements that control the processing of transcripts, such as a transcriptional terminator, and (4) a gene encoding a selectable marker, such as antibiotic resistance. The prokaryotic host cell produces FGF18 or trFGF18 upon introduction of an expression vector and addition of an appropriate inducer. Accordingly, the present invention contemplates expression vectors comprising a promoter, the FGF18 or trFGF18 optimized nucleotide sequence, and a terminator sequence. The exemplary optimized FGF18 nucleotide sequence is shown in SEQ ID NO:3, while the exemplary optimized trFGF18 sequence is shown in SEQ ID NO:5. In another embodiment, the expression vector further comprises a selectable marker. In one embodiment, the selectable marker is kanamycin resistance.<br>
Expression vectors can also comprise nucleotide sequences that encode a peptide tag to aid in purification of the desired protein. Peptide tags that are useful for isolating recombinant polypeptides include, for example, polyffistidine tags (which have an affinity for nickel-chelating resin), c-myc tags, calmodulin binding protein (isolated with calmodulin affinity chromatography), substance P, the RYIRS tag (which binds with anti-RYIRS antibodies), the Glu-GIu tag, and the FLAG tag (which binds with anti-FLAG antibodies). See, for example, Luo et aL, Arch. Biochem. Biophvs. 329:215 (1996), Morganti et al., Biotechnol. AppL Biochem. 23:67 (1996), and Zheng et al7 Gene 186:55 (1997). Nucleic acid molecules encoding such peptide tags are available, for example, from Sigma-Aldrich Corporation (St. Louis, MO).<br>
One of ordinary skill in the art will be familiar with a multitude of molecular techniques for the preparation of the expression vector. For example, the FGF18 or trFGF18 polynucleotide can be prepared by synthesizing nucleic acid molecules using mutually priming, long oligonucleotides and the nucleotide sequences described herein (see, for example, Ausubel (1995) at pages 8-8 to 8-9). Established techniques using the polymerase chain reaction provide the ability to synthesize DNA molecules at least two kilobases in length (Adang et al. Plant Molec. Biol. 21:1131 (1993), Bambot et aL, PCR<br><br>
Methods and Applications 2:266 (1993), Dillon et al., "Use of the Polymerase Chain Reaction for the Rapid Construction of Synthetic Genes," in Methods in Molecular Biology, Vol. 15: PCR Protocols: Current Methods and Applications, White (ed.), pages 263-268, (Humana Press, Inc. 1993), and Holowachuk et al, PCR Methods Appl. 4:299 (1995)).<br>
Another method for constructing expression systems utilizes homologous recombination using a yeast system. See U.S. Patent No. 6,207,442, Plasmid Construction by Homologous Recombination, incorporated herein by reference. The system provides a universal acceptor plasmid that can be used to clone a DNA encoding any polypeptide of interest, including polypeptide fusions. The system provides methods for preparing double stranded, circular DNA molecules comprising a region encoding a protein of interest One or more donor DNA fragments encoding the protein of interest, i.e., FGF18 or txFGF18, are combined with an acceptor plasmid, a first DNA linker, and a second DNA linker in a Saccharomyces cerevisiae host cell whereby the donor DNA fragment is joined to die acceptor plasmid by homologous recombination of the donor DNA, acceptor plasmid, and linkers to form the closed, circular plasmid.<br>
The nucleic acid molecules of the present invention can also be synthesized with "gene machines" using protocols such as the phosphoramidite metho± If chemically-synthesized, double stranded DNA is required for an application such as the synthesis of a gene or a gene fragment, then each complementary strand is made separately. The production of short genes (60 to 80 base pairs) is technically straightforward and can be accomplished by synthesizing the complementary strands and then annealing them. For the production of longer genes (&gt;300 base pairs), however, special strategies may be required, because the coupling efficiency of each cycle during chemical DNA synthesis is seldom 100%. To overcome this problem, synthetic genes (double-stranded) are assembled in modular form from single-stranded fragments that are from 20 to 100 nucleotides in length. For reviews on polynucleotide synthesis, see, for example, Glick and Pasternak, Molecular Biotechnology, Principles and Applications of Recombinant DNA (ASM Press 1994), Itakura et al, Annu, Rev. Biochem. 53:323 (1984), and Clirnie et al., Proc. Nat'l Acad. Sci. USA 87:633 (1990).<br>
Examples of alternate techniques that can be used to prepare the FGF18 or trFGF18 gene and expression vector include, for example, restriction endonuclease digestion and ligation, and polymerase chain reaction, all of which are well known in the art.<br>
A wide variety of selectable marker genes is available (see, for example, Kaufman, Meth. EnzvmoL 185:487 (1990); Kaufman, Meth. Enzvmol. 185:537 (1990)). It is common for expression vectors to comprise selection markers, such as tetracycline<br><br>
resistance, amplicillin resistance, kanamycin resistance, neomycin resistance, or chloramphenicol resistance. A selectable marker will permit selection and/or detection of cells that have been transformed with expression vector from cells that have not been transformed. An expression vector can carry more than one such antibiotic resistance gene. An example of selectable marker without antibiotic resistance uses the hok/sok system from plasmid Rl. The hok gene encodes the toxic Hok protein of 52 amino acids and the sok gene encodes an antisense RNA, which is complementary to the hok mRNA leader sequence. This selectable marker is known to one skilled in the art and is described in more detail by Gerdes, K et al., Genetic Engineering, 19:49-61,1997.<br>
A wide variety of suitable recombinant host cells is encompassed by the present invention and includes, but is not limited to, gram-negative prokaryotic host organisms. Suitable strains of E. coli include W3110, K12-derived strains MM294, TG-1, JM-107, BL21, and UT5600. Other suitable strains include: BL21(DE3), BL21(DE3)pLysS, BL21(DE3)pLysE, DH1, DH4I, DH5, DH5I, DH5IF, DH5IMCR, DH10B, DH10B/p3, DH11S, C600, HB101, JM101, JM105, JM109, JM110, K38, RR1, Y1088, Y1089, CSH18, ER1451, ER1647, E. coli K12, E. coli K12 RV308, E. coli K12 C600, £. coZiHBlOl, E. coli K12 C600 R.sub.k-M.sub.k-, £. coli K12 RR1 (see, for example, Brown (ed.), Molecular Biology Labfax (Academic Press 1991)). Other gram-negative prokaryotic hosts can include Serratia, Pseudomonas, Caulobacter. Prokaryotic hosts can include gram-positive organisms such as Bacillus, for example, B. subtilis and 5. thuringienesis, and B. thuringienesis var. israelensis, as well as Streptomyces, for example, S. lividans, S. ambofaciens, S. fradiae, and &amp; griseofuscus. Suitable strains of Bacillus subtilus include BR151, YB886, MI119, MI120, and B170 (see, for example, Hardy, "Bacillus Cloning Methods," in DNA Cloning: A Practical Approach, Glover (ed.) (TRL Press 1985)). Standard techniques for propagating vectors in prokaryotic hosts are well-known to those of skill in the art (see, for example, Ausubel et al. (eds.), Short Protocols in Molecular Biology. 3rd Edition (John Wiley &amp; Sons 1995); Wu et al, Methods in Gene Biotechnology (CRC Press, Inc. 1997)). For an overview of protease deficient strains in prokaryotes, see, Meerman et al.. Biotechnology 12:1107-1110, 1994. The present invention is exemplified using the W3110 strain, which has been deposited at the American Type Culture Collection (ATCC) as ATCC # 27325.<br>
Techniques for manipulating cloned DNA molecules and introducing exogenous DNA into a variety of host cells are disclosed by Sambrook et al., Molecular Cloning: A Laboratory Manual, 2nd ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989, and Ausubel et aL, eds., Cuirent Protocols in Molecular Biology, John Wiley and Sons, Inc., NY, 1987. Transformed or transfected host cells are cultured according to conventional procedures in a culture medium containing nutrients<br><br>
and other components required for the growth of the chosen host cells. A variety of suitable media, including defined media and complex media, are known in the art and generally include a carbon source, a nitrogen source, essential amino acids, vitamins and minerals. Media may also contain such components as growth factors or serum, as required. The growth medium will generally select for cells containing the exogenously added DNA by, for example, drug selection or deficiency in an essential nutrient that is complemented by the selectable marker carried on the expression vector or co-transfected into the host cell. Liquid cultures are provided with sufficient aeration by conventional means, such as shaking of small flasks or sparging of fermentors. Transformed cells can be selected and propagated to provide recombinant host ceils that express the gene of interest. FGF18 or trFGF18 can be expressed in E. coli using the MBP (maltose binding protein) fusion system (New England Biolabs (NEB; Beverly, MA)). In this system, the FGF18 or trFGF18 cDNA is attached to the 3f end of the malE gene to form an MBP-FGF18 or MBP-trFGF18 fusion protein. Fusion protein expression is driven by the tac promoter and is "off until the promoter is induced by addition of 1 mrnol IPTG. The constructs can be built as in-frame fusions with MBP in accordance with the Multiple Cloning Site (MCS) of the pMAL-c2 vector (NEB), and according to the manufacturer's specifications.<br>
FERMENTATION<br>
in one embodiment of the present invention a batch fermentation can be used, particularly when a large scale production of FGF18 or trFG18 using the expression system of the present invention is required. Generally, batch fermentation comprises that a first stage seed flask is prepared by growing E. coli strains expressing FGF18 or trFGF18 in a suitable medium in shake flask culture to allow for growth to an optical density (OD) of 5 to 20 at 600 ran. A suitable medium would contain nitrogen from a source(s) such as ammonium sulfate, ammonium phosphate, ammonium chloride, yeast extract, hydrolyzed animal proteins, hydrolyzed plant proteins or hydrolyzed caseins. Phosphate will be supplied from potassium phosphate, ammonium phosphate, phosphoric acid or sodium phosphate. Other components would be magnesium chloride or magnesium sulfate, ferric sulfate or ferric chloride, and other trace elements. Growth medium can be supplemented with carbohydrates, such as fructose, glucose, galactose, lactose, and glycerol, to improve growth. In certain embodiments, carbohydrate additions would be glycerol or glucose added from 1 to 20 g/L medium. In certain embodiments, the glycerol or glucose is 5- 10 g/L. Growth is started by inoculating a shake flask (baffled flask from 500 ml to 2000 ml) containing a preferred growth medium with Ecoli from an agar medium containing antibiotic, for example kanamycin at 10-50<br><br>
jig/ml, at the appropriate concentration or from a frozen stock culture. Growth in the shake flasks is at a temperature between 28 and 40°C. In certain embodiments, the shake flasks are grown at 30 to 37°C. The flasks are incubated with agitation set at 200 to 300 rpm.<br>
Fermentation vessels are prepared with a suitable growth medium and sterilized. The pH of the medium is adjusted to a pH 6.5 to 7.5. In certain embodiments, the pH is 6.8, 6.9, 7.0, 7.1 or 7.2. The vessels are set to the proper aeration and agitation levels and inoculated from a first stage seed flask culture that has been grown 10 to 20 hours and has an OD of 5 to 20 at 600 nm. The inoculation level is between 1% and 12% volume/volume (v/v). In certain embodiments, the inoculation level is at 3%, 4%, 5%, 6%, 7%, 8%, 9% or 10% v/v. The dissolved oxygen level is maintained above 20% saturation by increasing agitation speed, increasing the aeration rate, sparging in oxygen, or various combinations. The culture is grown until the OD 600 reaches 2 to 20 OD units. HTG is then added to the culture to a concentration 0.1 to 2.0 mM. The IPTG induces the tac promoter to express the FGF18 or trFGF18. Alternatively, lactose at 30% solution can be added at 10 g/1 at 24 hours for induction. The culture is then allowed to grow for an additional time between 2 and 8 hours. In certain embodiments, the culture is grown for 3-4 hours.<br>
In another embodiment, a fed batch culture is used to generate a high yield of FGF18 or trFGF18 protein. The FGF18- or trFGF18-producing E. coli strains are grown in a suitable medium in shake flask culture to allow for growth to an OD of 5 to 20 at 600 nm. A suitable medium would contain nitrogen from a source(s) such as ammonium sulfate, ammonium phosphate, ammonium chloride, yeast extract, hydrolyzed animal proteins, hydrolyzed plant proteins or hydrolyzed caseins. Phosphate will be supplied from potassium phosphate, ammonium phosphate, phosphoric acid or sodium phosphate. Other components would be magnesium chloride or magnesium sulfate, ferric sulfate or ferric chloride, and other trace elements. Growth medium can be supplemented with carbohydrates such as fructose, glucose, galactose, lactose and glycerol, to improve growth. In certain embodiments, carbohydrate additions would be glycerol or glucose added from 1 to 40 g/L medium. Li one embodiment, the glycerol or glucose is 5-10 g/L. Growth is started by inoculating a shake flask (baffled flask from 500 ml to 2000 ml) containing a preferred growth medium with Kcoli from an agar medium containing kanamycin (10-50 jig/ml) or from a frozen stock culture. Growth in the shake flasks is at a temperature of 28 to 40°C. In certain embodiments, growth temperature is 30 to 37°C. The flasks are incubated with agitation set at 200 to 300 rpm.<br>
A second stage vessel is prepared with a suitable growth medium and sterilized. A suitable medium would be, for example, Super Broth II (Becton Dickenson,<br><br>
Franklin Lakes, NJ), APS-Super Broth, Luria Broth, or ZSM (see, Tables 1-4) and kanamycin. Growth medium can be supplemented with carbohydrates to improve growth. Certain embodiments provide carbohydrate additions that have glycerol or glucose added from 1 to 40 g/L medium. In one embodiment, glycerol or glucose is 5-10 g/L, The pH of the medium is adjusted to a pH of 6*5 to 7.5. In certain embodiments, the pH is 6.8, 6.9, 7.0, 7.1 or 7.2. The vessels are set to the proper aeration and agitation levels. Growth is started by inoculating the vessel from a first stage seed flask culture that has been grown 10 to 20 hours and has an OD of 5 to 20 at 600 nm. The inoculation level is 1% to 12% v/v. In certain embodiments, the induction level will be 3%, 4%, 5%, 6%, 7%, 8%, 9% or 10% v/v. The dissolved oxygen level is maintained above 20% saturation by increasing agitation speed, increasing the aeration rate, sparging in oxygen or various combinations thereof.<br>
Fermentation vessels are prepared with a suitable growth medium (as described above) and sterilized. The pH of the medium is adjusted to a pH between 6.2, 6.3, 6.4, 6.5, 6.6, 6.7, 6.8, 6.9, 7.0, 7.1 or 7.2. In one embodiment, the medium is adjusted to pH 6.8. Growth medium can be supplemented with carbohydrates to improve growth. In some embodiments, carbohydrate additions are glycerol or glucose added from 5 to 40 g/L medium with certain embodiments having glycerol or glucose at 15-20 g/L. The vessels are set to the proper aeration and agitation levels and inoculated from a first stage seed flask culture or second stage seed vessel that has been grown to 10 to 20 hours and has an OD of 5 to 20 at 600 nm. The inoculation level is between 1% and 12% v/v. In certain embodiments, the inoculation level is 5%, 6%, 7%, 8%, 9% or 10% v/v. The dissolved oxygen level is maintained above 20% saturation by increasing agitation speed, increasing the aeration rate, sparging in oxygen or various combinations thereof.<br>
A carbohydrate solution is fed into the fermentor at a pre-determined rate starting at the beginning of the fermentation run, but generally after 6 hours elapsed fermentation time (EFT), and no longer than 12 hours EFT. The feed is continued until the end of the fermentation. The feed solution can be glycerol prepared at 40-70% v/v or glucose prepared at 40-70% weight/volume (w/v). In certain embodiments, glycerol or glucose are prepared at 70% v/v glycerol and 60% w/v glucose. Feed rates can vary between 5-15 grams of glucose or glycerol per liter per hour. In one embodiment the feed rate is 8, 9, or 10 g/L/hr. At a time of 20 to 30 hours EFT, for example at 24 hours, DPTG is added to the culture to a concentration of 0.5 to 2 mM. Alternatively, lactose at 30% solution can be added at 10 g/1 at 24 hours for induction. At a time of 48 to 56 hours EFT, the fermentation is harvested. Alternatively, an additional 0.5 to 2 mmol/L of<br><br>
IPTG is added to the fermentor culture. The fermentation is then harvested at 52 to 56 hours EFT.<br>
At the end of the fermentation run the temperature is adjusted downward to from 4° to 20°C, and the pH is either maintained or adjusted to 5.0 to 9.0. In certain embodiments, the range is 6.0 to 8.0 pH units. The fermentation broth is harvested by over-pressurization of the vessel and collection of the broth through the sample port. Alternatively, the broth can be pumped out through one of the sample ports. The fermentation broth can contain 10%-30% w/v solids.<br>
FGF18 OR TRFGF18 RECOVERY<br>
Following fermentation the cells are harvested by centrifugation, re-suspended in homogemzation buffer and homogenized, for example, in an APV-Gaulin homogenizer (Invensys APV, Tonawanda, New York) or other type of cell disruption equipment, such as bead mills and sonicators. Alternatively, the cells are taken directly from the fermentor and homogenized in an APV-Gaulin homogenizer. Alternatively, the fermentation broth may be diluted with water or buffer prior to homogenization.<br>
In one embodiment, the cells are homogenized directly in the fermentation broth. For example, an APV-Gaulin 1000 or APV-Gaulin 2000 homogenizer is chilled to 4°-15°C for at least 30 minutes. The fermentation broth is passed through the homogenizer and the cell suspension is collected. The homogenizer pressure should be set at 6000 to 14,000 psi for maximum cell disruption. In one embodiment, the pressure is set for 10,000 psi. The suspension is passed through the homogenizer between 1-5 times, for example, for 3 passes. In another embodiment, the broth is diluted with an equal volume of water prior to homogenization. The amount of DNA may be decreased by the addition of PEI, spermine or benzonase during or after the homogenization step.<br>
The homogenate is centrifuged, and the pellet containing the inclusion bodies is obtained after decanting the supernatant. The inclusion body pellet is washed in water, or Tris buffers with or without varying levels of the following compounds: sodium chloride, urea, Triton X-100, zinc chloride, sodium lauryl sulfate, sucrose.<br>
In another embodiment, the .cells are harvested by transferring the fermentation broth to centriftxge bottles and centrifuging at 2-8°C for 20-60 minutes. For example, a Beckman J6MI centrifuge with KompSpin KAJ7.100 rotor (Beckman Coulter, Fullerton, CA) at 7500 x G can be used to harvest cells. A Beckman Avanti JHC centrifuge with a Beckman JLA-8.1 fixed angle rotor (8,000 -15,800 x G) or an Aries JS 5.0 Swinging Bucket rotor with 2.25 L bottles at 7500 X G can be used as well.<br><br>
A continuous centrifuge such as those supplied by Cair Separations, Inc. (Franklin, MA) or Westfalia Separator, Inc. (Northvale, NJ) can also be used.<br>
The culture broth or supernatant is removed from the centrifuge bottles. The cell pellets are resuspended in homogenization buffer (100 mM Tris, 5 mM ZnCl23 pH 7.5) at 10-30% w/v solids. The fermentation broth is passed through the APV-Gaulin homogenizer and the cell suspension is collected. The homogenizer pressure should be set at 6000-14,000 psi for maximum cell disruption. In one embodiment, the pressure is 10,000 psi. The suspension is passed through the homogenizer for 1-5 passes, for example, 3 passes.<br>
Additionally, the methods of recovering FGF18 or trFGFl 8 can comprise a further step of precipitating, washing, and resolubilizing the FGF18 or trFGFl8. The washed inclusion bodies are solubilized in 6 M guanidine or 8 M urea, diluted 6-10 fold in water or buffer, incubated 30 minutes, and centrifuged or filtered. Alternatively, ultrafiltration or macrofiltration can be used wash inclusion bodies after homogenization. The resulting precipitate is washed in 2-6 M urea, and contains the FGF18 or trFGF18 protein. The precipalate is then washed with water prior to solublization. Addition of Al + or Fe3+ or anionic and cationic polymers or agents such as spermine, PEI and benzonase may be added to precipitate cell debris, soluble proteins, DNA, RNA, and carbohydrates.<br>
SOLUBILIZATION OF INCLUSION BODIES<br>
The washed inclusion body prep can be solubilized using guanidine hydrochloride (5-8 M), guanidine thiocyanate (5-6 M), or urea (7-8M) containing a reducing agent such as beta mercaptoethanol (10 - 100 mM), or dithiothreitol (5-50 mM). The solutions can be prepared in Tris, phopshate, HEPES or other appropriate buffers. Inclusion bodies can also be solubilized with urea (2-4 M) containing sodium lauryl sulfate (0.1-2%). Inclusion bodies from 1 liter of fermentation broth can be solubilized using 50 - 200 ml of the described solutions. The one method provides solubilizing the inclusion body pellets from 1 liter of fermentation broth in 150 ml of 6 M GuHCl prepared in 100 mM Tris, pH 8.0, containing 40 mM DTT. In another embodiment, an inclusion body slurry is mixed with 50-100 ml 8 M GuHCL. The slurry is re-suspended by mixing with a spatula followed by homogenization with an Omni EZ homogenizer (Omni International, Warrenton, VA) or mixing with a mechanical device. The suspension is mixed for 30 - 120 minutes, at 3-37°C. In one embodiment, the suspension is mixed at 15-25°C, to finish the solubilization process. The sample is then centrifuged at 7,500 - 16,000 x G at 4°C for 10 -30 minutes using an appropriate<br><br>
centrifuge. The supernatant sample containing the solubilized FGF18 or trFGFIS is decanted and retained.<br>
The concentration of the FGF18 or trFGF18 in the solubilized fraction is determined by reversed phase HPLC. A Jupiter C5 column (Phenomenex, Torrance, CA) is used with acetonitrile/trifluoroacetic acid as the mobile phase. FGF18 or trFGF18 standard is diluted in a guanidine/DTT/Tris-containing buffer and different amounts are injected onto the column. The area under the FGF18 or ixFGF18 peak is used to construct a standard curve. The solubilized FGF18 or trFGF18 sample is microfuged to remove particulates prior to injection on the HPLC column. Determination of the area under the FGF18 or trFGF8 peak allows quantification of the FGF18 or trFGFIS concentration from the standard curve.<br>
Additionally, the solubilized FGF18 or trFGF18 may be purified at this stage using tangential flow filtration, reverse phase HPLC of immobilized metal affinity chromatography.<br>
REFOLDING<br>
In one aspect of the invention, the process for recovering purified FGF18 or trFGF18 from transformed E. coli host strains in which the FGF18 or trFGF18 is expressed as refractile inclusion bodies, the cells are disrupted and the inclusion bodies are recovered by centrifugation.<br>
The inclusion bodies are then solubilized and denatured in 50 mM TRIS pH 8 and 6 M guanidine hydrochloride containing a reducing agent. The reduced FGF18 or trFGF18 is then oxidized in a controlled renaturation step. This step involves dilution in a refold buffer comprising 50 mM Tris and 120 mM NaCL. This buffer can also comprise arginine hydrochloride, additional salts, and an oxido-shuffling system, although these components are not necessary for successful refolding of either FGF18 or trFGF18. The oxido-shuffling system is used to initiate disulfide bonding of the FGF18 or trFGF18 molecule, and is based on mixtures of reduced and oxidized molecules such as cysteine and cystine, DTT and cystine, reduced glutathione and oxidized glutathione, and DTT and oxidized glutathione. The ratio of reduced to oxidized glutathione can range from 1:1 to 6:1 with a concentration range of 0.5 and 8 mM. In one embodiment, the optimal concentration is 4 mM reduced glutathione: 2 mM oxidized glutathione. The ratio of cysteine to cystine can range from 2:1 to 1:1 with a concentration range of 4 mM to 1 mM of either reagent. In one embodiment, the optimal concentration is 4 mM cysteine, with 2 mM cystine. Optimal refolding may also be achieved using 4 mM cystine and 2 mM DTT which form 4 mM cysteine and 2 mM cystine. Refolding may also be done by sulfitolysis in the presence of reagents such as sodium sulfite and sodium<br><br>
tetrathionate. This process "caps" free cysteines. Because FGF18 or trFGF18 contains only one disufide bond, this step is not generally necessary for successful refolding of FGF18 or trFGF18. The renatured FGF18 or trFGF18 is captured from the dilute refold buffer using cation exchange chromatography, and is purified using hydrophobic interaction chromatography and high performance cation exchange chromatography.<br>
The solute containing FGF18 or trFGF18 is added rapidly (1-5 minutes), or slowly (0.5-5 hours) to the refolding buffer with mixing. The refolding buffer contains Tris and NaCl but can also comprise arginine (0.5 to 1.25 M), PEG, and other salts. It may also include glycerol, guanidine HC1, urea, EDTA, protease inhibitors and chaperones, alcohol, detergents, glycerol and copper sulfate. The FGF18 or trFGF18 can be added in one addition, in multiple additions, or fed in over time. The FGF18 or trFGF18 is added to the refolding mixture to a final concentration of 0.05 to 1.2 mg/ml. The temperature range is 4-30°C. The pH is 7.3 to 8.5. The vessel containing, the refold mixture is left open to the atmosphere or can be sparged with air or nitrogen during renaturation. The refolding is allowed to take place 1 to 26 hours. In particular, in specific embodiements of the invention, the refolding time is 5 to 20 hours, 10 to 20 hours, 15 to 18 hours, or about 18 hours.<br>
Refolding can also be done in the presence of EDTA to decrease methionine oxidation, or on a size exclusion column, or using tangential flow filtration, or electrodialysis.<br>
CLARIFICATION AND CONCENTRATION OF REFOLDED FGF18 OR TRFGF18<br>
Refolded FGF18 or trFGF18 is passed through a 1.2μm filter for clarification and removal of insoluble protein. Because FGF18 and trFGF18 have a high PI, a drop in pH is general not necessary to clarify and concentrate the proteins. The filtered solution is concentrated 10-30 fold using tangential flow filtration on a plate and frame system or with a hollow fiber cartridge. The concentrate is then diluted 3-10 fold with buffer or water to allow unfolded and aggregated proteins to precipitate. The solution is then passed through a filter for clarification and removal of insoluble protein.<br>
Surprisingly, FGF18 or trFGIS can be refolded without the use of arginine, although its presense would not be expected to adversely effect the process.<br><br>
CAPTURE OF FGF18 OR TRFGF18<br>
In another aspect of the present invention, after the FGF18 or trFGF18 protein is refolded and concentrated, the methods of the present invention comprise capturing the refolded FGF18 or trFGF18 protein is captured in dilute buffer on a cation exchange column and purifying FGF18 or trFGF18 protein using hydrophobic interaction chromatography and high performance cation exchange chromatography.<br>
The capture step is designed to capture the diluted, folded FGF18 or trFGF18 and cany out initial purification. The clarified, diluted FGF18 or trFGF18 is captured on a cation exchange column at pH 8.0. Typically, SP Sepharose XL (Amersham Biosciences, Piscataway, NJ) or TOYOPEARL SP 550C (Tosoh Biosep, Montgomery, PA) is used. The equilibration buffer is the same as the refolding buffer described above and the bound FGF18 or trFGF18 is eluted with an increasing salt gradient. FGF18 or trFGF18 eiutes from the SP Sepharose XL using a NaCl step or NaCl gradient<br>
Expanded bed chromatography can also be used for FGF18 or trFGF18 capture following refolding. In that case the dilution step is carried out in-line while loading the FGF18 or trFGF18 onto the column. Streamline SP XL (Amersham Biosciences) is equilibrated with refolding buffer. FGF18 or trFGF18 is then loaded in upflow mode onto the equilibrated Streamline SP XL resin, which is maintained at twice the settled bed height, while diluting 1:3 inline with water. Following washing in both upflow and downflow modes, FGF18 or trFGF18 is eluted in downflow mode with a NaCl step or a NaCl gradient.<br>
The methods of the present invention provide the use of many different cation exchange resins for this step, including weak cation exchangers such as carboxymethyl, different types of solid supports such as agarose or cellulose, and different particle sizes. The methods of the present invention can also provide running the columns at different pHs in the range from 5.0 to 9.0, and with different buffers and salts. Alternatively, other chromatographic methods such as hydrophobic interaction, anion exchange, and metal chelate maybe used to capture the refolded FGF18 or trFGF18.<br>
PURIFICATION<br>
In one aspect of the present invention, there is an intermediate purification of FGF18 or trFGF18 protein. This step is designed to achieve further purification of the FGF18 or trFGF18 using hydrophobic interaction chromatography. Typically Butyl Sepharose FF (Amersham Biosciences) or TOYOPEARL butyl 650M (Tosoh Biosep) are resins used for this step.  The resin is equilibrated with 50 mM Tris, 1M NaCl, 1M<br><br>
(NH4)2S04) pH 8.0. FGF18 or trFGF18 that has been purified by cation exchange chromatography is adjusted to 1.5 M (NKL^SCU and then passed through a 0.45 (xm nominal filter. The adjusted and filtered FGF18 or trFGF18 is then loaded onto the equilibrated resin, which is then washed with equilibration buffer to remove unbound material FGF18 or trFGF18 is eluted with a linear gradient of PBS (7mM Na2HP04, 1.5 mM KH2PO4, 137 mM NaCI, 2.7 mM KC1, pH 7.5) and equilbration buffer. FGF18 or trFGF18 elutes from the column at approximately 120 mS/cm.<br>
Other hydrophobic interaction chromatography resins that can be used for this step include, for example, those substituted with phenyl or hexyl, different types of solid supports such as agarose or cellulose, and different particle sizes. The present invention also provides running the columns at different pHs in the range from 5.0 to 9.0, and with different buffers and salts. The present invention also provides running the column in such a manner that FGF18 or tri?GF18 does not bind.<br>
The final column eluate pool is concentrated using a 5 kDa molecular weight cut-off tangential flow filtration plate and frame membrane, diafiltered against PBS, pH 7.3, described above. Other membranes can be used, such as a 3 kDa or 8 kDa molecular weight cut-off plate and frame membrane or a 10 kDa molecular weight cutoff hollow fiber system to achieve this ultrafiltration/diafiltration step. The purity of the FGF18 or trFGF18 following these procedures is at least 95%, and typically greater than 98%, by sodium dodecyl sulfate polyacrylamide gel electrophoresis. The endotoxin level in the FGF18 or trFGF18 preparation following cation exchange chromatography capture, hydrophobic interaction chromatography purification, and buffer exchange, is generally 
Further purification of FGF18 or trFGF18 to remove the remaining impurities and contaminants may be desirable. For example, an anion exchange column can be used to reduce the endotoxin level. FGF18 or trFGF18 is diluted to a conductivity level of 
Other purification steps that could potentially be used to further purify FGF18    or   trFGF18    include   metal    chelate   chromatography,    anion    exchange<br><br>
chromatography, or hydrophobic interaction chromatography on a phenyl column. It is also possible to carry out purification prior to refolding the FGF18 or trFGF18, using for example reversed phase HPLC, ion exchange chromatography or metal chelate chromatography. Thus, the present invention further provides methods comprising the additional steps of purification disclosed herein.<br>
CHARACTERIZATION OF PURIFIED FGF18 OR TRFGF18<br>
Assays measuring cell proliferation or differentiation are well known in the art. Specifically, biological activity of FGF18 or trFGF18 can be measured using a BaF3 or<br>
Y<br>
CCC4 assay, described in detail below. Additional well know assays include assays measuring proliferation include such assays as chemosensitivity to neutral red dye (Cavanaugh et aL, Investigational New Drugs .8:347-354, 1990, incorporated herein by reference), incorporation of radiolabeled nucleotides (Cook et al., Analytical Biochem. 179:1-7, 1989, incorporated herein by reference), incorporation of 5-bromo-2'-deoxyuridine (BrdU) in the DNA of proliferating cells (Porstmann et al., J- Immunol. Methods 82:169-179, 1985, incorporated herein by reference), and use of tetrazolium salts (Mosmann, J. Immunol. Methods 65:55-63, 1983; Alley et al., Cancer Res. 48:589-601, 1988; Marshall et al., Growth Reg. 5:69-84, 1995; and Scudiero et al., Cancer Res. 48:4827-4833, 1988; all incorporated herein by reference). Assays measuring differentiation include, for example, measuring cell-surface markers associated with stage-specific expression of a tissue, enzymatic activity, functional activity or morphological changes (Watt, FASEB, 5:281-284,1991; Francis, Differentiation 57:63-75, 1994; Raes, Adv. Anim. Cell Biol. Technol. Bioprocesses, 161-171, 1989; all incorporated herein by reference). FGF18 or trFGF18 produced by the methods described herein is capable of stimulating proliferation of BaF3/FGF18R cells.<br>
Purified FGF18 or trFGF18 can be characterized by a number of physical methods. Optimally, amino acid analysis indicates the amino acid composition of all residues is within 10% of the expected values. N-terminal sequencing gives a single sequence beginning with methionine and corresponding to the sequence predicted from the FGF18 or trFGF18 expression vector. Endoproteinase Lys C digestion followed by liquid chromatography-mass spectrometry can be used to generate a peptide map in which all peaks correspond in mass to predicted tryptic peptides in FGF18 or trFGF18, and in which all predicted tryptic peptides from FGF18 or trFGF18 are identified.<br>
The invention is further illustrated by the following non-limiting examples.<br><br>
EXAMPLES<br>
Example 1<br>
Construction of expression vector, pTAP237<br>
Plasmid pTAP237 was generated by inserting a PCR-generated linker into the Smal site of pTAP186 by homologous recombination. Plasmid pTAP186 was derived from the plasmids pRS316 (a Saccharomyces cerevisiae shuttle vector) and pMAL-c2, an E. coli expression plasmid derived from pKK223-3 and comprising the tac promoter and the rrnB terminator.. Plasmid pTAP186 contains a kanamycin resistance gene in which the Sma I site has been destroyed and has NotI and Sfil sites flanking the yeast ARS-CEN6 and URA3 sequences, facilitating their removal from the plasmid by digestion with NotI. The PCR-generated linker replaced the expression coupler sequence in pTAP186 with the synthetic RBS II sequence. It was prepared from 100 pmoles each of oligonucleotides zc29,740 and zc29,741, as shown in SEQ ID NOS: 19 and 20, respectively, and approximately 5 pmoles each of oligonucleotides zc29,736 and zc29,738, as shown in SEQ ID NOS: 21 and 22, respectively. These oligonucleotides were combined by PCR for ten cycles of 94°C for 30 seconds, 50°C for 30 seconds, and 72°C for 30 seconds, followed by 4°C soak. The resulting PCR products were concentrated by precipitation with two times the volume of 100% ethanol. Pellet was resuspended in 10 fiL water to be used for recombining into the recipient vector pTAP186 digested with Smal to produce the construct containing the synthetic RBS II sequence. Approximately 1 fig of the PCR-generated linker and 100 ng of pTAP186 digested with Smal were mixed together and transformed into competent yeast cells (5. cerevisiae). The yeast was then plated onto -URA D plates and left at room temperature for about 72 hours. Then the Ura+ transformants from a single plate were resuspended in 1 mL H2O and spun briefly to pellet the yeast cells- The cell pellet was resuspended in 0.5 mL of lysis buffer. DNA was recovered and transformed into E. coli MC1061. Clones were screened by colony PCR as disclosed above using 20 pmoles each of oligonucleotides zc29,740 and zc29,741, as shown in SEQ ID NOS: 19 and 20, respectively. Clones displaying the correct size band on an agarose gel were subject to sequence analysis. The correct plasmid was designated pTAP237.<br>
Example 2<br>
Codon Optimization<br>
Test expression of human FGF18 in E. coli produced lower than ideal levels, as well as truncated products, specifically truncation of 11 amino acids from the C-terminal end of the protein. To address the low level of expression, examination of the codons used in the FGF18 coding sequence indicated that it contained an excess of<br><br>
the least frequently used codons in E. coli with a CAI value equal to 0.248. The CAI is a statistical measure of synonymous codon bias and can be used to predict the level of protein production (Sharp et al., Nucleic Acids Res. 15(3): 1281-95, 1987). Genes coding for highly expressed proteins tend to have high CAI values (&gt; 0.6), while proteins encoded by genes with low CAI values (
The production of truncated products was a second inefficiency in the process, requiring isolation of the truncated portion from the produced proteins. Once it was determined that the truncated protein had similar, if not identical biological activity to the full-length protein, a construct that deliberately produced the shorter product was constructed. Specifically, the protein was stopped after amino acid 196 (Lys).<br>
This left the codon issue. Thus, an attempt was made to resynthesizing the gene coding for FGF18 and trFGF18 with more appropriate codon usage provides an improved vector for expression of large amounts of FGF18 or trFGF18.<br>
The codon optimized trFGF18 coding sequence was constructed from twelve overlaping oligonucleotides: zc44,349 (SEQ ID NO:7), zc44,350 (SEQ ID NO:8), zc44,351 (SEQ ID NO:9), zc44,352 (SEQ ID NO: 10), zc44,355 (SEQ ID NO: 11), zc44,356 (SEQ ID NO:12), zc44,357 (SEQ ID NO:13), zc44,358 (SEQ ID NO:14), zc44,359 (SEQ ID NO: 15), zc44,360 (SEQ ID NO:16), zc44,361 (SEQ ID NO:17), zc44,362 (SEQ ID NO:18). Primer extension of these overlapping oligonucleotides followed by PCR amplication produced a full length trFGF18 gene with codons optimized for expression in E. coli.. The final PCR product was inserted into expression vector pTAP237 by yeast homologous recombination. The expression construct was extracted from yeast and transformed into competent £. coli DH10B. Clones resistance to kanamycin were identified by colony PCR. A positive clone was verified by sequencing and subsequently transformed into three production host strains BL21, W3110, and zGoldl. The expression vector with the optimized trFGF18 sequence was named pSDH174.<br>
Example 3<br>
Construction of pSDH170 and pSDH174<br>
The human FGF18 coding sequence was generated by PCR amplification using a CD3+ cDNA library pool as template and oligonucleotide primers. To optimize the translation process in E. coli, a primer was used to add an ATG initiation codon to the 5' end of the mature FGF18 coding sequence. The resulting gene sequence encoded the mature FGF18 with one extra methionine at the N-terminus. The final PCR product was inserted into expression vector pTAP237  (described in Example  1) by yeast<br><br>
homologous recombination (Raymond et al., Biotechniques. 26(1}: 134-8, 140-1, 1999; U.S. Patent 6,027,442, incorporated herein by reference). The expression construct, pSDH170 (SEQ ID NO:l), was extracted from yeast and transformed into a competent E. coli DH10B. Kanamycin resistant clones were identified by colony PCR. A positive clone was verified by sequencing and subsequently transformed into either production host strains BL21, W3110, and zGOLDl (described below).<br>
Example 4 Expression of trFGF18<br>
E. coli were inoculated into 100 mL Superbroth II medium (Becton Dickinson, Franklin Lakes, NJ) containing 0.01% Antifoam 289 (Sigma-Aldrich, St. Louis, MO) and 30 Mg/ml kanamycin, and cultured overnight at 37°C. A 10 mL inoculum was added to 500 mL of same medium in a 2 L culture flask that was shaken at 275 rpm at 37°C until the culture attained an ODaoo of 4. IPTG was then added to a final concentration of 1 mM and shaking was continued for another 2.5 hours. The cells were centrifuged at 4,000 x g for 10 min at 4°C. The cell pellets were frozen at -80°C for use at a later time.<br>
Expression of trFGF18 was performed on a larger scale in a 25 mL culture at 37°C. One mL of culture was collected 2 hours after IPTG induction. E. coli cells were resuspended in an equal volume BugBuster® Protein Extraction Reagent (Novagen, Madison, WI) at 4°C and incubated for 20 min. The soluble and insoluble fractions were separated by centrifugation at 16,000.x g for 10 min at 4°C.<br>
Recombinant trFGF18 accumulated as insoluble inclusion bodies. The recovery yield of trFGF18 from most of the E. coli strains was considered low. About 80 to 90% of trFGF18 in the inclusion bodies was lost within 20 min after cell lysis and incubation at 4°C. Lysing bacteria with 8 M urea did not improve recovery. However, including protease inhibitors, such as 5 mM ZnCfe and 0.5 mM Benzamidine, in the cell lysis buffer prevented the loss of trFGF18 from strain E104 (W3110 arabinose"). This indicated that a bacterial protease capable of cleaving trFGF18 under denaturing conditions was co-purifying with the inclusion bodies. It was observed that txFGF18 was stable in cells lysates from strain UT5600, but not in E104 cell lysates. This suggested that the protease was present in El 04 but not UT5600. Comparison of the genotypes of these strains revealed that OmpT, which cleaves between dibasic residues, was present in E104 but not in UT5600. OmpT is heat stable and active even under denaturing conditions (White et al., ibid. 1995). Examination of the amino acid sequence of FGF18 indicated that it contained at least eight potential OmpT cleavage sites. trFGF1.8 also demonstrated excellent stability in BL21, another OmpT deficient E. coli strain. These<br><br>
data suggested that OmpT protease activity was critical for the stability and recovery of FGF18. The use of E. coli strain UT5600 as the production host significantly improves the recovery of trFGF18. Overall the yields of trFGF18 were increased from 2 mg/L to 50-100 mg/L with the combination of construct and host strain improvement.<br>
Example 5 Characterization of FGF18<br>
The protein produced by the processes described above can be analyzed for polypeptide integrity and biological activity using methods well known in the art. For example, the protein can be examined using Western analysis.<br>
For Western analysis, protein samples were separated on a 4-20% MES-SDS NuPAGE gel (Invitrogen) under reducing conditions and transferred to nitrocellulose membrane (Invitrogen) at 30 V for 1 hour. The membrane was blocked with 5% non-fat milk in TIBS buffer (20 mM Tris pH 7.4, 160 mM NaCl, 0.1% Tween 20), Polyclonal antibody specific for human FGF18 was added in TIBS Buffer with 5% non-fat milk and incubated for 1 hour. After washing with TIBS, the blot was probed with HRP conjugated goat-anti rabbit IgG (Bio-Rad) for 1 hour. The blot was subsequently washed three times with TIBS before chemiluminescent detection with ECL reagent from Pierce.<br>
The activity of the FGF18 or trFGF18 protein can be examined using the BaF3 assay as described in Ellsworth et al„ Osteoarthritis Cartilage,lO:308(2002Y A second assay is the osteoblast cell line CCC4 assay.<br>
A high throughput in vitro bioassay for FGF18 was developed using an immortalized osteoblast cell line (CCC4) derived from P53 knockout mice. A P53 osteoblast line expressing the serum response element (SRE)-luciferase reporter system was established. The clonal cell line used was derived by transfecting the immortalized P53 osteoblast cell line CCC4 with a SRE-luciferase/puromycin reporter construct KZ125 and selecting for clones with high luciferase inducibility by serum, PDGF, and bFGF. The assay in its current format is sensitive, reproducible, and results can be generated within one day. CCC4/KZ125 is used in all experiments. Growth Medium:        alpha-MEM    500 ml<br>
15%HIA-FBS75ml<br>
1	mM Na Pyruvate     5 ml 2mM L-Glutamine     5 ml<br>
2	ug/ml puromycin     20 ul of 50 mg/ml stock<br><br>
Plating Medium:        alpha-MEM    500 ml<br>
l%mA-KBS 5 ml 1 mM Na Pyruvate     5 ml 2mM L-Glutamine     5 ml<br>
Assay Medium:	DMEM	500 ml<br>
1% BSA	66.6 ml of 7.5% BSA<br>
20mMHEPES	10 ml<br>
1 mM Na Pyruvate     5 ml<br>
2mML-Glutamine     5 ml<br>
(the BSA was tissue culture grade)<br>
Maintenance of Cell CCC4 cells: Cells are passaged twice weekly using split ratios of 1:10 (confluent in 2 days), 1:20 (confluent in 3 days), 1:40 (confluent in 4 days).<br>
Preparation of Assay Plates: Cells are trypsinized, resuspended in growth medium, then centrifuged at 1000 ipm for 5 min. The cell pellet is resuspended in plating medium, counted on a hemacytometer, adjusted to 2 X 105 cells/ml, and plated at 100 ul/well in Dynatech Microlite opaque white microtiter plates (Dynatech, #011-010-7418 and Linbro lids, ICN Biomedical, #76-205-05) and allowed to grow overnight at 37°C, 5%<br>
co2.<br>
Assay of SRE-Luciferase Induction:<br>
Test substances, bFGF and serum controls are diluted in assay medium to a concentration that twice (2X) the final desired assay concentration (FAC) and added (100 ul/well) to assay plates for a final assay volume of 200 ul/well. If test samples are in DMSO then samples are diluted such that the FAC of DMSO does not exceed 1% (or 0.25% DMF). Assay medium, without test substances or control compounds, is added to 4 wells/assay plate to determine the basal (uninduced) luciferase expression on each plate. bFGF at 3 ng/ml FAC is added to 4 wells/plate to determine the maximum possible luciferase induction for each plate. On at least one plate in each assay set a full bFGF (0.001 - 10 ng/ml FAC) dose response curve is run. If test substance stocks are in DMSO (or DMF), then DMSO (or DMF) is added to the basal and bFGF controls at a final assay concentration (FAC) equal to that in the test substances.<br><br>
Plates are incubated for 4 hours at 37°C, 5% C02. Assay medium is dumped from plates and IX lysis buffer (25ul/well) is added (Promega, #1501). Plates are then incubated for 15 min. at room temperature. (Plates may be frozen at this point for later assay of luciferase activity). Luciferasd substrate (Promega, #E1501) is added, 50 ul/well, using the Labsystems Luminoskan luminometer to do the substrate addition. Luciferase signal (RLU) is read for 2 seconds/well following a 1 second delay.<br>
Data Calculation: Average basal (uninduced) signal is subtracted from all readings and results are expressed as a percentage of the maximum induction produced by 3 ng/ml bFGF. The concentration of test substance (in ug/ml for crude extracts or uM for pure compounds) that produces 50% of the maximum possible luciferase induction is calculated (EC50).<br>
Typical standard curves for E. coli-derived recombinant human FGF18 spiked into assay medium are shown in Figures 1 and 2. The linear range and lower limit of detection of the assay are 0-3 ng/mL and -0.2 ng/rnL, respectively.<br>
Example 6<br>
Plasmid Stability Analysis<br>
E. coli was inoculated into 25 mL Superbroth II medium (Becton Dickinson) containing 0.01% Antifoam 289 (Sigma) and 30 Jig/ml kanamycin, and cultured overnight at 37°C. A 25 pL inoculum was added to 25 mL of same medium without kanamycin in a 25 mL culture flask which was shaken at 275 rpm at 37°C. 100 {iL of culture were collected at four different time points (when the culture reached OD$oo values of 2, 4, 6 and 8). The samples were diluted and plated on LB agar plates without any additives. After overnight incubation at 37°C, 100 E. coli colonies were replica plated onto a LB agar plate and a LB agar plate containing 30 |ig/ml kanamycin. After overnight incubation at 37°C, the number of colonies that formed on each plate was counted and compared. The number of colonies that grew on LB plus kanamycin relative to the number that grew on the plate without antibiotic reflected the percentage of cells still harboring the expression vector.<br>
When clones of W3110 or zGOLDl carrying the pSDH170 or pSDH174 expression vector were cultured for 12 hours in medium that did not contain kanamycin, more than 90% retained the plasmid. Clones carrying the expression vector without the FGF18 gene showed similar retention of the plasmid. These data demonstrate that the pSDH170 or pSDH174 expression vector carrying FGF18 or trFGF18 is stable in these production strains.<br>
E. coli strains, TGI and MM294, were not selected as the production host due to low productivity of FGF18 and serious plasmid instability. The most encouraging<br><br>
results came from the studies using E, coli strain W3110 (ATCC #27325) to produce FGF18. The productivity of W3110 was comparable to that of UT5600. Plasmid stability studies demonstrated that the expression vector, pTAP337, was maintained in W3110 as well. UT5600 is an auxotrophic strain and more difficult to grow at large scale. These considerations led to selection of W3110 as the preferred host strain for production of FGF18.<br>
Example 7<br>
Batch Fermentation<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with Difco APS Super Broth (Difco Laboratories, Detroit, MI), supplemented with glycerol at 5 g/L and kanamycin at 25 ug/ml. Growth was started by inoculating the shake flask with a loop full of E.coli W3110 or zGOLDl containing the expression vector pSDH170 or pSDH174 from a 24 hour old agar plate (Luria agar (Difco Laboratories) containing kanamycin 25 p,g/ml). Growth in the shake flask was at an appropriate temperature (generally, 37°C). The flask was incubated with agitation set at 250 rpm.<br>
A 2 L fermentation vessel was prepared with 2.0 L of Difco APS Super Broth and sterilized. The growth medium was supplemented with glycerol at 50 g/L and kanamycin at 25 ug/ml plus 5 mL/L 1 M MgS04- Aeration of the vessel was set to 1 vvm and agitation was set at 350 rpm. The temperature was set to 37°C. The fermentor was inoculated from a first stage seed flask culture grown for 16 hours to an optical density (OD) of 10-20 at 600 nm. The inoculation was 5% v/v. Dissolved oxygen was maintained above 20% saturation by increasing agitation speed or utilizing O2 supplementation if necessary, as is well known in the art.<br>
The culture was grown until the ODeoo reached 2.5 (approx 2.5 hours). IPTG was added to the culture to a concentration of 1.0 mM. The culture was then allowed to grow for an additional 2.5 hours.<br>
Example 8<br>
A.       Fed Batch Fermentation<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with Difco APS Super Broth, supplemented with glycerol at 5 g/L and kanamycin at 25 ug/ml. Growth was started by inoculating the shake flask with a loop full of E.coli W3110 or zGOLDl (described below) containing the expression vector pSDH170 or pSDH174 (described above) from a 24 hour old agar plate (Luria agar<br><br>
containing kanamycin 25 |ig/ml). The shake flask was incubated at 30°C with agitation set at 250 rpm.<br>
A 6 L fermentation vessel was prepared with 3.0 L of ZymoM growth medium and sterilized. The growth medium was supplemented with glycerol at 20 g/L and kanamycin at 25 μg/ml. The pH of the medium was adjusted to pH 6.4. Aeration was set to 1 vvm,agitation to 350 rpm, and temperature to 32°C. The fermentor was inoculated from a first stage seed flask culture that had been grown for 16 hours to an ODeoo of 10-20. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed<br>
A carbohydrate solution was fed into the fermentor starting at 10 hours EFT. The feed was continued until the end of the fermentation. The feed solution was glycerol prepared at 70% v/v. The feed rate was 6 grams of glycerol per liter per hour based on the initial starting volume. At 24 hours EFT, IPTG was added to the culture to a concentration of 2 mM. At 48 hours EFT, the fermentation was harvested.<br>
In an alternative fed batch process, a first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM, supplemented with glucose at 20 g/L and kanamycin at 25 (μg/ml. Growth was started by inoculating the shake flask with 300 μl Exoli W3110 frozen in 20% glycerol and containing the expression vector pTAP337. The culture was incubated at 30°C with agitation at 250 rpm.<br>
A 6 L fermentation vessel was prepared with 3.0 L of ZymoM growth medium and sterilized The growth medium was supplemented with glucose at 20 g/L and kanamycin at 25 (μg/ml. The pH of the medium was adjusted to 6.8. Aeration was set to 1 vvm, agitation to 350 rpm, and temperature to 37°C. The fermentor was inoculated from a first stage seed flask culture that had been grown for 16 hours to an OD600 of 16. Inoculation was 5% volume/volume and the dissolved oxygen level was maintained above 20% saturation by increasing agitation speed.<br>
A carbohydrate solution was fed into the fermentor starting at 10 hours EFT. The feed was continued until the end of the fermentation. The feed solution was glucose prepared at 60% v/v and the feed rate was 9.5 grams of glucose per liter per hour based on the initial starting volume. At 24 hours EFT, IPTG was added to the culture to a concentration of 2 mM. At 48 hours EFT, 2 mmol/1 of IPTG was added to the culture bringing the IPTG concentration to 4 mM. The fermentation was harvested at 56 hours.<br><br><br>
B.	Fed batch fermentation with PCOL22 medium<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial of W3110 or zGOLDl stably tranformed with pSDH170 or pSDH174. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 2 L fermentation vessel was prepared with 1.2 L of PCOL22 medium (ZSM plus the Trace D solution as described below) and sterilized. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25μg/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 37. The fermentor was inoculated from a first stage seed flask culture of EE410 that had been grown for 16 hours to an ODeoo nm of 10-16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed plus O2 supplementation. pH was controlled at 6.8 by addition of 5 N NH4OH.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 10 g of glucose / L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 1 mM. The fermentation was harvested at 48 hours EFT.<br><br>
C.	Fed batch fermentation with PCOL22 medium minus kanamycin<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25<br><br>
ug/ml. Growth was started by inoculating the flask with 300 μl of material from a thawed frozen vial containing the production strain. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 6 L fermentation vessel was prepared with 2.7 L of PCOL22 medium and sterilized. After cooling the growth medium was supplemented with, glucose at 20 g/Lt magnesium sulfate, and calcium chloride. No kanamycin was added. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a first stage seed flask culture of EE410 that had been grown for 16 hours to an ODgoo nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. pH was controlled at 6.8 by addition of 5 N NH4OH.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose/L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
D. Fed batch fermentation with PCOL22-L medium<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial containing the production strain. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 6 L fermentation vessel was prepared with 2.7 L of PCOL22 -L medium and sterilized. This medium contains citric acid and has one-third less-salts to prevent precipitation. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25 ug/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a first stage seed flask culture of EE410 that had been grown for 16 hours to an ODeoo nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. pH was controlled at 6.8 by addition of 5 N NH4OH.<br>
A glucose solution (60% w/v) minus magnesium sulfate was fed into the fermentor starting at 8 hours EFT, A constant feed rate of 9.5 g of glucose / L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG<br><br>
was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
E.	Fed batch fermentation with PCOL12 -L medium<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial containing the production strain. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 6 L fermentation vessel was prepared with 2.7 L of PCOL22 -L medium and sterilized. This medium contains V4 th less-salts to prevent precipitation. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25μg/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 vvm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a first stage seed flask culture of EE410 that had been grown for 16 hours to an ODeoo nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. pH was controlled at 6.8 by addition of 5 N NH4OH.<br>
A glucose solution (60% w/v) minus magnesium sulfate was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose/L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
F.	Fed batch fermentation with PCOL12 -R medium<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial containing the production strain. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 6 L fermentation vessel was prepared with 2.7 L of PCOL22 -R medium and sterilized. This medium contains increased levels of yeast extract and glucose to increase the growth of the host strain before glucose feeding is initiated. After cooling the growth medium was supplemented with, glucose at 40 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25 μg/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 vvm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a first stage<br><br>
seed flask culture of EE410 that had been grown for 16 hours to an OD600 nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose / L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
G. Fed batch fermentation in 20L vessel<br>
In an alternative fed batch process, a first stage seed vessel (6 1) was prepared with 3.0 L of ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the vessel with 3.0 ml of material from a thawed frozen vial containing the production strain. Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 32 C.<br>
A 20 L fermentation vessel was prepared with 10.8 L of PCOL22 medium and sterilized. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25 ug/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 vvm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from the first stage seed vessel culture of EE410 that had been grown for 16 hours to an ODgoo nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. Culture pH was controlled at 6.8 through addition of 5N ammonium hydroxide.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose / L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
H. Fed batch fermentation with 2 stage seed<br>
A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial containing the production strain. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A second stage seed vessel (6 1) was prepared with 3.0 L of ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 μg/ml. Growth was started by inoculating the vessel with 100 ml of material from a first stage seed flask<br><br>
containing the production strain EE410 (E. coli W3110 containing the expression vector pTAP337). Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 32 C.<br>
A 20 L fermentation vessel was prepared with 10.8 L of PCOL22 medium and sterilized. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25 μg/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 win, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a second stage seed vessel that had been grown for 12 hours to an OD600 nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. Culture pH was controlled at 6.8 through addition of 5N ammonium hydroxide.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose / L starting volume per hour was maintained throughout the fermentation. At 24 hours EFT, IPTG was added to the culture to a concentration of 0.5 mM. The fermentation was harvested at 48 hours EFT.<br>
L   Fed batch fermentation with zGOLD 1<br>
Construction of the expression vector zGOLDl is described in Example 19. A first stage seed flask (baffled 500 ml flask with 100 ml medium) was prepared with ZSM medium, supplemented with glucose at 20 g/L and kanamycin at 25 ug/ml. Growth was started by inoculating the flask with 300 ul of material from a thawed frozen vial containing the production strain E. coli W3110 ompT - (zGOLDl) containing the expression vector pSDH170 or pSHD174. The shake flask was incubated at 32 C with agitation set to 250 rpm.<br>
A 6 L fermentation vessel was prepared with 2.7 L of PCOL22 medium and sterilized. After cooling the growth medium was supplemented with, glucose at 20 g/L, magnesium sulfate, calcium chloride, and kanamycin at 25 ug/ml. The pH of the medium was adjusted to 6.8 with 5N ammonium hydroxide. Aeration was set to 1 wm, agitation was set to 350 rpm, and temperature to 37 C. The fermentor was inoculated from a first stage seed flask culture of EE410 that had been grown for 16 hours to an OD600 nm of 16. Inoculation was 5% v/v and dissolved oxygen was maintained above 20% saturation by increasing agitation speed. Culture pH was controlled at 6.8 through addition of 5N ammonium hydroxide.<br>
A glucose solution (60% w/v) was fed into the fermentor starting at 8 hours EFT. A constant feed rate of 9.5 g of glucose / L starting volume per hour was<br><br><br><br><br><br><br>
Example 9 FGF18	Disruption of harvested cells<br>
The harvested E. coli pellet was produced by fed-batch fermentation, and contained approximately 5-6 g/L of FGF18 or trFGF18 in inclusion body form. The fermentation broth (1L) was pelleted by centrifugation at 8000 x g for 30 minutes The pellet was resuspended in 850 ml of breakage buffer (100 mM Tris, pH 7.2, 5 mM ZnCk) and chilled on ice. The broth was passed through the APV homogenizer three times at 10,000 psi. The broth was then centrifuged at 8000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The pellet was washed twice by resuspension in 800 ml of DI water and centrifugation at 8000 x g for 40 minutes. The supernatant was discarded, taking care to retain the loose pellet. The inclusion body pellet was stored at -80°C or refolded without freezing.<br>
B.	Direct Disruption of harvested broth<br>
The harvested E. coli broth was produced by fed-batch fermentation, and contained approximately 6-7 g/L of FGF18 or trFGF18 in inclusion body form. The fermentation broth (0.5L) was diluted to 1.0 L with deionized water and passed through the APV homogenizer three times at 10,000 psi. The broth was then centrifuged at 15,000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The pellet was resuspended in 500 ml of DI water and centrifuged at 15,000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The washing step was repeated and the inclusion body pellet was stored at -80°C or refolded without freezing.<br>
C.	Solublization and precipitation<br>
1. Solubilization was achieved by suspension of the washed inclusion body pellel in 200 mL of 100 mM Tris, 6 M Guanidine hydrochloride, 5 mM ZnCfe, pH 7.2 at room temperature for one hour. The suspension was then centrifuged at 12000 g for 30 minutes. The supernatant was kept at 4°C. The supernatant was diluted 1:8 (v/v) into 100 mM Tris, 5 mM ZnCk, pH 7.2. The suspension was centrifuged at<br><br>
12000 g for 10 minutes. The supernatant was discarded. The pellet was resuspended in 200 ml of 100 mM Tris, 8 M Urea, pH 7.2. The suspension was centrifuged at 12000 g for 30 minutes. The supernatant was discarded. The washing procedure was repeated two more times. Resolubilization was achieved by suspension of the washed pellet in 200 mL of 100 mM Tris, 6 M Guanidine hydrochloride, 10 mM DTT, pH 72. The suspension was centrifuged at 12000 g for 30 minutes. The protein concentration in the supernatant as measured by HPLC protein assay was 10 mg/mL. The FGF18 sample was then stored at 4°C.<br>
2. The solublization of FGF18 or trFGF18 was achieved by suspending the washed inclusion body body pellet in 6 M Guanidine hydrochloride, 40 mM dithiothreitol (DTT) prepared in 100 Mm Tris, pH 8.0 (GDT40). Approximately 150 ml of GDT40 was used per liter of original ferementation broth. The solublization took place at room temperature for one hour. The suspension was then centrifuged. The supernatant from dissolved inclusion bodies was refolded by dilution (20-30 X) into a refolding buffer containing a 0.75 M arginine plus DTT/cystine oxidation-reduction pair. Refolding was allowed to take place for 5-16 hours after which the pH of the mixture was adjusted to pH 5.5 and filtered prior to delivery to purification.<br>
D. Direct disruption of harvested broth from zGOLDl<br>
The harvested E. coli zGOLDl broth was produced by fed-batch fermentation in PCOL22 medium (described above), and contained approximately 9 - 10 g/L of trFGF18 in inclusion body form. The fermentation broth (0.5L) was diluted to 1.0 L with dionized water and passed through the APV homogenizer three times at 10,000 psi. The broth was then centrifuged at 15,000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The pellet was resuspended in 500 ml of DI water and centrifuged at 15,000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The pellet was resuspended in 500 ml of DI water and centrifuged at 15,000 x g for 30 minutes. The supernatant was discarded, taking care to retain the loose pellet. The inclusion body pellet was stored at -80°C or refolded without freezing.<br>
Example 10<br>
A.       Solublization of Washed Inclusion bodies<br>
Solubilization was achieved by suspension of the washed inclusion body<br>
pellet in 150 mL of 100 mM Tris, 6 M Guanidine hydrochloride, 20 mM dithiothreitol<br>
pH 7.5 at room temperature for one hour. The suspension was then centrifuged at 12000<br><br>
g for 30 minutes. The protein concentration in the supernatant as measured by HPLC protein assay was 21 mg/mL. The FGF18 sample was then stored at 4°C.<br>
B.	Solublization of Washed Inclusion bodies from zGOLDl<br>
Solubilization was achieved by suspension of a washed inclusion body pellet from 1 liter of fermentation broth in 150 mL of 100 mM Tris, 6 M Guanidine hydrochloride, 40 mM dithiothreitol, pH 8.0 at room temperature for one hour. The suspension was then centrifuged at 15, 000 x g for 30 minutes. The protein concentration in the supernatant as measured by HPLC protein assay was 29 mg/mL. The IL21 sample was then stored at 4 °C.<br>
C.	Clarification of solubilized inclusion bodies<br>
Immobilized metal affinity chromatography (IMAC) resin was used to clarify solubilized FGF18 inclusion body pellets. In one example, washed inclusion body pellets were solubilized for 1 hour at room temperature in 6M guanidine HC1 containing lOmM Imidazole, pH 7.5, 1.0 ml Efts-trap columns (Amersham Biosciences) were charged with 0.5 ml of 0.1M NiS04. After charging and water washing, 5.0ml of binding buffer consisting of 6M GuHCl, 20mM Imidazole, 0.5M NaCl, and 20mM phosphate was used to equilibrate the column.<br>
The solute sample (1.0 ml) was applied to the column, and the column was washed with 5.0 ml of the binding buffer. FGF18 was eluted by applying 2.5ml of elution buffer (6M GuHCl, 0.5M Imidazole, 0.5M NaCl, and 20mM phosphate) to the column. The elution step was repeated, and the samples were analyzed for purity and clarification using SDS -Page gels.<br>
Example 11<br>
Refolding<br>
A.        Renaturation<br>
The concentration of FGF18 in the solubilized fraction was determined by reverse phase HPLC to be 21 mg/ml. Determination of the refolding buffer volume was based on the amount of solute and the concentration of FGF18 present in the solute. The refolding buffer (50 mM Tris pH 8 and 120 mM NaCl) was added followed by 0.5M CuS04 (1:1000 v/v) and stirred for 30 minutes.<br>
Example 12<br>
Capture of refolded FGF18<br>
A.        Cation Exchange using TOYOPEARL SP 550 C resin<br><br>
Following concentration, FGF18 is captured on a cation exchange column. In one example, the concentrated FGF18 is diluted 3-fold with water or 25 mM sodium acetate, pH 5.5. A precipitate is formed which is removed by filtration after 30 minutes incubation at room temperature. A Millipore 1.2 μm Polysep II filter (Millipore) or a 1.2-0.8 μm Cuno Zeta Plus A30MO3 membrane (Cuno, Meriden, CN) is used. The filtered FGF18 is loaded onto a column of TOYOPEARL_SP550C resin (Tosoh Biosep) equilibrated to equilibration buffer (25 mM sodium acetate, 0.2 M NaCl, pH 5.5). The column is loaded at a capacity of 6-10 g FGF18 per L resin, the bed height is 15 cm, UV absorbance at 280 nm and 215 nm is monitored, and a flow rate of 150 cm/hr is used. Following loading the column is washed with equilibration buffer until the UV absorbance returns to baseline. The column is then washed with 4 column volumes of 50 % equilibration buffer, 50 % elution buffer (25 mM sodium acetate, 1.0 M NaCl, pH 5.5). FGF18 is eluted from the column with 25 % equilibration buffer, 75 % elution buffer. Alternatively, following loading of FGF18 onto the column and washing with equilibration buffer, FGF18 is eluted from the column with a 10 column volume linear gradient from 100 % equilibration buffer to 100 % elution buffer.<br>
Alternatively, following pH adjustment, dilution, hold step, and filtration using depth filtration, the FGF18 is captured on cation exchange chromatography. The filtered solution is loaded onto a column of TOYOPEARL SP 550 C resin (Tosoh Biosep) and equilibrated to equilibration buffer conditions (25 mM sodium acetate, pH 5.5, 0.4 M NaCl). The column is loaded at a capacity of 6 to 15 g FGF18 per L resin. UV absorbance at 280 nm and 215 nm is monitored, and a flow rate of 150 cm/hr is used. Following loading, the column is washed with equilibration buffer until the UV absorbance returns to baseline. FGF18 is eluted from the column using a step gradient to 100% elution buffer (25 mM sodium acetate, pH 5.5,0.75 M NaCl).<br>
B.        Cation Exchange Chromatography Using SP Sepharose XL resin<br>
The concentrated FGF18 is diluted 10-fold with 25 mM sodium acetate, pH 5.5. A precipitate is formed which is removed by filtration after 30 minutes incubation at room temperature. A Millipore 1.2 μm Polypro XL filter (Millpore) is followed by a 0.45 μm Whatman Polycap 75 AS filter (Maidstone, Kent, UK). The filtered FGF18 is loaded onto a column of Amersham Biosciences SP Sepharose XL resin equilibrated to equilibration buffer (25 mM sodium acetate, 0.2 M NaCl, pH 5.5). The column is loaded at a capacity of 3-6 g FGF18 per L resin, the bed height is 15 cm, UV absorbance at 280 nm and 215 nm is monitored, and a flow rate of 150 cm/hr is used. Following loading the column is washed with equilibration buffer until the UV absorbance returns to baseline.  The column is then washed with 4 column volumes of<br><br>
25 % equilibration buffer, 75 % elution buffer (25 mM sodium acetate, 1.0 M NaCl, pH<br>
5.5).  FGF18 is eluted from the column with 50 % equilibration buffer, 50 % elution<br>
buffer.<br>
C.       Cation Exchange Chromatography using Streamline SP XL resin<br>
In another example, FGF18 is not concentrated by tangential flow filtration prior to capture by cation exchange chromatography. Following refolding, the pH is adjusted to 5.5 and the material is filtered through a 1.2 μm nominal cut off filter. An Amersham Biosciences Streamline column packed with Amersham Biosciences Streamline SP XL is equilibrated to equilibration buffer (25 mM sodium acetate, 0.2 M NaCl, pH 5.5). Following equilibration, the filtered, pH-adjusted, refolded FGF18 is loaded onto the column using in-line dilution, i.e. 30% filtered, pH-adjusted, refolded FGF18 and 70% water is loaded using the chromatography system to generate the correct ratio. The FGF18 is loaded onto the column in an upflow direction using a flow rate that causes a 2-fold expansion of the resin compared to the settled bed height. Once the filtered, pH-adjusted refolded FGF18 has been loaded it is replaced with equilibration buffer. Pumping onto the column is then continued with 30 % equilibration buffer and 70% water until the conductivity recorded at the column inlet is 
Example 13<br>
Intermediate Purification of FGF18 by hydrophobic interaction chromatography<br>
A.	Hydrophobic Interaction Chromatography (HIC) using butyl Sepharose resin<br>
FGF18 is adjusted to 1.5 M ammonium sulfate by adding 198 gr solid ammonium sulfate per liter FGF18 solution. The solution is stirred until the ammonium sulfate is dissolved and then solid material is removed by filtration through a 0.45 nm nominal cut-off filter. In one example a 15 cm high column of Amersham Biosciences butyl Sepharose 4 FF is equilibrated to equilibration buffer (25 mM sodium acetate, 50 mM sodium chloride, 1.5 M ammonium sulfate, pH 5.5). The adjusted, filtered FGF18 solution is loaded onto the column at a capacity of 1.0-2.5 g FGF18 per L resin at a flow rate of 150 cm/hr. UV absorbance at 280 nm and 215 nm is monitored. Following loading the column is washed with equilibration buffer until the UV absorbance returns<br><br>
to baseline. FGF18 is eluted from the column with 50% equilibration buffer and 50% elution buffer (25 mM sodium acetate, 50 mM sodium chloride, pH 5.5). Alternatively, following loading of FGF18 onto the column and washing with equilibfation buffer, FGF18 is eluted from the column with a 10 column volume linear gradient from 100% equilibration buffer to 100% elution buffer.<br>
B.       HIC using TOYOPEARL 650M resin<br>
In another example a different resin, Tosoh Biosep TOYOPEARL butyl 650M, is used to purify the FGF18. The method is the same as that used for the butyl Sepharose FF resin with the following exceptions: the cation exchange eluate is adjusted to 1.5 M CNH4)2S04 using a 3.5 M (NBU)2S04 stock solution; the adjusted, filtered FGF18 solution is loaded onto the column at a capacity of 10-12 g FGF18 per L resin; following loading, the column is washed with equilibration buffer until UV absorbance returns to baseline; FGF18 is eluted from the column with 100% elution buffer (25 mM sodium acetate, pH 5.5,0.05 M NaCl, 0.15 M (NH4)2S04).<br>
Example 14<br>
A.	Concentration and Buffer Exchange of purified FGF18 to phosphate buffered<br>
saline<br>
Following purification FGF18 is subject to ultrafiltration and diafiltration to concentrate it and exchange it to a buffer suitable for storage. A tangential flow filtration apparatus and membranes (Millipore Pellicon Biomax 5 kDa molecular weight cut-off plate and frame system) are sanitized using 0.5 M NaOH and rinsed with water. For purified FGF18 from 1 L of fermentation broth, 0.1 m2 or less of membrane area is used with a cross-flow rate of approximately 20-25 L/hr and a transmembrane pressure of 10 psi to 15 psL FGF18 is concentrated to approximately 15-20 mg/mL and then diafiltered against approximately 5-10 diavolumes of phosphate buffered saline, pH 6.0. The concentrated, buffer exchanged FGF18 is stored at -80°C.<br>
B.	Concentration and buffer exchange of purified FGF18 to histidine/mannitol<br>
buffer<br>
Following purification by SP HP Sepharose, FGF18 is subject to ultrafiltration and diafiltration to concentrate and exchange purified FGF18 into a buffer suitable for storage. A tangential flow filtration apparatus and membranes (Millipore Pellicon Biomax 5 kDa molecular weight cut-off plate and frame system) are sanitized using 0.5 M NaOH and rinsed with water. For purified FGF18, from 1 L of fermentation broth, 0.1 m or less of membrane area is used with a cross-flow rate of approximately<br><br>
30 L/hour at a transmembrane pressure of 25. FGF18 is concentrated to approximately 10-15 mg/ml, and then diafiltered against approximately 5-10 diavolumes of 10 mM histidine, 4.72% (w/v) mannitol, pH 5.0-5.3. The resulting solution is sterile filtered.<br>
Example 15<br>
Additional Purification of FGF18<br>
A.	Cation Exchange Chromatography using SP HP Sepharose resin for polishing<br>
Further purification using SP HP Sepharose is performed to further improve overall purity. The TOYOPEARL butyl 650M elutate is diluted to 30 mS/cm with water, and then adjusted to pH 6.0 using a dibasic sodium phosphate stock solution. The adjusted solution is then filtered using a 0.22 μm filter. The filtered material is loaded onto the column at 10-15 g FGF18 per L resin on a column equilibrated with 50 mM phosphate, pH 6.0, 0.3 M NaCl. UV 280 nm and UV 215 nm are used to monitor the chromatography. After loading, the column is washed with equilibration buffer until UV reaches baseline. FGF18 is eluted from the column using a 20-column volume gradient to 100% elution buffer (50 mM phosphate, pH 6.0, 0.7 M NaCl).<br>
B.	Anion Exchange Chromatography<br>
FGF18 is passed through an anion exchange column to remove endotoxin. A column of Amersham Biosciences Q Sepharose EF is equilibrated with equilibration buffer (20 mM Tris, pH 8.0). The FGF18 solution is adjusted to a conductivity of 
C.	Hydrophobic Interaction Chromatography<br>
In other examples, hydrophobic interaction chromatography, using conditions different than those described above with butyl resin, has been used to purify FGF18. Amersham Biosciences phenyl Sepharose FF high sub, Amersham Biosciences Phenyl Sepharose HP and Amersham Biosciences butyl Sepharose 4 FF can be used as resin in both binding and flow through modes. To bind FGF18, the columns are equilibrated to 25 mM sodium acetate, 50 mM sodium chloride, 1.5 M ammonium sulfate, pH 5.5. FGF18 is adjusted to 1.5 M ammonium sulfate by adding solid ammonium sulfate and stirring until it is dissolved.   The adjusted FGF18 solution is<br><br>
loaded onto the equilibrated column at a flow rate of 150 cm/hr. UV absorbance at 280 nm and 215 nm is monitored. Following washing, the FGF18 is eluted from the column with a 10 column volume linear gradient from 100 % equilibration buffer to 100 % elution buffer (25 mM sodium acetate, 50 raM NaCl, pH 5.5). In flow through mode the FGF18 containing solution is adjusted to 1.0 M or less ammonium sulfate, and loaded onto a column equilibrated with 25 mM sodium acetate, 50 mM NaCl, 1.0 M ammonium sulfate, pH 5.5. The flow through is collected.<br>
In other examples, hydrophobic interaction chromatography using sodium sulfate as salt, rather than ammonium sulfate, has been used to purify FGF18. Amersham Biosciences phenyl Sepharose FF high sub, Amershan Biosciences Phenyl Sepharose HP and Amersham Biosciences butyl Sepharose 4 FF can be used as resin. The columns are equilibrated to 25 mM sodium acetate, 50 mM sodium chloride, 1.5 M sodium sulfate, pH 5.5. FGF18 is adjusted to 1.5 M sodium sulfate by adding solid sodium sulfate and stirring until the sodium sulfate is dissolved. The adjusted FGF18 solution is loaded onto the equilibrated column at a flow rate of 150 cm/hr. UV absorbance at 280 nm and 215 nm is monitored. Following washing, the FGFlS'is eluted from the column with a 10 column volume linear gradient from 100% equilibration buffer to 100% elution buffer (25 mM sodium acetate, 50 mM NaCl, pH 5.5).<br>
In another example, HIC FPLC flow-through was performed on a BIOCAD 700E EPLC system (Perseptive Biosystems, Framingham, MA) equipped with Butyl Sepharose 4 FF column (Amersham Biosciences). The column was conditioned with 25 mM NaOAc, 600 mM NaCl, 1 M (NHLO2SO4. pH 5.5. Solid (NRO2SO4 was added to the cation-exchange eluate to a final concentration of 1M. The solution was loaded onto the column and FGF18 was collected in the flow-through.<br>
D.       IMAC using metal chelating Sepharose<br>
Amersham Biosciences Chelating Sepharose (Amersham) is used to further purify FGF18. Captured FGF18 CIE eluate is loaded onto a column charged with copper, zinc, or nickel ions then equilibrated with 25 mM sodium acetate, pH 5.5; 0.8 M NaCl UV 280 nm and UV 215 nm are used to monitor the chromatography. The column is then washed with equilibration buffer to baseline, and eluted using a 10 CV gradient to 100% elution buffer (25 mM sodium acetate, pH 5.5; 0.8 M NaCl, 0.5. M imidizole).<br>
Example 16<br>
A.       Reversed phase BDPLC analysis of solubilized FGF18 in acetonitrile buffer<br><br>
The method described here is used to quantify FGF18 in solubilized inclusion body samples and purified samples. A 4,6 x 50 mm Jupiter C5 column (300 A, 5 μm, Phenomenex) is used on an Agilent Technologies 1100 series HPLC system with thermostated autosampler and thermostatted column compartment. A 0.2 μm pre-column filter is placed before the column. Mobile phase A is 0.1% TFA in HPLC grade water and mobile phase B is 0.1% TFA in acetonitrile.<br><br>
The column is equilibrated to the initial conditions of the elution gradient/time table until a stable baseline is achieved.<br>
Method parameters are as follows:<br>
1.	Flow rate: 1 ml/min.<br>
2.	Total run time: 20 minutes<br>
3.	Column temperature: 40 °C<br>
4.	Autosampler temperature: 8 °C<br>
5.	Maximum column pressure; 240 bar<br>
6.	Injector draw speed: 100 μL/minute<br><br>
7.	Injector eject speed: 100 μL/minute<br>
8.	Diode array detector data collection wavelength: Signal A: 280 run, 25 nm bandwidth<br>
9.	Diode array detector data monitoring wavelength: Signal B: 215 nm, 10 nm bandwidth<br>
10.	Diode array detector data reference wavelength: Signal A: 350 nm, 25<br>
nm bandwidth; Signal B: 350 nm, 25 nm bandwidth<br>
11.	Diode Array Detector autobalance: Prerun/Postrun mode<br>
12.	Peak width response time: &gt; 0.1 min.<br>
13.	Slit width: 4 nm<br>
14.	Needle wash function: programmed to reduce the build-up of<br>
guanidine on the needle and needle seat.<br>
For quantitation of unfolded FGF18, FGF18 reference standard is diluted to 0.5 mg/mL with 50 mM Tris, pH 7.5, 6 M guanidine HC1,10 mM DTT and heated at 40°C for 20 minutes. Diluted reference standard is injected onto the column at least five levels between 10 \ig and 50 jxg (for example, 10, 20, 30 ,40 and 50 ]ig injections). Solubilized FGF18 samples are spun in a microfuge and diluted 1:10 in 50 mM Tris, pH 7.5,6 M guanidine HC1 prior to injection of 25 μl of sample.<br>
For quantitation of folded FGF18, FGF18 reference standard is diluted to 1:0 mg/ml with phosphate buffered saline, pH 6.0. Folded FGF18 samples are injected to the HPLC without any treatment. Following chromatography the area under the FGF18 peaks is integrated. A standard curve is constructed and the concentration of FGF18 in the samples is read off the standard curve.<br>
B.       Methanol-based RP-HPLC for quantitation of FGF18<br>
A fifteen-minute methanol-based RP-HPLC method may also be used to evaluate FGF18 preparations ranging from solubilized inclusion bodies through final product.<br>
Method Parameters for FGF18 Methanol-based RP-HPLC Analysis are as follows:<br>
Column: Zorbax 300SB-CN (4.6 x 50 mm), 3.5 micron Mobile Phase A: 0.154% TFA, HPLC grade Water Mobile Phase B: 0.154% TFA, Methanol<br><br><br>
Total Run-Time: 15 minutes<br>
Column Temperature: 40 °C<br>
Autosampler Temperature: 5°C<br>
Injector Draw Speed: 90fiL/minute<br>
Injector Eject Speed: 90 {-iiyminute<br>
DAD Monitoring Wavelength: Signal A: 280 nm, 8 nm bandwidth<br>
Signal B: 215 nm, 8 nm bandwidth<br>
Signal C: 280 nm, 6 nm bandwidth (Reference Wavelength OFF)<br>
DAD Data Collection Wavelength: Signal A: 280 nm, 8 nm bandwidth<br>
DAD Reference Wavelengths: Signals A and B, 360 nm, 16 nm bandwidth<br>
DAD Autobalance: Prerun/Postrun mode<br>
Peak Width Response Time: &gt; 0.1 min,<br>
Slit Width: 4nm<br>
Margin for Negative Absorbance: 100 mAu<br>
Standard Curve Load Amount Range: 1-20 (Xg<br>
Minimal Injection Volume: 5 μL<br>
Maximum Injection Volume: 100μL<br>
Pressure limit: 350 bar<br>
Normal Running Pressure: 130-200 bar<br>
Example 17<br>
OmpT deficient strain for expressing FGF18<br>
A.        Construction of a new host strain for production of heterologous genes<br><br>
The current process for production of FGF18 includes expression in the Kcoli host W3110 [F- mcrA mcrB XN(irnD- rmE)l k-]. While W3110 is a robust host, it is not ideal for downstream processing. Upon cell lysis, FGF18 is cleaved at amino acid 196 (Lys) (as shown in SEQ ID NO:6) by the OmpT protease present in the outer membrane. This protease is known to cleave other heterologous recombinant proteins, including FGF-18. Proteolysis of FGF18 does not occur in strains lacking OmpT, such as BL21 [F- ompT hsdSB (rB- mB-) gal dcm Ion]. While OmpT activity can be minimized during cell lysis with the addition of ZnS04 or Q1SO4, the purification scheme had to be designed to remove truncated FGF18 from the final product. In an effort to streamline the process for production of FGF18 and other heterologous genes, the OmpT protease was removed from W3110 to create a new production strain. The construction of this new E.coli host strain is described below.<br>
B.        Construction of plasmid pCHANl for expression of the Red recombinase operon<br>
A strategy based on homologous recombination was used to remove the OmpT<br>
protease from W3110. In order to delete genes efficiently from the E.coli chromosome<br>
by homologous recombinantion, certain enzymes with recombinase activity must he<br>
present within the cells. To accomplish this, a plasmid was. constructed harboring the<br>
Red recombinase operon from bacteriophage X.    A fragment containing the Red<br>
recombinase genes was synthesized from bacteriophage X DNA (New England Biolab)<br>
by PCR using recombination-specific primers ZC43,586 (SEQ ID NO:23) and ZC435587<br>
(SEQ ID NO:24)   The reaction contained 100 pmol each of primers ZC43,586 and<br>
ZC43,587, 10 pJ of 10X PCR buffer (Boehringer Mannheim), 1 ]id Pwo Polymerase<br>
(Boehringer Mannheim), 10 pi ®f 0.25 mM nucleotide triphosphate mix (Perkin Elmer), and dH20 in a final, volume of 100 |il. The PCR reaction consisted of a single 5 minute<br>
cycle at 94°C, followed by 30 cycles of 1 minute at 94°C, 1 minute at 50°C and 1 minute at 72°C. The last of the 30 cycles was followed by a 5-minute extension at 72°C and the reaction concluded with an overnight hold at 4°C. The resulting 1964 base pair (bp) fragment contained the Red recombinase operon (SEQ ID NO: 25). The nucleotide sequence as shown in SEQ ID NO:25 encodes for three genes, Gam(y) as shown from nucleotides 41-454, Bet{$) as shown from nucleotides 463-1245, and Exo as shown from nucleotides 1245-1922.<br>
The Red recombinase operon was incorporated into a plasmid by homologous recombination in yeast. Competent yeast cells (100 fil of S. cerevisiae SF838-9Dct) were combined with 100 ng of Smal-digested pTAP399 (deposited at American Type Culture Collection in Manassas, VA. (PA-4853), an expression vector for the production of IL-21), acceptor vector and 1 jag of the PCR fragment from above.<br><br>
The yeast/DNA mixture was transferred to a 0.2 cm electroporation cuvette and pulsed at 0.75 kV (5 kV/cm), infinite  25 /xF capacitor. The transformation mixture was then added to 1 ml of 1.2 M sorbitol and incubated at 30°C for 1 hour. The cells were plated in 500 fil aliquots onto two URA DS plates (2% dextrose, 2% sorbitol) and incubated at<br>
n<br>
30 C for 2 days. After about 48 hours the Ura+ yeast transformants from the plates were suspended in 2 ml H2O and pelleted by centrifugation. The cell pellet was resuspended<br>
in 1ml of Qiagen PI lysis buffer (Qiagen) and transferred to a fresh tube containing 1 ml<br>
of 0.5mm zirconia/silica beads (Biospec Products Inc.).  The cells were lysed, samples<br>
were allowed to settle, 250 |il of lysate were transferred to a fresh tube, and plasrnid<br>
DNA was isolated using the Qiagen Spin Miniprep kit according to the manufacturer's<br>
instructions.<br>
Electrocompetent Kcoli DH10B cells (Invitrogen) were transformed with<br>
1 p] of the yeast DNA prep. The cells were pulsed in 0.1 cm cuvettes at 2.0 kV, 25 ΜF<br>
and 100 £2. Following electroporation, 250 μl SOC (2% Bacto Tryptone (Difco, Detroit, MI), 0.5% yeast extract (Difco), 10 mM NaCl, 2.5 mM KC1, 10 mM MgCl2, 10 mM<br>
MgS04, 20 mM glucose) was added to each sample. Cells were allowed to recover at<br>
37°C for 2 hours. The entire 250 μl sample was plated in one aliquot on an LB plate (LB<br>
broth (Lennox), 1.8% Bacto Agar (Difco)) containing 25 mg/L kanamycin (Sigma).<br>
o<br>
Plates were incubated at 37 C overnight. Individual clones harboring the Red recombinase operon were identified by restriction digest to verify the presence of insert The inserts of positive clones were subjected to sequence analysis. A plasrnid containing the correct insert was designated pCHANl.<br>
The yeast sequence was then removed from the vector backbone of pCHANl. 3.0 μl of plasrnid DNA were incubated overnight with 24.3 \H H20, 2.7 pi buffer H (Roche) and 2.0 μl NotI (New England Biolabs) at 37°C. 5 pi of the overnight digest were mixed with 1 μl of 6x DNA sample dye (25% Ficoll Type 400 (Sigma), 0.25% Bromophenol blue (EM Science), 0.25% Xylene Cyanol (Kodak Biomedicals Inc.)), and 4 pi of this solution were run on a 1% agarose gel (EM Science) to verify complete digestion. To recircularize the plasrnid, 14 μl of the overnight NotI digest was mixed with 4 pi of 5x ligation buffer (Invitrogen) and 2 (μl ligase (Invitrogen). The ligation was incubated overnight at 25°C.<br>
The religated pCHANl was transformed into W3110. Electrocompetent W3110 cells (50 pi) were transformed with 1 μl pCHANl DNA using the electroportation protocol for Kcoli described above. After recovery, the entire 250 fil transformation mixture was plated in one aliquot on an LB plate containing 25 mg/L kanamycin. Plates were incubated at 37°C overnight and ten of the resulting clones were picked for further analysis. They were grown at 37°C overnight in 2.0 ml Superbroth H<br><br>
(Becton Dickinson) containing 25 μg/ml kanamycin. The following day, 1.0 ml of the overnight digest was used to confirm the presence of pCHANl. The Qiagen Spin Miniprep Kit was used to make plasmid DNA, following the manufacturer's instructions. The identity of the plasmid was confirmed by restriction digest using EcoRI (Gibco BRL) and NotI (New England Biolabs). Isolate #3 was selected for subsequent experimentation and named EE670.<br>
Generation of a tetracycline fragment for gene replacement in W3110<br>
The tetracycline gene was chosen as a suitable marker for homologous<br>
recombination into the OmpT locus, rendering the OmpT gene inactive. The tetracycline<br>
promoter::tetracycline (tetp::tet) fragment was generated by PCR from pBR322 DNA<br>
(New England Biolabs) using recombination-specific primers ZG45,112 (SEQ ID<br>
NO:26) and ZG45,171 (SEQ ID NO:27). The reaction mixture contained 100 pmol each<br>
of primers, ZG45.112 and ZG45.171,10 μl of 10X PCR buffer (Boehringer Mannheim),<br>
1 pi Pwo Polymerase (Boehringer Mannheim), 10 $ of 0.25 mM nucleotide triphosphate mix (Perkin Elmer), and dH20 in a final volume of 100 pi. The conditions for the PCR<br>
reaction were 1 cycle at 2 minutes at 94°C, followed by 30 cycles of 30 seconds at 94°C, 1 minute at 50°C and 2 minutes at 72°C. This was followed by a 7-minute extension at 72°C and an overnight hold at 4°C. The resulting 1590 bp fragment carries tetp::tet (SEQ IDNO:28).<br>
The PCR reaction was loadedonto a 1% agarose preparative gel to purify the tetp;;tet fragment. The tetp::tet fragment was cut out of the gel and placed in a 0.5 ml eppendorf tube with a small hole in the bottom that was lined with aquarium filter floss (Finny Products, Inc., Cincinnati, OH). The tube was inserted into a 1.5 ml eppendorf tube and spun in a tabletop centriftige at 14,000rpm for 10 minutes at 25°C. The liquid in the bottom of the 1.5 ml tube was mixed with 10% (vol/vol) 3M NaOAc and 2 volumes of 100% Ethanol. The sample was incubated at «20°C for 10 minutes and centrifuged for 10 minutes at 4°C in a tabletop centrifuge to precipitate the PCR fragment. The supernatant was aspirated and the pellet resuspended in 50 pi H2O. The tetp::tet fragment was at a working concentration of 50 ng/pl.<br>
The PCR fragment was ligated into the pCR4.0-BLUNT TOPO® vector (Invitrogen) to use as a positive control for the gene replacement experiments. The ligation was performed according to manufacturer's instructions. E.coli DH10B cells (Invitrogen) were transformed with 2 pi of the tetp::tet DNA fragment using the electroporation protocol for E.coli described above. Following recovery, the entire 250 pi transformation mixture was plated on an LB plate containing 100 mg/L Ampicillin (Sigma). Plates were incubated at 37°C overnight.<br><br>
Ten clones were picked for further analysis. They were grown overnight in 2.0 ml Superbroth II (Becton Dickinson) containing 100μg/ml ampicillin at 37°C. The following day, 1.0 ml of the overnight culture was used to confirm the presence of plasmid DNA. The Qiagen Spin Mmiprep Kit was used to make plasmid DNA, following the manufacturer's instructions. Plasmid DNA was subjected to restriction analysis using Sail (New England Biolabs) and PstI (New England Biolabs) to verify plasmid identity and insert orientation. Isolate #1 was picked for subsequent experimentation. The plasmid was named pSDH185 and the clone, EE686.<br>
Gene replacement in W3110: Deletion of the OmpT gene<br>
A 500 ml culture of W3110/pCHANl was grown at 37°C in SOB media [20 g/L tryptone, 5 g/L yeast extract, 0.5 g/L NaCl, 10 ml/L of 250 mM KC1, 5 ml/L of 2 M MgCb, pH7.0] to an ODeoo of 0.6. The culture was split into four 125 ml cultures. One culture was left as an uninduced control, while the other three were induced with 1 mM IPTG for 15 minutes, 30 minutes, or 60 minutes. At the end of their respective incubations, competent cells were made from all four cultures in the following manner. Cells were pelleted by centrifugation at 5000 rpm for 10 minutes. The supernatants were drained and each peUet was resuspended in 62.5 ml ice cold H2O. The cultures were pelleted again, the supernatant was drained, and each pellet was resuspended in 31.25ml cold 10% glycerol. The cultures were then centrifuged at 8000 rpm for 5 minutes. The pellets were drained well and resuspended in residual -10% glycerol.<br>
All four cultures were 'divided into six 50μl aliquots which were transformed in the following ways: 1) no DNA negative control, 2) 1 jxl (1 μgμl) pBR322 (New England Biolabs) positive control, 3) μl (1μg/μl) pTAP279 positive control, 4) 1 μl pSDH185 positive control, 5) 2 pi (50 ng/(il) tetp::tet fragment, and 6) 4 pi (50 ngμl) tetp::tet fragment. The cells were transformed by electroporation as described above for E.colt Entire transformation mixtures were plated on LB plates containing 10 mg/L tetracycline (Sigma) except for the pTAP279 controls, which were plated on LB plates containing 35 mg/L chloramphenicol (Sigma). Plates were incubated at 37°C overnight. In addition, 10"6 and 10"7 dilutions (in H20) of each four culture were plated on LB plates to evaluate overall efficiency of the recombination process by determining the cell number.<br>
The following day, control plates were taken out of the incubator and assessed. Samples transformed with the tetp::tet fragments were allowed to incubate for an additional 24 hours prior to assay. Twenty-six of the largest clones were identified for further analysis.<br><br>
Characterization of ompT deficient clones<br>
Each of the 26 selected clones was grown overnight at 37°C in 1 ml of LB with 5 fig/ml tetracycline. The following day, genomic DNA was generated from all 26 clones using the Genomic Prep DNA Isolation Kit (Amersham Pharmacia) according to the manufacturer's instructions.<br>
The genomic DNA from each clone was diluted 1:100 in dH20 to use as a<br>
template for PCR analysis. Each diluted sample was assayed using three different sets of<br>
PCR primers (three PCR reactions per clone). The reactions contained 100 pmol each of<br>
primer set #1: ZG45,357 (SEQ ID NO:29) and ZG45,350 (SEQ ID NO:30), or primer set<br>
#2: ZG45,353 (SEQ ID NO:31) and ZG45,355 (SEQ ID NO:32), or primer set #3:<br>
ZG45,354 (SEQ ID NO:33) and ZG45,359 (SEQ ID NO:34). The remainder of the 100<br>
pi final volume was made up of 10 pi of 10X PCR buffer (Boehringer Mannheim), 1 pi<br>
Pwo Polymerase (Boehringer Mannheim), 10 pi of 0.25 mM nucleotide triphosphate mix (Perkin Elmer) and dH20. The reaction conditions were: 1 cycle for 5 minutes at 94°C,<br>
followed by 30 cycles of 30 seconds at 94°C, 1 minute at 50°C and 2 minutes at 72°C. The PCR concluded with a 7-minute extension at 72°C and an overnight hold at 4°C. If the OmpT gene in W3110 was successfully replaced with the tetracycline gene, primer set #1 should amplify a 1584 bp band (SEQ ID NO:35), primer set #2 should amplify an 1190 bp band (SEQ ID NO;42). The results demonstrated that 25 of the 26 clones screened were ompT. W3110 ompT" clones #1 and #3 were selected for subsequent analysis.<br>
To confirm loss of proteolytic activity, H-21 was incubated with cell lysates from the newly derived ompT'strains and the W3110 parent. Lysate from the ompT strain, BL21, was included as a positive control. Cells were inoculated into Superbroth II and grown overnight at 37°C. Four 1 ml aliquots of each overnight culture were pelleted at room temperature and the cells were lysed using BugBuster® (Novagen) according to the manufacturer's instructions. Cell lysates were incubated at 25°C for 4 hours with either 1) 0.332 mg/ml of EL-21, or 2) 0.332 mg/ral of IL-21 in the presence of 5 mM ZnCl2- Each sample was mixed with an equal volume of NuPAGE 4x Sample Buffer (Invitrogen) containing 2% P-mercaptoethanol (Sigma). The reduced samples were heated for 5 min at 100°C and 10 fiL were loaded onto a 10% NuPAGE polyacrylamide gel (Invitrogen). Electrophoresis was conducted at 130v under denaturing conditions (SDS-PAGE) using lx MES running buffer (Invitrogen). Gels were stained with Simply Blue Safestain (Invitrogen) following the manufacturer's instructions.<br>
The results indicated that the OmpT protease was inactivated through gene replacement IL-21 was completely intact after a 4-hour incubation in lysates from<br><br>
BL21, W3110 ompT #1 and W3110 ompT" #3, but was completely degraded in a lysate from the W3110 parent. The activity of the OmpT protease was inhibited by zinc. In incubations containing 5 mM ZnCl2 the FGF18 remained intact, supporting that OmpT was responsible for the degradation. The newly constructed W3110 ompT strains were named zGOLDl (W3110 ompT #1; (deposited at American Type Culture Collection in Manassas, VA. (PA-4S53) )and zGOLD3 (W3110 ompT #3).<br>
Characterization of zGOLDl and zGOLD3<br>
zGOLDl and zGOLD3 were grown alongside the W3110 parent for assessment of growth. Cultures of all three strains were grown at 37°C in LB to an OD500 of 1.0. Cell density was measured hourly to assess growth. Dilutions (10-6,10"7 and 10" 8 in H2O) of each culture were plated on LB kanamycin plates (see above) to determine cell number. The results indicate that the growth of the zGOLD strains is equivalent to that of the W3110 parent strain.<br>
To assess transformation efficiency, cells were harvested and made competent for transformation as described above. Aliquots from each strain were transformed with either: 1) 1 |il pTAP337 (IL-21 expression plasmid; ATCC No. PA-4853), or 2) no DNA (negative control). Electroporation was carried out as described above. Following recovery, each transformation mixture was plated on an LB plate containing 25 mg/L kanamycin and incubated overnight at 37°C. The data indicate that transformation efficiency of W3110 was not affected by the removal of opmT.<br>
Ten clones of each zGOLD strain transformed with the FGF18 expression vector were selected to evaluate protein production. The clones were grown at 37°C overnight in Superbroth II (containing 25μg/ml kanamycin. The overnight cultures were used to inoculate roller drums containing Superbroth II with 25μgμl kanamycin. Cells were grown at 37°C. A second culture of one of the clones was grown and served as an uninduced control. When the ODeoo of each culture was L5-2.0, they were induced with ImM IPTG (ICN Biomedicals lac.)- Incubation of the cultures continued for another 5 hours. Samples of each culture were analyzed by SDS-PAGE on 4-12% gradient NuPAGE gel (Invitrogen) under reducing conditions as described above. The results indicate that FGF18 production by zGOLDl and zGOLD3 is equivalent to that of the W3110 parent strain. zGOLDl/pTAP337 (deposited at American Type Culture Collection in Manassas, VA. (PA-4853)) is an example of zGOLDl transformed with an expression vector.<br><br>
Example 18<br>
It was determined that a further modification of the zGOLDl strain was desired to increase the amount of heterologous protein by rendering the strain resistant to infection by particular bacteriophage. Using equivalent methods to those described above, the fhuA gene was identified in the zGOLDl strain and disrupted. This gene was originally identified by Fecker and Braun, J. Bacterio., 156:1301 (1983). fhuA" strains are well known in the art and convey resistance to infection by T-odd phages. Such strains are described, for example, in Boheivers et al. Biochem., 40:2606 (2001). The doubly disrupted strain (ompT, fhuA") was named zGOLD5.<br>
Generation of chloramphenicol actvltransferase gene fragment Oligos zc48544 (SEQ ID NO: 37) and zc48545 (SEQ ID NO: 38) were used to amplify the cat gene, coding for chloramphenicol acetyltransferase. zc48544, the sense oligo, consists of two parts. At the 5' end, the oligo is made up of 70 bases that flank the fhuA coding sequence. This sequence ends just 5' to the initial methionine codon. The second half of the oligo consists of 23 bases homologous to the promoter region of the cat gene.    zc48545, the antisense oligo, is designed much like the sense oligo. The 5' end of the oligo consists of 73 bases that flank the 3' end of the fhuA coding sequence, including the stop codon. The next 23 bases are homologous to the 3' end of the cat gene.<br>
To amplify the cat gene with the fhuA flanking sequence, the following final concentrations of reagents were used in a total reaction volume of 100μL: 0.2 pM of each oligo; 0.2 mM dNTPs; lx reaction buffer; 10% DMSO; and 0.05 UμL Pwo (Roche). The template used for amplification of the cat gene was pRARE (Novagen). The reaction consisted of 25 cycles of the following: 94°C for 30 seconds, 55°C for 30 seconds, and 72°C for 1 minute. This was done in duplicate. The resulting DNA fragment was analyzed by electrophoresis on a 1% IxTBE agarose gel. The size of the fragment was approximately 1 kb, as expected. The DNA fragment was cut from the gel and purified from the agarose using a QIAquick Gel Extraction Kit (Qiagen). The concentration of the recovered DNA was determined to be 110 ngμL by spectrophotometric analysis.<br>
Gene replacement: deletion of the fhuA gene in 7.GOT .PI<br>
Electrocompetent ZGOLDl cells containing pCHANl were prepared. One hundred microliters of the overnight culture were used to inoculate 50 mL of LB media, containing 25 fig/mL kanamycin, in a shake flask. The cultures were grown at 37°C with agitation until the ODeoo was approximately 0.6. At this point the cultures were induced using 1 mM IPTG and were allowed to grow for 15 minutes. The cultures<br><br>
were centrifuged for 10 minutes at 5000xg. The supernatant was discarded, and the pellet was resuspended in a half volume of ice-cold water. The cells were centrifuged again, the supernatant was discarded, and the resulting pellet was resuspended in a quarter volume of ice-cold 10% glycerol. The cells were centrifuged one last time, the supernatant was discarded, and the pellet was resuspended in the residual glycerol.<br>
Two hundred nanograms of the purified PCR product were transformed into the electrocompetent cells in 0.1cm cuvettes pulsed at 2.0 kV, 25 μF and 400 £2. Following electroporation, the cells were allowed to recover in 0.6 roL of SOC [2% Bacto Tryptone pifco, Detroit, MI), 0.5% yeast extract (Difco), 10 mM NaCl, 2.5 mM KC1, 10 mM MgC12, 10 mM MgS04, 20 mM glucose] for one hour at 37°C. The cells were plated as 10 pL and 100 \iL aliquots onto LB Cm plates [LB broth (Lennox), 1.8% Bacto Agar (Difco), 10μg/mL chloramphenicol (Sigma)]. In addition, the remaining cells were plated  The plated cells were incubated at 37°C overnight.<br>
Thus, this approach was used to inactivate the fhuA gene in ZGOLDl. The cat gene with the fhuA flanking sequence was generated by PCR. After gel purification, 200 ng of the purified PCR product were transformed into electrocompetent ZGOLDl cells containing pCHANl. The cells were plated on LB agar containing chloramphenicol at 10 μg/mL. The plated cells were incubated at 37°C overnight. Six colonies were picked for further analysis. A PCR screen for the presence of the cat gene in the transformed bacteria was performed using oligos zc46,701 (SEQ ID NO: 39) and zc46,702 (SEQ ID NO: 40). These two primers anneal to the genomic DNA flanking the fhuA coding sequence. Therefore, either fhuA or cat would be amplified. If fhuA were present, the resulting PCR product would be about 2.3kb, but if cat were present, the resulting PCR product would be about l.lkb. A ZGOLDl culture was included as a control. All the screened clones produced a band of approximately l.lkb, the size of the cat gene. The results indicate that fhuA is absent from the ZGOLDl genomic background.<br>
The clones lacking the fhuA gene were challenged with T-odd bacteriophage. W3110 is a phage sensitive strain and was chosen as a negative control. 27C7 (ATCC) is resistant to T-odd bacteriophage and was selected as a positive control. T5 bacteriophage, obtained from ATCC, were resuspended in LB and diluted 1:10, 1:100, 1:1000, and 1:10,000. Five microliters of phage and each phage dilution were spotted on solidified top agar containing bacteria. The plates were allowed to dry and were incubated at 37°C for 24 hours. The T5 bacteriophage formed plaques on the plates inoculated with W3110. In contrast, no plaques were formed on the plates inoculated with 27C7 and the six fhuA-deleted clones. The results indicate that all six fhuA-deleted clones are resistant to T-odd bacteriophage.<br><br>
The six fhuA-deleted clones were cultured overnight at 37°C in the absence of selection. Each clone was streaked onto LB agar containing chloramphenicol at 10μg/mL. Each clone was re-streaked onto LB agar with and without kanamycin at 25 Hg/mL. The E. coli cells were cultured overnight at 37°C. All clones grew on LB agar and no clone grew on LB agar containing kanamycin. The results demonstrate that all of the clones have lost the pCHANl plasmid. One isolate was chosen and saved as a glycerol stock. This isolate was designated ZGOLD5.<br>
From the foregoing, it will be appreciated that, although specific embodiments of the invention have been described herein for purposes of illustration, various modifications may be made without deviating from the spirit and scope of the invention. Accordingly, the invention is not limited except as by the appended claims.<br><br><br><br><br><br><br><br><br><br>
CLAIMS We claim:<br>
1.	An expression vector for producing FGF18 protein comprising<br>
the following operably linked elements!<br>
(a)	a prokaryotic origin of replication;<br>
(b)	a transcriptional initiation DNA element;<br>
(c)	a polynucleotide sequence as shown in SEQ ID NO:l; and<br>
(d)	a transcriptional terminator.<br><br>
2.	The expression vector of claim 1 which further comprises a selectable marker.<br>
3.	The expression vector of claim 1 or 2 wherein said selectable market is selected from the group consisting of tetracycline resistance, amplicillin resistance, kanamycin resistance, neomycin resistance, chlormaphenicpl resistance, and the hok/sok system.<br>
4,' The expression vector of claim 1 or 2 or 3 wherein the FGF18 protein is produced under the control of the Tac promoter, the vector comprises at least one of the Lacl-q gene, the ROP gene, and kanamycin resistence, and the origin of replication of the vector is-ColEl.<br>
5.	An expression vector comprising the pSDH170 vector.<br>
6.	A prokaryotic host cell transformed with the expression vector according to claims 1-5.<br>
7.	The prokaryotic host cell of claim 6 wherein the host cell is an £. coli OmpT protease deficient strain.<br>
8.	The host cell of claim 6, wherein the host cell is E. coli strain is selected from the group consisting of W3110, MM294, TG-1, JM-107, UT5600, and BL21.<br>
9.	A method for producing FGF18 proteins comprising:<br><br>
(a)	culturing a host cell according to claims 6, 7 or 8 in growth medium under conditions wherein FGF18 is expressed;<br>
(b)	recovering the host cells from the growth medium; and<br>
(c)	isolating the FGF18 protein from the host cells.<br>
10.	A method for producing FGF18 proteins comprising:<br>
(a)	culturing a host cell according to claim 6, 7 or 8 in growth medium by fed batch fermentation;<br>
(b)	recovering the host cells from the growth medium; and<br>
(c)	isolating the FGF18 protein from the host cells.<br>
11.	A method for producing an FGF18 protein comprising:<br>
(a)	culturing a host cell according to claim 6, 7, or 8 in a shake flask to an OD6oo of 5 to 20 in a growth medium;<br>
(b)	inoculating a fermentation vessel with 1 to 12% v/v of shake flask medium containing host cells;<br>
(c)	culturing the host cells in a growth medium at a pH of 6.2 to 7.2, wherein a feed solution is fed into the fermentation vessel before 15 hours elapsed fermentation time (EFT);<br>
(d)	adding an inducing agent to the fermentation vessel at 20 to 30 hours EFT; and<br>
(e)	harvesting the host cells at 48 to 56 hours EFT.<br><br>
12.	The method of claim 11, wherein the inducing agent is isopropyl B-D thiogalactopyranoside (IPTG) at 0.5 to 2 mM.<br>
13.	The method of claim 11, wherein the feed solution comprises a carbohydrate selected from the group consisting of glycerol and glucose at a concentration of growth medium, and a feed rate of 5-15 grams of carbohydrate per hour.<br>
14.	The method of claim 13, wherein the glycerol is 40 to 70% v/v glycerol or the glucose is 40 to 70% w/v glucose.<br>
15.	The method of claim 13, wherein the glycerol is about 70% v/v or the glucose is about 60% w/v.<br><br>
16.	A method of producing FGF18 protein comprising:<br>
(a)	seeding a flask with an inoculum comprising an E. coli W3110 host cell expressing an FGF18 polypeptide as shown in SEQ ID NO:4, or an E. coli W3110 host cell comprising pTAP337 vector wherein an FGF18 polypeptide is expressed, and with growth medium comprising about 5 g/L glycerol;<br>
(b)	culturing the inoculum in growth medium for 16-20 hours at about 30°C;<br>
(c)	transferring the cultured inoculum in growth medium to a batch fermentor at a concentration of 0.5-5% v/v inoculum;<br>
(d)	fermenting the batch fermentation at about 37° and about pH 6.8; with about 2% glycerol;<br>
(e)	introducing a glucose feed at about 8 hours elapsed fermentation time (EFT) of about 9.5 g glucose/liter/hour and continuing until end of a fermentation run;<br>
(f)	adding IPTG at about 24 hour EFT to final concentration of 0.5 to2mM;<br>
(g)	fermenting about 28 hours after addition of IPTG;<br>
(h)       harvesting fermentation broth from the fermenter;<br>
(i)       adding an equal volume of water to the fermentation broth; and (])       homogenizing  and  centrifuging the fermentation broth  to collect a cell pellet or cell slurry comprising FGF18 protein material.<br>
17.	A method for isolating insoluble FGF18 protein comprising a<br>
sequence of amino acid residues as shown in SEQ ID NO:4 comprising the steps of:<br>
(a)	separating water insoluble FGF18 protein material from a cell pellet or cell slurry;<br>
(b)	dissolving the insoluble FGF18 protein material in a chaotropic solvent;<br>
(c)	diluting the chaotropic solvent and refolding the FGF18 protein; and<br>
(d)	isolating the FGF18 protein, wherein the isolated FGF18 protein is capable of being biologically active.<br>
18.	The method of claim 17 wherein the isolated FGF18 protein is<br>
at least 90% pure.<br><br>
19.	The method of claim 17 wherein the isolated FGF18 protein is at least 90% pure and has an endotoxin level of less than 10 endotoxin units per mg FGF18 protein.<br>
20.	A method for isolating insoluble FGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:4 comprising the steps of:<br><br>
(a)	separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble FGF18 protein material;<br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble FGF18 protein material in a chaotropic solvent comprising a guanidine salt;<br>
(d)	diluting the chaotropic solvent by addition of a refolding buffer comprising arginine salts and a mixture of reducing and raiding components; ;<br>
(e)	isolating the FGF18 protein by removing unfolded and aggregated proteins by filtering; and<br>
(f)	purifying the FGF18 refolded protein on a cation exchange column; wherein the isolated and purified FGF18 protein is capable of being biologically active.<br>
21.	A method for isolating insoluble FGF18 protein comprising a<br>
sequence of amino acid residues as shown in SEQ ID NO:4 comprising the steps of:<br>
(a)	separating from a fermentation broth a cell pellet or cell slurry. comprising water insoluble FGF18 protein material;<br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble FGF18 protein material in a chaotropic solvent comprising a guanidine salt; and<br>
(d)	diluting the chaotropic solvent by addition of a refolding buffer comprising arginine salts and a mixture of reducing and oxidizing components;<br>
(e)	isolating the FGF18 protein by removing unfolded and aggregated proteins by filtering;<br>
(f)	purifying the FGF18 refolded protein on a cation exchange column; and<br><br>
(g) purifying the FGF18 eluate from step (f) on a hydrophobic interaction column, wherein the isolated and purified FGF18 protein is capable of being biologically active.<br>
22.	The method of claim 20 or 21 wherein the guanidine salt is selected from the group consisting of guanidine hydrochloride and guanidine thiocyanate.<br>
23.	A method for isolating insoluble FGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:4 comprising the steps of:<br><br>
(a)	separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble FGF18 protein material;<br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble FGF18 protein in a chaotropic solvent comprising about 6M guanidine hydrochloride, 40 mM dithiotfareitol (DTT) for about one hour at room temperature;<br>
(d)	refolding the dissolved inclusion bodies in a solution by diluting into refolding buffer comprising 50 mM Tris and 120 mM NaCl and adding 0.5MCuSO4;<br>
(e)	filtering the solution;<br>
(h) loading solution on resin column equilibrated to pH 8.0 using sodium acetate buffer,<br>
(f)	washing the resin column with about 0.4 M sodium chloride;<br>
(g)	washing the resin column with about 0.75 M sodium chloride to elute bound FGF18 protein;<br>
(h) adding ammonium sulfate to a concentration of about 1.5 M to eluate and filtering eluate solution;<br>
(i) loading eluate onto a Tosohaas butyl 650-M column equilibrated to 1.5 M ammonium sulfate, 0.05 M sodium chloride in sodium acetate buffer;<br>
(j) washing column with about 0.15 M ammonium sulfate, 0.05 sodium chloride in sodium acetate buffer;<br>
(k) diluting the eluate to a conductivity of about 30 mS/cm with water;<br><br>
(1) loading eluate onto a SP Sepharose HP column equilibrated with sodium acetate buffer;<br>
(m) washing column with 20-column volume linear gradient from 0.3 to 0.7 M sodium chloride;<br>
(n)      concentrating the FGF18 protein; and<br>
(o) exchanging buffer to formulation buffer using tangential flow ultrafiltration.<br>
24.	The method according to claims 11-23, wherein biological activity is measured using a FGF18 cell proliferation assay.<br>
25.	A host cell from a strain of zGOLDl transformed with an expression vector comprising a pSDH170 vector.<br>
26.	An expression vector for producing trFGF18 protein comprising the following operably linked elements:<br><br>
(a)	a prokaryotic origin of replication;<br>
(b)	a transcriptional initiation DNA element;<br>
(c)	a polynucleotide sequence as shown in SEQ ID NO:2; and<br>
(e)	a transcriptional terminator.<br>
27.	The expression vector of claim 26 which further comprises a selectable marker.<br>
28.	The expression vector of claim 26 or 27 wherein said selectable market is selected from the group consisting of tetracycline resistance, amplicillin resistance, kanamycin resistance, neomycin resistance, chlormaphenicol resistance, and the hok/sok system.<br>
29.	The expression vector of claim 26, 27 or 28 wherein the trFGF18 protein is produced under the control of the Tac promoter, the vector comprises at least one of the Lacl-q gene, the ROP gene, and kanamycin resistence, and the origin of replication of the vector is ColEl.<br>
30.	An expression vector comprising the pSDH174 vector.<br><br>
31.	A prokaryotic host cell transformed with the expression vector according to claims 26-30.<br>
32.	The prokaryotic host cell of claim 31 wherein the host cell is an E. coli OmpT protease deficient strain.<br>
33.	The host cell of claim 31 or 32, wherein the host cell is E. coli strain is selected from the group consisting of W3110, MM294, TG-1, JM-107, UT5600,andBL-2L<br>
34.	A method for producing trFGFl8 proteins comprising:<br><br>
(a)	culturing a* host cell according to claim 31, 32, or 33 in growth medium under conditions wherein trFGFl 8 is expressed;<br>
(b)	recovering the host cells from the growth medium; and<br>
(c)	isolating the trFGFl 8 protein from the host cells.<br>
35.	A method for producing trFGF18 proteins comprising:<br>
(a)	culturing a host cell according to claim 31, 32 or 33 in growth medium l&gt;y fed batch fermentation;<br>
(b)	recovering the host cells from the growth medium; and<br>
(c)	isolating the trFGFl8 protein from the host cells.<br>
36.	A method for producing trFGFl 8 protein comprising:<br>
(a)	culturing a host.cell according to claim claim 31, 32, or 33 in a shake flask to an ODeoo of 5 to 20 in a growth medium;<br>
(b)	inoculating a fermentation vessel with 1 to 12% v/v of shake flask medium containing host cells;<br>
(c)	culturing the host cells in a growth medium at a pH of 6.2 to 7.2, wherein a feed solution is fed into the fennentation vessel before 15 hours elapsed fennentation time (EFT);<br>
(d)	adding an inducing agent to the fermentation vessel at 20 to 30 hours EFT; and<br>
(e)	harvesting the host cells at 48 to 56 hours EFT.<br>
37.	The method of claim 36, wherein the inducing agent is<br>
isopropyl 8-D thiogalactopyranoside (IPTG) at 0.5 to 2 mM.<br><br>
38.	The method of claim 36, wherein the feed solution comprises a carbohydrate selected from the group consisting of glycerol and glucose at a concentration of growth medium, and a feed rate of 5-15 grams of carbohydrate per hour.<br>
39.	The method of claim 38, wherein the glycerol is 40 to 70% v/v glycerol or the glucose is 40 to 70% w/v glucose.<br>
40.	The method of claim 38, wherein the glycerol is about 70% v/v or the glucose is about 60% w/v.<br>
41.	A method of producing trF(3Fl 8 protein comprising:<br><br>
(a)	seeding a flask with an inoculum comprising an E. coli W3110 host cell expressing an trFGF18 polypeptide as shown in SEQ ID NO:6, or an £ coli W3110 host cell comprising pSDH174 vector wherein txFGF18 polypeptide is expressed, and with growth medium comprising about 5 g/L glycerol;<br>
(b)	culturing the inoculum in growth medium for 16-20 hours at about 30°C;<br>
(c)	transferring the cultured inoculum in growth medium to :a batch fennentor at a concentration of 0.5-5% v/v inoculum;<br>
(d)	fermenting the batch fermentation at about 37° and about pH 6.8; with about 2% glycerol;<br>
(e)	introducing a gjucose feed at about 8 hours elapsed fermentation time (EFT) of about 9.5 g glucose/litei/hour and continuing until end of a fermentation run;<br>
(f)	adding IPTG at about 24 hour EFT to final concentration of 0.5 to 2 mM;<br>
(g)	fermenting about 28 hours after addition of IPTG;<br>
(h)      harvesting fermentation broth from the feimenter;<br>
(i)       adding an equal volume of water to the fermentation broth; and (j)       homogenizing  and centrifiiging the fermentation broth  to collect a cell pellet or cell slurry comprising FGF18 protein material.<br>
42.	A method for isolating insoluble trFGF18 protein comprising a<br>
sequence of amino acid residues as shown in SEQ ED NO:6 comprising the steps of:<br><br>
(a)	separating water insoluble trFGFl 8 protein material from a cell pellet or cell slurry;<br>
(b)	dissolving the insoluble trFGF18 protein material in a chaotropic solvent;<br>
(c)	diluting the chaotropic solvent and refolding the trFGF18 protein; and<br>
(d)	isolating the trFGF18 protein, wherein the isolated trFGFl 8 protein is capable of being biologically active.<br><br>
43.	The method of claim 42 wherein the isolated FGF18 protein is at least 90% pure.<br>
44.	The method of claim 42 wherein the isolated trFGFl 8 protein is at least 90% pure and has an endotoxin level of less than 10 endotoxin units per mg HFGF18 protein.<br>
45.	A method for isolating insoluble trFGF18 proton comprising a sequence of amino acid residues as shown in SEQ ID NO:6 comprising the steps of:<br><br>
(a)	separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble trFGFl 8 protein material;<br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble trFGF18 protein material in a chaotropic solvent comprising a guanidine salt;<br>
(d)	diluting the chaotropic solvent by addition of a refolding buffer comprising argraine salts and a mixture of reducing and oxiding components;<br>
(e)	isolating the trFGFl 8 protein by removing unfolded and aggregated proteins by filtering; and<br>
(f)	purifying the trFGFl 8 refolded protein on a cation exchange column; wherein the isolated and purified ttFGF18 protein is capable of being biologically active.<br>
46.	A method for isolating insoluble trFGFl 8 protein comprising a<br>
sequence of amino acid residues as shown in SEQ ID NO:6 comprising the steps of:<br>
(a) separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble trFGFl8 protein material;<br><br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble trFGF18 protein material in a chaotropic solvent comprising a guanidine salt; and<br>
(d)	diluting the chaotropic solvent by addition of a refolding buffer comprising arginine salts and a mixture of reducing and oxidizing components;<br>
(e)	isolating the trFGF18 protein by removing unfolded and aggregated proteins by filtering;<br>
(f)	purifying the trFGF18 refolded protein on a cation exchange column; and<br>
(g)	purifying the trFGF18 eluate from step (f) on a hydrophobic interaction column, wherein the isolated and purified trFGF18 protein is capable of being biologically active.<br>
47. A method for isolating insoluble trFGF18 protein comprising a sequence of amino acid residues as shown in SEQ ID NO:6 comprising the steps of:<br>
(a)	separating from a fermentation broth a cell pellet or cell slurry comprising water insoluble FGF18 protein material;<br>
(b)	homogenizing the cell pellet or cell slurry to collect inclusion bodies;<br>
(c)	dissolving the insoluble txFGF18 protein in a chaotropic solvent comprising about 6M guanidine hydrochloride, 40 mM dithiothreitol (DTT) for about one hour at room temperature;<br>
(d)	refolding the dissolved inclusion bodies in a solution by diluting into refolding buffer comprising about 50 mM Tris pH8 and 120 mM NaCl and adding 0.5 mM CuS04,<br>
(e)	filtering the solution;<br>
(f)	loading solution on resin column equilibrated to pH 8.0 using sodium acetate buffer,<br>
(g)	washing the resin column with about 0.4 M sodium chloride;<br>
(h)      washing the resin column with about 0.75 M sodium chloride to<br>
elute bound trFGFl 8 protein;<br>
(i) adding ammonium sulfate to a concentration of about 1.5 M to eluate and filtering eluate solution;<br><br>
(j) loading eluate onto a Tosohaas butyl 650-M column equilibrated to L5 M ammonium sulfate, 0.05 M sodium chloride in sodium acetate buffer;<br>
(k) washing column with about 0.15 M ammonium sulfate, 0.05 sodium chloride in sodium acetate buffer;<br>
(1)       diluting the eluate;<br>
(m) loading eluate onto a SP Sepharose HP column equilibrated with sodium acetate buffer;<br>
(n) washing column with 20-column volume linear gradient from 0.3 to 0.7 M sodium chloride;<br>
(o)      concentrating the FGF18 protein; and<br>
(p) exchanging buffer to formulation buffer using tangential flow ultrafiltration.<br>
48.	The method of claim 46 or 47 wherein the guanidine salt is<br>
selected from the group consisting of guanidine hydrochloride and guanidine<br>
thiocyanate.<br>
49.	The method according to claims 34-48, wherein biological<br>
activity is measured using a FGF18 cell proliferation assay.<br>
50.	A host cell from a strain of zGOLDl transformed with an<br>
expression vector comprising a pSDH174 vector.<br><br><br>
FGF18 PRODUCTION IN PROKARYOT1C HOSTS<br>
The expression vectors and methods using an E. coli expression system for the large scale production of FGF18 are described. The vectors utilize the FGF18 coding sequence with specific changes in nucleotides in order to optimize codons and mRNA secondary structure for translation in E. coli. Using the expression vectors, the FGF18 gene was produced in £ coli to a level of greater than 1 g/L in fed batch fermentation. Also included are OmpT deficient E. coli strains, as well as OmpT and fhuA negative strains transformed with an FGF18 expression vector.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgQU1FTkRFRCAgUEFHRVMgT0YgU1BFQ0lGSUNBVElPTiAgMjItMDItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   AMENDED  PAGES OF SPECIFICATION  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgQU1FTkRFRCBDTEFJTVMgIDIyLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   AMENDED CLAIMS  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTICAgMDYtMDctMjAxMS5wZGY=" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   CORRESPONDENCE OTHERS   06-07-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgRk9STS0zICAyMi0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   FORM-3  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyMi0wMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   OTHER PATENT DOCUMENT  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICAgUE9XRVIgT0YgQVRUT1JORVkgIDIyLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007   POWER OF ATTORNEY  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1DSEVOUC0yMDA3ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDIyLTAyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">3049-CHENP-2007  EXAMINATION REPORT REPLY RECEIVED  22-02-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWFzc2lnbmVtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-assignement.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzA0OS1jaGVucC0yMDA3LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3049-chenp-2007-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252326-composition-for-controlling-animals-pests.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252328-a-fuel-detergent-additive-package.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252327</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3049/CHENP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>19/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-May-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-May-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>09-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ZYMOGENETICS ,INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1201 EASTLAKE AVENUE EAST , SEATTLE ,WASHINGTON 98102</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>REARDON, BRIAN</td>
											<td>3311 16TH AVENUE SOUTH , SEATTLE ,WASHINGTON 98144</td>
										</tr>
										<tr>
											<td>2</td>
											<td>JULIEN , SUSAN , D.,</td>
											<td>5022 37TH AVENUE NE, SEATTLE ,WA 98105 ,US</td>
										</tr>
										<tr>
											<td>3</td>
											<td>CHAN, CHUNG-LEUNG</td>
											<td>27316 SE 10TH COURT, SAMMAMISH ,WA 98075, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LIU, HONG, Y.,</td>
											<td>21752 34TH PLACE WEST ,BRIER,WASHINTON 98036, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/50</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US05/45166</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/635,023</td>
									<td>2004-12-10</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252327-fgf18-production-in-prokaryotic-hosts by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:25:49 GMT -->
</html>
