<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252653-analysis-chip-with-reference-scale-kits-and-analytical-methods by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:05 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252653:&quot;ANALYSIS CHIP WITH REFERENCE SCALE KITS AND ANALYTICAL METHODS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;ANALYSIS CHIP WITH REFERENCE SCALE KITS AND ANALYTICAL METHODS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to analysis chip of at least one analyte, said chip comprising at least one analysis spot for recognition and immobilization specific to the analyte; and a reference range (C) comprising several reference spots each arranged on said chip in a defined manner and independently of one another, each reference spot of this range comprising, immobilized on its surface and in a defined proportion P that is different and known for each spot relative to the other reference spots of said range: a probe reference molecule (PRM) permitting recognition and hybridization specifically to a defined target reference molecule (TRM) , and/or an inert oligonucleotide molecule (IM) incapable of recognition and hybridization with said PRM, these molecules both being unable to immobilize said analyte or analytes. The method of the invention comprises the use of said chip with its reference range for analysis of said, at least one, analyte.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>ANALYSIS CHIP WITH REFERENCE RANGE, KITS AND METHODS OF ANALYSIS<br>
DESCRIPTION<br>
TECHNICAL FIELD<br>
Tae present invention relates to an analysis chip comprising a reference range, to a method of analysis of a sample using said chip, and to a diagnostic or analytical kit containing said chip.<br>
The present invention is suitable for the qualitative and especially quantitative analysis of any analyre present in a sample.<br>
The present invention is therefore positioned notably in the area of chemical or biological chips, for example DNA chips, protein chips, antibody chips, antigen chips, cell chips, receptor chips, or chips for other iigands known to a person skilled in the art and capable cf specifically capturing and immobilizing one or more analyt.es.<br>
Usually,  applications  using  DNA  chips  are divided into two broad categories: gene expression (i) and genetyping (ii).<br>
(i) Gene expression comprises using a DNA chip or chips for studying the variation of all or part of the transcriptonie of a cell according to certain biological or Dhysio .Logical parameters: evolution of a cell curing the development of an organism, cancerization of a tissue, etc. It is becoming more and more evident that such expression profiles can also serve as diagnostic multifactorial  diseases,   notably  in<br><br>
(i-) In applications of the genotypirig type, the response required from the DNA chip is generally of the qualitative, yes/no type. In fact, we wish to know whether one or more particular sequence (s) is (are) present in the sample. When there is a great need for specificity, for example in the case of the detection of mutations, several oligonucleotides which only differ by one or two bases, are placed on the chip, and generally, a conclusion is made about the presence of the sequence corresponding to the strongest signal, e.g. of .fluorescence.<br>
The present invention is suitable for these two broad categories of applications and advantageously solves the many problems of the prior art that are describee below, in particular in the carrying out of such studies and profiles.<br>
Owing to its reference range, the chip of the present invention makes it possible to convert detect.ion and/or analysis signals from a chip, for example from a biological chip, to an absolute, reproducible, stable unit that is comparable to other measurement results obtained from the same conversion.<br>
PRIOR ART<br>
Fcr the quantification of variations in gene expression, the biologist nearly always uses two samples wnich are hybridized simultaneously on a DNA chip: a .reference sample, containing a certain mass of RNA or cDNA labelled with a first f luorochrome, for example t_uorescein (read in the green), and a sample of interest for which measurement has to be carried<br>
out, containing the same mass of RNA or cDNA labelled with a second f luorochrome, for example Cy3 or Cy5 (read in the orange or the red) . After hybridization, the DNA chip is read at the two wavelengths, and it is the ratio between the two intensities of the signals emitted which serves as the signal.<br>
1 t is therefore a question of a relative measurement with internal reference in each experiment. This method works well, but is quite laborious in use, and there are certain pitfalls, such as bias in labelling according to the sequences, or a deviation in reading tnat varies depending on the wavelength.<br>
Another technique of quantification is used for the gene expression chips developed by the company Affymetrix. In this method, each sequence placed on the chip is represented by a set of oligonucleotides, and each o1igonucleotide is in the form of two oligonucleotides, the one perfectly complementary to the required sequence, the other having a point mutat j.on ("mismatch"). The sample of interest, is then hybridized on the chip, a single colour being used. The complementary-mismatch difference is then used as primary signal, these signals being averaged over the set of oligonucleotides representing a gene. Moreover, a "reference gene" is arranged on the chip by the same technique; this reference gene, taken from the so-calied hoasekeeping genes of the organism, is assumed to maintain constant expression regardless of the physiological conditions of the cell. Then the "primary signal" of each gene is divided by that of the reference gene, for example to avoid the variations in<br>
brightness of the fluorescence due to the surroundings. It is then possible for different experiments to be compared with each other.<br>
This technique has the advantage of being monochrome, and therefore does not have the aforementioned drawbacks with the two markers. However, it necessarily uses short oligonucleotides (20 to 25 bases), and therefore there are other pitfalls, since some genes occur in various forms (alternative splicing;. This method therefore reguires more spots on the DNA chip and more knowledge about the various forms of the RNAs of the genes represented on the chip than with the other methods of the prior art. Moreover, the results vary depending on the choice of reference gene, even when the latter is selected very carefully. Therefore the results cannot always be compared.<br>
Tne aforementioned methods, provided certain controls and some repetitions of the experiments are employed, car; be relatively reliable and therefore offer a research tool that can be used by biologists. However, it must be noted that the measurements are relative, since the signal is a dimensionless number resulting from the ratio between two measurements with units that are generally arbitrary (arbitrary units of fluorescence). Therefore it is not possible to compare exDerimeri: s performed with different reference samples for example, without additional testing. Moreover, as the measurements are in arbitrary units, it is also ratner di fficult to know what causes the problems in the c<.ise of low intensities: it is difficult to know wher.ner an instrumental problem a></.ise>
connected with the chip itself, with the sample, with trie ope rating procedure, or with the result of the experiment.<br>
Jn applications of the genotyping type, the response required from the DNA chip is generally of the qualitative, yes/no type, as explained above. Furthermore, with the chips of the prior art, the results are relative, since the intensities of different oligonucleot: ides under investigation on one and the same chip are being compared.<br>
however, for certain applications, such as the detection of mixtures of several sequences, it would be advantageous to be able to compare the signals between several experiments, which is impossible at present with the chips currently available; it is constantly necessary to repeat one or more experiments to obtain relative measurements from which conclusions can be drawn, which ..s time-consuming and expensive.<br>
Moreover, in routine diagnosis, the problem of quantification of the results can be solved with a calibration curve. A reference sample is diluted in a predetermined manner at various concentrations in a suitable diluent, and the biological test is performed for each dilution away from the chip. This provides a reference' curve. When a real biological test is carried out, the result in arbitrary unit is plotted on the reference curve, and the concentration of analyte in the sample is deduced from this. However, it is import arr: for the test to be performed with exactly the same elements as those used for constructing the reference curve. Moreover, in tests with enzymatic<br>
development for reading, for example ELISA, ELOSA, et'.1., a reference curve is also required for each manufacturing batch of the tests. Moreover, additional calibration points for resetting the reference curve for each user must be performed tor correcting variations from one machine to another, from one temperature to another, from one laboratory to another, for possible aging of the reagents, etc. These additional calibration tests must be repeated regularly by the users, for example every week.<br>
This method of quantification, although usual, is therefore complicated and expensive. Moreover, it is not easily applicable to DNA chips; in fact, it would be necessary to use a DNA chip for each point of the reference range, and Keep repeating the experiment (at each change of batch of marker, or at regular intervals, for example every month, to check for possible drift of the instrumentation), which is too expensive for the laboratories.<br>
Therefore there is a real need for a chip and a method of analysis which enable the many aforementioned problems of the prior art to be solved, and notably which art reliable, accurate, and reproducible; which can be applied to the various known and future DNA chips; which make it possible to compare the signals between several experiments; which make it possible to avoid using a DNA chip for each point of the reference range; and which make it possible to avoid constantly repeating the calibration test (at each change of batch of marker, or at regular intervals, for example every month, and to check for possible drift of the<br>
instrumentation), so as to reduce the complexity and costs of analyses on chips, for example on biological chips, for the laboratories, in industry and in resea rch.<br>
In the following description, the references in square brackets [ ] refer to the list of references given after the examples of application of the invention described below.<br>
DESCRIPTION OF THE INVENTION<br>
The present invention meets precisely this need and solves the many aforementioned problems of the prior ai"! , and others, notably in that it provides an analysis chip, as well as a method of analysis using this chip, as defined below.<br>
The analysis chip of the present invention is an analys'is chip&gt; for at least one analyte present in a liquid sample, said chip comprising:<br>
at  least  one analysis  spot  of  said,  at<br>
least one, analyte, said analysis spot being arranged<br>
on the chip to permit the recognition and specific<br>
immobilization of the analyte; and<br>
a reference range (G), said reference range<br>
being  constituted  of  several  reference  spots  each<br>
arranged  on  said  chip  in  a  defined  manner,  and<br>
independently of one another, each reference spot of<br>
this range comprising, immobilised on its surface and<br>
in a defined proportion P that is different for each<br>
spot  relative to the other reference spots of said<br>
ranne:   at least one probe reference molecule (PRM)<br>
permit tin:.; the recognition and specific binding to a<br>
a)	defined target reference molecule (TRM), and/or (ii) at least one inert molecule (IM) incapable of recognizing and binding to said target reference molecule, the probe reference molecule and the inert molecule both being incapable of recognizing and immobilizing said<br>
number of PRMs<br>
analyte (s);    with   r =	    and<br>
number of PRMs + number of IMs<br>
the sum of the number of PRMs + number of IMs being constant from one reference spot to another.<br>
The method of analysis of the present invention is a method of analysis in vitro of at least one analyte that may be present in a liquid sample comprising the following stages:<br>
(a) optionally, immobilization of a first detecting means on the analyte;<br>
(£'! addition, to the sample to be analysed, comprising the optionally labelled analyte, of a target reference molecule (TRM) on which a second detecting means, .Ldentical to or different from the first detecting means, has optionally been immobilized, said TRM being capable of recognizing and of binding specifically to the probe reference molecule (PRM) of an analysis chip according to the invention, said TRM being added to said sample in a sufficient amount to saturate the PRM of the reference range (G) of said chip, thus creating a reference range that is a function of the defined proportion P which is different f gr ea chrererence spot;<br>
(y) bringing the sample to be analysed comprising the TRM into contact with said analysis chip<br>
comprising at least one analysis spot of said at least one analyte in physicochemical conditions such that: on the one hand, the analyte to be analysed, if it is present, binds to its analysis spot on the chip; and on the other hand, the TRM specifically recognizes the PRM arid binds to the latter on the various reference spots of the reference range of the chip;<br>
(5) determination of a reference signal emitted, if applicable owing to the second detecting means, by each reference spot of the reference range, said signal being a function of the amount of target reference molecule immobilized on the latter; and<br>
(s:) determination of an analysis signal emitted, if applicable owing to the first detecting means, by said, at least one, analysis spot and being a function of the amount of analyte immobilized by the analysis spot, and comparison with the signals determined in stage (5) for expressing this analysis signal as a function of P.<br>
The term "analyte" means all or part of the particle or molecule that we wish to analyse, i.e. detect and/or quantify, for example a microorganism, a bacterium, a fungus, a virus, a eukaryotic ceil, a cell such as a tumour cell, a chemical compound or molecule, a molecule such as a peptide, a protein, a glycoprotein, a lipoprotein, an enzyme, a polysaccharide, a lipid, a glycolipid, a lipopclysaccharide, a ribonucleic acid (RNA), a deoxyribonucleic acid (DNA), a hormone, an antigen, an antibody, a growth factor, a hapten, etc.<br>
The sample can be a solution, a cellular extract, or a sample obtained from an animal or veaetabic: organism. This sample can be diluted, if necessary, for use in the present invention. A person skilled in the art will be familiar with the handling of such samples and their dissolution and/or their di]ution, their purification and/or their concentra-ion, for analysis on an analysis chip, for example a biological or chemical chip. These same manipulations are employed for carrying out the present invention.<br>
The chip of the present invention comprises, in combination, on the one hand, at least one analysis spot of a" least one arialyte to be analysed, and on the other hand, a reference range, both arranged on the surface : f one and the same chip according to the present invention.<br>
Tne chip of the present invention is constituted in the same way as the analysis chips of the prior art, except that it additionally comprises a reference range in the sense of the present invention. The stages of production of the reference range are therefore added to those of fabrication of the chip. DNA cnips that can be used for carrying out the present invention arid their method (s) of production are described for example in document [1] of the bibliography. Protein chips that can be used for carrying out ::he present invention and their method (s) of production are described for example in documents |2| and of the bibliography.<br>
The chip) of the present invention therefore has a very wide range of uses as an analysis tool. Thus, by "analysis" we mean "qualitative analysis" of a sample, i.e. detection of analyte(s) present in a sample and/or "quant it ative analysis", i.e. quantitative determination of analyte(s) present in a sample. Some examples are given below.<br>
According to the invention, the "spots" comprise means for recognizing and immobilizing, exclusively and specifically, either "the analyte" (analysis spot) or "the target reference molecule" (spot of the reference range).<br>
for the "analysis spot", these means comprise the probe molecules specific to said analyte. The specific probe molecule can be for example a molecule of DNA or of RNA permitting recognition and immobilization of a complementary analyte (DNA or RNA) by hybridization; of an antigen or of an antibody recognizing and immobilizing the analyte by an interaction of the antigen/antibody type; of a protein recognizing and immobilizing the analyte by an interaction of the protein/protein type; of an enzyme or of a substrate recognizing and immobilizing the analyte by an interaction of the enzyme/substrate type;<br>
For example, when the analyte is a nucleic acid, Lh at least one, analysis spot is a spot f unct.1 ona ;. izea by a nucleic acid complementary to the latter. For example, when the analyte is an antibody or an, the, at least one, analysis spot is a spot<br>
Functionalized respectively by an antigen or an antibody -<br>
The methods of production of these analysis spots and their functdonalization are known to a person skilled in the art and are disclosed for example in the aforementioned documents. Functionalization of the analysis spots of the chip can be effected for example by mean;:- of a robot for dispensing drops of a functiona_ization solution, for example a robot of the Packard Instrument or GeSim type (registered trademarks).<br>
The chip of the present invention can comprise several analysis spots, which may be identical or different, ir. the same way as the analysis chips cur rent, ly used in the laboratory.<br>
The reference range of the present invention is constituted of reference spots, which are also spots for recognition and immobilization of a target, but these reierence spots are characterized in that they cannot, recognize and immobilize the analyte or analytes to be analysed, but only a defined target reference molecule. The reference spots of this reference range are as defined above in points (i) and (ii) of the method of the invention.<br>
According to the invention, the probe reference molecule, the target reference molecule, and the inert molecule are selected together and as a function of the ariatyte i .;&gt;r which the chip of the present invention is intenoed, so that they have least possible interaction with the analyte.<br>
According to the invention, preferably, the target, reference molecule is of the same nature as the analyte. It can also be of a different nature. The main point is that the target reference molecule should neither be recognized nor immobilized by said, at least one, analysis spot.<br>
According to the invention, the probe reference molecule is selected for specifically recognizing and immobilizing said target reference molecule. Preferably, this probe reference molecule is of the same nature as the probe molecule that is specific to the analyte. It can also be of a different nature. The main point is that the probe reference molecule neither recognizes nor immobilizes the analyte.<br>
F:or example, according to the invention, the target. reference molecule and the probe reference molecule can be, for example, complementary cligonucleotides (DNA or RNA) for recognition and immoDi 1 izatior, of the target reference molecule by hybriaization on the spots of the reference range comprising the probe molecule; for example an antigen and its specific antibody permit the target reference molecule to be recognized and immobilized by an interact.] en of the antigen/antibody type on the spots of the reference range comprising the probe molecule; for example proteins permitting the reference target to be recognized and immobilized by an interaction of the protein/protein type on the spots of the reference range comprising the probe molecule; etc.<br>
According to the invention, the at least one anaivte and the target reference molecule are<br>
preferably oligonucleotides or antibodies. The functionalization of the various spots (analysis and reference range) on a chip according to the present invention using probe and/or inert oligonucleotides or antibodies or antigens can be carried out using the techniques of functionalization with which a person skilled in the art is perfectly familiar in the field of chips with oligonucleotides or antibodies or antigens. Techniques that can be used are described for example in the aforementioned documents.<br>
According to the invention, the inert molecule of the reference range, also called "non-specific" molecule, is selected in such a way that it neither recognizes nor immobilizes the target reference molecule and the analyte or analytes. Preferably, accordinc; to the invention, the inert molecule is of the same nature as the probe reference molecule. In fact, this means it is not necessary to take inro account the non-specific interactions that might interfere with the analysis, while using the chip of the present invention, for example between the inert molecule and the analyte or analytes on the one hand and the target reference molecule on the other hand, and ontain a reliable reference range. In the preceding examples, the inert molecule of the reference range can therefore- be respectively a DNA or an RNA, an antigen or an antibody, a protein, an enzyme or a substrate. The reference inert molecule is also immobilized on the spots c; the reference range according to the techniques of chip functionalization known to a person<br>
skilled i:i the art, for example those described in the aforementioned documents.<br>
According to the invention, preferably the probe reference molecule, the target reference molecule and the inert molecule are oligonucleotides. The productior. of oligonucleotide sequences, for example artificial, meeting the definitions of the probe and target reference molecules and of the inert molecule of the present invention, and functionalization of spots on a chip- using these probe and inert oligonucleotides for the production of the reference range is in fact facilitated and uses the techniques with which a person skilled in the art is perfectly familiar in the field of oligonucleotide chips.<br>
In-situ systems: certain methods of production of DNA chips employ in-situ synthesis of the oligonucleotides (for example the methods of production of the GeneChip chips of the company Affymetrix (registered trademark), or the chips of the company Agilent). In these methods, each oligonucleotide is synthesized in situ on the chip. The three-dimensional addressing of the monomers to be coupled is effected either at the level of deprotection in a conventional cycle or synthesis of oligonucleotides (photo-deprotection for Affymetrix); or at the level of coupling o£ the monomers ("spotting" for Agilent). Such a technique of in-situ synthesis makes it possible to produce the analysis spots of the chip of the present invention, and, by adding as many cycles of synthesis as there are points in the reference range, and in these cycles of synthesis, by directly coupling the<br>
mixture of probe reference oligonucleotide/inert (non­specific i oligonucleotide at a suitable place on the substrate forming the chip (instead of only coupling one: monomer), a reference range according to the invention is produced.<br>
According to the invention, the reference range is constituted of several reference spots arranged on said chip in a defined manner and independent of one another. In fact, for the reference range to be usable, it is necessary for each reference spot to be distinct from the- other reference spots so that the various spots in this range supply signals that are independent ana can oe utilized for the analysis when using the chip.<br>
t.ach spot of the reference range is distinguished from the other spots of this range by its defined proportion P as defined above: the amount of probe reference molecules relative to the inert molecules is therefore known for each reference spot. This defined proportion P is obtained on each reference spot by iunctionalizing each spot, independently of the others, by means of a solution with a defined concentration of probe reference molecule and/or inert molecule. According to the invention this proportion P can range from 0, in the case when the reference spot only contains the inert molecule, to 1, in the case when the reference spot only contains the probe reference molecule. The main thing is that the total amount of probe reference molecule and inert molecule (PRM i IK) should be equal from one spot to another, for the reference range to be accurate.<br>
Preferably, in the example where the reference probe and inert molecules are oligonucleotides, they are selected in such a way that they have equal effectiveness of attachment to the spots of the reference range during fabrication of the chip. Thus, after attachment, the proportion of reference oligonucleotide is equal to that of the initial dilution. This makes it possible to pass advantageously from the unit "% of dilution" to the unit "% of reference oligonucleotides attached" in the course of an analysis using the chip of the present invention.<br>
In the example of oligonucleotide chips according to the present invention, in contrast to the cDNA chips or PCR product chips, preferably, the total amount "reference probe oligonucleotides + non-specific inert oligonucleotides" is equivalent to the amount of probe oligonucleotides used for each analysis spot for the fabrication of the chip. Oligonucleotides of the same lenath will also preferably be used. Thus, as the efficiency of attachment has little dependence on the sequence, for each spot, it is possible to express the signal in "percentage of spots hybridized", 100% corresponding to saturation of the surface.<br>
Preferably, the spots of the reference range are disposed on the chip linearly and are arranged in relation to the proportion P of each spot, for example from the spot with the highest concentration to the spot with the lowest concentration of probe reference molecule. In fact this gives a calibration scale that can be utilized more easily when using the chip. The increase in the proportion P from one reference spot to<br>
another close to it in such a range can be linear, for example, in percentage of probe reference molecule on successive different spots, of 0%; 20%; 40%; 60%; 80% and lCiO'6, or nonlinear, for example 0%, 5%, 10%, 20%, 50%, 100% or even 0%; 0.1%; 1%; 10% and 100%. There is no other restriction on application of the present in vent lor. than that of being able to assign clearly, to a siqnai supplied from a spot of the reference range, a defined proportion of probe reference molecule.<br>
The number of reference spots of the reference range: is determined as a function of the required accuracy of analysis and of the dynamic range of the test when using the chip. For example, three reference spots for example with, successively, 0%, 50% and 100% of probe reference molecule only provide three calibration results as a basis for the results of analysis of the analyte, whereas five reference spots for example with, successively, 0%, 20%, 40%, 60%, 80% and 100% of probe reference molecule provide six calibration results, giving a more accurate reading of the results of analysis. For example, the reference range can comprise from 1 to 50 reference spots, for example from 2 to 20. A person skilled in the art will easily determine, depending on the required accuracy of analysis and the dynamic range of the concentrations, the number of different reference spots required and their arrangement or disposition on the chip.<br>
The appended Fig. 1 shows schematically, in cross section, a reference range (G) on a chip (P) aocoiainq to the present invention. This diagram shows reference spots (la, Ib, Ic and Id) , on which<br>
the probe reference molecules (PRM) (shown as thick lines)1 and the inert molecules (IM) (shown as thin lines) d.re immobilized. The analysis spots are not shown. The left-most spot, only contains PRM (100%), and the right-most spot only contains IM (0% PRM). The intermediate spots contain intermediate proportions of PRM. The sum PRM 4 IM is constant from one spot to another.<br>
According to the invention, several, reference ranges, which may be identical or different, can be arranged on one and the same chip. The use of several reference ranges, although not obligatory, can in some cases .improve the accuracy of the analyses performed. Thi s may be so for example when the sample comprises several analytes to be analysed simultaneously on the chips, by an oligonucleotide and an antibody. A person skilled in the art will easily determine, depending on the analysis for which the chip is intended, the most appropriate implementation.<br>
Tne full importance of the present invention becomes clear when the chip having this reference range is used for the analysis of a sample comprising or likely to comprise one or more analytes to be analysed, notably in the method of the invention.<br>
In the method of the invention, the analyte and target reference molecule are preferably of the  nature, without being limited to this embodiment, preferably, the analyte and the target reference are oligonucleotides or antibodies,  without limited to this embodiment.  In a particularly advantageous embodiment of the invention,  the probe<br>
reference molecule, the target reference molecule and the inert molecule are oligonucleotides for the same reasons as those presented above.<br>
.Similarly, and for the same reasons as those presented above, the, at least one, analyte is advantageously a nucleic acid, and the analysis spot of said ana_yte on the chip is a spot functionalized oy a nucleic acid complementary to the latter. The analyte can also be an antigen or an antibody, and the analysis spot of said analyte on the chip can be a spot functionalized respectively by an antibody or an antigen recognizing and immobilizing the latter.<br>
Stage (a) of the method of the invention is optional. In this stage, a first detecting means can be immobilized on the analyte. It can be any means known to a person skilled in the art that can be used for detecting an analyte immobilized on a specific probe molecule (probe/analyte combination) immobilized on a chip. This means can involve for example a first marker, which can be immobilized either on the specific probe molecule, or on the analyte. This means can also be another molecule for indirect labelling, for example a biotin, depending on the labelling technique employed. The markers and the labelling techniques that can be used for carrying out the present invention are, without limitation, those known to a person skilled in the cirt in the field of analysis chips, for example biological chips, for example DNA or RNA chips, antibody;antigen chips or chips using an e n 2. vme /substrate interaction.<br><br>
The markers that can be used can be selected for example from the group comprising fluorescent markers; radioactive markers; standard, coloured or fluorescent latex particles; photonic crystals (also called "quantum dots"); colloidal gold; and enzymatic markers. Among the fluorescent markers, we may mention fluorescein, Cy3, Cy5 and rhodamine. Among the radioactive markers, we may mention for example P3", PJJ, S~'~', I125, H°. .Among the enzymatic markers, we may mention alkaline phosphatase, horse radish peroxidase (HRP'i, p-galactosidase, acetylcholinesterase. For these enzymes, there are coloured, fluorescent or luminescent markers. In this case, they are molecules transformed by the enzyme, which, after transformation, become coloured (or the absorption wavelength changes), fluorescent, luminescent or redox. The enzymatic markers that are used routinely in diagnostic iminunoanalyses can also be used. These fluorescent or enzymati: markers, and others, that can be used in the present invention, are available for example from the catalogues of the companies Sigma/Aldrich, Molecular Probes, Airier sham Pharmacia Biotech., Stratagene, etc. Documents [4j and [5] of the bibliography describe methods of using these markers that can be used for carrying out the present invention. Markers in the form of particles that can be used in the present invention are available from several suppliers, for example Molecular Probe, Miltenyi, Estapor/Merck, Polymer Solutions, etc. Document [6] of the bibliography describes methods employing these markers in the form of latex or colloidal particles that can be used for carrying out the present invention. For the use of photonic crystals it will be possible to use for example the method described in document [7] of the bibliography.<br>
For certain markers, especially the enzymatic markers, the detectable product may move some distance away from the spot where it was produced, by molecular diffusion, and contaminate neighbouring spots. This can cause a disturbance in the use both of the reference range and of the signals emitted by the spots for recognition of the analyte or analytes. That is why the inventors prefer luminescent markers, as there is limited diffusion: once the enzymatic product has emitted a photon, it cannot emit any more. To avoid the phenomenon of diffusion, it is also possible to use so-called precipitating enzymatic substrates: the enzymatic product is insoluble, and is precipitated in situ, i.e. on the spot. For example, tetramethylbenzidine (TMB) can be used for the HRP enzyme, and the combination of Nitroblue Tetrazolium with 5-bromo-4-chloro-indolyl phosphate (NET + BCIP) for alkaline phosphatase.<br>
According to the invention, in order to avoid these phenomena of diffusion which can interfere with the reading of the chip with certain markers, but also, in general, to avoid any phenomenon of diffusion between adjacent spots, the inventors recommend the use of chips on which the spots (for recognition and of the analyte and/or of the reference range are provided with means for ensuring that droplets of the sample obtained in stage (6) are formed<br>
exclusively on each of the spots, without these droplets being able to diffuse between them. Tnese means can be for example in the form of a boundary, for example of resin, surrounding each spot. These means can also be in the form of a ring around each spot and displaying much greater wettability for the sample than the rest of the chip surface, so that a droplet of sample is retained by each ring exclusively on each soot. This ring can be for example a ring of black silicon obtained by microgravure on a silica surface or an electrode for capturing a droplet of sample by electro-wetting. Regardless of the means employed for formation of droplets, the chip surface is preferably non-wetting with respect to the sample. The chip of the present invention can therefore advantageously be provided with these means in order to increase the quality or the reliability of the measurements obtained.<br>
In stage ((3), the target reference molecule on which a second detecting means identical to or different from the first detecting means is optionally immobilized, is added to the sample, as defined above. This second means can involve for example a second marker, which can be immobilized on the target reference molecule or on the probe reference molecule. It can also involve a molecule other than a marker, but which it employed in a method of indirect labelling (for example biotin) . This marker can be selected for example from those mentioned above. The second detecting means is preferably, but not necessarily, identi-ca.i to the first detecting means. The main thing<br>
is that the? immobilization of the target reference molecule on the probe reference molecule of the reference range can be detected on the chip by a detectable signal.<br>
According to the invention, the analyte alone may be labelled, or only the target reference molecule, or both, or neither of the two. In fact, some methods of detect.ion are able to detect molecular interaction (probe/target immobilization on a chip) without. a marker. ;\!ome examples are given below.<br>
The target reference molecule on which the second detecting means is optionally immobilized is added to the sample in sufficient quantity so that the maximum r.umber of probe molecules of the spots of the reference range recognize and immobilize the target molecule in the experimental conditions used for bringing the sample into contact with the chip (stage  Addition to the sample can be carried out by, preferably homogeneous, mixing with the sample.<br>
In stage  (y) ,  the mixture from stage  ((3)  is brought  into contact with the  chip of the present invention,  in the physicochemical conditions defined above. These conditions are known to a person skilled in the art in the field of analysis chips, for example biological chips,  for example  DNA chips,  RNA chips, antibody/antigen chips, protein chips, etc. These are, for   example  for  the  aforementioned  chips,   the conditions  cf  pH,  temperature  and  ionic  strength permitting the recognition and immobilization of the analyte  or  analytes  (DNA,  RNA,  antibody,  antigen, protein, etc.) on their recognition spot or spots, and<br><br>
of trie target reference molecule (DNA, RNA, antibody, antigen, protein, etc.) on the spots of the reference range comprising the probe reference molecule. The advantage or the present invention is that the recognition and immobilization of the analyte by the, at least one, analysis spot and the formation of the refer en c*--r range by the immobilization of the probe reierenc'r molecule on the reference spots are effected simultaneously and in the same operating conditions. Tne effect is obvious: the signals emitted by the reference range have been obtained in the same operating conditions as were used for immobilizing the analyte. The results of analysis are therefore more reliable than those obtained by the methods of the pr 1 or ar 7. <br>
..n general, the analysis spots and the spots of the reference range can be brought into contact with the sample by the usual means of the technology of analysis chips used for the distribution of a sample on functionalized zones.<br>
in a chip comprising means for limiting the formation. of droplets of the sample on the spots of the  bringing into contact can be effected very simply by covering the spots with the sample, then withdrawing the sample so as only to leave the droplets of sample captured by these means on the analysis spots and the spots of the reference range of the chip.<br>
'I he appended Fig. 2 shows schematically, in cross section, the reference range (G) of Fig. 1 after  stage of bringing into contact. This diagram, shows in target reference molecules (TRM) immobilized on the<br>
probe reference molecules (PRM). The TRM are labelled by means of a marker (Mq) . It is clear that the left­most spot 100% PRM) will give the strongest signal from the marker, and the right-most spot (0% PRM) will not. give a signal. Intermediate signals will appear on the spots in the middle.<br>
As with the use of the analysis chips known to a person skilled in the art, this stage (y) can be followed by stage(s) of washing and rinsing in physicochemical conditions which do not disrupt the immobilization of the analyte on its analysis spot, as well as the immobilization of the target reference molecule on the probe reference molecule. These stages of wash:ng and rinsing are known to a person skilled in the art: and can. be found in the aforementioned documents. They make it possible to remove the excess of reagents and molecules that are not immobilized on the chip...<br>
In stage (6) of the method of the invention, a signal emitted by each spot of the reference range, reflecting the amount of immobilization of target reference molecule immobilized on each spot of the reference range, is determined. In stage (c) of the method of the invention, notably a signal emitted by each analysis spot, reflecting the amount of anaiyte immobilized on each analysis spot, is determined.<br>
These determinations can be performed in various ways depending on whether or not markers are used in the method of the invention. The main thing is to oe able to assign a definite signal representative of the amount of targets immobilized on each of the<br>
spots of the reference range, and a definite signal representative of the amount of analyte immobilized on e a c r  a n a ": y s i s spot.<br>
Was markers are used, the technical means that are appropriate for the detection of these markers are used. If the first and second detecting means used employ the same markers, the detecting means will be the same for the various analysis spots and for the various spots of the reference range. These techniques of signal reading are known to a person skilled in the art. In the aforementioned examples of markers, they make it possible to detect for example an amount of fluorescence, an amount of radioactivity, an amount of enzymati-- reaction product, luminescence, etc. The aforementioned documents relating to markers present techniques for determination of these signals which can be used for implementing the method of the invention.<br>
According to the invention, the signal from each spot of the reference range and/or from each analysis spot can also be determined, especially if markers are not used, by a method selected from the group comprising the methods of detection by surface plasmon resonance, methods of photothermal detection, eliipsometric methods, methods of photometric detection and methods of acoustic detection. These methods are known to a person skilled in the art. Documents [8] and .91 of the bibliography disclose methods of photothermal detection that can be used for carrying out r. he method of the present invention for detecting a sicmal without a marker.<br><br>
According to the invention, it is possible to use different methods of determination of the signals, with or without markers, for the analysis spot and for the reference range. Thus, the method of the invention makes it possible to compare the results obtained with a method without labelling (analysis spot or reference range; with other results obtained on the same chip with labelling (analysis spot or reference range), for example by fluorescence. The main thing is that the operating conditions of immobilization of the analyte on its analysis spot and of the target reference molecule on the reference range should be effected simultaneously on the same chip and in the same opera t. inq conditions .<br>
."in stage (e), the signal emitted by each analysis spot is compared with the various signals emitted oy the reference range for expressing said signal of the analysis spot as a function of the proportion P defined above. In fact, the intensity of the signal emitted by each analysis spot is compared with the various signals of the reference range for deducing from that, relative to an equivalent signal of the' reference range, a signal value relative to the proportion P. Other comparisons which take account of other criteria of the reference range are also possible (IM number, PRIM number, TM/PPJM ratio, PRM/IM ratio, e 11. /<br>
Advantageously, if the nature of the analyte ana of trie reference probe is the same, the proportion of analy t.e ( s } on each analysis spot can be determined directly, reliably and reproducibly, by simple reading.<br>
determination is carried out by means of a reference range performed simultaneously, from the same sample, ana therefore in the same operating conditions, as for t h e a n a '1 y t e .<br>
of the nature of the analyte is different from that of :he probe reference molecule, it is possible to determin-- a relative value, which however has the advantage or being stable, reproducible, and of integrating the operating conditions of the analysis with these which permitted the simultaneous formation of the reference range, on the same chip, from the same sample. This relative value is therefore more reliable than those obtained by the methods of the prior art.<br>
in a particular example of application of the present invention, when the analyte, the probe molecules specific to the analyte, the probe reference molecules, reference targets and inert targets are ol i gcnucIeotides, stage (y) of bringing into contact is for the purpose of on-chip hybridization of the analyte with the probe reference molecule, and hybridization of the target reference molecule with the probe reference molecule. For this hybridization, a target reference oiigonuc1eotide containing the sequence complementary to the probe reference oligonucleotide of the reference range, and labelled preferably, but not necessarily, in the same way as the analyte, is added to the sample containing the analyte. This oligonucleotide is used in si 1 ant excess, in such a way that all the probe reference molecules of the spot of the reference range comprising 100? of reference oligonucleotide are<br>
hybridized in the experimental conditions of hybridisation used. Thus, for each spot of the reference range, all the spots that can be hybridized will be hybridized. It will then be possible to express the signal from each analysis spot of the DNA chip) in "'i of reference dilution". By using, in different analyses, the same reference oligonucleotides arid non­specific oligonucleotides (i.e. not recognizing and not immobilizing the analyte), this unit is independent of the sample, of the labelling chemistry, of the marker used (green, red or blue, fluorescent, photonic, etc.), of the conditions of hybridization (provided that the hybridization of the reference oligonucleotide is complete;, of the chip batch, of the efficiency of attachment, etc., enabling the experiments to be compared directly with one another, without additional experiments as was necessary in the methods of the prioi ar:. .<br>
Generally speaking, it is possible to perform a direct reading, or construct, from the reference range, a reference curve, for example (signal determined) -- f (P), which can be used for analysis of trie ana'iy-sis spots of said chip.<br>
The chip and the method of the present invention are therefore reliable, accurate, and the results are reproducible. The present invention applies to the various known and future DNA chips; it makes it possible to compare the signals between several experiments, avoids using a DNA chip for each point of the reference range, and avoids the constant repetition<br>
of the Galioration experiments as is necessary with the methods of the prior art.<br>
Advantageously, in the case of the methods of detection with enzymatic labelling, for example luminescence, the present invention makes it possible to avoid having to calibrate the activity of each batch of enzyme, at the manufacturer and at the customer. This therefore represents a considerable simplification of the "chain of metrology".<br>
Trie present invention therefore reduces the complexity and the costs of on-chip analyses, for example :n a biological chip, for the laboratories, in industry and in research.<br>
Since the present invention can be based on all the ana-lysis chips of the prior art employing spots for recognit-Lon and immobilization of an analyte, a person skilled in the art will readily understand that it will find application in all fields where this type of chip can be used.<br>
Thus, the present invention also relates to a diagnostic kit comprising a chip according to the invention. In this case, the analysis spots of the chip of the present invention comprise the probe molecules specific to recognition of the analyte required for obtaining the results which it will then be possible to interpret for establishing a diagnosis. For example, it car. be a question of probe molecules selected from oiigonucleotides, cDNA, antibodies, lectins, aptamers,<br>
the present invention also relates to an analysis kit comprising a chip according to the<br>
invention. In this case, the analysis spots of the chip of the present invention comprise the probe molecules specific to recognition of the analyte required in oraer to carry out said analysis. It can be a kit for qualitative or quantitative analysis. For example, it can oe a question of probe molecules selected from oligonucieotides, cDNA, antibodies, lectins, aptamers, etc.<br>
Trie present invention also relates to the use of a chip according to the present invention for monitoring variations in gene expression in tissue cells i,rj vitro. For example document [1] describes a protocol that can be used on the chip of the present invention for monitoring these variations in gene expression.<br>
The present invention also relates to the use of a chip according to the invention in a method of genotyping in vitro. For example document [1] describes a protocol that can be used on the chip of the present invention for carrying out a method of genotyping.<br>
For example, concerning the usefulness and for the use of tne gene expression chips according to the present invention, reference may be made to the description given in document [10] which describes protocols that can be used for carrying out the method of the invention in these applications.<br>
For example, regarding the use of the present invention in a method of genotyping, for example for diagnostic purposes, reference may be made to documents [ 1 ;. ] ano [12] each of which describes protocols that<br>
cars  of used  for  carrying  out  the  method  of  the invention in this application.<br>
Other characteristics and advantages will become clear on reading the following examples, given for purposes of illustration, referring to the appended drawings,<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 is a schematic representation of a reference range on a chip according to the present invent ion.<br>
Fig. 2 is a schematic representation of a reference range on a chip according to the present invention when the target reference molecules are immobilized on the probe reference molecules. Markers are also shown.<br>
Fig. 3 shows two negatives of images of a reference range on a chip according to the invention with exposure of 560 ms, gain 0 (left) or 16 (right).<br>
Fig. 4 is a graph showing curves of signals as a function of the proportion of reference oiigonucleotide on the capturing spots, for four independent experiments (Cl, C5, C6 and C7).<br>
These curves constitute reference curves obtained from the reference ranges according to the present invention.<br>
Fig. 5 is a graph summarizing the results of signals determined for each of tne spots of a reference range of the present invention with labelling with fluorescent particles.<br>
Fig. 6 is a schematic representation of the- setup used by the present inventors for the fabrication of chips according to the present invention.<br>
Fig. 7 is a schematic representation of the biochemical mechanism of indirect labelling, with latex particles, of immobilization of the target reference molecule on the reference probe molecule on a chip of the present invention.<br>
Fig. 8 is a photograph of a chip substrate used for fabrication of a chip according to the present invention.<br>
Fig. 9 is a schematic representation of an analysis chip according to the invention for detecting the presence of the viruses RSV A and RSV B responsible for bronchiolitis (RSV: Respiratory Syncytial Virus), and of the viruses Irif A (influenza A, and Inf B (influenza B) responsible for influenza in<br>
EXAMPLES<br>
Example 1: Chip according to the present invention 1.1   Chip substrate<br>
The substrate used for fabrication of a chip according to the Invention is a monochrome imager of the type "VV5501 VGA Monochrome Image Sensor" (registered trademark of ST Microelectronics) the main techr. ica... characteristics of which are shown in the f o} low j n," table :<br>
(Table Removed)<br>
A top view of this substrate is shown in the photograph in Fig. 8. It has a silicon surface at its centre. Kig. 6, described below, is a schematic cross section of this chip which also has a plastic tube glued on top (see below).<br>
This substrate is in fact intended for the manufacture of a low-cost digital camera with gain and exposure control functions and theoretically has a signal to noise ratio of 56 dB. Moreover, it only requires very few external components for its operation arid is provided with a kit (software and hardware) for management of the internal parameters of the camera.<br>
1.2 First stages in fabrication of the chip<br>
Chip substrates as described in the preceding paragraph 1.1 were modified for functionalizing the silicon surface (see Fig. 1). To gain access to the silicon surface of the photodetector (at the centre of the substrate), the glass cover was removed and the bonding connections were buried under globtop resin (a general t erm denoting any resin used for protecting the bonding connections). The passivation layer (silicon nitride) at the centre of the photodetector was functlonalized (silanization + attachment of the spot capture probes).<br>
for the subsequent experiments of functionaiiration of the silicon surface, a plastic tube was glued (UV adhesive) on the chip to define a<br>
reaction chamber above the active surface, in which the hybridizations, washings, and development were carried out. Fig. 6 is a schematic representation of the setup used: it comprises a resin (R) for protecting the connections of the chip, the silicon surface (P), a ceramic frame (Cc), the glued plastic tube (Tp) , and the spots (se = a reference spot, and sa = an analysis spot! on which either the probe molecules and/or inert standards or the probe molecules specific to the analyte are immobilized. This schematic diagram shows that the tube is provided for containment of a reaction liquid (Lr) in contact with the silicon surface.<br>
(S) and (Cc) are shown in top view in the photograph in Fig. 8, shown prior to gluing of the tube and deposition of the resin.<br>
1.3   Functionalization of the silicon surface by probe and inert molecules which are oligonucleotid.es 1.3.1  Revealing the silanols<br>
(i) Silanols are first revealed on the silicon nitride chip surface: Agitation for 2 h at room temperature in the solution: NaOH 1 g / water 3 ml / ethanol 99% 4 ml. Copious washing with water. Agitation for I h in 0.2 N HC1, then copious rinsing with water. Drying a: 80 °C in the stove.<br>
ii) silanization: the plates are immersed at room temperature for 24 hours in 3-aminop r:opy.l triethoxysilane at 10% (vol/vol) in etnanol, under argon. Then drying. Then washing with 99V ethariol, then a second washing with 99% ethanol<br>
+ ultrasound. Each washing takes several minutes. Drying. Annealing for 3 hours at 110 °C in the dry.<br>
J	i) pre-activation: the plates are immersed<br>
in KOH solution (1.5 g KOH in 20 ml water/. Incubation for 5 to 10 minutes, at room temperature, with agitation. Then rinsing with pure water.<br>
iv) activation: the plates are immersed in a solution of glutaraldehyde (4 ml of qlutaralaehyde in 16 ml ot water; . Incubation 1 h 30 min at room temperature. P.insing with pure water. Drying.<br>
1.3.2   Iirmobi lization of the oligonucleotides<br>
After activation of these NH; groups by giutaraldehyde, aminated oligonucleotides (ODN hereinafter) can be deposited on the various spots of the criip surface for their attachment, by means of a robot from tne company Karl Ziiss equipped with a pdpette firom the company Mikrodrop. The diameter of the spots was evaluated at approx. 140 urn.<br>
Preparation of the solutions of oliqonucleotides to be deposited: the solutions used each comprise 10 uM of ODN in total (probe r inert) to be deposited on each spot of the reference range of the chip. They contain a mixture of reference probe ODN and inert 'non-specific) ODN :tn proportions 100% reference, 10% reference, Is reference, 0.1% reference, and 100% non-specit! ic, i.e. a reference range produced with 5 spot: s or points. These 5 solutions were deposited on 5 of spots with the robot from the company Karl Ziiss, equipped with a piezoelectric pipette from the oompari;.' Mikrodrop The spots were deposited with<br>
spacing  of  200 urn.  The  distance  between  lines  is approx. 400 urn. Spot diameter is approx. 140 um.<br>
A	1  the  oligonucleotides  in  this  experiment<br>
have 3 length of 22 bases. The sequences used are as<br>
Sequence  of  the  probe	oligonucleotide<br>
(SEQ ID No.: 1):<br>
5'  (H2N-)ATGAACAAGTAGATAAATTAGT	3'<br>
Sequence  of  the  inert	oligonucleotide<br>
(SEQ ID No.: 2) :<br>
5'  (H2N-)CTAAAGGAATAGTGTAAATAAT	3'<br>
Tr.e -NH2 groups serve for attachment on the aldehydes of the glutaraldehyde used in this example.<br>
uhe sequence of the reference target oligonucleotide is described in example 3)<br>
1.4   Hybridization and labelling<br>
These  chips  were  hybridized  with  a  15 nM solution  of  HRP conjugate at  37°C  for  30 minutes, washed, tr.en imaged in the presence of Pierce substrate mixture  f "Super ELISA Femto Maximum Sensitivity Kit")<br>
(registered trademark). The HRP conjugate is an oligonucleotide that is complementary to the probe oligonucleotide, to which the HRP enzyme was coupled<br>
(HRP: Horse Radish Peroxidase).<br>
aj 1 the hybridizations were carried out, in a humid chamber, at: 37 °C for 30 minutes in TE IX, NaCl 1 M, Triton X-100 0.05%. The volume of hybridization is 200 ul (i.e. a liquid layer approx. 3 mm thick) . No<br>
agitation. Hybridization is followed by 3 washings of 400 p] in TE IX, NaCl 1 M, Triton X-100 0.05%, then one washing .in TE IX, NaCl 1 M. TE IX denotes: Tris 10 mM, EDTA 1 mM, pH 8.<br>
After hybridization and the washings, the chip is mounted on a reading card.<br>
1.5 Determination of the signals of the reference range and reading<br>
The rinsing medium is removed, then 200 ul of the substrate mixture SuperSignal ELISA Femto Maximum Sensitivity Substrate (Pierce, #37075 (registered trademark)) is added.<br>
linage acquisition: 50 images with gain 15 divisor 15 (1.8 images/s), 50 images gain 0 divisor 15 (1.8 imaoes/s) . The first image series enables weak signals to be visualized. The second permits visualization of strong signals, which saturate the sensor ir. the above conditions. The 50 luminescence images aie averaged pixel by pixel, as well as the 50 black images. Then the average of the black images is subtracted from the average of the luminescence images. On the resulting image obtained, the average intensity of each spot is taken, and the average value of the background noise around the spot is subtracted from this. The value obtained represents the luminescence signal of the spot.<br>
In  fact,  the  dynamic  range  of  the  sensor<br>
(estimate:; theoretically at 320)  is not sufficient to<br>
cover  the  dynamic  range  of  signals  generated  by<br>
biologics I  samples.  It  was  therefore  necessary  to<br>
perform two acquisitions with different settings for gain or exposure. The conditions selected (gain 0/exposure 560 ms, and gain 16/exposure 560 ms) make it possible to oDtain a non-saturated image for the strong signals dnd an image of the weak signals: photographs in Fig. 3, on the left gain 0 and on the right gain 16. In the image on the right, we can clearly distinguish, even before processing the image, four horizontal lines of spots, from top to bottom: 100%, 10%, 1%, 0.1% reference followed by a line of spots not containing any probe reference molecule, but only the inert (non­specific) molecule, which is not visible.<br>
The grey levels (GL) of the spots on the processed images were quantified, then normalized, and the values obtained are shown on the graph in Fig. 4. In Fig. 4, Cl, C5, C6 and C7 (C for chip) represent four independent experiments on four different chips according to the present invention. The corrected signal of  (in GL) is shown on the ordinate, and the percentage of probe molecule captured (% SCS) is shown on trie abscissa.<br>
It can be seen from this graph that detection by enzyme-coupled hybridization of reference targets makes it possible to distinguish the surface concentrations of each spot of the reference range (here, capture reference probes specific to the reference target) over a useful range of 1000 (from 0.1'. to 100%).<br>
This system simulates variation of the surface concentration of spots of captured reference targets, therefore possible to detect targets with a range<br>
of  concentrations,  i.e.  having  a  dynamic  range  of detection, of the order of 1000.<br>
In this example, carried out for purposes of demonstration, the chip was hybridized with an oligonucieotide directly coupled to the HRP enzyme. In real applications, there will certainly be other spots on the chip than just the reference spots. Therefore the chip will be hybridized for example with the sample previously labelled with a hapten, for example biotin. During hybridization, an oligonucieotide complementary to trie reference oligonucieotide, and labelled with biotin, will be added to the sample. After hybridization, a streptavidin-HRP conjugate, which will attach itself to the biotins present on the chip, will be incubated. Then development by luminescence will be performed, as described above. This "two-time" method was used in the following example, but with particles as markers, rather than enzymes.<br>
Example 2: direct labelling of the target reference molecule with fluorescent particles<br>
In this example, the target reference molecule (TRM) hybridized on the chip (C) is labelled with a hapten, oiotin (b), in this case by means of a detection ODN (ODNd) . Then a "staining" stage takes place, incubating the chip surface' with fluorescent particles (Pf) functionalized with streptavidin (St), which wiJ1 therefore be immobilized on the biotins (b) . The sequences are given below. Fig. 7 shows schematically the biochemical mechanism used in this method, on the surface (S) of the chip: a capture ODN<br>
is irnmob Llized on the substrate forming the chip and plays the role of the probe reference oligonucleotide<br>
(PRM) of the invention; the combination target + detection ODN plays the role of the complementary<br>
(target) oligonucleotide (TRM). The capture ODN (probe) is made up of 70 nucleotides. The target is made up of 513 nucleotides. The detection ODN is made up of 16 nucleotides. The sequences are as follows:<br>
Sequence of the probe oligonucleotide<br>
(SEQ ID No.: 3):<br>
5' TCACTATTAT CTTGTATTAC TACTGCCCCT TCACCTTTCC AGAGGAGC<br>
TT TGGTGGTCCT TTCCAAAGTG 3'<br>
Sequence of the inert oligonucleotide<br>
-'W -1- &gt;-&gt;<br>
5' ACTGTTACTG ACCTACCATT TGTTACCTAT GCTAAGCTCA TTGCACCT CT GATTGCGGAG GCCTTTCTTT 3'<br>
Sequence of the target oligonucleotide<br>
5' ACAGCAGTAC AAATGGCAGT ATTCATCCAC	AATTTTAAAA GAAAAGGG<br>
GG GATTGGGGGG TACAGTGCAG GGGAAAGAAT	AGTAGACATA ATAGCAAC<br>
AG ACATAGAAAC TAAAGAATTA CAAAAACCCT	TACAAAAATT CAAAATTT<br>
TC GGGTTTATTA CAGGGACAGC AGAAATCCAC	TTTGGAAAGG ACCAGCAA<br>
AG GTGCTGTGGA AAGGTGAAGG GGCAGTAGTA	ATACAAGATA ATAGTGAG<br>
AT AAAAGTAGTG CCAAGAAGAA AAGCAAAGAT	CATTAGGGAT TATGGAAA<br>
AC AGATGGCAGG TGATGATTGT GTGGCAAGTA	GACAGGATGA GATTAGAA<br>
CA TGGAAAAGTT TAGTAAAACA CCATATGTAT	GTTTCAGGGA AAGCTAGG<br>
GG TAGGTTTTAT AGACATCACT ATGAAAGCCC	TCATCCAAGA ATAAGTTC<br>
AG AAGTAAATCG AATTCCCGCG GCCATGGCGG	CCGGGAGCAT GCGACGTC<br>
GG GCGGAArnTCG CCC 3'<br>
Sequence  of  the  detection  oligonucleotide (ODNd.! (SEQ ID No. : 6) :<br>
TTCTGAACTTATTCTT 3'<br>
The efficiency of this staining stage is still poorly understood, and varies depending on the experimental conditions, the surface chemistry used for fabricating the DNA chip, the fluorescent particles used, etc. Thus, although the particles can be counted individually, it is nevertheless necessary, in' the methods of the prior art, to calibrate the measurements to be able to compare the experiments with one another.<br>
.Hence the benefit of the present invention for this type of labelling as well, as it makes it possible, advantageously, to dispense with this calibration.<br>
The substrate forming the chip in this example is a microscope slide, 25 x 75 mm. For all the operations, the microscope slide is immersed in a small beaker containing the required solution. The operations are a;: follows :<br>
S:ages (i) to (iv) of paragraph 1.3.1 of example 1 above are applied to this substrate. Then:<br>
il) Immobilization of the aforementioned probe and inert: oligonucleotides: Manual deposition, by itu, of aminated oligonucleotides at 10 uM, as a 3 iii droplet. The size of the spots is approx. 3 mm. Incabatior. in a humid chamber overnight. at room temper a .<br>
Preparation of the solutions of Gl;,aonucieotides to be deposited: the 10 uM mixture of capture ODN deposited contains a mixture of reference and non-specific ODN in proportions 100% reference, 1%<br>
reference, 0.2% reference, 0.1% reference, 0.033% and 100% inert (non-specific) molecule (i.e. a reference range produced with 6 points). These 6 solutions were deposited manually.<br>
;'2} Post-immobilization treatment. Reduction of the aldehydes still present: water 30 ml / NaBH4 105 mg; incubation 1 h at room temperature, without agitation. Rinsing with water 5 min, then with 5% sodium dodecylsuifate 5 min, then again in pure water 5 min. Drying.<br>
i 3) Hybridization of the chip and detection: the chips were hybridized with the aforementioned target molecule of 513 bases (SEQ ID No.: 5) at 100 nM for 30 min at room temperature in TE NaCl 1 M Triton 0.05%. Then these chips were hybridized with a 200 nM solution of the detection oligonucleotide at room temperature for 1 hour, then washed. This oligonucleotide is labelled with biotin. Next, the slide was incubated with neutravidin fluorescent particles of 100 nm diameter (Molecular Probes, T8861).<br>
Hybridization: all the hybridizations were carried out at room temperature in TE IX, NaCl 1 M, Triton X-100 0.05%. No agitation. Hybridization was followed by 3 washings in TE IX, NaCl 1 M, Triton X-100 0.05%, then one washing in TE IX, NaCl 1 M. TE IX means: Tris 10 mM, EDTA 1 mM, pH 8.<br>
:4) Development and reading: the fluorescent particles were incubated in TE NaCl 1 M, at a concentration of 10° particles per ul for 3 h, at room temperature. The particles have a diameter of 100 nanometres.<br>
5) Extraction of the numerical results: each imaged in the fluorescence microscope n cube). Then the particle density on each spot is counted, either manually, or automatically by means of a matlab routine (registered trademark). The results are consistent between these two methods.<br>
A usable reference range is certainly obtained. It is shown as the graph in Fig. 5, with the number of particles  per  10000 um~  on  the  ordinate,  and  P (I PRM/(IM+PRM)) on the aoscissa.<br>
When using, instead of the marker mentioned previously, Quantum Dot (registered trademark) fluorescent particles from the company Quantum Dot Corp., we can expect to obtain a far better labelling efficiency than that obtained here with fluorescent latices .<br>
Example 3: analysis of a sample<br>
We propose, as an example, a simple chip dedicated to initial screening of patients presenting an infection of the upper respiratory tract. The chip is intended more precisely for detecting the presence of the viruses RSV A and RSV B, responsible for bronchiol.ltis, and the presence of the viruses Influenza A (IrifA) and Influenza B (Inf B) , responsible for influenza.<br>
In all cases, they are very polymorphic RNA viruses; '".he symptoms are relatively similar, and these viruses can be dangerous in infants, in the elderly, and in ali immunodepressed patients in general.<br>
The chip used will be of exactly the same technology as in example 1, with functionalization chemistry identical to that of example 2.<br>
The spots (analysis spots as well as reference range spots) have a diameter of approx. 160 urn, and are arranged with spacing of 300 urn. This chip according to the present invention is shown schematically in Fig. 9. The spots are shown as being circular.<br>
For RSV A, RSV B, Inf A and Inf B, as well as for the "stain" control and development (T &amp; R), it is the same spot repeated five times.<br>
The molecule on the spots for "stain" control ana development (T &amp; R) is an oligonucleotide identical to the TRM molecules, already labelled, during production of the oligonucleotides, with biotin.<br>
Tne positive control (C +) is a reference range in ;:he sense of the invention, in the proportions 100%, 10cc, 1%, 0.1%, 0% (from left to right).<br>
Tne chip will be used in the following way: a)      the  biologist  purifies  the  RNAs  of  the samples according to the methods of the prior art.<br>
bi the oiological sample is mixed with the target reference molecule labelled with biotin during production.<br>
of the sample is labelled with biotin, according to one of the variants of the LDC method which was the object of patent applications filed by BioMerieux in the families of patents corresponding or documents [13], [14], [15] or [16] in the bibliography.<br>
di the assembly on the chip is incubated (hybridization stage), followed by washings, as in example 1.<br>
e; the streptavidin/HRP enzyme conjugate is added, as in example 1. In this stage, the conjugate attaches itself to the analytes present on the analysis spots, as well as to the TRM molecules present on the spots of the reference range, as well as to the molecules of the spots for "stain" control and development.<br>
f' development by luminescence is carried out as in example 1.<br>
The sequences of the TRM,  inert and PRM )ligonucleotides are identical to that of example 1.<br>
Bibliography<br>
[I]	Francois GELI,  "Puces a ADN et autres systemes<br>
d' analyse",  E.iofutur  n°206,  Le  technoscope  de<br>
Biofutur, pages 1-14, decembre 2000.<br>
[2] Cahill DJ, Nordhoff E., "protein arrays and their role in proteomics", Adv. Biochem. Eng. Biotechnol., 2003, 83, 177-187.<br>
[3] Lopez MF, Pluskal MG, "Protein micro- and macroarrays : digitizing the proteome", J. Chromatogr. B Analyt. Technol. Biomed. Life Sci., 2003, 787(1), 19-27.<br>
[4] EP-A-1 119 769. [5] FR-A-2 784 189. [6] WO-A-03/023061.<br>
[7] Bar-coding biomolecules with fluorescent nanocrystals, Nature Biotechnology, 2001, 19, 621-622.<br><br>
 [9]<br>
[10] Bertrand Jordan, "voyage au pays des puces", medecine/sciences, 1998, 14, 1097-1102.<br>
[II]	Troesch  A.,  et  al.,   "Mycobacterium  species<br>
identification and  rifampin  resistance  testing<br>
with  high-density  DNA probe  array",  J.  Clin.<br>
Mic.robiol., 1999, 37, 49-55.<br>
[12] ::hrjsten R. et Mabilat C., "Applications des puces a ADN en bacteriologie", Bull. Soc. Fr. Microbiol., 1998, 13, 10-17.<br>
[13] "Precede de marquage d'un acide ribonucieique et fragments d'ARN marques ainsi obtenus", FP 98 07870 et ses extensions dans les autres pays (fammille de brevets). Inventeur : A. Laayoun.<br>
[14] "Process for labelling a nucleic acid", EP-A-1 238 116. Inventeurs : A. Banerjee, A. Laayoun, M. Becker, K. Browne, M. Friedenberg, F. Hajjar.<br>
[15]  "Process	for  labelling  a  nucleic  acid  and<br>
labelled	RNA  fragments  which  are  obtained<br>
thereby",	WO-A-01/44506. Inventeurs : A. Laayoun,<br>
D.	Do, C.	Miyada.<br>
[16]  "Procede de marquage et de fragmentation	d'ADN",<br>
FP.    ri°01 06039.    Inventeurs :    J.	Lhomme,<br>
E.	rrevisiol, A. Laayoun, C. Bourget, K.	Kotera,<br>
C. Tora.<br><br><br><br><br><br><br>
We Claim;<br>
1. Analysis chip of at least one analyte present in a liquid sample, wherein the analyte, the target reference molecule and the probe reference molecule, identical or different, are all or part of a microorganism, a bacterium, a fungus, a virus, a eukaryotic cell, a cell, a chemical compound or molecule, a peptide, a protein, ac glycoprotein, a lipoprotein, an enzyme, a polysaccharide, a lipid, a glycolipid, a lipopolysaccharide, a ribonucleic acid (RNA), a deoxyribonucleic acid (DNA), a hormone, an antigen, an antibody, a growth factor and a hapten and wherein said chip comprising:<br>
(a)	at least one analysis spot of said, at least one, analyte, said analysis spot being arranged on the chip in such a way as to permit the recognition and immobilization specific to the analyte; and<br>
(b)	a reference range (G), said reference range comprising several reference spots each arranged on said chip in a defined manner and independently of one another, each reference spot of this range comprising, immobilized on its surface and in a defined proportion P different for each spot relative to the other reference spots of said range: (i) at least one probe reference molecule (PRM) for recognizing and specifically immobilizing a defined target reference molecule (TRM), and (ii) at least one inert molecule (IM) incapable of recognizing and attaching to said target reference molecule, the probe reference molecule and the inert molecule both being unable to recognize and immobilize said analyte or said analytes; with<br>
(Formula Removed)<br>
the number of PRMs + number of IMs being constant from one reference spot to another.<br>
2.	The chip as claimed in Claim 1, wherein the analyte and the target reference molecule are of the same nature.<br>
3.	The chip as claimed in claim 2, wherein the analyte and the target reference molecule are oligonucleotides or antibodies.<br>
4.	The chip as claimed in claim 1, wherein the probe reference molecule, the target reference molecule and the inert molecule are oligonucleotides.<br>
5.	The chip as claimed in claims 1 or 4, wherein the, at least one, analyte is a nucleic acid, and in which the, at least one, analysis spot is a<br>
spot functionalized by a nucleic acid complementary to the latter.<br>
6.	The chip as claimed in claim   1, wherein the spots of the reference range are disposed on the chip linearly and are arranged as a function of the proportion P of each spot.<br>
7.	Method of analysis in vitro of at least one analyte, that may be present in a liquid sample comprising the following stages:<br>
(a) optionally, immobilization of a first detecting means on the analyte;<br>
(P) addition, to the sample to be analysed, comprising the optionally labelled analyte, of a target reference molecule (TRM) on which a second detecting means, identical to or different from the first detecting means, has optionally been immobilized, said TRM being capable of recognizing and of binding specifically to the probe reference molecule (PRM) of an analysis chip as claimed in claim 1, said TRM being added to said sample in a sufficient amount to saturate the PRM of the reference range (G) of said chip, thus creating a reference range that is a function of the defined proportion P which is different for each reference spot;<br>
() bringing the sample to be analysed comprising the analyte and the TRM, optionally labelled, into contact with said analysis chip as claimed in claim 1 comprising at least one analysis spot of said at least one analyte in physicochemical conditions such that: on the one hand, the analyte to be analysed, if it is present, binds to its analysis spot on the chip; and on the other hand, the TRM specifically recognizes the PRM and binds to the latter on the various reference spots of the reference range of the chip;<br>
(δ) determination of a reference signal emitted, if applicable owing to the second detecting means, by each reference spot of the reference range, said signal being a function of the amount of target reference molecule immobilized on the latter; and<br>
(Σ) determination of an analysis signal emitted, if applicable owing to the first detecting means, by said, at least one, analysis spot and being a function of the amount of analyte immobilized by the analysis spot, and comparison with the signals determined in stage (8) for expressing this analysis signal as a function of P.<br>
8.	Method as claimed in claim 7, wherein the analyte and the target reference molecule are of the same nature.<br>
9.	The method as claimed in claim 8, wherein the analyte and the target reference molecule are oligonucleotides or antibodies.<br>
10.	The method as claimed in claim 7, wherein the probe reference molecule, the target reference molecule and the inert molecule are oligonucleotides.<br>
11.	The method as claimed in claims 7 or 10, wherein the, at least one,<br>
analyte is a nucleic acid, and in which the, at least one, analysis spot is a<br>
spot functionalized by a nucleic acid complementary to this nucleic acid.<br>
12.	The method as claimed in claim 7 or 11, wherein the first and the second detecting means involve respectively a first marker and a second marker.<br>
13.	The method as claimed in claim 12, wherein the first and the second markers are selected independently from the group comprising fluorescent markers, radioactive markers, enzymatic markers, latex particles, photonic crystals and colloidal gold.<br>
14.	The method as claimed in claim 7, wherein the determination of the signal of each spot of the reference range and/or of each analysis spot is performed by a method selected from the group comprising methods of detection by surface plasmon resonance, methods of photothermal detection, ellipsometric techniques and methods of photometric detection.<br>
15.	The method as claimed in claim 7, wherein the analysis is detection or a quantitative analysis.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUFic3RyYWN0LSgwMy0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Abstract-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUNsYWltcy0oMDMtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Claims-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUNsYWltcy0oMjUtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-Claims-(25-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUNvcnJlc3BvZGVuY2UgT3RoZXJzLSgwMy0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Correspodence Others-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-Correspondence Others-(25-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTMtMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-Correspondence-Others-(13-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LURyYXdpbmdzLSgwMy0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Drawings-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUZvcm0tMS0oMDMtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Form-1-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUZvcm0tMS0oMTMtMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-Form-1-(13-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUZvcm0tMTMgKDI4LTA2LTIwMDYpLnBkZg==" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Form-13 (28-06-2006).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMTMtKDEzLTA1LTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-13-(13-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LUZvcm0tMy0oMDMtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Form-3-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUZvcm0tMy0oMjUtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-Form-3-(25-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUdQQS0oMTMtMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-GPA-(13-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1ERUxOUC0yMDA2LUdQQS0oMjUtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-DELNP-2006-GPA-(25-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU5MS1kZWxucC0yMDA2LVBldGl0aW9uLTEzNy0oMDMtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3591-delnp-2006-Petition-137-(03-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="252652-a-dna-fragment-encoding-an-enzyme-having-polyphosphate-driven-adp-phosphorylation-activity.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252654-ammoniated-glycyrrhizin-modified-sweetened-beverage-products.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252653</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3591/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Jun-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-May-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>21-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>BIOMERIEUX SA</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>CHEMIN DE 1&#x27;ORME, F-69280 MARCY L&#x27;ETOILE, FRANCE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NOVELLI-ROUSSEAU</td>
											<td>25 RUE, RAMPEAU, F-38180 SEYSSINS, FRANCE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MALLARD FREDERIC</td>
											<td>63 RUE LAMBOURNAY, F-38340, VOREPPE, FRANCE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GINOT FREDERIC</td>
											<td>32, RUE CASIMIR BRENIER, F-38120 SAINT EGREVE, FRANCE.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RICOUL FLORENCE</td>
											<td>LIEU-DIT CHARRIERE, F-38950 QUAIX-EN-CHARTREUSE, FRANCE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>G06F</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/FR2004/050757</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-12-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0351222</td>
									<td>2003-12-29</td>
								    <td>France</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252653-analysis-chip-with-reference-scale-kits-and-analytical-methods by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:06 GMT -->
</html>
