<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252620-expression-of-mammalian-proteins-in-pseudomonas-fluorescenes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252620:&quot;EXPRESSION OF MAMMALIAN PROTEINS IN PSEUDOMONAS FLUORESCENES&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;EXPRESSION OF MAMMALIAN PROTEINS IN PSEUDOMONAS FLUORESCENES&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for producing a recombinant mammalian protein in a Pseudomonas fluorescens host cell comprising: transforming a Pseudomonas fluorescens host cell with a nucleic acid encoding a recombinant mammalian protein; growing the cell under conditions that allow expression of the recombinant mammalian protein; and isolating the recombinant mammalian protein; wherein the recombinant mammalian protein is produced as 5-75% total cell protein, wherein the recombinant mammalian protein is present in the host cell in a soluble form.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>EXPRESSION OF MAMMALIAN PROTEINS IN PSEUDOMONAS FLUORESCENS<br>
REFERENCE TO PRIOR APPLICATIONS<br>
This application claims priority to U.S. Provisional Application Nos. 60/564,798,<br>
entitled "Expression of Mammalian Proteins in Pseudomonas fluorescens," filed April 22,<br>
2004, and 60/537,148, entitled "Protein Expression Systems," filed January 16,2004.<br>
FIELD OF THE INVENTION<br>
The invention is a process for improved production of a recombinant mammalian<br>
protein by expression in a Pseudomonad, particularly in a Pseudomonas fluorescens<br>
organism. The process improves production of mammalian protein expression over known<br>
expression systems.<br>
BACKGROUND OF THE INVENTION<br>
More than 325 million people worldwide have been helped by the more than 155<br>
biotechnology drugs and vaccines approved by the U.S. Food and Drug Administration<br>
(FDA). In addition, there are more than 370 biotech drug products and vaccines currently in<br>
clinical trials targeting more than 200 diseases, including various cancers, Alzheimer's<br>
disease, heart disease, diabetes, multiple sclerosis, AIDS and arthritis. Unlike traditional<br>
small molecule therapeutics that are produced through classical chemical synthesis, proteins<br>
are ususally produced in living cells inefficiently and at high cost. Due to the high cost and<br>
complexity, there is a shortage of manufacturing capacity for protein-based therapeutics.<br>
The use of microbial cells to produce products has a very long history. As early as<br>
1897, Buchner discovered that enzymes extracted from yeast are effective in converting sugar<br>
into alcohol, leading to the production of key industrial chemicals using microorganisms. By<br>
the 1940s, large-scale production of penicillin via fermentation was achieved. Techniques for<br>
the insertion of foreign genes into bacteria were first developed in the early 1970s. Bacterial<br>
production of commercially viable recombinant mammalian protein was first exploited in the<br>
production of human insulin (Goeddel, et al., 1979a; Wong, 1997). Today fermentation and<br>
cell culture underlie the bulk of the industry's production of alcohol, antibiotics, biochemicals<br>
and therapeutic proteins. However, development and manufacturing of therapeutically useful<br>
proteins has been hampered due, in large part, to the limitations of the current organisms used<br>
to express these exogenous proteins.<br>
Prokaryotic vs. Eukaryotic Protein Expression<br>
Although bacterial expression system are often used to produce recombinant<br>
eukaryotic proteins, typically the proteins yielded differ significantly from their original<br>
counterparts. In general, it is a challenge to reproduce the eukaryotic secondary and tertiary<br>
structures in E. coli expression systems. At the same time, while the eukaryotic expression<br>
systems currently are better able to form the secondary and tertiary structures of recombinant<br>
eukaryotic proteins, the capacity of these systems to produce recombinant proteins in large<br>
quantity is limited.<br>
Post-translational modifications represent the most significant differences between<br>
prokaryotic and eukaryotic protein expression. Prokaryotes (i.e., bacteria) have a very simply<br>
cellular structure and no membrane-bound organelles. In eukaryotes, a protein is often<br>
modified after it is intially produced. These modifications, in many cases, are necessary to<br>
convert the peptide into a functional form. Thus, even when exisiting bacterial expression<br>
systems produce a protein with the correct primary structure, the protein may not be posttranslationally<br>
modified and is therefore often nonfunctional. Common modifications include<br>
disulfide bond formation, glycosylation, acetylation, acylation, phosphorylation, and gammacarboxylation,<br>
all of which can regulate protein folding and biological activity. Bacterial<br>
expression systems generally do not properly glycosylate, acetylate, acylate, phosphorylate, or<br>
gamma-carboxylate eukaryotic proteins.<br>
Bacteria, such as E. coli, can form disulfide bonds, but the bonds are often formed in<br>
the incorrect configuration required for biological activity; therefore, denaturation and<br>
refolding is usually required to produce active eukaryotic proteins. Molecular chaperone<br>
proteins are present in both prokaryotes and eukaryotes that facilitate the folding of other<br>
proteins. In the absence of such chaperones, unfolded or partially folded polypeptide chains<br>
are unstable within the cell, frequently folding incorrectly or aggregating into insoluble<br>
complexes. The binding of chaperones stabilizes these unfolded polypeptides, thereby<br>
preventing incorrect folding or aggregation and allowing the polypeptide chain to fold into its<br>
correct conformation. However, chaperones differ in each type of cell, and can be<br>
differentially expressed based on extracellular conditions.<br>
Problems With Current Expression Systems<br>
Escherichia coli (E. coif) is the most widely and routinely used protein expression<br>
system. Production in E. coli is inexpensive, fast, and well characterized. Further, scale-up<br>
and harvesting is possible and cGMP production is well established. However, there are<br>
significant limitations to the use of E. coli, which often prove difficult to overcome,<br>
particularly when expressing reconibinant mammalian proteins.<br>
Along with the limitations described above, the high-level expression of recombinant<br>
gene products in E. coli often results in the misfolding of the protein of interest and its<br>
subsequent degradation by cellular proteases or deposition into biologically inactive<br>
aggregates known as inclusion bodies. Protein found in inclusion bodies typically must be<br>
extracted and renautred for activity, adding time and expense to the process. Typical<br>
renaturation methods involve attempts to dissolve the aggregate in concentrated denaturant,<br>
and subsequent removal of the denaturant by dilution. Some of the factors which have been<br>
suggested to be involved in inclusion body formation include the high local concentration of<br>
protein; a reducing environment in the cytoplasm (E. coli cytoplasm has a high level of<br>
glutathione) preventing formation of disulfide bonds; lack of post-translational modifications,<br>
which can increase the protein solubility; improper interactions with chaperones and other<br>
enzymes involved in in vivo folding; intermolecular cross-linking via disulfide or other<br>
covalent bonds; and increased aggregation of folding intermediates due to their limited<br>
solubility. It is probably a combination of these factors, as well as a limited availability of<br>
chaperones, which most commonly lead to the formation of inclusion bodies.<br>
Yeast expression systems, such as Saccharomyces cerevisiae or Pichia pastoris, are<br>
also commonly used to produce proteins. These systems are well characterized, provide good<br>
expression levels and are relatively fast and inexpensive compared to other eukaryotic<br>
ecpression systems. However, yeast can accomplish only limited post-translational protein<br>
modifications, the protein may need refolding, and harvesting of the protein can be a problem<br>
due to the characteristics of the cell wall.<br>
Insect cell expression systems have also emerged as an attractive, but expensive,<br>
alternative as a protein expression system. Correctly folded proteins that are generally posttranslationally<br>
modified can sometimes be produced and extracellular expression has been<br>
achieved. However, it is not as rapid as bacteria and yeast, and scale-up is generally<br>
challenging.<br>
Mammalian cell expression systems, such as Chinese hamster ovary cells, are often<br>
used for complex protein expression. This system usually produces correctly folded proteins<br>
with the appropriate post-translational modifications and the proteins can be expressed<br>
extracellularly. However, the system is very expensive, scale-up is slow and often not<br>
feasible, and protein yields are lower than in any other system.<br>
Pseudomonas fluorescens (P. fluorescens)<br>
Pseudomonas fluorescens encompasses a group of common, nonpathogenic<br>
saprophytes that colonize soil, water and plant surface environments. P. fluorescens are<br>
extensively used in agricultural and industrial processes, including commercially for the<br>
production of non-mammalian industrial and agricultural proteins. Nonmammalian enzymes<br>
derived from P. fluorescens have been used to reduce environmental contamination, as<br>
detergent additives, and for stereoselective hydrolysis. Mycogen began expressing<br>
recombinant bacterial proteins in P. fluorescens in the mid-1980's and filed its first patent<br>
application on the expression of the Bacillus thuringiensis toxin in P. fluorescens on January<br>
22, 1985 ("Cellular encapsulation of biological pesticides"). Between 1985 and 2004,<br>
Mycogen, later Dow Agro Sciences, as well as other companies, capitalized on the<br>
agricultural use of P. fluorescens in patent applications on the production of pesticidal,<br>
insecticidal, and nematocidal toxins, as well as on specific toxic sequences and genetic<br>
manipulation to enhance expression of these. Examples of patent applications directed to the<br>
expression of recombinant bacterial proteins in P. fluorescens include: U.S. Patent Nos.<br>
3,844,893; 3,878,093, 4,169,010; 5,292,507; 5,558,862; 5,559,015; 5,610,044; 5,622,846;<br>
5,643,774; 5,662,898; 5,677,127; 5,686,282; 3,844,893; 3,878,093; 4,169,010; 5,232,840;<br>
5,292,507; 5,558,862; 5,559,015; 5,610,044; 5,622,846; 5,643,774; 5,662,898; 5,677,127;<br>
5,686,282; 5,686,283; 5,698,425; 5,710,031; 5,728,574; 5,731,280; 5,741,663; 5,756,087;<br>
5,766,926; 5,824,472; 5,869,038; 5,891,688; 5,952,208; 5,955,348; 6,051,383; 6,117,670;<br>
6,184,440; 6,194,194; 6,268,549; 6,277,625; 6,329,172; 6,447,770; as well as PCT<br>
Publication Nos. WO 00/15761; WO 00/29604; WO 01/27258; WO 02/068660; WO<br>
02/14551; WO 02/16940; WO 03/089455; WO 04/006657; WO 04/011628; WO 87/05937;<br>
WO 87/05938; WO 95/03395; WO 98/24919; WO 99/09834; and WO 99/53035.<br>
On October 8, 2003, Dow AgroSciences filed PCT Publication No. 04/087864<br>
entitled, "Amended Recombinant Cells (ARCs) for the Production and Delivery of Antiviral<br>
Agents, Adjuvants and Vaccine Accelerants". The application describes recombinant cells<br>
that can include at least one heterologous gene encoding a chemokine or a cytokine and the<br>
administration of such cells to a host to accelerate an immune response. The application<br>
demonstrates the production of bovine interferon-a and interferon-'y in P.fluorescens.<br>
Dow Global Technologies currently has several pending patent applications in the area<br>
of use of P. fluorescens to produce recombinant proteins. PCT Application WO 03/068926 to<br>
Dow Global Technologies, filed February 13, 2003, entitled, "Over-Expression of<br>
Extremozyme Genes in Pseudomonas and Closely Related Bacteria" describes an expression<br>
system in which pseudomonads, specifically P. fluorescens, can be used as host cells for the<br>
production of extremozyme enzymes. These enzymes are typically ancient, found in<br>
prokaryotes, eukaryotes including fungi, yeast, lichen, protists and protozoa, algae and<br>
mosses, tardigrades and fish. The patent discloses that enzymes can be derived from certain<br>
extremophilic fungi and yeast, but are typically derived from extremophilic bacteria.<br>
PCT publication No. WO 03/089455 to Dow Global Technologies, filed April 22,<br>
2003, entitled "Low-Cost Production of Peptides" describes a method of producing small<br>
peptides, primarily antimicrobial peptides, as concatameric precursors in Pseudomonads,<br>
specifically P. fluorescens.<br>
PCT publication No. WO 04/005221 to Dow Global Technologies, entitled "Benzoate<br>
and Antranilate Inducible Promoters" provides novel benzoate- or anthranilate-inducible<br>
promoters from P. fluorescens, as well as novel tandem promoters, variants and improved<br>
mutants thereof, that are useful for commercial prokaryotic fermentation systems.<br>
U.S. Patent No. 5,232,840 to Monsanto Co. describes the use of novel ribosomal<br>
binding sites to enhance expression of certain proteins in prokaryotic cells. In one example,<br>
the cells are used to express porcine growth hormone in several organisms, including E. coli,<br>
P. fluorescens, and P. putida. The data shows that P. fluorescens is less efficient at<br>
expressing the growth hormone when compared to E. coli. In contrast, when expressing a<br>
bacterial protein, P. fluorescens is much more effective at protein production than E. coli<br>
under comparable conditions. In fact, P. fluorescens cells described in this patent produce<br>
several-fold more bacterially-derived p-galactosidase than E. coli (compare table 4 to tables 1<br>
and 2).<br>
While progress has been made in the production of proteins of commercial interest, a<br>
strong need remains to improve the capability and production level of recombinant<br>
mammalian, and in particular human, proteins.<br>
Therefore, it is an object of the present invention to provide a process for the<br>
production of recombinant mammalian, in particular human, proteins that can be isolated and<br>
purified for therapeutic use, and cells which can accomplish this process.<br>
It is a further object of the present invention to provide improved processes for the<br>
production of active recombinant mammalian proteins, including complex mammalian<br>
proteins.<br>
It is a further object of the present invention to provide improved processes for the<br>
production of high levels of recombinant mammalian, in particular human, proteins.<br>
It is a further object of the present invention to provide transformed organisms that<br>
provide high expression levels of soluble or insoluble recombinant mammalian proteins.<br>
SUMMARY OF THE INVENTION<br>
It has been discovered that Pseudomonas fluorescens is a superior organism for the<br>
production of recombinant proteins, and in particular recombinant mammalian proteins, such<br>
as recombinant human proteins. Based on these discoveries, the present invention provides a<br>
process of producing recombinant mammalian or mammalian-derived proteins in P.<br>
fluorescens. In addition, the invention provides P. fluorescens transformed to produce<br>
recombinant mammalian, including human, proteins.<br>
In one embodiment, the invention provides a process of producing a mammalian<br>
protein in a P. fluorescens organism in which the protein is produced at a higher level or<br>
concentration per cell or per liter of fermentation reaction than in an E. coli organism under<br>
comparable conditions. In yet another embodiment, the invention provides a process of<br>
producing mammalian proteins in an P. fluorescens organism in a batch culture which<br>
produces higher amounts of protein per liter than a corresponding batch of recombinant E.<br>
coli organisms.<br>
Comparable conditions or substantially comparable conditions particularly refers to<br>
expression of recombinant protein using the same operably linked transcriptional promoter<br>
and ribosomal binding site in different organisms, and using the same initial induction<br>
conditions. Comparable conditions can further include using the same vector and associated<br>
regulatory elements, including, but not limited to, enhancer sequences, termination sequences,<br>
and origin or replication sequences. Comparable conditions can also include the same total<br>
volume of cell fermentation reaction. Comparable conditions can also include the same<br>
concentration of total cells per liter of reaction. In one embodiment, the conditions also<br>
include total induction times (before measurement) that are similar or the same. However, in<br>
another embodiment, the induction times can vary depending on the organism. Specifically,<br>
P. fluorescens has a capacity for increased growth time over E. coli without reducing protein<br>
production, such that protein production can be measured in P. fluorescens at a time point at<br>
which E. coli cells are largely silent. One way to measure the comparable conditions is to<br>
compare the percentage of recombinant protein per total cell protein. The comparable<br>
conditions also do not require identical media for growth. The media can be adjusted to<br>
ensure optimal production for the individual organisms.<br>
In another embodiment, the invention provides a process for producing recombinant<br>
mammalian proteins by producing the proteins in a P. fluorescens organism and isolating the<br>
produced protein. In one sub-embodiment, the process includes substantially purifying the<br>
protein. In one embodiment, the protein is derived from a human protein, or is humanized.<br>
The invention also provides the use of P. fluorescens in at least the following<br>
embodiments:<br>
(i) the production of recombinant mammalian, including human, proteins present in the<br>
cell in a range of between 1 and 75 percent total cell protein (%tcp), or in particular,<br>
at least greater than approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20%<br>
tcpormore;<br>
(ii) the production of recombinant mammalian, including human, proteins that are<br>
soluble and present in the cytoplasm of the cell in a range of between 1 and 75 %tcp<br>
or in particular, at least greater than approximately 5% tcp, 10% tcp, at least 15%<br>
tcp, at least 20% tcp or more;<br>
(iii) the production of recombinant mammalian, including human, proteins that are<br>
insoluble in the cytoplasm of the cell, in a range of between 1 and 75 % tcp or in<br>
particular, at least greater than approximately 5% tcp, 10% tcp, at least 15% tcp, at<br>
least 20% tcp or more;<br>
(iv) the production of recombinant mammalian, including human, proteins that are<br>
soluble in the periplasm of the cell in a range of between 1 and 75 % tcp or in<br>
particular, at least greater than approximately 5% tcp, 10% tcp, at least 15% tcp, at<br>
least 20% tcp or more;<br>
(v) the production of recombinant mammalian, including human, proteins that are<br>
insoluble in the periplasm in a range of between 1 and 75 % tcp or in particular, at<br>
least greater than approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp<br>
or more;<br>
(vi) the production or recombinant mammalian, including human, proteins in the cell in a<br>
range of between 1 and 75 %tcp, or particularly at least greater than approximately<br>
5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more, when grown at a cell<br>
density of at least 40g/L;<br>
(vii) the production of recombinant mammalian, including human, proteins present in the<br>
cell in an active form;<br>
(viii) the production of multi-subunit recombinant mammalian, including human, proteins<br>
in active form;<br>
(ix) the production of recombinant mammalian, including human, proteins that are then<br>
isolated and purified; and<br>
(x) the production of recombinant mammalian, including human, proteins that are<br>
renatured.<br>
In one embodiment, the recombinant mammalian protein is selected from the group<br>
consisting of a multi-subunit protein, a blood carrier protein, an enzyme, a full length<br>
antibody, an antibody fragment, or a transcriptional factor.<br>
In another embodiment, the invention includes:<br>
(i) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins at a higher level or concentration than a<br>
corresponding E. coli organism when grown under substantially corresponding<br>
conditions;<br>
(ii) Pseudomonas fluorescens organisms that are transformed to produce recorribinant<br>
mammalian, including human, proteins and peptides that are present in the cell in a<br>
range of between 1 and 75% tcp or in particular, at least greater than approximately<br>
5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more;<br>
(iii) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins that are present in the cell in active form;<br>
(iv) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins that are soluble in the cytoplasm of the cell<br>
in a range of between 1 and 75 % tcp or in particular, at least greater than<br>
approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more;<br>
(v) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins that are insoluble in the cytoplasm of the cell<br>
in a range of between 1 and 75 % tcp or in particular, at least greater than<br>
approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more;<br>
(vi) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins that are soluble in the periplasm of the cell in<br>
a range of between 1 and 75 % tcp or in particular, at least greater than<br>
approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more;<br>
(vii) Pseudomonas fluorescens organisms that are transformed to produce recombinant<br>
mammalian, including human, proteins that are insoluble in the periplasm of the cell<br>
in a range of between 1 and 75 % tcp or, in particular, at least greater than<br>
approximately 5% tcp, 10% tcp, at least 15% tcp, at least 20% tcp or more;<br>
(viii) Pseudomonas fluorescens organisms that are transformed to produce multi-subunit<br>
recombinant mammalian, including human, proteins;<br>
(ix) Pseudomonas fluorescens organisms that are transformed to produce multi-subunit<br>
recombinant mammalian, including human, proteins present in the cell in active<br>
form.<br>
In an alternative embodiment, a Pseudomonas organisms and closely related bacteria<br>
other than fluorescens are used as host cells in this invention, as described in more detail<br>
below. Li one embodiment, the host cell will be selected generally from the genus<br>
Pseudomonas and specifically from a nonpathogenic Pseudomonas species. Likewise, any<br>
Pseudomonas fluorescens strain can be used that accomplishes the desired inventive goal,<br>
including but not limited to strain MB 101, or a strain that is modified to include at least one<br>
host-cell-expressible, inserted copy of at least one Lac represser protein-encoding lacl<br>
transgene, such as MB214 and MB217. The Pseudomonas organism can also optionally be<br>
genetically modified to add or delete one or more genes to improve performance, processing,<br>
or other characteristics.<br>
In one embodiment, the Pseudomonas organism is transformed with a nucleic acid<br>
encoding a recombinant mammalian protein selected from the group consisting of a multisubunit<br>
protein, a blood carrier protein, an enzyme, a full length antibody, an antibody<br>
fragment, or a transcriptional factor. In one embodiment, the P. fluorescens organism<br>
expresses a recombinant mammalian protein selected from the group consisting of a multisubunit<br>
protein, a blood carrier protein, an enzyme, a full length antibody, an antibody<br>
fragment, or a transcriptional factor.<br>
The expressed recombinant mammalian or human protein will typically have a mass<br>
of at least about IkD, and up to about 100, 200, 300, 400 or 500 kD, often between about<br>
lOkD and about lOOkD, and usually greater than about 30kD.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is a graph showing hu-y-IFN purified from the soluble fraction of P.<br>
fluorescens samples displays activity comparable to &amp; commercially available standard.<br>
Figure 2 is a picture of an ELISA showing the activity of purified Gal 13 in P.<br>
fluorescens and E. coli.<br>
Figure 3 represents human growth hormone expression constructs. The amino acid<br>
sequence of human growth hormone lacking its native secretion signal sequence is shown in<br>
A. Plasmid constructs for expression in P. fluorescens (pDOW2400) and E. coli (412-<br>
001 .hGH) are shown in B.<br>
Figure 4 is a picture of an SDS-PAGE analysis of soluble and insoluble fractions of<br>
hGH expressed in P. fluorescens and E. coli. The time post-induction is denoted by 10,124,<br>
148, 0 or 3. The large arrows indicate the position of the 21 kDa hGH protein.<br>
Figure 5 shows an SDS-PAGE analysis of the expression of y-IFN in E. coli versus<br>
P.fluoresens cells. Soluble (S) and insoluble (I) fractions of samples taken at 0, 3, 24 and 48<br>
hours post-induction (10, etc.) were resolved. E. coli expressed y-IFN is shown in panel A, P.<br>
fluorescens expressed y-IFN is shown in panel B. 5uL of A575-20 samples were loaded onto<br>
a 10% Bis-Tris NuPAGE gel and resolved in IX MES. Arrows indicate the position of the<br>
recombinant protein. Western analyses are shown in panels C (E. coli) and D (P.<br>
fluorescens).<br>
Figure 6 shows the replacement of the BuiBui toxin gene with the BGI gene at the<br>
Spel andXhol sites of pMYC1803.<br>
Figure 7 shows that all the transformants selected had the desired interferon insert, as<br>
verified by sequencing the inserted DNA.<br>
Figure 8 represents the nucleotide sequence for the phosphate binding protein-ga!2<br>
single chain antibody fusion protein.<br>
Figure 9 represents the amino acid sequence for the phosphate binding protein-ga!2<br>
single chain antibody fusion protein.<br>
Figure 10 represents the nucleotide sequence for the phosphate binding proteinhuman<br>
growth hormone fusion protein.<br>
Figure 11 represents the amino acid sequence for the phosphate binding proteinhuman<br>
growth hormone fusion protein.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Process for producing recombinant mammalian proteins<br>
The invention provides processes and transformed Pseudomonas fluorescens<br>
organisms that produce recombinant mammalian proteins.<br>
In one embodiment, the invention provides a process for producing recombinant<br>
mammalian proteins by producing the proteins in a P. fluorescens organism and isolating the<br>
produced protein. The protein can be isolated after expression by techniques known in the art,<br>
including, but not limited to, affinity chromatography, ion-exchange chromatography,<br>
antibody affinity, size-exclusion, or any other method that eliminates a substantial portion of<br>
the cellular debris from the protein. In one sub-embodiment, the process provides a<br>
substantially purified protein. The isolated protein can have activity similar to that of the<br>
native protein that it is derived from. The protein can be isolated in a correctly folded state or<br>
conformation, approximating that of the native protein, or can be further renatured or<br>
modified to put it into a correctly folded conformation. In one sub-embodiment, the protein is<br>
derived from a human protein, or is humanized. A "humanized" protein is typically a<br>
chimeric mammalian-type protein which is partially comprised of a human-derived protein<br>
sequence. Humanization is particularly useful in antibody production and the development of<br>
humanized antibodies has been extensively described, for example in U.S. Patent No.<br>
6,800,738.<br>
In one embodiment, expression of the protein by the host cell is followed by isolation<br>
of the protein. In another embodiment, the protein of peptide is purified. In an alternative<br>
embodiment, the protein is purified following isolation of the protein. Optionally the isolated<br>
or purified protein can be renatured or refolded in order to produce active proteins.<br>
In another embodiment, the invention provides a process of producing a mammalian<br>
protein in a P. fluorescens organism in which the protein is produced at a higher level or<br>
concentration than in an E. coli organism. The suitability of P. fluorescens organisms for<br>
high level production of mammalian proteins was unexpected based on the lack of success in<br>
producing such proteins in these organisms in the prior art. The present inventors have found<br>
that these organisms are indeed capable of high levels of production of mammalian proteins,<br>
and typically express protein in higher yield or at higher levels than E. coli when tested in<br>
corresponding assays. In another embodiment, the invention provides a process of producing<br>
mammalian proteins in an P. fluorescens organism in a batch culture which produces higher<br>
amounts of protein per liter than a corresponding batch of recombinant E. coli organisms.<br>
In some embodiments, processes are provided that include producing recombinant<br>
mammalian, including human, multi-subunit proteins in active form in P. fluorescens;<br>
producing recombinant mammalian blood carrier proteins, including human blood carrier<br>
proteins such as transferrin and albumin in P. fluorescens; producing recombinant mammalian<br>
enzymes, including recombinant mammalian enzymes in active form in P. fluorescens;<br>
producing antibodies and antibody fragments, including single chain antibodies and Fab<br>
fragments in P. fluorescens; and producing recombinant mammalian, including human,<br>
transcriptional factors in P. fluorescens.<br>
In one embodiment, the recombinant mammalian protein is produced as a multimer, or<br>
in a concatameric precursor, for example, in the form of at least two small peptide (1-15<br>
amino acids) units in tandem. In an alternative embodiment, the recombinant mammalian<br>
protein is not produced as a multimer, or in concatameric precursors, but instead is produced<br>
as a single chain polypeptide.<br>
Screening of biomolecules<br>
A separate embodiment of the present invention provides P. fluorescens organisms in<br>
a process of screening libraries of mammalian biomolecules to identify at least one that<br>
exhibits a desired activity or property. The P. fluorescens cells can be transformed with a<br>
number of mammalian derived nucleic acids for which testing is desired, producing a library<br>
of transformed host cells. Upon expression, polypeptides encoded by at least some of the<br>
nucleic acids are produced for testing either in cytoplasm or following recovery from the cell.<br>
Examples of activities and properties for which testing may be performed include:<br>
polypeptide expression level; polypeptide stability; and biocatalytic activities and properties.<br>
Illustrative examples of biocatalytic activities and properties include: enzymatic activities;<br>
protein interactions/binding; protein stabilization; substrate usage; product formation; reaction<br>
conditions, such as pH, salinity, or reaction temperature; biocatalytic parameters for a given<br>
catalyzed reaction, such as Km and Vmax; and stability behavior, such as thermostability and<br>
biocatalyst half-life. The test results obtained may be used to selected library member(s) for<br>
further development.<br>
Protein expression<br>
A key aspect of this invention is the expression of high levels of recombinant<br>
mammalian, for example human, proteins in a range of between 1 and 75 percent total cell<br>
protein (%tcp) by expression in P. fluorescens organisms. The expressed proteins can be<br>
soluble or insoluble while in the P. fluorescens cell. Such high levels of soluble or insoluble<br>
recombinant mammalian proteins can be an improvement over previously known mammalian<br>
protein expression systems. In particular, high levels of recovered mammalian proteins in<br>
large scale fermentation reactions are not typically possible with known techniques.<br>
In one embodiment, the invention provides expression levels of mammalian proteins<br>
that exceed those found in E. coli expression systems. In one embodiment, the concentration<br>
of recombinant protein in each cell is higher than that found in E. coli in comparative assays.<br>
In one embodiment, the level of recombinant protein as compared to total cell protein<br>
measured in the P. fluorescens expression system is higher than that of the same recombinant<br>
protein expressed in E. coli. In another embodiment, the level or amount of soluble protein in<br>
the P. fluorescens expression system described herein is higher than the level or amount of<br>
soluble recombinant protein in a comparable E. coli expression system. In another<br>
embodiment, the total amount of active protein is higher than the amount derived from an E.<br>
coli expression system. In a separate embodiment, the level of recombinant active protein as<br>
compared to total cell protein measured in the P. fluorescens expression system is higher than<br>
that of the same recombinant protein expressed in E. coli. In one embodiment, the level,<br>
concentration, or amount of protein expressed in P. fluorescens is at least 2x, at least 3x, at<br>
least 4x, at least 5x, at least 6x, at least 7x, at least 8x, at least 9x, at least lOx, or more the<br>
level, concentration, or amount of recombinant protein expressed in E. coli in comparable<br>
assays.<br>
One of the benefits of P. fluorescens as an expression system is that the cells can be<br>
grown in large scale cultures without negatively impacting their capacity for protein<br>
production. This capacity exceeds that found in other bacterial systems, such as E. coli. In<br>
another embodiment, the process includes producing mammalian proteins in batch cultures in<br>
which the recombinant protein is produced at a higher total level in P. fluorescens than in E.<br>
coli batch cultures. In yet another embodiment, the invention provides a process of producing<br>
mammalian proteins in an P. fluorescens organism in a batch culture which produces higher<br>
amounts of protein per liter than a corresponding batch of recombinant E. coli organisms.<br>
The invention generally provides processes and transformed P. fluorescens organisms<br>
that afford expression levels of 1-75% total cell protein (tcp) of soluble or insoluble<br>
recombinant mammalian proteins. The recombinant mammalian proteins expressed in the<br>
cell can be expressed in an active form. In other embodiments, the P. fluorescens provides at<br>
least 1, 5, 10, 15, 20, 25, 30, 40, 50, 55, 60, 65, 70, or 75 % tcp of recombinant mammalian<br>
proteins.<br>
These proteins can be soluble, and when soluble, can be present in the cytoplasm or<br>
periplasm of the cell during production. Soluble proteins are readily released from the cell by<br>
methods including, but not limited to, rupturing of the cell membrane by pressure (i.e. the<br>
"French" press method), or by lysozyme degradation of the cell membrane. Cells can<br>
typically also be lysed using detergents, such as non-ionic detergents. Proteins that are<br>
soluble can be further stabilized by adjusting components of the buffer, such as buffer pH, salt<br>
concentrations, or additional protein components (for example, in multi-subunit complexes).<br>
The soluble proteins can be isolated or purified from other protein and cellular debris by, for<br>
example, centrifugation and/or chromatography such as size exclusion, anion or cation<br>
exchange, or affinity chromatography.<br>
The proteins can also be insoluble. Insoluble proteins are typically found in inclusion<br>
bodies in the cytoplasm, but are also often in the periplasm. Not all insoluble proteins are in<br>
inclusion bodies, and can also be found in membrane aggregates, as small protein aggregates<br>
or in any other insoluble form in the cytoplasm or periplasm. Insoluble proteins can typically<br>
be renatured using, for example, reducing agents such as urea or guanidine hydrochloride.<br>
Insoluble proteins or protein aggregates can be isolated, for example, by centrifugation and/or<br>
chromatography such as size exclusion chromatography. Proteins in insoluble aggregates can<br>
typically be separated by solubilization of the aggregates using, for example, micelles or<br>
reverse micelles as described in Vinogradov, et al. (2003) Anal Biochem. 15;320(2):234-8.<br>
In a particular embodiment, the Pseudomonas host cell can have a recombinant<br>
mammalian peptide, polypeptide, protein, or fragment thereof expression level of at least 1%<br>
tcp and a cell density of at least 40 g/L, when grown (i.e. within a temperature range of about<br>
4° C to about 55° C, inclusive) in a mineral salts medium. In a particular embodiment, the<br>
expression system will have a recombinant protein of peptide, including recombinant<br>
mammalian protein, expression level of at least 5% tcp and a cell density of at least 40 g/L,<br>
when grown (i.e. within a temperature range of about 4° C to about 55° C, inclusive) in a<br>
mineral salts medium at a fermentation scale of at least 10 Liters.<br>
Expression levels can be measured by standard techniques known in the art. In one<br>
embodiment, the amount of protein (in grams) is compared to the amount in grams of total<br>
cell protein in a given sample. In another embodiment, the measurement is a level of<br>
recombinant protein per liter. In another embodiment, the level or amount can be measured as<br>
compared to a known standard, such as a BSA control. The level or amount of recombinant<br>
protein can be measured, for example, by analyzing the light absorbtion of a purified protein,<br>
by measuring an affinity of an marker for the protein (such as an antibody affinity) and<br>
comparing that to a known standard, or by measuring the level of activity compared to a<br>
known standard (such as a known amount of purified, active protein).<br>
It has been found that, in certain situations, no additional disulfide-bond-promoting<br>
conditions or agents are required in order to recover disulfide-bond-containing target<br>
polypeptides in active, soluble form, when a Pseudomonas fluorescens bacteria is used as the<br>
expression host cell. Therefore, in one embodiment, the transgenic peptide, polypeptide,<br>
protein, or fragment contains at least one intramolecular disulfide bond in its native state. In<br>
other embodiments, the protein can contain up to 2, 4, 6, 8, 10, 12, 14, 16, 18, or 20 or more<br>
disulfide bonds in its native state.<br>
In some embodiments, the protein is expressed or found in the periplasm of the cell<br>
during production before purification or isolation. The protein can be secreted into the<br>
periplasm by being fused to an appropriate signal secretion sequence. In one embodiment, the<br>
signal sequence is a signal sequence that is native to the P. fluorescens genome. In specific<br>
embodiments, the signal sequence is a phosphate binding protein, a Lys-Arg-Orn binding<br>
protein (LAObp or KRObp) secretion signal peptide, an Outer Membrane Porin E (OpreE)<br>
secretion signal peptide, an azurin secretion signal peptide, an iron (III) binding protein<br>
(Fe(III)bp) secretion signal peptide, or a lipoprotein B (LprB) secretion signal peptide.<br>
In one embodiment, the recombinant peptide, polypeptide, protein, or fragment thereof<br>
has a folded intramolecular conformation in its active state. P. fluorescens typically produce<br>
mammalian proteins more efficiently in the correctly folded conformation. In one<br>
embodiment, more than 50% of the expressed, transgenic peptide, polypeptide, protein, or<br>
fragment thereof produced can be produced as single peptides, polypeptides, proteins, or<br>
fragments thereof in soluble, active form or insoluble, but renaturable form in the cytoplasm<br>
or periplasm. In another embodiment about 60% , 70%, 75%, 80%, 85%, 90%, 95% of the<br>
expressed protein is obtained in or can be renatured into active form.<br>
Definitions<br>
Throughout this specification, the term "protein" is used to include any amino acid<br>
concatamers or polymers. The terms "polypeptide," "peptide" and "protein" are used<br>
interchangeably and include amino acid polymers in which one or more amino acid residue is<br>
an artificial chemical analogue of a corresponding naturally occurring amino acid, as well as<br>
to naturally occurring amino acid polymers.<br>
The term "isolated" refers to nucleic acid, protein, or peptide that is substantially or<br>
essentially free from other material components which normally accompany it as found in its<br>
native state when in a cell, for example, other cellular components.<br>
The term "purified" or "substantially purified" is used to mean that the protein is<br>
separated from other cell components and is separated from other proteins and peptides found<br>
in the cell that are not in a native complex with the protein. In particular embodiments, the<br>
purified proteins are of a purity approved for therapeutic or veterinary used as defined by<br>
standard cGMP guidelines or approved by the FDA.<br>
The term "percent total cell protein" ("tcp") means the amount of protein in the host<br>
cell as a percentage of aggregate cellular protein. Alternatively, the term means a measure of<br>
the fraction of total cell protein that represents the relative amount of a given protein<br>
expressed by the cell.<br>
The term "operably attached" refers to any configuration in which the transcriptional<br>
and any translational regulatory elements are covalently attached to the encoding sequence in<br>
such disposition(s), relative to the coding sequence, that in and by action of the host cell, the<br>
regulatory elements can direct the expression of the coding sequence.<br>
As used herein, the term "mammal" is meant to include or designate any animal in the<br>
class Mammalia including human or non-human mammals, such as, but not limited, to<br>
porcine, ovine, bovine, rodents, ungulates, pigs, swine, sheep, lambs, goats, cattle, deer,<br>
mules, horses, monkeys, apes, dogs, cats, rats, and mice.<br>
As used herein, the term "recombinant mammalian protein" or peptide is meant to<br>
include proteins derived from a native mammalian protein sequence or derived or generated<br>
from a native mammalian nucleic acid sequence. Such recombinant proteins can be produced<br>
from nucleic acid sequence substantially corresponding to native mammalian mKNA or<br>
substantially corresponding cDNA, or fragements thereof. The sequence can be adjusted<br>
accordingly based on specific host cell codon usage as known in the art.<br>
The phrase "substantially corresponding" in the context of two nucleic acids or<br>
polypeptides refers to the residues in the two sequences that have at least 50%, 60%, 70%,<br>
80%, 90% or higher identity when aligned for maximum correspondence over a domain of the<br>
protein, as measured using an algorithms known in the art. Optimal alignment of sequences<br>
for comparison can be conducted, e.g., by algorithms known in the art (e.g. Smith &amp;<br>
Waterman (1981) Adv. Appl. Math. 2:482; Needleman &amp; Wunsch (1970) J. Mol Biol 48:443;<br>
Pearson &amp; Lipman (1988) Proc. Nat'l. Acad. Sci, USA 85:2444; Altschul et al. (1990) J. Mol.<br>
Biol. 215:403-410 (BLAST)), by computerized implementations of these algorithms (GAP,<br>
BESTFIT, FASTA, and TFASTA in the Wisconsin Genetics Software Package, Genetics<br>
Computer Group, 575 Science Dr., Madison, Wis.), or by inspection.. Software for<br>
performing BLAST analyses is publicly available through the National Center for<br>
Biotechnology Information (http://www.ncbi.nlm.nih.gov/).<br>
The term "fragment" means a portion or partial sequence of a nucleotide, protein, or<br>
peptide sequence.<br>
As used herein, the term "soluble" means that the protein is not precipitated by<br>
centrifugation at between approximately 5,000x and 20,000x gravity when spun for 10-30<br>
minutes in a buffer under physiological conditions. Soluble proteins are not part of an<br>
inclusion body or other precipitated mass.<br>
As used herein, the term "insoluble" means that the protein that can be precipitated by<br>
centrifugation at between 5,000x and 20,000x gravity when spun for 10-30 minutes in a<br>
buffer under physiological conditions. Insoluble proteins can be part of an includion body or<br>
other precipitated mass.<br>
The term "inclusion body" is meant to include any intracellular body contained within<br>
a cell wherein an aggregate of proteins have been sequestered.<br>
As used herein, the term "homologous" means either i) a protein that has an amino<br>
acid sequence that at least 70, 75, 80, 85, 90, 95, or 98% similar to the sequence of a given<br>
original protein and that retains a desired function of the original protein or ii) a nucleic acid<br>
that has a sequence that is at least 70, 75, 80, 5, 90, 95, or 98% similar to the sequence of a<br>
given nucleic acid and that retains a desired function of the original nucleic acid sequence, hi<br>
all of the embodiments of this invention and disclosure, any disclosed protein, peptide or<br>
nucleic acid can be substituted with a homologous or substantially homologous protein,<br>
peptide or nucleic acid that retains a desired function. In all of the embodiments of this<br>
invention and disclosure, when any nucleic acid is disclosed, it should be assumed that the<br>
invention also includes all nucleic acids that hybridize to the disclosed nucleic acid.<br>
In one non-limiting embodiment, the non-identical amino acid sequence of the<br>
homologous polypeptide can be amino acids that are members of any one of the 15<br>
conservative or semi-conservative groups shown in Table 1,<br>
(Table Removed)Types of Mammalian Proteins Produced<br>
hi general, the recombinant mammalian protein can be any mammalian protein of<br>
which an amino acid sequence is known or any putative mammalian or mammalian-derived<br>
protein for which an amino acid sequence is deduced. The proteins can be selected from the<br>
group consisting of a multi-subunit protein, a blood carrier protein, an enzyme, a full length<br>
antibody, an antibody fragment, or a transcriptional factor.<br>
The amino acid sequence of the protein can be altered to adjust for desired qualities,<br>
such as to ensure certain types of interactions. The sequence can, for example, be adjusted to<br>
reduce immunoreactivity, or to increase absorbtion, reduce excretion, or otherwise enhance<br>
bioavailability in an organism such as a mammal. The amino acid sequence of the protein can<br>
thus be adjusted to ensure certain post-translational modifications or protein conformations.<br>
hi one embodiment, the mammalian protein is a chemokin or cytokine. In another<br>
embodiment, the mammalian proteins is one of the following proteins: DL-1, IL-la, IL-lp,<br>
IL-2, IL-3, IL-4, IL-5, IL-6, IL-7, IL-8, IL-9, IL-10, IL-11, IL-12, IL-12elasti, IL-13, IL-15,<br>
IL-16, IL-18, IL-18BPa, IL-23, IL-24, VIP, erythropoietin, GM-CSF, G-CSF, M-CSF,<br>
platelet derived growth factor (PDGF), MSF, FLT-3 ligand, EOF, fibroblast growth factor<br>
(FGF; e.g., aFGF (FGF-1), bFGF (FGF-2), FGF-3, FGF-4, FGF-5, FGF-6, or FGF-7), insulinlike<br>
growth factors (e.g., IGF-1, IGF-2); tumor necrosis factors (e.g., TNF, Lymphotoxin),<br>
nerve growth factors (e.g., NGF), vascular endothelial growth factor (VEGF); interferons<br>
(e.g., IFN-a, IFN-J3, IFN-y); leukemia inhibitory factor (LIF); ciliary neurotrophic factor<br>
(CNTF); oncostatin M; stem cell factor (SCF); transforming growth factors (e.g., TGF-a,<br>
TGF-pl, TGF-pl, TGF-pl); TNF superfamily (e.g., LIGHT/TNFSF14, STALL-1/TNFSF13B<br>
(BLy5, BAFF, THANK), TNFalphaATNFSF2 and TWEAK/TNFSF12); or chemokines<br>
(BCA-l/BLC-1, BRAK/Kec, CXCL16, CXCR3, ENA-78/LIX, Eotaxin-1, Eotaxin-2/MPIF-2,<br>
Exodus-2/SLC, Fractalkine/Neurotactin, GROalpha/MGSA, HCC-1, I-TAC,<br>
Lymphotactin/ATAC/SCM, MCP-1/MCAF, MCP-3, MCP-4, MDC/STCP-l/ABCD-1, MIPla,<br>
MIP-1P, MIP-2a/GROp, MIP-Sa/Exodus/LARC, MIP-3a/Exodus-3/ELC, MIP-<br>
4/PARC/DC-CK1, PF-4, RANTES, SDFla, TARC, or TECK).<br>
Alternatively, the protein is not a chemoldne or cytokine. In another embodiment, the<br>
mammalian protein is not one of the following proteins: IL-1, IL-la, EL-lp, DL-2, IL-3, IL-4,<br>
IL-5, IL-6, IL-7, IL-8, IL-9, IL-10, IL-11, IL-12, IL-12elasti, IL-13, IL-15, IL-16, IL-18, ILISBPa,<br>
IL-23, IL-24, VIP, erythropoietin, GM-CSF, G-CSF, M-CSF, platelet derived growth<br>
factor (PDGF), MSF, FLT-3 ligand, EOF, fibroblast growth factor (FGF; e.g., aFGF (FGF-1),<br>
bFGF (FGF-2), FGF-3, FGF-4, FGF-5, FGF-6, or FGF-7), insulin-like growth factors (e.g.,<br>
IGF-1, IGF-2); tumor necrosis factors (e.g., TNF, Lymphotoxin), nerve growth factors (e.g.,<br>
NGF), vascular endothelial growth factor (VEGF); interferons (e.g., IFN-a, IFN-P, IFN-y);<br>
leukemia inhibitory factor (LIF); ciliary neurotrophic factor (CNTF); oncostatin M; stem cell<br>
factor (SCF); transforming growth factors (e.g., TGF-a, TGF-pl, TGF-pl, TGF-pl); TNF<br>
superfamily (e.g., LIGHT/TNFSF14, STALL-1/TNFSF13B (BLy5, BAFF, THANK),<br>
TNFalpha/TNFSF2 and TWEAK/TNFSF12); or chemokines (BCA-l/BLC-1, BRAK/Kec,<br>
CXCL16, CXCR3, ENA-78/LIX, Eotaxin-1, Eotaxin-2/MPIF-2, Exodus-2/SLC,<br>
Fractalkine/Neurotactin, GROalpha/MGSA, HCC-1, I-TAC, Lymphotactin/ATAC/SCM,<br>
MCP-1/MCAF, MCP-3, MCP-4, MDC/STCP-l/ABCD-1, MlP-la, MIP-lp, MIP-2a/GROp,<br>
MIP-3a/Exodus/LARC, MIP-3a/Exodus-3/ELC, MIP-4/PARC/DC-CK1, PF-4, RANTES,<br>
SDFla, TARC, or TECK). In one embodiment, the protein is not a porcine protein,<br>
particularly not a porcine growth factor.<br>
As yet a further embodiment of the present disclosure, the recombinant mammalian<br>
proteins, their fragments or other derivatives, or analogs thereof, can be antibodies. These<br>
antibodies can be, for example, polyclonal or monoclonal antibodies. This aspect of the<br>
present invention also includes chimeric, single chain, and humanized antibodies, as well as<br>
Fab fragments, or the product of an Fab expression library.<br>
Production of Multi-subunit Proteins<br>
In one embodiment of the present invention, the production of recombinant<br>
mammalian multi-subunit proteins by a host cell of the species Pseudomonas is provided, hi<br>
another embodiment, a host cell of the Pseudomonas species is provided that has been<br>
transformed to express a recombinant mammalian multi-subunit protein. In one embodiment,<br>
multisubunit proteins, including recombinant mammalian or human proteins, are expressed in<br>
a Pseudomonas host cell. In one embodiment, expression of the multi-subunit protein by the<br>
host cell is followed by isolation of the multi-subunit protein. In another embodiment, the<br>
multi-subunit protein of peptide is purified. The protein can be assembled by the cell before<br>
purification or isolation, or further assembly can be undertaken during or after isolation or<br>
purification. Optionally, the protein or any portion thereof can be renatured or refolded to<br>
produce active proteins.<br>
Any of a variety of vectors and expression systems can be used to express the multisubunit<br>
protein in the host cell. The multi-subunits can be located on a single vector,<br>
optionally operably linked to different promoters, optionally in a polycistronic sequence.<br>
Each subunit can also be on different vectors. Multiple vectors can be used. Each subunit can<br>
be under the control of one or more selection markers. Regulatory elements can be included<br>
on the vector, including periplasmic secretion signal sequences, internal ribosome entry sites,<br>
activator sequences, promoters, and termination signals.<br>
In one embodiment, multisubunit proteins are expressed in Pseudomonas using<br>
expression systems with auxotrophic selection markers as disclosed in U.S. Application No.<br>
10/994,138 to Dow Global Technologies filed November 19, 2004, wherein the control of<br>
each nucleic acid encoding a subunit is under the control of an auxotrophic selection marker.<br>
Multisubunit proteins that can be expressed include homomeric and heteromeric proteins.<br>
The multisubunit proteins may include two or more subunits, that may be the same or<br>
different. For example, the protein may be a homomeric protein comprising 2, 3, 4, 5, 6, 7, 8,<br>
9, 10, 11, 12 or more subunits. The protein also may be a heteromeric protein including 2, 3,<br>
4, 5, 6, 7, 8, 9,10,11,12, or more subunits.<br>
Exemplary multisubunit mammalian proteins include: receptors including ion channel<br>
receptors; signaling proteins such as kinases, GTPases, ATPases; transmembrane proteins;<br>
extracellular matrix proteins including chondroitin; collagen; immunomodulators including<br>
MHC proteins, full chain antibodies, and antibody fragments; enzymes including RNA<br>
polymerases, and DNA polymerases; and membrane proteins.<br>
Production of Blood Proteins<br>
In one embodiment of the present invention, the production of recombinant<br>
mammalian blood proteins is provided. In one embodiment, expression of the blood protein<br>
by the host cell is followed by isolation of the blood protein. In another embodiment, the<br>
blood protein is purified. In another embodiment, following isolation of the blood protein, the<br>
blood protein is purified. Optionally, the protein can be renatured or refolded to produce<br>
active protein. In general, a recombinant blood protein of this invention is produced by<br>
transforming a suitable host cell, such as a P. fluorescens host cell, with a nucleic acid<br>
construct encoding the blood protein, culturing the transformed host cell under conditions<br>
appropriate for expression, and optionally isolating, or isolating and purifying the<br>
recombinant blood protein expressed by the cell.<br>
In another embodiment, a host cell of the Pseudomonas species is provided that has<br>
been transformed to express a recombinant mammalian blood protein with an vector<br>
containing appropriate genes and regulatory elements for expression of the blood protein of<br>
interest is provided.<br>
1*he blood proteins that can be expressed include, but are not limited to: carrier<br>
proteins, such as albumin, including human albumin (Seq ID No. 1, Table 2) and bovine<br>
albumin; transferrin, including human transferrin (Seq ID No. 2, Table 2), bovine transferrin,<br>
rat transferrin, recombinant transferrin, recombinant transferrin half-molecules, recombinant<br>
transferrin half-molecules having altered properties; haptoglobin; fibrinogen and other<br>
coagulation factors; complement components; immunoglobulins; enzyme inhibitors;<br>
precursors of substances such as angiotensin and bradykinin; insulin; endothelin; globulin<br>
including alpha, beta, and gamma-globulin; and other types of proteins, peptides, and<br>
fragments thereof found primarily in the blood of mammals. The amino acid sequences for<br>
numerous blood proteins have been reported (see, S. S. Baldwin (1993) Comp. Biochem<br>
Physiol. 106b: 203-218), including the amino acid sequence for human serum albumin (Lawn,<br>
L.M., et al. (1981) Nucleic Acids Research 9:22; pp 6103-6114) and human serum transferrin<br>
(Yang, F. el al. (1984) Proc. Natl. Acad. Sci. USA 81; pp. 2752-2756).<br>
In a specific embodiment, the production of albumin in P. fluorescens, is provided,<br>
comprising transforming a P. fluorescens host cell with an expression vector containing a<br>
nucleic acid sequence or sequences and regulatory elements for expression of albumin,<br>
culturing the host cell under conditions suitable for expression of the albumin, and recovering<br>
the albumin expressed by P. fluorescens. According to this embodiment, the albumin<br>
expressed is selected from the group consisting of human albumin, bovine albumin, rabbit<br>
albumin, chicken albumin, rat albumin, and mouse albumin. In another embodiment, albumin<br>
can be fused to a therapeutically active polypeptide, which can be of mammalian or nonmammalian<br>
origin.<br>
In a further specific embodiment, the production of a transferrin in P. fluorescens is<br>
provided, comprising transforming a P. fluorescens host cell with an expression vector<br>
containing nucleic acid and regulatory elements for expression of the transferrin, culturing the<br>
host cell under conditions suitable for expression of the transferring. In another embodiment,<br>
following expression of the transferrin, and, in one embodiment, isolating the protein. In a<br>
further embodiment, the transferrin can be purified following isolation. The transferrin<br>
expressed is selected from the group consisting of human serum transferrin, glycosylated<br>
human transferrin, non-glycosylated human transferrin, the N-terminal half-molecule of<br>
human transferrin, bovine transferrin, rat transferrin, mouse transferrin, primate transferrin,<br>
recombinant transferrin, recombinant transferrin half-molecules, recombinant transferrin halfmolecules<br>
having altered properties, transferrin polynucleotides, transferrin polypeptides<br>
encoded by transferrin polypeptides, transferrin polypeptides, transferrin antibodies,<br>
transferrin fragments, and transferrin fused to a therapeutically active polypeptide.<br>
In yet another specific embodiment, the production of a globulin in P. fluorescens is<br>
provided, comprising transforming a P. fluorescens host cell with an expression vector<br>
containing nucleic acid and regulatory elements for expression of the globulin, culturing the<br>
host cell under conditions suitable for expression of the globulin and optionally isolating the<br>
protein. In a further embodiment, following expression, the globulin is isolated and purified<br>
from the host cell. The globulin expressed is selected from the group consisting of human<br>
globulin, bovine globulin, rabbit globulin, rat globulin, mouse globulin, sheep globulin,<br>
monkey globulin, steroid-binding globulins, and globulin fused to a therapeutically active<br>
polypepride.<br>
In a further embodiment, the production of an insulin in P. fluorescens is provided,<br>
comprising transforming a P. fluorescens host cell with an expression vector containing<br>
nucleic acid and regulatory elements for expression of the insulin, culturing the host cell<br>
under conditions suitable for expression of the insulin and optionally isolatig the protein. In a<br>
further embodiment, the insulin can be isolated and purified following production of the<br>
insulin by the host cell. The insulin expressed is selected from the group consisting of human<br>
insulin, bovine insulin, mouse insulin, rat insulin, porcine insulin, monkey insulin, and insulin<br>
fused to a therapeutically active polypeptide. The accession number for human insulin genes<br>
is J00265, and for synthetic human insulin gene the accession number is J02547.<br>
Full-length DNA fof production of recombinant blood proteins or truncated DNA<br>
encoding either the amino-terminal or carboxy-terminal lobe of blood proteins or a portion<br>
thereof can be obtained from available sources or can be synthesized according to the known<br>
sequences by standard procedures.<br>
(Table Removed)Seq. ID. No: 1<br>
Seq. ID. No: 2<br>
(Strausberg (2002) PNAS, 99:16899-16903)<br>
cDNA<br>
Sequence<br>
of Human<br>
Insulin<br>
Production of Enzymes<br>
In one embodiment of the present invention, the production of recombinant<br>
mammalian enzymes or co-factors by a host cell of the species Pseudomonas fluorescens is<br>
provided. In another embodiment, a host cell of the Pseudomonas species is provided that has<br>
been transformed to express a recombinant mammalian enzyme or co-factor.<br>
The enzymes and co-factors expressed in this embodiment include but are not limited<br>
to aldolases, amine oxidases, amino acid oxidases, aspartases, Bn dependent enzymes,<br>
carboxypeptidases, carboxyesterases, carboxylyases, chemotrypsin, CoA requiring enzymes,<br>
cyanohydrin synthetases, cystathione synthases, decarboxylases, dehydrogenases, alcohol<br>
dehydrogenases, dehydratases, diaphorases, dioxygenases, enoate reductases, epoxide<br>
hydrases, fumerases, galactose oxidases, glucose isomerases, glucose oxidases,<br>
glycosyltrasferases, methyltransferases, nitrile hydrases, nucleoside phosphorylases,<br>
oxidoreductases, oxynitilases, peptidases, glycosyltrasferases, peroxidases, and enzymes<br>
fused to a therapeutically active polypeptide.<br>
In another embodiment, the enzyme can be a mesophilic enzyme polypeptide, for<br>
example one that is desirable for human and/or veterinary therapeutic and/or diagnostic use.<br>
Examples of such therapeutic mesophilic enzymes include, e.g., tissue plasminogen activator;<br>
urokinase, reptilase, streptokinase; catalase, superoxide dismutase; DNAse, amino acid<br>
hydrolases (e.g., asparaginase, amidohydrolases); carboxypeptidases; proteases, trypsin,<br>
pepsin, chymotrypsin, papain, bromelain, collagenase; neuraminidase; lactase, maltase,<br>
sucrase, and arabinofuranosidases.<br>
Yet another embodiment provides for the production of recombinant enzyme<br>
replacements in P. fluorescens cells by transforming a P. fluorescens host cell with an<br>
expression vector containing nucleic acids and regulatory elements for xpression of<br>
recombinant enzyme replacements, and culturing the cell under conditions suitable for<br>
expression of the recombinant enzyme replacements. The recombinant enzyme replacements<br>
expressed in the host cell is selected from the group consisting of Algasidase beta,<br>
Laronidase, and recombinant enzyme replacements fused to a therapeutically active<br>
polypeptide.<br>
Production of Mammalian Antibodies and Antibody Fragments<br>
In one embodiment of the present invention, the production of recombinant<br>
mammalian single chain, Fab fragments and/or full chain antibodies or fragments or portions<br>
thereof by a host cell of the species P. fluorescens is provided. In one embodiment, following<br>
expression of the protein, the protein can be isolated and optionally purified. Optionally, the<br>
protein can be renatured to produce an active protein. The antibody or antibody fragments are<br>
optionally linked to a secretion signal sequence for targeting in the cell during production.<br>
In another embodiment, a host cell of the Pseudomonas species is provided that has<br>
been transformed to express a recombinant mammalian single chain, Fab fragments and/or<br>
full chain antibodies or fragments or portions thereof.<br>
In one embodiment, the P. fluorescens cell can produces a single chain antibody or<br>
fragments or portions thereof. A single-chain antibody can include the antigen-binding<br>
regions of antibodies on a single stably-folded polypeptide chain. Single-chain antibodies are<br>
of smaller size than classical immunoglobulins but can retain the antigen-specific binding<br>
properties of antibodies. Single chain antibodies can be used for therapeutics, such as "naked"<br>
single-chain antibodies, bi-specific antibody binders, radioconjugates or as fusions with<br>
effector domains, diagnostics, such as tumor imaging or in vivo or ex vivo cancer marker<br>
assays, research tools, such as protein purification and detection, including identification and<br>
characterization of novel therapeutic targets, antibody microarrays, display technologies<br>
and/or vehicles for gene or drug delivery.<br>
In another embodiment, the P, jluorescens cell produces Fab fragments or portions<br>
thereof. Fab fragments can be a piece of a particular antibody. The Fab fragment can contain<br>
the antigen binding site. The Fab fragment can contains 2 chains : a light chain and a heavy<br>
chain fragment. These fragments can be linked via alinker or a disulfide bond.<br>
In other embodiments of the present invention, full chain antibodies can be expressed<br>
in P. Jluorescens, and other Pseudomonas species. An intact antibody containing the Fc<br>
region can be more resistant against degradation and clearance in vivo, thereby having longer<br>
biological half life in circulation. Such antibodies can be used as a therapeutic agent for<br>
diseases requiring sustained therapies.<br>
In one embodiment, a process for producing a functional antibody or fragment thereof<br>
in Pseudomonas species is provided by providing an expression vector that contains separate<br>
cistronic or polycistronic sequences. The separate cistron expression vector can contain a first<br>
promoter-cistron pair for expression of an immunoglobulin light chain and a second<br>
promoter-cistron pair for expression of an immunoglobulin heavy chain, such that expression<br>
of the light chain and heavy chain are independently regulated by separate promoters. Each<br>
cistron within the expression cassette polynucleotide can include a translation initiation region<br>
(TDR.) operably linked to the nucleic acid sequence coding for the light chain or heavy chain<br>
of the full length antibody. In one embodiment, the TIR sequences can be manipulated to<br>
provide different translational strength combinations for light and heavy chains. In an<br>
alternative embodiment, a heavy chain coding sequence can be located on the same plasmid<br>
as a light chain coding sequence. In an alternative embodiment, the heavy and light chain<br>
sequences are found in a polycistronic sequence within a single plasmid, or coded into the<br>
genome of the host.<br>
In another embodiment, a process is provided for producing a functional antibody or<br>
fragment thereof in a host cell transformed with two separate translational units respectively<br>
encoding the light and heavy chains of the antibody. In one embodiment the process includes:<br>
a) culturing the host cell under suitable conditions so that the light chain and heavy chain are<br>
expressed in a sequential fashion, thereby temporally separating the production of the light<br>
and heavy chains; and b) allowing the assembly of the light and heavy chains to form the<br>
functional antibody or fragment thereof.<br>
In further embodiment, the Pseudomonas expression system can express human<br>
therapeutic single chain, Fab fragments or full chain antibodies or portions thereof, including,<br>
but not limited to Fab, Fab', F(ab')2, F(ab')2-leucine zipper, Fv, dsFv, anti-GDI8 antibody,<br>
chimeric antibodies, human antibodies, humanized antibodies, or those described in the Table<br>
3 below.<br>
(Table Removed)Sarcoma<br>
Allograft rejection<br>
Production of Transcriptional Factors<br>
In one embodiment of the present invention, the production of recombinant<br>
mammalian transcription factors by a host cell of the species Pseudomonas fluorescens is<br>
provided. In one embodiment, following expression of the protein, the protein can be<br>
isolated. In another embodiment, the protein can be purified. Optionally, the protein can be<br>
renatured to produce an active protein. In another embodiment, a host cell of the<br>
Pseudomonas species is provided that has been transformed to express a recombinant<br>
mammalian transcription factor.<br>
Transcription factors suitable for insertion into the expression systems of the present<br>
invention include those of the helix turn helix family and members of the Pac family, as well<br>
as other transcription factor families known in the art. Members of these families suitable for<br>
use with the present invention include mammalian and mammalian homologs and analogs of:<br>
transcriptional regulators; transcription factors of the of the ASNC family such as<br>
ASNC_trans_reg, putative transcriptional regulators; bacterial regulatory proteins of the luxR<br>
family; bacterial regulatory helix-turn-helix transcription factors; bacterial regulatory proteins<br>
of the arsR family; transcription factors of the helix-tum-helix domain, especially the rpiR<br>
family; bacterial regulatory protein transcription factors, bacterial regulatory helix-turn-helix<br>
transcription factors; DNA binding domain transcription factors; MarR family of transcription<br>
factors; the ROK family of transcription factors; the MerR family of regulatory proteins;<br>
arginine represser transcription factors; firmicute transcriptional factors; ferric uptake<br>
regulatory transcription factors; sigma transcription factors; response regulatory receiver<br>
transcription factors; tryptophan RNA-binding attenuator protein transcription factors;<br>
putative sugar-binding domain transcription factors; PRD domain transcription factors;<br>
nitrogen regulatory protein transcription factors; negative regulators of generic competence,<br>
such as MecA; negative transcriptional regulator transcription factors; bacterial transcriptional<br>
regulator transcription factors; glycerol-3-phosphate responsive transcription factors; iron<br>
dependent represser transcription factors; and numerous species specific transcriptional<br>
regulator transcription factors.<br>
Transcriptional factors expressed by Pseudomonas species can be utilized for<br>
diagnostic, therapeutic, and investigational applications.<br>
Vector Preparation<br>
Polvnucleotides<br>
The recombinant mammalian proteins and peptides can be expressed from<br>
polynucleotides in which the target polypeptide coding sequence is operably attached to<br>
transcription and translation regulatory elements forming a functional gene from which the<br>
host cell can express the protein. The coding sequence can be a native coding sequence for<br>
the target polypeptide, if available, but can also be a coding sequence that has been selected,<br>
improved, or optimized for use in the selected expression host cell: for example, by<br>
synthesizing the gene to reflect the codon use bias of a Pseudomonas species such as P.<br>
jluorescens. The gene(s) that result will have been constructed within or will be inserted into<br>
one or more vector, which will then be transformed into the expression host cell. Nucleic acid<br>
or a polynucleotide said to be provided in an "expressible form" means nucleic acid or a<br>
polynucleotide that contains at least one gene that can be expressed by the selected bacterial<br>
expression host cell.<br>
Regulatory Elements<br>
The regulatory elements used herein can be operably attached to the target<br>
recombinant mammalian protein encoding gene. The coding sequence of the proteinencoding<br>
gene used herein can contain, in addition to the mature polypeptide coding sequence<br>
and transcription-regulatory elements, further encoding elements, e.g., one or more of coding<br>
sequences for peptide tags, pre-peptides, pro-peptides, pre-pro-peptides, or other commonly<br>
utilized encoding elements known in the art, excluding secretion signal peptides functional in<br>
the selected expression host cell.<br>
The term "operably attached," as used herein, refers to any configuration in which the<br>
transcriptional and any translational regulatory elements are covalently attached to the coding<br>
sequence in such disposition(s), relative to the coding sequence, that the regulatory elements<br>
can direct the expression of the coding sequence. In one embodiment, the regulatory elements<br>
will be part of a whole gene before undergoing transformation into a host cell; however, in<br>
other embodiments the regulatory elements are part of another gene, which can be part of the<br>
host genome or can be part of a genome of another organism, or can be derived therefrom.<br>
Promoters and Accessory Elements<br>
The promoters used in accordance with the present invention may be constitutive<br>
promoters or regulated promoters. Common examples of useful regulated promoters include<br>
those of the family derived from the lac promoter (i.e. the lacZ promoter), especially the tac<br>
and trc promoters described in U.S. Patent No. 4,551,433 to DeBoer, as well as Ptacl6,<br>
Ptacl7, PtacII, PlacUVS, and the TVlac promoter.<br>
Common examples of non-lac-type promoters useful in expression systems according<br>
to the present invention include, e.g., those listed in Table 4.<br>
(Table Removed)See, e.g.: J. Sanchez-Romero &amp; V. De Lorenzo (1999) Manual of Industrial<br>
Microbiology and Biotechnology (A. Demain &amp; J. Davies, eds.) pp.460-74 (ASM Press,<br>
Washington, D.C.); H. Schweizer (2001) Current Opinion in Biotechnology 12:439-445; and<br>
R. Slater &amp; R. Williams (2000) Molecular Biology and Biotechnology (J. Walker &amp; R.<br>
Rapley, eds.) pp. 125-54. A promoter having the nucleotide sequence of a promoter native to<br>
the selected bacterial host cell may also be used to control expression of the transgene<br>
encoding the target polypeptide, e.g, a Pseudomonas anthranilate or benzoate operon<br>
promoter (Pant, Pben). Tandem promoters may also be used in which more than one<br>
promoter is covalently attached to another, whether the same or different in sequence, e.g., a<br>
Pant-Pben tandem promoter (interpromoter hybrid) or a Plac-Plac tandem promoter.<br>
Regulated promoters utilize promoter regulatory proteins in order to control<br>
transcription of the gene of which the promoter is a part. Where a regulated promoter is used<br>
herein, a corresponding promoter regulatory protein will also be part of an expression system<br>
according to the present invention. Examples of promoter regulatory proteins include:<br>
activator proteins, e.g., E. coli catabolite activator protein, MalT protein; AraC family<br>
transcriptional activators; represser proteins, e.g., E. coli LacI proteins; and dual-ruction<br>
regulatory proteins, e.g., E. coli NagC protein. Many regulated-promoter/promoterregulatory-<br>
protein pairs are known in the art.<br>
Promoter regulatory proteins interact with an effector compound, i.e. a compound that<br>
reversibly or irreversibly associates with the regulatory protein so as to enable the protein to<br>
either release or bind to at least one DNA transcription regulatory region of the gene that is<br>
under the control of the promoter, thereby permitting or blocking the action of a transcriptase<br>
enzyme in initiating transcription of the gene. Effector compounds are classified as either<br>
inducers or co-repressors, and these compounds include native effector compounds and<br>
gratuitous inducer compounds. Many regulated-promoter/promoter-regulatory-protein/<br>
effector-compound trios are known in the art. Although an effector compound can be used<br>
throughout the cell culture or fermentation, in one embodiment in which a regulated promoter<br>
is used, after growth of a desired quantity or density of host cell biomass, an appropriate<br>
effector compound is added to the culture in order directly or indirectly result in expression of<br>
the desired target gene(s).<br>
By way of example, where a lac family promoter is utilized, a lad gene can also be<br>
present in the system. The lad gene, which is (normally) a constitutively expressed gene,<br>
encodes the Lac represser protein (LacI protein) which binds to the lac operator of these<br>
promoters. Thus, where a lac family promoter is utilized, the lad gene can also be included<br>
and expressed in the expression system. In the case of the lac promoter family members, e.g.,<br>
the tac promoter, the effector compound is an inducer, such as a gratuitous inducer like IPTG<br>
(isopropyl-p-D-1-thiogalactopyranoside, also called "isopropylthiogalactoside").<br>
Other Elements<br>
Other regulatory elements can be included in an expression construct. Such elements<br>
include, but are not limited to, for example, transcriptional enhancer sequences, translational<br>
enhancer sequences, other promoters, activators, translational start and stop signals,<br>
transcription terminators, cistronic regulators, polycistronic regulators, tag sequences, such as<br>
nucleotide sequence "tags" and "tag" peptide coding sequences, which facilitates<br>
identification, separation, purification, or isolation of an expressed polypeptide.<br>
At a minimum, a protein-encoding gene according to the present invention can<br>
include, in addition to the mammalian protein coding sequence, the following regulatory<br>
elements operably linked thereto: a promoter, a ribosome binding site (RBS), a transcription<br>
terminator, translational start and stop signals. Useful RBSs can be obtained from any of the<br>
species useful as host cells in expression systems, such as from the selected host cell. Many<br>
specific and a variety of consensus RBSs are known, e.g., those described in and referenced<br>
by D. Frishman et al. (1999) Gene 234(2):257-65; and B.E. Suzek et al. (2001) Bioinformatics<br>
17(12): 123-30. In addition,. either native or synthetic RBSs may be used, e.g., those<br>
described in: EP 0207459 (synthetic RBSs); O. Ikehata et al. (1989) Eur. J. Biochem.<br>
181(3):563-70 (native RBS sequence of AAGGAAG). Further examples of methods, vectors,<br>
and translation and transcription elements, and other elements useful in the present invention<br>
are described in, e.g.: U.S. Patent Nos. 5,055,294 and 5,128,130 to Gilroy et al.; 5,281,532 to<br>
Rammler et al.; 4,695,455 and 4,861,595 to Barnes et al.; 4,755,465 to Gray et al.; and<br>
5,169,760 to Wilcox.<br>
Vectors<br>
Transcription of the DNA encoding the enzymes of the present invention by<br>
Pseudomonas is increased by inserting an enhancer sequence into the vector or plasmid.<br>
Typical enhancers are cis-acting elements of DNA, usually about from 10 to 300 bp in size<br>
that act on the promoter to increase its transcription. Examples include various Pseudomonas<br>
enhancers, as described elsewhere herein.<br>
Generally, the recombinant expression vectors will include origins of replication and<br>
selectable markers permitting transformation of the Pseudomonas host cell, e.g., the<br>
antibiotic-free resistance genes of P. fluorescens, and a promoter derived from a highlyexpressed<br>
gene to direct transcription of a downstream structural sequence. Such promoters<br>
can be derived from operons encoding the enzymes such as 3-phosphoglycerate kinase<br>
(PGK), acid phosphatase, or heat shock proteins, among others. The heterologous structural<br>
sequence is assembled in appropriate phase with translation initiation and termination<br>
sequences, and, in one embodiment, a leader sequence capable of directing secretion of the<br>
translated enzyme. Optionally, the heterologous sequence can encode a fusion enzyme<br>
including an N-terminal identification peptide imparting desired characteristics, e.g.,<br>
stabilization or simplified purification of expressed recombinant product.<br>
Useful expression vectors for use with P. fluorescens in expressing enzymes are<br>
constructed by inserting a structural DNA sequence encoding a desired protein together with<br>
suitable translation initiation and termination signals in operable reading phase with a<br>
functional promoter. The vector will comprise one or more phenotypic selectable markers<br>
and an origin of replication to ensure maintenance of the vector and to, if desirable, provide<br>
amplification within the host.<br>
Vectors are known in the art as useful for expressing recombinant proteins in host<br>
cells, and any of these may be used for expressing the genes according to the present<br>
invention. Such vectors include, e.g., plasmids, cosmids, and phage expression vectors.<br>
Examples of useful plasmid vectors include, but are not limited to, the expression plasmids<br>
pBBRlMCS, pDSK519, pKT240, pML122, pPSlO, RK2, RK6, pRO1600, and RSF1010.<br>
Other examples of such useful vectors include those described by, e.g.: N. Hayase (1994)<br>
Appl. Envir. Microbiol. 60(9):3336-42; A.A. Lushnikov et al. (1985) Basic Life Sci. 30:657-<br>
62; S. Graupner &amp; W. Wackernagel (2000) Biomolec. Eng. 17(1): 11-16.; H.P. Schweizer<br>
(2001) Curr. Opin. Biotech. 12(5):439-45; M. Bagdasarian &amp; K.N. Timmis (1982) Curr.<br>
Topics Microbiol. Immunol. 96:47-67; T. Ishii et al. (1994) FEMS Microbiol. Lett.<br>
116(3):307-13; I.N. Olekhnovich &amp; Y.K. Fomichev (1994) Gene 140(l):63-65; M. Tsuda &amp;<br>
T. Nakazawa (1993) Gene 136(l-2):257-62; C. Nieto et al. (1990) Gene 87(l):145-49; J.D.<br>
Jones &amp; N. Gutterson (1987) Gene 61(3):299-306; M. Bagdasarian et al. (1981) Gene 16(1-<br>
3):237-47; H.P. Schweizer et al. (2001) Genet. Eng. (NY) 23:69-81; P. Mukhopadhyay et al.<br>
(1990) J. Bact. 172(1):477-80; D.O. Wood et al. (1981) J. Bact. 145(3): 1448-51; and R.<br>
Hollwick et al. (2001) Microbiology 147(Pt 2):337-44.<br>
Further examples of expression vectors that can be useful in Pseudomonas host cells<br>
include those listed in Table 5 as derived from the indicated replicons.<br>
(Table Removed)The expression plasmid, RSFIOIO, is described, e.g., by F. Heflron et al. (1975) Proc.<br>
Nat'l Acad. Sci. USA 72(9):3623-27, and by K. Nagahari &amp; K. Sakaguchi (1978) J. Bact.<br>
133(3): 1527-29, Plasmid RSFIOIO and derivatives thereof are particularly useful vectors in<br>
the present invention. Exemplary, useful derivatives of RSFIOIO, which are known in the art,<br>
include, e.g., pKT212, pKT214, pKT231 and related plasmids, and pMYClOSO and related<br>
plasmids (see, e.g., U.S. Patent Nos. 5,527,883 and 5,840,554 to Thompson et al.), such as,<br>
e.g., pMYC1803. Plasmid pMYC1803 is derived from the RSFlOlO-based plasmid pTJS260<br>
(see U.S. Patent No. 5,169,760 to Wilcox), which carries a regulated tetracycline resistance<br>
marker and the replication and mobilization loci from the RSFIOIO plasmid. Other<br>
exemplary useful vectors include those described in U.S. Patent No. 4,680,264 to Puhler et al.<br>
Li a one embodiment, an expression plasmid is used as the expression vector. In<br>
another embodiment, RSFIOIO or a derivative thereof is used as the expression vector. In still<br>
another embodiment, pMYClOSO or a derivative thereof, or pMYC1803 or a derivative<br>
thereof, is used as the expression vector.<br>
The plasmid can be maintained in the host cell by use of a selection marker gene, also<br>
present in the plasmid. This may be an antibiotic resistance gene(s), in which case the<br>
corresponding antibiotic(s) will be added to the fermentation medium, or any other type of<br>
selection marker gene known as useful in the art, e.g., a prototrophy-restoring gene in which<br>
case the plasmid will be used in a host cell that is auxotrophic for the corresponding trait, e.g.,<br>
a biocatalytic trait such as an amino acid biosynthesis or a nucleotide biosynthesis trait or a<br>
carbon source utilization trait.<br>
Extensive sequence infonnation required for molecular genetics and generic<br>
engineering techniques is widely publicly available. Access to complete nucleotide sequences<br>
of mammalian, as well as human, genes, cDNA sequences, amino acid sequences and<br>
genomes can be obtained from GenBank at the URL address<br>
http: www.ncbi.nlm.nih.gov/Entrez. Additional information can also be obtained from<br>
GeneCards, an electronic encyclopedia integrating information about genes and their products<br>
and biomedical applications from the Weizmann Institute of Science Genome and<br>
Bioinformatics (http://bioinformatics.weizmann.ac.il/cards/), nucleotide sequence information<br>
can be also obtained from the EMBL Nucleotide Sequence Database (<br>
http: www.ebi.ac.uk/embl/) or the DNA Databank or Japan (DDBJ,<br>
httpwww.ddbj.nig.ac.jp/; additional sites for information on amino acid sequences include<br>
Georgetown's protein information resource website (http://www-nbrf.georgetown.edu/pir/)<br>
and Swiss-Prot (http://au.expasy.org/sprot/sprot-top.html).<br>
Transformation<br>
Transformation of the Pseudomonas host cells with the vector(s) may be performed<br>
using any transformation methodology known in the art, and the bacterial host cells may be<br>
transformed as intact cells or as protoplasts (i.e. including cytoplasts). Exemplary<br>
transformation methodologies include poration methodologies, e.g., electroporation,<br>
protoplast fusion, bacterial conjugation, and divalent cation treatment, e.g., calcium chloride<br>
treatment or CaCl/Mg2+ treatment, or other well known methods in the art. See, e.g., Morrison<br>
(1977) J. Bact. 132:349-351; Clark-Curtiss &amp; Curtiss (1983) Methods in Enzymology<br>
101:347-362, Sambrook et al. (1989) Molecular Cloning, A Laboratory Manual (2nd ed.);<br>
Kriegler (1990) Gene Transfer and Expression: A Laboratory Manual; and Ausubel et al.,<br>
eds. (1994) Current Protocols in Molecular Biology.<br>
Pseudomonas Organisms<br>
While the primary invention herein is the use of Pseudomonas fluorescens, other<br>
Pseudomonas and closely related bacterial organisms can be useful. Pseudomonas and<br>
closely related bacteria are generally part of the group defined as "Gram(-) Proteobacteria<br>
Subgroup 1" or "Gram-Negative Aerobic Rods and Cocci" (Buchanan and Gibbons (eds.)<br>
(197 4) Sergey's Manual of Determinative Bacteriology, pp. 217-289).<br>
(Table Removed)"Gram-negative Proteobacteria Subgroup 1" also includes Proteobacteria that would<br>
be classified in this heading according to the criteria used in the classification. The heading<br>
also includes groups that were previously classified in this section but are no longer, such as<br>
the genera Acidovorax, Brevundimonas, Burkholderia, Hydrogenophaga, Oceanimonas,<br>
Ralstonia, and Stenotrophomonas, the genus Sphingomonas (and the genus Blastomonas,<br>
derived therefrom), which was created by regrouping organisms "belonging to (and previously<br>
called species of) the genus Xanthomonas, the genus Acidomonas, which was created by<br>
regrouping organisms belonging to the genus Acetobacter as defined in Bergey (1974). In<br>
addition hosts can include cells from the genus Pseudomonas, Pseudomonas enalia (ATCC<br>
14393), Pseudomonas nigrifaciens (ATCC 19375), and Pseudomonas putrefaciens (ATCC<br>
8071), which have been reclassified respectively as Alteromonas haloplanktis, Alteromonas<br>
nigrifaciens , and Alteromonas putrefaciens. Similarly, e.g., Pseudomonas acidovorans<br>
(ATCC 15668) and Pseudomonas testosteroni (ATCC 11996) have since been reclassified as<br>
Comamonas acidovorans and Comamonas testosteroni, respectively; and Pseudomonas<br>
nigrifaciens (ATCC 19375) and Pseudomonas piscicida (ATCC 15057) have been<br>
reclassified respectively as Pseudoalteromonas nigrifaciens and Pseudoalteromonas<br>
piscicida. " Gram-negative Proteobacteria Subgroup 1" also includes Proteobacteria classified<br>
as belonging to any of the families: Pseudomonadaceae, Azotobacteraceae (now often called<br>
by the synonym, the "Azotobacter group" of Pseudomonadaceae), Rhizobiaceae, and<br>
Methylomonadaceae (now often called by the synonym, " Methylococcaceae").<br>
Consequently, in addition to those genera otherwise described herein, further Proteobacterial<br>
genera falling within " Gram-negative Proteobacteria Subgroup 1" include: 1) Azotobacter<br>
group bacteria of the genus Azorhizophilus; 2) Pseudomonadaceae family bacteria of the<br>
genera Cellvibrio, Oligella, and Teredinibacter; 3) Rhizobiaceae family bacteria of the genera<br>
Chelatobacter, Ensifer, Liberibacter (also called "Candidates Liberibacter"), and<br>
Sinorhizobium; and 4) Methylococcaceae family bacteria of the genera Methylobacter,<br>
Methylocaldum, Methylomicrobium, Methylosarcina, and Methylosphaera ,<br>
In another embodiment, the host cell is selected from " Gram-negative Proteobacteria<br>
Subgroup 2." " Gram-negative Proteobacteria Subgroup 2" is defined as the group of<br>
Proteobacteria of the following genera (with the total numbers of catalog-listed, publiclyavailable,<br>
deposited strains thereof indicated in parenthesis, all deposited at ATCC, except as<br>
otherwise indicated): Acidomonas (2); Acetobacter (93); Gluconobacter (37); Brevundimonas<br>
(23); Beijerinckia (13); Derxia (2); Brucella (4); Agrobacterium (79); Chelatobacter (2);<br>
Ensifer (3); Rhizobium (144); Sinorhizobium (24); Blastomonas (1); Sphingomonas (27);<br>
Alcaligenes (88); Bordetella (43); Burkholderia (73); Ralstonia (33); Acidovorax (20);<br>
Hydrogenophaga (9); Zoogloea (9); Methylobacter (2); Methylocaldum (1 at NCIMB);<br>
Methylococcus (2); Methylomicrobium (2); Methylomonas (9); Methylosarcina (1);<br>
Methylosphaera; Azomonas (9); Azorhizophilus (5); Azotobacter (64); Cellvibrio (3);<br>
Oligella (5); Pseudomonas (1139); Francisella (4); Xanthomonas (229); Stenotrophomonas<br>
(50); and Oceanimonas (4).<br>
Exemplary host cell species of " Gram-negative Proteobacteria Subgroup 2" include,<br>
but are not limited to the following bacteria (with the ATCC or other deposit numbers of<br>
exemplary strain(s) thereof shown in parenthesis): Acidomonas methanolica (ATCC 43581);<br>
Acetobacter aceti (ATCC 15973); Gluconobacter oxydans (ATCC 19357); Brevundimonas<br>
diminuta (ATCC 11568); Beijerinckia indica (ATCC 9039 and ATCC 19361); Derxia<br>
gummosa (ATCC 15994); Brucella melitensis (ATCC 23456), Brucella abortus (ATCC<br>
23448); Agrobacterium tumefaciens (ATCC 23308), Agrobacterium radiobacter (ATCC<br>
19358), Agrobacterium rhizogems (ATCC 11325); Chelatobacter heintzii (ATCC 29600);<br>
Ensifer adhaerens (ATCC 33212); Rhizobium leguminosarum (ATCC 10004); Sinorhizobium<br>
fredii (ATCC 35423); Blastomonas natatoria (ATCC 35951); Sphingomonas paucimobilis<br>
(ATCC 29837); Alcaligenes faecalis (ATCC 8750); Bordetella pertussis (ATCC 9797);<br>
Burkholderia cepacia (ATCC 25416); Ralstonia pickettii (ATCC 27511); Addovorax facilis<br>
(ATCC 11228); Hydrogenophaga flava (ATCC 33667); Zoogloea ramigera (ATCC 19544);<br>
Methylobacter luteus (ATCC 49878); Methylocaldum gracile (NCMB 11912);<br>
Methylococcus capsulatus (ATCC 19069); Methylomicrobium agile (ATCC 35068);<br>
Methylomonas methanica (ATCC 35067); Methylosarcina fibrata (ATCC 700909);<br>
Methylosphaera hansonii (ACAM 549); Azomonas agilis (ATCC 7494); Azorhizophilus<br>
paspali (ATCC 23833); Azotobacter chroococcum (ATCC 9043); Cellvibrio mixtus (UQM<br>
2601); Oligella urethralis (ATCC 17960); Pseudomonas aeruginosa (ATCC 10145),<br>
Pseudomonas fluorescens (ATCC 35858); Francisella tularensis (ATCC 6223);<br>
Stenotrophomonas maltophilia (ATCC 13637); Xanthomonas campestris (ATCC 33913); and<br>
Oceanimonas doudoroffii (ATCC 27123).<br>
In another embodiment, the host cell is selected from " Gram-negative Proteobacteria<br>
Subgroup 3." "Gram-negative Proteobacteria Subgroup 3" is defined as the group of<br>
Proteobacteria of the following genera: Brevundimonas; Agrobacterium; Rhizobium;<br>
Sinorhizobium; Blastomonas; Sphingomonas; Alcaligenes; Burkholderia; Ralstonia;<br>
Addovorax; Hydrogenophaga; Methylobacter; Methylocaldum; Methylococcus;<br>
Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas;<br>
Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter;<br>
Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas. ,<br>
In another embodiment, the host cell is selected from "Gram-negative Proteobacteria<br>
Subgroup 4." "Gram-negative Proteobacteria Subgroup 4" is defined as the group of<br>
Proteobacteria of the following genera: Brevundimonas; Blastomonas; Sphingomonas ;<br>
Burkholderia; Ralstonia; Addovorax; Hydrogenophaga; Methylobacter; Methylocaldum;<br>
Methylococcus; Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera;<br>
Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter;<br>
Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
In an embodiment, the host cell is selected from "Gram-negative Proteobacteria<br>
Subgroup 5." "Gram-negative Proteobacteria Subgroup 5" is defined as the group of<br>
Proteobacteria of the following genera: Methylobacter; Methylocaldum; Methylococcus;<br>
Methylomicrobium; Methylomonas; Methylosarcina; Methylosphaera; Azomonas;<br>
Azorhizophilus; Azotobacter; Cellvibrio; Oligella; Pseudomonas; Teredinibacter;<br>
Francisella; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 6."<br>
"Gram-negative Proteobacteria Subgroup 6" is defined as the group of Proteobacteria of the<br>
following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia;<br>
Acidovorax; Hydrogenophaga; Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella;<br>
Pseudomonas; Teredinibacter; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 7."<br>
"Gram-negative Proteobacteria Subgroup 7" is defined as the group of Proteobacteria of the<br>
following genera: Azomonas; Azorhizophilus; Azotobacter; Cellvibrio; Oligella;<br>
Pseudomonas; Teredinibacter; Stenotrophomonas; Xanthomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 8."<br>
"Gram-negative Proteobacteria Subgroup 8" is defined as the group of Proteobacteria of the<br>
following genera: Brevundimonas; Blastomonas; Sphingomonas; Burkholderia; Ralstonia;<br>
Acidovorax; Hydrogenophaga; Pseudomonas; Stenotrophomonas; Xanthomonas; and<br>
Oceanimonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 9."<br>
"Gram-negative Proteobacteria Subgroup 9" is defined as the group of Proteobacteria of the<br>
following genera: Brevundimonas; Burkholderia; Ralstonia; Acidovorax; Hydrogenophaga;<br>
Pseudomonas; Stenotrophomonas; and Oceanimonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 10."<br>
"Gram-negative Proteobacteria Subgroup 10" is defined as the group of Proteobacteria of the<br>
following genera: Burkholderia; Ralstonia; Pseudomonas; Stenotrophomonas; and<br>
Xanthomonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 11."<br>
"Gram-negative Proteobacteria Subgroup 11" is defined as the group of Proteobacteria of the<br>
genera: Pseudomonas; Stenotrophomonas; and Xanthomonas. The host cell can be selected<br>
from "Gram-negative Proteobacteria Subgroup 12." "Gram-negative Proteobacteria Subgroup<br>
12" is defined as the group of Proteobacteria of the following genera: Burkholderia;<br>
Ralstonia; Pseudomonas. The host cell can be selected from "Gram-negative Proteobacteria<br>
Subgroup 13." "Gram-negative Proteobacteria Subgroup 13" is defined as the group of<br>
Proteobacteria of the following genera: Burkholderia; Ralstonia; Pseudomonas; and<br>
Xanthomonas. The host cell can be selected from "Gram-negative Proteobacteria Subgroup<br>
14." "Gram-negative Proteobacteria Subgroup 14" is defined as the group of Proteobacteria<br>
of the following genera: Pseudomonas and Xanthomonas. The host cell can be selected from<br>
"Gram-negative Proteobacteria Subgroup 15." "Gram-negative Proteobacteria Subgroup 15"<br>
is defined as the group of Proteobacteria of the genus Pseudomonas.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 16."<br>
"Gram-negative Proteobacteria Subgroup 16" is defined as the group of Proteobacteria of the<br>
following Pseudomonas species (with the ATCC or other deposit numbers of exemplary<br>
strain(s) shown in parenthesis): P. abietaniphila (ATCC 700689); P. aentginosa (ATCC<br>
10145); P. alcaligenes (ATCC 14909); P. anguilliseptica (ATCC 33660); P. citronellolis<br>
(ATCC 13674); P.flavescens (ATCC 51555); P. mendocina (ATCC 25411); P. nitroreducens<br>
(ATCC 33634); P. oleovorans (ATCC 8062); P. pseudoalcaligenes (ATCC 17440); P.<br>
resinovorans (ATCC 14235); P. straminea (ATCC 33636); P. agarici (ATCC 25941); P.<br>
alcaliphila; P. alginovora; P. andersonii; P. asplenii (ATCC 23835); P. azelaica (ATCC<br>
27162); P. beijerinckii (ATCC 19372); P. borealis; P. boreopolis (ATCC 33662); P.<br>
brassicacearum; P. butanovora (ATCC 43655); P. cellulosa (ATCC 55703); P. aurantiaca<br>
(ATCC 33663); P. chlororaphis (ATCC 9446, ATCC 13985, ATCC 17418, ATCC 17461);<br>
P.fragi (ATCC 4973); P. lundensis (ATCC 49968); P. taetrolens (ATCC 4683); P. cissicola<br>
(ATCC 33616); P. coronqfaciens; P. diterpeniphila; P. elongata (ATCC 10144); P.flectens<br>
(ATCC 12775); P. azotoformans; P. brenneri; P. cedrella; P. corrugata (ATCC 29736); P.<br>
extremorientalis; P. fluorescens (ATCC 35858); P. gessardii; P. libanensis; P. mandelii<br>
(ATCC 700871); P. marginalis (ATCC 10844); P. migulae; P. mucidolens (ATCC 4685); P.<br>
orientalis; P. rhodesiae; P. synxantha (ATCC 9890); P. tolaasii (ATCC 33618); P. veronii<br>
(ATCC 700474); P. frederiksbergensis; P. geniculata (ATCC 19374); P. gingeri; P.<br>
graminis; P. grimontii; P. halodenitnficans; P. halophila; P. hibiscicola (ATCC 19867); P.<br>
huttiensis (ATCC 14670); P. hydrogenovora; P. jessenii (ATCC 700870); P. kilonensis; P.<br>
lanceolata (ATCC 14669); P. lini; P. marginata (ATCC 25417); P. mephitica (ATCC<br>
33665); P. denitrificans (ATCC 19244); P. pertucinogena (ATCC 190); P. pictorum (ATCC<br>
23328); P. psychrophila; P.fulva (ATCC 31418); P. monteilii (ATCC 700476); P. mosselii;<br>
P. oryzihabitans (ATCC 43272); P. plecoglossicida (ATCC 700383); P. pwtefa (ATCC<br>
12633); P. reactans; P. spinosa (ATCC 14606); P. balearica; P. luteola (ATCC 43273); P.<br>
stutzeri (ATCC 17588); P. amygdali (ATCC 33614); P. avellanae (ATCC 700331); P.<br>
caricapapayae (ATCC 3361 Sj; P. cicAoni (ATCC 10857); P. ficuserectae (ATCC 35104);<br>
P. juscovaginae; P. we/zae (ATCC 33050); P. syringae (ATCC 19310); P. viridijlava (ATCC<br>
13223); P. thermocarboxydovorans (ATCC 35961); P. thermotolerans; P. thivervalensis; P.<br>
vancouverensis (ATCC 700688); P. wisconsinensis; and P. xiamenensis.<br>
The host cell can be selected from "Gram-negative Proteobacteria Subgroup 17."<br>
"Gram-negative Proteobacteria Subgroup 17" is defined as the group of Proteobacteria known<br>
in the art as the "fluorescent Pseudomonads" including those belonging, e.g., to the following<br>
Pseudomonas species: P. azotoformans; P. brenneri; P. cedrella; P. corrugata; P.<br>
extremorientalis; Pseudomonas fluorescens; P. gessardii; P. libanensis; Pseudomonas<br>
mandelii; P. marginalis; P. migulae; P. mucidolens; P. orientalis; P. rhodesiae; P.<br>
synxantha; P. tolaasii; and P. veronii.<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 18." "Gram(-)<br>
Proteobacteria Subgroup 18" is defined as the group of all subspecies, varieties, strains, and<br>
other sub-special units of the species P. fluorescens, including those belonging, e.g., to the<br>
following (with the ATCC or other deposit numbers of exemplary strain(s) shown in<br>
parenthesis): P. fluorescens biotype A, also called biovar 1 or biovar I (ATCC 13525); P.<br>
fluorescens biotype B, also called biovar 2 or biovar II (ATCC 17816); P. fluorescens biotype<br>
C, also called biovar 3 or biovar III (ATCC 17400); P. fluorescens biotype F, also called<br>
biovar 4 or biovar IV (ATCC 12983); P. fluorescens biotype G, also called biovar 5 or biovar<br>
V (ATCC 17518); P. fluorescens biovar VI; P. fluorescens PfO-1; P. fluorescens Pf-5 (ATCC<br>
BAA-477); P. fluorescens SBW25; and P. fluorescens subsp. cellulosa (NCIMB 10462).<br>
The host cell can be selected from "Gram(-) Proteobacteria Subgroup 19." "Gram(-)<br>
Proteobacteria Subgroup 19" is defined as the group of all strains of P. fluorescens biotype A.<br>
A particularl strain of this biotype is P. fluorescens strain MB 101 (see US Patent No.<br>
5,169,760 to Wilcox), and derivatives thereof. An example of a derivative thereof is P.<br>
fluorescens strain MB214, constructed by inserting into the MB 101 chromosomal asd<br>
(aspartate dehydrogenase gene) locus, a native E. coli PlacI-lacI-lacZYA construct (i.e. in<br>
which PlacZ was deleted).<br>
In one embodiment, the host cell is any of the Proteobacteria of the order<br>
Pseudomonadales. In a particular embodiment, the host cell is any of the Proteobacteria of<br>
the family Pseudomonadaceae.<br>
Additional P. fluorescens strains that can be used in the present invention include P.<br>
fluorescens Migula and P. fluorescens Loitokitok, having the following ATCC designations:<br>
(NCIB 8286); NRRL B-1244; NCIB 8865 strain CO1; NCIB 8866 strain CO2; 1291 (ATCC<br>
17458; IFO 15837; NCIB 8917; LA; NRRL B-1864; pyrrolidine; PW2 (ICMP 3966; NCPPB<br>
967; NRRL B-899); 13475; NCTC 10038; NRRL B-1603 (6; IFO 15840); 52-1C; CCEB<br>
488-A (BU 140); CCEB 553 (IBM 15/47); IAM 1008 (AHH-27); IAM 1055 (AHH-23); 1<br>
(IFO 15842); 12 (ATCC 25323; NIH 11; den Dooren de Jong 216); 18 (IFO 15833; WRRL P-<br>
7); 93 (TR-10); 108 (52-22; IFO 15832); 143 (IFO 15836; PL); 149 (2-40-40; IFO 15838);<br>
182 (IFO 3081; PJ 73); 184 (IFO 15830); 185 (W2 L-l); 186 (IFO 15829; PJ 79); 187<br>
(NCPPB 263); 188 (NCPPB 316); 189 (PJ227; 1208); 191 (IFO 15834; PJ 236; 22/1); 194<br>
(Klinge R-60; PJ 253); 196 (PJ 288); 197 (PJ 290); 198 (PJ 302); 201 (PJ 368); 202 (PJ 372);<br>
203 (PJ 376); 204 (IFO 15835; PJ 682); 205 (PJ 686); 206 (PJ 692); 207 (PJ 693); 208 (PJ<br>
722); 212 (PJ 832); 215 (PJ 849); 216 (PJ 885); 267 (B-9); 271 (B-1612); 401 (C71A; IFO<br>
15831; PJ 187); NRRL B-3178 (4; IFO 15841); KY 8521; 3081; 30-21; (IFO 3081); N; PYR;<br>
PW; D946-B83 (BU 2183; FERM-P 3328); P-2563 (FERM-P 2894; IFO 13658); IAM-1126<br>
(43F); M-l; A506 (A5-06); A505 (A5-05-1); A526 (A5-26); B69; 72; NRRL B-4290; PMW6<br>
(NCIB 11615); SC 12936; Al (IFO 15839); F 1847 (CDC-EB); F 1848 (CDC 93); NCIB<br>
10586; P17; F-12; AmMS 257; PRA25; 6133D02; 6519E01; Nl; SC15208; BNL-WVC;<br>
NCTC 2583 (NCIB 8194); H13; 1013 (ATCC 11251; CCEB 295); IFO 3903; 1062; or Pf-5.<br>
Fermentation<br>
The term "fermentation" includes both embodiments in which literal fermentation is<br>
employed and embodiments in which other, non-fermentative culture modes are employed.<br>
Fermentation may be performed at any scale. In one embodiment, the fermentation medium<br>
may be selected from among rich media, minimal media, and mineral salts media; a rich<br>
medium may be used, but is typically avoided. In another embodiment either a minimal<br>
medium or a mineral salts medium is selected. In still another embodiment, a minimal<br>
medium is selected.<br>
Mineral salts media consists of mineral salts and a carbon source such as, e.g.,<br>
glucose, sucrose, or glycerol. Examples of mineral salts media include, e.g., M9 medium,<br>
Pseudomonas medium (ATCC 179), Davis and Mingioli medium (see, BD Davis &amp; ES<br>
Mingioli (1950) J. Bact. 60:17-28). The mineral salts used to make mineral salts media<br>
include those selected from among, e.g., potassium phosphates, ammonium sulfate or<br>
chloride, magnesium sulfate or chloride, and trace minerals such as calcium chloride, borate,<br>
and sulfates of iron, copper, manganese, and zinc. Typically, no organic nitrogen source,<br>
such as peptone, tryptone, amino acids, or a yeast extract, is included in a mineral salts<br>
medium. Instead, an inorganic nitrogen source is used and this may be selected from among,<br>
e.g., ammonium salts, aqueous ammonia, and gaseous ammonia. A mineral salts medium will<br>
typically contain glucose as the carbon source, hi comparison to mineral salts media, minimal<br>
media can also contain mineral salts and a carbon source, but can be supplemented with, e.g.,<br>
low levels of amino acids, vitamins, peptones, or other ingredients, though these are added at<br>
very minimal levels.<br>
In one embodiment, media can be prepared using the components listed in Table 7<br>
below. The components can be added in the following order: first (NHLOHPCU, KHfePCXj and<br>
citric acid can be dissolved in approximately 30 liters of distilled water; then a solution of<br>
trace elements can be added, followed by the addition of an antifoam agent, such as Ucolub N<br>
115. Then, after heat sterilization (such as at approximately 121°C), sterile solutions of<br>
glucose MgSC4 and thiamine-HCL can be added. Control of pH at approximately 6.8 can be<br>
achieved using aqueous ammonia. Sterile distilled water can then be added to adjust the<br>
initial volume to 371 minus the glycerol stock (123 mL). The chemicals are commercially<br>
available from various suppliers, such as Merck. This media can allow for a high cell density<br>
cultivation (HCDC) for growth of Pseudomonas species and related bacteria. The HCDC can<br>
start as a batch process which is followed by a two-phase fed-batch cultivation. After<br>
unlimited growth in the batch part, growth can be controlled at a reduced specific growth rate<br>
over a period of 3 doubling times in which the biomass concentration can increased several<br>
fold. Further details of such cultivation procedures is described by Riesenberg, D et al.<br>
 (1991) "High cell density cultivation of Escherichia coli at controlled specific growth rate" J<br>
Biotechnol20(l):l7-27.<br>
(Table Removed)The expression system according to the present invention can be cultured in any<br>
fermentation format. For example, batch, fed-batch, semi-continuous, and continuous<br>
fermentation modes maybe employed herein.<br>
The expression systems according to the present invention are useful for transgene<br>
expression at any scale (i.e. volume) of fermentation. Thus, e.g., microliter-scale, centiliter<br>
scale, and deciliter scale fermentation volumes may be used; and 1 Liter scale and larger<br>
fermentation volumes can be used. In one embodiment, the fermentation volume will be at or<br>
above 1 Liter. In another embodiment, the fermentation volume will be at or above 5 Liters,<br>
10 Liters, 15 Liters, 20 Liters, 25 Liters, 50 Liters, 75 Liters, 100 Liters, 200 Liters, 500<br>
Liters, 1,000 Liters, 2,000 Liters, 5,000 Liters, lO.OOOLiters or 50,000 Liters.<br>
In the present invention, growth, culturing, and/or fermentation of the transformed<br>
host cells is performed within a temperature range permitting survival of the host cells, such<br>
as a temperature within the range of about 4°C to about 55°C, inclusive. In addition,<br>
"growth" is used to indicate both biological states of active cell division and/or enlargement,<br>
as well as biological states in which a non-dividing and/or non-enlarging cell is being<br>
metabolically sustained, the latter use being synonymous with the term "maintenance."<br>
Cell Density<br>
An additional advantage in using P. fluorescens in expressing recombinant<br>
mammalian proteins includes the capacity of P. fluorescens to be grown in high cell densities<br>
compared to E. coli or other bacterial expression systems. To this end, P. fluorescens<br>
expressions systems according to the present invention can provide a cell density of about 20<br>
g/L or more. The P. fluorescens expressions systems according to the present invention can<br>
likewise provide a cell density of at least about 70 g/L, as stated in terms of biomass per<br>
volume, the biomass being measured as dry cell weight.<br>
In one embodiment, the cell density will be at least 20 g/L. In another embodiment,<br>
the cell density will be at least 25 g/L, 30 g/L, 35 g/L, 40 g/L, 45 g/L, 50 g/L, 60 g/L, 70 g/L,<br>
80 g/L, 90 g/L., 100 g/L, 110 g/L, 120 g/L, 130 g/L, 140 g/L, or at least 150 g/L.<br>
In another embodiments, the cell density at induction will be between 20 g/L and 150<br>
g/L;, 20 g/L and 120 g/L; 20 g/L and 80 g/L; 25 g/L and 80 g/L; 30 g/L and 80 g/L; 35 g/L<br>
and 80 g/L; 40 g/L and 80 g/L; 45 g/L and 80 g/L; 50 g/L and 80 g/L; 50 g/L and 75 g/L; 50<br>
g/L and 70 g/L; 40 g/L and 80 g/L.<br>
Isolation and Purification ,<br>
The proteins of this invention may be isolated purified to substantial purity by<br>
standard techniques well known in the art, including including, but not limited to, ammonium<br>
sulfate or ethanol precipitation, acid extraction, anion or cation exchange chromatography,<br>
phosphocellulose chromatography, hydrophobia interaction chromatography, affinity<br>
chromatography, nickel chromatography, hydroxylapatite chromatography, reverse phase<br>
chromatography, lectin chromatography, preparative electrophoresis, detergent solubilization,<br>
selective precipitation with such substances as column chromatography, immunopurification<br>
methods, and others. For example, proteins having established molecular adhesion properties<br>
can be reversibly fused a ligand.. With the appropriate ligand, the protein can be selectively<br>
adsorbed to a purification column and then freed from the column in a relatively pure form.<br>
The fused protein is then removed by enzymatic activity. In addition, protein can be purified<br>
using immunoaffinity columns or Ni-NTA columns. General techniques are further described<br>
in, for example, R. Scopes (1982) Protein Purification: Principles and Practice, Springer-<br>
Verlag: N.Y.; Deutscher (1990) Guide to Protein Purification, Academic Press; U.S. Patent<br>
No. 4,511,503; S. Roe (2001) Protein Purification Techniques: A Practical Approach,<br>
Oxford Press; D. Bollag, et al. (1996) Protein Methods, Wiley-Lisa, Inc.; AK Patra et al.<br>
(2000) Protein Expr Purif, 18(2):182-92; and R. Mukhija, et al. (1995) Gene 165(2):303-6.<br>
See also, for example, Deutscher (1990) "Guide to Protein Purification," Methods in<br>
Enzymology vol. 182, and other volumes in this series; Coligan, et al. (1996 and periodic<br>
Supplements) Current Protocols in Protein Science, Wiley/Greene, NY; and manufacturer's<br>
literature on use of protein purification products, e.g., Pharmacia, Piscataway, N.J., or Bio-<br>
Rad, Richmond, Calif. Combination with recombinant techniques allow fusion to appropriate<br>
segments, e.g., to a FLAG sequence or an equivalent which can be fused via a proteaseremovable<br>
sequence. See also, for example: Hochuli (1989) Chemische Industrie 12:69-70;<br>
Hochuli (1990) "Purification of Recombinant Proteins with Metal Chelate Absorbent" in<br>
Setlow (ed.) Genetic Engineering, Principle and Methods 12:87-98, Plenum Press, NY; and<br>
Crowe, et al. (1992) QIAexpress: The High Level Expression &amp; Protein Purification System<br>
QUIAGEN, Inc., Chatsworth, Calif.<br>
Detection of the expressed protein is achieved by methods known in the art and<br>
include, for example, radioimmunoassays, Western blotting techniques or<br>
immunoprecipitation.<br>
The recombinantly produced and expressed enzyme can be recovered and purified<br>
from the recombinant cell cultures by numerous methods, for example, high performance<br>
liquid chromatography (HPLC) can be employed for final purification steps, as necessary.<br>
Certain proteins expressed in this invention may form insoluble aggregates ("inclusion<br>
bodies"). Several protocols are suitable for purification of proteins from, inclusion bodies.<br>
For example, purification of inclusion bodies typically involves the extraction, separation<br>
and/or purification of inclusion bodies by disruption of the host cells, e.g., by incubation in a<br>
buffer of 50 mM TRIS/HC1 pH 7.5, 50 mM NaCl, 5 mM MgCl2, 1 mM DTT, 0.1 mM ATP,<br>
and 1 mM PMSF. The cell suspension is typically lysed using 2-3 passages through a French<br>
Press. The cell suspension can also be homogenized using a Polytron (Brinkrnan Instruments)<br>
or sonicated on ice. Alternate methods of lysing bacteria are apparent to those of skill in the<br>
art (see, e.g., Sambrook, J., E. F. Fritsch and T. Maniatis eds. (1989) "Molecular Cloning: A<br>
Laboratory Manual", 2d ed., Cold Spring Harbor Laboratory Press; Ausubel et al., eds. (1994)<br>
Current Protocols in Molecular Biology).<br>
If necessary, the inclusion bodies can be solubilized, and the lysed cell suspension<br>
typically can be centrifuged to remove unwanted insoluble matter. Proteins that formed the<br>
inclusion bodies may be renatured by dilution or dialysis with a compatible buffer. Suitable<br>
solvents include, but are not limited to urea (from about 4 M to about 8 M), formamide (at<br>
least about 80%, volume/volume basis), and guanidine hydrochloride (from about 4 M to<br>
about 8 M). Although guanidine hydrochloride and similar agents are denaturants, this<br>
denaturation is not irreversible and renaturation may occur upon removal (by dialysis, for<br>
example) or dilution of the denaturant, allowing re-formation of immunologically and/or<br>
biologically active protein. Other suitable buffers are known to those skilled in the art.<br>
Alternatively, it is possible to purify the recombinant proteins from the host periplasm.<br>
After lysis of the host cell, when the recombinant protein is exported into the periplasm of the<br>
host cell, the periplasmic fraction of the bacteria can be isolated by cold osmotic shock in<br>
addition to other methods known to those skilled in the art. To isolate recorribinant proteins<br>
from the periplasm, for example, the bacterial cells can be centrifuged to form a pellet. The<br>
pellet can be resuspended in a buffer containing 20% sucrose. To lyse the cells, the bacteria<br>
can be centrifuged and the pellet can be resuspended in ice-cold 5 mM MgSC&gt;4 and kept in an<br>
ice bath for approximately 10 minutes. The cell suspension can be centrifuged and the<br>
supernatant decanted and saved. The recombinant proteins present in the supernatant can be<br>
separated from the host proteins by standard separation techniques well known to those of<br>
skill in the art.<br>
An initial salt fractionation can separate many of the unwanted host cell proteins (or<br>
proteins derived from the cell culture media) from the recombinant protein of interest. One<br>
such example can be ammonium sulfate. Ammonium sulfate precipitates proteins by<br>
effectively reducing the amount of water in the protein mixture. Proteins then precipitate on<br>
the basis of their solubility. The more hydrophobic a protein is, the more likely it is to<br>
precipitate at lower ammonium sulfate concentrations. A typical protocol includes adding<br>
saturated ammonium sulfate to a protein solution so that the resultant ammonium sulfate<br>
concentration is between 20-30%. This concentration will precipitate the most hydrophobic<br>
of proteins. The precipitate is then discarded (unless the protein of interest is hydrophobic)<br>
and ammonium sulfate is added to the supernatant to a concentration known to precipitate the<br>
protein of interest. The precipitate is then solubilized in buffer and the excess salt removed if<br>
necessary, either through dialysis or diafiltration. Other methods that rely on solubility of<br>
proteins, such as cold ethanol precipitation, are well known to those of skill in the art and can<br>
be used to fractionate complex protein mixtures.<br>
The molecular weight of a recombinant protein can be used to isolated it from proteins<br>
of greater and lesser size using ultrafiltration through membranes of different pore size (for<br>
example, Amicon or Millipore membranes). As a first step, the protein mixture can be<br>
ultrafiltered through a membrane with a pore size that has a lower molecular weight cut-off<br>
than the molecular weight of the protein of interest. The retentate of the ultrafiltration can<br>
then be ultrafiltered against a membrane with a molecular cut off greater than the molecular<br>
weight of the protein of interest. The recombinant protein will pass through the membrane<br>
into the filtrate. The filtrate can then be chromatographed as described below.<br>
Recombinant protiens can also be separated from other proteins on the basis of its size,<br>
net surface charge, hydrophobicity, and affinity for ligands. In addition, antibodies raised<br>
against proteins can be conjugated to column matrices and the proteins ianmunopurified. All<br>
of these methods are well known in the art. It will be apparent to one of skill that<br>
chromatographic techniques can be performed at any scale and using equipment from many<br>
different manufacturers (e.g., Pharmacia Biotech).<br>
Renaturation and Refolding<br>
Insoluble protein can be renatured or refolded to generate secondary and tieriary<br>
protein structure conformation. Protein refolding steps can be used, as necessary, in<br>
completing configuration of the recombinant product. Refolding and renaturation can be<br>
accomplished using an agent that is known in the art to promote dissociation/association of<br>
proteins. For example, the proteincan be incubated with dithiothreitol followed by incubation<br>
with oxidized glutathione disodium salt followed by incubation with a buffer containing a<br>
refolding agent such as urea.<br>
Recombinant protein can also be renatured, for example, by dialyzing it against<br>
phosphate-buffered saline (PBS) or 50 mM Na-acetate, pH 6 buffer plus 200 mM NaCl.<br>
Alternatively, the protein can be refolded while immobilized on a column, such as the Ni-<br>
NTA column by using a linear 6M-1M urea gradient in 500 mM NaCl, 20% glycerol, 20 mM<br>
Tris/HCl pH 7.4, containing protease inhibitors. The renaturation can be performed over a<br>
period of 1.5 hours or more. After renaturation the proteins can be eluted by the addition of<br>
250 mM immidazole. Irnmidazole can be removed by a final dialyzing step against PBS or 50<br>
mM sodium acetate pH 6 buffer plus 200 mM NaCl. The purified protein can be stored at 4°C<br>
or frozen at -80°C.<br>
Other methods include, for example, those that may be described in: MH Lee et al.<br>
(2002) Protein Expr. Purif. 25(l):166-73; W.K. Cho et al. (2000) J. Biotechnology 77(2-<br>
3): 169-78; Deutscher (1990) "Guide to Protein Purification," Methods in Enzymology vol.<br>
182, and other volumes in this series; Coligan, et al. (1996 and periodic Supplements) Current<br>
Protocols in Protein Science, Wiley/Greene, NY; S. Roe (2001) Protein Purification<br>
Techniques: A Practical Approach, Oxford Press; D. Bollag, et al. (1996) Protein Methods,<br>
Wiley-Lisa, Inc.<br>
Active Protein or Peptide Analysis<br>
Typically, an "active" protein includes proteins that have a biological function or<br>
biological effect comparable to the corresponding native protein. In the context of proteins<br>
this typically means that a polynucleotide or polypeptide comprises a biological function or<br>
effect that has at least about 20%, about 50%, about 60%, about 70%, about 75%, about 80%,<br>
about 85%, about 90%, about 95%, about 98% or 100% biological function compared to the<br>
corresponding native protein using standard parameters. The determination of protein activity<br>
can be performed utilizing corresponding standard, targeted comparative biological assays for<br>
particular proteins. One indication that a recombinant protein biological function or effect is<br>
that the recombinant polypeptide is immnunologically cross reactive with the native<br>
polypeptide.<br>
Active proteins typically have a specific activity of at least 20%, 30%, 40%, 50%,<br>
60%, 70%, 80%, 90%, or 95% that of the native mammalian protein. Further, the substrate<br>
specificity (koat/Kn,) is optionally substantially similar to the native mammalian protein.<br>
Typically, kcat/Km will be at least 30%, 40%, 50%, 60%, 70%, 80%, or 90% that of the native<br>
protein. Methods of assaying and quantifying measures of protein and peptide activity and<br>
substrate specificity (kcat/Km), are well known to those of skill in the art.<br>
The activity of a recombinant -mammalian protein can be measured by any protein<br>
specific conventional or standard in vitro or in vivo assay known in the art. The activity of the<br>
Pseudonmonas produced recombinant mammalian protein can be compared with the activity<br>
of the corresponding native mammalian protein to determine whether the recombinant<br>
mammalian protein exhibits substantially similar or equivalent activity to the activity<br>
generally observed in the native protein under the same or similar physiological conditions.<br>
The activity of the recombinant protein can be compared with a previously established<br>
native protein standard activity. Alternatively, the activity of the recombinant protein can be<br>
determined in a simultaneous, or sunstantially simultaneous, comparative assay with the<br>
native protein. For example, an in vitro assays can be used to determine any detectable<br>
interaction between a recombinant protein and a target, e.g. between an expressed enzyme and<br>
substrate, between expressed hormone and hormone receptor, between expressed antibody<br>
and antigen, etc. Such detection can include the measurement of colorimetric changes,<br>
proliferation changes, cell death, cell repelling, changes in radioactivity, changes in solubility,<br>
changes in molecular weight as measured by gel electrophoresis and/or gel exclusion<br>
methods.phosphorylation abilities, antibody specificity assays such as ELISA assays, etc. In<br>
addition, in vivo assays include, but are not limited to, assays to detect physiological effects of<br>
the Pseudomonas produced protein in comparison to physiological effects of the native<br>
protein, e.g. weight gain, change in electrolyte balance, change in blood clotting time, changes<br>
in clot dissolution and the induction of antigenic response. Generally, any in vitro or in vivo<br>
assay can be used to determine the active nature of the Pseudomonas produced recombinant<br>
mammalian protein that allows for a comparative analysis to the native protein so long as<br>
such activity is assayable. Alternatively, the proteins produced in the present invention can be<br>
assayed for the ability to stimulate or inhibit interaction between the protein and a molecule<br>
that normally interacts with the protein, e.g. a substrate or a component of the signal pathway<br>
that the native protein normally interacts. Such assays can typically include the steps of<br>
combining the protein with a substrate molecule under conditions that allow the protein to<br>
interact with the target molecule, and detect the biochemical consequence of the interaction<br>
with the protein and the target molecule.<br>
Assays that can be utilized to determine protein activity are described, for example, in:<br>
Ralph, P. J., et al. (1984) J. Immunol. 132:1858; Saiki et al. (1981) J. Immunol. 127:1044;<br>
Steward, W. E. II (1980) The Interferon Systems. Springer-Verlag, Vienna and New York;<br>
Broxmeyer, H. E., et al. (1982) Blood 60:595; Sambrook, J., E. F. Fritsch and T. Maniatis eds.<br>
(1989) "Molecular Cloning: A Laboratory Manual", 2d ed., Cold Spring Harbor Laboratory<br>
Press; Berger, S. L. and A. R. Kimmel eds. (1987) "Methods in Enzymology: Guide to<br>
Molecular Cloning Techniques", Academic Press; AK Patra et al. (2000) Protein Expr Purif<br>
18(2): 182-92; Kodama et al. (1986) J. Biochem. 99:1465-1472; Stewart et al. (1993) Proc.<br>
Nat'lAcad. Sci. USA 90:5209-5213; Lombillo etal. (1995)/. CellBiol. 128:107-115; Vale et<br>
al. (1985) Cell 42:39-50.<br>
EXAMPLES<br>
Bacterial Strains and Growth Conditions.<br>
Unless otherwise specified, all strains used for all Pseudomonas expression testing<br>
were based on P. fluorescens strain MB 101. E. coli strains JM109 (Promega), XL2 Blue<br>
(Stratagene) or Top 10 (Invitrogen) were used for general cloning. For E. coli expression<br>
studies, BL21(DE3) Gold was used. P. fluorescens strains were grown in either LB or<br>
minimal salts medium supplemented with 15 ug/mL tetracycline and 30 ug/mL kanamycin as<br>
needed at 30°C. E. coli strains were grown in LB supplemented with 30 ug/mL kanamycin<br>
and/or 15 ug/mL chloramphenicol, or 15 ug/mL tetracycline as needed at 37°C. Cells were<br>
induced with 0.3mM IPTG following growth phase.<br>
Protein Activity Detection (ELISA Assay)<br>
Plates were coated by adding 200 \\L of the p-galactosidase solution at 10 |ig/mL in<br>
PBS (pH 7.6) to each well of the microtiter plate. Plates were incubated at room temperature<br>
for 16 hrs, then washed 3 times with 200 |aL PBS + 0.1% Tween-20 (PBS-T). Primary<br>
antibody was diluted in PBS with 2% nonfat dry milk (w/v). 200 of the diluted antibody<br>
was added to each well and incubated at room temperature for 1 hr. The plates were then<br>
washed 4 times with 200 PBS-T. The secondary antibody was also diluted in PBS with<br>
2% non fat dry milk (w/v) and to each well, 200 pL was added and incubated at room<br>
temperature for 1.5-2 hours. The plates were then washed 4 times with PBS-T. A tertiary<br>
antibody is used to detect the scFv antibodies: alkaline phosphatase conjugated sheep antimouse<br>
antibody (Sigma-Aldrich, St. Louis, MO, USA cat #A5324). To each desired well was<br>
added 200 \iL of diluted antibody solution (or PBS-T) and incubated at room temperature for<br>
1.5 hours. The plates were then washed 4 times with PBS-T. To each well was added 200 ul<br>
of the freshly prepared Sigma Fast pNPP substrate (Sigma catalogue #R-2770). After 30<br>
minutes, the reaction was stopped by adding 50 3M NaOH to each well and absorbance<br>
was read at 405 nm.<br>
Fermentation<br>
The inoculum for the fermentor culture for P.fluorescens is generated by inoculating a<br>
shake flask containing 600 mL of a chemically defined medium supplemented with yeast<br>
extract and dextrose. Tetracycline is typically added to ensure maintenance of the<br>
recombinant plasmid in the starter culture during its overnight incubation as well as in the<br>
fermentor. The shake flask culture is then aseptically transferred to a 20L fermentor<br>
containing a chemically defined medium designed to support a high biomass, without yeast<br>
extract supplementation. Oxygen is maintained at a positive level in the liquid culture by<br>
regulating the air flow into the fermentor and the agitator mixing rate; the pH is maintained at<br>
greater than 6.0 through the addition of aqueous ammonia. The fed-batch high density<br>
fermentation process is divided into an initial growth phase of approximately 24h and gene<br>
expression (induction) phase in which an inducer is added to initiate recombinant gene<br>
expression. Glucose, in the form of corn syrup, is fed throughout the fermentation process at<br>
limiting concentrations. The target cell density for initiating the induction phase is typically<br>
150 OD units at 575nm. The induction phase of the fermentation is typically allowed to go for<br>
approximately 45 to 55 hours. During this phase, samples are withdrawn from the fermentor<br>
for various analyses to determine the level of target gene expression, cell density, etc.<br>
For each fermentation experiment for E. coli, a frozen glycerol stock is removed from<br>
-80°C storage, thawed and diluted before inoculating a shake flask containing 600 mL of LB<br>
broth supplemented with kanamycin. The shake flask culture is incubated at 37°C with<br>
shaking at 300 rpm overnight and then aseptically transferred to a 20L fermentor containing<br>
complex medium. Temperature in the fermentor is maintained at 37°C, pH at 7 through the<br>
addition of aqueous ammonia and phosphoric acid, and dissolved oxygen at greater than 20%.<br>
After a brief initial batch phase, glycerol is fed at rates increased stepwise to maintain excess<br>
carbon. When the cell density reaches 24-28 OD units at 600nm, recombinant expression is<br>
effected by addition of an inducer, such as isopropyl-thiogalactoside (IPTG). The induction<br>
phase of the fermentation typically continues for approximately 3 to 5 hours as the fermentor<br>
reached volumetric capacity or as the growth rate began to decrease significantly. During this<br>
phase, samples are withdrawn from the fermentor for various analyses to determine the level<br>
of target gene expression, cell density, etc.<br>
Cellfractionation andSDS-PAGE analysis.<br>
Samples are normalized to A575=30, and 1 mL normalized culture is pelleted. Cells<br>
are resuspended in ImL lysis buffer (50 mM Tris base; 200mM NaCl; 5% v/v glycerol; 20<br>
mM EDTA disodium salt; 0.5% v/v Triton X-100; 1 mM DTT). A protease inhibitor cocktail<br>
specific for bacterial lysates (Sigma#P8465) is added to a IX concentration. The resuspended<br>
cells are added to a 2 ml screw cap microfuge tube approximately % full with 0.1 mm glass<br>
beads and the cells are mechanically lysed using 4, 1 minute incubations in a BioSpec bead<br>
mill at the highest setting. Cells are kept on ice between incubations. Approximately lOOuL of<br>
lysed cell solution is removed from beads, transferred into a new tube and pelleted. The<br>
supernatant (soluble fraction) is removed to a new tube. The pellet (insoluble fraction) is<br>
resuspended in an equal volume (100 uL) of lysis buffer plus protease inhibitor. Five uL of<br>
each sample is added to 5uL of 2X LDS loading buffer (Invitrogen) and loaded onto a 4-12%<br>
or 10% Bis-Tris NuPAGE gel (Invitrogen) and run in either IX MES or IX MOPS buffer as<br>
indicated.<br>
Example 1: Expression ofscFV in the Cytoplasm<br>
Single chain antibody fragments (scFV) are finding increased use as diagnostic and<br>
therapeutic agents. These relatively small proteins are made by fusing together genes coding<br>
the variable light and heavy chains of an immunoglobulin.<br>
Cloning ofGalB scFv<br>
The GallS scFv gene (Genbank accession number AF238290), cloned into the phage<br>
display vector pCANTAB6, (see P Martineau et al. (1998) /. Mol. Biol. 280(1): 117-27) was<br>
used as template to amplify a 774 base pair product, which was subsequently cloned into the<br>
pCR2.1 TOPO vector (Invitrogen, Carlsbad, CA, USA). The scFv gene was excised from the<br>
TOPO vector with Spel and Sail restriction enzymes (New England Biolabs, Beverly, MA,<br>
USA) and cloned into the Spel and Xhol sites of the P. fluorescens vector pMYC1803,<br>
downstream of the Ptac promoter, to produce pDOWlll?. The resulting plasmids were<br>
electroporated into P. fluorescens. The GallS gene was cloned into the pET24d+ expression<br>
vector (Novagen, Madison, WI, USA), following amplification such that Sail and Ncol sites<br>
flanked the coding sequence. The PCR products were digested with Sail and Ncol and cloned<br>
into the same sites of pET24d+ vector downstream of the T7 promoter. The newly formed<br>
construct was then used to transform XL2 Blue competent cells. Once sequence was<br>
confirmed, the DNA construct was used to transform BL21(DE3) Gold (Stratagene, San<br>
Diego, CA, USA) for expression.<br>
Expression of a Single Chain Antibody Fragment (scFv) in E. coli and P. fluorescens<br>
scFv molecules were expressed in both E. coli and P. fluorescens, among them an<br>
scFv with binding activity to the E. coli protein ji-galactosidase single chain antibody gal!3<br>
(P. Martmeau et al., "Expression of an antibody fragment at high levels in the bacterial<br>
cytoplasm, " J. Mol. Biol. 280(1):117-27 (1998)). P. fluorescens expressed about six-fold<br>
more protein than E. coli during 20L fermentation, with 3.1 g/L yield in P. fluorescens and<br>
0.5 g/L yield in E. coli as determined by SDS-PAGE and densitometry (see Table 8). P.<br>
fluorescens expressed about 96% soluble protein, whereas E. coli expresses only 48% soluble<br>
protein.<br>
(Table Removed)<br><br>
Material purified from both expression systems was found to be active in an enzyme-linked<br>
immunosorbant assay (ELISA) as shown in Figure 2. Material was also purified from the<br>
soluble fraction only of equal lysate volumes from lysates of both strains using affinity<br>
chromatography. Finally, the overall volumetric recovery for the P. fluorescens process is<br>
approximately 20 fold more efficient than for E. coli, 1.34 g/L vs. 0.07 g/L.<br>
Example 2; Expression of human y-IFN in the Cytoplasm<br>
Cloning of Human gamma-Interferon<br>
Human gamma interferon (hu-ylFN, Genbank accession XI3274) was amplified from<br>
a human spleen cDNA library (Invitrogen, Carlsbad, CA, USA; catalogue #10425-015) such<br>
that it lacked the native secretion signal, with the N-terminus of the recombinant y-IFN<br>
beginning as Met-Cys-Tyr-Cys-Gln-Asp-Pro as described in PW Gray et al. (1982) Nature<br>
298:859-63. The resulting product was cloned into the pCR2.1 TOPO vector and the<br>
sequence was confirmed. The hu-yIFN gene was excised from the TOPO vector with Spel<br>
and Xhol restriction enzymes and cloned into the same sites of pMYC1803. In a separate<br>
reaction, hu-ylFN was amplified such that Afllll and Xhol sites flanked that coding sequence.<br>
The resulting fragment was cloned into the TOPO-TA vector (Invitrogen) and transformed<br>
into chemically competent E. coli JM109 cells (Promega, Madison, WI, USA). The gene was<br>
isolated by digesting with Afllll and Xhol (New England Biolabs), cloned into the Ncol and<br>
Xhol sites of pET24d+ (Novagen, Madison, WI, USA) downstream of the T7 promoter, and<br>
transformed into JM109. A positive clone was transformed into E. coli BL21(DE3) cells<br>
(Novagen) to test for expression.<br>
Human gamma-Interferon Purification<br>
Frozen cell paste from P. fluorescens cultures was thawed and re-suspended in lysis<br>
buffer (50 mM potassium phosphate, pH 7.2 containing 50 mM NaCl, 10 mM EDTA<br>
(ethylenediaminetetraacetic acid, catalog number BPH8-500, Fisher Scientific, Springfield,<br>
NJ, USA), 1 mM PMSF (phenylmethylsulfonyl fluoride, catalog number P-7626, Sigma, St.<br>
Louis, MO), ImM dithiothreitol (catalog number D-0632, Sigma), and 1 mM benzamidine<br>
(catalog number B-6506, Sigma)) at a ratio of about 1 gram cell paste per 2 mL lysis buffer.<br>
Cells were broken by three passages through a microfluidizer (model HOY, Microfluidics<br>
Corporation, Newton, MA, USA). Cell debris and unbroken cells were removed by<br>
centrifugation (for 60 min at 23,708 x g and 4°C using a Beckman Coulter centrifuge; model<br>
JA 25.50, Beckman Coulter, Inc., Fullerton, CA, USA). The resulting supernatant (cell-free<br>
extracts) was clarified by adding 10% w/v diatomaceous earth (Celite product, World<br>
Minerals, Inc., Goleta, CA, USA) and passing the result through a paper filter (Whatman 1,<br>
catalog number 1001-150, Whatman Paper Ltd., Maidstone, Kent, UK)) with vacuum<br>
filtration.<br>
Clarified cell extracts were applied to a 3.2 cm x 13.5 cm chromatography column of<br>
SP-Sepharose FAST FLOW (6% cross-linked agarose bead material; catalog number 17-<br>
0709-10, Amersham Biosciences, Piscataway, NJ, USA) equilibrated in buffer A, at a flow<br>
rate of 0.5 mL/min. The composition of Buffer A was: 50mM HEPES, pH 7.8 (i.e. N-(2-<br>
hydroxyethyl)piperazine)"N'-(2-ethanesulfonic acid), from Fisher Scientific, catalog number<br>
BP-310-100), 50mM NaCl, ImM EDTA, and 0.02% sodium azide (catalog number 71289,<br>
Sigma Chemical Co.). After loading, the column was washed with 3 column volumes<br>
(column volume = 108 mL) buffer A and 5 column volumes of buffer A containing 0.4M<br>
NaCl. The column was further developed by applying a gradient of 0.4 M to 1 M NaCl in the<br>
same buffer at a flow rate of 2 mL/min for a total of 7 column volumes. Fractions containing<br>
pure IFN-y were then pooled and dialyzed against IX PBS (phosphate-buffered saline, pH<br>
7.2) at 4°C. Protein was concentrated by ultrafiltration (using a YM30 ultrafiltration<br>
membrane; catalog no. 13722, from Millipore, Bedford, MA USA), then frozen in liquid<br>
nitrogen and stored at 80°C.<br>
Expression of Human y-Interferon in E. coli and P. fluorescens<br>
Human y-interferon is produced commercially by fermentation of E. coli expressing<br>
the y-IFN gene. The protein is expressed cytoplasmically in an insoluble and inactive form.<br>
In order to produce the recombinant polypeptide as an active pharmaceutical ingredient, the<br>
interferon must be recovered, solubilized, refolded, and then purified. All these unit<br>
operations add greatly to the cost of goods (COGs) for this protein. A human spleen cDNA<br>
library was used as a template to amplify the ylFN cDNA without the native signal sequence<br>
and clone into E. coli and P. fluorescens expression vectors. P. fluorescens construct<br>
produced ~4g/L of ylFN protein during a typical 20L fermentation reaction. SDS-PAGE and<br>
Western analyses of soluble and insoluble fractions show that the majority of the protein<br>
(95%) is present in the soluble fraction. Figure 1 shows that hu-y-IFN purified from the<br>
soluble fraction of P. fluorescens samples displays activity comparable to a commercially<br>
available standard. Figure 5 and Table 9 show a comparison of expression of y-IFN between<br>
E. coli and P. fluorescens expression systems.<br>
(Table Removed)<br><br>
Cell lines and media: Hela cells (catalogue no. CCL-2) and encephalomyocarditis<br>
virus (ECMV, catalogue no. VR-129B) were obtained from the American Type Culture<br>
Collection (Manassas, VA). HeLa cells were maintained in Eagles Modified Essential<br>
Medium (Cellgro EMEM, Mediatech, Herdon, VA, USA) with 10% fetal bovine serum<br>
(Gibco, Invitrogen, Carlsbad, CA, USA) at 37°C/ 5% CO2.<br>
The activity of purified hu-ylFN was assayed using a viral inhibition assay as<br>
previously described (JA Lewis (1987) in Lymphokines and Interferons: A Practical<br>
Approach MJ Clemens et al. (eds.) (IRL Press Ltd, Oxford, England). Briefly, HeLa cells<br>
were seeded in a 96-well microtiter plate at 3 X 104 per well. After 24 hours, purified <br>
isolated from P. jluorescens, or E. coli recombinant hu-ylFN (from R&amp;D Systems,<br>
Minneapolis, MN, USA), was added to triplicate wells at 0, 0.01 or 0.05 ng per well. After<br>
preincubating the cells with hu-ylFN for 24 hours, ECMV was added at varying dilutions to<br>
sets of triplicate wells. The cells were incubated for 5 days, after which cell viability was<br>
measured using a cell proliferation ELISA that monitors 5-bromo-2'-deoxyuridine<br>
incorporation (catalogue no. 1647229, Roche Molecular Biochemicals, Indianapolis, IN,<br>
USA). Results are expressed as absorbance units, with greater absorbance resulting from the<br>
presences of a greater number of actively dividing (live) cells.<br>
Example 3: Expression of hGH in the Cytoplasm<br>
Primers were designed to amplify human growth hormone (hGH) from human cDNA<br>
libraries. For this study, hGH was amplified using AmpliTaq polymerase (Perkin Elmer)<br>
according to the manufacturer's protocol, using the above plasmid as template and primers<br>
ELVIrev and hgh-sig, with a PCR cycling profile of 95°C 2 minutes (95°C 60 seconds 428C<br>
120 seconds 72°C 3 minutes) 25X. The resulting product was purified using "Wizard PCR<br>
DNA purification kit (Promega), digested with Spel and Xhol restriction enzymes (New<br>
England Biolabs) and cloned into the same sites of pMYC1803 (see Figure 3). A mutation<br>
found in the amplified hGH was corrected by using the hgh-sigcorr primer with ELVIrev and<br>
repeating the PCR and cloning procedures.<br>
Primers used to clone hGH.<br>
Purification of hGH<br>
Following 20L fermentation, hGH was purified from the insoluble fraction of E. coli<br>
and P. fluorescens cells, with the exception that during DEAE FF elution a gradient from 0 to<br>
0.5M NaCl was used in place of a 0.25M NaCl step.<br>
Expression of human growth hormone in E. coli vs. P. fluorescens.<br>
The cDNA encoding human growth hormone was amplified from a human pituitary<br>
cDNA library. The native secretion signal sequence was removed, and an N-terminal<br>
methionine was engineered into the constructs for microbial expression. For E. coli<br>
expression, the pET25 vector containing the hGH gene was transformed into BL21(DE3),<br>
which contains an integrated T7 polymerase gene necessary for hGH transcription. P.<br>
fluorescens expression studies were carried out in the MB214 strain, which contains an<br>
integrated lacl gene to control expression from the Ptac promoter. Both expression systems<br>
were evaluated at the 20L fermentation scale. As shown in Table 10, P. fluorescens (Pf)<br>
outperformed E. coli (EC) in the amount of protein produced per gram of dry biomass (1.6X<br>
as much).<br>
(Table Removed)<br><br>
Cell fractionation and SDS-PAGE analysis show that hGH is found in the insoluble fraction<br>
in both expression systems (Figure 4). Surprisingly, approximately 7X more hGH monomer<br>
was purified from P. fluorescens, compared to E. coli, despite a difference of only 1.6X in<br>
protein production per gram of dry biomass.<br>
(Table Removed)<br><br>
Example 4: Expression of Proteins in thePeriplasm<br>
Characterization of Secretion Signal Peptides<br>
Pseudomonas fluorescens secretion signal peptides were discovered by formation and<br>
expression of alkaline phosphatase (phoA) coding sequence-genomic DNA fusions and are<br>
described in more detail in U.S. Application No. 10/996,007, filed November 22, 2004. Six<br>
of the expressed fusions were further characterized as follows.<br>
The cleavage site for the signal sequences for the secreted genes identified as phoA<br>
fusions was deduced by comparison to homolgous proteins from other Pseudomonads, by the<br>
SPScan program (Menne et al, 2000). The cleavage site of the putative lipoprotein was<br>
deduced by comparison to signal peptidase II motifs; signal peptidase II specifically cleaves<br>
the signal sequences of lipoproteins. All six of the signal peptides were analyzed using<br>
SignalP (a software program for analysis of putative signal peptides; available from the<br>
Center for Biological Sequence Analysis of the Technical University of Denmark, at<br>
http://www.cbs.dtu.dklservices/SignalP/.) Also see, Nielson et al. (1997) Protein Engineering<br>
10:1-6. hi some cases, a supplementary source was used to further characterize the identity of<br>
the signal peptide. This information is present in Table 12.<br>
(Table Removed)<br><br>
Western analysis of the phoA fusion proteins to detect fusion proteins<br>
To analyze whether the fusion proteins were produced, Western analysis with<br>
antibody to alkaline phosphatase was carried out on cultures separated by centrirugation into a<br>
whole-cell fraction (cytoplasm and periplasm) and a cell-free broth fraction. Of five strains<br>
for which the site of insertion was determined, four (putative azurin, putative phosphate<br>
binding protein, putative periplasmic lipoprotein B, putative Fe(III) binding protein) produced<br>
a fusion protein of the expected size, and one (putative oprE protein) produced a protein about<br>
40 kD smaller than predicted, and one (putative Lys-Arg-Orn binding protein) produced a<br>
protein about 20 kD smaller than predicted.<br>
Proteins were separated by SDS-PAGE and were transferred to nitrocellulose<br>
membrane at 40 V for one hour using the Xcell SureLockTM Mini-Cell and XCell IITM Blot<br>
Module (Invitrogen). Western experiments were performed using the instruction provided<br>
from SuperSignal West HisProbeTM Kit (Pierce).<br>
Construction, Expression, and Characterization of a pbp-hGH Fusion<br>
The P. fluorescens phosphate binding protein secretion leader was fused to the Nterminus<br>
of the mature domain of the human growth hormone (hGH) gene and tested for<br>
expression and secretion.<br>
The pbp signal-sequence coding region was PCR amplified from a clone of the P.<br>
fluorescens pbp signal sequence as template, using sig_pbp for<br>
(gctctagaggaggtaacttatgaaactgaaacg) and pbpjhgh (gggaatggttgggaaggccaccgcgttggc)<br>
primers, then gel-purified. This resulted in production of an oligonucleotide fragment<br>
containing the pbp signal peptide CDS and the coding sequence for the 5' end of the mature<br>
domain of hGH.<br>
A cDNA encoding the human growth hormone was PCR-amplified from a human<br>
pituitary cDNA library (Clontech, Palo Alto CA) using primers ELVIfor<br>
(agagaactagtaaaaaggagaaatccatggctacaggctcccggacgtcc) and ELVIrev<br>
(agagactcgagtcattagaagccacagctgccctccac), which were designed to amplify only the mature<br>
domain of hGH, and cloned into pMYCl 803/SpeI Xhol, forming pDOW2400. The mature<br>
hGH gene was amplified from pDOW2400, using primers pbp_hgh_revcomp<br>
(gccaacgcggtggccttcccaaccattccc) and hgh_rev (agagactcgagtcattagaagc<br>
cacagctgccctccacagagcggcac), then purified with Strataprep columns (Stratagene) to remove<br>
primers and other reaction components. To make the polynucleotide encoding the pbp-hGH<br>
fusion, the two PCR reactions were combined and amplified again with sig_pbp for and<br>
hgh_rev in order to link the two pieces. The expected 681 bp fragment was purified with<br>
Strataprep as above, restriction digested with Xbal and Xhol and ligated to dephosphorylated<br>
pDOW1269/XhoISpeI to form pDOW 1323-10, placing pbp-hGH under control of the tac<br>
promoter in a vector analogous to pMYC1803, but with a pyrF selectable marker in place of a<br>
tetR tetracycline resistance marker gene. The ligation mix was transformed into MB 101 pyrF<br>
proC lacl21. Inserts were sequenced by The Dow Chemical Company using the method<br>
described above. The DNA and amino acid sequence of this fusion is presented in (Figure 10)<br>
and (Figure 11), respectively.<br>
The resulting strains were tested first at the shake flask scale. Induced bands of the<br>
expected size for processed and unprocessed (22.2 kDa and 24.5 kDa, respectively) were<br>
detected by SDS-PAGE. About half of the protein was processed (indicating localization to<br>
the periplasm), and of the processed about half was in the soluble fraction and half in the<br>
insoluble fraction. Expression studies were scaled up to 20-L bioreactors. Densitometry of<br>
the Coomassie-stained SDS-PAGE gels showed that 18% of the total hGH produced was<br>
processed and soluble. The strain produced 3.2 g/L of all forms of hGH; processed and<br>
soluble was 0.6 g/L.<br>
Construction, Expression, and Characterization ofpbp-scFv Fusion<br>
63<br>
he putative 24 amino acid signal sequence of phosphate binding protein (i.e.<br>
including Metl) was fused to the open reading frame of the ga!2 scFv gene (gal2) at the +2<br>
amino acid (Ala) position. See Figure 8 and Figure 9. The signal sequence appears to be<br>
processed, indicating secretion to the periplasm. Moreover, there is secretion to the broth, in<br>
that protein was detected in the cell free culture supernatant. Surprisingly, fusion to the<br>
phosphate binding protein signal sequence appears to improve expression of ga!2 scFv in P.<br>
fluorescens. Without the secretion signal fused at the amino terminus, expression of ga!2<br>
scFv was not detectable.<br>
Cloning of GaI2<br>
PCR was performed using primers sig_pbp for (above) and pbp_ga!2SOE rev<br>
which contains a reverse complement of pbp_ga!2SOE for<br>
, and using a plasmid encoding the P. fluorescens pbp secretion<br>
signal peptide as template. This resulted in production of an oligonucleotide fragment<br>
containing the pbp signal peptide coding sequence (CDS) and a CDS for the 5' end of the ga!2<br>
single chain antibody (scAb or scFv).<br>
PCR was performed using primers pbp_ga!2SOE for and scFv2rev<br>
(acgcgtcgacttattaatggtg atgatggtgatgtgcggccgcacgtttgatc), and using a ga!2-encoding<br>
polynucleotide as template. This resulted in production of a polynucleotide fragment<br>
containing a CDS encoding the 3'end of the pbp signal peptide and the open reading frame<br>
(ORE) encoding ga!2.<br>
Reaction products were purified. About 15ng of each was used as a "template" DNA<br>
in a further PCR reaction using primers sig__pbp_for and scFv2rev. This resulted in<br>
production of a nucleic acid fragment with the pbp signal peptide CDS fused to the ga!2<br>
coding sequence.<br>
The predicted -1 amino acid of the signal sequence (that is the last amino acid prior to<br>
the proposed cleavage site) was fused to the +2 amino acid of the ga!2 scFv (Ala). The<br>
resulting fusion was cloned into the P. fluorescens vector pMYC1803 under control of the<br>
Ptac promoter to produce plasmid and pDOWl 123 (pbp:ga!2). The plasmid was transformed<br>
into P. fluorescens strain MB 101 carrying plasmid pCN51-lacI (described in U.S.<br>
Application No. 10/994,138, filed November 19,2004.<br>
Fusion of the putative phosphate binding protein signal sequence to gal2 scFv<br>
The phosphate binding protein signal sequence was fused to a single chain antibody<br>
gene and tested for secretion to the periplasm and/or to the culture supernatant.<br>
(Table Removed)<br><br>
The resulting strains were tested first at the shake flask scale. Induced bands of the<br>
expected size for unprocessed and processed ga!2 (29kDa and 27kDa) were detected via SDSPAGE<br>
in the insoluble protein fraction (data not shown). Expression studies were scaled up<br>
to 20L fermentation. Again, SDS-PAGE analysis showed that the majority of the induced<br>
protein is found in the insoluble protein fraction.<br>
The Western analysis also indicated that some processed ga!2 is present in the soluble<br>
protein fraction for pbp:ga!2 (pDOW1123). Western analysis of periplasmic fractions<br>
prepared from strains carrying pDOW 1123 (using the Epicentre periplast kit) showed the<br>
presence of soluble ga!2 protein.<br>
Recombinant was isolated from the cell extract of a shake flask experiment<br>
using the Ni-NTA protocol, then refolded as described in P. Marrineau et al., J Mol.<br>
Biol. 280:117-127 (1998). This antibody was found to be active against p-galactosidase in an<br>
ELISA assay.<br><br><br><br><br><br>
WE CLAM:<br>
1.	A method for producing a recombinant mammalian protein in a Pseudomonas fluoresceins host<br>
cell comprising:<br>
transforming a Pseudomonas fluorescens.host cell with a nucleic acid encoding a<br>
recombinant mammalian protein; growing the cell under conditions that allow expression of the recombinant mammalian<br>
protein; and isolating the recombinant mammalian protein; wherein the recombinant mammalian protein is produced as 5-75% total cell protein, wherein the recombinant mammalian protein is present in the host cell in a soluble form.<br>
2.	The method as claimed in claim 1 wherein the protein is expressed at an increased level when compared to an expression level of the protein under substantially comparable conditions in an E. coli expression system.<br>
3.	The method as claimed in claim 1 or claim 2, optionally comprising isolating the recombinant mammalian protein.<br>
4.	The method as claimed in any of claims 1 to 3, optionally comprising substantially purifying the recombinant mammalian protein.<br>
5.	The method as claimed in claim any of claims 1 to 4, wherein the recombinant mammalian protein is present in the cell in an active form.<br><br>
6.	The method as claimed in any of claims 1 to 5, wherein the recombinant mammalian protein is a human peptide.<br>
7.	The method as claimed in any of claims 1 to 6, wherein the recombinant protein is expressed as at least 5% of the total cell protein.<br>
8.	The method as claimed in claim any of claims 1 to 7, wherein the recombinant protein is produced at a concentration of at least lOg/L.<br>
9.	The method as claimed in any of claims 1 to 9, wherein the recombinant mammalian protein has a mass of between at least 1 kD and 500 kD.<br><br>
10.	The method as claimed in any of claims 1 to 5, wherein the recombinant mammalian protein is an antibody.<br>
11.	The method as claimed in claim 10, wherein said nucleic acid encoding the antibody comprises at least two expression vectors,<br>
12.	The method as claimed in claim 11, wherein said at least two expression vectors comprise a first promoter-cistron pair and a second promoter-cistron pair.<br>
13.	The method as claimed in claim  12, wherein said first promoter-cistron pair is an<br>
immunoglobulin light chain and said second promoter-cistron pair is an immunoglobulin heavy<br><br>
chain.<br>
14.	The method as claimed in claim 13, wherein the light chain and heavy chain are located on the same plasmid.<br>
15.	The method as claimed in any of claims 13-14, wherein said first or second promoter-cistron pair further comprises a translation initiation region operably linked to the nucleic acid encoding an antibody.<br>
16.	The method as claimed in claim 15, wherein the translation initiation region provides different translational strengths to the first and second promoter-cistron pair.<br>
17.	A method as claimed in 10 wherein said nucleic acid encodes an antibody or antibody fragment comprising at least two separate translational units and further wherein the translational units are expressed in sequential fashion.<br>
18.	The method as claimed in claim 17, wherein said at least two separate translational units comprise a light chain and a heavy chain.<br>
19.	The method as claimed in any of claims 10 to 18, wherein said antibody is a single chain antibody, a full chain antibody, or an antibody fragment selected from the group consisting of Fab, Fab'., F(ab')2, F(ab')2-Ieucine zipper, Fv, dsFv, and anti-CD 18 antibody.<br><br>
20.	The method as claimed in claim 21, wherein said Fab fragment comprises a light and a heavy chain fragment.<br>
21.	The method as claimed in any of claims 19 to 20, wherein said Fab comprises an antigen binding site.<br><br>
22.	The method as claimed in any of claims 10 to 21, wherein said antibody or antibody fragment is a chimeric antibody, human antibody or humanized antibody.<br>
23.	The method as claimed in any of claims 10 to 22, wherein said method further comprises linking a secretion signal sequence to said antibody.<br>
24.	The method as claimed in any of claims 10 to 23, wherein said antibody further comprises an<br>
Fc region.<br>
25.	The method as claimed in any of claims 10 to 24, wherein said antibody is a fusion protein.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LUFic3RyYWN0LSgwMS0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Abstract-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWFzc2lnbm1lbnRzLnBkZg==" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-assignments.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LUNsYWltcy0oMDEtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Claims-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDEtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Correspondence Others-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycyAxLnBkZg==" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LURyYXdpbmdzLSgwMS0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Drawings-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LUZvcm0tMy0oMDEtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Form-3-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LUdQQS0oMDEtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-GPA-(01-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0xMDYucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-106.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1kZWxucC0yMDA2LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">3608-delnp-2006-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzYwOC1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMDEtMDMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3608-DELNP-2006-Petition-137-(01-03-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252619-telecommunication-service-platform-and-method-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252621-an-isolated-nucleic-acid-peptide-a-recombinant-vector-cell-and-expression-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252620</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3608/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Jun-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-May-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW GLOBAL TECHNOLOGIES INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>WASHINGTON STREET, 1790 BUILDING, MIDLAND, MI 48674, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DIANE M. RETALLACK</td>
											<td>13420 CARRIAGE ROAD, POWAY, CA 92064, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHARLES H. SQUIRES</td>
											<td>15547 HIDDEN VALLEY DRIVE, POWAY, CA 92064, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>DAVID C. WATKINS</td>
											<td>185 SHADY HILL DRIVE, EAST GREENWICH, RI 02818, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>STACEY LYNN LEE</td>
											<td>13831 PASCO CARDIEL, SAN DIEGO, CA 92129, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>ROBERT SHUTTER</td>
											<td>16350 WOODSON VIEW ROAD, POWAY, CA 92064, USA</td>
										</tr>
										<tr>
											<td>6</td>
											<td>FRANK H. GAERTNER</td>
											<td>4461 OCEAN BOULEVARD #36 SAN DIEGO, CA 92109, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12P 21/06</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/001549</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-01-18</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/564,798</td>
									<td>2004-04-22</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/537,148</td>
									<td>2004-01-16</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252620-expression-of-mammalian-proteins-in-pseudomonas-fluorescenes by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:58 GMT -->
</html>
