<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/252648-an-isolated-polypeptide-for-treating-viral-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 252648:&quot;AN ISOLATED POLYPEPTIDE FOR TREATING VIRAL INFECTION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED POLYPEPTIDE FOR TREATING VIRAL INFECTION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method for detecting a mutation related to the gene encoding OAS1. This and other disclosed mutations correlate with resistance of humans to viral infection including hepatitis C. Also provided is a therapeutic agent consisting of a protein or polypeptide encoded by the mutated gene, or a polynucleotide encoding the protein or polypeptide. Inhibitors of human OAS1, including antisense oligonucleotides, methods, and compositions specific for human OAS1, are also provided.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DETECTION OF MUTATIONS IN A GENE ASSOCIATED WITH RESISTANCE TO VIRAL INFECTION, OAS1<br>
DESCRIPTION<br>
1.	Technical Field<br>
The present invention relates to a method for detecting a mutation in a human oligoadenylate synthetase gene, wherein a mutation confers resistance to flavivirus infection, including infection by hepatitis C virus, and a mutation relates to other disease states including prostate cancer and diabetes, and uses of the encoded proteins and antibodies thereto.<br>
2.	Background of the Invention<br>
A number of diseases have been identified to date in which natural resistance to infection exists in the human population. Alter and Mover, J. Acquir. Immune Defic. Syndr. Hum Retrovirol. 18 Suppl. 1 :S6-10 (1998) report hepatitis C viral infection (HCV) rates as high as 90% in high-risk groups such as injecting drug users. However, the mechanism by which the remaining 10% are apparently resistant to infection has not been identified in the literature. Proteins that play a role in HCV infection include the 2-prime, 5-prime oligoadenylate synthetases. OASs are interferon-induced proteins characterized by their capacity to catalyze the synthesis of 2-prime,5-prime oligomers of adenosine (2-5As), Hovanessian et al., EMBO 6: 1273-1280 (1987) found that interferon-treated human cells contain several OASs corresponding to proteins of 40 (OAS1), 46 (OAS1), 69, and 100 kD. Marie et al., Biochem. Biophys. Res. Commun. 160:580-587 (1989) generated highly specific polyclonal antibodies against p69, the 69-kD OAS. By screening an interferon-treated human cell expression library with the anti-p69 antibodies, Marie and Hovanessian, J. Biol. Chem. 267: 9933-9939 (1992) isolated a partial OAS 2 cDNA. They screened additional libraries with the partial cDNA and recovered cDNAs encoding two OAS2 isoforms. The smaller isoform is encoded by two mRNAs that differ in the length of the 3-prime untranslated region.<br>
Northern blot analysis revealed that OAS2 is expressed as four interferon-induced mKNAs in human cells. The predicted OAS2 proteins have a common 683-amino acid sequence and different 3-prime termini. According to SDS-PAGE of in vitro transcription/translation products, two isoforms have molecular masses of 69 and 71 kD. Both isoforms exhibited OAS activity in vitro. Sequence analysis indicated that OAS2 contains two OAS 1-homologous domains separated by a proline-ricli putative linker region. The N- and C-terminal domains are 41% and 53% identical to OAS1, respectively.<br>
By fluorescence in situ hybridization and by inclusion within mapped clones, Hovanian<br><br>
et al., Genomics 52: 267-277 (1998) determined that the OAS1, OAS2, and OAS3 genes are clustered with a 130-kb region on 12q24.2. 2-5As bind to and activate RNase I, which degrades viral and cellular RNAs, leading to inhibition of cellular protein synthesis and impairment of viral replication.<br>
A fourth human OAS gene, referred to as OASL, differs from OAS1, OAS2 and OAS3 in that OASL lacks enzyme activity. The OASL gene encodes a two-domain protein composed of an OAS unit fused to a 164 amino acid C-terminal domain that is homologous to a tandem repeat of ubiquitin. (Eskildsen et al., Nuc. Acids Res. 31:3166-3173,2003; Kakuta et al., J. Interferon &amp; Cytokine Res. 22:981-993,2002.)<br>
Because of their role in inhibiting viral replication and viral infection, there is a need in the art for methods and compositions that suppress viral replication related to OAS1 activity, including a profound need for inhibitor-based therapies that suppress HCV replication.<br>
BRIEF SUMMARY OF THE INVENTION<br>
The present invention relates to detecting hepatitis C resistance-related mutations which are characterized as point mutations in the oligoadenylate synthetase 1 gene. In one embodiment, a human genetic screening method is contemplated. The method comprises assaying a nucleic acid sample isolated from a human for the presence of an oligoadenylate synthetase 1 gene point mutation characterized as a base substitution at nucleotide position 2135728, 2135749, 2135978, 2144072, 2144088,2144116,2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, or 2156638 or a base deletion at nucleotide position 2156595 for oligoadenylate synthetase 1 gene (OAS1) with reference to Genbank Sequence Accession No. NT_009775.13 (consecutive nucleotides 2,130,000 - 2,157,999 of which are shown in Figure 2 as SEQ ID NO: 19).<br>
In a preferred embodiment, the method comprises treating, under amplification conditions, a sample of genomic DNA from a human with a polymerase chain reaction (PCR) primer pair for amplifying a region of human genomic DNA containing nucleotide position 2135728, 2135749, 2135978,2144072, 2144088, 2144116, 2144321,2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 of oligoadenylate synthetase 1 gene NT_009775.13. The PCR treatment produces an amplification product containing the region, which is then assayed for the presence of a point mutation.<br>
In a further embodiment, the invention provides a protein encoded by a gene having at least one mutation at position 2135728, 2135749, 2135978, 2144072,2144088, 2144116, 2144321,<br><br>
2131025, 2133961, 2139587, 2144294,2144985, 2156523, 2156595, or 2156638 of<br>
NT 009775.13, and use of the protein to prepare a diagnostic for resistance to viral infection,<br>
preferably flaviviral infection, most preferably hepatitis C infection. In specific embodiments,<br>
the diagnostic is an antibody.<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or<br>
inhibiting infection by a virus, preferably a flavivirus, most preferably hepatitis C virus,<br>
wherein the therapeutic compound is a protein encoded by an OAS 1 gene having at least one<br>
mutation at position 2135728, 2135749,2135978,2144072, 2144088, 2144116,2144321,<br>
2131025,2133961, 2139587,2144294, 2144985, 2156523, 2156595, or 2156638 of<br>
NTJ)Q9775.13. In other embodiments the therapeutic compound is a polynucleotide, such as<br>
DNA or RNA, encoding the protein.<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or<br>
inhibiting infection by a virus, preferably a liavivirus, most preferably a hepatitis C virus,<br>
wherein the therapeutic compound is a protein of the sequence: SEQ ID NO: 20, SEQ ID NO:<br>
21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ<br>
ID NO: 27, SEQ ID NO: 28, SEQ ID NO:29, SEQ ID NO:32, SEQ ID NO:33, SEQ ID NO:34,<br>
SEQ ID NO.-35, SEQ ID NO:46, SEQ ID NO:47 and/or SEQ ID NO:48.<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or<br>
inhibiting infection by a virus, preferably a flavivirus, most preferably hepatitis C virus, wherein<br>
the therapeutic compound mimics the beneficial effects of at least one mutation at position<br>
2135728,2135749,2135978,2144072,2144088,2144116,2144321,2131025,2133961,<br>
2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 ofNT_009775.13. The<br>
therapeutic compound can be a small molecule, protein, peptide, DNA or RNA molecule, or<br>
antibody,<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or<br>
treating cancer, preferably prostate cancer, wherein the therapeutic compound is a protein<br>
encoded by an OAS1 gene having at least one mutation at position 2135728, 2135749,2135978,<br>
2144072, 2144088, 2144116,2144321, 2131025, 2133961, 2139587, 2144294, 2144985,<br>
2156523, 2156595, or 2156638 ofNT_009775.13. mother embodiments the therapeutic<br>
compound is a polynucleotide, such as DNA or RNA, encoding the protein.<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or<br>
treating cancer, preferably prostate cancer, wherein the therapeutic compound is a protein of the<br>
sequence: SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24,<br>
SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO:29, SEQ ID<br><br>
NO:32, SEQ ID N0:33, SEQ ID NO:34, SEQ ID NO:35, SEQ ID N0:46, SEQ ID NO:47 and/or SEQ ID NO:48.<br>
In a still further embodiment, the invention provides a therapeutic compound for preventing or treating cancer, preferably prostate cancer, wherein the therapeutic compound mimics the beneficial effects of at least one mutation at position 2135728, 2135749, 2135978, 2144072, 2144088,2144116,2144321,2131025,2133961,2139587,2144294,2144985,2156523, 2156595, or 2156638 of NT_009775.13. The therapeutic compound can be a small molecule, protein, peptide, DNA or RNA molecule, or antibody.<br>
In further embodiments, the therapeutic compound is capable of inhibiting the activity of OAS1 or at least one sub-region or sub-function of the entire protein, and such compounds are represented by antisense molecules, ribozymes, and RNAi molecules capable of specifically binding to OAS1 polynucleotides, and by antibodies and fragments thereof capable of specifically binding to OAS1 proteins and polypeptides.<br>
The present invention provides, in another embodiment, inhibitors of OAS1. Inventive inhibitors include, but are not limited to, antisense molecules, ribozymes, RNAi, antibodies or antibody fragments, proteins or polypeptides as well as small molecules. Exemplary antisense molecules comprise at least 10, 15 or 20 consecutive nucleotides of, or that hybridize under stringent conditions to the polynucleotide of SEQ ID NO: 19. More preferred are antisense molecules that comprise at least 25 consecutive nucleotides of, or that hybridize under stringent conditions to the sequence of SEQ ID NO: 19.<br>
In a sf 11 further embodiment, inhibitors of OAS1 are envisioned that specifically bind to the region of the protein defined by the polypeptide of SEQ ID N0:30. Inventive inhibitors include but are not limited to antibodies, antibody fragments, small molecules, proteins, or polypeptides.<br>
In a still further embodiment, inhibitors of OAS1 are envisioned that are comprised of antisense or RNAi molecules that specifically bind or hybridize to the polynucleotide of SEQ ID NO:31.<br>
In further embodiments, compositions are provided that comprise one or more OAS1 inhibitors in a pharmaceutically acceptable carrier.<br>
Additional embodiments provide methods of decreasing OAS1 gene expression or biological activity.<br>
Additional embodiments provide for methods of specifically increasing or decreasing the expression of certain forms of the OAS1 gene having at least one mutation at position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587,<br><br>
2144294, 2144985, 2156523, 2156595, or 2156638 ofNT_009775.13.<br>
The invention provides an antisense oligonucleotide comprising at least one modified internucleoside linkage.<br>
The invention further provides an antisense oligonucleotide having a phosphorothioate linkage.<br>
The invention still further provides an antisense oligonucleotide comprising at least one modified sugar moiety.<br>
The invention also provides an antisense oligonucleotide comprising at least one modified sugar moiety which is a 2'-O-methyl sugar moiety.<br>
The invention further provides an antisense oligonucleotide comprising at least one modified nucleobase.<br>
The invention still further provides an antisense oligonucleotide having a modified nucleobase wherein the modified nucleobase is 5-methylcvtosine.<br>
The invention also provides an antisense compound wherein the antisense compound is a chimeric oligonucleotide.<br>
The invention provides a method of inhibiting the expression of human OAS1 in human cells or tissues comprising contacting the cells or tissues in vivo with an antisense compound or a ribozyme of 8 to 35 nucleotides in length targeted to a nucleic acid molecule encoding human OAS1 so that expression of human OAS1 is inhibited.<br>
The invention further provides a method of decreasing or increasing expression of specific forms of OAS 1 in vivo, such forms being defined by having at least one mutation at position 2135728, 2135749, 2135978, 2144072,2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523,2156595, or 2156638 of NT_009775.13, using antisense or RNAi compounds or ribozymes.<br>
The invention further provides a method of modulating growth of cancer cells comprising contacting the cancer cells in vivo with an antisense compound or ribozyme of 8 to 35 nucleotides in length targeted to a nucleic acid molecule encoding human OAS1 so that expression of human OAS1 is inhibited.<br>
The invention still further provides for identifying target regions of OAS 1 polynucleotides. The invention also provides labeled probes for identifying OAS1 polynucleotides by in situ hybridization.<br>
The invention provides for the use of an OAS1 inhibitor according to the invention to prepare a medicament for preventing or inhibiting HCV infection.<br>
The invention further provides for directing an OAS1 inhibitor to specific regions of the<br><br>
OAS1 protein or at specific functions of the protein.<br>
The invention also provides a pharmaceutical composition for inhibiting expression of OAS1, comprising an antisense oligonucleotide according to the invention in a mixture with a physiologically acceptable carrier or diluent.<br>
The invention further provides a ribozyme capable of specifically cleaving OAS1 RNA, and a pharmaceutical composition comprising the ribozyme.<br>
The invention also provides small molecule inhibitors of OAS1 wherein the inhibitors are capable of reducing the activity of OAS1 or of reducing or preventing the expression of OAS1 mRNA.<br>
The invention further provides for inhibitors of OAS 1 that modify specific functions of the protein other than the synthesis of 2'-5' oligoadenylates, such functions including interaction with other proteins such as Hepatitis C virus NS5A protein.<br>
The invention further provides for compounds that alter post-translational modifications of OAS1 including but not limited to glycosylation and phosphorylation. The invention further provides a human genetic screening method for identifying an oligoadenylate synthetase gene mutation comprising: (a) treating, under amplification conditions, a sample of genomic DNA from a human with a polymerase chain reaction (PCR) primer pair for amplifying a region of human genomic DNA containing nucleotide position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116,2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 of oligoadenylate synthetase gene, said treating producing an amplification product containing said region; and (b) detecting in the amplification product of step (a) the presence of an nucleotide point mutation at nucleotide position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638, thereby identifying said mutation.<br>
In certain embodiments of this method, the region comprises a nucleotide sequence represented by a sequence selected from the group consisting of SEQ ID NO: 1-7 and SEQ ID NO:57-64. In other embodiments, the region consists essentially of a nucleotide sequence selected from the group consisting of SEQ ID NO:l-7 and SEQ ID NO:57-64. Also provided is a method of detecting, wherein the detecting comprises treating, under hybridization conditions, the amplification product of step (a) above with an oligonucleotide probe specific for the point mutation, and detecting the formation of a hybridization product. In certain embodiments of the method, the oligonucleotide probe comprises a nucleotide sequence selected from the group consisting of SEQ ID NO:12-18.<br><br>
The invention also relates to a method for detecting in a human a hepatitis C infection resistance disease allele containing a point mutation comprising substitution of a non wild-type nucleotide for a wild-type nucleotide at nucleotide position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587,2144294, 2144985, 2156523, 2156595, or 2156638 of oligoadenylate synthetase gene (OAS1), which method comprises: (a) forming a polymerase chain reaction (PCR) admixture by combining, in a PCR buffer, a sample of genomic DNA from said human and an oligoadenylate synthetase gene-specific PCR primer pair selected from the group consisting of SEQ ID NO:8 and 9, and SEQ ID NO:10 and 11; (b) subjecting the PCR admixture to a plurality of PCR thermocycles to produce an oligoadenylate synthetase gene amplification product; and (c) treating, under hybridization conditions products produced in step (b), with a probe selected from the group consisting of SEQ ID NO: 12-18, thereby detecting said mutation.<br>
Also provided is an isolated OAS1 inhibitor selected from die group consisting of an antisense oligonucleotide, a ribozyme, a small inhibitory RNA (RNAi), a protein, a polypeptide, an antibody, and a small molecule. The isolated inhibitor may be an antisense molecule or the complement thereof comprising at least 15 consecutive nucleic acids of the sequence of SEQ ID NO: 19. In other embodiments, the isolated OAS1 inhibitor (antisense molecule or the complement thereof) hybridizes under high stringency conditions to the sequence of SEQ ID NO:19.<br>
The isolated OAS1 inhibitor may be selected from the group consisting of an antibody and an antibody fragment. Also provided is a composition comprising a therapeutically effective amount of at least one OAS1 inhibitor in a pharmaceutically acceptable carrier.<br>
The invention also relates to a method of inhibiting the expression of OAS 1 in a mammalian cell, comprising administering to the cell an OAS1 inhibitor selected from the group consisting of an antisense oligonucleotide, a ribozyme, a protein, an RNAi, a polypeptide, an antibody, and a small molecule.<br>
The invention further relates to a method of inhibiting the expression of OAS 1 gene expression in a subject, comprising administering to the subject, in a pharmaceutically effective vehicle, an amount of an antisense oligonucleotide which is effective to specifically hybridize to all or part of a selected target nucleic acid sequence derived from said OAS1 gene.<br>
The invention still further relates to a method of preventing infection by a flavivirus in a human subject susceptible to the infection, comprising administering to the human subject an OAS1 inhibitor selected from group consisting of an antisense oligonucleotide, a ribozyme, an RNAi, a protein, a polypeptide, an antibody, and a small molecule, wherein said OAS1 inhibitor<br><br>
prevents infection by said flavivirus.<br>
The invention still further relates to a method of preventing or curing infection by a flavivirus or other virus in a human subject susceptible to the infection, comprising administering to the human subject an OAS1 inhibitor selected from group consisting of an antisense oligonucleotide, a ribozyme, an RNAi, a protein, a polypeptide, an antibody, and a small molecule, wherein said OAS1 inhibitor prevents infection by said flavivirus or other virus and wherein said OAS1 inhibitor is directed at one or more specific forms of the protein defined by a mutation at position 2135728,2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985,2156523, 2156595, or 2156638 of NT_009775.13.<br>
The invention still further relates to a method of preventing or curing infection by a flavivirus or any other virus in a human subject susceptible to the infection by administering one of the polypeptides of the sequence: SEQ ID NO: 20, SEQ ED iNO: 21, SEQ ID NO: 22, SEQ ID i\O: 23, SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO:29, SEQ E&gt; NO:32, SEQ ID NO:33, SEQ ID NO:34, SEQ ID NO:35, SEQ ID NO:46, SEQ ID NO:47 and/or SEQ ID NO:48.<br>
The invention embodies also treatments for infection with the human immunodeficiency virus (HIV).<br>
The invention still further relates to a method of preventing insulin dependent diabetes mellitus (EDDM) in a human subject, comprising administering to the human subject an OAS1 inhibitor selected from group consisting of an antisense oligonucleotide, a ribozyme, an RNAi, a protein, a polypeptide, an antibody, and a small molecule, wherein said OAS1 inhibitor prevents IDDM.<br>
The invention still further relates to a method of preventing EDDM in a human subject, comprising administering to the human subject an OAS1 inhibitor selected from group consisting of an antisense oligonucleotide, a ribozyme, an RNAi, a protein, a polypeptide, an antibody, and a small molecule, wherein said OAS1 inhibitor prevents IDDM and wherein said OAS1 inhibitor is directed at one or more specific forms of the protein defined by a mutation at position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985,2156523, 2156595, or 2156638 of NT_009775.13.<br>
The invention still further relates to a method of treating cancer, such as prostate cancer by increasing expression of the OAS 1 gene or by therapeutic administration of polypeptides disclosed herein.<br>
Also provided is a method for inhibiting expression of a OAS1 target gene in a cell in<br><br>
vitro comprising introduction of a ribonucleic acid (RNA) into the cell in an amount sufficient to inhibit expression of the OAS1 target gene, wherein the RNA is a double-stranded molecule with a first strand consisting essentially of a ribonucleotide sequence which corresponds to a nucleotide sequence of the OAS1 target gene and a second strand consisting essentially of a ribonucleotide sequence which is complementary to the nucleotide sequence of the OAJS1 target gene, wherein the first and the second ribonucleotide strands are separate complementary strands that hybridize to each other to form said double-stranded molecule, and the double-stranded molecule inhibits expression of the target gene.<br>
In certain embodiments of the method, the first ribonucleotide sequence comprises at least 20 bases which correspond to the OAS1 target gene and the second ribonucleotide sequence comprises at least 20 bases which are complementary to the nucleotide sequence of the OAS1 target gene. In still further embodiments, the target gene expression is inhibited by at least 10%.<br>
In still further embodiments of the method, the double-stranded ribonucleic acid structure is at least 20 bases in length and each of the ribonucleic acid strands is able to specifically hybridize to a deoxyribonucleic acid strand of the OAS1 target gene over the at least 20 bases.<br>
The invention provides a polypeptide or protein capable of restoring function of OAS1 that may be diminished or lost due to gene mutation. In some embodiments the polypeptide or protein has the amino acid sequence of wild type OAS1 encoded by a gene comprising SEQ ED NO: 19. In other embodiments, wherein a mutation in the OAS1 gene confers increased activity, stability, and/or half life on the OAS1 protein, or other change making the OAJ31 protein more suitable for anti-viral activity, the protein or polypeptide encoded by the mutated OAS1 gene is preferred.<br>
Any of the foregoing proteins and polypeptides can be provided as a component of a therapeutic composition.<br>
Also provided is the use of any of the proteins consisting of SEQ ID NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24, SEQ ID NO: 25, SEQ ID 3^O: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO:29, SEQ ID NO:32, SEQ ID NO:33, SEQ ID NO:34, SEQ ID NO:35, SEQ ID NO:46, SEQ ED NO:47 and/or SEQ ID NO:48 as a component of a therapeutic composition.<br>
In a further embodiment, a nucleic acid encoding the OAS1 protein, OAS1 mutant protein, or OAS1 polvpetitide can be administered in the form of gene therapy.<br><br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is a Table showing the locations of the mutations of the present invention in the OAS1 gene, the allelic variants (base substitutions), coordinates of the mutation on the genomic sequence, and NCBI dbSNP ID if any.<br>
Figure 2 (SEQ ID NO:19) is a polynucleotide sequence consisting of the consecutive nucleotide bases at positions 2,130,000-2,157,999 of NCBI Accession No. NT_009775. 13, OAS1.<br>
Figure 3 shows SEQ ID NO: 20-64.<br>
Figure 4 shows approximate haplotype distributions within the OAS1 gene in a Caucasian population.<br>
Figure 5 A and B illustrates the variety of transcript variants disclosed by the invention and their relation to mutations disclosed by ihe invention. Figure 5A shows transcript Variant (TV) forms 1-6, and Figure 5B shows TV forms 7-10. Figure 5C further illustrates the inferred structure of chimpanzee and gorilla variants relative to human.<br>
Figure 6 is a table describing non-human primate mutations in the OAS1 gene relative to human and the coordinates of the mutations on the corresponding human genomic sequence.<br>
Figure 7 demonstrates the enzymatic activity of exemplary polypeptides of the present invention when expressed and recovered from E. coli.<br>
Figure 8 demonstrates the enzymatic activity of exemplary polypeptides resulting from polynucleotides of the present invention when expressed in HeLa cells.<br>
Figure 9 demonstrates development of antibodies to exemplary polypeptides of the present invention.<br>
Figure 10 is a table detailing exemplary protein transduction domain polypeptides.<br>
DETAILED DESCRIPTION OF THE INVENTION Introduction and Definitions<br>
This invention relates to novel mutations in the oligoadenylate synthetase gene, use of these mutations for diagnosis of susceptibility or resistance to viral infection, to proteins encoded by a gene having a mutation according to the invention, and to prevention or inhibition of viral infection using the proteins, antibodies, and related nucleic acids. These mutations correlate with resistance of the carrier to infection with tlavi virus, particularly hepatitis C virus. Much of current medical research is focused on identifying mutations and defects that cause or contribute to disease. Such research is designed to lead to compounds and methods of treatment<br><br>
aimed at the disease state. Less attention has been paid to studying the genetic influences that allow people to remain healthy despite exposure to infectious agents and other risk factors. The present invention represents a successful application of a process developed by the inventors by which specific populations of human subjects are ascertained and analyzed in order to discover genetic variations or mutations that confer resistance to disease. The identification of a su.b-population segment that has a natural resistance to a particular disease or biological condition further enables the identification of genes and proteins that are suitable targets for pharmaceutical intervention, diagnostic evaluation, or prevention, such as prophylactic vaccination.<br>
The sub-population segment identified herein is comprised of individuals who, despite repeated exposure to hepatitis C virus (HCV) have nonetheless remained sero-negative, while cohorts have become infected (sero-positive). The populations studied included hemophiliac patients subjected to repeated blood transfusions, and intravenous drug users who become exposed. through shared needles and other risk factors.<br>
HCV infection involves a complex set of proteins and immune system components that work together to achieve a level of infection that, while it causes disease, can develop into low steady state of virus in infected cells, apparently allowing HCV to escape from the host irnmuno-surveillance system, while enabling persistent viral infection. (Dansako et al., Virus Research 97:17-30,2003.) The present invention focuses on one component of this system, the interferon-inducible 2'-5'-oligoadenylate synthetase gene, specifically OASl. The OASl gene plays a major role in the antiviral activity of host cells in the human, by activating ribonuclease L (RNase L) to cleave viral RNA. HCV RNA activates the 2'-5 '-OAS/RNase L pathway.  As pointed out by Dansako et al., it may appear contradictory for HCV RNA to activate a pathway that leads to cleavage of the viral RNA. However, such activity may serve to retain a balance between the host immune defense and a level of infection that would kill the host. In view of this complex role of the OASl gene, it is of significant interest that the present invention has identified a strong correlation between mutations in the OASl gene, and resistance to HCV infection in carriers of these mutations. The presence of such individuals now permits the elucidation of how O AS 1 contributes to resistance to HCV infection despite repeated exposure to infectious levels of the virus. This information will then lead to development of methods and compositions for replicating the resistance mechanism in individuals lacking natural resistance.<br>
The present invention therefore provides that, regardless of the mechanism, the mutations identified herein are useful for identifying individuals who are resistant to HCV infection.   The<br><br>
resistance may come about through a loss of function of the OAS1 protein, in which case it is predicted that HCV viral levels would be high enough to prevent the virus from escapin g from the host immuno-surveillance system, hence facilitating destruction of the virus. The resistance may also come about through gain of function in that the OAS1 protein level is enhanced, the half life of the protein is increased, and/or the protein structure is affected in a way that enhances its ability to activate ribonuclease L to cleave viral RNA. The resistance may also come about through modifications to the OAS1 protein that prevent inhibition of normal OAS1 protein function by HCV viral proteins or nucleotides. The resistance may also come about through modifications to the OAS1 protein that prevent interaction of the protein with HCV viral proteins or nucleotides that are necessary for the normal HCV viral lifespan. The invention is not limited to one mechanism. Furthermore, although several different point mutations are disclosed herein, this is not intended to be indicative that each mutation has the same effect on OAS1 protein structure or function.<br>
OAS1 plays a role in infection by other viruses of the flavivirus family, of which HCV is a member. The flavivirus family also includes viruses that cause yellow fever, dengue fever, St. Louis encephalitis, Japanese encephalitis, and other viral diseases disclosed herein. The host defense to these viruses includes virus-inducible interferon. The interferon induces 2'-5 '-oligoadenylate synthetases, which as discussed above, are involved in the activation of KNaseL. RNaseL in turn cleaves viral RNA. Other viral infections may by amenable to prevention and/or inhibition by the methods disclosed herein, including RSV.<br>
In reference to the detailed description and preferred embodiment, the following definitions are used:<br>
A: adenine; C: cytosine; G: guanine; T: thymine (in DNA); and U: uracil (in RNA) Allele: A variant of DNA sequence of a specific gene. In diploid cells a maximum of two alleles will be present, each in the same relative position or locus on homologous chromosomes of the chromosome set. When alleles at any one locus are identical the individual is said to be homozygous for that locus, and when they differ the individual is said to be heterozygous for that locus. Since different alleles of any one gene may vary by only a single base, the possible number of alleles for any one gene is very large. When alleles differ, one is often dominant to the other, which is said to be recessive. Dominance is a property of the phenotype and does not imply inactivation of the recessive allele by the dominant. In numerous examples the normally functioning (wild-type) allele is dominant to all mutant alleles of more or less defective function. In such cases the general explanation is that one functional allele out of two is sufficient to produce enough active gene product to support normal development of the organism (i.e., there<br><br>
is normally a two-fold safety margin in quantity of gene product).<br>
Haplotype: One of many possible pluralities of Alleles, serially ordered by chromosomal<br>
localization and representing that set of Alleles carried by one particular homologous<br>
chromosome of the chromosome set.<br>
Nucleotide: A monomeric unit of DNA or RNA consisting of a sugar moiety (pentose), a<br>
phosphate, and a nitrogenous heterocyclic base. The base is linked to the sugar moiety via the<br>
glycosidic carbon (1' carbon of the pentose) and that combination of base and sugar is a<br>
nucleoside. When the nucleoside contains a phosphate group bonded to the 3' or 5' position of<br>
the pentose it is referred to as a nucleotide. A sequence of operatively linked nucleotides is<br>
typically referred to herein as a "base sequence" or "nucleotide sequence", and their<br>
grammatical equivalents, and is represented herein by a formula whose left to right orientation is<br>
in the conventional direction of 5'-terminus to 3'-terminus.<br>
Base Pair (bp): A partnership of adenine (A) with thymine (. I), or of cytosine (C) with guanine<br>
(G) in a double stranded DNA molecule. In RNA, uracil (U) is substituted for thymine. When<br>
referring to RNA herein, the symbol T may be used interchangeably with U to represent uracil at<br>
a particular position in the RNA molecule.<br>
Nucleic Acid: A polymer of nucleotides, either single or double stranded.<br>
Polynucleotide: A polymer of single or double stranded nucleotides. As used herein<br>
"polynucleotide" and its grammatical equivalents will include the full range of nucleic acids. A<br>
polynucleotide will typically refer to a nucleic acid molecule comprised of a linear strand of two<br>
or more deoxyribonucleotides and/or ribonucleotides. The exact size will depend on many<br>
factors, which in turn depends on the ultimate conditions of use, as is well known in the art. The<br>
polynucleotides of the present invention include primers, probes, RNA/DNA segments,<br>
oligonucleotides or "oligos" (relatively short polynucleotides), genes, vectors, plasmids, and the<br>
like.<br>
RNAi: RNA interference (RNAi) is a method whereby small interfering RNA (siRNA), a<br>
duplex typically about 21-23 nucleotides long, is introduced into a cell, leading ultimately to the<br>
degradation of messenger RNA of a targeted gene containing an identical or complementary<br>
sequence and effectively silencing it.<br>
Gene: A nucleic acid whose nucleotide sequence codes for an RNA or polypeptide. A. gene can<br>
be either RNA or DNA.<br>
Duplex DNA: A double-stranded nucleic acid molecule comprising two strands of substantially<br>
complementary polynucleotides held together by one or more hydrogen bonds between each of<br>
the complementary bases present in a base pair of the duplex. Because the nucleotides that form<br><br>
a base pair can be either a ribonucleotide base or a deoxyribonucleotide base, the phrase "duplex<br>
DNA" refers to either a DNA-DNA duplex comprising two DNA strands (ds DNA), or an RNA-<br>
DNA duplex comprising one DNA and one RNA strand.<br>
Complementary Bases: Nucleotides that normally pair up when DNA or RNA adopts a double<br>
stranded configuration.<br>
Complementary Nucleotide Sequence: A sequence of nucleotides in a single-stranded molecule<br>
of DNA or RNA that is sufficiently complementary to that on another single strand to<br>
specifically hybridize to it with consequent hydrogen bonding.<br>
Conserved: A nucleotide sequence is conserved with respect to a preselected (reference)<br>
sequence if it non-randomly hybridizes to an exact complement of the preselected sequence.<br>
Hybridization: The pairing of substantially complementary nucleotide sequences (strands of<br>
nucleic acid) to form a duplex or heteroduplex by the establishment of hydrogen bonds between<br>
complementary base pairs. It is a specific, i.e. non-random, interaction between two<br>
complementary polynucleotides that can be competitively inhibited.<br>
Nucleotide Analog: A purine or pyrimidine nucleotide that differs structurally from A, T, G, C,<br>
or U, but is sufficiently similar to substitute for the normal nucleotide in a nucleic acid molecule.<br>
DNA Homolog: A nucleic acid having a preselected conserved nucleotide sequence and a<br>
sequence coding for a receptor capable of binding a preselected ligand.<br>
Upstream: In the direction opposite to the direction of DNA transcription, and therefore going<br>
from 5' to 3' on the non-coding strand, or 3' to 5' on the mRNA.<br>
Downstream: Further along a DNA sequence in the direction of sequence transcription or read<br>
out, that is traveling in a 3'- to 5'-direction along the non-coding strand of the DNA or 5'- to 3 '-<br>
direction along the RNA transcript.<br>
Stop Codon: Any of three codons that do not code for an amino acid, but instead cause<br>
termination of protein synthesis. They are UAG, UAA and UGA and are also referred to as a<br>
nonsense or termination codon.<br>
Reading Frame: Particular sequence of contiguous nucleotide triplets (codons) employed in<br>
translation. The reading frame depends on the location of the translation initiation codon.<br>
Intron: Also referred to as an intervening sequence, a noncoding sequence of DNA that is<br>
initially copied into RNA but is cut out of the final RNA transcript.<br>
Modes of Practicing the Invention<br>
The present invention provides a novel method for screening humans for oligoadenylate<br>
synthetase alleles associated with resistance to infection by a flavivirus, particularly hepatitis C.<br><br>
The invention is based on the discovery that such resistance is associated with a point mutation (base substitution) in the oligoadenylate synthetase gene DNA sequence at nucleotide position 2135728,2135749, 2135978, 2144072, 2144088,2144116, 2144321, 2131025,2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 of Genbank Accession No. NT_009775.13 (SEQ ID NO: 19), which encodes the human OAS1 gene. This invention discloses the results of a study that identified populations of subjects resistant or partially resistant to infection with the hepatitis C virus (HCV) and that further identified genetic mutations that confer this beneficial effect. Several genetic mutations in the 2'-5'-oligoadenylate synthetase genes are identified, that are significantly associated with resistance to HCV infection. The study design used was a case-control, allele association analysis. Cases assigned as subjects had serially documented or presumed exposure to HCV, but who did not develop infection as documented by the development of antibodies to the virus (i.e. HCV seronegative). Control subjects were serially exposed subjects who did seroconveri to HCV positive. Case and control subjects were recruited from three populations, hemophilia patients from Vancouver, British Columbia, Canada; hemophilia patients from Northwestern France; and injecting drug users from the Seattle metropolitan region.<br>
Case and control definitions differed between the hemophilia and EDU groups and were based upon epidemiological models of infection risk published in the literature and other models developed by the inventors, as described herein. For the hemophilia population, control subjects were documented to be seropositive for antibodies to HCV using commercial diagnostics laboratory testing. Case subjects were documented as being HCV seronegative, having less than 5% of normal clotting factor, and having received concentrated clotting factors before January 1987. Control injecting drug users were defined as documented HCV seropositive. Case injecting drug users were defined as documented HCV seronegative, having injected drugs for more than ten years, and having reported engaging in one or more additional risk behaviors. Additional risk behaviors include the sharing of syringes, cookers, or cottons with another IDU. In one particular Caucasian population 20 cases and 42 controls were included in the study. Selection of case and control subjects was performed essentially as described in U.S. Patent Application 09/707,576 using the population groups at-risk affected ("controls") and at-risk unaffected ("cases").<br>
The present inventive approach to identifying gene mutations associated with resistance to HCV infection involved the selection of candidate genes. Approximately 50 candidate genes involved in viral binding to the cell surface, viral propagation within the cell, the interferon response, and aspects of the innate immune system and the antiviral response, were interrogated. Candidate<br><br>
genes were sequenced in cases and controls by using the polymerase chain reaction to amplify target sequences from the genomic DNA of each subject. PCR products from candidate genes were sequenced directly using automated, fluorescence-based DNA sequencing and an ABI3730 automated sequencer.<br>
Genetic mutations were identified in an oligoadenylate synthetase gene (OAS1) that either alone, or in combination, were significantly associated (p 
Population and subject parental haplotypes, comprising pluralities of the OAS1 mutations (SEQ ID NO:l-7 and SEQ ID NO:57-64), are computationally inferred from the case-control genotyping data set by Expectation Maximation methods as known to those skilled in the art (Excoffier and Slatkin, Mol. Biol. Evol. 12:921-927, 1995). The invention embraces both the use of the full range of OAS1 mutations for haplotypic analysis as well as the use of subsets of OAS1 mutations for computational convenience. Comparative analysis of patterns of segregation of haplotypes and haplotype subsets with the case and control groups identify mutations of particular potency with regard to viral resistance or susceptibility. In one illustrative example, haplotypes are computed comprising pluralities of the OAS1 mutations identified by SEQ ID NO: 2 through SEQ ED NO: 6. Several haplotypes are identified in a Caucasian case and control population by this analysis. The definition of these haplotypes is shown in Figure 4. Two common haplotypes (identified as HAP1 and HAP2) are identified that account for approximately 85% of inferred haplotypes and are in Hardy-Weinberg equilibrium, particularly with regard to the occurrence of haplotype homozygotes in the population. Further analysis of OAS 1 in various human populations and in primates indicates that HAP2 is the ancestral primate haplotype pre-dating the divergence of Old world monkeys and hominids. One additional haplotype (identified as HAPS) is associated with the persistently HCV-resistant case group in this particular population. Therefore subjects carrying the HAPS haplotype are at substantially lower risk of HCV infection. The HAPS haplotype appears to have arisen through a complex series of recombination and mutation originating from the ancestral haplotype. The combined rarity of such events combined with the considerable occurrence of haplotype HAPS suggests positive selection acted to develop and retain haplotype HAPS in the population, possibly as a response to recurring viral challenge over time. In this example, haplotype HAPS is<br><br>
the only haplotype occurring at an appreciable population frequency that combines the effects of a G nucleotide for mutation SEQ ED NO:2 together with an A micleotide for mutation SEQ ID NO:4 in a single pre-cursor RNA.<br>
The present invention is not limited by the foregoing illustrative example. Nor is the present invention limited by other illustrative examples that provide insight into the relevance and utility of particular OAS1 mutations. In another illustrative example, the substitution of a G nucleotide for an A nucleotide in SEQ ED NO:2 results in a predicted amino acid substitution of a Serine to Glycine. Computational prediction as known to those skilled in the art is highly suggestive that the Serine is a site of phosphorylation whereas the Glycine would not be phosphorylated. In a further illustrative example, the substitution of an A nucleotide for a G nucleotide in mutation SEQ ED NO:4 occurs in the consensus splice acceptor site for the wild-type sixth exon in OAS1. This substitution replaces the requisite G in the splice acceptor recognition signal but in the process creates a new splice recognition site one base pair downstream. The mutated form thereby creates a frameshift in the translated protein. The mutated site also is a less effective splice signal and as a result encourages additional alternative splicing of pre-cursor RNAs in addition to the frameshifted exon 6 splicing. Preferred embodiments of these alternative splice forms are provided in Figures 5 A and 5B. Further illustrative examples of genetic analysis are provided below.<br>
Promiscuous splicing of OAS1 transcript variants was independently confirmed by reverse transcription of RNA derived from both lymphocyte cell lines and peripheral blood mononuclear cells (PBMCs) isolated from fresh human serum. PCR analysis of reverse transcribed RNA products from various haplotype carrying cell lines and PBMCs indicated RNA forms carrying A nucleotides for mutation SEQ ED NO:4 resulted in a multitude of transcript variant OAS 1R forms. These OAS1R transcript variants are depicted graphically in Figure 5A and 5B and comprise SEQ ID NO:36 through SEQ ED NO:47 as provided in Figure 3. These variant OAS1R forms of the OAS1 gene and corresponding transcript variants are believed to encode one or more of the polypeptides consisting of SEQ ED NO: 20, SEQ ED NO: 21, SEQ ED NO: 22, SEQ ED NO: 23, SEQ ED NO: 24, SEQ ID NO: 25, SEQ ED NO: 26, SEQ ED NO: 27, SEQ ED NO: 28, SEQ FD NO:29, SEQ ED NO:32, SEQ ED NO: 35, SEQ ED NO: 46, and/or SEQ ED NO: 47. The foregoing polypeptides, either singly or plurally, may be referred to herein as OAS1R polypeptides or OAS1R proteins interchangeably. A common feature of many of the foregoing polypeptides is that they differ primarily in their carboxyl-terminus while conserving the ammo-terminal portion. In addition to the production of alternative transcripts themselves, the OAS1R forms of the gene<br><br>
may also contain or abolish specific sequence contexts (such as Exon Splice Enhancers) that modify the selective preference for specific transcript variants. This in turn would cause differing relative levels of abundance of the resulting proteins. These variant OAS1R forms of the OAS 1 gene may also modify localization or post-translational modification of the resulting proteins. Those skilled in the art will appreciate that increased abundance or other modifications that improve the activity, stability, or availability of a specific OAS1 protein form may improve the overall anti-viral performance of the 2'-5'-OAS/RNase L pathway. Those skilled in the art can likewise appreciate that depressing the activity or availability of a specific OAS1 form may also improve the overall anti-viral performance of the 2'-5'-OAS/RNase L pathway in cases where said specific protein is not advantaged, or even disadvantaged, over other specific OAS1 forms. Without limitation, one embodiment of a disadvantaged OAS1 protein is one which is specifically targeted by viral protein(s) in such a manner as to preclude the enzymatic activity of said specific OAS1 protein. A further embodiment of a non-advantaged OAS1 protein is one with lower enzymatic activity polymerizing with other active forms thereby lowering, or abolishing, the overall enzymatic activity (and hence decreasing overall anti-viral effect) of the polymerized protein. One or more of the foregoing mechanisms may contribute to resistance to viral infection. The present invention is not limited, however, by the specific mechanism of action of the disclosed variant polynucleotides or polypeptides.<br>
The invention therefore provides novel forms of the human 2'-5'-oligoadenylate synthetase gene, novel mRNA transcripts, and associated proteins. The invention also discloses utility for the novel mKNA transcripts and novel proteins. These novel forms are characterized by the presence in the gene of one or more of several rare genetic mutations or haplotypes not disclosed in the public databases. These novel forms of OAS 1, OAS1R, confer on carriers a level of resistance to the hepatitis C virus and associated flaviviruses including but not limited to the West Nile virus, dengue viruses, yellow fever virus, tick-borne encephalitis virus, Japanese encephalitis virus, St. Louis encephalitis virus, Murray Valley virus, Powassan virus, Rocio virus, louping-ill virus, Banzi virus, Ilheus virus, Kokobera virus, Kunjin virus, Alfuy virus, bovine diarrhea virus, and the Kyasanur forest disease virus. The OAS proteins have also been shown to be important in attenuating infection in experimental respiratory syncitial virus and picornavirus cell culture infection systems. Failure of human immunodeficiency virus-1 (HIV-1) infected cells to release virus has been correlated with high concentrations of OAS and/or 2-5A. Furthermore, HIV-1 transactivator protein (tat) has been shown to block activation of OAS (Muller et al, J Biol Chem. 1990 Mar 5;265(7):3803-8) thus indicating that novel forms of OAS might evade HIV-1 defense mechanisms and provide an effective therapy. Thus, these OAS1R<br><br>
forms of OAS1 disclosed herein may confer resistance to these non-flavivirus infectious agents as well.<br>
The present invention also provides novel description of the chimpanzee (Pan troglodytes) and gorilla (Gorilla gorilla) forms of OAS1, each of which leads to a novel mRNA and polypeptide with utility. While genes are typically very highly conserved in closely related primates, such as humans, chimpanzees and gorillas, important differences in OAS1 are observed between the three species. Chimpanzees, the closest human relative, possess a single base substitution within OAS1 exon 5 (at a site equivalent to 2,142,351 in NT_009775.13 in humans and defined by SEQ ID NO: 53) that causes a truncated protein product. The chimpanzee OAS1 polypeptide and mRNA sequences are provided by SEQ ID NO: 51 and SEQ ID NO: 55, respectively. Gorillas also possess a two base pair deletion (at the sites equivalent to 2,144,089-2,144,090 in NT 009775.13 in humans and defined by SEQ ID NO: 54) within exon 6 near the acceptor splice site that causes a premature stop of translation, ilie gorilla partial polypeptide and partial mRNA sequences are provided by SEQ ID NO: 52 and SEQ ID NO: 56, respectively. The inferred structure of the chimpanzee and gorilla transcripts relative to human is provided in Figure 5C. Each of these cases, like the human OAS1R polypeptides, possess highly conserved polypeptide sequences that differ most notably in the structure and content of the carboxyl-terminal tail. The common ammo-terminal portion of these polypeptides contains all of the elements previously demonstrated to be required for OAS1 enzymatic activity. As chimpanzees and gorillas have been subjected to viral challenges similar to that of humans in Africa, the prevalence of these distinct but functionally similar primate variants provides further evidence that the carboxy-terminal portions of longer OAS1 forms are unnecessary or even disadvantaging in surviving viral challenges. The fact that chimpanzees possess an outright truncation of the OAS1 polypeptide as opposed to the heterogeneity of transcript variants in humans is consistent with the observation that chimpanzees, while the only other primate known to be susceptible to HCV infection, do have an atypical infection relative to humans characterized by an increased frequency of viral clearance and the absence of resulting fibrosis or hepatocellular carcinoma.<br>
Each novel OAS1R cDNA is cloned from human subjects who are carriers of these mutations. Cloning is carried out by standard cDNA cloning methods that involve the isolation of RNA from cells or tissue, the conversion of RNA to cDNA, and the conversion of cDNA to double-stranded DNA suitable for cloning. As one skilled in the art will recognize, all of these steps are routine molecular biological analyses. Other methods include the use of reverse transcriptase PCR, 5'RACE (Rapid Amplification of cDNA Ends), or traditional cDNA library construction<br><br>
and screening by Southern hybridization. All OAS1R alleles described herein are recovered from patient carriers. Each newly cloned OAS1R cDNA is sequenced to confirm its identity and to identify any additional sequence differences relative to wild-type. Novel OAS1R gene mutations may affect resistance to viral infection by modifying the properties of the resulting OAS1 mRNA. Therefore, differences in mRNA stability between carriers of the OAS1R alleles and homozygous wild-type subjects are evaluated. RNA stability is evaluated and compared using known assays including Taqman and simple Northern hybridization. These constitute routine methods in molecular biology. OAS1R mutations may affect infection resistance by modifying the regulation of the OAS1 gene. It is known that expression of OAS genes is induced by interferon treatment and during viral infection. The OAS1R alleles may confer resistance to viral infection through constitutive expression, over-expression, or other disregulated expression. Several methods are used to evaluate gene expression with and without interleron or viral stimulation. These methods include expression microarray analysis, Northern hybridization, Taqman, and others. Samples are collected from tissues known to express the OAS genes such as the peripheral blood mononuclear cells. Gene expression is compared between tissues from OAS1R carriers and non-carriers. In one embodiment, peripheral blood mononuclear cells are collected from carriers and non carriers, propagated in culture, and stimulated with interferon. The level of expression of OAS1R alleles during interferon induction is compared to wild-type alleles. In another embodiment, human subjects are treated with interferon and the level of induction of the OAS1 gene is evaluated in carriers of the OAS1R mutations versus non-carriers. As one skilled in the art can appreciate, numerous combinations of tissues, experimental designs, and methods of analysis are used to evaluate OAS1R gene regulation.<br>
Once the novel cDNA for each OAS1R is cloned, it is used to manufacture recombinant OAS1R proteins using any of a number of different known expression cloning systems. In one embodiment of this approach, an OAS1R cDNA is cloned by standard molecular biological methods into an Escherichia coli expression vector adjacent to an epitope tag that contains a sequence of DNA coding for a polyhistidine polypeptide. The recombinant protein is then purified from Escherichia coli lysates using immobilized metal affinity chromatography or similar method. One skilled in the art will recognize that there are many different expression vectors and host cells that can be used to purify recombinant proteins, including but not limited to yeast expression systems, baculovirus expression systems, Chinese hamster ovary cells, and others. Computational methods are used to identify short peptide sequences from OAS1R proteins that<br><br>
uniquely distinguish these proteins from wild-type OAS1 proteins. Various computational methods and commercially available software packages can be used for peptide selection. These computationally selected peptide sequences can be manufactured using the FMOC peptide synthesis chemistry or similar method. One skilled in the art will recognize that there are numerous chemical methods for synthesizing short polypeptides according to a supplied sequence.<br>
Peptide fragments and the recombinant protein from the OAS1R gene can be used to develop antibodies specific to this gene product. As one skilled in the art will recognize, there are numerous methods for antibody development involving the use of multiple different host organisms, adjuvants, etc. In one classic embodiment, a small amount (150 micrograms) of purified recombinant protein is injected subcutaneously into the backs of New Zealand White Rabbits with subsequent similar quantities injected every several months as boosters. Rabbit serum is then collected by venipuncture and the serum, purified IgG, or affinity puniied antibody specific to the immunizing protein can be collected. As one skilled in the art will recognize, similar methods can be used to develop antibodies in rat, mouse, goat, and other organisms. Peptide fragments as described above can also be used to develop antibodies specific to the O AS 1R protein. The development of both monoclonal and polyclonal antibodies is suitable for practicing the invention. The generation of mouse hybridoma cell lines secreting specific monoclonal antibodies to the OAS1R proteins can be carried out by standard molecular techniques.<br>
Antibodies prepared as described above can be used to develop diagnostic methods for evaluating the presence or absence of the OAS1R proteins in cells, tissues, and organisms. In one embodiment of this approach, enzyme-linked immunosorbent assays can be developed using purified recombinant OAS1R proteins and specific antibodies in order to detect these proteins in human serum. These diagnostic methods can be used to validate the presence or absence of OASlRproteins in the tissues of carriers and non-carriers of the above-described genetic mutations.<br>
Antibodies prepared as described above can also be used to purify native OAS1R proteins from those patients who carry these mutations. Numerous methods are available for using antibodies to purify native proteins from human cells and tissues. In one embodiment, antibodies can be used in immunoprecipitation experiments involving homogenized human tissues and antibody capture using protein A. This method enables the concentration and further evaluation of mutant OAS 1R proteins. Numerous other methods for isolating the native forms of OAS1R are available including column chromatography, affinity chromatography, high pressure liquid<br><br>
chromatography, salting-out, dialysis, electrophoresis, isoelectric focusing, differential centrifugation, and others.<br>
Proteomic methods are used to evaluate the effect of OAS 1R mutations on secondary, tertiary, and quaternary protein structure. Proteomic methods are also used to evaluate the impact of OAS1R mutations on the post-translational modification of the OAS protein. There are many known possible post-translational modifications to a protein including protease cleavage, glycosylation, phosphorylation, sulfation, the addition of chemical groups or complex molecules, and the like. A common method for evaluating secondary and tertiary protein structure is nuclear magnetic resonance (NMR) spectroscopy. NMR is used to probe differences in secondary and tertiary structure between wild-type OAS1 proteins and OAS1R proteins. Modifications to traditional NMR are also suitable, including methods for evaluating the activity of functional sites including Transfer Nuclear Overhauser Spectroscopy (TrNOESY) and others. As one skilled in the art will recognize, numerous minor modifications to this approach and methods for data interpretation of results can be employed. All of these methods are intended to be included in practicing this invention. Other methods for determining protein structure by crystallization and X-ray diffraction are employed.<br>
Mass spectroscopy can also be used to evaluate differences between mutant and wild-type OAS proteins. This method can be used to evaluate structural differences as well as differences in the post-translational modifications of proteins. In one typical embodiment of this approach, the wild-type OAS1 protein and mutant OASlRproteins are purified from human peripheral blood mononuclear cells using one of the methods described above. These cells can be stimulated with interferon, as described above, in order to increase expression of the OAS proteins. Purified proteins are digested with specific proteases (e.g. trypsin) and evaluated using mass spectrometry. As one skilled in the art will recognize, many alternative methods can also be used. This invention contemplates these additional alternative methods. For instance, either matrix-assisted laser desorption/ ionization (MALDI) or electrospray ionization (ESI) mass spectrometric methods can be used. Furthermore, mass spectroscopy can be coupled with the use of two-dimensional gel electrophoretic separation of cellular proteins as an alternative to comprehensive pre-purification. Mass spectrometry can also be coupled with the use of peptide fingerprint database and various searching algorithms. Differences in post-translational modification, such as phosphorylation or glycosylation, can also be probed by coupling mass spectrometry with the use of various pretreatments such as with glycosylases and phosphatases. All of these methods are to be considered as part of this application. OAS1 is active as a tetramer, and mutations that interfere with self-association affect enzyme<br><br>
activity. Known methods are used to evaluate the effect of OASIR mutations on tetramer formation. For instance, immunoprecipitation with OASl and OASIR specific antibodies is performed in order to isolate OASl/OASIR complexes from patient cells, cell culture, or transfected cells over-expressing the OASl or OASIR. These complexes can then be evaluated by gel electrophoresis or other chromatographic methods which are well known to those skilled in the art.<br>
OASl may confer viral resistance by interaction with other proteins. The enzyme contains a region with structural homology to the BH3 domain of the bcl-2 family. This domain may be critical to the function of OASl. According to the invention, OASl-specific antibodies can be used to isolate protein complexes involving the OASl proteins from a variety of sources as discussed above. These complexes can then be evaluated by gel electrophoresis to separate members of the interacting complex. Gels can be probed using numerous methods including Western blotting, and novel interacting proteins can be isolated and identified using peptide sequencing. Differences in the content of OAS complexes in wild-type and OASIR extracts will also be evaluated. As one skilled in the art will recognize, the described methods are only a few of numerous different approaches that can be used to purify, identify, and evaluate interacting proteins in the OAS complex. Additional methods include, but are not limited to, phage display and the use of yeast two-hybrid methods.<br>
OASl is known to interact with hepatitis C virus NS5A protein (Taguchi, T. et al., J. Gen. Virol. 85:959-969, 2004). Without being bound by a mechanism, the invention therefore relates to OASl proteins that do not interact with the NS5A protein, wherein the proteins are polypeptides of the present invention, expressed by polynucleotides of the present invention, expressed by mRNA encoded by splice variants of OASl, by OASl polynucleotides containing at least one mutation of the present invention, by OAS 1 polynucleotides having at least one mutation in the coding region, and or by OASl polynucleotides having at least one base substitution, deletion or addition wherein binding to NS5 A protein is altered or prevented.<br>
NS5 A protein may exert a biological activity by inhibiting the antiviral activity of interferon. This antiviral activity is in one model normally implemented when interferon stimulates OASl activity in the presence of a co-factor, such as double stranded RNA. OASl polymerizes ATP into 2'-5'-linked oligoadenylates, which activate RNase L to cleave single stranded RNA including mRNA. Binding of NS5A to OASl can inhibit its activity, thereby inhibiting or preventing the cascade of activities that would otherwise lead to destruction of the viral RNA.<br>
Although the invention is not dependent on this model, the binding of NS5A to OASl is<br><br>
consistent with a model in which mutated forms of OAS1 avoid NS5A binding and inhibition and are thereby able to carry out the normal function of polymerizing ATP. In such cases, consistent with the clinical results described herein, a person carrying such a mutation is resistant to infection by hepatitis C virus. Similarly, the truncated form of OAS1 possessed by chimpanzees, as disclosed above, may elude binding by NS5A or other viral proteins and thereby allow the observed higher frequency of chimpanzee viral clearance. The mutation may in some cases directly affect the binding site of OAS1 for NS5A. In other cases the mutation may be at a site separate from the actual binding site, but causes a conformational change such that binding of OAS1 to NS5A is inhibited, slowed, or prevented.<br>
The binding of OAS1 to NS5A in a physiologically and pathologically relevant manner therefore provides an objective test for assaying the effect of a base mutation, deletion or addition in an OAS1 polynucleotide on a biological function of the encoded OAS1 protein. Such binding is assayed in a manner known in the art. hi one exemplary but not limiting method, such as described by Taguchi, T. et al. (J. Gen. Virol. 85:959-969, 2004), HeLa cells are transiently transfected with expression plasmids encoding GST-tagged NS5A and HA-tagged OAS1. By OAS1 in this example is meant OAS1 according to the invention, including OAS1 encoded by splice variants of OAS1, by OAS1 polynucleotides having at least one mutation in the coding region, and/or by OAS1 polynucleotides having at least one base substitution, deletion or addition. After an appropriate incubation time such as 12-16 hours, the cells are washed, lysed, and centrifuged, and the resulting supernatant is mixed with glutathione-conjugated Sepharose beads, which aid in separating GST-tagged proteins. Complexes of GST-tagged NS5A and HA-tagged OAS1 protein are identified by using and imaging antibodies to NS5A and anti-HA antibody. Variations on this method include using other tags for the individual proteins, such as FLAG-tag. La the context of the present invention, the main variable is the OAS1 protein or polypeptide. The ability of the OAS1 protein or polypeptide to carry out one biologically relevant activity (i.e. the binding to a hepatitis C protein that is known to be protective for the ability of the virus to replicate in the host) is objectively tested using these assays. OAS1 proteins and polypeptides that do not bind to NS5A are suitable candidates as therapeutic proteins.<br>
The OAS1 proteins are enzymes that catalyze the conversion of ATP into oligoadenylate molecules. Several methods are available to evaluate the activity of OAS 1 enzymes. These methods are employed to determine the effects of OAS1R mutations on the activity of the mutant proteins relative to the wild type enzyme. For example, oligoadenylate synthesis activity can be measured by quantifying the incorporation of 32P-radiolabeled ATP into polyadenylates.<br><br>
The radiolabeled polyadenylates can be quantified and characterized in terms of length by a number of chromatographic methods including electrophoresis or ion exchange chromatography. These assays also enable characterization of substrate (ATP) binding and enzyme kinetics. OAS1 is activated by dsKNA. The kinetics of this activation is analyzed in OAS1 and compared to OASIR using the activity assays described herein and synthetic dsRNAs as described in the art.<br>
The polypeptides of the present invention are demonstrated by these and other methods known in the art to possess oligoadenylate synthesis activity. Figures 7 and 8 demonstrate the activities of several exemplary polypeptides of the present invention. Regardless of their quantitative level of activity, this capacity to produce 2'-5'-oligodenylates is well understood by those skilled in the art to produce anti-viral effects through the activation of RNaseL. As such, the mere fact that the polypeptides of the present invention possess oligoadenylate synthesis activity indicates that said polypeptides have utility, particularly in consideration of therapeutic uses thereof which are disclosed below.<br>
Biological studies are performed to evaluate the degree to which OASIR mutant genes protect from viral infection. These biological studies generally take the form of introducing the mutant OASIR genes or proteins into cells or whole organisms, and evaluating their biological and antiviral activities relative to wild-type controls. In one typical embodiment of this approach, the OASIR genes are introduced into African Green monkey kidney (Vero) cells in culture by cloning the cDNAs isolated as described herein into a mammalian expression vector that drives expression of the cloned cDNA from an SV40 promoter sequence. This vector will also contain SV40 and cytomegalovirus enhancer elements that permit efficient expression of the OASIR genes, and a neomycin resistance gene for selection in culture. The biological effects of OASIR expression can then be evaluated in Vero cells infected with the dengue virus. In the event that OASIR confers broad resistance to multiple flaviviruses, one would expect an attenuation of viral propagation in cell lines expressing these mutant forms of OAS1 relative to wild-type. As one skilled in the art will recognize, there are multiple different experimental approaches that can be used to evaluate the biological effects of OASIR genes and proteins in cells and organisms and in response to different infectious agents. For instance, in the above example, different expression vectors, cell types, and viral species may be used to evaluate the OASIR resistance effects. Primary human cells in culture may be evaluated as opposed to cell lines. Cells may be stimulated with double-stranded RNA or interferon before introduction of the virus. Expression vectors containing alternative promoter and enhancer sequences may be evaluated. Viruses other than the flaviviruses (e.g. respiratory syncytial virus and picornavirus)<br><br>
are evaluated.<br>
Transgenic animal models are developed to assess the usefulness of mutant forms of OAS1 in protecting against whole-organism viral infection. In one embodiment, OAS1R genes are introduced into the genomes of mice susceptible to flavivirus infection (e.g. the C3H/He inbred laboratory strain). These OAS1R genes are evaluated for their ability to modify infection or confer resistance to infection in susceptible mice. As one skilled in the art will appreciate, numerous standard methods can be used to introduce transgenic human OAS1R genes into mice. These methods can be combined with other methods that affect tissue specific expression patterns or that permit regulation of the transgene through the introduction of endogenous chemicals, the use of inducible or tissue specific promoters, etc.<br>
As a model for hepatitis C infection, cell lines expressing OAS1R genes can be evaluated for susceptibility, resistance, or modification of infection with the bovine diarrheal virus (BVDV). BVDV is a commonly used model for testing the efficacy of potential anti-HCV antiviral drugs (Buckwold et. al., Antiviral Research 60:1-15, 2003). In one embodiment, the OAS1R genes can be introduced into KL (calf lung) cells using expression vectors essentially as described above and tested for their ability to modify BVDV infection in this cell line. Furthermore, mouse models of HCV infection (e.g. the transplantation of human livers into mice, the infusion of human hepatocyte into mouse liver, etc.) may also be evaluated for modification of HCV infection In the transgenic setting of OAS1R genes. Experiments can be performed whereby the effects of expression of OAS1R genes are assessed in HCV viral culture systems. Cell culture systems can also be used to assess the impact of the mutant OAS1R gene on promoting apoptosis under varying conditions, hi one embodiment, cell culture mutant forms of OAS1R can be assessed relative to wild-type OAS1 sequences for their ability to promote apoptosis in cells infected with a number of viruses including BVDV, HCV, and other flaviviruses. As one skilled in the art will recognize, numerous methods for measuring apoptosis are available. The most common method involves the detection of the characteristic genomic "DNA laddering" effect in apoptosing cells using fluorescent conjugation methods coupled to agarose gel electrophoresis.<br>
The ability of defective interfering viruses to potentiate the effects of OAS1R mutant forms can be tested in cell culture and in small animal models.<br>
The degree to which the presence or absence of OAS1R genotypes affects other human phenotypes can also be examined. For instance, OAS1R mutations are evaluated for their association with viral liter and spontaneous viral clearance in HCV infected subjects. Similar methods of correlating host OAS1 genotype with the course of other flavivirus infections can<br><br>
also be undertaken. The impact of OAS1R mutations on promoting successful outcomes during interferon or interferon with ribavirin treatment in HCV infected patients is also examined. These mutations may not only confer a level of infection resistance, but also promote spontaneous viral clearance in infected subjects with or without interferon-ribavirin treatment. Furthermore, it has been reported that schizophrenia occurs at a higher frequency in geographic areas that are endemic for flavivirus infection, suggesting an association between flavivirus resistance alleles and predisposition to schizophrenia. This link is evaluated by performing additional genetic association studies involving the schizophrenia phenotype and the OAS1R mutations. The impact of OAS 1R mutations on susceptibility to 1DDM, prostate and other cancers, and schizophrenia will also be evaluated.<br>
The invention discloses OAS1R variant mRNAs (identified as SEQ ID NO:36 through SEQ ID NO:43) thatare novel and have utility. The invention is not limited by the mode of use of the disclosed variant mRNAs. In one preferred embodiment, these variant mRNAs are used in differentially screening human subjects for increased or decreased viral (including HCV) susceptibility. In other preferred embodiments, these variant mRNAs are useful in screening for susceptibility to IDDM, prostate and other cancers, and/or schizophrenia. Such differential screening is performed by expression analyses known to those skilled in the art to determine relative amounts of one or more variant OAS1R mRNAs present in samples derived from a given human subject. Increased or decreased amounts of one or more OAS1R mRNA variants in a human subject's sample relative to a control sample is indicative of the subject's degree of susceptibility to viral, IDDM, prostate and other cancers, and/or schizophrenia, as appropriate to the test under consideration.<br>
As discussed herein, 2',5'-oligoadenylate synthetases (OAS) are a family of IFN-a-inducible, RNA dependent effector molecules enzymes that synthesize short 2' to 5' linked oligoadenylate (2-5A) molecules from ATP. OAS enzymes constitute an important part of the nonspecific immune defense against viral infections and have been used as a cellular marker for viral infection. In addition to the role in hepatitis C infection discussed herein, OAS activity is implicated in other disease states, particularly those in which a viral infection plays a role.<br>
While specific pathogenic mechanisms are subjects of current analysis, viral infections are believed to play a role in the development of diseases such as diabetes. Lymphocytic OAS activity is significantly elevated in patients with type 1 diabetes, suggesting that OAS may be an important link between viral infections and disease development. In a study involving diabetic twins frommonozygotic twin pairs, Bonnevie-Nielsen et al. (Clin Immunol. 2000 Jul;96(l):l 1-8) showed that OAS is persistently activated in both recent-onset and long-standing type 1<br><br>
diabetes. Continuously elevated OAS activity in type 1 diabetes is clearly different from a normal antiviral response and might indicate a chronic stimulation of the enzyme, a failure of down regulatory mechanisms, or an aberrant response to endogenous or exogenous viruses or their products.<br>
A more direct link between a viral infection and the development of diabetes is exemplified by a number of studies showing that between 13 and 33% of patients with chronic hepatitis C have diabetes mellitus (type 2 diabetes), a level that is significantly increased compared with that in matched healthy controls or patients with chronic hepatitis B (Knobler et al. Am J Gastroenterol. 2003 Dec;98(12):2751-6). While OAS has not to date been reported to play a role in the development of diabetes mellitus following hepatitis C infection, it may be a useful marker for the antiviral response system. Furthermore, the results reported according to the present invention illustrate that if hepatitis C infection is causally related to diabetes mellitus, inhibition or abolition of hepatitis C infection using the compositions and methods disclosed herein may be advantageous in preventing or alleviating development of diabetes mellitus.<br>
A further published study has shown that OAS plays an essential role in wound healing and its pathological disorders, particularly in the case of venous ulcers and diabetes-associated poorly-healing wounds (WO 02/090552). in the case of poor wound healing, OAS mRNA levels in the affected tissues were reduced, rather than elevated as in lymphocytes derived from patients suffering from type 1 diabetes. These findings point to OAS as an etiologically important marker of immune reactions in diabetes and diabetes-related wound healing.<br>
OAS may also play an intermediary role in cell processes involved in prostate cancer. A primary biochemical function of OAS is to promote the activity of RNaseL, a uniquely-regulated endoribonuclease that is enzymatically stimulated by 2-5A molecules. RNaseL has a well-established role in mediating the antiviral effects of IFN, and is a strong candidate for the hereditary prostate cancer 1 allele (HPC1). Mutations in RNaseL have been shown to predispose men to an increased incidence of prostate cancer, which in some cases reflect more aggressive disease and/or decreased age of onset compared with non RNase L-linked cases. Xiang et al. (Cancer Res. 2003 Oct 15;63(20):6795-801) demonstrated that biostable phosphorothiolate analogs of 2-5A induced RNaseL activity and caused apoptosis in cultures of late-stage metastatic human prostate cancer cell lines. Their findings suggest that the elevation of OAS activity with a concurrent increase in 2-5 A levels may facilitate the destruction of cancer cells through a potent apoptotic pathway. Thus, use of compositions and methods disclosed herein may find utility in the detection, treatment and/or prevention of prostate cancer.<br>
OAS may further play a role in normal cell growth regulation, either through its<br><br>
regulation of RJMaseL or through another as yet undiscovered pathway. There is considerable evidence to support the importance of OAS in negatively regulating cell growth. Rysiecki et al. (J. Interferon Res. 1989 Dec;9(6):649-57) demonstrated that stable transfection of human OAS into a glioblastoma cell line results in reduced cellular proliferation. OAS levels have also been shown to be measurable in several studies comparing quiescent versus proliferating cell lines (e.g. Hassel and Ts'O, Mol Carcinog. 1992;5(1):41-51 and Kimchi et al., Eur J Biochem. 1981;114(1):5-10) and in each case the OAS levels were greatest in quiescent cells. Other studies have shown a correlation between OAS level and cell cycle phase, with OAS levels rising sharply during late S phase and then dropping abruptly hi G2 (Wells and Mallucci, Exp Cell Res. 1985 Jul;159(l):27-36). Several studies have shown a correlation between the induction of OAS and the onset of antiproliferative effects following stimulation with various forms of interferon (see Player and Torrence, Pharmacol Ther. 1998 May;78(2):55-l 13). Induction of OAS has also been shown during cell differentiation (e.g. Saizberg et al., J Cell Sci. 1996 Jun;109(Pt 6):1517-26 and Schwartz and Nilson, Mol Cell Biol. 1989 Sep;9(9):3897-903). Other reports of induction of OAS by platelet derived growth factor (PDGF) (Zullo et al. Cell. 1985 Dec;43(3 Pt 2):793-800) and under conditions of heat-shock induced growth (Chousterman et al., J Biol Chem. 1987 Apr 5;262(10):4806-11) lead to the hypothesis that induction of OAS is a normal cell growth control mechanism. Thus, use of compositions and methods disclosed herein may find broad utility in the detection, treatment and/or prevention of cancer.<br>
Polvnucleotide Analysis<br>
An oligoadenylate synthetase gene is a nucleic acid whose nucleotide sequence codes for<br>
oligoadenylate synthetase, mutant oligoadenylate synthetase, or oligoadenylate synthetase<br>
pseudogene. It can be in the form of genomic DNA, an mRNA or cDNA, and in single or<br>
double stranded form. Preferably, genomic DNA is used because of its relative stability in<br>
biological samples compared to mRNA. The sequence of a polynucleotide consisting of<br>
consecutive nucleotides 2,130,000 - 2,157,999 of the complete genomic sequence of the<br>
reference oligoadenylate synthetase gene is provided in the Sequence Listing as SEQ ID NO: 19,<br>
and corresponds to Genbank Accession No. NT 009775.13.<br>
The nucleic acid sample is obtained from cells, typically peripheral blood leukocytes. Where<br>
mRNA is used, the cells are lysed under RNase inhibiting conditions. In one embodiment, the<br>
first step is to isolate the total cellular mRNA. Poly A+ mRNA can then be selected by<br>
hybridization to an oligo-dT cellulose column.<br>
In preferred embodiments, the nucleic acid sample is enriched for a presence of oligoadenylate<br><br>
syntiietase anenc material. Emichment is typically accomplished by subjecting the genomic DNA or mRNA to a primer extension reaction employing a polynucleotide synthesis primer as described herein. Particularly preferred methods for producing a sample to be assayed use preselected polynucleotides as primers in a polymerase chain reaction (PCR) to form an amplified (PCR) product. Preparation of Polynucleotide Primers<br>
The term "polynucleotide" as used herein in reference to primers, probes and nucleic acid fragments or segments to be synthesized by primer extension is defined as a molecule comprised of two or more deoxyribonucleotides or ribonucleotides, preferably more than three. Its exact size will depend on many factors, which in turn depends on the ultimate conditions of use. The term "primer" as used herein refers to a polynucleotide whether purified from a nucleic acid restriction digest or produced synthetically, which is capable of acting as a point of initiation of nucleic acid synthesis when placed under conditions in which synthesis of a primer extension product which is complementary to a nucleic acid strand is induced, i.e., in the presence of nucleotides and an agent for polymerization such as DNA polymerase, reverse transcriptase and the like, and at a suitable temperature and pH. The primer is preferably single stranded for maximum efficiency, but may alternatively be in double stranded form. If double stranded, the primer is first treated to separate it from its complementary strand before being used to prepare extension products. Preferably, the primer is a polydeoxyribonucleotide. The primer must be sufficiently long to prime the synthesis of extension products in the presence of the agents for polymerization. The exact lengths of the primers will depend on many factors, including temperature and the source of primer. For example, depending on the complexity of the target sequence, a polynucleotide primer typically contains 15 to 25 or more nucleotides, although it can contain fewer nucleotides. Short primer molecules generally require cooler temperatures to form sufficiently stable hybrid complexes with template.<br>
The primers used herein are selected to be "substantially complementary to the different strands of each specific sequence to be synthesized or amplified. This means that the primer must be sufficiently complementary to non-randomly hybridize with its respective template strand. Therefore, the primer sequence may or may not reflect the exact sequence of the template. For example, a non-complementary nucleotide fragment can be attached to the 5' end of the primer, with the remainder of the primer sequence being substantially complementary to the strand. Such non-complementary fragments typically code for an endonuclease restriction site. Alternatively, non-complementary bases or longer sequences can be interspersed into the primer, provided the primer sequence has sufficient complementarity with the sequence of the strand to<br><br>
be synthesized or amplified to non-randomly hybridize therewith and thereby form an extension product under polynucleotide synthesizing conditions.<br>
Primers of the present invention may also contain a DNA-dependent RNA polymerase promoter sequence or its complement. See for example, Krieg, et al., Nucl. Acids Res., 12:7057-70 (1984); Studier, et al., J. Mol. Biol., 189:113-130 (1986); and Molecular Cloning: A Laboratory Manual, Second Edition, Maniatis, et al., eds., Cold Spring Harbor, N.Y. (1989). When a primer containing a DNA-dependent RNA polymerase promoter is used, the primer is hybridized to the polynucleotide strand to be amplified and the second polynucleotide strand of the DNA-dependent RNA polymerase promoter is completed using an inducing agent such as E. coli DNA polymerase I, or the Klenow fragment of E. coli DNA polymerase. The starting polynucleotide is amplified by alternating between the production of an RNA polynucleotide and DNA polynucleotide.<br>
Primers may also contain a template sequence or replication initiation site for a RNA-directed RNA polymerase. Typical RNA-directed RNA polymerase include the QB replicase described by Lizardi, et al., Biotechnology, 6:1197-1202 1988). RNA-directed polymerases produce large numbers of RNA strands from a small number of template RNA strands that contain a template sequence or replication initiation site. These polymerases typically give a one million-fold amplification of the template strand as has been described by Kramer, et al., J. Mol. Biol., 89:719-736(1974).<br>
The polynucleotide primers can be prepared using any suitable method, such as, for example, the phosphotriester or phosphodiester methods see Narang, et al., Meth. Enzymol, 68:90, (1979); U.S. Pat. Nos. 4,356,270, 4,458,066, 4,416,988,4,293,652; and Brown, et al., Meth. Enzymol., 68:109(1979).<br>
The choice of a primer's nucleotide sequence depends on factors such as the distance on the nucleic acid from the hybridization point to the region coding for the mutation to be detected, its hybridization site on the nucleic acid relative to any second primer to be used, and the like. If the nucleic acid sample is to be enriched for oligoadenylate synthetase gene material by PCR amplification, two primers, i.e., a PCR primer pair, must be used for each coding strand of nucleic acid to be amplified. The first primer becomes part of the non-coding (anti-sense or minus or complementary) strand and hybridizes to a nucleotide sequence on the plus or coding strand. Second primers become part of the coding (sense or plus) strand and hybridize to a nucleotide sequence on the minus or non-coding strand. One or both of the first and second primers can contain a nucleotide sequence defining an endonuclease recognition site. The site can be heterologous to the oligoadenylate synthetase gene being amplified.<br><br>
in one embodiment, the present invention utilizes a set of polymicleotides that form primers having a priming region located at the 3 '-terminus of the primer. The priming region is typically the 3'-most (3'-terminal) 15 to 30 nucleotide bases. The 3'-terminal priming portion of each primer is capable of acting as a primer to catalyze nucleic acid synthesis, i.e., initiate a primer extension reaction off its 3' terminus. One or both of the primers can additionally contain a 5'-terminal (5'-most) non-priming portion, i.e., a region that does not participate in hybridization to the preferred template.<br>
In PCR, each primer works in combination with a second primer to amplify a target nucleic acid sequence. The choice of PCR primer pairs for use in PCR is governed by considerations as discussed herein for producing oligoadenylate synthetase gene regions. When a primer sequence is chosen to hybridize (anneal) to a target sequence within an oligoadenylate synthetase gene allele intron, the target sequence should be conserved among the alleles in order to insure generation of taigut sequence to be assayed. Polymerase Chain Reaction<br>
Oligoadenylate synthetase genes are comprised of polynucleotide coding strands, such as mRNA and/or the sense strand of genomic DMA. If the genetic material to be assayed is in the form of double stranded genomic DNA, it is usually first denatured, typically by melting, into single strands. The nucleic acid is subjected to a PCR reaction by treating (contacting) the sample with a PCR primer pair, each member of the pair having a preselected nucleotide sequence. The PCR primer pair is capable of initiating primer extension reactions by hybridizing to nucleotide sequences, preferably at least about 10 nucleotides in length, more preferably at least about 20 nucleotides in length, conserved within the oligoadenylate synthetase alleles. The first primer of a PCR primer pair is sometimes referred to herein as the "anti-sense primer" because it hybridizes to a non-coding or anti-sense strand of a nucleic acid, i.e., a strand complementary to a coding strand. The second primer of a PCR primer pair is sometimes referred to herein as the "sense primer" because it hybridizes to the coding or sense strand of a nucleic acid.<br>
The PCR reaction is performed by mixing the PCR primer pair, preferably a predetermined amount thereof, with the nucleic acids of the sample, preferably a predetermined amount thereof, in a PCR buffer to form a PCR reaction admixture. The admixture is thermocycled for a number of cycles, which is typically predetermined, sufficient for the formation of a PCR reaction product, thereby enriching the sample to be assayed for oligoadenylate synthetase genetic material. PCR is typically carried out by thermocycling i.e., repeatedly increasing and decreasing the<br><br>
temperature of a PCR reaction admixture within a temperature range whose lower limit is about 30 degrees Celsius (30 C.) to about 55 C. and whose upper limit is about 90 C. to about 100 0 C. The increasing and decreasing can be continuous, but is preferably phasic with time periods of relative temperature stability at each of temperatures favoring polynucleotide synthesis, denaturation and hybridization.<br>
A plurality of first primer and/or a plurality of second primers can be used in each amplification, e.g., one species of first primer can be paired with a number of different second primers to form several different primer pairs. Alternatively, an individual pair of first and second primers can be used. In any case, the amplification products of amplifications using the same or different combinations of first and second primers can be combined for assaying for mutations. The PCR reaction is performed using any suitable method. Generally it occurs in a buffered aqueous solution, i.e., a PCR buffer, preferably at a pH of 7-9, most preferably about 8. Preferably, a molar excess (for genomic nucleic acid, usually about 106:1 primentempiate) of the primer is admixed to the buffer containing the template strand. A large molar excess is preferred to improve the efficiency of the process.<br>
The PCR buffer also contains the deoxyribonucleotide triphosphates (polynucleotide synthesis substrates) dATP, dCTP, dGTP, and dTTP and a polymerase, typically thermostable, all in adequate amounts for primer extension (polynucleotide synthesis) reaction. The resulting solution (PCR admixture) is heated to about 90 C. -100 C. for about 1 to 10 minutes, preferably from 1 to 4 minutes. After this heating period the solution is allowed to cool to 54 C., which is preferable for primer hybridization. The synthesis reaction may occur at from room temperature up to a temperature above which the polymerase (inducing agent) no longer functions efficiently. The thermocycling is repeated until the desired amount of PCR product is produced. An exemplary PCR buffer comprises the following: 50 mM KC1; 10 mM Tris-HCl at pH 8.3; 1.5 mM MgCl.; 0.001% (wt/vol) gelatin, 200 uM dATP; 200 uM dTTP; 200 uM dCTP; 2002 uM dGTP; and 2.5 units Thermus aquaticus (Taq) DNA polymerase I (U.S. Pat. No. 4,889,818) per 100 microliters of buffer.<br>
The inducing agent may be any compound or system which will function to accomplish the synthesis of primer extension products, including enzymes. Suitable enzymes for this purpose include, for example, E. coli DNA polymerase I, Klenow fragment of E. coli DNA polymerase I, T4 DNA polymerase, other available DNA polymerases, reverse transcriptase, and other enzymes, including heat-stable enzymes, which will facilitate combination of the nucleotides in the proper manner to form the primer extension products which are complementary to each nucleic acid strand. Generally, the synthesis will be initiated at the 3' end of each primer and<br><br>
proceed in the 5' direction along the template strand, until synthesis terminates, producing molecules of different lengths. There may be inducing agents, however, which initiate synthesis at the 5' end and proceed in the above direction, using the same process as described above. The inducing agent also may be a compound or system which will function to accomplish the synthesis of RNA primer extension products, including enzymes. In preferred embodiments, the inducing agent may be a DNA-dependent RNA polymerase such as T7 RNA polymerase, T3 RNA polymerase or SP6 RNA polymerase. These polymerases produce a complementary RNA polynucleotide. The high turn-over rate of the RNA polymerase amplifies the starting polynucleotide as has been described by Chamberlin, et al., The Enzymes, ed. P. Boyer, pp. 87-108, Academic Press, New York (1982). Amplification systems based on transcription have been described by Gingeras, et al., in PCR Protocols, A Guide to Methods and Applications, pp. 245-252, Innis, et al., eds. Academic Press, Inc., San Diego, Calif. (1990). If the inducing agent is a DNA-dependent RNA polymerase and, therefore incorporates ribonucleotide triphosphates, sufficient amounts of ATP, CTP, GTP and UTP are admixed to the primer extension reaction admixture and the resulting solution is treated as described above. The newly synthesized strand and its complementary nucleic acid strand form a double-stranded molecule which can be used in the succeeding steps of the process.<br>
The PCR reaction can advantageously be used to incorporate into the product a preselected restriction site useful in detecting a mutation in the oligoadenylate synthetase gene. PCR amplification methods are described in detail in U.S. Pat. Nos. 4,683,192, 4,683,202, 4,800,159, and 4,965,188, and at least in several texts including PCR Technology: Principles and Applications for DNA Amplification, H. Erlich, ed., Stockton Press, New York (1989); and PCR Protocols: A Guide to Methods and Applications, Innis, et al., eds., Academic Press, San Diego, Calif. (1990).<br>
In some embodiments, two pairs of first and second primers are used per amplification reaction. The amplification reaction products obtained from a plurality of different amplifications, each using a plurality of different primer pairs, can be combined or assayed separately. However, the present invention contemplates amplification using only one pair of first and second primers. Exemplary primers for amplifying the sections of DNA containing the mutations disclosed herein are shown below in Table 1. AmpliconA corresponds to the polynucleotide sequence that contains the mutations referred to in SEQ ID NO: 1-3. AmpliconB corresponds to the polynucleotide sequence containing the mutations referred to in SEQ ID NO:4-7 and SEQ ID NO:60. AmpliconC corresponds to the polynucleotide sequence containing the mutation referred to in SEQ ID NO: 57. AmpliconD corresponds to the polynucleotide<br><br>
sequence containing the mutation referred to in SEQ ID NO:58. AmpliconE corresponds to the polynucleotide sequence containing the mutation referred to in SEQ ID NO: 59. AmpliconF corresponds to the polynucleotide sequence containing the mutation referred to in SEQ ID NO:61. AmpliconG corresponds to the polynucleotide sequences containing the mutation referred to in SEQ ID NO: 62-64.<br>
TABLE REMOVED<br>
Nucleic Acid Sequence Analysis<br>
Nucleic acid sequence analysis is approached by a combination of (a) physiochemical<br>
techniques, based on the hybridization or denaturation of a probe strand plus its complementary<br>
target, and (b) enzymatic reactions with endonucleases, ligases, and polymerases. Nucleic acid<br>
can be assayed at the DNA or KNA level. The former analyzes the genetic potential of<br>
individual humans and the latter the expressed information of particular cells.<br>
In assays using nucleic acid hybridization, detecting the presence of a DNA duplex in a process<br>
of the present invention can be accomplished by a variety of means.<br>
In one approach for detecting the presence of a DNA duplex, an oligonucleotide that is<br>
hybridized in the DNA duplex includes a label or indicating group that will render the duplex<br><br>
detectable. Typically such labels include radioactive atoms, chemically modified nucleotide bases, and the like.<br>
The oligonucleotide can be labeled, i.e., operatively linked to an indicating means or group, and used to detect the presence of a specific nucleotide sequence in a target template. Radioactive elements operatively linked to or present as part of an oligonucleotide probe (labeled oligonucleotide) provide a useful means to facilitate the detection of a DNA duplex. A typical radioactive element is one that produces beta ray emissions. Elements that emit beta rays, such as 3H, 12C, 32P and 35S represent a class of beta ray emission-producing radioactive element labels. A radioactive polynucleotide probe is typically prepared by enzymatic incorporation of radioactively labeled nucleotides into a nucleic acid using DNA kinase. Alternatives to radioactively labeled oligonucleotides are oligonucleotides that are chemically modified to contain metal complexing agents, biotin-containing groups, fluorescent compounds, and the like.<br>
One useful metal complexing agent is a lanthanide chelate formed by a lanthanide and an aromatic beta-diketone, the lanthanide being bound to the nucleic acid or oligonucleotide via a chelate-forniing compound such as an EDTA-analogue so that a fluorescent lanthanide complex is formed. See U.S. Pat. Nos. 4,374,120, 4,569,790 and published Patent Application EP0139675 and W087/02708.<br>
Biotin or acridine ester-labeled oligonucleotides and their use to label polynucleotides have been described. See U.S. Pat. No. 4,707,404, published Patent Application EP0212951 and European Patent No. 0087636. Useful fluorescent marker compounds include fluorescein, rhodamine, Texas Red, NBD and the like.<br>
A labeled oligonucleotide present in a DNA duplex renders the duplex itself labeled and therefore distinguishable over other nucleic acids present in a sample to be assayed. Detecting the presence of the label in the duplex and thereby the presence of the duplex, typically involves separating the DNA duplex from any labeled oligonucleotide probe that is not hybridized to a DNA duplex.<br>
Techniques for the separation of single stranded oligonucleotide, such as non-hybridized labeled oligonucleotide probe, from DNA duplex are well known, and typically involve the separation of single stranded from double stranded nucleic acids on the basis of their chemical properties. More often separation techniques involve the use of a heterogeneous hybridization format in which the non-hybridized probe is separated, typically by washing, from the DNA duplex that is bound to an insoluble matrix.  Exemplary is the Southern blot technique, in which the matrix is a nitrocellulose sheet and the label is 32P. Southern, J. Mol. Biol, 98:503 (1975).<br><br>
The oligonucleotides can also be advantageously linked, typically at or near their 5'-terminus, to a solid matrix, i.e., aqueous insoluble solid support. Useful solid matrices are well known in the art and include cross-linked dextran such as that available under the tradename SEPHADEX from Pharmacia Fine Chemicals (Piscataway, N.J.); agarose, polystyrene or latex beads about 1 micron to about 5 millimeters in diameter, polyvinyl chloride, polystyrene, cross-linked polyacrylamide, nitrocellulose or nylon-based webs such as sheets, strips, paddles, plates microtiter plate wells and the like.<br>
It is also possible to add "linking" nucleotides to the 5' or 3' end of the member oligonucleotide, and use the linking oligonucleotide to operatively link the member to the solid support. In nucleotide hybridizing assays, the hybridization reaction mixture is maintained in the contemplated method under hybridizing conditions for a time period sufficient for the oligonucleotides having complementarity to the predetermined sequence on the template to hybridize to complementary nucleic acid sequences present in the template to form a hybridization product, i.e., a complex containing oligonucleotide and target nucleic acid. The phrase "hybridizing conditions" and its grammatical equivalents, when used with a maintenance time period, indicates subjecting the hybridization reaction admixture, in the context of the concentrations of reactants and accompanying reagents in the admixture, to time, temperature and pH conditions sufficient to allow one or more oligonucleotides to anneal with the target sequence, to form a nucleic acid duplex. Such time, temperature and pH conditions required to accomplish hybridization depend, as is well known in the art, on the length of the oligonucleotide to be hybridized, the degree of complementarity between the oligonucleotide and the target, the guanine and cytosine content of the oligonucleotide, the stringency of hybridization desired, and the presence of salts or additional reagents in the hybridization reaction admixture as may affect the kinetics of hybridization. Methods for optimizing hybridization conditions for a given hybridization reaction admixture are well known in the art. Typical hybridizing conditions include the use of solutions buffered to pH values between 4 and 9, and are carried out at temperatures from 4 C. to 37 C., preferably about 12 C. to about 30 C., more preferably about 22 C., and for time periods from 0.5 seconds to 24 hours, preferably 2 minutes (min) to 1 hour.<br>
Hybridization can be earned out in a homogeneous or heterogeneous format as is well known. The homogeneous hybridization reaction occurs entirely in solution, in which both the oligonucleotide and the nucleic acid sequences to be hybridized (target) are present in soluble forms in solution. A heterogeneous reaction involves the use of a matrix that is insoluble in the reaction medium to which either the oligonucleotide, polynucleotide probe or target nucleic acid<br><br>
is bound.<br>
Where the nucleic acid containing a target sequence is in a double stranded (ds) form, it is<br>
preferred to first denature the dsDNA, as by heating or alkali treatment, prior to conducting the<br>
hybridization reaction. The denaturation of the dsDNA can be carried out prior to admixture<br>
with an oligonucleotide to be hybridized, or can be carried out after the admixture of the dsDNA<br>
with the oligoiiucleotide.<br>
Predetermined complementarity between the oligonucleotide and the template is achieved in two<br>
alternative manners. A sequence in the template DNA may be known, such as where the primer<br>
to be formed can hybridize to known oligoadenylate synthetase sequences and can initiate<br>
primer extension into a region of DNA for sequencing purposes, as well as subsequent assaying<br>
purposes as described herein, or where previous sequencing has determined a region of<br>
nucleotide sequence and the primer is designed to extend from the recently sequenced region<br>
into a region ot unknown sequence. This latter process has been referred to a "directed<br>
sequencing" because each round of sequencing is directed by a primer designed based on the<br>
previously determined sequence.<br>
Effective amounts of the oligonucleotide present in the hybridization reaction admixture are<br>
generally well known and are typically expressed in terms of molar ratios between the<br>
oligonucleotide to be hybridized and the template. Preferred ratios are hybridization reaction<br>
mixtures containing equimolar amounts of the target sequence and the oligonucleotide. As is<br>
well known, deviations from equal molarity will produce hybridization reaction products,<br>
although at lower efficiency. Thus, although ratios where one component can be in as much as<br>
100 fold molar excess relative to the other component, excesses of less than 50 fold, preferably<br>
less than 10 fold, and more preferably less than two fold are desirable in practicing the<br>
invention.<br>
Detection of Membrane-Immobilized Target Sequences<br>
In the DNA (Southern) blot technique, DNA is prepared by PCR amplification as previously<br>
discussed. The PCR products (DNA fragments) are separated according to size in an agarose gel<br>
and transferred (blotted) onto a nitrocellulose or nylon membrane. Conventional electrophoresis<br>
separates fragments ranging from 100 to 30,000 base pairs while pulsed field gel electrophoresis<br>
resolves fragments up to 20 million base pairs in length. The location on the membrane a<br>
containing particular PCR product is determined by hybridization with a specific, labeled<br>
nucleic acid probe.<br>
In preferred embodiments, PCR products are directly immobilized onto a solid-matrix<br>
(nitrocellulose membrane) using a dot-blot (slot-blot) apparatus, and analyzed by probe-<br><br>
hybridization. See U.S. Patents Nos. 4,582,789 and 4,617,261.<br>
Immobilized DNA sequences may be analyzed by probing with allele-specific oligonucleotide (ASO) probes, which are synthetic DNAn oligomers of approximately 15, 17, 20, 25 or up to about 30 nucleotides in length. These probes are long enough to represent unique sequences in the genome, but sufficiently short to be destabilized by an internal mismatch in their hybridization to a target molecule. Thus, any sequences differing at single nucleotides may be distinguished by the different denaturation behaviors of hybrids between the ASO probe and normal or mutant targets under carefully controlled hybridization conditions. Exemplary probes are disclosed herein as SEQ ID NO:l-7 and SEQ ID NO:57-64 (Table 3), but any probes are suitable as long as they hybridize specifically to the region of the OAS1 gene carrying the point mutation of choice, and are capable of specifically distinguishing between polynucleotide carrying the point mutation and a wild type polynucleotide. Detection ot Target Sequences in Solution<br>
Several rapid techniques that do not require nucleic acid purification or immobilization have been developed. For example, probe/target hybrids may be selectively isolated on a solid matrix, such as hydroxylapatite, which preferentially binds double-stranded nucleic acids. Alternatively, probe nucleic acids may be immobilized on a solid support and used to capture target sequences from solution. Detection of the target sequences can be accomplished with the aid of a second, labeled probe that is either displaced from the support by the target sequence in a competition-type assay or joined to the support via the bridging action of the target sequence in a sandwich-type format.<br>
In the oligonucleotide ligation assay (OLA), the enzyme DNA ligase is used to covalently join two synthetic oligonucleotide sequences selected so that they can base pair with a target sequence in exact head-to-tail juxtaposition. Ligation of the two oligomers is prevented by the presence of mismatched nucleotides at the junction region. This procedure allows for the distinction between known sequence variants in samples of cells without the need for DNA purification. The joining of the two oligonucleotides may be monitored by immobilizing one of the two oligonucleotides and observing whether the second, labeled oligonucleotide is also captured.<br>
Scanning Techniques for Detection of Base Substitutions<br>
Three techniques permit the analysis of probe/target duplexes several hundred base pairs in length for unknown single-nucleotide substitutions or other sequence differences. In the ribonuclease (RNase) A technique, the enzyme cleaves a labeled RNA probe at positions where it is mismatched to a target RNA or DNA sequence. The fragments may be separated according<br><br>
to size allowing for the determination of the approximate position of the mutation. See U.S. Pat.<br>
No. 4,946,773.<br>
In the denaturing gradient gel technique, a probe-target DNA duplex is analyzed by<br>
electrophoresis in a denaturing gradient of increasing strength. Denaturation is accompanied by<br>
a decrease in migration rate. A duplex with a mismatched base pair denatures more rapidly than<br>
a perfectly matched duplex.<br>
A third method relies on chemical cleavage of mismatched base pairs. A mismatch between T<br>
and C, G, or T, as well as mismatches between C and T, A, or C, can be detected in<br>
heteroduplexes. Reaction with osmium tetroxide (T and C mismatches) or hydroxylamine (C<br>
mismatches) followed by treatment with piperidine cleaves the probe at the appropriate<br>
mismatch.<br>
Therapeutic agents for restoring and/or enhancing OAS1 function<br>
Where a mutation in the OAS1 gene leads to defective OAS1 function and this defective<br>
function is associated with increased susceptibility of a patient to pathogenic infection, whether<br>
through lower levels of OAS 1 protein, mutation in the protein affecting its function, or other<br>
mechanisms, it may be advantageous to treat the patient with wild type OAS1 protein.<br>
Furthermore, if the mutation gives rise in infection-resistant carriers to a form of the protein that<br>
differs from the wild-type protein, and that has an advantage in terms of inhibiting HCV<br>
infection, it may be advantageous to administer a protein encoded by the mutated gene. As<br>
described previously, administration of either native or mutant forms of OAS 1 proteins or<br>
polypeptides may also be advantageous in the treatment of other indications including but not<br>
limited to cancer, diabetes mellitus, and wound healing. The discussion below pertains to<br>
administration of any of the foregoing proteins or polypeptides.<br>
The polypeptides of the present invention, including those encoded by OAS1R, may be a<br>
naturally purified product, or a product of chemical synthetic procedures, or produced by<br>
recombinant techniques from a prokaryotic or eukaryotic host (for example, by bacterial, yeast,<br>
higher plant, insect and mammalian cells in culture) of a polynucleotide sequence of the present<br>
invention. Depending upon the host employed in a recombinant production procedure, the<br>
polypeptides of the present invention may be glycosylated with mammalian or other eukaryotic<br>
carbohydrates or may be non-glycosylated. Polypeptides of the invention may also include an<br>
initial methionine amino acid residue (at position minus 1).<br>
The polypeptides of the present invention also include the protein sequences defined in SEQ ID<br>
NO: 20, SEQ ID NO: 21, SEQ ID NO: 22, SEQ ID NO: 23, SEQ ID NO: 24 and SEQ ID NO:<br>
25, SEQ ID NO: 26, SEQ ID NO: 27, SEQ ID NO: 28, SEQ ID NO: 29, SEQ ID NO: 32, SEQ<br><br>
ID NO: 33, SEQ ID NO: 34, SEQ ID NO: 35, SEQ ID NO: 46, and SEQ ID NO: 47 and derivatives thereof.<br>
In addition to naturally occurring allelic forms of the polypeptide(s) the present invention also embraces analogs and fragments thereof, which function similarly to the naturally occurring allelic forms. Thus, for example, one or more of the amino acid residues of the polypeptide may be replaced by conserved amino acid residues, as long as the function of the OAS1R protein is maintained. Examples 8-10, below, provide representative illustrations of suitable amino acid replacements with regard to the polypeptides of the present invention. As another example, the polypeptides of the present invention specifically include the truncated or analog forms of OAS1R defined in SEQ ID NO: 48, SEQ ID NO: 49, SEQ ID NO: 50, SEQ ID NO: 51 and SEQ ID NO:52. As discussed previously, SEQ ID NO: 51 represents the shortened form of OAS1 possessed by chimpanzees and SEQ ID NO: 52 represents a carboxyl-terminus fragment of the longer but still truncated form possessed by gorillas. SEQ ID NO: 49 and SEQ ID NO: 50 represent synthetic human OAS1R constructs truncated to the corresponding chimpanzee and gorilla sites of truncation, respectively. SEQ ID NO: 48 represents a synthetic human OAS1R polypeptide truncated to a length intermediate to the chimpanzee and gorilla forms. SEQ ID NO: 48 has further been demonstrated to be enzymatically active by methods known in the art as disclosed elsewhere herein. Correspondingly, the remaining highly similar truncated forms may also be demonstrated to be enzymatically active. As those skilled in the art will appreciate, therapeutic use of truncated but functional forms of OAS1R polypeptides can preclude the development of antibody response which would otherwise hinder the therapeutic efficacy of the polypeptide. The foregoing truncated polypeptides, and others that can be envisioned by one skilled in the art, maintain function but remove non-ubiquitous portions of the polypeptide that could induce antibody response in individuals not possessing the full length OAS1R polypeptide endogenously. Those skilled in the art will also appreciate that smaller polypeptides, in general, are more amenable to the complexities of manufacturing, delivery, and clearance typically encountered in therapeutic development. Additionally, those skilled in the art will appreciate that the occurrence of distinct homozygous truncating variants in chimpanzee and gorilla are also highly suggestive for the broad anti-viral potency of the presently disclosed truncated OAS1 forms. Although the truncated polypeptide forms specifically disclosed above represent truncations to the carboxyl-terminus of the polypeptide, the invention is not limited by the form of the fragment and specifically includes amino-terminus truncations and internal amino acid deletions that retain enzymatic function.<br>
Also included in the scope of the invention are polypeptides that retain at least one<br><br>
activity of a specific disclosed polypeptide, but differ from the disclosed amino acid sequence.<br>
Such polypeptides preferably have at least 80% sequence homology, preferably 85% sequence<br>
homology, more preferably 90% sequence homology, most preferably 95% more sequence<br>
homology to the corresponding disclosed SEQ ID NO: as calculated using standard methods of<br>
alignment.<br>
The polypeptides may also be employed in accordance with the present invention by expression<br>
of such polypeptides in vivo, which is often referred to as gene therapy. Thus, for example, cells<br>
may be transduced with a polynucleotide (DNA or RNA) encoding the polypeptides ex vivo<br>
with those transduced cells then being provided to a patient to be treated with the polypeptide.<br>
Such methods are well known in the art. For example, cells may be transduced by procedures<br>
known in the art by use of a retroviral particle containing RNA encoding the polypeptide of the<br>
present invention.<br>
Similarly, transduction of cells may be accomplished in vivo for expression of the polypeptide in<br>
vivo, for example, by procedures known in the art. As known in the art, a producer cell for<br>
producing a retroviral particle containing RNA encoding the polypeptides of the present<br>
invention may be administered to a patient for transduction in vivo and expression of the<br>
polypeptides in vivo.<br>
These and other methods for administering the polypeptides of the present invention by such<br>
methods should be apparent to those skilled in the art from the teachings of the present<br>
invention. For example, the expression vehicle for transducing cells may be other than a<br>
retrovirus, for example, an adenovirus which may be used to transduce cells in vivo after<br>
combination with a suitable delivery vehicle.<br>
In the case where the polypeptides are prepared as a liquid formulation and administered by<br>
injection, preferably the solution is an isotonic salt solution containing 140 millimolar sodium<br>
chloride and 10 millimolar calcium at pH 7.4. The injection may be administered, for example,<br>
in a therapeutically effective amount, preferably in a dose of about 1 ug/kg body weight to about<br>
5 mg/kg body weight daily, taking into account the routes of administration, health of the<br>
patient, etc.<br>
The polypeptide(s) of the present invention may be employed in combination with a suitable<br>
pharmaceutical carrier. Such compositions comprise a therapeutically effective amount of the<br>
protein, and a pharmaceutically acceptable carrier or excipient. Such a carrier includes but is not<br>
limited to saline, buffered saline, dextrose, water, glycerol, ethanol, and combinations thereof.<br>
The formulation should suit the mode of administration.<br>
The polypeptide(s) of the present invention can also be modified by chemically linking the<br><br>
polypeptide to one or more moieties or conjugates to enhance the activity, cellular distribution, or cellular uptake of the polypeptide(s). Such moieties or conjugates include lipids such as cholesterol, cholic acid, thioether, aliphatic chains, phospholipids and their derivatives, polyamines, polyethylene glycol (PEG), palmityl moieties, and others as disclosed in, for example, U.S. Patents 5,514,758, 5,565,552, 5,567,810, 5,574,142, 5,585,481, 5,587,371, 5,597,696 and 5,958,773.<br>
The polypeptide(s) of the present invention may also be modified to target specific cell types for a particular disease indication, including but not limited to liver cells in the case of hepatitis C infection. As can be appreciated by those skilled in the art, suitable methods have been described that achieve the described targeting goals and include, without limitation, liposomal targeting, receptor-mediated endocytosis, and antibody-antigen binding. In one embodiment, the asiaglycoprotein receptor may be used to target liver cells by the addition of a galactose moiety to the polypeptide(s). In another embodiment, mannose moieties may be conjugated to the polypeptide(s) in order to target the mannose receptor found on macrophages and liver cells. The polypeptide(s) of the present invention may also be modified for cytosolic delivery by methods known to those skilled in the art, including, but not limited to, endosome escape mechanisms or protein transduction domain (PTD) systems. PTD systems are disclosed in, for example, Vives E, et al. (1997) J. Biol. Chem. 272: 16010-16017, Derossi, et al. (1994) J. Biol. Chem. 269: 10444-10450, Elliott, G et al. (1997) Cell 88:223-233, Wadia, JS et al. (2004) Nat. Med 10:310-315, and Kabouridis, PS. (2003) Trends Biotech., 21: 498-503. Known endosome escape systems include the use of ph-responsive polymeric carriers such as poly(propylacrylic acid). Known PTD systems range from natural peptides such as HIV-1 TAT, HSV-1 VP22, Drosophila Antennapedia, or diphtheria toxin to synthetic pep tide carriers (Wadia and Dowdy, Cur. Opin. Biotech. 13:52-56, 2002; Becker-Hapak et. al., Methods 24:247-256,2001). Figure 10 provides detailed description of several of these exemplary PTDs. As one skilled in the art will recognize, multiple delivery and targeting methods may be combined. For example, the polypeptide(s) of the present invention may be targeted to liver cells by encapsulation within liposomes, such liposomes being conjugated to galactose for targeting to the asialoglycoprotein receptor.<br>
The invention also provides a pharmaceutical pack or kit comprising one or more containers filled with one or more of the ingredients of the pharmaceutical compositions of the invention. Associated with such containers) can be a notice in the form prescribed by a governmental agency regulating the manufacture, use or sale of Pharmaceuticals or biological products, which notice reflects approval by the agency of manufacture, use or sale for human administration. In<br><br>
addition, the polypeptide of the present invention may be employed in conjunction with other therapeutic compounds.<br>
When the OAS1 variants of the present invention are used as a pharmaceutical, they can be given to mammals, in a suitable vehicle. When the polypeptides of the present invention are used as a pharmaceutical as described above, they are given, for example, in therapeutically effective doses of about 10 ug/kg body weight to about 4 mg/kg body weight daily, taking into account the routes of administration, health of the patient, etc. The amount given is preferably adequate to achieve prevention or inhibition of infection by a virus, preferably a flavivirus, most preferably HCV, thus replicating the natural resistance found in humans carrying an OAS1R allele as disclosed herein.<br>
Inhibitor-based drug therapies that mimic the beneficial effects of at least one mutation at position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321,2131025, 2133961, 2139587, 2144294, 2144985, 2156523,2156595, or 2156638 of NT_009775.13 are also envisioned, as discussed in detail below. As discussed previously, one exemplary rationale for developing such inhibitors is the case where the beneficial mutation diminishes or eradicates expression, translation, or function of one or more particular isoforms of OAS 1. The present invention is not limited by the precise form or effect of the beneficial mutation nor the biological activity of the particular isoforms thereby affected. In such case, one skilled in the art will appreciate the utility of therapeutically inhibiting said particular isoform(s) of OAS 1. These inhibitor-based therapies can take the form of chemical entities, peptides or proteins, antisense oligonucleotides, small interference RNAs, and antibodies.<br>
The proteins, their fragments or other derivatives, or analogs thereof, or cells expressing them can be used as an immunogen to produce antibodies thereto. These antibodies can be, for example, polyclonal, monoclonal, chimeric, single chain, Fab fragments, or the product of an Fab expression library. Various procedures known in the art may be used for the production of polyclonal antibodies.<br>
Antibodies generated against the polypeptide encoded by OAS1R of the present invention can be obtained by direct injection of the polypeptide into an animal or by administering the polypeptide to an animal, preferably a nonhuman. The antibody so obtained will then bind the polypeptide itself. In this manner, even a sequence encoding only a fragment of the polypeptide can be used to generate antibodies binding the whole native polypeptide. Moreover, a panel of such antibodies, specific to a large number of polypeptides, can be used to identify and differentiate such tissue. As an example, Figure 9 demonstrates development of antibodies specific to particular exemplary polypeptides of the present invention.<br><br>
For preparation of monoclonal antibodies, any technique which provides antibodies produced by<br>
continuous cell line cultures can be used. Examples include the hybridoma technique (Kohler<br>
andMilstein, 1975, Nature, 256:495-597), the trioma technique, the human B-cell hybridoma<br>
technique (Kozbor, et al., 1983, Immunology Today 4:72), and the EBV-hybridoma technique to<br>
produce human monoclonal antibodies (Coe, et al., 1985, Monoclonal Antibodies and Cancer<br>
Therapy, Alan R. Liss, Inc. pp. 77-96).<br>
Techniques described for the production of single chain antibodies (U.S. Pat. No. 4,946,778) can<br>
be adapted to produce single chain antibodies to immunogenic polypeptide products of this<br>
invention.<br>
The antibodies can be used in methods relating to the localization and activity of the protein<br>
sequences of the invention, e.g., for imaging these proteins, measuring levels thereof in<br>
appropriate physiological samples, and the like.<br>
The present invention provides detectably labeled oligonucleotides for imaging OAS1<br>
polynucleotides within a cell. Such oligonucleotides are useful for determining if gene<br>
amplification has occurred, and for assaying the expression levels in a cell or tissue using, for<br>
example, in situ hybridization as is known in the art.<br>
Therapeutic agents for inhibition of OAS 1 function<br>
The present invention also relates to antisense oligonucleotides designed to interfere with the normal function of OAS 1 polynucleotides. Any modifications or variations of the antisense molecule which are known in the art to be broadly applicable to antisense technology are included within the scope of the invention. Such modifications include preparation of phosphorus-containing linkages as disclosed in U.S. Patents 5,536,821; 5,541,306; 5,550,111; 5,563,253; 5,571,799; 5,587,361, 5,625,050 and 5,958,773.<br>
The antisense compounds of the invention can include modified bases as disclosed in 5,958,773 and patents disclosed therein. The antisense oligonucleotides of the invention can also be modified by chemically linking the oligonucleotide to one or more moieties or conjugates to enhance the activity, cellular distribution, or cellular uptake of the antisense oligonucleotide. Such moieties or conjugates include lipids such as cholesterol, cholic acid, thioether, aliphatic chains, phospholipids, polyamines, polyethylene glycol (PEG), palmityl moieties, and others as disclosed in, for example, U.S. Patents 5,514,758, 5,565,552, 5,567,810, 5,574,142, 5,585,481, 5,587,371, 5,597,696 and 5,958,773.<br>
Chimeric antisense oligonucleotides are also within the scope of the invention, and can be prepared from the present inventive oligonucleotides using the methods described in, for<br><br>
example, U.S. Patents 5,013,830, 5,149,797, 5,403,711, 5,491,133, 5,565,350, 5,652,355, 5,700,922 and 5,958,773.<br>
Preferred antisense oligonucleotides can be selected by routine experimentation using, for example, assays described in the Examples. Although the inventors are not bound by a particular mechanism of action, it is believed that the antisense oligonucleotides achieve an inhibitory effect by binding to a complementary region of the target polynucleotide within the cell using Watson-Crick base pairing. Where the target polynucleotide is RNA, experimental evidence indicates that the RNA component of the hybrid is cleaved by RNase H (Giles et al., Nuc. Acids Res. 25:954-61, 1995; U.S. Patent No. 6,001,653). Generally, a hybrid containing 10 base pairs is of sufficient length to serve as a substrate for RNase H. However, to achieve specificity of binding, it is preferable to use an antisense molecule of at least 17 nucleotides, as a sequence of this length is likely to be unique among human genes.<br>
As disclosed in U.S. Patent No. 5,998,383, incorporated herein by reference, the oligonucleotide is selected such that the sequence exhibits suitable energy related characteristics important for oligonucleotide duplex formation with their complementary templates, and shows a low potential for self-dimerization or self-complementation (Anazodo et al., Biochem. Biophys. Res. Commun. 229:305-09, 1996). The computer program OLIGO (Primer Analysis Software, Version 3.4), is used to determined antisense sequence melting temperature, free energy properties, and to estimate potential self-dimer formation and self-complimentarity properties. The program allows the determination of a qualitative estimation of these two parameters (potential self-dimer formation and self-complimentary) and provides an indication of "no potential" or "some potential" or "essentially complete potential." Segments of OAS1 polynucleotides are generally selected that have estimates of no potential in these parameters. However, segments can be used that have "some potential" in one of the categories. A balance of the parameters is used in the selection.<br>
In the antisense art a certain degree of routine experimentation is required to select optimal antisense molecules for particular targets. To be effective, the antisense molecule preferably is targeted to an accessible, or exposed, portion of the target RNA molecule. Although in some cases information is available about the structure of target mRNA molecules, the current approach to inhibition using antisense is via experimentation. According to the invention, this experimentation can be performed routinely by transfecting cells with an antisense oligonucleotide using methods described in the Examples. mRNA levels in the cell can be measured routinely in treated and control cells by reverse transcription of the mRNA and assaying the cDNA levels. The biological effect can be determined routinely by measuring cell<br><br>
growth or viability as is known in the art.<br>
Measuring the specificity of antisense activity by assaying and analyzing cDNA levels is an art-recognized method of validating antisense results. It has been suggested that RNAfrom treated and control cells should be reverse-transcribed and the resulting cDNA populations analyzed. (Branch, A. D., T.I.B.S. 25:45-50, 1998.) According to the present invention, cultures of cells are transfected with two different antisense oligonucleotides designed to target OAS1. The levels of mRNA corresponding to OAS1 are measured in treated and control cells.<br>
Additional inhibitors include ribozymes, proteins or polypeptides, antibodies or fragments thereof as well as small molecules. Each of these OAS1 inhibitors share the common feature in that they reduce the expression and/or biological activity of OAS 1. In addition to the exemplary OAS 1 inhibitors disclosed herein, alternative inhibitors may be obtained through routine experimentation utilizing methodology either specifically disclosed herein or as otherwise readily available to and within the expertise of the skilled artisan. Ribozymes<br>
OAS 1 inhibitors may be ribozymes. A ribozyme is an RNA molecule that specifically cleaves RNA substrates, such as mRNA, resulting in specific inhibition or interference with cellular gene expression. As used herein, the term ribozymes includes RNA molecules that contain antisense sequences for specific recognition, and an RNA-cleaving enzymatic activity. The catalytic strand cleaves a specific site in a target RNA at greater than stoichiometric concentration.<br>
A wide variety of ribozymes may be utilized within the context of the present invention, including for example, the hammerhead ribozyme (for example, as described by Forster and Symons, Cell 48:211-20, 1987; Haseloff and Gerlach, Nature 328:596-600, 1988; Walbot and Bruening, Nature 334:196, 1988; Haseloff and Gerlach, Nature 334:585, 1988); the hairpin ribozyme (for example, as described by Haseloff et al., U.S. Patent No. 5,254,678, issued October 19, 1993 and Hempel et al., European Patent Publication No. 0 360 257, published March 26, 1990); and Tetrahymena ribosomal RNA-based ribozymes (see Cech et al., U.S. Patent No. 4,987,071). Ribozymes of the present invention typically consist of RNA, but may also be composed of DNA, nucleic acid analogs (e.g., phosphorothioates), or chimerics thereof (e.g.,DNA/RNA/RNA).<br>
Ribozymes can be targeted to any RNA transcript and can catalytically cleave such transcripts (see, e.g., U.S. Patent No. 5,272,262; U.S. Patent No. 5,144,019; and U.S. Patent Nos. 5,168,053, 5,180,818, 5,116,742 and 5,093,246 to Cech et al.). According to certain embodiments of the invention, any such OAS1 mRNA-specific ribozyme, or a nucleic acid<br><br>
encoding such a ribozyme, may be delivered to a host cell to effect inhibition of OAS1 gene expression. Ribozymes and the like may therefore be delivered to the host cells by DNA encoding the ribozyme linked to a eukaryotic promoter, such as a eukaryotic viral promoter, such that upon introduction into the nucleus, the ribozyme will be directly transcribed. RNAi<br>
The invention also provides for the introduction of RNA with partial or fully double-stranded character into the cell or into the extracellular environment. Inhibition is specific to the OAS1 expression in that a nucleotide sequence from a portion of the target OAS1 gene is chosen to produce inhibitory RNA. This process is (1) effective in producing inhibition of gene expression, and (2) specific to the targeted OAS1 gene. The procedure may provide partial or complete loss of function for the target OAS1 gene. A reduction or loss of gene expression in at least 99% of targeted cells has been shown using comparable techniques with other target genes. Lower doses of injected material and longer times after administration of dsRNAmay result in inhibition in a smaller fraction of cells. Quantitation of gene expression in a cell may show similar amounts of inhibition at the level of accumulation of target mRNA or translation of target protein. Methods of preparing and using RNAi are generally disclosed in U.S. Patent 6,506,559, incorporated herein by reference.<br>
The RNA may comprise one or more strands of polymerized ribonucleotide; it may include modifications to either the phosphate-sugar backbone or the nucleoside. The double-stranded structure may be formed by a single self-complementary RNA strand or two complementary RNA strands. RNA duplex formation may be initiated either inside or outside the cell. The RNA may be introduced in an amount which allows delivery of at least one copy per cell. Higher doses of double-stranded material may yield more effective inhibition. Inhibition is sequence-specific in that nucleotide sequences corresponding to the duplex region of the RNA are targeted for genetic inhibition. RNA containing a nucleotide sequence identical to a portion of the OAS1 target gene is preferred for inhibition. RNA sequences with insertions, deletions, and single point mutations relative to the target sequence have also been found to be effective for inhibition. Thus, sequence identity may optimized by alignment algorithms known in the art and calculating the percent difference between the nucleotide sequences. Alternatively, the duplex region of the RNA may be defined functionally as a nucleotide sequence that is capable of hybridizing with a portion of the target gene transcript.<br>
RNA may be synthesized either in vivo or in vitro. Endogenous RNA polymerase of the cell may mediate transcription in vivo, or cloned RNA polymerase can be used for transcription in vivo or in vitro. For transcription from a transgene in vivo or an expression construct, a<br><br>
regulatory region may be used to transcribe the RNA strand (or strands).<br>
For RNAi, the RNA may be directly introduced into the cell (i.e., intracellularly), or introduced extracellularly into a cavity, interstitial space, into the circulation of an organism, introduced orally, or may be introduced by bathing an organism in a solution containing RNA. Methods for oral introduction include direct mixing of RNA with food of the organism, as well as engineered approaches in which a species that is used as food is engineered to express an RNA, then fed to the organism to be affected. Physical methods of introducing nucleic acids include injection directly into the cell or extracellular injection into the organism of an RNA solution.<br>
The advantages of the method include the ease of introducing double-stranded RNA into cells, the low concentration of RNA which can be used, the stability of double-stranded RNA, and the effectiveness of the inhibition.<br>
Inhibition of gene expression refers to the absence (or observable decrease) in the level of protein and/or mRNA product from a OAS1 target gene. Specificity refers to the ability to inhibit the target gene without manifest effects on other genes of the cell. The consequences of inhibition can be confirmed by examination of the outward properties of the cell or organism or by biochemical techniques such as RNA solution hybridization, nuclease protection, Northern hybridization, reverse transcription, gene expression monitoring with a microarray, antibody binding, enzyme linked immunosorbent assay (ELISA), Western blotting, radioimmunoassay (R1A), other immunoassays, and fluorescence activated cell analysis (FACS). For RNA-mediated inhibition in a cell line or whole organism, gene expression is conveniently assayed by use of a reporter or drug resistance gene whose protein product is easily assayed. Such reporter genes include acetohydroxyacid synthase (AHAS), alkaline phosphatase (AP), beta galactosidase (LacZ), beta glucoronidase (GUS), chloramphenicol acetyltransferase (CAT), green fluorescent protein (GFP), horseradish peroxidase (HRP), luciferase (Luc), nopaline synthase (NOS), octopine synthase (OCS), and derivatives thereof. Multiple selectable markers are available that confer resistance to ampicillin, bleomycin, chloramphenicol, gentamycin, hygromycin, kanamycin, lincomycin, methotrexate, phosphinothricin, puromycin, and tetracyclin.<br>
Depending on the assay, quantitation of the amount of gene expression allows one to determine a degree of inhibition which is greater than 10%, 33%, 50%, 90%, 95% or 99% as compared to a cell not treated according to the present invention. Lower doses of injected material and longer times after administration of dsRNAmay result in inhibition in a smaller fraction of cells (e.g., at least 10%, 20%, 50%, 75%, 90%, or 95% of targeted cells).<br><br>
Quantitation of O AS I gene expression in a cell may show similar amounts of inhibition at the level of accumulation of OAS1 target mRNA or translation of OAS1 target protein. As an example, the efficiency of inhibition may be determined by assessing the amount of gene product in the cell: mRNA may be detected with a hybridization probe having a nucleotide sequence outside the region used for the inhibitory double-stranded RNA, or translated polypeptide may be detected with an antibody raised against the polypeptide sequence of that region.<br>
The RNA may comprise one or more strands of polymerized ribonucleotide. It may include modifications to either the phosphate-sugar backbone or the nucleoside. For example, the phosphodiester linkages of natural RNA may be modified to include at least one of a nitrogen or sulfur heteroatom. Modifications in RNA structure may be tailored to allow specific genetic inhibition while avoiding a general panic response in some organisms which is generated by dsRNA. Likewise, bases may be modified to block the activity of adenosine deaminase. RNA may be produced enzymatically or by partial/total organic synthesis, any modified ribonucleotide can be introduced by in vitro enzymatic or organic synthesis.<br>
The double-stranded structure may be formed by a single self-complementary RNA strand or two complementary RNA strands. RNA duplex formation may be initiated either inside or outside the cell. The RNA may be introduced in an amount which allows delivery of at least one copy per cell. Higher doses (eg., at least 5, 10, 100, 500 or 1000 copies per cell) of double-stranded material may yield more effective inhibition; lower doses may also be useful for specific applications. Inhibition is sequence-specific in that nucleotide sequences corresponding to the duplex region of the RNA are targeted for genetic inhibition.<br>
RNA containing a nucleotide sequences identical to a portion of the OAS1 target gene are preferred for inhibition. RNA sequences with insertions, deletions, and single point mutations relative to the target sequence may be effective for inhibition. Thus, sequence identity may optimized by sequence comparison and alignment algorithms known in the art (see Gribskov and Devereux, Sequence Analysis Primer, Stockton Press, 1991, and references cited therein) and calculating the percent difference between the nucleotide sequences by, for example, the Smith-Waterman algorithm as implemented in the BESTFIT software program using default parameters (e.g., University of Wisconsin Genetic Computing Group). Greater than 90% sequence identity, or even 100% sequence identity, between the inhibitory RNA and the portion of the OAS1 target gene is preferred. Alternatively, the duplex region of the RNA may be defined functionally as a nucleotide sequence that is capable of hybridizing with a portion of the OAS1 target gene transcript (e.g., 400 mM NaCl, 40 mM PIPES pH 6.4, 1 mM EDTA, 50C<br><br>
or 70C hybridization ibr 12-16 hours; followed by washing). The length of the identical nucleotide sequences may be at least 25, 50, 100, 200, 300 or 400 bases.<br>
100% sequence identity between the RNA and the OAS1 target gene is not required to practice the present invention. Thus the methods have the advantage of being able to tolerate sequence variations that might be expected due to genetic mutation, strain polymorphism, or evolutionary divergence.<br>
OAS1 RNA may be synthesized either in vivo or in vitro. Endogenous RNA polymerase of the cell may mediate transcription in vivo, or cloned RNA polymerase can be used for transcription in vivo or in vitro. For transcription from a transgene in vivo or an expression construct, a regulatory region (e.g., promoter, enhancer, silencer, splice donor and acceptor, polyadenylation) may be used to transcribe the RNA strand (or strands). Inhibition may be targeted by specific transcription in an organ, tissue, or cell type; stimulation of an environmental condition (e.g., infection, stress, temperature, chemical inducers); and/or engineering transcription at a developmental stage or age. The RNA strands may or may not be polyadenylated; the RNA strands may or may not be capable of being translated into a polypeptide by a cell's translational apparatus. RNA may be chemically or enzymatically synthesized by manual or automated reactions. The RNAmay be synthesized by a cellular RNA polymerase or a bacteriophage RNA polymerase (e.g., T3, T7, SP6). The use and production of an expression construct are known in (see WO 97/32016; U.S. Pat. Nos. 5,593,874, 5,698,425, 5,712,135, 5,789,214, and 5,804,693; and the references cited therein). If synthesized chemically or by in vitro enzymatic synthesis, the RNAmay be purified prior to introduction into the cell. For example, RNA can be purified from a mixture by extraction with a solvent or resin, precipitation, electrophoresis, chromatography, or a combination thereof. Alternatively, the RNA may be used with no or a minimum of purification to avoid losses due to sample processing. The RNAmay be dried for storage or dissolved in an aqueous solution. The solution may contain butlers or salts to promote annealing, and/or stabilization of the duplex strands.<br>
RNAmay be directly introduced into the cell (i.e., intracellularly); or introduced extracellularly into a cavity, interstitial space, into the circulation of an organism, introduced orally, or may be introduced by bathing an organism in a solution containing the RNA. Methods for oral introduction include direct mixing of the RNA with food of the organism, as well as engineered approaches in which a species that is used as food is engineered to express the RNA, then fed to the organism to be affected. For example, the RNA may be sprayed onto a plant or a plant may be genetically engineered to express the RNA in an amount sufficient to kill some or<br><br>
all of a pathogen known to infect the plant. Physical methods of introducing nucleic acids, for example, injection directly into the cell or extracellular injection into the organism, may also be used. Vascular or extravascular circulation, the blood or lymph system, and the cerebrospinal fluid are sites where the RNA may be introduced. A transgenic organism that expresses RNA from a recombinant construct may be produced by introducing the construct into a zygote, an embryonic stem cell, or another multipotent cell derived from the appropriate organism.<br>
Physical methods of introducing nucleic acids include injection of a solution containing the RNA, bombardment by particles covered by the RNA, soaking the cell or organism in a solution of the RNA, or electroporation of cell membranes in the presence of the RNA. A viral construct packaged into a viral particle would accomplish both efficient introduction of an expression construct into the cell and transcription of RNA encoded by the expression construct. Other methods known in the art for introducing nucleic acids to cells may be used, such as lipid-mediated carrier transport, chemical-mediated transport, such as calcium phosphate, and the like. Thus the RNA may be introduced along with components that perform one or more of the following activities: enhance RNA uptake by the cell, promote annealing of the duplex strands, stabilize the annealed strands, or other-wise increase inhibition of the target gene.<br>
The present invention may be used alone or as a component of a kit having at least one of the reagents necessary to carry out the in vitro or in vivo introduction of RNA to test samples or subjects. Preferred components are the dsRNA and a vehicle that promotes introduction of the dsRNA. Such a kit may also include instructions to allow a user of the kit to practice the invention.<br>
Suitable injection mixes are constructed so animals receive an average of 0.5xl06 to l.OxlO6 molecules of RNA. For comparisons of sense, antisense, and dsRNA activities, injections are compared with equal masses of RNA (i.e., dsRNA at half the molar concentration of the single strands). Numbers of molecules injected per adult are given as rough approximations based on concentration of RNA in the injected material (estimated from ethidium bromide staining) and injection volume (estimated from visible displacement at the site of injection). A variability of several-fold in injection volume between individual animals is possible. Proteins and PolvrjLerjtidgs<br>
In addition to the antisense molecules and ribozynies disclosed herein, OAS1 inhibitors of the present invention also include proteins or polypeptides that are effective in either reducing OAS1 gene expression or in decreasing one or more of OAS1 's biological activities, including but not limited to enzymatic activity; interaction with single stranded RNA, configurations; and<br><br>
binding to other proteins such as Hepatitis C virus NS5 A or a fragment thereof. A variety of methods are readily available in the art by which the skilled artisan may, through routine experimentation, rapidly identify such OAS1 inhibitors. The present invention is not limited by the following exemplary methodologies.<br>
Literature is available to the skilled artisan that describes methods for detecting and analyzing protein-protein interactions. Reviewed in Phizicky et al., Microbiological Reviews 59:94-123,1995, incorporated herein by reference. Such methods include, but are not limited to physical methods such as, e.g., protein affinity chromatography, affinity blotting, irnmunoprecipitation and cross-linking as well as library-based methods such as, e.g., protein probing, phage display and two-hybrid screening. Other methods that may be employed to identify protein-protein interactions include genetic methods such as use of extragenic suppressors, synthetic lethal effects and unlinked noncomplementation. Exemplary methods are described in further detail below.<br>
Inventive OAS1 inhibitors may be identified through biological screening assays that rely on the direct interaction between the OAS1 protein and a panel or library of potential inhibitor proteins. Biological screening methodologies, including the various "n-hybrid technologies," are described in, for example, Vidal et al., Nucl. Acids Res. 27(4):919-29,1999; Frederickson, R.M., Curr. Opin. Biotechnol. 9(l):90-96, 1998; Brachrnann et al., Curr. Opin. Biotechnol. S(5):561-68, 1997; and White, M.A., Proc. Natl. Acad. Sci. U.S.A. P3:10001-03, 1996, each of which is incorporated herein by reference.<br>
The two-hybrid screening methodology may be employed to search new or existing target cDNA libraries for OAS1 binding proteins that have inhibitory properties. The two-hybrid system is a genetic method that detects protein-protein interactions by virtue of increases in transcription of reporter genes. The system relies on the fact that site-specific transcriptional activators have a DNA-binding domain and a transcriptional activation domain. The DNA-binding domain targets the activation domain to the specific genes to be expressed. Because of the modular nature of transcriptional activators, the DNA-binding domain may be severed covalently from the transcriptional activation domain without loss of activity of either domain. Furthermore, these two domains may be brought into juxtaposition by protein-protein contacts between two proteins unrelated to the transcriptional machinery. Thus, two hybrids are constructed to create a functional system. The first hybrid, i.e., the bait, consists of a transcriptional activator DNA-binding domain fused to a protein of interest. The second hybrid, the target, is created by the fusion of a transcriptional activation domain with a library of proteins or polypeptides. Interaction between the bait protein and a member of the target library<br><br>
results in the juxtaposition of the DNA-binding domain and the transcriptional activation domain and the consequent up-regulation of reporter gene expression.<br>
A variety of two-hybrid based systems are available to the skilled artisan that most commonly employ either the yeast Gal4 or E. coli Lex A DNA-binding domain (BD) and the yeast Gal4 or herpes simplex virus VP16 transcriptional activation domain. Chien et al., Proc. Natl. Acad. Sci. U.S.A. 55:9578-82,1991; Dalton et al., Cell (55:597-612, 1992; Durfee et al., Genes Dev. 7:555-69, 1993; Vojtek et al, Cell 74:205-14, 1993; and Zervos et al, Cell 72:223-32, 1993. Commonly used reporter genes include the E. coli lacZ gene as well as selectable yeast genes such as HISS and LEW. Fields et al., Nature (London) 340:245-46, 1989; Durfee, T.K., supra; and Zervos, A.S., supra. A wide variety of activation domain libraries is readily available in the art such that the screening for interacting proteins may be performed through routine experimentation.<br>
Suitable bait proteins for the identification of OASl interacting proteins may be designed based on the OAS1 DNA sequence presented herein as SEQ ID NO: 19. Such bait proteins include either the full-length OASl protein or fragments thereof.<br>
Plasmid vectors, such as, e.g., pBTM116 and pAS2-l, for preparing OASl bait constructs and target libraries are readily available to the artisan and may be obtained from such commercial sources as, e.g., Clontech (Palo Alto, CA), Invitrogen (Carlsbad, CA) and Stratagene (La Jolla, CA). These plasmid vectors permit the in-frame fusion of cDNAs with the DNA-binding domains as Lex A or Gal4BD, respectively.<br>
OAS 1 inhibitors of the present invention may alternatively be identified through one of the physical or biochemical methods available in the art for detecting protein-protein interactions.<br>
Through the protein affinity chromatography methodology, lead compounds to be tested as potential OASl inhibitors may be identified by virtue of their specific retention to OASl when either covalently or non-covalently coupled to a solid matrix such as, e.g., Sepharose beads. The preparation of protein affinity columns is described in, for example, Beeckmans et al,Eur. J. Biochem. 777:527-35,1981, and Formosa et al., Methods Enzymol. 208:24-45, 1991. Cell lysates containing the full complement of cellular proteins may be passed through the OAS 1 affinity column. Proteins having a high affinity for OAS 1 will be specifically retained under low-salt conditions while the majority of cellular proteins will pass through the column. Such high affinity proteins may be eluted from the immobilized OASl under conditions of high-salt, with chaotropic solvents or with sodium dodecyl sulfate (SDS). In some embodiments, it may be preferred to radiolabel the cells prior to preparing the lysate as an aid in identifying the<br><br>
OAS1 specific binding proteins. Methods for radiolabeling mammalian cells are well known in the art and are provided, e.g., in Sopta et al., /. Biol. Chem. 260:10353-60, 1985.<br>
Suitable OAS1 proteins for affinity chromatography may be fused to a protein or polypeptide to permit rapid purification on an appropriate affinity resin. For example, the OAS1 cDNA may be fused to the coding region for glutathione S-transferase (GST) which facilitates the adsoiption of fusion proteins to glutathione-agarose columns. Smith et al., Gene 67:31-40, 1988. Alternatively, fusion proteins may include protein A, which can be purified on columns bearing immunoglobulin G; oligohistidine-containing peptides, which can be purified on columns bearing Ni2+; the maltose-binding protein, which can be purified on resins containing amylose; and dihydrofolate reductase, which can be purified on methotrexate columns. One exemplary tag suitable for the preparation of OAS1 fusion proteins that is presented herein is the epitope for the influenza virus hemagglutinin (HA) against which monoclonal antibodies are readily available and from which antibodies an affinity column may be prepared.<br>
Proteins that are specifically retained on a OAS1 affinity column may be identified after subjecting to SDS polyacrylamide gel electrophoresis (SDS-PAGE). Thus, where cells are radiolabeled prior to the preparation of cell lysates and passage through the OAS1 affinity column, proteins having high affinity for OAS1 may be detected by autoradiography. The identity of OAS 1 specific binding proteins may be determined by protein sequencing techniques that are readily available to the skilled artisan, such as Mathews, C.K. et al., Biochemistry, The Benjamin/Curnmings Publishing Company,Inc., 1990, pp. 166-70. Small Molecules<br>
The present invention also provides small molecule OAS1 inhibitors that may be readily identified through routine application of high-throughput screening (HTS) methodologies. Reviewed by Persidis, A., Nature Biotechnology /d:488-89, 1998. HTS methods generally refer to those technologies that permit the rapid assaying of lead compounds, such as small molecules, for therapeutic potential. HTS methodology employs robotic handling of test materials, detection of positive signals and interpretation of data. Such methodologies include, e.g., robotic screening technology using soluble molecules as well as cell-based systems such as the two-hybrid system described in detail above.<br>
A variety of cell line-based HTS methods are available that benefit from their ease of manipulation and clinical relevance of interactions that occur within a cellular context as opposed to in solution. Lead compounds may be identified via incorporation of radioactivity or through optical assays that rely on absorbance, fluorescence or luminescence as read-outs. See, e.g., Gonzalez et al., Cwr. Opin. Biotechnol 9(6):624-31, 1998, incorporated herein by<br><br>
reierence.<br>
HTS methodology may be employed, e.g., to screen for lead compounds that block one of OASl's biological activities. By this method, OASl protein may be immunoprecipitated from cells expressing the protein and applied to wells on an assay plate suitable for robotic screening. Individual test compounds may then be contacted with the immunoprecipitated protein and the effect of each test compound on OASl. Methods for Assessing the Efficacy of OASl inhibitors<br>
Lead molecules or compounds, whether antisense molecules or ribozymes, proteins and/or peptides, antibodies and/or antibody fragments or small molecules, that are identified either by one of the methods described herein or via techniques that are otherwise available in the art, may be further characterized in a variety of in vitro, ex vivo and in vivo animal model assay systems for their ability to inhibit OASl gene expression or biological activity. As discussed in further detail in the Examples provided below, OAS1 inhibitors of the present invention are effective in reducing OASl expression levels. Thus, the present invention further discloses methods that permit the skilled artisan to assess the effect of candidate inhibitors.<br>
Candidate OASl inhibitors may be tested by administration to cells that either express endogenous OASl or that are made to express OASl by transfection of a mammalian cell with a recombinant OAS 1 plasmid construct.<br>
Effective OASl inhibitory molecules will be effective in reducing the enzymatic activity of OASl or ability of OASl to respond to IFN induction. Methods of measuring OASl enzymatic activity and IFN induction are known in the art, for example, as described in Eskildsen et al., Nuc. Acids Res. 31:3166-3173, 2003; and Justesenet al., Nuc. Acids Res. 8:3073-3085, 1980, incorporated herein by reference. The effectiveness of a given candidate antisense molecule may be assessed by comparison with a control "antisense" molecule known to have no substantial effect on OASl expression when administered to a mammalian cell.<br>
OASl inhibitors effective in reducing OASl gene expression by one or more of the methods discussed above may be further characterized in vitro for efficacy in one of the readily available established cell culture or primary cell culture model systems as described herein, in reference to use of Vero cells challenged by infection with a flavivirus, such as dengue virus. Pharmaceutical Compositions<br>
The antisense oligonucleotides and ribozymes of the present invention can be synthesized by any method known in the art for ribonucleic or deoxyribonucleic nucleotides. For example, the oligonucleotides can be prepared using solid-phase synthesis such as in an Applied Biosystems 380B DNA synthesizer. Final purity of the oligonucleotides is determined<br><br>
as is known in the art.<br>
The antisense oligonucleotides identified using the methods of the invention modulate tumor cell proliferation. Therefore, pharmaceutical compositions and methods are provided for interfering with virus infection, preferably flavivirus, most preferably HCV infection, comprising contacting tissues or cells with one or more of antisense oligonucleotides identified using the methods of the invention.<br>
The invention provides pharmaceutical compositions of antisense oligonucleotides and ribozymes complementary to the OAS1 mRNAgene sequence as active ingredients for therapeutic application. These compositions can also be used in the method of the present invention. When required, the compounds are nuclease resistant. In general the pharmaceutical composition for inhibiting virus infection in a mammal includes an effective amount of at least one antisense oligonucleotide as described above needed for the practice of the invention, or a fragment thereof shown to have the same effect, and a pharmaceutically physiologically acceptable carrier or diluent.<br>
The compositions can be administered orally, subcutaneously, or parenterally including intravenous, intraarterial, intramuscular, intraperitoneally, and intranasal administration, as well as intrathecal and infusion techniques as required. The pharmaceutically acceptable carriers, diluents, adjuvants and vehicles as well as implant carriers generally refer to inert, non-toxic solid or liquid fillers, diluents or encapsulating material not reacting with the active ingredients of the invention. Cationic lipids may also be included in the composition to facilitate oligonucleotide uptake. Implants of the compounds are also useful. In general, the pharmaceutical compositions are sterile.<br>
By bioactive (expressible) is meant that the oligonucleotide is biologically active in the cell when delivered directly to the cell and/or is expressed by an appropriate promotor and active when delivered to the cell in a vector as described below. Nuclease resistance is provided by any method known in the art that does not substantially interfere with biological activity as described herein.<br>
"Contacting the cell" refers to methods of exposing or delivering to a cell antisense oligonucleotides whether directly or by viral or non-viral vectors and where the antisense oligonucleotide is bioactive upon delivery.<br>
The nucleotide sequences of the present invention can be delivered either directly or with viral or non-viral vectors. When delivered directly the sequences are generally rendered nuclease resistant. Alternatively, the sequences can be incorporated into expression cassettes or constructs such that the sequence is expressed in the cell. Generally, the construct contains the<br><br>
proper regulatory sequence or promotor to allow the sequence to be expressed in the targeted cell.<br>
Once the oligonucleotide sequences are ready for delivery they can be introduced into cells as is known in the art. Transfection, electroporation, fusion, liposomes, colloidal polymeric particles, and viral vectors as well as other means known in the art may be used to deliver the oligonucleotide sequences to the cell. The method selected will depend at least on the cells to be treated and the location of the cells and will be known to those skilled in the art. Localization can be achieved by liposomes, having specific markers on the surface for directing the liposome, by having injection directly into the tissue containing the target cells, by having depot associated in spatial proximity with the target cells, specific receptor mediated uptake, viral vectors, or the like.<br>
The present invention provides vectors comprising an expression control sequence operatively linked to the oligonucleotide sequences of the invention. The present invention further provides host cells, selected from suitable eukaryotic and prokaryotic cells, which are transformed with these vectors as necessary.<br>
Vectors are known or can be constructed by those skilled in the art and should contain all expression elements necessary to achieve the desired transcription of the sequences. Other beneficial characteristics can also be contained within the vectors such as mechanisms for recovery of the oligonucleotides in a different form. Phagemids are a specific example of such beneficial vectors because they can be used either as plasmids or as bacteriophage vectors. Examples of other vectors include viruses such as bacteriophages, baculo viruses and retroviruses, DNA viruses, liposomes and other recombination vectors. The vectors can also contain elements for use in either procaryotic or eucaryotic host systems. One of ordinary skill in the art will know which host systems are compatible with a particular vector.<br>
The vectors can be introduced into cells or tissues by any one of a variety of known methods within the art. Such methods can be found generally described in Sambrook et al., Molecular Cloning: A Laboratory Manual, Cold Springs Harbor Laboratory, New York, 1989, 1992; in Ausubel et al., Current Protocols in Molecular Biology, John Wiley and Sons, Baltimore, MD, 1989; Chang et al., Somatic Gene Therapy, CRC Press, Ann Arbor, Mich., 1995; Vega et al., Gene Targeting, CRC Press, Ann Arbor, Mich., 1995; Vectors: A Survey of Molecular Cloning Vectors and Their Uses, Butterworths, Boston, Mass., 1988; and Gilboa et al., BioTechniques 4:504-12, 1986, and include, for example, stable or transient transfection, lipofection, electroporation and infection with recombinant viral vectors.<br>
Recombinant methods known in the art can also be used to achieve the antisense<br><br>
inhibition of a target nucleic acid. For example, vectors containing antisense nucleic acids can be employed to express an antisense message to reduce the expression of the target nucleic acid and therefore its activity.<br>
The present invention also provides a method of evaluating if a compound inhibits transcription or translation of an OAS1 gene and thereby modulates (i.e., reduces) the ability of the cell to activate RNaseL, comprising transfecting a cell with an expression vector comprising a nucleic acid sequence encoding OAS1, the necessary elements for the transcription or translation of the nucleic acid; administering a test compound; and comparing the level of expression of the OAS1 with the level obtained with a control in the absence of the test compound.<br>
Preferred Embodiments<br>
Utilizing methods described above and others known in the art, the present invention contemplates a screening method comprising treating, under amplification conditions, a sample of genomic DNA, isolated from a human, with a PCR primer pair for amplifying a region of human genomic DNA containing any of nucleotide (nt) positions 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 of oligoadenylate synthetase (OAS1, SEQ ID NO:19). Amplification conditions include, in an amount effective for DNA synthesis, the presence of PCR buffer and a thermocycling temperature. The PCR product thus produced is assayed for the presence of a point mutation at the relevant nucleotide position. In one embodiment, the PCR product contains a continuous nucleotide sequence comprising about 358 base pairs (bp) written from 5' to 3' direction and including position 2135728 (mutation 1), 2135749 (mutation 2), 2135978 (mutation 3), 2144072 (mutation 4), 2144088 (mutation 5), 2144116 (mutation 6), or 2144321 (mutation 7) and the approximately 175 bases flanking the position at each side. In another embodiment, the amplicons as described above in Tables 1 and 2 are exemplary of the PCR products and corresponding primers.<br>
In one preferred embodiment, the PCR product is assayed for the corresponding mutation by treating the amplification product, under hybridization conditions, with an oligonucleotide probe specific for the corresponding mutation, and detecting the formation of any hybridization product. Preferred oligonucleotide probes comprise a nucleotide sequence indicated in Table 3 below. Oligonucleotide hybridization to target nucleic acid is described in U.S. Pat. No. 4,530,901.<br><br>
Table Removed<br><br>
Wherein R=A/G, S=C/G, and Y=C/T.<br>
The PCR admixture thus formed is subjected to a plurality of PCR thermocycles to produce<br>
OAS1 and OAS1R gene amplification products. The amplification products are then treated,<br>
under hybridization conditions, with an oligonucleotide probe specific for each mutation. Any<br>
hybridization products are then detected.<br>
The following examples are intended to illustrate but are not to be construed as limiting of the<br>
specification and claims in any way.<br>
EXAMPLES<br>
EXAMPLE 1<br>
PREPARATION AND PRELIMINARY SCREENING OF GENOMIC DNA<br>
This example relates to screening of DNA from two specific populations of patients, but is equally applicable to other patient groups in which repeated exposure to HCV is documented, wherein the exposure does not result in infection. The example also relates to screening patients who have been exposed to other flaviviruses as discussed above, wherein the exposure did not result in infection.<br>
Here, two populations are studied: (1) a hemophiliac population, chosen with the criteria of moderate to severe hemophilia, and receipt of concentrated clotting factor before January, 1987; and (2) an intravenous drug user population, with a history of injection for over 10 years, and evidence of other risk behaviors such as sharing needles. The study involves exposed but HCV negative patients, and exposed and HCV positive patients.<br>
High molecular weight DNA is extracted from the white blood cells from IV drug users, hemophiliac patients, and other populations at risk of hepatitis C infection, or infection by other flavivimses. For the initial screening of genomic DNA, blood is collected after informed consent from the patients of the groups described above and anticoagulated with a mixture of<br><br>
0.14M citric acid, 0.2M trisodium citrate, and 0.22M dextrose. The anticoagulated blood is centrifuged at 800 x g for 15 minutes at room temperature and the platelet-rich plasma supernatant is discarded. The pelleted erythrocytes, mononuclear and polynuclear cells are resuspended and diluted with a volume equal to the starting blood volume with chilled 0.14M phosphate buffered saline (PBS), pH 7.4. The peripheral blood white blood cells are recovered from the diluted cell suspension by centrifugation on low endotoxin Ficoll-Hypaque (Sigma Chem. Corp. St. Louis, Mo.) at 400 x g for 10 minutes at 18 C. (18 C.). The pelleted white blood cells are then resuspended and used for the source of high molecular weight DNA. The high molecular weight DNA is purified from the isolated white blood cells using methods well known to one skilled in the art and described by Maniatis, et al., Molecular Cloning: A Laboratory Manual, 2nd ed. Cold Spring Harbor Laboratory, Sections 9.16-9.23, (1989) and U.S. Pat. No. 4,683,195.<br>
Each sample of DNA is then examined for a point mutation of any one of the nucleotides at position 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 with reference to the nucleotides positions of Genbank Accession No. NT_009775.13, corresponding to the oligoadenylate synthetase 1 gene (OAS1).<br>
EXAMPLE 2<br>
MUTATIONS IN OAS1 GENE ASSOCIATED WITH RESISTANCE TO HCV INFECTION Using methods described in Example 1, a population of unrelated hemophiliac patients and intravenous drug users was studied, and the presence or absence of a mutation in OAS 1 as disclosed in SEQ ID NO:1-SEQ ID NO:7 and SEQ ID NO:57-64 was determined. In a study of 20 cases and 42 controls in a Caucasian population, these mutations were found in the context of resistance to hepatitis C infection. There was a statistically significant correlation between resistance to HCV infection and presence of a point mutation in OAS1.<br>
EXAMPLE 3<br>
PREPARATION AND SEQUENCING OF cDNA<br>
Total cellular RNA is purified from cultured lymphoblasts or fibroblasts from the patients having the hepatitis C resistance phenotype. The purification procedure is performed as described by Chomczynski, et al., Anal. Biochem., 162:156-159 (1987). Briefly, the cells are prepared as described in Example 1. The cells are then homogenized in 10 milliliters (ml) of a denaturing solution containing 4.0M guanidine thiocyanate, 0.1M Tris-HCl at pH 7.5, and 0.1M beta-mercaptoethanol to form a cell lysate. Sodium lauryl sarcosinate is then admixed to a final concentration of 0.5% to the cell lysate after which the admixture was centrifuged at 5000 X g<br><br>
for 10 minutes at room temperature. The resultant supernatant containing the total RNA is layered onto a cushion of 5.7M cesium chloride and 0.01M EDTA at pH 7.5 and is pelleted by centrifugation. The resultant RNA pellet is dissolved in a solution of 10 mM Tris-HCl at pH 7.6 and 1 mM EDTA (TE) containing 0.1% sodium docecyl sulfate (SDS). After phenolchloroform extraction and ethanol precipitation, the purified total cellular RNA concentration is estimated by measuring the optical density at 260 nm.<br>
Total RNA prepared above is used as a template for cDNA synthesis using reverse transcriptase for first strand synthesis and PCR with oligonucleotide primers designed so as to amplify the cDNA in two overlapping fragments designated the 5' and the 3' fragment. The oligonucleotides used in practicing this invention are synthesized on an Applied Biosystems 381A DNA Synthesizer following the manufacturer's instructions. PCR is conducted using methods known in the art. PCR amplification methods are described in detail in U.S. Pat. Nos. 4,683,192, 4,683,202, 4,800,159, and 4,965,188, and at least in several texts including PCR Technology: Principles and Applications for DNA Amplification, H. Erlich, ed., Stockton Press, New York (1989); and PCR Protocols: A Guide to Methods and Applications, Innis, et al., eds., Academic Press, San Diego, Calif. (1990) and primers as described in Table 1 herein. The sequences determined directly from the PCR-amplified DNAs from the patients with and without HCV infection, are analyzed. The presence of a mutation upstream from the coding region of the OAS gene can be detected in patients who are seronegative for HCV despite repeated exposures to the virus..<br>
EXAMPLE4<br>
PREPARATION OF PCR AMPLIFIED GENOMIC DNA CONTAINING A POINT MUTATION AND DETECTION BY ALLELE SPECIFIC OLIGONUCLEOTIDE HYBRIDIZATION<br>
The point mutation in an oligoadenylate synthetase (OAS1) gene at one of nucleotide positions 2135728, 2135749, 2135978, 2144072, 2144088, 2144116, 2144321, 2131025, 2133961, 2139587, 2144294, 2144985, 2156523, 2156595, or 2156638 can be determined by an approach in which PCR amplified genomic DNA containing the mutation is detected by hybridization with oligonucleotide probes that hybridized to that region. To amplify the region having the point mutation for hybridization with oligonucleotide specific probes, PCR amplifications are performed as essentially described in Example 3 with, for example, 180 ng of each of the primers shown in Table 1.<br>
Following the PCR amplification, 2 ul of the amplified oligoadenylate synthetase DNA products are spotted onto separate sheets of nitrocellulose. After the spotted amplified DNA has dried, the nitrocellulose is treated with 0.5N NaOH for 2 minutes, 1M Tris-HCl at pH 7.5 for 2<br><br>
minutes, followed by 0.5M Tris-HCl at pH 7.5 containing 1.5M NaCl for 2 minutes to denature and then neutralize the DNA. The resultant filters are baked under a vacuum for 1 hour at 80C., are prehybridized for at least 20 minutes at 42C. with a prehybridization solution consisting of 6X SSC (1X=0.15M NaCl, 0.15M sodium citrate), 5X Denhardt's solution (5X=0.1% polyvinylpyrrolidone, 0.1% ficoll, and 0.1% bovine serum albumin), 5 mM sodium phosphate buffer at pH 7.0, 0.5 mg/ral salmon testis DNA and 1% SDS. After the prehybridization step, the nitrocellulose filters are separately exposed to 32P-labeled oligonucleotide probes diluted in prehybridization buffer. Labeling of the probes with 32P is performed by admixing 2.5 ul of 10X concentrate of kinase buffer (10X=0.5M Tris[hydroxymethyl] aminomethane hydrochloride (Tris-HCl) atpH 7.6, 0.1M MgCl2, 50 mM dithiothreitol (DTT), 1 mM spermidine-HCl, and 1 mM ethylenediaminetetraacetic acid (EDTA)), 1.1 ul of 60 ug/ul of a selected oligonucleotide, 18.4 ul water, 2 ul of 6000 Ci/mM of gamma 32P ATP at a concentration of 150 mCi/ul, and 1 ui of 10 U/ul polynucleotide kinase. The labeling admixture is maintained for 20 minutes at 37C. followed by 2 minutes at 68C. The maintained admixture is then applied to a Sephadex G50 (Pharmacia, Inc., Piscataway, NJ) spin column to remove unincorporated 32P-labeled ATP.<br>
The oligonucleotide probes used to hybridize to the region containing the mutation are shown in Table 3 above. The underlined nucleotide corresponds to the mutation nucleotide. In probes for detecting wild type (normal), the underlined nucleotide is replaced with the wild-type nucleotide.<br>
Ten X 106 cpm of the normal and mutant labeled probes are separately admixed with each filter. The nitrocellulose filters are then maintained overnight at 42C. to allow for the formation of hybridization products. The nitrocellulose filters exposed to the normal probe are washed with 6X SSC containing 0.1% SDS at 46C. whereas the filters exposed to the mutant probe are washed with the same solution at a more stringent temperature of 52C. The nitrocellulose filters are then dried and subjected to radioautography.<br>
Only those products having the point mutation hybridize with the mutant probe. Positive and negative controls are included in each assay to determine whether the PCR amplification is successful. Thus, the patients' genomic DNA prepared in Example 1 are determined by this approach to have the unique point mutation of a non-wild type nucleotide substituted for a wild type nucleotide at the indicated position.<br>
EXAMPLES<br>
ANIJSENSE INHIBITION OF TARGET RNA \.        Preparation oXQliEonucleotides for Transfection<br><br>
A carrier molecule, comprising either a lipitoid or cholesteroid, is prepared for transfection by diluting to 0.5 mM in water, followed by sonication to produce a uniform solution, and filtration through a 0.45 um PVDF membrane. The lipitoid or cholesteroid is then diluted into an appropriate volume of OptiMEM (Gibco/BRL) such that the final concentration would be approximately 1.5-2 nmol lipitoid per (j.g oligonucleotide.<br>
Antisense and control oligonucleotides are prepared by first diluting to a working concentration of 100 uM in sterile Millipore water, then diluting to 2 uM (approximately 20 mg/mL) in OptiMEM. The diluted oligonucleotides are then immediately added to the diluted lipitoid and mixed by pipetting up and down.<br>
B.	Transfection<br>
Human PH5CH8 hepatocytes, which are susceptible to HCV infection and supportive of HCV replication, are used (Dansako et al., Virus Res. 97:17-30, 2003; Bceda et al., Virus Res. 56:157-167, 1998; Noguchi and Hirohashi, In Vitro CellDev. BiolAnim. 32:135-137, 1996.) The cells are transfected by adding the oligonucleotide/lipitoid mixture, immediately after mixing, to a final concentration of 300 nM oligonucleotide. The cells are then incubated with the transfection mixture overnight at 37C, 5% CO2 and the transfection mixture remains on the cells for 3-4 days.<br>
C.	Total RNA Extraction and Reverse Transcription<br>
Total RNA is extracted from the transfected cells using the RNeasy kit (Qiagen Corporation, Chatsworth, CA), following protocols provided by the manufacturer. Following extraction, the RNA is reverse-transcribed for use as a PCR template. Generally 0.2-1 |.ig of total extracted RNA is placed into a sterile microfuge tube, and water is added to bring the total volume to 3 uL. 7 (jL of a buffer/enzyme mixture is added to each tube. The buffer/enzyme mixture is prepared by mixing, in the order listed:<br>
4 joL 25 mM MgCl2<br>
2 (.iL 10X reaction buffer<br>
8f.iL2.5mMdNTPs<br>
1 |j.L MuLV reverse transcriptase (50 u) (Applied Biosystems)<br>
1 (J.L RNase inhibitor (20 u)<br>
1 |oL oligo dT (50 pmol)<br>
The contents of the microfuge tube are mixed by pipetting up and down, and the reaction is incubated for 1 hour at 42C. I &gt;         PCR AmplificatLQJlJIid Quantification of Target Sequences<br><br>
Following reverse transcription, target genes are amplified using the Roche Light CyclerTM real-time PCR machine. 20 \.\.L aliquots of PCR amplification mixture are prepared by mixing the following components in the order listed: 2 j.iL 10X PCR buffer II (containing 10 mM Tris pH 8.3 and 50 mM KC1, Perkin-Elmer, Norwalk, CT) 3 mM MgCl2, 140 j.iM each dNTP, 0.175 praol of each OAS1 oligo, 1:50,000 dilution of SYBR Green, 0.25 mg/mL BSA, 1 unit Tag polymerase, and HbO to 20 |.iL. SYBR Green (Molecular Probes, Eugene, OR) is a dye that fluoresces when bound to double-stranded DNA, allowing the amount of PCR product produced in each reaction to be measured directly. 2 u,L of completed reverse transcription reaction is added to each 20 (.iL aliquot of PCR amplification mixture, and amplification is carried out according to standard protocols.<br>
EXAMPLE 6<br>
TREATMENT OF CELLS WITH OAS1 RNAi<br>
Using the methods of Example 5, for antisense treatment, cells are treated with an oligonucleotide based on the OAS1 sequence (SEQ ID NO: 19). Two complementary ribonucleotide monomers with deoxy-TT extensions at the 3' end are synthesized and annealed. Cells of the PH3CH8 hepatocyte cell line are treated with 50-200 nM RNAi with 1:3 L2 lipitoid. Cells are harvested on day 1, 2, 3 and 4, and analyzed for OAS1 protein by Western analysis, as described by Dansako et al. Virus Res. 97:17-30, 2003.<br>
EXAMPLE 7<br>
OAS1 Interaction with Hepatitis C Virus NS5A Protein<br>
The ability of an OAS1 protein or polypeptide of the invention to interact with hepatitis C virus NS5A protein is assayed using a method described in Taguchi, T. et al., J. Gen. Virol. 85:959-969, 2004. Polynucleotides encoding OAS1 proteins and polypeptides are prepared as described above, and plasmids are constructed using routine methods, such as described in Taguchi, T. et al One plasmid contains a polynucleotide encoding an OAS1 protein or polypeptide, and a second plasmid contains polynucleotide encoding NS5A. The plasmids also encode appropriate tags for the respective proteins, such as FLAG-tag, HA, or GST. Suitable cells, such as HeLa cells, are transiently transfected with a plasmid encoding a tag and NS5A protein, and a plasmid encoding a different tag and an OAS1 protein or polypeptide. After incubation and preparation of supernatant as described (Taguchi, T. et al.), a variety of analytic techniques can be used to detect arid quantify the binding of NS5A with the OAS1 protein or polypeptide. Such techniques are known in the art and include co-precipitation analysis, immunofluorescence analysis, and immunoblot analysis. OAS1 proteins and polynucleotides that do not exhibit<br><br>
binding to NS5A are appropriate for further analysis as inhibitors of hepatitis C infection.<br>
EXAMPLE 8<br>
CHEMICALLY AND STERICALLY CONSERVED REGIONS OF OAS1<br>
As one skilled in the art will recognize, when modifying the structure of OAS1 to improve enzymatic activity or therapeutic potential, certain residues or regions of residues must be chemically and structurally conserved. By example, several conserved domains are described below. As one skilled in the art will recognize, chemically conservative changes to some ammo acids that preserve the structure and function of the protein may be tolerated. For example, Asp75 and Asp77 both coordinate catalytic divalent metal ions that are essential to OAS1 function. While modifications to these bases may be tolerated (e.g. to asparagine or glutamic acid), the essentially polar and acid nature of these residues must be preserved.<br>
As examples, with regard to SEQ ID NO: 26-29, SEQ ID NO:33, SEQ ID NO:34, and SEQ ID<br>
NO:50, the following polypeptide fragments represent conserved domains:<br>
Amino Acids 40-47: FLKERCFR (SEQ ID NO: 75)<br>
Amino acids 55-82: VSKWKGGSSGKGTTLRGRSDADLVVFL (SEQ ID NO: 76)<br>
Amino Acids 94-112: RRGEFIQEIRRQLEACQRE (SEQ ID NO: 77)<br>
Amino Acids 128-138: NPRALSFVLSS (SEQ ID NO: 78)<br>
Amino Acids 145-158: VEFDVLPAFDALGQ (SEQ ID NO: 79)<br>
Amino Acids 182-198: KEGEFSTCFTELQRDFL (SEQ ID NO: 80)<br>
Amino Acids 201-217: RPTKLKSLIRLVKHWYQ (SEQ ID NO: 81)<br>
Amino Acids 225-241: KLPPQYALELLTVYAWE (SEQ ID NO: 82)<br>
Amino Acids 296-307: PVILDPADPTGN (SEQ ID NO: 83)<br>
Amino Acids 337-343: GSPVSSW (SEQ ID NO:84)<br>
EXAMPLE 9<br>
AMINO ACIDS CHANGES THAT IMPROVE ENZYME ACTIVE SITE<br>
Changes in OAS amino acids sequences can be envisioned that improve the enzymatic activity of the protein. In one preferred embodiment, amino acids within the active site of the enzyme can be modified to improve ATP or metal ion binding, enzyme efficiency, and enzyme processivity. An example of such an alteration would be the substitution of a tyrosine for a glycine at amino acid position 61 of SEQ ID NO:26, SEQ ED NO:27, SEQ ID NO:33, SEQ ID NO:34, SEQ ID NO:35, or SEQ DD NO:48. Substitution of the chemically innocuous glycine for the polar tyrosine should facilitate hydrogen bonding between the N3 atom of ATP and this<br><br>
amino acid position, thereby improving the dissociation constant and energetics of this interaction. A tyrosine is found at this position in, for example, the more processive poly-A polymerase. As one skilled in the art will recognize, other modifications can be envisioned.<br>
EXAMPLE 10<br>
AMINO ACID CHANGES THAT IMPROVE DOUBLE-STRANDED RNA BINDING A second example of amino acid modifications to OAS that improve enzymatic activity would be those that stabilize the interaction between this protein and double-stranded viral RNA. The table below lists those amino acids in the RNA binding groove of the protein and several proposed changes designed to stabilize the interaction between the basic, positively charged amino acid side chains and the negatively charged ribonucleic acid. Changes are envisioned that increase the positive charge density in the RNA binding groove of the protein. As one skilled in the art will recognize, similar types of modifications to the RNA binding groove can be envisioned.<br>
TABLE 4 Proposed changes to amino acids in RNA binding groove<br>
TABLE REMOVED<br>
EXAMPLE 11<br>
ANALYSIS OF GENETIC MUTATIONS<br>
Those skilled in the art will recognize that numerous other analytical methods exist for assessing the evolutionary importance of particular mutations in a genetic analysis. One example is the well-known calculation of a linkage disequilibrium estimate, commonly referred to as D' (Lewontin, Genetics 49:49-67, 1964). Other particularly relevant methods attempt to estimate selective pressures and/or recent evolutionary events within a genetic locus (for example, selective sweeps) by comparing the relative abundance of high-, moderate-, or low-frequency mutations in the locus. Most familiar of these tests is the Tajima D statistic (Tajima, Genetics<br><br>
123:585-595, 1989). Fu and Li, Genetics 133:693-709 (1993) have also developed a variant to the Tajima and other statistics that also makes use of knowledge regarding the ancestral allele for each mutation. These and other methods are applied to the mutations of the present invention to assess relative contribution to the observed effects.<br>
The foregoing specification, including the specific embodiments and examples, is intended to be illustrative of the present invention and is not to be taken as limiting. Numerous other variations and modifications can be effected without departing from the true spirit and scope of the invention. All patents, patent publications, and non-patent publications cited are incorporated by reference herein.<br><br><br><br><br><br>
We claim:<br>
1. An isolated polypeptide for treating viral infection, said isolated polypeptide comprising of 335-346 amino acids in length, and comprising the fragment amino acid sequence of SEQ ID NO: 75, wherein said polypeptide has at least 95% sequence similarity to a polypeptide selected from a group consisting of SEQ ID NO:48, SEQ ID NO: 49, and SEQ ID NO: 50.<br>
2.	An isolated polypeptide as claimed in claim 1, wherein a single methionine residue is present at the N-terminus.<br>
3.	An isolated polypeptide as claimed in claim 1 or claim 2, optionally covalently attached to a polypeptide comprising a protein transduction domain.<br>
4.	An isolated polypeptide as claimed in claim 3 wherein the protein transduction domain is comprised of a polypeptide selected from the group consisting of SEQ ID NO: 85-94.<br>
5.	An isolated polypeptide as claimed in claim 3 wherein the protein transduction domain differs from a polypeptide selected from the group consisting of SEQ ID NO: 85-94 by the addition or substitution of an arginine, lysine, or histidine.<br>
6.	An isolated polypeptide as claimed in any one of claims 1-5 covalently attached to polyethylene glycol.<br>
7.	An isolated polypeptide as claimed in any one of claims 1-5 encapsulated in a liposome.<br><br>
8.	An isolated polypeptide as claimed in any one of claims 1-5 covalently attached to an endosome disrupting agent.<br>
9.	An isolated polypeptide as claimed in any one of claims 1-5 noncovalently attached to an endosome disrupting agent.<br>
10.	An isolated polypeptide as claimed in claims 8 or 9 wherein the endosome disrupting agent is pH sensitive.<br>
11.	An isolated polypeptide as claimed in any one of claims 1-5 covalently conjugated to a sugar moiety.<br>
12.	An isolated polypeptide as claimed in claim 11 wherein the sugar moiety is comprised of galactose or mannose.<br>
13.	An isolated polypeptide as claimed in any one of claims 1-5 which is produced by a recombinant prokaryotic or eukaryotic host cells, such as bacterial, yeast, higher plant, insect, and mammalian cells.<br>
14.	An isolated polypeptide as claimed in claim 1, wherein the amino acid sequence of said polypeptide is a variant of SEQ ID NO:48, SEQ ID NO: 49, and SEQ ID NO: 50 at one or more of positions 39, 42, 60, 61, 195, 199, and 204.<br>
15.	An isolated polypeptide as claimed in claim 14, wherein the amino acid at position 39 is Arg or Lys, at position 42 is Arg, at position 60 is Arg, at position 61 is Tyr, at position 195 is Lys, at position 199 is Arg, and/or at position 204 is Arg.<br><br>
16.      An isolated polypeptide as claimed in claim 1, wherein the said polypeptide is any one of SEQ ID NO: 48, SEQ ID NO: 49 and SEQ ID NO:50.<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUFic3RyYWN0LSgyMC0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Abstract-(20-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUFzc2lnbm1lbnQtKDMxLTA4LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Assignment-(31-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUNsYWltcyAoMDktMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Claims (09-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNsYWltcy0oMDQtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Claims-(04-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNsYWltcy0oMjAtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Claims-(20-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNsYWltcy0oMjQtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Claims-(24-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDQtMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Correspondence Others-(04-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjAtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Correspondence Others-(20-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjQtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Correspondence Others-(24-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMzEtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Correspondence Others-(31-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLW90aGVycyAoMDktMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Correspondence-others (09-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDQtMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Correspondence-Others-(04-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Correspondence-Others-(07-04-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTEtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Correspondence-Others-(11-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUZvcm0tMS0oMTEtMDgtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Form-1-(11-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUZvcm0tMS0oMjAtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Form-1-(20-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tMTMtKDExLTA4LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-13-(11-08-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUZvcm0tMTYtKDMxLTA4LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Form-16-(31-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1ERUxOUC0yMDA2LUZvcm0tMi0oMjAtMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-DELNP-2006-Form-2-(20-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUZvcm0tMy0oMDQtMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Form-3-(04-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LUdQQS0oMzEtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-GPA-(31-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LXBjdC0zMDYucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LVBldGl0aW9uIDEzNy0oMDQtMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Petition 137-(04-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjk2Ny1kZWxucC0yMDA2LVBldGl0aW9uIE90aGVycy0oMDctMDQtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2967-delnp-2006-Petition Others-(07-04-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="252647-a-composition-containing-a-lyophilisate-of-dedifferentiated-plant-cells-for-skin-depigmentation.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="252649-a-mycobacteria-cell-deficient-in-manlam-cap-specific-monnosyl-transferase-activity.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>252648</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2967/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>22/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Jun-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>25-May-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-May-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>ILLUMIGEN BIOSCIENCES, INC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>C/O CUBIST PHARMACEUTICALS,INC 65 HAYDEN AVENUE LEXINGTON,MA 02421,USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SHAWN P. IADONATO</td>
											<td>421 31ST AVENUE SEATTLE, WASHINGTON 98122 USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHARLES L. MAGNESS</td>
											<td>P.O.BOX 51182 SEATTLE, WASHINGTON 98115 USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>GARY ROSENBURG</td>
											<td>23519 23RD AVENUE SOUTHEAST BOTHELL, WASHINGTON 98021 USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>CHRISTINA A. SCHERER</td>
											<td>1837 12TH AVENUE WEST SEATTLE, WASHINGTON 98119 USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>THIERRY GUILLAUDEUX</td>
											<td>40, RUE MOTTE BARIL F-35000 RENNES. FRANCE USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 9/12</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/035284</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-10-22</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/605,243</td>
									<td>2004-08-26</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>69/513,888</td>
									<td>2003-10-23</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>3</td>
									<td>60/542,373</td>
									<td>2004-02-06</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>4</td>
									<td>60/554,758</td>
									<td>2004-03-19</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>5</td>
									<td>60/560,524</td>
									<td>2004-04-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/252648-an-isolated-polypeptide-for-treating-viral-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 13:16:14 GMT -->
</html>
