<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253277-an-isolated-mutant-flavivirus-e-glycoprotein-polypeptide-nucleic-acid-molecule-encoding-said-polypeptide-and-its-virus-like-particle by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:55:32 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253277:&quot;AN ISOLATED MUTANT FLAVIVIRUS E-GLYCOPROTEIN POLYPEPTIDE, NUCLEIC ACID MOLECULE ENCODING SAID POLYPEPTIDE AND ITS VIRUS-LIKE PARTICLE&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED MUTANT FLAVIVIRUS E-GLYCOPROTEIN POLYPEPTIDE, NUCLEIC ACID MOLECULE ENCODING SAID POLYPEPTIDE AND ITS VIRUS-LIKE PARTICLE&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An isolated mutant flavivirus E-glycoprotein polypeptide that exhibits measurably reduced antibody cross-reactivity as compared to the corresponding wild-type flavivirus E-glycoprotein, said mutant polypeptide comprising at least one amino acid substitution at position 104, 106, or 107 as compared to said corresponding wild-type flavivirus E-glycoprotein polypeptide.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>PRIORITY CLAIM<br>
This application claims the benefit of U.S. Provisional Patent Application No. 60/591,898 filed July 27, 2004, which is incorporated herein by reference in its entirety.<br>
STATEMENT OF GOVERNMENT SUPPORT<br>
This invention was made by the Centers for Disease Control and Prevention, an agency of the United States Government. Therefore, the U.S. Government has certain rights in this invention.<br>
FIELD<br>
This disclosure relates to a structure-based rational mutagenesis method for identifying flavivirus envelope (E)-glycoprotein cross-reactive epitopes. The disclosure further relates to flavivirus E-glycoprotein cross-reactive epitopes and mutants thereof having reduced or ablated cross-reactivity. Flavivirus cross-reactive E-glycoprotein epitopes with reduced or ablated cross-reactivity are useful in the diagnosis, inhibition and treatment of diseases caused by flaviviruses.<br>
BACKGROUND<br>
The Flaviviridae are a diverse family of enveloped viruses infecting both arthropods and vertebrates. Flaviviruses have a positive-sense single-stranded RNA genome 10.7 kb in length, transcribed into a single polyprotein precursor encoding three structural proteins, capsid, premembrane (prM), envelope (E), and seven non-structural proteins (Lindenbach &amp; Rice, Flaviviridae: the viruses and their replication.  In Fields Virology, 4th ed., Knipe and Howley. Eds., Philadelphia, Lippincott Williams &amp; Wilkins, pp. 991-1041, 2001; Rice et al, Science 229:726-33, 1985). The flavivirus E-glycoprotein is the primary antigen, inducing protective immunity; it is essential for membrane fusion, and mediates binding to cellular receptors (Allison et al., J. Virol. 75:4268-75, 2001; Crill &amp; Roehrig, J. Virol. 75:7769-73, 2001; Rey et al, Nature 375:291-98, 1995). Flavivirus E-glycoprotein therefore directly affects host range, tissue tropism, and the virulence of these viruses.<br>
The flavivirus E-glycoprotein contains three structural and functional domains. Domain I (DI) is an 8-stranded (3-barrel containing two large insertion loops that form the elongated finger-like domain II (DII) (Rey et al, Nature 375:291-98, 1995). DII is involved in stabilizing the E-glycoprotein dimer and contains the internal fusion peptide (Allison et al., J. Virol. 75:4268-75, 2001). Domain III (Dili) forms a ten-stranded P-barrel with an immunoglobulin-like fold and contains the cellular receptor-binding motifs (Crill &amp; Roehrig, J. Virol. 75:7769-73, 2001; Modis et al, PNAS 100:6986-91, 2003). DI and Dili contain predominately type-specific and subcomplex-reactive epitopes, whereas DII contains the major flavivirus group- and subgroup-cross-reactive<br>
epitopes, which are sensitive to reduction and denaturation and are formed from discontinuous amino<br>
acid sequences (Mandl et al, J. Virol. 63:564-71, 1989; Key et al, Nature 375:291-98, 1995;<br>
Roehrig et al., Virology 246:317-28, 1998).<br>
Members of the Flaviviridae family that infect humans frequently cause severe morbidity<br>
and mortality, and epidemics of flaviviruses continue to be a major public health concern worldwide.<br>
More than two billion people are at risk of being infected with members of the genus Flavivirus which<br>
includes at least 70 distinct virus species (Burke &amp; Monath, Flaviviruses. In Fields Virology, 4th ed.,<br>
Knipe and Howley. Eds., Philadelphia, Lippincott Williams &amp; Wilkins, pp. 1043-1125, 2001; Kuno et<br>
al.,J. Virol. 72:73-83, 1998; Solomon &amp; Mallewa, J. Infect. 42:104-15, 2001). The medically<br>
important flaviviruses include yellow fever (YF) virus in Africa, Latin and South America; Japanese<br>
encephalitis (JE) virus in Asia and Australia; West Nile (WN) virus in Africa, Central Europe, and<br>
most recently in North America; tick-borne encephalitis (TBE) complex viruses in the temperate<br>
regions of Europe, North America and Asia; and the four serotypes of dengue viruses (DEN-1, -2, -3,<br>
and -4) in tropical and subtropical regions of the world (Lindenbach &amp; Rice, Flaviviridae: the viruses<br>
and their replication. In Fields Virology, 4th ed., Knipe and Howley. Eds., Philadelphia, Lippincott<br>
Williams &amp; Wilkins, pp. 991-1041, 2001).<br>
Human infection by flaviviruses results in a humoral immune response involving virus<br>
species-specific as well as flavivirus cross-reactive antibodies (Calisher et al.,J. Gen. Virol. 70:37-43,<br>
1989; Tesh etal., Emerg. Inf. Dis. 8:245-51, 2002). The presence of flavivirus cross-reactive<br>
antibodies in human sera produces two public health concerns upon secondary infection with a<br>
heterologous flavivirus. Serodiagnosis of secondary flavivirus infections, especially in areas with<br>
multiple co-circulating flaviviruses, can be particularly difficult due to the inability to differentiate<br>
primary from secondary cross-reactive serum antibodies using currently available viral antigens.<br>
Therefore, definitive epidemiological information either cannot be obtained or is delayed to the point<br>
that effective control and prevention strategies may be delayed. Additionally, the presence of subneutralizing<br>
levels of flavivirus cross-reactive serum antibodies may result in increasing the severity<br>
of secondary flavivirus infections due to antibody-dependant enhancement (ADE), in particular,<br>
following secondary dengue virus infection (Ferguson et al.,PNAS 96:790-94, 1999; Halstead, Rev.<br>
Infect. Dis. 11:830-39, 1989; Takada &amp; Kawaoka, Rev. Med. Virol. 13:387-98, 2003; Wallace et al.,<br>
J. Gen Virol. 84:1723-28, 2003). Thus, there exists a need for a method for identifying and<br>
characterizing flavivirus cross-reactive epitopes for improved flavivirus serodiagnosis and<br>
development of flavivirus vaccines.<br>
SUMMARY OF THE DISCLOSURE<br>
Multiple flavivirus E-glycoprotein cross-reactive epitopes and mutant E-glycoprotein<br>
polypeptides thereof exhibiting reduced or ablated cross-reactivity have been identified. In various<br>
embodiments, these E-glycoprotein polypeptides with reduced or ablated cross-reactivity are capable<br>
of eliciting effective type-specific immune responses against flaviviruses. In one example, the<br>
identified cross-reactive epitopes incorporate the highly conserved Gly104, Glyioe, and Leuio? residues.<br>
In another example, the identified cross-reactive epitope centers on the strictly conserved Trp23l<br>
residue and its structurally related neighbors Glui26 and Thr226.<br>
Also described herein are recombinant flavivirus E-glycoprotein constructs that can be used<br>
directly or indirectly to stimulate flavivirus type-specific antibodies. These constructs are designed to<br>
elicit T-cell, B-cell, or both T-cell and B-cell responses against flavivirus type-specific epitopes. The<br>
constructs, when integrated into a vector, can be used as immunogens, can be used as DNA vaccines,<br>
and can be used as sources of recombinant protein for stimulation of immune responses in subjects, as<br>
well as for protein boosts to subjects who have received a nucleic acid construct previously. Also<br>
provided are methods of identifying and characterizing flavivirus E-glycoprotein amino acid residues<br>
incorporated into cross-reactive epitopes, using structure-based rational mutagenesis.<br>
The foregoing and other features and advantages will become more apparent from the<br>
following detailed description of several embodiments, which proceeds with reference to the<br>
accompanying figures,<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 is a diagrammatical representation of the quaternary structure of the DEN-2 virus<br>
E-glycoprotein homodimer, top view, looking down towards the viral surface, showing the locations<br>
of flavivirus cross-reactive epitope residues (space-filling representation). The structural and<br>
functional domains I, II, and III are also shown.<br>
Figure 2 is a series of diagrammatical representations of the structural locations of crossreactive<br>
epitope residues for flavivirus cross-reactive monoclonal antibodies (mAbs) in the atomic<br>
structure of the DEN-2 virus E-glycoprotein dimer, as well as a bar graph indicating fold reductions in<br>
mAb reactivities assayed by indirect immuno-fluorescence assay (IFA) and/or antigen-capture ELISA<br>
(Ag-ELISA) for mutations at these E-glycoprotein positions.<br>
FIG. 2A is a diagrammatical representation of a portion of the atomic structure of the DEN-2<br>
virus E-glycoprotein homodimer, showing the flavivirus group-reactive mAb 4G2 and 6B6C-1<br>
epitope residues from the fusion peptide region of DII. The flavivirus fusion peptide comprises the<br>
highly conserved E-glycoprotein residues 98-113, which form a surface exposed loop of hydrophobic<br>
residues rich in glycine at the tip of DII (Key et at, Nature 375:291-98, 1995; Allison et al.,J. Virol.<br>
75:4268-75, 2001). The view is looking downward toward the viral membrane surface at an angle of<br>
approximately 45°, while looking in towards the fusion peptide region about 45° off of parallel to the<br>
dimer's longitudinal axis. The molecular surfaces of DI and Dili from the alternate sub-unit are<br>
depicted as space-filling Van der Waals surfaces to highlight the close fitting of the fusion peptide<br>
into this region. Fusion peptide residues 100-108 are depicted as stick representations with the<br>
participating amino acids labeled. Glycan moieties attached to Asn153 and Asn67 are labeled CHOI53<br>
and CHO67, respectively.<br>
FIG. 2B is a diagrammatical representation of a portion of the atomic structure of the DEN-2<br>
virus E-glycoprotein homodimer, showing the flavivirus subgroup-reactive mAb 1B7-5 epitope<br>
residues. The view and labeling are the same as in FIG. 2A. Identified residues are depicted as sticks<br>
and labeled.<br>
FIG. 2C is a bar graph showing fold decreases in mAb reactivities in Ag-ELISA for DEN-2<br>
VLPs with substitutions at the listed residues. mAbs 4G2 and 6B6C-1 are flavivirus group-reactive<br>
and 1B7-5 is flavivirus subgroup-reactive. Substitutions at G)04 and W23] produced plasmids that<br>
were unable to secrete measurable VLP antigen into tissue culture media. Therefore, fold decreases<br>
in mAb reactivities for these two constructs are from IFA. Wild-type plasmid did not produce an<br>
endpoint nearly as far out in IFA as in Ag-ELISA (see Table 3), therefore the fold reductions for<br>
substitutions at G)04 and W23i were not as great as for other constructs measured by Ag-ELISA, even<br>
though substitutions at these two positions appeared to completely ablate mAb reactivity.<br>
Figure 3 is a bar graph showing the percent of cross-reactive epitope residue substitutions<br>
altering reactivities of mAbs of different cross-reactivities. The total number of SLEV and WNV<br>
mAbs of each type is shown in the legend on the y-axis, and the number of substitutions altering these<br>
mAbs is shown in the columns.<br>
BRIEF DESCRIPTION OF THE APPENDICES<br>
Appendix I contains Tables 1-13.<br>
SEQUENCE LISTING<br>
The nucleic and amino acid sequences listed in the accompanying sequence listing are<br>
shown using standard letter abbreviations for nucleotide bases, and three letter code for amino acids,<br>
as defined in 37 C.F.R. 1.822. Only one strand of each nucleic acid sequence is shown, but the<br>
complementary strand is understood as included by any reference to the displayed strand. In the<br>
accompanying sequence listing:<br>
SEQ ID NOs: 1-12 show the nucleic acid sequences of mutagenic primers used to generate<br>
the K64N mutation, T76M mutation, Q77R mutation, G104H mutation, G,06Q mutation, L107K mutation,<br>
E,26A mutation, T226N mutation, W23|F mutation, W23)L mutation, H244R mutation, and K247R<br>
mutation, respectively, in the pCB8D2-2J-2-9-l DEN-2 prM/E expression plasmid.<br>
SEQ ID NOs: 13 and 14 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen.<br>
SEQ ID NOs: 15 and 16 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the G104H substitution.<br>
SEQ ID NOs: 17 and 18 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the Gjo6Q substitution.<br>
SEQ ID NOs: 19 and 20 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the Li0?K substitution.<br>
SEQ ID NOs: 21 and 22 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the E126A substitution.<br>
SEQ ID NOs: 23 and 24 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the T226N substitution.<br>
SEQ ID NOs: 25 and 26 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the W23iF substitution.<br>
SEQ ID NOs: 27 and 28 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the W23iL substitution.<br>
SEQ ID NOs: 29 and 30 show the nucleic and amino acid sequences of a recombinant DEN-<br>
2 virus E-glycoprotein antigen incorporating the double E)26A/ T226N substitution.<br>
SEQ ID NOs: 31-79 show the nucleic acid sequences of mutagenic primers used to generate<br>
site-specific mutations into the SLEV and WNV E genes.<br>
SEQ ID NOs: 80 and 81 show the nucleic and amino acid sequences of a recombinant SLEV<br>
virus E-glycoprotein antigen.<br>
SEQ ID NOs: 82 and 83 show the nucleic and amino acid sequences of a recombinant SLEV<br>
virus E-glycoprotein antigen incorporating the G|06Q substitution.<br>
SEQ ID NOs: 84 and 85 show the nucleic and amino acid sequences of a recombinant WNV<br>
virus E-glycoprotein antigen.<br>
SEQ ID NOs: 86 and 87 show the nucleic and amino acid sequences of a recombinant WNV<br>
virus E-glycoprotein antigen incorporating the GmV substitution.<br>
DETAILED DESCRIPTION<br>
AbbreviationsADE antibody-dependant enhancementAg-ELISA antigen-capture ELISAD domainDEN dengue<br>
DENY dengue virus<br>
E envelope<br>
ELISA enzyme-linked immunoabsorbent assay<br>
IFA indirect immuno-fluorescence assay<br>
JE Japanese encephalitis<br>
JEV Japanese encephalitis virus<br>
mAb monoclonal antibody<br>
MHIAF murine hyper-immune ascetic fluid<br>
MVEV Murray Valley encephalitis virus<br>
PCR polymerase chain reaction<br>
prM premembrane<br>
SLE St. Louis encephalitis<br>
SLEV St. Louis encephalitis virus<br>
TBE tick-borne encephalitis<br>
VLP virus-like particle<br>
WN West Nile<br>
WNV West Nile virus<br>
YF yellow fever<br>
//. Terms<br>
Unless otherwise noted, technical terms are used according to conventional usage.<br>
Definitions of common terms in molecular biology may be found in Benjamin Lewin, Genes VII,<br>
published by Oxford University Press, 2000 (ISBN 019879276X); Kendrew et al. (eds.), The<br>
Encyclopedia of Molecular Biology, published by Blackwell Publishers, 1994 (ISBN 0632021829);<br>
and Robert A. Meyers (ed.), Molecular Biology and Biotechnology: a Comprehensive Desk<br>
Reference, published by Wiley, John &amp; Sons, Inc., 1995 (ISBN 0471186341); and other similar<br>
references.<br>
As used herein, the singular terms "a," "an," and "the" include plural referents unless context<br>
clearly indicates otherwise. Similarly, the word "or" is intended to include "and" unless the context<br>
clearly indicates otherwise. Also, as used herein, the term "comprises" means "includes." Hence<br>
"comprising A or B" means including A, B, or A and B. It is further to be understood that all base<br>
sizes or amino acid sizes, and all molecular weight or molecular mass values, given for nucleic acids<br>
or polypeptides are approximate, and are provided for description. Although methods and materials<br>
similar or equivalent to those described herein can be used in the practice or testing of the present<br>
invention, suitable methods and materials are described below. All publications, patent applications,<br>
patents, and other references mentioned herein are incorporated by reference in their entirety. In case<br>
of conflict, the present specification, including explanations of terms, will control. The materials,<br>
methods, and examples are illustrative only and not intended to be limiting.<br>
In order to facilitate review of the various embodiments of this disclosure, the following<br>
explanations of specific terms are provided:<br>
Animal: Living multi-cellular vertebrate organisms, a category that includes, for example,<br>
mammals and birds. The term mammal includes both human and non-human mammals. Similarly,<br>
the term "subject" includes both human and veterinary subjects, for example, humans, non-human<br>
primates, dogs, cats, horses, and cows.<br>
Antibody: A protein (or protein complex) that includes one or more polypeptides<br>
substantially encoded by immunoglobulin genes or fragments of immunoglobulin genes. The<br>
recognized immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon, and mu<br>
constant region genes, as well as the myriad immunoglobulin variable region genes. Light chains are<br>
classified as either kappa or lambda. Heavy chains are classified as gamma, mu, alpha, delta, or<br>
epsilon, which in turn define the immunoglobulin classes, IgG, IgM, IgA, IgD and IgE, respectively.<br>
The basic immunoglobulin (antibody) structural unit is generally a tetramer. Each tetramer<br>
is composed of two identical pairs of polypeptide chains, each pair having one "light" (about 25 kDa)<br>
and one "heavy" (about 50-70 kDa) chain. The N-terminus of each chain defines a variable region of<br>
about 100 to 110 or more amino acids primarily responsible for antigen recognition. The terms<br>
"variable light chain" (VL) and "variable heavy chain" (VH) refer, respectively, to these light andheavy chains.<br>
As used herein, the term "antibody" includes intact immunoglobulins as well as a number of<br>
well-characterized fragments. For instance, Fabs, Fvs, and single-chain Fvs (SCFvs) that bind to<br>
target protein (or epitope within a protein or fusion protein) would also be specific binding agents for<br>
that protein (or epitope). These antibody fragments are as follows: (1) Fab, the fragment which<br>
contains a monovalent antigen-binding fragment of an antibody molecule produced by digestion of<br>
whole antibody with the enzyme papain to yield an intact light chain and a portion of one heavy chain;<br>
(2) Fab', the fragment of an antibody molecule obtained by treating whole antibody with pepsin,<br>
followed by reduction, to yield an intact light chain and a portion of the heavy chain; two Fab'<br>
fragments are obtained per antibody molecule; (3) (Fab')2, the fragment of the antibody obtained by<br>
treating whole antibody with the enzyme pepsin without subsequent reduction; (4) F(ab')2, a dimer of<br>
two Fab' fragments held together by two disulfide bonds; (5) Fv, a genetically engineered fragment<br>
containing the variable region of the light chain and the variable region of the heavy chain expressed<br>
as two chains; and (6) single chain antibody, a genetically engineered molecule containing the<br>
variable region of the light chain, the variable region of the heavy chain, linked by a suitable<br>
polypeptide linker as a genetically fused single chain molecule. Methods of making these fragments<br>
are routine (see, for example, Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL,<br>
New York, 1999).<br>
Antibodies for use in the methods and compositions of this disclosure can be monoclonal or<br>
polyclonal. Merely by way of example, monoclonal antibodies can be prepared from murine<br>
hybridomas according to the classical method of Kohler and Milstein (Nature 256:495-97, 1975) or<br>
derivative methods thereof. Detailed procedures for monoclonal antibody production are described in<br>
Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
Antibody binding affinity: The strength of binding between a single antibody binding site<br>
and a ligand (e.g., an antigen or epitope). The affinity of an antibody binding site X for a ligand Y is<br>
represented by the dissociation constant (Kd), which is the concentration of Y that is required to<br>
occupy half of the binding sites of X present in a solution. A smaller (Kd) indicates a stronger or<br>
higher-affinity interaction between X and Y and a lower concentration of ligand is needed to occupy<br>
the sites. In general, antibody binding affinity can be affected by the alteration, modification and/or<br>
substitution of one or more amino acids in the epitope recognized by the antibody paratope.<br>
In one example, antibody binding affinity is measured by end-point titration in an Ag-ELISA<br>
assay. Antibody binding affinity is substantially lowered (or measurably reduced) by the modification<br>
and/or substitution of one or more amino acids in the epitope recognized by the antibody paratope if<br>
the end-point titer of a specific antibody for the modified/substituted epitope differs by at least 4-fold,<br>
such as at least 10-fold, at least 100-fold or greater, as compared to the unaltered epitope.<br>
Antigen: A compound, composition, or substance that can stimulate the production of<br>
antibodies or a T-cell response in an animal, including compositions that are injected or absorbed into<br>
an animal. An antigen reacts with the products of specific humoral or cellular immunity, including<br>
those induced by heterologous immunogens. In one embodiment, an antigen is a flavivirus antigen.<br>
cDNA (complementary DNA): A piece of DNA lacking internal, non-coding segments<br>
(introns) and regulatory sequences that determine transcription. cDNA is synthesized in the<br>
laboratory by reverse transcription from messenger RNA extracted from cells.<br>
Epitope: An antigenic determinant. These are particular chemical groups, such as<br>
contiguous or non-contiguous peptide sequences, on a molecule that are antigenic, that is, that elicit a<br>
specific immune response. An antibody binds a particular antigenic epitope based on the three<br>
dimensional structure of the antibody and the matching (or cognate) three dimensional structure of the<br>
epitope.<br>
A "cross-reactive epitope" is an epitope found in two or more antigens expressed by different<br>
genes, and responsible for inducing cross-reactive antibodies. For example, a "flavivirus crossreactive<br>
epitope" is a flavivirus epitope found in a peptide from two or more flaviviruses, and<br>
responsible for inducing flavivirus cross-reactive antibodies.<br>
A "substituted epitope" comprises at least one structural substitution in the epitope, such as a<br>
substitution of one amino acid for another. In certain provided embodiments, amino acid substitutions<br>
at probable or identified cross-reactive epitope residues are designed to reduce or ablate crossreactive<br>
antibody recognition without substantially altering E-glycoprotein structural conformation oraffecting type-specific antibody binding sites, disrupting dimer interactions, or impairing particle<br>
formation, maturation, or secretion.<br>
Flavivirus cross-reactive antibody: An antibody that recognizes (that is, specifically binds<br>
to) an epitope found on a peptide from more than one species of flavivirus. Flavivirus cross-reactive<br>
antibodies are classified as either complex cross-reactive or group cross-reactive antibodies.<br>
Complex cross-reactive antibodies recognize epitopes shared by all viruses within a complex, such as<br>
the JE virus complex or the DEN virus complex. Group cross-reactive antibodies recognize epitopes<br>
shared by all members of the genus Flavivirus.<br>
Antibody cross-reactivity is further refined within the sub-complex and sub-group crossreactive<br>
categories. Sub-complex cross-reactive antibodies recognize epitopes shared by most, but<br>
not all, members of a particular flavivirus complex (e.g., DENV-1, -2, and -3, but not DENV-4),<br>
while sub-group cross-reactive antibodies recognize epitopes shared by flaviviruses from several<br>
complexes, but not all members of the flavivirus group (e.g., all members of the DEN virus and JE<br>
virus complexes, but not all members of the tick-borne virus complex). Specific, non-limiting<br>
examples of flavivirus cross-reactive antibodies include the group cross-reactive mAbs 4G2 and<br>
6B6C-1, the sub-group cross-reactive mAb 1B7-5, and the sub-complex cross-reactive mAb 10A1D-<br>
2.<br>
Flavivirus E-glycoprotein: A structural envelope protein that mediates binding of<br>
flavivirus virions to cellular receptors on host cells. The flavivirus E-glycoprotein is required for<br>
membrane fusion, and is the primary antigen inducing protective immunity to flavivirus infection.<br>
Flavivirus E-glycoprotein affects host range, tissue tropism and viral virulence. The flavivirus Eglycoprotein<br>
contains three structural and functional domains, DI-DIII. In mature virus particles the<br>
E-glycoprotein forms head to tail homodimers lying flat and forming a dense lattice on the viral<br>
surface.<br>
Flavivirus E-glycoprotein domain: A domain of a protein is a part of a protein that shares<br>
common structural, physiochemical and/or functional features; for example hydrophobic, polar,<br>
globular, helical domains or properties, for example a DNA binding domain, an ATP binding domain,<br>
and the like. The flavivirus E-glycoprotein contains three recognized structural and functional<br>
domains, DI-DIII. DI is an 8-stranded P-barrel containing two large insertion loops that form the<br>
elongated finger-like DII. DII is involved in stabilizing the E-glycoprotein dimer and contains the<br>
internal fusion peptide that mediates flaviviral entry into host cells via membrane fusion. Dili forms a<br>
ten-stranded p-barrel with an immunoglobulin-like fold and contains the cellular receptor-binding<br>
motifs. DI and Dili contain predominately type- and subtype-specific epitopes, whereas DII contains<br>
the major flavivirus group and subgroup cross-reactive epitopes, which are sensitive to reduction and<br>
denaturation and are therefore believed to be formed from discontinuous amino acid sequences.<br>
Flavivirus type-specific antibody: An antibody that recognizes (that is, specifically binds<br>
to) an epitope found on a peptide from only.one specific member of the flaviviruses. Specific, nonlimiting<br>
examples of flavivirus type-specific antibodies include: DI mAb 9A4D-1, DII mAb 1A5D-1,<br>
and Dili mAbs 3H5, 9A3D-8 and 9D12, which only recognize epitopes found in the DENV-2 Eglycoprotein.<br>
Hybridization: Oligonucleotides and their analogs hybridize by hydrogen bonding, which<br>
includes Watson-Crick, Hoogsteen or reversed Hoogsteen hydrogen bonding, between<br>
complementary bases. Generally, nucleic acid consists of nitrogenous bases that are either<br>
pyrimidines (cytosine (C), uracil (U), and thymine (T)) or purines (adenine (A) and guanine (G)).<br>
These nitrogenous bases form hydrogen bonds between a pyrimidine and a purine, and the bonding of<br>
the pyrimidine to the purine is referred to as "base pairing." More specifically, A will hydrogen bond<br>
to T or U, and G will bond to C. "Complementary" refers to the base pairing that occurs between to<br>
distinct nucleic acid sequences or two distinct regions of the same nucleic acid sequence.<br>
"Specifically hybridizable" and "specifically complementary" are terms that indicate a<br>
sufficient degree of complementarity such that stable and specific binding occurs between the<br>
oligonucleotide (or its analog) and the DNA or UNA target. The oligonucleotide or oligonucleotide<br>
analog need not be 100% complementary to its target sequence to be specifically hybridizable. An<br>
oligonucleotide or analog is specifically hybridizable when binding of the oligonucleotide or analog<br>
10to the target DNA or RNA molecule interferes with the normal Function of the target DNA or RNA,<br>
and there is a sufficient degree of complementarity to avoid non-specific binding of the<br>
oligonucleotide or analog to non-target sequences under conditions where specific binding is desired,<br>
for example under physiological conditions in the case of in vivo assays or systems. Such binding is<br>
referred to as specific hybridization.<br>
Hybridization conditions resulting in particular degrees of stringency will vary depending<br>
upon the nature of the hybridization method of choice and the composition and length of the<br>
hybridizing nucleic acid sequences. Generally, the temperature of hybridization and the ionic strength<br>
(especially the Na+ and/or Mg++ concentration) of the hybridization buffer will determine the<br>
stringency of hybridization, though wash times also influence stringency. Calculations regarding<br>
hybridization conditions required for attaining particular degrees of stringency are discussed by<br>
Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring<br>
Harbor Laboratory Press, Cold Spring Harbor, NY, 1989, chapters 9 and 11; and Ausubel et al. Short<br>
Protocols in Molecular Biology, 4th ed., John Wiley &amp; Sons, Inc., 1999.<br>
For purposes of the present disclosure, "stringent conditions" encompass conditions under<br>
which hybridization will only occur if there is less than 25% mismatch between the hybridization<br>
molecule and the target sequence. "Stringent conditions" may be broken down into particular levels<br>
of stringency for more precise definition. Thus, as used herein, "moderate stringency" conditions are<br>
those under which molecules with more than 25% sequence mismatch will not hybridize; conditions<br>
of "medium stringency" are those under which molecules with more than 15% mismatch will not<br>
hybridize, and conditions of "high stringency" are those under which sequences with more than 10%<br>
mismatch will not hybridize. Conditions of "very high stringency" are those under which sequences<br>
with more than 6% mismatch will not hybridize.<br>
"Specific hybridization" refers to the binding, duplexing, or hybridizing of a molecule only<br>
or substantially only to a particular nucleotide sequence when that sequence is present in a complex<br>
mixture (for example, total cellular DNA or RNA). Specific hybridization may also occur under<br>
conditions of varying stringency.<br>
Immune stimulatory composition: A term used herein to mean a composition useful for<br>
stimulating or eliciting a specific immune response (or immunogenic response) in a vertebrate. The<br>
immune stimulatory composition can be a protein antigen or a plasmid vector used to express a<br>
protein antigen. In some embodiments, the immunogenic response is protective or provides<br>
protective immunity, in that it enables the vertebrate animal to better resist infection with or disease<br>
progression from the organism against which the immune stimulatory composition is directed.<br>
Without wishing to be bound by a specific theory, it is believed that an immunogenic<br>
response induced by an immune stimulatory composition may arise from the generation of an<br>
antibody specific to one or more of the epitopes provided in the immune stimulatory composition.<br>
Alternatively, the response may comprise a T-helper or cytotoxic cell-based response to one or more<br>
of the epitopes provided in the immune stimulatory composition. All three of these responses may<br>
originate from nai've or memory cells. One specific example of a type of immune stimulatory<br>
composition is a vaccine.<br>
In some embodiments, an "effective amount" or "immune-stimulatory amount" of an immune<br>
stimulatory composition is an amount which, when administered to a subject, is sufficient to engender<br>
a detectable immune response. Such a response may comprise, for instance, generation of an<br>
antibody specific to one or more of the epitopes provided in the immune stimulatory composition.<br>
Alternatively, the response may comprise a T-helper or CTL-based response to one or more of the<br>
epitopes provided in the immune stimulatory composition. All three of these responses may originate<br>
from nai've or memory cells. In other embodiments, a "protective effective amount" of an immune<br>
stimulatory composition is an amount which, when administered to a subject, is sufficient to confer<br>
protective immunity upon the subject.<br>
Inhibiting or treating a disease: Inhibiting the full development of a disease or condition,<br>
for example, in a subject who is at risk for a disease. Specific examples of diseases include dengue<br>
fever, dengue hemorrhagic fever, yellow fever, Japanese encephalitis, tick-borne encephalitis, and<br>
West Nile disease. "Treatment" refers to a therapeutic intervention that ameliorates a sign or<br>
symptom of a disease or pathological condition after it has begun to develop. As used herein, the<br>
term "ameliorating," with reference to a disease, pathological condition or symptom, refers to any<br>
observable beneficial effect of the treatment. The beneficial effect can be evidenced, for example, by<br>
a delayed onset of clinical symptoms of the disease in a susceptible subject, a reduction in severity of<br>
some or all clinical symptoms of the disease, a slower progression of the disease, a reduction in the<br>
number of relapses of the disease, an improvement in the overall health or well-being of the subject,<br>
or by other parameters well known in the art that are specific to the particular disease.<br>
Isolated: An "isolated" or "purified" biological component (such as a nucleic acid, peptide,<br>
protein, protein complex, or particle) has been substantially separated, produced apart from, or<br>
purified away from other biological components in the cell of the organism in which the component<br>
naturally occurs, that is, other chromosomal and extrachromosomal DNA and RNA, and proteins.<br>
Nucleic acids, peptides and proteins that have been "isolated" or "purified" thus include nucleic acids<br>
and proteins purified by standard purification methods. The term also embraces nucleic acids,<br>
peptides and proteins prepared by recombinant expression in a host cell, as well as chemically<br>
synthesized nucleic acids or proteins. The term "isolated" or "purified" does not require absolute<br>
purity; rather, it is intended as a relative term. Thus, for example, an isolated biological component is<br>
one in which the biological component is more enriched than the biological component is in its<br>
natural environment within a cell, or other production vessel. Preferably, a preparation is purified<br>
such that the biological component represents at least 50%, such as at least 70%, at least 90%, at least<br>
95%, or greater, of the total biological component content of the preparation.<br>
Nucleic acid molecule: A polymeric form of nucleotides, which may include both sense and<br>
anti-sense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the<br>
above. A nucleotide refers to a ribonucleotide, deoxynucleotide or a modified form of either type of<br>
nucleotide. The term "nucleic acid molecule" as used herein is synonymous with "nucleic acid" and<br>
"polynucleotide." A nucleic acid molecule is usually at least 10 bases in length, unless otherwise<br>
specified. The term includes single- and double-stranded forms of DNA. A polynucleotide may<br>
include either or both naturally occurring and modified nucleotides linked together by naturally<br>
occurring and/or non-naturally occurring nucleotide linkages.<br>
Oligonucleotide : A nucleic acid molecule generally comprising a length of 300 bases or<br>
fewer. The term often refers to single-stranded deoxyribonucleotides, but it can refer as well to<br>
single- or double-stranded ribonucleotides, RNA:DNA hybrids and double-stranded DNAs, among<br>
others. The term "oligonucleotide" also includes oligonucleosides (that is, an oligonucleotide minus<br>
the phosphate) and any other organic base polymer.<br>
In some examples, oligonucleotides are about 10 to about 90 bases in length, for example,<br>
12, 13, 14, 15, 16, 17, 18, 19 or 20 bases in length. Other oligonucleotides are about 25, about 30,<br>
about 35, about 40, about 45, about 50, about 55, about 60 bases, about 65 bases, about 70 bases,<br>
about 75 bases or about 80 bases in length. Oligonucleotides may be single-stranded, for example,<br>
for use as probes or primers, or may be double-stranded, for example, for use in the construction of a<br>
mutant gene. Oligonucleotides can be either sense or anti-sense oligonucleotides. An oligonucleotide<br>
can be modified as discussed above in reference to nucleic acid molecules. Oligonucleotides can be<br>
obtained from existing nucleic acid sources (for example, genomic or cDNA), but can also be<br>
synthetic (for example, produced by laboratory or in vitro oligonucleotide synthesis).<br>
Operably linked: A first nucleic acid sequence is operably linked with a second nucleic<br>
acid sequence when the first nucleic acid sequence is placed in a functional relationship with the<br>
second nucleic acid sequence. For instance, a promoter is operably linked to a coding sequence is the<br>
promoter affects the transcription or expression of the coding sequence. Generally, operably linked<br>
DNA sequences are contiguous and, where necessary to join two protein coding regions, in the same<br>
reading frame. If introns are present, the operably linked DNA sequences may not be contiguous.<br>
Paratope: That portion of an antibody that is responsible for its binding to an antigenic<br>
determinant (epitope) on an antigen.<br>
Polypeptide: A polymer in which the monomers are amino acid residues joined together<br>
through amide bonds. When the amino acids are alpha-amino acids, either the L-optical isomer or the<br>
D-optical isomer can be used, the L-isomers being preferred for many biological uses. The terms<br>
"polypeptide" or "protein" as used herein are intended to encompass any amino acid molecule and<br>
include modified amino acid molecules such as glycoproteins. The term "polypeptide" is specifically<br>
intended to cover naturally occurring proteins, as well as those which are recombinantly or<br>
synthetically produced.<br>
Probes and primers: A probe comprises an isolated nucleic acid molecule attached to a<br>
detectable label or other reporter molecule. Typical labels include radioactive isotopes, enzyme<br>
substrates, co-factors, ligands, chemiluminescent or fluorescent agents, haptens, and enzymes.<br>
Methods for labeling and guidance in the choice of labels appropriate for various purposes are<br>
discussed, for example, in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed.,<br>
vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989 and Ausubel et al.<br>
Short Protocols in Molecular Biology, 4th ed., John Wiley &amp; Sons, Inc., 1999.<br>
Primers are short nucleic acid molecules, for instance DNA oligonucleotides 6 nucleotides or<br>
more in length, for example that hybridize to contiguous complementary nucleotides or a sequence to<br>
be amplified. Longer DNA oligonucleotides may be about 10, 12, 15, 20, 25, 30, or 50 nucleotides or<br>
more in length. Primers can be annealed to a complementary target DNA strand by nucleic acid<br>
hybridization to form a hybrid between the primer and the target DNA strand, and then the primer<br>
extended along the target DNA strand by a DNA polymerase enzyme. Primer pairs can be used for<br>
amplification of a nucleic acid sequence, for example, by the polymerase chain reaction (PCR) or<br>
other nucleic-acid amplification methods known in the art. Other examples of amplification include<br>
strand displacement amplification, as disclosed in U.S. Patent No. 5,744,311; transcription-free<br>
isothermal amplification, as disclosed in U.S. Patent No. 6,033,881; repair chain reaction<br>
amplification, as disclosed in WO 90/01069; ligase chain reaction amplification, as disclosed in EPA-<br>
320 308; gap filling ligase chain reaction amplification, as disclosed in 5,427,930; and NASBA™<br>
RNA transcription-free amplification, as disclosed in U.S. Patent No. 6,025,134.<br>
Methods for preparing and using nucleic acid probes and primers are described, for example,<br>
in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring<br>
Harbor Laboratory Press, Cold Spring Harbor, NY, 1989; Ausubel et al. Short Protocols in<br>
Molecular Biology, 4* ed., John Wiley &amp; Sons, Inc., 1999; and Innis et al. PCR Protocols, A Guide<br>
to Methods and Applications, Academic Press, Inc., San Diego, CA, 1990. Amplification primer<br>
pairs can be derived from a known sequence, for example, by using computer programs intended for<br>
that purpose such as Primer (Version 0.5, © 199J, Whitehead Institute for Biomedical Research,<br>
Cambridge, MA). One of ordinary skill in the art will appreciate that the specificity of a particular<br>
probe or primer increases with its length. Thus, in order to obtain greater specificity, probes and<br>
primers can be selected that comprise at least 20, 25, 30, 35, 40, 45, 50 or more consecutive<br>
nucleotides of a target nucleotide sequences.<br>
Recombinant nucleic acid: A nucleic acid molecule that is not naturally occurring or has a<br>
sequence that is made by an artificial combination of two otherwise separated segments of sequence.<br>
This artificial combination is accomplished by chemical synthesis or, more commonly, by the<br>
artificial manipulation of isolated segments of nucleic acids, e.g., by genetic engineering techniques<br>
such as those described in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed.,<br>
vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. The term<br>
recombinant includes nucleic acids that have been altered solely by addition, substitution, or deletion<br>
of a portion of a natural nucleic acid molecule.<br>
Regulatory sequences or elements: These terms refer generally to a class of DNA<br>
sequences that influence or control expression of genes. Included in the term are promoters,<br>
enhancers, locus control regions (LCRs), insulators/boundary elements, silencers, matrix attachment<br>
regions (MARs, also referred to as scaffold attachment regions), represser, transcriptional<br>
terminators, origins of replication, centromeres, and meiotic recombination hotspots. Promoters are<br>
sequences of DNA near the 5'-end of a gene that act as a binding site for DNA-dependent UNA<br>
polymerase, and from which transcription is initiated. Enhancers are control elements that elevate the<br>
level of transcription from a promoter, usually independently of the enhancer's orientation or distance<br>
from the promoter. LCRs confer tissue-specific and temporally regulated expression to genes to<br>
which they are linked. LCRs function independently of their position in relation to the gene, but are<br>
copy-number dependent. It is believed that they function to open the nucleosome structure, so other<br>
factors can bind to the DNA. LCRs may also affect replication timing and origin usage. Insulators<br>
(also know as boundary elements) are DNA sequences that prevent the activation (or inactivation) of<br>
transcription of a gene, by blocking effects of surrounding chromatin. Silencers and repressers are<br>
control elements that suppress gene expression; they act on a gene independently of their orientation<br>
or distance from the gene. MARs are sequences within DNA that bind to the nuclear scaffold; they<br>
can affect transcription, possibly by separating chromosomes into regulatory domains. It is believedthat MARs mediate higher-order, looped structures within chromosomes. Transcriptional terminators<br>
are regions within the gene vicinity where RNA Polymerase is released from the template. Origins of<br>
replication are regions of the genome, during DNA synthesis or replication phases of cell division,<br>
that begin the replication process of DNA. Meiotic recombination hotspots are regions of the genome<br>
that recombine more frequently than average during meiosis.<br>
Sequence identity: The similarity between two nucleic acid sequences, or two amino acid<br>
sequences, is expressed in terms of the similarity between the sequences, otherwise referred to as<br>
sequence identity. Sequence identity is frequently measured in terms of percentage identity (or<br>
similarity or homology); the higher the percentage, the more similar the two sequences are.<br>
Methods of alignment of sequences for comparison are well known in the art. Various<br>
programs and alignment algorithms are described in: Smith and Waterman (Adv. Appl. Math., 2:482,<br>
1981); Needleman and Wunsch (J. Mol. Bio!., 48:443, 1970); Pearson and Lipman (Proc. Natl. Acad.<br>
Sci., 85:2444, 1988); Higgins and Sharp (Gene, 73:237-44, 1988); Higgins and Sharp (CABIOS,<br>
5:151-53, 1989); Corpet et al. (Nuc. Acids Res., 16:10881-90, 1988); Huang et al. (Comp. Appls.<br>
Biosci,, 8:155-65, 1992); and Pearson et al. (Meth. Mol. Biol, 24:307-31, 1994). Altschul et al.<br>
(Nature Genet., 6:119-29, 1994) presents a detailed consideration of sequence alignment methods and<br>
homology calculations.<br>
The alignment tools ALIGN (Myers and Miller, CABIOS 4:11-17, 1989) or LFASTA<br>
(Pearson and Lipman, 1988) may be used to perform sequence comparisons (Internet Program ©<br>
1996, W. R. Pearson and the University of Virginia, "fasta20u63" version 2.0u63, release date<br>
December 1996). ALIGN compares entire sequences against one another, while LFASTA compares<br>
regions of local similarity. These alignment tools and their respective tutorials are available on the<br>
Internet at the NCSA website. Alternatively, for comparisons of amino acid sequences of greater than<br>
about 30 amino acids, the "Blast 2 sequences" function can be employed using the default<br>
BLOSUM62 matrix set to default parameters, (gap existence cost of 11, and a per residue gap cost of<br>
1). When aligning short peptides (fewer than around 30 amino acids), the alignment should be<br>
performed using the "Blast 2 sequences" function, employing the PAM30 matrix set to default<br>
parameters (open gap 9, extension gap 1 penalties). The BLAST sequence comparison system is<br>
available, for instance, from the NCBI web site; see also Altschul et al.,J. Mol. Biol., 215:403-10,<br>
1990; Gish. and States, Nature Genet., 3:266-72, 1993; Madden et al., Meth. Enzymol., 266:131-41,<br>
1996; Altschul et al., Nucleic Acids Res., 25:3389-402, 1997; and Zhang and Madden, Genome Res.,<br>
7:649-56, 1997.<br>
Orthologs (equivalent to proteins of other species) of proteins are in some instances<br>
characterized by possession of greater than 75% sequence identity counted over the full-length<br>
alignment with the amino acid sequence of specific protein using ALIGN set to default parameters.<br>
Proteins with even greater similarity to a reference sequence will show increasing percentage<br>
identities when assessed by this method, such as at least 80%, at least 85%, at least 90%, at least 92%,<br>
at least 95%, or at least 98% sequence identity. In addition, sequence identity can be compared over<br>
the full length of one or both binding domains of the disclosed fusion proteins.<br>
When significantly less than the entire sequence is being compared for sequence identity,<br>
homologous sequences will typically possess at least 80% sequence identity over short windows of<br>
10-20, and may possess sequence identities of at least 85%, at least 90%, at least 95%, or at least 99%<br>
depending on their similarity to the reference sequence. Sequence identity over such short windows<br>
can be determined using LFASTA; methods are described at the NCSA website. One of skill in the<br>
art will appreciate that these sequence identity ranges are provided for guidance only; it is entirely<br>
possible that strongly significant homologs could be obtained that fall outside of the ranges provided.<br>
Similar homology concepts apply for nucleic acids as are described for protein. An alternative<br>
indication that two nucleic acid molecules are closely related is that the two molecules hybridize to<br>
each other under stringent conditions.<br>
Nucleic acid sequences that do not show a high degree of identity may nevertheless encode<br>
similar amino acid sequences, due to the degeneracy of the genetic code. It is understood that<br>
changes in nucleic acid sequence can be made using this degeneracy to produce multiple nucleic acidsequences that each encode substantially the same protein.Specific binding agent: An agent that binds bstantially only to a defined target. Thus aprotein-specific binding agent binds substantially only the defined protein, or to a specific region<br>
within the protein. As used herein, protein-specific binding agents include antibodies and other<br>
agents that bind substantially to a specified polypeptide. The antibodies may be monoclonal or<br>
polyclonal antibodies that are specific for the polypeptide, as well as immunologically effective<br>
portions ("fragments") thereof.<br>
The determination that a particular agent binds substantially only to a specific polypeptide<br>
may readily be made by using or adapting routine procedures. Examples of suitable in vitro assays<br>
which make use of the Western blotting procedure include IFA and Ag-ELISA, and are described in<br>
many standard texts, including Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL,<br>
New York, 1999.<br>
Transformed: A "transformed" cell is a cell into which has been introduced a nucleic acid<br>
molecule by molecular biology techniques. The term encompasses all techniques by which a nucleic<br>
acid molecule might be introduced into such a cell, including transfection with viral vectors,<br>
transformation with plasmid vectors, and introduction of naked DNA by electroporation, lipofection,<br>
and particle gun acceleration.<br>
Vector: A nucleic acid molecule as introduced into a host cell, thereby producing a<br>
transformed host cell. A vector may include nucleic acid sequences that permit it to replicate in a host<br>
cell, such as an origin of replication. A vector may also include one or more selectable marker genes<br>
and other genetic elements known in the art.<br>
///. Overview of Several Embodiments<br>
Isolated mutant flavivirus polypeptides exhibiting measurably reduced antibody crossreactivity<br>
(compared to corresponding wild-type polypeptides) are disclosed herein. In one<br>
embodiment, the isolated flavivirus polypeptides are flavivirus E-glycoproteins that include an amino<br>
acid sequence as shown in SEQ ID NO: 14, wherein at least one of the amino acids at position 104,<br>
106, 107, 126, 226, or 231 is substituted (compared to corresponding wild-type E-glycoproteins).<br>
Specific, non-limiting examples of the amino acid substitutions at positions 104, 106, 107, 126,226,<br>
and 231 include: G104H (SEQ ID NO: 16), G106Q (SEQ ID NO: 18), L107K (SEQ ID NO: 20), E126A<br>
(SEQ ID NO: 22), T226N (SEQ ID NO: 24), W23,F (SEQ ID NO: 26), and W231L (SEQ ID NO: 28).<br>
Also disclosed are isolated nucleic acid molecules encoding the flavivirus polypeptides with at least<br>
one amino acid substitution at position 104, 106, 107, 126, 226, or 231 of SEQ ID NO: 14.<br>
Representative nucleic acid molecules are shown in SEQ ID NOs: 15, 17, 19, 21, 23, 25, and 27.<br>
In another embodiment, the isolated flavivirus polypeptides are flavivirus E-glycoproteins<br>
that include an amino acid sequence as shown in SEQ ID NO: 81, wherein at least one of the amino<br>
acids at position 106 is substituted (compared to corresponding wild-type E-glycoproteins). Specific,<br>
non-limiting examples of the amino acid substitutions at position 106 include: G^Q (SEQ ID NO:<br>
83). Also disclosed are isolated nucleic acid molecules encoding the flavivirus polypeptides with at<br>
least one amino acid substitution at position 106 of SEQ ID NO: 81. A representative nucleic acid<br>
molecule is shown in SEQ ID NO: 82.<br>
In yet another embodiment, the isolated flavivirus polypeptides are flavivirus Eglycoproteins<br>
that include an amino acid sequence as shown in SEQ ID NO: 85, wherein at least one<br>
of the amino acids at position 106 is substituted (compared to corresponding wild-type Eglycoproteins).<br>
Specific, non-limiting examples of the amino acid substitutions at position 106<br>
include: GioeV (SEQ ID NO: 87). Also disclosed are isolated nucleic acid molecules encoding the<br>
flavivirus polypeptides with at least one amino acid substitution at position 106 of SEQ ID NO: 85. A<br>
representative nucleic acid molecule is shown in SEQ ID NO: 86.<br>
Pharmaceutical and immune stimulatory compositions are also disclosed that include one or<br>
more flavivirus E-glycoprotein polypeptides exhibiting measurably reduced antibody cross-reactivity,<br>
with at least one amino acid substitution at position 104, 106, 107, 126, 226, or 231 of SEQ ID NO:<br>
14. Also disclosed are pharmaceutical and immune stimulatory compositions that include one or<br>
more nucleic acid molecules encoding the flavivirus polypeptides with at least one amino acid<br>
substitution at position 104, 106, 107, 126, 226, or 231 of SEQ ID NO: 14. Representative nucleic<br>
acid molecules are shown in SEQ ID NOs: 15, 17, 19, 21, 23, 25, and 27.<br>
Also disclosed are pharmaceutical and immune stimulatory compositions that include one or<br>
more flavivirus E-glycoprotein polypeptides exhibiting measurably reduced antibody cross-reactivity,<br>
with at least one amino acid substitution at position 106 of SEQ ID NO: 81 or SEQ ID NO: 85. Also<br>
disclosed are pharmaceutical and immune stimulatory compositions that include one or more nucleic<br>
acid molecules encoding the flavivirus polypeptides with at least one ammo acid substitution at<br>
position 106 of SEQ ID NO: 81 or SEQ ID NO: 85. Representative nucleic acid molecules are shown<br>
in SEQ ID NOs: 82 and 86.<br>
In another embodiment, a method is provided for identifying and modifying a flavivirus<br>
cross-reactive epitope. This method includes selecting a candidate cross-reactive epitope using a<br>
structure-based design approach, and designing a substituted epitope including at least one amino acid<br>
residue substitution compared to the candidate epitope. The candidate epitope is then contacted with<br>
a specific binding agent under conditions whereby a candidate epitope/specific binding agent complex<br>
can form. Likewise, the substituted epitope is contacted with the same specific binding agent under<br>
the same conditions used for candidate epitope/specific binding agent complex formation. A<br>
candidate epitope is identified as a flavivirus cross-reactive epitope when the substituted epitope has a<br>
substantially lower binding affinity for the specific binding agent compared to the candidate epitope,<br>
and wherein the flavivirus cross-reactive epitope binds to a specific binding agent that binds to at least<br>
two flaviviruses. In specific, non-limiting examples, the at least two flaviviruses are selected from<br>
dengue serotype 1 virus, dengue serotype 2 virus, dengue serotype 3 virus, dengue serotype 4 virus,<br>
yellow fever virus, Japanese encephalitis virus, St. Louis encephalitis virus, and West Nile virus. In<br>
yet another specific example of the provided method, the specific binding agent is a flavivirus crossreactive<br>
antibody.<br>
In a further specific example of the provided method, the structure-based design approach<br>
includes identifying at least one conserved flavivirus amino acid between two or more flavivirus<br>
groups or subgroups, and mapping the conserved flavivirus amino acid onto a structure of a flavivirus<br>
E-glycoprotein. In another specific example, the conserved flavivirus amino acid exhibits two or<br>
more of the following structural characteristics: it is located in DII of the E-glycoprotein, it is<br>
conserved across the flaviviruses, it is on the outer or lateral surface of the E-glycoprotein dimer, it<br>
has at least 35% surface accessibility potential, its side chain projection is accessible for antibody<br>
paratopes, or it has a high p-factor.<br>
18<br>
In yet a further specific example of the provided method, the structure-based design<br>
approach includes identifying at least one conserved flavivirus amino acid between two or more<br>
flavivirus complexes or subcomplexes, and mapping the conserved flavivirus amino acid onto a<br>
structure of a flavivirus E-glycoprotein. In still another specific example, the conserved flavivirus<br>
amino acid exhibits two or more of the following structural characteristics: it has at least 35% surface<br>
accessibility potential, it is on the outer or lateral surface of the E-glycoprotein dimer, it is conserved<br>
across the flaviviruses, its side chain projection is accessible for antibody paratopes, or it has a high Pfactor.<br>
In another embodiment, a method is provided for detecting a flavivirus antibody in a sample.<br>
This method includes contacting the sample with the disclosed mutant flavivirus polypeptides under<br>
conditions whereby a polypeptide/antibody complex can form, and detecting polypeptide/antibody<br>
complex formation, thereby detecting a flavivirus antibody in a sample. Also disclosed are methods<br>
of diagnosing a flavivirus infection in a subject. In one embodiment, the method includes contacting<br>
a sample from the subject with the disclosed mutant flavivirus polypeptides under conditions whereby<br>
a polypeptide/antibody complex can form, and detecting polypeptide/antibody complex formation,<br>
thereby diagnosing a flavivirus infection in a subject.<br>
Also disclosed is a flavivirus E-glycoprotein engineered to comprise at least one amino acid<br>
residue substitution according to the methods described herein.<br>
IV. Identifying Flavivirus Cross-Reactive Epitopes<br>
The current disclosure provides methods for identifying flavivirus cross-reactive epitopes, as<br>
well as distinguishing such epitopes from species-specific (or type-specific) epitopes.<br>
In one embodiment, the method comprises a structure-based design approach, which<br>
optionally includes one or more of the following requirements in order to identify cross-reactive<br>
epitopes: 1) the epitope is located in DII of the E-glycoprotein, for example, amino acids 52-135 and<br>
195-285 in the THE virus E-glycoprotein, 52-132 and 193-280 in the DEN-2 virus E-glycoprotein,<br>
and conserved across the flaviviruses or multiple flaviviral species; 2) the epitope is on the outer or<br>
lateral surface of the E-glycoprotein dimer; 3) the epitope has at least 35% surface accessibility<br>
potential; 4) one or more side chain projections of amino acids within the epitope are accessible to<br>
antibody paratopes; and 5) residues with high temperature (P) factors are favored.<br>
In one embodiment, a structure-based design approach comprises a procedural algorithm<br>
developed to localize epitopes responsible for inducing flavivirus cross-reactive antibodies. Strictlyconserved<br>
flavivirus residues are initially identified. These residues are mapped, for example, onto a<br>
2.0 A resolution E-glycoprotein structure for TBE virus (Rey et al., Nature 375:291-98, 1995), a high<br>
resolution DEN-2 virus E-glycoprotein structure (Modis et al.,PNAS 100:6986-91, 2003), or other<br>
similar structure. Optionally, strictly-conserved flavivirus residues are also mapped onto a computer<br>
predicted homology model structure for the DEN-2 virus E-glycoprotein using, for example, the<br>
Swiss-Pdb Viewer 3.7 structure analysis software (Guex et al., Electrophoresis 18:2714-23, 1997).<br>
The following criteria (individually or in combination of two or more) are then employed in<br>
certain embodiments to select probable flavivirus group or subgroup cross-reactive epitope residues:<br>
1) an amino acid located in DII (for example, amino acids 52-135 and 195-285 in the TBE virus Eglycoprotein<br>
(Rey et al, Nature 375:291-98, 1995); 52-132 and 193-280 in the DEN-2 virus Eglycoprotein<br>
(Modis et al., PNAS 100:6986-91, 2003)), and conserved among more than one<br>
flavivirus; 2) amino acids on the outer or lateral surface of the E-glycoprotein dimer; 3) amino acids<br>
with at least 35% surface accessibility potential; 4) side chain projections accessible to antibody<br>
paratopes; and 5) residues with high temperature (P) factors should be favored, as these residues tend<br>
to be flexible and are able to conform to the antibody paratope, increasing the antibody-antigen (Ab-<br>
Ag) affinity.<br>
Similar criteria (individually or in combination of two or more) are employed in certain<br>
embodiments to select probable flavivirus complex or subcomplex cross-reactive epitope residues.<br>
The procedural algorithm for the identification of flavivirus complex and sub-complex cross-reactive<br>
epitopes utilizes the following optimaliry criteria: 1) The identification and selection of amino acid<br>
residues with &gt;35% of their surface solvent accessible. These residues are identified from the<br>
published atomic structure coordinates of the DENV-2 soluble ectodomain of the envelope<br>
glycoprotein and homology models of SLEV and WNV derived from the DENV-2 structure (Modis<br>
et al, Proc. Natl. Acad. Sci. USA 100:6986-91, 2003). In addition to examination of amino acid<br>
residues in structural domain II, residues in domains I and III were examined, since published results<br>
indicate that some complex and sub-complex cross-reactive epitopes are mapped onto domains I and<br>
III in addition to domain II (Roehrig et al., Virology 246:317-28, 1998). 2) Amino acids on the outer<br>
or lateral surface of the E-glycoprotein dimer, and accessible to antibody. 3) Amino acid<br>
conservation across the flavivirus complex (based upon a structural alignment of the protein<br>
sequences). Residues conserved across all member viruses of the same complex are favored. If<br>
conserved within but not across the entire complex, then residues with shared identities between<br>
WNV and SLEV are favored in the JEV complex, and residues with shared identities between<br>
DENV-2 and two or more other viruses in the DENV complex are favored over those shared with<br>
DENV-2 and only one other DENV complex virus. 4) Side chain projections exposed towards the<br>
outer surface and accessible to antibody paratopes. 5) Residues with high temperature (P-) factors<br>
should be favored, as these residues tend to be flexible and are able to conform to the antibody<br>
paratope, increasing the antibody-antigen affinity. Amino acid residues with high temperature factors<br>
are more commonly found in antigen epitopes than lower temperature factor residues. 6) Following<br>
identification of potential individual flavivirus complex and sub-complex cross-reactive epitope<br>
residues, all residues are mapped and highlighted on the same E-glycoprotein dimer structure<br>
together. With this technique, groups of potential cross-reactive epitope residues forming clusters<br>
(and hence probable epitopes) are readily identified. 7) Residues fitting all of these criteria and<br>
occurring in structural clusters approximately 20 x 30 A2 (which is the average "footprint" of an<br>
antibody Fab that interacts with an antigen epitope) are favored over residues that are more isolated<br>
in the protein structure. 8) Within an identified structural cluster of potential epitope residues,<br>
residues that more completely satisfy greater numbers of the optimality criteria are selected for the<br>
first round of mutagenesis analysis.<br>
A. Outer and/or Lateral Surface Amino Acids<br>
In one embodiment, the outer and/or lateral surface of the E-glycoprotein dimer comprises<br>
those residues which are exposed on the surface of the E-glycoprotein dimer in a way that they are<br>
physically capable of interacting with a host-derived immunoglobulin antibody molecule. The<br>
flavivirus virion contains a host cell-derived lipid bilayer, with E-glycoprotein dimers imbedded<br>
within this lipid bilayer via their trans-membrane domains. The ectodomains of the E-glycoprotein<br>
dimers lie on top of this bilayer, forming a dense lattice and essentially coating the virion in a protein<br>
shell. Because of this structural organization, there are regions of the E-glycoprotein that, under<br>
general assembled virion conditions, cannot physically interact with an immunoglobulin molecule,<br>
and therefore are highly unlikely to form part of an antibody epitope. Such inaccessible regions<br>
include the trans-membrane domains (because they are imbedded within the lipid bilayer and are<br>
covered by the ectodomain) and more than two-thirds of the residues of the ectodomain itself, which<br>
are either on the bottom surface of the dimer (and therefore packed between the lipid layer and the<br>
ectodomain), or are packed into the interior of this globular protein rather than on its surface.<br>
Because of these structural constraints, under normal conditions immunoglobulin molecules can only<br>
interact with residues on the outer exposed surface of the E-glycoprotein dimer, and with a subset of<br>
residues on the outer lateral surface. Because of the close packing of E-glycoprotein dimers into a<br>
network across the surface of the virion, and the difficulty of a large immunoglobulin molecule<br>
accessing these narrow spaces, it is believed that only some of the lateral surface residues are<br>
available for immunoglobulin interaction. For these reasons, only residues located on the outer or<br>
lateral surface of the E-glycoprotein are considered as participating in possible flavivirus crossreactive<br>
epitopes. An inspection of the location of a residue (e.g., a residue conserved among more<br>
than one flavivirus, such as Gly104, Gly]06, LeU|07, or Trp23i) in the E-glycoprotein dimer atomic<br>
structure allows for a determination as to whether or not a residue is located on the outer or lateral<br>
surface of the dimer.<br>
B. Surface Accessibility Potential<br>
In one embodiment, surface accessibility potential comprises that portion of the predicted<br>
electron density surrounding any amino acid residue's side chain that is exposed on the surface of the<br>
protein, and theoretically available to interact with another molecule. For any given "surface"<br>
residue, its surface accessibility is affected by the local (and surrounding) secondary structure of the<br>
alpha-carbon main chain, and the positions and types of immediately surrounding side-chain<br>
projections. Thus, by definition, maximum accessibility would be for a residue X in the peptide<br>
GGXGG in an extended conformation, as the glycine residues have no side chains and therefore<br>
amino acid X's surface accessibility is not constrained by either the alpha-carbon backbone shape or<br>
the surrounding residues' side chain projections (see, e.g., Li et al, Nature Struct. Bio. 10:482-88,<br>
2003; and Faelber et al.,J. Mol. Biol. 313:83-97, 2001).<br>
C. Accessible Side Chain Projections<br>
In one embodiment, the side chain projection(s) accessible for antibody paratopes comprises<br>
a qualitative assessment of how exposed and/or available a given amino acid's reactive side chain is<br>
to interact with a hypothetical immunoglobulin molecule. The angle of projection of a side chain is<br>
determined primarily by its position in the primary amino acid chain. However, upon folding of this<br>
polypeptide chain, the side chain projections are additionally altered or affected by electrostatic and<br>
other forces from surrounding residues. The accessibility of an amino acid's side chain projections to<br>
be bound by antibody is a specific criterion that is inherent in an amino acid's "surface accessibility."<br>
Hence, theoretical amino acid X could have 50% surface accessibility and yet its side-chain may still<br>
be directed towards the interior of the protein and therefore be unlikely to interact energetically with<br>
an immunoglobulin molecule (see, e.g., Li et al., Nature Struct. Bio. 10:482-88, 2003; Faelber et al.,<br>
J. Mol. Biol. 313:83-97, 2001; and Eyal et al.,J. Comp. Chem. 25: 712-24, 2003).<br>
D. High Temperature Factors<br>
In one embodiment, a temperature or p-factor comprises a criterion which represents a<br>
particular amino acid's potential flexibility within the protein. Any given atom within a protein<br>
structure is defined by four parameters, the three x, y and z coordinates, defining its position in space,<br>
and its (5- or temperature factor. For well defined, high-resolution crystal structures, p-values are<br>
typically 40 A2 can be a signal that there is little confidence in<br>
the assignment of these atoms within the protein (for example, if the protein is disordered and does<br>
not consistently fold into the same structure). However, in well-defined atomic-level resolution<br>
protein structures, high p-factors associated with particular atoms for individual amino acids are<br>
typically interpreted as indicators of that residue or atom's potential flexibility. This criterion is<br>
relevant to epitope determination, as shape complementarity of the molecular surfaces of both the<br>
antibody paratope and the antigen epitope is know to be an important factor effecting antibody<br>
avidity. Flexible residues, identified by their higher p-factors, are better able to make slight<br>
positional adjustments, thereby improving shape complementarity and the energetics of the Ag-Ab<br>
interaction. It has been demonstrated that epitope amino acids involved in antibody interactions are<br>
more likely to have high p-factors than are amino acids from the same protein that do not interact<br>
with antibodies (see, e.g., Mylvaganam et al.,J. Mol. Biol. 281:301-22, 1998).<br>
Amino acid substitutions at probable cross-reactive epitope residues are modeled, selecting<br>
substitutions that should reduce or ablate antibody recognition without altering E-glycoprotein<br>
structural conformation, disrupting dimer interactions, or impairing particle formation, maturation, or<br>
secretion. For this reason, cysteine residues otherwise satisfying the cross-reactive epitope criteria<br>
are not recommended for mutagenesis because their involvement in disulphide bridging is believed to<br>
be necessary for proper E-glycoprotein structure and function (Modis et al, PNAS 100:6986-91,<br>
2003; Key et al., Nature 375:291-98, 1995). Stability calculations are performed for all possible<br>
amino acid substitutions of candidate residues using, for example, the FOLD-X server (Guerois et ai,<br>
J. Mol. Biol. 320:369-87, 2002; available on the internet) and the TBE virus E-glycoprotein pdb file<br>
coordinates (Rey et al., Nature 375:291-98, 1995). By way of example, amino acid substitutions<br>
modeled in the TBE virus E-glycoprotein with free energies of folding equal to or less than that of the<br>
non-mutated wild-type E-glycoprotein are re-examined with the Swiss-PdbViewer software, to<br>
identify those substitutions that minimized local structural disturbances while maintaining structurally<br>
relevant biochemical interactions such as hydrogen bonding and/or charge interactions with<br>
neighboring amino acids.<br>
Optionally, upon the successful identification of cross-reactive epitope residues, the Eglycoprotein<br>
structure can be further analyzed to identify additional residues forming cross-reactive<br>
epitopes. By way of example, a "nearest neighbor" search is conducted of the surface of the Eglycoprotein<br>
structure, looking for additional residues located within 10-15 A of the identified<br>
residue. This distance is within the binding footprint of a single antibody paratope (Faebler et al.,J.<br>
Mol. Biol. 313:83-97, 2001). In this second iteration of cross-reactive epitope residue identification,<br>
the same five optimality criterion as above are used, with one change. The criterion of strict<br>
conservation across the flaviviruses is relaxed to now include variable residues. In this way, residues<br>
either conserved in their physiochemical nature and/or conserved only within a particular flavivirus<br>
complex (such as the four DEN serotypes) or subgroup can be identified.<br>
Also provided are methods for designing a substituted epitope comprising at least one amino<br>
acid residue substitution compared to a wild-type candidate epitope; obtaining a first sample<br>
comprising the candidate epitope; obtaining a second sample comprising the substituted epitope;<br>
contacting the first sample with a specific binding agent; and contacting the second sample with the<br>
specific binding agent, wherein the cross-reactive epitope is identified when the substituted epitope<br>
has a substantially lower binding affinity for the specific binding agent compared to the candidate<br>
epitope. Antibody binding affinities can be determined by many methods well known in the art, such<br>
as end-point titration in an Ag-ELISA assay, competition binding in an ELISA assay, a solid-phase<br>
radioimmunoassay, and the Biacore® surface plasmon resonance technique (Malmqvist, Biochem.<br>
Soc. Trans. 27:335-40, 1999; and Drake et al.,Anal. Biochem. 328:35-43, 2004).<br>
In some embodiments the specific binding agent is an antibody, for example, a polyclonal<br>
antibody or a mAb. A specific, non-limiting example of a polyclonal antibody is polyclonal anti-<br>
DEN-2 MHIAF. Specific, non-limiting examples of mAbs include 4G2 (ATCC No. HB-112), 6B6C-<br>
1, 1B7-5, 10A1D-2, 1A5D-1, and 1B4C-2 (Roehrig et al, Virology 246:317-28, 1998).<br>
V. Flavivirus Cross-Reactive Epitopes and Variants Thereof<br>
The disclosure also provides an isolated polypeptide comprising at least one flavivirus<br>
cross-reactive epitope residue, wherein the antibody cross-reactivity of the at least one flavivirus<br>
cross-reactive epitope has been reduced or ablated. In one embodiment, one or more amino acid<br>
substitutions of one or more flavivirus cross-reactive epitope residues causes the reduction or ablation<br>
of antibody cross-reactivity. In another embodiment, the at least one flavivirus cross-reactive epitope<br>
residue with reduced or ablated cross-reactivity has measurably lower binding affinity with one or<br>
more flavivirus group-reactive mAbs, due to substitution of the flavivirus cross-reactive epitope<br>
residue(s), but its binding with one or more DEN-2 virus type-specific mAbs is not affected.<br>
Specific, non-limiting examples of an isolated polypeptide comprising at least one flavivirus<br>
cross-reactive epitope residue with reduced or ablated cross-reactivity include, the amino acid<br>
sequences shown in SEQ ID NO: 16 (G104H), SEQ ID NO: 18 (GmQ), SEQ ID NO: 20 (L,07K), SEQ<br>
ID NO: 22 (E126A), SEQ ID NO: 24 (T226N), SEQ ID NO: 26 (W23iF), SEQ ID NO: 28 (W231L), SEQ<br>
ID NO: 30 (E,26A/ T226N), SEQ ID NO: 83, and SEQ ID NO: 87.<br>
Manipulation of the nucleotide sequence of a flavivirus cross-reactive epitope using standard<br>
procedures, including for instance site-directed mutagenesis or PCR and M13 primer mutagenesis, can<br>
be used to produce variants with reduced or ablated cross-reactivity. Details of these techniques are<br>
provided in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold<br>
Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. The simplest modifications involve<br>
the substitution of one or more amino acids for amino acids having similar physiochemical and/or<br>
structural properties. These so-called conservative substitutions are likely to have minimal impact on<br>
the activity and/or structure of the resultant protein. Examples of conservative substitutions are shown<br>
below.<br>
Conservative substitutions generally maintain (a) the structure of the polypeptide backbone<br>
in the area of the substitution, for example, as a sheet or helical conformation, (b) the charge or<br>
hydrophobicity of the molecule at the target site, or (c) the bulk of the side chain. Substitutions that<br>
should reduce or ablate antibody recognition without altering E-glycoprotein structural conformation,<br>
disrupting dimer interactions, or impairing particle formation, maturation, or secretion include: Gly to<br>
His, Gly to Gin, Leu to Lys, Glu to Ala, Thr to Asn, Trp to Phe, and Trp to Leu.<br>
The substitutions which in general are expected to produce the greatest changes in protein<br>
properties will be non-conservative, for instance changes in which (a) a hydrophilic residue, for<br>
example, seryl or threonyl, is substituted for (or by) a hydrophobic residue, for example, leucyl,<br>
isoleucyl, phenylalanyl, valyl or alanyl; (b) a cysteine or proline is substituted for (or by) any other<br>
residue; (c) a residue having an electropositive side chain, for example, lysyl, arginyl, or histadyl, is<br>
substituted for (or by) an electronegative residue, for example, glutamyl or aspartyl; or (d) a residue<br>
having a bulky side chain, for example, phenylalanine, is substituted for (or by) one not having a side<br>
chain, for example, glycine.<br>
The disclosure also provides isolated nucleic acids that encode the described polypeptldes.<br>
Nucleic acids of the invention thus include nucleic acids that encode: 1) polypeptides comprising at<br>
least one flavivirus cross-reactive epitope with reduced or ablated cross-reactivity; and 2)<br>
polypeptides that that are at least 95% identical to the polypeptides comprising at least one flavivirus<br>
cross-reactive epitope with reduced or ablated cross-reactivity.<br>
Recombinant nucleic acids may, for instance, contain all or part of a disclosed nucleic acid<br>
operably linked to a regulatory sequence or element, such as a promoter, for instance, as part of a<br>
clone designed to express a protein. Cloning and expression systems are commercially available for<br>
such purposes and are well known in the art. The disclosure also provides cells or organisms<br>
transformed with recombinant nucleic acid constructs that encode all or part of the described<br>
polypeptides. Also disclosed are virus-like particles (VLPs) that include one or more of the described<br>
flavivirus E-glycoprotein polypeptides.<br>
VI. Specific Binding Agents<br>
This disclosure provides specific binding agents that bind to polypeptides disclosed herein,<br>
e.g., flavivirus E-glycoprotein polypeptides with reduced or ablated cross-reactivity. The binding<br>
agent may be useful for identifying flavivirus cross-reactive epitopes, and for detecting and purifying<br>
polypeptides comprising flavivirus cross-reactive epitopes. Examples of the binding agents are a<br>
polyclonal or monoclonal antibody, and fragments thereof, that bind to polypeptides disclosed herein.<br>
A specific, non-limiting example of a polyclonal antibody is polyclonal anti-DEN-2 MHIAF.<br>
Specific, non-limiting examples of mAbs include 4G2, 6B6C-1, 1B7-5, 10A1D-2, 1A5D-1, and<br>
1B4C-2.Monoclonal or polyclonal antibodies can be raised to recognize the polypeptides described<br>
herein, or variants thereof. Optimally, antibodies raised against these polypeptides will specifically<br>
detect the polypeptide with which the antibodies are generated. That is, antibodies raised against the<br>
polypeptide will recognize and bind the polypeptide, and will not substantially recognize or bind to<br>
other polypeptides or antigens. The determination that an antibody specifically binds to a target<br>
polypeptide is made by any one of a number of standard immunoassay methods; for instance, the<br>
Western blotting technique (Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed.,<br>
vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989), Ag-ELISA and IFA.<br>
Substantially pure flavivirus recombinant polypeptide antigens suitable for use as<br>
immunogens can be isolated from the transformed cells described herein, using methods well known<br>
in the art. Monoclonal or polyclonal antibodies to the antigens can then be prepared.<br>
Monoclonal antibodies to the polypeptides can be prepared from murine hybridomas<br>
according to the classic method of Kohler &amp; Milstein (Nature 256:495-97, 1975), or a derivative<br>
method thereof. Briefly, a mouse is repetitively inoculated with a few micrograms of the selected<br>
protein immunogen (for example, a polypeptide comprising at least one flavivirus cross-reactive<br>
epitope with reduced or ablated cross-reactivity, a portion of a polypeptide comprising at least one<br>
flavivirus cross-reactive epitope with reduced or ablated cross-reactivity, or a synthetic peptide<br>
comprising at least one flavivirus cross-reactive epitope with reduced or ablated cross-reactivity) over<br>
a period of a few weeks. The mouse is then sacrificed, and the antibody-producing cells of the spleen<br>
isolated. The spleen cells are fused by means of polyethylene glycol with mouse myeloma cells, and<br>
the excess unfused cells destroyed by growth of the system on selective media comprising<br>
aminopterin (HAT media). The successfully fused cells are diluted and aliquots of the dilution placed<br>
in wells of a microtiter plate where growth of the culture is continued. Antibody-producing clones are<br>
identified by detection of antibody in the supernatant fluid of the wells by immunoassay procedures,<br>
such as ELISA, as originally described by Engvall (Meth. EnzymoL, 70:419-39, 1980), or a derivative<br>
method thereof. Selected positive clones can be expanded and their monoclonal antibody product<br>
harvested for use. Detailed procedures for monoclonal antibody production are described in Harlow<br>
and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
Polyclonal antiserum containing antibodies can be prepared by immunizing suitable animals<br>
with a polypeptide comprising at least one flavivirus cross-reactive epitope with reduced or ablated<br>
cross-reactivity, a portion of a polypeptide comprising at least one flavivirus cross-reactive epitope<br>
with reduced or ablated cross-reactivity, or a synthetic peptide comprising at least one flavivirus<br>
cross-reactive epitope with reduced or ablated cross-reactivity, which can be unmodified or modified,<br>
to enhance immunogenicity.<br>
Effective antibody production (whether monoclonal or polyclonal) is affected by many<br>
factors related both to the antigen and the host species. For example, small molecules tend to be less<br>
immunogenic than others and may require the use of carriers and adjuvant. Also, host animals vary in<br>
response to site of inoculations and dose, with either inadequate or excessive doses of antigen<br>
resulting in low liter antisera. Small doses (ng level) of antigen administered at multiple intradermal<br>
sites appear to be most reliable. An effective immunization protocol for rabbits can be found in<br>
Vaitukaitis et al. (J. Clin. Endocrinol. Metab., 33:988-91, 1971).<br>
Booster injections can be given at regular intervals, and antiserum harvested when the<br>
antibody titer thereof, as determined semi-quantitatively, for example, by double immunodiffusion in<br>
agar against known concentrations of the antigen, begins to fall. See, for example, Ouchterlony et al.,<br>
Handbook of Experimental Immunology, Wier, D. (ed.), Chapter 19, Blackwell, 1973. A plateau<br>
concentration of antibody is usually in the range of 0.1 to 0.2 mg/ml of serum (about 12<br>
26<br>
Affinity of the antisera for the antigen is determined by preparing competitive binding curves, as<br>
described, for example, by Fisher (Manual of Clinical Immunology, Ch. 42, 1980).<br>
Antibody fragments may be used in place of whole antibodies and may be readily expressed<br>
in prokaryotic host cells. Methods of making and using immunologically effective portions of<br>
monoclonal antibodies, also referred to as "antibody fragments," are well known and include those<br>
described in Better &amp; Horowitz, Methods Enzymol. 178:476-96, 1989; Glockshuber et ai,<br>
Biochemistry 29:1362-67, 1990; and U.S. Patent Nos. 5,648,237 (Expression of Functional Antibody<br>
Fragments); 4,946,778 (Single Polypeptide Chain Binding Molecules); and 5,455,030<br>
(Immunotherapy Using Single Chain Polypeptide Binding Molecules), and references cited therein.<br>
Conditions whereby a polypeptide/binding agent complex can form, as well as assays for the<br>
detection of the formation of a polypeptide/binding agent complex and quantitation of binding<br>
affinities of the binding agent and polypeptide, are standard in the art. Such assays can include, but<br>
are not limited to, Western blotting, immunoprecipitation, immunofluorescence,<br>
immunocytochemistry, immunohistochemistry, fluorescence activated cell sorting (FACS),<br>
fluorescence in situ hybridization (FISH), immunomagnetic assays, ELISA, ELISPOT (Coligan et at,<br>
Current Protocols in Immunology, Wiley, NY, 1,995), agglutination assays, flocculation assays, cell<br>
panning, etc., as are well known to one of skill in the art.<br>
Binding agents of this disclosure can be bound to a substrate (for example, beads, tubes,<br>
slides, plates, nitrocellulose sheets, etc.) or conjugated with a detectable moiety, or both bound and<br>
conjugated. The detectable moieties contemplated for the present disclosure can include, but are not<br>
limited to, an immunofluorescent moiety (for example, fluorescein, rhodamine), a radioactive moiety<br>
(for example, 32P, I251,35S), an enzyme moiety (for example, horseradish peroxidase, alkaline<br>
phosphatase), a colloidal gold moiety, and a biotin moiety. Such conjugation techniques are standard<br>
in the art (for example, see Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New<br>
York, 1999; Yang et al., Nature, 382:319-24, 1996).<br>
VII. Detection ofFlavivirus Antibodies<br>
The present disclosure further provides a method of detecting a flavivirus-reactive antibody<br>
in a sample, comprising contacting the sample with a polypeptide or peptide of this disclosure under<br>
condition whereby an antibody/polypeptide complex can form; and detecting formation of the<br>
complex, thereby detecting flavivirus antibody in a sample.<br>
The method of detecting flavivirus-reactive antibody in a sample can be performed, for<br>
example, by contacting a fluid or tissue sample from a subject with a polypeptide of this disclosure<br>
and detecting the binding of the polypeptide to the antibody. A fluid sample of this method can<br>
comprise any biological fluid which could contain the antibody, such as cerebrospinal fluid, blood,<br>
bile plasma, serum, saliva and urine. Other possible examples of body fluids include sputum, mucus<br>
and the like.<br>
Enzyme immunoassays such as IFA, ELISA and immunoblotting can be readily adapted to<br>
accomplish the detection of flavivirus antibodies according to the methods of this disclosure. An<br>
ELISA method effective for the detection of the antibodies can, for example, be as follows: 1) bind<br>
the polypeptide to a substrate; 2) contact the bound polypeptide with a fluid or tissue sample<br>
containing the antibody; 3) contact the above with a secondary antibody bound to a detectable moiety<br>
which is reactive with the bound antibody (for example, horseradish peroxidase enzyme or alkaline<br>
phosphatase enzyme); 4) contact the above with the substrate for the enzyme; 5) contact the above<br>
with a color reagent; and 6) observe/measure color change or development.<br>
Another immunologic technique that can be useful in the detection of flavivirus antibodies<br>
uses mAbs for detection of antibodies specifically reactive with flavivirus polypeptides in a<br>
competitive inhibition assay. Briefly, a sample is contacted with a polypeptide of this invention<br>
which is bound to a substrate (for example, a 96-well plate). Excess sample is thoroughly washed<br>
away. A labeled (for example, enzyme-linked, fluorescent, radioactive, etc.) mAb is then contacted<br>
with any previously formed polypeptide-antibody complexes and the amount of mAb binding is<br>
measured. The amount of inhibition of mAb binding is measured relative to a control (no antibody),<br>
allowing for detection and measurement of antibody in the sample. The degree of mAb binding<br>
inhibition can be a very specific assay for detecting a particular flavivirus variety or strain, when<br>
based on mAb binding specificity for a particular variety or strain of flavivirus. mAbs can also be<br>
used for direct detection of flavivirus in cells by, for example, IFA according to standard methods.<br>
As a further example, a micro-agglutination test can be used to detect the presence of<br>
flavivirus antibodies in a sample. Briefly, latex beads, red blood cells or other agglutinable particles<br>
are coated with a polypeptide of this disclosure and mixed with a sample, such that antibodies in the<br>
sample that are specifically reactive with the antigen crosslink with the antigen, causing agglutination.<br>
The agglutinated polypeptide-antibody complexes form a precipitate, visible with the naked eye or<br>
measurable by spectrophotometer.<br>
In yet another example, a microsphere-based immunoassay can be used to detect the<br>
presence of flavivirus antibodies in a sample. Briefly, microsphere beads are coated with a<br>
polypeptide of this disclosure and mixed with a sample, such that antibodies in the sample that are<br>
specifically reactive with the antigen bind the antigen. The bead-bound polypeptide-antibody<br>
complexes are allowed to react with fluorescent-dye labeled anti-species antibody (such as FITClabeled<br>
goat anti-human IgM), and are measured, using a microsphere reader (such as a Luminex<br>
instrument).<br>
The present disclosure further provides a method of diagnosing a flavivirus infection in a<br>
subject, comprising contacting a sample from the subject with the polypeptide of this disclosure<br>
under conditions whereby an antibody/polypeptide complex can form; and detecting<br>
antibody/polypeptide complex formation, thereby diagnosing a flavivirus infection in a subject.<br>
In examples of the diagnostic methods, the polypeptide of this disclosure can be bound to a<br>
substrate and contacted with a fluid sample such as blood, serum, urine or saliva. This sample can be<br>
taken directly from the patient or in a partially purified form. In this manner, antibodies specific for<br>
the polypeptide (the primary antibody) will specifically react with the bound polypeptide. Thereafter,<br>
a secondary antibody bound to, or labeled with, a detectable moiety can be added to enhance the<br>
detection of the primary antibody. Generally, the secondary antibody will be selected for its ability to<br>
react with multiple sites on the primary antibody. Thus, for example, several molecules of the<br>
secondary antibodies can react with each primary antibody, making the primary antibody more<br>
detectable.<br>
The detectable moiety allows for visual detection of a precipitate or a color change, visual<br>
detection by microscopy, or automated detection by spectrometry, radiometric measurement or the<br>
like. Examples of detectable moieties include fluorescein, rhodamine, Cy5, and Cy3 (for<br>
fluorescence microscopy and/or the microsphere-based immunoassay), horseradish peroxidase (for<br>
either light or electron microscopy and biochemical detection), biotin-streptavidin (for light or<br>
electron microscopy) and alkaline phosphatase (for biochemical detection by color change).<br>
Vlll. Pharmaceutical and Immune Stimulatory Compositions and Uses Thereof<br>
Pharmaceutical compositions including flavivirus nucleic acid sequences or flavivirus<br>
polypeptides comprising at least one flavivirus cross-reactive epitope with reduced or ablated crossreactivity<br>
are also encompassed by the present disclosure. These pharmaceutical compositions<br>
include a therapeutically effective amount of one or more active compounds, such as flavivirus<br>
polypeptides comprising at least one flavivirus cross-reactive epitope with reduced or ablated crossreactivity,<br>
or one or more nucleic acid molecules encoding these polypeptides, in conjunction with a<br>
pharmaceutically acceptable carrier. It is contemplated that in certain embodiments, flavivirus nucleic<br>
acid sequences or flavivirus polypeptides comprising multiple flavivirus cross-reactive epitopes with<br>
reduced or ablated cross-reactivity will be useful in preparing the pharmaceutical compositions of the<br>
disclosure.<br>
Disclosed herein are substances suitable for use as immune stimulatory compositions for the<br>
inhibition or treatment of a flavivirus infection, for example, a dengue virus infection. In one<br>
embodiment, an immune stimulatory composition contains a flavivirus polypeptide including at least<br>
one flavivirus cross-reactive epitope with reduced or ablated cross-reactivity. In a further<br>
embodiment, the immune stimulatory composition contains a nucleic acid vector that includes<br>
flavivirus nucleic acid molecules described herein, or that includes a nucleic acid sequence encoding<br>
at least one flavivirus cross-reactive epitope with reduced or ablated cross-reactivity. In a specific,<br>
non-limiting example, a nucleic acid sequence encoding at least one flavivirus cross-reactive epitope<br>
with reduced or ablated cross-reactivity is expressed in a transcriptional unit, such as those described<br>
in published PCT Application Nos. PCT/US99/12298 and PCT/US02/10764 (both of which are<br>
incorporated herein in their entirety).<br>
The provided immune stimulatory flavivirus polypeptides, constructs or vectors encoding<br>
such polypeptides, are combined with a pharmaceutically acceptable carrier or vehicle for<br>
administration as an immune stimulatory composition to human or animal subjects. In a particular<br>
embodiment, the immune stimulatory composition administered to a subject directs the synthesis of a<br>
mutant flavivirus E-glycoprotein as described herein, and a cell within the body of the subject, after<br>
incorporating the nucleic acid within it, secretes VLPs comprising the mutant E-glycoprotein with<br>
reduced or ablated cross-reactivity. It is believed that such VLPs then serve as an in vivo immune<br>
stimulatory composition, stimulating the immune system of the subject to generate protective<br>
immunological responses. In some embodiments, more than one immune stimulatory flavivirus<br>
polypeptide, construct or vector may be combined to form a single preparation.<br>
The immunogenic formulations may be conveniently presented in unit dosage form and<br>
prepared using conventional pharmaceutical techniques. Such techniques include the step of bringing<br>
into association the active ingredient and the pharmaceutical carrier(s) or excipient(s). In general, the<br>
formulations are prepared by uniformly and intimately bringing into association the active ingredient<br>
with liquid carriers. Formulations suitable for parenteral administration include aqueous and nonaqueous<br>
sterile injection solutions which may contain anti-oxidants, buffers, bacteriostats and solutes<br>
which render the formulation isotonic with the blood of the intended recipient; and aqueous and nonaqueous<br>
sterile suspensions which may include suspending agents and thickening agents. The<br>
formulations may be presented in unit-dose or multi-dose containers, for example, sealed ampules and<br>
vials, and may be stored in a freeze-dried (lyophilized) condition requiring only the addition of a<br>
sterile liquid carrier, for example, water for injections, immediately prior to use. Extemporaneous<br>
injection solutions and suspensions may be prepared from sterile powders, granules and tablets<br>
commonly used by one of ordinary skill in the art.<br>
In certain embodiments, unit dosage formulations are those containing a dose or unit, or an<br>
appropriate fraction thereof, of the administered ingredient. It should be understood that in addition<br>
to the ingredients particularly mentioned above, formulations encompassed herein may include other<br>
agents commonly used by one of ordinary skill in the art.<br>
The compositions provided herein, including those for use as immune stimulatory<br>
compositions, may be administered through different routes, such as oral, including buccal and<br>
sublingual, rectal, parenteral, aerosol, nasal, intramuscular, subcutaneous, intradermal, and topical.<br>
They may be administered in different forms, including but not limited to solutions, emulsions and<br>
suspensions, microspheres, particles, microparticles, nanoparticles, and liposomes.<br>
The volume of administration will vary depending on the route of administration. By way of<br>
example, intramuscular injections may range from about 0.1 ml to about 1.0 ml. Those of ordinary<br>
skill in the art will know appropriate volumes for different routes of administration.<br>
A relatively recent development in the field of immune stimulatory compounds (for example,<br>
vaccines) is the direct injection of nucleic acid molecules encoding peptide antigens (broadly<br>
described in Janeway &amp; Travers, Immunobiology: The Immune System In Health and Disease, page<br>
13.25, Garland Publishing, Inc., New York, 1997; and McDonnell &amp; Askari, N. Engl. J. Med. 334:42-<br>
45, 1996). Vectors that include nucleic acid molecules described herein, or that include a nucleic acid<br>
sequence encoding a flavivirus polypeptide comprising at least one flavivirus cross-reactive epitope<br>
with reduced or ablated cross-reactivity may be utilized in such DNA vaccination methods.<br>
Thus, the term "immune stimulatory composition" as used herein also includes nucleic acid<br>
vaccines in which a nucleic acid molecule encoding a flavivirus polypeptide comprising at least one<br>
flavivirus cross-reactive epitope with reduced or ablated cross-reactivity is administered to a subject<br>
in a pharmaceutical composition. For genetic immunization, suitable delivery methods known to<br>
those skilled in the art include direct injection of plasmid DNA into muscles (Wolff et al, Hum. Mol.<br>
Genet. 1:363, 1992), delivery of DNA complexed with specific protein carriers (Wu et al., J. Biol.<br>
Chem. 264:16985, 1989), co-precipitation of DNA with calcium phosphate (Benvenisty and Reshef,<br>
Proc. Natl. Acad. Sci. 83:9551, 1986), encapsulation of DNA in liposomes (Kaneda et al., Science<br>
243:375, 1989), particle bombardment (Tang et al, Nature 356:152, 1992; Eisenbraun et al., DNA<br>
Cell Biol. 12:791, 1993), and in vivo infection using cloned retroviral vectors (Seeger et al., Proc.<br>
Natl. Acad. Sci. 81:5849, 1984). Similarly, nucleic acid vaccine preparations can be administered via<br>
viral carrier.<br>
The amount of immunostimulatory compound in each dose of an immune stimulatory<br>
composition is selected as an amount that induces an immunostimulatory or immunoprotective<br>
response without significant, adverse side effects. Such amount will vary depending upon which<br>
specific immunogen is employed and how it is presented. Initial injections may range from about 1<br>
ug to about 1 mg, with some embodiments having a range of about 10 ug to about 800 ug, and still<br>
other embodiments a range of from about 25 ug to about 500 ug. Following an initial administration<br>
of the immune stimulatory composition, subjects may receive one or several booster administrations,<br>
adequately spaced. Booster administrations may range from about 1 ug to about 1 mg, with other<br>
embodiments having a range of about 10 ug to about ,750 ug, and still others a range of about 50 ug to<br>
about 500 ug. Periodic boosters at intervals of 1-5 years, for instance three years, may be desirable to<br>
maintain the desired levels of protective immunity.<br>
It is also contemplated that the provided immunostimulatory molecules and compositions can<br>
be administered to a subject indirectly, by first stimulating a cell in vitro, which stimulated cell is<br>
thereafter administered to the subject to elicit an immune response. Additionally, the pharmaceutical<br>
or immune stimulatory compositions or methods of treatment may be administered in combination<br>
with other therapeutic treatments.<br>
IX. KitsAlso provided herein are kits useful in the detection and/or diagnosis of flaviviruses. An<br>
example of an assay kit provided herein is a recombinant flavivirus polypeptide (or fragment thereof)<br>
as an antigen and an enzyme-conjugated anti-human antibody as a second antibody. Examples of<br>
such kits also can include one or more enzymatic substrates. Such kits can be used to test if a sample<br>
from a subject contains antibodies against a flavivirus-specific protein. In such a kit, an appropriate<br>
amount of a flavivirus polypeptide (or fragment thereof) is provided in one or more containers, or<br>
eld on a substrate. A flavivirus polypeptide can be provided in an aqueous solution or as a freezedried<br>
or lyophilized powder, for instance. The container(s) in which the flavivirus polypeptide(s) are<br>
supplied can be any conventional container that is capable of holding the supplied form, for instance,<br>
microfuge tubes, ampoules, or bottles.<br>
The amount of each polypeptide supplied in the kit can be any appropriate amount, and can<br>
depend on the market to which the product is directed. For instance, if the kit is adapted for research<br>
or clinical use, the amount of each polypeptide provided would likely be an amount sufficient for<br>
several assays. General guidelines for determining appropriate amounts can be found, for example, in<br>
Ausubel et al. (eds.), Short Protocols in Molecular Biology, John Wiley and Sons, New York, NY,<br>
1999 and Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999.<br>
The subject matter of the present disclosure is further illustrated by the following nonlimiting<br>
Example 1<br>
Identification of DII cross-reactive epitope residues<br>
This example demonstrates the identification of flavivirus cross-reactive epitopes using a<br>
structure-based rational mutagenesis method.<br>
Cell culture, virus strain and recombinant plasmid<br>
COS-1 cells (ATCC CRL 1650; Manassas, VA) were grown at 37°C with 5% CO2 on<br>
Dulbecco's modified Eagle's minimal essential medium (DMEM, GIBCO, Grand Island, NY)<br>
supplemented with 10% heat-inactivated fetal bovine serum (FBS, Hyclone Laboratories, Inc.,<br>
Logan, UT), 110 mg/1 sodium pyruvate, 0.1 mM nonessential amino acids, 2 mM L-glutamine, 20<br>
ml/1 7.5% NaHCO3, lOOU/ml penicillin, and 100 ug/ml streptomycin. CHO cells (ATCC CCL 61;<br>
Manassas, VA) were grown under the same conditions as COS-1 cells with DMEM/F12 nutrient<br>
mixture (GIBCO, Grand Island, NY).<br>
Flavivirus plasmids capable of expressing extracellular VLPs composed of prM/M and Eglycoproteins<br>
for JE, WN, SLE, and the four DEN virus serotypes have been constructed (Chang et<br>
al, J. Virol. 74:4244-52, 2000; Chang et al, Virology 306:170-80, 2003; Davis et al, J. Virol.<br>
75:4040-47, 2001). These VLPs, produced by recombinant plasmid-transformed eukaryotic cells,<br>
contain the flavivirus prM/M and E-glycoproteins in their native viral conformations, and although<br>
non-infectious, they maintain many of the same properties as mature virus particles including,<br>
hemagglutination activity, membrane fusion, and the induction of protective immune responses in<br>
animals (Chang et al.,J. Virol. 74:4244-52, 2000; Chang et al, Virology 306:170-80, 2003; Davis et<br>
al, J. Virol 75:4040-47, 2001; Hunt, et al, J. Virol. Methods 97:133-49, 2001).<br>
The recombinant expression plasmid pCB8D2-2J-2-9-l (the DEN-2 prM/E expression<br>
plasmid, Chang et al, Virology 306:170-80, 2003) was used as the template DNA for both sitedirected<br>
mutagenesis and for transient expression of DEN-2 recombinant antigen (see below). This<br>
plasmid includes the human cytomegalovirus early gene promoter, Kozak sequence, JE virus signal<br>
sequence, DEN-2 virus prM/M gene, DEN-2 virus chimeric E gene (with amino-terminal 80% from<br>
DEN-2 virus and carboxy-terminal 20% from JE virus), and bovine growth hormone poly(A) signal.<br>
The replacement of the terminal 20% of DEN-2 virus E gene sequences with JE virus E gene<br>
sequences dramatically increases the secretion of extracellular VLPs into the culture medium without<br>
altering the native DEN-2 virus E-glycoprotein conformation (Chang el al., Virology 306:170-80,<br>
2003),<br>
Procedural algorithm<br>
To localize the epitopes responsible for inducing flavivirus cross-reactive antibodies, the<br>
following procedural algorithm was developed: Strictly-conserved flavivirus residues were initially<br>
identified. These residues were mapped onto the 2.0 A resolution E-glycoprotein structure for TBE<br>
virus (Rey et al, Nature 375:291-98, 1995) and onto a computer predicted homology model structure<br>
for the DEN-2 virus E-glycoprotein using the Swiss-Pdb Viewer 3.7 structure analysis software<br>
(Guex et al., Electrophoresis 18:2714-23, 1997; available on the ExPASy Molecular Biology<br>
Server). A brief review of high resolution structures for antigen-antibody complexes revealed that<br>
10-20 residues typically are involved in making direct contacts between the antigen epitope and<br>
antibody paratope. These contacts result in 20-30 residues that are "buried" by the typical antibody<br>
footprint, measuring approximately 20 x 30 A. On average however, only 25% of the buried side<br>
chains, or 4-6 residues, account for most of the mAb binding energy (Arevalo et al, Nature 356:859-<br>
63, 1993; Bhrtet al., PNAS 91:1089-93, 1994; Davies &amp; Cohen, PNAS 93:7-12, 1996; Faebler eta!.,<br>
J. Mol Biol 313:83-97, 2001; Fleury et al, Nature St. Biol. 6:530-34, 1999; Li etal, Biochemistry<br>
39:6296-6309, 2000; Lo et al.,J. Mol. Biol. 285:2177-98, 1999; and Mylvaganam et al.,J. Mol. Biol.<br>
281:301-22, 1998).<br>
The following criteria were developed to select probable flavivirus group cross-reactive<br>
epitope residues: 1) an amino acid located in DII (for example, amino acids 52-135 and 195-285 in<br>
the TBE virus E-glycoprotein (Rey et al., Nature 375:291-98, 1995); 52-132 and 193-280 in the<br>
DEN-2 virus E-glycoprotein (Modis et al, PNAS 100:6986-91, 2003)) and conserved among more<br>
than one flavivirus; 2) amino acids on the outer or lateral surface of the E-glycoprotein dimer; 3)<br>
amino acids with at least 35% surface accessibility potential; 4) side chain projections accessible to<br>
antibody paratopes; and 5) residues with high temperature (p1-) factors should be favored, as these<br>
residues tend to be flexible and are able to conform to the antibody paratope, increasing the antibodyantigen<br>
affinity.<br>
Using this structure-based design approach, candidate flavivirus cross-reactive epitope<br>
residues were narrowed down from a total of 53 conserved amino acids in DII (38 invariant and 15<br>
almost completely conserved), to less than ten probable DII cross-reactive epitope residues. Amino<br>
acid substitutions at these probable cross-reactive epitope residues were computer modeled, selecting<br>
substitutions that should reduce or ablate antibody recognition without altering E-glycoprotein<br>
33<br>
structural conformation, disrupting dimer interactions, or impairing particle formation, maturation, or<br>
secretion. For this reason, cysteine residues otherwise satisfying the cross-reactive epitope criteria<br>
were not considered for mutagenesis because of their involvement in disulphide bridging necessary<br>
for proper E-glycoprotein structure and function (Modis et al, PNAS 100:6986-91, 2003; Rey et al.,<br>
Nature 375:291-98, 1995).<br>
Stability calculations were performed for all possible amino acid substitutions of candidate<br>
residues using the FOLD-X server (Guerois et al, J. Mol. Biol. 320:369-87, 2002; available on the<br>
internet) and the TBE virus E-glycoprotein pdb file coordinates (Rey et al, Nature 375:291-98,<br>
1995). Amino acid substitutions modeled in the TBE virus E-glycoprotein with free energies of<br>
folding equal to or less than that of the non-mutated wild-type E-glycoprotein were re-examined with<br>
the Swiss-PdbViewer software to identify those substitutions that minimized local structural<br>
disturbances while maintaining structurally relevant biochemical interactions such as hydrogen<br>
bonding and/or charge interactions with neighboring amino acids. Because the outer surface of<br>
mature flavivirus particles are covered in a dense network of E and prM/M proteins, any<br>
conformational changes in the E-glycoprotein are likely to induce concerted reorganization across the<br>
surface of the virion (Kuhn et al, Cell 108:717-25, 2002; Modis et al, PNAS 100:6986-91, 2003). A<br>
comparison of the a priori stability calculations based on the TBE virus E-glycoprotein structure with<br>
a posteriori stability calculations from the DEN-2 virus atomic structure are shown in Table 2.<br>
Site-directed mutagenesis<br>
Site-specific mutations were introduced into the DEN-2 virus E gene using the Stratagene<br>
Quick Change® multi site-directed mutagenesis kit (Stratagene, La Jolla, CA) and pCB8D2-2J-2-9-l<br>
as DNA template following the manufacturer's recommended protocols. The sequences of the<br>
mutagenic primers used for all constructs are listed in Table 1. Four or five colonies from each<br>
mutagenic PCR transformation were selected and grown in 5 ml LB broth cultures, mini-prepped and<br>
sequenced. Structural gene regions and regulatory elements of all purified plasmids were sequenced<br>
entirely upon identification of the correct mutation. Automated DNA sequencing was performed<br>
using a Beckman Coulter CEQ™ 8000 genetic analysis system (Beckman Coulter, Fullerton, CA)<br>
and analyzed using Beckman Coulter CEQ™ 8000 (Beckman Coulter, Fullerton, CA) and<br>
Lasergene® software (DNASTAR, Madison, WI).<br>
Transient expression of DEN-2 virus recombinant antigens in COS-] or CHO cells<br>
COS-1 and CHO cells were electroporated with pCB8D2-2J-2-9-l using the protocol<br>
described by Chang et al. (J. Virol 74:4244-52, 2000). Electroporated cells were recovered in 50 ml<br>
DMEM, seeded into 150 cm2 culture flasks for VLP expression and into 96 well tissue culture plates<br>
(Costar® #3603; Corning, Inc., Corning, NY) for IFA, and incubated at 37°C with 5% CO2. Six to<br>
eight hours following electroporation, the growth medium in the 150 cm2 culture flasks was replaced<br>
with DMEM containing 2% FBS. Cells in 96 well plates for IFA were fixed 14-18 hours post<br>
34<br>
electroporation. Tissue-culture medium and cells were harvested 48 and 96 hours post<br>
electroporation for antigen characterization.<br>
Characterization of mutant pCB8D2-J2-2-9-l infected cells and secreted antigen<br>
Fourteen to eighteen hours following electroporation, 96 well tissue culture plates containing<br>
cells transformed with the mutated pCB8D2-2J-2-9-l clones were washed twice with phosphate<br>
buffered saline (PBS), fixed with 3:1 acetone:PBS for 10 minutes and air dried. E-glycoproteinspecific<br>
mAbs specific for each of the three E-glycoprotein domains were used to determine affinity<br>
reductions in DII cross-reactive epitopes by indirect IFA as described by Chang et al. (J. Virol.<br>
Tissue culture medium was harvested 48 hours and 96 hours following electroporation. Cell<br>
debris was removed from tissue culture media by centrifugation for 30 minutes at 10,000 rpm. Ag-<br>
ELISA was used to detect secreted antigen from the mutagenized pCB8D2-2J-2-9-l transformed<br>
COS-1 cells. Secreted antigen was captured with polyclonal rabbit anti-DEN-2 sera (Roehrig et al,<br>
Virology 246:317-28, 1998) at a 1:10,000 dilution. Murine hyper-immune ascetic fluid (MHIAF)<br>
specific for DEN-2 virus was used at a 1:3000 dilution to detect captured antigen, and this MHIAF<br>
was detected using horseradish peroxidase conjugated goat anti-mouse HIAF at a 1:5000 dilution.<br>
Secreted antigen from tissue culture medium was concentrated by centrifugation overnight at 19,000<br>
rpm, and resuspended in TNE buffer (50mM Tris, lOOmM NaCl, lOmM EDTA, pH 7.5) to 1/200*<br>
the original volume. Concentrated antigen was analyzed with a panel of anti-DEN-2 mAbs in Ag-<br>
ELISA to determine mAb end point reactivities of the mutated antigens following the protocol of<br>
Roehrig et al. (Virology 246:317-28, 1998).<br>
Affinity reductions in DII cross-reactive epitopes<br>
Three anti-DEN-2 mAbs, 4G2, 6B6C-1 and 1B7-5, were used to determine affinity<br>
reductions in DII cross-reactive epitopes. These three mAbs share several characteristics: they<br>
recognize surface accessible epitopes in DII, they are flavivirus group- or subgroup-reactive, they are<br>
reduction-denaturation sensitive, they block virus-mediated cell-membrane fusion, they neutralize<br>
virus infectivity, and tryptic fragment mapping indicates that the binding domains of these three<br>
mAbs are formed by two discontinuous DEN-2 virus E-glycoprotein peptide fragments, aal-120 and<br>
158-400 (Aaskov et al.,Arch Virol, 105:209-21, 1989; Henchal et al, Am. J. Trap. Med. Hyg.<br>
34:162-69, 1985; Megret etal, Virology 187:480-91, 1992; Roehrig et al., Virology 246:317-28,<br>
1998). Prospective cross-reactive epitope residues were assessed by looking for decreases in the<br>
reactivity of these three DII flavivirus cross-reactive mAbs for the mutant plasmid transfected cells<br>
by IFA, and mutant VLPs in Ag-ELISA. Proper E-glycoprotein folding and structural conformation<br>
was assessed with a panel of E-glycoprotein DEN virus complex-, subcomplex-, and type-specific<br>
mAbs.<br>
Four potential flavivirus cross-reactive epitope residues were initially focused on. Single<br>
amino acid substitutions were introduced into the DEN-2 prM/E expression plasmid at the following<br>
positions (of SEQ ID NO: 14): Gly,06 to Glu (G,06Q), Trp23i to Phe (W23|F), His244 to Arg (H244R),<br>
and Lys247 to Arg (K247R) (Table 1). Substitutions at Gly,06 and Trp23, strongly interfered with the<br>
binding of flavivirus cross-reactive mAbs (Table 3), However, substitutions at His244 and Lys247 did<br>
not have a measurable effect on the binding of the cross reactive mAbs or of any other mAbs from<br>
the panel.<br>
Glyioe is located within the fusion peptide at the very tip of DII in the E-glycoprotein<br>
monomer (Allison et al., J. Virol. 73:5605-12, 1999; FIGS. 1 and 2). As with the other fusion<br>
peptide residues, Glyioe is strongly conserved across the flaviviruses, the one exception being Modoc<br>
virus with alanine at this position (Table 4). Gly106 is located at the distal end of each E-monomer<br>
along the upper and outer-lateral surface of the dimer. This residue has moderately high surface<br>
accessibility, and its relatively high temperature (p-) factor suggests its potential flexibility. The<br>
substitution of a large, bulky, polar glutamine for the glycine at this position was modeled. The<br>
glutamine substitution fit well into the surrounding region, did not appear to disrupt the local<br>
hydrogen bonding network, and produced acceptable stability calculations using the TBE virus Eglycoprotein<br>
structure coordinates (Table 2).<br>
Trp231 is located in a long intervening loop sequence between DII p-strands h and i (Modis<br>
et al, PNAS 100:6986-91, 2003; FIG. 1). Trp231 lays in a trough on the upper and outer surface of<br>
DII (FIG. 2). It is structurally close to the glycan on Asns7, and lies laterally exterior to the disulfide<br>
bridge between Cys60 and Cys,21. The large hydrophobic side chain lays parallel to the dimer surface<br>
within this trough. This residue is only moderately surface accessible yet its high temperature (P-)<br>
factor and the lack of hydrogen bonding from surrounding residues to the side chain suggest its<br>
potential flexibility. Although all of the substitutions that were modeled at Trp23) were predicted to<br>
induce substantially high energetic costs from the stability analyses, the phenylalanine substitution<br>
was the least costly substitution at this position (Table 2). The phenylalanine fit well into the<br>
surrounding molecular region with limited disruption of the local hydrogen bonding network.<br>
Binding of the GioeQ mutant to either of the two flavivirus group-reactive mAbs, 4G2 and<br>
6B6C-1, was not detected (Table 3). DEN-2 type-specific mAbs 1A5D-1 and 1B4C-2 (DII and DI,<br>
respectively) exhibited reduced affinities for G106Q transfected cells and for secreted VLPs. Dengue<br>
complex-specific mAb 10A1D-2 also exhibited moderately reduced reactivity for the G106Q VLP<br>
antigen (Table 3). However, the reactivity of the GioeQ mutant was unchanged from the reactivity of<br>
the wild-type pCB8D2-2J-2-9-l antigen for polyclonal anti-DEN-2 MHIAF, as well as for the<br>
remaining subcomplex- and type-specific mAbs: 9A4D-1 (DI), 4E5 (DII), and 3H5, 9A3D-8,<br>
10A4D-2, 9D12, and 1A1D-2 (Dili) (Table 3).<br>
The W231F substitution also abolished the binding of both flavivirus group-reactive mAbs,<br>
4G2 and 6B6C-1, as well as that of flavivirus subgroup-reactive mAb 1B7-5 (Table 3). This<br>
substitution additionally interfered with the binding of type-specific DI mAb 1B4C-2, but the binding<br>
of the remaining subcomplex- and type-specific DI, DII and Dili mAbs and a polyclonal DEN-2<br>
MHIAF were unchanged relative to the non-mutated wild-type plasmid (Table 3). In three separate<br>
experiments, secretion of W23]F VLP antigen into the tissue culture medium from transiently<br>
transfected COS-1 cells was not detected. Consequently, the effects of this substitution could only be<br>
analyzed by IFA of plasmid transfected cells.<br>
The H244R and K247R substitutions did not have an effect on the binding of any mAbs in<br>
either IFA of transfected cells, or in Ag-ELISA of secreted VLP antigen.<br>
Example 2<br>
Identification of additional cross-reactive epitopes<br>
through nearest neighbor search<br>
This example demonstrates the identification of additional cross-reactive epitopes using a<br>
"nearest neighbor" search.<br>
Following the identification of cross-reactive epitope residues G\6 and W231 (DEN-2<br>
numbering), the E-glycoprotein atomic structure was reexamined to search for additional flavivirus<br>
cross-reactive epitope'residues. A "nearest neighbor" search was conducted of the surface of the Eglycoprotein<br>
structure, looking for additional residues located within 10-15 A of the identified<br>
residue. This distance is within the binding footprint of a single antibody paratope (Faebler et al.,J.<br>
Mol. Biol. 313:83-97, 2001). In this second iteration of cross-reactive epitope residue identification<br>
the same five optimality criterion as above were used, with one change. The criterion of strict<br>
conservation across the flaviviruses was relaxed to now include variable residues. In this way,<br>
residues either conserved in their physiochemical nature and/or conserved only within a particular<br>
flavivirus complex (such as the four DEN virus serotypes) could be identified.<br>
This nearest neighbor search yielded another seven potential cross-reactive epitope residues.<br>
Amino acid substitutions at these positions were modeled into the TBE virus E-glycoprotein<br>
structure as described above. Mutagenic PCR primers were then synthesized (Table 1) and used to<br>
introduce mutations into the wild-type DEN-2 prM/E expression plasmid. Plasmids were transiently<br>
transfected into CHO cells, and transfected cells and secreted VLP antigen were analyzed with the<br>
anti-DEN-2 mAb panel (Table 3). The substitutions introduced at these positions (of SEQ ID NO:<br>
14) were: Lys64 to Asn (K^N), Thr76 to Met (T76M), Gln77 to Arg (Q77R), Gly)04 to His (Gi04H),<br>
Leu,o7 to Lys (L)07K), Ghii26 to Ala (Ei26A), and Thr226 to Asn (T226N) (Table 2). A single double<br>
mutant combining substitutions at positions 126 and 226 (E126A/T226N) was also examined. Since the<br>
initial W23]F substitution interfered with antigen secretion, the effects of an alternative substitution at<br>
this position, Trp23i to Leu (W23iL), were also examined.<br>
The G,04H, Li07K, and W23]L substitutions had the greatest effect on decreasing the<br>
reactivities of DII cross-reactive mAbs. Gly104 is located on the upper surface of the dimer at the tip<br>
of the tight loop structure which the fusion peptide adopts in the E-glycoprotein dimer (Modis et al.,<br>
PNAS 100:6986-91, 2003; FIG. 2). The residue has moderately high surface accessibility and a<br>
relatively high temperature (p-) factor. The replacement of this small aliphatic glycine was modeled<br>
with a large polar histidine at this position. The histidine residue fits well into this pocket and was<br>
predicted not to alter the hydrogen-bond network in the region. Moreover, because the tick-borne<br>
flaviviruses have a histidine at this position (Table 4) it seemed probable that this substitution would<br>
not disrupt the structure in this localized region or elsewhere within DII. In fact, a posteriori stability<br>
calculations based upon the DEN-2 E atomic structure (Modis et al., PNAS 100:6986-91, 2003)<br>
indicate that the G104H substitution is energetically favorable (Table 2).<br>
The Gio4H substitution, like both substitutions examined at Trp23i, produced a plasmid that<br>
was unable to secrete measurable VLP antigen into the tissue culture medium upon transfection of<br>
either COS-1 or CHO cells. Consequently, the effects of G^H and W23iL substitutions were<br>
analyzed solely by IFA of plasmid transfected cells, as described above for W23iF. The GWH<br>
substitution ablated the reactivity of all three of the flavivirus cross-reactive mAbs, 4G2, 6B6C-1,<br>
and 1B7-5. The type-specific DII mAb 1A5D-1 also showed strongly reduced reactivity for cells<br>
transiently transcribed with this plasmid Table 3). W23iL showed a reduction in mAb reactivities<br>
very similar to W23iF, knocking out any discernable recognition of all three cross-reactive mAbs<br>
(Table 3). The reactivity of DI mAb 1B4C-2 was also reduced by this mutation, but there were no<br>
discernable changes in the reactivities of the remaining subcomplex- and type-specific mAbs or the<br>
anti-DEN-2 MHIAF for either the Gi04H or W231L plasmid constructs (Table 3).<br>
The L107K substituted plasmid exhibited a pattern of reduced reactivities for flavivirus crossreactive<br>
mAbs unlike any of the other substitutions. Leu)07 sits directly below GlyI06 on the outer<br>
lateral surface of the E-protein dimer. This residue has relatively high surface accessibility and<br>
temperature (p-) factor, and its hydrophobic side-chain is directed laterally away from the dimer.<br>
This residue is also strongly conserved across the flaviviruses; the exceptions being the tick-borne<br>
Powassan virus, JE virus strain SA-14-14-2, and DEN-2 virus strain PUO-280 (Table 4). All of these<br>
viruses have a phenylalanine instead of a leucine at this position. A large basic lysine was substituted<br>
for the leucine at this position. Modeling of this L107K substitution indicated that it too was unlikely<br>
to alter the localized hydrogen bonding network. This observation and the low thermodynamic free<br>
energy (ddG) stability calculation (Table 2) suggested that this substitution was unlikely to induce<br>
localized or domain associated conformational changes.<br>
Flavivirus group-reactive mAb 4G2 showed no discernable reactivity for this construct in<br>
either IFA of plasmid transfected cells, or by Ag-ELISA of secreted VLP antigen. However, the<br>
reactivities of the other two cross-reactive mAbs, 6B6C-1 and 1B7-5, were unchanged for this<br>
construct relative to the non-mutated wild-type plasmid (Table 3). L^K plasmid-transfected cells<br>
and secreted VLP antigen also showed moderately reduced reactivity for mAbs 1A5D-1, 10A1D-2<br>
and 1B4C-2, while all other mAbs and the polyclonal MHIAF reactivities were not significantly<br>
different than they were for the wild-type plasmid (Table 3).<br>
Unlike Leu107, Glui26 appears to be incorporated into epitopes recognized by flavivirus<br>
group-reactive mAb 6B6C-1 and subgroup-reactive mAb 1B7-5, but not in the epitope of flavivirus<br>
38<br>
group-reactive mAb 4G2. Glu126 is located 10-12A from Trp231 in the same trough on the upper and<br>
outer surface of DII. The bulky side chain projects from the a-carbon backbone up into this trough<br>
producing a moderately high surface accessibility and a high p-factor (Fig. 2). The replacement of<br>
this large, negatively charged acidic glutamine was modeled with a small hydrophobic alanine at this<br>
position. This substitution was predicted to induce a moderately high, but acceptable energetic cost<br>
in the free energy stability analysis based on the TBE virus E-glycoprotein structure coordinates<br>
(THE virus equals Lys,26, Table 2).<br>
The Ei26A substitution reduced the reactivity of flavivirus group-reactive mAb 6B6C-1, and<br>
moderately reduced the reactivity of subgroup-reactive mAb 1B7-5 (Table 3). However, mAb 6B6C-<br>
1 exhibited reduced reactivity only by IFA of mutant plasmid transfected cells, and 1B7-5 only<br>
showed reactivity reductions for this construct in Ag-ELISA (Table 3). Similarly, type-specific DII<br>
mAb 1A5D-1 exhibited moderately reduced reactivity by Ag-ELISA, but there was no detectable<br>
reduction in its reactivity by IFA (Table 3). The T226N substitution did not alter the reactivity of any<br>
of the flavivirus group-reactive mAbs relative to the non-mutated wild-type plasmid, and the<br>
E|26A/T226N double mutant generally showed a similar pattern of reduction of mAb reactivity as did<br>
Ei26A alone. The two exceptions to this correlation were in the reactivities of mAbs 1B7-5 and<br>
10A1D-2. E|26A/T226N exhibited the same moderate 87% reduction in Ag-ELISA reactivity for<br>
flavivirus subgroup-reactive mAb 1B7-5 as did E^A. However, the double mutant also exhibited a<br>
strong 97% reduction for this same mAb by IFA, which was not observed for either single mutant<br>
(Table 3). DEN virus complex-specific mAb 10A1D-2 also exhibited moderate reactivity decreases<br>
by IFA for this double mutant (Table 3).<br>
K64N, T76M, and ChvR were all unchanged in their reactivities for the flavivirus crossreactive<br>
mAbs. The T76M VLP antigen did however show reduced reactivity for DII type-specific<br>
mAb 1A5D-1 and for DI mAb 1B4C-2 in Ag-ELISA (Table 3).<br>
Example 3<br>
Spatial characterization and organization of flavivirus group-reactive<br>
epitope residues<br>
This example describes the spatial characterization and organization of exemplary flavivirus<br>
cross-reactive epitope residues.<br>
The six residues (G104, G106, L\07, Ei26, T226, and W23i) identified as participating in the<br>
flavivirus cross-reactive epitopes are spatially arranged on the DEN-2 virus E-glycoprotein surface in<br>
two clusters (FIG. 1). The most prominent grouping of these residues is the clustering of three<br>
residues from the highly conserved fusion peptide region of DII (Allison et al.,J. Virol. 75:4268-75,<br>
2001). These residues, Glyio4, GlyI06, and Leu107, are almost completely conserved across the<br>
flaviviruses (Table 4).<br>
The cross-reactive mAbs most strongly affected by substitutions in this region were 4G2 and<br>
6B6C-1. These two mAbs are considered to be quite similar; both are flavivirus group-reactive and<br>
have been grouped into the Al epitope of the E-glycoprotein (Gentry et al.,Am, J. Trap. Med. Hyg.<br>
31:548-55, 1982; Henchal etal.,Am. J. Trap. Med. Hyg. 34:162-69, 1985; Mandl etal, J. Virol<br>
63:564-71, 1989; Roehrig et al, Virology 246:317-28, 1998). The data disclosed herein demonstrate<br>
that although the epitopes of these two mAbs spatially overlap, they do not contain exactly the same<br>
residues. Substitutions at GI04, G106, or L107 knock out the ability of mAb 4G2 to bind to the Eglycoprotein.<br>
However, only substitutions at G104 and G|06 interfere with the binding ability of mAb<br>
6B6C-1. LIO? is therefore not a component of the flavivirus group-reactive epitope recognized by<br>
mAb6B6C-l.<br>
The G104H substitution dramatically reduced the reactivities of all three of the flavivirus<br>
cross-reactive mAbs for this construct (Table 3). Without being bound by theory, it is unlikely that a<br>
glycine residue, with no side chain, would directly participate in the binding energetics of an<br>
antibody-antigen (Ab-Ag) interaction. However, if a glycine residue is included in the buried surface<br>
area of this antibody epitope, the introduction of a large bulky hydrophobic side chain is likely to<br>
disrupt the Ab-Ag shape complementarity and hence increase the dissociation rate-constant (Kj) of<br>
the Ab-Ag interaction (Li et al., Nature Struct. Biol. 10:482-88, 2003). Gi04H also reduced the<br>
recognition of type-specific DII mAb 1A5D-1 (Table 3). The 1A5D-1 epitope is non-neutralizing,<br>
reduction sensitive and moderately surface accessible (Roehrig et al., Virology 246:317-28, 1998).<br>
All of the fusion peptide substitutions introduced into this region reduced the reactivity of 1A5D-1,<br>
consistent with the interpretation that the buried surface area footprint of this mAb not only includes<br>
DEN-2 virus serotype-specific residues, but also includes these strongly conserved residues as well.<br>
A comparison of the DEN-2 atomic structure with flavivirus E-glycoprotein alignments identifies at<br>
least two unique DEN-2, DII, surface accessible residues (Glu7l and Asn83), and a third residue<br>
variable within DEN-2 but distinct from the other DEN virus serotypes (Thr8i). All of these residues<br>
are within 10-22 A of Gly104, a distance well within the buried surface area of a typical Ab-Ag<br>
interface (Lo et al.,J. Mol. Biol. 285:2177-98, 1999). Alternatively, less surface accessible typespecific<br>
residues nearby could participate in mAb 1A5D-1 binding since this epitope itself is only<br>
moderately surface accessible (Roehrig et al., Virology 246:317-28, 1998). Since this mAb is DEN-2<br>
virus specific, these type-specific residues would be expected to provide the majority of the binding<br>
energy for 1A5D-1.<br>
The GioeQ substitution also knocked out the reactivities of both of the flavivirus groupreactive<br>
mAbs, 4G2 and 6B6C-1, though it did not alter the binding of subgroup-reactive mAb 1B7-5<br>
(Table 3, FIG. 2). Type-specific DII mAb 1A5D-1 again lost all measurable reactivity to the G106Q<br>
construct, as did 1B4C-2. The 1A5D-1 epitope footprint appears to include conserved fusion peptide<br>
residues in addition to DEN-2 serotype-specific residues as discussed herein. The reduced reactivity<br>
of DI mAb 1B4C-2 for the GjoeQ construct is difficult to explain. Because of the lack of biological<br>
activity of DI, epitope assignments to this domain can be problematic (Roehrig et al., Virology<br>
246:317-28, 1998). Without being bound by theory, the involvement of Gly)06 as well as that of<br>
Leu 107 are consistent with the possibility that either the previous DI assignment is incorrect, or that<br>
40<br>
the 1B4C-2 mAb footprint includes residues from both DI and DII. However, if 1B4C-2 recognizes<br>
such an inter-domain epitope, this high affinity mAb would be expected to interfere with the Eglycoprotein<br>
dimer to trimer reorganization associated with virus-mediated membrane fusion, which<br>
it does not.<br>
Leu,o7 is the third residue identified in the fusion peptide region of DII that is incorporated<br>
into flavivirus cross-reactive epitopes. Unlike the substitutions at E-glycoprotein positions 104 and<br>
106, the Lio?K substitution knocked out the reactivity of flavivirus group-reactive mAb 4G2, but it<br>
did not alter the reactivity of the other flavivirus group-reactive mAb, 6B6C-1 (Table 3, FIG. 2).<br>
Beyond this discrepancy, the reactivity patterns of the rest of the mAbs for this construct were similar<br>
to that observed for the other fusion peptide substitutions. mAbs 1A5D-1, 10A1D-2, and 1B4C-2 all<br>
showed little to no reactivity for the L^K. construct (Table 3).<br>
Previous studies have examined the effects of mutagenesis in this fusion peptide region.<br>
Pletnev et al. (J. Virol. 67:4956-63, 1993) performed mutagenesis to fusion peptide residues 104 and<br>
107 in a chimeric infectious clone containing the TBE virus structural genes and DEN-4 virus nonstructural<br>
genes. TBE virus has a histidine at position 104 as do all of the tick-borne flavi viruses.<br>
Pletnev et al. constructed the opposite substitution that was constructed herein, H|04G, replacing the<br>
tick-associated histidine with the mosquito-associated glycine, but they were unable to recover live<br>
virus from this construct. They also constructed a double mutant H^G/ L^F from which they were<br>
able to recover virus; however, they were unable to detect any effect of these mutations on mouse<br>
neurovirulence. Allison et al. (J. Virol. 75:4268-75, 2001) also performed mutagenesis at Leu)07<br>
examining the role of this residue in virus-mediated membrane fusion using TBE virus VLPs. They<br>
replaced Leu107 with phenylalanine, threonine, or aspartic acid. They found that all of these<br>
mutations reduced the rate of fusion. Moreover, consistent with the results presented herein, they<br>
found that the L^D substitution appeared to completely abolish the binding of their DII flavivirus<br>
group-reactive mAb A1.<br>
The fourth residue identified as having a major effect on the flavivirus cross-reactive mAbs<br>
was Trp23i, an invariant residue across the flaviviruses (Table 4). Both substitutions introduced at<br>
Trp23i dramatically reduced the reactivity of all three of the flavivirus cross-reactive mAbs, 4G2,<br>
6B6C-1, and 1B7-5. This residue is structurally distant from the fusion peptide region (FIGS. 1 and<br>
2). It is somewhat surprising that substitutions at this residue affect the binding of mAbs also shown<br>
to recognize the distant fusion peptide residues. With out being bound by theory, the strict<br>
conservation of tryptophan (Table 4) and the predicted high energetic costs of substitutions at this<br>
position (Table 2) suggest that this residue could be important for proper DI/DII conformational<br>
structure and function. If this were the case, the loss of reactivity of mAbs recognizing fusion peptide<br>
residues could occur from the induction of localized structural disturbances across DII occurring at a<br>
distance from Trp23i. However, the Trp23i substitutions did not significantly affect the binding of<br>
any of the remaining DII mAbs, 4E5, 1A5D-1, and 10A1D-2 (DI or DII); whereas mAb 1A5D-1<br>
reactivity was reduced or ablated by all of the fusion peptide substitutions. mAb 4E5 does not<br>
recognize native virus yet it blocks virus-mediated cell-membrane fusion, presumably by recognizing<br>
an epitope that is exposed only during or after low-pH-catalyzed conformational changes (Roehrig et<br>
al., Virology 246:317-28, 1998). Without being bound by theory, if substitutions at Trp23i induced<br>
domain wide structural alterations, a loss of reactivity of mAb 1A5D-1 (and the possible exposure of<br>
the non-native-accessible mAb 4E5 epitope, resulting in an increase, or at least a change in, the<br>
reactivity of mAb 4E5 by IFA for these constructs), would be expected. Moreover, the reactivities of<br>
polyclonal MHIAF and of all of the Dili mAbs were no different for these constructs than they were<br>
for the non-mutated wild-type plasmid transfected cells (Table 3). Dili however, is reductiondenaruration<br>
stable and folds into its native IgC like conformation even when this domain is<br>
expressed alone without the remainder of the E-glycoprotein (Bhardwaj et al.,J. Virol. 75:4002-07,<br>
2001).<br>
Both W23iF and W231L plasmids, as well as the G)04H plasmid, failed to secrete measurable<br>
VLP antigen into tissue culture media following transient transfection of COS-1 or CHO cells. The<br>
inability of cells transfected with these plasmids to secrete VLP antigen into tissue-culture media<br>
could result from the disruption of a variety of protein maturation processes. Without being bound<br>
by theory, interference with particle maturation could occur via disruption of E-prM/M<br>
intermolecular interactions, E-glycoprotein dimer interactions, or via the disruption of dimer<br>
organization into the surface lattice covering mature particles. Although the two processes are<br>
interdependent, these substitutions may not interfere with particle formation per se, but may directly<br>
interfere with particle secretion itself. In fact, the IFA staining pattern of DEN-2 GKMH and of<br>
W23|F/L transfected cells was highly punctate and localized within inclusion bodies. Similar IFA<br>
staining patterns have.been observed with non-secreting constructs of dengue and other flaviviruses<br>
(Chang et al, Virology 306:170-80, 2003). Studies with TBE virus VLPs have shown that<br>
interactions between prM and E are involved in prM-mediated intracellular transport of prM-E<br>
heterodimers (Allison et al.,J. Virol. 73:5605-12, 1999). The location of G\ym near the interiorlateral<br>
edge of DII puts it very close to the E-dimer "hole" where the prM/M proteins are located in<br>
the heterodimer (Kuhn et al., Cell 108:717-25, 2002; FIG. 1). Therefore, it seems likely that G^H<br>
interferes with VLP secretion via disruption of the prM-E interactions necessary for intracellular<br>
transport and secretion. The identity of this residue is positively correlated with arthropod vector,<br>
The mosquito-born flaviviruses have a glycine at this position whereas the tick-borne flaviviruses<br>
have a histidine. Interestingly, Pletnev et al. (J. Virol. 67:4956-63, 1993) introduced the reverse<br>
substitution, H,04G, into the TBE virus E-glycoprotein in a TBE/DEN-4 chimeric infectious clone,<br>
and they were unable to recover virus from this mutant. The inability of G^H transfected cells to<br>
secrete VLP antigen similarly suggests that this too could be a lethal substitution in DEN-2 virus.<br>
Taken together, these two results are consistent with the idea that vector-specific selection has<br>
produced strong epistasis between this residue and other unidentified residue(s) elsewhere in the Eor<br>
prM/M proteins.<br>
Example 4<br>
Identification of flavivirus complex and sub-complex cross-reactive<br>
epitope residues<br>
This example demonstrates the identification of flavivirus complex and sub-complex crossreactive<br>
epitopes using a structure-based rational mutagenesis method.<br>
Cell culture, virus strains and recombinant plasmids<br>
CHO cells (ATCC CCL 61; Manassas, VA) were grown at 37°C with 5% CO2 on Dulbeco's<br>
modified Eagle's minimal essential medium with F-12 nutrient mixture (D-MEM/F-12, GIBCO,<br>
Grand Island, NY) supplemented with 10% heat-inactivated fetal bovine serum (FBS, Hyclone<br>
Laboratories, Inc., Logan, UT), 110 mg/1 sodium pyruvate, 0.1 mM nonessential amino acids, 2 mM<br>
L-glutamine, 2.438 g/L NaHCO3, 100 U/ml penicillin, and 100 ug/ml streptomycin.<br>
The recombinant expression plasmids pCB8SJ2 and pCBWN were used as template DNAs<br>
for both site-directed mutagenesis and for transient expression of St. Louis encephalitis virus (SLEV)<br>
and West Nile virus (WNV) recombinant antigen (see below). The pCB8SJ2 plasmid includes the<br>
human cytomegalovirus early gene promoter, Japanese encephalitis virus (JEV) signal sequence,<br>
SLEV prM and E gene region (amino-terminal 80%), JEV carboxyl terminal 20%, and bovine growth<br>
hormone poly(A) signal. The replacement of the terminal 20% of SLEV E with JEV E gene<br>
sequences dramatically increases the secretion of extracellular VLPs into the culture medium without<br>
altering the native SLEV E glycoprotein conformation (Purdy et al.,J. Clin. Micro. 42:4709-17,<br>
2004). The pCBWN plasmid includes the human cytomegalovirus early gene promoter, JEV signal<br>
sequence, WNV prM and E gene region in its entirety, and bovine growth hormone poly(A) signal<br>
(Davis et al.,J. Viral. 75:4040-47, 2001).<br>
Procedural algorithm<br>
Following the identification and ablation of flavivirus group cross-reactive epitopes,<br>
flavivirus complex and sub-complex cross-reactive epitopes have been identified. Two different<br>
flavivirus complexes, the JEV complex and the DENV complex, were focused on. The DENV<br>
complex consists of the four dengue serotypes, DENV-1, DENV-2, DENV-3, and DENV-4. The<br>
large JEV complex includes JEV, WNV, Murray Valley encephalitis virus (MVEV), and SLEV.<br>
The procedural algorithm for the identification of flavivirus complex and sub-complex crossreactive<br>
epitopes utilizes the following optimality criteria: 1) The identification and selection of<br>
amino acid residues with &gt;35% of their surface solvent accessible. These residues are identified from<br>
the published atomic structure coordinates of the DENV-2 soluble ectodomain of the envelope<br>
glycoprotein and homology models of SLEV and WNV derived from the DENV-2 structure (Modis<br>
et al, Proc. Natl. Acad. Sci. USA 100:6986-91, 2003). In addition to examination of amino acid<br>
residues in structural domain II, residues in domains I and III were examined, since published results<br>
indicate that some complex and sub-complex cross-reactive epitopes are mapped onto domains I and<br>
III in addition to domain II (Roehrig et al., Virology 246:317-28, 1998). 2) Amino acids on the outer<br>
or lateral surface of the E-glycoprotein dimer, and accessible to antibody. 3) Amino acid<br>
conservation across the flavivirus complex (based upon a structural alignment of the protein<br>
sequences). Residues conserved across all member viruses of the same complex are favored. If<br>
conserved within but not across the entire complex, then residues with shared identities between<br>
WNV and SLEV are favored in the JEV complex, and residues with shared identities between<br>
DENV-2 and two or more other viruses in the DENY complex are favored over those shared with<br>
DENV-2 and only one other DENY complex virus. 4) Side chain projections exposed towards the<br>
outer surface and accessible to antibody paratopes. 5) Residues with high temperature (P-) factors<br>
should be favored, as these residues tend to be flexible and are able to conform to the antibody<br>
paratope, increasing the antibody-antigen affinity. Amino acid residues with high temperature factors<br>
are more commonly found in antigen epitopes than lower temperature factor residues. 6) Following<br>
identification of potential individual flavivirus complex and sub-complex cross-reactive epitope<br>
residues, all residues are mapped and highlighted on the same E-glycoprotein dimer structure<br>
together. With this technique, groups of potential cross-reactive epitope residues forming clusters<br>
(and hence probable epitopes) are readily identified. 7) Residues fitting all of these criteria and<br>
occurring in structural clusters approximately 20 x 30 A2 (which is the average "footprint" of an<br>
antibody Fab that interacts with an antigen epitope) are favored over residues that are more isolated in<br>
the protein structure. 8) Within an identified structural cluster of potential epitope residues, residues<br>
that more completely satisfy greater numbers of the optimality criteria are selected for the first round<br>
of mutagenesis analysis.<br>
Site-directed mutagenesis<br>
Site-specific mutations were introduced into the SLEY and WNV E genes using the<br>
Stratagene Quick Change® multi site-directed mutagenesis kit (Stratagene, La Jolla, CA) and<br>
pCB8SJ2 and pCBWN as DNA templates following the manufacturer's recommended protocols. The<br>
sequences of the mutagenic primers used for all constructs are listed in Table 5. Four or five colonies<br>
from each mutagenic PCR transformation were selected and grown in 5 ml LB broth cultures. DNA<br>
was mini-prepped and sequenced from these cultures. Structural gene regions and regulatory<br>
elements of all purified plasmids were sequenced entirely upon identification of the correct mutation.<br>
Automated DNA sequencing was performed using a Beckman Coulter CEQ™ 8000 genetic analysis<br>
system (Beckman Coulter, Fullerton, CA) and analyzed using Beckman Coulter CEQ™ 8000<br>
(Beckman Coulter, Fullerton, CA) and Lasergene® software (DNASTAR, Madison, WI).<br>
Transient expression of SLEV and WNV recombinant antigens by CHO cells<br>
CHO cells were electroporated with pCB8SJ2 or pCBWN using the protocol described by<br>
Chang et al (J. Virol, 74:4244-52, 2000). Electroporated cells were recovered in 50 ml DMEM,<br>
seeded into 150 cm2 culture flasks for VLP expression and into 96-well tissue culture plates for IFA,<br>
and incubated at 37°C with 5% C02. Cells in 96 well plates for IFA were fixed 14-24 hours post<br>
electroporation. Tissue-culture medium and cells were harvested 48-72 hours post electroporation for<br>
antigen characterization.<br>
Characterization of mutant pCB8SJ2 andpCBWN infected cells and secreted antigen<br>
Fourteen to twenty four hours following electroporation, 96-well tissue culture plates<br>
(Costar® #3603 Coming, Inc., Corning, NY) containing cells transformed with the mutated pCB8SJ2<br>
or pCBWN clones were washed twice with PBS, fixed with 3:1 acetone:PBS (v:v) for 10 minutes and<br>
air dried. E-glycoprotein-specific mAbs recognizing each of the three E-glycoprotein domains (Table<br>
6) were used to determine affinity reductions in cross-reactive epitopes by IFA as described by Chang<br>
et al. (J. Virol. 74:4244-52, 2000).<br>
Tissue culture medium was harvested 48-72 hours following electroporation. Cell debris<br>
was removed from tissue culture media by centrifugation for 30 minutes at 10,000 rpm. Ag-ELISA<br>
was used to detect secreted antigen from the mutagenized pCB8SJ2 and pCBWN transformed CHO<br>
cells. Secreted antigen was captured with polyclonal rabbit anti-SLEV and rabbit anti-pCBWN sera<br>
at 1:30,000 and 1:50,000 dilutions, respectively. MHIAF specific for SLEV and WNV was used at a<br>
1:15,000 dilution to detect captured antigen, and this MHIAF was detected using horseradish<br>
peroxidase conjugated goat anti-mouse HIAF at a 1:5000 dilution.<br>
Secreted antigen was concentrated from positive tissue culture medium by centrifugation<br>
overnight at 19,000 rpm, and resuspended in TN buffer (50 mM Tris, 100 mM NaCl, pH 7.5) to<br>
1/100* the original volume. Alternatively, some antigens were concentrated using Millipore's<br>
Amicon® Ultra PL-100 (Millipore, Billerica, MA) centrifugal filter devices. Concentrated antigen<br>
was analyzed with a panel of anti-flavivirus mAbs in Ag-ELISA to determine mAb end point<br>
reactivities of the mutated antigens, following the protocol of Roehrig et al. (Virology 246:317-28,<br>
1998). This Ag-ELISA protocol is the same as that used herein to detect secreted antigen, with the<br>
exception of using the specified mAbs (Table 6) instead of polyclonal MHIAF.<br>
Antigenic characterization and MAb screening of potential cross-reactive epitope residue mutants<br>
Using the structure-based design approach described above, candidate flavivirus complex<br>
and sub-complex cross-reactive epitope residues were narrowed down to 34 in DENV-2 and 31 each<br>
in WNV and SLEV. From these residues and with reiterative application of the optimality criteria<br>
described herein 17 DENV-2, 13 WNV, and 11 SLEV residues were chosen as most likely to be<br>
incorporated into complex and sub-complex cross-reactive epitopes (highlighted in Tables 7-9).<br>
Amino acid substitutions were modeled at these probable cross-reactive epitope residues, selecting<br>
substitutions that should potentially disrupt or ablate antibody recognition without altering Eglycoprotein<br>
structural conformation, disrupting dimer interactions, or impairing particle formation,<br>
maturation, or secretion. Stability calculations were performed for all possible amino acid<br>
substitutions of candidate residues using the PoPMuSiC server, (available on the Universite Libre de<br>
Bruxelles' web site) and the DENV-2 E-glycoprotein pdb file coordinates (Modis et al,, Proc. Natl.<br>
Acad. Sci. 100:6986-91, 2003) or homology model coordinates for WNV and SLEV. Amino acid<br>
substitutions modeled in the E-glycoprotein structures with free energies of folding equal to or less<br>
than that of the non-mutated wild-type E-glycoprotein were re-examined with the Swiss-Pdb Viewer<br>
software (available on the Swiss Institute of Bioinformatics' web site) to identify those substitutions<br>
that minimized local structural disturbances while maintaining structurally relevant biochemical<br>
interactions such as hydrogen bonding and/or charge interactions with neighboring amino acids.<br>
Substitutions at 11 of 16 potential cross-reactive epitope residues selected for mutagenesis in<br>
pCB8SJ2 altered the reactivities of all 14 of the anti-SLE mAbs, relative to wild-type pCB8SJ2<br>
(Table 10). Eight of the 14 MAbs were flavivirus group- or subgroup-cross-reactive (see Table 6).<br>
Substitutions at nine of the 16 residues analyzed altered the reactivity of all eight of the flavivirus<br>
group- or subgroup-cross-reactive mAbs. Substitutions at four of 16 potential cross-reactive epitope<br>
residues altered all three of the JEV complex- and subcomplex-cross reactive mAbs. Only one<br>
substitution however, affected type-specific mAb reactivities (Fig. 3). The effect of this substitution<br>
(GioeQ) on type-specific mAb reactivities was to actually increase the reactivity of these mAbs<br>
relative to that of the wild-type unaltered pCB8SJ2. Without being bound by a single theory, such<br>
increase in the reactivity of type-specific antibodies is believed to be beneficial for the development<br>
of type-specific flavivirus antigens.<br>
Substitutions at 14 of 17 residues selected for mutagenesis in pCBWN altered the reactivities<br>
of all 10 of the anti-WNV mAbs, relative to wild-type pCBWN (Table 11). Six of the 10 anti-WNV<br>
mAbs were flavivirus group- or subgroup-cross-reactive, two were JEV complex cross-reactive and<br>
two were WNV type-specific (see Table 6). Nine of the 17 substitutions examined altered the<br>
reactivities of all six group- and subgroup-cross-reactive mAbs; 12 of these 17 substitutions affected<br>
the reactivities of both of the JEV complex cross-reactive mAbs. The GioeV substitution in pCBWN<br>
was the only substitution to alter type-specific mAb reactivities, and, as with pCB8SJ2, this<br>
substitution actually increased the reactivity of the type-specific mAbs (Fig. 3).<br>
The outcome that many of these substitutions altered mAb reactivities (Tables 10 and 11;<br>
Fig. 3) illustrates not only the efficiency of the described algorithms for identifying cross-reactive<br>
epitope residues, but also that these cross-reactive epitopes can be altered to ablate or appreciably<br>
interfere with the ability of an antibody to recognize these modified antigens. For example, 82% and<br>
69% of the potential cross-reactive epitope residue substitutions examined in pCBWN and pCB8SJ2,<br>
respectively, affected all of the cross-reactive antibodies reactive to these two viruses from the<br>
antibody panel (see Fig. 3). The high percentage of residues, selected a priori, affecting mAb<br>
reactivities illustrates the accuracy of the cross-reactive epitope residue selection algorithms.<br>
The mAb characterization of potential cross-reactive epitope residue mutants illustrates the<br>
importance of the E-protein fusion peptide region as a potently cross-reactive antigenic determinant.<br>
As described herein, substitutions at fusion peptide residues GKM, G106, and LIQ? strongly affected<br>
many of the mAb reactivities for DENV-2, SLEV and WNV (see Tables 10 and 11). Without being<br>
bound by a single theory, Gioe appears to be the most important cross-reactive antigenic determinant<br>
of these residues. Substitutions at G,06 altered the reactivities of 7 of 10 cross-reactive mAbs<br>
recognizing SLEV, and 7 of 8 cross-reactive mAbs recognizing WNV (see Tables 10 and 11).<br>
Substitutions at fusion peptide residue G104 also affected the reactivities of many mAbs for each of<br>
these viruses. However, all substitutions examined at this position produced plasmids that were<br>
unable to efficiently secrete VLP antigen upon transient transformation into eukaryotic cells. This<br>
observation was true for all three flaviviruses examined: DENV-2, SLEV and WNV.<br>
Substitutions at fusion peptide residue Gjoe had a variety of effects on mAb reactivities for<br>
both pCBWN and pCB8SJ2. The majority of the substitutions at this residue reduced or ablated a<br>
mAb's ability to recognize the antigen. This occurred with cross-reactive mAbs 4G2, 6B6C-1, 4A1B-<br>
9, and 2B5B-3 in Gi06V-pCBWN and with 4G2 and 2B5B-3 for G106Q-pCB8SJ2 (see Tables 10 and<br>
11), indicating that the substituted residue is a part of the antigenic epitope recognized by these<br>
antibodies.<br>
Example 5<br>
Human IgM MAC-ELISA Serology<br>
This example demonstrates the representative nature of a murine antibody response as a<br>
model of human antibody response to substitutions in the flavivirus cross-reactive epitopes.<br>
Human sera<br>
Well-characterized serum specimens were assembled from the Diagnostic and Reference<br>
Laboratory, Arbovirus Diseases Branch, Division of Vector-Borne Infectious Diseases, US Centers<br>
for Disease Control and Prevention. A serum panel (see Table 12) was assembled from patients<br>
infected in the US between 1999 and 2004 with either WNV («=6) or SLEV («=10), as determined by<br>
the standard 90% plaque-reduction neutralization (PRNT) assay. SLEV is endemic to North<br>
America, whereas WNV was first introduced into North America in 1999 and has spread epidemically<br>
since that time.<br>
The flavivirus responsible for the most recent infection was determined as that with the<br>
highest neutralizing antibody titer, which had to be at least four-fold greater than that for any other<br>
virus tested. Because of the high level of cross-reactivity between the SLEV and WNV viruses, it is<br>
often difficult to determine the infecting virus by ELISA, thus requiring the PRNT. SLEV infected<br>
sera with measurably high levels of cross-reactivity for WNV were purposefully selected in order to<br>
maximize the ability to asses for improved discrepancy (specificity) of the pCBWN-GioeV versus the<br>
pCBWN wild-type antigen. SLEV infected patient sera were split into two groups based upon<br>
previously determined (Diagnostic and Reference Laboratory) positive to negative (P/N) ratios for<br>
SLEV and for WNV. 'Equivocal' SLEV sera («=5) were those that were clear SLEV infections from<br>
the PRNT data, yet had MAC-ELISA P/N ratios that were not statistically different between SLEV<br>
and WNV. Three of these equivocal SLEV samples were negative (P/N 
was presumptive positive (P/N &gt;2.0 and 5.0) for both<br>
viruses. 'Misleading' SLEV sera («=5) were SLEV positive in the PRNT, yet had MAC-ELISA P/N<br>
ratios that were not only positive for both viruses, but were actually greater for WNV than for SLEV.<br>
Definitive 'positive' WNV infected patient sera («=6) were selected based on MAC-ELISA results<br>
from the Diagnostic and Reference Laboratory collection for use as positive control sera to assess the<br>
accuracy of the pCBWN-G,o6V plasmid derived antigen.<br>
IgM ELISA protocols<br>
IgM ELlSAs were performed following the protocols of Purdy et al. (J. Clin. Micro.<br>
42:4709-17, 2004) and Holmes et al. (J. Clin. Micro. 43:3227-36, 2005). Briefly, the inner 60 wells<br>
of Immulon II HB flat-bottomed 96-well plates (Dynatech Industries Inc., Chantilly, VA) were coated<br>
overnight at 4°C in a humidified chamber with 75 ul of goat anti-human IgM (Kierkegaard &amp; Perry<br>
Laboratories, Gaithersburg, MD) diluted at 1:2000 in coating buffer (0.015 M sodium carbonate,<br>
0.035 M sodium bicarbonate, pH 9.6). Wells were blocked with 300 ul of InBlock blocking buffer<br>
(Inbios, Seattle, WA, L/N FA1032) for 60 minutes at 37°C in a humidified chamber. 50 ul of sera<br>
were added to each well and incubated again for 60 minutes at 37°C in a humidified chamber. Human<br>
test sera were diluted 1:400 in sample dilution buffer (Inbios, L/N FA 1055). Positive control sera<br>
were diluted 1:3000 for SLEV and 1:800 for WNV. Positive and negative control VLP antigens were<br>
tested on all patient sera in triplicate by diluting appropriately in sample dilution buffer and adding<br>
50 ul to appropriate wells for incubation overnight at 4°C in a humidified chamber. Captured<br>
antigens were detected with 50 ul/well of polyclonal rabbit anti-pCBWN diluted 1:1000 in sample<br>
dilution buffer and incubated for 60 m at 37°C in a humidified chamber. Rabbit sera was detected<br>
with horseradish peroxidase conjugated goat anti-rabbit sera diluted 1:8000 in IgM conjugate dilution<br>
buffer (Inbios, L/N FA 1056) and incubated for 60 m at 37°C in a humidified chamber. Bound<br>
conjugate was detected with 75 ul of 3,3'5,5'-tetramethylbenzidine (Neogen Corp, Lexington, KY)<br>
substrate, incubated at RT for 10 min, stopped with 50 ul of 2N H2SO4, and then read at A450 using an<br>
ELx405HT Bio-Kinetics microplate reader (Bio-Tek Instruments Inc., Winooski, VT).<br>
IgM test validation and interpretation<br>
Test validation and P/N values were determined according to the procedure of Martin et al.<br>
(J. Clin. Micro. 38:1823-26, 2000), using internal positive and negative serum controls included in<br>
each 96-well plate. Positive (P) values for each specimen were determined as the average A45o for the<br>
patient serum sample incubated with positive VLP antigen. Negative (N) values were determined for<br>
each plate as the average A4so for the normal human serum control incubated with positive VLP<br>
antigen.<br>
Human serology<br>
To determine how representative the murine antibody response (mAb data) is as a model of<br>
the human antibody response (serological data) to the viral substitution antigens described herein,<br>
serological assays were performed with single substitution, prototype type-specific antigens. As the<br>
mAb screening results indicated that fusion peptide residue 106 was incorporated into multiple crossreactive<br>
epitopes for both WNV and SLEV, this substitution was selected to conduct MAC-ELISA<br>
serum tests.<br>
The prototype type-specific GioeV-WNV Ag dramatically outperformed the wild-type (wt)-<br>
WNV Ag when tested on 10 difficult to discern 'equivocal' or positively 'misleading' SLEV-infected<br>
patient sera (Table 12). Six of 10 of these SLEV infected sera were correctly diagnosed as WNVnegative<br>
by MAC-ELISA (P/N 
&gt;2.0 
WNV Ag, only four sera were correctly scored as WNV negative, one was equivocal, and five were<br>
misdiagnosed as WNV positive with this unmodified Ag. When antigens were directly compared on<br>
each individual serum sample, the G105V-WNV Ag produced lower P/N ratios than did the wt-WNV<br>
Ag in nine of 10 cases on these SLEV infected sera, indicating that the G105V-WNV Ag exhibits<br>
improved specificity and reduced cross-reactivity relative to the wt-WNV Ag.<br>
The prototype type-specific GioeV-WNV Ag also outperformed the unaltered wt-WNV Ag in<br>
MAC-ELISA sensitivity tests on positive WNV infected human sera (Table 12). Five of six WNV<br>
infected patient sera had positive P/N ratios when tested with the GioeV-WNV Ag, whereas four were<br>
positive with the wt-WNV Ag. The single WNV positive serum sample that tested negative with the<br>
wt-Ag and equivocal with the GioaV Ag had the lowest neutralizing tilers of the WNV sera in the<br>
PRNT (see Table 12), indicative of a weak antibody titer.<br>
In addition to improved accuracy with the GioeV-WNV Ag, it was also more sensitive than<br>
was the wt-WNV Ag. In 5 of the 6 WNV infected sera, the MAC-ELISA P/N ratios were higher with<br>
the G]06V- than with the wt-WNV Ag (Table 12). Higher P/N ratios are expected from an improved<br>
type-specific Ag relative to the cross-reactive wt Ag when tested on sera infected with the same virus.<br>
The prototype type-specific GjoeV-WNV Ag exhibited improved specificity, accuracy, and<br>
sensitivity relative to the unmodified wt-WNV Ag. The GioeV-WNV Ag was more specific and<br>
accurate for WNV diagnosis than was the wt Ag, correctly diagnosing more WNV infected sera as<br>
positive and fewer SLEV infected sera as negative, than did the wt-WNV Ag. The GioeV-WNV Ag<br>
was also more sensitive at detecting WNV antibody in WNV infected serum than was the wt-WNV<br>
Ag. The positive signal indicating the presence of WNV antibody (P/N ratios) was greater for GioeVWNV<br>
Ag than it was for the wt-Ag when testing WNV infected sera, and less than that of the wt-Ag<br>
when testing non-WNV infected sera.<br>
Example 6<br>
Murine Immunization<br>
This example demonstrates the ability of prototypical type-specific flavivirus mutant<br>
compositions to generate type-specific neutralizing antibody responses in mice.<br>
Mouse vaccination<br>
Groups of six female outbred ICR mice were used in this study. Mice were immunized by<br>
injection with pCB8D2-2J-2-9-l, pCB8D2-2J-2-9-l-G106Q, pCBWN, pCBWN-G,o6V, pCB8SJ2, or<br>
pCB8SJ2-G,o6Q expression plasmids as described herein. Each mouse was injected with 100 ug of<br>
Picogreen® fluorometer quantified plasmid DNA in PBS pH 7.5, at a concentration of 1 ug/ul. Mice<br>
were immunized with 50 ug of plasmid DNA injected intramuscularly into each thigh on weeks 0 and<br>
3. Mice were bled on week six following initial vaccination.<br>
Plaque reduction neutralization assays<br>
Six week post-vaccination serum specimens were tested for the presence of type-specific<br>
neutralizing (Nt) antibody (Ab) by plaque reduction neutralization test (PRNT). PRNT was<br>
performed with freshly confluent Vero cell monolayers as described by Chang et al. (J. Virol.<br>
74:4244-52, 2000) using DENV-2 (16681), WNV (NY-99), and SLEV (MSI-7) viruses.<br>
Neutralizing antibody responses<br>
Mice were immunized with wild-type and G!06 substituted plasmids for WNV, SLEV, and<br>
DENV-2 to determine if there were differences between the wild-type and G]06 prototype typespecific<br>
antigens for type-specific Nt Ab liter, cross-reactive Nt Ab titer, and protection from virus<br>
challenge. The type-specific Nt Ab titer results are shown in Table 13. There was little difference in<br>
the 75% PRNT titer between wt and G106 substituted plasmids for all three viruses. The 75% Nt Ab<br>
titer was greater than or equal to 1:128 for almost all of the mice immunized with both the DENV-2<br>
and both the WNV DNA vaccines. One mouse immunized with the wt DENV-2 DNA vaccine had a<br>
75% PRNT titer of 1:64, and two mice immunized with the pCBWN-Gio6V DNA vaccine had 75%<br>
PRNT liters of 1:64 and 1:16.<br>
These results demonstrale lhat for all three flaviviruses tesled, mere was little to no<br>
detectable difference in type-specific neutralizing antibody liter between the prototype type-specific<br>
G)06 mutant vaccines and their wt counterparts. These results also illustrate lhat the methods<br>
described herein for ablating cross-reaclive epitope residues can be used to generate type-specific<br>
flavivirus prM/E expression plasmids for use as DNA vaccines that still maintain potent type-specific<br>
neutralizing immunogenicily.<br>
Example 7<br>
Reduction of Cross-reactive Immunogenicity of Type-specific Genetic Vaccines<br>
This example provides methods by which prototypical type-specific flavivirus mutant<br>
compositions can be used lo generate a reduced cross-reactive neutralizing antibody response relalive<br>
to the unaltered wild-lype composilions.<br>
Mouse vaccination and plaque reduction neutralization assays<br>
Female outbred ICR mice (such as the mice in Example 6) can be used in this study.<br>
Twelve-week post vaccination serum samples from immunized mice will be tested for cross-reactive<br>
(heterologous) Nt antibody response by PRNT. Unlike the type-specific PRNTs performed in<br>
Example 6, the cross-reactive PRNTs will be performed by examining Nt of immunized mouse sera<br>
not only for the type-specific virus used for immunization, but also for Nt of the seven other medically<br>
important flaviviruses. Thus, all 12-week mouse sera will be tested for neutralization against eight<br>
different flaviviruses: all four dengue serocomplex viruses, DENV-1 (16007), DENV-2 (16681),<br>
DENV-3 (H87), and DENV-4 (H241); three JEV serocomplex viruses, JEV (SA14-14-2), WNV<br>
(NY-99) and SLEV (MSI-7); and the single medically important member of the yellow fever virus<br>
serocomplex, YFV (17D).<br>
Predicted antibody response<br>
Without being bound by theory, similar type-specific Nt Ab titers between the prototype<br>
type-specific Gioe mutant vaccines and their wt counterparts are expected. Thus, both pCBWN and<br>
pCBWN-GioeV vaccinated mouse sera are predicted to have similar Nt Ab titers against WNV, and<br>
pCB8D2-2J-2-9-l and pCB8D2-2J-2-9-l-G106Q will have similar Nt Ab titers against DENV-2.<br>
However, when these same sera are tested for Nt against the heterologous flaviviruses, it is expected<br>
that significantly lower PRNT titers for prototype type-specific G)06 mutant vaccinated mouse sera<br>
will be observed than for the counterpart wt vaccinated mouse sera. For example, mice immunized<br>
with pCBWN and pCBWN-G,06V will both have similar PRNT titers against WNV, whereas,<br>
pCBWN-G106V immunized mice will have significantly lower PRNT titer against SLEV, JEV, YF,<br>
and the four dengue serotype viruses, than wild-type pCBWN immunized mice.<br>
Example 8<br>
Combining Multiple Cross-reactive Epitope Substitutions into Single Plasmid Constructs<br>
This example provides methods by which individual substitutions affecting different<br>
flavivirus cross-reactive epitopes can be combined into a single construct.<br>
Individual substitutions affecting different flavivirus cross-reactive epitopes (such as those<br>
disclosed herein) can be combined into a single construct based, for example, on mAb screening<br>
results disclosed herein (see, Tables 3, 10 and 11), as well as additional mAb screening studies. For<br>
example, a mutagenesis primer has been designed for SLEV to introduce both the GioeQ and L^K<br>
substitutions into a single pCB8SJ2 plasmid (see, Table 5). This double mutation plasmid has been<br>
constructed, and its sequence confirmed.<br>
Cells can be transformed with this double mutated plasmid (or another plasmid containing a<br>
sequence encoding an E glycoprotein having a combination of two or more mutated amino acids),<br>
and the antigen characterized. In SLEV, the G106Q substitution alone alters the reactivities of many<br>
mAbs recognizing distinct cross-reactive epitopes (Table 10). However, this substitution alone has<br>
no significant effect on the flavivirus group cross-reactive epitope recognized by MAb T-23-1. The<br>
Lio7K substitution does knock out the ability of mAb T-23-1 to recognize the flavivirus cross-reactive<br>
epitope. Without being bound by theory, this suggests that L107 is incorporated in the cross-reactive<br>
epitope recognized by mAb T-23-1, while doe is not.<br>
Because of the generally additive effects observed when combining these substitutions into<br>
single constructs (see, Tables 10 and 11), it is expected that GioeQ/LioyK. antigen will combine the<br>
different effects observed from mAb screening of the individual mutants into a single, multiple<br>
substituted mutant. Upon transfection into mammalian cells, such a multiple mutant plasmid can be<br>
used to produce improved type-specific antigens. When utilized as genetic vaccines, these plasmids<br>
are expected to exhibit further reductions in cross-reactive immunogenicity while still inducing a<br>
potent type-specific immune response.<br>
Example 9<br>
Immune Stimulatory Compositions for the Inhibition or Treatment of a Flavivirus Infection<br>
This example provides methods for administering substances suitable for use as immune<br>
stimulatory compositions for the inhibition or treatment of a flavivirus infection.<br>
An immune stimulatory composition containing a therapeutically effective amount of a<br>
flavivirus polypeptide that includes at least one flavivirus cross-reactive epitope with reduced or<br>
ablated cross-reactivity (particularly in an E glycoprotein) can be administered to a subject at risk for,<br>
or exposed, to a flavivirus (e.g., a dengue virus, West Nile virus, etc.). Alternatively, an immune<br>
stimulatory composition containing a therapeutically effective amount of a nucleic acid vector that<br>
includes flavivirus nucleic acid molecules described herein, or that includes a nucleic acid sequence<br>
encoding at least one flavivirus cross-reactive epitope with reduced or ablated cross-reactivity<br>
(particularly in an E glycoprotein), can be administered to a subject at risk for, or exposed to a<br>
flavivirus.<br>
Dosages and routes of administration for the immune stimulatory composition can be readily<br>
determined by one of ordinary skill in the art. Therapeutically effective amounts of an immune<br>
stimulatory composition can be determined, in one example, by in vitro assays or animal studies.<br>
When in vitro or animal assays are used, a dosage is administered to provide a target tissue<br>
concentration similar to that which has been shown to be effective in the in vitro or animal assays.<br>
While this disclosure has been described with an emphasis on preferred embodiments, it will<br>
be apparent to those of ordinary skill in the art that variations and equivalents of the preferred<br>
embodiments may be used and it is intended that the disclosure may be practiced otherwise than as<br>
specifically described herein. Accordingly, this disclosure includes all modifications encompassed<br>
within the spirit and scope of the disclosure as defined by the claims below.<br>
APPENDIX I<br>
TABLES<br>
Table 1. Nucleotide sequence of primers used for mutagenesis. The mismatched nucleotides causing<br>
the desired substitutions are underlined.<br>
Sequence<br>
5'-TGTTGTTGTGTTGGTTAGGTTTGCCTCTATACAG-3'<br>
5'-TGGGTTCCCCTTGCATTGGGCAGCGAGATTCTGTTGTTG-3'<br>
5 ' -TTC ATTTAGGCTGGGTTCCCCTCGTGTTGGGC AG-3 '<br>
5 '-CCCTTTCCAAATAGTCCAC AGTG ATTTCCCCATCCTCTGTCTACC-3 '<br>
5 ' -GCCTCCCTTTCCAAATAGTTG ACATCC ATTTCCCCA-3 '<br>
5 '-GGTCACAATGCCTCCCTTTCCAAATTTTCCACATCCATTTCCCC-3 '<br>
5 '-AGTTTTCTGGTTGCACAACTTTTCCTGCCATGTTCTTTTTGC-3 '<br>
5 '-GTATCCAATTTGACCCTTGATTGTCCGCTCCGGGCAACC-3 '<br>
5 '-GTCTCTTTCTGTATGAAATTTGACCCTTGTGTGTC-3 '<br>
5 '-AATGTCTCTTTCTGTATCAGATTTGACCCTTGTGTGTCCGCTCC-3 '<br>
5 '-TCCTGTTTCTTCGCACGGGG ATTTTTGAAAGTGACC-3 '<br>
5 ' -AC AAC AACATCCTGTCGCTTCGCATGGGG ATTTTTG-3 '<br><br>
Table 2. Stability free energy (ddG) calculations for putative domain II cross-reactive epitope<br>
substitutions based upon the published pdb coordinates for the DEN-2 virus (Modis et al, PNAS<br>
100:6986-91, 2003) and the TBE virus (Rey et al, Nature 375:291-98, 1995) E-glycoprotein<br>
na: not applicable (these constructs did not secrete VLP antigen and thus could not be examined by Ag-ELISA); nd: not determined.<br>
55<br>
Table 4. Amino acid sequence variability for proposed cross-reactive epitope residues in domain II of the flavivirus E protein.<br>
Japanese Encephalitis<br>
Murray Valley encephalitis<br>
West Nile<br>
St. Louis encephalitis<br><br>
DENV-2 strains containing variable amino acid sequences at these positions are indicated below with their GenBank accession numbers (all incorporated<br>
by reference as of the date of filing of this application).<br>
64R:AF359579; 77L:M24449, X15434, X15214; 107F: M24446<br>
126K:L10053, D00346, M29095, AF204178, M24450, M24451, AF410348, AF410361, AF410362, AF410365, AF204177, D10514<br>
Table 5. Nucleotide sequence of primers used for mutagenesis. The mismatched nucleotides causing<br>
the desired substitutions are underlined.<br><br>
CTCCCTTTTCCAAACAGACCACAGTGGTTACCCCATCCGC<br>
CTCCCTTTTCCAAACAGACCACAGTTGTTACCCCATCCGC<br>
CCCTTTTCCAAACAGACCACAGTCGTTACCCCATCCGC<br>
CTCCCTTTTCCAAACAGACCACACTTGTTACCCCATCCGC<br>
CTCCCTTTTCCAAACAGCTGACATCCGTTACCCCATCCGC<br>
CTCCCTTTTCCAAACAGCTTACATCCGTTACCCCATCCGC<br>
TCCCTTTTCCAAACAGTACACATCCGTTACCCCATCCGC<br>
CTTTTCCAAACAGATCACATCCGTTACCCCATCCGC<br>
CTCCCTTTTCCAAAGAAACCACATCCGTTACCCCATCCGC<br>
AATGCTCCCTTTTCCAAAGAACTGACATCCGTTACCCCATCCGC<br>
CGGGCTTATGGTGAATTGAGCCGCTTGGTTTTTTCC<br>
TTCCATACTCGCCCATGTTGGCAATAAAGGACGGTG<br>
GTAACTGTTCCATACTCGGACATGTTGGCCGTAAAGG<br>
GTAACTGTTCCATAGTTGCCCATGTTGGCCGTAAAGG<br>
CTCTGTTGCGCCAATCGTTTGTGGCAGGGCTCGTC<br>
TTCTCTGTTGCGGAAATCAGTTGTGGCAGGGCTCGTC<br>
TACTACAGTTTGCTTGGTGGCACGCGGTTCCTC<br>
TGATTGCAAGGTTAGGGTTGATCCGCTAACAGTGGC<br>
CGTTCCCTTGATTTTGACGTAGTCAAGCTTAGCTCTGC<br>
ACACATGCCATATGCCGTTCCCTTGATTTTGACC<br>
CAGGGTCCGTTGCTTCCATCATACTGCAGTTCCAC<br>
CGATCATGACCTTGTTGTTCGATCCCCCTGTGC<br>
TTCGATCATGACCTTGTTGAACGCTCCCCCTGTGC<br>
TTTGCCAAATAGTCCGCAGTTGTTGCCCCAGCCCC<br>
TTGCCAAATAGTCCGCAGTCGTTGCCCCAGC<br>
CCTTTGCCAAATAGTCCGCACTTGTTGCCCCAGCCCC<br>
TTGCCAAATAGTCCGCATGCGTTGCCCCAGC<br>
TTTGCCAAATAGGACGCAGCCGTTGCCCCAGCC<br>
TTTGCCAAATAGCCTGCAGCCGTTGCCCCAGCC<br>
CCTTTGCCAAATAGGTAGCAGCCGTTGCCCCAGCCCC<br>
TTTGCCAAATAGAGCGCAGCCGTTGCCCCAGCC<br>
TTCCTTTGCCAAAGTATCCGCAGCCGTTGCCCCAGCC<br>
CCTTTGCCAAAGAATCCGCAGCCGTTGCCCCAGC<br>
CCTTTGCCAAAATGTCCGCAGCCGTTGCCCCAGC<br>
CCTTTGCCAAATCTTCCGCAGCCGTTGCCCCAGC<br>
CTTGGTAGAGCAGGCAAATACGGCGCATGTGTC<br>
CCAACCTGTGTGGAGTAGTCTCCGTGCGAC<br>
CCAACCTGTGTGGAGCCGTTTCCGTGCGACTC<br>
CTGAGTGGCTCCAACATCTGTGGAGTAGTTTCCGTGCG<br>
AGGAGTGATGCTGAAGTACCCTGCCTGAGTGG<br>
CCAAGCTTTAGTACGTATGAAGGCGCCGCAGGAG<br>
CCTCTCCATAGCCTCCAAGCTTTAGTGTGTATGAAGG<br>
AACGTCTCTCTGTTCCTGAACACAGTACTTCCAGCAC<br>
CCGACGTCAACTTGACAGTGTTGTCTGAAAATTCCACAGG<br>
GTTCCCTTCAACTGCAAGTTTTCCATCTTCACTCTACAC<br><br>
ACAGACGCCATAGTTTGTTCCCTTCAACTGCAATTTTTCC<br>
CCATCCGTGCCGTTGTACTGCAATTCCAACACCACAG<br>
GGACCTTAGCGTTGACCGTGGCCACTGAAAC<br>
ACCTTAGCGCTGGCCGTGGCCACTGAAAC<br>
Table 6. E-glycoprotein-specific mAbs recognizing each of the three E-glycoprotein domains<br>
sub-grp: DEN+JE comp<br>
sub-grp: DEN+JE comp<br>
sub-grp. JE comp + YF<br>
sub-grp: DEN-2,3 ,4 + JE<br>
sub-grp: DEN-2, JE, SLE<br>
sub-grp: DEN- 1, 2, JE, WN*<br>
sub-grp: DEN-1,2,3,4 + SLE<br>
JE comp.<br>
JE comp.<br>
sub-comp: DEN-2,3<br>
sub-comp: DEN-1,2,3<br>
sub-comp: SLE + JE<br>
sub-comp: DEN-2 + JEV<br>
sub-comp: DEN-1,2,3<br>
top outer edge<br>
top outer edge SDM in D2<br>
out-mid-lat. SDM in D2<br>
out low lat mid SDM in D216681 -yes<br>
up-mid-top below CHO-153<br>
up-mid-top near CHO 153<br>
up-in-top near CHO &amp; prM<br>
up-mid-top near prM<br>
up-mid-top high.exp.<br>
out-lat mid DI/II region<br>
DI/II border out-mid<br>
DI/II border up-out edge<br>
DI/III up-out edge<br>
DI/III up-out edge<br>
in-mid-up<br>
in-mid-up no SDM in D2<br>
up-out-middle (SDM in D2)<br>
up-out-mid SDM previous<br>
up-out-middle<br>
out-up lat DI/III face<br>
out-up lat DI/III face, above D290<br>
mid-out-end DI/III SDMD2?~yes<br>
mid-out-end DI/III flavis=S/T YES<br>
mid-out-end DI/III<br>
out-up DI-DII interface<br>
out-up DI-DII interface<br>
up-out-top "end" Dili<br>
up-out-top "end" Dili<br>
tip-top-mid Dili<br>
tip-top-mid Dili<br>
tip-top-mid Dili<br>
"RGD" loop up-out-lat<br>
"ROD" loop up-out-lat<br>
"RGD" loop up-out-lat<br>
* not identified as &gt; 35% SA in this particular structure/model<br>
B-f: p-factor (temperature factor) a qualitative assessment of the scale (5-60A2).<br>
SC?: is the amino acid side chain accessible and available for antibody binding<br>
Ep?: might this amino acid be incorporated into an antigen epitope?<br>
DVc: DENY 1-4 complex;<br>
Jec: JE complex (medically important clade =JE, MVE, WN, SLE)<br>
SDM: site-directed mutagenesis<br>
Table 8. Potential JEV complex- and sub-complex cross-reactive epitope residues from WNV, with residues<br>
chosen for mutagenesis highlighted<br>
D2# SLE# WN# Pom. B-f. Location? SC? Conservation ED?<br>
top inner edge near N67 D2<br>
out-lateral-low-mid.<br>
outer lateral middle<br>
out-up-lat. loop near FP<br>
out-mid-lat.<br>
out-up-lat<br>
up-mid-top near N67 in D2<br>
up-mid-top<br>
out mid lat DI/II border<br>
up-mid-top below CHO-153<br>
up-mid-top CHO on D2-153<br>
up-mid-top near CHO 153 yes<br>
up-in-top near CHO &amp; prM<br>
up-mid-top near prM<br>
up-mid-top high.exp.<br>
DI/II border out-mid<br>
DI/II border up-out-top edge<br>
DI/III border outer edge yes<br>
DI/III up-out edge<br>
DI/III up-out edge<br>
up-out-mid. SDM?<br>
prM hole low but above P243<br>
prM hole low, above 243/244<br>
up-top-in good aa JEc YES<br>
out-up lat DI/III face<br>
mid-out-end DI/III<br>
mid-out-end DI/III<br>
up-out-top "end" Dili<br>
tip-top-mid Dili YES<br>
tip-top-mid Dili YES<br>
"ROD" loop up-out-lat yes<br>
"ROD" loop up-out-lat ~y/na<br>
-yes WN, SLE=L ?yes<br>
yes "all" mosq.+ticks ?<br>
no variable, Q=DVc ?<br>
yes variable type? Yes<br>
-yes JEc=A(SLE=S) yes<br>
-no? variable ?possible<br>
yes JEc=K, SDM in WN YES<br>
yes var. yes<br>
-yes var. L=WN,SLE ?/YES<br>
yes H=DVc; yes if<br>
no CHO<br>
.yes "allmosq"=N YES<br>
Y=JEc :SDMinWN YES<br>
yes Q=Jec, SDMinWN YES<br>
yes DVc=TJEc=I/V YES<br>
yes JEc=R, except JE=K yes<br>
yes P=mosq. (D2,3=S) yes?<br>
yes DVc=E, JEc=T yes<br>
var. JEc=K (SLE=N) yes?<br>
yes DVc=var, JEc=G yes<br>
yes "all"mosq=neg D2=G yes<br>
yes DVc, JEc= T yes?<br>
yes all flavis no?<br>
yes "all" flavis no?<br>
JEc=S, SDMWN YES<br>
-yes flavis=K yes?<br>
-yes DVc=M JEc=T ?pos?<br>
YES flavis=S/T YES<br>
YES JEc=T ex JE=S YES<br>
JEc=A SDMinWN YES<br>
JEc=N SDMiaWN YES<br>
JEc=R YES<br>
mosq=G or E YES<br>
* not identified as &gt; 35% SA in this particular structure/model<br>
B-f: p-factor (temperature factor) a qualitative assessment of the scale (5-60A2).<br>
SC?: is the amino acid side chain accessible and available for antibody binding<br>
Ep?: might this amino acid be incorporated into an antigen epitope?<br>
DVc: DENV1-4 complex;<br>
Jec: JE complex (medically important clade =JE, MVE, WN, SLE)<br>
SDM: site-directed mutagenesis<br>
Table 9. Potential JEV complex- and sub-complex cross-reactive epitope residues from SLEV, with residues<br>
chosen for mutagenesis highlighted<br>
D2# SLE# WN# Pom. B-f. Location? SC? Conservation Ep?<br>
T68* L68 L68 DII low top inner edge near N67 D2<br>
T76 T76 T76 DII low out-lateral-low-mid.<br>
Q77 T77* M77 DII med outer lateral middle<br>
N83 T83 D83 DII med out-up-lat. loop near FP<br>
Q86 S86 A86 DII high out-mid-lat.<br>
K88 P88 P88 DII high out-up-lat<br>
K122 K122 SI 22 DII med+ up-mid-top<br>
Q131 L131 L131 DII/DI high out low lat mid<br>
H149* S149 V149 DI high up-mid-top below CHO-153<br>
N153* N154 N154 DI med+ up-mid-top CHO on D2-153<br>
NA* Y155 Y155 DI na up-mid-top near CHO 153 yes<br>
D154 Q158 Q158 DI high up-in-top near CHO &amp; prM<br>
T155* 1159* V159 DI high up-mid-top near prM<br>
E161 R166 R166 DI high up-mid-top high.exp.<br>
SI69 PI74 PI74 DI med+ DI/II border out-mid<br>
E172 T177 T177 DI high DI/II border up-out-top edge<br>
El74 N179 K179 DI high DI/III border outer edge<br>
T176 GI81 G181 DI med DI/III up-out edge<br>
G177* El82 El82 DI med DI/III up-out edge<br>
T226 T231 T231 DII higli up-out-mid. SDM prev.<br>
H244* H246 H246 DII med prM hole low but above P243<br>
K247 K249 K249 DII med prM hole low, above 243/244<br>
S274 S276 S276 DII/I high up-top-in good aa JEc YES<br>
K291 K294 K294 DI med+ out-up lat DI/III face<br>
M297 T300 T300 DIII/I high mid-out-end DI/III<br>
S298 T301 T301 DIII/I med+ mid-out-end DI/III<br>
E327* T330 T330 Dili high up-out-top "end" Dili<br>
K361 A367 A367 Dili high tip-top-mid Dili<br>
D362 N368 N368 Dili med tip-top-mid Dili<br>
V382* R388 R388 Dili low "ROD" loop up-out-lat<br>
E383 G389* G389 Dili high "ROD" loop up-out-lat<br>
-yes WN,SLE=L ?yes<br>
yes "all" mosq. + ticks ?<br>
no variable, Q=DVc ?<br>
yes variable type? Yes<br>
-yes JEc=A(SLE=S) yes<br>
-no? variable ?possible<br>
yes var. yes<br>
-yes var. L=WN,SLE ?/no<br>
yes H=DVc; yes if<br>
no CHO<br>
yes "all mosq"=N YES<br>
Y=JEc SDM SLE YES<br>
yes Q=Jec, SDM SLE YES<br>
yes DVc=TJEc=I/V YES<br>
yes JEc=R, except JE=K yes<br>
yes P=mosq. (D2,3=S) yes?<br>
yes DVc=E,JEc=T YES<br>
yes var. JEc=K (SLE=N) yes?<br>
yes DVc=var, JEc=G yes<br>
yes "all"inosq=neg D2=G yes<br>
yes DVc,JEc=T<br>
all flavis<br>
"all" flavis<br>
SDM SLE<br>
flavis=K<br>
DVc=M JEc=T<br>
flavis=S/T<br>
JEc=T ex JE=S<br>
YES JEc=A, SDM in SLE YES<br>
YES JEc=N, SDM in SLE YES<br>
-yes JEc=R YES<br>
~y/na mosq=G or E YES<br>
as &gt; 35% SA in this particular structure/model<br>
B-f: p-factor (temperature factor) a qualitative assessment of the scale (5-60A2).<br>
SC?: is the amino acid side chain accessible and available for antibody binding<br>
Ep?: might this amino acid be incorporated into an antigen epitope?<br>
DVc: DENV1-4 complex;<br>
Jec: JE complex (medically important clade =JE, MVE, WN, SLE)<br>
SDM: site-directed mutagenesis<br>
Table 10. Inverse Iog10 end-point tilers of anli-SLEV mAbs determined by Ihe AG-ELISA for antigens expressed by wild-type pCB8SJ2 and cross-reactive reduced<br>
mutated conslrucls<br>
Mabs: MHIAF 4G2 T-23-1 T-23-2 6B6C-1 2B6B-2 4A1B-9 1B7-S 2BSB-3 T-16 6B4A-10 1B5D-1<br>
supr supr JE JEV +<br>
6B5A-2 4A4C-4 IB2C-5<br>
Shaded block: Significantly altered endpoints relative to pCB8SJ2 derived wild-type VLP antigens. Most substitutions reduced mAb reactivity, however, some mAbs<br>
reactivity increased.<br>
Table 11. Inverse logic end-point titers of anti-WNV mAbs determined by the AG-ELISA for antigens expressed by wild-type pCBWN and cross-reactive reduced<br>
mutated constructs<br>
Mutants<br>
Shaded block: Significantly altered endpoints relative to pCB8SJ2 derived wild-type VLP antigens. Most substitutions reduced mAb reactivity, however, some mAbs<br>
reactivity increased.<br>
Table 12. Comparative detection of human IgM antibody by MAC-ELISA with wild type (wt-) and<br>
G106V- prototype type-specific antigens.<br>
Serum Specimen Description<br>
Infecting Virii<br>
' Sera were assigned to one of three classes; positive, equivocal, or misleading as described in<br>
materials and methods. Assignments were based upon previously determined P/N ratios3 reported<br>
by the Diagnostics and Reference Laboratory, Arbovirus Diseases Branch, Division of Vector-<br>
Borne Diseases, US Centers for Disease Control and Prevention.<br>
2 PRNT90, Plaque reduction neutralization test; tilers represent inverse 90% plaque reduction<br>
endpoints as reported by the Diagnostics and Reference Laboratory, ADD, DVBID, CDC.<br>
3 Values represent ratios calculated as described in Materials and Methods. Positive ratios &gt;3.0 are<br>
shown in bold<br>
4 Ratios reported by the Diagnostics and Reference Laboratory, ADB, DVBID, CDC.<br>
5 Ratios determined in this study comparing wild-type (wt-) WNV Ag. with prototype cross-reactivity<br>
reduced G106V-WNV Ag.<br>
68<br>
Table 13. Type-specific neutralizing antibody tilers as determined by PRNT<br>
Plasmid DNA used for<br>
immunization1<br><br>
'Mice were immunized intramuscularly with lOOug of plasmid DNA on weeks 0 and 3.<br>
2PRNT plaque reduction neutralization tesl, 75% neulralizalion endpoint tilers on mouse<br>
sera collected 6 weeks post vaccination.<br><br><br><br><br><br>
We Claim:<br>
1.	An isolated mutant flavivirus E-glycoprotein polypeptide that exhibits measurably reduced antibody cross-reactivity as compared to the corresponding wild-type flavivirus E-glycoprotein, said mutant polypeptide comprising at least one amino acid substitution at position 104, 106, or 107 as compared to said corresponding wild-type flavivirus E-glycoprotein polypeptide.<br>
2.	The isolated mutant flavivirus E-glycoprotein polypeptide as claimed in claim 1, wherein the amino acid substitution is selected from the group consisting of G104H, G104N, G106Q, G106V, L107K, L107Y, and a combination of two or more thereof.<br>
3.	The isolated mutant flavivirus E-glycoprotein polypeptide as claimed in claim 1, comprising an amino acid sequence as shown in SEQ ID NO: 14, having at least one amino acid substitution at position 104, 106, or 107 compared to a wild-type flavivirus E-glycoprotein polypeptide.<br>
4.	The isolated mutant flavivirus E-glycoprotein polypeptide as claimed in claim 2, wherein the amino acid substitution is selected from the group consisting of:<br>
G104H; G106Q; L107K; and a combination of two or more thereof.<br>
5.	An isolated nucleic acid molecule encoding a polypeptide as claimed in claim 1.<br>
6.	A recombinant nucleic acid molecule, comprising a regulatory sequence operably linked to the nucleic acid molecule as claimed in claim 5.<br>
7.	A virus-like particle, comprising the polypeptide as claimed in claim 1.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUFic3RyYWN0LSgwMi0wNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Abstract-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUFic3RyYWN0LSgxMC0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Abstract-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUNsYWltcy0oMDItMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Claims-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUNsYWltcy0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Claims-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDItMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Correspondence Others-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Correspondence Others-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW9oZXJzLTEucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-correspondence-ohers-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlLU90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDEwLTA4LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Description (Complete)-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LURyYXdpbmdzLSgxMC0wOC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Drawings-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUZvcm0tMS0oMDItMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Form-1-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUZvcm0tMS0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Form-1-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUZvcm0tMi0oMDItMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Form-2-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUZvcm0tMi0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Form-2-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUZvcm0tMy0oMDItMDUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Form-3-(02-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUZvcm0tMy0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Form-3-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-Form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LUdQQS0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-GPA-(10-08-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LUdQQS5wZGY=" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-GPA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1ERUxOUC0yMDA3LVBDVC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">1310-DELNP-2007-PCT-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LXBjdC0zMjYucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LXBjdC0zNzMucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTMxMC1kZWxucC0yMDA3LVBldGl0aW9uLTEzNy0oMTAtMDgtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1310-delnp-2007-Petition-137-(10-08-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="253276-polyphosphate-derivative-of-a-1-3-5-triazine-compound-a-process-for-the-production-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253278-a-process-for-producing-ferritic-bainitic-dual-phase-steel-sheet-for-wheel-rim-disc-application.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253277</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1310/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>28/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jul-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>10-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>19-Feb-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY OF THE DEPARTMENT OF HEALTH AND HUMAN SERVICES</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGY TRANSFER OFFICE, 4770 BUFORD HIGHWAY (K79) ATLANTA, GA 30341, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GWONGJEN J. CHANG</td>
											<td>4237 BEAVER CREEK DRIVE, FT. COLLINS, CO 80526, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WAYNE D. CRILL</td>
											<td>1121 AKIN AVENUE, FORT COLLINS, CO 80521, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/10, A61K 39/12</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/026672</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-07-27</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/591,898</td>
									<td>2004-07-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253277-an-isolated-mutant-flavivirus-e-glycoprotein-polypeptide-nucleic-acid-molecule-encoding-said-polypeptide-and-its-virus-like-particle by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:55:33 GMT -->
</html>
