<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253211-anti-cd16-binding-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:57:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253211:ANTI-CD16 BINDING MOLECULE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ANTI-CD16 BINDING MOLECULE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to binding molecules that specifically bind to the human Fc gamma receptor expressed on the surface of natural killer (NK) cells and macrophages (i.e. FcgRIIIA), and in particular binding molecules that specifically bind the A form FcgRIII but do not bind to the B form of FcgRIII, as well as to the use of such binding molecules in the diagnosis and treatment of disease. The invention further extends to polynucleotides encoding such binding molecules, host cells comprising such polynucleotides and methods of producing binding molecules of the invention using such host cells.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 <br>
(39 of 1970)<br>
&amp; <br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
ANTI-CD16 BINDING MOLECULES;<br>
AFFIMED THERAPEUTICS AG, A CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF GERMANY, WHOSE ADDRESS IS NEUENHEIMER FELD 582, 69120 HEIDELBERG, GERMANY<br>
THE	FOLLOWING	SPECIFICATION<br>
PARTICULARLY	DESCRIBES	THE<br>
INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED. <br><br>
The present invention relates to binding molecules that specifically bind to the human Fc gamma III receptor expressed on the surface of natural killer (NK) cells and macrophages (i.e. FcgRIIIA), and in particular binding molecules that specifically bind the A form FcgRIII but do not bind to the B form of FcgRIII, as well as to the use of such binding molecules in the diagnosis and treatment of disease. The invention further extends to polynucleotides encoding such binding molecules, host cells comprising such polynucleotides and methods of producing binding molecules of the invention using such host cells.<br>
CD 16 (also known as FcgRIII), a low affinity receptor for the Fc portion of some IgGs known to be involved in antibody-dependent cellular cytotoxicity (ADCC), is the best-characterized membrane receptor responsible for triggering of target cell lysis by NK cells (Mandelboim et al, 1999, PNAS 96:5640-5644). Human NK cells comprise approximately 15% of all lymphocytes and are defined phenotypically by their expression of CD56 and lack of expression of CD3 (Robertson and Ritz, 1990, Blood 76:2421-2438). The majority (approximately 90%) of human NK cells express CD56 at low density (CD56dim) and FcgRIII (CD16) at a high level (Cooper et al, 2001, Trends Immunol. 22:633-640). Human FcgRIII exists as two isoforms, FcgRIIIA and FcgRIIIB, that share 96% sequence identity in their extracellular immunoglobulin-binding regions (van de Winkel and Capel, 1993, Immunol. Today 14(5):215-221).<br>
FcgRIIIA is expressed on macrophages, mast cells, and NK cells as a transmembrane receptor. On NK cells, the alpha chain of FcgRIIIA associates with the immunoreceptor tyrosine-based activation motif (ITAM) containing FceRI y-chain and/or the T-cell receptor (TCR)/CD3 ^-chain to mediate signalling (Wirthmueller et al, 1992, J. Exp. Med. 175:1381-1390). The interaction of FcgRIIIA with different combinations of homo- and hetero-dimers of the y and C, chains has been observed in NK cells, offering the possibility of different signalling pathways via variations of the FcgRIIIA complex in NK cells (Anderson et al, 1990, PNAS 87(6):2274-2278; Ackerly etal, 1992, Int. J. Cancer Suppl. 7:11-14).<br>
FcgRIIIB is present on polymorphonuclear granulocytes (PMN) as a glycosyl-phosphatidylinositol (GPI)-anchored receptor (FcyRIIIB isoform), which cannot trigger tumour cell killing (van de Winkel and Capel, 1993, supra). In addition, FcyRIIIB exists<br><br><br>
as a soluble receptor in serum, and upon binding to antibodies may trigger side-effects via the formation of immune complexes in vivo.<br>
FcgR-expressing effector cells have been shown to be involved in destroying tumour cells via ADCC. This has lead to the development of several immunotherapeutic approaches to cancer therapy, which involve the use of FcgR activities, for example, tumour specific monoclonal antibodies can mediate destruction of tumour cells by ADCC induced via binding to FcgRs, and bi-specific molecules with one specificity for tumour cells and the other specificity for either an FcgRI on granulocytes or FcgRIII on NK cells have also been developed in efforts to improve effector cell recruitment (van Spriel et al, 2000 Immunol. Today, 21:391-397).<br>
As cellular mediators of innate immunity, NK cells are professional cell killers and, unlike T cells, tend to exist in constitutively activated states not requiring additional (pre-)stimulation. In contrast, resting cytotoxic T-cells require activation via the binding of an antigen-MHC complex and subsequent co-stimulation via CD28. Thus, amongst the immune effector cells, the NK cell is one of the more attractive candidates for use in immunotherapy, and bi-specific antibodies with specificity to CD 16 (FcgRIII) and HER-2/neu or CD16 and CD30 have been developed (Arndt et al, 1999 Blood 94:2562-2568; McCall etal, 2001 J. Immunol. 166:6112-6117).<br>
The majority of anti-CD 16 antibodies that have been produced to date, are unable to distinguish between the two CD 16 isoforms, FcgRIIIA and FcgRIIIB, with the exceptions being monoclonal antibody (MAb) 1D3, which only binds to the form of CD 16 expressed on neutrophils i.e. FcgRIIIB, and the monoclonal antibodies which recognise different allotypic forms of FcgRIIIB (Ory et al. 1989, J. Clin. Invest. 83:1676-81; Huizinga et al. 1990 Blood 75:213-7; Tamm &amp; Schmidt 1996, J. Immunol. 1996 157:1576-1581). However, to date, there have been no examples of molecules that bind specifically to FcgRIIIA, the form expressed on NK cells, and not to FcgRIIIB.<br>
An antibody specific for FcgRIIIA, however, would be particularly useful as a component of a bi- or multispecific binding molecule that is directed against disease-associated cells, as it would mainly recruit NK cells, and would not be bound by circulating soluble FcgRIIIB or diverted from NK cell binding by binding to neutrophils or activated eosinophils. To efficiently mediate NK-cell killing, the FcgRIIIA-specific antibody must not only bind NK cells but also activate them upon binding.<br><br><br>
We have identified several new anti-CD 16 scFvs, which exhibit this useful property, i.e. they have specificity for FcgRIIIA, they do not bind with specificity to FcgRIIIB and they are also able to activate NK cells upon binding.<br>
It is also known that FcgRIIIA exhibits several allelic polymorphisms, in particular at positions 48 and 158 of the IgG binding domain. Previously described anti-CD16 antibodies have been shown to bind with different affinities to the different polymorphic forms of FcgRIIIA and work by Koene et al. (1997, Blood 90:1109-1114) attributes the difference in binding of some of these antibodies to the polymorphism at position 158. Since the gene frequencies of the FcgRIIIA158F allele and of the FcgRIIIA158V allele are approximately 0.6 and 0.4 respectively, it would be advantageous for any anti CD16A antibody to recognize both allelic forms, to ensure that in any given patient the anti-CD 16A antibody would be capable of binding and activating the NK cell population.<br>
Described herein is an anti-CD 16A scFv which recognizes both FcgRIIIA158v and FcgRIIIA158F with approximately equal affinity, but which does not bind to FcgRIIIB.<br>
The new scFvs described herein may thus be used to produce novel binding molecules, which may be used as therapeutic agents. The term "binding molecule" as used herein encompasses antibodies and antigen-binding fragments thereof, wherein said antibodies or antigen binding fragments selectively and specifically bind to FcgRIIIA and do not bind with specificity to FcgRIIIB, as well as fusion proteins of such antibodies or antigen-binding fragments, and protein conjugates comprising such antibodies or antigen-binding fragments.<br>
Thus according to the first aspect of the invention, there is provided a binding molecule having specificity for at least one antigen, wherein the at least one antigen is FcgRIIIA and wherein the binding molecule does not specifically bind to FcgRIIIB.<br>
In a preferred embodiment, the specificity for FcgRIIIA is provided by an antibody or antigen-binding fragment thereof. Accordingly, binding molecules of the invention may comprise, or in certain embodiments consist of, at least one antibody (or an antigen binding fragment thereof) which has specificity for FcgRIIIA but not FcgRIIIB.<br>
Antibodies conferring specificity to FcgRIIIA may be naturally occurring antibodies, or they may be recombinant antibodies. Antibodies of the invention may be of any immunoglobulin class, e.g. they may be an IgA, IgD, IgE, IgG or IgM antibody.<br><br><br>
Preferably the antibody will be an IgG antibody. Antigen-binding fragments suitable for use in the invention may be naturally occurring or recombinant and include immunoglobulin light chains, immunoglobulin heavy chains, VH domains, VL domains, Fvs, single chain antibodies (including scFvs), Fabs, di-Fabs, Fab's, F(ab')2S, and CDRs. The skilled man will be very familiar with such antigen-binding fragments and their preparation. For the avoidance of doubt the term single chain antibody as used herein refers to a genetically engineered single chain molecule comprising the variable region of a light chain and the variable region of a heavy chain, linked by a suitable flexible polypeptide linker. Examples of single chain antibodies include a scFv and a diabody.<br>
In one embodiment, the antibody or antigen-binding fragment conferring specificity to FcgRIIIA will comprise at least one human CDR, preferably selected from the CDRs described in Table 1 below.<br>
Table 1. Amino acid sequences of human light and heavy chain CDRs from scFvs that selectively and specifically bind to FcgRIIIA and do not bind with specificity to FcgRIIIB<br><br>
Chain<br><br>
CDR1<br><br>
CDR2<br><br>
CDR3<br><br><br><br>
1.	Heavy Chain<br>
2.	Light Chain<br>
3.	Light Chain<br>
4.	Light Chain<br>
5.	Light Chain<br>
6.	Light Chain<br>
7.	Light Chain<br>
8.	Light Chain<br><br>
TSYYMH (SEQIDN0 17) SGDKLEEKYVS (SEQ ID NO 20) GGNNIESRNVH (SEQ ID NO 23) GGNNIGSKNVH (SEQ ID NO 26)<br>
EGNNIGSKNVH (SEQ ID NO 29) GGNNIGSKNVH (SEQ ID NO 26)<br>
GANDIGKRNVH (SEQ ID NO 33) GGHNIGSKNVH (SEQ ID NO 34)<br><br>
IINPSGGSTSYAQKFQG<br>
(SEQ ID NO 18)<br>
QDNKRPS<br>
(SEQ ID NO 21)<br>
RDNNRPS<br>
(SEQ ID NO 24)<br>
RDSNRPS<br>
(SEQ ID NO 27)<br>
DDSDRPS (SEQ ID NO 30) RDSSRPS<br>
(SEQ ID NO 31)<br>
QDNKRPS<br>
(SEQ ID NO 21)<br>
QDNKRPS<br>
(SEQ ID NO 35)	<br><br>
GSAYYYDFADY (SEQ ID NO 19) QVWDNYSVL (SEQ ID NO 22) QVWDNYTVL (SEQ ID NO 25) QVWDNYIVL<br>
(SEQ ID NO 28) QVWDNYSVL (SEQ ID NO 22) QVWDDYIW (SEQ ID NO 32)<br>
QVWDNYSVL (SEQ ID NO 22) QVWDNYSVL (SEQ ID NO 22)<br><br>
The skilled man will appreciate that antibodies and antigen-binding fragments thereof, with the exception of single CDRs, in general comprise three light chain CDRs and/or three heavy chain CDRs. Accordingly in further embodiments, the antibody or antigen binding fragment with specificity for FcgRIIIA comprises three human light chain CDRs and/or three human heavy chain CDRs. Preferably the human light chain CDR 1 will have an amino acid sequence selected from SEQ ID NO 20, SEQ ID NO 23, SEQ ID NO 26, SEQ ID NO 29, SEQ ID NO 33, and SEQ ID NO 34, the human light chain CDR 2<br><br><br>
will have an amino acid sequence selected from SEQ ID NO 21, SEQ ID NO 24, SEQ ID NO 27, SEQ ID NO 30, SEQ ID NO 31 and SEQ ID NO 35 and the human light chain CDR3 will have an amino acid selected from SEQ ID NO 22, SEQ ID NO 25, SEQ ID NO 28 and SEQ ID NO 32. Preferably the human heavy chain CDR1 will have the amino acid sequence of SEQ ID NO 17, the human heavy chain CDR2 will have the amino acid sequence of SEQ ID NO 18 and the human heavy chain CDR3 will have the amino acid sequence of SEQ ID NO 19.<br>
Where the antibody or antigen-binding fragment comprises or consists of the variable domain of a light chain it may have one of the following amino acid sequences: SEQ ID NO 10:<br>
SYELMQPPSVSVSSGQTASIPCSGDKLEEKYVSWYQQRPGQSPVLVIYQDNKRPS GIPERFSGSNSGNTATLTISGTQAMDEADYYCQVWDNYSVLFGGGTKLTVL; SEQ ID NO 11:<br>
SYELTQPLSESVAQGQTARITCGGNNIESRNVHWYQQKPGQAPVLVIYRDNNRP SGIPERFSGSNSGNMATLTISRAQAGDAADYYCQVWDNYTVLFGGGTKLTVL; SEQ ID NO 12:<br>
SYELTQPPSVAVAPGKTARITCGGNNIGSKNVHWYQQKPGQAPVLVIYRDSNRP SGIPERFSGSNSGNTATLTISRAQAGDEADFYCQVWDNYIVLFGGGTKLTVL; SEQ ID NO 13:<br>
QAVLTQPPSVSVAPGQTARIPCEGNNIGSKNVHWYRQKPGQVPVLVMYDDSDR PSGIPERFSGSNSGNTATLTISGTQAMDEADYYCQVWDNYSVLFGGGTKLTVL; SEQ ID NO 14:<br>
QPVLTQPLSVSVAPGQTARITCGGNNIGSKNVHWYQQKPGQAPVLVIYRDSSRP SGIPERLSGSNSGDTATLTISRAQAGDEADYYCQVWDDYIVVFGGGTKLTVL; SEQ ID NO 15:<br>
SYELTQPPSVSVTPGQTATITCGANDIGKRNVHWYQQRPGQSPVLVIYQDNKRPS GIPERFSGSNSGNTATLTISGTQAMDEADYYCQVWDNYSVLFGGGTKLTVL; SEQ ID NO 16:<br>
QPVLTQPSSVSVAPGQTATISCGGHNIGSKNVHWYQQRPGQSPVLVIYQDNKRP SGIPERFSGSNSGNTATLTISGTQAMDEADYYCQVWDNYSVLFGGGTKLTVL.<br>
Where the antibody or antigen-binding fragment comprises or consists of the variable domain of a heavy chain, it may have the following amino acid sequence (SEQ ID NO 9): EVQLVQSGAEVKKPGESLKVSCKASGYTFTSYYMHWVRQAPGQGLE<br><br><br>
WMGIINPSGGSTSYAQKFQGRVTMTRDTSTSTVYMELSSLRSEDTAVYYCARGS AYYYDFADYWGQGTLVTVSS.<br>
In a preferred embodiment, the antibody or antigen-binding fragment that provides specificity to FcgRIIIA will be fully human.<br>
In further preferred embodiments the binding molecule of the invention will be capable of activating human cells expressing FcgRIIIA upon binding to the FcgRIIIA expressed on such cells, i.e. the binding molecule will be capable or eliciting a biological response, in particular a signaling response, upon binding to FcgRIIIA. Preferably the binding molecule will also be capable of triggering cell killing, in a manner analogous to antibody-dependent cellular cytotoxicity (ADCC), by virtue of its binding to such cells.<br>
In order to test whether binding molecules of the invention are capable of activating human cells expressing FcgRIIIA, they may be tested for their ability to elicit calcium mobilization in peripheral blood NK cells, for example as described herein in the Examples. The ability of binding molecules of the invention to trigger NK-cell mediated cytotoxicity may also be tested as described herein in the Examples, using for example a calcein-release assay to detect re-directed cell lysis using freshly isolated NK cells.<br>
The present invention also provides binding molecules that not only have specificity for FcgRIIIA whilst lacking specificity for FcgRIIIB, but which also have specificity for one or more additional antigens. Thus binding molecules of the invention may be multispecific. The term multispecific as used herein means that a binding molecule of the invention, whilst lacking specificity for FcgRIIIB, has at least two specificities, one of which is for FcgRIIIA. Multispecific binding molecules of the invention thus encompass at least bi-specific, tri-specific, and tetra-specific molecules, such as for example, bi-specific antibodies including single-chain diabodies, (scFv)2, tandem diabodies (TandAbs); tri-bodies (tri-specific antibody); tetra-bodies (tetraspecific antibody); and flexibodies (see Le Gall et al. 1999 FEBS Letts 453:164-168 and WO 03/025018).<br>
By virtue of their multispecificity, binding molecules according to this aspect of the invention may target macrophages or NK cells to further antigens or even to cells bearing such antigens, so that the antigen or cell bearing the antigen may be eradicated via phagocytosis or NK cell mediated cell killing. For example, a binding molecule of the invention may have additional specificity for one or more tumor specific antigens and can thus target NK-cells to the tumor cell. Activation of the NK cell caused by the binding molecule binding to FcgRIIIA will lead to killing of the tumor cells.   In this<br><br><br>
manner binding molecules of the invention may be used to treat and/or eliminate diseased cells and infectious agents (such as for example, bacteria, fungi, mycoplasmas, viruses, parasites, prions), depending on the additional specificity or specificities of the binding molecule.<br>
Thus in a further aspect the invention provides a binding molecule as described hereinbefore, wherein the binding molecule has specificity for at least one additional antigen. In one embodiment the at least one additional antigen is a cell-surface antigen. Examples of cell surface antigens to which binding molecules according to this embodiment of the invention exhibit specificity are CD19, CD20, CD30, the laminin receptor precursor protein (also known as the onco-foetal antigen immature laminin receptor OFA-iLR), EGFR1 (also known as HER-1, ErbBl), EGFR2 (also known as HER-2, ErbB2, neu), EGFR3 (also known as HER-3), Ep-CAM, PLAP (placental alkaline phosphatase), Thomsen-Friedenreich (TF) antigen, MUC-1 (mucin), IGFR, IL4-R alpha, IL13-R, FcsRI and CD5.<br>
In a further embodiment, the at least one additional antigen is an antigen from an infectious agent, for example, the antigen may be secreted from or expressed on the outer surface of a host cell, infected cell, bacterium, fungus, mycoplasma, parasite or virus, or it may be released upon lysis of such an infectious agent. For example, a binding molecule of the invention may have additional specificity for a viral protein such as gp34 or gp68 on the surface of CMV-infected cells, or for haemaglutinin on the presence of the influenza virus, and may thus target either a virally infected cell or the virus directly. Alternatively, where the infectious agent is a prion, the prion protein per se may be the antigen.<br>
In a further embodiment the at least one additional antigen is an allergen, or a molecule involved in the response of the body to an allergen, such as for example circulating IgE, or the FceRI receptor on (in particular) mast cells.<br>
Preferably, in binding molecules according to this aspect of the invention, the specificity for the at least one additional antigen is provided by an antibody or an antigen-binding fragment thereof. As described above, the skilled man will be familiar with the types of antibodies (IgA, IgD, IgE, IgG and IgM) and with antigen-binding fragments derived from such antibodies (for example, immunoglobulin light chains, immunoglobulin heavy chains, VH domains, VL domains, Fvs, scFvs, Fabs, di-Fabs, Fab's, F(ab')2S, and CDRs) which may also be used in this aspect of the invention. In one preferred embodiment the antibody will be an IgG antibody.<br><br><br>
Multispecific, and in particular bispecific, binding molecules of the invention may thus have any of the following formats: bispecific IgGs, IgG-scFv2, (scFv)4-IgG, (Fab')2, (scFv)2, (dsFv)2, Fab-scFv fusion proteins, (Fab-scFv)2, (scFv)2-Fab, (scFv-CH2-CH3-scFv)2, bibody, tribody, bispecific diabody, disulfide-stabilized (ds) diabody, 'knob-into-whole' diabody, single-chain diabody (scDb), tandem diabody (TandAb), flexibody, DiBi miniantibody, [(scFv)2-Fc]2, (scDb-CH3)2, (scDb-Fc)2, Di-diabody, Tandemab (reviewed in Kipriyanov &amp; Le Gall, 2004, Curr. Opin. Drug Discov. Devel., 7:233-242 and Marvin and Zhu, 2005, Acta Pharmacol. Sin., 26:649-658). In preferred embodiments the binding molecule will be TandAb or Flexibody. Unlike many other bispecific antibody formats, the TandAb is a homodimer comprising only antibody variable domains, and its formation is determined by the association of complementary VH and VL domains located on different polypeptide chains. The TandAb is twice the size of the diabody and (scFv)2, is able to bind bivalently both effector and target cells and possesses improved pharmacokinetic characteristics relative to the diabody and (scFv)2 e.g. it has greater stability and enhanced biological activity both in vitro and in vivo (Kipriyanov et al, 1999, J. Mol. Biol., 293:41-56; Cochlovius et al, 2000, Cancer Res., 60:4336-4341; Reusch et al, 2004, Int. J. Cancer, 112:509-518.). The interchain pairing of the cognate VH and VL domains of the same specificity could be also used for the formation of bispecific scFvB-diabodyA-scFvB tandem molecules, so-called 'flexibodies' (A and B may have the same or different specificity). Depending on the tightness of association of complementary domains involved into the interchain pairing and on the length of the linker separating them, dimers, trimers and even tetramers of bispecific single-chain molecules may be formed.<br>
Examples of antibodies and antigen-binding fragments, which may be used in this aspect of the invention and which exhibit specificity for CD 19, CD20, CD30, the laminin receptor precursor protein, EGFR1, EGFR2, EGFR3, Ep-CAM, PLAP, Thomsen-Friedenreich (TF) antigen, MUC-1 (mucin), IGFR, CD5, IL4-R alpha, IL13-R, FcsRI and IgE, are described in the art, as are antibodies to infectious agents (see for example, EP-A-1314741, Reff et al. 1994 Blood 83:435-445, EP-A-1005870, WO 01/92340, US 6,033,876, WO 86/03223, Buto et al. 1997 Int. J. Biol. Markers 12:1-5, US 6,699,473, WO 98/50433, US 5,770195, EP-A-0502812, Carter et al. 1992 PNAS 89:4285-4289, US 5,968,511, WO 04/106383, Nouri et al 2000 BJU Int. 86:894-900, EP-A-0429242, Ravn et al. 2004 J. Mol. Biol. 343:985-996, Dahlenborg et al. 1997 Int. J. Cancer 70:63-71, WO 88/05054, WO 02/053596, WO 04/071529, Studnicka et al. 1994 Protein Eng.<br><br><br>
7:805-814, Presta et al. 1993 J. Immunol. 151:2623-2632). Where these citations provide amino acid sequences of, or nucleic acid sequences encoding, such antibodies or antigen-binding fragments, the skilled man will be able to utilize such sequences directly in the production of binding molecules of the invention. Where these citations describe hybridomas or cell lines producing such antibodies or antigen-binding fragments, the skilled man will be able to obtain the nucleic acid sequences encoding the desired antibody or antigen-binding fragment from the appropriate hybridoma or cell line using standard molecular biology techniques, and once obtained the nucleic acid sequences may be used in the construction of binding molecules of the invention.<br>
A further aspect of the invention provides for a binding molecule according to any one of the aspects or embodiments described hereinbefore, which is capable of binding to the FcgRIIIA158F allele and the FcgRIIIAl58V allele with approximately the same affinity, i.e. the affinity of the binding molecule for the two 158 alleles of FcgRIIIA differs by no more than two-fold. In preferred embodiments according to this aspect of the invention the specificity for FcgRIIIA will be provided by an antibody or antigen binding fragment derived from the 4-LS21 scFv described herein. In particularly preferred embodiments binding molecules according to this aspect of the invention will be in a TandAb format and will comprise antigen binding fragments derived from the 4-LS21 scFv described herein as well as antigen binding fragments having specificity for CD19 or CD30.<br>
The skilled man will appreciate that where it is intended that binding molecules of the invention are to be used in the treatment and/or therapy of humans, the potential immunogenicity of the binding molecule should be minimized. Accordingly, where the specificity for FcgRIIIA and/or the additional antigen(s) is(are) provided by an antibody or antigen-binding fragment, it is preferred that the antibody or antigen-binding fragment is chimeric, CDR-grafted, humanized or fully human. More preferably the antibody or antigen-binding fragment will be humanized or fully human. The use of standard molecular biological techniques to effect the humanization of antibodies is now routine in the art, and provided with a non-human antibody sequence, the skilled man would readily be able to produce a humanized version of such an antibody. The term "fully human" as used herein insofar as it relates to an antibody or antigen-binding fragment means that the amino acid sequences of the antibody or antigen-binding fragment are derived from (i.e. originate or may be found in) humans.<br><br><br>
Another technique which may be employed either as an alternative, or in addition, to the methods described above for reducing immunogenicity, is that of deimmunisation. Deimmunisation technology involves the identification and removal of T helper (Th) cell epitopes from antibody and other protein biological therapeutic agents. Th cell epitopes comprise short peptide sequences within proteins that have the capacity to bind to MHC class II molecules. The peptide-MHC class II complexes can be recognized by T cells and can trigger the activation and differentiation of Th cells, which is required to initiate and sustain immunogenicity through interaction with B cells, thus resulting in the secretion of antibodies that bind specifically to the administered biological therapeutic agent. For antibody deimmunisation, the Th-cell epitopes are identified within the antibody sequence, for example by a computer-based method for predicting the binding of peptides to human MHC class II molecules (see Altuvia et al, 1995, J. Mol. Biol., 249:244-2450, and Schueler-Furman et al, 2000, Protein Sci., 9:1838-1846). To avoid recognition by T cells, the Th cell epitopes thus identified are eliminated from the protein sequence by amino acid substitutions. This may be achieved through the use of standard molecular biology techniques, such as for example site-directed mutagenesis to alter the nucleic acid sequence encoding the Th cell epitope in the therapeutic protein. In this way, an antibody or antigen-binding fragment may be modified so that HAMA (Human anti mouse antigenic) and/or anti-idiotypic response(s) are reduced or avoided (Bander et al, 2003, Semin. Oncol., 30:667-676; Nanus et al, 2003, J. Urol., 170:S84-S89). Thus in further embodiments, binding molecules of the invention are modified to remove any Th cell epitopes present in their sequence. Such binding molecules are referred to herein as deimmunised binding molecules.<br>
In a further aspect, binding molecules of the invention comprise a further functional domain. This functional domain may confer effector properties on the binding molecule, for example, the functional domain may be a FcR-binding peptide or the Fc domain of an antibody which, through their capability to bind to Fc receptors, confer effector properties on the binding molecule. Alternatively, the functional domain may be an enzyme that is capable of converting a pro-drug to an active drug. In this way binding molecules of the invention may be used in antibody-dependent enzyme pro-drug therapy (ADEPT). In a further embodiment, the functional domain is a protein or peptide that confers an increased serum half-life on the binding molecule. An example of such a protein is serum albumin or the Fc portion of IgG, which may increase serum half-life of<br><br><br>
the binding molecule by virtue of its ability to bind to FcgRn (the neonatal Fc receptor). In a further embodiment the functional domain may be a cytokine.<br>
In a further aspect, binding molecules of the invention may be conjugated to a labeling molecule, or a toxin. Where the labeling molecule or toxin is a protein, conjugation to the binding molecule may occur through a peptide bond or through chemical conjugation. Thus a binding molecule according to this aspect of the invention may be in the form of a fusion protein where the labeling molecule or toxin is linked to the binding molecule by a peptide bond, preferably by a peptide linker, or it may be in the form of a chemical conjugate. For the avoidance of doubt, the term conjugation is used herein to mean that two components are physically linked together via a chemical bond, which includes a peptide bond (thus conjugates include fusion proteins), ester linkage, or disulphide bridge.<br>
Conjugation of a binding molecule of the invention to a labeling molecule, such as a radiolabel or a fluorescent or luminescent (including chemiluminescent) label allows the binding molecule to be used as an immunological staining reagent. Such a reagent may be used in detecting, for example, tissue-infiltrating NK cells, macrophages or mast cells expressing FcgRIIIA, or, where the binding molecule exhibits specificity for an additional antigen, in detecting NK-cell-binding molecule-additional antigen complexes. Detection of the latter may be particularly useful in the diagnosis of disease or in the monitoring of disease progression or remission.<br>
Where a binding molecule of the invention is conjugated to a toxin molecule, such as a ribosyl transferase, serine protease, guanyl cylcase activator, calmodulin-dependent adenyl cylcase, ribonuclease, DNA alkylating agent or mitosis inhibitor (e.g. doxorubicin) and the like, it may be used to target and kill NK cells and macrophages. Such a binding molecule may thus be used as an immunosuppressive agent. The skilled man will appreciate that in this aspect of the invention, it is preferred that the binding molecule exhibits specificity for only one antigen, i.e. FcgRIIIA. Suppression of the immune system may also be achieved using binding molecules comprising or consisting of a monovalent antigen binding fragment which binds selectively and specifically to FcgRIIIA. Such monovalent binding molecules are preferred as they will act as molecules that block the FcgRIIIA receptor, will not be cross-linked and will thus not activate the NK-cell or macrophage. Preferred monovalent anti-FcgRIIIA antigen-binding fragments include, scFv, Fab, VH, VL and a CDR.<br><br><br>
The skilled man will appreciate that as an alternative to being tagged with a labeling molecule or toxin via chemical conjugation, peptide labels or peptide toxins may also be used. For example, the binding molecule of the invention may be expressed as a fusion protein with an N- or C-terminal peptide tag such as a tetra- penta- or hexa-histidine tag, a c-myc tag or the like.<br>
Binding molecules of the invention may be produced by expressing nucleic acids encoding them in any suitable protein expression system. Thus in a further aspect the invention provides polynucleotides encoding binding molecules of the invention. Polynucleotides that are particularly suitable for use in this aspect of the invention will encode the CDRs described in Table 1 above. In particular, the skilled man armed with the amino acid sequence of the light and heavy chain CDRs from table 1, will be able to derive polynucleotide sequences encoding the various light and heavy chain CDRs from the following nucleic acid sequences: <br>
SEQ ID NO l:<br>
gaggtccagctggtacagtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacac cttcaccagctactatatgcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtgg tagcacaagctacgcacagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagctt agcagcctgagatctgaggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggcca gggaaccctggtcaccgtctcctca;<br>
SEQ ID NO 2:<br>
tcctatgagctgatgcagccaccctcagtgtccgtgtcctcaggacagacagccagcatcccctgctctggagataaattggag<br>
gaaaaatatgtttcctggtatcaacagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcagggat<br>
ccctgagcgattctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctgact<br>
actattgtcaggtgtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 3:<br>
tcctatgagctgacacagccactctcagagtcagtggcccagggacagacggccaggattacctgtgggggaaacaacattga aagtagaaatgttcactggtaccagcagaagccaggccaggcccctgtgttggtcatctatagggataacaaccggccctctgg gatccctgagcgattctctggctccaattcggggaacatggccaccctgaccatcagcagagcccaagccggggatgcagctg actattactgtcaggtgtgggacaactacactgtgctattcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 4:<br><br>
tcctatgagctgacacagccaccctcagtggcagtggccccaggaaagacggccaggattacctgtgggggaaacaacattg gaagtaaaaatgtgcactggtaccagcagaagccaggccaggcccctgtgctggtcatctatagggatagcaaccggccctct gggatccctgagcgattctctggctccaactcggggaacacggccaccctgaccatcagcagagcccaagccggggatgag gctgacttttattgtcaggtgtgggacaactatattgtgctgttcggcggagggaccaagctgaccgtcctg<br>
SEQ ID NO 5:<br>
caggctgtgctgactcagccgccctcagtgtcagtggccccaggacagacggccaggattccctgtgagggaaacaacattg gaagtaaaaatgtccactggtatcggcagaagccaggccaggtccctgtcctggtcatgtatgatgatagcgaccggccctcag ggatccctgagcgattctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggct gactactattgtcaggtgtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 6:<br>
cagcctgtgctgactcagccactctcagtgtcagtggccccgggacagacggccaggattacctgtgggggaaacaacattgg aagtaaaaatgtgcactggtaccagcagaagccaggccaggcccctgtactggtcatctatagggacagcagccggccctctg ggatccctgagcgactctctggctccaactcgggggacacggccaccctgaccatcagcagagcccaggccggggatgagg ctgactattactgtcaggtgtgggacgactacattgtggtcttcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 7:<br>
tcctatgagctgacacagccaccctcggtgtcagtgaccccaggacagacggccacgattacctgcggggcaaacgacattg gaaaaagaaatgtccactggtaccaacagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcag ggatccctgagcgattctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggct gactactattgtcaggtgtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 8:<br>
cagcctgtgctgactcagccatcctcggtgtcagtggccccaggacagacggccacgatctcctgtgggggacacaacattgg gagtaaaaatgtgcactggtaccagcagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcagg gatccctgagcgattctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctg actactattgtcaggtgtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtccta<br>
SEQ ID NO 1 encodes the heavy chain variable region from a human scFv that selectively and specifically binds to FcgRIIIA but which does not bind with specificity to FcgRIIIB. SEQ ID NOs 2-8 encode light chain variable regions from human scFvs that selectively and specifically bind to FcgRIIIA, but which do not bind with specificity to<br><br><br>
FcgRIIIB. Accordingly, SEQ ID NOs 1 to 8 are also examples of polynucleotides of the invention, and may be used in producing further binding molecules of the invention.<br>
Polynucleotides encoding multispecific binding molecules of the invention, in particular bi-specific binding molecules, may be constructed as described in WO 03/025018, using nucleic acids encoding the VH and VL domains of an antibody which binds selectively and specifically to FcgRIIIA but which does not bind specifically to FcgRIIIB, in combination with nucleic acids encoding the VH and VL domains of an antibody having specificity for the additional antigen.<br>
In order for a binding molecule to be expressed in a suitable protein expression system the polynucleotide encoding the binding protein will be operably linked to a suitable promoter, and optionally a suitable transcription terminator. This may be achieved by introducing a polynucleotide of the invention into the genome of a host, and utilising a promoter present within the genome to drive expression of the polynucleotide. Alternatively, a polynucleotide of the invention may be introduced into host cells on a vector comprising the polynucleotide operably linked to promoter and optionally a terminator region. Such vectors form yet a further aspect of the invention.<br>
In general, the promoter to which the polynucleotide encoding the binding molecule is operably linked in the vector, will be any promoter capable of driving expression of the binding molecule in the host cell into which the polynucleotide and vector are to be introduced. Thus, if it is desired that the binding molecule be expressed in a bacterial cell, the promoter will be operable in that bacterial cell. Similarly if it is desired that the binding molecule be expressed in a fungal expression system the promoter will be operable in a fungal cell and the same logic prevails if the construct is to be introduced into a mammalian cell culture expression system, an insect cell expression system or a plant expression system. Where polynucleotides of the invention are operably linked to a suitable transcriptional terminator region, this will be one that mediates the termination of transcription in the host cell in which the binding molecule of the invention is to be expressed. Transcriptional terminator regions suitable for this purpose are described in the art.<br>
In preferred embodiments, the polynucleotide of the invention may also be operably linked to a signal sequence, which is capable of directing expression of the binding molecule to a desired cellular or extracellular location. For example, in some embodiments it may be desirable to secrete the binding molecule from the host cell in which it is expressed. Where this is the case, the signal sequence will encode a secretion<br><br><br>
signal. Suitable bacterial secretion signals are described in the art and one such example is the PelB leader sequence. In other embodiments, for example, those involving expression in eukaryotes, it may be desirable to retain expression of the binding molecule in the endoplasmic reticulum (ER). In this case the signal sequence will comprise an ER retention sequence encoding the "KDEL" (SEQ ID NO 54) motif, and in particular encoding the sequence "SEKDEL" (SEQ ID NO 55).<br>
Polynucleotides of the invention may also be codon optimised according to methods readily available in the art, with the codon bias being altered to suit the particular expression host chosen.<br>
Polynucleotides and vectors of the invention may be introduced into host cells using any appropriate methodology. In eukaryotic cells such techniques include calcium phosphate transfection, DEAE-Dextran, electroporation, particle bombardment, liposome-mediated transfection or transduction using retrovirus, adenovirus or other viruses, such as vaccinia or, for insect cells, baculovirus. In bacterial cells, suitable techniques may include calcium chloride transformation, electroporation or transfection using bacteriophage. In plant cells suitable techniques include Agrobacterium mediated transformation, electroporation, microinjection of plant cells and protoplasts, microprojectile bombardment, bacterial bombardement in particular the "fibre" or "whisker" method, depending upon the particular plant species being transformed. Thus in a further aspect the invention provides host cells comprising polynucleotides or vectors of the invention. Host cells from the organisms described below and which comprise polynucleotides or vectors of the invention are thus to be considered as specific embodiments of this aspect.<br>
Suitable expression systems for expression of binding molecules of the invention include microbial expression systems such as bacterial (e.g. E. coli, Bacillus expression systems) and fungal systems e.g. yeasts (such as, for example, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Pichia species, Hanensula species), and other fungal expression systems (e.g. filamentous fungal expressions systems derived from Aspergillus species, Trichoderma reesei, and Neurospora crassa); mammalian expression systems e.g. CHO cells, NS0 cells, HEK 293 cells; insect cell expression systems and plant expression systems. Safflower is particularly preferred as a plant expression system. The skilled man will be familiar with these expression systems, which are described fully in the art.<br><br><br>
The present invention also provides methods for the production of binding molecules of the invention. Thus in a further aspect there is provided a method for the production of a binding molecule as described herein, wherein said method comprises introducing into a host cell a polynucleotide or vector of the invention and culturing said host cell under conditions whereby the binding molecule is expressed. Conditions for the growth and maintenance of host cells, and for mediating the expression of binding molecules of the invention from such host cells, are described fully in the art. In one embodiment the method will further comprise the isolation and optionally further purification of the binding molecule.<br>
Where a binding molecule of the invention comprises a labelling molecule or a toxin molecule that has been chemically conjugated to the binding molecule, the conjugated binding molecule may be produced by introducing a polynucleotide or vector of the invention into a host cell, culturing the host cell under conditions whereby a binding molecule of the invention is expressed, isolating the expressed binding molecule and conjugating the isolated binding molecule with an appropriate label or toxin. Typically the conjugation step will involve the use of heterobifunctional agents which result in disulphide or thioether linkages as described in the art (see for example, Hermanson, G.T. "Bioconjugate Techniques" Academic Press, London, 1996, for standard methodologies relating to the use of cross-linking reagents).<br>
The skilled man will appreciate that with some embodiments of the invention it may be desirable to increase the circulatory half-life if the binding molecule in the body. This may be achieved in various ways, for example, through constructing the binding molecule as a fusion protein with, for example, albumin, as described hereinbefore. Further methods of increasing serum half-life of a molecule are well known and include for example, pegylation, sialylation, and glycosylation. Accordingly the present invention encompasses binding molecules as described herein, which have been pegylated, sialylated or glycosylated.<br>
The present invention also comprises compositions comprising a binding molecule and at least one additional component. For example, the skilled man will appreciate that in isolating binding molecules from host cells post expression, compositions may be obtained which comprise substantially purified binding molecule but which also contain an impurity, a diluent, a buffer or the like. Such intermediate compositions may be directly suitable for further use, or the binding molecule may be further purified therefrom.<br><br><br>
Compositions of the invention may also comprise a binding molecule of the invention in combination with any suitable pharmaceutically acceptable carrier, excipient, diluent and/or stabiliser. The skilled man will be familiar with such components, which are described in the art. Preferably such a composition will be in any form suitable for administration to a patient, in particular in a form suitable for parenteral administration, for example by injection or infusion.<br>
Where the composition is for injection or infusion it may take the form of a suspension, solution or emulsion in an oily or aqueous vehicle and it may contain formulatory agents such as suspending, preservative, stabilising and/or dispersing agents. Alternatively the binding molecule or composition may be in dry form, for reconstitution before use with an appropriate sterile liquid.<br>
Binding molecules and compositions of the invention, in particular multispecific binding molecules and compositions containing the same, may be used in the diagnosis and/or treatment of disease, such as for example, autoimmune disease, inflammatory disease, infectious disease (including graft-versus-host disease), allergy and cancer (e.g. non-Hodgkin's lymphoma; chronic lymphocytic leukaemia; Hodgkin's lymphoma/disease; solid tumours e.g. those occurring in breast cancer, ovarian cancer, colon cancer, cancer of the kidney, or cancer of the bile duct; minimal residual disease; metastatic tumours e.g. those metastasising in the lungs, bones, liver or brain).<br>
Multispecific binding molecules of the invention wherein the at least one additional specificity is towards the CD 19 or CD20 antigen may be of particular use in the treatment of non-Hodgkin's lymphoma.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards EGFR1 may be of particular use in the treatment of cancers wherein EGFR1 expression is up-regulated or altered, for example in cancers of the breast, bladder, head and neck, prostate, kidney, non-small cell lung cancer, colorectal cancer and glioma.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards TF-antigen may be particularly useful in treating breast or colon cancer and/or liver metastases.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards CD30 may be particularly useful in treating Hodgkin's disease.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards the alpha chain of the IL4 receptor (IL4R alpha) may be particularly useful in<br><br><br>
treating solid tumors, in particular carcinomas of the breast, ovaries, renal system, head and neck, malignant melanoma and AIDS related Kaposi's sarcoma.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards EGFR3/HER3 and/or EGFR2/neu may be particularly useful in treating breast cancer.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards IGFR may be particularly useful in treating prostate cancer, colorectal cancer, ovarian cancer or breast cancer.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards CD5 may be particularly useful in treating chronic lymphocytic leukaemia<br>
Multispecific binding molecules wherein the at least one additional specificity is towards MUC-1 may be particularly useful in the treatment of gastric cancer and ovarian cancer.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards EpCAM may be particularly useful in the treatment of carcinomas of the colon, kidney, and breast.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards PLAP may be of particular use in the treatment of ovarian or testicular cancer.<br>
Multispecific binding molecules wherein the at least one additional specificity is towards OFA-iLR may be particularly useful in the treatment of metastatic tumours.<br>
In a further aspect the present invention provides the use of a binding molecule as described hereinbefore in the manufacture of a medicament for the treatment of autoimmune disease, inflammatory disease, infectious disease, allergy or cancer (e.g. non-Hodgkin's lymphoma; chronic lymphocytic leukaemia; Hodgkin's lymphoma; solid tumours e.g. those occurring in breast cancer, ovarian cancer, colon cancer, cancer of the kidney, or cancer of the bile duct; minimal residual disease; metastatic tumours e.g. those metastatising in the lungs, bones, liver or brain). Where specified multispecific binding molecules have been described above as having a particular utility in the treatment of a specified disease, these binding molecules may also be used in the manufacture of a medicament for that specified disease.<br>
In yet a further aspect, binding molecules of the invention may be used as a reagent to stain cells expressing FcgRIIIA. Where the binding molecule has specificity for at least one further antigen, it may be used as the reagent by which the FcgRIIIA-expressing cell-binding molecule-antigen complex can be identified. Binding molecules<br><br><br>
of the invention may also be used to analyse and type patient samples ex vivo, as a biomarker, and to isolate NK cells for ex-vivo therapy.<br>
Thus in a further aspect, there is provided a kit comprising a binding molecule as described hereinbefore and means for detecting the binding molecule when bound to FcgRIIIA. Where the binding molecule is radiolabeled or labelled with a chemiluminescent label, the detection means may comprise film sensitive to the radio- or chemiluminescent label. Where the binding molecule is tagged with a histidine or c-myc tag, the kit may comprise an antibody which recognises that tag.<br>
Various aspects and embodiments of the present invention will now be illustrated in more detail by way of example. It will be appreciated that modification of detail may be made with out departing from the scope of the invention, and the skilled man will appreciate that further antibodies and antigen binding fragments for use in the invention may be obtained as described herein in Example 2, and their properties with respect to binding to the different isoforms of FcgRIII and even to different allelic forms of FcgRIIIA, may be tested as described herein in the Examples.<br>
For the avoidance of doubt, the terms "FcgRIII", "FcgRIIIA" and "FcgRIIIB" are used herein interchangeably with the terms "CD 16", "CD 16A" (denoting the A isoform of CD 16) and "CD16B" (denoting the B isoform of CD 16), respectively.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1:    ELISA results demonstrating binding of scFv 50NI to streptavidin<br>
captured CD16-Fc<br>
ELISA plates were coated with 500 ng Streptavidin and 300 ng human IgG (Sigma-Aldrich, Taufkirchen, Germany) per well in 100 mM NaHC03, pH 8.6. Subsequently, 300 ng biotinylated CD16A-Fc was absorbed for 30 min at room temperature in PBS. Streptavidin captured CD16A-Fc, Streptavidin and human IgG were incubated with 1 ug/mL murine MAb A9 or 50 ml crude periplasmic extract from clone 50NI in PBS/TWEEN 0.1%/2% skimmed milk powder. Binding of MAb A9 was detected with a goat anti-mouse antibody-HRP conjugate. Binding of scFv 50NI was detected using anti-Penta His antibody-HRP-conjugate. The absorbance at 450nm of the HRP-substrate tetramethyl benzidine (TMB) was measured following termination of the reaction with 50ml 0.5M sulphuric acid.<br><br><br>
Figure 2: Affinity-matured human anti-CD16 scFv bind to cell transfectants expressing CD16A but not cells expressing CD16B. Murine BW cells and BW cells stably transfected with CD16A (BW/CD16A), HEK-293 cells and HEK-293 cells transfected with CD16B (293/CD16B), CD16A (293/CD16A) or NKp46 (293/NKp46) were stained with 10 mg/mL of the MAb A9 (anti-CD 16) or MAb 195314 (anti-NKp46) followed by 15 mg/mL FITC-conjugated goat anti-mouse IgG. The anti-CD 16 scFv were used at a concentration of 50 mg/mL and were detected with 10 mg/mL MAb 13/45/31-2 (anti-Hexa His) followed by FITC-conjugated goat anti-mouse IgG. The mean fluorescence intensities obtained from flow cytometric analysis were corrected by subtracting the fluorescence values from cell stainings with the secondary reagents alone, and plotted in the diagram.<br>
Figure 3: ELISA on recombinant CD16A-Fc and CD16B. Recombinant CD16A-Fc and CD16B proteins were coated at 80 nM in 0.1 M NaHC03, pH 8.8. Wells were blocked with PBS, 2% skimmed milk. The binding of the scFv to the recombinant proteins and to gp34-Fc (negative control) was detected using an anti-c-Myc-HRP conjugate (10mg/ml). Incubation with anti-CD 16 MAb A9 (lmg/mL) was followed by a goat anti-mouse IgG HRP-conjugate (0.5 mg/mL). Coated recombinant CD16B was directly detected by staining with an anti-His-HRP (lmg/mL) and coating of the Fc-Fusion proteins was tested by an anti-human IgG-HRP conjugate (0.5 mg/mL). As HRP-substrate, 50 mL of TMB were used. After termination of the reaction with 50 mL 0.5 M H2SO4, absorbance was measured at 450 nm.<br>
Figure 4: Triggering of redirected cytotoxicity in freshly isolated NK cells by human, affinity-matured anti-CD16 scFv. NK cells were isolated and enriched from the peripheral blood of a healthy donor and used as effector cells in a cytotoxicity assay with calcein-labelled P815 as target cells. Effector (E) and target (T) cells were incubated for 3 hours at an E:T-ratio of 10:1 in the presence of 1 mg/mL of the indicated antibodies. Percentage of specific lysis was calculated from the measured fluorescence counts of the released calcein in the supernatant. Mean values and standard deviations from triplicates were plotted.<br>
Figure 5: Flow cytometric analysis of affinity-matured anti-CD16 scFv on PMN and NK cells.   Polymorphonuclear cells (PMN) and natural killer (NK) cells were<br><br><br>
isolated from peripheral blood from a healthy donor and used for flow cytometric staining and analysis. The cells were stained with 10mg/mL of the anti-CD 16 MAb A9 and anti-CD56 MAb B159 followed by 15mg/mL FITC-conjugated goat anti-mouse IgG. All scFv were used at a concentration of 50mg/mL and were detected with 10mg/mL anti-(His)6 followed by 15mg/mL FITC-conjugated goat anti-mouse IgG. The mean fluorescence intensities obtained from flow cytometric analysis were corrected by subtracting the fluorescence values from cell stainings with the secondary reagents alone and were plotted in the diagram.<br>
Figure 6: Amino acid sequence alignment of allelic variants of the CD16A and CD16B isoforms. The signal peptides is shown as shaded. Asterisks and italic text mark the differences within the allelic variants of CD16B. Variations between the CD16A and CD16B isoforms are shown in bold and underlined. The position of the CD16A 158 Val/Phe polymorphism is indicated by the box.<br>
Figure 7: ELISA on different CD16 isoforms. Wells of a Maxisorp™ plate were coated with 100 mM NaHC03 containing 200ng of CD16A-Fc48R/158V, CD16A-Fc48R/l58F, CD16B-FcSH, CD16B-FcNA\ CD16B-FcNA2 and NKp46-Fc. All antigens were incubated with [A] the scFv 4-LS21 followed by a HRP conjugated anti-His antibody or [B] with the MAb A9 followed by a goat anti-mouse HRP conjugate (minimal cross reactivity with human IgG). An anti-NKp46 scFv used as a primary antibody acted as a negative control.<br>
Figure 8: Target-specific killing by TandAb-activated NK cells. 1x104 of the indicated calcein-labelled L540CY (results shown in panel [A]) or JOK-1 target cells (results shown in panel [B]) were incubated for 3 hours with lxl05 NK cells in the presence of the indicated concentrations of CD30xCD16 TandAb (filled rhombus) or CD19xCD16 TandAb (open rhombus). As a control, target cells were incubated with NK cells without antibody (filled square). Percent specific lysis was calculated from the measured fluorescence counts of the released calcein and analysed by non-linear regression using the GraphPadPrism software.<br><br><br>
EXAMPLES<br>
Example 1:   Transient expression and purification of a human CD16A-Fc fusion<br>
protein<br>
A secreted, soluble form of the CD16A-Fc fusion protein was produced in HEK-293 cells after transient transfection with plasmid pCDM-CD16-Fc encoding human CD16A fused to the Fc portion of human IgGl (Mandelboim et al, 1999, supra).<br>
HEK-293 cells (ATCC accession number CRL-1573) were cultured in complete DMEM medium (DMEM medium supplemented with 10% heat-inactivated fetal calf serum (FCS), 2 mM L-glutamine, 100U/mL penicillin G sodium and 100 mg/mL streptomycin sulfate; all from Invitrogen, Karlsruhe, Germany) at 37 °C in a humidified atmosphere with 5% CO2. Calcium phosphate mediated transfection of these cells was effected as follows.<br>
One day prior to transfection 1.5xl06 HEK-293 cells were used to seed 10 mL complete DMEM medium in a 10 cm diameter cell culture plate. 10 mg plasmid DNA was mixed with 500mL 250mM CaCl2, which was then added drop-wise to 500mL 2xHEBS buffer (280mM NaCl, 1.5 mM Na2HP04, 50mM HEPES, pH7.05) and incubated for 15 min at room temperature. 9 mL complete DMEM medium was added to this transfection cocktail, mixed and transferred to one dish of HEK-293 cells from which the culture medium had been removed. The next day, the transfection medium was exchanged for serum-free DMEM medium (DMEM medium supplemented with 2 mM L-glutamine, 100U/mL penicillin G sodium and 100 fig/mL streptomycin sulfate and 1 x Insulin-Transferrin-Selenium-A supplement (all from Invitrogen). Supernatants containing the secreted CD16A-Fc fusion protein were collected twice in a period of two to six days after transfection, depending on the viability of the cells. After each harvest, fresh serum-free medium was added to the cells. Harvested supernatants were centrifuged to remove cells or cell debris and stored at -80°C until used for purification of the fusion protein. Transfections were repeated until approximately 1.2 L of supernatant containing CD16A-Fc fusion proteins were harvested.<br>
Purification of the recombinant CD16A-Fc fusion protein was performed on a 2 mL Protein A column (Protein A Sepharose™ Fast Flow, Amersham Pharmacia). The column was equilibrated with ten volumes DMEM pH 7 (Invitrogen; supplemented with 100mg/mL Streptomycin, 100 u/ml Penicillin, 1 x Insulin-Transferrin-Selenium-A and 2 mM L-Glutamine). Before loading the cell culture supernatant (1200 ml) on the column, it was passed through a 0.2 mm sterile filter and adjusted to pH 7 by adding 0.1 M<br><br><br>
glycine/HCl pH 2.7. The column was run by gravity flow at 4°C. Subsequently, the column was washed with 10 volumes DMEM pH 7. Elution was carried out with 6 volumes 0.1 M glycine/HCl pH 2.7. 1 ml fractions were collected directly into 55 ml 1 M Tris/HCl pH 8.8.<br>
Analysis of fraction numbers 1-10 in a 12% PAA-gel showed the vast majority of the fusion protein in fractions 2-7. These fractions were pooled and dialysed twice against 5 L PBS pH 7.0. The determination of protein concentration according to Bradford (Bradford, 1976, Anal. Biochem. 72:248-254) revealed a total yield of 21 mg CD16A Fc-fusion protein i.e. 17.5 mg of fusion protein was produced per litre of culture supernatant.<br>
Example 2: Identification and isolation of a CD16-A specific antibody.<br>
The CD16A-Fc fusion protein was used to screen a large human IgM-derived scFv phage display library, which had been produced using methods similar to those described previously (Dorsam et al, 1997 FEBS Letts 414:7-13, Little et al, 1999 J. Immunol. Methods 231:3-9, Schwarz et al, FASEB J. 18:1704-6). The clones in this library contained nucleotide sequences between the scFv coding sequence and the pill gene of phage M13, which encoded a hexa-histidine tag, an amber stop codon and a c-myc epitope.<br>
500 mg CD16A Fc-fusion protein was biotinylated using the ECL protein biotinylation module (Amersham Pharmacia Biotech, Uppsala Sweden) and used in three rounds of selection against the phage display library. Prior to selection the library was consecutively pre-adsorbed to polystyrene coated streptavidin and to human IgG (Sigma-Aldrich, Taufkirchen, Germany) in order to remove potential binders to these proteins. Approximately 1012 phages from the library were resuspended in PBS, 0.1% tween, 2 % skimmed milk were incubated with soluble biotinylated CD16A Fc-fusion. Phage antigen complexes were captured by streptavidin coated magnetic beads using a magnetic separator (Dynal, Oslo, Norway). Phage that did not specifically bind to the target antigen were removed by ten washing steps with PBS, 0.1% tween. Bound entities were eluted by using glycine-HCl pH 2.2, and after neutralization with 2 M Tris/HCl, pH 8.0, the eluate was used for infection of freshly grown E. coli XL1 Blue cells at mid log phase (OD600 0.2-0.5). Cells that had been successfully transformed with phagemids encoding the human scFvs were selected for ampicillin resistance and were subsequently infected with M13K07 helper phage to generate phage progeny displaying scFv for the<br><br><br>
following in vitro selection. After the 3 round of selection, individual colonies were grown in LB medium containing 100 mg/mL ampicillin and 20 mg/mL tetracycline at 30 °C in a PP-Masterblock 2 mL (Greiner, Frickenhausen, Germany). Cells were harvested by centrifugation and resuspended in 200 mL 200 mM Tris-HCl, pH 7.5, 20% Sucrose, 1 mM EDTA. During a 1 hour incubation on ice the outer membrane was destroyed and soluble periplasmic proteins including the scFv were released into the culture medium. After elimination of spheroplasts and cellular debris by centrifugation, the crude periplasmic extracts were tested in ELISA for scFv binding to the CD16A-Fc fusion protein. One clone (5ONI) was found on three separate occasions.<br>
Since the affinity selection of this clone from the phage display library was performed on a human Fc-fusion protein, it was necessary to demonstrate specificity of the clone 50NI for the CD 16 portion of the molecule.<br>
A periplasmic extract from clone 50NI was prepared as described above and was tested in ELISA for its ability to bind to the CD16A-Fc fusion protein and to human IgG. Anti-CD 16 murine monoclonal antibody A9 (Hombach et al. 1993 Int. J. Cancer 55:830-836) was used as a control. A Maxisorb™ ELISA plate was pre-coated with 500 ng Streptavidin and 300 ng human IgG (Sigma-Aldrich, Taufkirchen, Germany) per well in 100 mM NaHC03, pH 8.6. Subsequently 300 ng biotinylated CD16A-Fc was captured for 30 min at room temperature in PBS.<br>
Streptavidin captured CD16A-Fc, streptavidin and human IgG were incubated with 1 mg/mL MAb A9 or 50 \x\ crude periplasmic extract from clone 50NI in PBS/TWEEN 0.1%/2% skimmed milk powder. Binding of MAb A9 was detected with a goat anti-mouse-HRP conjugate (0,5 mg/mL, Dianova, Hamburg, Germany), whereas binding of scFv 50NI was detected by an anti-Penta His HRP-conjugate (lmg/mL, Qiagen, Hilden, Germany). 50 mL of TMB (KPL, Maryland, USA) was used as HRP-substrate and the colour allowed to develop. The reaction was stopped with 50 mL 0.5 M H2SO4, and absorbance was measured at 450 nm.<br>
The results are illustrated in Figure 1. MAb A9 and the scFv 50NI clearly recognize the biotinylated streptavidin captured CD16A-Fc fusion protein. The scFv 50NI reveals neither a signal on the human IgG nor on streptavidin alone. The anti-Penta His HRP-conjugate used as a secondary reagent revealed no non-specific binding to the antigens. However, the goat anti-mouse-HRP conjugate used as a secondary reagent for MAb A9 detection caused some background staining due to cross reactivity with the<br><br><br>
human Fc portion of the CD 16 fusion protein and the human IgG used as a negative control.<br>
These results are strongly indicative that the scFv 50NI exhibits specificity for the CD16A portion of the fusion protein and not the Fc portion.<br>
Example 3: Testing scFv 50NI in FACS on different cells lines stably expressing the CD16 isoforms, human PBMC and granulocytes<br>
To determine whether the 50NI scFv reveals binding to the native CD16A protein on the cell surface, a crude periplasmic extract of the scFv was tested by flow cytometry on two different cell lines expressing CD16A or CD16B, respectively. For this purpose, the cell line BW/CD16A expressing CD16A (Mandelboim et al, 2001, Nature 409:1055-1060);) and the cell line 293-CD16 (HEK-293 cells stably transfected with a plasmid coding for the NA-2 allele of the human CD16B isoform) were used.<br>
To assess whether the 50NI scFv has the ability to bind to CD16A expressed on the surface of natural killer (NK) cells and/or to CD16B present on the surface of neutrophilic granulocytes (van de Winkel and Capel, 1993, supra; van Spriel et al, 2000, Immunol. Today 21(8):391-397), it was tested for binding to freshly isolated peripheral blood mononuclear cells (PBMC) consisting of 10-20% NK cells and on polymorphonuclear cells (PMN) representing more than 95% CD16B-positive neutrophilic granulocytes, using flow cytometry.<br>
Murine MAb A9 and a periplasmic extract from the murine clone scFvjg A9 (a scFv derived from MAb A9) were used as controls.<br>
3.1      Growth of cell lines and isolation of PBMC and PMN.<br>
BW/CD16A cells were cultured in DMEM medium supplemented with 10% heat-inactivated foetal calf serum (FCS), 2mM L-glutamine, 100U/mL penicillinG sodium, 100 mg/mL streptomycin sulphate, 1% non essential amino acids and 5mg/mL G418. The 293/CD16B cells were cultured in DMEM medium supplemented with 10% heat-inactivated fetal calf serum (FCS), 2mM L-glutamine, 100U/mL penicillinG sodium, 100 Hg/mL streptomycin sulfate, and 0.5mg/mL G418. The non-transfected parental cell lines BW and 293 were cultured as described above for the stable transfectants, but in medium lacking G418.<br><br><br>
PBMC and PMN were isolated from heparinized peripheral blood from a healthy donor by density gradient centrifugation. The blood was twice diluted with PBS, layered on a double cushion consisting of 12mL Histopaque-1119™ (Sigma-Aldrich) and an upper cushion of 12 mL Histopaque-1077. Centrifugation was carried out at 800g for 25 min. PBMC located on the upper Histopaque-1077™ (Sigma-Aldrich) cushion and the PMN located on the Histopaque-1119 ™ cushion were collected and washed 3 times with PBS before use.<br>
3.2	Isolation of periplasmic extracts containing scFv 50NI and scFv18 A9<br>
Crude periplasmic extracts containing scFvs were isolated from 5 mL bacterial cultures. The freshly inoculated bacteria were grown overnight at 37°C in LB medium containing 50 mM glucose, 100 mg/mL ampicillin and 20 mg/mL tetracycline. The overnight cultures were diluted to OD600nm = 0.1 in LB medium with ampicillin and tetracycline and further grown at 37°C till OD600nm = 0.8. The bacteria were harvested by centrifugation and resuspended in LB medium containing 0.1 mM isopropyl-P-D-thiogalactopyranoside (IPTG), 100 mg/mL ampicillin and 20 mg/mL tetracycline. After overnight incubation of induced bacteria at 21 °C, 300 mL periplasmic extracts were isolated as described for ELISA. Before use in FACS analysis, the periplasmic extracts were dialyzed against PBS overnight.<br>
3.3	Flow cytometry<br>
For flow cytometric analysis, lx106 cells BW, BW/CD16A, HEK-293, 293/CD16B, PBMC and PMN were stained with either the monoclonal antibody A9 or a periplasmic extract containing scFv18A9 and 50NI.<br>
lxl06 cells were incubated with 100 mL of the dialyzed periplasmic extracts or with MAb A9 (10mg/mL ) for 45 min on ice. Antibody/scFv incubations with PBMC and PMN were supplemented with polyclonal human IgG at lmg/mL to block Fc receptors on the surface of subpopulations of PBMC and PMN. After washing with FACS buffer (PBS supplemented with 2% heat-inactivated FCS and 0.1% sodium azide) the cells were incubated on ice for 45 minutes with 0.1 mL of 0.01 mg/mL anti-(His)6 mouse MAb 13/45/31-2 (Dianova, Hamburg, Germany) in the same buffer. After a second washing cycle, the cells were incubated with 0.1 mL of 0.015 mg/mL FITC-conjugated goat anti-mouse IgG antibodies (Dianova, Hamburg, Germany) under the same conditions. The cells were then washed again and resuspended in 0.5 mL of FACS<br><br><br>
buffer containing 2mg/mL propidium iodide (Sigma-Aldrich) to exclude dead cells. The fluorescence of 1 x 104 stained cells was measured using a Beckman-Coulter Epics XL flow cytometer. Mean fluorescence intensities were calculated using System-II and Expo32 software (Beckman-Coulter, Krefeld, Germany). Fluorescence caused by background staining of the cells with the secondary agents alone was subtracted from the results and histograms plotted (not shown) to aid analysis.<br>
3.4      Results<br>
The murine monoclonal anti-CD 16 antibody A9 and the A9-derived anti-CD 16 scFv18 served as positive controls and revealed clear binding to BW/CD16A in comparison to the non-transfected parental cells (BW). The scFv 50NI also showed unambiguous staining of the BW/CD16A cells at a level comparable to A9.<br>
Both monoclonal antibody A9 and scFv18 A9 exhibited strong binding to 293/CD16B transfectants, which express the CD16B isoform on their surface, whilst 50 NI did not bind to this cell line.<br>
The monoclonal antibody A9 and the scFv18 A9 revealed a clear staining of a small subpopulation of the PBMC representing the NK cell fraction. The scFv 50NI similarly exhibited binding to this subpopulation.<br>
In contrast to A9 and the scFv derived from A9, both of which showed a strong signal on the purified granulocytes, the 50 NI scFv failed to bind to PMN.<br>
This data indicates that the 50NI scFv binds specifically to the CD16A isoform and does not bind to the CD16B isoform.   The scFv 50NI also recognizes not only recombinant CD16A but also the native antigen as expressed on the surface of NK cells. Example 4: Binding of human anti-CD16A scFv 50NI to enriched NK cells<br>
To demonstrate that the PBMC subpopulation that was recognized by 50NI as shown in example 3 comprises natural killer (NK) cells, an enriched population of NK cells was isolated from PBMC and analysed for binding to scFv 50NI by flow cytometry.<br>
PBMCs were isolated from heparinized peripheral blood from a healthy donor by density gradient centrifugation. The blood sample was twice diluted with PBS, layered on a cushion of Histopaque-1077™ (Sigma-Aldrich) and centrifuged at 800g for 25 min. PBMC located in the interface were collected and washed 3 times with PBS. For isolation of NK cells, the PBMC were resuspended in 2% FCS/PBS and were subjected to one round of negative selection using the EasySEP™ negative NK cell enrichment kit (CellSystems) according to the manufacturers instructions.   Staining of the PBMC and<br><br><br>
NK cells and flow cytometric analysis was performed essentially as described in example 3 above: lxl06 of the cells were stained with 10mg/mL of the MAb A9 or the anti-CD56 monoclonal antibody MAb B159 (BD Biosciences/Pharmingen) followed by 15mL FITC-conjugated goat anti-mouse IgG. The murine scFv18 A9 and the human scFv 50NI were used for staining at a concentration of 5mg/mL, and cell bound scFv were detected with 10mg/mL MAb anti-(His)6 followed by 15mL FITC-conjugated goat anti-mouse IgG. All stainings were performed in the presence of 1 mg/mL polyclonal human IgG. Dead cells were excluded by propidium iodide staining as described above. Data analysis was carried out as described in Example 3 above.<br>
4.1      Results<br>
Staining of PBMC with the anti-CD56 and anti-CD 16 antibodies clearly identified a subpopulation representing the NK cell fraction. The scFv 50NI also recognized a similar small cell population indicating binding to NK cells.<br>
After negative enrichment of the NK cells, all four antibodies/scFvs revealed unambiguous staining. Whilst the anti CD56 MAb exhibited binding to the entire NK cell population, a small fraction of the cells were not stained using the antibodies directed against CD 16. This might reflect the presence of some cells with a CD16-negative phenotype.<br>
Example 5: Ca2+ flux measurement after scFv 50NI binding<br>
CD 16A, the low affinity receptor for IgG, is expressed on the majority of human peripheral blood NK cells in association with the immunoreceptor tyrosine-based activation motif (ITAM) containing the y chain from FcsRI or the C, chain from the TCR/CD3 complex. Binding of CD16A to antibodies or immune complexes triggers NK cell cytotoxicity and cytokine secretion. The activation involves phosphorylation of the y chain or C, chain, respectively, and subsequent intracellular signal transduction such as Ca2+ mobilization.<br>
To test whether the human anti-CD 16 scFv 50NI possesses NK-activating properties, Ca2+ flux measurements with freshly isolated NK cells were performed after ligation of surface receptors with anti-CD 16 monoclonal antibodies or scFv. Since some anti-CD 16 antibodies need cross-linking by secondary reagents to elicit a clear increase of intracellular calcium, anti-c-myc and anti-(His)6 in combination with anti-mouse IgG were used to cross-link cell-bound anti-CD 16 antibodies.<br><br><br>
For NK preparation, PBMC were isolated from heparinized peripheral blood from a healthy donor by density gradient centrifugation. The blood sample was twice diluted with PBS, layered on a cushion of Histopaque-1077™ and centrifuged at 800g for 25 min. PBMC located in the interface were collected and washed 3 times with PBS. For isolation of NK cells the PBMC were resuspended in 2%FCS/PBS and subjected to one round of negative selection using the EasySEP™ negative NK cell enrichment kit according to the instructions of the manufacturer (Cell Systems). The NK cells were washed once with HBSS and then labelled with the calcium-indicator Fluo-4-acetoxymethyl (AM) ester (Molecular Probes, Leiden, The Netherlands) at a concentration of 2mM for 30 min at room temperature in the dark. After washing with HBSS the cells were incubated for further 20 min in HBSS to allow complete de-esterification of intracellular AM esters and washed again with HBSS. For flow cytometric analysis of intracellular calcium changes aliquots of labelled NK cells were first measured without any antibody on a Beckman-Coulter Epics XL flow cytometer to determine background fluorescence. After approximately 60 sec, anti-CD 16 antibodies (2mg MAb A9 or 5mg scFv18 A9 or 5mg 50NI scFv) were added and the measurement was continued. At 180 sec after starting, 20mg polyclonal goat anti-mouse IgG antibodies (GAM) were added to the MAb A9 sample to cross-link A9 bound to CD 16 on the surface of NK cells. To cross-link the scFvs 2.5u£ anti-(His)6 mouse MAb 13/45/31-2 and 2.5 ug anti-c-myc MAb 9E10 were applied before continuing measurement. In the case of the samples with the scFvs 20mg polyclonal goat anti-mouse IgG antibodies were added at later time points to further cross-link anti-(His)6 and/or anti-c-myc MAbs bound to the scFv. The mean fluorescent intensities from short time intervals were determined using the Expo32 software (Beckman-Coulter, Krefeld, Germany) and plotted against time (not shown).<br>
The fiuorometry tracings obtained with freshly isolated NK cells clearly showed that MAb A9 elicits a strong increase in the intracellular calcium concentration upon cross-linking with anti-mouse IgG antibodies, which is reflected by higher fluorescence signals. In contrast to MAb A9, the scFv derived from A9 (scFv18 A9) induced little or no increase of intracellular calcium even after cross-linking with anti-(His)6, anti-c-myc and anti-mouse IgG antibodies. The human scFv 50NI, however, elicited a clear (although slightly weak) fluorescence signal after ligation of the cell surface bound scFv with cross-linking antibodies. This demonstrates that the human scFv 50NI possesses binding characteristics that are suitable for triggering NK cell activity via CD16A.<br>
Example 6: Affinity maturation of the 50NI scFv by light chain shuffling<br>
To obtain anti-CD 16 scFvs with increased affinity, a chain shuffled library based on the clone 50NI was generated by rearrangement of the 50NI VH chain with a VL gene repertoire.<br>
The VK and V\ gene repertoires were excised from pEXHAMl DNA originating from a large human naive scFv library previously described (Schwarz et ah supra, 2004). This repertoire reflects approximately 105 individual VK and "V\ chains each. Rearrangement with the 50NI heavy chain by cloning into the Mlul and Notl sites of pEXHAMl and transformation of 1 mg into E. coli XL1 blue (1.8 kV, 0.1 cm gap cuvette 25mF, 200 D) resulted in a 2.6 x 106 (VK) and 2.2 x 106 (Vx) member library.<br>
Sequence analysis of 32 randomly picked clones after transformation showed that each clone contained an individual light chain and 24 clones revealed expression of full length scFvs detectable by an anti-His antibody (Qiagen, Hilden, Germany) in a Western Blot.<br>
Four rounds of selection were performed on a biotinylated CD16A-Fc fusion protein as described in example 2. Selection pressure favouring high affinity binders was created by successively limiting the antigen concentration (1st round 20 nM, 2nd round 1 nM, 3round round 0.1 nM, 4th round 0.01 nM. The K-and X rearranged libraries were kept separately.<br>
Approximately 10 phage from each library resuspended in PBS/0.1 % TWEEN/2% skimmed milk were incubated with biotinylated CD16A-Fc fusion. Phage antigen complexes were rescued by adding streptavidin coated magnetic beads and nonspecific antigen binding phage were removed by ten washing steps with PBS/0.1 % TWEEN. In a first step, bound entities were eluted by using glycine-HCl pH 2.2. After neutralisation with 2 M Tris/HCl pH 8, freshly grown E. coli XL1 Blue (mid log phase OD600 0.2-0.5) were added in order to recover phage which could not be removed by low pH. Cells eluted by acidic elution were also mixed with logarithmic growing XL1 blue host cells.<br>
Cells successfully transduced with phagemid, encoding a human scFv were selected for ampicillin resistance and subsequently infected with M13K07 helper phage to generate phage presenting scFv for the following in vitro selection. In subsequent rounds of selection, phage recovered by acidic elution and infection of XL1 blue were<br><br><br>
handled separately. After the 3r and 4 round of affinity selection, individual colonies were tested in ELISA as described in example 2.<br>
None of the clones tested from the rearranged K-library showed significant signals above that of the parental clone 5 ONI grown as reference in the same master block. However, several clones were sequenced and proved to be identical to the parental clone, indicating that no suitable K-chain was found which improved the binding properties.<br>
In contrast, analysis of clones derived from the rearranged X-library revealed numerous clones generating signals up to five fold over the reference. 30 clones were selected for sequencing after the 3rd and 34 clones were selected for sequencing after the 4lh round of bio panning. Sequencing was performed with the Long Read Tower (Visible Genetics, Toronto Canada) using the Deaza Kit (Visible Genetics, Toronto Canada) following the manufacturers guidelines. Acidic elution of bound phage particles or direct infection of the host cells had no obvious effect on the later distribution of the clones. Five different groups of enriched clones and several individuals were identified. Table 2 gives an overview of the different groups.<br>
Table 2   Designations of clones representing a certain group in subsequent examples are highlighted in bold<br><br>
Group	Total Frequency	3m round 30 clones sequenced	4tn round 34 clones sequenced<br>
A	19/64	3-LB1, 3-LB24, 3-LB36, 3-LB48, 3-LS1.3-LS4, 3-LS19	4-LB28, 4-LB34, 4-LB27, 4-LB43, 4-LS6, 4-LS10, 4-LS22, 4-LS26, 4-LS68; 4-LS80, 4-LS88<br>
B	9/64	3-LB3,        3-LB41,        3-LS23, 3-LS35, 3-LS87	4-LB46, 4-LS38, 4-LS12, 4-LS39<br>
C	13/64	3-LB5,        3-LB10,        3-LB32, 3-LS17,       3-LS22,       3-LS79, 3-LS36, 3-LS89	4-LB4, 4-LB13.4-LB41 4-LS25, 4-LB25<br>
D	5/64	3-LS72	4-LS73, 4-LB32, 4-LB33, 4-LB33<br>
E	5/64		4-LS14, 4-LS37, 4-LS87, 4-LS72, 4-LS44<br>
individuals		3-LB6,        3-LB23,        3-LS49; 3-LS34,       3-LS60,       3-LS28, 3-LS30, 3-LS55, 3-LS40	4-LS5, 4-LB14, 4-LS21, 4-LS46<br>
Clones forming group A represent the parental clone 50NI which was also present in the shuffled library due to re-ligated or uncut parental vector or due to rearrangement<br><br><br>
during library construction. This finding correlates with the fact that clones from this group yielded the weakest ELISA signals.<br>
All other groups contain newly arranged light chains revealing increased ELISA signals compared to the parental clone. Clones from group C and D were found in comparable numbers after the 3rd and 4th selection round.<br>
Analysis of the sequences using a V-base alignment software (http://vbase.mrc-cpe.cam.ac.uk) revealed that all new V\ chains belong to the VL3 family as well as the VL chain from the parental clone.<br>
Amino acid sequences of the common VH chain shared by all clones is given herein as SEQ ID NO 9 and the specific VL chain for clones 50NI, 3-LB3, 3-LB5, 3-LB6, 3-LS49, 4-LS14 and 4-LS21 are given herein as SEQ ID NOs 10-16 respectively. The nucleic acid sequence encoding the common VH chain is given herein as SEQ ID NO 1, and the nucleic acid sequences encoding the individual VL chains for clones 50NI, 3-LB3, 3-LB5, 3-LB6, 3-LS49, 4-LS14 and 4-LS21 are given herein as SEQ ID NOs 2-8 respectively. The amino acid sequences of the individual heavy and light chain CDRs are given in Table 1, with the common heavy chain CDRs shown as the first entry in the table, followed by light chain CDRs for 50NI, 3-LB3, 3-LB5, 3-LB6, 3-LS49, 4-LS14 and 4-LS21 (shown as entries 2-8, respectively).<br>
Example 7: FACS analysis of affinity matured scFv on cell lines transfected with CD16A and CD16B<br>
A preliminary FACS analysis with crude periplasmic extracts from all individual clones showed that all chain shuffled scFv retained their ability to bind CD16A. For this test, HEK-293 cells transiently transfected with CD16A were used (data not shown). Based on this test, the clones 3-LB3, 3-LB5, 3-LB6, 3-LS49, 4-LS14 and 4-LS21 were selected for further characterisation.<br>
Nucleic acids encoding these scFvs were subcloned into the expression vector pSKK2, where they were cloned in-frame with a N-terminal PelB leader sequence, and C-terminal hexa-histidine and c-myc tags. The nucleotide sequences of these sub-clones are as follows:<br>
pSKK2-50NI (SEQ ID NO 36)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br><br><br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtatcctatgagctgatgc<br>
agccaccctcagtgtccgtgtcctcaggacagacagccagcatcccctgctctggagataaattggaggaaaaatatgtttcctg<br>
gtatcaacagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcagggatccctgagcgattctct<br>
ggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctgactactattgtcaggtgtg<br>
ggacaattacagtgtgctattcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgccccctcggtcactctgtt<br>
cccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcactaa<br>
pSKK2-3-LB3 (SEQ ID NO 37)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtatcctatgagctgacac<br>
agccactctcagagtcagtggcccagggacagacggccaggattacctgtgggggaaacaacattgaaagtagaaatgttcac<br>
tggtaccagcagaagccaggccaggcccctgtgttggtcatctatagggataacaaccggccctctgggatccctgagcgattc<br>
tctggctccaattcggggaacatggccaccctgaccatcagcagagcccaagccggggatgcagctgactattactgtcaggt<br>
gtgggacaactacactgtgctattcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgccccctcggtcactc<br>
tgttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcact<br>
aa<br>
pSKK2-3-LB5 (SEQ ID NO 38)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtatcctatgagctgacac<br>
agccaccctcagtggcagtggccccaggaaagacggccaggattacctgtgggggaaacaacattggaagtaaaaatgtgca<br>
ctggtaccagcagaagccaggccaggcccctgtgctggtcatctatagggatagcaaccggccctctgggatccctgagcgat<br>
tctctggctccaactcggggaacacggccaccctgaccatcagcagagcccaagccggggatgaggctgacttttattgtcag<br><br><br>
gtgtgggacaactatattgtgctgttcggcggagggaccaagctgaccgtcctgggtcagcccaaggctgccccctcggtcact ctgttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcac taa<br>
pSKK2-3-LB6 (SEQ ID NO 39)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtacaggctgtgctgactc<br>
agccgccctcagtgtcagtggccccaggacagacggccaggattccctgtgagggaaacaacattggaagtaaaaatgtcca<br>
ctggtatcggcagaagccaggccaggtccctgtcctggtcatgtatgatgatagcgaccggccctcagggatccctgagcgatt<br>
ctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctgactactattgtcaggt<br>
gtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgccccctcggtcactct<br>
gttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcacta<br>
a<br>
pSKK2-3-LS49 (SEQ ID NO 40)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtacagcctgtgctgactc<br>
agccactctcagtgtcagtggccccgggacagacggccaggattacctgtgggggaaacaacattggaagtaaaaatgtgca<br>
ctggtaccagcagaagccaggccaggcccctgtactggtcatctatagggacagcagccggccctctgggatccctgagcga<br>
ctctctggctccaactcgggggacacggccaccctgaccatcagcagagcccaggccggggatgaggctgactattactgtca<br>
ggtgtgggacgactacattgtggtcttcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgcccccccggtc<br>
actctgttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccat<br>
cactaa<br>
pSKK2-4-LS14 (SEQ ID NO 41)<br><br><br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtatcctatgagctgacac<br>
agccaccctcggtgtcagtgaccccaggacagacggccacgattacctgcggggcaaacgacattggaaaaagaaatgtcca<br>
ctggtaccaacagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcagggatccctgagcgattc<br>
tctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctgactactattgtcaggtg<br>
tgggacaattacagtgtgctattcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgccccctcggtcactctg<br>
ttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcactaa<br>
pSKK2-4-LS21 (SEQ ID NO 42)<br>
atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcggaggtccagctggtaca<br>
gtctggagcagaggtgaaaaagcccggggagtctctgaaggtttcctgcaaggcatctggatacaccttcaccagctactatat<br>
gcactgggtgcgacaggcccctggacaagggcttgagtggatgggaataatcaaccctagtggtggtagcacaagctacgca<br>
cagaagttccagggcagagtcaccatgacccgggacacgtccacgagcacagtctacatggagcttagcagcctgagatctg<br>
aggacacggccgtgtattactgtgctagaggtagtgcttattactacgattttgctgactactggggccagggaaccctggtcacc<br>
gtctcctcagggagtgcatccgccccaacccttaagcttgaagaaggtgaattttcagaagcacgcgtacagcctgtgctgactc<br>
agccatcctcggtgtcagtggccccaggacagacggccacgatctcctgtgggggacacaacattgggagtaaaaatgtgca<br>
ctggtaccagcagaggccaggccagtcccctgtgttggtcatttatcaggataataagcggccctcagggatccctgagcgatt<br>
ctctggctccaactctgggaacacagccactctgaccatcagcgggacccaggcgatggatgaggctgactactattgtcaggt<br>
gtgggacaattacagtgtgctattcggcggagggaccaagctgaccgtcctaggtcagcccaaggctgccccctcggtcactct<br>
gttcccgccgtccgcggccgctggatccgaacaaaagctgatctcagaagaagacctaaactcacatcaccatcaccatcacta<br>
a<br>
The nucleotide sequence encoding the PelB leader sequence is atgaaatacctattgcctacggcagccgctggcttgctgctgctggcagctcagccggccatggcg (SEQ ID NO 43).<br>
The nucleotide sequence encoding the linker region between the light and heavy chain variable domains of each scFv is gaagaaggtgaattttcagaagca (SEQ ID NO 44).<br>
The nucleotide sequence of the c-myc and hexa-histidine tags are gaacaaaagctgatctcagaagaagaccta (SEQ ID NO 45) and catcaccatcaccatcac (SEQ ID NO 46), respectively.<br><br><br>
The vectors comprising the sub-clones were transformed into E. coli RV308, expressed in 2 liter shake flask cultures and purified by immobilized metal ion affinity chromatography (IMAC) according to Le Gall et al. (2004, J. Immunol. Methods 285:111-127).<br>
To assess whether the chain shuffled scFv retained their specificity for CD16A as indicated for 50NI in Example 3, transfected cell lines expressing either CD16A or CD16B were tested with IMAC purified scFv. For flow cytometric analysis, the following cell lines were used: BW and BW/CD16A transfectants were cultured as described previously. 293 cells and 293-CD16B transfectants were handled as described previously. 293-CD16A and 293-NKp46 were generated by transient transfection of 293 cells with plasmids encoding the extracellular domains of CD16A or NKp46, respectively, fused to the transmembrane and cytoplasmic domain of the human CD3^ chain (pcDNA3-CD16A-zeta and pcDNA3-NKp46-zeta, obtained from Dr. O. Mandelboim, University of Jerusalem, Israel). The transient transfections were performed using the calcium phosphate method as described in Example 1 and were used for flow cytometric staining and analysis two days after transient transfection. Flow cytometric staining and analysis was performed as essentially described before, lxl06 of the indicated cells were stained with 50ug/mL of the purified scFv followed by lOug/mL MAb 13/45/31-2 (anti-hexa His) and 15ug/mL FITC-conjugated goat anti-mouse IgG.<br>
To demonstrate that the transfected cells express the corresponding antigens on their cell surface, cells were stained in parallel with lOug/mL of MAb A9 and anti-NKp46 monoclonal, MAb 195314 (R&amp;D Systems, Wiesbaden, Germany). MAbs A9 and 195314 were detected with 15(ig/mL FITC-conjugated goat anti-mouse IgG. The mean fluorescence intensities obtained from the flow cytometric analysis were corrected by subtracting the fluorescence values from stainings with the secondary reagents alone, and the results shown in Figure 2.<br>
MAb A9 used as a positive control stains bound as expected to all cell lines transfected with CD 16, irrespective of the isoform. No binding to the NKp46-transfectants was observed, whereas staining with MAb 195314 directed against NKp46 clearly showed that the cells were successfully transfected with the control plasmid encoding NKp46.<br>
The parental human anti-CD 16 scFv 5 ONI used as a reference revealed weak staining of the two cell lines expressing CD16A on their surface (the BW/CD16 A and 293/CD16A cells).   In contrast, the newly isolated scFv with rearranged light chains<br><br><br>
exhibited highly improved binding to these cells. None of the new variants bound 293 cells stably transfected with CD16B, demonstrating specificity for the A isoform of CD16.<br>
Example 8:  Testing affinity matured scFv fragments on freshly isolated PMN and enriched NK-cells<br>
To assess whether the improved scFv retained their ability to specifically bind to CD16A expressed on leukocytes, freshly isolated PMN and enriched NK cells from donor blood were stained as described previously using IMAC-purified scFv. Enrichment of NK cells was performed by negative selection as described above. Mean fluorescence intensities obtained from flow cytometric analysis are shown in Figure 5.<br>
Staining of the enriched leukocyte population with anti-CD 16 MAb A9 and anti-CD56 MAb B159 clearly demonstrates that the population consists of CD16+/CD56+ peripheral blood NK cells. The MAb A9 and the murine scFv derived from A9 stained the CD16A-positive NK cell fraction and showed an even stronger signal on CD16B-positive granulocytes.<br>
MAb A9 recognized a subpopulation (18.5%) of PBMCs corresponding to CD16-positive NK cells. Strong binding was observed on enriched NK cells and isolated granulocytes (data not shown).<br>
The parental clone 5 ONI revealed weak but clear binding to enriched NK-cells, whilst no staining of the granulocytes was visible. The affinity matured variants 3-LB3, 3-LB5, 3-LB6, 3-LS49, 4-LS14 and 4-LS21 revealed staining patterns resembling that of 50NI, but with improved binding signals. This data provides further support for the fact that the affinity matured scFvs are able to distinguish specifically between the two isoforms of CD 16 and bind specifically to CD 16A.<br>
Example 9: ELISA on recombinant CD16A-Fc and CD16B<br>
In order to generate further supporting data concerning the CD16A specificity of the scFvs, an ELISA was performed using the CD16A-Fc fusion protein (Example 1) and commercially available recombinant CD16B-Fc (Allele NA-2; BSA free; RD Systems).<br><br><br>
A gp34-Fc fusion protein was used as a negative control. The recombinant proteins were coated overnight at a concentration of 80 nM in 100 mM NaHCC&gt;3 at pH 8.8.<br>
Detection of scFv bound to the recombinant proteins was carried out by using an anti-c-myc-HRP conjugate (lOug/ml). The positive control comprised MAb A9 (lug/mL) followed by a goat anti-mouse IgG HRP-conjugate (0.5 fag/mL). Binding to the recombinant CD16B was tested by directly staining with anti-His-HRP (lug/mL) and binding to the Fc-Fusion proteins was tested by an anti-human IgG-HRP conjugate (0.15 ug/mL).<br>
All of the scFv showed strong binding to CD16A-Fc, whereas no binding to CD16B was observed (Figure 3). A slightly increased background binding was noticed with gp34-Fc. This was probably due to a small amount of unspecific binding to gp34-Fc, since all secondary antibodies tested alone also revealed slightly increased background signals on this protein.<br>
As expected, A9 strongly recognized CD16A-Fc. However, it must be considered that the secondary goat anti-mouse-HRP conjugated antibody shows significant cross reactivity with the human Fc fusion (see control with goat anti mouse-HRP alone). Reproduction of this assay using a pre-adsorbed goat anti-mouse IgG-HRP conjugate with minimal cross reactivity to human IgG (data not shown) revealed no significant reduction in background binding.<br>
MAb A9 showed a clear signal on CD16B, which was not influenced by the secondary reagent.<br>
Example 10: Western Blot on recombinant CD16A-Fc under reducing and non reducing conditions<br>
For further characterization all of the anti-CD 16A scFv were tested for binding in Western Blot on recombinant CD16A-Fc and NKp46-Fc.<br>
CD16A-Fc and NKp46-Fc serving as a negative control were separated by 12% SDS-gel under reducing condition and on a 8% gel under non reducing conditions. The proteins were transferred on a nitrocellulose membrane by electroblotting. After staining with Ponceau S, the membrane was cut into strips of 5 mm width. Each nitrocellulose strip was individually incubated with scFv (4ug/ml) in PBS, 0.1% tween, 2% skimmed milk. Detection was carried out with the anti-Penta His HRP-conjugate (lug/mL) using diaminobenzidine (DAB) as a peroxidase substrate. Control strips were incubated with A9 (1 ug/mL) or anti-NKp46 195314 followed by a goat anti-mouse IgG HRP-conjugate<br><br><br>
(0.2 ug/mL, minimal cross reactivity). As an additional positive control, a goat anti-human IgG HRP-conjugate (0.16 ug/mL) was used for detection of the Fc portion of the fusion proteins.<br>
The monoclonal antibodies A9 and 195314 recognized their antigen under reducing as well as under non reducing conditions. The goat anti-mouse-HRP conjugate used as a secondary reagent for MAb A9 revealed slight cross reactivity with the Fc portion of the CD16A-Fc fusion protein. In contrast, binding of the anti-CD 16 scFvs was only observed under non-reducing conditions indicating that a secondary structure stabilized by disulfide bridges is required for recognition. The parental clone 50NI shows weaker signals compared to the affinity matured scFvs. No binding of the scFv 3-LB6 could be detected and none of the scFv revealed cross reactivity with the NKp46-Fc control protein, confirming specificity for CD 16.<br>
Example 11: Analysis of the scFv by size exclusion chromatography on a Superdex 200 column.<br>
To determine whether the higher affinities of the scFv variants are primarily due to improved monovalent binding, the protein preparations were analysed for their oligomeric composition on a Superdex 200 gel filtration column. The LMW Gel Filtration Calibration Kit (Amersham Pharmacia Biosciences, Freiburg, Germany) served as a standard. Table 3 gives an summary of the molecular forms present in the individual protein preparations.<br>
Table. 3 Molecular forms of scFvs present in individual protein preparations. 3-LB5 was not analysed due to low protein concentration.<br><br>
scFv	Monomer	Dimer<br>
3-LS49	67.2%	32.8%<br>
3-LB3	87.8%	12.2%<br>
3-LB6	100%	-<br>
4-LS14	57.6%	42.4%<br>
4-LS21	100%	-<br>
Size exclusion chromatography clearly demonstrated that the scFv 3-LB6 and 4-LS21 were solely monomeric.<br>
In contrast 3-LS49, 3-LB3 and 4-LS14 revealed significant proportions of dimeric molecules, indicating that the improved binding properties of these molecules may not be<br><br><br>
only due to higher affinity, as for 3-LB6 and 4-LS21, but may also reflect an avidity effect due to dimer formation.<br>
Example 12: Determination of scFv affinity by surface plasmon resonance (SPR)<br>
The affinity constants (KD values) of the scFv were determined by SPR using the BIAcore 2000 biosensor system (Amersham-Pharmacia). The CD16A-Fc fusion protein and BSA used as a negative control were immobilized on the carboxymethyldextran-coated sensor chip CM5 using conventional NHS/EDC chemistry for the amine coupling procedure (BIAcore). Purified scFvs were diluted in HBS-EP buffer (BIAcore, Uppsala, Sweden). Binding data was generated by passing varying concentrations of the scFv antibodies (100 nM, 200 nM, 400 nM, 800 nM and 1600 nM) over the immobilized antigen. The anti-CD 16 MAb A9 served as positive control. All SPR measurements were performed at a constant flow rate of 20 uL/min in HBS-EP buffer at 25°C. Each injected sample (100 ul) was in contact with antigen for 5 minutes. The dissociation was followed for 30 minutes. After each cycle, the surface of the sensor chip was flushed with HBS-EP buffer. Signals from the control channel coated with BSA were automatically subtracted. Kinetic constants were calculated according to the 1/1 Langmuir binding model using the BIAevaluation version 3.0 software (BIAcore). The calculated off- and on-rate constants and respective KD values are summarized in table 3. For MAb A9, used as a control, a KD of 1.75 x 10"9 M was calculated correlating with the value of 1,55 x 10"9 M described in the literature (Renner et ah, 2001 Cancer Immunol. Immunother. 50:102-108). Additionally, equilibrium dissociation constants were deduced from steady state analysis for the scFv 50NI, 4-LS21 and 3-LB6 of the SPR data.<br>
Table 4 Data for the kinetics of binding for individual scFvs<br><br>
scFv	kon (M-V1)	kof,(s1)	KD = koff/k0n(M)	Keq(M)<br>
50NI	(1.76 ± 1.37) x104	(2.17 ± 0.55) x10"2	1.23 x10"6	2.56 x 10<br>
4-LS21	(5.60 ± 1.88) x104	(6.30 ± 0.48) x10"3	1.13 x10"7	2.03x10<br>
3-LB6	(5.48 ± 0.48) x104	(6.52 ± 0.44) x10"3	1.19 x10"7	1.06x10'<br>
3-LS49	(2.93 ± 1.05) x104	(4.01 ± 0.93) x10"3	1.37 x10"7	n.d.<br>
3-LB3	(2.16 ± 0.44) x104	(2.97 ± 0.48) x10"3	1.38 x10"7	n.d.<br>
4-LS14	(5.09 ± 0.99) x104	(5.57 ± 1.31) x10"3	1.09 x10"7	n.d.<br>
3-LB5	(7.33 ± 0.44) x104	(2.97 ± 0.48) x10"3	4.05 x10"8	n.d.<br>
MAbA9	(1.01 ±0.46)x105	(1.77 ± 0.84) x10"4	1.75 x10"9	n.d.<br><br><br>
Kinetic constants calculated independently by these two methods correlate well and direct comparison with the parental clone 50NI revealed more than twenty-fold improvement of affinity by light chain shuffling. All three scFv form solely monomeric protein populations (compare Table 3, Example 11) indicating that the improved binding properties are due to higher affinity based on optimisation of the antigen scFv interface and not to avidity effects.<br>
The scFv 3-LS49, 3-LB-3 and 3-LS14 reveal comparable Kp values but show an inhomogeneous molecule population comprising significant percentages of dimeric molecules (compare table 2, example 13). Thus, improved affinities of these antibodies may be at least partially due to multivalent binding.<br>
Example 13: Induction of Ca2+ flux in NK cells by binding of affinity matured anti-CD16 scFv<br>
To assess whether the affinity-matured scFv retained their ability to elicit Ca2+ mobilization in peripheral blood NK cells as shown for the parental anti-CD 16 scFv 50NI, NK cells were isolated and enriched from peripheral blood and used for Ca2+ flux measurements by flow cytometry as described above in Example 5.<br>
The results clearly demonstrate that the addition of MAb anti-(His)6 followed by polyclonal goat anti-mouse IgG antibodies (GAM) alone did not induce significant changes of the intracellular calcium concentration in freshly isolated NK cells. However, a clear increase of the fluorescence signal was observed when scFv 5 ONI was added and cross-linked by MAb anti-(His)6 followed by GAM as already shown in Example 5.<br>
When the affinity-matured anti-CD 16 scFv (3-LS49 or 4-LS21, used as illustrative examples) were applied, the increase in induced signal after cross-linking by anti-(His)6 followed by GAM was at least as strong as the signal triggered by 50NI. Thus, it may be concluded that all scFv that are derived from the human scFv 50NI by chain-shuffling retain their ability to activate NK cells and to trigger an intracellular calcium mobilization by binding to CD 16A.<br>
Example 14: Triggering of cytolytic activity in freshly isolated NK cells<br>
CD16A is involved in antibody-dependent cellular cytotoxicity (ADCC) mediated by NK cells. In vitro it was shown that activating monoclonal antibodies of the IgG isotype specific for CD 16 can trigger killing of FcR-positive target cell lines. To determine whether the affinity-matured anti-CD 16A scFv that induce calcium mobilization in NK<br><br>
cells are also able to mediate NK cytotoxicity, these scFv were tested in a redirected killing assay with freshly isolated NK cells against murine FcR-positive P815 cells. Since the scFv lack the Fc portion of IgG, they were used together with an anti-(His)6 murine monoclonal antibody. The redirected cell lysis was measured in a calcein-release assay.<br>
NK cells were isolated and enriched from PBMC of a healthy donor as described above. Flow cytometric analysis demonstrated that the NK population consists of 81% CD16-positive and 94% CD56-positive cells; CD3-positive or CD19-positive cells could not be detected in significant amounts (data not shown). The murine P815 cell line (kindly provided by Dr. G. Moldenhauer, DKFZ, Heidelberg, Germany) was cultured in RPMI 1640 medium supplemented with 10% heat-inactivated fetal calf serum (FCS), 2mM L-glutamine, lOOU/mL penicillinG sodium, 100 ug/mL streptomycin sulfate, and 50uM P-mercaptoethanol. For the assay, the P815 target cells were labelled with lOuM CalceinAM (Molecular Probes) in RPMI 1640 medium without FCS for 30 min at 37°C. After washing twice with RPMI 1640 medium the labelled P815 cells were resuspended in RPMI 1640 medium supplemented with 10% heat-inactivated FCS, 2mM L-glutamine, lOOU/mL penicillinG sodium, 100 ug/mL streptomycin sulfate, to a density of lxl06/mL. From this suspension, 50uL corresponding to lxl04 cells were seeded in individual wells of a 96 well round bottom microtiter plate. lxlO5 freshly isolated NK cells were added in 50uL to the wells giving an effector-to-target ratio of 10:1. Anti-CD^ antibodies and control antibodies were added to the wells to a final concentration of lug/mL and where indicated lug/mL anti-(His)6 mouse MAb 13/45/31-2 was applied. Monoclonal antibodies OKT3 (an anti-CD3 MAb), A9 and 13/45/31-2 anti-(His)6 were a kind gift from Dr. G. Moldenhauer, DKFZ, Heidelberg, Germany. Antibody B159 anti-CD56 was purchased from BD Biosciences/Pharmingen (Heidelberg, Germany) and 195314 anti-NKp46 from R&amp;D Systems (Wiesbaden, Germany). The plates were incubated for 3 hours at 37°C in a humidified atmosphere with 5% CO2. To some wells, 1 % Triton X 100 was added (Roth, Karlsruhe, Germany) to give maximum lysis of target cells. Spontaneous release of calcein was determined by incubating the target cells without effector cells and without antibodies. After incubation, the microtiter plates were centrifuged at 500g for 5min and lOOuL of the supernatant was harvested to measure fluorescence (F) of released calcein with a multilabel reader at 520nm (Victor3, Perkin Elmer, Rodgau, Germany).  The percentage of lysis was calculated according to<br><br><br>
the formula: [F(sampie)-F(sPontaneous)]/[F(maximumrF(Spontaneous)]xlOO%. Mean values and standard deviations were determined from triplicate samples and plotted in Figure 4.<br>
As predicted the OKT3 anti-CD3, B159 anti-CD56 and anti-(His)6 MAbs that were used as control antibodies induced only background killing similar to that of samples without any antibody. This demonstrates that binding of an antibody to a non-activating receptor like CD56 is not sufficient to induce NK cell cytotoxicity. In addition, the control antibodies exclude cytotoxic effects of IgGs that are binding to Fc receptors on P815 target cells but not to effector molecules on NK cells. In contrast, MAb 195314 (anti-NKp46) and A9 (anti-CD 16), both targeting activating NK receptors, triggered a strong lysis of P815 targets that was in the range of 35% and 65%, respectively. The lysis of the target cells in the presence of the anti-CD 16 scFv was between 1.8% and 11.5% indicating that the scFv alone could not activate NK cells. However, the addition of MAb anti-(His)6 that facilitates cross-linking and binding to FcR-positive P815 target cells triggered a strong lysis in all samples. In the case of the affinity-matured scFv 4-LS21, almost 60% of the target cells were lysed. In comparison, the parental anti-CD 16 scFv 50NI in combination with MAb anti-(His)6 induced only a slight enhancement of the redirected killing of P815 targets by NK cells.<br>
The results from the redirected cytotoxicity assay with the affinity-matured anti-CD^ scFv clearly demonstrate that the anti-CD 16A scFv not only elicit calcium mobilization in NK cells but also trigger NK cytotoxicity.<br>
Example 15: Generation of the nucleic sequences encoding for the human CD16A-Fc48R/i58F variant and for the three different human CD16B-Fc allelic variants<br>
Human FcyRIII exists as two isoforms, FcyRIIIA (transmembrane protein) and FcyRIIIB (GPI-anchored), that share 96% sequence identity in their extracellular immunoglobulin-binding regions (van de Winkel and Capel, 1993, Immunol. Today 14(5):215-221). For these two isoforms different allelic variants are described (Koene et al, 1997 Blood 90:1109-1114; Koene et al, 1998 Blood 91:673-679). Figure 6 shows an alignment of the amino acid sequences of the allelic variants of the A and B isoforms.<br>
The soluble CD16A-Fc fusion protein produced in HEK-293 cells after transient transfection with the plasmid pCDM-CD16-Fc and described in Example 1, is the human CD16A-Fc48R/158v (Mandelboim et al, 1999, supra). In position 158 of this isoform a bi-allelic polymorphism (FcyRIII-A 158Val/Phe), occurs in humans, which influences the affinity of the Fey RIII-A receptor on natural killer cells for IgG. For FcyRIII-A 158 Val,<br><br><br>
a higher affinity for IgG is reported in comparison to the allelic 158 Phe form (Koene et ah, 1997 supra).<br>
To demonstrate binding of the scFv 4-LS21 to the Phe variant, site-directed mutagenesis of the pCDM-CD16A-Fc48R/158v was performed to create the pCDM-CD16A-Fc48R/158F. 10 ng of the pCDM-CD16A-Fc48R/158V plasmid was incubated with 125 ng of the primers P41, 5' CTGCAGGGGGCTTTTTGGGAGTAAAAATGTG 3'(SEQ ID NO 47) and P42, 5' CACATTTTTACTCCCAAAAAGCCCCCTGCAG 3' (SEQ ID NO 48), 25mM of each dNTP and 2.5 units of the PfuUltra HF DNA polymerase (Stratagene). The reaction was performed for 30 sec at 95°C, followed by 16 cycles with 30 sec at 95°C, 60 sec at 66 °C and 318 sec at 68°C. The product was digested by Dpnl restriction enzyme for 3 hours and the reaction was stopped finally by incubation at 80 °C for 20 min. Chemocompetent TOP10/P3 cells (Invitrogen) were transformed with 2 ul of the reaction mixture and 150 ul spread on LB plates containing 10 ug/ml tetracycline and 25 ug/ml ampicillin. The nucleic sequence encoding for the pCDM-CD16A-Fc48R/158F was confirmed by sequencing. CD16A-Fc48R/158F was expressed and purified as described Example 1.<br>
The FcyRIIIB receptor is expressed in humans as three different allelic variants designated NA1, NA2 and SH respectively (Koene et ah, 1997 supra; Koene et al., 1998 supra). Three synthetic genes encoding the human extracellular domain of the CD16B allelic forms were synthesized by Geneart (Regensburg, Germany). The nucleic acid sequences were cloned as a Hindlll/BamHl fragment into the pSEC plasmid containing the nucleic sequence of the Fc region of the receptor.  CD16B-FcNA1' CD16B-FcNA2 and<br>
CO<br>
CD16B-Fc    were expressed and purified as described Example 1.<br>
Example 16: Western-blot on different allelic variants of human CD16<br>
To demonstrate specificity of the scFv 4-LS21 for the CD16A isoform, Western-Blot analysis was performed on the different allelic variants of CD16A and CD16B. 750 ng of each of CD16A-Fc48R/158V, CD16A-Fc48R/158F, CD16B-FcSH, CD16B-FcNA1, CD16B-FcNA2 and NKp46-Fc (negative control) were loaded on an 8% SDS gel under non-reducing conditions and blotted onto a membrane.<br>
The anti-CD 16 scFv A9, the anti-CD 16A scFv 4-LS21 and the anti-NKp46 scFv (negative control) were incubated at 4ug/ml with the blot. Detection of scFv bound to the recombinant proteins was carried out using an anti-His-HRP conjugate (lug/mL). As<br><br><br>
controls HRP conjugated anti-human IgG and anti-His-HRP conjugate (lug/mL) were<br>
used.<br>
The 4-LS21 scFv revealed strong binding to the allelic forms of CD16A only: no binding<br>
to the CD16B alleles was observed.  This clearly demonstrated specificity of the human<br>
4-LS21 scFv for the A isoform of CD 16 and that 4-LS21 recognised both allelic variants<br>
of CD 16A.   In contrast the murine anti-CD 16 scFv A9 showed uniform binding to all<br>
allelic variants of both CD16A and CD16B.<br>
Example 17: ELISA on different allelic variants of human CD16<br>
To further demonstrate specificity of the scFv 4-LS21 for the CD16A isoform, an ELISA was performed using the different allelic variants. Wells of a Maxisorp™ plate were coated with 100 mM NaHC03 containing 200ng of CD16A-Fc48R/158v, CD16A-Fc48R/158F, CD16B-FcSH, CD16B-FcNA1, CD16B-FcNA2 or NKp46-Fc (negative control), respectively.<br>
The anti-CD 16A 4-LS21 scFv and the anti-NKp46 scFv were incubated with the plate at a concentration of 4 mg/ml. Detection of scFv bound to the recombinant proteins was carried out using an anti-His-HRP conjugate (1 mg/ml). The scFv 4-LS21 showed strong binding to both allelic forms of CD16A, whereas no binding to the CD16B alleles was observed (Figure 7A), thus demonstrating clear specificity of the 4-LS21 scFv for the CD16A isoform.<br>
In a parallel assay (Figure 7B) anti-CD 16 MAb A9 (1 mg/ml) was assayed against an identically coated plate. The MAb A9 (1 mg/mL) incubation was followed by a goat anti-mouse IgG HRP-conjugate at 0.5 mg/ml (minimal cross reactivity to human Fc). MAb A9 bound strongly to all allelic variants (both CD16A and CD16B).<br>
Example 18: Comparison of the affinity of the scFv 4-LS21 and the monoclonal Ab A9 to different allelic forms of human CD16 by surface plasmon resonance (SPR)<br>
The affinity constants (KD values) of scFv 4-LS21 and MAb A9 were determined by SPR for the allelic variants CD16A-Fc48R/158V, CD16A-Fc48R/l58F, CD16B-FcSH, CD16B-FcNA1 and CD16B-FcNA2 using the BIAcore 2000 biosensor system (Amersham-Pharmacia) according to the protocol described in Example 12. The calculated off- and on-rate constants and the respective KD values are summarized below in Table 5.<br><br><br>
As expected for MAb A9 (used as a control) a KD in the range of 4-6 x 10"9 M was calculated for all allelic variants, demonstrating promiscuous binding of MAb A9 to all alleles of all forms. In contrast scFv 4-LS21 again proved to be highly specific for the CD16A isoform. The 4-LS21 scFv exhibited a KD in the region of 6 x 10"8M for CD16A-Fc48R/158V and in the region of 4 x 10"8M for CD16A-Fc48R/158F, indicating that 4-LS21 scFv binds to both allelic forms of CD16A with approximately equal affinity. In contrast 4-LS21 exhibited no binding to any of the allelic forms of CD16B.<br><br><br>
Table 5: Data for the kinetics of binding of the scFv 4-LS21 and the MAb A9 to allelic variants of the CD16A and CD16B isoforms.<br><br>
scFv/Mab	Isoform &amp; allelic variant	kon <m-v koffts1 kd kon></m-v>
4-LS21 MAbA9	CD16Ap  48R/158V	5.54 ±1.97x10" 7.86 ± 2.16 x104	3.42 ± 0.14 x10'3 4.06 ±1.0x10""	6.17 x10"8 5.17 x10"9<br>
4-LS21 MAbA9	CD16Ap   48R/158F	7.13 ± 2.47x104 9.94 ±3.46x10"	3.23 ± 0.09 x10'3 3.80 ±1.0x10""	4.53 x10s 3.82 x10"9<br>
4-LS21 MAbA9	CD16B FcSH	7.79 ±1.90x10"	NO BINDING 5.28 ±4.93x10""	1.14 x10"8<br>
4-LS21 MAbA9	CD16B FcNA1	7.72 ±4.86x10"	NO BINDING 4.87 ±4.72x10""	8.91 x 10"9<br>
4-LS21 MAbA9	CD16B FcNA2	7.57 ±2.90x10"	NO BINDING 4.78 ±4.23x10'"	9.63 x10'9<br>
Example 19: Construction of the plasmid pSKK3 CD30xCD16 LSZI TandAb/(G2S)3 for expression of TandAb molecule in bacteria<br>
The genes encoding the VH and VL variable domains of the anti-CD30 and anti-CD 16A antibodies, were derived from the hybridoma HRS-3 or from the human scFv library, respectively, using the plasmids pHOG_scFv(G2S)3 ocCD30 (HRS-3) and PSKK2_scFv aCD16A (LS21).<br>
The VH aCD16LS21 was amplified by polymerase chain reaction with the primers P34, 5' CATCACGATATCAGAACCACCGGAGCCGCCGCTACCACCTGAGGACACGGTG ACCAGGGTTCCC 3'(SEQ ID NO 49) , and P36, 5' CCGGCCATGGCGCAGGTC CAGCTGGTACAGTCTGG 3'(SEQ ID NO 50), to introduce a nucleic acid sequence coding for the 9 amino acid linker (GlyGlySer)3 (SEQ ID NO 51) at the 3' end of the PCR product The resulting PCR fragment was digested with the restriction enzymes Nco HEco RV and cloned into the Nco l/Eco RV pHOG_scFv(G2s)3aCD30 (HRS-3) to create the hybrid plasmid PHOG_VHaCD16LS2l(G2S)3VLaCD30(G2S)3VHaCD30.<br>
The VL of the anti-CD 16A antibody 4-LS21 was amplified by polymerase chain reaction with the primers P35, 5' GGTCACCGTCTCCTCAGGTGGTAGCGGCGGCTC CGGTGGTTCTTCCTATGTGCTGACTCAGCCATCCTC 3' (SEQ ID NO 52), and P37, 5' CCTCTAGATTAGTGATGGTGATGGTGATGGGATCCTAGGACGGTCAG<br><br><br>
CTTGGTCCC 3' (SEQ ID NO 53), to introduce the nucleic sequence coding for the 9 amino acid linker (GlyGlySer)3 (SEQ ID NO 52) at the 5' end of the PCR product. The resulting PCR fragment was digested with the restriction enzymes Bsm Bl/Xba I and was cloned into the linearized Bsm Bl/Xba I digested hybrid plasmid pHOG_VHaCD16LS21(G2S)3VLaCD30(G2S)3VHaCD30 resulting in the plasmid pHOG CD30xCD16LS21 TandAb/(G2S)3. This plasmid was cut by NcoVXbal and ligated into the NcoVXbal linearized plasmid pSKK3.<br>
Example 20: Construction of the plasmids pSKK3 CD19xCD16 LS21 Tandab/(G2S)3 for expression of TandAb molecules in bacteria<br>
The genes encoding the VH and VL variable domains of the anti CD 19 antibody was derived from the hybridoma HD37 using the plasmid pSKK3_scFv(G2S)3 ocCD19 (HD37).<br>
Plasmid pSKK3_scFv(G2s)3 ocCD19 (HD37) was cut by Nco VBsm BI and the insert was purified on an agarose gel. The purified fragment was then digested with the restriction enzyme Eco RV. The Eco RV/ Bsm BI digested fragment was cloned into the linearized Eco RV/ Bsm BI plasmid pHOG CD30xCD16 LS21 TandAb/(G2S)3, resulting in the plasmid pHOG CD19xCD16 LS21 TandAb/(G2S)3. The plasmid pHOG CD19xCD16 LS21 TandAb/(G2S)3 was cut by NcoVXbal and ligated in the NcoVXbal linearized plasmid pSKK3. The resulting plasmid is pSKK3 CD19xCD16A LS21 TandAb/(G2S)3 encoding the anti CD 19 x anti CD16A TandAb.<br>
Example 21: Expression and purification of the CD30xCD16 LS21 TandAb/(G2S)3 and CD19xCD16 LS21 TandAb/(G2S)3 molecules in bacteria<br>
Samples of E. coli K12 strain RV308 (Maurer et al., 1980, J. Mol. Biol. 139:147-161) transformed with the expression plasmids pSKK3 CD30xCD16 LS21 TandAb/(G2S)3 and pSKK3 CD19xCD16 LS21 TandAb/(G2S)3 were grown overnight in 2xYT medium with 50 ug/ml ampicillin and 100 mM glucose (2xYTGA) at 28°C. Dilutions (1:50) of the overnight cultures in 2XYTGA were grown as flask cultures at 28°C with shaking at 200 rpm. When cultures reached an OD600 of 0.8, bacteria were sedimented by centrifugation at 9500g for 15 min and 20°C and resuspended in the same volume of fresh YTBS medium (2xYT containing 1 M sorbitol and 2.5 mM glycine betaine; Blacwell &amp; Horgan, 1991, FEBS Letters 295:10-12) supplemented with 50 ug/ml ampicillin.    IPTG was added to a final concentration of 0.2 mM and growth was<br><br><br>
continued at 21°C for 18-20 h. Cells were harvested by centrifugation at 14000g for 20 min at 4°C. To isolate soluble periplasmic proteins, the sedimented bacteria were resuspended in 5% of the initial volume of ice-cold 200 mM Tris-HCl, 20% sucrose, 1 mM EDTA, pH 8.0. After a 1 hour incubation on ice with occasional stirring, the spheroplasts were centrifuged at 14000g for 60 min at 4°C leaving the soluble periplasmic extract in the supernatant, and the spheroplasts plus the insoluble periplasmic material in the pellet. The periplasmic fractions were dialyzed against start buffer (50 mM Tris-HCl, 1 M NaCl, 50 mM Imidazole pH 7.0) at 4°C. The dialyzed solution containing recombinant product was centrifuged at 14000 g for 30 min at 4°C. Immobilized metal affinity chromatography (IMAC) was performed at 4°C using a 1 ml column of Chelating Sepharose Fast Flow (GE Healthcare Biosciences, Freiburg, Germany) charged with Cu2+ and equilibrated with 50 mM Tris-HCl, 1 M NaCl, pH 7.0 (start buffer). The sample was loaded by passing the sample over the column. It was then washed with twenty column volumes of start buffer followed by start buffer containing 50 mM imidazole until the absorbency (280 nm) of the effluent was minimal (about thirty column volumes). Absorbed material was eluted with 50 mM Tris-HCl, 1 M NaCl, 300 mM imidazole, pH 7.0. The eluted fractions containing the TandAb molecules were identified by Western-blot analysis using a Penta His™ Antibody BSA-free (Qiagen, Hilden, Germany) followed by a goat anti-mouse IgG labeled with horseradish peroxidase (Dianova, Hamburg, Germany).<br>
The positive fractions were pooled and subjected to buffer exchange with 50 mM Imidazole, 50 mM NaCl (pH 6.0) or 50 mM MES, 50 mM NaCl (pH 5.5) using prepacked PD-10 (GE Healthcare Biosciences, Freiburg, Germany) (CD30xCD16 LS21 TandAb/(G2S)3 and CD19xCD16LS21 TandAb/(G2S)3, respectively). The turbidity of protein solution was cleared by centrifugation. The final purification was achieved by ion-exchange chromatography on a MonoS HR5/5 column (GE Healthcare Biosciences, Freiburg, Germany) in 50 mM Imidazole, 50 mM NaCl (pH 6.0) or 50 mM MES, 50 mM NaCl (pH 5.5) (CD30xCD16 LS21 TandAb/(G2S)3 and CD19xCD16 LS21 TandAb/(G2S)3, respectively), with a linear 0-1 M NaCl gradient. The eluted fractions containing the TandAb molecules were identified by reducing 15% SDS-PAGE followed by Coomassie staining. The positive fractions were collected and buffer exchanged with PBS containing 50 mM Imidazole andlO mM Trehalose (pH 6.0 or 7.0) (CD30xCD16 LS21 TandAb/(G2S)3 and CD19xCD16LS21 TandAb/(G2S)3, respectively) using pre-packed PD-10 (GE Healthcare Biosciences, Freiburg, Germany).<br><br><br>
Example   22:   Characterization   of   the   CD30xCD16 "*'   Tandab/(G2S)3   and CD19xCD16 LS21 TandAb/(G2S)3 molecules by flow cytometry<br>
To measure binding to CD16A, HEK-293 cells were transiently transfected using the calcium phosphate method with the pcDNA3-CD16A-zeta plasmid (Dr. O. Mandelboim, University of Jerusalem, Israel) and harvested 40 hours after transfection for flow cytometry as described in Example 7. The hairy cell leukemia cell line JOK-1 (Schwartz-Albiez R, Dorken B, Monner DA, Moldenhauer G, (1991) Int Immunol 3:623, a kind gift from Dr. G. Moldenhauer, DKFZ, Heidelberg) was used to determine binding to CD19. For testing reactivity with CD30, staining was performed with the human Hodgkin cell line L540CY (kindly provided by Dr. V. Diehl, University of Cologne, Germany; Kapp U, Wolf J, von Kalle C, Tawadros S, Rottgen A, Engert A, Fonatsch C, Stein H, Diehl V. (1992) Ann Oncol. Sep;3 Suppl 4:21-3). Cell staining and analysis carried out essentially as described supra. All recombinant antibodies were detected by 10 |ig/mL MAb anti-hexa His 13/45/31-2 (Dianova, Hamburg, Germany) followed by 15 ug/mL FITC-conjugated goat anti-mouse IgG (Dianova).<br>
The flow cytometric analyses demonstrated strong binding of both the CD30xCD16LS21 TandAb and the CD19xCD16LS21 TandAb antibody to cells expressing the corresponding antigen. Both TandAb antibodies bound to CD16A-expressing cells with approximately equal affinity.<br>
Example 23: Affinity measurements<br>
Determination of the KD values of the scFv 4-LS21 on the allelic variants 48R/158V and 48R/158F of the CD16A antigen revealed comparable KD values of about 5 x 10'8M, as measured by surface plasmon resonance, indicating that both allelic variants are recognized with approximately the same affinity (see Example 12). This finding was confirmed with the 4-LS21 derived TandAb CD30xCD16LS21, which was also shown to bind both allelic variants of CD16A with almost identical affinities. In comparison to the 4-LS21 scFv, an improvement in affinity of more than factor 10 was observed with the TandAb anitbody due to increased avidity.<br>
Example   24:   Characterization   of   the   CD30xCD16LS21   TandAb/(G2S)3   and CD19xCD16LS21 TandAb/(G2S)3 molecules by cytotoxicity assays<br><br><br>
To determine whether the CD30xCD16Lbil and CD19xCD16L;521 TandAb antibodies could mediate in vitro cell cytotoxicity by recruiting NK cells to target cells and activation of the NK cells by crosslinking activatory receptors, non-radioactive cytotoxicity assays were performed essentially as described by T. Dreier et al. (2002, Int J Cancer 100:690-697). NK cells that were used as effector cells were enriched from PBMC as described previously. Purity and antigen expression of the enriched NK cells was checked by flow cytometry in each case (data not shown).<br>
CD30+ L540CY target cells or CD19+ JOK-1 or Raji target cells were cultured in RPMI 1640 medium supplemented with 10%FCS, 2mM L-glutamine and 100 IU/mL penicillin G sodium and lOOug/mL streptomycin sulfate (herein referred to as RPMI medium; all components from Invitrogen). For the cytotoxicity assay cells were labeled with lOuM calcein AM (Molecular Probes/Invitrogen) for 30 min in RPMI medium at 37°C. After gently washing the labeled cells were resuspended in RPMI medium to a density of lxl05/mL. lxlO4 target cells were then seeded together with lxl05 enriched NK cells with either the CD30xCD16LS21 TandAb or the CD19xCD16ALS21 TandAb in individual wells of a round-bottom 96-well microtiter plate in a total volume of 200uL/well. After centrifugation for 2 min at 200g the assay was incubated for 3 hours at 37°C in a humidified atmosphere with 5%C02. 15 minutes prior to the end of incubation 20uL of 10% Triton X-100 in RPMI medium was added to the wells containing target cells only. 20uL RPMI medium was added to all other wells. IOOUL of cell culture supernatant was harvested from each well after an additional centrifugation for 5min at 500g, and the fluorescence of the released calcein was measured at 520nm using a fluorescence plate reader (Victor 3, Perkin Elmer). On the basis of the measured counts, the specific cell lysis was calculated according to the following formula: [fluorescence (sample) - fluorescence (spontaneous)] / [fluorescence (maximum) - fluorescence (spontaneous)] x 100%. Fluorescence (spontaneous) represents the fluorescent counts from target cells in the absence of effector cells and antibodies and fluorescence (maximum) represents the total cell lysis induced by the addition of Triton X-100. Sigmoidal dose response curves were plotted using the Prism software (GraphPad Software) see Figure 8.<br>
The results demonstrate the specific lysis of target cells by TandAb-activated NK cells. In contrast, the CD19xCD16LS21 TandAb did not mediate lysis of CD19- L540CY target cells, and vice-versa, the CD30xCD16LS21 TandAb did not activate NK cells to kill<br><br><br>
CD30" JOK-1 cells. In the absence of NK cells neither the CD30xCD16L;&gt;iI TandAb nor the CD19xCD16LS21 TandAb showed any cytotoxic activity against target cells.<br><br>
WE CLAIM :<br>
1.	A binding molecule having specificity for at least one antigen, wherein the at least one antigen is FcgRIIIA and wherein the binding molecule does not specifically bind to FcgRIIIB.<br>
2.	A binding molecule according to claim 1, wherein the specificity for FcgRIIIA is provided by an antibody or antigen-binding fragment thereof.<br>
3.	A binding molecule according to claim 2 wherein the specificity for FcgRIIIA is provided by the antigen-binding fragment.<br>
4.	A binding molecule according to claim 3, wherein the antigen-binding fragment is selected from the group consisting of an immunoglobulin light chain, an immunoglobulin heavy chain, a VH domain, a VL domain, Fv, Fab, di-Fab, Fab', F(ab')2, scFv, and a CDR.<br>
5.	A binding molecule according to any one of claims 2 to 4, wherein the antibody or antigen-binding fragment comprises at least one human CDR.<br>
6.	A binding molecule according to claim 5, wherein the CDR has an amino acid sequence selected from the group consisting of SEQ ID NOs 20, 23, 26, 29, 33, 34,21,24, 27, 30, 31, 35,22, 25, 28, 32,17,18 and 19.<br>
7.	A binding molecule according to any one of claims 2 to 6, wherein the antibody of antigen-binding fragment comprises three human light chain CDRs.<br>
8.	A binding molecule according to claim 7, wherein the sequence of the human light chain CDR1 is selected from the group consisting of SEQ ID NOs 20, 23, 26, 29, 33 and 34, the sequence of the human light chain CDR2 is selected from the group consisting of SEQ ID NOs 21, 24, 27, 30, 31 and 35, and the sequence of the human light chain CDR3 is selected from the group consisting of SEQ ID NOs 22,25, 28 and 32.<br><br><br>
9.	A binding molecule according to any one of claims 2 to 8, wherein the antibody or antigen-binding fragment comprises three human heavy chain CDRs.<br>
10.	A binding molecule according to claim 9, wherein the sequence of the human heavy chain CDR1 is SEQ ID NO 17, the sequence of the human heavy chain CDR2 is SEQ ID NO 18 and the sequence of the human heavy chain CDR3 is SEQ ID NO 19.<br>
11.	A binding molecule according any one of claims 2 to 8, wherein the antibody or antigen-binding fragment comprises a human light chain variable region.<br>
12.	A binding molecule according to claim 11, wherein the human light chain variable region has an amino acid sequence selected from the group consisting of SEQ ID NOs 10-16.<br>
13.	A binding molecule according to any one of claims 2 to 12 wherein the antibody or antigen-binding fragment comprises a human heavy chain variable region.<br>
14.	A binding molecule according to claim 13, wherein the human heavy chain variable region has the amino acid sequence of SEQ ID NO 9.<br>
15.	A binding molecule according to any one of claims 2 to 14, wherein the antibody or antigen-binding fragment is fully human.<br>
16.	A binding molecule according to any one of the preceding claims, which is capable of activating human cells expressing FcgRIIIA.<br>
17.	A binding molecule according to claim 16, which is capable of inducing NK-cell mediated cell killing.<br>
18.	A binding molecule according to claim 15, which inhibits activation of human cells expressing FcgRIIIA.<br><br><br>
19.	A binding molecule according to any one of the preceding claims, wherein said binding molecule has specificity for at least one additional antigen.<br>
20.	A binding molecule according to claim 19, wherein the at least one additional antigen is a cell-surface antigen.<br>
21.	A binding molecule according to claim 20, wherein the cell-surface antigen is selected from the group consisting of CD 19, CD20, CD30, the Laminin Receptor Precursor, EGFR1, EGFR2, EGFR3, Ep-CAM, PLAP, Thomsen-Friedenreich antigen, MUC-1, IL4-R alpha, IL13-R, IGFR, FcsRI and CD5.<br>
22.	A binding molecule according to claim 20 or claim 21, wherein the cell-surface antigen is CD 19 or CD30.<br>
23.	A binding molecule according to claim 19 or claim 20, wherein the at least one additional antigen is from an infectious agent.<br>
24.	A binding molecule according to claim 23 wherein the infectious agent is a virus, a bacterium, a fungus, a mycoplasma, a parasite or a prion.<br>
25.	A binding molecule according to any one of claims 19 to 24 wherein the specificity for the additional antigen is provided by an antibody or antigen-binding fragment thereof.<br>
26.	A binding molecule according to claim 25, wherein the antibody is selected from the group consisting of a chimeric antibody, a CDR-grafted antibody, a humanized antibody, and a fully human antibody.<br>
27.	A binding molecule according to claim 25 or claim 26, wherein the specificity for the at least one additional antigen is provided by the antigen-binding fragment.<br>
28.	A binding molecule according to claim 26, wherein the antigen-binding fragment is selected from the group consisting of an immunoglobulin light chain, an<br><br><br>
immunoglobulin heavy chain, a VH domain, a VL domain, Fv, Fab, di-Fab, Fab', F(ab')2, scFv, and a CDR.<br>
29.	A binding molecule according to claim 27 or claim 28 wherein the binding molecule is a diabody or a TandAb.<br>
30.	A binding molecule according to any one of the preceding claims wherein said binding molecule further comprises a functional domain.<br>
31.	A binding molecule according to claim 30, wherein the functional domain is selected from the following: an antibody Fc domain, an enzyme for use in antibody-dependent enzyme prodrug therapy, a peptide capable of binding to Fc receptors, a protein or peptide for increasing the serum half-life of the binding molecule.<br>
32.	A binding molecule according to any one of the preceding claims wherein said binding molecule is capable of binding to the FcgRIIIA158F allelic variant with approximately the same affinity with which it binds to the FcgRIIIA158V allelic variant.<br>
33.	A binding molecule according to any one of the preceding claims, wherein the binding molecule is conjugated to a labeling molecule or a toxin.<br>
34.	A binding molecule according to claim 33, wherein the labeling molecule is a radiolabel, a fluorescent label or a luminescent label.<br>
35.	A composition comprising a binding molecule according to any one of the preceding claims and at least one additional component.<br>
36.	A composition according to claim 35, comprising a suitable pharmaceutical carrier, excipient, diluent and/or stabilizer.<br>
37.	A polynucleotide encoding a binding molecule as defined in any one of claims 1 to 34.<br><br><br>
38.	A polynucleotide according to claim 37, wherein the antibody or antigen-binding<br>
fragment having specificity for FcgRIIIA comprises<br>
a)	a human light chain variable region encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 2-8 and/or;<br>
b)	a human heavy chain variable region encoded by the nucleic acid sequence of SEQ ID NO 1.<br><br>
39.	A polynucleotide according to claim 37 or claim 38 wherein said polynucleotide encodes: a first light chain variable region encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 2-8, a first heavy chain variable region encoded by a nucleic acid sequence of SEQ ID NO 1, a second light chain variable region, and a second heavy chain variable region, wherein said second light and heavy chain variable regions have affinity for CD 19.<br>
40.	A polynucleotide according to claim 37 or claim 38 wherein said polynucleotide encodes: a first light chain variable region encoded by a nucleic acid sequence selected from the group consisting of SEQ ID NOs 2-8, a first heavy chain variable region encoded by a nucleic acid sequence of SEQ ID NO 1, a second light chain variable region, and a second heavy chain variable region, wherein said second light and heavy chain variable regions have affinity for CD30.<br>
41.	A vector comprising a polynucleotide according to any one of claims 37 to 40, operably linked to a promoter region and optionally to a transcriptional terminator region.<br>
42.	A vector according to claim 41, wherein the polynucleotide is operably linked to a promoter region and to a signal sequence.<br>
43.	A host cell comprising a polynucleotide according to any one of claims 37 to 40, or a vector according to claim 41 or claim 42.<br><br><br>
44.	A host cell according to claim 43, wherein said host cell is selected from the group consisting of a bacterial cell, a fungal cell, insect cell, mammalian cell, or plant cell.<br>
45.	A method for the production of a binding molecule according to any one of claims 1 to 34 comprising, introducing into a host cell a polynucleotide according to any one of claims 37 to 40, or a vector according to claim 41 or claim 42, and culturing the host cell under conditions whereby the binding molecule is expressed.<br>
46.	A method according to claim 45, further comprising the isolation and optionally further purification of the binding molecule.<br>
47.	A method for the production of a binding molecule according to claim 33 comprising;<br><br>
(a)	introducing into a host cell a polynucleotide according to any one of claims 37 to 40, or a vector according to claim 4l or claim 42,<br>
(b)	culturing  the  host  cell  under  conditions  whereby  a binding  molecule according to any one of claims 1 to 32 is expressed,<br>
(c)	isolating the binding molecule expressed in step (b), and<br>
(d)	conjugating the isolated binding molecule from step (c) with a label or a toxin.<br><br>
48.	A binding molecule according to any one of claims 1 to 34, or a composition according to claim 35 or claim 36, for use in the diagnosis or treatment of an autoimmune disease, inflammatory disease, infectious disease, allergy or cancer.<br>
49.	A binding molecule or composition according to claim 48, wherein the cancer is selected from the group consisting of non-Hodgkin's lymphoma, chronic lymphocytic leukemia, Hodgkin's disease, solid tumour, minimal residual disease, and metastatic tumour.<br>
50.	Use of a binding molecule according to any one of claims 1 to 34, or a composition according to claim 35 or claim 36, in the manufacture of a<br><br><br>
medicament for the treatment of non-Hodgkin's lymphoma, chronic lymphocytic leukemia, Hodgkin's disease, solid tumour, minimal residual disease, or metastatic tumour.<br>
51.	Use of binding molecule according to claim 33 or claim 34 as reagent for: staining of cells expressing FcgRIIIA, analyzing patient cell profiles ex-vivo, or isolating NK cells for ex-vivo therapy.<br>
52.	A kit comprising a binding molecule according to any one of claims 1 to 34 and means for detecting the binding molecule when bound to FcgRIIIA.<br><br><br><br><br>
ABSTRACT<br>
The present invention relates to binding molecules that specifically bind to the human Fc gamma receptor expressed on the surface of natural killer (NK) cells and macrophages (i.e. FcgRIIIA), and in particular binding molecules that specifically bind the A form FcgRIII but do not bind to the B form of FcgRIII, as well as to the use of such binding molecules in the diagnosis and treatment of disease. The invention further extends to polynucleotides encoding such binding molecules, host cells comprising such polynucleotides and methods of producing binding molecules of the invention using such host cells.<br><br><br><br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUFCU1RSQUNUKEdSQU5URUQpLSg0LTctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-ABSTRACT(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWFic3RyYWN0LmRvYw==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUFOTkVYVVJFIDEoMjgtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-ANNEXURE 1(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNBTkNFTExFRCBQQUdFUygyOC01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CANCELLED PAGES(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNMQUlNUyhBTUVOREVEKS0oMi0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CLAIMS(AMENDED)-(2-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNMQUlNUyhBTUVOREVEKS0oMjgtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CLAIMS(AMENDED)-(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNMQUlNUyhHUkFOVEVEKS0oNC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CLAIMS(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNMQUlNUyhNQVJLRUQgQ09QWSktKDItMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CLAIMS(MARKED COPY)-(2-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNMQUlNUyhNQVJLRUQgQ09QWSktKDI4LTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CLAIMS(MARKED COPY)-(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNPUlJFU1BPTkRFTkNFIDIwLTYtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CORRESPONDENCE 20-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNPUlJFU1BPTkRFTkNFKDExLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CORRESPONDENCE(11-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWNvcnJlc3BvbmRlbmNlKDIxLTUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-correspondence(21-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNPUlJFU1BPTkRFTkNFKDI3LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CORRESPONDENCE(27-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNPUlJFU1BPTkRFTkNFKDgtOS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CORRESPONDENCE(8-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUNPUlJFU1BPTkRFTkNFKElQTyktKDQtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-CORRESPONDENCE(IPO)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWNvcnJlc3BvbmRlbmNlLXJlY2VpdmVkLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-correspondence-received.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LURFU0NSSVBUSU9OKEdSQU5URUQpLSg0LTctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-DESCRIPTION(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LURSQVdJTkcoR1JBTlRFRCktKDQtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-DRAWING(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0gMSgxNC01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form 1(14-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUZPUk0gMTgoOC05LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-FORM 18(8-9-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUZPUk0gMihHUkFOVEVEKS0oNC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-FORM 2(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0gMih0aXRsZSBwYWdlKS0oMjItMTEtMjAwNykucGRm" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form 2(title page)-(22-11-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDQtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-FORM 2(TITLE PAGE)-(GRANTED)-(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0gMygyMS01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form 3(21-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LUZPUk0gMyg2LTktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-FORM 3(6-9-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tMi0xLmRvYw==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-2-1.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tMi5kb2M=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LWZvcm0tcGN0LXJvLTEwMS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-form-pct-ro-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3KDItMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-PETITION UNDER RULE 137(2-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVBPV0VSIE9GIEFUVE9STkVZIDIwLTYtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-POWER OF ATTORNEY 20-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgyLTMtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-REPLY TO EXAMINATION REPORT(2-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCg2LTktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-REPLY TO EXAMINATION REPORT(6-9-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVJFUExZIFRPIEhFQVJJTkcoMjgtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-REPLY TO HEARING(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVNFUVVFTkNFIExJU1RJTkcoMi0zLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-SEQUENCE LISTING(2-3-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVNFUVVFTkNFIExJU1RJTkcoNC03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-SEQUENCE LISTING(4-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1NVU1OUC0yMDA3LVUuUy4gRE9DVU1FTlRTKDYtOS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">1966-MUMNP-2007-U.S. DOCUMENTS(6-9-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTk2Ni1tdW1ucC0yMDA3LXdvIGludGVybmF0aW9uYWwgcHVibGljYXRpb24gcmVwb3J0KDIyLTExLTIwMDcpLnBkZg==" target="_blank" style="word-wrap:break-word;">1966-mumnp-2007-wo international publication report(22-11-2007).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="253210-recombinant-human-interferon-like-protein.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253212-system-for-controlling-regeneration-of-lean-nox-traps.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253211</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1966/MUMNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>27/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>06-Jul-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>22-Nov-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>AFFIMED THERAPEUTICS AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>NEUENHEIMER FELD 582, 69120 HEIDELBERG,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>KIPRIJANOV SERGEJ</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KNACKMUSS STEFAN HANS JOACHIM</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LE GALL FABRICE</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
										<tr>
											<td>4</td>
											<td>LITTLE MELVYN</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
										<tr>
											<td>5</td>
											<td>REUSCH UWE</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
										<tr>
											<td>6</td>
											<td>HOFFMANN KARIN</td>
											<td>AFFIMED THERAPEUTICS AG, IM NEUENHEIMER FELD 582, 69120 HEIDELBERG</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K16/28 A61K39/395</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2006/005057</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-05-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0510790.9</td>
									<td>2005-05-26</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253211-anti-cd16-binding-molecule by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:57:45 GMT -->
</html>
