<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253257-method-for-detection-of-mutant-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:56:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253257:METHOD FOR DETECTION OF MUTANT GENE</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHOD FOR DETECTION OF MUTANT GENE</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed is a method for detecting a mutation in nucleotide sequence, which comprises performing a nucleic acid amplification reaction by using an oligonucleotide or a salt thereof as a primer and using a nucleic acid in a sample as a template and detecting a reaction product, wherein the oligonucleotide is so modified at the second nucleotide from the 3&quot;-terminus as to inhibit the nucleic acid synthesis. Also disclosed is a kit for the method. It becomes possible to completely eliminate any false positive result in the determination and deal with various mutation patterns in a single run of PCR1 reaction. Therefore, it becomes possible to contract a rug-resistance determination system which can detect any possible genetic mutation by a single run of multiplex PCR.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970<br>
(39 of 1970)<br>
&amp;<br>
THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
(See Section 10; rule 13)<br>
"METHOD FOR DETECTION OF MUTANT GENE"<br><br><br>
WAKO PURE CHEMICAL INDUSTRIES, Ltd. of the address: 1-2, Doshomachi 3-Chome, Chuo-Ku, Osaka-shi, Osaka 540-8605, Japan.<br>
The following specification particularly describes and ascertains the nature of this invention and the manner in which it has to be performed:<br><br><br>
SPECIFICATION<br>
METHOD FOR DETECTION OF MUTANT GENE<br>
TECHNICAL FIELD<br>
The present invention relates to a specific method for detecting mutation(s) in nucleotide sequence and a kit to be<br>
used therefor.<br>
BACKGROUND ART<br>
In case of human, for example, one of the reasons for difference between individual subjects in susceptibility to<br>
diseases, reactivity to medical drugs, susceptibility to side effect and the like may be due to the involvement of a<br>
genetic mutation (base polymorphism of gene).    And so, focusing attention on this difference of nucleotide<br>
sequence, studies on the exploration of a suitable therapy for the individual constitution have been progressing.<br>
In addition, the base polymorphism of gene can also be genetic markers for various diseases.    Therefore, the<br>
clarification of such base polymorphism is important from the clinical point of view, and the establishment of a<br>
method for analyzing nucleotide sequence which is capable of detecting the respective mutant gene and a method<br>
for identifying such base polymorphism has long been desired.<br>
In addition, in the field of diagnosis of infectious diseases, there is an occasion where the infecting microorganism<br>
has developed resistance to drug by a genetic mutation.    Therefore, for the purpose of determining whether the<br>
microorganism isolated from the patient has developed resistance to the medical agent, the detection of a genetic<br>
mutation in the microorganism isolated from the patient is particularly important from a viewpoint of diagnostic<br>
methodology and also for the selection of appropriate therapeutic treatment.<br>
For example, it has been clarified that a mutation ofrpoB gene coding for RNA polymerase B subunit of genus<br>
Mycobacterium is involved in development of resistance to rifampicin in human tubercle bacillus {Mycobacterium<br>
tuberculosis), and the mutation in rpoB gene was observed for about 95% of the rifampic in-resistant tubercle<br>
bacilli.    And so, it may be possible to figure out whether the tubercle bacillus is resistant to rifampicin or not by<br>
detecting this gene and by determining the existence or non-existence of genetic mutation.<br>
In addition, as to the mutation in rpoB gene involved in the rifampicin resistance of tubercle bacillus, various<br>
patterns have been reported (VIVEKKAPUR, et al., J. Clin. Microbiol. 1944. p. 1095-1098).    Therefore, if the<br>
detection of plural number of mutant genes can be performed at the same time, improved accuracy in the<br>
diagnosis and increased throughput (the amount of information processed in a certain period) can be expected.<br>
As to the tubercle bacillus, the genetic mutation involved in the resistance to other medical drugs has also been<br>
2<br><br>
studied, and it is now coming to light the patterns for the respective mutations.<br>
Another example showing effectiveness of the detection of mutation is the detection of MRSA.    That is, as the<br>
methicillin-resistant Streptococcus aureus (MRSA) has mec gene, the resistance to methicillin can be recognized<br>
by the detection of the mec gene (MRSA can be detected).<br>
However, in order to detect the existence of mutant gene such as a point mutation, a single nucleotide<br>
polymorphism, and the like, difference of only one base in a huge genomic nucleotide sequence has to be detected,<br>
and in consequence, a very high specificity is required.<br>
Examples of known analytical technique of the nucleotide sequence which is conventionally and commonly used<br>
are, for example, the determination method of nucleotide sequence determination (sequencing method),<br>
TaqMan™ probe method (Genome Res., vol. 6, p.986, 1996), RFLP (restriction fragment length polymorphism)<br>
method (J. Clin. Invest., vol. 72, p.1262, 1983), ASP (allele-specific primer) method (WO 01/42498), ASO<br>
(allele-specific oligoprobe) method (Nature, vol. 324, p. 163, 1986), Single base extension method (Proc. Natl.<br>
Acad. Sci. USA, vol. 94, p.10756, 1997), Pyrosequencing method (Analytical Biochemistry, vol. 244, p.367,<br>
1997), Invader™ method (Nature Biotech., vol. 17, p.292, 1999), and so on.<br>
Among these methods, the method of nucleotide sequence determination, TaqMan™ probe method, RFLP method,<br>
ASO method, Single base extension method, Pyrosequencing method and the like are the methods in which a<br>
region comprising a mutation or a polymorphism is amplified in advance using nucleic acid amplification<br>
technique such as Polymerase Chain Reaction (PCR). then the detection procedure is performed.    Therefore, in<br>
order to obtain a clear signal with reduced background, to obtain a specific amplification product is primarily<br>
quite important.<br>
Invader™ method is an accepted method in principle, which can detect the mutation and polymorphism without<br>
any amplification reaction.    However, there are many cases where the sensitivity is insufficient for the detection<br>
when the method is practiced without amplification reaction.    Practically, same as the case with other methods, it<br>
is important to amplify the nucleotide of the measuring object to be detected in advance.<br>
Further, in the ASP method, after amplifying the mutant allele specifically by PCR using a mutant DNA-specific<br>
oligonucleotide comprised of about 20 bases and a template DNA, the PCR product obtained is subjected to gel<br>
electrophoresis, and the existence of a mutant allele is detected by detecting a band.    This method is eventually<br>
suitable for testing a large number of samples efficiently.<br>
However, in the conventional methods using PCR as described above including the ASP method, there are some<br>
3<br><br>
cases where a primer extension product is obtained even when mismatches exist in the primer, and so that there<br>
exists difficulty in obtaining a specific amplification product.    Therefore, the method is problematic in terms of<br>
stringency.<br>
In addition, there is a method which uses a primer having a nucleoside with a base which is not compJementary to<br>
a target gene at the second position from 3'end, and setting the polymorphic site to be detected at the 3'end<br>
thereof; incidentally the method has been modified from the ASP method based on the knowledge that the<br>
polymorphism-specific primer having SNP site at the second position from the 3'end thereof recognizes the<br>
polymorphism precisely.    There is a publication reporting that, in this method, detection of the polymorphic site<br>
existing at the 3'end is improved, in comparison with a primer having a nucleoside with a base complementary to<br>
a target gene at the second position from 3 'end (Non-patent Document 1).    However, even though this method is<br>
used, in some cases, such a primer extension product is obtained even when mismatches exist at the 3'end of the<br>
primer, and eventually, the method still had a problem in terms of detection sensitivity.<br>
Further, a method using an oligonucleotide as a primer has been reported (Patent Document 1), in which the 3'end<br>
of the nucleotide sequence thereof is set in a polymorphic site, and a 2'-0, 4'-C-ethylene nucleotide unit is used<br>
as the third nucleotide from 3'end thereof.<br>
In the above-described Patent Document, using a primer complementary to a wild type sequence in which the<br>
third nucleotide from 3:end thereof is a 2'-0, 4'-C-ethylene nucleotide unit (Example 1 of Patent Document 1)<br>
and a primer complementary to a mutant type sequence in which the third nucleotide from 3 'end thereof is a 2'-0,<br>
4'-C-ethylene nucleotide unit (Example 2 of Patent Document 1), PCR has been performed using a wild type<br>
sample as a template (Test Example 1 of Patent Document 1).    In this case, amplification of gene was confirmed<br>
when the primer complementary to the wild type DNA sequence was used, and amplification of gene was not<br>
confirmed when the primer complementary to the mutant type DNA sequence was used.    For this reason, in the<br>
above-described reference, it has been stated that the selective amplification of gene can be performed by this<br>
method.<br>
However, it cannot be concluded only by the above-described results that the above-described primer, which is<br>
complementary to the wild type sequence, is specific for the detection of the wild type.    That is, unless the result<br>
showing that when PCR with a wild type DNA sample as a template is performed by using a primer<br>
complementary to a wild type sequence, the primer extension products can be obtained, however, when PCR with<br>
a mutant type DNA sample as a template is performed by using the same primer, the primer extension products<br>
4<br><br>
can not be obtained, there can not be a proof showing that the above-described primer is capable of detecting wild type specifically.<br>
In fact, even when PCR is performed using "a primer which amplifies mutant type" and a sample from wild type gene according to this method, there is a problem that the primer extension products is obtained (emergence of false positive).    Namely, there is a problem that even when the primer extension product could be obtained by performing PCR using this primer for a sample, which sample is unclear whether it is wild type or mutant type, still the sample cannot be determined whether it is wild type.<br>
As to a possible cause of the above-described problem, the following reason can be considered.    That is, the thermostable DNA polymerase used for PCR has a 3' —» 5' polymerase activity and concurrently has a 3' —» 5' exonuclease activity.    Therefore, when a primer capable of amplifying the mutant type is bound to a sample comprising the wild type gene, the nucleotide base allocated at the 3'end of the primer, which base is the same or complementary to the mutant nucleotide gene, will be cleaved at the PCR step by the 3' —&gt; 5' exonuclease activity of the thermostable DNA polymerase.    Therefore, the 3'end of the above-described primer recognizes completely the gene sequence of the wild type sample, which normally should not be recognized, and the extension reaction according to the primer sequence is started by the action of the DNA polymerase.    In consequence, it is considered that, even though a primer which should amplify the mutant type has been used, and PCR has been performed using a sample comprising wild type gene as a template, the primer extension product has been produced, and thus such a nonspecific amplification is occurred.<br>
In addition, in general, the "specificity" is defined as the ability to give a decision for negative as negative.    And so, as a method for improving the specificity in the analysis of mutation and polymorphism using a nucleic acid amplification reaction, a method for inhibiting nonspecific amplification (emergence of false positive amplification) can be considered.    As a means for this purpose, optimization of amplification condition (increase of anneal temperature, decrease of cycle number, lowering of enzyme amount, change of enzyme species, decrease of dNTP concentration, decrease of Mg concentration, decrease of template DNA concentration, and the like), modification of primer design, utilization of hot start method using antibody or aptamer, utilization of modified oligonucleotide (PNA and LNA, and the like) which have been said to increase the specificity, and the like are considered.    However, there are some cases where these methods are effective for the suppression of nonspecific amplification, but in many cases the effectiveness is insufficient and optionally the suppression is also<br>
occurred in the specific amplification, and these each has drawback.<br>
5<br><br>
From these circumstances, establishment of a method for detecting mutation(s) in nucleotide sequence with<br>
exclusion of false positive and with high specificity has been desired.<br>
Patent Document 1: JP-A-2005-323583<br>
Patent Document 2; JP-A-] 993-271224<br>
Non-patent Document 1: Bioorganic &amp; Medical Chemistry, 2003, vol.11, p.2211-2226<br>
Non-patent Document 2: V1VEK KAPUR et al., J. Clin. Microbiol., vol.32, No.4, 1994, pl095-1098<br>
Non-patent Document 3: Koji Morita et al., Bioorganic &amp; Medicinal Chemistry Letters, vol.12, 2002, p.73-76<br>
DISCLOSURE OF THE INVENTION<br>
PROBLEM TO BE SOLVED BY THE INVENTION<br>
It is an object of the present invention to provide a method for detecting mutation(s) in nucleotide sequence with<br>
improved specificity and a kit for detecting genetic polymorphism.<br>
MEANS FOR SOLVING PROBLEMS<br>
The present invention has been made for the purpose of solving the above-described problems, and comprises the<br>
following constituents (I) and (II).<br>
(T) A method for detecting mutation(s) in nucleotide sequence comprising:<br>
performing a nucleic acid amplification reaction by using any one of the following oligonucleotides (a) to (d) or a<br>
salt thereof as a primer, and using a nucleic acid in a sample as a template, and<br>
detecting reaction product;<br>
(a) an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(b) an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end<br>
6<br><br>
position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position   where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene except for its<br>
3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the<br>
5'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(d)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibitrng a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof.<br>
(II) A kit for detecting mutation(s) in nucleotide sequence, comprising any one of the following oligonucleotides<br>
(a) to (d) or a salt thereof as a primer:<br>
(a) an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end<br>
7<br><br>
position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene except for its<br>
3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the<br>
5'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(d)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
8<br><br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof.<br>
The present inventor has studied to solve the above-described problems with respect to the method for detecting a<br>
gene mutation.    As a result, an oligonucleotide in which, once the position of gene mutation is designed to be at<br>
the 3'end thereof, and a nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis<br>
(optionally described as "nucleotide analog") is disposed at the second position from 3'end thereof, was designed.<br>
And then the present inventor has found that when this oligonucleotide is used as a primer for PCR, all the<br>
problems as described above, for example, reading mistake of the template occurred when a single base deletion<br>
is caused by the 3' —&gt; 5' exonuclease activity of the DNA polymerase utilized, can be solved, and as the result, the<br>
generation of false positive result in the primer extension product is suppressed, and the genetic mutation can be<br>
detected with high precision, and thus completed the present invention.<br>
EFFECTS OF THE INVENTION<br>
The detection method of the present invention can be used, for example, for the microbial sensitivity test to check<br>
the resistance characteristics of bacteria which has been an essential test in the diagnosis (treatment) of<br>
tuberculosis.    In particular, by using in-line (on chip PCR system) separation detection method, the gene<br>
diagnosis and determination of drug resistance of tubercle bacillus and nontuberculous acid-fast bacterium disease<br>
can be performed at the same time in a single tube (in a single line).    This is a point of great advantage.<br>
Particularly, in tubercle bacilli, 90 to 95% of rifampicin-resistant bacteria have a mutation in rpoB gene.    And so,<br>
if the detection method of the present invention is performed, a mutation of the tubercle bacillus isolated from<br>
patient can be detected in a short time.    By doing this, it will be possible to determine whether the patient is<br>
infected with rifampicin-resistant bacteria in short order, and to treat promptly.    In addition, according to the<br>
present invention, the possibility of plural number of mutations in rpoB gene of the rifampicin-resistant bacteria<br>
can be detected and determined at once.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
Fig. 1 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using the Primer rpoB_2 and using plasmid DNA comprising mutant type rpoB gene<br>
9<br><br>
sequence_2 as a template obtained in Example 1.<br>
Fig. 2 shows a result of an analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using the Primer rpoB_2 and using plasmid DNA comprising wild type rpoB gene sequence<br>
as a template obtained in Example 1.<br>
Fig.3 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using the Primer rpoB_2_n3 and using plasmid DNA comprising mutant type rpoB gene<br>
sequence_2 as a template obtained in Comparative Example 1.<br>
Fig.4 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using the Primer rpoB_2_n3 and using piasmid DNA comprising wild type rpoB gene<br>
sequence_2 as a template obtained in Comparative Example 1.<br>
Fig. 5 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a Primer rpoB_6 and using plasmid DNA comprising mutant type rpoB gene<br>
sequence_6 as a template obtained in Example 2.<br>
Fig. 6 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a Primer rpoB_6 and using plasmid DNA comprising wild type rpoB gene sequence as<br>
a template obtained in Example 2.<br>
Fig. 7 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a mixed primer of Primer rpoB_2, Primer rpoB_3 and Primer rpoB_4 and using plasmid<br>
DNA comprising mutant type rpoB gene sequence_2 as a template obtained in Example 3.<br>
Fig. 8 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a mixed primer of Primer rpoB_2, Primer rpoB_3 and Primer rpoB_4 and using plasmid<br>
DNA comprising mutant type rpoB gene sequence_3 as a template obtained in Example 3.<br>
Fig. 9 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a mixed primer of Primer rpoB_2, Primer rpoB_3 and 'Primer rpoB_4 and using plasmid<br>
DNA comprising mutant type rpoB gene sequence_4 as a template obtained in Example 3.<br>
Fig. 10 shows a result of analysis of melting curve obtained based on the results of the real-time PCR by the<br>
intercalator method using a mixed primer of Primer rpoB_2. Primer rpoB_3 and Primer rpoB_4 and using plasmid<br>
DNA comprising wild type rpoB gene sequence as a template obtained in Comparative Example 2.<br>
BEST MODE FOR CARRYING OUT THE INVENTION<br>
10<br><br>
The method for detecting mutation(s) in nucleotide sequence according to the present invention is<br>
"A method for detecting mutation(s) in nucleotide sequence comprising:<br>
performing a nucleic acid amplification reaction by using any one of the following oligonucleotides (a) to (d) or a<br>
salt thereof as a primer, and using a nucleic acid in a sample as a template, and<br>
detecting reaction product;<br>
(a)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene except for its<br>
3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the<br>
5'-side from a position where the reference nucleotide exist in the target gene, and<br>
11<br><br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof;<br>
(d) an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof."<br>
In the present invention, the term "mutation(s) in nucleotide sequence" includes, for example, "genetic<br>
polymorphism".    The term "genetic polymorphism" is, as used in this field in the customary sense, and also<br>
referred to as SNP (single nucleotide polymorphism).<br>
Specifically, for example, in a certain gene locus, a single base substitution that means substitution of a single<br>
base with another base, a single base deletion that means a single base is deleted, and a single base insertion that<br>
means a single base is inserted, are included.<br>
In the present invention, it should be noted that, "mutation" and "base polymorphism" are referred to having a<br>
different nucleotide sequence from the wild type sequence.    Generally, the mutation frequency of less than 1% is<br>
referred to as "mutation", and that of 1% or more is referred to as polymorphism.    However, rigid distinction in<br>
wording should not be intended in the present specification.<br>
The term "target gene" in the present invention includes a gene to be targeted in the detection of mutation(s) in<br>
nucleotide sequence, at least a region of the nucleotide sequence is known, and a has a possibility of mutation(s)<br>
in nucleotide sequence.    Specifically, for example, genes such as rpoB gene of human tubercle bacillus<br>
(Non-patent Document 2), cytochrome P4501A2, cytochrome P4502A6, cytochrome P4502C9, cytochrome<br>
P4502C19, cytochrome P4502D6, cytochrome P4502E1, thiopurine methyltransferase, N-acetyltransferase,<br>
UDP-glucuronosyltransferase, glutathione S-transferase, HLA, TCRa, APOE4, dopamine D3 receptor, tryptophan<br>
hydroxylase, angiotensin precursor, blood coagulation factor VII, leptin and the like are included.<br>
12<br><br>
It should be noted that the number of mutation(s) in a gene is not specifically limited.<br>
In the present invention, a nucleotide sequence which has not caused the "mutation(s) in nucleotide sequence"<br>
such as mutation (substitution), deletion and insertion as stated in the present invention (for example, nucleotide<br>
sequence of gene of wild type bacteria) is referred to as a "reference sequence".    In addition, in the reference<br>
sequence, a nucleotide at the position where the mutation may be caused is referred to as a "reference nucleotide".<br>
Also, "mutant nucleotide" is referred to as a nucleotide at the mutation position where the mutation has been<br>
caused for the reference sequence.<br>
In the present invention, the term "complementary nucleotide" is used for meaning in the customary sense in this<br>
field.    That is, it refers to a nucleotide composed of a nucleic acid base which is complementary to the base of<br>
the target nucleotide.<br>
In the present invention, the nucleic acid base composing a nucleotide may be represented by the abbreviations<br>
(adenine as "A", guanine as "G", cytosine as "C", and thymine as "T") as used customarily in the field of the<br>
present invention.<br>
The term "salt" used in the "oligonucleotide or the salt thereof of the present invention includes, for example,<br>
salts of alkaline metal such as sodium, potassium, or lithium; salts of alkaline earth metal such as calcium or<br>
magnesium; salts of metal such as aluminum, iron, zinc, copper, nickel, or cobalt; inorganic salts such as<br>
ammonium salt; organic acid salts such as salts of amine such as t-octylamine, dibenzylamine, morpholine,<br>
ethylenediamine, N-methylglucamine? guanidine, diethylamine, triethy 1 amine, dicyclohexylamine,<br>
N,N'-dibenzylethylenediamine, chloroprocaine, procaine, diethanolamine, N-benzyl-phenethylamine, piperazine,<br>
tetramethyl ammonium, or tri sfhydroxym ethyl am inomethane); salts of amino acid such as glycine, lysine, arginine,<br>
ornithine, glutamic acid, or aspartic acid.<br>
In the term "nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis" in the<br>
present invention, the term "having a modification capable of inhibiting a reaction of nucleic acid synthesis"<br>
means the use of, for example, a ribose derivative or an oxetane derivative having a structure "capable of<br>
inhibiting a reaction of nucleic acid synthesis" in place of, for example, normal 2-deoxy-D-ribose composing a<br>
nucleotide.    And the nucleotide or the salt thereof having such modification is the one which inhibits the reaction<br>
of nucleic acid synthesis and does not provide any replication products (amplification products), when the nucleic<br>
acid amplification reaction described hereinafter is performed using an oligonucleotide wherein this nucleotide is<br>
set at the 3'end thereof as a primer.<br>
13<br><br>
A specific example of the "nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis" in the present invention includes, for example, a nucleotide derivative represented by the following formula [I] or [I]],<br><br><br>
wherein B represents a nucleic acid base; R] represents an oxygen atom, a -NH-group or a lower alkylene group; R2 represents a lower alkylene group;<br><br><br>
[II]<br><br>
wherein B represents a nucleic acid base; R3 represents an oxygen atom, a nitrogen atom, a -NH-group or a lower<br>
alkylene group.<br>
In the above-described formula [I] and formula [11], the lower alkylene group shown as R|, R2 and R3 include<br>
methylene group, ethylene group, propylene group, butylene group, pentylene group, and hexylene group (these<br>
may be either straight chain, branched chain or cyclic).<br>
The specific examples of the nucleotide having a modification capable of inhibiting a reaction of nucleic acid<br>
synthesis represented by the formula [I] of the present invention are shown in Table 1 below.<br>
Table 1<br>
14<br><br><br>
15<br>
The specific examples ofthe nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis represented by the formula [II] of the present invention are shown in Table 2 below. Table 2<br><br><br>
The "nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis" represented by the formula [I] of the present invention can be synthesized using appropriate materials according to a synthetic method described, for example, inKoji Morita et al., Bioorganic &amp; Medicinal Chemistry Letters, vol.12, 2002, p.73-76 (Non-patent Document 3).    In this regard, 2,-0,4'-C-Ethylene-bridged Nucleic Acids (ENA) is commercially available and can the commercially available products may be employed in the present invention. The "nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis" represented by the formula [II] of the present invention can be synthesized using appropriate materials according to a synthetic method described, for example, in JP-A-1993-271224 (Patent Document 2).<br>
It should be noted that, as to the "nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis" of the present invention, any kind of nucleotide which has the same property as described above can be used, and the nucleotides are not limited to the nucleotides shown in Table 1 and Table 2. The primer of the present invention includes,<br>
(a)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof (hereinafter, optionally referred to as "Primer a");<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
16<br><br>
or a salt thereof (hereinafter, optionally referred to as "Primer b");<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the 3'end<br>
position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene except for its<br>
3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the<br>
5'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof (hereinafter, optionally referred to as "Primer c");<br>
(d)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target<br>
gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the<br>
target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis,<br>
or a salt thereof (hereinafter, optionally referred to as "Primer d").<br>
For example, when the Primer a of the present invention is designed, an oligonucleotide comprising a part or<br>
entire of the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from the<br>
position in the above-described target gene where the mutant nucleotide of the detection target may exist, except<br>
that the mutant nucleotide in the genetic mutation (for example, genetic polymorphism) of detection target is set at<br>
the 3'end thereof, and a nucleotide having a modification capable of inhibiting a reaction of nucleic acid synthesis<br>
is set at the nucleotide at the second position from 3'end" is designed; and then in consideration of the melting<br>
temperature (Tm value) and the like, the primer with an appropriate length of appropriate region may be designed<br>
from the above-described oligonucleotide.<br>
When the Primer b is designed, an oligonucleotide in which a nucleotide complementary to a mutant nucleotide<br>
(for example, genetic polymorphism) in a target gene is set at the 3'end thereof, and the other region of the<br>
17<br><br>
oligonucleotide is an oligonucleotide comprising a part or entire of the nucleotide sequence complementary to the<br>
nucleotide sequence of the above-described target gene toward the 3'-side from the position where the mutant<br>
nucleotide may exist in the above-described target gene, and the nucleotide at the second position from the 3'end<br>
thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis as described above, may be<br>
designed; and then in consideration of the melting temperature (Tm value) and the like, the primer with an<br>
appropriate length of appropriate region may be designed from the above-described oligonucleotides.<br>
When the Primer c is designed, a oligonucleotide in which the same nucleotide as a "reference nucleotide at the<br>
position in the target gene where the mutation may be caused" is set at the 3'end thereof, and the region except for<br>
its 3'end position is the same sequence as a nucleotide sequence (reference sequence) of the target gene, and has<br>
the same sequence as the nucleotide sequence of the target gene toward the 5'-side from a position where the<br>
reference nucleotide exist in the above-described target gene, and the nucleotide at the second position from the<br>
3'end thereof has the modification capable of inhibiting a reaction of nucleic acid synthesis is designed; and then<br>
in consideration of the melting temperature (Tm value) and the like, the primer with an appropriate length of<br>
appropriate region may be designed from the above-described oligonucleotide.<br>
When the Primer d is designed, an oligonucleotide in which a nucleotide complementary to the "reference<br>
nucleotide at the position in the target gene where a mutation may be caused" is set at the 3'end thereof, and the<br>
region except for its 3'end is complementary to the nucleotide sequence (reference sequence) of the target gene,<br>
and has the sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from the<br>
position in the above-described target gene where the above-described reference nucleotide exist; and the<br>
nucleotide at the second position from the 3 'end thereof has the modification capable of inhibiting a reaction of<br>
nucleic acid synthesis is designed; and ten in consideration of the melting temperature (Tm value) and the like, the<br>
primer with an appropriate length of appropriate region may be designed from the above-described<br>
oligonucleotide.<br>
The length of the primer of the present invention includes, preferably 10 bases or more, which has been<br>
considered for the base number necessary to maintain the specificity as primer sequence, and a primer having the<br>
length with 20 or more of bases is more preferable.    For example, the ones comprised of 15 to 30 bases are<br>
included.<br>
In order to obtain the primer of the present invention, it is preferable to prepare by the method of chemical<br>
synthesis well known per se, using a nucleotide which has a modification capable of inhibiting a reaction of<br>
18<br><br>
nucleic acid synthesis of the present invention obtained by the above-described method and a normal nucleotide.<br>
By this method, an oligonucleotide with constant quality can be obtained, easily, in large amount at low cost.<br>
For example, an oligonucleotide is synthesized using a DNA synthesizer usually used for the DNA synthesis by<br>
the commonly used phosphoramidite method, then purified by the conventional method using an an ion-exchange<br>
column chromatography, and thus the objective oligonucleotide of the present invention can be obtained.<br>
It should be noted that the oligonucleotides having a "nucleotide which has a modification capable of inhibiting a<br>
reaction of nucleic acid synthesis" of the present invention is available through the use of vendor's custom service<br>
of nucleic acid synthesis.<br>
The principle of the method for detecting mutation(s) in nucleotide sequence according to the present invention is<br>
described as follows:<br><principle>    the Primer a or the Primer b of the present invention is used:<br>
For example, when mutation in nucleotide sequence of a single-base mutation is intended to be detected,<br>
above-described Primer a or Primer b, an oligonucleotide primer capable of amplifying a sequence of interest by<br>
the nucleic acid amplification reaction in pairs with the above described primer, and a sample (nucleic acid<br>
comprising a nucleotide sequence to be detected for mutation in nucleotide sequence) are subjected to the nucleic<br>
acid amplification reaction in a reaction solution.    And, if the nucleotide of 3'end of the Primer a or the<br>
nucleotide of 3'end of the Primer b matches (or the base is complementary), namely if there exist a single-base<br>
mutation, the nucleic acid amplification reaction takes place.    In contrast, if the nucleotide of the 3'end does not<br>
match (or the base is not complementary), namely if there exist no single-base mutation, the nucleic acid<br>
amplification reaction does not take pface.<br><principle>    the Primer c or the Primer d of the present invention is used:<br>
For example, when mutation(s) in nucleotide sequence of a single-base mutation is intended to be detected,<br>
above-described Primer c or primer d, an oligonucleotide primer capable of amplifying a sequence of interest by<br>
the nucleic acid amplification reaction in pairs with the above-described primer, and a sample (nucleic acid<br>
comprising a nucleotide sequence to be detected for mutation(s) in nucleotide sequence) are subjected to the<br>
nucleic acid amplification reaction in a reaction solution.    And, if the nucleotide of 3 'end of the Primer c or the<br>
nucleotide of 3'end of the Primer d matches (or the base is complementary), namely if there exist the same<br>
nucleotide as the reference nucleotide of detection target, the nucleic acid amplification reaction takes place.    In<br>
contrast, if the nucleotide of 3'end does not match (or the base is not complementary), namely if there exist a<br>
19<br><br>
single-base mutation, the nucleic acid amplification reaction does not take place.<br>
For example, focusing on the mutation of rpoB gene of human tuberc]e bacillus as a genetic polymorphism to be detected, the Primer a to be used when the detection is performed according to the above-described <principle> may be designed as follows.<br>
Namely, for the detection of a mutation in rpoB gene, an oligonucleotide or the salt thereof in a target gene, (i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide having a mutant base at the 3'end position thereof, and (ii) wherein the oligonucleotide has the same nucleotide sequence except for its 3'end position, as the nucleotide sequence of rpoB gene toward the 5'-side from the position in rpoB gene where the above-described mutant nucleotide may exist, and (iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleotide synthesis, may be designed. Focusing on the mutation oirpoB gene of human tubercle bacillus as a genetic polymorphism to be detected, the Primer c to be used when the detection is performed according to the above-described <principle> may be designed as follows.<br>
Namely, for the detection of a mutation in rpoB gene, an oligonucleotide or the salt thereof, (i) wherein the oligonucleotide has the same nucleotide as the reference nucleotide in rPaBB gene at the 3'end position thereof; and, (ii) wherein the oligonucleotide has the same nucleotide sequence as the nucleotide sequence of rpoB gene at the position except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the rpoB gene toward the 5-side from a position where the above-described  reference nucleotide exist in rpoB gene, and (iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis; may be designed.<br>
As to the method for designing a primer of the present invention, for example, a specific example of designing the Primer a for detecting mutation of rpoB gene by the method based on the <pnnciple> is described below. The rpoB gene of human tubercle bacillus and the type of mutation for rifampcin resistance (RFP resistance) and its position in the gene have been described in FIG. I of VIVEK KAPIJR et al., J. Clin. Microbiol., vol.32, No.4, 1994. pl095-1098 (Non-patent Document 2).    Therefore, referring the description in this FIG 1, at first, a sequence comprising a single-base mutation is designed.<br>
For example, in rpoB gene of the wild type human tubercle bacillus, the mutation exists in some cases at the oligonucleotide region of 69 base pairs coding for the 511lh to 533rd amino acids.<br>
The oligonucleotide sequence of these 69 base pairs of the wild type hijman tubercle bacillus (hereinafter, referred<br>
20<br><br>
to as "the sequence of a hot spot mutation region in the wild type rpoB gene") is as follows.<br>
5'-CTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACCCACAAGCGCCGACTGTCGGC GCTGGGG-3' (SEQ ID NO: 1)<br>
And, as one of the mutations of this region, in the sequence of a hot spot mutation region in the wild type rpoB<br>
gene, the 46th base "C" (cytosine) from the 5'end in the sequence of the above-described SEQ ID NO:l is possible<br>
to be changed to "T" (thymine).    Consequently, an oligonucleotide with the following sequence and yet with a<br>
functional length as a primer can be designed so that this mutation would be exist at 3'end (namely, its 3'end<br>
would be "T"), and also the other region of it would be the same sequence as the nucleotide sequence of wild type<br>
rpoB gene toward the 5'-side from the above-described 46th base in the sequence of a hot spot mutation region in<br>
the wild type rpoB gene.    The underlined base corresponds to the mutant base.<br>
5'-GCTGTCGGGGTTGACCT-3' (SEQ ID NO:7)<br>
In the next place, the second base "C from the 3'end in the above-described sequence is designed to be the "C"<br>
having "a modification capable of inhibiting a reaction of nucleic acid synthesis of the present invention".    This<br>
is hereinafter referred to as "Primer rpoB_2".<br>
When the nucleic acid amplification reaction (PCR) is performed by using this primer, existence or nonexistence<br>
of the mutation at the 46th base "C" to "T" in the sequence of a hot spot mutation region in the wild type rpoB<br>
gene can be detected.<br>
In addition, in an example of another mutation, as to the oligonucleotide of SEQ ID NO:l, in the sequence of the<br>
hot spot mutation region in the wild type rpoB gene, the 68th base "T" from the 5'end in the sequence is possible<br>
to be changed to "C".    Consequently, an oligonucleotide with the following sequence and yet with a functional<br>
length as a primer can be designed so that this mutation would exist at 3'end (namely, 3'end would be "C"), and<br>
also the other region of it would be the same sequence as the nucleotide sequence of wild type rpoB gene toward<br>
the 5'-side from the above-described 68th base in the sequence of a hot spot mutation region in the wild type rpoB<br>
gene.    The underlined base corresponds to the mutant base.<br>
5'-CCGACTGTCGGCGCC-3'(SEQlDNO:8)<br>
In the next place, the second base "C" from the 3'end in the above-described sequence is designed to be the "C"<br>
having "a modification capable of inhibiting a reaction of nucleic acid synthesis of the present invention".    This<br>
is hereinafter referred to as "Primer rpoB_6".<br>
If the nucleic acid amplification reaction (PCR) is performed using this primer, existence or nonexistence of the<br>
21<br><br>
mutation at the 68th base "T" to "C" in the sequence of a hot spot mutation region in the wild type rpoB gene can<br>
be detected.<br>
Further, in an example of another mutation, as to an oligonucleotide of SEQ ID NO: 1, in the sequence of the hot<br>
spot mutation region in the wild type rpoB gene, the 46,h base "C" from the 5'end in the sequence is possible to be<br>
changed to "G" (guanine).    Consequently, an oligonucleotide with the following sequence and yet with a<br>
functional length as a primer can be designed so that this mutation would exist at 3'end (namely, its 3'end would<br>
be "G"), and also the other region of it would be the same sequence as the nucleotide sequence of wild type rpoB<br>
gene toward the 5'-side from the above-described 46lh base in the sequence of a hot spot mutation region in the<br>
wild type rpoB gene.    The underlined base corresponds to the mutant base.<br>
5'-GCTGTCGGGGTTGACCG-3' (SEQ ID NO:9)<br>
In the next place, the second base "C" from the 3'end in the above-described sequence is designed to be "C"<br>
having "a modification capable of inhibiting a reaction of nucleic acid synthesis of the present invention".    This<br>
is hereinafter referred to as "Primer rpoB_3".<br>
When the nucleic acid amplification reaction (PCR) is performed using this primer, existence or nonexistence of<br>
the mutation at the 46th base "C" to "G" in the sequence of a hot spot mutation region in the wild type rpoB gene<br>
can be detected.<br>
And further, in an example of another mutation, as to an oligonucleotide of SEQ IDNO:l, in the sequence of the<br>
hot spot mutation region in the wild type rpoB gene, the 47' base "A" (adenine) from the 5'end in the sequence is<br>
possible to be changed to "T".    Consequently, an oligonucleotide with the following sequence and yet with a<br>
functional length as a primer can be designed so that this mutation would exist at 3'end (namely, its 3'end would<br>
be "T"), and also the other region of it would be the same sequence as the nucleotide sequence of wild type rpoB<br>
gene toward the 5'-side from the above-described 47th base in the sequence of a hot spot mutation region in the<br>
wild type rpoB gene.    The underlined base corresponds to the mutant base.<br>
5'-CTGTCGGGGTTGACCCT-3' (SEQ ID NO: 10)<br>
In the next place, the second base "C" from the 3'end of the above-described sequence is designed to be "C"<br>
having "a modification capable of inhibiting a reaction of nucleic acid synthesis of the present invention".    This<br>
is hereinafter referred to as "Primer rpoB_4".<br>
When the nucleic acid amplification reaction (PCR) is performed by using this primer, existence or nonexistence<br>
of the mutation at the 47th base "A" to "T" in the sequence of a hot spot mutation region in the wild type rpoB<br>
22<br><br>
gene can be detected.<br>
The above-described examples are only the explanations of method of designing a primer of the present invention,<br>
and thus, when the other mutation would be detected, the oligonucleotides of the present invention of (a) - (d) or<br>
salt thereof having the structure as described above may be designed.<br>
It should be noted that the primer of the present invention may be labeled with a labeling substance.<br>
As to the labeling substance to be used for labeling the primer of the present invention with a labeling substance,<br>
any of the known labeling substances such as radioisotope, enzyme, fluorescent substance, luminescent substance<br>
and biotin may be used.<br>
For example, radioisotope such as 32P, 33P and 35S, enzymes such as alkaline phosphatase and horseradish<br>
peroxidase, fluorescent substance such as Cyanine Dye series of Cy3, Cy5 (Amersham Biosciences K.K.), and<br>
fluorescein, luminescent substance such as chemoluminescent reagent comprising acridinium ester, are included.<br>
When the primer of the present invention is labeled with radioisotope, a method of labeling by incorporation of a<br>
radioisotope-labeled nucleotide into the primer at the time when the primer is synthesized, or a method of labeling<br>
with radioisotope after the primer is synthesized or the like are included.    Specifically, a frequently-used random<br>
primer method, nick-translation method, 5'-terminal labeling method using T4 polynucleotide kinase, 3'-terminal<br>
labeling method using terminal deoxynucleotidyl transferase and RNA labeling method are included.<br>
included.<br>
When the primer of the present invention is labeled with an enzyme, the conventional technique in this field of<br>
direct labeling method by which the primer to be labeled is directly linked covalently with an enzyme molecule<br>
such as alkaline phosphatase, horseradish peroxidase or the like can be employed.<br>
When the primer of the present invention is labeled with fluorescent substance, for example, the fluorescently<br>
labeled nucleotide can be incorporated into the primer by conventional labeling technique in this field.    In<br>
addition, by a method of replacing a sequence with a nucleotide having a linker arm as a member of a<br>
oligonucleotide (See. for example, Nucleic Acids Res., 1986, vol. 14, p. 6115), the nucleotide can also be labeled<br>
with the fluorescent substance.    In that case, there can also be a method that a uridine having a linker arm on<br>
5-position is synthesized chemically from a deoxyuridine by a synthetic method disclosed in JP-A-1985-500717<br>
and then a fluorescent substance is introduced into the above-described oligonucleotide. In the methods of labeling<br>
with a luminescent substance and with biotin, the labeling can be carried out according to the conventional<br>
technique of luminescent-labeling or biotin-labeling of nucleotide usually conducted in this field.<br>
23<br><br>
An oligonucleotide primer capable of amplifying a sequence of interest by nucleic acid amplification reaction in<br>
pairs with the above-described primer of the present invention is an appropriate oligonucleotide capable of<br>
amplifying a sequence possible to contain the target mutant sequence (a sequence of interest) in the known<br>
sequence of detection target.<br>
The method for designing "an oligonucleotide primer capable of amplifying a sequence of interest in pairs with<br>
the primer of the present invention by nucleic acid amplification reaction when the nucleic acid amplification<br>
reaction is performed by using an oligonucleotide of the present invention as a primer", and the method for<br>
obtaining thereof include, for example, a method which is designed so as to prevent interference reaction with the<br>
primer of the present invention through the use of a primer prediction software such as Primer 3 (Whitehead<br>
Institute for Biomedical Research).<br>
In addition, if available, an oligonucleotide primer which has a characteristics of "an oligonucleotide primer<br>
capable of amplifying a sequence of interest in pairs with the primer of the present invention by the nucleic acid<br>
amplification reaction when the nucleic acid amplification reaction is performed by using an oligonucleotide of<br>
the present invention as the primer" as described above, and the one usually used in this field can be used.    The<br>
various types of primers are commercialized (for example, M13 Forward Primer, TAKARA BIO INC.), and are<br>
also available.<br>
The "nucleic acid amplification reaction" involved in the "method for detecting mutation(s) in nucleotide<br>
sequence of the present invention comprising: performing a nucleic acid amplification reaction by using the<br>
primer of the present invention" include, a conventional method usually conducted in this field in which, primer<br>
extension is performed using, for example, a primer of the present invention, by hybridizing this with a nucleic<br>
acid in a sample, and by conducting nucleic acid amplification with DNA polymerase and the like [for example,<br>
PCR (JP-A-1985-281), LAMP (Loop-mediated Isothermal Amplification) method (Tsugunori Notomi et al.,<br>
Nucleic Acid Res., 28, e63, 2000), ICAN (Isothermal and Chimeric primer-initiated Amplification of Nucleic<br>
acids) method (Rinsho Byori (Jpn. J. Clin. Pathol.) 51(11), 1061-1067, 2003, Nov.), LCR (ligase chain reaction)<br>
method (JP-A-1992-211399), SDA (strand displacement amplification) method (JP-A-I996-19394)].    The<br>
reaction condition of the nucleic acid amplification reaction including PCR, the operation procedure and the like<br>
may be according to the conventional method usually used in this field.<br>
The "method for detecting reaction product obtained by the nucleic acid amplification reaction" of the present<br>
invention includes various detection methods such as, for example, an intercalator method, a TaqMan™ real-time<br>
24<br><br>
PCR method (see, for example, the specification of US Patent No. 5,538,848), MGB Eclipse Probe System method (see, for example, the specification of US Patent No. 5,803,155), Molecular Beacons Probe Technology method (see, for example, the specification of US Patent No. 5,925,517), LUX Fluorogenic Primer method (Invitrogen Corporation), Quenching probe-PCR (QP) method (see, for example, the specification of US Patent No. 6,492,121), a method in which, after the nucleic acid amplification reaction is performed, the primer extension products obtained are subjected to the electrophoresis, and the detection is performed based on the results of the electrophoresis, a method in which the determination is performed by measuring the signal derived from the primer extension product obtained by the nucleic acid amplification reaction using the labeled primer. As to a method for determining amplification products (primer extension products) obtained by the nucleic acid amplification reaction, taking the following 4 methods for instance, an explanation will be given for each method; however, as a matter of course, it should not be limited thereto.<br>
(1)	Intercalator method.<br>
(2)	TaqMar\7M reaf-ti'me PCR method.<br>
(3)	a method in which, after the nucleic acid amplification reaction is performed, the primer extension products obtained .are subjected to the electrophoresis, and the detection is performed based on the results of the electrophoresis,<br>
(4)	a method in which the determination is performed by measuring the signal derived from the primer extension product obtained by the nucleic acid amplification reaction vising the labeled primer,<br>
(1) Intercalator method<br>
The conventional intercalator method in which the real-time PCR is performed using known intercalator can be<br>
utilized.<br>
For example, a method of performing the real-time PCR utilizing conventional intercalator method using a primer<br>
of the present invention and an oligonucleotide primer capable of amplifying a sequence of interest by nucleic<br>
acid amplification reaction in pairs with the above-described primer, is included.<br>
The intercalator is a reagent capable of generating fluorescence by binding specifically with a double-stranded<br>
DNA, and generates fluorescence when exposed to excitation light.    When the DNA is increased by repeated<br>
amplification of PCR, the intercalator is incorporated into the DNA accordingly.    That is, as the amount of<br>
intercalator incorporated into DNA is proportional to the amount of the amplification product, the amount of the<br>
primer extension product can be determined by detecting the fluorescence intensity derived from the intercalator.<br>
25<br><br>
In this regard, however, as the intercalator binds to the entire double-stranded DNA, the melting curve analysis is<br>
performed based on the results of measurement of the fluorescent intensity.    That is, after PCR, fluorescence<br>
intensity from the intercalator is measured in parallel with increasing the temperature of the reaction solution of<br>
PCR comprising the obtained amplification products.    In the beginning, fluorescence is generated because the<br>
amplification product forms double strand.    However, when the temperature of the reaction solution of PCR<br>
reaches to a certain temperature, the amplification products will dissociate to single strand, and the fluorescence<br>
intensity from the intercalator decreases immediately.    The temperature at the time is the melting temperature<br>
(Tm value), and is an intrinsic value of the sequence of a primer extension product.    A specific product (an<br>
objective product) and a non-specific product may be identified from this Tm value.<br>
This intercalator method does not require electrophoresis after the real-time PCR, and therefore, is an effective<br>
method when the rapid determination of mutation(s) is required in such a case of clinical testing (diagnosis).<br>
The intercalator to be used for the present invention includes, while any type of the intercalator usually used in<br>
this field, for example, SYBR™ Green I (Molecular Probes Inc.), ethidium bromide, fluorine and the like can be<br>
utilized.<br>
The example of "the method for detecting mutation in nucleotide sequence by performing intercalator method" of<br>
the present invention would be explained as follows:<br><method><br>
Using a Primer a or a Primer b of the present invention, an oligonucleotide primer capable of amplifying a<br>
sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer, and<br>
SYBRT M Green I as the intercallator, and using a purified DNA sample purified from a sample to be detected for<br>
mutation as a template, real-time PCR is performed using Taq DNA polymerase.    And the fluorescent intensity<br>
emitted from SYBRT M Green I intercalated into the amplification products is measured.<br>
In order to detect mutation, for example, analysis on the melting curve of amplification products may be<br>
performed.    That is, the melting curve is made by plotting the melting temperature of amplification product<br>
(double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical<br>
axis, and detection of peak is examined.    When the Tm value of the detected peak of each primer extension<br>
product is identical with that of the primer extension product predicted to be amplified by the real-time PCR using<br>
the same combination of primers, it can be determined that there is a objective, that is, mutation of detection<br>
target(s) in nucleotide sequence in the purified DNA sample.<br>
26<br><br>
In addition, a method may also be used is that, at first, as to the amplified product for each dilution series of solution comprising purified DNA sample used as a template, the melting curve is made by plotting the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical axis, and detection of peak is examined.    When the Tm value of the peaks of each primer extension product of each dilution series is identical, it can be determined that there is a target mutation in nucleotide sequence in the purified DNA sample.<br>
In addition, as a reference experiment, the real-time PCR is performed by the same method as described above except for using a purified DNA sample derived from the wild type instead of a purified DNA sample purified from a sample to be detected for mutation as a template, and by the same method, the fluorescence intensity emitted by the SYBR      Green I is measured, and then the melting curve analysis may be performed.    In this case, as there is no mutation in nucleotide sequence in the sample, fluorescence could not be detected, and therefore, a peak should not appear in the melting curve analysis.    To make the determination of mutation in nucleotide sequence more assured, it is preferable to conduct such a control experiment in parallel. <method><br>
Using a Primer c or a Primer d of the present invention, an oligonucleotide primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer, and SYBR      Green I as the intercallator, and using a purified DNA sample purified from a sample to be detected for mutation as a template, real-time PCR is performed using Taq DNA polymerase.    And the fluorescent intensity emitted from SY"BRl'   Green I intercalated into the amplification products is measured. In order to detect mutation, for example, analysis on the melting curve of amplification products may be performed.    That is, the melting curve is made by plotting the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical axis, and detection of peak is examined.    When the Tm value of the detected peak of each primer extension product is identical with that of the primer extension product predicted to be amplified by the real-time PCR using the same combination of primers, it can be determined that there is no target mutation in nucleotide sequence in the purified DNA sample.    On the other hand, when the Tm value is different, it can be determined that there is a target mutation(s) in nucleotide sequence in the purified DNA sample. In addition, a method may also be used is that, at first, as to the amplified product for each dilution series of<br>
solution comprising purified DNA sample used as a template, the melting curve is made by plotting the melting<br>
27<br><br>
temperature of amplification product (double-stranded DNA) as horizontal axis and the first derivation (variation)<br>
of fluorescent intensity as vertical axis, and detection of peak is examined_    When the Tm value of the peak of<br>
each primer extension product of each dilution series is identical, it Can determined that there is no target<br>
mutation in nucleotide sequence in the purified DNA sample.    On the other hand, when the Tm value is different,<br>
it can be determined that there is a target mutation(s) in nucleotide sequence in the purified DNA sample.<br>
As an example, a method for detecting mutation in nucleotide sequence  by the real-time PCR detection system<br>
using intercalator according to the present invention will be explained by taking a case where a single-base<br>
mutation in the rpoB gene of human tubercle bacillus is detected using the above-described "Primer rpoB 2"<br>
(corresponding to the Primer a of the present invention), (a case where the mutation of base "C" to "T" at the 46th<br>
from the 5'end in the sequence of a hot spot mutation region in the wild type rpoB gene is detected as described<br>
above) (<method>) for instance.<br>
Al first, by the known method, the purified DNA sample is obtained from a sample to be detected for a mutation<br>
in nucleotide sequence.<br>
In the next place, using the Primer rpoB_2 and an oligonucleotide primer capable of amplifying a sequence of<br>
interest by the nucleic acid amplification reaction in pairs with the Primer rp0B_2, for example as described below,<br>
the real-time PCR is performed.<br>
That is, 10 mM Tris-HCl buffer solution (pH 8.9) comprising each 50 to 2000 nM of the Primer rpoB_2 and an<br>
oligonucleotide primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in<br>
pairs with the Primer rpoB_2, about 5000 - 1000000 times dilution (final concentration) of the concentrate<br>
solution of SYBR™ Green I (trade name of Molecular Probe Inc.), 1-0 to 4.0 mM MgCl2, 80 mM KCI, 500<br>
u.g/ml of BSA, 0.1% sodium cholate, 0.005 to 0.2% TritonX-100, 0.2 rnM each of dATP, dCTP, dGTP and dTTP,<br>
and ] 0 to 80 U/ml of Taq DNA polymerase is prepared, and used as a faction solution for PCR.    To a 20 u.1 of<br>
the reaction solution for PCR, 1 ng of purified DNA sample purified from a sample to be detected for mutation is<br>
added, and used as a DNA sample for PCR. This sample is placed in the of 96-well reaction plate, and the<br>
real-time PCR is performed using appropriate real-time PCR detection equipment and the like.    The reaction is<br>
repeated for 30 to 50 cycles, and the fluorescent intensity emitted by SyBRT M Green I intercalated into the<br>
amplification products is measured at each cycle.<br>
In the next place, the melting curve is made by plotting the melting tenthpeature of amplification product<br>
(double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical<br>
28<br><br>
axis, and detection of peak is examined.<br>
In this case, when the Tm value of the detected peak of the primer extension product is identical with that of the<br>
primer extension product predicted to be amplified by the real-time PCR using the same combination of primers,<br>
it can be determined that there is a mutation in rpoB gene which is the detection target in the purified DNA<br>
sample.<br>
In addition, it may be such a method that when the Tm value of the peak obtained by the melting curve analysis<br>
for each primer extension product of each dilution series is identical, it can be determined that there is a mutation<br>
in rpoB gene in the purified DNA sample.<br>
In addition, as a reference, either an oligonucreotide comprising rpoB gene of wild type human tubercle bacillus is<br>
synthesized by conventional method, or an oligonucreotide comprising rpoB gene is extracted and purified from<br>
wild type human tubercle bacillus.   And, the real-time PCR is performed by the same method as described above<br>
except for using this oligonucleotide instead of "a purified DNA sample purified from a sample to be detected for<br>
mutation as a template, and the fluorescence intensity of the SYBR™ Green I is measured by the same way, and<br>
then the melting curve analysis may be performed.    In this case, as there is no mutation in nucleotide sequence in<br>
the sample, fluorescence could not be detected, and therefore, a peak should not appear in the melting curve<br>
analysis.    To make the determination of mutation in nucleotide sequence more assured, it is preferable to conduct<br>
such a control experiment in parallel.<br>
In addition, a method for detecting a single-base mutation in rpoB gene of human tubercle bacillus by the<br>
above-described  using the Primer c or the Primer d of the present invention includes, for example,<br>
the following methods.<br>
That is, using the Primer c or the Primer d of the present invention which are designed based on the<br>
above-described hot spot mutation region in rpoB gene of the wild type human tubercle bacillus, an<br>
oligonucleotide primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in<br>
pairs with the above-described primer, and the intercallator, and using a purified DNA sample purified from a<br>
sample to be detected for a mutation as a template, real-time PCR method by known intecalator method and the<br>
following melting curve analysis are performed through the use of known intercalator method by the same<br>
method as the above-described <method>.<br>
In this case, when the Tm value of a peak for the primer extension product detected in the melting curve analysis<br>
is identical with Tm value of the primer extension product predicted to be amplified by the real-time PCR using<br>
29<br><br>
the same combination of primers, it can be determined that there is nci target mutation in the rpoB gene in the<br>
purified DNA sample.    On the other hand, when the peaks having the same Tm value cannot be obtained, it can<br>
be determined that there is a target mutation(s) in the target rpoB gene in  the purified DNA sample.<br>
In addition, it may be such a method that, as a consequence of the meting curve analysis, when the Tm value of<br>
the peak obtained for each primer extension product of each dilution seriesof the purified DNA sample solution is<br>
identical, it can be determined that there is no mutation in rpoB gene in the purified DNA sample.    On the other<br>
hand, when the Tm value of the peaks is different, it can be determined that  here is mutation(s) in rpoB gene in<br>
the purified DNA sample.<br>
It should be noted that the real-time PCR may be performed using any one kind of Primer a, b, c or d of the<br>
present invention, or otherwise, the real-time PCR may be performed by the same method using several kinds of<br>
primers of the present invention in the same time.    When there exist Some  sort 0f mutation in the sample DNA, if<br>
only one kind of primer is used, it can be determined only whether the corresponding type of mutation exist or not<br>
in the sample DNA.    On the other hand, when the several kinds of primers 0f the present invention are used in<br>
the same time, it can be determined whether there exists any of mutatition detectable by the several kinds of the<br>
applied primers at once, and therefore, this method is extremely effective m the field of clinical laboratory test<br>
where swiftness of the determination is required.<br>
The example of the case where the several kinds of primers of the presnt invention are used includes for<br>
example, a case where several kinds of the Primer a is used, a case where several kinds of the Primer b is used<br>
and a case where (several kinds of) the Primer a and (several kinds of) the Primer b are used appropriately in<br>
combination.<br>
For example, when a mutation in rpoB gene of tubercle bacillus is detected  if the real-time PCR is performed<br>
using only the Primer rpoB_2, it can be determined only whether the notation detectable by the Primer rpoB_2<br>
exist or not, namely only whether the mutation of base "C" at the 46th t0 "T" exists or not in the sequence of a hot<br>
spot mutation region in the wild type rpoB gene, and if there exist another mutation, the existence of such<br>
mutation can not be determined.    On the other hand, for example, if the real-time PCR is performed using the<br>
Primer rpoB_2, the Primer rpoB_2_n3, the Primer rpoB_6 and the Prirner rp0B_3 together in the same time, and<br>
if there exist any of a mutation of base "C" at the 46th to "T", a mutatic,n 0f base "T" at the 68th to "C", a mutation<br>
of base "C" at the 46lh to "G", a mutation of base "A" at the 47lh to "T" a peak can be obtained by the melting<br>
curve analysis, and thereby it can be determined that there are some sort of mutation among the above-described<br>
30<br><br>
four kinds of mutations in rpoB gene of the sample.    In addition, by analyzing the position where peaks appear, it<br>
can be determined specifically which mutation among four types of mutations exists.<br>
(2) TaqMan™ real-time PCR method<br>
In the method for detecting mutation(s) in nucleotide sequence of the present invention, TaqMan™ real-time<br>
amplification detection method (see, for example, the documents in US Patent No. 5210015 and US Patent No.<br>
5538848) may also be used.<br>
The TaqMan™ real-time amplification detection method (TaqMan™ real-time PCR method) is, in more<br>
specifically, the method capable of detecting target of small amount of target DNA quantitatively with high<br>
sensitivity by the real-time PCR method using a probe in which the 5'end thereof is labeled with a fluorescent dye<br>
(reporter) such as FAM, and the 3'end thereof is labeled, for example with a quencher dye such as TAMRA (see,<br>
for example, the document in US Patent No. 5538848).<br>
The principle of the TaqMan™ real-time PCR method is explained as follows:<br>
That is, an oligonucleotide probe, which the 5'end thereof is labeled with a fluorescent dye (reporter) and the<br>
3'end thereof is labeled with a quencher dye, and capable of hybridizing with a specific region in the target gene,<br>
is used.    In the above-described probe, the fluorescence of the reporter is suppressed by the quencher dye under<br>
normal condition.    In a state where this fluorescent probe is hybridized completely with the target gene, PCR is<br>
performed from the outside thereof using DNA polymerase.    According to the progression of the extension<br>
reaction by DNA polymerase, the fluorescent probe is hydrolyzed from the 5'end by its exonuclease activity, and<br>
the released reporter dye generates fluorescence.    In the TaqMan™ real-time PCR method, by monitoring the<br>
intensity of this fluorescence in real time, the initial amount of the template DNA can be quantified accurately.<br>
As a probe to be used for the probe labeled with a fluorescent dye (reporter) at the 5'end thereof and with a<br>
quencher dye at the 3'end thereof to be used for the TaqMan™ real-time PCR detection method of the present<br>
invention, a probe having a nucleotide sequence of primer extension product obtained (predicted to be amplified)<br>
when the real-time PCR is performed by the combinational use of the selected forward primer and a reverse<br>
primer, or a probe having a nucleotide sequence designed further from such sequence may be used.<br>
In addition, the reporter fluorescent substance for labeling the 5'end includes carboxyfluorescein (FAM),<br>
hexachlorofluorescein (HEX), tetrachlorofluorescein (TET), Cy5 (Amersham Biosciences K.K.) and the like,<br>
however, FAM is preferable among them.    The quencher dye for labeling its 3'end includes fluorescent<br>
substance such as carboxytetramethyl-rhodamine (TAMRA), nonfluorescent substance such as Black Hole<br>
31<br><br>
Quencher dye (for example, BHQ2), 4-((4-(dimethylamino) phenyl)azo)benzoic acid (DABCYL), and TAMRA is<br>
preferable among them.<br>
Other reagents to be used for the TaqMan™ real-time PCR method such as deoxyribonucleoside 3-phosphate<br>
(dATP, dCTP, dGTP, dTTP) and DNA polymerase to be used the same reagents as usually used in the<br>
conventional real-time PCR.    The procedure of the TaqMan™ real-time PCR may be performed according to the<br>
customary protocol of the TaqMan™ real-time PCR except for the use of the primer and the probe of the present<br>
invention.<br>
The example of "the method for detecting mutation(s) in nucleotide sequence by the TaqMan™ real-time PCR<br>
method" of the present invention would be explained as follows:<br><method><br>
The Primer a or the Primer b of the present invention and an oligonucleotide primer capable of amplifying a<br>
sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer are used.<br>
And besides, based on the nucleotide sequence of the primer extension product predicted to be amplified by PCR<br>
using the combination of the above-described primers, an oligonucleotide with appropriate length is designed and<br>
synthesized by a conventional method.    Using the oligonucleotide which is labeled with a reporter dye on its<br>
5'end and labeled with a quencher dye on its 3'end as a labeled probe, and using a purified DNA sample purified<br>
from a sample to be detected for mutation as a template, the TaqMan™ real-time PCR is performed.    When a<br>
fluorescence derived from the reporter dye is measured, it can be determined that there is a target mutation in<br>
nucleotide sequence in the purified DNA sample.<br>
When the <method> is practiced, as the similar method in the intercalator method, it is preferable to conduct<br>
a control experiment using purified DNA derived from the wild type as a template in parallel.<br><method><br>
The Primer c or the Primer d of the present invention and an oligonucleotide primer capable of amplifying a<br>
sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer are used.<br>
And besides, based on the nucleotide sequence of the primer extension product predicted to be amplified by PCR<br>
using the combination of the above-described primers, an oligonucleotide with appropriate length is designed and<br>
synthesized by a conventional method.    Using the oligonucleotide which is labeled with a reporter dye on its<br>
5'end with a quencher dye on its 3;end as a labeled probe, and using a purified DNA sample purified from a<br>
sample to be detected for a mutation as a template, the TaqMan™ real-time PCR is performed.    When a<br>
32<br><br>
fluorescence derived from the reporter dye is measured, it can be determined that there is no target mutation in<br>
nucleotide sequence in the purified DNA sample.    On the other hand, when fluorescence derived from the<br>
reporter dye is not measured, it can be determined that there is a target mutation in nucleotide sequence in the<br>
purified DNA sample.<br>
The plural number of mutation may be determined at the same time by conducting the TaqMan™ real-time PCR<br>
using several kinds of primers of the present invention.    And, this is the same situation as the case where the<br>
determination is performed by the above-described intercalation method.<br>
As an example, a method for detecting mutation in nucleotide sequence by the TaqMan™ real-time PCR detection<br>
method of the present invention will be explained by taking a case where a single-base mutation in the rpoB gene<br>
of human tubercle bacillus is detected using the above-described "Primer rpoB_2" (as described above, a case<br>
where the mutation of base "C" to "T" at the 46th from the 5'end in the sequence of a hot spot mutation region in<br>
the wild type rpoB gene is detected) (<method>) for instance as follows.<br>
At first, the purified DNA sample is obtained from a sample to be detected for a mutation in nucleotide sequence<br>
by the known method.<br>
In the next place, based on the nucleotide sequence of the primer extension product predicted to be amplified by<br>
the PCR using the combination of the Primer rpoB_2 and an oligonucleotide primer capable of amplifying a<br>
sequence of interest by the nucleic acid amplification reaction in pairs with the Primer rpoB_2, an oligonucleotide<br>
with appropriate length is designed and synthesized by a conventional method.    The 5'end of this<br>
oligonucleotide is bound with a reporter dye ofFAM and the 3'end thereof is bound with a quencher dye (reporter<br>
quenching agent) of TAMRA by the conventional method, and thus a labeled probe is obtained.<br>
In the next place, using the Primer rpoB_2, an oligonucleotide primer capable of amplifying a sequence of interest<br>
by the nucleic acid amplification reaction in pairs with the Primer rpoB_2, and a labeled probe obtained above,<br>
the TaqMan™ real-time PCR is performed, for example, as described below:<br>
That is, 10 mM Tris-HCI buffer solution (pH 8.9) comprising each 1 uM of the Primer rpoB_2 and an<br>
oligonucleotide primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in<br>
pairs with the Primer rpoB_2, 100 to ] 000 JIM the labeled probe, 1.0 to 4.0 mM MgCI2, 80 mM K.C1, 500 µg/ml<br>
of BSA, 0.1% sodium cholate, 0.005 to 0.2% TritonX-100, 0.2 mM each of dATP, dCTP, dGTP and dTTP, and 10<br>
to 80 µ/ml of Taq DNA polymerase is prepared, and used as a reaction solution for PCR.    To a 20µl of<br>
above-described reaction solution for PCR, 1 ng of purified DNA sample purified from a sample to be detected<br>
33<br><br>
for mutation is added, and used as a DNA sample for PCR.    This sample is placed in the well of 96-well reaction<br>
plate, and the real-time PCR is performed using appropriate real-time PCR detection equipment and the like.<br>
The reaction is repeated for 30 to 50 cycles, and the fluorescence intensity derived from the reporter dye is<br>
measured at each cycle.<br>
In this case, .when the fluorescence derived from the reporter dye is measured, it can be determined that there is a<br>
mutation in the target rpoB gene in the purified DNA sample.<br>
In addition, as described above, it is preferable to conduct a control experiment using purified DNA comprising<br>
rpoB gene of wild type human tubercle bacillus as a template in parallel.<br>
A method for detecting single base mutation in rpoB gene of human tubercle bacillus by the above-described<br><method> using the Primer c or the Primer d of the present invention includes, for example, the following<br>
methods:<br>
That is, using the Primer c or the Primer d of the present invention which are designed based on the<br>
above-described hot spot mutation region in rpoB gene of the wild type human tubercle bacillus, an<br>
oligonucleotide primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in<br>
pairs with the above-described primer, and a labeled probe prepared by the same method as described in <method></method>
(2)-l&gt;, the TaqMan™ real-time PCR is performed by the same method as above-described <method>.<br>
In this case, when the fluorescence derived from the reporter dye is measured, it can be determined that there is no<br>
mutation in the target rpoB gene in the purified DNA sample.    On the other hand, when the fluorescence is not<br>
measured, it can be determined that there is a mutation in the target rpoB gene in the purified DNA sample.<br>
In addition, present invention can be applied in the nucleic acid amplification process with a detection method<br>
using RNA replication product.    For example, NASBA (nucleic acid sequence based amplification) method<br>
(Japanese Patent No. 2650159), 3SR (self-sustained sequence replication) method (JP-B-Hei 7-114718), TAS<br>
(transcription based amplification system) method (JP-A-Hei 2-500565; International Publication No.<br>
WO88/10315), and TMA (transcription mediated amplification) method (JP-A-1999-46778) are included,<br>
however, among them, isothermal nucleic acid amplification method utilizing concerted reaction of reverse<br>
transcriptase and RNA polymerase (a reaction is perfumed under the condition where reverse transcriptase and<br>
RNA polymerase act conceitedly) is suitable for the automation of the determination system.<br>
(3) A method in which, after the nucleic acid amplification reaction is performed, the primer extension products<br>
obtained are subjected to electrophoresis<br>
34<br><br>
This method includes, for example,<br>
"the method for detecting mutation(s) in nucleotide sequence comprising, the following steps of:<br>
(i) performing a nucleic acid amplification reaction (PCR) by using a primer selected from the Primer a, the<br>
Primer b, the Primer c or the Primer d of the present invention, and an oligonucleotide primer capable of<br>
amplifying a sequence of interest by nucleic acid amplification reaction in pairs with the above-described primer,<br>
and using the nucleic acid in a sample as a template,<br>
(ii) electrophoresing the primer extension product obtained in the above (i), and detecting mutation(s) in<br>
nucleotide sequence is detected on the basis of the obtained result.".<br>
A method for detecting mutation(s) in nucleotide sequence based on the results of electrophoresis includes, for<br>
example,<br>
(3-1) the method for detection by confirming a fraction of primer extension product having objective size (number<br>
of base pair);<br>
(3-2) the method for detection by hybridization using labeled probe.<br>
Condition and operation procedure of the electrophoresis may be according to the conventional method usually<br>
performed in this field.<br>
(3-1) The method for detection by confirming a fraction of primer extension product having objective size<br>
(number of base pair)<br>
As to the method for detection by confirming a fraction of primer extension product having objective size<br>
(number of base pair), for example, first of all, PCR is performed, and then the primer extension product obtained<br>
is subjected to electrophoresis,    The size (number of base pair) of the primer extension product predicted to be<br>
amplified by PCR using the combination of the primers of the present invention and an oligonucleotide primer<br>
capable of amplifying a sequence of interest by the nucleic acid amplification reaction in a pair with the<br>
above-described primer is estimated in advance.    And, the confirmation whether the size of the fraction obtained<br>
is relevant to the size of the predicted amplification product may be performed by the conventional method.    For<br>
example, the method in which, in such a method that the obtained fraction is visualized by staining with ethidium<br>
bromide and the like, the amplification product is detected (confirmed) based on the characteristic size thereof is<br>
included.<br>
The example of "the method for detection by confirming the fraction of the primer extension product having<br>
objective size (number of base pair)" of the present invention would be explained as follows:<br>
35<br><br><method><br>
The Primer a or the Primer b of the present invention and an oligonucleotide primer capable of amplifying<br>
a sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer are<br>
selected.    Further, the size (number of base pair) of the primer extension product predicted to be amplified by the<br>
PCR using this combination of the primers is estimated in advance.    Using the combination of the<br>
above-described primers, and using a purified DNA sample purified from a sample to be detected for a mutation<br>
as a template, PCR is performed, and then the primer extension product obtained is subjected to electrophoresis.<br>
In the next place, the confirmation whether the size of the fraction obtained is relevant to the size of the predicted<br>
amplification product may be performed by the conventional method.    For example, in such a method that the<br>
obtained fraction is visualized by staining with ethidium bromide and the like, the amplification product is<br>
detected by the characteristic size thereof, and when the size of the fraction obtained is confirmed to correspond to<br>
the size of the predicted amplification product, it can be determined that there is target mutation in nucleotide<br>
sequence in the purified DNA sample.<br><method><br>
The Primer c or the Primer d of the present invention, and an oligonucleotide primer capable of amplifying a<br>
sequence of interest by the nucleic acid amplification reaction in pairs with the above-described primer is selected.<br>
Further, the size (number of base pair) of the primer extension product predicted to be amplified by the PCR using<br>
this combination of the primers is estimated in advance.    Using the combination of the above-described primers,<br>
and using a purified DNA sample purified from a sample to be detected for mutation as a template, PCR is<br>
performed, and then the primer extension product obtained is subjected to electrophoresis.    In the next place, the<br>
confirmation whether the size of the fraction obtained is relevant to the size of the predicted amplification product<br>
may be performed by the conventional method.    For example, in such a method that the obtained fraction is<br>
visualized by staining with ethidium bromide and the like, the amplification product is detected by the<br>
characteristic size thereof, and when the size of the fraction obtained is confirmed to correspond to the size of the<br>
predicted amplification product, it can be determined that there is no target mutation in nucleotide sequence in the<br>
purified DNA sample.    On the other hand, when such fraction is not confirmed, it can be determined that there is<br>
target mutation in nucleotide sequence in the purified DNA sample.<br>
The plural number of mutation may be determined at the same time by conducting PCR and electrophoresis as<br>
described above using several kinds of primers of the present invention.    And, this is the same situation as the<br>
36<br><br>
case where the determination is performed by the above-described intercalation method.<br>
As an example, a method for detecting mutation in nucleotide sequence by the method of (3-1) for the present<br>
invention will be explained by taking a case where a single-base mutation in the rpoB gene of human tubercle<br>
bacillus is detected using the above-described "Primer rpoB_2" (as described above, a case where the mutation of<br>
base "C"to "T" at the 46th from the 5 'end in the sequence of a hot spot mutation region in the wild type rpoB<br>
gene is detected) (<method>) for instance as follows.<br>
At first, the purified DNA sample is obtained from a sample to be detected for a mutation in nucleotide sequence<br>
by the known method.<br>
In the next place, using the Primer rpoB_2 and an oligonucleotide primer capable of amplifying a sequence of<br>
interest by the nucleic acid amplification reaction in pairs with the Primer rpoB 2, and using a purified DNA<br>
sample purified from a sample to be detected for a mutation as a template, PCR is performed.    Then, the<br>
obtained primer extension product is subjected to electrophoresis.    The size (number of base pair) of the<br>
amplification product predicted to be amplified by PCR using this combination of primers is estimated in advance.<br>
The confirmation whether the size of the fraction obtained is relevant to that of the predicted amplification<br>
product may be performed by conventional method.   When the size of the fraction obtained is confirmed to<br>
correspond to the size of the predicted amplification product, it can be determined that there is a mutation in the<br>
target rpoB gene in the purified DNA sample.<br>
In addition, a method for detecting single-base mutation in rpoB gene of human tubercle bacillus by the<br>
above-described <method> using the Primer c or the Primer d of the present invention includes, for<br>
example, the following methods.<br>
That is, using the Primer c or the Primer d which are designed based on the above-described hot spot mutation<br>
region in rpoB gene of the wild type human tubercle bacillus and an oligonucleotide primer capable of amplifying<br>
a sequence of interest by nucleic acid amplification reaction in pairs with the above-described primer, and using a<br>
purified DNA sample purified from a sample to be detected for mutation as a template, PCR is performed by the<br>
same method as above-described <method>.    Then, the primer extension product obtained is subjected to<br>
electrophoresis.    The size (number of base pair) of the amplification product predicted to be amplified by the<br>
PCR using this combination of the primers is estimated in advance.    The confirmation whether the size of the<br>
fraction obtained is relevant to the size of the predicted amplification product may be performed by the<br>
conventional method.    When the size of the fraction obtained is confirmed to correspond to the size of the<br>
37<br><br>
predicted amplification product, it can be determined that there is no mutation in the target rpoB gene in the purified DNA sample.    On the other hand, when such fraction is not confirmed, it can be determined that there is a mutation in the target rpoB gene in the purified DNA sample.<br>
(3-2) The method for determination by hybridization using labeled probe. As to the method for determination by hybridization using labeled probe, for example, the following methods are included.    At first, PCR amplification product is subjected to the electrophoresis.    In the next place, an oligonucleotide, which comprises a sequence to be detected and labeled with a labeling substance, is used as a labeled probe; the obtained fraction is subjected to hybridization with the above-described labeled probe.    Then, by detecting signal from the above-described labeled probe, the existence or nonexistence of the hybridized fraction with the above-described labeled probe is confirmed, and the existence of mutation in nucleotide sequence is detected.<br>
The "oligonucleotide which comprises a sequence to be detected" used in the above-described method includes an oligonucleotide having the same nucleotide sequence as a nucleotide sequence of an amplification product predicted to be amplified by PCR using the combination of the primer, or an oligonucleotide further designed from the sequence and having an appropriate length.<br>
The specific example of labeling substance with which the probe to be used for the above-described method, and the method for labeling the probe are the same labeling substance and the same labeling method as used for the above-described labeled primer.<br>
The example of "the method for determination by hybridization using labeled probe" of the present invention would be explained as follows: <method><br>
PCR is performed by using the Primer a or the Primer b of the present invention, and an oligonucleotide primer capable of amplifying a sequence of interest by nucleic acid amplification reaction in pairs with the above-described primer, and using a purified DNA sample purified from a sample to be detected for a mutation as a template, PCR is performed.    Then, the primer extension product obtained is subjected to electrophoresis. The labeled probe is prepared in advance by labeling an oligonucleotide comprising a sequence to be detected with a labeling substance. The obtained fraction is subjected to hybridization with the above-described labeled probe.    The signal from the labeled probe is examined, and when a fraction hybridized with the above-described labeled probe is confirmed, it can be determined that there is a target mutation in nucleotide sequence in the purified DNA sample.<br>
38<br><br><method><br>
PCR is performed by using the Primer c or the Primer d of the present invention, and an oligonucleotide<br>
primer capable of amplifying a sequence of interest by nucleic acid amplification reaction in pairs with the<br>
above-described primer, and using a purified DNA sample purified from a sample to be detected for a mutation as<br>
a template, PCR is performed.    Then, the primer extension product obtained is subjected to electrophoresis.<br>
The labeled probe is prepared in advance by labeling an oligonucleotide comprising a sequence to be detected<br>
with a labeling substance.    The obtained fraction is subjected to hybridization with the above-described labeled<br>
probe.    The signal from the labeled probe is examined, and when a fraction hybridized with the above-described<br>
labeled probe is confirmed, it can be determined that there is no target mutation in nucleotide sequence in the<br>
purified DNA sample.    On the other hand, when such fraction is not confirmed, it can be determined that there is<br>
a target mutation in nucleotide sequence in the purified DNA sample.<br>
The plural number of mutation may be determined at the same time by conducting PCR and electrophoresis as<br>
described above using several kinds of primers of the present invention.    And, this is the same situation as the<br>
case where the determination is performed by the above-described intercalation method.<br>
As an example, a method for detecting mutation(s) in nucleotide sequence by the method of (3-2) of the present<br>
invention will be explained by taking a case where a single-base mutation in the rpoB gene of human tubercle<br>
bacillus is detected by using the above-described "Primer rpoB_2" (as described above, a case where the mutation<br>
of base "C" to "T" at the 46th from the 5'end in the sequence of a hot spot mutation region in the wild type rpoB<br>
gene is detected) (<method>) for instance as follows.<br>
At first, the purified DNA sample is obtained from a sample to be detected for a mutation in nucleotide sequence<br>
by the known method.<br>
In the next place, using the Primer rpoB_2 and an oligonucleotide primer capable of amplifying a sequence of<br>
interest by the nucleic acid amplification reaction in pairs with the Primer rpoB_2, and using a purified DNA<br>
sample purified from a sample to be detected for mutation as a template, PCR is performed.    Then, the obtained<br>
primer extension product is subjected to electrophoresis.    Based on the nucleotide sequence of the amplification<br>
product predicted to be amplified by PCR using this combination of primers, a sequence having an appropriate<br>
length for use in a probe is designed, and an oligonucleotide with this sequence is synthesized by the conventional<br>
method in advance.    The labeled probe is also prepared in advance by labeling this oligonucleotide with a<br>
labeling substance.    The obtained fraction is subjected to hybridization with the above-described labeled probe.<br>
39<br><br>
When a fraction hybridized with the above-described labeled probe is confirmed by detecting a signal from the<br>
labeled probe, it can be determined that there is a target mutation in rpoB gene in the purified DNA sample.<br>
In addition, a method for detecting single-base mutation in rpoB gene of human tubercle bacillus by the<br>
above-described <method> using the Primer c or the Primer d of the present invention includes, for<br>
example, the following methods.<br>
That is, using the Primer c or the Primer d which are designed based on the above-described hot spot mutation<br>
region in rpoB gene of the wild type human tubercle bacillus, an oligonucleotide primer capable of amplifying a<br>
sequence of interest by nucleic acid amplification reaction in pairs with the above-described primer, and using a<br>
purified DNA sample purified from a sample to be detected for a mutation as a template, PCR and subsequently<br>
the electrophoresis are carried out by the same method as above-described <method>.    Based on the<br>
nucleotide sequence of the amplification product predicted to be amplified by PCR using this combination of<br>
primers, a sequence having an appropriate length for use in a probe is designed, and an oligonucleotide with this<br>
sequence is synthesized by the conventional method in advance.    The labeled probe is also prepared in advance<br>
by labeling this oligonucleotide with a labeling substance.    The obtained fraction is subjected to hybridization<br>
with the above-described labeled probe.    When a fraction hybridized with the above-described labeled probe is<br>
confirmed by detecting a signal from the labeled probe, it can be determined that there is no target mutation in<br>
rpoB gene in the purified DNA sample.    On the other hand, when such fraction is not confirmed, it can be<br>
determined that there is a target mutation in rpoB gene in the purified DNA sample.<br>
(4) A method for measuring the signal of the primer extension product obtained by the nucleic acid amplification<br>
reaction using a labeled primer<br>
This method includes a method comprising using a labeled primer prepared from the primer of the present<br>
invention by the above-described labeling method and an oligonucleotide primer (it is not labeled.) capable of<br>
amplifying a sequence of interest by nucleic acid amplification reaction in pairs with the above-described primer,<br>
and using the nucleic acid in a sample as a template, performing PCR, and then measuring the signal of the<br>
obtained primer extension product.<br>
The example would be explained as follows:<br><method><br>
PCR is performed by using the labeled Primer a or the labeled Primer b, an oligonucleotide primer (it is not<br>
labeled.) capable of nucleic acid amplification reaction in pairs with the above-described primer, and using a<br>
40<br><br>
purified DNA sample purified from a sample to be detected for a mutation as a template.    After that the free-labeled primer is removed, and then the signal from the primer extension product is measured.    When the signal is detected, it can be determined that there is a target mutation in nucleotide sequence in the purified DNA sample. <method><br>
PCR is performed by using the labeled Primer c or the labeled Primer d, an oligonucleotide primer (it is not labeled.) capable of nucleic acid amplification reaction in pairs with the above-described primer, and using a purified DNA sample purified from a sample to be detected for a mutation as a template.    After that, the free-labeled primer is removed, and then the signal from the primer extension product is measured.    When the signal is detected, it can be determined that there is no target mutation in nucleotide sequence in the purified DNA sample.    On the other hand, when the signal is not detected, it can be determined that there is a target mutation in nucleotide sequence in the purified DNA sample.<br>
In the above-described method, the method of removing the free labeled primer includes a method in which after the primer extension product in the reaction mixture obtained by PCR is precipitated by the conventional nucleic acid precipitation method (ethanol precipitation method, a method of the precipitation using isopropanol, and the like), the supernatant solution containing the not-precipitated free labeled primer is removed, and the like. In addition, the method of separating the primer extension product from the free labeled primer by treating the reaction products obtained by PCR with gel chromatography under appropriate condition, or a electrophoretic separation method, and the like are also included.<br>
It should be noted that, as the reagents to be used in the method for detecting mutation(s) in nucleotide sequence of the present invention, the reagents which are usually used in this field such as, for example, buffering agents, stabilizers and preservatives, and which do not inhibit the stability of coexisting reagents and the like, and does not inhibit PCR and the hybridization reaction, may be utilized.    Also, the concentration of the reagent may be selected appropriately from the range of concentration usually used in this field.<br>
Specific example of buffer solution includes, all the buffer solutions customarily used when the conventional PCR and the hybridization reaction are performed, for example, Tris buffer, phosphate buffer, veronal buffer, borate buffer and Good's buffer, and the pH thereof is not specifically limited, but normally the pH range of 5 to 9 is preferable.<br>
In addition, as need arises, a nucleic acid synthesizing enzyme (DNA polymerase, RNA polymerase, reverse<br>
41<br><br>
transcriptase and the like), a substrate appropriate for the enzyme (dNTP, rNTP and the like), and also, a double strand intercalator (ethidium bromide, SYBR™ Green and the like) or a signal detection substance such as FMA and TAMRA and the like may be used.<br>
A kit for detecting mutation(s) in nucleotide sequence of the present invention includes "a kit for detecting mutation(s) in nucleotide sequence, comprising any one of oligonucleotide described in the following (a) - (d) or a salt thereof as a primer:<br>
(a)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene except<br>
42<br><br>
for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof; (d) an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of<br>
inhibiting a reaction of nucleic acid synthesis,<br>
or a salt thereof.<br>
The preferable aspect and the specific example of the constitutive reagents composing these kits are as described<br>
above.<br>
In addition, the above-described kit further includes a kit comprising "1) nucleoside 3-phosphate, 2) nucleic acid<br>
synthesizing enzyme, 3) an oligonucleotide capable of amplifying a sequence of interest by the nucleic acid<br>
amplification reaction in pairs with above-described oligonucleotide, and 4) PCR buffer solution".<br>
The nucleic acid synthesizing enzyme of the above-described 2) includes, for example, UNA polymerase, RNA<br>
polymerase and reverse transcriptase.<br>
The specific example of buffer solution to be used for PCR buffer solution of the above-described 4) includes, all<br>
the buffer solutions customarily used when the conventional PCR and the hybridization reaction are performed,<br>
for example, Tris buffer, phosphate buffer, veronal buffer, borate buffer and Good's buffer, and the pH thereof is<br>
also not specifically limited, but normally the pH range of 5 to 9 is preferable.<br>
In addition, the kit for detecting mutation(s) in nucleotide sequence of the present invention may comprise, as<br>
need arises, a substrate appropriate for the enzyme (dNTP, rNTP and the like), and also for example, a double<br>
strand intercalator such as SYBR™ Green I (trade name of Molecular Probes Inc.), ethidium bromide, fluorenes a<br>
signal detection substance such as FAM and TAMRA.    In addition, the reagent which is usually used in this field<br>
43<br><br>
such as, for example, stabilizers and preservatives, and which does not inhibit the stability of coexisting reagents<br>
and the like, and does not inhibit PCR and the hybridization reaction, may be comprised.    Also, the<br>
concentration of the reagent may be selected appropriately from the range of concentration usually used in this<br>
field.<br>
The material to be used for the method for detecting nucleotide sequence of the present invention includes<br>
biological samples obtained from test subject such as plasma, serum, cerebrospinal fluid, component extraction<br>
solutions from various body tissues, tissue sections, fecal matter, and urine.    In addition, cells derived from<br>
above-described various tissues are also included.<br>
For example, the material to be used for the method for detecting mutation(s) of human tubercle bacillus includes<br>
various clinical specimens such as expectorate sputum, transbronchial specimen, pharyngeal mucus, gastric fluid,<br>
puncture fluid such as bronchial washing solution, pleural effusion, and the like.<br>
The sample to be used for the method for detecting mutation(s) in nucleotide sequence of the present invention includes the sample comprising nucleic acid.    It may be the nucleic acid isolated and purified from the above-described materials, or may be the nucleic acid amplified by the nucleic acid amplification and detection<br>
system and the like.<br>
The present invention will be described in more detail by way of example thereof.    It should be noted, however,<br>
that the present invention should not be limited thereto.<br>
It should be noted that, in the example and the comparative example, "A" shown as a nucleic acid base represents<br>
adenine, "G" represents guanine, "C" represent cytosine, and "T" represents thymine respectively.<br>
EXAMPLES<br>
Example 1. Detection of drug resistant tubercle bacillus: model experiment 1<br>
(1) Construction of plasmid DNA<br>
1) Construction of plasmid DNA comprising mutant type rpoB gene sequence_2<br>
Referring to "the sequence of a hot spot mutation region in rpoB gene" reported by VIVEK KAPUR et al.<br>
(Non-patent Document 2: J. Clin. Microbiol., vol.32, No.4, 1994. pl095-1098,  FIG.1), a "plasmid DNA having a<br>
sequence with a single-base mutation in rpoB gene of tubercle bacillus" was prepared by gene engineering.    It<br>
should be noted that this sequence of a hot spot mutation region in rpoB gene is identical between human tubercle<br>
bacillus {Mycobacterium tuberculosis) and bovine tubercle bacillus(Mycobacterium bovis).<br>
First, using genomic DNA extracted and purified from Mycobacterium bovis BCG (bovine tubercle bacillus,<br>
44<br><br>
Strain No. RJMD1314006, Japanese Society for Bacteriology) as a starting material, and according to the known method of introducing site-directed mutagenesis by PCR described in "Toru Komano ed. Seibutsukagaku Jikkenhou vol.47; A manual of PCR experiments", introduction of mutation into a base at the 46,h from the 5'end ofthe sequence (SEQ ID NO: 1) of a hot spot mutation region in rpoB gene in the genomic DNA of the tubercle bacillus to "T': was performed, while it should be "C in the wild type.<br>
The mutation-introduced DNA fragment was purified using a column produced by QIAGEN K.K., and then inserted into a cloning vector (produced by Invitrogen Corporation).    After that, using QIAprep™ Spin Miniprep Kit (produced by QIAGEN K.K.), the plasmid DNA comprising objective sequence was purified and<br>
recovered.<br>
The plasmid DNA obtained by the above-described method comprises a nucleotide sequence shown in SEQ ID<br>
NO:3 described below.    This nucleotide sequence shown in SEQ ID NO:3 is hereinafter described as "mutant<br>
type rpoB gene sequence_2".    In the sequence ofthe "mutant type rpoB gene sequence_2", the underlined base<br>
at the 64th from the 5'end thereof corresponds to the base at the 46th from the 5'end in a hot spot mutation region<br>
in rpo B gene.    And, a base at this position in the wild type rpo B gene is "C", but is substituted into "T" in the<br>
"mutant type rpoB gene sequence_2".<br>
SEQIDNO:3:<br>
5'-TTCTTCGGCACCAGCCAGCTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACClAC<br>
AAGCGCCGACTGTCGGCGCTGGGG-3' (mutant type rpoB gene sequenceJ2)<br>
2) Construction of plasmid DNA comprising wild type rpoB gene sequence<br>
The plasmid DNA comprising wild type rpoB gene sequence was obtained by the same method as described<br>
above except for not introducing base mutation.    That is, this plasmid DNA comprises a nucleotide sequence<br>
shown in the following SEQ ID NO:2.<br>
SEQIDNO:2:<br>
5'-TTCTTCGGCACCAGCCAGCTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACCCA<br>
CAAGCGCCGACTGTCGGCGCTGGGG-3' (2) Construction of a primer (Primer rpoB_2) for detection of mutant type rpoB gene<br>
Based on the mutant type rpoB gene sequence_2, the primer was designed by the following method.<br>
First, the primer sequence for the detection ofthe mutant type rpoB gene was designed so that (i) the position to<br>
be introduced with a mutation in the mutant type rpoB gene sequence_2, namely, the base "T" at the 64th from the<br>
45<br><br>
5'end thereof, will be set at the 3'end of the primer, (ii) the other sequence of the primer will be set to be identical<br>
to the nucleotide sequence with 16 bases toward the 5'-side from the base "T" at the 64th from the 5'end in the<br>
mutant type rpoB gene sequence_2, and moreover (iii) the base "C" at the second position from the 3'end of the<br>
primer will substituted with the "C" which is modified with 2'-0,4-C-Ethylene-bridged Nucleic Acids.    The<br>
oligonucleotide designed in this method is described as "Primer rpoB_2".    The nucleotide sequence of the<br>
"Primer rpoB_2" is shown in the following SEQ ID NO:7.<br>
That is, in the nucleotide sequence shown in the SEQ ID NO:7 described below, the underlined nucleotide at the<br>
3'end is the position where the "C" in the wild type is substituted into "T".    In addition, the base "C" at the<br>
second position from the 3 'end is the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.<br>
SEQ IDNO:7 : 5'-GCTGTCGGGGTTGACCT-3'<br>
The designed "Primer rpoB_2" was obtained through the use of custom service of Sigma Genosys.<br>
(3) Detection of the mutant type rpoB gene by the real-time PCR detection and determination of the effects from<br>
false positive reactions<br>
1)	Preparation of DNA sample for PCR<br>
By measuring absorbance of the plasmid DNA sample comprising the mutant type rpoB gene sequence_2 obtained in the above-described (1), the amount of DNA in the plasmid DNA sample was measured.    In the next place, the plasmid DNA sample was diluted using 10 mM Tris-HCl buffer (pH 8.9) to prepare a dilution series of 105, 104, 103, and 102 copies/uJ, and used them as DNA samples for PCR.<br>
2)	Preparation of the reaction solution for PCR<br>
A 10 mM Tris-HCl (pH 8.9) containing each 300 nM of the Primer rpoB_2 obtained in the above-described (2) and a universal primer (Ml3-forward primer, Takara Bio Inc.), a 30 times dilution (30000 times dilution in final) of the original concentration of SYBR™ Green I (trade name of Molecular Probes Inc.) as a intercalating dye, 1.5 mM MgCI2, 80 mM KC1, 500 µg/ml of BSA, 0.1% sodium cholate, 0.1% TritonX-100, 0.2 mM each of dATP, dCTP, dGTP and dTTP, and 40 µ/ml of Taq DNA polymerase (produced by Nippon Gene Co., Ltd.) was prepared, and used the solution as a reaction solution for PCR.<br>
It should be noted that, in this time, as a partner primer to be combined with the Primer rpoB_2, M13-forward primer was used; and the reason for this combination was because in this case the employed DNA sample for PCR was a plasmid DNA comprising the mutant type rpoB gene sequence_2 (hereinafter, the same as above).<br>
3)	Real-time PCR<br>
46<br><br>
Using a plasmid DNA comprising the mutant type rpoB gene sequence_2 prepared in the above-described (1) 1) as a template DNA (an amplification target), real-time PCR was performed.    In addition, by the method described below, the quantitative monitoring by intercalation method was performed, and the results of real-time PCR were studied and evaluated.<br>
That is, 1 µ1 of each dilution series of the DNA sample for PCR prepared in the above-described (3) 1) and 19µl of the reaction solution for PCR prepared in the above-described (3) 2) were placed in a well of a 96-well reaction plate (MicroAmp Optical 96-Well Reaction Plate, produced by Aplied Biosystems Japan), and the real-time PCR was performed using personal thermal cycler and detection system for TaqMan™ PCR (ABI 7500, produced by Applied Biosystems Japan).    Namely, after the reaction solution was heated at 95 °C for 10 minutes, a reaction cycle consisting a reaction at 95 °C for 15 seconds and a reaction at 60 °C for 1 minute was repeated for 40 cycles. After that, the fluorescent intensity emitted from SYBRT M Green I corresponding to the amplification products was measured.<br>
In addition, using a plasmid DNA comprising the wild type rpoB gene sequence prepared in the above-described (1) 2) as a template DNA, and by the same method as above-described (3) except for the use of above-described plasmid DNA, the preparation of DNA sample for PCR, the preparation of the reaction solution for PCR and the real-time PCR were performed.<br>
(4)	Melting curve analysis<br>
The melting curve was made for each amplification product for the dilution series of the DNA sample for PCR by plotting the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical axis, and detection of peak was examined.<br>
(5)	Results<br>
The real-time PCR was performed using each DNA sample as a template, and the melting curves which was made based on the obtained results were shown in Fig. 1 and Fig. 2.<br>
Fig. 1 shows a result of the real-time PCR performed by using the Primer rpoB_2 of the present invention and using a plasmid DNA comprising mutant type rpoB gene sequence_2 as a template.<br>
On the other hand. Fig. 2 shows a result of the real-time PCR performed by using the Primer rpoB 2 of the present invention and using a plasmid DNA comprising the wild type rpoB gene sequence as a template. As is clear from Fig. 1, when the real-time PCR is performed by using the Primer rpoB_2 of the present invention<br>
and using a plasmid DNA comprising mutant type rpoB gene sequence_2 as a template, the amplification of DNA<br>
47<br><br>
was confirmed.    In addition, the peak positions, namely, the Tm value of the melting curve for each DNA sample<br>
with 105, 104, 103 and 102 copies were overlapped.<br>
On the other hand, as is clear from Fig. 2, when the real-time PCR is performed in the same method using the<br>
Primer rpoB_2 of the present invention and using a plasmid DNA comprising wild type rpoB gene sequence as a<br>
template, the peak did not appear, and the primer extension product of the DNA was not obtained.<br>
From the results described above, it is found that it becomes possible to eliminate completely any false positive<br>
result in the detection, and to detect only the target having a mutant sequence, namely, the DNA sample having a<br>
mutant sequence specifically and with high accuracy, by conducting the real-time PCR using a primer of the<br>
present invention which has a single base substitution at the 3'end thereof, and has a nucleotide with a<br>
modification capable of inhibiting the reaction of the nucleic acid synthesis at the second position from the 3'end<br>
thereof.<br>
Comparative Example 1:<br>
(1)	Plasmid DNA comprising the mutant type rpoB gene sequence<br>
The "mutant type rpoB gene sequence_2" obtained in Example 1 (1) was used.<br>
(2)	Primer for the detection of mutant type rpoB gene (Construction of Primer rpoB_2_n3)<br>
A primer having a nucleotide sequence shown in the same SEQ ID NO:7 as that for the Primer rpoB_2 obtained in Example 1 (2), while the 3rd "C" from the 3'end thereof is modified with a nucleotide analog (2'-0,4'-C-Ethylene-bridged Nucleic Acids) and the second "C" from the 3'end thereof is not modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids, was designed.    This primer is described as "Primer rpoB_2_n3". The designed "Primer rpoB_2_n3" was obtained through the use of custom service of Sigma Genosys.<br>
(3)	Detection of the mutant type rpoB gene by the real-time PCR detection and determination of the effects from<br>
false positive reactions<br>
1)	Preparation of DNA sample for PCR<br>
The same one as prepared in Example 1 (3) 1) (the dilution series of plasmid DNA comprising the mutant type rpoB gene sequence_2) was used.<br>
2)	Preparation of the reaction solution for PCR<br>
A 10 mM Tris-HCI (pH 8.9) containing each 300 nM of the Primer rpoB_2_n3 obtained in the above-described<br>
(2), a universal primer (Ml3-forward primer, Takara Bio Inc.), a 30 times dilution (30000 times dilution in final)<br>
of the original concentration of SYBR™ Green I (trade name of Molecular Probes Inc.) as intercalating dye, 1.5<br>
48<br><br>
mM MgCl2, 80 mM KC1, 500 µg/ml of BSA, 0.1% sodium cholate, 0.1% TritonX-100, 0.2 mM each of dATP,<br>
dCTP, dGTP and dTTP, and 40 U/ml of Taq DNA polymerase (produced by Nippon Gene Co., Ltd.) was prepared,<br>
and used the solution as a reaction solution for PCR.<br>
3) Real-time PCR<br>
The real-time PCR was performed by the same method as described in  Example 1 (3) 3) except for using the<br>
reaction solution for PCR prepared in the above-described (3) 2) as a reaction solution, and the fluorescent<br>
intensity emitted from SYBRT M Green I intercalating into the ampliflication products was measured.<br>
In addition, using a plasmid DNA comprising the wild type rpoB gene sequence prepared in Example 1 (1) 2) as a<br>
DNA sample (template DNA), and by the same method except for the use of above-described plasmid DNA, the<br>
preparation of DNA sample, the preparation of the reaction solution and the real-time PCR were performed.<br>
(4) Melting curve analysis<br>
The melting curve was made for each amplification product for the dilution series of the DNA sample by plotting<br>
the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first<br>
derivation (variation) of fluorescent intensity as vertical axis, and detection of peak was examined.<br>
(5) Results<br>
The real-time PCR was performed using each DNA sample as a template, and the melting curves which was made<br>
based on the obtained results were shown in Fig. 3 and Fig. 4.<br>
Fig. 3 shows a result of the real-time PCR performed by using the Primer rp0B_2_n3 and using a plasmid DNA<br>
comprising mutant type rpoB gene sequence_2 as a template.<br>
On the other hand, Fig. 4 shows a result of the real-time PCR performed by using the Primer rpoB 2 n3 and<br>
using a plasmid DNA comprising the wild type rpoB gene sequence_2 as a template.<br>
As is clear from Fig. 3, when the real-time PCR is performed by using the Primer rpoB_2_n3 and using a plasmid<br>
DNA comprising mutant type rpoB gene sequence_2 as a template, th&amp; amplification of DNA was confirmed.<br>
However, as is clear from Fig.4, when the real-time PCR is performed jn the same method using the Primer<br>
rpoB_2_n3 and using a plasmid DNA comprising wild type rpoB gene sequence as a template, the amplification<br>
of DNA was also confirmed.    That is, the peak of false positive ampliflicatij0n was confirmed.<br>
From the results described above, it is found that when PCR is performed using an oligonucleotide wherein a<br>
nucleotide having a modification capable of inhibiting the reaction of rlucleic acid synthesis is allocated at the 3rd<br>
position from 3'end thereof as a template, the accurate determination of mutant type can not be made.<br>
49<br><br>
Example 2. Detection of drug resistant tubercle bacillus: model experiment 2<br>
(1)	Construction of plasmid DNA comprising mutant type rpoB gene sequence_6<br>
By the same method as Example 1 (1) I), introduction of mutation into a base at the 68th from the 5'end of the<br>
sequence of a hot spot mutation region in rpoB gene to "C" was performed, while it should be "TM in the wild<br>
type.<br>
The mutation-introduced DNA fragment was purified using a column produced by QIAGEN K.K., and then<br>
inserted into a cloning vector (produced by Invitrogen Corporation).    After that, using QIAprep™ Spin<br>
Miniprep Kit (produced by QIAGEN K.K.), the plasmid DNA comprising objective sequence was purified and<br>
recovered.<br>
The plasmid DNA obtained by the above-described method comprises a nucleotide sequence shown in SEQ ID<br>
NO:4 described below.    This nucleotide sequence shown in SEQ ID NO:4 is hereinafter described as "mutant<br>
type rpoB gene sequence 6".    In the sequence of "the mutant type rpoB gene sequence_6", the underlined base<br>
at the 86th from the 5'end thereof corresponds to the base at the 68th from the 5'end in a hot spot mutation region<br>
in rpo B gene.    And, a base at this position in the wild type rpo B gene is "T", but is substituted into "C in the<br>
"mutant type rpoB gene sequence_6".<br>
SEQ ID NO:4 :<br>
5'-TTCTTCGGCACCAGCCAGCTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACCTAC<br>
AAGCGCCGACTGTCGGCGCCGGGG-3' (mutant type rpoB gene sequence_6)<br>
(2)	Construction of a primer (Primer rpoB_6) for detecting mutant type rpoB gene<br>
Based on the mutant type rpoB gene sequence_6, primer was designed by the following method.<br>
First, the primer sequence for the detection of the mutant type rpoB gene was designed so that (i) the position to<br>
be introduced with a mutation in the mutant type rpoB gene sequence 6, namely, the base "C" at the 86th from the<br>
5'end thereof, will be set at the 3'end of the primer, (ii) the other sequence of the primer will be set to be identical<br>
to the nucleotide sequence with 14 bases toward 5'-side from the base "C" at the 86th from the 5'end of the mutant<br>
type rpoB gene sequence_6, and moreover (iii) the base "C" at the second position from the 3'end of the primer<br>
will be substituted with the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.    The<br>
oligonucleotide designed in this method is described as "Primer rpoB_6".    The nucleotide sequence of "Primer<br>
rpoB_6" is shown in the following SEQ ID NO:8.<br>
That is, in the nucleotide sequence shown in SEQ ID NO:8 described below, the underlined nucleotide at the<br>
50<br><br>
3'end is the position where the "T" in the wild type is substituted into "C".    In addition, the base "C" at the<br>
second position from the 3'end is the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.<br>
SEQ ID NO:8 : 5'-CCGACTGTCGGCGCC-3'<br>
The designed "Primer rpoB_6" was obtained through the use of custom service of Sigma Genosys.<br>
(3) Detection of the mutant type rpoB gene by the real-time PCR detection and determination of the effects from<br>
false positive reactions<br>
1) Preparation of DNA sample for PCR<br>
By measuring absorbance of the plasmid DNA sample comprising the mutant type rpoB gene sequence_6<br>
obtained in the above-described (1), the amount of DNA in the plasmid DNA sample was measured.    In the next<br>
place, the plasmid DNA sample was diluted using 10 mM Tris-HCl buffer (pH 8.9) to prepare a dilution series of<br>
105, 104, 103, and 102 copies/ul, and used as DNA samples for PCR.<br>
2)	Preparation of the reaction solution for PCR<br>
A 10 mM Tris-HCl (pH 8.9) containing each 300 nM of the Primer rpoB_6 obtained in the above-described (2) and a universal primer (M13-forward primer, Takara Bio Inc.), a 30 times dilution (30000 times dilution in final) of the original concentration of SYBR™ Green I (trade name of Molecular Probes Inc.) as intercalating dye, 1.5 mM MgCI2, 80 mM KCl, 500 ng/ml of BSA, 0.1% sodium cholate, 0.1%TritonX-100, 0.2 mM each of dATP, dCTP, dGTP and dTTP, and 40 U/ml of Taq DNA polymerase (produced by Nippon Gene Co., Ltd.) was prepared, and used as a reaction solution for PCR.<br>
3)	Real-time PCR<br>
Using a plasmid DNA comprising the mutant type rpoB gene sequence_6 prepared in the above-described (1) as a<br>
template DNA (an amplification target), real-time PCR was performed.    In addition, by the method described<br>
below, the quantitative monitoring by intercalation method was performed, and the result of real-time PCR were<br>
studied and evaluated.<br>
That is, 1 µl of each dilution series of the DNA sample for PCR prepared in the above-described (3) 1) and 19 ul<br>
of the reaction solution for PCR prepared in the above-described (3) 2) were placed in a well of a 96-well reaction<br>
plate (MicroAmp Optical 96-Well Reaction Plate, produced by Aplied Biosystems Japan), and the real-time PCR<br>
was performed by using personal thermal cycler and detection system for TaqMan™ PCR (ABI 7500, produced<br>
by Applied Biosystems Japan).    Namely, after the reaction solution was heated at 95 °C for 10 minutes, a<br>
reaction cycle consisting a reaction at 95 °C for 15 seconds and a reaction at 60 °C for 1 minute was repeated for<br>
51<br><br>
40 cycles.    After that, the fluorescent intensity emitted from SYBR™ Green I intercalated into the amplification products was measured.<br>
In addition, by the same method as described in Example 2 (3) except f°r tne use ofthe plasmid DNA comprising the wild type rpoB gene sequence prepared in Example 1 (1) 2) as a ternPIate DNA,the preparation of DNA sample for PCR, preparation of the reaction solution for PCR and the real"time PCR were performed.<br>
(4)	Melting curve analysis<br>
The melting curve was made for each amplification product for the dilution series of the DNA sample for PCR by plotting the melting temperature of amplification product (double-straded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical axis, and detect100 or peak was examined.<br>
(5)	Results<br>
The real-time PCR was performed by using each DNA sample as a template  and the melting curves which was<br>
made based on the obtained results were shown in Fig. 5 and Fig. 6.<br>
Fig. 5 shows a result of the real-time PCR performed by using the Primer rp06-6 of the present invention, and<br>
using a plasmid DNA comprising mutant type rpoB gene sequence-6 as a template.<br>
On the other hand, Fig. 6 shows a result of the real-time PCR performed bY using the Primer rpoB_6 of the<br>
present invention and a plasmid DNA comprising the wild type rpoBgene sequence as a template.<br>
As is clear from Fig. 5, when the real-time PCR is performed by using the Primer rpoB_6 of the present invention<br>
and using a plasmid DNA comprising mutant type rpoB gene sequence " as a template, the amphiication of DNA<br>
was confirmed.    In addition, the peak position of the melting curve for each DNA sample with 10,10,10 and<br>
10 copies were overlapped.    Namely, the Tm value was consistent.<br>
On the other hand, as is clear from Fig. 6, when the real-time PCR is performed using the Primer rpoB_6 of the<br>
present invention and using a plasmid DNA comprising wild type rpoB Sene sequence as a template, the peak did<br>
not appear, and the primer extension product of the DNA was not obtained-<br>
From the results described above, it is found that, any false positive resistin the detection can be eliminated<br>
completely and only the target having a mutant sequence can be detected specifically and with high accuracy, by<br>
conducting the real-time PCR by using a primer of the present invention wnich has a single base substitution at<br>
the 3'end thereof, and has a nucleotide with a modification capable of inhibiting the reaction of nucleic acid<br>
synthesis allocated at the second position from the 3'end thereof<br>
Example 3. Simultaneous detection of plural mutant gene<br>
52<br><br>
(1) Construction of plasmid DNA comprising mutant type rpoB gene sequence<br>
1) Construction of plasmid DNA comprising mutant type rpoB gene sequence_3<br>
By the same method as Example 1 (1) 1), introduction of mutation into a base at the 46th from the 5'end of the<br>
sequence of a hot spot mutation region in rpoB gene to "G" was performed, while it should be "C" in the wild<br>
type.<br>
The mutation-introduced DNA fragment was purified using a column produced by QIAGEN K.K., and then<br>
inserted into a cloning vector (produced by Invitrogen Corporation).    After that, using QIAprep™ Spin<br>
Miniprep Kit (produced by QIAGEN K.K.), the plasmid DNA comprising objective sequence was purified and<br>
recovered.<br>
The plasmid DNA obtained by the above-described method comprises a nucleotide sequence shown in SEQ ID<br>
NO:5 below.    This nucleotide sequence shown in SEQ IDN NO:5 is hereinafter described as "mutant type rpoB<br>
gene sequence_3".    In the sequence of the "mutant type rpoB gene sequence_3", the underlined base at the 64lh<br>
from the 5'end thereof corresponds to the base at the 46llV from the 5'end in a hot spot mutation region in rpo B<br>
gene.    And, a base at this position in the wild type rpo B gene is "C", but is substituted into "G" in the "mutant<br>
type rpoB gene sequence_3".<br>
SEQIDNO:5 :<br>
5'-TTCTTCGGCACCAGCCAGCTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACCGA<br>
CAAGCGCCGACTGTCGGCGCTGGGG-3' (mutant type rpoB gene sequence_3)<br>
2) Construction of plasmid DNA comprising mutant type rpoB gene sequence_4<br>
By the same method as Example 1 (I) 1), introduction of mutation into a base at the 47lh from the 5'end of the<br>
sequence of a hot spot mutation region in rpoB gene to "V* was performed, while it should be "A" in the wild<br>
type.<br>
The mutation-introduced DNA fragment was purified using a column produced by QIAGEN K.K., and then<br>
inserted into a cloning vector (produced by Invitrogen Corporation).    After that, using QIAprep7 M Spin<br>
Miniprep Kit (produced by QIAGEN K.K.), the plasmid DNA comprising objective sequence was purified and<br>
recovered.<br>
The plasmid DNA obtained by the above-described method comprises a nucleotide sequence shown in SEQ ID<br>
NO:6 described below.    This nucleotide sequence shown in SEQ ID NO:6 is hereinafter described as "mutant<br>
type rpoB gene sequence_4".    In the sequence of the "mutant type rpoB gene sequence_4", the underlined base<br>
53<br><br>
at the 65,h from the 5'end thereof corresponds to the base at the 47,h from the 5'end in a hot spot mutation region<br>
in rpo B gene.    And, a base at this position in the wild type rpo B gene is "A", but is substituted into "V in the<br>
"mutant type rpoB gene sequence_4'"<br>
SEQ ID NO:6 :<br>
5'-TTCTTCGGCACCAGCCAGCTGAGCCAATTCATGGACCAGAACAACCCGCTGTCGGGGTTGACCCTC<br>
AAGCGCCGACTGTCGGCGCTGGGG-3' (mutant type rpoB gene sequence--4)<br>
(2) Construction of a primer for detection of mutant type rpoB gene<br>
1) Construction of "Primer rpoB_3"<br>
Based on the mutant type rpoB gene sequence_3, the primer was designed by the following method.<br>
First, the primer sequence for the detection of the mutant type rpoB gene was designed so that (i) the position to<br>
be introduced with a mutation in the mutant type rpoB gene sequence_3, namely, the base "G" at the 64th from the<br>
5'end thereof, will be setthe3'end of the primer, (ii)the other sequence of the primer will be set to be identical to<br>
the nucleotide sequence with 16 bases toward 5'-side from the base "G" at the 64th from the 5'end in the mutant<br>
type rpoB gene sequence_3, and moreover (iii) the base "C" at the second position from the 3'end of the primer<br>
will be substituted with the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.    The<br>
oligonucleotide designed in this method is described as "Primer rpoB_3".    The nucleotide sequence of the<br>
"Primer rpoB_3" is shown in the following SEQ ID NO:9.<br>
That is, in the nucleotide sequence shown in the SEQ ID NO:9 described below, the underlined nucleotide at the<br>
3'end is the position where the "C" in the wild type is substituted into "G".    In addition, the base "C" at the<br>
second position from the 3'end is the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.<br>
SEQ ID NO:9 : 5'-GCTGTCGGGGTTGACCG-3'<br>
The designed "Primer rpoB_3" was obtained through the use of custom service of Sigma Genosys.<br>
[02401<br>
2) Construction of "Primer rpoB_4"<br>
Based on the mutant type rpoB gene sequence_4, the primer was designed by the following method.<br>
First, the primer sequence for the detecting the mutant type rpoB gene was designed so that (i) the position to be<br>
introduced with a mutation in the mutant type rpoB gene sequence_4, namely, the base "T" at the 47th from the<br>
5'end thereof, will be set at the 3'end of the primer, (ii) the other sequence of the primer will be set to be identical<br>
to the nucleotide sequence with 16 bases toward the 5'-side from the base "T" at the 47th from the 5'end in the<br>
54<br><br>
mutant type rpoB gene sequence_4, and moreover (iii) the base "C" at the second position from the 3'end of the<br>
primer will be substituted with the "C" which is modified with 2,-0,4'-C-Ethylene-bridged Nucleic Acids.    The<br>
oligonucleotide designed in this method is described as "Primer rpoB_4".    The nucleotide sequence of the<br>
"Primer rpoB_4" is shown in the following SEQ ID NO: 10.<br>
That is, in the nucleotide sequence shown in the SEQ ID NO: 10 described below, the underlined nucleotide at the<br>
3'end is the position where the "A" in the wild type is substituted into "T".    In addition, the base "C" at the<br>
second position from the 3'end is the "C" which is modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids.<br>
SEQ IDNO:10 : S'-CTGTCGGGGTTGACCCT^'<br>
The designed "Primer rpoB_4" was obtained through the use of custom service of Sigma Genosys.<br>
(3) Detection of the mutant type rpoB gene by the real-time PCR detection and determination of the effects from<br>
false positive reactions<br>
1) Preparation of DNA sample for PCR<br>
By measuring absorbance of the plasmid DNA sample comprising the mutant type rpoB gene sequence_3<br>
obtained in the above-described (1) 1) and the plasmid DNA sample comprising the mutant type rpoB gene<br>
sequence_4 obtained in the above-described (1) 2) , the amount of DNA in each plasmid DNA samples was<br>
measured.<br>
In the next place, the plasmid DNA samples comprising the mutant type rpoB gene sequence_2 obtained in<br>
Example 1 (1) 1), the mutant type rpoB gene sequence_3 obtained in the above-described (1) 1) and the mutant<br>
type rpoB gene sequence_4 obtained in the above-described (1) 2) were each diluted using 10 mM Tris-HCI<br>
buffer (pH 8.9) to prepare each dilution series of 105, 104, 103, and 102 copies/µl, and eash were used as DNA<br>
samples for PCR.<br>
2) Preparation of the reaction solution for PCR<br>
A 10 mM Tris-HCI (pH 8.9) containing each 300 nM of the Primer rpoB _2 obtained in (2) of Example 1, the<br>
Primer rpoB _3 and the Primer rpoB _4 obtained in the above-described (2), a universal primer (M13-forward<br>
primer, Takara Bio Inc.), a 30 times dilution (30000 times dilution in final) of the original concentration of<br>
SYBR™ Green I (trade name of Molecular Probes Inc.) as intercalating dye, 1.5 mM MgCl2, 80 mM KC1, 500<br>
µg/ml of BSA, 0.1% sodium cholate, 0.1% TritonX-100, 0.2 mM each of dATP, dCTP, dGTP and dTTP, and 40<br>
U/ml of Taq DNA polymerase (produced by Nippon Gene Co., Ltd.) was prepared, and used as a reaction solution<br>
for PCR.    That is, in this reaction solution, three kinds of primers, the Primer rpoB_2, the Primer rpoB_3 and the<br>
55<br><br>
Primer rpoB_4 are contained for the detection of mutant type rpoB gene. 3) Real-time PCR<br>
Using the plasmid DNA samples comprising the mutant type rpoB gene sequence_2 obtained in Example 1 (1) 1), the mutant type rpoB gene sequence_3 obtained in the above-described (1) 1), or the mutant type rpoB gene sequence_4 obtained in the above-described (1) 2) as a template DNA (an amplification target), real-time PCR was performed.    In addition, by the method described below, the quantitative monitoring by intercalation method was performed, and the results of real-time PCR were studied and evaluated.<br>
That is, 1 jil of each dilution series of the DNA sample for PCR prepared in the above-described (3) I) and 19 u.1 of the reaction solution for PCR prepared in the above-described (3) 2) were placed in a well of a 96-well reaction plate (MicroAmp Optical 96-Well Reaction Plate, produced by Aplied Biosystems Japan), and the real-time PCR was performed by using personal thermal cycler and detection system for TaqMan™ PCR (ABI 7500, produced by Applied Biosystems Japan).    Namely, after the reaction solution was heated at 95 °C for 10 minutes, a reaction cycle consisting of a reaction at 95 °C for 15 seconds and a reaction at 60 °C for 1 minute was repeated for 40 cycles.    After that the fluorescent intensity emitted from SYBRT M Green I intercalated into the amplification products was measured.<br>
(4)	Melting curve analysis<br>
The melting curve was made for each amplification product for the dilution series of each DNA sample for PCR by plotting the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first derivation (variation) of fluorescent intensity as vertical axis, and detection of peak was examined.<br>
(5)	Results<br>
Using each DNA sample as a template, and using a mixed primer comprising Primer rpoB_2, Primer rpoB_3 and<br>
Primer rpoB_4, the real-time PCR was performed, and the melting curves was made based on the obtained results<br>
were shown in Fig. 7 to Fig. 9.<br>
Fig. 7 shows a result of the real-time PCR using a plasmid DNA comprising mutant type rpoB gene sequence_2<br>
as a template.<br>
Fig. 8 shows a result of the real-time PCR using a plasmid DNA comprising mutant type rpoB gene sequence_3<br>
as a template.<br>
Fig. 9 shows a result of the real-time PCR using a plasmid DNA comprising mutant type rpoB gene sequence__4<br>
as a template.<br>
56<br><br>
Comparative Example 2<br>
By the same method as described in Example 3 (3) except for the use of a plasmid DNA comprising the wild type rpoB gene<br>
sequence obtained in Example 1 (1) 2) as a template DNA, preparation of DNA sample for PCR, preparation of reaction<br>
solution for PCR, and the real-time PCR was performed by using the Primer rpoB_2, the Primer rpoB_3, the Primer rpoB_4<br>
and M-13 forward primer, and then the fluorescent intensity emitted from S YBRT M Green I intercalated into the<br>
amplification products was measured.<br>
In the next place, the melting curve was made for each amplification product for the dilution series of each DNA sample for<br>
PCR by plotting the melting temperature of amplification product (double-stranded DNA) as horizontal axis and the first<br>
derivation (variation) of fluorescent intensity as vertical axis, and detection of peak was examined.<br>
The real-time PCR was performed using each DNA sample as a template, and the melting curves made based on the obtained<br>
results were shown in Fig. 10.<br>
As is clear from Fig. 10, when the real-time PCR is performed using several kinds of primers of the present invention and<br>
using a plasmid DNA comprising wild type sequence as a template, the peak of the primer extension product did not appear.<br>
On the other hand, as is clear from Fig. 7 to 9, when the real-time PCR is performed using several kinds of primers of the<br>
present invention and using a plasmid DNA comprising mutant type rpoB gene sequence_2, mutant type rpoB gene<br>
sequence_3 or mutant type rpoB gene sequence_4 as a template, the primer extension product was obtained in each case.<br>
That is, by using three kinds of primers of the present invention at the same time, three kinds of mutant gene can be detected.<br>
From the results described above, it is found that, even when the real-time PCR is performed using plural primers of the<br>
present invention, it is possible to eliminate completely any false positive result in the detection, and moreover, as single run<br>
of PCR can deal with various types of mutation pattern, it is possible to establish a determination system for drug resistance<br>
by multiplex PCR.<br>
INDUSTRIAL APPLICABILITY<br>
For example, for the microbial sensitivity test which is an examination on drug resistance property of bacteria and accepted<br>
as an essential test in diagnosis (treatment) of tuberculosis, through the use of in-Line separate detection (On chip PCR<br>
system) as a genetic test item, and in combination with the genetic diagnosis of tubercle bacillus and<br>
nontuberculous acid-fast bacterium disease, it becomes possible to perform the determination of drug-resistance<br>
in a single tube (1 line).<br>
Particularly, according to the present invention, it becomes possible to completely eliminate any false positive<br>
result in the determination and to deal with various mutation patterns in a single run of PCR reaction.    Therefore,<br>
it becomes possible to construct a drug-resistance determination system, which can detect any possible genetic<br>
mutation by a single run of multiplex PCR.<br>
57<br><br>
We claim:-<br>
1. A method for detecting mutation(s) in nucleotide sequence comprising:<br>
performing a nucleic acid amplification reaction by using any one of the following oligonucleotides (a) to (d) or a salt thereof as a primer, and using a nucleic acid in a sample as a template, and detecting reaction product; (a) an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the * 3'end position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucfei'c acid" synthesis, or a salt thereof;<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the<br>
3'end position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
58<br><br>
toward the 5'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(d) an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof.<br>
2.	The method according to claim }, wherein the method further comprises an oligonucleotide as a primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in pairs with the oligonucleotide to be used as the primer in claim 1.<br>
3.	The method according to claim 1 or claim 2, wherein the primer comprises 15 to 30 bases.<br>
4.	The method according to claim 2, wherein the combination of primers comprising "an oligonucleotide or a salt thereof having the same nucleotide as a mutant nucleotide in a target gene at the 3'end position thereof; having the same nucleotide sequence as a nucleotide sequence of the target gene except for its 3'end position, and having the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from the position where the mutant nucleotide may exist in the target gene; and further having a modification capable of inhibiting a reaction of nucleic acid synthesis at the nucleotide at the second position from 3 'end thereof, and "an oligonucleotide capable of amplifying a sequence of interest by the nucleic acid amplification reaction in pairs with the oligonucleotide" is such a combination that when the nucleic acid amplification reaction is performed with the nucleic acid in the sample as a template, the objective reaction product is obtained if the nucleic acid in the sample is mutant type, and the objective reaction products is not obtained if the nucleic acid in the sample is<br>
59<br><br>
wild type, and<br>
wherein the detection is performed by detecting the mutant type of nucleic acid specifically by using the combination of primers.<br>
5.	The method according to claim 1, wherein the mutation(s) in nucleotide sequence is a genetic polymorphism.<br>
6.	The method according to claim 5, wherein the genetic polymorphism is due to a single base substitution, a single base deletion or a single base insertion.<br>
7.	The method according to claim 1, wherein the nucleotide having a modification capable of inhibiting the reaction of nucleic acid synthesis is represented by the following formula [I] or [II],<br><br>
wherein B represents a nucleic acid base; R1 represents an oxygen atom, a -NH-group or a lower alkylene group; R2 represents a lower alkylene group;<br><br><br>
[II]<br><br>
wherein B represents a nucleic acid base; R3 represents an oxygen atom, a nitrogen atom, a -NH-group or a lower alkylene group.<br>
8. The method according to claim 1, wherein the nucleotide having a modification capable of inhibiting the reaction of nucleic acid synthesis is a nucleotide modified with 2'-0,4'-C-Ethylene-bridged Nucleic Acids (ENA).<br>
60<br><br>
9.	The method according to claim 1, wherein the nucleic acid amplification reaction is polymerase chain reaction<br>
(PCR).<br>
10.	A kit for detecting mutation(s) in nucleotide sequence, comprising any one of the following oligonucleotides<br>
(a) to (d) or a salt thereof as a primer:<br>
(a) an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has the same nucleotide sequence as a nucleotide sequence of the target gene except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene toward the 5'-side from the position where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(b)	an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a mutant nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position    where the mutant nucleotide may exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(c)	an oligonucleotide,<br>
i) wherein the oligonucleotide has the same nucleotide as a reference nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has the same sequence as a nucleotide sequence of the target gene<br>
except for its 3'end position, and has the same nucleotide sequence as the nucleotide sequence of the target gene<br>
toward the 5'-side from a position where the reference nucleotide exist in the target gene, and<br>
61<br><br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof;<br>
(d) an oligonucleotide,<br>
i) wherein the oligonucleotide has a nucleotide complementary to a reference nucleotide in a target gene at the 3'end position thereof,<br>
ii) wherein the oligonucleotide has a nucleotide sequence complementary to a nucleotide sequence of the target gene except for its 3'end position, and has a nucleotide sequence complementary to the nucleotide sequence of the target gene toward the 3'-side from a position where the reference nucleotide exist in the target gene, and<br>
iii) wherein the nucleotide at the second position from the 3'end thereof has a modification capable of inhibiting a reaction of nucleic acid synthesis, or a salt thereof.<br>
11. The kit according to claim 10, wherein the kit further comprises an oligonucleotide as a primer capable of amplifying a sequence of interest by the nucleic acid amplification reaction in pairs with the oligonucleotide. ] 2. The kit according to claim 11, wherein the kit further comprises,<br>
1)	nucleoside 3-phosphate,<br>
2)	nucleic acid synthesizing enzyme,<br>
3)	an oligonucleotide capable of amplifying a sequence of interest by the nucleic acid amplification reaction in pairs with the oligonucleotide, and<br>
4)	PCR buffer solution.<br><br>
13.	The method according to claim 5, wherein the genetic polymorphism is due to substitution, deletion or insertion.<br>
14.	A method substantially as herein described with reference to the foregoing description, examples, tables and the accompanying drawings.	<br>
Dated this 7th day of October 2008<br>
SHARAD VADEHRA<br>
       OF KAN  AND KRISHME<br>
ATTORNEY FOR THE APPLICANTS<br><br><br><br>
	62<br></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></method></pnnciple></principle></principle></principle></principle></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LS1DT1JSRVNQT05ERU5DRSgyMC0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008--CORRESPONDENCE(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUFCU1RSQUNUKDctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-ABSTRACT(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUFCU1RSQUNUKEdSQU5URUQpLSg2LTctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-ABSTRACT(GRANTED)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWFic3RyYWN0LmRvYw==" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUFOTkVYVVJFIDEgVE8gMyg3LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-ANNEXURE 1 TO 3(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUFOTkVYVVJFIDEoMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-ANNEXURE 1(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWFzc2lnbm1lbnQoMjQtMTAtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-assignment(24-10-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNBTkNFTExFRCBQQUdFUygyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CANCELLED PAGES(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNMQUlNUyhBTUVOREVEKS0oMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CLAIMS(AMENDED)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNMQUlNUyhBTUVOREVEKS0oNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CLAIMS(AMENDED)-(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNMQUlNUyhHUkFOVEVEKS0oNi03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CLAIMS(GRANTED)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUFkgT0YgRk9STUFMIERSQVdJTkcoNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-COPY OF FORMAL DRAWING(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDE3LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(17-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDItMTItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(2-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDIwLTEwLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDI4LTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(28-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDQtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(4-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKElQTyktKDYtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-CORRESPONDENCE(IPO)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWNvcnJlc3BvbmRlbmNlLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5kb2M=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-description(complete).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LURFU0NSSVBUSU9OKEdSQU5URUQpLSg2LTctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-DESCRIPTION(GRANTED)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWRyYXdpbmcucGRm" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUVOR0xJU0ggVFJBTlNMQVRJT04oNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-ENGLISH TRANSLATION(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUVQIERPQ1VNRU5UKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-EP DOCUMENT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gIDMoNy0xMi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM  3(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMSgyMC0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 1(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMTMoMjAtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 13(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMTgoMjgtMS0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 18(28-1-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMihHUkFOVEVEKS0oNi03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 2(GRANTED)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMihUSVRMRSBQQUdFKS0oR1JBTlRFRCktKDYtNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 2(TITLE PAGE)-(GRANTED)-(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gMih0aXRsZSBwYWdlKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 2(title page).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gMi5kb2M=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMygyLTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 3(2-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk0gMygyMC0xMC0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORM 3(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUZPUk1BTCBEUkFXSU5HKDctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-FORMAL DRAWING(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LUpBUEFORVNFIERPQ1VNRU5UKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-JAPANESE DOCUMENT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LU9USEVSIERPQ1VNRU5UKDIwLTEwLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-OTHER DOCUMENT(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1tdW1ucC0yMDA4LXBjdC1pYi0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2152-mumnp-2008-pct-ib-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVBPV0VSIE9GIEFUVE9STkVZKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-POWER OF ATTORNEY(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVBPV0VSIE9GIEFUVE9STkVZKDctMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-POWER OF ATTORNEY(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVBSSU9SSVRZIERPQ1VNRU5UIEpBUEFORVNFKDE3LTEyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-PRIORITY DOCUMENT JAPANESE(17-12-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVBST1NFQ1VUSU9OIEhJU1RPUlkgT0YgRVAgRE9DVU1FTlQoMjAtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-PROSECUTION HISTORY OF EP DOCUMENT(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVBST1NFQ1VUSU9OIEhJU1RPUlkgT0YgVVMgRE9DVU1FTlQoMjAtMTAtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-PROSECUTION HISTORY OF US DOCUMENT(20-10-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCg3LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-REPLY TO EXAMINATION REPORT(7-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVJFUExZIFRPIEhFQVJJTkcoMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-REPLY TO HEARING(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVNFUVVFTkNFIExJU1RJTkcoNC01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-SEQUENCE LISTING(4-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVNFUVVFTkNFIExJU1RJTkcoNi03LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-SEQUENCE LISTING(6-7-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVNQRUNJRklDQVRJT04oQU1FTkRFRCktKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-SPECIFICATION(AMENDED)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVNQRUNJRklDQVRJT04oTUFSS0VEIENPUFkpLSgyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-SPECIFICATION(MARKED COPY)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjE1Mi1NVU1OUC0yMDA4LVVTIERPQ1VNRU5UKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2152-MUMNP-2008-US DOCUMENT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="253256-novel-polymeric-blue-anthraquinone-derivative-colorants.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253258-a-method-of-assessing-a-risk-of-an-adverse-drug-reaction.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253257</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2152/MUMNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>28/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>13-Jul-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>06-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Oct-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>WAKO PURE CHEMICAL INDUSTRIES, LTD.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1-2, DOSHOMACHI 3-CHOME, CHUO-KU, OSAKA-SHI, OSAKA 540-8605,</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>TOMOKAZU ISHIKAWA</td>
											<td>6-1, TAKADA-CHO, AMAGASAKI-SHI, HYOGO 661-0963, JAPAN, A JAPANESE CITIZEN.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/JP2007/054785</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2007-03-12</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>2006-067495</td>
									<td>2006-03-13</td>
								    <td>Japan</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253257-method-for-detection-of-mutant-gene by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:56:13 GMT -->
</html>
