<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253468-detection-of-nucleic-acid-by-target-specific-hybrid-capture-method by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:44:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253468:DETECTION OF NUCLEIC ACID BY TARGET-SPECIFIC HYBRID CAPTURE METHOD</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">DETECTION OF NUCLEIC ACID BY TARGET-SPECIFIC HYBRID CAPTURE METHOD</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Target-specific hybrid capture (TSHC) provides a nucleic acid detection method that is not only rapid and sensitive, but is also highly specific and capable of discriminating highly homologous nucleic acid sequences. The method produces DNA:RNA hybrids which can be detected by a variety of methods.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp; THE PATENS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
DETECTION OF NUCLEIC ACIDS BY TARGET-SPECIFIC HYBRID CAPTURE METHOD:<br><br><br>
DIGENE	CORPORATION,	A<br>
CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF UNITED STATES OF AMERICA, WHOSE ADDRESS IS 1201 CLOPPER ROAD, GAITHERSBURG, MD 20878, U.S.A.<br><br>
THE FOLLOWING SPECIFICATION PARTICULARLY DESCRIBES THE INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED.<br><br><br>
This application is a continuation-in-part application under 35 U.S.C. §120 of U.S. patent application Serial No. 11/005,617, filed December 6, 2004, entitled, "Detection Of Nucleic Acids By Target-Specific Hybrid Capture Method," which is a continuation-in-part of U.S. patent application Serial No. 10/971,251, filed October 20, 2004, now abandoned, which is a continuation-in-part of U.S. patent application Serial No. 09/594,839, filed June 15, 2000, all of which are herein incorporated by reference.<br>
FIELD OF INVENTION<br>
This invention relates to the field of nucleic acid detection methods in general and more particularly relates to the detection of nucleic acids by target-specific hybrid capture method.<br>
BACKGROUND OF THE INVENTION<br>
The detection of specific nucleic acid sequences present in a biological sample is important for identifying and classifying microorganisms, diagnosing infectious diseases, detecting and characterizing genetic abnormalities, identifying genetic changes associated with cancer, studying genetic susceptibility to disease, and measuring response to various types of treatment. Common techniques for detecting and quantitating specific nucleic acid sequences are nucleic acid hybridization and target amplification.<br>
Various hybridization methods are available for the detection and study of nucleic acids. In a traditional hybridization method, the nucleic acids to be identified are either in a solution or affixed to a solid carrier. The nucleic acids are<br>
2.<br><br>
detected using labeled nucleic acid probes which are capable of hybridizing to the nucleic acids. Recently, new hybridization methods have been developed to increase the sensitivity and specificity of detection. One example is the Hybrid Capture® method described in U.S. Application Serial No. 07/792,585 and U.S. Patent No. 6,228,578. Although these new hybridization methods offer significant improvements over the traditional methods, they still lack the ability to fully discriminate between highly homologous nucleic acid sequences.<br>
The polymerase chain reaction (PCR) is the most commonly used target nucleic acid amplification method. However, PCR is limited to some extent when a large number of different targets are to be amplified simultaneously, i.e., multiplex reactions, which may cause not only PCR artifacts such as primer-dimers, but also spurious target amplification. In an attempt to overcome this limitation, consensus primers may be used when a number of targets have homologous regions. However, homology between species is never 100%, and as a result, the primers will have several mismatches with different targets, causing non-uniform amplification. When different amounts of targets are present in a sample, the amplification efficiency of different PCRs also varies, leading to non-uniform and non-specific amplification of different targets.<br>
It is therefore an object of the present invention to provide a method for detecting target nucleic acid sequences which not only provides increased rapidity and sensitivity, but which is also highly specific and capable of discriminating between multiple highly homologous nucleic acid target sequences<br>
SUMMARY OF THE INVENTION<br>
The present invention provides a novel nucleic acid detection method, referred to herein as target-specific HYBRID CAPTURE ("TSHC"). TSHC is a rapid, highly specific, and sensitive method capable of discriminating between and detecting highly homologous nucleic acid target sequences.<br>
One embodiment of the invention relates to a method of detecting and/or quantifying one or more target nucleic acids, comprising the steps of target<br>
3<br><br>
enrichment, amplification, and detection for the rapid and sensitive detection of the target nucleic acid sequences.<br>
In one embodiment of the claimed method, one or more target nucleic acids are detected by: capturing the target nucleic acids to a solid support by mixing the target nucleic acids, nucleic acid probes complementary to the target nucleic acids, wherein one is RNA and the other is DNA, and a solid support; removing unbound target nucleic acids and nucleic acid probes; amplifying the captured target nucleic acids or nucleic acid probes, forming a plurality of amplicons, where the presence of the amplicons is indicative of the presence of the target nucleic acids; and detecting the target nucleic acids by mixing the target nucleic acids with selectable and distinguishable oligonucleotides which hybridize to a portion of the target nucleic acids, (i.e., capture sequence probes; CSPs) and nucleic acid probes complementary to a different portion of the target nucleic acids (i.e., signal sequence probes; SSPs), wherein either the probe or target is an RNA and the other is DNA, where DNA:RNA hybrids are detected by DNA:RNA hybrid-specific binding agents, which are directly or indirectly labeled, thereby detecting the target nucleic acids. The SSPs are not limited to serving as only a means for producing a signal for detection; but may be used in the target enrichment step by hybridizing to the target nucleic acid, enabling capture with a DNA:RNA hybrid-specific binding agent.<br>
In yet another embodiment, a plurality of target nucleic acids are detected by: hybridizing a plurality of target nucleic acids to nucleic acid probes which are complementary to the target nucleic acids, forming DNA:RNA hybrids; capturing the DNA:RNA hybrids with DNA:RNA hybrid-specific antibodies conjugated to solid supports; removing unbound target nucleic acids and nucleic acid probes; amplifying the captured target nucleic acids or nucleic acid probes, forming a plurality of amplicons, using random primers and DNA polymerase, where the presence of the plurality of amplicons is indicative of the presence of the target nucleic acids; hybridizing nucleic acid probes complementary to a portion of the target nucleic acid sequences, forming DNA-RNA hybrids between targets and probes; hybridizing oligonucleotides conjugated to a solid support to a different portion of the target<br>
A<br><br>
nucleic acids, wherein the solid support is selectable; selecting the oligonucleotide complexes; and detecting the plurality of target nucleic acids by binding DNArRNA hybrid-specific binding agents to the DNA:RNA hybrids.<br>
In a further embodiment, one or more target DNAs are detected by a multiplex method having the steps of: hybridizing a plurality of target DNAs to RNA probes which are complementary to the target DNAs, forming DNArRNA hybrids; capturing the DNA:RNA hybrids with DNA:RNA hybrid-specific antibodies which are conjugated to beads; removing unbound nucleic acids and nucleic acid probes by washing excess nucleic acids and probes; isothermally amplifying the target DNAs using random primers and DNA polymerase, forming a plurality of amplicons; hybridizing RNA probes complementary to a portion of the target DNAs (i.e., SSPs), forming DNArRNA hybrids; hybridizing specific DNA oligonucleotides to a different portion of the target DNAs, wherein the DNA oligonucleotides are conjugated to selectable beads; and detecting the plurality of target DNAs by binding detectably labeled DNA:RNA hybrid-specific antibodies to the DNA:RNA hybrids and selecting target DNA using selectable oligonucleotide-conjugated beads (i.'e., CSPs), wherein the DNArRNA hybrid-specific antibodies are detectably and distinguishably labeled. The presence of each target is detected by the labeled DNArRNA antibody through SSPs which fonn DNA:RNA hybrids with the target, while the various targets are separated or selected based on the oligonucleotide-conjugated bead {i.e., CSP). The presence of amplicon and DNArRNA hybrids is indicative of the presence of the target DNAs.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention provides a method for enriching, amplifying, and detecting the presence of a nucleic acid or a plurality of nucleic acids in test samples. More specifically, the invention provides a highly specific and sensitive method which<br>
5<br><br>
is capable of discriminating between and detecting highly homologous nucleic acid sequences.<br>
One embodiment of the invention is directed to rapid, sensitive methods for multiplex detection of target nucleic acid sequences, where a plurality of different target nucleic acid sequences may be detected simultaneously. This method may be automated. The method essentially comprises three steps: target enrichment; target amplification; and target detection, which presents a semiquantitative or qualitative approach for nucleic acid diagnostics. This embodiment of the invention may be performed in multiplex format detecting target nucleic acid sequences in purified or unpurified samples, where a concentration of target may be as low as about 50 copies or fewer per milliliter of sample to about 100 copies of target, to as great as 108 copies per milliliter. Detection of individual pathogens or targets in biological samples, such as but not limited to HSV and HPV types, may be performed in multiplex, preferably but not limited to, 50-plexes.<br>
As is beneficial, this multiplex method of detecting a plurality of target nucleic acid sequences is a relatively rapid, sensitive, and accurate method having two levels of specificity which is an advantageous feature for clinical assays. The two levels of specificity are achieved in the target enrichment and detection steps by using sequences specific for the target.<br>
Target enrichment is the first step which prepares the sample for amplification by separating non-specific nucleic acids and contaminants from the specific target. The presence of non-specific or undesirable nucleic acid sequences decreases the sensitivity of target amplification by interfering with amplification and detection. The target enrichment step purifies the sample from possible inhibitors and eliminates non-specific nucleic acids thereby providing the first level of specificity. Eliminating non-specific nucleic acids allows efficient amplification in the isothermal amplification step and thereby minimizes the competition between homologous targets. Target enrichment also denatures double-stranded targets, creating single-stranded DNA for efficient amplification. Various reagents may be used in the target enrichment step in order to improve target capture. For example, subtil isins may be<br>
6<br><br>
used to improve target capture, especially in clinical samples that are bloody. Subtilisins, or carbonyl hydrolases, are alkaline proteases that are secreted by members of the genus Bacillus. When target nucleic acids are captured by beads, the presence of subtilisin causes the beads to form tight pellets, thereby enhancing the ease of washing away unbound or undesirable materials. Furthermore, the target enrichment step also concentrates target nucleic acid in a small volume.<br>
Enrichment of the target is achieved by mixing a target nucleic acid, a DNA:RNA hybrid-specific binding agent, such as a nucleic acid probe which is complementary to the target nucleic acid, for example, a signal sequence probe, and a solid support, where the target nucleic acid and the DNA:RNA hybrid-specific binding agent hybridize to form a DNA:RNA hybrid on a solid support, and removing any nucleic acids which do not form a DNArRNA hybrid or are not captured to a solid support.<br>
In particular, separation of the desired target nucleic acid and the non¬specific nucleic acid is accomplished by the formation of DNA:RNA hybrids that are captured on a solid phase or solid support, such as for example, paramagnetic beads modified with DNA:RNA hybrid-specific binding agents such as, but not limited to, antibodies specific for DNArRNA hybrids (e.g., HYBRID CAPTURE antibody, HC-Ab), monoclonal or polyclonal antibodies, or fragments thereof, proteins, catalytically inactive RNase H, nucleic acids, nucleic acid aptamers, or oligonucleotides having the ability to bind and form a triplex structure. The binding affinity of RNase H for DNArRNA hybrids in the absence of Mg2+ and catalysis via surface plasmon resonance is reported by Haruki, et al. ["Kinetic and stoichiometric analysis for the binding of Escherichia coli ribonuclease HI to RNA-DNA hybrids using surface plasmon resonance" J. Biol Chem. 272:22015-22022,1997, incorporated by reference]. This target enrichment step may be carried out on any solid support, such as on microtiter plates, microchips, beads, paramagnetic/non-paramagnetic beads or any of the previously mentioned solid phases. For example, target DNA, RNA probe, or set of different RNA probes (if in a multiplex reaction), and beads bound with conjugated DNA:RNA HC- antibodies are mixed. The target DNAs and RNA probes<br>
7<br><br>
form DNA:RNA hybrids. The DNA:RNA HC-antibodies are conjugated to solid supports, such as paramagnetic beads. Once the DNA:RNA hybrids are captured onto solid supports, all unbound nucleic acid sequences and contaminants are washed away, preferably by repeated washes using buffers which do not degrade or affect the hybrids.<br>
As an alternative or in addition to using the DNA:RNA hybrid-specific antibodies, oligonucleotides or polynucleotides of any length that are complementary to the targets conjugated to a solid phase may be used for capturing target nucleic acid sequences. For example, oligonucleotides or probes which recognize a specific HPV type may be conjugated to a solid phase, such as a plate, chip, or bead. Several beads conjugated to specific and known probes form a bead set. For example, each bead set is specific for one HPV type. In a multiplex format, multiple bead sets which identify various HPV types, one HPV type per bead set, may be used. Oligonucleotides useful in the capturing the target nucleic acid may be partial or complete, locked nucleic acids (LNA), peptide nucleic acids (PNA), or have other modifications [Current Protocols in Nucleic Acid Chemistry, Eds. Serge L. Beaucage, et al., John Wiley &amp; Sons © 2004]. The nucleic acid probe may have a length up to about 100% of the target nucleic acid length, and range from about 50 bases to about 10 kilobases in length. The target nucleic acid may be either DNA or RNA. If the target nucleic acid is RNA, then cDNA may be generated from the RNA target by any of the commonly known methods, or the target RNA may be captured by a DNA probe, where hybridized DNA probe will be amplified and detected.<br>
The target enrichment step is not limited by the embodiments described herein. One skilled in the art will understand how to purify a target nucleic acid in a sample such that contaminants, inhibitors and other non-specific nucleic acids are removed or eliminated from the sample based upon the principles described herein. Sample preparations are known and understood in the art through commercially available methods and kits, such as plasmid mini- or maxi- preparation kits, gel extraction kits, and DNA and RNA purification kits. These may be used to facilitate<br>
8<br><br>
the target enrichment step in certain circumstances where lower sensitivity is acceptable.<br>
After target enrichment, the sample undergose nucleic acid amplification. Isothermal target amplification of nucleic acids using oligonucleotide primers of random sequences and a DNA polymerase with strand-displacing activity addresses many of the limitations such as, but not inclusive, of limited multiplex capabilities and non-uniform amplification of multiplex PCR. The amplified sequences, or amplicons, are identified through hybridization to the sequence specific oligonucleotides. Amplification may also be performed with the individual targets rather than in multiplex format and amplicons may be combined for the hybridization assay.<br>
DMA polymerase may be used to extend the primers and activate strand displacement. Alternatively, if a DNA probe is used to capture the target RNA, then the hybridized DNA probe is amplified and detected. Another embodiment entails reverse transcribing target RNA to produce cDNA, as the DNA target, which may be captured as described above. This second step preferably uses isothermal amplification of a target nucleic acid. Specifically, oligonucleotide primers of random sequence and a DNA polymerase having strand-displacing activity may be useful in isothermal amplification. In one embodiment, a solid phase or support such as, but not limited to magnetic beads, which has a captured target nucleic acid may be used directly in an amplification mixture. DNA targets may be amplified using any DNA polymerases, (including but not limited to, phi29 DNA polymerase, Bst DNA polymerase, T4 DNA polymerase, T7 DNA polymerase, and DNA Polymerase I), that are primed with random primers.<br>
Random primers may have a specific ratio of dG, dC, dT and dA monomers for optimal efficiency. One or two nucleotide bases could be completely omitted for optimal performance. The random primer pool does not need to include all of the possible sequences. Sequences included in the pool also do not need to be at equimolar concentrations. In fact, random primer pools may include a subset of primer sequences selective for particular diseases. The length may vary from about 4<br>
9<br><br>
to about 20 nucleotides, and preferably from about 5 to about 8 nucleotides. Length may depend on the ratio of monomer within the primer. An optimal amplification temperature varies between about 25°C and about 70°C, preferably about 28°C to about 40°C, depending on the length of the primers which may be determined by one skilled in the art and without undue experimentation. The optimal temperature could be estimated by using commercially available software that predicts the annealing temperature of the oligonucleotides. One example of such commercially available software is OLIGO™ sVersion 6.0 or 6.41 (Molecular Biology Insights; Cascade, CO). For example, the target DNA may be amplified for 2 hours with pentamer primers in combination with phi29 DNA polymerase.<br>
Pentamer primers, for example, are protected from exonuclease degradation by modifications. Phosphorothioate bonds or 2'-0-methyI groups may be included into the structure of the primers. The amplification reaction preferably occurs over 1-3 hours depending on the required sensitivity, which is substantially shorter than the duration of published protocols for phi29 DNA polymerase reactions [Gary J. Vora, et al., Applied andEnvironmental Microbiology, 70(5): 3047-3054,2004, incorporated by reference]. A unique formula of reagents combined with the chosen target enrichment step and the target detection step enables the relatively short amplification time.<br>
The target amplification step of the invention preferably utilizes isothermal amplification which enables: amplification of an unlimited number of targets with uniform efficiency; amplification of the entire sequence, not a fragment as is the case in the polymerase chain reaction (PCR); and allows target detection even if a part of the target sequence in the biological sample is removed. For example, the LI region is often deleted in HPV; however, by using a hybridization probe for the LI and E regions [MH Einstein and GN Goldberg, Cancer Invest. 20:1080-1085,2002], the deleted sequences may still be detected with the same level of sensitivity, thus allowing detection of the target even though part of the sequence is absent. Multiplex isothermal amplification may be performed at a temperature ranging from about 25°C to about 40oC with a reaction time ranging from 1 to 3 hoursV 'Non-limiting examples<br>
10<br><br>
of isothermal amplification include rolling circle amplification [Lizardi P., Huang X., Zhu Z., Bray-Ward P., Thomas D., Ward D. "Mutation detection and single molecule counting using isothermal rolling circle amplification" Nat. Genet. 1998, 19:225-232]; multiple displacement amplification [Little M., et al. "Strand displacement amplification and homogenous real-time detection incorporated in a second generation probe system" Clin. Chem. 1999, 45:777-784]; and protein-primed DNA amplification [Blanco M., Lazaro J., de Vega M, Bonnin A., Salas M "Terminal protein-primed DNA amplification" Proc. Natl. Acad. Sci. USA, 1994,91:12198-12202]. Other target amplification embodiments include those where the target is released from the solid support before adding it to the amplification mixture. Non-limiting ways for releasing the target nucleic acid include, alkali treatment, high temperature incubation, and RNaseH treatment. A unique formula of reagents combined with the chosen target enrichment step and the target detection step permits use of the relatively short amplification time. The reagents for isothermal amplification may be performed simultaneously with target amplification, optionally including RNase H, which would increase the number of priming sites on the target DNA. These methods allow denaturation as well as solid support release enabling more efficient target amplification.<br>
The next step involves the detection of individual target nucleic acid sequences within the amplified nucleic acid. The sensitivity and specificity of this step usually depends on the amplification efficiency. The detection and elucidation of the target nucleic acid is performed by hybridizing different portions of the amplified target product, or ampl icons, to individual capture sequence probes which are conjugated to a solid support and specific probes complementary to the target sequence forming a target complex. Hybridization of different portions of the target nucleic acid sequence with two probes provides two levels of specificity, ensuring the detection and identification of a specific target in a sample.<br>
Oligonucleotides or capture sequence probes conjugated to a solid support, preferably beads which are selectable, are useful in obtaining one level of specificity. Hybridization of the CSPs to the amplified target nucleic acids followed<br>
11<br><br>
by subsequent selection of the beads, enables the separation of the target complex from the unbound or partially bound entities. The.CSPs are generally added in excessive amounts in order to ensure binding. .Preferably, the CSPs and SSPs do not have overlapping sequences; however, it is possible to perform the method in one embodiment where the CSPs and SSPs have overlapping sequences complementary to the target nucleic acid sequence.<br>
Capture sequence probes (CSP) or oligonucleotides may be conjugated or attached to a solid phase, solid support, or solid matrix which includes, for example, polystyrene, polyethylene, polypropylene, polycarbonate or any solid plastic material in the shape of plates, slides, dishes, beads, particles, cups, strands, chips, strips, microplates, and microarrays. A solid phase also includes glass beads, glass test tubes and any other appropriate glass product. A functionalized solid phase such as plastic or glass that has been modified so that the surface contains carboxyl, amino, hydrazide, aldehyde groups, nucleic acid or nucleotide derivatives can also be used. Any solid phase such as plastic, metal, magnetic or glass microparticles, beads, strips, test tubes, slides, strands, chips, microchip or microtiter plates can be used.<br>
Capture sequence probes or oligonucleotides are nucleic acid probes which comprise at least 8 bases, preferably 15 to 100 bases, and more preferably 20 to 40 bases. The capture sequence probes are preferably identifiable, either by their known locations on a solid support, such as a sub-well plate, or when, for example, conjugated to beads, by a distinguishable colored dye. The means for identifying the capture sequence probes are known in the art, and are exemplified in the instant specification.<br>
Hybridization to the sequence specific probes which may be attached to the solid support not only allows for identification of the specific targets, but also provides a second level of specificity to the assay, making it a more reliable clinical assay. If lower sensitivity is required, signal amplification is not necessary and detection using (signal) oligonucleotides or polynucleotides attached to'the reporter (biotin, fluorophore, enzyme, etc.) may be used. There may be more than one detection oligonucleotide per target. Both capture and detection oligonucleotides may<br>
12<br><br>
be additionally modified, e.g., PNA, LNA, etc. Oligonucleotides or nucleic acid probes may have a length ranging from about 15 bases to about 10 kilobases, or up to 100 % of the target length.<br>
A signal sequence probe (SSP) may be an unlabeled RNA that is used to form an DNA:RNA hybrid recognized by, for example, a labeled hybrid-specific binding agent during the detection step. Since the SSP is unlabeled, it does not create background noise when unbound to the target, thereby resulting in more specificity. A signal amplified by approximately 400 fold may be achieved if, for example, the size of the RNA signal probe is 8 kilobases (kb) and each antibody binds to 20 bases. As previously described, generally, the signal sequence probe comprise at least 15 bases, but may be up to or greater than about 1000 bases, preferably between about 15 to 100 bases. Other non-limiting examples of a signal sequence probe include labeled RNA probe and labeled DNA probe. Hybridization is not limited to non-overlapping regions, but may in fact include overlapping regions. The detection of the RNA:DNA hybrid complex bound to a solid support may be performed in a multiplex format using, for example, a PE-labeled antibody, carboxylated distinguishable beads, and detected by flow-cytometry.<br>
One embodiment for target (or amplicon) detection utilizes a liquid-based array. Bead arrays are commercially available and in this embodiment, carboxylated polystyrene bead arrays are preferable. Each well of a 96-welI plate, for example, has a mixture of bead sets. A 13-plex has 13 bead sets where each bead set has a specific "signature" and the signature is provided by dyes that are inside each bead. The ratio of these dyes is specific for each bead set, and enables differentiation between each of the bead sets. Capture sequence probes (CSPs) or oligonucleotides specific for one target nucleic acid are applied or conjugated to one particular bead set. When the target is hybridized to the bead conjugated CSPs or oligonucleotides, selection of a particular bead set and then detection occurs using the complementary nucleic acid probe and labeled DNA:RNA hybrid-specific binding agent. The selection or separation may be carried out in a fiow-cytometer, where the beads proceed one-by-one through two lasers: one of which selects the signature on the bead,<br>
13<br><br>
while the other detects the target as identified by the labeled DNA:RNA hybrid-specific binding agents. In this way, multiple targets may be differentiated and detected. Additionally, the labeled DNA:RNA reagent allows enhanced signal detection, thereby increasing both the specificity and sensitivity of the assay.<br>
A further embodiment to the liquid-based bead array of the instant invention utilizes sub-well plates. A sub-well plate is a microtiter plate, having, for example, 96 primary wells, where each individual primary well is subdivided into a number of sub-wells. The sub-well plate as a platform for multiplex has been previously described [A. Roda, et al. Clin.Chem 46: 1654-1660, 2002, incorporated by reference]. However, the subwell platform, as previously used and described, has limited use because of the "cross-talk" problem between different sub-wells. Whereas, the modified sub-well platform described here essentially eliminates the problem of cross-talk between different subwells by the use of masks. Each sub-well preferably has a known target-specific capture sequence probe attached to the well. Target nucleic acids may be added to the primary well.where the target nucleic acid hybridizes to its complementary nucleic acid probe, i.e., signal sequence probe. DNA:RNA hybrids are formed between a portion of the target nucleic acid and the signal sequence probe. The capture sequence probe binds a different portion of the target nucleic acid, thereby capturing the DNA:RNA hybrid to the solid phase. Detection of the target nucleic acid occurs as previously described using complementary nucleic acid probes which hybridizes to the target nucleic acid, and labeled DNA:RNA hybrid-specific antibodies, for example, and by the known capture sequence probe. Signal amplification and the use of a mask which acts as a lid to eliminate the cross-talk between the sub-wells both enable the specific and sensitive detection of target nucleic acids with minimal background noise.<br>
A wide variety of methods are available that may be used for detecting and identifying the target sequence including, but not limited to dot blot hybridization, reverse blot hybridization on nylon membrane, hybridization of slide arrays, chip arrays, bead arrays, arrays on the bottom of a multi-well plates, or the like. Non-limiting examples of solid supports for hybridization include bead arrays (for example,<br>
14<br><br>
bead array products from commercially available sources), slide arrays, plate arrays (e.g., arrays on the bottom of each well of a microtiter plate), sub-well plates (microtiter plate that has 16-64 small sub-wells within each well), and arrays of electrodes such as the GenOHM system (GeneOhm Sciences, Inc.). Drummond, et al., Nature Biotech, 21: 1192-1199 (2003). Different types of reporters may be implemented to generate signals for detection through, for example, fluorescence, chemiluminescence, and gold nanoparticles. As previously discussed, the detector means may include fluorescence, any enzyme-based signal, chemiluminescent or colorimetric signals, light scattering with gold particles, or the like.<br>
In one embodiment, a reporter may be attached to a hybrid-specific binding agent, such as but not limited to, antibodies specific for RNA:DNA hybrids (HC-Ab), proteins, catalytically inactive RNase H, nucleic acids, nucleic acid aptamers, or oligonucleotides having the ability to bind and form a triplex structure. Each labeled binding agent binds a nucleotide segment of the DNA:RNA hybrid, for example, at least 20 nucleotides in length, providing signal amplification.<br>
A further embodiment of the invention for the multiplex detection of a plurality of target DNA in an unpurified sample, comprises the following steps: 1) denaturing target DNA to create a single-stranded target sequence; 2) adding RNA probes and magnetic beads, where the beads have been previously conjugated to antibodies that are specific for hybrids; 3) removing any unbound targets and contaminants by washing; 4) performing the isothermal amplification of the captured targets for a relatively short period of time, such as for example 1-3 hours, in the presence of DNA polymerase and other reagents to initiate amplification and formation of amplicons; 4) denaturing the amplicons to create single-stranded sequences; 5) hybridizing the amplified target to oligonucleotides conjugated to a solid support (capture probes), such as beads or a microtiter plate, and to a signal sequence probe; and 6) detecting the presence or absence of each target DNA by differentiating the targets based on the capture probe features and detecting the signal sequence probe complex.<br>
15<br><br>
In one embodiment of the invention, the hybridization of a target and probe may occur simultaneously with the capture step by a hybrid-binding agent while in the same mixture and at an elevated temperature. The elevated temperature during the entire process may allow an increase in specificity of target capture, while decreasing the reaction time. It is to be understood that the low, moderate and high stringency hybridization / washing conditions may be varied using a variety of ingredients, buffers and temperatures well known to and practiced by the skilled artisan. For additional stringency conditions, see T. Maniatis et al., Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY ,(1982). The one step hybridization and capture may also be more efficient than performing hybridization and capture sequentially, depending on the overall assay conditions.<br>
A method of detecting target nucleic acids of interest in the presence of other nucleic acids, by target enrichment; target amplification; and target detection, provides for a rapid, sensitive method of simultaneously detecting multiple targets in the same reaction sample. This method is useful in clinical diagnostic applications for identifying, many disease states for example, whether a patient is infected with the Human Papillomavirus (HPV) and determining which specific HPV type or types the patient has, in order to better diagnose and treat the patient. Other diseases related to specific nucleic acids are readily known in the art and would be identifiable using the detection method described in the invention.<br>
Any source of nucleic acid, in purified or non-purified form, can be utilized as the test sample. For example, the test sample may be a food or agricultural product, or a human or veterinary clinical specimen. Typically, the test sample is a biological fluid such as urine, blood, plasma, serum, sputum or the like. Alternatively the test sample may be a tissue specimen suspected of carrying a nucleic acid of interest. The target nucleic acid in the test sample may be present initially as a discrete molecule so that the sequence to be detected constitutes the entire nucleic acid, or may only be a component of a larger molecule. It is not necessary that the nucleic acid sequence to be detected be present initially in a. pure form. The test sample may<br>
&gt;S<br><br>
contain a complex mixture of nucleic acids, of which the target nucleic acid may correspond to a gene of interest contained in total human genomic DNA or RNA or a portion of the nucleic acid sequence of a pathogenic organism which organism is a minor component of a clinical sample.<br>
The target nucleic acid in a test sample can be DNA or RNA, such as messenger RNA, from any source, including bacteria, yeast, viruses, and the cells or tissues of higher organisms such as plants or animals. Target could also be a cDNA synthesized from the RNA by reverse transcription. Methods for the extraction and/or purification of such nucleic acids are well known in the art. Target nucleic acids may be double-stranded or single-stranded. In one embodiment of the present method, the target nucleic acids are single-stranded or made single-stranded by conventional denaturation techniques prior to the hybridization and amplification steps of the method. In one embodiment, a base denaturation technique or high temperatures are used to denature the double-stranded target DNA.<br>
The term "oligonucleotide" as the term is used herein refers to a nucleic acid molecule comprised of two or more deoxyribonucleotides or ribonucleotides. A desired oligonucleotide may be prepared by any suitable method, such as purification from a naturally occurring nucleic acid, by molecular biological means, or by de novo synthesis. Non-limiting examples of oligonucleotides and nucleic acid probes are described herein.<br>
Nucleic acid probes are nucleic acid sequences that hybridize to complementary RNA or DNA sequences in a test sample. Detection of the probe indicates the presence of a particular nucleic acid sequence in the test sample. The nucleic acid probes may be directly or indirectly detected. In one embodiment, the target-specific HYBRID CAPTURE method employs two types of nucleic acid sequence probes: capture sequence probe (CSP) and signal sequence probe (SSP).<br>
A capture sequence probe or oligonucleotide comprises a nucleic acid sequence which is capable of hybridizing to unique region(s) of a target nucleic acid and being captured onto a solid phase or which enables the capture of the target nucleic acid sequence. The CSP used in the detection method can be DNA, RNA,<br>
17<br><br>
peptide nucleic acids (PNAs) or other nucleic apid analogues. PNAs are oligonucleotides in which the sugar-phosphate backbone is replaced with a polyamide or "pseudopeptide" backbone. In a preferred embodiment, the CSP is DNA. The CSP has a minimum length of at least 8 bases, preferably between 15 and 100 bases, and more preferably between 20 and 40 bases. The CSP is substantially complementary to a target nucleic acid sequence to which the CSP hybridizes. The sequence of a CSP is preferably at least 75% complementary to the target hybridization region, more preferably, 100% complementary to this sequence. It is also preferred that the CSP contains less than or equal to 75% sequence identity, more preferably less than 50% sequence identity, to undesirable non-target sequences believed to be present in a test sample. The sequence within a target nucleic acid to which a CSP binds is preferably at least about 12 bases, more preferably 20-40 bases. The target nucleic acid sequences to which the CSP hybridizes are preferably unique sequences or group-specific sequences. Group-specific sequences are multiple related sequences that form discrete groups. For example, the CSPs may contain sequences which recognize specific human papillomavirus types, such as but not limited to, HPV-16, HPV-18, and HPV-31.<br>
In one embodiment, the CSP used in the detection method may contain one or more modifications in the nucleic acid which allows specific capture of the probe onto a solid phase. For example, the CSP may be modified by tagging it with at least one ligand by methods well-known to those skilled in the art including, for example, nick-translation, chemical or photochemical incorporation. In addition, the CSP may be tagged at multiple positions with one or multiple types of labels. For example, the CSP may be tagged with biotin, which binds to streptavidin; or digoxigenin, which binds to anti-digoxigenin; or 2,4-dinitrophenol (DNP), which binds to anti-DNP. Fluorogens can also be used to modify the probes. Examples of fluorogens include fluorescein and derivatives, phycoerythrin, allo-phycocyanin, phycocyanin, rhodamine, Texas Red or other proprietary fluorogens. The fluorogens are generally attached by chemical modification and bind to a fluorogen-specific antibody, such as anti-fluorescein. It will be understood by those skilled in the art that<br>
18<br><br>
the CSP can also be tagged by incorporation of a modified base containing any chemical group recognizable by specific antibodies. Other tags and methods of tagging nucleotide sequences for capture onto &amp; solid phase coated with substrate are well known to those skilled in the art. A review of nucleic acid labels can be found in the article by Landegren, et al, "DNA Diagnostics-Molecular Techniques and Automation", Science, 242:229-237 (1988), which is incorporated herein by reference. In a further embodiment, the CSP is tagged with biotin on both the 5' and the 3' ends of the nucleotide sequence. In another embodiment, the CSP is not modified, but is captured on a solid matrix by virtue of sequences contained in the CSP capable of hybridization to the matrix.<br>
The SSP used in the detection method may be a DNA or RNA. A signal sequence probe comprises a nucleic acid sequence which is capable of hybridizing to regions of a target nucleic acid that are adjacent to the unique regions recognized by the CSP. In one particular embodiment of the invention, the SSP and target nucleic acid form a DNA:RNA hybrid. Therefore, in this embodiment, if the target nucleic acid is a DNA, then the preferred SSP is an RNA. Similarly, if the target nucleic acid is RNA, then the preferred SSP is a DNA. The SSP is generally at least 15 bases long. However, the SSP may be up to or greater than 1000 bases long. Longer SSPs are preferred. The SSP may comprise a single nucleic acid fragment, or multiple smaller nucleic acid fragments each of which is preferably between 15 to 100 bases in length.<br>
The sequences of CSP and SSP are selected so that they do not hybridize to the same region of a target nucleic acid or to each other. The SSP sequence and CSP sequence which correspond to regions of a target nucleic acid sequence, may be modified in order to eliminate competition between the CSP and SSP. For example, the SSP may have deletions that the CSP does not include. In addition, the CSPs and the SSPs are selected to hybridize to regions of the target within, for example, 50,000 bases of each other. The distance between the sequence to which the CSP hybridizes within the target nucleic acid and-the sequence to which the SSP hybridizes is preferably between about 1 and 50,000 bases. More preferably, the<br>
19<br><br>
distance between the CSP and SSP on a target nucleic acid is less than 3,000 bases, and most preferably, the distance is less than 1,000 bases.<br>
In another embodiment, a portion of the SSP used in the detection method forms a DNA:RNA hybrid with a single-stranded target nucleic acid sequence, which is detected by DNA:RNA hybrid-specific binding agents, and another portion of the SSP is capable of hybridizing to the target nucleic acid. The SSP may be prepared by first cloning a single-stranded DNA sequence complementary to sequences within the target nucleic acid into a single-stranded DNA vector, then hybridizing RNA complementary to the DNA vector sequence to generate a DNA:RNA hybrid. For example, if Ml 3 is used as the DNA vector, M13 RNA is hybridized to the M13 DNA sequence in the vector to generate a DNA:RNA hybrid. The resulting SSP forms a DNA:RNA hybrid portion as well as a single-stranded portion capable of hybridizing to sequences within the target nucleic acid. The single-stranded DNA should be at least 10 bases long, and may be up to or greater than 1000 bases long. Alternatively, the DNA.RNA hybrid portion of the SSP may be formed during or after the reaction in which the single-stranded portion of the SSP is hybridized to the target nucleic acid. The SSP can be linear, circular, or a combination of two or more forms. The DNA.-RNA hybrid portion of the SSP provides amplified signals for the detection of captured hybrids using, for example, DNA:RNA hybrid specific antibodies, which are labeled or are recognized by a labeled entity, as described herein.<br>
In yet another embodiment, the SSP used in the detection method is a molecule which does not contain sequences that are capable of hybridizing to the target nucleic acid. In this embodiment, bridge probes comprising sequences capable of hybridizing to the target nucleic acid as well as sequences capable of hybridizing to the SSP are used. The bridge probes can be DNA, RNA, peptide nucleic acids (PNAs) or other nucleic acid analogues.<br>
In a further embodiment, a portion of the SSP and a complementary nucleic acid probe form a DNA:RNA hybrid and a single-stranded portion of the SSP contains sequences complementary to sequences within a bridge probe. The bridge probe, which is capable of hybridizing to both the target nucleic acid and the SSP,<br>
20<br><br>
serves as an intermediate for connecting the SSP to the target nucleic acid. The DNA:RNA hybrid is detected by a DNA:RNA hybrid-specific binding agent which may be labeled or is detected by a labeled entity. The CSP hybridizes to a different portion of the target nucleic acid sequence, thereby capturing the target, bridge probe, and DNA:RNA hybrid complex to a solid phase or support. The SSP may be prepared as described above.<br>
In another embodiment, the SSP used in one target nucleic acid detection method comprises multiple sets of repeat sequences which are complementary to a single-stranded RNA sequence capable of hybridizing to a bridge probe. A DNA oligonucleotide probe containing sequences complementary to the repeat sequences may be used to hybridize to the SSP to generate the DNA:RNA duplex needed for signal amplification.<br>
In yet another embodiment, the bridge probe contains a poly(A) tail in addition to sequences which are capable of hybridizing to the target nucleic acid. The SSP used in this example comprises poly (dT) DNA sequences. The bridge probe is therefore capable of hybridizing to the SSP via its poly(A) tail. A RNA probe comprising poly(A) sequences may be used to hybridize to the remaining poly (dT) DNA sequences within SSP to form a DNA:RNA hybrid. The SSP comprising poly (dT) sequences and the RNA probe comprising poly (A) sequences are preferably 100 to 5,000 bases long.<br>
The SSP used in the methods of detecting target nucleic acid sequences of the instant invention may be unmodified or modified as with the CSP using methods described above and/or known in the art. In a preferred embodiment, the SSP is a covalently unmodified probe.<br>
It is understood that multiple CSPs and/or SSPs can be employed in the detection method of the invention.<br>
In another embodiment, an oligonucleotide probe comprising complementary sequences of two or more distinct regions of the target nucleic acid are fused together and used as the capture sequence probe in the method of the invention. Alternatively a single probe can be designed and produced which contains sequences<br>
21<br><br>
complementary to single or multiple target nucleic acids. This type of probe is also referred to herein as a "fused" CSP. The fused capture sequence probe works as effectively as the combination of two unfused CSPs when used at the same concentration.<br>
The nucleic acid probes of the invention may be produced by any suitable method known in the art, including for example, by chemical synthesis, isolation from a naturally-occurring source, recombinant production and asymmetric PCR (McCabe, 1990 In: PCR Protocols: A guide to methods and applications. San Diego, CA., Academic Press, 76-83). It may be preferred to chemically synthesize the probes in one or more segments and subsequently link the segments, as may be the case for preparing an oligonucleotide microarray on a microchip. Several chemical synthesis methods are described by Narang et aL (1979 Meth. Enzymol. 68:90), Brown et al. (1979 Meth. Enzymol. 68:109) and Caruthers et al. (1985 Meth. Enzymol. 154:287), all of which are incorporated herein by reference. Alternatively, cloning methods may provide a convenient nucleic acid fragment which can be isolated for use as a promoter primer. A double-stranded DNA probe is first rendered single-stranded using, for example, conventional denaturation methods prior to hybridization to the target nucleic acids.<br>
Hybridization is conducted under standard hybridization conditions well-known to those skilled in the art. Reaction conditions for hybridization of a probe to a nucleic acid sequence vary from probe to probe, depending on factors such as probe length, the number of G and C nucleotides in the sequence, and the composition of the buffer utilized in the hybridization reaction. Moderately stringent hybridization conditions are generally understood by those skilled in the art as conditions approximately 25°C below the melting temperature of a perfectly base-paired double stranded DNA. Higher specificity is generally achieved by employing incubation conditions having higher temperatures, in other words more stringent conditions. Chapter 11 of the well-known laboratory manual of Sambrook et al., MOLECULAR CLONING: A LABORATORY MANUAL, second edition, Cold Spring Harbor Laboratory Press, New York (1990) (which is incorporated by reference<br>
2.2.<br><br>
herein), describes hybridization conditions for oligonucleotide probes in great detail, including a description of the factors involved and the level of stringency necessary to guarantee hybridization with specificity. Hybridization is typically performed in a buffered aqueous solution, for which conditions such as temperature, salt concentration, and pH are selected to provide sufficient stringency such that the probes hybridize specifically to their respective target nucleic acid sequences but not any other sequence.<br>
Generally, the efficiency of hybridization between probe and target improve under conditions where the amount of probe added is in molar excess to the template, preferably a 2 to 106 molar excess, more preferably 103 to 106 molar excess. The concentration of each CSP provided for efficient capture is at least 25 fmoles/ml (25 pM) in the final hybridization solution, preferably between 25 fmoles to 104 fmoles/ml (10 nM). The concentration of each SSP is at least 15 ng/ml in the final hybridization solution, preferably 150 ng/ml. Table A shows the conversion of SSP concentrations expressed in ng/ml to molar basis.<br>
Table A<br><br>
Conversion of SSP Concentration From ng/ml to fmoles/ml<br>
SSP Concentration in ng/ml	SSP Concentration in fmoles/ml (pM)<br><br>
	SSP is a 3kb RNA	SSP is a 5kb RNA<br>
15 ng/ml	15.1	9<br>
150 ng/ml	151	90<br>
600 ng/ml	606	364<br>
Hybridization of the CSP and the SSP to the target nucleic acid in the detection step may be performed simultaneously or sequentially and in either order. In one embodiment, hybridization of the CSP and hybridization of the SSP to the target nucleic acid are performed simultaneously. In one embodiment, the DNA:RNA hybrid formed may then be captured onto a solid phase. The solid phase may be coated with<br>
23<br><br>
a substrate to which ligand attached to the CSP binds with specificity. A portion of the target nucleic acid sequence and the SSP form a DNA:RNA hybrid, while another portion of the target nucleic acid sequence hybridizes to a CSP which is attached to a solid phase. In another embodiment, hybridization of the SSP to the target nucleic acid is performed after the hybridization of the CSP to the target nucleic acid. In this case, the CSP may be immobilized on a solid phase before or after hybridization. Both the CSP and the target nucleic acid sequence may be bound to the solid phase during the SSP hybridization reaction.<br>
It will be understood by those skilled in the art that a solid phase, solid support, or solid matrix, may be used interchangeably and includes, for example, glass, silicon, metal, nitrocellulose, polystyrene, polyethylene, polypropylene, polycarbonate or any solid plastic material in the shape of plates, slides, dishes, beads, microbeads, particles, microparticles, cups, test tubes, slides, strands, chips, microchips, strips, membranes, microplates, microtiter plates with subwells, and microarrays. A solid phase also includes glass beads, glass test tubes and any other appropriate glass product. A functionalized solid phase such as plastic or glass that has been modified so that the surface contains carboxyl, amino, hydrazide, aldehyde groups, nucleic acid or nucleotide derivatives can also be used.<br>
In one embodiment, the CSP is labeled with biotin, and streptavidin-coated or avidin-coated solid phase is employed to capture the hybrid. In another embodiment, streptavidin-coated microtiter plates or microplates are used. These plates may be coated passively or covalently. A further embodiment utilizes microtiter plates where each individual well has several sub-wells, where a CSP is individually attached in each sub-well. Another embodiment employs CSPs bound to beads by any of the known and commonly used methods of binding, such as using spacers and linkers. When the CSP is conjugated to a solid support, the CSP may be used for capturing and/or selecting for the target nucleic acid to which it is complementary.<br>
The captured DNA:RNA hybrid may be detected by conventional means well-known in the art. The DNA:KNA hybrid may be detected directly or indirectly using a labeled or distinguishable DNA:RNA hybrid-specific binding agent,<br>
24<br><br>
such as a labeled polyclonal or monoclonal antibody, or fragment thereof, specific for the DNA:RNA hybrid, an antibody specific for one or more ligands attached to the SSP, a labeled antibody, or a detectable modification on the SSP itself.<br>
One preferred method detects the captured hybrid by using a DNA:RNA hybrid-specific antibody. In this embodiment, the DNA:RNA hybrid-specific antibody is preferably labeled with an enzyme, a fluorescent molecule or a biotin-avidin conjugate, and is preferably non-radioactive. The label can be detected directly or indirectly by conventional means known in the art such as a colorimeter, a luminometer, or a fluorescence detector. One preferred label is, for example, alkaline phosphatase. Other labels known to one skilled in the art can also be employed as a means of detecting the bound double-stranded hybrid. Detection is performed by methods conventionally used in the art, for example, colorimetry or chemiluminescence as described at Coutlee, et al, J. Clin. Microbiol. 27:1002-1007 (1989), incorporated by reference. Preferably, bound alkaline phosphatase conjugate is detected by chemiluminescence by adding a substrate which can be activated by alkaline phosphatase. Chemiluminescent substrates that are activated by alkaline phosphatase are well known in the art.<br>
Detection of captured DNA:RNA hybrids is preferably achieved by binding the conjugated DNA:RNA hybrid-specific binding agent, such as but not limited to a DNA:RNA hybrid-specific antibody, to the DNA:RNA hybrid during an incubation step. In one embodiment, the DNA.-RNA hybrid-specific antibody is conjugated to a solid phase, for example, a paramagnetic bead. By placing the paramagnetic beads conjugated to the DNA:RNA hybrid-specific antibodies under a magnetic force, thereby capturing the DNArRNA hybrids, surfaces may then be washed to remove any excess antibodies and other non-hybridized materials. This step enriches or purifies the target nucleic acid sequence. These target enrichment techniques are known in the art. For example, manual washes may be performed using either an repeat pipettors or syringes, a simple pump regulated by a variostat, or by gravity flow from a reservoir with attached tubing. Commercially available tube washing systems available can also be used.<br>
25<br><br>
Non-limiting examples of capture sequence probes useful in the invention include those having sequences for HSV-1; sequences for HSV-2. CSPs for HPV include those having SEQ ID NOs: 1-64. The HPV types that are encompassed by these sequences include: HPV 16, HPV 18, HPV26, HPV31, HPV33, HPV35, HPV39, HPV45, HPV51, HPV52, HPV56, HPV58, HPV59, HPV66, HPV68, HPV73, and HPV82, several of which are high risk HPVs.<br>
In yet another embodiment, the method employs blocker probes in addition to the CSP and SSP. A blocker probe comprises sequences that are substantially complementary to the sequences of the CSP. The sequence of a blocker probe is preferably at least about 75% complementary to the sequence of the CSP, more preferably, 100% complementary to the CSP. The addition of the blocker probes to the hybridization reaction mixture prevents non-hybridized CSP from hybridizing to cross-reactive nucleic acid sequences present in the target and therefore increases the specificity of the detection. The blocker probe is generally at least about 5 bases to about 12 bases. In one embodiment of the invention, the concentration of the blocker probe in the hybridization reaction is preferably in excess to that of the CSP and SSP. Preferably, the blocker probe is present in about a 2-fold molar excess, although, it may be present in an up to 10,000-fold molar excess. The blocker probes can be DNA, RNA, peptide, non-specific nucleic acids (PNAs) or other nucleic acid analogues.<br>
In one embodiment, blocker probes complementary to the full-length or near full-length of the CSP are used. Following the reaction in which the hybrid between CSP, SSP and the target nucleic acid is formed, one or more blocker probes may be added to the reaction and the hybridization is continued for a desired time. The hybridization products are then detected as described above.<br>
In another embodiment, blocker probes complementary to only a portion of the CSP and shorter than the CSP are used. These blocker probes have lower melting temperatures than the CSPs. Preferably, the melting temperature of the blocker probe is 10 degrees lower than that of the CSP. In this embodiment the blocker probe is preferably added to the target nucleic acids simultaneously with the CSP and the SSP. Since the blocker probe has a lower melting temperature than the<br>
26<br><br>
CSP, the initial temperature for hybridization is chosen such that the blocker probe does not interfere with the hybridization of the CSP to its target sequences. However, when the temperature of the hybridization mixtures is adjusted below the temperature used for target hybridization, the blocker probe hybridizes to the CSP and effectively blocks the CSP from hybridizing to cross-reactive nucleic acid sequences. For example, when the hybridization products are incubated at room temperature on a streptavidin-coated microtiter plate during hybrid capture, the blocker probes may be added. Non-limiting examples of blocker probes are found in Table 2.<br>
The following examples illustrate use of the present amplification method and detection assay and kit. These examples are offered by way of illustration, and are not intended to limit the scope of the invention in any manner. All references described herein are expressly incorporated in toto by reference.<br>
EXAMPLE 1 TARGET-SPECIFIC HYBRID CAPTURE (TSHC) ASSAY PROTOCOL<br>
Herpes Simplex Virus 1 (HSV-1) and Herpes Simplex Virus 2 (HSV-2) viral particles of known concentration (Advanced Biotechnologies, Inc., Columbia, MD) or clinical samples were diluted using either Negative Control Media (Digene Corp., Gaithersburg, MD) or Negative Cervical Specimens (Digene Corp). Various dilutions were made and aliquoted into individual microfuge tubes. A half volume of the Denaturation Reagent 5100-0431 (Digene Corp.) was added. Test samples were incubated at 65°C for 45 minutes to allow denaturation of nucleic acids in the samples.<br>
Following denaturation, a hybridization solution containing signal sequence probes (SSPs) (600 ng/ml each) and capture sequence probes (CSPs) (2.5 pmoles/ml each) was added to the sample, and incubated at 74°C for 1 hour. Blocker probes in a solution containing one volume of 4x Probe Diluent (Digene Corp.), one volume of Denaturation Reagent and two volumes of the Negative Control Media were then added to the hybridization mixture and incubated at.74°C for 15 minutes.<br>
27<br><br>
In a second series of experiments, following the denaturation of nucleic acids, a hybridization mixture containing SSPs (600 ng/ml each), CSPs (2.5 pmoles/ml each), and blocker probes (250 pmoles/ml each) was added to the samples and incubated at 74°C for 1 hour.<br>
Tubes containing reaction mixtures were cooled at room temperature for 5 minutes, and aliquots were taken from each tube and transferred to individual wells of a 96-well streptavidin capture plate (Digene Corp.). The plates were shaken at 1100 rpms at room temperature for 1 hour. The supernatants were then decanted and the plates were washed twice with SNM wash buffer (Digene Corp.) and inverted briefly to remove residual wash buffer. The alkaline-phosphatase anti-RNA/DNA antibody DR-1 Reagent (Digene Corp.) was then added to each well and incubated at room temperature for 30 minutes. The wells were then subjected to multiple wash steps which include: 1) three washes with Sharp wash buffer (Digene Corp.) at room temperature; 2) incubation of the plate with the Sharp wash buffer at 60°C for 10 minutes on a heat block; 3) two washes with the Sharp wash buffer at room temperature; and 4) one wash with the SNM wash buffer (Digene Corp.) at room temperature. Following removal of the residual liquid, luminescent substrate 5100-0350 (Digene Corp.) was added to each well and incubated at room temperature for 15 minutes. The individual wells were then read on a plate luminometer to obtain the relative light unit (RLU) signal.<br>
Solutions containing Negative Control Media or known HSV Negative Cervical Specimens were used as negative controls for the test samples. The signal to noise ratio (S/N) was calculated as the ratio of the average RLU obtained from a test sample to the average RLU of the negative control. The signal to noise ratio was used as the basis for determining capture efficiency and the detection of target nucleic acids. A S/N value of 2 or greater was arbitrarily assigned as a positive signal while a S/N values less than 2 was considered negative. The coefficient of variation (CV) which is a determination of the variability of the experiment within one sample set was calculated by taking the standard deviation of the replicates, dividing them by the average and multiplying that value by 100 to give a percent value.<br>
28<br><br>
The capture sequence probes and the blocker probes used in the experiments were synthesized using the method described by Cook et aL (1988 Nucl. Acid. Res., 16: 4077-95). Unless otherwise noted, the capture sequence probes used in the experiments described herein were labeled with biotins at their 5' and 3' ends.<br>
The signal sequence probes used in the experiments are RNA probes. These probes were prepared using the method described by Yisraeli et al. (1989, Methods in Enzymol, 180:42-50).<br>
AH of the CSPs were assayed for their type specific abilities in Type Specific HYBRID CAPTURE assays described herein. The following CSPs, i.e., SEQ ID NOs: 36-56, did not show cross-reactive properties with other types and were accepted for use in the assay.<br>
Table 1: Capture Sequence Probes For HPV<br>
Probe	Sequence	9 Size (bp)	HPV Type and Sequence Location<br>
ZL-1	GTACAGATGGTACCGGGGTTGTAGAAGTATCTG        33 [SEQ  ID NO: I]	HPV16 5360-5392<br>
ZL-4	CTGCAACAAGACATACATCGACCGGTCCACC              31 [SEQ lD NO:21	HPV 16 495-525<br>
DP-1	GAAGTAGGTGAGGCTGCATGTGAAGTGGTAG             31 [SEQ lD NO:31	HPV16 5285-5315<br>
DP-4	CAGCTCTGTGCATAACTGTGGTAACTTTCTGGG           33 [SEQ ID NO:41                                                                 |	HPV16 128-160<br>
SH-1	GAGGTCTTCTCCAACATGCTATGCAACGTCCTG [SEQ ID NO:5]	33	HPV31 505-537<br>
SH-4	GTGTAGGTGCATGCTCTATAGGTACATCAGGCC [SEQ lD NO:6]	33	HPV31 5387-5419<br>
VS-1	CAATGCCGAGCTTAGTTCATGCAATTTCCGAGG [SEQ ID NO:7]	33	HPV31 132-164<br>
VS-4	GAAGTAGTAGTTGCAGACGCCCCTAAAGGTTGC [SEQ ID  NO:8]	33	HPV31 5175-5207<br>
AH-1	GAACGCGATGGTACAGGCACTGCAGGGTCC [SEQ ID NO:9]	30	HPV18 5308-5337<br>
AH-2	GAACGCG A TGGTACAGGCACTGCA [SEQ ID NO: 10]	24	HPV18 5314-5337<br>
AL-1	ACGCCCACCCAATGGAATGTACCC ■' [SEQ ID NO: 11]	24	HPV18 4451-4474<br>
PA-4	TCTGCGTCGTTGGAGTCGTTCCTGTCGTGCTC [SEQlDNO:12]	32	HPV18 535-566<br>
18-1AB	'(TTATTATTA)CTACATACATTGCCGCCATGTTCG CCA [SEQ ID NO: 13]	36	HPV18 1369-1395<br>
18-2AB	(TTATTATTAyTGTTGCCCTCTGTGCCCCCGTTGTC	46	HPV18<br>
29<br><br>
Table 1: Capture Sequence Probes For HPV<br>
Probe	Sequence	Size bp	HPV Type and Sequence Location<br>
	TATAGCCTCCGT [SEQ ID NO:14]		1406-1442<br>
18-3AB	(TTATTATTA)GGAGCAGTGCCCAAAAGATTAAAG TTTGC [SEQ ID NO:15]	38	HPV18 7524-7552<br>
I8-4AB	(TTATTATTA)CACGGTGCTGGAATACGGTGAGGG GGTG[SEQ ID NO:16]	37	HPV18 3485-3512<br>
18-5AB	(TTATTATTA)ACGCCCACCCAATGGAATGTACCC [SEQ ID NO: 17]	33	HPV 18 4451-4474<br>
18-6AB	(TTATTATTA)ATAGTATTGTGGTGTGTTTCTCACA T [SEQ ID NO: 18]	35	HPV18 81-106<br>
18-7AB	(TTATTATTA)GTTGGAGTCGTTCCTGTCGTG [SEQ ID NO: 19]	30	HPV 18 538-558<br>
18-8AB	(TTATTATTA)CGGAATTTCATTTTGGGGCTCT [SEQ ID NO:2Q]	31	HPV18 634-655<br>
PE-1	GCTCGAAGGTCGTCTGCTGAGCTTTCTACTACT [SEQ ID NO:21]	33	HPV 18 811-843<br>
PZ-2	GCGCCATCCTGTAATGCACTTTTCCACAAAGC [SEQ ID NO:22]	32	HPV45 77-108<br>
PZ-5	TAGTGCTAGGTGTAGTGGACGCAGGAGGTGG [SEQ ID NO:23]	31	HPV45 5295-5325<br>
CS-1	GGTCACAACATGTATTACACTGCCCTCGGTAC [SEQ ID NO:24]	32	HPV45 500-531<br>
CS-4	CCTACGTCTGCGAAGTCTTTCTTGCCGTGCC [SEQ ID NO:25]	31	HPV45 533-563<br>
PF-1	CTGCATTGTCACTACTATCCCCACCACTACTTTG [SEQ ID NO:26]	34	HPV45 1406-1439<br>
PF-4	CCACAAGGCACATTCATACATACACGCACGCA [SEQ ID NO:27]	32	HPV45 7243-7274<br>
PA-1	GTTCTAAGGTCCTCTGCCGAGCTCTCTACTGTA [SEQ ID NO:28]	33	HPV45 811-843<br>
45-5AB	(TTATTATTA)TGCGGTTTTGGGGGTCGACGTGGA GGC [SEQ ID NO:29]	36	HPV45 3444-3470<br>
45-6AB	(TTATTATTA)AGACCTGCCCCCTAAGGGTACATA GCC [SEQ ID NO:30]	36	HPV45 4443-4469<br>
45-8AB	(TTATTATTA)CAGCATTGCAGCCTTTTTGTTACTT GCTTGTAATAGCTCC [SEQ ID NO:31]	49	HPV45 1477-1516<br>
45-9AB	(TTATTATTA)ATCCTGTAATGCACTTTTCCACAAA [SEQ ID NO:32]	34	HPV45 79-103<br>
45-1OAB	(TTATTATTA)GCCTGGTCACAACATGTATTAC [SEQ ID NO:33]	31	HPV45 514-535<br>
45-11AB	(TTATTATTA)CAGGATCTAATTCATTCTGAGGTT rSEO ID NO:341	33	HPV45 633-656<br>
ON-1	TGCGGTTTTGGGGGTCGACGTGGAGGC [SEQ ID NO.-351	27	HPV45 3444-3470<br>
ZZ-1	GGCGCAACCACATAACACACAGAACCACAAAAC [SEQ ID NO:36]	33	HPV 18 5285-5315<br>
DXA-1	GTTCTACACGGGTTTGCAGCACGATCAACAACG [SEQ ID NO:37]	33	HPV33 1175-1207<br>
PRA-1	CGCTGCTTGTGGTGGTCGGTTATCGTTGTCTG	32	HPV33<br>
3o<br><br>
Table 1: Capture Sequence Probes For HPV<br>
^^^^"™" ■'■""    ■ ■■■"■		ii	ii I.I     i—■**-	II<br>
Probe	Sequence	——<br>
Size<br>
(bp)	HPV Type and Sequence Location<br>
	[SEQ ID NO:38]		3389-3420<br>
TT-1	GACGTAGTGTCGCCTCACATTTACAACAGGAC. [SEQ ID NO:39]	32	HP35 732-763<br>
TT-7	CTCGCTTGGTGGGGTTGTAGGGGAGCTCGG [SEQ ID NO:40]	30	HPV35 3432-3461<br>
NH-1	GCTGTAGTTGTCGCAGAGTATCCCGTGAGG [SEQ ID N0:4I]	30	HPV39 833-862<br>
FT-7	GTGAGCCTGTGTTATATGTAGTGCCCGAATCCC [SEQ ID NO:42]	33	HP39 5358-5390<br>
SHA-4	CCACCTCCTGCGTCCACTACACCTAGCACTA [SEQ ID NO:43]	31	HPV45 5295-5325<br>
SHA-1	TGCGTGCGTGTATGTATGAATGTGCCTTGTGG [SEQ ID NO:44]	32	HPV45 7243-7274<br>
OCA-1	AATTAGCGCATTGCCCCGTCCAACGTCCCG [SEQ ID NO:45]	30	HPV51 482-511<br>
TAA-7	CGCCGTGCACGTGTAGCCACCATATTTAATCAC [SEQ ID NO.-46]	33	HPV51 4124-4156<br>
LFA-1	CGAATTGTGTGAGGTGCTGGAAGAATCGGTGC [SEQ ID NO:47]	32	HPV52 140-171<br>
PTA-7	GATCGTTCACAACTTTTACCTGCACCGGATCC [SEQ ID NO:48]	32	HPV52 5066-5097<br>
ZOA-1	CTAGGTTCTCTAGATGTTTGTCTCCAGCACCCC [SEQ ID NO:49]	33	HPV56 521-553<br>
NAA-4	CTGTCGGTATTGTCTGTGTCGCTGATGTGTG [SEQ ID NO: 50]	31	HPV56 3543-3573<br>
LA-1	GATACACACACATTTGCAGCCCGGTCCACACA [SEQ ID NO:51]	32	HPV58 1180-1211<br>
LA-7	GGTGGCAAAGGACGTATGTGAGTGCAGAGGAC [SEQ ID NO:52]	32	HPV58 5376-5407<br>
ZV-4	GCGTTGCGGAGGGGTATGATAGTTGCTCAGAAG [SEQ ID NO:53]	33	HPV59 3385-3417<br>
ZV-10	GTCTAGGCGTGTAGGAGGAAACAAGATGGGG [SEQ ID NO:54]	31	HPV59 7543-7573<br>
PNA-1	CTGAACACAGCAGTTCTCTATACCAATGGCGCTA TTTC [SEQ ID NO:55]	38	HPV68 81-118<br>
PNA-4	TTGGTTGCCCCTGAGCAGTCGGACCCTATGGATA fSEQ ID NO.-56]	34	HPV68 5211-5244<br>
CDA-4	GCGCCGCATTGCTGCACCTCGTTTATATAGCAGG GCATTTTC [SEQ ID NO:57]	42	HPV82 4827-4868<br>
CDA-11	CCTGGCGCATGTCATACACACCACATTACTC [SEQ ID NO:58]	31	HPV82 7596-7626<br>
CTA-1	CACGAAGTGTCAGTGCACAGTATGCCTTGC [SEQ ID NO:59]	30	HPV73 6006-6045<br>
CTA-16	GCCATGTACTTCACAAACTGTTAATACTGGTGAT TGTCCC [SEQ ID NO:60]	40	HPV73 720-749<br>
DLA-14	CCTACACAGTACAAGTGGAGGCCATCACCCG [SEQ ID NO:61]	31	HPV26 3566-3596<br>
RBA-16    1 GTCTGACACATACTGTTGTAACCCATAGTTAAAC	39   1           HPV26<br>
31<br><br><br><br>
Sequences in parentheses are "tail" sequences not directed at HSV.<br>
Table 2: Blocker Probes For HPV<br>
PROBE	SEQUENCE	SIZE	CSP to which binds<br>
PA-6	ACTCCAACGACGCAGA [SEQ ID NO:65]	16	PA-4<br>
ZZ-2	TTTTGTGGTTCTGTGTG [SEQ ID NO:66]	17	ZZ-1<br>
ZZ-3	TTATGTGGTTGCGC [SEQ ID NO:67]	14	ZZ-1<br>
DXA-2	CGTTGTTGATCGTGC [SEQ ID N0.68]	15	DXA-1<br>
DXA-3	TGCAAACCCGTGTAG [SEQ ID NO:69]	15	DXA-1<br>
PRA-2	CAGACAACGATAACCG [SEQ ID NO:70]	16	PRA-1<br>
PRA-3	ACCACCACAAGCAGC [SEQ ID NO:71]	15	PRA-1<br>
TT-2	GTCCTGTTGTAAATGTG [SEQ ID NO:72]	17	TT-I<br>
TT-3	AGGCGACACTACGTC [SEQ ID NO:73]	15	TT-1<br>
TT-8	CGAGCTCCCCTACAA [SEQ ID NO:74]	15	TT-7<br>
TT-9	CCCCACCAAGCGA [SEQ ID NO:75]	13	TT-7<br>
NH-2	CCTCACGGGATACTC [SEQ ID NO:76]	15	NH-1<br>
NH-3	TGCGACAACTACAGC [SEQ ID NO:77]	15	NH-1<br>
FT-8	GGATTCGGGCACTA [SEQ ID NO:78]	14	FT-7<br>
FT-9	CATATAACACAGGCTCAC [SEQ ID NO.-79]	18	FT-7<br>
SHA-5	TAGTGCTAGGTGTAGTGG [SEQ ID NO:80]	18	SHA-4<br>
SHA-6	ACGCAGGAGGTGG [SEQ ID NO:81]	13	SHA-4<br>
SHA-2	TACATACACGCACGCA [SEQ ID NO:82]	16	SHA-1<br>
SHA-3	CCACAAGGCACATTCA [SEQ ID NO:83]	16	SHA-1<br>
TAA-8	GCTACACGTGCACGGCG [SEQ ID NO:84]	17	TAA-7<br>
TAA-9	GTGATTAAATATGGTGG [SEQ ID NO:85]	17	TAA-7<br>
OCA-2	GGGACGTTGGACG [SEQ ID NO:86]	13	OCA-1<br>
OCA-3	GGCAATGCGCTAAT [SEQ ID NO: 87]	14	OCA-1<br>
LFA-2	CACCGATTCTTCCAG [SEQ ID NO:88]	15	LFA-1<br>
LFA-3	CACCTCACACAATTCG [SEQ ID NO:89]	16	LFA-1<br>
PTA-8	GGATCCGGTGCAG [SEQ ID NO:90]	13	PTA-7<br>
PTA-9	GTAAAAGTTGTGAACGATC [SEQ ID NO:91]	19	PTA-7<br>
ZOA-2	GGTGCTGGAGACAA [SEQ ID NO:92]	14	ZOA-1<br>
ZOA-3	ACATCTAGAGAACCTAG [SEQ ID NO:93]	17	ZOA-I<br>
NAA-5	CACACATCAGCGACA [SEQ ID NO:94]	15	NAA-4<br>
NAA-6	CAGACAATACCGACAG [SEQ ID NO:95]	16	NAA-4<br>
LA-2	TGTGTGGACCGGG [SEQ ID NO:96]	13	LA-1<br>
LA-3	CTGCAAATGTGTGTGTAT [SEQ ID NO:97]	18	LA-1<br>
LA-8	GTCCTCTGCACTCACAT [SEQ ID NO:981	17	LA-7<br>
LA-9	ACGTCCTTTGCCAC [SEQ ID NO:99]	14	LA-7<br>
ZV-5	TTCTGAGCAACTATCATAC [SEQ ID NO: 100]	19	ZV-4<br>
32<br><br>
Table 2: Blocker Probes For HPV<br>
PROBE	SEQUENCE	SIZE 12	CSP to which binds ZV-4<br>
ZV-6	CCCTCCGCAACG [SEQ ID NO: 101]	<br><br><br>
ZV-11	CCCCATCTTGTTTCC [SEQ ID NO: 102]	15	ZV-10<br>
ZV-12	TCCTACACGCCTAGAC [SEQ ID NO: 103]	16	ZV-10<br>
PNA-2	TATAGAGAACTGCTGTGTTC [SEQ ID NO: 104]	20	PNA-1<br>
PNA-3	GAAATAGCGCCATTG [SEQ ID NO: 105]	15	PNA-1<br>
PNA-5	CTCAGGGGCAACCA [SEQ ID NO: 106]	14	PNA-4<br>
PNA-6	ATCCATAGGGTCCGAC [SEQ ID NO: 107]	16	PNA-4<br>
CDA-5	CAATGCGGCGC [SEQ ID NO:108]	11	CDA-4<br>
CDA-6	TATAAACGAGGTGCAG [SEQ ID NO: 109]	16	CDA-4<br>
CDA-7	GAAAATGCCCTGCTA [SEQ ID NG':110]	15	CDA-4<br>
CDA-12	ACATGCGCCAGG [SEQ ID NO:111]	12	CDA-11<br>
CDA-13	GAGTAATGTGGTGTGTATG [SEQ ID NO:112]	19	CD A-11<br>
CTA-2	GCAAGGCATACTGTG [SEQ ID NO: 113]	15	CTA-1<br>
CTA-3	CACTGACACTTCGTG [SEQ ID NO: 114]	15	CTA-1<br>
CTA-17	GGACAATCACCAGTATTA [SEQ ID NO: 115]	18	CTA-16<br>
CTA-18	AGTTTGTGAAGTACATGG [SEQ ID NO: 116]	18	CTA-16<br>
DLA-15	CGGGTGATGGCC [SEQ ID NO: 117]	12	DLA-14<br>
DLA-16	CCACTTGTACTGTGTAGG [SEQ ID NO:l 18]	18	DLA-I4<br>
RBA-17	CTGTGTTTAACTATGGGT [SEQ ID NO:l 19]	18	RBA-16<br>
RBA-18	TACAACAGTATGTGTCAGAC [SEQ ID NO: 120]	20	RBA-16<br>
BNA-2	AAGACAGGGAGACAGC [SEQ ID NO: 121]	16	BNA-1<br>
BNA-3	CTTATAAACAATACACAGG [SEQ ID NO: 122]	19	BNA-1<br>
DLA-33	ATGCACTATAGTAACACACC [SEQ ID NO: 123]	20	DLA-32<br>
DLA-34	ACTCCATTTTAGTGCTGTA [SEQ ID NO: 124]	19	DLA-32<br>
EXAMPLE 2<br>
EFFECT OF THE DISTANCE BETWEEN THE CSP AND THE SSP TARGET SITES ON CAPTURE EFFICIENCY<br>
The effect of the distance between capture sequence probe (CSP) and<br>
signal sequence probe (SSP) hybridization sites on a HSV-1 target nucleic acid on<br>
capture efficiency was evaluated. CSPs that hybridize to HSV-1 nucleic acid<br>
sequences which are located 0.2kb, 3kb, 18kb, 36kb and 46kb from the site of SSP<br>
hybridization were tested. The general TSHC method described in Example 1 was<br>
employed. The capture efficiencies were 100%, 50%, 30%, 19% and 7%, respectively<br>
(Table 3). A steady decline in relative capture efficiencies was observed as the<br>
distance increased from 0.2 Kb to 46 Kb.<br>
 33<br><br>
Table 3: Effect of Distance between Target Sites<br>
	on Capture Efficiency	<br>
CSP	SSP	Distance Between<br>
Target Site<br>
0.2 Kb	Relative Capture<br>
Efficiency<br>
100%<br>
BRH19	H19	<br><br><br>
F1SR	H19	3 Kb	50%<br>
F6R	RH5B	18 Kb	30%<br>
F15R	RJH5B	36 Kb	19%<br>
F6R	H19	46 Kb	7%<br>
EXAMPLE 3<br>
CAPTURE EFFICIENCY OF VARIOUS CSPS AND SSPs IN TSHC DETECTION OF HSV-1<br>
The capture efficiency of capture sequence probes (CSPs) for each of the four HSV-1 specific signal sequence probes (SSPs), H19, RH5B, RH3 and R10, in the detection of HSV-1 by TSHC were evaluated. The criteria used for designing the capture sequence probes were: 1) the CSP hybridization site is within 1 kb either 5' or 3' of the SSP hybridization site on the HSV-1 nucleic acid sequence, preferably within 0.5 kb; and 2) the CSPs contain sequences that are unique to HSV-1, with no stretches of sequence homology to HSV-2 greater than 10 bases. The CSPs were designed to target the 5" and 3' regions adjacent to the SSP hybridization site, preferably with a 5' CSP and a 3' CSP for each SSP. The commercially available OMIGA software (Oxford Molecular Group; Campbell, CA) was instrumental in the identification of such sites. The melting temperature (Tm) of the CSPs was designed to be between 70°C to 85°C. The general TSHC method described in Example 1 was employed. Eleven CSPs (which bind to 6 different sites) for HI 9, six CSPs (which bind to three unique sites) for RH5B, six CSPs (which bind to six unique sites) for RH3, and two CSPs for R10 were tested. As shown in Table 4, efficient capture sequence probes were found for signal sequence probes H19, RH5B and R10.<br>
34<br><br><br>
EXAMPLE 5<br>
EFFECT OF COMBINING PROBES IN TSHC DETECTION OF HSV The effect of combining HSV-1 specific signal sequence probe and capture sequence probe sets on HSV-1 detection was assessed. TSHC detection of HSV-1 and HSV-2 cross-reactivity was performed separately with two different sets of RNA signal sequence probe/biotinylated capture sequence probe combinations (Set #1: HI9 plus HZ-1; and Set #2: RH5b plus the TS-1 and TS-2). TSHC was also performed with both RNA signal sequence probe/biotinylated capture sequence probe sets combined to assess the effect of combining the two probe sets on sensitivity and cross-reactivity. The general TSHC method described in Example 1 was employed. The results shown in Table 7 clearly demonstrate an additive effect of combining the two probe sets for HSV-1 detection with no apparent increase in HSV-2 cross-reactivity.<br>
36<br><br><br>
Table 7: Sensitivity is Improved by Combining HSV-1 Specific CSPs and SSPs<br>
EXAMPLE 6<br>
DETECTION OF HPV TYPES IN MULTIPLEX FORMAT Rapid, sensitive and specific detection of 13 sequences representing different types of Human Papillomavirus (HPV) in a single biological sample was performed in a multiplex format. This assay can analyze up to 96 samples in a 96 well micrpplate format within a 5 hour period of time. At least 500 copies of one type of virus may be detected within 5 hours after beginning the assay.<br>
Protein G-paramagnetic beads (Dynal Corp.; Brown Deer, Wl) and Luminex® 100™ carboxylated beads (Luminex-Corp., Austin, TX) were used as solid supports. HPV RNA was prepared by in vitro transcription of HPV plasmids as commonly understood in the art and described by Ausubel, et al. [Current Protocols in Molecular Biology, New York, Wiley Publishing, 1993, incorporated by reference]. The novel assay has three steps including 1) target enrichment; 2) target amplification; and 3) target detection.<br>
37<br><br>
Target enrichment:<br>
A sample was prepared for amplification by separating unwanted or non-specific DNA and contaminants from preselected specific targets. This step removes unwanted DNA, the presence of which substantially decreases the sensitivity of target amplification. Target enrichment was performed by capturing sequence specific RNA:DNA hybrids onto paramagnetic beads. The beads were initially modified with antibodies specific for RNA:DNA hybrids or HYBRID CAPTURE antibodies (Digene, Corp.). More specifically, the HC-Ab paramagnetic beads were prepared by mixing 1 mL of Protein G Magnetic Beads (2.7 X 109 beads/mL; Dynal, Corp.) with 375 fig of HC-Ab and incubating at room temperature in PBS for 40 minutes forming bead-antibody (bead-Ab) complexes. The bead-Ab complexes were accumulated at the bottom of the well by placing the plate upon a magnetic grid (Dynal, Corp.) and washed once with 0.2 M Triethanolamine, pH 8.2. Antibodies were cross-linked to the Protein G in a reagent 6f 20 mM DMP/0.2M Triethanolamine, pH 8.2 for 30 minutes. The bead-Ab complexes were washed 3X with 1 mL PBS-0.05% Tween20. Beads were then resuspended inl mL of PBS-Tween and stored at 4°C.<br>
The sample was serial diluted with various HPV plasmids (10 kb) prepared in a solution composed of 100 jig/ml Herring Sperm DNA, 1 M Guanidine -HC1, 10 mM Tris - HC1 (pH 8.0), 10 mM EDTA, 0.05% Sodium Azide in deionized water. Each of the diluted samples (50 ul) was mixed in a 96-well microplate containing 25 p.1 of 1.75M NaOH per well and incubated at 50°C for 15 minutes to denature the sample. A cocktail of 13 types of HPV RNA selected from Table 1(15 ng each per assay) was added to the denatured samples in neutralizing solution: 25  µL of 0.125 M sodium citrate, 0.125 M sodium phosphate, 0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5 mM EDTA, 0.05% sodium azide, and 0.4% Tween-20 (pH 3.6-4.1). The HYBRID CAPTURE antibody (HC-Ab) -conjugated paramagnetic beads (5.4 x 106 beads per assay) were then immediately added to the neutralized sample. The HPV RNA and paramagnetic bead-antibody conjugates were hybridized to the sample target sequences for 30 min at 65°C with<br>
38<br><br>
shaking at 1100 rpm. The magnetic bead-target complexes were accumulated at the bottom of the well by placing the plate upon a magnetic grid (Dynal, Corp.)- The supernatant was aspirated by pipette and the beads were washed three times using a wash buffer (40 mM Tris pH 8.2, 100 mM sodium chloride, 0.05% sodium azide and 0.05% Tween-20), thereby removing any contaminants.<br>
Target Amplification:<br>
Isothermal amplification of the sample target DNA was performed using random primers and a DNA polymerase having strand displacement activity. The DNA targets were amplified using phi29 DNA polymerase primed with random pentamers. The pentamer primers were synthesized with two phosphorothioate bonds at the 3'-end to prevent degradation by the nuclease. Other modifications may include, but are not limited to, 2'O-methyl groups which are incorporated into the structure of the primers. The target DNA was amplified for 2 hours. Paramagnetic beads with captured target nucleic acid obtained in the target enrichment step were resuspended in 20 ul reaction composed of 50 mM Tris-HCl, p*H 7.5, 10 mM MgCl2, 10 mM (NH4)2 S04,4 mM DTT, 200 u.g/ml acetylated BSA, 400 u,M each dNTP, 125   µ M random pentamers and 1U of phi29 DNA polymerase (New England Biolabs, Inc) and incubated for 2 hours at 30°C.<br>
Primers which may be useful in the amplification step are listed below.<br>
Probe	HPV16 5'-\FAM\ TCAGGACCCACAGGAGCGACCCAG-3'\BHQ<br>
(SEQ ID NO: 125) Forward      5'-GCACCAAAAGAGAACTGCAATGT-3' (SEQ ID NO: 126) primer Reverse       5'-CATATACCTCACGTCGCAGTAACT-3' (SEQ ID NO: 127)<br>
primer<br>
*	duel-labeled probe and primers (Integrated DNA Technologies, Inc.; Salt Lake City, UT)<br>
*	FAM-fluorescent reporter; BHQ-black hole quencher<br>
Target Detection:<br>
Amplified sequences were detected using a liquid-based bead array system that integrates optics, fluidics, and signal processing enabling multiplex capabilities (Luminex technology). The Luminex technology uses a highly sensitive<br>
39<br><br>
assay system based on fluorescent microbeads and reporter molecules that can detect multiple targets in the same assay well (see, U.S. Patent Nos. 5,981,180; 6,524,793; 5,736,330; 6,449,562; 6,592,822; 6,632,526; 6,514,295; 6,599,331; 6,046,807?; and 6,366,354). The Luminex detection instrument uses two lasers: one for the detection of the fluorescent bead itself and the other for the fluorescent reporter. The bi-colored fluorescent beads enabled multiplex detection of up to about 100 different targets. Luminex technology was applied to HPV typing in this example. This method increased the sensitivity and robustness of the platform.<br>
Two oligonucleotide sequences complementary to the late (L)- and early (E)- regions [MH Einstein and GN Goldberg, Cancer Invest. 20: 1080-1085, 2002, incorporated by reference] on the specific HPV type were conjugated to the commercially available differently colored Luminex carboxylated beads using the manufacturer's protocol. Sequences of capture oligonucleotides were presented in Table 1. Two oligonucleotides of one specific HPV type were attached to the same bead set, thereby eliminating false negative results caused by the deletion of the HPV L-region which may occur in cancer cells. An amplicon purification step was not required before the detection step since hybridization had been performed in the same tube which held the paramagnetic beads introduced during the first target enrichment step.<br>
The detection step of the assay began with amplicon denaturation in 75  µ1 of 438 mM NaOH at 70°C for 15 minutes. A cocktail of 13 different types of HPV RNA (15 ng each per assay) was added to the denatured sample in 25 ulof 0.125 M sodium citrate, 0.125 M sodium phosphate, 0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1). Then, the 13 types of oligonucleotides are each conjugated to carboxylated Luminex polystyrene beads (5 x 103 beads per assay) and immediately added to the neutralized sample. The RNA and bead-oligonucleotide conjugates were hybridized to the target for 30 minutes at 65°C while shaking at 1100 rpm. The samples were then filtered through the 96-well filter plate (Whatman) and the beads were resuspended in 100  µl of lx phosphate-buffered saline (PBS), 0.05% Tween-20,<br>
40<br><br>
10 % goat serum and 10 ng of mouse monoclonal DNA:RNA hybrid-specific antibodies labeled with phycoerythrin (PE-HC-Ab). The samples were incubated for 30 minutes at room temperature (18-25°C) while shaking at 1100 rpm. The excess antibody was removed by vacuum filtration and the beads with target complexes were resuspended in 100µL PBS with 0.05% Tween-20. Samples were then analyzed by Luminex flowcytometer after adjusting the gain of the photomultiplier tube to 700 volts.<br>
The improved sensitivity of this step resulted from long RNA signal sequence probes (6500 bp, approximate length) that permit binding of multiple phycoerythrin antibodies (PE-Ab), which thereby resulted in about 500 fold amplification of signal compared to other detection techniques, such as those that use fluorophores conjugated to short hybridization probes. The RNA signal sequence probes may have a length ranging from about 500 bases to about 10 kilobases, or up to just less than 100% of the target nucleic acid length.<br>
The results shown in Table 8 demonstrate the sensitivity of the detection of HPV 16 in the presence of all 13 HPV Type bead sets. The sensitivity of detection for plasmids was greater than 500 copies per assay as indicated by the robust signal to noise ratio. The initial sample volume was 50  µl, but may be increased to 500 ul or even the entire sample (for use in tube-based assays; not shown), which will increase the assay sensitivity to an even greater degree. The results also demonstrated the specificity for HPV 16 detection. Essentially no signal was detected on bead sets specific for HPV targets other men HPV 16. The signals of Table 8 (average, n=4 replicates, % CV between 3 and 31 %) are the median fluorescent intensities (MFI) for 100 beads (events) per sample counted by the flowcytometer. The background or noise value is the MFI of samples containing no target.<br>
Table 9 demonstrates the sensitivity and specificity of detection of all 13 HPV types. The target HPV types are listed along the x-axis and the nucleic acid probes are listed along the y-axis. The same HPV target and HPV probe for each type is highlighted diagonally across the table (positive result). These boxes demonstrate the high level of specificity of the probe for its HPV target. This experiment confirms<br>
41<br><br>
that the probes are specific for corresponding HPV type targets. The results in Table 9 show that, for example, HPV 16 target binds only the corresponding HPV 16 probe (positive result) and does not bind the probes for other HPV types (negative result).<br>
Table 8: Sensitivity Of The Detection Of HPV 16<br>
Target input (copies/ assay)	Average Signal /Noise Ratio for each bead set specific for the individual HPV type<br><br>
	16                  18    31     33    35    39    45	51    52    56    58    59    68<br>
5 x I02	194	1	1	1	1	2	1	1	1	1	1	1	1<br>
5 x l03	419	1	1	1	1	2	1	I	1	1	1	2	1<br>
5 x l04	553	2	1	1	1	3	1	2	1	1	1	2	1<br>
Table 9: Specificity Of HPV Types<br><br>
EXAMPLE 7 DETECTION OF HPV 16, HPV58 AND HPV33 Cervical samples infected with multiple types of HPV were modeled by mixing three plasmids each inserted with a different type of HPV sequence (10 kb) in the sample buffer indicated in Example 6. This example demonstrates the capability<br>
42<br><br>
of the assay to detect multiple infections, when more than one HPV type is present in the sample.<br>
The assay of multiple target samples was performed as illustrated in Example 21. Samples consisted of IxlO2 copies of HPV 16, lxlO4 copies of HPV 58 and lx 106 copies of HPV 33. Results in Figure 7 demonstrated that HPV 16 target in low abundance was detected even in the highly abundant presence of other HPV types. Figure 7 demonstrates that HPV 16 has a low signal to noise ratio, where the signals (average, n=4 replicates, % CV (coefficientof variation) between 3% and 31%) are the median fluorescent intensities (MFI) for 100 beads (events) per sample counted by the flow-cytometer. The noise value is the MFI of samples containing no target plasmids.<br>
EXAMPLE 8 DETECTION OF HPV IN CLINICAL SAMPLES<br>
HPV was detected in cervical samples, where the cervical samples were collected in specimen transport media (STM; Digene Corp., Gaithersburg, MD) and initially screened for the high-risk HPV types using the commercially available HC2 High-Risk HPV DNA Test™ assay (Digene Corp.). The HC2 assay results were compared with those of the HPV typing assay described in this invention.<br>
Cervical samples were collectedyusing a brush and kept in a storage medium (1 M Guanidine - HC1, 10 mM Tris - HC1 (pH 8.0), 10 mM EDTA, and 0.05% Sodium Azide in deionized water) until use. Aliquots (50 pi) of cervical samples were analyzed by HC2 (Digene Corp) according to the manufacturer's recommendations or by the method described in Example 6. In order to compare the sensitivity of the two assays, the positive sample was serially diluted in a cervical sample without HPV infection.<br>
43<br><br>
Table 10: Comparison Of Detection Sensitivity Of Clinical Samples<br><br>
Positive sample dilutions	HC2, Signal/CutOff	Typing Assay, S/N<br>
No dilution	187.2	585.1<br>
1:10	21.2	310.4<br>
1:100	2.3	199.2<br>
1:1000	
1:10,000	
The sensitivity of HC2 on clinical samples is reported in Table 10 and shows a Signal/Cutoff of 2.3 (100 fold diluted sample) which corresponds approximately to 10,000 HPV targets per assay; Accordingly, when compared to the sensitivity of the HPV typing assay of the invention, the invention is at least 100 fold greater and equal to 100 copies per assay with a signal to noise ratio (S/N) of about 43.<br>
EXAMPLE 9<br>
DETECTION OF CIRCULAR, LINEAR AND INTEGRATED FORMS OF HPV DNA<br>
The assay of the invention may detect DNA sequences that are linear, as well as, circular. Thus, the amplification that occurs in the inventive assay does not necessarily occur by the "rolling circle" phenomenon. Therefore, the targets of the inventive assay need not be circular. They may be linear, either because the circular pathogenic DNA is nicked by denaturation, or because the pathogenic (viral) DNA targets is integrated into the linear host (human) genome, or because the pathogenic DNA is a linear genome.<br>
Samples including serial dilutions of HPV 16 plasmid that were either linear or circular or serial dilutions of purified DNA from CaSki cells were diluted in TE buffer. Linear plasmids were created by restriction endonuclease digestion of the circular plasmids using standard techniques. CaSki cell DNA is human genomic DNA that contains 500 copies of integrated HPV 16 viral genomes per human genome. The concentration of HPV 16 copies per assay for the CaSki cell DNA was calculated from<br>
44<br><br>
the sizes and ratios of the two genomes, human and HPV, and the optical density measurements of the purified CaSki DNA.<br>
Targets were heat denatured at 95°C for 3 minutes in the presence of random primers immediately before amplification in 50 mM Tris-HCl (pH 8.0), 0.5 mM EDTA and 50 µM random hexamer primer (phosphorothioated at ultimate 3' bonds). Single-stranded targets were then amplified for 120 minutes at 30°C in 20  µl reactions composed of 50 mM Tris-HCl (pH 7.5), 10 mM MgCl2, 10 mM (NH4)2S04, 4 mM DTT, 0.2 mg/ml acetylated BSA, 400 uM each dNTPs, 0.13 mM random hexamers and 10 U of phi29 DNA polymerase (Epicentre, Inc).<br>
The amplified targets were detected as described in Example 6 with the following exception. The RNA:DNA hybrid complexes were tagged first with a mouse monoclonal anti-RNA:DNA primary antibody (10 ug/assay) and then with a goat-anti-mouse IgG secondary antibody conjugated with phycoerythrin (1.6 Ug/assay).<br>
The results in Table 11 demonstrate that the assay of the invention detected circular, linear and integrated DNA with robust Signal/Noise ratios, thereby demonstrating that the effectiveness of the assay was not dependent on the "rolling-circle" mechanism. However, the "rolling circle" mechanism could have accelerated the reaction, because the circular target was amplified more efficiently than the linear target.<br>
Table 11: Detection Of Circular Linear And Integrated Targets With The Proposed Method<br>
1 Target (copies/assay)		Signal/Noise	<br><br>
	Linear DNA	Circular DNA	Integrated DNA<br>
1X10Z	71	123	21<br>
1X104	96	130	83<br>
1X102	89	142	77<br>
The signals were averages (n=2) of the median fluorescent intensities (MFI) for 100 beads (events) per sample counted by the flowcytometer. The noise value is the average MFI of samples containing no target plasrnids.<br>
45<br><br>
EXAMPLE 10 ISOTHERMAL AMPLIFICATION CATALYZED BY PHI29 DNA POLYMERASE USING RANDOM OLIGONUCLEOTIDES Another embodiment of the target amplification step utilizes modified oligonucleotides optimized in the length of the random primers. The random pentamers, 5'-NpNpNpsNpsN-3' were made by IDT, Inc. For the data generated in this Example, HPV 16 plasmid was amplified in a 20uL total reaction volume with isothermal amplification for 2 hours at 30°C. The DNA target was heat denatured for 3 minutes at 95°C in 10 ul of 50 mM Tris-HCl, pH 8.0, 0.5 mM EDTA, 50 yM of primers. Isothermal amplification was performed in 20 (j.1 of 50 mM Tris-HCl, pH 7.5, 10 mM MgCfe, 10 mM (NH^ S04) 4 mM DTT, 200 ug/ml acetylated BSA, 400 uM each dNTP, 125 uM random pentamers and 1 U of phi29 DNA polymerase (New England Biolabs, Inc) for 2 hours at 30°C. Following the amplification, half the volumes (10|J.L) of the amplicons were detected in the Luminex fiow-cytometer system using the "Primary/Secondary" antibody detection protocol as described in Example 11.<br>
Table 12: Random DNA Pentamers are the Optimal Primer Length. All primers Have Two Phosphorothioate Modifications at the 3'End.<br><br>
J Copies/assay	5-mer	6-mer	7-mer	8-mer	9-mer 9	10-mer<br>
0	10	9	7	13	<br>
	14<br>
1x10^	5873	990	16	10	17	12<br>
lxl 0*	9864	5766	1469	176	36	26<br>
1x10°	12257	7567	7187	5444	2421	1621<br>
Values = Luminex. Average MFI<br>
46<br><br>
Table 13: Comparison of Different Random Primer lengths and Modifications<br><br>
* Values = Luminex Average MFI<br>
Tables 12 and 13 show that random DNA hexamer primers were optimal with one phosphorothioate modification and primers that were random DNA pentamers were optimal with two phosphorothioate modifications.<br>
EXAMPLE 11 DIFFERENT METHODS OF DETECTING AMPLIFIED TARGET<br>
Three different protocols of detecting amplified targets were compared. One means utilized biotinylated probes which are oligodeoxynucleotides labeled with biotin at the 5' end. These oligonucleotides were prepared by the chemical synthesis by Integrated DNA Technology, Inc.; Salt Lake City, UT. The two other approaches used labeled primary HC-Ab (Digene Corp.) and sequentially used primary HC-Ab and labeled secondary antibody (Digene Corp.), representing two embodiments of the invention. The combination of flow-cytometry detection of antibody-labeled or colored beads, and signal amplification resulted in an increased signal, and hence the sensitivity of the assay increased more then 10 fold. Signal amplification would normally increase the background of the assay, as would the presence of a large complex on the beads, thus decreasing or eliminating sensitivity. However, the described inventive method provides for an increase in signal and sensitivity. The large complex on the beads could interfere with the detection mode that relies on a certain size of the beads in the Luminex flow-cytometer system. Both of these problems were overcome by the inventive method.<br>
47<br><br>
A.	In one method, a biotinylated probe complementary to the target<br>
nucleic acid was applied to streptavidin-conjugated to phycoethithrin (SA-PE; Pierce).<br>
Amplified HPV 16 plasmid was denatured in 75 µ1 of 438 mM NaOH at 70°C for 15<br>
minutes. Two nanograms each of two 5'-biotinylated probes (5'- Biotin-<br>
TATTTTATATGACACAATGT-3' (SEQ ID NO: 128); 5'- Biotin-<br>
GGTTTGTGCTAACAATAAATGTATCCATAG-3' (SEQ ID NO: 129)) were added<br>
to the denatured target nucleic acid sample in 25 u.1 of 0.125 M sodium citrate, 0.125<br>
M sodium phosphate, 0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3%<br>
polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1).<br>
Approximately 1000 polystyrene beads were conjugated to oligonucleotides specific<br>
for HPV 16 L- and E- regions (see Table 1) and immediately added to the neutralized<br>
sample. Hybridization was performed for 30 minutes at 50°C while shaking at 1100<br>
rpm. SA-PE (100 ng) in 100 u.1 of 438 mM NaOH, 0.125 M sodium citrate, 0.125 M<br>
sodium phosphate, 0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3%<br>
polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 was added to<br>
each reaction and incubated at room temperature for 5 minutes. The samples were<br>
then transferred to a filter plate where the unbound materials were removed by vacuum<br>
filtration. The beads were resuspended in 100 u.1 of PBS-0.05% Tween-20 and<br>
analyzed by Luminex flowcytometer after adjusting the gain of the photomultiplier<br>
tube to 700 volts.<br>
B.	In the second method, amplified HPV 16 plasmid was denatured<br>
in 75  µ1 of 438 mM NaOH at 70°C for 15 minutes. A cocktail of 13 different types of<br>
HPV RNA (each 15 ng per assay, obtained as described in Example 6) was added to<br>
the denatured sample in 25 u.1 of 0.125 M sodium citrate, 0.125 M sodium phosphate,<br>
0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5<br>
mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1). Thirteen sets of<br>
oligonucleotide-conjugated polystyrene beads (5 x 103 beads per assay) were<br>
immediately added to the neutralized sample. Each bead set had two oligonucleotide<br>
sequences complementary to the late (L)- and early (E)- regions on the specific HPV<br>
48<br><br>
type. Sequences of capture oligonucleotides were presented in Table 1. The RNA and bead-oligonucleotide conjugates were hybridized to the target nucleic acid for 30 minutes at 65°C with shaking at 1100 rpm. The samples were then filtered in a 96-well filter plate (Whatman) and 1 u.g of primary HC-Ab in 100 u.1 of PBS- 0.05% Tween-20 was added. After shaking at room temperature for 5 minutes, reactions were filtered and 8  µg of PE-labeled secondary (goat-antimouse) antibody in 100  µl of PBS, 0.05% Tween-20, and 10 % goat serum was added. Incubation continued for 20 minutes while shaking at room temperature. Tlie excess antibody was removed by vacuum filtration and the beads with target complexes were resuspended in 100  µl PBS with 0.05% Tween-20. Samples were then analyzed by Luminex flow-cytometer after adjusting the gain of the photomultiplier tube to 700 volts.<br>
C.        In the third method, amplified HPV 16 plasmid was denatured in 75 u.1 of 438 mM NaOH at 70°C for 15 minutes. A cocktail of 13 different types of HPV RNA (each 15 ng per assay, obtained as described in Example 6) was added to the denatured sample in 25 |ul of 0.125 M sodium citrate, 0.125 M sodium phosphate, 0.6 M triethanolamine, 0.6 BES, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1). Thirteen sets of oligonucleotide-conjugated polystyrene beads (5 x 103 beads per assay) were immediately added to the neutralized sample. Each bead set had two oligonucleotide sequences complementary to the late (L)- and early (E)- regions of the specific HPV type. Sequences of capture oligonucleotides were presented in Table 1. The RNA and bead-oligo conjugates were hybridized to the target for 30 minutes at 65 C with shaking at 1100 rpm. The samples were then filtered through the 96-well filter plate (Whatman) and the beads were resuspended in 100 u.1 of PBS, 0.05% Tween-20, 10 % goat serum and 10 ng of mouse monoclonal RNA:DNA specific antibodies labeled with phycoerythrin (PE-HC-Ab). The samples were then incubated for 30 minutes at room temperature (18°-25°C) while shaking at 1100 rpm. Excess antibody was removed by vacuum filtration and the beads with target complexes were resuspended<br>
49<br><br>
in 100  µI PBS with 0.05% Tween-20. Samples were then analyzed by Luminex flow-cytometer after adjusting the gain of the photomultiplier tube to 700 volts.<br>
Isothermal amplification of HPV 16 target was performed as described in the Example6. The amplified target was detected using one of three protocols as described in the above protocols A, B or C. The results are presented in Table 14.0f the three described protocols, detection with the biotinylated probes (Protocol A) involved the least amount of time to perform the method. However, this detection method also produced the lowest signal intensity. The greatest signal intensity and S/N ratio was achieved using the labeled primary antibody detection scheme (Table 14), which implemented HC-Ab signal amplification (Protocols B and C). Protocol C utilizes one fewer step than in protocol B and would be preferable.<br>
Table 14: Comparison Of HPV 16 Detection After Isothermal Amplification<br><br>
Target input (copies/assay)	Average of 3 median fluorescent intensities (MFI)<br><br>
	Protocol A	Protocol B	Protocol C<br>
5x102	174	4165	2506<br>
5xl03	517	7466	4715<br>
5x104	776	9310	7001<br>
EXAMPLE 12 HPV TYPING IN A 96-WELL NUCLEOLINK PLATE In an alternative embodiment, a microtiter plate may be used instead of beads as a solid support for sequence detection. Oligonucleotides specific for individual HPV types were conjugated to the bottom of the wells. The target nucleic acid hybridized to the capture oligonucleotides and then detected by chemiluminescence.<br>
In preparation of the NucleoLink™ multi-well plate (NalgeNunc International; New York), two oligonucleotide sequences complementary to the late (L)- and early (E)- regions on the specific HPV type were conjugated to the Nucleolink plate using the protocol described below. Sequences of capture<br>
50<br><br>
oligonucleotides are presented in Table 1. By placing two oligonucleotides in each well, false negative results caused by the deletion of HPV L-region common in cancer cells may be eliminated. Type specific capture oligonucleotides (two per each HPV type) containing a 5'-C12-amino linker were prepared at a final concentration 100 nM (0.1 pmole/ µL) in a freshly prepared solution of 10 mM l-(3-dimethylaminopropyl)-3-ethylcarbodimide hydrochloride containing 10 mM l-methylimidizole, pH 7. This solution was added at 100  µL per well (one type per well) to a Nucleolink 96-weII plate, sealed with thermostable tape and incubated at 50°C for 4 hours. The solution was then decanted and the wells were washed three times, soaked for 5 minutes and washed three times all with 100 mM Tris-HCl, pH 7.5, 150 mM sodium chloride, and 0.1% Tween-20 at room temperature. The wells were then washed three times, soaked for 5 minutes and washed three times with molecular biology grade water. The plates were then allowed to dry for 30 minutes at room temperature. The plates were stored at 4°C in an etfiylene bag with desiccant until needed.<br>
The NucIeoLink plate with covalently bound oligonucleotides was brought to room temperature after storage. The plate was blocked with 290  µL per well of 6% casein in 100 mM Tris-HCl, pH 7.2, 0.05% sodium azide, for 30 min at room temperature. Model samples (100 pg/tnL) of dilutions of HPV plasmids (10 kb) prepared in a solution composed of 100  µg/mL Herring Sperm DNA, 1 M Guanidine-HCI, 10 mM Tris-HCl (pH 8.0), 10 mM EDTA, 0.05% Sodium Azide in deionized water. The samples (50 µ1) were mixed in a 96-well microplate with 25  µL of 1.75M NaOH and incubated at 70°C for 30 minutes. Unlabeled RNA probe (15 ng per assay) in 0.125 M sodium citrate, 0.125 M sodium phosphate, 0.6 M triethanolamine, 0.6 N,N-bis(2-hydroxyethyl)-2-aminoethane sulfonic acid, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1) was added to each well in the hybridization plate. The solution became clear or slightly yellow. The contents of each well were transferred to their respective wells in the Nucleolink plate. The plate was incubated at 65°C while shaking at 1150 rpm for 1 hour. The plate was cooled to room temperature and the solution was decanted. A solution (100  µL/well) of alkaline phosphatase conjugated mouse monoclonal anti-<br>
51<br><br>
RNA:DNA antibody in 0.1M mM Tris-pH 7.4,0.6 M sodium chloride, 0.1 mM zinc chloride, 1 mM magnesium chloride 0.05% sodium azide, 0.25% tween-20, 0.2 mg/mL RNase, 4% hydroxypropyl-p-cyclodextrin, 0.05% goat IgG, 0.0008% Sulforhodamine B, with 30% blocking solution (6% casein in 100 mM Tris-HCl, pH 7.2, 0.05% sodium azide) was added to each well and the plate was incubated at 37°C for 30 minutes. The solution was decanted and the plate was washed twice with 128 mM Tris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide; 0.24% Tween and then 250 uX of 128 mMTris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide; 0.24% Tween was added to each well. The plate was sealed with thermostable tape and incubated at 60oC for 10 minutes. The solution was decanted and the plate was washed twice more with 128 mM Tris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide. The plate was then washed twice with 40 mM Tris pH 8.2, 100 mM sodium chloride, 0.05% sodium azide. The solution was decanted and 100 u.L of CDP-Star substrate was added to each well. The plate was incubated for 20 minutes in the dark and read on the DML luminometer. Table 15 shows the resulting signal to noise ratios of 100 pg/mL of 7 different HPV cloned targets as detected by the method described above.<br>
Table 15: Typing of HPV 16,18, 31,33, 35, 45 and 52 on a 96 well plate<br><br>
Capture	HPV targets, Signal/Noise<br>
probes	,6	18	31	33	35	45	52	No target<br>
16	30.6	1.2	1.1	0.7	0.3	0.6	0.8	1.0<br>
18	0.5	21.9	1.1	0.5     .	1.1	1.5	0.7	1.0<br>
31	0.6	0.5	264.4	0.6	0,6	1.7	1.8	1.0<br>
33	0.4	1.2	1.8	91.7	0.5	1.2	0.4	1.0<br>
35              0.4	0.3	1.7	0.4	103.6	0.7	0.4	1.0<br>
45        I      1.1	1.9	0.8	0.8	1.0	34.3	0.5	1.0<br>
52	0.7	0.5	0.5	0.5	0.7	0.5	44.4	1.0<br>
EXAMPLE 13<br>
52<br><br>
EFFECTS OF USING A MASK IN CONJUNCTION WITH A SUB-WELL PLATE<br>
A solution of alkaline phosphatase was added to CDP-Star substrate. This solution (20uL) was added to the indicated sub-wells of a 384 well plate (Digene Corp.) and incubated for 20 minutes. The plate was then read on the 384 well plate luminometer at about 450 nm with and without the addition of a mask (Digene Corp.). Table 16 shows that a mask which covers each well/subwell increases the S/N ratio by more then 100-fold.  The sub-well plate with the mask is advantageous for overcomiiig the previously known cross-talk between subwells. Each box of Table 16 represents a single sub-well. Only two subwells have chemiluminescent substrate, which are represented in bold face type.<br>
Table 16: Cross-Talk Between Neighboring Sub Wells<br>
RELATIVE LIGHT UNITS WITHOUT MASK<br><br>
1712	5598	• 2192	1210<br>
4076	21986	5181	1262<br>
1594	5024	2830	6046<br>
1004	1051	4844	30128<br>
1712	5598	2192	1210<br><br>
RELATIVE LIGHT UNITS WITH MASK<br>
15	48	15	8<br>
33	56548	40	26<br>
18	24	12	16<br>
31	16	23	64942<br>
15	48	15	8<br>
*Sub-weIls with added chemiluminescent substrate are shown in bold.<br>
EXAMPLE 14 HPV TYPING IN A 384-SUB-WELL PLATE. Detection of individual as well as multiple types of HPV in a sub-well plate format was performed. In this example, sample was added to each well and the individual targets migrated and hybridized to the corresponding capture sequence probes. The detection reagent was added to the well, where it bound to the target in the positive sub-wells. Chemiluminescence was used for detection.<br>
53<br><br>
Type specific capture oligonucleotides (two oligonucleotides per each HPV type) containing a 5'-C12-Amino linker were prepared at a final concentration of 5  µM (5 pmoIe/u.L) to a solution of 500 mM sodium hydrogen phosphate, pH 8.5 containing 1 mM ethylenediaminetetraacetic acid disodium salt. This solution was added at 20 µL per well to each sub-well of the sub-well plate (one type per sub-well), sealed with thermostable tape and incubated at 42°C for 4 hours. The solution was decanted and the wells were washed three times with TBS. The plate was then used immediately in the HPV typing assay.<br>
The plate was blocked with 6% casein in 100 mM Tris-HCl, pH 7.2, 0.05% sodium azide (1 mL per large well), for 30 min at room temperature. Model samples (100 pg/mL) consist of dilution of HPV plasmids (10 kb) prepared in a solution composed of 100 |xg/mL Herring Sperm DNA, 1 M Guanidine-HCl, 10 mM Tris-HCl (pH 8.0), 10 mM EDTA, 0.05% Sodium Azide in deionized water. The samples (500  µl) are mixed in an eppendorf tube with 250  µL of 1.75M NaOH and incubated at 70°C for 30 min. Next, RNA Probe (15 ng per assay) in 0.125 M sodium citrate, 0.125 M sodium phosphate, 0.6 M triethanolamine, 0.6 N,N-bis(2-hydroxyethyl)-2-aminoethane sulfonic acid, 0.83 M glacial acetic acid, 3% polyacrylic acid, 5 mM EDTA, 0.05% sodium azide and 0.4% Tween-20 (pH 3.6-4.1) was added to each tube. The solution is now clear or slightly yellow. The contents of each tube (lmL) were transferred to the respective well in the subwell plate. The plate was incubated at 65°C with shaking at 1150 rpm for 1 hour. The plate was cooled to room temperature and the solution was decanted. A solution (lmL per large well) of alkaline phosphatase conjugated mouse monoclonal anti RNArDNA antibody in 100 mM Tris-pH 7.4, 0.6 M sodium chloride, 0.1 mM zinc chloride, 1 mM magnesium chloride 0.05% sodium azide, 0.25% tween-20, 0.2 mg/mL RNase, 4% hydroxypropyl-p-cyclodextrin, 0.05% goat IgG, 0.0008% Sulforhodamine B, with 30% blocking solution (6% casein in 100 mM Tris-HCl, pH 7.2,0.05% sodium azide) was added to each well and the plate was incubated at 37°C for 30 minutes. The solution was decanted and the plate was washed twice with 128 mM Tris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide; 0.24% Tween and then 1 mL of 128 mM<br>
54<br><br>
Tris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide; 0.24% Tween was added to each well. The plate was sealed with thermostable tape and incubated at 60°C for 10 minutes. The solution was decanted and the plate was washed twice more with 128 mM Tris pH 7.2; 0.6 M sodium chloride; 0.05% sodium azide; 0.24% Tween. The plate was then washed twice with 40 mM Tris pH 8.2, 100 mM sodium chloride, 0.05% sodium azide. The solution was decanted and 20 u.L of CDP-Star substrate (Applera Corp.) was added to each well. The plate was incubated for 20 minutes in the dark and read on the 384-Iuminometer with the mask at about 450 nm as described in Example 13.<br>
55<br>
The results in Table 17 demonstrate the specificity of the detection of Individual and multiple HPV types in the subwell plate. Signals shown in bold face type demonstrate the specificity of the detection. For example, HPV16 target gave a positive signal when presented with HPV16 probe only and gives no signal with probes for other HPV types. However, multiple infections give multiple signals.<br><br><br>
EXAMPLE 15<br>
COMPARISON OF HPV DETECTION METHODS USING A MICROTITER PLATE AND BEADS<br>
Two assay formats of the invention were compared where the detection step is performed on a microtiter plate (as detected by chemiluminescence) or on Luminex carboxylated beads (as detected by fluorescence).<br>
The target enrichment and target amplification steps were performed as described in the Example 6. The third step, involving hybridization and detection, was performed in two formats: one on Luminex carboxylated beads as described in Example 6, and the other on a microtiter plate as described in the Example 14.<br>
The results in Table 18 show that the plate assay can detect 500 input copies in 30 minutes of amplifying with an S/N ratio of 9. At 1 hour of amplification, the plate assay is at least one order of magnitude more sensitive than the Luminex format.<br>
Table 18: HPV detection using Beads and Plate detection format.<br>
Target Input copies/assay	Beads (fluorescence)		<br><br>
	30min amplification	60 min amplification	120 min amplification<br><br>
	S/N	%CV	S/N	%CV	S/N	%CV<br>
0	1	20%	1	21%	1	16%<br>
5X102	1	8%	24	3%	1009	13%<br>
5X103	2	8%	281	21%	1548	5%<br>
5X104	18	14%	1288	9%	1520	19%<br>
Target Input copies/assay	<br><br>
	Plate (chemiluminescence)<br><br>
	30min amplification	60 min amplification	120 min amplification<br><br>
	S/N	%CV	S/N	%CV	S/N	%CV<br>
0	1	42%	1	81%	1	57%<br>
5X102	9	8%	1046	29%	12320	7%<br>
5X103	102	15%	9054	6%	11715	15%<br>
(          5X104	929	8%	13019	5%	12649	2%<br>
56<br><br>
EXAMPLE 16 EFFECT OF NUCLEOTIDE RATIO IN RANDOM PRIMERS ON THE EFFICIENCY OF ISOTHERMAL AMPLIFICATION Random primers were chemically synthesized using different combinations and ratios of nucleotides in the target amplification step of the method described in Example 6. The results in Table 19 show that dG and T play a major role in achieving efficient amplification. The omission of dA had a less pronounced effect and omitting dC had no effect at all. Isothermal amplification was performed for 2 hours with pentamer primers. The complete assay was performed as described in Example 6. Relative efficiency was calculated as signal to noise, where the efficiency of the standard primer composition was (1.0).<br>
Table 19: Relative efficiency of target detection depending on the composition of random primers<br>
Primer composition	Target input (copies/assay)	Relative assay |         efficiency<br>
dA:T:dG:dC 1:   1:   I:   1	100	(1.0)<br><br>
	1000	(1.0)<br>
dA:T:dG 1:   1:   1	100	0.95<br><br>
	1000	1.0<br>
T:dG:dC 1:   1:   1	100	0.4<br><br>
	1000	0.6<br>
dA:dG:dC 1:    1:   1	100	0.10<br><br>
	1000	0.12<br>
dA:T:dC 1:   1: I	100	
<br>
	1000	
dA:T:dG:dC 1:   1:  3:   1	100	0.7<br><br>
	1000	0:6<br>
dA:T:dG:dC 2.5:2.5:4:  1	100	0.95<br><br>
	1000	1.0<br>
dA:T:dG \:   \:  2	100	1.5<br><br>
	1000	1.2<br>
EXAMPLE 17<br>
HPV TYPING ON A MICROCHIP ARRAY<br>
Detection of HPV types, individual as well as multiple types of HPV,<br>
on a microchip array format may be performed. Target nucleic acid capture and<br>
i amplification are performed according to Example 6. Reagents and washing protocols<br>
57<br><br>
are performed according to commercially available products and instructions for washing slides, membranes, chips, etc. The target detection step comprises target denaturation and hybridization to the RNA probes, or signal sequence probes, both according the Example 6.<br>
In this example, hybridization of target to the sequence specific capture sequence oligonucleotide probe occurs on a spotted oligonucleotide array rather than on beads, as described in Example 6. The term "spotted array" refers to a 2-dimensional series of elements on the surface of a solid support, where each element is an aliquot of oligonucleotide deposited in a specific location. Solid supports for this format include: a glass slide, nylon membrane, or a microchip (glass or silicon). Further examples and description are found in "Microarrays And Cancer Research" Ed. J. Warrington, Eaton Publishing, 2002.<br>
Type-specific capture sequence oligonucleotide probes (two oligonucleotides per each HPV type) modified with a 5'-C12-Amino linker is spotted on a solid support, such as a microarray, each spot corresponding to a specific HPV type. Oligonucleotides are attached through, for example, the primary amino group. The solid support is equilibrated with 2X SSC (sodium chloride and sodium citrate) buffer at room temperature for 10 min, followed by 30 min incubation at 37°C in pre-hybridization buffer (2X SSC/0.05% blocking reagent/5% dextran sulfate/0.1% SDS) with 50 micrograms per mL of denatured salmon sperm DNA. Hybridization is then performed in the same pre-hybridization buffer with the addition of amplified target nucleic acids and RNA signal sequence probes/ Hybridization of the target nucleic acids, RNA signal sequence probes, and type-specific capture sequence oligonucleotide probes occurs while incubating at 37°C with shaking for 3 hours in rehybridization solution.<br>
Detection of the spots containing amplicon may be performed in two ways: a) using HYBRID CAPTURE antibodies (HC-Ab) labeled with Cy-fluorescent dye, with subsequent scanning using commercially available scanners; or b) by using alkaline phosphatase labeled HC-Ab with subsequent application of precipitating substrate (for example, 5-bromo-4-chloro-3-indolyl-phosphate and nitroblue<br>
58<br><br>
tetrazolium: NBT/BCEP). In the latter case, the signal is detectable without instrumental assistance.<br>
The above description of various embodiments has been presented for purposes of illustration and description. It is not intended to be exhaustive or limiting to the precise forms disclosed. Obvious modifications or variations are possible in light of the above teachings. The embodiments discussed were chosen and described to provide illustrations and its practical application to thereby enable one of ordinary skill in the art to utilize the various embodiments and with various modifications as are suited to the particular use contemplated. All such modifications and variations are within the system as determined by the appended claims when interpreted in accordance with the breadth to which they are fairly, legally and equitably entitled.<br>
59<br><br>
WE CLAIM:<br>
1.	A method of detecting each of a plurality of target nucleic acids<br>
comprising:<br>
a)	capturing a plurality of target nucleic acids onto a solid support by mixing: said plurality of target nucleic acids, nucleic acid probes complementary to at least a portion of the target nucleic acids, wherein one is RNA and the other is DNA, and a solid support, forming a plurality of enriched target nucleic acids;<br>
b)	amplifying the enriched nucleic acids or nucleic acid probes, forming a plurality of amplified targets; and<br>
c)	detecting the target nucleic acids by detection of the plurality of amplified targets by mixing: the plurality of amplified targets; selectable oligonucleotides which hybridize to a first portion of the amplified targets; and nucleic acid probes complementary to a second portion of the amplified targets, forming a DNA:RNA hybrid oligonucleotide complex, wherein the DNA:RNA hybrids are detectable by DNA:RNA hybrid-specific binding agents, and selectable by the oligonucleotide complex separation, thereby detecting each of the plurality of target nucleic acids.<br><br>
2.	The method according to claim 1, wherein in the capturing step, the solid support is conjugated to a DNA:RNA hybrid-specific binding agent, thereby capturing the target nucleic acids to a solid support.<br>
3.	The method according to claim 1, wherein the DNA:RNA hybrid-specific binding agent is selected from the group consisting of: DNA:RNA hybrid-specific antibodies, monoclonal or polyclonal antibodies, or fragments thereof, proteins, catalytically inactive RNase H, and nucleic acids, nucleic acid aptamers, or oligonucleotides which bind and form a triplex structure.<br>
4.	The method according to claim 2, wherein the DNA:RNA hybrid-specific binding agent is a DNA:RNA hybrid-specific antibody.<br>
60<br><br>
5.	The method according to claim 2, wherein the DNA:RNA hybrid-specific binding agent is a nucleic acid probe complementary to the target nucleic acid.<br>
6.	The method according to claim 3, wherein the DNA:RNA hybrid-specific binding agent is detectably labeled.<br>
7.	The method according to claim 6, wherein the label is detected by: colorimetry, chemiluminescence, fluorescence, and light-scattering methods.<br>
8.	The method according to claim 1, wherein the captured target nucleic acids or nucleic acid probes are amplified by isothermal amplification.<br>
9.	The method according to claim 8, wherein DNA polymerase is used in the isothermal amplification.<br>
10.	The method according to claim 8, wherein random primers are used in the isothermal amplification.<br>
11.	The method according to claim 10, wherein the random primers are pentamers.<br>
12.	The method according to claim 10, wherein the random primers contain a subset of primer sequences specific for a disease.<br>
13.	The method according to claim 1, wherein the selectable oligonucleotides are bound to a solid support.<br>
14.	The method according to claim 1, wherein the solid support is selected from the group consisting of: plates, microplates, microliter plates, slides, dishes, beads, particles, microparticles, cups, strands, chips, microchips, strips, membranes, microarrays, and test tubes.<br>
15.	The method according to claim 14, wherein the solid support is made of material selected from the group consisting of: glass, silicon, plastic, polystyrene, polyethylene, polypropylene, and polycarbonate.<br>
16.	The method according to claim 14, wherein the solid support is modified to contain: has been modified so that the surface contains carboxyl, amino,<br>
61<br><br>
hydrazide, aldehyde groups, nucleic acid or nucleotide derivatives, primary amino groups, and dyes.<br>
17.	The method according to claim 1, further comprising adding blocker probes.<br>
18.	A method of detecting each of a plurality of target nucleic acids comprising:<br><br>
a)	hybridizing a plurality of target nucleic acids to nucleic acid probes which are complementary to at least a portion of the target nucleic acids, forming DNA:RNA hybrids;<br>
b)	capturing the DNA:RNA hybrids with DNA:RNA hybrid-specific antibodies conjugated to solid supports;<br>
c)	separating unbound target nucleic acids from the DNA:RNA hybrids;<br>
d)	amplifying the captured target nucleic acids or nucleic acid probes, forming a plurality of amplified targets, using random primers and DNA polymerase;<br>
e)	hybridizing nucleic acid probes complementary to a first portion of the amplified targets, forming DNA:RNA hybrids;<br>
f)	hybridizing oligonucleotides conjugated to a plurality of solid supports to a second portion of the amplified targets, wherein the solid support is selectable; and<br>
g)	selecting each of the plurality of amplified targets by the selectable solid support; and<br>
h)        detecting each of the plurality of amplified targets by binding DNA:RNA hybrid-specific binding agents to the DNA:RNA hybrids, wherein the presence of the plurality of amplicons is indicative of the presence of the target nucleic acids.<br>
62<br><br>
19.	The method according to claim 18, wherein the solid support conjugated to oligonucleotides are distinguishable beads.<br>
20.	The method according to claim 19, wherein the distinguishable beads are selected from a group consisting of detectable labels: a fluorescent label, a gold label, and an enzyme label.<br>
21.	A multiplex method of detecting a plurality of target DNAs comprising:<br><br>
a)	hybridizing a plurality of target DNAs to RNA probes which are complementary to at least a portion of the target DNAs, forming DNA:RNA hybrids;<br>
b)	capturing the DNA:RN hybrids with DNA:RNA hybrid-specific antibodies conjugated to beads;<br>
c)	separating unbound nucleic acids from DNA:RNA hybrids, forming enriched targets;<br>
d)	amplifying the enriched target DNAs, forming a plurality of amplified targets, using random primers and DNA polymerase;<br>
e)	hybridizing RNA probes complementary to a first portion of the amplified target DNAs, forming DNA:RNA hybrids;<br>
f)	hybridizing DNA oligonucleotides to a second portion of the target DNAs, wherein the DNA oligonucleotides are conjugated to a plurality of selectable beads; and<br>
g)	detecting the plurality of amplified target DNAs by binding DNA:RNA hybrid-specific antibodies to the DNA:RNA hybrids and selecting each of the amplified targets based on the particular selectable bead.<br>
22.      The method of claim 1, wherein the nucleic acid probes complementary to at least a portion of the target nucleic acids and oligonucleotide sequences are selected from the group consisting of:<br>
GTACAGATGGTACCGGGGTTGTAGAAGTATCTG [SEQ IDNOrl]; CTGCAACAAGACATACATCGACCGGTCCACC [SEQ ID NO:2];<br>
63<br><br>
GAAGTAGGTGAGGCTGCATGTGAAGTGGTAG [SEQ ID NO:3]; CAGCTCTGTGCATAACTGTGGTAACTTTCTGGG   [SEQ ID NO:4]; GAGGTCTTCTCCAACATGCTATGCAACGTCCTG [SEQ ID NO:5]; GTGTAGGTGCATGCTCTATAGGTACATCAGGCC [SEQ ID NO:6]; CAATGCCGAGCTTAGTTCATGCAATTTCCGAGG [SEQ ID NO:7]; GAAGTAGTAGTTGCAGACGCCCCTAAAGGTTGC [SEQ ID N0:8]; GAACGCGATGGTACAGGCACTGCAGGGTCC [SEQ ID NO:9]; GAACGCGATGGTACAGGCACTGCA [SEQ ID NO: 10]; ACGCCCACCCAATGGAATGTACCC [SEQ ID NO: 11]; TCTGCGTCGTTGGAGTCGTTCCTGTCGTGCTC [SEQ ID NO: 12]; *(TTATTATTA)CTACATACATTGCCGCCATGTTCGCCA [SEQ ID NO: 13]; (TTATTATTA)TGTTGCCCTCTGTGCCCCGGTTGTCTATAGCCTCCGT [SEQ ID NO: 14]; (TTATTATTA)GGAGCAGTGCCCAAAAGATTAAAGTTTGC [SEQ ID NO: 15]; (TTATTATTA)CACGGTGCTGGAATACGGTGAGGGGGTG [SEQ ID NO: 16]; (TTATTATTA)ACGCCCACCCAATGGAATGTACCC [SEQ ID NO: 17]; (TTATTATTA)ATAGTATTGTGGTGTGTTTCTCACAT [SEQ ID NO: 18]; (TTATTATTA)GTTGGAGTCGTTCCTGTCGTG [SEQ ID NO: 19]; (TTATTATTA)CGGAATTTCATTTTGGGGCTCT [SEQ ID NO:20]; GCTCGAAGGTCGTCTGCTGAGCTTTCTACTACT [SEQ ID NO:21]; GCGCCATCCTGTAATGCACTTTTCCACAAAGC [SEQ ID NO:22]; TAGTGCTAGGTGTAGTGGACGCAGGAGGTGG [SEQ ID NO:23]; GGTCACAACATGTATTACACTGCCCTCGGTAC [SEQ ID NO:24]; CCTACGTCTGCGAAGTCTTTCTTGCCGTGCC [SEQ ID NO:25]; CTGCATTGTCACTACTATCCCCACCACTACTTTG [SEQ IDNO:26]; CCACAAGGCACATTCATACATACACGCACGCA [SEQ ID NO:27]; GTTCTAAGGTCCTCTGCCGAGCTCTCTACTGTA [SEQ ID NO:28]; (TTATTATTA)TGCGGTTTTGGGGGTCGACGTGGAGGC [SEQ ID NO:29]; (TTATTATTA)AGACCTGCCCCCTAAGGGTACATAGCC [SEQ ID NO:30]; (TTATTATTA)CAGCATTGCAGCCTTTTTGTTACTTGCTTGTAATAGCTCC [SEQ ID NO:31]; (TTATTATTA)ATCCTGTAATGCACTTTTCCACAAA [SEQ ID<br>
64<br><br>
NO:32]; (TTATTATTA)GCCTGGTCACAACATGTATTAC [SEQ ID NO:33]; (TTATTATTA)CAGGATCTAATTCATTCTGAGGTT [SEQ ID NO:34]; TGCGGTTTTGGGGGTCGACGTGGAGGC [SEQ ID NO:35]; GGCGCAACCACATAACACACAGAACCACAAAAC [SEQ ID NO:36]; GTTCTACACGGGTTTGCAGCACGATCAACAACG [SEQ ID NO:37]; CGCTGCTTGTGGTGGTCGGTTATCGTTGTCTG [SEQ ID NO:38]; GACGTAGTGTCGCCTCACATTTACAACAGGAC [SEQ ID NO :3 9]; CTCGCTTGGTGGGGTTGTAGGGGAGCTCGG [SEQ ID NO:40]; GCTGTAGTTGTCGCAGAGTATCCCGTGAGG [SEQ ID NO:41]; GTGAGCCTGTGTTATATGTAGTGCCCGAATCCC [SEQ ID NO:42]; CCACCTCCTGCGTCCACTACACCTAGCACTA [SEQ ID NO:43]; TGCGTGCGTGTATGTATGAATGTGCCTTGTGG [SEQ ID NO:44]; AATTAGCGCATTGCCCCGTCCAACGTCCCG [SEQ ID NO:45]; CGCCGTGCACGTGTAGCCACCATATTTAATCAC [SEQ ID NO:46]; CGAATTGTGTGAGGTGCTGGAAGAATCGGTGC [SEQ ID NO:47]; GATCGTTCACAACTTTTACCTGCACCGGATCC [SEQ ID NO:48]; CTAGGTTCTCTAGATGTTTGTCTCCAGCACCCC [SEQ ID NO:49]; CTGTCGGTATTGTCTGTGTCGCTGATGTGTG [SEQ ID NO:50]; GATACACACACATTTGCAGCCCGGTCCACACA [SEQ IDNO:51]; GGTGGCAAAGGACGTATGTGAGTGCAGAGGAC [SEQ ID NO:52]; GCGTTGCGGAGGGGTATGATAGTTGCTCAGAAG [SEQ ID NO:53]; GTCTAGGCGTGTAGGAGGAAACAAGATGGGG [SEQ ID NO:54]; CTGAACACAGCAGTTCTCTATACCAATGGCGCTATTTC [SEQ ID NO:55]; TTGGTTGCCCCTGAGCAGTCGGACCCTATGGATA [SEQ ID NO:56]; GCGCCGCATTGCTGCACCTCGTTTATATAGCAGGGCATTTTC [SEQ ID NO:57]; CCTGGCGCATGTCATACACACCACATTACTC [SEQ ID NO:58j; CACGAAGTGTCAGTGCACAGTATGCCTTGC [SEQ ID NO:59j; GCCATGTACTTCACAAACTGTTAATACTGGTGATTGTCCC [SEQ ID NO.60]; CCTACACAGTACAAGTGGAGGCCATCACCCG [SEQ ID NO:61]; GTCTGACACATACTGTTGTAACCCATAGTTAAACACAGG [SEQ ID NO:62];<br>
65<br><br>
GCTGTCTCCCTGTCTTCCTGTGTATTGTTTATAAGTGTATT [SEQ ID NO:63]; and GTACAGCACTAAAATGGAGTTTGGTGTGTTACTATAGTGCATAC [SEQ ID NO:64].<br>
23.       A composition for use in the method of claim 17, wherein the the nucleic acid probes complementary to at least a portion of the target nucleic acids, oligonucleotide sequences, and blocker probes are selected from the group consisting of: GTACAGATGGTACCGGGGTTGTAGAAGTATCTG [SEQ ID NO:l]; CTGCAACAAGACATACATCGACCGGTCCACC [SEQ ID NO:2]; GAAGTAGGTGAGGCTGCATGTGAAGTGGTAG [SEQ ID NO:3]; CAGCTCTGTGCATAACTGTGGTAACTTTCTGGG   [SEQ ID NO:4]; GAGGTCTTCTCCAACATGCTATGCAACGTCCTG [SEQ ID NO:5]; GTGTAGGTGCATGCTCTATAGGTACATCAGGCC [SEQ ID NO:6]; CAATGCCGAGCTTAGTTCATGCAATTTCCGAGG [SEQ ID NO:7]; GAAGTAGTAGTTGCAGACGCCCCTAAAGGTTGC [SEQ ID NO:8]; GAACGCGATGGTACAGGCACTGCAGGGTCC [SEQ ID NO:9]; GAACGCGATGGTACAGGCACTGCA [SEQ ID NO: 10]; ACGCCCACCCAATGGAATGTACCC [SEQ ID NO: 11]; TCTGCGTCGTTGGAGTCGTTCCTGTCGTGCTC [SEQ ID NO: 12]; *(TTATTATTA)CTACATACATTGCCGCCATGTTGGCCA [SEQ ID NO: 13]; (TTATTATTA)TGTTGCCCTCTGTGCCCCCGTTGTCTATAGCCTCCGT [SEQ ID NO: 14]; (TTATTATTA)GGAGCAGTGCCCAAAAGATTAAAGTTTGC [SEQ ID NO: 15]; (TTATTATTA)CACGGTGCTGGAATACGGTGAGGGGGTG [SEQID NO:16]; (TTATTATTA)ACGCCCACCCAATGGAATGTACCC [SEQ ID NO:17]; (TTATTATTA)ATAGTATTGTGGTGTGTTTCTCACAT [SEQ ID NO: 18]; (TTATTATTA)GTTGGAGTCGTTCCTGTCGTG [SEQ ID NO: 19]; (TTATTATTA)CGGAATTTCATTTTGGGGCTCT [SEQ ID NO:20]; GCTCGAAGGTCGTCTGCTGAGCTTTCTACTACT [SEQ ID NO:21]; GCGCCATCCTGTAATGCACTTTTCCACAAAGC [SEQ ID NO:22]; TAGTGCTAGGTGTAGTGGACGCAGGAGGTGG [SEQ ID NO:23]; GGTCACAACATGTATTACACTGCCCTCGGTAC [SEQ ID NO:24];<br>
66<br><br>
CCTACGTCTGCGAAGTCTTTCTTGCCGTGCC [SEQ ID NO:25]; CTGCATTGTCACTACTATCCCCACCACTACTTTG [SEQ ID NO:26]; CCACAAGGCACATTCATACATACACGCACGCA [SEQ ID NO:27]; GTTCTAAGGTCCTCTGCCGAGCTCTCTACTGTA [SEQ ID NO:28]; (TTATTATTA)TGCGGTTTTGGGGGTCGACGTGGAGGC [SEQ ID NO:29]; (TTATTATTA)AGACCTGCCCCCTAAGGGTACATAGCC [SEQ ID NO:30]; (TTATTATTA)CAGCATTGCAGCCTTTTTGTTACTTGCTTGTAATAGCTCC [SEQ ID NO:31]; (TTATTATTA)ATCCTGTAATGCACTTTTCCACAAA [SEQ ID NO:32]; (TTATTATTA)GCCTGGTCACAACATGTATTAC [SEQ ID NO:33]; (TTATTATTA)CAGGATCTAATTCATTCTGAGGTT [SEQ ID NO:34]; TGCGGTTTTGGGGGTCGACGTGGAGGC [SEQ ID NO:35]; GGCGCAACCACATAACACACAGAACCACAAAAC [SEQ ID NO:36]; GTTCTACACGGGTTTGCAGCACGATCAACAACG [SEQ ID NO:37]; CGCTGCTTGTGGTGGTCGGTTATCGTTGTCTG [SEQ ID NO:38]; GACGTAGTGTCGCCTCACATTTACAACAGGAC [SEQ ID NO:39]; CTCGCTTGGTGGGGTTGTAGGGGAGCTCGG [SEQ ID NO:40]; GCTGTAGTTGTCGCAGAGTATCCCGTGAGG [SEQ ID NO:41]; GTGAGCCTGTGTTATATGTAGTGCCCGAATCCC [SEQ ID NO:42]; CCACCTCCTGCGTCCACTACACCTAGCACTA [SEQ ID NO:43]; TGCGTGCGTGTATGTATGAATGTGCCTTGTGG [SEQ ID NO:44]; AATTAGCGCATTGCCCCGTCCAACGTCCCG [SEQ ID NO:45]; CGCCGTGCACGTGTAGCCACCATATTTAATCAC [SEQ ID NO:46]; CGAATTGTGTGAGGTGCTGGAAGAATCGGTGC [SEQ ID NO:47]; GATCGTTCACAACTTTTACCTGCACCGGATCC [SEQ ID NO:48]; CTAGGTTCTCTAGATGTTTGTCTCCAGCACCCC [SEQ DDNO:49]; CTGTCGGTATTGTCTGTGTCGCTGATGTGTG [SEQ ID NO:50]; GATACACACACATTTGCAGCCCGGTCCACACA [SEQ ID NO:51]; GGTGGCAAAGGACGTATGTGAGTGCAGAGGAC [SEQ ID NO:52]; GCGTTGCGGAGGGGTATGATAGTTGCTCAGAAG [SEQ ID NO:53]; GTCTAGGCGTGTAGGAGGAAACAAGATGGGG [SEQ ID NO:54];<br>
67<br><br>
CTGAACACAGCAGTTCTCTATACCAATGGCGCTATTTC [SEQ ID NO:55]; TTGGTTGCCCCTGAGCAGTCGGACCCTATGGATA [SEQ ID NO:56]; GCGCCGCATTGCTGCACCTCGTTTATATAGCAGGGCATTTTC [SEQ ID NO:57]; CCTGGCGCATGTCATACACACCACATTACTC [SEQ ID NO:58]; CACGAAGTGTCAGTGCACAGTATGCCTTGC [SEQ ID NO:59]; GCCATGTACTTCACAAACTGTTAATACTGGTGATTGTCCC [SEQ ID NO:60]; CCTACACAGTACAAGTGGAGGCCATCACCCG [SEQ ID NO:61]; GTCTGACACATACTGTTGTAACCCATAGTTAAACACAGG [SEQ ID NO:62]; GCTGTCTCCCTGTCTTCCTGTGTATTGTTTATAAGTGTATT [SEQ ID NO:63]; GTACAGCACTAAAATGGAGTTTGGTGTGTTACTATAGTGCATAC [SEQ ID NO:64]; and ACTCCAACGACGCAGA [SEQ ID NO:65];   , TTTTGTGGTTCTGTGTG [SEQ ID NO:66]; TTATGTGGTTGCGC [SEQ ID NO:67]; CGTTGTTGATCGTGC [SEQ ID NO:68];TGCAAACCCGTGTAG [SEQ ID NO:69]; CAGACAACGATAACCG [SEQ ID NO:70]; ACCACCACAAGCAGC [SEQ ID N0:71]; GTCCTGTTGTAAATGTG [SEQ ID NO:72]; AGGCGACACTACGTC [SEQ ID NO:73]; CGAGCTCCCCTACAA [SEQ ID NO:74]; CCCCACCAAGCGA [SEQ ID NO:75]; CCTCACGGGATACTC [SEQ ID NO:76]; TGCGACAACTACAGC [SEQ ID NO:77]; GGATTCGGGCACTA [SEQ ID NO:78]; CATATAACACAGGCTCAC [SEQ ID NO:79]; TAGTGCTAGGTGTAGTGG [SEQ ID NO:89]; ACGCAGGAGGTGG [SEQ ID NO:81]; TACATACACGCACGCA [SEQ ID NO:82]; CCACAAGGCACATTCA [SEQ ID NO:83j; GCTACACGTGCACGGCG [SEQ ID NO:84]; GTGATTAAATATGGTGG [SEQ ID NO:85]; GGGACGTTGGACG [SEQ ID NO:86]; GGCAATGCGCTAAT [SEQ ID NO:87]; CACCGATTCTTCCAG [SEQ ID NO:88]; CACCTCACACAATTCG [SEQ ID NO:89]; GGATCCGGTGCAG [SEQ ID NO:90]; GTAAAAGTTGTGAACGATC [SEQ ID NO:91]; GGTGCTGGAGACAA [SEQ ID NO:92]; ACATCTAGAGAACCTAG [SEQ ID NO:93]; CACACATCAGCGACA [SEQ ID NO:94]; CAGACAATACCGACAG [SEQ ID NO:95]; TGTGTGGACCGGG [SEQ ID NO:96]; CTGCAAATGTGTGTGTAT [SEQ ID NO:97]; GTCCTCTGCACTCACAT [SEQ<br>
68<br><br>
ID NO:98]; ACGTCCTTTGCCAC [SEQ ID NO: 99]; TTCTGAGCAACTATCATAC [SEQ ID NO:100]; CCCTCCGCAACG [SEQ ID NO:101]; CCCCATCTTGTTTCC [SEQ ID NO: 102]; TCCTACACGCCTAGAC [SEQ ID NO: 103]; TATAGAGAACTGCTGTGTTC [SEQ ID NO: 104]; GAAATAGCGCCATTG [SEQ ID NO: 105]; CTCAGGGGCAACCA [SEQ ID NO: 106]; ATCCATAGGGTCCGAC [SEQ ID NO:107];CAATGCGGCGC [SEQ ID NO:108];TATAAACGAGGTGCAG [SEQ ID NO:109]; GAAAATGCCCTGCTA [SEQ ID NO:l 10]; ACATGCGCCAGG [SEQ ID NO:l 11];<br>
GAGTAATGTGGTGTGTATG [SEQ ID NO: 112]; GCAAGGCATACTGTG [SEQ ID NO:l 13]; CACTGACACTTCGTG [SEQ ID NO: 114]; GGACAATCACCAGTATTA [SEQ ID NO: 115]; AGTTTGTGAAGTACATGG [SEQ ID NO:l 16]; CGGGTGATGGCC [SEQ ID NO:l 17]; CCACTTGTACTGTGTAGG [SEQ ID NO: 118]; CTGTGTTTAACTATGGGT [SEQ ED NO:l 19]; TACAACAGTATGTGTCAGAC [SEQ ID NO:120]; AAGACAGGGAGACAGC [SEQ ID NO:121]; CTTATAAACAATACACAGG [SEQ ED NO: 122]; ATGCACTATAGTAACACACC [SEQ ID NO: 123]; and ACTCCATTTTAGTGCTGTA [SEQ ID NO: 124].<br>
24.       A composition of matter comprising a nucleic acid sequence selected from the group consisting of: SEQ ID NOs: 1-124.<br>
Dated this 23rd day of May, 2008<br>
FOR DIGENE CORPORATION<br>
By their Agent<br>
(UMA BHATTAD) KRISHNA &amp; SAURASTRI<br>
69<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWFic3RyYWN0LmRvYw==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNMQUlNUyhBTUVOREVEKS0oMjgtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CLAIMS(AMENDED)-(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNMQUlNUyhBTUVOREVEKS0oOS0yLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CLAIMS(AMENDED)-(9-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNMQUlNUyhNQVJLRUQgQ09QWSktKDI4LTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CLAIMS(MARKED COPY)-(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNMQUlNUyhNQVJLRUQgQ09QWSktKDktMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CLAIMS(MARKED COPY)-(9-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWNsYWltcy5kb2M=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFICgxMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE (11-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFIDI0LTYtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE 24-6-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDExLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(11-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDE1LTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(15-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDI5LTctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(29-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDMtMTEtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(3-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDMtMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(3-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDQtMi0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUNPUlJFU1BPTkRFTkNFKDctMTAtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-CORRESPONDENCE(7-10-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWNvcnJlc3BvbmRlbmNlLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5kb2M=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-description(complete).doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMSgyMy01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 1(23-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMSgyOS03LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 1(29-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMSg0LTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 1(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMTMoNC0yLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 13(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMTgoMy0xMS0yMDA5KS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 18(3-11-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMTgoQ09QWSktKDQtMi0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 18(COPY)-(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMih0aXRsZSBwYWdlKS0oMjMtNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 2(title page)-(23-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMihUSVRMRSBQQUdFKS0oNC0yLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 2(TITLE PAGE)-(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMih0aXRsZSBwYWdlKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 2(title page).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMi5kb2M=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMi5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMygxNC02LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 3(14-6-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMygxNS0yLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 3(15-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMygyMy01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 3(23-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMyg0LTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 3(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gMyg3LTEwLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 3(7-10-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gNSgyMy01LTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 5(23-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LUZPUk0gNSg0LTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-FORM 5(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVBFVElUSU9OIFVOREVSIFJVTEUgMTM3KDE0LTYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-PETITION UNDER RULE 137(14-6-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVBPV0VSIE9GIEFUVE9STkVZKDI5LTctMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-POWER OF ATTORNEY(29-7-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVBPV0VSIE9GIEFUVE9STkVZKDQtMi0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-POWER OF ATTORNEY(4-2-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCgxNC02LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-REPLY TO EXAMINATION REPORT(14-6-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVJFUExZIFRPIEVYQU1JTkFUSU9OIFJFUE9SVCg5LTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-REPLY TO EXAMINATION REPORT(9-2-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVJFUExZIFRPIEhFQVJJTkcoMjgtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-REPLY TO HEARING(28-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LXNlcXVlbmNlIGxpc3RpbmcoMjMtNS0yMDA4KS5wZGY=" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-sequence listing(23-5-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1NVU1OUC0yMDA4LVVTIEdSQU5URUQgUEFURU5UICYgRVAgU0VBUkNIIFJFUE9SVCgxNC02LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">1042-MUMNP-2008-US GRANTED PATENT &amp; EP SEARCH REPORT(14-6-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTA0Mi1tdW1ucC0yMDA4LXdvIGludGVybmF0aW9uYWwgcHVibGljYXRpb24gcmVwb3J0KDIzLTUtMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">1042-mumnp-2008-wo international publication report(23-5-2008).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="253466-a-dendritic-compound.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253469-a-cobalt-based-alloy-for-the-coating-of-organs-subject-to-erosion-by-liquid.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253468</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1042/MUMNP/2008</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jul-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>23-May-2008</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>QIAGEN GAITHERSBURG, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1201 CLOPPER ROAD, G AITHERSBURG, MD 20878, U. S. A.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>NAZARENKO, IRINA</td>
											<td>3 RUDIS WAY, GAITHERSBURG, MD 20878,</td>
										</tr>
										<tr>
											<td>2</td>
											<td>LORINCZ, ATTILA T.</td>
											<td>6 CHINABERRY COURT, NORTH POTOMAC, MD 20878,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LOWE, BRIAN</td>
											<td>17717 OLD BALTIMORE ROAD, OLNEY , MD 21212,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>THAI, HA</td>
											<td>7111 WOODMONT AVENUE #113, CHEVY CHASE, MD 20815,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>EDER, PAUL</td>
											<td>10 WINDERMERE COURT, BETHESDA, MD 20852,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>MALLONEE, RICHARD</td>
											<td>7112 WARDMAN ROAD, BALTIMORE, MD 21212,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/060603</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-11-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>11/269,003</td>
									<td>2005-11-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253468-detection-of-nucleic-acid-by-target-specific-hybrid-capture-method by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:44:55 GMT -->
</html>
