<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253465-vaccine-for-prevention-and-treatment-of-hiv-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:45:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253465:&quot;VACCINE FOR PREVENTION AND TREATMENT OF HIV-INFECTION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;VACCINE FOR PREVENTION AND TREATMENT OF HIV-INFECTION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to novel HIV polypeptide and polynucleotide fusions of Gag, Pol and Nef which are useful in immunogenic compositions and vaccines. The invention relates in particular to a polypeptide which comprises Nef or an immunogenic fragment or derivative thereof, p17 Gag and p24 Gag or immunogenic fragments or derivatives thereof, wherein there is at least one HIV antigen or immunogenic fragment between p17 Gag and p24 Gag, the polypeptide further comprising RT or an immunogenic fragment or derivative thereof, and wherein the immunogenic fragments or derivatives remain capable of raising an immune response against the native antigen. Figure 20.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The gp 120 protein is the principal target of neutralizing antibodies, but unfortunately the<br>
most immunogenic regions of the proteins (V3 loop) are also the most variable parts of<br>
the protein. Therefore, the use of gp120 (or its precursor gp160) as a vaccine antigen to<br>
elicit neutralizing antibodies is thought to be of limited use for a broadly protective<br>
vaccine. The gp120 protein does also contain epitopes that are recognized by cytotoxic T<br>
lymphocytes (CTL). These effector cells are able to eliminate virus-infected cells, and<br>
therefore constitute a second major antiviral immune mechanism. In contrast to the target<br>
regions of neutralizing antibodies some CTL epitopes appear to be relatively conserved<br>
among different HIV strains. For this reason gp120 and gp160 maybe useful antigenic<br>
components in vaccines that aim at eliciting cell-mediated immune responses<br>
(particularly CTL).<br>
Non-envelope proteins of HIV-1 include for example internal structural proteins such as<br>
the products of the gag and pol genes and other non-structural proteins such as Rev, Nef,<br>
Vif and Tat (Green et al., New England J. Med, 324, 5, 308 et seq (1991) and Bryant et<br>
al. (Ed. Pizzo), Pediatr. Infect. Dis. J., 11, 5, 390 et seq (1992).<br>
HIV Nef is an early protein,that is it is expressed early in infection and in the absence of<br>
structural protein.<br>
The Nef gene encodes an early accessory HIV protein which has been shown to possess<br>
several activities. For example, the Nef protein is known to cause the down regulation of<br>
CD4, the HIV receptor, and MHC class I molecules from the cell surface, although the<br>
biological importance of these functions is debated. Additionally Nef interacts with the<br>
signal pathway of T cells and induces an active state, which in turn may promote more<br>
efficient gene expression. Some HIV isolates have mutations in this region, which cause<br>
them not to encode functional protein and are severely compromised in their replication<br>
and pathogenesis in vivo.<br><br>
The Gag gene is translated as a precursor polyprotein that is cleaved by protease to yield<br>
products that include the matrix protein (p17), the capsid (p24), the nucleocapsid (p9), p6<br>
and two space peptides, p2 and p1.<br>
The Gag gene gives rise to the 55-kilodalton (kD) Gag precursor protein, also called p55,<br>
which is expressed from the unspliced viral mRNA. During translation, the N terminus<br>
of p55 is myristoylated, triggering its association with the cytoplasmic aspect of cell<br>
membranes. The membrane-associated Gag polyprotein recruits two copies of the viral<br>
genomic RNA along with other viral and cellular proteins that triggers the budding of the<br>
viral particle from the surface of an infected cell. After budding, p55 is cleaved by the<br>
virally encoded protease (a product of the pol gene) during the process of viral maturation<br>
into four smaller proteins designated MA (matrix [p17]), CA (capsid [p24]), NC<br>
(nucleocapsid [p9]), and p6.<br>
In addition to the 3 major Gag proteins, all Gag precursors contain several other regions,<br>
which are cleaved out and remain in the virion as peptides of various sizes. These<br>
proteins have different roles e.g. the p2 protein has a proposed role in regulating activity<br>
of the protease and contributes to the correct timing of proteolytic processing.<br>
The p17 (MA) polypeptide is derived from the N-terminal, myristoylated end of p55.<br>
Most MA molecules remain attached to the inner surface of the virion lipid bilayer,<br>
stabilizing the particle. A subset of MA is recruited inside the deeper layers of the virion<br>
where it becomes part of the complex which escorts the viral DNA to the nucleus. These<br>
MA molecules facilitate the nuclear transport of the viral genome because a karyophilic<br>
signal on MA is recognized by the cellular nuclear import machinery. This phenomenon<br>
allows HIV to infect non-dividing cells, an unusual property for a retrovirus.<br>
The p24 (CA) protein forms the conical core of viral particles. Cyclophilin A has been<br>
demonstrated to interact with the p24 region of p55 leading to its incorporation into HIV<br>
particles. The interaction between Gag and cyclophilin A is essential because the<br>
disruption of this interaction by cyclosporin A inhibits viral replication.<br><br>
The NC region of Gag is responsible for specifically recognizing the so-called packaging<br>
signal of HIV. The packaging signal consists of four stem loop structures located near<br>
the 5' end of the viral RNA, and is sufficient to mediate the incorporation of a<br>
heterologous RNA into HIV-1 virions. NC binds to the packaging signal through<br>
interactions mediated by two zinc-finger motifs. NC also facilitates reverse transcription.<br>
The p6 polypeptide region mediates interactions between p55 Gag and the accessory<br>
protein Vpr, leading to the incorporation of Vpr into assembling virions. The p6 region<br>
also contains a so-called late domain which is required for the efficient release of budding<br>
virions from an infected cell.<br>
The Pol gene encodes two proteins containing the two activities needed by the virus in<br>
early infection, the RT and the integrase protein needed for integration of viral DNA into<br>
cell DNA. The primary product of Pol is cleaved by the virion protease to yield the<br>
amino terminal RT peptide which contains activities necessary for DNA synthesis (RNA<br>
and DNA-dependent DNA polymerase activity as well as an RNase H function) and<br>
carboxy terminal integrase protein. HIV RT is a heterodimer of full-length RT (p66) and<br>
a cleavage product (p51) lacking the carboxy terminal RNase H domain.<br>
RT is one of the most highly conserved proteins encoded by the retroviral genome. Two<br>
major activities of RT are the DNA Pol and Ribonuclease H. The DNA Pol activity of<br>
RT uses RNA and DNA as templates interchangeably and like all DNA polymerases<br>
known is unable to initiate DNA synthesis de novo, but requires a pre existing molecule<br>
to serve as a primer (RNA).<br>
The RNase H activity inherent in all RT proteins plays the essential role early in<br>
replication of removing the RNA genome as DNA synthesis proceeds. It selectively<br>
degrades the RNA from all RNA - DNA hybrid molecules. Structurally the polymerase<br>
and ribo H occupy separate, non-overlapping domains with the Pol covering the amino<br>
two thirds of the Pol.<br><br>
The p66 catalytic subunit is folded into 5 distinct subdomains. The amino terminal 23 of<br>
these have the portion with RT activity. Carboxy terminal to these is the RNase H Domain.<br>
WO 03/025003 describes DNA constructs encoding HIV-1 p17/24 Gag, Nef and RT, wherein<br>
the DNA sequences may be codon optimized to resemble highly expressed human genes.<br>
These constructs are useful in DNA vaccines.<br>
Fusion proteins containing multiple HIV antigens have been suggested as vaccine candidates<br>
for HIV, for example the Nef-Tat fusion as described in WO 99/16884. However, fusion<br>
proteins are not straightforward to produce; there can be difficulties in expressing them<br>
because they do not correspond to native proteins. There can be difficulties at the<br>
transcription level, or further downstream. Also, they may not be straightforward to<br>
formulate into a pharmaceutically acceptable composition. Notably, the majority of<br>
approaches to HIV vaccines that involve multiple antigens fused together, are DNA or live<br>
vector approaches rather than polypeptide fusion proteins.<br>
Brief Description of Accompanying Figures<br>
Figure 1A shows Coomassie staining and Western blot of F4 (p24-RT-Nef-p17) - 10% SDS-<br>
PAGE-Reducing.<br>
Figure 1B shows a solubility assay for F4 (p24-RT-Nef-p17) on Coomassie staining and<br>
Western blot.<br>
Figure 2 shows detection of recombinant p24-RT-Nef -p17 by Coomassie blue staining and<br>
on Western blot.<br>
Figure 3 shows an RT sequence alignment.<br>
Figure 4 shows expression of p51 and p66 on Coomassie stained gel and Western B1ot.<br>
Figure 5 shows a solubility assay for p51 and p66 under reducing as well as non-reducing<br>
conditions.<br>
Figure 6 shows expression of Nef-p17 and p17-Nef fusions, with and without a linker.<br>
Figure 7 shows a solubility assay for Nef-p17 and p 17-Nef fusions on Coomassie stained gel<br>
and Western blot.<br><br>
Figure 8 shows expression of F4* in B834(DE3) cells.<br>
Figure 9 shows expression of F3* in cellular extracts<br>
Figure 10 shows expression of F3* in B834(DE3) cells.<br>
Figure 11 shows expression of F4(p51) in B834(DE3) cells.<br>
Figure 12 shows expression of F4(p51 )* in B834(DE3) cells.<br>
Figure 13 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4-containing<br>
fractions collected during the purification of F4.<br>
Figure 14 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4(p51)*-<br>
containing fractions collected during the purification of F4(p51*).<br>
Figure 15 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4*-<br>
containing fractions collected during the purification of F4*.<br>
Figure 16 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the three fusion<br>
proteins F4, F4(p51)* and F4*. Different level of heterogeneity of the constructs after the Q<br>
sepharose step and after elimination of LMW bands by the Superdex 200 column are<br>
compared.<br>
Figure 17 shows a SDS gel of the F4-containing fractions collected during the purification of<br>
F4co and the purification of carboxyamidated F4co ("F4coca").<br>
Figure 18 compares the different purified bulks that were obtained from the two different<br>
purification methods (I and II) developed to purify the different F4 constructs.<br>
Figure 19 shows antibody responses mounted by OF1 mice to all four F4 components. The<br>
responses observed are shown in Figure 19. +/- indicates that the response observed was<br>
weak or only observed with one of the two adjuvant.<br>
Figure 20 shows antibody responses mounted by DBA mice to all four F4 components. The<br>
CD4+IL-2+ and CD4+IFNy+ responses observed are shown.<br><br>
AMENDED 1<br>
Summary of the Invention<br>
The present invention provides novel constructs for use in vaccines for the prophylaxis and<br>
treatment of HIV infections and AIDS.<br>
In one aspect the invention provides a polypeptide which comprises Nef or an immunogenic<br>
fragment or derivative thereof, and p 17 Gag and or p24 Gag or immunogenic fragments or<br>
derivatives thereof, wherein when both p17 and p24 Gag are present there is at least one HIV<br>
antigen or immunogenic fragment between them.<br>
In the constructs and compositions according to the invention as described herein, the Nef is<br>
preferably a full length Nef.<br><br>
In the constructs according to the invention the p17 Gag and p24 Gag are preferably full<br>
length p17 and p24 respectively.<br>
In one embodiment the polypeptide comprises both p17 and p24 Gag or immunogenic<br>
fragments thereof. In such a construct the p24 Gag component and p17 Gag component<br>
are separated by at least one further HIV antigen or immunogenic fragment, such as Nef<br>
and/or RT or immunogenic fragments or derivatives thereof.<br>
Alternatively p 17 or p24 Gag may be provided separately. Thus the invention also<br>
provides a composition comprising (i) a polypeptide which comprises Nef or an<br>
immunogenic fragment or derivative thereof and p17 Gag or an immunogenic fragment<br>
or derivative thereof, and (ii) p24 Gag or an immunogenic fragment or derivative thereof;<br>
or (i) a polypeptide which comprises Nef or an immunogenic fragment or derivative<br>
thereof and p24 Gag or an immunogenic fragment or derivative thereof, and (ii) p17 Gag<br>
or an immunogenic fragment or derivative thereof.<br>
In another embodiment the polypeptide construct according to the invention further<br>
comprises Pol or a derivative of Pol such as RT or an immunogenic fragment or<br>
derivative thereof. Particular fragments of RT that are suitable for use in the invention<br>
are fragments in which the RT is truncated at the C terminus, preferably such that they<br>
lack the carboxy terminal RNase H domain. One such fragment lacking the carboxy<br>
terminal Rnase H domain is the p51 fragment described herein.<br>
Preferably the RT or immunogenic fragment in the fusion proteins described herein is<br>
p66 RT or p51 RT.<br>
The RT component of the fusion protein or composition according to the invention<br>
optionally comprises a mutation at position 592, or equivalent mutation in strains other<br>
than HXB2, such that the methionine is removed by mutation to another residue e.g.<br>
lysine. The purpose of this mutation is to remove a site which serves as an internal<br>
initiation site in prokaryotic expression systems.<br><br>
The RT component also, or alternatively, comprises a mutation to remove the enzyme<br>
activity (reverse transcriptase). Thus K231 may be present instead of W.<br>
In fusion proteins according to the invention which comprise p24 and RT, it may be<br>
preferable that the p24 precedes the RT in the construct because when the antigens are<br>
expressed alone in E. coli better expression of p24 than of RT is observed.<br>
Preferred constructs according to the invention include the following:<br>
1.	p24-RT-Nef-p17<br>
2.	P24-RT*-Nef-p17<br>
3.	P24-p51RT-Nef-p17<br>
4.	P24-p51RT*-Nef-p17<br>
5.	P17-p51RT-Nef<br>
6.	p17-p51RT*-Nef<br>
7.	Nef-p17<br>
8.	Nef-p17 with linker<br>
9.	p17-Nef<br>
10.	p17-Nef with linker<br>
* represents RT methionine592 mutation to lysine<br>
The linker included in the constructs listed above may be any short amino acid sequence<br>
for decreasing potential interactions between the two fusion partners that it links together.<br>
The linker may be for example from 4-10 amino acids in length. For example, it may be<br>
a 6 amino acid sequence such as the GSGGGP sequence described herein in the<br>
examples.<br>
In another aspect the present invention provides a fusion protein of HIV antigens<br>
comprising at least four HIV antigens or immunogenic fragments, wherein the four<br>
antigens or fragments are or are derived from Nef, Pol and Gag. Preferably Gag is<br><br>
present as two separate components which are separated by at least one other antigen in<br>
the fusion. Preferably the Nef is full length Nef. Preferably the Pol is p66 or p51RT.<br>
Preferably the Gag is p 17 and p24 Gag. Other preferred features and properties of the<br>
antigen components of the fusion in this aspect of the invention are as described herein.<br>
Preferred embodiments of this aspect of the invention are the four component fusions as<br>
already listed above:<br>
1.	p24-RT-Nef-p17<br>
2.	P24-RT*-Nef-p17<br>
3.	p24-p51RT-Nef-p17<br>
4.	p24-P51RT*-Nef-p17<br>
The term "derived from" or "derivative" in relation to the HIV antigens included in the<br>
invention means that the antigens may have been altered in a limited way compared to<br>
their native counterparts. This includes point mutations which may change the properties<br>
of the protein for example by improving expression in prokaryotic systems or removing<br>
undesirable activity including undesirable enzyme activity. The point mutations<br>
described herein for RT are designed to achieve these things. However, the antigens<br>
must remain sufficiently similar to the native antigens such that they retain the antigenic<br>
properties desirable in a vaccine and thus they remain capable of raising an immune<br>
response against the native antigen. Whether or not a particular derivative raises such an<br>
immune response may be measured by a suitable immunological assay such as an ELISA<br>
(for antibody responses) or flow cytometry using suitable staining for cellular markers<br>
and cytokines (for cellular responses).<br>
The polypeptide constructs of HIV antigens according to the invention are capable of<br>
being expressed in in vitro systems including prokaryotic systems such as E. coli.<br>
Advantageously they can be purified by conventional purification methods.<br>
The fusions described herein are preferably soluble when expressed in a selected<br>
expression system, that is they are present in a substantial amount in the supernatant of a<br><br>
crude extract from the expression system. The presence of the fusion protein in the crude<br>
extract can be measured by conventional means such as running on an SDS gel,<br>
coomassie staining and checking the appropriate band by densitometric measurement.<br>
Fusion proteins according to the invention are preferably at least 50% soluble, more<br>
preferably at least 70% soluble, most preferably 90% soluble or greater as measured by<br>
the techniques described herein in the Examples. Techniques to improve solubility of<br>
recombinantly expressed proteins are known, for example in prokaryotic expression<br>
systems solubility is improved by lowering the temperature at which gene expression is<br>
induced.<br>
The fusion proteins described herein can be purified. In particular they can be purified<br>
while remaining soluble or significantly soluble.<br>
Immunogenic fragments as described herein will contain at least one epitope of the<br>
antigen and display HIV antigenicity and are capable of raising an immune response<br>
when presented in a suitable construct, such as for example when fused to other HIV<br>
antigens or presented on a carrier, the immune response being directed against the native<br>
antigen. Typically the immunogenic fragments contain at least 20, preferably 50, more<br>
preferably 100 contiguous amino acids from the HIV antigen.<br>
The invention provides in a further aspect polynucleotides encoding the polypeptides according to the invention.<br>
Polynucleotides according to the invention may be used as polynucleotide vaccines. The<br>
polynucleotides may be present within any of a variety of delivery systems known to<br>
those of ordinary skill in the art, including nucleic acid expression systems such as<br>
plasmid DNA, bacterial and viral expression systems. Numerous gene delivery<br>
techniques are well known in the art, such as those described by Rolland, Crit. Rev.<br>
Therap. Drug Carrier Systems 15:143-198, 1998 and references cited therein.<br>
Appropriate nucleic acid expression systems contain the necessary DNA sequences for<br>
expression in the patient (such as a suitable promoter and terminating signal). When the<br><br>
expression system is a recombinant live microorganism, such as a virus or bacterium, the<br>
gene of interest can be inserted into the genome of the live recombinant virus or<br>
bacterium. Inoculation and in vivo infection with this live vector will lead to in vivo<br>
expression of the antigen and induction of immune responses. Viruses and bacteria used<br>
for this purpose are for instance: poxviruses (e.g; vaccinia, fowlpox, canarypox,<br>
modified poxviruses e.g. Modified Virus Ankara (MVA)), alphaviruses (Sindbis virus,<br>
Semliki Forest Virus, Venezuelan Equine Encephalitis Virus), flavi viruses (yellow fever<br>
virus, Dengue virus, Japanese encephalitis virus), adenoviruses, adeno-associated virus,<br>
picornaviruses (poliovirus, rhinovirus), herpesviruses (varicella zoster virus, etc),<br>
morbilliviruses (e.g. measles), Listeria, Salmonella, Shigella, Neisseria, BCG. These<br>
viruses and bacteria can be virulent, or attenuated in various ways in order to obtain live<br>
vaccines. Such live vaccines also form part of the invention.<br>
A preferred measles vector for use as a live vector according to the invention is the<br>
Schwartz strain or a strain derived therefrom.<br>
A preferred adenovirus for use as a live vector is a low sero-βrevalent human adenovirus<br>
such as Ad5 or Ad35 or a non-human originating adenovirus such as a non-human<br>
primate adenovirus such as a simian adenovirus. Such low sero-βrevalent human or<br>
similar adenoviruses will have less than 60, typically less than 50% sero-βrevelance in<br>
the population. Preferably the vectors are replication defective. Typically these viruses<br>
contain an E1 deletion and can be grown on cell lines that are transformed with an E1<br>
gene. Preferred simian adenoviruses are viruses isolated from chimpanzee. In particular<br>
C68 (also known as Pan 9) (See US patent No 6083 716) and Pan 5, 6 and Pan 7 (WO<br>
03/046124) are preferred for use in the present invention. These vectors can be<br>
manipulated to insert a heterologous polynucleotide according to the invention such that<br>
the polypeptides according to the invention maybe expressed. The use, formulation and<br>
manufacture of such recombinant adenoviral vectors is described in detail in WO<br>
03/046142.<br><br>
Thus, the Nef, p 17 and p24 Gag and RT of a preferred vaccine according to the invention<br>
may be provided in the form of a polynucleotide encoding the desired polypeptide.<br>
Polynucleotides according to the invention may be used to express the encoded<br>
polypeptides in a selected expression system. At least one of the HIV antigens, for<br>
example the RT, may be encoded by a codon optimized sequence in the polynucleotide,<br>
that is to say the sequence has been optimized for expression in a selected recombinant<br>
expression system such as E. coli.<br>
In another aspect the invention provides a p51 RT polypeptide or derivative thereof or a<br>
polynucleotide encoding it, preferably codon optimized for expression in a suitable<br>
expression system, particularly a prokaryotic system such as E. coli.<br>
The p51 RT polypeptide or polynucleotide may be used alone; or in combination with a<br>
polypeptide or polynucleotide construct according to the invention. Thus in a further<br>
aspect the invention provides a composition comprising (i) a polypeptide which<br>
comprises Nef or a fragment containing a Nef epitope and p17 Gag and/or p24 Gag,<br>
wherein when both p 17 and p24 Gag are present there is at least one HIV antigen or<br>
immunogenic fragment between them and (ii) a p51 RT polypeptide. The invention<br>
further provides polynucleotides encoding these.<br>
According to this embodiment (i) may be selected from for example:<br>
1.	Nef-p17<br>
2.	Nef- p17 with linker<br>
3.	p17-Nef<br>
4.	p17 - Nef with linker<br>
Preferably Nef is full length Nef. Preferably p17 is full length p17.<br>
The polypeptides and polynucleotides according to the invention may be combined with<br>
other antigens or polynucleotides encoding other antigens. In particular, this may include<br>
HIV env proteins or fragments or derivatives thereof. Preferred forms of env are gp120,<br><br>
gp140 and gp160. The env may be for example the envelope protein described in WO<br>
00/07631 from an HIV-1 clade B envelope clone known as R2, or a fragment or<br>
derivative thereof. Thus the invention further provides a composition comprising any of<br>
the polypeptides or polypeptide compositions according to the invention, together with an<br>
HIV env protein or fragment or derivative thereof. Similarly the invention provides a<br>
composition comprising a polynucleotide or polynucleotides encoding a polypeptides or<br>
polypeptides according to the invention and a polynucleotide encoding an HIV env<br>
protein or fragment or derivative thereof.<br>
The invention further provides methods of preparing the polypeptides described herein<br>
which method comprises expressing a polynucleotide encoding the polypeptide in a<br>
suitable expression system, particularly a prokaryotic system such as E. coli and<br>
recovering the expressed polypeptide. Preferably expression is induced at a low<br>
temperature, that is a temperature below 37°, to promote the solubility of the polypeptide.<br>
The invention further provides a process for purifying a polypeptide as described herein,<br>
which process comprises:<br>
i). providing a composition comprising the unpurified polypeptide;<br>
ii). Subjecting the composition to at least two chromatographic steps;<br>
iii). Optionally carboxyamidating the polypeptide;<br>
iv) Performing a buffer exchange step to provide the protein in a suitable<br>
buffer for a pharmaceutical formulation.<br>
The carboxyamidation may be performed between the two chromatographic steps. The<br>
carboxyamidation step may be performed using iodoacetimide.<br>
In one example, the process according to the invention uses no more than two<br>
chromatographic steps.<br><br>
The invention further provides pharmaceutical compositions and immunogenic<br>
compositions and vaccines comprising the polypeptides and polynucleotides according to<br>
the invention, in combination with a pharmaceutically acceptable adjuvant or carrier.<br>
Vaccines according to the invention may be used for prophylactic or therapeutic<br>
immunization against HIV.<br>
The invention further provides the use of the polypeptides and polypeptide compositions<br>
and the polynucleotides and polynucleotide compositions as described herein, in the<br>
manufacture of a vaccine for prophylactic or therapeutic immunization against HIV.<br>
The vaccine of the present invention will contain an immunoprotective or<br>
immunotherapeutic quantity of the polypeptide and/or polynucleotide antigens and may<br>
be prepared by conventional techniques.<br>
Vaccine preparation is generally described in New Trends and Developments in<br>
Vaccines, edited by Voller et al., University Park Press, Baltimore, Maryland, U.S.A.<br>
1978. Encapsulation within liposomes is described, for example, by Fullerton, U.S.<br>
Patent 4,235,877. Conjugation of proteins to macrornolecules is disclosed, for example,<br>
by Likhite, U.S. Patent 4,372,945 and by Armor et al., U.S. Patent 4,474,757.<br>
The amount of protein in the vaccine dose is selected as an amount which induces an<br>
immunoprotective response without significant, adverse side effects in typical vaccinees.<br>
Such amount will vary depending upon which specific immunogen is employed and the<br>
vaccination regimen that is selected. Generally, it is expected that each dose will<br>
comprise 1-1000 ug of each protein, preferably 2-200 ug, most preferably 4-40 ug of the<br>
polypeptide fusion. An optimal amount for a particular vaccine can be ascertained by<br>
standard studies involving observation of antibody titres and other immune responses in<br>
subjects. Following an initial vaccination, subjects may receive a boost in about 4 weeks,<br>
and a subsequent second booster immunisation.<br><br>
The proteins of the present invention are preferably adjuvanted in the vaccine formulation<br>
of the invention. Adjuvants are described in general in Vaccine Design - the Subunit and<br>
Adjuvant Approach, edited by Powell and Newman, Plenum Press, New York, 1995.<br>
Suitable adjuvants include an aluminium salt such as aluminium hydroxide or aluminium<br>
phosphate, but may also be a salt of calcium, iron or zinc, or may be an insoluble<br>
suspension of acylated tyrosine, or acylated sugars, cationically or anionically derivatised<br>
polysaccharides, or polyphosphazenes.<br>
In the formulation of the invention it is preferred that the adjuvant composition induces a<br>
preferential Th1 response. However it will be understood that other responses, including<br>
other humoral responses, are not excluded.<br>
An immune response is generated to an antigen through the interaction of the antigen<br>
with the cells of the immune system. The resultant immune response may be broadly<br>
distinguished into two extreme catagories, being humoral or cell mediated immune<br>
responses (traditionally characterised by antibody and cellular effector mechanisms of<br>
protection respectively). These categories of response have been termed Th1-type<br>
responses (cell-mediated response), and Th2-type immune responses (humoral response).<br>
Extreme Th1-type immune responses may be characterised by the generation of antigen<br>
specific, haplotype restricted cytotoxic T lymphocytes, and natural killer cell responses.<br>
In mice Th1-type responses are often characterised by the generation of antibodies of the<br>
IgG2a subtype, whilst in the human these correspond to IgG1 type antibodies. Th2-type<br>
immune responses are characterised by the generation of a broad range of<br>
immunoglobulin isotypes including in mice IgG1, IgA, and IgM.<br>
It can be considered that the driving force behind the development of these two types of<br>
immune responses are cytokines, a number of identified protein messengers which serve<br>
to help the cells of the immune system and steer the eventual immune response to either a<br>
Th1 or Th2 response. Thus high levels of Th1-type cytokines tend to favour the<br><br>
induction of cell mediated immune responses to the given antigen, whilst high levels of<br>
Th2-type cytokines tend to favour the induction of humoral immune responses to the<br>
antigen.<br>
It is important to remember that the distinction of Th1 and Th2-type immune responses is<br>
not absolute. In reality an individual will support an immune response which is described<br>
as being predominantly Th1 or predominantly Th2. However, it is often convenient to<br>
consider the families of cytokines in terms of that described in murine CD4 +ve T cell<br>
clones by Mosmann and Coffman (Mosmann, T.R. and Coffman. R.L. (1989) TH1 and<br>
TH2 cells: different patterns of lymphokine secretion lead to different functional<br>
properties. Annual Review of Immunology, 7, pl 45-173). Traditionally, Th1-type<br>
responses are associated with the production of the INF-y and IL-2 cytokines by T-<br>
lymphocytes. Other cytokines often directly associated with the induction of Th1-type<br>
immune responses are not produced by T-cells, such as IL-12. In contrast, Th2- type<br>
responses are associated with the secretion of IL-4, IL-5, IL-6, IL-10 and tumour necrosis<br>
factor-β(TNF-β).<br>
It is known that certain vaccine adjuvants are particularly suited to the stimulation of<br>
either Th1 or Th2 - type cytokine responses. Traditionally the best indicators of the<br>
Th1:Th2 balance of the immune response after a vaccination or infection includes direct<br>
measurement of the production of Th1 or Th2 cytokines by T lymphocytes in vitro after<br>
restimulation with antigen, and/or the measurement of the IgG1:IgG2a ratio of antigen<br>
specific antibody responses.<br>
Thus, a Th1-type adjuvant is one which stimulates isolated T-cell populations to produce<br>
high levels of Th1-type cytokines when re-stimulated with antigen in vitro, and induces<br>
antigen specific immunoglobulin responses associated with Th1-type isotype.<br>
Preferred Th1-type immunostimulants which may be formulated to produce adjuvants<br>
suitable for use in the present invention include and are not restricted to the following.<br><br>
Monophosphoryl lipid A, in particular 3-de-O-acylated monophosphoryl lipid A (3D-<br>
MPL), is a preferred Th1-type immunostimulant for use in the invention. 3D-MPLisa<br>
well known adjuvant manufactured by Ribi Immunochem, Montana. Chemically it is<br>
often supplied as a mixture of 3-de-O-acylated monophosphoryl lipid A with either 4, 5,<br>
5 or 6 acylated chains. It can be purified and prepared by the methods taught in IN 156144A1, IN 157241 A<br>
and IN 157910A1 t which reference aiso discloses the preparation of diphosphoryl lipid A, and 3-<br>
O-deacylated variants thereof. Other purified and synthetic lipopolysaccharides have<br>
been described (US 6,005,099 and EP 0 729 473 B1; Hilgers et al., 1986,<br>
Int.Arch.Allergy.Immunol, 79(4):392-6; Hilgers et al., 1987, Immunology, 60(l):141-6;<br>
10 and EP 0 549 074 B1). A preferred form of 3 D-MPL is in the form of a particulate<br>
formulation having a small particle size less than 0.2uxn in diameter, and its method of<br>
manufacture is disclosed in EP 0 689 454.<br>
Saponins are also preferred Th1 imrnunostimulants in accordance with the invention.<br>
15 Saponins are well known adjuvants and are taught in: Lacaille-Dubois, M and Wagner H.<br>
(1996. A review of the biological and pharmacological activities of saponins.<br>
Phytomedicine vol 2 pp 363-386). For example, Quil A (derived from the bark of the<br>
South American tree Quillaja Saponaria Molina), and fractions thereof, are described in<br>
US 5,057,540 and "Saponins as vaccine adjuvants", Kensil, C. R., Crit Rev Ther Drug<br>
20 Carrier Syst, 1996, 12 (l-2):l-55; and EP 0 362 279 B1. The haemolytic saponins QS21<br>
and QS17 (HPLC purified fractions of Quil A) have been described as potent systemic<br>
adjuvants, and the method of their production is disclosed in US Patent No. 5,057,540<br>
and EP 0 362 279 B1. Also described in these references is the use of QS7 (a non-<br>
haemolytic fraction of Quil-A) which acts as a potent adjuvant for systemic vaccines. Use<br>
25 of QS21 is further described in Kensil et al. (1991. J. Immunology vol 146, 431-437).<br>
Combinations of QS21 and polysorbate or cyclodextrin are also known (WO 99/10008).<br>
Particulate adjuvant systems comprising fractions of QuilA, such as QS21 and QS7 are<br>
described in WO 96733739 and WO 96/11711. One such system is known as an Iscorn<br>
and may contain one or more saponins.<br>
30<br><br>
Another preferred immunostimulant is an irnmunostimulatory oligonucleotide containing<br>
unmethylated CpG dinucleotides ("CpG"). CpG is an abbreviation for cytosine-<br>
guanosine dinucleotide motifs present in DNA. CpG is known in the art as being an<br>
adjuvant when administered by both systemic and mucosal routes (WO 96/02555, EP<br>
468520, Davis et al, J.Immunol, 1998, 160(2):870-876; McCluskie and Davis,<br>
JJmmunol, 1998, 161(9):4463-6). Historically, it was observed that the DNA fraction of<br>
BCG could exert an anti-tumour effect. In further studies, synthetic oligonucleotides<br>
derived from BCG gene sequences were shown to be capable of inducing<br>
irnmunostimulatory effects (both in vitro and in vivo). The authors of these studies<br>
concluded that certain palindromic sequences, including a central CG motif, carried this<br>
activity. The central role of the CG motif in immunostimulation was later elucidated in a<br>
publication by Krieg, Nature 374, p546 1995. Detailed analysis has shown that the CG<br>
motif has to be in a certain sequence context, and that such sequences are common in<br>
bacterial DNA but are rare in vertebrate DNA. The irnmunostimulatory sequence is<br>
often: Purine, Purine, C, G, pyrimidine, pyrimidine; wherein the CG motif is not<br>
methylated, but other unmethylated CpG sequences are known to be irnmunostimulatory<br>
and may be used in the present invention.<br>
In certain combinations of the six nucleotides a palindromic sequence is present. Several<br>
of these motifs, either as repeats of one motif or a combination of different motifs, can be<br>
. present in the same oligonucleotide. The presence of one or more of these<br>
irnmunostimulatory sequences containing oligonucleotides can activate various immune<br>
subsets, including natural killer cells (which produce interferon γ and have cytolytic<br>
activity) and macrophages (Wooldrige et al Vol 89 (no. 8), 1977). Other unmethylated<br>
CpG containing sequences not having this consensus sequence have also now been<br>
shown to be immunomodulatory.<br>
CpG when formulated into vaccines, is generally administered in free solution together<br>
with free antigen (WO 96/02555; McCluskie and Davis, supra) or covalently conjugated<br>
to an antigen (WO 98/16247), or formulated with a carrier such as aluminium hydroxide<br><br>
((Hepatitis surface antigen) Davis et al. supra; Brazolot-Millan et al.,<br>
Proc.Natl.Acad.Sci., USA, 1998, 95(26), 15553-8).<br>
Such immunostimulants as described above may be formulated together with carriers,<br>
such as for example liposomes, oil in water emulsions, and or metallic salts, including<br>
aluminium salts (such as aluminium hydroxide). For example, 3D-MPL may be<br>
formulated with aluminium hydroxide (EP 0 689 454) or oil in water emulsions (WO<br>
95/17210); QS21 may be advantageously formulated with cholesterol containing<br>
liposomes (WO 96/33739), oil in water emulsions (WO 95/17210) or alum (WO<br>
98/15287); CpG may be formulated with alum (Davis et al. supra ; Brazolot-Millan<br>
supra) or with other cationic carriers.<br>
Combinations of immunostimulants are also preferred, in particular a combination of a<br>
monophosphoryl lipid A and a saponin derivative (WO 94/00153; WO 95/17210; WO<br>
96/33739; WO 98/56414; WO 99/12565; WO 99/11241), more particularly the<br>
combination of QS21 and 3D-MPL as disclosed in WO 94/00153. Alternatively, a<br>
combination of CpG plus a saponin such as QS21 also forms a potent adjuvant for use in<br>
the present invention. Alternatively the saponin may be formulated in a liposome or in an<br>
Iscorn and combined with an immunostimulatory oligonucleotide.<br>
Thus, suitable adjuvant systems include, for example, a combination of monophosphoryl<br>
lipid A, preferably 3D-MPL, together with an aluminium salt.<br>
An enhanced system involves the combination of a monophosphoryl lipid A and a<br>
saponin derivative particularly the combination of QS21 and 3D-MPL as disclosed in<br>
WO 94/00153, or a less reactogenic composition where the QS21 is quenched in<br>
cholesterol containing liposomes (DQ) as disclosed in WO 96/33739. This combination<br>
may additionally comprise an immunostimulatory oligonucleotide.<br>
A particularly potent adjuvant formulation involving QS21, 3D-MPL &amp; tocopherol in an<br>
oil in water emulsion is described in WO 95/17210 and is another preferred formulation<br>
for use in the invention.<br><br>
Another preferred formulation comprises a CpG oligonucleotide alone or together with an<br>
aluminium salt.<br>
In a further aspect of the present invention there is provided a method of manufacture of a<br>
vaccine formulation as herein described, wherein the method comprises admixing a<br>
polypeptide according to the invention with a suitable adjuvant.<br>
Particularly preferred adjuvant combinations for use in the formulations according to the<br>
invention are as follows:<br>
i)	3D-MPL + QS21 in a liposome<br>
ii)	Alum + 3D-MPL<br>
iii) Alum + QS21 in a liposome + 3D-MPL<br>
iv)	Alum + CpG<br>
v)	3D-MPL + QS21 + oil in water emulsion<br>
vi)	CpG<br>
Administration of the pharmaceutical composition may take the form of one or of more<br>
than one individual dose, for example as repeat doses of the same polypeptide containing<br>
composition, or in a heterologous "prime-boost" vaccination regime. A heterologous<br>
prime-boost regime uses administration of different forms of vaccine in the prime and the<br>
boost, each of which may itself include two or more administrations. The priming<br>
composition and the boosting composition will have at least one antigen in common,<br>
although it is not necessarily an identical form of the antigen, it may be a different form<br>
of the same antigen.<br>
Prime boost immunisations according to the invention may be performed with a<br>
combination of protein and DNA-based formulations. Such a strategy is considered to be<br>
effective in inducing broad immune responses. Adjuvanted protein vaccines induce<br>
mainly antibodies and T helper immune responses, while delivery of DNA as a plasmid<br>
or a live vector induces strong cytotoxic T lymphocyte (CTL) responses. Thus, the<br><br>
combination of protein and DNA vaccination will provide for a wide variety of immune<br>
responses. This is particularly relevant in the context of HIV, since both neutralising<br>
antibodies and CTL are thought to be important for the immune defence against HIV.<br>
In accordance with the invention a schedule for vaccination may comprise the sequential<br>
("prime-boost") or simultaneous administration of polypeptide antigens and DNA<br>
encoding the polypeptides. The DNA may be delivered as naked DNA such as plasmid<br>
DNA or in the form of a recombinant live vector, e.g. a poxvirus vector, an adenovirus<br>
vector, a measles virus vector or any other suitable live vector. Protein antigens may be<br>
injected once or several times followed by one or more DNA administrations, or DNA<br>
may be used first for one or more administrations followed by one or more protein<br>
immunisations.<br>
A particular example of prime-boost immunisation according to the invention involves<br>
priming with DNA in the form of a recombinant live vector such as a modified poxvirus<br>
vector, for example Modified Virus Ankara (MVA) or an alphavirus, for example<br>
Venezuelian Equine Encephalitis Virus, or an adenovirus vector, or a measles virus<br>
vector, followed by boosting with a protein, preferably an adjuvanted protein.<br>
Thus the invention further provides a pharmaceutical kit comprising:<br>
a)	a composition comprising a polypeptide comprising Nef or an<br>
immunogenic fragment or derivative thereof and p17 and/or p24 Gag or an<br>
immunogenic fragment or derivative thereof, wherein when both p17 and p24<br>
Gag are present there is at least one HIV antigen or immunogenic fragment or<br>
derivative between them, together with a pharmaceutically acceptable excipient;<br>
and<br>
b)	a composition comprising a polynucleotide encoding one or more<br>
of Nef and Gag or an immunogenic fragment or derivative of Nef or Gag<br>
containing a Nef or Gag epitope present in the polypeptide of a), together with a<br>
pharmaceutically acceptable excipient.<br><br>
Preferably the polypeptide of a) further comprises RT or an immunogenic fragment or<br>
derivative thereof such as p51RT.<br>
In an alternative embodiment the pharmaceutical kit comprises:<br>
a)	a composition comprising a polynucleotide encoding a polypeptide<br>
comprising Nef or an immunogenic fragment or derivative thereof and p17 and/or<br>
p24 Gag or an immunogenic fragment or derivative thereof, wherein when both<br>
p17 and p24 Gag are present there is at least one HIV antigen or immunogenic<br>
fragment or derivative between them, together with a pharmaceutically acceptable<br>
excipient; and<br>
b)	a composition comprising a polypeptide comprising one or more of<br>
Nef and Gag or an immunogenic fragment or derivative of Nef or Gag containing<br>
a Nef or Gag epitope present in the polypeptide of a), together with a<br>
pharmaceutically acceptable excipient.<br>
Preferably the polynucleotide of a) encodes a polypeptide which further comprises RT or<br>
an immunogenic fragment or derivative thereof such as p51RT.<br>
Preferred polypeptides and polynucleotides for use in a prime/boost kit according to the<br>
invention are the polypeptides and polynucleotides as described herein. Thus, the protein<br>
component of a protein/DNA type prime boost approach may be any of the preferred<br>
fusion proteins described herein. Likewise, the DNA component may be a<br>
polynucleotide encoding any of the preferred proteins.<br>
Thus for example, the p24 - RT - Nef- p17, p24 - RT* - Nef- p17, p24 - p51RT - Nef<br>
-p17,p24-p51RT*-Nef-p17,p17-p51RT-Neforp17-p51RT*-Nef fusions or<br>
any of the p17-Nef fusions as described herein may be provided in a prime boost kit<br>
wherein the priming composition comprises the fusion protein and the boosting<br>
composition comprises a polynucleotide encoding the fusion protein, or the priming<br><br>
composition comprises the polynucleotide and the boosting composition comprises the<br>
fusion protein.<br>
Both the priming composition and the boosting composition may be delivered in more<br>
than one dose. Furthermore the initial priming and boosting doses may be followed up<br>
with further doses which may be alternated to result in e.g. a DNA plasmid prime /<br>
protein boost / further DNA plasmid dose / further protein dose.<br>
By codon optimisation it is meant that the polynucleotide sequence, is optimised to<br>
resemble the codon usage of genes in the desired expression system, for example a<br>
prokaryotic system such as E. coli. In particular, the codon usage in the sequence is<br>
optimised to resemble that of highly expressed E. coli genes.<br>
The purpose of codon optimizing for expression in a recombinant system according to the<br>
invention is twofold: to improve expression levels of the recombinant product and to<br>
render expression products more homogeneous (obtain a more homogeneous expression<br>
pattern). Improved homogeneity means that there are fewer irrelevant expression<br>
products such as truncates. Codon usage adaptation to E.coli expression can also<br>
eliminate the putative "frame-shift" sequences as well as premature termination and/or<br>
internal initiation sites.<br>
The DNA code has 4 letters (A, T, C and G) and uses these to spell three letter "codons"<br>
which represent the amino acids the proteins encoded in an organism's genes. The linear<br>
sequence of codons along the DNA molecule is translated into the linear sequence of<br>
amino acids in the protein(s) encoded by those genes. The code is highly degenerate,<br>
with 61 codons coding for the 20 natural amino acids and 3 codons representing "stop"<br>
signals. Thus, most amino acids are coded for by more than one codon - in fact several<br>
are coded for by four or more different codons.<br>
Where more than one codon is available to code for a given amino acid, it has been<br>
observed that the codon usage patterns of organisms are highly non-random. Different<br><br>
species show a different bias in their codon selection and, furthermore, utilisation of<br>
codons may be markedly different in a single species between genes which are expressed<br>
at high and low levels. This bias is different in viruses, plants, bacteria and mammalian<br>
cells, and some species show a stronger bias away from a random codon selection than<br>
others. For example, humans and other mammals are less strongly biased than certain<br>
bacteria or viruses. For these reasons, there is a significant probability that a viral gene<br>
from a mammalian virus expressed in E. coli, or a foreign or recombinant gene expressed<br>
in mammalian cells will have an inappropriate distribution of codons for efficient<br>
expression. It is believed that the presence in a heterologous DNA sequence of clusters<br>
of codons or an abundance of codons which are rarely observed in the host in which<br>
expression is to occur, is predictive of low heterologous expression levels in that host.<br>
In the polynucleotides of the present invention, the codon usage pattern may thus be<br>
altered from that typical of human immunodeficiency viruses to more closely represent<br>
the codon bias of the target organism, e.g. E. coli.<br>
There are a variety of publicly available programs useful for codon optimization, for<br>
example "CalcGene" (Hale and Thompson, Protein Expression and Purification 12: 185-<br>
189 (1998).	<br><br>
EXAMPLES<br>
Example 1: Construction and expression of HIV-1 p24 - RT - Nef - p17 fusion F4<br>
and F4 codon optimized (co)<br>
1. F4 Non-codon-optimised<br>
HIV-1 gag p24 (capsid protein) and p17 (matrix protein), the reverse transcriptase and<br>
Nef proteins were expressed in E.coli B834 strain (B834 (DE3) is a methionine<br>
auxotroph parent of BL21 (DE3)), under the control of the bacteriophage T7 promoter<br>
(pET expression system).<br>
They were expressed as a single fusion protein containing the complete sequence of the<br>
four proteins. Mature p24 coding sequence comes from HIV-1 BH10 molecular clone,<br>
mature p17 sequence and RT gene from HXB2 and Nef gene from the BRU isolate.<br>
After induction, recombinant cells expressed significant levels of the p24-RT-Nef-p17<br>
fusion that amounted to 10% of total protein.<br>
When cells were grown and induced at 22°C, the p24-RT-Nef-p17 fusion protein was<br>
confined mainly to the soluble fraction of bacterial lysates (even after freezing/thawing).<br>
When grown at 30°C, around 30% of the recombinant protein was associated with the<br>
insoluble fraction.<br>
The fusion protein p24-RT-Nef-p17 is made up of 1136 amino acids with a molecular<br>
mass of approximately 129 kDa. The full-length protein migrates to about 130 kDa on<br>
SDS gels. The protein has a theoretical isoeleectric point (pl) of 7.96 based on its amino<br>
acid sequence, confirmed by 2D-gel electrophoresis.<br>
Details of the recombinant plasmid:<br>
name:	pRIT15436 (or lab name pET28b/p24-RT-Nef-p17 )<br>
host vector: pET28b<br><br>
replicon:	colE1<br>
selection: kanamycin<br>
promoter: T7<br>
insert:	p24-RT-Nef-p17 fusion gene.<br>
Details of the recombinant protein:<br>
p24-RT-Nef-p17 fusion protein : 1136 amino acids.<br>
N-term - p24: 232a.a. - hinge:2a.a. - RT: 562a.a. -hinge:2a.a. - Nef: 206a.a. -<br>
- P17: 132a.a. -C-term<br><br><br><br>
p24 sequence is in bold<br>
Nef sequence is underlined<br>
Boxes: nucleotides introduced by genetic construction<br>
Amino-Acid sequence<br><br><br>
Expression of the recombinant protein:<br>
In pET plasmid, the target gene (p24-RT-Nef-p17) is under control of the strong<br>
bacteriophage T7 promoter. This promoter is not recognized by E.coli RNA polymerase<br>
and is dependent on a source of T7 RNA polymerase in the host cell. B834 (DE3) host<br><br>
cell contains a chromosomal copy of the T7 RNA polymerase gene under lacUV5 control<br>
and expression is induced by the addition of IPTG to the bacterial culture.<br>
Pre-cultures were grown, in shake flasks, at 37°C to mid-log phase (A620:0.6) and then<br>
stored at 4°C overnight (to avoid stationary phase cultures). Cultures were grown in LBT<br>
medium supplemented with 1% glucose and 50 µg/mlg/ml kanamycin. Addition of glucose to<br>
the growth medium has the advantage to reduce the basal recombinant protein expression<br>
(avoiding cAMP mediated derepression of lacUV5 promoter)<br>
Ten ml of cultures stored overnight at 4°C were used to inoculate 200 ml of LBT medium<br>
(without glucose) containing kanamycin. Cultures were grown at 30°C and 22°C and<br>
when O.D.620 reached 0.6, IPTG was added (ImM final). Cultures were incubated for<br>
further 3, 5 and 18 hours (overnight). Samples were collected before and after 3, 5 and 18<br>
hours induction.<br>
Extract preparation was as follows:<br>
Cell pellets were suspended in breaking buffer* (at a theoretical O.D. of 10) and<br>
disrupted by four passages in French press (at 20.000psi or 1250 bars). Crude extracts<br>
(T) were centrifuged at 20.000g for 30 min to separate the soluble (S) and insoluble (P)<br>
fractions.<br>
Breaking buffer: 50mM Tris-HCL pH 8.0, ImM EDTA, ImM DTT + protease<br>
inhibitors cocktail (Complete/Boerhinger).<br>
SDS-PAGE and Western B1ot analysis:<br>
Fractions corresponding to insoluble pellet (P), supernatant (S) and crude extract (T) were<br>
run on 10 % reducing SDS-PAGE. p24-RT-Nef -p17recombinant was detected by<br>
Coomassie blue staining and on Western blot (WB).<br>
Coomassie staining: p24-RT-Nef-βl 7 protein appears as:<br>
one band at ± 130 kDa (fitting with calculated MW)<br><br>
MW theoretical:	128.970 Daltons<br>
MW apparent:	130 kDa<br>
Western blot analysis:<br>
Reagents = - Monoclonal antibody to RT (p66/p51)<br>
Purchased from ABI (Advanced Biotechnologies)<br>
dilution: 1/5000<br>
-Alkaline phosphatase-conjugate anti-mouse antibody<br>
dilution: 1/7500<br>
Expression level: - Very strong p24-RT-Nef-p 17 specific band after 20h<br>
induction at 22°C, representing up to 10% of total<br>
protein (See Figure 1 A).<br>
Recombinant protein "solubility":<br>
"Fresh" cellular extracts (T.S,P fractions): With growth/induction at 22°C/20h, almost all<br>
p24-RT-Nef-p17 fusion protein is recovered in the soluble fraction of cellular extract<br>
(Figure 1A). With growth/induction at 30°C/20h, around 30% of p24-RT-Nef-p17<br>
protein is associated with the insoluble fraction (Figure 1A).<br>
"Freezing/thawing" (S2, P2 fractions):<br>
Soluble (S1) fraction (20h induction at 22°C) conserved at -20°C. Thawed and<br>
centrifuged at 20.000g/30 min : S2 and P2 (resuspended in 1/10 vol.)<br>
Breaking buffer with DTT : almost all p24-RT-Nef-p17 fusion<br>
protein still soluble (only 1-5 % precipitated) (see Figure 1B)<br>
Breaking buffer without DTT: 85-90 % of p24-RT-Nef-p17 still<br>
soluble (Figure 1B)<br>
Figures:<br>
Figure 1A - Coomassie staining and western blot.<br><br>
Figure 1B -p24-RT-Nef-p17 solubility assay<br>
The F4 protein was purified using purification method I in Example 7.<br>
The cell growth and induction conditions and cellular extracts preparation for the<br>
examples which follow are as described in Example 1 unless other conditions are<br>
specified (e.g. temperature, composition of breaking buffer).<br>
2. F4 codon-optimised<br>
The following polynucleotide sequence is codon optimized such that the codon usage<br>
resembles the codon usage in a highly expressed gene in E.coli. The amino acid sequence<br>
is identical to that given above for F4 non-codon optimized.<br><br><br><br>
p24 sequence is in bold<br>
Nef sequence is underlined<br>
Boxes: nucleotides introduced by genetic construction<br><br>
The procedures used in relation to F4 non-codon optimized were applied for the codon-<br>
optimised sequence.<br>
Details of the recombinant plasmid:<br>
name:	pRIT15513 (lab name: pET28b/p24-RT-Nef-p17 )<br>
host vector:	pET28b<br>
replicon:	colE1<br>
selection:	kanamycin<br>
promoter:	T7<br>
insert:	p24-RT-Nef-p17 fusion gene, codon-optimized<br>
The F4 codon-optimised gene was expressed in E. coli BLR(DE3) cells, a recA"<br>
derivative of B834(DE3) strain. RecA mutation prevents the putatitve production of<br>
lambda phages.<br>
Pre-cultures were grown, in shake flasks, at 37°C to mid-log phase (A620:0.6) and then<br>
stored at 4°C overnight (to avoid stationary phase cultures).<br>
Cultures were grown in LBT medium supplemented with 1% glucose and 50 µg/ml<br>
kanamycin. Addition of glucose to the growth medium has the advantage to reduce the<br>
basal recombinant protein expression (avoiding cAMP mediated derepression of lacUV5<br>
promoter).<br>
Ten ml of cultures stored overnight at 4°C were used to inoculate 200 ml of LBT<br>
medium (without glucose) containing kanamycin. Cultures were grown at 37°C and<br>
when O.D.620 reached 0.6, IPTG was added (1mM final). Cultures were incubated for<br>
further 19 hours (overnight), at 22°C. Samples were collected before and 19 hours<br>
induction.<br>
Extract preparation was as follows:<br>
Cell pellets were resuspended in sample buffer (at a theoretical O.D. of 10), boiled and<br>
directly loaded on SDS-PAGE.<br><br>
SDS-PAGE and Western B1ot analysis:<br>
Crude extracts samples were run on 10 % reducing SDS-PAGE.	<br>
p24-RT-Nef-p17 recombinant protein is detected by Coomassie blue staining (Figure 2)<br>
and on Western blot.<br>
Coomassie staining: p24-RT-Nef-p17 protein appears as:<br>
one band at ± 130 kDa (fitting with calculated MW)<br>
MW theoretical:	128.967 Daltons<br>
MW apparent:	130 kDa<br>
Western blot analysis:<br>
Reagents = - Rabbit polyclonal anti RT (rabbit P03L16)<br>
dilution: 1/10.000<br>
-	Rabbit polyclonal anti Nef-Tat (rabbit 388)<br>
dilution 1/10.000<br>
-	Alkaline phosphatase-conjugate anti- rabbit antibody .<br>
dilution: 1/7500<br>
After induction at 22°C over 19 hours, recombinant BLR(DE3) cells expressed the F4<br>
fusion at a very high level ranging from 10-15% of total protein.<br>
In comparison with F4 from the native gene, the F4 recombinant product profile from the<br>
codon-optimised gene is slightly simplified. The major F4-related band at 60 kDa, as<br>
well as minor bands below, disappeared (see Figure 2). Compared to the B834(DE3)<br>
recombinant strain expressing F4, the BLR(DE3) strain producing F4co has the following<br>
advantages: higher production of F4 full-length protein, less complex band pattern of<br>
recombinant product.<br><br>
Example 2: Construction and expression of PS1 RT (truncated, codon-optimised<br>
RT)<br>
The RT/p66 region between amino acids 428-448 is susceptible to E.coli proteases. The<br>
P51 construct terminates at Leu 427 resulting in the elimination of RNaseH domain (see<br>
RT sequence alignment in Figure 3).<br>
The putative E.coli "frameshift" sequences identified in RT native gene sequence were<br>
also eliminated (by codon-optimization of p51 gene).<br>
p51 synthetic gene design/construction:<br>
The sequence of the synthetic p51 gene was designed according to E.coli codon usage.<br>
Thus it was codon optimized such that the codon usage resembles the codon usage in a<br>
highly expressed gene in E.coli. The synthetic gene was constructed as follows: 32<br>
oligonucleotides were assembled in a single-step PCR. In a second PCR the full-length<br>
assembly was amplified using the ends primers and the resulting PCR product was cloned<br>
into pGEM-T intermediate plasmid. After correction of point errors introduced during<br>
gene synthesis, the p51 synthetic gene was cloned into pET29a expression plasmid. This<br>
recombinant plasmid was used to transform B834 (DE3) cells.<br><br><br><br>
Boxes: amino-acids introduced by genetic construction.<br>
K (Lysine): instead of Tryptophan (W). Mutation<br>
introduced to remover enzyme activity.<br>
Length, Molecular Weight, Isoelectric Point (IP):<br>
433 AA, MW : 50.3 kDa„ IP: 9.08<br><br>
p51 expression in B834(DE3) cells:<br>
P51 expression level and recombinant protein solubility were evaluated, in parallel to<br>
RT/p66 production strain.<br>
p51 expression level:<br>
Induction condition: cells grown/inducedat 37°C (+lmMIPTG), during 5 hours.<br>
Breaking buffer: 50 mM Tris/HCl, pH:7.5, ImMEDTA, +/- ImMDTT.<br>
Western blot analysis:<br>
Reagents- - rabbit polyclonal anti RT (rabbit P03L16) (dilution: 1/10,000)<br>
- Alkaline phosphatase-conjugate anti-rabbit antibody (dilution: 1/7500)<br>
Cellular fractions corresponding to crude extracts (T), insoluble pellet (P) and supernatant<br>
(S) were run on 10 % reducing SDS-PAGE.<br>
As illustrated on Coomassie stained gel and Western Blot (Figure 4) very high expression<br>
of P51 (15-20% of total protein) was observed, higher than that observed for P66.<br>
For both p51 and p66 proteins (after 5h induction at 37°C), 80% of the recombinant<br>
products were recovered in the soluble fraction (S1) of cellular extracts (See Figure 4).<br>
When expressed at 30°C, 99% of recombinant proteins were associated with the soluble<br>
fraction (data not shown).<br>
The p51 Western B1ot pattern was multiband, but less complex than that observed for<br>
P66.<br>
Solubility assay<br>
Solubility assay: Freezing/thawing of Soluble (S1) fraction (5h induction, 37°C)<br>
prepared under reducing (breaking buffer with DTT) and non-reducing conditions. After<br>
thawing, S1 samples were centrifuged at 20.000g/30 minutes, generating S2 and P2 (p2<br>
is resuspended in 1/10 vol.).<br><br>
After freezing/thawing of soluble fractions (S1), prepared under reducing as well as non-<br>
reducing conditions, 99% of p51 and p66 are still recovered in soluble (S2) fraction. Only<br>
1% is found in the precipitate (P2). This is shown in Figure 5.<br>
Example 3: Construction and expression of p17-Nef and Nef-p17 with or without<br>
linker<br>
The double fusion proteins were constructed with and without linkers. The linkers aimed<br>
to decrease potential interactions between the two fusion partners and are as follows:<br><br>
Recombinant plasmids construction:<br>
•	pET29a/Nef-p17 expression vector:<br>
Nef-p17 fusion gene was amplified by PCR from the F4 recombinant plasmid.<br>
The PCR product was cloned into the intermediate pGEM-T cloning vector and<br>
subsequently into the pET29a expression vector.<br>
• pET28b/p17-Nef expression vector:<br>
Nef gene was amplified by PCR from the F4 recombinant plasmid. The PCR<br>
product was cloned into the intermediate pGEM-T cloning vector and subsequently into<br>
the pET28b/p17 expression vector, as a C-terminal in frame fusion with the p17 gene.<br>
•	pET29a/Nef-linker-p17 and pET28b/p17-linker-Nef expression vector:<br>
A 18 bp DNA fragment coding for the hexapeptide linker (GSGGGP) was<br>
inserted between Nef and p17 fusion partners, by site-directed mutagenesis (using the<br>
"GeneTailor Site-Directed Mutagenesis System", Invitrogen).<br><br>
Recombinant protein characteristics:<br>
•	Length, Molecular Weight, Isoelectric Point (IP)<br>
Nef-p17 (namedNP): 340 AA, MW: 38.5 kDa, IP:7.48<br>
Nef-GSGGGP|-P17 (named NLP): 346 AA, MW:38.9 kDa, IP: 7.48<br>
p17-Nef (named PN): 342 AA, MW: 38.7 kDa, IP: 7.19<br>
p 17-GSGGGP|-Nef (named PLN): 348 AA, MW: 39.1kDa, IP: 7.19<br>
•	Amino-acid sequences and polynucleotide sequences:<br><br><br><br><br><br><br><br>
Hexapeptide linker<br>
Box: amino-acids introduced by genetic construction.<br>
Comparative expression of Nef-p17 , p17-Nef fusions, with and w/o linkers:<br>
The four recombinant strains were induced at 30°C over 3 hours, in parallel to F4 and<br>
Nef producing strains. Crude extracts were prepared and analyzed by Coomassie stained<br>
gel and Western blotting.<br>
Western blot analysis:<br>
Reagents: - rabbit polyclonal anti RT (rabbit P03L16) (dilution: 1/10,000)<br>
- Alkaline phosphatase-conjugate anti-rabbit antibody (dilution: 1/7500)<br><br>
As illustrated in Figure 6, Nef-pl 7 and p17-Nef fusions, with and w/o linker, are<br>
expressed at a high level (10% total proteins).<br>
In the Western blot: the four double fusion constructs present a multi-band pattern, but<br>
less complex than what was observed for F4. When expressed alone, the Nef and p17<br>
proteins present single band patterns.<br>
Strains expressing Nef-p17 (NP) and p17-Nef (PN) fusions, without linker peptide, were<br>
further analysed (solubility assays, see below).<br>
Nef-p17 and p17-Nef solubility assay:<br>
Nef-p17 and p17-Nef proteins were induced, in parallel to F4 and Nef producing strains.<br>
Induction condition: cells grown / induced at 30°C (+lmMlPTG), over 3 hours.<br>
Breaking buffer: 50mM Tris/HCl pH:8, 50mM NaCl, ImM EDTA<br>
Fresh cellular extracts:<br>
Cellular extracts were prepared (under non-reducing conditions) and fractions<br>
corresponding to crude extracts (T), insoluble pellet (P), and supernatant (S1) were<br>
analyzed on Coomassie stained gel and Western blot.<br>
As illustrated in Figure 7 on Coomassie stained gel and Western blot, almost all Nef-p17,<br>
p17-Nef, as well as Nef proteins are recovered in the soluble fraction (S) of cellular<br>
extracts. For F4 construct: 5-10% of recombinant protein already recovered in the pellet<br>
fraction.<br>
Conclusions:<br>
All double fusion constructs tested are highly expressed ( &gt; 10% of total protein). P17-<br><br>
Nef and Nef-p17 fusion proteins are more soluble than F4. Both present a less complex<br>
WB pattern.<br>
Example 4: Construction and expression of p24-RT*-Nef-p17 (F4*)<br>
F4* is a mutated version of the F4 (p24-RT/p66-Nef-p17) fusion where the Methionine at<br>
position 592 is replaced by a Lysine. This methionine is a putative internal transcriptional<br>
"start" site, as supported by N-terminal sequencing performed on a Q sepharose eluate<br>
sample of F4 purification experiment. Indeed, the major F4-related small band at 62 kDa<br>
present in the Q eluate sample starts at methionine 592.<br>
Methionine is replaced by a lysine: RMR → RKR. The RKR motif is naturally present in<br>
clade A RT sequences.<br>
The impact of this mutation on CD4-CD8 epitopes was evaluated:<br>
-	one HLA-A3 CTL epitope (A* 3002) is lost, but 9 other HLA-A3 epitopes are present<br>
in the RT sequence.<br>
-	No helper epitope identified in this region.<br>
Recombinant protein characteristics:<br>
N-term - p24: 232a.a. - |hinge:2a.al - |RT: 562a.a.] -jhinge:2a.al - [Nef: 206a.a. -<br>
- P17: 132a.a. - C-term<br>
• Length, Molecular Weight, Isoelectric Point (IP):<br>
1136 AA, 129 kDa, IP: 8.07<br><br><br><br><br><br><br><br>
F4* expression in B834(DE3) ceils:<br>
F4* recombinant strain was induced at 22°C during 18h3 in parallel to F4 non-mutated<br>
construct. Crude extracts were prepared and analyzed by Coomassie stained gel and<br>
Western blotting.<br>
As illustrated in Figure 8, F4* was expressed at a high level (10% total protein), slightly<br>
higher compared to F4 and the small 62 kDa band disappeared.<br>
Western blot analysis:<br>
Reagents: -βool 3 Mabs anti p24 (JC13.1, JC16.1, IG8.1. l)(dilution 1/5000)<br>
-	rabbit polyclonal anti RT (rabbit P03L16) (dilution: 1/10 000)<br>
-	rabbit polyclonal anti Nef-Tat (rabbit 388) (dilution 1/10 000)<br>
-Alkaline phosphatase-conjugate anti-rabbit antiboby (dilution:<br>
1/7500)<br>
-Alkaline phosphatase-conjugate anti-mouse antiboby (dilution:<br>
1/7500)<br>
Example S: Construction and expression of F3 and F3* (mutated F3)<br><br>
F3 (p17-p51-Nef) and F3* (p17-p51*-Nef) in which the putative internal Methionine<br>
initiation site replaced by Lysine.<br>
F3 and F3* fusions could be used in combination with p24.<br>
Recombinant plasmids construction:<br>
F3: The sequence encoding p51 was excized (as Seal and StuI DNA fragment) from<br>
pET29a/p51 expression plasmid and ligated into pET28b/p17-Nef plasmid , at the StuI<br>
site (located between p17 and Nef gene), as an in frame fusion with p17 and Nef<br>
sequences. The resulting fusion construct p17-p51-Nef is named F3.<br>
F3*: Mutation of the putative internal methionine initiation site was achieved using the<br>
"Gene Tailor Site-Directed Mutagenesis system" (Invitrogen), generating F3* construct.<br>
F3 and F3* plasmids were used to transform B834 (DE3) cells.<br><br><br><br><br><br><br>
"Fresh" cellular extracts<br>
Cellular fractions corresponding to crude extracts (T), insoluble pellet (P) and supernatant<br>
(S) were analyzed on 10% reducing SDS-PAGE. As illustrated in Figure 9, the F3 fusion<br>
protein is expressed at a high level (10% total protein). Almost all F3 is recovered in the<br>
soluble fraction (S) of cellular extracts, although 5-10% of F4 product are already<br>
associated with the pellet fraction. The WB pattern is simplified compared to F4.<br>
F3* expression in B834(DE3) cells:<br>
F3* recombinant strain was induced at 37°C over 3h, in parallel to F3 non-mutated<br>
constructed. Crude cellular extracts were prepared and analyzed by Coomassie stained<br>
gel and Western blotting. As illustrated in Figure 10, the F3* fusion protein is expressed<br>
at a very high level ( 10-20% total protein). There was a simplified WB pattern<br>
compared to F3; a very faint band at +/- 32kDa (detected on WB only) had disappeared.<br>
Example 6: Construction and expression of F4(p51) and F4(p51)*<br>
RT/p51 was used in the F4 fusion construct (in place of RT/p66).<br>
F4(p51) = p24-p51-Nef-p17<br>
F4(p51)* = p24-p51*-Nef-p17 - Mutated F4(p51): putative internal Methionine<br>
initiation site (present in RT portion) replaced by Lysine, to further simplify the antigen<br>
pattern.<br>
Recombinant plasmids construction:<br>
F4(p51): The sequence encoding p51 was amplified by PCR from pET29a/p51<br><br>
expression plasmid. Restriction sites were incorporated into the PCR primers (Ndel and<br>
StuI at the 5' end. AvrII at the 3' end of the coding sequence). The PCR product was<br>
cloned into pGem-T intermediate plasmid and sequenced. pGem-T/p51 intermediate<br>
plasmid was restricted by Ndel and Avrll and the p51 fragment was ligated into<br>
pET28b/p24-RT/p66-Nef-p17 expression plasmid restricted by Ndel and Nhel (resulting<br>
in the excision of RT/p66 sequence). Ligation was performed by combining digestion<br>
reactions in appropriate concentrations, in the presence of T4 DNA ligase. Ligation<br>
product was used to transform DH5α E.coli cells. Verification of insertion of p51 into the<br>
correct translational reading frame (in place of RT/p66 in the f4 fusion) was confirmed by<br>
DNA sequencing. The resulting fusion construct p24-RT/p51-Nef-p17 is named F4(p51).<br>
F4(p51)*: Mutation of the putative internal methionine initiation site (present in RT/p51)<br>
was achieved with "GeneTailor Site-Directed Mutagenesis system" (Invitrogen),<br>
generating F4(p51)* construct.<br>
F4(p51) and F4(p51)* expression plasmids were used to transform B834(DE3) cells.<br><br><br><br><br><br><br>
enzyme activity.<br>
F4(p51) expression in B834(E3) cells:<br>
F4(p51) expression level and recombinant protein solubility were evaluated, in parallel to<br>
F4 expressing strain.<br>
Induction condition: cells grown at 37°C/ induced at 22°C (+lmMIPTG), over 19h.<br>
Breaking buffer: 50mMTris/HCl pH:7.5. ImMEDTA, ImMDTT<br>
Western blot analysis:<br>
reagents	- rabbit polyclonal anti RT (rabbit P03L16) (dilution: 111 0 000)<br>
- rabbit polyclonal anti Nef-Tat (rabbit 388) (dilution 1/10 000)<br>
-Alkaline phosphatase-conjugate anti-rabbit antiboby (dilution: 1/7500)<br>
Cellular fractions corresponding to crude extracts (T), insoluble pellet (P) and supernatant<br>
(S) were analyzed on 10% reducing SDS-PAGE.<br>
As illustrated in Figure 11, F4(p51) was expressed at a high level (10% of total protein),<br>
similar to F4. Almost all F4(p51) is recovered in the soluble fraction (S) of cellular<br>
extracts. Upon detection with an anti-Nef-tat reagent, F4(p51) the WB pattern was<br>
shown to be simplified (reduction of truncated products below +/- 60kDa).<br>
F4(pSl)* expression in B834(DE3) cells:<br>
F4(p51)* recombinant strain was induced at 22°C over 18h, in parallel to F4(p51) non-<br>
mutated construct, F4 and F4*. Crude cellular extracts were prepared and analyzed by<br>
Coomassie stained gel and Western blotting. As illustrated in Figure 12 high expression<br>
of F4(p51) and F4(p51)* fusions was observed, representing at least 10% of total<br>
protein. WB pattern: reduction of truncated products below +/- 60kDa. In addition, for<br><br>
F4(p51)* construct, the 47kDa band (due to internal start site) has disappeared.<br>
Example 7: Purification of F4, F4(pSl)* and F4* - Purification Method I<br>
The fusion protein F4, comprising the 4 HIV antigens p24-RT-Nef-p17, was purified<br>
from a E. coli cell homogenate according to purification method I, which comprises the<br>
following principal steps:<br>
•	Ammonium sulfate precipitation of F4<br>
	S03 Fractogel cation-exchange chromatography (positive mode)<br>
.ctyl sepharose hydrophobic interaction chromatography (positive mode)<br>
.Q sepharose FF anion-exchange chromatography (positive mode)<br>
.Superdex 200 gel filtration chromatography in presence of SDS<br>
.Dialysis and concentration<br>
Additionally, the F4(p51)* fusion protein (RT replaced by the codon optimized p51<br>
carrying an additional mutation Met592Lys) and the F4* protein ( F4 carrying an<br>
additional Met592Lys mutation) were purified using the same purification method I.<br>
Protein quantification<br>
• Total protein was determined using the Lowry assay. Before measuring the protein<br>
concentration all samples are dialyzed overnight against PBS, 0.1% SDS to remove<br>
interfering substances (urea, DTT). BSA (Pierce) was used as the standard.<br>
SDS-PAGE and western blot<br>
	Samples were prepared in reducing or non-reducing SDS-PAGE sample buffer (+/-<br>
β-mercaptoethanol) and heated for 5 min at 95°C.<br>
•	Proteins were separated on 4-20% SDS-plyacrylamide gels at 200 V for 75 min<br>
using pre-cast Novex Tris-glycine gels or Criterion gels (Bio-Rad), 1 mm thick.<br>
•	Proteins were visualized with Coomassie-blue R250.<br><br>
	For the western blots (WB), the proteins were transferred from the SDS-gel onto<br>
nitrocellulose membranes (Bio-Rad) at 4°C for 1.5 h at 100 V or overnight at 30 V.<br>
	F4 was detected using monoclonal antibodies against the different antigens, anti-<br>
p24, anti-Nef-Tat, anti-RT (sometimes a mixture of anti-p24 and anti Nef-Tat was<br>
used to detect a maximum number of protein bands).<br>
	Alkaline-βhosphatase conjugated anti-mouse or anti-rabbit antibodies were bound<br>
to the primary antibodies and protein bands were visualized using BCIP and NBT<br>
as the substrates.<br>
anti-E. coli western blot<br>
	5 ug protein (Lowry) were separated by SDS-PAGE and transferred onto<br>
nitrocellulose membranes as above.<br>
	Residual host cell proteins were detected using polyclonal anti-E. coli antibodies.<br>
Protein bands were visualized with the alkaline-posphatase reaction as above.<br>
Purification Method I<br>
Method I comprises a precipitation by ammonium sulfate and four chromatographic<br>
steps:<br>
	E.coli cells were homogenized in 50mM Tris buffer at pH 8.0 in the presence of<br>
10 DTT, ImM PMSF, ImM EDTA at OD50 (-360 ml). 2 Rannie passages<br>
were applied at 1000 bars.<br>
	Cells debris and insoluble material were removed by centrifugation at 14400 ˟ g for<br>
20min.<br>
• Ammonium sulfate (AS) was added from a 3.8M stock solution to the clarified<br>
supernatant to a final concentration of 1.2M. Proteins were precipitated for ~2<br>
hours at room temperature (RT) and then pelleted by centrifugation (10 min at<br>
14400 x g). The pellet was resuspended in 8M urea, 10mM DTT in 10mM<br>
phosphate buffer at pH 7.0.<br>
	The antigen was captured on a S03 Fractogel column (Merck) in the presence of<br>
8M urea and 10mM DTT at pH 7.0 in phosphate buffer. The column was washed to<br>
elute non-bound protein followed by a pre-elution step with 170mM NaCl to<br><br>
remove bound host cell proteins (HCP). F4 was then eluted with 460mM NaCl, 8M<br>
urea, 10mM DTT in phosphate buffer at pH 7.0.<br>
• The S03 eluate was 2fold diluted with 10mM phosphate buffer, pH 7, and loaded<br>
onto a Octyl sepharose column (Amersham Biosciences) in the presence of 4M<br>
urea, lmM DTT, 230mM NaCl in phosphate buffer at pH 7.0. Following a washing<br>
step (equilibration buffer) bound F4 was eluted with 8M urea, lmM DTT in 25mM<br>
Tris buffer at pH 8.0.<br>
	The Octyl eluate was diluted and adjusted to pH 9.0 and F4 was then bound to an Q<br>
sepharose column (Amersham Bioscience) in the presence of 8M urea at pH 9.0<br>
(25mM Tris). Unbound protein was washed off (8M urea, 25mM Tris at pH 9.0)<br>
and a pre-elution step (90mM NaCl in 8M urea, 25mM Tris, pH 9.0) removed HCP<br>
and F4-degradation products. F4 was desorped from the column with 200mM<br>
NaCl, 8M urea in Tris buffer at pH 9.0.<br>
	An aliquot of the Q eluate was spiked with 1% SDS and dialyzed against PBS<br>
buffer containing 0.1% SDS and lmM DTT to remove the urea prior to injecting<br>
the sample onto the gel filtration column (prep grade Superdex 200, two 16 x 60<br>
cm columns connected in a row). The relevant fractions were pooled after in-<br>
process SDS-PAGE analysis.<br>
	Samples were dialyzed twice at RT in dialysis membranes (12-14 kDa cut-off)<br>
overnight against 110.5M Arginine, 10mM Tris, 5mM G1utathione, pH 8.5.<br>
The sequential purification steps are shown in the flowchart below.<br>
Purification Flowsheet<br>
360 ml homogenate OD50 (Rannie)<br>
50 mM Tris pH 8.0, 1 mM PMSF, 10mM DTT, 2 mM EDTA<br>
Clarification<br>
20 min centrifugation at 14400 x g<br><br><br>
Ammonium sulfate precipitation<br>
1.2 M AS, 2 h at RT, centrifugation 14400 x g, 10 min<br><br>
pellet resuspended in 8 M urea, 10 mM P04, 10mM DTT, pH 7.0<br><br>
(+) S03 Fractogel EMD 650 (M) chromatography<br>
pH 7.0, 8 M urea, 10mM DTT, pre-elution at 170 mM NaCl, elution 460 mM NaCl<br><br>
2 x dilution to pH 7.0,4 M urea, 5 mM DTT, 230 mM NaCl<br><br>
(+) Octyl sepharose chromatography<br>
pH 7.0,4 M urea 230 mM NaCl, elution 8 M urea, 20 mM Tris pH 8.0<br>
~2 x dilution, adjustment to pH 9.0 (NaOH)<br><br>
(+) Q Sepharose FF chromatography<br>
Tris pH 9.0, 8 M urea, pre-elution 90 mM NaCl, elution 200mM NaCl<br><br>
addition of 1%SDS<br><br>
dialysis<br>
- TBS, 0.1% SDS, pH 8.5<br><br>
Superdex 200 gel filtration chromatography 16 x 120 cm<br>
TBS, 0.1% SDS, pH 8.5<br><br>
IPA SDS-PAGE<br><br>
pool/concentration/dialysis<br>
- formulation compatible buffer<br>
IPA - In process analysis<br><br>
All buffers contain 1 mM DTT if not otherwise specified.<br>
Results Purification of F4<br>
SDS-PAGE/Western B1ot Follow-up of The Purification Process<br>
Figure 13 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4-<br>
containing fractions collected during the purification of F4.<br>
The E. coli homogenate is shown in Fig. 13, lane 2, with F4 estimated to represent about<br>
10% of the total proteins (density scans of Coomassie blue stained SDS-gels). After<br>
centrifugation, the soluble fraction of F4 was recovered in the clarified supernatant (lane<br>
3). The ammonium sulfate precipitation step eliminated many impurities (lane 4) and<br>
reduced the proteic charge for the subsequent chromatographic step. Additionally, the 8M<br>
urea used to resuspend the precipitate dissociated complexes of F4 with HCP and allowed<br>
both complete capture of F4 by and quantitative elution from the S03 resin. The S03<br>
eluate shown in lane 5 was considerably enriched in F4 but the heterogeneous pattern<br>
remained principally unchanged. The hydrophobic Octyl sepharose column mainly<br>
removed low molecular weight (LMW) HCP and F4-degradation products (lane 6),<br>
thereby simplifying the F4 pattern. The Q sepharose chromatography further simplified<br>
the F4 pattern and removed many impurities (lane 7). Final purity in terms of coli<br>
impurities was obtained after this step. In fact, no host cell proteins were detected in the<br>
Q eluate by anti-E. coli western blot analysis. The purified F4 thus produced is referred<br>
to as F4Q. The Superdex 200 column separated LMW F4-degradation products from the<br>
full length F4 improving F4 homogeneity in the Superdex 200 eluate (lane 8). The term<br>
F4S may be used to refer to F4 purified according to the full scheme of method I.<br>
An anti-E. coli western blot was done of the same fractions collected during the<br>
purification of F4. The absence of visible bands on the anti-E. coli western blot indicated<br>
HCP contamination below 1% in the Q eluate and in the Superdex eluate.<br><br>
F4 and Protein Recovery<br>
F4 recovery at each step of the purification process was estimated from SDS-PAGE and<br>
western blot analysis. To estimate F4 recovery from SDS-gels, the sample volumes<br>
loaded onto the SDS-gels corresponded to the volumes of the different fractions collected<br>
during the purification.<br><br>
The table shows the amount of protein in the homogenate and the soluble material,<br>
including F4, recovered in the supernatant after the clarification step. The AS-<br>
precipitation step removed a great amount of HCP and only a slight loss of F4 was<br>
observed on the SDS-gel. The S03 chromatography additionally removed many<br>
impurities and the SDS-gel indicated a high recovery of F4. In contrast, the -50% protein<br>
recovery measured with both the Octyl sepharose and the Q sepharose columns were also<br><br>
accompanied by losses of F4. Protein recovery after the gel filtration chromatography<br>
was about 50%. The SDS-gel shows that many LMW-βrotein bands (F4-degradation<br>
bands) were removed, concomitantly reducing F4 recovery.<br>
F4 Yield<br>
Table 1 above shows that about 36 mg purified F4 could be obtained from 360 ml<br>
homogenate at OD50. Therefore, 1 1 homogenate at OD 50 should yield about 100 mg<br>
purified F4. Since ODs of 70 - 90 were achieved during the fermentation process, the<br>
yield per liter fermenter would accordingly be in the range of 140 to 180 mg F4.<br>
Results Purification of F4(p51)*<br>
The F4(p51)* fusion construct was purified using purification method1described above<br>
without modifications.<br>
SDS-PAGE/Western B1ot Follow-up of The Purification Process<br>
Figure 14 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4(p51)*-<br>
containing fractions collected during the purification of F4(p51*).<br>
The SDS-gel and the western blot demonstrate that the F4(p51)* fusion protein globally<br>
behaved similarly to F4 at the ammonium sulfate precipitation step as well as during the<br>
chromatographic steps. Purified F4(p51)* had a heterogeneity pattern similar to purified<br>
F4.<br>
An anti E. coli western blot indicated that HCP contamination was below 1 % in both the<br>
Q eluate and the Superdex eluate.<br><br>
About 25% of F4(p51)* were lost in the insoluble fraction of the homogenate.<br>
Additionally, because the purification method was not adapted to this protein, losses were<br>
observed at the chromatographic steps. Therefore the overall recovery of F4(p51)* was<br>
reduced to about 25 mg per liter homogenate (OD50). Extrapolated to 1 litre culture at<br>
OD 177, the yield would accordingly be in the range of 85 mg F4(p51)*.<br>
Results Purification of F4*<br>
The F4* fusion construct was purified using purification method I described above<br>
without modifications.<br>
SDS-PAGE/Western B1ot Follow-up of The Purification Process<br>
Figure 15 shows the SDS gel and the anti-p24/anti-Nef-Tat western blot of the F4*-<br>
containing fractions collected during the purification of F4*.<br>
As with F4(p51)* it can also be noted that F4* globally behaved quite similarly to F4<br>
during the purification procedure. The protein was recovered in the expected fractions as<br>
shown by the SDS-gel and the western blot. An anti-Ecoli western blot also<br>
demonstrated elimination of most HCP already after the Q sepharose column.<br>
Yield<br>
The global recovery was about 17 mg purified F4* obtained from 465 ml homogenate<br>
OD50. Extrapolated to 1 1 culture at OD 140, the yield would accordingly be in the range<br>
of 100 mg F4*.<br>
In summary, the three fusion proteins F4, F4(p51)* and F4* were purified employing<br>
purification method I. The SDS gel in Figure 16 compares the three purified proteins<br>
showing the different level of heterogeneity of the constructs after the Q sepharose step<br>
and after elimination of LMW bands by the Superdex 200 column.<br><br>
Example 8: Purification of F4 and F4co (codon optimized) - Purification Method II<br>
Purification Method II<br>
A simplified purification procedure, method II as compared to method I, was also<br>
developed. Method II consists of only 2 chromatographic steps and a final<br>
dialysis/diafiltration for buffer exchange. Notably, a CM hyperZ chromatographic<br>
column (BioSepra) was introduced to replace the clarification step, the ammonium sulfate<br>
precipitation and the S03 chromatography of method I (Example 7). Method II was used<br>
to purify both F4 and full-codon optimized F4 ("F4co"). For F4co, two different forms of<br>
method II were performed, one involving carboxyamidation and one not. The purpose of<br>
the carboxyamidation step was to prevent oxidative aggregation of the protein. This<br>
carboxyamidation is performed after the 1st chromatographic step (CM hyperZ).<br>
	E.coli cells (expressing F4 or F4co) were homogenized in 50mM Tris buffer at<br>
pH 8.0 in the presence of 10mM DTT, at OD90. 2 Rannie passages were applied at<br>
1000 bars.<br>
8M urea were added to the homogenate before application to the CM hyperZ<br>
resin (BioSepra) equilibrated with 8M urea in phosphate buffer at pH 7. Antigen<br>
capture was done in a batch mode. The resin was then packed in a column, unbound<br>
proteins were washed off with the equilibration buffer and bound host cell proteins<br>
(HCP) were removed by a pre-elution step with 120mM NaCl. F4co was then eluted<br>
with 360mM NaCl, 8M urea, 10mM DTT in phosphate buffer at pH 7.0.<br>
To control oxidative aggregation of the fusion protein, the cysteine groups of<br>
F4co can be carboxyamidated with idoacetamide. Therefore, optionally, 50 mM<br>
iodoacetarnide was added to the CM hyperZ eluate and carboxyamidation was done<br>
for 30 min at room temperature in the dark.<br>
The CM hyperZ eluate was then adequately diluted (about 5-8 fold) and<br>
adjusted to pH 9.0. F4co or F4coca was then bound to a Q sepharose column<br>
(Amersham Bioscience) in the presence of 8M urea in Tris buffer at pH 9.0. Unbound<br>
protein was washed off with the equilibration buffer and a pre-elution step with 90mM<br><br>
NaCl (only with non-carboxyamidated protein) in the same buffer removed bound<br>
HCP. F4co was desorped from the column with 200mM NaCl, 8M urea in Tris buffer<br>
atpH9.0.<br>
Samples were dialyzed twice at RT in dialysis membranes (12-14 kDa cut-off)<br>
overnight against 1 1 0.5M Arginine, 10mM Tris buffer, 10mM G1utathione (only<br>
added to the non-carboxyamidated protein), pH 8.5. Alternatively, buffer exchange<br>
was accomplished by diafiltration against 10 sample volumes of the same buffer using<br>
a tangential-flow membrane with 30 or 50 kDa cut-off.<br>
	Finally, the dialyzed product was sterile filtered through a 0.22 µm membrane.<br>
The sequential purification steps are shown in the flowchart below.<br>
Purification Flowsheet<br>
Homogenate OD90 (Rannie)<br>
50mM Tris pH 8.0, 10mM DTT<br><br>
addition of 8M urea, adjusting to pH 7.0<br>
(+) CM hyperZ chromatography<br>
pH 7.0, 8M urea, 10mM DTT, pre-elution at 120mM NaCl, elution 360mM NaCl<br><br>
optional carboxyamidation: addition of 50mM iodoacetamide, 30 min at RT<br><br>
dilution and adjusting to pH 9.0,8M urea<br><br>
(+) Q Sepharose FF chromatography<br>
Tris pH 9.0, 8 M urea, pre-elution, elution with NaCl*<br><br>
dialysis/dlafiltration<br>
-&gt; phosphate buffer, 0.5M Arginine, pH 8.5 (10mM G1utathione)<br><br><br>
Sterile filtration<br>
All buffers contained DTT if F4co was not carboxyamidated and G1utathione in the<br>
purified bulk. Reducing agents were omitted once the protein was carboxyamidated.<br>
*NaCl - for F4co this was 200mM NaCl, for F4coca elution was by gradient of NaCl.<br>
This step can be further optimized for F4coca by pre-eluting with 60mM NaCl and<br>
eluting with 100mM NaC; and for F4co by eluting with 100mM NaCl (no pre-elution<br>
step needed).<br>
Results: Purification of F4co<br>
Figure 17 shows a SDS gel of the F4-containing fractions collected during the<br>
purification of F4co and the purification of carboxyamidated F4co ("F4coca").<br>
The CM hyperZ resin completely captured F4co from the crude homogenate (lane 1) in<br>
the presence of 8M urea and quantitative elution was achieved with 360mM NaCl. The<br>
CM hyperZ eluate shown in lane 2 was considerably enriched in F4co. After appropriate<br>
dilution and adjustment of the sample to pH 9, F4co or F4coca was bound to a Q<br>
sepharose column. F4co or F4coca was then specifically eluted with 200mM NaCl as<br>
shown in lane 3. This chromatography not only removed remaining host cell proteins but<br>
also DNA and endotoxins. To bring the purified material in a formulation-compatible<br>
buffer, the Q sepharose eluate was dialyzed against 10mM Tris buffer, 0.5M Arginine,<br>
10mM G1utathione pH 8.5 in a dialysis membrane with 12-14 kDa cut-off. G1utathione<br>
was omitted with the carboxyamidated protein.<br>
Purification of both F4co and F4coca yielded about 500 mg purified material per L of<br>
culture OD130. This was in a similar range as observed before with the non-codon-<br>
optimized F4.<br><br>
As described above, two different purification methods (I and II) have been developed to<br>
purify the different F4 constructs. Figure 18 compares the different purified bulks that<br>
were obtained.<br>
The SDS gel in Figure 18 clearly illustrates the distinct pattern of the two different<br>
proteins, F4 and F4co. Whereas F4 presented several strong low molecular weight<br>
(LMW) bands, only faint bands were visible with the codon-optimized F4co. Method I<br>
and method II produce a very similar F4co pattern. Anti-E coli western blot analysis<br>
confirmed the purity of the purified proteins indicating host cell protein contamination<br>
below 1% in all the preparations.<br>
Example 9: Immunogeniciry of F4 in mice<br>
Formulation:<br>
Adjuvant formulation 1B:<br>
To prepare Adjuvan t formulation 1 B, A mixture of lipid (such as phosphatidylcholine<br>
either from egg-yolk or synthetic) and cholesterol and 3 D- MPL in organic solvent, is<br>
dried down under vacuum (or alternatively under a stream of inert gas). An aqueous<br>
solution (such as phosphate buffered saline) is then added, and the vessel agitated until<br>
all the lipid is in suspension. This suspension is then microfluidised until the liposome<br>
size is reduced to about 100 nm, and then sterile filtered through a 0.2 µm filter.<br>
Extrusion or sonication could replace this step.<br>
Typically the cholesterol:phosphatidylcholine ratio is 1:4 (w/w), and the aqueous solution<br>
is added to give a final cholesterol concentration of 5 to 50 mg/ml..<br>
The liposomes have a defined size of 100 nm and are referred to as SUV (for small<br>
unilamelar vesicles). If this solution is repeatedly frozen and thawed the vesicles fuse to<br>
form large multilamellar structures (MLV) of size ranging from 500nm to 15 pm.<br>
The liposomes by themselves are stable over time and have no fusogenic capacity.<br><br>
QS21 in aqueous solution is added to the liposomes to reach a final 3 D- MPL and QS21<br>
concentrations of 100 µg/ml.<br>
Formulation 2A: 3 De acylated monophoshphoryl lipid A and QS21 In an oil in water<br>
emulsion;<br>
Preparation of oil in water emulsion can be made by following the protocol as set forth in<br>
WO 95/17210. In detail the emulsion contains: 5% Squalene 5% tocopherol 2.0% tween<br>
80; the particle size is 180 nm.<br>
Preparation of Oil in water emulsion (2 fold concentrate)<br>
Tween 80 is dissolved in phosphate buffered saline (PBS) to give a 2% solution in the<br>
PBS. To provide 100 ml two fold concentrate emulsion 5g of DL alpha tocopherol<br>
and 5ml of squalene are vortexed to mix thoroughly. 90ml of PBS/Tween solution is<br>
added and mixed thoroughly. The resulting emulsion is then passed through a syringe<br>
and finally microfluidised by using an M110S microfluidics machine. The resulting oil<br>
droplets have a size of approximately 180 nm.<br>
Sterile bulk emulsion is added to PBS to reach a final concentration of 500 ul of<br>
emulsion per ml (v/v). 3 D- MPL is then added to reach a final concentration of 100ug.<br>
QS21 is then added to reach a final concentration of 100µg per ml. Between each<br>
addition of component, the intermediate product is stirred for 5 minutes<br>
F4Q not codon optimized, purified according to purification method I, was diluted in a<br>
phosphate/Arginine buffer pH 6.8. The dilution was mixed with two different<br>
concentrated adjuvants (adjuvants 2A and 1B) in order to obtain a final formulation of<br>
40ug/dose of 500µl of F4 in presence of 290 (for adjuvant 2A) - 300 (for adjuvant 1B)<br>
mM Argnine, 50µg MPL and 50µg QS2I. 100µl of each formulation were injected in<br>
mice.<br>
Mouse immunogenicity studies were performed to evaluate the cellular and humoral<br>
immune responses to the four antigens found within F4 (p24, p17, RT and Nef).<br><br>
Due to the complexicity of the F4 antigen, eight strains of mice, each with a different<br>
genetic background, were immunised twice at day 0 and day 21 with 8ug of adjuvanted<br>
F4 protein prepared as described above, in a 100 µl volume. Serum and spleen samples<br>
were collected 14 days following the last immunisation (day 35) for analysis of the<br>
humoral and cellular responses to each of the four components of F4 (p24, p17, RT and<br>
Nef), as well as F4.<br>
Total antibody responses were characterised by ELISAs specific for p24, p17, RT, Nef<br>
and F4. The following table, Table 2, summarises where antigen specific humoral<br>
responses were observed in each strain. The results indicate the presence or absence of<br>
antibodies compared to control animals immunized with adjuant alone. The results<br>
presented are a compilation from two separate but identical experiments. In the table, 2A<br>
refers to antigen formulated with 3D-MPL and QS21 in an oil in water emulsion and 1B<br>
refers to antigen formulated with 3D-MPL, QS21 and cholesterol containing liposomes.<br><br><br>
0F1 mice mounted antibody responses to all four F4 components. The responses<br>
observed are shown in Figure 19. +/- indicates that the response observed was weak or<br>
only observed with one of the two adjuvant. For example, B6D2F1 mice p17 responses:<br>
+/- overall with a +2A and -1B means that there was a response with 2A (not weak) and<br>
none with 1B. Balb/c mice p17 responses: - overall, with a +/- 2A and a - 1B, here the<br>
+/- means that the response with adjuvant 2A was weak.<br>
Cellular responses were characterised by flow cytometry staining for CD4 and CD8,<br>
IFNy and IL-2 expression (intracellular cytokine staining for IFNy and IL-2 expression),<br>
following restimulation of spleen cells with p24, p17, RT or Nef specific peptides, using<br>
peptide library pools of 15 mers with 11 mer overlap. GD4 responses were the dominant<br>
cellular response observed. The following table, Table 3, summarises where antigen<br>
specific CD4+IL-2+ responses were observed for each mouse strain. Again, this is<br>
shown as presence or absence of a response.<br><br><br>
DBA mice mounted CD4 responses to all four F4 components. The CD4+IL-2+ and<br>
CD4+IFNγ+ responses observed for this mouse strain are shown in Figure 20.<br>
In summary, F4 formulated in either of the two adjuvant formulations is able to promote<br>
humoral and cellular responses to p24, p17, RT and Nef. This shows that each region of<br>
F4 is immunogenic in an in vivo situation.<br><br>
We Claim :<br>
1.	A polypeptide which comprises Nef or an immunogenic fragment or derivative<br>
thereof, such as herein described, p17 Gag and p24 Gag or immunogenic fragments or<br>
derivatives thereof, such as herein described, wherein there is at least one HIV antigen<br>
or immunogenic fragment between p17 Gag and p24 Gag, the polypeptide further<br>
comprising RT or an immunogenic fragmentor derivative thereof, such as herein<br>
described, and wherein the immunogenic fragments or derivatives remain capable of<br>
raising an immune response against the native antigen.<br>
2.	The polypeptide as claimed in claim 1, wherein the Pol or an immunogenic<br>
fragment or derivative thereof, wherein the immunogenic fragment or derivative<br>
remains capable of raising an immune response against the native antigen.<br>
3.	The polypeptide as claimed in claim 1 or claim 2, wherein the RT or<br>
immunogenic, fragment is a fragment in which RT is truncated at the C terminus such<br>
that it lacks the carboxy terminal RNase H domain.<br>
4.	The polypeptide as claimed in claim 3, wherein the RT fragment is the p51<br>
fragment.<br>
5.	The polypeptide as claimed in any one of claims 2 to 4, wherein the RT<br>
comprises a mutation at position 592 to replace the methionine by another residue e.g. •<br>
lysine.<br>
6.	The polypeptide as claimed in any one of claims 1 to 4, wherein the Nef is full<br>
length Nef.<br>
7.	A polypeptide selected from one of the following:<br><br>
8.	p24-RT-Nef-p17<br>
9.	p24-RT*-Nef-p17<br>
10.	p24-p51RT-Nef-p17<br>
11.	p24-p51RT*-Nef-p17<br>
* represents RT methionine592 mutation to lysine<br>
8.	A process for purifying a polypeptide as claimed in any one of claims 1 to 7,<br>
which process comprises:<br>
i) providing a composition comprising the unpurified polypeptide;<br>
ii) subjecting the composition to at least two chromatographic steps;<br>
iii) optionally carboxyarnidating the polypeptide;<br>
iv) performing a buffer exchange step to provide the protein in a suitable<br>
buffer for a pharmaceutical formulation.<br>
9.	The process as claimed in claim 8,. wherein there are no more than two<br>
chromatographic steps.<br>
10.	The process as claimed in claim 8 or claim 9, wherein the carboxyamidation is<br>
performed between the two chromatographic steps.<br>
11.	A composition comprising a polypeptide as claimed in claim 1.<br>
12.	A composition as claimed in claim 11, wherein the RT or immunogenic<br>
fragment thereof is a fragment in which RT is truncated at the C terminus such that it<br>
lacks the carboxy terminal RNase H domain.<br><br>
13.	The composition as claimed in claim 12, wherein the RT fragment is the p51<br>
fragment.<br>
14.	The composition as claimed in any one of claims 11 to 13, wherein the RT<br>
comprises a mutation at position 592 such that the methionine is replaced by another<br>
residue e.g. lysine.<br>
15.	The composition as claimed in any one of claims 11 to 14, wherein the Nef is<br>
full length Nef.<br>
16.	A polynucleotide or polynucleotides encoding a polypeptide or composition of<br>
polypeptides as claimed in any one of claims 1 to 7 and 11 to 15.<br>
17.	A pharmaceutical composition comprising a polypeptide or polynucleotide or<br>
composition of polypeptides or polynucleotides as claimed in any previous claim or a<br>
polypeptide purified according to a process as claimed in any previous claim, together<br>
with a pharmaceutically acceptable carrier or adjuvant<br>
18.	A pharmaceutical composition as claimed in claim 17, wherein the adjuvant is a<br>
Th1 inducing adjuvant such as QS21 or 3D-MPL or a combination of QS21 and 3D-<br>
MPL.<br>
19.	A pharmaceutical kit comprising:<br>
a)	a composition comprising a polypeptide as claimed in any one of<br>
claims 1 to 7, together with a pharmaceutically acceptable excipient; and<br>
b)	a composition comprising a polynucleotide encoding one or more<br>
of Nef and Gag or an immunogenic fragment or derivative of Nef or Gag<br>
containing a Nef or Gag epitope present in the polypeptide of a), together with<br><br>
a pharmaceutically acceptable excipient, wherein any irnmunogenic fragment<br>
or derivative remains capable of raising an immune response against the native<br>
antigen.<br>
20. A pharmaceutical kit comprising:<br>
a)	a composition comprising a polynucleotide encoding a<br>
polypeptide as claimed in any one of claims I to 7, together with a<br>
pharmaceutically acceptable excipient; and<br>
b)	a composition comprising a polypeptide comprising one or more<br>
of Nef and Gag or an immunogenic fragment or derivative of Nef or Gag<br>
containing a Nef or Gag epitope present in the polypeptide of a), together with<br>
a pharmaceutically acceptable excipient, wherein any immunogenic fragment<br>
or derivative remains capable of raising an immune response against the native<br>
antigen.<br><br><br><br>
ABSTRACT<br><br>
VACCINE FOR PREVENTION AND TREATMENT OF HIV-INFECTION<br>
This invention relates to novel HIV polypeptide and polynucleotide fusions of<br>
Gag, Pol and Nef which are useful in immunogenic compositions and vaccines. The<br>
invention relates in particular to a polypeptide which comprises Nef or an<br>
immunogenic fragment or derivative thereof, p17 Gag and p24 Gag or immunogenic<br>
fragments or derivatives thereof, wherein there is at least one HIV antigen or<br>
immunogenic fragment between p17 Gag and p24 Gag, the polypeptide further<br>
comprising RT or an immunogenic fragment or derivative thereof, and wherein the<br>
immunogenic fragments or derivatives remain capable of raising an immune response<br>
against the native antigen.<br>
Figure 20.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjEta29sbnAtMjAwNy1hc3NpZ25tZW50LTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">00621-kolnp-2007-assignment-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjEta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZS0xLjIucGRm" target="_blank" style="word-wrap:break-word;">00621-kolnp-2007-correspondence-1.2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjEta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZS0xLjMucGRm" target="_blank" style="word-wrap:break-word;">00621-kolnp-2007-correspondence-1.3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjEta29sbnAtMjAwNy1mb3JtIDE4LnBkZg==" target="_blank" style="word-wrap:break-word;">00621-kolnp-2007-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDA2MjFrb2xucC0yMDA3LWNvcnJlc3BvbmRlbmNlLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">00621kolnp-2007-correspondence-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGNvcnJlc3BvbmRlbmNlIG90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGludGVybmF0aW9uYWwgcHVibGljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IGludGVybmF0aW9uYWwgc2VhcmNoIGF1dGhvcml0eSByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 international search authority report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IHBjdCBmb3JtLnBkZg==" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 pct form.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDYyMS1rb2xucC0yMDA3IHByaW9yaXR5IGRvY3VtZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">0621-kolnp-2007 priority document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQUJTVFJBQ1QtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-ABSTRACT-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQU1BTkRFRCBDTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQU1BTkRFRCBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-AMANDED PAGES OF SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQU5FWFVSRSBUTyBGT1JNIDMucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-ANEXURE TO FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQVNTSUdOTUVOVC5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-ASSIGNMENT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQ0FOQ0VMTExFRCBQQUdFUy5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-CANCELLLED PAGES.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQ09SUkVTUE9OREVOQ0UgMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-CORRESPONDENCE 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctQ09SUkVTUE9OREVOQ0UucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctREVTQ1JJUFRJT04gKENPTVBMRVRFKS0xLjEucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-DESCRIPTION (COMPLETE)-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRFJBV0lOR1MtMS4xLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-DRAWINGS-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRVhBTUlOQVRJT04gUkVQT1JULnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-EXAMINATION REPORT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSAxLTEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 1-1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSAxMyAxLjEucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 13 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSAxMy5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctRk9STSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-FORM 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1BBLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GPA.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1DTEFJTVMucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1EUkFXSU5HUy5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-DRAWINGS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1GT1JNIDEucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-FORM 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctR1JBTlRFRC1TUEVDSUZJQ0FUSU9OLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-GRANTED-SPECIFICATION.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctT1RIRVJTICAxLjEucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-OTHERS  1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctT1RIRVJTLnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcucGRm" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUIDEuMS5wZGY=" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-REPLY TO EXAMINATION REPORT 1.1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjIxLUtPTE5QLTIwMDctUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JULnBkZg==" target="_blank" style="word-wrap:break-word;">621-KOLNP-2007-REPLY TO EXAMINATION REPORT.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="253464-flavor-ingredients-for-cooling-preparations.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253466-a-dendritic-compound.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253465</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>621/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>30/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>27-Jul-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>24-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>20-Feb-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GLAXOSMITHKLINE BIOLOGICALS S.A.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>ABRECHT HELGE</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MARCHAND MARTINE</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MATHY NATHALIE LOUISE</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>4</td>
											<td>PERMANNE PHILIPPE JEAN GERVAIS GHISLAIN</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>5</td>
											<td>VOSS GERALD HERMANN</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
										<tr>
											<td>6</td>
											<td>DELCHAMBRE MARTINE</td>
											<td>GLAXOSMITHKLINE BIOLOGICALS S.A., RUE DE I&#x27;INSTITUT 89, B-1330 RIXENSART,</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/49; A61K39/21</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/008434</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-08-03</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0417494.2</td>
									<td>2004-08-05</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253465-vaccine-for-prevention-and-treatment-of-hiv-infection by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:45:03 GMT -->
</html>
