<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253563-genetically-modified-yeast-of-the-species-issatchenkia-orientalis-and-closely-related-species-and-fermentation-processes-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:38:48 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253563:&quot;GENETICALLY MODIFIED YEAST OF THE SPECIES ISSATCHENKIA ORIENTALIS AND CLOSELY RELATED SPECIES AND FERMENTATION PROCESSES USING SAME&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;GENETICALLY MODIFIED YEAST OF THE SPECIES ISSATCHENKIA ORIENTALIS AND CLOSELY RELATED SPECIES AND FERMENTATION PROCESSES USING SAME&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Cells of the species Issatchenkia orientalis and closely related yeast species are transformed with a vector to introduce an exogenous lactate dehydrogenase gene. The cells produce lactic acid efficiently and are resistant at low pH, high lactate titer conditions.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>This invention was made under contract no. DE-FC07-02ID 14349 with the United States Department of Energy. The United States Government has certain rights to this invention.<br>
This application claims benefit of United States Provisional Patent Application 60/686,899, filed June 2, 2005.<br>
This invention relates to certain genetically modified yeast species.<br>
Certain organic acids such as lactic acid are manufactured through an industrial fermentation process. The fermentation is conducted using various types of bacterial species, which consume sugars (principally glucose) and convert those sugars to the desired acid.<br>
There are several reasons why it would be desirable to develop a yeast or fungal biocatalyst for producing organic acids from sugar substrates. Many bacteria are unable to synthesize some of the amino acids or proteins they need to grow and metabolize sugars efficiently. As a result, bacteria often must be fed a somewhat complex package of nutrients. This increases the direct expense required to operate the fermentation. The increased complexity of the broth makes it more difficult to recover the fermentation product in reasonably pure form, so increased operating and capital costs are incurred to recover the product. On the other hand, many yeast species can synthesize their needed amino acids or proteins from inorganic nitrogen compounds. They often grow and ferment well in so-called "defined" media, \\hidi are simplified, often less expensive and present fewer difficulties in product recovery operations.<br>
Another reason that yeast are of interest as a biocatalyst for organic acid production has to do with the nature of the product itself. To have an economically viable process, a high concentration of the organic acid product must accumulate in the fermentation broth. In addition to the normal concerns about toxieity (the fermentation product may be toxic to the biocatalyst when present in high concentrations), an additional concern about acidity exists when the fermentation product is an acid. The media will become increasingly acidic as more of the organic acid is produced. Most bacteria that produce these organic acids do not perform well in strongly acidic environments—they either do not survive under those conditions or else produce the product so slowly as to be economically unviable.<br>
For this reason, commercial acid fermentation processes are buffered by the addition of an agent which neutralizes the acid as it formed. This maintains the broth at or near a neutral pH and allows the bacteria to grow and produce efficiently. However, this converts the acid to a salt, which subsequently must be split to obtain the product in its desired acid form.<br>
The most common buffering agent is a calcium compound, which neutralizes the organic acid to form the corresponding calcium salt. After the calcium salt is recovered from the fermentation broth, it is split by the addition of a mineral acid, typically sulphuric acid, to regenerate the organic acid and form an insoluble calcium salt of the mineral acid. This process therefore involves direct expense for the buffering agent and mineral acid, as well as costs for handling and disposing the unwanted calcium salt by-product. These costs could be reduced or eliminated if the biocatalyst could grow and produce efficiently under lower pH conditions.<br>
Yeast species have been considered as candidates for such low pll fermentations. Many yeast species naturally ferment hexose sugars to ethanol, hut few if any naturally produce desired organic acids such as lactic acid. Efforts have been made to genetically modify various yeast species to insert one or more genes that will enable the cell to produce lactic acid. In order to divert sugar metabolism from ethanol production to lactic acid production, these cells have also heen genetically modified to disrupt or delete the native pyruvate decarboxylase (PDC) gene. This work is described, for example, in WO 99/14335, WO 00/71738 Al, \YO 02/42471 A2, WO 03/049525 A2, WO 03/102152 A2 and WO 03/102201A2. Much of the efforts described in these various publications center on Kluyveromyces species, such as K. marxianus, and certain species classified under the genus Candida, such as C. sonorensis and C. methanosorbosa.<br>
There remains a desire to provide even better biocatalysts for organic acid fermentation processes. A biocatalyst for these fermentation processes desirably can achieve high volumetric and specific productivities; a high yield of the desired organic acid from the fermentation substrate; the ability to grow and produce with reasonable efficiency under acidic conditions; and the ability to grow and produce with reasonable efficiency under microaerobic and especially anaerobic conditions. The biocatalyst preferably can achieve these results using a simplified defined media.<br>
In one aspect, this invention is a genetically modified yeast cell of a species within the Issatchenkia orientalis/Pichia fermentans clade having at least one exogenous lactate dehydrogenase gene integrated into its genome.<br>
This invention is also a fermentation process in which a genetically modified yeast cell of the first aspect is cultured under fermentation conditions in a fermentation broth that includes a fermentable sugar to produce lactic acid or a salt thereof.<br>
This invention is in addition a fermentation process in which a genetically modified yeast cell of the first aspect is cultured under fermentation conditions in a fermentation broth that includes a fermentable sugar to produce lactic acid or a salt thereof, wherein the pH of the fermentation broth during at least a portion of the period of fermentation is in the range of from about 1.5 to about 4.5.<br>
It has surprisingly been found that modified cells of the invention exhibit an excellent tolerance to moderately low pH, high lactic acid titer conditions, and can grow and produce lactic acid at good rates in an unbuffered medium, even under anaerobic or quasi-anaerobic conditions. In addition, the modified cells grow well on defined media.<br>
Figure 1 is a diagram depicting the pMI318 plasmid.<br>
Figure 2 is a diagram depicting the pM!320 plasmid:<br>
Figure 3 is a diagram depicting the pMI321 plasmid.<br>
Figure 4 is a diagram depicting the pMI355 plasmid.<br>
Figure 5 is a diagram depicting the pMI356 plasmid.<br>
Figure 6 is a diagram depicting the pMI357 plasmid.<br>
Figure 7 is a diagram depicting the pM!319 plasmid.<br>
Figure 8 is a diagram depicting the pMI433 plasmid.<br>
Figure 9 is a diagram depicting the pMM25 plasmid.<br>
Figure 10 is a diagram depicting the pMM28 plasmid.<br>
Figure 11 is a diagram depicting the pMI445 plasmid.<br>
Figure 12 is a diagram depicting the pMM35 plasmid.<br>
Figure 13 is a diagram depicting the pMI446 plasmid.<br>
Figure 14 is a diagram depicting the pMI447 plasmid.<br>
Figure 15 is a diagram depicting the pMI448 plasmid.<br>
Figure 16 is a diagram depicting the pMI457 plasmid.<br>
Figure 17 is a diagram depicting the pMI458 plasmid.<br>
Figure 18 is a diagram depicting the pMI449 plasmid.<br>
Figure 19 is a diagram depicting the pMI454 plasmid.<br>
Figure 20 is a diagram depicting the pBH165 plasmid.<br>
Figure 21 is a diagram depicting the pMI464 plasmid.<br>
The genetically modified yeast of the invention is made by performing certain genetic modifications to a host yeast cell. The host cell is of a species contained within the 7. orientalis/P. ferrnentans clade. This clade is the most terminal clade that contains at least the species Issatchenkia orientalis, Pichia galeiformis, Pichia sp. YB-4149 (NRRL designation), Candida ethanolica, P. descrticola, P. mernbranifaciens and P. ferrnentans. Members of the I. orientalis/P. fcnncnl<in clade are identified by analysis of the variable d1 domain ri bosom a i dna yeast species using method described kurtzman and robnett in phylogeny ascomycetous yeasts from nuclear large subunit ribosomal partial sequences antonie van leeuwenhock incorporated herein reference. see especially p. hundreds ofascomycet.es has revealed that orientalis ferrnentans contains very closely related species. members exhibit greater similarity to other than outside clade. therefore can be comparison domains their respective comparing methods.></in>
When first characterized, the species 7. orientalis was assigned the name Pichia kudriavzevii. The anamorph (asexual form) of 7. orientalis is known as Candida krusei.<br>
A particularly suitable host cell is 7. orientalis strain ATCC 32196. Another particularly suitable host cell is 7. orientalis strain ATCC PTA-6658.<br>
The cell of the invention contains at least one functional, exogenous lactate dehydrogenase (LDH) gene integrated into its genome. An LD77 gene is any gene that encodes for a lactate dehydrogenase enzyme, i.e., one having lactate dehydrogenase activity. "Lactate dehydrogenase activity" refers to the ability of the protein to catalyze the reaction of pyruvate to lactate. Lactate dehydrogenase<br>
en/ymes   include   (but   are   not   limited   to)   those   categorized   by   the   Kn/ymc Commission numbers 1.1.1.27 and 1.1.1.28.<br>
In this context, "exogenous" means that the genetic material under consideration (in this case, the LDH gene) is not native to the host strain. The term "native" is used herein with respect to genetic materials (e.g., a gene, promoter or terminator) that are found (apart from individual-to-individual mutations which do not affect function) within the genome of wild-type cells of the host cell.<br>
The LDH gene may enable the modified cell to produce either the L- or D-lactic acid stereoisomer. It is possible that the modified cell of the invention contains both L- and D-LDH genes, and thus is capable of producing both lactic acul stereoisomers. However, it is preferred that only L- or only D-LDH genes are present, so the cell produces a more optically pure lactic acid product.<br>
Suitable LDH genes include those obtained from bacterial, fungal, yeast or mammalian sources. Examples of specific L-LDH genes are those obtained from Lactobacillus helveticus, L. casei, Bacillus rnegaterium, Pediococcus (widilaclici. Rhizopus oryzae and bovine sources such as Bos taurus. Examples of specific: I)-MM) genes are those obtained from L. helveticus, L. johnsonii, L. bulguricun. //. delbrueckii, L. pla.nta.rum, L. pentosus and P. acidilactici. Functional genes that are identical to such L-LDH or D-LDH genes or which have an identities score of a! least 35%, 60%, 70% or 80% relative to such genes at the amino acid level are suitable. The native genes obtained from any of these sources may be subjected to mutagenesis if necessary to provide a coding sequence starting with the usual eukaryotic starting codon (ATG), or for other purposes. A preferred L-LDH gene is that obtained from /,. helveticus or one that has an identities score relative to such gene of at least 35%. 60%, 70%, 80%, 85%, 90% or 95%. Another preferred L-LDH gene is that obtained from B, rnegaterium or one that has an identities score of at least 35%, 60%, 70%, 80%, 85%, 90% or 95% compared with such gene. Another preferred L-LDH gene is that obtained from Bos. taurus or one that has an identities score of at least 35'};., 60%, 70%, 80%, 85%, 90% or 95% compared with such gene. A preferred D-LDH gene is that obtained from L. helveticus or one that has an identities score of at least 45%, 60%, 70%, 80%, 85%, 90% or 95% compared with such gene.<br>
Identities scores of amino acid sequences of DNA, RNA or proteins are, for purposes of this invention computed using BLAST version 2.2.1 software with default parameters.<br>
Particularly suitable LDH genes include those that encode for an en/yme with an amino acid sequence that has an identities score of at least 60%, especially at least 80%, 85% or 95%, compared with the sequence identified as SEQ. ID. NO. 93 (which appears as SEQ. ID. NO. 45 in WO 03/049525) or compared with that identified as SEQ. ID. NO. 94 (which appears as SEQ. ID. NO. 49 in WO 03/0-19525). Particularly suitable LDH genes also include those that encode an enzyme having a protein sequence that has an identities score of at least 60%, 80%&gt;, 85% or 95% compared to SEQ. ID. NO. 95 or SEQ. ID. NO. 96 (which appear as SEQ TD. NO. Hi and 50, respectively, in WO 03/049525).<br>
The transformed cell may contain a single LDH gene or multiple LDH gem-s, such as from 1-10 LDH genes, especially from 1-5 LDH genes. When the transformed cell contains multiple LDH genes, the individual genes may be copies of the same gene, or include copies of two or more different LDH genes. Multiple copies of tIn-exogenous LDH gene may be integrated at a single locus (so they are adjacent to each other), or at several loci within the host cell's genome.<br>
The exogenous LDH gene is under the transcriptional control of one or more promoters and one or more terminators, both of which are functional in the modi tied yeast cell. As used herein, the term "promoter" refers to an untranslated sequence located upstream (i.e., 5') to the translation start codon of a structural gene (generally within about 1 to 1000 bp, preferably 1-500 bp, especially 1-100 bp) and which controls the start of transcription of the structural gene. Similarly, the term "terminator" refers to an untranslated sequence located downstream (i.e., 3') to the translation finish codon of a structural gene (generally within about 1 to 1000 bp, more typically 1-500 base pairs and especially 1-100 base pairs) and which controls the end of transcription of the structural gene. A promoter or terminator is "operatively linked" to a structural gene if its position in the genome relative to that of the structural gene is such that the promoter or terminator, as the case may he, performs its transcriptional control function.<br>
Promoters and terminator sequences may be native to I. orientalis or exogenous to the cell. Useful promoter and terminator sequences include those that are highly identical (i.e., have an identities score of 90% or more, especially 95% or more, most preferably 99%) or more) in their functional portions compared to the functional portions of promoter and terminator sequences, respectively, that are<br>
native to the host cell, particularly when the insertion of the exogenous gene is targeted at a specific site in the cell's genome.<br>
One suitable type of promoter has an identities score at least 90%, 95% or 99% relative to a promoter that is native to a yeast gene. A more suitable type of promoter has an identities score of at least 90%, 95% or 99% compared to a promoter for a gene that is native to the host cell. Particularly useful promoters include promoters for yeast pyruvate decarboxylase (PDC), phosphoglycerate kinase (PCJK), and transcription elongation factor-1 (TEF-1) genes, especially from the respective /. orientalis genes. An especially useful promoter includes the functional portion of a promoter for an /. orientalis PGK gene.<br>
One suitable type of terminator has an identities score of at least 90%, 95% or 99% compared to a terminator for a gene that is native to a yeast. The terminator may have an identities score at least 90%, 95% or 99% homologous to a terminator for a gene that is native to the host cell. Particularly useful terminators include terminators for yeast pyruvate decarboxylase (PDC), xylose reductase, (XR). xylitol dehydrogenase (XDH) or iso-2-cytochrome c (CYC) genes, or a terminator from the galactose family of genes in yeast, particularly the so-called GAL10 terminator-. An especially preferred terminator includes a functional portion of a terminator for a PDC gene of the host cell.<br>
The use of native (to the host cell) promoters and terminators, together with their respective upstream and downstream flanking regions, can permit the targeted integration of the LDH gene into specific loci of the host cell's genome, and for simultaneous integration the LDH gene and deletion of another native gene, such as, for example, a PDC gene.<br>
It is possible for different exogenous LDH genes to be under the control of different types of promoters and/or terminators.<br>
The exogenous LDH gene may be integrated randomly into the host cell's genome or inserted at one or more targeted locations. Examples of targeted locations include the loci of a gene that is desirably deleted or disrupted, such as a PDC gene. Integration at the PDC locus may be accomplished with or without deletion or disruption of the native PDC gene, but it is generally preferred to disrupt or delete the PDC gene, so the modified cell produces less ethanol.<br>
The host cell may contain multiple PDC genes. Native /. orientalis cells, for example, contain two PDC genes, which are designated herein as loPDClA and<br>
loPDClB. In some strains, including ATCC PTA-6658, these are visualized as ~S kbp and -10 kbp Hindlll bands, respectively, upon Southern analysis of the wild-type organism. Other /. orientalis strains, such as ATCC 32196, appear to have two alleles that produce bands of similar size. When the host cell contains multiple IM.)C genes, it is preferred to delete or disrupt at least one of them and more preferred to disrupt all of them, as this destroys the cell's ability to produce ethanol. Thus, in /. orientalis, it is preferred to disrupt the loPDCIA or loPDClB genes and more preferred to delete or disrupt both the loPDCIA and loPDClB genes.<br>
By "delete or disrupt", it is meant that the entire coding region of the gene is eliminated (deletion), or the gene or its promoter and/or terminator region is modified (such as by deletion, insertion, or mutation) so that the gene no longer produces an active enzyme, or produces an enzyme with severely reduced activity. The deletion or disruption can be accomplished by genetic engineering methods, forced evolution or mutagenesis and/or selection or screening. A preferred way of accomplishing this is to replace the PDC gene with a cassette containing an LDH gene, as described more fully below.<br>
Genetic modification of the host cell is accomplished in one or more steps via the design and construction of appropriate vectors and transformation of the host cell with those vectors. Electroporation and/or chemical (such as calcium chloride- or lithium acetate-based) transformation methods can be used. Methods for transforming yeast strains to insert an exogenous LDH gene are described in WO 99/14335, WO 00/71738, WO 02/42471, WO 03/102201, WO 03/102152 and WO 03/049525; these methods are generally applicable for transforming I. orientalis cells in accordance with this invention. The vectors can either be cut with particular restriction enzymes or used as circular DNA.<br>
In general, a vector is prepared that contains the LDH gene and associated promoter and terminator sequences. The vector may contain restriction sites of various types for linearization or fragmentation. Vectors may further contain a backbone portion (such as for propagation in E. coli) many of which are conveniently obtained from commercially available yeast or bacterial vectors.<br>
The vector preferably contains one or more selection marker gene cassettes. A "selection marker gene" is one that encodes a protein needed for the survival and/or growth of the transformed cell in a selective culture medium. Typical selection marker genes encode proteins that (a) confer resistance to antibiotics or other toxins<br>
such as zeocin (sh ble gene from Streptoalloteichus hindustcmus), G418 (kanamycin resistance gene of Tn903), hygromycin (aminoglycoside antibiotic resistance gene from E. coli), ampicillin, tetracycline, or kanamycin), (b) complement auxotropliic deficiencies of the cell. Two prominent examples of auxotrophic deficiencies are the amino acid leucine deficiency (e.g. LEU2 gene) or uracil deficiency (e.g. UK A3 gene). Cells that are orotidine-5'-phosphate decarboxylase negative (ura3-) cannot grow on media lacking uracil. Thus a functional URA3 gene can be used as a marker on a cell having a uracil deficiency, and successful transformants can be selected on a medium lacking uracil. Only cells transformed with the functional URA3 gene are able to synthesize uracil and grow on such medium. If the wild-type strain does not have a uracil deficiency (as is the case with /. orientalis, for example), an auxotrophic mutant having the deficiency must be made in order to use URA3 as a selection marker for the strain. Methods for accomplishing this are well known in the art.<br>
Preferred selection makers include the zeocin resistance gene, G418 resistance gene, hygromycin resistance gene and MEL5 (melibiase gene). The selection marker cassette will further include a promoter and terminator sequence, operatively linked to the selection marker gene, and which are operable in 7. orientalis. Suitable promoters include those described above with respect to the LDH gene, as well as others such as are described in WO 99/14335, WO 00/71738, WO 02/42471, WO 03/102201, WO 03/102152 and WO 03/049525. An especially preferred promoter is a PGK or PDC promoter (or functional portion thereof) of the host strain, or a sequence having an identities score of at least 80, 85, 90 or 95% compared to such a PGK or PDC promoter. Suitable terminators include those described above with respect to the LDH gene. Either the promoter or terminator (or both) may be the same as that used with the LDH gene.<br>
Targeted integration can be accomplished by constructing a vector having regions that are highly identical to (i.e. identities score of 80% or more, preferably 95% or more and most preferably 100%) to the upstream (5'-) and downstream (.T-) flanks of the target gene. Either or both of these regions may include a portion of the coding region of the target gene as well as a portion or all of the respective promoter or terminator regions. The LDH cassette (including associated promoters and terminators if different from those of the target gene) and selection marl<er associated promoters and terminators as may be needed will reside on the vector between regions that are highly identical to upstream downstream flanks></er>
of the target gene. As mentioned, a preferred target gene is the loPDCIA nr loPDClB gene (or both) as disruption or deletion of one or both of those genes is preferred. However, other native genes may serve as targets for insertion of I he LIHI gene cassette.<br>
Successful transformants can be selected for in known manner, by taking advantage of the attributes contribvited by the marker gene, or by other characteristics (such as ability to produce lactic acid, inability to produce ethanol, or ability to grow on specific substrates) contributed by the inserted genes. Screening can be performed by PCR or Southern analysis to confirm that the desired insertions and deletions have taken place, to confirm copy number and to identify the point of integration of genes into the host cell's genome. Activity of the enzyme encoded by the inserted gene and/or lack of activity of enzyme encoded by the deleted gene can be confirmed using known assay methods.<br>
Deletion or disruption of the PDC genes can be accomplished in a variety of ways, including, for example, methods analogous to those described in WO 99/14335, WO 02/42471, WO 03/049525, WO 03/102152 and WO 03/102201. In a method of particular interest (with respect to transforming I. orientalis), (1) the 5' and 3' flanking regions of one of the /. orientalis PDC genes are cloned, optionally together with a portion of the functional PDC gene; (2) a vector containing the 5' and 3' flanking regions is produced, and (3) the /. orientalis cell is transformed with the vector. A homologous recombination event results in a deletion of the functional PDC' gene. It has been found that the 5' and 3' flanking regions for loPDCIA can be used in the vector to delete or disrupt both loPDCIA and loPDClB. Analogous methods may be used to disrupt or delete one or more PDC genes in other host cells useful in the invention.<br>
The PDC deletion or disruption vector may include one or more functional structural genes, notably an LDH gene as described above, inserted between the f&gt;' and 3' flanking portions of one of the PDC genes of the host cell. The functional gene preferably includes functional promoter and terminator sequences operatively linked to the structural gene. This approach allows for the simultaneous deletion of the PDC gene and insertion of the functional gene. The vector may include a select ion marker gene instead of or in addition to the structural gene. Again, the selection marker gene is positioned on the vector between the 5' and 3' flanking portions of the PDC gene being targeted, and becomes inserted in the locus of the functional PDC<br>
gene. The use of a selection marker gene has the advantage of introducing a mcnns of selecting for successful transformants. However, it may be possible to .select for successful transformants based on their reduced or eliminated ability to produce ethanol, especially in transformants having disruptions or deletions of multiple I'DC genes (such as both the loPDCIA and loPDClB genes of/, orientalis).<br>
In /, orientalis, it is possible to eliminate both loPDCIA and loPDClB in a single step by transforming the host strain with a single vector containing the .V flank of either target gene, a functional gene cassette including associated promoters and terminators, and/or a selection marker gene cassette including associated promoters and terminators, and the 3' flank of either target gene. An example of such a vector is that designated pMI356 in Example 2B. Transformations of /. orientalis with such a vector produces transformants having a single PDC allele deleted and transformants having a deletion of both the loPDCIA and JnPDCIB alleles. Typically, at least one of the PDC alleles is replaced by a functional 1 J)TI gene (or other structural gene). Again, analogous methods are applicable with respect to other host cells having multiple PDC alleles.<br>
Alternatively, it is possible in I. orientalis to delete loPDCIA and foPDCJH in a two-step process. For example, /. orientalis can be transformed with a vector as described above, and single PDC deletion strains transformed a second time with a like or similar vector to eliminate the second PDC allele. Examples 2D and 3B below illustrate such an approach.<br>
The genetically modified yeast cell of the invention may include additional genetic modifications that provide some desired attribute to the cells. Additional modification(s) of particular interest confer to the cell the ability to ferment pcntosc sugars to desirable fermentation products. Among such modifications are (1) insertion of a functional exogenous xylose isomerase gene, (2) a deletion or disruption of a native gene that produces an enzyme that catalyzes the conversion of xylose to xylitol, (3) a deletion or disruption of a functional xylitol dehydrogenase gene and/or (4) modifications that cause the cell to overexpress a functional xylulokinase. Methods for introducing such modifications into yeast cells are described, for example, in WO 04/099381, incorporated herein by reference.<br>
In the fermentation process of the invention, the cell of the invention is cultivated in a fermentation medium that includes a sugar that is fermentable by the transformed cell. The sugar may be a hexose sugar such as glucose, glycan or other<br>
polymer of glucose, glucose oligomers such as maltose, maltotriose and isomaltotnose, panose, fructose, and fructose oligomers. If the cell is modified to impart an ability to ferment pentose sugars, the fermentation medium may include a pentose sugar such as xylose, xylan or other oligomer of xylose. Such pentose sugars are suitably hydrolysates of a hemicelluose-containing biomass, In case of oligomeric sugars, it may be necessary to add enzymes to the fermentation broth in order to digest these to the corresponding monomeric sugar for fermentation by the cell.<br>
The medium will typically contain nutrients as required by the particular coll, including a source of nitrogen (such as amino acids, proteins, inorganic nitrogen sources such as ammonia or ammonium salts, and the like), and various vitamins, minerals and the like. /. orientalis is capable of satisfying its needs for nitrogen, phosphorus and magnesium from inorganic sources and so is capable of growing and fermenting in a chemically defined medium containing inorganic sources of these elements. Thus, the cells of the invention can be cultured in such a chemically defined medium. However, it is also possible to culture the cells of the invention in a complex medium that is not chemically defined and which may contain organic nitrogen sources such as proteins, partially digested proteins, or amino acids.<br>
Other fermentation conditions, such as temperature, cell density, selection of substrate(s), selection of nutrients, and the like are not considered to be critical to the invention and are generally selected to provide an economical process. Temperatures during each of the growth phase and the production phase may range from above the freezing temperature of the medium to about 50°C. A preferred temperature, particularly during the production phase, is from about 30-45°C.<br>
During the production phase, the concentration of cells in the fermentation medium is typically in the range of about 0.1-20, preferably about 0.1-5, even more preferably about 1-3 g dry cells/liter of fermentation medium.<br>
The fermentation may be conducted aerobically, microaerobically or anaerobically. Quasi-anaerobic conditions, in which no oxygen is added during the fermentation but dissolved oxygen is present in the fermentation medium at the start of the fermentation, can also be used. If desired, specific oxygen uptake rate can be used as a process control, as described in WO 03/102200. The cells of the invention exhibit a good ability to ferment sugars to lactic acid or lactic acid/ethanol mixtures. at good volumetric and specific productivities under even anaerobic conditions.<br>
When the fermentation product is an acid, the medium may be buffered during the production phase of the fermentation so that the pH is maintained in a range of about 5.0 to about 9.0, preferably about 5.5 to about 7.0. Suitable buffering agents are basic materials that neutralize lactic acid as it is formed, and include, for example, calcium hydroxide, calcium carbonate, sodium hydroxide, potassiunv hydroxide, potassium carbonate, sodium carbonate, ammonium carbonate, ammonia. ammonium hydroxide and the like. In general, those buffering agents that have been used in conventional fermentation processes are also suitable here.<br>
In a buffered fermentation, acidic fermentation products such as lactic acid are neutralized to the corresponding salt as they are formed. Recovery of the acid therefore involves regenerating the free acid. This is typically done by removing tin-cells and acidulating the fermentation broth with a strong acid such as sulfuric acid. A salt by-product is formed (gypsum in the case where a calcium salt is the neutralizing agent and sulfuric acid is the acidulating agent), which is separated from the acid. The acid is then recovered through techniques such as liquid-liquid extraction, distillation, absorption, etc., such as are described in T.B. Vickroy, Vol. 3, Chapter 38 of Comprehensive Biotechnology, (ed. M. Moo-Young), Pergamon, Oxford, 1985; R. Datta, et al., FEMS Microbiol. Rev., 1995; 16:221-231; U.S. Patent Nos. 4,275,234, 4,771,001, 5,132,456, 5,420,304, 5,510,526, 5,641,406, and 5,831,122, and WO 93/00440.<br>
It is preferred, however, to conduct the fermentation so that the pH of the medium at the end of the fermentation is at or below the pKa of the acid fermentation product. A suitable final pH is suitably in the range of about 1.5 to about 3.5, in the range of from about 1.5 to about 3.0, or in the range from about 1.5 to about 2.5. The starting pH may be somewhat higher, such as from about 3.5 to about 6.0, especially from about 3.5 to about 5.5, or may be adjusted to a more acidic pH of 1.5 to about 3.5. The cells of this invention have been shown to have an unexpected ability to grow and produce well even in acidic fermentation media where the pH is below 3.5, below 3.0, below 2.5, and even below 2.0. /. orientalis cells with only basic genetic modifications, such as the insertion of an LDH gene and deletion of loPDClA and loPDClB genes, have been found to produce from 15-20 g/L lactic acid under anaerobic conditions in an unbuffered medium originally containing 10% by weight glucose.<br>
Recovery of lactic acid from a low pH fermentation medium can he conducted using methods such as those described in U. S. Patent Nos. 6,229,046.<br>
The process of the invention can be conducted continuously, batch-wise, or some combination thereof.<br>
The following examples are provided to illustrate the invention, but arc not intended to limit the scope thereof. All parts and percentages are by weight unless otherwise indicated.<br>
Example 1A: Cloning of /. orientalis PGK (loPGKl) promoter region; construction of a plasmid (pMI318, Fig. 1) having the E. coll hygromycin gene under the control of the loPGKl promoter and the S. cerevisiae GAL JO terminator.<br>
A 920 bp probe fragment of the C, sonorensis PGK1 gene is obtained from the<br>
genomic DNA of C. sonorensis in the same manner as described in Example 22 of WO 02/042471, using primers identified as SEQ. ID. NO. 1 and SEQ. ID. NO. 2. Genomie DNA is isolated from a growing /. orientalis strain and resuspended in 10 mM Tris-HC1 + 1 mM EDTA (pH 8) (TE). The /. orientalis genomic DNA is cut with Hind]]], and a Southern blot is prepared and hybridized using standard methods with the C. sonorensis PGK1 gene as a probe. Fragments of ~2 kb size are isolated from agarose gel and cloned into a Hindlll-cut plasmid. Colony hybridization of the K. co/i transformants with the PGK1 probe result in isolation of a Hindlll fragment containing most of the /. orientalis PGKl (loPGKl) protein coding sequences but no promoter sequences, as verified by sequencing.<br>
Genomic fragments containing the loPGKl promoter region are obtained with ligation-mediated PCR amplification (Mueller, P.R. and Wold, B, 1989, "In vivo footprinting of a muscle specific enhancer by ligation mediated PCR." Science 246:780-786). A mixture of a linker identified as SEQ. ID. NO. 3 and a linker identified as SEQ. ID. NO. 4 is ligated to Haelll-digested 7. orientalis genomic DNA with T4 DNA ligase (New England BioLabs). Samples of the ligation mixtures are used as templates for 50 ul PCR reactions containing 0.1 uM of a primer identified as SEQ. ID. NO. 5 and 1 uM of a primer identified as SEQ. ID. NO. 6. The reaction mixture is heated at 94°C for 3 minutes after 2 U of Dynazyme EXT is added. The reactions are cycled 30 times as follows: 1 minute at 94°C, 2 minutes at 68°C and 2 minutes at 72°C, with a final extension of 10 minutes at 72°C. A diluted sample of this first PCR-amplification is used as the template in a nested PCR reaction (50 ul) containing 0.05 uM of a primer identified as SEQ. ID. NO. 7 and 0.5 uM of a primer<br>
identified as SEQ. ID. NO. 8. The reaction mixture is heated at 94°C for 3 minutes after 2 U of Dynazyme EXT is added. The reactions are then cycled 30 times as follows: 1 minute at 94°C, 2 minutes at 67°C and 2 minutes at 72°C, with a final extension of 10 minutes at 72°C.<br>
A -600 bp PCR fragment is isolated and sequenced. Nested primers identified as SEQ. ID. NO. 9 and SEQ. ID. NO. 10 are designed and used in a ligation-medial.ed PCR amplification together with oligonucleotides identified as SEQ. ID. NO. 11 and SEQ. ID. NO. 12 similarly as above except that digested I. orientalis DNA is used and the PCR is carried out using an annealing temperature of 65°C.<br>
The I. orientalis PGK1 promoter is PCR amplified using primers identified as SEQ. ID. NO. 13 and SEQ. ID. NO. 14 and the /. orientalis genomic DNA as the template. The fragment is filled in using the Klenow enzyme and then cut with Xlxil. A 633 bp fragment is gel isolated and ligated to a 4428 bp fragment obtained by digesting a plasmid designated as pMI270 (described in Fig. 4 of WO 03/049525) with A7?.ol, filling the fragment in using the Klenow enzyme and 0.1 mM dNTP, and digesting with Xbal. Plasmid pMI270 contains the E. coli hygromycin gene linked to a C. sonorensis PGKl promoter and a S. cerevisiae GAL10 terminator. The resulting plasmid is designated pMI318 (Fig. 1). Plasmid pMI318 contains the E. coli hygromycin gene under the control of the 7. orientalis PGKl promoter and the S. cerevisiae GAL 10 terminator.<br>
Example IB: Construction of a plasmid (pMI321, Fig. 3) containing the hygromycin gene under the control of the loPGKl promoter and the S. cerevisiae GAL10 terminator, and the L. helveticus LDH gene under the control of the loPGKl promoter and S. cerevisiae CYC1 terminator.<br>
The /. orientalis PGKl promoter from Example 1A is PCR amplified using<br>
primers identified as SEQ. ID. NO. 15 and SEQ. ID. NO. 16 and the I. orientalis genomic DNA as the template. The fragment is filled in using the Klenow enzyme and 0.1 mM dNTP, and then cut with Ncol. A 633 bp fragment is gel isolated.<br>
Plasmid pVRl (described in WO 03/102152 Figure 7) contains the L. helvi'ticus LDH gene under the control of the S, cerevisiae TEF1 promoter and the S. cererisi.ac CYCl terminator. Plasmid pVRl is digested with Xhol, filled in using the Klenow enzyme, and cut with Ncol. A 7386 bp fragment from plasmid pVRl is ligated to the 633 bp loPGKl promoter fragment. The resulting plasmid is designated pMI320 (Fig. 2). Plasmid pMI320 contains the L. helveticus LDH gene under the control of the loPGKl promoter and S. cerevisiae CYCl terminator.<br>
Plasmids pMI318 (Ex. 1A, Fig. 1) and pMI320 are digested with Apa\ and Not}, A 5008 bp fragment from plasmid pMI318 is ligated to a 1995 bp fragment from plasmid pMI320 to form plasmid pMI321 (Fig. 3).<br>
The hygromycin gene (and its terminator) is positioned on plasmid pMITV21 between two copies of the JoPGKl promoter. This construct can permit a cell transformed with plasmid pMI321 to engage in a homologous recombination to "loop out" the hygromycin gene and terminator, together with one copy of the JoPGKl promoter.<br>
Example 1C: Generation of an /. orientalis mutant (CD 990) with integrated LDH gene and hygromycin resistance genes by transforming wild-type I. orientalis with partially digested plasmid pMI321 (Fig. 3, Ex. IB).<br>
Plasmid pMI321 is partially restricted with Xhol, and the resulting mixture of<br>
linear and circularized DNA is used to transform a wild-type 7. orientalis strain designated as ATCC PTA-6658, using standard lithium acetate methods as described in Gietz et al. (1992, Nucleic Acids Rs. 20:1425), The transformed cells are screened for hygromycin resistance. Several hygromycin-resistant colonies are cultured and the culture medium analyzed for the production of lactic acid. A hygromycin-resistant colony that produces lactic acid is designated strain CD990.<br>
Example ID: Generation of an /. orientalis mutant with an integrated LDH gene and hygromycin resistance gene by transforming wild-type 7. orientalis with partially digested plasmid pMI321 (Fig. 3, Ex. IB).<br>
Plasmid pMI321 is partially restricted with Xhol, and the resulting mixture of<br>
linear and circularized DNA is used to transform wild-type I, orientalis strain ATCC 32196, using standard lithium acetate methods as described before. The transformed cells are screened for hygromycin resistance. Several hygromycin-resistant colonies are cultured and the culture medium analyzed for the production of lactic acid. Several colonies are found to produce lactic acid.<br>
Example IE: Generation of an 7. orientalis mutant with an integrated LDH gene and hygromycin resistance gene by transforming wild-type 7. orientalis with partially digested plasmid pMI320 and digested plasmid pMI321 (Figs. 2, 3, Ex. IB).<br>
Plasmid pMI320 is partially digested with Xhol.  Plasmid pMI321 is digested<br>
with Srnal and Sail. The digested materials are combined and used to transform wild-type 7. orientalis strain ATCC PTA-6658, using standard lithium acetate methods as described before. The transformed cells are screened for hygromycin<br>
resistance. Several hygromycin-resistant colonies are cultured and the culture medium analyzed for the production of lactic acid. Several colonies are found to produce lactic acid.<br>
Example IF: Generation of an /. orientalis mutant with an integrated LDf'f gene and hygromycin resistance genes by transforming wild-type /, orientalis with partially digested plasmid pMI320 and digested plasmid pMI321 (Figs 2, 3, Ex. IB).<br>
Plasmid pMI320 is partially digested with Xhol.  Plasmid pMI321 is digested<br>
with Smal and Sail. The digested materials are combined and used to transform a wild-type /. orientalis strain designated as ATCC 32196, using standard lithium acetate methods as described before. The transformed cells are screened for hygromycin resistance. Several hygromycin-resistant colonies are cultured and the culture medium analyzed for the production of lactic acid. One colony is found to produce lactic acid.<br>
Example 2A: Cloning of /. orientalis PDC (loPDCIA) promoter region; construction of a plasmid (pMI355, Fig. 4) having the E, coli hygromycin gene under the control of the loPGKl promoter and the S. cerevisiae GAL10 terminator, the L. helveticus LDH gene under the control of the loPGKl promoter and S, cerevisiae CYC1 terminator, and the loPDCIA 5' flanking region.<br>
A genomic library of the  native /.  orientalis strain ATCC PTA-6658  is<br>
constructed into the SuperCosl (Stratagene) cosmid vector according to instructions provided by the manufacturer. PDC-like sequences are amplified by PCR from the genomic DNA of the strain with primers designated as SEQ. ID. NO. 17 and SEQ. ID. NO. 18. A 700 bp fragment of a PDC gene is amplified. The genomic library is screened using hybridization techniques with labeled PCR fragments as probes as described in WO 03/049525 and cosmid clones containing the PDC gene are isolated and sequenced. The I. orientalis PDC1A 5' region from 1000 bp to 167 bp upstream of the start of the open reading frame is PCR amplified using primers identified as SEQ. ID. NO. 19 and SEQ. ID. NO. 20 and the /. orientalis PDC cosmid DNA as the template. The amplified gene (from start to finish codons) has the sequence identified as SEQ. ID. NO. 97. The fragment is cut with Sail and Sad. An 836 bp fragment is gel isolated and ligated to a 6992 bp fragment obtained by digesting plasmid pMI321 (Example IB, Fig. 3) with Sail and Sad. The resulting plasmid is named pMI355 (Fig. 4).<br>
Example 2B: Cloning of 7. orientalis PDC (loPDCIA) terminator region; construction of plasmids (pMI356 and pMI357, Figs. 5 and 6) having the loPDCIA 5' flanking region, the E, coli hygromycin gene under the control of the loPGKl promoter and the S. cerevisiae GAL10 terminator, the L. helveticus LDH gene vinder the control of the loPGKl promoter and 8, cerevisiae CYC1 terminator, and the loPDCIA 3' flanking region.<br>
The I. orientalis PDC 3' region corresponding to sequences from 233 bp to 872<br>
bp downstream of the PDC translation stop codon is PCR amplified using primers identified as SEQ. ID. NO. 21 and SEQ. ID. NO. 22 and the 7. orientalis PDCJA cosmid DNA (Example 2A) as the template. The fragment is cut with Apal and Smal. A 630 bp fragment is gel isolated and ligated to a 7809 bp fragment obtained by digesting plasmid pM1355 (Ex. 2A, Fig. 4) with Apal and Smal. The resulting plasmid is named pMI356 (Fig. 5). It contains the hygromycin and LDH cassettes from plasmid pMI355 between the 5' flank and a portion of the 3' flank of the loPDCIA gene.<br>
The /. orientalis PDC1A 3' region corresponding to sequences from 524 bp upstream to 217 bp downstream of the PDC translation stop codon is PCR amplified using primers identified as SEQ. ID. NO. 23 and SEQ. ID. NO. 24 and the /. orientalis PDC cosmid DNA (Example 2A) as the template. The fragment is cut with Apal and Smal. A 764 bp fragment is gel isolated and ligated to a 7809 bp fragment obtained by digesting plasmid pMI355 with Apal and Smal." The resulting plasmid is named pMI357 (Fig. 6). It contains the hygromycin and LDH cassettes from plasmid pMI355 between the 5' flank and a portion of the 3' flank of the loPDCIA gene. Plasmid pM!357 differs from plasmid pMI356 with respect to the portion of the ,'V loPDCIA flank that is present.<br>
Example 2C: Generation of an /. orientalis mutant (ATCC/357-5) with deleted PDC gene and integrated L, helveticus LDH gene and hygromycin resistance gene by transforming wild-type 7. orientalis with plasmid p'MI357 (Fig. 6, Ex. 2B).<br>
Plasmid pMI357 is restricted with Sod and Apal and used to transform /.<br>
orientalis strain ATCC 32196, using standard chemical methods as described before,<br>
Colonies that grow on the hygromycin media are subjected to Southern analysis to confirm the integration of the LDH gene from plasmid pMI357 and to confirm the deletion of the loPDCIA and/or loPDClB allele. A transformam containing the LDH gene and a deletion of one of the loPDCIA or loPDClB alleles is designated as ATCC/357-5.<br>
Example 2D: Generation of 7. orientalis mutants (CD1027 and CD1030) with deleted PDC gene and integrated L. helveticus LDH gene and hygromycin resistance gene, by transforming wild-type /. orientalis with plasmid pMI356 (Fig. 5, Ex. 2A).<br>
Plasmid pMI356 is restricted with SocI and Apal and used to transform /. orientalis strain ATCC PTA-6658, using standard chemical methods as described before.<br>
Colonies that grow on the hygromycin media are selected. Southern analysis of Hindll-Xbal cut genomic DNA hybridized with LhLDH and PDC 5' probes confirm the integration of the LDH gene from plasmid pMI356 and the deletion of the loPDClB gene in a transformant, which is designated as CD1030. A transformant having integrated LDH and a deletion of the loPDCIA gene is designated as CD 1027,<br>
Example 2E: Generation of an 7. orientalis mutant (ATCC/356-23) with deleted loPDCIA gene and integrated L. helveticus LDH gene and hygromycin resistance gene, by transforming wild-type 7. orientalis with plasmid pMI356 (Fig. 5, Ex. 2A).<br>
Plasmid pMI356 is restricted with SocI and Apal and used to transform wild-type /. orientalis strain ATCC 32196, using standard chemical methods as described before.<br>
Colonies that grow on the hygromycin media are selected. Southern analysis of Hindll-Xbal cut genomic DNA hybridized with LhLDH and PDC 5' probes confirm the integration of the LDH gene from plasmid pMI356 and the deletion of one of the loPDCIA or loPDClB alleles in a transformant, which is designated as ATCC/356-23.<br>
Example 3A: Construction of plasmid pM!433 (Fig. 8) containing the loPDCIA 5' flanking region, the ScMELS gene under the control of the loPGKl promoter, the L. helveticus LDH gene under the control of the loPGKl promoter and ScCYCl terminator, and the loPDCIA 3' flanking region.<br>
The /. orientalis PGKl promoter is PCR amplified using primers identified as<br>
SEQ. ID. NO. 25 and SEQ. ID. NO. 26 and the 7. orientalis genomic DNA as the template. The fragment is filled in using the Klenow enzyme and 0.1 mm dNTP, and then cut with Sphl. A 669 bp fragment is gel isolated. A plasmid designated as pM1233 (described in Fig. 23C of WO 03/049525) is cut with Xhol. The fragment is filled in with the Klenow enzyme and cut with Sphl. The 4534 bp and the 669 bp fragments are ligated and the resulting plasmid is named pMI319 (Fig. 7). Plasmid<br>
pMI319 contains the S. cerevisiae MEL5 (ScMELS) gene and the loPGKl promoter region.<br>
Plasmid pMI319 plasmid is cut with Apal, made blunt ended with T4 polymerase, and cut with Noil. A 2317 bp fragment is gel isolated. It is ligated to a 6498 bp fragment obtained by digesting plasmid pMI357 (Example 2B, Fig. 6) with Sail, making it blunt ended with the Klenow enzyme and then cutting with ATo/I. The resulting plasmid contains the ScMELS gene (with its native terminator) in place of the hygromycin gene of plasmid pMI357. The resulting plasmid is designated PMI433 (Fig. 8).<br>
Example 3B: Generation of /. orientalis mutants (C258/433-3 and C258/483-4) with deleted loPDCJA and loPDClB genes and integrated L, helveticus LDH gene and ScMELS gene by transforming mutant strain CD1027 (Ex. 2B) with plasmid pMI433 (Fig. 8, Ex. 3A).<br>
Mutant strain CD 1027 is transformed with a 5.9kb Sad/Apal fragment from<br>
plasmid pMI433 using standard chemical methods. Southern analysis is conducted on transformants that exhibit melibiase activity, using genomic DNA digested with various enzyme combinations and carried out with LhLDH and PDC 5' probes. Two transformants that have lost the loPDClB allele and gained a second copy of the LhLDH gene are designated C258/433-3 and C258/433-4, respectively. However, the integration occurs differently in the two transformants, in that the LhLDH 3' hand corresponding to the LhLDH cassette of plasmid pM!433 appears differently in the two strains. It is not clear whether the inserted LhLDH expression cassette is intact in these transformants.<br>
Example 4: Generation of an /. orientalis mutant (CD1184) with deleted loPDCIA and loPDClB alleles and integrated LhLDH gene in one stop by transforming wild-type 7. orientalis strain with plasmid pMI356 (Fig, 5, Ex. 2B).<br>
7. orientalis strain ATCC PTA-6658 is transformed with plasmid pMI356 using<br>
the general method described in Example 3B. Transformed strains that grow on hygromycin plates are cultured. A transformant that does not produce ethanol is selected for Southern analysis, which confirms the deletion of both the loPDCIA and loPDClB alleles and insertion of at least one copy of the LhLDH gene. This strain is designated CD 1184.<br>
Example 5: Generation of an /. orientalis mutant (CD 1270) with deleted loPDCIA and loPDClB alleles and integrated LhLDH gene in one step by transforming wild-type I, orientalis strain ATCC32196 with plasmid pMI356 (Fig. 5, Ex. 2B).<br>
I. orientalis strain ATCC 32196 is transformed with plasmid pMI356 using the<br>
general method described in Example 3B. Transformed strains that grow on hygromycin plates are cultured. A transformant that does not produce ethanoJ is selected for Southern analysis, which confirms the deletion of both the loPDCIA and loPDClB alleles and the insertion of a copy of the LhLDH gene. This strain is designated CD 1270.<br>
Example 6: Microaerobic shake flask characterizations of strains CD 990 (Ex. 1C) ATCC/357-5 (Ex. 2C), ATCC 356-23 (Ex. 2E), CD1030 (Ex. 2D), CD1184 (Ex. 4) and CD1270 (Ex, 5) in non-buffered defined medium.<br>
Transformants  CD990, ATCC/357-5,  ATCC 356-23,  CD1030,  GDI 184   and<br>
CD 1270 are separately cultivated in 50 mL yeast nitrogen base (YNB) without amino acids, supplemented with 100 g/L glucose in a 250 mL baffled flask. The cultivations are not buffered, so pH within the medium falls as lactic acid is produced, to a final pH of 2.0 ± 0.1 in each instance. Each flask is inoculated to an ODeoo of 0.2 with cells grown on yeast peptone plus glucose plates. The cultivations are maintained at a temperature of 30°C with shaking at 100 rpm. Samples are withdrawn periodically during cultivation, and ODeoo is measured. Cells are recovered from each sample by centrifugation and the supernatant analyzed by HPLC for lactic acid, glucose and ethanol.<br>
HPLC analyses are conducted with a Waters 2690 Separation Module and Water System Interfase Module liquid chromatography coupled with a Waters 2414 differential refractometer and Waters 2487 dual X absorbance detector. The liquid chromatography columns are a 50 X 7.8 mm Fast Juice column from Phenomenex and a 100 X 7.8 mm Fast Acid Analysis column from Bio-Rad. The columns arc equilibrated with 2.5 mM H2S04 in water at 60°C and samples are eluted with 2."&gt; mM HbSCU in water at 0.5 ml/min flow rate. Data acquisition is done using Waters Millennium software.<br>
The single PDC deletant strains (ATCC/357-5, ATCC/356-23 and CD 1030) all produce both ethanol and lactic acid. Glucose consumption for these three .strains is nearly linear throughout the 168 hour cultivation, with each consuming about 50% of the glucose after about 72 hours and essentially all of the glucose after about IfiS hours. Each of these strains produces about 20-24 g/L of ethanol after 168 hour,-.<br>
Lactic acid production peaks after about 96 hours at a level of 15-20 g/L for each of these strains. The strains thereafter consume a small amount of lactic acid. Lactic acid yields for these strains peak at about 35% after 48 hours, and decline thereafter due to lactic acid consumption and continued production of ethanol.<br>
Strain CD990, which has no PDC deletion, performs similarly to the single PDC deletant strains.<br>
The double PDC deletant strains (CD 1184 and CD 1270) produce lactic acid bvit no ethanol. These strains consume glucose more slowly than do the others, with approximately 48-55% of the glucose being unconsumed after 168 hours. Strain CD 1270 produces lactic acid through the first 96 hours of cultivation, after which lactic acid titers increase only slightly. Strain CD 1270 produces a peak lactic acid yield of about 56%. Strain GDI 184 continues to produce lactic acid through the entire cultivation period, with lactic acid titer at the end of the cultivation being about 31 g/L. Lactic acid yield for this strain is about 55%.<br>
Example 7: Microaerobic two-stage shake flask characterizations of strains CD990 (Ex. 1C), CD1030 (Ex. 2D), CD1184 (Ex. 4) and CD1270 (Ex. 5) in buffered defined medium.<br>
Transformants CD 990, CD 1030, CD 1184 and CD 1270 are separately grown<br>
on yeast peptone plus 5% glucose. The cells are used to inoculate separate flasks containing YNB + 5% glucose + 0.5 M MES, pH 5.5 media to ODeoo = 0.1. The flasks are incubated overnight at 30°C and 250 rpm shaking. The cells are then transferred to separate flasks containing 50 ml YNB + 10% glucose + 4 g CaCOs to ODeoo =12, and incubated for 5 days at 30°C and 100 rpm shaking. Samples are withdrawn periodically during cultivation, and ODeoo is measured. Cells are recovered from each sample by centrifugation and the supernatant analyzed by HPLC for lactic acid, glucose and ethanol as described in Example 6.<br>
The single PDC deletant strain CD 1030 produces both ethanol and lactic acid. It, consumes all of the glucose within 48 hours and produces about 56 g/L lactic acid. Lactic acid yield for this strain is just under 60%. This strain also produces about l;~{ g/L of ethanol. This performance is very similar to that of CD990, which has the LhLDH gene without either PDC deletion.<br>
The double PDC deletant strains (CD1184 and CD1270) again produce lactic acid but no ethanol. Strain CD1270 consumes glucose slightly faster than strain CD 1184, but at about the same rate as strain CD 1030. Lactic acid titer for strain<br>
CD 1270 peaks at about 85 g/L after 50 hours, and declines slightly thereafter as the strain begins to consume lactic acid when glucose becomes depleted. Lactic acid yield for this strain is about 85% after 50 hours. Strain CD 1184 consumes about 90% of the glucose after 50 hours, and consumes the remainder over the next 72 hours. It produces a maximum lactic acid titer of about 73 g/L and a maximum lactic acid yield of about 80%.<br>
Example 8: Quasi-anaerobic two-stage shake flask characterizations of strains CD990 (Ex. 1C), CD1030 (Ex. 2D), CD1184 (Ex. 4) and CD1270 (Ex. 5) in buffered defined medium.<br>
Transformants CD 990, CD 1030, CD 1184 and CD 1270 are separately grown<br>
on yeast peptone plus 2% glucose. The cells are used to inoculate separate flasks containing yeast peptone + 10% glucose and incubated overnight at 30°C and 250 rpm shaking. The cells are then transferred to separate flasks containing 50 ml yeast peptone + 10% glucose to ODnoo = 13. The flasks are sealed with water locks and incubated for about 6 days at 30°C and 100 rpm shaking in yeast peptone + 10% glucose. However, residual air is not removed from the head space of the flask and no measures are taken to remove dissolved oxygen. These cultivations are therefore not strictly anaerobic, as some oxygen is available at least at the beginning of the cultivation.<br>
The pH of the broths falls to 3.2 ±0.1 during the cultivations due to the production of lactic acid.<br>
Samples are withdrawn periodically during cultivation, and ODcon is measured. Cells are recovered from each sample by centrifugation and the supernatant analyzed by HPLC for lactic acid, glucose and ethanol as described in Example 6.<br>
The single PDC deletant strain CD1030 produces about 19 g/L lactic acid after 24 hours, about 24 g/L lactic acid after 72 hours and slightly more than 25 g/L lactic acid after 141 hours. Strain CD990, which has no PDC deletion, produces about, 20 g/L lactic acid after 24 hours and about 22 g/L lactic acid after 141 hours. Strains CD990 and CD 1030 both produce ethanol as well as lactic acid.<br>
The double PDC deletant strain CD 1184 produces about 15 g/L lactic acid after 24 hours and about 18 g/L lactic acid after 72 hours. The double PDC doletant strain CD 1270 produces about 15.5 g/L lactic acid after 24 hours, about 14.5 g/L lactic acid after 72 hours and about 19.5 g/L lactic acid after 141 hours.<br>
Example 9A: Microaerobic batch culture cultivation of strain GDI 184 (Ex. 4) at pH 3.<br>
Duplicate single-stage batch culture reactors containing a defined medium that includes ammonium sulphate, potassium dihydrogen phosphate and magnesium sulphate, trace elements, vitamins and 83 g/L glucose are inoculated with 1 mL strain CD1184. pH of the medium is adjusted to 3.3 prior to adding the cells. The cells are cultured at 30°C under 380 rpm agitation and 0.1 vvm aeration. These conditions lead to an oxygen uptake rate (see WO 03/102200) of 2.9-3.1 mmol/L/h. The pH of the culture is allowed to drop to 3.0 as cells grow and begin to produce lactic acid. Afterward, pH is maintained at 3.0 by addition of potassium hydroxide.<br>
HPLC analyses are conducted as described above. Under these conditions, the organism produces 67 g/L lactic acid after -120 hours fermentation. The lactate production rate is 0.62 g/l/hr and the yield of lactate on glucose is 0.76 g/g.<br>
Example 9B: Aerobic batch culture cultivation of strain CD1184 (Ex. 4) at PH3.<br>
A  single-stage   hatch   culture   reactor  containing   a  defined  medium   that<br>
includes ammonium sulphate, potassium dihydrogen phosphate and magnesium sulphate, trace elements, vitamins, defoaming agent, and 90 g/L glucose arc inoculated with 1 mL strain CD1184. . The pH of the medium is adjusted to about 3.3 prior to adding the cells. The pH of the culture is allowed to drop to 3.0 as cells grow and begin to produce lactic acid. Afterward, pH is maintained at about 3.0 by addition of potassium hydroxide. The cells are cultured at 30°C under 490 rpm agitation and 0.1 vvm aeration. These conditions lead to an oxygen uptake rate (sec WO 03/102200) of about 8 mmol/L/hr. An additional 40 g/L glucose is added to the fermentation after about 50 hours of fermentation<br>
HPLC analyses are conducted as described above. Under these conditions, the organism produces 80 g/L lactic acid after ~90 hours fermentation (including a las phase of about 18 hours during which little fermentation occurs). Over the entire batch fermentation, the lactate production rate is 1.0 g/L/hr and the yield of lactate on glucose is 0.71 g/g. Over the period of time between the end of the lag phase (18 hours) and a titer of 70 g/L lactate is reached (69 hours), the lactate production rate is 1.5 g/L/hr and the yield of lactate on glucose is 0.75 g/g after accounting for dilution effects from the addition of glucose and potassium hydroxide.<br>
Example 10A: Construction of a plasmid (pMI445, Fig. 11) containing the S. cerevisiae MEL5 gene cassette between identical K. thermotolerans repeats.<br>
The entire K. marxianus CYB2 (KmCYBS) gene cassette, including promoter<br>
and terminator regions, is PCR amplified from the genomic DNA of a native A', marxianus strain, with introduction of BamHI and Sail restriction sites, by PCR using primers identified as SEQ. ID. NO. 27 and SEQ. ID. NO. 28. The PCR product is ligated to a commercial vector designated as pUCIS (from Invitrogen Corp., Carlsbad, CA) that is digested with BamHI and Sa/7. The resulting plasmid i.s designated as pMM25 (Fig. 9).<br>
A 705 bp sequence identified as SEQ. ID. NO. 29 is PCR-amplified from the genomic DNA of K. thermotolerans, with introduction of SphI and Sail restriction sites, using primers identified as SEQ. ID. NO. 30 and SEQ. ID. NO. 31. This K. thermotolerans sequence does not encode for any active protein. Plasmid pMM25 is digested with SphI and Sail and the K. thermotolerans sequence is ligated to it upstream (5') to the KmCYB2 cassette to form a plasmid designated as pMM27.<br>
An identical K. thermotolerans sequence is PCR-amplified with addition of BamHI and Xmal restriction sites, using primers identified as SEQ. ID. NO. 32 and<br>
SEQ. ID. NO. 33.   Plasmid pMM27 is digested with BamHI and Xmal and the K.<br><br>
thermotolerans sequence is ligated to it downstream (3') from the KmCYB2 cassette to form a plasmid designated as pMM28 (Fig. 10). Plasmid pMM28 contains the KtnCYB2 cassette flanked by identical K. thermotolerans sequences, both oriented in the same direction.<br>
Plasmid pMM28 is digested with BamHI, filled in with the Klenow enzyme, and digested with So/7. A 4077 bp fragment so obtained is ligated to a 2317 bp fragment obtained by digesting pMI433 (Fig. 8, Ex. 3A) with NotI, filling the overhangs in with the Klenow enzyme, and digesting with Sail. The resulting plasmid is designated pMI445 (Fig. 11).<br>
Example 10B: Isolation of CYB2 homologues from 7. orientalis.<br>
The KmCYB2 gene is used as a probe to isolate homologous genes from n<br>
library of genomic DNA obtained from a growing 7. orientalis strain. The probe is synthesized by PCR using oligonucleotides SEQ. ID. NO. 34 and SEQ. TD. NO. 85 H* primers and K. marxianus genomic DNA as the template, and labeled with '-P. The KmCYB2 gene so obtained is used to isolate 7. orientalis CYB2 genes from a genomic<br>
library of I. orientalis. A Southern blot containing EcoRI-digested DNA from six /. orientalis cosmid clones and genomic DNA from a wild-type I. orientalis strain are prepared and hybridized with the KrnCYB2 gene. ~1.5 kbp bands are delected, isolated from gel and cloned into an jE'ccxfijT-digested pBluescript SK(-) plasmid. The bands are sequenced using M13 reverse and forward primers. Sequence-specific primers are designed based on the sequences so obtained. Two CYB2 genes are identified, which are designated loCYBSA and IoCYB2B. The coding region and approximately 1 kbp of the 5' and 3' flanking regions of each of the clones are sequenced. The sequences are identified as SEQ. ID. NO. 36 and SEQ. ID. NO. 37, respectively.<br>
Example IOC: Construction of a plasmid (pMI447, Fig. 14) containing the I. orientalis PDC1A promoter, K. thermotolerans sequence, ScMELl cassette, second identical K. thermotolerans sequence, LhLDH gene cassette and J. orientalis PDC1A terminator.<br>
A vector designated as pNC16 is obtained from the National Research KIUT^A<br>
Laboratories in Golden, Colorado. This plasmid contains the S. cerevisiae MELl pene under the control of the S. cerevisiae PDCl promoter and S. cerevisiae GAlJO terminator. The MELl gene cassette is PCR-amplified with addition of Bglll and Sad restriction sites using primers designated as SEQ. ID. NO. 38 and SEQ. ID. NO. 39. Plasmid pMM28 (Ex. 1.0A. Fig. 10) is digested with Bglll and Sad and ligated to the MELl cassette. This simultaneously deletes the KmCYB2 cassette of plasmid pMM28 and replaces it with the MELl cassette. The resulting plasmid is designated pMM31. It contains the MELl cassette flanked by the repeating K. thermotolerans sequences.<br>
A ~2kbp flanking region directly 3' of the KmCYB2 coding region is amplified with introduction of Xmal and Sad restriction sites by PCR using primers identified as SEQ. ID. NO. 40 and SEQ. ID. NO. 41 and genomic DNA as the template, The resulting fragment is ligated to Xwa//Sac/-digested plasmid pMMSl to insert the 3' CYB2 flank downstream (3') of the K. thermotolerans sequence that is itself downstream of the MELl cassette. The resulting plasmid is designated as pMM32.<br>
A ~2kbp flanking region directly 5' of the KmCYB2 coding region is amplified with introduction of AatJJ and Narl restriction sites by PCR using primers identified as SEQ. ID. NO. 42 and SEQ. ID. NO. 43 and genomic DNA as the template. The resulting fragment is Ligated to the Aa/!/7/A7ar/-digested plasmid pMJV132. The<br>
resulting plasmid (designated pMM35, Fig. 12) contains, in order, the 5' KmCYB2 flanking region, a first identical K. thermotolerans sequence, the MELl cassette, the second identical K. thermotolerans sequences and the 3' KmCYB2 flanking region,<br>
The K. thermotolerans sequence is amplified by PCR using primers identified as SEQ. ID. NO. 44 and SEQ. ID. NO. 45, with plasmid pMM35 as the template. The PCR product is digested with NotI and Spel. The resulting 712 bp fragment is ligated to an 8798 bp fragment obtained by digesting pM!433 (Ex. 3A, Fig. 8) with Spel and NotI. The resulting plasmid is designated pMI446 (Fig. 13). It contains, in order, the /. orientalis PDC promoter, ScMELS gene cassette, the K. thermotolerans sequence, the LhLDH cassette, and /. orientalis PDC1A terminator.<br>
The K. thermotolerans sequence is amplified by PCR using primers identified as SEQ. ID. NO. 46 and SEQ. ID. NO. 47, using plasmid pMM35 as the template. The PCR product is digested with Sail. The resulting 711 bp fragment is ligated to a 9510 bp Sail fragment of plasmid pMI446. The resulting plasmid is designated pMI447 (Fig. 14). It contains, in order, the /. orientalis PDC promoter, first K. thermotolerans repeating sequence, ScMELS cassette, second K. thermotolerans repeating sequence, LhLDH gene cassette and I. orientalis PDC terminator.<br>
Example 10D: Construction of a plasmid (pMI448, Fjg. 15) containing a K. thermotolerans sequence, ScMELS gene cassette, second identical K. thermotolerans sequence and IoCYB2A terminator.<br>
The 3' flanking region of the IoCYB2A gene from -90-676 bp downstream of<br>
the open reading frame is amplified by PCR using primers identified as SEQ. ID. NO. 48 and SEQ. ID. NO. 49 and a cosmid clone containing the /. orientalis CYB2A gene as the template. The PCR product is digested with Sad and Smal. A 607 bp fragment is obtained and ligated to a 6386 fragment obtained by digesting plasmid pMI445 (Example 10A, Fig. 11) with SacI and Smal. The resulting plasmid is designated pMI448 (Fig. 15).<br>
Example 10E: Isolation of URA3 gene from /. orientalis.<br>
A fragment of the 7. orientalis URA3 gene (loURAS) is amplified by PCR from<br>
genomic DNA of a native strain of 7. orientalis using primers identified as SEQ. ID. NO. 50 and SEQ. ID. NO. 51. A ~650 bp fragment is obtained, which is sequenced to confirm close homology to the URA3 genes of other yeasts. This ~650 bp fragment is then used as a probe for isolating the full length gene from a genomic cosmid library of the 7, orientalis native strain. A clone is obtained, which is purified and sequenced.<br><br>
The clone includes the loURA3 functional gene and flanking regions, and includes a sequence identified as SEQ. ID. NO. 52. The open reading frame of this gene encodes for a protein having 262 amino acids. This amino acid sequence is identified as SEQ. ID. NO. 53.<br>
Example 10F: Construction of transformation plasmid pMI457 (Fig. 16) containing the loURA3 promoter, ScMEL5 gene cassette between identical K. thermotolerans sequences, LhLDH gene cassette and loURA3 terminator and transformation plasmid pMI458 (Fig. 17) containing the loURA3 promoter, ScMEL5 gene cassette between identical K. thermotolerans sequences, and loURA3 terminator.<br>
The loURAS 3' flanking sequence of /. orientalis is amplified with primers<br>
identified as SEQ. ID. NO. 54 and SEQ. ID. NO. 55 with an /. orientalis cosmic! clone-containing the URA3 gene as the template. A 630 bp fragment is obtained, which is cut with Smal and Apal and ligated to a Smal/Apal fragment of plasmid pMl447 (Ex. 1OC, Fig. 14) to produce a plasmid designated pMI455. Plasmid pMI455 contains the 7. orientalis PDC promoter, ScMEL5 gene cassette between repeating K. thermotolerans sequences, LhLDH gene cassette and loURA3 3' flank.<br>
The loURA3 5' flanking sequence of /. orientalis is amplified with primers identified as SEQ. ID. NO. 56 and SEQ. ID. NO. 57 with the 7, orientalis cosmicl clone containing the URA3 gene as the template. A 554 bp fragment is obtained, which is cut with SphI and ligated to a 6994 bp Sphl-cut fragment of plasmid pM!44H (Ex]\ IOC, Fig. 15) to produce plasmid pMI456. Plasmid pMI456 contains the loURA.'i promoter, the ScMELS gene cassette between repeating K. thermotolerans sequences and the 7. orientalis CYB2A terminator.<br>
Plasmid pMI455 is cut with Sac7 and Sail. The resulting 8542 bp fragment, is ligated to a 1264 bp SacI-XhoI fragment of plasmid pM!456 to produce plasmid pM!457 (Fig. 16).<br>
Plasmid pMI457 is cut with NotI and Smal, filled in using the Klenow enzyme and the resulting 7834 bp fragment religated to form plasmid pMI458 (Fig. 17).<br>
Example 10G: Generation of/, orientalis mutants (CD1439 and CD1440) with two (CD1439) and one (CD1440) copies of LhLDH gene and ScMEL5 cassette inserted at locus of native UPA3 gene (and URA3 deletion) by transforming mutant strain CD 1184 (Ex. 4) with plasmids pMI457 (Ex. 10F, Fig. 16) and PMI458 (Ex. 10F, Fig. 17).<br>
Mutant stain CD 1184 is transformed with plasmid pM!457 using the general<br>
method   described   in   Example   3B   and   plated   onto   YPD   +   X-a-gal   plates.<br><br>
Transformants containing the ScMELS gene are identified based on blue color. Those transformants are screened via PCR using primers identified as SEQ. ID. NO. 58 and SEQ. ID. NO. 59 to identify URA3 integrants. Positive transformants are further identified by Southern analysis of EcoRV-Hindlll and NcoI-Bsml-digested DNA. A digoxigenin-labeled URA3 probe is synthesized using primers identified as SEQ. TD. NO. 56 and SEQ. ID. NO. 55 and the cosmid clone containing the /. orientalis URA3 gene as the template. A transformant that produces the expected bands is identified as strain CD 1439. Strain CD 1439 has the same genetic background as strain GDI 184, except it has an extra copy of the LhLDH cassette and ScMELS cassette at the locus of a native URA3 gene.<br>
Strain CD1440 is produced in the same manner, except it is transformed with plasmid pMI458. Plasmid pM!458 lacks the LhLDH cassette, but otherwise effects the same transformation as plasmid pMl457. Strains CD1439 and CD1440 therefore have the same genetic background except for the number of LhLDH cassettes.<br>
Example 11 - Microaerobic shake flask characterizations of strains CD 1184 (Ex. 4), CD1439 (Ex. 10F) and CD1440 (Ex. 10F).<br>
Strains CD 1439 and CD 1440 are separately inoculated to an initial OD600 of<br>
0.15 into 50 ml of YP+lOOg/L glucose in non-baffled flasks. The flasks are incubated at 30°C with 100 rpm shaking and assayed after 22, 47, 62, 91, 119 and 143 hours.<br>
Samples for enzyme activity measurements (5 mL) are collected periodically by centrifugation. The cell pellets are washed with 1 mL of cold 10 mM K2HPOi/KH2PO (pH 7.5) supplemented with 2 mM EDTA. The washed pellets are resuspended in 1 mL of the same buffer and stored at -70°C. Samples are thawed at room temperature and washed in homogenization buffer (100 mM KoHPO4/KH2PO4 (pH 7.5) supplemented with 2 mM MgCb, 1 mM DTT and Protease Inhibitor (EDTA free, Roche). Washed samples are resuspended in 0.5 mL of homogenization buffer and twice homogenized for 30 seconds with 0.5 mL glass beads using a Bead Beater homogenizer. Samples are then centrifuged at 14,000 rpm for 30 minutes at 4°f. LDH activity is determined by analyzing the supernatant spectrophotometricallv (A340) using a Cobas MTRA automated analyzer at 30°C in sodium acetate buffer containing 0.4 mM NADH, 5 mM fructose-1,6-diphosphate and 2 mM pyruvatc. 1 U of activity is defined as the amount of activity converting 1 umol of NADH to NAD+/minute. Protein concentrations are determined using the Bio-Rad method, with bovine gamma-globulin used as a protein standard.<br>
Strains CD1184, CD1439 and CD1440 all consume glucose at approximately the same rate, and all produce approximately 55-60 g/L of lactate. Each produces approximately 0.6 g/L pyruvate and 6 g/L glycerol. The LDH activity of strain CD 1439 is approximately 40% higher than that of CD 1440 throughout the cultivation, due to the presence of the second copy of the LhLDH cassette.<br>
Example   12A:  Cloning of /.  orientalis native  GPD1 gene  together  with upstream and downstream flanking region.<br>
Known glycerol-3-phosphate dehydrogenase genes from several yeast species (S. cerevisiae, K. tnarxianus, Y. lipolytica, P. jadinii, D, hansenii and C. glabrata) are aligned and regions which are highly conserved among the various genes are identified. Two sets of degenerate primers were designed in these regions of high homology. These sets are identified as SEQ. ID. NO. 60 and SEQ. ID. NO. 61, and SEQ. ID. NO. 62 and SEQ. ID. NO. 63, respectively. PCR is performed using the first set of primers and I. orientalis genomic DNA as the template, and a ~200 bp product is obtained as expected. PCR is again performed using the second set of primers and I. orientalis genomic DNA as the template, and a ~400 bp product is obtained as expected. The two PCR products are gel purified and sequenced using the same primers. Using the partial sequence so obtained, primers are designed for genome walking. Genome walking is performed using the BD Clontech Genome Walking Kit according to the manufacturer's instructions using primary PCR primers identified as SEQ. ID. NO. 64 and SEQ. ID. NO. 65 and nested PCR primers identified as SEQ. I [). NO. 66 and SEQ. ID. NO. 67. Sequences obtained from both upstream and downstream genome walks are aligned and merged with the previously obtained partial sequence to construct the I. orientalis glycerol-3-phosphate dehydrogenase gene.<br>
Example 12B: Construction of plasmids pMI449 (Fig. 18) and pMI454 (Fig. 19) containing /. orientalis CYB2 5' flanking region, ScMELS gene cassette between K. thermotolerans direct repeat sequences and /. orientalis CYB2 :3 flanking region.<br>
Plasmid pMM28 (Fig. 10, Ex. 10A) is digested with BαmHI, filled in with the<br>
Klenow enzyme, and digested with Sail. The 4077 bp fragment so obtained is ligated to a 2317 bp NotI (filled in with Klenow enzyme)-Sα/I fragment of pM!433 (Fig. 8, E\. 3A). The resulting plasmid is designated pMI445.<br>
The 3' flanking region of the /. orientalis L-lactate:ferricytochrome c oxidoreductase (IoCYB2A) gene (corresponding to sequences from 90 to 67ft bp downstream of the predicted open reading frame) is amplified by PCR using primers identified as SEQ. ID. NO. 68 and SEQ. ID. NO. 69, using a CYB2-2 cosmid clone as a template. The PCR product is digested with Sad and Smal and the 607 bp fragment is ligated to the 6386 bp Sad • Smal fragment of plasmid pMI445. The resulting plasmid is designated pMI448.<br>
The IoCYB2A 5' flanking region (corresponding to sequences from 913 to 187 bp upstream of the predicted open reading frame) is amplified by PCR using primers identified as SEQ. ID. NO. 70 and SEQ. ID. NO. 71, again using the CYB2-2 cosmic! clone as a template. The PCR product is digested with Sphl and the 454 bp fragment is ligated to the 6993 bp Sphl fragment obtained by partially digesting pMI448. The resulting plasmid is designated pM!449 (Fig. 18).<br>
The loCYB2A 5' flanking region (corresponding to sequences from 466 to 7 bp upstream of the predicted open reading frame) is amplified by PCR using primers identified as SEQ. ID. NO. 72 and SEQ. ID. NO. 73, once again using the CYB2-2 cosmid clone as the template. The PCR product is digested with Sphl and the 493 bp fragment is ligated to the 6993 bp Sphl fragment obtained by partially digesting plasmid pM!448. The resulting plasmid is designated pM!45'3.<br>
The IoCYB2A 3' flanking region (corresponding to sequences from -102 bp upstream to 77 bp downstream of the predicted stop codon) is amplified by PCR using primers identified as SEQ. ID. NO. 74 and SEQ. ID. NO. 75, using the CYB2-2 cosmid as a template. The PCR product is digested with Apal and Smal and the 506 bp fragment is ligated to the 6886 bp Apal - Smal fragment of plasmid pMI453. The resulting plasmid is designated pM!454 (Fig. 19).<br>
Example 12C: Construction of a plasmid (pBH165, Fig. 20) containing" an upstream fragment of the loGPDl gene, a first K. thermotolerans direct repeat section, a MEL5 gene cassette, a second K, thermotolerans clireci repeat section, and a downstream fragment of the loGPDl gene.<br>
Plasmid pMI449 (Fig, 18, Ex. 12B) is digested with Ndel and Sbfl to excise I he<br>
5' CYB2A flanking homology. A 6.8 kbp fragment is gel purified and dephosphorylated. A 302 bp fragment of the loGPDl gene from Example 12A (corresponding to base pairs 1-302 from the start codon of the gene) is amplified by PCR using primers identified as SEQ. ID. NO. 76 and SEQ. ID. NO. 77. The IVK product is gel purified, digested with Ndel and Sbfl, and ligated to the (i.S kbp<br>
fragment from plasmid pMI449 to produce plasmid pBH!64. Plasmid pBH164 is then digested with Xmal and KcoRI to excise the 3' CYB2A flanking homology. A 6.5 kbp fragment is gel purified and dephosphorylated. A 346 bp fragment of the loGPDl gene from Example 12A (corresponding to base pairs 322-668 from the start codon) is amplified by PCR using primers identified as SEQ. ID. NO. 78 and SEQ. ID. NO. 79. The PCR product is gel purified, digested with Xmal and EcoRI, and ligated to the 6.5 kbp fragment obtained from pBH164 to produce pBH!65 (Fig. 20).<br>
Plasmid pBH165 contains, in order of transcription, the 302 bp fragment of the loGPDl gene, a first K. thermotolerans direct repeat section, a MEL5 gene cassette, a second K. thermotolerans direct repeat section, and the 346 bp fragment of the loGPDl gene. It is designed for insertion at the locus of the native loGPDl gene (with disruption of the gene), followed by a loop-out of the MEL5 gene cassette.<br>
Example 12D: Generation of /. orientalis mutant strain (CD1496) by successively transforming strain CD1184 (Ex. 4) with plasmids pMI449 (Ex. 12B, Fig. 18) and pMI454 (Ex. 12B, Fig. 19), followed by mutagenesis.<br>
Strain CD 1184 is transformed with plasmid pM!449 using the lithium acetate<br>
method and transformants (blue colonies) are selected based on melibiase activity on YPD X-a-gal plates. The replacement of the IoCYB2A gene of strain CD 1184 is confirmed by colony PCR and Southern analysis in some of the transformants. The MKL5 marker is looped out from one of those transformants via a homologous recombination event through the K. thermotolerans repeat sequences, as confirmed by Southern analysis. The second CYB2A allele is then deleted from this transformant using plasmid pMI454. Transformants are analyzed by colony PCR for the absence of a 1000 bp CYB2A-specific PCR product. The MEL5 marker from plasmid pMI454 is looped out of a transformant having a deletion of the second CYB2A allele via recombination as before. This transformant is designated strain CD 1-136, Strain CD 1436 has a deletion of both PDCl alleles (with replacement by a functional L-L/)// gene cassette), and a deletion of each of its two native IoCYB2 genes.<br>
Cells of strain CD1436 from a fresh YPD plate are resuspended in 2 nil, phosphate-buffered saline to an approximate OD600of 6. Twelve 200 µl aliquots of I his cell suspension are pipeted into twelve 14mL snap-cap tubes, and 8 µL of ethyl methanesulfonate (EMS, Sigma Chemical Co., St. Louis, MO, catalog # MO880, 1.17g/mL solution) is added to ten of the twelve tubes. The remaining two tubes serve as mock-treated controls. The tubes are then incubated at 30°C with agitation<br>
(225rpm) for 60 minutes, to kill 90-99% of the cells. Following exposure to EMS, the cells from the twelve tubes are pelleted, washed twice with 5.0% Na2S2O3 to neutralize the EMS and washed once with water. Mutagenized cells are allowed to recover for 6 hours in 200 µL of YP + 20g/L glucose media and then plated onto PDA + 35 g/L lactic acid plates and incubated for one week at 30°C. A strain that produces more lactate and less glycerol than strain CD 1436 is designated as strain CD 1496.<br>
Example 12E: Transformation of strain CD 1496 (Ex. 12D) with plasmid pBH165 (Ex. 12C, Fig. 20), followed by loop-out of the selection marker to produce transformant strain CD1671 which has a single GPD1 allele deleted.<br>
Strain CD 1496 is grown and transformed with 5 ug of the 4.4 kbp fragment<br>
obtained by digesting plasmid pBH165 with Ndel and EcoRI. Transform ants are selected on yeast nitrogen base (YNB) + 2% melibiose plates overlaid with x-a-gal (5-bromo-4-chloro-3-indolyl-aD-galactoside). Blue-colored transformants are visible after ~4 days of growth at 30°C. Eight transformants are picked and plated for .single colonies on YP + 20 g/L glucose plates containing x-a-gal. A single blue colony for each transformant is picked and restreaked to YP + 20 g/L glucose plates. Genomic DNA is isolated from the transformants. Disruption of one allele of the loGPDl genc is verified by PCR using primers identified as SEQ. ID. NO. 80 and SEQ. ID. NO. 81. One transformant that exhibits the expected ~2 kb product is designated as strain CD 1657. Disruption of one copy of the native loGPDl gene is further verified by PCR using primers designated as SEQ. ID. NO. 82 and SEQ. ID. NO. 83.<br>
Strain CD 1657 is grown for several rounds in YP + lOOg/L glucose at 3()°C. A dilution series is plated onto YP + 20 g/L plates overlaid with x-a-gal, and grown overnight at 30°C. A white colony (indicative of the loop-out of the MEL 5 marker cassette) is selected and restreaked to YP + 20 g/L glucose + x-a-gal plates. A white colony is selected. Disruption of one allele of the native loGPDl gene is verified by PCR using primers identified as SEQ. ID. NO. 84 and SEQ. ID. NO. 85. This transformant is designated as strain CD 1671.<br>
Example 12F: Transformation of strains CD1671 (Ex. 12E) with plasmid pBH1.65 (Ex. 12C, Fig. 20) to produce transformant strain CD!690 with both loGPDl alleles deleted.<br>
Strain CD1671 is transformed with 5 ug of a 4.4 kbp fragment obtained by<br>
i,<br>
digesting plasmid pBH l65 with Ndel and EcoRI. Transformants are selected on YNB + 2% melibiose plates overlaid with x-a-gal. Blue-colored transformants are visible<br>
after ~4 days of growth at 30°C. Ten transformants are picked and plated fov single colonies on YP + 20 g/L glucose plates containing x-α-gal. A single blue colony foe each transformant is picked and restreaked to YP + 20 g/L glucose. Genomic DNA is isolated from the transformants. Disruption of the second allele of the loGPDl gene is verified in a transformant by PCR using primers identified as SEQ. ID. NO 86 and SEQ. ID. NO. 86. This transformant is designated as strain CD 1690.<br>
Example 12G - Microaerobic shake flask characterizations of strain GDI 690 (Ex. 12F).<br>
Strain CD 1690 is inoculated to an initial OD600 of 0.2 into YP + l00g/L<br>
glucose in 3-liter batch fermenter. The flasks are incubated for 40 hours at 38-4C)°C with 100 rpm shaking. The cultivation is buffered to pH 5.5 throughout the cultivation. Under these conditions, strain CD 1690 produces a yield of 88 grams of L-lactic acid/100 grams of glucose that is consumed. L-lactic acid productivity is 2.6 g/L/hr. Yields for by-products are C02: 8%; biomass: 2.4%, and pyruvate: 1%. The final OD600 is 6.3.<br>
Example 13A: Cloning of P. membranifaciens native PDC1 gene fragment.<br>
A pair of degenerate primers is designed to clone a portion of the PDCl gene in P. membranifaciens. These primers are identified as SEQ. ID. NO. 88 and SEQ. ID. NO. 89. PCR is performed using the primers and P. membranifaciens genomif DNA as the template, and a ~700 bp product is obtained. The fragment is cloned onto a commercial TOPO vector to produce a plasmid designated as plasmid PDC-7 clone. The fragment is sequenced using primers identified as SEQ. ID. NO. 90 and SEQ. ID. NO. 91. The nucleotide sequence of the fragment is identified as SEQ. ID. NO. 92. The fragment has high identify with other known yeast PDC gene sequences.<br>
Example 13B: Construction of plasmid pMI464 (Fig. 21) containing P. membranifaciens PDCl gene fragment, hygromycin expression cassette and LhLDH expression cassette.<br>
Plasmid pMI357 (Fig. 6, Ex. 2B) is digested with Sad and Sal} to form a<br>
-7735 bp fragment. Plasmind PDC-y clone is digested with Sad and Xhol to produce1 a -700 bp fragment. The two fragments are ligated together to form a plasmid designated as plasmid pMT464.<br>
Example 13C: Generation of strain CD1598 by transformation of a wild-type P. membranifaciens strain with plasmid pMI464 to integrate the LhLDH gene cassette.<br>
A   wild-type   P.    membranifaciens   strain   designated   as   NCYC269G   is<br>
transformed with a fragment obtained by digesting plasmid pMI464 with AgcI Transformants are selected on YDP + hygromycin plates and streaked onto YDP +200 µ,g/mL hygromycin. One colony is designated as strain CD 1598, The presence of the LhLDH gene cassette in the strain is verified by PCR.<br>
Example 13D - Microaerobic shake flask characterization of strain CD1598 (Ex. 13C).<br>
Strain CD 1598 is inoculated to an initial OD600 of 0.2 into 50 ml of non-buffered YP + 10% glucose medium in a shake flask. The flask is incubated for 7 days at 30°C with 100 rpm shaking. Strain CD1598 produces lactic acid to a titer of 17 g/L. Lactic acid yield is 70% based on glucose consumed. Lactic acid production rate is 0.41 g/L/hr. The strain does not produce ethanol.<br><br><br><br><br><br><br>
We claim:<br>
1.	A genetically modified yeast cell of a species selected from a group comprising of Issatchenkia orientalis, Pichia galeiformis, Pichia sp. YB-4149 (NRRL designation), Candida ethanolica, P. deserticola, P. membranifaciens or P. fermentans clade having minimum one exogenous lactate dehydrogenase (LDH) gene, wherein the exogenous LDH gene encodes a functional protein that is minimum 60% identical to at least one of SEQ. ID. NO. 93, SEQ. ID. NO. 94, SEQ. ID. NO. 95 or SEQ. ID. NO. 96.<br>
2.	The yeast cell as claimed in claim 1, wherein the exogenous LDH gene is integrated into the genome of the yeast cell.<br>
3.	The yeast cell as claimed in any of claims 1 to 2 has a deletion or disruption of a native pyruvate decarboxylase (PDC) gene.<br>
4.	The yeast cell as claimed in claim 3 which is unable to produce ethanol.<br>
5.	The yeast cell as claimed in any of claims 1 to 4 wherein the exogenous LDH gene is<br>
under the transcriptional control of a promoter that is native to a yeast cell.<br>
6.	The yeast cell as claimed in claim 5 wherein the promoter is native to the host cell.<br>
7.	The yeast cell as claimed in any of claims 1 to 2 wherein the LDH gene is integrated at a locus of a native PDC gene.<br>
8.	The yeast cell as claimed in any of claims 1 to 2 wherein the host cell is I. orientalis or P. membranifaciens.<br>
9.	The yeast cell as claimed in claim 3 wherein the host cell is /. orientalis or P. membranifaciens.<br>
10.	The yeast cell as claimed in claim 4 wherein the host cell is / orientalis or P. membranifaciens.<br>
11.	The yeast cell as claimed in any of claims 1 to 2 optionally have an additional genetic<br>
modification selected from a group comprising of (1) an insertion of a functional<br>
exogenous xylose isomerase gene, (2) a deletion or disruption of a native gene that<br>
produces an enzyme that catalyzes the conversion of xylose to xylitol, (3) a deletion<br>
or disruption of a functional xylitol dehydrogenase gene and/or (4) a modification that<br>
causes the cell to overexpress a functional xylulokinase.<br>
12.	A fermentation process in which a genetically modified yeast cell of claim 1 is<br>
cultured under fermentation conditions in a fermentation broth that includes a<br>
fermentable sugar to produce lactic acid or a salt thereof.<br>
13.	The fermentation process as claimed in claim 12 wherein the fermentable sugar<br>
includes glucose.<br>
14.	The process as claimed in claim 12 or 13 wherein the pH of the fermentation broth<br>
during at least a portion of the period of fermentation is in the range of from  1.5 to 4.5.<br>
15.	The process as claimed in claim 14 wherein the pH of the fermentation broth during at<br>
least a portion of the period of fermentation is in the range of 1.9 to 3.5.<br>
16.	The process as claimed in claim 15 wherein the pH of the fermentation broth is within<br>
the range of 1.9 to 3.5 throughout the period of fermentation.<br>
17.	The process as claimed in claim 15 wherein the pH of the fermentation broth at the<br>
beginning of fermentation is from    3.5 to    6, and the pH drops during the fermentation to  1.9 to 3.5.<br>
18.	The process as claimed in claim 12 or 13 which is an anaerobic or microaerobic<br>
fermentation.<br>
19.	The process as claimed in claim 12 or 13 which is an anaerobic or quasi-anaerobic<br>
fermentation.<br>
20.	The process as claimed in claim 12 or 13 wherein the yield of lactate on glucose is 0.55-0.95 g/g.<br>
21.	The process as claimed in claim 12, optionally comprising recovering lactate from the<br>
fermentation broth.<br>
22.	The process as claimed in claim 21, optionally comprising producing lactide from the<br>
recovered lactate.<br>
23.	The process as claimed in claim 22, optionally comprising polymerizing the lactide to<br>
from a poly(lactide) polymer.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1BYnN0cmFjdC0oMTUtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Abstract-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1Bc3NpZ25tZW50LSgxNS0xMS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Assignment-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1hc3NpZ25tZW50LnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1DbGFpbXMtKDE0LTA1LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-Claims-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1DbGFpbXMtKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Claims-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE0LTA1LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-Correspondence Others-(14-05-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Correspondence Others-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1jb3JyZXNwb25kZW5jZS1vdGhlcnMgMS5wZGY=" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1EcmF3aW5ncy0oMTUtMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Drawings-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1Gb3JtLTEtKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Form-1-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1mb3JtLTE4LnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1Gb3JtLTMtKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Form-3-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1HUEEtKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-GPA-(15-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1wY3QtMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtZGVsbnAtMjAwNy1wY3QtMzA2LnBkZg==" target="_blank" style="word-wrap:break-word;">10071-delnp-2007-pct-306.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1QZXRpdGlvbi0xMzctKDE1LTExLTIwMTEpLTEucGRm" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Petition-137-(15-11-2011)-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTAwNzEtREVMTlAtMjAwNy1QZXRpdGlvbi0xMzctKDE1LTExLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">10071-DELNP-2007-Petition-137-(15-11-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="253562-process-for-producing-purified-alcohols.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253564-method-for-expressing-an-immunotoxin-in-pichia-pastoris.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253563</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>10071/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Aug-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Dec-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CARGILL, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>15407 MCGINTY ROAD WEST, WAYZATA MINNESOTA 55391, UNITED STATES OF AMERICA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SUOMINEN, PIRKKO</td>
											<td>7801 KINGSVIEW LANE NORTH, MAPLE GROVE, MN 55311,UNITED STATES OF AMERICA.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NA</td>
											<td>NA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>ARISTIDOU,ARISTOS</td>
											<td>7909 KINGSVIEW LANE NORTH, MAPLE GROVE, MN 55311, UNITED STATES OF AMERICA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>ROBERT - PEREZ, KEVIN</td>
											<td>2250 CLEVELAND STREET, MINNEAPOLIS, MN 55418, UNITED STATES OF AMERICA.</td>
										</tr>
										<tr>
											<td>5</td>
											<td>PENTTILA, MERJA</td>
											<td>VATTUNIEMENKATU 2 A 72, FI- 00210 HELSINKI, FINLAND</td>
										</tr>
										<tr>
											<td>6</td>
											<td>ILMEN, MARJA</td>
											<td>SANKARITIR 7 A 19, FI-00320 HELSINKI, FINLAND.</td>
										</tr>
										<tr>
											<td>7</td>
											<td>RUOHONEN, LAURA</td>
											<td>TIIRASAARENTIE 11 A 4, FI-0020 HELSINKI, FINLAND.</td>
										</tr>
										<tr>
											<td>8</td>
											<td>KOIVURANTA, KARI</td>
											<td>MAKITORPANTIE 17 D 39, FI-00640 HELSINKI, FINLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 1/18</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2006/020782</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2006-05-30</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/686,899</td>
									<td>2005-06-02</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253563-genetically-modified-yeast-of-the-species-issatchenkia-orientalis-and-closely-related-species-and-fermentation-processes-using-same by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:38:49 GMT -->
</html>
