<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253525-cry-1f-transgenic-cotton-lines-and-event-specific-identification-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:41:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253525:CRY 1F TRANSGENIC COTTON LINES AND EVENT-SPECIFIC IDENTIFICATION THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">CRY 1F TRANSGENIC COTTON LINES AND EVENT-SPECIFIC IDENTIFICATION THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>This invention relates to plant breeding and the protection of plants from insects. More specifically, this invention includes novel transformation events of cotton plants comprising one or more polynucleotide sequences, as described herein, inserted into specific site(s) within the genome of a cotton cell. In highly preferred embodiments, said polynucleotide sequences encode &quot;stacked&quot; Cry IF and Cry 1 Ac lepidopteran insect inhibitory proteins. However, the subject invention includes plants having single cry IF or cry 1 Ac events, as described herein. Additionally, the invention is related to cotton plants derived from that transformation event and to assays for detecting the presence of the event in a sample. More specifically, the present invention provides DNA and related assays for detecting the presence of certain insect-resistance events in cotton. The assays are based on the DNA sequences of recombinant constructs inserted into the cotton genome and of the genomic sequences flanking the insertion sites. These sequences are unique. Based on these insert and border sequences, event-specific primers were generated. PCR analysis demonstrated that these cotton lines can be identified in different cotton genotypes by analysis of the PCR amplicons generated with these event-specific primer sets. Thus, these and other related procedures can be used to uniquely identify these cotton lines. Kits and conditions useful in conducting the assays are also provided. These materials and methods can also be used to assist breeding programs to further develop traits in cotton.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br><br>
CRYIF AND CRYIAC TRANSGENIC CUTTON LINES<br>
AND EVENT-SPECIFIC IDENTIFICATION THEREOF<br>
Background of the Invention<br>
[0001]	Cotton is an important fiber crop. Breeding and biotechnology have been applied to<br>
cotton to improve its agronomic traits and the quality of the product. One such agronomic trait is resistance to insects, the advantages of which are readily apparent. Genes encoding insecticidal proteins have been introduced into cotton plants. In order to alleviate any concern that a given type of insect could develop resistance to a single type of insecticidal protein, plants are often developed that produce two different types of insecticidal proteins. Thus, the odds of an insect being hypothetically capable of developing resistance to two different insecticidal proteins are extremely low.<br>
[0002]	Cry 1 Ac insecticidal proteins and genes are known in the art. See, e.g., U.S. Patent Nos.<br>
6,1 14,138; 5,710,020; 6,251,656; and 6,229,004. CryIF insecticidal proteins and genes are also known in the art. See, e.g, U.S. Patent Nos. 5,188,960; 5,691308: 6,096/708; and 6,573,240.<br>
[0003]	The expression of foreign genes in plants is influenced by where the foreign gene is<br>
inserted in the chromosome. This could be due to chromatin structure (e.g., heterochromatin) or the proximity of transcriptional regulation elements (e.g.. enhancers) close to the integration site (Weising et al, Ann. Rev. Genet 22:421-477, 1988). For example, the same gene in the same type of transgenic plant (or other organism) can exhibit a wide variation in expression level amongst different events. Tliere may also be differences in spatial or temporal patterns of expression. For example, differences in the relative expression of a transgene in various plant tissues may not correspond to the patterns expected from transcriptional regulatory elements present in the introduced gene construct.<br>
[0004]	Thus, it is necessary to create and screen a large number of events in order to identify an<br>
event that optimally expresses an introduced gene of interest. For commercial purposes, it is common to produce hundreds to thousands of different events and to screen those events for a single event that has desired transgene expression levels and patterns. An event that has desired levels and/or patterns of transgene expression is useful for introgressing the transgene into other genetic backgrounds by sexual outcrossing using conventional breeding methods. Progeny of such crosses maintain the transgene expression characteristics of the original transformant This<br><br>
strategy is used to ensure reliable gene expression in a number of varieties that are well adapted to local growing conditions.<br>
[0005]	It would be advantageous to be able to detect the presence of a particular event in order to<br>
determine whether progeny of a sexual cross contain a transgene of interest. In addition, a method for detecting a particular event would be helpful for complying with regulations requiring the pre-market approval and labeling of foods derived from recombinant crop plants, for example. It is possible to detect the presence of a transgene by any well-known nucleic acid detection method such as polymerase chain reaction (PCR) or DNA hybridization using nucleic acid probes. These detection methods generally focus on frequently used genetic elements, such as promoters, terminators, marker genes, and the like. As a result, such methods may not be useful for discriminating between different events, particularly those produced using the same DNA construct, unless the sequence of chromosomal DNA adjacent to the inserted DNA (flanking  DNA") is known. Ah event-specific PCR assay is discussed, for example, by Windels<br>
ei al (Med. Fac. Landbouww, Univ. Gent 64/5b:459462.1999). This related to the identification of glyphosate tolerant soybean event 40-3-2 by PCR using a primer set spanning the junction between the insert and flanking DNA. More specifically, one primer included sequence from the insert and a second primer included sequence from flanking DNA.<br>
[0006]	U.S. Patent Apps. 20020120964 Al and 20040009504 Al relate to cotton event PV-<br>
GHGT07(1445) and compositions and methods for the detection thereof. WO 02/100163 relates to cotton event MONI5985 and compositions and methods for the detection thereof. WO 2004/011601 relates to com event MON863 plants and compositions and methods for the detection thereof. WO 2004/072235 relates to cotton event MON 88913 and compositions and methods for the detection thereof.<br>
fooo7]	However, no such procedures and materials were specificalty known, heretofore, that<br>
could be used to specifically identify CryIF and/or Cry1Ac stacked cotton as discussed below.<br>
Brief Summary of the Invention<br>
[0008]	This invention relates to plant breeding and the protection of plants from insects. More<br>
specifically, this invention includes novel transformation events of cotton plants comprising one or more polynucleotide sequences, as described herein, inserted into specific site(s) within the geoome of a cotton cell. In highly preferred embodiments, said polynucleotide sequences encode<br><br>
"stacked" CrylF and CrylAc lepidopteran insect inhibitory proteins.  However, the subject<br>
invention includes plants having single CrylF or CrylAc events, as described herein.<br>
[0009]	Additionally, fee subject invention provides assays for detecting the presence of one or<br>
more of the subject events in a sample. The present invention provides DNA and related assays for detecting the presence of certain insect-resistance events in cotton. The assays are based on the DNA sequences of recombinant constructs inserted into the cotton genome and of the genomic sequences flanking the insertion sites. Kits and conditions useful in conducting the assays are also provided.<br>
 Thus, the subject invention relates in part to the cloning and analysis of the DNA sequences of a whole cryIF insert, whole Cry1Ac inserts, and the border regions thereof (in transgenic cotton lines). These sequences are unique. Based on these insert and border sequences, event-specific primers were generated. PCR analysis demonstrated that these events can be identified by analysis of the PCR amplicons generated with these event-specific primer sets. Thus, these and other related procedures can be used to uniquely identify cotton lines comprising one or more events of the subject invention.<br>
Brief Description of the Figures<br>
 Figure 1 illustrates the inserted crylF transgene and flanking sequences for cotton event<br>
281-24-236. This Figure also shows amplicons and primers as described herein.<br>
[0012]	Figure 2 illustrates an inserted cryl Ac transgene and flanking sequences for cotton event<br>
3006-210-23. This Figure also shows amplicons and primers as described herein.<br>
[00 13]<br>
Brief Description of the Sequences<br>
[oo 14]	SEQ ID NO:1 is the DNA sequence for the cry IF event 281 -24-236 insert and its border<br>
sequences.<br>
[0015]	SEQ ID NO:2 is the DNA sequence for the crylAc event 3006-210-23 insert and its<br>
border sequences.<br>
[0016]	SEQ ID NO:3 is the sequence of forward primer "281-14" used with reverse primer<br>
"281-15" to amplify a 603 bp amplicon that spans the 5' junction between the flanking and insert regions of crylF event 281-24-236.<br><br>
[0017]	SEQ ID NO:4 is the sequence of the reverse primer "281-15" used with forward primer<br>
"281-14" to amplify a 603 bp amplicon that spans the 5' junction between the flanking and insert<br>
regions of crylF event 281-24-236.<br>
[0018]	SEQ ID NO:5 is the 603 bp sequence of the amplicon produced using the primers of<br>
SEQ ID NOS:3 and 4.<br>
[0019]	SEQ ID NO:6 is the sequence of forward primer "281 -9" used with reverse primer "281 -<br>
10" to amplify a 562 bp amplicon that spans the 3' junction between the insert and flanking<br>
regions of cry1F event 281-24-236.<br>
[0020]	SEQ ID NO:7 is the sequence of the reverse primer "281-10" used with forward primer<br>
"281-9" to amplify a 562 bp amplicon that spans the 3' junction between the flanking and insert<br>
regions of crylF event 281-24-236.<br>
[0021]	SEQ ID NO:8 is the 562 bp sequence of the amplicon produced using the primers of<br>
SEQE)NOS:6and7.<br>
[0022]	SEQ ID NO:9 is the sequence of forward primer "3006-20" used with reverse primer<br>
"3006-22" to amplify a 614 bp amplicon that spans the 5' junction between the flanking and<br>
insert regions of ciyl Ac event 3006-210-23.<br>
[0023]	SEQ ID NO:10 is the sequence of the reverse primer "3006-22" used with forward<br>
primer "3006-20" to amplify a 614 bp amplicon that spans the 5' junction between the flanking<br>
and insert regions of ay I Ac event 3006-210-23.<br>
[0024]	SEQ ID NO: 11 is the 614 bp sequence of the amplicon produced using the primers of<br>
SEQIDNOS:9andlO.<br>
[0025]	SEQ ID NO:12 is the sequence of forward primer "3006-9" used with reverse primer<br>
"3006-12" to amplify a 662 bp amplicon that spans the 3' junction between the insert and<br>
flanking regions of cryAc event 3006-210-23.<br>
[0026]	SEQ ID NO:13 is the sequence of the reverse primer "3006-12" used with forward<br>
primer "3006-9" to amplify a 662 bp amplicon that spans the 3' junction between the flanking<br>
and insert regions of aylAc event 3006-210-23.<br>
[0027]	SEQ ID NO:14 is the 662 bp sequence of the amplicon produced using the primers of<br>
SEQK)NOS:12and l3.<br>
[0028]	SEQ ID NO:15 is a segment of genomic cotton DNA for event 281-24-236 (53 missing<br>
bases).<br><br>
[0029]	SEQ ID NO:16 is a segment of genomic cotton DNA for event 3006-210-23 (16 missing<br>
bases).<br>
Detailed Description of the Invention<br>
[0030]	This invention relates to plant breeding and the protection of plants from insects. More<br>
specifically, this invention includes novel transformation events of cotton plants (e.g. Gossypium hirsutum and Gossypium barbadense) comprising one or more polynucleotide sequences, as described herein, inserted into specific site(s) within the genome of a cotton cell. In highly preferred embodiments, said polynucleotide sequences encode "stacked" CrylF and CrylAc lepidopteran insect inhibitory proteins. However, the subject invention includes plants having single CrylF or CrylAc events, as described herein.<br>
[0031]	Additionally, the subject invention provides assays for detecting the presence of one or<br>
more of the subject events in a sample. Aspects of the subject invention include methods of designing and/or producing any of the diagnostic nucleic acid molecules exemplified or suggested herein, particularly those based wholly or partially on the subject flanking sequences.<br>
[0032]	More specifically, the subject invention relates in part to two transgenic cotton events<br>
(crylF 281-24-236 and aylAc 3006-210-23), plant lines comprising these events, and the cloning and analysis of the DNA sequences of this cylF insert, these ciyl Ac inserts, and/or the border regions thereof. Plant lines of the subject invention can be detected using sequences disclosed and suggested herein.<br>
[00331	In preferred embodiments, this invention relates to insect-resistant cotton lines, and the<br>
identification thereof, that produces two "stacked" insecticidal proteins known as CryIF and Cryy 1 Ac. In preferred embodiments, a plant line of the subject invention comprises CryIF event 281-24-236 and crylAc event 3006-210-23. However, plants of the subject invention can comprise any one or, preferably, both of the events discussed herein.<br>
[0034]	As alluded to above in the Background section, the introduction and integration of a<br>
transgene into a plant genome involves some random events (hence the name "event" for a given insertion that is expressed). That is, with many transformation techniques such as Agrobacterium transformation, the "gene gun," and WHISKERS, it is unpredictable where in the genome a transgene will become inserted Thus, identifying the flanking plant genomic DNA on both sides of the insert can be important for identifying a plant that has a given insertion event.  For<br><br>
example, PCR primers can be designed that generate a PCR amplicon across the junction region of the insert and the host genome. This PCR amplicon can be used to identify a unique or distinct type of insertion event<br>
[0035]	As "events" are random events and generally cannot be duplicated, as part of this<br>
disclosure at least 2500 seeds of a cotton line, comprising the cry1F event 281-24-236 and aylAo event 3006-210-23, have been deposited, and made available to the public without restriction (but subject to patent rights), with the American Type Culture Collection (ATCC), Rockville, Md. 20852. The deposit has been designated as ATCC Deposit No. PTA-6233. The deposit will be maintained without restriction at the ATCC depository, which is a public depository, for a period of 30 years, or five years after the most recent request, or for the effective life of the patent, whichever is longer, and will be replaced if it becomes nonviable during that period.<br>
[0036]	The deposited seeds are part of the subject invention. Clearly, cotton plants can be grown<br>
from these seeds, and such plants are part of the subject invention. The subject invention also relaxes to DNA sequences contained in these cotton plants that are useful for detecting these plants and progeny thereof. Detection methods and kits of the subject invention can be directed to identifying any one, twos or even all three of these events, depending on the ultimate purpose of the test.<br>
[0037]	Definitions and examples are provided herein to help describe the present invention and<br>
to guide those of ordinary skill in the art to practice the invention. Unless otherwise noted, terms are to be understood according to conventional usage by those of ordinary skill in the relevant art. The nomenclature for DNA bases as set forth at 37 CFR § 1.822 is used.<br>
[0038]	A transgenic "event" is produced by transformation of plant cells with heterologous DNA,<br>
i.e., a. nucleic acid construct that includes a transgene of interest regeneration of a population of plants resulting from the insertion of the transgene into the genome of the plant, and selection of a particular plant characterized by insertion into aparticular genome location. The term "event" refers to the original transformant and progeny of the transformant that include the heterologous DNA. The term "event" also refers to progeny produced by a sexual outcross between the transformant and another variety that includes the genomic/transgene DNA. Even after repeated back-crossing to a recurrent parent, the inserted transgene DNA and flanking genomic DNA (genomic/transgene DNA) from the transformed parent is present in the progeny of the cross at<br><br>
the same chromosomal location. The term "event" also refers to DNA from the original transform ant and progeny thereof comprising the inserted DNA and flanking genomic sequence immediately adjacent to the inserted DNA that would be expected to be transferred to a progeny that receives inserted DNA including the transgene of interest as the result of a sexual cross of one parental line that includes the inserted DNA (e.g., the original transformant and progeny resulting from selfing) and a parental line that does not contain the inserted DNA.<br>
[0039]	A "junction sequence" spans the point at which DNA inserted into the genome is linked<br>
to DNA from the cotton native genome flanking the insertion point, the identification or detection of one or the other junction sequences in a plant's genetic material being sufficient to be diagnostic for the event. Included are the DNA sequences that span the insertions in herein-described cotton events and similar lengths of flanking DNA. Specific examples of such diagnostic sequences are provided herein; however, other sequences that overlap the junctions of the insertions, or the junctions of the insertions and the genomic sequence, are also diagnostic and could be used according to the subject invention.<br>
[0040]	The subject invention relates to the identification of such flanking, junction, and insert<br>
sequences. Related PCR primers and amplicons are included in the invention. According to the subject invention, PCR analysis methods using amplicons that span across inserted DNA and its borders (of crylF 281-24-236 and/or crylAo 3006-210-23) can be used to detect or identify commercialized transgenic cotton varieties or lines derived from the subject proprietary transgenic cotton lines.<br>
[0041]	The entire sequences of each of these inserts, together with the respective flanking<br>
sequences, are provided herein as SEQ ID NO: 1 (cry IF 281 -24-236) and SEQ ID NO:2 (ay I Ac 3006-210-23). Table I provides the coordinates of the insert and flanking sequences for these events.<br><br><br>
[0042]	These insertion events, and further components thereof, are further illustrated in Figures<br>
1 and 2. These sequences (particularly the flanking sequences) are unique. Based on these insert and border sequences, event-specific primers were generated PCR analysis demonstrated that these cotton lines can be identified in different cotton genotypes by analysis of the PCR amplicons generated with these event-specific primer sets, Thus? these and other related procedures can be used to uniquely identify these cotton lines. The sequences identified herein are unique. For example, BLAST searches aginst GENBANK databases did not reveal any significant homology between the cloned border sequences and sequences in the database.<br>
[0043]	Detection techniques of the subject invention are especially useful in conjunction with<br>
plant breeding, to determine which progeny plants comprise a given event, after a parent plant comprising an event of interest is crossed with another plant line in an effort to impart one or more additional traits of interest in the progeny. These PCR analysis methods benefit cotton breeding programs as well as quality control, especially for commercialized transgenic cottonseeds. PCR detection kits for these transgenic cotton lines can also now be made and used. This can also benefit product registration and product stewardship.<br>
[0044]	Furthermore, flanking cotton sequences can be used to specifically identify the genomic<br>
location of each insert. This information can be used to make molecular marker systems specific to each event. These can be used for accelerated breeding strategies and to establish linkage data.<br>
[0045]	Still further, the flanking sequence information can be used to study and characterize<br>
transgene integration processes, genomic integration site characteristics, event sorting, stability of transgenes and their flanking sequences, and gene expression (especially related to gene silencing, transgene methylation patterns, position effects, and potential expression-related elements such as MARS [matrix attachment regions], and the like).<br>
[0046]	In light of all the subject disclosure, it should be clear that the subject invention includes<br>
seeds available under ATCC Deposit No. PTA-6233. The subject invention also includes an insect-resistant cotton plant grown from a seed deposited with the ATCC under accession number PTA-6233. The subject invention further includes parts of said plant, such as leaves, tissue samples, seeds produced by said plant, pollen,  and the like.<br>
[0047]	Still further, the subject invention includes descendant and/or progeny plants of plants<br>
grown from the deposited seed, preferably an insect-resistant cotton plant wherein said plant has a genome comprising a detectable wild-type genomic DNA/insert DNA junction sequence as<br><br>
-described herein. As used herein, the term "cotton" means Gossypium hirsututn and includes all plant varieties fhat can be bred with cotton, including Gossypium bar-badense.<br>
[0048]	This invention further includes processes of making crosses using a plant of the subject<br>
invention as at least one parent For example, the subject invention includes an F, hybrid plant having as one or both parents any of the plants exemplified herein. Also within the subject invention is seed produced by such F1 hybrids of the subj ect invention. This invention includes a method for producing an F 1 hybrid seed by crossing an exemplified plant with a different (e.g. inbred parent) plant and harvesting the resultant hybrid seed. The subject invention includes an exemplified plant that is either a female parent or a male parent. Characteristics of the resulting plants may be improved by careful consideration of the parent plants.<br>
[0049]	An insect-resistant cotton plant can be bred by first sexually crossing a first parental<br>
cotton plant consisting of a cotton plant grown from seed of any one of the lines referred to herein, and a second parental cotton plant, thereby producing a plurality of first progeny plants; and then selecting a first progeny plant that is resistant to insects (or that possesses at least one of the events of the subject invention); and selfing the first progeny plant, thereby producing a plurality of second progeny plants; and then selecting from the second progeny plants a plant that is resistant to insects (or that possesses at least one of the events of the subject invention). These steps can further include the back-crossing of the first progeny plant or the second progeny plant to the second parental cotton plant or a third parental cotton plant. A cotton crop comprising cotton seeds of the subject invention, or progeny thereof, can then be planted.<br>
[ooso]	It is also to be understood that two different transgenic plants can also be mated to<br>
produce offspring that contain two independently segregating added, exogenous genes, Selfing of appropriate progeny can produce plants that are homozygous for both added, exogenous genes. Back-crossing to a parental plant and out-crossing with a non-transgenic plant are also contemplated, as is vegetative propagation. Other breeding methods commonly used for different traits and crops are known in the art. Backcross breeding has been used to transfer genes for a simply inherited, highly heritable trait into a desirable homozygous cultivar or inbred line, which is the recurrent parent. The source of the trait to be transferred is called the donor parent. The resulting plant is expected to have the attributes of the recurrent parent (e.g., cultivar) and the desirable trait transferred from the donor parent. After the initial cross, individuals possessing the phenotype of the donor parent are selected and repeatedly crossed (backcrossed) to the<br><br>
recurrent parent The resulting parent is expected to have the attributes of the recurrent parent (e.g., cultivar) and the desirable trait transferred from the donor parent<br>
[0051]	The DNA molecules of the present invention can be used as molecular markers in a<br>
marker assisted breeding (MAB) method. DNA molecules of the present invention can be used in methods (such as; AFLP markers, RFLP markers, RAPD markers, SNPs, and SSRs) that identify genetically linked agronomicaily useful traits, as is known in the art. The insect-resistance trait can be tracked in the progeny of a cross with a cotton plant of the subject invention (or progeny thereof and any other cotton cultivar or variety) using the MAB methods. The DNA molecules are markers for this trait, and MAB methods that are well known in the art can be used to track the insect-resistance trait(s) in cotton plants where at least one cotton line of the subject invention, or progeny thereof, was a parent or ancestor. The methods of the present invention can be used to identify any cotton variety having the insect-resistance event from cotton line 281 -24-236 (CTFIF) and/or 3006-210-23 (crylAc).<br>
[0052]	Methods of the subject invention include a method of producing an insect-resistant cotton<br>
plant wherein said method comprises breeding with a plant of the subject invention. More specifically, said methods can comprise crossing two plants of the subject invention, or one plant of the subject invention and any other plant. Preferred methods further comprise selecting progeny of said cross by analyzing said progeny for an event detectable according to the subject invention.<br>
[0053]	A preferred plant, or a seed, of the subject invention comprises in its genome at least one<br>
of the insert sequences, as identified in Table I, together with at least 20-500 or more contiguous flanking nucleotides on both sides of the insert, as identified in Table I. Unless indicated otherwise, "crylF cotton event 281-24-236" refers to DNA of SEQ ID NO:1 that includes the heterologous DNA inserted in the original transformant (nucleotides 2075-12,748 of SEQ ID NO:1) and all or part of both of the flanking genomic sequences of SEQ ID NO:1 (nucleotide residues 1-2074 and 12,749-15,490) immediately adjacent to the inserted DNA that would be expected to be transferred to progeny that receives the inserted DNA as a result of a sexual cross of a parental line that includes the event Similarly, unless indicated otherwise, "cryl Ac cotton event 3006-210-23" refers to DNA of SEQ ID NO:2 that includes the heterologous DNA inserted in the original transformant (nucleotides 528-8900 of SEQ ID NO:2) and all or part ofboth of the flanking genomic sequences of SEQ ID NO:2 (residues 1-527 and 8901-9382) immediately<br><br>
adjacent to the inserted DNA that would be expected to be transferred to progeny that receives the inserted DNA as a result of a sexual cross of a parental line that includes the event<br>
[0054]	The subject invention includes tissue cultures of regenerable cells of a plant of the subject<br>
invention. Also included is a plant regenerated from such tissue culture, particularly where said plant is capable of expressing all the morphological and physiological properties of an exemplified variety. Preferred plants of the subject invention have all the physiological and morphological characteristics of a plant grown from the deposited seed. This invention further comprises progeny of such seed and seed possessing the quality traits of interest.<br>
[0055]	Manipulations (such as mutation, further transfection, and further breeding) of plants or<br>
seeds, or parts thereof, may lead to the creation of what may be termed "essentially derived" varieties. The International Union for the Protection of New Varieties of Plants (UPOV) has provided the following guideline for determining if a variety has been essentially derived from a protected variety:<br>
loose]	[A] variety shall be deemed to be essentially derived from another variety ("the initial<br>
variety") when<br>
[0057]	(i)       it is predominantly derived from the initial variety, or from a variety that is itself<br>
predominantly derived from the initial variety, while retaining the expression of the essential characteristics that result from the genotype or combination of genotypes of the initial variety;<br>
[0058]	(ii)      it is clearly distinguishable from the initial variety; and<br>
[0059]	(iii)     except for the differences which result from the act of derivation it conforms to<br>
the initial variety in the expression of the essential characteristics that result from the genotype or combination of genotypes of the initial variety.<br>
[0060]	UPOV, Sixth Meeting with International Organizations, Geneva. Oct. 3 0,1992; document<br>
prepared by the Office of the Union.<br>
[0061]	As used herein, a "line" is a group of plants that display little or no genetic variation<br>
between individuals for at least one trait. Such lines may be created by several generations of self-pollination and selection, or vegetative propagation from a single parent using tissue or cell culture techniques.<br>
[0062]	As used herein, the terms "cultivar" and "variety" are synonymous and refer to a line<br>
which is used for commercial production.<br><br>
[0063]	"Stability" or "stable" means that with respect to the given component, the component is<br>
maintained from generation to generation and, preferably, at least three generations at substantially the same leveL eg., preferably ± 5%, more preferably±10%, most preferably ±5%. The stability may be affected by temperature, location, stress and the time of planting. Comparison of subsequent generations under field conditions should produce the component in a similar manner.<br>
[0064]	"Commercial Utility" is defined as having good plant vigor and high fertility, such that<br>
the crop can be produced by farmers using conventional fanning equipment, and the oil with the described components can be extracted from the seed using conventional crushing and extraction equipment. To be commercially useful, the yield, as measured by seed weight, oil content, and total oil produced per acre, is within 15% of the average yield of an otherwise comparable commercial canola variety without the premium value traits grown in the same region.<br>
[0065]	"Agronomically elite" means that a line has desirable agronomic characteristics such as<br>
yield, maturity, disease resistance, and the like, in addition to the insect resistance due to the subject event(s).<br>
[0066]	As one skilled in the art will recognize in light of this disclosure, preferred embodiments<br>
of detection kits, for example, can include probes and/or primers directed to and/or comprising "junction sequences" or "transition sequences" (where the cotton genomic flanking sequence meets the insert sequence). For example, this includes a polynucleotide probe, primer, or amplicon comprising a sequence including residues 2074-2075 or 12,748-12,749 of SEQ ID NO:1, or residues 527-528 or 8,900-8,901 of SEQ ID NO:2? as indicated in Table I. To be diagnostic for these particular events, preferred "junction primers" should include at least -15 residues of the adjacent flanking sequence and at least -15 residues of the adjacent insert sequence. With this arrangement, another primer in either the flanking or insert region can be used to generate a detectable amplicon that indicates the presence of an event of the subject invention. In preferred embodiments, however, one primer binds in the flanking region and one binds in the insert, and these primers can be used to generate an amplicon that spans (and includes) a junction sequence as indicated above.<br>
[0067]	One skilled in the art will also recognize that primers and probes can be designed to<br>
hybridize, under a range of standard hybridization and/or PCR conditions, to a segment of SEQ ID NO: 1, SEQ ID NO:2, and complements thereof wherein the primer or probe is not perfectly<br><br>
complementary to the exemplified sequence. That is, some degree of mismatch can be tolerated. For an approximately 20 nucleotide primer, for example, typically one or two or so nucleotides do not need to bind with the opposite strand if the mismatched base is internal or on the end of the primer that is opposite the amplicon. Various appropriate hybridization conditions are provided below. Synthetic nucleotide analogs, such as inosine, can also be used in probes. Peptide nucleic acid (PNA) probes, as well as DNA and RNA probes, can also be used. What is important is that such probes and primers are diagnostic for (able to uniquely identify and distinguish) the presence of an event of the subject invention.<br>
[0068]	It should be further noted that errors in PCR amplification can occur which might result in<br>
minor sequencing errors, for example. That is, unless otherwise indicated, the sequences listed herein were determined by generating long amplicons from cotton genomic DNAs, and then cloning and sequencing the amplicons. It is not unusual to find slight differences and minor discrepancies in sequences generated and determined in this manner, given the many rounds of amplification that are necessary to generate enough amplicon for sequencing from genomic DNAs. For example, the following differences between the determined sequences of the event-flanking DNAs and the corresponding, known / wild-type / genomic DNAs are noted. In the 55 flank for the subject cry IF event residue 2037 of SEQ ID NO: 1 was determined to be / is listed as "G" whereas the corresponding residue of the 281-24-236 locus of the known genomic sequence is "A" (R can be used in a consensus sequence, according to standard IUPAC-IUB conventions). In the 3' flank of this event, residue 12,781 of SEQ ID NO: 1 is listed herein as T whereas C is provided in the published genomic sequence at the corresponding location (Y is the consensus code). Position 12,811 of SEQ ID NO: 1 is C whereas T is provided for the genome (Y would be the consensus). Position 12,866 is listed as C in SEQ ID NO:1 whereas T appears in the genome (Y is the consensus). Position 12.882 is listed as G in SEQ ID NO:1 whereas A appears for the genome (R is the consensus). Position 12,918 is listed as A in SEQ ID NO:1 wheres G appears in the genome (R is the consensus). Residue 13,129 is listed as G in SEQ ID NO:1 whereas A appears in the genome (R is the consensus). Residue 13,222 is listed as C in SEQ ID NO:1 whereas T appears in the genomic sequence (Y is the consensus). At position 13-441 in SEQ ID NO: 1, a T appears whereas there is no corresponding residue in the genomic listing. Thus, this apparent insertion would shift the downstream numbering of SEQ ID NO: 1 accordingly, as compared to the genomic sequence. One skilled in the art should recognize and<br><br>
be put on notice than any adjustments needed due to these types of common sequencing errors or discrepancies are wifhin the scope of the subject invention.<br>
[0069]	Similar differences also appear in the 5' flank for the subject cry 1 Ac event At positions<br>
149,153,159,165, and 244 of SEQ ID NO:2, the following residues are listed respectively C; G, C, C. and C. in the genoomic sequence at the 3006-210-23 locus, the following residues appear, respectively, at corresponding locations: A, A, A, A, and A. Consensus codes for these substitutions are, respectively, M, R, M, M, and M. Adjustments to probes and primers can be made accordingly, and corresponding differences might be noted in amplicons that span or include any of the above residues.<br>
[0070]	It should also be noted that it is not uncommon for some genomic sequence to be deleted<br>
when a sequence is inserted during the creation of an event This was the case for both events of the subject invention. That is, SEQ ID NO:1 provides a 53-base segment of geaiomic cotton DNA for event 281 -24-236 that was deleted during the insertion. This "interior segment3' occurs between residues 2074 and 12,749 of SEQ ID NO:1 in the non-transformed cotton genome. Similarly, SEQ ED NO;2 provides a 16-base segment of genomic cotton DNA for event 3006-210-23 that was deleted during the insertion. This "interior segment" occurs between residues 527 and 8,901 of SEQ ED NO:2 in the non-transformed cotton genome.<br>
[007 J]	As illustrated in Figures 1 and 2, the components of each of the "inserts" are as follows.<br>
The transgene genetic element DNA molecules contained in the subject event CrylF 281 -24-236 consists of the maize ubiquitin 1 promoter, operably connected to the phosphinothricin N-acetyltransferase (PAT) from Streptomyces viridochromogenes, operably connected to the ORF25 polyadenylation sequences (Baker et alt Plant Molecular Biology 2:335-350,1983); the chimeric promoter [(4OCS)5MAS] containing a partially deleted mannopines synthase promoter with 4 enhancer elements from the octopine synthase promoter, operably connected to the CrylF(synpro) from Bacillus thuringiensis var. aizawai, operably connected to ORF25 polyadenylation sequences (Baker et al, Plant Molecular Biology 2:335-350, 1983); and the maize ubiquitin 1 promote unoperably connected to a partial pat sequence. The DNA polynucleotide sequences or fragments of these components can be used as DNA primers or probes in the methods of the present invention.<br>
[0072]	The transgene genetic element DNA molecules contained in the subject event CrylAc<br>
3006-210-23 consists of the (4OCS)5MAS promoter operably connected to the PAT (as<br><br>
described above), operably connected to the ORF25; and the maize ubiquitin 1 promoter opearably connected to the Cry1Ac (synpro) from Bacillus thuringiensis var. kurstaki, operably connected to the the ORP25 polyadenylafion sequences. The DNA polynucleotide sequences of these components, or fragments thereof can be used as DNA primers or probes in the methods of the present invention.<br>
[0073]	In some embodiments of the invention, compositions and methods are provided for<br>
detecting the presence of the transgene/genornic insertion region, in plants and seeds and the like, from a cotton plant designated WIDESTRIKE comprising CrylF event 281 -24-236 and QylAo event 3006-210-23. DNA sequences are provided that comprise at least one transgene/genomic insertion region junction sequence provided herein in SEQ ID NO;15 SEQ ID NO:25 segments thereof, and complements of the exemplified sequences and any segments thereof. The insertion region junction sequence spans the junction between heterologous DNA inserted into the genome and the DNA from the cotton cell flanking the insertion site. Such sequence are diagnostic for one or more of the given events.<br>
[0074]	Based on these insert and border sequences, event-specific primers were generated. PCR<br>
analysis demonstrated that these cotton lines (CrylF 281 -24-236 and cry1Ac 3006-210-23) can be identified in different cotton genotypes by analysis of the PCR amplicons generated with these event-specific primer sets. These and other related procedures can be used to uniquely identify these cotton lines. Thus, PCR amplicons derived from such primer pairs are unique and can be used to identify these cotton lines.<br>
[0075]	In some embodiments, DNA sequences that comprise at least one of the novel<br>
transgene/genomic insertion regions are an aspect of this invention. Included are DNA sequences that comprise a sufficient length of polynucleotides of transgene insert sequence and a sufficient length of polynucleotides of cotton genomic sequence from one or more of the three aforementioned cotton plants and/or sequences that are useful as primer sequences for the production of an amplicon product diagnostic for one or more of these cotton plants.<br>
[0076]	Related embodiments pertain to DNA sequences that comprise at least 2, 3,4,5,6, 7.85<br>
9,10,11,12,13,14,15,16,17,18,19,20,2L 2123,24,25, or more contiguous nucleotides of a transgene portion of a DNA sequence selected from the group consisting of SEQ ID NO: 1 and SEQ ID NO:2. or complements thereof, and a similar length of flanking cotton DNA sequence from these sequences, or complements thereof. Such sequences are useful as DNA primers in<br><br>
DNA amplification methods. The amplicons produced using these primers are diagnostic for any of the cotton events referred to herein. Therefore, the invention also includes the amplicons produced by such DNA primers-s and homologous primers.<br><br>
[0078]	This invention also includes methods of detecting the presence of DNA, in a sample, that<br>
corresponds to at least one of the cotton events referred to herein. Such methods can comprise: (a) contacting the sample comprising DNA with a primer set that, when used in a nucleic acid amplification reaction with DNA from at least one of these cotton events, produces an amplicon that is diagnostic for said event(s); (b) performing a nucleic acid amplification reaction, thereby producing the amplicon; and (c) detecting the amplicon.<br>
[0079]	Further detection methods of the subject invention include a method of detecting the<br>
presence of a DNA, in a sample, corresponding to at least one of said events, wherein said method comprises: (a) contacting the sample comprising DNA with a probe that hybridizes under stringent hybridization conditions with DNA from at least one of said cotton events and which does not hybridize under the stringent hybridization conditions with a control cotton plant (non-event-of-interest DNA); (b) subjecting the sample and probe to stringent hybridization conditions; and (c) detecting hybridization of the probe to the DNA.<br>
[ooso]	In still further embodiments, the subject invention includes methods of producing a cotton<br>
plant comprising a cry IF and/or a cry1 Ac event of the subject invention, wherein said method comprises the steps of: (a) sexually crossing a first parental cotton line (comprising an expression<br><br>
cassettes of the present invention, which confers said insect resistance trait to plants of said line) and a second parental cotton line (that lacks this insect tolerance trait) thereby producing a plurality of progeny plants; and (b) selecting a progeny plant by the use of molecular markers. Such methods may optionally comprise the further step of back-crossing the progeny plant to the second parental cotton line to producing a true-breeding cotton plant that comprises said insect tolerance trait.<br>
[0081]	According to another aspect of the-invention, methods of determining the zygosity of<br>
progeny of a cross with any one (or more) of said three events are provided. Said methods can comprise contacting a sample, comprising cotton DNA, with a primer set of the subject invention. Said primers, when used in a nucleic-acid amplification reaction with genomic DNA from at least one of said cotton events, produces a first amplicon that is diagnostic for at least one of said cotton events. Such methods further comprise performing a nucleic acid amplification reaction, thereby producing the first amplicon; detecting the first amplicon; and contacting the sample comprising cotton DNA with said primer set (said primer set when used in a nucleic-acid amplification reaction with genomic DNA from cotton plants, produces a second amplicon comprising the native cotton genomic DNA homologous to the cotton genomic region of a transgene insertion identified as one of said cotton events); and performing a nucleic acid amplification reaction, thereby producing the second amplicon. The methods further comprise detecting the second amplicon, and comparing the first and second amplicons in a sample, wherein the presence of both amplicons indicates that the sample is heterozygous for the transgene insertion.<br>
[0082]	DNA detection kits can be developed using the compositions disclosed herein and<br>
methods well known in the art of DNA detection. The kits are useful for identification of the subject cotton event DNA in a sample and can be applied to methods for breeding cotton plants containing this DNA. The kits contain DNA sequences homologous or complementary to the amplicons, for example, disclosed herein, or to DNA sequences homologous or complementary to DNA contained in the transgene genetic elements of the subject events. These DNA sequences can be used in DNA amplification reactions or as probes in a DNA hybridization method. The kits may also contain the reagents and materials necessary for the performance of the detection method.<br><br>
[0083]	A "probe" is an isolated nucleic acid molecule to which is attached a conventional<br>
detectable label or reporter molecule (such as a radioactive isotope, ligand, chemiluminescent agent, or enzyme). Such a probe is complementary to a strand of a target nucleic acid, in the case of the present invention, to a strand of genomic DNA from one of said cotton events, whether from a cotton plant or from a sample that includes DNA from the event. Probes according to the present invention include not only deoxyribonucleic or ribonucleic acids but also polyamides and other probe materials that bind specifically to a target DNA sequence and can be used to detect the presence of that target DNA sequence..<br>
[0084]	"Primers" are isolated nucleic acids that are annealed to a complementary target DNA<br>
strand by nucleic acid hybridization to form a hybrid between the primer and the target DNA strand, then extended along the target DNA strand by a polymerase, e.g.. a DNA polymerase. Primer pairs of the present invention refer to their use for amplification of a target nucleic acid sequence, e.g., by the polymerase chain reaction (PCR) or other conventional nucleic-acid amplification methods.<br>
[0085]	Probes and primers are generally 5,6,7, 8,9.10,11,12,13,14,15,16,17,18,19,20,21,<br>
22,23, 24,25,26, 27,28,29, 30,31, 32, 33, 34,35, 36,37,38,39,40,41,42,43,44,45, 46,47, 48,49, 50, 51, 52, 53, 54, 55; 56, 57, 58,59, 60, 61,62,63,64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74,75.. 76,77,78,79, 80, 81, 82,83, 84, 85, 86,87, 88,89,90,91,92,93,94, 95,96,97,98,99, 100,101,102,103,104,105, 106, 107,108,109,110, 111, 112,113,114,115,116,117,118, 119, 120, 121,122,123, 124,125,126,127, 128,129,130,131, 132, 133, 134, 135, 136, 137, 138,139, 140,141,142, 143,144, 145,146,147,148,149,150,151,152,153,154,155,156, 157, 158, 159, 160,161,162, 163, 164,165,166,167,168, 169, 170, 171, 172, 173, 174, 175, 176,177,178,179,180,181,182,183,184,185,186,187,188,189,190,191,192,193,194, 195, 196,197,198,199,200, 201,202,203,204, 205,206,207, 208,209, 210, 211, 212, 213, 214, 215, 216,217, 218, 219, 220, 221, 222,223,224,225, 226,227, 228, 229, 230,231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243,244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260:261, 262,263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276,277,278,279,280, 281,282,283, 284,285, 286, 287, 288, 289, 290,291, 292,293, 294, 295,296, 297,298,299, 300, 301,302, 303, 304, 305, 306,307, 308, 3O9; 310, 311,312, 313, 314, 315, 316,317,318, 319,320,321, 322, 323, 324, 325, 326, 327, 328,329, 330, 331, 332,333,334, 335,336, 337, 338, 339, 340, 341, 342, 343, 344, 345: 346,<br><br>
347, 348, 349,350, 351, 352,353, 354, 355, 3565 357,358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 3685 369, 370, 371,3725 373, 374, 375, 376, 377, 378,379, 380,381,3825 383, 384, 385, 386, 387, 388,389,390, 391,392,393, 394, 395,396, 397, 398: 399,4005 401,402; 403, 404, 405,406; 4079 4085 409:410,411,412,413,414,415,416,417,418,4195 420,421,422? 423, 424, 425,426, 427, 428,429,430,431,432,433,434, 435,436,437,438,439,440,441, 442, 443,444,445, 446, 447,448, 449,450,451,452,453, 454,455, 456,457, 458,459, 460, 461, 462, 463,464, 465,466,467,468,469,470,471, 472,473,474,475,476,477,478, 479, 480,481, 482,483, 484, 485, 486, 487, 488,489,490,491,492,493, 494,495,496,497, 498, 499, or 500 polynucleotides or more in length. Such probes and primers hybridize specifically to a target sequence under high stringency hybridization conditions. Preferably, probes and primers according to the present invention have complete sequence similarity with the target sequence, although probes differing from the target sequence and that retain the ability to hybridize to target sequences may be designed by conventional methods.<br>
[0086]	Methods for preparing and using probes and primers are described, for example, in<br>
Molecular Cloning: A Laboratory Manual, 2nd ed, vol. 1-3. ed. Sambrook et al, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y., 1989. PCR-primer pairs can be derived from a known sequence, for example, by using computer programs intended for that purpose.<br>
[0087]	Primers and probes based on the flanking DNA and insert sequences disclosed herein can<br>
be used to confirm (and, if necessary, to correct) the disclosed sequences by conventional methods, e.g., by re-cloning and sequencing such sequences.<br>
[0088]	The nucleic acid probes and primers of the present invention hybridize under stringent<br>
conditions to a target DNA sequence. Any conventional nucleic acid hybridization or amplification method can be used to identify the presence of DNA from a transgenic event in a sample. Nucleic acid molecules or fragments thereof are capable of specifically hybridizing to other nucleic acid molecules under certain circumstances. As used herein, two nucleic acid molecules are said to be capable of specifically hybridizing to one another if the two molecules are capable of forming an anti-parallel, double-stranded nucleic acid structure. A nucleic acid molecule is said to be the "complement" of another nucleic acid molecule if they exhibit complete complementarity. As used herein, molecules are said to exhibit "complete complementarity" when every nucleotide of one of the molecules is complementary to a nucleotide of the other. Two molecules are said to be "minimally complementary" if they can<br><br>
hybridize to one another with sufficient stability to permit them to remain annealed to one another under at least conventional 'low-stringency" conditions. Similarly, the molecules are said to be "complementary" if they can hybridize to one another with sufficient stability to permit them to remain annealed to one another under conventional "high-stringency"  conditions. Conventional stringency conditions are described by Sambrook et al, 1989. Departures from complete complementarity are therefore permissible, as long as such departures do not completely preclude the capacity of the molecules to form a double-stranded structure. In order for a nucleic acid molecule to serve as a primer or probe it need only be sufficiently complementary in sequence to be able to form a stable double-stranded structure under the particular solvent and salt concentrations employed<br>
[0089]	As used herein, a substantially homologous sequence is a nucleic acid sequence that will<br>
specifically hybridize to the complement of the nucleic acid sequence to which it is being compared under high stringency conditions. The term "stringent conditions" is functionally defined with regard to the hybridization of a nucleic-acid probe to a target nucleic acid (z. e., to a particular nucleic-acid sequence of interest) by the specific hybridization procedure discussed in Sambrook et al, 1989, at 9.52-9.55. See also, Sambrook et al, 1989 at 9.47-9.52 and 9.56-9.58, Accordingly, the nucleotide sequences of the invention may be used for their ability to selectively form duplex molecules with complementary stretches of DNA fragments.<br>
[0090]	Depending on the application envisioned, one can use varying conditions of hybridization<br>
to achieve varying degrees of selectivity of probe towards target sequence. For applications requiring high selectivity, one will typically employ relatively stringent conditions to form the hybrids, e.g., one will select relatively low salt and/or high temperature conditions, such as provided by about 0.02 M to about 0.15 M NaCl at temperatures of about 50° C to about 70° C. Stringent conditions, for example, could involve washing the hybridization filter at least twice with high-stringency wash buffer (0.2X SSC, 0.1% SDS, 65° C). Appropriate stringency conditions which promote DNA hybridization, for example. 6.0X sodium chloride/sodium citrate (SSC) at about 45° C, followed by a wash of 2.0X SSC at 50° C are known to those skilled in the art, 6.3.1-6.3.6. For example, the salt concentration in the wash step can be selected from a low stringency of about 2.0X SSC at 50° C to a high stringency of about 0.2X SSC at 50° C. In addition, the temperature in the wash step can be increased from low stringency conditions at room temperature, aboiit 22° C to high stringency conditions at about 65° C. Both temperature<br><br>
and salt may be varied, or either the temperature or the salt concentration may be held constant while the other variable is changed. Such selective conditions tolerate little, if any, mismatch between the probe and the template or target strand. Detection of DNA sequences via hybridization is well-known to those of skill in the art, and the teachings of U.S. Patent Nos. 4,965,188 and 5.176,995 are exesuplary of the methods of hybridization analyses.<br>
[0091]	In a particularly preferred embodiment, a nucleic acid of the present invention will<br>
specifically hybridize to one or more of the primers (or amplicons or other sequences) exemplified or suggested herein, including complements and fragments thereof, under high stringency conditions. In one aspect of the present invention, a marker nucleic acid molecule of the present invention has the nucleic acid sequence set forth in SEQ ID NOS:3-14, or complements and/or fragments thereof.<br>
[0092]	In another aspect of the present invention, a marker nucleic acid molecule of the present<br>
invention shares between 80% and 100% or 90% and 100% sequence identity with such nucleic acid sequences- In a further aspect of the present invention, a madcer nucleic acid molecule of the present invention shares between 95% and 100% sequence identity with such sequence. Such sequences may be used as markers in plant breeding methods to identify the progeny of genetic crosses. The hybridization of the probe to the target DNA molecule can be detected by any number of methods known to those skilled in the art, these can include, but are not limited to, fluorescent tags, radioactive tags, antibody based tags, and chemiluminescent tags.<br>
[0093]	Regarding the amplification of a target nucleic acid sequence (e.g., by PCR) using a<br>
particular amplification primer pair, "stringent conditions" are conditions that permit the primer pair to hybridize only to the target nucleic-acid sequence to which a primer having the corresponding wild-type sequence (or its complement) would bind and preferably to produce a unique amplification product, the amplicon.<br>
[0094]	The term "specific for (a target sequence)" indicates that a probe or primer hybridizes<br>
under stringent hybridization conditions only to the target sequence in a sample comprising the target sequence.<br>
[0095]	As used herein, "amplified DNA" or "amplicon" refers to the product of nucleic-acid<br>
amplification of a target nucleic acid sequence that is part of a nucleic acid template. For example, to determine whether the cotton plant resulting from a sexual cross contains transgenic event genomic DNA from the cotton plant of the present invention, DNA extracted from a cotton<br><br>
plant tissue sample may be subjected to nucleic acid amplification method using a primer pair that includes a primer derived from flanking sequence in the genome of the plant adjacent to the insertion site of inserted heterologous DNA, and a second primer derived from the inserted heterologous DNA to produce an amplicon that is diagnostic for the presence of the event DNA. The amplicon is of a length and has a sequence that is also diagnostic for the event. The amplicon may range in length from the combined length of the primer pairs plus one nucleotide base pair, and/or the combined length of the primer pairs plus about 2, 3,4,5,6,7,.8,9,10,11,12,13,14, 15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40, 41,42,43,44,45,46,47, 48, 49, 50, 51, 52, 53,54, 55, 56, 57, 58,59,60, 61, 62, 63,64,65, 66, 67,68, 69,70, 71,72,73,74,75,76, 77,78, 79,80,81,82,83, 84, 85,86, 87,88, 89,90,91,92, 93,94,95,96, 97,98,99,100, 101,102,103,104,105,106,107,108,109,110, 111, 112,113, 114,115,116,117, 118,119,120, 121, 122, 123, 124,125, 126, 127,128, 129, 130,131, 132, 133, 134, 135, 136, 137, 138,139, 140,141,142,143,144,145,146,147, 148, 149, 150, 151, 152, 153: 154,155, 156, 157,158, 159, 160, 161, 162,163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181,182,183,184,185,186,187,188,189, 190, 191, 192, 193, 194, 195, 196,197, 198, 199, 200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211,212, 213, 214, 215, 216, 217, 218, 219, 220,221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257,258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274,275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 291, 292,293,294, 295,296,297, 298,299, 300, 301, 302, 303, 304, 305, 306, 307, 308, 309, 310,311, 312, 313, 314, 315, 316, 317, 318, 319, 320, 321, 322, 323, 324, 325, 326, 327, 328, 329, 330, 331, 332, 333, 334, 335, 336, 337, 338, 339, 340, 341, 342, 343, 344, 345, 346, 347, 348, 349, 350, 351, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 376, 377, 378, 379, 380, 381, 382, 383, 384, 385, 386, 387, 388, 389, 390, 391, 392, 393, 394, 395, 396, 397, 398, 399, 400,401, 402, 403, 404, 405, 406, 407, 408, 409,410, 411, 412,413, 414, 415, 416, 417, 418,419,420,421,422,423,424,425,426,427,428,429,430,431,432, 433,434,435,436, 437,438,439,440, 441, 442,443,444,445,446,447, 448,449,450,451,452,453,454,455, 456, 457,458, 459: 460,461, 462, 463,464,465,466,467,468,469,470,471,472,473,474, 475, 476,477,478,479,480,481, 482,483,484, 485, 486, 487,488,489,490,491,492,493,<br><br>
494, 495, 496, 497; 498, 499, or 500, 750,1000,1250, 15005 1750, 2000, or more nucleotide base pairs (plus or minus any of the increments listed above). Alternatively, a primer pair can be derived from flanking sequence on both sides of the inserted DNA so as to produce an amplicon that includes the entire insert nncleotide sequence. A member of a primer pair derived from the plant genomic sequence may be located a distance from the inserted DNA sequence. This distance can range from one nucleotide base pair up to about twenty thousand nucleotide base pairs. The use of the term "amplicon" specifically excludes primer dimers that may be formed in the DNA thermal amplification reaction.<br>
[0096]	Nucleic-acid amplification can be accomplished by any of the various nucleic-acid<br>
amplification methods known in the art, including the polymerase chain reaction (PCR). A variety of amplification methods are known in the art and are described, inter alia, in U.S. Patent No. 4,683,195 andU.S. Patent No. 4;683?202, PCR amplification methods have been developed to amplify up to 22 kb of genomic DNA. These methods as well as other methods known in the art of DNA amplification may b e used in the practice of the present invention. The sequence of the heterologous transgene DNA insert or flanking genomic sequence from a subject cotton event can be verified (and corrected if necessary) by amplifying such sequences from the event using primers derived from the sequences provided herein followed by standard DNA sequencing of the PCR amplicon or of the cloned DNA.<br>
[0097]	The amplicon produced by these methods may be detected by a plurality of techniques.<br>
Agarose gel electrophoresis and staining with ethidium bromide is a common well known method of detecting DNA amplicons. Another such method is Genetic Bit Analysis where an DNA oligonucleotide is designed which overlaps both the adjacent flanking genomic DNA sequence and the inserted DNA sequence. The oligonucleotide is immobilized in wells of a microwell plate. Following PCR of the region of interest (using one, primer in the inserted sequence and one in the adjacent flanking genomic sequence), a single-stranded PCR product can be hybridized to the immobilized oligonucleotide and serve as a template for a single base extension reaction using a DNA polymerase and labelled ddNTPs specific for the expected next base. Readout may be fluorescent or ELISA-based. A signal indicates presence of the insert/flanking sequence due to successful amplification, hybridization, and single base extension.<br><br>
[0098]	Another method is the Pyrosequencing technique as described by Winge (Innov. Phanna.<br>
Tech. 00:18*24,2000). In this method an oligonucleotide is designed that overlaps the adjacent genomic DNA and insert DNA junction. The oligonucleotide is hybridized to single-stranded PCR product from the region of interest (one primer in the inserted sequence and one in the flanking genomic sequence) and incubated in the presence of a DNA polymerase, ATP, sulfurylase, luciferase, apyrase, adenosine 51 phosphosulfate and luciferin. DNTPs are added individually and the incorporation results in a light signal that is measured. A light signal indicates the presence of the transgene insert/flanking sequence due to successful amplification, hybridization, and single or multi-base extension.<br>
[0099]	Fluorescence Polarization is another method that can be used to detect an amplicon of the<br>
present invention. Following this method, an oligonucleotide is designed which overlaps the genomic flanking and inserted DNA junction. The oligonucleotide is hybridized to single-stranded PCR product from the region of interest (one primer in the inserted DNA and one in fhe flanking genomic DNA sequence) and incubated in the presence of a DNA polymerase and a fluorescent-labeled ddNTP. Single base extension results in incorporation of the ddNTP. Incorporation can be measured as a change in polarization using a fluorometer. A change in polarization indicates the presence of the transgene insert/flanking sequence due to successful amplification, hybridization- and single base extension.<br>
[0010]	TAQMAN (PE Applied Biosystems, Foster City, Calif.) is a method of detecting and<br>
quantifying the presence of a DNA sequence. Briefly, a FRET oligonucleotide probe is designed that overlaps the genomic flanking and insert DNA junction. The FRET probe and PCR primers (one primer in the insert DNA sequence and one in the flanking genomic- sequence) are cycled in the presence of a thermostable polymerase and dNTPs. Hybridization of the FRET probe results in cleavage and release of the fluorescent moiety away from the quenching moiety on the FRET probe. A fluorescent signal indicates the presence of the flanking/transgene insert sequence due to successful amplification and hybridization.<br>
[00101]	Molecular Beacons have been described for use in sequence detection. Briefly, a FRET<br>
oligonucleotide probe is designed that overlaps the flanking genomic and insert DNA junction. The unique structure of the FRET probe results in it containing secondary structure that keeps the fluorescent and quenching moieties in close proximity. The FRET probe and PCR primers (one primer in the insert DNA sequence and one in the flanking genomic sequence) are cycled in the<br><br>
presence of a thermostable polymerase and dNTPs. Following successful PCR amplification,<br>
hybridization of the FRET probe to the target sequence results in the removal of the probe<br>
secondary structure and spatial separation of the fluorescent and quenching moieties. A<br>
fluorescent signal results. A fluorescent signal indicates the presence of the flanking<br>
genomic/transgene insert sequence due to successful amplification and hybridization.<br>
[00102	Having disclosed two general locations in the cotton genome that are excellent for<br>
insertions, the subject invention also comprises a cotton seed and/or a cotton plant comprising at least one non-crylF and non-crylAc insert in the general vicinity of one or both of these locations. One option is to substitute a different insert in place of the crylF and/or cryl Ac insert exemplified herein. In these generally regards, targeted homologous recombination, for example, can be used according to the subject invention. This type of technology is the subject of, for example, WO 03/080809 A2 and the corresponding published U.S. application (USPA 20030232410).<br>
 All patents, patent applications, provisional applications, and publications referred to or<br>
cited herein are incorporated by reference in their entirety to the extent they are not inconsistent with the explicit teachings of this specification.<br>
[00104]	The following examples are included to illustrate procedures for practicing the invention<br>
and to demonstrate certain preferred embodiments of the invention. These examples should not be construed as limiting. It should be appreciated by those of skill in the art that the techniques disclosed in the following examples represent specific approaches used to illustrate preferred modes for its practice. However, those of skill in the art should, in light of the present disclosure, appreciate that many changes can be made in these specific embodiments while still obtaining like or similar results without departing from the spirit and scope of the invention. Unless otherwise indicated, all percentages are by weight and all solvent mixture proportions are by volume unless otherwise noted.<br>
Example 1 - Production of Deposited Seed<br>
[00105]	WideStrike™ brand insect resistance for cotton is a transgenic trait developed by Dow<br>
AgroSciences that provides in-plant insect resistance against Lepidoptera. It contains two insect<br><br>
tolerance genes, cry 1 Ac and ay IF, which were derived from Bacillus thuringiensis subspecies kurstak  and Bacillus thuringiensis subspecies aizawai, respectively. Bacillus thxcringiensis (B.t) is a common, gram-positive, soil-borne bacterium. In its spore-forming stage, it produces several insecticidal protein crystals (known as delta-endotoxins) including CrylAc and crylF. These proteins are toxic to certain lepidopteran insects. In susceptible insects, they bind to specific receptors present on niidgiit epithelial cells, forming pores that disrupt osmotic balance and eventually result in cell lysis and death. crylAc and Cryl have been shown to be non-toxic to humans, livestock, and beneficial insects, which do not have binding sites for the delta-endotoxin. Using two delta-endotoxins rather than one will provide improved insect resistance because the two Qy proteins provide a greater spectrum of control than either does alone and have differential activity against the lepidopteran pests that they are effective against. More important!)', it may help delay the development of resistant insects.<br>
[00106]	The- crvlAc and cry1F genes in WideStrike were introduced using Asrobacterium<br>
mediated transformation into GO510 cotton (Gossypium hirsutum L.) plants in two separate transformation events, 3006-210-23 and 281-24-236. Following crossing into an elite cotton variety, these events were combined by conventional breeding to produce cotton bearing the WideStrike insect-resistance trait WideStrike also contains the pat gene from Streptomyces viridochromogenes, a common aerobic soil bacteria. The pat gene codes for the Phosphinothricin Acetyl Transferase (PAT) enzyme, which detoxifies glufosinate ammonium into an inactive compound by acetylation. The pat gene was included to allow for selection of transformed cotton plants.<br>
Example 2 - Diagnostic test for Cry IF cotton event 281-24-236<br>
[00307]	DNA from CiylY event 281-24-236 and cryl Ac events 3006-210-23, and non-transgenic<br>
cotton PCS355 was extracted from cotton leaves using QIAGEN's Plant DNeasy kit (catalog # 69181, Qiagen, Valencia, CA, USA). The manufacturer's suggested protocol was followed. In briel leaf discs were disrupted in an RNAse supplemented preheated buffer using a tungsten carbide bead (0.125 mm diameter) and a Retsch MM3000 Mixer Mill. The mixture was centrifuged at room temperature, and the supernatant was subsequently captured by running through a DNeasy 96 plate. DNA was eluted in an elution buffer and stored frozen until use.<br><br>
 the DNa extracted from the cotton leaf tissue was used in a PCR DNA amplification of<br>
the 5' genomic/ trangene sequences in Cry1F even 281-24 -236 using primer 281-14(SEQ<br>
ID &gt;*O3, 5TGTCGGCIGAAGGTAGGGAGG31) aad primer 281-15 (^Q ID N CCGGACATGAAGCCAnTAC3'), and the 3' genomic/transgene insert sequenc flanking using primer 281-9 (SEQIDNO:6, STCTCTAGAaAGGGGCACGACCB1) and primer 281-10 (SEQ ID NO:7, 5lCGAGCTGGAGAGACCGGTQAC3r). The PCR DNA amplification analyses were conducted using genomic DNA extracted from cotton event CrylF 281-24-236 and non-transgenic cotton line PCS355. The amplification, reaction for the 5' flacking genomic sequeaice was conducted using QIAGEN HotStarTaq PCR kit (catalog # 203203 or 203205, QIAGEN, Vakaoria, CA, USA) with a final concentration of 0.4 μM for Primer 281-14 and Primer 281-15 in a 50 μl  reaction volume. The reactions were performed using a GenAmp PCR System 9600 (AppBed Biosystem- Foster City, CA) under the following cycling conditions: 1 cycle at 95o C fir 15  minute35 cycles of 94° C for 30 seconds. 57° C for 30 seconds, 72° C for  60 seconds1 cycle 72 C  C for 10 miraTteg The PCR for the 3' flanking genomic sequnce was conduncted. using Takaia ExTaq PCR kit (Catalog # RR001A, Panvera. Madison, WI) in a 50 ul reaction volume containing a final concentration of 0.4 μM of Primer 281-9 and Primer 281-10. The reactions were performed using a GenAmp PCR System 9600 (Applied Biosystem. Posts: City. CA) under the following cycling conditions: 1 cycle at 95° C for S mintrte; 35 cycles of 94° C for 30 seconds, 60° C for 30 seconds, 72° C for 60 seconds; 1 cycle at 72° C for 10 minutes, The PCR products were separated using 1.0% agarose gel electrophoresis at 100 V for about 1 hour and visualized by ethidium bromide staining.<br>
 The 5! PCR product DNA sequence was determined resulting in a 603 nucleotide base<br>
pair sequence representing the 5' genomic/transgene insert sequence of cotton CrylF event 281-<br>
24-236 and identified as SEQ ID NO:5. The 3' PCR product DNA sequence was determined<br>
resulting in a 562 nncieotide base pair sequence representing the 3' genomic/transgene insert<br>
sequence of cotton CrylF event 281-24-236 and identified in SEQ ID NO:S.<br>
[OOJIOJ	The genamic/transgene junction sequences, SEQ ID NO:5 and SEQ ID NO:8 are novel<br>
DNA seqamces m CryYS event 281-24-236 that are diagnostic for cotton plant CryYB event 281-24-236 md. its progeny.<br><br><br><br><br><br><br><br>
	Having illustrated and and described 1he principles of  the present invetnion, it should be<br>
without departing from such such principles. We claim all modifications that are within the spirit and scope  of the appended claims.<br><br><br><br><br><br><br><br>
Claims<br>
1.	A cation seed comprising in its genome cryIF  cotton EVENT 281-24-236 AND CRYl a cotton<br>
2.	event 3006-210-23 and having representative seed deposited with American Type culture<br>
3.	Collection (ATCC) with Accession No. PTA-6233.<br>
4.	A cotton seed comprising in its genome crylF cotton event 281-24-236.<br>
5.	A cotton seed comprising in its genome crylAc cotton event 3006-210-23.<br>
6.	A cotton, plant produced by growing the seed of claim 1.<br>
7.	A progeny cotton plant of the plant of claim 4.<br>
8.	An insect-resistant progeny cotton plant of fee plant of claim 4.<br>
9.	The plant of claim 4 wherein said cotton plant comprises a genome comprising a DNA<br>
10.	sequence selected from residues 2055-12,768 of SEQ ID NO: 1 and 508-8920 of SEQ ID NO.2.<br>
11.	The plant of claim 4 wherein said cotton plant comprises residues 205 5-12,7 68 of SEQ ID<br>
12.	NO:1 and 508-8920 of SEQ ID NO:2.<br>
13.	A part of the cotton plant of claim 4 wherein said part is selected from the group<br>
14.	consisting of pollen, ovale. flowers, bolls, lint, shoots, roots, and leaves.<br>
15.	A transgenic cotton plant comprising an insert that disrupts a genomic sequence selected<br>
16.	from the group consisting of<br>
a. a first DNA sequence comprising a 5' end comprising nucleotides 1 -2074 of SEQ ID NO.l, a first interior segment comprising SEQ ID NO:15, and a 3' end comprising nucleotides 12,749-15,490 of SEQ ID NO: 1, and<br><br>
b- a second DNA sequence comprising a 5' end comprising nucleotides 1-527 of SEQ ID NO.2, a second interior segment comprising SEQ ID No;16 and a 3'<br>
end comprising nucleotides 8,901-9,382 of SEQ ID NO-2<br>
11.	An isolated polynucleotide molecule -wherein said molecule comprises at least 15<br>
12.	nucleotides and maintains hybridization under stringent wash conditions with a nucleic acid<br>
13.	sequence selected from the group consisting of residues 1-2074 of SEQ ID NO:15 residues<br>
14.	12,749-15,490 of SEQ ID NO: 1, residues 1-527 of SEQ ID  NO:2, residues 8,901-9,382 of SEQ<br>
15.	ID NO:2, and complements thereof.<br>
16.	The polynucleotide of claim 11 wherein said polynucleotide comprises a nucleotide<br>
17.	sequence selected from the group consisting of SEQ ID NO3. SEQ ID NO:7, SBQ ID NO:9, and<br>
18.	SBQIDNO.B.<br>
19.	An isolated polynucleotide comprising a nucleotide sequence selected from the group<br>
20.	consisting of residues 2060 to 2090 of SEQ ID NO:L residues 12,733 to 12,765 of SEQ ID<br>
21.	NO:L residues 512-543 of SEQ ID NO:2; and residues 8,885 to 8,916 of SEQ IDNO:2.<br>
22.	The polynucleotide of claim 13 wherein said polynucleotide is an amplicon generated by<br>
23.	polymerase chain reaction and said amplicon comprises a sequence selected from the groin)<br>
24.	consisting of SEQ ID NO:5, SEQ ID NO:8, SEQ ID NO:1 L and SEQ ID NO:14.<br>
25.	A method of detecting the presence of a cotton event in a sample comprising cotton DNA<br>
26.	wherein said method comprises contacting said sample with at least one polynucleotide that is<br>
27.	diagnostic for a cotton event selected from the group consisting of crylF cotton event 281 -24-236<br>
28.	and cryl Ac cotton event 3006-210-23, as present in seed deposited with American Type Culture<br>
29.	Collection (ATCC) with Accession No. PTA-6233.<br><br>
16.      The method of claim 15 wherein said method comprise contacting said sample with a.       a first primer that binds  to a flanking sequence selected from the group consisting<br>
 residues 1-2074 of SEQ ID NO: L residues 12.749-15,490 of SQ ID NO:1, and complements thereof and<br>
b. a second primer that binds to an insert sequence selected from the group consisting of residues 207542,748 of SEQID NO:1 and the complement thereof; and<br>
subjecting said sample to polymerase chain reaction; and assaying for an amplicon generated between said primers.<br>
! 7.      The method of claim 15 wherein said method comprises contacting said sample wife<br>
a,	a  first primer that binds to a flanking sequence selected from the group consisting<br>
of residues 1-527 of SEQ ID NO:2, residues 8,901-9382 of SEQ ID NO:2 and<br>
complements thereof and<br>
b.	a second primer that binds to an insert sequence selected from the group<br>
consisting of residues 528-8,900 of SEQ ID NO:2. and the complement thereof<br>
and<br>
subjecting said sample to polymerase chain reaction; and assaying for an amplicon generated between said primers.<br>
18.	The method of claim 16 wherein said second primer is selected from the group consisting<br>
19.	of SEQ ID NO:4 and SEQ ID NO:6.<br>
20.	The method of claim 17 wherein said second primer is selected from the group consisting<br>
21.	of SEQ ID NO:10 and SEQ ID NO: 12.<br>
22.	The method of claim 15 wherein said polynucleotide comprising at least 30 nucleotides<br>
23.	and hybridizes under stringent conditions with a sequence selected from the group consisting of<br>
24.	residues 2060 to 2090 of SEQ ID NO:1, residues 12,733 to 12,765 of SEQ ID NO:1, residues<br>
25.	512-543 of SEQ ID NO2, residues 8,885 to 8,916 of SEQ ID NO:2, and complements thereof<br>
26.	and -wherein said method rafter comprises subjecting said sample and said polynucleotide to<br><br>
 stringen hybridization conditions, and assaying said sample for hybridization of said polynucleotide to said DNA.<br>
21. A DNA detection kit comprising a first primer and a second primer wherein said primers<br>
are as defined in claim 16.<br>
22.	A DNA detection kit comprising a first primer and a second primer wherein said primers<br>
are as defined in claim 17.<br>
23.	A DNA detection kit comprising a polynucleotide as defined in claim 20.<br>
24.	A polynucleotide comprising a sequence selected from the group consisting of SEQ 3D<br>
NO. 1 and  SBQ ID NO.2.<br>
25.	A plant comprising a genomic sequence comprising  a polynucleotuide according  to<br>
claim 24.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICAgQ09SUkVTUE9OREVOQ0UgT1RIRVJTIDE0LTEwLTIwMTEucGRm" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006   CORRESPONDENCE OTHERS 14-10-2011.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBBTUVOREVEIENMQUlNUyAxMS0wNi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  AMENDED CLAIMS 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBBTUVOREVEIFBBR0VTIE9GIFNQRUNJRklDQVRJT04gMTEtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  AMENDED PAGES OF SPECIFICATION 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgMTEtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  EXAMINATION REPORT REPLY RECEIVED 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBGT1JNLTEgMTEtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  FORM-1 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBGT1JNLTMgMTEtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  FORM-3 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgMTEtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  OTHER PATENT DOCUMENT 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1DSEVOUC0yMDA2ICBQT1dFUiBPRiBBVFRPUk5FWSAxMS0wNi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">3542-CHENP-2006  POWER OF ATTORNEY 11-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzU0Mi1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">3542-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="253524-ionic-liquids-having-uronium-or-thiouronium-cations.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253526-apparatus-for-facilitating-communication-of-data-and-method-thereof.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253525</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3542/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>31/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>03-Aug-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>26-Jul-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Sep-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>DOW AGROSCIENCES LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>9330 ZIONSVILLE ROAD, INDIANAPOLIS, IN 46268-1054</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SONG, PING</td>
											<td>9330 ZIONSVILLE ROAD, INDIANAPOLIS, IN 46268-1054</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TAGLIANI,LAURA</td>
											<td>9330 ZIONSVILLE ROAD, INDIANAPOLIS, IN 46268-1054</td>
										</tr>
										<tr>
											<td>3</td>
											<td>PELLOW,JOHN,W.</td>
											<td>9330 ZIONSVILLE ROAD, INDIANAPOLIS, IN 46268-1054</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/325</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US04/33844</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-10-13</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/556,586</td>
									<td>2004-03-26</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/613,851</td>
									<td>2004-09-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253525-cry-1f-transgenic-cotton-lines-and-event-specific-identification-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:41:18 GMT -->
</html>
