<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/253998-a-2-m-family-plasmid-and-its-method-of-preparation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:09:55 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 253998:&quot;A 2µM-FAMILY PLASMID AND ITS METHOD OF PREPARATION&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;A 2µM-FAMILY PLASMID AND ITS METHOD OF PREPARATION&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A 2µm-family plasmid characterized in that it comprise a polynucleotide sequence insertion, deletion and/or substitution between the first base after the last functional codon of at least one of either a REP2 gene or an FLP gene and the last base before the FRT site in an inverted repeat adjacent to said gene.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to a 2µm family plasmid and its method of preparation. FIELD OF THE INVENTION<br>
The present application relates to modified plasmids and uses thereof.<br>
BACKGROUND OP THE INVENTION<br>
Certain closely related species of budding yeast have been shown to contain naturally occurring circular double stranded DNA plasmids.  These plasmids, collectively termed 2µm-family plasmids, include pSR1, pSB3 and pSB4 from Zygosaccharomyces ronxii (formerly classified as Zygosaccharomyces bisporus), plasmids pSBl and pSB2 from Zygosaccharomyces bailii, plasmid pSMl from Zygosaccharomyces fermentati, plasmid pKD1   from.  Khyveromyces   drosphilarum,   an   un-narned   plasmid   from   Pichia membranaefaciens (hereinafter referred to as "pPM1") 'and the 2µm plasmid and variants (such as Scp1, Scp2 and Scp3) from Saccharomyces cerevisiae (Volkert, et ah, 1989, Microbiological RevieM's, 53, 299; Painting, et al., 1984, J. Applied Bacteriology, 5b, 331) and other Saccharomyces species, such as  S. carlsbergensis.    As a family of plasmids these molecules share a series of common features in. that they possess two inverted repeats on opposite sides of the plasmid. have a similar size around 6-kbp (range 4757 to 6615-bp), at least three open reading frames, one of which encodes for a site specific recombinase (such as FLP in 2 µm) and an autonornously replicating sequence (ARS), also known as an origin of replication (ori), located close to the end of one of the inverted repeats. (Futcher, 1988, Yeast, 4, 27; Murray et al, 1988, J. Mol Biol. 200, 601 and Toh-e et al, 1986, Basic Life Sci. 40, 425).   Despite their lack of discernible DNA sequence homology, their shared molecular architecture and the conservation of function of the open reading frarnes have demonstrated a common link between the family members.<br>
The 2 µm plasmid (Figure 1) is a 6,318-bp double-stranded DNA plasmid, endogenous in most Saccharomyces cerevisiae strains at 60-100 copies per haploid genome. The 2 µm plasmid comprises a small-unique (US) region and a large unique (UL) region, separated<br>
, by two 599-bp inverted repeat sequences. Site-specific recombination of the inverted<br>
repeat sequences results in inter-conversion between the A-fonn and B-form of the<br>
plasmid 277 vivo (Volkert &amp; Broach, 1986, Cell, 46, 541). The two forms of 2um differ<br>
only in the relative orientation of their unique regions.<br>
While DNA sequencing of a cloned 2um plasmid (also known as Scpl) from<br>
Saccharomyces cerevisiae gave a size of 6,318-bp (Hartley and Donelson, 19805 Nature,<br>
286, 860), other slightly smaller variants of 2uni, Scp2 and Scp3, are known to exist as a<br>
result of small deletions of 125-bp and 220-bp, respectively, in a region known as STB<br>
(Cameron et al, 1977, Nucl Acids Res., 4, 1429: Kikuchi, 1983, Cell, 35, 487 and<br>
Livingston &amp; Hahne, 1979., Proc. Natl. Acad. Sci. USA, 76, 3727). In one study about<br>
80% of natural Saccharomyces strains from around the world contained DNA<br>
homologous to 2pm (by Southern blot analysis) (Hollenberg, 1982, Current Topics in<br>
Microbiology and Immunobiology, 96, 119). Furthermore, variation (genetic<br>
polymorphism.) occurs within the natural population of 2 urn plasmids found in S.<br>
cerevisiae and S. carlsbergensis, with the NCBI sequence (accession number<br>
NC_001398) being one example.<br>
The 2p.m. plasmid has a nuclear localisation and displays a high level of mitotic stability<br>
(Mead et al, 1986, Molecular &amp; General Genetics, 205, 417). The inherent stability of<br>
the 2pm. plasmid results from a plasmid-encoded copy number amplification and<br>
partitioning mechanism, which is easily compromised during the development of<br>
chimeric vectors (Futcher &amp; Cox, 1984, J. Bacterial, 157, 283; Bachmair &amp; Ruis, 1984,<br>
Monatshefte fiir Chemie, 115, 1229). A yeast strain, which contains a 2 urn plasmid is<br>
known as [cir*], while a yeast strain which does not contain a 2um plasmid is known as<br>
The US-region contains the REP2 and FLP genes, and the UL-region contains the KEPI<br>
and D (also known as RAF) genes, the STB-locus and the origin of replication (Broach &amp;<br>
Hicks, 1980, Cell, 21, 501; Sutton &amp; Broach, 1985, Mol Cell. Bipl, 5, 2770). The Flp<br>
recombinase binds to FRT-sites (Flp Recognition Target) within the inverted repeats to<br>
mediate site-specific recombination, which is essential for natural plasmid amplification<br>
j*and control ofplasniid copy number m vivo (Senecoffef al. 19S5, Proc. Natl Acad. &amp;ci.<br>
U.S.A: 82, 7270: Jayaram, 1985. Proc. Natl Acad. Sci. U.S.A., 82, 5875). The copy<br>
number of 2um-family plasmids can be significantly affected by changes in Flp<br>
recombmase activity (Sleep e1 al, 2001, Yeast, 18, 403; Rose &amp; Broach, 1990, Methods<br>
Enzymol, 185. 234). The Repl and Rep2 proteins mediate plasmid segregation, although<br>
then mode of action is unclear (Sengupta et al. 2001, J. Bacterial, 1&amp;3, 23 06). They also<br>
repress transcription of the FLP gene (Reynolds et al, 1987, Mol. Cell JBioL, 7, 3566).<br>
The FLP and REP2 genes are transcribed from divergent promoters, with apparently no<br>
rntenrening sequence defined between them. The FLP and REP2 transcripts both<br>
terminate at the same sequence motifs within the inverted repeat sequences, at 24-bp and<br>
17S-bp respectively after their translation tennrnation codons (Sutton &amp; Broach. 1985,<br>
Mol. Cell. Bid., 5, 2770).<br>
In the case of FLP, the C-terrninal coding sequence also lies within the inverted repeat<br>
sequence. Furthermore, fhe two inverted repeat sequences are highly conserved over<br>
599-bp, a feature considered advantageous to efficient plasmid replication and<br>
amplification in vivo, although only the FRT-sites (less than 65-bp) are essential for sitespecific<br>
recombination in -vitro (Senecoff et al, 1985, Proc. Natl. Acad. Sci. U.S.A., 82,<br>
7270; Jayaram, 1985, Proc. Natl. Acad. Sci. U.S.A., 82, 5875; Meyer-Leon et al, 1984,<br>
Cold Spring Harbor Symposia On Quantitative Biology, 49, 797). The key catalytic<br>
residues of Flp are argiiri.ne-308 and tyrosine-343 (which is essential) with strand-cutting<br>
facilitated by histidine-309 and Mstidine 345 (Prasad et al, 1987, Proc. Natl Acad. Sci.<br>
U.S.A., 84, 2189; Chen et al, 1992, Cell, 69, 647; Grainge et al, 2001, J. Mol. Biol, 314,<br>
717).<br>
Two functional domains are described in Rep2. Residues 15-58 form a Repl-binding<br>
domain, and residues 59-296 contain a self-association and STB-binding region<br>
(Sengupta et al, 2001, J. Bacteriol, 183, 2306).<br>
Clumeric or large deletion mutant derivatives of 2um which lack many of the essential<br>
functional regions of the 2 urn plasmid but retain functional the cis element ARS and STB,<br>
cannot effectively partition between mother and daughter cells at cell division. Such<br>
plasrnids can do so if these functions are supplied in trans, by for instance the provision<br>
of a functional 2 urn plasmid within the host, a so called [cir+] host.<br>
Genes of interest have previously been inserted into the UL-region of the 2um plasrnid.<br>
For example, see plasmid pSACSUl in EP 0 286 424. However, there is likely to be a<br>
limit to the amount of DNA that can usefully be inserted into the UL-region of the 2(jm<br>
plasmid without generating excessive asymmetry between the US and UL-regions.<br>
Therefore, the US-region of the 2pm plasmid is particularly attractive for the insertion of<br>
additional DNA sequences, as this would tend to equalise the length of DNA fragments<br>
either side of the inverted repeats.<br>
This is especially true for expression vectors, such as that shown in Figure 2, in which the<br>
plasmid is already crowded by the introduction of a yeast selectable marker and adjacent<br>
DNA sequences. For example, the plasmid shown in Figure 2 includes a (3-lactamase<br>
gene (for ampicillin resistance), a LEU2 selectable marker and an oligonucleotide linker,<br>
the latter two of which are inserted into a unique SndBI-site within the UL-region of the<br>
2(.tm-family disintegration vector, pSACS (see EP 0 286 424). The E. coli DNA between<br>
the Xbal-sites that contains the ampicillin resistance gene is lost from the plasmid shown<br>
in Figure 2 after transformation into yeast. This is described in Chinery &amp; Hinchliffe,<br>
1989, Curr. Genet, 16, 21 and EP 0 286 424, where these types of vectors are designated<br>
"disintegration vectors". In the crowded state shown in Figure 2, it is not readily<br>
apparent where further polynucleotide insertions can be made. A JVM-site within the<br>
linker has been used for the insertion of additional DNA fragments, but tin's contributes to<br>
further asymmetry between the UL and US regions (Sleep et al, 1991, Biotechnology (N<br>
Y), 9, 183).<br>
We had previously attempted to insert additional DNA into the US-region of the 2um<br>
plasrnid and maintain its high inherent plasmid stability. In the 2um-family<br>
disintegration plasrnid pS ACS 00, a 1.1-Kb DNA fragment containing the URA3 gene was<br>
inserted into JSagl-site between REP2 and FLP in US-region in such a way that<br>
transcription from the URA3 gene was in same direction as REP2 transcription (see EP 0<br>
*286 -:24']. When S150-2B [cirL'J was transformed to uracil protorrophy by pSAC300. it<br>
was shown to be considerably less stable (50% plasrnid loss in under 30 generations) than<br>
comparable vectors with URA3 inserted into the UL-region of 2um (0-.10% plasmid loss<br>
m under 30 generations) (Chinery £ Hinchliffe, 1989, Ctirr. Genet, 16, 21; EP 0 286<br>
424). Thus, insertion at the Eagl site may have interfered with FLP expression and it was<br>
concluded that the insertion position could have a profound effect upon the stability of<br>
the resultant plasmid, a conclusion confirmed by Bijvoet et al., 1993, Yeast,. 1, 347.<br>
It is desirable to insert further potynucleotide sequences into 2p.ni-family plasniids. For<br>
example, the insertion of polynucleotide sequences that encode host derived proteins.<br>
recombinant proteins, or non-coding antisense or UNA interference (RNAi) transcripts<br>
may be desirable. Moreover, it is desirable to introduce multiple farther polynucleotide<br>
sequences into 2ajn-family plasmids, thereby to provide a plasrnid which encodes, for<br>
example, multiple separate!}' encoded multi-subunit proteins, different members of the<br>
same metabolic pathway, additional selective markers or a recombinant protein (single or<br>
multi-subunit) and a chaperone to aid the expression of the recombinant protein.<br>
However, the 6,31£-bp 2um plasmid, and other 2 jam-family plasmids, are crowded with<br>
functional genetic elements (Sutton &amp; Broach, 1985, Mol. Cell. Biol, 5, 2770; Broach ei<br>
al, 1979. Cell, 16, 827), with no obvious positions existing for the insertion of additional<br>
DNA sequences without a concomitant loss in plasrnid stability. In fact, except for the<br>
region between the origin of replication and the D gene locus, the entire 2pm plasmid<br>
genome is transcribed into at least one poly(A)+ species and often more (Sutton &amp;<br>
Broach, 1985, Mol. Cell. Biol, 5, 2770). Consequently, most insertions might be<br>
expected to have a detrimental impact on plasmid function 777 vivo.<br>
Indeed, persons skilled in the art have given np on inserting heterologous porynucleotide<br>
sequences into 2urn-family plasmids.<br>
Robinson et al, 1994, Sio/TechnoJog)', 12, 3S1-384 reported that a recombinant<br>
additional PDI gene cop}' in Saccharomyces cerevisiae could be used to increase the<br>
recombinant expression of human platelet derived growth factor (PDGF) B homodinier<br>
by ten-fold and Schizosacharomyces pombe acid phosphatase by four-fold. Robiiison<br>
obtained the observed increases in expression of PDGF and S. pombe acid phosphatase<br>
using an additional chromosomally integrated PDI gene copy. Robinson reported that<br>
attempts to use the multi-copy 2um expression vector to increase PDI protein levels had<br>
had a detrimental effect on heterologous protein secretion.<br>
Shusta et al, 1998, Nature Biotechnology/, 16, 773-777 described the recombinant<br>
expression of single-chain antibody fragments (scFv) in Saccharomyces cerevisiae.<br>
Shusta reported that in yeast systems, the choice between integration of a transgene into<br>
the host chromosome versus the use of episomal' expression vectors can greatly affect<br>
secretion and., with reference to Parekh &amp; Wittrup, 1997, Biotechnol. Prog., 13,117-122,<br>
that stable integration of the scFv gene into the host chromosome using a 5 integration<br>
vector was superior to the use of a 2um-based expression plasmid. Parekh &amp; Wittrup,<br>
op. cit, had previously taught that the expression of bovine pancreatic trypsin inhibitor<br>
(BPTI) was increased by an order of magnitude using a 5 integration vector rather than a<br>
2 urn-based, expression plasmid. The 2{.im-based expression plasmid was said to be<br>
counter-productive for the production of heterologous secreted protein.<br>
Bao et al, 2000, Yeast, 16, 329-341, reported mat the KIPDI1 gene had been introduced<br>
into K. lactis on a multi-copy plasmid, pKan707, and that the presence of the plasmid<br>
caused the strain to grow poorly, hi the light of the earlier findings in Bao et al, 2000,<br>
Bao &amp; Fukuliara, 2001, Gene, 272, 103-110, chose to introduce a single duplication of<br>
KJ.PDI1 on the host chromosome.<br>
Accordingly, the art teaches the skilled person to integrate transgenes into the yeast<br>
chromosome, rather than into a multicopy vector. There is, therefore, a need for<br>
alternative ways of transforming yeast.<br>
DESCRIPTION OF THE INVENTION<br>
The present invention relates to recornbinantry modified versions of 2pm~faniiry<br>
plasmids.<br>
A 2pjn-iamily plasmid is a circular, double stranded DMA plasmid. It is Typically small,<br>
sucl) as between 3,000 to ] 0.000 bp, preferabty between 4,500 to 7000 bp, excluding<br>
reconibinantly inserted sequences. Preferred 2 urn-family plasmids for use in the present<br>
invention comprise sequences derived from one or more of plasmids pSE.1, pSB3, or<br>
pSB4 as obtained from Zygosaccharomyces rouxii, pSBl or pSB2 both as obtained from<br>
Zygosaccharomyces bailli, pSMl as obtained from Zygosaccharomyces fennemati,<br>
pKDl as obtained from Khryveromyces1 drosophilarum, pPMl as obtained from Pichia<br>
nembranaefadens and the 2f.Lm plasmid and variants (such as Scpl. Scp2 and Scp3) as<br>
obtained from Saccharoinyces cerevisiae. for example as described hi Vollcert e1 al, 1989,<br>
Microbiological Reviews, 53(3), 299-317, Murray et al 1988, Afo/. BioL, 200, 601-607<br>
and Painting, et al., 1984, J. Applied Bacteriology, 56. 331.<br>
A 2|im.-family plasmid is capable of stable multicopy maintenance within a yeast<br>
population, although not necessaiity all 2um-faniiry plasmids will be capable of stable,<br>
multicopy maintenance within all types of yeast population. For example, the 2inn<br>
plasmid is capable of stable multicopy maintenance, inter alia, within Saccharomvcescerevisiae<br>
and Saccharomyces carlsbergensis.<br>
By "multicopy maintenance" we mean that the plasmid is present in multiple copies<br>
within each yeast cell. A yeast cell comprising 2 jim-family plasmid is designated [cir4],<br>
whereas a yeast cell that does not comprise 2pjn-family plasmid is designated [cir°J. A<br>
[cir+] j^east cell typically comprises 10-100 copies of 2um-famil3' plasmid per haploid<br>
genome, such as 20-90, more typically 30-80, preferably 40-70, more preferably 50-60<br>
copies per haploid genome. Moreover, the plasmid copy number can be affected by the<br>
genetic background of the host which can increase the plasmid copy number of 2u.m-like<br>
plasmid to above 100 per haploid genome (Gerbaud and Ohierineau, 1980, Citrr.<br>
Genetics, 1, 219, Holm, 19S2, Cell, 29, 585, Sleep et al., 20013 Yeast, 18, 403 and<br>
W099/00504). Multicopy stability is defined below.<br>
A 2um~fainily plasmid typically comprises at least three open reading frames ("OJRPs")<br>
that each encode a protein that functions in the stable maintenance of the 2u,m-family<br>
plasmid as a: multicopy plasmid. The proteins encoded by the three ORFs can be<br>
designated FLP, RJ3P1 and REP2. Where a 2jim-family plasmid comprises not all three<br>
of the ORFs encoding FLP, REP1 and REP2 then ORFs encoding the missing protein(s)<br>
should be supplied in. trans, either on another plasmid or by chromosomal integration.<br>
A "FLP" protein is a protein capable of catalysing the site-specific recombination<br>
between inverted repeat sequences recognised by FLP. Hie inverted repeat sequences are<br>
termed FLP recombination target (FRT) sites and each is typically present as part of a<br>
larger inverted repeat (see below). Preferred FLP proteins comprise the sequence of the<br>
FLP proteins encoded by one of plasmids pSRl, pSBl, pSB2, pSB3, pSB4, pSMl,<br>
pKDl, pPMl and the 2um plasmid, for example as described in Volkert et al, op. cit,<br>
Murra}' et al, op. cit and Painting et al, op. cit. Variants and fragments of these FLP<br>
proteins are also included in the present invention. "Fragments" and "variants" are those<br>
which retain the ability of the native protein to catalyse the site-specific recombination<br>
between the same FRT sequences. Such variants and fragments will usually have at least<br>
50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, or more, homology with an FLP protein<br>
encoded by one of plasmids pSRl, pSBl, pSB2, pSB3, pSMl, pKDl and the 2um<br>
plasmid. Different FLP proteins can have different FRT sequence specificities. A typical<br>
FRT site may comprise a core nucleotide sequence flanked by inverted repeat sequences.<br>
In the 2um plasmid, the FRT core sequence is 8 nucleotides in length and the flanking<br>
inverted repeat sequences are 13 nucleotides in length (Volkert et al, op. cit.'), However<br>
the FRT site recognised by any given FLP protein may be different to the 2|-im plasmid<br>
FRT site.<br>
REP1 and REP2 are proteins involved in the partitioning of plasmid copies during cell<br>
division, and may also have a role in the regulation of FLP expression. Considerable<br>
sequence divergence has been observed between REP1 proteins from different 2 urnfamily<br>
plasmids., whereas no sequence alignment is currently possible between REP2<br>
proteins derived from different 2 urn-family plasmids. Preferred REP1 and REP2<br>
proteins comprise the sequence of the REP1 and REP2 proteins encoded by one of<br>
plasmids pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl and the 2j.im plasmid,<br>
for example as described in Vollcert et al, op. cit, Murray et al, op. cit. and Painting et al,<br>
%JP. cii. Variants and fragments of these RE?] and R_h;.P2 proteins are. also included in the<br>
present invention. "Fragments" and '"valiants" of REP 1 and REP2 are those which, when<br>
encoded by the plasmid in place of the native ORF, do not disrupt the stable multicopy'<br>
maintenance of the plasmid within a suitable yeast population. Such variants and fragments<br>
of KEPI and REP2 wall usually have at least 5%, 10%r 20%, 30%, 40%., 50%, 60%, 70%,<br>
80%, 90%, 95%, 98%, 99%, or more, honiology with a REP1 and REP2 protein,<br>
respectively, as encoded by one of plasrnids pSRl, pSBl, pSB2; pSB3, pSB4, pSMl.<br>
pICDl, pPMl and the 2 urn plasmid.<br>
The REP1 and REP2 proteins encoded'by the ORFs on the plasmid must be compatible.<br>
REP1 and REP2 are compatible if they contribute, in combination with the other<br>
functional elements of the plasmid, towards the stable multicopy maintenance of the<br>
plasmid which encodes them. Whether or not a REP1 and REP2 ORF contributes<br>
towards the stable multicopy maintenance of the plasmid which encodes them can be<br>
determined by preparing mutants of the plasmid in which, each of the REP1 and REP2.<br>
ORF's are specificall}' disrupted. If the disruption of an ORF impairs the stable multicop}^<br>
maintenance of the plasmid then the ORF' can be concluded to contribute towards the<br>
stable multicopy maintenance of the plasmid in the non-mntated version. It is preferred<br>
that the REP1 and REP2 proteins have the sequences of REP1 and PJBP2 proteins<br>
encoded by the same naturally occurring 2urn-family plasmid. such as pSRl, pSBl,<br>
pSB2, pSB3, pSB4, pSMl, pKDl; pPMl and the 2um plasmid, or variant or fragments<br>
thereof.<br>
A 2urn-family plasmid comprises two inverted repeat sequences. The inverted repeats<br>
ma}1 be any size, so long as they each contain an FRT site (see above). The inverted<br>
repeats are typically highly homologous. They may share greater than 50%, 60%, 70%,<br>
• 80%, 90%, 95%, 96%, 97%, 98%, 99%, 99.5% or more sequence identity. In a preferred<br>
embodiment they are identical. Typically the inverted repeats are each between 200 to<br>
1000 bp in length. Prefen-ed inverted repeat sequences may each have a length of from<br>
200 to 300 bp, 300 to 400 bp., 400 to 500 bp, 500 to 600 bp, 600 to 700 bp, 700 to 800 bp,<br>
800 to 900 bp, or 900 to 1000 bp. Particularly preferred inverted repeats are those of the<br>
plasmids pSRl (959 bp), pSBl (675 bp), pSB2 (477 bp),pSB3 (391 bp), pSMl (352 bp),<br>
pKDl (346 bp), the 2 urn plasmid (599 bp), pSB4 and pPMl.<br>
The sequences of the inverted repeats may be varied. However, the sequences of the<br>
FRT site in each inverted repeat should be compatible with the specificity of the FLP<br>
protein encoded by the plasmid, thereby to enable the encoded FLP protein to act to<br>
catalyse the site-specific recombination between the inverted repeat sequences of the<br>
plasmid. Recombination between inverted repeat sequences (and thus the ability of the<br>
FLP protein to recognise the FRT sites with the plasmid) can be determined by methods<br>
known in the art. For example, a plasmid in a yeast cell under conditions that favour FLP<br>
expression can be assayed for changes in the restriction profile of the plasmid which<br>
would result from a change in the orientation of a region of the plasmid relative to<br>
another region of the plasmid. The detection of changes in restriction profile indicate that<br>
the FLP protein is able to recognise the FRT sites in the plasmid and therefore that the<br>
FRT site in each inverted repeat are compatible with the specificity of the FLP protein<br>
encoded by the plasmid.<br>
In a particularly preferred embodiment, the sequences of inverted repeats, including the<br>
FRT sites, are derived from the same 2um-family plasmid as the ORF encoding the FLP<br>
protein, such as pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl or the 2um<br>
plasniid.<br>
The inverted repeats are typically positioned within the 2p,m-family plasmid such that the<br>
two regions defined between the inverted repeats (e.g. such as defined as UL and US in<br>
the 2um plasmid) are of approximately similar size, excluding exogenously introduced<br>
sequences such as transgenes. For example, one of the two regions ma}' have a length<br>
equivalent to at least 40%, 50%, 60°/o, 70%, 80%, 90%, 95% or more, up to 100%, of the<br>
length of the other region.<br>
A 2urn-family plasniid comprises the ORF that encodes FLP and one inverted repeat<br>
(arbitrarily termed "TR1" to distinguish it from the other inverted repeat mentioned in the<br>
next paragraph) juxtaposed in such a manner that TR1 occurs at the distal end of the FLP<br>
ORP, without any intervening coding sequence, for example as seen in the 2 pro plasmid.<br>
By "distal end" in this context we mean the end of the FLP ORF opposite to the end from<br>
which the promoter initiates its transcription. In a preferred embodiment, the distal end<br>
of the FLP ORF overlaps with IPU.<br>
A 2 urn-family plasmid comprises the ORF that encodes REP2 and the oilier inverted<br>
repeat (arbitrarily termed "IEJ2" to distinguish it from IR.1 mentioned in the previous<br>
paragraph) juxtaposed in sncfi a manner that IR2 occurs at the distal end of the REP2<br>
ORF, without an}' intervening coding sequence, for example as seen in the 2f.im plasmid.<br>
By '"distal end" in this context we mean the end of the REP2 ORF opposite to the end<br>
from which the promoter initiates its transcription.<br>
In one embodiment, the OEFs encoding REP2 and FLP ma}' he present on the same<br>
region of the two regions defined between the inverted repeats of the 2j.Lm-farnily<br>
plasmid, which region ma}' be the bigger or smaller of the regions (if there is any<br>
inequality in size between the two regions).<br>
In one embodiment, the ORJFs encoding REP2 and FLP may be transcribed from<br>
divergent promoters.<br>
Typically, the regions defined between the inverted repeats (e.g. such as defined as UL<br>
and US in the 2um plasmid) of a 2um-family plasmid ma}' comprise not more than two<br>
endogenous genes that encode a protein that functions in the stable maintenance of the<br>
2urn-family plasmid as a multicopy plasmid. Thus in a preferred embodiment, one<br>
region of the plasmid defined between the inverted repeats may comprise not more than<br>
the ORFs encoding FLP and REP2; FLP and REPl; or REPl and REP2, as endogenous<br>
coding sequence,<br>
A 2um-family plasmid comprises an origin of replication (also laiown as an<br>
autonomously replicating sequence - "ARS"), which is typically bidirectional. Any<br>
appropriate ARS sequence can be present. Consensus sequences typical of yeast<br>
chromosomal origins of replication may be appropriate (Broach ei ol, 1982, Cold Spr<br>
•ing<br>
Harbor Symp. Quant. Biol, 41, 1165-1174; Williamson, Yeast, 1985,1, 1-14). Preferred<br>
ARSs include those isolated from pSRl, pSBl, pSB2, pSB3, pSB4, pSMl, pKDl, pPMl<br>
and the 2um plasmid.<br>
Thus, a 2(.Lm-family plasmid typically comprises at least ORFs encoding FLP and REP2,<br>
two inverted repeat sequences each inverted repeat comprising an FRT site compatible<br>
with FLP protein, and an ARS sequence. Preferably the plasmid also comprises an ORF<br>
encoding REPl, although it may be supplied in trans, as discussed above. Preferably the<br>
FRT sites are derived from the same 2um-famiry plasmid as the sequence of the encoded<br>
FLP protein. Preferably the sequences of the encoded REPl and REP2 proteins are<br>
derived from the same 2pm-family plasmid as each other. More preferably, the FRT sites<br>
are derived from the same 2am-fainily plasmid as the sequence of the encoded FLP,<br>
REPl and REP2 proteins. Even more preferably, the sequences of the ORFs encoding<br>
FLP, REPl and REP2, and the sequence of the inverted repeats (including the FRT sites)<br>
are derived from the same 2j.iin-family plasmid. Yet more preferably, the ARS site is<br>
obtained from the same 2urn-family plasmid as one or more of the ORFs of FLP, REPl<br>
and REP2, and the sequence of the inverted repeats (including the FRT sites). Preferred<br>
plasmids include plasroids pSRl, pSB3 and pSB4 as obtained from Zygosaccharomyces<br>
rouxii, pSBl or pSB2 both as obtained from Zygosaccharomyces bailli, pSMl as<br>
obtained from Zygosaccharomyces fermentati, pKDl as obtained from Kluyveromyces<br>
drosophilarum, pPMl as obtained from Pichia membranaefaciens, and the 2um plasmid<br>
as obtained from Saccharomyces cerevisiae, for example as described in Volkert et al,<br>
1989, op. tit, Murray et al, op. tit. and Painting et al, op. tit.<br>
Optionally, a 2um-farnily plasmid may comprise a region equivalent to the STB region<br>
(also known as REPS) of the 2um plasmid., as defined in Volkert et al, op. tit. The STB<br>
region in a 2jim-family plasmid of the invention may comprise two or more tandern<br>
repeat sequences, such as three, four, five or more. Alternatively, no tandem repeat<br>
sequences may be present. The tandem repeats may be any size, such as 10, 20, 30, 40,<br>
50, 60 70, 80, 90, 100 bp or more in length. The tandem repeats in the STB region of the<br>
2(.im plasmid are 62 bp in length. It is not essential for the sequences of the tandem<br>
repeats to be identical. Slight sequence variation can be tolerated. It may be preferable<br>
12<br>
%&amp; select an STB region from the- same plasmid as either or both of the REP1 and Klil-'2<br>
OPLFs. The STB region is thought to be a czj-acting element and preferably is not<br>
transcribed.<br>
Optional!)-, a 2f.im-family plasmid ma}' comprise an additional ORP that encodes a<br>
protein that functions in the stable maintenance of the 2uni-farnily plasmid as a<br>
multicopy plasmid. The additional protein can be designated RAP or D, ORFs encoding<br>
the RAP or D gene can be seen on, for example, the 2f.Lm plasmid and pSMl. Thus a<br>
RAP or D ORP can comprise a sequence suitable to encode the protein product of the<br>
RAP or D gene ORFs encoded by the 2urn plasmid or pSMl, or variants and fragments<br>
thereof. Thus variants and fragments of the protein products of the RAF or D genes of the<br>
2um plasmid or pSMl are also included in the present invention. "Fragments" and<br>
'Variants" of the protein products of the RAF or D genes of the 2um plasmid or pSMl are<br>
those which., when encoded by the 2uxn plasmid or pSMl in place of the native ORP, do<br>
not disrupt the stable multicopy maintenance of the plasmid within a suitable yeast<br>
population Such variants and fragments will usually have at least 5%, 10%, 20%, 30%,<br>
40%, 50%, 60%, 70%, 80%, 90%, 95%, 98%, 99%, or more, homology with the protein<br>
product of the RAF or D gene ORFs encoded by the 2urn plasmid or pSMl.<br>
The present invention provides a 2pm-fainily plasmid comprising a polynucleotide<br>
sequence insertion, deletion and/or substitution between the first base after the last<br>
functional codon of at least one of either a REP2 gene or an FLP gene and the last base<br>
before the FRT site in an inverted repeat adjacent to said gene.<br>
A polynucleotide sequence insertion is an}' additional polynucleotide sequence inserted<br>
into the plasniid. Preferred polynucleotide sequence insertions are described "below. A<br>
deletion is removal of one or more base pairs, such as the removal of up to 2, 3, 4, 5. 6, 7,<br>
8; 9, 10, 20, 30, 40, 50, 60, 70, SO, 90, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000<br>
or more base pairs, which may be as a single contiguous sequence or from spaced apart<br>
regions within a DNA sequence. A substitution is the replacement of one or more base<br>
pairs, such as the replacement of up to 2, 3, 4, 5, 6, 7, 8, 9, 10, 20, 30, 40, 50, 60, 70, SO,<br>
90, 100, 200, 300, 400, 500, 600, 700, 800, 900, 1000 or more base pairs, which may be<br>
as a single contiguous sequence or from spaced apart regions within a DNA sequence. It<br>
is possible for a region to be modified by any two of insertion, deletion or substitution, or<br>
even all three.<br>
The last functional codon of either a RJEP2 gene or a FLP gene is the codon in the open<br>
reading frame of the gene that is furthest downstream from the promoter of the gene<br>
whose replacement by a stop codon -will lead to an unacceptable loss of multicopy<br>
stability of the plasmid, when determined by a test such as defined in Ginnery &amp;<br>
Hinchliffe (1989, Ciirr. Genet., 16, 21-25). It may be appropriate to modify the test<br>
defined by Chinery &amp; Hinchcliffe, for example to maintain exponential logarithmic<br>
growth over the desired number of generations, by introducing modifications to the<br>
inocula or sub-culturing regime. This can help to account for differences between the<br>
host strain under analysis and S. cere-visiae S150-2B used by Chinery &amp; Hinchcliffe,<br>
and/or to optimise the test for the individual characteristics of the plasmid(s) under assay,<br>
which can be determined by the identity of the insertion site within the small US-region<br>
of the 2j.im-like plasmid, and/or other differences in the 2|ini-like plasmid, such as the<br>
size and nature of the inserted sequences within the 2j.im-like plasmid and/or insertions<br>
elsewhere in the 2um-Iike plasmid. For yeast that do not grow in the non-selective<br>
medium (YPD, also designated YEPD) defined in Chinery &amp; Hinchliffe (1989, Cvrr.<br>
Genet., 16, 21-25) other appropriate non-selective media might be used. A suitable<br>
alternative non-selective medium typically permits exponential logarithmic growth over<br>
the desired number of generations. For example, sucrose or glucose might be used as<br>
alternative carbon sources. Plasmid stability may be defined as the percentage cells<br>
remaining prototrophic for the selectable marker after a defined number of generations.<br>
The number of generations will preferably be sufficient to show a difference between a<br>
control plasmid, such as p SACS 5 or pSACS 10, or to show comparable stability to such a<br>
control plasmid. The number of generations may be 1, 2, 3, 4, 5, 6, 7, 8, 9, 10,11, 12, 13,<br>
14, 15, 16, 17, 18, 19, 20, 25, 30, 35, 40, 45, 50, 60, 70, 80, 90, 100 or more. Higher<br>
numbers are preferred. The acceptable plasmid stability might be 1%, 2%, 3%, 4%, 5%,<br>
10%, 15%, 20%, 25%, 30%, 40%, 50°/o, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%,<br>
97%, 98%, 99%, 99.9% or substantially 100%. Higher percentages are preferred. The<br>
skilled person will appreciate that, even though a plasmid may have a stability less than<br>
(iO'J'(i when grown on non-selective media, that plasmid can siil] be of use \vhen cultured<br>
in selective media. For example plasmid pDE2711 as described in the examples is only<br>
10% stable, when the stability is determined accordingly to Example 1, but provides a 15-<br>
fold increase in recombinant Transferrin productivity in shake flask culture under<br>
selective growth conditions,<br>
Thus., disruption of the REP2 or FLP genes at any point downstream of the last functional<br>
codoD in either gene, by insertion of a po^imcleotide sequence insertion, deletion or<br>
substitution wili not lead to an unacceptable loss of multicopy' stability of the plasmid.<br>
We have surprisingly found that the REP2 gene of the 2pm plasmid can be disrupted<br>
after codon 59 and that the FLP gene of the 2um plasmid can be disrupted after codon<br>
344, each without leading to an unacceptable loss of multicopy stability of the plasmid.<br>
The last functional codon in equivalent genes hi other 2um-farnily plasmids can be<br>
determined routinely by modifying the relevant genes and detenniuing stability as •<br>
described above. Typicalty, therefore, modified plasmids of the present invention are.,<br>
stable, in the sense that the modifications made thereto do not lead to an unacceptable<br>
loss of multicopy stability of the plasmid.<br>
The REP 2 and FLP genes hi a 2um plasmid of the invention each have an inverted repeat<br>
adjacent to them. The inverted repeat can be identified because (when reversed) it<br>
matches the sequence of another inverted repeat within the same plasmid. By "adjacent"<br>
is meant that the FLP or REP2 gene and its inverted repeat are juxtaposed in such a<br>
manner that the inverted repeat occurs at the distal end of the gene, without airy<br>
inteiTening coding sequence, for example as seen hi the 2um plasmid. By "distal end' in<br>
this context we mean the end of the gene opposite to the end from which the promoter<br>
initiates its transcription. In a preferred embodiment, the distal end of the gene overlaps<br>
with the inverted repeat.<br>
hi a first preferred aspect of the invention, the polynucleotide sequence insertion, deletion<br>
and/or substitution occurs between the first base after the last functional codon of the<br>
REP2 gene and the last base before the FRT site in an inverted repeat adjacent to said<br>
gene, preferably between the first base of the inverted repeat and the last base -before the<br>
FRT site, even more preferably at a position after the translation tennination codon of the<br>
REP2 gene and before the last base before the FRT site.<br>
The term "between", in this context, includes the defined outer limits and so, for<br>
example, an insertion, deletion and/or substitution "between the first base after the last<br>
functional codon of the REP 2 gene and the last base before the FRT site" includes<br>
insertions, deletions and/or substitutions at the first base after the last functional codon of<br>
the REP2 gene and insertions, deletions and/or substitutions at the last base before the<br>
FRT site.<br>
In a second preferred aspect of the invention, the polynucleotide sequence insertion,<br>
deletion and/or substitution occurs between the first base after the last functional codon<br>
of the FLP gene and the last base before the FRT site in an inverted repeat adjacent to<br>
said gene, preferably between the first base of the inverted repeat and the last base before<br>
the FRT site, more preferably between the first base after the end of the FLP coding<br>
sequence and the last base before the FRT site, such as at the first base after the end of<br>
the FLP coding sequence. The polynucleotide seqxience insertion, deletion and/or<br>
substitution may occur between the last base after the end of FLP and the .Fjpl-site in the<br>
inverted repeat, but optionally not within the Fspl-site.<br>
In one embodiment, other than the polynucleotide sequence insertion, deletion and/or<br>
substitution, the FLP gene and/or the REP2 gene has the sequence of a FLP gene and/or a<br>
REP2 gene, respectively, derived from a naturally occurring 2um-famiry plasmid.<br>
The term "derived from" includes sequences having an identical sequence to the<br>
sequence from which they are derived. However, variants and fragments thereof, as<br>
defined above, are also included. For example, an FLP gene having a sequence derived<br>
from the FLP gene of the 2um plasmid may have a modified promoter or other regulatory<br>
sequence compared to that of the naturally occurring gene. Alternatrvefy, an FLP gene<br>
having a sequence derived from the FLP gene of the 2um plasmid may have a modified<br>
nucleotide sequence in the open reading frame which may encode the same protein as the<br>
* natural] y occurring eerie, or may encode a modified FLP protein. The same<br>
considerations apply to 1~IEP2 genes having a sequence derived from a particular source.<br>
A natural]}' occurring 2u,m-family plasmid is any plasniid having the features defined<br>
above as being essential features for a 2 jam-family plasniid. which plasmid is found to<br>
naturally exist in yeast, i.e. has not been recombiaantly modified to include heterologous<br>
sequence. Preferably the naturally occurring 2uxn--fami]y plasmid is selected from pSRl<br>
(Accession No. X02398), pSB3 (Accession No. X02608) or pSB4 as obtained from<br>
Zygosaccharomyces ronxii, pSBl or pSB2 (Accession No. NC_002055 or Ml 8274) both<br>
as obtained from Zygosaccharomyces bailli, pSMl (Accession No. NC_002054) as<br>
obtained from Zygosaccharomyces fermentati, pKDl (Accession No. X03961) as<br>
obtained from. Khryveromyoes drosophilarum, pPMl as obtained from Pichia<br>
membranaefaciens, or, most preferably, the 2p.ni plasmid (Accession No.. NC_OQ13 98 or<br>
J01347) as obtained from Saccharomyces cerevisiae. Accession numbers refer to<br>
deposits at the NCBI<br>
Preferably, other than the polynucleotide sequence insertion, deletion and/or substitution,<br>
the sequence of the inverted repeat adjacent to said FLP and/or KEP2 gene is derived<br>
from the sequence of the corresponding inverted repeat in the same naturally occurring<br>
2um-family plasmid as the sequence from which the gene is derived. Thus, for example,<br>
if the FLP gene is derived from the 2 urn plasmid as obtained from S. cerevisiae., then it is<br>
preferred that the inverted repeat adjacent to the FLP gene has a sequence derived from<br>
the inverted repeat that is adjacent to the FLP gene in the 2 urn plasniid as obtained from<br>
S cerevisiae. If the REP2 gene is derived from the 2um plasmid as obtained from S.<br>
cerevisiae, then it is preferred that the inverted repeat adjacent to the REP2 gene has a<br>
sequence derived from the inverted repeat that is adjacent to the REP2 gene in the 2j.im<br>
plasmid as obtained from S. cerevisiae.<br>
Where, in the first preferred aspect of the invention, other than the polynucleotide<br>
sequence insertion, deletion and/or substitution, the REP2 gene and the inverted repeat<br>
sequence have sequences derived from the corresponding regions of the 2jam plasmid as<br>
obtained from S. cerevisiae, then it is preferred that the polynucleotide sequence<br>
insertion, deletion and/or substitution occurs at a position between the first base of codon<br>
59 of the REP gene and the last base before the FRT site in the adjacent inverted repeat,<br>
more preferably at a position between the first base of the inverted repeat and the last<br>
base before the FRT site, even more preferably at a position after the translation<br>
termination codon of the REP2 gene and before the last base before the FRT site, such as<br>
at the first base after the end of the REP2 coding sequence.<br>
Where, other than the polynucleotide sequence insertion, deletion and/or substitution, the<br>
REP2 gene and the inverted repeat sequence have sequences derived from the<br>
corresponding regions of the 2um plasmid as obtained from S. cerevisiae, then in one<br>
embodiment, other than the polynucleotide sequence insertion, deletion and/or<br>
substitution, the sequence of the REP2 gene and the adjacent inverted repeat is as defined<br>
by SEQ ID N0:l or variant thereof. In SEQ ID N0:l, the first base of codon 59 of the<br>
^<br>
REP2 gene is represented by base number 175 and the last base before the FRT site is<br>
represented by base number 1216. The FRT sequence given here is the 55-base-pair<br>
sequence from Sadowski et al, 1986, pp7-10, Mechanisms of Yeast Recombination<br>
(Current Communications in Molecular Biology) CSHL. Ed. Klar, A. Strathern, J. N. In<br>
SEQ ID NO:1, the first base of the inverted repeat is represented by base number 887 and<br>
the first base after the translation termination codon of the REP2 gene is represented by<br>
base number 892.<br>
In an even more preferred embodiment of the first aspect of the invention, other than the<br>
polynucleotide sequence insertion, deletion and/or substitution, the REP2 gene and the<br>
inverted repeat sequence have sequences derived from the conesp ending regions of the<br>
2jjm plasmid as obtained from S. cerevisiae and, in the absence of the interruption the<br>
polynucleotide sequence insertion, deletion and/or substitution, comprise an Xcml site or<br>
an Fspl site within the inverted repeat and the polynucleotide sequence insertion, deletion<br>
and/or substitution occurs at Has Xcml site, or at the Fspl site. In SEQ ID N0:l, Has Xcml<br>
site is represented by base numbers 935-949 and the Fspl site is represented by base<br>
numbers 1172-1177.<br>
;re. in the- second preferred aspect of the invention, other than the polynucleotide<br>
sequence insertion, deletion and/or substitution, the FLP gene and the adjacent inverted<br>
repeat sequence have sequences derived from the corresponding regions of the 2am<br>
plasmid as obtained from S. cerevisiae., then it is preferred that the potyimcleotide<br>
sequence insertion, deletion and/or substitution occurs at a position, between the first base<br>
of codon 344 of the FLP gene and the last base before the FP,T site, more preferably<br>
between the first base of the inverted repeat and the last base before the FRT site, yet<br>
more preferably between the first base after the end of the FLP coding sequence and the<br>
last base before the FRT site, such as at the first base after the end of the FLP coding<br>
sequence. The Fspl site between the FLP gene and the FRT site can be avoided as an<br>
insertion site.<br>
Where, other than the polynucleotide sequence insertion, deletion and/or substitution, the<br>
FLP gene and the adjacent inverted repeat sequence have sequences derived from the<br>
corresponding regions of the 2um plasmid as obtained from S. cerevisiae, then in one<br>
embodiment other than the polynucleotide sequence insertion, deletion and/or<br>
substitution, the sequence of the FLP gene and the inverted repeat that follows the FLP<br>
gene is as denned by SEQ TD NO:2 or variant thereof. In SEQ ID NO:25 the first base of<br>
codon 344 of the FLP gene is represented by base number 1030 and the last base before<br>
the FRT site is represented by base number 1419, the first base of the inverted repeat is<br>
represented by base number 1090, and the first base after the end of the FLP coding<br>
sequence is represented by base number 1273.<br>
In an even more preferred embodiment of the second preferred aspect of the invention,<br>
other than the pofynucleotide sequence insertion, deletion and/or substitution, the FLP<br>
gene and the adjacent inverted repeat sequence have sequences derived from the<br>
corresponding regions of the 2um plasmid as obtained from S. cerevisiae and, in the<br>
absence of the potynucleotide sequence insertion, deletion and/or substitution, comprise<br>
an Hgal site or an Fspl sits "within the inverted repeat and the polynucleotide. sequence<br>
insertion, deletion and/or substitution occurs at the cut formed by the action of Hgal on<br>
the Hgal site (Hgal cuts outside the 5bp sequence that it recognises), or at the Fspl. In<br>
SEQ ID NO:2, the Hgal site is represented by base numbers 1262-1266 and the Fspl site<br>
is represented by base numbers 1375-1380.<br>
The skilled person will appreciate that the features of the plasinid defined by the first and<br>
second preferred aspects of the present invention are not mutually exclusive. Thus, a<br>
plasmid according to a third preferred aspect of the present invention may comprise<br>
polynucleotide sequence insertions, deletions and/or substitutions between the first bases<br>
after the last functional codons of both of the REP2 gene and the FLP gene and the last<br>
bases before the FRT sites in the inverted repeats adjacent to each of said genes, which<br>
polynucleotide sequence insertions, deletions and/or substitutions can be the same or<br>
different. For example, a plasmid according to a third aspect of the present invention<br>
may, other than the polynucleotide sequence insertions, deletions and/or substitutions,<br>
comprise the sequence of SEQ ID N0:l or variant thereof and the sequence of SEQ ID<br>
NO:2 or variant thereof, each comprising a polynucleotide sequence insertion, deletion<br>
and/or substitution at a-position as defined above for the first and second preferred<br>
aspects of the invention, respectively.<br>
The skilled person will appreciate that the features of the plasmid defined by the first,<br>
second and third preferred aspects of the present invention do not exclude the possibility<br>
of the plasmid also having other sequence modifications. Thus, for example, a 2umfamily<br>
plasmid of the first, second and third preferred aspects of the present invention<br>
may additionally comprise a polynucleotide sequence insertion, deletion and/or<br>
substitution which is not at a position as defined above. Accordingly, the plasmid may<br>
additionally carry transgenes at a site other than the insertion sites of the invention.<br>
Alternative insertion sites in 2um plasmids are known in the art, but do not provide the<br>
advantages of using the insertion sites defined by the present invention. Nevertheless,<br>
plasrnids which already include a polynucleotide sequence insertion, deletion and/or<br>
substitution at a site known in the art can be further modified by making one or more<br>
further modifications at one or more of the sites defined by the first, second and third<br>
preferred aspects of the present invention. The skilled person will appreciate that, as<br>
discussed in the introduction to this application, there are considerable technical<br>
limitations placed on the insertion of Transgenes at sites of -urn-family plasmids o~ther<br>
than as defined by the first and second aspects of the invention.<br>
Typical modified 2,um plasmids laiown in the art include those described in Rose &amp;<br>
Broach (1990, Methods En=ymol, 185, 234-279), such as plasmids pCV19, pCV20,<br>
CVneo, which utilise aa insertion at EcoRl in FLP, plasmids pCV21. pGT41 and pYE<br>
which utilise EcoPJ in D as the insertion site, plasniid pHKB52 which utilises Pstl in D<br>
as the insertion site, plasmid pJDB248 which utilises an insertion al Pstl in D and JEc-oFJ<br>
in D. plasmid pJDB219 in which Pstl in D and EcoPJ in FLP are used as insertion sites,<br>
plasmid Gl 8. plasmid pABIS which utilises an insertion at Clal in FLP, plasmids pGT39<br>
and pA3, plasmids pYTl 1, pYT14 and pYTl 1-LEU which use Pstl in D as the insertion<br>
site, and plasniid PTY39 which uses Eco'KL in FLP as the insertion site. Other 2.pm<br>
plasmids include pSAC3, pSACSUl, pSAC3U2, pSACSOO, pSAC310, pSACBCl,<br>
pSACSPLl, pSAC3SL4, and pSACSSCl are described in EP 0 286 424 and Chrnery &amp;<br>
HincHiffe (1989, Cvrr. Genet., 16, 21-25) which also described Pstl, EagI or Sna&amp;I as...<br>
appropriate 2um insertion sites. Further 2p:m plasmids include pAYE255. pAYE316,<br>
pAYE443, pA^T522 (Kerry-Williams et al, 1998, Yeast, 14, 161-169), pDB2244 (TWO<br>
00/44772) and pAYE329 (Sleep et al, 2001, Yeast, IS, 403-421).<br>
In one preferred embodiment, a 2fim-lrke plasmid as defined by the first, second and<br>
third preferred aspects of the present invention additionally comprises a po^TtucIeotide<br>
sequence insertion, deletion and/or substitution which occurs within an untranscribed<br>
region around the ARS sequence. For example, in the 2um plasmid obtained from S.<br>
cerevisiae, the untranscribed region around the ARS sequence, extends from end of th_e D<br>
gene to the beginning of ARS sequence. Insertion into SnaBl (near the origin of<br>
replication sequence ARS) is described in Chinery &amp; Hincliliffe, 1989, Curr. Genet, 16,<br>
21-25. The skilled person will appreciate that an additional polynucleotide sequence<br>
insertion., deletion and/or substitution can also occur within the imtranscribed region at<br>
neighbouring positions to the SnaBl site described by Chinery &amp; Hhchliffe.<br>
A plasmid according to any of the first, second or third aspects of the present invention<br>
may be a plasmid capable of autonomous replication in yeast, such as a member of the<br>
Saccharomyces, Khnweromyces, Zygosaccharomyces, or Pichia genus, such<br>
Saccharomyces cerevisiae, Saccharomyces carlsbergensis,, Kluyveromyces lactis, Pichia<br>
pastoris and Pichia membranaefaciens, Zygosaccharomyces roiixii, Zygosaccharomyces<br>
bailii, Zygosaccharomyces fermentati, or Kluyveromyces drosphilarum. S. cerevisiae and<br>
5*. carlsbergensis are thought to provide a suitable host cell for the autonomous<br>
replication of all known 2u-m plasmids.<br>
In a preferred embodiment, the, or at least one, polynucleotide sequence insertion,<br>
deletion and/or substitution included in a 2um-family plasmid of the invention is a<br>
polynucleotide sequence insertion. Any polynucleotide sequence insertion may be used,<br>
so long as it is not unacceptably detrimental to the stability of the plasmid, by which we<br>
mean that the plasmid is at least 1%, 2%, 3%, 4%, 5%, 10%, 15%, 20%, 25%, 30%, 4-0%<br>
50%, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%, 99.9% or<br>
substantially 100% stable on non-selective media such as YEPD media compared to the<br>
unmodified plasmid, the latter of which is assigned a stability of 100%. Preferably, the<br>
above mentioned level of stability is seen after separately culturing yeast cells comprising<br>
the modified and unmodified plasmids in a culture medium for one, two, three, four, five,<br>
six, seven, eight, nine ten, 11, 12, 13, 14, 15, 16, 17, 18, 19 20,25, 30, 35, 40, 45, 50, 60,<br>
70, 80, 90,100 or more generations.<br>
Where the plasmid comprises a selectable marker, higher levels of stability can. be<br>
obtained when transformants are grown under selective conditions (e.g. in minimal<br>
medium), since the medium can place a selective pressure on the host to retain the<br>
plasmid.<br>
Stability in non-selective and selective (e.g. minimal) media can be determined using the<br>
methods set forth above. Stability in selective media can be demonstrated by the<br>
observation that the plasmids can be used to transform }^east to prototrophy.<br>
Typically, the polynucleotide sequence insertion will be at least 4, 6, 8, 10,20, 30, 40, 50,<br>
60, 70, 80, 90, 100, 150,200, 250, 300, 350, 400, 450, 500 or more base pairs hi length.<br>
Usually, the polynucleotide sequence insertion will be up to 1kb, 2kb, 3kb, 4kb, 5kb, 6kb,<br>
7kb: 8kb. 9kb, 10kb or more in length. The sldlled person will appreciate that the 2 urn<br>
plasmid of the present invention may comprise multiple polynuclecnide sequence<br>
insertions at different sites "within the plasmid. Typically, the total length of<br>
pohoiucleoticle sequence insertions is no more than 5kb, 10kb. 15kb. 20kb, 25kb or 30kb<br>
although .greater total length insertion ma)' be possible.<br>
The polynucleotide sequence may or may not be a linker sequence used to introduce new<br>
restriction sites. For example a synthetic linker may or may not be introduced at the Fspl<br>
site after the FLP gene, such as to introduce a further restriction site (e.g. BamKI).<br>
The polynucleotide sequence insertion may contain a transcribed region or ma)' contain<br>
no transcribed region. A transcribed region may encode an open reading frame, or may<br>
be non-coding. The polynucleotide sequence insertion may contain both, transcribed and<br>
non-transcribed regions.<br>
A transcribed region is a region of DNA that can be transcribed by RNA pobynierase,<br>
typically yeast RNA porymerase. A transcribed region can encode a functional RNA<br>
molecule, such as ribosomal or transfer RNA or an RNA molecule that can function as an<br>
aniisense or RNA interference ("RNAi") molecule. Alternatively a transcribed region<br>
can encode a messenger RNA molecule (rnRNA), which inRNA can contain an open<br>
reading frame (ORF) which can be translated 271 vivo to produce a protein. The term<br>
"protein" as used herein includes all natural and non-natural proteins, porypeptides and<br>
peptides. Preferably, the ORF encodes a heterologous protein. B)' "heterologous<br>
protein" we mean a protein that is not naturally encoded by a 2|jm-family plasmid (i.e. a<br>
c:non- 2j.m&gt;famiry plasmid protein"). For convenience the terms "heterologous protein"<br>
and "non- 2um-fainily plasmid protein" are used synonymous!}7 throughout this<br>
application. Preferably, therefore, the heterologous protein is not a FLP, REP15 REP2, or<br>
a RAFfD protein as encoded by any one of pSRl, pSB3 or pSB4 as obtained from Z.<br>
roiKii, pSBl or pSB2 both as obtained from Z. bailli, pSMl as obtained from Z<br>
fermentati, pKDl as obtained from X. drosophilarum, pPMl as obtained from P.<br>
membranaefaciens and the 2um plasmid as obtained from S. cerevisiae.<br>
Where the polynucleotide sequence insertion encodes an open reading frame, then it may<br>
additionally comprise some polynucleotide sequence that does not encode an open<br>
reading frame (termed "non-coding region").<br>
Non-coding region in the polynucleotide sequence insertion may contain one or more<br>
regulatory sequences, operatively linked to the open reading frame, which allow for the<br>
transcription of the open reading frame and/or translation of the resultant transcript.<br>
The term "regulatory sequence" refers to a sequence that modulates (i.e., promotes or<br>
reduces) the expression (i.e., the transcription and/or translation) of an open reading<br>
frame to which it is operably linked. Regulatory regions typically include promoters,<br>
terminators, ribosome binding sites and the like. The skilled person will appreciate that<br>
the choice of regulatory region will depend upon the intended expression system. For<br>
example, promoters may be constitutive or inducible and may be cell- or tissue-type<br>
specific or non-specific,<br>
Where the expression system is yeast, such as Saccharomyces cerevisiae, suitable<br>
promoters for S. cerevisiae include those associated with the PGK1 gene, GAL1 or<br>
GAL10 genes, TEF1, TEF2, PYK1, PklAl, CYC1, PHO5, TRP1, ADH2, ADH2, the genes<br>
for glyceraldehyde-3-phosphate dehydrogenase, hexolcinase, pyruvate decarboxylase,<br>
phosphofructokinase, triose phosphate isomerase, phosphoglucose isomerase,<br>
glucokinase, oc-mating factor pherornone, a-mating factor pheromone, the PRB1<br>
promoter, the PRA1 promoter,, the GPD1 promoter, and hybrid promoters involving<br>
hybrids of parts of 5' regulatory regions with parts of 5' regulatory regions of other<br>
promoters or with upstream activation sites (e.g. the promoter of EP-A-258 067).<br>
Suitable transcription termination signals are well known in the art. Where the host cell<br>
is eukaryotic, the transcription termination signal is preferably derived from the 3'<br>
flanking sequence of a eukaryotic gene, which contains proper signals for transcription<br>
termination and polyadenylation. Suitable 3' flanking sequences may, for example, be<br>
those of the gene naturally linked to the expression control sequence used, i.e. may<br>
correspond to the promoter. Alternatively, they may be different. In that case, and where<br>
be host is a yeast, preferably S. fji-cvisiac, then the termination signal of the 5.<br>
ewisi-acADHJ.ADH2, CYCJ, or PGICJ genes are preferred.<br>
It may be beneficial for the promoter and open reading frame of the heterologous gene,<br>
such as the those of the chaperone PDIL to be flanked by transcription temiination<br>
sequences so that the transcription termination sequences are located both upstream and<br>
downstream of the promoter and open reading frame, hi order to prevent transcriptional<br>
read-through into neighbouring genes, such as 2 urn genes, and vice versa.<br>
hi one embodiment, the favoured regulator)' sequences hi yeast, such as Saccharomyces<br>
cerevisiae. include: a yeast promoter (e.g. the Saccharomyces cerevisiae PRB1<br>
promoter), as taught in EP 431 880; and a transcription terminator, preferably the<br>
terminator from Saccharomyces ADH1 , as taught in EP 60 057.<br>
It may be beneficial for the non-coding region to incorporate more than one DNA<br>
sequence encoding a translational stop codon. such as UAA. UAG or UGA, hi order to<br>
minimise traaslational read-through and thus avoid the production of elongated, nonaatural<br>
fusion proteins. The translation stop codon UAA is preferred. Preferably, at least<br>
two translation stop codons are incorporated.<br>
The term "operabty linked" includes within its meaning that a regulatory sequence is<br>
positioned within any non-coding region such that it forms a relationship with an open<br>
reading frame that permits the regulatory region to exert an effect on the open reading;<br>
frame, in its intended manner. Thus a regulatory region "operably linked" to an open<br>
reading frame is positioned in such a way that the regulatory region is able to influence<br>
transcription and/or translation of the open reading frame in the intended manner, under<br>
conditions compatible with the regulator)' sequence.<br>
Where the potynucleotide sequence insertion as defined by the first second or third<br>
aspects of the present invention includes an open reading frame that encodes a protein<br>
then it ma}' be advantageous for the encoded protein to be secreted. In that case, a<br>
sequence encoding a secretion leader sequence may be included in the open reading<br>
frame.<br>
For production of proteins in eukaryotic species such as the yeasts Saccharomyces<br>
cerevisiae, Zygosaccharomyces species, Kinder omyces lactis and Pichia pastoris,<br>
laiown leader sequences include those from the S. cerevisiae acid phosphatase protein<br>
(Pho5p) (see EP 366 400), the invertase protein (Suc2p) (see Smith et al. (1985) Science,<br>
229, 1219-1224) and heat-shock protein-150 (Hspl50p) (see WO 95/33833).<br>
Additionally, leader sequences from the S. cerevisiae mating factor alpha-1 protein<br>
(MFoc-1) and from the human lysozyme and human serum albumin (HSA) protein have<br>
been used, the latter having been used especially, although not exclusively, for secreting<br>
human, albumin. WO 90/01063 discloses a fusion of the MFa-1 and HSA leader<br>
sequences, which advantageously reduces the production of a contarninating fragment of<br>
human albumin relative to the use of the MFcc-1 leader sequence. In addition, the natural<br>
transferrin leader sequence may be used to direct secretion of transferrin and other<br>
heterologous proteins.<br>
Alternatively, the encoded protein may be intracellular.<br>
hi one preferred embodiment, at least one polynucleotide sequence insertion as defined<br>
by the first, second or third aspects of the present invention includes an open reading<br>
frame comprising a sequence that encodes a yeast protein, hi another preferred<br>
embodiment, at least one polynucleotide sequence insertion as defined by the first,<br>
second or third aspects of the present invention includes an open reading frame<br>
comprising a sequence that encodes a yeast protein from tke same host from which the<br>
2um-lilce plasmid is derived.<br>
hi another preferred embodiment, at least one polynucleotide sequence insertion as<br>
defined by the first, second or third aspects of the present invention includes an open<br>
reading frame comprising a sequence that encodes a protein involved in protein folding,<br>
or which has chaperone activity or is involved hi the unfolded protein response (Stanford<br>
Genome Database (SGD), htrp:://db.yeastgenome.org). Preferred proteins may be<br>
"selected from protein encoded by ANAL CCT2, CCT3, CCT4, CCT5, CCT6.. CCT7,<br>
CCTS, CtfSJ, CPR3, CPR6: EROJ, EUG1, FMO.L HCHL HSPKL HSPJ2.. KSPJ04,<br>
HSP26, HSP30: HSP42: ESP60, PISP7S, HSP82, JEM1, MDJ1, MDJ2, MPDL MPD2,<br>
PDnj'PD],ABCl,APJLATPn,ATP12,.BTTl, CDC57, CNSJ: CPR6; CPR7,HSC82,<br>
KAX2, LP1S1, MGEL MRS11, NOBJ, ECM10, SSAJ, SSA2, SSA3, SSA4, SSCJ, SSE2&gt;<br>
SILL. SLSJ, OJtMJ, UBJ4, ORM2, PEfJ, PTC2, PSE1 and EAC1 or a truncated<br>
introoless HA CJ (Valkonen el al. 2003, Applied Environ. Micro. 69, 2065).<br>
A preferred protein involved in protein foldings or protein with chaperone activit3r or a<br>
protein involved in the unfolded protein response may be:<br>
• a heat shock protein, such as a protein that is a member of the hsp70 family of<br>
proteins (including Kar2p, SSA and SSB proteins, for example proteins encoded<br>
by SSA1, SSA2, SSA3, SSA4, SSS1 and SSB2), a protein that is a member of tie<br>
HSP90-famiry, or a protein that is a member of the HSP40-family or proteins<br>
involved in their modulation (e.g. SiJlp), including DNA-J and DNA-J-like<br>
proteins (e.g. Jemlp, Mdj2p):<br>
• a protein that is a member of the IcaiyopherhVimportin famify of proteins, such as<br>
the alpha or beta families of Icaiyopherin/importin proteins, for example the<br>
Icaryopherin beta protein encoded byPSEJ;<br>
• a protein that is a member of the ORMDL farnity described by Hjelmqvist ef al,<br>
2002, Genome Biology. 3(6), research0027.1-0027.16, such as Orm2p.<br>
a protein that is naturally located in the endoplasmic reticulum or elsewhere in the<br>
secretory patlrwa}', such as the golgi. For example, a protein that naturalty acts in<br>
the lumen of the endoplasmic reticulum (ER), particularly in secretory cells, such<br>
asPDJ<br>
• a protein that is a transmembraiie protein anchored in the ER, such as a member<br>
of the ORMDL family described by Hjelmqvist et al, 2002, supra, (for example,<br>
Onn2p);<br>
• a protein that acts in the cytosol, such as the hsp70 proteins, including SSA and<br>
SSB proteins, for example proteins encoded by SSA2, SSA2, SSA3, SSA4, SSB1<br>
andSSB2;<br>
• a protein that acts in the nucleus, the nuclear envelope and/or the cytoplasm, such<br>
' as Pselp;<br>
• a protein that is essential to the viability of the cell, such as PDI or an essential<br>
karyopherin protein, such as Pselp;<br>
• a protein that is involved in sulphydryl oxidation or disulphide bond formation,<br>
breakage or isomerization, or a protein that catalyses thiolidisulphide interchange<br>
reactions in proteins, particularly during the biosynthesis of secretory and cell<br>
surface proteins, such as protein disulphide isomerases (e.g. Pdilp, Mpdlp),<br>
homologues (e.g. Euglp) and/or related proteins (e.g. Mpd2p, Fmolp, Erolp);<br>
• a protein that is involved in protein synthesis, assembly or folding, such as PDI<br>
and Ssalp;<br>
• a protein that binds preferentially or exclusively to unfolded, rather than mature<br>
protein, such as the hsp70 proteins, including SSA and SSB proteins, for example<br>
proteins encoded by SSA1, SSA2, SSA3, SSA4, SSB1 andSSB2;<br>
• a protein that prevents aggregation of precursor proteins in the cytosoi, such as the<br>
hsp70 proteins, including SSA and SSB proteins, for example proteins encoded<br>
by SSA2, SSA2, SSA3, SSA4, SSB1 and SSB2;<br>
• a protein that binds to and stabilises damaged proteins, for example Ssalp;<br>
•d protein that is involved in the unfolded protein response or provides for<br>
increased resistance to agents (such as tunicanrycin and dittoothreitol) that induce<br>
the unfolded protein response., such as a member of the ORMDL family described<br>
by Pljehnqvist el al, 2002., supra (for example, Orm2p) or a protein involved in<br>
the response to stress (e.g. Ubi4p);<br>
a protein that is a co-chaperone and/or a protein indirectly involved in protein<br>
folding and/or the unfolded protein response (e.g. hsp!04p, Mdjlp);<br>
a protein that is involved in the nucleocytoplasmic transport of macroinolecules.<br>
suchasPselp;<br>
a protein that mediates the transport of macromolecules across the nuclear<br>
membrane by recognising nuclear location sequences and nuclear export<br>
sequences and interacting with the nuclear pore complex, such as Pselp;<br>
a protein that is able to reactivate ribonuclease activity against RNA of scrambled<br>
ribonuclease as described in as described in EP 0 746 611 and Hillson et al, 1984,<br>
Methods Ercymol, 107, 281-292, such as PDI;<br>
a protein that has an acidic pi (for example, 4.0-4.5), such as PDI;<br>
a protein that is a member of the Hsp70 family, and preferably possesses an Nterminal<br>
ATP-binding domain and a C-terminal peptide-binding domain, such as<br>
Ssalp.<br>
a protein that is a peptidyl-prolyl cis-trans isomerases (e.g. CprSp, Cpr6p):<br>
a protein that is a liomologues of known chaperones (e.g. HsplOp);<br>
a protein that is a mitochondria! chapeione (e.g CprSp);<br>
• a protein that is a cytoplasrnic or nuclear chaperone (e.g Cnslp);<br>
• a protein that is a membrane-bound chaperone (e.g, Orm2p, Fmolp);<br>
• a protein that has chaperone activator activity or chaperone regulatory activity<br>
(e.g. Alialp, Haclp, Hchlp);<br>
• a protein that transiently binds to polypeptides in their immature form to cause<br>
proper folding transportation and/or secretion, including proteins required for<br>
efficient translocation into the endoplasmic reticulum (e.g. Lhslp) or their site of<br>
action within the cell (e.g. Pselp);<br>
• a protein mat is a involved in protein complex assembly and/or ribosome<br>
assembly (e.g. Atpl Ip, Pselp, Noblp);<br>
• a protein of the chaperonin T-complex (e.g. Cct2p); or<br>
• a protein of the prefoldin complex (e.g. Pfdlp).<br>
One preferred chaperone is protein disulphide isomerase (PDI) or a fragment or variant<br>
thereof having an equivalent ability to catalyse the formation of disulphide bonds within<br>
the lumen of the endoplasmic reticulum (ER). By "PDI" we include any protein having<br>
the ability to reactivate the ribonuclease activity against RNA of scrambled ribonuclease<br>
as described in EP 0 746 611 and Hillson et al, 1984, Methods Enzymol. 107, 281-292.<br>
Protein disulphide isomerase is an enzyme which typically catalyzes thiolrdisulphide<br>
interchange reactions, and is a major resident protein component of the E.R. lumen in<br>
secretory cells. A body of evidence suggests that it plays a role in secretory protein<br>
biosynthesis (Freedman, 1984, Trends Biochem. Set., 9, 438-41) and this is supported by<br>
direct cross-linking studies in situ (Roth and Pierce, 1987, Biochemistry, 26, 4179-82).<br>
The finding that microsomal membranes deficient in PDI show a specific defect in<br>
cotranslational protein disulphkl? fonnation I'Bulleid and i-'reedman, I98S, A'ahire. 335,<br>
649-51) implies that the enzyme functions as a catal.vsi of native disulpliide bond<br>
formation duiiiig the biosynthesis of secretory and cell surface proteins. Tins role is<br>
consistent with what is known of the enzyme's catalytic properties 177 vitro: it catatyz.es<br>
tliiol: disulphide interchange reactions leadhig to net protein disulphide formation,<br>
breakage or isomerization, and can typically catalyze protein folding and the formation of<br>
native disulphide bonds in a wide variety of reduced, unfolded protein substrates<br>
(Treedman et al., 19S9, Biochem. Soc. Symp., 55. 167-192). PDI also functions as a<br>
diaperone since mutant PDI lacking isomerase activity accelerates protein folding<br>
fHayano et al, 1995, FEBS Letters, 377, 505-511). Recently, sulphyoryl oxidation, not<br>
disulphide isornerisation was reported to be the principal function of Protein Disulphide<br>
Isomerase in S. cerevisiae (Solovyov et al, 2004, J. Biol. Chem., 279 (33) 34095-34100).<br>
The DNA and arnino acid sequence of the enzyme is known for several species (Scherens<br>
et al, 1991, Yeast, 1, 185-193; Farquhar et al 1991, Gene, 108, 81-89; EP074661;<br>
EP0293793; EP0509S41) and there is increasing information on the mechanism of action<br>
of the enzyme purified to homogeneity from mammalian liver (Creighlon et al, 1980., J.<br>
Mol. Biol, 142, 43-62: Fieedman et al, 1988, Biochem, Soc. Trans., 16, 96-9; Gilbert,<br>
1989,Biochemistry, 28, 7298-7305; Lundstrom and Holmgren, 1990, J. Biol. Chem., 265,<br>
9114-9120: Hawkins and Freedman, 1990, Biochem. J., 275, 335-339). Of the many<br>
protein factors currently implicated as mediators of protein folding, assembly and<br>
trans-location in the cell (Rothrnan, 1989, Cell, 59, 591-601), PDI has a well-defined<br>
catalytic activity.<br>
The deletion or inactivation of the endogenous PDI gene in a host results in the production<br>
of an inviable host. In other words, the endogenous PDI gene is an "essential" gene.<br>
PDI is readily isolated from mammalian tissues and the homogeneous enzyme is a<br>
homodirner (2x57 kD) with characteristically acidic pi (4.0-4.5) (Hillson et al. 1984,<br>
Methods ErirymoL, 107, 281-292). The enzyme has also been purified from wheat and<br>
from the alga Chlamydomonas reinhardii (Kaska et al, 1990, Biochem. J., 268, 63-68),<br>
rat (Edman et al, 1985, Nature, 317., 267-270), bovine (Yamauchi et al, 19S7, Biochem.<br>
Biophys. Res. Comm., 146, 1485-1492), human (Piluajaniemi et al, 1987, EhfBO J., 6,<br>
643-9), yeast (Scherens et al, supra; Farquhar ef al, supra] and chick (Parldconen et al,<br>
1988, Biochem. J., 256, 1005-1011). The proteins from these vertebrate species show a<br>
high degree of sequence conservation throughout and all show several overall features<br>
first noted in the rat PDI sequence (Edman et al., 1985, op. cit),<br>
A yeast protein disulphide isomerase precursor, PDI1, can be found as Genbank<br>
accession no. CAA42373 or BAA00723, It has the following sequence of 522 amino<br>
acids:<br>
1 mkfsagavls wsslllassv faqqeavape dsawklatd sfneyiqshd Ivlaeffapw<br>
61 cghcknmape yvkaaetlve knitlagidc tenqdlcmeh nipgfpslki fknsdvnnsi<br>
121 dyegprtaea ivqfmikqsq pavawadlp aylanetfvt pvivqsgkid adfnatfysm<br>
181 ankhfndydf vsaenadddf klsiylpsam depwyngkk adiadadvfe kwlgvealpy<br>
241 fgeidgsvfa qyvesglplg ylfyndeeel eeykplftel akknrglmnf vsidarkfgr<br>
301 hagnlnmkeq fplfaihdmt edlkyglpql seeafdelsd kivleskaie slvkdflkgd<br>
361 aspivksqei fenqdssvfq Ivgknhdeiv ndpkkdvlvl yyapwcghck rlaptyqela<br>
421 dtyanatsdv liakldhten dvrgvviegy ptivlypggk ksesvvyqgs rsldslfdfi<br>
481 kenghfdvdg kalyeeaqek aaeeadadae ladeedaihd el<br>
An alternative PDI sequence can be found as Genbanlc accession no. CAA38402. It has<br>
the following sequence of 530 amino acids<br>
1 mkfsagavls wsslllassv faqqeavape dsawklatd sfneyi'qshd Ivlaeffapw<br>
61 cghcknmape yvkaaetlve knitlaqidc tenqdlcmeh nipgfpslki fknrdvnnsi<br>
121 dyegprtaea ivqfmikqsq pavawadlp aylanetfvt pvivqsgkid adfnatfysm<br>
161 ankhfndydf vsaenadddf klsiylpsam depvvyngkk adiadadvfe kwlqvealpy<br>
241 fgeidgsvfa qyvesglplg ylfyndeeel eeykplftel akknrglmnf vsidarkfgr<br>
301 hagnlnmkeq fplfaihdmt edlkyglpql seeafdelsd kivleskaie slvkdflkgd<br>
361 aspivksqei fenqdssvfq Ivgknhdeiv ndpkkdvlvl yyapwcghck rlaptyqela<br>
421 dtyanatsdv liakldhten dvrgvviegy ptivlypggk ksesvvyqgs rsldslfdfi<br>
481 kenghfdvdg kalyeeaqek aaeeaeadae aeadadaela deedaihdel<br>
Variants and fragments of the above PDI sequences, and variants of other naturally<br>
occurring PDI sequences are also included in the present invention. A "variant", in the<br>
context of PDI, refers to a protein wherein at one or more positions there have been amino<br>
acid insertions, deletions, or substitutions, either conservative or non-conservative, provided<br>
"*tliai such changes resuli in a protein whose basic properties, ior example enzymatic aciivity<br>
(type of and specific activity;, thennostability, activity in a certain pH-range (pH-stability')<br>
have, not significant!}7 been changed. "Significant]}'" in tliis contort' means that one skilled in<br>
the art would say that the properties of the variant may still be different but would not be<br>
unobvious over Hie ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val. Be, Leu. Ala, Met;<br>
Asp, Glu; Asn, Gin: Ser, Tbr. Gly, Ala; Lys, Arg, His; and Phe, Tyr, Tip. Preferred<br>
conservative substitutions include Glys Ala; Val, He, Leu; Asp, Glu: Asn, Gin: Ser, Thr;<br>
Lys, Arg: and Phe. Tyr.<br>
A "variaat" typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably<br>
at least 80%, more preferably at least 90%, even more preferably7 at least 95%. yet moie<br>
preferably at least 99%, most preferably at least 99.5% sequence identity to the polypeptide<br>
from which, it is derived.<br>
The percent sequence identity between two polypeptides may be determined using<br>
suitable computer programs, as discussed below. Such variants may be natural or made<br>
using the methods of protein engineering and site-directed mutagenesis as are well known in<br>
the ait.<br>
A "fragment", in the context of PDI, refers to a protern wherein at one or more positions<br>
there have been deletions. Thus the fragment may comprise at most 5, 10, 20, 30. 40 or<br>
50%, typically up to 60%, more typically up to 70%. preferably up to 80%. more preferabhy<br>
up to 90%, even more preferably up to 95%, yet more preferably up to 99% of the complete<br>
sequence of the full mature PDI protein. Particularly preferred fragments of PDI protein<br>
comprise one or more whole domains of the desired protein.<br>
A fragment or variant of PDI may be a protein that, when expressed recombinantty in a<br>
host cell, such as S. cerevisiae, can complement the deletion of the endogenously<br>
encoded PDI gene in the host cell and may, for example, be a natural!}' occmrizig<br>
homolog of PDI, such as a hornolog encoded by another organism, sucli as another yeast<br>
or other fungi, or another eukaryote such as a human or other vertebrate, or animal or by<br>
a plant.<br>
Another preferred chaperone is SSA1 or a fragment or variant thereof having an<br>
equivalent chaperone-like activity. SSA1, also known as YG100, is located on<br>
chromosome I of the S. cerevisiae genome and is 1.93-kbp in size.<br>
One published protein sequence of SSA1 is as follows:<br>
MSKAVGIDLGTTYSCVAHFANDRVDIIANDQGNRTTPSFVAFTDTERLIGDAAKNQAAMN<br>
PSNTVFDAKRLIGRNFNDPEVQADMKHFPFKLIDVDGKPQIQVEFKGETKNFTPEQISSM<br>
VLGKMKETAESYLGAKVNDAWTVPAYFNDSQRQATKDAGTIAGLNVLRIINEPTAMIA<br>
YGLDPCKGKEEHVLIFDLGGGTFDVSLLFIEDGIFEVKATAGDTHLGGEDFDNRLVNHFIQ<br>
EFKRKNKKDLSTNQRALRRLRTACERAKRTLSSSAQTSVEIDSLFEGIDFYTSITRRRFE<br>
ELCADLFRSTLDPVEKVLRDAKLDKSQVDEIVLVGGSTRIPKVQKLVTDYFNGKEPNRSI<br>
MPDEAVAYGAAVQAAILTGDESSKTQDLLLLDVAPLSLGIETAGGVMTKLIPRNSTISTK<br>
KFEIFSTYADNQPGVLIQVFEGERAKTKDNNLLGKFELSGIPPAPRGVPQIEVTFDVDSN<br>
GILNVSAVEKGTGKSNKITITNDKGRLSKEDIEKMVAEAEKFKEEDEKESQRIASKNQLE<br>
SIAYSLKNTISEAGDKLEQADKDTVTKKAEETISWLDSNTTASKEEFDDKLKELQDIANP<br>
IMSKLYQAGGAPGGAAGGAPGGFPGGAPPAPEAEGPTVEEVD<br>
A published coding sequence for SSA1 is as follows, although it will be appreciated that<br>
the sequence can be modified by degenerate substitutions to obtain alternative nucleotide<br>
sequences which encode an identical protein product:<br>
ATGTCAAAAGCTGTCGGTATTGATTTAGGTACAACATACTCGTGTGTTGCTCACTTTGCT<br>
AATGATCGTGTGGACATTATTGCCAACGATCAAGGTAACAGAACCACTCCATCTTTTGTC<br>
GCTTTCACTGACACTGAAAGATTGATTGGTGATGCTGCTAAGAATCAAGCTGCTATGAAT<br>
CCTTCGAATACCGTTTTCGACGCTAAGCGTTTGATCGGTAGAAACTTCAACGACCCAGAA<br>
GTGCAGGCrGACATGAAGCACTTCCCATTCAAGTTGATCGATGTTGACGGTAAGCCTCAA<br>
ATTCAAGTTGAATTTAAGGGTGAAACCAAGAACTTTACCCCAGAACAAATCTCCTCCATG<br>
GTCTTGGGTAAGATGAAGGAAACTGCCGAATCTTACTTGGGAGCCAAGGTCAATGACGCT<br>
GTCGTCACTGTCCCAGCTTACTTCAACGATTCTCAAAGACAAGCTACCAAGGATGCTGGT<br>
ACCATTGCTGGTTTGAATGTCTTGCGTATTATTAACGAACCTACCGCCGCTGCCATTGCT<br>
TACGGTTTGGACAAGAAGGGTAAGGAAGAACACGTCTTGATTTTCGACTTGGGTGGTGGT<br>
ACTTTCGATGTCTCTTTGTTGTTCATTGAAGACGGTATCTTTGAAGTTAAGGCCACCGCT<br>
GGTGACACCCATTTGGGTGGTGAAGATTTTGACAACAGATTGGTCAACCACTTCATCCAA<br>
TCT\ACCAiC^<br>
ATT'GACTCTTTGTTCGMGGTATCGATTTCTACACTTCCATCACCAGAGCCAGATT'CGAA<br>
GAATTGTGTGCTGACTTGTTCAGATCTACTTTGGACCCAGTTGAAAAGGTCTTGAGAGAT<br>
GCTAAATTGGACAAATCTCAAGTCGATGAAATTGTCTTGGTCGGTGGTTCTACCAGAATT<br>
CCAAAGGTCCAAAAATTGGTCACTGACTACTTCAACGGTAA.GGAACCAAACAGATCTATC<br>
AACCCAGATGAAGCTGTTGCTTACGGTGCTGCTGTTCAAGCTGCTATTrTGACTGGTGAC<br>
GAATCTTCCAAGACTCAAGATCTATTGTTGTTGGATGTCGCTCCATTArCCTTGGGTA.TT<br>
GA&gt;ACrGCTGGTGGTGTCATGACCAAGTTGA-TTCCAAGAAACTCTACCATrTCAACAAAG<br>
AA.GTTCGAGATCTTTTCCACTTATGCTGATAACCAACCAGGTGTCTTGATTCAAGTCTTT<br>
GAAGGrGAAAGAGCCAAGACTAAGGACAACAACTTGT7GGG?AAGTTCGAATTGAGTGGT<br>
ATTCCACCAGCTCCAAGAGGTGTCCCACAAATTGAAGTCACTTTCGATGTCGACTCTAAC<br>
GGrATTTTGAArGTTTCCGCCGTCGAAAAGGGTACTGGTAAGTCTAACAAGATCACTATT<br>
ACCAACGACAAGGGTAGATTGrCCAAGGAAGATATCGAAAAGATGGTTGCTGAAGCCGAA<br>
AAATTCAAGGAAGAAGATGAZaAGGAATCTCAAAGAATTSCTTCCAAGAACCAATTGGAA<br>
TCCATTGCTTACTCTTTGAAGAACACCATTTCTGAAGCTGGTGACAAATTGGAACAAGCT<br>
GACAAGGACACCGTCACCAAGAAGGCTGAAGAGACTATTTCTTGGTTAGACAGCAACACC<br>
ACTGCCAGCAAGGAAGAATTCSATGACAAGTTGAAGGAGTTGCAAGACATTGCCAACCCA<br>
ATCATGTCTAAGTTGTACCAZkGCTGGTGGTGCrCCAGGTGGCGCTGCAGGTGGTGCTCCA<br>
GGCGGTTTCCCAGGTGGTGCTCCTCCAGCTCCAGAGGCTGAAGGTCCAACCGTTGAAGAA<br>
GTTGATTAA<br>
Tlie protein Ssalp belongs to the Hsp70 family of proteins and is resident in the cytosol.<br>
Hsp70s possess the ability to perform a number of chaperone activities; aiding protein<br>
sj'nthesis., assembly and folding: mediating translocation of polypeptides to various<br>
intracellulai' locations, and resolution of protein aggregates (Becker &amp; Craig, 1994, Ew.<br>
J. Biochem. 219, 11-23). Hsp70 genes are highly conserved, possessing an N-terrninal<br>
ATP -binding domain and a C-tenninal peptide-binding domain. Hsp70 proteins interact<br>
with the peptide backbone of, mainly unfolded, proteins. The binding and release of<br>
peptides by hsp70 proteins is an ATP-dependent process and accompanied by a<br>
conformational change in the hsp70 (Becker &amp; Craig, 1994, supra').<br>
Cytosolic hsp70 proteins are particularly involved in the synthesis, folding and secretion<br>
of proteins (Becker &amp; Craig, 1994, supra). In S. cerevisiae C3^tosolic hsp70 proteins have<br>
been divided into two groups: SSA (SSA 1-4) and SSB (SSB 1 and 2) proteins, which are<br>
functionally distinct from each other. The SSA family is essential in that at least one<br>
protein from the group must be active to maintain cell viability (Becker &amp; Craig, 1994,<br>
suprci). Cytosolic hsp70 proteins bind preferentially to unfolded and not mature proteins.<br>
This suggests that they prevent the aggregation of precursor proteins, by maintaining<br>
them in an unfolded state prior to being assembled into multimolecular complexes in the<br>
cytosol and/or facilitating their translocation to various orgauelles (Becker &amp; Craig,<br>
1994, supra). SSA proteins are particularly involved in post-translational biogenesis and<br>
maintenance of precursors for translocation into the endoplasmic reticulum and<br>
mitochondria (Kim eta!., 1998, Proc. Natl. Acad. Sci. USA. 95,12860-12865; Ngosuwan<br>
et aL, 2003, J. Biol Chem. 278 (9), 7034-7042). Ssalp lias been shown to bind damaged<br>
proteins, stabilising them in a partially unfolded form and allowing refolding or<br>
degradation to occur (Becker &amp; Craig, 1994, supra; Glover &amp; Lindquist, 1998, Cell. 94,<br>
73-82).<br>
Variants and fragments of SSA1 are also included hi the present invention. A "variant', in<br>
the context of SSA1, refers to a protein having the sequence of native SSA1 other than for at<br>
one or more positions where there have been amino acid insertions, deletions, or<br>
substitutions, either conservative or non-conservative, provided that such changes result in a<br>
protein whose basic properties, for example enzymatic activity (type of and specific<br>
activity), thermostability, activity in a certain pH-range (pH-stability) have not significantly<br>
been changed. "Significantly" in this context means that one skilled hi the art would say that<br>
the properties of the variant may still be different but would not be unobvious over the ones<br>
of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val, lie, Leu, Ala, Met;<br>
Asp, Glu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Tip. Preferred<br>
conservative substitutions include Gly, Ala; Val, He, Leu; Asp, Glu; Asn, Gin; Ser, Thr;<br>
Lys, Arg; and Phe, Tyr.<br>
A "variant" of SSA1 typically has at least 25%, at least 50%, at least 60% or at least 70%,<br>
preferably at least 80%, more preferably at least 90%, even more preferably at least 95%,<br>
yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the<br>
sequence of native SSA1.<br>
The percent sequence identify between TWO poK'pepiides may be: determined using<br>
suitable computer programs, as discussed below. Such variaits may be natural or made<br>
using th.e methods of protein engineering and site-directed mutagenesis as are well l;nown in<br>
the art.<br>
A "fragment", hi the context of SSAl . refers to a protein having the sequence of native<br>
SSAl oilier than for at one or more positions where there have been deletions. Thus the<br>
fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more<br>
typically up to 70%, preferably up to 80%, more preferably up to 90%, even more<br>
preferably up to 95%, yet more preferably up to 99% of the complete sequence of the. full<br>
mature SSAl protein. Particularly preferred fragments of SSAl protein comprise- one or<br>
more whole domains of the desired protein.<br>
A fragment or variant of SSAl may be a protein that, when expressed recombinantly in a<br>
host cell, such as S. cerevisiae, can complement the deletion of the endogenous^<br>
encoded SSAl gene in the host cell and may, for example, be a naturally occurring<br>
homolog of SSAl. such as a hornolog encoded by another organism, such as another<br>
yeast or other fungi, 01 another eukaryote such as a human or other vertebrate, or animal<br>
or by a plant.<br>
Another preferred chapsrone is PSE1 or a fragment or variant thereof having equivalent<br>
chapero tie-like activity.<br>
PSEJ, also known as /C4.?-?2J, is an essential gene, located on chromosome XItL<br>
A published protein sequence for the protein pselp is as follows:<br>
MSALPE EVWRTLLQIVQAFAS PDNQIRSVAEKS.LSEEWI rENNIEYLLTFLAEQAAFS QD<br>
TTVAALSAVIIFRKLJl!iLKfl.PPSSKLMIMSKl&gt;riTBIPJCEVLAQIRSSLLKGFISEEADSIRH<br>
KLSDArAECVQDDLPAWPELLQALIESLKSGNPWFRESSFRILTTVPYLITAVDINSILP<br>
IFQSGFTDASD1WKIAAVTAFVGYFKQLPKSEWSKLGILLPSLLNSLPRFLDDGKDDALA<br>
SVFESL, IELVELAPKLFKDMFDQI IQFTDMVI IWKDLEPPARTTALELLTVFSEWAPQMC<br>
KSHQNYGQTLVlWTLIMMTEVSIDDDDAaEWISSDDTDDEEEVTYDHARQALDRVALKLG<br>
HPRVQYGCCNVLGQISTDFSPFIQRTAHDRILPALISKLTSECTSRVQTHAAAALVNFSE<br>
FASKDILEPYLDSLLTNLLVLLQSNKLYVQEQALTTIAFIAEAAKNKFIKYYDTLMPLLL<br>
NVLKVWNKDNSVLKGKCMECATLIGFAVGKEKFHEHSQELISILVALQNSDIDEDDALRS<br>
YLEQSWSRICRILGDDFVPLLPIVIPPLLITAKATQDVGLIEEEEMNFQQYPDWDVVQV<br>
QGKHIAIHTSVLDDKVSAMELLQSYATLLRGQFAVYVKEVMEEIALPSLDFYLHDGVRAA<br>
GATLIPILLSCLLAATGTQNEELVLLWHKASSKLIGGLMSEPMPEITQVYHNSLVNGIKV<br>
MG DNCLSEDQLAAFTKGVSANLT DT YERMQDRHGDG DEYNENIDEEEDFTDEDLLDE INK<br>
SIAAVLKTTNGHYLKNLENIWPMINTFLLDNEPILVIFALVVIGDLIQYGGEQTASMKNA<br>
FIPKVTECLISPDARIRQAASYIIGVCAQYAPSTYADVCIPTLDT1VQIVDFPGSKLEEN<br>
RSSTENASAAIAKILYAYNSNIPNVDTYTANWFKTLPTITDKEAASFNYQFLSQLIENNS<br>
PIVCAQSNISAVVDSVIQALNERSLTEREGQTVISSVKKLLGFLPSSDAMAIFNRYPADI<br>
MEKVHKWFA*<br>
A published nucleotide coding sequence of PSE1 is as follows, although it will be<br>
appreciated that the sequence can be modified by degenerate substitutions to obtain<br>
alternative nucleotide sequences which encode an identical protein product:<br>
ATGTCTGCTTTACCGGAAGAAGTTAATAGAACATTACTTCAGArTGTCCAGGCGTTTGCT<br>
TCCCCTGACAATCAAATACGTTCTGTAGCTGAGAAGGCTCTTAGTGAAGAATGGATTACC<br>
GAAAACAATATTGAGTATCTTTTAACTTTTTTGGC'TGAACAAGCCGCTTTCTCCCAAGAT<br>
ACAACAGTTGCAGCATTArCTGCTGTTCTGTTTAGAAAATTAGCATTAAAAGCTCCCCCr<br>
TCTTCGAAGCTTATGATrATGTCCAAAAATATCACACATATTAGGAAAGAAGTTCTTGCA<br>
CAAATTCGTTCTTCATTGTTAAAAGGGTTTTTGTCGGAAAGAGCTGATTCAATTAGGCAC<br>
AAACTATCTGATGCTATTGCTGAGTGTGTTCAAGACGACTTACCAGCATGGCCAGAATTA<br>
CTACAAGCTTTAATAGAGTCTTTAAAAAGCGGTAACCCAAATTTTAGAGAATCCAGTTTT<br>
AGAATTTTGACGACTGTACCTTATTTAATTACCGCTGTTGACATCAACAGTATCTTACCA<br>
ATTTTTCAATCAGGCTTTACTGATGCAAGTGATAATGTCAAAATTGCTGCAGTTACGGCT<br>
TTCGTGGGTTATTTTAAGCAACTACCAAAATCTGAGTGGTCCAAGTTAGGTATTTTATTA<br>
CCAAGTCTTTTGAATAGTTTACCAAGATTTTTAGATGATGGTAAGGACGATGCCCTTGCA<br>
TCAGTTTTTGAATCGTTAATTGAGTTGGTGGAATTGGCACCAAAACTATTCAAGGATATG<br>
TTTGACCAAATAATACAATTCACTGATATGGTTATAAAAAATAAGGATTTAGAACCTCCA<br>
GCAAGAACCACAGCACTCGAACTGCTAACCGTTTTCAGCGAGAACGCTCCCCAAATGTGT<br>
AAATCGAACCAGAATTACGGGCAAACTTTAGTGATGGTTACTTTAATCATGATGACGGAG<br>
GTATCCATAGATGATGATGATGCAGCAGAATGGATAGAATCTGACGATACCGATGATGAA<br>
GAGGAAGTTACATATGACCACGCTCGTCAAGCTCTTGATCGTGTTGCTTTAAAGCTGGGT<br>
GGTGAATATTTGGCTGCACCATTGTTCCAATATTTACAGCAAATGATCACATCAACCGAA<br>
TGGAGAGAAAGATTCGCGGCCATGATGGCACTTTCCTCTGCAGCTGAGGGTTGTGCTGAT<br>
GTTCTGATCGGCGAGATCCCAAAAATCCTGGATATGGTAATTCCCCTCATCAACGATCCT<br>
CATCCAAGAGTACAGTATGGATGTTGTAATGTTTTGGGTCAAATATCTACTGATTTTTCA<br>
TCAG^JiTGCACGTCAAGAGrTCAT-ACGCACGCCGCAGCGGCTCTGGTTAACTTT'TCTGAA<br>
TTCGCTTCGAAGGA^TATTCrTGAGCCTTACTTGGArAGTCTATTGACAAATTTATTAGTT<br>
TTATTACAAAGCA?i.CAAACTTTACGTACAGGAACAGGCCCTAACAACCATTGCATTTATT<br>
GCTGAAGCTGCARAGAATAAS.™TT?.TCAAGTATTACGATACTCTAATGCCATTATTATTA<br>
AATGTTTTGAAGGTTAACAATAAAGAT&gt;JlTAGTGTTTTGAAAGGTAAATGTATGGAATGT<br>
GC&gt;ACTCTGATTGGTTTTGCCGTTGGTAAGGAAA?L»TTTCATGAGCACTCTCAAGAGCTG<br>
ATTTCTATATTGGTCGCTTTACAAAACTCAGATATCGATGAAGATGATGCGCTCAGATCA<br>
TACrTAGAACAAASTTGGAGCAGGATTTGCCGAATTCTGGGTGATGATTTTGTTCCGTTG<br>
TTACCGATTGTTArACCACCCCTGCTAATTACTGCCAAAGCAACGCAAGACGrCGGTTTA<br>
ATTGAAGAAGAAGAAGCAGCAAATTTCCAACAATATCCAGATTGGGATGTTGTTCAAGTr<br>
CAGGGAAAACACATTGCTATTCACACATCCGTCCTTGACGATAAAGTATCAGCAATGGAG<br>
CrArTACAAAGCTATGCGACACTTTTAAGAGGCCAATTTGCTGTATATGTTAAAGAAGTA<br>
ArGGAAGAAATAGCTCTACCATCGCTTGACTTTTACCTACArGACGGTGTTCGrGCTGCA<br>
SGAGCAACTTTAArTCCTATTCTATTATCTTGTTTACTTGCAGCCACCGGTACTC.AAAAC<br>
GAGGAATTGGTATrGTTGTGGCATAAAGCTTCGTCTAAACTAATCGGAGGCTrAATGTCA<br>
GAACCAATGCCAGAAATCACGCAAGTTTATCACAACTCGTTAGTGAATGGTATTAAAGTC<br>
ATGGGTGACAATTGCTTAAGCGAAGACCAATTAGCGGCATTTACTAAGGGTGTCTCCGCC<br>
A» CTTAAC TGACAC T TACGAAAG GATGCAGGAT C G C CATGG TGAT GGT GAT GAATATAAT<br>
GAAAATATTGATGAAGAGGAAGACTTTACTGACGAAGATCrTCTCGATGAAATCAACAAG<br>
TCTATCGCGGCCGTTTTGAA-RACCACAAATGGTCATrATCTAAAGAATTTGGAGAATATA<br>
TGGCCTATGATAAACACArrCCrTTTAGATAATG-AACCAATTTTAGTCATTTrTGCATTA<br>
GTAGTGATTGGTGACTTGATrCAATATGGTGGCGAACAAACTGCTASCATGAAGAACGCA<br>
TTTATTCCAAAGGTTACCGAGTGCTTGATTTCTCCrGACGCTCGTATrCGCCAAGCTGCT<br>
TCrTATAT.AATCGGTGTTTGTGCCCAATACGCTCCATCTACATArGCTGACGTTTGCATA<br>
CCGACTTTAGATACACTTGTTCAGATTGTCGATrTTCCAGGCTCCAaACTGGAAGAAAAT<br>
CGTTCTTCAACAGAGAATGCCAGTGCAGCCATCGCCAAAATTCTTTATGCATACAATTCC<br>
AACATTCCTAACGTAGACACGTACACGGCTAATTGGTTCAAAACGTTACCAACAATAACT<br>
GACAAAGAAGCTGCCTCATTCAACTATCAATTTTTGAGTCAATrGATTGAAAATAATTCG<br>
CCAATTGTGTGTGCTCAATCTAATATCTCCGCTGTAGTTGATTCAGTCATACAAGCCTTG<br>
AATGAGAGAAGTTTGACCGAaAGGGAAGGCCAAACGGTGATAAGTTCAGTTAAAAAGTTG<br>
TTGGGATTTTTGCCTTCTAGTGATGCTATGGCAATTTTCAATAGATATCCAGCTGATATT<br>
ATGGAGAAAGTACATAAATGGTTTGCATAA<br>
The PSEl gene is 3.25-kbp in size. Pselp is involved in the nucjeocj'toplasinic transport<br>
ofmacroinolecules (Seedorf &amp; Silver, 1997, Proc. Nail Acad. Sci USA. 94, 8590-8595).<br>
This process occurs via the nuclear pore complex (NPC) embedded in the nuclear<br>
envelope and made up of nucleoporiiis (Ryan &amp; Wente., 2000, Ciirr. Opin. Cell Biol. 12,<br>
361-371). Proteins possess specific sequences that contain the information required for<br>
nuclear import, nuclear localisation sequence (NLS) and export, nuclear export sequence<br>
(NES) (Pemberton et aL, 1998, Ciar. Opin. Cell Biol 10, 392-399). Pselp is a<br>
karyopherin/importin, a group of proteins, which have been divided up into a and (3<br>
families. Karyopherins are soluble transport factors that mediate the transport of<br>
rnacromolecules across the nuclear membrane by recognising NLS and NES, and interact<br>
with and the NPC (Seedorf &amp; Silver, 1997, supra; Pemberton et aL, 1998, supra; Ryan &amp;<br>
Wente, 2000, supra). Translocation through the nuclear pore is driven by GTP<br>
hydrolysis, catalysed by the small GTP-binding protein, Ran (Seedorf &amp; Silver, 1997,<br>
supra). Pselp has been identified as a karyopherin p. 14 karyopherin (3 proteins have<br>
been identified in S. cerevisiae, of which only 4 are essential. This is perhaps because<br>
multiple karyopherins may mediate the transport of a single macromolecule (Isoyama et<br>
aL, 2001, J. Biol. Chem. 276 (24), 21863-21869). Pselp is localised to the nucleus, at the<br>
nuclear envelope, and to a certain extent to the cytoplasm. This suggests the protein<br>
moves in and out of the nucleus as part of its transport function (Seedorf &amp; Silver, 1997,<br>
supra). Pselp is involved in the nuclear import of transcription factors (Isoyama et al.,<br>
2001, supra; Ueta et aL, 2003, J. Biol. Chem. 278 (50), 50120-50127), Mstones<br>
(Mosammaparast et al., 2002, J. Biol. Chem. Ill (1), 862-86S), and ribosomal proteins<br>
prior to their assembly into ribosomes (Pemberton et al., 1998, supra}. It also mediates<br>
the export of mRNA from the nucleus. Karyopherins recognise and bind distinct NES<br>
found on RNA-binding proteins, which coat the RNA before it is exported from the<br>
nucleus (Seedorf &amp; Silver, 1997, Pemberton et aL, 1998, supra}.<br>
As nucleocytoplasmic transport of macromolecules is essential for proper progression<br>
through the cell cycle, nuclear transport factors, such as pselp are novel candidate targets<br>
for growth control (Seedorf &amp; Silver, 1997, supra}Overexpression of Pselp (protein secretion enhancer) on a multicopy plasmid in S.<br>
cerevisiae has also been shown to increase protein secretion levels of a repertoire of<br>
biologically active proteins (Chow et aL, 1992; J. Cell. Sci. 101 (3), 709-719).<br>
Variants and fragments of I'Sj^l are also included in the present invention. A "variant", in<br>
the context of PSE1. refers to &amp; protein having the sequence of native PSE1 other than for at<br>
one or more positions where there have been aniino acid insertions, deletions, or<br>
substitutions, either consen'ative or-non-conservative, provided that such changes result in a<br>
protein whose basic properties, for example enzymatic activity (type of and specific<br>
activity), thennoslabiiity. activity' in a certain pH-range (pH-stability) have not significantly<br>
been changed. "Significantly" in this context means that one skilled in the art would say that<br>
the properties of the variant may still be different but would not be tmobvious over the ones<br>
of the original protern.<br>
63' "conservative substitutions" is intended combinations such as Val. lie. Leu, Ala. Met;<br>
Asp, Glu; Asn: Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, Tyr, Trp. Preferred<br>
conservative substitutions include Gly, Ala; Val. lie. Leu; Asp, Glu; Asn, Gin; Ser. Thr;<br>
Lys, Arg; and Phe, Tyr.<br>
A "variant" of PSE1 typicalty has at least 25%, at least 50%, at least 60% or at least 70%,<br>
preferably at least 80%, more preferably at least 90%, even more preferably at least 95%,<br>
3ret more preferably at least 99%, most preferably at least 99.5% sequence identity to the<br>
sequence of native PSE1.<br>
The percent sequence identity between two polypeptides may be determined using<br>
suitable computer programs, as discussed below. Such variants may be natural or made<br>
using the methods of protein engineering and site-directed rrmtagenesis as are well known in<br>
the art.<br>
A "fragment", in the context of PSE1, refers to a protein having the sequence of native<br>
PSE1 other than for at one or more positions where there have been deletions. Thus the<br>
fragment may comprise at most 5, 10, 20. 30, 40 or 50%, typically up to 60%, more<br>
typically up to 70%, preferably up to 80%, more preferably up to 90%, even more<br>
preferably up to 95%, yet more preferably up to 99% of the complete sequence of the full<br>
mature PSE1 protein. Particularly preferred fragments of PSE1 protein comprise one or<br>
more whole domains of the desired protein.<br>
A fragment or variant of PSE1 may be a protein that, when expressed recombinantly in a<br>
host cell, such as S. cerevisiae, can complement the deletion of the endogenous PSE1<br>
gene in the host cell and may, for example, be a naturally occurring homolog of PSE1,<br>
such as a hornolog encoded by another organism, such as another yeast or other fungi, or<br>
another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
Another preferred chaperone is ORM2 or a fragment or variant thereof having equivalent<br>
chaperone-like activity.<br>
ORM2, also known as YLR350W, is located on chromosome XII (positions S28729 to<br>
829379) of the S. cerevisiae genome and encodes an evolutionarily conserved protein<br>
with similarity to the yeast protein Ormlp. Hjelrnqvist et al, 2002, Genome Biology,<br>
3(6), research0027.1-0027.16 reports that ORM2 belongs to gene family comprising three<br>
human genes (ORMDL1, ORMDL2 and ORMDL3) as well as homologs in<br>
microsporidia, plants, Drosophila, tirochordates and vertebrates. The ORMDL genes are<br>
reported to encode transmembrane proteins anchored hi the proteins endoplasmic<br>
reticurum (ER).<br>
The protein Orm2p is required for resistance to agents that induce the unfolded protein<br>
response, Hjelmqvist et al, 2002 (supra] reported that a double knockout of the two S.<br>
cerevisiae ORMDL homologs (ORM1 and ORM2) leads to a decreased growth rate and<br>
greater sensitivity to runicamycin and ditMothreitol.<br>
One published sequence of Orm2p is as follows:<br>
MIDRTKKIESPAFEESPLTPMVSNLKPFPSQSNKISTPVTDHRRRRSSSVISHVEQETFED<br>
ENDQQMLPNMNATWVDQRGAWLIHIWIVLLRLFYSLFGSTPKWTWTLTNMTYIIGFYIM<br>
FHLVKGTPFDFNGGAYDNLTMWEQINDETLYTPTRKFLLIVPIVLFLISNQYYRNDMTLF<br>
LSNLAVTVLIGWPKLGITHRLRISIPGITGRAQIS*<br>
The above protein is encoded in S. cerevisiae by the following coding nucleotide<br>
sequence, although it will be appreciated that the sequence can be modified by<br>
degenerate substitutions to obtain alternative nucltolide sequences which encode<br>
an identical protein product:<br>
ATGATTGACCGCACTAAAAACGAATCTCCAGCTTTTGAaGAGTCTCCGCTTACCCCCA?iT<br>
GTGrCTAACCTGAAACCATTCCCTTCTCAAA.GCAACAAAATATCCACTCCAGTGACCGAC<br>
CATAGGAGAAGACGGTCATCCAGCGTAATATCACATGTGGAACAGGAAACCTTCGAAGAC<br>
GMAATGACCAGCAGATGCTTCCCAACATGA.ACGCTACGTGGGTCGACCAGCGAGGCGCG<br>
TGGTTGATTCATATCGTCGTAATAGTACTCTTGAGGCTCTTCTACTCCTTGTTCGGGTCG<br>
ACGCCCAAATGGACGTGGACTTTAACAAACATGACCTACATCATCGGATTCTAT-ATCATG<br>
TTCCACCTTGTCAAAGGTACGCCCTTCGACTTTAACGGTGGTGCGTACGACAACCTGACC<br>
ATGTGGGAGCAGATTAACGATGAGACTTTGTACACACCCACTAGAAAATTTCTGCTGATT<br>
GTACCCATTGTGTTGTTCCTGATTAGCAACCAGTACTACCGCAA.CGACATGACACTATTC<br>
CTCTCCAACCTCGCCGTGACGGTGCTTATTGGTGTCGTTCCTAAGCTGGGAATTACGCAT<br>
AGACTAAGAATATCCATCCCTGGTATTACGGGCCGTGCTCAAATTAGTTAG<br>
Variants and fragments of ORM2 are also included in the present invention. A 'Variant", in<br>
the context of ORM2, refers to a protein having the sequence of native ORM2 other than for -<br>
at one or more positions where there have been arnhio acid insertions, deletions, or<br>
substitutions, either conservative or non-conservative, provided that such changes result in a<br>
protein whose basic properties, for example enzymatic activity (type of and specific<br>
activity), thermostability, activity in a certain pH-range (pH-stability) have not significantly<br>
been changed. "Significantly" in this context means that one skilled in the art would say that<br>
the properties of the variant may still be different but Avould not be unobvious over the ones •<br>
of the original protein.<br>
By "conservative substitutions" is intended combinations such as Val, He, Leu, Ala, Met;<br>
Asp, Giu; Asn, Gin; Ser, Thr, Gly, Ala; Lys, Arg, His; and Phe, T)?, Trp. Preferred<br>
conservative substitutions include G]y, Ala; Val, lie. Leu; Asp, Glu; Asn, Gin; Ser, Tlir;<br>
Lys, Arg; and Phe, T}?.<br>
A "vaiiant" of ORM2 topically has at least 25%, at least 50%, at least 60% or at least 70%,<br>
preferabl)7 at least §0%, more preferably at least 90%, even more preferably at least 95%,<br>
yet more preferably at least 99%, most preferably at least 99.5% sequence identity to the<br>
sequence of native ORM2.<br>
The percent sequence identity between two polypeptides may be determined using<br>
suitable computer programs, as discussed below. Such variants may be natural or made<br>
using the methods of protein engineering and site-directed mutagenesis as are well known in<br>
the art.<br>
A "fragment", in the context of ORM2, refers to a protein having the sequence of native<br>
ORM2 other than for at one or more positions where there have been deletions. Thus the<br>
fragment may comprise at most 5, 10, 20, 30, 40 or 50%, typically up to 60%, more<br>
typically up to 70%, preferably up to 80%, more preferably up to 90%, even more<br>
preferably up to 95%, yet more preferably up to 99% of the complete sequence of the MI<br>
mature ORM2 protein. Particularly preferred fragments of ORM2 protein comprise one or<br>
more whole domains of the desired protein.<br>
A fragment or variant of ORM2 ma}' be a protein that, when expressed recom.bman.tly in<br>
a host cell, such as S. cerevisiae, can complement the deletion of the endogenous ORM2<br>
gene in the host cell and may, for example, be a naturally occurring homolog of ORM2,<br>
such as a homolog encoded by another organism, such as another yeast or other fungi, or<br>
another eukaryote such as a human or other vertebrate, or animal or by a plant.<br>
It is particularly preferred that a plasmid according to a first, second or third aspects of<br>
the invention includes, either within a polynucleotide sequence insertion, or elsewhere on<br>
the plasmid, an open reading frame encoding a protein comprising the sequence of<br>
albumin or a fragment or variant thereof. Alternatively, the host cell into which the<br>
plasmid is transformed may include within its genome a polynucleotide sequence<br>
encoding a protein comprising the sequence of albumin or a fragment or variant thereof,<br>
either as an endogenous or heterologous sequence.<br>
By "albumin" we include a protein having the sequence of an albumin protein obtained<br>
from any source. Typically the source is mammalian. In one preferred embodiment the<br>
serum albumin is human serum albumin ("HSA"). The term "human serum albumin"<br>
includes the meaning of a serum albumin having an amino acid sequence naturally<br>
occurring in humans, and valiants thereof. Preferably the albumin has the amino acid<br>
sequence disclosed in "WO 90/13633 or a variant thereof. The HSA coding sequence is<br>
obtainable by Icnown methods for isolating cDNA corresponding to human genes, and is<br>
also disclosed in, for example, EP 73 646 and EP 286 424.<br>
In another preferred embodiment the "albumin" has the sequence of bovine serum<br>
albumin. The term "bovine serum albumin' includes the meaning of a serum albumin<br>
having an arnino acid sequence naturally occurring in cows, for example as taken from<br>
Swissprot accession number P02769, and variants thereof as defined below. The term<br>
"bovine serum albumin1' also includes the meaning of fragments of full-length bovine<br>
serum albumin or variants thereof, as defined below.<br>
In another preferred embodiment the albmrun is an albumin derived from (i.e. lias the<br>
sequence of) one of serum albumin from dog (e.g. see Swissprot accession number<br>
P49822), pig (e.g. see Swissprot accession number P08835). goat (e.g. as available from<br>
Sigma as product no. A2514 or A4164), turkey (e.g. see. Swissprot accession number<br>
073860), baboon (e.g. as available from Sigma as product no. A1516), cat (e.g. see<br>
Swissprot accession number P49064), chicken (e.g. see Swissprot accession number<br>
P19121). ovalbumin (e.g. chicken ovalburnin) (e.g. see Swissprot accession number<br>
P01012). donkey (e.g. see Swissprot accession number P39090), guinea pig (e.g. as<br>
available from Sigma as product no. A3060, A2639. O5483 or A6539). hamster (e.g. as<br>
available from Sigrna as product no. A5409), horse (e.g. see Swissprot accession number<br>
P35747), rhesus monkey (e.g. see Swissprot accession number Q28522), mouse (e.g. see<br>
Swissprot accession nranber 089020), pigeon (e.g. as defined by Khan el al, 2002. Ini. J.<br>
Biol. MacromoL, 30(3-4),!71-8), rabbit (e.g. see Swissprot accession number P49065),<br>
rat (e.g. see Swissprot accession number P36953) and sheep (e.g. see Swissprot accession<br>
number P14639) and includes variants and fragments thereof as defined below.<br>
Man)' naturally occurring mutant forms of albumin are known. Many are described in<br>
Peters, (1996, All About Albumin: Biochemistry, Genetics and Medical Applications,<br>
Academic Press, Inc., San Diego, California., p.170-1 SI). A variant as defined above<br>
be one of these naturally occurring mutants.<br>
A "variant albumin" refers to an albumin protein wherein at one or more positions there<br>
have been amiuo acid insertions, deletions, or substitutions, either conservative or nonconservative,<br>
provided that such changes result in an albumin protein for which at least one<br>
basic property, for example binding activity (type of and specific activity e.g. binding to<br>
bilinibin), osmolarity (oncotic pressure, colloid osmotic pressure), behaviour in a certain<br>
pH-raiige (pH-stability) has not significantly been changed "Significantly" in this context<br>
means that one skilled in the art would say that the properties of the variant may still be<br>
different but would not be unobvious over the ones of the original protein.<br>
By "conservative substitutions" is intended combinations such as Gly, Ala; Val, lie, Leu;<br>
Asp, Glu; Asn, Gin; Ser, Thr; Lys, Arg; and Phe, Tyr. Such variants may be made by<br>
techniques well known in the art, such as by site-directed mutagenesis as disclosed in US<br>
Patent No 4,302,386 issued 24 November 1981 to Stevens, incorporated herein by reference.<br>
Typically an albumin variant will have more than 40%, usually at least 50%, more typically<br>
at least 60%, preferably at least 70%, more preferably at least 80%, yet more preferably at<br>
least 90%, even more preferably at least 95%, most preferably at least 98% or more<br>
sequence identity with naturally occurring albirmin. The percent sequence identity between<br>
two polypeptides may be determined using suitable computer programs, for example the<br>
GAP program of the University of Wisconsin Genetic Computing Group and it will be<br>
appreciated that percent identity is calculated in relation to polypeptides whose sequence<br>
has been aligned optimally. The alignment may alternatively be carried out using the<br>
Clustal W program (Thompson et al, 1994). The parameters used may be as follows:<br>
Fast pairwise alignment parameters: K-tuple(word) size; 1, window size; 5, gap penalty;<br>
3, number of top diagonals; 5. Scoring method: x percent. Multiple alignment<br>
parameters: gap open penalty; 10, gap extension penalty; 0.05. Scoring matrix:<br>
BLOSUM.<br>
The term "fragment" as used above includes any fragment of full-length albumin or a<br>
variant thereof, so long as at least one basic property, for example binding activity (type of<br>
and specific activity e.g. binding to bilirubin), osmolarity (oncotic pressure, colloid osmotic<br>
pressure), behaviour in a certain pH-range (pH-stability) has aiot significantly been changed.<br>
"Significant!}'" in this context means- that one skilled in the ait would say thai the properties<br>
of the variant may still be different but would not be unobvious over the ones of the original<br>
protein. A fragment will typically be at least 50 amino acids long. A fragment ma}'<br>
comprise at least one whole sub-domain of albumin. Domains of HSA have been<br>
expressed as recombinant proteins (Dockal, M. ef a?., 1999., J. Eiol. Chem., 274, 29303-<br>
29310), where domain I was defined as consisting of amino acids 1-197, domain II was<br>
defined as consisting of arnino acids 189-385 and domain HI was defined as consisting of<br>
amino acids 381-5S5. Partial overlap of the domains occurs because of the extended ahelix<br>
structure (hlO-hl) which exists between domains I and n, and between domains II<br>
and III (Peters, 1996, op. cit, Table 2-4). HSA also comprises six sub-domains (subdomains<br>
LA, IB, HA, HJ3, IDA and IHB). Sub-domain LA. comprises amino acids 6-105,<br>
sub-domain IB comprises amino acids 120-177, sub-doinaia HA comprises aminn acids<br>
200-291, sub-domain HB comprises amino acids 316-369, sub-domain IHA comprises<br>
amino acids 392-491 and sub-domain IHB comprises amino acids 512-583. A fragment<br>
may comprise a whole or part of one or more domains or sub-domains as defined above,<br>
or any combination of those domains and/or sub-domains.<br>
Thus the pohynucleotide insertion may comprise an open reading frame that encodes<br>
albumin or a variant or fragment thereof.<br>
Alternatively, it is preferred that a plasmid according to a first, second or third aspects of<br>
the invention includes, either within a polynucleotide sequence insertion, or elsewhere on<br>
the plasmid, an open reading frame encoding a protein comprising the sequence of<br>
transferrin or a variant or fragment thereof. Alternatively, the host cell into which the<br>
plasmid is transformed msy include within its genome a potynueleotide sequence<br>
encoding a protein comprising the sequence of transferrin or a variant or fragment<br>
thereof, either as an endogenous or heterologous sequence.<br>
Tie term "rransferrin" as used herein includes all members of the traiisferrin family<br>
(Testa, Proteins of iron metabolism, CRC Press. 2002; Harris &amp; Aiseii, Iron carriers and<br>
iron proteins, Vol. 5, Physical Bioinorganic Chemistry, VCH, 1991) and their<br>
derivatives, such as transferrin, mutant transferrins (Mason et al, 1993, Biochemistry. 32;<br>
5472; Mason et a!, 1998, Biochem. J., 330(1), 35), truncated transfemns, transferrin lobes<br>
(Mason et al, 1996, Protein Expr. Purif., 8, 119; Mason et al, 1991, Protein Expr. Purif.,<br>
2, 214), lactoferrin, mutant lactoferrins, truncated lactoferrins, lactoferrin lobes or fusions<br>
of any of the above to other peptides, polypepticles or proteins (Shin et al, 1995, Proc.<br>
Natl. Acad. Sci. USA, 92, 2820; Ali et al, 1999, J. Biol Ckem., 274, 24066; Mason et al,<br>
2002, Biochemistry, 41, 9448).<br>
The transferrin may be human transferrin. The term "human transferrin" is used herein to<br>
denote material which is indistinguishable from transfenin derived from a human or<br>
which is a variant or fragment thereof. A "variant" includes insertions, deletions and<br>
substitutions, either conservative or non-conservative, where such changes do not<br>
substantially alter the useful ligand-binding or immunogenic properties of transferrin.<br>
Mutants of transferrin are included hi the invention. Such mutants may have altered<br>
irrrmunogenicity. For example, transferrin mutants may display modified (e.g. reduced)<br>
glycosylation. The N-linked glycosylation pattern of a transferrin molecule can be<br>
modified by adding/removing ammo acid glycosylation consensus sequences such as NX-<br>
S/T, at any or all of the N, X, or S/T position. Transferrin mutants may be altered in<br>
their natural binding to metal ions and/or other proteins, such as transferrin receptor. An<br>
example of a transferrin mutant modified in this manner is exemplified below.<br>
We also include, naturally-occurring polymorphic variants of human transferrin or human<br>
transferrin analogues. Generally, variants or fragments of human transferrin will have at<br>
least 50% (preferably at least 80%, 90% or 95%) of human transferring ligand binding<br>
activity (for example iron-binding), weight for weight. The iron binding activity of<br>
transferrin or a test sample can be determined spectrophotometrically by 470nm:280nm<br>
absorbance ratios for the proteins in their iron-free and fully iron-loaded states. Reagents<br>
should be iron-free unless stated otherwise. Iron can be removed from transferrin or the<br>
test sample by dialysis against 0.1M citrate, 0.1M acetate, lOmM EDTA pH4.5. Protein<br>
should be at approximately 20mg/mL in lOOmM HEPES, lOrnM NaHC03 pHS.O.<br>
Measure the 470nm:280nm absorbance ratio of apo-transferrin (Calbiochem, CN<br>
Biosciences, Nottingham, UK) diluted in water so that, absorbance at 280nm can be<br>
accurately determined spectrophotometrically (0% iron binding). Prepare 20mM iron-<br>
nitrilotri acetate (FeNTA) solution by dissolving I9hng nitroiriaceiic acid in 2mL ]M<br>
NaOH, then add 2mL 0.5M fenic chloride. Dilute to 50mL with deionised watex. Full}'<br>
load apo-transfenin with iron (100% iron binding) by adding a sufficient excess of<br>
freshly prepared 20rnM FeNTA. then dialyse tlie halo-transferriii preparation completely'<br>
against lOQmM HEPES, lOmM NaHC03 pHS.O to remove remaining FeNTA before<br>
measuring the. absorbance ratio at 47Gnrn:280nm. Repeat the procedure using lest<br>
sample, which should initially be free from iron, and compare final ratios to the control.<br>
Additionally, single or multiple heterologous fusions comprising any of the above; or<br>
single or multiple heterologous fusions to albumin, transfenin or hnmunoglobins. or a<br>
variant or fragment of any of these may be used. Such fusions include albumin Nterminal<br>
fusions, albumin C-terrninal fusions and co-N-temiinal and C-terminal albumin<br>
fusions as exemplified by WO 01/79271, and transfenin N-terminal fusions, transferrin<br>
C-terminal fusions, and co-N-terminal and C-terminal transfenin fusions.<br>
The skilled person will also appreciate that the open reading frame of airy other gene or<br>
variant, or part or either, can be utilised to form a whole or part of an open reading frame<br>
in fanning a pohoiucleotide sequence insertion for use with the present invention. For<br>
example, the open reading frame may encode a protein comprising an}'- sequence, be it a<br>
natural protein (including a zymogen), or a variant, or a fragment (winch may. for<br>
example, be a domain) of a natural protein: or a totally s}'nthetic protein; or a single or<br>
multiple fusion of different proteins (natural or synthetic). Such proteins can be talc en,<br>
but not exclusively, from the lists provided in WO 01/79258, WO 01/79271, WO<br>
01/79442, WO 01/79443, WO 01/79444 and WO 01/79480, or a variant or fragment<br>
thereof; the disclosures of which are incorporated herein b}' reference. Although these<br>
patent applications present the list of proteins in the context of fusion partners for<br>
albumin, the present invention is not so limited and, for the purposes of the present<br>
invention, any of the proteins listed therein ma}' be presented alone or as fusion partners<br>
for albumin, the Fc region of urummoglobului, transfenin., lactofenin or an}' other protein<br>
or fragment or variant of any of the above, including fusion proteins comprising any of<br>
the above, as a desired pohypeptide. Further examples of transfenin fusions are given in<br>
US patent applications US2003/0221201 and US2003/0226155.<br>
Preferred other examples of desirable proteins for expression by the present invention<br>
includes sequences comprising the sequence of a monoclonal antibody, an etoposide, a<br>
serum protein (such as a blood clotting factor), antistasin, a tick anticoagulant peptide,<br>
transferrin, lactoferrin., endostatin, angio statin, collagens, immuiio globulins or<br>
immunoglobulin-based molecules or fragment of either (e.g. a Small Modular<br>
ImmmioPhaiinaceutical™ ("SMTP") or dAb, Fab' fragments, F(ab')2, scAb, scFv or scFv<br>
fragment), a Kmiitz domain protein (such as those described hi WO 03/066824-, with or<br>
without albumin fusions) interferons, interleuldns, IL10, IL11, IL2, interferon a species<br>
and sub-species, interferon P species and sub-species, interferou y species and subspecies,<br>
leptin, CNTF, CNTFAxis, IL1-receptor antagonist, erythropoetin (EPO) and EPO<br>
mimics, thrombopoetin (TPO) and TPO mimics., prosaptide, cyanovirin-N, 5-helix, T20<br>
peptide, T1249 peptide, HTV gp41, HTV gp!20, uroldnase, prourolcinase, tPA (tissue<br>
plasminogen activator), hirudin, platelet derived growth, factor, parathyroid hormone,<br>
pro insulin, insulin, glucagon, glucagon-lilce peptides, insulin-like growth factor,<br>
calcitonin, growth honnone, transforming growth factor (3, tumour necrosis factor, GCSF,<br>
GM-CSF, M-CSF, FGF, coagulation factors in both pre and active forms, including<br>
but not limited to plasminogen, fibrinogen, thrombin, pre-thrombin, pro-thrombin, von<br>
Willebrand's factor, ai-antitrypsin, plasminogen activators, Factor VIE, Factor VIII,<br>
Factor IX, Factor X and Factor XTfl, nerve growth factor, LACI (lipoprotein associated<br>
coagulation inhibitor, also known as tissue factor pathway inhibitor or extrinsic pathway<br>
inhibitor), platelet-derived endothelial cell growth factor (PD-ECGF), glucose oxidase,<br>
serum cholinesterase, aprotinin, amyloid precursor, inter-alpha trypsin inhibitor,<br>
antithrombin III, apo-lipoprotein species, Protein C, Protein S, a variant or fragment or<br>
fusion protein of any of the above. The protein may or may not be Mrudin.<br>
A "variant", in the context of the above-listed proteins, refers to a protein wherein at one or<br>
more positions there have been amino acid insertions, deletions, or substitutions, either<br>
conservative or non-conservative, provided that such changes result in a protein whose basic<br>
properties, for example enzymatic activity or receptor binding (type of and specific activity),<br>
thermostability, activity in a certain pH-range (pH-stability) have not significantly been<br>
changed, "Significantly" in this context means that one skilled in the art would say that the<br>
 properties of the variant may still be different but would not be vmobvious over the ones of<br>
the original protein.<br>
By "conservative substitutions" is intended combinations such as Val. He. Leu. Ala, Met;<br>
Asp. Glu; Asa, Gin; Ser, Thr, Gly, Ala; Lys, Arg. His: and Phe, 131', Trp. Preferred<br>
conservative substitutions include Gly. Ala.; Val, lie. Leu; Asp, Glu: Asn, Gin: Ser, Thr;<br>
Lys, Arg; and Phe, Tyr.<br>
A "variant" typically has at least 25%, at least 50%, at least 60% or at least 70%, preferably<br>
at least 80%, more preferably at least 90%, even more preferably -at least 95%, yet more<br>
preferably at least 99%, most preferably at least 99.5% sequence identity to the polypeptide<br>
from which it is derived.<br>
The percent sequence identity between two polypeptides may be determined using<br>
suitable computer programs, for example the GAP program of the University' of<br>
Wisconsin Genetic Computing Group and it will be appreciated that percent identity is<br>
calculated in relation to polypeptides whose sequence has been aligned optimally.<br>
The alignment may alternatively be carried out using the Clustal W program (Thompson<br>
st al, (1994) Nucleic Acids Res., 22(22), 4673-80). The parameters used imy be as<br>
follows:<br>
• Fast pairwise alignment parameters: K-tuplefword) size; 1, window size; 5; gap<br>
penalty; 3, number of top diagonals; 5. Scoring method: x percent.<br>
• Multiple alignment parameters: gap open penalty; 10, gap extension penalty; 0.05.<br>
• Scoring matrix: BLOSUM.<br>
Such variants may be natural or made using the methods of protein engmeering and sitedirected<br>
mutagenesis as are well known in the art.<br>
A "fragment", in the context of the above-listed proteins, refers to a protein wherein at one<br>
or more positions there have been deletions. Thus the fragment ma}' comprise at most 5,103<br>
20. 30, 40 or 50% of the complete'sequence of the full mature polypeptide. Typically a<br>
fragment comprises up to 60%, more typically up to 70%, preferably up to 80%, more<br>
preferably up to 90%, even more preferably up to 95%, yet more preferably up to 99% of<br>
the complete sequence of the full desired protein. Particularly preferred fragments of a<br>
desired protein comprise one or more whole domains of the desired protein.<br>
It is particularly preferred that a plasrnid according to a first, second or third aspects of<br>
the invention includes, either within a polynucleotide sequence insertion, or elsewhere on<br>
the plasmid, an open reading frame encoding a protein comprising the sequence of<br>
albumin or a fragment or variant thereof, or any other protein take from the examples<br>
above (fused or unfused to a fusion partner) and at least one other heterologous sequence,<br>
wherein the at least one other heterologous sequence may contain a transcribed region,<br>
such as an open reading frame. In one embodiment, the open reading frame may encode<br>
a protein comprising the sequence of a yeast protein, hi another embodiment the open<br>
reading frame may encode a protein comprising the sequence of a protein involved in<br>
protein folding, or which has chaperone activity or is involved in the unfolded protein<br>
response, preferably protein disulphide isomerase.<br>
The resulting plasmids may or ma}' not have symmetry between the US and UL regions.<br>
For example, a size ratio of 1:1, 5:4, 5:3, 5:2, 5:1 or 5:
and UL or between UL and US regions. The benefits of the present invention do not rely<br>
on symmetry being maintained.<br>
The present invention also provides a method of preparing a plasmid of the invention,<br>
which method comprises -<br>
(a) providing a 2jim-family plasmid comprising a REP2 gene or an FLP gene and an<br>
inverted repeat adjacent to said gene;<br>
(b) providing a polynucleotide sequence and inserting the pofynucleotide sequence<br>
into the plasmid at a position according to the first, second or third prefen-ed<br>
aspects of the invention; and/or<br>
 (c) additionally or as an alternative to step (b). deleuug some or all of the riucleotide<br>
bases at the positions according to the first, second or third preferred aspects of<br>
the invention: and/or<br>
(d) additionall}' or as an alternative to either of steps (b) and (c). substituting some or<br>
all of the nucleoti.de bases at the positions according to the first, second or third<br>
preferred aspects of the invention with alternative nucleotide bases.<br>
Steps (b), (c) and (d) can be achieved using techniques well known in the art, including<br>
cloning techniques, site-directed niutagenesis and the lice, such as are described in b}'<br>
Sambrook et al, Molecular Cloning: A Laboratory Manual, 2001, 3rd edition, the<br>
contents of which are incorporated herein by reference. For example, one such method<br>
involves ligation via cohesive ends. Compatible cohesive ends can be generated on a DNA<br>
fragment for insertion and plasmid by the action of suitable restriction engines. These ends<br>
will rapidly anneal through complementary base pairing and remaining nicies can be closed. -<br>
by the action of DNA ligase.<br>
A further method uses synthetic double stranded oligonucleotide linkers and adaptors. DNA<br>
fragments with blunt ends are generated ~bj bacteriophage T4 DNA porymerase or E. call<br>
DNA polyrnerase I which remove protruding 3' termini and £11 in recessed 3s ends.<br>
Synthetic linkers and pieces of blunt-ended double-stranded DNA, which contain<br>
recognition sequences for defined restliction enzymes, can be ligated to blunt-ended DNA<br>
fragments by T4 DNA ligase. They are subsequently digested with appropriate restriction<br>
enzymes to create cohesive, ends and ligated to an expression vector with compatible<br>
tenrrini. Adaptors are also chemically synthesised DNA fragments which contain one blunt<br>
end used for hgation but which also possess one preformed cohesive end. Alternatively a<br>
DNA fragment or DNA fragments can be ligated together by the action of DNA ligase in<br>
the presence or absence of one or more S}'nthetic double stranded oligonucleotides<br>
optionally containing cohesive ends.<br>
Synthetic linkers containing a variety of restriction endonuclease sites are commercially<br>
available from a number of sources including Sigrna-Genosys Ltd, London Road,<br>
Pampisfordj Cambridge, United Kingdom.<br>
According]y, the present invention also provides a plasmid obtainable by the above<br>
method.<br>
The present invention also provides a host cell comprising a plasmid as defined above.<br>
The host cell may be any type of cell. Bacterial and yeast host cells are preferred.<br>
Bacterial host ceils may be useful for cloning purposes. Yeast host cells may be useful<br>
for expression of genes present in the plasmid.<br>
In one embodiment the host cell is a cell in which the plasmid is stable as a multicopy<br>
plasmid. Plasmids obtained from one yeast type can be maintained in other yeast types<br>
(Me et al, 1991, Gene, 108(1), 139-144; Me et al, 1991, Mol Gen. Genet, 225(2), 257-<br>
265). For example, pSRl from Zygosaccharomyces rouxii can be maintained in<br>
Saccharomyces cerevisiae. Where the plasmid is based on pSRl, pSB3 or pSB4 the host<br>
cell may be Zygosaccharomyces rouxii, where the plasmid is based on pSBl or pSB2 the<br>
host cell may be Zygosaccharomyces bailli, where the plasmid is based on pPMl the host<br>
cell may be Pichia membranaefaciens, where the plasmid is based on pSMl the host cell<br>
may be Zygosaccharomyces• fermentati, where the plasmid is based on pKDl the host cell<br>
may be Kluyveromyces drosophilarum and where the plasmid is based on the 2|im<br>
plasmid the host cell may be Saccharomyces cerevisiae or Saccharomyces<br>
carlsbergensis. A 2um-family plasmid of the invention can be said to be "based on" a<br>
naturally occurring plasmid if it comprises one, two or preferably three of the genes FLP,<br>
REPJ and RJEP2 having sequences derived from that naturally occurring plasmid.<br>
A plasmid as defined above, may be introduced into a host through, standard techniques.<br>
With regard to transformation of prokaryotic host cells, see, for example, Cohen eta! (1972)<br>
Proc, Natl. Acad. Sci. USA 69, 2110 and Sarnbrook et al (2001) Molecular Cloning, A<br>
Laboratory Manual, 3rd Ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.<br>
Transformation of yeast cells is described in Sherman et al (1986) Methods In Yeast<br>
Genetics, A Laboratory Manual, Cold Spring Harbor, NY. The method of Beggs (1978)<br>
Nature 275, 104-109 is also useful. Methods for the transformation of S, cerevisiae are<br>
taught generally in EP 251 744, EP 258 067 arid WO 90/01063, all of which are<br>
incorporated herein by reference. With regard to vertebrate cells, reagents useful in<br>
transfecting such cells, fox example calcium phosphate and DEAE-dextran or liposome<br>
formulations, are available from Stratageue Cloning Systems, or Life Technologies Inc.,<br>
Gaithersburg, I\dD 20877, USA.<br>
Electroporation is also useful for transfonning cells and is well known, in the art for<br>
transfonning yeast cell, bacterial cells and vertebrate, cells. Methods for transformation of<br>
yeast by electroporation axe disclosed in Becker &amp; Guarente (1990) Methods En^nnol 194,<br>
182.<br>
Generally, the plasmid will transform not all of "die hosts and it will therefore be necessary to<br>
select for transformed host cells. Thus, a plasmid according to any one of the first, second<br>
or third aspects of the present invention ms.y comprise a selectable marker, either within a<br>
polynucleotide sequence insertion, or elsewhere on the plasmid, including but not limited<br>
to bacterial selectable marker and/or a j^east selectable marker. A typical bacterial<br>
selectable marker is the p-lactamase gene although many others are known in the art.<br>
Suitable yeast selectable marker include LEU2 (or an equivalent gene encoding a protein<br>
with the activity of p-lactamase malate dehydrogenase), TRP1, HISS, H1S4, URA3,<br>
URA5, SFA1, ADE2, METIS, L7S5, LYS2, ILV2, FBA1, PSE1, PDI1 and PGK1. In light<br>
of the different options available, the most suitable selectable markers can be chosen. If<br>
it is desirable to do so. URA3 and/or LEU2 can be avoided. Those skilled in the art will<br>
appreciate that an)' gene whose chromosomal deletion or inactivation results in an<br>
inviable host, so called essential genes, can be used as a selective marker if a functional<br>
gene is provided on the plasmid, as demonstrated for PGK1 in apgkl yeast strain (Piper<br>
and Curran, 1990, Curr. Genet. 17, 119). Suitable essential genes can be found within<br>
the Stanford Genome Database (SGD). http:://db.yeastgenorne,org).<br>
Additionally, a plasmid according to any one of the first, second or third aspects of the<br>
present invention ma]' comprise more than one selectable marker, either within a<br>
polynucleotide sequence insertion, or elsewhere on the plasmid.<br>
One selection technique involves incorporating into the expression vector a DNA sequence<br>
marker, Avith any necessary control elements, that codes for a selectable trait in the<br>
transformed cell. These markers include dihydrofolate reductase, G418 or neomycin<br>
resistance for eukaryotic cell culture, and tetrac3'clin, kanamycin or ampicillin (i.e. (3-<br>
lactamase) resistance genes for culttuing in Kcoli and other bacteria. Alternatively, the<br>
gene for such selectable trait can be on another vector, which is used to co-transform the<br>
desired host cell.<br>
Another method of identifying successfully transformed cells involves growing the cells<br>
resulting from the introduction of a plasmid of the invention, optionally to allow the<br>
expression of a recornbinant polypeptide (i.e. a polypeptide which is encoded by a<br>
polynucleotide sequence on the plasmid and is heterologous to the host cell, in the sense that<br>
that polypeptide is not naturally produced by the host). Cells can be harvested and lysed and<br>
their DNA or RNA content examined for the presence of the recornbinant sequence using a<br>
method such as that described by Southern (1975) J. Mol Biol. 98, 503 or Berent et al<br>
(1985) Biotech. 3/208, or other methods of DNA and RNA analysis common in the ait<br>
Alternatively, the presence of a polypeptide in the supernatant of a culture of a transformed<br>
cell can be detected using antibodies.<br>
In addition to directly assaying for the presence of recornbinant DNA, successful<br>
transformation can be confirmed by well known irnmunological methods when the<br>
recornbinant DNA is capable of directing the expression of the protein. For example, cells<br>
successfully transformed with an expression vector produce proteins displaying appropriate<br>
antigenicity. Samples of cells suspected of being transformed are harvested and assayed for<br>
the protein using suitable antibodies.<br>
L<br>
Thus, in addition to the transformed host cells themselves, the present invention also<br>
contemplates a culture of those cells, preferably a monoclonal (clonally homogeneous)<br>
culture, or a culture derived from a monoclonal culture, in a nutrient medium. Alternatively,<br>
transformed cells may themselves represent an industrially/commercially or<br>
pharmaceutically useful product and can be purified from a culture medium and optionally<br>
formulated with a carrier or diluent in a manner appropriate to their intended<br>
industrial/commercial or pharmaceutical use, and optionally packaged and presented in a<br>
manner suitable for that use. For example, whole cells could be immobilised; or iised to<br>
spray a cell culture directly on to/into a process, crop or other desired target. Similarly,<br>
whole cell such as yeast cells can be used as capsules for a huge variety of applications,<br>
5 sucli as fragrances, flavours and phamiaceuticals.<br>
Transformed host cells may then be cultured for a sufficient time and under appropriate<br>
conditions known to those skilled in the art. and in view of the teachings disclosed herein,<br>
to permit the expression of an}' ORF(s) in the one or more polynucleotide sequence<br>
10 insertions within the plasmid.<br>
The present invention thus also provides a method for producing a protein comprising the<br>
steps of (a) providing a plasmid according to the first, second or third aspects of the<br>
invention as defined above; (b) providing a suitable host cell; (c) transforming the host<br>
15 cell with the plasmid; and (d) culturing the transformed host cell in a culture medium,.,<br>
thereby to produce the protein.<br>
Many expression systems are known, including bacteria (for example E. coli and Bacillus<br>
subnlis], yeasts, filamentous fungi (for example Aspergilhis], plant cells, whole plants,<br>
20 animal cells and insect cells.<br>
In one embodiment the preferred host cells are the yeasts in which the plasmid is capable<br>
of being maintained as a stable multicopy plasmid. Such yeasts include Saccharomyces<br>
cerevisiae, J&amp;vyvercmiyces lactis, Pichia pastoris, Zygosaccharomyces rouxii,<br>
25 Zygosaccharojnyces bailli, Zygos-accharomyces ferm.enta.ti, and Kluyveromyces<br>
drosophilarum.<br>
A plasmid is capable of being maintained as a stable multicopy plasmid in a host, if the<br>
plasmid contains, or is modified to contain, a selectable (e.g. LEU2) marker, and stability,<br>
30 as measured by the loss of the marker, is at least 1%, 2%, 3%, 4%, 5%, 10%, 15%, 20%,<br>
25%, 30%, 40%5 50%, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%,<br>
99.9% or substantially 100% after one, two, three, four, five, six, seven, eight, nine- ten,<br>
11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 25., 30, 35, 40, 45, 50, 60, 70, SO, 90, 100 or more<br>
PCT/GB 2004 / 0 0 5 4 3 5<br>
WO 2005/061719 PCT/CB2004/005435<br>
^ generations. Loss of a marker can be assessed as described above, with reference to<br>
Chinery &amp; Hinchliffe (1989, Cior. Genet, 16, 21-25).<br>
It is particularly advantageous to use a yeast deficient in one or more protein rnannosyl<br>
5 transferases involved in 0-glycosylation of proteins, for instance by disruption of the<br>
gene coding sequence.<br>
Recombinantly expressed proteins can be subject to undesirable post-translational<br>
modifications by the producing host cell. For example, the albumin protein sequence<br>
10 does not contain any sites for N-linked glycosylation and has not been reported to be<br>
modified., in nature, by 0-linked glycosylation. However, it has been found that<br>
recombinant human albumin ("rHA") produced in a number of yeast species can be<br>
modified by 0-linked glycosylation, generally involving mannose. The mannosylated<br>
albumin is able to bind to the lectin Concanavalin A. The amount of mannosylated<br>
15 albumin produced by the yeast can be reduced by using a yeast strain deficient in one or<br>
more of the PMT genes (WO 94/04687). The most convenient way of achieving this is to<br>
create a yeast which has a defect in its genome such that a reduced level of one of the<br>
Pint proteins is produced. For example, there may be a deletion, insertion or<br>
transposition in the coding sequence or the regulatory regions (or in another gene<br>
20 regulating the expression of one of the PMT genes) such that little or no Pint protein is<br>
produced. Alternatively, the yeast could be transformed to produce an anti-Pmt agent,<br>
such as an anti-Pmt antibody.<br>
If a 3'east other man S. cerevisiae is used, disruption of one or more of the genes<br>
25 equivalent to the PMT genes of S. cerevisiae is also beneficial, e.g. in Pichia pastoris or<br>
Khiyveromyces lactis. The sequence of PMTl (or any other PMT gene) isolated from S.<br>
cerevisiae may be used for the identification or disruption of genes encoding similar<br>
enzymatic activities in other fungal species. The cloning of the PMTl homologue of<br>
Kluyveromyces lactis is described in WO 94/04687.<br>
The yeast will advantageously have a deletion of the HSP150 and/or YAPS genes as<br>
taught respectively hi WO 95/33833 and WO 95/23857.<br>
58<br>
The preseirl application also provides a method of producing, a protein comprising the<br>
steps of providing a host ceil as defined above,, which host cell comprises a plasniid of<br>
the present invention and culturing the host cell in a culture medium thereby to produce<br>
the protein, The culture medium may be non-selective or place a selective pressure on<br>
the stable multicopy maintenance of the plasmid.<br>
A method of producing a protein expressed from a plasmid of the invention preferably<br>
further comprise the step of isolating the thus produced protein from the cultured host cell<br>
or the culture medium..<br>
The thus produced protein may be present intracellularly or. if secreted, in the culture<br>
medium and/or periplasrnic space of the host cell. The protein may be isolated from the<br>
cell and/or culture medium by many methods known in the art. For example purification<br>
techniques for the recovery of recombinantly expressed albumin have been disclosed in:<br>
WO 92/04367, removal of matrix-derived dye; EP 464 590, removal of yeast-derived.<br>
colorants; EP 319 067, alkaline precipitation and subsequent application of the albumin<br>
to a lipophihc phase; and WO '96/37515, US 5 728 553 and WO 00/44772, which<br>
describe complete purification processes; all of which are incorporated herein by<br>
reference. Proteins other than albumin may be purified from the culture medium by any<br>
technique that has been found to be useful for purifying such proteins.<br>
Such well-known methods include ammonium sulphate or ethanol precipitation, acid or<br>
solvent extraction, anion or cation exchange chromatography, phosphocellulose<br>
chrornatography, hydrophobia interaction chromatography, affinity chromatograplxy,<br>
hydroxylapatite chromatography., lectin chromatograprry, concentration, dilution, pH<br>
adjustment, diafiltration, ultrafiltration, high performance liquid chromatography ("HPLC"),<br>
reverse phase HPLC, conductivity adjustment and the like.<br>
hi one embodiment, any one or more of the above mentioned techniques may be used to<br>
further purifying the thus isolated protein to a commercially acceptable level of purity.<br>
By commercially acceptable level of purity, we include the provision of the protein at a<br>
concentration of at least 0.01 g.L'J, 0.02 g.L"1, 0.03 g.L'3, 0.04 g.L"1, 0.05 g.L~].,0,06 g.L"<br>
],0.07 gU\ O.OS gr1, 0.09 g.U\ 0.1 g.1/1, 0.2 gL~\ 0.3 gU1, 0.4 g.L'1, 0.5 g.L"3, 0.6 g.L'1,<br>
59<br>
0.7 g.1/1, 0.8 g.L-', 0.9 g.1/1, 1 gX-', 2 g.1/1, 3 g.L'1, 4 g.1/1, 5 gi;1, 6 gl"1, 7 g.I/1, S g.U9 gX-1, 10 gX-1, 15 gX-1, 20 g.1/1, 25 gX'1, 30 gX'1, 40 g-I/'pO gX'1, 60 gL'1, 70 g.L'1, 70<br>
g.1/1, 90 g.L/1, 100 g.1/1, 150 gX-1, 200 g.L/1 ,250 g.L'1, 300 g.L/1, 350 g.1/1, 400 g.1/1, 500<br>
gX"1, 600 g.L/1, 700 g.1/1, 800 g.1/1, 900 g.L/1, 1000 gX'1, or more.<br>
The thus purified protein may be lyophilised. Alternatively it may be formulated with a<br>
carrier or diluent, and optionally presented in a unit form.<br>
It is preferred that the protein is isolated to achieve a pharmaceutically acceptable level of<br>
purity. A protein has a pharmaceutically acceptable level of purity if it is essentially<br>
pyrogen free and can be administered in a pharmaceutically efficacious amount without<br>
causing medical effects not associated with the activity of the protein.<br>
The resulting protein may be used for any of its known utilities, which, in the case of<br>
albumin, include i.v. administration to patients to treat severe bums, shock aad blood<br>
loss, supplementing culture media., and as an excipient in formulations of other proteins.<br>
Although it is possible for a therapeutically useful desired protein obtained by a process of<br>
the invention to be administered alone, it is preferable to present it as a pharmaceutical<br>
formulation, together with one or more acceptable carriers or diluents. The canier(s) or<br>
diluent(s) must be "acceptable" in the sense of being compatible with the desired protein and<br>
not deleterious to the recipients thereof. Typically, the carriers or diluents will be water or<br>
saline which will be sterile and pyrogen free.<br>
Optionally the thus formulated protein will be presented in a unit dosage form, such as in<br>
the form of a tablet, capsule, injectable solution or the like.<br>
We have also demonstrated that a rilasTrn'd-bnme gene<br>
sequence of an "essential" protein can be used to stably maintain the plasrnid in a host<br>
cell that, in the absence of the plasmid, does not produce the essential protein. A<br>
preferred essential protein is an essential chaperone, which can provide the further<br>
advantage that, as well as acting as a selectable marker to increase plasmid stability, its<br>
expression simultaneously increases the expression of a heterologous protein encoded by<br>
a Tecombinant gene within the host cell. This system is advantageous because it allows<br>
the use]- to minimise the number of recornbrnant genes that need to he carried by a<br>
plasniid. For example, typical prior art plasmids carry marker genes (such as those as<br>
described above) thai enable the plasniid to he stably maintained during bost cell<br>
culturing process. Such marker genes need to be retained on the plasmid in addition to<br>
any further gexies that are required to achieve a desired effect. However, the ability of<br>
plasmids to incorporate exogenous DNA sequences is limited and it is therefore<br>
advantageous to minimise the number of sequence insertions required to achieve a<br>
desired effect. Moreover, some marker genes (such as auxotrophic marker genes) require<br>
the ctilturing process to be conducted under specific conditions in order to obtain the<br>
effect of the marker gene. Such specific conditions may not be optimal for cell grcnvth or<br>
protein production, or may require inefficient or unduly expensive growth systems to be<br>
used.<br>
Thus, it is possible to use a gene that recombin.an.tly encodes a protein comprising the.,<br>
sequence of an "essential protein" as a plasmid-bome gene to increase plasmid stability,<br>
where the plasmid is present within a cell that, in the absence of the plasmid. is unable to<br>
produce the 'essential protein".<br>
It is preferred that the "essential protein" is one that, when its encoding gene(s) in a host<br>
cell are deleted or inactivated, does not result in the host cell developing an. auxotrophic<br>
(bios3rnthetic) requirement. By "auxotrophic (bios37nthetic) requirement" we include a<br>
deficiency that can be complemented by additions or modifications to tbe growth,<br>
medium. Therefore, an "essential marker gene" which encodes an "essential protein"., in<br>
the context of the present invention is one that, when deleted or inactivated in a host cell,<br>
results in a deficiency which cannot be complemented by additions or modifications to<br>
the growth medium. The advantage of this S3rstem is that the "essential marker gene" can<br>
be used as a selectable marker on a plasmid in host cell that, in the absence of the<br>
plasmid, is unable to produce that gene product, to achieve increased plasmid stability<br>
without the disadvantage of requiring the cell to be cultured under specific selective (e.g.<br>
selective nutrient) conditions. Therefore, the host cell can be cultured under conditions<br>
that do not have to be adapted for any particular marker gene, without losing plasmid<br>
stability. For example, host cells produced using this system can be cultured in non-<br>
61<br>
selective media such as complex or rich media, which may be more economical than the<br>
minimal media that are commonly used to give auxotrophic marker genes their effect.<br>
The cell may, for example, have its endogenous gene or genes deleted or otherwise<br>
inactivated.<br>
It is particularly preferred if the "essential protein" is an "essential" chaperone, as this<br>
can provide the dual advantage of improving plasrnid stability without the need for<br>
selective growth conditions and increasing the production of proteins, such as<br>
endogenously encoded or a heterologous proteins, in the host cell. This s)rsteni also has<br>
the advantage that it minimises the number of recombinant genes that need to be<br>
earned by the plasrnid if one chooses to use over-expression of an essential<br>
chaperone to increase protein production by the host cell.<br>
Preferred "essential proteins" for use in this aspect of the invention include the<br>
"essential" chaperones PDI1 and PSE1, and other "essential" gene products such as<br>
PGK1 or FBA1 which, when the endogenous gene(s) encoding these proteins are deleted<br>
or inactivated in a host cell, do not result hi the host cell developing an auxotrophic<br>
(biosynthetic) requirement.<br>
Accordingly, in a fourth aspect, the present invention also provides a host cell comprising<br>
a plasrnid (such as a plasrnid according to any of the first, second or third aspects of the<br>
invention), the plasrnid comprising a gene that encodes an essential chaperone wherein,<br>
in the absence of the plasmid, the host cell is unable to produce the chaperone.<br>
Preferably, in the absence of the plasmid, the host cell is inviable. The host cell may<br>
further comprise a recombinant gene encoding a heterologous (or homologous, in the<br>
sense that the recombinant gene encodes a protein identical in sequence to a protein<br>
encoded by the host cell) protein, such as those described above in respect of earlier<br>
aspects of the invention.<br>
The present invention also provides, in a fifth aspect, a plasrnid comprising, as the sole<br>
selectable marker, a gene encoding an essential chaperone. The plasmid may further<br>
comprise a gene encoding a heterologous; protein. The plasmid ma)-' be a 2urn-famihr<br>
plasmid and is preferably a plasmid according to any of the first, second or third aspects<br>
of the invention.<br>
The present invention also provides, in a sixth aspect a method for producing a<br>
lieterologous protein comprising the steps of: providing a host cell comprising a plasmid,<br>
the plasmid comprising a gene that encodes an essential chaperone wherein., in the<br>
absence of the plasmid, the host cell is unable to produce the chaperone and wherein the<br>
host cell further comprises a recombinant gene encoding a heterologous protein;<br>
culturing the host cell in a culture medium under conditions that allow the expression of<br>
the essential chaperone and the heterologous protein; and optionally purifying the thus<br>
expressed heterologous protein from the cultured host cell or the culture medium; and<br>
further optionally, lyopMlising the thus purified protein.<br>
The method ma}' further comprise the step of formulating the purified heterologous<br>
protein with a carrier or diluent and optional!}7 presenting the thus formulated protein in a<br>
unit dosage form, in the manner discussed above. In one preferred embodiment, the<br>
method involves culturing the host cell in non-selective media, such as a rich media.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
Figure 1 shows a plasmid map of the 2 urn plasmid.<br>
Figure 2 shows a plasmid map of pSAC35.<br>
Figure 3 shows some exemplified FLP insertion sites.<br>
Figures 4 to 8,10,11,13 to 32, 36 to 42, 44 to 46, 4S to 54 and 57 to 76 show maps of<br>
various plasmids.<br>
Figure 9 shows the DNA fragment from pDB2429 containing the PD11 gene.<br>
Figure 12 shows some exemplified REP2 insertion sites,<br>
Figure 33 shows table 3 as referred to in the Examples.<br>
Figure 34 shows the sequence of SEQ ID NO: 1.<br>
Figure 35 shows the sequence of SEQ ID NO: 2.<br>
Figure 43 shows the sequence of PCR primers DS248 andDS250.<br>
Figure 47 shows plasmid stabilities with increasing number of generation growth in nonselective<br>
liquid culture for S. cerevisiae containing various pSAC35-derived plasmids.<br>
Figure 55 shows the results of RIB. lOmL YEPD shake flasks were inoculated with<br>
DXY1 trplA [pDB2976], DXY1 trplA [pDB2977], DXY1 tiplA [pDB2978], DXY1<br>
trplA £pDB2979], DXY1 ti-plA [pDB2980] or DXY1 trplA [pDB2981] transformed to<br>
tryptophan prototrophy with a 1.41kb Notl/Pstl pdil::TKPl disrupting DNA fragment<br>
was isolated from pDB3078. Transformants were grown for 4-days at 30°C, 200rpm.<br>
4jiL culture supernatant loaded per well of a rocket immunoelectrophoresis gel (Weeke,<br>
B. 1976. Rocket immunoelectrophoresis. In N. H. Azelsen, J. Kroll, and B. Weeke [eds.],<br>
A manual of quantitative immunoelectrophoresis. Methods and applications.<br>
Universitetsforlaget, Oslo, Norway). rHA standards concentrations are in p.g/mL. 700uL<br>
Precipin was stained with Uoomassie blue. isolates seiecieu iur mruicr auaiysiis are<br>
indicated (*)•<br>
Figure 56 shows the results of RJE. lOmL YEPD shake flasks were inoculated with<br>
DXY1 [pDB2244], DXY1 [pDB2976], DXY1 trplA pdil;:TRPl [pDB2976], DXY1<br>
[pDB2978], DXY1 trplA pdil::TRPl [pDB2978], DXY1 [pDB2980]5 DXY1 trplA<br>
pdil::TRPl [pDB2980], DXY1 [r,DB2977], DXY1 ti-plA pdil::TRPl \pDB2971],<br>
DXY1 [pDB2979] DXY1 trplApdil::TKPl [pDB2979], DXY1 [pDB2981] and DXY1<br>
trplApdil::TRPl [pDB2981]3 and were grown for 4-da}'s at 30°C, 200ipm. 4(j,L culture<br>
 supernatant loaded per well of a rocket ummmoelectrophoresis gel. rHA standards<br>
concentrations are in ug/mL. SOOuL goai anti-HA (Sigma product A-l 151 resuspended<br>
in 5mL water) /50mL agarose. Precipm was stained with Coomassie blue. Isolates<br>
selected for furthei analysis are indicated (*)<br>
EXAMPLES<br>
These example describes the insertion of additional DNA sequences into a number of<br>
positions, defined by restriction endonuclease sites, within the US-region of a 2jj.rnfamily<br>
plasmid, of the type shown hi Figure 2 and generally designated pSAC355 which<br>
includes a |3-lactarnase gene (for ampicillin resistance, which is lost from the plasmid<br>
following transformation into 3?east), a LEU2 selectable marker and an oligonucleotide<br>
linlcer, the latter two of which are inserted into a unique SnaBI-site within the UL-region<br>
of the 2}4.m-farrhly disintegration vector, pSACS (see EP 0 286 424). The sites chosen<br>
were towards the 3'-ends of the REP2 and FLP coding regions or in the downstream<br>
inverted repeat sequences. Short synthetic DNA linkers were inserted into each site, and<br>
the relative stabilities of the modified plasmids were compared during growth on nonselective<br>
media. Preferred sites for DNA insertions were identified. Insertion of larger<br>
DNA fragments containing "a gene of interest" was demonstrated b}^ inserting a DNA<br>
fragment containing the PDJ1 gene into the Ac/nl-site after REP2.<br>
EXA1S1PLE 1<br>
Insertion of Synthetic DNA Linker into Xcml-Sites in the Small Unique Region of<br>
pSACSS<br>
Sites assessed initially for insertion of additional DNA into the US-region of pSAC35.<br>
were the JLcml-sites in the 599-bp inverted repeats. One JTcml-site cuts 51-bp after the<br>
REP2 translation termination codon, whereas the other Xcml-site cuts 127-bp before the<br>
end of the FLP coding sequence, due to overlap with the inverted repeat (see Figure 3).<br>
The sequence inserted was a 52-bp linker made by annealing
oligonucleotides CF86 and CF87. This DNA linker contained a core region "SndBIPacI-<br>
Fsel/Sfil-Smal-SnaBl", which encoded restriction sites absent from pSAC35.<br>
Xcml Linker rCF86+CF87)<br>
Sfil<br>
Pad SnaBI<br>
SnaBI Fsal Smal<br>
CF86 GGAGTGGTA CGTATTAATT AAGGCCGGCC AGGCCCGGGT ACGTACCAAT TGA<br>
CF87 TCCTCACCAT GCATAATTAA TTCCGGCCGG TCCGGGCCCA TGCATGGTTA AC<br>
Plasmid pSAC35 was partially digested with Xcml, the linear 11-lcb fragment was<br>
isolated from a 0.7%(w/v) agarose gel, ligated with the CFS6/CFS7Xcml linker (neat, 10"<br>
1 and 10"2 dilutions) and transformed into E. coll DH5a. Ampicillin resistant<br>
transformants were selected and screened for the presence of plasmids that could be<br>
linearised by Smal digestion.v Restriction enzyme analysis identified pDB2688 (Figure 4)<br>
with the linker cloned into the Jfowl-site after REP2. DNA sequencing using<br>
oligonucleotides primers CF88, CF98 and CF99 (Table 1) confirmed the insertion<br>
contained the correct linker sequence.<br>
Table 1: Oligonucleotide sequencing primers: <br>
(Table Removed)<br>
Restriction enzyme anafysis also identified pDB2689 (Figure 5), with the linker cloned<br>
into the Xcml-silz in the PLP gene. However, the. linlcer in pDB2689 was sho^vn by DNA<br>
sequencing using primers CF90 and CF91 to have a missing G:C hase-pair within, the<br>
FseUSfil site (marked above in bold in the CFS6-J-CFS7 linker). This generated a coding<br>
sequence for a mutant Pip-protein, with 39 C-terminal amino acid residues replaced loy 56<br>
different amino acids before the translation termination codon.<br>
The missing base-pair in the pDB2689 linker sequence was corrected to produce<br>
pDB27S6 (Figure 6). To achieve this, a 31-bp 5'-phosphoiylated Sjia'BI-lmk&amp;i was made<br>
67<br>
from oligonucleotides CF104 and CF105. This was ligated into the SnaBl site of<br>
pDB2689, which had previously been treated with calf intestinal alkaline phosphatase.<br>
DMA sequencing with primers CF90, CF91, CF100 and CF101 confirmed the correct<br>
DNA linker sequence in pDB2786. This generated a coding sequence for a mutant Flpprotein,<br>
with 39 C-temoinal residues replaced by 14 different residues before translation<br>
termination.<br>
SndBI Linker rCF104+CF105)<br>
Sfil<br>
Fsel<br>
Pad Smal<br>
CF104 P1-GTA.TTAATTA AGGCC6GCCA GGCCCGGGTA C<br>
CF105 CATAATTAAT TCCGGCCGGT CCGGGCCCAT G-Pi<br>
An. additional plasmid, pDB2798 (Figure 7), was also produced by ligation of the<br>
linker in the opposite direction to pDB2786. The linker sequence in pDB2798 was<br>
coiafirmed by 33NA sequencing. Plasmid pDB2798 contained a coding sequence for a<br>
mutant Flp-protein, with 39 C-terminal residues replaced by 8 different residues before<br>
translation termination.<br>
A linker was also cloned into the JTcml-site in the FLP gene to truncate the Flp protein at<br>
the site of insertion. The linker used was a 45 -bp 5'-phosphorylated JTeml-linker made<br>
from oligonucleotides CF120 and CF121.<br>
Xcml Linker rCF120+CF121)<br>
Sfil<br>
Pad SnaBI<br>
SnaBI Fsel Smal<br>
CF120 Pi-GTAATAATA CGTATTAATT AAGGCCGGCC AGGCCCGGGT ACGTAA<br>
CF121 TCATTATTAT GCATAATTAA TTCCGGCCGG TCCGGGCCCA TGCAT-Pi<br>
This CF120/CF121 Xcml linker was ligated with 11-kb pSAC35 fragments produced by<br>
partial digestion with Xcml, followed by treatment with calf intestinal alkaline<br>
phosphatase. Analysis of ampicilHn resistant K coli DH5a transformants identified<br>
:lons£ containing pDB2823 (Figure 8). DMA sequencing with primers CF90. CFP1.<br>
CF100 and CF]f)3 confirmed the linker sequence in pDB2823. Translaiion temiination<br>
witlini the linlcer inserted would result in the production of Flp (1-382), which lacked 41<br>
C-tenninal residues.<br>
The impact on plasmid stability from insertion of linlcer sequences into the J£c7j?I-sites<br>
within the US-region of pSAC35 was assessed for pDB2688 and pDB26S9. Plasmid<br>
stability' was determined in a S. cerevisiae strain by loss of the LEU2 marker during nonselective<br>
grown, on YEPS. The same yeast strain, transformed with pSAC35, which is<br>
structuraUy similar to pSAC3, but contains additional DNA inserted at the SndQl site that<br>
contained &amp;LEU2 selectable marker (Chinery &amp; Hinchliffe, 1989, Cwr. Genet., 16, 21),<br>
was used as the control.<br>
The yeast strain was transformed to leucine prototrophy using a modified lithium acetate<br>
method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et al, 1983, J.<br>
BacferioL, 153, 163; Bible, 1992. Bioteclmiques, 13, IS)). Transformants were selected<br>
on BMMD-agar plates, and were subsequent!}' patched out on BMMD-agar plates.<br>
Cryopreserved trehalose stocks were prepared from lOmL BMMD shake flask cultures<br>
(24hrs530°C,20Qipm).<br>
The composition of YEPD and BMMD is described by Sleep et a!., 2002, Yeast IB, 403.<br>
YEPS and BMMS are similar in composition to YEPD and BMMD accept that 2% (w/v)<br>
sucrose was substituted for the 2% (w/v) glucose as the sole initial carbon source.<br>
For the determination of plasmid stability a ImL Cryopreserved stock was thawed and<br>
inoculated into lOOinL YEPS (initial OD6oo ~ 0.04-0.09) in a 250mL conical flask and<br>
grow for approximately 72 hours (70-74 hrs) at 30DC in an orbital shaker (200 rpm,<br>
Innova 4300 incubator shaker. New Brunswick Scientific).<br>
Samples were removed from each flask, diluted in YEPS-broth (10~2 to ID"3 dilution), and<br>
lOOuL aliquots plated in duplicate onto YEPS-agar plates. Cells were grown at 30°C for<br>
3-4 days to allow single colonies to develop. For each yeast stock analysed, 100 random<br>
colonies were patched in replica onto BMMS-agar plates followed by YEPS-agar plates.<br>
After growth at 30°C for 3-4 days the percentage of colonies growing on both. BMMSagar<br>
plates and YEPS-agar plates was determined as the measure of plasmid stability.<br>
In the above analysis to measure the loss of the LEU2 marker from transformants,<br>
pSAC35 and pDB2688 appeared to be 100% stable, whereas pDB26S9 was 72% stable.<br>
Hence, insertion of the linker into the Xcml-site after £EP2 had no apparent effect on<br>
plasmid stability, despite altering the transcribed sequence and disrupting the homology<br>
between the 599-bp inverted repeats. Insertion of the linker at the Xcml-site in FLP also<br>
resulted in a surprisingly stable plasmid, despite both disruption of the inverted repeat<br>
and mutation of the Flp protein.<br>
EXAMPLE 2<br>
Insertion ofthePDIl Gene, into the XanILinker ofpDB2688<br>
The insertion of a large DNA fragment into the US-region of 2(.un-lilce vectors was<br>
demonstrated by cloning the S. cerevisiae PDI1 gene into the A'cml-linker of pDB2688.<br>
The PDI1 gene (Figure 9) was cloned on a 1.9-kb SacI-Spel fragment from a larger S.<br>
cerevisiae SKQ2n genomic DNA fragment containing the PDI1 gene (as provided in the<br>
plasmid pMA3a:C7 that is described in US 6,291,205 and also described as Clone C7 in<br>
Crouzet &amp; Tuite, 1987, Mol Gen. Genet, 210, 581-583 and Farquhar et al, 1991, supra),<br>
which had been cloned into YIplac211 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534) and<br>
had a synthetic DNA linker containing a Sad restriction site inserted at a unique Bsit3 61-<br>
site in the 3' untranslated region of the PDI1 gene. The 1.9-kb Sacl-Spel fragment was<br>
treated with T4 DNA polymerase to fill the Spel 5'-overhang and remove the iSizcI 3'-<br>
overhang. This PDI1 fragment included 212-bp of the PDI1 promoter upstream of the<br>
translation initiation codon, and 148-bp downstream of the translation termination codon.<br>
This was ligated with Smal linearised/calf intestinal alkaline phosphatase treated<br>
pDB2688, to create plasmid pDB2690 (Figure 10), with the PDI1 gene transcribed in the<br>
same direction as REP2. A S. cerevisiae strain was transformed to leucine prototrophy<br>
with pDB2690.<br>
An expression cassette for a human transferrin mutant {'N413Q. N6I1Q) was<br>
subsequently cloned into the jVo/I-site of pDB2690 to create pDB27J I (Figure 1 ]). The<br>
expression cassette in pDB2711 contains the S. cerevisiae PRB] promoter, an HSA/MFa<br>
fusion leader sequence (EP 387319; Sleep el al 1990, Bio1eclmolog)&gt; (N.Y.), 8, 42)<br>
followed by a coding sequence for the human transferrin mutant (N413Q, N611Q) and<br>
the S. cerevisiae ADHJ terminator. Plasmid pDB253b (Figure 36) was constructed<br>
similarly by insertion of the same expression cassette into the Nofl-site of pSAC35.<br>
The advantage of inserting "genes of interest" into the US-region of 2jam-vectors was<br>
demonstrated by the approximate 7-fold increase in recombinant transferrin N413Q,<br>
N611Q secretion during fermentation of )feast transformed with pDB2711, compared to<br>
the same yeast transformed with pDB2536. An approximate 15-fold increase in<br>
recombinant transfenin N413Q, N611Q secretion was observed in shake flask culture<br>
(data not shown).<br>
The relative stabilities of plasmids pDB2688, pDB2690a pDB2711, PDB2536 and<br>
pSAC35 were determined in the same yeast strain grown in YEPS media, using the<br>
method described above (Table 2).<br>
In this analysis, pDB2690 was 32% stable, compared to 100% stability for pDB2688<br>
without the PD11 insert. This decrease in plasmid stability was less than the decrease in<br>
plasmid stability observed with pDB2536, due to insertion of the.rTF (N413Q, N611Q)<br>
expression cassette into the JVorl-site within the large unique region of pSAC35 (Table 2).<br>
Furthermore, selective growth in minimal media during high cell density fennentations<br>
could overcome the increased plasmid instability due to the PDI1 insertion observed in<br>
YEPS medium, as the rTF (N413Q, N611Q) 37ield from the same yeast transformed with<br>
pDB2711 did not decrease compared to that achieved from the same yeast transformed<br>
with pDB2536.<br>
Table 2: 
region of pS ACS 5. Data from 3 days growth in non-selective shake flask culture before<br>
plating on YEPS-agar.<br>
(Table Removed)<br>
EXAMPLES<br>
Insertion of DNA Linkers into the JREP2 Gene and Downstream Sequences in the<br>
Inverted Repeat ofpSA C35<br>
To define the useful limits for insertion of additional DNA into the REP2 gene and<br>
sequences in the inverted repeat downstream of it, further linkers were inserted into<br>
pSAC35. Figure 12 indicates the restriction sites used for these insertions and the effects<br>
on the Rep2 protein of translation termination at these sites.<br>
The linker inserted at the Xmnl-site in REP2 was a 44-bp sequence made from<br>
oligonucleotides CF108 and CF109.<br>
.Ymrcl Linlcer fCFl 08+CF1 09)<br>
Pad<br>
SnaEI Fsel Swal<br>
CF10E: ATAATAATAC GTATTAATTA AGGCCGGCCA GGCCCGGGTA CGTA<br>
CF109 TATTATTATG CATAATTAAT TCCGGCCGGT CCGGGCCCAT GCAT<br>
To avoid insertion into other Amjd-sites in pSAC35. the 3,076-bp Xbal fragment from<br>
pSAC35 that contained the REP2 and FLP genes was first sub-cloned into the E. call<br>
clearing -vector pDB2685 (Figure. 13) to produce pDB2783 (Figure 14).<br>
Plasmid pDB2685 is a pU CIS-like cloning vector derived from pCF17 containing<br>
apramycin resistance gene aac(3)IV from Klebsiella pneumonias (Rao el al, 1983.,<br>
Antimicrob. Agents Chemother., 24, 689) and multiple cloning site from pMCSS<br>
(Hoheisel, 1994, Biotechmques, 17, 456). pCF17 was made from pIJ8600 (Sun et al,<br>
1999, Microbiology, 145(9), 2221-7) by digestion with EcoRl, Nliel and the Klenow..<br>
fragment of DNA polymerase I. and self-ligation, followed by isolation from the reaction<br>
products by transformation of competent E. col: DH5a cells and selection with<br>
apramycin sulphate. Plasmid pDB2685 was constructed by cloning a 439bp Sspl-Swal<br>
fragment from pMCSS into pCF17, wliich had been cut with Msd and treated with calf<br>
intestinal alkaline phosphatase. Blue/white selection is not dependant on IPTG induction.<br>
Plasmid pDB2783 was linearised with AM and ligated with the CF108/CF109 Xmnllinker<br>
to produce pDB2799 (Figure 15) and pDB2780 (not shown). Plasmid pDB2799<br>
contained the CF108/CF109 Xmnl linlcer in me correct orientation for translation<br>
tennination at the insertion site to produce Rep2 (1-244), whereas pDB27SO contained<br>
the linlcer cloned in the opposite orientation. DNA sequencing with primers CF98 and<br>
CF99 confirmed the correct linlcer sequences.<br>
The 3,120bp Xbal fragment from pDB2799 was subsequently ligated with a 7,961-bp<br>
pSAC35 fragment which had been produced by partial Xbal digestion and treatment with<br>
calf intestinal alkaline phosphatase, to create plasmid pDB2817 (B-foim) and pDB2818<br>
(A-form) disintegration vectors ("Figures 16 and 17 respecth^ely).<br>
Insertion of linkers at the Apal-site in pSAC35 was performed with and without 3'-5'<br>
exonuclease digestion by T4 DNA polymerase. This produced coding sequences for<br>
either Rep2 (1-271) or Rep2 (1-269) before translation termination. In the following<br>
figure, the sequence GGCC marked with diagonal lines was deleted from the 3 '-overhang<br>
produced after Apal digestion resulting in removal of nucleotides from the codons for<br>
Glycine-170 (GGC) andProline-171.<br>
Thr He Thr GIu<br>
ACCATCACT<br>
TGGTAGTGA<br>
Apal<br>
The linker inserted at the Apal-site without exonuclease digestion was a 50-bp<br>
5'-phosphorylated linker made from oligonucleotides CF116 and CF117.<br>
^pal-Linker fCF116+CFl 17)<br>
Sfil<br>
Pad SnaBI<br>
SnaEI Fsel Smal<br>
CF116 Pi-CTTAAT AATACGTATT AATTAAGGCC GGCCAGGCCC GGGTACGTAG GGCC<br>
CF117 CCGGGAATTA TTATGCATAA TTAATTCCGG CCGGTCCGGG CCCATGCATC-Pi<br>
This was ligated with pSACSS, which, had been linearised with Apal and treated with calf<br>
intestinal alkaline phosphatase, to produce pDB2788 (Figure 18) and pDB2789 (not<br>
shown). Within pDB278S, the linker was in the correct orientation for translation<br>
termination after proline-271, whereas hi pDB2789 the linker was in the opposite<br>
orientation.<br>
The linker inserted at the Apal-site with exonuclease digestion by T4 DNA polymerase<br>
was a 43-bp 5'-phosphorylated linker made from oligonucleotides CF106 and CF107,<br>
which was called the core termination linker.<br>
Core. Termination-Linker CCF106+CF107')<br>
Sill<br>
Ps cl S.naB:<br>
SnaEI Fsei Smal<br>
CF106 Pi-TAATAATACG TATTAATTAA GGCCGGCCAG GCCCGGGTAC GTA<br>
CF107 ATTATTATGC ATAATTAATT CCGGCCGGTC CGGGCCCATG CAT-Pi<br>
The core termination linker was ligaied with pSAC.35, which had been linearised with<br>
Apal, digested with T4 DNA polymerase and treated with calf intestinal alkaline<br>
phosphatase. This ligation produced pDB2787 (Figure 19) with the linker cloned in the<br>
correct orientation for translation termination after glutamate-269.<br>
The correct DNA sequences were confirmed in all clones containing the Apdl-\ink.Krs,<br>
using oligonucleotide primers CF98 and CF99.<br>
The core termination linker (CF106-+CF107) was also used for insertion into the pisites<br>
of pDB2783 (Figure 14), The core terrnination linker (CF106-I-CF107) was ligated<br>
into pDB2783 linearised by partial Fspl digestion, which had been treated with calf<br>
intestinal alkaline phosphatase. Plasmids isolated from apramycin resistant E. coll DH5a<br>
transformants were screened by digestion with Fspl, and selected clones were sequenced<br>
with Ml3 forward and reverse primers.<br>
Plasrnid pDB2801 (not shown) was identified containing two copies of the linker cloned<br>
in the ccorect orientation (with the Pad-site nearest the REP 2 gene). The extra cop}' of<br>
the linker was subsequent!}7 removed b}; first deleting a 116-bp Nrnl-Hpal fragment<br>
containing an T^el-site from the multiple cloning site region, followed by digestion with<br>
Fsel and re-ligation to produce pDB2802 (Figure 20). DNA sequencing using<br>
oligonucleotide CF126 confirmed the correct linker sequence.<br>
The 3,119-bp pDB2802 Xbal fragment was subsequently ligated with a 7,961-bp<br>
pSAC35 fragment produced by partial Xbal digestion and treatment with calf intestinal<br>
alkaline phosphatase to create pDB2805 (B-fonn) and pDB2806 (A-form) disintegration<br>
vectors (Figures 21 and 22, respectively)-<br>
EXAMPLE 4<br>
Insertion of DNA Linkers into the FLP Gene and Downstream Sequences in the<br>
Inverted Repeat ofpSAC35<br>
DNA linkers were inserted into pSAC35 to define the useful limits for insertion of<br>
additional DNA into the FLP gene and sequences downstream in the inverted repeat.<br>
Figure 3 indicates the restriction sites used for these insertions and the affects on the Flp<br>
protein of translation teamination at these sites.<br>
The linker inserted at the JBc/I-site was a 49-bp 5'-phosphorylated linker made from<br>
oligonucleotides CF118 and CF119.<br>
ffcfl Linker (CF118+CF119)<br>
Sfil<br>
Pad SnaBI<br>
S.naBI Fsel Smal<br>
CF118 Pi-GATCACTAATAATACGTATTAATTAAGGCCGGCCAGGCCCGGGTACGTA<br>
CF119 TGATTATTATGCATAATTAATTCCGGCCGGTCCGGGCCCATGCATCTAG-Pi<br>
Due to Dam-methylation of the .BcZI-site rn pSAC35, the 5c/I-linker was cloned into nonmethylated<br>
pSAC35 DNA, which had been isolated from the K coli straia ET12567<br>
pUZ8002 (MacNeil et al, 1992, Gene, 111, 61; Kieser et al, 2000, Practical Streptomyces<br>
Genetics, The John Innes Foundation, Norwich). Plasmid pSAC35 was linearised with<br>
Bell, treated with calf intestinal alkaline phosphatase, and ligated with the 5c/l-linker to<br>
create pDB2816 (Figure 23). DNA sequencing with oligonucleotide primers CF91 and<br>
CF100 showed that three copies of the .Sc/I-lmker were present in pDB2816, which were<br>
all in the correct orientation for translational termination of Flp after histidine-353.<br>
Digestion of pDB2816 with Pad followed by self-ligation, was performed to produce<br>
pDB2814 and pDB2815, containing one and two copies of the .Bc/I-linker respectively<br>
(Figures 24 and 25). The DNA sequences of the linkers were confirmed using primers<br>
CF.9] and CF100. In S. cercvisiae a truncated Fip (1-353) protein will be produced by<br>
yeast transformed with pDB2814, pDB2815 or pDB2Sl 6.<br>
An additional plasmid pDB2846 (data not shown) was also produced by ligation of a<br>
single copy of the JJc/l-linker hi the opposite orientation to pDB2814. This has the<br>
coding sequence for the first 352-residues from Flp followed by 14 different residues<br>
before translation termination.<br>
The linker inserted at the Hgal-site, was a 47-bp 5 '-phosphorylated linker made from<br>
oligonucleotides CF114 and CF115.<br>
JfegJ Linker CCF114+CF115)<br>
Sfil<br>
Pad SnaBI<br>
5naBI Fsel Smal<br>
CF114 Pi-AGTACTATAAIACGTATTAATTAAGGCCGGCCAGGCCCGGGTACGTA<br>
CF115 ATATTATGCATAATTAATTCCGGCCGGTCCGGGCCCATGCATTCATG-Pi<br>
The Hgal-linker was ligated with pDB27S3, which had been linearised b}' partial Hgal<br>
digestion and treated with calf intestinal allcalrne phosphatase. to create pDB2811 (Figure<br>
26). DNA sequencing with oh'gonucleotides CF90, CF91 and CF100 confirmed the<br>
correct hnker insertion.<br>
The 3,123-bp Xbal fi-agment from pDB2Sl 3 was subsequently ligated with the 7,961-bp<br>
pSAC35 fragment, produced by partial Xbal digestion and treatment with calf intestinal<br>
alkaline phosphatase to produce pDB2S12 (B-form) and pDB2S13 (A-form)<br>
disintegration vectors containing DNA inserted at the Hgal-site (Figures 27 and 28<br>
respectively).<br>
Plasmids pDB2S03 and pDB2S04 (Figures 29 and 30, respectively) with the core<br>
termination linker (CF106+CF107) inserted at the Fspl after FLP, were isolated by the<br>
same method used to construct pDB2801. The correct liiiker insertions were confirmed<br>
by DNA sequencing, Plasmid pDB2S04 contained the linker inserted in the correct<br>
orientation (with the Pad-site closest to the FLP gene), whereas pDB2803 contained the<br>
linker in the opposite orientation.<br>
The pDB2804 3,119-bp Xbal fragment was ligated with the 7,961-bp pSAC35 fragment<br>
produced by partial Xbal digestion and treatment with calf intestinal alkaline phosphatase<br>
to create pDB2807 (B-form) and pDB2808 (A-fornf) disintegration vectors containing<br>
DNA inserted at the Fspl-site after FLP (Figures 31 and 32 respectively).<br>
EXAMPLE 5<br>
Relative Stabilities of the LEU2 Marker in Yeast Transformed with pSACS 5-Like<br>
Plasmids Containing DNA Linkers Inserted into the Small Unique Region and<br>
Inverted Repeats<br>
A S. cerevisiae strain was transformed with the pSAC35-like plasmids containing DNA<br>
linkers inserted into the US-region and inverted repeats. Cryopreserved trehalose stocks<br>
were prepared for testing plasmid stabilities (Table 3). Plasmid stabilities were analysed<br>
as described above for linkers inserted at the Xcml-sites in pSAC35. Duplicate flasks<br>
were set up for each insertion site analysed. In addition, to the analysis of colonies<br>
derived from cells after 3-days in shake flake culture, colonies were grown and analysed<br>
from cells with a further 4-days shake flask culture. For this, lOOuL samples were<br>
removed from each 3-day old flask and sub-cultured in lOOmL YEPS broth for a further<br>
period of approximately 96 hours (94-98 hrs) at 30.0°C in an orbital shaker, after which<br>
single colonies were obtained and analysed for loss of the LEU2 marker. In this case<br>
analysis was resticted to a single flask from selected strains, for which 50 colonies were<br>
picked. The overall results are summarised in Table 4.<br>
Table 4: Summary ofplasmid stability7 daia for DNA insertions into pSAC35<br>
Set 1 represents data from 3 days in non-selective shake flask culture,<br>
Set 2 represents data from 1 days in non-selective shake flask culture.<br>
A) REP2 Insertion Sites<br>
(Table Removed)<br>
All of the modified pSAC35 plasmids were able to transform }'east to leucine<br>
prototrophy, indicating that despite the additional DNA inserted within the functionally<br>
crowded regions of 2u.m DNA, all could replicate and partition in S. cerevisiae. This<br>
applied to plasmids with 43-52 base-pair linkers inserted at all the sites in the 2u,m USregion,<br>
as well as the larger DNA insertion containing the PDI1 gene.<br>
For the linker insertion sites, data was reproducible between "both experiments and<br>
duplicates. All sites outside REP2 or FLP open reading frames, but within inverted<br>
repeats appeared to be 100% stable under the test conditions use;d. Plasmid instability<br>
(i.e. plasmid loss) was observed for linkers inserted into sites within the REP2 or FLP<br>
open reading frames. The observed plasmid instability of REP2 insertions was greater<br>
than for FLP insertions. For the REP2 insertions, loss of the LEU2 marker continued<br>
with the extended growth period in non-selective media, whereas there was little<br>
difference for the FLP insertions.<br>
Insertions into the REP2 gene produced Rep2 polypeptides truncated within a region<br>
known to function in self-association and binding to the STB-locus of 2jim (Sengupta et<br>
al, 2001, J. Bacterial., 183, 2306).<br>
Insertions into the FLP gene resulted in truncated Flp proteins. All the insertion sites<br>
were after tyrosine-343 in the C-terminal domain, which is essential for correct<br>
functioning of the Flp protein (Prasad et al, 1987, Proc. Nati Acad. Set. U.S.A., 84,2189;<br>
Chen etal, 1992, Cell, 69, 647; Grainge et al, 2001, J. Mol. BioL, 314, 717).<br>
None of the insertions into the inverted repeat regions resulted in plasmid instability<br>
being detected, except for the insertion into the FLP Xcml-site, which also truncated the<br>
Flp protein product. The insertions at the Fspl-sites in the inverted repeat regions were<br>
pSAC35-lilce plasmids have been constructed with 43-52 base-pair DNA linkers inserted<br>
into the REP2 open reading frame, or the FLP open reading frame or the inverted repeat<br>
sequences, hi-addition, a 1.9-kb DNA fragment containing the PJDI2 gene was inserted<br>
into a DNA linker at the Jfcml-site after REP2.<br>
All of the pSAC35-like vectors with additional DNA inserted were able to transform<br>
yeast to leucine prototrophy. Therefore, despite inserting DNA into functionally crowded<br>
regions of 2|im plasmid DNA, the plasmid replication and partitioning mechanisms had<br>
not been abolished.<br>
Detennination of plasmid stability7 by measuring loss of the LEU2 selectable marker<br>
during growth in non-selective medium indicated mat inserting DNA linkers into the<br>
inverted repeats had not destabilised the plasmid, whereas plasmid stability had been<br>
reduced by insertions into the REP2 and FLP open reading frames. However, despite a<br>
reduction in plasmid stability under non-selective media growth conditions when<br>
insertions were made into the REP 2- and FLP open reading frames at some positions<br>
defined by the first and second aspects of the invention, the resulting plasmid<br>
nevertheless has a sufficiently lugh stability for use in yeast when grown on selective<br>
media.<br>
EXAMPLE 6<br>
Insertion of DNA Sequences Immediately after the REP2 Gene in the Small Unique<br>
Region ofpSAC35<br>
To further define the useful limits for insertion of additional DNA into the REP2 gene<br>
and sequences hi the inverted repeat downstream of it, a synthetic DNA linker was<br>
inserted into pSAC35 immediately after the REP2 translation termination codon (TGA).<br>
As there were no naturally occurring restriction endonuclease sites conveniently located<br>
immediately after the REP2 coding sequence in 2urn (or pSAC35), a SwaBl-site was.<br>
introduced at this position by oligonucleotide directed mutagenesis. The pSAC35<br>
derivative with a unique SnaBI-site immediately downstream of REP2 was named<br>
pDB2938 (Figure 37). In pDB2938, the end of the inverted repeat was displaced from<br>
the rest of the inverted repeat by insertion of the iSnoBI-site. pDB2954 (Figure 38) was<br>
subsequently constructed with a 31-bp sequence identical to the Sno5I-linker made from<br>
oligonucleotides CF104 and CF105 (supra] inserted into the unique SnaBI site of<br>
pDB293S. such mat the order of restriction endonuclease sites located immediate]}'- after<br>
the TGA translation termination codon ofREP2 was SndBl-Pacl-FsellSfil-Smdl.-SnaB'l.<br>
To construct pDB2938, the 1,085-bp Ncol-BanKL fragment from. pDB27S3 (Figure 14)<br>
was first sub-cloned into pMCSS (Hoheisel, 1994, Biolechniques, 17, 456), which had<br>
81<br>
been digested with Ncol, BamHI and calf intestinal alkaline phosphatase. This produced<br>
pDB2809 (Figure 39), which was subsequent!}' mutated using oligonucleotides CF127<br>
and CF128, to generate pDB2920 (Figure. 40).<br>
The 51-bp mutagenic oligonucleotides CF127 and CF128<br>
The SndBl recognition sequence is underlined<br>
CF127 5 ' -CGTAATACTTCTAGGGTATGATACGTATCCAATATCAAAGGAAATGATAGC-3 '<br>
CF12B 5 ' -GCTATCATTTCCTTTGATATTGGATACGTATCATACCCTAGAAGTATTACG-3 '<br>
Oligonucleotide directed mutagenesis was performed according to the instruction manual<br>
of the Statagene's QuiclcChange™ Site-Directed Mutagenesis Kit. SndBl and Hindis.<br>
restriction digestion of plasmid DNA was used to identify the arnpicillin resistant E. coli<br>
transformants that contained pDB2920. The inserted 6-bp sequence of the SndBl<br>
restriction site and the correct DNA sequence for the entire 1,091-bp Ncol-BamHL<br>
fragment was confirmed in pDB2920 by DNA sequencing using oligonucleotide primers<br>
CF98, CF99, CF129, CF130, CF131 and M13 forward and reverse primers (Table 1).<br>
The 1,091-bp Ncol-BamHl fragment from pDB2920 was isolated by agarose gel<br>
purification and ligated with the approximately 4.7-kb Ncol-BamHl fragment from<br>
pDB2783 to produce pDB2936 (Figure 41). The pDB2783 4.7-kb Ncol-BamHL fragment<br>
was isolated by complete BamHL digestion of pDB2783 DNA that had first been<br>
linearised by partial digestion with Ncol and purified by agarose gel electrophoresis. E.<br>
coli DHScc cells were transformed to apramycin resistance by the ligation products.<br>
pDB2936 was identified by SndBl digestion of plasmid DNA isolated from the<br>
apramycin resistant clones.<br>
The 3,082-bp Xbal fragment from pDB2936 was subsequently ligated with a 7,961-bp<br>
pSAC35 fragment, which had been produced by partial .So! digestion and treatment with<br>
calf intestinal alkaline phosphatase, to create the disintegration vector pDB2938 (2pm Bform,<br>
Figure 37)<br>
pDB2938 was digested with SndBl and calf intestinal phosphatase and ligated with an<br>
approximately 2-lcb SndBl fragment from pDB2939 (Figure 42). pDB2939 was produced<br>
by PCR amplifying the PD11 gene from S. ccrcvi.si.ac S2SSc genomic DNA v^ith,<br>
olieoiiucleotide pruners DS24S and DS250 (Figure 43). followed by digesting the PCR<br>
products with EcoRl and i'^mHI and cloning the approximately 1.98-l:b fragment into<br>
YIplac211 (Gietz &amp; Sugino, 1988, Gene, 14, 527-534), that had been cut with EcoRl and<br>
Bamlll. DNA sequencing of pDB2939 identified a missing "G' from within the DS24S<br>
sequence, winch is marked in bold in Figure 43. The approximately 2-kb SnaBl fragment<br>
from pDB2939 was subsequently cloned into the mn'que SnaBl-site of pDB2938 to<br>
produce plasmid pDB2950 (Figure 44'). The PDI1 gene in pDB2950 is transcribed in -the<br>
same direction as the REP2 gene.<br>
pDB2950 was subsequently digested with Smal and the approximately 11.1-lcb DNTA<br>
fragment was circularised to delete the S28Sc PDIJ sequence, This produced plasmid<br>
pDB2954 (Figure 38) with the SnaBI-Pacl-FseUSfil-Smd[-SnaBI linker located<br>
immediately after the TGA translation termination codon of KEP2.<br>
hi addition to cloning the S. cerevlsiae S288c PDI1 gene into the unique SnaBl-site. of<br>
pDB293S5 the S. cerevisiae SICQ2n PDI1 gene was similarly inserted at this site. The &amp;<br>
cerevisiae SKQ2n PDI1 gene sequence was PCR amplified from plasmid DNTA<br>
containing the PDI1 gene from pMA3a:C7 (US 6,291,205), also known as Clone C7<br>
(Crouzet &amp; Tuite; 1987, supra; Farquhar et al 1991, supra). The SKQ2n PDI1 gene was<br>
amplified using oligonucleotide primers DS248 and DS250 (Figure 43). The<br>
approximately 2-kb PCR product was digested with EcoEI and BaniHI and ligated into<br>
Ylplac211 (Gietz &amp; Sugino, 1988, Gene, 74, 527-534) that has been cut with£coRI aoad<br>
BamHI, to produce plasmid pDB2943 (Figure 45). The 5' end of the SKQ2n POI1<br>
sequence is analogous to a blunt-ended Spel-site extended to include the EcoKL, Sael,<br>
SnaBl, Pad, Fsel, Sfil and Smal sites, the 3' end extends up to a site analogous to a<br>
blunt-ended Bsu36l site, extended to include a Smal, SnaBl and BaniHI sites. The PD&gt;I1<br>
promoter length is approximately 2lObp. The entire DNA sequence, was determined for<br>
the PDI1 fragment and shown to code for the PDI protein ofS. cerevisiae strain SKQ2n<br>
sequence (NCBI accession number CAA38402), but with a serine residue at position 1 14<br>
(not an arginine residue). Similarly to the S. cerevisiae S28Sc sequence in pDB2939,<br>
pDB2943 had a missing 'G' from within the DS248 sequence, which is marked in bold in<br>
Figure 43. The approximately 1,989-bp SnaBl fragment from pDB2943 was<br>
S3<br>
subsequently cloned into the unique SnaBl-sitQ in pDJS2938. This produced plasmid<br>
pDB2952 (Figure 46), in which the SKQ2n PDI1 gene is transcribed in the same<br>
direction as REP 2.<br>
EXAMPLE 7<br>
Relative Stabilities of the LEU2 Marker in Yeast Transformed with pSAC35-Like<br>
Plasmids Containing DNA Inserted Immediately after the KEP2 gene<br>
The impact on plasmid stability from insertion of the linker sequence at the SnaBl-site<br>
introduced after the KEP2 gene in pSAC35 was assessed for pDB2954. This was<br>
determined in the same S. cerevisiae strain as used in the earlier examples by loss of the<br>
LEU2 marker during non-selective growth on YEPS. The stability of pDB2954 was<br>
compared to the stabilities of pSAC35 (control plasmid), pDB2688 (A'c/nl-linker) and<br>
pDB2817 (Imrcl-linker) by the method described in Example 1.<br>
The 3'east strain was transformed to leuciie prototrophy using a modified lithium acetate<br>
method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et at, 1983, J.<br>
Bacteriol.) 153, 163; Elble, 1992, Biotechniques, 13, 18)), Transformants were selected<br>
on BMMD-agar plates, and were subsequently patched out on BMMD-agar plates.<br>
Cryopreserved trehalose stocks were prepared from lOmL BMMD shake flask cultures<br>
(24 hrs, 30°C, 200rpm) by mixing with an equal volume of sterile 40% (w/v) trehalose<br>
o o and freezing aliquots at-80 C (i.e. minus 80 C).<br>
For the determination of plasmid stability, a ImL cryopreserved stock was thawed and<br>
inoculated into lOOmL YEPS (initial ODeoo « 0.04-0.09) in a 250mL conical flask and<br>
grown for approximately 72 hours (typically 70-74 hrs) at 30°C in an orbital shaker (200<br>
rpm, Innova 4300 incubator shaker, New Brunswick Scientific). Each strain was<br>
analysed in duplicate.<br>
Samples were removed from each flask, diluted in YEPS-broth (10'2 to 10"5 dilution), and<br>
1 OOf.iL aliquots plated in duplicate onto YEPS-agar plates. Cells were grown at 30°C for<br>
4 days to allow single colonies to develop. Fox each yeast stocl: analysed. 100 random<br>
colonies were patched in replica onto BMMS-agar piaies followed by YEPS-agar plates.<br>
After growth at 30°C for 3-4 days the percentage of colonies growing on both BlvlMSagar<br>
plates and YEPS-agar plates was determined as the measure of plasmid stability.<br>
The results of the above analysis are shown below in Table 5A. These results indicate<br>
thai pDE2954 is essential!)' as stable as the pSAC35 control and pDB26SS. In this type<br>
of assay a low level of instability can occasionally be detected even with the pSAC35<br>
control (see Table 4). Hence, the iSnoBI-site artificially introduced into the inverted<br>
repeat sequence immediately after the translation termination codon of JfLE'P2 appeared to<br>
be equivalent to the Xcml-stie. in the inverted repeat for insertion of sj^ntaetic linker<br>
sequences. However, the JicwI-site appeared to be preferable to the iSnaBI-site for<br>
insertion of the approximately 2-kb DNA fragment containing the PDI1 gene.<br>
Table 5A: Relative stabilities of pSAC35-based vectors containing various DNA<br>
insertions<br>
(Table Removed)<br>
A "zero percent stability" result of this assay for plasmids pDB2952 and pDB2950 was<br>
obtained in non-selective media, which gives an indication of the relative plasmid<br>
stabilities. This assay was optimised to compare the relative stabilities of the different<br>
linker inserts. In selective media, plasmids with PDIl at the SnaBI-site (even when<br>
comprising an additional transferrin gene at the Not/ site, which is known to further<br>
destabilise the plasmid (such as pDB2959 and pDB2960 as described below)) produced<br>
"precipitin halos" of secreted transferrin on both non-selective YEPD-agar and selective<br>
BMMD-agar plates containing anti-transferrin antibodies. Precipitin halos of secreted<br>
transferrin were not observed from pDB2961, without the PDIl gene inserted at the<br>
SwoBI-site. These results demonstrate that the iSfooBI-site is useful for the insertion of<br>
large genes such as PDIl, which can increase the secretion of heterologous proteins.<br>
These results were all generated in the control strain. An increase was also seen for<br>
Strain A containing pDB2959 and pDB2960, but in this case there was also a lower level<br>
of secretion observed with pDB2961 (because of the extra PDIl gene in the genome of<br>
Strain A). Results from the control strain are summarised in Table SB below. Antibody<br>
plates were used contained lOOuL of goat polyclonal anti-transferrin autiserum<br>
(Calbiochem) per 25mL BMMD-agar or YEPD-agar. Strains were patched onto antibody<br>
plates and grown for 48-72 hours at 30°C, after which the precipitin "halos" were<br>
observed within the agar around colonies secreting high levels of recombinant transferrin.<br>
Very low levels of transferrin secretion are not observed in this assay.<br>
Plasmids pDB2959, pDB2960 and pDB2961 were constructed from pDB2950 (Figure<br>
44), pDB2952 (Figure 46) and pDB2954 (Figure 38) respectively, by inserting the same<br>
3.27-kb Not/ cassette for rTf (N413Q, N611Q) as found in pDB2711 (Figure 11), into the<br>
unique Notl-site, in the same orientation as pDB2711.<br>
Table 5E: Increased transfenin secretion from the Control Strain transformed with pSAC3&gt;<br>
bassd A'ectors containing various PDl] gene insertions inxcnediately-site after REP 2<br>
(Table Removed)<br>
EXAJSCPLE 8<br>
Stabilities of the LEU2 Marker in Yeast Transformed with pSACBS-Like Plasmids<br>
Determined Over Thirty Generations of Growth in Non-Selective Conditions<br>
The stabilities of pSAC35-like plasmids with DNA inserted in the US-region were<br>
determined using a method analogous to that defined by Chinery &amp; Hinchcliffe (1989,<br>
Curr. Genet., 16, 21-25) This was determined in the same S. cerevisiae strain as used in<br>
previous examples by loss of the LEU2 marker during logarithmic growth on nonselective<br>
YEPS medium over a defined number of generations. Thirty generations was<br>
suitable to show a difference between a control plasmid, pSAC35, or to shown<br>
comparable stability to the control plasmid. Plasmids selected for analysis by this assay<br>
were; pSAC35 (control), pDB2688 (A'crol-linker), pDB2812 (Hgal-lmk&amp;i), pDB2S17<br>
(Jlffjnl-linker), pDB2960 (PDIl gene inserted at Xcml site after REP 2} and pDB2711<br>
(PDIl gene inserted &amp;\XcmI site after REP 2 and a transferrin expression cassette inserted<br>
at the Nofl-site in the UL-region).<br>
Strains were grown to logarithmic phase in selective (BMMS) media at 30°C and used to<br>
inoculate lOOmL non-selective (YEPS) media pre-warmed to 30°C in 250mL conical<br>
flasks, to give between 1.25x105 and 5x10D cells/ml. The number of cells inoculated into<br>
each flask was determined accurately by using a haemocytometer to count the number of<br>
cells in culture samples. Aliquots were also plated on non-selective (YEPS) agar and<br>
incubated at 30°C for 3-4 days, after winch for each stock analysed, 100 random colonies<br>
were replica plated on selective- (BMMS) agar and non-selective (YEPS) agar to assess<br>
the proportion of cells retaining the plasmid. After growth at 30°C for 3-4 days the<br>
percentage of colonies growing on both BMMS agar and YEPS agar plates was<br>
determined as a measure of plasmid stability.<br>
Non-selective liquid cultures were incubated at 30°C with shaking at 200rprn for 24<br>
hours to achieve approximately 1x107 cells/nil, as determined by haemocytometer counts.<br>
The culture was then re-inoculated into fresh pre-warmed non-selective media to give<br>
between 1.25xl05 and 5x10D cells/ml. Aliquots were again plated on non-selective agar,<br>
and subsequently replicated plated on selective agar and non-selective agar to assess<br>
retention of the plasmid. Hence, it was possible to calculate the number of cell<br>
generations in non-selective liquid media. Exponential logarithmic growth was<br>
maintained for thirty generations in liquid culture, which was sufficient to show<br>
comparable stability to a control plasmid, such as pSAC35. Plasmid stability was defined<br>
as the percentage cells maintaining the selectable LEU2 marker.<br>
Results of the above analysis to measure the retention of the plasmid-encoded phenotype<br>
through growth in non-selective media are shown in Table 6 and Figure 47.<br>
Table- 6: The Relative Stabilities of Selected pSAC35-Like Plasmids in a 5. ccrevisiae<br>
Strain grown for Thirty Generaiionr, in Non-Selective Media<br>
(Table Removed)<br>
Figure 47 shows the loss of the LEU2 marker with increasing number generation in nonselective<br>
liquid culture for each strain analysed.<br>
The control plasmidpS ACS 5 remained 100% stable over the entire 30-generations of this<br>
assay. Plasmids pBD268S and pDB2812 both appeared to be as stable as pSAC35.<br>
Therefore, insertion of the linlcer into the Xcml-site after KEP2 or the Hgal-site. after FLP<br>
respectively had no apparent effect on plasrnid stability. In contrast., insertion of the<br>
Xmnl-linker within the REP 2 gene appeared to have reduced plasrnid stability.<br>
Plasrnid pDB2690, which contains a S. cerevisiae PDIl gene in the _l'c777l-linker after<br>
KEPI, was approximately 33% stable after thirty generations growth, indicating that<br>
insertion of this large DNA fragment into the US-region of the 2um-based vector caused<br>
a decrease in plasmid stability. However, this decrease in stability was less than that<br>
observed with pDB2711, where insertion of the recombinant transferrin (N413Q, N611Q)<br>
expression cassette into the Notl-site within the large unique region of pSAC35 acted to<br>
further destabilise the plasmid. These observations are consistent with the results of<br>
Example 2 (see Table 2).<br>
The stability of plasmid pDB2711 was assessed by the above method in an alternative<br>
strain of S. cerevisiae, and similar results were obtained (data not shown). This indicates<br>
that the stability of the plasmid is not strain dependent.<br>
EXAMPLE 9<br>
PDI1 gene disruption, combined with a PDI1 gene on the 2pn-based plasmid<br>
enhanced plasmid stability<br>
Single stranded oligonucleotide DNA primers listed in Table 7 were designed to amplify<br>
a region upstream of the yeast PDI1 coding region and another a region downstream of<br>
the yeast PDI1 coding region.<br>
Table 7: Oligonucleotide primers <br>
(Table Removed)<br>
Primers DS2P9 and DS300 amplified the 5' region of PDI1 by PCR, while primers<br>
DS301 and DS302 amplified a region 3' of AD/7, using gsnomic DNA derived S288c as<br>
a template. The PCR conditions were as follows: 1 uL S2S8c template DNA (at<br>
0.01ng/uL; 0.1ng/j.LL, Ing/^L. lOng/uL and lOOng/pL), 5|iL lOXBuffer (Fast Start<br>
Taq+Mg, (Roche))., luL lOmlvf dNTP's, 5uL each primer DfiMl G.4jiL Fast Stait Taq,<br>
made up to 50uL with FbO. PCRs were performed using a Perkin-Ehner Thermal Cycler<br>
9700. The conditions were: denature at 95°C for 4min [HOLD], then [CYCLE] denature<br>
at 95°C for 30 seconds, anneal at 45DC for 30 seconds, extend at 72°Cfor 45 seconds for<br>
20 cycles., then [HOLD] 72°C for lOmin and then [HOLD] 4°C. The 0.221cbp PDI1 5'<br>
PCR product was cut with Noil and HindlR, while the 0.34kbp PDI1 3' PCR product was<br>
cut with Hindlll and PstL<br>
Plasmid pMCSS (Hoheisel, 1994, Biotechniques 17, 456-460) (Figure 48) was digested to<br>
completion with HindUl, blunt ended with T4 DNA polymerase plus dNTPs and<br>
religated to create pDB2964 (Figure 49).<br>
Plasmid pDB2964 was HindlR digested, treated with caLf intestinal phosphatase, and<br>
ligated with the 0.221chp PDI1 5' PCR product digested with Notl and HinSSl and the<br>
0.34kbp PDI1 3' PCR product digested with HinSHl and Pstl to create pDB3069 (Figure<br>
50) which, was sequenced with forward and reverse universal primers and the DNA<br>
sequencing primers DS303, DS304; DS305 andDS306 (Tahle 7).<br>
Primers DS234 and DS235 (Table 8) were used to amplify the modified TRP1 marker<br>
gene from YIplac204 (Gietz &amp; Sugrno, 1988, Gene, 74, 527-534), incorporating HindUl<br>
restriction sites at either end of the PCR product. The PCR conditions were as follows:<br>
IjiL template YIplac204 (at 0.01ng/uL3 O.lng/uL, Ing/uL, lOng/uL and lOOng/jaL), 5uL<br>
lOXBuffer (Fast Start Taq+Mg, (Roche)), IjiL IQmM dNTP's, 5j.tL each primer (2uM),<br>
0.4p.L Fast Stait Taq, made up to 50uL with H?0. PCRs were performed using a Perlcin-<br>
Elmer Thermal Cycler 9600. The conditions were: denature at 95°C for 4min [HOLD],<br>
then [CYCLE] denature at 95°C for 30 seconds, anneal for 45 seconds at 45°C; extend at<br>
72°C for 90sec for 20 cycles, then [HOLD] 72°C for lOmin and then [HOLD] 4°C. The<br>
O.S6kbp PCR product was digested with Hindlll and cloned into the HindHI site of<br>
pMCSS to create pDB2778 (Figure 51). Restriction enzyme digestions and sequencing<br>
with universal forward and reverse primers as well as DS236, DS237, DS238 and DS239<br>
(Table 8) confirmed that the sequence of the modified TRP1 gene was correct<br>
Table 8: Oligonucleotideprimers<br>
(Table Removed)<br>
The 0.86kbp TRP1 gene was isolated from pDB2778 by digestion with HindTH. and<br>
cloned into the HmSEL site of pDB3069 to create pDB3078 (Figure 52) andpDB3079<br>
(Figure 53). A 1.41kb pdil::TRPl disrupting DNA fragment was isolated from<br>
pDB307S or pDB3079 by digestion with Notl/Pstl.<br>
Yeast strains incorporating a TRP1 deletion (trplA) were to be constructed in such a way<br>
that no homology to the TRP1 marker gene (pDB2778) should left in the genome once<br>
the trplA had been created, so preventing homologous recombination between future<br>
TRP1 containing constructs and the TRP1 locus. In order to achieve the total removal of<br>
the native TKP1 sequence from the genome of the chosen host strains, oligonucleotides<br>
were designed to amplify areas of the 5' UTR and 3' UTR of the TRP1 gene outside of<br>
TIJ'J marker gene present on integrating vector YIplac204 f Gietz &amp; Sugmo, 1988. to,*,<br>
74. 527-534J. The YIplac204 TRP1 marker gene differs from the native/chromosomal<br>
IKW sene in that internal P/mdllL Pstl and -M sites were removed by site directed<br>
inutaaenesis (Gietz &amp; Sugmo, 1988, Gem, 74, 527-534). The YIPla,204 modified TRP1<br>
marker gene was constructed from a 1.453kbp blunt-ended genomic fragment EcoXl<br>
fragment which contained the TKPJ gene and only 102bp of the TKP1 promoter f Gietz<br>
&amp; Sugmo, 1988, Gene, 74, 527-534). Although tins was a relatively short promoter<br>
sequence it was clearly sufficient to complement tyl auxotrophic mutations (Gietz &amp;<br>
Sugmo, 1988, Gene, 74, 527-534). Only DNA sequences upstream of the EcoPJ site,<br>
positioned 102bp 5' to the start of the TRP1 ORF were used to create the 5' TRP1 UTR.<br>
The selection of the 3' UTR was less critical as long as it -was outside the 3' end of the<br>
iunctional modified TSP1 marker, which was chosen to be 85bp downstream of the<br>
translation stop codon.<br>
Single stranded oligonucleotide DNA primers were designed and constructed to amplify<br>
the°5' UTR and 3' UTR regions of the TSP1 gene so that during the PCR amplification<br>
restriction enzyme sites would be added to the ends of the PCR products to be used in<br>
later cloning steps. Primers DS230 and DS231 (Table 8) amplified the 5' region of Wl<br>
by PCR, while primers DS232 andDS233 (Table 8) amplified a region 3' of TRP1, using<br>
S2S8c genomic DNA as a template. The PCR conditions were as follows: 1 uL template<br>
S288c genomic DNA (at O.Olng/uL, O.lng/pX, Ing/pL, 10ng/uL and lOOng/pX), 5uL<br>
lOXBuffer (Fast Start Taq+Mg, (Roche)), luL lOmM dNTP's, 5|aL each primer (2^M),<br>
0.4aL Fast Start Taq, made up to 50uL with H20. PCRs were performed using a Perldn-<br>
Elmer Thennal Cycler 9600. The conditions were: denature at 95"C for 4mm [HOLD],<br>
then [CYCLE] denature at 95°C for 30 seconds, anneal for 45 seconds at 45°C, extend at<br>
72°C for 90sec for 20 cycles, fhen [HOLD] 72°C for lOmin and then [HOLD] 4°C.<br>
The 0.19kbp TRP1 5' UTR PCR product was cut with EcoKL and HindEl, while the<br>
0 2kbp TRP1 3' UTR PCR product was cut with BamSL 'and Hindlll and ligated into<br>
pAYE505 linearised with BwWEcoXl to create plaSmid-pDB2777 (Figure 54). The<br>
constmction of pAYESOS is described in WO 95/33833. DNA sequencing using forward<br>
and reverse primers, designed to prune from the plasmid backbone and sequence the<br>
cloned inserts, confirmed that in both cases the cloned 5' and 3' UTR sequences of the<br>
TRP2 gene had the expected DNA sequence. Plasmid pDB2777 contained a TRP1<br>
disrupting fragment that comprised a fusion of sequences derived from the 5' and 3'<br>
UTRs ofTRPl. This 0.383kbp TRP1 disrupting fragment was excised from pDB2777 by<br>
complete digestion with Eco~Rl.<br>
Yeast strain DXY1 (Kerry-Williams et al, 1998, Yeast, 14, 161-169) was transformed to<br>
leucine prototrophy with the albiunin expression plasmid pDB2244 using a modified<br>
lithium acetate method (Sigma yeast transformation kit, YEAST-1, protocol 2; (Ito et al,<br>
1983, J. Bacterial, 153, 163; Elble, 1992, Biotechnigues, 13, 18)) to create yeast strain<br>
DXY1 [pDB2244J. The construction of the albumin expression plasmid pDB2244 is<br>
described in WO 00/44772. Transformants were selected on BMMD-agar plates, and<br>
were subsequently patched out on BMMD-agar plates. Cryopreserved trehalose stocks<br>
were prepared from 1 OmL BMMD shake flask cultures (24 hrs, 30°C, 200rpm).<br>
DXY1 [pDB2244] was transformed to tryptophan autotrophy with the 0.383kbp EcoRI<br>
TRP1 disrupting DNA fragment from pDB2777 using a nutrient agar incorporating the<br>
counter selective tryptophan analogue, 5-fluoroanthranilic acid (5-FAA), as described by<br>
Toyn et al, (2000 Yeast 16, 553-560). Colonies resistant to the toxic effects of 5-FAA<br>
were picked and streaked onto a second round of 5-FAA plates to confirm that they really<br>
were resistant to 5-FAA and to select away from any background growth. Those colonies<br>
which, grew were then were re-patched onto BMMD and BMMD plus tryptophan to<br>
identify which were tryptophan auxotrophs.<br>
Subsequently colonies that had been shown to be tryptophan auxotrophs were selected for<br>
further analysis by transformation with YCplac22 (Gietz &amp; Sugino, 1988, Gene, 74, 527-<br>
534) to ascertain which isolates were trpl.<br>
PCR amplification across the TRP1 locus was used to confirm that the trp" phenotype was<br>
due to a deletion in this region. Genomic DNA was prepared from isolates identified as<br>
resistant to 5-FAA and unable to grow on minimal media without the addition of<br>
tryptophan. PCR amplification of the genomic TRP1 locus with primers CED005 and<br>
CED006 (Table S) was achieved as follows: luL template genomic DNA, 5uL<br>
lOXBuffer (Fast Start Taq+Mg, (Roche)), IjiL lOmM dNTP's, 5uL each primer (2uM),<br>
0.4|.iJL Fast Start Taq, made up to 50uL with EkO. PCRs were performed using a Perlcin-<br>
Elmer Thermal Cycler %00. The conditions were: denature al 94°C for 1 Omm [HOLD],<br>
then [CYCLE] denature at 94 °C for 30 seconds, anneal for 30 seconds at 55°Cr extend at<br>
72°C for 120sec for 40 cycles, then [HOLD] 72*C for lOmni and then [HOLD] 4°C.<br>
PCR amplification of the wild type TRJP1 locus resulted in a PCR product of 1.34kbp in<br>
size, whereas amplification across the deleted TEfl region resulted in a PCR product<br>
0.84kbp smaller at O.SOkbp. PCR analysis identified a DXY1 derived tip- strain (DXY1<br>
trplA [pDB2244]) as having the expected deletion event.<br>
The yeast strain DXY1 trplA [pDB2244] was cured of the expression plasmid pDB2244<br>
as described by Sleep el al, 1991, Bio/Technology, 9,183-187. DXY1 trplA cir° was retransformed<br>
the leucme prototrophy with either PDB2244, pDB2976, PDB2977,<br>
pDB297S, pDB2979, PDB2980 or pDB2981 (the production of pDB2976, pDB2977 and<br>
pDB2980 or pDB2981 is discussed further in Example 10) using a modified lithium<br>
acetate method (Sigma yeast transformation lat, YEAST-1, protocol 2; ato et al, 1983, J.<br>
BacterioL 153, 163; Elble, 1992, Biotechnigues, 13, 18)). Transformants were selected<br>
on BMMD-agar plates supplemented with tryptophan, and were subsequently patched out<br>
on BMMD-agar plates supplemented with tr3'PtoPhan. Cryopreserved trehalose stocks<br>
were prepared from IGmL BMMD shake flask cultures supplemented with tryptophan<br>
(24hrs,30°C,200rpm).<br>
The yeast strains DXY1 *plA [pDB2976], DXY1 trplA [pDB2977], DXY1 trplA<br>
[PDB3078], DXY1 *plA [pDB3079]9 DXY1 n-plA [pDB29BO] or DXY1 trplA<br>
[PDB^981] was transformed to tryptophan prototrophy using the modified lithium acetate<br>
xnetiod (Sigma yeast transformation lot YEAST-1, protocol 2; (Ito ,/ al, 1983, J.<br>
Bacterial, 153, 163; Elble, 1992, Biotechnig^s, 13, 18)) with a 1.41kb Pdil::TRPl<br>
disrupting DNA fragment was isolated from pDB3078 by digestion with NofUPstl.<br>
Transformants were selected on BMMD-agar plates and were subsequently patched out<br>
on BMMD-agar plates.<br>
Six transformants of each strain were inoculated into 10mL YEPD in 50mL shake flasks<br>
and incubated in an orbital shaker at 30°C, 200rpm for 4-days. Cultoe supernatant* and<br>
cell biomass were harvested Genomic DNA was prepared (Lee, 1992, Biotechnigues,<br>
V 677) from the tryptophan prototrophs and DXY1 [pDB2244]. The genomic PDI1<br>
locus amplified by PCR of with primers DS236 and DS303 (Table 7 and 8) was achieved<br>
as follows: luL template genomic DNA, 5uL lOXBuffer (Fast Start Taq+Mg, (Roche)),<br>
luL lOmM dNTP's, 5uL each primer (2uM), 0.4uL Fast Start Taq, made up to 50fj.L<br>
with HjO. PCRs were performed using a Perkin-Elmer Thermal Cycler 9700. The<br>
conditions were: denature at 94°C for 4min [HOLD], then [CYCLE] denature at 94°C for<br>
30 seconds, anneal for 30 seconds at 50°C, extend at 72°C for 60sec for 30 cycles, then<br>
[HOLD] 72°C for lOmin and then [HOLD] 4°C. PCR amplification of the wild type<br>
PDI1 locus resulted in no PCR product, whereas amplification across the deleted PDI1<br>
region resulted in a PCR product 0.65kbp. PCR analysis identified that all 36 potential<br>
pdil::TRPl strains tested had the expectedpdil::TRPl deletion.<br>
The recombinant albumin titres were compared by rocket imniunoelectrophoresis (Figure<br>
55). Within each group, all six pdil::TRPl disruptants of DXY1 trplA [pDB2976],<br>
DXY1 trplA |pDB2978], DXY1 trplA [pDB2980], DXY1 trplA [pDB2977] and DXY1<br>
trplA [pDB2979] had very similar rHA productivities. Only the six pdil::TRPl<br>
disruptants of DXY1 trplA [pDB2981] showed variation in rHA expression titre. The<br>
saipdil::TRPl disruptants indicated in Figure 55 were spread onto YEPD agar to isolate<br>
single colonies and then re-patched onto BMMD agar.<br>
Three single celled isolates of DXY1 trplA pdil::TEPl [pDB2976], DXY1 trplA<br>
pdil::TRJPl [pDB2978], DXY1 trplApdil::TRPl [pDB2980], DXY1 trplApdil::TRPl<br>
[pDB2977], DXY1 trplA pdil::TKPl [pDB2979] and DXY1 trplA pdil::TRPl<br>
[pDB2981] along with DXY1 [pDB2244], DXY1 [pDB2976], DXY1 [pDB2978], DXY1<br>
[pDB2980], DXY1 (j&gt;DB2977], DXY1 [pDB2979] and DXY1 [pDB2981] were<br>
inoculated into 1 OmL YEPD in 50mL shake flasks and incubated in an orbital shaker at<br>
30°C, 200rpm for 4-days. Culture supernatants were harvested and the recombinant<br>
albumin titres were compared by rocket imniunoelectrophoresis (Figure 56). The thirteen<br>
wild type PDI1 and pdil::TRPl disruptants indicated in Figure 56 were spread onto<br>
YEPD agar to isolate single colonies. One hundred single celled colonies from each<br>
strain were then re-patched onto BMMD agar or YEPD agar containing a goat anti-HSA<br>
antibody to detect expression of recombinant albumin (Sleep et a/., 1991,<br>
Bk<: and the leu- leu or ler-></:>
of each colony scored (Table 9).<br>
Table 9:<br>
(Table Removed)<br>
These data indicate plasmid retention is increased when the PDI1 gene is used as a<br>
selectable marker on a plasmid in a host strain having no chromosomalty encoded PDl,<br>
even in non-selective media such as this rich medium. These show that an "essential"<br>
chaperone (e.g PDI1 or PSE1), or any other any "essential" gene product (e.g. PGKl Or<br>
FBA1) which, when deleted or inactivated, does not result hi an auxotroplijc<br>
(biosjaithetic) requirement,, can be used as a selectable marker on a plasmid in a host cell<br>
that, in the absence of the plasmid., is unable to produce that gene product, to achieve<br>
increased plasmid stability without the disadvantage of requiring the cell to be cultured<br>
under specific selective conditions. By "auxotrophic (biosjTithetic) requirement" -Ve<br>
include a deficiency, which can be complemented by additions or modifications to the<br>
growth medium. Therefore, "essential marker genes" hi the context of the present<br>
; invention are those that, when deleted or inactivated in a host cell, result in a deficiency<br>
which can not be complemented by additions or modifications to the growth medium.<br>
EXAMPLE 10<br>
The construction 0/ expression vectors containing various PDIl genes and the<br>
expression cassettes for various heterologous proteins on the same 2 fun-like plasmid<br>
PCR amplification and cloning of PDIl genes into YIplac211<br>
The PDIl genes from S. cerevisiae S288c and S. cerevisiae SKQ2n were amplified by<br>
PCR to produce DNA fragments with different lengths of the 5'-untranslated region<br>
containing the promoter sequence. PCR primers were designed to permit cloning of the<br>
PCR products into the EcoRI and BamHL sites of YIplac211 (Gietz &amp; Sugino, 1988,<br>
Gene, 74, 527-534). Additional restriction endonuclease sites were also incorporated into<br>
PCR primers to facilitate subsequent cloning. Table 10 describes the plasmids<br>
constructed and Table 11 gives the PCR primer sequences used to amplify the PDIl<br>
genes. Differences in the PDIl promoter length within these YIplac211-based plasmids<br>
are described in Table 10.<br>
pDB2939 (Figure 57) was produced by PCR amplification of the PDIl gene from S.<br>
cerevisiae S288c genomic DNA with oligonucleotide primers DS248 and DS250 (Table<br>
11), followed by digesting the PCR product with JEcoRI and BamHI and cloning the<br>
approximately 1.98-kb fragment into YIplac211 (Gietz &amp; Sugino, 1988, Gene, 74, 527-<br>
534), that had been cut with EcoEl and BamHL DNA sequencing of pDB2939 identified<br>
a missing CG' from within the DS248 sequence, which is marked in bold in Table 5.<br>
Oligonucleotide primers used for sequencing the PDIl gene are listed in Table 6, and<br>
were designed from the published S288c PDIl gene sequence (PDI1/YCL043C on<br>
chromosome III from coordinates 50221 to 48653 plus 1000 basepairs of upstream<br>
sequence and 1000 basepairs of downstream sequence, (http://www.veastgenome.org/<br>
Genebank Accession number NC001135).<br>
3(1: Ylplac211-based Plasmids Containing PD11 Genes<br>
(Table Removed)<br>
Table 11: Oligonucleotide Primers for PCR Amplification of S. cerevisiae PDI2 Genes<br>
(Table Removed)<br>
Table 12: Oligonucleotide Primers for DNA Sequencing S. cerevisiae PDIl Genes<br>
(Table Removed)<br>
Plasmids pDB2941 (Figure 58) and pDB2942 (Figure 59) were constructed similarly<br>
using the PCR primers described in Tables 10 and 11, and by cloning the approximately<br>
1.90-kb and 1.85-lcb EcoEl-JBamHL fragments, respectively, into YIplac211. The con-ect<br>
DNA sequences were confirmed for the PDIl genes in pDB2941 and pDB2942.<br>
The S. cerevisiae SKQ2n PDIl gene sequence was PCR amplified from plasmid DNA<br>
containing the PDIl gene from pMA3a:C7 (US 6,291,205), also Icnown as Clone C7<br>
(Crouzet &amp; Tuite, 1987, supra', Farquhar et al, 1991, supra}. The SKQ2n PDIl gene<br>
100<br>
was amplified using oligonucleotide primers US24B and DS250 (Tables 10 and 11). The<br>
approximately 2.0]-l;b PCR product was digested with £coJ"J and BanJHi and ligated<br>
into YIplac211 (Gietz &amp; Sugiiio, 1988, Gene, 14, 527-534'] that has been cut with EcoRl<br>
and Bamlil, to produce plasmid pDB2943 (Figure 60). The 5' end of the SKQ2n PDI1<br>
sequence is analogous to a blunt-ended Spel-sile. extended to include the .EcoRl, SacL<br>
iSrcoBl, Pad, Fsel, Sfil and Smal sites, the 3' end extends up to a site analogous to a<br>
blunt-ended Bsu36I site, extended to include, a Smal, SnaBl and BamHl sites. The PDI1<br>
promoter length is approximately 210bp. The entire DNA sequence was determined for<br>
ih&amp; PDIJ fragment using oligonucleotide primers given in Table 12. This confirmed the<br>
presence of a coding sequence, for the PDI protein of S. cerevisiae strain SICQ2n (NCBI<br>
accession number CAA3S402). but with a serine residue at position 114 (not an arginirte<br>
residue as previously published). Similar!}7, in the same wa}' as in the S. cerevisiae S2SSc<br>
sequence in pDB2939, pDB2943 also had a missing !G' from v\dthin the DS248<br>
sequence, which is marked in bold in Table 5.<br>
Plasmids pDB2963 (Figure 61) and pDB2945 (Figure 62) were constructed similarly<br>
using the PCR primers described in Tables 10 and 11, and by cloning the approximately<br>
1.94-lcb and 1.87-kb Eco'Rl-BamHI fragments, respectively, into "\lplac211. The<br>
expected DNA sequences were confirmed for the PDI] genes in pDB2963 and pDB29455<br>
with a serine codon at the position of ammo acid 114.<br>
The constructioD of pSAC35-based rHA expression plasmids with different PDI1<br>
genes inserted at the Xcml-site^ after KEP2:<br>
pSAC35-based plasmids were constructed for the co-expression of rHA with different<br>
PDI1 genes (Table 13).<br>
Table 13: pSAC35-based plasmids for co-expression of rHA with different PDI1 genes<br>
(Table Removed)<br>
.The rHA expression cassette from pDB2243 (Figure 63, as described in WO 00/44772)<br>
was first isolated on a 2,992-bp Notl fragment, which subsequently was cloned into the<br>
Noti-site of pDB2688 (Figure 4) to produce pDB2693 (Figure 64). pDB2693 was<br>
digested with SndBls treated with calf intestinal alkaline phosphatase, and ligated with<br>
SnaBI fragments containing the PDI1 genes from pDB2943, pDB2963, pDB2945,<br>
pDB2939, pDB2941 and pDB2942. This produced plasmids pDB2976 to pDB2987<br>
(Figures 65 to 70. PDI1 transcribed in the same orientation as REP2 was designated<br>
"orientation A", whereas PDI1 transcribed in opposite orientation to REP2 was<br>
designated "orientation B" (Table 13).<br><br><br><br><br><br>
We Claim:<br>
1.	A 2µm-family plasmid characterized in that it comprise a polynucleotide sequence insertion, deletion and/or substitution between the first base after the last functional codon of at least one of either a REP2 gene or an FLP gene and the last base before the FRT site in an inverted repeat adjacent to said gene.<br>
2.	The 2µm-family plasmid as claimed in claim 1 wherein, other than the polynucleotide sequence insertion, deletion and/or substitution, the FLP gene and/or the REP2 gene has the sequence of a FLP gene and/or a REP2 gene, respectively, derived from a naturally occurring 2µm-family plasmid.<br>
3.	The 2µm-family plasmid as claimed in claim 1 wherein the naturally occurring 2µm-family plasmid is selected from pSR1, pSB3 or pSB4 as obtained from Zygosaccharomyces rouxii, pSB1 or pSB2 both as obtained from Zygosaccharomyces bailli, pSM1 as obtained from Zygosaccharomyces fermentati, pKD1 as obtained from Kluyveromyces drosophilarum, pPM1 as obtained from Pichia membranaefaciens, and the 2µm plasmid as obtained from Saccharomyces cerevisiae.<br>
4.	The 2µm-family plasmid as claimed in claim 2 or 3 wherein the sequence of the inverted repeat adjacent to said FLP and/or REP2 gene is derived from the sequence of the corresponding inverted repeat in the same naturally occurring 2µm-family plasmid as the sequence from which the gene is derived.<br>
5.	The 2µm-family plasmid as claimed in any one of claims 2 to 4 wherein the naturally occurring 2µm-family plasmid is the 2µm plasmid as obtained from Saccharomyces cerevisiae.<br>
6.	The 2µm-family plasmid as claimed in claim 5 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at a position between the first base of codon 59 of the REP2 gene and the last base before the FRT site in the adjacent inverted repeat.<br>
7.	The 2µm-family plasmid as claimed in claim 5 or 6 wherein, other than the polynucleotide sequence insertion, deletion and/or substitution, the sequence of the REP2<br><br>
gene and the adjacent inverted repeat is as defined by SEQ ID N0:1 or variant thereof of the kind such as herein described.<br>
8.	The 2µm-family plasmid as claimed in any one of claims 1 to 7 wherein polynucleotide sequence insertion, deletion and/or substitution occurs at a position between the first base of the inverted repeat and the last base before the FRT site.<br>
9.	The 2µm-family plasmid as claimed in any one of claims 1 to 7 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs between the first base after the end of the REP2 coding sequence and the last base before the FRT site, such as at the first base after the end of the REP 2 coding sequence.<br>
10.	The 2µm-family plasmid as claimed in any one of claims 1 to 7 wherein, other than the polynucleotide sequence insertion, deletion and/or substitution, the inverted repeat that follows the REP2 coding sequence has a sequence derived from the corresponding region of the 2µm plasmid as obtained from Saccharomyces cerevisiae.<br>
11.	The 2µm-family plasmid as claimed in a claim 10 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at an XcmI site or an FspI site within the inverted repeat<br>
12.	The 2µm-family plasmid as claimed in claim 5 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at a position between the first base of codon 344 of the FLP gene and the last base before the FRT site in the adjacent inverted repeat.<br>
13.	The 2µm-family plasmid as claimed in claim 5, 11 or 12 wherein, other than the polynucleotide sequence insertion, deletion and/or substitution, the sequence of the FLP coding sequence and the adjacent inverted repeat is as defined by SEQ ID NO:2 or variant thereof of the kind such as herein described.<br>
14.	The 2µm-family plasmid as claimed in claim 11, 12 or 13 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at a position between the first base of the inverted repeat and the last base before the FRT site.<br><br>
15.	The 2µm-family plasmid as claimed in claim 14 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at a position between the first base after the end of the FLP coding sequence and the last base before the FRT site.<br>
16.	The 2µm-family plasmid as claimed in claim 15 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at the first base after the end of the FLP coding sequence.<br>
17.	The 2µm-family plasmid as claimed in any one of claims 11 to 16 wherein, other than the polynucleotide sequence insertion, deletion and/or substitution, the inverted repeat that follows the FLP gene has a sequence derived from the corresponding region of the 2µm plasmid as obtained from Saccharomyces cerevisiae.<br>
18.	The 2µm-family plasmid as claimed in claims 17 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs at an HgaI site or an FspI site within the inverted repeat<br>
19.	The 2µm-family plasmid as claimed in any one of the preceding claims comprising polynucleotide sequence insertions, deletions and/or substitutions between the first bases after the last functional codons of both of the REP2 gene and the FLP gene and the last bases before the FRT sites in the inverted repeats adjacent to each of said genes, which polynucleotide sequence insertions, deletions and/or substitutions can be the same or different.<br>
20.	The 2µm-family plasmid as claimed in any preceding claim optionally comprising a polynucleotide sequence insertion, deletion and/or substitution which is not at a position as defined in any one of the preceding claims.<br>
21.	The 2µm-family plasmid as claimed in claim 20 wherein the polynucleotide sequence insertion, deletion and/or substitution occurs within an untranscribed region around an ARS sequence.<br>
22.	The 2µm-family plasmid as claimed in any one of the preceding claims wherein the, or at least one, polynucleotide sequence insertion, deletion and/or substitution is a polynucleotide sequence insertion.<br><br>
23.	The 2µm-family plasmid as claimed in claim 22 in which the polynucleotide sequence insertion encodes an open reading frame.<br>
24.	The 2µm-family plasmid as claimed in claim 23 in which the open reading frame encodes a non- 2µm-family plasmid protein.<br>
25.	The 2µm-family plasmid as claimed in claim 24 in which the non- 2µm-family plasmid protein comprises the sequence of a known protein involved in protein folding, or which has chaperone activity or is involved in the unfolded protein response, albumin, a monoclonal antibody, an etoposide, a serum protein (such as a blood clotting factor), antistasin, a tick anticoagulant peptide, transferrin, lactoferrin, endostatin, angiostatin, collagens, immunoglobulins or immunoglobulin-based molecules or fragments of either (e.g. a dAb, Fab' fragments, F(ab')2, scAb, scFv or scFv fragment), a Kunitz domain protein, interferons, interleukins, IL10, IL11, IL2, interferon a species and sub-species, interferon ß species and sub-species, interferon y species and sub-species, leptin, CNTF, CNTFAX15, IL1-receptor antagonist, erythropoietin (EPO) and EPO mimics, thrombopoietin (TPO) and TPO mimics, prosaptide, cyanovirin-N, 5-helix, T20 peptide, T1249 peptide, HIV gp41, HIV gpl20, urokinase, prourokinase, tPA, hirudin, platelet derived growth factor, parathyroid hormone, proinsulin, insulin, glucagon, glucagon-like peptides, insulin-like growth factor, calcitonin, growth hormone, transforming growth factor ß, tumour necrosis factor, G-CSF, GM-CSF, M-CSF, FGF, coagulation factors in both pre and active forms, including but not limited to plasminogen, fibrinogen, thrombin, pre-thrombin, pro-thrombin, von Willebrand's factor, α1-antitrypsin, plasminogen activators, Factor VII, Factor VIII, Factor IX, Factor X and Factor XIII, nerve growth factor, LACI, platelet-derived endothelial cell growth factor (PD-ECGF), glucose oxidase, serum cholinesterase, aprotinin, amyloid precursor protein, inter-alpha trypsin inhibitor, antithrombin III, apo-lipoprotein species, Protein C, Protein S, or a variant or fragment of any of the above of the kind such as herein described.<br>
26.	The 2µm-family plasmid as claimed in claim 25 in which the non- 2µm-family plasmid protein comprises the sequence of albumin, a variant or fragment thereof of the kind such as herein described, or a fusion protein comprising the sequence of any of these.<br><br>
27.	The 2µm-family plasmid as claimed in claim 25 in which the non- 2µm-family plasmid protein comprises the sequence of transferrin, a variant or fragment thereof of the kind such as herein described, or a fusion protein comprising the sequence of any of these.<br>
28.	The 2µm-family plasmid as claimed in claim 25 in which the non- 2µm-family plasmid protein comprises the sequence of lactoferrin, a variant or fragment thereof of the kind such as herein described, or a fusion protein comprising the sequence of any of these.<br>
29.	The 2µm-family plasmid as claimed in claim 25 in which the non- 2µm-family plasmid protein comprises the sequence of Fc, a variant or fragment thereof of the kind such as herein described, or a fusion protein comprising the sequence of any of these.<br>
30.	The 2µm-family plasmid as claimed in claim 25 in which the non- 2µm-family plasmid protein comprises the sequence of a protein involved in protein folding, or which has chaperone activity or is involved in the unfolded protein response as encoded by any one of AHA1, CCT2, CCT3, CCT4, CCT5, CCT6, CCT7, CCT8, CNS1, CPR3, CPR6, EPS1, ERO1, EUG1, FMO1, HCH1, HSP10, HSP12, HSP104, HSP26, HSP30, HSP42, HSP60, HSP78, HSP82, JEM1, MDJ1, MDJ2, MPD1, MPD2, PDJ1, PFD1, ABC1, APJ1, ATP11, ATP12, BTT1, CDC37, CPR7, HSC82, KAR2, LHS1, MGE1, MRS11, NOB1, ECM10, SSA1, SSA2, SSA3, SSA4, SSC1, SSE2, SIL1, SLS1, UBI4, ORM1, ORM2, PER1, PTC2, PSE1 and HAC1 or a truncated intronless HAC1.<br>
31.	The 2µm-family plasmid as claimed in claim 25 or 30 in which the chaperone is protein disulphide isomerase (PDI), or is a protein encoded by ORM2, SSA1 or PSE1.<br>
32.	The 2µm-family plasmid as claimed in any one of claims 24 to 31 in which the non- 2µm-<br>
family plasmid protein comprises a secretion leader sequence.<br>
33.	The 2µm-family plasmid as claimed in claim 24 in which the non- 2µm-family plasmid<br>
protein comprises the sequence of a bacterial selectable marker and/or a yeast selectable<br>
marker.<br>
34.	The 2µm-family plasmid as claimed in claim 33 in which the bacterial selectable marker is<br>
a ß-lactamase gene and/or the yeast selectable marker is a LEU2 selectable marker.<br><br>
35.	The 2µm-family plasmid as claimed in any one of the preceding claims which plasmid<br>
comprises (i) a heterologous sequence encoding a non- 2µm-family plasmid protein; (ii) a<br>
heterologous sequence encoding a protein comprising the sequence of a protein involved<br>
in protein folding, a chaperone or a protein involved in the unfolded protein response,<br>
preferably protein disulphide isomerase; and (iii) a heterologous sequence encoding a<br>
protein comprising the sequence of a selectable marker; wherein at least one of the<br>
heterologous sequences occurs at a position as defined by any one of Claims 1 to 16.<br>
36.	A method of preparing a plasmid as claimed in any one of the preceding claims<br>
comprising -<br>
(a)	providing a plasmid comprising the sequence of a REP2 gene and the inverted repeat that follows the REP2 gene, or a FLP gene and the inverted repeat that follows the FLP gene, in each case the inverted repeat comprising an FRT site;<br>
(b)	providing a polynucleotide sequence and inserting the polynucleotide sequence into the plasmid at a position as defined in any one of Claims 1 to 18; and/or<br>
(c)	deleting some or all of the nucleotide bases at the positions defined in any one of Claims 1 to 18; and/or<br>
(d)	substituting some or all of the nucleotide bases at the positions defined in any one of Claims 1 to 18 with alternative nucleotide bases.<br>
37.	A plasmid obtainable by the method as claimed in claim 36.<br>
38.     A host cell comprising a plasmid as claimed in any one of claims 1 to 35 and 37.<br>
39.	A host cell as claimed in claim 38 which is a yeast cell.<br>
40.	A host cell as claimed in claim 38 or 39 in which the plasmid is stable as a multicopy plasmid.<br>
41.	A host cell as claimed in claim 40 in which the plasmid is based on pSR1, pSB3 or pSB4 and the yeast cell is Zygosaccharomyces rouxii, the plasmid is based on pSBl or pSB2 and<br><br>
the yeast cell is Zygosaccharomyces bailli, the plasmid is based on pSM1 and the yeast cell is Zygosaccharomyces fermentati, the plasmid is based on pKDl and the yeast cell is Kluyveromyces drosophilarum, the plasmid is based on pPMl and the yeast cell is Pichia membranaefaciens or the plasmid is based on the 2jxm plasmid and the yeast cell is Saccharomyces cerevisiae or Saccharomyces carlsbergensis.<br>
42.	A host cell as claimed in claim 40 or 41 in which, if the plasmid contains, or is modified to contain, a selectable marker then stability, as measured by the loss of the marker, is at least 1%, 2%, 3%, 4%, 5%, 10%, 15%, 20%, 25%, 30%, 40%, 50%, 60%, 70%, 75%, 80%, 85%, 90%, 95%, 96%, 97%, 98%, 99%, 99.9% or 100% after 5 generations.<br>
43.	A method as claimed in producing a protein comprising the steps of-<br><br>
(a)	providing a plasmid as claimed in any one of claims 1 to 35 or 37<br>
(b)	providing a suitable host cell;<br>
(c)	transforming the host cell with the plasmid; and<br>
(d)	culturing the transformed host cell in a culture medium;<br>
(e)	thereby to produce the protein.<br><br>
44.	A method as claimed in producing a protein comprising the steps of providing a host cell as defined by any one of Claims 38 to 42 which host cell comprises a plasmid as defined by any one of Claims 1 to 35 or 37 and culturing the host cell in a culture medium thereby to produce the protein.<br>
45.	A method as claimed in claim 43 or 44 further comprising the step of isolating the thus produced protein from the cultured host cell or the culture medium.<br>
46.	A method as claimed in claim 45 further comprising the step of purifying the thus isolated<br>
protein to a commercially acceptable level of purity.<br><br>
47.     A method as claimed in claim 45 further comprising the step of purifying the thus isolated protein to a pharmaceutically acceptable level of purity.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUFic3RyYWN0LSgxNS0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Abstract-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUFic3RyYWN0LSgxNi0wNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Abstract-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUFzc2lnbm1lbnQgKDE1LTEwLTIwMDkpLnBkZg==" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Assignment (15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNsYWltcy0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Claims-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNsYWltcy0oMTYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Claims-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Correspondence Others-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Correspondence Others-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Correspondence-Others (15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Correspondence-Others-(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-correspondence-others-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE1LTA2LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Description (Complete)-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDE2LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Description (Complete)-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LURyYXdpbmdzLSgxNS0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Drawings-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMS0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-1-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMS0oMTYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-1-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMi0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-2-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMi0oMTYtMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-2-(16-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMy0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-3-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUZvcm0tMy0oMTYtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Form-3-(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUdQQSAoMTUtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-GPA (15-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LUdQQS0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-GPA-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1kZWxucC0yMDA2LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">3664-delnp-2006-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMTUtMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2006-Petition-137-(15-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA5LUNvcnJlc3BvbmRlbmNlLU90aGVycyAoMjAtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2009-Correspondence-Others (20-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzY2NC1ERUxOUC0yMDA5LUdQQSAoMjAtMTAtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">3664-DELNP-2009-GPA (20-10-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="253997-process-for-manufacturing-a-coated-steel-part-having-very-high-strength-after-heat-treatment.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="253999-an-improved-process-for-the-synthesis-of-quinoline-derivatives.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>253998</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3664/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Sep-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>12-Sep-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>26-Jun-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NOVOZYMES BIOPHARMA DK A/S</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>KROGSHOEJVEJ 36, DK-2880 BAGSVAERD, DENMARK</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DARRELL SLEEP</td>
											<td>66 LADYBAG ROAD, WEST BRIDGFORD, NOTTINGHAM NG2 5DS, ENGLAND</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CHRISTOPHER JOHN ARTHUR FINNIS</td>
											<td>74 HARLAXTON DRIVE, LENTON, NOTTINGHAM NG7 1JB, ENGLAND</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/81</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/GB2004/005435</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-12-23</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0329722.3</td>
									<td>2003-12-23</td>
								    <td>U.K.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/253998-a-2-m-family-plasmid-and-its-method-of-preparation by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:09:56 GMT -->
</html>
