<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/254018-modulation-of-replicative-fitness-by-deoptimization-of-synonymous-codons by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:08:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 254018:&quot;MODULATION OF REPLICATIVE FITNESS BY DEOPTIMIZATION OF SYNONYMOUS CODONS&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;MODULATION OF REPLICATIVE FITNESS BY DEOPTIMIZATION OF SYNONYMOUS CODONS&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A method of reducing replicative fitness of a virus, comprising: deoptimizing at least twenty codons in a coding sequence of the virus by replacing in the coding sequence with a synonymous codon less frequently used in the virus, thereby generating a deoptimized coding sequence to reduce replicative fitness of the virus.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>MODULATION OF REPLICATIVE FITNESS BY DEOPTIMIZATION OF SYNONYMOUS<br>
CODONS<br>
CROSS REFERENCE TO RELATED APPLICATION<br>
[001]    This application claims priority to U.S. Provisional Application No. 60/617,545 filed October 8, 2004, herein incorporated by reference in its entirety.<br>
FIELD<br>
[002]    This disclosure relates to methods of reducing the replicative fitness of a pathogen by deoptimizing codons. Pathogens with deoptimized codons can be used to increase the phenotypic stability of attenuated vaccines.<br>
ACKNOWLEDGMENT OF GOVERNMENT SUPPORT<br>
[003]    This invention was made by the National Center for Infectious Diseases, Centers for Disease Control and Prevention, an agency of the United States Government. Therefore, the U.S. Government has certain rights in this invention.<br>
BACKGROUND<br>
[004]    Infections by intracellular pathogens such as viruses, bacteria and parasites, are cleared in most cases after activation of specific T cellular immune responses that recognize foreign antigens and eliminate infected cells. Vaccines against those infectious organisms have been traditionally developed by administration of whole live attenuated or inactivated microorganisms. Although research has been performed using subunit vaccines, the levels of cellular immunity induced are usually low and not capable of eliciting complete protection against diseases caused by intracellular microbes.<br>
[005]    One problem encountered when using live attenuated vaccines is the development of adverse events in some patients. Typical reactions associated with live viral and bacterial vaccines, such as measles, mumps, rubella (MMR) and varicella vaccines, often resemble attenuated forms of the disease against which the vaccine is directed. However, more severe adverse affects have been reported. For example, there is an association between the Urabe strain of mumps vaccine and viral meningitis (Dubey and Banerjee, Indian J, Pediatr. 70:579-84, 2003). In addition, vaccine associated thrombocytopenia has been reported. Although epidemiological studies do not support a causative link between MMR and autism (Chen et al., Psychol. Med. 34:543-53, 2004), the fear remains and likely contributes to poor vaccine acceptance in some regions and sections of society.<br>
[006]    In addition, documented safety concerns with vaccines demonstrate the harm that vaccines can cause. For example, the currently available attenuated Sabin oral polio vaccine (OPV) strains are genetically unstable, principally because only 2-5 base substitutions confer the attenuated phenotype<br>
 (Renetal. J. Viral. 65:1377-82,1991). This instability is the underlying cause of vaccine-associated paralytic poliomyelitis in immunologically normal (Strebel et al., Clin. Infect. Dis. 14:568-79, 1992) and in people with B-cell immunodeficiencies (Kew et al, J. Clin. Microbiol. 36:2893-9; Khetsuriani et al., J. Infect. Dis 188:1845-52, 2003; Yang et al., J. Virol. 79:12623-34), and of outbreaks associated with circulating vaccine-derived polioviruses (Kew et al., Science 296: 356-9, 2002; Yang et al., J. Virol. 77:8366-77, 2003; Rousset et al.. Emerg. Inf. Dis: 9:885-7, 2003; Kew et al., Bull. WHO 82:16-23, 2004; Shimizu et al, J. Virol 78:13512-21, 2004; Kew et al, Ann. Rev. Microbiol. 59:587-635, 2005). In addition, the CDC recommended suspending use of the rhesus-human rotavirus reassortant-tetravalent vaccine (RRV-TV) due to cases of intussusception (a bowel obstruction in which one segment of bowel becomes enfolded within another segment) among infants who received the vaccine (MMWR Morb Mortal Wkly Rep. 53:786-9, 2004).<br>
[007]    Although the primary mode of protective immunity induced by OPV is the production of neutralizing antibody by B-cells, OPV stimulates an immune response similar to that of a natural infection. Immunity against paralytic disease is further enhanced by the production of antibodies in the gastrointestinal tract that limit poliovirus replication, and, thus, person-to-person transmission. The stimulation of intestinal immunity, along with ease of administration, has made OPV the vaccine of choice for global polio eradication (Aylward and Cochi, Bull WHO 82:40-6, 2004). Therefore, there is a need to identify methods of making an attenuated vaccine that reduces the safety concerns with currently available live attenuated vaccines while retaining the advantages of attenuated vaccines.<br>
SUMMARY<br>
[008]    The inventors have determined that replacement of one or more natural (or native) codons in a pathogen with synonymous unpreferred codons can decrease the replicative fitness of the pathogen, thereby attenuating the pathogen. The unpreferred synonymous codon(s) encode the same amino acid as the native codon(s), but have nonetheless been found to reduce a pathogen's replicative fitness. The introduction of deoptimized codons into a pathogen can limit the ability of the pathogen to mutate or to use recombination to become virulent. The disclosed compositions and methods can be used in attenuated vaccines having well-defined levels of replicative fitness and enhanced genetic stabilities.<br>
[009]    Methods of reducing a pathogen's replicative fitness are disclosed. In some examples, the method includes deoptimizing at least one codon in a coding sequence of the pathogen, thereby generating a deoptimized coding sequence. Such deoptimization reduces replicative fitness of the pathogen. In some examples, more than one coding sequence of the pathogen is deoptimized, such as at least one, at least two, or at least 5 coding sequences, such as deoptimizing 1, 2, 3, 4, 5, 6, 7, 8, 9 or 10 coding sequences of the pathogen.<br>
[010]    More than one codon in the one or more coding sequences can be deoptimized, such as at least 15 codons, at least 20 codons, at least 30 codons, at least 40 codons, at least 50 codons, at least 60 codons, at least 70 codons, at least 100 codons, at least 200 codons, at least 500 codons, or even at least 1000 codons, in each coding sequence. In some examples, at least 20% of the coding sequence of each desired gene is deoptimized,,such as at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, at least 90%, at least 95%, or even at least 97% deoptimized.<br>
[Oil]    In particular examples, deoptimizing the codon composition alters the G+C content of a coding sequence, such as increases or decreases the G+C content by at least 10%, for example increases the G+C content of a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%, or decreases the G+C content of a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%. However, the G+C content can be altered in combination with deoptimizing one or more codons in a pathogen sequence. For example, some of the nucleotide substitutions can be made to deoptimize codons (which may or may not alter the G+C content of the sequence), and other nucleotide substitutions can be made to alter the G+C content of the sequence (which may or may result in a deoptimized codon). Altering the G+C content of the sequence may also result in a deoptimized codon, but is not required in all instances.<br>
[012]    For example, if the pathogen is a rubella virus, whose UNA genome has a high G+C content and consequently has a high rate of usage of rare codons rich in G+C. Therefore, deoptimization of rubella virus can be achieved by decreasing the G+C content of one or more coding sequences, for example decreasing the G+C content by at least 10%, such as at least 20%, or even by at least 50%. In another example, the pathogen is a poliovirus, and deoptimization can be achieved by increasing the G+C content of one or more coding sequences, for example increasing the G+C content by at least 10%, such as at least 20%, or even by at least 50%.<br>
[013]    In some examples, deoptimizing the codon composition alters the frequency of CG dinucleotides, TA dinucleotides, or both, in a coding sequence, such as increases or decreases the frequency of CG or TA dinucleotides by at least 10%, for example increases in the number of CG or TA dinucleotides in a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 100%, at least 200%, or even by at least 300%, or decreases in the number of CG or TA dinucleotides in a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%. However, the number of CG or TA dinucleotides can be altered in combination with deoptimizing one or more codons in a pathogen sequence. For example, some of the nucleotide substitutions can be made to deoptimize codons (which may or may not alter the number of CG or TA dinucleotides in the sequence), and other nucleotide substitutions can be made to alter the number<br>
of CG or TA dinucleotides in the coding sequence (which may or may result in a deoptimized codon). Altering the number of CG or TA dinucleotides in the sequence may also result in a deoptimized codon, but is not required in all instances.<br>
[014]    For example, if the pathogen is a poliovirus or eukaryotic virus, deoptimization can be achieved by increasing the number of CG or TA dinucleotides in one or more coding sequences, for example increasing the number of CG or TA dinucleotides by at least 10%, such as at least 30%, or even by at least 300%. In another example, the pathogen is a bacterium, and deoptimization can be achieved by decreasing the number of CG or TA dinucleotides in one or more coding sequences, for example decreasing the number of CG or TA dinucleotides by at least 10%, such as at least 30%, or even by at least 50%.<br>
[015]    In particular examples, methods of reducing the replicative fitness of a pathogen include analysis of a codon usage table for the pathogen to identify amino acids that are encoded by at least 2 different codons, (such as 2 different codons, 3 different codons, 4 different codons, or 6 different codons), and choosing the codon used least frequently (lowest codon usage frequency) of the different codons in the pathogen. The one or more low-frequency codons chosen are used to replace the appropriate one or more codons in the native sequence, for example using molecular biology methods, thereby generating a deoptimized sequence that reduces the replicative fitness of the pathogen. For example, if the pathogen uses the CCU, CCC, CCA and CCG codons to encode for Pro at 12, 19, 21 and 9% frequency respectively, the CCG codon can be used to replace at least one CCU, CCC, or CCA codon in the native pathogen sequence, thereby generating a deoptimized sequence. In this example, the use of the CCG codon may also increase the number of CG dinucleotides in the sequence, and may also increase the G+C content of the sequence. In examples where the amino acid is encoded by only two different codons, one of the two codons can be selected and used in the deoptimized sequence if the codon usage is highly biased, such as a difference of at least 10%, at least 20%, or at least 30%. For example, if the pathogen uses the codons CAA and CAG to encode for Gin at 60% and 40% frequency respectively, the CAG codon is used to replace at least one CAA codon in the native sequence, thereby generating a deoptimized sequence. In this example, the use of the CAG codon may also increase the G+C content of the sequence.<br>
[016]    In some examples, when choosing a low frequency codon, the codon chosen based on its ability to alter the G+C content of the deoptimized sequence or alter the frequency of CG or TA dinucleotides. For example, if the pathogen uses the CCU, CCC, CCA and CCG codons to encode for Pro at 9, 19, 21 and 12% frequency respectively, the CCG codon can be used to replace at least one CCU, CCC, or CCA codon in the native pathogen sequence, if the presence of increased G+C content or increased numbers of CG dinucleotides is desired in the deoptimized sequence. Even though CCG is not the most infrequently used codon, the use of this codon will will increase the number of CG dinucleotides in the sequence and may increase the G+C content of the deoptimized<br>
sequence. In contrast, if the presence of decreased G+C content or decreased numbers of CG dinucleotides is desired in the deoptimized sequence, the CCU codon could be used to replace at least one CCG, CCC, or CCA codon in the native pathogen sequence.<br>
[017]    In some examples, there may be two or more codons used at low frequencies that are similar in value, such as codon usages that are within 0.01-2% of each other (for example within 0.1-2%, 0.5-2% or 1-2% of each other). In this case, one can opt to not choose the codon with the lowest codon usage frequency. In some examples, the codon chosen is one that will alter the G+C content of the deoptimized sequence, such as increase or decrease the G+C content of the sequence. In other examples, the codon chosen is one that increases or decreases the frequency of a specific dinucleotide pair (such as a CG or TA dinucleotide pair) found at low frequencies in that genome (such as no more than 4%, for example no more than 3%). Such dinucleotide pairs can fall across codon boundaries, or be contained within the codon.<br>
[018]    The codon usage table used can include codon usage data from the complete genome of the pathogen (or 2 or more genomes, for example from different strains of the pathogen), codon usage data from one or more genes (such as 1 gene, at least 2 genes, at least 3 genes, at least 5 genes, or even at least 10 genes), for example one or more genes involved in the antigenicity of the pathogen.<br>
[019]    Specific non-limiting examples of deoptimized coding sequences for several pathogens are disclosed herein. In some examples, a deoptimized coding sequence includes a nucleic acid sequence having at least 90% sequence identity, such as at least 95% sequence identity, to any of SEQ ID NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69. Sequences that hybridize to any of SEQ ID NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69, for example under stringent conditions, are also disclosed. In some examples, a deoptimized coding sequence includes a nucleic acid sequence shown in any of SEQ ID NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69.<br>
[020]    In particular examples, more than one coding sequence in the pathogen is deoptimized, such as at least 2 coding sequences, such as at least 3, at least 4, at least 5, at least 6, at least 7, at least 8, at least 9, or even at least 10 coding sequences. Any coding sequence can be deoptimized. In one example, one of the deoptimized coding sequences encodes for a housekeeping gene. Particular examples of coding sequences that can be deoptimized in a pathogen, include, but are not limited to, sequences that encode a viral capsid, a viral spike glycoprotein (for example the gH and gE surface glycoproteins of varicella-zoster virus); glycoprotein B, glycoprotein D, glycoprotein H, and glycoprotein N of human cytomegalovirus; glycoprotein D, tegument protein host shut-off factor, ribonucleotide reductase large subunit of human herpes simplex viruses; the fusion (F) protein and glycoprotein (G) of respiratory syncytial virus; the hemagglutinin (HA) and neuraminidase (NA)<br>
glycoproteins of influenza virus; the env protein of human immunodeficiency virus type 1 (HIV-1), ArgS and TH/A gene products of Escherichia coli, or combinations thereof.<br>
[021]    The replicative fitness of the pathogen can be reduced by any amount sufficient to attenuate the pathogen. In some examples, the replicative fitness of the deoptimized pathogen is reduced by at least 20%, such as at least 30%, at least 40%, at least 48%, at least 50%, at least 75%, at least 80%, at least 90%, at least 95%, or even at least 97%, as compared to replicative fitness of a pathogen (of the same species and strain) having a coding sequence with an optimized codon composition.<br>
[022]    Any pathogen can be attenuated using the disclosed methods. Particular examples include, but are not limited to, viruses (such as positive-strand RNA viruses, negative-strand RNA viruses, DNA viruses, and retroviruses), bacteria, fungi, and protozoa.<br>
[023]    In one specific example, the pathogen is a poliovirus. For example, when the natural codons of the Sabin type 2 (Sabin 2) OPV strain (Sabin and Boulger. J. Biol. Stand. 1:115-8; 1973; Toyoda et al., J. Mol. Biol. 174:561-85, 1984) were replaced with synonymous unpreferred codons in sequences encoding capsid proteins, virus plaque size and yield in cell culture decreased in proportion to the number of unpreferred codons incorporated into the capsid sequences. The altered codon composition was largely conserved during 25 serial passages in HeLa cells. Fitness for replication in HeLa cells of both the unmodified Sabin 2 and modified constructs increased with higher passage; however, the relative fitness of the modified constructs remained lower than that of the unmodified construct.<br>
[024]    Attenuated pathogens produced by the methods disclosed herein are also provided. In one example, immunogenic compositions include an attenuated pathogen produced by the disclosed methods. Such immunogenic compositions can include other agents, such as an adjuvant, a pharmaceutically acceptable carrier, or combinations thereof.<br>
[025]    Methods are disclosed for eliciting an immune response against a pathogen in a subject, using the disclosed attenuated pathogens. In one example, the method includes administering an immunologically effective amount of the disclosed attenuated pathogens to a subject, thereby eliciting an immune response in the subject. In particular examples, the disclosed attenuated pathogens are present in an immunogenic composition which is administered to a subject. Subjects include human and veterinary subjects, such as cats, dogs, cattle, sheep, pigs and horses.<br>
[026]    The foregoing and other features and advantages of the disclosure will become more apparent from the following detailed description of a several embodiments.<br>
BRIEF DESCRIPTION OF THE FIGURES<br><br>
[027]    FIG. 1A is a schematic drawing showing the locations of the codon replacement cassettes<br>
A-D in the infectious Sabin 2 (S2R9) cDNA clone. The restriction sites used for construction of the<br>
codon replacement constructs are indicated at the appropriate positions, in the context of the mature<br>
viral proteins.	.     .       .<br>
[028]    FIGS. 1B-D is a sequence showing original S2R9 Sabin 2 triplets (ABCD, SEQ ID NO: 3) above the codon-replacement residues; the deduced amino acids for both constructs are indicated below the triplets (SEQ ID NO: 4). The fully replaced sequence (abed, SEQ ID NO: 5) is referred to S2R23.<br>
[029]	FIG. 2 is a<br>
schematic drawing showing exemplary Sabin 2 codon replacement constructs. The Sabin 2 genome is represented with open rectangles. Filled rectangles indicate the locations of individual cassettes, black-filled rectangles indicate cassettes with replacement codons. Unmodified cassettes are indicated by upper case letters; the corresponding cassettes with replacement codons are indicated by lower case letters.<br>
[030]    FIG. 3A is a graph showing mean plaque area in HeLa cells versus the number of nucleotide substitutions in the capsid region. The coefficient of determination (R2) for the regression line was 0.88.<br>
[031]    FIG. 3B is a graph showing virus yields (12-hour postinfection) of a single-step growth curve versus the number of nucleotide substitutions in the capsid region. The coefficient of determination (/?2) for the regression line was 0.94.<br>
[032]    FIG. 3C is a digital image showing plaque phenotypes at 35°C in HeLa cells.<br>
[033]    FIG. 3D is a graph showing the inverse linear relationship observed between plaque area and number of replacement codons in Sabin 2.<br>
[034]    FIG. 3E is a graph showing the inverse linear relationship observed between plaque area and number of CG pairs in Sabin 2.<br>
[035]    FIG. 4A and 4B are graphs showing single-step growth curves in HeLa S3 cells at 35°C.<br>
[036]    FIGS. 5A and SB are digital images showing production of intracellular Poliovirus-specific proteins produced by ABCD, ABCd, and abed viruses in vivo and in vitro. (A) Lysates of infected HeLa cells labeled with [35S]methionine at 4 to 7 hours postinfection. (B) In vitro translation products from rabbit reticulocyte lysates programmed with 250 ng of RNA transcripts from cDNAs ABCD, ABCd, and abed. Noncapsid proteins were identified by their electrophoretic mobilities and<br>
band intensities; capsid proteins were identified by their comigration with proteins from purified virions.<br>
[037]    FIGS. 5C and SD are digital images showing production of intracellular MEF Poliovirus-specific proteins produced by ABC, ABc, and abc viruses in vivo and in vitro. (A) Lysates of infected HeLa cells labeled with [35S]methionine at 4 to 7 hours postinfection. (B) In vitro translation products from rabbit reticulocyte lysates programmed with 250 ng of RNA transcripts from cDNAs ABC, ABc, and abc. Noncapsid proteins were identified by their electrophoretic mobilities and band intensities; capsid proteins were identified by their comigration with proteins from purified virions.<br>
[038]    FIG. 6A and B are graphs showing RNA yields from (A) ABCD, ABCd, and abed Sabin 2 viruses obtained in the single-step growth experiments described in FIGS. 4A and 4B, and for (B) ABC, ABc, and abc MEF1 viruses. RNA levels were determined by quantitative PCR using primers and a probe targeting 3Dp0'region sequences. One pg of poliovirus RNA corresponds to -250,000 genomes.<br>
[039]    FIG. 7 shows MinE RNA secondary structures for complete genomes of ABCD, ABCd, and abcd viruses calculated by using the mfold algorithm. Base positions are numbered in increments of 1000. Triangles mark boundaries of codon-replacement cassettes: beginning of cassette A (nt 657); beginning of cassette D (nt 2616); end of cassette D (nt 3302). Only intervals bounded by filled triangles had replacement codons.<br>
[040]    FIG. 8A is a graph showing mean plaque areas of evolving viruses using a plaque assay of HeLa cells after 60 hours incubation at 35°C.<br>
[041]    FIG. 8B is a graph showing virus tilers determined by plaque assay of HeLa cells at 35°C on every fifth passage.<br>
[042]    FIG. 8C is a digital image showing plaque phenotypes at 35°C in HeLa cells (35°C, 60 hours).<br>
[043]    FIGS. 9A-E show an original MEF1 capsid sequence (SEQ ID NO: 6; GenBank Accession No. AY082677) above the codon-replacement residues for an MEF1 de-optimized capsid sequence (SEQ ID NO: 8) (only replaced nucleotides are indicated); the deduced amino acids for both the constructs are indicated below the triplets (SEQ ID NO: 7).<br>
[044]    FIG. 9F is a graph showing the inverse linear relationship observed between plaque area and number of replacement codons in MEF1.<br>
[045]    FIG. 9G is a graph showing the inverse linear relationship observed between plaque area and number of CG pairs in MEF1.<br>
[046]    FIG. 9H is a graph showing plaque yields over time for native and deoptimized MEF1 constructs.<br>
[047]    FIG. 91 is a graph showing the inverse linear relationship observed between plaque size and number of nucleotide changes in MEF1.<br>
[048]    FIG. 9J is a graph showing the inverse linear relationship observed between viral liter and number of nucleotide changes in MEF1.<br>
[049]    FIGS. 10A-B show an original FMDV capsid sequence (SEQ ID NO: 9; GenBank Accession No. AJ539141) above the codon-replacement residues for an FMDV de-optimized capsid sequence (SEQ ID NO: 11) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 10).<br>
[050]    FIGS. 11A-C show an original SARS spike glycoprotein sequence (SEQ ID NO: 12; GenBank Accession No. AY278741) above the codon-replacement residues for a de-optimized SARS spike glycoprotein sequence (SEQ ID NO: 14) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 13).<br>
[051]    FIGS. 12A-G shows an original rubella sequence (SEQ ID NO: 15; GenBank Accession No. L78917) above the codon-replacement residues for a de-optimized rubella sequence (SEQ ID NO: 18) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NOS: 16 and 17).<br>
[052]    FIGS. I3A-B show an original VZV gH sequence (GenBank Accession No. AB097932, SEQ ID NO: 19) above the codon-replacement residues for a de-optimized VZV gH sequence (SEQ ID NO: 21) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 20).<br>
[053]    FIGS. 14A-B show an original VZV gE sequence (GenBank Accession No. AB097933, SEQ ID NO: 22) above the codon-replacement residues for a de-optimized VZV gE sequence (SEQ ID NO: 24) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 23).<br>
[054]    FIGS. 15A-B show an original measles F sequence (SEQ ID NO: 25; GenBank Accession No. AF266287) above the codon-replacement residues for a de-optimized measles F sequence (SEQ<br>
ID NO: 27) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 26).<br>
[055]    FIGS. 16A-B show an original measles hemagglutinin (H) sequence (SEQ ID NO: 28; GenBank Accession No. AF266287) above the codon-replacement residues for a de-optimized measles H sequence (SEQ ID NO: 30) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 29).<br>
[056]    FIGS. 17A-B show an original RSV F sequence (SEQ ID NO: 31; GenBank Accession No. U63644) above the codon-replacement residues for a de-optimized RSV F sequence (SEQ ID NO: 33) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 32).<br>
[057]    FIG. 18 shows an original RSV G sequence (SEQ ID NO: 34; GenBank Accession No. U63644) above the codon-replacement residues for a de-optimized RSV G sequence (SEQ ID NO: 36) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 35).<br>
[058]    FIG. 19 shows an original influenza HA sequence (SEQ ID NO: 37) above the codon-replacement residues for a de-optimized influenza HA sequence (SEQ ID NO: 39) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 38).<br>
[059]    FIG. 20 shows an original influenza NA sequence (SEQ ID NO: 40) above the codon-replacement residues for a de-optimized influenza NA sequence (SEQ ID NO: 42) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 41).<br>
[060]    FIGS. 21A-B show an original HIV-1 env sequence (SEQ ID NO: 43; GenBank Accession No. AF110967) above the codon-replacement residues for a de-optimized HIV-1 env sequence (SEQ ID NO: 45) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 44).<br>
[061]    FIGS. 22A-B show an original E. coli ArgS sequence (SEQ ID NO: 46; GenBank Accession No. U0096) above the codon-replacement residues for a de-optimized E. coli ArgS sequence (SEQ ID NO: 48) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 47).<br>
[062]    FIG. 23 shows an original E. coli TufA. sequence (SEQ ID NO: 49; GenBank Accession No. JO 1690) above the codon-replacement residues for a de-optimized E. coli TufA. sequence (SEQ ID NO: 51) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (SEQ ID NO: 50).<br>
[063]    FIGS. 24 A-M show exemplary codon usage tables for various pathogens.<br>
[064]    FIG. 25 shows a Sabin 2 virus cassette d (VP1 region) sequence that has been altered by reducing the number of CG dinucleotides. The original sequence (nucleotides 1975- 2664 of SEQ ID NO: 3) is shown above the codon-replacement residues for an altered Sabin 2 cassette d (VP1 region) sequence (SEQ ID NO: 65) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4).<br>
[065]    FIG. 26 shows a Sabin 2 virus cassette d (VP1 region) sequence that has been altered by decreasing the number of CG and TA dinucleotides. The original sequence (nucleotides 1975- 2664 of SEQ ID NO: 3) is shown above the codon-replacement residues for an altered Sabin 2 cassette d (VP1 region) sequence (SEQ ID NO: 66) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4).<br>
[066]    FIG. 27 shows a Sabin 2 virus cassette d (VP1 region) sequence that has been altered by increasing the number of CG dinucleotides. The original sequence (nucleotides 1975-2664 of SEQ ID NO: 3) is shown above the codon-replacement residues for a de-optimized Sabin 2 cassette d (VP1 region) sequence (SEQ ID NO: 67) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4). Original CG dinucleotides retained after codon changes are underlined.<br>
[067]    FIG. 28 shows a Sabin 2 virus cassette d (VP1 region) sequence that has been altered by increasing the number of CG and TA dinucleotides. The original sequence (nucleotides 1975- 2664 of SEQ ID NO: 3) is shown above the codon-replacement residues for a de-optimizedSabin 2 cassette d (VP1 region) sequence (SEQ ID NO: 68) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4). Original CG, TA dinucleotides retained after codon changes are underlined.<br>
[068]    FIG. 29 shows a Sabin 2 virus cassette d (VP1 region) sequence having maximum codon deoptimization. The original sequence (nucleotides 1975-2664 of SEQ ID NO: 3) is shown above the codon-replacement residues for the de-optimized Sabin 2 cassette d (VP1 region) sequence (SEQ ID NO: 69) (only replaced nucleotides are indicated); the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4). Original CG dinucleotides retained after codon changes are underlined.<br>
[069]    FIG. 30 shows a Sabin 2 virus cassette d (VP1 region) sequence that has MEF1 codons for Sabin 2 amino acids. The original sequence (nucleotides 1975- 2664 of SEQ ID NO: 3) is shown above the codon-replacement residues; the deduced amino acids are indicated below the triplets (amino acids 623 - 852 of SEQ ID NO: 4). The altered Sabin 2 cassette d (VP1 region) sequence<br>
(SEQ ID NO: 70) is shown below the original sequence (only replaced nucleotides are indicated). The amino acids that differ between Sabin 2 and MEF-1 are underlined.<br>
SEQUENCE LISTING<br>
[070]    The nucleic acid and amino acid sequences listed in the accompanying sequence listing are shown using standard letter abbreviations for nucleotide bases, and three-letter code for amino acids. Only one strand of each nucleic acid sequence is shown, but the complementary strand is understood as included by any reference to the displayed strand.<br>
[071]    SEQ ID NO: 1 is a primer sequence used to reverse transcribe poliovirus cDNA. [072]    SEQ ID NO: 2 is a primer sequence used to long PCR amplify poliovirus cDNA.<br>
[073]    SEQ ID NO: 3 is a capsid nucleic acid coding sequence of Sabin 2 (construct S2R9) poliovirus.<br>
[074]	SEQ ID NO: 4 is a protein sequence encoded by SEQ ID NO: 3.<br>
[075]	SEQ ID NO: 5 is a Sabin 2 codon-deoptimized nucleic acid sequence.<br>
[076]	SEQ ID NO: 6 is a capsid nucleic acid coding sequence of MEF1 poliovirus.<br>
[077]	SEQ ID NO: 7 is a protein sequence encoded by SEQ ID NO: 6.<br>
[078]	SEQ ID NO: 8 is an MEF1 codon-deoptimized nucleic acid sequence.<br>
[079]	SEQ ID NO: 9 is a capsid nucleic acid coding sequence of FMDV.<br>
[080]	SEQ ID NO: 10 is a protein sequence encoded by SEQ ID NO: 9.<br>
[081]	SEQ ID NO: 11 is an FMDV codon-deoptimized capsid nucleic acid sequence.<br>
[082]	SEQ ID NO: 12 is a spike glycoprotein nucleic acid coding sequence of SARS coronavirus.<br>
[083]	SEQ ID NO: 13 is a protein sequence encoded by SEQ ID NO: 12.<br>
[084]    SEQ ID NO: 14 is a SARS coronavirus codon-deoptimized spike glycoprotein nucleic acid sequence.<br>
[085]    SEQ ID NO: 15 is a nucleic acid coding sequence of rubella virus.<br>
[086]	SEQ ID NOS: 16 and 17 are protein sequences encoded by SEQ ID NO: 15.<br>
[087]	SEQ ID NO: 18 is a rubella codon-deoptimized nucleic acid sequence.<br>
[088]	SEQ ID NO: 19 is a gH nucleic acid coding sequence of VZV.<br>
[089]	SEQ ID NO: 20 is a protein sequence encoded by SEQ ID NO: 18.<br>
[090]	SEQ ID NO: 21 is a VZV codon-deoptimized gH nucleic acid sequence.<br>
[091]	SEQ ID NO: 22 is a gE nucleic acid coding sequence of VZV.<br>
[092]	SEQ ID NO: 23 is a protein sequence encoded by SEQ ID NO: 21.<br>
[093]	SEQ ID NO: 24 is a VZV codon-deoptimized gE nucleic acid sequence.<br>
[094]	SEQ ID NO: 25 is an F nucleic acid coding sequence of measles virus.<br>
[095]	SEQ ID NO: 26 is a protein sequence encoded by SEQ ID NO: 24.<br>
[096]	SEQ ID NO: 27 is a measles virus codon-deoptimized F nucleic acid sequence.<br>
[097]	SEQ ID NO: 28 is a hemagglutinin (H) nucleic acid coding sequence of measles virus.<br>
[098]	SEQ ID NO: 29 is a protein sequence encoded by SEQ ID NO: 27.<br>
[099]	SEQ ID NO: 30 is a measles codon-deoptimized H nucleic acid sequence.<br>
[0100] SEQ ID NO: 31 is an F nucleic acid coding sequence of RSV.<br>
[0101] SEQ ID NO: 32 is a protein sequence encoded by SEQ ID NO: 30.<br>
[0102] SEQ ID NO: 33 is a RSV codon-deoptimized F nucleic acid sequence.<br>
[0103]	SEQ ID NO: 34 is a G nucleic acid coding sequence of RSV.<br>
[0104]	SEQ ID NO: 35 is a protein sequence encoded by SEQ ID NO: 33.<br>
[0105]	SEQ ID NO: 36 is a RSV codon-deoptimized G nucleic acid sequence.<br>
[0106]	SEQ ID NO: 37 is a HA nucleic acid coding sequence of influenza virus.<br>
[0107]	SEQ ID NO: 38 is a protein sequence encoded by SEQ ID NO: 36.<br>
[0108] SEQ ID NO: 39 is an influenza virus codon-deoptimized HA nucleic acid sequence.<br>
[0109] SEQ ID NO: 40 is a NA nucleic acid coding sequence of influenza virus.<br>
[0110] SEQ ID NO: 41 is a protein sequence encoded by SEQ ID NO: 39.<br>
[0111] SEQ ID NO: 42 is an influenza codon-deoptimized NA nucleic acid sequence.<br>
[0112] SEQ ID NO: 43 is an env nucleic acid coding sequence of HIV-1.<br>
[0113] SEQ ID NO: 44 is a protein sequence encoded by SEQ ID NO: 42.<br>
[0114] SEQ ID NO: 45 is an HIV-1 codon-deoptimized env nucleic acid sequence.<br>
[0115] SEQ ID NO: 46 is an ArgS nucleic acid coding sequence of E. coli.<br>
[0116] SEQ ID NO: 47 is a protein sequence encoded by SEQ ID NO: 45.<br>
[0117] SEQ ID NO: 48 is an E. coli codon-deoptimized ArgS nucleic acid sequence.<br>
[0118] SEQ ID NO: 49 is an 7w/A nucleic acid coding sequence of E. coli.<br>
[0119] SEQ ID NO: 50 is a protein sequence encoded by SEQ ID NO: 48.<br>
[0120] SEQ ID NO: 51 is an E. coli codon-deoptimized 7w/A nucleic acid sequence.<br>
[0121] SEQ ID NO: 52 is a nucleic acid sequence showing the sequence of MEF1R1 or uncloned.<br>
[0122] SEQ ID NO: 53 is a nucleic acid sequence showing the sequence of MEF1R2.<br>
[0123] SEQ ID NO: 54 is a nucleic acid sequence showing the sequence of MEF1R5.<br>
[0124] SEQ ID NO: 55 is a nucleic acid sequence showing the sequence of MEF1R6.<br>
[0125] SEQ ID NO: 56 is a nucleic acid sequence showing the sequence of MEF1R7.<br>
[0126] SEQ ID NO: 57 is a nucleic acid sequence showing the sequence of MEF1R8.<br>
[0127] SEQ ID NO: 58 is a nucleic acid sequence showing the sequence of MEF1R9.<br>
[0128] SEQ ID NOS: 59-60 are primer sequences used to amplify the 3D region of Sabin 2.<br>
[0129] SEQ ID NO: 61 is a TaqMan probe used to detect the yield of amplicon generated using SEQ ID NOS: 59 and 60.<br>
[0130] SEQ ID NOS: 62-63 are primer sequences used to amplify the 3Dpo1 region of MEF1.<br>
[0131] SEQ ID NO: 64 is a TaqMan probe used to detect the yield of amplicon generated using SEQ ID NOS: 62 and 63.<br>
[0132] SEQ ID NO: 65 is a Sabin 2 cassette d (VP1 region) sequence with a reduced number of CG dinucleotides.<br>
[0133] SEQ ID NO: 66 is a Sabin 2 cassette d (VP1 region) sequence with a reduced number of CG and TA dinucleotides.<br>
[0134] SEQ ID NO: 67 is a Sabin 2 cassette d (VP1 region) sequence with an increased number of CG dinucleotides.<br>
[0135] SEQ ID NO: 68 is a Sabin 2 cassette d (VP1 region) sequence with an increased number of CG and TA dinucleotides.<br>
[0136] SEQ ID NO: 69 is an exemplary deoptimized Sabin 2 cassette d (VP1 region) sequence.<br>
[0137] SEQ ID NO: 70 is a Sabin 2 cassette d (VP1 region) sequence that uses MEF1 codons for Sabin 2 amino acids.<br>
DETAILED DESCRIPTION OF SEVERAL EMBODIMENTS<br>
Abbreviations and Terms<br>
[0138] The following explanations of terms and methods are provided to better describe the present disclosure and to guide those of ordinary skill in the art in the practice of the present disclosure. The singular forms "a," "an," and "the" refer to one or more than one, unless the context clearly dictates otherwise. For example, the term "comprising a nucleic acid molecule" includes single or plural nucleic acid molecules and is considered equivalent to the phrase "comprising at least one nucleic acid molecule." The term "or" refers to a single element of stated alternative elements or a combination of two or more elements, unless the context clearly indicates otherwise. As used herein, "comprises" means "includes." Thus, "comprising an alteration in the number of TA or CG dinucleotides," means "including an alteration in the number of TA dinucleotides, the number of CG dinucleotides, or the number of CG and TA dinucleotides," without excluding additional elements.<br>
[0139] Unless explained otherwise, all technical and scientific terms used herein have the same meaning as commonly understood to one of ordinary skill in the art to which this disclosure belongs.<br>
Although methods and materials similar or equivalent to those described herein can be used in the practice or testing of the present disclosure, suitable methods and materials are described below. The materials, methods, and examples are illustrative only and not intended to be limiting.<br>
[0140] OPV: oral poliovirus vaccine<br>
[0141] PV: poliovirus<br>
[0142] VAPP: vaccine-associated paralytic poliomyelitis<br>
[0143] VDPV: vaccine-derived poliovirus<br>
[0144] Adjuvant: A compound, composition, or substance that when used in combination with an immunogenic agent augments or otherwise alters or modifies a resultant immune response. In some examples, an adjuvant increases the titer of antibodies induced in a subject by the immunogenic agent. In another example, if the antigenic agent is a multivalent antigenic agent, an adjuvant alters the particular epitopic sequences that are specifically bound by antibodies induced in a subject.<br>
[0145] Exemplary adjuvants include, but are not limited to, Freund's Incomplete Adjuvant (IFA), Freund's complete adjuvant, B30-MDP, LA-15-PH, montanide, saponin, aluminum salts such as aluminum hydroxide (Amphogel, Wyeth Laboratories, Madison, NJ), alum, lipids, keyhole lympet protein, hemocyanin, the MF59 microemulsion, a mycobacterial antigen, vitamin E, non-ionic block polymers, muramyl dipeptides, polyanions, amphipatic substances, ISCOMs (immune stimulating complexes, such as those disclosed in European Patent EP 109942), vegetable oil, Carbopol, aluminium oxide, oil-emulsions (such as Bayol F or Marcol 52), E. coli heat-labile toxin (LT), Cholera toxin (CT), and combinations thereof.<br>
[0146] In one example, an adjuvant includes a DNA motif that stimulates immune activation, for example the innate immune response or the adaptive immune response by T-cells, B-cells, monocytes, dendritic cells, and natural killer cells. Specific, non-limiting examples of a DNA motif that stimulates immune activation include CG oligodeoxynucleotides, as described in U.S. Patent Nos. 6,194,388; 6,207,646; 6,214,806; 6,218,371; 6,239,116; 6,339,068; 6,406,705; and 6,429,199, and IL-2 or other immunomodulators.<br>
[0147] Administration: To provide or give a subject an agent, such as an immunogenic composition disclosed herein, by any effective route. Exemplary routes of administration include, but are not limited to, oral, injection (such as subcutaneous, intramuscular, intradermal, intraperitoneal, and intravenous), sublingual, rectal, transdermal, intranasal, vaginal, intraocular, and inhalation routes.<br>
[0148] Agent: Any substance, including, but not limited to, a chemical compound, molecule, peptidomimetic, pathogen, or protein.<br>
[0149] Antibody: A molecule including an antigen binding site which specifically binds (immunoreacts with) an antigen. Examples include polyclonal antibodies, monoclonal antibodies, humanized monoclonal antibodies, or immunologically effective portions thereof.<br>
[0150] Includes immunoglobulin molecules and immunologically active portions thereof. Immunoglobulin genes include the kappa, lambda, alpha, gamma, delta, epsilon, and mu constant region genes, as well as the myriad immunoglobulin variable region genes. Light chains are classified as either kappa or lambda. Heavy chains are classified as gamma, mu, alpha, delta, or epsilon, which in turn define the immunoglobulin classes, IgG, IgM, IgA, IgD and IgE, respectively.<br>
[0151] Antigen: A compound, composition, or substance that can stimulate the production of antibodies or a T-cell response in an animal, including compositions that are injected or absorbed into an animal. An antigen reacts with the products of specific humoral or cellular immunity, including those induced by heterologous immunogens. The term "antigen" includes all related antigenic epitopes. In one example, an antigen is an attenuated pathogen.<br>
[0152] Attenuated pathogen: A pathogen with a decreased or weakened ability to produce disease while retaining the ability to stimulate an immune response like that of the natural pathogen. In one example, a live pathogen is attenuated by deoptimizing one or more codons in one or more genes, such as an immunogenic surface antigen or a housekeeping gene. In another example, a pathogen is attenuated by selecting for avirulent variants under certain growth conditions (for example see Sabin and Boulger. J. Biol. Stand. 1:115-8; 1973; Sutler et al., 2003. Poliovirus vaccine - live, p. 651-705. In S. A. Plotkin and W. A. Orenstein (ed.), Vaccines, Fourth ed. W.B. Saunders Company, Philadelphia).<br>
[0153] Codons can be deoptimized, for example, by manipulating the nucleic acid sequence using molecular biology methods. Attenuated pathogens, such as an attenuated virus or bacterium, can be used in an immune composition to stimulate an immune response in a subject. For example, attenuated pathogens can be used in an attenuated vaccine to produce an immune response without causing the severe effects of the disease. Particular examples of attenuated vaccines include, but are not limited to, measles, mumps, rubella, polio, typhoid, yellow fever, and varicella vaccines.<br>
[0154] cDNA (complementary DNA): A piece of DNA lacking internal, non-coding segments (introns) and regulatory sequences that determine transcription. cDNA can be synthesized in the laboratory by reverse transcription from messenger RNA or viral extracted from cells or purified viruses.<br>
[0155] Cellular immunity: An immune response mediated by cells or the products they produce, such as cytokines, rather than by an antibody. It includes, but is not limited to, delayed type hypersensitivity and cytotoxic T cells.<br>
[0156] CG dinucleotide: A cytosine nucleotide immediately followed by a guanine in a nucleic acid sequence. Similarly, a TA (or UA) dinucleotide is a thymine (or uracil) nucleotide immediately followed by a adenine in a nucleic acid sequence. For example, the sequence GTAGTCGACT (nucleotides 1-10 of SEQ ID NO: 2) has one CG dinucleotide and one TA dinucleotide (underlined).<br>
[0157] Codon: A specific sequence of three adjacent nucleotide bases on a strand of DNA or RNA that provides genetic code information for a particular amino acid or a termination signal.<br>
[0158] Conservative substitution: One or more amino acid substitutions for amino acid residues having similar biochemical properties. Typically, conservative substitutions have little to no impact on the activity of a resulting polypeptide. For example, a conservative substitution is an amino acid substitution in an antigenie epitope of a pathogenic peptide that does not substantially affect the ability of an antibody that specifically binds to the unaltered epitope to specifically bind the epitope including the conservative substitution. Thus, in some examples, a conservative variant of an epitope is also a functional variant of the epitope.<br>
[0159] Methods which can be used to determine the amount of recognition by a variant epitope are disclosed herein. In addition, an alanine scan can be used to identify which amino acid residues in a pathogenic epitope can tolerate an amino acid substitution. In one example, recognition is not decreased by more than 25%, for example not more than 20%, for example not more than 10%, when an alanine, or other conservative amino acid (such as those listed below), is substituted for one or more native amino acids. Similarly, an ELISA assay can be used that compares a level of specific binding of an antibody that specifically binds a particular antigenic peptide to a level of specific binding of the antibody to a corresponding peptide with the substitution(s) to determine if the substitution(s) does not substantially affect specific binding of the substituted peptide to the antibody.<br>
[0160] In one example, one, two, three, five, or ten conservative substitutions are included in the peptide. In another example, 1-10 conservative substitutions are included in the peptide. In a further embodiment, at least 2 conservative substitutions are included in the peptide. A peptide can be produced to contain one or more conservative substitutions by manipulating the nucleotide sequence that encodes that polypeptide using, for example, standard procedures such as site-directed mutagenesis or PCR. Alternatively, a polypeptide can be produced to contain one or more conservative substitutions by using standard peptide synthesis methods.<br>
[0161] Substitutional variants are those in which at least one residue in the amino acid sequence has been removed and a different residue inserted in its place. Conservative substitution tables providing<br>
functionally similar amino acids are well known in the art. Examples of amino acids which may be substituted for an original amino acid in a protein and which are regarded as conservative substitutions include: Ser for Ala; Lys for Arg; Gin or His for Asn; Glu for Asp; Ser for Cys; Asn for Gin; Asp for Glu; Pro for Gly; Asn or Gin for His; Leu or Val for He; He or Val for Leu; Arg or Gin for Lys; Leu or lie for Met; Met, Leu or Tyr for Phe; Thr for Ser; Ser for Thr; Tyr for Trp; Trp or Phe for Tyr; and He or Leu for Val.<br>
[0162] Further information about conservative substitutions can be found, among other sources, Ben-Bassat et a!., (J. Bacterial. 169:751-7, 1987), O'Regan et al., (Gene 77:237-51, 1989), Sahin-Toth et al., (Protein Sci. 3:240-7, 1994), Hochuli et a\., (Bio/Technology 6:1321-5, 1988) and in standard textbooks of genetics and molecular biology.<br>
[0163] DNA (deoxyribonucleic acid): A long chain polymer which includes the genetic material of most living organisms (many viruses have genomes containing only ribonucleic acid, RNA). The repeating units in DNA polymers are four different nucleotides, each of which includes one of the four bases, adenine, guanine, cytosine and thymine bound to a deoxyribose sugar to which a phosphate group is attached. Triplets of nucleotides, referred to as codons, in DNA molecules code for amino acid in a polypeptide. The term codon is also used for the corresponding (and complementary) sequences of three nucleotides in the mRNA into which the DNA sequence is transcribed,<br>
[0164] Degenerate variant: A nucleic acid sequence encoding a peptide that includes a sequence that is degenerate as a result of the genetic code. There are 20 natural amino acids, most of which are specified by more than one of the 61 codons of the "universal" genetic code used by most cells and viruses. For example, the amino acid Ala is encoded by four codon triplets: GCU, GCG, GCA, and GCC. Therefore, all degenerate nucleotide sequences are included as long as the amino acid sequence of the peptide encoded by the nucleotide sequence is unchanged.<br>
[0165] Deoptimization of a codon: To replace a preferred codon in a nucleic acid sequence with a synonymous codon (one that codes for the same amino acid) less frequently used (unpreferred) in the organism. Each organism has a particular codon usage bias for each amino acid, which can be determined from publicly available codon usage tables (for example see Nakamura et al., Nucleic Acids flay. 28:292, 2000 and references cited therein; Sharp et al., Nucleic Acids Res. 16:8207-11, 1988; Chou and Zhang, AIDS Res. Hum. Retroviruses. Dec;8(12):1967-76, 1992; West and Iglewski et al, Nucleic Acids Res. 16:9323-35, 1988, Rothberg and Wimmer, Nucleic Acids Res. 9:6221-9, 1981; Jenkins et al, J. Mol Evol 52:383-90, 2001; and Watterson, Mol Biol Evol. 9:666-77, 1992; all herein incorporated by reference). In addition, codon usage tables are available for several organisms on the internet at GenBank's website.<br>
 [0166] For example, if an organism has a codon usage for the amino acid Val of 15% for GUU, 10% for GUC, 50% for GUA, and 25% for GUG, the "least frequently used codon" is GUC. Therefore, to deoptimize a Val codon, the codon GUC could be used to replace one or more of the codons GUU, GUA, or GUG in a native sequence. Similarly, the codon GUU is a "less frequently used codon" than the GUA codon, and therefore, GUU could be used to replace GUA.<br>
[0167] In some examples, the choice of the less frequently used codon is made depending on whether the codon will alter the G+C conent, the number of CG dinucleotides, the number of TA(UA) dinucleotides, or combinations thereof, in the deoptimized sequence. For example, if an organism has a codon usage for the amino acid Val of 50% for GUU, 10% for GUC, 15% for GUA, and 25% for GUG, the codon GUA is a "less frequently used codon" than the GUU codon, and could be used to replace GUU, for example if it was desired to increase the number of UA (TA) dinucleotides in the deoptimized sequence. Similarly, the the codon GUG is a "less frequently used codon" than the GUU codon, and could be used to replace GUU, for example if it was desired to increase the G+C conent of the deoptimized sequence.<br>
[0168] Deoptimized pathogen: A pathogen having a nucleic acid coding sequence with one or more deoptimized codons, which decrease the replicative fitness of the pathogen. In some examples, refers to the isolated deoptimized nucleic acid sequence itself, independent of the pathogenic organism.<br>
[0169] Epitope: An antigenic determinant. Chemical groups or peptide sequences on a molecule that are antigenic, that is, that elicit a specific immune response. An antibody binds a particular antigenic epitope, or a T-cell reacts with a particular antigenic epitope bound to a specific MHC molecule. In some examples, an epitope has a minimum sequence of 6-8 amino acids, and a maximum sequence of about 100 amino acids, for example, about 50, 25 or 18 amino acids in length.<br>
[0170] Functional variant: Sequence alterations in a peptide, wherein the peptide with the sequence alterations retains a function or property (such as immunogenicity) of the unaltered peptide. For example, a functional variant of an epitope can specifically bind an antibody that binds an unaltered form of the epitope or stimulates T-cell proliferation to an extent that is substantially the same as the unaltered form of the epitope. Sequence alterations that provide functional variants can include, but are not limited to, conservative substitutions, deletions, mutations, frameshifts, and insertions. Assays for determining antibody binding and T-cell reactivity are well known in the art.<br>
[0171] Screens for immunogenicity can be performed using well known methods such as those described in Harlow and Lane, Antibodies: A Laboratory Manual, Cold Spring Harbor Laboratory, 1988, or in Paul, Fundamental Immunology, 3rd ed., 243-247 (Raven Press, 1993) and references cited therein. For example, a peptide can be immobilized on a solid support and contacted with<br>
subject sera to allow binding of antibodies within the sera to the immobilized polypeptide. Unbound sera may then be removed and bound antibodies detected using, for example, I25l-labeled Protein A. The ability of a functional variant to react with antigen-specific antisera may be unchanged relative to original epitope, or may be enhanced or diminished by less than 30%, for example, less than 20%, such as less than 10%, relative to the unaltered epitope.<br>
[0172] G+C content: The amount of guanine (G) and cytosine (C) in a nucleic acid sequence (such as a pathogen coding sequence). In particular examples, the amount can be expressed in mole fraction or percentage of total number of bases in the sequence. For example, the sequence GTAGTCGACT (nucleotides 1-10 of SEQ ID NO: 2) would be said to have a G+C content of 50% (5 of the 10 bases are guanine and cytosine).<br>
[0173] Humoral immunity: Immunity that can be transferred with immune serum from one subject to another. Typically, humoral immunity refers to immunity resulting from the introduction of specific antibodies or stimulation of the production of specific antibodies, for example by administration of one or more of the pathogens with decreased replicative fitness disclosed herein.<br>
[0174] Hybridization: The binding of a nucleic acid molecule to another nucleic acid molecule, for example the binding of a single-stranded DNA or RNA to another nucleic acid, thereby forming a duplex molecule. The ability of one nucleic acid molecule to bind to another nucleic acid molecule can depend upon the complementarity between the nucleotide sequences of two nucleic acid molecules, and the stringency of the hybridization conditions.<br>
[0175] Methods of performing hybridization are known in the art (such as those described in sections 7.39-7.52 of Sambrook et al., (1989) Molecular Cloning, second edition, Cold Spring Harbor Laboratory, Plainview, NY.). For example, Southern or Northern analysis can be used to determine if one nucleic acid sequence hybridizes to another nucleic acid sequence.<br>
[0176] Deoptimized nucleic acid molecules are disclosed herein, such as SEQ ID NOs: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, and 69. However, the present disclosure encompasses other deoptimized nucleic acid molecules that can hybridize to any of SEQ ID NOs: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42,45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69, under moderate or high stringent conditions. In some examples, sequences that can hybridize to any of SEQ ID NOs: 5, 8, 11, 14, 18, 21,. 24, 27, 30, 33, 36, 39,42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69 are at least 100 nucleotides in length (such as at least 500, at least 750, at least 1000, at least 2500, or at least 5000 nucleotides in length) and hybridize, under moderate or high hybridization conditions, to the sense or antisense strand of any of SEQ ID NOs: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69.<br>
[0177] Moderately stringent hybridization conditions are when the hybridization is performed at about 42°C in a hybridization solution containing 25 mM KPO4 (pH 7.4), 5X SSC, 5X Denhart's solution, 50 (xg/mL denatured, sonicated salmon sperm DNA, 50% formamide, 10% Dextran sulfate, and 1-15 ng/mL probe (about 5xl07 cpm/ug), while the washes are performed at about 50°C with a wash solution containing 2X SSC and 0.1% sodium dodecyl sulfate.<br>
[0178] Highly stringent hybridization conditions are when the hybridization is performed at about 42°C in a hybridization solution containing 25 mM KPO4 (pH 7.4), 5X SSC, 5X Denhart's solution, 50 ug/mL denatured, sonicated salmon sperm DNA, 50% formamide, 10% Dextran sulfate, and 1-15 ng/mL probe (about 5xl07 cpm/ug), while the washes are performed at about 65°C with a wash solution containing 0.2X SSC and 0.1% sodium dodecyl sulfate.<br>
[0179] Immune response: A response of a cell of the immune system, such as a B-cell, T-cell, macrophage, monocyte, or polymorphonucleocyte, to an immunogenic agent (such as the disclosed pathogens having decreased replicative fitness or sequences therefrom) in a subject. An immune response can include any cell of the body involved in a host defense response, such as an epithelial cell that secretes interferon or a cytokine. An immune response includes, but is not limited to, an innate immune response or inflammation.<br><br>
a<br>
[0180] The response can be specific for a particular antigen (an "antigen-specific response"). In particular example, an immune response is a T cell response, such as a CD4+ response or a CD8+ response. In another example, the response is a B cell response, and results in the production of specific antibodies to the immunogenic agent.<br>
[0181] In some examples, such an immune response provides protection for the subject from the immunogenic agent or the source of the immunogenic agent. For example, the response can protect a subject, such as a human or veterinary subject, from infection by a pathogen, or interfere with the progression of an infection by a pathogen. An immune response can be active and involve stimulation of the subject's immune system, or be a response that results from passively acquired immunity.<br>
[0182] Immunity: The state of being able to mount a protective response upon exposure to an immunogenic agent (such as the disclosed pathogens having decreased replicative fitness or sequences therefrom). Protective responses can be antibody-mediated or immune cell-mediated, and can be directed toward a particular pathogen. Immunity can be acquired actively (such as by exposure to an immunogenic agent, either naturally or in a pharmaceutical composition) or passively (such as by administration of antibodies).<br>
[0183] Immunogen: An agent (such as a compound, composition, or substance) that can stimulate or elicit an immune response by a subject's immune system, such as stimulating the production of<br>
antibodies or a T-cell response in a subject. Immunogenic agents include, but are not limited to, pathogens (such as the disclosed pathogens having decreased replicative fitness or sequences therefrom) and their corresponding proteins. One specific example of an immunogenic composition is a vaccine.<br>
[0184] Immunogenic carrier: An immunogenic macromolecule to which an antigenic molecule (such as a pathogen with decreased replicative fitness) is bound. When bound to a carrier, the bound molecule becomes more immunogenic, such as an increase of at least 5%, at least 10%, at least 20%, or even at least 50%. Carriers can be used to increase the immunogenicity of the bound molecule or to elicit antibodies against the carrier which are diagnostically, analytically, or therapeutically beneficial. Covalent linking of a molecule to a carrier confers enhanced immunogenicity and T-cell dependence (Pozsgay et al., PNAS 96:5194-97, 1999; Lee etal, J. Immunol. 116:1711-18, 1976; Dintzis et al., PNAS 73:3671-75, 1976). Exemplary carriers include polymeric carriers, which can be natural (for example, polysaccharides, polypeptides or proteins from bacteria or viruses), semi-synthetic or synthetic materials containing one or more functional groups to which a reactant moiety can be attached.<br>
[0185] Examples of bacterial products for use as carriers include, but are not limited to, bacterial toxins, such as B. anthracis PA (including fragments that contain at least one antigenic epitope and analogs or derivatives capable of eliciting an immune response), LF and LeTx, and other bacterial toxins and toxoids, such as tetanus toxin/toxoid, diphtheria toxin/toxoid, P. aeruginosa exotoxin/toxoioV, pertussis toxin/toxoid, and C. perfringem exotoxin/toxoid. Viral proteins, such as hepatitis B surface antigen and core antigen can also be used as carriers, as well as proteins from higher organisms such as keyhole limpet hemocyanin, horseshoe crab hemocyanin, edestin, mammalian serum albumins, and mammalian immunoglobulins. Additional bacterial products for use as carriers include, but are not limited to, bacterial wall proteins and other products (for example, streptococcal or staphylococcal cell walls and lipopolysaccharide (LPS)).<br>
[0186] Immunogenicity: The ability of an agent to induce a humoral or cellular immune response. Immunogenicity can be measured, for example, by the ability to bind to an appropriate MHC molecule (such as an MHC Class I or II molecule) and to induce a T-cell response or to induce a B-cell or antibody response, for example, a measurable cytotoxic T-cell response or a serum antibody response to a given epitope. Immunogenicity assays are well-known in the art and are described, for example, in Paul, Fundamental Immunology, 3rd ed., 243-247 (Raven Press, 1993) and references cited therein.<br>
[0187] Immunologically Effective Dose: A therapeutically effective amount of an immunogen (such as the disclosed pathogens having decreased replicative fitness or sequences therefrom) that<br>
will prevent, treat, lessen, or attenuate the severity, extent or duration of a disease or condition, for example, infection by a pathogen.<br>
[0188] Isolated: An "isolated" biological component (such as, a nucleic acid molecule or protein) has been substantially separated, produced apart from, or purified away from other biological components in the cell of the organism in which the component occurs, for example, other chromosomal and extra-chromosomal DNA and RNA, and proteins. Nucleic acid molecules and proteins which have been "isolated" include nucleic acid molecules and proteins purified by standard purification methods. The term also embraces nucleic acid molecules and proteins prepared by recombinant expression in a host cell as well as chemically synthesized proteins and nucleic acids. Samples of isolated biological components include samples of the biological component wherein the biological component represents greater than 90% (for example, greater than 95%, such as greater than 98%) of the sample.<br>
[0189] An "isolated" microorganism (such as a virus, bacterium, fungus, or protozoa) has been substantially separated or purified away from microorganisms of different types, strains, or species. Microorganisms can be isolated by a variety of techniques, including serial dilution and culturing.<br>
[0190] Lymphocytes: A type of white blood cell involved in the immune defenses of the body. There are two main types of lymphocytes: B-cells and T-cells.<br>
[0191] Mimetic: A molecule (such as an organic chemical compound) that mimics the activity of another molecule.<br>
[0192] Nucleic acid molecule: A deoxyribonucleotide or ribonucleotide polymer including, without limitation, cDNA, mRNA, genomic DNA, genomic RNA, and synthetic (such as chemically synthesized) DNA. Includes nucleic acid sequences that have naturally-occurring, modified, or non-naturally-occurring nucleotides linked together by naturally-occurring or non-naturally-occurring nucleotide linkages. Nucleic acid molecules can be modified chemically or biochemically and can contain non-natural or derivatized nucleotide bases. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with analogs, and internucleotide linkage modifications.<br>
[0193] Nucleic acid molecules can be in any topological conformation, including single-stranded, double-stranded, partially duplexed, triplexed, hairpinned, circular, linear, and padlocked conformations. Where single-stranded, a nucleic acid molecule can be the sense strand or the antisense strand. Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known and include, for example, molecules in which peptide linkages are substituted for phosphate linkages in the backbone.<br>
[0194] The disclosure includes isolated nucleic acid molecules that include specified lengths of a nucleotide sequence. Such molecules can include at least 10, at least 15, at least 20, at least 25, at least 30, at least 35, at least 40, at least 45, at least 50, at least 100, at least 300 or at least 500 nucleotides of these sequences or more, and can be obtained from any region of a nucleic acid molecule.<br>
[0195] Nucleotide: A subunit of DNA or RNA including a nitrogenous base (adenine, guanine, thymine, or cytosine in DNA; adenine, guanine, uracil, or cytosine in RNA), a phosphate molecule, and a sugar molecule (deoxyribose in DNA and ribose in RNA).<br>
[0196] Passive immunity: Immunity acquired by the introduction by immune system components into a subject rather than by stimulation.<br>
[0197] Pathogen: A disease-producing agent. Examples include, but are not limited to microbes such as viruses, bacteria, fungi, and protozoa.<br>
[0198] Peptide, polypeptide, and protein: Polymers of amino acids (typically L-amino acids) or amino acid mimetics linked through peptide bonds or peptide bond mimetic to form a chain. The terminal amino acid at one end of the chain typically has a free amino group (the amino-terminus), while the terminal amino acid at the other end of the chain typically has a free carboxyl group (the carboxy terminus). Encompasses any amino acid sequence and includes modified sequences such as glycoproteins. The terms cover naturally occurring proteins, as well as those which are recombinantly or synthetically produced.<br>
[0199] As used herein, the terms are interchangeable since they all refer to polymers of amino acids (or their analogs) regardless of length. Non-natural combinations of naturally- or non-naturally occurring sequences of amino acids may also be referred to as "fusion proteins."<br>
[0200] Pharmaceutically Acceptable Carriers: The pharmaceutically acceptable carriers (vehicles) useful in this disclosure are conventional. Remington's Pharmaceutical Sciences, by E. W. Martin, Mack Publishing Co., Easton, PA, 15th Edition (1975), describes compositions and formulations suitable for pharmaceutical delivery of one or more therapeutic compounds or molecules, such as one or more nucleic acid molecules, proteins or immunogenic compositions disclosed herein.<br>
[0201] In general, the nature of the carrier will depend on the particular mode of administration being employed. For instance, parenteral formulations can include injectable fluids that include pharmaceutically and physiologically acceptable fluids such as water, physiological saline, balanced salt solutions, aqueous dextrose, glycerol or the like as a vehicle. In addition to biologically-neutral "carriers, pharmaceutical compositions to be administered can contain minor amounts of non-toxic<br>
auxiliary substances, such as wetting or emulsifying agents, preservatives, and pH buffering agents and the like, for example sodium acetate or sorbitan monolaurate, sodium lactate, potassium chloride, calcium chloride, and triethanolamine oleate.<br>
[0202] Poliovirus (PV): An enterovirus of the Picornaviridae family that is the causative agent of poliomyelitis (polio).<br>
[0203] Purified: The term purified does not require absolute purity; rather, it is intended as a relative term. Thus, for example, a purified peptide preparation is one in which the peptide is more enriched than the peptide is in its natural environment within a cell or cell extract. In one example, a preparation is purified such that the purified peptide represents at least 50% of the total peptide content of the preparation. In other examples, a peptide is purified to represent at least 90%, such as at least 95%, or even at least 98%, of all macromolecular species present in a purified preparation prior to admixture with other formulation ingredients, such as a pharmaceutical carrier, excipient, buffer, absorption enhancing agent, stabilizer, preservative, adjuvant or other co-ingredient. In some examples, the purified preparation is be essentially homogeneous, wherein other macromolecular species are not detectable by conventional techniques.<br>
[0204] Such purified preparations can include materials in covalent association with the active agent, such as glycoside residues or materials admixed or conjugated with the active agent, which may be desired to yield a modified derivative or analog of the active agent or produce a combinatorial therapeutic formulation, conjugate, fusion protein or the like. The term purified thus includes such desired products as peptide and protein analogs or mimetics or other biologically active compounds wherein additional compounds or moieties are bound to the active agent in order to allow for the attachment of other compounds or provide for formulations useful in therapeutic treatment or diagnostic procedures.<br>
[0205] Quantitating: Determining a relative or absolute quantity of a particular component in a sample. For example, in the context of quantitating antibodies in a sample of a subject's blood to detect infection by a pathogen, quantitating refers to determining the quantity of antibodies using an antibody assay, for example, an ELISA-assay or a T-cell proliferation assay.<br>
[0206] Recombinant: A recombinant nucleic acid molecule is one that has a sequence that is not naturally occurring or has a sequence that is made by an artificial combination of two otherwise separated segments of sequence. This artificial combination can be accomplished by chemical synthesis or by the artificial manipulation of isolated segments of nucleic acids, for example, by genetic engineering techniques such as those described in Sambrook et al. (ed.), Molecular Cloning: A Laboratory Manual, 2nd ed., vol. 1-3, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989. The term recombinant includes nucleic acid molecules that have been altered solely by<br>
addition, substitution, or deletion of a portion of the nucleic acid molecule. Similarly, a recombinant protein is one encoded for by a recombinant nucleic acid molecule.<br>
[0207] Replicative fitness: The ability of a pathogen to produce mature infectious progeny. In some examples, introduction of one or more deoptimized codons into a pathogen reduces the replicative fitness of the pathogen, as compared to a pathogen containing native codons. In particular examples, introduction of one or more deoptimized codons into a pathogen, in combination with altering the G+C content or alterning the number of CG or TA dinucleotides in a coding sequence, reduces the replicative fitness of the pathogen, as compared to a pathogen containing native codons. In some examples, such replicative fitness is reduced by at least 10%, such as at least 20%, at least 50%, or even at least 90% as compared to a pathogen containing native codons.<br>
[0208] Methods that can be used to determine replicative fitness are disclosed herein and are known in the art. For example, to determine the replicative fitness of a virus, plaque size can be determined, infectious center assays can be used, viral liter by TCID50 (tissue-culture infectious doses 50%) or plaque assay, replication in single-step growth curves, temperature-sensitivity or cold-sensitivity of plaques determined, unusual host range observed, or competition assays with a related virus can be determined. To determine the replicative fitness of a bacterium or fungus, exemplary replicative fitness assays include assays for colony-forming activity, temperature-sensitivity, cold-sensitivity, slow growth under certain conditions, increased or rapid bacterial death, reduced ability of the bacteria or fungi to survive various stress conditions (such as nutrient deprivation), altered host range, enzymatic assays indicating reduced activity of a key enzyme, or assays for reduced pathogenicity due to decreased expression of an important protein (such as LPS).<br>
[0209] Specific Binding Agent: An agent that binds substantially only to a defined target. Thus a protein-specific binding agent binds substantially only the defined protein, or to a specific region within the protein. As used herein, a specific binding agent includes antibodies and other agents that bind substantially to a specified peptide.<br>
[0210] The determination that a particular agent binds substantially only to a specific peptide can readily be made by using or adapting routine procedures. One suitable in vitro assay makes use of the Western blotting procedure (described in many standard texts, including Harlow and Lane, Using Antibodies: A Laboratory Manual, CSHL, New York, 1999).<br>
[0211] Specifically bind: Refers to the ability of a particular agent (a "specific binding agent") to specifically react with a particular analyte, for example to specifically immunoreact with an antibody, or to specifically bind to a particular peptide sequence. The binding is a non-random binding reaction, for example between an antibody molecule and an antigenic determinant. Binding specificity of an antibody is typically determined from the reference point of the ability of the antibody to differentially bind the specific antigen and an unrelated antigen, and therefore distinguish<br>
between two different antigens, particularly where the two antigens have unique epitopes. An antibody that specifically binds to a particular epitope is referred to as a "specific antibody".<br>
[0212] In particular examples, two compounds are said to specifically bind when the binding constant for complex formation between the components exceeds about 104 L/mol, for example, exceeds about 106 L/mol, exceeds about 108 L/mol, or exceeds about 1010L/mol. The binding constant for two components can be determined using methods that are well known in the art.<br>
[0213] Subject: Living multi-cellular organisms, a category that includes human and non-human mammals, as well as other veterinary subjects such as fish and birds.<br>
[0214] Therapeutically effective amount: An amount of a therapeutic agent (such as an immunogenic composition) that alone, or together with an additional therapeutic agent(s), induces the desired response, such as a protective immune response or therapeutic response to a pathogen. In one example, it is an amount of immunogen needed to increase resistance to, prevent, ameliorate, or treat infection and disease caused by a pathogenic infection in a subject. Ideally, a therapeutically effective amount of an immunogen is an amount sufficient to increase resistance to, prevent, ameliorate, or treat infection and disease caused by a pathogen without causing a substantial cytotoxic effect in the subject. The preparations disclosed herein are administered in therapeutically effective amounts.<br>
[0215] In general, an effective amount of a composition administered to a human or veterinary subject will vary depending upon a number of factors associated with that subject, for example whether the subject previously has been exposed to the pathogen. An effective amount of a composition can be determined by varying the dosage of the product and measuring the resulting immune or therapeutic responses, such as the production of antibodies. Effective amounts also can be determined through various in vitro, in vivo or in situ immunoassays. The disclosed therapeutic agents can be administered in a single dose, or in several doses, as needed to obtain the desired response. However, the effective amount of can be dependent on the source applied, the subject being treated, the severity and type of the condition being treated, and the manner of administration.<br>
[0216] The disclosed therapeutic agents can be administered alone, or in the presence of a pharmaceutically acceptable carrier, or in the presence of other agents, for example an adjuvant.<br>
[0217] In one example, a desired response is to increase an immune response in response to infection with a pathogen. For example, the therapeutic agent can increase the immune response by a desired amount, for example by at least 5%, at least 10%, at least 15%, at least 20%, at least 25%, at least 30%, at least 50%, at least 75%, or even at least 90%, as compared to an immune response in the absence of the therapeutic agent. This increase can result in decreasing or slowing the progression of, a disease or condition associated with a pathogenic infection.<br>
[0218] In another example, a desired response is to decrease the incidence of vaccine-associated paralytic poliomyelitis in response to an attenuated Sabin oral polio vaccine. The incidence of vaccine-associated paralytic poliomyelitis does not need to be completely eliminated for a therapeutic agent, such as a pharmaceutical preparation that includes an immunogen, to be effective. For example, the therapeutic agent (such as a codon-deoptimized oral polio vaccine) can decrease the incidence of vaccine-associated paralytic poliomyelitis or the emergence of circulating vaccine-derived polioviruses by a desired amount, for example by at least 5%, at least 10%, at least 15%, at least 20%, at least 25%, at least 30%, at least 50%, at least 75%, or even at least 90%, as compared to the incidence of vaccine-associated paralytic poliomyelitis or the emergence of circulating vaccine-derived polioviruses in the presence of a oral polio vaccine containing native codons.<br>
[0219] Treating a disease: "Treatment" refers to a therapeutic intervention that ameliorates a sign or symptom of a disease or pathological condition related to a disease, even if the underlying pathophysiology is not affected. Reducing a sign or symptom associated with a pathogenic infection can be evidenced, for example, by a delayed onset of clinical symptoms of the disease in a susceptible subject, a reduction in severity of some or all clinical symptoms of the disease, a slower progression of the disease, a reduction in the number of relapses of the disease, an improvement in the overall health or well-being of the subject, or by other parameters well known in the art that are specific to the particular disease.<br>
[0220] Treatment can also induce remission or cure of a condition, such as a pathogenic infection or a pathological condition associated with such an infection. In particular examples, treatment includes preventing a disease, for example by inhibiting or even avoiding altogether the full development of a disease or condition, such as a disease associated with a pathogen, such as polio. Thus, prevention of pathogenic disease can include reducing the number of subjects who acquire a disease associated with a pathogenic infection (such as the development of polio or poliomyelitis by the polio virus or development of rabies by the rabies virus) in a population of subjects receiving a preventative treatment (such as vaccination) relative to an untreated control population, or delaying the appearance of such disease in a treated population versus an untreated control population. Prevention of a disease does not require a total absence of disease. For example, a decrease of at least 50% can be sufficient.<br>
[0221] Unit dose: A physically discrete unit containing a predetermined quantity of an active material calculated to individually or collectively produce a desired effect such as an immunogenic effect. A single unit dose or a plurality of unit doses can be used to provide the desired effect, such as an immunogenic effect. In one example, a unit dose includes a desired amount of one or more of the disclosed pathogens having reduced replicative fitness.<br>
[0222] Vaccine: An immunogenic composition that can be administered to an animal or a human to confer immunity, such as active immunity, to a disease or other pathological condition. Vaccines can be used prophylactically or therapeutically. Thus, vaccines can be used reduce the likelihood of infection or to reduce the severity of symptoms of a disease or condition or limit the progression of the disease or condition. In one example, a vaccine includes one or more of the disclosed pathogens having reduced replicative fitness.<br>
[0223] Vector: A nucleic acid molecule as introduced into a host cell, thereby producing a transformed host cell. A vector can include nucleic acid sequences that permit it to replicate in the host cell, such as an origin of replication. A vector can also include one or more therapeutic genes or selectable marker genes and other genetic elements known in the art. A vector can transduce, transform or infect a cell, thereby causing the cell to express nucleic acid molecules or proteins other than those native to the cell. A vector optionally includes materials to aid in achieving entry of the nucleic acid into the cell, such as a viral particle, liposome, protein coating or the like. In one example, a vector is a viral vector. Viral vectors include, but are not limited to, retroviral and adenoviral vectors.<br>
Deoptimizing Codon Usage to Decrease Replicative Fitness<br>
[0224] This disclosure provides methods of decreasing the replicative fitness of a pathogen by deoptimizing codon usage in one or more genes of the pathogen. Such methods can be used to increase the genetic stability of the attenuated phenotype of currently available attenuated vaccines, as well as to generate new attenuated pathogens that can be used in immunogenic compositions. For example, the attenuated Sabin oral polio vaccine (OPV) strains are genetically unstable. This instability is the underlying cause of vaccine-associated paralytic poliomyelitis and the emergence of circulating vaccine-derived polioviruses. Therefore, the disclosed compositions and methods can be used to reduce the incidence of vaccine-associated paralytic poliomyelitis and other disorders caused by currently available live attenuated vaccines. The disclosed methods and compositions increase the genetic stability of pathogens by distributing attenuating mutations over many sites within the pathogen's genome.<br>
[0225] Codon usage bias, the use of synonymous codons at unequal frequencies, is ubiquitous among genetic systems (Ikemura, J. Mol. Biol. 146:1-21, 1981; Ikemura, J. Mol. Biol. 158:573-97, 1982). The strength and direction of codon usage bias is related to genomic G + C content and the relative abundance of different isoaccepting tRNAs (Akashi, Curr. Opin, Genet. Dev. 11:660-6, 2001; Duret, Curr. Opin. Genet. Dev. 12:640-9, 2002; Osawaetal., Microbiol. Rev. 56:229-64, 1992). Codon usage can affect the efficiency of gene expression. In Escherichia coli (Ikemura, J. Mol. Biol. 146:1-21, 1981; Xia Genetics 149:37-44, 1998), Saccharomyces cerevisiae (Bennetzen and Hall, J. Biol. Chem. 257:3026-31, 1982; Ikemura, J. Mol. Biol. 158:573-97, 1982), Caenorhabditis elegans<br>
(Duret, Curr. Opin. Genet. Dev. 12:640-9, 2002), Drosophila melanogaster (Moriyama and Powell, J. Mol. Evol. 45:514-23, 1997), and Arabidopsis thaliana (Chiapello et al. Gene 209:GC1-GC38, 1998) the most highly expressed genes use codons matched to the most abundant tRNAs (Akashi and Eyre-Walker, Curr. Opin. Genet. Dev. 8:688-93, 1998). By contrast, in humans and other vertebrates, codon usage bias is more strongly correlated with the G + C content of the isochore where the gene is located (Musto et al., Mol. Biol. Evol. 18:1703-7, 2001; Urrutia and Hurst, Genetics 159:1191-9, 2001) than with the breadth or level of gene expression (Duret, Curr. Opin. Genet. Dev. 12:640-9, 2002) or the number of tRNA genes (Kanaya et al., J. Mol. Evol. 53:290-8, 2001).<br>
[0226] The deoptimized nucleic acid sequences of the present application include one or more codons that are degenerate as a result of the genetic code. There are 20 natural amino acids, most of which are specified by more than one codon. However, organisms have codons which are used more frequently, and those that are used less frequently (deoptimized). All possible deoptimized nucleotide sequences are included in the disclosure as long as the deoptimized nucleotide sequence retains the ability to decrease replicative fitness, for example by at least 10%, at least 20%, at least 50% or even at least 75% as compared to the replicative fitness of a pathogen with a codon optimized nucleic acid sequence.<br>
[0227] Optimization of codon composition is frequently required for efficient expression of genes in heterologous host systems (Andre et al., J. Virol 72:1497-503, 1998; Kane, Curr. Opin. Biotech. 6:494-500, 1995; Smith, Biotech. Prog. 12:417-22, 1996; Yadava and Ockenhouse. Infect. Immun. 71:4961-9, 2003). Conversely, engineered codon deoptimization can dramatically decrease the efficiency of gene expression in several organisms (Robinson et al., Nucleic Acids Res. 12:6663-71, 1984; Hoekema et al., Mol. Cell Biol. 7:2914-24, 1987; Garlini and Stephan. Genetics 163:239-43, 2003; and Zhou et al., J. Virol. 73:4972-82, 1999). However, it has not been previously taught or suggested that deoptimization of sequences of a microbial pathogen (such as a housekeeping or antigenic sequence) could be used to systematically reduce the replicative fitness of the pathogen, thereby producing a novel approach for developing attenuated derivatives of the pathogen having well-defined levels of replicative fitness, and increasing the genetic stability of the attenuated phenotype.<br>
Selection of codons to deoptimize<br>
[0228] The methods provided herein include deoptimizing at least one codon in a coding sequence of a pathogen, thereby generating a deoptimized coding sequence. Such deoptimization reduces replicative fitness of the pathogen. In particular examples, methods of reducing the replicative fitness of a pathogen include identifying one or more amino acids that are encoded by at least 2 different codons in the pathogen (such as 2 different codons, 3 different codons, 4 different codons, or 6 different codons). In some examples, the codon used least frequently (lowest codon usage frequency) for a particular amino acid is incorporated into the sequence of the pathogen (to replace the<br>
appropriate one or more codons in the native sequence), thereby deoptimizing the pathogen sequence and reducing the replicative fitness of the pathogen. In other examples, a codon used with a lower frequency than at least one other codon (but not necessarily the codon with the lowest frequency) for a particular amino acid is incorporated into the sequence of the pathogen (to replace the appropriate one or more codons in the native sequence), for example to alter the G+C content of the sequence or alter the number of CG or TA dinucleotides in the sequence, thereby deoptimizing the pathogen sequence and reducing the replicative fitness of the pathogen. Identification of infrequently used codons can be made by analyzing one or more codon usage tables for the pathogen. The codon usage table used can include codon usage data from the complete genome of the pathogen (or 2 or more genomes, for example from different strains of the pathogen), codon usage data from one or more genes (such as 1 gene, at least 2 genes, at least 3 genes, at least 5 genes, or even at least 10 genes), for example one or more genes involved in the antigenicity of the pathogen. Codon usage tables are publicly available for a wide variety of pathogens (for example see Nakamura et al, Nucleic Acids Res. 28:292, 2000; Sharp et al, Nucleic Acids Res. 16:8207-11, 1988; Chou and Zhang, AIDS Res. Hum. Retroviruses. Dec;8(12):1967-76, 1992; West and Iglewski et al., Nucleic Acids Res. 16:9323-35, 1988, Rothberg and Wimmer, Nucleic Acids Res. 9:6221-9, 1981; Jenkins et al., J. Mol. Evol. 52:383-90, 2001; and Watterson, Mol. Biol. Evol. 9:666-77, 1992; all herein incorporated by reference).<br>
[0229] For example, if the pathogen uses the ACU, ACC, ACA, and ACG codons to encode for Thr at 45, 24, 20 and 11 % frequency respectively, the ACG codon can be chosen to replace at least one ACU, ACC, or ACA codon sequence of the native pathogen sequence, thereby generating a deoptimized sequence. This selection would also increase the number of CG dinucleotides in the deoptimized sequence. However, if it was desired to decrease the G+C content of the deoptimized sequence, the ACA codon (for example instead of ACG) can be chosen to replace the ACU codon. In examples where the amino acid is encoded by only two different codons, one of the two codons can be selected and used in the deoptimized sequence if the codon usage is highly biased, such as a difference of at least 10%, at least 20%, or at least 30%. For example, if the pathogen uses the codons UAU and UAC to encode for Tyr at 90% and 10% frequency respectively, the UAC codon is used to replace at least one UAU codon of the native pathogen sequence, thereby generating a deoptimized sequence. In contrast, if the pathogen uses the codons UAU and UAC to encode for Tyr at 49% and 51 % frequency respectively, Tyr codons would not likely be chosen as the codons to deoptimize.<br>
[0230] In some examples, there may be two or more codons used at low frequencies that are similar in value, such as codon usages that are within 0.01-2% of each other (for example within 0.1-2%, 0.5-2% or 1-2% of each other). In some examples, the codon with the lowest codon usage frequency is not chosen to replace a codon more frequently used. In some examples, the codon chosen is one that alters the G+C content of the deoptimized sequence. In other examples, the codon chosen is one that<br>
alters the frequency of a specific dinucleotide pair (such as CG or TA) found at low frequencies in that genome (such as no more than 3-4%). One example is the CG dinucleotide, which is strongly suppressed in mammalian genomes and in the genomes of many RNA viruses (Karlin et al, J. Virol. 68:2889-2897, 1994). Such dinucleotide pairs can fall across codon boundaries, or be contained within the codon.<br>
Reducing replicative fitness<br>
[0231] The replicative fitness of a pathogen is the overall replicative capacity of the pathogen to produce mature infectious progeny. By introducing one or more deoptimized codons into a coding region of a pathogen's gene(s), the replicative fitness of the pathogen decreases. In some examples, replicative fitness is decreased by at least 10%, at least 20%, at least 30%, at least 50%, at least 75%, at least 90%, at least 95%, or even at least 98%, as compared to an amount of replicative fitness by the a pathogen of the same species and strain in the absence of deoptimized codons. The disclosed methods can be used for making vaccines because the replicative fitness of the pathogen can be modulated by introducing different numbers of nucleotide changes. This flexibility can allow one to alter systematically the replicative fitness of a candidate vaccine strain in order to allow sufficient replication to induce an immune response, but not enough replication to cause pathogenicity.<br>
[0232] Methods that can be used to measure the replicative fitness of a pathogen are known in the art and disclosed herein. For example, to measure the replicative fitness of a virus, plaque size can be measured, infectious center assays can be used, viral liter by TCID50 (tissue-culture infectious doses 50%) or plaque assays can be used, replication in single-step growth curves can be determined, temperature-sensitivity or cold-sensitivity of plaques determined, determination of whether the virus has an unusual host range, or competition assays with a related virus can be determined. To determine the replicative fitness of a bacterium or fungus, exemplary replicative fitness assays include assays for colony-forming activity, temperature-sensitivity, cold-sensitivity, slow growth under certain conditions, increased or rapid bacterial or fungal death, reduced ability of the bacteria or fungi to survive various stress conditions (such as nutrient deprivation), altered host range, enzymatic assays indicating reduced activity of a key enzyme, or assays for reduced pathogenicity due to decreased expre'ssion of an important protein (such as LPS). To measure the replicative fitness of a protozoan, exemplary replicative fitness assays include competitive growth assays with unmodified homologues, temperature-sensitivity, cold-sensitivity, slow growth under certain conditions, increased or rapid senescence, reduced ability to survive various stress conditions, altered host range, enzymatic assays indicating reduced activity of a key enzyme, or assays for reduced pathogenicity due to decreased expression of an important protein (such as surface antigens).<br>
[0233] This disclosure provides several specific examples of pathogens containing deoptimized codons in various genes, including housekeeping genes and genes encoding proteins that are determinants of immunity. However, one skilled in the art will understand how to use the disclosed<br>
methods to deoptimize one or more codons in any pathogen of interest using publicly available codon usage tables and publicly available pathogen sequences In particular examples, a pathogen includes one or more deoptimized codons, for example at least 2, at least 3, at least 4, at least 5, at least 6, at least 7, at least 8, at least 9, at least 10, at least 20, at least 50, at least 100, at least 200, at least 300, at least 400, at least 500, at least 1000, or even at least 2000 deoptimized codons.<br>
[0234] In some examples, a pathogen includes deoptimization of at least 5% of the codons in a gene that encode a particular amino acid, such as deoptimization of at least 5% of the codons that encode Ala (or another amino acid such as Leu, Thr, etc.), for example at least 10% of the codons that encode Ala (or another amino acid), at least 20% of the codons that encode Ala (or another amino acid), at least 50% of the codons that encode Ala (or another amino acid), or at least 90% of the codons that encode Ala (or another amino acid) in a gene. In particular examples, a pathogen includes deoptimization of at least 5% of the codons in one or more coding sequences, such as deoptimization of at least 10%, at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even at least 90% of the codons in one or more coding sequences.<br>
[0235] In one example, viral pathogen sequences are deoptimized in one or more nucleic acid sequences that encode proteins encoding surface antigens which are determinants of immunity, such as a capsid sequences, or spike glycoproteins.<br>
[0236] In particular examples, deoptimizing the codon composition results in an altered G+C content of a coding sequence. For example, deoptimizing one or more codons can increase or decrease the G+C content by at least 10%, such as increase the G+C content of a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%, or decrease the G+C content of a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%. Whether the G+C content is increased or decreased will depend on the sequence of the pathogen of interest.<br>
[0237] However, the G+C content can be deliberately altered in combination with deoptimizing one or more codons in a pathogen sequence. For example, some of the nucleotide substitutions can be made to deoptimize codons, and other nucleotide substitutions can be made to alter the G+C content of the sequence. Altering the G+C content of the sequence may also result in a deoptimized codon, but is not required in all instances.<br>
[0238] In one example, the pathogen is a rubella virus, whose RNA genome has a high G+C content. Therefore, deoptimization of rubella can be achieved by decreasing the G+C content of one or more coding sequences of rubella, for example decreasing the G+C content by at least 10%, such as at least 20%, or even by at least 50%. In another example, the pathogen is a poliovirus or other eukaryotic virus, and deoptimization can be achieved by increasing the G+C content of one or more<br>
coding sequences, for example increasing the G+C content by at least 10%, such as at least 20%, or even by at least 50%. Such changes in G+C content can be achieved as a result of deoptimizing one or more codons, or in addition to deoptimizing one or more codons.<br>
[0239] In some examples, deoptimizing the codon composition results in an altered frequency (number) of CG dinucleotides, TA dinucleotides, or both, in a coding sequence. For example, deoptimization of one or more codons may increase or decrease the frequency of CG or TA dinucleotides in the sequence by at least 10%, for example increase the number of CG or TA dinucleotides in a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 100%, at least 200%, or even by at least 300%, or decrease in the number of CG or TA dinucleotides in a coding sequence by at least 10%, such as at least 20%, at least 30%, at least 40%, at least 50%, at least 60%, at least 70%, at least 80%, or even by at least 90%. Whether the number of CG or TA dinucleotides is increased or decreased will depend on the sequence of the pathogen of interest.<br>
[0240] However, the number of CG or TA dinucleotides can be deliberately altered in combination with deoptimizing one or more codons in a pathogen sequence. For example, some of the nucleotide substitutions can be made to deoptimize codons, and other nucleotide substitutions can be made to alter the number of CG or TA dinucleotides in the coding sequence. Altering the number of CG or TA dinucleotides in the sequence may also result in a deoptimized codon, but is not required in all instances.<br>
[0241] In one example, the pathogen is a poliovirus or eukaryotic virus, and deoptimization can be achieved by increasing the number of CG or TA dinucleotides in one or more coding sequences, for example increasing the number of CG or TA dinucleotides by at least 10%, such as at least 30%, or even by at least 300%. In another example, the pathogen is a bacterium, and deoptimization can be achieved by decreasing the number of CG or TA dinucleotides in one or more coding sequences, for example decreasing the number of CG or TA dinucleotides by at least 10%, such as at least 30%, or even by at least 50%.<br>
[0242] In a particular example, the pathogen is a bacterium. Several methods can be used to deoptimize one or more codons in bacterial coding sequences. For example, one or more codons can be deoptimized such that a single rare codon (such as AGG) is used to force exclusive AGG usage in the mRNA encoding the arginyl tRNA synthetase, potentially limiting the pools of charged arginyl-tRNAs in the cell, and therefore synergistically further limiting the production of arginyl tRNA synthetase. In another example, one or more codons are deoptimized (for example by exclusively using AGG to encode for Arg residues) in one or more of the most highly expressed essential genes (such as translation factors). In yet another example, the distribution of codon-deoptimized genes<br>
along the genome is chosen to reduce the likelihood that all deoptimized genes could be exchanged out by any single natural recombination event.<br>
Exemplary pathogens<br>
[0243] Any pathogen can be attenuated by deoptimizing one or more codons in one or more coding sequences. Exemplary pathogens include, but are not limited to, viruses, bacteria, fungi, and protozoa. For example, viruses include positive-strand RNA viruses and negative-strand RNA viruses. Exemplary positive-strand RNA viruses include, but are not limited to: Picornaviruses (such as Aphthoviridae [for example foot-and-mouth-disease virus (FMDV)]), Cardioviridae; Enteroviridae (such as Coxsackie viruses, Echoviruses, Enteroviruses, and Polioviruses); Rhinoviridae (Rhinoviruses)); Hepataviridae (Hepatitis A viruses); Togaviruses (examples of which include rubella; alphaviruses (such as Western equine encephalitis virus, Eastern equine encephalitis virus, and Venezuelan equine encephalitis virus)); Flaviviruses (examples of which include Dengue virus, West Nile virus, and Japanese encephalitis virus); and Coronaviruses (examples of which include SARS coronaviruses, such as the Urbani strain). Exemplary negative-strand RNA viruses include, but are not limited to: Orthomyxyoviruses (such as the influenza virus), Rhabdoviruses (such as Rabies virus), and Paramyxoviruses (examples of which include measles virus, respiratory syncytial virus, and parainfluenza viruses).<br>
[0244] Polioviruses are small (28 nm diameter), non-enveloped viruses whose single-stranded genome is enclosed in a capsid of 60 identical subunits arranged in icosahedral symmetry. Their positive-stranded genomes (-7500 nt) can serve directly as a messenger RNA, which is translated as a large (-250 kD) polyprotein from a single ORF. The polyprotein is post-translationally processed in a proteolytic cascade catalyzed by virus-encoded proteases, producing at least 10 distinct final cleavage products. Polioviruses grow rapidly in a wide variety of cultured human and simian cells, yielding 103 to 104 infectious particles per infected cell in - 8 hours. As with other RNA viruses, the poliovirus replicase lacks proofreading activity and consequently has a very high rate of base mis incorporation (-KT4 base substitution per base pair per replication; see Domingo et al. 2002. Error frequencies of picomavirus RNA polymerases: evolutionary implications for virus populations, p. 285-298. In B. L. Semler and E. Wimmer (ed.), Molecular Biology of Picornaviruses. ASM Press, Washington, D.C.; Drake and Holland, Proc. Natl. Acad. Sci. USA 96:13910-13, 1999). Polioviruses exist as three stable serotypes, and for each serotype strains with reduced replicative fitness (the "attenuated" Sabin oral poliovirus vaccine [OPV] strains) have been used throughout the world as live virus vaccines; see Sutler et al., 2003. Poliovirus vaccine -- live, p. 651-705. In S. A. Plotkin and W. A. Orenstein (ed.), Vaccines, Fourth ed. W.B. Saunders Company, Philadelphia).<br>
[0245] Viruses also include DNA viruses. DNA viruses include, but are not limited to: Herpesviruses (such as Varicella-zoster virus, for example the Oka strain; cytomegalovirus; and Herpes simplex virus (HSV) types 1 and 2), Adenoviruses (such as Adenovirus type 1 and<br>
Adenovirus type 41), Poxviruses (such as Vaccinia virus), and Parvoviruses (such as Parvovirus B19).<br>
[0246] Another group of viruses includes Retroviruses. Examples of retro viruses include, but are not limited to: human immunodeficiency virus type 1 (HIV-1), such as subtype C, HIV-2; equine infectious anemia virus; feline immunodeficiency virus (FIV); feline leukemia viruses (FeLV); simian immunodeficiency virus (SIV); and avian sarcoma virus.<br>
[0247] Another type of pathogen are bacteria. Bacteria can be classified as gram-negative or gram-positive. Exemplary gram-negative bacteria include, but are not limited to: Escherichia coll (K-12 and O157:H7), Shigella dysenteriae, and Vibrio cholerae.. Exemplary gram-positive bacteria include, but are not limited to: Bacillus anthracis, Staphylococcus aureus, pneumococcus, gonococcus, and streptococcal meningitis.<br>
[0248] Protozoa, nemotodes, and fungi are also types of pathogens. Exemplary protozoa include, but are not limited to, Plasmodium, Leishmania, Acanthamoeba, Giardia, Entamoeba, Cryptosporidium, Isospora, Balantidium, Trichomonas, Trypanosoma, Naegleria, and Toxoplasma. Exemplary fungi include, but are not limited to, Coccidiodes immitis and Blastomyces dermatitidis. There is a great need for effective vaccines against protozoan pathogens. No effective vaccines for fungal pathogens have yet been identified.<br>
Exemplary genes which can be deoptimized<br>
[0249] The gene(s) (for example its corresponding coding sequence) chosen for codon deoptimization can vary depending on the pathogen of interest. In one example, one of the coding sequences deoptimized is a single copy gene that is important for survival of the pathogen, such as a "housekeeping" gene. In some examples, one of the coding sequences deoptimized is a determinant of immunity, such as a viral capsid coding sequence.<br>
[0250] In one example, the virus is a positive strand virus, such as a picornavirus, for example a poliovirus, (for example the Sabin type 2 OPV strain or the MEF1 reference strain used in the inactivated poliovirus vaccine [IPV]) or foot-and-mouth-disease virus (FMDV) (such as serotype O), having one or more codons deoptimized in the capsid region of the virus. In one example, one or more of the Arg codons (such as all of the Arg codons in a reading frame) are replaced with a rare Arg codon, such as CGG. Such CGG-deoptimized picornaviruses can be used to produce inactivated poliovirus vaccine (IPV) in Vero cells expressing elevated levels of the corresponding rare tRNA. Such CGG-deoptimized IPV seed strains are less likely to infect workers in IPV production facilities, enhancing poliovirus containment after global polio eradication.<br>
 [0251] In one example, the positive strand virus is a togavirus, such as a rubella virus or alphavirus. In a particular example, the complete genome of such a virus is de-optimized. However, particular coding sequences can be de-optimized, such as envelope (E) protein El, E2 or core protein.<br>
[0252] In a specific example, the positive strand virus is a flavivirus, such as a dengue virus, West Nile virus, or Japanese encephalitis virus, and one or more codons in the coding sequence of a surface glycoprotein gene deoptimized (such as 8 different amino acid codons).<br>
[0253] In a specific example, the positive strand virus is a coronavirus, such as the SARS coronaviruses (for example the Urbani strain). Such viruses can have one or more codons deoptimized in the coding sequence of a spike glycoprotein region (such as at least 5 different amino acid codons deoptimized).<br>
[0254] In one example, the pathogen is an RNA virus, such as a negative-strand RNA virus. In a specific example, the virus is an orthomyxyovirus, such as an influenza virus (such as strain H3N2), having one or more codons deoptimized in a hemagglutinin (HA) or neuraminidase (NA) coding sequence. In one example, the virus is a paramyxovirus, such as a measles virus having one or more codons deoptimized in a fusion (F) or hemagglutinin (H) coding sequence, or a respiratory syncytial virus having one or more codons deoptimized in a fusion (F) or glycoprotein (G) coding sequence.<br>
[0255] In one example, the pathogen is a retrovirus, such as HIV-1 or HIV-2, and one or more codons are deoptimized in an envelope (env) or group antigen (gag) coding sequence.<br>
[0256] In one example, the pathogen is a DNA virus, such as herpes viruses. In a specific example, the virus is a varicella zoster virus (such as the Oka strain), and one or more codons are deoptimized in a glycoprotein E or H coding sequence. In another specific example, the virus is a cytomegalovirus, and one or more codons are deoptimized in a glycoprotein B, H, or N coding sequence. In yet another specific example, the virus is herpes simplex virus types 1 or 2, and one or more codons are deoptimized in genes encoding surface glycoprotein B, glycoprotein D, integument protein, or the large subunit of ribonucleotide reductase.<br>
[0257] In one example, the pathogen is a bacterium, such as gram-positive or gram-negative bacteria. In one gram-negative example, the bacterium is Escherichia coli (such as strains K-12 or 0157:H7), and one or more Arg codons (such as all Arg codons) are replaced with the rare codon AGG in the ArgS gene (arginyl synthetase gene) and the highly expressed TufA gene (translation factor U). In another example, the bacterium is a Shigella dysenteriae, and one or more Arg codons (such as all Arg codons) are replaced with AGG in the RdsB gene. In one gram-positive example, the bacterium is Staphylococcus aureus, and one or more Arg codons (such as all Arg codons) are replaced with AGG in the RplB and FusA genes.<br>
Pathogens with Deoptimized Codon Sequences as Immunogenic Compositions<br>
[0258] The disclosed attenuated pathogens having a nucleic acid coding sequence with one or more deoptimized codons can be used in an immunogenic composition. In some examples, the deoptimized pathogens are further attenuated, for example by passage at suboptimal growth temperatures. Such immunogenic compositions can be used to produce an immune response against the pathogen in a subject, for example to treat a subject infected with the pathogen, decrease or inhibit infection by the pathogen, or reduce the incidence of the development of clinical disease.<br>
[0259] In forming a composition for generating an immune response in a subject, or for vaccinating a subject, a purified, diluted, or concentrated pathogen can be utilized.<br>
Compositions including a deoptimized pathogen<br>
[0260] In one example, purified or concentrated (or diluted) deoptimized pathogens that have one or more codons deoptimized are provided. In some examples, the immunogenic compositions are composed of non-toxic components, suitable for infants, children of all ages, and adults. Also disclosed are methods for the preparation of a vaccine, which include admixing a deoptimized pathogen of the disclosure and a pharmaceutjcally acceptable carrier. Although particular examples of deoptimized sequences are provided herein, one skilled in the art will appreciate that further modifications to the nucleic acid or protein sequence of the pathogen can be made without substantially altering the reduced replicative fitness due to the deoptimized codons. Examples of such further modifications include one or more deletions, substitutions, insertions, or combinations thereof, in the nucleic acid or protein sequence. In one example, such further modifications to a deoptimized pathogenic sequence do not increase the replicative fitness of the deoptimized pathogenic sequence by more than 5%, such as no more than 10%, as compared to an amount of replicative fitness by the deoptimized pathogen.<br>
[0261] In one example, deoptimized pathogen sequences that include additional amino acid deletions, amino acid replacements, isostereomer (a modified amino acid that bears close structural and spatial similarity to the original amino acid) substitutions, isostereomer additions, and amino acid additions can be utilized, so long as the modified sequences do not increase the replicative fitness of the deoptimized pathogenic sequence by more than 5%, and retain the ability to stimulate an immune response against the pathogen. In another example, deoptimized pathogen sequences that include nucleic acid deletions, nucleic acid replacements, and nucleic acid additions can be utilized, so long as the modified sequences do not increase the replicative fitness of the deoptimized pathogenic sequence by more than 5%, and retains the ability to stimulate an immune response against the pathogen.<br>
[0262] In one example, the deoptimized pathogenic nucleic acid sequences are recombinant.<br><br>
[0263] The deoptimized pathogens can be replicated by methods known in the art. For example, pathogens can be transferred into a suitable host cell, thereby allowing the pathogen to replicate. The cell can be prokaryotic or eukaryotic.<br>
[0264] The disclosed deoptimized pathogens can be used as immunogenic compositions, such as a vaccine. In one example, an immunogenic composition includes an immunogenically effective amount (or therapeutic amount) of an attenuated deoptimized pathogen of the disclosure, such as a viral, bacterial, fungal, or protozoan deoptimized pathogen. Immunogenically effective refers to the amount of attenuated deoptimized pathogen (live or inactive) administered at vaccination sufficient to induce in the host an effective immune response against virulent forms of the pathogen. An effective amount can being readily determined by one skilled in the art, for example using routine trials establishing dose response curves. In one example, the deoptimized pathogen can range from about 1% to about 95% (w/w) of the composition, such as at least 10%, at least 50%, at least 75%, or at least 90% of the composition.<br>
[0265] Pharmaceutical compositions that include a deoptimized pathogen can also include other agents, such as one or more pharmaceutically acceptable carriers or other therapeutic ingredients (for example, antibiotics). In one example, a composition including an immunogenically effective amount of attenuated deoptimized pathogen also includes a pharmaceutically acceptable carrier. Particular examples of pharmaceutically acceptable carriers include, but are not limited to, water, culture fluid in which the pathogen was cultured, physiological saline, proteins such as albumin or casein, and protein containing agents such as serum. Other agents that can be included in the disclosed pharmaceutical compositions, such as vaccines, include, but are not limited to, pH control agents (such as arginine, sodium hydroxide, glycine, hydrochloric acid, citric acid, and the like), local anesthetics (for example, benzyl alcohol), isotonizing agents (for example, sodium chloride, mannitol, sorbitol), adsorption inhibitors (for example, Tween 80), solubility enhancing agents (for example, cyclodextrins and derivatives thereof), stabilizers (for example, serum albumin, magnesium chloride, and carbohydrates such as sorbitol, mannitol, starch, sucrose, glucose, and dextran), emulsifiers, preservatives, (such as chlorobutanol and benzalkonium chloride), wetting agents, and reducing agents (for example, glutathione).<br>
[0266] When the immunogenic composition is a liquid, the tonicity of the formulation, as measured with reference to the tonicity of 0.9% (w/v) physiological saline solution taken as unity, can be adjusted to a value at which no substantial, irreversible tissue damage will be induced at the site of administration. Generally, the tonicity of the solution is adjusted to a value of about 0.3 to about 3.0, such as about 0.5 to about 2.0, or about 0.8 to about 1.7.<br>
DNA immunogenic compositions<br>
[0267] In one example, an immunogenic composition includes a deoptimized nucleic acid coding sequence instead of (or in addition to) the entire deoptimized pathogen. In particular examples, the sequence includes a sequence having at least 90%, at least 95%, or 100% sequence identity to any of SEQ ID NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42,45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69. In some examples, an immunogenic composition includes a full-length deoptimized genome, for example a deoptimized poliovirus genome. However, one skilled in the art will appreciate that fragments of the deoptimized full-length genome can also be used (and in some examples ligated together). The DNA including the deoptimized coding sequence can be part of a vector, such as a plasmid, which is administered to the subject. Such DNA immunogenic compositions can be used to stimulate an immune response using the methods disclosed herein.<br>
[0268] In one example, a deoptimized nucleic acid coding sequence from a pathogen is present in a colloidal dispersion system. Colloidal dispersion systems include macromolecule complexes, nanocapsules, microspheres, beads, and lipid-based systems including oil-in-water emulsions, micelles, mixed micelles, and liposomes. Large uni-lamellar vesicles (LUV), which range in size from 0.2-4.0 urn can encapsulate a substantial percentage of an aqueous buffer containing large macromolecules. RNA, DNA and intact virions can be encapsulated within the aqueous interior and be delivered to cells in a biologically active form (Fraley et al., Trends Biochem. Sci. 6:77, 1981).<br>
[0269] The composition of a liposome is usually a combination of phospholipids, particularly high-phase-transition-temperature phospholipids, usually in combination with steroids, such as cholesterol. Examples of lipids useful in liposome production include phosphatidyl compounds, such as phosphatidylglycerol, phosphatidylcholine, phosphatidylserine, phosphatidylethanolamine, sphingolipids, cerebrosides, and gangliosides. Particularly useful are diacylphosphatidyl-glycerols, where the lipid moiety contains from 14-18 carbon atoms, such as 16-18 carbon atoms, and is saturated. Illustrative phospholipids include egg phosphatidylcholine, dipalmitoylphosphatidylcholine and distearoylphosphatidylcholine.<br>
Inducing an Immune Response<br>
[0270] Methods are disclosed for stimulating an immune response in a subject using the disclosed deoptimized pathogens (such as a pathogen that includes a sequence having at least 90%, at least 95% or 100% sequence identity to any of SEQ ID NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 48, 51, 54, 55, 56, 57, 58, 67, 68, or 69) and immunogenic compositions. The method includes administering to a subject an immunologically effective amount of a deoptimized pathogen having a nucleic acid coding sequence with one or more deoptimized codons, which reduce the replicative fitness of the pathogen (for example by at least 20%, at least 50%, or even at least 99%). Such administration can be broadly effective for treatment and prevention of disease caused by a pathogen,<br>
and one or more associated symptoms thereof. In one example, the immunogenic compositions and methods are designed to confer specific immunity against infection with a pathogen, and to induce antibodies specific to the pathogen. The deoptimized pathogens can be delivered to a subject in a manner consistent with conventional methodologies associated with management of the disorder for which treatment or prevention is sought.<br>
[0271] In selected examples, one or more symptoms or associated effects of exposure to or infection with a pathogen is prevented or treated by administration to a subject at risk of being infected by the pathogen, or presenting with one or more symptoms associated with infection by the pathogen, of an effective amount of a deoptimized pathogen of the disclosure. Therapeutic compositions and methods of the disclosure for prevention or treatment of toxic or lethal effects of pathogen infection are applicable to a wide spectrum of infectious agents.<br>
Administration of deoptimized pathogens<br>
[0272] For administration to animals or humans, the immunogenic compositions of the present disclosure, including vaccines, can be given by any method determined appropriate by a clinician. In addition, the immunogenic compositions disclosed herein can be administered locally or systemically. Types of administration include, but are not limited to, intramuscular, subcutaneous, oral, intravenous, intra-atrial, intra-articular, intraperitoneal, parenteral, intraocular, and by a variety of mucosal administration modes, including by oral, rectal, intranasal, intrapulmonary, or transdermal delivery, or by topical delivery to other surfaces.<br>
[0273] The disclosed methods include administering a therapeutically effective amount of an attenuated pathogen having one or more deoptimized codon sequences (a deoptimized pathogen) to generate an immune response against the pathogen. Specific, non-limiting examples of an immune response are a B cell or a T cell response. Upon administration of the deoptimized pathogen, the immune system of the subject responds to the immunogenic composition (such as a vaccine) by producing antibodies, both secretory and serum, specific for one or more pathogen epitopes. Such a response signifies that an immunologically effective dose of the deoptimized pathogen was delivered. An immunologically effective dosage can be achieved by single or multiple administrations. In some examples, as a result of the vaccination, the subject becomes at least partially or completely immune to infection by the pathogen, resistant to developing moderate or severe pathogen infection, or protected from disease associated with infection by the pathogen. For example, an effective dose can be measured by detection of a protective antibody liter in the subject.<br>
[0274] Typical subjects that can be treated with the compositions and methods of the present disclosure include humans, as well as veterinary subjects such as dogs, cats, horses, chickens, cows, fish, sheep, and pigs. To identify subjects for treatment according to the methods of the disclosure, accepted screening methods can be employed to determine risk factors associated with a targeted or<br>
suspected disease of condition (for example, polio) as discussed herein, or to determine the status of an existing disease or condition in a subject. These screening methods include, for example, conventional work-ups to determine environmental, familial, occupational, and other such risk factors that may be associated with the targeted or suspected disease or condition, as well as diagnostic methods, such as various ELISA and other immunoassay methods, which are available and well known in the art to detect or characterize disease-associated markers, such as antibodies present in the serum of a subject indicating that they were previously infected with a particular pathogen. The vaccines can also be administered as part of a routine health maintenance program in at risk individuals, such as the administration of meningococcal vaccines in children and pneumococcal or influenza vaccines in the elderly. These and other routine methods allow a clinician to select subjects in need of therapy using the methods and pharmaceutical compositions of the disclosure. In accordance with these methods and principles, a deoptimized pathogen can be administered using the methods disclosed herein as an independent prophylaxis or treatment program, or as a follow-up, adjunct or coordinate treatment regimen to other treatments, such as surgery, vaccination, or immunotherapy.<br>
[0275] The compositions including deoptimized pathogens can be used for therapeutic purposes, such as prophylactically. When provided prophylactically, deoptimized pathogens are provided in advance of any symptom associated with the pathogen against which the prophylaxis is provided. The prophylactic administration of deoptimized pathogens serves to prevent or ameliorate any subsequent infection. When provided therapeutically, deoptimized pathogens are provided at (or shortly after) the onset of a symptom of disease or infection. The disclosed deoptimized pathogens can thus be provided prior to the anticipated exposure to a particular pathogen, so as to attenuate the anticipated severity, duration or extent of an infection or associated disease symptoms, after exposure or suspected exposure to the pathogen, or after the actual initiation of an infection.<br>
[0276] The deoptimized pathogens disclosed herein can be administered to the subject in a single bolus delivery, via continuous delivery (for example, continuous transdermal, mucosal, or intravenous delivery) over an extended time period, or in a repeated administration protocol (for example, by an hourly, daily, weekly, or monthly repeated administration protocol). In one example, administration of a daily dose can be carried out both by single administration in the form of an individual dose unit or else several smaller dose units and also by multiple administrations of subdivided doses at specific intervals.<br>
[0277] The therapeutically effective dosage of a deoptimized pathogen can be provided as repeated doses within a prolonged prophylaxis or treatment regimen that will yield clinically significant results to alleviate one or more symptoms or detectable conditions associated with a targeted disease or condition as set forth herein. Determination of effective dosages are typically based on animal model studies followed up by human clinical trials and is guided by administration protocols that<br>
significantly reduce the occurrence or severity of targeted disease symptoms or conditions in the subject. Various considerations are described, e.g., in Oilman et al, eds., Goodman and Oilman: The Pharmacological Bases of Therapeutics, 8th ed., Pergamon Press, 1990; and Remington's Pharmaceutical Sciences, 17th ed., Mack Publishing Co., Easton, PA, 1990, each of which is herein incorporated by reference. Suitable models in this regard include, for example, murine, rat, porcine, feline, non-human primate, and other accepted animal model subjects known in the art.<br>
[0278] Immunologically effective dosages can also be determined using in vitro models (for example, immunologic and histopathologic assays). Using such models, only ordinary calculations and adjustments are used to determine an appropriate concentration and dose to administer a therapeutically effective amount of the deoptimized pathogen (for example, amounts that are effective to elicit a desired immune response or alleviate one or more symptoms of a targeted disease). In some examples, amounts administered are those amounts adequate to achieve tissue concentrations at the site of action which have been found to achieve the desired effect in vitro. In alternative examples, an effective amount or effective dose of the deoptimized pathogens can decrease or enhance one or more selected biological activities correlated with a disease or condition.<br>
[0279] For example, deoptimized pathogens of the present application can be tested using in vitro and in vivo models to confirm adequate attenuation, genetic stability, and immunogenicity for vaccine use. In a particular example, an in vitro assay is used to determine the attenuation and genetic stability of a deoptimized pathogen, for example using the plaque assays and virus yield, single-step growth assays described herein. In another example, deoptimized pathogens are further tested in animal models of infection, for example using the methods described herein. For example, a deoptimized pathogen can be administered to an animal model, and an amount of immunogenic response to the deoptimized pathogen determined, for example by analyzing antibody, T-cell or B-cell production. In some examples, the animal is further exposed to the pathogen, and resistance to infection determined.<br>
[0280] The actual dosage of the deoptimized pathogen can vary according to factors such as the disease indication and particular status of the subject (for example, the subject's age, weight, fitness, extent of symptoms, susceptibility factors, and the like), time and route of administration, the type of pathogen against which vaccination is sought, other drugs or treatments being administered concurrently, as well as the specific pharmacology of the deoptimized pathogens for eliciting the desired activity or biological response in the subject. Dosage regimens can be adjusted to provide an optimum prophylactic or therapeutic response. A therapeutically effective amount is also one in which any toxic or detrimental side effects of a deoptimized pathogen are outweighed in clinical terms by therapeutically beneficial effects.<br>
[0281] In one example, an immunogenic composition includes any dose of deoptimized bacteria sufficient to evoke an immune response, such as a range of between 103 and 1010 bacteria per dose, for example at least 103 bacteria, at least 104 bacteria, at least 105 bacteria, at least 108 bacteria, or at least 109 bacteria per dose. In one example, an immunogenic composition includes any dose of deoptimized virions sufficient to evoke an immune response, such as a range of between 103 to 1010 plaque forming units (PFU) or more of virus per subject, such as 104 to 105 PFU virus per subject, for example at least 103 PFU virus per subject, at least 104 PFU virus per subject, at least 105 PFU virus per subject, or at least 109 PFU virus per subject. In another example, an immunogenic composition includes any dose of deoptimized protozoa sufficient to evoke an immune response, such as at least 102 infectious units per subject, for example at least 103 infectious units per subject, or a range of between 102 to 106 infectious units per subject. In any event, the immunogenic compositions ideally provide a quantity of deoptimized pathogen sufficient to effectively protect the subject against serious or life-threatening pathogen infection.<br>
[0282] For each particular subject, specific dosage regimens can be evaluated and adjusted over time according to the individual need and professional judgment of the person administering or supervising the administration of the deoptimized pathogen. For example, in neonates and infants, multiple administrations can be required to elicit sufficient levels of immunity. In some examples, administration of the disclosed immunogenic compositions begins within the first month of life and continues at intervals throughout childhood, such as at two months, six months, one year and two years, as necessary to maintain sufficient levels of protection against pathogen infection. Similarly, adults who are particularly susceptible to repeated or serious infection by pathogens, such as health care workers, day care workers, elderly individuals, and individuals with compromised cardiopulmonary function, may require multiple immunizations to establish or maintain protective immune responses. Levels of induced immunity can be monitored by measuring amounts of neutralizing secretory and serum antibodies, and dosages adjusted or vaccinations repeated as necessary to maintain desired levels of protection.<br>
[0283] The antibody response of a subject administered the compositions of the disclosure can be determined by using effective dosages/immunization protocols. In some examples, it is sufficient to assess the antibody liter in serum or plasma obtained from the subject. Decisions as to whether to administer booster inoculations or to change the amount of the immunogenic composition administered to the individual can be at least partially based on the antibody titer level. The antibody liter level can be based on, for example, an immunobinding assay which measures the concentration of antibodies in the serum which bind to a specific antigen present in the pathogen. The ability to neutralize in vitro and in vivo biological effects of the pathogen of interest can also be assessed to determine the effectiveness of the treatment.<br>
[0284] Dosage can be varied by the attending clinician to maintain a desired concentration at a target site. Higher or lower concentrations can be selected based on the mode of delivery. Dosage can also be adjusted based on the release rate of the administered formulation. To achieve the same serum concentration level, for example, slow-release particles with a release rate of 5 nanomolar (under standard conditions) would be administered at about twice the dosage of particles with a release rate of 10 nanomolar.<br>
Kits<br>
[0285] The instant disclosure also includes kits, packages and multi-container units containing the herein described deoptimized pathogens, alone or in the presence of a pharmaceutically acceptable carrier, and in some examples, an adjuvant. Such kits can be used in the treatment of pathogenic diseases in subjects. In one example, these kits include a container or formulation that contains one or more of the deoptimized pathogens described herein. In one example, this component is formulated in a pharmaceutical preparation for delivery to a subject. The deoptimized pathogens can be contained in a bulk dispensing container or unit or multi-unit dosage form.<br>
[0286] Optional dispensing means can be provided, for example a pulmonary or intranasal spray applicator, or a needle. Packaging materials optionally include a label or instruction indicating for what treatment purposes, or in what manner the pharmaceutical agent packaged therewith can be used.<br>
[0287] The subject matter of the present disclosure is further illustrated by the following non-limiting Examples.<br>
Example 1<br>
Codon usage in Poliovirus [0288] This example describes methods used to determine codon usage in poliovirus.<br>
[0289] Mononucleotide and dinucleotides frequencies, and codon usage were analyzed in the original reports of poliovirus genomic sequences (Kitamura et al. 1981. Nature 291:547-53; Racaniello and Baltimore. 1981. Proc. Natl. Acad. Sci. USA 78:4887-91; Rothberg and Wimmer. 1981. Nucleic Acids Res. 9:6221-9; Toyoda et al. 1984. J. Mol. Biol. 174:561-85). The mono-, di-, and trinucleotide frequency patterns are similar for the three Sabin strains (Toyoda et al. 1984. J. Mol. Biol. 174:561-85) and appear to be conserved across poliovirus genotypes (Hughes et al. 1986. J. Gen. Virol. 67:2093-102; Kew et al. 2002. Science 296:356-9; La Monica et al. 1986. J. Virol. 57:515-25; Liu et al. 2003. J. Virol. 77:10994-1005; Martin et al. 2000. Virology 278:42-9; Yang et al. 2003. J. Virol. 77:8366-77) and human enterovirus species C serotypes (Brown et al. 2003. J. Virol. 77:8973-84).<br>
[0290] As with other enteroviruses, the component bases in the Sabin 2 ORF are present in approximately equal proportions (24.0% U, 22.9% C, 29.9% A, and 23.1% G; see Rezapkin et a!., Virology 258:152-60, 1999; Toyoda et al., J. Mol. Biol 174:561-85, 1984), thus permitting a low bias in codon usage (Osawa et al, Microbiol. Rev. 56:229-264, 1992). Indeed, all codons are used in poliovirus ORFs (Toyoda et al., J. Mol. Biol. 174:561-85, 1984), and the overall degree of codon usage bias is low (Jenkins and Holmes. Virus Res. 92:1-7, 2003).<br>
[0291] One measure of codon usage bias is the number of effective codons (Nc), which can vary from 20 (only one codon used for each amino acid) to 61 (all codons used randomly) (Wright, Gene 87:23-9, 1990). The Nc values for Sabin 2 are 56.0 for the capsid region and 54.6 for the complete ORF. As with the genomes of vertebrates and most RNA viruses, the dinucleotide CG is suppressed in the Sabin 2 genome (Toyoda et al., J. Mol. Biol. 174:561-85, 1984), and the observed pattern of codon usage reflects this CG suppression (Table 1).<br>
Table 1: Codon usage in mutagenized capsid interval and complete ORF in unmodified and deoptimized Sabin 2 genomes.<br>
(Table Removed)<br><br>
 Unpreferred codons used as replacement codons are shown in boldface font.<br>
b ABCD represents virus construct S2R9, which differs from the reference Sabin 2 strain sequence at<br>
three synonymous third-position sites: A26]6 —* G (VP1 region), A3303 —»• T (VP1 region), and T5640 —»<br>
A (3Cpro region).<br>
c ABCd represents virus construct S2R19, which has replacement codons across an interval spanning<br>
76%oftheVPl region.<br>
d abed represents virus construct S2R23, which has replacement codons across an interval spanning<br>
97% of the capsid region.<br>
Example 2<br>
Poliovirus Containing a Deoptimized Capsid Region<br>
[0292] This example describes methods used to generate a poliovirus containing deoptimized codons in the capsid region. Briefly, the original capsid region codons of the Sabin type 2 oral polio vaccine strain were replaced with synonymous codons less frequently used in poliovirus genomes. An unpreferred synonymous codon was used nearly exclusively to code for each of nine amino acids. Codon changes were introduced into four contiguous intervals spanning 97% of the capsid region.<br>
[0293] The strategy for codon replacement was as follows. Despite the low overall bias in codon usage in Sabin 2, some synonymous codons are used at much lower frequencies than others (Table 1). To determine codon usage in Sabin 2, the preferred codons for each of nine amino acids were replaced with a synonymous unpreferred codon (Table 1). The codon replacements shown in Table 1 were introduced only within the capsid sequences, because those sequences uniquely identify a poliovirus serotype, as both noncapsid and 5'-UTR region sequences are exchanged out by recombination with other species C enteroviruses during poliovirus circulation.<br>
[0294] Because codon usage bias was very low for most two-fold degenerate codons (except codons for His and Tyr), only six-fold, four-fold, and three-fold degenerate codons were replaced. Synonymous codons for nine amino acids were replaced by a single unpreferred codon: CUU for Leu, AGC for Ser, CGG for Arg, CCG for Pro, GUC for Val, ACG for Thr, GCG for Ala, GGU for Gly, and AUC for He (Table 1). Whenever possible, codons with G or C at degenerate positions (the<br>
nucleotides that differ within the codons that encode for a particular amino acid) were chosen to increase the G + C content of the modified viral genomes.<br>
[0295] For example, as shown in Table 1, the amino acid Leu is encoded by 6 different codons in Sabin 2. However, the codon CUU is used the least frequently of the six. Therefore, it was selected to replace the other five codons. Similarly, the amino acid Pro is encoded by four different codons in Sabin 2. However, the codon CCG is used the least frequently of the four. Therefore, it was selected to replace the other three codons. A similar analysis was performed for the least frequently used codon for Thr and, Ala. For the amino acid Ser, although the codon UCG was less frequently used than AGC in Sabin 2, AGC was chosen to deoptimize the sequence because it was the least preferred Ser codon among a larger collection of VP1 sequences of wild polioviruses. Similarly, GGU was the least preferred Gly codon among a larger collection of VP1 sequences of wild polioviruses. Codons CGG and AUC were selected for Arg and He, respectively, because they were not preferred and their usage would increase the G+C content of the poliovirus genome.<br>
[0296] In addition, some codons did not display a significant amount of bias, and were therefore not selected. For example, the amino acid Asp is encoded in the Sabin 2 capsid region by 19 and 23 GAU and GAC codons, respectively. Similarly, the amino acid Glu is encoded in the Sabin 2 capsid region by 16 and 19 GAA and GAG codons, respectively. Since these values are similar, it is not likely that substitution of one for the other would reduce replicative fitness of the pathogen. Ideally, in the case where there are at least two codons that encode for an amino acid in the pathogen, there is at least a 20% difference between the selected codon and one or more of the other codons that encode the amino acid, such as an at least 30% difference, or an at least 50% difference.<br>
[0297] Replacement .codons were introduced into a full-length infectious cDNA clone derived from Sabin 2 (construct S2R9) within an interval (nt 748 to 3302) spanning all but the last 27 codons of the capsid region (FIGS. 1A-D). The capsid interval was divided into four mutagenesis cassettes: A (nt 657 to 1317; 661 bp),B(nt 1318 to 2102; 785 bp), C(nt 2103 to 2615; 513 bp), and D(nt 2616 to 3302; 687 bp) (FIG. 1 A). Mutagenesis cassette A, bounded by restriction sites BstZlll and Avrll, includes the last 91 nucleotides of the 5'-UTR, but no 5'-UTR sequences were modified in cassette A. Within each cassette, synonymous codons for the nine amino acids were comprehensively replaced except at 15 positions (replacement at 11 of these positions would have eliminated desirable restriction sites or generated undesirable restriction sites). Unmodified cassettes are identified by uppercase italic letters; the corresponding cassettes with replacement codons are identified by lowercase italic letters. Thus, as shown in FIG. 2, the reference Sabin 2 derivative (derived from cDNA construct S2R9) is identified as ABCD (SEQ ID NO: 3), and the fully modified virus (derived from cDNA construct S2R23) is identified as abed (SEQ ID NO: 5).<br>
[0298] The methods described below were used to generate the deoptimized polioviruses.<br>
[0299] Virus and cells. The Sabin Original + 2 (Sabin and Boulger. J. Biol. Stand. 1:115-8, 1973) master seed of the Sabin type 2 oral poliovaccine strain (P712 ch 2ab) was provided by R. Mauler of Behringwerke AG (Marburg, Germany). Virus was grown at 35°C in suspension cultures as previously described (Rueckert and Pallansch. Meth. Enzymol. 78:315-25, 1981) of S3 HeLa cells (human cervical carcinoma cells; ATCC CCL- 2.2) or in monolayer cultures of HeLa (ATCC CCL-2), and RD (human rhabdomyosarcoma cells; ATCC CCL-136) cells. Some initial plaque assays were performed in HEp-2C cells (Chen, Cytogenet. Cell Genet. 48:19-24, 1988).<br>
[0300] Preparation of Infectious Sabin 2 clones. Poliovirus RNA was extracted from 250 ^1 of cell culture lysate (from -75,000 infected cells) by using TRIZOL LS reagent (Life Technologies, Rockville, MD) and further purified on CENTR1-SEP columns (Princeton Separations, Adelphia, NJ). Full-length cDNA was reversed transcribed (42°C for 2 hours) from ~1 ug of viral RNA in a 20 ul reaction containing 500 uM dNTP (Roche Applied Science, Indianapolis, IN), 200 U Superscript II Reverse Transcriptase (Life Technologies), 40 U RNase-inhibitor (Roche), 10 mM dithiothreitol, and 500 ng primer S2-7439A-B [CCTAAGC(T)30CCCCGAATTAAAGAAAAATT TACCCCTACA; SEQ ID NO: 1] in Superscript II buffer.<br>
[0301] After reverse transcription, 2 U RNase H (Roche) was added and incubated at 37°C for 40 min. Long PCR amplification of viral cDNA was performed using TaqPlus Precision (Stratagene, La Jolla, CA) and AmpliWax PCR Gem 100 beads (Applied Biosystems, Foster City, CA) for "hot start" PCR in thin-walled tubes. The bottom mix (50 ul) contained 200 uM each dNTP (Roche) and 250 ng each of primers S2-7439A-B and S2-1S-C<br>
(GTAGTCGACTAATACGACTCACTATAGGTTAAAACAGCTCTGGGGTTG; SEQ ID NO: 2) in TaqPlus Precision buffer. A wax bead was added to each tube, and samples were heated at 75°C for 4 minutes and cooled to room temperature. The top mix (50 jjl) contained 2 ul of the cDNA and 10 U TaqPlus Precision in TaqPlus Precision buffer. The samples were incubated in a thermal cycler at 94°C for 1 minute and then amplified by 30 PCR cycles (94°C for 30 seconds, 60°C for 30 seconds, and 72°C for 8 minutes), followed by a final 94°C for 1 minute and final extension of 72°C for 20 minutes.<br>
[0302] PCR products were purified using QIAquick PCR purification kit (Qiagen, Valencia, CA) and sequentially digested for 2 hours at 37°C with Sal I and Hind III prior to gel purification. PCR products were ligated to pUC19 plasmids following standard methods (Sambrook and Russell. 2001. Molecular Cloning: A Laboratory Manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York) and ligated plasmids were transformed into XL-10 Gold supercompetent E. coli cells (Stratagene). Colonies were screened for recombinant plasmids on X-gal indicator plates (Sambrook and Russell. 2001. Molecular Cloning: A Laboratory Manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York) and 6 white colonies were transferred to 1.5 ml Luria-Bertani broth containing 50 ng/ml ampicillin (LB/amp) (Roche). Plasmids were purified using<br><br>
QIAprep Spin Miniprep columns and sequences of the inserts were determined by cycle sequencing using an automated DNA sequencer (Applied Biosystems, Foster City, CA) (Liu et al, J. Virol. 74:11153-61, 2000). The full-length viral insert was sequenced in both orientations using overlapping sense and antisense primers spaced -500 nt apart. Selected clones were grown in 50 ml LB/amp, and recombinant plasmids were purified using the QIAfilter Plasmid Maxi kit.<br>
[0303] Virus Preparation. Plasmids were linearized with Hind III and purified using QIAquick columns prior to RNA transcription from 1 ug of plasmid DNA using the Megascript T7 In Vitro Transcription kit (Ambion, Austin, TX). RNA yields were estimated using DNA Dipsticks (Invitrogen, Carlsbad, CA) and RNA chain length was analyzed by electrophoresis on 1% formaldehyde gels prior to transfection. RD cells were transfected with transcripts of viral RNA by using Tfx-20 (Promega, Madison, WI). Briefly, semi-confluent RD cells in 12-well cell culture plates were inoculated with 500 ul MEM (MEM incomplete) (Life Technologies) containing 0.1 ug viral RNA transcript and 0.45 ul Tfx-20 Reagent. Plates were incubated for 1 hour at 35°C prior to addition of 1.5 ml MEM complete [MEM incomplete supplemented with 100 U penicillin and 100 ug streptomycin, 2 mM L-glutamine, 0.075% NaHCO3, 10 uM HEPES (pH 7.5)] (Life Technologies) containing 3% fetal calf serum (PCS; HyClone, Logan, UT). Negative controls were performed using RNA transcribed from pBluescriptll SK+ (Stratagene) containing a viral insert truncated at base 7200 by digestion with BamHI and transcribed in a reverse orientation from a T3 promoter.<br>
[0304] Complete CPE was observed after incubation at 35°C for 18-20 hours at which time 400 ul from the transfected wells were transferred to a confluent RD cell monolayer in 75 cm2 flasks containing MEM complete. Complete CPE was observed in the second passage after 24 hours at 35°C, and virus was liberated from the infected cells by three freeze-thaw cycles and clarification by centrifugation for 15 minutes at 15,000 x g. Control wells were passaged once and monitored for 72 hours post-transfection. The sequences of all virus stocks were verified by RT-PCR amplification of two large overlapping fragments and subsequent sequence analysis of the PCR product.<br>
[0305] Site-Directed Mutagenesis. Single-base substitutions were introduced using the QuikChange Site-Directed Mutagenesis Kit (Stratagene). Briefly, two complementary primers containing the desired mutation were designed for PCR amplification of the plasmid containing the Sabin 2 insert. Amplification was performed using Pfu Turbo DNA polymerase on 5 ng of template DNA for 15 cycles at 95°C for 30 s, 50°C for 1 minute, and 68°C for 23 minutes. PCR products were digested for 1 hour at 37°C with 10 U of Dpn I prior to transformation in XL-1 Blue Supercompetent cells. Colonies were grown and screened by sequencing as described above.<br>
[0306] Assembly PCR. Multiple base substitutions were introduced by assembly PCR using previously described methods (Stemmer et al., Gene 164:49-53, 1995). Briefly, primers were designed to span the region of interest with complementary 40-mers overlapping by 10 nt on each<br>
end. A first round of assembly (30 PCR cycles of 94°C for 45 seconds, 52°C for 45 seconds, and 72°C for 45 seconds) was performed with a 20 ul reaction mixture containing Taq Plus Precision buffer, 10 U Taq Plus Precision, 5 pmoles of each primer, and 200 uM dNTP. A second round of assembly (25 PCR cycles of 94°C for 45 seconds, 50°C for 45 seconds, and 72°C for 2 minutes) was performed using the outermost sense and antisense primers in a 100 ul reaction mixture in Taq Plus Precision buffer containing 2 ul of product from the first assembly round, 10 U Taq Plus Precision, 200 ng of each primer, and 400 uM dNTP. PCR products were column purified prior to digestion, ligation, and transformation into XL-10 gold supercompetent E. coli cells. Clones were grown and screened by sequencing of insert as described.<br>
[0307] Construction of recombinant clones. The sequence of the full-length Sabin 2 infectious clone, S2R9, differed from the published sequence of a reference Sabin 2 strain (Rezapkin et al., Virology 258:152-60, 1999) at three synonymous third-codon positions: G2ei6 (in VP1 region; A replaced to introduce an EagI restriction site) T3303 (in VP1 region; A replaced to introduce a Xhol site), A564o (in 3Cpro region). The S2R9 construct was used as the reference Sabin 2 strain. Recombinant clones having different combinations of blocks of replacement codons were constructed using standard methods (Kohara et al., J. Virol. 53:786-92, 1985).<br>
[0308] As shown in Tables 1 and 2, the modifications introduced dramatically altered the mono-, di-, and trinucleotide (codon) frequencies in the capsid region. In the fully modified construct, abed, nearly half (427/879; 48.6%) of the capsid region codons were replaced, and a total of 544 substitutions (90 first codon position, 44 second position, and 410 third position) were introduced into the 2555 mutagenized capsid region nucleotides. This strategy for codon deoptimization increased the number of CG dinucleotides in the poliovirus templates. CG was the least abundant dinucleotide (181 occurrences) in the unmodified ABCD construct and the most abundant dinucleotide (386 occurrences) in the highly modified abed construct. Compared with ABCD, the Nc values in the capsid region of abed fell from 56.2 to 29.8, the number of CG dinucleotides rose from 97 to 302, and the %G + C increased from 48.4% to 56.4% (Table 2). These changes were nearly uniformly distributed over the mutagenized capsid region (Table 2).<br>
Table 2. Effective number of codons used (Nc), number of CG dinucleotides, and G+C content in mutagenized capsid region sequences.<br>
(Table Removed)<br>
(Table Removed)<br>
 Constructs correspond to the following infectious cDNA plasmids, clones, and virus derivatives:<br>
ABCD, S2R9; aBCD, S2R28; AbCD, not constructed; ABcD, S2R20; ABCd, S2R19; abed, S2R23;<br>
Nc, number of CG dinucleotides, and %G+C of all other constructs can be calculated from table.<br>
 NC: effective number of codons used (1); one replacement codon spanned the Eagl restriction<br>
cleavage site and was counted as part of cassette D.<br>
c One CG dinucleotide spanned the Eagl restriction cleavage site and was counted as part of the<br>
cassette D.<br>
d orig/mod: original construct/modified codon-replacement construct.<br>
" Complete capsid region: nt 748 to 3384.<br>
f The S2R9 (ABCD) sequence differs from the reference Sabin 2 sequence at three synonymous<br>
third-position sites (see Table 1).<br>
s Does not include the 3'-terminal 91 bases of the 5'-UTR at the 5'-end of cassette A (nt 657 to 747)<br>
that were not modified.<br>
Example 3<br>
Growth Properties of Codon-Deoptimized Constructs<br>
[0309] This example describes methods used to determine the growth properties of the deoptimized Sabin 2 polioviruses generated in Example 2. Similar methods can be used to determine the replicative fitness of any deoptimized virus.<br>
[0310] Briefly, RNA transcripts of constructs with different combinations of codon-replacement cassettes (FIG. 2) were transfected into RD cells as described above. Virus obtained from the primary transfection was passaged again in RD cells to increase virus liters as described above. The growth properties of the virus constructs in HeLa cells were measured by plaque assays (FIGS. 3A-E) and single-step growth experiments (FIGS. 4A-B).<br>
[0311] Plaque assays were performed by a modification of previously described methods (Yang et al. J. Virol. 77:8366-77, 2003). Briefly, confluent HeLa cell monolayers in 100 cm2 cell culture dishes were washed, inoculated with virus in MEM incomplete, and incubated at room temperature for 30 minutes prior to the addition of 0.45% SeaKem LE Agarose (BioWhittaker Molecular, Rockland, ME) in MEM complete containing 2% PCS. Plates were incubated for 52-60 hours at 35°C, fixed with 0.4% formaldehyde and stained with 3% crystal violet. Plaque size was quantified by scanning plates on a FOTO/Analyst Archiver system (Fotodyne, Hartland, WI) and subsequent image analysis using Scion Image for Windows (Scion Corp., Frederick, MD).<br>
[0312] As shown in FIGS. 3A and 3C, an approximately linear inverse relationship was observed between mean plaque area in HeLa cells and the number of nucleotide changes in the capsid region. Similar inverse linear relationships were observed when the abscissa was rescaled to the number of replacement codons (FIG. 3D) or to the number of CG dinucleotides (FIG. 3E). There was no strong<br>
polarity to the effects of codon replacement within the capsid region, as introduction of replacement codons into any combination of the four cassettes reduced plaque areas approximately in proportion to the total number of replacement codons. However, replacement of codons into VP1 (cassette D) appeared to have slightly stronger effects than replacement elsewhere. Codon replacement in three or four cassettes generally conferred a minute-plaque phenotype (mean plaque area 
[0313] Measurement of plaque areas and total plaque number became difficult as plaque size decreased. The diameters of poliovirus plaques are typically heterogeneous, and this heterogeneity was observed with the plaques of all constructs. Precise measurement was most difficult with the smallest of the minute plaques, as was discriminating very minute plaques from other small defects in the cell monolayers. Extended incubation of plaque cultures to 72 hours increased plaque diameters but did not markedly increase the plaque counts. Growth properties of all constructs were also determined by plaque assays and limit dilution infectivity assays in HEp-2(C) cells at 35°C. For some of the constructs (abed, abCD, AbcD, ABcd, and aBCd), the limit dilution infectivity liter was 2-10 fold higher than the plaque liters. For the olher conslrucls, limil dilution infectivily and plaque liters were similar. The plaque liters mighl have been undereslimaled for some constructs because of the difficulty in seeing the liniesl plaques.<br>
[0314] A plaque is the result of several cycles of replication, which effectively amplifies any difference in replication rale. To determine Ihe relalionship belween plaque size, virus growth rales, and virus yield, single-step growth experiments (input MOI: 5 PFU/cell) were performed as follows. S3 HeLa suspension cells (1 x 107) were infected at a multiplicity of infection (MOI) of 5 PFU/cell with stirring for 30 minutes al 25°C. After 30 minutes, cells were sedimented by low-speed centrifugalion and resuspended in 2.5 ml warm complete media SMEM containing glutamine, 5% PCS, penicillin-streptomycin, and 25mM HEPES (pH 7.5). Incubation conlinued al 35°C in a water balh with orbital shaking at 300 rpm. Samples were withdrawn at 2-hour intervals from 0 to 14 hours poslinfeclion, and titered by plaque assay in Hep-2(C) cells (35°C, 72 hours).<br>
[0315] As shown in FIGS. 3B, 4A and 4B, mean virus yields from the single-step growlh assays generally decreased as Ihe number of replacement codons increased. Virus yields were highest (-200 PFU/cell) for the ABCD prototype and constructs ABcD and aBCD. Yields were 4- lo 8-fold lower wilh constructs ABCd, abCD, and ABcd, 12- to 24-fold lower wilh constructs abcD and aBcd, 30- to 45-fold lower with constructs Abed and abCd, and -65-fold lower with construct abed. Moreover, production of infectious virus appeared to be slower in Ihe codon-replacemenl conslrucls lhan in Ihe unmodified ABCD construct. Although maximum plaque yields were obtained at 10-12 hours for all<br>
constructs, proportion of the final yields detected at 4 hours were lower for the codon-deoptimized constructs (FIGS. 4A and 4B).<br>
[0316] In summary, although the Sabin 2 OPV strain has a relatively low codon usage bias, its replicative fitness in cell culture was reduced by replacement of preferred codons in the capsid region with synonymous unpreferred codons. The reduction in fitness, as measured by plaque area, was approximately proportional to the length of the interval containing replacement codons. Plaque areas were reduced by -90% and virus burst yields by -98% in the abed construct, in which the replacement interval spanned nearly the entire capsid region. The fitness declines in the replacement codon constructs are not attributable to amino acid substitutions because all constructs encoded the same reference Sabin 2 polyprotein sequence. Virus yields varied over a -65-fold range in response to the extent of codon deoptimization.<br>
[0317] Multiple synonymous capsid codon replacements increase the ability to detect discernible reductions in poliovirus fitness. For example, replacement of 3 to 14 Arg codons in VP1 (0.3% to 1.6% of capsid codons) with CGG (among the least preferred codons in the poliovirus genome) did not result in any apparent reduction in plaque areas. The ability to detect small declines in poliovirus fitness might be improved by replacing the plaque assay, which invariably gives heterogeneous plaques, with a biochemical assay. However, one advantage of the plaque assay and other virus infectivity assays is their high sensitivities to very low levels of biological activity.<br>
Example 4<br>
In vivo Protein Synthesis by Deoptimized Pathogen Sequences<br>
[0318] This example describes methods used to determine if there was a change in the amount of protein synthesis due to the presence of deoptimized codons. Similar methods can be used to measure protein synthesis by any deoptimized pathogen sequence.<br>
[0319] Monolayer HeLa cells were plated at 8 xlO5 per well in a 6-well dish. On the following day, the cells were washed in MEM without serum. Cells were infected at a multiplicity of infection (moi) of 25 in complete MEM with 2% serum. Cells were incubated in a C02 incubator at 35°C or 37°C for 4 hours. Viruses tested were Sabin 2 and MEF1; constructs tested were S2R9 (Sabin 2 prototype genome; ABCD; SEQ ID NO: 3), S2R19 (deoptimized VP3-VP1 genome; ABCd), S2R23 (deoptimized Pl/capsid region; abed; SEQ ID NO: 5), MEF1R2 (MEF1 prototype genome; ABC), MEF1R5 (deoptimized VP3-VP1 genome; ABc), and MEF1R9 (deoptimized Pl/capsid region; abc).<br>
[0320] Media was removed, and 1.9 ml. of labeling media (200uCi 35S-met in a mixture of 1 volume regular complete MEM containing 2% serum and 7 volumes of met-deficient complete MEM containing 2% serum) were added. Cultures were incubated in CO2 incubator at 35 or 37°C for 3 hours. Radioactive media was removed, and cells were rinsed twice with PBS. Cells were lysed in 1 ml lysis buffer (lOmM NaCl, lOmM Tris-Cl pH 7.5, 1.5mM MgCl2, containing 1% NP-40) at 35°C<br>
for one minute. The lysed cell-media mixture was transferred to a screw-cap Eppendorf tube on ice. 0.2 ml. lysis buffer was added to the plate, and this lysate was added to the original lysate. The lysate was spun at 2000xg 2 minutes 4°C, and the supernatant was removed to a new tube. SDS was added to the sup to make a final concentration of 1% SDS, and samples were frozen. Samples (4 ul) were run on SDS-10% PAGE gels (Laemmli). Gels were fixed, washed, dried on a vacuum gel drier, and exposed to Kodak BioMax film for 1 - 3 days at room temperature.<br>
[0321] Although it was thought that replacement of preferred codons with unpreferred codons would lower replicative fitness primarily by reducing the rate of translation (at the level of polypeptide chain elongation) of viral proteins and potentially disrupting their proteolytic processing in infected cells, unexpectedly, it was observed that the electrophoretic profiles of the labeled virus-specific proteins were similar for all S2R viruses, both in the relative intensities of the labeled viral protein bands and in the total amounts of labeled viral proteins produced in the infected cells (FIG. 5 A). The four S2R viruses were similar in the efficiency of shutoff of host cell protein synthesis and in the synthesis and processing of viral proteins in infected HeLa cells. Similar results were obtained with MEF1 viruses (see Example 10, FIG. 5C).<br>
Example 5 In Vitro Translation<br>
[0322] This example describes methods used to determine the ability of deoptimized poliovirus RNA transcripts to serve as templates for in vitro translation in rabbit reticulocyte lysates. Similar methods can be used to measure in vitro protein synthesis by any deoptimized pathogen sequence.<br>
[0323] For preparation of truncated polio proteins that include the entire capsid protein and terminate in the 2C noncapsid portion of the poliovirus genome, plasmid DNAs were digested with SnaBl. Full-length and partial viral RNAs were transcribed as described herein. In vitro-transcribed RNAs were subjected to phenol/chloroform extraction and two successive ammonium acetate isopropanol precipitations, including 70% ethanol washes. The RNA pellets were air-dried for 5 minutes and then resuspended in a small volume of RNAse-free water. The resuspended RNA was quantitated by measuring OD26o absorbance in a spectrophotometer.<br>
[0324] In vitro translation was performed using a nuclease-treated rabbit reticulocyte lysate (Promega, Madison, WI) supplemented with an uninfected HeLa cell extract (Brown and Ehrenfeld Virology 97: 396-405, 1979), according to the manufacturer's instructions. The HeLa extract has been found to improve the fidelity of initiation of translation. Briefly, 35 ul micrococcal nuclease-treated, supplemented rabbit reticulocyte lysate was mixed with 7 ul HeLa cell extract, lul ImM amino acid mix (minus methionine), various amounts of RNA (0.2 - lug), 30 uCi 35S-met at 15mCi/ml, and 1 ul RNasin (40 u/ul) in a final volume of 50 ul. The reactions were incubated at 30°C for 3 hours. Samples (4 ul) were run on SDS-10% PAGE gels (Laemmli). Gels were fixed,<br>
washed, dried on a vacuum gel drier, and exposed to Kodak BioMax film for 1 - 3 days at room temperature.<br>
[0325] The efficiency of the poliovirus RNA transcripts to serve as templates for in vitro translation in rabbit reticulocytes was similar for all of the viruses tested (S2R9, S2R19, S2R23, MEF1R1, MEF1R2, MEF1R5, and MEF1R9). No decline in translational efficiency was observed with increasing numbers of replacement codons in the in vitro translation systems tested (FIG. 6). The observation that codon replacement had little detectable effect in vivo upon viral protein synthesis and processing was mirrored by the results of in vitro translation experiments in rabbit reticulocyte lysates. Full-length in vitro transcripts from cDNA constructs ABCD, ABCd, and abcd(S2R9, S2R19, S2R23), ABC, ABc and abc (MEF1R2, MEF1R5, and MEF1R9) programmed the in vitro synthesis and processing of virus-specific proteins with nearly equal efficiency (FIGS. 5B and 5D). The in vivo and in vitro protein synthesis results indicate that the reduced replicative fitness of the codon-replacement viruses is not primarily attributable to impairment of translation and processing of viral proteins.<br>
[0326] The protein synthesis results are somewhat surprising, since translational effects have been previously observed when unpreferred codons were introduced into the coding region of some genes of bacteria (Barak et al, J. Mol. Biol. 256:676-84, 1996), yeast (Hoekema et al, Mol. Cell Biol. 7:2914-24, 1987), yeast, and one animal virus (Zhou et al., J. Virol. 73:4972-82, 1999). It is possible that translational effects were not observed because some of the codons that are rarely used in poliovirus genomes are used frequently in highly expressed mammalian genes, such that the levels of the tRNAs for these codons may be high and therefore difficult to deplete. Another possible explanation is that poliovirus RNA is not equivalent to a highly expressed gene, as it is not translated as efficiently as mRNAs of the most highly expressed mammalian genes. Polypeptide chain elongation rates are -220 amino acids per min for poliovirus in HeLa cells at 37°C (Rekosh, J. Virol. 9:479-87, 1972) compared with -600 amino acids per min for the a-chain of hemoglobin in rabbit reticulocytes (Hunt et al., J. Mol. Biol. 43:123-33, 1969). The translation results do not exclude the possibility that there are local conditions in certain cells in an infected person or animal that result in decreased translational efficiency.<br>
Example 6 Specific Infectivities of Virions of Codon-Replacement Viruses<br>
[0327] This example describes methods used to measure the infectivity of the deoptimized Sabin viruses described in Example 2.. Similar methods can be used to measure the infectivity of any pathogen with one or more deoptimized sequences.<br>
[0328] Virus was propagated in RD cells, liberated by freeze-thaw, and concentrated by precipitation with polyethylene glycol 6000 (Nottay et al., Virology 108:405-23, 1981). Virions were<br>
purified by pelleting, isopycnic centrifugation in CsCl, and repelleting essentially as described by Nottay et al., (Virology 108:405-23, 1981). The number of virus particles in each preparation recovered from the CsCl band with a buoyant density of 1.34 g/ml was calculated from the absorbance at 260 nm using the relationship of 9.4 x 1012 virions per OD26o unit (Rueckert, R. R. 1976. On the structure and morphogenesis of picornaviruses, p. 131-213. In H. Fraenkel-Conrat and R. R. Wagner (ed.), Comprehensive Virology, vol. 6. Plenum Press, New York.).<br>
[0329] The poliovirions produced by HeLa cells infected with viruses ABCD (S2R9), ABCd (S2R19), and abcd(S2R23) were analyzed. Purified infectious virions of all three viruses had similar electrophoretic profiles and the high VP2/VPO ratios typical of mature capsids. However, the specific infectivities of the purified virions decreased with increased numbers of replacement codons. For example, the particle/PFU ratios increased from 293 (ABCD) to 1221 (ABCd) to 5392 (abed). The magnitude of the decline in specific infectivity was dependent upon the infectivity assay used, and was steeper with the plaque assay than with the limit dilution assay. This difference arose because the CCID50/PFU ratio in HeLa cells increased with the number of replacement codons, from 1.1 (ABCD) to 5.4 (abed).<br>
Example 7<br>
Measurement of Viral RNA in Infected Cells<br>
[0330] Alterations in the primary sequence of the viral genome could affect the levels of RNA in infected HeLa cells by modifying the rates of RNA synthesis or by changing the stabilities of the intracellular viral RNA molecules. This example describes methods used to measure the amount of viral RNA produced in cells infected with the deoptimized viruses described in Example 2. However, one skilled in the art will recognize that similar methods can be used to measure the amount of viral RNA produced in cells infected with any pathogen with one or more deoptimized sequences.<br>
[0331] Production of viral RNA in infected HeLa cells during the single-step growth assays described above was measured by quantitative RT-PCR using a Stratagene MX4000 PCR system programmed to incubate at 48°C for 30 min, 95°C for 10 min, followed by 60 PCR cycles (95°C for 15 sec, 60°C for 1 min). Sequences within the 3' half of the 3Dpo1 region of Sabin 2 were amplified using primers S2/7284A (ATTGGCACACTCCTGATTTTAGC; SEQ ID NO: 59) and S2/7195S (CAAAGGATCCCAGAAACACACA; SEQ ID NO: 60), and the amplicon yield measured by the fluorescence at 517 nm of the TaqMan probe S2/7246AB (TTCTTCTTCGCCGTTGTGCCAGG; SEQ ID NO: 61) with FAM atttached to the 5' end and BHQ-1 (Biosearch Technologies, Novato, Calif.) attached to the 3' end. Stoichiometric calculations used a value of 2.4 * io6 for the molecular weight of Sabin 2 RNA (Kitamura, et al., Nature 291:547-53, 1981; Toyoda et al., J. Mol. Biol. 174:561-85, 1984).<br>
[0332] Total levels of viral RNA present in infected HeLa cells were measured at 2 h intervals from 0 to 12 hours in the single-step growth experiments described above and shown in FIGS. 4A and 4B. Viral RNA was measured by quantitative PCR using primers targeting 3DP°' sequences shared among all viruses. After 12 hours, total viral RNA yields were highest (915 ng/ml; equivalent to -57,000 RNA molecules/cell) forABCD, lower (569 ng/ml; -35,000 RNA molecules/cell) forABCd, and lowest (330 ng/ml; -20,000 RNA molecules/cell) for abed (FIG. 6A). Plaque yields, by contrast, had followed a steeper downward trend, from -130 PFU/ cell (ABCD), to -30 PFU/ cell (ABCd), to -2 PFU/ cell (abed) (FIGS. 3B and 4A-B). Combining these values, the following yields are obtained: -440 RNA molecules/PFU (ABCD), -1200 RNA molecules/PFU (ABCd), and -10,000 RNA molecules/PFU (abed). Although the RNA molecules/PFU ratios were similar to the particle/PFU ratios determined above for each virus, the number of RNA molecules produced in infected cells is typically about twice the number of virus particles, because only about 50% of the viral RNA product is encapsidated (Hewlett et al, Biochem. 16:2763-7, 1977). Nonetheless, the two sets of values clearly followed similar trends, as RNA yields and specific infectivities declined with increased number of replacement codons.<br>
[0333] Because the particle/PFU (or RNA molecule/PFU) ratios were higher for the codon-replacement viruses than for the unmodified ABCD prototype, substantially more ABCd and abed virion particles were used to initiate the single-step growth infections, even though the input MOIs varied over a narrow (~4-fold) range (FIGS. 4A-B). Consequently, the initial input RNA levels were high for ABCd and very high for abed, such that the extent of amplification of viral RNA at 12 h was -4000-fold for ABCD, -1000-fold forABCd, and only -20-fold for abed (FIG. 6).<br>
[0334] The observation that the eclipse phases in the single-step growth experiments were increasingly prolonged as the number of replacement codons increased indicates that codon-replacement viruses were less efficient at completing an early step (or steps) of the infectious cycle. This view is reinforced by the observation that the particle/PFU and RNA molecule/PFU ratios increased sharply with the number of replacement codons. It thus appears that a larger number of codon-replacement virus particles are needed to initiate a replicative cycle, but once the cycle had started the synthesis and processing of viral proteins is nearly normal. Although total viral RNA yield was reduced by only -3-fold in the most highly modified abed virus, its viral RNA amplification was only ~20-fold, indicating that impairment of viral RNA synthesis can also contribute to reduced replicative fitness.<br>
Examples<br>
RNA secondary structures of Codon Deoptimized Sequences<br>
[0335] This example describes methods used to predict RNA secondary structures of the deoptimized Sabin 2 codon genomes generated in Example 2.<br>
[0336] Prediction of the secondary structure of the RNA templates of virus constructs S2R9, S2R19, and S2R23 was performed using the mfold v. 3.1 program (Zuker, Science 244: 48-52, 1989; Mathews et al, J. Mol. Biol, 288:911-40, 1999; Palmenberg and Sgro, Semin. Virol. 8:231-41, 1997) that implements an energy minimization algorithm that finds a structure lying within a percentage (P) of the calculated minimum energy (MinE). Running parameters were set to default except folding temperature (T), which was set to 35°C. The free energy increment (AAG35°C), dependent on P, is set to 1 kcal/mol or 12 kcal/mol (SubE12) when the calculated AAG35°C values lie below or above these values.<br>
[0337] The genomic RNAs of polioviruses and other enteroviruses appear to have relaxed secondary structures outside of the 5'-UTR, the 3'-UTR, and the ere element within the 2C region (Palmenberg and Sgro, Semin. Virol. 8:231-41, 1997; Witwer et al., Nucleic Acids Res. 29:5079-89, 2001). Accordingly, under physiological conditions, most bases within the ORF can pair with more than one partner, and poliovirus genomes can fold into many different secondary structures having similar thermodynamic stabilities (Palmenberg and Sgro, Semin. Virol. 8:231-41, 1997). However, the incorporation of numerous base substitutions into the codon-replacement constructs and the concomitant increase in G+C content might destabilize folding patterns that had been subject to natural selection and stabilize other pairings absent from the unmodified Sabin 2 genome.<br>
[0338] To determine the effects of codon replacement on RNA folding patterns, the secondary structures of the complete genomes ofABCD, ABCd, and abcdwere calculated using the mfold v. 3.1 algorithm. The calculated global thermodynamic stabilities (expressed as minimum free-energy at 35°C [AG35°C] or MinE) of the RNA secondary structures increased with increasing G+C content (ABCD, AG35°C = -2047 kcal/mol; ABCd, AG35°C = -2078 kcal/mol; abed, AG35°C = -2191 kcal/mol), and the number of predicted stem structures increased from 546 (ABCD), to 557 (ABCd), to 562 (abed). The calculated MinE structures for the three viruses also differed (FIG. 7). However, the in vivo pairings are likely to be much more flexible and dynamic than indicated by the static structures shown in FIG. 7, as many alternative structures having nearly equivalent (+12 kcal/mol) MinE values are predicted (SubE12). A more informative measure of structural rigidity is the p-num value, which gives the number of alternative pairings for each base. Unaltered in all viruses were the stable (low p-num values, colored red) secondary structures in the 5'-UTR, the 3'-UTR, and the ere element, as well as the close apposition of the 5' and 3' termini. However, some folding patterns were modified in the codon-replacement viruses, and the structural perturbations extended beyond the boundaries of the modified cassettes. Alterations in stable pairings were most extensive with abed, where the long Pl/capsid region:P3/noncapsid region pairings (nt 1480 - 1714:nt 5998 - 5864) predicted for Sabin 2 RNA were destabilized and other pairings formed (FIG. 7).<br>
Example 9<br>
Stability of the Mutant Phenotypes<br>
[0339] This example describes methods used to determine the stability of the codon-deoptmized polioviruses during serial passage in HeLa cells.<br>
[0340] Three constructs generated as described in Example 2 were examined: ABCD (unmodified prototype), ABCd (modified VP1 region), and abed (modified Pl/capsid region). Poliovirus constructs S2R9 (ABCD), S2R19 (ABCd), and S2R23 (abed) were serially passaged in HeLa cell monolayers in T75 flasks at 35°C for 36 hours, at an input MOI ranging from 0.1 PFU/cell to 0.4 PFU/cell. Each virus was passaged 25 times (at 35°C for 36 hours), wherein each passage represented at least two rounds of replication. At every fifth passage, virus plaque areas, plaque yields, and the genomic sequences of the bulk virus populations were determined, and the MOI was readjusted to -0.1 PFU/cell.<br>
[0341] All three constructs evolved during serial passage, as measured by increasing plaque size, increasing virus yield, and changing genomic sequences (Table 3; FIGS. 8A-C). Evolution of the ABCD prototype was the least complex. Plaque areas increased ~6-fold from passage 0 to passage 15, and this was accompanied by nucleotide substitutions at 6 sites. By contrast, virus yields increased 2.5-fold over the 25 passages. Two substitutions (Ui439—&gt;C and C269—»U) were fixed by passage 10, three more (U3424—&gt;C, A3586—»G, and A550i—*G) by passage 15, and all 6 substitutions were fixed by passage 20. Mixed bases were found at passage 5 (Ci439&gt;U, C2609&gt;U, and U3 passage 10 (C3424&gt;U, G3586»A, and G550i&gt;A) and passage 15 (A5630&gt;U). No evidence of back mutation or serial substitutions at a site was observed.<br>
Table 3. Nucleotide substitutions in ABCD, ABCd, and abed during passage.<br><br>
(Table Removed)<br>
 Virus constructs: ABCD, S2R9; ABCd, S2R19; abed, S2R23.<br>
 Nucleotides immediately preceding (-1 nt) and immediately following (+4 nt) codon.<br>
 Varying nucleotide is shown in boldface font.<br>
Rjghtward pointing arrows indicate substitutions that steadily accumulated with increased passage; bidirectional arrows indicate bidirectional fluctuations among substitutions.<br>
CG dinucleotides, including those across codons, are underlined.<br>
' Location of amino acid replacements: S, virion surface residue; NAg, neutralizing antigenic site (1,2); ~NAg, adjacent to neutralizing antigenic site; I, internal capsid residue not exposed to virion surface; NC, non-consensus amino acid; V, variable amino acid. g Represents direct reversion of engineered codon change.<br><br>
[0342] All substitutions mapped to the coding region, and 2 of 6 (33%) mapped to the capsid region, which represents 35.4% of the genome. In distinct contrast to the pattern of poliovirus evolution in humans, where the large majority of base substitutions generate synonymous codons, all six of the observed base substitutions (4 at the second codon position and 2 at the first codon position) generated amino acid replacements (Table 3). None of the substitutions involved loss of a CG dinucleotide.<br>
[0343] Evolution of the codon-replacement constructs was more complex and dynamic. In construct ABCd, 4 of the 8 (50%) variable positions mapped to VP1 (12.1% of genome), and 3 of these 4 mapped within the replacement-codon d interval (9.2% of genome) (Table 3). Substitutions at half of the positions involved the apparent loss of CG dinucleotides (6.3% of total genome), although in all instances the loss from the virus population was incomplete. One d interval substitution (G3|2o—*A) eliminating a CG dinucleotide represented a back mutation to the original synonymous codon. A second d interval substitution (G278o—*A) reduced the frequency of a CG dinucleotide by HeLa passage 10, but the CG dinucleotide predominated in the population by HeLa passage 25. Another substitution (C3377—&gt;U), which resulted in the partial loss of a CG dinucleotide, mapped just downstream from the d interval. Two adjacent substitutions, mapping to positions 3808 and 3809 in 2A, resulted in a complex pattern of substitution involving first and second positions of the same codon. The ABCd construct resembled the ABCD prototype in that substitutions in 6 of the 8 generated amino acid replacements. By contrast, the ABCd construct differed markedly from the ABCD prototype because the dynamics of substitution had apparently not stabilized by passage 25, and mixed bases were found at all 8 positions of variability (Table 3). The active sequence evolution was accompanied by progressively increasing plaque areas over a ~6-fold range, while virus yields fluctuated over a narrow (~2-fold) range (FIGS. 8A-C).<br>
[0344] Evolution of the abed construct was the most dynamic, as determined by expanding plaque areas, increasing virus yields, and nucleotide substitutions. Plaque areas increased -15-fold from passage 0 to passage 15, and then stabilized (FIGS. 8A-C). Virus yields increased most sharply (~4-fold) between passages 5 and 10, but remained ~4-fold lower than those of the ABCD and ABCd constructs at passage 25 (FIG. 8B). Among the 13 sites of nucleotide variability, most (11/13; 84.6%) mapped to the capsid region, all within the codon-replacement interval, 8 within VP1, 3 within VP2, and none within VP3 (Table 3). As with the other constructs, most (8/13; 61.5%) of the substitutions encoded amino acid replacements. Substitutions at six sites involved partial, transient, or complete loss of CG dinucleotides.<br>
[0345] As in the ABCd construct, a G312o-&gt;A substitution eliminated a CG dinucleotide and restored the original Sabin 2 base. Interestingly, this same reversion was observed in 8 other independent passages of the abed construct (data not shown). The two variable sites outside of the capsid region<br><br>
(one in 2A, the other in 2C) stabilized with new substitutions by HeLa passage 20, whereas 8 of the 11 variable sites within the capsid region still had mixed bases at passage 25. Apart from the back-mutation at position 3120, all other variable sites differed between the ABCD, ABCd, and abed constructs. No net changes were observed at site A48| (in the 5'-UTR), and U29o9 (in the VP1 region), known to be strongly selected against when Sabin 2 replicates in the human intestine.<br>
[0346] In addition to the elimination of several CG dinucleotides, there was also a net loss (1 lost, 5 partially lost, 1 gained) of UA dinucleotides in the high-passage isolates (Table 3). In the codon-replacement constructs, elimination of UA dinucleotides was incomplete up to passage 25. Most (4 of 6) UA losses involved amino acid replacements. Unlike codons most frequently associated with loss of CG dinucleotides, none of the codons associated with loss of UA dinucleotides were replacement codons. While not as strongly suppressed as CG dinucleotides, UA dinucleotides are underrepresented in poliovirus genomes and human genes.<br>
[0347] Most (8 of 13) of the capsid amino acid replacements mapped within or near surface determinants forming neutralizing antigenic sites. For example, four replacements mapped to NAg-Isite and four to NAg-2 site (Table 3). Although surface determinants are generally the most variable, amino acid replacements also occurred in naturally variable non-surface residues in VP1 (Lys—»Glu) and 2Apro (Ser—»Arg). Most of the synonymous mutations mapped to codons for conserved amino acids. However, several of the amino acid replacements, including 5 of the 6 in the ABCD construct, were substitutions to non-consensus residues (Table 3).<br>
[0348] Sequence evolution in HeLa cells of the unmodified ABCD virus differed in many respects from the codon-replacement ABCd and abed viruses. Nucleotide substitutions in the ABCD progeny were dispersed across the ORF, dimorphic variants emerged in the early passages, all 6 mutations were fixed by passage 20, and a single dominant master sequence emerged. By contrast, populations of the ABCd and abed progeny were complex mixtures of variants at least up to passage 25, and the majority base at the variable sites typically fluctuated from passage to passage. Apparently the incorporation of unpreferred codons into the ABCd and abed genomes led to an expansion of the mutant spectrum and to the emergence of complex and unstable quasispecies populations.<br>
[0349] To identify potential critical codon replacements, substitutions that accumulated in the genomes of codon-replacement viruses upon serial passage in HeLa cells were identified. Only one substitution, G3120—&gt;A, a direct back mutation to the original sequence, was shared between derivatives of the ABCd and abed viruses after serial passage. The 19 other independent substitutions found among the ABCd and abed high-passage derivatives were associated with 12 different codon triplets. Codon replacement in the VP1 region appeared to have proportionately greater effects on replicative fitness than replacements in other capsid intervals, an observation reinforced by the finding that 8 of the 13 sites that varied upon serial passage of abed mapped to the VP1 region.<br>
Replacement of VP1 region codons in the genome of the unrelated wild poliovirus type 2 prototype strain, MEF1, also had a disproportionately high impact on growth.<br>
[0350] The pattern of reversion among high-passage progeny of the codon-replacement virus constructs indicates that increased numbers of CG dinucleotides may contribute to the reductions in fitness. The codon replacements raised the number of CG dinucleotides in the poliovirus complete ORFs from 181 (ABCD) to 386 (abed). Although the biological basis for CG suppression in RNA viruses is poorly understood (Karlin et al., J. Virol. 68:2889-97, 1994), selection against CG dinucleotides during serial passage of ABCdand abed was sufficiently strong at some sites as to drive amino acid substitutions into the normally well conserved poliovirus capsid proteins. In every instance, the CG suppression was incomplete, and was frequently reversed upon further passage. The most stable trends toward CG suppression involved nucleotide positions 3120 and 3150 and were not associated with amino acid changes.<br>
[0351] Although fitness of the ABCdand abed constructs increased during serial passage in HeLa cells, the virus yields of the ABCd and abed derivatives were still below that of the unmodified ABCD construct. In addition, the substitutions accumulating in the ABCdand abed derivatives during cell culture passage were distinct from the Sabin 2 mutations known to accumulate during propagation in cell culture,<br>
[0352] In summary, replicative fitness of both codon-deoptimized and unmodified viruses increased with passage in HeLa cells. After 25 serial passages (-50 replication cycles), most codon modifications were preserved and the relative fitness of the modified viruses remained below that of the unmodified virus. The increased replicative fitness of high-passage modified virus was associated with the elimination of several CG dinucleotides.<br>
[0353] Codon replacement in VP1 appeared to have greater relative effects on replicative fitness than replacements in other capsid intervals, an observation confirmed in similar experiments with the wild poliovirus type 2 prototype strain, MEF1, and reinforced by the finding that 8 of the 13 sites that varied upon serial passage of the abed construct mapped to VP1.<br>
Example 10<br>
Deoptimized Poliovirus MEF1<br>
[0354] This example describes methods used to generate a deoptimized MEF1 virus, and the effects of deoptimizing the sequence.<br>
[0355] Methods used were similar to those for Sabin 2 (see Example 2). FIGS. 9A-E show a capsid coding sequence for the poliovirus type 2, strain MEF1 which is deoptimized. The prototype strain is listed on the top (SEQ ID NO: 6), the nucleotide codon change is indicated below that line (SEQ ID NO: 8), and the single-letter amino acid code is included as the third line (SEQ ID NO: 7).<br>
[0356] Replacement codons were introduced into an infectious cDNA clone derived from MEF1 (MEF1R2) within an interval (nt. 748 to 3297) spanning all but the last 29 codons of the capsid region.<br>
[0357] R5 VIRUS	Cassette Afel-XhoI most of VP1 (SEQIDNO:54)<br>
[0358] R6 VIRUS	Cassette EcoRV-Agel VP4-VP2 (SEQ ID NO: 55)<br>
[0359] R7 VIRUS	Cassette Agel-Afel VPS-partial VP1 (SEQ ID NO: 56)<br>
[0360] R8 VIRUS	Cassette EcoRV-Afel VP4-VP2-VP3-partial VP1 (SEQ ID NO: 57)<br>
[0361] R9 VIRUS	Cassette EcoRV-XhoI Complete capsid (almost) (SEQ ID NO: 58)<br>
[0362] Within each cassette, synonymous codons for the nine amino acids were comprehensively replaced except at 2 positions (replacement at 2 of these positions would have generated undesirable restriction sites). Unmodified cassettes were identified by uppercase italic letters; the corresponding cassettes with modified codons were identified by lowercase italic letters. Thus, the reference MEF1R2 clone was identified as ABC (SEQ ID NO: 53), and the fully modified construct (MEF1R9), was identified as abc (SEQ ID NO: 58).<br>
[0363] The effect of increasing numbers of replacement codons on growth properties was similar to that observed for Sabin 2. An approximately linear inverse relationship was observed between mean plaque area in HeLa cells and the number of nucleotide changes in the capsid region (FIGS. 9F and 9G). Similar inverse linear relationships were observed when the abscissa was rescaled to the number of replacement codons or to the number of CG dinucleotides. There was no strong polarity to the effects of codon replacement within the capsid region, as introduction of replacement codons into any combination of the three cassettes reduced plaque areas approximately in proportion to the total number of replacement codons. However, replacement of codons into VP1 (cassette C) appeared to have slightly stronger effects than replacement elsewhere. Codon replacement across the entire PI/capsid region (construct abc) conferred a minute-plaque phenotype (mean plaque area 
[0364] Mean virus yields from the single-step growth assays of MEF1 constructs generally decreased as the number of replacement codons increased. As observed for the Sabin 2 codon replacement constructs, production of infectious virus appeared to be slower in the MEF1 codon-replacement constructs than in the unmodified ABC construct. Although maximum plaque yields<br>
were obtained at 10-12 hours for all constructs, proportion of the final yields detected at 4 hours were lower for the codon-deoptimized constructs (FIG. 9H). An approximately linear inverse relationship was observed between the log 10 virus yield at 8-12 hours postinfection in the single-step growth curve in HeLa cells and the number of nucleotide changes in the capsid region (FIG. 91). Plaque size also exhibited a linear inverse relationship with the number of nucleotide changes in the capsid region (FIG. 9J).<br>
[0365] The effect on protein translation in vivo and in vitro of the deoptimized MEF viruses was determined using the methods described in Examples 4 and 5. As was observed for the deoptimized Sabin 2 polioviruses, the MEF1 deoptimized viruses had little detectable effect in vivo upon viral protein synthesis and processing (FIG. 5C) or on in vitro translation (FIG. 5D).<br>
[0366] The effect on RNA yields of the deoptimized MEF viruses was determined using the methods described in Example 7, except that the following primers were used to RT-PCR the sequence, CTAAAGATCCCAGAAACACTCA and ATTGGCACACTTCTAATCTTAGC (SEQ ID NOS:. 62 and 63), and amplicon yield measured using CTCTTCCTCGCCATTGTGCCAAG (SEQ ID NO: 64). As was observed for the deoptimized Sabin 2 polioviruses, RNA yields declined with increased number of replacement codons. Total viral RNA yields were highest for ABC, lower for ABc, and lowest for abc (MEF1R9) (FIG. 6B). No increase in viral RNA was observed during the s.s. growth curve for MEF1R9 in HeLa S3 cells.<br>
[0367] The MEF1 viruses were purified using the methods described in Example 6. In addition to the virus band at 1.34g/ml, a large amount of material was observed above the virus band. Some of this material was located where empty capsids might be found in the gradient, but the band was diffuse and quite wide. SDS-PAGE analysis of the material revealed VPO, VP1, VP2 and VP3, which is consistent with an immature virus particle.<br>
[0368] The ratio of infectivity on RD cells compared to HeLa cells (CCID50) increased as the numbers of nt substitutions increased (Table 4). The ratio for MEF1R2 was 4, whereas the ratio for MEF1R9 was 40. Codon deoptimization had a bigger determinental effect on the virus titer measured by plaque assay than the virus titer measured by limiting dilution (CCID50) in HeLa cells. For S2R and MEF1R viruses, CCID50 tilers were higher than PFU liters (Table 4), with S2R23 and MEF1R9 having the highest ratios of CCID50/PFU. Codon deoptimization had a dramatic effect on the specific infectivity of purified MEF1R viruses, as described for S2R. The particle/HeLa PFU ratios ranged from 182 for MEF 1R2 to 18,564 for MEF 1R9. The particle/HeLa CCIDSOs also increased with increased numbers of substitutions, but the effect was more moderate (~4fold for MEF1R9).<br>
Table 4. Infectivity of native and modified polioviruses<br><br>
(Table Removed)<br>
 [0369] In summary, the replicative fitness of Sabin 2 and MEF1 in cell culture was reduced by replacement of preferred codons in the capsid region with synonymous unpreferred codons. The reduction in fitness, as measured by plaque area, was approximately proportional to the length of the interval containing replacement codons.<br>
Example 11<br>
Additional Deoptimization of Poliviruses<br>
[0370] This example describes additional changes that can be made to the Sabin 2 poliovirus capsid sequences disclosed in Example 2, or the MEF1 poliovirus sequences disclosed in Example 10. Such modified sequences can be used in an immunogenic composition<br>
[0371] In one example, the codon deoptimized Sabin 2 poliovirus capsid sequences disclosed in Example 2 (such as SEQ ID NO: 5), or the codon deoptimized MEF1 poliovirus capsid sequences disclosed in Example 10 (such as SEQ ID NO: 58) can be further deoptimized. For example, additional codon substitutions (for example AUA (He), AAA (Lys), and CAU (His)), as well as and redesigned codon substitutions (for example UCG (Ser)) codon substitutions, which are better matched to the least abundant tRNA genes in the human genome (International Human Genome Sequencing Consortium. Nature 409:860-921, 2001), can be used to further impair translational efficiency and reduce replicative fitness. Such substitutions can be made using routine molecular biology methods.<br>
Example 12<br>
Additional Methods to Decrease Replicative Fitness<br>
[0372] This example describes additional or alternative substitutions that can be made to a pathogen sequence to increase the replicative fitness of a pathogen. In addition to changing codon usage, alterations in G+C content and the frequency of CG or TA dinucleotide pairs can be used to decrease the replicative fitness of a pathogen. For example, a pathogen sequence that includes one or more deoptimized codons can further include an alteration in the overall G+C content of the sequence, such<br>
as an increase or decrease of at least 10% in the G+C content in the coding sequence (for example without altering the amino acid sequence of the encoded protein). In another or additional example, a pathogen sequence that includes one or more deoptimized codons can further include an alteration in the number of CG or TA dinucleotides in the sequence, such as an increase or decrease of at least 20% in the number of CG or TA dinucleotides in the coding sequence.<br>
Altering G+C content<br>
[0373] The replicative fitness of a pathogen can be altered by changing the G+C content of a pathogen coding sequence. For example, to increase the G+C content, codons used less frequently by the pathogen that include a "G" or "C" in the third position instead of an "A" or "T" can be incorporated into the deoptimized sequence. Such methods can be used in combination with the other methods disclosed herein for decreasing replicative fitness of a pathogen, for example in combination with deoptimizing codon sequences or altering the frequency of CG or TA dinucleotides.<br>
[0374] In one example, the G+C content of a pathogen coding sequence is reduced to decrease replicative fitness. For example, the G+C content of a rubella virus coding sequence can be reduced to decrease replicative fitness of this virus. In one example, the G+C content of a rubella sequence is decreased by at least 10%, at least 20%, or at least 50%, thereby decreasing replicative fitness of the virus. Methods of replacing C and G nucleotides as well as measuring the replicative fitness of the virus are known in the art, and particular examples are provided herein.<br>
[0375] In another example, the G+C content of a pathogen coding sequence is increased to decrease replicative fitness. For example, the G+C content of a poliovirus coding sequence can be reduced to decrease replicative fitness of this virus. In one example, the G+C content of a poliovirus sequence is increased by at least 10%, at least 20%, or at least 50%, thereby decreasing replicative fitness of the virus. Methods of replacing A and T nucleotides with C and G nucleotides are known in the art, and particular examples are provided herein.<br>
Altering frequency ofCG or TA dinucleotides to decrease replicative fitness [0376] The replicative fitness of a pathogen can be altered by changing the number of CG dinucleotides, the TA dinucleotides, or both, in a pathogen coding sequence. For example, to increase the number of CG dinuclotides in a deoptimized sequence, codons used less frequently by the pathogen that include a CG in the second and third position instead of another dinucleotide can be incorporated into the deoptimized sequence. Such methods can be used in combination with the other methods disclosed herein for decreasing replicative fitness of a pathogen, for example in combination with deoptimizing codon sequences.<br>
[0377] The dinucleotides CG and TA (UA) are known to be suppressed in poliovirus genomes (Karlin et al, J. Virol. 68:2889-97; Kanaya et al., J. Mol. Evol. 53, 290-8; Toyoda et al. J. Mol. Biol.<br>
174:561-85). The results described herein with the Sabin 2 constructs indicate that increased numbers of CG and TA dinucleotides are associated with reductions in replicative fitness. Therefore, the number of CG or TA dinucleotides can be increased in polio and other eukaryotic viruses (such is those in which CG is strongly suppressed in the genome) to decrease their replicative fitness. In one example, the number of CG or TA dinucleotides in a virus sequence is increased by at least 10%, at least 30%, at least 100%, or at least 300%, thereby decreasing replicative fitness of the virus. The number of CG dinucleotides, TA dinucleotides, or both can be increased in a viral sequence using routine molecular biology methods, and using the methods disclosed herein. For example, additional CG dinucleotides can be incorporated into the ORF by uniform replacement of degenerate third-position bases with C when the first base of the next codon is G. Replacement of codons specifying conserved amino acids can be used to further stabilize the reduced fitness phenotype, as restoration of fitness may strictly require synonymous mutations.<br>
Exemplary sequences<br>
[0378] Provided herein are exemplary modified Sabin 2 sequences that have silent (synonymous) nucleotide substitutions in the cassette d (VP1 region). Such modified sequences can be used in an immunogenic composition<br>
[0379] SEQ ID NO: 65 (and FIG. 25) show a Sabin 2 sequence with a reduced number of CG dinucleotides (number of CG dinucleotides reduced by 94%). SEQ ID NO: 66 (and FIG. 26) show a Sabin 2 sequence with a reduced number of both CG dinucleotides and UA dinucleotides (number of CG dinucleotides reduced by 94% and number of TA dinucleotides reduced by 57%). These sequences will likely have similar replicativie fitness as a native poliovirus, and therefore can be used as a control.<br>
[0380] SEQ ID NO: 67 (and FIG. 27) show a Sabin 2 sequence with an increased number of CG dinucleotides (number of CG dinucleotides increased by 389%). SEQ ID NO: 68 (and FIG. 28) show a Sabin 2 sequence with an increased number of both CG dinucleotides and UA dinucleotides, with a priority placed on increasing CG dinucleotides (number of CG dinucleotides increased by 389% and number of TA dinucleotides increased by 203%). These sequences will likely have reduced replicativie fitness compared to a native poliovirus, and therefore can be used in immunogenic compositions.<br>
[0381] SEQ ID NO: 69 (and FIG. 29) show a Sabin 2 sequence having maximum codon deoptimization. In this sequence, the least favored codons were selected without reference to CG or TA dinucleotides. This sequences will likely have reduced replicativie fitness compared to a native poliovirus, and therefore can be used in an immunogenic composition.<br>
[0382] SEQ ID NO: 70 (and FIG. 30) show a Sabin 2 sequence using MEF1 codons for Sabin 2 amino acids. This provides a means of using different, naturally occurring codons. This sequences will likely have similar replicativie fitness as a native poliovirus, and therefore can be used as a control.<br>
Example 13<br>
Determination of the Replication Steps Altered in Highly Modified Viruses [0383] This example describes methods that can be used to identify the defective replication step in a virus whose coding sequence has been altered to reduce replicative fitness of the virus.<br>
[0384] A modified virus, such as a highly modified viruses (for example S2R23 (SEQ ID NO: 5) and MEF1R9 (SEQ ID NO: 58)) can be screened using routine methods in the art. For example, the effects of deoptimizing codons on virus binding, eclipse, uncoating, and particle elution steps can be determined using known methods (Kirkegaard, J. Virol. 64:195-206 and Labadie et al. Virology 318:66-78, 2004, both herein incorporated by reference as to the methods). Briefly, binding assays (Kirkegaard, J. Virol. 64:195-206) could involve determining the percentage of 3H-labeled virions onto HeLa or other cells. After incubation with 3H-labeled purified poliovirus (such as those shown in SEQ ID NOS: 5 and 58), cells are washed extensively with PBS and the initial and remaining radioactivity counts determined by tricholoroacetic acid precipitation and filtering of the labeled particles.<br>
[0385] For conformational alteration assays (Kirkegaard, J. Virol. 64:195-206), polioviruses (such as those shown in SEQ ID NOS: 5 and 58) are prebound to a HeLa monolayer at 4°C for 60 minutes at MOIs of 0.1 PFU/cell. The monolayers are washed three times with PBS and incubated for various time periods at 35°C. Cells are harvested by scraping, and cytoplasmic extracts are titered by plaque assay on HeLa cells. An alternate method (Pelletier et al, Virol. 305:55-65) is to use [35S]-methionine-labeled purified virus particles. Infections are synchronized by a 2.5-hour period of adsorption at 0°C, and then conformational transitions initiated by incubation at 37°C for 3 or 10 minutes. Cell-associated virus particles are separated by centrifugation in sucrose gradients (15-30% w/v) (Pelletier et al., Cell. Mol. Life Sci. 54:1385-402, 1998).<br>
[0386] For KNA release assays (Kirkegaard, J. Virol. 64:195-206), neutral red-containing virus is prepared by harvesting virus (such as those shown in SEQ ID NOS: 5 and 58) from HeLa monolayer grown in the presence of lOug of neutral red per ml. Time courses of UNA release are determined by pre-binding approximately 200 PFU of each virus to HeLa monolayers at 4°C for 60 minutes, followed by washing twice with PBS, and agar overlay. Duplicate plates are irradiated for 8 minutes after various times of incubation at 35°C. The numbers of plaques on the irradiated plates are expressed as a percentage of the number of plaques on the unirradiated control.<br>
[0387] Protein synthesis and the kinetics of host cell shutoff of protein synthesis can be determined by using pulse-chase experiments in infected cells and other standard methods. Pactamycin will be used to study translational elongation rates (Rekosh, J. Virol. 9:479-487). The spectrum of virus particles produced by highly modified viruses can be characterized using fractions from a CsCl density gradient.<br>
[0388] Infectivities in different cell types, such as Vero (African green monkey cell line) and human (and possibly murine) neuroblastoma cell lines, can also be determined using routine methods, such as those disclosed herein.<br>
Example 14<br>
Deoptimized Picornaviruses<br>
[0389] Examples 14-17 describe methods that can be used to generate a deoptimized positive-strand RNA virus. This example describes methods that can be used to generate a deoptimized Picornavirus sequence, which can be used in an immunogenic composition. Particular examples of foot-and-mouth disease virus (FMDV) and polioviruses are described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any Picornavirus.<br>
[0390] Sequences for FMDV are publicly available (for example see GenBank Accession Nos: AJ539141; AY333431; NC_003992; NC_011452; NC_004915; NC_004004; NC_002554; AY593852; AY593851; AY593850; and AY593849). Using publicly available FMDV sequences, along with publicly available codon usage tables from FMDV (for example see Sanchez et al., J. Virol. 77:452-9, 2003; and Boothroyd et al., Gene 17:153-61, 1982, herein incorporated by reference and FIG. 24A), one can generate deoptimized FMDV sequences.<br>
[0391] Using the methods described above in Examples 1 and 2, the capsid of FMDV can be deoptimized. FIGS. 10A-B (and SEQ ID NO: 11) show an exemplary FMDV, serotype 0 strain UKG/35/2001 capsid sequence having codons deoptimized for 9 amino acids (see Table 5). FMDV containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Table 5, one or more other FMDV coding sequences can be deoptimized. In addition, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an FMDV coding sequence, for example to further decrease the replicative fitness of FMDV.<br>
Table 5: Deoptimized FMDV codons<br><br>
(Table Removed)<br>
 [0392] Sequences for poliovirus are publicly available (for example see GenBank Accession Nos: API 11984; NC_002058; AY560657; AY278553; AY278552; AY278551; AY278550; AY27849; AF538843; AF538842; AF538840; AY177685; AY 184221; AY 184220; AY184219; and AY238473). Using publicly available human poliovirus sequences, along with publicly available codon usage tables for poliovirus (Rothberg and Wimmer, Nucleic Acids Res. 9:6221-9, 1981, as well as the tables disclosed herein), one can generate deoptimized poliovirus sequences.<br>
[0393] Using the methods described above (for example see Examples 1 and 2), the capsid of poliovirus can be deoptimized. FIGS. 9A-E (SEQ ID NO: 8) shows an exemplary poliovirus type 2, strain MEF1 capsid sequence having all Arg codons deoptimized to CGG. Poliovirus containing these substitutions can be generated using standard molecular biology methods.<br>
[0394] Similarly, using the methods described above (for example, see Examples 1 and 2), poliovirus types 1 and 3 can be deoptimized (for example by deoptimization of the capsid sequence). For example, the neurovirulent wild strains type 1 Mahoney/USA41 (POLIOIB; GenBank Accession No: V01149) and type 3 Leon/USA37 (POL3L37; GenBank Accession No: K01392), and their Sabin strain derivatives LSc 2ab (Sabin type 1) (GenBank Accession No: V01150), and Leon 12 a,b (Sabin type 3) (GenBank Accession No: X00596) can be deoptimized.<br>
Example 15 Deoptimized Coronaviruses<br>
[0395] This example describes methods that can be used to generate a deoptimized Coronavirus sequence, which can be used in an immunogenic composition. A particular example of a SARS virus is described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any Coronavirus.<br><br>
 [0396] Sequences for SARS are publicly available (for example, see GenBank Accession Nos: NC_004718; AY654624; AY595412; AY394850; AY559097; AY559096; AY559095; AY559094; AY559093; AY559092; AY559091; AY559090; AY559089; AY559088; AY274119; and AY278741). Using publicly available SARS sequences, along with publicly available codon usage tables from SARS (for example, see Rota et al, Science 300:1394-1399, 2003, herein incorporated by reference, and FIG. 24B), one can generate deoptimized SARS sequences.<br>
[0397] Using the methods described above in Examples 1 and 2, the spike glycoprotein of SARS can be deoptimized. FIGS. 11A-C (and SEQ ID NO: 14) shows an exemplary SARS, strain Urbani spike glycoprotein sequence having codons deoptimized for 9 amino acids (see Table 6). SARS containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Table 6, one or more SARS coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an SARS coding sequence, for example to further decrease the replicative fitness of SARS.<br>
Table 6: Deoptimized SARS codons<br>
(Table Removed)<br>
Example 16<br>
Deoptimized Togaviruses<br>
[0398] This example describes methods that can be used to generate a deoptimized togavirus sequence, which can be used in an immunogenic composition. A particular example of a rubella virus is described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any togavirus.<br>
[0399] Sequences for rubella virus are publicly available (for example see GenBank Accession Nos: L78917; NC_001545; AF435866; AF188704 and AB047329). Using publicly available rubella sequences, along with publicly available codon usage tables from rubella virus (for example see Nakamura et al., Nucleic Acids Res. 28:292, 2000 and FIG. 24C), one can generate deoptimized rubella virus sequences. Similar methods can be used to generate a deoptimized sequence for any togavirus.<br>
[0400] Using the methods described above in Examples 1 and 2, the coding sequence of a togavirus can be deoptimized. FIGS. 12A-G (and SEQ ID NO: 18) shows an exemplary rubella virus sequence having codons deoptimized for 10 amino acids (see Table 7). Rubella viruses containing the substitutions shown in FIG. 11 can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Table 7, one or more other rubella coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in a rubella coding sequence, for example to further decrease the replicative fitness of rubella.<br>
Table 7. Deoptimized rubella codons<br><br>
(Table Removed)<br>
Example 17<br>
Deoptimized Flaviviruses<br>
[0401] This example describes methods that can be used to generate a deoptimized flavivirus sequence, which can be used in an immunogenic composition. Particular examples of a Dengue I and Dengue II viruses are described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any flavivirus.<br>
[0402] Sequences for Dengue type 1 and Dengue type 2 virus are publicly available (for example see GenBank Accession Nos: M87512; U88535 and U88536 for type 1 and M19197; M29095 and AF022434 for type 2). Using publicly available Dengue 1 and Dengue 2 sequences, along with publicly available codon usage tables from Dengue type 1 and Dengue type 2 virus (for example see Nakamura et al., Nucleic Acids Res. 28:292, 2000 and FIGS. 22 D and E, respectively), one can generate deoptimized Dengue type I and Dengue type II virus sequences. Similar methods can be used to generate a deoptimized sequence for any flavivirus.<br>
[0403] Using the methods described above in Examples 1 and 2, the coding sequence of a flavivirus can be deoptimized. Flaviviruses, such as Dengue type 1 and 2 viruses, containing these substitutions can be generated using standard molecular biology methods, based on the deoptimized codons provided in Tables 8 and 9. Furthermore, the methods described in Example 12 can be used to alter<br>
-77-<br>
the G+C content or the number of CG or TA dinucleotides in a Flavivirus coding sequence, for example to further decrease the replicative fitness of the Flavivirus.<br>
(Table Removed)<br>
Example 18<br>
Deoptimized Herpesviruses<br>
[0404] This example describes methods that can be used to generate a deoptimized herpesvirus sequence, which can be used in an immunogenic composition. A particular example of a varicella-zoster virus (human herpesvirus 3) is described. In addition, provided is a list of deoptimized codon sequences that can be used for HSV-1 or HSV-2, as well as human cytomegalovirus (CMV; human herpesvirus 5). However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any herpesvirus.<br>
[0405] Sequences for varicella-zoster virus are publicly available (for example see GenBank Accession Nos: NC_001348; AY548170; AY548171; AB097932 and AB097933). Using publicly available varicella-zoster virus sequences, along with publicly available codon usage tables from varicella-zoster virus (for example see Nakamura et al, Nucleic Acids Res. 28:292, 2000 and FIG. 24F), one can generate deoptimized varicella-zoster virus sequences.<br>
[0406] Using the methods described above in Examples 1 and 2, the gH and gE coding sequence of a herpesvirus can be deoptimized. FIGS. 13A-B and 14A-B (and SEQ ID NOS: 21 and 24) show exemplary varicella-zoster virus gH and gE sequences having codons deoptimized for 9 amino acids (see Table 10). Varicella-zoster virus containing these substitutions can be generated using standard molecular biology methods. Using the methods described above in Examples 1 and 2, and standard<br><br>
molecular biology methods, the coding sequence of one or more VZV genes can be deoptimized. In addition, based on the deoptimized codons provided in Table 10, one or more other VZV coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in a VZV coding sequence, for example to further decrease the replicative fitness of the VZV.<br>
Table 10. Deoptimized varicella-zoster codons<br><br>
(Table Removed)<br>
 [0407] Sequences for human cytomegalovirus (CMV; human herpesvirus 5) are publicly available (for example see GenBank Accession Nos: AY446894; BK000394; AC146999; NC_001347; and AYS 15197). Using publicly available CMV sequences, along with publicly available codon usage tables from CMV (for example see Nakamura et al, Nucleic Acids Res. 28:292, 2000 and FIG. 24G), one can generate deoptimized CMV sequences.<br>
[0408] Table 11 shows CMV deoptimized codon sequences for 9 amino acids. The complete genome of CMV is about 233-236 kb. Using the methods described above in Examples 1 and 2, and standard molecular biology methods, glycoprotein B (UL55), glycoprotein H (UL75), and glycoprotein N (UL73) coding sequences of a CMV can be deoptimized. In addition, based on the deoptimized codons provided in Table 11, one or more other CMV coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in a CMV coding sequence, for example to further decrease the replicative fitness of CMV.<br>
(Table Removed)<br>
[0409] Sequences for herpes simplex virus 1 and 2 (HSV1 and HSV2) are publicly available (for example see GenBank Accession Nos: X14112 and NC_001806 for HSV1 and NC_001798 for HSV2). Using publicly available HSV1 and HSV2 sequences, along with publicly available codon usage tables from HSV1 and HSV2 (for example see Nakamura et al, Nucleic Acids Res. 28:292, 2000 and FIG. 24H), one can generate deoptimized HSV1 and HSV2 sequences.<br>
[0410] Table 12 shows HSV1 and HSV2 deoptimized codon sequences for 11 amino acids. The codon choices for HSV1 and 2 are very similar and where there are differences they are small. Therefore, the same codon choices can be used for both HSV1 and HSV2. The complete genome of HSV1 and HSV2 is about 152kb and 155kb, respectively. Using the methods described above in Examples 1 and 2, and standard molecular biology methods, glycoprotein B (UL27), glycoprotein D (US6), tegument protein host shut-off factor (UL41; see Geiss, J. Virol. 74:11137, 2000), and ribonucleotide reductase large subunit (UL39; see Aurelian, Clin. Diag. Lab. Immunol. 11:437-445, 2004) coding sequences of HSV 1 or HSV2 can be deoptimized.   In addition, based on the deoptimized codons provided in Table 12, one or more other HSV1 or HSV2 coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in a HSV1 or HSV2 coding sequence, for example to further decrease the replicative fitness of HSV 1 or HSV2.<br>
Table 12: Deoptimized HSV1 and HSV2 codons<br><br>
(Table Removed)<br>
Example 19<br>
Deoptimized Paramyxoviruses<br>
[0411] Examples 19 and 20 describe methods that can be used to generate a deoptimized negative-strand RNA virus. This example describes methods that can be used to generate a deoptimized paramyxovirus sequence, which can be used in an immunogenic composition. Particular examples of measles and respiratory syncytial viruses (RSV) are described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any paramyxovirus.<br>
[0412] Sequences for measles and RSV are publicly available (for example see GenBank Accession Nos: NC_001498; AF266287; AY486084; AF266291; and AF266286 for measles; and NC_001781; U63644; AY353550; NC_001803; AF013254 and U39661 for RSV). Using publicly available measles and RSV sequences, along with publicly available codon usage tables from measles and RSV (for example see Nakamura et al, Nucleic Acids Res. 28:292, 2000 and FIG. 241), one can generate deoptimized measles and RSV sequences. Similar methods can be used to generate a deoptimized sequence for any paramyxovirus.<br>
[0413] Using the methods described above in Examples 1 and 2, the fusion (F) or hemagglutinin (H) coding sequence of a paramyxovirus can be deoptimized. FIGS. 15A-B and 16A-B show exemplary measles F and G sequences having codons deoptimized for 8 amino acids (SEQ ID NOS: 27 and 30, respectively). FIGS. 17A-B and 18 (and SEQ ID NOS: 33 and 36) show exemplary RSV F and glycoprotein (G) sequences having codons deoptimized for 8 amino acids (see Tables 13 and 14). Measles and RSV viruses containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Tables 13 and 14, one or more other measles or RSV coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in a RSV coding sequence, for example to further decrease the replicative fitness of RSV.<br>
Table 13. Deoptimized measles codons<br><br><br>
(Table Removed)<br>
Example 20<br>
Deoptimized Orthomyxyoviruses<br>
[0414] This example describes methods that can be used to generate a deoptimized orthomyxyovirus sequence, which can be used in an immunogenic composition. A particular example of an influenza virus is described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any orthomyxyovirus.<br>
[0415] Sequences for influenza virus are publicly available (for example see NC_002204 and AY253754). Using publicly available influenza sequences, along with publicly available codon usage tables from influenza (for example see Nakamura et al., Nucleic Acids Res. 28:292, 2000 and FIG. 24J), one can generate deoptimized influenza sequences. Similar methods can be used to generate a deoptimized sequence for any orthomyxyovirus.<br>
[0416] Using the methods described above in Examples 1 and 2, the hemagglutinin (HA) or neuraminidase (NA) coding sequences of an orthomyxyovirus can be deoptimized. FIGS. 17 and 18 show an exemplary influenza virus HA (FIG. 19 and SEQ ID NO: 39) and a NA gene (FIG. 20 and SEQ ID NO: 42) sequence having codons deoptimized for 8 amino acids (see Table 15). Influenza viruses containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Table 15, one or more other influenza coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an influenza coding sequence, for example to further decrease the replicative fitness of influenza.<br>
Table 15. Deoptimized influenza codons<br>
(Table Removed)<br><br>
Example 21<br>
Deoptimized Retroviral Codons<br>
[0417] This example describes methods that can be used to generate a deoptimized retrovirus sequence, which can be used in an immunogenic composition. Particular examples of an HIV type 1 (HIV-1), subtype C, retrovirus, and a lentivirus, are described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any retrovirus.<br><br>
 [0418] Sequences for HIV-1 are publicly available (for example see GenBank Accession Nos: API 10967; AY322191; AY682547; AY536234; AY536238; AY332236; AY331296 and AY331288). Using publicly available HIV-1 sequences, along with publicly available codon usage tables from HIV-1 (for example see Nakamura.e? al, Nucleic Acids Res. 28:292, 2000; Chou and Zhang, AIDS Res. Hum. Retro-viruses. 8:1967-76, 1992; Kyprand Mrazek, Nature. 327(6117):20, 1987, all herein incorporated by reference, and FIG. 24K), one can generate deoptimized HIV-1 sequences. Similar methods can be used to generate a deoptimized sequence for any retrovirus.<br>
[0419] Using the methods described above in Examples 1 and 2, the env coding sequence of HIV-1 can be deoptimized. FIGS. 21A-B (and SEQ ID NO: 45) shows an exemplary HIV-1 env sequence having codons deoptimized for 8 amino acids (see Table 16). HIV-1 containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codons provided in Table 16, one or more other HIV-1 coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an HIV-1 coding sequence, for example to further decrease the replicative fitness of HIV-1.<br>
Table 16: Deoptimized HIV-1 codons<br><br>
(Table Removed)<br>
 [0420] The equine infectious anemia virus (EIAV) is a lentivirus. Sequences for EIAV are publicly available (for example see GenBank Accession Nos: M87581; X16988; NC_001450 and AF327878). Using publicly available EIAV sequences, along with publicly available codon usage tables from EIAV (for example see Nakamura et al., Nucleic Acids Res. 28:292, 2000, herein incorporated by reference, and FIG. 24L), one can generate deoptimized EIAV sequences. Similar methods can be used to generate a deoptimized sequence for any lentivirus.<br>
[0421] Using the methods described above in Examples 1 and 2, the env coding sequence of EIAV can be deoptimized, for example using the deoptimized codons provided in Table 17. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an EIAV coding sequence, for example to further decrease the replicative fitness of EIAV.<br>
Table 17. Deoptimized equine infectious anaemia virus (EIAV) codons<br><br>
(Table Removed)<br>
Example 22<br>
Deoptimized Bacterial Codons<br>
[0422] This example describes methods that can be used to generate a deoptimized bacterial sequence, which can be used in an immunogenic composition. Particular optimized E. coli sequences are described. However, one skilled in the art will appreciate that similar (and in some examples the same) substitutions can be made to any baterial coding sequence.<br>
[0423] Sequences for E. coli are publicly available (for example see GenBank Accession Nos: NC_002695; NC_000913; BA000007; NC_004431; and AE014075). Using publicly available E. coli sequences, along with publicly available codon usage tables from E. coli (for example see Nakamura et al., Nucleic Acids Res. 28:292, 2000 and Sharp et al., Nucleic Acids Res. 16:8207-11, 1988, all herein incorporated by reference, and FIG. 24M), one can generate deoptimized E. coli sequences. Similar methods can be used to generate a deoptimized sequence for any bacterium.<br>
[0424] Using the methods described above in Examples 1 and 2, the ArgS or Tufk coding sequences of E. coli can be deoptimized. FIGS. 22A-B and 23 shows exemplary E. coli ArgS and TufA sequences (and SEQ ID NOS: 48 and 51), respectively, having codons deoptimized for 1 amino acid. E. coli containing these substitutions can be generated using standard molecular biology methods. In addition, based on the deoptimized codon provided in Table 18, one or more other E. coli coding sequences can be deoptimized. Furthermore, the methods described in Example 12 can be used to alter the G+C content or the number of CG or TA dinucleotides in an E. coli coding sequence, for example to further decrease the replicative fitness of E. coli.<br>
Table 18. Deoptimized E. coli K12 codon<br><br>
(Table Removed)<br>
Example 23<br>
Pharmaceutical Compositions<br>
[0425] The disclosed immunogenic deoptimized pathogenic sequences can be incorporated into pharmaceutical compositions (such as immunogenic compositions or vaccines). Pharmaceutical compositions can include one or more deoptimized pathogenic sequences and a physiologically acceptable carrier. Pharmaceutical compositions also can include an immunostimulant. An immunostimulant is any substance that enhances or potentiates an immune response to an exogenous antigen. Examples of immunostimulants include adjuvants, biodegradable microspheres (such as polylactic galactide microspheres) and liposomes (see, for example, U.S. Pat. No. 4,235,877). Vaccine preparation is generally described, for example, in M. F. Powell and M. J. Newman, eds., Vaccine Design: the submit and adjuvant approach, Plenum Press, NY, 1995. Pharmaceutical compositions within the scope of the disclosure can include other compounds, which may be either biologically active or inactive.<br>
[0426] A pharmaceutical composition can include DNA having a deoptimized coding sequence. The DNA can be present within any of a variety of delivery systems known to those of ordinary skill in the art, including nucleic acid expression systems, bacteria and viral expression systems. Numerous gene delivery techniques are well known in the art, including those described by Holland, Crit. Rev. Therap. Drug Carrier Systems 15: 143-198, 1998, and references cited therein. Appropriate nucleic acid expression systems contain DNA sequences for expression in the subject (such as a suitable promoter and terminating signal). Bacterial delivery systems involve the administration of a bacterium (such as Bacillus-Calmette-Guerrin) that expresses the polypeptide on its cell surface or secretes it. In one example, the DNA is introduced using a viral expression system (such as vaccinia or other pox virus, retrovirus, or adenovirus), which can involve the use of a non-pathogenic (defective), replication competent virus. Suitable systems are disclosed, for example, in Fisher-Hoch et al., Proc. Natl. Acad. Sci., USA 86:317-21, 1989; Flexner et al., Ann. N.Y. Acad Sci. 569:86-103, 1989; Flexner et al., Vaccine 8:17-21, 1990; U.S. Pat. Nos. 4,603,112, 4,777,127, 4,769,330, and 5,017,487; PCT publications WO 89/01973 and WO 91/02805; Berkner, Biotechniques 6:616-27, 1988; Rosenfeld et al., Science 252:431-4, 1991; Kolls et al., Proc. Natl. Acad. Sci. USA 91:215-9, 1994; Kass-Eisler et al., Proc. Natl. Acad Sci. USA 90:11498-502, 1993; Guzman et al., Circulation 88:2838-48, 1993; and Guzman et al., Cir. Res. 73:1202-7, 1993. Techniques for incorporating DNA into such expression systems are known. DNA can also be incorporated as "naked DNA," as described, for example, in Ulmer et al., Science 259:1745-9, 1993 and Cohen, Science 259:1691-2, 1993. Uptake of naked DNA can be increased by coating the DNA onto biodegradable beads.<br>
[0427] While any suitable carrier known to those of ordinary skill in the art can be employed in the pharmaceutical compositions, the type of carrier will vary depending on the mode of administration.<br>
Pharmaceutical compositions can be formulated for any appropriate manner of administration, including for example, oral (including buccal or sublingual), nasal, rectal, aerosol, topical, intravenous, intraperitoneal, intradermal, intraocular, subcutaneous or intramuscular administration. For parenteral administration, such as subcutaneous injection, exemplary carriers include water, saline, alcohol, fat, wax, buffer, or combinations thereof. For oral administration, any of the above carriers or a solid carrier can be employed. Biodegradable microspheres (such as polylactate polyglycolate) can also be employed as carriers for the pharmaceutical compositions. Suitable biodegradable microspheres are disclosed, for example, in U.S. Pat. Nos. 4,897,268 and 5,075,109.<br>
[0428] The disclosed pharmaceutical compositions can also include buffers (such as neutral buffered saline or phosphate buffered saline), carbohydrates (such as glucose, mannose, sucrose or dextrans), mannitol, and additional proteins, polypeptides or amino acids such as glycine, antioxidants, chelating agents such as EDTA or glutathione, and immunostimulants (such as adjuvants, for example, aluminum phosphate) or preservatives.<br>
[0429] The compositions of the present disclosure can be formulated as a lyophilizate, or stored at temperatures from about 4°C to -100°C. Compositions can also be encapsulated within liposomes using well known technology. Furthermore, the compositions can be sterilized, for example, by filtration, radiation, or heat.<br>
[0430] Any of a variety of immunostimulants can be employed in the pharmaceutical compositions that include an immunogenically effective amount of attenuated deoptimized pathogen. In some examples, an immunostimulatory composition also includes one or more compounds having adjuvant activity, and can further include a pharmaceutically acceptable carrier.<br>
[0431] Adjuvants are non-specific stimulators of the immune system that can enhance the immune response of the host to the immunogenic composition. Some adjuvants contain a substance designed to protect the antigen from rapid catabolism, for example, aluminum hydroxide or mineral oil, and a stimulator of immune responses, such as lipid A, Bordatella pertussis or Mycobacterium tuberculosis derived proteins. Suitable adjuvants are commercially available as, for example, Merck Adjuvant 65 (Merck and Company, Inc., Rahway, N.J.), TiterMax Gold (TiterMax, Norcross, GA), ISA-720 (Seppic, France) ASO-2 (SmithKlineGlaxo, Rixensart, Belgium); aluminum salts such as aluminum hydroxide (for example, Amphogel, Wyeth Laboratories, Madison, NJ) or aluminum phosphate; salts of calcium, iron or zinc; an insoluble suspension of acylated tyrosine; acylated sugars; cationically or anionically derivatized polysaccharides; polyphosphazenes; biodegradable microspheres; monophosphoryl lipid A and saponins such as quil A and QS-21 (Antigenics, Framingham, MA). Cytokines, such as GM-CSF or interleukin-2, -7, or -12, can be used as adjuvants.<br>
[0432] The adjuvant composition can be designed to induce an immune response predominantly of the Thl type. High levels of Thl-type cytokines (such as IFN-y, TNF-a, IL-2 and IL-12) tend to<br>
favor the induction of cell mediated immune responses to an administered antigen. In contrast, high levels of Th2-type cytokines (such as IL-4, IL-5, IL-6 and IL-10) tend to favor the induction of humoral immune responses. Following administration of a pharmaceutical composition as provided herein, a subject may support an immune response that includes Thl- and Th2-type responses. However, in examples where the response is predominantly a Thl-type, the level of Thl-type cytokines increases to a greater extent than the level of Th2-type cytokines. The levels of these cytokines can be readily assessed using standard assays.<br>
[0433] Adjuvants for use in eliciting a predominantly Th 1 -type response include, but are not limited to, a combination of monophosphoryl lipid A, such as 3-de-O-acylated monophosphoryl lipid A (3D-MPL) (Corixa, Hamilton IN), together with an aluminum salt. MPL adjuvants are available from Corixa (Seattle, WA; see also U.S. Pat. Nos. 4,436,727; 4,877,611; 4,866,034 and 4,912,094). CG-containing oligonucleotides (in which the CG dinucleotide is unmethylated) also induce a predominantly Thl response. Such oligonucleotides are well known and are described, for example, in PCT publications WO 96/02555 and WO 99/33488. Immunostimulatory DNA sequences are also described, for example, by Sato et al., Science 273:352, 1996. Another adjuvant is a saponin such as QS21 (Antigenics, Framingham, MA), which may be used alone or in combination with other adjuvants. For example, an enhanced system involves the combination of a monophosphoryl lipid A and saponin derivative, such as the combination of QS21 and 3D-MPL as described in WO 94/00153, or a less reactogenic composition where the QS21 is quenched with cholesterol, as described in WO 96/33739. Other formulations include an oil-in-water emulsion and tocopherol. An adjuvant formulation involving QS21, 3D-MPL and tocopherol in an oil-in-water emulsion is described in WO 95/17210.<br>
[0434] Still further adjuvants include Montanide ISA 720 (Seppic, France), SAP (Chiron, California, United States), ISCOMS (CSL), MF-59 (Chiron), the ASO-2 series of adjuvants (SmithKlineGlaxo, Rixensart, Belgium), Detox (Corixa, Seattle, WA), RC-529 (Corixa, Seattle, WA), Aminoalkyl glucosaminide 4-phosphates (AGPs), copolymer adjuvants, CG oligonucleotide motifs and combinations of CG oligonucleotide motifs, bacterial extracts (such as mycobacterial extracts), detoxified endotoxins, and membrane lipids. Combinations of two or more adjuvants can also be used.<br>
[0435] Still other adjuvants include polymers and co-polymers. For example, copolymers such as polyoxyethylene-polyoxypropylene copolymers and block co-polymers can be used. A particular example of a polymeric adjuvant is polymer P1005.<br>
[0436] Adjuvants are utilized in an adjuvant amount, which can vary with the adjuvant, subject, and immunogen. Typical amounts of non-emulsion adjuvants can vary from about 1 ng to about 500 mg per administration, for example, from 10 ug to 800 ug, such as from 50 ng to 500 ug. For emulsion<br>
adjuvants (oil-in-water and water-in-oil emulsions) the amount of the oil phase can vary from about 0.1% to about 70%, for example between about 0.5% and 5% oil in an oil-in-water emulsion and between about 30% and 70% oil in a water-in-oil emulsion. Those skilled in the art will appreciate appropriate concentrations of adjuvants, and such amounts can be readily determined.<br>
[0437] Any pharmaceutical composition provided herein can be prepared using well known methods that result in a combination of deoptimized pathogen (or deoptimized DNA coding sequence), alone or in the presence of an immunostimulant, carrier or excipient, or combinations thereof. Such compositions can be administered as part of a sustained release formulation (such as a capsule, sponge or gel that includes the deoptimized pathogen) that provides a slow release of the composition following administration. Such formulations can be prepared using well known technology (see, for example, Coombes et al., Vaccine 14:1429-38, 1996) and administered by, for example, subcutaneous implantation at the desired target site. Sustained-release formulations can contain a deoptimized pathogen dispersed in a carrier matrix or contained within a reservoir surrounded by a rate controlling membrane.<br>
[0438] Carriers for use with the disclosed compositions are biocompatible, and can also be biodegradable, and the formulation can provide a relatively constant level of active component release. Suitable carriers include, but are not limited to, microparticles of poly(lactide-co-glycolide), as well as polyacrylate, latex, starch, cellulose and dextran. Other delayed-release carriers include supramolecular biovectors, which comprise a non-liquid hydrophilic core (such as a cross-linked polysaccharide or oligosaccharide) and, optionally, an external layer comprising an amphiphilic compound, such as a phospholipid (see, for example, U.S. Pat. No. 5,151,254 and PCT publications WO 94/20078, WO/94/23701 and WO 96/06638). The amount of active compound contained within a sustained release formulation depends upon the site of implantation, the rate and expected duration of release and the nature of the condition to be treated or prevented.<br>
[0439] Any of a variety of delivery vehicles can be employed with the disclosed pharmaceutical compositions to facilitate production of an antigen-specific immune response to a deoptimized pathogen. Exemplary vehicles include, but are not limited to, hydrophilic compounds having a capacity to disperse the deoptimized pathogen and any additives. The deoptimized pathogen can be combined with the vehicle according to methods known in the art. The vehicle can be a solvent or dispersion medium containing, for example, water, ethanol, polyol (for example, glycerol, propylene glycol, liquid polyethylene glycol, and the like), and suitable mixtures thereof. Other exemplary vehicles include, but are not limited to, copolymers of polycarboxylic acids or salts thereof, carboxylic anhydrides (for example, maleic anhydride) with other monomers (for example, methyl (meth)acrylate, acrylic acid and the like), hydrophilic vinyl polymers, such as polyvinyl acetate, polyvinyl alcohol, polyvinylpyrrolidone, cellulose derivatives, such as hydroxymethylcellulose,<br>
hydroxypropylcellulose and the like, and natural polymers, such as chitosan, collagen, sodium alginate, gelatin, hyaluronic acid, and nontoxic metal salts thereof.<br>
[0440] A biodegradable polymer can be used as a base or vehicle, such as polyglycolic acids and polylactic acids, poly(lactic acid-glycolic acid) copolymer, polyhydroxybutyric acid, poly(hydroxybutyric acid-glycolic acid) copolymer, and mixtures thereof. Other biodegradable or bioerodable polymers include, but are not limited to, such polymers as poly(epsilon-caprolactone), poly(epsilon-aprolactone-CO-lactic acid), poly(epsilon.-aprolactone-CO-glycolic acid), poly(beta-hydroxy butyric acid), poly(alkyl-2-cyanoacrilate), hydrogels, such as poly(hydroxyethyl methacrylate), polyamides, poly(amino acids) (for example, L-leucine, glutamic acid, L-aspartic acid and the like), poly(ester urea), poly(2-hydroxyethyl DL-aspartamide), polyacetal polymers, polyorthoesters, polycarbonate, polymaleamides, polysaccharides, and copolymers thereof. In some examples, vehicles include synthetic fatty acid esters such as polyglycerin fatty acid esters and sucrose fatty acid esters. Hydrophilic polymers and other vehicles can be used alone or in combination, and enhanced structural integrity can be imparted to the vehicle by partial crystallization, ionic bonding, cross-linking and the like.<br>
[0441] The vehicle can be provided in a variety of forms, including, fluid or viscous solutions, gels, pastes, powders, microspheres and films. In one example, pharmaceutical compositions for administering a deoptimized pathogen are formulated as a solution, microemulsion, or other ordered structure suitable for high concentration of active ingredients. Proper fluidity for solutions can be maintained, for example, by the use of a coating such as lecithin, by the maintenance of a desired particle size in the case of dispersible formulations, and by the use of surfactants.<br>
[0442] Delivery vehicles include antigen presenting cells (APCs), such as dendritic cells, macrophages, B cells, monocytes and other cells that can be engineered to be efficient APCs. Such cells can, but need not, be genetically modified to increase the capacity for presenting the antigen, to improve activation or maintenance of the T cell response, to have anti-pathogen effects, or to be immunologically compatible with the receiver (matched HLA haplotype). APCs can generally be isolated from any of a variety of biological fluids and organs, including tumor and peritumoral tissues, and may be autologous, allogeneic, syngeneic or xenogeneic cells.<br>
[0443] In certain examples, the deoptimized pathogen is administered in a time release formulation. These compositions can be prepared with vehicles that protect against rapid release, and are metabolized slowly under physiological conditions following their delivery (for example in the presence of bodily fluids). Examples include, but are not limited to, a polymer, controlled-release microcapsules, and bioadhesive gels. Many methods for preparing such formulations are well known to those skilled in the art (see, for example, Sustained and Controlled Release Drug Delivery Systems, J. R. Robinson, ed., Marcel Dekker, Inc., New York, 1978).<br>
[0444] Pharmaceutical compositions can be presented in unit-dose or multi-dose containers, such as sealed ampoules or vials. Such containers are typically hermetically sealed to preserve sterility of the formulation until use. In general, formulations can be stored as suspensions, solutions or as emulsions in oily or aqueous vehicles. Alternatively, a pharmaceutical composition can be stored in a freeze-dried condition requiring only the addition of a sterile liquid carrier immediately prior to use.<br>
[0445] The pharmaceutical compositions of the disclosure typically are sterile and stable under conditions of manufacture, storage and use. Sterile solutions can be prepared by incorporating the disclosed deoptimized pathogens (alone or in the presence of a pharmaceutically acceptable carrier, adjuvant, or other biologically active agent) in the desired amount in an appropriate solvent followed by sterilization, such as by filtration. Generally, dispersions are prepared by incorporating the deoptimized pathogen into a sterile vehicle that contains a dispersion medium and other desired ingredients. In the case of sterile powders, methods of preparation include vacuum drying and freeze-drying which yields a powder of the deoptimized pathogen plus any additional desired ingredient from a previously sterile-filtered solution thereof. For vaccine use, the deoptimized pathogens of the disclosure can be used directly in vaccine formulations, or lyophilized, as desired, using lyophilization protocols well known in the art. Lyophilized pathogen is typically be maintained at about 4°C. When ready for use the lyophilized pathogen can be reconstituted in a stabilizing solution (such as saline).<br>
Example 24<br>
Methods of Stimulating an Immune Response<br>
[0446] This example describes methods using the disclosed immunogenic compositions that can be used to stimulate an immune response in a subject, such as a human. Methods for inoculation are routine in the art. In some examples, a determination is made as to whether the subject would benefit from administration of a deoptimized pathogen sequence, prior to administering the immunogenic composition. Administration can be achieved by any method known in the art, such as oral administration or inoculation (such as intramuscular, ip, or subcutaneous). In some examples, the deoptimized pathogen is administered, for example an inactivated or live pathogen. In particular examples, the deoptimized nucleic acid molecule or protein molecule is administered. In some examples, combinations of these agents are administered, alone or in the presence of other agents, such as an adjuvant.<br>
[0447] The amount of deoptimized pathogen (or part thereof such as DNA sequence) administered is sufficient to induce in the host an effective immune response against virulent forms of the pathogen. An effective amount can being readily determined by one skilled in the art, for example using routine trials establishing dose response curves. The immunogenic compositions disclosed herein can be administered to the subject as needed to confer immunity against the pathogen to the subject. For example, the composition can be administered in a single bolus delivery (which can be<br>
followed by one or more booster administrations as needed), via continuous delivery over an extended time period, in a repeated administration protocol (for example, by an hourly, daily, weekly, or monthly repeated administration protocol).<br>
[0448] In some examples, a deoptimized viral sequence is administered to a subject. The sequence can be administered as a nucleic acid molecule, the virus itself, or combinations thereof. In one example, a deoptimized DNA sequence is administered to the subject, for example in the presence of a carrier molecule, such as a lipid (for example a liposome). The amount of DNA administered can be determined by routine methods in the art. In some examples, the amount of DNA administered (for example by orally or inoculation) is 0.1 ng - 1000 ug DNA, such as 10-100 jig DNA, such as at least 10 ug DNA. In particular examples, a deoptimized virus (live or inactivated, and in some examples lyophilized) is administered to the subject (for example orally or via injection). Exemplary doses of virus, include, but are not limited to, 103 to 1010 plaque forming units (PFU) or more of virus per dose, such as 104 to 10s PFU virus per dose, for example at least 103 PFU virus per dose, at least<br>
104	PFU virus per dose, at least 105 PFU virus per dose, or at least 109 PFU virus per dose.<br>
[0449] In some examples, a deoptimized bacterial sequence is administered to a subject. The sequence can be administered as a nucleic acid molecule, or as the bacterium. In examples wherein a deoptimized bacterial DNA sequence is administered, the methods described above can be used. In particular examples, a deoptimized bacterium (such as an inactivated whole-cell vaccine) is administered to the subject (for example orally or via injection). Exemplary doses of bacteria (as measured by colony-forming units), include, but are not limited to, 103 - 1010 bacteria per dose, for example at least 103 bacteria, at least 104 bacteria, at least 105 bacteria, at least 10s bacteria, or at least 109 bacteria per dose.<br>
[0450] In some examples, a deoptimized parasitic sequence is administered to a subject. The sequence can be administered as a nucleic acid molecule, or as the parasite. In examples wherein a deoptimized parasitic DNA sequence is administered, the methods described above can be used. In particular examples, a deoptimized parasite (such as a live or inactivated parasite) is administered to the subject (for example orally or via injection). Exemplary doses of parasites, include, but are not limited to, 103 - 1010 parasites per dose, for example at least 103 parasites, at least 104 bacteria, at least<br>
105	parasites, at least 10s parasites, or at least 109 parasites per dose.<br>
Example 25<br>
Attenuated Poliovirus as an Immunogen<br>
[0451] This example describes methods that can be used to demonstrate the ability of an attenuated poliovirus to be used as an immunogen.<br>
Wild-type mouse neurovirulence using deoptimized MEFl viruses<br>
[0452] The method of Ford et al. (Microbial Pathogenesis 33:97-107, 2002, herein incorporated by reference) can be used. Wild-type mice are infected with the wild type 2 poliovirus strain MEFl. MEFl is a mouse-adapted type 2 polio strain that cannot infect mice via the oral route, but can infect via injection. Briefly, wild-type mice (such as six-week old, adult, male Swiss mice (laconic Labs, Germantown, NY)) are anesthetized with isofluorane and subsequently administered the virus via intramuscular injection (right medial gastrocnoemius) utilizing a 26.5 gauge needle. In some examples, the virus is injected into the brain or spinal cord. Mice each are administered approximately 1010- 10" TCID50 (amount of virus required for 50% infectivity of susceptible cells in tissue culture) of MEF1R2 (an MEFl clone with an extra silent restriction site; SEQ ID NO: 53), MEFl (non-clone; SEQ ID NO: 52), MEF1R5 (Wl alterations; SEQ ID NO: 54), MEF1R9 (SEQ ID NO: 58), or with phospho-buffered saline (PBS) as a negative control.<br>
[0453] All inoculated animals are observed daily for signs of disease (paralysis, encephalitis, or death). Paralysis is defined as limb weakness and delineated between spastic/hypertonic and flaccid/hypotonic by a neurologist. Tone is determined by manual manipulation of the limb and compared with normal tone in uninoculated mice. Blood will be collected from mice 21 days after infection. Serum samples are analyzed for the presence of neutralizing antibody to poliovirus. Blood will be collected before euthanasia when necessary.<br>
[0454] The following methods can be used to assess immunogenicity of the deoptimized viruses. The presence of neutralizing antibodies can be assessed by using the neutralization test (standard WHO method), as described in Horie et al. (Appl. Environ. Microbiol. 68:138-42, 2002). Following immunization, sera is obtained from immunized and non-immunized subjects. About 50 ul of sera dilution series is prepared, in duplicate, in Eagle's minimal essential medium (MEM) supplemented with 2% PCS in a 96-weIl microtiter plate. Then 50 ul of 100 50% cell culture infectious doses (CCID50) of each isolate, Sabin type 2 vaccine strain, or type 2 wild strain MEFl is added to each well. After incubation at 36°C for 2 hours, 100 ul of a cell suspension containing 104 HEp2-C cells in MEM supplemented with 5% PCS are added to each well. The plates are then scored or CPE after 7 days of incubation at 36°C in a CO2 atmosphere. The calculation of the neutralizing titer of each sample can be determined by the Karber method (see World Health Organization. 1990. Manual for the virological investigation of poliomyelitis. World Health Organization, Expanded Programme on Immunization and Division of Communicable Diseases. W.H.O. publication no. W.H.O./EPI/CDS/POLIO/90.1. World Health Organization, Geneva, Switzerland).<br>
[0455] Production of specific neutralizing antibodies when inoculated with codon-deoptimized virus constructs of MEFl would give evidence of protective immunity. Protection from paralysis<br>
upon challenge with dosages of MEF1 sufficient to cause paralysis in unprotected mice would be confirmation of protective immunity.<br>
Transgenic mice bearing the human poliovirus receptor<br>
[0456] As an alternative to using wild-type mice, transgenic mice expressing the human poliovirus receptor can be used (PVR-Tg21 mice, Central Laboratories for Experimental Animals, Kanagawa, Japan), using the methods described above. Briefly, transgenic PVR-Tg21 mice at 8 - 10 weeks of age are administered the deoptimized virus (such as a sequence that includes SEQ ID NO: 5 or 58), wild-type virus, other polio virus, or buffer alone. Administration can be by any mode, such as injection into the muscle as described above, intranasal, intraspinal or intracerebral inoculation. However, injection into muscle in some examples requires a higher dose of virus than intraspinal or intracerebral inoculation. Intraspinal injection can be performed as described in Horie et al. (Appl. Envir. Microbiology 68:138-142, 2002). Briefly, the desired virus is serially diluted 10-fold, and 5 ul of each dilution inoculated into the spinal cord of 5-10 mice per dilution. Intracerebral injection can be performed as described in Kew et al. (Science 296:356- 9, 2002). Briefly, mice are inoculated (30 ul/mouse) intracerebrally for each virus dilution (in 10-fold increments). Intranasal infection can be performed using the method ofNagata et al. (Virology 321:87-100, 2004), as transgenic mice are susceptible to polio infection via the intranasal route.<br>
Analysis of challenge/protection<br>
[0457] After the neurovirulence properties of the codon-deoptimized viruses are determined, challenge studies can be used to demonstrate that the codon-deoptimized viruses protect mice from disease. Briefly, mice are inoculated with a codon-deoptimized virus using conditions that induce neutralizing antibody. Immunized mice are challenged 21 days later with neurovirulent type 2 MEF1 virus at paralytic doses. The absence of paralytic signs when challenged with neurovirulent prototype MEF1 indicates that the transgenic PVR-Tg21 mice are protected by their prior exposure to codon-deoptimized MEF1 virus. The type-specificity of protection is measured by challenge with the neurovirulent type 1 poliovirus, Mahoney and neurovirulent type 3 poliovirus.<br>
Monkey Neurovirulence<br>
[0458] As an alternative to using mice, the ability of a deoptimized poliovirus to be used as an immunogen can be determined in rhesus monkeys. Deoptimized polioviruses, such as those disclosed herein, can be administered to monkeys and neurovirulence assayed. Examples of deoptimized viruses include, but are not limited to sequences that include SEQ ID NOS: 5, 8, 58, or 65-70). Briefly, intraspinal inoculation of rhesus monkeys will be performed according to the recommendations of the World Health Organization for Type 2 OPV (WHO Tech. Rep. Ser. 800, 30-65, 1990). Requirements for poliomyelitis vaccine (oral), and the United States Code of Federal Regulations, Title 21, Part 630.16 (1994). For example, 10-14 juvenile rhesus monkeys will be inoculated in the lumbar region of the spinal cord with 0.1-0.2 ml of virus (6-7 logio<br><br>
CCID5o/monkey). The ability of the deoptimized virus to stimulate an immune response in the treated monkeys can be determined as described above.<br>
Example 26<br>
Methods of Determining Replicative Fitness<br>
[0459] This example describes methods that can be used to measure the replicative fitness of a virus or bacteria. One skilled in the art will appreciate that other methods can also be used.<br>
[0460] In one example, the replicative fitness of a deoptimized virus is determined by calculation of plaque size and number. Briefly, RNA transcripts of viral sequences having a deoptimized sequence or a native sequence are transfected into the appropriate cell line. The resulting virus obtained from the primary transfection can be passaged again to increase virus liters. The virus is then used to infect cells (such as confluent HeLa cell monolayers), and incubated at room temperature for 10-60 minutes, such as 30 minutes, prior to the addition of 0.45% SeaKem LE Agarose (BioWhittaker Molecular, Rockland, ME) in culture medium. Plates are incubated for 50-100 hours at 35°C (or at a temperature most appropriate for the virus strain under study), fixed with 0.4% formaldehyde and stained with 3 % crystal violet. Plaque size is the quantified, for example by manual measurement and counting of the plaques, or by scanning plates (for example on a FOTO/Analyst Archiver system, Fotodyne, Hartland, WI) and subsequent image analysis (for example using Scion Image for Windows, Scion Corp., Frederick, MD). A codon-deoptimized virus is considered to have reduced replicative fitness when the size or number of plaques is reduced by at least 50%, for example at least 75%, as compared to the size or number of plaques generated by the native virus.<br>
[0461] The replicative fitness of a virus can also be determined using single-step growth experiments. Virus (deoptimized and native) is generated as described above. The appropriate cells (such as HeLa cells) are infected at a multiplicity of infection (MOI) of 1-10 PFU/cell with stirring for 10-60 minutes at 35°C. Cells are then sedimented by low-speed centrifugation and resuspended in culture media. Incubation continued at 35°C in a water bath with orbital shaking at 300 rpm. Samples are withdrawn at 2-hour intervals from 0 to 14 hours postinfection, and titered by plaque assay as described above.<br>
[0462] To determine the replicative fitness of a bacterium or yeast pathogen, a colony-forming assay can be performed. Briefly, bacterial or yeast suspensions can be plated onto agar plates containing solidified medium with the appropriate nutrients, and after incubation (normally at 37 °C), the number of colonies are counted. Alternatively, growth rates can be measured spectrophotometrically by following the increase in optical density of the appropriate liquid medium after inoculation with the bacterial or yeast cultures. Another method to measure growth rates would use quantitative PCR to determine the rate of increase of specific nucleic acid targets as the bacterial or yeast cells are incubated in the appropriate liquid medium.<br><br>
[0463] In view of the many possible embodiments to which the principles of our invention may be applied, it should be recognized that the illustrated examples are only particular examples of the invention and should not be taken as a limitation on the scope of the invention. Rather, the s<br>
cope of the invention is defined by the following claims. We therefore claim as our invention all that comes within the scope and spirit of these claims.<br><br><br><br><br><br>
WE CLAIM:<br>
1.	A method of reducing replicative fitness of a virus, comprising:<br>
deoptimizing at least twenty codons in a coding sequence of the virus by replacing in the coding sequence with a synonymous codon less frequently used in the virus, thereby generating a deoptimized coding sequence to reduce replicative fitness of the virus.<br>
2.	The method as claimed in claim 1, wherein the replicative fitness of the virus is reduced by at<br>
least 20% as compared to an amount of replicative fitness by the virus having a coding sequence<br>
with a native codon composition.<br>
3.	The method as claimed in *claim 1, wherein at least 50% of the coding sequence is<br>
deoptimized.	.<br>
4.	The method as claimed in claim 1, wherein deoptimizing the codon composition alters G+C content in the coding sequence by at least 20%.<br>
5.	The method as claimed in claim 4, wherein deoptimizing the codon composition increases G+C content in the coding sequence by at least 40%.<br>
6.	The method as claimed m claim 4, wherein deoptimizing the codon composition decreases G+C content in the coding sequence by at least 40%.<br>
7.	The method as claimed in claim 5, wherein deoptimizing the at least twenty codons increases<br>
, G+C content in the coding sequence by at least 48%.<br>
8.	The method as claimed in claim 1, wherein deoptimizing the codon composition alters the number of CG dinucleotides, the number of T A dinucleotides, or the number of CG dinucleotides and TA nucleotides in the coding sequence by at least 30%.<br>
9.	The method as claimed in claim 8, wherein deoptimizing the codon composition increases the number CG dinucleotides or T A dinucleotides in the coding sequence by at least 100%.<br><br>
10.	The method as claimed in claim 1, wherein the virus is a positive-strand RNA virus.<br>
11.	The method as claimed in claim 10, wherein at least 20 codons in a capsid coding sequence are deoptimized.<br>
12.	The method as claimed in claim 11, wherein at least 50 codons in a capsid coding sequence are deoptimized.<br>
13.	The method as claimed in claim 1, wherein at least 97% of the capsid coding sequence is deoptimized.<br>
14.	The method as claimed in claim 10, wherein the posifive-strand RNA virus is a Coronavirus, and wherein at least 20 codons in a spike glycoprotein coding sequence are deoptimized.<br>
15.	The method as claimed in claim 1, wherein the virus is a herpesvirus, and wherein at least 20 codons in a gH or gE coding sequence are deoptimized.<br>
16.	The method of claim 1, wherein the virus is a herpesvirus, and wherein at least 20 codons in a glycoprotein B, glycoprotein H, or glycoprotein N coding sequence are deoptimized.<br>
17.	The method as claimed in claim 1, wherein the virus is a herpesvirus, and wherein at least 20 codons in a glycoprotein B, glycoprotein D, tegument protein host shut-off factor, or ribonucleotide reductase large subunit coding sequence are deoptimized.<br>
18.	The method as claimed in claim 10, wherein the positive-strand RNA virus is a togavirus, and wherein at least 20 codons are optimized for a human codon usage.<br>
19.	The method as claimed in claim 1, wherein the virus is a negative-strand RNA virus.<br><br>
20.	The method as claimed in claim 19, wherein the negative-strand RNA virus is a<br>
paramyxovirus, and wherein at least 20 codons in a fusion (F) or glycoprotein (G) coding<br>
sequence are deoptimized.<br>
21.	The method as claimed in claim 19, wherein the negative-strand RNA virus is an<br>
orthomyxyovirus, and wherein at least 20 codons in a hemagglutinin (HA) or neuraminidase<br>
(NA) coding sequence are deoptimized.<br>
22.	The method as claimed in claim 1, wherein the virus is a retrovirus.<br>
23.	The method as claimed in claim 22, wherein the retrovirus virus is a human<br>
immunodeficiency virus (HIV) and wherein at least 20 codons in an eny coding sequence are<br>
deoptimized.<br>
24.	The method as claimed in any one of claims 10 to 13, wherein the positive strand virus is a<br>
poliovirus.<br>
25.	The method as claimed in claim 1, wherein the deoptimized coding sequence comprises a<br>
sequence shown in any of SEQ ED NOS: 5, 8, 11, 14, 18, 21, 24, 27, 30, 33, 36, 39, 42, 45, 54,<br>
55, 56, 57, 58, 67, 68, or 69.<br>
26.	An attenuated virus, made by the method as claimed in claim 1.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUFic3RyYWN0LSgxMC0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Abstract-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUFic3RyYWN0LSgxNy0wNy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Abstract-(17-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUNsYWltcy0oMTAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Claims-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUNsYWltcy0oMTctMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Claims-(17-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Correspondence Others-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMDctMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Correspondence Others-(17-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LURyYXdpbmdzLSgxMC0wMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Drawings-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUZvcm0tMy0oMTAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Form-3-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LUdQQS0oMTAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-GPA-(10-01-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1kZWxucC0yMDA3LXBjdC0yMzcucGRm" target="_blank" style="word-wrap:break-word;">3346-delnp-2007-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MzM0Ni1ERUxOUC0yMDA3LVBldGl0aW9uLTEzNy0oMTAtMDEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3346-DELNP-2007-Petition-137-(10-01-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="254017-method-of-generating-an-image-incorporating-a-microstructure-and-method-of-generating-a-security-document.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="254019-novel-compounds-of-imidazo-fused-4-hydroxy-quinoline-carboxamides.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>254018</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3346/DELNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>37/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>14-Sep-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>13-Sep-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>04-May-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>THE GOVERNMENT OF THE UNITED STATES OF AMERICA AS REPRESENTED BY THE SECRETARY OF THE DEPARTMENT OF HEALTH AND HUMAN SERVICES, CENTERS FOR DISEASE CONTROL AND PREVENTION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>TECHNOLOGY TRANSFER OFFICE, 4770 BUFORD HIGHWAY (K79) ATLANTA, GA 30341, USA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>OLEN M.KEW</td>
											<td>270 NORTH PEAK DRIVE, ALPHARETTA, GA 30022, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>CARA C.BURNS</td>
											<td>3256 WYNN DRIVE, AVONDALE ESTATES, GA 30002, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>JING SHAW</td>
											<td>2614 WILLOW COVE, DECATUR, GA 30033, USA</td>
										</tr>
										<tr>
											<td>4</td>
											<td>RAYMOND CAMPAGNOLI</td>
											<td>108 MOCKINGBIRD LANE, DECATUR, GA 30030, USA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>JACQUELINE QUAY</td>
											<td>901 CEDAR FORK TRAIL, CHAPEL HILL, NC 27514, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 5/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2005/036241</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-10-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/617,545</td>
									<td>2004-10-08</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/254018-modulation-of-replicative-fitness-by-deoptimization-of-synonymous-codons by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 12:08:36 GMT -->
</html>
