<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/254355-antisense-oligonucleotide-of-mycobacterial-peptide-deformylase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:46:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 254355:ANTISENSE OLIGONUCLEOTIDE OF MYCOBACTERIAL PEPTIDE DEFORMYLASE.</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">ANTISENSE OLIGONUCLEOTIDE OF MYCOBACTERIAL PEPTIDE DEFORMYLASE.</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to the design of the Antisense-oligonucleotide complementary to the specific region of peptide deformylase gene from Mycobacterium tuberculosis. The use of this Antisense-oligonucleotide on mycobacterial culture inhibits the production of the peptide deformylase enzyme by hybridizing within the region, which is found to be responsible for maintaining stability as well as retaining the functionality of the enzyme and thus in turn affecting the growth of the cells. This invention also establishes the essentiality of the peptide deformylase enzyme in mycobacteria and claims it as a drug target in this microorganism.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Field of the invention<br>
The present invention relates to identification of a specific region in the mycobacterial peptide deformylase enzyme useful as a potential drug target against Mycobacteria. The present invention further relates to the design of an antisense oligonucleotide complementary to the specific region of peptide deformylase gene from Mycobacterium tuberculosis. The use of this antisense oligonucleotide on mycobacterial culture inhibits the production of the peptide deformylase enzyme by hybridizing within the region, which is found to be responsible for maintaining stability as well as retaining the functionality of the enzyme and thus in turn affecting the growth of the cells. This invention also establishes the essentiality of the peptide deformylase enzyme in mycobacteria and claims it as a drug target in this microorganism.<br>
Background of the invention<br>
In the past, few decades' tuberculosis has emerged as a cause of considerable human mortality worldwide. It has been found that there is a steady increase in the frequency of mycobacterial strains, which have developed resistance against one or more anti-mycobacterial agents commonly used in treatment. Therefore, to overcome the situation there is a need to have better drug intervention strategies, which can be achieved by identification of new drug targets. In this consequence, the enzyme peptide deformylase is involved in deformylation of nascent polypeptides, which appears to be a mandatory step in mycobacterial protein synthesis in general. Therefore, any biotic/abiotic factor(s) inhibiting this enzyme may prevent protein synthesis in general in mycobacteria and thus specifically inhibits its growth.<br>
Description of the prior art<br>
Drug resistance in pathogenic microorganisms has emerged as a great threat to public health worldwide. Although there is large number of antibiotics used, the variety of target they inhibit is very limited. Consequences of the prolonged and excessive use of these antibiotics outlay multi-drug resistance in the pathogenic microorganisms.<br><br>
Therefore, in order to diversify the spectrum of antimicrobial agents, there is an urgent need to frame new intervention strategies, based on rational approaches, which would allow improved drug design.<br>
Protein synthesis has always been proven to be a rich source of targets for antimicrobials. In contrast to the eukaryotes, protein synthesis in prokaryotes is initiated with N-formyl-methionyl-tRNA leading to formylation of all nascent polypeptides at the amino-terminal end. The N-formylmethionine, however, is not retained in mature proteins of eubacteria and has been reported to be deformylated by peptide deformylase. This formylation/deformylation event appears to be a mandatory step in eubacterial protein synthesis and therefore, the importance of this enzyme has long been envisaged.<br>
Available genome sequencing data revealed the presence of putative gene encoding the peptide deformylase (def) throughout the eubacterial lineage including pathogens like Mycobacterium tuberculosis (NCBI general identification Gl: 38490165; SEQ ID NO: 8), Staphylococcus aureus, (NCBI general identification Gl: 57651784) Streptococcus pneumoniae (NCBI general identification Gl: 16272565 SEQ ID NO: 2), Haemophillus influenzae (NCBI general identification Gl: 16272565; SEQ ID NO: 3), Leptospira interrogans (NCBI general identification Gl: 14626937; SEQ ID NO: 4), Enterococcus feacelis (NCBI general identification Gl: 29377524; SEQ ID NO: 5), Helicobacter pyroli (NCBI general identification Gl: 49089809; SEQ ID NO: 6) and Bacillus subtilis (NCBI general identification Gl: 16078635; SEQ ID NO: 7). etc. Earlier studies have shown the identification and use of various compounds or preparations and their derivative inhibiting the activity of peptide deformylase in various microorganisms (Patent no: W00138561, W02005026133, W02005037272, W02005092872 efc).<br>
The article by Tomioka, H (Prospects for development of new antituberculous drugs. Kekkaku. Aug; 77[8] 573-84, 2002) in general describes the pharmacological status of certain new derivatives of existing drugs such as rifamycin (rifabutin, rifapentine, and<br>
rifalazil), fluoroquinolones (ciprofloxacin, ofloxacin, sparfloxacin, levofloxacin, gatifloxacin, sitafloxacin, moxifloxacin, and others), and new macrolides (clarithromycin, azithromycin, and roxithromycin). This review also discusses the importance of the development of new antimycobacterial, especially antituberculous agents including oxazolidinone (PNU-100480), 5'-nitroimidazole (CGI 17341), 2-pyridone (ABT-255), new riminophenazines, nitroimidazopyran (PA-824), new ketolides (ABT-773, telithromycin) and defensins (human neutrophil peptide-l). Moreover, authors have described the possibility of designing inhibitors (certainly one of the strategy could be an antisense technology) specific to mycobacterial genes encoding certain metabolic enzymes or virulence factors as a new drug targets. Infact, use of antisense oligonucleotides to shut down the expression of mycobacterial genes is a very familiar technique (For reference: Harth ef a/., Proc. Natl. Acad. Sci. U. S. A. 99, 15614-15619,2002) and is not first time mentioned in this paper.<br>
The present invention highlights the importance of Insertion sequence specifically present in mycobacterial peptide deformylase (consisting of Amino acids 74-85, (Please refer Fig. 1 and Fig. 2) responsible for maintaining the functionality of the enzyme (Fig. 5, where it is shown that deletion mutant of this region did not show any enzyme activity). Furthermore, the use of antisense oligonucleotide (complementary to the corresponding nucleic acid sequence) against the insertion region reduces the expression of peptide deformylase enzyme (as shown in Fig. 8 by western blotting using anti-mPDF antibody), which in turn leads to the growth inhibition of mycobacteria in culture (Fig. 6A and left panel of Fig. 7). These results therefore describe the novelty of the insertion region of mycobacterial enzyme, which we have invented, in terms of the possibility of designing inhibitors based on this insertion region (Antisense molecule has been used to elucidate the importance of the region in contributing mycobacterial growth).<br>
In another article by Cynamon, et al. 2004. Journal of Antimicrobial Chemotherapy. 53: 403-405 it is recited that actinonin an antibiotic isolated from class Actinomycetes as well as BB3497 (a hydroxamic acid derivative of actinonin) showed inhibition for PDF<br>
enzyme activity from different microorganisms by binding to the active site. The mentioned article describes the inhibitory effect of BB3497 on the growth of mycobacteria in culture possibly by inhibiting PDF enzyme activity. Cynamon, et al. 2004 in their paper showed a known peptide deformylase inhibitor inhibits mycobacterial growth. On the other hand, we initiated our studies through characterization of mPDF and established that despite the commonality, it is distinctly different from other bacterial homologues. Sequence analysis of peptide deformylase of M. tuberculosis revealed the presence of characteristic insertions (residues 74-85) between motifs I and II (Fig.1). The result of the instant application with deletion mutant indicates the contribution of this region towards functionality of the enzyme (Fig. 5). Among PDFs characterized to-date, our analysis revealed that the constituent amino acids of the insertion region is typical of mycobacterial species (Fig. 2). Moreover, using 5'-phosphothiorate-modified antisense oligodeoxyribonucleotides directed against this insertion region, we showed inhibition of mycobacterial growth in cultures, establishing the importance of this region (Fig. 6A). Furthermore, antisense oligonucleotide directed against insertion sequence specific to mycobacteria has no effect on the functionality of PDF enzyme from other bacteria such as Escherchia coli (as shown in right panel of Fig. 7). Thus our results clearly establish that the antisense oligonucleotide directed against the insertion region specifically inhibits the expression of the mycobacterial peptide deformylase enzyme (Fig. 8) and therefore, the growth of the mycobacteria (Fig. 6A and left panel of Fig. 7). Hence, we claim that we have identified a region in mycobacterial peptide deformylase enzyme (amino acid residues 74-85), which is important towards the functionality of the enzyme in mycobacteria. Any molecule (biotic or abiotic) that interacts with this region of the mycobacterial enzyme and affects the expression or production of this enzyme can inhibit mycobacterial growth. (We established this by using an antisense oligonucleotide directed against this region. So it is an approach to validate our conclusion/invention). Therefore, this region (amino acid residues 74-85), which we have identified in mycobacteria for the first time as well as established its importance (Figs. 5 to 8) is definitely a drug target for development of antimycobacterials.<br>
Huntington, K.M. 2000. Biochemistry. Apr 18; 39[15]: 4543-51 reports the recent information on the whole genome of various pathogenic bacteria including M. tuberculosis certainly provides a good platform to promote the progression in the identification of genes that code for new drug targets. Essential genes encoding proteins involved in metabolism and survival of pathogenic microorganisms are always being preferential vaccine candidates. Similarly, peptide deformylase is among one of the essential enzyme, which is involved in posttranslational modification of N-formylated polypeptides in prokaryotes (Mazel et al., 1994, Margolis et al., 2000 and 2001). It has been characterized as either zinc or ferrous containing metalloprotease in many eubacteria. Its essential character in bacterial cells makes it an attractive target for antibacterial drug design. Authors in the above mentioned article showed that they have rationally designed and synthesized a series of peptide thiols that act as potent, reversible inhibitors of purified recombinant peptide deformylase from Escherichia coli and Bacillus subtilis by binding to the active site. The PDF inhibitors induce bacterial cell lysis and have been tested to be bactericidal to B. subtilis, Staphylococcus epidermidis, Enterococcus faecalis, and £. coli. However, the present invention is specifically focused to M. tuberculosis. Authors have nowhere mentioned the effect of these compounds on the activity of purified mycobacterial enzyme as well as on the growth of mycobacteria. On the other hand, our work specifically deals with mycobacterial PDF and claims for the first time that an insertion sequence specific to mycobacterial enzyme could be focused to develop new antimycobacterials.<br>
Recently, we have PCR amplified the 594 base pair def gene from M. tuberculosis and following cloning in pET28c vector, expressed it as a histidine-tagged fusion protein in Escherichia coli (Saxena and Chakraborti, Biochem Biophys Res Commun (332): 418-425, (2005)). Although atomic absorption spectroscopy revealed that mPDF was a +2<br>
Fe -containing enzyme, its activity was very stable at 30°C with a half-life of ~4h. Furthermore, it maintained its distinction by exhibiting resistance to oxidizing agents, like H2<d2 and chakraborti biochem biophys res commun saxena j.bacteriol since conversion of fe to by environmental oxygen resulted in inactivation this metallo-></d2>
protease in E. coli (Rajagopalan, et. al., J. Biol. Chem 36: 13910-13918, 1997), this seems to be an important observation considering the fact that M. tuberculosis has to cope up with oxidative stress for its survival within the host as a successful pathogen.<br>
This led us to characterize the mycobacterial peptide deformylase enzyme. In contrast to other studies (Patent no. W002074903), our invention is related to use of an antisense oligonucleotide complementary to specific nucleotide region of the mycobacterial peptide deformylase gene (def), which inhibits enzyme activity, as well as the growth of this microorganism in culture establishing its essentiality and its potential as a drug target.<br>
Objects of the invention<br>
The main object of the invention is to provide the mycobacterial peptide deformylase<br>
[def] gene sequence, represented by sequence ID No. 14.<br>
Another object of the invention is thus to provide the amino acid sequence 74 to 85 of<br>
the def gene of Mycobacteria, useful as a potential drug target against Mycobacteria.<br>
Another object of the present invention is to provide an antisense oligonucleotide<br>
against Mycobacterial Peptide deformylase.<br>
Yet another object of the present invention is to provide an oligonucleotide useful for<br>
inhibiting the activity and growth of Mycobacteria.<br>
Still another object of the present invention is to provide a modified antisense<br>
oligonucleotide against Mycobacterial Peptide Deformylase.<br>
A further object of the invention is to provide a process for the preparation of said<br>
antisense oligonucleotide.<br>
Yet another objective of the invention is to provide a pharmaceutical composition<br>
useful for the treatment of tuberculosis comprising an oligonucleotide, optionally along<br>
with pharmaceutically acceptable carriers, additives or diluents.<br>
Summary of the invention<br>
Accordingly, the present invention provides an antisense oligonucleotide complementary to the mycobacterial peptide deformylase [def] gene sequence, represented by sequence ID No. 14, which correspond to 12 amino acids represented by XTXRRRGWINP, wherein X is any one of the 20 known amino acids. The present invention is further related to the use of Antisense-oligonucleotide on mycobacterial culture for inhibiting the production of the peptide deformylase enzyme by hybridizing within this region and thus in turn affecting the growth of the mycobacterial cells. The region (amino acid sequence 74 to 85) within the peptide deformylase enzyme from M. tuberculosis is found to be involved in maintaining the enzymatic stability as well as retaining the functionality of the mycobacterial enzyme and thus highlighting its importance. The prevention of growth of mycobacterial cells in culture treated with the said oligonucleotide further establishes the essentiality of the peptide deformylase enzyme in mycobacteria and therefore, claims it as a drug target in this microorganism. The invention further provides the mycobacterial peptide deformylase [def] sequence comprising 12 amino acids represented by XTXRRRGWINP, wherein X = any one of the 20 known amino acids, is 90 to 95% similar in M. tuberculosis, M. smegmatis, M. bovis, M. avium and M. leprae. The said amino acid sequence of the def gene of Mycobacteria is a potential drug target against Mycobacteria. In one embodiment of the present invention, the mycobacterial peptide deformylase [del] gene sequence is represented by sequence ID No. 14.<br>
In another embodiment of the present invention, the said sequence is useful as a potential drug target against Mycobacteria.<br>
In yet another embodiment of the present invention, the sequence comprises 12 amino acids represented by XTXRRRGWINP, wherein X = any one of the 20 known amino acids.<br>
In a further embodiment of the present invention, the amino acid sequence is 90 to 95% similar in M. tuberculosis, M. smegmatis, M. bovis, M. avium and M. leprae. In another embodiment of the present invention is an antisense oligonucleotide complementary to the gene sequence represented by sequence ID No. 14.<br>
In a further embodiment of the present invention,  the said oligonucleotide is<br>
characterized in that it is either a single (5') or throughout phosphorothioate modified<br>
oligodeoxynucleotide.<br>
In yet another embodiment of the present invention, the said oligonucleotide inhibits<br>
the production of the enzyme peptide deformylase by hybridizing within the short<br>
region of mycobacterial peptide deformylase (def) gene.<br>
In another embodiment of the present invention, the said oligonucleotide is a potential<br>
drug against Mycobacteria<br>
In yet another embodiment of the present invention is a process for the preparation of<br>
an antisense oligonucleotide, the said process comprising the steps of isolating<br>
polynucleotide sequence from M. tuberculosis comprising nucleic acid sequence (594<br>
bp) encoding a polypeptide (197 amino acids) having peptide deformylase activity<br>
wherein,  the polypeptide is present in different mycobacterial  species like  M.<br>
tuberculosis, M. smegmatis, M. bovis, M. avium, M. leprae represented by SEQ ID<br>
NO: 8, 9, 10, 11, 12 and having at least 90 to 95%, sequence similarity among<br>
themselves; identifying a region within mycobacterial peptide deformylase enzyme<br>
isolated from step (a) represented by polynucleotide sequence ID No. 14 and amino<br>
acid sequences 74 to 85 involved  in maintaining the enzymatic stability and<br>
functionality, the said region being conserved in all of the mycobacterial species;<br>
preparing an antisense oligonucleotide or its permissive modifications, against the<br>
conserved region of peptide deformylase enzyme; inhibiting the enzyme activity as<br>
well as growth of the mycobacteria using the antisense oligonucleotide.<br>
In a further embodiment of the present invention is the use of the polynucleotide<br>
sequence as a potential drug target against Mycobacteria.<br>
In yet another embodiment of the present invention is the use of the amino acid<br>
sequence of the def gene of Mycobacteria as a potential drug target against<br>
Mycobacteria.<br>
In another embodiment of the present invention is the use of the oligonucleotide for<br>
inhibiting the activity and growth of Mycobacteria.<br>
In a further embodiment of the present invention is provided a pharmaceutical<br>
composition, comprising an oligonucleotide optionally along with pharmaceutically<br>
acceptable carriers, additives or diluents, the said composition being useful for the treatment of tuberculosis.<br>
Detailed description of the invention<br>
The present invention deals with peptide deformylase from pathogenic bacterium M. tuberculosis causing dreadful disease tuberculosis. The present invention is related to the designing of the Antisense-oligonucleotide complementary to the specific region of peptide deformylase from Mycobactehum tuberculosis. The region within the peptide deformylase enzyme from M. tuberculosis is involved in maintaining the enzymatic stability as well as retaining the functionality of mycobacterial enzyme. The use of /Inf/sense-oligonucleotide on mycobacterial culture inhibits the production of the peptide deformylase enzyme by hybridizing within this region and thus in turn affecting the growth of the mycobacterial cells.<br>
In the context of this disclosure, a number of terms shall be utilized. The terms "polynucleotide sequence", "nucleic acid sequence", "nucleic acid fragment", "isolated polynucleotide sequence", "polypeptide", and "polypeptide sequence" are used interchangeably herein. These terms encompass nucleotide/amino acid sequences and the like. A polynucleotide may be a polymer of RNA or DNA, which is either single- or double-stranded. Similarly, polypeptide is a polymer of 20 different amino acids arranged in various fashions to translate for a functional protein. A polynucleotide in the form of a polymer of DNA may be comprised of a sequence of genomic DNA or synthetic DNA.<br>
As used herein "Gene" refers to a nucleic acid fragment that expresses a specific<br>
protein.<br>
A "protein" or "polypeptide" is a chain of amino acids arranged in a specific order<br>
determined by the coding sequence in a polynucleotide encoding the polypeptide.<br>
Each protein or polypeptide has a unique function.<br>
"Mature protein" or the term "mature" when used in describing a protein refers to a<br>
post-translationally processed polypeptide.<br>
As used herein, the term "region" refers to the short conserved sequences of nucleic<br>
acids or amino acids that comprise part of a longer sequence and is expected that<br>
such conserved subsequences would be important for function, and could be used to<br>
identify new targets.   It is expected that one or two of the conserved amino acids in<br>
any given conserved sequence may differ in a true homologue.<br>
As used herein, "substantially similar" refers to nucleic acid fragments wherein<br>
changes in one or more nucleotide bases results in substitution of one or more amino<br>
acids, but do not affect the functional properties of the polypeptide encoded by the<br>
nucleotide sequence.<br>
"Substantially similar" also refers to nucleic acid fragments wherein changes in one or<br>
more nucleotide bases does not affect the ability of the nucleic acid fragment to<br>
mediate alteration of gene expression by gene silencing through for example<br>
antisense technology.<br>
An Antisense oligodeoxynucleotide used in the present study, designed on the basis of specific sequence of M. tuberculosis inhibits the growth of M. smegmatis without sharing 100% sequence identity in between two sequences. Moreover, substantially similar nucleic acid fragments may also be characterized by their ability to hybridize. Estimates of such homology are provided by either DNA-DNA or DNA-RNA hybridization under conditions of stringency.<br>
Furthermore, "substantial portion" of an amino acid or nucleotide sequence comprises an amino acid or a nucleotide sequence that is sufficient to afford putative identification of the protein or gene that the amino acid or nucleotide sequence comprises.<br>
As used herein, "growth inhibition" is related in terms of difference in colony forming unit and growth curves of Antisense-oligonucleotides treated and untreated microorganism.<br>
"PCR" or "polymerase chain  reaction"  is well  known technique used for the amplification of specific DNA segments (U.S. Pat. Nos. 4,683,195 and 4,800,159).<br>
SEQ ID NO: 1 is the amino acid sequence of Staphylococcus aureus polypeptide deformylase (NCBI general identification Gl: 57651784).<br>
SEQ ID NO: 2 is the amino acid sequence of Streptococcus pneumoniae polypeptide deformylase (NCBI general identification Gl: 15858846).<br>
SEQ ID NO: 3 is the amino acid sequence of Haemophillus influenzae polypeptide deformylase (NCBI general identification Gl: 16272565).<br>
SEQ ID NO: 4 is the amino acid sequence of Leptospira interrogans polypeptide deformylase (NCBI general identification Gl: 14626937).<br>
SEQ ID NO: 5 is the amino acid sequence of Enterococcus feacelis polypeptide deformylase (NCBI general identification Gl: 29377524).<br>
SEQ ID NO: 6 is the amino acid sequence of Helicobacter pylori polypeptide deformylase (NCBI general identification Gl: 49089809).<br>
SEQ ID NO: 7 is the amino acid sequence of Bacillus subtilis polypeptide deformylase (NCBI general identification Gl: 16078635).<br>
SEQ ID NO: 8 is the amino acid sequence of Mycobacterium tuberculosis polypeptide deformylase (NCBI general identification Gl: 38490165).<br>
SEQ ID NO: 9 is the amino acid sequence of Mycobactehum smegmatis polypeptide deformylase (MSMEG0826 peptide deformylase (def) [3.5.1.88] {M. smegmatis MC2} (http://cmr.tigr.org/tigr-scripts/CMR/GenomePage.cgi?og search=&amp;ora=ams).<br>
SEQ ID NO: 10 is the amino acid sequence of Mycobacterium bovis polypeptide deformylase (NCBI general identification Gl: 31617046).<br>
SEQ ID NO: 11 is the amino acid sequence of Mycobacterium avium polypeptide deformylase (NCBI general identification Gl: 41398721).<br>
SEQ ID NO: 12 is the amino acid sequence of Mycobacterium leprae polypeptide deformylase (NCBI general identification Gl: 13093428).<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
Characterization of peptide deformylase open reading frame from Mycobacterium tuberculosis (mPDF):<br>
Genomic DMA was isolated from M. tuberculosis strain H37Ra and used for PCR amplification of mPDF gene (def). Primers (CR1: 5' CATATGGCAGTGGTACCC 3' where Nde\ site was incorporated and CR3: 5' CCATTAGTGACCGAACGGG 3') used were designed based on def (Rv0429c) sequence of published M. tuberculosis genome (Cole et al., Nature. 393 537-544 (1998)). The def open reading frame (594 bp) was PCR amplified using Expand long template PCR system (Roche, Germany) following manufacturer's recommended protocol. Following treatment with DNA polymerase I (Klenow), the PCR-amplified fragment was initially cloned in pUC19 vector (pUC-PDF; Fig. 3) and its nucleic acid sequence was determined using an automated sequencer. The construct was subsequently used for subcloning of def open reading frame at Nde\IHindIII sites of pET28c and transformed in E. coli strain<br>
DH5a (pET-PDF; Fig. 3). Clones containing gene of interest were confirmed by restriction analysis.<br>
The pET-PDF was transformed into E coli strain BL21(DE3) for over-expression. For purification of proteins, overnight culture of these colonies (-15 h at 37°C in LB broth containing 50 ug/ml of kanamycin) were re-inoculated and grown until OD600 of ~ 0.8.<br>
Cells were then induced with 0.4 mM IPTG at 25°C, harvested after 12 h and suspended in lysis buffer (20 mM phosphate buffer, pH 7.4 containing 5 mM DTT, 10 µg/ml of catalase, 1 mM phenylmethylsulfonyl fluoride, 1 ug/ml of pepstatin and 1 ug/ml of leupeptin). Cells were sonicated and the pellet fraction (-12, 000 x g for 30 min at 4°C) was resuspended in lysis buffer containing 3M urea and 2% Triton X100. Following centrifugation, supernatant fraction was dialyzed (14 h at 4°C) to remove urea and purified on Ni-NTA column (Qiagen) following manufacturer's recommended protocol. Finally, mPDF was eluted in elution buffer (20 mM phosphate buffer, pH 7.4 containing 300 mM NaCI, 250 mM imidazole and 10 ug/ml of catalase) and protein concentration was estimated following Bradford's method (Bradford M. M, Anal. Biochem. 72 248-254 1976).<br>
The mPDF protein at different stages of purification was run in 12% SDS-PAGE and its identity was confirmed by Western blotting using anti-histidine antibody (Fig. 4). The protein concentration of the purified mPDF stock was maintained at 3.5 mg/ml and stored at -80°C until used. mPDF was diluted in dilution buffer (20 mM phosphate buffer, pH 7.4 containing 1 mg/ml of BSA and 10 µg/ml of catalase) adjusting protein concentration at 3.5 ug/ml prior to use in assays. Atomic absorption spectroscopy was carried out by injecting 30 ug of mPDF sample (prepared in 20mM phosphate buffer, pH 7.4) to graphite furnace. Iron content present in mPDF was calculated using standard curve prepared using 1-10 µg of FeS04. It revealed presence of iron at the catalytic core (0.94 ± 0.21 mol of iron /mol of mPDF polypeptide, n = 4).<br>
The ability of mPDF to deformylate methionine was assessed in a spectrophotometric assay following the method described by Groche et al (Groche et al., Biochem. Biophys. Res. Commun. 246 342-346 1998) with slight modification. The assay was carried out in 50µl reaction volume mPDF protein (usually 70 ng) in 1X assay buffer (100 mM phosphate buffer, pH 7.4 containing 100 µg/ml catalase) was incubated with the substrate (0 to 80 mM of N-formyl-Met-Ala, Sigma, USA) at 30°C for 30 min. The reaction was terminated by addition of 50 µl of 4% HCI04 and further incubated (37°C for 2 h) with TNBSA reagent (0.01% in 0.1 M NaHCO3 buffer, pH 8.4). Following<br>
addition of 10% SDS (250 pi) and 1N HCI (125 µl), the highly chromogenic derivative generated due to reaction of primary amine with TNBSA was measured at 335 nm (Hermanson G, Bioconjugate techniques, Academic press, San Diego, California, 1996, pp, 112-113). The values obtained were corrected by subtracting the blank (all ingredients except mPDF enzyme) readings. Standard curves were prepared with known amounts (0-42.8 nmoles) of methionine. The determination of the catalytic parameters from three independent experiments using N-formyl-methionine-alanine as the substrate indicated that mPDF is an active enzyme with Michalis-Menton constant (Km) of 4.1 ± 0.2 mM, velocity maxima (Vmax)of 13.3 ± 0.7 umoles/min/mg protein and catalytic efficiency () of 1220 ± 6 M'V1.<br>
Mycobacterial peptide deformylase enzyme activity was highly stable and resistant to oxidizing agent like hydrogen peroxide:<br>
The enzyme activity of the recombinant protein (maintained at a concentration of 3.5 µg/ ml) IN TNBSA assay as mentioned above when monitored as the function of time, exhibited a half-life of 4.1 ± 0.7 h. Thus, despite being Fe*2 at its metal binding core, the recombinant mPDF found to be very stable compared to that of E. coli. This observation together with the fact that M. tuberculosis has to cope up with oxidative stress for its survival within the host, led us to monitor the effect of oxidizing agent, like<br>
H202 (hydrogen peroxide), on the deformylating ability of mPDF. While micromolar<br>
concentration has been reported to cause rapid and complete inactivation of E. coli enzyme (Rajagopalan and. Pei, J. Biol. Chem. 273 22305-22310 1998), we found pre-incubation (up to 2 h at 30°C with 70 ng protein/reaction) with 500 mM of H202, did not<br>
show any significant effect on the deformylating ability of mPDF compared to the untreated control. Thus, our results established that despite the commonality with other bacterial homologues, mPDF certainly maintained distinction in its behavior.<br>
Identification of an insertion region in mycobacterial peptide deformylase enzyme that is involved in maintaining enzymatic stability:<br>
Like other gram-positive bacteria (type II class), mPDF possessed insertions (amino acid residues 74-85; IR in Fig. 1), between conserved motifs I and II. We created deletion mutants of mPDF, at the insertion sequences (designated as IR where twelve amino acids "MTARRRGWINP" were deleted) employing PCR based mutagenesis approach (Shirley, K., et al., PCR Primer: A Laboratory Manual pp143-155 in C.W. Dieffenbach, G.S. Dveksler, (ed.). Cold Spring HarborLaboratory, Cold Spring Harbor, NY. 1995). This was followed by assessment of the enzyme activity to evaluate contribution of these regions on the deformylation ability of mPDF. The enzyme activity of mPDF was determined in the presence of catalase and BSA using N-formyl-Met-Ala as the substrate in TNBSA assay as mentioned above. The expressed mutant proteins (IR) were recognized by the anti-his tag antibody as evidenced by the Western blotting. On use of even excess amount of protein (20 ug incubated with 5mM of N-formyl-Met-Ala) in assays, IR mutant hardly showed any deformylase activity (Fig. 5).<br>
Antisense oligonucleotide against insertion region inhibits mycobacterial growth in culture and peptide deformylase enzyme production:<br>
Essentiality of def genes in many pathogenic bacteria led to its use as a promising drug target (Yuan et al., Drug Discov. Today 6, 954-961(2001). It has also been<br>
reported that cultures incubated with inhibitors of this enzyme affect the growth of the bacteria (Clements, et al., Antimicrob. Agents. Chemother 45, 563-570 2001 and Cynamon, et al., J. Antimicrob. Chemother 53, 403-405 2004). Since insertion sequences are crucial for maintaining the enzymatic activity of mPDF, we further examined the contribution of this region on the growth profile of Mycobacterium smegmatis strain mc2155, a fast growing saprophyte which has often been used as a model for genetic studies of M. tuberculosis (Flint, et al., Proc. Natl. Acad. Sci. U. S. A. 101, 12598-12603 2004). For this purpose, the bacterial culture was grown in the presence of 5' phosphothiorate modified antisense oligodeoxyribonucleotide, PS-ODN1, designed to span the region (bases 219-249 of M. tuberculosis def) mostly conserved in all mycobacterial species (~73% homology at the nucleotide level between defs of M. tuberculosis and M. smegmatis).<br>
Growth profile of the bacterium was monitored at different time intervals (0-24 h) by recording the absorbance at 600 nm as well as by counting colony forming units. Compared to the untreated culture, our results showed a five-fold decrease in growth of M. smegmatis at 24 h when treated with PS-ODN1 (Figs. 6A and 7, left panel). This finding was confirmed by using another antisense oligodeoxyribonucleotide (PS-ODN2) within this region (spanning bases 229-255 of M. tuberculosis def, 86% homology at the nucleotide sequences between M. tuberculosis and M. smegmatis) where all bases had phosphothiorate modification (inset of Fig. 6A). We did not observe such growth inhibition when the bacterial culture was treated with a nonspecific 5' phosphothiorate modified antisense oligodeoxyribonucleotide, PS-ODN3, designed based on non-homologous sequences (22% homology between bases 100-117 of def of M. tuberculosis and M. smegmatis). Since PS-ODN1 was mycobacteria specific (insertion sequences were absent in other bacteria), it had no effect on growth profile of E. coli (Figs. 7, right panel).<br>
To ensure that PS-ODN1 permeabilized within the M. smegmatis cells, it was conjugated with flourescein at the 3'-end (PS-ODN4) and following treatment for 24 h, when visualized in a confocal microscope, exhibited fluorescence (Fig. 6B). Thus, all<br>
these lines of evidence establish that PS-ODNs targeted against the insertion region typical of mycobacterial species, permeabilized inside the cell and specifically inhibited the growth of M. smegmatis.<br>
To determine whether PS-ODNs inhibit expression of the native PDF protein in M. smegmatis, cultures were grown either in presence or absence of PS-ODN1 for 24 h. Following pelleting of cultures, the soluble fractions of both treated and untreated cell, lysates were prepared in 20 mM phosphate buffer (pH 7.4). These samples were then subjected to SDS-PAGE (amount of protein loaded = 50 µg per slot) and Western blotting using polyclonal antibody against recombinant mPDF. Compared to the untreated control (see Ponceau S stained blot which served as a loading control, Fig. 8, upper panel), significant reduction in the level of expression of endogenous PDF protein was noticed in M. smegmatis cells treated with PS-ODN1 (Fig. 8, lower panel). Taken together our results establish that the insertion region plays a pivotal role towards the functionality of this enzyme.<br>
The following examples are given by way of illustration of the present invention and therefore should not be construed to limit the scope of the present invention.<br>
EXAMPLES Example 1<br>
Nucleotide derived amino acid sequence of mPDF was compared with 'nr' database in BLAST-P programme using mail server at NIH (Altschul et al., Nucleic. Acids. Res. 25 3389-3402 1997). The multiple sequence alignments of the retrieved sequences were carried out using the Clustal X 1.81 program (Thompson et al., Nucleic. Acids. Res. 25: 4876-4882 1997.). Analyses of amino acid sequences of all eubacterial PDFs revealed the presence of three (I: GXGXAAXQ, II: EGCLS and III: QHEXXH where X is any hydrophobic residue) highly conserved motifs (Fig. 1), despite their broad categorization in the literature as type I (gram-negative) and type II (gram-positive) classes. We compared nucleotide derived amino acid sequence of mPDF with well-<br>
characterized representatives belonging to both type I (E coli) and type II (Staphylococcus aureus) classes. Analysis of M. tuberculosis peptide deformylase sequence revealed that the mPDF possesses an insertion (amino acid residues 74-85; denoted as IR in Fig. 1). This was followed by alignment of PDF sequences of different mycobacterial species. When compared between different mycobacterial species, the insertion region of M. tuberculosis peptide deformylase exhibits ~84% identity (Fig. 2).<br>
(Figure Removed)<br>
Sequence alignment of M. tuberculosis enzyme with that of other bacterial iron-containing peptide deormylases: Nucleotide derived amino acid sequences of iron-containing peptide deformylase from E. coli, Staphylococcus aureus and M. tuberculosis were aligned using the Clustal X 1.84 programme. Asterix and dots are used to denote identical and similar amino acids respectively. Amino acids constituting insertion region (74-85) deleted to create AIR mutant are underlined.<br>
(Figure Removed)<br>
Multiple sequence alignment of peptide deformylase enzyme from different mycobacterial species: Mycobacterial sequences retrieved through PSI-BLAST were aligned by Clustal X 1.84 programme. Insertion region specific to mycobacterial deformylase containing conserved residues is underlined. Asterix and dots are used to denote identical and similar amino acids respectively.<br>
Example 2<br>
The def open reading frame (594 bp) was PCR amplified at annealing temperature of 50°C using Genomic DMA from M. tuberculosis. Primers (CR1: 5' CATATGGCAGTGGTACCC 3' where Ndel site was incorporated and CR3: 5' CCATTAGTGACCGAACGGG 3') used were designed based on def (Rv0429c) sequence of published M. tuberculosis genome (Cole et al., Nature. 393 537-544 1998). The PCR was carried out using Expand long template PCR system (Roche) following manufacturer's recommended protocol.<br>
Following treatment with DNA polymerase I (Klenow), the PCR-amplified fragment was initially cloned at Smal site of pUC19 vector (pUC-mPDF) following standard protocols (Sambrook, J. and Russel, D.. Molecular cloning : a Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold spring Harbor, New York, USA 2001) and its nucleic acid sequence was determined using an automated sequencer (Applied Biosystems). Sequencing of this fragment following cloning in pUC19 indicated 100% identity at the nucleotide level with the published def sequence of M. tuberculosis (Cole et al., Nature. 393 537-544 1998).<br>
In order to characterize the enzymatic properties of mPDF, 594 bp fragment containing the def open reading frame was excised out by restriction digestion of pUC-mPDF with A/ctel/Hinddlll restriction enzymes and ligated to the corresponding sites of pET28c following standard procedures (Sambrook, J. and Russel, D.. Molecular cloning : a Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold spring Harbor, New York, USA 2001). This resulted in a construct designated as pET-mPDF (Fig. 3), which was transformed in E. coli strain BL21(DE3) to obtain recombinant protein . SDS-PAGE analysis of the cell lysate prepared from host cells harboring plasmid pET-mPDF indicated over-expression of a ~30 kDa protein following IPTG induction (Fig. 4, left panel, compare lanes 2 and 3).<br>
The over-expressed protein was found in the pellet fraction (Fig. 4, left panel, compare lanes 4 and 5), it was solubilized with 3 M urea (Fig. 4, left panel, lane 6) and dialyzed against 20 mM phosphate buffer. Subsequent affinity purification of the soluble protein (Fig.4, left panel, lane 8) on Ni-NTA column revealed the molecular mass of 31 ±1.4 kDa (Mean ± SD, n = 7). The expressed protein had an additional 19 amino acids (HHHHHHSSGLVPRGSH) at the amino-terminal end from the vector including that of a poly histidine region (six residues). The amino-terminal extension not only facilitated the affinity purification of the protein but also provided the means of detection of mPDF protein in Western blot using anti-his tag monoclonal antibody (Fig. 4, right panel, lanes 2 and 3). This construct was designated as the wild-type in our studies.<br>
(Figure Removed)<br>
Schematic representation of cloning of mycobacterial peptide deformylase gene in expression vector: The def open reading frame (594 bp) was PCR amplified using Genomic DMA from M. tuberculosis Following treatment with DMA polymerase I (Klenow), the PCR-amplified fragment was cloned in Sma\ site of pUC19 vector (pUC-PDF) and its nucleic acid sequence was confirmed by restriction digestion followed by nucleic acid sequencing. The construct was subsequently used for subcloning of the open reading frame at Ndel/H/ndlll sites of pET28c and transformed in E. coli strain DH5a to obtain pET-PDF (WT). PCR amplified products containing mutation at the desired sites were digested with unique Sacll/Hindlll and incorporated at the corresponding site in pET-PDF construct.<br>
(Figure Removed)<br>
Fig. 4: Purification of peptide deformylase of M. tuberculosis expressed in E. coli. Overnight cultures of BL21(DE3) cells harbouring pET-PDF or deletion mutant (IR) were processed as described in the text. Protein samples at various stages of purification were subjected to 12% SDS-PAGE followed by Coomassie Brilliant Blue staining (left panel) and Western blotting using anti-histidine antibody (right panel). Left panel: Lane 1, molecular weight marker; lane 2, crude extract of cells transformed with pET-PDF; lane 3, crude extract of cells transformed with pET-PDF following IPTG induction; lane 4, low speed supernatant fraction; lane 5, pellet fraction obtained after low speed centrifugation; lane 6, urea solubilized supernatant fraction; lane 7, urea solubilized pellet fraction after low speed centrifugation; lane 8, Ni-NTA resin purified protein. Right panel: Lane 1, crude extract of cells transformed with pET-PDF; lane 2, crude extract of cells transformed with pET-PDF following IPTG induction; lane 3, Ni-NTA resin purified protein. The arrow heads indicate the position of purified mPDF. Numbers denote<br>
To establish importance of the insertion region 74 to 85, we constructed a mutant deleting amino acid residues 74-85 of M. tuberculosis peptide deformylase using PCR-based approaches. The mutant was generated using pUC-PDF as the template following PCR-based methods (Shirley, K., et al., PCR Primer: A Laboratory Manual pp143-155 in C.W. Dieffenbach, G.S. Dveksler, (ed.). Cold Spring HarborLaboratory, Cold Spring Harbor, NY. 1995). The PCR was carried out with two external (primers CR26: 5' GGAATTCCATATGGCAGTCGTACCC3' and CR27: 5'CCCAA GCTT TTAGTGACCGAACGG3') and two internal primers (CR88: 5'GCGGACCGCGCA GTG CTTGAGACCTC 3' and CR87: 5'GAGGTCTCAAGCACTGCGCGGTCCG 3' designed eliminating 36 base-pairs corresponding to amino acids residue 74-85). To generate desired mutation two sets of primary PCR reactions (using PCR primers CR27/CR87 and CR26/CR88 and pUC-PDF as the template) were carried out. The PCR amplified product obtained in primary reactions was mixed at the ratio of 1:1. Following mixing, the PCR product was used as template to carry out secondary PCR with external primer (CR26/CR27). The final PCR product containing desired mutation was purified in 0.8% agarose gel and digested with Sacll/H/ndlll and incorporated in the corresponding sites of pET-mPDF (Fig. 3). The mutant construct obtained in this way was designated as pET-AIR PDF. This (mutant construct) was expressed and purified similarly as mentioned for the wild type. This was followed by monitoring of enzyme activities of wild-type and mutant proteins.<br>
The ability of mPDF or mutant protein to deformylate methionine was assessed in a spectrophotometric assay following the method described elsewhere (Hermanson G, Bioconjugate techniques, Academic press, San Diego, California, 1996, pp, 112-113) with slight modification. Briefly, in 50ul reaction volume mPDF or mutant protein (usually 32ng- 20ug) in 1X assay buffer (100 mM phosphate buffer, pH 7.4 containing 100 µg/ml catalase) was incubated with the substrate (5 mM of N-formyl-Met-Ala, Sigma, USA) at 30°C for 30 min. The reaction was terminated by addition of 50 ul of 4% HCI04 and further incubated (37°C for 2 h) with Tri nitrobenzenesulphonic acid<br>
(TNBSA) reagent (0.01% in 0.1 M NaHC03 buffer, pH 8.4). Following addition of 10% SDS (250 ul) and 1N HCI (125 pi), the highly chromogenic derivative generated due to<br>
reaction of primary amine with TNBSA was measured at 335 nm. The values obtained were corrected by subtracting the blank (all ingredients except enzyme) readings. Standard curves were prepared with known amounts (0-42.8 nmoles) of methionine and the enzyme activity of mPDF was expressed as nmoles of free amino group produced/min/ mg protein. Finally, the data presented in the form of Mean ± SD from at least three independent experiments. The deletion of the entire insertion region (AIR mutant spanning residues 74-85) completely abolished the enzyme activity when monitored as a function of protein concentrations (Fig. 5). Thus this result indicated the importance of the insertion region towards the enzyme activity of M. tuberculosis peptide deformylase.<br>
(Figure Removed)<br>
Effect of mutations on the enzyme activity of M. tuberculosis peptide deformylase.<br>
Deletion mutant (AIR 74-85) was created using PCR based mutagenesis method as described in text. Following expression, wild type and mPDF mutant proteins were purified. The deformylation ability of mutant AIR (74-85) and wild type (WT) were compared as a function of increasing protein concentrations (0.032, 0.16, 0.8, 4.0, .20 ug) using 5 mM N-formyl-Met-Ala as substrate.<br>
Example 3<br>
We further examined the contribution of this region on the growth profile of Mycobacterium smegmatis strain mc2155, a fast growing saprophyte which has often been used as a model for genetic studies of M. tuberculosis (Flint, et al., Proc. Natl. Acad. Sci. U. S. A. 101, 12598-12603 2004). -1x105 cells of M. smegmatis (obtained from confluent culture and cell number adjusted by serial dilution) were incubated with 10uM PS-ODN1 in 3ml broth (7H9 Middlebrook media supplemented with 10% ADC). The PS-ODN1 was designed to span the region (bases 219-249 of M. tuberculosis def) mostly conserved in all mycobacterial species (-73% homology at the nucleotide level between defs of M. tuberculosis and M. smegmatis). Small aliquots were removed at different time intervals (0, 6, 12, 24 hr) and optical density at 600 nm was recorded to obtain a growth profile of bacterial cultures for treated and untreated with PS-ODN1. Simultaneously, the bacterial cells withdrawn at different time intervals were washed, plated on 7H10 Middlebrook agar (supplemented with 10% ADC) following serial dilution and enumerated for colony forming units after incubation for 3 days at 37°C. Compared to the untreated culture, our results showed a five-fold decrease (Fig. 6A and 7) in growth of M. smegmatis cultures grown in the presence of PS-ODN1 (similar growth profiles were obtained when growth monitored by determining optical density of the culture at 600 nm and by counting the number of colonies obtained on plates). This finding was confirmed by using another antisense oligodeoxyribonucleotide (PS-ODN2) within this region (spanning bases 229-255 of M. tuberculosis def, 86% homology at the nucleotide sequences between M. tuberculosis and M. smegmatis) where all bases had phosphothiorate modification (inset of Fig. 6A).<br>
Further, to ensure that PS-ODN1 permeabilized within the M.smegmatis cells, PS-ODN1 were conjugated with 3'Flourescein label and used for the treatment of mycobacterial culture (-1x105 cells of M. smegmatis were incubated with 10µM PS-ODNs in 3ml 7H9 Middlebrook broth supplemented with 10% ADC and grown at 37°C/200 rpm for 24 hrs). At the end of the experiment, following washing with 1x PBS (pH7.4) when cells treated with PS-ODN1 conjugated with 3'Flourescein were<br>
visualized in a confocal microscope exhibited fluorescence (Fig. 6B). Since PS-ODN1 was mycobacteria specific, in E. coli where this insertion sequence is absent, when culture was treated with 10µM PS-ODN1 had no effect on its growth. (Fig. 7, right panel). Thus, all these lines of evidence establish that PS-ODNs targeted against the insertion region typical of mycobacterial species, permeabilized inside the cell and specifically inhibited the growth of M. smegmatis.<br>
(Figure Removed)<br>
Effect of antisense oligonucleotides of conserved insertion region of mycobacterial peptide deformylase on growth. M. smegmatis culture (1x105 cells in 3 ml Middlebrook 7H9 medium supplemented with 10% cocktail of albumin, dextrose and catalase) were incubated with PS-ODNs (10 µM) designed against insertion region typical to mycobacterial species. Aliquots were removed at different time intervals (0, 6, 12, 24 hr) and optical density at 600 nm was recorded to obtain a growth profile of bacterial cultures treated and untreated with PS-ODNs. Inset: Mycobacterial culture when treated with a non-specific 5' phosphothiorate modified antisense oligodeoxyribonucleotide, PS-ODN3, designed based on non-homologous sequences (22% homology between bases 100-117 of def of M. tuberculosis and M. smegmatis) and grown as described above, we did not observe such growth inhibition (B) Bacterial cells treated with 3'Flourescein conjugated PS-ODN for 24 h were visualized under confocal microscope. Upper panel: untreated M. smegmatis cells, lower panel: M. smegmatis treated with 3'Flourescein conjugated PS-ODN.<br>
(Figure Removed)<br>
Different bacterial growth in response to antisense oligonucleotides of conserved insertion region of mycobacterial peptide deformylase. Bacterial cultures grown (1x105 cells in 3 ml Middlebrook 7H9 medium supplemented with 10% cocktail of albumin, dextrose and catalase for M. smegmatis and Lauria-Bertani medium for E. coli) in absence and presence of 10 µMPS-ODNs (designed against insertion region specific to mycobacterial species) removed at different time intervals were washed and plated on Middlebrook 7H10-Agar supplemented with 10% cocktail of albumin, dextrose and catalase (M. smegmatis) or Lauria-Bertani-Agar (E. coli) plates following serial dilution. Colonies obtained after incubation for 3 days (M. Smegmatis) and 12 hrs (E. coli) at 37°C were enumerated and plotted as a percent growth of untreated cultures.<br>
Example 4<br>
To determine whether PS-ODNs inhibit expression of the native PDF protein in M. smegmatis, cultures were grown either in presence or absence of PS-ODN1 for 24 h. Following pelleting of cultures, the soluble fractions of both treated and untreated cell, lysates were prepared in 20 mM phosphate buffer (pH 7.4). These samples were then subjected to SDS-PAGE (amount of protein loaded = 50 µg per slot) and Western blotting using polyclonal antibody against recombinant mPDF. Compared to the<br>
untreated control (see Ponceau S stained blot which served as a loading control, Fig. 8, upper panel), significant reduction in the level of expression of endogenous PDF protein was noticed in M. smegmatis cells treated with PS-ODN1 (Fig. 8, lower panel). Taken together our results establish that the insertion region plays a pivotal role towards the functionality of this enzyme. <br>
(Figure Removed)<br>
Fig. 8: Expression of peptide deformylase protein in response to antisense oligonucleotide treatment. . M. smegmatis culture (1x105 cells in 3 ml Middlebrook 7H9 medium supplemented with 10% cocktail of albumin, dextrose and catalase) were incubated with PS-ODNs (10 µM) designed against insertion region typical to mycobacterial species for 24 h. Cells were harvested, sonicated and supernatant fraction (13200 rpm) following protein estimation was used subsequently. Proteins resolved at 12% SDS-PAGE (loaded 50 µg protein per slot for treated or untreated samples) were subjected to Western blot analysis using polyclonal antibodies raised against recombinant purified mPDF. Upper panel: Blot probed with polyclonal antibody against mPDF, Lane 1\ Ni-NTA purified mPDF (as control), Lane 2, supernatant fraction from untreated M. smegmatis, Lane 3, supernatant fraction from M. smegmatis treated with 10uM PS-ODNs, Lane 4, Prestained protein molecular weight marker. Lower panel:<br>
Advantages:<br>
In the past few decades, tuberculosis has re-emerged as a global health hazard causing millions of deaths worldwide. Although there are several anti-tuberculosis drugs are known, the emergence of single or multidrug resistant strains of pathogenic mycobacterial species has widely been regarded as one of the prime causes for the resurgence of this dreadful disease. To overcome the situation there is an urgent need to develop novel drug intervention strategies. To achieve this objective, identification of drug target is a prime requirement. In this context, the present invention is focused on protein synthesis in mycobacteria in general, which has always been proven to be a rich source of targets for screening of antibacterial compounds. In contrast to synthesis of cytosolic proteins in eukaryotes, the formylation/deformylation event appears to be a mandatory step in eubacteria and therefore, the importance of PDF enzyme has long been envisaged. Despite commonality with different bacterial PDFs, the mycobacterial PDF has several distinctive features. Among them, the contribution of insertion (residues 74-85) sequences (specific to mycobacterial species only) in maintaining the enzymatic stability as well as functionality of this protein is the significant feature, which has not been reported to-date from any other bacteria. The phosphothiorate modified antisense oligonucleotide designed and synthesized against the insertion sequence hampered mycobacterial growth in culture as well as expression of the mycobacterial peptide deformylase enzyme. Thus, these results highlighted the novelty of the insertion region of mycobacterial enzyme based on which rational drug designing is possible. Hence, this invention will definitely be advantageous in identifying/developing of any antimycobacterial compound (biotic or abiotic) that interacts with this region of the mycobacterial enzyme as well as affects the expression or production of this enzyme can inhibit mycobacterial growth.<br><br><br><br>
We claim:<br>
1.	An antisense oligonucleotide represented by sequence ID No. 14, which is complementary to the polynucleotide sequences encoding the insertion sequence of mycobacterial deformylase peptide.<br>
2.	The antisense oligonucleotide as claimed in claim 1, wherein the mycobacterial deformylase peptide comprises 12 amino acids represented by XTXRRRGVVINP, wherein X = any one of the 20 known amino acids.<br>
3.	An antisense oligonucleotide as claimed in claim 2, wherein the amino acid sequence is 90 to 95% similar in M. tuberculosis, M. smegmatis, M. bovis, M. avium and M. leprae.<br>
4.	An antisense oligonucleotide as claimed in claim 1, wherein the said oligonucleotide is characterized in that it is either a single (5') or throughout phosphorothioate modified oligodeoxynucleotide.<br>
5.	A process for the preparation of (i) the antisense oligonucleotide as claimed in claim 1 or (ii) an antisense oligonucleotide as claimed in claim 4, the said process comprising the steps of:<br>
a.	Isolating polynucleotide sequence from M. tuberculosis comprising nucleic<br>
acid sequence (594 bp) encoding a polypeptide (197 amino acids) having<br>
peptide deformylase activity wherein, the polypeptide is present in different<br>
mycobacterial species like M. tuberculosis, M. smegmatis, M. bovis, M. avium,<br>
M. leprae represented by SEQ ID NO: 8, 9, 10, 11, 12 and having at least 90<br>
to 95%, sequence similarity among themselves;<br>
b.	Identifying  a  region  within  mycobacterial  peptide  deformylase  enzyme<br>
isolated from step (a) and amino acid sequences 74 to 85 involved in maintaining the enzymatic stability and functionality, the said region being conserved in all of the mycobacterial species as given in step (a);<br>
c.	preparing<br><br>
d.   (i) the antisense oligonucleotide as claimed in claim 1 or<br>
(ii) an antisense oligonucleotide characterized in that it is either a single (5') or throughout phosphorothioate modified oligodeoxynucleotide against the conserved region of peptide deformylase enzyme, as identified in step (b).<br>
6.	A pharmaceutical composition comprising an antisense oligonucleotide as claimed in<br>
claim 1 optionally along with pharmaceutically acceptable carriers, additives or<br>
diluents useful for the treatment of tuberculosis<br>
7.	An antisense oligonucleotide, process for its preparation and use thereof substantially as herein described with reference to the foregoing examples and drawings.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">1763-del-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1ERUwtMjAwNi1DbGFpbXMtKDE5LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-DEL-2006-Claims-(19-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDAxLTA2LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-del-2006-Correspondence Others-(01-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1ERUwtMjAwNi1Db3JyZXNwb25kZW5jZSBPdGhlcnMtKDE5LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-DEL-2006-Correspondence Others-(19-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1jb3JyZXNwb25kZW5jZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1kZXNjcmlwdGlvbiAoY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-del-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1mb3JtLTEucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1mb3JtLTIucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1Gb3JtLTMtKDAxLTA2LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-del-2006-Form-3-(01-06-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1ERUwtMjAwNi1Gb3JtLTMtKDE5LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-DEL-2006-Form-3-(19-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1mb3JtLTMucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1ERUwtMjAwNi1Gb3JtLTUtKDE5LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-DEL-2006-Form-5-(19-07-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1kZWwtMjAwNi1mb3JtLTUucGRm" target="_blank" style="word-wrap:break-word;">1763-del-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MTc2My1ERUwtMjAwNi1QZXRpdGlvbi0xMzctKDE5LTA3LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">1763-DEL-2006-Petition-137-(19-07-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="254354-a-compound-which-is-n-3-fluorophenyl-1-4-3s-3-methyl-1-piperazinyl-methyl-phenyl-acetyl-4-piperidinamine-ana-pharmaceutically-acceptable-salt-thereof.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="254356-access-control-padlock.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>254355</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>1763/DEL/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Nov-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Oct-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-Aug-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>COUNCIL OF SCIENTIFIC &amp; INDUSTRIAL RESEARCH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ANUSANDHAN BHAWAN, RAFI MARG, NEW DELHI - 110001</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>PRADIP K. CHAKRABORTI</td>
											<td>INSTITUTE OF MICROBIAL TECHNOLOGY, SECTOR 39A CHANDIGARH-160036</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RAHUL SAXENA</td>
											<td>INSTITUTE OF MICROBIAL TECHNOLOGY, SECTOR 39A CHANDIGARH-160036</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N15/11; A61K31/70</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/254355-antisense-oligonucleotide-of-mycobacterial-peptide-deformylase by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:46:18 GMT -->
</html>
