<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/254410-il-7-variants-with-reduced-immunogenicity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:42:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 254410:IL-7 VARIANTS WITH REDUCED IMMUNOGENICITY</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">IL-7 VARIANTS WITH REDUCED IMMUNOGENICITY</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A polypeptide comprising a modified human IL-7 molecule or an active portion thereof having a T-cell epitope modified to reduce an anti-IL-7 T-cell response, said polypeptide further comprising an Fc portion of an immunoglobulin molecule fused via its C-terminal to the N-terminal of said modified IL-7 molecule, wherein said fused Fc-IL7 molecule is selected from the group consisting of: (i) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L128S), wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 31; (ii) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L77D, L128S), wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D L128S) is IL-7 containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 32; (iii) huFcy2(h) - L - IL-7 (F39P, F57N, L77D, L128S), wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D, L128S) is IL-7 containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 33; and (iv) huFcy2(h) - L - IL-7 (F39P, F57N, L128S), wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 34.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>WO 2006/061219	PCT/EP2005/013145<br>
1<br>
IL-7 Variants with Reduced Immunogenicity<br>
Cross Reference to Related Applications<br>
[0001]      This application claims priority to and the benefit of U.S. Provisional<br>
Patent Application No. 60/634,470, filed on December 9, 2004, the entire<br>
disclosure of which is incorporated by reference herein.<br>
Field of the Invention<br>
[0002]      The invention relates generally to IL-7 molecules modified to reduce<br>
their immunogenicity. These molecules include also fusion proteins comprising<br>
said modified IL-7 molecules and immunoglobulin molecules or portions thereof,<br>
especially corresponding Fc fusion proteins.<br>
Background<br>
[0003]      Cytokines are stimulators of the immune system and are thus useful<br>
as drugs. For example, interferon-alpha (IFN-a), interferon-beta (IFN-fi),<br>
inierleuk/n-2 (IL-2), and granuiocyte/macrophage-colony stimulating factor (GM-<br>
CSF) are all approved drugs used to treat viral infections, cancer, immune system<br>
misregulation such as autoimmune disease, and to promote recovery of the<br>
immune system after cancer chemotherapy. Unfortunately, these proteins can<br>
stimulate an immune response against themselves, causing patients to develop<br>
antibodies against the therapeutic protein. These antibodies can also inhibit<br>
function of the same protein endogenously produced within the patient, resulting<br>
in potential long-term consequences for patient health.<br>
[0004]      lnterleukin-7 is a cytokine that promotes survival and proliferation of T-<br>
cells, B-cells, and other immune cells. It is also potentially a therapeutic protein<br>
to treat patients whose immune systems have been damaged by cancer<br>
chemotherapy, HIV infection, or other diseases, disorders, or chemical<br>
exposures. However, based on its immunostimulatory properties, therapeutically<br>
administered IL-7 is expected to induce an antibody response against itself.<br>
Therefore, there is a need in the art for improved versions of IL-7 that are less<br>
immunogenic, but that retain the property of stimulating the immune system.<br>
CONFIRMATION COPY<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
2<br>
Summary of the Invention<br>
[0005]      The present invention is directed to interleukin-7 (IL-7) which has been<br>
modified to reduce its immunogenicity in comparison to wild-type IL-7. More<br>
specifically, the IL-7 proteins of the invention are modified to remove potential T-<br>
cell epitopes. As a result, IL-7 proteins, including immunoglobulin -IL7 fusion<br>
proteins, preferably Fc-IL7 fusion proteins, of the invention have improved<br>
biological properties compared to wild-type IL-7.<br>
[0006]      Accordingly, in one aspect, the invention features a polypeptide at<br>
least 80% identical to a human IL-7 moijety or an active portion thereof,<br>
comprising an amino acid substitution at one or more residues corresponding to<br>
Gln22, Leu24, He30, Phe39, Met54, Phe57, Arg58, Ala60, Leu63, Lys68, Met69,<br>
Leu77, Ile88, Val96, Leu 104, Leu 128, Met147, Thr149, or Lys150. These amino<br>
acid modifications can be used singly or in combination to reduce an anti-IL-7 T-<br>
cell response. Thus, the invention encompasses IL-7 moieties with for example,<br>
one, at least two, at least four, or at least eight amino acid modifications at<br>
positions selected from Gin22, Leu24, He30, Phe39, Met54, Phe57, Arg58, Ala60,<br>
Leu63, Lys68, Met 69, Leu77, Ile88, Val96, Leu104, Leu128, Met147, Thr149,<br>
and Lys150. In one embodiment, the IL-7 moiety incorporates one, two, three,<br>
four, five or more of the following substitutions: Gln22Asp, Leu24Asp, IIe30Thr,<br>
Phe39Pro, Met54Ala, Phe57Lys, Phe57Asn, Arg58Asp, Ala60Ser, Arg61Glu,<br>
Leu77Asp, Leu104Ser, Leu104Vai, Leu128Ala, Leu128Val, Leu128Pro,<br>
Leu128Ser, Met147Lys, Thr149Ser, or Lys150Stop.<br>
[0007]      In one embodiment, the polypeptide contains a substitution or<br>
substitutions at one or more at Phe39, Phe57, Leu77, and Leu 128. In a further<br>
embodiment, the polypeptide has one or more of substitutions Phe39Pro,<br>
Phe57Asn, Leu77Asp, and Leu128Ser. In another embodiment, the polypeptide<br>
includes the substitutions Phe39Pro, Phe57Asn] Leu77Asp, and Leu128Ser,<br>
while in a further embodiment, the polypeptide includes the substitutions<br>
Phe39Pro, Phe57Asn, and Leu128Ser.<br>
Preferred substitutions according to this invention are at postions:<br>
Phe39, or Phe57, or Leu77, or Leu128;<br>
Phe39 and Phe57, or Phe39 and Leu77, or Phe57 and Leu77,or Plne39 and<br>
Leu128, or Phe57 and Leu128, or Leu77 and Leu128;<br>
Phe39 and Phe 57 and Leu77, or Phe39 and Phe57 and Leu128, or<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
3<br>
Phe57 and Leu77 and Leu128, or Phe39 and Leu77 and Leu128;<br>
Phe39 and Phe 57 and Leu77 and Leu128.<br>
Preferred specific substitutions according to this invention are:<br>
Phe39Pro, or Phe57Lys, or Leu77Asp, or Leu128Ser;<br>
Phe39Pro and Phe57Lys, or Phe39Pro and Leu77Asp, or Phe57Lys and<br>
Leu77Asp, or Phe39Pro and Leu128Ser, or Phe57Lys and Leu128Ser, or<br>
Leu77Asp and Leu128Ser;<br>
Phe39Pr and Phe 57Lys and Leu77Asp, or Phe39Pro and Phe57Lys and<br>
Leu128Ser, or<br>
Phe57Lys and Leu77Asp and Leu128Ser, or Phe39Pro and Leu77Asp and<br>
Leu128Ser;<br>
Phe39Pro and Phe 57Lys and Leu77Asp and Leu128Ser.<br>
[0008]      In certain embodiments of the invention, the polypeptide with at least<br>
80% identity with a human IL-7 moiety further comprises an immunoglobulin (Ig)<br>
moiety, such as a human Ig moiety, In one embodiment, the Ig moiety is lgG2.<br>
In some embodiments, the Ig moiety is an Fc portion. The invention also relates<br>
to a cell comprising a nucleic acid sequence encoding a polypeptide modified<br>
according to the invention. In one embodiment, the cell is a prokaryotic cell.<br>
[0009]      In a further embodiment, the polypeptide has at least 90% identity to a<br>
human IL-7 moiety or an active portion thereof, while in another embodiment, the<br>
polypeptide has at least 95% identity to a human IL-7 moiety or an active portion<br>
thereof.<br>
[0010]      The invention also features a method of treating a patient comprising<br>
administering a therapeutically effective amount of a polypeptide of the invention<br>
to, for example, a patient diagnosed with cancer or HIV. In one embodiment, the<br>
invention provides for administration of between about 0.01 and about 10<br>
mg/kg/day or between 0.01 and 10.00 mg/kg/day of a polypeptide of the<br>
invention.<br>
Brief Description of the Drawings<br>
[0011]      Figure 1 depicts the amino acid sequence for human IL-7. The signal<br>
sequence is shown in bold. Also depicted in bold and italics is a stretch of<br>
eighteen amino acids which can be deleted from the IL-7 sequence.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
4<br>
[0012]      Figure 2 depicts the amino acid sequence for cow IL-7. The signal<br>
sequence is shown in bold.<br>
[0013]      Figure 3 depicts the amino acid sequence for sheep IL-7. The signal<br>
sequence is shown in bold.<br>
[0014]      Figure 4 depicts the amino acid sequence of an exemplary<br>
deimmunized human IL-7 wherein T cell epitope sequences have been modified.<br>
[0015]      Figure 5 depicts the amino acid sequence of a bacterially produced<br>
deimmunized human IL-7.<br>
[0016]      Figure 6 depicts the nucleic acid sequence encoding mature IL-7<br>
incorporating codons for the mutations F39P, F57N, and L128S.<br>
[0017]      Figure 7 depicts the amino acid sequence of mature IL-7 with the<br>
mutations F39P, F57N and L128S.<br>
[0018]      Figure 8 depicts the nucleic acid sequence encoding mature IL-7<br>
incorporating codons for the mutations F39P, F57N, L77D, and L128S.<br>
[0019]      Figure 9 depicts the amino acid sequence of mature IL-7 with the<br>
mutations F39P, F57N, L77D and L128S.<br>
[0020]      Figure 10 depicts the nucleic acid sequence encoding bacterially<br>
produced deimmunized IL-7 (bDel-IL-7), codon-optimized for £. co//with codons<br>
for the amino acid mutations K68D, M69D, I88T, V96G.<br>
[0021]      Figure 11 depicts the nucleic acid sequence encoding a mature IL-7<br>
variant, deimmunized IL-7 (De!-lL-7), with codons for the amino acid mutations<br>
K68D, M69D, I88T, V96G.<br>
[0022]      Figure 12 depicts the amino acid sequence of FCK1-IL-7, where the Fc<br>
portion consists of a K1 hinge, y\ CH2, and y\ CH3 region.<br>
[0023]      Figure 13 depicts the amino acid sequence of human<br>
Fcy2(h)(FN&gt;AQ)-IL-7, which is an Fc portion with a K1 hinge, a y2 CH2 domain,<br>
and a y2 CH3 domain. The Fc portion incorporates the amino acid mutations<br>
F296A and N297Q.<br>
[0024]      Figure 14 depicts the amino acid sequence of human FC)/1-(linker1)-<br>
IL-7, which is an Fc portion consisting of a y1 hinge, CH1, and CH2 region<br>
connected to an IL-7 moiety via a poiypeptide linker of amino acid sequence<br>
GGGGSGGGGSGGGGS.<br>
[0025]      Figure 15 depicts the amino acid sequence of human Fcy1 (YN&gt;AQ)-<br>
(iinker 2)-lL-7, which Is a yi Fc portion with a y\ hinge, CH1, and CH2 region,<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
5<br>
incorporating the mutations Y296A and N297Q, connected to an IL-7 moiety via a<br>
polypeptide linker of amino acid sequence GGGGSGGGG.<br>
[0026]      Figure 16 depicts the amino acid sequence of human Fcj/1 (YN&gt;AQ,d)-<br>
(linker 2)-IL-7, which is a K1 FC portion with a y\ hinge, CH1, and CH2 domains,<br>
incorporating the mutations Y296A and N297Q, as well deletion of the C-terminal<br>
lysine and preceding glycine of the Fc moiety. The Fc portion is connected to an<br>
IL-7 moiety via a polypeptide linker of amino acid sequence GGGGSGGGG.<br>
[0027]      Figure 17 depicts the nucleic acid sequence of Fcy1, an Fc portion<br>
with a hinge, CH1 domain and CH2 domain, all of lgG1.<br>
[0028]      Figure 18 depicts the nucleic acid sequence of Fcp1(YN&gt;AQ), which is<br>
an Fc portion with a hinge, CH1 domain and CH2 domain, all of lgG1. The Fc<br>
portion incorporates the mutations Tyr296Ala and Asn297Gln.<br>
[0029]      Figure 19 depicts the nucleic acid sequence of Fcy2(h), which is an Fc<br>
portion with an lgG1 hinge and lgG2 CH2 and CH3 domains.<br>
[0030]       Figure 20 depicts the nucleic acid sequence of FcK2(h){FN&gt;AQ),<br>
which is an Fc portion with an lgG1 hinge and lgG2 CH2 and CH3 domains. The<br>
Fc portion incorporates the mutations F296A and N297Q.<br>
[0031]      Figure 21 depicts the amino acid sequence of mature human<br>
deimmunized IL-7.1, wherein the IL-7 incorporates the substitutions L24D, M54A,<br>
F57K, A60S, R61E, M147K, T149S, and deletes residues K150, E151, and H152.<br>
[0032]      Figure 22 depicts the nucleic acid sequence encoding the amino acid<br>
sequence of Figure 21.<br>
[0033]      Figure 23 depicts the amino acid sequence of mature human<br>
deimmunized IL-7.2 which incorporates the substitutions D76N, L77D, T87Q,<br>
i88T, V96G, L119S, L128V, M147K, T149S, and deletes K150, E151, and H152.<br>
[0034]      Figure 24 depicts the nucleic acid sequence encoding the amino acid<br>
sequence of Figure 23.<br>
[0035]      Figure 25 depicts the amino acid sequence of mature human<br>
deimmunized IL-7.3 which incorporates the substitutions L24D, I30T, F39P,<br>
M54A, F57K, A60S, R61E, M68D, N69D, L77D, T87Q, I88T, V96G, L119S,<br>
L128A, M147K, T149S, and deletes K150, E151, and H152.<br>
[0036]      Figure 26 depicts the nucleic acid sequence encoding the amino acid<br>
sequence of Figure 25.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
6<br>
[0037]      Figure 27 depicts the nucleic acid sequence encoding the linker<br>
sequence GGGGSGGGG followed by mature human IL-7 containing the amino<br>
acid substitution F39P, F57N, and L128S (PNS), and which contains flanking<br>
restriction sites Xma I and Xho I at the 51 and 3! ends respectively.<br>
[0038]      Figure 28 depicts the nucleic acid sequence of mature<br>
huFcy2(h)(FN&gt;AQ)(linker2)-IL-7(F39P, F57N, L77D, L128S) which is a human Fc<br>
portion with an lgG1 hinge, and lgG2 CH2 and CH3 domains incorporating the<br>
mutations F296A and N297Q connected to the N-terminus of a human IL-7<br>
moiety incorporating the mutations F39P, F57N, L77D, and L128S. The Fc<br>
portion and the IL-7 moiety are connected by a linker sequence GGGGSGGGG.<br>
[0039]       Figure 29 depicts the nucleic acid sequence of mature huFcK2(h)-<br>
(Iinker2)-IL-7(F39P, F57N, L77D, L128S) which is a human Fc portion with an<br>
lgG1 hinge, and lgG2 CH2 and CH3 domains connected to the N-terminus of a<br>
human IL-7 moiety incorporating the mutations F39P, F57N, L77D and L128S.<br>
The Fc portion and the IL-7 moiety are connected by a linker sequence<br>
GGGGSGGGG.<br>
[0040]      Figure 30 depicts the nucleic acid sequence of mature huFcy2(h)-<br>
(Iinker2)-IL-7(F39P, F57N, L128S) which is a human Fc portion with an lgG1<br>
hinge, and lgG2 CH2 and CH3 domains connected to the N-terminus of a human<br>
IL-7 moiety incorporating the mutations F39P, F57N, and L128S. The Fc portion<br>
and the IL-7 moiety are connected by a linker sequence GGGGSGGGG.<br>
[0041]      Figure 31 depicts the amino acid sequence of mature<br>
huFcy2(h)(FN&gt;AQ)-(linker2)-IL-7(F39P, F57N, L128S), which is a human Fc<br>
portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2. The Fc portion<br>
contains the mutations F296A and N297Q. The Fc portion is linked to the IL-7<br>
moiety via a linker of sequence GGGGSGGGG. The IL-7 moiety contains the<br>
mutations F39P, F57N, and L128S.<br>
[0042]      Figure 32 depicts the amino acid sequence of mature<br>
huFcy2(h)(FN&gt;AQMIinker2)-IL~7(F39P, F57N, L77D, L128S), which is a human<br>
Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2. The Fc<br>
portion contains the mutations F296A and N297Q. The Fc portion is linked to the<br>
IL-7 moiety via a linker of sequence GGGGSGGGG. The IL-7 moiety contains<br>
the mutations F39P, F57N, L77D and L128S.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
7<br>
[0043]      Figure 33 depicts the amino acid sequence of mature huFcy2(h)-<br>
(Iinker2)-IL-7(F39P, F57N, L77D, L128S), which is a human Fc portion with a<br>
hinge of lgG1 and CH2 and CH3 domains of lgG2. The Fc portion is linked to the<br>
IL-7 moiety via a linker of sequence GGGGSGGGG. The IL-7 moiety contains<br>
the mutations F39P, F57N, L77D and L128S.<br>
[0044]      Figure 34 depicts the amino acid sequence of mature huFcy2(h)<br>
(Iinker2)-IL-7(F39P, F57N, L128S), which is a human Fc portion with a hinge of<br>
igG1 and CH2 and CH3 domains of lgG2. The Fc portion is linked to the IL-7<br>
moiety via a linker of sequence GGGGSGGGG. The IL-7 moiety contains the<br>
mutations F39P, F57N, L77D and L128S.<br>
[0045]      Figure 35 is an amino acid sequence alignment of IL-7 proteins from<br>
human, chimpanzee, baboon, macaque, bovine, pig, sheep, rat, and murine<br>
sources.<br>
[0046]      Figure 36 depicts Fc-IL-7 plasma concentrations in μg/ml for both test<br>
mice and control mice administered Fc-IL-7 subcutaneously.<br>
[0047]      Figure 37 shows values for the average fold change in plasma Fc-IL-7<br>
concentrations between day 0 and day 2, and between day 2 and 4 in test mice<br>
administered Fc-IL-7 subcutaneously (SC).<br>
[0048]      Figure 38 depicts the average organ weights of organs taken from test<br>
mice sacrificed on day 7 compared to the average organ weights of mice in the<br>
control group.<br>
[0049]      Figure 39 depicts a comparison of the frequency of granulocyte Gr-1 +<br>
cells in cells/μL in the peripheral blood of mice from the control group, 0.5 mg/kg<br>
dosage group, 5.0 mg/kg dosage group, and 25 mg/kg dosage group on day 7.<br>
[0050]      Figure 40 depicts a comparison of the frequency of CD19+ cells in<br>
cellslμL in the peripheral blood of mice from the control group, 0.5 mg/kg dosage<br>
group, 5.0 mg/kg dosage group, and 25 mg/kg dosage group on day 7.<br>
[0051]      Figure 41 depicts a comparison of the frequency of CD4+ cells in<br>
cells/μL in the peripheral blood of test from the control group, 0.5 mg/kg dosage<br>
group, 5.0 mg/kg dosage group, and 25 mg/kg dosage group on day 7.<br>
[0052]      Figure 42 depicts a comparison of the frequency of CD8+ cells in<br>
cells/μL in the peripheral blood of test from the control group, 0.5 mg/kg dosage<br>
group, 5.0 mg/kg dosage group, and 25 mg/kg dosage group on day 7.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
8<br>
[0053] Figure 43 depicts the activity of Fc-lL-7 as compared to wild type IL-7<br>
based on incorporation of tritiated thymidine in counts per minute versus IL-7/Fc-<br>
IL-7 concentration in a standard cell proliferation assay.<br>
Detailed Description of the Invention<br>
[0054]      The invention is directed to IL-7 proteins that have reduced<br>
immunogenicity as compared to wild-type IL-7, as well as methods for making<br>
and using such proteins. More specifically, the invention provides mutations<br>
within IL-7 moieties that have the effect of reducing the immunogenicity of IL-7<br>
itself, primarily by removing T-cell epitopes within IL-7 that may stimulate to an<br>
immune response. The invention also encompasses fusion proteins<br>
incorporating IL-7 moieties modified according to the teachings of the invention.<br>
[0055]      T-cell epitopes can be identified by a variety of computer and non-<br>
computer methods, including predictions based on structure-based computer<br>
modeling or by synthesis of peptides and testing for binding to specific MHC<br>
Class II molecules or in an immunogenicity assay. According to the invention, a<br>
potential T-cell epitope is a sequence that, when considered as an isolated<br>
peptide, is predicted to bind to an MHC Class II molecule or an equivalent in a<br>
non-human species. A potential T-cell epitope is defined without consideration of<br>
other aspects of antigen processing, such as the efficiency of protein uptake into<br>
antigen-presenting cells, the efficiency of cleavage at sites in an intact protein to<br>
yield a peptide that can bind to MHC Class II, and so on. Thus, the set of T-cell<br>
epitopes that are actually presented on MHC Class II after administration of a<br>
protein to an animal is a subset of the potential T-cell epitopes. According to the<br>
invention, a T-cell epitope is an epitope on a protein that interacts with an MHC<br>
class II molecule. Without wishing to be bound by theory, it is understood that a<br>
T-cell epitope is an amino acid sequence in a protein that failed to undergo the<br>
negative T-cell selection process during T-cell development and therefore will be<br>
expected to be presented by an MHC Class II molecule and recognized by a T-<br>
cell receptor.<br>
[0056]      B-cell epitopes are also identified by a variety of computer and non-<br>
computer methods, including predictions based on structure-based computer<br>
modeling or by synthesis of peptides and testing for binding to specific B-cell<br>
antigen receptor molecules or in an immunogenicity assay. According to the<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
9<br>
invention, a potential B-cell epitope is a sequence that, when considered as an<br>
isolated peptide, is predicted to bind to a B-celi antigen receptor or an equivalent<br>
in a non-human species. A B-cell epitope is an epitope that does bind or is<br>
recognized by a B-cell antigen receptor and is a subset of potential B-ce!l<br>
epitopes.<br>
[0057]      The invention provides methods related to reducing the<br>
immunogenicity of IL-7. According to one embodiment of the invention, potential<br>
non-self T-cell epitopes are identified in sequences of IL-7. For example,<br>
potential non-self T-cell epitopes are identified by computational methods based<br>
on modeling peptide binding to MHC Class II molecules. Substitutions are then<br>
made such that the ability of peptides containing potential T-cell epitopes to bind<br>
to MHC Class II is reduced or eliminated. This process of identifying and<br>
modifying peptides which bind to MHC Class II is termed "de-immunization" and<br>
the resultant modified protein molecules are termed "de-immunized."<br>
[0058]      According to the invention, MHC Class II binding can be removed in<br>
situations where a protein is to be produced in bacteria or in an organism that<br>
does not generate a mammalian glycosylation pattern, such as yeast or insect<br>
cells.<br>
[0059]      The invention provides non-computer methods for reducing or<br>
eliminating the number of T-cell epitopes in IL-7 without requiring elaborate<br>
computer simulations or protein three-dimensional structures. In one<br>
embodiment, a method of the invention takes advantage of the fact that a core<br>
segment of nine amino acids interacts with both the MHC class II molecule as<br>
well as the T-cell receptor during antigen presentation. The most N-termina!<br>
amino acid, the "anchor" position, binds to a deep pocket within the MHC class II<br>
molecule. One of the following amino acids is typically present at the anchor<br>
position, which is important for binding to an MHC class II molecule: leucine,<br>
valine, isoleucine, methionine, phenylalanine, tyrosine and tryptophan. According<br>
to the invention, an additional 2 to 3 amino acids adjacent to the core 9 amino<br>
acids also affect the interaction with MHC molecules.<br>
[0060]      A general method of the invention includes mutating any leucines,<br>
valines, isoleucines, methionines, phenylalanines, tyrosines or tryptophans that<br>
occur in IL-7. In one embodiment, one or more of these amino acids in a<br>
candidate T-cell epitope is mutated to a threonine, an alanine or a proline,<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
10<br>
thereby retaining some of the hydrophobic nature of the amino acid that is<br>
replaced. In further embodiments of the invention, one or more of the above-<br>
mentioned amino acids is deleted from a candidate T-cell epitope or potential T-<br>
cell epitope, or replaced with an appropriate amino acid analog. According to the<br>
invention, if an amino acid is deleted to destroy a potential T-cell epitope, care<br>
should be taken not to generate a new T-cell epitope that includes amino acids<br>
near the deletion.<br>
[0061]      Thus, the invention provides nucleic acid sequences and proteins that<br>
are useful in construction of less immunogenic IL-7 proteins. Specifically, the<br>
invention provides proteins with mutations of leucines, valines, isoleucines,<br>
methionines, phenylalanines, tyrosines, or tryptophans. Any aliphatic or aromatic<br>
residue (leucine, valine, isoleucine, methionine, phenylaianine, tryptophan or<br>
tyrosine) presents a high risk of creating an MHC binding peptide with the amino<br>
acid in the first position (anchor position) that binds the pocket of the MHC<br>
molecule. Therefore, substitution of any of the above-mentioned annino acids,<br>
with an amino acid that is not one of the above-mentioned amino acids, or with<br>
alanine, praline, orthreonine, will remove a candidate T-cell epitope.<br>
[0062]      The proteins can be human proteins with sequences that generally<br>
correspond to sequences found in the human body. The invention also provides<br>
nucleic acid sequences encoding such proteins. The nucleic acid sequences for<br>
this aspect of the invention may exist as plasmids, PCR-generated fragments, or<br>
nucleic acids produced by chemicai synthesis.<br>
[0063]      As used herein, the term "interleukin-7" or "IL-7" means IL-7<br>
polypeptides and derivatives and analogs thereof having substantial amino acid<br>
sequence identity to wild-type mature mammalian IL-7. For example, IL-7 refers<br>
to an amino acid sequence of a recombinant or non-recombinant polypeptide<br>
having an amino acid sequence of: i) a native or naturally-occurring allelic variant<br>
of an IL-7 polypeptide, ii) a biologically active fragment of an IL-7 polypeptide, iii)<br>
a biologically active polypeptide analog of an IL-7 polypeptide, or iv) a biologically<br>
active variant of an IL-7 polypeptide.<br>
[0064]      IL-7 polypeptides modified according to the invention can be derived<br>
from any species, e.g., human, cow or sheep. IL-7 nucleic acid and amino acid<br>
sequences are well known in the art. For example, the human IL-7 amino acid<br>
sequence has a Genbank accession number of NM 000880 (SEQ ID NO:1) and<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
11<br>
is shown in Figure 1; the mouse IL-7 amino acid sequence has a Genbank<br>
accession number of NM 008371; the rat IL-7 amino acid sequence has a<br>
Genbank accession number of AF 367210; the cow IL-7 amino acid sequence<br>
has a Genbank accession number of NM 173924 (SEQ ID NO:2) and is shown in<br>
Figure 2; and the sheep IL-7 amino acid sequence has a Genbank accession<br>
number of U10089 (SEQ ID NO:3) and is shown in Figure 3. The signal<br>
sequence for each of the polypeptide species is shown in bold in each of the<br>
figures and is typically not included where the IL-7 portion is fused C-terminal to<br>
the carrier protein.<br>
[0065]      In addition, in Figure 35, an alignment of various mammalian IL-7<br>
sequences is shown. IL-7 from non-human primates is generally more than 90%<br>
identical to human IL-7. Although the murine IL-7 sequence is the most divergent<br>
from the human IL-7 sequence, with less than 70% identity, it is nevertheless<br>
capable or activating the human IL-7 receptor. Therefore, IL-7 moieties from a<br>
range of species are particularly useful in accordance with the teachings of the<br>
invention.<br>
[0066]      A "variant" of an IL-7 protein is defined as an IL-7 amino acid<br>
sequence that is altered by one or more amino acids as compared to wild-type IL-<br>
7. The variant can have "conservative" changes, wherein a substituted amino<br>
acid has similar structural or chemical properties, e.g., replacement of leucine<br>
with isoleucine. More rarely, a variant can have "nonconservative" changes, e.g.,<br>
replacement of a glycine with a tryptophan. Similar minor variations can also<br>
include amino acid deletions or insertions, or both.<br>
[0067]      Variant IL-7 proteins also include polypeptides that have at least about<br>
70%, 75%, 80%, 81%, 82%, 83%, 84%, 85%, 86%, 87%, 88%, 89%, 90%, 91%,<br>
92%, 93%, 94%, 95%, 96%, 97%, 98%, 99%, or more sequence identity with<br>
wild-type IL-7. To determine the percent identity of two amino acid sequences or<br>
of two nucleic acids, the sequences are aligned for optimal comparison purposes<br>
(e.g., gaps can be introduced in the sequence of a first amino acid or nucleic acid<br>
sequence for optimal alignment with a second amino acid or nucleic acid<br>
sequence). The percent identity between the two sequences is a function of the<br>
number of identical positions shared by the sequences (i.e.., % homology = (# of<br>
identical positions/total # of positions)times 100). The determination of percent<br>
homology between two sequences can be accomplished using a mathematical<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
12<br>
algorithm. A non-limiting example of a mathematical algorithm utilized for the<br>
comparison of two sequences is the algorithm of Karlin and Altschul, (1990) Proc.<br>
Natl. Acad. Sci. USA, 87:2264-68, modified as in Karlin and Altschul, (1993) Proc.<br>
Natl. Acad. Sci. USA, 90:5873-77. Such an algorithm is incorporated into the<br>
NBLAST and XBLAST programs of Altschul et a/., (1990) J. Mol. Biol., 215:403-<br>
10. BLAST nucleotide searches can be performed with the NBLAST program,<br>
score=100, wordlength=12. BLAST protein searches can be performed with the<br>
XBLAST program, score=50, wordlength=3. To obtain gapped alignments for<br>
comparison purposes, Gapped BLAST can be utilized as described in Altschul et<br>
al., (1997) Nucleic Acids Research, 25(17):3389-3402. When utilizing BLAST<br>
and Gapped BLAST programs, the default parameters of the respective programs<br>
(e.g., XBLAST and NBLAST) can be used.<br>
[0068]       Furthermore, the invention also includes IL-7 fusion proteins wherein<br>
the IL-7 moiety contains a deletion and which retain comparable activity<br>
compared to the corresponding unmodified IL-7 fusion proteins. For example, the<br>
invention provides a form of lg-IL-7 or IL-7 in which the IL-7 moiety contains an<br>
eighteen amino acid internal deletion corresponding to the sequence<br>
VKGRKPAALGEAQPTKSL (SEQ ID NO:25), of wild-type human IL-7. (See SEQ<br>
ID NO:1). In addition, the invention provides an active form of IL-7 wherein<br>
Lys150 is deleted. Glu151and His152 may also be deleted in conjunction with<br>
Lys150 while still leaving an active form of IL-7.<br>
[0069]      Throughout this application, the positions of amino acid residues in the<br>
IL-7 sequence are given with reference to the mature human IL-7 protein. For<br>
example, the cysteine in the N-terminal sequence MDCDIEGK...(SEQ ID NO:22)<br>
of bacterially produced human IL-7 protein, which includes a start methionine, is<br>
still referred to as Cys2.<br>
Modifying IL-7 Proteins<br>
[0070]      One aspect of the invention derives from the insight that IL-7 produced<br>
by bacterial expression will not contain post-translational modifications that are<br>
characteristic of eukaryotes, such as mammals. For example, IL-7 contains three<br>
predicted N-linked glycosylation sites at positions 70, 91, and 116. In an Fc-IL-7<br>
fusion protein expressed in mammalian cells, the asparagines at positions 70 and<br>
91 are glycosylated, while the asparagine at position 116 is not. It is likely that IL-<br>
7 endogenously produced in the human body is also N-glycosylated, at least at<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
13<br>
positions 70 and 91, and possibly at position 116. These N-linked glycosylations<br>
are not present in bacterially produced lL-7, and represent sequences that might<br>
be recognized by the human immune system as "non-self," i.e., not normally<br>
present in the human body. As such, the invention encompasses deimmunizing<br>
these potential epitope regions on IL-7 to reduce the immunogenicity of IL-7 and<br>
related proteins.<br>
[0071]      According to the invention, T-cell epitopes are present in IL-7 that<br>
include positions 70 and 91, as described in Table 1. The epitopes shown in<br>
Table 1 are defined in terms of a minimal 9-mer peptide, with the strong MHC<br>
Class II anchor residue in the first position.<br>
Table 1<br><br>
[0072]      According to the invention, one method for reducing the<br>
immunogenicity of bacterially produced IL-7 is to introduce one or more of the<br>
following mutations: Leu63Ala, Leu63Val, Leu63Pro, Leu63Thr, Lys68Asp,<br>
Met69Asp, Lys68Glu, Met69Glu, He88Thr, lle88Ala, He88Val, and Val96Gly.<br>
Other mutations may be introduced at positions 63, 68, 69, 88 and/or 94. Some<br>
mutations are particularly useful in combination, such as the pairs Lys68Asp<br>
coupled with Met69Asp and/or He88Thr coupled with Val96Gly.<br>
[0073]      When these mutations are introduced into IL-7 or a fusion protein<br>
comprising IL-7, the resulting mutant protein generally has enough IL-7 biological<br>
activity to be useful as a therapeutic protein. In fact, the biological activity of the<br>
IL-7 moiety is at least 10%, 20%, 50%, 70%, 80%, 90%, 95%, 99% or 100% in<br>
comparison to the biological activity of wild type IL-7. Activity of the IL-7 of the<br>
invention can be tested in an in vitro or in vivo assay. Example 9 shows an assay<br>
for testing biological activity of the IL-7 variants of the invention.<br>
[0074]      In addition, the mutations generally allow proper folding of the IL-7<br>
moiety so that a pure protein, largely free of high-molecular weight aggregates<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
14<br>
and incorrectly disulfide-bonded forms, may be isolated. However, the folding<br>
and biological activity that results from any particular combination should be<br>
tested, for example as illustrated in the Examples, to verify that the desired<br>
activity is obtained.<br>
[0075]      According to the invention, an alternative strategy for reducing the<br>
immunogenicity of bacterially produced IL-7 is to alter Asn70 and Asn91 to<br>
aspartic acid. Without wishing to be bound by theory, the mutation of Asn70 and<br>
Asn91 to aspartic acid may be useful for the following reasons.<br>
[0076]      The immunogenicity of an exogenously administered therapeutic<br>
protein is mediated, in part, through the presentation of T-cell epitopes derived<br>
from the therapeutic protein. Such presentation is thought to occur through the<br>
following mechanism. A therapeutic protein is taken up by an antigen-presenting<br>
cell (APC), such as a dendritic cell, macrophage, or B-cell by endocytosis. The<br>
protein is transported into a series of vesicles termed endosomes, including the<br>
early, middle and late endosomes. In these vesicles, the environment becomes<br>
progressively more harsh and less favorable for extracellular, disulfide-bonded<br>
proteins that may be stably folded at neutral pH. Proteases termed cathepsins<br>
degrade internalized proteins into small peptides. A proportion of these protein<br>
fragments then become bound by MHC Class I! proteins which transport the<br>
fragments to the cell surface as MHC Class ll/peptide complexes. Such<br>
complexes are recognized by T-cell receptors on CD4+ T-cells.<br>
[0077]      In the case of peptides deriving from foreign proteins, presentation of<br>
an MHC Class ll/peptide complex may stimulate an immune response. However,<br>
in the case of peptides deriving from self proteins, there are multiple mechanisms<br>
by which T-cells recognizing MHC Class ll/peptide complexes are deleted or<br>
prevented from activating an immune response.<br>
[0078]      With the preceding two paragraphs taken as background, it is<br>
important to consider how an N-glycosylated protein would be processed in the<br>
endosome. Such a protein could be degraded into N-linked oligosaccharide-<br>
containing peptides that could bind to MHC Class II molecules. According to an<br>
insight of the invention, the endosome also contains an endoglycosidase that<br>
sometimes removes the oligosaccharide from asparagine, and in doing so,<br>
converts the asparagine into aspartic acid. Thus, self protein sequences that<br>
contain asparagine-iinked oiigosaccharides may be presented by MHC Class II<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
15<br>
as peptides containing the asparagine linked to an oligosaccharide, or as<br>
corresponding peptides containing aspartic acid instead of asparagine.<br>
[0079]      As part of the invention, it is also recognized that this strategy for<br>
reducing the immunogenicity of mammalian proteins that are expressed in<br>
bacteria may be applied in a general manner. Specifically, the substitution of<br>
aspartic acid for asparagine at a site of N-linked glycosylation generally has the<br>
effect of reducing the immunogenicity of a mammalian protein that is expressed<br>
in a prokaryote.<br>
[0080]      The invention contains additional mutations that reduce the<br>
immunogenicity of IL-7 and IL-7-containing fusion proteins when expressed in<br>
either bacterial or mammalian cells. These mutations include those listed in<br>
Table 2 below. An IL-7 or IL-7 containing fusion protein may comprise one or<br>
more of these mutations. For example, in one embodiment, IL-7 is modified to<br>
incorporate one or more of L24D, M54A, F57K, A60S, R61E, M147K, and T149S,<br>
with K150, E151 and H152 being deleted. In another embodiment, IL-7 is<br>
modified to incorporate one or more of D76N, L77D, T87Q, I88T, V96G, L119S,<br>
M147K, and T149S, with K150, E151 and H152 being deleted. In a further<br>
embodiment, IL-7 can be modified to incorporate one or more of L24D, I30T,<br>
F39P, M54A, F57K, A60S, R61E, M68D, N69D, L77D, T87Q, I88T, V96G,<br>
L119S, L128A, M147K, and T149S, with K150, E151 and H152 being deleted.<br>
[0081]      In another embodiment, an IL-7 molecule or an IL-7 containing fusion<br>
protein may include mutations to one or more of residues 39, 57, 77 and/or 128 of<br>
IL-7. For example, IL-7 in one embodiment, includes a mutation at residue 39. in<br>
another embodiment, IL-7 includes a mutation at residue 57. In a further<br>
embodiment, IL-7 includes mutations at both residues 39 and 57. In yet another<br>
embodiment, IL-7 includes mutations at residues 39, 57 and 128, while in another<br>
embodiment, IL-7 includes mutations at residues 39, 57 and 77. In yet another<br>
embodiment, IL-7 includes mutations at residues 39, 57,77 and 128. In a further<br>
embodiment, the phenylalanine residue at position 39 is replaced by a proline<br>
residue (F39P). In another embodiment, the phenylalanine residue at 57 is<br>
replaced by an asparagines residue (F57N). In another embodiment, the leucine<br>
residue at position 77 is replaced by aspartic acid (L77D). In yet another<br>
embodiment, the leucine residue at position 128 is replaced by serine (L128S).<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
16<br><br>
Verification of the Reduced Immunogenicity of the Proteins of the Invention<br>
[0082]      To check that a mutation of the invention has indeed resulted in<br>
reduced immunogenicity, standard experimental tests, which are well known in<br>
the art, may be employed. For example, a T-cell stimulation assay may be used<br>
(e.g. Jones et ai, (2004), J. Interferon Cvtokine Res., 24:560). In such an assay,<br>
human peripheral blood mononuclear cells (PBMCs) are obtained and cultured<br>
according to standard conditions. After an optional pre-stimulation, a peptide<br>
corresponding to a potential MHC Class II epitope is added to the culture of<br>
PBMCs; the PBMCs are further incubated, and at a later time tritiated thymidine<br>
is added. The peptide may be a minima! 9-mer, or may have about 10 to 15 or<br>
more amino acids. After further incubation of the cells, incorporation of tritiated<br>
thymidine into DNA is then measured by standard techniques.<br>
[0083]      The T-ce!l stimulation assay is thought to work by the following<br>
mechanisms. First, if a peptide is used as a stimulator, the peptide must first bind<br>
to an MHC Class II molecule present on a cell among the PBMCs. Second, the<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
17<br>
MHC Class ll/peptide complex must interact productively with a T-celi receptor on<br>
a CD4+ T-cell. If the test peptide is unable to bind sufficiently tightly to an MHC<br>
Class II molecule, no signal will result. If the peptide is able to bind an MHC<br>
Class II molecule and there are T-cells expressing an appropriately rearranged T-<br>
cell receptor capable of recognizing a particular MHC Class ll/peptide complex, a<br>
signal should result. However, if such T-cells have been deleted as a result of a<br>
negative selection process, no signal will result. These mechanisms are<br>
considered relevant to the immunogenicity of a protein sequence, as inferred<br>
from the stimulation or lack of stimulation by a given peptide.<br>
[0084]       If recognizing T-cells are present in very low numbers in the PBMC<br>
population for stochastic reasons relating to failure of an appropriate T-cell<br>
receptor to take place or proliferation of other, unrelated T-cells followed by<br>
homeostasis of the T-cell population, there may also be no signal even though a<br>
signal is expected. Thus, false negative results may occur. Based on these<br>
considerations, it is important to use a large number of different sources of<br>
PBMCs and to test these samples independently. It is also generally useful to<br>
test PBMCs from an ethnically diverse set of humans, and to determine the MHC<br>
Class II alleles present in each PBMC population.<br>
[0085]      The standard T-cell assay has the disadvantage that the tritium<br>
incorporation signal is often only two-fold greater than the background<br>
incorporation. The proteins and peptides of the invention may also be tested in a<br>
modified T-cell assay in which, for example, purified CD4+ T-cells and purified<br>
dendritic cells are co-cultured in the presence of the test peptide, followed by<br>
exposure to tritiated thymidine and then assayed for tritiated thymidine<br>
incorporation. This second assay has the advantage that tritiated thymidine<br>
incorporation into irrelevant cells, such as CD8+ T-cells, is essentially eliminated<br>
and background is thus reduced.<br>
[0086]      A third assay involves the testing of a candidate protein with reduced<br>
immunogenicity in an animal such as a primate. Such an assay would generally<br>
involve the testing of an entire IL-7 protein or IL-7-containing fusion protein in<br>
which the IL-7 moiety had been designed by testing individual component<br>
peptides for potential immunogenicity in a cell-based assay such as one<br>
described above. Once such a candidate IL-7-containing protein is designed and<br>
expressed, the protein is tested for immunogenicity by injection into an animal.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
18<br>
[0087] Injection of the modified IL-7-containing protein is generally performed<br>
in the same manner as the anticipated route of delivery during therapeutic use in<br>
humans. For example, intradermal, subcutaneous, intramuscular, intraperitoneal<br>
injection or intravenous infusion may be used. If more than one administration is<br>
used, the administrations may be by different routes.<br>
[0088]      For immunogenicity testing purposes, it may be useful to coadminister<br>
an adjuvant to increase the signal and minimize the number of animals that need<br>
to be used. If an adjuvant is used, it is possible to use an adjuvant lacking a<br>
protein component, such as non-coding DNA with unmethylated CpG<br>
dinucleotides, bacterial lipid A, N-formyl methionine, or other bacterial non-protein<br>
components. Without wishing to be bound by theory, the rationale for avoiding<br>
protein-containing adjuvants is that other proteins may provide T-cell epitopes<br>
that will ultimately contribute to an antibody response against the candidate<br>
protein.<br>
[0089]      After one or more administrations of the candidate IL-7-containing<br>
protein, the presence of anti-IL-7 antibodies is tested according to standard<br>
techniques, such as the ELISA method. It is found that the altered IL-7-<br>
containing molecules of the invention induce antibody formation less frequently,<br>
and to a lesser extent, than corresponding molecules containing normal human<br>
IL-7.<br>
[0090]      Many of the proteins of the invention alter surface residues of IL-7. It<br>
is contemplated that the proteins of the invention, while being less immunogenic<br>
than corresponding proteins containing human IL-7, may still occasionally induce<br>
formation of antibodies. Because the B-cell epitopes of the proteins of the<br>
invention are generally different from those of unmodified IL-7, antibodies to the<br>
proteins of the invention will generally not cross-react with endogenous IL-7, and<br>
formation of antibodies to the proteins of the invention will have no long-term<br>
consequences for the health of the patient.<br>
Fc-IL-7 Fusion Proteins<br>
[0091]      A key aspect of the invention is that IL-7 modified according to the<br>
invention may be fused to a carrier protein to create a fusion protein. In one<br>
embodiment, the carrier protein is disposed towards the N-terminus of the fusion<br>
protein and the IL-7 is disposed towards the C-terminus. In another embodiment,<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
19<br>
the IL-7 is disposed towards the N-terminus of the fusion protein and the carrier<br>
protein is disposed towards the C-terminus.<br>
[0092]      The carrier protein can be any polypeptide covalently fused to the IL-7<br>
protein. In one embodiment, the carrier protein is albumin, for example, human<br>
serum albumin. The albumin moiety may be fused to the C-terminal or N-terminal<br>
end of the IL-7 moiety. In another embodiment, the carrier protein is an<br>
immunoglobulin (Ig) moiety, such as an ig heavy chain. The Ig chain may be<br>
derived from IgA, IgD, IgE, IgG, or IgM. According to the invention, the Ig moiety<br>
may be an intact antibody and may direct the IL-7 fusion protein to specific target<br>
sites in the body. Fusion proteins making use of antibody targeting are known to<br>
those in the art.<br>
[0093]      In one embodiment, the Ig moiety comprises an Fc region. As used<br>
herein, "Fc portion" encompasses domains derived from the constant region of an<br>
immunoglobulin, such as a human immunoglobulin, including a fragment, analog,<br>
variant, mutant or derivative of the constant region. Suitable immunoglobulins<br>
include lgG1, lgG2, lgG3, lgG4, and other classes. The constant region of an<br>
immunoglobulin is defined as a naturally-occurring or synthetically-produced<br>
polypeptide homologous to the immunoglobulin C-terminal region, and can<br>
include a hinge, a CH2 domain, a CH3 domain, or a CH4 domain, separately or in<br>
any combination. In the present invention, the Fc portion typically includes at<br>
least a CH2 domain. For example, the Fc portion can include hinge-CH2-CH3.<br>
Alternatively, the Fc portion can include all or a portion of the hinge region, the<br>
CH2 domain and/or the CH3 domain. Methods for making Fc-IL-7 fusion proteins<br>
are disclosed in U.S. Provisional Patent Application No. 60/533,406.<br>
[0094]      The constant region of an immunoglobulin is responsible for many<br>
important antibody functions including Fc receptor (FcR) binding and complement<br>
fixation. There are five major classes of heavy chain constant region, classified<br>
as IgA, IgG, IgD, IgE, and IgM. For example, IgG is separated into four y<br>
subclasses: y 1, y2, y3, and yA, also known as IgGl, lgG2, lgG3, and lgG4,<br>
respectively.<br>
[0095]      IgG molecules interact with multiple classes of cellular receptors,<br>
including three classes of FCK receptors (Fc y R) specific for the IgG class of<br>
antibody, namely Fcj/RI, FcyRII, and Fc^RIII. The important sequences for the<br>
binding of IgG to the FcyR receptors have been reported to be located in the CH2<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
20<br>
and CH3 domains. The serum half-life of an antibody is influenced by the ability<br>
of that antibody to bind to an Fc receptor (FcR). Similarly, the serum half-life of<br>
immunoglobulin fusion proteins is also influenced by the ability to bind to such<br>
receptors (Gillies et a/., (1999) Cancer Res. 59:2159-66). Compared to those of<br>
lgG1, CH2 and CH3 domains of lgG2 and lgG4 have biochemically undetectable<br>
or reduced binding affinity to Fc receptors. It has been reported that<br>
immunoglobulin fusion proteins containing CH2 and CH3 domains of lgG2 or<br>
lgG4 had longer serum half-lives compared to the corresponding fusion proteins<br>
containing CH2 and CH3 domains of lgG1 (U.S. Patent No. 5,541,087; Lo et ai,<br>
(1998) Protein Engineering. 11:495-500). Accordingly, in certain embodiments of<br>
the invention, CH2 and CH3 domains are derived from an antibody isotype with<br>
reduced receptor binding affinity and effector functions, such as, for example,<br>
lgG2 or lgG4.<br>
[0096]      The hinge region is normally located C-terminal to the CH1 domain of<br>
the heavy chain constant region. In the IgG isotypes, disulfide bonds typically<br>
occur within this hinge region, permitting the final tetrameric molecule to form.<br>
This region is dominated by pralines, serines and threonines. When included in<br>
the present invention, the hinge region is typically at least homologous to the<br>
naturally-occurring immunoglobulin region that includes the cysteine residues to<br>
form disulfide bonds linking the two Fc moieties. Representative sequences of<br>
hinge regions for human and mouse immunoglobulins are known in the art and<br>
can be found in Borrebaeck, C. A. K., ed., (1992) Antibody Engineering, A<br>
Practical Guide, W. H. Freeman and Co. Suitable hinge regions for the present<br>
invention can be derived from lgG1, lgG2, lgG3, lgG4, and other immunoglobulin<br>
classes.<br>
[0097]      The lgG1 hinge region has three cysteines, two of which are involved<br>
in disulfide bonds between the two heavy chains of the immunoglobulin. These<br>
same cysteines permit efficient and consistent disulfide bonding formation of an<br>
Fc portion. Therefore, a hinge region of the present invention in one embodiment<br>
is derived from lgG1, such as human lgG1. When the lgG1 hinge is used, the<br>
first cysteine can be mutated to another amino acid, such as serine.<br>
[0098]      The lgG2 isotype hinge region has four disulfide bonds that tend to<br>
promote oligomerization and possibly incorrect disulfide bonding during secretion<br>
in recombinant systems. A suitable hinge region can be derived from an lgG2<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
21<br>
hinge. In one embodiment, the first two cysteines of the lgG2 hinge are mutated<br>
to another amino acid.<br>
[0099]      The hinge region of lgG4 is known to form interchain disulfide bonds<br>
inefficiently. However, a suitable hinge region for the present invention can be<br>
derived from the lgG4 hinge region, and can contain a mutation that enhances<br>
correct formation of disulfide bonds between heavy chain-derived moieties (Angal<br>
et a/., (1993) Mol. Immunol., 30:105-8).<br>
[00100]    In accordance with the present invention, the Fc portion can contain<br>
CH2 and/or CH3 and/or CH4 domains and a hinge region that are derived from<br>
different antibody isotypes, i.e., a hybrid Fc portion. For example, in one<br>
embodiment, the Fc portion contains CH2 and/or CH3 domains derived from<br>
lgG2 or lgG4 and a mutant hinge region derived from lgG1. Alternatively, a<br>
mutant hinge region from another igG subclass is used in a hybrid Fc portion.<br>
For example, a mutant form of the lgG4 hinge that allows efficient disulfide<br>
bonding between the two heavy chains can be used. A mutant hinge can also be<br>
derived from an lgG2 hinge in which the first two cysteines are each mutated to<br>
another amino acid. Such hybrid Fc portions facilitate high-level expression and<br>
improve the correct assembly of the Fc-IL-7 fusion proteins. Assembly of such<br>
hybrid Fc portions is known in the art and has been described in U.S. Published<br>
Patent Application No. 2003-0044423.<br>
[00101]    In some embodiments, the Fc portion contains amino acid<br>
modifications that generally extend the serum half-life of an Fc fusion protein.<br>
Such amino acid modifications include mutations substantially decreasing or<br>
eliminating Fc receptor binding or complement fixing activity. For example, the<br>
glycosylation site within the Fc portion of an immunoglobulin heavy chain can be<br>
removed. In lgG1, the glycosylation site is Asn297 within the amino acid<br>
sequence GIn-Tyr-Asn-Ser (SEQ ID NO:26). In other immunoglobulin isotypes,<br>
the glycosylation site corresponds to Asn297 of lgG1. For example, in lgG2 and<br>
lgG4, the glycosylation site is the asparagine within the amino acid sequence<br>
Gln-Phe-Asn-Ser (SEQ ID NO:28). Accordingly, a mutation of Asn297 of lgG1<br>
removes the glycosylation site in an Fc portion derived from lgG1. In one<br>
embodiment, Asn297 is replaced with Gin. In other embodiments, the tyrosine<br>
within the amino acid sequence GIn-Tyr-Asn-Ser (SEQ ID NO:26) is further<br>
mutated to eliminate a potential non-self T-cell epitope resulting from asparagine<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
22<br>
mutation. For example, the amino acid sequence Gln-Tyr-Asn-Ser (SEQ ID<br>
NO:26) within an lgG1 heavy chain can be replaced with a Gln-Ala-Gln-Ser (SEQ<br>
ID NO:27) amino acid sequence.<br>
[00102]    Similarly, in lgG2 or lgG4, a mutation of asparagine within the amino<br>
acid sequence Gln-Phe-Asn-Ser (SEQ ID NO:28) removes the glycosylation site<br>
in an Fc portion derived from lgG2 or lgG4 heavy chain. In one embodiment, the<br>
asparagine is replaced with a glutamine. In other embodiments, the<br>
phenylalanine within the amino acid sequence Gln-Phe-Asn-Ser (SEQ ID NO:28)<br>
is further mutated to eliminate a potential non-self T-cell epitope resulting from<br>
asparagine mutation. For example, the amino acid sequence Gln-Phe-Asn-Ser<br>
(SEQ ID NO:28) within an lgG2 or lgG4 heavy chain can be replaced with a Gln-<br>
Ala-Gln-Ser (SEQ ID NO:27) amino acid sequence. Other mutations that are<br>
useful in reducing Fc receptor binding are disclosed in U.S. Patent Application<br>
No. 09/256,156.<br>
[00103]     It has also been observed that alteration of amino acids near the<br>
junction of the Fc portion and the non-Fc portion can dramatically increase the<br>
serum half-life of the Fc fusion protein. (U.S. Published Patent Application No.<br>
2002-0147311). Accordingly, the junction region of an Fc-IL-7 or IL-7-Fc fusion<br>
protein of the present invention can contain alterations that, relative to the<br>
naturally-occurring sequences of an immunoglobulin heavy chain and IL-7, lie<br>
within about 10 amino acids of the junction point. These amino acid changes can<br>
cause an increase in hydrophobicity by, for example, changing the C-terminal<br>
lysine of the Fc portion to a hydrophobic amino acid such as alanine or leucine.<br>
(See e.g. SEQ ID NO:34). In yet another embodiment of the invention, the C-<br>
terminal lysine and preceding glycine of the Fc portion is deleted. (See e.g. SEQ<br>
IDNO:35).<br>
[00104]    In other embodiments, the Fc portion contains amino acid alterations<br>
of the Leu-Ser-Leu-Ser segment near the C-terminus of the Fc portion of an<br>
immunoglobulin heavy chain. The amino acid substitutions of the Leu-Ser-Leu-<br>
Ser (SEQ ID NO:29) segment eliminate potential junctional T-cell epitopes. In<br>
one embodiment, the Leu-Ser-Leu-Ser (SEQ ID NO:29) amino acid sequence<br>
near the C-terminus of the Fc portion is replaced with an Ala-Thr-Ala-Thr (SEQ ID<br>
NO:30) amino acid sequence. In other embodiments, the amino acids within the<br>
Leu-Ser-Leu-Ser (SEQ ID NO:29) segment are replaced with other amino acids<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
23<br>
such as glycine or praline. Detailed methods of generating amino acid<br>
substitutions of the Leu-Ser-Leu-Ser (SEQ ID NO:29) segment near the C-<br>
terminus of an lgG1, lgG2, lgG3, lgG4, or other immunoglobulin class molecules,<br>
as well as other exemplary modifications for altering junctional T-cell epitopes,<br>
have been described in U.S. Published Patent Application No. 2003-0166877.<br>
[00105]    In one embodiment, a spacer or linker peptide is inserted between the<br>
carrier protein and the IL-7 protein. The spacer or linker peptide can be non-<br>
charged or non-polar or hydrophobic. The length of a spacer or linker peptide is<br>
between 1 and about 100 amino acids, or between 1 and about 50 amino acids,<br>
or between 1 and about 25 amino acids, or between 1 and about 15 amino acids.<br>
In one embodiment, the spacer contains a sequence (G4S)n, where n is less than<br>
10. In another embodiment, the linker sequence is GGGGSGGGG (SEQ ID<br>
NO:67). In yet another embodiment, the spacer contains a motif that is<br>
recognized as an N-linked glycosylation site. In another embodiment of the<br>
invention, the carrier protein and the IL-7 fusion protein are joined via a spacer or<br>
linker peptide. In an alternative embodiment of the invention, the carrier protein<br>
and IL-7 fusion protein are separated by a synthetic spacer, for example a PNA<br>
spacer. The spacer can be non-charged, or non-polar or hydrophobic.<br>
Production of IL-7 fusion proteins<br>
[00106]     Fusion proteins containing IL-7 modified according to the teachings of<br>
the invention can be synthesized by the non-limiting methods described herein.<br>
Assays useful for testing pharmacokinetic activities of fusion proteins containing<br>
IL-7 modified according to the invention in in vivo animal models are also<br>
described herein.<br>
[00107]    The IL-7 fusion proteins of the invention can be produced using<br>
recombinant expression vectors known in the art. The term "expression vector"<br>
refers to a replicable DNA construct used to express DNA which encodes the<br>
desired IL-7 fusion protein and which includes a transcriptional unit comprising an<br>
assembly of (1) genetic element(s) having a regulatory role in gene expression,<br>
for example, promoters, operators, or enhancers, operatively linked to (2) a DNA<br>
sequence encoding the desired IL-7 fusion protein which is transcribed into<br>
mRNA and translated into protein, and (3) appropriate transcription and<br>
translation initiation and termination sequences. The choice of promoter and<br>
other regulatory elements generally varies according to the intended host cell.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
24<br>
[00108]    The nucleic acid encoding the IL-7 fusion protein is transfected into a<br>
host cell using recombinant DNA techniques. In the context of the present<br>
invention, the foreign DNA includes a sequence encoding the inventive proteins.<br>
Suitable host cells include prokaryotic, yeast or higher eukaryotic cells. In one<br>
embodiment, the host is a prokaryotic organism.<br>
[00109]    The recombinant IL-7 fusion proteins can be expressed in yeast hosts,<br>
such as from Saccharomyces species, such as S. cerevisiae. Yeast of other<br>
genera such as Pichia or Kluyveromyces may also be employed. Yeast vectors<br>
will generally contain an origin of replication from a yeast plasmid or an<br>
autonomously replicating sequence (ARS), a promoter, DNA encoding the IL-7<br>
fusion protein, sequences for polyadenylation and transcription termination and a<br>
selection gene. Suitable promoter sequences in yeast vectors include the<br>
promoters for metallothionein, 3-phosphoglycerate kinase or other glycolytic<br>
enzymes, such as enolase, glyceraldehyde-3-phosphate dehydrogenase,<br>
hexokinase, pyruvate decarboxylase, phosphofructokinase, glucose-4-phosphate<br>
isomerase, 3-phosphoglycerate mutase, pyruvate kinase, triosephosphate<br>
isomerase, phosphoglucose isomerase and glucokinase.<br>
[00110]    Various mammalian or insect cell culture systems can be employed to<br>
express recombinant protein. Baculovirus systems for production of proteins in<br>
insect cells are well known in the art. Examples of suitable mammalian host cell<br>
lines include NS/0 cells, L cells, C127, 3T3, Chinese hamster ovary (CHO), HeLa,<br>
and BHK cell lines. Additional suitable mammalian host cells include CV-1 cells<br>
(ATCC CCL70) and COS-7 cells both derived from monkey kidney. Another<br>
suitable monkey kidney cell line, CV-1/EBNA, was derived by transfection of the<br>
CV-1 cell line with a gene encoding Epstein-Barr virus nuclear antigen-1 (EBNA-<br>
1) and with a vector containing CMV regulatory sequences (McMahan etal.,<br>
(1991), EMBQ J., 10:2821). The EBNA-1 gene allows for episomal replication of<br>
expression vectors, such as HAV-EO or pDC406, that contain the EBV origin of<br>
replication.<br>
[00111]    Mammalian expression vectors may comprise non-transcribed<br>
elements such as an origin of replication, a suitable promoter and enhancer<br>
linked to the gene to be expressed, and other 5' or 3' flanking nontranscribed<br>
sequences, and 5! or 3' nontranslated sequences, such as necessary ribosome<br>
binding sites, a poly-adenylation site, splice donor and acceptor sites, and<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
25<br>
transcriptional termination sequences. Commonly used promoters and<br>
enhancers are derived from Polyoma, Adenovirus 2, Simian Virus 40 (SV40), and<br>
human cytomegalovirus. DNA sequences derived from the SV40 viral genome,<br>
for example, SV40 origin, early and late promoter, enhancer, splice, and<br>
polyadenylation sites may be used to provide the other genetic elements required<br>
for expression of a heterologous DNA sequence.<br>
[00112]    When secretion of the IL-7 fusion protein from the host cell is desired,<br>
the expression vector may comprise DNA encoding a signal or leader peptide. In<br>
the present invention the native signal sequence of IL-7 can be used, or<br>
alternatively, a heterologous signal sequence may be added, such as the signal<br>
sequence from interleukin-4.<br>
[00113]    The present invention also provides a process for preparing the<br>
recombinant proteins of the present invention including culturing a host cell<br>
transformed with an expression vector comprising a DNA sequence that encodes<br>
the IL-7 fusion protein under conditions that promote expression. The desired<br>
protein is then purified from culture media or cell extracts. For example,<br>
supernatants from expression systems that secrete recombinant protein into the<br>
culture medium can be first concentrated using a commercially available protein<br>
concentration filter, for example, an Amicon or Millipore Pellicon ultrafiltration unit.<br>
Following the concentration step, the concentrate can be applied to a suitable<br>
purification matrix, as known in the art.<br>
[00114]    An "isolated" or "purified" IL-7 fusion protein or biologically active<br>
portion thereof is substantially free of cellular material or other contaminating<br>
proteins from the cell or tissue source from which the IL-7 fusion protein is<br>
derived, or substantially free from chemical precursors or other chemicals when<br>
chemically synthesized. The language "substantially free of cellular material"<br>
includes preparations of IL-7 fusion protein in which the protein is separated from<br>
cellular components of the cells from which it is isolated or recombinantly<br>
produced. In one embodiment, the language "substantially free of cellular<br>
material" includes preparations of IL-7 fusion protein having less than about 30%<br>
(by dry weight) of non-lL-7 fusion protein (also referred to herein as a<br>
"contaminating protein"), less than about 20% of non-IL-7 fusion protein, less than<br>
about 10% of non-IL-7 fusion protein, or less than about 5% non-IL-7 fusion<br>
protein. When the IL-7 fusion protein or biologically active portion thereof is<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
26<br>
purified from a recombinant source, it is, in one embodiment, substantially free of<br>
culture medium, i.e.., culture medium represents less than about 20%, less than<br>
about 10%, or less than about 5% of the volume of the protein preparation.<br>
[00115]    The term "substantially pure lg-IL-7 fusion protein" or "substantially<br>
pure IL-7 fusion protein" refers to a preparation in which the IL-7 comprising<br>
fusion protein constitutes at least 60%, 70%, 80%, 90%, 95% or 99% of the<br>
proteins in the preparation.<br>
Methods of Treatment Using IL-7 Proteins<br>
[00116]    The IL-7 proteins, including fusion proteins, of the invention are useful<br>
in treating immune deficiencies and in accelerating the natural reconstitution of<br>
the immune system that occurs, for example, after diseases or treatments that<br>
are immunosuppressive in nature. For example, IL-7 proteins can be used to<br>
treat viral infections, immune disorders, and to enhance the growth (including<br>
proliferation) of specific cell types. Moreover, the IL-7 proteins can be in the<br>
treatment of cancers such as bladder cancer, lung cancer, brain cancer, breast<br>
cancer, skin cancer, and prostate cancer. In one example, it is useful to treat<br>
patients who have undergone one or more cycles of chemotherapy with IL-7<br>
proteins as described above to help their immune cells replenish. Alternatively, it<br>
is also useful to administer the IL-7 proteins described above to patients with HIV,<br>
the elderly, patients receiving a transplant or other patients with suppressed<br>
immune system function.<br>
Administration<br>
[00117]    Both the IL-7 and IL-7 fusion proteins of the invention can be<br>
incorporated into a pharmaceutical composition suitable for administration. Such<br>
compositions typically comprise IL-7 or an IL-7 fusion protein and a<br>
pharmaceutically-acceptable carrier. As used herein the language<br>
"pharmaceutically-acceptable carrier" is intended to include any and all solvents,<br>
dispersion media, coatings, antibacterial and antifungal agents, isotonic and<br>
absorption delaying agents, and the like, compatible with pharmaceutical<br>
administration. The use of such media and agents for pharmaceutically active<br>
substances is well known in the art.<br>
[0118]        A pharmaceutical composition of the invention is formulated to be compatible<br>
with its intended route of administration. Examples of routes of administration include<br>
parenteral, e.g., intravenous, intradermal, subcutaneous, oral (e.g., inhalation),<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
27<br>
transdermal (topical), transmucosal, and rectal administration. Solutions or suspensions<br>
used for parenteral, intradermal, or subcutaneous application can include the following<br>
components: a sterile diluent such as water for injection, saline solution, fixed oils,<br>
polyethylene glycols, glycerine, propylene glycol or other synthetic solvents; antibacterial<br>
agents such as benzyl alcohol or methyl parabens; antioxidants such as ascorbic acid or<br>
sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid; buffers such as<br>
acetates, citrates or phosphates and agents for the adjustment of tonicity such as sodium<br>
chloride or dextrose. pH can be adjusted with acids or bases, such as hydrochloric acid or<br>
sodium hydroxide. The parenteral preparation can be enclosed in ampoules, disposable<br>
syringes or multiple dose vials made of glass or plastic.<br>
[0119]      Medicaments that contain the IL-7 proteins of the invention can have a<br>
concentration of 0.01 to 100% (w/w), though the amount varies according to the<br>
dosage form of the medicaments.<br>
[0120]      Administration dose depends on the body weight of the patients, the<br>
seriousness of the disease, and the doctor's opinion. However, it is generally<br>
advisable to administer between about 0.01 to about 10 mg/kg body weight a<br>
day, about 0.02 to about 2 mg/kg/day in case of injection, or about 0.5 mg/kg/day.<br>
The dose can be administered once or several times daily according to the<br>
seriousness of the disease and the doctor's opinion.<br>
[0121]      Compositions of the invention are useful when co-administered with<br>
one or more other therapeutic agents, for example, a molecule also known to be<br>
useful to replenish blood cells. For example, the molecule may be erythropoietin<br>
which is known to be used to replenish red blood cells, G-CSF which is used to<br>
replenish neutrophils or GM-CSF which is used to replenish granulocytes and<br>
macrophages.<br>
[0122]      Aspects of invention are further illustrated by the following examples.<br>
Examples<br>
Example 1: Identification of T-cell epitopes by computational methods<br>
[0123]      According to the invention, epitopes of IL-7 can be modified using<br>
methods for introducing mutations into proteins to modulate their interaction with<br>
the immune system. These methods are similar to those disclosed in U.S.<br>
Published Patent Application No. 2003-0166877. According to the invention,<br>
known methods in the art that can be adapted according to the invention include<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
28<br>
those described in the prior art (WO 92/10755 and WO 96/40792 (Novo Nordisk),<br>
EP 0519 596 (Merck &amp; Co.), EP 0699 755(Centro de Immunologia Moelcular),<br>
WO 98/52976 and WO 98/59244 (Biovation Ltd.) or related methods.<br>
[0124]      Advantageous mutant proteins, however, can be obtained if the<br>
identification of said epitopes is realized by the following method which is<br>
described herewith in detail and applied to IL-7. There are a number of factors<br>
that play important roles in determining the total structure of a protein,<br>
polypeptide or immunoglobulin. First, the peptide bond, i.e.., that bond which<br>
joins the amino acids in the chain together, is a covalent bond. This bond is<br>
planar in structure, essentially a substituted amide. An "amide" is any of a group<br>
of organic compounds containing the grouping -CONH-.<br><br>
[0125]      The planar peptide bond linking Ca of adjacent amino acids may be<br>
represented as depicted below:<br>
[0126]      Because the O=C and the C-N atoms lie in a relatively rigid plane, free<br>
rotation does not occur about these axes. Hence, a plane schematically depicted<br>
by the interrupted line is sometimes referred to as an "amide" or "peptide plane"<br>
plane wherein lie the oxygen (O), carbon (C), nitrogen (N), and hydrogen (H)<br>
atoms of the peptide backbone. At opposite corners of this amide plane are<br>
located the Ca atoms. Since there is substantially no rotation about the O=C and<br>
C-N atoms in the peptide or amide plane, a polypeptide chain thus comprises a<br>
series of planar peptide linkages joining the Ca atoms.<br>
[0127]      A second factor that pla     n important role in defining the total<br>
structure or conformation of a polypeptide or protein is the angle of rotation of<br>
each amide plane about the common Ca linkage. The terms "angle of rotation"<br>
and "torsion angle" are hereinafter regarded as equivalent terms. Assuming that<br>
the O, C, N, and H atoms remain in the amide plane (which is usually a valid<br>
assumption, although there may be some slight deviations from planarity of these<br>
atoms for some conformations), these angles of rotation define the N and R<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
29<br>
polypeptide's backbone conformation, i.e.., the structure as it exists between<br>
adjacent residues. These two angles are known as <j> and \\i. A set of the angles<br>
i, Ψi where the subscript i represents a particular residue of a polypeptide chain,<br>
thus effectively defines the polypeptide secondary structure. The conventions<br>
used in defining the , Ψ angles, i.e.., the reference points at which the amide<br>
planes form a zero degree angle, and the definition of which angle is , and which<br>
angle is Ψ, for a given polypeptide, are defined in the literature. (See, e.g.,<br>
Ramachandran et a/., (1968), Adv. Prot. Chem. 23:283-437, at pages 285-94).<br>
[0128]      The method can be applied to any protein, and is based in part upon<br>
the discovery that in humans the primary Pocket 1 anchor position of MHC Class<br>
II molecule binding grooves has a well designed specificity for particular amino<br>
acid side chains. The specificity of this pocket is determined by the identity of the<br>
amino acid at position 86 of the beta chain of the MHC Class II molecule. This<br>
site is located at the bottom of Pocket 1 and determines the size of the side chain<br>
that can be accommodated by this pocket. Marshall, J. Immunol., (1994),<br>
152:4946-4956. If this residue is a glycine, then all hydrophobic aliphatic and<br>
aromatic amino acids (hydrophobic aliphatics being: valine, leucine, isoleucine,<br>
methionine and aromatics being: phenylalanine, tyrosine and tryptophan) can be<br>
accommodated in the pocket, with a preference being for the aromatic side<br>
chains. If this pocket residue is a valine, then the side chain of this amino acid<br>
protrudes into the pocket and restricts the size of peptide side chains that can be<br>
accommodated such that only hydrophobic aliphatic side chains can be<br>
accommodated. Therefore, in an amino acid residue sequence, wherever an<br>
amino acid with a hydrophobic aliphatic or aromatic side chain is found, there is<br>
the potential for a MHC Class II restricted T-cell epitope. If the side-chain is<br>
hydrophobic aliphatic, however, it is approximately twice as likely to be<br>
associated with a T-cell epitope than an aromatic side chain (assuming an<br>
approximately even distribution of Pocket 1 types throughout the global<br>
population).<br>
[0129]      An exemplary computational method profiles the likelihood of peptide<br>
regions of IL-7 to contain T-cell epitopes as follows: (1) The primary sequence of<br>
a peptide segment of predetermined length is scanned, and all hydrophobic<br>
aliphatic and aromatic side chains present are identified. (2) The hydrophobic<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
30<br>
aliphatic side chains are assigned a value greater than that for the aromatic side<br>
chains; preferably about twice the value assigned to the aromatic side chains,<br>
e.g., a value of,2 for a hydrophobic aliphatic side chain and a value of 1 for an<br>
aromatic side chain. (3) The values determined to be present are summed for<br>
each overlapping amino acid residue segment (window) of predetermined uniform<br>
length within the peptide, and the total value for a particular segment (window) is<br>
assigned to a single amino acid residue at an intermediate position of the<br>
segment (window), preferably to a residue at about the midpoint of the sampled<br>
segment (window). This procedure is repeated for each sampled overlapping<br>
amino acid residue segment (window). Thus, each amino acid residue of the<br>
peptide is assigned a value that relates to the likelihood of a T-cell epitope being<br>
present in that particular segment (window). (4) The values calculated and<br>
assigned as described in Step 3, above, can be plotted against the amino acid<br>
coordinates of the entire amino acid residue sequence being assessed. (5) All<br>
portions of the sequence which have a score of a predetermined value, e.g., a<br>
value of 1, are deemed likely to contain a T-cell epitope and can be modified, if<br>
desired.<br>
[0130]      This particular aspect of the present invention provides a general<br>
method by which T-cell epitopes of lL-7 can be described. Modifications to the<br>
peptide in these regions have the potential to modify the MHC Class II binding<br>
characteristics.<br>
[0131]      According to another aspect of the present invention, T-cell epitopes<br>
can be predicted with greater accuracy by the use of a more sophisticated<br>
computational method which takes into account the interactions of peptides with<br>
models of MHC Class II alleles.<br>
[0132]      The computational prediction of T-cell epitopes present within a<br>
peptide according to this particular aspect contemplates the construction of<br>
models of at least 42 MHC Class II alleles based upon the structures of all known<br>
MHC Class II molecules and a method for the use of these models in the<br>
computational identification of T-cell epitopes, the construction of libraries of<br>
peptide backbones for each model in order to allow for the known variability in<br>
relative peptide backbone alpha carbon (Ca) positions, the construction of<br>
libraries of amino-acid side chain conformations for each backbone dock with<br>
each model for each of the 20 amino-acid alternatives at positions critical for the<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
31<br>
interaction between peptide and MHC Class II molecule, and the use of these<br>
libraries of backbones and side-chain conformations in conjunction with a scoring<br>
function to select the optimum backbone and side-chain conformation for a<br>
particular peptide docked with a particular MHC Class II molecule and the<br>
derivation of a binding score from this interaction.<br>
[0133]      Models of MHC Class II molecules can be derived via homology<br>
modeling from a number of similar structures found in the Brookhaven Protein<br>
Data Bank ("PDB"). These may be made by the use of semi-automatic homology<br>
modeling software (Modeller et al., (1993), J. Mol. Biol., 234:779-815) which<br>
incorporates a simulated annealing function, in conjunction with the CHARMm<br>
force-field for energy minimization (available from Molecular Simulations Inc., San<br>
Diego, Ca.). Alternative modeling methods can be utilized as well.<br>
[0134]      Other computational methods which use libraries of experimentally<br>
derived binding data of each amino-acid alternative at each position in the binding<br>
groove for a small set of MHC Class II molecules (Marshall et al., (1995), Biomed.<br>
Pept. Proteins Nucleic Acids, 1(3):157-162) are known, as are yet other<br>
computational methods which use similar experimental binding data in order to<br>
define the binding characteristics of particular types of binding pockets within the<br>
groove, again using a relatively small subset of MHC Class II molecules, and then<br>
'mixing and matching1 pocket types from this pocket library to artificially create<br>
further 'virtual' MHC Class II molecules (Stumiolo et al., (1999), Nat. Biotech,<br>
17(6): 555-561. Both methods suffer the major disadvantage that, due to the<br>
complexity of the assays and the need to synthesize large numbers of peptide<br>
variants, only a small number of MHC Class II molecules can be experimentally<br>
scanned. Therefore the first method can only make predictions for a small<br>
number of MHC Class II molecules. The second method also makes the<br>
assumption that a pocket lined with similar amino-acids in one molecule will have<br>
the same binding characteristics when in the context of a different Class II allele<br>
and suffers further disadvantages in that only those MHC Class II molecules can<br>
be 'virtually' created which contain pockets contained within the pocket library.<br>
Using the modeling approach described herein, the structure of any number and<br>
type of MHC Class II molecules can be deduced, therefore alleles can be<br>
specifically selected to be representative of the global population. In addition, the<br>
number of MHC Class II molecules scanned can be increased by making further<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
32<br>
models further than having to generate additional data via complex<br>
experimentation.<br>
[0135]      The use of a backbone library allows for variation in the positions of<br>
the Ca atoms of the various peptides being scanned when docked with particular<br>
MHC Class II molecules. This is again in contrast to the alternative<br>
computational methods described above which rely on the use of simplified<br>
peptide backbones for scanning amino-acid binding in particular pockets. These<br>
simplified backbones are not likely to be representative of backbone<br>
conformations found in 'real' peptides leading to inaccuracies in prediction of<br>
peptide binding. The present backbone library is created by superposing the<br>
backbones of all peptides bound to MHC Class II molecules found within the<br>
Protein Data Bank and noting the root mean square (RMS) deviation between the<br>
Ca atoms of each of the eleven amino-acids located within the binding groove.<br>
While this library can be derived from a small number of suitable available mouse<br>
and human structures (currently 13), in order to allow for the possibility of even<br>
greater variability, the RMS figure for each C"- a position is increased by 50%.<br>
The average Ca position of each amino-acid is then determined and a sphere<br>
drawn around this point whose radius equals the RMS deviation at that position<br>
plus 50%. This sphere represents all allowed Ca positions.<br>
[0136]      Working from the Ca with the least RMS deviation (that of the amino-<br>
acid in Pocket 1 as mentioned above, equivalent to Position 2 of the 11 residues<br>
in the binding groove), the sphere is three-dimensionally gridded, and each vertex<br>
within the grid is then used as a possible location for a Ca of that amino-acid.<br>
The subsequent amide plane, corresponding to the peptide bond to the<br>
subsequent amino-acid is grafted onto each of these Cas and the 		 and 	Ψ angles<br>
are rotated step-wise at set intervals in order to position the subsequent Ca. If<br>
the subsequent Ca falls within the 'sphere of allowed positions' for this Ca than<br>
the orientation of the dipeptide is accepted, whereas if it falls outside the sphere<br>
then the dipeptide is rejected. This process is then repeated for each of the<br>
subsequent Ca positions, such that the peptide grows from the Pocket 1 Ca<br>
'seed1, until all nine subsequent Cas have been positioned from all possible<br>
permutations of the preceding Cas. The process is then repeated once more for<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
33<br>
the single Ca preceding pocket 1 to create a library of backbone Ca positions<br>
located within the binding groove.<br>
[0137]      The number of backbones generated is dependent upon several<br>
factors: The size of the 'spheres of allowed positions'; the fineness of the<br>
gridding of the 'primary sphere' at the Pocket 1 position; the fineness of the step-<br>
wise rotation of the 	 and 	Ψ angles used to position subsequent Cas. Using this<br>
process, a large library of backbones can be created. The larger the backbone<br>
library, the more likely it will be that the optimum fit will be found for a particular<br>
peptide within the binding groove of an MHC Class il molecule. Inasmuch as all<br>
backbones will not be suitable for docking with all the models of MHC Class II<br>
molecules due to clashes with amino-acids of the binding domains, for each allele<br>
a subset of the library is created comprising backbones which can be<br>
accommodated by that allele. The use of the backbone library, in conjunction<br>
with the models of MHC Class II molecules creates an exhaustive database<br>
consisting of allowed side chain conformations for each amino-acid in each<br>
position of the binding groove for each MHC Class II molecule docked with each<br>
allowed backbone. This data set is generated using a simple steric overlap<br>
function where a MHC Class II molecule is docked with a backbone and an<br>
amino-acid side chain is grafted onto the backbone at the desired position. Each<br>
of the rotatable bonds of the side chain is rotated step-wise at set intervals and<br>
the resultant positions of the atoms dependent upon that bond noted. The<br>
interaction of the atom with atoms of side-chains of the binding groove is noted<br>
and positions are either accepted or rejected according to the following criteria:<br>
the sum total of the overlap of all atoms so far positioned must not exceed a pre-<br>
determined value. Thus the stringency of the conformational search is a function<br>
of the interval used in the step-wise rotation of the bond and the pre-cletermined<br>
limit for the total overlap. This latter value can be small if it is known that a<br>
particular pocket.is rigid; however, the stringency can be relaxed if the positions<br>
of pocket side-chains are known to be relatively flexible. Thus allowances can be<br>
made to imitate variations in flexibility within pockets of the binding groove. This<br>
conformationa! search is then repeated for every amino-acid at every position of<br>
each backbone when docked with each of the MHC Class II molecules to create<br>
the exhaustive database of side-chain conformations.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
34<br>
[0138]      A suitable mathematical expression is used to estimate the energy of<br>
binding between models of MHC Class II molecules in conjunction with peptide<br>
ligand conformations which have to be empirically derived by scanning the large<br>
database of backbone/side-chain conformations described above. Thus a protein<br>
is scanned for potential T-cell epitopes by subjecting each possible peptide of<br>
length varying between 9 and 20 amino-acids (although the length is kept<br>
constant for each scan) to the following computations: An MHC Class II molecule<br>
is selected together with a peptide backbone allowed for that molecule and the<br>
side-chains corresponding to the desired peptide sequence are grafted on. Atom<br>
identity and interatomic distance data relating to a particular side-chain at a<br>
particular position on the backbone are collected for each allowed conformation<br>
of that amino-acid (obtained from the database described above). This is<br>
repeated for each side-chain along the backbone and peptide scores derived<br>
using a scoring function. The best score for that backbone is retained and the<br>
process repeated for each allowed backbone for the selected model. The scores<br>
from all allowed backbones are compared and the highest score is deemed to be<br>
the peptide score for the desired peptide in that MHC Class II model. This<br>
process is then repeated for each model with every possible peptide derived from<br>
the protein being scanned, and the scores for peptides versus models are<br>
displayed.<br>
[0139]      In the context of the present invention, each ligand presented for the<br>
binding affinity calculation is an amino-acid segment selected from a peptide or<br>
protein as discussed above. Thus, the ligand is a selected stretch of amino acids<br>
about 9 to 20 amino acids in length derived from a peptide, polypeptide or protein<br>
of known sequence. The terms "amino acids" and "residues" are hereinafter<br>
regarded as equivalent terms. The ligand, in the form of the consecutive amino<br>
acids of the peptide to be examined grafted onto a backbone from the backbone<br>
library, is positioned in the binding cleft of an MHC Class II molecule from the<br>
MHC Class II molecule mode! library via the coordinates of the C"-a atoms of the<br>
peptide backbone and an allowed conformation for each side-chain is selected<br>
from the database of allowed conformations. The relevant atom identities and<br>
interatomic distances are also retrieved from this database and used to calculate<br>
the peptide binding score. Ligands with a high binding affinity for the MHC Class<br>
II binding pocket are flagged as candidates for site-directed mutagenesis. Amino-<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
35<br>
acid substitutions are made in the flagged ligand (and hence in the protein of<br>
interest) which is then retested using the scoring function in order to determine<br>
changes which reduce the binding affinity below a predetermined threshold value.<br>
These changes can then be incorporated into the protein of interest to remove T-<br>
cell epitopes.<br>
[0140]      Binding between the peptide ligand and the binding groove of MHC<br>
Class II molecules involves non-covalent interactions including, but not limited to:<br>
hydrogen bonds, electrostatic interactions, hydrophobic (lipophilic) interactions<br>
and van der Waal's interactions. These are included in the peptide scoring<br>
function as described in detail below. It should be understood that a hydrogen<br>
bond is a non-covalent bond which can be formed between polar or charged<br>
groups and consists of a hydrogen atom shared by two other atoms. The<br>
hydrogen of the hydrogen donor has a positive charge where the hydrogen<br>
acceptor has a partial negative charge. For the purposes of peptide/protein<br>
interactions, hydrogen bond donors may be either nitrogens with hydrogen<br>
attached or hydrogens attached to oxygen or nitrogen. Hydrogen bond acceptor<br>
atoms may be oxygens not attached to hydrogen, nitrogens with no hydrogens<br>
attached and one or two connections, or sulphurs with only one connection.<br>
Certain atoms, such as oxygens attached to hydrogens or imine nitrogens {e.g.<br>
C=NH) may be both hydrogen acceptors or donors. Hydrogen bond energies<br>
range from 3 to 7 Kcal/mol and are much stronger than van der Waal's bonds, but<br>
weaker than covalent bonds. Hydrogen bonds are also highly directional and are<br>
at their strongest when the donor atom, hydrogen atom and acceptor atom are<br>
co-linear. Electrostatic bonds are formed between oppositely charged ion pairs<br>
and the strength of the interaction is inversely proportional to the square of the<br>
distance between the atoms according to Coulomb's law. The optimal distance<br>
between ion pairs is about 2.8A. In protein/peptide interactions,, electrostatic<br>
bonds may be formed between arginine, histidine or lysine and aspartate or<br>
glutamate. The strength of the bond will depend upon the pKa of the ionizing<br>
group and the dielectric constant of the medium although they are approximately<br>
similar in strength to hydrogen bonds.<br>
[0141]      Lipophilic interactions are favorable hydrophobic-hydrophobic contacts<br>
that occur between the protein and the peptide ligand. Usually, these will occur<br>
between hydrophobic amino acid side chains of the peptide buried within the<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
36<br>
pockets of the binding groove such that they are not exposed to solvent.<br>
Exposure of the hydrophobic residues to solvent is highly unfavorable since the<br>
surrounding solvent molecules are forced to hydrogen bond with each other<br>
forming cage-like clathrate structures. The resultant decrease in entropy is highly<br>
unfavorable. Lipophilic atoms may be sulphurs which are neither polar nor<br>
hydrogen acceptors and carbon atoms which are not polar.<br>
[0142]      van der Waal's bonds are non-specific forces found between atoms<br>
which are 3- 4A apart. They are weaker and less specific than hydrogen and<br>
electrostatic bonds. The distribution of electronic charge around an atom<br>
changes with time and, at any instant, the charge distribution is not symmetric.<br>
This transient asymmetry in electronic charge induces a similar asymmetry in<br>
neighboring atoms. The resultant attractive forces between atoms reaches a<br>
maximum at the van der Waal's contact distance but diminishes very rapidly at<br>
about 1Ǻ to about 2Ǻ. Conversely, as atoms become separated by less than the<br>
contact distance, increasingly strong repulsive forces become dominant as the<br>
outer electron clouds of the atoms overlap. Although the attractive forces are<br>
relatively weak compared to electrostatic and hydrogen bonds (about 0.6<br>
Kcal/mol), the repulsive forces in particular may be very important in determining<br>
whether a peptide ligand may bind successfully to a protein.<br>
[0143]       In one embodiment, the Bohm scoring function (SCORE1 approach)<br>
is used to estimate the binding constant. (Bohm, H.J.. (1994), J. Comput. Aided<br>
Moi. Des., 8(3):243-256) which is hereby incorporated in its entirety). In another<br>
embodiment, the scoring function (SCORE2 approach) is used to estimate the<br>
binding affinities as an indicator of a ligand containing a T-cell epitope (Bohm,<br>
H.J., (1998), J. Comput. Aided Mol. Des., 12(4):309-323) which is hereby<br>
incorporated in its entirety). However, the Bohm scoring functions as described<br>
in the above references are used to estimate the binding affinity of a ligand to a<br>
protein where it is already known that the ligand successfully binds to the protein<br>
and the protein/ligand complex has had its structure solved, the solved structure<br>
being present in the Protein Data Bank ("PDB"). Therefore, the scoring function<br>
has been developed with the benefit of known positive binding data. In order to<br>
allow for discrimination between positive and negative binders, a repulsion term<br>
must be added to the equation. In addition, a more satisfactory estimate of<br>
binding energy is achieved by computing the lipophilic interactions in a pairwise<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
37<br>
manner rather than using the area based energy term of the above Bohm<br>
functions. Therefore, in one embodiment, the binding energy is estimated using a<br>
modified Bohm scoring function. In the modified Bohm scoring function, the<br>
binding energy between protein and ligand (∆Gbind) is estimated considering the<br>
following parameters: The reduction of binding energy due to the overall loss of<br>
translational and rotational entropy of the ligand (∆G0); contributions from ideal<br>
hydrogen bonds (∆Ghb) where at least one partner is neutral; contributions from<br>
unperturbed ionic interactions (AGiOnic); lipophilic interactions between lipophilic<br>
ligand atoms and lipophilic acceptor atoms (AG|jP0); the loss of binding energy<br>
due to the freezing of internal degrees of freedom in the ligand, i.e.., the freedom<br>
of rotation about each C-C bond is reduced (∆Grot); the energy of the interaction<br>
between the protein and ligand (Evdw). Consideration of these terms gives<br>
equation 1:<br>
(∆Gbind)=(∆Go)+(∆GhbXNhb)+(∆GionicXNioniC)+(∆GlipoXNlipo)+(∆Grot+NrQt)+(Evdw)-<br>
where N is the number of qualifying interactions for a specific term and, in one<br>
embodiment, ∆Go, ∆Ghb, ∆GiOnic. ∆Glipo and ∆Grat are constants which are given<br>
the values: 5.4, -4.7, -4.7, -0.17, and 1.4, respectively.<br>
[0144]      The term Nhb is calculated according to equation 2:<br>
Nhb = ∑h-bondsf(	∆R, ∆α) X f(Nneighb) X fpcs<br>
[0145]      f(AR, Aa) is a penalty function which accounts for large deviations of<br>
hydrogen bonds from ideality and is calculated according to equation 3:<br>
f(∆R,∆-α) = f1(	∆R) x f2(∆		α)<br>
where:	fl(∆R)   =1 if ∆R
or       = 1 - (∆R - TOL)/0.4 if AR 
or       = 0 if ∆R &gt;0.4 + TOL<br>
and:	f2(∆α) = 1 if 		∆α 
or       = 1 -(∆	α - 30)/50 if 		∆α 
or       = 0 if ∆α &gt;80°<br>
[0146]      TOL is the tolerated deviation in hydrogen bond length = 0.25Ǻ;<br>
∆R is the deviation of the H-O/N hydrogen bond length from the ideal value = 1.9Ǻ;<br>
∆α is the deviation of the hydrogen bond angle 
[0147]      f(Nneighb) distinguishes between concave and convex parts of a protein<br>
surface and therefore assigns greater weight to polar interactions found in<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
38<br>
pockets rather than those found at the protein surface. This function is calculated<br>
according to equation 4 below:<br>
f(Nneighb) = (Nneighb/Nneighb,o)a        where α = 0.5.<br>
[0148]       Nneighb is the number of non-hydrogen protein atoms that are closer<br>
than 5Ǻ to any given protein atom.<br>
[0149]       Neeighb,o is a constant = 25<br>
[0150]      fpcs is a function which allows for the polar contact surface area per<br>
hydrogen bond and therefore distinguishes between strong and weak hydrogen<br>
bonds and its value is determined according to the following criteria:<br>
fpcs= β When APolar/NHB 
or fPcs= I when Apolar/NHB &gt; 10 Ǻ2<br>
Apolar is the size of the polar protein-ligand contact surface<br>
NHB is the number of hydrogen bonds<br>
β is a constant whose value = 1.2<br>
[0151]       For the implementation of the modified Bohm scoring function, the<br>
contributions from ionic interactions, ∆GiOnic, are computed in a similar fashion to<br>
those from hydrogen bonds described above since the same geometry<br>
dependency is assumed.<br>
[0152]      The term Nlipo is calculated according to equation 5 below:<br>
Nlipo	= ∑ILf(riL)<br>
[0153]      f(riL) is calculated for all lipophilic ligand atoms, I, and all lipophilic<br>
protein atoms, L, according to the following criteria:<br>
f(rIL) =1 when riL  R1<br>
f(rIL&gt; =0 when rIL &gt;= R2<br>
where: R1=r,vdw + rLvdw +0.5<br>
and R2 = R1 + 3.0<br>
and rivdw is the van der Waal's radius of atom I<br>
and rLVdw is the van der Waal's radius of atom L<br>
[0154]      The term Nrot is the number of ratable bonds of the amino acid side<br>
chain and is taken to be the number of acyclic sp3 - sp3 and sp3 - sp2 bonds.<br>
Rotations of terminal -CH3 or -NH3 are not taken into account.<br>
[0155]      The final term, Evdw, is calculated according to equation 6 below:<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
39<br>
Evdw = ε1ε2((r1vdw +r2vdw)12/r12 - (nvdw +r2vdw)6/r6), where:<br>
ε1 and ε2 are constants dependent upon atom identity;<br>
r1vdw +r2vdw are the van der Waal's atomic radii; and<br>
r is the distance between a pair of atoms.<br>
    [0156]      With regard to equation 6, in one embodiment, the constants ε1 and ε2<br>
are given the atom values: C: 0.245, N: 0.283, O: 0.316, S: 0.316, respectively<br>
(i.e.. for atoms of Carbon, Nitrogen, Oxygen and Sulfur, respectively). With<br>
regards to equations 5 and 6, the van der Waal's radii are given the atom values<br>
C: 1.85, N: 1.75, O: 1.60, S: 2.00		Ǻ.<br>
[0157]      It should be understood that all predetermined values and constants<br>
given in the equations above are determined within the constraints of current<br>
understandings of protein ligand interactions with particular regard to the type of<br>
computation being undertaken herein.<br>
[0158]      As described above, the scoring function is applied to data extracted<br>
from the database of side-chain conformations, atom identities, and interatomic<br>
distances. For the purposes of the present description, the number of MHC<br>
Class II molecules included in this database is 42 models plus four solved<br>
structures. It should be apparent from the above descriptions that the modular<br>
nature of the construction of the computational method of the present invention<br>
means that new models can simply be added and scanned with the peptide<br>
backbone library and side-chain conformational search function to create<br>
additional data sets which can be processed by the peptide scoring function as<br>
described above. This allows for the repertoire of scanned MHC Class II<br>
molecules to easily be increased, or structures and associated data to be<br>
replaced if data are available to create more accurate models of the existing<br>
alleles.<br>
[0159]      The present prediction method can be calibrated against a data set<br>
comprising a large number of peptides whose affinity for various MHC Class II<br>
molecules has previously been experimentally determined. By comparison of<br>
calculated versus experimental data, a cut of value can be determined above<br>
which it is known that ail experimentally determined T-cell epitopes are correctly<br>
predicted.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
40<br>
[0160]      It should be understood that, although the above scoring function is<br>
relatively simple compared to some sophisticated methodologies that are<br>
available, the calculations are performed extremely rapidly. It should also be<br>
understood that the objective is not to calculate the true binding energy perse for<br>
each peptide docked in the binding groove of a selected MHC Class II protein.<br>
The underlying objective is to obtain comparative binding energy data as an aid<br>
to predicting the location of T-cell epitopes based on the primary structure (i.e..<br>
amino acid sequence) of a selected protein. A relatively high binding energy or a<br>
binding energy above a selected threshold value would suggest the presence of a<br>
T-ce!l epitope in the iigand. The ligand may then be subjected to at least one<br>
round of amino-acid substitution and the binding energy recalculated. Due to the<br>
rapid nature of the calculations, these manipulations of the peptide sequence can<br>
be performed interactively within the program's user interface on cost-effectively<br>
available computer hardware. Major investment in computer hardware is thus not<br>
required.<br>
[0161]       It would be apparent to one skilled in the art that other available<br>
software could be used for the same purposes. In particular, more sophisticated<br>
software which is capable of docking ligands into protein binding-sites may be<br>
used in conjunction with energy minimization. Examples of docking software are:<br>
DOCK (Kuntz et al., (1982), J. Mol. BioL 161:269-288), LUDI (Bohm, H.J.,<br>
(1994), J. Comput Aided Mol. Des.. 8:623-632) and FLEXX (Rarey et al., (1995),<br>
ISMB, 3:300-308). Examples of molecular modeling and manipulation software<br>
include: AMBER (Tripos) and CHARMm (Molecular Simulations Inc.). The use of<br>
these computational methods would severely limit the throughput of the method<br>
of this invention due to the lengths of processing time required to make the<br>
necessary calculations. However, it is feasible that such methods could be used<br>
as a 'secondary screen' to obtain more accurate calculations of binding energy<br>
for peptides which are found to be 'positive binders' via the method of the present<br>
invention. The limitation of processing time for sophisticated molecular mechanic<br>
or molecular dynamic calculations is one which is defined both by the design of<br>
the software which makes these calculations and the current technology<br>
limitations of computer hardware. It may be anticipated that, in the future, with<br>
the writing of more efficient code and the continuing increases in speed of<br>
computer processors, it may become feasible to make such calculations within a<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
41<br>
more manageable time-frame. Further information on energy functions applied to<br>
macromolecules and consideration of the various interactions that take place<br>
within a folded protein structure can be found in: Brooks etal., (1983), J. Comput.<br>
Chem., 4:187-217 and further information concerning general protein-ligand<br>
interactions can be found in: Dauber-Osguthorpe etal., (1988), Proteins, 4(1):31-<br>
47. Useful background information can also be found, for example, in Fasman,<br>
G.D., ed., Prediction of Protein Structure and the Principles of Protein<br>
Conformation, Plenum Press, New York, ISBN: 0-306 4313-9.<br>
Example 2: In vitro analysis of IL-7 derived peptides as potential CD4+ T helper<br>
cell epitopes by unfractionated PBMC cultures<br>
[0162]      Based on in silico predictions that sequences surrounding N-linked<br>
glycosylation sites of the IL-7 protein are immunogenic, peptides encompassing<br>
these regions, spanning for example Leu63 to Ser71 (LRQFLKMNS SEQ ID<br>
NO:4) or Ile88 to Val96 (ILLNCTGQV SEQ ID NO:7) in mature human IL-7<br>
protein, are analyzed for their immunogenicity, which is measured by their ability<br>
to induce T-cell proliferation in vitro. In essence, PBMCs isolated from human<br>
blood are incubated with individual overlapping 15-mer peptides, and proliferative<br>
responses are measured by 3H-thymidine incorporation. In principle, T-cells<br>
within the mixture of PBMCs will only proliferate if they recognize individual<br>
peptide-MHC complexes on autologous APCs (antigen presenting cells), and<br>
thus proliferation is an indication of peptide immunogenicity.<br>
[0163]      For example, 15-mer peptides that are staggered by three amino acids<br>
and span the region from, for example, Met54 to Leu80 in human IL-7 are<br>
synthesized (Pepscan Systems, Netherlands), resuspended in DMSO (Sigma<br>
Chemical, St. Louis, MO, U.S.A.), and used at a final concentration of 5 //M in<br>
0.5% DMSO in culture media.<br>
[0164]      PBMCs are isolated from peripheral blood from healthy donors by<br>
Ficoll-Hypaque gradient centrifugation, and are stored frozen in liquid nitrogen. In<br>
addition, each PBMC sample is HLA typed, using a SSP PCR typing kit (Bio-<br>
Synthesis, Lewisville, TX) on DNA isolated with a QiaAmp Tissue Kit (Qiagen,<br>
Valencia, CA).<br>
[0165]      In a typical proliferation assay, each of the overlapping 15-mer<br>
peptides is assayed in sextuplicate PBMC cultures derived from 40 naTve donors.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
42<br>
Briefly, 2 x105 PBMCs, thawed rapidly before use, are mixed with 5 pM of each<br>
peptide and incubated at 37°C in 5% CO2 for 7 days. As a positive control,<br>
samples are incubated with the tetanus toxin derived peptide MQYIKANSKFIGI<br>
(SEQ ID NO:15), whereas negative control samples are incubated with 0.5%<br>
DMSO. During the last 12 hours of incubation the cultures are pulsed with of<br>
[methyl-3H]thymidine (0.5 //Ci/well) (NEN Life Science Products, Boston, MA), the<br>
cultures are harvested onto filter mats and thymidine incorporation is measured<br>
as counts per minute (CPM) using a Wallac microplate beta top plate scintillation<br>
counter (Perkin Elmer, Boston, MA). The stimulatory index for each peptide is<br>
calculated by dividing the CPM value of a given peptide divided by the CPM value<br>
obtained from negative controls.<br>
[0166]       It is found that the stimulatory index of the positive control peptide is<br>
significantly greater than 1 (the stimulatory index of the negative control), and<br>
peptides containing the core sequence LRQFLKMNS (SEQ ID NO:4),<br>
FLKMNSTGD (SEQ ID NO:5) or LKMNSTGDF (SEQ ID NO:6), such as, for<br>
example peptides ARKLRQFLKMNSTGD (SEQ ID NO:10),<br>
LRQFLKMNSTGDFDL (SEQ ID NO:11), or FLKMNSTGDFDLHLL (SEQ ID<br>
NO: 12), have an increase in stimulatory index. Therefore peptide sequences<br>
such as LRQFLKMNS (SEQ ID NO:4) or LKMNSTGDF (SEQ ID NO:6) indeed<br>
represent potential T-cell epitopes.<br>
[0167]      A similar analysis is performed with a series of 15-mer peptides<br>
encompassing a region defined by the core peptides ILLNCTGQV (SEQ ID NO:7)<br>
and LLNCTGQVK (SEQ ID NO:8), and it is found that these peptide sequences<br>
as well represent potential T-cell epitopes.<br>
Example 3: Mapping of CD4+ T helper cell epitopes using differentiated human<br>
dendritic cells (DCs) in vitro<br>
[0168]      Mature dendritic cells (DCs) are potent antigen-presenting cells<br>
(APCs) that present antigenic peptides or whole proteins to T-cells efficiently.<br>
Isolated DCs, pulsed with antigenic peptides in vitro, are used to induce primary<br>
T-cell responses that can be measured in in vitro proliferation assays.<br>
Differentiated DCs are generated, for example by the following procedure: first,<br>
human monocytes are generated by allowing PBMCs to adhere to plastic tissue<br>
culture flasks or by purifying CD14+ PBMCs with magnetically labeled antibodies<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
43<br>
(Miltenyi Biotec, Auburn, CA). The purified monocytes (0.5 to 1.5 x 10° cells/ml)<br>
are then cultured in AIM V media (GIBCO BRL, Grand Island, NY, U.S.A.)<br>
containing 1000 U7ml GM-CSF (Endogen; Woburn, MA) and 500 U/ml IL-4<br>
(Endogen; Woburn, MA) for 3 days. Subsequently, these immature DCs are<br>
pulsed with 5 ^/g/ml of experimental or control peptides and further incubated with<br>
a combination of 1000 U/ml TNF-a (Endogen; Woburn, MA), 1000 U/ml GM-CSF<br>
and 500 U/ml IL-4 for another 48 hours. Mature DCs are monitored by high<br>
surface expression levels of CD80+, CD86+ and HLA-DR.<br>
[0169]      These mature antigen-pulsed DCs are irradiated with 4200 Rads and<br>
are used in a proliferation assay with purified autologous CD4+ T-cells. (CD4+ T-<br>
cells are purified with magnetically labeled antibodies (Miltenyi Biotec, Auburn,<br>
CA), using frozen PBMC aliquots from the same donor that provides monocytes<br>
for the in vitro DC differentiation.) In a typical assay, antigen-pulsed DCs (2 *<br>
105/mL) are incubated together with autologous CD4+ T-cells (2 x 106 cells/mL) in<br>
round-bottomed 96-well plates at 37°C in 5% CO2 for 7 days, [methyl-<br>
3H]thymidine (NEN Life Science Products, Boston, MA) is added during the last<br>
12 hours of incubation at 0.5 pCi/well, the samples are harvested, lysed onto<br>
glass filters, and 3H-thymidine incorporation is measured in a scintillation counter.<br>
[0170]      15-mer peptides, as described for Example 1, are tested in this assay<br>
and compared to reference peptides and other controls. It is found that this<br>
assay is more sensitive than the assay described in Example 1, allowing better<br>
differentiation between the ability of individual peptides to induce T-cell<br>
proliferation. It is found, that IL-7 peptides containing the core sequences<br>
LRQFLKMNS (SEQ ID NO:4), FLKMNSTGD (SEQ ID NO:5), LKMNSTGDF<br>
(SEQ ID NO:6), ILLNCTGQV(SEQ ID NO:7) or LLNCTGQVK (SEQ ID NO:8) do<br>
indeed induce significant T-cell proliferation and therefore these sequences do<br>
represent potential T-cell epitopes.<br>
Example 4: In vitro analysis of de-immunizing amino acid substitutions in IL-7<br>
[0171]      Amino acid substitutions in the peptide regions described above which<br>
are considered to render the IL-7 protein less immunogenic are tested in in vitro<br>
assays, as described in Example 1 and Example 2. For example, the variant IL-7<br>
peptide encompassing the sequence LRQFLDDNS (SEQ ID NO: 13) is expected<br>
to generate a significantly decreased T-cell proliferative response compared to<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
44<br>
the wild-type parental peptide encompassing LRQFLKMNS (SEQ ID NO:4).<br>
Similarly, the variant IL-7 peptide encompassing the sequence TLLMCTGQG<br>
(SEQ ID NO: 14) is expected to generate a significantly decreased T-cell<br>
proliferative response compared to the wild-type parental peptide encompassing<br>
ILLNCTGQV (SEQ ID NO:7)<br>
[0172]      A series of IL-7 derived 15-mer peptides is synthesized that<br>
encompass the variant IL-7 sequences LRQFLDDNS (SEQ ID NO:13) or<br>
TLLNCTGQG (SEQ ID NO:14), as described in Example 1. In addition, the<br>
variant IL-7 proteins, or fusion proteins containing variant IL-7, which include<br>
substitutions of the invention are produced either in a prokaryotic or eukaryotic<br>
expression system (Del-IL-7's). For example, a variant IL-7 is produced that<br>
includes the amino acid substitutions K68D, M69D, I88T and V94G. (In addition,<br>
the prokaryotically produced IL-7 proteins include a start methionine.) These<br>
peptides and purified proteins, and their parental counterparts, are tested in their<br>
ability to induce T-cell proliferation, in assays with whole PBMC cultures as<br>
described in Example 1 or by pulsing human DCs as described in Example 2. 25<br>
	μg/ml of protein is used to stimulate PBMCs or to pulse DCs.<br>
[0173]      It is found that in general, peptides derived from the variant IL-7<br>
sequences have a significantly reduced ability to induce T-cell proliferation<br>
compared to the corresponding peptides derived from the wild-type human IL-7<br>
protein. Therefore, these variant peptide sequences are much poorer potential T-<br>
cell epitopes. Likewise, the bacterially produced variant IL-7 protein also has a<br>
reduced ability to induce T-cell proliferation than wild-type IL-7, indicating that<br>
these mutated regions may be significant contributors to the immunogenicity of<br>
prokaryotically produced IL-7.<br>
EXAMPLE 5: Analysis of IL-7 derived peptides as potential B-cell epitopes<br>
[0174]      For bacterially-produced, unglycosylated human IL-7 protein,<br>
sequences surrounding N-Iinked glycosylation sites of IL-7 may be recognized as<br>
"non-self by the human immune system, and elicit an antibody response.<br>
Essentially, to assess if these sequences represent linear B-cell epitopes,<br>
peptides spanning these sequences are used to immunize rabbits, and the<br>
reactivity of resulting antibodies toward bacterially-produced native human IL-7<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
45<br>
and denatured human IL-7 is tested. As a further control, a native eukaryotically-<br>
produced glycosylated huFc-IL-7 fusion protein is used.<br>
[0175]      Methods and materials to raise polyclonal antibodies against a specific<br>
peptide antigen in, for example, rabbits, and their subsequent purification are<br>
generally known to those skilled in the art, and references thereto may be found,<br>
for example, in: Antibodies: A Laboratory Manual, E. Hariow and D. Lane, Cold<br>
Spring Harbor Press.<br>
[0176]      Briefly, in one example, a peptide containing the core sequence<br>
FLKMNSTGD (SEQ ID NO:5), such as LRQFLKMNSTGDFDL[C] (SEQ ID NO:<br>
18) or [CjLRQFLKMNSTGDFDL (SEQ ID NO:19) is coupled via an added<br>
terminal cysteine to three different carrier proteins, keyhole limpet hemocyanin<br>
(KLH, EMD Biosciences, San Diego, CA), BSA (EMD Biosciences, San Diego,<br>
CA), and ovalbumin (Pierce, Rockford, IL) using a coupling agent such as SMCC<br>
(Pierce, Rockford, IL), and multiple rabbits are immunized by successive<br>
injections with each of the peptide conjugate in the presence of adjuvant. The<br>
immune response is boosted with further injections of one of the peptide-carrier<br>
conjugates at monthly intervals, and the resulting antiserum from each rabbit is<br>
affinity-purified over a Sulfo-Link column (Pierce, Rockford, IL) to which the<br>
peptide is coupled, and the antibody is further concentrated over a hydroxyapatite<br>
column (Bio-Rad Laboratories, Hercules, CA).<br>
[0177]      The purified antibodies are tested against bacterially-produced native<br>
human IL-7, denatured human IL-7 and a eukaryotically-produced glycosylated<br>
huFc-IL-7 fusion protein in an ELISA assay, following standard procedures.<br>
Briefly, ELISA plates, coated with the purified protein preparations, are incubated<br>
with the test antibody samples, the plates are washed, incubated with a<br>
secondary antibody such as horseradish peroxidase-coupled anti-rabbit IgG,<br>
washed again and incubated with a chromogenic substrate solution to indicate<br>
the concentration of bound antibody.<br>
[0178]      Similarly, antibodies are raised to other peptides encompassing the<br>
...MNSTG...(SEQ ID NO:20) glycosylation site (at Asn70) in human IL-7, or to<br>
peptides encompassing the ...LNCTG...(SEQ ID NO:21) glycosylation site (at<br>
Asn91). Using this approach, it is found that generally the denatured bacterially-<br>
produced human IL-7 is well recognized by the antibodies and that the<br>
glycosylated huFc-IL-7 fusion protein is not. Peptides that give rise to antibodies<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
46<br>
reacting with the bacterially-produced native human IL-7 protein indicate that a<br>
linear B-cell epitope at the glycosylation site is recognized. It is further found that,<br>
in a cell-based proliferation assay as described in Example 9, the antibodies<br>
raised against peptides of this Example have the effect of inhibiting IL-7-<br>
stimulated cell proliferation. This result indicates that these antibodies have<br>
neutralizing activity.<br>
EXAMPLE 6: Construction of human IL-7 variants that lack potential T-cell<br>
epitopes<br>
[0179]      Nucleic acids are constructed that encode versions of human IL-7<br>
variants either suitable for bacterial expression or suitable for eukaryotic<br>
expression, for instance as a fusion protein. For example, nucleic acids encoding<br>
a mature human IL-7 variant containing the substitutions K68D, M69D, I88T, and<br>
V96G (Del-IL-7 SEQ ID NO:16) are constructed, using standard methods familiar<br>
to those skilled in the art. Figure 11 shows an example of such a DNA sequence<br>
encoding a mature IL-7 variant, Del-IL-7, with codon substitutions of amino acid<br>
residues K68D, M69D, I88T, and V96G (SEQ ID NO:24).<br>
[0180]      For bacterial expression, the protein sequence of Del-IL-7 including a<br>
start methionine (bDel-IL-7 SEQ ID NO:17) is reverse-translated using a codon<br>
bias appropriate for optimal E. coli expression. The resulting nucleic acid<br>
sequence is further adapted to include desired (or to exclude undesired) features<br>
such as a stop codon or restriction sites, and sequences are added that facilitate<br>
cloning into a bacterial expression vector, for example an appropriate vector from<br>
the pET series (EMD Biosciences, San Diego, CA). The nucleic acid sequence is<br>
synthesized by total gene synthesis (Blue Heron Biotechnology, Bothell, WA) and<br>
inserted into the expression vector. An example of a DNA sequence encoding<br>
bDel-IL-7, codon-optimized for E. coli with codon substitutions of amino acid<br>
residues K68D, M69D, I88T, V96G, is shown in Figure 10 (SEQ ID NO:23).<br>
[0181]      For eukaryotic expression as a huFc-Dei-IL-7 fusion protein, the<br>
nucleic acid sequence of the mature human IL-7 is modified to incorporate<br>
codons for desired amino acid mutations of the invention as described above.<br>
(See e.g. SEQ ID NO:24). The sequence is further adapted to incorporate<br>
flanking sequences with unique restriction sites for insertion as a Xma I / Xho I<br>
fragment in-frame into a pdCs-huFc expression vector encoding the hinge, CH2<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
47<br>
and CH3 region of lgG1 (see Lo et a/., (1998), Protein Engineering 11:495), and<br>
is synthesized by total gene synthesis (Blue Heron Biotechnology, Bothell, WA).<br>
The synthetic Xma I / Xho I Del-IL-7 fragment is then cloned into the pdCs-huFc<br>
vector, yielding an expression plasmid encoding huFc-Del-lL-7. Other IL-7 and<br>
Fc-IL-7 variants of the invention can be produced by similar methods.<br>
[0182]      Specifically, nucleic acids encoding the human deimmunized Fc-IL-7<br>
fusion proteins huFcy2(h)(FN&gt;AQ)-(linker2)-IL-7(PNS) and huFcy2(h)(FN&gt;AQ)-<br>
(Iinker2)-IL-7(PNDS) were generated as follows. huFcy2(h)(FN&gt;AQ)-(linker2)-IL-<br>
7(PNS) is a human Fc-IL-7 fusion protein comprising the N-terminus of human IL-<br>
7 genetically fused to the C-terminus of a human lgG2 Fc domain with an lgG1<br>
hinge via a linker sequence GGGGSGGGG. The Fc portion contains the<br>
mutations Phe296Ala and Asn297Gln. The IL-7 portion contains the mutations<br>
F39P, F57N and L128S. huFcy2(h)(FN&gt;AQ)-(linker2)-IL-7(PNDS) is the same as<br>
huFcy2(hXFN&gt;AQ)-(linker2)-IL-7(PNS), but for containing an additional mutation<br>
in the IL-7 moiety, L77D. The sequence also contains codons for the mutation of<br>
the LSLS sequence near the C-terminus of the Fc portion to be replaced by<br>
ATAT. In addition, the nucleic acid sequence includes a codon to replace the C-<br>
terminal lysine of the Fc portion with an alanine residue.<br>
[0183]      A nucleic acid of the sequence presented in Figure 27 was<br>
synthesized de novo (Blue Heron Biotechnology, Bothell, WA), which encodes<br>
the linker sequence GGGGSGGGG followed by mature human IL-7 containing<br>
the amino acid substitutions F39P, F57N, and L128S (IL-7(PNS)) and which<br>
contains flanking restriction sites Xma I and Xho I at the 5'- and 3' ends,<br>
respectively. This purified Xma I / Xho I fragment was ligated to a likewise<br>
digested and purified vector fragment of the pdCs-huFc series, pdC10-<br>
huFcy2(h)(FN&gt;AQ), generating a plasmid encoding huFcy2(h)(FN&gt;AQ)-(linker2)-<br>
IL-7(PNS). Lo et a/., (1998), Protein Engineering 11:495. The coding sequence<br>
was ascertained by sequencing.<br>
[0184]      The further introduction of the substitution L77D into IL-7(PNS) was<br>
performed by standard PCR mutagenesis methods, using mutagenic primers<br>
M(s) (5'-TGACTTTGATGACCACCTGTTAAAAGTTTC-3' (SEQ ID NO: 50);<br>
mutated codon underlined) and M(a)<br>
(5'-AACAGGTGGICATCAAAGTCACCAGTGC-31) (SEQ ID NO:51). Briefly,<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
48<br>
separate PCR reactions were performed on a plasmid template containing<br>
(Iinker2)-IL-7(PNS), one with M(s) and the downstream primer<br>
5' - CTCGAGTCAGTGTTCTTTAGTGCCCATC - 3' (SEQ ID NO:52), the other<br>
with M(a) and the upstream primer<br>
5' - CCCGGGTGCTGGAGGTGGAGGATCAGGTG- 3' (SEQ ID NO:53), the PCR<br>
fragments were purified and combined as the template for a secound round of<br>
PCR using again the upstream primer<br>
5' - CCCGGGTGCTGGAGGTGGAGGATCAGGTG- 3'(SEQ ID NO:54) and the<br>
downstream primer 5' - CTCGAGTCAGTGTTCTTTAGTGCCCATC - 3'(SEQ ID<br>
NO:55). The resultant purified fragment was inserted into a TA cloning vector<br>
pCR2.1 ((Invitrogen, Carlsbad, CA), and its sequence was confirmed. An Xma I /<br>
Xho I fragment encoding (Iinker2)-IL-7(PNDS) was excised, and ligated to a<br>
likewise digested and purified vector fragment of the pdCs-huFc series, pdC10-<br>
huFcy2(h)(FN&gt;AQ), generating a plasmid encoding huFcy2(h)(FN&gt;AQ)-(linker2)-<br>
IL-7(PNDS).<br>
[0185]      Similarly, plasmids encoding variants of these fusion proteins differing<br>
in the Fc moiety are obtained; for example a plasmid encoding<br>
huFcy2(h)(iinker2)-IL-7(PNDS) is obtained by ligating an Xma I /Xho I fragment<br>
encoding (Iinker2)-IL-7(PNDS) to a likewise digested and purified vector fragment<br>
of the pdCs-huFc series, pdC10-huFcy2(h).<br>
EXAMPLE 7: Expression and purification of lL-7 variants<br>
[0186]      For eukaryotic expression of the huFc-Del-IL-7 fusion protein,<br>
electroporation is used to introduce the DNA encoding the fusion protein into a<br>
mouse myeloma NS/0 cell line. To perform electroporation NS/0 cells are grown<br>
in Dulbecco's modified Eagle's medium supplemented with 10% heat-inactivated<br>
fetal bovine serum, 2 mM glutamine and penicillin/streptomycin. About 5x106<br>
cells are washed once with PBS and resuspended in 0.5 ml PBS. 10 ug of<br>
linearized plasmid DNA for huFc-Del-IL-7 is then incubated with the cells in a<br>
Gene Pulser Cuvette (0.4 cm electrode gap, BioRad) on ice for 10 min.<br>
Electroporation is performed using a Gene Pulser (BioRad, Hercules, CA) with<br>
settings at 0.25 V and 500 	μF. Cells are allowed to recover for 10 min on ice,<br>
after which they are resuspended in growth medium and plated onto two 96 well<br>
plates.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
49<br>
[0187]      Stably transfected clones are selected by their growth in the presence<br>
of 100 nM methotrexate (MTX), which is added to the growth medium two days<br>
post-transfectiom The cells are fed every 3 days two to three more times, and<br>
MTX-resistant clones appear in 2 to 3 weeks. Supematants from clones are<br>
assayed by anti-Fc ELISA to identify clones that produced high amounts of the IL-<br>
7 fusion protein. High producing clones are isolated and propagated in growth<br>
medium containing 100 nM MTX. Typically, a serum-free growth medium, such<br>
as H-SFM or CD medium (Life Technologies), is used.<br>
[0188]      A standard purification of Fc-containing fusion proteins is performed<br>
based on the affinity of the Fc protein moiety for Protein A. Briefly, NS/0 cells<br>
expressing the fusion protein, such as huFc-Del-IL-7, are grown in tissue culture<br>
medium and the supernatant containing the expressed protein is collected and<br>
loaded onto a pre-equilibrated Fast Flow Protein A Sepharose column. The<br>
column is then washed extensively with buffer (such as 100 mM sodium<br>
phosphate, 150 mM NaCI at neutral pH). Bound protein is eluted at a low pH (pH<br>
2.5 - 3) in same buffer as above and fractions are immediately neutralized.<br>
[0189]      Bacterial expression and purification of bDel-IL-7 is performed<br>
essentially as described by Cosenza et al. for bacterially-produced IL--7 (see<br>
Cosenza et al., (1997) JBC, 272:32995). In essence, bDel-IL-7 is isolated from<br>
inclusion bodies, denatured and refolded. Briefly, bacterial expression cultures<br>
transformed with the expression vector encoding, for example, bDel-IL-7 are<br>
grown to mid-log phase and recombinant protein expression is induced.<br>
Following induction, the bacteria are harvested and lysed by sonication, and<br>
inclusion bodies are isolated in buffer A (50 mM Tris HCI (7.5), 5 mM EDTA, 20%<br>
sucrose). After extensive washes, the inclusion bodies are resuspended in a<br>
guanidine denaturation buffer (50 mM Tris-HCI (pH8.0), 5 M guanidine HCI, 5 mM<br>
EDTA), briefly sonicated and reduced in 6 mM DTT. Denatured bDel-IL-7 protein<br>
is then further purified by denaturing size exclusion HPLC. The protein is then<br>
refolded in refolding buffer (50 mM glycine, 30 mM NaOH, 0.4 M L-arginine, 1<br>
mM DTT, pH 10), dialyzed into a phosphate buffer, and further purified by size<br>
exclusion HPLC.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
50<br>
EXAMPLE 8: Biochemical analysis of IL-7 variants<br>
[0190] The effect of the introduced mutations on the integrity of IL-7 proteins<br>
is assessed by routine reducing and non-reducing SDS-PAGE analysis and size<br>
exclusion chromatography.<br>
[0191]       For example, the fusion protein huFc-Del-IL-7, expressed from NS/0<br>
cells, is captured on Protein A Sepharose beads (Repligen, Needham, MA) from<br>
the tissue culture medium into which it is secreted, and eluted by boiling in protein<br>
sample buffer, with or without a reducing agent such as /?-mercaptoethanol. The<br>
sample is fractionated by SDS-PAGE and the protein bands are visualized by<br>
Coomassie staining. It is expected that a fusion protein containing IL-7 mutations<br>
that sufficiently interfere with proper folding is more likely to show degradation<br>
products by SDS-PAGE.<br>
[0192]       Purified huFc-Del-IL-7 is also analyzed by size exclusion<br>
chromatography (SEC) to assess the extent to which the fusion protein is<br>
aggregated. Briefly, the cell culture supernatant is loaded onto a pre-equilibrated<br>
Fast-Flow Protein A Sepharose column, the column is washed extensively in a<br>
physiological buffer (such as 100 mM Sodium Phosphate, 150 mM NaCI at<br>
neutral pH), and the bound protein is eluted at about pH 2.5 to 3 in same salt<br>
buffer as above. Fractions are immediately neutralized, peak fractions are<br>
pooled, and an aliquot is fractionated over an analytical SEC column.<br>
EXAMPLE 9: In vitro activity of IL-7 variants<br>
[0193]      To determine whether the IL-7 variants containing mutations of the<br>
invention retain their cytokine activity in vitro cellular proliferation bioassays are<br>
performed. Human PBMC (Peripheral Blood Mononuclear Cells) are activated by<br>
PHA-P to produce cells which are responsive to IL-7. Proliferation is measured in<br>
a standard thymidine incorporation assay.<br>
[0194]       For example, the cytokine activity of huFc-Del-IL-7 and bDel-IL-7 is<br>
determined. Briefly, PBMC's are first incubated for five days with 10<br>
microgram/ml PHA-P, cells are washed and then incubated in medium with huFc-<br>
Del-IL-7 or bDe!-IL-7, prepared as a dilution series, for a total of 48 hours. During<br>
the final 12 hours, the samples are pulsed with 0.3 JJC\ of [methyl-3H]thymidine<br>
(Dupont-NEN-027). Cells are then washed extensively, harvested and lysed onto<br>
glass filters. 3H-thymidine incorporated into DNA is measured in a scintillation<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
51<br>
counter. As a standard, wild type hulL-7 protein, obtained from R&amp;D Systems<br>
(Minneapolis, MN), or obtained from the National Institute for Biological<br>
Standards and Control (NIBSC), is assayed.<br>
[0195]      An ED50 value of cell proliferation for huFc-Del-IL-7 or bDel-lL-7 is<br>
obtained from plotting a dose response curve according to standard techniques,<br>
and determining the protein concentration that results in half-maximal response.<br>
Example 10: Induction of anti-human IL-7 antibodies in monkeys by wild-type IL-<br>
7 and IL-7 variants<br>
[0196]      It is known that bacterial-derived wild-type human IL-7 administered to<br>
monkeys often results in neutralizing anti-human IL-7 antibody titers (Storek et<br>
a/., (2003), Blood, 101:4209; Fry etai, (2003), Blood, 101:2294). Thus, the<br>
propensity of prokaryotically produced variant IL-7 and wild-type IL-7 proteins, as<br>
well as eukaryotically produced fusion proteins containing wild-type or variant IL-7<br>
polypeptides, to induce neutralizing antibodies in nonhuman primates is<br>
assessed. In a typical experiment, rhesus macaques are injected with 40//g/kg<br>
of the protein samples subcutaneously once a day for four weeks. For example,<br>
the protein samples are commercially available prokaryotically-produced IL-7<br>
(PeproTech, Rocky Hill, NJ), the prokaryotically produced variant IL-7(K68D,<br>
M69D, I88T, V94G), and the equivalent Fc-IL-7 fusion proteins produced in a<br>
mammalian expression system. At regular intervals, serum is obtained from the<br>
animals, and serum concentrations of antibodies against human IL-7 are<br>
measured by ELISA using human IL-7 coated 96 well plates (Nunc, Naperville,<br>
IL). Typically, serial dilutions of each serum sample are added to each well in<br>
triplicate for two hours, washed with 0.05% Tween (Tween 20) in PBS and<br>
blocked with 1% BSA/1% goat serum in PBS. To each sample a horseradish<br>
peroxidase-conjugated anti-macaque IgG is added (1:60,000 in sample buffer),<br>
incubated at 37°C for 2 hr, and the plate is washed 8 times with 0.05% Tween in<br>
PBS. Samples are then assayed using the colorimetric substrate solution OPD<br>
(o-phenylenediamine dihydrochloride) by measuring the OD at 490 nm,<br>
subtracting the background OD reading at 650 nm.<br>
[0197]      It is found that prokaryotically produced wild-type IL-7 protein indeed<br>
gives rise to high anti-IL-7 antibody titers. In contrast, the antibody titers of the<br>
prokaryotically produced variant IL-7 gives rise to significantly lower titers of anti<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
52<br>
IL-7 antibodies. It is also found that the differences in the levels of anti IL-7<br>
antibody titers produced by animals administered mammalian produced wild-type<br>
and variant Fc-IL-7 fusion proteins, (with mutations around the N-linked<br>
glycosylation sites) are not as pronounced. This result may indicate that the lack<br>
of glycosylation at these sites in the prokaryotically produced proteins contributes<br>
to the immunogenicity of these proteins.<br>
Example 11: Acute tolerability of Fc-IL-7 in immunocompetent mice<br>
[0198]     The Fc-IL-7 fusion protein huFcK2(h)(N&gt;Q)(linker2)-hulL-7 was<br>
prepared according to the method described in Example No. 6, purified, then<br>
formulated in 50mM phosphate, 150mM sodium chloride pH 7.00, 0,05% (v/v)<br>
Tween 80. The protein concentrations of diluted solutions were determined using<br>
the absorbance at 280nm and the theoretical extinction coefficient of 0.98<br>
mg/OD280, based on the known protein sequence. For dosing mice, an aliquot of<br>
each sample was removed from stock vials and diluted with 0.9% saline within<br>
one hour of dosing.<br>
[0199]      C57B1/6 mice (Charles River Laboratories, Wilmington, MA), 17<br>
weeks of aged were divided into groups of 2 mice each and were administered<br>
the Fc-IL-7 fusion proteins subcutaneously for 5 consecutive days. Groups<br>
received dosages of either 0.5, 5.0, 25 mg/kg or the vehicle control each day. All<br>
mice survived the treatment through day 7, at which point the mice were<br>
sacrificed.<br>
[0200]      Fc-IL-7 plasma levels were determined by obtaining blood samples<br>
from the retro-orbital sinus 6 hours after dosing on days 0, 2, 4, and 7. Blood<br>
samples were collected in tubes containing heparin to prevent clotting. Cells<br>
were removed by centrifugation and the concentration of intact Fc-IL-7 fusion<br>
protein in the plasma was measured using standard ELISA procedures. Plasma<br>
levels of Fc-IL-7 in //g///l for test mice are shown in Figure 36. The plasma of<br>
mice showed a dose dependent increase in Fc-IL-7 concentration at all time<br>
points tested and further increased following each dose. However, as shown in<br>
Figure 37, the magnitude of the increase lessened after each dosing.<br>
[0201]      The functional activity of Fc-IL-7 was confirmed by measuring<br>
increases in B cells and T cells on day 7 following the initiation of dosing. Since<br>
IL-7 boosts the production of immune effector cells such as B cells and T cells,<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
53<br>
the cellularity and weight of the spleen is expected to increase. Mice were<br>
sacrificed on day 7 and organs were removed and weighed. Figure 38 shows the<br>
average organ weights on day 7. As expected, spleen weight increased 3 to 5<br>
fold 1 week after the initial dose. Lung weights increased 2 fold following the 2<br>
    higher doses of Fc-IL-7 due to lymphotcytic infiltration. No weight changes were<br>
observed in the kidney or liver.<br>
[0202]      The response of B cells (CD19+, CD4+, CD8+ and granulocytes (Gr-<br>
I	+)) in all groups was observed on day 7. Fig. 39 shows the frequency of Gr-1 +<br>
cells in the peripheral blood of two mice in each group. As the data shows,<br>
    granulocytes were generally unresponsive to Fc-IL-7. Fig. 40 shows the<br>
frequency of CD19+ cells in the peripheral blood of two mice in each group. Fig.<br>
41 shows the frequency of CD4+ cells in the peripheral blood of two mice in each<br>
group, while Fig. 42 shows the frequency of CD8+ cells in the peripheral blood of<br>
two mice in each group. The increases in B cell (Fig. 40 and T cell (Figs. 41 and<br>
    42 numbers were maximal for the 5 mg/kg dosage group with each mouse tested<br>
showing significantly increased T cell and B cell numbers over the control group<br>
and the 0.5 mg/kg dosage group. However, T cell numbers either declined or<br>
increased for mice in the 25 mg/kg dosage group. All measurements of cells are<br>
shown in cells per/;L of blood.<br>
Example 12. Assessment of Human Fc-IL-7 Activity<br>
[0203]      The biological activity of the Fc-IL-7 fusion protein tested in Example<br>
11	was measured by tritiated thymidine uptake in a standard cell proliferation<br>
assay using peripheral blood mononuclear cell (PBMC) PHA blasts according to<br>
the method described in Yokota et al, (1986), Proc. Natl. Acad. Sci. USA,<br>
83:5894; and Stern et al., (1990), Proc. Natl. Acad. Sci. USA, 87:6808-6812, with<br>
human IL-7 used as a standard. As shown in Figure 43, cellular proliferation as<br>
measured by the uptake of tritiated thymidine for the Fc-IL-7 molecule is similar to<br>
that of the standard NIBSC human IL-7 (World Health Organization), indicating<br>
that the activity of the Fc-iL-7 molecule is similar to wild-type human IL-7.<br>
Equivalents<br>
[0204]      The invention may be embodied in other specific forms without<br>
departing from the spirit or essential characteristics thereof. The foregoing<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
54<br>
embodiments are therefore to be considered in all respects illustrative rather than<br>
limiting on the invention described herein. Scope of the invention is thus<br>
indicated by the appended claims rather than by the foregoing description, and all<br>
changes which come within the meaning and range of equivalency of the claims<br>
are intended to be embraced therein.<br><br>
WO 2006/061219	PCT/EP2005/013145<br>
55<br>
WE CLAIM:<br>
1. A polypeptide comprising a modified human IL-7 molecule or an active portion thereof having a T-cell epitope modified to reduce an anti-IL-7 T-cell response, said polypeptide further comprising an Fc portion of an immunoglobulin molecule fused via its C-terminal to the N-terminal of said modified IL-7 molecule, wherein said fused Fc-IL7 molecule is selected from the group consisting of:<br>
(i) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 31; (ii) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L77D, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D L128S) is IL-7 containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 32; (iii) huFcy2(h) - L - IL-7 (F39P, F57N, L77D, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D, L128S) is IL-7<br><br>
containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 33; and<br>
(iv) huFcy2(h) - L - IL-7 (F39P, F57N, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 34.<br>
2.	A polypeptide as claimed in claim 1, wherein said immunoglobulin molecule is a human<br>
immunoglobulin.<br>
3.	A polypeptide as claimed in claim 1 or 2, wherein said immunoglobulin molecule is IgG2.<br>
4.	A nucleic acid molecule encoding a polypeptide as claimed in any one as claimed in claims 1-3.<br><br>
5.	An expression vector comprising a nucleic acid molecule as claimed in claim 4.<br>
6.	A prokaryotic cell comprising an expression vector as claimed in claim 5.<br><br>
A polypeptide comprising a modified human IL-7 molecule or an active portion thereof having a T-cell epitope modified to reduce an anti-IL-7 T-cell response, said polypeptide further comprising an Fc portion of an immunoglobulin molecule fused via its C-terminal to the N-terminal of said modified IL-7 molecule, wherein said fused Fc-IL7 molecule is selected from the group consisting of:<br>
(i) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 31;<br>
(ii) huFcy2(h)(FN&gt;AQ) - L - IL-7(F39P, F57N, L77D, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 comprising the mutations F296A and N297Q, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D L128S) is IL-7 containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 32;<br>
(iii) huFcy2(h) - L - IL-7 (F39P, F57N, L77D, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2, L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L77D, L128S) is IL-7 containing the mutations F39P, F57N, L77D and L128S; said molecule having the sequence as depicted in Fig. 33; and<br>
(iv) huFcy2(h) - L - IL-7 (F39P, F57N, L128S),<br>
wherein Fcy2(h) is a human Fc portion with a hinge of lgG1 and CH2 and CH3 domains of lgG2 L is a linker with the sequence GGGGSGGGG, and IL-7(F39P, F57N, L128S) is IL-7 containing the mutations F39P, F57N, and L128S; said molecule having the sequence as depicted in Fig. 34.<br></j></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1jb3JyZXNwb25kZW5jZSBvdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-correspondence others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1kZXNjcmlwdGlvbiBjb21wbGV0ZS5wZGY=" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-description complete.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1mb3JtIDIucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1ncGEucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHB1YmxpY2F0aW9uLnBkZg==" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-international publication.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1pbnRlcm5hdGlvbmFsIHNlYXJjaCByZXBvcnQucGRm" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-international search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1wY3QgcmVxdWVzdC5wZGY=" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-pct request.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDI1MTYta29sbnAtMjAwNy1zZXF1ZW5jZSBsaXN0aW5nLnBkZg==" target="_blank" style="word-wrap:break-word;">02516-kolnp-2007-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgyMC0wNi0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(20-06-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgyMC0wNi0yMDEyKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(20-06-2012)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgyNC0wNy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(24-07-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgyNC0wNy0yMDEyKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(24-07-2012)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1BQlNUUkFDVC5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-ABSTRACT.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1BTUFOREVEIENMQUlNUy5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-AMANDED CLAIMS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1DT1JSRVNQT05ERU5DRS5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-CORRESPONDENCE.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1ERVNDUklQVElPTiAoQ09NUExFVEUpLnBkZg==" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-DESCRIPTION (COMPLETE).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1FWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-EXAMINATION REPORT REPLY RECEIVED.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1GT1JNLTEtMS5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-FORM-1-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1GT1JNLTEucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-FORM-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1GT1JNLTIucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-FORM-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1GT1JNLTMucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-FORM-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1PVEhFUlMgUENUIEZPUk0ucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-OTHERS PCT FORM.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1PVEhFUlMucGRm" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-OTHERS.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1QQS1DRVJUSUZJRUQgQ09QSUVTLnBkZg==" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-PA-CERTIFIED COPIES.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNy0xLnBkZg==" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-PETITION UNDER RULE 137-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1LT0xOUC0yMDA3LSgzMC0wMy0yMDEyKS1QRVRJVElPTiBVTkRFUiBSVUxFIDEzNy5wZGY=" target="_blank" style="word-wrap:break-word;">2516-KOLNP-2007-(30-03-2012)-PETITION UNDER RULE 137.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUxNi1rb2xucC0yMDA3LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2516-kolnp-2007-form-18.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="254409-growth-factor-binding-compounds-attached-to-a-calixarene.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="254411-a-cell-header-compression-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>254410</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2516/KOLNP/2007</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>44/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>02-Nov-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>31-Oct-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>06-Jul-2007</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MERCK PATENT GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>FRANKFURTER STRASSE 250 64293 DARMSTADT</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>GILLIES, STEPHEN, D</td>
											<td>47 SWANSON LANE, CARLISLE, MA 01741</td>
										</tr>
										<tr>
											<td>2</td>
											<td>WAY, JEFFREY, C</td>
											<td>108, FAYERWEATHER STREET, UNIT 2, CAMBRIDGE, MA 02138</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/54</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/013145</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-12-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/634,470</td>
									<td>2004-12-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/254410-il-7-variants-with-reduced-immunogenicity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:42:42 GMT -->
</html>
