<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/254638-a-peptide-and-a-method-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:27:35 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 254638:A PEPTIDE AND A METHOD THEREOF</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">A PEPTIDE AND A METHOD THEREOF</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Disclosed herein is a small peptide, LaR2C, corresponding to the C terminus of RRM2 of the human La protein that binds to the IRES element of hepatitis C virus RNA and its derivatives. This invention demonstrates that human La protein interacts with the HCV IRES element both in vitro and in vivo and also shown that this interaction enhances the efficiency of viral RNA translation (Pudi et al, J of Biol Chem, 2003). La protein has three putative RNA recognition motifs (RRM1-3). It has been established that RRM2 binds with high affinity around the GCAC sequence near the initiator AUG and the binding induces a conformational change in the HCV IRES which is critical for the internal initiation of translation (Pudi et al, J of Biol Chem, 2004). ABSTRACT An improved process for preparing isoxazolyl penicillin derivatives of Formula I, and their pharmaceutically acceptable salts.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td> <br><br>
This invention relates to a small peptide, LaR2C, corresponding to the C terminus of RRM2 of the human La protein that binds to the IRES element of hepatitis C virus RNA, uses of the peptide in preparing nucleic acid sequence, a polynucleotide, a vector and protein and also novel antiviral agents comprising the said peptide.<br>
BACKGROUND OF THE INVENTION:<br>
Hepatitis C virus (HCV), a member of the Flaviviridae family, is an enveloped positive-sense, single-stranded RNA virus. The 9.6 kb long genome encodes a single polyprotein of about 3,000 amino acids. The polyprotein is processed by host cell and viral proteases into three major structural proteins and several non-structural proteins necessary for viral replication. HCV causes a variety of liver-diseases in humans including liver cirrhosis and hepatocellular carcinoma. It is estimated that about 3% of the world population is infected with HCV and about 85% of infected individuals develop chronic infection.<br>
Translation initiation of HCV occurs in cap-independent maimer wherein the ribosomes are recruited onto an internal ribosome entry site (IRES) located mostly within the 5' untranslated region (UTR) and extending a few nucleotides into the coding region (2). HCV IRES has been shown to form three complex stem-loops and a pseudoknot, which encompasses the initiator AUG codon. Although the HCV IRES binds to the 40S ribosomal subunit specifically and stably even in absence of any initiation factors, efficient translation requires canonical initiation factors like eIF2 and eIF3 and other non-canonical trans-acting cellular proteins including polypyrimidine-tract binding protein (PTB), La autoantigen, poly (rC) binding protein (PCBP), heterogeneous nuclear ribonucleoprotein L (p68). Recently, binding of a 25kDa cellular protein (p25) to HCV IRES has been shown to be important for the efficient translation initiation. p25 was originally suggested to be ribosomal protein S9 but later identified as rpS5.<br>
Human La protein is known to interact with HCV IRES and stimulate translation initiation both in vitro and in vivo (3-6). La protein has been shown to specifically interact with both the 5' and 3'UTR of hepatitis C virus RNA. Sequestration of La in RRL inhibits HCV IRES mediated translation, which can be rescued by exogenous addition of purified La protein. Due to the critical role played by the La protein in HCV<br><br>
IRES translation, the disruption of its interaction with HCV IRES is an attractive target for inhibiting HCV IRES activity. A 60-nucleotide RNA (I-RNA) from the yeast Saccharomyces cerevisiae which preferentially blocked HCV and Poliovirus IRES mediated translation appeared to inhibit the translation by virtue of its ability to bind La protein (4). Recently, it has been shown that a synthetic peptide corresponding to N-terminal 'La motif of human La autoantigen inhibits HCV IRES mediated translation possibly by binding to other essential cellular proteins (5).<br>
LIMITATIONS OF THE PRIOR ART:<br>
HCV establishes persistent liver infection leading to chronic hepatitis, liver cirrhosis and hepatocellular carcinoma. The translation of the positive stranded genomic RNA to produce the viral proteins required for replication is an early obligatory step of the infection process. The translation initiation of the uncapped viral RNA is mediated by the interaction of ribosome and cellular proteins with an internal ribosome entry site (IRES) located within the 5'untranslated region (5'UTR). The translation of the viral RNAs is believed to be controlled by binding of some trans acting cellular proteins to highly structured cis acting RNA elements. Human La autoantigen (originally detected in patients with Lupus Erythematosus) was one of the first IRES trans acting factors (ITAFs) to be identified to interact with Poliovirus IRES element.<br>
Current treatment options in treating HCV involve a-interferon alone or in combination with ribavirin. However, these treatments fail to achieve sustained virological response in majority of patients thus emphasizing the need for novel therapeutic approaches to combat HCV infection (1). Another major limitation is the stability of the peptide inside the cells which cannot be maintained by anything disclosed in the prior art.<br>
SUMMARY OF THE INVENTION:<br>
This invention demonstrates that human La protein interacts with the HCV IRES element in vivo and also shown that this interaction enhances the efficiency of viral RNA translation (
 <br>
have shown that RRM2 binds with high affinity around the GCAC sequence near the initiator AUG and the binding induces a conformational change in the HCV IRES, which is critical for the internal initiation (7).<br>
Since the mechanism ribosome assembly during internal initiation is fundamentally different from the 5'capdependent translation of cellular mRNAs (8-9), it is exploited as a target for developing antiviral agents. In order to develop effective antiviral agent, we have targeted the La protein interaction with the HCV IRES RNA.<br>
This invention demonstrates a novel approach to inhibit HCV IRES mediated translation using small peptide derived from the C terminus region of RRM2 of La protein. We have shown that a small peptide, LaR2C (24-aminacid long) is capable of binding to IRES element of hepatitis C virus RNA and significantly competes out the interaction of cellular La protein to HCV RNA. The peptide has been shown to prevent the ribosome assembly on HCV IRES and thus effectively block the translation of the viral RNA<br>
This invention thus provides a small peptide, LaR2C, corresponding to the C terminus of RRM2 of the human La protein binds to the IRES element of hepatitis C virus RNA. The LaR2C peptide competes with the cellular La protein for binding to HCV IRES RNA and act as dominant negative. The LaR2C peptide effectively blocks the ribosome assembly on the HCV RNA. The peptide inhibits the internal initiation of translation of hepatitis C virus in vitro and in vivo. The interaction between the ribosome and HCV RNA could be targeted for designing novel antiviral agents. The sequence of the LaR2C peptide {KYKETDLULFKDDYFAKKNEERK) or its derivative that blocks ribosome interaction with the HCV-IRES and inhibit viral RNA translation.<br>
This invention also provides a novel antiviral agents comprising the novel peptide LaR2C corresponding to the C terminus of RRM2 of the human La protein that binds to the IRES element of hepatitis C virus RNA and that effectively blocks the ribosome assembly on the HCV RNA and also the use of LaR2C peptide for treatment of any viral infection and particularly Hepatitis C viral infection.<br><br>
DETAILED DESCRIPTION OF THE INVENTION: EXPERIMENTAL APPROACH &amp; RESULTS<br>
a.	RRM2 of La protein binds to HCV IRES through its C-terminal residues<br>
Previously, it has been shown that RRM2 of La protein binds to HCV IRES with high affinity. To precisely identify the region that is important for the binding, two deletion constructs of La-RRM2 (LalOO-180 and Lal20-208) with deletions of 20 aminoacids from N-terminus and 28 aminoacids from C-terminal region were generated (figure 1A). The over-expressed and purified proteins were analyzed by gel electrophoresis followed by silver staining (figure IB) and used to study their ability to bind HCV IRES RNA using filter-binding assay. [32P] labeled HCV IRES RNA was incubated with increasing concentration of La-RRM2 (La 100-208), LalOO-180 or Lal20-208 proteins in RNA-binding buffer. The RNA-protein complexes were bound to nitrocellulose filters and washed with binding buffer to remove unbound RNA. The counts retained were plotted against the protein concentration to obtain saturation curve. The results showed that Lal20-208 retained the ability to bind HCV IRES RNA with an affinity of 0.16^iM which was comparable to the affinity of La-RRM2 (0.14|iM), while the RNA binding ability was drastically affected in Lai 00-180, where the C-terminal 28 aminoacids were deleted (figure 1C). The results indicated that the C-terminal aminoacids of La-RRM2 contribute to the binding with HCV IRES RNA.<br>
b.	Peptide LaR2C derived from the C terminus of La-RRM2 is capable of binding to<br>
HCV IRES RNA<br>
Based on the filter-binding assay results and the information obtained from the computational modeling (6) a synthetic peptide (LaR2C) of 24 amino acids (Lai 74-197), corresponding to C-terminus of La-RRM2, was custom synthesized and tested for its ability to bind to HCV IRES using filter-binding assay. A non-specific peptide of similar length was used as a negative control. The integrity of both the peptides, the La-peptide (LaR2C) and the nonspecific peptide (NSP), were checked by gel electrophoresis followed by silver staining (figure 2A). The filter-binding results showed that LaR2C retained the ability to bind to HCV IRES with an affinity of 75^iM, although, the affinity<br><br>
was much lower compared to that of La protein (0.12^iM). The non-specific peptide, NSP, did not show any binding to the RNA (figure 2B). The results were further confirmed using UV cross-linking assay where increasing concentration of LaR2C or NSP were used to bind to [32P] labeled HCV IRES RNA. After the removal of unbound RNA by RNase A treatment, the protein-nucleotidyl complexes were subjected to SDS-15%PAGE followed by phosphor imaging analysis. LaR2C showed efficient binding to HCV IRES RNA in a dosage dependent manner (Figure 2C, lanes 2-4), while the NSP of similar length did not show any binding (Figure 2C, lanes 5-7). To study the specificity of the interaction further, competition UV cross-linking assay was performed where the binding of LaR2C with [32P] HCV IRES was competed with 100 and 200 fold molar excess of unlabeled HCV IRES RNA (self RNA) or with mutant IRES RNA, M2, where GCAC residues in SL IV have been substituted with ACCG (7). The results showed that the unlabeled HCV wild-type IRES RNA efficiently competed out the binding (Figure 2D, lanes 3-4) confirming the specificity of the interaction of LaR2C with HCV IRES RNA. Even 100-200 fold excess of M2 RNA failed to compete significantly with the [32P] HCV IRES RNA for binding to LaR2C (Figure 2D, lanes 5-6) clearly indicating that the LaR2C specifically binds to HCV IRES RNA and perhaps at the same site within the SL IV region where La-RRM2 has been shown to interact.<br>
c. LaR2C binds to HCV IRES in SL IV region similar to La-RRM2<br>
To further confirm the region on HCV IRES RNA where LaR2C binds, primer extension-inhibition assay (toe-printing) was performed using HCV IRES RNA in absence or presence of increasing concentrations of the peptide LaR2C. The full-length La protein and also the RRM2 protein were included in the assay. Increasing concentration of LaR2C or the purified recombinant proteins were incubated with 5 pmoles of in vitro transcribed HCV IRES RNA. To this complex, [32P] end-labeled primer complimentary to the 3' end of the HCV-383 was added and extended using AMV-RT. The resulting extended products were analyzed on a 6%polyacrylamide-7M urea denaturing gel. For precise mapping of the contact points, a DNA sequencing reaction corresponding to the HCV 383 RNA and using the same end labeled primer was electrophoresed alongside. The results demonstrated specific RT pauses (toe-prints) with the addition of increasing<br><br>
concentrations of the proteins (Figure 3) indicating possible binding sites. Two specific toe-prints corresponding to C334 and A342 were observed around the initiator AUG (iAUG) in SL IV region, which showed increase in band intensity with the addition of increasing concentrations of La Ml-length and RRM2 protein (Figure 3, panel A).<br>
When LaR2C peptide was added, similar toe-prints (A342 and C334) were obtained around the iAUG within SL IV region (Figure 3, panel B). However, no significant increases in toe-prints were observed when similar concentration of NSP peptide was used (Figure 3, panel B). The results indicate that the LaR2C peptide might have contact points within SLIV region at similar positions as observed with the full-length La and RRM2 protein.<br>
d. LaR2C competes with binding of full-length La protein to the HCV IRES RNA<br>
We have shown earlier that RRM2 of human La protein binds to the HCV IRES RNA near initiator AUG (6-7). Since, we have also observed that LaR2C binds to SLIV region of the HCV IRES RNA near the initiator AUG, we were interested to investigate whether LaR2C can compete with the full-length protein for binding to the HCV IRES RNA and if so, what could be the possible consequences. To address this, competition UV cross-linking experiment was performed with purified recombinant La protein and [32P] HCV IRES RNA in absence or presence of increasing concentration of LaR2C peptide or NSP. The results showed that the binding of La protein to HCV IRES RNA was partially competed out (upto 40%) with the addition 20|*M and 40|iM of LaR2C peptide in the binding reaction (Figure 4, lanes 3-4). At further high concentration of the peptide (60^iM) the competition was much more pronounced (upto 70%) (Figure 4A, lane 5). However, the NSP did not show a similar effect (Figure 4A, lanes 6-8). The result is consistent with the observation that the peptide LaR2C encompassing region of the full-length La protein contributed significantly to its binding to HCV IRES RNA and when added in trans, this region can significantly compete with La protein binding to HCV IRES RNA.<br>
To further verify whether LaR2C peptide can compete with full-length La protein in the context of cytoplasmic extract, UV cross-linking experiment was performed using [ P]<br><br>
HCV IRES RNA and He La cell extract in absence or presence of increasing concentrations of LaR2C. The results demonstrated that in presence of increasing concentration of LaR2C, the binding of a 52kDa polypeptide to the [32P] HCV IRES RNA was found to be drastically reduced (figure 4B, lanes 2-4). Immunoprecipitation of the UV cross-linked protein-nucleotide complex using anti-La antibody showed a 52kDa cross-linked band from He La cell extract, corresponding in size to the 52kDa protein band observed in the above UV cross-linking experiment (data not shown). Results suggest, the 52kDa cross-linked band, which showed decreased binding with HCV IRES upon addition of the LaR2C peptide is indeed the La protein. Interestingly, in presence of LaR2C peptide significant changes in band intensities were observed in the binding of some other cellular proteins to HCV IRES RNA (as indicated with asterisks in figure 4). The sizes of some of them were found to be similar to that of eIF3 subunits (pi70, p66) that were reported to interact with the HCV IRES RNA. From the above observation it appears that the LaR2C peptide can compete with the full-length La protein for binding to the HCV-IRES RNA even in presence of other transacting factors in the He La cytoplasmic extract.<br>
e. LaR2C inhibits HCV IRES mediated translation in vitro<br>
Human La autoantigen has been shown to enhance HCV IRES mediated translation. Since the RNA-protein interaction studies revealed that the peptide LaR2C binds to HCV IRES near iAUG and competes with the binding of full-length La protein to HCV RNA, it was interesting to study whether LaR2C has any effect on HCV IRES mediated translation. For this purpose, monocistronic RNA, HCV-GFP containing the reporter gene GFP downstream of HCV IRES was translated in vitro in rabbit reticulocyte lysate using [35]S methionine, in absence or presence of increasing concentration of LaR2C (20^M, 40JIM and 60JIM). The cap-independent initiation of translation, occurring internally from HCV IRES resulted in the synthesis of GFP. The translated GFP was analyzed by electrophoresis followed by phosphorimaging analysis and the band intensities were quantitated by densitometry. It was observed that addition of the peptide LaR2C lead to a significant decrease in the HCV IRES activity (80 to 95%) in a dosage dependent manner leading to reduced synthesis of GFP (Figure 5A, lanes 2-4). However,<br><br>
addition of similar concentrations of NSP to the translation reactions did not show any significant effect (Figure 5A, lanes 5-7). Also, the peptide LaR2C didn't affect the translation when capped GFP RNA was used. (Figure 5B, lanes 2-4), indicating the specificity of the peptide to selectively inhibit HCV IRES mediated translation. Furthermore, addition of similar concentrations of LaR2C peptide didn't show inhibition of HAV IRES (Fig 5C). However, addition of increasing concentration (2pjxM, 40|iM, 60^M) of LaR2C peptide showed only up to 44-49% inhibition of the PV IRES mediated translation of the reporter gene firefly luciferase. The results suggest that the LaR2C peptide might be more effective in inhibiting HCV IRES compared to other IRES function.<br>
f. LaR2C inhibits HCV IRES mediated translation in vivo<br>
As the LaR2C showed dominant negative effect on HCV IRES mediated translation in vitro, we wanted to investigate whether the same peptide can inhibit the HCV IRES mediated translation in vivo as well As a first step, sequence coding for the peptide LaR2C was cloned into bacterial expression vector, pTAT which would produce the peptide as a fusion of HIV-TAT peptide that has the property to internalize into mammalian cells when supplied exogenously into the medium (10) (Figure 6A). Huh7 monolayer cells were first transfected with the pcDNA3-HCV bicistronic construct and the transfectants were selected and maintained in Neomycin containing medium. These selected cells, which constitutively express both the reporter genes, Flue and Rluc from the HCV bicistronic construct (Fig 6B), were washed and layered with medium containing lOOnM of TAT-LaR2C fusion protein and incubated for 10 minutes. Similarly, as a negative control, another set of dishes was layered with TAT-HA protein. After incubation the cells were washed and incubated with fresh medium and then harvested at different time intervals, 10 min, 6hr, 12hr. The cells were then lysed and Flue and Rluc reporter activities were measured. The relative luciferase activities were represented as a ratio of Flue to Rluc for normalization. The results indicated that, at all the time points, the cells layered with TAT-LaR2C showed a decrease in the HCV IRES mediated translation over the control cells, while the TAT-HA did not have any effect on HCV IRES mediated translation (Figure 6B). At 6hr post transfection the inhibitory<br><br>
effect was much pronounced (inhibition up to 60%), while the inhibitory effect was found to be gradually decreasing upon prolonged incubation (12hr time point), possibly due to degradation of TAT-LaR2C within the cells. The results indicated that LaR2C does compete with the interaction of cellular La protein to HCV IRES RNA and exert a dominant negative effect to inhibit HCV IRES mediated translation. g. LaR2C peptide prevented the assembly of ribosomal complexes on the HCV IRES<br>
Earlier we have demonstrated that full-length La protein binding at the SLIV triggers a conformational change that facilitates ribosome assembly at the HCV IRES RNA. Since the LaR2C peptide successfully competes with the full-length La protein for binding with the HCV IRES, we have investigated the effect of ribosome assembly upon addition of the LaR2C peptide in the reaction. For this puipose, sucrose gradient centrifugation experiments were performed followed by the analysis of 48S and 80S ribosomal peaks. [32P] labeled HCV IRES RNA was incubated in translation reactions containing RRL and aminoacid mixture and loaded onto 5-30% sucrose gradient and ultracentrifuged for 3hrs at 30,000 rpm. The fractions were collected from the bottom of the tube and scintillation counts were measured. The percentage of counts in each fraction was plotted against the fraction volume of the gradient. When wildtype HCV IRES RNA was used, two peaks were observed in equilibrium, corresponding to 80S and 48S ribosomes associated with the labeled HCV RNA (figure 7, panel A). Upon addition of 20^iM of the LaR2C peptide the peaks corresponding to 80S and 48 S ribosomes were found to be reduced to some extent (Figure 7, compare panel A with panel B). At higher concentration (40|oM) of LaR2C, the 80S peak was found to be completely abolished and even the 48S peak was further reduced compared to control (compare panel C with panel A). However, addition of 40^M concentration of the non-specific peptide (NSP) didn't show significant alterations in either of the 48S or 80S peaks in the ribosomal assembly experiment (figure7, panel D). Interestingly, addition of further high concentration (60|xM) of LaR2C peptide, even the 48S peak was found to be drastically reduced along with the 80S peak (data not shown).<br><br>
It has been demonstrated that in the case of HCV IRES RNA, the 40S ribosomal subunit can form stable binary complex which is distinct from the 48S preinitiation complex (8). However, this binary complex might not be separable from the true 48 S peak in the sucrose gradient centrifugation. The actual 48 S prenitiation complex would be competent for the 60S joining, whereas the binary complex would not lead to functional initiation complex formation. To further clarify, whether the inhibition at the 80S complex formation by the LaR2C peptide was due to its effect at the preinitiation complex formation, we have performed the ribosome assembly reaction in presence of lOmM GMP-PNP. GMP-PNP, which is a non-hydrolyzable analog of GTP, inhibits translation initiation at the 48S stage by preventing the release of eIF2. As expected, addition of GMP-PNP abolished the 80S peak and caused the accumulation of 48S ribosome preinitiation complexes in the control reaction (figure 7, panel E). Interestingly, when LaR2C peptide (40JIM) was added to the reaction (in presence of lOmM GMP-PNP), in addition to a drastic decrease in 80S peak, a significant decrease in the 48S peak was also observed (figure 7, compare panel A, E and F). The results suggest that LaR2C peptide might interfere with the transition step between the HCV-40S binary complex to the 48S complex, resulting in formation of defective complex, which is incompetent for 60S joining. Since addition of LaR2C didn't show accumulation of 48S peak (as observed in case GMP-PNP), rather it decreases 48S peak at higher concentration, it seems that the complex might be dissociating eventually.<br>
The result clearly reconfirms the role of La protein in the formation of functional 48S ribosome complex on HCV IRES, as suggested by our earlier work (ref 6, 7) and also a recent report (11).<br>
Taken together the results strongly suggest that the LaR2C peptide acts as dominant negative and might compete for the binding of La protein to SL IV region of the HCV-IRES, as a consequence of which the formation of functional ribosomal initiation complex is severely affected resulting in the inhibition of HCV IRES mediated translation (Fig 8, Inhibition model). From Fig. 8, it can be observed that La protein binding near initiator AUG of the HCV IRES facilitates the joining of 40S ribosomal subunit into the initiation complex. If the La-peptide (LaR2C) is added in trans, it binds<br><br>
to GCAC near initiator AUG and blocks La protein interaction at this site in a dominant negative manner (indicated with the arrow). This prevents the joining of ribosomal sub unit on the HCV IRES and thereby inhibits HCV IRES-mediated translation.<br>
Figure Legends:<br>
Figure 1: Effect of deletions on the binding of La-RRM2 to HCV IRES RNA. A:<br>
Schematic representation of N- and C terminal deletions within La-RRM2. The corresponding aminoacid numbers of the truncated proteins are indicated. B: The purified protein samples (as indicated on top of the lanes) were analyzed by SDS-10% Tris Tricine gel followed by silver staining. C: Filter-binding assay to study the binding of La 101-208, La 101-180 and La 120-208 to HCV IRES. [32P] labeled HCV IRES RNA was bound to increasing concentrations of La RRM2 or truncated proteins (as indicated by the symbols within the panel). The amount of bound RNA was determined by binding to the nitrocellulose filters. The percentage of bound RNA was plotted against the protein concentration (nM).<br>
Figure 2: Ability of the peptide, LaR2C to bind to HCV IRES RNA. A: The sequence of the peptides LaR2C and NSP are indicated. The LaR2C and the NSP peptides were analyzed by resolving in SDS-12% Tris tricine gel electrophoresis followed by silver staining. B: Filter-binding assay to study the binding of the peptide, LaR2C to HCV IRES. [32P] labeled HCV IRES RNA was bound to increasing concentrations of the peptides, LaR2C or NSP (as indicated by the symbols within the panel). The amount of bound RNA was determined by binding to the nitrocellulose filters. The percentage of bound RNA was plotted against the peptide concentration (pM). C: UV cross-linking of LaR2C and NSP to HCV IRES. [32P] labeled HCV IRES RNA was UV cross-linked increasing concentration (20,40 and 60 jiM) of either LaR2C or NSP (as indicated above the panel), digested with RNase A and resolved by SDS-15% PAGE followed by phosphorimaging. Lane 1 represents the no protein control. Lane M represents [14C] protein molecular weight marker. The corresponding molecular masses are indicated to the left of the panel. D: Competition assay to determine specificity of the binding to<br><br>
LaR2C to HCV IRES RNA. LaR2C pre-incubated with 100 and 200 fold excess of unlabeled HCV wild type RNA or HCV M2 RNA (where SL IV region was mutated) as indicated above the lanes, was bound to [32P] labeled HCV IRES RNA and UV cross-linked. The complexes were treated with RNase A and resolved by SDS-15% PAGE followed by phosphorimaging. The band corresponding to LaR2C is indicated to the right of the panels by arrows. Lane 1 represents the no protein control. Lane M represents [14C] protein molecular weight marker. The corresponding molecular masses are indicated to the left of the panel.<br>
Figure 3: Primer extension inhibition (toe-printing) analysis. A: HCV IRES RNA (18-383nt) was incubated with increasing concentrations of La full-length protein (200, 400ng), La-RRM2 (50-100ng) or LaR2C (2, 4 ^g) as indicated above the lanes, and analyzed by primer extension. Lane 5 shows the no protein control. The toe-prints, which showed increased intensity upon addition of proteins are indicated by asterisks and the toe-prints which showed reduced intensity upon addition of La protein are indicated by filled circles on the right. GCAC motif near AUG is marked on the left of the panel. B: Increasing concentration of the LaR2C peptide (2 and 4\ig, lanes 6-7) or 4 ^ig NSP (lane 9) were incubated with HCV IRES RNA (18-383nt) RNA as above and analyzed for the toe-prints. Lanes 5 and 8 represents the no protein control.<br>
The lanes 1-4 in each panel shows the DNA sequencing ladder corresponding to the HCV 18-383 RNA obtained by using the same end-labeled primer. The nucleotides indicated on the right of each panels signifies the corresponding positions on the HCV IRES RNA.<br>
Figure 4: Effect of LaR2C on the binding of recombinant La protein and other cellular proteins to HCV IRES RNA. A: [32P] labeled HCV IRES RNA was preincubated with increasing concentrations (20, 40 and 60pM) of LaR2C or NSP as indicated above the lanes and then bound to recombinant purified La protein. The UV cross-linked complexes were treated with RNase A and resolved by SDS-10% PAGE followed by phosphorimaging. Band corresponding to rLa are indicated to the right of the panel. Lane M represents [14C] protein molecular weight marker. The corresponding<br><br>
molecular masses are indicated to the left of the panel. The intensity of the band corresponding to La protein in each lane was quantitated using densitometry and represented in numbers below the lanes. B: [32P] labeled HCV IRES RNA was preincubated with increasing concentrations (20,40, 60 \xM) of LaR2C and then bound to HeLa cytoplasmic extract (2.5fig). The UV cross- linked RNA-protein complexes were treated with RNase A and resolved by SDS-5-15% gradient PAGE followed by phosphorimaging. Bands corresponding to p52 is indicated to the right of the panel. The protein bands whose intensity was reduced are indicated by asterisks. The intensity of the band corresponding to p52 in each lane was quantitated using densitometry and represented in numbers below the respective lanes. Lane M represents [14C] protein molecular weight marker. The corresponding molecular masses are indicated to the left of the panel.<br>
Figure 5: Effect of LaR2C on HCV IRES mediated translation in vitro. A: l|ig of uncapped HCV IRES-GFP RNA was translated in rabbit reticulocyte lysate (RRL) absence (lane 1) or presence of increasing concentrations (20, 40 and 60^M) of either LaR2C (lane 2-4) or NSP (lane 5-7). The translation of GFP was analyzed on SDS-12.5% polyacrylamide gel followed by phosphorimaging. The band corresponding to GFP is indicated to the right of the panel. B: l^ig of capped GFP RNA was translated in RRL absence (lane 1) or presence of increasing concentrations (20, 40 and 60^M) of LaR2C (lane 2-4). The translation of GFP was analyzed on SDS-12.5% polyacrylamide gel followed by phosphorimaging. The band corresponding to GFP is indicated to the right of the panel. Panel C and D: 2fig of either capped HAV-bicistronic RNA (containing Fluc-HAV-GFP in order) or PV bicistronic RNA (containing Rluc-PV-FLuc in order) was translated in absence (lane 1) and presence of increasing concentrations (20, 40, 60pM) of LaR2C (lane 2-4). The translation of the reporter genes were analyzed on SDS-12.5% polyacrylamide gel followed by phosphorimaging. The bands corresponding to GFP (representing HAV-IRES function, panel C) and FLuc (representing PV-IRES function, panel D) are only shown in the picture for clarity. The intensity of the GFP band and FLuc band in each lane was quantitated using densitometry and represented in numbers below the respective lanes.<br><br>
Figure 6: Effect of TAT-LaR2C fusion protein on HCV IRES mediated translation in vivo. A: Schematic representation of the TAT-LaR2C fusion protein. The aminoacid sequence of TAT and LaR2C are highlighted. B: Huh7 monolayer cells expressing HCV bicistronic RNA were overlaid with lOOnM of either HA-TAT or TAT-LaR2C fusion protein for 10 minutes. The cells were then harvested after different time points (10 min, 6h and 12h), lysed and the Rluc and Flue were measured using Dual Luciferase assay system. The relative ratio of Flue to Rluc was plotted at each time point. Black bars represent control cells, white bars represent cells overlaid with HA-TAT and gray bars represent cells treated with TAT-LaR2C fusion protein. The HCV bicistronic construct used in the cell line is indicated on top of the panel B. Data from the transfection experiments is expressed as mean Â± SD of three independent replicates. C: Absolute levels of RLuc and FLuc activities (in relative light units) of a representative experiment are presented in the table.<br>
Figure 7: Effect of LaR2C peptide on ribosomal assembly on the HCV IRES RNA.<br>
Sucrose gradient sedimentation profiles of [32P] labeled HCV IRES RNA in absence (panel A) or presence of increasing concentrations of the LaR2C peptide (panel B, and panel C) or the NSP (panel D) after incubation in RRL and separated on 5-30% sucrose gradient. The fractions (200 Dl) were manually collected from the bottom of the tube and scintillation counts were measured. The counts per minute of each fraction, shown as percentage of the total counts added to the reaction (~2xl05cpm) and were plotted against the volume of the gradient solution (0-8 ml). The ribosomal peaks corresponding to 48S and 80S are indicated. Panel E represents sedimentation profile of HCV IRES RNA incubated in RRL in presence of lOmM GMP-PNP alone and panel F is that obtained in presence of both LaR2C (40DM) and GMP-PNP (lOmM).<br>
Figure 8: This illustrates the inhibition of hepatitis C RNA translation by blocking the ribosome assembly with the help of La-peptide.<br><br>
ADVANTAGES OF THE PRESENT INVENTION:<br>
The translation of the positive stranded genomic RNA to produce the viral proteins required for replication is an early obligatory step of the infection process. The translation initiation of the uncapped HCV RNA takes place through the highly structured IRES element located in the 5'UTR of the viral RNA. Thus the process of IRES-mediated translation is an attractive target for antiviral drug design.<br>
The selective inhibition of HCV IRES-mediated mechanism by LaR2C peptide has a potential to be used as a therapeutic strategy with many associated advantages. Firstly, as the interactions between host cellular proteins and a highly conserved region of the viral RNA is targeted, the chance of generation of viral escape mutants is very low. Approaches like siRNA treatment (RNA silencing) has demonstrated rapid emergence of escape mutants in poliovirus. Although the rate of HCV replication is not as high as that of poliovirus, any sequence-specific antiviral molecule would exert a selection pressure for the generation of escape variants, unlike a strategy targeting host protein-viral RNA interactions. Secondly, the peptide molecule being a part of the host genome, if administered prophylactically to patients harbouring the viral RNA, it is not expected to give rise to non-specific immune responses. Thirdly, as the binding of the cellular proteins is known to be dependent on the secondary structure, more stable derivatives and small molecule structural analogs of the peptide could be utilized. The use of a smaller derivative of the peptide might result in better stability thereby significantly increasing the effectiveness of treatment against Hepatitis C viral infection.<br><br>
References<br>
1.	Choo, Q. -L., G. Kuo, A. J. Weiner, L. R. Overby, D. W. Bradley, and M. Houghton 1989. Isolation of a cDNA clone derived from a blood-borne non-A, non-B viral hepatitis genome.   Science. 244: 359-362.<br>
2.	Tsukiyama-Kohara, Z., N. Iizuka, M. Kohara, and A. Nomoto. 1992. Internal ribosome entry site within hepatitis C virus RNA. J. Virol. 66: 1476-1483.<br>
3.	Ali, N., G. J. M. Pruijn, D. J. Kenan, J. D. Keene, and A. Siddiqui. 2000. Human La Antigen Is Required for the Hepatitis C Virus Internal Ribosome Entry Site-mediated Translation. J. Biol. Chem. 275: 27531-27540.<br>
4.	Das, S., M. Ott, A. Yamane, W. Tsai, M. Gromier, F. Lahser, S. Gupta, and A. Dasgupta. 1998. A Small Yeast RNA Blocks Hepatitis C Virus Internal Ribosome Entry Site (HCV IRES)-Mediated Translation and Inhibits Replication of a Chimeric Poliovirus under Translational Control of the HCV IRES Element. J. Virol. 72: 5638-5647.<br>
5.	Izumi, R. E., S. Das, B. Barat, S. Raychaudhuri, and A. Dasgupta. 2004. A peptide from autoantigen La blocks poliovirus and hepatitis C virus cap-independent translation and reveals a single tyrosine critical for La RNA binding and translation stimulation. J. Virol. 78: 3763-3776.<br>
6.	Pudi, R., S. Abhiman, N. Srinivasan, and S. Das. 2003. Hepatitis C Virus Internal Ribosome Entry Site-mediated Translation Is Stimulated by Specific Interaction of Independent Regions of Human La Autoantigen. J. Biol. Chem. 278: 12231-12240.<br>
7.	Pudi, R., P. Srinivasan, and S. Das. 2004. La Protein Binding at the GCAC Site Near the Initiator AUG Facilitates the Ribosomal Assembly on the Hepatitis C Virus RNA to Influence Internal Ribosome Entry Site-mediated Translation. J. Biol. Chem. 279: 29879 -29888.<br>
8.	Otto G.A., and J.D. Puglisi. 2004. The pathway of HCV IRES-mediated translation initiation. Cell. 119(3): 369-80.<br>
9.	Spahn, C.M.T, Kieft, J.S., Grassucci, R.A., Penczek, P.A., Zhou, K., Doudna, J.A and Frank, J. (2001) Hepatitis C virus IRES RNA-induced change in the conformation of the 40S ribosomal subunit. Science, 291, 1959-1962.<br><br>
lO.Becker-Hapak, M., S. S. McAllister, and S.F. Dowdy. 2001. TAT-Mediated protein transduction into Mammalian cells. Methods. 24: 247-56.<br>
11. Costa-Mattioli, M., Y. Svitkin, and N. Sonenberg. 2004. La autoantigen is necessary for optimal function of the poliovirus and hepatitis C virus internal ribosome entry site in vivo and in vitro. Mol. Cell. Biol. 24: 6861-6870.<br><br><br>
WE CLAIM:<br>
1.	A small peptide, LaR2C, corresponding to the C terminus of RRM2 of the human La protein that binds to the IRES element of hepatitis C virus RNA.<br>
2.	The LaR2C peptide that competes with the cellular La protein for binding to HCV IRES RNA and act as dominant negative.<br>
3.	The LaR2C peptide that effectively blocks the ribosome assembly on the HCV RNA.<br>
4.	The peptide that inhibits the internal initiation of translation of hepatitis C virus in vitro and in vivo.<br>
5.	The amino acid sequence of the LaR2C peptide {KYKETDLULFKDDYFAKKNEERK) or its derivative that blocks ribosome interaction with the HCV-IRES and inhibit viral RNA translation.<br>
6.	A polynucleotide comprising the nucleic acid sequence encoding LaR2C peptide iKYKETDLULFKDDYFAKKNEERK) or its derivative that blocks ribosome interaction with the HCV-IRES and inhibit viral RNA translation.<br>
7.	A recombinant vector comprising the polynucleotide as claimed in claim 6.<br>
8.	A fusion peptide obtained by cloning of nucleic acid sequence encoding the LaR2C peptide (KYKEIDLULFKDDYFAKKNEERK) and bacterial expression vector, pTAT.<br><br>
9.	A fusion protein TAT-LaR2C comprising the fusion peptide as claimed in claim<br>
8.<br>
10.	A novel antiviral agent comprising the novel peptide LaR2C corresponding to the<br>
C terminus of RRM2 of the human La protein that binds to the IRES element of<br>
hepatitis C virus RNA and that effectively blocks the ribosome assembly on the<br>
HCV RNA.<br>
11.	The use of LaR2C peptide for treatment of any viral infection and particularly<br>
Hepatitis C viral infection.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1hYnN0cmFjdC5wZGY=" target="_blank" style="word-wrap:break-word;">0520-che-2005-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1jbGFpbXMucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1jb3JyZXNwb25kbmVjZS1vdGhlcnMucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1kZXNjcmlwdGlvbihjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1kZXNjcmlwdGlvbihwcm92aXNpb25hbCkucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-description(provisional).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1kcmF3aW5ncy5wZGY=" target="_blank" style="word-wrap:break-word;">0520-che-2005-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1mb3JtIDEucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1mb3JtIDI2LnBkZg==" target="_blank" style="word-wrap:break-word;">0520-che-2005-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1mb3JtIDMucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MDUyMC1jaGUtMjAwNS1mb3JtIDUucGRm" target="_blank" style="word-wrap:break-word;">0520-che-2005-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgRk9STS0xMyAgMjQtMDgtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">520-CHE-2005   FORM-13  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgRk9STS0xMy0xICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005   FORM-13-1  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgRk9STS00ICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005   FORM-4  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgRk9STS01ICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005   FORM-5  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgT1RIRVIgUEFURU5UIERPQ1VNRU5UICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005   OTHER PATENT DOCUMENT  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICAgUE9XRVIgT0YgQVRUT1JORVkgIDI0LTA4LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">520-CHE-2005   POWER OF ATTORNEY  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBBTUVOREVEICBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAgMjQtMDgtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">520-CHE-2005  AMENDED  PAGES OF SPECIFICATION   24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBBTUVOREVEIENMQUlNUyAgIDI0LTA4LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">520-CHE-2005  AMENDED CLAIMS   24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDI0LTA4LTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">520-CHE-2005  EXAMINATION REPORT REPLY RECEIVED  24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBFWEFNSU5BVElPTiBSRVBPUlQgUkVQTFkgUkVDRUlWRUQgIDI2LTEwLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">520-CHE-2005  EXAMINATION REPORT REPLY RECEIVED  26-10-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBGT1JNLTEgICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005  FORM-1   24-08-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NTIwLUNIRS0yMDA1ICBGT1JNLTMgICAyNC0wOC0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">520-CHE-2005  FORM-3   24-08-2012.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="254637-an-apparatus-and-method-for-archiving-data-exchanged-in-a-session-with-a-wireless-communication-network.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="254639-a-method-for-preparing-a-dry-powder-formulation-suitable-for-inhalation.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>254638</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>520/CHE/2005</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>48/2012</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>30-Nov-2012</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>29-Nov-2012</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>02-May-2005</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INDIAN INSTITUTE OF SCIENCE</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BANGALORE 560 012 KARNATAKA STATE</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DR. SAUMITRA DAS</td>
											<td>DEPARTMENT OF MICROBIOLOGY AND CELL BIOLOGY INDIAN INSTITUTE OF SCIENCE BANGALORE 560 012 KARNATAKA STATE</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MS. RENUKA PUDI</td>
											<td>DEPARTMENT OF MICROBIOLOGY AND CELL BIOLOGY INDIAN INSTITUTE OF SCIENCE BANGALORE 560 012 KARNATAKA STATE</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MR. SUDHAMANI SONNY</td>
											<td>DEPARTMENT OF MICROBIOLOGY AND CELL BIOLOGY INDIAN INSTITUTE OF SCIENCE BANGALORE 560 012 KARNATAKA STATE</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 7/06</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>N/A</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td></td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td></td>
									<td></td>
								    <td>NA</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/254638-a-peptide-and-a-method-thereof by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:27:36 GMT -->
</html>
