<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/254919-binding-molecules-capable-of-neutralizing-rabies-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:12:09 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 254919:BINDING MOLECULES CAPABLE OF NEUTRALIZING RABIES VIRUS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">BINDING MOLECULES CAPABLE OF NEUTRALIZING RABIES VIRUS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides binding molecules that specifically bind to rabies virus and are capable of neutralizing the virus. The invention further provides nucleic acid molecules encoding the binding molecules, compositions comprising the binding molecules and methods of identifying or producing the binding molecules. The binding molecules can be used in the diagnosis, prophylaxis and/or treatment of a condition resulting from rabies virus. Preferably, they can be used in the postexposure prophylaxis of rabies.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>TITLE OF THE INVENTION Binding molecules capable of neutralizing rabies virus and uses thereof<br>
FIELD OF THE INVENTION The invention relates to medicine. In particular the invention relates to rabies virus neutralizing binding molecules. The binding molecules are useful in the postexposure prophylaxis of rabies.<br>
BACKGROUND OF THE INVENTION<br>
Rabies is a viral infection with nearly worldwide distribution that affects principally wild and domestic animals but also involves humans, resulting in a devastating, almost invariable fatal encephalitis. Annually, more than 70,000 human fatalities are estimated, and millions of others reguire postexposure treatment.<br>
The rabies virus is a bullet-shaped, enveloped, single-stranded RNA virus classified in the rhabdovirus family and Lyssavirus genus. The genome of rabies virus codes for five viral proteins: RNA-dependent RNA polymerase (L); a nucleoprotein (N); a phosphorylated protein (P); a matrix protein (M) located on the inner side of the viral protein envelope; and an external surface glycoprotein (G).<br>
The G protein (62-67 kDa) is a type-I glycoprotein composed of 505 amino acids that has two to four potential N-glycosylation sites, of which only one or two are glycosylated depending on the virus strains. The G protein forms the protrusions that cover the outer surface of the virion envelope and is known to induce virus-neutralizing antibodies.<br>
Rabies can be treated or prevented by both passive and active immunizations. Rabies postexposure prophylaxis includes<br><br>
prompt local wound care and administration of both passive (anti-rabies immunoglobulins) and active (vaccines) immunizations.<br>
Currently, the anti-rabies immunoglobulins (RIG) are prepared from the serum samples of either rabies virus-immune humans (HRIG) or rabies virus-immune horses (ERIG). A disadvantage of ERIG as well as HRIG is that they are not available in sufficient amounts and, in case of HRIG, are too expensive. In addition, the use of ERIG might lead to adverse reactions such as anaphylactic shock. The possibility of contamination by known or unknown pathogens is an additional concern associated with HRIG. To overcome these disadvantages it has been suggested to use monoclonal antibodies capable of neutralizing rabies virus in postexposure prophylaxis. Rabies-virus neutralizing murine monoclonal antibodies are known in the art (see Schumacher et al., 1989). However, the use of murine antibodies in vivo is limited due to problems associated with administration of murine antibodies to humans, such as short serum half life, an inability to trigger certain human effector functions and elicitation of an unwanted dramatic immune response against the murine antibody in a human (the “human anti-mouse antibody” (HAMA) reaction).<br>
Recently, human rabies virus-neutralizing monoclonal antibodies have been described (see Dietzschold et al., 1990, Champion et al.r 2000, and Hanlon et al., 2001). For human anti-rabies monoclonal antibodies to be as effective as HRIG in postexposure prophylaxis a mixture of monoclonal antibodies should be used. In such a mixture each antibody should bind to a different epitope or site on the virus to prevent the escape of resistant variants of the virus.<br>
Currently, there is still a significant need for new human rabies virus-neutralizing monoclonal antibodies having<br><br>
improved postexposure prophylactic potential, particularly antibodies having different epitope-recognition specificities. The present invention provides such human monoclonal antibodies that offer the potential to be used in mixtures useful in the postexposure prophylaxis of a wide range of rabies viruses and neutralization-resistant variants thereof.<br>
DESCRIPTION OF THE FIGURES Figure 1 shows the comparison of the amino acid sequences of the rabies virus strain CVS-11 and E57 escape viruses. Virus-infected cells were harvested 2 days post-infection and total RNA was isolated. cDNA was generated and used for DNA sequencing. Regions containing mutations are shown and the mutations are indicated in bold. Figure 1A shows the comparison of the nucleotide sequences. Numbers above amino acids indicate amino acids numbers from rabies virus glycoprotein including signal peptide. Figure IB shows the comparison of amino acid sequences. Schematic drawing of rabies virus glycoprotein is shown on top. The black box indicates the signal peptide, while the gray box indicates the transmembrane domain. The sequences in Figure 1 are also represented by SEQ ID Nos:130-141.<br>
Figure 2 shows the comparison of the amino acid sequences of the rabies virus strain CVS-11 and EJB escape viruses. Virus-infected cells were harvested 2 days post-infection and total RNA was isolated. cDNA was generated and used for DNA sequencing. Regions containing mutations are shown and the mutations are indicated in bold. Figure 2A shows the comparison of the nucleotide sequences. Numbers above amino acids indicate amino acid numbers from rabies virus glycoprotein including the signal peptide. Figure 2B shows the<br><br>
comparison of amino acid sequences. Schematic drawing of rabies virus glycoprotein is shown on top. The black box indicates the signal peptide, while the gray box indicates the transmembrane domain. The sequences in Figure 2 are also represented by SEQ ID Nos:142-151.<br>
Figure 3 shows the vector PDV-C06.<br>
Figure 4 shows a competition ELISA of anti-rabies virus scFvs and the biotinylated anti-rabies virus antibody called CR-57. ELISA plates coated with purified rabies virus G protein were incubated with the respective scFvs before addition of CR-<br>
57bio (0.5 μg/ml). Subsequently, CR-57bio binding was monitored in absence and presence of scFvs.<br>
Figure 5 shows a competition ELISA of anti-rabies virus scFvs and the anti-rabies virus antibody called CR-57. ELISA plates coated with purified rabies virus G protein were incubated<br>
with CR-57 (1 μg/ml) before addition of excess scFvs. Subsequently, scFv binding was monitored in absence and presence of CR-57.<br>
Figure 6 shows a competition ELISA assay of anti-rabies virus G protein IgGs and the anti-rabies virus antibody called CR-57. G protein (ERA strain) was incubated with unlabeled IgGs (shown on the X-axis). Biotinylated CR57 (CR57bio) was added and allowed to bind to the G protein before visualization by means of streptavidin-HRP. ELISA signals are shown as percentage of CR57bio binding alone.<br>
Figure 7 shows a competition FACS assay of anti-rabies virus G protein IgGs and the anti-rabies virus antibody called CR-57.<br><br>
G protein (ERA strain) expressing PER.C6 cells were incubated with unlabeled IgGs (shown on the X-axis). Biotinylated CR57 (CR57bio) was added and allowed to bind to the G protein expressing cells before visualization by means of streptavidin-PE. FACS signals are shown as percentage of CR57bio binding alone.<br>
Figure 8 shows the comparison of the amino acid sequences of CVS-11 and E98 escape viruses. Virus-infected cells were harvested 2 days post-infection and total RNA was isolated. cDNA was generated and used for DNA sequencing. Region containing a point mutation is shown and the mutation is indicated in bold. Figure 8A shows the comparison of the nucleotide sequences. The number above the nucleotide indicates the mutated nucleotide (indicated in bold) from rabies virus glycoprotein open reading frame without signal peptide sequence. Figure 8B shows the comparison of amino acid sequences. The number above the amino acid indicates the mutated amino acid (indicated in bold) from rabies virus glycoprotein without signal peptide sequence.<br>
Figure 9 shows a phylogenetic tree of 123 rabies street viruses (123 rabies virus G glycoprotein sequences, Neighbor joining, Kimura-2-parameter method, 500 bootstraps). Bold indicates viruses harboring the N&gt;D mutation as observed in E98 escape viruses.<br>
Figure 10 shows neutralizing epitopes on rabies glycoprotein. A schematic drawing of the rabies virus glycoprotein is shown depicting the antigenic sites including the novel CR57 epitope. The signal peptide (19 amino acids) and transmembrane domain are indicated by black boxes. Disulfide bridges are<br><br>
indicated. Amino acid numbering is from the mature protein minus the signal peptide sequence.<br>
DESCRIPTION OF THE INVENTION Herebelow follow definitions of terms as used in the invention.<br>
DEFINITIONS Binding molecule<br>
As used herein the term “binding molecule” refers to an intact immunoglobulin including monoclonal antibodies, such as chimeric, humanized or human monoclonal antibodies, or to an antigen-binding and/or variable domain comprising fragment of an immunoglobulin that competes with the intact immunoglobulin for specific binding to the binding partner of the immunoglobulin, e.g. rabies virus or a fragment thereof such as for instance the G protein. Regardless of structure, the antigen-binding fragment binds with the same antigen that is recognised by the intact immunoglobulin. An antigen-binding fragment can comprise a peptide or polypeptide comprising an amino acid sequence of at least 2 contiguous amino acid residues, at least 5 contiguous amino acid residues, at least 10 contiguous amino acid residues, at least 15 contiguous amino acid residues, at least 20 contiguous amino acid residues, at least 25 contiguous amino acid residues, at least 30 contiguous amino acid residues, at least 35 contiguous amino acid residues, at least 40 contiguous amino acid residues, at least 50 contiguous amino acid residues, at least 60 contiguous amino residues, at least 70 contiguous amino acid residues, at least contiguous 80 amino acid residues, at least contiguous 90 amino acid residues, at least contiguous<br><br>
100 amino acid residues, at least contiguous 125 amino acid residues, at least 150 contiguous amino acid residues, at least contiguous 175 amino acid residues, at least 2 00 contiguous amino acid residues, or at least contiguous 250 amino acid residues of the amino acid sequence of the binding molecule.<br>
The term “binding molecule”, as used herein includes all immunoglobulin classes and subclasses known in the art. Depending on the amino acid sequence of the constant domain of their heavy chains, binding molecules can be divided into the five major classes of intact antibodies: IgA, IgD, IgE, IgG, and IgM, and several of these may be further divided into subclasses (isotypes), e.g., IgAl, IgA2, IgGl, IgG2, IgG3 and IgG4.<br>
Antigen-binding fragments include, inter alia, Fab, F(abf), F(ab’)2, Fv, dAb, Fd, complementarity determining region (CDR) fragments, single-chain antibodies (scFv), bivalent single-chain antibodies, single-chain phage antibodies, diabodies, triabodies, tetrabodies, (poly)peptides that contain at least a fragment of an immunoglobulin that is sufficient to confer specific antigen binding to the (poly)peptide, etc. The above fragments may be produced synthetically or by enzymatic or chemical cleavage of intact immunoglobulins or they may be genetically engineered by recombinant DNA techniques. The methods of production are well known in the art and are described, for example, in Antibodies: A Laboratory Manual, Edited by: E. Harlow and D, Lane (1988), Cold Spring Harbor Laboratory, Cold Spring Harbor, New York, which is incorporated herein by reference. A binding molecule or antigen-binding fragment thereof may have one or more binding sites. If there is more than one binding<br><br>
site, the binding sites may be identical to one another or they may be different.<br>
The binding molecule can be a naked or unconjugated binding molecule but can also be part of an iinmunoconjugate, A naked or unconjugated binding molecule is intended to refer to a binding molecule that is not conjugated, operatively linked or otherwise physically or functionally associated with an effector moiety or tag, such as inter alia a toxic substance, a radioactive substance, a liposome, an enzyme. It will be understood that naked or unconjugated binding molecules do not exclude binding molecules that have been stabilized, multimerized, humanized or in any other way manipulated, other than by the attachment of an effector moiety or tag. Accordingly, all post-translationally modified naked and unconjugated binding molecules are included herewith, including where the modifications are made in the natural binding molecule-producing cell environment, by a recombinant binding molecule-producing cell, and are introduced by the hand of man after initial binding molecule preparation. Of course, the term naked or unconjugated binding molecule does not exclude the ability of the binding molecule to form functional associations with effector cells and/or molecules after administration to the body, as some of such interactions are necessary in order to exert a biological effect. The lack of associated effector group or tag is therefore applied in definition to the naked or unconjugated binding molecule in vitro, not in vivo.<br>
Complementarity determining regions (CDR)<br>
The term “complementarity determining regions” as used herein means sequences within the variable regions of binding molecules, such as immunoglobulins, that usually contribute to<br><br>
a large extent to the antigen binding site which is complementary in shape and charge distribution to the epitope recognized on the antigen. The CDR regions can be specific for linear epitopes, discontinuous epitopes, or conformational epitopes of proteins or protein fragments, either as present on the protein in its native conformation or, in some cases, as present on the proteins as denatured, e.g., by solubilization in SDS. Epitopes may also consist of posttranslational modifications of proteins.<br>
Functional variant<br>
The term “functional variant”, as used herein, refers to a binding molecule that comprises a nucleotide and/or amino acid sequence that is altered by one or more nucleotides and/or amino acids compared to the nucleotide and/or amino acid sequences of the parent binding molecule and that is still capable of competing for binding to the binding partner, e.. g. rabies virus or a fragment thereof, with the parent binding molecule. In other words, the modifications in the amino acid and/or nucleotide sequence of the parent binding molecule do not significantly affect or alter the binding characteristics of the binding molecule encoded by the nucleotide sequence or containing the amino acid sequence, i.e. the binding molecule is still able to recognize and bind its target. The functional variant may have conservative sequence modifications including nucleotide and amino acid substitutions, additions and deletions. These modifications can be introduced by standard techniques known in the art, such as site-directed mutagenesis and random PCR-mediated mutagenesis, and may comprise natural as well as non-natural nucleotides and amino acids.<br>
Conservative amino acid substitutions include the ones in which the amino acid residue is replaced with an amino acid<br><br>
residue having similar structural or chemical properties. Families of amino acid residues having similar side chains have been defined in the art. These families include amino acids with basic side chains (e.g., lysine, arginine, histidine), acidic side chains (e.g., aspartic acid, glutamic acid), uncharged polar side chains (e.g., asparagine, glutamine, serine, threonine, tyrosine, cysteine, tryptophan), nonpolar side chains (e.g., glycine, alanine, valine, leucine, isoleucine, proline, phenylalanine, methionine), beta-branched side chains (e.g., threonine, valine, isoleucine) and aromatic side chains (e.g., tyrosine, phenylalanine, tryptophan). It will be clear to the skilled artisan that other classifications of amino acid residue families than the one used above can also be employed. Furthermore, a variant may have non-conservative amino acid substitutions, e. g., replacement of an amino acid with an amino acid residue having different structural or chemical properties. Similar minor variations may also include amino acid deletions or insertions, or both. Guidance in determining which amino acid residues may be substituted, inserted, or deleted without abolishing immunological activity may be found using computer programs well known in the art.<br>
A mutation in a nucleotide sequence can be a single alteration made at a locus (a point mutation), such as transition or transversion mutations, or alternatively, multiple nucleotides may be inserted, deleted or changed at a single locus. In addition, one or more alterations may be made at any number of loci within a nucleotide sequence. The mutations may be performed by any suitable method known in the art.<br>
Host<br><br>
The term “host”, as used herein, is intended to refer to an organism or a cell into which a vector such as a cloning vector or an expression vector has been introduced. The organism or cell can be prokaryotic or eukaryotic. It should be understood that this term is intended to refer not only to the particular subject organism or cell, but to the progeny of such an organism or cell as well. Because certain modifications may occur in succeeding generations due to either mutation or environmental influences, such progeny may not, in fact, be identical to the parent organism or cell, but are still included within the scope of the term “host” as used herein.<br>
Human<br>
The term “human”, when applied to binding molecules as defined herein, refers to molecules that are either directly derived from a human or based upon a human sequence. When a binding molecule is derived from or based on a human sequence and subsequently modified, it is still to be considered human as used throughout the specification. In other words, the term human, when applied to binding molecules is intended to include binding molecules having variable and constant regions derived from human germline immunoglobulin sequences based on variable or constant regions either or not occurring in a human or human lymphocyte or in modified form. Thus, the human binding molecules may include amino acid residues not encoded by human germline immunoglobulin sequences, comprise substitutions and/or deletions (e.g., mutations introduced by for instance random or site-specific mutagenesis in vitro or by somatic mutation in vivo). “Based on” as used herein refers to the situation that a nucleic acid sequence may be exactly copied from a template, or with minor mutations, such as by<br><br>
error-prone PCR methods, or synthetically made matching the template exactly or with minor modifications. Semisynthetic molecules based on human sequences are also considered to be human as used herein.<br>
Monoclonal antibody<br>
The term “monoclonal antibody” as used herein refers to a preparation of antibody molecules of single molecular composition, i.e. primary structure, i.e. having a single amino acid sequence. A monoclonal antibody displays a single binding specificity and affinity for a particular epitope. Accordingly, the term “human monoclonal antibody” refers to an antibody displaying a single binding specificity which have variable and constant regions derived from or based on human germline immunoglobulin sequences or derived from completely synthetic sequences. The method of preparing the monoclonal antibody is not relevant.<br>
Nucleic acid molecule<br>
The term “nucleic acid molecule” as used in the present invention refers to a polymeric form of nucleotides and includes both sense and antisense strands of RNA, cDNA, genomic DNA, and synthetic forms and mixed polymers of the above. A nucleotide refers to a ribonucleotide, deoxynucleotide or a modified form of either type of nucleotide. The term also includes single- and double-stranded forms of DNA. In addition, a polynucleotide may include either or both naturally-occurring and modified nucleotides linked together by naturally-occurring and/or non-naturally occurring nucleotide linkages. The nucleic acid molecules may be modified chemically or biochemically or may contain non-natural or derivatized nucleotide bases, as will be readily<br><br>
appreciated by those of skill in the art. Such modifications include, for example, labels, methylation, substitution of one or more of the naturally occurring nucleotides with an analog, internucleotide modifications such as uncharged linkages (e.g., methyl phosphonates, phosphotriesters, phosphoramidates, carbamates, etc.)/ charged linkages (e.g., phosphorothioates, phosphorodithioates, etc.), pendent moieties (e.g., polypeptides), intercalators (e.g., acridine, psoralen, etc.), chelators, alkylators, and modified linkages (e.g., alpha anomeric nucleic acids, etc.). The above term is also intended to include any topological conformation, including single-stranded, double-stranded, partially duplexed, triplex, hairpinned, circular and padlocked conformations. Also included are synthetic molecules that mimic polynucleotides in their ability to bind to a designated sequence via hydrogen bonding and other chemical interactions. Such molecules are known in the art and include, for example, those in which peptide linkages substitute for phosphate linkages in the backbone of the molecule. A reference to a nucleic acid sequence encompasses its complement unless otherwise specified. Thus, a reference to a nucleic acid molecule having a particular sequence should be understood to encompass its complementary strand, with its complementary sequence. The complementary strand is also useful, e.g., for antisense therapy, hybridization probes and PCR primers.<br>
Pharmaceutically acceptable excipient<br>
By “pharmaceutically acceptable excipient” is meant any inert substance that is combined with an active molecule such as a drug, agent, or binding molecule for preparing an agreeable or convenient dosage form. The “pharmaceutically acceptable excipient” is an excipient that is non-toxic, or at least of<br><br>
which the toxicity is acceptable for its intended use, to recipients at the dosages and concentrations employed and is compatible with other ingredients of the formulation comprising the drug, agent or binding molecule.<br>
Post exposure prophylaxis	v<br>
“Post exposure prophylaxis” (PEP) is indicated for persons possibly exposed to a rabid animal. Possible exposures include bite exposure (i.e. any penetration of the skin by teeth) including animal bites, and non-bite exposure. Non-bite exposures include exposure to large amounts of aerosolized rabies virus in laboratories or caves and surgical recipients of corneas transplanted from patients who died of rabies. The contamination of open wounds, abrasions, mucous membranes, or theoretically, scratches, with saliva or other potentially infectious material (such as neural tissue) from a rabid animal also constitutes a non-bite exposure. Other contact by itself, such as petting a rabid animal and contact with blood, urine, or feces of a rabid animal, does not constitute an exposure and is not an indication for prophylaxis. PEP should begin as soon as possible after an exposure. If no exposure has occurred post exposure prophylaxis is not necessary. In all post exposure prophylaxis regimens, except for persons previously immunized, active and passive immunizations should be used concurrently.<br>
Specifically Binding<br>
The term “specifically binding”, as used herein, in reference to the interaction of a binding molecule, e. g. an antibody, and its binding partner, e.g. an antigen, means that the interaction is dependent upon the presence of a particular structure, e.g. an antigenic determinant or epitope, on the<br><br>
binding partner. In other words, the antibody preferentially binds or recognizes the binding partner even when the binding partner is present in a mixture of other molecules or organisms. The binding may be mediated by covalent or non-covalent interactions or a combination of both. In yet other words, the term “specifically binding” means<br>
immunospecifically binding to an antigen or a fragment thereof and not immunospecifically binding to other antigens. A binding molecule that immunospecifically binds to an antigen may bind to other peptides or polypeptides with lower affinity as determined by, e.g., radioimmunoassays (RIA), enzyme-linked immunosorbent assays (ELISA), BIACORE, or other assays known in the art. Binding molecules or fragments thereof that immunospecifically bind to an antigen may be cross-reactive with related antigens. Preferably, binding molecules or fragments thereof that immunospecifically bind to an antigen do not cross-react with other antigens.<br>
Therapeutically effective amount<br>
The term “therapeutically effective amount” refers to an amount of the binding molecule as defined herein that is effective for post-exposure prophylaxis of rabies.<br>
Vector<br>
The term “vector” denotes a nucleic acid molecule into which a second nucleic acid molecule can be inserted for introduction into a host where it will be replicated, and in some cases expressed. In other words, a vector is capable of transporting a nucleic acid molecule to which it has been linked. Cloning as well as expression vectors are contemplated by the term “vector”, as used herein. Vectors include, but are not limited to, plasmids, cosmids, bacterial artificial chromosomes (BAC)<br><br>
and yeast artificial chromosomes (YAC) and vectors derived from bacteriophages or plant or animal (including human) viruses. Vectors comprise an origin of replication recognized by the proposed host and in case of expression vectors, promoter and other regulatory regions recognized by the host. A vector containing a second nucleic acid molecule is introduced into a cell for example by transformation, transfection, or by making use of bacterial or viral entry mechanisms. Other ways of introducing nucleic acid into cells are known, such as electroporation or particle bombardment often used with plant cells, and the like. The method of introducing nucleic acid into cells depends among other things on the type of cells, and so forth. This is not critical to the invention. Certain vectors are capable of autonomous replication in a host into which they are introduced (e.g., vectors having a bacterial origin of replication can replicate in bacteria). Other vectors can be integrated into the genome of a host upon introduction into the host, and thereby are replicated along with the host genome.<br>
SUMMARY OF THE INVENTION The invention provides binding molecules capable of specifically binding to and neutralizing rabies virus. Furthermore, the invention pertains to nucleic acid molecules encoding at least the binding region of these binding molecules. The invention further provides for the use of the binding molecules of the invention in the post exposure prophylaxis of a subject at risk of developing a condition resulting from rabies virus.<br>
DETAILED DESCRIPTION OF THE INVENTION<br><br>
In a first aspect the present invention encompasses binding molecules capable of specifically binding to rabies virus. Preferably, the binding molecules of the invention also have rabies virus-neutralizing activity. Preferably, the binding molecules of the invention are human binding molecules. Alternatively, they may also be binding molecules of other animals. Rabies virus is part of the Lyssavirus genus. In total, the Lyssavirus genus includes eleven genotypes: rabies virus (genotype 1), Lagos bat virus (genotype 2), Mokola virus (genotype 3), Duvenhage virus (genotype 4), European bat lyssavirus 1 (genotype 5), European bat lyssavirus 2 (genotype 6), Australian bat lyssavirus (genotype 7), Aravan virus (genotype 8), Khujand virus (genotype 9), Irkut virus (genotype 10) and West Caucasian virus (genotype 11). Besides binding to rabies virus, the binding molecules of the invention may also be capable of binding to other genotypes of the Lyssavirus genus. Preferably, the binding molecules may also be capable of neutralizing other genotypes of the Lyssavirus genus. Furthermore, the binding molecules of the invention may even be capable of binding to and/or neutralizing viruses, other than Lyssaviruses, of the rhabdovirus family. This family includes the genera cytorhabdovirus, ephemerovirus, lyssavirus, nucleorhabdovirus, rhabdovirus and vesiculovirus.<br>
The binding molecules may be capable of specifically binding to rabies virus in its natural form or in its inactivated/attenuated form. Inactivation of rabies virus may be performed by treatment with inter alia beta-propiolactone (BPL) (White and Chappel, 1982), heating at 56°C for more than 30 minutes, gamma irradiation, treatment with<br>
acetylethylenimine or ethylenimine or treatment with ascorbic acid and copper sulfate for 72 hours (Madhusudana et al.,<br><br>
2004). General viral inactivation methods well known to the skilled artisan such as inter alia pasteurization (wet heat), dry heat treatment, vapor heat treatment, treatment with low pH, treatment with organic solvent/detergent, nanofiltration; UV light irradiation may also be used. Preferably, the inactivation is performed by treatment with beta-propiolactone (BPL). Methods to test if rabies virus is still infective or partly or completely inactivated are well known to the person skilled in the art and can among others be found in Laboratory techniques in rabies, Edited by: F.-X Meslin, M.M. Kaplan and H. Koprowski (1996), 4th edition, Chapter 36, World Health Organization, Geneva.<br>
The binding molecules may also be capable of specifically binding to one or more fragments of the rabies virus such as inter alia a preparation of one or more proteins and/or (poly)peptides derived from rabies virus or a cell transfected with a rabies virus protein and/or (poly)peptide. For methods of treatment and/or prevention such as methods for post exposure prophylaxis of rabies virus the binding molecules are preferably capable of specifically binding to surface accessible proteins of rabies virus such as the M (see Ameyama et al. 2003) or G protein. For diagnostical purposes the human binding molecules may also be capable of specifically binding to proteins not present on the surface of rabies virus. The amino acid sequence of surface accessible and internal proteins of various known strains of rabies virus can be found in the EMBL-database and/or other databases.<br>
Preferably, the fragment at least comprises an antigenic determinant recognized by the human binding molecules of the invention. An “antigenic determinant” as used herein is a moiety, such as a rabies virus (poly)peptide, (glyco)protein, or analog or fragment thereof, that is capable of binding to a<br><br>
human binding molecule of the invention with sufficiently high affinity to form a detectable antigen-binding molecule complex.<br>
The binding molecules according to the invention can be intact immunoglobulin molecules such as polyclonal or monoclonal antibodies, in particular human monoclonal antibodies, or the binding molecules can be antigen-binding fragments including, but not limited to, Fab, F(ab!), F(ab?)2r Fv, dAb, Fd, complementarity determining region (CDR) fragments, single-chain antibodies (scFv), bivalent single-chain antibodies, single-chain phage antibodies, diabodies, triabodies, tetrabodies, and (poly)peptides that contain at least a fragment of an immunoglobulin that is sufficient to confer specific antigen binding to the rabies virus or fragment thereof. The binding molecules of the invention can be used in non-isolated or isolated form. Furthermore, the binding molecules of the invention can be used alone or in a mixture comprising at least one human binding molecule (or variant or fragment thereof). In other words, the binding molecules can be used in combination, e.g., as a pharmaceutical composition comprising two or more binding molecules, variants or fragments thereof. For example, binding molecules having rabies virus-neutralizing activity can be combined in a single therapy to achieve a desired prophylactic, therapeutic or diagnostic effect.<br>
RNA viruses such as rabies virus make use of their own RNA polymerase during virus replication. These RNA polymerases tend to be error-prone. This leads to the formation of so-called quasi-species during a viral infection. Each quasi-species has a unique RNA genome, which could result in differences in amino acid composition of viral proteins. If such mutations occur in structural viral proteins, the virus<br><br>
could potentially escape from the host’s immune system due to a change in T or B cell epitopes. The likelihood of this to happen is higher when individuals are treated with a mixture of two binding molecules, such as human monoclonal antibodies, than with a polyclonal antibody mixture (HRIG). Therefore, a prerequisite for a mixture of two human monoclonal antibodies for treatment of rabies is that the two antibodies recognize non-overlapping, non-competing epitopes on their target antigen, i.e. rabies virus glycoprotein. The chance of the occurrence of rabies escape viruses is thereby minimized. As a consequence thereof, the binding molecules of the invention preferably are capable of reacting with different, non-overlapping, non-competing epitopes of the rabies virus, such as epitopes on the rabies virus G protein. The mixture of binding molecules may further comprise at least one other therapeutic agent such as a medicament suitable for the post exposure prophylaxis of rabies.<br>
Typically, binding molecules according to the invention can bind to their binding partners, i.e. rabies virus or fragments thereof such as rabies virus proteins, with an affinity constant (Kd-value) that is lower than 0.2*10”4 M, 1.0*10”5 M, 1.0*10”6 M, 1.0*10”7 M, preferably lower than 1.0*10” 8 M, more preferably lower than 1.0*10~9 M, more preferably lower than 1.0*10~10 M, even more preferably lower than 1.0*10” 11 M, and in particular lower than 1.0*10”12 M. The affinity constants can vary for antibody isotypes. For example, affinity binding for an IgM isotype refers to a binding affinity of at least about 1.0*10~7 M. Affinity constants can for instance be measured using surface plasmon resonance, i.e. an optical phenomenon that allows for the analysis of real-time biospecific interactions by detection of alterations in protein concentrations within a biosensor matrix, for example<br><br>
using the BIACORE system (Pharmacia Biosensor AB, Uppsala, Sweden).<br>
The binding molecules according to the invention may bind to rabies virus in purified/isolated or non-purified/non-isolated form. The binding molecules may bind to rabies virus in soluble form such as for instance in a sample or may bind to rabies virus bound or attached to a carrier or substrate, e.g., microtiter plates, membranes and beads, etc. Carriers or substrates may be made of glass, plastic (e.g., polystyrene), polysaccharides, nylon, nitrocellulose, or teflon, etc. The surface of such supports may be solid or porous and of any convenient shape. Alternatively, the binding molecules may also bind to fragments of rabies virus such as proteins or (poly)peptides of the rabies virus. In an embodiment the binding molecules are capable of specifically binding to the rabies virus G protein or fragment thereof. The rabies virus proteins or (poly)peptides may either be in soluble form or may bind to rabies virus bound or attached to a carrier or substrate as described above. In another embodiment cells tranfected with the G protein may be used as binding partner for the binding molecules.<br>
In a preferred embodiment of the invention, the binding molecules of the invention neutralize rabies virus infectivity. This may be achieved by preventing the attachment of rabies virus to its receptors on host cells, such as inter alia the murine p75 neurotrophin receptor, the neural cell adhesion molecule (CD56) and the acetylcholine receptor, or inhibition of the release of RNA into the cytoplasm of the cell or prevention of RNA transcription or translation. In a specific embodiment, the binding molecules of the invention prevent rabies virus from infecting host cells by at least 99%, at least 95%, at least 90%, at least 85%, at least 80%,<br><br>
at least 75%, at least 70%, at least 60%, at least 50%, at least 45%, at least 40%, at least 45%, at least 35%, at least 30%, at least 25%, at least 20%, or at least 10% relative to infection of host cells by rabies virus in the absence of said binding molecules. Neutralization can for instance be measured as described in Laboratory techniques in rabies, Edited by: F.-X Meslin, M.M. Kaplan and H. Koprowski (1996), 4th edition, Chapters 15-17, World Health Organization, Geneva. Furthermore, the human binding molecules of the invention may be complement fixing binding molecules capable of assisting in the lysis of enveloped rabies virus. The human binding molecules of the invention might also act as opsonins and augment phagocytosis of rabies virus either by promoting its uptake via Fc or C3b receptors or by agglutinating rabies virus to make it more easily phagocytosed.<br>
In a preferred embodiment, the binding molecules according to the invention comprise at least a CDR3 region comprising the amino acid sequence selected from the group consisting of SEQ ID NO:l, SEQ ID NO:2, SEQ ID NO:3, SEQ ID NO:4, SEQ ID NO:5, SEQ ID NO:6, SEQ ID NO:7, SEQ ID NO:8, SEQ ID NO:9, SEQ ID NO:10, SEQ ID NO:ll, SEQ ID NO:12, SEQ ID NO:13, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16, SEQ ID NO:17, SEQ ID NO:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23 and SEQ ID NO:24. In an embodiment the CDR3 region is a heavy chain CDR3 region.<br>
In yet another embodiment, the binding molecules according to the invention comprise a variable heavy chain comprising essentially an amino acid sequence selected from the group consisting of SEQ ID NO:26, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO:30, SEQ ID NO:31, SEQ ID NO:32, SEQ ID NO:33, SEQ ID NO:34, SEQ ID NO:35, SEQ ID NO:36, SEQ ID NO:37, SEQ ID NO:38, SEQ ID NO:39, SEQ ID NO:40, SEQ ID NO:41,<br><br>
SEQ ID NO:42, SEQ ID NO:43, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:48 and SEQ ID NO:49. In a preferred embodiment the binding molecules according to the invention comprise a variable heavy chain comprising essentially an amino acid sequence comprising amino acids 1-119 of SEQ ID NO:335.<br>
In a further embodiment, the binding molecules according to the invention comprise a variable heavy chain comprising the amino acid sequence of SEQ ID NO:26 and a variable light chain comprising the amino acid sequence of SEQ ID NO:50, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:27 and a variable light chain comprising the amino acid sequence of SEQ ID NO:51, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:28 and a variable light chain comprising the amino acid sequence of SEQ ID NO:52, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:29 and a variable light chain comprising the amino acid sequence of SEQ ID NO:53, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:30 and a variable light chain comprising the amino acid sequence of SEQ ID NO:54, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:31 and a variable light chain comprising the amino acid sequence of SEQ ID NO:55, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:32 and a variable light chain comprising the amino acid sequence of SEQ ID NO:56, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:33 and a variable light chain comprising the amino acid sequence of SEQ ID NO:57, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:34 and a variable light chain comprising the amino acid sequence of SEQ ID NO:58, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:35 and a variable light chain comprising the amino acid<br><br>
sequence of SEQ ID NO:59, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:36 and a variable light chain comprising the amino acid sequence of SEQ ID NO:60, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:37 and a variable light chain comprising the amino acid sequence of SEQ ID NO:61, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:38 and a variable light chain comprising the amino acid sequence of SEQ ID NO:62, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:39 and a variable light chain comprising the amino acid sequence of SEQ ID NO:63, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:40 and a variable light chain comprising the amino acid sequence of SEQ ID NO:64, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:41 and a variable light chain comprising the amino acid sequence of SEQ ID NO:65, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:42 and a variable light chain comprising the amino acid sequence of SEQ ID NO:66, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:43 and a variable light chain comprising the amino acid sequence of SEQ ID NO:67, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:44 and a variable light chain comprising the amino acid sequence of SEQ ID NO:68, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:45 and a variable light chain comprising the amino acid sequence of SEQ ID NO:69, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:46 and a variable light chain comprising the amino acid sequence of SEQ ID NO:70, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:47 and a variable light chain comprising the amino acid sequence of SEQ ID NO:71, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:48 and a variable light<br><br>
chain comprising the amino acid sequence of SEQ ID NO:72, a variable heavy chain comprising the amino acid sequence of SEQ ID NO:49 and a variable light chain comprising the amino acid sequence of SEQ ID NO:73. In a preferred embodiment the human binding molecules according to the invention comprise a variable heavy chain comprising the amino acid sequence comprising amino acids 1-119 of SEQ ID NO:335 and a variable light chain comprising the amino acid sequence comprising amino acids 1-107 of SEQ ID NO:337.<br>
In a preferred embodiment the binding molecules having rabies virus neutralizing activity of the invention are administered in IgG format, preferably IgGl format.<br>
Another aspect of the invention includes functional variants of binding molecules as defined herein. Molecules are considered to be functional variants of a binding molecule according to the invention, if the variants are capable of competing for specifically binding to rabies virus or a fragment thereof with the parent binding molecules. In other words, when the functional variants are still capable of binding to rabies virus or a fragment thereof. The functional variants should also still have rabies virus neutralizing activity. Functional variants include, but are not limited to, derivatives that are substantially similar in primary structural sequence, but which contain e.g. in vitro or in vivo modifications, chemical and/or biochemical, that are not found in the parent binding molecule. Such modifications include inter alia acetylation, acylation, ADP-ribosylation, amidation, covalent attachment of flavin, covalent attachment of a heme moiety, covalent attachment of a nucleotide or nucleotide derivative, covalent attachment of a lipid or lipid derivative, covalent attachment of phosphotidylinositol, cross-linking, cyclization, disulfide bond formation,<br><br>
demethylation, formation of covalent cross-links, formation of cystine, formation of pyroglutamate, formylation, gamma -carboxylation, glycosylation, GPI-anchor formation, hydroxylation, iodination, methylation, myristoylation, oxidation, pegylation, proteolytic processing, phosphorylation, prenylation, racemization, selenoylation, sulfation, transfer-RNA mediated addition of amino acids to proteins such as arginylation, ubiquitination, and the like.<br>
Alternatively, functional variants can be binding molecules as defined in the present invention comprising an amino acid sequence containing substitutions, insertions, deletions or combinations thereof of one or more amino acids compared to the amino acid sequences of the parent binding molecules. Furthermore, functional variants can comprise truncations of the amino acid sequence at either or both the amino or carboxy termini. Functional variants according to the invention may have the same or different, either higher or lower, binding affinities compared to the parent binding molecule but are still capable of binding to rabies virus or a fragment thereof and are still capable of neutralizing rabies virus. For instance, functional variants according to the invention may have increased or decreased binding affinities for rabies virus or a fragment thereof compared to the parent binding molecules or may have a higher or lower rabies virus neutralizing activity. Preferably, the amino acid sequences of the variable regions, including, but not limited to, framework regions, hypervariable regions, in particular the CDR3 regions, are modified. Generally, the light chain and the heavy chain variable regions comprise three hypervariable regions, comprising three CDRs, and more conserved regions, the so-called framework regions (FRs). The hypervariable regions comprise amino acid residues from CDRs and amino acid<br><br>
residues from hypervariable loops. Functional variants intended to fall within the scope of the present invention have at least about 50% to about 99%, preferably at least about 60% to about 99%, more preferably at least about 70% to about 99%, even more preferably at least about 80% to about 99%, most preferably at least about 90% to about 99%, in particular at least about 95% to about 99%, and in particular at least about 97% to about 99% amino acid sequence homology with the parent binding molecules as defined herein. Computer algorithms such as inter alia Gap or Bestfit known to a person skilled in the art can be used to optimally align amino acid sequences to be compared and to define similar or identical amino acid residues.<br>
In another embodiment, functional variants may be produced when the parent binding molecule comprises a glycosylation site in its sequence that results in glycosylation of the binding molecule upon expression in eukaryotic cells and hence might abrogate the binding to the antigen. The functional variant produced no longer contains the glycosylation site, but will be capable of binding to rabies virus and still have neutralizing activity.<br>
Functional variants can be obtained by altering the parent binding molecules or parts thereof by general molecular biology methods known in the art including, but not limited to, error-prone PCR, oligonucleotide-directed mutagenesis and site-directed mutagenesis. Furthermore, the functional variants may have complement fixing activity, be capable of assisting in the lysis of enveloped rabies virus and/or act as opsonins and augment phagocytosis of rabies virus either by promoting its uptake via Fc or C3b receptors or by agglutinating rabies virus to make it more easily phagocytosed.<br><br>
In yet a further aspect, the invention includes immunoconjugates, i.e. molecules comprising at least one binding molecule or functional variant thereof as defined herein and further comprising at least one tag, such as inter alia a detectable moiety/agent. Also contemplated in the present invention are mixtures of immunoconjugates according to the invention or mixtures of at least one immunoconjugate according to the invention and another molecule, such as a therapeutic agent or another binding molecule or immunoconjugate. In a further embodiment, the immunoconjugates of the invention may comprise one or more tags. These tags can be the same or distinct from each other and can be joined/conjugated non-covalently to the binding molecules. The tag(s) can also be joined/conjugated directly to the binding molecules through covalent bonding, including, but not limited to, disulfide bonding, hydrogen bonding, electrostatic bonding, recombinant fusion and conformational bonding. Alternatively, the tag(s) can be joined/conjugated to the binding molecules by means of one or more linking compounds. Techniques for conjugating tags to binding molecules are well known to the skilled artisan.<br>
The tags of the immunoconjugates of the present invention may be therapeutic agents, but preferably they are detectable moieties/agents. Immunoconjugates comprising a detectable agent can be used diagnostically to, for example, assess if a subject has been infected with rabies virus or monitor the development or progression of a rabies virus infection as part of a clinical testing procedure to, e.g., determine the efficacy of a given treatment regimen. However, they may also be used for other detection and/or analytical and/or diagnostic purposes. Detectable moieties/agents include, but are not limited to, enzymes, prosthetic groups, fluorescent<br><br>
materials, luminescent materials, bioluminescent materials, radioactive materials, positron emitting metals, and nonradioactive paramagnetic metal ions.<br>
The tags used to label the binding molecules for detection and/or analytical and/or diagnostic purposes depend on the specific detection/analysis/diagnosis techniques and/or methods used such as inter alia immunohistochemical staining of (tissue) samples, flow cytometric detection, scanning laser cytometric detection, fluorescent immunoassays, enzyme-linked immunosorbent assays (ELISA’s), radioimmunoassays (RIA’s), bioassays (e.g., neutralization assays), Western blotting applications, etc. For immunohistochemical staining of tissue samples preferred labels are enzymes that catalyze production and local deposition of a detectable product. Enzymes typically conjugated to binding molecules to permit their immunohistochemical visualization are well-known and include, but are not limited to, acetylcholinesterase, alkaline phosphatase, beta-galactosidase, glucose oxidase, horseradish peroxidase, and urease. Typical substrates for production and deposition of visually detectable products are also well known to the skilled person in the art. Next to that, immunoconjugates of the invention can be labeled using colloidal gold or they can be labeled with radioisotopes, such as 33P, 32P, 35S, 3H, and 1251. Binding molecules of the invention can be attached to radionuclides directly or indirectly via a chelating agent by methods well known in the art.<br>
When the binding molecules of the present invention are used for flow cytometric detections, scanning laser cytometric detections, or fluorescent immunoassays, they can usefully be labeled with fluorophores. A wide variety of fluorophores useful for fluorescently labeling the binding molecules of the present invention are known to the skilled artisan. When the<br><br>
binding molecules of the present invention are used for secondary detection using labeled avidin, streptavidin, captavidin or neutravidin, the binding molecules may be labeled with biotin to form suitable prosthetic group complexes.<br>
When the immunoconjugates of the invention are used for in vivo diagnostic use, the binding molecules can also be made detectable by conjugation to e.g. magnetic resonance imaging (MRI) contrast agents, such as gadolinium<br>
diethylenetriaminepentaacetic acid, to ultrasound contrast agents or to X-ray contrast agents, or by radioisotopic labeling.<br>
Furthermore, the binding molecules, functional variants thereof or immunoconjugates of the invention can also be attached to solid supports, which are particularly useful for in vitro immunoassays or purification of rabies virus or a fragment thereof. Such solid supports might be porous or nonporous, planar or nonplanar and include, but are not limited to, glass, cellulose, polyaerylamide, nylon, polystyrene, polyvinyl chloride or polypropylene supports. The human binding molecules can also for example usefully be conjugated to filtration media, such as NHS-activated Sepharose or CNBr-activated Sepharose for purposes of immunoaffinity chromatography. They can also usefully be attached to paramagnetic microspheres, typically by biotin-streptavidin interaction. The microspheres can be used for isolation of rabies virus or a fragment thereof from a sample containing rabies virus or a fragment thereof. As another example, the human binding molecules of the present invention can usefully be attached to the surface of a microtiter plate for ELISA.<br><br>
The binding molecules of the present invention or functional variants thereof can be fused to marker sequences, such as a peptide to facilitate purification. Examples include, but are not limited to, the hexa-histidine tag, the hemagglutinin (HA) tag, the myc tag or the flag tag.<br>
Alternatively, an antibody can be conjugated to a second antibody to form an antibody heteroconjugate. In another aspect the human binding molecules of the invention may be conjugated/attached to one or more antigens. Preferably, these antigens are antigens which are recognized by the immune system of a subject to which the binding molecule-antigen conjugate is administered. The antigens may be identical but may also differ from each other. Conjugation methods for attaching the antigens and binding molecules are well known in the art and include, but are not limited to, the use of cross-linking agents. The human binding molecules will bind to rabies virus and the antigens attached to the human binding molecules will initiate a powerful T-cell attack on the conjugate which will eventually lead to the destruction of the rabies virus.<br>
Next to producing immunoconjugates chemically by conjugating, directly or indirectly via for instance a linker, the immunoconjugates can be produced as fusion proteins comprising the human binding molecules of the invention and a suitable tag. Fusion proteins can be produced by methods known in the art such as, e.g., recombinantly by constructing nucleic acid molecules comprising nucleotide sequences encoding the human binding molecules in frame with nucleotide sequences encoding the suitable tag(s) and then expressing the nucleic acid molecules.<br>
It is another aspect of the present invention to provide a nucleic acid molecule encoding at least a binding molecule<br><br>
or functional variant thereof according to the invention. Such nucleic acid molecules can be used as intermediates for cloning purposes, e.g. in the process of affinity maturation described above - In a preferred embodiment, the nucleic acid molecules are isolated or purified.<br>
The skilled man will appreciate that functional variants of these nucleic acid molecules are also intended to be a part of the present invention. Functional variants are nucleic acid sequences that can be directly translated, using the standard genetic code, to provide an amino acid sequence identical to that translated from the parent nucleic acid molecules.<br>
Preferably, the nucleic acid molecules encode binding molecules comprising a CDR3 region, preferably a heavy chain CDR3 region, comprising an amino acid sequence selected from the group consisting of SEQ ID N0:1, SEQ ID N0:2, SEQ ID N0:3, SEQ ID N0:4, SEQ ID N0:5, SEQ ID N0:6, SEQ ID NO:7, SEQ ID N0:8, SEQ ID N0:9, SEQ ID NO:10, SEQ ID N0:11, SEQ ID N0:12, SEQ ID N0:13, SEQ ID N0:14, SEQ ID N0:15, SEQ ID N0:16, SEQ ID N0:17, SEQ ID N0:18, SEQ ID NO:19, SEQ ID NO:20, SEQ ID NO:21, SEQ ID NO:22, SEQ ID NO:23 and SEQ ID NO:24.<br>
Even more preferably, the nucleic acid molecules encode human binding molecules comprising a variable heavy chain comprising essentially an amino acid sequence selected from the group consisting of SEQ ID NO:26, SEQ ID NO:27, SEQ ID NO:28, SEQ ID NO:29, SEQ ID NO:30, SEQ ID N0:31, SEQ ID NO:32, SEQ ID NO:33, SEQ ID NO:34, SEQ ID NO:35, SEQ ID NO:36, SEQ ID NO:37, SEQ ID NO:38, SEQ ID NO:39, SEQ ID NO:40, SEQ ID N0:41, SEQ ID NO:42, SEQ ID NO:43, SEQ ID NO:44, SEQ ID NO:45, SEQ ID NO:46, SEQ ID NO:47, SEQ ID NO:48 and SEQ ID NO:49. In a particularly preferred embodiment the nucleic acid molecules encode binding molecules comprising a variable heavy chain<br><br>
comprising essentially an amino acid sequence comprising amino acids 1-119 of SEQ ID NO:335.<br>
In yet another embodiment, the nucleic acid molecules encode binding molecules comprising a variable heavy chain comprising the amino acid sequence of SEQ ID NO:2 6 and a variable light chain comprising the amino acid sequence of SEQ ID NO:50, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:27 and a variable light chain comprising the amino acid sequence of SEQ ID NO:51, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:28 and a variable light chain comprising the amino acid sequence of SEQ ID NO:52, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:2 9 and a variable light chain comprising the amino acid sequence of SEQ ID NO:53, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:30 and a variable light chain comprising the amino acid sequence of SEQ ID NO:54, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:31 and a variable light chain comprising the amino acid sequence of SEQ ID NO:55, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:32 and a variable light chain comprising the amino acid sequence of SEQ ID NO:56, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:33 and a variable light chain comprising the amino acid sequence of SEQ ID NO:57, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:34 and a variable light chain comprising the amino acid sequence of SEQ ID NO:58, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:35 and a variable light chain comprising the amino acid sequence of SEQ ID NO:59, or they encode a variable heavy chain comprising the amino acid<br><br>
sequence of SEQ ID NO:36 and a variable light chain comprising the amino acid sequence of SEQ ID NO:60, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:37 and a variable light chain comprising the amino acid sequence of SEQ ID NO:61, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:38 and a variable light chain comprising the amino acid sequence of SEQ ID NO:62, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:39 and a variable light chain comprising the amino acid sequence of SEQ ID NO:63, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:40 and a variable light chain comprising the amino acid sequence of SEQ ID NO:64, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:41 and a variable light chain comprising the amino acid sequence of SEQ ID NO:65, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:42 and a variable light chain comprising the amino acid sequence of SEQ ID NO:66, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:43 and a variable light chain comprising the amino acid sequence of SEQ ID NO:67, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:44 and a variable light chain comprising the amino acid sequence of SEQ ID NO:68, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:45 and a variable light chain comprising the amino acid sequence of SEQ ID NO:69, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:4 6 and a variable light chain comprising the amino acid sequence of SEQ ID NO:70, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:47 and a variable light chain comprising the amino acid sequence of SEQ ID NO:71, or they<br><br>
encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:48 and a variable light chain comprising the amino acid sequence of SEQ ID NO:72, or they encode a variable heavy chain comprising the amino acid sequence of SEQ ID NO:49 and a variable light chain comprising the amino acid sequence of SEQ ID NO:73. In a preferred embodiment the nucleic acid molecules encode human binding molecules comprising a variable heavy chain comprising the amino acid sequence comprising amino acids 1-119 of SEQ ID NO:335 and a variable light chain comprising the amino acid sequence comprising amino acids 1-107 of SEQ ID NO:337.<br>
In a specific embodiment of the invention the nucleic acid molecules encoding the variable heavy chain of the binding molecules of the invention comprise essentially a nucleotide sequence selected from the group consisting of SEQ ID NO:74, SEQ ID NO:75, SEQ ID NO:76, SEQ ID NO:77, SEQ ID NO:78, SEQ ID NO:79, SEQ ID NO:80, SEQ ID NO:81, SEQ ID NO:82, SEQ ID NO:83, SEQ ID NO:84, SEQ ID NO:85, SEQ ID NO:86, SEQ ID NO:87, SEQ ID NO:88, SEQ ID NO:897 SEQ ID NO:90, SEQ ID NO:91, SEQ ID NO:92, SEQ ID NO:93, SEQ ID NO:94, SEQ ID NO:95, SEQ ID NO:96 and SEQ ID NO:97. Preferably, the nucleic acid molecules encoding the variable heavy chain of the binding molecules of the invention comprise essentially a nucleotide sequence comprising nucleotides 1-357 of SEQ ID NO:334.<br>
In yet another specific embodiment of the present invention, the nucleic acid molecules encoding the variable light chain of the binding molecules of the invention comprise essentially a nucleotide sequence selected of the group consisting of SEQ ID NO:98, SEQ ID NO:99, SEQ ID NO:100, SEQ ID NO:101, SEQ ID NO:102, SEQ ID NO:103, SEQ ID NO:104, SEQ ID NO:105, SEQ ID NO:106, SEQ ID NO:107, SEQ ID NO:108, SEQ ID NO:109, SEQ ID NO:110, SEQ ID NO:lll, SEQ ID NO:112, SEQ ID<br><br>
N0:113, SEQ ID N0:114, SEQ ID N0:115, SEQ ID N0:116, SEQ ID N0:117, SEQ ID N0:118, SEQ ID N0:119, SEQ ID NO:120 and SEQ ID NO:121. Preferably, the nucleic acid molecules encoding the variable light chain of the human binding molecules of the invention comprise essentially a nucleotide sequence comprising nucleotides 1-321 of SEQ ID NO:336.<br>
It is another aspect of the invention to provide vectors, i.e. nucleic acid constructs, comprising one or more nucleic acid molecules according to the present invention. Vectors can be derived from plasmids such as inter alia F, Rl, RP1, Col, pBR322, TOL, Ti, etc; cosmids; phages such as lambda, lambdoid, M13, Mu, PI, P22, Qp, T-even, T-odd, T2, T4, T7, etc; plant viruses such as inter alia alfalfa mosaic virus, bromovirus, capillovirus, carlavirus, carmovirus, caulivirus, clostervirus, comovirus, cryptovirus, cucumovirus, dianthovirus, fabavirus, fijivirus, furovirus, geminivirus, hordeivirus, ilarvirus, luteovirus, machlovirus, marafivirus, necrovirus, nepovirus, phytorepvirus, plant rhabdovirus, potexvirus, potyvirus, sobemovirus, tenuivirus, tobamovirus, tobravirus, tomato spotted wilt virus, tombusvirus, tymovirus, etc; or animal viruses such as inter alia adenovirus, arenaviridae, baculoviridae, birnaviridae, bunyaviridae, calciviridae, cardioviruses, coronaviridae, corticoviridae, cystoviridae, Epstein-Barr virus, enteroviruses, filoviridae, flaviviridae, Foot-and-Mouth disease virus, hepadnaviridae, hepatitis viruses, herpesviridae, immunodeficiency viruses, influenza virus, inoviridae, iridoviridae, orthomyxoviridae, papovaviruses, paramyxoviridae, parvoviridae, picornaviridae, poliovirus, polydnaviridae, poxviridae, reoviridae, retroviruses, rhabdoviridae, rhinoviruses, Semliki Forest virus, tetraviridae, togaviridae, toroviridae, vaccinia virus, vescular stomatitis virus, etc. Vectors can be used for<br><br>
cloning and/or for expression of the human binding molecules of the invention and might even be used for gene therapy purposes. Vectors comprising one or more nucleic acid molecules according to the invention operably linked to one or more expression-regulating nucleic acid molecules are also covered by the present invention. The choice of the vector is dependent on the recombinant procedures followed and the host used. Introduction of vectors in host cells can be effected by inter alia calcium phosphate transfection, virus infection, DEAE-dextran mediated transfection, lipofectamin transfection or electroporation. Vectors may be autonomously replicating or may replicate together with the chromosome into which they have been integrated. Preferably, the vectors contain one or more selection markers. The choice of the markers may depend on the host cells of choice, although this is not critical to the invention as is well known to persons skilled in the art. They include, but are not limited to, kanamycin, neomycin, puromycin, hygromycin, zeocin, thymidine kinase gene from Herpes simplex virus (HSV-TK), dihydrofolate reductase gene from mouse (dhfr). Vectors comprising one or more nucleic acid molecules encoding the human binding molecules as described above operably linked to one or more nucleic acid molecules encoding proteins or peptides that can be used to isolate the binding molecules are also covered by the invention. These proteins or peptides include, but are not limited to, glutathione-S-transferase, maltose binding protein, metal-binding polyhistidine, green fluorescent protein, luciferase and beta-galactosidase.<br>
Hosts containing one or more copies of the vectors mentioned above are an additional subject of the present invention. Preferably, the hosts are host cells. Host cells include, but are not limited to, cells of mammalian, plant,<br><br>
insect, fungal or bacterial origin. Bacterial cells include, but are not limited to, cells from Gram positive bacteria such as several species of the genera Bacillus, Streptomyces and Staphylococcus or cells of Gram negative bacteria such as several species of the genera Escherichia, such as E. coli, and Pseudomonas. In the group of fungal cells preferably yeast cells are used. Expression in yeast can be achieved by using yeast strains such as inter alia Pichia pastoris, Saccharomyces cerevisiae and Hansenula polymorpha. Furthermore, insect cells such as cells from Drosophila and Sf9 can be used as host cells. Besides that, the host cells can be plant cells. Transformed (transgenic) plants or plant cells are produced by known methods, for example, Agrobacterium-mediated gene transfer, transformation of leaf discs, protoplast transformation by polyethylene glycol-induced DNA transfer, electroporation, sonication, microinjection or bolistic gene transfer. Additionally, a suitable expression system can be a baculovirus system. Expression systems using mammalian cells such as Chinese Hamster Ovary (CHO) cells, COS cells, BHK cells or Bowes melanoma cells are preferred in the present invention. Mammalian cells provide expressed proteins with posttranslational modifications that are most similar to natural molecules of mammalian origin. Since the present invention deals with molecules that may have to be administered to humans, a completely human expression system would be particularly preferred. Therefore, even more preferably, the host cells are human cells. Examples of human cells are inter alia HeLa, 911, AT1080, A549, 293 and HEK293T cells. Preferred mammalian cells are human retina cells such as 911 cells or the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury,<br><br>
Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6® (PER.C6 is a registered trademark of Crucell Holland B.V.). For the purposes of this application “PER.C6” refers to cells deposited under number 96022940 or ancestors, passages up-stream or downstream as well as descendants from ancestors of deposited cells, as well as derivatives of any of the foregoing.<br>
In preferred embodiments, the human producer cells comprise at least a functional part of a nucleic acid sequence encoding an adenovirus El region in expressible format. In even more preferred embodiments, said host cells are derived from a human retina and immortalised with nucleic acids comprising adenoviral El sequences, such as the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6®. Production of recombinant proteins in host cells can be performed according to methods well known in the art. The use of the cells marketed under the trademark PER.C6® as a production platform for proteins of interest has been described in WO 00/63403 the disclosure of which is incorporated herein by reference in its entirety.<br>
A method of producing a binding molecule or a functional variant according to the invention is an additional part of the invention. The method comprises the steps of a) culturing a host according to the invention under conditions conducive to the expression of the binding molecule or functional variant thereof, and b) optionally, recovering the expressed binding molecule or functional variant thereof. The expressed binding molecules or functional variants thereof can be recovered from the cell free extract, but preferably they are<br><br>
recovered from the culture medium. Methods to recover proteins, such as binding molecules, from cell free extracts or culture medium are well known to the man skilled in the art. Binding molecules or functional variants thereof as obtainable by the above described method are also a part of the present invention.<br>
Alternatively, next to the expression in hosts, such as host cells, the binding molecules or functional variants thereof of the invention can be produced synthetically by conventional peptide synthesizers or in cell-free translation systems using RNA nucleic acid derived from DNA molecules according to the invention. Binding molecule or functional variants thereof as obtainable by the above described synthetic production methods or cell-free translation systems are also a part of the present invention.<br>
In another embodiment binding molecules or functional variants thereof according to the present invention may be generated by transgenic non-human mammals, such as for instance transgenic mice or rabbits, that express human immunoglobulin genes. Preferably, the transgenic non-human mammals have a genome comprising a human heavy chain transgene and a human light chain transgene encoding all or a portion of the human binding molecules as described above. The transgenic non-human mammals can be immunized with a purified or enriched preparation of rabies virus or a fragment thereof. Protocols for immunizing non-human mammals are well established in the art. See Using Antibodies: A Laboratory Manual, Edited by: E. Harlow, D. Lane (1998), Cold Spring Harbor Laboratory, Cold Spring Harbor, New York and Current Protocols in Immunology, Edited by: J.E. Coligan, A.M. Kruisbeek, D.H. Margulies, E.M. Shevach, W. Strober (2001), John Wiley &amp; Sons Inc., New York, the disclosures of which are incorporated herein by reference.<br><br>
In a further aspect, the invention provides a method of identifying binding molecules such as human monoclonal antibodies or fragments thereof according to the invention or nucleic acid molecules according to the invention capable of specifically binding to rabies virus and comprises the steps of a) contacting a collection of binding molecules on the surface of replicable genetic packages with the rabies virus or a fragment thereof under conditions conducive to binding, b) selecting at least once for replicable genetic packages binding to the rabies virus or the fragment thereof, and c) separating and recovering the replicable genetic packages binding to the rabies virus or the fragment thereof.<br>
The selection step may be performed in the presence of rabies virus. The rabies virus may be isolated or non-isolated, e.g. present in serum and/or blood of an infected individual. In another embodiment the rabies virus is inactivated. Alternatively, the selection step may be performed in the presence of a fragment of rabies virus such as an extracellular part of the rabies virus, one or more (poly)peptides derived from rabies virus such as the G protein, fusion proteins comprising these proteins or (poly)peptides, and the like. In another embodiment cells transfected with rabies virus G protein are used for selection procedures.<br>
In yet a further aspect, the invention provides a method of obtaining a binding molecule or a nucleic acid molecule according to the invention, wherein the method comprises the steps of a) performing the above described method of identifying binding molecules, such as human monoclonal antibodies or fragments thereof according to the invention, or nucleic acid molecules according to the invention, and b) isolating from the recovered replicable genetic packages the<br><br>
binding molecule and/or the nucleic acid encoding the binding molecule. Once a new monoclonal antibody has been established or identified with the above mentioned method of identifying binding molecules or nucleic acid molecules encoding the binding molecules, the DNA encoding the scFv or Fab can be isolated from the bacteria or replicable genetic packages and combined with standard molecular biological techniques to make constructs encoding bivalent scFvs or complete human immunoglobulins of a desired specificity (e.g. IgG, IgA or IgM). These constructs can be transfected into suitable cell lines and complete human monoclonal antibodies can be produced (see Huls et al., 1999; Boel et al., 2000).<br>
A replicable genetic package as used herein can be prokaryotic or eukaryotic and includes cells, spores, bacteria, viruses, (bacterio)phage and polysomes. A preferred replicable genetic package is a phage. The human binding molecules, such as for instance single chain Fv’s, are displayed on the replicable genetic package, i.e. they are attached to a group or molecule located at an exterior surface of the replicable genetic package. The replicable genetic package is a screenable unit comprising a human binding molecule to be screened linked to a nucleic acid molecule encoding the binding molecule. The nucleic acid molecule should be replicable either in vivo (e.g., as a vector) or in vitro (e.g., by PCR, transcription and translation). In vivo replication can be autonomous (as for a cell), with the assistance of host factors (as for a virus) or with the assistance of both host and helper virus (as for a phagemid). Replicable genetic packages displaying a collection of human binding molecules are formed by introducing nucleic acid molecules encoding exogenous binding molecules to be displayed into the genomes of the replicable genetic packages to form<br><br>
fusion proteins with endogenous proteins that are normally expressed from the outer surface of the replicable genetic packages. Expression of the fusion proteins, transport to the outer surface and assembly results in display of exogenous binding molecules from the outer surface of the replicable genetic packages. In a further aspect the invention pertains to a human binding molecule capable of binding rabies virus or a fragment thereof and being obtainable by the identification method as described above.<br>
In yet a further aspect the invention relates to a method of identifying a binding molecule potentially having neutralizing activity against rabies virus, wherein the method comprises the steps of (a) contacting a collection of binding molecules on the surface of replicable genetic packages with the rabies virus under conditions conducive to binding, (b) separating and recovering binding molecules that bind to the rabies virus from binding molecules that do not bind, (c) isolating at least one recovered binding molecule, (d) verifying if the binding molecule isolated has neutralizing activity against the rabies virus, characterized in that the rabies virus in step a is inactivated. The inactivated rabies virus may be purified before being inactivated. Purification may be performed by means of well known purification methods suitable for viruses such as for instance centrifugation through a glycerol cushion. The inactivated rabies virus in step a may be immobilized to a suitable material before use. Alternatively, the rabies virus in step a may still be active. In another alternative embodiment a fragment of a rabies virus, such as a polypeptide of a rabies virus such as the G protein, is used in step a. In yet another embodiment cells transfected with rabies virus G protein are used for selecting binding molecule potentially having neutralizing activity<br><br>
against rabies virus. As indicated herein, when cells expressing rabies virus G protein were included in the selection method the number of selected neutralizing antibodies was higher compared to selection methods wherein only purified rabies virus G protein and/or inactivated rabies virus was used.<br>
In a further embodiment the method of identifying a binding molecule potentially having neutralizing activity against rabies virus as described above further comprises the step of separating and recovering, and optionally isolating, human binding molecules containing a variable heavy 3-30 germline gene. A person skilled in the art can identify the specific germline gene by methods known in the art such as for instance nucleotide sequencing. The step of separating and recovering binding molecules containing a variable heavy 3-30 germline gene can be performed before or after step c. As indicated below the majority of rabies virus neutralizing human monoclonal antibodies found in the present invention comprises this specific VH germline gene.<br>
Phage display methods for identifying and obtaining (neutralizing) binding molecules, e.g. antibodies, are by now well-established methods known by the person skilled in the art. They are e.g. described in US Patent Number 5,696,108; Burton and Barbas, 1994; de Kruif et al., 1995b; and Phage Display: A Laboratory Manual. Edited by: CF Barbas, DR Burton, JK Scott and GJ Silverman (2001), Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York. All these references are herewith incorporated herein in their entirety.<br>
For the construction of phage display libraries, collections of human monoclonal antibody heavy and light chain variable region genes are expressed on the surface of bacteriophage, preferably filamentous bacteriophage,<br><br>
particles, in for example single-chain Fv (scFv) or in Fab format (see de Kruif et al., 1995b). Large libraries of antibody fragment-expressing phages typically contain more than 1.0*109 antibody specificities and may be assembled from the immunoglobulin V regions expressed in the B lymphocytes of immunized- or non-immunized individuals. In a specific embodiment of the invention the phage library of human binding molecules, preferably scFv phage library, is prepared from RNA isolated from cells obtained from a subject that has been vaccinated against rabies or exposed to a rabies virus. RNA can be isolated from inter alia bone marrow or peripheral blood, preferably peripheral blood lymphocytes. The subject can be an animal vaccinated or exposed to rabies virus, but is preferably a human subject which has been vaccinated or has been exposed to rabies virus. Preferably the human subject has been vaccinated. A collection of human binding molecules on the surface of replicable genetic packages, such as a scFv phage library, as described above is another aspect of the present invention.<br>
Alternatively, phage display libraries may be constructed from immunoglobulin variable regions that have been partially assembled in vitro to introduce additional antibody diversity in the library (semi-synthetic libraries). For example, in vitro assembled variable regions contain stretches of synthetically produced, randomized or partially randomized DNA in those regions of the molecules that are important for antibody specificity, e. g. CDR regions. Rabies virus specific phage antibodies can be selected from the libraries by immobilising target antigens such as antigens from rabies virus on a solid phase and subsequently exposing the target antigens to a phage library to allow binding of phages expressing antibody fragments specific for the solid phase-<br><br>
bound antigen(s). Non-bound phages are removed by washing and bound phages eluted from the solid phase for infection of Escherichia coli {E.coli) bacteria and subsequent propagation. Multiple rounds of selection and propagation are usually required to sufficiently enrich for phages binding specifically to the target antigen(s). If desired, before exposing the phage library to target antigens the phage library can first be subtracted by exposing the phage library to non-target antigens bound to a solid phase. Phages may also be selected for binding to complex antigens such as complex mixtures of rabies virus proteins or (poly)peptides, host cells expressing one or more rabies virus proteins or (poly)peptides of rabies virus, or (inactivated) rabies virus itself. Antigen specific phage antibodies can be selected from the library by incubating a solid phase with bound thereon a preparation of inactivated rabies virus with the phage antibody library to let for example the scFv or Fab part of the phage bind to the proteins/polypeptides of the rabies virus preparation. After incubation and several washes to remove unbound and loosely attached phages, the phages that have bound with their scFv or Fab part to the preparation are eluted and used to infect Escherichia coli to allow amplification of the new specificity. Generally, one or more selection rounds are required to separate the phages of interest from the large excess of non-binding phages. Alternatively, known proteins or (poly)peptides of the rabies virus can be expressed in host cells and these cells can be used for selection of phage antibodies specific for the proteins or (poly)peptides. A phage display method using these host cells can be extended and improved by subtracting non-relevant binders during screening by addition of an excess of host cells comprising no target molecules or non-target<br><br>
molecules that are similar, but not identical, to the target, and thereby strongly enhance the chance of finding relevant binding molecules (This process is referred to as the MAbstract® process. MAbstract® is a registered trademark of Crucell Holland B.V., see also US Patent Number 6,265,150 which is incorporated herein by reference).<br>
In yet a further aspect, the invention provides compositions comprising at least one binding molecule, at least one functional variant or fragment thereof, at least one immunoconjugate according to the invention or a combination thereof. The compositions may further comprise inter alia stabilising molecules, such as albumin or polyethylene glycol, or salts. Preferably, the salts used are salts that retain the desired biological activity of the human binding molecules and do not impart any undesired toxicological effects. If necessary, the human binding molecules of the invention may be coated in or on a material to protect them from the action of acids or other natural or non-natural conditions that may inactivate the binding molecules.<br>
In yet a further aspect, the invention provides compositions comprising at least one nucleic acid molecule as defined in the present invention. The compositions may comprise aqueous solutions such as aqueous solutions containing salts (e.g., NaCl or salts as described above), detergents (e.g., SDS) and/or other suitable components.<br>
Furthermore, the present invention pertains to pharmaceutical compositions comprising at least one n binding molecule according to the invention, at least one functional variant or fragment thereof, at least one immunoconjugate according to the invention, at least one composition according to the invention, or combinations thereof. The pharmaceutical<br><br>
composition of the invention further comprises at least one pharmaceutically acceptable excipient.<br>
In a preferred embodiment the pharmaceutical composition according to the invention comprises at least one additional binding molecule, i.e. the pharmaceutical composition can be a cocktail/mixture of binding molecules. The pharmaceutical composition may comprise at least two binding molecules according to the invention or at least one binding molecule according to the invention and at least one further anti-rabies virus binding molecule. Said further binding molecule preferably comprises a CDR3 region comprising the amino acid sequence of SEQ ID NO:25. The binding molecule comprising the CDR3 region comprising the amino acid sequence of SEQ ID NO:25 may be a chimeric or humanized monoclonal antibody or functional fragment thereof, but preferably it is a human monoclonal antibody or functional fragment thereof. In an embodiment, the binding molecule comprises a heavy chain variable region comprising the amino acid sequence SEQ ID NO:273. In another embodiment, the binding molecule comprises a light chain variable region comprising the amino acid sequence SEQ ID NO:275. In yet another embodiment the binding molecule comprises a heavy and light chain comprising the amino acid sequences of SEQ ID NO:123 and SEQ ID NO:125, respectively. The binding molecules in the pharmaceutical composition should be capable of reacting with different, non-competing epitopes of the rabies virus. The epitopes may be present on the G protein of rabies virus and may be different, non-overlapping epitopes. The binding molecules should be of high affinity and should have a broad specificity. Preferably, they neutralize as many fixed and street strains of rabies virus as possible. Even more preferably, they also exhibit neutralizing activity towards other genotypes of the<br><br>
Lyssavirus genus or even with other viruses of the rhabdovirus family, while exhibiting no cross-reactivity with other viruses or normal cellular proteins. Preferably, the binding molecule is capable of neutralizing escape variants of the other binding molecule in the cocktail.<br>
Another aspect of the present invention pertains to a pharmaceutical composition comprising at least two rabies virus neutralizing binding molecules, preferably (human) binding molecules according to the invention, characterized in that the binding molecules are capable of reacting with different, non-competing epitopes of the rabies virus. In an embodiment the pharmaceutical composition comprises a first rabies virus neutralizing binding molecule which is capable of reacting with an epitope located in antigenic site I of the rabies virus G protein and a second rabies virus neutralizing binding molecule which is capable of reacting with an epitope located in antigenic site III of the rabies virus G protein. The antigenic structure of the rabies glycoprotein was initially defined by Lafon et al. (1983). The antigenic sites were identified using a panel of mouse mAbs and their respective mAb resistant virus variants. Since then, the antigenic sites have been mapped by identification of the amino acid mutations in the glycoprotein of mAb resistant variants (see Seif et al., 1985; Prehaud et al., 1988; and Benmansour et al., 1991). The majority of rabies neutralizing mAbs are directed against antigenic site II (see Benmansour et al., 1991), which is a discontinuous conformational epitope comprising of amino acid 34-42 and amino acid 198-200 (see Prehaud et al., 1988). Antigenic site III is a continuous conformational epitope at amino acid 330-338 and harbors two charged residues, K330 and R333, that affect viral pathogenicity (see Seif et al., 1985; Coulon et al., 1998; and<br><br>
Dietzschold et al., 1983). The conformational antigenic site I was defined by only one mAb, 509-6, and located at amino acid 231 (see Benmansour et al., 1991; and Lafon et a_Z., 1983). Antigenic site IV is known to harbor overlapping linear epitopes (see Tordo, 1996; Bunschoten et al., 1989; Luo et al., 1997; and Ni et al.f 1995). Benmansour et al. (1991) also described the presence of minor site a located at position 342-343, which is distinct from antigenic site III despite its close proximity. Alignment of the CR-57 epitope with the currently known linear and conformational neutralizing epitopes on rabies glycoprotein (Figure 10) revealed that the CR-57 epitope is located in the same region as the conformational antigenic site I, defined by the single mAb 509-6. Based on nucleotide and amino acid sequences of the glycoprotein of the escape viruses of CR04-098, the epitope recognized by this antibody appears to be located in the same region as the continuous conformational antigenic site III.<br>
In a preferred embodiment the pharmaceutical composition comprises a first rabies virus neutralizing binding molecule comprising at least a CDR3 region, preferably heavy chain CDR3 region, comprising the amino acid sequence of SEQ ID NO:25 and a second rabies virus neutralizing binding molecule comprising at least a CDR3 region, preferably heavy chain CDR3 region, comprising the amino acid sequence selected from the group consisting of SEQ ID NO:4, SEQ ID NO:10, SEQ ID NO:14, SEQ ID NO:15, SEQ ID NO:16 and SEQ ID NO:22. More preferably, the second rabies virus neutralizing binding molecule comprises at least a CDR3 region, preferably heavy chain CDR3 region, comprising the amino acid sequence of SEQ ID NO:14. Preferably, the first rabies virus neutralizing binding molecule comprises a heavy and light chain comprising the amino acid sequences of SEQ ID NO:123 and SEQ ID NO:125,<br><br>
respectively, and the second rabies virus neutralizing binding molecule comprises a heavy and light chain comprising the amino acid sequences of SEQ ID NO:335 and SEQ ID NO:337, respectively. Preferably, the heavy and light chain of the first rabies virus neutralizing binding molecule are encoded by SEQ ID NO:122 and SEQ ID NO:124, respectively, and the heavy and light chain of the second rabies virus neutralizing binding molecule are encoded by SEQ ID NO:334 and SEQ ID NO:336, respectively.<br>
A pharmaceutical composition comprising two binding molecules, wherein the pI of the binding molecules is divergent may have a problem when choosing a suitable buffer which optimally stabilizes both binding molecules. When adjusting the pH of the buffer of the composition such that it increases the stability of one binding molecule, this might decrease the stability of the other binding molecule. Decrease of stability or even instability of a binding molecule may lead to its precipitation or aggregation or to its spontaneous degradation resulting in loss of the functionality of the binding molecule. Therefore, in another aspect the invention provides a pharmaceutical composition comprising at least two binding molecules, preferably human binding molecules, characterized in that the binding molecules have isoelectric points (pI) that differ less than about 1.5, 1.4, 1.3, 1.2, 1.1, 1.0, 0.9, 0.8, 0.7, 0.6, 0.5, 0.4, 0.3, preferably less than (and including) 0.25 pI units from one another. The pI can be measured experimentally, e.g. by means of isoelectric focusing, or be calculated based on the amino acid sequence of the binding molecules. In an embodiment the binding molecules are binding molecules according to the present invention and the pharmaceutical composition is a pharmaceutical composition according to the invention. Preferably, the binding molecules<br><br>
are monoclonal antibodies, e.g. human monoclonal antibodies such as IgGl antibodies. Preferably, the binding molecules are capable of binding to and/or neutralizing an infectious agent, e.g. a virus, a bacterium, a yeast, a fungus or a parasite. In an embodiment the binding molecules are capable of binding to and/or neutralizing a lyssavirus, e.g. rabies virus. In a specific embodiment both binding molecules have a calculated pI that is in the range between 8.0-9.5, preferably 8.1-9.2, more preferably 8.2-8.5. Preferably, the binding molecules have the heavy chain CDR3 region of SEQ ID NO:14 and SEQ ID NO:25, respectively.<br>
In another embodiment the invention provides a cocktail of two or more human or other animal binding molecules, including but not limited to antibodies, wherein at least one binding molecule is derived from an antibody phage or other replicable package display technique and at least one binding molecule is obtainable by a hybridoma technique. When divergent techniques being used, the selection of binding molecules having a compatible pI is also very useful in order to obtain a composition wherein each binding molecule is sufficiently stable for storage, handling and subsequent use.<br>
In another embodiment the binding molecules present in the pharmaceutical composition of the invention augment each others neutralizing activity, i.e. they act synergistically when combined. In other words, the pharmaceutical compositions may exhibit synergistic rabies virus, and even lyssavirus, neutralizing activity. As used herein, the term “synergistic” means that the combined effect of the binding molecules when used in combination is greater than their additive effects when used individually. The ranges and ratios of the components of the pharmaceutical compositions of the invention should be determined based on their individual potencies and tested in<br><br>
in vitro neutralization assays or animal models such as hamsters.<br>
Furthermore, the pharmaceutical composition according to the invention may comprise at least one other therapeutic, prophylactic and/or diagnostic agent. Said further therapeutic and/or prophylactic agents may be anti-viral agents such as ribavirin or interferon-alpha.<br>
The binding molecules or pharmaceutical compositions of the invention can be tested in suitable animal model systems prior to use in humans. Such animal model systems include, but are not limited to, mice, rats, hamsters, monkeys, etc.<br>
Typically, pharmaceutical compositions must be sterile and stable under the conditions of manufacture and storage. The human binding molecules, variant or fragments thereof, immunoconjugates, nucleic acid molecules or compositions of the present invention can be in powder form for reconstitution in the appropriate pharmaceutically acceptable excipient before or at the time of delivery. In the case of sterile powders for the preparation of sterile injectable solutions, the preferred methods of preparation are vacuum drying and freeze-drying (lyophilization) that yield a powder of the active ingredient plus any additional desired ingredient from a previously sterile-filtered solution thereof.<br>
Alternatively, the binding molecules, variant or fragments thereof, immunoconjugates, nucleic acid molecules or compositions of the present invention can be in solution and the appropriate pharmaceutically acceptable excipient can be added and/or mixed before or at the time of delivery to provide a unit dosage injectable form. Preferably, the pharmaceutically acceptable excipient used in the present invention is suitable to high drug concentration, can maintain proper fluidity and, if necessary, can delay absorption.<br><br>
The choice of the optimal route of administration of the pharmaceutical compositions will be influenced by several factors including the physico-chemical properties of the active molecules within the compositions, the urgency of the clinical situation and the relationship of the plasma concentrations of the active molecules to the desired therapeutic effect. For instance, if necessary, the human binding molecules of the invention can be prepared with carriers that will protect them against rapid release, such as a controlled release formulation, including implants, transdermal patches, and microencapsulated delivery systems. Biodegradable, biocompatible polymers can inter alia be used, such as ethylene vinyl acetate, poly-anhydrides, poly-glycolic acid, collagen, poly-orthoesters, and poly-lactic acid. Furthermore, it may be necessary to coat the human binding molecules with, or co-administer the binding molecules with, a material or compound that prevents the inactivation of the human binding molecules. For example, the human binding molecules may be administered to a subject in an appropriate carrier, for example, liposomes, or a diluent.<br>
The routes of administration can generally be divided into two main categories, oral and parenteral administration. The preferred administration of the human binding molecules and pharmaceutical compositions of the invention is into and around the wound and intramuscularly in the gluteal region. Formulations of the human binding molecules and pharmaceutical compositions are dependent on the routes of administration.<br>
In a further aspect, the binding molecules, functional variants, immunoconjugates, compositions, or pharmaceutical compositions of the invention can be used as a medicament. So, a method of treatment and/or prevention of a lyssavirus infection using the human binding molecules, functional<br><br>
variants, immunoconjugates, compositions, or pharmaceutical compositions of the invention is another part of the present invention. The lyssavirus can be a virus from any of the known genotypes, but is preferably rabies virus. The above-mentioned molecules or compositions can be used in the postexposure prophylaxis of rabies.<br>
The molecules or compositions mentioned above may be employed in conjunction with other molecules useful in diagnosis, prophylaxis and/or treatment of rabies virus. They can be used in vitro, ex vivo or in vivo. For instance, the human binding molecules, functional variants, immunoconjugates or pharmaceutical compositions of the invention can be co-administered with a vaccine against rabies. Alternatively, the vaccine may also be administered before or after administration of the molecules or compositions of the invention. Administration of the molecules or compositions of the invention with a vaccine is suitable for post exposure prophylaxis. Rabies vaccines include, but are not limited to, purified chick embryo cell (PCEC) vaccine (RabAvert), human diploid cell vaccine (HDCV; Imovax vaccine) or rabies vaccine adsorbed (RVA).<br>
The molecules are typically formulated in the compositions and pharmaceutical compositions of the invention in a therapeutically or diagnostically effective amount. Dosage regimens can be adjusted to provide the optimum desired response (e.g., a therapeutic response). A suitable dosage range may for instance be 0.1-100 IU/kg body weight, preferably 1.0-50 IU/kg body weight and more preferably 10-30 IU/kg body weight, such as 20 IU/kg body weight.<br>
Preferably, a single bolus of the binding molecules or pharmaceutical compositions of the invention are administered. The molecules and pharmaceutical compositions according to the<br><br>
present invention are preferably sterile. Methods to render these molecules and compositions sterile are well known in the art. The dosing regimen of post exposure prophylaxis is administration of five doses of rabies vaccine intramuscularly in the deltoid muscle on days 0, 3, 7, 14 and 28 days after exposure in individuals not previously immunized against rabies virus. The human binding molecules or pharmaceutical compositions according to the invention should be administered into and around the wounds on day 0 or otherwise as soon as possible after exposure, with the remaining volume given intramuscularly at a site distant from the vaccine. Non-vaccinated individuals are advised to be administered anti-rabies virus human binding molecules, but it is clear to the skilled artisan that vaccinated individuals in need of such treatment may also be administered anti-rabies virus human binding molecules.<br>
In another aspect, the invention concerns the use of binding molecules or functional variants thereof, immunoconjugates according to the invention, nucleic acid molecules according to the invention, compositions or pharmaceutical compositions according to the invention in the preparation of a medicament for the diagnosis, prophylaxis, treatment, or combination thereof, of a condition resulting from an infection by a lyssavirus. The lyssavirus can be a virus from any of the known genotypes but is preferably rabies virus. Preferably the molecules mentioned above are used in the preparation of a medicament for the post exposure prophylaxis of rabies.<br>
Next to that, kits comprising at least one binding molecule according to the invention, at least one functional variant thereof according to the invention, at least one immunoconjugate according to the invention, at least one<br><br>
nucleic acid molecule according to the invention, at least one composition according to the invention, at least one pharmaceutical composition according to the invention, at least one vector according to the invention, at least one host according to the invention or a combination thereof are also a part of the present invention. Optionally, the above described components of the kits of the invention are packed in suitable containers and labeled for diagnosis, prophylaxis and/or treatment of the indicated conditions. The above-mentioned components may be stored in unit or multi-dose containers, for example, sealed ampoules, vials, bottles, syringes, and test tubes, as an aqueous, preferably sterile, solution or as a lyophilized, preferably sterile, formulation for reconstitution. The containers may be formed from a variety of materials such as glass or plastic and may have a sterile access port (for example the container may be an intravenous solution bag or a vial having a stopper pierceable by a hypodermic injection needle). The kit may further comprise more containers comprising a pharmaceutically acceptable buffer, such as phosphate-buffered saline, Ringer? s solution and dextrose solution. It may further include other materials desirable from a commercial and user standpoint, including other buffers, diluents, filters, needles, syringes, culture medium for one or more of the suitable hosts. Associated with the kits can be instructions customarily included in commercial packages of therapeutic, prophylactic or diagnostic products, that contain information about for example the indications, usage, dosage, manufacture, administration, contraindications and/or warnings concerning the use of such therapeutic, prophylactic or diagnostic products.<br>
Currently, HRIG products are used for post exposure prophylaxis of rabies. An adult dose of HRIG of 1500 IU (75 kg<br><br>
individual 2 0 IU/kg) is only available in a volume of 10 ml. More concentrated HRIG products are not possible as the currently obtainable 10 ml dose contains 1-1.5 gram of total IgG. In view thereof the current HRIG products have two drawbacks. Firstly, it is often not anatomically feasible to administer the recommended full dose in and around the bite wounds and secondly the administration of the current volume dose of HRIG is associated with significant pain. The present invention gives a solution to these drawbacks as it provides a pharmaceutical composition comprising a full adult dose in a volume of approximately 2 ml or less, if desirable. Such a pharmaceutical composition may comprise for example two binding molecules capable of neutralizing rabies virus, preferably CR57 and CR04-098. The pharmaceutical composition further comprises a pharmaceutically acceptable excipient and has a volume of around 2 ml. More is also possible, but less desirable in view of the pain associated with injecting larger volumes. Less than 2 ml is also possible. The pharmaceutical composition comprises the full adult dose (in IU) necessary for successful post exposure prophylaxis. In an embodiment the pharmaceutical composition is stored in 10 ml vial such as for instance a 10 ml ready-to-use vial (type I glass) with a stopper. By providing a 10 ml vial the option is given to dilute the pharmaceutical composition towards a higher volume in case an individual presents a large wound surface area. The invention also provides a kit comprising at least a container (such as a vial) comprising the pharmaceutical composition. The kit may further comprise a second container which holds a diluent suitable for diluting the pharmaceutical composition towards a higher volume. Suitable diluents include, but are not limited to, the pharmaceutically acceptable excipient of the pharmaceutical composition and a saline solution.<br><br>
Furthermore, the kit may comprise instructions for diluting the pharmaceutical composition and/or instructions for administering the pharmaceutical composition, whether diluted or not.<br>
The invention further pertains to a method of detecting a rabies virus in a sample, wherein the method comprises the steps of a) contacting a sample with a diagnostically effective amount of a binding molecule, a functional variant or an immunoconjugate according to the invention, and b) determining whether the binding molecule, functional variant, or immunoconjugate specifically binds to a molecule of the sample. The sample may be a biological sample including, but not limited to blood, serum, tissue or other biological material from (potentially) infected subjects. The (potentially) infected subjects may be human subjects, but also animals that are suspected as carriers of rabies virus might be tested for the presence of rabies virus using the human binding molecules, functional variants or immunoconjugates of the invention. The sample may first be manipulated to make it more suitable for the method of detection. Manipulation means inter alia treating the sample suspected to contain and/or containing rabies virus in such a way that the rabies virus will disintegrate into antigenic components such as proteins, (poly)peptides or other antigenic fragments. Preferably, the binding molecules, functional variants or immunoconjugates of the invention are contacted with the sample under conditions which allow the formation of an immunological complex between the human binding molecules and rabies virus or antigenic components thereof that may be present in the sample. The formation of an immunological complex, if any, indicating the presence of rabies virus in the sample, is then detected and measured by suitable means.<br><br>
Such methods include, inter alia, homogeneous and heterogeneous binding immunoassays, such as radioimmunoassays (RIA) , ELISA, immunofluorescence, immunohistocheiriistry, FACS, B1ACORE and Western blot analyses.<br>
Furthermore, the binding molecules of the invention can be used to identify epitopes of rabies virus proteins such as the G protein. The epitopes can be linear, but also structural and/or conformational. In one embodiment, binding of binding molecules of the invention to a series of overlapping peptides, such as 15-mer peptides, of a protein from rabies virus such as the rabies virus G protein can be analyzed by means of PEPSCAN analysis (see inter alia WO 84/03564, WO 93/09872, Slootstra et al. 1996). The binding of human binding molecules to each peptide can be tested in a PEPSCAN-based enzyme-linked immuno assay (ELISA). In another embodiment, a random peptide library comprising peptides from rabies virus proteins can be screened for peptides capable of binding to the human binding molecules of the invention. In the above assays the use of rabies virus neutralizing human binding molecules may identify one or more neutralizing epitopes. The peptides/epitopes found can be used as vaccines and for the diagnosis of rabies.<br>
In a further aspect, the invention provides a method of screening a binding molecule or a functional variant of a binding molecule for specific binding to a different, preferably non-overlapping epitope of rabies virus as the epitope bound by a binding molecule or functional variant of the invention, wherein the method comprises the steps of a) contacting a binding molecule or a functional variant to be screened, a binding molecule or functional variant of the invention and rabies virus or a fragment thereof (such as for instance the rabies virus G protein), b) measure if the<br><br>
binding molecule or functional variant to be screened is capable of competing for specifically binding to the rabies virus or fragment thereof with the binding molecule or functional variant of the invention. If no competition is measured the binding molecules or functional variants to be screened bind to a different epitope. In a specific embodiment of the above screening method, human binding molecules or functional variants thereof may be screened to identify human binding molecules or functional variants capable of binding a different epitope than the epitope recognized by the binding molecule comprising the CDR3 region comprising the amino acid sequence of SEQ ID NO:25. Preferably, the epitopes are non-overlapping or non-competing. It is clear to the skilled person that the above screening method can also be used to identify binding molecules or functional variants thereof capable of binding to the same epitope. In a further step it may be determined if the screened binding molecules that are not capable of competing for specifically binding to the rabies virus or fragment thereof have neutralizing activity. It may also be determined if the screened binding molecules that are capable of competing for specifically binding to the rabies virus or fragment thereof have neutralizing activity. Neutralizing anti-rabies virus binding molecules or functional variants thereof found in the screening method are another part of the present invention. In the screening method “specifically binding to the same epitope” also contemplates specific binding to substantially or essentially the same epitope as the epitope bound by the human binding molecules of the invention. The capacity to block, or compete with, the binding of the human binding molecules of the invention to rabies virus typically indicates that a binding molecule to be screened binds to an epitope or binding site on the rabies<br><br>
virus that structurally overlaps with the binding site on the rabies virus that is immunospecifically recognized by the binding molecules of the invention. Alternatively, this can indicate that a binding molecule to be screened binds to an epitope or binding site which is sufficiently proximal to the binding site immunospecifically recognized by the binding molecules of the invention to sterically or otherwise inhibit binding of the binding molecules of the invention to rabies virus or a fragment thereof.<br>
In general, competitive inhibition is measured by means of an assay, wherein an antigen composition, i.e. a composition comprising rabies virus or fragments (such as G proteins) thereof, is admixed with reference binding molecules and binding molecules to be screened. In an embodiment the reference binding molecule may be one of the human binding molecules of the invention and the binding molecule to be screened may be another human binding molecule of the invention. In another embodiment the reference binding molecule may be the binding molecule comprising the CDR3 region comprising the amino acid sequence of SEQ ID NO:25 and the binding molecule to be screened may be one of the human binding molecules of the invention. In yet another embodiment the reference binding molecule may be one of the human binding molecule of the invention and the binding molecule to be screened may be the binding molecule comprising the CDR3 region comprising the amino acid sequence of SEQ ID NO:25. Usually, the binding molecules to be screened are present in excess. Protocols based upon ELISAs are suitable for use in such simple competition studies. In certain embodiments, one may pre-mix the reference binding molecules with varying amounts of the binding molecules to be screened (e.g., 1:10, 1:20, 1:30, 1:40, 1:50, 1:60, 1:70, 1:80, 1:90 or 1:100) for a<br><br>
period of time prior to applying to the antigen composition. In other embodiments, the reference binding molecules and varying amounts of binding molecules to be screened can simply be admixed during exposure to the antigen composition. In any event, by using species or isotype secondary antibodies one will be able to detect only the bound reference binding molecules, the binding of which will be reduced by the presence of a binding molecule to be screened that recognizes substantially the same epitope. In conducting a binding molecule competition study between a reference binding molecule and any binding molecule to be screened (irrespective of species or isotype), one may first label the reference binding molecule with a detectable label, such as, e.g., biotin, an enzymatic, a radioactive or other label to enable subsequent identification. In these cases, one would pre-mix or incubate the labeled reference binding molecules with the binding molecules to be screened at various ratios (e.g., 1:10, 1:20, 1:30, 1:40, 1:50, 1:60, 1:70, 1:80, 1:90 or 1:100) and (optionally after a suitable period of time) then assay the reactivity of the labeled reference binding molecules and compare this with a control value in which no potentially competing binding molecule was included in the incubation. The assay may again be any one of a range of immunological assays based upon antibody hybridization, and the reference binding molecules would be detected by means of detecting their label, e.g., using streptavidin in the case of biotinylated reference binding molecules or by using a chromogenic substrate in connection with an enzymatic label (such as 3,3 T 5,5’-tetramethylbenzidine (TMB) substrate with peroxidase enzyme) or by simply detecting a radioactive label. A binding molecule to be screened that binds to the same epitope as the reference binding molecule will be able to effectively compete for<br><br>
binding and thus will significantly reduce reference binding molecule binding, as evidenced by a reduction in bound label. Binding molecules binding different non-competing epitopes will show no reduction. The reactivity of the (labeled) reference binding molecule in the absence of a completely irrelevant binding molecule would be the control high value. The control low value would be obtained by incubating the labeled reference binding molecule with unlabelled reference binding molecules of exactly the same type, when competition would occur and reduce binding of the labeled reference binding molecule. In a test assay, a significant reduction in labeled reference binding molecule reactivity in the presence of a binding molecule to be screened is indicative of a binding molecule that recognizes the same epitope, i.e., one that “cross-reacts” with the labeled reference binding molecule. If no reduction is shown, the binding molecule may bind a different non-competing epitope.<br>
Binding molecules identified by these competition assays (“competitive binding molecules”) include, but are not limited to, antibodies, antibody fragments and other binding agents that bind to an epitope or binding site bound by the reference binding molecule as well as antibodies, antibody fragments and other binding agents that bind to an epitope or binding site sufficiently proximal to an epitope bound by the reference binding molecule for competitive binding between the binding molecules to be screened and the reference binding molecule to occur. Preferably, competitive binding molecules of the invention will, when present in excess, inhibit specific binding of a reference binding molecule to a selected target species by at least 10%, preferably by at least 25%, more preferably by at least 50%, and most preferably by at least 75%-90% or even greater. The identification of one or more<br><br>
competitive binding molecules that bind to about, substantially, essentially or at the same epitope as the binding molecules of the invention is a straightforward technical matter. As the identification of competitive binding molecules is determined in comparison to a reference binding molecule, it will be understood that actually determining the epitope to which the reference binding molecule and the competitive binding molecule bind is not in any way required in order to identify a competitive binding molecule that binds to the same or substantially the same epitope as the reference binding molecule. Alternatively, binding molecules binding to different non-competing epitopes identified by these competition assays may also include, but are not limited to, antibodies, antibody fragments and other binding agents.<br>
In another aspect the invention provides a method of identifying a binding molecule potentially having neutralizing activity against an infectious agent causing disease in a living being or a nucleic acid molecule encoding a binding molecule potentially having neutralizing activity against an infectious agent causing disease in a living being, wherein the method comprises the steps of a) contacting a collection of binding molecules on the surface of replicable genetic packages with at least a cell expressing a protein of the infectious agent causing disease in a living being on its surface under conditions conducive to binding, b) separating and recovering binding molecules that bind to the cell expressing a protein of the infectious agent causing disease in a living being on its surface from binding molecules that do not bind said cell, c) isolating at least one recovered binding molecule, d) verifying if the binding molecule isolated has neutralizing activity against the infectious agent causing disease in a living being. The cell expressing a<br><br>
protein of the infectious agent causing disease in a living being on its surface can be a cell transfected with the protein. A person skilled in the art is aware that antigens of the infectious agent other than proteins can also be successfully used in the method. In a specific embodiment the cell is a PER.C6® cell. However, other (El-immortalized) cell lines could also be used to express the proteins such as BHK, CHO, NSO, HEK293, or 911 cells. In an embodiment the binding molecule is human. The infectious agent can be a virus, a bacterium, a yeast, a fungus or a parasite. In an embodiment the protein is a protein normally expressed on the surface of the infectious agent or comprises at least a part of a protein that is surface accessible. In a specific embodiment the collection of binding molecules on the surface of replicable genetic packages are subtracted/counterselected with the cells used for expressing of the protein of the infectious agent, i.e. the cells are identical to the cells used in step a with the proviso that they do not express the protein of the infectious agent on their surface. The cells used for subtraction/counterselection can be untransfected cells. Alternatively, the cells can be transfected with a protein or (extracellular) part thereof that is similar and/or highly homologous in sequence or structure with the respective protein of the infectious agent and/or that is derived from an infectious agent of the same family or even genus.<br>
Another aspect of the invention pertains to a binding molecule as defined herein having rabies virus neutralizing activity, characterized in that the human binding molecule comprises at least a heavy chain CDR3 region comprising the amino acid sequence comprising SEQ ID NO:25 and further characterized in that the human binding molecule has a rabies virus neutralizing activity of at least 2500 IU/mg protein.<br><br>
More preferably, said human binding molecule has a rabies virus neutralizing activity of at least 2800 IU/mg protein, at least 3000 IU/mg protein, at least 3200 IU/mg protein, at least 3400 IU/mg protein, at least 3600 IU/mg protein, at least 3800 IU/mg protein, at least 4000 IU/mg protein, at least 4200 IU/mg protein, at least 4400 IU/mg protein, at least 4600 IU/mg protein, at least 4800 IU/mg protein, at least 5000 IU/mg protein, at least 5200 IU/mg protein, at least 5400 IU/mg protein. The neutralizing activity of the binding molecule was measured by an in vitro neutralization assay (modified RFFIT (rapid fluorescent focus inhibition test)). The assay is described in detail in the example section infra.<br>
In an embodiment the binding molecule comprises a variable heavy chain comprising the amino acid sequence comprising SEQ ID NO:273. In another embodiment the binding molecule comprises a heavy chain comprising the amino acid sequence comprising SEQ ID NO:123. The variable light chain of the binding molecule may comprise the amino acid sequence comprising SEQ ID NO:275. The light chain of the binding molecule may comprise the amino acid sequence comprising SEQ ID NO:125.<br>
A nucleic acid molecule encoding the binding molecules as described above is also a part of the present invention. Preferably, the nucleic acid molecule comprises the nucleotide sequence comprising SEQ ID NO:122. In addition the nucleic acid molecule may also comprise the nucleotide sequence comprising SEQ ID NO:124. A vector comprising the nucleic acid molecules and a host cell comprising such a vector are also provided herein. Preferably, the host cell is a mammalian cell such as a human cell. Examples of cells suitable for production of human binding molecules are inter alia HeLa,<br><br>
911, AT1080, A549, 293 and HEK293T cells. Preferred mammalian cells are human retina cells such as 911 cells or the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6® (PER.C6 is a registered trademark of Crucell Holland B.V.). Eor the purposes of this application “PER.C6” refers to cells deposited under number 96022940 or ancestors, passages up-stream or downstream as well as descendants from ancestors of deposited cells, as well as derivatives of any of the foregoing.<br>
EXAMPLES To illustrate the invention, the following examples are provided. The examples are not intended to limit the scope of the invention in any way.<br>
Example 1<br>
Epitope recognition of human anti-rabies antibodies CR-5 7 and<br>
CR-JB<br>
To address whether the human monoclonal antibodies called CR-57 and CR-JB recognize non-overlapping, non-competing epitopes, escape viruses of the human monoclonal antibodies called CR-57 and CR-JB were generated. CR-57 and CR-JB were generated essentially as described (see Jones et al.r 2003), via introduction of the variable heavy and light chain coding regions of the corresponding antibody genes into a single human IgGl expression vector named pcDNA3002(Neo). The resulting vectors pgS057Cll and pgSOJBCll were used for transient expression in cells from the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR,<br><br>
Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6®. The nucleotide and amino acid sequences of the heavy and light chains of these antibodies are shown in SEQ ID NO:122 - 129, respectively. Serial dilutions (0.5 ml) of rabies virus strain CVS-11 (dilutions ranging from 10”1 - 10”8) were incubated with a constant amount (~4 IU/ml) of antibody CR-57 or CR-JB (0.5 ml) for 1 hour at 37°C/5% C02 before addition to wells containing mouse neuroblastoma cells (MNA cells) or BSR cells (Baby Hamster Kidney-like cell line). After 3 days of selection in the presence of either human monoclonal antibody CR-57 or CR-JB, medium (1 ml) containing<br>
potential escape viruses was harvested and stored at 4°C until further use. Subsequently, the cells were acetone-fixed for 20<br>
minutes at 4°C, and stained overnight at 37°C/5% CO2 with an anti-rabies N-FITC antibody conjugate (Centocor). The number of foci per well were scored by immunofluorescence and medium of wells containing one to six foci were chosen for virus amplification. All E57 escape viruses were generated from 1 single focus with the exception of E57B1 (3 foci). EJB escape viruses were isolated from 1 focus (EJB3F), 3 foci (EJB2B, 4 foci (EJB2C), 5 foci (EJB2E, 2F), or 6 foci (EJB2D), respectively. Each escape virus was first amplified on a small scale on BSR or MNA cells depending on their growth characteristics. These small virus batches were then used to further amplify the virus on a large scale on MNA or BSR cells. Amplified virus was then titrated on MNA cells to determine the titer of each escape virus batch as well as the optimal dilution of the escape virus (giving 80-100 % infection after 24 hours) for use in a virus neutralization assay.<br><br>
Modified RFFIT (rapid fluorescent focus inhibition test) assays were performed to examine cross-protection of E57 (the escape viruses of CR-57) and EJB (the escape viruses of CR-JB) with CR-JB and CR-57, respectively. Therefore, CR-57 or CR-JB was diluted by serial threefold dilutions starting with a 1:5 dilution. Rabies virus (strain CVS-11) was added to each dilution at a concentration that gives 80-100% infection.<br>
Virus/IgG mix was incubated for 1 hour at 37°C/5% C02 before<br>
addition to MNA cells. 24 hours post-infection (at 34°C/5% C02)<br>
the cells were acetone-fixed for 20 minutes at 4°C, and stained for minimally 3 hours with an anti-rabies virus N-FITC antibody conjugate (Centocor). The wells were then analyzed for rabies virus infection under a fluorescence microscope to determine the 50% endpoint dilution. This is the dilution at which the virus infection is blocked by 50% in this assay. To calculate the potency, an international standard (Rabies Immune Globulin Lot R3, Reference material from the laboratory of Standards and Testing DMPQ/CBER/FDA) was included in each modified RFFIT. The 50% endpoint dilution of this standard corresponds with a potency of 2 IU/ml. The neutralizing potency of the single human monoclonal antibodies CR-57 and CR-JB as well as the combination of these antibodies were tested.<br>
EJB viruses were no longer neutralized by CR-JB or CR-57 (see Table 1), suggesting both antibodies bound to and induced amino acid changes in similar regions of the rabies virus glycoprotein. E57 viruses were no longer neutralized by CR-57, whereas 4 out of 6 E57 viruses were still neutralized by CR-JB, although with a lower potency (see Table 1). A mixture of the antibodies CR-57 and CR-JB (in a 1:1 IU/mg ratio) gave<br><br>
similar results as observed with the single antibodies (data not shown).<br>
To identify possible mutations in the rabies virus glycoprotein the nucleotide sequence of the glycoprotein open reading frame (ORF) of each of the EJB and E57 escape viruses was determined. Viral RNA of each of the escape viruses and CVS-11 was isolated from virus-infected MNA cells and converted into cDNA by standard RT-PCR. Subsequently, cDNA was used for nucleotide sequencing of the rabies virus glycoprotein ORFs in order to identify mutations.<br>
Both E57 and EJB escape viruses showed mutations in the same region of the glycoprotein (see Figure 1 and 2, respectively; see for all the sequences described in Figures 1 and 2 SEQ ID NO:130 - 151). This indicates that both antibodies recognize overlapping epitopes. From the above can be concluded that the combination of CR-57 and CR-JB in a cocktail does not prevent the escape of neutralization-resistant variants and is therefore not an ideal immunoglobulin preparation for rabies post exposure prophylaxis.<br>
Example 2<br>
Construction of a ScFv phage display library using peripheral<br>
blood lymphocytes of rabies vaccinated donors.<br>
From four rabies vaccinated human subjects 50 ml blood was drawn from a vein one week after the last boost. Peripheral blood lymphocytes (PBL) were isolated from these blood samples using Ficoll cell density fractionation. The blood serum was saved and frozen at -20°C. The presence of anti-rabies antibodies in the sera was tested positive using a FACS staining on rabies virus glycoprotein transfected 293T cells. Total RNA was prepared from the PBL using organic phase<br><br>
separation (TRIZOL M) and subsequent ethanol precipitation. The obtained RNA was dissolved in DEPC-treated ultrapure water and the concentration was determined by OD 260 nm measurement. Thereafter, the RNA was diluted to a concentration of 100 ng/pl. Next, 1 pg of RNA was converted into cDNA as follows: To 10 pI total RNA, 13 pI DEPC-treated ultrapure water and 1 111 random hexamers (500 ng/pl) were added and the obtained mixture was heated at 65CC for 5 minutes and quickly cooled on wet-ice. Then, 8 pI 5X First-Strand buffer, 2 pI dNTP (10 mM each), 2 ul DTT (0.1 M) , 2 pI Rnase-inhibitor (40 TJ/\il) and 2 lil Superscript™III MMLV reverse transcriptase (200 U/pl) were added to the mixture, incubated at room temperature for 5 minutes and incubated for 1 hour at 50°C. The reaction was terminated by heat inactivation, i.e. by incubating the mixture for 15 minutes at 75°C.<br>
The obtained cDNA products were diluted to a final volume of 200 pI with DEPC-treated ultrapure water. The OD 260 nm of a 50 times diluted solution (in 10 mM Tris buffer) of the dilution of the obtained cDNA products gave a value of 0.1.<br>
For each donor 5 to 10 pI of the diluted cDNA products were used as template for PCR amplification of the immunoglobulin gamma heavy chain family and kappa or lambda light chain sequences using specific oligonucleotide primers (see Tables 2-7). PCR reaction mixtures contained, besides the diluted cDNA products, 25 pmol sense primer and 25 pmol anti-sense primer in a final volume of 50 pI of 20 mM Tris-HCl (pH 8.4), 50 mM KC1, 2.5 mM MgC12, 250 pM dNTPs and 1.25 units Taq polymerase. In a heated-lid thermal cycler having a temperature of 96°C, the mixtures obtained were quickly melted for 2 minutes, followed by 30 cycles of: 30 seconds at 96CC, 30 seconds at 60°C and 60 seconds at 72°C-<br><br>
In a first round amplification, each of seventeen light chain variable region sense primers (eleven for the lambda light chain (see Table 2) and six for the kappa light chain (see Table 3) were combined with an anti-sense primer recognizing the C-kappa called HuCk 5’-ACACTCTCCCCTGTTGAAGCT CTT-3’ (see SEQ ID NO:152) or C-lambda constant region HuCA2 5’-TGAACATTCTGTAGGGGCCACTG~3’ (see SEQ ID NO:153) and HuCA7 5’-AGAGCATTCTGCAGGGGCCACTG~3’ (see SEQ ID NO:154) (the HuCX2 and HuCX7 anti-sense primers were mixed to equimolarity before use), yielding 4 times 17 products of about 600 basepairs. These products were purified on a 2% agarose gel and isolated from the gel using Qiagen gel-extraction columns. 1/10 of each of the isolated products was used in an identical PCR reaction as described above using the same seventeen sense primers, whereby each lambda light chain sense primer was combined with one of the three Jlambda-region specific anti-sense primers and each kappa light chain sense primer was combined with one of the five Jkappa-region specific anti-sense primers. The primers used in the second amplification were extended with restriction sites (see Table 4) to enable directed cloning in the phage display vector PDV-C06 (see Figure 3 and SEQ ID NO:155). This resulted in 4 times 63 products of approximately 350 basepairs that were pooled to a total of 10 fractions. This number of fractions was chosen to maintain the natural distribution of the different light chain families within the library and not to over or under represent certain families. The number of alleles within a family was used to determine the percentage of representation within a library (see Table 5). In the next step, 2.5 μg of pooled fraction and 100 μg PDV-C06 vector were digested with Sail and NotI and purified from gel. Thereafter, a ligation was performed overnight at 16°C as follows. To 500 ng PDV-C06 vector 70 ng pooled<br><br>
fraction was added in a total volume of 50 μl ligation mix containing 50 mM Tris-HCl (pH 7.5), 10 mM MgCl2, 10 mM DTT, 1 mM ATP, 25 μg/ml BSA and 2.5 μl T4 DNA Ligase (400 U/μl) . This procedure was followed for each pooled fraction. The ligation mixes were purified by phenol/chloroform, followed by a chloroform extraction and ethanol precipitation, methods well known to the skilled artisan. The DNA obtained was dissolved in 50 μl ultrapure water and per ligation mix two times 2,5 yl aliquots were electroporated into 40 μl of TGI competent E. coli bacteria according to the manufacturer’s protocol (Stratagene). Transformants were grown overnight at 37°C in a total of 30 dishes (three dishes per pooled fraction; dimension of dish: 240 mm x 240 mm) containing 2TY agar supplemented with 50 yg/ml ampicillin and 4.5% glucose. A (sub)library of variable light chain regions was obtained by scraping the transformants from the agar plates. This (sub)library was directly used for plasmid DNA preparation using a Qiagen™ QIAFilter MAXI prep kit.<br>
For each donor the heavy chain immunoglobulin sequences were amplified from the same cDNA preparations in a similar two round PCR procedure and identical reaction parameters as described above for the light chain regions with the proviso that the primers depicted in Tables 6 and 7 were used. The first amplification was performed using a set of nine sense directed primers (see Table 6; covering all families of heavy chain variable regions) each combined with an IgG specific constant region anti-sense primer called HuCIgG 5’-GTC CAC CTT GGT GTT GCT GGG CTT-3’ (SEQ ID NO:156) yielding four times nine products of about 650 basepairs. These products were purified on a 2% agarose gel and isolated from the gel using Qiagen gel-extraction columns. 1/10 of each of the isolated products was used in an identical PCR reaction as described<br><br>
above using the same nine sense primers, whereby each heavy chain sense primer was combined with one of the four JH-region specific anti-sense primers. The primers used in the second round were extended with restriction sites (see Table 7) to enable directed cloning in the light chain (sub)library vector. This resulted per donor in 36 products of approximately 350 basepairs. These products were pooled for each donor per used (VH) sense primer into nine fractions. The products obtained were purified using Qiagen PCR Purification columns. Next, the fractions were digested with Sfil and Xhol and ligated in the light chain (sub)library vector, which was cut with the same restriction enzymes, using the same ligation procedure and volumes as described above for the light chain (sub)library. Alternatively, the fractions were digested with Ncol and Xhol and ligated in the light chain vector, which was cut with the same restriction enzymes, using the same ligation procedure and volumes as described above for the light chain (sub)library. Ligation purification and subsequent transformation of the resulting definitive library was also performed as described above for the light chain (sub)library and at this point the ligation mixes of each donor were combined per VH pool. The transformants were grown in 27 dishes (three dishes per pooled fraction; dimension of dish: 240 mm x 240 mm) containing 2TY agar supplemented with 50 μg/ml ampicillin and 4.5% glucose. All bacteria were harvested in 2TY culture medium containing 50 μg/ml ampicillin and 4.5% glucose, mixed with glycerol to 15% (v/v) and frozen in 1.5 ml aliquots at -80°C. Rescue and selection of each library were performed as described below.<br>
Example 3<br><br>
Selection of phages carrying single chain Fv fragments specifically recognizing rabies virus glycoprotein<br>
Antibody fragments were selected using antibody phage display libraries, general phage display technology and MAbstract® technology, essentially as described in US Patent Number 6,265,150 and in WO 98/15833 (both of which are incorporated by reference herein). The antibody phage libraries used were two different semi-synthetic scFv phage libraries (JK1994 and WT2000) and the immune scFv phage libraries (RAB-03-G01 and RAB-04-G01) prepared as described in Example 2 above. The first semi-synthetic scFv phage library (JK1994) has been described in de Kruif et al. (1995b), the second one (WT2000) was build essentially as described in de Kruif et al. (1995b). Briefly, the library has a semi-synthetic format whereby variation was incorporated in the heavy and light chain V genes using degenerated oligonucleotides that incorporate variation within CDR regions. Only VH3 heavy chain genes were used, in combination with kappa- and lambda light chain genes. CDRl and CDR3 of the heavy chain and CDR3 of the light chain were recreated synthetically in a PCR-based approach similar as described in de Kruif et al. (1995b). The thus created V region genes were cloned sequentially in scFv format in a phagemid vector and amplified to generate a phage library as described before. Furthermore, the methods and helper phages as described in WO 02/103012 (incorporated by reference herein) were used in the present invention. For identifying phage antibodies recognizing rabies virus glycoprotein phage selection experiments were performed using whole rabies virus (rabies virus Pitman-Moore strain) inactivated by treatment with beta-propiolactone, purified rabies virus glycoprotein (rabies<br><br>
virus ERA strain), and/or transfected cells expressing rabies virus G protein (rabies virus ERA strain).<br>
The G protein was purified from the rabies virus ERA strain as follows. To a virus solution, 1/10 volume of 10% octyl-beta-glucopyranoside was added and mixed gently. Upon a 30 minutes incubation at 4°C the virus sample was centrifuged (36,000 rpm, 4°C) in a SW51 rotor. The supernatant was collected and dialyzed overnight at 4°C against 0.1 M Tris/EDTA. Subsequently, the glycoprotein was collected from<br>
the dialysis chamber, aliquotted, and stored at -80°C until further use. The protein concentration was determined by OD 280 nm and the integrity of the G protein was analyzed by SDS-PAGE.<br>
Whole inactivated rabies virus or rabies virus G protein were diluted in phosphate buffered saline (PBS), 2-3 ml was added to MaxiSorp Nunc-Immuno Tubes (Nunc) and incubated<br>
overnight at 4°C on a rotating wheel. An aliquot of a phage library (500 μl, approximately 1013 cfu, amplified using CT helper phage (see WO 02/103012)) was blocked in blocking buffer (2% Protifar in PBS) for 1-2 hours at room temperature. The blocked phage library was added to the immunotube (either preincubated with or without CR-57 scFv to block the epitope recognized by CR-57), incubated for 2 hours at room temperature, and washed with wash buffer (0.1% Tween-20 (Serva) in PBS) to remove unbound phages. Bound phages were then eluted from the antigen by incubation for 10 minutes at room temperature with 1 ml of 50 mM Glycine-HCl pH 2.2. Subsequently, the eluted phages were mixed with 0.5 ml of 1 M Tris-HCl pH 7.5 to neutralize the pH. This mixture was used to infect 5 ml of a XLl-Blue JE. coli culture that had been grown at 37°C to an OD 600 nm of approximately 0.3. The phages were<br><br>
allowed to infect the XLl-Blue bacteria for 30 minutes at 37°C. Then, the mixture was centrifuged for 10 minutes, at 3200*g at room temperature and the bacterial pellet was resuspended in 0.5 ml 2-trypton yeast extract (2TY) medium. The obtained bacterial suspension was divided over two 2TY agar plates supplemented with tetracyclin, ampicillin and glucose. After incubation overnight of the plates at 37°C, the colonies were scraped from the plates and used to prepare an enriched phage library, essentially as described by De Kruif et al. (1995a) and WO 02/103012. Briefly, scraped bacteria were used to inoculate 2TY medium containing ampicillin, tetracycline and glucose and grown at a temperature of 37°C to an OD 600 nm of -0.3. CT helper phages were added and allowed to infect the bacteria after which the medium was changed to 2TY containing ampicillin, tetracycline and kanamycin. Incubation was continued overnight at 30°C. The next day, the bacteria were removed from the 2TY medium by centrifugation after which the phages in the medium were precipitated using polyethylene glycol (PEG) 6000/NaCl. Finally, the phages were dissolved in 2 ml of PBS with 1% bovine serum albumin (BSA), filter-sterilized and used for the next round of selection. Phage selections were also performed with rabies virus glycoprotein transfected cells. The cells used were cells from the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6®. They are hereinafter referred to as PER.C6® cells. Here, the blocked phage library (2 ml) was first added to 1*107 subtractor cells (in DMEM/10% FBS) and<br>
incubated for 1 hour at 4°C on a rotating wheel. The subtractor cells were PER.C6® cells that expressed the Vesicular Stomatitis Virus (VSV) glycoprotein ecto domain on their<br><br>
surface fused to the rabies virus transmembrane and cytoplasmic domain. With this subtraction step phages recognizing either VSV glycoprotein or antigens specific for PER.C6 cells were removed from the phage library. The<br>
phage/cell mixture was centrifuged (5 minutes at 4°C at 500xg) to remove cell-bound phages, and the supernatant was added to a new tube containing 3 ml of 1*107 subtractor cells. The subtraction step was repeated twice with the respective supernatant. Subsequently, the subtracted phages were<br>
incubated for 1.5 hours at 4°C on a rotating wheel with the rabies virus glycoprotein expressing transfected cells (PER.C6 cells (3*106 cells)). Before that, the transfected cells were preincubated either with or without CR-57 scFv to block the epitope recognized by CR-57. After incubation the cells were washed five times with 1 ml of DMEM/10%FBS (for each wash, the cells were resuspended and transferred to new tube), phages were eluted and processed as described above.<br>
Typically, two rounds of selections were performed before isolation of individual phage antibodies. After the second round of selection, individual E. coli colonies were used to prepare monoclonal phage antibodies. Essentially, individual colonies were grown to log-phase in 96 well plate format and infected with VCSM13 helper phages after which phage antibody production was allowed to proceed overnight. The produced phage antibodies were PEG/NaCl-precipitated and filter-sterilized and tested in ELISA for binding to both whole inactivated rabies virus and purified rabies virus G protein. From the selection a large panel of phage antibodies was obtained that demonstrated binding to both whole inactivated rabies virus and rabies virus G protein (see example below). Two selection strategies were followed with the above-described immune libraries. In the first strategy 736 phage<br><br>
antibodies were selected after two selection rounds using in the first and second selection round inactivated virus or purified G protein. In the second strategy 736 phage antibodies were selected after two selection rounds using in the first selection round cell surface expressed recombinant G protein and in the second selection round inactivated virus or purified G protein. The number of unique phage antibodies obtained by the first strategy was 97, while the second strategy yielded 70 unique ones. The 97 unique phage antibodies found by means of the first strategy gave rise to 18 neutralizing antibodies and the 70 unique clones identified by means of the second strategy yielded 33 neutralizing antibodies.1 This clearly demonstrates that selections that included rabies virus glycoprotein tranfected cells, i.e. cell surface expressed recombinant G protein, as antigen appeared to yield more neutralizing antibodies compared to selections using only purified G protein and/or inactivated virus.<br>
Example 4<br>
Validation of the rabies virus glycoprotein specific single-chain phage antibodies.<br>
Selected single-chain phage antibodies that were obtained in the screens described above, were validated in ELISA for specificity, i.e. binding to rabies virus G protein, purified as described supra. Additionally, the single-chain phage antibodies were also tested for binding to 5% FBS. For this purpose, the rabies virus G protein or 5% FBS preparation was coated to Maxisorp™ ELISA plates. After coating, the plates were blocked in PBS/1% Protifar for 1 hour at room temperature. The selected single-chain phage antibodies were incubated for 15 minutes in an equal volume of PBS/1% Protifar to obtain blocked phage antibodies. The plates were emptied,<br><br>
and the blocked phage antibodies were added to the wells. Incubation was allowed to proceed for one hour, the plates were washed in PBS containing 0.1% Tween-20 and bound phage antibodies were detected (using OD 492 run measurement) using an anti-M13 antibody conjugated to peroxidase. As a control, the procedure was performed simultaneously using no single-chain phage antibody, a negative control single chain phage antibody directed against CD8 (SC02-007) or a positive control single chain phage antibody directed against rabies virus glycoprotein (scFv S057). As shown in Table 8, the selected phage antibodies called SC04-001, SC04-004, SC04-008, SC04-010, SC04-018, SC04-021, SC04-026, SC04-031, SC04-038, SC04-040, SC04-060, SC04-073, SC04-097, SC04-098, SC04-103, SC04-104, SC04-108, SC04-120, SC04-125, SC04-126, SC04-140, SC04-144, SC04-146, and SC04-164 displayed significant binding to the immobilized purified rabies virus G protein, while no binding to FBS was observed. Identical results were obtained in ELISA using the whole inactivated rabies virus prepared as described supra (data not shown).<br>
Example 5<br>
Characterization of the rabies virus specific scFvs<br>
From the selected specific single chain phage antibody (scFv) clones plasmid DNA was obtained and nucleotide sequences were determined according to standard techniques. The nucleotide sequences of the scFvs (including restriction sites for cloning) called SC04-001, SC04-004, SC04-008, SC04-010, SC04-018, SC04-021, SC04-026, SC04-031, SC04-038, SC04-040, SC04-060, SC04-073, SC04-097, SC04-098, SC04-103, SC04-104, SC04-108, SC04-120, SC04-125, SC04-126, SC04-140, SC04-144, SC04-146, and SC04-164 are shown in SEQ ID NO:157, SEQ ID NO:159, SEQ ID NO:161, SEQ ID NO:163, SEQ ID NO:165, SEQ ID<br><br>
NO:167, SEQ ID NO:169, SEQ ID N0:171, SEQ ID NO:173, SEQ ID NO:175, SEQ ID NO:177, SEQ ID NO:179, SEQ ID N0:181, SEQ ID NO:183, SEQ ID NO:185, SEQ ID NO:187, SEQ ID NO:189, SEQ ID N0:191, SEQ ID NO:193, SEQ ID NO:195, SEQ ID NO:197, SEQ ID NO:199, SEQ ID NO:201 and SEQ ID NO:203, respectively. The amino acid sequences of the scFvs called SC04-001, SC04-004, SC04-008, SC04-010, SC04-018, SC04-021, SC04-026, SC04-031, SC04-038, SC04-040, SC04-060, SC04-073, SC04-097, SC04-098, SC04-103, SC04-104, SC04-108, SC04-120, SC04-125, SC04-126, SC04-140, SC04-144, SC04-146, and SC04-164 are shown in SEQ ID NO:158, SEQ ID NO:160, SEQ ID NO:162, SEQ ID NO:164, SEQ ID NO:166, SEQ ID NO:168, SEQ ID NO:170, SEQ ID NO:172, SEQ ID NO:174, SEQ ID NO:176, SEQ ID NO:178, SEQ ID NO-.180, SEQ ID NO:182, SEQ ID NO:184, SEQ ID NO:186, SEQ ID NO:188, SEQ ID NO:190, SEQ ID NO:192, SEQ ID NO:194, SEQ ID NO:196, SEQ ID NO:198, SEQ ID NO-.200, SEQ ID NO:202 and SEQ ID NO:204, respectively.<br>
The VH and VL gene identity (see Tomlinson IM, Williams SC, Ignatovitch 0, Corbett SJ, Winter G. V-BASE Sequence Directory. Cambridge United Kingdom: MRC Centre for Protein Engineering (1997)) and heavy chain CDR3 compositions of the scFvs specifically binding the rabies virus G protein are depicted in Table 9.<br>
Example 6<br>
In vitro neutralization of rabies virus by rabies virus<br>
specific scFvs (modified RFFIT)<br>
In order to determine whether the selected scFvs were capable of blocking rabies virus infection, in vitro neutralization assays (modified RFFIT) were performed. The scFv preparations were diluted by serial threefold dilutions starting with a 1:5 dilution. Rabies virus (strain CVS-11) was<br><br>
added to each dilution at a concentration that gives 80-100 % infection. Virus/scFv mix was incubated for 1 hour at 37°C/5% CO2 before addition to MNA cells. 24 hours post-infection (at 34°C/5% C02) the cells were acetone-fixed for 20 minutes at<br>
4°C/ and stained for minimally 3 hours with an anti-rabies N-FITC antibody conjugate (Centocor). The cells were then analyzed for rabies virus infection under a fluorescence microscope to determine the 50% endpoint dilution. This is the dilution at which the virus infection is blocked by 50% in this assay (see Example 1). Several scFvs were identified that showed neutralizing activity against rabies virus (see Table 10) .<br>
Additionally, it was investigated by means of the in vitro neutralization assay (modified RFFIT) as described above, if the selected scFvs were capable of neutralizing the E57 escape viruses as prepared in Example 1 (E57A2, E57A3, E57B1, E57B2, E57B3 and E57C3). Several scFvs were identified that showed neutralizing activity against the E57 escape viruses (see Tables 11A and 11B).<br>
Example 7<br>
Rabies virus G protein competition ELISA with scFvs<br>
To identify antibodies that bind to non-overlapping, non-competing epitopes, a rabies glycoprotein competition ELISA was performed. Nunc-Immuno™ Maxisorp F96 plates (Nunc) were coated overnight at 4°C with a 1:1000 dilution of purified rabies virus glycoprotein (1 mg/ml; rabies virus ERA strain) in PBS (50 μl) . Uncoated protein was washed away before the wells were blocked with 100 μl PBS/1% Protifar for 1 hour at room temperature. Subsequently, the blocking solution was discarded and 50 μl of the non-purified anti-rabies virus scFvs<br><br>
in PBS/1% Protifar (2x diluted) was added. Wells were washed five times with 100 μl of PBS/0.05% Tween-20. Then, 50 μl biotinylated anti-rabies virus competitor IgG, CR-57bio, was added to each well, incubated for 5 minutes at room<br>
temperature, and the wells were washed five times with 100 μl<br>
of PBS/0.05% Tween-20. To detect the binding of CR-57bio, 50 μl of a 1:2000 dilution of streptavidin-HRP antibody (Becton Dickinson) was added to the wells and incubated for 1 hour at room temperature. Wells were washed again as above and the ELISA was further developed by addition of 100 μ1 of OPD<br>
reagens (Sigma) . The reaction was stopped by adding 50 μl 1 M H2SO4 before measuring the OD at 492 nm.<br>
The signal obtained with CR-57bio alone could be reduced to background levels when co-incubated with scFv S057, i.e. the scFv form of CR-57 (for nucleotide and amino acid sequence of S057 see SEQ ID NO:205 and 206, respectively) or scFv SOJB, i.e. the scFv form of CR-JB (for nucleotide and amino acid sequence of SOJB see SEQ ID NO:312 and 313, respectively). This indicates that the scFvs S057 and SOJB compete with the interaction of CR-57bio to rabies virus glycoprotein by binding to the same epitope or to an overlapping epitope as CR-57bio, respectively. In contrast, an irrelevant scFv called SC02-007, i.e. a scFv binding to CD8, did not compete for binding. The anti-rabies virus scFvs called SC04-004, SC04-010, SC04-024, SC04-060, SC04-073, SC04-097, SC04-098, SC04-103, SC04-104, SC04-120, SC04-125, SC04-127, SC04-140, SC04-144 and SC04-146 did also not compete with CR-57bio, indicating that these scFvs bind to a different epitope than the epitope recognized by CR-57 (see Figure 4).<br>
Similar results were obtained with the following experiment. First, the rabies virus antibody CR-57 was added<br><br>
to wells coated with rabies virus G protein. Next, the competing scFvs were added. In this set-up the anti-rabies virus scFvs were detected with anti-VSV-HRP by virtue of the presence of a VSV-tag in the scFv amino acid sequences (see Figure 5).<br>
Example 8<br>
Construction of fully human immunoglobulin molecules (human monoclonal anti-rabies virus antibodies) from the selected anti-rabies virus single chain Fv’s<br>
Heavy and light chain variable regions of the scFvs called SC04-001, SC04-008, SC04-018, SC04-040 and SC04-126 were PCR-amplified using oligonucleotides to append restriction sites and/or sequences for expression in the IgG expression vectors pSyn-C03-HCYl (see SEQ ID No:277) and pSyn-C04-CA (see SEQ ID No:278), respectively. The VH and VL genes were amplified using the oligonucleotides as shown in Table 12 and 13, respectively, and the PCR products were cloned into the vectors pSyn-C03-HCYl and pSyn-C04-CA, respectively.<br>
Heavy and light chain variable regions of the scFvs called SC04-0 04, SC04-Q10, SC04-021, SC04-02 6, SC04-031, SC04-038, SC04-060, SC04-073, SC04-097, SC04-098, SC04-103, SC04-104, SC04-108, SC04-120, SC04-125, SC04-140, SC04-144, SC04-146 and SC04-164 were also PCR-amplified using oligonucleotides to append restriction sites and/or sequences for expression in the IgG expression vectors pSyn-C03-HCyl and pSyn-C05-CK (see SEQ ID No:279), respectively. The VH and VL genes were amplified using the oligonucleotides as given in Table 12 and 13, respectively, and the PCR products were cloned into the vectors pSyn-C03-HCyl and pSyn-C05-Ck, respectively. The oligonucleotides are designed such that they correct any deviations from the germline sequence that have<br><br>
been introduced during library construction, due to the limited set of oligonucleotides that have been used to amplify the large repertoire of antibody genes. Nucleotide sequences for all constructs were verified according to standard techniques known to the skilled artisan.<br>
The resulting expression constructs pgGl04-001C03, pgG104-008C03, pgG104-018C03, pgG104-040C03 and pgG104-126C03 encoding the anti-rabies virus human IgGl heavy chains in combination with the relevant pSyn-C04-VA construct encoding the corresponding light chain were transiently expressed in 293T cells and supernatants containing IgGl antibodies were obtained. The expression constructs pgG104-004C0 3, pgG104-010C03, pgG104-021C03, pgGl04-026C03, pgG104-031C03, pgG104-038C03, pgG104-060C03, pgGl04-073C03, pgG104-097C03, pgG104-098C03, pgG104-103C03, pgG104-104C03, pgG104-108C03, pgG104-120C03, pgG104-125C03, pgG104-140C03, pgG104-144C03, pgG104-146C03 and pgG104-164C03 encoding the anti-rabies virus human IgGl heavy chains in combination with the relevant pSyn-C05-VK construct encoding the corresponding light chain were transiently expressed in 293T cells and supernatants containing IgGl antibodies were obtained.<br>
The nucleotide and amino acid sequences of the heavy and light chains of the antibodies called CR04-001, CR04-004, CR04-008, CR04-010, CR04-018, CR04-021, CR04-026, CR04-031, CR04-038, CR04-040, CR04-060, CR04-073, CR04-097, CR04-098, CR04-103, CR04-1G4, CR04-108, CR04-120, CR04-125, CR04-126, CR04-140, CR04-144, CR04-146 and CR04-164 were determined according to standard techniques. Subsequently, the recombinant human monoclonal antibodies were purified over a protein-A column followed by a buffer exchange on a desalting column using standard purification methods used generally for<br><br>
immunoglobulins (see for instance WO 00/63403 which is incorporated by reference herein).<br>
Additionally, for CR04-098r a single human IgGl expression vector named pgG104-098C10 was generated as described above for vectors pgS057Cll and pgSOJBCll encoding CR-57 and CR-JB, respectively (see Example 1). The nucleotide and amino acid sequences of the heavy and light chains of antibody CR04-098 encoded by vector pgG104-098C10 are shown in SEQ ID NO:334 - 337, respectively. Vectors pgS057Cll (see Example 1) and pgG104-098C10 were used for stable expression of CR-57 and CR04-098, respectively, in cells from the cell line deposited at the European Collection of Cell Cultures (ECACC), CAMR, Salisbury, Wiltshire SP4 OJG, Great Britain on 29 February 1996 under number 96022940 and marketed under the trademark PER.C6®. The stably produced CR-57 and CR04-098 have a calculated isoelectric point of 8.22 and 8.46, respectively. The experimentally observed isoelectric points are between 8.1-8.3 for CR-57 and 9.0-9.2 for CR04-098. The recombinant human monoclonal antibodies were purified as described above. Unless otherwise stated, for CR04-001, CR04-004, CR04-008, CR04-010, CR04-018, CR04-021, CR04-026, CR04-031, CR04-038, CR04-040, CR04-060, CR04-073, CR04-097, CR04-098, CR04-103, CR04-104, CR04-108, CR04-120, CR04-125, CR04-126, CR04-140, CR04-144, CR04-146 and CR04-164 use was made of recombinant human monoclonal antibodies transiently expressed by the two vector system as described above and for CR57 use was made of recombinant human monoclonal antibody transiently expressed by the one vector system as described in Example 1.<br><br>
Example 9<br>
Rabies virus G protein competition ELISA with IgGs<br>
To address whether the human monoclonal anti-rabies virus G protein IgGs bind to non-overlapping, non-competing epitopes, competition experiments are performed. Wells with coated rabies virus G protein are incubated with increasing<br>
concentrations (0-50 |xg/ml) of unlabeled anti-rabies virus G<br>
protein IgG for 1 hour at room temperature. Then, 50 μl of a<br>
different biotinylated anti-rabies virus IgG (1 μg/ml) is added to each well, incubated for 5 minutes at room temperature, and immediately washed five times with 100 μ1 of PBS/0.05% Tween-20. Subsequently, wells are incubated for 1 hour at room temperature with 50 μ1 of a 1:2000 dilution of streptavidin-HRP (Becton Dickinson), washed and developed as described above. A decrease in signal with increasing concentration of unlabeled IgG indicates that the two antibodies are competing with each other and recognize the same epitope or overlapping epitopes. Alternatively, wells coated with rabies virus G protein<br>
(ERA strain) were incubated with 50 μg/ml of unlabeled anti-rabies virus G protein IgG for 1 hour at room temperature. Then, 50 \xl of biotinylated CR57 (0.5-5 μg/ml; at subsaturated levels) was added to each well. The further steps were performed as described supra. The signals obtained were compared to the signal obtained with only biotinylated CR57 (see Figure 6; no competitor). From Figure 6 can be deduced that the signal could not be reduced with the antibody called CR02-428 which served as a negative control. In contrast, competition with unlabeled CR57 (positive control) or CR-JB reduced the signal to background levels. From Figure 6 can further be deduced that none of the anti-rabies virus G<br><br>
protein IgGs competed significantly with CR-57, which is in agreement with the scFv competition data as described in Example 7.<br>
In addition, competition experiments were performed on rabies virus G protein (ERA strain) transfected PER.C6 cells by means of flow cytometry. Transfected cells were incubated with<br>
20 μl of unlabeled anti-rabies virus G protein IgG (50 jig/ml)<br>
for 20 minutes at 4 °C. After washing of the cells with PBS<br>
containing 1% BSA, 20 μl of biotinylated CR57 (0.5-5 μq/ml; at subsaturated levels) were added to each well, incubated for 5 minutes at 4°C, and immediately washed twice with 100 μl of PBS containing 1% BSA. Subsequently, wells were incubated for 15<br>
minutes at 4°C with 20 μl of a 1:200 dilution of streptavidin-PE (Caltag), washed and developed as described above. The signal obtained with biotinylated CR57 could not be reduced significantly with the negative control antibody CR02-428 (see Figure 7). In contrast, competition with unlabeled CR57 (positive control) or CR-JB reduced the signal to background levels. None of the anti-rabies virus G protein IgGs competed significantly with CR-57, with the exception of CR04-126 which reduced the signal to approximately 30% (see Figure 7). The latter did not compete in ELISA (see Figure 6). This may be caused by the difference in the way the glycoprotein is presented to the antibody in FACS experiments compared to ELISA experiments. The binding of CR04-126 could be more dependent on the conformation of the glycoprotein, resulting in the competitive effect observed with CR04-126 in the FACS-based competition assay and not in the ELISA-based competition assay. Additionally, CR04-008 and CR04-010 reduced the signal to approximately 50% (see Figure 7) in the FACS-based competition assay indicating that they might compete with<br><br>
CR57. For CR04-010 this was however not confirmed by the scFv competition data or the ELISA-based competition assay. For the other IgGs, the FACS data were in agreement with the respective ELISA data of both the scFvs and the IgGs.<br>
Example 10<br>
Additive/synergistic effects of anti-rabies IgGs in in vitro<br>
neutralization of rabies virus (modified RFFIT)<br>
In order to determine whether the anti-rabies virus G protein IgGs have additive or synergistic effects in neutralization of rabies virus, different combinations of the IgGs are tested. First, the potency (in IU/mg) of each individual antibody is determined in a modified RFFIT (see Example 1). Then, antibody combinations are prepared based on equal amounts of IU/mg and tested in the modified RFFIT. The potencies of each antibody combination can be determined and compared with the expected potencies. If the potency of the antibody combination is equal to the sum of the potencies of each individual antibody present in the combination, the antibodies have an additive effect. If the potency of the antibody combination is higher, the antibodies have a synergistic effect in neutralization of rabies virus.<br>
Alternatively, additive or synergistic effects can be determined by the following experiment. First, the potency of the antibodies to be tested, e.g. CR-57 and CR04-098, is determined in a standard RFFIT (see Laboratory techniques in rabies, Edited by: F.-X Meslin, M.M. Kaplan and H. Koprowski (1996), 4th edition, Chapter 15, World Health Organization, Geneva). Then, the antibodies are mixed in a 1:1 ratio based on IU/ml. This antibody mixture, along with the individual antibodies at the same concentration, are tested in six independent RFFIT experiments to determine the 50%<br><br>
neutralizing endpoint. Subsequently, the combination index (CI) is determined for the antibody mixture using the formula CI = (Cl/Cxl) + (C2/Cx2) + (ClC2/CxlCx2) as described by Chou et al. (1984). CI and C2 are the amount (in \xq) of monoclonal antibody 1 and monoclonal antibody 2 that lead to 50% neutralization when used in combination and Cx1 and Cx2 are<br>
the amount (in μg) of monoclonal antibody 1 and monoclonal antibody 2 that lead to 50% neutralization when used alone. CI = 1, indicates an additive effect, CI  1 indicates an antagonistic effect of the monoclonal antibodies.<br>
Example 11<br>
Identification of epitopes recognized by recombinant human<br>
anti-rabies virus antibodies by PEPSCAN-ELISA<br>
15-mer linear and looped/cyclic peptides were synthesized from the extracellular domain of the G protein of the rabies virus strain ERA (see SEQ ID NO:207 for the complete amino acid sequence of the glycoprotein G of the rabies virus strain ERA, the extracellular domain consists of amino acids 20-458; the protein-id of the glycoprotein of rabies virus strain ERA in the EMBL-database is J02293) and screened using credit-card format mini-PEPSCAN cards (455 peptide formats/card) as described previously (Slootstra et al., 1996; WO 93/09872). All peptides were acetylated at the amino terminus. In all looped peptides position-2 and position-14 were replaced by a cysteine (acetyl-XCXXXXXXXXXXXCX-minicard). If other cysteines besides the cysteines at position-2 and position-14 were present in a prepared peptide, the other cysteines were replaced by an alanine. The looped peptides were synthesized using standard Fmoc-chemistry and deprotected using trifluoric acid with scavengers. Subsequently, the deprotected peptides<br><br>
were reacted on the cards with an 0.5 mM solution of 1,3-bis(bromomethyl)benzene in ammonium bicarbonate (20 mM, pH 7.9/acetonitril (1:1 (v/v)). The cards were gently shaken in the solution for 30-60 minutes, while completely covered in the solution. Finally, the cards were washed extensively with excess of H20 and sonicated in disrupt buffer containing 1%<br>
SDS/0.1% beta-mercaptoethanol in PBS (pH 7.2) at 70°C for 30 minutes, followed by sonication in H2O for another 45 minutes. The human monoclonal antibodies were prepared as described above. Binding of these antibodies to each linear and looped peptide was tested in a PEPSCAN-based enzyme-linked immuno assay (ELISA). The 455-well creditcard-format polypropylene cards, containing the covalently linked peptides, were incubated with the antibodies (10 μg/ml; diluted in blocking solution, which contained 5% horse-serum (v/v) and 5% ovalbumin (w/v)) (4°C, overnight). After washing, the peptides were incubated with anti-human antibody peroxidase (dilution 1/1000) (1 hour, 25°C), and subsequently, after washing the peroxidase substrate 2,2 ? -azino-di-3-ethylbenzthiazoline sulfonate (ABTS) and 2 μl/ml 3% H202 was added. Controls (for linear and looped) were incubated with anti-human antibody peroxidase only. After 1 hour the color development was measured. The color development of the ELISA was quantified with a CCD-camera and an image processing system. The set-up consisted of a CCD-camera and a 55 mm lens (Sony CCD Video Camera XC-77RR, Nikon micro-nikkor 55 mm f/2.8 lens), a camera adaptor (Sony Camera adaptor DC-77RR) and the Image Processing Software package Optimas, version 6.5 (Media Cybernetics, Silver Spring, MD 20910, U.S.A.). Optimas ran on a pentium II computer system.<br>
The human anti-rabies virus G protein monoclonal antibodies were tested for binding to the 15-mer linear and<br><br>
looped/cyclic peptides synthesized as described supra- A peptide is considered to relevantly bind to an antibody when OD values are equal to or higher than two times the average OD-value of all peptides (per antibody). See Table 14 for results of the binding of the human monoclonal antibodies called CR57, CRJB and CR04-010 to the linear peptides of the extracellular domain of glycoprotein G of rabies virus strain ERA, Regions showing significant binding to the respective antibodies are highlighted in grey (see Table 14).<br>
Antibody CR57 bound to the linear peptides having an amino acid sequence selected from the group consisting of SLKGACKLKLCGVLG (SEQ ID NO:314), LKGACKLKLCGVLGL (SEQ ID NO:315), KGACKLKLCGVLGLR (SEQ IDNO:316), GACKLKLCGVLGLRL (SEQ ID NO:317), ACKLKLCGVLGLRLM (SEQ ID NO:318), CKLKLCGVLGLRLMD (SEQ ID NO:319), KLKLCGVLGLRLMDG (SEQ ID NO:320), LKLCGVLGLRLMDGT (SEQ ID NO:321) and KLCGVLGLRLMDGTW (SEQ ID NO:322) (see Table 14). The peptides having the amino acid sequences GACKLKLCGVLGLRL (SEQ ID NO:317), ACKLKLCGVLGLRLM (SEQ ID NO:318) have an OD-value that is lower than twice the average value. Nevertheless these peptides were claimed, because they are in the near proximity of a region of antigenic peptides recognised by antibody CR57. Binding was most prominent to the peptide with the amino acid sequence KLCGVLGLRLMDGTW (SEQ ID NO:322).<br>
Antibody CR04-010 bound to the linear peptides having an amino acid sequence selected from the group consisting of GFGKAYTIFNKTLME (SEQ ID NO:323), FGKAYTIFNKTLMEA (SEQ ID NO:324), GKAYTIFNKTLMEAD (SEQ IDNO:325), KAYTIFNKTLMEADA (SEQ ID NO:326), AYTIFNKTLMEADAH (SEQ ID NO:327), YTIFNKTLMEADAHY (SEQ ID NO:328), TIFNKTLMEADAHYK (SEQ ID NO:329), IFNKTLMEADAHYKS (SEQ ID NO:330) and FNKTLMEADAHYKSV (SEQ ID NO:331). The peptides having the amino acid sequences<br><br>
AYTIFNKTLMEADAH (SEQ ID NO:327), YTIFNKTLMEADAHY (SEQ ID NO:328) have an OD-value that is lower than twice the average value. Nevertheless these peptides were claimed, because they are in the near proximity of a region of antigenic peptides recognized by antibody CR04-010. Binding was most prominent to the peptides with the amino acid sequence TIFNKTLMEADAHYK (SEQ ID NO:329), IFNKTLMEADAHYKS (SEQ ID NO:330) and FNKTLMEADAHYKSV (SEQ ID NO:331).<br>
CRJB and the antibodies called CR04-040, CR04-098 and CR04-103 (data not shown) did not recognize a region of linear antigenic peptides.<br>
Any of the above peptides or parts thereof represents good candidates of a neutralizing epitope of rabies virus and could form the basis for a vaccine or for raising neutralizing antibodies to treat and/or prevent a rabies virus infection.<br>
SLKGACKLKLCGVLGLRLMDGTW (SEQ ID NO:332) and GFGKAYTIFNKTLMEADAHYKSV (SEQ ID NO:333) are particularly interesting regions of the glycoprotein based on their high reactivity in PEPSCAN.<br>
From the above PEPSCAN data can further be deduced that the human monoclonal antibodies called CR57 and CR04-010 bind to different regions of the rabies virus G protein indicating that they recognize non-competing epitopes.<br>
Example 12<br>
Determination of neutralizing potency of anti-rabies G protein<br>
IgGs using an in vitro neutralization assay (modified RFFIT).<br>
The neutralizing potency of each of the produced human monoclonal antibodies was determined in a modified RFFIT as described in Example 1. Sixteen IgGs neutralized rabies strain CVS-11 with a potency higher than 1000 IU/mg, whereas only two IgGs had a potency lower than 2 IU/mg (see Table 15). Eight of<br><br>
the sixteen antibodies outperformed transiently produced CR-57 with regard to potency, suggesting a higher efficiency in post exposure prophylaxis of rabies virus than CR-57, The potency of transiently produced CR-57 was approximately 3800 IU/mg protein (see Tables 1 and 15), whereas stably produced CR-57 displayed a potency of 5400 IU/mg protein (data not shown). Interestingly, the majority of the neutralizing human monoclonal antibodies identified contain a variable heavy 3-30 germline gene (see Table 9).<br>
Based on the affinity of the antibodies for rabies virus (data not shown) and 100% endpoint dilution of the antibodies in a modified RFFIT assay (data not shown), a panel of six unique IgGs, i.e. CR04-010, CR04-040, CR04-098, CR04-103, CR04-104, and CR04-144, were chosen for further development. Within this panel, antibody CR04-098 was particularly interesting as it displayed the highest potency, i.e. approximately 7300 IU/mg protein (see Table 15). A similar potency was also found for stably produced CR04-098 (data not shown).<br>
Example 13<br>
In vitro neutralization of £57 escape viruses by anti-rabies<br>
virus IgGs<br>
To further characterize the novel human monoclonal anti-rabies antibodies the neutralizing activity of the IgGs against E57 escape viruses was tested in a modified RFFIT as described above. The majority of the anti-rabies virus IgGs had good neutralizing activity against all six E57 escape viruses (see Table 16). In contrast, CR04-008, CR04-018 and CR04-126 did not neutralize 6/6, 2/6 and 3/6 E57 escape viruses, respectively. No neutralization means that no 50% endpoint was reached at an antibody dilution of 1:100. CR04-<br><br>
021, CR04-108, CR04-120, CR04-125, and CR04-164 showed a significant decrease in neutralizing activity against a number of escape viruses. This suggests that the epitope of these antibodies has been affected either directly or indirectly in the E57 escape virus glycoprotein. On the basis of the above several anti-rabies virus IgGs may be compatible with CR-57 in an anti-rabies cocktail for post exposure prophylaxis treatment. In particular, the panel of six unique IgGs as identified above, i.e. antibodies CR04-010, CR04-040, CR04-098, CR04-103, CR04-104, and CR04-144, displayed good neutralizing potency towards the E57 escape viruses suggesting that epitope(s) recognized by these antibodies was/were not affected by the amino acid mutations induced by CR-57. Antibody CR04-098 appeared most promising since it had a potency higher than 3000 IU/mg for each of the escape viruses.<br>
Example 14<br>
Epitope recognition of anti-rabies antibodies CR-57 and CR04-<br>
098<br>
To confirm that the human monoclonal antibodies called CR-57 and CR04-0 98 recognize non-overlapping, non-competing epitopes, escape viruses of the human monoclonal antibody called CR04-098 were generated essentially as described for escape viruses of CR57 (see Example 1). In short, the number of foci per well was scored by immunofluorescence and medium of wells containing preferably one focus were chosen for virus amplification. All E98 escape viruses were generated from 1 single focus with the exception of E98-2 (2 foci) and E98-4 (4 foci). A virus was defined as an escape variant if the neutralization index was 
 <br>
plus monoclonal antibody (~ 4 IU/ml) from the number of infectious virus particles/ml produced in BSR or MNA cell cultures infected with virus alone ([log focus forming units/ml virus in absence of monoclonal antibody minus log ffu/ml virus in presence of monoclonal antibody]). An index lower than 2.5 logs was considered as evidence of escape.<br>
To further investigate that CR04-098 binds to a different non-overlapping, non-competing epitope compared to CR-57, CR-57 was tested against E98 escape viruses in a modified RFF1T assay as described above. As shown in Table 17, CR-57 had good neutralizing activity against all five E98 escape viruses. Additionally, antibodies CR04-010 and CR04-144 were tested for neutralizing activity against the E98 escape viruses. Both antibodies did not neutralize the E98 escape viruses (data not shown) suggesting that the epitope recognized by both antibodies is either directly or indirectly affected by the amino acid mutation induced by antibody CR04-098. The antibodies CR04-018 and CR04-126 were tested for neutralizing activity against only one of the E98 escape viruses, i.e. E98-4. CR04-018 was capable of neutralizing the escape virus, while CR04-126 only had a weak neutralizing potency towards the escape virus. This suggests that the epitope recognized by CR04-018 is not affected by the mutation induced by antibody CR04-098. Additionally, the antibodies CR04-010, CR04-038, CR04-040, CR04-073, CR04-103, CR04-104, CR04-108, CR04-120, CR04-125, CR04-164 did not neutralize E98-4 suggesting that they recognize the same epitope as CR04-098 (data not shown).<br>
To identify possible mutations in the rabies glycoprotein of each of the E98 escape viruses, the nucleotide sequence of the glycoprotein open reading frame (ORF) was determined as described before for the E57 and EJB escape viruses. All E98 escape viruses showed the mutation N to D at amino acid<br><br>
position 336 of the rabies glycoprotein (see Figure 8). This region of the glycoprotein has been defined as antigenic site III comprising of amino acids 330-338 (numbering without signal peptide). In contrast, CR-57 recognized an epitope located at amino acids 226-231 (numbering without signal peptide), which overlaps with antigenic site I. In addition to the N336D mutation the E98 escape virus called E98-5 showed the mutation H to Q at amino acid position 354 (codon change CAT to CAG) of the rabies glycoprotein (data not shown).<br>
Moreover, pepscan analysis of binding of CR57 to peptides harbouring a mutated CR57 epitope (as observed in E57 escape viruses) did show that interaction of CR57 was abolished (data not shown). Strikingly, CR04-098 was still capable of binding to the mutated glycoprotein (comprising the N336D mutation) expressed on PER.C6® cells, as measured by flow cytometry (data not shown), even though viruses containing this mutation were no longer neutralized.<br>
Furthermore, epitope mapping studies and affinity ranking studies were performed using surface plasmon resonance analysis using a BIAcore3000™ analytical system. Purified rabies glycoprotein (ERA strain) was immobilized as a ligand on a research grade CM5 4-flow channel (Fc) sensor chip (Biacore AB, Sweden) using amine coupling. Ranking was performed at 25CC with HBS-EP (Biacore AB, Sweden) as running buffer. 50 μl of each antibody was injected at a constant flow rate of 20 μl/min. Then, running buffer was applied for 750 seconds followed by regeneration of the CM5 chip with 5 μl 2M NaOH, 5 μl 45 mM HCl and 5 μl 2 mM NaOH. The resonance signals expressed as resonance units (RU) were plotted as a function of time and the increase and decrease in RU as a measure of association and dissociation, respectively, were determined and used for ranking of the antibodies. The actual KD values.<br><br>
for CR57 and CR04-098 as determined by surface plasmon resonance analysis were 2.4 nM and 4.5 nM, respectively. The epitope mapping studies further confirmed that CR57 and CR04-098 bind to different epitopes on rabies glycoprotein. Injection of CR57 resulted in a response of 58 RU (data not shown). After injection of CR04-098 an additional increase in response level (24 RU) was obtained, suggesting that binding sites for CR04-098 were not occupied (data not shown). Similar results were observed when the reverse order was applied showing that each antibody reached similar RU levels regardless of the order of injection (data not shown). These results further demonstrate that CR57 and CR04-098 can bind simultaneously and recognize different epitopes on the rabies virus glycoprotein.<br>
Overall, the above data further confirm that the antibodies CR-57 and CR04-098 recognize distinct non-overlapping epitopes, i.e. epitopes in antigenic site I and III, respectively. The data are in good agreement with the ELISA/FACS competition data indicating that CR-57 and CR04-098 do not compete for binding to ERA G and the good neutralizing activity of antibody CR04-098 against all E57 escape viruses. On the basis of these results and the fact that in vitro exposure of rabies virus to the combination of CR57 and CR04-098 (selection in the presence of 4 IU/ml of either antibody) yielded no escape viruses (data not shown), it was concluded that the antibodies CR-57 and CR04-098 recognize non-overlapping, non-competing epitopes and can advantageously be used in an anti-rabies virus antibody cocktail for post-exposure prophylaxis treatment.<br>
Example 15<br><br>
Assessment of conservation of the epitope recognized by CR57 and CR04-098<br>
The minimal binding region of CR-57 (amino acids KLCGVL within SEQ ID NO:332, the region of the glycoprotein of rabies virus recognized by CR57 as determined by means of PEPSCAN and alanine scanning technology) was aligned with nucleotide sequences of 22 9 genotype 1 rabies virus isolates to assess the conservation of the epitope (see Table 18). The sample set contained human isolates, bat isolates and isolates from canines or from domestic animals most likely bitten by rabid canines. Frequency analysis of the amino acids at each position within the minimal binding region revealed that the critical residues constituting the epitope were highly conserved. The lysine at position one was conserved in 99.6% of the isolates, while in only 1/229 isolates a conservative K&gt;R mutation was observed. Positions two and three (L and C) were completely conserved. It is believed that the central cysteine residue is structurally involved in the glycoprotein folding and is conserved among all lyssaviruses (see Badrane and Tordo, 2001). The glycine at position four was conserved in 98.7% of the isolates, while in 3/229 isolates mutations towards charged amino acids (G&gt;R in 1/229; G&gt;E in 2/229) were observed. The fifth position was also conserved with the exception of one isolate where a conservative V&gt;I mutation was observed. At the sixth position, which is not a critical residue as determined by an alanine-replacement scan, significant heterogeneity was observed in the street isolates: L in 70.7%, P in 26.7% and S in 2.6% of the strains, respectively. Taken together, approximately 99 percent of the rabies viruses that can be encountered are predicted to be recognized by the CR-57 antibody.<br><br>
123 of these 229 virus isolates were analyzed for the presence of mutations in both the CR-57 and CR04-098 epitope. None of these 123 street viruses did contain mutations in both epitopes. The N&gt;D mutation as observed in the E98 escape viruses was present in only five virus isolates. These viruses were geographically distinct and isolated from animals in Africa (see Figure 9 for phylogenetic tree; the five virus isolates, i.e. AF325483, AF325482, AF325481, AF325480 and AF325485, are indicated in bold). The phylogenetic analysis of glycoprotein sequences revealed that rabies viruses with mutated CR57 epitopes are only distantly related to rabies viruses bearing a mutated CR04-098 epitope. Therefore, the likelihood of encountering a rabies virus resistant to neutralization by a cocktail of CR-57 and CR04-098 is virtually absent.<br><br>
Table 1: Neutralizing potency of CR-57 and CR-JB against wild-type and escape viruses.<br><br><br><br><br><br><br>
Table 4: Human kappa chain variable region primers extended with Sail restriction sites (sense), human kappa chain J-region primers extended with NotI restriction sites (anti-sense) , human lambda chain variable region primers extended with Sail restriction sites (sense) and human lambda chain J-region primers extended with NotI restriction sites (anti-sense) .<br><br><br><br><br><br><br>
Table 5: Distribution of the different light chain products over the 10 fractions.<br><br><br><br><br>
Table 7: Human IgG heavy chain variable region primers extended with Sfil/Ncol restriction sites (sense) and human IgG heavy chain J-region primers extended with XhoI/BstEII restriction sites (anti-sense).<br><br><br><br><br>
Table 8: Binding of single-chain (scFv) phage antibodies to rabies virus G protein (ERA strain) and to FBS as measured by ELISA.<br><br><br>
Table 9: Data of the single-chain Fv’s capable of binding rabies virus G protein.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Table 14: Binding of the human monoclonal antibodies CR57, CRJB and CR04-010 (10 lag/ml) to linear peptides of the extracellular domain of glycoprotein G of rabies virus strain ERA.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
REFERENCES<br>
Ameyama S, Toriumi H, Takahashi T, Shimura Y, Nakahara T,<br>
Honda Y, Mifune K, Uchiyama T and Kawai A (2003) , Monoclonal<br>
antibody #3-9-16 recognizes one of the two isoforms of rabies<br>
virus matrix protein that exposes its N-terminus on the virion<br>
surface. Microbiol. Immunol. 47:639-651.<br>
Badrane H and Tordo N (2001), Host switching in Lyssavirus history from rhe Chirozrtera to the Camivora orders. J. Virol.<br><br><br>
monoclonal human rabies virus-neutralizing antibodies as a substitute for pooled human immune globulin in the prophylactic treatment of rabies virus exposure. J. Immunol. Methods 235:61-90.<br>
Chou TC and Talalay ? (1984), Quantitative analysis of dose-effect relationships: the combined effects of multiple drugs or enzyme inhibitors. Adv. Enzyme Regul. 22:27-55.<br>
Coulon ?, Temaux J?, Flamand A, and Tuffereau C (1SSS), An avirulent mutant of rabies virus is unable to infect motoneurons in vivo and In vitro. J. Virol. 72:273-27 6.<br><br><br>
human monoclonal antibodies to rabies virus. J. Virol. 64:3087-3090.<br>
Hanlon CA, DeMattos CA, DeMattos CC, Niezgoda M, Hooper DC, Koprowski H, Not kins A, and Rupprecht CE (2001), Experimental utility of rabies virus neutralizing human monoclonal antibodies in post-exposure prophylaxis. Vaccine 19:3834-3842.<br>
Huls G, Heijnen IJ, Cuomo E, van der Linden J, Boel Z, van de Winkel J and Logtenberg T (1999) , Antitumor immune effector mechanisms recruited by phage display-derived fully human IgGl and IgAl monoclonal antibodies. Cancer Res. 59:577 8-57 84.<br><br><br>
Ni Y, Tominaga Y, Honda Y, Morimoto K, Sakamoto S, and Kawai A (19S5)r Mapping and characterization of a sequential epitope on the rabies virus glycoprotein which is recognized by a neutralizing monoclonal antibody, RG719. Microbiol- Immunol. 39:693-702.<br>
Prehaud C, Coulon ?, LaFay F, Thiers C, and Flamand A (1988) f Antigenic site II of the rabies virus glycoprotein:structure and role in viral virulence. J. Virol. 62:1-7.<br>
Schumacher CL, Dietzschold 3, Ertl HC, Niu HS, Rupprecht CE, and Koprowski 3 (19B3) , Use of mouse anti-rabies monoclonal antibodies in postexposure treatment of rabies. J. Clin.<br><br><br>
Clin. Microbiol. 16:253-256.<br>
CLAIMS<br>
1. A method of identifying a binding, molecule specifically binding to a rabies virus or a nucleic acid molecule encoding a binding molecule specifically binding to a rabies virus, wherein .the method comprises the steps<br>
a)	contacting a collection of binding molecules on the surface of replicable genetic packages with a rabies virus under conditions conducive to binding, wherein the collection of binding molecules is prepared from RRA isolated from cells obtained from a subject that has been vaccinated against rabies or that has been exposed to rabies virus.<br>
b)	selecting at least once for binding molecules binding to one rabies virus, and<br><br><br>
vaccinated against rabies or that has been exposed to rabies virus.<br>
b)	separating and recovering binding molecules that bind to the rabies virus from binding molecules that do not bind,<br>
c)	isolating at least one recovered binding molecule,<br>
d)	verifying if the binding molecule isolated has neutralizing activity against rabies virus.<br><br><br>
8.	A collection of human binding molecules on the surface of replicable generic packages, characterized in that the collection is prepared from RKA isolated from cells obtained from a subject that has been vaccinated against rabies or has been exposed to rabies virus.<br>
9.	A collection according to claim 8, characterized in that the collection is a single chain Fv library.<br>
10.	A collection according to claim 8 or 9, characterized in that the subject is a human individual which has been vaccinated against rabies.<br>
11.	A binding molecule having rabies virus neutralizing activity, characterized in that the binding molecule comprises at least a CDR3 region comprising the amino acid sequence of SEQ to NO: 14<br><br><br>
variant according to claim 14, the immunoconjugate further comprising at least one tag.<br>
16.	A nucleic acid molecule encoding a binding molecule according to any of the claims 11-13 or a functional variant according to claim 14.<br>
17.	A vector comprising at least one nucleic acid molecule according to claim 16.<br>
13. A host cell comprising at least one vector according to c1aim 17.<br>
19. A host cell according to claim 18, which is a human cell.<br><br><br>
22.	A pharmaceutical composition according to claim 21 comprising at least one additional binding molecule, the additional binding molecule comprising at least a CDR3 region comprising the amino acid sequence of SEQ ID NO:25, wherein the binding molecules are capable of reacting with different, non-competing epitomes of the rabies virus.<br>
23.	A pharmaceutical composition according to claim 22, characterized in that the additional binding molecule comprises a variable heavy chain comprising the amino acid sequence of SEQ ID NO:273.<br>
24.	A pharmaceutical composition comprising at least two rabies virus neutralizing binding molecules that are capable of reacting with different, non-competing epitopes of the rabies virus, characterized in that a first rabies virus neutralicinc binding molecule is capable of reactin-<br><br><br>
26.	A binding molecule according to any of the claims 11-13, a functional variant according to claim 14, an immunoconjugate according to claim 15, or a pharmaceutical composition according to claim 21-25 for use as a medicament.<br>
27.	A binding molecule according to any of the claims 11-13, a functional variant according to claim 14, an immunoconjugate according to claim 15, or a pharmaceutical composition according to claim 21-25 for use in the diagnosis, prophylaxis, treatment, or combination thereof, of a condition resulting from rabies virus.<br>
28. Use of a binding molecule according to any of the claims 11-13, a functional variant according to claim 14, an immuneconjugate according to claim 15, or a pharmaceutical composition according to claim 21-25 in the <br>
preparation of a medicament for the diagnosis, prephylaxis, treatment, or medication thereof, of a <br>
condition resulting from rabies virus.<br><br>
29. A kit comprising at least one binding molecule according to any of the claims 11-13, at least one<br>
functional variant according to claim 14, at least one immunoconjugate according to claim 15, or at least<br>
one pharmaceutional composition according to claim 21-25 or a combination thereof.<br><br>
30. A pharmaceutical composition comprising at least two human monoclonal antibodies, <br>
characterized in that the human monoclonal antibodies have isoelectric points as <br><br>
measured by isoelectric focusing that differ less than 1.5 <br>
pI units.<br>
31.	A pharmaceutical composition according to claim 30, characterized in rhat the isoelectric points differ less than 1.2 pI units.<br>
32.	A pharmaceutical composition according to claim 30 or 31, characterized in that both human monoclonal antibodies are capable of neutralizing an infectious agent.<br>
33.	A pharmaceutical composition according to any of the claims 30-32, characterized in that both human monoclonal antibodies are capable of neutralizing rabies virus.<br><br><br>
37. A binding molecule according to any of the claims 34-36, characterized in that the binding molecule is human.<br>
38. A nucleic acid molecule encoding a binding molecule according to any of the claims 34-37.<br>
39.	A nucleic acid molecule according to claim 38, characterized in that the nucleic acid molecule comprises the nucleotide sequence comprising SEQ ID NO:122.<br>
40.	A method of producing a rabies virus neutralizing binding molecule which comprises at least a heavy chain CDR3 region comprising the amino acid sequence comprising SEQ ID NO:25 in a cell, the method comprising the steps of: providing the cell with a nucleic acid molecule encoding the binding molecule, culzuring the cell under conditions conducive to the expression of the binding molecule and<br>
and the binding molecule has a rabies virus neutralizing activity of at least 2500 IU/mg protein.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICAgRVVST1BFQU4gU0VBUkNIIFJFUE9SVCAgMTQtMTItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006   EUROPEAN SEARCH REPORT  14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICAgRVhBTUlOQVRJT04gUkVQT1JUIFJFUExZIFJFQ0VJVkVELiAxNC0xMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006   EXAMINATION REPORT REPLY RECEIVED. 14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBBTUVOREVEICBQQUdFUyBPRiBTUEVDSUZJQ0FUSU9OICAxNC0xMi0yMDEyLnBkZg==" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  AMENDED  PAGES OF SPECIFICATION  14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBBTUVOREVEIENMQUlNUyAgMTQtMTItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  AMENDED CLAIMS  14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBDT1JSRVNQT05ERU5DRSBPVEhFUlMgMjUtMDYtMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  CORRESPONDENCE OTHERS 25-06-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBGT1JNLTEgMTQtMTItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  FORM-1 14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBGT1JNLTMgMTQtMTItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  FORM-3 14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBPVEhFUiBQQVRFTlQgRE9DVU1FTlQgIDE0LTEyLTIwMTIucGRm" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  OTHER PATENT DOCUMENT  14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2ICBQT1dFUiBPRiBBVFRPUk5FWSAgMTQtMTItMjAxMi5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006  POWER OF ATTORNEY  14-12-2012.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1DSEVOUC0yMDA2IEZPUk0tMTMgMDgtMDUtMjAwOC5wZGY=" target="_blank" style="word-wrap:break-word;">4346-CHENP-2006 FORM-13 08-05-2008.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWNvcnJlc3BvbmRuZWNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-correspondnece-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWRlc2NyaXB0aW9uKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWZvcm0gMS5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWZvcm0gMjYucGRm" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-form 26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWZvcm0gMy5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LWZvcm0gNS5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NDM0Ni1jaGVucC0yMDA2LXBjdC5wZGY=" target="_blank" style="word-wrap:break-word;">4346-chenp-2006-pct.pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="254918-aqeuous-based-froth-of-dispersed-olefin-polymer.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="254920-a-process-for-the-preparation-of-an-ester.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>254919</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>4346/CHENP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>02/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>11-Jan-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>04-Jan-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>27-Nov-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>CRUCELL HOLLAND B.V</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>ARCHIMEDSWEG 4,NL-2333 CN LEIDEN,THE NETHERLANDS</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BAKKER,ALEXANDER,BERTHOLD,HENDRIK</td>
											<td>BURGERMEESTER DE KATLAAN 6, NL-2182 GJ HILLEGEN,THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>2</td>
											<td>MARRISSEN,WILLEM,EGBERT</td>
											<td>WALLOORD 74,NL-3448 BH WOERDEN,THE NETHERLANDS</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KRAMER,ROBERT,ARGEN</td>
											<td>LOMBARDIJE 15, NL-3524 KV UTRECHT</td>
										</tr>
										<tr>
											<td>4</td>
											<td>DE KRUEF, CORNELIS, ADRIAAN</td>
											<td>WESTERLAAN 40,NL-3731 EL DE BILT, THE NETHELANDS</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 16/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP05/52410</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-05-26</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/575,023</td>
									<td>2004-05-27</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/254919-binding-molecules-capable-of-neutralizing-rabies-virus by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:12:10 GMT -->
</html>
