<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/255053-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-psph-gene-expression-level by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:03:13 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 255053:AN IN-VITRO METHOD OF PREDICTING THE RESPONSE OF A NON SMALL CELL LUNG CANCER PATIENT TO ERLOTINIB BY DETERMINING PSPH GENE EXPRESSION LEVEL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN-VITRO METHOD OF PREDICTING THE RESPONSE OF A NON SMALL CELL LUNG CANCER PATIENT TO ERLOTINIB BY DETERMINING PSPH GENE EXPRESSION LEVEL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a biomarker that is predictive for the response to treatment with an EGFR inhibitor in cancer patients</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp; THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
EGFR INHIBITOR TREATMENT MARKER;<br>
F.	HOFFMANN-LA        ROCHE        AG.,        A<br>
CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF SWITZERLAND, WHOSE ADDRESS IS GRENZACHERSTRASSE 124, CH 4070 BASEL, SWITZERLAND.<br>
THE	FOLLOWING	SPECIFICATION<br>
PARTICULARLY DESCRIBES THE INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED.<br><br>
The present invention provides a biomarker that is predictive for the response to treatment with an EGFR inhibitor in cancer patients<br>
A number of human malignancies are associated with aberrant or over-expression of the epidermal growth factor receptor (EGFR). EGF, transforming growth factor a (TGF-a ), and a number of other ligands bind to the EGFR, stimulating autophosphorylation of the intracellular tyrosine kinase domain of the receptor. A variety of intracellular pathways are subsequently activated, and these downstream events result in tumour cell proliferation in vitro. It has been postulated that stimulation of tumour cells via the EGFR may be impor-tant for both tumour growth and tumour survival in vivo.<br>
Early clinical data with Tarceva , an inhibitor of the EGFR tyrosine kinase, indicate that the compound is safe and generally well tolerated at doses that provide the targeted effective concentration (as determined by preclinical data). Clinical phase 1 and II trials in patients with advanced disease have demonstrated that Tarceva has promising clinical activity in a range of epithelial tumours. Indeed, Tarceva has been shown to be capable of inducing durable partial remissions in previously treated patients with head and neck can-cer, and NSCLC (Non small cell lung cancer) of a similar order to established second line chemotherapy, but with the added benefit of a better safety profile than chemo therapy and improved convenience (tablet instead of intravenous [i.v.] administration). A recently completed, randomised, double-blind, placebo-controlled trial (BR.21) has shown that single agent Tarceva significantly prolongs and improves the survival of NSCLC patients for whom standard therapy for advanced disease has failed.<br>
Tarceva (erlotinib) is a small chemical molecule; it is an orally active, potent, selective inhibitor of the EGFR tyrosine kinase (EGFR-TKI).<br>
Lung cancer is the major cause of cancer-related death in North America and Europe. In the United States, the number of deaths secondary to lung cancer exceeds the combined total deaths from the second (colon), third (breast), and fourth (prostate) leading causes of cancer deaths combined. About 75% to 80% of all lung cancers are non-small cell lung cancer (NSCLC), with approximately 40% of patients presenting with locally advanced and/or unresectable disease. This group typically includes those with bulky stage IIIA and IIIB disease, excluding malignant pleural effusions.<br>
The crude incidence of lung cancer in the European Union is 52.5, the death rate 48.7 cases/100000/year. Among men the rates are 79.3 and 78.3, among women 21.6 and 20.5,<br><br><br>
respectively. NSCLC accounts for 80% of all lung cancer cases. About 90% of lung cancer mortality among men, and 80% among women, is attributable to smoking.<br>
In the US, according to the American Cancer Society, during 2004, there were approximately 173,800 new cases of lung cancer (93,100 in men and 80,700 in women) and were accounting for about 13% of all new cancers. Most patients die as a consequence of their disease within two years of diagnosis. For many NSCLC patients, successful treat-ment remains elusive. Advanced tumours often are not amenable to surgery and may also be resistant to tolerable doses of radiotherapy and chemotherapy. In randomized trials the currently most active combination chemotherapies achieved response rates of approxi-mately 30% to 40% and a 1-year survival rate between 35% and 40%. This is really an ad-vance over the 10% 1-year survival rate seen with supportive care alone.<br>
Until recently therapeutic options for patients following relapse were limited to best supportive care or palliation. A recent trial comparing docetaxel (Taxotere ) with best supportive care showed that patients with NSCLC could benefit from second line chemother-apy after cisplatin-based first-line regimens had failed. Patients of all ages and with ECOG performance status of 0, 1, or 2 demonstrated improved survival with docetaxel, as did those who had been refractory to prior platinum-based treatment. Patients who did not benefit from therapy included those with weight loss of 10%, high lactate dehydrogenase levels, multi-organ involvement, or liver involvement. Additionally, the benefit of do-cetaxel monotherapy did not extend beyond the second line setting. Patients receiving do-cetaxel as third-line treatment or beyond showed no prolongation of survival. Single-agent docetaxel became a standard second-line therapy for NSCLC. Recently another randomized phase III trial in second line therapy of NSCLC compared pemetrexed (AlimtaÂ®) with do-cetaxel. Treatment with pemetrexed resulted in a clinically equivalent efficacy but with significantly fewer side effects compared with docetaxel.<br>
It has long been acknowledged that there is a need to develop methods of individualising cancer treatment. With the development of targeted cancer treatments, there is a particular interest in methodologies which could provide a molecular profile of the tumour target, (i.e. those that are predictive for clinical benefit). Proof of principle for gene expres-sion profiling in cancer has already been established with the molecular classification of tumour types which are not apparent on the basis of current morphological and immunohis-tochemical tests.<br><br>
Therefore, it is an aim of the present invention to provide expression biomarkers that are predictive for the response to EGFR inhibitor treatment in cancer patients.<br>
In a first object the present invention provides an in vitro method of predicting the response of a cancer patient to treatment with an EGFR inhibitor comprising the steps:<br>
determining an expression level of a gene PSPH in a tumour sample of a patient and corn-paring the expression level of the gene PSPH to a value representative of the gene PSPH expression level in tumours of a non responding patient population, wherein a higher expression level of the gene PSPH in the tumour sample of the patient is indicative for a patient who will respond to the treatment.<br>
The term "a value representative of an expression level of the gene PSPH in tumours of a non responding patient population" refers to an estimate of the mean expression level of the marker gene in tumours of a population of patients who do not respond to the treat-ment with the EGFR inhibitor.<br>
In a preferred embodiment, the marker gene PSPH (Seq. Id No, 1) shows typically between 1.8 and 3.7 ore more fold higher expression in the tumour sample of the respond-ing patient compared to a value representative of the gene PSPH expression level in tu-mours of a non responding patient population.<br>
In a preferred embodiment, the expression level of the gene PSPH is determined by microarray technology or other technologies that assess RNA expression levels like quantitative RT-PCR or by any method looking at the expression level of the respective protein, eg immunohistochemistry (IHC). The construction and use of gene chips are well known in the art. see, U. S. Pat Nos. 5,202,231; 5,445,934; 5,525,464; 5,695,940; 5,744,305; 5,795, 716 and 1 5,800,992. See also, Johnston, M. Curr. Biol. 8:R171-174 (1998); Iyer VR et alâ Science 283:83-87 (1999). Of course, the gene expression level can be determined by other methods that are known to a person skilled in the art such as e.g. northern blots, RT-PCR, real time quantitative PCR, primer extension, RNase protection, RNA expression profiling.<br>
The marker gene of the present invention can be combined with other biomarkers to biomarker sets. Biomarker sets can be built from any combination of predictive biomarkers to make predictions about the effect of EGFR inhibitor treatment in cancer patients. The biomarkers and biomarkers sets described herein can be used, for example, to predict how patients with cancer will respond to therapeutic intervention with an EGFR inhibitor.<br>
The term "gene" as used herein comprises variants of the gene. The term "variant" relates to nucleic acid sequences which are substantially similar to the nucleic acid se-<br><br>
quences given by the GenBank accession number. The term "substantially similar" is well understood by a person skilled in the art. In particular, a gene variant may be an allele which shows nucleotide exchanges compared to the nucleic acid sequence of the most prevalent allele in the human population. Preferably, such a substantially similar nucleic acid sequence has a sequence similarity to the most prevalent allele of at least 80%, pref-erably at least 85%, more preferably at least 90%, most preferably at least 95%. The term "variants" is also meant to relate to splice variants.<br>
The EGFR inhibitor can be selected from the group consisting of gefitinib, erlotinib, PKI-166, EKB-569, GW2016, CI-1033 and an anti-erbB antibody such as trastuzumab and cetuximab.<br>
In another embodiment, the EGFR inhibitor is erlotinib. In yet another embodiment, the cancer is NSCLC.<br>
Techniques for the detection of gene expression of the genes described by this invention include, but are not limited to northern blots, RT-PCR, real time quantitative PCR, primer extension, RNase protection, RNA expression profiling and related techniques. These techniques are well known to those of skill in the art see e.g. Sambrook J et al., Mo-lecular Cloning: A Laboratory Manual, Third Edition (Cold Spring Harbor Press, Cold Spring Harbor, 2000).<br>
Techniques for the detection of protein expression of the respective genes described by this invention include, but are not limited to immunohistochemistry (IHC).<br>
In accordance with the invention, cells from a patient tissue sample, e.g., a tumour or cancer biopsy, can be assayed to determine the expression pattern of one or more bio-markers. Success or failure of a cancer treatment can be determined based on the biomarker expression pattern of the cells from the test tissue (test cells), e.g., tumour or cancer biopsy, as being relatively similar or different from the expression pattern of a control set of the one or more biomarkers. In the context of this invention, it was found that the PSPH gene is up-regulated i.e. shows a higher expression level, in tumours of patients who respond to the EGFR inhibitor treatment compared to tumours of patients who do not respond to the EGFR inhibitor treatment. Thus, if the test cells show a biomarker expression profile which corresponds to that of a patient who responded to cancer treatment, it is highly likely or predicted that the individual's cancer or tumour will respond favourably to treatment with the EGFR inhibitor. By contrast, if the test cells show a biomarker expression pattern corresponding to that of a patient who did not respond to cancer treatment, it is highly likely or<br><br><br>
predicted that the individual's cancer or tumour will not respond to treatment with the EGFR inhibitor.<br>
The biomarker of the present invention is a first step towards an individualized ther-apy for patients with cancer, in particular patients with refractory NSCLC. This individual-ized therapy will allow treating physicians to select the most appropriate agent out of the existing drugs for cancer therapy, in particular NSCLC. The benefit of individualized ther-apy for each future patient are: response rates / number of benefiting patients will increase and the risk of adverse side effects due to ineffective treatment will be reduced.<br>
In a further object the present invention provides a therapeutic method of treating a cancer patient identified by the in vitro method of the present invention. Said therapeutic rnethod comprises administering an EGFR inhibitor to the patient who has been selected for treatment based on the predictive expression pattern of the PSPH gene. A preferred EGFR inhibitor is erlotinib and a preferred cancer to be treated is NSCLC.<br>
Short description of the figures<br>
Figure 1 shows the study design; Figure 2 shows the scheme of sample processing.<br>
Figure 3a shows expression levels versus clinical outcome for GenechipÂ® profiling; Figure 3b shows expression levels versus clinical outcome for qRT-PCR and Figure 3c shows the'correlation between GenechipÂ® and qRT-PCR measurements for PSPH.<br>
Experimental part<br>
Rationale for the Study and Study Design<br>
Recently mutations within the EGFR gene in the tumour tissue of a subset of NSCLC patients and the association of these mutations with sensitivity to erlotinib and gefitinib were described (Pao W, et al. 2004; Lynch et al. 2004; Paez et al. 2004). For the patients combined from two studies, mutated EGFR was observed in 13 of 14 patients who re-sponded to gefitinib and in none of the 11 gefitinib-treated patients who did not respond. The reported prevalence of these mutations was 8% (2 of 25) in unselected NSCLC pa-tients. These mutations were found more frequently in adenocarcinomas (21%), in tumours from females (20%), and in tumours from Japanese patients (26%). These mutations result in increased in vitro activity of EGFR and increased sensitivity to gefitinib. The relation-ship of the<br><br><br><br>
mutations to prolonged stable disease or survival duration has not been prospec-tively evaluated.<br>
Based on exploratory analyses from the BR.21 study, it appeared unlikely that the observed survival benefit is only due to the EGFR mutations, since a significant survival benefit is maintained even when patients with objective response are excluded from analy-ses (data on file). Other molecular mechanisms must also contribute to the effect.<br>
Based on the assumption that there are changes in gene expression levels that are predictive of response / benefit to Tarceva treatment, microarray analysis was used to detect these changes<br>
This required a clearly defined study population treated with Tarceva monotherapy after failure of 1st line therapy. Based on the experience from the BR.21 study, benefiting population was defined as either having objective response, or disease stabilization for 12 weeks. Clinical and microarray datasets were analyzed according to a pre defined statistical plan.<br>
The application of this technique requires fresh frozen tissue (FFT). Therefore a mandatory biopsy had to be performed before start of treatment. The collected material was frozen in liquid nitrogen (N2).<br>
A second tumour sample was collected at the same time and stored in paraffin (formalin fixed paraffin embedded, FFPE). This sample was analysed for alterations in the EGFR signalling pathway.<br>
The ability to perform tumour biopsies via bronchoscopy was a prerequisite for this study. Bronchoscopy is a standard procedure to confirm the diagnosis of lung cancer. Although generally safe, there is a remaining risk of complications, e.g. bleeding.<br>
This study was a first step towards an individualized therapy for patients with refractory NSCLC. This individualized therapy will allow treating physicians to select the most appropriate agent out of the existing drugs for this indication.<br>
Once individualized therapy will be available, the benefit for each future patient will outweigh the risk patients have to take in the present study:<br>
response rates /number of benefiting patients will increase,<br>
the risk of adverse side effects due to ineffective treatment will be reduced.<br>
Rationale for Dosage Selection<br>
Tarceva was given orally once per day at a dose of 150 mg until disease progres-sion, intolerable toxicities or death. The selection of this dose was based on pharmacoki-netic<br><br><br><br>
parameters, as well as the safety and tolerability profile of this dose observed in Phase I, II and III trials in heavily pre-treated patients with advanced cancer. Drug levels seen in the plasma of patients with cancer receiving the 150 mg/day dose were consistently above the average plasma concentration of 500 ng / ml targeted for clinical efficacy. BR.21 showed a survival benefit with this dose.<br>
Objectives of the Study<br>
The primary objective was the identification of differentially expressed genes that are predictive for benefit (CR, PR or SD ? 12 weeks) of Tarceva treatment. Identification of differentially expressed genes predictive for "response" (CR, PR) to Tarceva treatment was an important additional objective.<br>
The secondary objectives were to assess alterations in the EGFR signalling pathways with respect to benefit from treatment.<br>
Study Design<br>
Overview of Study Design and Dosing Regimen<br>
This was an open-label, predictive marker identification Phase II study. The study was conducted in approximately 26 sites in about 12 countries. 264 patients with advanced NSCLC following failure of at least one prior chemotherapy regimen were enrolled over a 12 month period. Continuous oral Tarceva was given at a dose of 150 mg/day. Dose re-ductions were permitted based on tolerability to drug therapy. Clinical and laboratory pa-rameters were assessed to evaluate disease control and toxicity. Treatment continued until disease progression, unacceptable toxicity or death. The study design is depicted in figure 1.<br>
Tumour tissue and blood samples were obtained for molecular analyses to evaluate the effects of Tarceva and to identify subgroups of patients benefiting from therapy.<br>
Predictive Marker Assessments<br>
Biopsies of the tumour were taken within 2 weeks before start of treatment. Two different samples were collected:<br>
The first sample was always frozen immediately in liquid N2 The second sample was fixed in formalin and embedded in paraffin Snap frozen tissue had the highest priority in this study. Figure,2 shows a scheme of the sample processing.<br><br>
Microarray Analysis<br>
The snap frozen samples were used for laser capture microdissection (LCM) of ru-mour cells to extract tumour RNA and RNA from tumour surrounding tissue. The RNA was analysed on Affymetrix microarray chips (HG-U133A) to establish the patients' tu-mour gene expression profile. Quality Control of Affymetrix chips was used to select those samples of adequate quaJity for statistical comparison.<br>
Single Biomarker Analyses on Formalin Fixed Paraffin Embedded Tissue<br>
The second tumour biopsy, the FFPE sample, was used to perform DNA mutation, IHC and ISH analyses as described below. Similar analyses were performed on tissue col-lected at initial diagnosis.<br>
The DNA mutation status of the genes encoding EGFR and other molecules involved in the EGFR signalling pathway were analysed by DNA sequencing. Gene amplification of EGFR and related genes were be studied by FISH.<br>
Protein expression analyses included immunohistochemical [IHC] analyses of EGFR and other proteins within the EGFR signalling pathway.<br>
Response Assessments<br>
The RECIST (Uni-dimensional Tumour Measurement) criteria were used to evaluate response. These criteria can be found under the following link: http://www.eortc.be/recist/<br>
Note that:To be assigned a status of CR or PR, changes in tumour measurements must be confirmed by repeated assessments at least 4 weeks apart at any time during the treatment period.<br>
In the case of SD, follow-up measurements must have met the SD criteria at least once after study entry at a minimum interval of 6 weeks.<br>
In the case of maintained SD, follow-up measurements must have met the SD criteria at least once after study entry with maintenance duration of at least 12 weeks.<br>
Survival Assessment<br>
A regular status check every 3 months was performed either by a patient's visit to the clinic or by telephone. All deaths were recorded. At the end of the study a definitive confirmation of survival was required for each patient.<br><br><br>
Response to erlotinib treatment<br>
A total of 264 patients from 12 countries and 26 centres were enrolled in the study. 26% had Stage IIIB and 24% Stage IV NSCLC. 13.6% (n=36) of patients achieved an objective response while 31.4% (n=83) had clinical benefit (defined as having either an objective response or stable disease for 12 weeks or more). Median overall survival was 7.6 (CI 7-9) months and median progression-free survival was 11.3 (CI 8-12) weeks. Full de-tails about the clinical data are shown in Table 1.<br>
Fresh frozen bronchoscopic biopsies were collected from all subjects, but either not all samples had sufficient tumour content prior to microdissection (LCM) or did not have sufficient RNA yield after LCM to proceed to microarray analysis, so that tumour material was only available for 125 patients; 122 of these had evaluable RNA. Another set of 20 samples did not pass our quality control assessment of the microarray data. Of the 102 microarray data sets that were suitable for statistical analysis, the clinical characteristics are shown in Table 1. While 36 patients in the overall study achieved an objective response, only 6 of these had microarray data; similarly for those achieving clinical benefit the num-ber of subjects with microarray data was only 21 as compared to 83 in the full data set. 6 were judged to be partial responders (PR), 31 had SD and 49 had PD; of the 6 patients with a PR, 5 had adenocarcinoma and one had squamous cell carcinoma. There were no patients achieving a CR in the data set.<br>
Methods<br>
RNA sample preparation and quality control of RNA samples<br>
All biopsy sample processing was handled by a pathology reference laboratory; fresh frozen tissue samples were shipped from investigator sites to the Clinical Sample Opera-tions facility in Roche Basel and from there to the pathology laboratory for further process-ing. Laser capture microdissection was used to select tumour cells from surrounding tis-sue. After LCM, RNA was purified from the enriched tumour material. The pathology laboratory then carried out a number of steps to make an estimate of the concentration and quality of the RNA..<br>
RNases are RNA degrading enzymes and are found everywhere and so all procedures where RNA will be used must be strictly controlled to minimize RNA degradation. Most mRNA species themselves have rather short half-lives and so are considered quite unstable.<br><br><br>
Therefore it is important to perforin RNA integrity checks and quantification before any assay.<br>
RNA concentration and quality profile can be assessed using an instrument from Agilent (Agilent Technologies, Inc., Palo Alto, CA) called a 2100 BioanalyzerÂ®. The instrument software generates an RNA Integrity Number (RIN), a quantitation estimate (Schroeder, A., et al., The RIN: an RNA integrity number for assigning integrity values to RNA measurements. BMC Mol Biol, 2006. 7: p. 3), and calculates ribosomal ratios of the total RNA sample. The RfN is determined from the entire electrophoretic trace of the RNA sample, and so includes the presence or absence of degradation products.<br>
The RNA quality was analysed by a 2100 BioanalyzerÂ®. Only samples with at least one rRNA peak above the added poly-1 noise and sufficient RNA were selected for further analysis on the Affymetrix platform. The purified RNA was forwarded to the Roche Cen-tre for Medical Genomics (RCMG; Basel, Switzerland) for analysis by microarray. 122 RNA samples were received from the pathology laboratory for further processing.<br>
Target Labeling of tissue RNA samples<br>
Target labeling was carried out according to the Two-Cycle Target Labeling Amplification Protocol from Affymetrix (Affymetrix, Santa Clara, California), as per the manufacturer's instructions.<br>
The method is based on the standard Eberwine linear amplification procedure but uses two cycles of this procedure to generate sufficient labeled cRNA for hybridization to a microarray.<br>
Total RNA input used in the labeling reaction was 10ng for those samples where more than lOng RNA was available; if less than this amount was available or if there was no quantity data available (due to very low RNA concentration), half of the total sample was used in the reaction. Yields from the labeling reactions ranged from 20-180fig cRNA. A normalization step was introduced at the level of hybridization where 15ug cRNA was used for every sample.<br>
Human Reference RNA (Stratagene, Carlsbad, CA, USA) was used as a control sample in the workflow with each batch of samples. lOng of this RNA was used as input alongside the test samples to verify that the labeling and hybridization reagents were working as expected.<br><br>
Microarrav hybridizations<br>
Affymetrix HG-U133A microarrays contain over 22,000 probe sets targeting approximately 18,400 transcripts and variants which represent about 14,500 well-characterized genes.<br>
Hybridization for all samples was carried out according to Affymetrix instructions (Affymetrix Inc., Expression Analysis Technical Manual, 2004). Briefly, for each sample, 15ug of biotin-labeled cRNA were fragmented in the presence of divalent cations and heat and hybridized overnight to Affymetrix HG-U133A full genome oligonucleotide arrays. The following day arrays were stained with streptavidin-phycoerythrin (Molecular Probes; Eugene, OR) according to the manufacturer's instructions. Arrays were then scanned using a GeneChip Scanner 3000 (Affymetrix), and signal intensities were automatically calcu-lated by GeneChip Operating Software (GCOS) Version 1.4 (Affymetrix).<br>
Statistical Analysis<br>
Analysis of the Affymetrixâ¢ data consisted of four main steps.<br>
Step 1 was quality control. The goal was to identify and exclude from analysis array data with a sub-standard quality profile.<br>
Step 2 was pre-processing and normalization. The goal was to create a normalized and scaled "analysis data set", amenable to inter-chip comparison. It comprised background noise estimation and subtraction, probe summarization and scaling.<br>
Step 3 was exploration and description. The goal was to identify potential bias and sources of variability. It consisted of applying multivariate and univariate descriptive analysis techniques to identify influential covariates.<br>
Step 4 was modeling and testing. The goal was to identify a list of candidate markers based on statistical evaluation of the difference in mean expression level between "Responded" (patients with "Partial Response" or "Complete Response" as best response) and "Non Responders" (patients "Progressive Disease" as best response). It consisted of fitting an adequate statistical model to each probe-set and deriving a measure of statistical signifi-cance.<br>
All analyses were performed using the R software package.<br>
Step 1 : Quality Control<br>
The assessment of data quality was based on checking several parameters. These included standard Affymetrix GeneChipâ¢ quality parameters, in particular: Scaling Factor, Percentage of Present Call, and Average Background. This step also included visual in-<br><br><br>
spection of virtual chip images for detecting localized hybridization problems, and com-parison of each chip to a virtual median chip for detecting any unusual departure from median behaviour. Inter-chip correlation analysis was also performed to detect outlier sam-ples. In addition, ancillary measures of RNA quality obtained from analysis of RNA sam-ples with the Agilent Bioanalyzerâ¢ 2100 were taken into consideration.<br>
Based on these parameters, data from 20 arrays were excluded from analysis. Thus data from a total of 102 arrays representing 102 patients was included in the analysis. The clinical description of these 102 patients set is reported in table 1.<br>
Table 1: Description of clinical characteristics of patients included in the analysis<br><br>
Variable	Value	n=102<br>
Best Response	N/A PD SD PR	n   (%) 16 (15.7%) 49 (48.0%) 31 (30.4%)<br>
6 (5.9%)<br>
Clinical Benefit	NO<br>
YES	81 (79.4%) 21 (20.6%)<br>
SEX	FEMALE MALE	25 (24.5%) 77 (74.5%)<br>
ETHNICITY	CAUCASIAN ORIENTAL	65 (63.7%) 37 (36.3%)<br>
Histology	ADENOCARCINOMA<br>
SQUAMOUS<br>
OTHERS	35 (34.3%) 53 (52.0%) 14 (13.7%)<br>
Ever-Smoking	NO YES	20 (19.6%) 82 (80.4%)<br>
Step 2 : Data pre-processing and normalization<br>
The rma algorithm (Irizarry, R.A., et al., Summaries of Affymetrix GeneChip probe level data. Nucl. Acids Res., 2003. 31(4): p. el5) was used for pre-processing and normalization. The mas5 algorithm (AFFYMETRIX, GeneChipÂ® Expression: Data Analysis Fundamentals. 2004, AFFYMETRIX) was used to make detection calls for the individual probe-sets. Probe-sets called "absent" or "marginal" in all samples were removed from further analysis; 5930 probe-sets were removed from analysis based on this criterion. The analysis<br><br>
data set therefore consisted of a matrix with 16353 (out of 22283) probe-sets measured in 102 patients.<br>
Step 3 : Data description and exploration<br>
Descriptive exploratory analysis was performed to identify potential bias and major sources of variability. A set of covariates with a potential impact on gene expression pro-files was screened. It comprised both technical and clinical variables. Technical covariates included: date of RNA processing (later referred to as batch), RIN (as a measure of RNA quality/integrity), Operator and Center of sample collection. Clinical covariates included: histology type, smoking status, tumour grade, performance score (Oken, M.M., et al., Toxicity and response criteria of the Eastern Cooperative Oncology Group. Am J Clin Oncol, 1982. 5(6): p. 649-55), demographic data, responder status and clinical benefit status.<br>
The analysis tools included univariate ANOVA and principal component analysis. For each of these covariates, univariate ANOVA was applied independently to each probe-set.<br>
A significant effect of the batch variable was identified. In practice, the batch vari-able captured differences between dates of sample processing and Affymetrix chip lot. Af-ter checking that the batch variable was nearly independent fromthe variables of interest, the batch effect was corrected using the method described in Johnson, W.E., C. Li, and A. Rabinovic, Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostat, 2007. 8(1): p. 118-127.<br>
The normalized data set after batch effect correction served as the analysis data set in subsequent analyses.<br>
Histology and RIN were two additional important variables highlighted by the descriptive analysis.<br>
Step 4 : Data modeling and testing.<br>
A linear model was fitted independently to each probe-set. Variables included in the model are reported in table 2. The model parameters were estimated by the maximum likelihood technique. The parameter corresponding to the "Response" variable (XI) was used to assess the difference in expression level between the group "responding" and "non responding" patients.<br>
Table 2: Description of the variables included in the linear model. The linear model defined by these variables, including a normally distributed error term, was fitted to each probe-set.<br>
Variable	Type	Values<br><br>
gene expression	Dependent (Yip)	Normalized log2 intensity of probe-set i in patient p.<br>
Intercept	Overall mean (u)	<br>
Response	Predictor of interest (XI)	YES / NO<br>
Histology	Adjustment Covariate (X2)	ADENOCARCINOMA / SQUAMOUS / OTHERS<br>
RACE	Adj. Cov. (X3)	ORIENTAL / CAUCASIAN<br>
SEX	Adj Cov. (X4)	FEMALE / MALE<br>
RIN	Adj Cov. (X5)	[2,...,7.9]<br>
Patients with a stable disease status were excluded (n=31). The rationale was that by focusing on patients with the more marked response to therapy, the non-responder group would become more homogeneous.<br>
For each probe-set i, the aim of the statistical test was to reject the hypothesis that the mean expression levels in patients with response to treatment and patients without response to treatment are equal, taking into account the other adjustment covariates listed in table 2. Formally, the null hypothesis of equality was tested against a two sided alternative. Under the null hypothesis, the distribution of the Wald-statistic for this test follows a Chi Square distribution with 64 degrees of freedom. The corresponding p-values are reported in table 3.<br>
The choice of linear model was motivated by two reasons. Firstly, linear modeling is a versatile, well-characterized and robust approach that allows for adjustment of confound-ing variables when estimating the effect of the variable of interest. Secondly, given the sample size of 71, and the normalization and scaling of the data set, the normal distribution assumption was reasonable and justified.<br>
The issue of multiple testing was dealt with by using a False Discovery Rate (FDR) criterion for identifying the list of differential expressed genes (Benjamini et af., Journal of the Royal Statistical Society Series B-Methodological, 1995. 57(1): p. 289-300). Probe-sets with an FDR below the 0.3 threshold are declared significant. The 0.3 cut-off was cho-sen as a reasonable compromise between a rigorous correction for multiple testing with a stringent control of the risk of false positive and the risk of missing truly differential mark-ers. The identified marker gene is reported in Table 3,<br><br>
Table 3: Markers based on comparing "Responders" to "Progressors". Responders were defined as patients with Best Response equal to "Partial Response" (PR). "Progres-sors" were defined as patients having best response equal to "Progressive Disease"(PD) or no assessment available.<br>
Patients with no tumour assessment were included in the "Progressors" groups be-cause in the majority of cases, assessment was missing because of early withdrawal due to disease progression or death.<br>
Column 1 is the Affymetrix identifier of the probe-set. Column 2 is the GenBank accession number of the corresponding gene sequence. Corumn 3 is the corresponding offi-ciai gene name. Column 4 is the corresponding adjusted mean fold change in expression level between "responders" and "progressors", as estimated with the linear model. Column 5 is the p-value for the test of difference in expression level between "responders" and "progressors" as derived from the linear model. Column 6 is the 95% confidence interval for the adjusted mean fold change in expression level.<br><br>
Affymetrix Probe Set ID<br><br>
GenBank<br><br>
Gene<br><br>
Adjusted Mean Fold Change<br><br>
P-value<br><br>
CI 95%<br><br><br><br>
205194 at<br><br>
NM 004577<br><br>
PSPH<br><br>
2.6<br><br>
1.40E-05<br><br>
1.8,3.7<br><br>
For each probe-set, the assumption of homogeneity of variance was evaluated using Fligner-Killeen tests based on the model residuals. The analysis consisted of three steps:<br>
Test all categorical variables for equality of residual variance between their levels<br>
Note the variable V with the least p-value<br>
If the least p-value is less than 0.001, re-fit the model allowing the different level of variables V to have a different variance.<br>
Further statistical analysis<br>
For the selected candidate marker PSPH, the following additional analyses were performed in a validated environment by an independent statisticians :<br>
â¢	Univariate Cox Regression for PFS (Progression free survival) from Primary Affymetrix Analysis,<br>
â¢	Univariate Logistic Regression for Response from Primary Affymetrix Analysis, and<br><br>
As reported above, Affymetrix GenechipÂ® gene expression profiles were determined for 1 02 patients included in this study. Among these patients, qRT-PCR results were obtained for 75 (table 3). The demographics and clinical characteristics of the patients with qRT-PCR results were similar to those of the entire population (n=264) and of the patients with GenechipÂ® gene expression profiles available.<br>
Table 3: Baseline characteristics: patients with qRT-PCR analyses (n=75)<br><br>
Characteristic	<br>
Age {median, range)	62 (39-85)<br>
Gender; n (%)	<br>
Male	19 (25)<br>
Female	56 (75)<br>
ECOG performance status; n (%)	<br>
0	7(9)<br>
1	45 (60)<br>
2	23(31)<br>
Histology; n (%)	<br>
Adenocarcinoma	27 (36)<br>
Squamous-cell carcinoma	34 (45)<br>
Large-cell carcinoma	2(3)<br>
Other	12(16)<br>
Disease stage; n (%)	<br>
1MB	22(29)<br>
IV	53(71)<br>
Number of prior chemotherapy regimens; n (%)	<br>
0	19(25)<br>
1	36 (48)<br>
&gt;2	20 (27)<br>
Ethnicity; n (%)	<br>
Caucasian	51 (68)<br><br>
Asian	24 (32)<br>
Smoking history; n (%)	<br>
Never	12 (16)<br>
Current	24 (32)<br>
Former	39 (52)<br>
Of the 75 patients with qRT-PCR results, 4 (5%) had partial response (PR), 23 (31%) had SD, 39 (52%) had PD, and 9 (12%) were not evaluable. These results were very similar to those observed in the entire study population (n=264).<br>
Correlation between Affymetrix GenechipÂ® data, qRT-PCR data and clinical outcomes<br>
Figure 3 shows relative mRNA levels for PSPH in individual patients, as assessed by Affymetrix GenechipÂ® profiling and qRT-PCR. Fig. 3a shows expression levels versus clinical outcome for GenechipÂ® profiling and Fig. 3b shows expression levels versus clini-cal outcome for qRT-PCR.<br>
Figure 3c shows the correlation between GenechipÂ® and qRT-PCR measurements for PSPH. The association between PSPH mRNA levels and clinical outcome with erlotinib observed in the GenechipÂ® gene expression data was preserved in the qRT-PCR data.<br>
Identification of genes associated with response to erlotinib<br>
Responders were defined as patients whose best response was partial response, while "progressors" were defined as patients having either progressive disease or for whom no assessment was made (in most cases as a result of early withdrawal due to disease progression or death). Thus in this model 6 "responders" were compared to 65 "progressors".<br>
A linear model was fitted independently to each of the 16353 remaining probe-sets used in the analysis after removal of those probe-sets that were not present in any sample from the total 22283 on the HG-U133A microarray. A p-value was calculated for the dif-ference in expression between response and non-response for each probe-set. A false dis-covery rate (FDR) of 0.3 was applied to correct for multiple testing. The marker identified from this analysis is shown in Table 3.<br>
Discussion<br><br>
Targeting the Epidermal Growth Factor Receptor (EGFR) as a means of cancer therapy was proposed based on its ubiquitous aberrant expression in several epithelial cancers. EGFR is implicated in the pathogenesis and progression of many tumours including 40-80% of NSCLC tumours, as a result of activating mutations in the tyrosine kinase domain and / or its amplification. Upon activation, the receptor undergoes dimerization, resulting in phosphorylation of downstream targets with roles in cellular proliferation, metastasis, inhibition of apoptosis and neoangiogenesis.<br>
Two major classes of EGFR inhibitors have been developed, monoclonal antibodies targeting the extracellular domain of the receptor, and small molecule tyrosine kinase inhibitors targeting the catalytic domain of the receptor. The latter include erlotinib which competes with ATP for the intracellular binding site.<br>
It has emerged in recent years that several factors play a role in sensitivity to erlotinib including female gender, non-smoker status, Asian origin and adenocarcinoma histology; given that enhanced response rates are evident in such clinical subsets of patients, exten-sive efforts are ongoing to elucidate predictive molecular markers for patient stratification. Mutations in the EGFR, amplification of the EGFR gene locus and overexpression of EGFR on the protein level, have all been associated with response to varying degrees, though these are not the only molecular determinants of response.<br>
By analyzing tissue samples with high-density oligonucleotide microarray technol-ogy, and applying statistical modeling to the data, we have been able to identify a gene whose expression level is predictive of response to erlotinib (comparison of PR versus PD) (Table 3). This gene is PSPH (2.6 fold upregulated; p = 0.000014).<br>
Phosphoserine phosphatase (PSPH) is an intermediary enzyme in gluconeogenesis and amino acid biosynthesis, and is specifically responsible for the last step in L-serine formation. PSPH enzyme activity has been found to be higher in cancerous tissue than in normal lung tissue as well as in highly-differentiated lung tumours as compared to undif-ferentiated lung tumours or mesothelioma. On the mRNA transcript level PSPH has been evaluated as a marker for micrometastases in advanced gastric cancer and showed in-creased specificity in combination with a standard marker, carcinoembryonic antigen (CEA).<br><br>
We Claim :<br>
1.	An in vitro method of predicting the response of a cancer patient to treatment with an EGFR inhibitor comprising: determining an expression level of a gene PSPH in a tumour sample of a patient and comparing the expression level of the gene PSPH to a value representative of a gene PSPH expression Jevd in tumours of a non responding patient population, wherein a higher expression level of the PSPH gene in the tumour sample of the patient is indicative for a patient who will respond to the treatment.<br>
2.	The method of claim 1, wherein the expression level is determined by microarray technology,<br>
3.	The method of claim 1 or 2, wherein the marker gene shows between 1.8 and 3.7 or more fold higher expression in the tumour sample of the responding patient compared to the value representative of the gene PSPH expression level in tumours of a non respond-ing patient population.<br>
4.	The method of claims 1 to 3, wherein the EGFR inhibitor is erlotinib.<br>
5.	The method of claims 1 to 4, wherein the cancer is NSCLC.<br>
6.	Use of a PSPH gene for predicting the response of a cancer patient to EGFR inhibitor treatment.<br>
7.	The use of claim 6, wherein the cancer is NSCLC.<br>
8.	The use of claim 6 or 7, wherein the EGFR inhibitor is erlotinib.<br>
9.	A method of treating a cancer patient identified by a method of claims 1 to 5 comprising administering an EGFR inhibitor to the patient.<br><br>
10.	The method of claim, 9, wherein the EGFR inhibitor is erlotinib.<br>
11.	The method of claim 9 or 10, wherein the cancer is NSCLC.<br><br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQU5ORVhVUkUgQSgyMi01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-ANNEXURE A(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSgyMi01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(AMENDED)-(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSgyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(AMENDED)-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(AMENDED)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMjItNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(MARKED COPY)-(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMjgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(MARKED COPY)-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oNC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CLAIMS(MARKED COPY)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ09NTVVOSUNBVElPTlMgRlJPTSBFUE8oMTktMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-COMMUNICATIONS FROM EPO(19-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ09SUkVTUE9OREVOQ0UoMTktMS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CORRESPONDENCE(19-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtQ09SUkVTUE9OREVOQ0UoOC0zLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-CORRESPONDENCE(8-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZHJhd2luZy5wZGY=" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRVAgRE9DVU1FTlQoMjItNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-EP DOCUMENT(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAxKDIyLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 1(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAxKDQtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 1(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAyKFRJVExFIFBBR0UpLSgyMi01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 2(TITLE PAGE)-(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAyKFRJVExFIFBBR0UpLSg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 2(TITLE PAGE)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAyKHRpdGxlIHBhZ2UpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 2(title page).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAyLmRvYw==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAzKDE5LTEtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 3(19-1-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAzKDIyLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 3(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtRk9STSAzKDgtMy0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-FORM 3(8-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSBwY3QtaWItMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form pct-ib-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSBwY3QtaXNhLTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form pct-isa-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtZm9ybSBwY3Qtcm8tMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-form pct-ro-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtR0VORVJBTCBQT1dFUiBPRiBBVFRPUk5FWSgyMi01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-GENERAL POWER OF ATTORNEY(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtb3RoZXIgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-other document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMjItNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-PETITION UNDER RULE 137(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtcG93ZXIgb2YgYXR0b3JuZXkucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-power of attorney.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDIyLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-REPLY TO EXAMINATION REPORT(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtUkVQTFkgVE8gSEVBUklORygyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-REPLY TO HEARING(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtUkVQTFkgVE8gSEVBUklORyg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-REPLY TO HEARING(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtc2VxdWVuY2UgbGlzdGluZy5wZGY=" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LU1VTU5QLTIwMTAtVVMgRE9DVU1FTlQoMjItNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">285-MUMNP-2010-US DOCUMENT(22-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtd28gaW50ZXJuYXRpb25hbCBwdWJsaWNhdGlvbiByZXBvcnQgYTIucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-wo international publication report a2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mjg1LW11bW5wLTIwMTAtd28gaW50ZXJuYXRpb25hbCBwdWJsaWNhdGlvbiByZXBvcnQgYTMucGRm" target="_blank" style="word-wrap:break-word;">285-mumnp-2010-wo international publication report a3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="255052-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-ptprf-gene-expression-level.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="255054-an-in-vitro-method-of-predicting-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-mrna-expression-level-of-a-rapgef5-gene.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>255053</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>285/MUMNP/2010</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Jan-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Jan-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>11-Feb-2010</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F.HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>GRENZACHERSTRASSE 124, CH-4070 BASEL, SWITZERLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LUTZ, VERENA</td>
											<td>LINDENSCHMITSTRASSE 25, D-81371 MUENCHEN, GERMANY.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>DELMAR,PAUL</td>
											<td>HAMMERSTRASSE 89, CH-4057 BASEL, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>KLUGHAMMER,BARBARA</td>
											<td>IM ADLERGARTEN 11, D-79618 RHEINFELDEN, GERMANY.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MCLOUGHLIN,PATRICIA</td>
											<td>CLARAGRABEN 115, CH-4057 BASEL, SWITZERLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/006520</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-08-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07114312.7</td>
									<td>2007-08-14</td>
								    <td>EPO</td>
								</tr>
								<tr>
									<td>2</td>
									<td>08156511.1</td>
									<td>2008-05-20</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/255053-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-psph-gene-expression-level by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:03:14 GMT -->
</html>
