<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/255084-an-immunoglobulin-binding-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:01:07 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 255084:&quot;AN IMMUNOGLOBULIN-BINDING PROTEIN&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN IMMUNOGLOBULIN-BINDING PROTEIN&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>An immunoglobulin-binding protein of SEQ ID NO. 1 or SEQ ID NO. 2, capable of binding to regions of an immunoglobulin molecule other than the complementarity determining regions (CDR), the improvement consisting of mutating one asparagines residue of the protein having an amino acid sequence of SEQ ID NO. 1 or 2 with an amino acid other than glutamine, wherein said mutated asparagines is N23T, which mutation confers an increased chemical stability at alkaline pH- value.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>A MUTATED IMMUNOGLOBULIN-BINDING PROTEIN<br>
Technical field<br>
The present invention relates to the field of mutant proteins, and more specifically to a mutant protein that exhibits improved stability compared to the parental molecule as well as to a method of producing a mutant protein according to the invention. The invention also relates to an affinity separation matrix, wherein a mutant protein according to the invention is used as an affinity li-gand.<br>
Background<br>
A great number of applications in the biotechnological and pharmaceutical industry require comprehensive attention to definite removal of contaminants. Such contaminants can for example be non-eluted molecules adsorbed to the stationary phase or matrix in a chromatographic procedure, such as non-desired biomolecules or microorganisms, including for example proteins, carbohydrates, lipids, bacteria and viruses. The removal of such contaminants from the matrix is usually performed after a first elution of the desired product in order to regenerate the matrix before subsequent use. Such removal usually involves a procedure known as cleaning-in-place (CIP), wherein agents capable of eluting contaminants from the stationary phase are used. One such class of agents often used is alkaline solutions that are passed over said stationary phase. At present the most extensively used cleaning and sanitising agent is NaOH, and the concentration thereof can range from .0.1 up to e.g. 1 M, depending on the degree and nature of contamination. NaOH is known to be an effective CIP agent achieving multilog reduction of contaminants, such as microbes, proteins, lipids and nucleic acids. Another advantage of NaOH is that it can easily be disposed of without any further treatment. However, tins strategy is associated with exposing the matrix for pH-values above 13. For many affin-lty chromatography matrices containing proteinaceous affinity ligands such al-<br>
kaline environment is a very harsh condition and consequently results in decreased capacities owing to instability of the ligand to the high pH involved.<br>
An extensive research has therefore been focussed on the development of engineered protein ligands that exhibit an improved capacity to withstand alkaline pH-values. For example, Gulich et al (Susanne Giilibri, Martin Linhult, Per-Alee Nygren, Mathias Uhlen, Sophia Hober, Journal of Biotechnology 80 (2000), 169-178: Stability towards alkaline conditions can be engineered into aprotein ligand) suggested protein engineering to improve the stability properties cf a Streptococcal albumin-binding domain (ABD) in alkaline environments. Previously, it was shown that structural modification, such as deamidatfon and cleavage of the peptide backbone, of asparagine and glutamine residues in alkaline conditions is the main reason for loss of activity upon treatment in alkaline solutions, and that asparagine is the most.sensitive of the two (Geiger, T., and S. Clarke. 1987. Deamidation, Isomerization, and Racemization at Aspara-ginyl and Aspartyl Residues in Peptides. J. -Biol. CHem. 262:785-794). It is also known that the deamidation rate is highly specific and conformation dependent (Kosky, A.A., U.O. Razzaq, M.J. Treuheit, and D.N. Brems. 1999. The effects of alpha-helix on the stability of Asn residues: deamidation rates in peptides of varying helicity. Protein Sci. 8:2519-2523; Kossiakoff, A.A. 1988. Tertiary structure is aprincipal determinant to protein deamidation. Science. 240:191-194; and Lura, R., and V. Schirch". 1988. Role of peptide "conformation in the rate and mechanism of deamidation of asparaginyl residues. Biochemistry. 27:7671-7677), and the shortest deamidation'h'alftimes have been associated with the sequences -asparagine-glycine- and'-asparagine-serine. Accordingly, Gulich et al created a mutant of-ABD,'whefein.'all trie four aspargine residues of native ABD have been replaced by leucine (one residue), asparte (two residues) and lysine (one residue). Further, Gulich et al report that their mutant exhibits a target protein binding behaviour similar to that of the native protein, and that affinity columns containing the engineered ligand show'higher binding capaci-<br>
ties after repeated exposure to alkaline conditions than columns prepared using the parental non-engineered ligand. Thus, it is concluded therein that all four asparagine residues can be replaced without any significant effect on structure and function-Thus, the studies performed by Giilich et al were performed on a Streptococcal albumin-binding domain. However, affinity chromatography is also used in protocols for purification of other molecules, such as immunoglobulins, e.g. for pharmaceutical applications. A particularly interesting class of affinity reagents is proteins capable of specific binding to invariable parts of an antibody molecule, such interaction being independent on the antigen-binding specificity of the antibody. Such reagents can be,, widely used for affinitychromatography recovery of immunoglobulins from different samples" siich'as but not limited to serum or plasma preparations or cell culture derived feed stocks. An example of such a protein is staphylococcal protein A, containing domains capable of binding to the Fc and Fab portions of IgG immunoglobulins from different species.<br>
Staphylococcal protein A (SpA) based reagents have due to their high affinity and selectivity found a widespread use in the field of biotechnology, e.g. in affinity chromatography for capture and purification of antibodies as well as for detection. At present, SpA-basedaffihitymedium probably is the most widely used affinity medium for isolation of monoclonal, antibodies and their fragments from different samples including industrial feed stocks from cell cultures. Accordingly, various matrices comprising protein A-ligands are commercially available, for example, in the form of native protein A (e.g. Protein A Sepharose™, Amersham Biosciences, Uppsala, Sweden) and also comprised of recombinant protein A (e.g. rProtein A Sepharose™, Amersham Biosciences, Uppsala, Sweden). More specifically, the genetic manipulation performed in<br>
said commercial recombinant protein A product is aimed at facilitating the attachment thereof to a support.<br>
Accordingly, there is a need in this field to obtain protein ligands capable of binding immunoglobulins, especially via the Fc-fragments thereof, which are also tolerant to one or more cleaning procedures using alkaline agents.<br>
Summary of the present invention<br>
One object of the present invention is to provide a mutated immunoglobulin-binding protein ligand that exhibits an improved stability at increased pH-values, and accordingly an improved tolerance,to cleaning under alkaline conditions, as compared to the parental molecule.<br>
Another object of the invention is to provide such a protein ligand, which binds specifically to the Fc-fragment of hrrmunoglobulins, such as IgG, IgA and/or IgM.<br>
Yet another object of the inventionis to provide, a protein ligand as described above, which also exhibits an affinity which is retained for a longer period of time in alkaline conditions than that of the parental molecule.<br>
A farther object of the present invention is to provide an'affinity separation matrix, which comprises mutant protein ligands capable of binding immunoglobulins, such as IgG, IgA and/or IgM, preferably via their Fc-fragments, which ligands exhibit an improved tolerance to cleaning under alkaline conditions, as compared to the parental molecule ligand.<br>
One or more of the above-defined objects can be achieved as described in the appended claims.<br>
Brief description of drawings<br>
Figure lshows amino acid alignments of the five homologous domains (E, D,<br>
A, B and C) of SpA.<br>
Figure 2 (a) and (b) illustrates the results obtained after alkaline treatment<br>
(cleaning-in-place) of mutant proteins according to the invention as compared<br>
to the destabilised protein Z.<br>
Figure 3 shows the gene encoding the Z(N23T/N3A/N6D)-Cys after insertion<br>
into vector as described in example 4(a).<br>
Figure 4 shows a plasmid map of the plasmid pAY91, as described in example<br>
4(a).<br>
Figure 5 shows the gene encoding the Z(N23T/N3A/N6D) after insertion into<br>
vector as described in example 4(b).<br>
Figure 6 shows an example of plasmid map.for the plasmid pAYlOO as de-<br>
scribed in example 5.<br>
Figure 7 shows the adapter for introducing a Kpnl-siiE into'a vector with SPA<br>
promoter and signal sequence according to example 6.<br>
Figure 8 shows the plasmid pAY104, which contains SPA promoter and signal<br>
sequence to be used for introduction of an adapter containing a Kpnl-site, as<br>
described in example 6.<br>
Figure 9 shows the resulting plasmid, pAY128, after insertion of the adapter<br>
according to example 6.<br>
Figure 10 shows the constructed cloning cassette of example 6, where the<br>
original adapter is underlined.<br>
Figure 11 shows plasmid pAYl 14 after insertion of the Z(N23T/N3 A/N6D)-<br>
Cys-tetramer as described in Example 6.<br>
Figure 12 shows the constructed cloning cassette of example 7, where the<br>
original adapter is underlined.<br>
Figure 13 shows the resulting plasmid, pAY129, after insertion of the adapter<br>
according to example 7.<br>
Figure 14 shows plasmid pAY125 after insertion of the<br>
Z(N23T/N3A7N6D)tetramer-Cys as described in example 7.<br>
Figure 15 is a chromatogram obtained from a separation of human IgG (hlgG),<br>
as described in example 8.<br>
Figure 16 shows graphs that represent the remaining dynamic binding capacity<br>
of the matrices in accordance with example 8.<br>
Definitions<br>
The term "protein" is used herein to describe proteins as well as fragments thereof. Thus, any chain of amino acids that exhibits, a three dimensional structure is included in the term "protein", and protein fragments are accordingly embraced.<br>
The term "functional variant" of a protein means herein a variant protein, wherein the function, in relation to the invention defined as affinity and stability, are essentially retained. Thus, one or more amino acids that are not relevant for said function may have been exchanged.<br>
The term "parental molecule" is used herein for the corresponding protein in the form before a mutation according to the invention has.been introduced. The term "structural stability" refers to the integrity o'f three-dimensional form of a molecule, while "chemical stability" refers to the ability to withstand chemical degradation.<br>
The term "Fc fragment-binding" protein means that the protein is capable of<br>
binding to the Fc fragment of an immunoglobulin. However, it is not excluded<br>
that an Fc fragment-binding protein also can bind other regions, such as Fab<br>
regions of immunoglobulins.	' '	'<br>
In the present specification, if not referred to by their full names, amino acids are denoted with the conventional one-letter symbols.<br>
Mutations are defined herein by the number of the position exchanged, preceded by the wild type or non-mutated amino acid and followed by the mutated<br>
amino acid. Thus, for example, the mutation of an asparagine in position 23 to a threonine is denoted N23T.<br>
Detailed description of the invention<br>
In one aspect, the present invention relates to an immuno globulin-binding protein capable of binding to other regions of the immunoglobulin molecule than the complementarity determining regions (CDR), wherein at least one asparagine residue of a parental immunoglobulm-bindrng protein has been mutated to an amino acid other than gmtamine, which mutation confers an increased chemical stability at alkaline pH-values compared to the parental molecule. The increased stability means that the mutated protein's initial affmity for immunoglobulin is essentially retained for a prolonged period of time, as will be discussed below.<br>
The retained affinity for the target protein achieved according to the invention is in part due to a retained spatial conformation of the mutant protein. The affinity of mutated proteins to immunoglobulins can for example be tested by the skilled person using biosensor technology using for example a Biacore™ 2000 standard set-up (Biacore AB, Uppsala, Sweden), as will be illustrated in the experimental part below. In this context, it is understood' from the term "essentially" retained that the mutated protein exhibits an affinity for immunoglobulin winch is of the same order of magnitude as that of the parental molecule. Accordingly, in an initial phase, the binding capacity of the mutated protein is comparable with that of the parental molecule However, due to the below-discussed chemical stability of the mutated protein, which' is retained in time, its binding capacity will decrease more slowly than that of the parental molecule in an alkaline environment. The environment can be defined as alkaline, meaning of an increased pH-value, for example above about 10, such as up to about 13 or 14, i.e. from 10-13 or 10-14, in general denoted alkaline conditions. Alternatively, the conditions can be defined by the concentration of NaOH,<br>
which can be up to about 1.0 M, such as 0.7 M or specifically about 0.5 M, accordingly within a range of 7-1.0 M.<br>
The increased chemical stability of the mutated protein according to the invention can easily be confirmed by the skilled person in this field e.g. by routine treatment with NaOH at a concentration of 0.5 M, as will be described in the experimental part below. In this context, it is to be understood that similar to what is said above, an "increased" stability means that the initial stability is retained during a longer period of time than what is achieved by the parental molecule. Even though similar mutations have been reported for a Streptococcal albumin-binding domain (Giilich et al, see above), if is well kribwn that the rate of the deamidation involved in protein susceptibility to degradation in alkaline environments is highly sequence and conformation dependent. Since the amino acid sequence of ABD comprises no amino acid sequence similarity to immunoglobulin-binding proteins such as the individual domains staphylococcal protein A, it would not appear as though the teachings of Giilich et al could be applied also to immunoglobulin-binding proteins. However, the present invention shows for the first time that mutation of one or more asparagine residues of an immunoglobulin-binding protein surprisingly provides an improved chemical stability and hence a decreased degradation rate in environments wherein the pH is above about 10, such as up to about 13. or 14.<br>
Thus, the present invention provides a mutated protein, which is useful e.g. as a protein ligand in affinity chromatography for selective adsorption of immunoglobulins, such as IgG, IgA and/or IgM, preferably IgG, from a mammalian species, such as a human. The purpose of the adsorption can be either to produce a purified product, such as a pure immunoglobulin fraction or a liquid from which the immunoglobulin has been removed, or to detect the presence of immunoglobulin in a sample. The ligand according to the invention exhibits a chemical stability sufficient to withstand conventional alkaline cleaning for a<br>
prolonged period of time, which renders the ligand an attractive candidate for cost-effective large-scale operation where regeneration of the columns is a necessity.<br>
Accordingly, in the protein according to the invention, one or more asparagine (N) residues have been mutated to amino acids selected from the group that consists of glycine (G), alanine (A)',' valine (V), leucine'(L), isoleucine (I), serine (S), threonine (T), cysteine (C), methionine (M), phenylalanine (F), tyrosine (Y), tryptophan (W), glutamic acid (E), arginine (R), histidine (H), lysine (K) or proline (P), or any modified amino acid that is not susceptible to the undesired deamidation and isomerisation. Alternatively, one or more asparagine (N) residues have been mutated to glutamine (Q).<br>
The irnrnunoglobulin-binding protein can be any protein with a native immuno-globulin-binding capability, such as Staphylococcal protein A (SpA) or Streptococcal protein G (SpG). For a review of other such proteins, see e.g. Kron-vail, G., Jonsson, K. Receptins: a novel term for an expanding spectrum of natural and engineered microbial proteins with binding properties for mammalian proteins, J. Mol. Recognit. 1999 Jan-Feb; 12(l):3'8-44. Review.<br>
In one embodiment, the present invention is a mutated protein, which comprises at least the binding region of an immunoglobulin-binding protein and wherein at least one such asparagine mutation is present within said region. Accordingly, in this embodiment, a mutated protein according to the invention comprises at least about 75%, such as at least about 80% or preferably at least about 95%, of the sequence as defined in SEQ ID NO. 1 or.2, with the proviso that the asparagine mutation is not in position 21.<br>
In the present specification, SEQ ID NO 1 defines the amino acid sequence of the B-domain of SpA and SEQ ID NO 2 defines a protein known as protein Z.<br>
Protein Z is synthetic construct derived from the B-domain of SpA, wherein the glycine in position 29 has been exchanged for alanine, and it has been disclosed in the literature, see e.g. Stahl et al,' 1999:'Affinity fusions in biotechnology: focus on protein A and protein G, in The Encyclopedia of Bioprocess Technology: Fermentation, Biocatalysis and Bioseparation. M.C, Fleckinger and S.W. Drew, editors. John Wiley and Sons Inc., New York, 8-22. Further, protein Z has been used both as a ligand in affinity chromatography. However, even though protein Z exhibits an improved chemical stability to certain chemicals other than NaOH as compared to the SpA B-domain, it is still not as stable in conditions of increased pH-values as required to withstand the many CIP regeneration steps desired in an economic industrial plant.<br>
In one embodiment, the above described mutant protein is comprised of the amino acid sequence defined in SEQ ID NO 1 or 2, or is- a functional variant thereof. The term "functional variant" as used in this context includes any similar sequence, which comprises one or more further variations in amino acid positions that have no influence on the mutant protein's affinity to immunoglobulins or its improved chemical stability in environments of increased pH~ values.<br>
In an advantageous embodiment, the present mutation(s) are selected from the group that consists of N23T; N23T and-N43E; N28A; N6A; Nl IS; NlIS and N23T; andN6A andN23T; and wherein the parental molecule comprises the sequence defined by SEQ ID NO 2. As mentioned above, in order to achieve a mutant protein useful as a ligand with high binding capacity for a prolonged period of time in alkaline conditions, mutation of the asparagine residue in position 21 is avoided. In one embodiment, the asparagine residue in position 3 is not mutated.<br>
In the most advantageous embodiment, in me present protein, an asparagine residue located between a leucine residue and a glutamine residue has been mutated, for example to a threonine residue. Thus,'in one embodiment, the asparagine residue in position 23 of the sequence defined in SEQ ID NO 2 has been mutated, for example to a threonine residue. In a specific embodiment, the asparagine residue in position 43 of the sequence defined in SEQ ID NO 2 has also been mutated, for example to a glutamic acid. In the embodiments where amino acid number 43 has been mutated, it appears to most advantageously be combined with at least one further mutation, such as N23T.<br>
The finding according to the invention that the various asparagine Tesidues of the B-domain of SpA and protein Z can be ascribed different contributions to affinity and stability properties of the mutated protein was quite unexpected, especially in view of the above discussed teachings of Gulich et al wherein it was concluded that all the asparagine residues of ABD could be mutated without any internal discrimination.<br>
Thus, the invention encompasses the abbve-discussed monomeric mutant proteins. However, such protein monomers can be combined into multimeric proteins, such as dirners, tnmers, tetramers, pentamers etc. Accordingly, another aspect of the present invention is a multimer comprised of at least one of the mutated proteins according to the invention together with one or more further units, preferably also mutant proteins according to the invention. Thus, the present invention is e.g. a dimer comprised of two repetitive units.<br>
In one embodiment, the multimer according to the invention comprises monomer units linked by a stretch of amino acids preferably ranging from 0 to 15 amino acids, such as 5-10. The nature of such a link should preferably not destabilise the spatial conformation of the protein units. Furthermore, said link<br>
should preferably also be sufficiently stable in alkaline environments not to impair the properties of the mutated protein units.<br>
In the best embodiment at present, the multimer is a tetramer of protein Z comprising the mutation N23T, wherein the length of the'linking units are 5-10 amino acids. In one embodiment, the present multimer comprises the sequence VDAKFN-Z(N23T)-QAPKVDAKFN-Z(N23T)QAPKC. In another embodiment, the multimer comprises the sequence VDAKFD-Z(N23T)-QAPKVDAKFD-Z(N23T)-ZQAPKC.<br>
In a specific embodiment, the present multimer also"comprises one or more of the E, D, A, B, and C domains of Staphylococcal protein A. In this embodiment, it is preferred that asparagine residues located in loop regions have been mutated to more hydrolysis-stable.amino acids. In an embodiment advantageous for structural stability reasons, the glycine residue in position 29 of SEQ ID NOS. 1 has also been mutated, preferably to an alanine residue. Also, it is advantageous for the structural stability to avoid mutation of the asparagine residue in position 52, since it has been found to contribute to the a-helical secondary structure content of the protein A molecule.<br>
In a further aspect, the present invention relates to a nucleic acid encoding a mutant protein or multimer as described above. Accordingly, the invention embraces a DNA sequence that can be used in the production of mutant protein by expression thereof in a recombinant host according to .well-established biotech-nologicalmethods. Consequently,-another aspect of thepresent invention is an expression system, which enables production of a mutant protein as described above. Bacterial hosts can conveniently be used, e.g. as described in the experimental part below. In an alternative embodiment, the present invention is a cell line that has been genetically manipulated to express a mutant protein according to the invention. For methods to this end, see e.g. Sambrook et al.,<br>
Molecular Cloning: A Laboratory Manual (2n ed); vols. 1-3, Cold Spring Harbor Laboratory, (1989).<br>
Naturally, once the desired sequence has been established, the mutant protein according to the invention can alternatively be produced by synthetic methods.<br>
Accordingly, the present invention also includes a biotechnological or synthetic method of producing a mutant protein or a multimer according to the invention.<br>
In another aspect, the present invention relates to a matrix for affinity separation, which matrix comprises ligands that comprise irnrnunoglobulm-binding protein coupled to a solid support, in which protein at least.one asparagine residue has been mutated to an amino acid other than glutamine. The present matrix, when compared to a matrix comprised of the parental molecule as ligand, exhibits an increased binding capacity during two or more separations with intermittent alkaline cleaning. The mutated protein ligand is preferably an Fc-fragment-binding protein, and can be used for selective binding of IgG, IgA and/or IgM, preferably IgG.<br>
The matrix according to the invention can comprise the mutant protein as described above in any embodiment thereof as ligand: In the most preferred embodiment, the ligands present on the solid support comprise a multimer as described above.<br>
The solid support of the matrix according to the invention can be of any suitable well-known kind. A conventional affinity separation matrix is often of organic nature and based on polymers that expose a hydrophrlic surface to the aqueous media used, i.e. expose hydroxy (-OH), carboxy (-COOH), carbox-amido (-CONH2, possibly in N- substituted forms), amino (.-NH2, possibly in substituted form), oligo- or polyethylenoxy groups on their external and, if pre-<br>
sent, also on internal surfaces. In one embodiment, the polymers may, for instance, be based on polysaccharides, such as dextran, starch, cellulose, pullulan, agarose etc, which advantageously have been cross-linked, for instance with bisepoxides, epihalohydrins, 1,2,3-trihalo substituted lower hydrocarbons, to provide a suitable porosity and rigidity. In the most preferred embodiment, the solid support is porous agarose beads. The supports used in the present invention can easily be prepared according to standard methods, such as inverse suspension gelation (S Hjerten: Biochirn Bioptrys Acta 79(2), 393-398 (1964). Alternatively, the base matrices are commercially available products, such as Sepharose™ FF (Amersham Biosciences, Uppsala, Sweden). In an embodiment, which is especially advantageous for large-scale separations, the support has been adapted to increase its rigidity, and hence renders the matrix more suitable for high flow rates.<br>
Alternatively, the solid support is based on synthetic polymers, such as polyvinyl alcohol, polyhydroxyalkyl acrylates, polyhydroxyalkyl methacrylates, poly-acrylamides, polymethacrylamides etc. In case of Hydrophobic polymers, such as matrices based on divinyl and monovinyl-substituted benzenes, the surface of the matrix is often hydrophilised to expose hydrophilic groups as defined above to a surrounding aqueous liquid. Such polymers are easily produced according to standard methods, see e.g. "Styrene based polymer supports developed by suspension polymerization" (R Arshady: Chimica e L'Industria 70(9), 70-75 (1988)). Alternatively, a commercially available product, such as Source™ (Amersham Biosciences, Uppsala, Sweden) is used.<br>
In another alternative, the solid support according to the invention comprises a support of inorganic nature, e.g. silica, zirconium oxide etc.<br>
In yet another embodiment, the solid support is in another form such as a surface, a chip, capillaries, a or filter.<br>
As regards the shape of the matrix according to the invention, in one embodiment the matrix is in tire form of a porous monolith. In an alternative embodiment, the matrix is in beaded or particle form that can be porous or non-porous. Matrices in beaded or particle form can be used as a packed bed or in a suspended form. Suspended forms include those known as expanded beds and pure suspensions, in which the particles or beads are free to move. In case of monoliths, packed bed and expanded beds, the separation procedure commonly follows conventional chromatography with a concentration gradient. In case of pure suspension, batch-wise mode will be used.<br>
The ligand may be attached to tire support .via conventional coupling techniques utilising, e.g. amino and/or carboxy groups present in. the ligand. Bisepoxides, epichlorohydrin, CNBr, N-hydroxysuccinimide (HHS)'etc are well-known coupling reagents. Between the support and the ligand, a molecule known as a spacer can be introduced, which will improve the availability of the ligand and facilitate the chemical coupling of the ligand to the support. Alternatively, the ligand may be attached to the support by non-covalent-bonding, such as physical adsorption or biospecific adsorption.<br>
In an advantageous embodiment, the present ligand has been coupled to the support by thioether bonds. Methods for performing such coupling are well-known in this field and easily performed by the skilled person in this field using standard techniques and equipment. In an advantageous' embodiment, the ligand is firstly provided with a terminal cysteine residue for subsequent use in the coupling. The skilled person in this field also easily performs appropriate steps of purification.<br>
As mentioned above, the affinity to immunoglobulin i.e. the binding properties of the present ligand, and hence the capacity of the matrix, is not essentially chan ged in time by treatment with an alkaline agent. Conventionally, for a<br>
cleaning in place treatment of an affinity separation matrix, the alkaline agent used is NaOH and the concentration thereof is up to 0.75 M, such as 0.5 M.<br>
Thus, another way of characterising the matrix according to the invention is that due to the above discussed mutations, its binding capacity will decrease to less than about 70 %, preferably less than about 50% and more preferably less than about 30%, such as about 28%, after treatment with 0.5 M NaOH for 7.5 h.<br>
In a further aspect, the present invention relates to a method of isolating an immunoglobulin, such as IgG, IgA and/or IgM, wherein a mutant protein, a mul-timer or a matrix according to the invention is used. Thus, the invention encompasses a process of chromatography, wherein at least.one target compound is separated from a liquid by adsorption to a mutant protein or a multimer or matrix described above. The desired product can be the separated compound or the liquid. Thus, this aspect of the invention relates to affinity chromatography, which is a widely used and well-known separation technique. In brief, in a first step, a solution comprising the target compounds, preferably antibodies as mentioned above, is passed over a separation matrix under conditions allowing adsorption of the target compound to ligands present on said matrix. Such conditions are controlled e.g. by pH and/or salt concentration i.e. ionic strength in the solution. Care should be taken not to exceed the capacity of the matrix, i.e. the flow should be sufficiently slow to allow a satisfactory adsorption. In this step, other components of the solution will pass through in principle unimpeded. Optionally, the matrix is then washed, e.g. with an aqueous solution, in order to remove retained and/or loosely bound substances. The present matrix is most advantageously used with a washing step utilising' an alkaline agent, as discussed above. In a next step, a second solution denoted an eluent is passed over the matrix under conditions that provide desorption i.e. release of the target compound. Such conditions are commonly provided by a change of the pH,<br>
the salt concentration i.e. ionic strength, hydrophobicity etc. Various elution schemes are known, such as gradient elution and step-wise elution. Elution can also be provided by a second solution comprising a competitive substance, which will replace the desired antibody on the matrix. For a general review of the principles of affinity chromatography, see e.g. Wilchek, M., and Chaiken, I. 2000. An overview of affinity chromatography. Methods Mol. Biol. 147: 1-6.<br>
In an alternative embodiment, a mutant protein according to the invention is used as a lead compound in a process wherein an organic compound is modelled to resemble its three dimensional structure. The so modelled compound is known as a mimetic. Mimetic design, synthesis and testing can be used to avoid randomly screening large number of molecules. In brief, such a method can involve determining the particular parts of th'o protein that are critical and/or important for a property such as immunoglobulin-binding. Once these parts have been identified, its structure is modelled according to its-physical properties, e.g. stereochemistry, bonding, size, charge etc using data from a range of sources, such as spectroscopic techniques, X-ray diffraction data and NMR. Computational analysis, similarity mapping and other techniques can be used in this process. Important considerations in this kind of process are the ease to synthesise a compound, pharmacological acceptance, degradation pattern in vivo etc.<br>
Finally, the present invention also comprises'other uses'of the mutant protein described above, such as in analytical methods, for medical purposes, e.g. for diagnosis, in arrays etc.<br>
Detailed description of the drawings<br>
Figure 1 shows amino acid alignments of the five homologous domains (E,D,A,B and C) of SpA. Horizontal lines indicate amino acid identity. The three boxes show the a-helices of Zwt as determined by Tashiro and co-workers (Tashiro et ai, 1997). The asparagine residues, and also one glycine residue in the B domain, which were replaced, are underlined in the figure. Also, amino acid alignments for Zwt and Z(N23T) are shown.<br>
Figure 2 illustrates the results obtained after alkaline treatment (cleaning-in-place) of mutant proteins according to the invention as compared to the destabilised protein Z. A comparison of the capacity after repeated CIP-treatment following an ordinary affinity chromatography scheme. 0.5 M NaOH was used<br>
as cleaning agent. The protocol was run 16 times and the duration for the alkaline sanitisation was 30 minutes in each round. Figure 2(a) shows the inactiva-tion pattern for Z(F30A) and variants thereof, whereas figure 2(b) shows the inactivation pattern for Zwt and Z(N23T).<br>
Figure 3 shows the gene encoding the Z(N23T/N3A/N6D)-Cys after insertion<br>
into vector as described in example 4(a). The mutations are marked with *.<br>
Figure 4 shows a plasmid map of the plasmid pAY91, which contains the gene<br>
encoding Z(N23T/N3A/N6D)-Cys as described in example 4(a).<br>
Figure 5 shows the gene encoding the Z(N23T/N3A/N6D) after insertion into<br>
vector as described in example 4(b). The mutations.are marked with *.<br>
Figure 6 shows an example of plasmid map for the plasmid pAY100 expressing<br>
the tetramer of Z(N23T/N3A/N6D)-Cys as described in example 5.<br>
Figure 7 shows the adapter for introducing a Kpnl-site, into a vector with SPA<br>
promoter and signal sequence according to example 6.<br>
Figure 8 shows the plasmid pAY104, which contains SPA promoter and signal<br>
sequence to be used for introduction of an adapter containing a KpnI-site, as<br>
described in example 6.<br>
Figure 9 shows the resulting plasmid, pAY128, after insertion of the adapter<br>
according to example 6.<br>
Figure 10 shows the constructed cloning cassette of example 6, where the original adapter is underlined.<br>
Figure 11 shows plasmidpAY114 after insertion of the Z(N23T/N3A/N6D)-Cys-tetramer as described in Example 6.<br>
Figure 12 shows the constructed cloning cassette of example 7, where the original adapter is underlined.<br>
Figure 13 shows the resulting plasmid, pAY129, after insertion of the adapter according to example 7.<br>
Figure 14 shows plasmid pAY125 after insertion of the Z(N23T/N3A/N6D)tetramer-Cys as described in example 7. Figure 15 is a chromatogram obtained from a run as described in example 8, where the first peak corresponds to the flow-through material and the second peak corresponds to eluted hlgG.<br>
Figure 16 shows graphs that represent the remaining dynamic binding capacity of the matrices in accordance with example 8, From top to bottom they represent Z(N23T/N3A/N6D)dimer-Cys, Z(N23T/N3A)dimer-Cys, Z(N23T)dimer-Cys and Z(N23T/K4G)dimer-Cys respectively. Due to software problems the last two measure points for Z(N23T/N3A)dimer-Cys are lacking.<br>
EXPERIMENTAL PART ' Below, the present invention will be described by way of examples, which are provided for illustrative purposes only and accordingly are not to be construed as limiting the scope of the present invention as. defined by the appended claims. All references given below and elsewhere in this application are hereby included herein by reference.<br>
In this part, since Z in its original form already has a significant but non-sufficient stability towards alkaline treatment, it was assumed that small changes in stability due to the mutations would be difficult to assess in labora-<br><br>
tory testings. Therefore, a suppressor mutation method (kotsuka, T., S. Akanuma, M. Tomuro, A. Yamagishi, and T. Oshima. 1996. Further stabilisation of 3-isopropylmalate dehydrogenase of an extreme thermophile, Thermus thermophilus, by a suppressor mutation method. J Bacterial. 178:723-727; and Sieber, V., A. Pliickthun, and F.X. Schmidt. 1998. Selecting proteins with improved stability by a phage-based method. Nature Biotechnology. 16:955-960) was used to provide a variant of the Z domain with a decreased structural stability. According to this strategy the destabilised variant of protein Z, herein denoted Z(F30A) (Cedergren et al., 1993, supra) was used as scaffold for subsequent introduction of additional mutations related to investigations of alkaline stability. The binding properties of this variant'are similar to native protein Z, since F30 is not involved in the Fc-binding.<br>
Further, Zwt denotes the wild type Z domain, not containing the F30A substitution.<br>
Experimental strategy<br>
To analyze which asparagines in the Z domain that are responsible for its instability in alkaline conditions, a mutational analysis was performed. In order to enable detection of improvements regarding the alkaline stability of the Z domain, it was decided to use a mutated variant, Z(F30A), since the Z-domain already possesses a significant but non-sufficient stability towards alkaline con-ditions. Z(F30A) has earlier been shown to possess an affinity to IgG that is similar to the wild type, but also a remarkably decreased structural stability due to the mutation of an amino acid that normally takes part in the hydrophobic core (Cedergren et al., 1993, supra; Jendeberg, L.,B. Persson, R. Andersson, R. Karlsson, M. Uhlen, and B. Nilsson. 1995. Kinetic analysis of the interaction between protein A domain variants and human Fc using plasmon resonance detection. Journal of Molecular Recognition. 8:270-278). The Z-domam is a three-helix bundle consisting of 58 amino acids, including eight asparagines (N3, N6, Nil, N21, N23, N28, N43 andN52) (Figure 1) (Nilsson, B., T. Moks,<br>
B. Jansson, L. Abrahmsen, A. Elmblad, E. Holmgren, C. Henrichson, T.A. Jones, and M. Uhlen. 1987. A synthetic IgG~binding domain based on staphylococcal protein A. Protein Eng. 1:107-113). To evaluate the effect of the different asparagrnes on the deactivation rate in alkaline conditions, seven of these residues were exchanged for other amino acids. Since N3 is located in the flexible amino-terrninal of the domain, it was excluded from the study. It was assumed that a degradation of this amino acid would not affect the activity of a monomeric ligand and would therefore not be detectable in the present assay, which measures the retained activity. Moreover, since the amino acid is located outside the structured part of the domain it will presumably be easily replaceable during a multimerisation of the domain to achieve a protein A"-like molecule. To facilitate the protein design, a comparison with the homologous sequences from the other domains of protein A was made (Fig. 1) (Gulich et al., 2000a). From the comparison, it was decided to .exchange asparagine 11 for a serine and 23 for threonine and finally 43 for a glutamic acid. Asparagine 6 was exchanged for alanine since the alternative when looking on the homologous sequences was asparric acid, which also has been reported to be sensitive in alkaline conditions. All five domains of protein A have asparagines in the other positions (21, 28, 52). Hence, they were exchanged for alanines.<br>
Example 1: Mutagenesis, expression and purification of mutant protein Z Materials and methods<br>
Site-directed mutagenesis was performed using a two-step PCR-technique (Higuchi et al., 1988). Plasmid pDHZF30A (Cedergren et al, 1993) was used as template. Oligonucleotides coding for the different asparagine replacements and the A29G replacement were synthesised by.Interac.tiva (Interactiva Bio-technologie GmbH, Ulm, Germany). The restriction enzymes Xbal and Hindlll (MBI Fermentas Inc., Amhurst, NY) were'used for cloning into the vector pDHZ (Jansson et al., 1996) that was performed according to Sambrook (Sambrook et al., 1987). To create pTrpZ, the Z domain was amplified by PCR,<br>
using plasmid pKNl as template (Nord et al., 1995). The fragment was restricted with Xbaland PstI and ligated into tire vector pTrpABDTlT2 (Kraulis et al., 1996) that had been restricted with same enzymes.A MegaBACE 1000 DNA Sequencing System (Amersham Biosciences, Uppsala, Sweden) was used to verify correct sequence of inserted fragments. MegaBACE terminator chemistry (Amersham Biosciences, Uppsala, Sweden) was' utilised according to the supplier's recommendations in a cycle sequencing protocol based on the dide-oxy method (Sanger et al., 1977). During cloning procedures, Escherichia coli strain RR1AM15 (American Type Culture Collection, Rockville, MA) was used, whereas for expression of the different gene products 017 (Olsson, M.O., and L.A. Isaksson. 1979. Analysis of rpsD Mutations in Escherichia coli. I: Comparison of Mutants with Various Alterations in Ribosomal Protein S4. Molec. gen. Genet. 169:251-257) was used."' ,<br>
Production and purification of Z(F30A) and the different constructs thereof were performed according to the protocol outlined by Giilich (Giilich et al., 2000b, see above). The production of Z and pZ(N23T) were performed as described in Kraulis et al (Kraulis, P.J., P. Jonasson, P.-A. Nygren, M. Uhlen, L. Jendeberg, B. Nilsson, and J. Kordel. 1996. The serum albumin-binding domain of streptococcal proteinG is a three-helix bundel: aheteronuclear NMR study. FEBS lett. 378:190-194). Relevant fractions were lyophilised. The amount of protein was estimated by absorbance measurements at 280.nm using the specific absorbance coefficient, a (1g-1 cm-1), Z 0.15.6; Z(N23T), 0.169; Z(F30A), Z(F30A,N43E), Z(F30A,N23T,N43E).0.157; Z(F30A,N6A), Z(F30A,N1 IS), Z(F30A,N21A), Z(F30A,N23T), Z(F30A,N28A), Z(F30A,N52A), Z(F30A,N6A,N23T), Z(F30A,N11S,N23T) 0.158. The concentration was confirmed by amino acid analysis (BMC, Uppsala, Sweden). The homogeneity was analysed by Sodium Dodecyl Sulfate PolyAcrylamide Gel Electrophoresis (SDS-PAGE) ( Laemmli, U.K. 1970. Cleavage of Structural Proteins during the Assembly of the Head of Bacteriophage T4. Nature.' 227:680-685) using the<br>
Phast system. Lyophilised proteins were loaded on high-density gels (Amer-sham Biosciences, Uppsala, Sweden) under reducing conditions and stained with Coomassie Brilliant Blue according to the supplier's recommendations. The homogeneity and the molecular weights were further confirmed by mass spectrometry.<br>
For CD spectroscopy, protein samples were prepared in a phosphate buffer (8.1 mM K2HP04, 1.9 mM KH2P04, pH 7.5) to a concentration of 10 µM. Spectra were recorded using a J-720 spectropolarimeter (JASCO, Tokyo, Japan) in the far UV region from 250 to 200 nm at RT in a quartz cell of path length 0.1 cm and with a scan speed of 10 nm min" . Each spectrum was the mean of five accumulated scans and the final spectra were converted into mean residue ellip-ticity (MRE) (deg cm2 dmol-1).<br>
Results (example 1)<br>
All Z variants were successfully produced intracellular in E. coli at 37°C and show the same expression levels, approximately 50 mgA as estimated from SDS-PAGE. The proteins were all purified by IgG affinity chromatography. After the purification, samples were analysed with.SDS-PAGE (data not shown), lyophilised and stored for further analyses. Themolecular mass for protein Z and the different mutants thereof were also confirmed by mass spectrometry. The data confirmed correct amino acid content for all mutants (data not shown). Also, structural analyses were performed on a Circular Dichroism (CD) equipment, since it previously has been proven to be suitable for detecting structural changes in α-helical proteins (Johnson, C.W., Jr. 1990. Protein secondary structure and circular dichroism: a practical guide. Proteins. 7:205-214; and Nord, K., J. Nilsson, B. Nilsson, M. Uhlen, and P.-A. Nygren. 1995. A combinatorial library of an a-helical bacterial receptor domain. Prot. eng. 8:601-608). All spectra show a minimum at208 nm and at 222 nm in combination with a maximum around 195 nm, indicating a similar structure for the mu-<br>
tants and the parental molecule. However, Z(F30A,N52A) seems to have a somewhat lower α-helicity than the wild type Z and the other mutants thereof (data not shown).<br>
Example 2: Biospecific interaction analysis Materials and methods<br>
Differences in affinity and kinetic constants of the-.association and dissociation states were detected on a Biacore™ 2000 instrument (Biacore, Uppsala, Sweden). Human polyclonal IgG and HSA (negative reference) were immobilised by amine coupling on the carboxylated dextran layer of a CMS sensor chip (Biacore) according to the supplier's recommendations. The immobilisation of IgG resulted in approximately 2000 RU. Z, ZF30A, and the different mutants were prepared in FfBS (10 mM HEPES, 0.15 M NaCl, 3.4 mM EDTA, 0.005% surfactant P20, pH 7.4) at 10 different concentrations (100-550 nM). The samples were injected over the surfaces as duplicates in'random, order at a flow rate of 30 µl min-1. 10 mM HC1 was used to regenerate the surface. The data was analysed using the BIA evaluation 3.0.2b software (Biacore AB). The signals from a control surface immobilized with HSA were subtracted from the IgG surface. A 1:1 Langmuir model was assumed and apparent kinetic constants and also affinity constants were calculated. Also, the change in free binding energy (AAG = -RTln Kaff, mutant(/Kaffi native) in relation to the native molecule was calculated.<br>
Results (example 2)<br>
To determine the differences in affinity for the Z variants towards IgG, surface plasmon resonance (SPR) using a Biacore was carried out, The aim was to compare the affinity for the different mutated Z variants according to the invention with the parental molecule. As mentioned above, due to the high alkaline stability of the parental Z domain it was decided to use a structurally destabilised variant of Z including the F30A mutation (Cedergren, L., R. Andersson,<br>
B. Jansson, M. Uhlen, and B. Nilsson. 1993. Mutational analysis of the interaction between staphylococcal protein A and human IgG]. Protein eng. 6:441-448). Therefore, it was of importance to first confirm that the affinity between the mutated molecule and IgG was retained despite the mutation. As can be seen in table 1 below, the affinity of Z(F30A) is not significantly affected. The very small change in affinity gives a slightly higher stability to the complex of Z(F30A) and IgG compared to the parental molecule Z and IgG. This is in accordance with results earlier reported by Jendeberg et al. (Cedergren et al., 1993, supra; Jendeberg et al., 199i, supra). All mutants constructed with Z(F30A) as scaffold were analysed and compared with their parental molecule (Z(F30A)). The results show that the overall affinity is not significantly affected by the mutations, indicating that none of the asparagine mutations ac-cording to the invention are very important for the binding to IgG (see table 1 below). In all Z variants including the N21A or the N43E mutation, only a slightly lower affinity constant was observed. For mutants with the N23T mutation, surprisingly, the affinity even seems to be slightly higher. Also, in the case of the N28A-mutation, the decrease in affinity is very small, and cannot be expected to have any essential influence if the mutant protein is used e.g. as a protein ligand. Furthermore, all constructs including the N28A-mutation have a remarkably increased off-rate. For the mutants including the N23T mutation the somewhat increased affinity seems to be due to a.slightly increased on-rate. Also, the N6A-mutation gives a higher on-rate, but the'affinity constant is not affected because of the increased off-rate that also follows the mutation.<br>
Table 1<br>
An overview of the kinetic study on the different Z domains carried out using the Biacore.<br>
(Table Removed) <br>
Zwt was used as an internal standard during the different measurements. The<br>
differences in free binding energy are calculated relative to Zwt and Z(F30A)<br>
respectively.<br>
Example 3: Stability towards alkaline conditions Materials and methods<br>
The behaviour of the variants of domain Z as affinity ligands was analysed by immobilisation to a standard affinity matrix. Z, Z(F30A), and mutated variants were covalently coupled to HiTrap™ affinity columns (Amersham Biosciences, Uppsala, Sweden) using the N-hydroxysuccinimide chemistry according to the<br>
manufacturer's recommendations. The columns were pulsed with TST and 0.2 M HAc, pH 3.1. Human polyclonal IgG in TST was prepared and injected onto the columns in excess. A standard affinity chromatography protocol was followed for 16 cycles on the AKTA™Explorer 10 (Amersham Biosciences, Uppsala, Sweden). Between each cycle a CIP-step was integrated. The cleaning agent was 0.5 M NaOH and the contact time for each pulse was 30 minutes, resulting in a total exposure time of 7.5 hours. Eluted material was detected at 280 run.<br>
Results (example 3)<br>
Z, Z(F30A), and mutants thereof were covalently attached to HiTrap™ columns using NHS-chemistry. IgG in excess was loaded and the amount of eluted IgG was measured after each cycle to determine the total capacity of the column. Between each cycle the columns were exposed to CTP treatment consisting of 0.5 M NaOH. After 16 pulses, giving a total exposure time of 7.5 hours, the column with the Z(F30A)-matrix shows a 70 % decrease of the capacity. The degradation data in figure 2a suggest that four of the exchanged asparagines (N6, N11, N43 andN52) are less sensitive to the alkaline conditions the mutants are exposed for in this experiment. In contrast, N23 seems to be very important for the stability of Z(F30A). Z(F30A,N23T) shows only a 28% decrease of capacity despite the destabilising F30A-mutation. Hence, the Z(F30A,N23T) is almost as stable as Zwt and thereby the most stabilised variant with Z(F30A) as scaffold. Also the Z(F30A)-domain with two additional mutations Z(F30A,N23T,N43E) shows the same pattern of degradation as Z(F30A,N23T). An exchange of N28 to an alanine also improves the stability of Z(F30A) towards alkaline conditions. Surprisingly, the column with Z(F30A,N21A) as affinity ligand reveals a dramatic loss of capacity when exposed to NaOH compared to the parental molecule. These data make Z(N23T) to a very advantageous candidate as ligand in affinity purification of IgG.<br>
To finally prove the reliability of the strategy using a structurally destabilised variant of a molecule in order to make small changes in stability detectable, the N23T-mutation was grafted into the parental Z-domain. Both the parental Z-domain and Z(N23T) were coupled to HiTrap-columns and exposed to alkaline conditions in the same way as for the already mentioned mutants. As can be seen in figure 2b, the Z(N23T)-mutant shows higher stability than Zwt when exposed to high pH.<br>
Example 4: Construction of monomers of Z-mutants with and without a C-terminal cysteine Three different mutations were introduced in a gene encoding Z(N23T): K4G, N3A and the double-mutation N3A/N6D.<br>
The mutations were originally introduced into two different vectors: one with a cysteine in the C-terminus and one without the cysteine. This was done to later facilitate the construction of multimers with one single C-terminal cysteine.<br>
Example 4(a): Cysteme-contaming monomer construction As template for the construction, a plasmid denoted "pGEM ZN23T", was used. This already contained the N23T-mutation in the Z-gene. A PCR-reaction was performed with this plasmid as template and the two oligonucleotides<br>
(Sequence Removed) <br>
for the N3A-mutation and,<br>
(Sequence Removed) <br>
for the N3A/N6D-mutation.<br>
PCR reaction tubes containing: 0,5 µl template pGEM ZN23T [500 ng/µl], 5 pmol of each primer (Interactiva, Thermo Hybaid GmbH, Ulm, Germany), 5 µl of dNTP-mix ([10 mM], Applied Biosystems, CA, USA), 5 µl of PCR-buffer 10x (Applied Biosystems, CA, USA), 0,1 µl of AmpliTaq ([5 U/µl], Applied Biosystems, CA, USA) and sterile water to a final volume of 50 µl. The PCR-program consisted of 2 min at 94°C followed by 30 cycles of 15 sec at 96°C, 15 sec at 50°C, 1 min at 72°C and concluded with an additional min at 72°C. The PCR reactions were performed on GeneAmp® PCR System 9700 (Applied Biosystems, CA, USA).<br>
The PCR-product was analysed on 1% agarose gel and, after confirming an obtained product of correct size, purified with QIAquick   PCR purification kit (QIAGEN GmbH, Hilden, Germany).<br>
The PCR-products were cleaved according to Sambrook (Sambrook et al.) with the restriction enzymes Accl and Pstl (New England Biolabs, NEB, MA, USA). The cleavage products were analysed on agarose gel and purified from the agarose with QIAquick® Gel Extraction Kit (QLAGEN GmbH, Hilden, Germany) prior to ligation. The fragments were ligated into a vector denoted "pTrp-protA-stab-(multi9)", already cleaved with the enzymes Accl and Pstl and purified, by adding T4 DNA ligase and ligation buffer (MBI Fermentas, Lithuania), and subsequently transformed into RRIAM15-cells (ATCC, MA, USA). The constructs were given the names pAY87 (Z(N23T/K4G)-Cys), pAY89 (Z(N23T/N3A)-Cys) andPAY91 (Z(N23T/N3A/N6D)-Cys), respectively. A MegaBACE™ 1000 DNA Sequencing System (Amersham Biosciences, Uppsala, Sweden) was used to verify correct sequences of inserted fragments. MegaBACE™ terminator chemistry (Amersham Biosciences, Uppsala, Swe-<br>
den) was utilised according to the supplier's recommendations in a cycle sequencing protocol based on the dideoxy method (Sanger et al., 1977).<br>
Example 4(b): Non-cysteine-containing monomer construction <br>
As template for the construction, a plasmid denoted "pTrp(-N)ZN23T-Cys'n, was used. This plasmid already contained the gene with the N23T-mutation. A PCR-reaction was performed with this plasmid as template and the two oligonucleotides<br>
(Sequence Removed) <br>
for the N3 A-mutation,<br>
(Sequence Removed) <br>
and for the N3A/N6D-mutation.<br>
PCR-reaction tubes containing: 0,5 µl template pTrp(-N)ZN23T-Cys [500 ng/µl], 5 pmol of each primer (Interactiva, Thermo Hybaid GmbH, Ulm, Germany), 5 µl of dNTP-mix (10 mM, Applied Biosystems, CA, USA), 5 µl of PCR-buffer 10x (Applied Biosystems, CA, USA), 0,1 µl of AmphTaq ([5 U/µl], Applied Biosystems, CA, USA) and sterile water to a final volume of 50 µl. The PCR-program consisted of 2 min at 94°C followed by 30 cycles of 15 sec at 96°C, 15 sec at 50°C, 1 min at 72°C and concluded with an additional min at 72°C. The PCR reactions were performed on GeneAmp® PCR System 9700 (Applied Biosystems, CA, USA).<br>
The PCR-products were directly TA-cloned into the vector pGEM according to the manufacturer's instructions (Promega, Wl, USA) and subsequently trans-<br>
formed into RRIAM15-cells (ATCC, MA, USA). The constructs were given the names pAYS6 (Z(N23T/K4G), pAY88 (Z(N23T/N3A) andpAY90 (Z(N23T/N3A/N6D) respectively.<br>
A MegaBACE™ 1000 DNA Sequencing System (Amersham Biosciences, Uppsala, Sweden) was used to verify correct sequences of inserted fragments. MegaBACE™ terminator chemistry (Amersham Biosciences, Uppsala, Sweden) was utilised according to the supplier's recommendations in a cycle sequencing protocol based on tire dideoxy method (Sanger et al., 1977).<br>
Example 5: Construction of the gene encoding monomers and oligomers with a C-terminal cysteine in pTip-vector All of the above described plasmids pAY 86 to pAY91 (a total of six plasmids) were cleaved with the restriction enzyme Accl. This resulted in releasing the Z-mutants completely from the pAY86-, pAY88- and pAY90-vectors and a single opening at the 3 '-end of the gene in the pAY87-, pAY89- and pAY91-vectors. The cleaved vectors were treated with Calf Intestine Alkaline Phosphatase (CIAP, MBI Fermentas, Lithuania) according to the manufacturer's recommendations. This step was performed to dephosphorylate the 5'-ends to avoid self-ligation of the vectors.<br>
The released Z-mutant-fragments were analysed on agarose gel and subsequently purified from the agarose before the fragments was ligated into the opened vectors according to the following: fragment from pAY86 to pAY87 fragment from pAY88 to pAY89 fragment from pAY90 to pAY91<br>
For the ligation reactions, different proportions of fragment versus vector were mixed and the result was that a range of different multimers, as expected, ranging from dimers to pentamers was obtained.<br>
The different multimers were transformed into RRIAM15-cells (ATCC, MA, USA) and the correct sequences were verified by analysis on a sequencing equipment at the Royal Institute of Technology as described above. The newly constructed plasmids were denoted as shown in the table below:<br>
Table 2<br>
Summary of constructed plasmids<br>
(Table Removed) <br>
The above described plasrmd vectors, except pAY86, pAY88 and pAY90 have Trp promoter, Trp leader sequence and a gene for kanamycin (Km) resistance. pAY86, pAY88 and pAY90 have a gene for ampicillm resistance instead.<br>
Example 6: Construction of fienes encoding monomers and oligomers with a C-terminal cysteine in pK4-vector The genes encoding the proteins as summarised in Table 2 were to be transferred to a vector containing SPA promoter and signal sequence. To enable this procedure, an adapter containing the cleavage site for the restriction enzyme Kpnl (New England Biolabs, NEB, MA, USA) was to be constructed. The adapter was constructed by the two oligonucleotides (Interactiva, Thermo Hy-baid GmbH, Ulm, Germany)<br>
The plasmid pAY104 (pK4-cys-ABDstabdimer) was cleaved with FspI and Pstl (New England Biolabs, NEB, MA, USA). The vector was purified on agarose gel and the released fragment was removed and the remaining vector was purified from the agarose with QIAquick   Gel Extraction Kit (QIAGEN GmbH, Hilden, Germany).<br>
The two oligomers AFFI-88 and AFFI-89 were mixed in ligation buffer (MBI Fermentas, Lithuania) and heated to 50°C and the mixture was allowed to cool to room temperature where after the cleaved plasmid vector was added together with T4 DNA ligase (MBI Fermentas, Lithuania). After the ligation reaction, the product was transformed into RRIAM15-cells and the correct sequence was verified as described above. The resulting plasmid was denoted pAY128.<br>
The plasmid pAY128 was then cleaved with the restriction enzymes Kpnl and Pstl and the cleaved vector was analysed on agarose gel and subsequently purified from the agarose with QIAquick® Gel Extraction Kit (QIAGEN GmbH, Hilden, Germany). The fragments expressing the two mutated Z-genes Z(N23T/N3A) and Z(N23T/N3A/N6A) from pAY86 to pAY103 were cleaved with Kpnl and Pstl (New England Biolabs, NEB, MA, USA), separated and purified after agarose gel separation. The different fragments were ligated into the<br>
cleaved vector originating from pAY128 and the resulting plasmids were, after verifying correct sequences, denoted pAYl 07  to pAY116 as summarised in Ta-ble3.<br>
Table 3<br>
Summary of constructed plasmids with SPA promoter and signal sequence.<br>
(Table Removed) <br>
Example 7: Construction of genes encoding a part of the E-gene (TO from protein A N-terminally fused to monomers and oligomers with a C-terminal cysteine in pK4-vector The genes encoding the proteins, as summarised in Table 2, were transferred to a vector containing the SPA promoter and signal sequence and apart of the gene encoding the E-region of protein A (E'). It has earlier been shown that an addition of the N-terminal IgG-binding part of the mature protein A (region E), or parts thereof, may increase correct processing and also facilitate secretion of the gene product to the surrounding culture medium (Abrahmsen et al., 1985). An adapter containing the cleavage site for the restriction enzyme Kpnl and a part of region E from protein A (E') was constructed by the two oligonucleotides (Interactiva, Thermo Hybaid GmbH, Ulm, Germany)<br>
The plasmid pAY104 (pK4-cys-ABDstabdimer) was cleaved with FspI and Pstl (New England Biolabs, NEB, MA, USA). The vector was purified on agarose gel and the released fragment was removed and the remaining vector was purified from the agarose with QIAquick® Gel Extraction Kit (QIAGEN GmbH, Hilden, Germany). The two oligonucleotides were mixed in ligation buffer and heated to 75°C and the mixture was allowed to cool to room temperature where after the cleaved plasmid vector was added, together with T4 DNA ligase (MB I Fermentas, Lithuania). After the ligation reaction the product was transformed into RRIAM15-cells and the correct sequence was verified as described above. The resulting plasmid was denoted pAY129.<br>
The plasmid pAY129 was then cleaved with the restriction enzymes Kpnl and Pstl and the cleaved vector was analysed on agarose gel and subsequently purified from the agarose with QIAquick® Gel Extraction Kit (QIAGEN GmbH, Hilden, Germany). The fragments expressing the two mutated Z-genes Z(N23T/N3A) and Z(N23T/N3A/N6A) from pAY86 to pAY103 were cleaved with Kpnl and Pstl, separated and purified after agarose gel separation. The different fragments were ligated into the cleaved vector originating from pAY129 and the resulting plasmids were, after verifying correct sequences, denoted pAYl 18 to pAY127 as summarised in Table 4.<br>
Table 4<br>
Summary of constructed plasmids with SPA promoter and signal sequence and<br>
a part of region E from protein A - E'.<br>
(Table Removed) <br>
Example 8: Stabihty towards alkaline conditions To evaluate the stabrlity of the proteins towards alkaline conditions, four different proteins were tested in an environment of high pH. The different proteins were Z(N23T)drmer-Cys, Z(N23T/K4G)dimer-Cys, Z(N23T/N3A)dimer-Cys and Z(N23T/N3A/N6D)dimer-Cys.<br>
(Z(N23T)dimer-Cys), (Z(N23T/N3A)dimer-Cys), (Z(N23T/N3A/N6D)dimer-Cys) and (Z(N23T/K4G)dimer-Cys) were cultivated in fermenters. The harvested media were purified and coupled to HF Agarose (Amersham Bio-sciences, Uppsala, Sweden) using standard methods before the alkaline tests. The HF agarose-coupled proteins were denoted as follows<br>
Z(N23T)dimer-Cys	U631049<br>
Z(N23T/K4G)dimer-Cys	U631079<br>
Z(N23T/N3A)dimer-Cys	U631064<br>
Z(N23T/N3A/N6D)dimer-Cys U631063 The matrices were packed in columns (HR 5/2, Amersham Biosciences, Uppsala, Sweden) to a final volume ranging from 0,1 to 0,3 ml. The purifica-<br>
tion equipment used was an AKTA™ Explorer 10 (Amersham Biosciences, Uppsala, Sweden) with a UV sample flow cell with a path length of 2 mm (Amersham Biosciences, Uppsala, Sweden).<br>
The buffers used contained<br>
Running buffer: 25 mM Tris-HCl, 1 mM EDTA, 200 mM NaCl, 0,05% Tween 20, 5 mM ammonium acetate, pH 8,0<br>
•	Elution buffer. 0,2 M acetic acid (HAc), pH 3,1<br>
Cleaning-In-Place (CIP) buffer. 0,5 M NaOH<br>
A typical chromatographic run cycle consisted of<br>
•	Equilibrium of the column with running buffer<br>
•	Sample application of 10 mg polyclonal human IgG (hlgG) at 0,2 ml/min<br>
•	Extensive washing-out of unbound proteins<br>
•	Elution at 1,0 ml/min with elution buffer<br>
•	Re-equilibration with running buffer<br>
•	Cleaning-In-Place (CIP) with CIP-buffer with a contact time between column matrix and 0,5 M NaOH of 1 hour<br>
•	Re-equilibration with miming buffer<br>
The amount of hlgG loaded at each run was well above the total dynamic binding capacity of the column since the breakthrough of unbound protein was considerable when loading the sample onto the columns in all cases. After one cycle, including the steps above, a new cycle was started which again included one hour of exposure of 0,5 M sodium hydroxide. To measure the decrease of the dynamic binding capacity of the column the peak area of the eluted peak was compared with the original peak area of the eluted peak when the matrix had not been exposed to the sodium hydroxide. Setting the original peak area as 100% of binding capacity the decrease of the binding capacity of<br>
hIgG was observed. The peak area was calculated with the UNICORN™ software accompanying the purification system.<br>
Each cycle was repeated 21 times resulting in a total exposure time between the matrix and the sodium hydroxide of 20 hours for each different matrix. The normalised peak areas were visualised in a graph as can be seen below (Figure 16). All 21 cycles were repeated for each mutant.<br>
Both Z(N23T/N3A/N6D)dimer-Cys and Z(N23T/N3A)dimer-Cys showed improved stability against alkaline conditions compared to the originally produced Z(N23T)dimer-Cys.<br><br><br><br><br>
We claim<br>
1.	An immunoglobulin-binding protein of SEQ ID NO. 1 or SEQ ID NO. 2, capable of binding to regions of an immunoglobulin molecule other than the complementarity determining regions (CDR), the improvement consisting of mutating one asparagines residue of the protein having an amino acid sequence of SEQ ID NO. 1 or 2 with an amino acid other than glutamine, wherein said mutated asparagines is N23T, which mutation confers an increased chemical stability at alkaline pH- value.<br>
2.	A protein as claimed in claim 1 which binds to an Fc fragment and the Fc portion of a full antibody.<br>
3.	A multimer comprised of two or more proteins as claimed in claim 1.<br>
4.	A multimer as claimed in claim 3 , wherein the proteins are linked by elements including up to 15 amino acids.<br>
5.	A multimer as claimed in claim 3, further comprising one or more of the E, D, A, B, and C domains of Staphylococcus protein A.<br>
6.	A multimer as claimed in claim 3, comprising three of said proteins.<br>
7.	A multimer as claimed in claim 3, further comprising the protein defines by SEQ ID NO. 2.<br>
8.	A nucleic acid encoding an immunoglobulin-binding protein as claimed in claim 1.<br>
9.	An expression system, which includes the nucleic acid as claimed in claim 8.<br>
10.	A matrix for affinity chromatography, comprising a plurality of ligands which include the immunoglobulin-binding protein as claimed in claim 1 coupled to a solid support.<br>
11.	 A matrix as claimed in claim 10, wherein the ligands are coupled to the support by thioether bonding.<br><br>
12.	A matrix as claimed in claim 10, wherein the support is a natural polymer material.<br>
13.	A matrix as claimed in claim 10, selectively binds an immunoglobulin selected from the group consisting of IgG, IgA and IgM.<br>
14.	A matrix as claimed in claim 10, wherein the ligands exhibit an increased binding capacity, during two or more separations with intermittent alkaline cleaning, as compared to the ligands which include the protein having an amino acid sequence of SEQ ID NO. 1 or 2.<br>
15.	A matrix as claimed in claim 14, wherein the cleaning is performed with NaOH and the concentration thereof is up to 1M<br>
16.	A nucleic acid encoding a multimer as claimed in claim 3.<br>
17.	An expression system, which includes the nucleic acid as claimed in claim 16.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUFic3RyYWN0LSgxOC0xMC0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Abstract-(18-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUFic3RyYWN0LSgyMi0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Abstract-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNsYWltcy0oMTgtMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Claims-(18-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNsYWltcy0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Claims-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNvcnJlc3BvZGVuY2UgT3RoZXJzLSgyMi0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Correspodence Others-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDYtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Correspondence Others-(06-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTMtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Correspondence Others-(13-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMTctMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Correspondence Others-(17-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTYtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Correspondence-Others-(16-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTctMDItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Correspondence-Others-(17-02-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTgtMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Correspondence-Others-(18-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjktMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Correspondence-Others-(29-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LURyYXdpbmdzLSgyMi0wNy0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Drawings-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMS0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-1-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tMTMucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-13.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMi0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-2-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMy0oMDYtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-3-(06-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMy0oMTMtMDQtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-3-(13-04-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUZvcm0tMy0oMTYtMDctMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Form-3-(16-07-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUZvcm0tMy0oMTctMDItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Form-3-(17-02-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1ERUxOUC0yMDA0LUZvcm0tMy0oMTctMTEtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-DELNP-2004-Form-3-(17-11-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMy0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-3-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUZvcm0tMy0oMjktMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Form-3-(29-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LUdQQS0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-GPA-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC0xMDEucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC0yMDEucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-201.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC00MDkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-409.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LXBjdC00MTYucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-pct-416.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LVBldGl0aW9uIDEzNy0oMjItMDctMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Petition 137-(22-07-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LVBldGl0aW9uIDEzNy0oMjktMTEtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Petition 137-(29-11-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjUyMy1kZWxucC0yMDA0LVBldGl0aW9uLTEzNy0oMDYtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2523-delnp-2004-Petition-137-(06-09-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="255083-compressed-pharmaceutical-compositions-comprising-peg-and-electrolytes.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="255085-a-method-and-arrangement-for-enhanced-transmission-quality-in-push-to-talk-system.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>255084</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2523/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Jan-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>22-Jan-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Aug-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>GE HEALTHCARE BIO-SCIENCES AB</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BJORKGATAN 30, S-751 84 UPPSALA, SWEDEN</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>SOPHIA HOBER</td>
											<td>SVEAVAGEN 61, S-113 59 STOCKHOLM, SWEDEN</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C07K 14/31</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/SE2003/00475</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-03-20</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>0200943-9</td>
									<td>2002-03-25</td>
								    <td>Sweden</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/255084-an-immunoglobulin-binding-protein by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:01:08 GMT -->
</html>
