<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/255051-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-ptp4a1-gene-expression-level by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:03:21 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 255051:AN IN-VITRO METHOD OF PREDICTING THE RESPONSE OF A NON SMALL CELL LUNG CANCER PATIENT TO ERLOTINIB BY DETERMINING PTP4A1 GENE EXPRESSION LEVEL</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">AN IN-VITRO METHOD OF PREDICTING THE RESPONSE OF A NON SMALL CELL LUNG CANCER PATIENT TO ERLOTINIB BY DETERMINING PTP4A1 GENE EXPRESSION LEVEL</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention provides a biomarkei&amp;quot; which is predictive for the clinical benefit of EGFR inhibitor treatment in cancer palienls.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>FORM 2<br>
THE PATENTS ACT, 1970 (39 of 1970)<br>
&amp; THE PATENTS RULES, 2003<br>
COMPLETE SPECIFICATION<br>
[See section 10, Rule 13]<br>
PREDICTIVE   MARKER   FOR   EGFR   INHIBITOR<br>
TREATMENT;<br>
F.	HOFFMANN-LA        ROCHE        AG.,        A<br>
CORPORATION ORGANIZED AND EXISTING UNDER THE LAWS OF SWITZERLAND, WHOSE ADDRESS IS GRENZACHERSTRASSE 124, CH-4070 BASEL, SWITZERLAND.<br>
THE	FOLLOWING	SPECIFICATION<br>
PARTICULARLY DESCRIBES THE INVENTION AND THE MANNER IN WHICH IT IS TO BE PERFORMED.<br><br>
The present invention provides a biomarker that is predictive for the clinical benefit of EGFR inhibitor treatment in cancer patients.<br>
A number of human malignancies are associated with aberrant or over-expression of the epidermal growth factor receptor (EGFR). EGF, transforming growth factor-a (TGF-a), and a number of other ligands bind to the EGFR, stimulating autophosphorylation of the intracellular tyrosine kinase domain of the receptor. A variety of intracellular pathways are subsequently activated, and these downstream events result in tumour cell proliferation in vitro. It has been postulated that stimulation of tumour cells via the EGFR may be important for both tumour growth and tumour survival in vivo.<br>
Early clinical data with Tarceva™, an inhibitor of the EGFR tyrosine kinase, indicate that the compound is safe and generally weil tolerated at doses that provide the targeted effective concentration (as determined by preclinical data). Clinical phase I and II trials in patients with advanced disease have demonstrated (hat Tarceva™ has promising clinical activity in a range of epithelial tumours. Indeed, Tarceva™ has been shown to be capable of inducing durable partial remissions in previously treated patients with head and neck cancer, and NSCLC (Non small cell lung cancer) of a similar order to established second line chemotherapy, but with the added benefit of a better safety profile than chemo therapy and improved convenience (tablet instead of intravenous [i.v.] administration). A recently completed, randomised, double-blind, placebo-controlled trial (BR.21) has shown that single agent Tarceva™ significantly prolongs and improves the survival of NSCLC patients for whom standard therapy for advanced disease has failed.<br>
Tarceva™ (erlotinib) is a small chemical molecule; it is an orally active, potent, selective inhibitor of the EGFR tyrosine kinase (EGFR-TKI).<br>
Lung cancer is the major cause of cancer-related death in North America and Europe. In the United States, the number of deaths secondary to lung cancer exceeds the combined total deaths from the second (colon), third (breast), and fourth (prostate) leading causes of cancer deaths combined. About 75% to 80% of all iung cancers are NSCLC, with approximately 40% of patients presenting with locally advanced and/or unresectable disease. This group typically includes those with bulky stage IIIA and IIIB disease, excluding malignant pleural effusions.<br>
The crude incidence of lung cancer in the European Union is 52.5, the death rate 48,7 cases/100000/year. Among men the rates are 79.3 and 78.3, among women 21.6 and 20,5,<br><br><br>
respectively. NSCLC accounts for 80% of all lung cancer cases. About 90% of lung cancer mortality among men, and 80% among women, is attributable to smoking.<br>
In the US, according to the American Cancer Society, during 2004, there were approximately 173,800 new cases of lung cancer (93,100 in men and 80,700 in women) and were accounting for about 13% of all new cancers. Most patients die as a consequence of their disease within two years of diagnosis. For many NSCLC patients, successful treatment remains elusive. Advanced tumours often are not amenable to surgery and may also be resistant to tolerable doses of radiotherapy and chemotherapy. In randomized trials the currently most active combination chemotherapies achieved response rates of approximately 30% to 40% and a 1-year survival rate between 35% and 40%. This is really an advance over the 10% 1-year survival rate seen with supportive care alone (Shepherd 1999).<br>
Until recently therapeutic options for relapsed patients following relapse were limited to best supportive care or palliation. A recent trial comparing docetaxel (Taxotere) with best supportive care showed that patients with NSCLC could benefit from second line chemotherapy after cisplatin-based first-line regimens had failed. Patients of all ages and with ECOG performance status of 0, I, or 2 demonstrated improved survival with docetaxel, as did those who had been refractory to prior platinum-based treatment. Patients who did not benefit from therapy included those with weight loss of 10%, high lactate dehydrogenase levels, multi-organ involvement, or liver involvement. Additionally, the benefit of docetaxel monotherapy did not extend beyond the second line setting. Patients receiving docetaxel as third-line treatment or beyond showed no prolongation of survival. Single-agent docetaxel became a standard second-line therapy for NSCLC. Recently another randomized phase III trial in second line therapy of NSCLC compared pemetrexed (Alimta®) with docetaxel. Treatment with pemetrexed resulted in a clinically equivalent efficacy but with significantly fewer side effects compared with docetaxel.<br>
It has long been acknowledged that there is a need to develop methods of individualising cancer treatment. With the development of targeted cancer treatments, there is a particular interest in methodologies which could provide a molecular profile of the tumour target, (i.e. those that are predictive for clinical benefit). Proof of principle for gene expression profiling in cancer has already been established with the molecular classification of tumour types which are not apparent on the basis of current morphological and immunohistochemical tests. Two separate disease entities were differentiated with differing<br><br>
prognoses from the single current classification of diffuse large B-cell lymphoma using gene expression profiling.<br>
Therefore, it is an aim of the present invention to provide expression biomarkers that are predictive for the clinical benefit of EGFR inhibitor treatment in cancer patients.<br>
In a first object the present invention provides an in vitro method of predicting the clinical benefit of a cancer patient in response to treatment with an EGFR inhibitor comprising the steps: determining an expression level of a PTP4A1 gene in a tumour sample of a patient and comparing the expression level of the PTP4A1 gene to a value representative of an expression level of the PTP4A1 gene in tumours of a population of patients deriving no clinical benefit from the treatment, wherein a lower expression level of the PTP4A1 gene in the tumour sample of the patient is indicative for a patient who will derive clinical benefit from the treatment.<br>
The abbreviation PTP4A1 means protein tyrosine phosphatase type IVA, member 1. Seq. Id. No. 1 shows the nucleotide sequence of human PTP4A1.<br>
The term "a value representative of the expression level of the PTP4A1 gene in tumours of a population of patients deriving no clinical benefit from the treatment" refers to an estimate of the mean expression level of the marker gene in tumours of a population of patients who do not derive a clinical benefit from the treatment. Clinical benefit was defined as either having an objective response or disease stabilization for &gt;12 weeks.<br>
In a further preferred embodiment, the PTP4A1 gene shows between 1.2 and 1.8 or more fold lower expression level in the tumour sample of the patient compared to a value representative of the population of patients deriving no clinical benefit from the treatment.<br>
In a preferred embodiment, the expression level of the marker gene is determined by microarray technology or other technologies that assess RNA expression levels like quantitative RT-PCR, or by any method looking at (he expression level of the respective protein, eg immunohistochemistry (IHC). The construction and use of gene chips are well known in the art. see, U. S. Pat Nos. 5,202,23 1; 5,445,934; 5,525,464; 5,695,940; 5,744,305; 5,795, 716 and 1 5,800,992. See also, Johnston, M. Curr. Biol. 8:R171-174 (1998); Iyer VR et al., Science 283:83-87 (1999). Of course, the gene expression level can be determined by other methods that are known to a person skilled in the art such as e.g. northern blots, RT-PCR, real time quantitative PCR, primer extension, RNase protection, RNA expression profiling.<br><br><br>
The marker gene of the present invention can be combined with other biomarkers to biomarker sets. Biomarker sets can be built from any combination of predictive biomarkers to make predictions about the effect of EGFR inhibitor treatment in cancer patients. The biomarkers and biomarkers sets described herein can be used, for example, to predict how patients with cancer will respond to therapeutic intervention with an EGFR inhibitor.<br>
The term "gene" as used herein comprises variants of the gene. The term "variant" relates to nucleic acid sequences which are substantially similar to the nucleic acid sequences given by the GenBank accession number. The term "substantially similar" is well understood by a person skilled in the art. In particular, a gene variant may be an allele which shows nucleotide exchanges compared to the nucleic acid sequence of the most prevalent allele in the human population. Preferably, such a substantially similar nucleic acid sequence has a sequence similarity to the most prevalent allele of at least 80%, preferably at least 85%, more preferably at least 90%, most preferably at least 95%. The term "variants" is also meant to relate to splice variants.<br>
The EGFR inhibitor can be selected from the group consisting of gefitinib, erlotinib, PKI-166, EKB-569, GW2016, CI-1033 and an anti-erbB antibody such as trastuzumab and cetuximab.<br>
In another embodiment, the EGFR inhibitor is erlotinib.<br>
In yet another embodiment, the cancer is NSCLC.<br>
Techniques for the detection and quantification of gene expression of the genes described by this invention include, but are not limited to northern blots, RT-PCR, real time quantitative PCR, primer extension, RNase protection, RNA expression profiling and related techniques. These techniques are well known to those of skill in the art see e.g. Sambrook J et al., Molecular Cloning: A Laboratory Manual, Third Edition (Cold Spring Harbor Press, Cold Spring Harbor, 2000).<br>
Techniques for the detection of protein expression of the respective genes described by this invention include, but are not limited to immunohistochemistry (IHC).<br>
In accordance with the invention, cells from a patient tissue sample, e.g., a tumour or cancer biopsy, can be assayed to determine the expression pattern of one or more biomarkers. Success or failure of a cancer treatment can be determined based on the biomarker expression pattern of the cells from the test tissue (test cells), e.g., tumour or cancer biopsy, as being relatively similar or different from the expression pattern of a control set of the one or more biomarkers. In the context of this invention, it was found that the gene of table 3 is down<br><br>
regulated i.e. shows a lower expression level, in tumours of patients who derived clinical benefit from EGFR inhibitor treatment compared to tumours of patients who did not derive clinical benefit from the EGFR inhibitor treatment. Thus, if the test cells show a biomarker expression profile which corresponds to that of a patient who responded to cancer treatment, it is highly likely or predicted that the individual's cancer or tumour will respond favorably to treatment with the EGFR inhibitor. By contrast, if the test cells show a biomarker expression pattern corresponding to that of a patient who did not respond to cancer treatment, it is highly likely or predicted that the individual's cancer or tumour will not respond to treatment with the EGFR inhibitor.<br>
The biomarker of the present invention i.e. the gene listed in table 3, is a first step towards an individualized therapy for patients with cancer, in particular patients with refractory NSCLC. This individualized therapy will allow treating physicians to select the most appropriate agent out of the existing drugs for cancer therapy, in particular NSCLC. The benefit of individualized therapy for each future patient are: response rates / number of benefiting patients will increase and the risk of adverse side effects due to ineffective treatment will be reduced.<br>
In a further object the present invention provides a therapeutic method of treating a cancer patient identified by the in vitro method of the present invention. Said therapeutic method comprises administering an EGFR inhibitor to the patient who has been selected for treatment based on the predictive expression pattern of the gene of table 3. A preferred EGFR inhibitor is erlotinib and a preferred cancer to be treated is NSCLC.<br>
Short description of the figures<br>
Figure 1 shows the study design;<br>
Figure 2 shows the scheme of sample processing;<br>
Figure 3a shows PTP4A1 expression levels versus clinical outcome for Genechip® profiling;<br>
Figure 3b shows PTP4A1 expression levels versus clinical outcome for qRT-PCR and<br>
Figure 3c shows the correlation between Genechip® and qRT-PCR measurements for PTP4A1.<br>
Experimental part<br>
Rationale for the Study and Study Design<br><br>
Recently mutations within the EGFR gene in the tumour tissue of a subset of NSCLC patients and the association of these mutations with sensitivity to erlotinib and gefitinib were described (Pao W, et al. 2004; Lynch et al. 2004; Paez et al. 2004). For the patients combined from two studies, mutated EGFR was observed in 13 of 14 patients who responded to gefitinib and in none of the 11 gefitinib-lreated patients who did not respond. The reported prevalence of these mutations was 8% (2 of 25) in unselected NSCLC patients. These mutations were found more frequently in adenocarcinomas (21%), in tumours from females (20%), and in tumours from Japanese patients (26%). These mutations result in increased in vitro activity of EGFR and increased sensitivity to gefitinib. The relationship of the mutations to prolonged stable disease or survival duration has not been prospectively evaluated.<br>
Based on exploratory analyses from ihe BR.21 study, it appeared unlikely that the observed survival benefit is only due to the EGFR mutations, since a significant survival benefit is maintained even when patients with objective response are excluded from analyses (data on file). Other molecular mechanisms must also contribute to the effect.<br>
Based on the assumption that there are changes in gene expression levels that are predictive of response / benefit to Tarceva™ treatment, microarray analysis was used to detect these changes<br>
This required a clearly defined study population treated with Tarceva™ monotherapy after failure of 1st line therapy. Based on the experience from the BR.21 study, benefiting population was defined as either having objective response, or disease stabilization for 12 weeks. Clinical and microarray datasets were analyzed according to a pre-defined statistical plan.<br>
The application of this technique requires fresh frozen tissue (FFT). Therefore a mandatory biopsy had to be performed before start of treatment. The collected material was frozen in liquid nitrogen (N2).<br>
A second tumour sample was collected at the same time and stored in paraffin (formalin fixed paraffin embedded, FFPE). This sample was analysed for alterations in the EGFR signaling pathway.<br>
The ability to perform tumour biopsies via bronchoscopy was a prerequisite for this study. Bronchoscopy is a standard procedure to confirm the diagnosis of lung cancer. Although generally safe, there is a remaining risk of complications, e.g. bleeding.<br><br>
This study was a first step towards an individualized therapy for patients with refractory NSCLC. This individualized therapy will allow treating physicians to select the most appropriate agent out of the existing drugs for this indication.<br>
Once individualized therapy will be available, the benefit for each future patient will outweigh the risk patients have to take in the present study:<br>
response rates / number of benefiting patients will increase,<br>
the risk of adverse side effects due to ineffective treatment will be reduced.<br>
Rationale for Dosage Selection<br>
Tarceva™ was given orally once per day at a dose of 150 mg until disease progression, intolerable toxicities or death. The selection of this dose was based on pharmacokinetic parameters, as well as the safety and tolerability profile of this dose observed in Phase 1, II and III trials in heavily pre-treated patients with advanced cancer. Drug levels seen in the plasma of patients with cancer receiving the 150 mg/day dose were consistently above the average plasma concentration of 500 ng / ml targeted for clinical efficacy. BR.21 showed a survival benefit with this dose.<br>
Objectives of the Study<br>
The primary objective was the identification of differentially expressed genes that are predictive for benefit (CR, PR or SD ? 12 weeks) of Tarceva™ treatment. Identification of differentially expressed genes predictive for "response" (CR, PR) to Tarceva™ treatment was an important additional objective.<br>
The secondary objectives were to assess alterations in the EGFR signaling pathways with respect to benefit from treatment.<br>
Study Design<br>
Overview of Study Design and Dosing Regimen<br>
This was an open-label, predictive marker identification Phase II study. The study was conducted in approximately 26 sites in about 12 countries. 264 patients with advanced NSCLC following failure of at least one prior chemotherapy regimen were enrolled over a 12 month period. Continuous oral Tarceva™ was given at a dose of 150 mg/day. Dose reductions were permitted based on tolerability to drug therapy. Clinical and laboratory parameters were assessed to evaluate disease control and toxicity. Treatment continued until disease progression, unacceptable toxicity or death. The study design is depicted in figure 1.<br><br><br>
Tumour tissue and blood samples were obtained for molecular analyses to evaluate the effects of Tarceva™ and to identify subgroups of patients benefiting from therapy.<br>
Predictive Marker Assessments<br>
Biopsies of the tumour were taken within 2 weeks before start of treatment. Two different samples were collected:<br>
The first sample was always frozen immediately in liquid N2<br>
The second sample was fixed in formalin and embedded in paraffin<br>
Snap frozen tissue had the highest priority in this study.<br>
Figure 2 shows a scheme of the sample processing.<br>
Microarray Analysis<br>
The snap frozen samples were used for laser capture microdissection (LCM) of tumour cells to extract tumour RNA and RNA from tumour surrounding tissue. The RNA was analysed on Affymetrix microarray chips (HG-U133A) to establish the patients' tumour gene expression profile. Quality Control of Affymetrix chips was used to select those samples of adequate quality for statistical comparison.<br>
Single Biomarker Analyses on Formalin Fixed Paraffin Embedded Tissue<br>
The second tumour biopsy, the FFPE sample, was used to perform DNA mutation, IHC and ISH analyses as described below. Similar analyses were performed on tissue collected at initial diagnosis.<br>
The DNA mutation status of the genes encoding EGFR and other molecules involved in the EGFR signaling pathway were analysed by DNA sequencing. Gene amplification of EGFR and related genes were be studied by FISH.<br>
Protein expression analyses included immunohistochemical [IHC] analyses of EGFR and other proteins within the EGFR signalling pathway.<br>
Response Assessments<br>
The RECIST (Uni-dimensional Tumour Measurement) criteria were used to evaluate response. These criteria can be found under the following link: http://www.eortc.be/recist/<br>
Note that:To be assigned a status of CR or PR, changes in tumour measurements must be confirmed by repeated assessments at least 4 weeks apart at any time during the treatment period.<br><br>
In the case of SD, follow-up measurements must have met the SD criteria at least once after study entry at a minimum interval of 6 weeks.<br>
In the case of maintained SD, follow-up measurements must have met the SD criteria at least once after study entry with maintenance duration of at least 12 weeks.<br>
Survival Assessment<br>
A regular status check every 3 months was performed either by a patient's visit to the clinic or by telephone. All deaths were recorded. At the end of the study a definitive confirmation of survival was required for each patient.<br>
Methods<br>
RNA sample preparation and quality control of RNA samples<br>
All biopsy sample processing was handled by a pathology reference laboratory; fresh frozen tissue samples were shipped from investigator sites to the Clinical Sample Operations facility in Roche Basel and from there to the pathology laboratory for further processing. Laser capture microdissection was used to select tumour cells from surrounding tissue. After LCM, RNA was purified from the enriched tumour material. The pathology laboratory then carried out a number of steps to make an estimate of the concentration and quality of the RNA.<br>
RNases are RNA degrading enzymes and are found everywhere and so all procedures where RNA will be used must be strictly controlled to minimize RNA degradation. Most mRNA species themselves have rather short half-lives and so are considered quite unstable. Therefore it is important to perfonn RNA integrity checks and quantification before any assay.<br>
RNA concentration and quality profile can be assessed using an instrument from Agilent (Agilent Technologies, Inc., Palo Alto, CA) called a 2100 Bioanalyzer®. The instrument software generates an RNA Integrity Number (RIN), a quantitation estimate (Schroeder, A., et al., The RIN: an RNA integrity number for assigning integrity values to RNA measurements. BMC Mol Biol, 2006. 7: p. 3), and calculates ribosomal ratios of the total RNA sample. The RIN is determined from the entire electrophoretic trace of the RNA sample, and so includes the presence or absence of degradation products.<br>
The RNA quality was analysed by a 2100 Bioanalyzer®. Only samples with at least one rRNA peak above the added poly-1 noise and sufficient RNA were selected for further<br><br><br>
analysis on the Affymetrix platform. The purified RNA was forwarded to the Roche Centre for Medical Genomics {RCMG; Basel, Switzerland) for analysis by microarray. 122 RNA samples were received from the pathology lab for further processing.<br>
Target Labeling of tissue RNA samples<br>
Target labeling was carried out according to the Two-Cycle Target Labeling Amplification Protocol from Affymetrix (Affymetrix, Santa Clara, California), as per the manufacturer's instructions.<br>
The method is based on the standard Eberwine linear amplification procedure but uses two cycles of this procedure to generate sufficient labeled cRNA for hybridization to a microarray.<br>
Total RNA input used in the labeling reaction was 1 Ong for those samples where more than lOng RNA was available; if less than this amount was available or if there was no quantity data available (due to very low RNA concentration), half of the total sample was used in the reaction. Yields from the labeling reactions ranged from 20-180fxg cRNA. A normalization step was introduced at the level of hybridization where 15ug cRNA was used for every sample.<br>
Human Reference RNA (Stratagene, Carlsbad, CA, USA) was used as a control sample in the workflow with each batch of samples. lOng of this RNA was used as input alongside the test samples to verify that the labeling and hybridization reagents were working as expected.<br>
Microarray hybridizations<br>
Affymetrix HG-U133A microarrays contain over 22,000 probe sets targeting approximately 18,400 transcripts and variants which represent about 14,500 well-characterized genes.<br>
Hybridization for ail samples was carried out according to Affymetrix instructions (Affymetrix Inc., Expression Analysis Technical Manual, 2004). Briefly, for each sample, 15(ig of biotin-labeled cRNA were fragmented in the presence of divalent cations and heat and hybridized overnight to Affymetrix HG-U133A full genome oligonucleotide arrays. The following day arrays were stained with streptavidin-phycoerythrin (Molecular Probes; Eugene, OR) according to the manufacturer's instructions. Arrays were then scanned using a<br><br><br>
GeneChip Scanner 3000 (Affymetrix), and signal intensities were automatically calculated by GeneChip Operating Software (GCOS) Version 1.4 (Affymetrix).<br>
Statistical Analysis<br>
Analysis of the Affymetrix™ data consisted of five main steps.<br>
Step 1 was quality control. The goal was to identify and exclude from analysis array data with a sub-standard quality profile.<br>
Step 2 was pre-processing and normalization. The goal was to create a normalized and scaled "analysis data set", amenable to inter-chip comparison. It comprised background noise estimation and subtraction, probe summarization and scaling.<br>
Step 3 was exploration and description. The goal was to identify potential bias and sources of variability. It consisted of applying multivariate and univariate descriptive analysis techniques to identify influential covariates.<br>
Step 4 was modeling and testing. The goal was to identify a list of candidate markers based on statistical evaluation of the difference in mean expression level between "clinical benefit" and "no clinical benefit" patients. It consisted in fitting an adequate statistical model to each probe-set and deriving a measure of statistical significance.<br>
Step 5 was a robustness analysis. The goal was to generate a qualified list of candidate markers that do not heavily depend on the pre-processing methods and statistical assumptions. It consisted in reiterating the analysis with different methodological approaches and intersecting the list of candidates. All analyses were performed using the R software package.<br>
Step 1: Quality Control<br>
The assessment of data quality was based on checking several parameters. These included standard Affymetrix GeneChip™ quality parameters, in particular: Scaling Factor, Percentage of Present Call and Average Background. This step also included visual inspection of virtual chip images for detecting localized hybridization problems, and comparison of each chip to a virtual median chip for detecting any unusual departure from median behaviour. Inter-chip correlation analysis was also performed to detect outlier samples. In addition, ancillary measures of RNA quality obtained from analysis of RNA samples with the Agilent Bioanalyzer™ 2100 were taken into consideration.<br>
Based on these parameters, data from 20 arrays were excluded from analysis. Thus data from a total of 102 arrays representing 102 patients was included in the analysis. The clinical description of these 102 samples set is reported in table 1.<br><br>
Table I: Description of clinical characteristics of patients included in the analysis<br><br>
Variable	Value	n = 102<br>
		n   (%)<br>
Best Response	N/A	16 (15.7%)<br>
	PD	49 (48.0%)<br>
	SD	31 (30.4%)<br>
	PR	6 (5.9%)<br>
Clinical Benefit	NO	81 (79.4%)<br>
	YES	21 (20.6%)<br>
SEX	FEMALE	25 (24.5%)<br>
	MALE	77 (74,5%)<br>
ETHNICITY	CAUCASIAN	65 (63.7%)<br>
	ORIENTAL	37 (36.3%)<br>
Histology	ADENOCARCINOMA	35 (34.3%)<br>
	SQUAMOUS	53 (52.0%)<br>
	OTHERS	14 {13.7%)<br>
Ever-Smoking	NO	20 (19.6%)<br>
	YES	82 (80.4%)<br>
Step 2: Data pre-processing and normalization<br>
The rma algorithm (Irizarry, R.A., et al., Summaries of Affymetrix GeneChip probe level data. Nucl. Acids Res., 2003. 31(4): p. el5) was used for pre-processing and normalization. The mas5 algorithm (AFFYMETRIX, GeneChip® Expression: Data Analysis Fundamentals. 2004, AFFYMETRIX) was used to make detection calls for the individual probe-sets. Probe-sets called "absent" or "marginal" in all samples were removed from further analysis; 5930 probe-sets were removed from analysis based on this criterion. The analysis data set therefore consisted of a matrix with 16353 (out of 22283) probe-sets measured in 102 patients.<br>
Step 3: Data description and exploration<br>
Descriptive exploratory analysis was performed to identify potential bias and major sources of variability. A set of covariates with a potential impact on gene expression profiles was screened. It comprised both technical and clinical variables. Technical covariates included: date of RNA processing (later referred to as batch), RIN (as a measure of RNA quality/integrity), Operator and Center of sample collection. Clinical covariates included: Histology type, smoking status, tumour grade,  performance score (Oken, M.M., et al.,<br><br><br><br>
Toxicity and response criteria of the Eastern Cooperative Oncology Group. Am J Clin Oncol, 1982. 5(6): p. 649-55), demographic data, rcsponder status and clinical benefit status.<br>
The analysis tools included univariate ANOVA and principal component analysis. For each of these covariates, univariate ANOVA was applied independently to each probe-set.<br>
A significant effect of the batch variable was identified. In practice, the batch variable captured differences between dates of sample processing and Affymetrix chip lot. After checking that the batch variable was nearly independent from the variables of interest, the batch effect was corrected using the method described in Johnson, W.E., C. Li, and A. Rabinovic, Adjusting batch effects in microarray expression data using empirical Bayes methods. Biostat, 2007. 8(1): p. 118-127.<br>
The normalized data set after batch effect correction served as the analysis data set in subsequent analyses.<br>
Histology and RIN were two additional important variables highlighted by the descriptive analysis.<br>
Step 4: Data modeling and testing.<br>
A linear model was fitted independently to each probe-set. Variables included in the model are reported in table 2. The model parameters were estimated by the maximum likelihood technique. The parameter corresponding to the "Clinical Benefit" variable (XI) was used to assess the difference in expression level between the group of patients with clinical benefit and the group with no clinical benefit.<br>
Table 2: Description of the variables included in the linear model.<br><br>
Variable	Type	Value<br>
gene expression	Dependent (Yip)	log2 intensity of probe-set i in patient p.<br>
Intercept	Overall mean <u></u>
Clinical Benefit	Predictor of interest (XI)	YES / NO<br>
Histology	Adjustment Covariate (X2)	ADENO. / SQUAM. / OTHERS<br>
RACE	Adj. Cov. (X3)	ORIENT. / CAUCAS.<br>
SEX	Adj. Cov. (X4)	FEMALE / MALE<br>
RIN	Adj. Cov. (X5)	[2,...,7.9]<br><br>
SMOKER	Adj. Cov. (X6)	C U RRENT/PAST/NE VER<br>
Stage	Adj. Cov. (X7)	UNRESECT.III / IV<br>
For each probe-set i, the aim of the statistical test was to reject the hypothesis that the mean expression levels in patients with clinical benefit and patients without clinical benefit are equal, taking into account the other adjustment covariates listed in table 2. Formally, the null hypothesis of equality was tested against a two sided alternative.The corresponding p-values are reported in table 3.<br>
The choice of linear model was motivated by two reasons. Firstly, linear modeling is a versatile, well-characterized and robust approach that allows for adjustment of confounding variables when estimating the effect of the variable of interest. Secondly, given the sample size of 102, and the normalization and scaling of the data set, the normal distribution assumption was reasonable and justified.<br>
For each probe-set, the assumption of homogeneity of variance was evaluated using Fligner-Killeen tests based on the model residuals. The analysis consisted of 3 steps :<br>
1.	Test each categorical variables for homogeneity of residual variance<br>
2.	Note the variable V with the least p-value<br>
3.	If the least p-value is less than 0.001, re-fit the model allowing the different level of variables V to have a different variance.<br>
Step 5: Robustness<br>
The goal of the robustness analysis was to reduce the risk that the results of the analysis might be artifactual and a result of the pre-processing steps or assumptions underlying the statistical analysis. The following three aspects were considered: a) inclusion or exclusion of a few extra chips at the quality control step; b) pre-processing and normalization algorithm; c) statistical assumptions and testing approach.<br>
The list of candidate markers was defined as the subset of genes consistently declared as significant with different analysis settings. The different applied analysis options were the following:<br>
a)	An additional subset of 8 chips was identified based on more stringent quality<br>
control criteria. A "reduced data set" was defined by excluding these 8 chips.<br><br>
b)	MAS5 was identified as an alternative to rma for pre-processing and normalization. MAS5 uses different methods for background estimation, probe summarization and normalization.<br>
c)	Two additional statistical tests were employed.<br>
a.	A wilcoxon test for the difference between clinical and no clinical benefit and<br>
b.	a likelihood ratio test (LRT) testing for the logistic regression model where<br>
clinical benefit was taken as the response variable and gene expression as covariate. These<br>
two additional tests rely on a different set of underlying statistical assumptions. For each<br>
probe-set, the LRT was following a Chi-square with 1 degree of freedom.<br>
In summary, two sets of samples (the "full" data-set and the "reduced" data-set), and 2 pre-processing algorithm (mas5 and rma) were considered; this resulted in four different analysis data sets. To each of these four data sets, three different statistical tests were applied. Therefore, for each probe-set, three p-values were calculated. In each analysis data set, a composite criterion was applied to identify the list of differentially regulated genes. This composite criterion was defined as: the maximum p-value is less than 0.05 and the minimum p-values is less than 0.001. The robustness analysis using criterion 1 for identifying marker genes yielded PTP4A1 as predictive marker for EGFR inhibitor treatment.<br>
Table 3: Gene marker for Clinical Benefit based on the robustness analysis after application of the composite Criterion.<br>
Column 1 is the Affymetrix identifier of the probe-set Column 2 is the GenBank accession number of the corresponding gene sequence. Column 3 is the corresponding official gene name. Column 4 is the corresponding adjusted mean fold change in expression level between clinical and no clinical benefit patient, as estimated from the linear model. Column 5 is the p-value for the test of difference in expression level between clinical benefit and no clinical benefit patients as derived from the linear model. Column 6 is the 95% confidence interval for the adjusted mean fold change in expression level.<br><br>
Affymetrix	GenBank	Gene	Adjusted	P-value	CI 95%<br>
Probe Set ID			Mean Fold Change		<br>
200732_s_at	NM 003463 (Seq.   Id.   No.	PTP4A1	-1.44	3.2.1E-3	-1.8,-1.2<br>
	1)				<br><br>
Further statistical analysis<br>
For the selected candidate marker PTP4A1. the following additional analyses were performed in a validated environment by an independent statisticians :<br>
•	Univariate Cox Regression  for PFS (Progression free survival) from Primary Affymetrix Analysis,<br>
•	Univariate Logistic Regression  for Clinical  Benefit from Primary Affymetrix Analysis, and<br>
•	Univariate Cox Regression for Survival from Primary Affymetrix Analysis<br>
The results of these analysis are presented below. They are consistent with the results of the primary analysis and confirm the choice of the selected marker.<br>
Results: Univariate Cox Regression for PFS (Progression free survival) from Primary<br>
Affymetrix Analysis:<br>
Gene	No. of patients        Hazard ratio        95   %   CI   for   p-Value<br>
Hazard ratio<br>
PTP4A1	102	1.69	1.23; 2.33	0.0011<br>
Results: Univariate Cox Regression  for Clinical benefit from Primary Affymetrix Analysis:<br>
Gene	No. of patients        Odds ratio	95   %   CI   for    p-Value<br>
Odds ratio<br>
PTP4A1	102	0.22	0.08; 0.57	0.0018<br>
Results: Univariate Cox Regression for Survival from Primary Affymetrix Analysis:<br>
Gene	No. of patients	Hazard ratio        95   %   CI   for   p-Value<br>
Hazard ratio<br>
PTP4A1	102	1.63	1.15; 2.30	0.0055<br>
qRT-PCR<br>
cDNA was synthesized using SuperScriptTM ill First-strand Synthesis SuperMix for qRT-PCR (Invitrogen, CA, USA) according to the manufacturer's instructions but without inclusion of an RNase H digest.<br><br>
Quantitative PCR was performed using TaqMan® Gene Expression Assays on an ABI PRISM® 7900HT Sequence Detection System according to the manufacturer's recommendations (Applied Biosystems, CA, USA). All assays were performed in triplicate,<br>
The used primers and probes crossed exon boundaries or were within the Affymetrix Genechip® probe sequence of interest. Two house-keeping genes were included as endogenous controls: beta-2-microglobulin (B2M; Assay Hs99999907_ml) and hypoxanthinephosphoribosyl transferase (HPRT; Assay Hs99999909_ml).<br>
All runs included a calibrator sample (MVPTM total RNA from human adult lung; Stratagene, CA, USA) and a standard curve. Universal Human Reference total RNA (Stratagene, CA, USA) was used as template for PTP4A1 standard curves. All samples were measured in triplicate. Relative quantification was performed using the -ACt method.<br>
Results<br>
As reported previously, Affymetrix Genechip® gene expression profiles were determined for 102 patients included in this study. Among these patients, qRT-PCR results were obtained for 75 (table 4). The demographics and clinical characteristics of the patients with qRT-PCR results were similar to those of the entire population (n=264) and of the patients with Genechip® gene expression profiles available.<br>
Table 4: Baseline characteristics: patients with qRT-PCR analyses (n=75)<br><br>
Characteristic	<br>
Age (median, range)	62 (39-85)<br>
Gender; n (%)	<br>
Male	19(25)  .<br>
Female	56 (75)<br>
ECOG performance status; n (%)	<br>
0	7(9)<br>
1	45 (60)<br>
2	23(31)<br>
Histology; n (%)	<br>
Adenocarcinoma	27 (36)<br>
Squamous-cell carcinoma	34 (45)<br>
Large-cell carcinoma	2(3)<br>
Other	12(16)<br>
Disease stage; n (%)	<br>
1MB	22 (29)<br>
IV	53(71)<br>
Number of prior chemotherapy regimens; n (%)	<br>
0	19(25)<br>
1	36 (48)<br>
&gt;2	20 (27)<br>
Ethnicity; n (%)	<br><br>
Caucasian	51 (68)<br>
Asian	24 (32)<br>
Smoking history; n (%)	<br>
Never	12(16)<br>
Current	24 (32)<br>
Former	39 (52)<br>
Of the 75 patients with qRT-PCR results, 4 (5%) had partial response (PR), 23 (31%) had SD, 39 (52%) had PD, and 9 (12%) were not evaluable. These results were very similar to those observed in the entire study population (n=264).<br>
Figure 3 shows relative mRNA levels for PTP4A1 in individual patients, as assessed by Affymetrix Genechip® profiling and qRT-PCR. Figure 3a shows expression levels versus clinical outcome for Genechip® profiling and Figure 3b shows expression levels to qRT-PCR.<br>
There was a good correlation between Genechip® and qRT-PCR measurements of the PTP4A1 mRNA transcript (Figure 3c; pearson's p=0.6, p
Discussion<br>
By analyzing tissue samples with high-density oligonucleotide microarray technology, and applying statistical modeling to the data, we have been able to identify genes whose expression levels may be predictive of patients deriving a clinical benefit from treatment with erlotinib.<br>
A composite criterion (defined above) was applied. It resulted in PTP4A1 as predictive marker for EGFR inhibitor treatment.<br>
PTP4A1 is the second member of the protein tyrosine phosphatase family. It is located on chromosome 6ql2 and encodes a protein belonging to a small class of prenylated protein tyrosine phosphatases (PTPs), which contains a PTP domain and a characteristic C-terminal prenylation motif. This tyrosine phosphatase is a nuclear protein, but may be primarily associated with the plasma membrane.<br>
Several reports have associated high expression of PTP4A1 with oncogenic activity, specifically enhanced proliferation rate and cell motility and invasiveness. A recent study in non small cell lung cancer has demonstrated a functional role of PTP4A1 in the processes of cell migration and invasion,<br><br><br>
In this study, PTP4A1 was found to be down regulated in patients deriving clinical benefit from treatment with erlotinib. The oncogenic function of this gene is a clear argument in favor of its utility as a tumour marker.<br><br><br><br><br><br>
We Claim :<br>
1.	An in vitro method of predicting the response of a cancer patient to treatment with .<br>
EGFR inhibitor comprising:<br>
determining an expression level of a PTP4A1 gene in a tumour sample of a patient ai comparing the expression level of the PTP4A1 gene to a value representative of ; expression level of the PTP4A1 gene in tumours of a population of patients deriving i clinical benefit from the treatment, wherein a lower expression level of the PTP4A1 gene the tumour sample of the patient is indicative for a patient who will derive clinical bene from the treatment.<br>
2.	The method of claim 1, wherein the expression level is determined by microarr; technology.<br>
3.	The method of claim 1 or 2, wherein the PTP4A1 gene shows between 1.2 and 1.8 more fold lower expression level in the tumour sample of the patient compared to the vali representative of an expression level of the PTP4A1  gene in tumours of a population patients deriving no clinical benefit from the treatment.<br>
4.	The method of claims 1 to 3. wherein the EGFR inhibitor is erlotinib.<br>
5.	The method of claims 1 to 4, wherein the cancer is NSCLC.<br>
6.	Use of a PTP4A1 gene for predicting the response of a cancer patient to EGF inhibitor treatment.<br>
7.	The use of claim 6, wherein the cancer is NSCLC.<br>
8.	The use of claim6 or 7, wherein the EGFR inhibitor is erlotinib.<br>
9.	A method of treating a cancer patient identified by a method of claims 1 to comprising administering an EGFR inhibitor to the patient.<br><br>
10.	The method of claim 9, wherein the EGFR inhibitor is erlotinib.<br>
11.	The method of claim 9 or 10, wherein the cancer is NSCLC. Dated this 8th day of February, 2010<br></td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtYWJzdHJhY3QuZG9j" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-abstract.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQU5ORVhVUkUgQSgyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-ANNEXURE A(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtYXNzaWdubWVudC5wZGY=" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSgyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(AMENDED)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSgyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(AMENDED)-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKEFNRU5ERUQpLSg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(AMENDED)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(MARKED COPY)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oMjgtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(MARKED COPY)-(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ0xBSU1TKE1BUktFRCBDT1BZKS0oNC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CLAIMS(MARKED COPY)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtY2xhaW1zLmRvYw==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-claims.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtQ09SUkVTUE9OREVOQ0UoOC0zLTIwMTApLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-CORRESPONDENCE(8-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtY29ycmVzcG9uZGVuY2UucGRm" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-correspondence.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZGVzY3JpcHRpb24oY29tcGxldGUpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-description(complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZHJhd2luZy5wZGY=" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-drawing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRVAgRE9DVU1FTlQgKDE2LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-EP DOCUMENT (16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRVAgRE9DVU1FTlQoMTYtMTItMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-EP DOCUMENT(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRVAgRE9DVU1FTlQoMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-EP DOCUMENT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAxKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 1(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAxKDQtMTItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 1(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSAxLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAxOC5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAyKFRJVExFIFBBR0UpLSgyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 2(TITLE PAGE)-(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAyKFRJVExFIFBBR0UpLSg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 2(TITLE PAGE)-(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSAyKHRpdGxlIHBhZ2UpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 2(title page).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSAyLmRvYw==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 2.doc</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSAyLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAzKDE2LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 3(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAzKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 3(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtRk9STSAzKDgtMy0yMDEwKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-FORM 3(8-3-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSAzLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSA1LnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form 5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSBwY3QtaWItMzA0LnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form pct-ib-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSBwY3QtaXNhLTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form pct-isa-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSBwY3QtaXNhLTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form pct-isa-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtZm9ybSBwY3Qtcm8tMTAxLnBkZg==" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-form pct-ro-101.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtR0VORVJBTCBQT1dFUiBPRiBBVFRPUk5FWSgyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-GENERAL POWER OF ATTORNEY(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtb3RoZXIgZG9jdW1lbnQucGRm" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-other document.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcoMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-PETITION UNDER RULE 137(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUEVUSVRJT04gVU5ERVIgUlVMRSAxMzcyMS01LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-PETITION UNDER RULE 13721-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtcG93ZXIgb2YgYXR0b3JuZXkucGRm" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-power of attorney.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDE2LTEyLTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-REPLY TO EXAMINATION REPORT(16-12-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUkVQTFkgVE8gRVhBTUlOQVRJT04gUkVQT1JUKDIxLTUtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-REPLY TO EXAMINATION REPORT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUkVQTFkgVE8gSEVBUklORygyOC0xMi0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-REPLY TO HEARING(28-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtUkVQTFkgVE8gSEVBUklORyg0LTEyLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-REPLY TO HEARING(4-12-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtc2VxdWVuY2UgbGlzdGluZy5wZGY=" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-sequence listing.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LU1VTU5QLTIwMTAtVVMgRE9DVU1FTlQoMjEtNS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">248-MUMNP-2010-US DOCUMENT(21-5-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ4LW11bW5wLTIwMTAtd28gaW50ZXJuYXRpb25hbCBwdWJsaWNhdGlvbiByZXBvcnQgYTEucGRm" target="_blank" style="word-wrap:break-word;">248-mumnp-2010-wo international publication report a1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QxLmpwZw==" target="_blank" style="word-wrap:break-word;">abstract1.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="255050-methods-for-repearing-cycloalkylidene-bisphenols.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="255052-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-ptprf-gene-expression-level.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>255051</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>248/MUMNP/2010</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>04/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>25-Jan-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>17-Jan-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>08-Feb-2010</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>F.HOFFMANN-LA ROCHE AG</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>GRENZACHERSTRASSE 124, CH-4070 BASEL, SWITZERLAND.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>DELMAR,PAUL</td>
											<td>HAMMERSTRASSE 89, CH-4057 BASEL, SWITZERLAND.</td>
										</tr>
										<tr>
											<td>2</td>
											<td>KLUGHAMMER,BARBARA</td>
											<td>IM ADLERGARTEN 11, D-79618 RHEINFELDEN, GERMANY.</td>
										</tr>
										<tr>
											<td>3</td>
											<td>LUTZ,VERENA</td>
											<td>LINDENSCHMITSTRASSE 25, D-81371 MUENCHEN, GERMANY.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>MCLOUGHLIN,PATRICIA</td>
											<td>CLARAGRABEN 115, CH-4057 BASEL, SWITZERLAND.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2008/006512</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2008-08-07</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>07114309.3</td>
									<td>2007-08-14</td>
								    <td>EPO</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/255051-an-in-vitro-method-of-predicting-the-response-of-a-non-small-cell-lung-cancer-patient-to-erlotinib-by-determining-ptp4a1-gene-expression-level by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 11:03:22 GMT -->
</html>
