<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/255387-an-isolated-polynucleotide-and-recombinant-vector-encoding-a-polypeptide-having-desaturase-acitivity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:42:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 255387:&quot;AN ISOLATED POLYNUCLEOTIDE AND RECOMBINANT VECTOR ENCODING A POLYPEPTIDE HAVING DESATURASE ACITIVITY&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;AN ISOLATED POLYNUCLEOTIDE AND RECOMBINANT VECTOR ENCODING A POLYPEPTIDE HAVING DESATURASE ACITIVITY&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>The present invention relates to an isolated polynucleotide comprising a nucleic acid sequence that encodes a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 15, wherein the polynucleotide is selected from the group consisting of: (a) a polynucleotide encoding the polypeptide of SEQ ID NO: 3, SEQ ID NO: 5 or SEQ ID NO: 34; (b) a polynucleotide comprising the nucleic acid sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO:4 or SEQ ID NO: 33; (c) a polynucleotide hybridizing to one or more of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 4 or SEQ ID NO: 33, or a complement thereof, under conditions of 5X SSC, 50% formamide and 42°C. The present invention also relates to a recombinant vector comprising the polynucleotide that encodes a polypeptide having desaturase activity.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The present invention relates to an isolated polynucleotide and recombinant vector encoding a polypeptide having desaturase activity.<br>
BACKGROUND OF THE INVENTION<br>
This application claims the priority of U.S. Provisional Patent Application Ser. No, 60/382,391, filed May 22 2002, and U.S. Provisional Patent Application Ser. No.60/453,125, filed March 7, 2003. The entire disclosure of each of the above applications is specifically incorporated herein by reference in the entirety.<br>
1,	Field of the Invention<br>
The invention relates generally to desaturase enzymes that modulate the number and location of double bonds in long chain poly-unsaturated fatty acids (LC-PUFA's), methods of use thereof and compositions derived therefrom. In particular, the invention relates to improved ferry acid profiles using desaturase enzymes and nucleic acids encoding for such enzymes identified in fungi<br>
2.	Description of the Related Art<br>
The primary products of fatty acid biosynthesis in most organisms are 16- and 18-carbon compounds. The relative ratio of chain lengths and degree of unsaturation of these fatty acids vary widely among species. Mammals, for example, produce primarily saturated and monosaturated fatty adds, while most higher plants produce fatty acids with one, two, or three double bonds, the latter two comprising polyunsaturated fatty acids (PUFA's).<br>
Two main families of PUFAs are the omega-3 ferry acids (also represented as "n-3" fatty acids), exemplified by eicosapentaenoic acid (EPA, 20:4, n-3), and the omega-6 fatty acids (also represented as "n-6" fatty acids), exemplified by arachidonic acid (ARA, 20:4, n-6}. PUFAs are important components of the plasma membrane of the cell and adipose tissue, where they may be found in such forms as phospholipids and as triglycerides, respectively. PUFAs are necessary for proper development in mammals, particularly in the developing infant brain, and for tissue formation and repair.<br>
Several disorders respond to treatment with fatty acids. Supplementation with PUFAs has been shown to reduce the rate of restenosis after angioplasty. The health benefits of certain dietary omega-3 fatty acids for cardiovascular disease and rheumatoid arthritis also have been<br>
weth documented (Simopoulos, 1997; James et al, 2000). Further, PUFAs have been suggested for use in treatments for asthma and psoriasis. Evidence indicates that PUFAs may be involved in calcium metabolism, suggesting that PUFAs may be useful in the treatment or prevention of osteoporosis and of kidney or urinary tract stones. The majority of evidence for health benefits applies to the long chain omega-3 fats, EPA and DHA which are in fish and fish oil. With this base of evidence, health authorities and nutritionists in Canada (Scientific Review Committee, 1990, Nutrition Recommendations, Minister of National Health and Welfare, Canada, Ottowa), Europe (de Deckerer et al, 1998), the United Kingdom (The British Nutrition Foundation, 1992, Unsaturated fatty-acids - nutritional and physiological significance: The report of the British Nutrition Foundation's Task Force, Chapman and Hall, London), and the United States (Simopoulos et al, 1999) have recommended increased dietary consumption of these PUFAs.<br>
PUFAs also can be used to treat diabetes (U.S. Pat. No. 4,826,877; Horrobin et al, 1993). Altered fatty acid metabolism and composition has been demonstrated in diabetic animals. These alterations have been suggested to be involved in some of the long-term complications resulting from diabetes, including retinopathy, neuropathy, nephropathy and reproductive system damage. Primrose oil, which contains GLA, has been shown to prevent and reverse diabetic nerve damage.<br>
PUFAs, such as linoleic acid (LA, 18:2, A9, 12) and a-linolenic acid (ALA18:3, A9, 12, 15), are regarded as essential fatty acids in the diet because mammals lack the ability to synthesize these acids. However, when ingested, mammals have the ability to metabolize LA and ALA to form the n-6 and n-3 families of long-chain polyunsaturated fatty acids (LC-PUFA). These LC-PUFA's are important cellular components conferring fluidity to membranes and functioning as precursors of biologically active eicosanoids such as prostaglandins, prostacyclins, and leukotrienes, which regulate normal physiological functions.<br>
In mammals, the formation of LC-PUFA is rate-limited by the step of A6 desaturation, which converts LA to y-linolenic acid (GLA, 18:3, A6, 9,12) and ALA to SDA (18:4, A6, 9,12, 15). Many physiological and pathological conditions have been shown to depress this metabolic step, and consequently, the production of LC-PUFA. However, bypassing the A6-desaturation via dietary supplementation with EPA or DHA can effectively alleviate many pathological diseases associated with low levels of PUFA. However, as set forth in more detail below, currently available sources of PUFA are not desirable for a multitude of reasons. The need for a reliable and economical source of PUFA's has spurred interest in alternative sources of PUFA's.<br>
Major long chain PUFAs of importance include docosahexaenoic acid (DHA, 22:6, n-3) and EPA, which are primarily found in different types of fish oil, and arachidonic acid (ARA, 20:4, n-6), found in filamentous fungi. For DHA, a number of sources exist for commercial production including a variety of marine organisms, oils obtained from cold water marine fish, and egg yolk fractions. Commercial sources of SDA include the genera Trichodesma and Echium. However, there are several disadvantages associated with commercial production of PUFAs from natural sources. Natural sources of PUFAs, such as animals and plants, tend to have highly heterogeneous oil compositions. For example, oil from the seeds of Echum, in addition to SDA, contain almost equivalent levels of the omega-6 fatty acid GLA. The oils obtained from these sources therefore can require extensive purification to separate out one or more desired PUFAs or to produce an oil which is enriched in one or more PUFA.<br>
Natural sources of PUFAs also are subject to uncontrollable fluctuations in availability. Fish stocks may undergo natural variation or may be depleted by overfishing. In addition, even with overwhelming evidence of their therapeutic benefits, dietary recommendations regarding omega-3 fatty acids are not heeded. Fish oils have unpleasant tastes and odors, which may be impossible to economically separate from the desired product, and can render such products unacceptable as food supplements. Ammal oils, and particularly fish oils, can accumulate environmental pollutants. Foods may be enriched with fish oils, but again, such enrichment is problematic because of cost and declining fish stocks worldwide. This problem is an impediment to consumption and intake of whole fish. Nonetheless, if the health messages to increase fish intake were embraced by communities, there would likely be a problem in meeting demand for fish. Furthermore, there are problems with sustainability of this industry which relies heavily on wild fish stocks for aquaculture feed (Naylor et al, 2000).<br>
Other natural limitations favor a novel approach for the production of omega-3 fatty acids. Weather and disease can cause fluctuation in yields from both fish and plant sources. Cropland available for production of alternate oil-producing crops is subject to competition from the steady expansion of human populations and the associated increased need for food production on the remaining arable land. Crops that do produce PUFAs, such as borage, have not been adapted to commercial growth and may not perform well in monoculture. Growth of such crops is thus not economically competitive where more profitable and better-established crops can be grown. Large scale fermentation of organisms such as Mortierella is also expensive.  Natural animal tissues contain low amounts of ARA and are difficult to process.<br>
Misroorganisms such as Porphyridium and Mortierella are difficult to cultivate on a commercial scale.<br>
A number of enzymes are involved in PUFA biosynthesis. LA, (18:2, A9, 12) is produced from oleic acid (OA, 18:1, A9) by a Δ12-desaturase while ALA (18:3) is produced from LA by a Δ15-desaturase. SDA (18:4, A6, 9, 12, 15) and GLA (18:3, A6, 9, 12) are produced from LA and ALA by a A6-desaturase. However, as stated above, mammals cannot desaturate beyond the A9 position and therefore cannot convert oleic acid into LA. Likewise, ALA cannot be synthesized by mammals. Other eukaryotes, including fungi and plants, have enzymes which desaturate at the carbon 12 and carbon 15 position. The major poly-unsaturated fatty acids of animals therefore are derived from diet via the subsequent desaturation and elongation of dietary LA and ALA.<br>
U.S. Pat. No. 5,952,544 describes nucleic acid fragments isolated and cloned from Brassica napus that encode fatty acid desaturase enzymes. Expression of the nucleic acid fragments of the '544 patent are expressed in plants and result in accumulation of ALA. However, in transgenic plants expressing the plant Δ15-desaturase, substantial LA remains unconverted by the desaturase. A more active enzyme that converts more LA to ALA would be advantageous. Increased conversion from LA to ALA would create greater amounts of ALA. Increased ALA levels allow the A6-desaturase, when co-expressed with nucleic acid encoding for the Δ15-desaturase, to act upon the ALA, thereby producing greater levels of SDA. Because of the multitude of beneficial uses for SDA, there is a need to create a substantial increase in the yield of SDA. Nucleic acids from various sources have been sought to increase SDA yield. However, innovations that would allow for improved commercial production in land-based crops are still highly desired. (See, e.g., Reed et al, 2000). Furthermore, the use of desaturase polynucleotides derived from Caenorhabditis elegans (Meesapyodsuk et al, 2000) is not ideal for the commercial production of enriched plant seed oils.<br>
Nucleic acids encoding Δ15-desaturases have been isolated from several species of cyanobacteria and plants, including Arabidopsis, soybean, and parsley. The deduced arnino acid sequences of these desaturases demonstrate a high degree of similarity, most notable in the<br>
t<br>
region of three histidine-rich motifs that, without being bound by any one theory, are believed to be involved in iron-binding. However, no Δ15-desaturase has been isolated from any fungal species. Furthermore, even with the genomes of several fungal species having been sequenced, and using sophisticated algorithms, searches utilizing known Δ15-desaturase cDNA and amino<br>
acid saliences against Aspergillus and Neurospora DNA databases have not yielded Δ15-desaturases.<br>
Therefore, it would he advantageous to obtain genetic material involved in PUFA biosynthesis and to express the isolated material in a plant system, in particular, a land-based terrestrial crop plant system, which can be manipulated to provide production of commercial quantities of one or more PUFA's. There is also a need to increase omega-3 fat intake in humans and animals. Thus there is a need to provide a wide range of omega-3 enriched foods and food supplements so that subjects can choose feed, feed ingredients, food and food ingredients which suit their usual dietary habits. Currently there is only one omega-3 fatty acid, ALA, available in vegetable oils. However, there is poor conversion of ingested ALA to the longer-chain omega-3 fatty acids such as EPA and DHA. It has been demonstrated in copending U.S. Application Ser. No. 10/384,369 for "Treatment And Prevention Of Inflammatory Disorders," that elevating ALA intake from the community average of 1/g day to 14 g/day by use of flaxseed oil, only modestly increased plasma phospholipid EPA levels. A 14-fold increase in ALA intake resulted in a 2-fold increase in plasma phospholipid EPA (Manzioris et al, 1994).<br>
Thus, to that end, there is a need for efficient and commercially viable production of PUFAs using fatty acid desaturases, genes encoding them, and recombinant methods of producing them. A need also exists for oils containing higher relative proportions of and/or enriched in specific PUFA's and food compositions and supplements containing them. A need also exists for reliable economical methods of producing specific PUFA's.<br>
Despite inefficiencies and low yields as described above, the production of omega-3 fatty acids via the terrestrial food chain is an enterprise beneficial to public health and, in particular, the production of SDA. SDA in particular is important because, as described above, there is low conversion of ALA to EPA. This is because in this three enzyme process (requiring A6, A12, and A15) the initial enzyme, A6-desaturase, has low activity in humans and is rate-limiting. Evidence that A6-desaturase is rate-limiting is provided by studies which demonstrate that the conversion of its substrate, ALA, is less efficient than the conversion of its product, SDA to EPA in mice and rats (Yamazaki et al, 1992; Huang, 1991).<br>
Based on such studies, it is seen that in commercial oilseed crops, such as canola, soybean, com, sunflower, safflower, or flax, the conversion of some fraction of the mono and polyunsaturated fatty acids that typify their seed oil to SDA, requires the seed-specific expression of multiple desaturase enzymes, including A6- and A12, and an enzyme that has Δ15-<br>
desaturase activity.  Oils derived from plants expressing elevated levels of A6, A12, and Δ15-<br>
desahcrases are rich in SDA and other omega-3 fatty acids. Such oils can be utilized to produce foods and food supplements enriched in omega-3 fatty acids and consumption of such foods effectively increases tissue levels of EPA and DHA. Foods and food stuffs, such as milk, margarine and sausages, all made or prepared with omega-3 enriched oils will result in therapeutic benefits. It has been shown that subjects can have an omega-3 intake comparable to EPA and DHA of at least 1.8 g/day without altering their dietary habits by utilizing foods containing oils enriched with omega-3 fatty acids (Naylor, supra.). Thus, there exists a strong need for novel nucleic acids of Δ15-desaturases for use in transgenic crop plants to produce oils enriched in PUFAs. New plant seed oils enriched for PUFAs and, particular, omega-3 fatty acids such as stearidonic acid are similarly needed.<br>
SUMMARY OF THE INVENTION<br>
In one aspect, the invention provides isolated nucleic acids encoding a polypeptide capable of desaturating a fatty acid molecule at carbon 15 (Δ15-desaturase). These may be used to transform cells or modify the fatty acid composition of a plant or the oil produced by a plant. One embodiment of the invention is an isolated polynucleotide sequence isolated from a fungal species having unique desaturase activity. The isolated polynucleotides may be isolated from fungal species preferably belonging to a phyla selected from the group consisting of zygomycota, basidiomycota, and ascomycota. In certain embodiments, the isolated polynucleotides are isolated from a fungal species selected from the group consisting of Neiirospora crassa, Aspergillus nidulans,, snABotrytis cinerea.<br>
In another aspect, the invention provides an isolated polynucleotide comprising a<br>
sequence selected from the group consisting of: (a) a polynucleotide encoding the polypeptide of<br>
SEQ ID NO:3, SEQ ID NO:5 or SEQ ID NO:34; (b) a polynucleotide comprising the nucleic<br>
acid sequence of SEQ TD NO:l, SEQ ID NO:2, SEQ ID NO:4 or SEQ ID NO: 33; (c) a<br>
polynucleotide hybridizing to one or more of SEQ ID NO:l, SEQ ID NO:2, SEQ ID NO:4 or<br>
SEQ ID NO: 33, or a complement thereof, under conditions of 5X SSC, 50% formamide and<br>
42°C; and (d) a fungal polynucleotide encoding a polypeptide having at least one of the amino<br>
acid motifs:<br>
(Sequence Removed) <br>
Irryet another aspect, the invention provides a recombinant vector comprising an isolated polynucleotide in accordance with the invention. The term "recombinant vector" as used herein, includes any recombinant segment of DNA which one desires to introduce into a host cell, tissue and/or organism, and specifically includes expression cassettes isolated from a starting polynucleotide. A recombinant vector may be linear or circular. In various aspects, a recombinant vector may comprise at least one additional sequence chosen from the group consisting of: regulatory sequences operatively coupled to the polynucleotide; selection markers operatively coupled to the polynucleotide; marker sequences operatively coupled to the polynucleotide; a purification moiety operatively coupled to the polynucleotide; and a targeting sequence operatively coupled to the polynucleotide.<br>
In still yet another aspect, the invention provides cells, such as mammal, plant, insect, yeast and bacteria cells transformed with the polynucleotides of the instant invention. In a further embodiment, the cells are transformed with recombinant vectors containing constitutive and tissue-specific promoters in addition to the polynucleotides of the instant invention. In certain embodiments of the invention, such cells may be further defined as transformed with a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 6.<br>
In still yet another aspect, the invention provides a polypeptide, including fragments and<br>
proteins having desaturase activity that desaturates a fatty acid molecule at carbon 15. In one<br>
embodiment of the invention, the polypeptide comprises at least one of the amino acid motifs:<br>
(Sequence Removed) <br>
	(SEQ	ID	N0:13)	or<br>
HisValVafflisHisLeuPheXaaArgneProPheTyr (HWHHLFXRIPFY) (SEQ ID NO: 14 or SEQ ID NO:22). In further embodiments the polypeptide is further defined as comprising all of said amino acid motifs. The invention also provides a fungal polypeptide comprising the amino acid sequence of SEQ ID NO:3, SEQ ID NO:5 or SEQ ID NO:34; or a fragment thereof having desaturase activity that desaturates a fatty acid molecule at carbon 15.<br>
Still yet another aspect of the invention provides a method of producing seed oil containing omega-3 fatty acids from plant seeds, comprising the steps of (a) obtaining seeds of a plant according to the invention; and (b) extracting the oil from said seeds. Examples of such a plant seed include canola, soy, soybeans, rapeseed, sunflower, cotton, cocoa, peanut, saffiower, coconut, flax, oil palm, oilseed Brassica napus, and corn. Preferred methods of transforming such plant cells include the use of Ti and Ri plasmids of Agrobacterium, electroporation, and high-velocity ballistic bombardment.<br>
In still yet another aspect, a method is provided of producing a plant comprising seed oil containing altered levels of omega-3 fatty acids comprising introducing a recombinant vector of the invention into an oil-producing plant. In the method, introducing the recombinant vector may comprise plant breeding and may comprise the steps of: (a) transforming a plant cell with the recombinant vector; and (b) regenerating said plant from the plant cell, wherein the plant has altered levels of omega-3 fatty acids. In the method, the plant may, for example, be selected from the group consisting of Arabidopsis thaliana, oilseed Brassica, rapeseed, sunflower, saffiower, canola, corn, soybean, cotton, flax, jojoba, Chinese tallow tree, tobacco, cocoa, peanut, fruit plants, citrus plants, and plants producing nuts and berries. The plant may be further defined as transformed with a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 6 and the plant may have SDA increased. The method may also further comprise introducing the recombinant vector into a plurality of oil-producing plants and screening the plants or progeny thereof having inherited the recombinant vector for a plant having a desired profile of omega-3 fatty acids.<br>
In still yet another aspect, the invention provides an endogenous canola seed oil having a SDA content of from about 8% to about 27% and an oleic acid content of from about 40% to about 70%. In certain embodiments, the canola seed oil may be further defined as comprising less than 10% combined ALA acid, LA and GLA. The oil may also comprise a SDA content further defined as from about 10% to about 20%, including from about 12% to about 20%, about 15% to about 20%, about 10% to aoout 17% and about 12% to about 17%.    In further<br>
embcxi saents of the invention, the canola seed oil may have an oleic acid content further defined as from about 45% to about 65%, including from about 50% to about 65%, from about 50% to about 60% and from about 55% to about 65%. In still further embodiments of the invention, the SDA content is further defined as from about 12% to about 17% and the oleic acid content is further defined as from about 55% to about 65%. In one embodiment of the invention, a canola seed oil is from Brassica napus or Brassica rapa seed. Jn certain embodiments, an oil provided has a ratio of omega-6 to omega-3 fatty acids of from about 1:1 to about 1:4, including from about 1:2 to about 1:4.<br>
In still yet another aspect, the invention provides a method of increasing the nutritional value of an edible product for human or animal consumption, comprising adding a canola seed oil provided by the invention to the edible product, hi certain embodiments, the product is human and /or animal food. The edible product may also be animal feed and/or a food supplement. In the method, the canola seed oil may increase the SDA content of the edible product and/or may decrease the ratio of omega-6 to omega-3 fatty acids of the edible product. The edible product may lack SDA prior to adding the canola seed oil.<br>
In still yet another aspect, the invention provides a method of manufacturing food or feed, comprising adding a canola seed oil provided by the invention to starting food or feed ingredients to produce the food or feed. In certain embodiments, the method is further defined as a method of manufacturing food and/or feed. The invention also provides food or feed made by the method.<br>
In still yet another aspect, the invention comprises a method of providing SDA to a human or animal, comprising administering the canola seed oil of claim 1 to said human or animal, hi the method, the canola seed oil may be administered in an edible composition, including food or feed. Examples of food include beverages, infused foods, sauces, condiments, salad dressings, fruit juices, syrups, desserts, icings and fillings, soft frozen products, confections or intermediate food. The edible composition may be substantially a liquid or solid. The edible composition may also be a food supplement and/or nutraceutical. In the method, the canola seed oil may be administered to a human and/or an animal. Examples of animals the oil may be administered to include livestock or poultry.<br>
BRIEF DESCRIPTION OF THE FIGURES<br>
The following drawings form part of the present specification and are included to further demonstrate certain aspects of the present invention. The invention may be better understood by reference to one or more of these drawings in combination with the detailed description of specific embodiments presented herein. The invention can be more fully understood from the following description of the figures:<br>
FIG. 1 shows the fungal Δ15-desaturase NcD15D coding region in a pCR2.1 cassette (pMON67004).<br>
FIG. 2 shows the fungal Δ15-desaturase NcD15D coding region in the yeast expression vector pYES 2.1 (pMON77208).<br>
FIG. 3 shows the ALA levels in 200 half-seeds (seeds cut in half), ordered from lowest to highest ALA.<br>
FIG. 4 shows a flow chart or plasmids maps resulting in plasmids pMON77214 and pMON77217.<br>
FIG. 5 shows an exemplary dendrogram of desaturase polypeptides, including N. crassa Δ15-desaturase.<br>
FIG. 6 shows a sequence alignment of exemplary desaturase polypeptides relative to N. crassa Δ15-desaturase.<br>
FIG. 7A-7H shows plasmid maps of constructs prepared.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
The invention overcomes the limitations of the prior art by providing methods and compositions for creation of plants with improved PUFA content. The modification of fatty acid content of an organism such as a plant presents many advantages, including improved nutrition and health benefits. Modification of fatty acid content can be used to achieve beneficial levels or profiles of desired PUFA's in plants, plant parts, and plant products, including plant seed oils. For example, when the desired PUFA's are produced in the seed tissue of a plant, the oil may be isolated from the seeds typically resulting in an oil high in desired PUFAs or an oil having a desired fatty acid content or profile, which may in turn be used to provide beneficial characteristics in food stuffs and other products. The invention in particular provides endogenous canola oil having SDA while also containing a beneficial oleic acid content.<br>
Various aspects of the invention include methods and compositions for modification of PUFA content of a cell, for example, modification of the PUFA content of a plant cell(s). Compositions related to the invention include novel isolated polynucleotide sequences, polynucleotide constructs and plants and/or plant parts transformed by polynucleotides of the invention. The isolated polynucleotide may encode fungal fatty acid desaturases and, in particular, may encode a fungal Δ15-desaturase. Host cells may be manipulated to express a polynucleotide encoding a desaturase polypeptide(s) which catalyze desaturation of a fatty acid(s).<br>
Some aspects of the invention include various desaturase polypeptides and polynucleotides encoding the same. Various embodiments of the invention may use a combinations of desaturase polynucleotides and the encoded polypeptides that typically depend upon the host cell, the availability of substrate(s), and the desired end produces). "Desaturase" refers to a polypeptide that can desaturate or catalyze formation of a double bond between consecutive carbons of one or more fatty acids to produce a mono- or poly-unsaturated fatty acid or precursor thereof. Of particular interest are polypeptides which can catalyze the conversion of stearic acid to oleic acid, oleic acid to LA, LA to ALA, or ALA to SDA, which includes enzymes which desaturate at the 12, 15, or 6 positions. The term "polypeptide" refers to any chain of amino acids, regardless of length or post-translational modification (e.g., glycosylation or phosphorylation). Considerations for choosing a specific polypeptide having desaturase activity include, but are not limited to the pH optimum of the polypeptide, whether the polypeptide is a rate limiting enzyme or a component thereof, whether the desaturase used is essential for synthesis of a desired PUFA, and/or a co-factor is required by the polypeptide. The expressed polypeptide preferably has characteristics that are compatible with the biochemical environment of its location in the host cell. For example, the polypeptide may have to compete for substrate(s).<br>
Analyses of the Km and specific activity of a polypeptide in question may be considered in determining the suitability of a given polypeptide for modifying PUFA(s) production, level, or profile in a given host cell. The polypeptide used in a particular situation is one which typically can function under the conditions present in the intended host cell, but otherwise may be any desaturase polypeptide having a desired characteristic or being capable of modifying the relative production, level or profile of a desired PUFA(s) or any other desired characteristics as discussed herein. The substrate(s) for the expressed enzyme may be produced by the host cell or may be<br>
exogmously supplied.   To achieve expression, the polypeptide(s) of the instant invention are encoded by polynucleotides as described below.<br>
The inventors have isolated and produced enzymes of fungal origin which exhibit Δ15-desaturase activity. Fungal sources include, but are not limited to the genus Aspergillus, e.g., Aspergillus nidulans; the genus Botrytis, e.g., Botrytis cinerea; the genus Neurospora, e.g., Neurospora crassa; and other fungi that exhibit Δ15-desaturase activity.<br>
Of particular interest are Neurospora crassa and/or Aspergillus nidulans Δ15-desaturase(s). The amino acid sequence of the N. crassa Δ15-desaturase, set forth in SEQ ID NO:3 and encoded by the nucleotide sequence in SEQ ID NO:l and SEQ ID NO:2, was determined to have a molecular weight of approximately 49,123.37 Daltons. The sequence consists of 429 arnino acids; 32 of which are strongly basic (lysine, arginine); 35 of which are strongly acidic (aspartic acid, glutamic acid); 170 hydrophobic amino acids (alanine, isoleucine, leucine, phenylalanine, tryptophan, valine); and 100 polar amino acids (asparagine, cysteine, glutamine, serine, threonine, tyrosine). SEQ ID NO:3 has an isoelectric point of 7.187; a charge of 1.634 at pH 7.0; a Davis, Botsein, Roth Melting Temperature of 89.65°C and a Wallace Temperature of 5098.00.<br>
The arnino acid sequence of the A. nidulans Δ15-desaturase, set forth in SEQ ID NO:5 and encoded by the nucleic acid sequence set forth in SEQ ID NO:4, was determined to have a molecular weight of approximately 46,300 Daltons. The sequence consists of 401 amino acids; of which 31 are strongly basic (lysine, arginine); 34 are strongly basic (aspartic acid, glutamic acid); 161 hydrophobic amino acids (alanine, isoleucine, leucine, phenylalanine, tryptophan, valine); and 100 polar arnino acids (asparagine, cysteine, glutamine, serine, threonine, tyrosine). SEQ ID NO:5 has an isoelectric point of 6.83.<br>
The sequences encoding the Neurospora crassa and/or the Aspergillus nidulans Δ15-desaturase may b&amp; expressed in transgenic plants, microorganisms or animals to effect greater synthesis of ALA from LA, as well as SDA. Other polynucleotides which are substantially identical to the N. crassa and/or the A. nidulans Δ15-desaturase polynucleotide, or which encode polypeptides which are substantially identical to the N crassa and/or the A. nidulans Δ15-desaturase polypeptide, also can be used. "Substantially identical" refers to an amino acid sequence or nucleic acid sequence exhibiting in order of increasing preference at least 80%, 90% or 95% identity to the TV. crassa and/or the A. nidulans Δ15-desaturase amino acid sequence or nucleic acid sequence encoding the amino acid sequence.    Polypeptide or polynucleotide<br>
comparisons may be carried out using sequence analysis software, for example, the Sequence Analysis software package of theGCG Wisconsin Package (Accelrys, San Diego, CA), MEGAlign (DNAStar, Inc., 1228 S. Park St., Madison, Wis. 53715), and Mac Vector (Oxford Molecular Group, 2105 S. Bascom Avenue, Suite 200, Campbell, Calif. 95008). Such software matches similar sequences by assigning degrees of similarity or identity.<br>
Encompassed by the present invention are related desaturases from the same or other related organisms. Such related desaturases include variants of the disclosed Δ15-desaturases naturally occurring within the same or different species of fungus. Related desaturases can be identified by their ability to function substantially the same as the disclosed desaturases; that is, are still able to effectively convert LA to ALA and GLA to SDA. Related desaturases also can be identified by screening sequence databases for sequences homologous to the disclosed desaturases, by hybridization of a probe based on the disclosed desaturases to a library constructed from the source organism, or by RT-PCR using rnRNA from the source organism and primers based on the disclosed desaturases.<br>
Certain aspects of the invention include variants and fragments of a fungal Δ15-desaturase polypeptide and the nucleic acids encoding such that retain desaturase activity. In another aspect of the invention, a vector containing a nucleic acid, or fragment thereof, containing a promoter, a Δ15-desaturase coding sequence and a termination region may transferred into an organism in which the promoter and termination regions are functional. Accordingly, organisms producing recombinant Δ15-desaturase are provided by this invention. Yet another aspect of this invention provides isolated Δ15-desaturase, which can be purified from the recombinant organisms by standard methods of protein purification. (For example, see AusubeletaL, 1987).<br>
Various aspects of the invention include nucleic acid sequences that encode desaturases, described herein. Nucleic acids may be isolated from fungi including, but not limited to Neurospora crassa, Aspergillus nidulans, Botrytis cinerea and the like. The genomes of these fungi have all been sequenced and it has been determined that each is rich in ALA. A cloning strategy based on oligonucleotide primers designed to amplify sequences identified as potential fatty acid desaturases, based on BLAST searches of the N. a-assa genomic DNA database, may be used to sequence individual clones. These clones may then be functionally characterized.<br>
Nucleic acid constructs may be provided that integrate into the genome of a host cell or are autonomously replicated (e.g., episomally replicated) in the host cell.   For production of<br>
All and/or SDA, the expression cassettes, {i.e., a polynucleotide encoding a protein that is operatively linked to nucleic acid sequence(s) that directs the expression of the polynucleotide) generally used include an expression cassette which provides for expression of a polynucleotide encoding a Δ15-desaturase. In certain embodiments a host cell may have wild type oleic acid content.<br>
Methods and compositions for the construction of expression vectors, when taken in light of the teachings provided herein, for expression of fungal desaturase enzymes will be apparent to one of ordinary skill in the art. Expression vectors, as described herein, are DNA or RNA molecules engineered for controlled expression of a desired polynucleotide, e.g., the Δ15-desaturase encoding polynucleotide. Examples of vectors include plasmids, bacteriophages, cosmids or viruses. Shuttle vectors, e.g. (Wolk et al. 1984; Bustos et ah, 1991) are also contemplated in accordance with the present invention. Reviews of vectors and methods of preparing and using them can be found in Sambrook et al. (1989); Goeddel (1990); and Perbal (1988). Sequence elements capable of effecting expression of a polynucleotide include promoters, enhancer elements, upstream activating sequences, transcription termination signals and polyadenylation sites.<br>
Polynucleotides encoding desaturases may be placed under transcriptional control of a strong promoter. In some cases this leads to an increase in the amount of desaturase enzyme expressed and concomitantly an increase in the fatty acid produced as a result of the reaction catalyzed by the enzyme. There is a wide variety of plant promoter sequences which may be used to drive tissue-specific expression of polynucleotides encoding desaturases in transgenic plants. For instance, the napin promoter and the acyl carrier protein promoters have previously been used in the modification of seed oil composition by expression of an antisense form of a desaturase (Knutzon et al. 1999). Similarly, the promoter for the p-subunit of soybean p-conglycinin has been shown to be highly active and to result in tissue-specific expression in transgenic plants of species other than soybean (Bray et al, 1987). Arondel et al. (1992) increased the amount of linolenic acid (18:3) in tissues of transgenic Arabidopsis plants by placing the endoplasmic reticulum-localized fad3 gene under transcriptional control of the strong constitutive cauliflower mosaic virus 35S promoter.<br>
The ordinarily skilled artisan can determine vectors and regulatory elements (including operably linked promoters and coding regions) suitable for expression in a particular host cell. "Operably linked" in this context means that the promoter and terminator sequences effectively function to regulate transcription. As a further example, a vector appropriate for expression of<br>
Δ15-(J' aturase in transgenic plants can comprise a seed-specific promoter sequence derived from helianthinin, napin, or glycinin operably linked to the Δ15-desaturase coding region and further operably linked to a seed storage protein termination signal or the nopaline synthase termination signal. As a still further example, a vector for use in expression of Al 5-desaturase in plants can comprise a constitutive promoter or a tissue specific promoter operably linked to the Δ15-desaturase coding region and further operably linked to a constitutive or tissue specific terminator or the nopaline synthase termination signal.<br>
Modifications of the nucleotide sequences or regulatory elements disclosed herein which maintain the functions contemplated herein are within the scope of this invention. Such modifications include insertions, substitutions and deletions, and specifically substitutions which reflect the degeneracy of the genetic code.<br>
Standard techniques for the construction of such recombinant vectors are well-known to those of ordinary skill in the art and can be found in references such as Sambrook et al. (1989), or any of the myriad of laboratory manuals on recombinant DNA technology that are widely available. A variety of strategies are available for ligating fragments of DNA, the choice of which depends on the nature of the termini of the DNA fragments. It is further contemplated in accordance with the present invention to include in a nucleic acid vector other nucleotide sequence elements which facilitate cloning, expression or processing, for example sequences encoding signal peptides, a sequence encoding KDEL, which is required for retention of proteins in the endoplasmic reticulum or sequences encoding transit peptides which direct Δ15-desaturase to the chloroplast. Such sequences are known to one of ordinary skill in the art. An optimized transit peptide is described, for example, by Van den Broeck et al. (1985). Prokaryotic and eukaryotic signal sequences are disclosed, for example, by Michaelis et al. (1982).<br>
In certain embodiments, the expression cassettes may include a cassette which provides for A6- and/or Al 5-desaturase activity, particularly in a host cell which produces or can take up LA or ALA, respectively. Production of omega-6 type unsaturated fatty acids, such as LA, is favored in a host organism which is incapable of producing ALA. The host ALA production can be removed, reduced and/or inhibited by inhibiting the activity of a Δ15-desaturase. This can be accomplished by standard selection, providing an expression cassette for an antisense Δ15-desaturase, by disrupting a target Δ15-desaturase gene through insertion, deletion, substitution of part or all of the target gene, or by adding an inhibitor of Δ15-desaturase. Similarly, production of LA or ALA is favored in a microorganism or animal having A6-desaturase activity by<br>
proving an expression cassette for an antisense A6 transcript, by disrupting a A6-desaturase gene, or by use of a A6-desaturase inhibitor.<br>
Polynucleotides encoding desired desaturases can be identified in a variety of ways. As an example, a source of the desired desaturase, for example genomic or cDNA libraries from Neurospora, is screened with detectable enzymaticalry- or chemically-synthesized probes, which can be made from DNA, RNA, or non-naturally occurring nucleotides, or mixtures thereof. Probes may be enzymatically synthesized from polynucleotides of known desaturases for normal or reduced-stringency hybridization methods. Oligonucleotide probes also can be used to screen sources and can be based on sequences of known desaturases, including sequences conserved among known desaturases, or on peptide sequences obtained from the desired purified protein. Oligonucleotide probes based on amino aci^ sequences can be degenerate to encompass the degeneracy of the genetic code, or can be biased in favor of the preferred codons of the source organism. Oligonucleotides also can be used as primers for PCR from reverse transcribed mRNA from a known or suspected source; the PCR product can be the full length cDNA or can be used to generate a probe to obtain the desired full length cDNA. Alternatively, a desired protein can be entirely sequenced and total synthesis of a DNA encoding that polypeptide performed.<br>
Once the desired genomic or cDNA has been isolated, it can be sequenced by known methods. It is recognized in the art that such methods are subject to errors, such that multiple sequencing of the same region is routine and is still expected to lead to measurable rates of mistakes in the resulting deduced sequence, particularly in regions having repeated domains, extensive secondary structure, or unusual base compositions, such as regions with high GC base content. When discrepancies arise, resequencing can be done and can employ special methods. Special methods can include altering sequencing conditions by using: different temperatures; different enzymes; proteins which alter the ability of oligonucleotides to form higher order structures; altered nucleotides such as ITP or methylated dGTP; different gel compositions, for example adding formamide; different primers or primers located at different distances from the problem region; or different templates such as single stranded DNAs. Sequencing of mRNA also can be employed.<br>
Some or all of the coding sequence for a polypeptide having desaturase activity may be from a natural source. In some situations, however, it is desirable to modify all or a portion of the codons, for example, to enhance expression, by employing host preferred codons.   Host preferred codons can be determined from the codons of highest frequency in the proteins expressed in the largest amount in a particular host species of interest.    Thus, the coding<br>
seqt|_jice for a polypeptide having desaturase activity can be synthesized in whole or in part. All or portions of the DNA also can be synthesized to remove any destabilizing sequences or regions of secondary structure which would be present in the transcribed rnRNA. All or portions of the DNA also can be synthesized to alter the base composition to one more preferable in the desired host cell. Methods for synthesizing sequences and bringing sequences together are well established in the literature. In vitro mutagenesis and selection, site-directed mutagenesis, or other means can be employed to obtain mutations of naturally occurring desaturase genes to produce a polypeptide having desaturase activity in vivo with more desirable physical and kinetic parameters for function in the host cell, such as a longer half-life or a higher rate of production of a desired polyunsaturated fatty acid.<br>
Once the polynucleotide encoding a desaturase polypeptide has been obtained, it is placed in a vector capable of replication in a host cell, or is propagated in vitro by means of techniques such as PCR or long PCR. Replicating vectors can include plasmids, phage, viruses, cosmids and the like. Desirable vectors include those useful for mutagenesis of the gene of interest or for expression of the gene of interest in host cells. The technique of long PCR has made in vitro propagation of large constructs possible, so that modifications to the gene of interest, such as mutagenesis or addition of expression signals, and propagation of the resulting constructs can occur entirely in vitro without the use of a replicating vector or a host cell.<br>
For expression of a desaturase polypeptide, functional transcriptional and translational initiation and termination regions are operably linked to the polynucleotide encoding the desaturase polypeptide. Expression of the polypeptide coding region can take place in vitro or in a host cell. Transcriptional and translational initiation and termination regions are derived from a variety of nonexclusive sources, including the polynucleotide to be expressed, genes known or suspected to be capable of expression in the desired system, expression vectors, chemical synthesis, or from an endogenous locus in a host cell.<br>
Expression in a host cell can be accomplished in a transient or stable fashion. Transient expression can occur from introduced constructs which contain expression signals functional in the host cell, but which constructs do not replicate and rarely integrate in the host cell, or where the host cell is not proliferating. Transient expression also can be accomplished by inducing the activity of a regulatable promoter operably linked to the gene of interest, although such inducible systems frequently exhibit a low basal level of expression. Stable expression can be achieved by introduction of a construct that can integrate into the host genome or that autonomously replicates in the host cell.  Stable expression of the gene of interest can be selected for through<br>
the of a selectable marker located on or transfected with the expression construct, followed by selection for cells expressing the marker. When stable expression results from integration, integration of constructs can occur randomly within the host genome or can be targeted through the use of constructs containing regions of homology with the host genome sufficient to target recombination with the host locus. Where constructs are targeted to an endogenous locus, all or some of the transcriptional and translational regulatory regions can be provided by the endogenous locus.<br>
When increased expression of the desaturase polypeptide in the source organism is desired, several methods can be employed. Additional genes encoding the desaturase polypeptide can be introduced into the host organism. Expression from the native desaturase locus also can be increased through homologous recombination, for example by inserting a stronger promoter into the host genome to cause increased expression, by removing destabilizing sequences from either the mRNA or the encoded protein by deleting that information from the host genome, or by adding stabilizing sequences to the mRNA (U.S. Pat. No. 4,910,141).<br>
It is contemplated that more than one polynucleotide encoding a desaturase or a polynucleotide encoding more than one desaturase may be introduced and propagated in a host cell through the use of episomal or integrated expression vectors. Where two or more genes are expressed from separate replicating vectors, it is desirable that each vector has a different means of replication. Each introduced construct, whether integrated or not, should have a different means of selection and should lack homology to the other constructs to maintain stable expression and prevent reassortment of elements among constructs. Judicious choices of regulatory regions, selection means and method of propagation of the introduced construct can be experimentally determined so that all introduced polynucleotides are expressed at the necessary levels to provide for synthesis of the desired products.<br>
When necessary for transformation, the Δ15-desaturase coding sequences of the present invention can be inserted into a plant transformation vector, e.g. the binary vector described by Bevan (1984). Plant transformation vectors can be derived by modifying the natural gene transfer system of Agrobacterium tumefaciens. The natural system comprises large Ti (tumor-inducing)-plasrnids containing a large segment, known as T-DNA, which is transferred to transformed plants. Another segment of the Ti plasmid, the vir region, is responsible for T-DNA transfer. The T-DNA region is bordered by terminal repeats. In the modified binary vectors the tumor-inducing genes have been deleted and the functions of the vir region are utilized to transfer foreign DNA bordered by the T-DNA border sequences.  The T-region also contains a<br>
selectable marker for antibiotic resistance, and a multiple cloning site for inserting sequences for transfer. Such engineered strains are known as "disarmed" A. tumefaciens strains, and allow the efficient transformation of sequences bordered by the T-region into the nuclear genomes of plants.<br>
The subject invention finds many applications. Probes based on the polynucleotides of<br>
the present invention may find use in methods for isolating related molecules or in methods to<br>
detect organisms expressing desaturases.    When used as probes, the polynucleotides or<br>
oligonucleotides must be detectable. This is usually accomplished by attaching a label either at<br>
an internal site, for example via incorporation of a modified residue, or at the 5' or 3' terminus.<br>
Such labels can be directly detectable, can bind to a secondary molecule that is detectably<br>
labeled, or can bind to an unlabelled secondary molecule and a detectably labeled tertiary<br>
molecule; this process can be extended as long as is practical to achieve a satisfactorily<br>
detectable signal without unacceptable levels of background signal.   Secondary, tertiary, or<br>
bridging systems can include use of antibodies directed against any other molecule, including<br>
labels or other antibodies, or can involve any molecules which bind to each other, for example a<br>
biotin-streptavidin/avidin system.    Detectable labels typically include radioactive isotopes,<br>
molecules which chemically or enzymatically produce or alter light, enzymes which produce<br>
detectable reaction products, magnetic molecules, fluorescent molecules or molecules whose<br>
fluorescence or hght-emitting characteristics change upon binding.    Examples of labeling<br>
methods can be found in U.S. Pat. No. 5,011,770. Alternatively, the binding of target molecules<br>
can be directly detected by measuring the change in heat of solution on binding of probe to target<br>
via isothermal titration calorimetry, or by coating the probe or target on a surface and detecting<br>
the change in scattering of light from the surface produced by binding of target or probe,<br>
respectively, as may be done with the BIAcore system.<br>
Constructs comprising the gene of interest may be introduced into a host cell by standard techniques. For convenience, a host cell which has been manipulated by any method to take up a DNA sequence or construct will be referred to as "transformed" or "recombinant" herein. The subject host will have at least have one copy of the expression construct and may have two or more, for example, depending upon whether the gene is integrated into the genome, amplified, or is present on an extrachromosomal element having multiple copy numbers.<br>
The transformed host cell can be identified by selection for a marker contained on the introduced construct. Alternatively, a separate marker construct may be introduced with the desired construct, as many transformation techniques introduce many DNA molecules into host<br>
cel| Typically, transformed hosts are selected for their ability to grow on selective media. Selective media may incorporate an antibiotic or lack a factor necessary for growth of the ^transformed host, such as a nutrient or growth factor. An introduced marker gene therefor may confer antibiotic resistance, or encode an essential growth factor or enzyme, and permit growth on selective media when expressed in the transformed host. Selection of a transformed host can also occur when the expressed marker protein can be detected, either directly or indirectly. The marker protein may be expressed alone or as a fusion to another protein. The marker protein can be detected by its enzymatic activity; for example, beta-galactosidase can convert the substrate X-gal to a colored product, and luciferase can convert luciferin to a Hght-emitting product. The marker protein can be detected by its light-producing or modifying characteristics; for example, the green fluorescent protein of Aequorea victoria fluoresces when illuminated with blue light. Antibodies can be used to detect the marker protein or a molecular tag on, for example, a protein of interest Cells expressing the marker protein or tag can be selected, for example, visually, or by techniques such as FACS or panning using antibodies. Desirably, resistance to kanamycin and the amino glycoside G418 are of interest, as well as ability to grow on media lacking uracil, leucine, lysine or tryptophan.<br>
Of particular interest is the Δ15-desaturase-mediated production of PUFA's in eukaryotic host cells. Eukaryotic cells include plant cells, such as those from oil-producing crop plants, and other cells amenable to genetic manipulation including fungal cells. The cells may be cultured or formed as part or all of a host organism including a plant. Li a preferred embodiment, the host is a plant cell which produces and/or can assimilate exogenously supplied substrate(s) for a Δ15-desaturase, and preferably produces large amounts of one or more of the substrates.<br>
The transformed host cell is grown under appropriate conditions adapted for a desired end result. For host cells grown in culture, the conditions are typically optimized to produce the greatest or most economical yield of PUFA's, which relates to the selected desaturase activity. Media conditions which may be optimized include: carbon source, nitrogen source, addition of substrate, final concentration of added substrate, form of substrate added, aerobic or anaerobic growth, growth temperature, inducing agent, induction temperature, growth phase at induction, growth phase at harvest, pH, density, and maintenance of selection.<br>
Another aspect of the present invention provides transgenic plants or progeny of plants<br>
containing the isolated DNA of the invention. Both monocotyledonous and dicotyledonous<br>
plants are contemplated.   Plant cells are transformed with an isolated DNA encoding Δ15-<br>
deastrase by any of the plant transformation methods described above. The transformed plant<br>
cell, usually in a callus culture or leaf disk, is regenerated into a complete transgenic plant by<br>
methods well-known to one of ordinary skill in the art (e.g. Horsch et al., 1985). In one<br>
embodiment, the transgenic plant is selected from the group consisting of Arabidopsis thaliana,<br>
canola, soy, soybean, rapeseed, sunflower, cotton, cocoa, peanut, safflower, coconut, flax, oil<br>
palm, oilseed Brassica napus, com, jojoba, Chinese tallow tree, tobacco, fruit plants, citrus<br>
plants or plants producing nuts and berries. Since progeny of transformed plants inherit the<br>
polynucleotide encoding Δ15-desaturase, seeds or cuttings from transformed plants may be used<br>
to maintain the transgenic plant line.	1<br>
The present invention further provides a method for providing transgenic plants with an increased content of ALA and/or SDA. This method includes, for example, introducing DNA encoding Δ15-desaturase into plant cells which lack or have low levels of ALA or SDA but contain LA, and regenerating plants with increased ALA and/or SDA content from the transgenic cells. In certain embodiments of the invention, a DNA encoding a A6- and/or Δ12-desaturase may also be introduced into the plant cells. Such plants may or may not also comprise endogenous A6- and/or Δ12-desaturase activity. In certain embodiments, modified commercially grown crop plants are contemplated as the transgenic organism, including, but not limited to, Arabidopsis thaliana, canola, soy, soybean, rapeseed, sunflower, cotton, cocoa, peanut, safflower, coconut, flax, oil palm, oilseed Brassica napus, corn, jojoba, Chinese tallow tree, tobacco, fruit plants, citrus plants or plants producing nuts and berries.<br>
The present invention further provides- a method for providing transgenic plants which may contain elevated levels of ALA and/or SDA, wherein said elevated levels are greater than levels found in non-transformed plants. This method may comprise introducing one or more polynucleotide encoding Δ15-desaturase into a plant which lacks or has low levels of ALA, but contains LA. Expression vectors comprising DNA encoding a Δ15-desaturase, or a Δ15-desaturase and a A6-desaturase, can be constructed by methods of recombinant technology known to one of ordinary skill in the art (Sambrook et al., 1989). In particular, commercially grown crop plants are contemplated as the transgenic organism, including, but not limited to, Arabidopsis thaliana, canola, soy, soybean, rapeseed, sunflower, cotton, cocoa, peanut, safflower, coconut, flax, oil palm, oilseed Brassica napus, and com.<br>
For dietary supplementation, the purified PUFAs, transformed plants or plant parts, or derivatives thereof, may be incorporated into cooking oils, fats or margarines formulated so that in normal use the recipient would receive the desired amount.    The PUFAs may also be<br>
incorporated into infant formulas, nutritional supplements or other food products, and may find use as anti-inflammatory or cholesterol lowering agents.<br>
As used herein, "edible composition" is defined as compositions which may be ingested by a mammal such as foodstuffs, nutritional substances and pharmaceutical compositions. As-used herein "foodstuffs" refer to substances that can be used or prepared for use as food for a mammal and include substances that maybe used in the preparation of food (such as frying oils) or food additives. For example, foodstuffs include animals used for human consumption or any product therefrom, such as, for example, eggs. Typical foodstuffs include but are not limited to beverages, (e.g., soft drinks, carbonated beverages, ready to mix beverages), infused foods (e.g. fruits and vegetables), sauces, condiments, salad dressings, fruit juices, syrups, desserts (e.g., puddings, gelatin, icings and fillings, baked goods and frozen desserts such as ice creams and sherbets), soft frozen products (e.g., soft frozen creams, soft frozen ice creams and yogurts, soft frozen toppings such as dairy or non-dairy whipped toppings), oils and emulsified products (e.g., shortening, margarine, mayonnaise, butter, cooking oil, and salad dressings) and intermediate moisture foods (e.g., rice and dog foods).<br>
Furthermore, edible compositions described herein can also be ingested as an additive or supplement contained in foods and drinks. These can be formulated together with a nutritional substance such as various vitamins and minerals and incorporated into substantially liquid compositions such as nutrient drinks, soymilks and soups; substantially solid compositions; and gelatins or used in the form of a powder to be incorporated into various foods. The content of the effective ingredient in such a functional or health food can be similar to the dose contained in a typical pharmaceutical agent.<br>
The purified PUFAs, transformed plants or plant parts may also be incorporated into animal, particularly livestock, feed. In this way, the animals themselves may benefit from a PUFA rich diet, while human consumers of food products produced from such livestock may benefit as well. It is expected in certain embodiments that SDA will be converted to EPA in animals and thus such animals may benefit from an increase in EPA by consumption of SDA.<br>
For pharmaceutical use (human or veterinary), the compositions may generally be<br>
administered orally but can be administered by any route by winch they may be successfully<br>
absorbed, e.g., parenterally (i.e. subcutaneously, intramuscularly or intravenously), rectally,<br>
vaginally or topically, for example, as a skin ointment or lotion. The PUFAs transformed plants<br>
or plant parts of the present invention may be administered alone or 'in combination with a<br>
pharmaceutically acceptable carrier or excipient.   Where available, gelatin capsules are the<br>
prefered form of oral administration. Dietary supplementation as set forth above can also provide an oral route of administration. The unsaturated acids of the present invention may be administered in conjugated forms, or as salts, esters, amides or prodrugs of the fatty acids. Any pharmaceutically acceptable salt is encompassed by the present invention; especially preferred are the sodium, potassium or lithium salts. Also encompassed are the N-alkylpolyhydroxamine salts, such as N-methyl glucamine, found in PCT publication WO 96/33155. The preferred esters are the ethyl esters. As solid salts, the PUFAs also can be administered in tablet form. For intravenous administration, the PUFAs or derivatives thereof may be incorporated into commercial formulations such as Intralipids.<br>
If desired, the regions of a desaturase polypeptide important for desaturase activity can be determined through routine mutagenesis followed by expression of the resulting mutant polypeptides and determination of their activities. Mutants may include substitutions, deletions, insertions and point mutations, or combinations thereof. Substitutions may be made on the basis of conserved hydrophobicity or hydrophilicity (Kyte and Doolittle, 1982), or on the basis of the ability to assume similar polypeptide secondary structure (Chou and Fasman, 1978). A typical functional analysis begins with deletion mutagenesis to determine the N- and C-terminal limits of the protein necessary for function, and then internal deletions, insertions or point mutants are made to further determine regions necessary for function. Other techniques such as cassette mutagenesis or total synthesis also can be used. Deletion mutagenesis is accomplished, for example, by using exonucleases to sequentially remove the 5' or 3' coding regions. Kits are available for such techniques. After deletion, the coding region is completed by ligating oligonucleotides containing start or stop codons to the deleted coding region after 5' or 3' deletion, respectively. Alternatively, oligonucleotides encoding start or stop codons are inserted into the coding region by a variety of methods including site-directed mutagenesis, mutagenic PCR or by ligation onto DNA digested at existing restriction sites.<br>
Internal deletions can similarly be made through a variety of methods including the use of existing restriction sites in the DNA, by use of mutagenic primers via site directed mutagenesis or mutagenic PCR. Insertions are made through methods such as Imker-scanning mutagenesis, site-directed mutagenesis or mutagenic PCR. Point mutations are made through techniques such as site-directed mutagenesis or mutagenic PCR. Chemical mutagenesis may also be used for identifying regions of a desaturase polypeptide important for activity. Such stracture-function analysis can determine which regions may be deleted, which regions tolerate insertions, and" which point mutations allow the mutant protein to,function in substantially the<br>
sambway as the native desaturase. All such mutant proteins and nucleotide sequences encoding them are within the scope of the present invention.<br>
As described herein above, certain embodiments of the current invention concern plant transformation constructs. For example, one aspect of the current invention is a plant transformation vector comprising one or more desaturase gene(s) or cDNA(s). Exemplary coding sequences for use with the invention include Neurospora crassa gene Δ15-desaturase NcD15D (SEQ ID NO:l and SEQ ID NO:2) and Aspergillus nidulans Δ15-desaturase AnD15D (SEQ ID NO:4). In certain embodiments, antisense desaturase sequences can also be employed with the invention. Exemplary desaturase encoding nucleic acids include at least 20, 40, 80, 120, 300 and up to the full length of the nucleic acid sequences of SEQ ID NO:l, SEQ ID NO:2, SEQ ID NO:4 or SEQ ID NO:33 may be used. In certain aspects, a nucleic acid may encode 1, 2, 3, 4, or more desaturase enzymes. In particular embodiments, a nucleic acid may encode a A6- and a Δ15-desaturase.<br>
In certain embodiments of the invention, coding sequences are provided operably linked to a heterologous promoter, in either sense or antisense orientation. Expression constructs are also provided comprising these sequences, as are plants and plant cells transformed with the sequences. The construction of constructs which may be employed in conjunction with plant transformation techniques using these or other sequences according to the invention will be known to those of skill of the art in light of the present disclosure (see, for example, Sambrook et al., 1989; Gelvin et al., 1990). The techniques of the current invention are thus not limited to any particular nucleic acid sequences.<br>
One use of the sequences provided by the invention will be in the alteration of plant phenotypes, e.g., oil composition, by genetic transformation with desaturase genes, in particular embodiments a fungal Δ15-desaturase. The desaturase gene may be provided with other sequences. Where an expressible coding region that is not necessarily a marker coding region is employed in combination with a marker coding region, one may employ the separate coding regions on either the same or different DNA segments for transformation. In the latter case, the different vectors are delivered concurrently to recipient cells to maximize cotransformation.<br>
The choice of any additional elements used in conjunction with the desaturase coding sequences will often depend on the purpose of the transformation. One of the major purposes of transformation of crop plants is to add commercially desirable, agronomically important traits to the plant.   As PUFAs are known to confer many beneficial effects on health, concomitant<br>
iruazases in SDA production may also be beneficial and could be achieved by expression of fungal Δ15-desaturase. Such increasing of SDA may, in certain embodiments of the invention, comprise expression of A6 and/or A12 desaturase, including fungal or plant A6 and/or A12 desaturases.<br>
Vectors used for plant transformation may include, for example, plasmids, cosmids, YACs (yeast artificial chromosomes), BACs (bacterial artificial chromosomes) or any other suitable cloning system, as well as fragments of DNA therefrom. Thus when the term "vector" or "expression vector" is used, all of the foregoing types of vectors, as well as nucleic acid sequences isolated therefrom, are included. It is contemplated that utilization of cloning systems with large insert capacities will allow introduction of large DNA sequences comprising more than one selected gene. In accordance with the invention, this could be used to introduce various desaturase encoding nucleic acids. Introduction of such sequences may be facilitated by use of bacterial or yeast artificial chromosomes (BACs or YACs, respectively), or even plant artificial chromosomes. For example, the use of BACs for Agrobacterium-mediated transformation was disclosed by Hamilton et al. (1996).<br>
Particularly useful for transformation are expression cassettes which have been isolated from such vectors. DNA segments used for transfoirning plant cells will, of course, generally comprise the cDNA, gene or genes which one desires to introduce into and have expressed in the host cells. These DNA segments can further include structures such as promoters, enhancers, polylinkers, or even regulatory genes as desired. The DNA segment or gene chosen for cellular introduction will often encode a protein which will be expressed in the resultant recombinant cells resulting in a screenable or selectable trait and/or which will impart an improved phenotype to the resulting transgenic plant. However, this may not always be the case, and the present invention also encompasses transgenic plants incorporating non-expressed transgenes. Preferred components likely to be included with vectors used in the current invention are as follows.<br>
In one embodiment the instant invention utilizes certain promoters. Examples of such promoters that may be used with the instant invention include, but are not limited to, the 35S CaMV (cauliflower mosaic virus), 34S FMV (figwort mosaic virus) (see, e.g., U.S. Patent No. 5,378,619, the contents of which are herein incorporated in their entirety), Napin (from Brassica), 7S (from soybean), Glob and Lee (from corn). The 35S CaMV promoter and promoters, which are regulated during plant seed maturation, are of particular interest for use with the instant invention. All such promoter and transcriptional regulatory elements, singly or<br>
in cd ^bination, are contemplated for use in the present replicable expression vectors and are known to one of ordinary skill in the art.<br>
The CaMV 35S promoter is described, for example, by Restrepo et al. (1990). Genetically transformed and mutated regulatory sequences which lead to seed-specific expression may also be employed for the production of modified seed oil composition. Such modifications of the invention described here will be obvious to one skilled in the art.<br>
The DNA sequence between the transcription initiation site and the start of the coding sequence, i.e., the untranslated leader sequence, can also influence gene expression. One may thus wish to employ a particular leader sequence with a transformation construct of the invention. Preferred leader sequences are contemplated to include those which comprise sequences predicted to direct optimum expression of the attached gene, i.e., to include a preferred consensus leader sequence which may increase or maintain mRNA stability and prevent inappropriate initiation of translation. The choice of such sequences will be known to those of skill in the art in light of the present disclosure. Sequences that are derived from genes that are highly expressed in plants will typically be preferred.<br>
Transformation constructs prepared in accordance with the invention will typically include a 3' end DNA sequence that acts as a signal to terminate transcription and allow for the poly-adenylation of the mRNA produced by coding sequences operably linked to a desaturase gene (e.g., cDNA). In one embodiment of the invention, the native terminator of a desaturase gene is used. Alternatively, a heterologous 3' end. may enhance the expression of desaturase coding regions. Examples of terminators deemed to be useful include those from the nopaline synthase gene of Agrobacterium tumefaciens (nos 3' end) (Bevan et al., 1983), the terminator for the T7 transcript from the octopine synthase gene of Agrobacterium tumefaciens, the 3' end of the protease inhibitor I or II genes from potato or tomato and the CaMV 35S terminator (tml3'). Regulatory elements such as an Adh intron (Callis et al., 1987), sucrose synthase intron (Vasil et al., 1989) or TMV omega element (Gallie et al., 1989), may further be included where desired.<br>
By employing a selectable or screenable. marker protein, one can provide or enhance the<br>
ability to identify transformants.  "Marker genes" are genes that impart a distinct phenotype to<br>
cells expressing the marker protein and thus allow such transformed cells to be distinguished<br>
from cells that do not have the marker. Such genes may encode either a selectable or screenable<br>
marker, depending on whether the marker confers a trait which one can "select" for by chemical<br>
means, i.e., through the use of a selective agent (e.g., a herbicide, antibiotic, or the like), or<br>
whether it is simply a trait that one can identify through observation or testing, i.e., by<br>
"scrisning" (e.g., the green fluorescent protein).  Of course, many examples of suitable marker proteins are known to the art and can be employed in the practice of the invention.<br>
Suitable methods for transformation of plant or other cells for use with the current invention are believed to include virtually any method by which DNA can be introduced into a cell, such as by direct delivery of DNA such as by PEG-mediated transformation of protoplasts (Omirulleh et al., 1993), by desiccation/inhibition-mediated DNA uptake (Potrykus et al., 1985), by electroporation (U.S. Patent No. 5,384,253, specifically incorporated herein by reference in its entirety), by agitation with silicon carbide fibers (Kaeppler et al., 1990; U.S. Patent No. 5,302,523, specifically incorporated herein by reference in its entirety; and U.S. Patent No. 5,464,765, specifically incorporated herein by reference in its entirety), by Agrobacterium-mediated transformation (U.S. Patent No. 5,591,616 and U.S. Patent No. 5,563,055; both specifically incorporated herein by reference) and by acceleration of DNA coated particles (U.S. Patent No. 5,550,318; U.S. Patent No. 5,538,877; and U.S. Patent No. 5,538,880; each specifically incorporated herein by reference in its entirety), etc. Through the application of techniques such as these, the cells of virtually any plant species may be stably transformed, and these cells developed into transgenic plants.<br>
After effecting delivery of exogenous DNA to recipient cells, the next steps generally concern identifying the transformed cells for further culturing and plant regeneration. In order to improve the ability to identify transformants, one may desire to employ a selectable or screenable marker gene with a transformation vector prepared in accordance with the invention. In this case, one would then generally assay the potentially transformed cell population by exposing the cells to a selective agent or agents, or one would screen the cells for the desired marker gene trait.<br>
Cells that survive the exposure to the selective agent, or cells that have been scored positive in a screening assay, may be cultured in media that supports regeneration of plants. In an exemplary embodiment, MS and N6 media may be modified by including further substances such as growth regulators. One such growth regulator is dicamba or 2,4-D. However, other growth regulators may be employed, including NAA, NAA + 2,4-D or picloram. Media improvement in these and like ways has been found to facilitate the growth of cells at specific developmental stages. Tissue may be maintained on a basic media with growth regulators until sufficient tissue is available to begin plant regeneration efforts, or following repeated rounds of manual selection, until the morphology of the tissue is suitable for regeneration, typically at least 2 weeks, then transferred to media conducive to maturation of embryoids.    Cultures are<br>
transi rred every 2 weeks on this medium. Shoot development will signal the time to transfer to medium lacking growth regulators.<br>
To confirm the presence of the exogenous DNA or "transgene(s)" in the regenerating plants, a variety of assays may be performed. Such assays include, for example, "molecular biological" assays, such as Southern and Northern blotting and PCR™; "biochemical" assays, such as detecting the presence of a protein product, e.g., by immunological means (ELISAs and Western blots) or by enzymatic function; plant part assays, such as leaf or root assays; and also, by analyzing the phenotype of the whole regenerated plant.<br>
In addition to direct transformation of a particular plant genotype with a construct prepared according to the current invention, transgenic plants may be made by crossing a plant having a selected DNA of the invention to a second plant lacking the DNA. Plant breeding techniques may also be used to introduce a multiple desaturases, for example A6, A12, and/or Al 5-desaturase(s) into a single plant. In this manner, Δ15-desaturase can be effectively up-regulated. By creating plants homozygous for a Δ15-desaturase activity and/or other desaturase activity (e.g., A6- and/or Δ12-desaturase activity) beneficial metabolites can be increased in the plant.<br>
As set forth above, a selected desaturase gene can be introduced into a particular plant variety by crossing, without the need for ever directly transforming a plant of that given variety. Therefore, the current invention not only encompasses a plant directly transformed or regenerated from cells which have been transformed in accordance with the current invention," but also the progeny of such plants. As used herein the term "progeny" denotes the offspring of any generation of a parent plant prepared in accordance with the instant invention, wherein the progeny comprises a selected DNA construct prepared in accordance with the invention. "Crossing" a plant to provide a plant line having one or more added transgenes or alleles relative to a starting plant line, as disclosed herein, is defined as the techniques that result in a particular sequence being introduced into a plant line by crossing a starting line with a donor plant line that comprises a transgene or allele of the invention. To achieve this one could, for example, perform the following steps: (a) plant seeds of the first (starting line) and second (donor plant line that comprises a desired transgene or allele) parent plants; (b) grow the seeds of the first and second parent plants into plants that bear flowers; (c) pollinate a flower from the first parent plant with pollen from the second parent plant; and (d) harvest seeds produced on the parent plant bearing the fertilized flower.<br>
Backcrossing is herein defined as the process including the steps of: (a) crossing a plant of a first genotype containing a desired gene, DNA sequence or element to a plant of a second genotype lacking said desired gene, DNA sequence or element; (b) selecting one or more progeny plant containing the desired gene, DNA sequence or element; (c) crossing the progeny plant to a plant of the second genotype; and (d) repeating steps (b) and (c) for the purpose of transferring a desired DNA sequence from a plant of a first genotype to a plant of a second genotype.<br>
Introgression of a DNA element into a plant genotype is defined as the result of the process of backcross conversion. A plant genotype into which a DNA sequence has been introgressed may be referred to as a backcross converted genotype, line, inbred, or hybrid. Similarly a plant genotype lacking the desired DNA sequence may be referred to as an unconverted genotype, line, inbred, or hybrid.<br>
EXAMPLES<br>
The following examples are included to illustrate embodiments of the invention. It should be appreciated by those of skill in the art that the techniques disclosed in the examples which follow represent techniques discovered by the inventor to function well in the practice of the invention. However, those of skill in the art should, in light of the present disclosure, appreciate that many changes can be made in the specific embodiments which are disclosed and still obtain a like or similar result without departing from the concept, spirit and scope of the invention. More specifically, it will be apparent that certain agents which are both chemically and physiologically related may be substituted for the agents described herein while the same or similar results would be achieved. All such similar substitutes and modifications apparent to those skilled in the art are deemed to be within the spirit, scope and concept of the invention as defined by the appended claims.<br>
Example 1<br>
Strains and Growth Conditions<br>
Neurospora crassa mating type A and Aspergillus nidulans Glasgow wild type were obtained from the Fungal Genetics Stock Center. Cultures were grown in Vogel's medium N. (Case et ah, Neurospora Newsletter, 8:25-26, 1965). Liquid cultures were inoculated with ascospores and grown for three days at 15°~C with shaking at 100 RPM. Mycelium was harvested by filtration in a Buchner funnel through Whatman number 1 paper and stored at 80°C<br>
for NA isolation or directly lyophilized for fatty acid composition determination by gas chromatography. The Saccharomyces cerevisiae strain used was INVScl, a diploid strain that is auxotrophic for histidine, leucine, tryptophan, and uracil (rnvitrogen). Cells were maintained on YPD media at 30°C.<br>
Example 2<br>
Isolation of Fungal RNA<br>
Total RNA was isolated from fungal mycelium of the 3 strains described in Example 1 using the acid guanidinium-phenol-chloroform method of Chomczynski and Sacchi, (1987, Tri-Reagent, SIGMA). This method provides 500 mg of mycelium being ground in liquid nitrogen then added to 7 ml of Tri-Reagent. Chloroform was added to separate the aqueous phase from the organic phase. The RNA was precipitated with isopropanol then washed with 70% ethanol before being resuspended in deionized water.<br>
Example 3<br>
Cloning of the N. crassa A12 and Δ15-Desaturase Sequences<br>
Based on sequence comparisons to the N. crassa genomic sequences, gene specific primers were designed to amplify the full-length coding regions of the putative Δ12-desaturase (NclllF2 and NclllR3) and the putative Δ15-desaturase (Nc94F6 and Nc94R8). Forward primers were designed to include three nucleotides 5' of the start Met site<br>
 (SEQIDNO:15)<br>
(Sequence Removed) <br>
 (SEQIDNO:16)<br>
(Sequence Removed) <br>
S' (SEQ ID NO: 17). TheiVM site added to the 5' end of the oligonucleotide is italicized.<br>
(Sequence Removed) <br>
 (SEQ ID NO:18). The iS!?e83871 site added to the 5' end of the oligonucleotide is italicized.<br>
The cDNA for N. crassa was prepared using the Marathon cDNA Amplification kit (Clontech Laboratories).   These primers were used with 3'-RACE ready cDNA to amplify<br>
putative desaturases using a Gene Amp PCR system 9700 (PE Applied Biosystems) with the<br>
recdapnended cycle conditions. The PCR product generated with oligonucleotides Nc94F6 and Nc94R8 was ligated into pCR2.1-TOPO (Invitrogen) and named pMON67004 (FIG. 1). The cDNA was sequenced and three "His-boxes", a conserved feature among membrane-bound desaturases, were found to be present at amino acid positions 124-128, 160-164, and 359-363.<br>
When compared to other membrane-bound A12 and Δ15-desaturases, the final "HXXHH" histidine box motif was found to be intact as well. The corresponding nucleotide and polypeptide sequences for the Δ15-desaturase (NcD15D) are given in SEQ ID NO:2 and SEQ ID NO:3, respectively, and the genomic clone is given in SEQ ID NO:l. pMON67004 was digested with EcoRl and ligated into the EcoRl site of the yeast expression vector pYES2/CT to generate pMON77208 (FIG. 2). For the plant transformation vectors, pMON67004 was digested with EcoRl, followed by a fill-in reaction, and then cut by Sse8387I. The gene fragment was ligated into the binary vector, pMON73270, which was digested by NotI, followed by a fill-in reaction, and then by Sse8387I. This gave rise to vector pMON77214 (FIG. 4) in which the Δ15-desamrase gene, NcD15D, was under regulation of the seed-specific Napin promoter. The EcoRl/Sse8387I-digested DNA fragment was also ligated into the binary vector, pMON73273, giving rise to pMON77217 (FIG. 4), in which NcD15D was under regulation of the constitutive 35S promoter.<br>
The PCR product generated with oligonucleotides NclllF2 and NclllR3 was ligated directly into pYES2.1/V5-His-TOPO (Invitrogen) to generate pMON67005 (FIG.7A). The cDNA was sequenced and three "His-boxes" were found to be present at amino acid positions 158-162, 194-198, and 394-398. When compared to other membrane-bound A12 and Δ15-desaturases, the final "HXXHH" histidine box motif was found to be intact as well. The corresponding nucleotide and polypeptide sequences for the putative Δ12-desaturase (NcD12D) are given in SEQ ID NO:39 and SEQ JD NO:40, respectively.<br>
Example 4<br>
Yeast Transformation and Expression<br>
Constructs pMON67005 and pMON77208 were introduced into the host strain S. cerevisiae INVScl (auxotrophic for uracil) using the PEG/Li Ac protocol as described in the Invitrogen manual for pYES2.1 /V5-His-TOPO. Transformants were selected on plates made of SC minimal media minus uracil with 2% glucose. Colonies of transformants were used to inoculate 5 "ml of SC minimal media minus uracil and 2% glucose grown overnight at 30°C. For<br>
invetion, stationary phase yeast cells were pelleted and resuspended in SC minimal media minus uracil supplemented with 2% galactose and grown for 3 days at 15°C. When exogenous fatty acids were provide to the cultures, 0.01% LA (A9,12-18:2) was added with the emulsifier 0.1% Tergitol. The cultures were grown for 3 days at 15°C, and subsequently harvested by centrifugation. Cell pellets were washed once with sterile TE buffer pH 7.5, to remove the media, and lyophilized for 24 h. The host strain transformed with the vector containing the LacZ gene was uses as a negative control in all experiments.<br>
For fatty acid analysis, the extraction of the yeast lipids followed the procedures described previously. Briefly, lyophilized yeast pellets were extracted with 15 mL of methanol and 30 mL of chloroform containing 100 jig of tridecanoin. After extraction, the yeast lipids were first saponified, and the liberated fatty acids were methylated. The distribution of fatty acid methyl esters was then analyzed by gas chromatography (GC) using a Hewlett-Packard 5890 II Plus gas chromatograph (Hewlett-Packard, Palo Alto, CA) equipped with a flame-ionization detector and a fused-silica capillary column (Supelcomega; 50 m x 0.25 mm, i.d., Supelco, Bellefonte, PA).<br>
In yeast transformed with the expression vector containing LacZ as a control, no LA or ALA (18:3) was measured in lines grown in the absence of added LA. In yeast transformed with an expression vector containing NcD15D or BnD15D, in the absence of added LA, no ALA accumulated. In yeast transformed with an expression vector containing NcD12D, without added LA, LA accumulated to 22% of the fatty acids, indicative of D12D activity. When LA was added to the yeast line expressing NcD15D, ALA compromised 1% of the fatty acids. In the yeast line expressing the Brassica napus Δ15-desaturase (BnD15D), ALA compromised 0.2% of the fatty acids after addition of LA. In the LacZ control, no ALA was detected after LA addition.<br>
TABLE 1 Yeast Expression Data<br>
(Table Removed) <br>
Example 5<br>
Arabidopsis Transformation with NcD15D<br>
This example describes the transformation and regeneration of transgenic Arabidopsis thaliana plants expressing a heterologous Δ15-desaturase coding sequence. Arabidopsis plants were grown by sowing seeds into 4 inch pots containing reverse osmosis water (ROW) and saturated MetroMix 200 (The SCOTTS Co., Columbus, OH). The plants were vernalized by placing the pots in a flat, covered with a humidity dome, in a growth chamber at 4-7°C, 8 hours light/day for 4-7 days. The flats were transferred to a growth chamber at 22° C, 55% relative humidity, and 16 hours light/day at an average intensity of 160-200 ~Mol/sec*m2. After germination, the dome was lifted and slid back 1" to allow for mild air circulation without desiccation. The humidity dome was removed when the true leaves had formed. The plants were bottom watered, as needed, with ROW until 2-3 weeks after germination. Plants were then bottom watered, as needed, with PLANTEX 18-18-15 solution (Plantex Corporation Ottawa, Canada) at 50 ppm N2. Pots were thinned so that 1 plant remained per pot at 2-3 weeks after germination. Once the plants began to bolt, the primary inflorescence was trimmed to encourage the growth of axillary bolts.<br>
The transformation vectors pMON77214 and pMON77217 were introduced into Agrobacterium tumefaciens strain ABI using methodology well known in the art. Transgenic A. thaliana plants were obtained as described by Bent et al. (1994)^ Bechtold et al. (1993). Briefly, cultures of Agrobacterium containing binary vectors pMPON77214 or pMON77217, were grown overnight in LB (10% bacto-tryptone, 5% yeast extract, and 10% NaCl with kanamycin (75 mg/L), chloramphenicol (25 mg/L), and spectinomycin (100 mg/L)). The bacterial culture was centrifuged and resuspended in 5% sucrose +.05% Silwet-77. The aerial portion of whole A. thaliana plants (-5-7 weeks of age) were immersed in the resulting solution for 2-3 seconds. The excess solution was removed by blotting the plants on paper towels. The dipped plants were placed on their side in a covered flat and transferred to a growth chamber at 19°C. After 16 to 24 hours the dome was removed and the plants were set upright. When plants reached maturity, water was withheld for 2-7 days prior to seed harvest.. Harvested seed was passed through a stainless steel mesh screen.<br>
To select transformants, seed was plated on agar medium containing 50 mg/L glyphosate. Green seedlings were rescued and transplanted into 4" pots and grown under the conditions<br>
desd   ,ed above. Leaves were harvested for fatty acid analysis when the rosette was at the 4-leaf stage. After lyophylization, leaf fatty acids were analyzed as described above.<br>
Example 6<br>
Functional Expression of N. crassa Clones<br>
In order to assess the functional specificity of the N. crassa D15D clone, the coding region from pMON67004 was cloned into a plant expression vector in which the constitutive 35S promoter drives expression of the transgene. The resulting construct, pMON77217, was transformed into A. thaliana and leaves of transformed T2 plants were analyzed for fatty acid composition. In non-transformed lines, approximately 20% of the fatty acids were LA, and approximately 48% ALA. In two independent A. thaliana transformation events, LA levels were reduced to approximately 3% and 5%, and ALA levels increased to 65% and 63%, respectively, indicating Δ15-desaturase activity inplanta. These data are summarized in Table 2. Controls are designated as CONT.<br>
TABLE 2 Fatty Acid Content of Arabidopsis Leaves<br>
(Table Removed) <br>
In order to assess the functional specificity of the N. crassa D15D clone to direct production of ALA in seeds, the coding region of pMON67004 was cloned into a seed-specific expression vector in which the Napin promoter drives expression of the transgene. The resulting construct, pMON77214, was transformed into A. thaliana and seeds of transformed T2 plants were analyzed for fatty acid composition. In non-transformed lines, approximately 26% of the seed lipids was present as LA, and approximately 18% as ALA. In two independents, thaliana transformation events, LA acid levels were reduced to approximately 14% and 13%, and ALA acid levels increased to 26% and 30%, respectively, indicating Δ15-desaturase activity in seeds. These data are shown in Table 3.<br>
TABLE 3 Fatty Acid Content of Arabidopsis Seeds<br>
(Table Removed) <br>
These results indicate that the protein encoded by the Neurospora NcD15D cDNA is a functional Δ15-desaturase in plants and can direct synthesis of ALA in leaves and in seeds.<br>
Example 7<br>
Activity of the Neurospora crassa Δ15-Desaturase in Canola<br>
Lines were transformed with construct pMON77214, which contains the Neurospora Δ15-desaturase driven by the Napin promoter. Both Quantum and Ebony canola varieties were transformed and controls for both varieties included. Data shown in Table 4 is percent 18:2 (LA) and 18:3 (ALA) in pools of 20 seeds from Ro plants.<br>
TABLE 4 Percent PUFAs in Pools of 20 Seeds from Ro Plants.<br>
(Table Removed) <br>
 The production of ALA at levels greater than -12% of seed fatty acids in these lines was indicative of the heterologous Δ15-desaturase activity. The highest level of ALA observed from this transformation was in line BNJ31395, which contains 30.17% ALA.<br>
For several of the lines expressing pMON77214, fatty acids in single seeds were determined and lines advanced to the next generation. As expected, ALA levels increased up to nearly 2-fold in individual seeds relative to the pools, indicative of homozygosity for the transgenes in individual segregants within each silique.   In line BN_1296, Pooled Rl seed<br>
contained 25.04% ALA. In the highest single seed from this line (BN_G1296-14), 48.2% ALA was-observed. The ALA levels in 200 half-seeds, ordered from lowest to highest ALA, is shown in FIG. 3.<br>
Example 8<br>
Cloning of the Δ15-desaturase Sequence from A nidulans and the Δ12- and Δ15-Desaturase<br>
Sequences from B. cinerea<br>
Based on sequence comparisons to the A. nidulans genomic sequence, gene specific primers were designed to amplify the full-length coding regions of the putative Δ15-desaturase (AnD15-Fl and AnD15-Rl). The forward primer was designed to include three nucleotides 5' of the start Met site<br>
AnD15-Fl: 5'-AATATGGCTGCAACTGCAACAACCC-3' (SEQIDNO:23)<br>
AnD15-Rl: 5'-TTCCGCTTTGGCACCCTTCTTC-3' (SEQ IDNO:24)<br>
Oligonucleotide primers BcD12Fl and BcD12Rl were designed from a partial genomic sequence (Monsanto proprietary partial gDNA clone found with BLASTALL) to amplify the full-length coding regions of B. cinerea Δ12-desaturase. The degenerate primer D15D-R9 was designed to amplify any putative B. cinerea Δ 15-desaturase in a 5'-RACE reaction. Oligonucleotide BCD15-F1 was designed for a 3' RACE reaction of the PCR product generated from oligonucleotide D15D-R9. Oligonucleotides BcD15F3 and BcD15Rl F were designed to amplify the full-length coding region of a putative B. cinerea  Δ15-desaturase.<br>
end contains Sail -3'(SEQIDNO:25).<br>
end contains PstI(SEQIDNO:26)<br>
 (SEQ ID NO:27)<br>
 (SEQ ID NO:28)<br>
end contains Sail (SEQ ID NO:29)<br>
end contains Pstl (SEQ E) NO:30) (Sequence Removed) <br><br>
cDNA for A. nidulans and B. cinerea were prepared using the GeneRacer kit (Tnvitrogen). These primers were used with 3'-RACE ready cDNA to amplify putative desaturases using a Gene Amp PCR system 9700 (RE APPLIED BIOSYSTEMS) with the recommended cycle conditions. The PCR product encoding A. nidulans Δ15-desaturase was generated with oligonucleotides AnD15-Fl and AnD15-Rl, was ligated into pYES2.1-TOPO (Invitrogen) and named pMON67010 (FIG. 7B). The cDNA was sequenced and three "His-boxes", a conserved feature among membrane-bound desaturases, were found to be present at amino acid positions 93-97, 129-133, and 327-331. The corresponding nucleotide and polypeptide sequences for the Δ15-desaturase (AnD15D) are given in SEQ ID NO:4 and SEQ ID NO:5, respectively.<br>
A B. cinerea Δ12-desaturase-encoding cDNA was amplified by PCR with oligonucleotides BcD12Fl and BcD12Rl and subsequently ligated directly into pYES2.1/V5-His-TOPO (Invitrogen) to generate pMON67022 (FIG. 7D). The cDNA was sequenced and three "His-boxes", a conserved feature among membrane-bound desaturases, were found to be present at amino acid positions 155-159, 191-195, and 390-394. The corresponding nucleotide and polypeptide sequences for the putative Δ12-desaturase (BcD12D) are given in SEQ ID NO:31 and SEQ ID NO:32, respectively.<br>
To clone a Δ15-desaturase from B. cinerea a degenerate oligonucleotide was generated<br>
based on an amino acid sequence alignment of the N. crassa, and Aspergillus sp. Δ12 and Δ15-<br>
desaturases. A 5'-RACE reaction was performed using a GeneRacer Kit (Invitrogen, Carlsbad<br>
CA) following the conditions recommended by the manufacturer.  Following cDNA synthesis,<br>
the 5' end of a putative Δ15-desaturase cDNA was amplified by PCR using the degenerate<br>
oligonucleotide D15D-R9 and ligated into pCR2.1-TOPO.  The resulting 742 bp fragment was<br>
sequenced and determined by deduced amino acid ahgmnent to be similar to the other fungal<br>
Δ15-desaturases.   A 3'-RACE reaction was used to amplify 664 bp from the 3' end of the<br>
putative B. cinerea Δ15-desaturase using oligonucleotide BcD15-Fl and ligated into pCR2.1-<br>
TOPO. Oligonucleotides BcD15F3 and BcD15Rl were designed from the composite sequence<br>
of the 5'- and 3'- RACE products, and used to amplify a full length B. cinerea putative A15-<br>
desaturase cDNA by 3'-RACE reaction and ligated into pYES2.1-TOPO.  The resulted plasmid<br>
was named pMON67021 (FIG. 7C). The corresponding nucleotide and polypeptide sequences<br>
for the putative Δ15-desaturase (BcD15D) are given in SEQ ID NO:33 and SEQ ID NO;34,<br>
respectively.<br>
To assess A15-desaturase activity of the putative AILDISD in the yeast expression assay, yeast expressing the putative A15-desaturase were fed the substrate for this enzyme, i.e., LA, and the production of ALA quantified. These data, in which the production of ALA by the K crassa A15-desaturase, pMON67023, was compared with that of the A. nidulans A15-desaturase, are shown in the Table 5. pMON67023 (FIG. 7E) was constructed as follows:<br>
 (SEQIDNO:35)<br>
 (SEQ ID NO:36) (Sequence Removed) <br><br>
Primers Nc94F2 and Nc94R2 were used to amplify the coding region of NcDISD without a stop codon. The resulting fragment was ligated into pYES2.1-TOPO to generate an inframe fusion between the NcDl5D coding region and the V5 epitope and 6-His region contained on the pYES2.1 expression vector.<br>
TABLE 5 Production of ALA by Neurospora crassa A15-Desaturase and   Aspergillus nidulans A15-desaturase<br>
(Table Removed) <br>
These results indicate that in this expression system; the A. nidulans desaturase is approximately 2-fold more active than NcD15D.<br>
Table 6 Analysis of AnD15D Substrate Utilization in Yeast<br>
(Table Removed) <br>
These results indicate that in this expression system, the A. nididans D15D is capable of desaturating both LA and GLA.<br>
Example 9<br>
Codon Optimization of the A15-Desaturases From A nidulans and N. crassa for Soybean<br>
A codon usage table was constructed from 8 highly expressed seed specific proteins from soybean (conglycinin, glycinin, globulin) and 17 highly expressed seed specific proteins from canola (cuciferin, napin, oleosin). The NcD15D and AnD15D nucleic acid sequences, along with the codon usage table described above, were sent to Blue Heron Biotechnology lac, (Bothell, Wa), who then utilized a proprietary algorithm to generate the final codon-optimized sequences with the lowest free energy-of-forming RNA secondary structures. The codon-optimized sequence of NcD15D was synthesized by Blue Heron Biotechnology inc., and named NcD15Dnno (SEQ ED NO:37). The codon-optimized sequence of AnD15D was synthesized by Midland (Midland, TX), and named AnD15Dnno (SEQ ID NO:38).<br>
Example 10<br>
Activity oftheNeurospora Al5-desaturase in combination with the A6- and A12-<br>
desaturases from Mortierella alpina<br>
The activity of the Neurospora A15-desaturase in combination with the A6- and A12-desaturases from Mortierella alpina was evaluated by fransforrning canola with construct pMON77216 (FIG. 7G), which contains the three desaturases under the control of the Napin promoter. In a number of lines obtained, however, the A12-desaturase was found to have been partially deleted. Fatty acid content of 10-seed pools from individual R0 plants was determined. The levels of stearic acid (18:0) (SA), oleic acid (18:l)(OA), LA, ALA, SDA and GLA are shown in Table 6 below. The control line was Ebony. Pooled seed from a majority of the transgenic events produced contained measurable SDA and in 8 events SDA accumulated to greater than 10% of the fatty acids<br>
TABLE 6 Relative Area Percent Results (Approx. wt percent) from pooled Rl seeds<br>
(Table Removed) <br>
Fatty acid data from single seeds of event BN_G1824, including both homozygotes and heterozygotes, is shown below in Table 7. In one case, 18.6.% SDA, 17.8% ALA, 11.2% LA, 24% oleic acid and 18.8% GLA were observed. This event is referred to as a high SDA/high GLA,eyent. In another seed from this event, 16.8% SDA, 7% ALA, 2% LA, 62.1% oleic acid and 3.1% GLA were observed. This event is referred to as a high SD A/low GLA line. Molecular data indicated that, in the high SD A/low GLA lines, the A12 coding sequence was not functional. In particular, it was indicated that the high SD A/low GLA lines were comprised of a single copy of a single partial T-DNA insert that has lost all insert DNA between the left border and the terminal 51 base pairs of the coding region of the Mortierella alpina A12-desaturase (e.g., last 51 bp of SEQ ID NO:41). Notable in the high SDA/low GLA line is that oleic acid is nearly at wild type levels whereas in the high SDA/high GLA lines, oleic acid is reduced approximately 2.5 fold with respect to wildtype. The lines that display the high SD A/high oleic phenotype are highlighted with grey.<br>
TABLE 7 Relative Area Percent Results (Approx. wt percent) Rl Single Seed of BN_G1190<br>
(Table Removed) <br>
In order to further assess the activity of the Neiirospora crassa A15-desaturase in combination with the M. alpina A6- and Al2-desaturases, lines homozygous for construct pCGN5544 (containing M. alpina A6- and A12-desaturases), which contained up to 35% GLA in seed oils, were re-transformed with construct pMON77214 containing NcD15D.  Twenty-seed<br>
pq'" J from 11 Ro plants were analyzed. The LA, ALA, SDA and GLA in these lines are shown in Table 8.<br>
TABLE 8 Relative Area Percent Results (Approx. wt percent) Analysis of Rl Pool seed<br>
(Table Removed) <br>
Example 11<br>
Activity of the Neurospora crassa A15-Desaturase in Combination With the A6-Desatnrase<br>
From Mortierella alpina<br>
The activity of the Neurospora crassa A15-desaturase in combination with the A6-desaturase from Mortierella alpina was evaluated by transforming canola with the construct pMON77215 (FIG. 7F), which contains the two desaturases under the control of the Napin promoter. This vector was constructed by digesting pCGN5536 (U.S. Pat. No. 6,459,018 Bl), which contains the Napin promoter driving expression of the M. alpina A6-desaturase (MaD6D), with NotI and then by ligating the expression cassette fragment into the Not I site of the binary vector, pMON70660, to form pMON77212. The pMON77215 plasmid was constructed by digesting pMON77214 with Pmel and AscI and then by ligating the resulting Napin-NcD15D expression cassette fragment into the Swal and AscI sites of pMON77212, to give a construct containing both MaD6D and NcD15D.<br>
Fatty acid content of 10-seed pools from individual RO canola transformants was determined. The levels of SA, OA, LA, ALA, SDA and GLA are shown in Table 9 below. The control line was Ebony (SP30052). Pooled seed from a majority of the transgenic events produced contained measurable SDA and in 25% of the events (10 out of 40) SDA accumulated to greater than 10% of the fatty acids.<br>
TABLE 9 Relative Area Percent Results (Approx. wt percent) for pMON77215 Pooled Rl Seecr<br><br>
(Table Removed) <br>
Fatty acid data from single seeds of from event BN_G1860, including both homozygotes and heterozygotes, is shown below in Table 10. In one case, up to 19% SDA, 10% ALA, 7% LA, 48% Oleic acid and 5% GLA was observed.<br>
TABLE 10 Relative Area Percent Results (Approx. wt percent) for pMON77215 Single Rl Seed of BN_G1860<br><br>
(Table Removed) <br>
Example 12<br>
Codon Optimization of the Al5-Desaturase Sequence from N. crassa for Maize<br>
A codon usage table was constructed from 9 highly expressed seed-specific genes from maize (six zeins and three oleosins). Using this table, two codons of NcD15D were mutated using the QuikChange Site-Directed Mutagenesis Kit (Stratagene, La Jolla, CA) and the resulting sequence was named NcFAD3m (SEQ ID NO:42). The codons changed were as follows: 1) to make a more preferred translational start site, an alanine in SEQ ID NO:2 is substituted with a threonine by changing the first base of the second codon (position 4 in SEQ ID NO: 42) from an ACG to GCG; and 2) to remove a rare codon, a valine codon was changed from GTA to GTG at position 882 (SEQ ID NO: 42).<br>
Example 13<br>
EPA Equivalence<br>
One measure of seed oil quality for health value is EPA equivalence. The value reflects the metabolic conversion rate to EPA. This is calculated by adding the % ALA divided by 14 and the % SDA divided by 4. The canola oil compositions obtained by the inventors had a high EPA equivalence, indicating excellent characteristics for achieving the health benefits associated increased EPA levels in humans and animals. An example of the analysis is given below by comparison of conventional canola oil relative to an example of a typical high SDA oil composition of 10% ALA and 15% SDA. Canola oil from conventional varieties has approximately 12% ALA and 0% SDA and thus has an EPA equivalence of 12/14 + 0/4= 0.8. ha contrast, the high SDA oil composition example has an EPA equivalence of 10/14 + 15/4=4.4. The relative values are shown below. Values are by wt %, not on a serving basis. The vast difference shows the importance of producing SDA in canola oil.<br>
TABLE 11 EPA Equivalence Comparison<br>
(Table Removed) <br>
REFERENCES<br>
The references listed below are incorporated herein by reference to the extent that they supplement, explain, provide a background for, or teach methodology, techniques, and/or compositions employed herein.<br>
U.S. Patent 4,535,060<br>
U.S. Patent 4,666,701<br>
U.S. Patent 4,758,592<br>
U.S. Patent 4,826,877<br>
U.S. Patent 4,910,141<br>
U.S. Patent 5,011,770<br>
U.S. Patent 5,116,871<br>
U.S. Patent 5,302,523<br>
U.S. Patent 5,322,783<br>
U.S. Patent 5,378,619<br>
U.S. Patent 5,384,253<br>
U.S. Patent 5,464,765<br>
U.S. Patent 5,508,184<br>
U.S. Patent 5,508,468<br>
U.S. Patent 5,538,877<br>
U.S. Patent 5,538,880<br>
U.S. Patent 5,545,818<br>
U.S. Patent 5,550,318<br>
U.S. Patent 5,563,055<br>
U.S. Patent 5,591,616<br>
US. Patent 5,610,042<br>
U.S. Patent 5,952,544<br>
Abdullah et al., Biotechnology, 4:1087,1986.<br>
Axondel  science, 258(5086):1353-1355,1992.<br>
Ausubel et al, ha: Current Protocols in Molecular Biology, Green Publishing Assoc, NY, 1987.<br>
Barton et al, Cell, 32:1033,1983,<br>
Bates, Mol BiotechnoL, 2(2):135-45,1994.<br>
Battraw and Hall, Theor. App. Genet., 82(2):161-168,1991.<br>
BechUfld et al, CR. Acad. Sci. Life Set, 316:1194-1199,1993.<br>
Becker and Guarente, Methods Enzymol, 194:182-187,1991.<br>
Bent etal, Science, 265:1856-1860,1994.<br>
Bent etal (1994), Science 265:1856-1860<br>
Bevan  et al.,   Nucleic Acids Res., ll(2):369-385, 1983.<br>
Bevan, TVucfezc Acids Res., 12:8111,1984.<br>
Bhattacharjee, An, Gupta, J. Plant Bioch. Biotech., 6(2):69-73.1997.<br>
Bouchez et al, EMBO Journal, 8( 13):4197-4204,1989.<br>
Bower et al, J. Plant, 2:409-416. 1992.<br>
Bray, 1987<br>
Brenner et al.,Adv. Exp. Med. Biol, 83:85-101,1976.<br>
Buchanan-Wollaston et al, Plant Cell Reports, 11:627-631. 1992<br>
BuisingandBenbow,M?/. Gen. Genet, 243(1):71-81,1994.<br>
Bustos et al, J. Bacteriol, 174:7525-7533,1991.<br>
Ca\hsetal, Genes Dev., 1:1183-1200,1987.<br>
Casa et al,Proc. Natl Acad. Sci. USA, 90(23): 11212-11216,1993.<br>
Case et al, Neurospora Newsletter, 8:25-26,1965.<br>
Chandler et al, The Plant Cell, 1:1175-1183, 1989.<br>
Chomczynski and Sacchi, Anal Biochem., 162(1):I56-159,1987.<br>
Chou and Fasman, Adv. Enzymol, 47:45-148, 1978.<br>
Christou etal, Proc. Natl Acad. Sci. USA, 84(12):3962-3966,1987.<br>
Conkling etal., Plant Physiol, 93:1203-1211, 1990.<br>
Crozier, Lipids, 24:460,1989.<br>
De Block et al, Plant Physiol, 91:694-701,1989.<br>
De Block et al, The EMBO Journal, 6(9):2513-2518, 1987.<br>
De Deckerer, Eur. J. Clin. Nutr., 52:749,1998.<br>
DeBlock etal, J. EMBO, 2:2143,1984.<br>
Dellaporta et al, In:   Chromosome Structure and Function:   Impact of New Concepts, 18th<br>
Stadler Genetics Symposium, 11:263-282,1988. Denmark Appl. DE 3642 829 D'Halluin et al, Plant Cell, 4(12):1495-1505, 1992. Ebertetal,Proc. Natl. Acad. Sci. USA, 84:5745-5749, 19S7 Ellis et al, EMBO Journal, 6(ll):3203-3208,1987. Europ. Appl. 154,204<br>
Fraley et L/., Bio/Technology, 3:629-635,1985.<br>
Fromm et al, Nature 319:791-793,1986<br>
Fromm et al, Nature, 319:791-793,1986.<br>
Gallie etal, The Plant Cell, 1:301-311,1989.<br>
Geivmetal.,In: Plant Molecular Biology Manual, 1990.<br>
Ghosh-Biswas et al., J. Biotechnol, 32(1):1-10,1994.<br>
Goeddel, In: Methods in Enzymology, Perbal (Ed.), Academic Press, John Wiley and Sones, 185,<br>
1988. Hagio, Blowers, Earle,Plajit Cell Rep., 10(5):260-264,1991. Hamilton et al, Proc. Natl. Acad. Sci. USA, 93(18):9975-9979,1996. Haseloffetal,Proc. Natl. Acad. Sci. USA, 94(6):2122-2127,1997. He et al, Plant Cell Reports, 14 (2-3): 192-196, 1994. Hensgens et al, PlantMol Biol, 22(6):1101-1127,1993. Hiei et al, Plant. Mol. Biol, 35(l-2):205-218,1997. Hinchee et al, Bio/technol, 6:915-922,1988. Horrobin^a/.,^/«. J. Clin. Nutr., 57(Suppl):732S-737S, 1993. Horschetal, Science, 223:496,1984. Horsch et al, Science, 227:1229,1985. Hon and Lin, Plant Physiology, 111:166,1996. Huang, Biqchem. Biophys. Acta, 1082:319,1991. Hudspeth and Grula, Plant Mol Biol, 12:579-589,1989. JJaitaetal,Bio/technol, 8:241-242,1990. hhidia etal, Nat. Biotechnol, 14(6):745-750,1996. JUP AC-IUB, Nad. Acid Res., 13:3021-3030,1985. James et al, Can. J. Physiol Pharmacol, 75:234,1997. James et al, Semin. Arthritis Rheum., 28:85,2000. Kaeppler et al, Plant Cell Reports, 9:415-418, 1990. Kaeppler et al, TJteor. Appl. Genet, 84(5-6):560-566,1992. Katz etal, J. Gen. Microbiol, 129:2703-2714,1983. Klee et al, Bio-Technology, 3(7):637-642,1985. Klein etal, Nature, 327:70,1987. Knittel et al, Plant Cell Reports, 14(2-3): 81-8 6,1994. Y^mtzon etal., Proc. Natl. Acad. Sci. USA, 89(7):2624-2628,1992. Kyte andDoolittle, J. Mol. Biol, 157:105-132, 1982.<br>
Lawto et al., Plant Mol. Biol. 9:315-324,1987. Lazzeri, Methods Mol Biol, 49:95-106,1995. Lee et al, Environ. Mol Mutagen., 13(l):54-59,1989. Lorz etal, Mol Gen Genet, 199:178-182,1985. Manzioris et al, Am. J. Clin. Nutr., 59:1304,1994. Marcotte et al, Nature, 335:454,1988. McCabe,Martinell, Bio-Technology, ll(5):596-598,1993. McConnac et al, Euphytica, 99(1): 17-25,1998. Meesapyodsuk et al, Biochemistry, 39(39):11948-11954,2000. Michaelis etal, Ann. Rev. Microbiol, 36:425,1982. Murakami et al, Mol. Gen. Genet, 205:42-50,1986. Nagatanief al,Biotech. Tech., ll(7):471-473,1997. Naylor et al, Nature, 405:1017,2000. Odell et al, Nature, 313:810-812,1985. Ogawaefa/., ScL Rep., 13:42-48, 1973. Omirulleh et al, Plant Mol Biol, 21(3):415-28,1993. Ow et al, Science, 234:856-859,1986. PCTAppl. WO 9217598 PCTAppl. WO 94/09699 PCTAppl. WO 95/06128 PCTAppl. WO 96/33155 PCTAppl. WO 97/4103 PCTAppl. WO 97/41228<br>
Potrykusera/.,Mo/. Gen. Genet, 199(2): 169-77,1985. Potrykusetal,Mol. Gen. Genet, 199:183-188,1985. Prasher etal.,Biochem. Biophys. Res. Commun., 126(3):1259-1268,1985. Reed etal, Plant Physiol, 122:715-720, 2000. Reichel et al, Proc. Natl. Acad. Set USA, 93 (12) p. 5888-5893.1996 Restrepo et al, Plant Cell, 2:987,1990. Rhodes et al, Methods Mol. Biol, 55:121-131,1995. Ritala et al, Plant Mol Biol, 24(2):317-325,1994. Rogers et al, Methods Enzymol., 153:253-277,1987.<br>
Sambrook et al, In: Moleadar cloning, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.<br>
Sheea     al, Plant Journal, 8(5):777-784,1995.<br>
Smiopoulos etal, Am. Coll.Nutr., 18:487,1999.<br>
Smgsit etal, Transgenic Res., 6(2):169-176,1997.<br>
Spencer etal, Plant Molecular Biology, 18:201-210,1992.<br>
Stalker et al, Science, 242:419-422, 1988.<br>
Sullivan etal.,Mol. Gen. Genet, 215(3):431-440,1989.<br>
Sutcliffe, Proc. Natl. Acad. Sci. USA, 75:3737-3741,1978.<br>
Thillet et al.,J. Biol Chem., 263:12500-12508,1988.<br>
Thompson etal., Euphytica, 85(l-3):75-80,1995.<br>
Thompson et al, The EMBO Journal, 6(9):2519-2523,1987.<br>
Tian etal, Plant Cell Rep., 16:267-271,1997.<br>
Tingay et al, Plant J., 11(6):1369-1376,1997.<br>
Tomes et al, Plant Mol Biol, 14(2):261-268,1990.<br>
Torbet et al, Crop Science, 38(1):226-231, 1998.<br>
Torbet et al, Plant Cell Reports, 14(10):635-640,1995.<br>
Toriyama et al, TheorAppl. Genet., 73:16,1986.<br>
Tsukada et al, Plant Cell Physiol, 30(4)599-604,1989.<br>
Twell et al, Plant Physiol, 91:1270-1274,1989.<br>
Uchimiya et al., Mol. Gen. Genet, 204:204, 1986.<br>
Van den Broeck et al, Nature, 313:358,1985.<br>
Van Eck, Blowers; Earle, Plant Cell Reports, 14(5):299-304, 1995.<br>
Vasil et al, Plant Physiol, 91:1575-1579,1989.<br>
Vasil et al, Plant Physiol, 91:1575-1579,1989.<br>
Walter et al, Proc. Natl. Acad. Sci. USA, 84:6624-6628,1987.<br>
Waavg etal,Molec. Cell Biol, 12(8):3399-3406,1992.<br>
wolk vet al, Proc. Natl Acad. Sci. USA, 1561-1565,1984.<br>
Yamada et al, Plant Cell Rep., 4:85,1986.<br>
Yamazaki et al, Biochem. Biophys. Acta, 1123:18,1992.<br>
Yang and Russell, Proc. Natl Acad. Sci. USA, 87:4144-4148, 1990.<br>
Zheng and Edwards, J. Gen. Virol, 71:1865-1868,1990.<br>
Zhou et al, Plant Cell Reports, 12(11):612-616,1993.<br>
Zukowsky et al., iVoc. Natl Acad. Sci. USA, 80:1101-1105,1983.<br><br><br><br><br><br><br>
WE CLAIM:<br>
1.	An isolated polynucleotide comprising a nucleic acid sequence that encodes a<br>
polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 15,<br>
wherein the polynucleotide is selected from the group consisting of:<br>
(a)	a polynucleotide encoding the polypeptide of SEQ ID NO: 3, SEQ ID NO: 5 or SEQ ID NO: 34;<br>
(b)	a polynucleotide comprising the nucleic acid sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO:4 or SEQ ID NO: 33;<br>
(c)	a polynucleotide hybridizing to one or more of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 4 or SEQ ID NO: 33, or a complement thereof, under conditions of 5X SSC, 50% formamide and 42°C.<br>
2.	The recombinant vector comprising the polynucleotide comprising a nucleic acid<br>
sequence that encodes a polypeptide having desaturase activity that desaturates a fatty acid<br>
molecule at carbon 15, wherein the polynucleotide is selected from the group consisting of:<br>
(a)	a polynucleotide encoding the polypeptide of SEQ ID NO: 3, SEQ ID NO: 5 or SEQ ID NO: 34;<br>
(b)	a polynucleotide comprising the nucleic acid sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO:4 or SEQ ID NO: 33;<br>
(c)	a polynucleotide hybridizing to one or more of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 4 or SEQ ID NO: 33, or a complement thereof, under conditions of 5X SSC, 50% formamide and 42°C; and<br>
(d)	a fungal polynucleotide encoding a polypeptide having at least one of the amino acid<br>
motifs: TrpIleLeuAlaHisGluCysGlyHisGlyAlaSerPhe (WILAHECGHGASF) (SEQ ID NO: 6);<br>
LeuAlaHisGluCysGlyHis	(LAHECGH)	(SEQ	ID	NO:	7);<br>
HisSerPheLeuLeuValProTyrPheSerTrpLys (HSFLLVPYFSWK) (SEQ ID NO: 8);<br>
LeuLeuValProTyrPheSerTrpLys (LLVPYFSWK) (SEQ ID NO:	9);<br>
His(His/Ala)ArgHisHisArg(Phe/Tyr)ThrThr (H(H/A)RHHR(F/Y)TT) (SEQ ID NO: 10, SEQ ID<br>
NO: 19, SEQ ID NO: 20, or SEQ ID NO: 21);<br>
TrpValHisHisTrpLeuValAlalleThr-TyrLeu(His/Gln)HisThrHis(WVHHWLVAITYL(H/Q)HTH)<br>
(SEQ ID NO: 11); AlaIleThrTyrLeu(His/Gln)HisThr (AITYL(H/Q)HT) (SEQ ID NO: 12);<br>
GlyAlaLeuAlaThrValAspArg        (GALATVDR)        (SEQ        ID        NO:         13)        or<br>
HisValValHisHisLeuPheXaaArglleProPheTyr (HWHHLFXRIPFY) (SEQ ID NO: 14 or SEQ ID NO: 22).<br>
3.	The recombinant vector as claimed in claim 2, wherein said polynucleotide is from a phyla selected from the group consisting of zygomycota, basidiomycota, and, ascomycota.<br>
4.	The recombinant vector as claimed in claim 2, wherein the polynucleotide is from a species selected from the group consisting of Neurospora crassa, Aspergillus nidulans and Botrytis cinerea.<br>
5.	The recombinant vector as claimed in claim 2, wherein the polynucleotide encodes a polypeptide having at least one of the amino acid motifs: TrpIleLeuAlaHisGluCysGlyHisGlyAlaSerPhe (WILAHECGHGASF) (SEQ ID NO: 6); LeuAlaHisGluCysGlyHis	(LAHECGH)	(SEQ	ID	NO:	7); HisSerPheLeuLeuValProTyrPheSerTrpLys (HSFLLVPYFSWK) (SEQ ID NO: 8); LeuLeuValProTyrPheSerTrpLys (LLVPYFSWK) (SEQ ID NO:	9); His(His/Ala)ArgHisHisArg(Phe/Tyr)ThrThr (H(H/A)RHHR(F/Y)TT) (SEQ ID NO: 10, SEQ ID NO: 19, SEQ ID NO: 20, or SEQ ID NO: 21); TrpValHisHisTrpLeuValAlalleThr-TyrLeu(His/Gln)HisThrHis(WVHHWLVAITYL(H/Q)HTH) (SEQ ID NO: 11); AlaIleThrTyrLeu(His/Gln)HisThr (AITYL(H/Q)HT) (SEQ ID NO: 12); GlyAlaLeuAlaThrValAspArg (GALATVDR) (SEQ ID NO: 13) or HisValValHisHisLeuPheXaaArglleProPheTyr (HWHHLFXRIPFY) (SEQ ID NO: 14 or SEQ ID NO: 22).<br>
6.	The recombinant vector as claimed in claim 2, wherein the polynucleotide encodes the polypeptide of SEQ ID NO: 3, SEQ ID NO:5 or SEQ ID NO:34.<br>
7.	The recombinant vector as claimed in claim 2, wherein the polynucleotide comprises the nucleic acid sequence of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 4 or SEQ ID NO: 33.<br>
8.	The recombinant vector as claimed in claim 2, wherein the polynucleotide hybridizes to one or more of SEQ ID NO: 1, SEQ ID NO: 2, SEQ ID NO: 4 or SEQ ID NO: 33 under conditions of 5X SSC, 50% formamide and 42°C.<br>
9.	The recombinant vector as claimed in claim 2, optionally comprising at least one<br>
additional sequence chosen from the group consisting of:<br>
(a)	regulatory sequences operatively linked to the polynucleotide;<br>
(b)	selection markers operatively linked to the polynucleotide;<br>
(c)	marker sequences operatively linked to the polynucleotide;<br>
(d)	a purification moiety operatively linked to the polynucleotide; and<br>
(e)	a targeting sequence operatively linked to the polynucleotide.<br>
(f)	promoter operably linked to said isolated polynucleotide.<br><br>
10.	The recombinant vector as claimed in claim 9, wherein the promoter is a developmentally regulated, organelle-specific, tissue-specific, constitutive or cell-specific promoter.<br>
11.	The recombinant vector as claimed in claim 9, wherein said promoter is selected from the group consisting of 35S CaMV, 34S FMV, Napin, 7S, Glob, and Lec.<br>
12.	The recombinant vector as claimed in claim 2, defined as an isolated expression cassette.<br>
13.	The recombinant vector as claimed in claim 2, optionally defined as comprising a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 6 and/or a nucleic acid sequence encoding a polypeptide having desaturase activity that 15 desaturates a fatty acid molecule at carbon 12.<br>
14.	An isolated polynucleotide comprising a nucleic acid sequence that encodes a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 12, wherein the polynucleotide is selected from the group consisting of.<br><br>
(a)	a polynucleotide encoding the polypeptide of SEQ ID NO: 32 or SEQ ID NO: 40;<br>
(b)	a polynucleotide comprising the nucleic acid sequence of SEQ ID NO: 31 or SEQ ID NO: 39; and<br>
(c)	a polynucleotide hybridizing to one or both of SEQ ID NO: 31 or SEQ ID NO: 39, or a complement thereof, under conditions of 5X SSC, 50% formamide and 42°C.<br><br>
15.	The isolated polynucleotide as claimed in claim 14, wherein said polynucleotide is from a phyla selected from the group consisting of zygomycota, basidiomycota, and ascomycota.<br>
16.	The isolated polynucleotide as claimed in claim 14, wherein the polynucleotide is from a species selected from the group consisting of Neurospora crassa and Botrytis cinerea.<br>
17.	A recombinant vector comprising the isolated polynucleotide as claimed in claim 14.<br>
18.	The recombinant vector as claimed in claim 17, further defined as comprising a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 6 and/or a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 15.<br>
19.	A fungal polypeptide or fragment thereof encoded by the recombinant vector as claimed in claim 2 having desaturase activity that desaturates a fatty acid molecule at carbon 15 and comprises at least one of the amino acid motifs: TrpIleLeuAlaHisGluCysGlyHisGlyAlaSerPhe (WILAHECGHGASF) (SEQ ID NO: 6); LeuAlaHisGluCysGlyHis (LAHECGH) (SEQ ID NO:7); HisSerPheLeuLeuValProTyrPheSerTrpLys (HSFLLVPYFSWK) (SEQ ID NO: 8); LeuLeuValProTyrPheSerTrpLys	(LLVPYFSWK)	(SEQ	ID	NO:9); His(His/Ala)ArgHisHisArg(Phe/Tyr)ThrThr (H(H/A)RHHR(F/Y)TT) (SEQ ID NO: 10, SEQ ID NO: 19, SEQ ID NO: 20, or SEQ ID NO: 21); TrpValHisHisTrpLeuValAlaIleThrTyrLeu(His/Gln)HisThrHis<br>
(WVHHWLVAITYL(H/Q)HTH) (SEQ ID NO: 11); AlaIleThrTyrLeu(His/Gln)HisThr (AITYL(H/Q)HT) (SEQ ID NO: 12); GlyAlaLeuAlaThrValAspArg (GALATVDR) (SEQ ID NO: 13) or HisValValHisHisLeuPheXaaArglleProPheTyr (HVVHHLFXRIPFY) (SEQ ID NO: 14 or SEQ ID NO: 22).<br>
20.	The fungal polypeptide as claimed in claim 19, optionally defined as comprising all of said amino acid motifs.<br>
21.	A fungal polypeptide encoded by the polynucleotide as claimed in claim 1 comprising the amino acid sequence of SEQ ID NO: 3, SEQ ID 25 NO: 5 or SEQ ID NO: 34; or a fragment thereof having desaturase activity that desaturates a fatty acid molecule at carbon 15.<br>
22.	A fungal polypeptide encoded by the polynucleotide as claimed in claim 14 comprising the amino acid sequence of SEQ ID NO: 32 or SEQ ID NO: 40; or a fragment thereof having desaturase activity that desaturates a fatty acid molecule at carbon 12.<br>
23.	A method of producing seed oil containing omega-3 fatty acids from plant seeds, comprising the steps of:<br>
(a) obtaining seeds of a plant transformed with the recombinant vector as claimed in<br>
claim 2; and<br>
(b) extracting the oil from said seeds.<br>
24.	A method of producing a plant comprising seed oil containing altered levels of omega-3 fatty acids comprising introducing the recombinant vector as claimed in claim 2 into an oil-producing plant.<br>
25.	The method as claimed in claim 24, wherein introducing the recombinant vector comprises plant breeding.<br>
26.	The method as claimed in claim 24, wherein introducing the recombinant vector comprises the steps of:<br><br>
(a)	transforming a plant cell with the recombinant vector as claimed in claim 2; and<br>
(b)	regenerating said plant from the plant cell, wherein the plant has altered levels of omega-3 fatty acids.<br><br>
27.	The method as claimed in claim 24, wherein the plant is a plant selected from the group consisting of Arabidopsis thaliana, oilseed Brassica, rapeseed, sunflower, saffiower, canola, corn, soybean, cotton, flax, jojoba, Chinese tallow tree, tobacco, cocoa, peanut, fruit plants, citrus plants, and plants producing nuts and berries<br>
28.	The method as claimed in claim 24, wherein the plant is further defined as transformed with a nucleic acid sequence encoding a polypeptide having desaturase activity that desaturates a fatty acid molecule at carbon 6.<br>
29.	The method as claimed in claim 28, wherein stearidonic acid is increased.<br>
30.	The method as claimed in claim 24, further defined as comprising introducing the recombinant vector as claimed in claim 2 into a plurality of oil-producing plants and screening said plants or progeny thereof having inherited the recombinant vector for a plant having a desired profile of omega-3 fatty acids.<br>
31.	A method of producing a plant comprising seed oil containing altered levels of omega-3 fatty acids comprising introducing the recombinant vector as claimed in claim 14 into an oil-producing plant.<br>
32.	The method as claimed in claim 31, wherein introducing the recombinant vector comprises plant breeding.<br>
33.	The method as claimed in claim 31, wherein introducing the recombinant vector comprises the steps of:<br><br>
(a)	transforming a plant cell with the recombinant vector as claimed in claim 14; and<br>
(b)	regenerating said plant from the plant cell, wherein the plant has altered levels of omega-3 fatty acids.<br><br>
34.	The method as claimed in claim 31, wherein the plant is a plant selected from the group consisting of Arabidopsis thaliana, oilseed Brassica, rapeseed, sunflower, safflower, canola, corn, soybean, cotton, flax, jojoba, Chinese tallow tree, tobacco, cocoa, peanut, fruit plants, citrus plants, and plants producing nuts and berries.<br>
35.	An endogenous canola seed oil having a stearidonic acid content of from about 8% to about 27% and an oleic acid content of from about 40% to about 70%.<br>
36.	The canola seed oil as claimed in claim 35, further defined as comprising less than 10% combined alpha-linoleic acid, linoleic acid and gamma-linoleic acid.<br>
37.	The canola seed oil as claimed in claim 35, wherein the stearidonic acid content is further defined as from about 10% to about 20%.<br>
38.	The canola seed oil as claimed in claim 35, wherein the stearidonic acid content is further defined as from about 12% to about 17%.<br>
39.	The canola seed oil as claimed in claim 35, wherein the oleic acid content is further defined as from about 45% to about 65%.<br>
40.	The canola seed oil as claimed in claim 35, wherein the oleic acid content is further defined as from about 55% to about 65%.<br>
41.	The canola seed oil as claimed in claim 35, wherein the stearidonic acid content is further<br>
defined as from about 12% to about 17% and the oleic acid content is further defined as from<br>
about 55% to about 65%.<br>
42.	The canola seed oil as claimed in claim 35, further defined as from Brassica napus seed.<br>
43.	The canola seed oil as claimed in claim 35, further defined as from Brassica rapa seed.<br>
44.	The canola seed oil as claimed in claim 35, wherein the ratio of omega-6 to omega-3 fatty acids in the oil is from about 1:1 to about 1:4.<br>
45.	The canola seed oil as claimed in claim 44, wherein the ratio of omega-6 to omega-3 fatty acids in the oil is further defined as from about 1:2 to about 1:4.<br>
46.	A method of increasing the nutritional value of an edible product for human or animal consumption, comprising adding the canola seed oil as claimed in claim 35 to the edible product.<br>
47.	The method as claimed in claim 46, wherein the edible product is human food.<br>
48.	The method as claimed in claim 46, wherein the edible product is animal feed.<br>
49.	The method as claimed in claim 46, wherein the edible product is a food supplement.<br>
50.	The method as claimed in claim 46, wherein the canola seed oil increases the stearidonic acid content of the edible product.<br>
51.	The method as claimed in claim 46, wherein the canola seed oil decreases the ratio of omega-6 to omega-3 fatty acids of the edible product.<br>
52.	The method as claimed in claim 46, wherein the edible product lacks stearidonic acid prior to adding the canola seed oil.<br>
53.	A method of manufacturing food or feed, comprising adding the canola seed oil as claimed in claim 35 to starting food or feed ingredients to produce the food or feed.<br>
54.	The method as claimed in claim 53, further defined as a method of manufacturing food.<br>
55.	The method as claimed in claim 53, further defined as a method of manufacturing feed.<br>
56.	Food or feed made by the method as claimed in claim 53.<br>
57.	A method of providing stearidonic acid to a human or animal, comprising administering the canola seed oil as claimed in claim 35 to said human or animal.<br>
58.	The method as claimed in claim 57, wherein the canola seed oil is administered in an edible<br>
composition.<br>
59.	The method as claimed in claim 58, wherein the edible composition is food or feed.<br>
60.	The method as claimed in claim 59, wherein the food comprises beverages, infused foods,<br>
sauces, condiments, salad dressings, fruit juices, syrups, desserts, icings and fillings, soft frozen<br>
products, confections or intermediate food.<br>
61.	The method as claimed in claims 58, wherein the edible composition is substantially a liquid.<br>
62.	The method as claimed in claim 58, wherein the edible composition is substantially a solid.<br>
63.	The method as claimed in claim 58, wherein the edible composition is a food supplement.<br>
64.	The method as claimed in claim 58, wherein the edible composition is a nutraceutical.<br>
65.	The method as claimed in claim 57, wherein the canola seed oil is administered to a human.<br>
66.	The method as claimed in claim 57, wherein the canola seed oil is administered to an animal.<br>
67.	The method as claimed in claim 66, wherein the canola seed oil is administered to livestock or poultry.<br>
68.	A method of producing a corn plant with increased a-linolenic acid (18:3, n-3) and/or<br>
stearidonic acid (18:4, n-3) levels comprising:<br>
(a)	transforming a corn plant cell with polynucleotides encoding A6-desaturase and A15-desaturase; and<br>
(b)	regenerating a corn plant with increased a-linolenic acid and/or stearidonic acid content from said plant cell.<br>
69.	A method of producing a corn oil containing omega-3 fatty acids from corn seeds,<br>
comprising the steps of:<br>
(a)	obtaining a plant transformed with polynucleotides encoding A6-desaturase and A15-desaturase<br>
(b)	crushing seeds produced from said plant; and<br>
(c)	extracting said oil from said seeds.<br><br>
70.	An oil produced by the method as claimed in claim 69.<br>
71.	An edible composition comprising a food stuff comprising an oil as set forth in claim 70.<br>
72.	The edible composition as claimed in claim 71, wherein the food stuff comprises beverages, infused, foods, sauces, condiments, salad dressings, fruit juices, syrups, desserts, icings and fillings, soft frozen products, confections and intermediate food.<br>
73.	The edible composition as claimed in claim 71, wherein the edible composition is substantially is liquid.<br>
74.	The edible composition as claimed in claim 71, wherein the edible composition is substantially a solid.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUFic3RyYWN0LSgyMi0wOS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Abstract-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUNsYWltcy0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Claims-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Correspondence Others-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjUtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Correspondence Others-(25-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjctMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Correspondence Others-(27-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTUtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Correspondence-Others-(15-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMTktMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-Correspondence-Others-(19-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjItMTAtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-Correspondence-Others-(22-10-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LURlc2NyaXB0aW9uIChDb21wbGV0ZSktKDIyLTA5LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Description (Complete)-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LURyYXdpbmdzLSgyMi0wOS0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Drawings-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUZvcm0tMS0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Form-1-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUZvcm0tMi0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Form-2-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUZvcm0tMy0oMTUtMDQtMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Form-3-(15-04-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LUZvcm0tMy0oMTktMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-Form-3-(19-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUZvcm0tMy0oMjUtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Form-3-(25-03-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUZvcm0tMy0oMjctMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Form-3-(27-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LUdQQS0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-GPA-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LXBjdC0yMTAucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LXBjdC0yMjAucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-pct-220.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1kZWxucC0yMDA0LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">3876-delnp-2004-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LVBldGl0aW9uIDEzNy0oMjItMDktMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Petition 137-(22-09-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=Mzg3Ni1ERUxOUC0yMDA0LVBldGl0aW9uIDEzNy0oMjUtMDMtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">3876-DELNP-2004-Petition 137-(25-03-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="255386-a-process-for-preparing-a-polymer-composition.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="255388-novel-crystalline-polymorphic-form-of-a-camptothecin-analouge.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>255387</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>3876/DELNP/2004</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>08/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>22-Feb-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>18-Feb-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>07-Dec-2004</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>MONSANTO TECHNOLOGY, LLC</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>800 NORTH LINDBERGH BLVD., ST. LOUIS, MO 63167, UNITED STATES OF AMERICA</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>VIRGINIA M. URSIN</td>
											<td>27232 MEADOWBROOK DR., DAVIS, CA 95616, USA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>TONI VOELKER</td>
											<td>1008 VILLANOVA, DAVIS, CA 95616, USA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>BYRON FORMAN</td>
											<td>1402 LOCUST PLACE, DAVIS, DA 95616, USA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>A01H 5/10</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2003/16144</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2003-05-21</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/382,391</td>
									<td>2002-05-22</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/453,125</td>
									<td>2003-03-07</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/255387-an-isolated-polynucleotide-and-recombinant-vector-encoding-a-polypeptide-having-desaturase-acitivity by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:42:39 GMT -->
</html>
