<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/255540-screening-method-for-identifying-pufa-pks-in-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:32:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 255540:SCREENING METHOD FOR IDENTIFYING PUFA PKS IN SAMPLES</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">SCREENING METHOD FOR IDENTIFYING PUFA PKS IN SAMPLES</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>SCREENING METHOD FOR IDENTIFYING PUFA PKS IN SAMPLES Abstract of the Invention (EN) The invention relates to a method for the rapid and simple identification of PUFA PKS in microorganisms. Said method is characterized by the fact that DNA sections representing PKS that produce specifically for PUFA (polyunsaturated fatty acids) are reproduced in vitro. The PUFA PKS-specific amino acid sequence LGIDSIKRVEIL makes it possible to derive oligonucleotides which are used for the efficient PCR screening for PUFA PKS genes or PUFA-producing microorganisms. The inventive method is particularly suitable for the high throughput screening of microorganisms for PUFA PKS genes.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>The invention describes a method for the rapid and simple identification of PUFA-PKS genes (PUFA - polyunsaturated fatty acids; PKS = polyketide synthase) in samples such as, e.g., biomass, especially in microorganisms. It is characterized by an in vitro reproduction of DNA sections specific for PUFA-producing PKS. The invention is also based, in addition to the identification of the appropriate DNA sequences, on the establishment of the experimental conditions for their multiplication.<br>
The term PUFAs (polyunsaturated fatty acids) denotes multiply unsaturated long-chain fatty acids with a chain length &gt; C12 and at least two double bonds. There are two main families of PUFA, which differ according to the position of the first double bond, relative to the alkyl end, in omega-3 and in omega-6 fatty acids. They are important components of cell membranes, where they are present in the form of lipids, especially phospholipids. PUFAs also function as preliminary stages of important molecules in humans and in animals such as, g, prostaglandins, leukotrienes and prostacyclins (A.P. Simopoulos, essential fatty Acids in health and chronic disease, Am. J. Clin. Nutr. 1999 (70), pp. 560-569), Important representatives of the group of omega-3 fatty acids are DHA (docosahexaenoic acid) and EPA (eicosapentaenoic acid), which can be found in fish oils .and in marine microorganisms. An important representative of omega-6 fatty acids is ARA (arachidonic acid) that occurs, e.g., in filamentary fungi but can also be isolated from animal tissues such as liver and kidney. DHA and ARA occur next to one another in human mother's milk.<br>
PUFAs are essential for a human as regards an appropriate development, in particular for the developing brain, tissue formation and its repair. Thus, DHA is an important component of human cell membranes, especially those of the nerves. Furthermore, DNA plays an important part in the maturing of brain function and is essential for the development of vision. Omega-3 PUFAs such as DHA and EPA are used as nutrient supplement since a balanced nourishment with a sufficient supply of DHA is advantageous for the prophylaxis of certain diseases {A.P. Simopoulos, Essential fatty acids in health and chronic disease, American Journal of Clinical<br>
Nutrition 1999 (70), pp.560-569). For example, adults with non-insulin-dependent diabetes exhibit a deficiency or at least an unbalanced DMA balance related to cardiac problems occurring later. Likewise, neuronal diseases such as, e.g., Alzheimer's or schizophrenia are accompanied by low DMA levels. There is a large number of sources for the commercial extraction of DMA, such as, e.g., oils from marine cold-water fish, egg yolk fractions or marine microorganisms. Microorganisms suitable for the extraction of n-3 PUFA are found, e.g., in bacteria in the in the*genus Vibrio (e.g., Vibrio marinus) or in the dinoflagellates (Dinophyta), in which in particular the genus Crypthecodinium, such as C. cohnii or in the Stramenopiles (or Labyrinthulomycota), such as the Pinguiophyceae such as, e.g., GJossomastix, Phaeomonas, Pinguiochrysis, Pinguiococcus and Polypodochrysis. Other preferred microorganisms for producing PUFA belong in particular to the order Thraustochytriales, (Thraustchytriidea) with the genera Japonochytrium, Schizochytrium, Thraustochytrium, Althornia, Labyrinthuloides, Aplanochytrium and<br>
.i.<br>
Ulkenia. Microorganisms of the genera Mortierella, Entomophthora, Phytium and Porphyridium are used for the commercial production of ARA.<br>
Commercially used sources for PUFA such as plants or animals are often characterized by a very heterogeneous composition of the oils extracted from them. The oils extracted in this manner must be subjected to expensive purification processes in order to be able to enrich one or several PUFAs. The supplying with PUFA from such sources is also subjected to uncontrollable fluctuations. Thus, disease and weather influences can reduce animal and also vegetable yields. The extraction of PUFA from fish is subject to seasonal fluctuations and can even be temporarily sharply limited due to overfishing or climatic changes (e.g., el Nifio). Animal oils, especially fish oils, can accumulate noxious substances from the environment via the food chain. It has become known that animals are highly stressed by organochJorides such as, e.g., polychlorinated biphenyls, in particular in commercial fish farms, that counteract the healthy aspects of fish consumption (Hites et al. 2004, Global assessment of organic contaminants in farmed salmon, Science 303, pp. 226-229). The resulting loss in quality of fish products results in a decreasing acceptance of consumers for fish and fish oils as omega-3 PUFA sources. Furthermore, the purification of DMA from fish is relatively expensive on account of high technical requirements. On the other hand, DMA is present in a few marine microorganisms in amounts of approximately 50% of the total fat component<br>
of the cell and they can be cultivated relatively economically in large fertnenters. Another advantage of microorganisms is a composition of the oils extracted from them that is limited to a few components.<br>
Two different biocatalytic paths are known for the biosynthesis of long-chain PL)FA. In the case of the so-called Sprecher Pathway, long-chain PUFAs such as DHA and EPA are synthesized starting from palmitic acid by a stream of elongation- and desaturation steps and terminating shortenings (H. Sprecher, Metabolism of highly unsaturated n-3 and n-6 fatty acids. Biochimica et Biophysica Acta 1486 (2000) pp. 219-231). This biosynthesis path is taken as described or in a similar manner in most organisms, even in humans and in plants. However, a certain number of marine organisms takes a different biosynthesis path for the production of EPA and DHA. These PUFA-producing microorganisms include marine representatives of gamma proteobacteria and, up to the present, the eukaryotic protist Schizochytrium. They synthesize long-chain PUFA via so-called polyketide synthases (PKS). These PKSs represent large enzymes that catalyze the synthesis of secondary metabolites consisting of ketide units (G.W. Wallis, J.L Watts and J. Browse, Polyunsaturated fatty acid synthesis: what will they think of next? Trends in Biochemical Sciences 27 (9) (2000) pp. 467-473). The synthesis of polyketides contains a number of enzymatic reactions that are analogous to those of fatty acid synthesis (Hopwood &amp; Sherman Annu. Rev. Genet. 24 (1990) pp. 37-66; Katz &amp; Donadio Annu. Rev. of Microbiol. 47 (1993) pp. 875-912).<br>
Gene sequences of different PUFA - PKSs (PUFA-synthesizing PKSs) are already known. Thus, a 38 kb genomic fragment was isolated from the marine bacterium Shewanella sp. that contains the information for the production of eicosapentaenoic acid (EPA). It was posstibie to produce EPA in E. coli and in Synechoccus by the transfer of the gene clusters contained in the genomic fragment. Subsequent sequencing of this fragment resulted in the identification of 8 open reading frames (ORFs. Open reading frames) (H. Takeyama et al., Microbiology 143 (1997) pp. 2725-2731). Five of these open reading frames from Shewanella are closely related to polyketide synthase genes. Further PKS-like gene clusters were also found in other PUFA-producing marine bacteria such as, e.g., Vibrio marinus (M. Tanaka, et al. 21 (1999) pp. 939-945). Analogous PUFA-producing, PKS-like ORFs were also able to be identified in the eukaryotic protist Schizochytrium (Metz et al, Science 293<br>
(2001) pp. 290-293 and WO 00/42195). Three ORFs were determined in Schizochytrium that display partial identities with the EPA gene cluster from Shewanella. The existence of these preserved PKS genes in a few prokaryotes and the eukaryote Schizochytrium furnishes an indication for a possible horizontal gene transfer of PUFA-PKS genes between pro- and eukaryotes.<br>
Very little is still known at present about the distribution of PKS between the individual species. Thus, e.g., a phylogenetically close relative of Schizochytrium, the marine protist Thraustochytrium sp., appears to have no PKS even though it is rich in DHA like Schizochytrium. It produces long-chain PUFA, among other things, using a very seldom occurring delta-4 desaturase (X Qiu et al. J. Biol. Chem. (2001) pp. 31561-31566). Both, Thraustochytrium and Schizochytrium, belong to the order Thraustochytriales but have totally different biosynthesis paths for the production of long-chain PUFA. Therefore, there is great interest in determining the distribution of PUFA-PKS genes in marine microorganisms, especially as regards the discovery Of new potential PUFA producers for the possible production of individual PUFAs on a commercial scale. In addition, there is currently a need for especially efficient screening methods in order to examine the large number of marine microorganisms with a high throughput for PUFA producers. Furthermore, a knowledge of many different PUFA-PKS should furnish information about the gene arrangement and structure of the corresponding enzymes and therewith for the production of different PUFAs. This is particularly important for many further applications such as, e.g., for the production of PUFA in transgenic microorganisms or plants. Designer oils with different PUFA combinations could be produced transgenically by the variation of genes.<br>
Patent application WO 02/083870 A2 describes a method for identifying organisms containing the PUFA-PKS genes. It is based on the one hand on five selection criteria concerning the fatty acid spectrum that should be given under certain cultivation conditions in order to function as indicator for a PUFA-PKS system. The more selection criteria are met, the stronger the indication for a PUFA-PKS system. It is based on the other hand on Southern blot analyses in which restriction-cleaved genomic DMA transferred onto blot membranes is hybridized from potential PUFA-PKS candidates (meeting of the five selection criteria) with PUFA-PKS-specific nucleic acid sequences. This second detection step was subsequently expanded in<br>
the exemplary embodiment for the verification of the result to the screening of a genomic DMA bank. Moreover, patent application WO 02/083870 describes strategies for the enrichment and selection of suitable microorganisms as a preselection for the above-cited screening method.<br>
However, it is apparent for those skilled in the art that the screening method described in WO 02/083870 A2 is very expensive and therefore unsuitable for^ high-throughput screening. Moreover, the evaluation of the selection parameters in the first screening step appears to be very vague, which can potentially result in negative results in the second screening step.<br>
The present invention therefore had the task in view of the state of the art of making available a method for identifying PUFA-PKS genes in different microorganisms. The method should make possible a broad screening of microorganisms efficiently, economically and in a short time. The screening should take place with a high throughput without expensive sample preparation.<br>
This task as well as other ones not explicitly cited but which can be readily derived or concluded from the initially discussed contexts in this document are solved by the subject matter defined in the claims of the present invention.<br>
An advantageous method for identifying PUFA-PKS genes in microorganisms is made available by the method defined in Claim 1, This method comprises the in vitro amplification of nucleic acids from samples, preferably biomass, especially from microorganisms, by means of polymerase chain reaction (PCR: polymerase chain reaction) using degenerated oligonucleotides (primers) derived from the amino acid succession LGIDSIKRVEIL (SEQ ID No. 5). A nucleic acid sequence derived from the amino acid succession LGIDSIKRVEIL (SEQ ID No. 5) represents, e.g., the sequence succession 5'- CTC GGC ATT GAC TCC ATC AAG CGT GTC GAG ATT CTC -3' (SEQ ID No. 6). The use in accordance with the invention of the degenerated primers described here leads to the identification of PKS gene fragments in PUFA-producing microorganisms.<br>
The method in accordance with the invention surprisingly makes do with oligonucleotides derived from only this one amino acid sequence section cited above.<br>
Furthermore and amazingly, heavily degenerated oligonucleotides containing a large number of N bases or, e.g., inosines, can be largely dispensed with.<br>
The method contains all oligonucleotides that can be derived from the above-cited amino acid sequence LGIDSIKRVEIL (SEQ ID No. 5) for amplifying and identifying PUFA-PKS genes. The selection of the oligonucleotides is independent of the length of the selected partial sequence and of its orientation (sense or antisense or complementary or non-complementary).<br>
In a preferred form oligonucleotides with a length of 10-36 bp, preferably 15-25 bp and especially preferably 18 bp are used in the detection method of the invention.<br>
The amount of oligonucleotides used can vary as long as there is no negative effect on the detection method for PUFA-PKS present. This also applied to all other components used in the PCR reaction.<br>
In a preferred embodiment the hybridization of the oligonucleotides used takes place at an annealing temperature of 45 °C - 65 °C, preferably 50 °C - 60 °C and especially preferably at 53 °C - 57 °C.<br>
The duration of the individual phases of the PCR, that is, of the denaturing, of the annealing and of the elongation can also vary as long as there is no negative effect on the detection method for PUFA-PKS present.<br>
The number of PCR cycles can also vary but is preferably between 20 and 40 cycles, especially preferably between 25 and 35 cycles and quite especially preferably approximately 30 cycles.<br>
All isolatable DNA and RNA nucleic acids from the microorganisms to be investigated as well as cDNA generated from mRNA can be used as template for the PCR. In a special embodiment entire cells or biomass can also be used as template for the PCR.<br>
In a further embodiment the oligonucleotides in accordance with the invention can also be used as hybridization probes for detecting complementary nucleic acid sequences.<br>
In particular, the method in accordance with the invention is suitable fo'r a high throughput screening of microorganisms for PUFA-PKS genes.<br>
Accordingly, the present invention also comprises a nucleic acid obtainable (identifiable) with the method of Claims 1 to 7 or by using the nucleic acid sequences according to Claim 9 as hybridization probes.<br>
'-•^<br>
A microorganism containing a nucleic acid according to Claim 11 is also comprised.<br>
Regardless of the great demand for PUFA-producing microorganisms, prior to the present invention there was no known efficient detection method based on PCR for identifying PUFA-PKS-containing microorganisms. A paper by Gentile et al. does describe the possible use of oligonucleotides for the amplification of PUFA-PKS gene sequences; however, the oligonucleotides described it are not derived from the AGP domains or from the amino acid sequence LGIDSIKRVEIL on which the invention is based (Gentile et al. 2003 J. Appl. Microbiol. (95) pp. 1124-1133).<br>
It is suspected that the amplification of a sequence section (APC domains of PKS) already present in multiple copy is the basis of the high efficiency of the PCR method described here. The presence of a large number of target sequences at the beginning of the PCR probably results in an increase of the efficiency. This results for its part in a higher hit ratio during screening. However, it was very surprising that specifically PUFA-PKS genes were able to be isolated with the aid of oligonucleotides derived from the amino acid sequence LGIDSIKRVEIL since APC domains occur in quite a number of other genes, e.g., PKS not specific for PUFA as well as peptide synthases and fatty acid synthases in general. This is also viewed as the reason that cloning tests of PUFA-PKS genes with derivation of oligos from the LGIDSIKRVEIL sequence were previously not attempted.<br>
Otherwise, up to the present only much more time-consuming and less reliable screening methods based on biomarkers for identifying PUFA producers were developed (D.S. Nichols and T.A. McMeekin 2002 J. Microbiol. Methods 48 (2-O, pp. 161-170).<br>
Figure 1    shows a comparison of the position and number of the acyl carrier protein domains of a few previously known PUFA-PKSs from Moritella marina,<br>
Photobacterium profundum (strain SS9) and Schizochytrium. The number of repetitions of the preserved sequence LGIDSIKRVEIL is also shown.<br>
Figure 2 shows the sequence homology of the PCR product (ACP domain), amplified with oligonucleotides, derived from the sequence LGIDSIKRVEIL, and from Ulkenia sp. SAM 2179 to the PUFA-PKS from Schizochytrium.<br>
The detection method constituting the basis of the method in accordance with the invention is described in the following using a few examples. However, the detection method and the invention are not limited to these examples.<br>
EXAMPLES Example 1:<br>
Amplification of a PUFA-PKS-specific sequence from isolated DNA from Ulkenia sp. SAM2179<br>
1.1 Isolation of genomic DNA<br>
50 ml DH1 medium (50 g/l glucose; 12.5 g/l yeast extract; 16.65 g/l Tropic Marin; pH 6.0) was inoculated in a 250 ml Erlenmeyer flask with flow spoiler with \Jlkenia sp. SAM 2179 (Ulkenia spec BP-5601; WO9803671) and cultivated '48 h at 28 °C and 150 rpm. The cells were subsequently washed with sterile tap water, centrifuged off and the cell sediment frozen at -85 °C. A cell mass of approximately 1.25 g dry weight was achieved. For the further workup the cell sediment was then transferred into a mortar and comminuted under liquid nitrogen with a pestle to a fine powder. Then, approximately 1/10th of the pulverized cell material was compounded with 2 ml lysis buffer (50 mM tris/CI pH 7.2; 50 mM EDTA; 3% (v/v) SDA; 0.01% (v/v) 2-mercaptoethanol) and incubated 1 h at 68 °C. 2 ml phenoi/chloroform/isoamylalcohol (25:24:1) were subsequently added, agitated and centrifuged 20 min at 10000 rpm. After removal of the upper aqueous phase the latter was transferred into two new reaction vessels at 600 ul each and again compounded with 600 ul each phenol/chloroform/isoamylalcohol (25:24:1), agitated and centrifuged 15 min at 13000 rpm. Each 400 ul of a particular upper phase was then transferred into a new reaction vessel and inverted two to three times after the addition of 1 ml ethanol<br>
(100%) in each instance. Then, the precipitated DNA was wound on a glass rod, washed with 70% ethanol, dried and dissolved in 50 M' H2OdiSt., compounded with 2 ul RNase A and stored at 4°C.<br>
1.2 PCR reaction using motive-specific oligonucleotides<br>
The PCR primers MOF1 and MOR1 were used as motive-specific oligonucleotides.<br>
"'•&gt;<br>
MOF1: 5' - CTC GGC ATT GAG TCC ATC - 3' (Seq ID No. 7) MOR1: 5'- GAG AAT CTC GAC ACG CTT - 3' (Seq ID No. 8)<br>
The genomic DNA from Ulkenia sp. SAM2179 described as in 1.1 was diluted 1:100. 2 ul of this dilution were then transferred into a 50 pi volume PCR reaction mixture (1 x buffer (Sigma); dNTPs (200 \iM each); MOF1 (20 pmol), MOR1 (20 pmol) and 2.5U Taq-DNA polymerase (Sigma). The PCR was carried out under the following conditions: Initial denaturing 94 °C for 3 min, followed subsequently by 30 cycles at 94 °C each for 1 min, 55 °C for 1 min, 72 °C 1 min and finally 8 min 72 °C. The PCR products were then analyzed by gel electrophoresis and fragments with an appropriate size incorporated into vector pCR2.1 TOPO via T/A cloning (Invitrogen). After transformation of E. coli TOP 10F1, plasmid DNA was isolated (Qiaprep Spin, QUAGEN) and sequenced.The sequence data obtained (SEQ ID No. 1) was compared with the officially accessible EMBL Nucleotide Sequence Database (http://www.ebi.ac.uik/embl/) and evaluated. The sequence comparisons obtained with FASTAX yielded for the main product of the PCR from Ulkenia sp. SAM 2179 a partial identity, that was approximately 90% on the amino acid level, with the acyl carrier protein of PUFA-PKS (ORF A; ORF: open reading frame) from Schizochytrium sp. ATCC 20888 (figure 7). Surprisingly, only a single PCR experiment had to be carried out in order to determine this PUFA-PKS in Ulkenia sp. SAM 2179.<br>
Example 2:<br>
Amplification of a PUFA-PKS-specific sequence from isolated DNA from Schizochytrium sp. SR21<br>
2.1	Isolation of genomic DNA<br>
50 ml DH1 medium (50 g/l glucose; 12.5 g/l yeast extract; 16.65 g/l TropksJMarin; pH 6/0) was inoculated in a 250 ml Erlenmeyer flask with flow spoiler with Schizochytrium sp. SR21 (Schizochytrium spec., MYA-1381; EP0823475) and cultivated 48 h at 28 °C and 150 rpm. The cells were subsequently washed twice with tap water, centrifuged off and the cell sediment frozen at -85 °C. A cell mass of approximately 1.4 g dry weight was obtained. Then, for the further workup the cell sediment was transferred into a mortar and treated as previously described (example 1) for isolating the genomic DNA.<br>
2.2	PCR reaction using motive-specific oligonucleotides<br>
The PCR primers MOF1 and MOR1 (see example 1) were used as motive-specific oligonucleotides.<br>
The PCR took place as described in 1.2 with 2 ul genomic DNA from Schizochytrium sp. SR21.<br>
The sequence data obtained (SEQ ID No. 2) was compared with the officially accessible EMBL Nucleotide Sequence Database (http://www.ebi.ac.uk/embl/) and evaluated. The sequence comparisons obtained with FASTAX yielded for the main product of the PCR from Schizochytrium sp. SR21 an approximately 90% partial identity with the acyl carrier protein of the PUFA-PKS (ORF A; ORF: open reading frame) from Schizochytrium sp. ATCC 20888. Surprisingly, only a single PCR experiment had to be carried out also for determining this PUFA-PKS in Schizochytrium sp. SR21.<br>
Example 3:<br>
Amplification of a PUFA-PKS-specific sequence directly from the biomass of Schizochytrium sp. SR21<br>
3.1	Obtention of biomass<br>
50 ml DH1 medium (50 g/l glucose; 12.5 g/l yeast extract; 16.65 g/l TropfcHVIarin; pH 6/0) was inoculated in a 250 ml Erlenmeyer flask with flow spoiler with Schizochytrium sp. SR21 and cultivated 48 h at 28 °C and 150 rpm. The cells were subsequently washed twice with tap water and centrifuged off. The biomass obtained in this manner was subsequently added directly into a corresponding PCR reaction.<br>
3.2	PCR reaction using motive-specific oligonucleotides<br>
The PCR primers MOF1 and MOR1 (see example 1) were used as motive-specific oligonucleotides.<br>
An aliquot of the biomass from Schizochytrium sp. SR21 obtained in 3,1 was taken up with a sterile toothpick and transferred into a 50 ul by volume PCR reaction mixture (1 x buffer (Sigma); dNTPs (200 uM each); MOF1 (20 pmol), MOR1 (20 pmol) and 2.5U Taq DMA polymerase (Sigma). The PCR was performed as described in point 1.2.<br>
The sequence data obtained (SEQ ID No. 2) was compared with the officially accessible EMBL Nucleotide Sequence Database (http://www.ebi.ac.uk/embl/) and evaluated. The sequence comparisons obtained with FASTAX yielded for the main product of the PCR from Schizochytrium sp. SR21 an approximately 90% partial identity with the acyl carrier protein of the PUFA-PKS (ORF A; ORF: open reading frame) from Schizochytrium sp. ATCC 20888. The sequence of the PCR product obtained from the biomass of Schizochytrium was identical to that in example 2.<br>
Surprisingly, only a single PCR experiment had to be carried out even here for determining the PUFA-PKS from the biomass of Schizochytrium sp. SR21.<br>
Example 4:<br>
Amplification of a PUFA-PKS-specific sequence directly from the biomass of different ulkenias<br>
4.1	Obtention of biomass<br>
50 ml DH1 medium (50 g/l glucose; 12.5 g/l yeast extract; 16.65 g/l Tropic* Marin; pH 6/0) was inoculated in a 250 ml Erlenmeyer flask with flow spoiler with either Ulkenia sp. SAM 2179 or Ulkenia visurgensis or another Ulkenia sp, and cultivated 48 h at 28 °C and 150 rpm. The cells were subsequently washed twice with tap water and centrifuged off. The biomass obtained in this manner was subsequently added directly into an appropriate PCR reaction.<br>
4.2	PCR reaction using motive-specific oligonucleotides<br>
The PCR primers MOF1 and MOR1 (see example 1) were used as motive-specific oligonucleotides.<br>
Aliquots of the biomasses from different ulkenias obtained in 4.1 were taken up with a sterile toothpick and each transferred into a 50 ul by volume PCR reaction mixture (1 x buffer (Sigma); dNTPs (200 uM each); MOF1 (20 pmol), MOR1 (20 pmol) and 2.5U Taq DNA polymerase (Sigma). The PCR was performed as described in point 1.2.<br>
The sequence data obtained (SEQ ID No. 1, 3 and 4) was compared with the officially accessible EMBL Nucleotide Sequence Database (http://www.ebi.ac. uk/embl/) and evaluated. The sequence comparisons obtained with FASTAX yielded high partial identities with the acyl carrier protein of the PUFA-PKS (ORF A; ORF: open reading frame) from Schizochytrium sp. ATCC 20888. The sequence of the PCR product obtained from the biomass of Ulkenia sp. SAM 2179 was identical to that in example 1.<br>
Surprisingly, only a single PCR experiment had to be carried out even here each time for determining the particular PUFA-PKS from the biomass of different ulkenias.<br><br>
1.	A method for the demonstration of PUFA-PKS gene-specific nucleic acid<br>
sequences in samples, in which<br>
(a)	A   polymerase   chain   reaction   (PCR)   is   performed   using   specific<br>
oligonucleotides and with a small part of the sample as template, and<br>
(b)	Obtained PCR products are sequenced and the obtained sequence<br>
information is compared with databanks in order to identify new PUFA-<br>
PKS sequence information by partial agreement with already known<br>
sequences,<br>
characterized in that the oligonucleotides used are derived from the amino acid sequence LGIDSIKRVEIL.<br>
2.	The method according to Claim 1, characterized in that the oligonucleotides are<br>
degenerated.<br>
3.	The method according to any one of the preceding claims, characterized in that<br>
the  length of the oligonucleotides is  10-36 bp,  preferably 15-25 bp and<br>
especially preferably 18 bp.<br>
4.	The method according to any one of the preceding claims, characterized in that<br>
the amount of oligonucleotides used is preferably approximately 20 pmol.<br>
5.	The method according to any one of the preceding claims, characterized in that<br>
the annealing temperature is approximately 45-65 °C, preferably approximately<br>
50-60 °C and especially preferably approximately 53-57 °C.<br>
6.	The method according to any one of the preceding claims, characterized in that<br>
the number of cycles is approximately 20-40, preferably approximately 25-35<br>
and especially preferably approximately 30.<br>
7.	The method according to any one of the preceding claims, characterized in that<br>
nucleic acids isolated from the biomass or the biomass itself is/are used as<br>
template for the PCR.<br>
8.	The amino acid sequence LGIDSIKRVEIL represented in SEQ ID No. 5.<br><br>
9.	Nucleic acid sequences derivable from SEQ ID No. 5, preferably the nucleic<br>
acids represented in SEQ ID No. 6, 7 and 8.<br>
10.	The nucleic acid according to Claim 9 for use as hybridization probes.<br>
1 1 .   The nucleic acid obtainable (identifiable) with the method of Claims 1 to 7 or by using the nucleic acid sequences according to Claim 9 as hybridization probes.<br>
1 --»<br>
12.   A microorganism containing a nucleic acid according to Claim 11.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LUNsYWltcy0oMjMtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Claims-(23-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjMtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Correspondence Others-(23-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Correspondence Others-(28-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycyAoMjUtMDItMjAwOCkucGRm" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-correspondence-others (25-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWNvcnJlc3BvbmVkZW5jLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-corresponedenc-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LURlc2NyaXB0aW9uIChDb21wbGV0ZS0oMjMtMDgtMjAxMikpLnBkZg==" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Description (Complete-(23-08-2012)).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LUZvcm0tMTMtKDIzLTA4LTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Form-13-(23-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tMTggKDI1LTAyLTIwMDgpLnBkZg==" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-18 (25-02-2008).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tMjYucGRm" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-26.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1ERUxOUC0yMDA2LUZvcm0tMy0oMjgtMDItMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">6375-DELNP-2006-Form-3-(28-02-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LXBjdC0zMDgucGRm" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-pct-308.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LXBjdC1ub3RpZmljYXRpb24ucGRm" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-pct-notification.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=NjM3NS1kZWxucC0yMDA2LXBjdC1zZWFyY2ggcmVwb3J0LnBkZg==" target="_blank" style="word-wrap:break-word;">6375-delnp-2006-pct-search report.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=YWJzdHJhY3QuanBn" target="_blank" style="word-wrap:break-word;">abstract.jpg</a></p>
		<br>
		<div class="pull-left">
			<a href="255539-a-process-for-the-preparation-of-dehydrated-bamboo-shoots-with-improved-reconstitution.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="255541-nanostructures-formed-of-branched-nanowhiskers-and-methods-of-producing-the-same.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>255540</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>6375/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>09/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>01-Mar-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>01-Mar-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>30-Oct-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>NUTRINOVA NUTRITION SPECIALITIES &amp; FOOD INGREDIENTS GMBH</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>INDUSTRIEPARK HOCHST GEB,D706,65926 FRANKFURT AM MAIN (DE)</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>LUY,MARKUS</td>
											<td>FONTANEWEG 9,61267 NEU-ANSPACH,GERMANY</td>
										</tr>
										<tr>
											<td>2</td>
											<td>RUSING,MATTHIAS</td>
											<td>FREILIGRATHSTRASSE 50,50935 KOLN,GERMANY</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12Q 1/68</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/EP2005/003702</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2005-04-08</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10 2004 017 369,9</td>
									<td>2004-04-08</td>
								    <td>Germany</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/255540-screening-method-for-identifying-pufa-pks-in-samples by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 10:32:26 GMT -->
</html>
