<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256154-methods-for-the-production-of-insulin-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:51:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256154:METHODS FOR THE PRODUCTION OF INSULIN IN PLANTS</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">METHODS FOR THE PRODUCTION OF INSULIN IN PLANTS</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>Methods for the production of insulin in plants are described. In one embodiment, the present invention provides a method for the expression of insulin in plants comprising: (a) providing a chimeric nucleic acid construct comprising in the 5&#x27; supl; to 3&#x27; direction of transcription as operably linked components: (i) a nucleic acid sequence capable of controlling expression in plant seed cells; and (ii) a nucleic acid sequence encoding an insulin polypeptide; (b) introducing the chimeric nucleic acid construct into a plant cell; and (c) growing the plant cell into a mature plant capable of setting seed wherein the seed expresses insulin.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>Title: METHODS FOR THE PRODUCTION OF INSULIN FIELD OF THE INVENTION<br>
The  present invention  relates to plant genetic engineering methods and to the production of insulin. More specifically, the present invention relates to methods for the production of insulin in the seeds of plants. BACKGROUND OF THE INVENTION<br>
Insulin is an important peptide hormone required to maintain blood glucose homeostasis in mammals, including humans, and other vertebrates. In healthy individuals an increase in blood glucose level stimulates the p-cells of the pancreas to secrete insulin. The insulin polypeptide then binds to specific receptors in muscle, liver, and adipose tissue leading to an increase in glucose uptake by these targeted tissues, an increase in metabolism, and a decrease in hepatic glucose production. The cumulative effects of these responses serve to keep blood glucose concentrations at a constant level.<br>
In individuals suffering from diabetes mellitus, an abnormally low insulin concentration presents itself as chronic hyperglycemia. The clinical manifestations of chronic hyperglycemia are manifold, and include blindness, kidney failure and, if left untreated, will ultimately result in death. Estimates place diabetes mellitus as the third largest cause of death in industrialized countries, after cardiovascular diseases and cancer (Barfoed, H.C., 1987, Chem. Eng. Prog. 83:49-54). In order to allow efficient uptake and metabolism of blood glucose by the cells, diabetic individuals may be treated by the routine administration of insulin. Approximately 0.7% of the world's population suffers from insulin-dependent diabetes (diabetes mellitus Type I) (Winter, J. et al., 2000, J. of Biotechnol. 84:175-185). In addition, it is estimated<br>
that the number of individuals diagnosed with diabetes will double to approximately 300 million, in the next 25 years (Kjeldsen, T. et al., 2001, Biotechnol. Gen. Eng. Rev. 18:89-121). Consequently, the ability to cost effectively manufacture human insulin in quantities to satisfy the anticipated growing world demand for insulin is highly desirable.<br>
In vivo the human insulin polypeptide is produced by the pancreatic p-cells as a single 110 amino acid polypeptide chain precursor, preproinsulin, which includes an N-terminally located 24 amino acid pre-sequence that is cleaved immediately upon completion of the chain's biosynthesis (Steiner, D.F. 2000. J. Ped. Endocrinol. Metab. 13:229-239). Proinsulin consists of a B and A chain, linked by a connecting peptide (C-peptide). During packaging of the hormone for secretion the C-peptide is cleaved and removed by prohormone convertases, PC2 and PC1/PC3 (Steiner, D.F. 2000. J. Fed. Endocrinol. Metab. 13:229-239). What remains is mature human insulin, a 51 amino acid protein consisting of two polypeptide chains, A (21 amino acids in length) and B (30 amino acids in length), linked by two inter-chain disulphide bonds. Additionally, the A chain comprises one intra-chain disulphide bond.<br>
Human insulin has been prepared using a variety of different methodologies. Microorganisms such as Escherichia coli (Frank et al., 1981, in Peptides: Proceedings of the 7th American Peptide Chemistry Symposium (Rich &amp; Gross, eds.), Pierce Chemical Co., Rockford. Ill pp 729-739; Chan et al., 1981, Proc Natl. Acad. Sci. USA 78: 5401-5404), Saccharomyces cerevisiae (Thim et al., 1986, Proc. Natl. Acad. Sci. USA 83: 6766-6770) are routinely employed to recombinantly produce insulin. Wang et al. (Biotechnol. Bioeng., 2001, 73:74-79) have shown that fungi, such as Pichia pastoris, are also suitable for insulin production. Alternative manufacturing options<br>
include production in non-human mammalian cell lines (Yanagita, M., et al., 1992, FEBS Lett 311:55-59), isolation from human pancreas, peptide synthesis, or the semisynthetic conversion to human insulin from porcine and bovine insulin. However, all of these methods suffer from lower yields and higher costs than desired.<br>
The use of plants as bioreactors for the large scale production of recombinant proteins is well known, and numerous proteins, including human therapeutic proteins, have been produced. For example, US Patents 4,956,282, 5,550,038 and 5,629,175 disclose the production of γ-interferon in plants; US Patents 5,650,307, 5,716,802 and 5,763,748 detail the production of human serum albumin in plants and US Patents 5,202,422, 5,639,947 and 5,959,177 relate to the production of antibodies in plants. One of the significant advantages offered by plant-based recombinant protein production systems is that by increasing the acreage of plants grown, protein Preduction can be inexpensively scaled up to provide for large quantities of tein. By contrast, fermentation and cell culture systems have large space, equipment and energy requirements, rendering scale-up of production costly. However, despite the fact that the use of plants as bioreactors is amply documented, and despite the above mentioned anticipated prodigious increase in need for large volumes of insulin, the prior art provides only a limited number of methods which demonstrably result in the production of insulin in plants (see: Arakawa et al. Nature Biotech., 1998,16: 934 - 938; PCT 01/72959).<br>
Arakawa et al. disclose the production of a fusion protein comprising insulin in the tubers of transgenic potato plants. However insulin represents only up to 0.05% of the total soluble protein content present in the transgenic tubers. At a level of 0.05% of total soluble protein, large amounts<br>
of biomass must be subjected to protein extraction rendering the production economics associated with the use of potato tubers unfavorable. Furthermore, Arakawa et al. are not concerned with the isolation of insulin from the potato tuber tissue, but instead suggest an approach prevent the onset of Type I diabetes by inducing immunotolerance which involves oral administration of insulin through the feeding of transgenic potato tubers.<br>
PCT Patent Application WO 01/72959 discloses the production of a fusion protein comprising insulin in chloroplasts of transgenic tobacco. However, while purportedly addressing shortcomings with respect to the accumulation levels of human proteins in plant tissue, the invention to which WO 01/72959 pertains is limited in that production in chloroplasts results in the accumulation of-insulin in green tissue, primarily the tobacco leaves. Due to the relatively high water content of green tissue, a large amount of biomass must be processed. Furthermore production of insulin would require immediate extraction from the biomass upon harvesting, as leaf material will rapidly deteriorate when stored.<br>
Thus in view of the shortcomings associated with the methods for the recombinant production of insulin in plants provided by the prior art, it is presently unclear whether and how the synthetic capacity of plants may be harnessed to achieve the commercial production of insulin in plants. There is a need in the art to improve methods for the commercial production of insulin in plants. SUMMARY OF THE INVENTION<br>
The present invention relates to improved methods for the production of insulin in plants. In particular the present invention relates to methods for the production of insulin in seeds.<br>
Accordingly, the present invention provides a method for the expression of insulin in plants comprising:<br>
(a)	providing    a    chimeric    nucleic    acid    construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii) a nucleic acid sequence encoding an insulin polypeptide;<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell; and<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin.<br>
In a preferred embodiment of the present invention the nucleic sequence capable of controlling expression in plant seed cells is a seed-preferred promoter, such as a phaseolin promoter.<br>
In a preferred embodiment of the present invention insulin is expressed a manner that permits accumulation of the insulin polypeptide in a membrane enclosed intracellular compartment within the seed cells. Accordingly the present invention provides a method for the expression of insulin in plants which comprises:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii)      a nucleic acid sequence encoding an insulin<br>
polypeptide; and (iii)     a     nucleic     acid     sequence    encoding    a<br>
polypeptide capable of retaining the insulin<br>
polypeptide     in     a     membrane     enclosed<br>
intracellular compartment<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell; and<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin.<br>
In a further preferred embodiment of the present invention, the membrane enclosed intracellular compartment is the endoplasmic reticulum (ER) or an ER-derived storage vesicle. Accordingly, the present invention provides a method for the expression of insulin in plants which comprises:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii) a nucleic acid sequence encoding an insulin polypeptide;<br>
(iii) a nucleic acid sequence encoding a polypeptide capable of retaining the insulin polypeptide in the ER or in an ER derived storage vesicle<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell; and<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin.<br>
In a further preferred embodiment the chimeric nucleic acid construct is introduced into the plant cell under nuclear genomic integration conditions. Under such conditions the chimeric nucleic acid sequence is stably integrated in the plant's genome.<br>
In a yet further preferred embodiment the nucleic acid sequence encoding insulin is optimized for plant codon usage and the nucleic acid sequence encoding the connecting peptide (C-peptide) is shortened. Preferred nucleic acid sequences used in accordance with the present invention encode human, bovine or porcine insulin. In accordance with the present invention a nucleic acid sequence encoding a proinsulin sequence is used wherein the proinsulin is modified in that the C-peptide is shortened in length.<br>
In another aspect, the present invention provides a method of recovering plant seeds comprising insulin. Accordingly, pursuant to the present invention a method is provided for obtaining plant seeds comprising insulin comprising:<br>
(a)	providing    a    chimeric    nucleic    acid    construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i)	a nucleic acid sequence capable of<br>
controlling expression in plant seed cells; and<br>
(ii)	a nucleic acid sequence encoding an insulin<br>
polypeptide;<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell;<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed; and<br>
(d)	obtaining seeds from said plant wherein the seed<br>
comprises insulin.<br>
Preferably, at least 0.1% of the total seed protein present in the seed is insulin.<br>
The seeds may be used to obtain a population of progeny plants each comprising a plurality of seeds expressing insulin.<br>
The present invention also provides plants capable of setting seed expressing insulin. In a preferred embodiment of the invention, the plants capable of setting seed comprise a chimeric nucleic acid sequence comprising in the 5' to 3' direction of transcription:<br>
(a)	a first nucleic acid sequence capable of controlling<br>
expression in a plant seed cell operatively linked to;<br>
(b)	a second nucleic acid sequence encoding an insulin<br>
polypeptide, wherein the seed contains insulin.<br>
Preferably, at least 0.1% of the total seed protein present in the seed is insulin.<br>
In a preferred embodiment the chimeric nucleic acid sequence is integrated in the plant's nuclear genome.<br>
In a further preferred embodiment of the present invention the plant that is used is a safflower, a flax plant or an Arabidopsis plant.<br>
In yet another aspect, the present invention provides plant seeds expressing insulin. In a preferred embodiment of the present invention, the plant seeds comprise a chimeric nucleic acid sequence comprising in the 5' to 3' direction of transcription:<br><br>
(a)	a first nucleic acid sequence capable of controlling<br>
expression in a plant seed cell operatively linked to;<br>
(b)	a second nucleic acid sequence encoding an insulin<br>
polypeptide.<br>
Preferably, at least 0.1% of the total seed protein present in the seed is insulin. The seeds are a source whence the desired insulin polypeptide, which is synthesized by the seed cells, may be extracted and the insulin may be used to treat diabetic patients.<br>
Other features and advantages of the present invention will become readily apparent from the following detailed description. It should be understood, however, that the detailed description and the specific examples while indicating preferred embodiments of the invention, are given by way of illustration only, since various changes and modifications within the spirit and scope of the invention will become readily apparent to those skilled in the art from this detailed description. BRIEF DESCRIPTION OF THE DRAWINGS<br>
The invention will now be described in relation to the drawings in which:<br>
Figure 1 depicts the nucleotide sequence (SEQ ID NO. 1) and deduced amino acid sequence (SEQ ID NO. 2) of the insulin fusion protein (PRS-D9scFv-Klip27-MI-KDEL) of pSBS4404. The predicted amino acid sequence is shown in single letter code. The deduced amino acid sequence of the PRS signal peptide is in italics, the deduced amino acid sequence of the D9 scFv is in bold, the deduced amino acid sequence of the KLIP 27 sequence is underlined, the deduced amino acid sequence of the mini-insulin sequence is in italics and bold and finally the KDEL sequence is bolded and underlined.<br><br>
Figure 2 depicts the nucleotide sequence (SEQ ID NO. 3) and deduced ammo acid sequence (SEQ ID NO. 4) of the insulin fusion protein (OLEO-KLIP8-KLIP27-MI) of PSBS4405. The predicted amino acid sequences are shown in single letter code. The deduced amino acid sequence of the Arabidopsis thaliana 18 kDa oleosin is in italics, the deduced amino acid sequence of the KLIP 8 sequence is in bold, the deduced amino acid sequence of the KLIP27 sequence is underlined and the deduced amino acid sequence of the mini-insulin is in italics and bold.<br>
Figure 3 depicts the complete nucleic acid sequence (SEQ ID NO:5) and amino acid sequence (SEQ ID NO:6) of the 4414 insulin fusion protein (PRS-MI-tetrabasic linker-D9Scfv-KDEL). The predicted amino acid sequences are shown in single letter code. The deduced amino acid sequence of the PRS signal peptide is in italics, the deduced amino acid sequence of the mini-insulin (B30 tetrabasic) is in bold, the deduced amino acid sequence of the tetra-basic linker sequence is underlined, the deduced amino acid sequence of the D9 scFv is in italics and bold and finally the KDEL sequence is bolded and underlined.<br>
Figures 4(A-D) depicts recombinant expression of insulin fusion proteins in transformed Arabidopsis thaliana lines (4404-2, -17, -20, and 4405-4) on the basis of Coomassie-stained SDS-PAGE and Western Blot analysis. The arrows denote the position of the migrating 38.5 kDa and 34.2 kDa fusion polypeptides, PRS-D9(scfv)-KLIP27-MIw/KDEL and OLEO-KLIP8-KLIP27-MI respectively, under reducing conditions. Figures 4A (Coomassie-stained gel) and 4B (corresponding Western Blot probed with anti-insulin E2E3) depicts total seed protein from wild type (wt) and transgenic seed lines expressing the 4404 and 4405 constructs. Figures 4B (Coomassie-stained gel) and 4D (corresponding Western Blot probed with anti-insulin E2E3) depicts oil body<br>
protein prepared from wild type and transgenic seed expressing the same 4404 and 4405 constructs. Figures 4 (E-F) depicts the recombinant expression of insulin fusion proteins in transformed Arabidopsis thaliana lines (4419-9 and 4414-20) on the basis of Coomassie-stained SDS-PAGE and Western Blot analysis. The molecular weight markers (M) are 10, 15, 25, 37, 50, 75,100, 150 kDa. Controls include, hIN (recombinant human insulin standard) and hProIN (recombinant human proinsulin standard), separated under non-reducing conditions.<br>
Figure 5 depicts determined expression levels in the available T3 seed lines (4404-2, -17, -20, 4405-4, -13, -19) and T2 seed lines (4414-9 and-20). The levels of transgene and % molar MI expression were determined on the basis of densitometry.<br>
Figure 6 depicts Coomassie-stained SDS-PAGE (15%) analysis of oil body preps prior to elution (-OB), OB prep after elution with formic acid (-OB'), and the concentrated eluted material (-E). The arrow denotes the position of the migrating fusion polypeptide. The wild type control is essentially free of any major proteins following elution whereas the concentrated 4404 material contains the fusion protein, some truncated products (possible hydrolyzed fusion protein) and possibly some albumins that co-eluted.<br>
Figures 7 depicts the chromatograms depicting the characteristic retention times of the human insulin standard in comparison to the trypsin cleaved eluted 4404 fusion protein on the C18 column. The hIN standard is recombinant human insulin standard (0.5 μg).<br>
Figure 8 depicts the mass spectral analysis of human insulin standard (A) in comparison to trypsin cleaved and HPLC purified 4404 (B) fractions collected from 17.0-17.5 minutes.<br>
Figure 9 depicts the Coomassie-stained SDS-PAGE (15%) analysis of total extractable seed protein and oil body (OB) prepared protein from lines expressing 4405 in comparison to wild type (nonrecombinant) seed. The arrow denotes the position of the migrating fusion polypeptide.<br>
Figures 10 depicts the chromatograms depicting the characteristic retention times of the human insulin standard in comparison to the trypsin cleaved 4405 OB preparations by RP-HPLC on the C18 column. The hIN standard is recombinant human insulin standard (0.5μg).<br>
Figure 11 depicts the mass spectral analysis of human insulin standard (A) in comparison to trypsin cleaved and HPLC purified 4405 (B) fractions collected from 17.0-17.5 minutes.<br>
Figure 12 depicts the chromatogram of trypsin cleaved 4405 oil body preparations (dashed line) in comparison to human insulin standard (solid line). Fractions of the eluted cleaved insulin was collected between 7-35 mS/cm and concentrated by lyophilization for insulin bioassay.<br>
Figure 13 depicts the changes in serum glucose levels in male B6 mice following injection of negative controls (open circles= saline placebo, closed circles^ trypsin cleaved wild type oil bodies), positive controls (open squares = Humulin R, open triangles = Roche hIN) in comparison to plant derived insulin prepared from 4405 oil bodies (closed diamonds= SBS hIN DesB30).<br>
Figure 14 depicts a Coomassie-stained gel of oil body proteins from two representative lines (4409-6 and 4409-8) comparing the migration of Oleosin-hPIN fusion protein (as denoted by the black arrow) to non-transformed (wt) Arabidopsis. The level of expression was determined by densitometry to measure on average about 0.10% of total seed protein. This level was calculated above and beyond the co-migration of an endogenous<br>
protein of the same molecular weight in the non-transformed seed (wt) which constituted approximately 0.04% of the total seed protein. DETAILED DESCRIPTION OF THE INVENTION<br>
As hereinbefore mentioned, the present invention relates to improved methods for the production of insulin in transgenic plants. The present inventors have surprisingly found that levels of insulin accumulation exceeding 0.1% of total cellular protein may be achieved in plants by recombinantly producing insulin in the seeds of plants. These expression levels, which are at least ten times higher as those heretofore achieved, render commercial production of insulin in plants viable. Production in seeds offers flexibility in storage and shipment of insulin as a raw material, since insulin retains its activity upon extraction from stored seed. Furthermore, the amount of biomass that needs to be subjected to extraction is limited due to the relatively low water content present in plant seeds.<br>
Accordingly, pursuant to the present invention a method for the expression insulin in plants is provided in which the method comprises:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii) a nucleic acid sequence encoding an insulin polypeptide;<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell; and<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin.In accordance with the present invention it has surprisingly been found that insulin accumulates to levels in plant seeds heretofore not achieved if insulin is expressed in seed in a manner that permits sequestration of the insulin polypeptide within the seed cells in a membrane enclosed intracellular compartment. Accordingly, pursuant to the present invention a preferred method for the expression of insulin in plants is provided in which the method comprises:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii) a nucleic acid sequence encoding an insulin polypeptide;<br>
(iii) a nucleic acid sequence encoding a polypeptide capable of retaining the insulin polypeptide in a membrane enclosed intracellular compartment<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell; and<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin.<br>
Terms and Definitions<br>
Unless defined otherwise, all technical and scientific terms used herein shall have the same meaning as is commonly understood by one skilled in the art to which the present invention belongs. Where permitted, all patents, applications, published applications, and other publications,<br>
including nucleic acid and polypeptide sequences from GenBank, SwissPro and other databases referred to in the disclosure are incorporated by reference in their entirety.<br>
The term "nucleic acid sequence" as used herein refers to a sequence of nucleoside or nucleotide monomers consisting of naturally occurring bases, sugars and intersugar (backbone) linkages. The term also includes modified or substituted sequences comprising non-naturally occurring monomers or portions thereof. The nucleic acid sequences of the present invention may be deoxyribonucleic acid sequences (DNA) or ribonucleic acid sequences (RNA) and may include naturally occurring bases including adenine, guanine, cytosine, thymidine and uracil. The sequences may also contain modified bases. Examples of such modified bases include aza and deaza adenine, guanine, cytosine, thymidine and uracil; and xanthine and hypoxanthine.<br>
The terms "nucleic acid sequence encoding insulin" and "nucleic acid sequence encoding an insulin polypeptide", which may be used interchangeably herein, refer to any and all nucleic acid sequences encoding an insulin polypeptide, including the insulin polypeptides listed in Table 1 (SEQ ID NO: 7 to 145) as well as any mammalian insulin polypeptide and any nucleic acid sequences that encode pro-insulin and preproinsulin. As used herein "proinsulin" refers to an insulin polypeptide which includes the connecting peptide or "C-peptide" linking the B and A insulin polypeptide chains. In native human insulin the C-peptide is the 31 amino acid residue polypeptide chain connecting residue B30 to residue A1. The term "preproinsulin" refers to a proinsulin molecule additionally comprising an N-terminal signal sequence which directs translation to occur on the ER ribosomes. Nucleic acid sequences encoding an insulin polypeptide further<br>
include any and all nucleic acid sequences which (i) encode polypeptides that are substantially identical to the insulin polypeptide sequences set forth herein; or (ii) hybridize to any nucleic acid sequences set forth herein under at least moderately stringent hybridization conditions or which would hybridize thereto under at least moderately stringent conditions but for the use of synonymous codons.<br>
By the term "substantially identical" it is meant that two polypeptide sequences preferably are at least 75% identical, and more preferably are at least 85% identical and most preferably at least 95% identical, for example 96%, 97%, 98% or 99% identical. In order to determine the percentage of identity between two polypeptide sequences the amino acid sequences of such two sequences are aligned, preferably using the Clustal W algorithm (Thompson, JD, Higgins DG, Gibson TJ, 1994, Nucleic Acids Res. 22 (22): 4673-4680, together with BLOSUM 62 scoring matrix (Henikoff S. and Henikoff J.G., 1992, Proc. Natl. Acad. Sci. USA 89: 10915 - 10919) and a gap opening penalty of 10 and gap extension penalty of 0.1, so that the highest order match is obtained between two sequences wherein at least 50% of the total length of one of the sequences is involved in the alignment. Other methods that may be used to align sequences are the alignment method of Needleman and Wunsch (J. Mol. Biol., 1970, 48: 443), as revised by Smith and Waterman (Adv. Appl. Math., 1981, 2: 482) so that the highest order match is obtained between the two sequences and the number of identical amino acids is determined between the two sequences. Other methods to calculate the percentage identity between two amino acid sequences are generally art recognized and include, for example, those described by Carillo and Lipton (SIAM J. Applied Math., 1988, 48:1073) and those described in Computational Molecular Biology, Lesk, e.d. Oxford University Press, New York, 1988,<br>
Biocomputing: Informatics and Genomics Projects. Generally, computer programs will be employed for such calculations. Computer programs that may be used in this regard include, but are not limited to, GCG (Devereux et al., Nucleic Acids Res., 1984,12: 387) BLASTP, BLASTN and FASTA (Altschul et al., J. Molec. Biol., 1990: 215: 403).<br>
By "At least moderately stringent hybridization conditions" it is meant that conditions are selected which promote selective hybridization between two complementary nucleic acid molecules in solution. Hybridization may occur to all or a portion of a nucleic acid sequence molecule. The hybridizing portion is typically at least 15 (e.g. 20, 25, 30, 40 or 50) nucleotides in length. Those skilled in the art will recognize that the stability of a nucleic acid duplex, or hybrids, is determined by the Tm, which in sodium containing buffers is a function of the sodium ion concentration and temperature (Tm = 81.5°C - 16.6 (Logic [Na+]) + 0.41(%(G+C) - 600/1), or similar equation). Accordingly, the parameters in the wash conditions that determine hybrid stability are sodium ion concentration and temperature. In order to identify molecules that are similar, but not identical, to a known nucleic acid molecule a 1% mismatch may be assumed to result in about a 1°C decrease in Tm, for example if nucleic acid molecules are sought that have a &gt;95% identity, the final wash temperature will be reduced by about 5°C. Based on these considerations those skilled in the art will be able to readily select appropriate hybridization conditions. In preferred embodiments, stringent hybridization conditions are selected. By way of example the following conditions may be employed to achieve stringent hybridization: hybridization at 5 x sodium chloride/sodium citrate (SSC)/5 x Denhardt's solution/1.0% SDS at Tm - 5°C based on the above equation , followed by a wash of 0.2 x SSC/0.1% SDS at 60°C . Moderately stringent hybridization<br>
conditions include a washing step in 3 x SSC at 42°C. It is understood however that equivalent stringencies may be achieved using alternative buffers, salts and temperatures. Additional guidance regarding hybridization conditions may be found in: Current Protocols in Molecular Biology, John Wiley &amp; Sons, N.Y., 1989, 6.3.1. - 6.3.6 and in: Sambrook et al., Molecular Cloning, a Laboratory Manual, Cold Spring Harbor Laboratory Press, 1989, Vol.3.<br>
As used herein the terms "insulin" and "insulin polypeptide", which may be used interchangeable herein, refer to any and all insulin polypeptides including the insulin polypeptides listed in Table 1 (SEQ ID NO: 7 to 145) as well as a polypeptide molecule comprising a sequence of amino acid residues which is (i) substantially identical to the amino acid sequences constituting any insulin polypeptides set forth herein or (ii) encoded by a nucleic acid sequence capable of hybridizing under at least moderately stringent conditions to any nucleic acid sequence encoding insulin set forth herein or capable of hybridizing under at least moderately stringent conditions to any nucleic acid sequence encoding insulin set forth herein but for the use of synonymous codons. The terms insulin and insulin polypeptide include pro-insulin polypeptides and mini-insulin polypeptides. The insulin polypeptide is preferably of human, porcine or bovine origin.<br>
The term "polypeptide capable of retaining the insulin polypeptide within a membrane enclosed intracellular compartment" as used herein refers to any polypeptide, which when linked to an insulin polypeptide, is capable of sequestering the insulin polypeptide in a subcellular structure surrounded by a membrane and located within the intracellular space of the plant cell as defined by the plant cell's plasma membrane.<br><br>
The term "a polypeptide capable of retaining the insulin polypeptide in the ER or in an ER derived storage vesicle" as used herein refers to any polypeptide, which when linked to an insulin polypeptide is capable of sequestering the insulin polypeptide either in the endoplasmic reticulum or in a storage compartment which is derived from an endoplasmic reticulum, such as for example an oil body, within a plant cell.<br>
The term "oil body" or "oil bodies" as used herein refers to any oil or fat storage organelle in a plant seed cell (described in for example: Huang (1992) Ann. Rev. Plant Mol. Biol. 43:177-200).<br>
The term "chimeric" as used herein in the context of nucleic acid sequences refers to at least two linked nucleic acid sequences which are not naturally linked. Chimeric nucleic acid sequences include linked nucleic acid sequences of different natural origins. For example a nucleic acid sequence constituting a plant promoter linked to a nucleic acid sequence encoding human insulin is considered chimeric. Chimeric nucleic acid sequences also may comprise nucleic acid sequences of the same natural origin, provided they are not naturally linked. For example a nucleic acid sequence constituting a promoter obtained from a particular cell-type may be linked to a nucleic acid sequence encoding a polypeptide obtained from that same cell-type, but not normally linked to the nucleic acid sequence constituting the promoter. Chimeric nucleic acid sequences also include nucleic acid sequences comprising any naturally occurring nucleic acid sequence linked to any non-naturally occurring nucleic acid sequence. Preparation of recombinant expression vectors comprising chimeric nucleic acid sequences encoding insulin and a promoter capable of controlling expression in a plant seed cell<br>
The nucleic acid sequences encoding insulin that may be used in accordance with the methods and compositions provided herein may be any nucleic acid sequence encoding an insulin polypeptide, including any proinsulin and preproinsulin.<br>
Exemplary nucleic acid sequences encoding insulin are well known to the art and are generally readily available from a diverse variety of mammalian sources including human (Bell, G.I. et al., 1980, Nature 284:26-32), porcine (Chance, R.E. et al., 1968, Science 161:165-167), bovine (D'Agostino, J. et al., 1987, Mol. Endocrinol. 1:327-331), ovine (Peterson, J.D. et al., 1972, Biol. Chem. 247:4866-4871) and the like, as well as from plant sources (Oliveira, A.E.A. et al., 1999, Protein Pept. Lett. 6:15-21). Insulin encoding sequences that may used include those encoding polypeptide chains set forth as SEQ ID NO:7 to SEQ ID NO: 145. The respective corresponding nucleic acid sequences encoding the insulin polypeptide chains can be readily identified via the Swiss Protein identifier numbers provided in Table 1. Using these nucleic acid sequences, additional novel insulin encoding nucleic acid sequences may be readily identified using techniques known to those of skill in the art. For example libraries, such as expression libraries, cDNA and genomic libraries, may be screened, and databases containing sequence information from sequencing projects may be searched for similar sequences. Alternative methods to isolate additional nucleic acid sequences encoding insulin polypeptides may be used, and novel sequences may be discovered and used in accordance with the present invention. In preferred embodiments nucleic acid sequences encoding insulin are human, porcine and bovine insulin.<br>
Numerous insulin analogs are known to the prior art (see for example US Patents 5,461,031; 5,474,978; 5,164,366 and 5,008,241) and may be<br><br>
used in accordance with the present invention. Analogs that may be used herein include human insulin molecules in which amino acid residue 28 of the B-chain (B28) has been changed from its natural proline residue into aspartate, lysine or isoleucine. In another embodiment the lysine residue at B29 is modified to a proline. Furthermore, the aspargine at A21 may be changed to alanine, glutamine, glutamate, glycine, histidine, isoleucine, leucine, methionine, serine, threonine, tryptophan, tyrosine or valine. Also, aspargine at B3 may be modified to lysine. Further examples of insulin analogs that may used herein include: human insulin lacking the B30 residue, also frequently referred to as "desBSO" or "B(l-29)"; those lacking the last 3 amino acid residues insulin "B(l-27)"; insulin molecules lacking the phenylalanine residue at Bl; and analogs wherein the A-chain or the B-chain have an N-terminal or C-terminal extension, for example the B-chain may be N-terminally extended by the addition of two arginine residues.<br>
In preferred embodiments, the nucleic acid sequence encoding insulin that is used is proinsulin. In further preferred embodiments nucleic acid sequence molecules encoding insulin are used in which the C-peptide has been modified relative to its native form. Amino acid residues in the C-peptide may be substituted and the C-peptide may be lengthened or shortened. In this regard, as used herein the term "mini-insulin" refers to an insulin polypeptide which has been modified so that the C-peptide has been shortened in length relative to its native form. In preferred embodiments a mini-insulin is used. Preferably the C-peptide of a mini-insulin molecule will be shorter than 20 amino acid residues, more preferably shorter than 15 amino acid residues and most preferably shorter than 9 amino acid residues, for example 7, 5, or 3 residues. Preferably, as is the case with natural insulin molecules, the mini-insulin C-peptide will comprise a cleavage site at its C-<br><br><br>
and N-termini. Such cleavage sites may be any convenient sites known to the art, for example methionine, cleavable by cyanogen bromide, a single basic residue or a pair of basic residues, cleavable by trypsin or trypsin like proteases or by a carboxy-peptidase. For example, the C-peptide may comprise a C-terminal lysine, for example Ala-Ala-Lys (SEQ ID NO: 146), or a dibasic processing site immediately prior to the GlyAl residue, for example Asn-Lys-Arg (SEQ ID NO: 147), or Arg-Arg-Lys-Gln-Lys-Arg. (SEQ ID NO: 148) or a tetrabasic processing site immediately prior to the Gly Al residue, for example Arg-Arg-Lys-Arg (SEQ ID NO:149). Accordingly, mini-insulin molecules that may be used in accordance with the present invention include:<br>
B(l-29/30)-Xi-X2-X3-Yi-A(l-21) wherein<br>
X1 is any amino acid;<br>
X2 is any amino acid;<br>
X3 is Lys or Arg;<br>
Y1 is peptide bond or 1-17 amino acid residues;<br>
B(l-29/30) is the B chain of the human insulin B-chain containing<br>
amino acid residues 1-29 or 1-30; and<br>
A(l-21) is the A chain of the human insulin A-chain containing<br>
amino acid residues 1-21.<br>
In a preferred embodiment, X1 is a basic amino acid residue (Lys or Arg) and Y1 is either a peptide bond or 1-17 amino acid residues where the C-terminal residue is a basic amino acid residue (Lys or Arg).<br>
Further, mini-insulin molecules that may be used herein comprise those that may be represented by the formula:<br>
B(l-27)-X2-X3-X1-Y-A(l-21)<br><br>
wherein<br>
X1 is a peptide of 1-8 amino acid residues comprising at least<br>
one aromatic amino acid residue;<br>
X2 is one of Pro, Asp, Lys or I1e at position 28 of the B-chain;<br>
X3 is one of Pro, Lys, Ala, Arg or Pro-Thr at position 29 of the B-<br>
chain;<br>
Y is Lys or Arg;<br>
B(l-27) is the B chain of the human insulin B-chain containing<br>
amino acid residues 1-27; and<br>
A(l-21) is the A chain of the human insulin A-chain containing<br>
amino acid residues 1-21.<br>
Additional examples of nucleic acid molecules encoding a mini-insulin polypeptides that may be used in accordance with the present invention include those described in: Markussen et al, Walter de Gruyter &amp;Co. 1987, in: Peptides pp 189-194; Thim et al., 1989, in: Genetics and molecular biology of industrial microorganisms, American Society for Microbiology pp 322-328; and those set forth in US Patents 4,916,212; 5,324,641 and 6,521,738. Alterations to the nucleic acid sequence encoding insulin to prepare insulin analogs may be made using a variety of nucleic acid modification techniques known to those skilled in the art, including for example site directed mutagenesis, targeted mutagenesis, random mutagenesis, the addition of organic solvents, gene shuffling or a combination of these and other techniques known to those of skill in the art (Shraishi et al., 1988, Arch. Biochem. Biophys, 358:104-115; Galkin et al., 1997, Protein Eng. 10: 687-690; Carugo et al., 1997, Proteins 28: 10-28; Hurley et al., 1996, Biochem, 35 : 5670-5678; Holmberg et al., 1999, Protein Eng. 12: 851-856).<br><br>
In accordance with the present invention it has surprisingly been found that insulin accumulates to levels in plant seeds heretofore not achieved if the insulin is expressed in seed, preferably in such a manner that the insulin polypeptide within the seed cells is sequestered in a membrane enclosed intracellular compartment. In preferred embodiments of the present invention the insulin polypeptide is sequestered in the ER or in an ER derived storage vesicle. In order to achieve such sequestration of insulin in the ER or an ER derived storage vesicle, in accordance with the present invention, the polypeptide encoding insulin is linked to a polypeptide which causes the insulin polypeptide to be retained in the ER or in an ER-derived storage vesicle rather than being transported out of the ER to, for example, the apoplast. Polypeptides that may be used in accordance with the present invention to retain the insulin polypeptide in the ER include any polypeptide capable of sequestering the insulin in the ER. Such polypeptides may be synthesized or obtained from any biological source. In a preferred embodiment of the present invention, the polypeptide that is capable of retaining the insulin is a polypeptide comprising a C-terminal ER retention motif. Examples of such C-terminal ER retention motifs include KDEL, HDEL, DDEL, ADEL and SDEL sequences (SEQ ID NO:150 to 154 respectively). Other examples include HDEF (SEQ ID NO:155) (Lehmann et al., 2001. Plant Physiol 127(2): 436-49.), or two arginine residues close to the N-terminus located at positions 2 and 3, 3 and 4, or 4 and 5 (Abstract from Plant Biology 2001 Program, ASPB, July 2001, Providence, Rhode Island, USA). Nucleic acid sequences encoding a C-terminal ER retention motif are preferably linked to the nucleic acid sequence encoding the insulin polypeptide in such a manner that the polypeptide capable of retaining the insulin in the ER is linked to the C-terminal end of the insulin polypeptide.<br>
In order to achieve sequestration of the insulin polypeptide in an ER derived storage vesicle, the insulin polypeptide is linked to a polypeptide that is capable of retaining the insulin polypeptide in an ER derived storage vesicle. The polypeptide capable of retaining the insulin polypeptide in an ER-derived storage vesicle that may be used in accordance with the methods herein may be any polypeptide capable of sequestering the insulin polypeptide in an ER-derived storage vesicle. Polypeptides capable of retaining insulin in an ER derived storage vesicle may be synthesized or obtained from any biological source. In a preferred embodiment the ER-derived storage vesicle is an oil body and the insulin polypeptide is linked to an oil body protein or a sufficient portion thereof capable of retaining the insulin polypeptide in the ER-derived storage vesicle. Oil body proteins that may be used in this regard include any protein that naturally associates with an oil body. Oil body proteins that are particularly preferred are oleosins, for example an Arabidopsis oleosin (van Rooijen et al. (1991) Plant Mol Biol. 18: 1177-1179) corn oleosin (Bowman-Vance et al., 1987, J. Biol. Chem. 262:11275-11279; Qu et al., 1990, J. Biol. Chem. 265: 2238-2243), carrot oleosin (Hatzopoulos et al. (1990) Plant Cell 2: 457-457) or Brassica oleosin (Lee et al., 1991, Plant Physiol. 96: 1395-1397), caleosins, see for example Genbank accession number AF067857) and steroleosins (Lin et al., 2002 Plant Physiol. 128(4):1200-11). In a further preferred embodiment, the oil body protein is a plant oleosin and shares sequence similarity with other plant oleosins such as the oleosin isolated from Arabidopsis thaliana (SEQ ID NO:156) or Brassica napus (SEQ ID NO:157). In another embodiment, the oil body protein is a caleosin or calcium binding protein from plant, fungal or other sources and shares sequence homology with plant caleosins such as the caleosin isolated from Arabidopsis thaliana (SEQ ID NO:158 and SEQ ID NO:159) In another<br><br>
embodiment the oil body protein is a steroleosin (SEQ ID NO.-160), a sterol binding dehydrogenase (Lin L-J et al, (2002) Plant Physiol 128: 1200-1211). The polypeptide encoding insulin may be linked to the oil body protein to the N-terminus as well as to the C-terminus and to fragments of an oil body protein, such as for example the central domain of an oleosin. New oil body proteins may be discovered for example by preparing oil bodies (for methodologies to prepare oil bodis see for example US Patent 6,650,554) and identifying proteins in oil body preparations through for example SDS gel electrophoresis. Polyclonal antibodies may be raised against these proteins and used to screen cDNA libraries in order to identify nucleic acid sequences encoding oil body proteins. New oil body proteins further may be discovered using known nucleic acid sequences encoding oil body proteins, using for example the hereinbefore mentioned oil body protein sequences encoding oil body proteins, to probe for example cDNA or genomic libraries for the presence of oil body proteins.<br>
Polypeptides capable of retaining the insulin in the ER or an ER derived storage organelle are typically not cleaved and the insulin may accumulate in the form of a fusion protein, which is, for example, typically the case when a KDEL retention signal is used to retain the polypeptide in the ER or when an oil body protein is used to retain the polypeptide in an ER derived storage organelle.<br>
The chimeric nucleic acid sequence additionally may contain a nucleic sequence which targets the nucleic acid sequence to the endomembrane system ("signal peptide"). In embodiments of the present invention in which the insulin polypeptide is retained in the ER using a sequence capable of retaining the polypeptide in the ER, such as KDEL, HDEL or SDEL polypeptide, it is particularly desirable to include a nucleic<br><br>
acid sequence encoding a signal peptide. Exemplary signal peptides that may be used herein include the tobacco pathogenesis related protein (PR-S) signal sequence (SEQ.ID.NO:161) (Sijmons et al., 1990, Bio/technology, 8:217-221), lectin signal sequence (Boehn et al., 2000, Transgenic Res, 9(6):477-86), signal sequence from the hydroxyproline-rich glycoprotein from Phaseolus vulgaris (Van et al., 1997, Plant Phyiol. 115(3):915-24 and Corbin et al., 1987, Mol Cell Biol 7(12):4337-44), potato patatin signal sequence (Iturriaga, G et al., 1989, Plant Cell 1:381-390 and Bevan et al., 1986, Nuc. Acids Res. 41:4625-4638.) and the barley alpha amylase signal sequence (Rasmussen and Johansson, 1992, Plant Mol. Biol. 18(2):423-7). Such targeting signals may in vivo be cleaved off from the insulin sequence, which for example is typically the case when an apoplast targeting signal, such as the tobacco pathogenesis related protein-S (PR-S) signal sequence (Sijmons et al., 1990, Bio/technology, 8:217-221) is used. Other signal peptides can be predicted using the SignalP World Wide Web server (http://www.cbs.dtu.dk/services/SignalP/) which predicts the presence and location of signal peptide cleavage sites in amino acid sequences from different organisms. In general there is little conservation of the primary amino acid sequence, although general physiochemical properties are conserved to some extent. The generic structure of signal peptides has 3 regions, the short amino-terminal "n-region" contains positively charged residues, the central hydrophobic "h-region" ranges in size from 7 to 15 amino acids and the carboxy-terminal "c-region" contains polar amino acids and a cleavage site that is recognized by membrane bound signal peptidase enzymes (Nakai K., 2000, Advances in Protein Chem 54:277-344). A targeting signal that also may be used in accordance herewith includes the native insulin signal sequence (24 amino acids in length in case of the human sequence). In preferred embodiments<br>
hereof an N-terminally located apoplast targeting sequence, such as the hereinbefore mentioned tobacco PR-S sequence is used combined with a C-terminally located ER retention sequence such as the KDEL sequence.<br>
In a further preferred embodiment, a nucleic acid sequence encoding a yeast a-factor leader sequence is linked to the N-terminal end of the nucleic acid sequence encoding insulin. Yeast leader sequences or sequences derived from yeast leader sequences that may be used in accordance herewith include those listed in SEQ ID NO: 162 to SEQ ID NO: 171 (Kjeldsen et al., 2001, Biotechnology and Genetic Engineering Reviews 18: 89-121). Such leader sequences may further comprise a spacer peptide located C-terminally of the nucleic acid encoding the leader sequence and N-terminally of the sequence encoding insulin. In accordance herewith such spacer sequences typically are between 2 and 20 amino acids in length. Thus, for example, spacer sequences SEQ ID NO: 172 and SEQ ID NO: 173 (Kjeldsen et al., 2001, Biotechnology and Genetic Engineering Reviews 18: 89-121) may be used. In embodiments of the present invention in which a yeast leader sequence is used, the nucleic acid sequence encoding the insulin polypeptide is preferably a mini-insulin polypeptide. In accordance herewith, in a particularly preferred embodiment a nucleic acid sequence encoding a single-chain antibody linked to a nucleic acid sequence encoding a yeast secretion leader peptide is used, as further described in Example 1 hereof.<br>
The chimeric nucleic acid sequence may also comprise polypeptides resulting in N-and/or C-terminal stabilizing protein extensions. Such extensions may be used to stabilize and/or assist in folding of the insulin polypeptide chain and additionally may be used to facilitate purification of insulin. Polypeptide extensions that may be used in this regard include for example a nucleic acid sequence encoding a single chain antibody, a nucleic<br>
acid encoding an Affibody® molecule (Affibody AB), a nucleic acid sequence encoding the non-toxic B subunit of cholera toxin (CTB) (Arakawa, T. et al., 1998, Nat. Biotechnol. 16:938) or combinations of such polypeptides. In a particularly preferred embodiment, the insulin polypeptide is retained in a membrane enclosed compartment, such as the ER, using for example a KDEL sequence as hereinbefore described, combined with a stabilizing polypeptide that permits the association of the insulin polypeptide with an oil body upon breakage of the integrity of the plant cell such as will occur when the insulin polypeptide is recovered from the plant cell. An example of such a stabilizing polypeptide is a single chain antibody with specificity for an oil body. Nucleic acid sequences encoding single chain antibodies with specificity for an oil body may be prepared from hybridoma cell lines expressing monoclonal antibodies raised against an oil body protein. In one embodiment, the single chain antibody specifically binds an oleosin, as described by Alting-Mees et al. (2000) IBC's International Conference on Antibody Engineering, Poster #1. This embodiment of the present invention is further detailed in Example 1 hereof.<br>
In a further embodiment, a cleavage site may be located upstream of the N-terminus and downstream the C-terminus of the insulin allowing for the insulin polypeptide to be cleaved from the fusion partner, thereby obtaining isolated insulin. Examples of such cleavage sites can be found in WO 98/49326 (Method for the cleavage of fusion proteins) and related applications and LaVallie et al. (1994) Enzymatic and chemical cleavage of fusion proteins In Current Protocols in Molecular Biology pp 16.4.5-16.4.17, John Wiley and Sons, Inc., New York NY. In a preferred embodiment, the cleavage site is a tetrabasic linker (for example Arg-Arg-Lys-Arg - SEQ ID NO: 149) which is cleavable by trypsin. In a further<br><br><br>
is<br>
preferred embodiment, the cleavage site is KLIP 8 (SEQ ID NO: 174) which i cleavable by aspartic proteases including chymosin.<br>
The invention further provides methods for the separation of heterologous proteins from host cell components by partitioning of the oil body fraction and subsequent release of the heterologous protein via specific cleavage of the heterologous protein - oil body protein fusion. Optionally a cleavage site may be located upstream of the N-terminus and downstream of the C-terminus of the heterologous polypeptide allowing the fusion polypeptide to be cleaved and separated by phase separation into its component pep tides.<br>
The nucleic acid sequence encoding insulin may be altered, to further improve expression levels for example, by optimizing the nucleic acid sequence in accordance with the preferred codon usage for a particular plant cell type which is selected for the expression of the insulin polypeptide, or by altering motifs known to destabilize mRNAs (see for example: PCT Patent Application 97/02352). Comparison of the codon usage of the nucleic acid sequence encoding the insulin polypeptide with the codon usage of the plant cell type will enable the identification of codons that may be changed. Construction of synthetic genes by altering the codon usage is described in for example PCT Patent Application 93/07278.<br>
In a preferred embodiment, the nucleic acid sequence encoding insulin that is used is represented by SEQ ID NO:1, SEQ ID NO:3, SEQ ID NO:5, or SEQ ID NO: 195.<br>
In accordance herewith the nucleic acid sequence encoding insulin is linked to a promoter capable of controlling expression of the insulin polypeptide in a plant seed cell. Accordingly, the present invention also provides a nucleic acid sequence encoding insulin linked to a promoter<br><br>
capable of controlling expression in a plant seed cell. Promoters that may be used herein will be generally art recognized and include any plant derived promoter capable of controlling expression of polypeptides in plants. Generally, promoters obtained from dicotyledonous plant species will be used when a dicotyledonous plant is selected in accordance herewith, while a monocotyledonous plant promoter will be used when a monocotyledonous plant species is selected. Constitutive promoters that may be used include, for example, the 35S cauliflower mosaic virus (CaMV) promoter (Rothstein et al., 1987, Gene 53: 153-161), the rice actin promoter (McElroy et al., 1990, Plant Cell 2: 163-171; US Patent 6,429,357), a ubiquitin promoter, such as the corn ubiquitin promoter (US Patents 5,879,903; 5,273,894), and the parsley ubiquitin promoter (Kawalleck, P. et al., 1993, Plant Mol. Biol. 21:673-684).<br>
In preferred embodiments, the promoter that is used is a promoter which results in preferential expression of the insulin polypeptide in seed tissue. "Seed-preferred promoters" in this regard are promoters which control expression of a recombinant protein'(i.e. insulin) so that preferably at least 80% of the total amount of recombinant protein present in the mature plant is present in the seed. More preferably at least 90% of the total amount of recombinant protein present in the mature plant is present in the seed. Most preferably at least 95% of the total amount of recombinant protein present in the mature plant is present in the seed. Seed-preferred promoters that may be used in this regard include, for example, the bean phaseolin promoter (Sengupta-Gopalan et al., 1985, Proc. Natl. Acad. Sci. USA 82: 3320-3324); the Arabidopsis 18 kDa oleosin promoter (US Patent 5,792,922) or the flax oleosin promoter (WO 01/16340); the flax legumin like seed storage protein (linin) promoter (WO 01/16340); the flax 2S storage protein promoter (WO 01/16340); an endosperm preferred promoter such as the Amy32b<br>
promoter (Rogers and Milliman, J. Biol. Chem., 1984, 259: 12234-12240, the Amy6-4 promoter (Kursheed and Rogers, J. Biol. Chem., 1988, 263: 18953-18960 or the Aleurain promoter (Whittier et al., 1987, Nucleic Acids Res., 15: 2515-2535) or the bean arcelin promoter Qaeger GD, et al., 2002, Nat. Biotechnol. Dec; 20:1265-8). New promoters useful in various plants are constantly discovered. Numerous examples of plant promoters may be found in Ohamuro et al. (Biochem. of Pints., 1989,15: 1-82).<br>
Certain genetic elements capable of enhancing expression of the insulin polypeptide may be used herein. These elements include the untranslated leader sequences from certain viruses, such as the AMV leader sequence (Jobling and Gehrke, 1987, Nature, 325: 622-625) and the intron associated with the maize ubiquitin promoter (US Patent 5,504,200). Generally the chimeric nucleic acid sequence will be prepared so that genetic elements capable of enhancing expression will be located 5' to the nucleic acid sequence encoding the insulin polypeptide.<br>
In accordance with the present invention the chimeric nucleic acid sequences comprising a promoter capable of controlling expression in plant seeds linked to a nucleic acid sequence encoding an insulin polypeptide can be integrated into a recombinant expression vector which ensures good expression in the seed cell. Accordingly, the present invention includes a recombinant expression vector comprising in the 5' to 3' direction of transcription as operably linked components:<br>
(i)       a nucleic acid sequence capable of controlling<br>
expression in plant seed cells; and (ii)      a nucleic acid sequence encoding an insulin polypeptide;<br>
wherein the expression vector is suitable for expression in a seed cell. The term "suitable for expression in a seed cell" means that the recombinant expression vector comprises the chimeric nucleic acid sequence of the present invention linked to genetic elements required to achieve expression in a seed cell. Genetic elements that may be included in the expression vector in this regard include a transcriptional termination region, one or more nucleic acid sequences encoding marker genes, one or more origins of replication and the like. In preferred embodiments, the expression vector further comprises genetic elements required for the integration of the vector or a portion thereof in the plant cell's nuclear genome, for example the T-DNA left and right border sequences which facilitate the integration into the plant's nuclear genome in embodiments of the invention in which plant cells are transformed using Agrobacterium.<br>
As hereinbefore mentioned, the recombinant expression vector generally comprises a transcriptional terminator which besides serving as a signal for transcription termination further may serve as a protective element capable of extending the mRNA half life (Guarneros et al., 1982, Proc. Natl. Acad. Sci. USA, 79: 238-242). The transcriptional terminator is generally from about 200 nucleotides to about 1000 nucleotides and the expression vector is prepared so that the transcriptional terminator is located 3' of the nucleic acid sequence encoding insulin. Termination sequences that may be used herein include, for example, the nopaline termination region (Sevan et al., 1983, Nucl. Acids. Res., 11: 369-385), the phaseolin terminator (van der Geest et al., 1994, Plant J. 6: 413-423), the arcelin terminator ((Jaeger GD, et al., 2002, Nat. Biotechnol.Dec; 20:1265-8)), the terminator for the octopine synthase genes of Agrobacterium tumefaciens or other similarly functioning elements.<br><br>
Transcriptional terminators may be obtained as described by An (An, 1987, Methods in Enzym. 153: 292).<br>
Pursuant to the present invention the expression vector may further contain a marker gene. Marker genes that may be used in accordance with the present invention include all genes that allow the distinction of transformed cells from non-transformed cells, including all selectable and screenable marker genes. A marker gene may be a resistance marker such as an antibiotic resistance marker against, for example, kanamycin (US Patent 6,174,724), ampicillin, G418, bleomycin, hygromycin which allows selection of a trait by chemical means or a tolerance marker against a chemical agent, such as the normally phytotoxic sugar mannose (Negrotto et al., 2000, Plant Cell Rep. 19: 798-803). Other convenient markers that may be used herein include markers capable of conveying resistance against herbicides such as glyphosate (US Patents 4,940,935; 5,188,642), phosphinothricin (US Patent 5,879,903) or sulphonyl ureas (US Patent 5,633,437). Resistance markers, when linked in close proximity to nucleic acid sequence encoding the insulin polypeptide, may be used to maintain selection pressure on a population of plant cells or plants that have not lost the nucleic acid sequence encoding the insulin polypeptide. Screenable markers that may be employed to identify transformants through visual inspection include p-glucuronidase (GUS) (US Patents 5,268,463 and 5,599,670) and green fluorescent protein (GFP) (Niedz et al., 1995, Plant Cell Rep., 14: 403).<br>
Recombinant vectors suitable for the introduction of nucleic acid sequences into plants include Agrobacterium and Rhizobium based vectors, such as the Ti and Ri plasmids, including for example pBIN19 (Bevan, Nucl. Acid. Res., 1984, 22: 8711-8721), pGKB5 (Bouchez et al., 1993, C R Acad. Sci. Paris, Life Sciences, 316:1188-1193), the pCGN series of binary<br><br>
vectors (McBride and Summerfelt, 1990, Plant Mol. Biol., 14:269-276) and other binary vectors (e.g. US Patent 4,940,838).<br>
The recombinant expression vectors, nucleic acid sequences and chimeric nucleic acid sequences of the present invention may be prepared in accordance with methodologies well known to those skilled in the art of molecular biology. Such preparation will typically involve the bacterial species Escherichia coli as an intermediary cloning host. The preparation of the E. coli vectors as well as the plant transformation vectors may be accomplished using commonly known techniques such as restriction digestion, ligation, gel ectrophoresis, DNA sequencing, the Polymerase Chain Reaction (PCR) and other methodologies. These methodologies permit the linking of nucleic acid sequences and polypeptides to which the present invention pertains. A wide variety of cloning vectors is available to perform the necessary steps required to prepare a recombinant expression vector. Among the vectors with a replication system functional in E. coli, are vectors such as pBR322, the pUC series of vectors, the M13mp series of vectors, pBluescript etc. Typically, these cloning vectors contain a marker allowing selection of transformed cells. Nucleic acid sequences may be introduced in these vectors, and the vectors may be introduced in E. coli grown in an appropriate medium. Recombinant expression vectors may readily be recovered from cells upon harvesting and lysing of the cells. Further, general guidance with respect to the preparation of recombinant vectors may be found in, for example: Sambrook et al., Molecular Cloning, a Laboratory Manual, Cold Spring Harbor Laboratory Press, 1989, Vol.3. Preparation of plants comprising seeds capable of expressing insulin<br>
In accordance with the present invention the chimeric nucleic acid sequence is introduced into a plant cell and the cells are grown into<br><br>
mature plants capable of setting seed, wherein the seed expresses the insulin polypeptide.<br>
In accordance herewith any plant species or plant cell may be selected. Particular cells used herein include cells obtainable from Arabidopsis thaliana, Brazil nut (Betholettia excelsa); castor bean (Riccinus communis); coconut (Cocus nudfera); coriander (Coriandrum sativum); cotton (Gossypium spp.); groundnut (Arachis Hypogaea); jojoba (Simmondsia chinensis); linseed/flax (Linum usitatissimum); maize (Zea mays); mustard (Brassica spp. and Sinapis alba); oil pahn (Elaeis guineeis); olive (Olea eurpaea); rapeseed (Brassica spp.); rice (Oryza sativa); safflower (Carthamus tinctorius); soybean (Glycine max); squash (Cucurbita maxima); barley (Hordeum vulgare); wheat (Traeticum aestivum); and sunflower (Helianthus annuus).<br>
In accordance herewith in a preferred embodiment plant species or plant cells from oil seed plants are used. Oil seed plants that may be used herein include peanut (Arachis hypogaea); mustard (Brassica spp. and Sinapis alba); rapeseed (Brassica spp.); chickpea (Cicer arietinum); soybean (Glycine max); cotton (Gossypium hirsutum); sunflower (Helianthus annuus); (Lentil Lens culinaris); linseed/ flax (Linum usitatissimum); white clover (Trifolium repens); olive (Olea eurpaea); oil pahn (Elaeis guineeis); safflower (Carthamus tinctorius) and narbon bean (Vicia narbonensis).<br>
In accordance herewith in a particularly preferred embodiment safflower, Arabidopsis or flax is used.<br>
Methodologies to introduce plant recombinant expression vectors into a plant cell, also referred to herein as "transformation", are well known to the art and typically vary depending on the plant cell that is selected. General techniques to introduce recombinant expression vectors in cells include, electroporation; chemically mediated techniques, for example<br><br>
CaCl2 mediated nucleic acid uptake; particle bombardment (biolistics); the use of naturally infective nucleic acid sequences, for example virally derived nucleic acid sequences, or Agrobacterium or Rhizobium derived sequences, polyethylene glycol (PEG) mediated nucleic acid uptake, microinjection and the use of silicone carbide whiskers.<br>
In preferred embodiments, a transformation methodology is selected which will allow the integration of the chimeric nucleic acid sequence in the plant cell's genome, and preferably the plant cell's nuclear genome. In accordance herewith this is considered particularly desirable as the use of such a methodology will result in the transfer of the chimeric nucleic acid sequence to progeny plants upon sexual reproduction. Transformation methods that may be used in this regard include biolistics and Agrobacterium mediated methods.<br>
Transformation methodologies for dicotyledenous plant species are well known. Generally, Agrobacterium mediated transformation is used because of its high efficiency, as well as the general susceptibility by many, if not all, dicotyledenous plant species. Agrobacterium transformation generally involves the transfer of a binary vector, such as one of the hereinbefore mentioned binary vectors, comprising the chimeric nucleic acid sequence of the present invention from E. coli to a suitable Agrobacterium strain (e.g. EHA101 and LBA4404) by, for example, tri-parental mating with an £. coli strain carrying the recombinant binary vector and an £. coli strain carrying a helper plasmid capable of mobilizing the binary vector to the target Agrobacterium strain, or by DNA transformation of the Agrobacterium strain (Hofgen et al., Nucl. Acids Res., 1988, 16:9877). Other techniques that may be used to transform dicotyledenous plant cells include biolistics (Sanford, 1988, Trends in Biotechn. 6:299-302); electroporation (Fromm et al., 1985, Proc. Natl.<br><br>
Acad. Sci. USA., 82:5824-5828); PEG mediated DNA uptake (Potrykus et al., 1985, Mol. Gen. Genetics, 199:169-177); microinjection (Reich et al., Bio/Techn., 1986, 4:1001-1004); and silicone carbide whiskers (Kaeppler et al,<br>
1990,	Plant Cell Rep., 9:415-418) or in planta transformation using, for<br>
example, a flower dipping methodology (Clough and Bent, 1998, Plant J.,<br>
16:735-743).<br>
Monocotyledonous plant species may be transformed using a variety of methodologies including particle bombardment (Christou et al.,<br>
1991,	Biotechn. 9:957-962; Weeks et al., Plant Physiol., 1993, 102:1077-1084;<br>
Gordon-Kamm et al., Plant Cell, 1990, 2:5603-618); PEG mediated DNA<br>
uptake (European Patents 0292 435; 0392 225) or Agrobacterium mediated<br>
transformation (Goto-Fumiyuki et al., 1999, Nature-Biotech. 17:282-286).<br>
The exact plant transformation methodology may vary somewhat depending on the plant species and the plant cell type (e.g. seedling derived cell types such as hypocotyls and cotyledons or embryonic tissue) that is selected as the cell target for transformation. As hereinbefore mentioned in a particularly preferred embodiment safflower, Arabidopsis or flax is used. A methodology to obtain safflower transformants is available in Baker and Dyer (Plant Cell Rep., 1996, 16:106-110). Additional plant species specific transformation protocols may be found in: Biotechnology in Agriculture and Forestry 46: Transgenic Crops I (Y.P.S. Bajaj ed.), Springer-Verlag, New York (1999), and Biotechnology in Agriculture and Forestry 47: Transgenic Crops II (Y.P.S. Bajaj ed.), Springer-Verlag, New York (2001).<br>
Following transformation, the plant cells are grown and upon the emergence of differentiating tissue, such as shoots and roots, mature plants are regenerated. Typically a plurality of plants is regenerated. Methodologies to regenerate plants are generally plant species and cell type<br><br>
dependent and will be known to those skilled in the art. Further guidance with respect to plant tissue culture may be found in, for example: Plant Cell and Tissue Culture, 1994, Vasil and Thorpe Eds., Kluwer Academic Publishers; and in: Plant Cell Culture Protocols (Methods in Molecular Biology 111), 1999, Hall Eds, Humana Press.<br>
In one aspect, the present invention provides a method of recovering plant seeds comprising insulin. Accordingly, the present invention provides a method for obtaining plant seeds comprising insulin comprising:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br>
(i)	a nucleic acid sequence capable of<br>
controlling expression in plant seed cells; and<br>
(ii)	a nucleic acid sequence encoding an insulin<br>
polypeptide;<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell;<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed; and<br>
(d)	obtaining seeds from said plant wherein the seed<br>
comprises insulin.<br>
In preferred embodiments, a plurality of transformed plants is obtained, grown, and screened for the presence of the desired chimeric nucleic acid sequence, the presence of which in putative transformants may be tested by, for example, growth on a selective medium, where herbicide resistance markers are used, by direct application of the herbicide to the<br>
plant, or by Southern blotting. If the presence of the chimeric nucleic acid sequence is detected, transformed plants may be selected to generate progeny and ultimately mature plants comprising a plurality of seeds comprising the desired chimeric nucleic acid sequence. Such seeds may be used to isolate insulin or they may be planted to generate two or more subsequent generations. It will generally be desirable to plant a plurality of transgenic seeds to obtain a population of transgenic plants, each comprising seeds comprising a chimeric nucleic acid sequence encoding insulin. Furthermore, it will generally be desirable to ensure homozygosity in the plants to ensure continued inheritance of the recombinant polypeptide. Methods for selecting homozygous plants are well known to those skilled in the art. Methods for obtaining homozygous plants that may be used include the preparation and transformation of haploid cells or tissues followed by the regeneration of haploid plantlets and subsequent conversion to diploid plants for example by the treatment with colchine or other microtubule disrupting agents. Plants may be grown in accordance with otherwise conventional agricultural practices.<br>
In another aspect, the present invention also provides plants capable of setting seed expressing insulin. In a preferred embodiment of the invention, the plants capable of setting seed comprise a chimeric nucleic acid sequence comprising in the 5' to 3' direction of transcription:<br>
(a)	a first nucleic acid sequence capable of controlling<br>
expression in a plant seed cell operatively linked to;<br>
(b)	a second nucleic acid sequence encoding an insulin<br>
polypeptide, wherein the seed contains insulin.<br>
In a preferred embodiment the chimeric nucleic acid sequence is stably integrated in the plant's nuclear genome.<br><br>
In yet another aspect, the present invention provides plant seeds expressing insulin. In a preferred embodiment of the present invention, the plant seeds comprise a chimeric nucleic acid sequence comprising in the 5' to 3' direction of transcription:<br>
(a)	a first nucleic acid sequence capable of controlling<br>
expression in a plant seed cell operatively linked to;<br>
(b)	a second nucleic acid sequence encoding an insulin<br>
polypeptide.<br>
In accordance with the present invention, seed is obtained which in which preferably at least 0.1% of the total soluble protein present in the seed is insulin. In further preferred embodiments of the present invention, seed is obtained in which at least 0.2%, 0.3%, 0.5%, or 1.0% of the total soluble protein present in the seed is insulin. The insulin polypeptide may be present in a variety of different types of seed cells including, for example, the hypocotyls and the embryonic axis, including in the embryonic roots and embryonic leafs, and where monocotyledonous plant species, including cereals and com, are used in the endosperm tissue. Preparation of insulin from plant seeds<br>
Once the plant seeds have been obtained the insulin protein may be purified from the seed using any protein purification methodologies known to the art.<br>
Accordingly, pursuant to the present invention a method of purifying insulin from plant seeds is provided in which the method comprises:<br>
(a)	providing   a   chimeric   nucleic   acid       construct<br>
comprising in the 5' to 3' direction of transcription as operably<br>
linked components:<br><br>
(i) a nucleic acid sequence capable of controlling expression in plant seed cells; and<br>
(ii) a nucleic acid sequence encoding an insulin polypeptide;<br>
(b)	introducing the chimeric nucleic acid construct into a<br>
plant cell;<br>
(c)	growing the plant cell into a mature plant capable of<br>
setting seed wherein the seed expresses insulin;<br>
(d)	obtaining seed expressing insulin; and<br>
(e)	purifying said insulin from the seed.<br>
The plant seeds may be ground using any comminuting process resulting in a substantial disruption of the seed cell membrane and cell walls. Both dry and wet milling conditions (US Patent 3,971,856; Lawhon et al., 1977, J. Am. Oil Chem. Soc., 63:533-534) may be used. Suitable milling equipment in this regard include colloid mills, disc mills, IKA mills, industrial scale homogenizers and the like. The selection of the milling equipment will depend on the seed type and throughput requirements. Solid seed contaminant such as seed hulls, fibrous materials, undissolved carbohydrates, proteins and other water insoluble contaminants may be removed from the seed fraction using for example size-exclusion based methodologies, such as filtering or gravitational based processes such as centrifugation. In preferred embodiments, the use of organic solvents commonly used in oil extraction, such as hexane, is avoided because such solvents may damage the insulin polypeptide. Substantially pure insulin may be recovered from the seed using a variety of additional purification methodologies such as centrifugation based techniques; size exclusion based methodologies, including for example membrane ultrafiltration and crossflow ultrafiltration; and chromatographic<br>
techniques, including for example ion-exchange chromatography, size exclusion chromatography, affinity chromatography, high performance liquid chromatography (HPLC), fast protein liquid chromatography (FPLC), hydrophobic interaction chromatography and the like. Generally, a combination of such techniques will be used to obtain substantially pure insulin.<br>
In a particularly preferred embodiment of the present invention the insulin polypeptide is isolated from the seed contaminants by contacting the insulin polypeptide with oil bodies. This method is considered to be particularly advantageous as it permits the removal seed contaminants including seed proteins in a particularly efficacious and inexpensive manner. As hereinbefore mentioned such contacting of the insulin polypeptide with the oil bodies may be achieved by linking the insulin polypeptide to an oil body protein or by linking the insulin polypeptide to a polypeptide with affinity for an oil body, such as a single chain antibody with affinity for an oil body. In the former embodiment, insulin polypeptide will be sequestered within the cell on the oil bodies and hence co-purify with the oil bodies. In the latter embodiment, upon being expressed in a membrane enclosed intracellular compartment such as the ER, the insulin polypeptide will associate with the oil body upon breakage of the seed cells during the comminuting process. A process for isolating oil bodies is described in US Patent 5,650,554.<br>
Pharmaceutical insulin formulations may be prepared from the purified insulin and such formulations may be used to treat diabetes. Generally the purified insulin will be admixed with a pharmaceutically acceptable carrier or diluent in amounts sufficient to exert a therapeutically useful effect in the absence of undesirable side effects on the patient treated.<br><br>
To formulate an insulin composition, the weight fraction of insulin is dissolved, suspended, dispersed or otherwise mixed in a selected carrier or diluent at an effective concentration such that the treated condition is ameliorated. The pharmaceutical insulin formulations are preferably formulated for single dosage administration. Therapeutically effective doses for the parenteral delivery of human insulin are well known to the art. Where insulin analogs are used or other modes of delivery are used therapeutically effective doses may be readily empirically determined by those of skill in the art using known testing protocols or by extrapolation of in-vivo or in-vitro test data. It is understood however that concentrations and dosages may vary in accordance with the severity of the condition alleviated. It is further understood that for any particular subject, specific dosage regimens may be adjusted over time according to individual judgement of the person administering or supervising administration of the formulations.<br>
Pharmaceutical solutions or suspensions may include for example a sterile diluent such as, for example, water, lactose, sucrose, dicalcium phosphate, or carboxymethyl cellulose. Carriers that may be used include water, saline solution, aqueous dextrose, glycerol, glycols, ethanol and the like, to thereby form a solution or suspension. If desired the pharmaceutical compositions may also contain non-toxic auxiliary substances such a wetting agents; emulsifying agents; solubilizing agents; antimicrobial agents, such as benzyl alcohol and methyl parabens; antioxidants, such as ascorbic acid and sodium bisulfite; chelating agents such as ethylenediaminetetraacetic acid (EDTA); pH buffering agents such as actetate, citrate or phosphate buffers; and combinations thereof.<br>
The final formulation of the insulin preparation will generally depend on the mode of insulin delivery. The insulin prepared in accordance<br><br>
with the present invention may be delivered in any desired manner; however parenteral, oral, pulmonary, buccal and nasal forms of delivery are considered the most likely used modes of delivery. Parenteral preparations can be enclosed in ampoules, disposable syringes or single or multiple dose vials made of glass, plastic or other such suitable materials.<br>
Examples<br>
The following examples are offered by way of illustration and not by limitation. Example 1<br>
Preparation of an insulin protein expressed as a Mini-Insulin (MI) fusion protein with a trypsin cleavable pro-peptide. Construction of pSBS4404: PRS-D9scFv-Klip27-MI-KDEL fusion protein<br>
One of the fusion proteins studied began with the tobacco pathogen related sequence (PRS) (Sijmons et al., 1990, Bio/technology, 8:217-221) which served as the signal peptide to target expression to the ER in a co-translational manner. Immediately downstream was a sequence encoding for a single-chain Fv antibody (scFv) with species-specific affinity against the 18 kDa oleosin from Arabidopsis thaliana denoted D9scFv, followed by a trypsin cleavable pro-peptide (KLIP27) derived from the TA57 pro-peptide of yeast (Kjeldsen et al., 2001, Biotechnology and Genetic Engineering Reviews 18:89-121). This was followed by mini-insulin (MI) described by Kjeldsen et al. (2001) with the addition of a KDEL ER-retention signal at the C-terminal end of the polypeptide.<br>
The backbone of this plasmid, PSBS4055, was based on the plant binary vector, pPZP200, described by Hajdukiewicz et al. (Plant Molecular Biology, 1994, 25:989-994). In place of the described multiple cloning site, a pat gene conferring host plant phosphinothricine resistance (Wohlleben et al.,<br><br>
1988, Gene 70:25-37)) driven by the ubiquitin promoter/terminator from<br>
Petroselinum crispum (Kawalleck et al., 1993, Plant. Mol. Bio., 21:673-684), was<br>
inserted between the left and right border sequences. In addition to this<br>
cassette, the p-phaseolin promoter/terminator from Phaseolus vulgaris<br>
(Slightom et al., 1983, Proc. Natl. Acad. Sc. USA 80:1897-1901) driving PRS<br>
was sub-cloned. Standard PCR (Horton et al., 1989, Gene 77:61-68) was used<br>
to fuse the synthetic PRS-encoding sequence with attached Sphl/Hindlll<br>
restriction endonuclease sites to the 3' end of the phaseolin promoter to yield<br>
pSBS4011. A Sp/zI-D9scFv-X/zoI, Swal, Hindlll insert sequence was generated<br>
by PCR amplification of a D9scFv cDNA clone (Sean Hemmingsen lab,<br>
unpublished) ' with primers	1325 (GCATGCTGACATTG<br>
TGATGACACAGTC) - SEQ ID NO:175 and 1326 (AAGCTTGCATTTAAATACTCGAGACTGTGAGAGTGGTGCCTTG) - SEQ ID NO: 176. Subsequent ligation of this fragment at the Sphl/Hindlll sites of pSBS4011 resulted in plasmid pSBS4055.<br>
The Klip27-MI sequence was synthesized from four partially<br>
overlapping oligonucleotides which incorporated Arabidopsis thaliana codon<br>
usage to increase the success of efficient translation in a plant-based<br>
expression	system.	Oligonucleotides	1324<br>
(GAAGAAGGAGAGCCTAAGTTTGTTAATCAACATCTTTGTGGATCTCAT<br>
CTTGTTGAGGCTCTCTACCTTG) - SEQ ID NO:177 and 1323 (CCTTA<br>
GGAGTGTAGAAAAATCCTCTTTCTCCACACACAAGGTAGAGAGCCTC<br>
AACA) - SEQ ID NO: 178 were annealed at their complimentary 20 nucleotide<br>
overlap and extended to form the 5' end of the Klip27-MI fusion while the<br>
same	was	done	with	oligonucleotides	1322<br>
(CTAAGGCTGCTAAGGGAATTG) - SEQ ID NO:179 and 1321 (AAGCTTCAGTTGCAATAGTTCTCCAATTGGTAAAGTGAGCAAATAGA<br><br>
AGTGCAACATTGTTCAACAATTCCCTTAGCAGCCTT) - SEQ ID NO:180<br>
to form the 3' end. The two halves were ligated following restriction<br>
digestion with Bsu36I, to yield the full Klip27-MI coding sequence. PCR of<br>
this	gene	fusion	using	primers	1364<br>
(CTCGAGTCAACCAATTGATGACACTGAATC) - SEQ ID NO:181 and 1334<br>
(AAGCTTCAAAGTTCATCCTTGTTGCAATAGTTCTCCAATTG) - SEQ ID<br>
NO: 182 attached a 5' Xhol restriction endonuclease cleavage site and the 3'<br>
KDEL DNA sequence plus Hindlll cleavage site for subsequent ligation into<br>
XM/HmdIII-cut pSBS4055. The result was plasmid pSBS4404: a DNA<br>
sequence encoding the PRS-D9scFv-Klip27-MI-KDEL fusion protein being<br>
placed in a binary vector under expression control of the phaseolin<br>
promoter/terminator. The phaseolin promoter controls the specific-temporal<br>
and tissue-specific expression of the transgene during seed development. The<br>
complete nucleic acid sequence (SEQ ID NO:1) and amino acid sequence (SEQ<br>
ID NO:2) of the 4404 insulin fusion protein (PRS-D9ScFv-Klip27-MI-KDEL) is<br>
shown in Figure 1.	,<br>
Construction of pSBS4405:OLEO-Klip8-Klip27-MI fusion protein<br>
The second fusion protein studied began with the 18 kDa oleosin from Arabidopsis thaliana followed in-frame by a chymosin cleavable pro-peptide (Klip8) - SEQ ID NO:175. Immediately downstream was a sequence encoding for the trypsin cleavable pro-peptide (Klip27) derived from the TA57 pro-peptide of yeast as described above (Kjeldsen et al., 2001, Biotechnology and Genetic Engineering Reviews 18:89-121). This was fused to mini-insulin (MI) described above (Kjeldsen et al., 2001). The expression of this fusion protein was targeted to the nascent oil bodies formed during the development of the embryo.<br><br>
The backbone of this plasmid, pSBS4055, was based on the plant binary vector, pPZP200, described by Hajdukiewicz et al. (Plant Molecular Biology, 1994, 25:989-994).  In place of the described multiple cloning site, a pat gene conferring host plant phosphinothricine resistance (Wohlleben et al., 1988, Gene 70:25-37)) driven by the ubiquitin promoter/terminator from Petroselinum crispum (Kawalleck et al., 1993, Plant. Mol. Bio., 21:673-684), was inserted between the left and right border sequences.   In addition to this cassette,   the   p-phaseolin   promoter/terminator   from   Phaseolus   vulgaris (Slightom et al., 1983, Proc. Natl. Acad. Sc. USA 80:1897-1901) driving the Arabidopsis 18 kDa oleosin genomic sequence-Klip8 fusion was sub-cloned. Standard PCR (Horton et al., 1989, Gene 77:61-68) was used to fuse the oleosin gene-Klip8 sequence with attached X/zoI/HndIII restriction endonuclease sites to the 3' end of the phaseolin promoter to yield pSBS4010.<br>
The Klip27-MI sequence was synthesized from four partially<br>
overlapping oligonucleotides which incorporated Arabidopsis thaliana codon<br>
usage to increase the success of efficient translation in a plant-based<br>
expression	system.	Oligonucleotides	1324<br>
(GAAGAAGGAGAGCCTAAGTTTGTTAATCAACATCTTTGTGGATCTCAT CTTGTTGAGGCTCTCTACCTTG)     -    SEQ     ID     NO:177     and     1323 (CCTTAGGAGTGTAGAAAAATCCTCTTTCTCCACACACAAGGTAGAGA GCCTCAACA) - SEQ ID NO: 178 were annealed at their complimentary 20 nucleotide overlap and extended to form the 5' end of the Klip27-MI fusion while      the       same      was       done      with      oligonucleotides       1322 (CTAAGGCTGCTAAGGGAATTG)    -    SEQ    ID    NO:179    and    1321 (AAGCTTCAGTTGCAATAGTTCTCCAATTGGTAAAGTGAGCAAATAGA AGTGCAACATTGTTCAACAATTCCCTTAGCAGCCTT) - SEQ ID NO:180 to form the 3' end.    The two halves were ligated following restriction<br><br>
digestion with Bsw36I, to yield the full Klip27-MI coding sequence. PCR of<br>
this	gene	fusion	using	primers	1364<br>
(CTCGAGTCAACCAATTGATGACACTGAATC) - SEQ ID NO:181 and 1329 (AAGCTTCAGTTGCAATAGTTC) - SEQ ID NO: 183 attached a 5' Xhol restriction endonuclease cleavage site and a 3' Hmdlll cleavage site, respectively, for subsequent ligation into X/zoI/HmdIII-cut pSBS4010. The result was plasmid pSBS4405: a DNA sequence encoding the Oleosin-KlipS-Klip27-MI fusion protein being placed in a binary vector under expression control of the phaseolin promoter/terminator. The phaseolin promoter controls the specific-temporal and tissue-specific expression of the transgene during seed development. The complete nucleic acid sequence SEQ ID NO:3 and amino acid sequence SEQ ID NO:4 of the 4405 insulin fusion protein (OLEO-Klip8-Klip27-MI) is shown in Figure 2.<br>
Construction of pSBS4414: PRS-MI-tetrabasic linker-D9Scfv-KDEL fusion protein<br>
Another fusion protein studied began with the tobacco pathogen related sequence (PRS) (Sijmons et al, 1990, Bio/technology, 8:217-221) which served as the signal peptide to target expression to the ER in a co-translational manner. Immediately downstream was the sequence encoding mini-insulin (MI) described by Kjeldsen et al. (2001) with the exception that the mini-C propeptide region (AAK - SEQ ID NO: 146) was replaced with an intervening B3o Threonine-tetrabasic site (BsoT-RRKR) (SEQ ID NO: 149) sequence between the Bo-29)- and Ao-2i)-chains of human insulin. This was immediately followed by a sequence encoding for a second tetrabasic linker followed by a single-chain Fv antibody (scFv) with species-specific affinity against the 18 kDa oleosin from Arabidopsis thaliana denoted D9scFv. At the<br><br>
C-terminal end of the polypeptide was the addition of a KDEL ER-retention signal.<br>
The plasmid backbone, pSBS4055, was based on the plant binary vector, pPZP200, described by Hajdukiewicz et al. (Plant Molecular Biology, 1994, 25:989-994). In place of the described multiple cloning site, a pat gene conferring host plant phosphinothricine resistance (Wohlleben et al, 1988, Gene 70:25-37)) driven by the ubiquitin promoter/terminator from Petroselinum crispum (Kawalleck et al., 1993, Plant. Mol. Bio., 21:673-684), was inserted between the left and right border sequences. In addition to this cassette, the p-phaseolin promoter/terminator from Phaseolus vulgaris (Slightom et al., 1983, Proc. Natl. Acad. Sc. USA 80:1897-1901) driving PRS was sub-cloned. Standard PCR (Horton et al., 1989, Gene 77:61-68) was used to fuse the synthetic PRS-encoding sequence with attached Sphl/Hindlll restriction endonuclease sites to the 3' end of the phaseolin promoter to yield pSBS4011.<br>
The Klip27-MI sequence was synthesized from four partially<br>
overlapping oligonucleotides which incorporated Arabidopsis thaliana codon<br>
usage to increase the success of efficient translation in a plant-based<br>
expression	system.	Oligonucleotides	1324<br>
(GAAGAAGGAGAGCCTAAGTTTGTTAATCAACATCTTTGTGGATCTCAT<br>
CTTGTTGAGGCTCTCTACCTTG) SEQ ID NO:177 and 1323 (CCTTA<br>
GGAGTGTAGAAAAATCCTCTTTCTCCACACACAAGGTAGAGAGCCTC<br>
AACA) SEQ ID NO: 178 were annealed at their complimentary 20 nucleotide<br>
overlap and extended to form the 5' end of the Klip27-MI fusion while the<br>
same	was	done	with	oligonucleotides	1322<br>
(CTAAGGCTGCTAAGGGAATTG)     SEQ     ID     NO: 179	and     1321<br>
(AAGCTTCAGTTGCAATAGTTCTCCAATTGGTAAAGTGAGCAAATAGA<br><br>
AGTGCAACATTGTTCAACAATTCCCTTAGCAGCCTT) SEQ ID NO:180 to<br>
form the 3' end. The two halves were ligated following restriction digestion<br>
with Bsw36I, to yield the full Klip27-MI coding sequence. PCR of this gene<br>
fusion using primers 1363 (GCATGCCCAACCAATTGATGACACTG) SEQ<br>
ID	NO:184	and	1334<br>
(AAGCTTCAAAGTTCATCCTTGTTGCAATAGTTCTCCAATTG) SEQ ID NO:182 attached a 5' Sphl restriction endonuclease cleavage site and the 3' KDEL DNA sequence plus HindIII cleavage site for subsequent ligation into Sphl/Hindlll-cut pSBS4011. The result was plasmid pSBS4402: a DNA sequence encoding the PRS-Klip27-MI-KDEL fusion protein being placed in a binary vector under expression control of the phaseolin promoter/terminator. The phaseolin promoter controls the specific-temporal and tissue-specific expression of the transgene during seed development. The plant expression vector, pSBS4402, served as the template to introduce tetrabasic sites between the B and A chains of insulin and between MI and D9 Scfv.<br>
An intravening tetrabasic (BsoT-RRKR) site was placed between<br>
the authentic B(i-29&gt;- and Ao-2i)-chains of human insulin by PCR using primers<br>
1515 (GCATGCATGCCTTTGTTAATCAACATCTTTGTGG) SEQ ID NO:185<br>
and	1518<br>
(ACATTGTTCAACAATTCCTCTCTTTCTTCTAGTCTTAGGAGTGTAG AAAAATCC) SEQ ID NO:186 using pSBS4402 as the template. The resulting 124 bp fragment was used in combination with primer 1517 (GCATAAGCTTCAAAGCTCATCCTTTGAGC) SEQ ID NO:187 using pSBS3400 as a template. Note that pSBS3400 is a plasmid that contains the D9 scFv-KDEL fragment with a Hindlll restriction site. This PCR reaction produced a 955 bp product which introduced a tetrabasic (RRKR)-D9 Scfv-KDEL-Hindlll onto the 124 bp Sphl-Ml fragment. The 955 bp fragment was<br><br>
then ligated and subcloned in pGEM-T (Promega) to result in pSBS3403. The entire Sphl-Ml (with BaoT-RRKR modified C propeptide)-RRKR-D9 Scfv-KDEL-Hindlll fragment was inserted into precut (SphllHindlll) pSBS4402 to generate pSBS4414. The complete nucleic acid sequence SEQ ID NO:5 and amino acid sequence SEQ ID NO:6 of the 4414 insulin fusion protein (PRS-Ml-tetrabasic linker-D9Scfv-KDEL) is shown in Figure 3. Transformation and Growth of recombinant E. coli and Agrobacterium with pSBS4404, pSBS4405 or pSBS4414.<br>
After confirming the integrity of the cDNA encoding for the fusion protein by sequence analysis, the plasmids pSBS4404, pSBS4405 and pSBS4414 were transformed into £. coli strain DH5a to allow for high level of expression. Isolated plasmid DNA (100 ng) was mixed on ice with 100 ul of DHScc competent cells for 20 min. The cells were then heat shocked at 42°C for 45 seconds and returned to ice for 2 min. Then 1 ml of SOC media was added and the cells were incubated at 37°C on an enviro-shaker at 225 rpm for 1 hr before plating transformed cells on LB-spectinomycin plates (10 g/L tryptone, 5 g/L yeast extract, 5 g/L NaCl, 15 g/L agar) and incubating overnight at 37°C. A single colony was used to inoculate 5 ml LB-spectinomycin broth. These cultures were grown overnight at 37°C. The recombinant plasmid was isolated from 1 ml of the overnight culture according to QIAprep®Spin Miniprep Kit (Qiagen)'. The isolated plasmid was then used to transform competent Agrobacterium strain EH101 (Hood et al, 1986; J. Bacteriol. 144: 732-743) by electroporation (25 uF, 2.5 kV, 200 Q). Recombinant Agrobacterium were plated on AB-spectinomycin/kanamycin (20x AB salts, 2 M glucose, 0.25 mg/ml FeSo4-7H2O, 1 M MgSo4, 1 M CaCk) and a single colony was used to inoculate 5 ml of AB-spectinomycin/kanamycin broth. These cultures were grown overnight at<br><br>
28°C. The recombinant Agrobacterium were then used to transform Arabidopsis thaliana plants by the flower dipping method (Clough et al., 1998, Plant J., 16:735-743). Arabidopsis thaliana cv. (C24) is used for all the experiments. Seeds are planted on the surface of a soil mixture (two-thirds Redi-earth and one-third perlite with a pH = 6.7) or an Arabidopsis soil mixture supplied by Lehle Seeds (perlite, vermiculite, peat, terra-green, with a pH = 5.5) in 4 inch pots. The seedlings are allowed to grow to a rosette stage of 6-8 leaves to a diameter of approximately 2.5 cm. The pots are placed inside a dome at 4°C for four days for a cold treatment and subsequently moved to 24°C growth room with constant light at about 150 μE and 60-70% relative humidity. The plants are irrigated at 2-3 day interval and fertilized weekly with 1% of Peters 20-19-18. Each pot contains five to six plants. When plants reach about 2 cm in height, the primary bolts are cut to encourage the growth of secondary and tertiary bolts. 4 to 5 days after cutting the primary bolts, the plants are ready to be infected with Agrobacterium. The pots with Arabidopsis plants are inverted to allow the Arabidopsis plants being infected with 500 ml of a re-suspension an overnight Agrobacterium culture containing the plant transformation vector of interest for 20 seconds. It is important that the Agrobacterium culture contains 5% sucrose and 0.05% of the surfactant Silwet L-77 (Lehle Seeds). The pots are subsequently covered with a transparent plastic dome for 24 hours to maintain higher humidity. The plants are allowed to grow to maturity and a mixture of seeds, untransformed and transformed, are harvested. For selection of transgenic lines, the putative transformed seeds are sterilized in a quick wash of 70% ethanol, then a 20% commercial bleach for 15 min and then rinsed at least four times with ddH2O. About 1000 sterilized seeds are mixed with 0.6% melted top agar and evenly spread on a half strength MS plate (Murashige and Skoog, 1962, Physiologia<br><br>
Plantarum 15: 473-497) containing 0.3% sucrose and 80 μM of the herbicide phosphinothricin (PPT) DL. The plates are then placed in a growth room with light regime 8 hr dark and 16 hr light at 24°C. After 7 to 10 days, putative transgenic seedlings are green and growing whereas untransformed seedlings are bleached. After the establishment of roots the putative transgenic seedlings are individually transferred to pots (the individually plants are irrigated in 3 day interval and fertilized with 1% Peters 20-19-18 in 7 day interval) and allowed to grow to maturity. The pots are covered with a transparent plastic dome for three days to protect the sensitive seedlings. After 7 days the seedlings are covered with a seed collector system from Lehle Seeds to prevent seed loss due to scattering. Seeds from these transgenic plants are harvested individually and ready for analysis. Example 2 Expression levels of insulin in Arabidopsis thaliana<br>
In the second example, expression levels of the fusion protein D9scfv-KLIP27-MI-KDEL (4404), OLEO-KLIP8-KLIP27-MI (4405) and PRS-MI-RRKR-D9Scfv-KDEL (4414) were determined in transgenic Arabidopsis thaliana mature seed. The transgene product was shown to be present in the cellular extracts of mature seed. Approximately 40 transgenic Arabidopsis thaliana seeds were ground with a mortar and pestle in 50 ul of 50 mM Tris-HC1 pH 8.0. Then, a reducing SDS-PAGE sample buffer (6x SDS sample buffer, 0.35 M Tris-HCl pH 6.8, 30% glycerol, 10% SDS, 0.012% bromophenol blue, 5% fi-mercaptoethanol) was added to the slurry and mixed by briefly vortexing. The sample was then briefly centrifuged and placed at 99°C for 10 minutes. After cooling on ice for 2 minutes the sample was centrifuged briefly. Samples were loaded (10 ul - equivalent to approximately 7 seeds) under reducing conditions.<br><br>
For oil body prepared samples the transgenic and wild type seed (20 mg) were ground in 250 ful oil body extraction buffer (0.4 M sucrose, 0.5 M NaCl, 50 mM Tris-HCl pH 8.0). Samples were microfuged at 10 000 g for 10 min. The soluble aqueous fraction was removed with a 26 G 5/8 1 ml syringe and the fat pad was re-suspended in 100 ul phosphate buffer supplemented with salt (20 mM Na2HPO4 pH 8.0, 0.5 M NaCl). The re-suspended fat pad was transferred to a clean microfuge tube and centrifuged again at 10 000 g for 10 min. The procedure was repeated 3 more times with a final re-suspension of the fat pad in 100 ul phosphate buffer without salt (20 mM N32HPO4 pH 8.0). An additional two more washes in phosphate buffer without salt were performed with intermittent centrifugation steps as outlined above. The final fat pellet was re-suspended in 10 ul phosphate buffer (20 mM NA2HPO4 pH 8.0). A 5 ul aliquot was taken and the oil body protein solubilized by boiling in 1/10 (v/v) 50 mM Tris-HCl pH 8.0 with 2% SDS. The sample was cooled on ice for 2 min and centrifuged at 10 000 g for 5 min. The protein content of the undematant was determined by BCA protein assay (Pierce, Rockford, IL). For coomassie-stained gels and Western blot analysis, 20 μg of total protein was separated on 15% SDS-PAGE gels under reducing conditions using the SDS-PAGE sample buffer.<br>
The sample(s) were then loaded on discontinuous 15% SDS-PAGE gels and separated at 150 volts for approximately 1.5 hours. Gels were then either Coomassie-stained or blotted onto PVDF membrane (Immobilon-P, Millipore Corporation, Bedford, MA) for Western blot analysis. Blotted samples were probed with monoclonal antibody directed against insulin (Clone E2-E3; Roth et al., 1992) purchased from Abeam (Cambridge, UK). Insulin bands were detected using a secondary Sheep X mouse IgG F(ab')2 AP-conjugate (Chemicon International, Temecula, CA) and developed using<br><br>
NBT-BCIP in GARAP buffer (Tris-HCl pH 9.5, 100 mM NaCl, 5 mM Mg Cb). The immunoreactive band corresponded to a polypeptide band, migrating at the predicted molecular weight of the fusion protein, as shown in Figures 4A-4F. Figures 4(A-F) show recombinant expression of insulin fusion proteins in transformed Arabidopsis thaliana lines (4404-2, -17, -20, 4405-4, 4414-19 and 4414-20) on the basis of Coomassie-stained SDS-PAGE and Western Blot analysis. The arrows denote the position of the migrating 38.5 kDa, 34.2 kDa and 34.2 kDa fusion polypeptides, PRS-D9(scfv)-KLIP27-MIw/KDEL (4404), OLEO-KLIP8-KLIP27-MI (4405) and PRS-MI-RRKR-D9Scfv-KDEL (4414) respectively, under reducing conditions. It should be noted 4414 fusion protein has an expected molecular weight of 34.2 kDa but has a higher apparent molecular weight on a SDS-PAGE gel). Figures 4A (Coomassie-stained gel) and 4B (corresponding Western blot probed with anti-insulin E2E3) show total seed protein from wild type (wt) and transgenic seed lines expressing the 4404 and 4405 constructs. Figures 4C (Coomassie-stained gel) and 4D (corresponding Western blot probed with anti-insulin E2E3) show oil body protein prepared from wild type and transgenic seed expressing the same 4404 and 4405 constructs. Figures 4D (Coomassie-stained gel) and 4E (corresponding Western Blot probed with anti-insulin E2E3) show oil body protein prepared from wild type and transgenic seed expressing the same 4414 constructs. The molecular weight markers (M) are 10,15, 20, 25, 37, 50, 75, 100, 150, 250 kDa. Controls include, hIN (recombinant human insulin standard) and hProIN (recombinant human proinsulin standard), separated under non-reducing conditions. Differences in expression levels are the result of clonal variation amongst transformants. The approximate protein levels of transgene and MI expression are shown in Figure 5. Levels of expression were determined using the 18 kDa oleosin band as an internal standard<br><br>
(equivalent to 1.5% total seed protein) by densitometry of the transgene band. The average level of expression for the PRS-D9(scfv)-KLIP27-MIw/KDEL (4404), OLEO-KLIP8-KLIP27-MI (4405) and PRS-MI-RRKR-D9Scfv-KDEL (4414) constructs were 0.21% total seed protein, 0.12% total seed protein and 0.79% total seed protein respectively. Example 3<br>
Cleavage of pSBS4404 and HPLC purification Elutionfrom oil body<br>
In the third example, 1 g of transgenic seed was homogenized in 12 ml extraction buffer (0.4 M sucrose, 0.5 M NaCl, 50 mM Tris-HCl pH 8.0) and centrifuged at 10 000 g for 10 min, the fat pads were removed and placed in 1 ml of 20 mM NaaHPCu, 0.5 M NaCl and re-centrifuged as above. This was repeated twice, before washing and centrifuging the fat pad twice in 750 |al 20 mM NaaHPCU. The 4404 fusion protein was eluted from the oil body into the undernatant by washing the final fat pad 5 times in 750 ul 20 mM formic acid pH 4.1, with 10 000 g centrifugation steps in between each wash. The collected elution fractions (undernatants) were pooled and neutralized with 2 N NaOH to pH 8.0. The entire solution was then placed at -80°C to freeze, and lyophilized overnight to concentrate the fusion protein. The lyophilized sample was re-suspended in 500 jal 50 mM Tris-HCl pH 8.0. The resuspended 4404 fusion protein was then desalted on a NAP-5 column (Amersham Pharmacia Biotech Ab, Uppsala, Sweden) and re-exchanged with buffer (50 mM Tris-HCl pH 8.0). The desalted fraction was then frozen again and lyophilized overnight to concentrate. The final concentrated sample was re-suspended in a final volume of 105 ul double distilled H20. The elution results are presented in Figure 6. Figure 6 is a Coomassie-stained SDS-PAGE (15%) analysis of oil body preps prior to elution (-OB), OB prep after elution<br><br>
with formic acid (-OB'), and the concentrated eluted material (-E). The arrow denotes the position of the migrating fusion polypeptide. The wild type control is essentially free of any major proteins following elution whereas the concentrated 4404 material contains the fusion protein, some truncated products (possible hydrolyzed fusion protein) and possibly some albumins that co-eluted. Cleavage and HPLC analysis of 4404 expressing Arabidopsis Seed<br>
The concentrated sample was resuspended in 105 ul of double distilled water and protein content was assessed by BCA protein assay according to the manufacture (Pierce, Rockford, IL, USA). Samples were then cleaved with trypsin (1:300 trypsin:total protein ratio, in 50 mM Tris-HCl pH 8.0, on ice for 90 min). The reactions were stopped with a 10 fold molar excess of TLCK (N-p-tosyl-L-lysine chloromethyl ketone). The entire reactions were then filtered through 0.2 um filters (Aerodisc® 13mm Syringe filter with 0.2 um Supof® membrane, Pall Corporation, Ann Arbor, MI, USA) and analyzed by reversed phase (RP)-HPLC using a CIS column (Zorbax 300SB-C18, Agilent Technologies, Waldbronn, Germany). Samples were loaded onto the column and eluted off at 1.0 ml/min using a 19-min linear gradient of 5-50% (v/v) acetonitrile in 0.1% (v/v) TFA. The chromatograph resulting from this analysis is seen in Figure 7. The trace reveals a trypsin cleaved product from 4404 fusion protein, with nearly identical properties on the column as the human insulin standard (retention times of 17.011 min and 17.179 min, respectively). The HPLC fraction was collected from 17.0-17.5 min and analyzed by PSD MALDI/TOF mass spectrometry using a Voyager-DE STR mass spectrometer (Applied Biosystems). MS analysis was performed by the BioAnalytical Spectroscopy service provided through NRC-Plant Biotechnology Institute, Saskatoon, Saskatchewan, Canada. Resolution<br><br>
of the cleaved 4404 product purified by HPLC as described above is shown in<br>
Figure 8B in comparison to human insulin standard shown in Figure 8A. The<br>
observed mass of cleaved 4404 fusion protein with trypsin was 6191.51 Da.<br>
The discrepancy between human insulin standard (Figure 8A) and the<br>
cleaved 4404 product (Figure 8B) corresponds to a Des-Bso Insulin with the<br>
KDEL signal retained on the A-chain of the cleaved product (Des-Bao Insulin-<br>
KDEL).<br>
Example 4<br>
Cleavage of pSBS4405 and HPLC purification<br>
Oil Body Preparation<br>
Fusion protein (OLEO-KLIP8-KLIP27-MI) can be partially purified by performing oil body preparations as described below. Approximately, 1 g of transgenic seed was homogenized in 12 ml extraction buffer (0.4 M sucrose, 0.5 M Nad, 50 mM Tris-HCl pH 8.0) and centrifuged at 10 000 g for 10 min, the fat pads were removed and placed in 1 ml of 50 mM Tris-HCl pH 8.0, 0.5 M NaCl and re-centrifuged as above. This was repeated twice, before washing and centrifuging the fat pad twice in 750 ^1 50 mM Tris-HCl pH 8.0. The oil body preparation results in the removal of the majority of background proteins. The typical protein profile of an oil body preparation from transgenic Arabidopsis seed expressing the 4405 construct is demonstrated in Figure 9. Cleavage and HPLC analysis of 4405 expressing Arabidopsis Seed<br>
The total protein content from resuspended oil bodies was assessed by solubilizing a fraction of the preparation (5μl) diluted 10 times in 2% SDS, 50 mM Tris-HCl pH 8.0, boiled for 5 minutes and centrifuged for 3 min at 10 000 g. Thereafter, the protein content was determined by BCA protein assay according to the manufacture (Pierce, Rockford, IL, USA).<br><br>
Samples were then cleaved with trypsin (1:300 trypsinrtotal protein ratio, in 50 mM Tris-HCl pH 8.0, on ice for 90 min) to release the Klip27-MI fragment from the fusion protein. The reactions were stopped with a 10 fold molar excess of TLCK (N-p-tosyl-L-lysine chloromethyl ketone). Samples were centrifuged at 10 000 g for 10 min and the undernatants of the entire reactions were then filtered through 0.2 μm filters (Aerodisc® 13mm Syringe filter with 0.2 μm Supof® membrane, Pall Corporation, Ann Arbor, MI, USA). Figure 9 depicts the Coomassie-stained SDS-PAGE (15%) analysis of total extractable seed protein and oil body (OB) prepared protein from lines expressing 4405 in comparison to wild type (nonrecombinant) seed. The arrow denotes the position of the migrating fusion polypeptide. The undernatants were further analyzed by reversed phase (RP)-HPLC using a CIS column (Zorbax 300SB-C18, Agilent Technologies, Waldbronn, Germany). Samples were loaded onto the column and eluted off at 1.0 ml/min using a 19-min linear gradient of 5-50% (v/v) acetonitrile in 0.1% (v/v) TFA. The chromatograph resulting from this analysis is seen in Figure 10. The trace reveals a trypsin cleaved product from 4405 fusion protein, with nearly identical properties on the column as the human insulin standard (retention times of 17.220 min and 17.179 min, respectively). The HPLC fraction was collected from 17.0-17.5 min and analyzed by PSD MALDI/TOF mass spectrometry using a Voyager-DE STR mass spectrometer (Applied Biosystems). MS analysis was performed by the BioAnalytical Spectroscopy service provided through NRC-Plant Biotechnolgoy Institute, Saskatoon, Saskatchewan, Canada. As shown in Figure 11, the observed mass of cleaved 4405 fusion protein with trypsin was 5706.30 Da. The discrepancy between human insulin standard (Figure 8A) and the cleaved 4405 product (Figure 11) correspond to a Des-Bao Insulin<br><br>
product (Des-Bso Insulin).  The Des-Bso Insulin is the product expected from<br>
correct trypsin maturation of the 4405 fusion.<br>
Example 5<br>
Purification of Trypsin Cleaved MI using AKTA Explorer (FPLC)<br>
Purification of cleaved MI from 4405 was also partially purified from upscaled cleavage reactions by anion exchange (Mono Q FF ImL, Amersham Pharmacia) on an AKTA explorer (Amersham Pharmacia). Cleavage reactions were performed on 4405 oil body as described above prepared from up to 30 g transgenic seed. The undernatant from cleavage reactions were either filtered through 0.2 urn filters or concentrated by lyophilization on a Savant Speed Vac. Sample reactions filtered could be applied to the column directly, but concentrated samples required the removal of salts to effectively bind the column. Concentrated samples could be desalted by passing cleaved material through a PD-10 column (Amersham Pharmacia), by dialysis, or dilution to a salt concentration equivalent or less than 5 mS/cm. Desalted samples were equilibrated with 20 mM Tris-HCl pH 6.5. Samples were separated using a step gradient with NaCl of 0-40% NaCl with a 1 ml/min flow rate. Detection was performed at 214 nm (detection at 280 nm is relatively poor because of the low content of aromatic amino acids in insulin). Solvent A was 20 mM Tris-HCl pH 6.5 while solvent B was 20 mM Tris-HCl pH 6.5, 1.0 M NaCl. Fractions (1 ml) eluting at the same conductivity as Roche insulin standard, between 7-35 mS/cm, were collected (refer to Figure 12). Figure 12 shows the chromatogram of trypsin cleaved 4405 oil body preparations (dashed line) in comparison to human insulin standard (solid line). The presence of Insulin was verified in the collected fractions by HPLC, ELISA, or Western analysis (data not shown). Samples<br><br>
collected were then concentrated by lyophilization and used in the insulin<br>
bioassay described in example 6.<br>
Example 6<br>
Insulin Tolerance Test: Bioassay in C57BI/6 (B6) Male Mice<br>
This bioassay was performed to determine the in vivo effect of recombinant plant-derived (DesB30 IN) from trypsin cleaved 4405 in comparison to human insulin. Glucose plasma levels in B6 mice were determined prior to and following the intraperitoneal injection of insulin standards, negative controls, and SBS insulin. Fifteen male C57BI/6 (B6) mice approximately 2 months of age were purchased from Jackson Laboratories (Bar Harbor, ME). Plasma glucose levels were determined with an automatic glucometer (OneTouch Ultra, Lifescan, Johnson and Johnson). Positive controls included HumulinR® (Eli Lilly) and yeast recombinant human insulin standard from Roche. A saline solution served as the placebo. A negative control was included which represented trypsin cleaved oil bodies purified from wild type (non-recombinant) Arabdiopsis seed that was processed identically to recombinant 4405 trypsin cleaved oil body preparations.<br>
B6 mice were housed and fed ad libitum on a 12-hour dark-light cycle. For insulin tolerance tests mice were injected intraperitoneally (IP) with insulin (lU/kg body weight) and glucose levels determined at 0, 15, 30, and 60 minutes using the automatic glucometer. All insulin tolerance tests were performed at the same interval each day (9:00 am). Insulin tolerance tests were performed with at least 2 days interevening between administering the next test. The results for the insulin tolerance tests are depicted in Figure 13. The SBS DesB30 Insulin derived from 4405 seed (closed diamonds) behaved almost identically (was statistically not different, p
<br>
R® (open squares) and Roche Insulin (open triangles standards following<br>
injection over the course of the study.    All insulin(s) tested significantly<br>
reduced plasma glucose levels (p
circles) and typsin cleaved wild type Arabidopsis oil bodies (closed circles)<br>
(negative control).<br>
Example 7<br>
Construction of pSBS4401: PRS- Klip27-MI- fusion protein<br>
One of the fusion proteins studied began with the tobacco pathogen related sequence (PRS) (Sijmons et al., 1990, Bio/technology, 8:217-221) which served as the signal peptide to target expression to the ER in a co-translational manner. Immediately downstream was a trypsin cleavable pro-peptide (KLIP27) derived from the TA57 pro-peptide of yeast (Kjeldsen et al., 2001, Biotechnology and Genetic Engineering Reviews 18:89-121). This was followed by mini-insulin (MI) described by Kjeldsen et al. (2001).<br>
The backbone of this plasmid, pSBS4055, was based on the plant binary vector, pPZP200, described by Hajdukiewicz et al. (Plant Molecular Biology, 1994, 25:989-994). In place of the described multiple cloning site, a pat gene conferring host plant phosphinothricine resistance (Wohlleben et al., 1988, Gene 70:25-37)) driven by the ubiquitin promoter/terminator from Petroselinum crispum (Kawalleck et al., 1993, Plant. Mol. Bio., 21:673-684), was inserted between the left and right border sequences. In addition to this cassette, the ß-phaseolin promoter/terminator from Phaseolus vulgaris (Slightom et al., 1983, Proc. Natl. Acad. Sc. USA 80:1897-1901) driving PRS was sub-cloned. Standard PCR (Horton et al., 1989, Gene 77:61-68) was used to fuse the synthetic PRS-encoding sequence with attached Sp/zI/HwdIII restriction endonuclease sites to the 3' end of the phaseolin promoter to yield pSBS4011.<br><br>
The Klip27-MI sequence was synthesized from four partially<br>
overlapping oligonucleotides which incorporated Arabidopsis thaliana codon<br>
usage to increase the success of efficient translation in a plant-based<br>
expression	system.	Oligonucleotides	1324<br>
(GAAGAAGGAGAGCCTAAGTTTGTTAATCAACATCTTTGTGGATCTCAT<br>
CTTGTTGAGGCTCTCTACCTTG) - SEQ ID NO:177 and 1323 (CCTTA<br>
GGAGTGTAGAAAAATCCTCTTTCTCCACACACAAGGTAGAGAGCCTC<br>
AACA) - SEQ ID NO: 178 were annealed at their complimentary 20 nucleotide<br>
overlap and extended to form the 5' end of the Klip27-MI fusion while the<br>
same	was	done	with	oligonucleotides	1322<br>
(CTAAGGCTGCTAAGGGAATTG) - SEQ ID NO:179 and 1321<br>
(AAGCTTCAGTTGCAATAGTTCTCCAATTGGTAAAGTGAGCAAATAGA<br>
AGTGCAACATTGTTCAACAATTCCCTTAGCAGCCTT) - SEQ ID NO:180<br>
to form the 3' end. The two halves were ligated following restriction<br>
digestion with Bsu36l, to yield the full Klip27-MI coding sequence. PCR of<br>
this	gene	fusion	using	primers	1363<br>
(GCATGCCCAACCAATTGATGACACTG) - SEQ ID NO:184 and primer 1329 (AAGCTTCAGTTGCAATAGTTC) - SEQ ID NO:183 attached a 5' SphI and 3' Hindlll restriction endonuclease cleavage site for subsequent ligation into Sp/zI/HmdIII-cut pSBS4011 (as mentioned above). The result was plasmid pSBS4401: a DNA sequence (SEQ ID NO:188) encoding the PRS-Klip27-MI fusion protein (SEQ ID NO: 189) being placed in a binary vector under expression control of the phaseolin promoter/terminator. The phaseolin promoter controls the specific-temporal and tissue-specific expression of the transgene during seed development.<br>
Transformation and Growth of recombinant E. coli and Agrobacterium with pSBS4401.<br><br>
After confirming the integrity of the cDNA encoding for the<br>
fusion protein by sequence analysis, the plasmids pSBS4401 was transformed<br>
into £. coli strain DH5a to allow for high level of expression. Isolated plasmid<br>
DNA (100 ng) was mixed on ice with 100 jal of DH5a competent cells for 20<br>
min. The cells were then heat shocked at 42°C for 45 seconds and returned to<br>
ice for 2 min. Then 1 ml of SOC media was added and the cells were<br>
incubated at 37°C on an enviro-shaker at 225 rpm for 1 hr before plating<br>
transformed cells on LB-spectinomycin plates (10 g/L tryptone, 5 g/L yeast<br>
extract, 5 g/L NaCl, 15 g/L agar) and incubating overnight at 37°C. A single<br>
colony was used to inoculate 5 ml LB-spectinomycin broth. These cultures<br>
were grown overnight at 37°C. The recombinant plasmid was isolated from 1<br>
ml of the overnight culture according to Qiagen mini prep. The isolated<br>
plasmid was then used to transform competent Agrobacterium strain EH101<br>
(Hood et al, 1986; J. Bacteriol. 144: 732-743) by electroporation (25 uF, 2.5 kV,<br>
200 Ω).	Recombinant Agrobacterium were plated on AB-<br>
spectinomycin/kanamycin (20x AB salts, 2 M glucose, 0.25 mg/ml FeSo4 7H2<d m mgso4 cack and a single colony was used to inoculate ml of ab-spectinomycin broth. these cultures were grown overnight at the recombinant agrobacterium then transform arabidopsis thaliana plants by flower dipping method et al. plant j. as described in example expression levels insulin></d>
Expression levels of the fusion protein KLIP27-MI (4401) was determined in transgenic Arabidopsis thaliana mature seed using the procedure outlined in Example 2 above.   The transgene product was not found to be present in the cellular extracts of mature seed. Example 8<br><br>
Construction of pSBS4409:OLEO-human proinsulin (OLEO-hPIN) fusion protein<br>
This fusion protein began with the 18 kDa oleosin from Arabidopsis thaliana followed in-frame by the gene encoding for human proinsulin (hPIN). The expression of this fusion protein was targeted to the nascent oil bodies formed during the development of the embryo.<br>
The backbone of this plasmid, pSBS4008, was based on the plant binary vector, pPZP200, described by Hajdukiewicz et al. (Plant Molecular Biology, 1994, 25:989-994). In place of the described multiple cloning site, a pat gene conferring host plant phosphinothricine resistance (Wohlleben et al., 1988, Gene 70:25-37)) driven by the ubiquitin promoter/terminator from Petroselinum crispum (Kawalleck et al., 1993, Plant. Mol. Bio., 21:673-684), was inserted between the left and right border sequences. In addition to this cassette, the p-phaseolin promoter/terminator from Phaseolus vulgaris (Slightom et al., 1983, Proc. Natl. Acad. Sc. USA 80:1897-1901) driving the Arabidopsis 18 kDa oleosin genomic sequence was sub-cloned. Standard PCR (Horton et al., 1989, Gene 77:61-68) was used to fuse the oleosin gene sequence (minus stop codon) with attached Ncol and HmdIII restriction endonuclease sites to the 3' end of the phaseolin promoter to yield pSBS4008.<br>
An Ncol-human pre-proinsulin gene-Hzndlll was synthesized as a single 335 bp piece by Aptagen using preferred plant codon usage. Subsequent ligation into NcoI/HmdIII-cut pSBS4008 resulted in the plasmid pSBS4400: a DNA sequence encoding the Oleosin-human pre-proinsulin fusion protein being placed in a binary vector under expression control of the phaseolin promoter/terminator. The pSBS4400 plasmid served as the template to generate human proinsulin (hPIN) by standard PCR using pfu DNA polymerase with primers directed against the 5' end (1457 oligo<br><br>
TTCGTGAACCAACACTTG - SEQ ID NO: 190) and 3' end (1458 oligo<br>
AAGCTTTCAGTTACAGTAGT - SEQ ID NO:191) including the Hindlll site<br>
of the existing proinsulin region of the vector. A second fragment was<br>
amplified using pfu DNA polymerase with a primer directed against the<br>
available SphI site (oligo 1455 GCATGCATGTGTTGAGC - SEQ ID NO:192)<br>
to the 3'end of the Arabidopsis oleosin gene (oligo 1456<br>
GGTAGTGTGCTGGCCA - SEQ ID NO:193) within the pSBS4400 vector.<br>
Following PCR, products were separated on an agarose gel and bands<br>
corresponding to a 267 bp (hPIN-HindIII) and 360 bp (SphI-OLEO(3'end))<br>
fragment were gel purified using a gel extraction kit (Qiagen). The two<br>
fragments were fused by a second round of PCR amplification using Taq<br>
DNA polymerase with primers 1455 (SEQ ID NO:192) and 1458 (SEQ ID<br>
NO:193) in combination with 0.001 µM of an overlapping bridging PCR<br>
primer	(oligo	1459<br>
GGTGGCCAGCACACTACCTTCGTGAACCAACACTTGTG - SEQ ID NO:194) for two cylces with an annealing temperature of 58°C followed by 31 cycles at 52 °C in order to amplify a 627 bp SpW-OLEO(3'end)-hPIN-HmdIIJ fragment. The 627 bp SphI-OLEO(3'end)-hPIN-HindIII fragment was then ligated into the T/A overhang of pGEMT Easy Vector System™ (Promega) and used to transform DHScc bacteria to result in pSBS3409 (pGEMT-SphI-OLEO(3'end)-hPIN-HindIII.<br>
The Sphl/Hindlll fragment from pSBS3409 was exchanged with the Sphl/Hindlll fragment from pSBS4400. Standard restriction digests on both pSBS3409 and pBSB4404 were performed using Sphl/Hindlll (New England Biolabs). Fragments were separated on 1.5% agarose gels and purified using a gel extraction kit (Qiagen). The 617 bp Sphl/Hindlll fragment liberated from pSBS3409 was then ligated into the Sphl/Hindlll acceptor site in pre-cut<br><br>
pSBS4400 (internal Sphl/Hindlll fragment removed) vector backbone using T4 DNA ligase (NEB) overnight at 15°C.<br>
The result was plasmid pSBS4409: a DNA sequence (SEQ ID NO:195) encoding the OLEOSIN-hPIN fusion protein (SEQ ID NO:196 being placed in a binary vector under the expression control of the phaseolin promoter/terminator. The phaseolin promoter controls the specific-temporal and tissue-specific expression of the transgene during seed development. Transformation and Growth of recombinant E. coli and Agrobacterium with pSBS4409.<br>
After confirming the integrity of the cDNA encoding for the<br>
fusion protein by sequence analysis, the plasmids pSBS4409 was transformed<br>
into E. coli strain DH5a to allow for high level of expression. Isolated plasmid<br>
DNA (100 ng) was mixed on ice with 100 ul of DHSct competent cells for 20<br>
min. The cells were then heat shocked at 42°C for 45 seconds and returned to<br>
ice for 2 min. Then 1 ml of SOC media was added and the cells were<br>
incubated at 37°C on an enviro-shaker at 225 rpm for 1 hr before plating<br>
transformed cells on LB-spectinomycin plates (10 g/L tryptone, 5 g/L yeast<br>
extract, 5 g/L NaCl, 15 g/L agar) and incubating overnight at 37°C. A single<br>
colony was used to inoculate 5 ml LB-spectinomycin broth. These cultures<br>
were grown overnight at 37°C. The recombinant plasmid was isolated from 1<br>
ml of the overnight culture according to Qiagen mini prep. The isolated<br>
plasmid was then used to transform competent Agrobacterium strain EH101<br>
(Hood et al., 1986; J. Bacteriol. 144: 732-743) by electroporation (25 uF, 2.5 kV,<br>
200 Q).	Recombinant Agrobacterium were plated on AB-<br>
spectinomycin/kanamycin (20x AB salts, 2 M glucose, 0.25 mg/ml FeSo4 THzO, 1 M MgSo4, 1 M CaCb) and a single colony was used to inoculate 5 ml of AB-spectinomycin/kanamycin broth. These cultures were grown overnight at<br><br>
28°C. The recombinant Agrobacterium were then used to transform Arabidopsis thaliana plants by the flower dipping method (dough et al., 1998, Plant }., 16:735-743) as described in Example 1. Expression levels of insulin in Arabidopsis thaliana<br>
Expression levels of the fusion protein OLEO-hPIN (4409) was determined in transgenic Arabidopsis thaliana mature seed using the procedure outlined in Example 2 above. Coomassie-stained gel of oil body proteins from two representative lines (4409-6 and 4409-8) comparing the migration of Oleosin-hPIN fusion protein (as denoted by the black arrow) to non-transformed (wt) Arabidopsis (Figurel4). The level of expression was determined by densitometry to measure on average about 0.10% of total seed protein. This level was calculated above and beyond the co-migration of an endogenous protein of the same molecular weight in the non-transformed seed (wt) which constituted approximately 0.04% of the total seed protein. Example 9 Transformation of safflower<br>
This transformation protocol is similar to that outlined by Orlilcowska T. K. et al. ((1995) Plant Cell, Tissue and Organ Culture 40: 85-91), but with modifications and improvements both for transforming S-317 and for using phosphinothricin as the selectable marker. Decontaminate seeds from S-317 California variety of safflower, which are not damaged, cracked or diseased, in 0.1% HCk for 12 minutes followed by 4-5 rinses with sterile distilled water. Germinate sterile seeds in the dark on MS medium (Murashige T. &amp; Skoog F (1962) Physiol. Plant. 15: 473-497) with 1% sucrose and 0.25% Gelrite. Initiate Agrobacterium cultures from frozen glycerol stocks in 5 ml AB minimal liquid media with antibiotic selection, and grow for 48 hours at 28° C. Grow an aliquot of this culture grown overnight in 5 ml of<br><br>
Luria broth with selection for transformation. Wash 6-8 ml of bacterial cells twice with AB media, and make up to a final cell density of 0.4 -0.5 (OD600).<br>
Remove two-day-old cotyledons from germinated seedlings, dip in the prepared Agrobacterium cells, and plate on MS medium with 3% sucrose, 4 ^M N6-benzyladenine (BA) and 0.8 jaM naphthaleneacetic acid (NAA). Incubate plates at 21° C under dark conditions. After 3 days, transfer to the same medium with 300 mg/L timentin. After an additional 4 days, move all cultures to the light. After 3 days, place explants on selection medium with phosphinothricin added at 0.5 mg/L. For continued bud elongation, transfer explants weekly onto MS medium without phytohormones but with twice the basal amount of KNOs. Excise shoots that had elongated to greater than 10 mm from the initial explant and individually grow on selection. For rooting, place green shoots, representing putative transgenic tissue, on MS medium with 2% sucrose, 10 µM indolebutyric acid and 0.5 µM NAA. Transfer rooted shoots to a well drained soil-less mix and grow under high humidity and 12 hours of light. Example 10 Flax transformation protocol<br>
This transformation procedure is similar to that outlined by Dong J. and McHughen A. (Plant Cell Reports (1991) 10:555-560), Dong J. and McHughen A. (Plant Sciences (1993) 88:61-71) and Mlynarova et al. (Plant Cell Reports (1994) 13: 282-285). Decontaminate flax seeds, which are not damaged, cracked or diseased, in a 70% ethanol solution for 5 to 7 minutes, followed by 25 minutes in a 50% bleach solution with Tween 20 (3-4 drops per 100ml) with continuous stirring. Rinse seeds 5 to 7 times with sterile distilled water. Germinate decontaminated seeds in the light on MS medium<br><br>
(Murashige T. &amp; Skoog F (1962) Physiol. Plant. 15: 473-497) with 2% sucrose and 0.3% Gelrite® in Magenta jars. For transformation, grow Agrobacterium cultures overnight in AB broth plus the appropriate antibiotic for selection. Wash 6 to 8 ml of overnight cells twice, and re-suspended in 5 ml of AB broth; add 2 ml of this stock to 98 ml of induction medium (MS basal medium with 3% sucrose, 5 jaM 6-benzylaminopurine (BA) and 0.25 jaM alpha-naphthalene acetic acid (NAA) and adjust for a final ODsoo of 1.0.<br>
Section hypocotyl explants, and inoculate in the prepared Agrobacterium cell solution for about 4 h (stir plates gently 1-2 times during this period). After the infection period, remove explants from the liquid inoculation medium and blot on sterile filter paper. Plate 15-20 explants on 0.7% agar-solidified induction medium in tissue culture plates. Seal the plates with plastic wrap, and co-cultivate explants for 48 h under light conditions (23-24 °C). After 2 days, transfer the green, meristematic explants to the same medium containing 300 mg/L Timentin (pre-selection media) and wrap with plastic wrap. After 3 days, transfer the cultures to the above medium containing 10 mg/L DL PPT (Selection 1). Wrap the plates with Parafilm® and incubate at 24 CC under light conditions. Transfer cultures every two weeks and keep on this media for one month. For shoot elongation, transfer the cultures every two weeks on selection medium II (MS basal medium containing 2% sucrose, 500mg/L MES buffer, 300 mg/L Timentin and 10 mg/L DL PPT) in Magenta jars. Putative transformed shoots, which survived selection, are dark green and form vigorous roots in 7-10 days when planted individually on selection II media. Transfer rooted shoots to sterilized greenhouse soil mix in small pots and cover plantlets with clear plastic cups for acclimatization. For maturation, transfer actively growing plants to one-<br><br>
gallon pots with a well-drained soil mix and grow under greenhouse conditions.<br>
While the present invention has been described with reference to what are presently considered to be the preferred examples, it is to be understood that the invention is not limited to the disclosed examples. To the contrary, the invention is intended to cover various modifications and equivalent arrangements included within the spirit and scope of the appended claims.<br>
All publications, patents and patent applications are herein incorporated by reference in their entirety to the same extent as if each individual publication, patent or patent application was specifically and individually indicated to be incorporated by reference in its entirety.<br>
(Table Removed)<br>
SUMMARY OF SEQUENCES<br>
SEQ ID NO:1 and 2 set forth the nucleotide sequence and the deduced amino acid sequence, respectively, of the PRS-D9scFv-KLIP27-MI-KDEL fusion protein in the plasmid pSBS4404.<br>
SEQ ID NO: 3 and 4 set forth the nucleotide sequence and the deduced amino acid sequence, respectively, of the Oleo-KLIP8-KLIP27-MI fusion protein in plasmid pSBS4405.<br>
SEQ ID NO:5 and 6 set forth the nucleotide sequence and the deduced amino acid sequence, respectively, of the PRS-MI-tetrabasic linker-D9Scfv-KDEL fusion protein in plasmid pSBS4414.<br>
SEQ ID NO:7 to 145 set forth known insulin sequences which are described in Table 1.<br>
SEQ ID NO:146 to 148 set forth the amino acid sequences of fragments of the insulin C-peptide.<br>
SEQ ID NO: 149 sets forth the amino acid sequence of the tetrabasic processing peptide.<br>
SEQ ID NO:150 to 155 sets forth the amino acid sequence of the polypeptides capable of retaining the insulin polypeptide to the ER.<br>
SEQ ID NO: 156 to 160 sets forth the amino acid sequences of the polypeptides capable of retaining the insulin polypeptide to an ER derived storage organelle.<br>
SEQ ID NO:161 sets forth the amino acid sequence of a PRS signal sequence.<br>
SEQ ID NO: 162 to 171 set forth the amino acid sequences of yeast leader sequences and sequences derived there from.<br><br>
SEQ ID NO: 172 to 173 set forth the amino acid sequences of spacer peptides.<br>
SEQ ID NO:174 sets forth the amino acid sequence of the KLIPS sequence<br>
SEQ ID NO:175 sets forth the nucleotide sequence of the forward primer 1325 which is complementary to the 5' region of the D9ScFv cDNA clone and is designed to add a Sphl site to the 5' region facilitate subsequent ligation.<br>
SEQ ID NO: 176 sets forth the nucleotide sequence of the reverse primer 1326 which is complementary to the 3' region of the D9ScFv cDNA clone and is designed to add a Xhol site to the 3' region facilitate subsequent ligation.<br>
SEQ ID NO:177 sets forth the nucleotide sequence of the forward primer 1324 which is complementary to a 20 nucleotide region of reverse primer 1323 and is designed to form the 5' end of the Klip27-MI fusion.<br>
SEQ ID NO:178 sets forth the nucleotide sequence of the reverse primer 1323 which is complementary to a 20 nucleotide region of forward primer 1324 and is designed to form the 5' end of the Klip27-MI fusion.<br>
SEQ ID NO: 179 sets forth the nucleotide sequence of the forward primer 1322 which is complementary to a 19 nucleotide region of reverse primer 1321 and is designed to form the 3' end of the Klip27-MI fusion.<br>
SEQ ID NO:180 sets forth the nucleotide sequence of the reverse primer 1321 which is complementary to a 19 nucleotide region of forward primer 1322 and is designed to form the 3' end of the Klip27-MI fusion.<br>
SEQ ID NO: 181 sets forth the nucleotide sequence of the forward primer 1364 which is complementary to the 5' region of the Klip27-MI<br><br>
sequence and is designed to add a Xhol site to the 5' region facilitate subsequent ligation.<br>
SEQ ID NO:182 sets forth the nucleotide sequence of the reverse primer 1334 which is complementary to the 3' region of the Klip27-MI sequence and is designed to add a Hindlll site to the 3' region facilitate subsequent ligation and a 3' KDEL sequence.<br>
SEQ ID NO: 183 sets forth the nucleotide sequence of the reverse primer 1329 which is complementary to the 3' region of the Klip27-MI sequence and is designed to add a Hindlll site to the 3' region facilitate subsequent ligation.<br>
SEQ ID NO: 184 sets forth the nucleotide sequence of the forward primer 1363 which is complementary to the 5' region of the Klip27-MI sequence and is designed to add a SphI site to the 5' region facilitate subsequent ligation.<br>
SEQ ID NO:185 sets forth the nucleotide sequence of the forward primer 1515 which is complementary to the 5' region of the insulin B chain sequence and is designed to insert the intervening tetrabasic site between the authentic A and B chains of human insulin in conjunction with reverse primer 1518.<br>
SEQ ID NO: 186 sets forth the nucleotide sequence of the reverse primer 1518 which is complementary to the 3' region of the insulin B chain and 5' region of the insulin A chain with the intervening tetrabasic mini-c-peptide sequence and is designed to insert the intervening tetrabasic site between the authentic A and B chains of human insulin.<br>
SEQ ID NO: 187 sets forth the nucleotide sequence of the reverse primer 1517 which is complementary to the 3' region of the_D9 scFV/KDEL<br><br>
sequence and is designed to amplify the entire Ml-tetrabasic linker-D9Scfv-KDEL to create the pSBS4414 insert.<br>
SEQ ID NO:188 and 189 set forth the nucleotide sequence and the deduced amino acid sequence, respectively, of the PRS- Klip27-MI fusion protein in plasmid pSBS4401.<br>
SEQ ID NO: 190 sets forth the nucleotide sequence of the forward primer 1457 which is complementary to the 5' region of the insulin B chain sequence and is designed to generate the human proinsulin (hRIN) fragment in conjunction with reverse primer 1591.<br>
SEQ ID NO: 191 sets forth the nucleotide sequence of the reverse primer 1458 which is complementary to the 3' region of human proinsulin (hPIN) is designed 'to generate the human pro(hPIN) and add a 3' Hindlll cloning site.<br>
SEQ ID NO: 192 sets forth the nucleotide sequence of the forward primer 1455 which is complementary to the 5' region of the Sphi site of pSBS4404 and is designed amplify the Arabidopsis oleosin gene in conjunction with reverse primer 1456.<br>
SEQ ID NO:193 sets forth the nucleotide sequence of the reverse primer 1456 which is complementary to the 3' region of the Arabidopsis oleosin gene and is designed amplify the Arabidopsis oleosin gene in conjunction with forward primer 1455.<br>
SEQ ID NO: 194 sets forth the nucleotide sequence of the overlapping bridging PCR primer which is complementary to the 3' region of the Arabidopsis oleosin gene and the 5' end of the human proinsulin gene and is designed to create the pSBS4409 insert inconjunction with forward primer 1455 and reverse primer 1456.<br><br><br><br><br><br>
We claim:<br>
1.	A method for the expression of insulin in plant seeds comprising:<br>
(a)	providing a chimeric nucleic acid construct comprising in the 5' to 3'<br>
direction of transcription as operably linked components:<br>
(i)        a nucleic acid sequence comprises a seed-preferred promoter;<br>
(ii)       a nucleic acid sequence encoding an insulin polypeptide; and<br>
(iii)	a nucleic acid sequence encoding a polypeptide which is<br>
capable of retaining the insulin polypeptide in the endoplasmic reticulum (ER) or an ER derived storage vesicle, and<br>
(b)	introducing the chimeric nucleic acid construct into a plant cell,<br>
wherein the plant cell is grown into a mature plant capable of setting<br>
seed wherein the seed expresses insulin wherein said insulin<br>
polypeptide accumulates within a membrane enclosed intracellular<br>
compartment within the plant cell wherein said membrane enclosed<br>
intracellular compartment is the endoplasmic reticulum (ER) or an ER<br>
derived storage vesicle and wherein at least 0.1% of the total seed<br>
protein present in the seed is insulin; and<br>
(c)	obtaining substantially pure insulin from said seeds.<br>
2.	A method as claimed in claim 1 wherein said polypeptide retaining the insulin polypeptide in the ER is selected from the group consisting of KDEL, HDEL, DDEL, ADEL and SDEL.<br>
3.	A method as claimed in claim 1 wherein said polypeptide retaining the insulin polypeptide in the ER is selected from the group consisting of SEQ ID NO:150, SEQ ID NO:151, SEQ ID NO:152, SEQ ID NO:153 and SEQ ID NO:154.<br><br>
4.	A method as claimed in claims 2 or 3 wherein said insulin polypeptide additionally comprises a nucleic acid sequence encoding a signal peptide.<br>
5.	A method as claimed in claim 4 wherein said signal peptide is a tobacco pathogenesis related protein (PR-S) signal sequence.<br>
6.	A method as claimed in claim 4 wherein said signal sequence is SEQ.ID.NO.:161.<br>
7.	A method as claimed in claim 1 wherein said ER-derived storage organelle is an oil body.<br>
8.	A method as claimed in claim 1 wherein said polypeptide retaining the insulin polypeptide in an ER derived storage organelle is an oil body protein.<br>
9.	A method as claimed in claim 8 wherein said oil body protein is selected from the group of oil body proteins consisting of oleosin, caleosin and steroleosin.<br>
10.	A method as claimed in claim 8 wherein said oil body protein is selected from the group consisting of SEQ ID NO:156, SEQ ID NO:157 and SEQ ID NO:158, SEQ ID NO:159, and SEQ ID NO:160.<br>
11.	A method as claimed in claim 1 wherein said chimeric nucleic acid additionally contains a nucleic acid sequence encoding a stabilizing protein fused in reading frame to the nucleic acid sequence encoding insulin.<br>
12.	A method as claimed in claim 11 wherein said chimeric nucleic acid additionally contains a nucleic acid sequence encoding a signal peptide sequence fused in reading frame to the nucleic acid sequence encoding insulin.<br><br>
13.	A method as claimed in claim 12 wherein said signal peptide is a tobacco pathogenesis related protein (PR-S) signal sequence.<br>
14.	A method as claimed in claim 13 wherein said signal peptide is SEQ.ID.NO..-161.<br>
15.	A method as claimed in claim 11 wherein said nucleic acid encoding said stabilizing protein permits association of the insulin polypeptide with the oil body upon harvesting and grinding of the seed.<br>
16.	A method as claimed in claim 15 wherein said stabilizing protein encodes a single chain antibody with specificity to an oil body.<br>
17.	A method as claimed in claim 11 wherein the nucleic acid sequence encoding a stabilizing protein fused in reading frame to the nucleic acid sequence encoding insulin is selected from the group of polypeptides consisting of a single chain antibody and cholera toxin B subunit.<br>
18.	A method as claimed in claim 1 wherein the chimeric nucleic acid sequence is introduced in into the plant cell under nuclear genomic integration conditions.<br>
19.	A method as claimed in claim 1 wherein the seed-preferred promoter is a phaseolin promoter.<br>
20.	A method as claimed in claims 1, 2, 3 and 7 to 18 wherein the nucleic acid sequence encoding insulin is selected from the group of nucleic acid sequences consisting of human insulin, porcine insulin and bovine insulin.<br>
21.	A method as claimed in claims 1, 2, 3 and 7 to 20 wherein the nucleic acid encoding insulin is a mini-insulin.<br><br>
22.     A method to any one of claims 1, 2, 3 and 7 to 21 wherein the nucleic acid sequence encoding insulin is optimized for plant codon usage.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQWJzdHJhY3QtKDA2LTAzLTIwMTIpLnBkZg==" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Abstract-(06-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtYWJzdHJhY3QucGRm" target="_blank" style="word-wrap:break-word;">225-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQ2xhaW1zLSgwNi0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Claims-(06-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQ2xhaW1zLSgyMS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Claims-(21-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtQ2xhaW1zLSgyNS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-Claims-(25-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtY2xhaW1zLnBkZg==" target="_blank" style="word-wrap:break-word;">225-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UgT3RoZXJzLSgwNi0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Correspondence Others-(06-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLSgyMS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Correspondence-Others-(21-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtQ29ycmVzcG9uZGVuY2UtT3RoZXJzLnBkZg==" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Correspondence-Others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZGVzY3JpcHRpb24gKGNvbXBsZXRlKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZHJhd2luZ3MucGRm" target="_blank" style="word-wrap:break-word;">225-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtRm9ybS0xLnBkZg==" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtRm9ybS0xOC5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZm9ybS0yLnBkZg==" target="_blank" style="word-wrap:break-word;">225-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtRm9ybS0zLSgwNi0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Form-3-(06-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZm9ybS0zLnBkZg==" target="_blank" style="word-wrap:break-word;">225-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZm9ybS01LnBkZg==" target="_blank" style="word-wrap:break-word;">225-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtR1BBLSgyMS0wOS0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-GPA-(21-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtZ3BhLnBkZg==" target="_blank" style="word-wrap:break-word;">225-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtcGN0LTIxMC5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-pct-210.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtcGN0LTIzNy5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-pct-237.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtcGN0LTMwNC5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtcGN0LTMyNi5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-pct-326.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LWRlbG5wLTIwMDYtcGN0LTM3My5wZGY=" target="_blank" style="word-wrap:break-word;">225-delnp-2006-pct-373.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1LURFTE5QLTIwMDYtUGV0aXRpb24tMTM3LSgwNi0wMy0yMDEyKS5wZGY=" target="_blank" style="word-wrap:break-word;">225-DELNP-2006-Petition-137-(06-03-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjI1Mi1kZWxucC0yMDEwLUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMjUtMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2252-delnp-2010-Correspondence-Others-(25-09-2012).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="256153-continuous-connection-profile-for-attachment-of-sheet-piles-to-supporting-elements.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256155-a-solid-laundry-detergent-composition-comprising-anionic-detersive-surfactant-and-a-highly-porous-carrier-material.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256154</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>225/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>19/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-May-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-May-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>12-Jan-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>SEMBIOSYS GENETICS, INC.</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>BAY #110, 2985-23rd AVENUE, N.E., CALGARY, ALBERTA T1Y 7L3, CANADA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>BOOTHE, JOSEPH</td>
											<td>#302, 332 6th AVENUE N.W., CALGARY, ALBERTA T2E 0L9, CANADA</td>
										</tr>
										<tr>
											<td>2</td>
											<td>NYKIFORUK, CORY</td>
											<td>23 HARVEST ROSE CIRCLE, N.W., CALGARY, ALBERTA T1K 4M5, CANADA</td>
										</tr>
										<tr>
											<td>3</td>
											<td>MOLONEY, MAURICE M.</td>
											<td>34 EDGEBROOKE COVE, N.W., CALGARY, ALBERTA, T3A 5N5, CANADA.</td>
										</tr>
										<tr>
											<td>4</td>
											<td>KEON, RICHARD</td>
											<td>302, 602 2nd AVENUE, N.W., CALGARY, ALBERTA T2N 0E1, CANADA</td>
										</tr>
										<tr>
											<td>5</td>
											<td>VAN ROOIJEN, GIJS</td>
											<td>3223 BEARSPAW DRIVE N.W., CALGARY, ALBERTA, T2L 1T1, CANADA</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N 15/82</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/CA2004/000896</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-06-17</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>60/549,539</td>
									<td>2004-03-04</td>
								    <td>U.S.A.</td>
								</tr>
								<tr>
									<td>2</td>
									<td>60/478,818</td>
									<td>2003-06-17</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256154-methods-for-the-production-of-insulin-in-plants by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:51:58 GMT -->
</html>
