<!DOCTYPE html>
<html lang="en">
  
<!-- Mirrored from www.allindianpatents.com/patents/256158-dna-cloning-vector-plasmids-and-methods-for-their-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:51:41 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Indian Patents. 256158:&quot;DNA CLONING VECTOR PLASMIDS AND METHODS FOR THEIR USE.&quot;</title>
    <meta content="authenticity_token" name="csrf-param" />
<meta content="cYcP52B8zyTWKbLwby2YPh9z/gvY/RLjWOwY4YXkiXg=" name="csrf-token" />

    <!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
    <!--[if lt IE 9]>
      <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
    <![endif]-->

    <link href="../assets/application-e80cf34975c5b1730c80b2f7170e7d26.css" media="all" rel="stylesheet" type="text/css" />

  </head>
  <body>

    <div class="navbar navbar-fluid-top">
      <div class="navbar-inner">
        <div class="container-fluid">
          <a class="btn btn-navbar" data-target=".nav-collapse" data-toggle="collapse">
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </a>
          <a class="brand" href="../index.html">Indian Patents</a>
          <div class="container-fluid nav-collapse">
            <ul class="nav">
              <li><a href="../recently-granted.html">Recently Granted Patents</a></li>
              <li><a href="../recently-published.html">Recently Published Patents</a></li>
            </ul>
            <form id="gform" class="navbar-search pull-right" action="https://www.google.com/search" method="get" target="_blank" onsubmit="document.getElementById('gform').q.value='site:http://www.allindianpatents.com '+document.getElementById('gform').q.value">
                <input type="text" name="q" id="q" class="search-query" placeholder="Search" onclick="this.value=''" autocomplete="off">
            </form>
          </div><!--/.nav-collapse -->
        </div>
      </div>
    </div>

    <div class="container-fluid">
      <div class="row-fluid">
        <div class="span12">

          <style>
          .allindianpatents-top { width: 320px; height: 50px; }
          @media(min-width: 500px) { .allindianpatents-top { width: 468px; height: 60px; } }
          @media(min-width: 800px) { .allindianpatents-top { width: 728px; height: 90px; } }
          </style>
          <center>
          </center>
          
          <div class="row-fluid">
	<div class="span8">

		<table class="table">
			<tr>
				<th>Title of Invention</th>
				<td><h1 style="font-size:large;">&quot;DNA CLONING VECTOR PLASMIDS AND METHODS FOR THEIR USE.&quot;</h1></td>
			</tr>
			<tr>
				<th>Abstract</th>
				<td>A group of cloning vector plasmids for use in constructing DNA molecules, such as transgenes, for the purpose of gene expression or analysis of gene expression. The present invention is also a method for using the cloning vector plasmids in a variable series of cloning steps to produce a final transgene product. The plasmid cloning vectors are engineered to minimize the amount of manipulation of DNA fragment components by the end user of the vectors and the methods for their use. Transgenes produced using the invention may be used in a single organism, or in a variety of organisms including bacteria, yeast, mice, and othereukaryotes with little or no further modification.</td>
			</tr>
		</table>

					<style>
					.allindianpatents-post-abstract { width: 320px; height: 50px; }
					@media(min-width: 880px) { .allindianpatents-post-abstract { width: 468px; height: 60px; } }
					@media(min-width: 1267px) { .allindianpatents-post-abstract { width: 728px; height: 90px; } }
					</style>
					<center>
					<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
					<!-- AllIndianPatents-post-abstract -->
					<ins class="adsbygoogle allindianpatents-post-abstract"
					     style="display:inline-block"
					     data-ad-client="ca-pub-7914358224572760"
					     data-ad-slot="9152759240"></ins>
					<script>
					(adsbygoogle = window.adsbygoogle || []).push({});
					</script>					
					</center>

		<table class="table">
			<tr>
				<th>Full Text</th>
				<td>DNA Cloning Vector Plasmids and Methods for Their Use<br>
FIELD OF INVENTION<br>
[0001 ] The present invention relates to the field of cloning vector plasmids, and to the use of cloning vector plasmids to build DNA constructs or transgenes.<br>
BACKGROUND OF THE INVENTION<br>
[0002] The foundation of moleoular biology is recombinant DNA technology, which can here be summarized as the modification and propagation of nucleic acids for the purpose of studying the structure and function of the nucleic acids and their protein products.<br>
[0003] Individual genes, gene regulatory regions, subsets of genes, and indeed entire chromosomes in which they are contained, are all comprised of double-stranded anti-parallel sequences of nucleotides identified conventionally by the initials A, T. G, and C. These DNA sequences, as well as cDNA sequences derived from mRNA molecules, can be cleaved into distinct fragments, isolated, and inserted into a vector such as a bacterial plasmid to study the gene products. A plasmid is an extra-chromosomal piece of DNA that was originally derived from bacteria, and can be manipulated and reintroduced into a host bacterium for the purpose of study or production of a gene product. The DNA of a plasmid is similar to all chromosomal DNA, in that it is composed of the same A, T, G, and C nucleotides encoding genes and gene regulatory regions, however, it is a relatively small molecule comprised of less than approximately 30,000 base-pairs, or 30 kilobases (kb). In addition, the nucleotide base pairs of a double-stranded plasmid form a continuous circular molecule, also distinguishing plasmid DNA from that of chromosomal DNA.<br>
[0004] Plasmids enhance the rapid exchange of genetic material between bacterial organisms and allow rapid adaptation to changes in environment, such as temperature, food supply, or other challenges. Any plasmid acquired must express a gene or genes that contribute to the survival of the host, or it will be destroyed or discarded by the organism since the maintenance of unnecessary plasmids would be a wasteful use of resources.   A clonal population of cells contains identical genetic material, including any plasmids it might harbor. Use of a cloning vector plasmid with a DNA insert in such a clonal<br>
population of host cells will amplify the amount of the DNA of interest available. The * * DNA so cloned may then be isolated and recovered for subsequent manipulation in the<br>
steps required for building a DNA construct. Thus, it can be appreciated that cloning vector plasmids are useful tools in the study of gene fuction.<br>
[0005] While some elements found in plasmids are naturally occurring, others have been engineered to enhance the usefulness of plasmids as DNA vectors. These include antibiotic- or chemical-resistance genes and a multiple cloning site (MCS), among others. Each of these elements has a role in the present invention, as well as in the prior art. Description of the role each element plays will highlight the limitations of the prior art and demonstrate the utility of the present invention.<br>
(0006) A particularly useful plasmid-born gene that can be acquired by a host is one that would confer antibiotic resistance. In the daily practice of recombinant DNA technology, antibiotic resistance genes are exploited as positive or negative selection elements to preferentially enhance the culture and amplification of the desired plasmid over that of other plasmids.<br>
[0007] In order to be maintained by a host bacterium a plasmid must also contain a segment of sequences that direct the host to duplicate the plasmid. Sequences known as the origin of replication (ORI) element direct the host to use its cellular enzymes to make copies of the plasmid. When such a bacterium divides, the daughter cells will each retain a copy or copies of any such plasmid. Certain strains of E. coli bacteria have been derived to maximize this duplication, producing upwards of 300 copies per bacterium. In this manner, the cultivation of a desired plasmid can be enhanced.<br>
[0008] Another essential element in any cloning vector is a location for insertion of the genetic materials of interest. This is a synthetic element that has been engineered into "wild type" plasmids, thus conferring utility as a cloning vector. Any typical commercially-available cloning vector plasmid contains at least one such region, known as a multiple cloning site (MCS).   A MCS typically comprises nucleotide sequences that may be cleaved by a single or a series of restriction endonuclease enzymes (hereafter referred to as "restriction enzymes"), each of which has a distinct recognition sequence and cleavage pattern. The so-called recognition sequences (which are referred to as restriction enzyme "sites") encoded in the DNA molecule comprise a double-stranded palindrornic sequence.   For some restriction enzymes, as few as 4-6 nucleotides are<br>
sufficient to provide a recognition site, while some restriction enzymes require a sequence dr8 or more nucleotides. Hie enzyme EcoRl, for example, recognizes the hexanucleotide sequence:J&gt; G-A-A-T-T-C3&gt;, wherein 5' indicates the end of the molecule known by convention as the "upstream" end, and 3' likewise indicates the "downstream" end. The complementary strand of the recognition sequence would be its anti-parallel strand,3 G-A-A-T-T-C-'. Thus the double stranded recognition site can be represented within the larger double-stranded molecule in which it occurs as:<br>
5'	G-A-A-T-T-C	3<br>
3   	C-T-T-A-A-G	5<br>
[0009] Like many other restriction enzymes, EcoRl does not cleave exactly at the axis of dyad symmetry, but at positions four nucleotides apart in the two DNA strands between the nucleotides indicated by a "/":<br>
v	G/A-A-T-T-C	3'<br>
3'	C-T-T-A-A/G	5'<br>
such that double-stranded DNA molecule is cleaved and has the resultant configuration of nucleotides at the newly formed "ends":<br>
5	G 3	5 A-A-T-T-C	3<br>
3	C-T-T-A-A 5'	3 G	5<br>
[0010] This staggered cleavage yields fragments of DNA with protruding 5' termini. Because A-T and G-C pairs are spontaneously formed when in proximity with each other, protruding ends such as these are called cohesive or sticky ends. Any one of these termini can form hydrogen bonds with any other complementary termini cleaved with the same restriction enzyme. Since any DNA that contains a specific recognition sequence will be cut in the same manner as any other DNA containing the same sequence, those cleaved ends will be complementary.  Therefore, the ends of any DNA molecules cut with the same restriction enzyme "match" each other in the way adjacent pieces of a jigsaw puzzle "match", and can be enzymatically linked together. It is this property that permits the formation of recombinant DNA molecules, and allows the introduction of foreign DNA fragments into bacterial plasmids, or into any other DNA molecule.<br>
[0011 ] A further general principle to consider when building recombinant DNA molecules is that all restriction sites occurring within a molecule will be cut with a particular restriction enzyme, not just the site of interest. The larger a DNA molecule, the<br>
more likely it is that any restriction site will reoccur. Assuming that any restriction sites Pare distributed randomly along a DNA molecule, a tetranucleotide site will occur, on the average, once every 44 (i.e., 256) nucleotides, whereas a hexanucleotide site will occur once every 46 (i.e., 4096) nucleotides, and octanucleotide sites will occur once every 48 (i.e., 114,688) nucleotides. Thus, it can be readily appreciated that shorter recognition sequences will occur frequently, while longer ones will occur rarely. When planning the construction of a transgene or other recombinant DNA molecule, this is a vital issue, since such a project frequently requires the assembly of several pieces of DNA of varying sizes. The larger these pieces are, the more likely that the sites one wishes to use occur in several pieces of the DNA components, making manipulation difficult, at best.<br>
[0012] Frequently occurring restriction enzymes are herein referred to as common restriction enzymes, and their cognate sequences are referred to as common restriction sites. Restriction enzymes with cognate sequences greater than 6 nucleotides are referred to as rare restriction enzymes, and their cognate sites as rare restriction sites. Thus, the designations "rare" and common" do not refer to the relative abundance or availability of any particular restriction enzyme, but rather to the frequency of occurrence of the sequence of nucleotides that make up its cognate recognition site within any DNA molecule or isolated fragment of a DNA molecule, or any gene or its DNA sequence.<br>
[0013] A second class of restriction endonuclease enzymes has recently been isolated, called homing endonuclease (HE) enzymes. HE enzymes have large, asymmetric recognition sites (12-40 base pairs). HE recognition sites are extremely rare. For example, the HE known as I-Scel has an 18 bp recognition site (5'...TAGGGATAACAGGGTAAT...3'), predicted to occur only once in every 7x 10'° base pairs of random sequence. This rate of occurrence is equivalent to only one site in 20 mammalian-sized genomes. The rare nature of HE sites greatly increases the likelihood that a genetic engineer can cut a final transgene product without disrupting the integrity of the transgene if HE sites were included in appropriate locations in a cloning vector plasmid.<br>
[0014] Since a DNA molecule from any source organism will be cut in identical fashion by its cognate restriction enzyme, foreign pieces of DNA from any species can be cut with a restriction enzyme, inserted into a bacterial plasmid vector that was cleaved with the same restriction enzyme, and amplified in a suitable host cell. For example, a<br>
human gene may be cut in 2 places with EcoRl, the fragment with EcoRl ends isolated •nd mixed with a plasmid that was also cut with EcoRl in what is commonly known as a ligation reaction or ligation mixture. Under the appropriate conditions in the ligation mixture, some of the isolated human gene fragments will match up with the ends of the plasmid molecules. These newly joined ends can link together and enzymatically recircularize the plasmid, now containing its new DNA insert. The ligation mixture is then introduced into E. coli or another suitable host, and the newly engineered plasmids will be amplified as the bacteria divide.  In this manner, a relatively large number of copies of the human gene may be obtained and harvested from the bacteria. These gene copies can then be further manipulated for the purpose of research, analysis, or production of its gene product protein.<br>
|0015 j Recombinant DNA technology is frequently embodied in the generation of so-called "iransgenes". Transgenes frequently comprise a variety of genetic materials that are derived from one or more donor organisms and introduced into a host organism. Typically, a transgene is constructed using a cloning vector as the starting point or "backbone" of the project, and a series of complex cloning steps are planned to assemble the final product within that vector. Elements of a transgene, comprising nucleotide sequences, include, but are not limited to 1) regulatory promoter and/or enhancer elements, 2) a gene that will be expressed as a mRNA molecule, 3) DNA elements that provide mRNA message stabilization, 4) nucleotide sequences mimicking mammalian intronic gene regions, and 5) signals for mRNA processing such as the poly-A tail added to the end of naturally-occurring mRNAs. In some cases, an experimental design may require addition of localization signal to provide for transport of the gene product to a particular subcellular location. Each of these elements is a fragment of a larger DNA molecule that is cut from a donor genome, or, in some cases, synthesized in a laboratory. Each piece is assembled with the others in a precise order and 5'-3' orientation into a cloning vector plasmid.<br>
[0016] The promoter of any gene may be isolated as a DNA fragment and placed within a synthetic molecule, such as a plasmid, to direct the expression of a desired gene, assuming that the necessary conditions for stimulation of the promoter of interest can be provided. For example, the promoter sequences of the insulin gene may be isolated, placed in a cloning vector plasmid along with a reporter gene, and used to study the<br>
conditions required for expression of the insulin gene in an appropriate cell type. Arernatively, the insulin gene promoter may be joined with the protein coding-sequence of any gene of interest in a cloning vector plasmid, and used to drive expression of the gene of interest in insulin-expressing cells, assuming that all necessary elements are present within the DNA transgene so constructed.<br>
[0017] A reporter gene is a particularly useful component of some types of transgenes.   A reporter gene comprises nucleotide sequences encoding a protein that will be expressed under the direction of a particular promoter of interest to which it is linked in a transgene, providing a measurable biochemical response of the promoter activity. A reporter gene is typically easy to detect or measure against the background of endogenous cellular proteins. Commonly used reporter genes include but are not limited to LacZ. green fluorescent protein, and luciferase, and other reporter genes, many of which are well-known to those skilled in the art.<br>
[0018] Introns are not found in bacterial genomes, but are required for proper formation of mRNA molecules in mammalian cells. Therefore, any DNA construct for use in mammalian systems must have at least one intron. Introns may be isolated from any mammalian gene and inserted into a DNA construct, along with the appropriate splicing signals that allow mammalian cells to excise the intron and splice the remaining mRNA ends together,<br>
[0019] An mRNA stabilization element is a sequence of DNA that is recognized by binding proteins that protect some mRNAs from degradation. Inclusion of an mRNA stabilization element will frequently enhance the level of gene expression from that mRNA in some mammalian cell types, and so can be useful in some DNA constructs or transgenes. An mRNA stabilization element can be isolated from naturally occurring DNA or RNA, or synthetically produced for inclusion in a DNA construct.<br>
[0020] A localization signal is a sequence of DNA that encodes a protein signal for subcellular routing of a protein of interest. For example, a nuclear localization signal will direct a protein to the nucleus; a plasma membrane localization signal will direct it to the plasma membrane, etc. Thus, a localization signal may be incorporated into a DNA construct to promote the translocation of its protein product to the desired subcellular location.<br>
[0021 ] A tag sequence may be encoded in a DNA construct so that the protein j|fcduct will have an unique region attached. This unique region serves as a protein tag that can distinguish it from its endogenous counterpart.  Alternatively, it can serve as an identifier that may be detected by a wide variety of techniques well known in the art, including, but not limited to, RT-PCR, immunohistochemistry, or in situ hybridization.<br>
[0022] With a complex transgene, or with one that includes particularly large regions of DNA, there is an increased likelihood that there will be multiple recognition sites in these pieces of DNA. Recall that the recognition sequences encoding any one hexanucleotide site occur every 4096 bp. If a promoter sequence is 3000 bp and a gene of interest of 1500 bp are to be assembled into a cloning vector of 3000 bp, it is statistically very likely that many sites of 6 or less nucleotides will not be useful, since any usable sites must occur in only two of the pieces. Furthermore, the sites must occur in the appropriate areas of the appropriate molecules that are to be assembled.    In addition, most cloning projects will need to have additional DNA elements added, thereby increasing the complexity .of the growing molecule and the likelihood of inopportune repetition of any particular site. Since any restriction enzyme will cut at all of its sites in a molecule, if a restriction enzyme site reoccurs, all the inopportune sites will be cut along with the desired sites, disrupting the integrity of the molecule. Thus, each cloning step must be carefully planned so as not to disrupt the growing molecule by cutting it with a restriction enzyme that has already been used to incorporate a preceding element. And finally, when a researcher wishes to introduce a completed transgene into a mammalian organism, the fully-assembled transgene construct frequently must be linearized at a unique restriction site at at least one end of the transgene, thus requiring yet another unique site found nowhere else in the construct. Since most DNA constructs are designed for a single purpose, little thought is given to any future modifications that might need to be made, further increasing the difficulty for future experimental changes.<br>
[0023] Traditionally, transgene design and construction consumes significant amounts of time and energy for several reasons, including the following:<br>
[0024] 1. There is a wide variety of restriction and HE enzymes available that will generate an array of termini, however most of these are not compatible with each other. Many restriction enzymes, such as EcoRl, generate DNA fragments with protruding 5' cohesive termini or "tails"; others (e.g., Pstl) generate fragments with 3' protruding tails,<br>
whereas still others (e.g., Ball) cleave at the axis of symmetry to produce blunt-ended Higments. Some of these will be compatible with the termini formed by cleavage with other restriction and HE enzymes, but the majority of useful ones will not. The termini that can be generated with each DNA fragment isolation must be carefully considered in designing a DNA construct.<br>
[0025] 2.   DNA fragments needed for assembly of a DNA construct or transgene must first be isolated from their source genomes, placed into plasmid cloning vectors, and amplified to obtain useful quantities. The step can be performed using any number of commercially-available or individually altered cloning vectors. Each of the different commercially available cloning vector plasmids were, for the most part, developed independently, and thus contain different sequences and restriction sites for the DNA fragments of genes or genetic elements of interest. Genes must therefore be individually tailored to adapt to each of these vectors as needed for any given set of experiments. The same DNA fragments frequently will need to be altered further for subsequent experiments or cloning into other combinations for new DNA constructs or transgenes. Since each DNA construct or transgene is custom made for a particular application with no thought or knowledge of how it will be used next, it frequently must be "retro-fitted" for subsequent applications.<br>
[0026] 3. In addition, the DNA sequence of any given gene or genetic element varies and can contain internal restriction sites that make it incompatible with currently available vectors, thereby complicating manipulation. This is especially true when assembling several DNA fragments into a single DNA construct or transgene.<br>
[0027] Thus, there is a need for a system that would allow the user to rapidly assemble a number of DNA fragments into one molecule, despite redundancy of restriction sites found at the ends and within the DNA fragments. Such a system might also provide a simple means for rapidly altering the ends of the fragments so that other restriction sequences are added to them. Inclusion of single or opposing pairs of HE restriction sites would enhance the likelihood of having unique sites for cloning. A system that would also allow easy substitutions or removal of one or ore of the fragments would add a level of versatility not currently available to users. Therefore, a "modular system, allowing one to insert or remove DNA fragments into or out of<br>
"cassette" regions flanked by rare restriction sites within the cloning vector would be pespecially useful, and welcome to the field of recombinant DNA technology.<br>
BRIEF SUMMARY OF THE INVENTION<br>
[0028] The present invention relates to a cloning vector plasmid, wherein sequential elements encoding a modular structure comprise: three non-variable and unique common restriction sites, a 5' oligonucleotide primer site, a unique HE site in a forward orientation, a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences, a fixed grouping of non-variable rare restriction sites to define a 5' portion of a promoter module, random nucleotide sequences, a fixed grouping of non-variable rare restriction sites that define a shared junction between a 3' position relative to the Promoter/intron module and a 5' position relative to an Expression module, random nucleotide sequences, a fixed grouping of non-variable rare restriction sites that define a junction of a 3' position relative to the Expression module and a 5' position relative to a 3' Regulatory module, random nucleotide sequences, a fixed grouping of non-variable rare restriction sites that define a 3' position relative to a 3' Regulatory module, a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences, a unique HE site in reverse orientation that is the same HE site as that placed 3' of the 5' oligonucleotide primer site, a 3' oligonucleotide primer site in reverse orientation, and four non-variable and unique common restriction sites that define a 3' insertion site.<br>
[0029] The present invention relates to a cloning vector plasmid , wherein sequential elements encoding a molecular structure comprise: two non-variable and unique common restriction sites that define a 5' insertion site, an oligonucleotide primer site, a pair of unique HE sites in opposite orientation flanking random nucleotide sequences, a non-variable and unique, common restriction site that allows cloning of a shuttle vector module downstream of the pair of unique HE sites, a fixed grouping of non-variable rare restriction sites, random nucleotide sequences, a fixed grouping of non-variable rare restriction sites, a unique HE site in a forward orientation, a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences, an oligonucleotide primer site, a pair of unique BstX I sites in opposite orientations (wherein the variable nucleotide region in the BstX I recognition site is defined by nucleotides identical to the non-complementary tails generated by the ordering of two identical HE<br>
recognition sites arranged in reverse-complement orientation, a pair of unique HE sites in p$&gt;posite orientations flanking random nucleotide sequences; an oligonucleotide primer site in reverse-orientation, a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences, a unique HE site in reverse orientation, with the HE site being the same as the HE site in a forward orientation, a fixed grouping of non-variable rare restriction sites, random nucleotide sequences, a fixed grouping of non-variable rare restriction sites, a non-variable and unique, common restriction site, a pair of unique HE sites in opposite orientation flanking random nucleotide sequences, an oligonucleotide primer site in reverse orientation, and three non-variable and unique common restriction sites.<br>
|0030] The present invention also relates to a method for constructing a transgene, comprising the steps of: providing a cloning vector plasmid comprising a sequential arrangement of Promoter, Expression, and 3' Regulatory modules; introducing first nucleotide sequences to be included in the transgene into a shuttle vector; transferring the first nucleotide sequences from the shuttle vector to the cloning vector plasmid comprising modular elements; sequentially introducing any other desired nucleotide sequences into another shuttle vector; and transferring the other desired nucleotide sequences into the cloning vector plasmid which received the first nucleotide sequences.<br>
BRIEF DESCRIPTION OF THE DRAWINGS<br>
[00311 FIGURE 1 is a linear map of the module concept of the invention.<br>
[0032] FIGURE 2 is a Docking Plasmid map.<br>
[0033] FIGURE 3 is a linear restriction map illustrating an example of restriction enzyme sites that can be included in the Docking Plasmid MCS.<br>
[0034] FIGURE 4 is a Primary Docking Plasmid map.<br>
[0035] FIGURE 5 is a linear restriction map illustrating an example of restriction enzyme sites that can be included in the Primary Docking Plasmid MCS.<br>
[0036] FIGURE 6 is a Shuttle Vector P (SVP) plasmid map<br>
[0037] FIGURE 7 is a linear restriction site map illustrating an example of restriction enzyme sites that can be included in the SVP MCS.<br>
[0038] FIGURE 8 is a Shuttle Vector E (SVE) plasmid map.<br>
[0039] FIGURE 9 is a linear restriction site map illustrating an example of |%estriction enzyme sites that can be included in the SVE MCS. [0040] FIGURE 10 is a Shuttle Vector 3' (SV3) map. [0041] FIGURE 11 is a linear restriction site map illustrating an example of restriction enzyme sites that can be included in the SV3 MCS.<br>
BRIEF DESCRIPTION OF SEQUENCE LISTING<br>
[0042] SEQ:ID 01 is an example of a nucleotide sequence for a PE3 Docking Plasmid MCS.<br>
[0043] SEQ:ID 02 is an example of a nucleotide sequence for a PE3 Docking Plasmid.<br>
(0044] SEQ:ID 03 is an example of a nucleotide sequence for a Primary Docking Plasmid MCS.<br>
[0045] SEQ:ID 04 is an example of a nucleotide sequence for a Primary Docking Plasmid.<br>
[0046] SEQ:ID 05 is an example of a nucleotide sequence for a SVP Plasmid MCS.<br>
[0047] SEQ:ID 06 is an example of a nucleotide sequence for a SVP Plasmid.<br>
|0048] SEQ:ID 07 is an example of a nucleotide sequence for a SVE Plasmid MCS.<br>
[0049] SEQ:ID 08 is an example of a nucleotide sequence for a SVE Plasmid.<br>
[0050] SEQ:ID 09 is an example of a nucleotide sequence for a SV3 Plasmid MCS.<br>
[0051 ] SEQ:ID 10 is an example of a nucleotide sequence for a SV3 Plasmid.<br>
DEFINITIONS OF TERMS USED TO DESCRIBE THE INVENTION [0052] As used herein, the terms "cloning vector" and "cloning vector plasmid" are used interchangeably to refer to a circular DNA molecule minimally containing an Origin of Replication, a means for positive selection of host cells harboring the plasmid such as an antibiotic-resistance gene; and a multiple cloning site.<br>
[0053] As used herein, the term "Origin of Replication" (ORI) refers to nucleotide uences that direct replication or duplication of a plasmid within a host cell<br>
[0054] As used herein, the term "multiple cloning site" refers to nucleotide sequences comprising restriction sites for the purpose of cloning DNA fragments into a cloning vector plasmid.<br>
[0055] As used herein, the term "cloning" refers to the process of ligating a DNA<br>
molecule into a plasmid and transferring it an appropriate host cell for duplication during<br>
* propagation of the host.<br>
[0056] As used herein, the term "DNA construct" refers to a DNA molecule synthesized by consecutive cloning steps within a cloning vector plasmid, and is commonly used to direct gene expression in any appropriate cell host such as cultured cells in vitro, or a transgenic mouse in vivo. A transgene used to make such a mouse can also be referred to as a DNA construct, especially during the period of time when the transgene is being designed and synthesized.<br>
10057] As used herein, the term "Shuttle Vector" refers to a specialized cloning vector plasmid used in the invention to make an intermediate molecule that will modify the ends of a DNA fragment.<br>
[0058] As used herein, the term "Docking Plasmid" refers to a specialized cloning vector plasmid used in the invention to assemble DNA fragments into a DNA construct.<br>
[0059] As used herein, the terms "restriction endonuclease" or "restriction enzyme" refers to a member or members of a classification of catalytic molecules that bind a cognate sequence of DNA and cleave the DNA molecule at a precise location within that sequence.<br>
[0060] As used herein, the terms "cognate sequence" or "cognate sequences" refer to the minimal string of nucleotides required for a restriction enzyme to bind and cleave a DNA molecule or gene.<br>
|0061] As used herein, the term "DNA fragment" refers to any isolated molecule of DNA, including but not limited to a protein-coding sequence, reporter gene, promoter, enhancer, intron, exon, poly-A tail, multiple cloning site, nuclear localization signal, or mRNA stabilization signal, or any other naturally occurring or synthetic DNA molecule. Alternatively, a DNA fragment may be completely of synthetic origin, produced in vitro.<br>
Furthermore, a DNA fragment may comprise any combination of isolated naturally<br>
"Ti<br>
J&gt;bccurring and/or synthetic fragments.<br>
[0062] As useu herein, the terms "gene promoter" or "promoter" (P) refer to a nucleotide sequence required for expression of a gene.<br>
[0063] As used herein, the term "enhancer region" refers to a nucleotide sequence that is not required for expression of a target gene, but will increase the level of gene expression under appropriate conditions.<br>
[0064] As used herein, the term "reporter gene" refers to a nucleotide sequences encoding a protein useful for monitoring the activity of a particular promoter of interest.<br>
[0065] As used herein, the term "chromatin modification domain" (CMD) refers to nucleotide sequences that interact with a variety of proteins associated with maintaining and/or altering chromatin structure.<br>
[0066| As used herein, the term "poly-A tail" refers to a sequence of adenine (A) nucleotides commonly found at the end of messenger RNA (mRNA) molecules. A Poly-A tail signal is incorporated into the 3' ends of DNA constructs or transgenes to facilitate expression of the gene of interest.<br>
[0067] As used herein, the term "intron" refers to the nucleotide sequences of a non-protein-coding region of a gene found between two protein-coding regions or exons.<br>
[0068] As used herein, the term "untranslated region" (UTR) refers to nucleotide sequences encompassing the non-protein-coding region of an mRNA molecule. These untranslated regions can reside at the 5' end (5' UTR) or the 3' end (3' UTR) an mRNA molecule.<br>
[0069] As used herein, the term "mRNA stabilization element" refers a sequence of DNA that is recognized by binding proteins thought to protect some mRNAs from degradation.<br>
[0070] As used herein, the term "localization signal" (LOG) refers to nucleotide sequences encoding a signal for subcellular routing of a protein of interest.<br>
[0071] As used herein, the term "tag sequence" (TAG) refers to nucleotide sequences encoding a unique protein region that allows it to be detected, or in some cases, distinguished from any endogenous counterpart.<br>
[0072] As used herein, the term "primer site" refers to nucleotide sequences that perve as DNA templates onto which single-stranded DNA oligonucleotides can anneal for the purpose of initiating DNA sequencing, PCR amplification, and/or RNA transcription.<br>
[0073] As used herein, the term "gene expression host selector gene" (GEH-S) refers to a genetic element that can confer resistance or toxicity to cells or organisms when treated with an appropriate antibiotic or chemical.<br>
[0074] As used herein, the term "recombination arm" refers to nucleotide sequences that facilitate the homologous recombination between transgenic DNA and genomic DNA. Successful recombination requires the presence of a left recombination arm (LRA) and a right recombination arm (RRA) flanking a region of transgenic DNA to be incorporated into a host genome via homologous recombination.<br>
[0075] As used herein, the term "pUC19" refers to a plasmid cloning vector well-known to those skilled in the art, and can be found in the NCBI Genbank database as Accession # L09137.<br>
•[0076] As used herein, the term "random nucleotide sequences" refers to any combination of nucleotide sequences that do not duplicate sequences encoding other elements specified as components of the same molecule.<br>
[0077] As used herein, the term "unique" refers to any restriction endonuc lease .or HE site that is not found elsewhere within a DNA molecule.<br>
DETAILED DESCRIPTION OF THE INVENTION<br>
[0078] The present invention is a group of cloning vectors optimized to reduce the amount of manipulation frequently needed to assemble a variety of DNA fragments into a de novo DNA construct or transgene. The primary vector, herein referred to as a Docking Plasmid, contains a multiple cloning site (MCS) with 3 sets of rare restriction and/or HE sites arranged in a linear pattern.  This arrangement defines a modular architecture that allows the user to assemble multiple inserts into a single transgene construct without disturbing the integrity of DNA elements already incorporated into the Docking Plasmid in previous cloning steps.<br>
[0079] Two recognition sites for at least three HE are placed in opposite orientation to flank three modular regions for the purpose of creating a gene cassette acceptor site that cannot self-anneal. Because HE sites are asymmetric and non-<br>
palindromic, it is possible to generate non-complementary protruding 3' cohesive tails by placing two HE recognition sites in opposite orientation. Thus, the HE I-Scel cuts its cognate recognition site as indicated by "/":<br>
5'...TAGGGATAA/CAGGGTAAT...3' ,<br>
3'...ATCCC/TATTGTCCCATTA...5' .<br>
[0080] The reverse placement of a second site within an MCS would generate two non-complementary cohesive protruding tails:<br>
5'...TAGGGATAA	CCCTA...3'<br>
3'...ATCCC	AATAGGGAT...5'.<br>
[0081] This is particularly useful when it is necessary to subclone larger transgenes into a vector. Due to the size of the insert, it is thermodynamically more favorable for a vector to self anneal rather than accept a large insert. The presence of non-complementary tails generated by this placement of restriction sites provides chemical forces to counteract the thermodynamic inclination for self-ligation.<br>
[0082] The asymmetric nature of most HE protruding tails also creates a powerful cloning tool when used in combination with the BstX I restriction enzyme site (5' CCANNNNN/NTGG 3'). The sequence-neutral domain of BstX I can be used to generate compatible cohesive ends for two reverse-oriented HE protruding tails, while precluding self-annealing.<br>
BstX I (I-Sce I Fwd.)      I-Sce I Forward         I-Sce I Reverse       BxtX I (I-Sce I Rev.)<br>
5'-CCAGATAA	CAGGGTAAT//ATTACCCTGTTAT	GTGG-3'<br>
3'-GGTC	TATTGTCCCATTA//TAATGGGAC	AATACACC-5-<br>
[0083] The secondary vectors of the invention, herein known as Shuttle vectors, contain multiple cloning sites with common restriction sites flanked by rare restriction and/or HE sites. The shuttle vectors are designed for cloning fragments of DNA into the common restriction sites between the rare sites. The cloned fragments can subsequently be released by cleavage at the rare restriction or HE site or sites, and incorporated into the Docking Plasmid using the same rare restriction and/or HE site or sites found in the shuttle vectors.<br>
[0084] Thus, unlike conventional cloning vectors, the design of the MCS allows "cassettes" or modules of DNA fragments to be inserted into the modular regions of the<br>
Docking Plasmid. Likewise, each can be easily removed using the same rare restriction and/or HE enzymes, and replaced with any other DNA fragment of interest. This feature allows the user to change the direction of an experimental project quickly and easily without having to rebuild the entire DNA construct. Thus, the cloning vector plasmids of the present invention allow the user to clone a DNA fragment into an intermediate vector using common restriction sites, creating a cassette-accepting module, and to then transfer that fragment to the desired modular spot in the final construct by means of rare restriction sites. Furthermore, it allows future alterations to the molecule to replace individual modules in the Docking Plasmid with other cassette modules.The following descriptions highlight distinctions of the present invention compared with the prior art.<br>
[0085] Individual components of a transgene (the promoter enhancer P, expressed protein E, and/or 3' regulatory region 3) can be assembled as modules transferred from shuttle vectors into the PE3 Docking Station Plasmid. If higher orders of complexity are needed, the assembled transgenes, or other nucleotide sequences, can then be transferred into a Primary Docking Plasmid. Each of the five types of cloning vector plasmids will be explained in greater detail to provide an understanding of the components incorporated into each, beginning with the more complex PE3 Docking Station Plasmid and the Primary Docking Plasmid.<br>
[0086] The PE3 Docking Plasmid (FIGURE 2) comprises a pUC19 backbone with the following modifications, wherein the sequences are numbered according to the pUC19 Genbank sequence file, Accession # L09137:<br>
1.	Only sequences from 806 to 2617 (Afl3-Aat2) are used in the Docking<br>
Plasmid,<br>
2.	The BspHl site at 1729 in pUC19 is mutated from TCATGA to GCATGA,<br>
3.	The Acl 1 site at 1493 in pUC19 is mutated from AACGTT to AACGCT,<br>
4.	The Acll site at 1120 in pUC19 is mutated from AACGTT to CACGCT,<br>
5.	The Ahdl site in pUC19 is mutated from GACNNNNNGTC to<br>
CACNNNNNGTC,<br>
6.	Sequences encoding BspHl/I-Ppo 1/BspHl are inserted at the only remaining<br>
BspHl site in pUC19 following the mutation step 2 in the list above.<br>
[0087] The multiple cloning site (MCS) in the PE3 Docking Plasmid (FIGURE 3)<br>
n<br>
Iromprises the following sequential elements, in the order listed:<br>
1.	Three non-variable and unique common restriction sites that define a 5'<br>
insertion site for the mutated pUC19 vector described above (for example, Aat II, Blp I,<br>
andEcoO109I),<br>
2.	A T7 primer site.<br>
3.	A unique HE site (for example, I-Scel (forward orientation)),<br>
4.	A pair of non-variable and unique, common restriction sites flanking random<br>
nucleotide sequences that can serve as a chromatin modification domain acceptor module<br>
(RNAS-CMD-1) (for example, Kpn 1 and Avr II),<br>
5.	A fixed grouping of non-variable rare restriction sites that define the 5' portion<br>
of the promoter module (for example, AsiS I and SgrA I),<br>
6.	Random nucleotide sequences that can serve as a Promoter/intron acceptor<br>
module (RNAS-P).<br>
7.	A fixed grouping of non-variable rare restriction sites that define the shared<br>
junction between the 3' portion of the Promoter/intron module and the 5' portion of the<br>
Expression module (for example, PacI, AscI, and Mlul),<br>
8.	Random nucleotide sequences that can serve as an expression acceptor module<br>
(RNAS-E),<br>
9.	A fixed grouping of non-variable rare restriction sites that define the junction of<br>
the 3' portion of the Expression module and the 5' portion of the 3' Regulatory module<br>
(for example, SnaB I, Not I, and Sal I),<br><br>
10.	Random nucleotide sequences that can serve as a 3' regulatory domain<br>
acceptor module (RNAS-3),<br>
11.	A fixed grouping of non-variable rare restriction sites that define the 3'<br>
portion of the 3' Regulatory module (for example, Swa I, Rsr II, and BsiW I),<br>
12.	A pair of non-variable and unique, common restriction sites flanking a random<br>
nucleotide sequence of DNA that can serve as a chromatin modification domain acceptor<br>
module (RNAS-CMD-2) (for example, Xho I and Nhe I),<br>
13.	A unique HE site in reverse orientation that is identical to that in item 3,<br>
above,<br>
14.	A T3 primer site in reverse orientation, and<br>
15. Four non-variable and unique common restriction sites that define a 3' i$irtion site for the mutated pUC19 vector described above (for example, BspE I, Pme I, Sap I, and BspH I).<br>
[0088] The Primary Docking Plasmid (FIGURE 4) can be used to assemble two completed transgenes that are first constructed in PE3 Docking Station Plasmids, or two homology arms needed to construct a gene-targeting transgene, or to introduce two types of positive or negative selection elements. The multiple cloning site (MCS) in the Primary Docking Plasmid (FIGURE 5) comprises the following sequential elements, in the order listed:<br>
1.	Two non-variable and unique common restriction sites that define a y insertion<br>
site for the mutated pUCIQ vector described above (for example, Aat II and Blp I).<br>
2.	An M13 Rev. primer site,<br>
3.	A pair of unique HE sites in opposite orientation flanking a random nucleotide<br>
sequence of DNA that can serve as -a genome expression host selector gene acceptor<br>
module (RNAS-GEH-S1) (for example, Pl-Scel (forward orientation) and Pl-Scel<br>
(reverse orientation)),<br>
4.	A non-variable and unique, common restriction site that allows cloning of a<br>
shuttle vector module downstream of the HE pair (for example, EcoOl 091),<br>
5.	A fixed grouping of non-variable rare restriction sites that define the 5' portion<br>
a Left Recombination Arm module (for example, SgrA I and AsiS I),<br>
6.	Random nucleotide sequences that can serve as a Left Recombination Arm<br>
acceptor module (RNAS-LRA),<br>
7.	A fixed grouping of non-variable rare restriction sites that define the 3' portion<br>
of the Left Recombination Arm acceptor module (for example, PacI, Mlul, and Ascl),<br>
8.	A unique HE site (for example, I-Ceu I (forward orientation)),<br>
9.	A pair of non-variable and unique, common restriction sites flanking a random<br>
nucleotide sequence of DNA that can serve as a chromatin modification domain acceptor<br>
module (RNAS-CMD-1) (for example, Kpn I and Avr II),<br><br>
10.	AT7 primer site,<br>
11.	A pair of unique BstX I sites in opposite orientation (wherein the variable<br>
nucleotide region in the BstX I recognition site is defined by nucleotides identical to the<br>
non-complementary tails generated by the ordering of two identical HE recognition sites<br>
JL<br>
pranged in reverse-complement orientation; for example, Pl-Scel (forward orientation)<br>
and Pl-Scel (reverse orientation)) flanking a random nucleotide sequence of DNA that can serve as a complex transgene acceptor module (RNAS-PE3-1),<br>
12.	A pair of unique HE sites in opposite orientation flanking a random nucleotide<br>
sequence of DNA that can serve as a complex transgene acceptor module (RNAS-PE3-2)<br>
(for example, I-Scel (forward orientation) and I-Scel (reverse orientation)),<br>
13.	A T3 primer site in reverse-orientation,<br>
14.	A pair of non-variable and unique, common restriction sites flanking a random<br>
nucleotide sequence of DNA that can serve as a chromatin modification domain acceptor<br>
module (RNAS-CMD-2) (for example, Xho I and Nhe I),<br>
15.	A unique HE site in reverse orientation that is identical to that in item 8 above,<br>
16.	A fixed grouping of non-variable rare restriction sites that define the 5'<br>
portion a Right Recombination Arm module (for example, SnaB I, Sal I, and Not I),<br>
17.	Random nucleotide sequences that can serve as a Right Recombination Arm<br>
acceptor module (RNAS-RRA),<br>
18.	A fixed grouping of non-variable rare restriction sites that define the 3'<br>
portion of the Right Recombination Arm acceptor module (for example, Rsr II, Swa I, and<br>
BsiW I),<br>
19.	A non-variable and unique, common restriction site that allows cloning of a<br>
shuttle vector module upstream of an HE pair (for example, BspE I),<br>
20.	A pair of unique HE sites in opposite orientation flanking a random nucleotide<br>
sequence of DNA that can serve as a genome expression host selector gene acceptor<br>
module (RNAS-GEH-S2) (for example, PI-Psp I (forward orientation) and PI-Psp I<br>
(reverse orientation)),<br>
21.	An M13 Forward primer site placed in reverse orientation,<br>
22.	Three non-variable and unique common restriction sites that define a 3'<br>
insertion site for the mutated pUC19 vector described above (for example, Pme I. Sap I,<br>
and BspH I).<br>
[0089] Three cloning vector plasmids of the invention are known as Shuttle Vectors. The Shuttle Vectors, like the PE3 and Primary Docking Plasmids, are also<br><br>
constructed from a pUC19 backbone.  Just like the PE3 and Primary Docking Plasmids,<br>
ch Shuttle Vector has the same modifications to the pUC19 backbone listed as 1 through 6, above. The individual Shuttle Vectors (SV) are identified as Shuttle Vector Promoter/intron (P), Shuttle Vector Expression (E), and Shuttle Vector 3'Regulatory (3); henceforth SVP, SVE, and SV3, respectively. Each is described more fully below.<br>
Shuttle Vector P (SVP):<br>
[0090] SVP is a cloning vector plasmid that one can be used to prepare promoter and intron sequences for assembly into a transgene construct (FIGURE 6). An example of an SVP Plasmid can comprise the following sequential elements in the MCS (FIGURE 7), in the order listed:<br>
1.	Two non-variable and unique, common restriction sites that define a 5"<br>
insertion site for the mutated pUC19 vector described above (for example, Aatll and<br>
BlpI),<br>
2.	A T7 primer site,<br>
3.	A non-variable and unique, common restriction site that allows efficient<br>
cloning of a shuttle vector module downstream of the T7 primer site (for example,<br>
Eco0109I),<br>
4.	A fixed grouping of non-variable rare restriction sites that define the 5' portion<br>
of the promoter module (for example, AsiSI and SgrAI),<br>
5.	A variable MCS comprising any grouping of common or rare restriction sites<br>
that are unique to the shuttle vector (for example, the series of restriction sites illustrated<br>
in FIGURE 7),<br>
6.	A fixed grouping of non-variable rare restriction sites that define the 3" portion<br>
of the promoter module (for example, PacI, AscI, and Mlul)<br>
7.	A non-variable and unique, common restriction site that allows efficient<br>
cloning of a shuttle vector module upstream of the T3 primer site (for example, BspEI)<br>
8.	A reverse-orientation T3 primer site, and<br>
9.	Two non-variable and unique, common restriction sites that define a 3'<br>
insertion site for the mutated pUC19 vector described above (for example, Pmel and<br>
Sapl).<br>
Shuttle Vector E (SVE):<br>
[0091] This is a cloning vector plasmid that can be used to prepare sequences to be expressed by the transgene for assembly into a transgene construct (FIGURE 8). An example of an SVE plasmid can comprise the following sequential elements in the MCS (FIGURE 9), in the order listed:<br>
1.	Two non-variable and unique, common restriction sites that define a 5'<br>
insertion site for the mutated pUC19 vector described above (for example, Aatll and Blp<br>
MX<br>
2.	A T7 primer site,<br>
3.	A non-variable and unique, common restriction site that allows efficient<br>
cloning of a shuttle vector module downstream of the T7 primer site (for example,<br>
Eco0109\I),<br>
4.	A fixed grouping of non-variable rare restriction sites that define the 5' portion<br>
of the expression module (for example, PacI, AscI, and Mlul),<br>
5.	A variable MCS consisting of any grouping of common or rare restriction sites<br>
that are unique to the shuttle vector (for example, the series of restriction sites illustrated<br>
in FIGURE 9),<br>
6.	A fixed grouping of non-variable rare restriction sites that define the 3' portion<br>
of the expression module (for example, SnaBI, NotI, and Sail),<br>
7.	A non-variable and unique, common restriction site that allows efficient<br>
cloning of a shuttle vector module upstream of the T3 primer site (for example, BspEI)<br>
8.	A reverse-orientation T3 primer site, and<br>
9.	Two non-variable and unique, common restriction sites that define a 3'<br>
insertion site for the mutated pUC!9 vector described above (for example, Pmel and<br>
Sapl).<br>
Shuttle Vector 3 (SV3):<br>
[0092] This is a cloning vector plasmid that can be used to prepare 3" regulatory sequences for assembly into a transgene construct (FIGURE 10). An example of an SV3 plasmid can comprise the following elements in the MCS (FIGURE 11), in the order listed:<br>
1.	Two non-variable and unique, common restriction sites that define a 5'<br>
insertion site for the mutated pUC19 vector described above (for example, Aatll and<br>
BlpI),<br>
2.	A T7 primer site,<br>
3.	A non-variable and unique, common restriction site that allows efficient<br>
cloning of a shuttle vector module downstream of the T7 primer (for example, Eco0109I),<br>
4.	A fixed grouping of non-variable rare restriction sites that define the 5' portion<br>
of the 3' regulatory module (for example, SnaBI, NotI, and Sail),<br>
5.	A variable MCS consisting of any grouping of common or rare restriction sites<br>
that are unique to the shuttle vector (for example, the series of restriction sites illustrated<br>
in FIGURE 11),<br>
6.	A fixed grouping of non-variable rare restriction sites that define the 3' portion<br>
of the 3' regulator}!' module (for example, Swal, RsrII, and BsiWI),<br>
7.	A non-variable and unique, non-rare restriction site that allows efficient cloning<br>
of a shuttle vector module upstream of the T3 primer site (for example, BspEI),<br>
8.	A reverse-orientation T3 primer site, and<br>
9.	Two non-variable and unique, non-rare restriction sites that define a 3'<br>
insertion site for the mutated pUC19 vector described above (for example, Pmel and<br>
Sapl).<br>
[0093] While the present invention discloses methods for building transgenes in plasmid cloning vectors, similar methods can be used to build transgenes in larger extrachromosomal DNA molecules such as cosmids or artificial chromosomes, including bacterial artificial chromosomes (BAG). The wide variety of genetic elements that can be incorporated into the plasmid cloning vectors also allow transfer of the final transgene products into a wide variety of host organisms with little or no further manipulation.<br>
[0094J As an example of the method of practicing the present invention, a transgene can be constructed containing these elements:<br>
1.	Nucleotide sequences of the human promoter for surfactant protein C (SP-C),<br>
2.	Sequences encoding the protein product of the mouse gene granulocyte-<br>
macrophage colony-stimulating factor-receptor beta c (GMR(k)<br>
3.	Rabbit betaglobin intron sequences, and<br>
4.	SV40 poly-A signal.<br>
[0095] The SP-C sequences contain internal BamHl sites, and can be released f 'from its parental plasmid only with Notl and EcoRl. GMRpc has an internal Notl site,<br>
and can be cut from its parental plasmid with BamHl and Xhol. The rabbit betaglobin intron sequences can be cut out of its parental plasmid with EcoRl. The SV-40 poly-A tail can be cut from its parental plasmid with Xhol and Sacl. Because of redundancy of several of restriction sites, none of the parental plasmids can be used to assemble all the needed fragments.<br>
[0096] The steps used to build the desired transgene in the PE3 Docking Plasmid invention are as follows.<br>
1.	Since Notl and PspOMl generate compatible cohesive ends, the human SP-C<br>
promoter sequences are excised with Notl and EcoRl and cloned into the PspOMl and<br>
EcoRl sites of Shuttle Vector P.   The product of this reaction is called pSVP-SPC<br>
2.	Following propagation and recovery steps well known to those skilled in the<br>
art, the rabbit betaglobin intron sequences are cloned into the EcoRl site of pSVP-SPC.<br>
Orientation of the intron in the resultant intermediate construct is verified by sequencing<br>
the product, called pSVP-SPC-rpG.<br>
3.	The promoter and intron are excised and isolated as one contiguous fragment<br>
from pSVP-SPC-rpG using AsiSl and Ascl. Concurrently, the PE3 Docking Plasmid is<br>
cut with AsiSl and Ascl in preparation for ligation with the promoter/intron segment.<br>
The promoter/intron fragment is ligated into the Docking Plasmid, propagated, and<br>
recovered.<br>
4.	The Xhol site of the GMRpc fragment is filled in to create a blunt 3' end,<br>
using techniques well known to those skiilled in the art. It is then cloned into the BamHl<br>
site and the blunt-ended Pvu2 site of pSVP-SPC-rpG. The resultant plasmid (pDP-SPC-<br>
GMRpc-rpG) was propagated and recovered.<br>
5.	The final cloning step is the addition of the SV-40 Poly-A tail. The SV40-<br>
polyA fragment is cut out with Xhol and Sacl, as is the recipient vector pDSl-SPC-<br>
GMRpc-rbpG.   Both pieces of DNA are gel purified and recovered. A ligation mix is<br>
prepared with a 10:1 molar ratio of SV-40polyA to pDSl-SPC-GMRpc-rpG. The<br>
ligation products are propagated and harvested. The new plasmid, pDSl-SPC- GMRpc-<br>
rpG-pA contains all elements required for the transgene, including a unique restriction site<br>
at the 3' end with which the entire pDSl -SPC- GMRpc-rpG-pA plasmid can be linearized<br>
for transfection into eukaryotic cells or microinjection into the pronucleus of a fertilized ovum.<br><br><br><br><br><br><br>
We claim;<br>
1.     A cloning vector plasmid, wherein sequential elements encoding a modular structure comprise<br>
a.	three non-variable and unique common restriction sites,<br>
b.	a 5'oligonucleotide primer site,<br>
c.	a unique HE site in a forward orientation,<br>
d.	a pair of non-variable and unique, common restriction sites flanking random<br>
nucleotide sequences,<br>
e.	a fixed grouping of non-variable rare restriction sites to define a 5'portion of a<br>
promoter module,<br>
f.	random nucleotide sequences,<br>
g.	a fixed grouping of non-variable rare restriction sites that define a shared junction<br>
between a 3' position relative to the Promoter/intron module and a 5' position<br>
relative to an Expression module,<br>
h. random nucleotide sequences,<br>
i. a fixed grouping of non-variable rare restriction sites that define a junction of a 3' position relative to the Expression module and a 5' position relative to a 3' Regulatory module,<br>
j. random nucleotide sequences,<br>
k. a fixed grouping of non-variable rare restriction sites that define a 3' position relative to a 3' Regulatory module,<br><br>
1. a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences,<br>
m.a unique HE site in reverse orientation that is the same HE site as that placed 3'of the 5'oligonucleotide primer site,<br>
n. a 3'oligonucleotide primer site in reverse orientation, and<br>
o. four non-variable and unique common restriction sites that define a 3' insertion site.<br>
2.	The cloning vector plasmid as claimed in claim 1, wherein the modular structure of the sequential elements is ordered to allow introduction of three discrete regions of nucleotide sequences, such that the nucleotide sequences so introduced replace the random nucleotide sequences in the Promoter, Expression, and 3' regulatory modules.<br>
3.	A cloning vector plasmid, wherein sequential elements encoded comprise<br>
a.	two non-variable and unique common restriction sites that define a 5' insertion<br>
site,<br>
b.	an oligonucleotide primer site,<br>
c.	a pair of unique HE sites in opposite orientation flanking random nucleotide<br>
sequences,<br>
d.	a non-variable and unique, common restriction site that allows cloning of a shuttle<br>
vector module downstream of the pair of unique HE sites,<br><br>
e.	a fixed grouping of non-variable rare restriction sites,<br>
f.	random nucleotide sequences,<br>
g.	a fixed grouping of non-variable rare restriction sites,<br>
h. a unique HE site in a forward orientation,<br>
i. a   pair   of   non-variable   and   unique,   common   restriction   sites    flanking random nucleotide sequences,<br>
j. an oligonucleotide primer site,<br>
k. a pair of unique BstX I sites in opposite orientations (wherein the variable nucleotide region in the BstX I recognition site is defined by nucleotides identical to the non-complementary tails generated by the ordering of two identical HE recognition sites arranged in reverse-complement orientation,<br>
1. a pair of unique HE sites in opposite orientations flanking random nucleotide sequences<br>
m.an oligonucleotide primer site in reverse-orientation,<br>
n. a pair of non-variable and unique, common restriction sites flanking random nucleotide sequences,<br>
o. a unique HE site in reverse orientation, with the HE site being the same as the HE site in a forward orientation,<br>
p. a fixed grouping of non-variable rare restriction sites,<br>
q. random nucleotide sequences,<br><br>
r. a fixed grouping of non-variable rare restriction sites,<br>
s. a non-variable and unique, common restriction site,<br>
t. a pair of unique HE sites in opposite orientation flanking random nucleotide sequences,<br>
u. an oligonucleotide primer site in reverse orientation, v. three non-variable and unique common restriction sites.<br>
4.	The cloning vector plasmid as claimed in claim 3, wherein the modular structure of the sequential elements is ordered to allow introduction of two discrete transgenes, such that the nucleotide sequences so introduced replace random nucleotide sequences.<br>
5.	The cloning vector plasmid as claimed in claim 3, wherein the modular structure of the sequential elements is ordered to allow introduction of two discrete regions of nucleotide sequences such as are needed to induce homologous recombination of a transgene to achieve targeted mutation of a gene.<br>
6.	The cloning vector plasmid as claimed in claim 3, wherein the modular structure of the sequential elements is ordered to allow introduction of two distinct positive or negative selection elements.<br>
7.	A method for constructing a transgene, comprising the steps of<br>
a.	providing the cloning vector plasmid as claimed in claim 1<br>
b.	introducing first nucleotide  sequences  to be included in  the  transgene into a<br>
shuttle vector,<br>
c.	transferring the first nucleotide sequences from the shuttle vector to the   cloning<br>
vector plasmid of claim 1<br><br>
d.	sequentially introducing any other desired nucleotide sequences into another shutde<br>
vector, and<br>
e.	transferring the other desired nucleotide sequences into the cloning vector plasmid<br>
of claim 1.<br>
8.	The method as claimed in claim 7, further comprising transferring the nucleotide<br>
sequences from the cloning vector plasmid of claim 1 to the cloning vector plasmid of claim 3.</td>
			</tr>
		</table>	
		<br>
		<h3>Documents:</h3>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUFic3RyYWN0LSgwNy0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Abstract-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWFic3RyYWN0LnBkZg==" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-abstract.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWFzc2lnbm1lbnQucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-assignment.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNsYWltcy0oMDUtMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Claims-(05-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNsYWltcy0oMDctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Claims-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LUNsYWltcy0oMDctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-Claims-(07-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWNsYWltcy5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-claims.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDMtMDUtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Correspondence Others-(03-05-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Correspondence Others-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMDctMDktMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-Correspondence Others-(07-09-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LUNvcnJlc3BvbmRlbmNlIE90aGVycy0oMjgtMDgtMjAxMikucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-Correspondence Others-(28-08-2012).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycyAxLnBkZg==" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-correspondence-others 1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDUtMDUtMjAwOSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Correspondence-Others-(05-05-2009).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUNvcnJlc3BvbmRlbmNlLU90aGVycy0oMDctMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Correspondence-Others-(07-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWNvcnJlc3BvbmRlbmNlLW90aGVycy5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-correspondence-others.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWRlc2NyaXB0aW9uIChjb21wbGV0ZSkucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-description (complete).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LURyYXdpbmdzLSgwNy0wNi0yMDExKS5wZGY=" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Drawings-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWRyYXdpbmdzLnBkZg==" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-drawings.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWZvcm0tMS5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-form-1.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUZvcm0tMTMtKDA3LTA2LTIwMTEpLnBkZg==" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Form-13-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWZvcm0tMTgucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-form-18.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWZvcm0tMi5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-form-2.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUZvcm0tMy0oMDctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Form-3-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUZvcm0tMy0oMDctMTItMjAxMCkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Form-3-(07-12-2010).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWZvcm0tMy5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-form-3.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWZvcm0tNS5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-form-5.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LUdQQS0oMDctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-GPA-(07-06-2011).pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LWdwYS5wZGY=" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-gpa.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1kZWxucC0yMDA2LXBjdC0zMDQucGRm" target="_blank" style="word-wrap:break-word;">2494-delnp-2006-pct-304.pdf</a></p>
				<p><a href="http://ipindiaonline.gov.in/patentsearch/GrantedSearch/pdfviewer.aspx?AppNo=MjQ5NC1ERUxOUC0yMDA2LVBldGl0aW9uLTEzNy0oMDctMDYtMjAxMSkucGRm" target="_blank" style="word-wrap:break-word;">2494-DELNP-2006-Petition-137-(07-06-2011).pdf</a></p>
		<br>
		<div class="pull-left">
			<a href="256157-shading-dye-granule-its-use-in-a-detergent-formulation-and-process-to-make-it.html">&laquo; Previous Patent</a>
		</div>
		<div class="pull-right">
			<a href="256159-a-method-for-detection-of-collision-with-an-object-or-another-vehicle-using-narrow-light-beams.html">Next Patent &raquo;</a>
		</div>			
	</div><!-- /span8 -->
	<div class="span4">
		<div class="well infobox">
			<table class="table table-condensed">
				<tr>
					<th>Patent Number</th>
					<td>256158</td>
				</tr>
				<tr>
					<th>Indian Patent Application Number</th>
					<td>2494/DELNP/2006</td>
				</tr>
				<tr>
					<th>PG Journal Number</th>
					<td>19/2013</td>
				</tr>
				<tr>
					<th>Publication Date</th>
					<td>10-May-2013</td>
				</tr>
				<tr>
					<th>Grant Date</th>
					<td>09-May-2013</td>
				</tr>
				<tr>
					<th>Date of Filing</th>
					<td>03-May-2006</td>
				</tr>
				<tr>
					<th>Name of Patentee</th>
					<td>INTREXON CORPORATION</td>
				</tr>
				<tr>
					<th>Applicant Address</th>
					<td>1872 PRATT DRIVE, VTCRC LAB SUITE 1400, BLACKSBURG, VIRGINIA 24060, USA.</td>
				</tr>
				<tr>
					<td colspan=2>
								<h5>Inventors:</h5>
								<table class="table">
									<tr>
										<th>#</th>
										<th>Inventor's Name</th>
										<th>Inventor's Address</th>
									</tr>

										<tr>
											<td>1</td>
											<td>REED THOMAS D.</td>
											<td>1512 NORTHVIEW AVENUE, CINCINNATI, OH 45223 USA.</td>
										</tr>
								</table>
					</td>
				</tr>
				<tr>
					<th>PCT International Classification Number</th>
					<td>C12N</td>
				</tr>
				<tr>
					<th>PCT International Application Number</th>
					<td>PCT/US2004/013517</td>
				</tr>
				<tr>
					<th>PCT International Filing date</th>
					<td>2004-05-04</td>
				</tr>
				<tr>
					<td colspan=2>
						<h5>PCT Conventions:</h5>
						<table class="table">
							<tr>
								<th>#</th>
								<th>PCT Application Number</th>
								<th>Date of Convention</th>
								<th>Priority Country</th>
							</tr>

								<tr>
									<td>1</td>
									<td>10/682,764</td>
									<td>2003-10-09</td>
								    <td>U.S.A.</td>
								</tr>

						</table>
					</td>
				</tr>
			</table>
		</div><!-- /well -->
	</div><!-- /span4 -->
</div><!-- /row-fluid -->

        </div>

      </div><!--/row-->

      <footer class="footer">

        <style>
        .allindianpatents-footer { width: 320px; height: 50px; }
        @media(min-width: 500px) { .allindianpatents-footer { width: 468px; height: 60px; } }
        @media(min-width: 800px) { .allindianpatents-footer { width: 728px; height: 90px; } }
        </style>
        <center>
        </center>

        <p>&copy; All Indian Patents, 2013-2021.</p>
        <p>Patent data available in the public domain from Indian Patents Office, Department of Industrial Policy and Promotions, Ministry of Commerce and Industry, Government of India.</p>
      </footer>

    </div> <!-- /container -->

    <!-- Javascripts
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="../assets/application-95f297ff0d8d2015987f04b30593c800.js" type="text/javascript"></script>

    <!-- Start of StatCounter Code for Default Guide -->
    <script type="text/javascript">
    var sc_project=8902313; 
    var sc_invisible=1; 
    var sc_security="3c1f8147"; 
    var scJsHost = (("https:" == document.location.protocol) ?
    "https://secure." : "http://www.");
    document.write("<sc"+"ript type='text/javascript' src='" +
    scJsHost+
    "statcounter.com/counter/counter.js'></"+"script>");
    </script>
    <noscript><div class="statcounter"><a title="web stats"
    href="http://statcounter.com/free-web-stats/"
    target="_blank"><img class="statcounter"
    src="http://c.statcounter.com/8902313/0/3c1f8147/1/"
    alt="web stats"></a></div></noscript>
    <!-- End of StatCounter Code for Default Guide -->

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','http://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-244143-31', 'allindianpatents.com');
      ga('send', 'pageview');

    </script>

  </body>

<!-- Mirrored from www.allindianpatents.com/patents/256158-dna-cloning-vector-plasmids-and-methods-for-their-use by HTTrack Website Copier/3.x [XR&CO'2014], Fri, 05 Apr 2024 09:51:42 GMT -->
</html>
